# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATION DECISION SUMMARY

A. 510(k) Number: K180966

B. Purpose for Submission: To obtain a substantial equivalence determination for the FilmArray Pneumonia Panel

# C. Measurand:

Acinetobacter calcoaceticus-baumannii complex, Enterobacter cloacae complex, Escherichia coli, Haemophilus influenzae, Klebsiella aerogenes, Klebsiella oxytoca, Klebsiella pneumoniae group, Moraxella catarrhalis, Proteus spp., Pseudomonas aeruginosa, Serratia marcescens, Staphylococcus aureus, Streptococcus agalactiae, Streptococcus pneumoniae, Streptococcus pyogenes, Chlamydia pneumoniae, Legionella pneumophila, Mycoplasma pneumoniae, Adenovirus, Coronavirus, Human Metapneumovirus, Human Rhinovirus/Enterovirus, Influenza A, Influenza B, Parainfluenza Virus, Respiratory Syncytial Virus, CTX-M, IMP, KPC, NDM, OXA-48-like, VIM, mecA/C and MREJ.

# D. Type of Test:

Qualitative and quantitative nucleic acid amplification assay

# E. Applicant:

BioFire Diagnostics, LLC

F. Proprietary and Established Names: FilmArray Pneumonia Panel

# G. Regulatory Information:

1. Regulation section:

21 CFR 866.3965 - Device to detect and identify microorganisms and associated resistance marker nucleic acids directly in respiratory specimens

2. Classification: Class II (Special Controls)

3. Product code: QBH

4. Panel: 83-Microbiology

# H. Indications for use:

1. Indications for use(s):

The FilmArray Pneumonia Panel is a multiplexed nucleic acid test intended for use with FilmArray, FilmArray 2.0, or FilmArray Torch systems for the simultaneous detection and identification of multiple respiratory viral and bacterial nucleic acids, as well as select antimicrobial resistance genes, in sputum-like specimens (induced or expectorated sputum, or endotracheal aspirates) or bronchoalveolar lavage (BAL)-like specimens (BAL or mini-BAL) obtained from individuals suspected of lower respiratory tract infection.

The following bacteria are reported semi-qualitatively with bins representing approximately $1 0 ^ { 4 }$ , $1 0 ^ { 5 }$ , $1 0 ^ { 6 }$ , or ${ \geq } 1 0 ^ { 7 }$ genomic copies of bacterial nucleic acid per milliliter (copies $/ \mathrm { m L }$ ) of specimen, to aid in estimating relative abundance of nucleic acid from these common bacteria within a specimen:

<table><tr><td colspan="3">Bacteria reported with bins of 104, 105, 106, or ≥107 copies/mL</td></tr><tr><td>Acinetobacter calcoaceticus- baumannii complex</td><td>Klebsiella oxytoca</td><td>Serratia marcescens</td></tr><tr><td>Enterobacter cloacae complex</td><td>Klebsiella pneumoniae group</td><td>Staphylococcus aureus</td></tr><tr><td>Escherichia coli</td><td>Moraxella catarrhalis</td><td>Streptococcus agalactiae</td></tr><tr><td>Haemophilus influenzae</td><td>Proteus spp.</td><td>Streptococcus pneumoniae</td></tr><tr><td>Klebsiella aerogenes</td><td>Pseudomonas aeruginosa</td><td>Streptococcus pyogenes</td></tr></table>

The following atypical bacteria, viruses, and antimicrobial resistance genes are reported qualitatively:   

<table><tr><td colspan="3">Atypical Bacteria</td></tr><tr><td>Chlamydia pneumoniae</td><td>Legionella pneumophila</td><td>Mycoplasma pneumoniae</td></tr><tr><td></td><td>Viruses</td><td></td></tr><tr><td>Adenovirus Coronavirus Human Metapneumovirus</td><td>Human Rhinovirus/Enterovirus Influenza A Influenza B</td><td>Parainfluenza Virus Respiratory Syncytial Virus</td></tr><tr><td colspan="3">Antimicrobial Resistance Genes</td></tr><tr><td>CTX-M</td><td>NDM</td><td>mecA/C and MREJ</td></tr><tr><td>IMP</td><td>OXA-48-like VIM</td><td></td></tr></table>

The detection and identification of specific viral and bacterial nucleic acids, as well as the estimation of relative abundance of nucleic acid from common bacterial analytes, within specimens collected from individuals exhibiting signs and/or symptoms of a respiratory infection, aids in the diagnosis of lower respiratory infection if used in conjunction with other clinical and epidemiological information. The results of this test should not be used as the sole basis for diagnosis, treatment, or other patient management decisions.

Negative results in the setting of a respiratory illness may be due to infection with pathogens that are not detected by this test, pathogens below the limit of detection, or in the case of bacterial analytes, present at levels below the lowest reported $\bar { 1 } 0 ^ { 4 }$ copies/mL bin. Detection of analytes does not rule out co-infection with other organisms: the agent(s) detected by the FilmArray Pneumonia Panel may not be the definite cause of disease. Additional laboratory testing (e.g., bacterial and viral culture, immunofluorescence, and radiography) may be necessary when evaluating a patient with possible lower respiratory tract infection.

Detection of bacterial nucleic acid may be indicative of colonizing or normal respiratory flora and may not indicate the causative agent of pneumonia. Semi-quantitative bin (copies/mL) results generated by the FilmArray Pneumonia Panel are not equivalent to $\mathrm { C F U / m L }$ and do not consistently correlate with the quantity of bacterial analytes compared to $\mathrm { C F U / m L }$ . For specimens with multiple bacteria detected, the relative abundance of nucleic acids (copies/mL) may not correlate with the relative abundance of bacteria as determined by culture (CFU/mL). Clinical correlation is advised to determine significance of semi- quantitative bin (copies/mL) for clinical management.

The antimicrobial resistance gene detected may or may not be associated with the agent(s) responsible for disease. Negative results for these antimicrobial resistance gene assays do not indicate susceptibility to corresponding classes of antimicrobials, as multiple mechanisms of antimicrobial resistance exist.

Antimicrobial resistance can occur via multiple mechanisms. A “Not Detected” result for a genetic marker of antimicrobial resistance does not indicate susceptibility to associated antimicrobial drugs or drug classes. A “Detected” result for a genetic marker of antimicrobial resistance cannot be definitively linked to the microorganism(s) detected. Culture is required to obtain isolates for antimicrobial susceptibility testing, and FilmArray Pneumonia Panel results should be used in conjunction with culture results for determination of bacterial susceptibility or resistance.

Due to the genetic similarity between human rhinovirus and enterovirus, the test cannot reliably differentiate them. A positive Rhinovirus/Enterovirus result should be followed up using an alternate method (e.g., cell culture or sequence analysis) if differentiation is required.

Culture is required to identify pathogens not detected by the FilmArray Pneumonia Panel, to further speciate analytes in genus, complex, or group results if desired, to identify bacterial pathogens present below the $1 0 ^ { 4 }$ copies/mL bin if desired, and for antimicrobial susceptibility testing.

2. Indication(s) for use: Same as Intended Use

3. Special conditions for use statement(s):

For prescription use only For in vitro diagnostic use only

# Limitations:

For prescription use only. The FilmArray Pneumonia Panel has not been validated for testing of specimens other than unprocessed sputum-like and BAL-like specimens. Contact or treatment of specimens with decontaminating agents (bleach, MycoPrep (NaOH and NALC), $2 \%$ NaOH and $5 \%$ Oxalic acid) can cause false negative results (see Interference section). The performance of the FilmArray Pneumonia Panel has not been established for monitoring treatment of infection. The effect of antibiotic treatment on test performance including semi-quantitative bin results has not been specificall evaluated. Viral and bacterial nucleic acids may persist in vivo independent of organism viability. Detection of organism target(s) does not imply that the corresponding organisms are infectious or are the causative agents for clinical symptoms. The FilmArray Pneumonia Panel results for bacteria are provided as a qualitative Detected/Not Detected result with an associated semi-quantitative bin result of $1 0 ^ { 4 }$ , $1 0 ^ { 5 }$ , $1 0 ^ { 6 }$ , or $\geq 1 0 ^ { 7 }$ copies of genomic nucleic acid per milliliter of specimen. An exact quantitative value is not provided. The semi-quantitative (copies/mL) bin result does not distinguish between nucleic acid from live or dead bacteria.   
• A negative FilmArray Pneumonia Panel result does not exclude the possibility of viral or bacterial infection. Negative test results may occur from the presence of sequence variants in the region targeted by the assay, the presence of inhibitors, technical error, sample mix-up or an infection caused by an organism not detected by the panel. Test results may also be affected by concurrent antiviral/antibacterial therapy or levels of organism in the specimen that are below the limit of detection for the test or below the reportable level for bacterial analytes. Negative results should not be used as the sole basis for diagnosis, treatment, or other patient management decisions. Concomitant culture of specimens is required with the FilmArray Pneumonia Panel. Culture is needed for recovery of isolates and antimicrobial susceptibility testing, as well as further speciation of genus, complex, or group level results (if desired).   
• Due to the genetic similarity between human rhinovirus and enterovirus, the FilmArray Pneumonia Panel cannot reliably differentiate them. A FilmArray Pneumonia Panel Human Rhinovirus/Enterovirus Detected

result should be followed-up using an alternate method (e.g. cell culture or sequence analysis) if differentiation between the viruses is required.

The in-silico analyses performed to predict amplification and detection of organisms and antimicrobial resistance genes were based on a comparison of target gene sequences available in GenBank to FilmArray Pneumonia Panel primer sequences. In-silico analyses were performed between January 2016 and January 2018. Entries of new sequences added to the database after these dates have not been evaluated. Additional limitations on reactivity may be identified as new sequence data are deposited and/or as new sequence variants emerge.   
• Based on in-silico analysis, the MREJ assay (which is only reported if Staphylococcus aureus is detected and the mecA/C assay is also positive) is predicted to have impaired reactivity or to be non-reactive with MREJ types ix, xv and xviii, as well as types xix and xx (associated with methicillin-sensitive S. aureus; MSSA), and MREJ sequences annotated from non-aureus Staphyloccoccus species and non-Staphyloccci such as Bacillus cereus, Bacillus thuringiensis, Macrococcus caseolyticus, Clostridium acidurici, and Rummeliibacillus stabekisii. Positive and negative predictive values are highly dependent on prevalence. False negative test results are more likel during peak activity when prevalence of disease is high. False positive test results are more likely during periods when prevalence is moderate to low. Performance characteristics for influenza A were established during the 2016-2017 influenza season. When other novel influenza A viruses are emerging, performance characteristics may vary. If infection with a novel influenza A virus is suspected based on current clinical and epidemiological screening criteria recommended by public health authorities, specimens should be collected with appropriate infection control precautions for novel virulent influenza viruses and sent to state or local health departments for testing. Viral culture should not be attempted in these cases unless a BSL $^ { 3 + }$ facility is available to receive and culture specimens.   
• Due to the small number of positive specimens collected for certain organisms during the prospective clinical study, performance characteristics for several analytes in one or both matrices were primarily established using archived and/or contrived specimens as detailed in the Clinical Performance section.

4. Special instrument requirements:

The FilmArray Pneumonia Panel is performed on FilmArray, FilmArray 2.0, or FilmArray Torch systems.

# I. Device Description:

The FilmArray Pneumonia Panel is designed to simultaneously identify 26 potential pathogens of lower respiratory tract infection (LRTI) and associated antimicrobial resistance (AMR) genes from a sputum-like (induced and expectorated sputum as well as endotracheal aspirate, ETA) or bronchoalveolar lavage (BAL)-like (BAL and miniBAL) specimens obtained from individuals with signs and/or symptoms of lower respiratory tract infection in a time ( ${ \bf \tilde { \rho } } ^ { \prime } { \bf \tilde { \rho } } ^ { 1 }$ hour). The FilmArray Pneumonia Panel is compatible with BioFire Diagnostics’ (BioFire) PCR-based in vitro diagnostic FilmArray, FilmArray 2.0, and FilmArray Torch systems for infectious disease testing. A specific software module (i.e. FilmArray Pneumonia Panel pouch module) is used to perform FilmArray Pneumonia Panel testing on these systems.

A test is initiated by loading Hydration Solution into one port of the FilmArray pouch and a sputum-like or BAL-like sample mixed with the provided Sample Buffer into the other port of the FilmArray Pneumonia Panel pouch and placing it in a FilmArray instrument. The pouch contains all the reagents required for specimen testing and analysis in a freeze-dried format; the addition of Hydration Solution and Sample/Buffer Mix rehydrates the reagents. After the pouch is prepared, the FilmArray Software guides the user though the steps of placing the pouch into the instrument, scanning the pouch barcode, entering the sample identification, and initiating the run.

The FilmArray instrument contains a coordinated system of inflatable bladders and seal points, which act on the pouch to control the movement of liquid between the pouch blisters. When a bladder is inflated over a reagent blister, it forces liquid from the blister into connecting channels. Alternatively, when a seal is placed over a connecting channel it acts as a valve to open or close a channel. In addition, electronically-controlled pneumatic pistons are positioned over multiple plungers in order to deliver the rehydrated reagents into the blisters at the appropriate times. Two Peltier devices control heating and cooling of the pouch to drive the PCR reactions and the melt curve analysis.

Nucleic acid extraction occurs within the FilmArray pouch using mechanical and chemical lysis followed by purification using standard magnetic bead technology. After extracting and purifying nucleic acids from the unprocessed sample, the FilmArray performs a nested multiplex PCR that is executed in two stages. During the first stage, the FilmArray performs a single, large volume, highly multiplexed reverse transcription PCR (rt-PCR) reaction. The products from first stage PCR are then diluted and combined with a fresh, primer-free master mix and a fluorescent double stranded DNA binding dye (LC Green Plus, BioFire Diagnostics). The solution is then distributed to each well of the array. Array wells contain sets of primers designed specifically to amplify sequences internal to the PCR products generated during the first stage PCR reaction. The 2nd stage PCR, or nested PCR, is performed in singleplex fashion in each well of the array. At the end of the 2nd stage PCR, the array is interrogated by melt curve analysis for the detection of signature amplicons denoting the presence of specific targets. A digital camera placed in front of the 2nd stage PCR captures fluorescent images of the PCR reactions and software interprets the data.

The FilmArray Software automatically interprets the results of each DNA melt curve analysis and combines the dat with the results of the internal pouch controls to provide a test result for each organism on the panel.

# Materials provided in each FilmArray Pneumonia Panel kit:

Each kit contains sufficient reagents to test 30 samples (30-test kit; RFIT-ASY-0144) or 6 samples (6-test kit; RFITASY-0145):

Individually-packaged FilmArray Pneumonia Panel pouches Single-use $( 1 . 0 \mathrm { m L }$ ) Sample Buffer ampoules

Single-use pre-filled $( 1 . 5 \mathrm { m L }$ ) Hydration Injection Vials (blue) Single-use Sample Injection Vials (red) Individually-packaged Transfer Pipettes

# Materials required but not provided:

• FilmArray system including: - FilmArray 2.0 or FilmArrayTouch and accompanying software - FilmArray Pouch Loading Station   
• $10 \%$ bleach solution

# Interpretation of Results

When PCR2 is complete, the FilmArray instrument performs a DNA melting analysis on the PCR products and measures the fluorescence signal generated in each well (for more information see appropriate FilmArray Operator’s Manual). The FilmArray Software then performs several analyses and assigns a final assay result. The steps in the analyses are described below.

# Analysis of Melt Curves

The FilmArray Software evaluates the DNA melt curve for each well of the PCR2 array to determine if a PCR product was present in that well. If the melt profile indicates the presence of a PCR product, then the analysis software calculates the melting temperature $\mathrm { ( T m ) }$ of the curve and compares it against the expected Tm range for the assay. If the software determines that the Tm of the curve is within the assay-specific Tm range, the melt curve is called positive. If the software determines that the Tm of the curve is not in the appropriate Tm range, the melt curve is called negative.

# Analysis of Replicates

Once positive melt curves have been identified, the software evaluates the replicates for each assay to determine the assay result. For an assay to be called positive, two associated melt curves must be called positive, and both Tms must be similar. Assays that do not meet these criteria are called negative.

# Analysis of Assay Results for Bacteria

The assays in the FilmArray Pneumonia Panel for detection of bacteria that are reported semi-quantitatively are designed to amplify genes that are present in single copies within the chromosome of the target bacterium and are used to estimate genomic copies of bacterial nucleic acid per milliliter (copies $/ \mathrm { m L }$ ) of specimen. The FilmArray Software calculates an approximate value for each gene target based on real-time PCR amplification data relative to the QSM (internal reference of known quantity). Assays with no measurable

amplification or a value below $1 0 ^ { 3 . 5 }$ copies/mL are called negative. Assays with a value equal to or greater than $1 0 ^ { 3 . 5 }$ copies/mL are called positive.

# Organism and Antimicrobial Resistance Gene Interpretation

Each positive and negative assay result is interpreted by the FilmArray Software to provide results for the identification of specific bacteria, atypical bacteria, viruses, and antimicrobial resistance (AMR) genes as shown in the Table below. For most analytes detected by the FilmArray Pneumonia Panel, interpretations are based on the result of a single assay. However, results for Staphylococcus aureus, Adenovirus, and the AMR genes require interpretation based on more than one assay result, as discussed in the relevant sections below.

<table><tr><td rowspan=1 colspan=1>Bacteria  Quantitative Results</td><td rowspan=1 colspan=1>Antimicrobial Resistance Genes</td></tr><tr><td rowspan=1 colspan=1>Acinetobacter calcoaceticus-baumannii complex</td><td rowspan=1 colspan=1>blaCTX-M (Extended spectrum beta-lactamase</td></tr><tr><td rowspan=1 colspan=1>Enterobacter cloacae complex</td><td rowspan=1 colspan=1>blaIMP (Carbapenem resistance)</td></tr><tr><td rowspan=1 colspan=1>Escherichia coli</td><td rowspan=1 colspan=1>blaKPC (Carbapenem resistance)</td></tr><tr><td rowspan=1 colspan=1>Haemophilus influenzae</td><td rowspan=1 colspan=1>mecA/mecC and MREJ (Methicillin resistance)</td></tr><tr><td rowspan=1 colspan=1>Klebsiella aerogene</td><td rowspan=1 colspan=1>blaNDM (Carbapenem resistance)</td></tr><tr><td rowspan=1 colspan=1>Klebsiella oxytoca</td><td rowspan=1 colspan=1>blaOxa48-like (Carbapenem resistance)</td></tr><tr><td rowspan=1 colspan=1>Klebsiella pneumoniae group</td><td rowspan=1 colspan=1>blaVIM (Carbapenem resistance)</td></tr><tr><td rowspan=1 colspan=1>Moraxella catarrhalis</td><td rowspan=1 colspan=1>Viruses</td></tr><tr><td rowspan=1 colspan=1>Proteus spp.</td><td rowspan=1 colspan=1>Adenovirus</td></tr><tr><td rowspan=1 colspan=1>Pseudomonas aeruginosa</td><td rowspan=1 colspan=1>Coronavirus</td></tr><tr><td rowspan=1 colspan=1>Serratia marcescens</td><td rowspan=1 colspan=1>Human Metapneumovirus</td></tr><tr><td rowspan=1 colspan=1>Streptococcus agalactiae</td><td rowspan=1 colspan=1>Human Rhinovirus/Enterovirus</td></tr><tr><td rowspan=1 colspan=1>Streptococcus pneumoniae</td><td rowspan=1 colspan=1>Influenza A</td></tr><tr><td rowspan=1 colspan=1>Streptococcus pyogenes</td><td rowspan=1 colspan=1>Influenza B</td></tr><tr><td rowspan=1 colspan=1>Bacteria (Atypical) - Qualitative Results</td><td rowspan=1 colspan=1>Parainfluenza Virus</td></tr><tr><td rowspan=1 colspan=1>Chlamydia pneumoniae</td><td rowspan=1 colspan=1>Respiratory Syncytial Virus</td></tr><tr><td rowspan=1 colspan=1>Legionella pneumophila</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Mycoplasma pneumoniae</td><td rowspan=1 colspan=1></td></tr></table>

# Interpretations and Semi-quantitative Bin Results for Bacteria

The FilmArray Pneumonia Panel provides a Detected or Not Detected result as well as a semi-quantitative bin result ( $1 0 ^ { 4 }$ copies/mL, $1 0 ^ { 5 }$ copies/mL, $1 0 ^ { 6 }$ copies/mL or ${ \geq } 1 0 ^ { 7 }$ copies/mL) for most bacteria. The bin result represents the approximate number of specific bacterial genomes in the specimen and is intended to provide a simple assessment of relative quantities of different bacteria in a lower respiratory specimen based on a molecular method.

For bacteria, negative assays (no measurable amplification or value less than $1 0 ^ { 3 . 5 }$ copies/mL) are reported as Not Detected. Positive assays are reported as Detected and a bin result is assigned based on the assay value. Each bin is defined by discrete upper and lower limits spanning a 1-log range of values (see Table 1) such that the bin result reflects the assay value within the nearest $\pm 0 . 5$ -log.

Table 1: FilmArray Pneumonia Panel Bin Results for Bacteria   

<table><tr><td rowspan=1 colspan=2>Assay Result</td><td rowspan=1 colspan=2>Reported Result and Bin Result</td></tr><tr><td rowspan=1 colspan=1>Negative OR</td><td rowspan=1 colspan=1>&lt;103.5 copies/mL</td><td rowspan=1 colspan=2>Not Detected</td></tr><tr><td rowspan=1 colspan=1>Positive AND</td><td rowspan=1 colspan=1>≥103.5  &lt;1045 copies/mL</td><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>104 copies/mL</td></tr><tr><td rowspan=1 colspan=1>Positive AND</td><td rowspan=1 colspan=1>≥104.5 - &lt;105.5 copies/mL</td><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>105 copies/mL</td></tr><tr><td rowspan=1 colspan=1>Positive AND</td><td rowspan=1 colspan=1>≥105.5 - &lt;106.5 copies/mL</td><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>106 copies/mLD</td></tr><tr><td rowspan=1 colspan=1>Positive AND</td><td rowspan=1 colspan=1>≥106.5 copies/mL</td><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>≥107 copies/mL</td></tr></table>

# Staphylococcus aureus

The FilmArray Pneumonia Panel pouch contains two different assays (Saureus1 and Saureus2) for the detection of Staphylococcus aureus. The FilmArray Software interprets each of these assays independently (as described above) and if one or a combination of the assays is positive, the result will be Staphylococcusaureus Detected with the appropriate bin result. If both assays are negative the result will be Staphylococcusaureus Not Detected.

# Interpretations for Atypical Bacteria and Viruses

Results for most Atypical Bacteria and Viruses are reported qualitatively as Detected or Not Detected based on an individual corresponding assay result. If the assay is positive the result will be Detected, and if the assay is negative, the result will be Not Detected.

# Adenovirus

The FilmArray Pneumonia Panel pouch contains three different assays (Adenovirus2, Adenovirus3, and Adenovirus7) for the detection of all species and serotypes of Adenovirus. The FilmArray Software interprets each of these assays independently (as described above) and the results are combined as a final result for the virus. If one or any combination of assays is positive, the result will be Adenovirus Detected. If all assays are negative, the result will be Adenovirus Not Detected.

Results for AMR genes are also reported qualitatively (Detected/Not Detected) based on corresponding assays, but only if an applicable bacterium (i.e. potential carriers of the AMR gene; Table 2) is also detected $( \geq 1 0 ^ { \bar { 3 } . 5 }$ copies/mL) in the sample.

The results for each of the antimicrobial resistance genes will be listed as either:

Detected – when an applicable bacterium is detected AND the antimicrobial resistance gene assay(s) are positive.   
Not Detected – when an applicable bacterium is detected AND the antimicrobial resistance gene assay(s) are negative.   
N/A – when all applicable bacteria are Not Detected, regardless of the result for the antimicrobial resistance gene assay(s).

Table 2: Antimicrobial Resistance (AMR) Genes and Applicable Organisms   

<table><tr><td>AMR Gene Result</td><td>Applicable Bacteria</td></tr><tr><td>mecA/C and MREJ</td><td>Staphylococcus aureus</td></tr><tr><td>CTX-M IMP KPC NDM VIM</td><td>Acinetobacter calcoaceticus-baumannii complex Enterobacter cloacae complex Escherichia coli Klebsiella aerogenes Klebsiella oxytoca Klebsiella pneumoniae group Proteus spp. Pseudomonas aeruginosa Serratia marcescens</td></tr><tr><td>OXA-48-like</td><td>Enterobacter cloacae complex Escherichia coli Klebsiella aerogenes Klebsiella oxytoca Klebsiella pneumoniae group Proteus spp. Serratia marcescens</td></tr></table>

Each AMR gene result is associated with a single corresponding assay except for the mecA/C and MREJ result, which is dependent on both the mecA/C assay and the MREJ assay. Detection of both Staphylococcus aureus and the mecA/C and MREJ markers is indicative of Methicillin Resistant Staphylococcus Aureus (MRSA).

# Run Information

The Run Information section is displayed at the top of both pages of the test report. It provides information about the sample and the run including: Sample ID, Protocol (sample type), pouch information (Pouch Type, Lot Number, and Serial Number), Run Date, Run Status (Completed, Incomplete, Aborted, Instrument Error, Instrument Communication Error, or Software Error), , the identity of the operator who performed the test, and the instrument used to perform the test. Control results are reported as Passed, Failed, or Invalid. The Table 3 below provides additional information for each of the possible control field results.

Table 3: Interpretation of Controls Field on the FilmArray Pneumonia Panel Test Report   

<table><tr><td rowspan=1 colspan=1>ControlResult</td><td rowspan=1 colspan=1>Explanation</td><td rowspan=1 colspan=1>Action</td></tr><tr><td rowspan=1 colspan=1>Passed</td><td rowspan=1 colspan=1>The run was successfully completedANDBoth pouch controls weresuccessful.</td><td rowspan=1 colspan=1>NoneReport the results provided on the test report.</td></tr><tr><td rowspan=1 colspan=1>Failed</td><td rowspan=1 colspan=1>The run was completedBUTAt least one of the pouch controls(RNA Process Control and/orQSM) failed.</td><td rowspan=1 colspan=1>Repeat the test using a new pouch.If the error persists, contact CustomerTechnical Support for further instruction.</td></tr><tr><td rowspan=1 colspan=1>Invalid</td><td rowspan=1 colspan=1>The controls are invalid because therun did not complete.(Typically this indicates a softwareor hardware error).</td><td rowspan=1 colspan=1>Note any error codes displayed during the runand the Run Status field in the RunInformation section of the report. Refer to theappropriate FilmArray Operator&#x27;s Manual orcontact Customer Technical Support forfurther instruction.Once the error is resolved, repeat the test orrepeat the test using another instrument.</td></tr></table>

# Detection Summary

The Detection Summary section is displayed on the first page of the report and lists the Detected results under each category (Bacteria, Antimicrobial Resistance Genes, Atypical Bacteria, and Viruses), including the semiquantitative ‘Bin (copies/mL)’ results for Bacteria. If there are no Detected results in a specific category, the result shown is Detected: None.

The Results Summary is displayed on the second page of the report and provides a full list of test results for each organism and antimicrobial resistance gene including the ‘Bin (copies/mL)’ result for Bacteria. Possible results for each organism are Detected, Not Detected, Invalid, and N/A. The Table provides an explanation for each interpretation and any follow-up necessary to obtain a final result.

Table 4: Reporting of Results and Required Actions   

<table><tr><td>Result</td><td>Explanation</td><td>Action</td></tr><tr><td>Detected</td><td>The run was successfully completed AND The pouch controls were successful (Passed) AND</td><td>Report results.</td></tr><tr><td>Not Detected</td><td>The assay(s) for the organism were POSITIVEa The run was successfully completed AND The pouch controls were successful (Passed) AND</td><td>Report results.</td></tr><tr><td>Invalid</td><td>The assay(s) for the organism were NEGATIVEb The pouch controls were not successful (Failed) OR The run was not successful (Run Status displayed as: Aborted, Incomplete, Instrument Error or Software Error)</td><td>SeeTable 3 for instruction.</td></tr><tr><td>N/A (Antimicrobial Resistance Genes only</td><td>The run was successfully completed AND The pouch controls were successful (Passed) AND The assay(s) for the organism(s) associated with the antimicrobial resistance gene were NEGATIVE so the results of the antimicrobial resistance gene are not applicable to the test results.</td><td>Report results.</td></tr></table>

a. For bacteria, the organism calculated value must be greater than or equal to $\overline { { 1 0 ^ { 3 . 5 } } }$ copies $/ \mathrm { m L }$ for the assay to be POSITIVE. b. For bacteria, a NEGATIVE assay result may indicate no amplification or amplification with an organism calculated value less than $1 0 ^ { 3 . 5 }$ copies $/ \mathrm { m L }$

# J. Substantial Equivalence Information:

1. Predicate device name(s): Curetis Unyvero LRT Application

2. Predicate 510(k) number(s):

DEN170047

3. Comparison with predicate:

<table><tr><td colspan="3">Similarities</td></tr><tr><td>Item</td><td>New Device: FilmArray Pneumonia Panel (K180966)</td><td>Predicate: Curetis Unyvero LRT Application (DEN170047)</td></tr><tr><td rowspan="4">Indication for Use</td><td>The FilmArray Pneumonia Panel is a multiplexed nucleic acid test intended for use with FilmArray, FilmArray 2.0, or FilmArray Torch systems for the simultaneous detection and identification of multiple respiratory viral and bacterial nucleic acids, as well as select antimicrobial resistance genes, in sputum-like specimens (induced or expectorated sputum, or endotracheal aspirates) or bronchoalveolar lavage (BAL)-like specimens (BAL or mini- BAL) obtained from individuals suspected of lower respiratory tract infection.</td><td>The Unyvero LRT Application is a qualitative nucleic acid multiplex test intended for the simultaneous detection and identification of nucleic acid sequences from the following microorganisms and antibiotic resistance markers in endotracheal aspirates from adult hospitalized patients with suspected lower respiratory tract infections. Mircoorganisms: Acinetobacter spp., Chlamydia pneumoniae, Citrobacter freundii, Escherichia coli, Enterobacter cloacae complex, Haemophilus influenzae, Klebsiella oxytoca, Klebsiella</td></tr><tr><td>The following bacteria are reported semi-qualitatively with bins representing approximately 104, 105, 106, or ≥107 genomic copies of bacterial nucleic acid per milliliter (copies/mL) of specimen, to aid in estimating relative abundance of nucleic acid from these common bacteria within a specimen: Bacteria reported with bins of 104, 105, 106, or ≥107 copies/mL: Acinetobacter calcoaceticus- baumannii complex, Enterobacter cloacae complex, Escherichia coli, Haemophilus influenzae, Klebsiella aerogenes, Klebsiella oxytoca, Klebsiella pneumoniae group, Moraxella catarrhalis, Proteus spp., Pseudomonas aeruginosa, Serratia marcescens, Staphylococcus aureus,</td><td>pneumoniae, Klebsiella variicola, Legionella pneumophila, Moraxella catarrhalis, Morganella morganii, Mycoplasma pneumoniae, Proteus spp, Pseudomonas aeruginosa, Serratia marcescens, Staphylococcus aureus, Stenotrophomonas maltophilia, Streptococcus pneumoniae Associated Antimicrobial Resistance Markers: ctx-M (blaCTX-M, subgroup 1 only), kpc (blaKPC), mecA, ndm (blaNDM), oxa-23 (blaOXA-23), oxa-24 (blaOXA-24),</td></tr><tr><td>Streptococcus agalactiae, Streptococcus pneumoniae, Streptococcus pyogenes Atypical Bacteria: Chlamydia pneumoniae, Legionella pneumophila, Mycoplasma pneumoniae Antimicrobial Resistance Genes: CTX-M, IMP, KPC, mecA/C + MREJ, NDM, Oxa48-like, VIM Viruses: Adenovirus, Coronavirus, Human Metapneumovirus, Human Rhinovirus/Enterovirus, Influenza A, Influenza B,</td><td>0xa-48 (blaOXA-48), oxa-58 (blaOXA-58), tem (blaTEM), vim (blaVIM) The Unyvero LRT Application performed on the Unyvero System is indicated as an aid in the diagnosis of lower respiratory tract infection in adult hospitalized patients with signs and symptoms of lower respiratory infection; results should be used in conjunction with other clinical and laboratory findings. As tracheal aspirates</td></tr><tr><td>Parainfluenza virus, Respiratory Syncytial virus The detection and identification of specific viral and bacterial nucleic acids, as well as the estimation of relative abundance of nucleic acid from common bacterial analytes, within specimens collected from individuals exhibiting signs and/or symptoms of a</td><td>commonly contain colonizing microorganisms, detection of Unyvero LRT microbial targets does not indicate that the microorganism is the cause of the disease. Unyvero positive results do not rule out co-infection with microorganisms not detected by the Unyvero LRT Application. Negative results do not preclude lower respiratory infection, as the causative agent may be a microorganism not detected by this test.</td></tr><tr><td colspan="2">Similarities respiratory infection, aids in the diagnosis of lower respiratory A negative result for any antibiotic resistance marker does not</td></tr><tr><td>management decisions. resistance.</td><td>infection if used in conjunction with other clinical and indicate that detected microorganisms are susceptible to applicable epidemiological information. The results of this test should not be antimicrobial agents. Detected resistance markers cannot be used as the sole basis for diagnosis, treatment, or other patient definitively linked to specific microorganisms, and may be present in organisms that are not detected by the Unyvero LRT Application such as organisms present as colonizing or normal Negative results in the setting of a respiratory illness may be due flora. to infection with pathogens that are not detected by this test, pathogens below the limit of detection, or in the case of bacterial Microbiology cultures of aspirates should be performed to obtain analytes, present at levels below the lowest reported 104 copies/mL isolates for species identification and antimicrobial susceptibility bin. Detection of analytes does not rule out co-infection with other testing, to differentiate quantities of identified microorganisms as organisms: the agent(s) detected by the FilmArray Pneumonia well as normal flora present in the specimen and to identify Panel may not be the definite cause of disease. Additional potential microorganisms not targeted by the Unyvero LRT laboratory testing (e.g. bacterial and viral culture, Application. immunofluorescence, and radiography) may be necessary when evaluating a patient with possible lower respiratory tract infection. Detection of bacterial nucleic acid may be indicative of colonizing or normal respiratory flora and may not indicate the causative agent of pneumonia. Semi-quantitative bin (copies/mL) results generated by the FilmArray Pneumonia Panel are not equivalent to CFU/mL and do not consistently correlate with the quantity of bacterial analytes compared to CFU/mL. For specimens with multiple bacteria detected, the relative abundance of nucleic acids (copies/mL) may not correlate with the relative abundance of bacteria as determined by culture (CFU/mL). Clinical correlation is advised to determine significance of semi-quantitative bin (copies/mL) for clinical management. The antimicrobial resistance gene detected may or may not be associated with the agent(s) responsible for disease. Negative results for these antimicrobial resistance gene assays do not indicate susceptibility to corresponding classes of antimicrobials, as multiple mechanisms of antimicrobial resistance exist. Antimicrobial resistance can occur via multiple mechanisms. A "Not Detected" result for a genetic marker of antimicrobial resistance does not indicate susceptibility to associated antimicrobial drugs or drug classes. A "Detected" result for a genetic marker of antimicrobial resistance cannot be definitively linked to the microorganism(s) detected. Culture is required to obtain isolates for antimicrobial susceptibility testing, and FilmArray Pneumonia Panel results should be used in conjunction with culture results for the determination of susceptibility or Due to the genetic similarity between human rhinovirus and</td></tr></table>

<table><tr><td rowspan=1 colspan=3>Similarities</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>enterovirus, the test cannot reliably differentiate them. A positiveRhinovirus/Enterovirus result should be followed up using analternate method (e.g., cell culture or sequence analysis) ifdifferentiation is required.Culture is required to identify pathogens not detected by theFilmArray Pneumonia Panel, to further speciate analytes in genus,complex, or group results if desired, to identify bacterial pathogenspresent below the 104 copies/mL bin if desired, and forantimicrobial susceptibility testing.</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>TechnologicalPrinciples</td><td rowspan=1 colspan=1>Multiplex nucleic acid</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=1>TestInterpretation</td><td rowspan=1 colspan=1>Automated test interpretation and report generation.</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=1>Controls</td><td rowspan=1 colspan=1>Two controls are included in each reagent pouch to controlfor sample processing and both stages of PCR and meltanalysis.</td><td rowspan=1 colspan=1>Internal control (a synthetic gene without any homology toknown sequences) is processed in every chamber in order toverify the DNA purification, amplification, arrayhybridization, and detection</td></tr><tr><td rowspan=1 colspan=1>UserComplexity</td><td rowspan=1 colspan=1>Moderate/Low</td><td rowspan=1 colspan=1>Same</td></tr></table>

<table><tr><td colspan="3" rowspan="1">Differences</td></tr><tr><td colspan="1" rowspan="1">Item</td><td colspan="1" rowspan="1">New Device: FilmArray Pneumonia Panel(K180966)</td><td colspan="1" rowspan="1">Predicate: Curetis Unyvero LRT Application(DEN170047)</td></tr><tr><td colspan="1" rowspan="1">SpecimenTypes</td><td colspan="1" rowspan="1">Sputum-like (induced or expectorated sputum,endotracheal aspirates) and BAL- like (BAL or mini-BAL)specimens</td><td colspan="1" rowspan="1">Endotracheal Aspirate specimens</td></tr><tr><td colspan="1" rowspan="1">Analyte</td><td colspan="1" rowspan="1">DNA/RNA</td><td colspan="1" rowspan="1">DNA only</td></tr><tr><td colspan="1" rowspan="1">Resulttypes</td><td colspan="1" rowspan="1">Detection with bin values (in 1-log rounded copies/mLbins) for BacteriaQualitative for Atypical Bacteria, Antimicrobial ResistanceGenes, and Viruses</td><td colspan="1" rowspan="1">Qualitative for all analytes</td></tr><tr><td colspan="1" rowspan="1">Instrumentation</td><td colspan="1" rowspan="1">FilmArray, FilmArray 2.0, or FilmArray Torch</td><td colspan="1" rowspan="1">Unyvero System</td></tr><tr><td colspan="1" rowspan="1">Time toresult</td><td colspan="1" rowspan="1">About 1 hour</td><td colspan="1" rowspan="1">About 4-5 hours</td></tr><tr><td colspan="1" rowspan="1">ReagentStorage</td><td colspan="1" rowspan="1">Room temperature</td><td colspan="1" rowspan="1">Store the Unyvero LRT Cartridge kit at 15°- 25°C. Do notrefrigerate.Store the Unyvero Master Mix Tubes frozen at -20°C orbelow.Thawed Master Mix tubes may be stored at 2-8°C for up to7 days. Do not re-freeze</td></tr></table>

# K. Standard/Guidance Document Referenced:

Guidance for Industry and Food and Drug Administration Staff – Highly Multiplexed Microbiological/Medical Countermeasure In Vitro Nucleic Acid Based Diagnostic Devices, (August 27, 2014) Guidance for Industry and FDA Staff - Class II Special Controls Guidance Document: Respiratory Viral Panel Multiplex Nucleic Acid Assay (October 9, 2009) Guidance for Industry and FDA Staff - Class II Special Controls Guidance Document: Testing for Detection and Differentiation of Influenza A Virus Subtypes Using Multiplex Nucleic Acid Assays (October 9, 2009) Guidance for Industry and FDA Staff - Class II Special Controls Guidance Document: Testing for Human Metapneumovirus (hMPV) Using Nucleic Acid Assays (October 9, 2009) Guidance for Industry and FDA Staff- Statistical Guidance on Reporting Results from Studies Evaluating Diagnostic Tests, (March 13, 2007) Guidance for Sponsors, Institutional Review Boards, and Food and Drug Administrative Staff – Guidance on Informed Consent for In Vitro Diagnostic Device Studies Using Leftover Human Specimens that are Not Individually Identifiable, (April 25, 2006) Evaluation of Stability of In Vitro Diagnostic Reagents; Approved Guideline, Clinical and Laboratory Standards Institute (CLSI) – First Edition, EP25-A (September 23, 2009). Interference Testing in Clinical Chemistry; Approved Guideline, Clinical and Laboratory Standards Institute (CLSI) – Second Addition, EP07-A2 (November, 2005). Molecular Diagnostic Methods for Infectious Diseases; Approved Guideline, Clinical and Laboratory Standards Institute (CLSI), MM3-A2 (February 2006) User Protocol for Evaluation of Qualitative Test Performance; Approved Guideline, Clinical and Laboratory Standards Institute (CLSI)– Second Edition, EP12-A2 (January, 2008) ISO 13485:2016/EN ISO 13485:2016, ‘Medical devices – Quality management systems – Requirements for regulatory purposes’. ISO 14971:2007 ‘Medical devices – Application of risk management to medical devices’. EN 13612:2002, ‘Performance evaluation of in vitro diagnostic devices’. EN 13641:2002, ‘Elimination or reduction of risk of infection related to in vitro diagnostic reagents’.

• EN 62366:2008/IEC 62366-1:2015, ‘Medical Devices-Application of usability engineering to medical devices’.   
• EN ISO 23640:2015, ‘In vitro diagnostic medical devices – Evaluation of stability of in vitro diagnostic reagents’.   
• Guidance for the Content of Premarket Submissions for Software Contained in Medical Devices, FDA Guidance Document (May 11, 2005) Guidance for Industry, FDA Reviewers and Compliance on Off-The-Shelf Software Use in Medical Devices (September 9, 1999) General Principle of Software Validation; Final Guidance for Industry and FDA Staff (January 11, 2002) Content of Premarket Submissions for Management of Cybersecurity in Medical Devices (October 02, 2014) Postmarket Management of Cybersecurity in Medical Devices (December 28, 2016)   
• ISO 62304:2006, ‘Medical device software – Software life-cycle processes’ – IEC 62304:2006, November 27, 2008.   
• Use of Symbols on Labels and in Labeling of In Vitro Diagnostic Devices Intended for Professional Use, FDA Guidance Document (November 30, 2004)   
• Guidance for Industry and FDA on Alternative to Certain Prescription Device Labeling Requirements (January 1, 2000)   
• ISO 15223-1:2016, ‘Medical Devices – Symbols to be used with medical device labels, labeling and information to be supplied – Part 1: General requirements’.   
C EN ISO 18113-1:2011, ‘In vitro diagnostic medical devices – Information supplied by the manufacturer (labeling) – Part 1: Terms, definition and general requirements’.   
• EN ISO 18113-2:2011, ‘In vitro diagnostic medical devices – Information supplied by the manufacturer (labeling) – Part 2: In vitro diagnostic reagents for professional use’.

# L. Test Principle:

The FilmArray Pneumonia Panel pouch is a closed system disposable that stores all the necessary reagents for sample preparation, reverse transcription, polymerase chain reaction (PCR), and detection in order to isolate, amplify, and detect nucleic acid from multiple lower respiratory pathogens within a single bronchoalveolar lavage (BAL)-like (BAL or mini-BAL) or sputum-like (sputum or ETA) specimen. After sample collection, the user injects hydration solution and sample combined with sample buffer into the pouch, places the pouch into a FilmArray instrument, and starts a run. The entire run process takes about one hour. Additional detail can be found in the appropriate FilmArray Operator’s Manual.

During a run, the FilmArray system:

• Lyses the sample by agitation (bead beading).

• Extracts and purifies all nucleic acids from the sample using magnetic bead technology.

• Performs nested multiplex PCR by:

o First performing reverse transcription and a single, large volume, massively-multiplexed reaction (PCR1)   
o Then performing multiple singleplex second-stage PCR reactions (PCR2) to amplify sequences within the PCR1 products

• Uses endpoint melting curve data to detect and generate a result for each target on the FilmArray Pneumonia Panel array.

For the FilmArray Pneumonia Panel, the system also uses real-time amplification data from the assays relative to a Quantified Standard Material (QSM) included in the pouch to provide an estimated value in genomic copies per milliliter (copies/mL) for bacterial analytes.

# M. Performance Characteristics (if/when applicable):

# 1. Analytical performance:

a. Analytical Sensitivity:

A limit of detection (LoD) was established for atypical bacteria and viruses detected by the FilmArray Pneumonia Panel. LoD was estimated by testing dilutions of contrived BAL or sputum samples containing known concentrations of organisms. Confirmation of LoD was achieved by testing at least 20 replicates per samples type on FilmArray, FilmArray 2.0 and FilmArray Torch systems (60 replicates total per sample type). LoD concentration was confirmed when the analyte was detected in at least $9 5 \%$ of the replicates tested.

The confirmed LoD for each atypical bacterium or virus detected by the panel (including a LoD for more than one isolate of the more genetically diverse viruses) is listed in Table 5. The LoD concentration is based on quantification of each culture in viable units $\mathrm { ( T C I D 5 0 / m L }$ or $\mathrm { C F U / m L }$ ) and a corresponding molecular LoD concentration (DNA or RNA copies $/ \mathrm { m L }$ ) is provided based on quantitative real-time or digital PCR.

Table 5: Summary of Limit of Detection (LoD) for FilmArray Pneumonia Panel Atypical Bacteria and Viruses   

<table><tr><td colspan="4" rowspan="1">Bacteria and Viruses</td></tr><tr><td colspan="1" rowspan="2">Analyte</td><td colspan="1" rowspan="2">IsolateStrain/Serotype/Source ID</td><td colspan="2" rowspan="1">LoD Concentrationa</td></tr><tr><td colspan="1" rowspan="1">Viable Units</td><td colspan="1" rowspan="1">Molecular (DNA orRNA)</td></tr><tr><td colspan="4" rowspan="1">Atypical Bacteria</td></tr><tr><td colspan="1" rowspan="1">Chlamydia pneumoniae</td><td colspan="1" rowspan="1">TW183ATCC VR-2282</td><td colspan="1" rowspan="1">5.0E-01 TCID50/mLb</td><td colspan="1" rowspan="1">3.3E+02 copies/mLb</td></tr><tr><td colspan="1" rowspan="1">Legionella pneumophila</td><td colspan="1" rowspan="1">Philadelphia-1ATCC 33152</td><td colspan="1" rowspan="1">5.0E+02 CFU/mL</td><td colspan="1" rowspan="1">1.6E+03 copies/mL</td></tr><tr><td colspan="1" rowspan="1">Mycoplasmapneumoniae</td><td colspan="1" rowspan="1">M129Zeptometrix 0801579</td><td colspan="1" rowspan="1">7.5E+01 TCID50/mL b</td><td colspan="1" rowspan="1">3.5E+03 copies/mL b</td></tr><tr><td colspan="4" rowspan="1">Viruses</td></tr><tr><td colspan="1" rowspan="6">Adenovirus</td><td colspan="1" rowspan="1">Species A(A18) ATCC VR-19</td><td colspan="1" rowspan="1">5.0E+01 TCID50/mL</td><td colspan="1" rowspan="1">9.2E+03 copies/mL</td></tr><tr><td colspan="1" rowspan="1">Species B (B3)Zeptometrix 0810062CF</td><td colspan="1" rowspan="1">1.0E+00 TCID50/mL</td><td colspan="1" rowspan="1">1.8E+03 copies/mL</td></tr><tr><td colspan="1" rowspan="1">Species C (C2)ATCC VR-846</td><td colspan="1" rowspan="1">5.0E+00 TCID50/mL</td><td colspan="1" rowspan="1">7.5E+03 copies/mL</td></tr><tr><td colspan="1" rowspan="1">Species D (D37)Zeptometrix 0810119CF</td><td colspan="1" rowspan="1">2.5E-01 TCID50/mLb</td><td colspan="1" rowspan="1">2.9E+03 copies/mL b</td></tr><tr><td colspan="1" rowspan="1">Species E (E4)Zeptometrix 0810070CF</td><td colspan="1" rowspan="1">1.0E-01 TCID50/mL b</td><td colspan="1" rowspan="1">3.5E+04 copies/mLb</td></tr><tr><td colspan="1" rowspan="1">Species F (F41)ATCC VR-930</td><td colspan="1" rowspan="1">5.0E+00 TCID50/mL</td><td colspan="1" rowspan="1">5.5E+03 copies/mL</td></tr><tr><td colspan="1" rowspan="4">Coronavirus</td><td colspan="1" rowspan="1">229E</td><td colspan="1" rowspan="1">5.0E-01 TCID50/mL</td><td colspan="1" rowspan="1">8.1E+01 copies/mL</td></tr><tr><td colspan="1" rowspan="1">HKU1Clinical Specimend</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1.0E+04 copies/mL</td></tr><tr><td colspan="1" rowspan="1">NL63BEI NR-470</td><td colspan="1" rowspan="1">2.5E+00 TCID50/mLc</td><td colspan="1" rowspan="1">5.4E+02 copies/mLc</td></tr><tr><td colspan="1" rowspan="1">OC43ATCC VR-759</td><td colspan="1" rowspan="1">5.0E+02 TCID50/mL</td><td colspan="1" rowspan="1">9.3E+03 copies/mLc</td></tr><tr><td colspan="1" rowspan="1">HumanMetapneumovirus</td><td colspan="1" rowspan="1">16 Type A1Zeptometrix 0810161CF</td><td colspan="1" rowspan="1">5.0E+01 TCID50/mL</td><td colspan="1" rowspan="1">5.9E+03 copies/mL</td></tr><tr><td colspan="1" rowspan="2">Human Rhinovirus/Enterovirus</td><td colspan="1" rowspan="1">Rhinovirus Type 1AZeptometrix 810012CFN</td><td colspan="1" rowspan="1">1.5E+01 TCID50/mLb</td><td colspan="1" rowspan="1">6.6E+03 copies/mL b</td></tr><tr><td colspan="1" rowspan="1">Echovirus 6Zeptometrix 0810076CF</td><td colspan="1" rowspan="1">1.0E+02 TCID50/mL</td><td colspan="1" rowspan="1">5.7E+02 copies/mL</td></tr><tr><td colspan="1" rowspan="2">Analyte</td><td colspan="1" rowspan="2">IsolateStrain/Serotype/Source ID</td><td colspan="2" rowspan="1">LoD Concentrationa</td></tr><tr><td colspan="1" rowspan="1">Viable Units</td><td colspan="1" rowspan="1">Molecular (DNA orRNA)</td></tr><tr><td colspan="1" rowspan="2">Influenza A</td><td colspan="1" rowspan="1">H1N1pdm09A/SwineNY/03/09Zeptometrix 0810249CF</td><td colspan="1" rowspan="1">2.5E+00 TCID50/mLc</td><td colspan="1" rowspan="1">1.7E+03 copies/mLc</td></tr><tr><td colspan="1" rowspan="1">H3N2A/Port Chalmers/1/73ATCC VR-810</td><td colspan="1" rowspan="1">1.0E+00 TCID50/mLb</td><td colspan="1" rowspan="1">2.1E+02 copies/mLb</td></tr><tr><td colspan="1" rowspan="1">Influenza B</td><td colspan="1" rowspan="1">B/FL/04/06 Zeptometrix0810255CF</td><td colspan="1" rowspan="1">5.0E+00 TCID50/mL</td><td colspan="1" rowspan="1">4.2E+02 copies/mLc</td></tr><tr><td colspan="1" rowspan="4">Parainfluenza Virus</td><td colspan="1" rowspan="1">Type 1Zeptometrix 0810014CF</td><td colspan="1" rowspan="1">2.5E+01 TCID50/mL</td><td colspan="1" rowspan="1">5.2E+03 copies/mL</td></tr><tr><td colspan="1" rowspan="1">Type 2Zeptometrix 0810015CF</td><td colspan="1" rowspan="1">2.5E+01 TCID50/mLc</td><td colspan="1" rowspan="1">1.5E+03 copies/mLc</td></tr><tr><td colspan="1" rowspan="1">Type 3Zeptometrix 0810016CF</td><td colspan="1" rowspan="1">2.5E+01 TCID50/mL</td><td colspan="1" rowspan="1">3.8E+02 copies/mLc</td></tr><tr><td colspan="1" rowspan="1">Type 4AZeptometrix 0810060CF</td><td colspan="1" rowspan="1">2.5E+02 TCID50/mL</td><td colspan="1" rowspan="1">8.1E+03 copies/mL</td></tr><tr><td colspan="1" rowspan="1">Respiratory SyncytialVirus</td><td colspan="1" rowspan="1">Type AZeptometrix 0810040ACF</td><td colspan="1" rowspan="1">1.0E+00 TCID50/mL</td><td colspan="1" rowspan="1">4.3E+02 copies/mL</td></tr></table>

a. The listed concentration was confirmed with $2 9 5 \%$ detection on each FilmArray system in artificial BAL (aBAL) and/or sputum. b. LoD confirmation $( \geq 9 5 \%$ detection) was achieved at a 2 to 5-fold lower concentration in aBAL. c. LoD confirmation $( \geq 9 5 \%$ detection) was achieved at a 2 to 5-fold lower concentration in sputum. d. No cultured isolates of Coronavirus HKU1 were available for testing.

For bacteria, the FilmArray Pneumonia Panel reports a Detected result when the estimated bacterial nucleic acid abundance is $\geq 1 0 ^ { 3 . 5 }$ copies/mL, and the panel reports a Not Detected if there is no amplification or the estimated bacterial nucleic acid abundance is $< 1 0 ^ { 3 . 5 }$ copies/mL. No assay-specific LoD concentrations were determined for the bacterial analytes, however, each assay was determined to be linear (slope $\approx 1 . 0$ and coefficient of determination (Adj R2) ${ \tt > } 0 . 9 5$ ) and estimates of nucleic acid abundance were determined to be accurate within 0.5 log10 copies/mL when compared to an input concentration determined by digital PCR.

Antimicrobial resistance (AMR) genes are reported as Detected when an applicable bacterium is detected and the assay for the AMR gene is positive. No AMR gene assay-specific LoD concentrations were determined for the AMR gene, but positive AMR gene assay results were recorded in $2 9 5 \%$ of 90 replicates of applicable bacteria tested at concentrations of $1 . 0 \mathrm { E } { + } 0 4$ copies $/ \mathrm { m L }$ or less in the precision evaluation (see Precision below).

# b. Reproducibility:

A multisite reproducibility study of the FilmArray Pneumonia Panel was performed with contrived BAL samples over multiples days at three laboratory locations (sites) on a combination of FilmArray, FilmArray 2.0 and FilmArray Torch systems. Testing incorporated a range of potential sources of variability, including run-to-run, day-to-day, and site-to-site variability, which also encompassed different operators, instruments, systems and reagent lots for a total of 30 tests per system and 90 total replicates per sample/concentration.

Evaluation of the reproducibility of Detected/Not Detected results for atypical bacteria and viruses included samples containing combinations of five different analytes, at Negative, Low Positive $\left( 1 \times \mathrm { L o D } \right)$ , and Moderate Positive $\left( 3 { \times } \mathrm { L o D } \right)$ concentrations. Negative results were obtained from up to 24 additional samples (see evaluation of precision for bacterial analytes below) that were not spiked with the analyte.

A summary of results (percent $( \% )$ agreement with the expected Detected or Not Detected result) for each atypical bacterium and virus (by site and system) is provided Table 6 below.

Table 6: Reproducibility of FilmArray Pneumonia Panel Atypical Bacteria and Virus Results   
Viruses   

<table><tr><td colspan="1" rowspan="3">Analyte</td><td colspan="1" rowspan="3">Concentration Tested</td><td colspan="1" rowspan="3">ExpectedResult</td><td colspan="4" rowspan="1">Agreement with Expected Result</td></tr><tr><td colspan="1" rowspan="1">FilmArray</td><td colspan="1" rowspan="1">FilmArray2.0</td><td colspan="1" rowspan="1">FilmArrayTorch</td><td colspan="1" rowspan="2">All Sites [95%CI]</td></tr><tr><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">Site C</td></tr><tr><td colspan="7" rowspan="1">Atypical Bacteria</td></tr><tr><td colspan="1" rowspan="1">Chlamydiapneumoniae</td><td colspan="1" rowspan="1">None(No Analyte)</td><td colspan="1" rowspan="1">Not Detected</td><td colspan="1" rowspan="1">780/780100%</td><td colspan="1" rowspan="1">780/780100%</td><td colspan="1" rowspan="1">780/780100%</td><td colspan="1" rowspan="1">2,340/2,340100%[99.8%-100%]</td></tr><tr><td colspan="1" rowspan="3">Legionella pneumophila Philadelphia-1ATC 33152</td><td colspan="1" rowspan="1">Moderate Positive3× LoD1.5E+03 CFU/mL</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">90/90100%[96.0%-100%]</td></tr><tr><td colspan="1" rowspan="1">Low Positive1× LoD5.0E+02 CFU/mL</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">90/90100%[96.0%-100%]</td></tr><tr><td colspan="1" rowspan="1">None(No Analyte)</td><td colspan="1" rowspan="1">Not Detected</td><td colspan="1" rowspan="1">720/720100%</td><td colspan="1" rowspan="1">720/720100%</td><td colspan="1" rowspan="1">720/720100%</td><td colspan="1" rowspan="1">2,160/2,160100%[99.8%-100%]</td></tr><tr><td colspan="1" rowspan="1">Mycoplasma pneumoniae</td><td colspan="1" rowspan="1">None(No Analyte)</td><td colspan="1" rowspan="1">Not Detected</td><td colspan="1" rowspan="1">780/780100%</td><td colspan="1" rowspan="1">780/780100%</td><td colspan="1" rowspan="1">780/780100%</td><td colspan="1" rowspan="1">2,340/2,340100%[99.8%-100%]</td></tr><tr><td colspan="1" rowspan="1">FilmArray</td><td colspan="1" rowspan="1">FilmArray2.0</td><td colspan="1" rowspan="1">FilmArrayTorch</td><td colspan="1" rowspan="2">All Sites [95%CI</td></tr><tr><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">Site C</td></tr><tr><td colspan="1" rowspan="3">AdenovirusSpecies B Serotype 3 ZeptoMetrix0810062CF</td><td colspan="1" rowspan="1">Moderate Positive 3xLoD3.0E+00 TCID50/mL</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">90/90100%[96.0%-100%]</td></tr><tr><td colspan="1" rowspan="1">Low Positive1× LoD1.0E+00 TCID50/mL</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">90/90100%[96.0%-100%]</td></tr><tr><td colspan="1" rowspan="1">None(No Analyte)</td><td colspan="1" rowspan="1">Not Detected</td><td colspan="1" rowspan="1">720/720100%</td><td colspan="1" rowspan="1">720/720100%</td><td colspan="1" rowspan="1">720/720100%</td><td colspan="1" rowspan="1">2,160/2,160100%[99.8%-100%]</td></tr><tr><td colspan="1" rowspan="1">Coronavirus</td><td colspan="1" rowspan="1">None(No Analyte)</td><td colspan="1" rowspan="1">Not Detected</td><td colspan="1" rowspan="1">780/780100%</td><td colspan="1" rowspan="1">776/78099.5%</td><td colspan="1" rowspan="1">780/780100%</td><td colspan="1" rowspan="1">2,336/2,34099.8%[99.6%-100%]</td></tr><tr><td colspan="1" rowspan="3">Human Metapneumovirus16 Type A1 ZeptoMetrix 0810161CF</td><td colspan="1" rowspan="1">Moderate Positive3× LoD1.5E+02 TCID50/mL</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">90/90100%[96.0%-100%]</td></tr><tr><td colspan="1" rowspan="1">Low Positive1× LoD5.0E+01 TCID50/mL</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">29/3096.7%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">89/9098.9%[94.0%-100%]</td></tr><tr><td colspan="1" rowspan="1">None(No Analyte)</td><td colspan="1" rowspan="1">Not Detected</td><td colspan="1" rowspan="1">720/720100%</td><td colspan="1" rowspan="1">720/720100%</td><td colspan="1" rowspan="1">720/720100%</td><td colspan="1" rowspan="1">2,160/2,160100%[99.8%-100%]</td></tr><tr><td colspan="1" rowspan="1">Human Rhinovirus/Enterovirus</td><td colspan="1" rowspan="1">None(No Analyte)</td><td colspan="1" rowspan="1">Not Detected</td><td colspan="1" rowspan="1">779/78099.9%</td><td colspan="1" rowspan="1">780/780100%</td><td colspan="1" rowspan="1">779/78099.9%</td><td colspan="1" rowspan="1">2,338/2,34099.9%[99.7%-100%]</td></tr><tr><td colspan="1" rowspan="2">Influenza AH3N2A/Port Chalmers/1/73 ATCC VR-810</td><td colspan="1" rowspan="1">Moderate Positive3× LoD1.5E+00 TCID50/mL</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">90/90100%[96.0%-100%]</td></tr><tr><td colspan="1" rowspan="1">Low Positive1× LoD0.5E-01 TCID50/mL</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">29/3096.7%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">89/9098.9%[94.0%-100%]</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">None(No Analyte)</td><td colspan="1" rowspan="1">Not Detected</td><td colspan="1" rowspan="1">720/720100%</td><td colspan="1" rowspan="1">720/720100%</td><td colspan="1" rowspan="1">720/720100%</td><td colspan="1" rowspan="1">2,160/2,160100%[99.8%-100%]</td></tr><tr><td colspan="1" rowspan="1">Influenza B</td><td colspan="1" rowspan="1">None(No Analyte)</td><td colspan="1" rowspan="1">Not Detected</td><td colspan="1" rowspan="1">780/780100%</td><td colspan="1" rowspan="1">780/780100%</td><td colspan="1" rowspan="1">780/780100%</td><td colspan="1" rowspan="1">2,340/2,340100%[99.8%-100%]</td></tr><tr><td colspan="1" rowspan="2">Parainfluenza VirusType 2 ZeptoMetrix 0810015CF</td><td colspan="1" rowspan="1">Moderate Positive3× LoD7.5E+01 TCID50/mL</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">90/90100%[96.0%-100%]</td></tr><tr><td colspan="1" rowspan="1">Low Positive1× LoD2.5E+01 TCID50/mL</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">90/90100%[96.0%-100%]</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">None(No Analyte)</td><td colspan="1" rowspan="1">Not Detected</td><td colspan="1" rowspan="1">720/720100%</td><td colspan="1" rowspan="1">720/720100%</td><td colspan="1" rowspan="1">720/720100%</td><td colspan="1" rowspan="1">2,160/2,160100%[99.8%-100%]</td></tr><tr><td colspan="1" rowspan="1">Respiratory Syncytial Virus</td><td colspan="1" rowspan="1">None(No Analyte)</td><td colspan="1" rowspan="1">Not Detected</td><td colspan="1" rowspan="1">780/780100%</td><td colspan="1" rowspan="1">780/780100%</td><td colspan="1" rowspan="1">780/780100%</td><td colspan="1" rowspan="1">2,340/2,340100%[99.8%-100%]</td></tr></table>

Precision for bacterial analytes was measured at each concentration as 1) precision of bin results and 2) reproducibility of analyte detection. When a sample containing one or more bacteria is tested repeatedly, the precision of the bin results (probability that each replicate will receive the same bin result) will vary based on the concentration of nucleic acid measured and the relation of that concentration to the limits of each bin. Bin precision may be as low as $50 \%$ for values at a bin limit and precision will increase (up to $90 \%$ or higher) as the distance of the measured value from a bin limit increases. The precision of FilmArray Pneumonia Panel bin results will follow the model illustrated in Figure 1:

$5 9 0 \%$ at a bin center (Scenario 1) • ${ \sim } 6 0 - 9 0 \%$ between a bin limit and bin center (Scenario 2) Q ${ \sim } 5 0 \%$ at bin limits (Scenario 3)

![](images/0e05b130f2cad3f10b198e20c3d3f57b561752c4c9aa0987d48a59b375b0c4e8.jpg)

Top: The probability (0.0 – 1.0) of the same bin results for each replicate tested varies based on proximity of the measured value to a bin limit.

Bottom: Expected distribution of bin results at different mean measured values.

Samples containing bacteria and corresponding antimicrobial (AMR) genes were tested at six different concentrations over the reportable range and below. A summary of the bin precision (percent $( \% )$ of replicates reported in each bin) and the reproducibility of detection is shown at each concentration tested in Table 7 below.

Table 7: Reproducibility of FilmArray Pneumonia Panel Bacterial Bin Results on FilmArray, FilmArray 2.0 and FilmArray TorchGrey shading indicates the expected bin results based on the analyte concentration and bold font indicates the bin with the greatest percentage of results at each concentration.   

<table><tr><td colspan="1" rowspan="2">Analyte</td><td colspan="1" rowspan="2">Concentration(log10copies/mL)</td><td colspan="5" rowspan="1">% Replicates Reported in Each Bin Result</td><td colspan="1" rowspan="2">TotalDetected</td></tr><tr><td colspan="1" rowspan="1">≥107</td><td colspan="1" rowspan="1">106</td><td colspan="1" rowspan="1">105</td><td colspan="1" rowspan="1">104</td><td colspan="1" rowspan="1">ND</td></tr><tr><td colspan="1" rowspan="2">Acinetobacterbaumannii(NDM-1)</td><td colspan="1" rowspan="1">7.5</td><td colspan="1" rowspan="1">90/90(100%)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">90/90100%</td></tr><tr><td colspan="1" rowspan="1">6.5</td><td colspan="1" rowspan="1">87/90(96.7%)</td><td colspan="1" rowspan="1">3/90(3.3%)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">90/90100%</td></tr><tr><td colspan="1" rowspan="5">AR-BANK#0033</td><td colspan="1" rowspan="1">5.5</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">82/90(91.1%)</td><td colspan="1" rowspan="1">8/90(8.9%)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">90/90100%</td></tr><tr><td colspan="1" rowspan="1">4.5</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1/90(1.1%)</td><td colspan="1" rowspan="1">80/90(88.9%)</td><td colspan="1" rowspan="1">9/90(10.0%)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">90/90100%</td></tr><tr><td colspan="1" rowspan="1">3.5</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1/90(1.1%)</td><td colspan="1" rowspan="1">74/90(82.2%)</td><td colspan="1" rowspan="1">15/90(16.7%)</td><td colspan="1" rowspan="1">75/9083.3%</td></tr><tr><td colspan="1" rowspan="1">2.5</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1/90(1.1%)</td><td colspan="1" rowspan="1">89/90(98.9%)</td><td colspan="1" rowspan="1">1/901.1%</td></tr><tr><td colspan="1" rowspan="1">None(No Analyte)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1800/1800(100%)</td><td colspan="1" rowspan="1">0/18000.0%</td></tr><tr><td colspan="1" rowspan="7">Enterobacteraerogenes13048</td><td colspan="1" rowspan="1">7.5</td><td colspan="1" rowspan="1">90/90(100%)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">90/90100%</td></tr><tr><td colspan="1" rowspan="1">6.5</td><td colspan="1" rowspan="1">65/90(72.2%)</td><td colspan="1" rowspan="1">25/90(27.8%)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">90/90100%</td></tr><tr><td colspan="1" rowspan="1">5.5</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">52/90(57.8%)</td><td colspan="1" rowspan="1">38/90(42.2%)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">90/90100%</td></tr><tr><td colspan="1" rowspan="1">4.5</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">38/90(42.2%)</td><td colspan="1" rowspan="1">51/90(56.7%)</td><td colspan="1" rowspan="1">1/90(1.1%)</td><td colspan="1" rowspan="1">89/9098.9%</td></tr><tr><td colspan="1" rowspan="1">3.5</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">33/90(36.7%)</td><td colspan="1" rowspan="1">57/90(63.3%)</td><td colspan="1" rowspan="1">33/9036.7%</td></tr><tr><td colspan="1" rowspan="1">2.5</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1/90(1.1%)</td><td colspan="1" rowspan="1">89/90(98.9%)</td><td colspan="1" rowspan="1">1/901.1%</td></tr><tr><td colspan="1" rowspan="1">None(No Analyte)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1800/1800(100%)</td><td colspan="1" rowspan="1">0/18000.0%</td></tr><tr><td colspan="1" rowspan="7">Enterobactercloacae(VIM)AR-BANK#0154</td><td colspan="1" rowspan="1">7.0</td><td colspan="1" rowspan="1">90/90(100%)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">90/90100%</td></tr><tr><td colspan="1" rowspan="1">6.0</td><td colspan="1" rowspan="1">4/90(4.4%)</td><td colspan="1" rowspan="1">86/90(95.6%)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">90/90100%</td></tr><tr><td colspan="1" rowspan="1">5.0</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">6/90(6.7%)</td><td colspan="1" rowspan="1">80/90(88.9%)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">4/90(4.4%)</td><td colspan="1" rowspan="1">86/9095.6%</td></tr><tr><td colspan="1" rowspan="1">4.0</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">6/90(6.7%)</td><td colspan="1" rowspan="1">83/90(92.2%)</td><td colspan="1" rowspan="1">1/90(1.1%)</td><td colspan="1" rowspan="1">89/9098.9%</td></tr><tr><td colspan="1" rowspan="1">3.0</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1/90(1.1%)</td><td colspan="1" rowspan="1">4/90(4.4%)</td><td colspan="1" rowspan="1">85/90(94.4%)</td><td colspan="1" rowspan="1">5/905.6%</td></tr><tr><td colspan="1" rowspan="1">2.0</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">90/90(100%)</td><td colspan="1" rowspan="1">0/900.0%</td></tr><tr><td colspan="1" rowspan="1">None (NoAnalyte)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1800/1800(100%)</td><td colspan="1" rowspan="1">0/18000.0%</td></tr><tr><td colspan="1" rowspan="7">Escherichia coli(IMP)GRE 1062016</td><td colspan="1" rowspan="1">7.0</td><td colspan="1" rowspan="1">90/90(100%)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">90/90100%</td></tr><tr><td colspan="1" rowspan="1">6.0</td><td colspan="1" rowspan="1">7/90(7.8%)</td><td colspan="1" rowspan="1">82/90(91.1%)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1/90 (1.1%)</td><td colspan="1" rowspan="1">89/9098.9%</td></tr><tr><td colspan="1" rowspan="1">5.0</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">12/90(13.3%)</td><td colspan="1" rowspan="1">78/90(86.7%)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">90/90100%</td></tr><tr><td colspan="1" rowspan="1">4.0</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">12/90(13.3%)</td><td colspan="1" rowspan="1">77/90(85.6%)</td><td colspan="1" rowspan="1">1/90(1.1%)</td><td colspan="1" rowspan="1">89/9098.9%</td></tr><tr><td colspan="1" rowspan="1">3.0</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1/90(1.1%)</td><td colspan="1" rowspan="1">15/90(16.7%)</td><td colspan="1" rowspan="1">74/90(82.2%)</td><td colspan="1" rowspan="1">16/9017.8%</td></tr><tr><td colspan="1" rowspan="1">2.0</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">90/90(100%)</td><td colspan="1" rowspan="1">0/900.0%</td></tr><tr><td colspan="1" rowspan="1">None (NoAnalyte)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1800/1800(100%)</td><td colspan="1" rowspan="1">0/18000.0%</td></tr><tr><td colspan="1" rowspan="7">HaemophilusinfluenzaeATCC 10211</td><td colspan="1" rowspan="1">7.0</td><td colspan="1" rowspan="1">89/90(98.9%)</td><td colspan="1" rowspan="1">1/90(1.1%)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">90/90100%</td></tr><tr><td colspan="1" rowspan="1">6.0</td><td colspan="1" rowspan="1">35/90(48.9%)</td><td colspan="1" rowspan="1">55/90(61.1%)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">90/90100%</td></tr><tr><td colspan="1" rowspan="1">5.0</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">49/90(54.4%)</td><td colspan="1" rowspan="1">40/90(44.4%)</td><td colspan="1" rowspan="1">1/90(1.1%)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">90/90100%</td></tr><tr><td colspan="1" rowspan="1">4.0</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">41/90(45.6%)</td><td colspan="1" rowspan="1">49/90(54.4%)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">90/90100%</td></tr><tr><td colspan="1" rowspan="1">3.0</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">42/90(46.7%)</td><td colspan="1" rowspan="1">48/90(53.3%)</td><td colspan="1" rowspan="1">42/9046.7%</td></tr><tr><td colspan="1" rowspan="1">2.0</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">90/90(100%)</td><td colspan="1" rowspan="1">0/900.0%</td></tr><tr><td colspan="1" rowspan="1">None (NoAnalyte)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1800/1800(100%)</td><td colspan="1" rowspan="1">0/18000.0%</td></tr><tr><td colspan="1" rowspan="5">Klebsiellaoxytoca(CTX-M)GRE 1254054</td><td colspan="1" rowspan="1">7.5</td><td colspan="1" rowspan="1">90/90(100%)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">90/90100%</td></tr><tr><td colspan="1" rowspan="1">6.5</td><td colspan="1" rowspan="1">90/90(100%)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">90/90100%</td></tr><tr><td colspan="1" rowspan="1">5.5</td><td colspan="1" rowspan="1">1/90(1.1%)</td><td colspan="1" rowspan="1">84/90(93.3%)</td><td colspan="1" rowspan="1">3/90(3.3%)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2/90(2.2%)</td><td colspan="1" rowspan="1">88/9097.8%</td></tr><tr><td colspan="1" rowspan="1">4.5</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">89/90(98.9%)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1/90(1.1%)</td><td colspan="1" rowspan="1">89/9098.9%</td></tr><tr><td colspan="1" rowspan="1">3.5</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">90/90(100%)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">90/90100%</td></tr><tr><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="1">2.5</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1/90(1.1%)</td><td colspan="1" rowspan="1">1/90(1.1%)</td><td colspan="1" rowspan="1">88/90(97.8%)</td><td colspan="1" rowspan="1">2/902.2%</td></tr><tr><td colspan="1" rowspan="1">None (NoAnalyte)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1800/1800(100%)</td><td colspan="1" rowspan="1">0/18000.0%</td></tr><tr><td colspan="1" rowspan="7">Klebsiellapneumoniae(KPC)AR-BANK#0097</td><td colspan="1" rowspan="1">7.00</td><td colspan="1" rowspan="1">90/90(100%)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">90/90100%</td></tr><tr><td colspan="1" rowspan="1">6.00</td><td colspan="1" rowspan="1">12/90(13.3%)</td><td colspan="1" rowspan="1">78/90(86.7%)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">90/90100%</td></tr><tr><td colspan="1" rowspan="1">5.00</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">15/90(16.7%)</td><td colspan="1" rowspan="1">75/90(83.3%)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">90/90100%</td></tr><tr><td colspan="1" rowspan="1">4.00</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">23/90(25.6%)</td><td colspan="1" rowspan="1">66/90(73.3%)</td><td colspan="1" rowspan="1">1/90(1.1%)</td><td colspan="1" rowspan="1">89/9098.9%</td></tr><tr><td colspan="1" rowspan="1">3.00</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">15/90(16.7%)</td><td colspan="1" rowspan="1">75/90(83.3%)</td><td colspan="1" rowspan="1">15/9016.7%</td></tr><tr><td colspan="1" rowspan="1">2.00</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">90/90(100%)</td><td colspan="1" rowspan="1">0/900.0%</td></tr><tr><td colspan="1" rowspan="1">None (NoAnalyte)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1800/1800(100%)</td><td colspan="1" rowspan="1">0/18000.0%</td></tr><tr><td colspan="1" rowspan="7">MoraxellacatarrhalisATCC 8176</td><td colspan="1" rowspan="1">7.0</td><td colspan="1" rowspan="1">90/90(100%)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">90/90100%</td></tr><tr><td colspan="1" rowspan="1">6.0</td><td colspan="1" rowspan="1">26/90(28.9%)</td><td colspan="1" rowspan="1">64/90(71.1%)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">90/90100%</td></tr><tr><td colspan="1" rowspan="1">5.0</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">6/90(6.7%)</td><td colspan="1" rowspan="1">83/90(92.2%)</td><td colspan="1" rowspan="1">1/90(1.1%)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">90/90100%</td></tr><tr><td colspan="1" rowspan="1">4.0</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">4/90(4.4%)</td><td colspan="1" rowspan="1">86/90(95.6%)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">90/90100%</td></tr><tr><td colspan="1" rowspan="1">3.0</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">4/90(4.4%)</td><td colspan="1" rowspan="1">86/90(95.6%)</td><td colspan="1" rowspan="1">4/904.4%</td></tr><tr><td colspan="1" rowspan="1">2.0</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">90/90(100%)</td><td colspan="1" rowspan="1">0/900.0%</td></tr><tr><td colspan="1" rowspan="1">None(No Analyte)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1800/1800(100%)</td><td colspan="1" rowspan="1">0/18000.0%</td></tr><tr><td colspan="1" rowspan="3">ProteusmirabilisATCC 35659</td><td colspan="1" rowspan="1">7.0</td><td colspan="1" rowspan="1">88/90(97.8%)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2/90(2.2%)</td><td colspan="1" rowspan="1">88/9097.8%</td></tr><tr><td colspan="1" rowspan="1">6.0</td><td colspan="1" rowspan="1">27/90(30.0%)</td><td colspan="1" rowspan="1">63/90(70.0%)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">90/90100%</td></tr><tr><td colspan="1" rowspan="1">5.0</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">26/90(28.9%)</td><td colspan="1" rowspan="1">64/90(71.1%)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">90/90100%</td></tr><tr><td colspan="1" rowspan="4"></td><td colspan="1" rowspan="1">4.0</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">14/90(15.6%)</td><td colspan="1" rowspan="1">75/90(83.3%)</td><td colspan="1" rowspan="1">1/90(1.1%)</td><td colspan="1" rowspan="1">89/9098.9%</td></tr><tr><td colspan="1" rowspan="1">3.0</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">28/90(31.1%)</td><td colspan="1" rowspan="1">62/90(68.9%)</td><td colspan="1" rowspan="1">28/9031.1%</td></tr><tr><td colspan="1" rowspan="1">2.0</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">90/90(100%)</td><td colspan="1" rowspan="1">0/900.0%</td></tr><tr><td colspan="1" rowspan="1">None(No Analyte)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1800/1800(100%)</td><td colspan="1" rowspan="1">0/18000.0%</td></tr><tr><td colspan="1" rowspan="7">PseudomonasaeruginosaATCC 10145</td><td colspan="1" rowspan="1">7.0</td><td colspan="1" rowspan="1">90/90(100%)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">90/90100%</td></tr><tr><td colspan="1" rowspan="1">6.0</td><td colspan="1" rowspan="1">20/90(22.2%)</td><td colspan="1" rowspan="1">70/90(77.8%)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">90/90100%</td></tr><tr><td colspan="1" rowspan="1">5.0</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">24/90(26.7%)</td><td colspan="1" rowspan="1">66/90(73.3%)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">90/90100%</td></tr><tr><td colspan="1" rowspan="1">4.0</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">16/90(17.8%)</td><td colspan="1" rowspan="1">74/90(82.2%)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">90/90100%</td></tr><tr><td colspan="1" rowspan="1">3.0</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">14/90(15.6%)</td><td colspan="1" rowspan="1">76/90(84.4%)</td><td colspan="1" rowspan="1">14/9015.6%</td></tr><tr><td colspan="1" rowspan="1">2.0</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">90/90(100%)</td><td colspan="1" rowspan="1">0/900.0%</td></tr><tr><td colspan="1" rowspan="1">None(No Analyte)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1800/1800(100%)</td><td colspan="1" rowspan="1">0/18000.0%</td></tr><tr><td colspan="1" rowspan="7">Serratiamarcescens(OXA-48-like)GRE 1659005</td><td colspan="1" rowspan="1">7.0</td><td colspan="1" rowspan="1">90/90(100%)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">90/90100%</td></tr><tr><td colspan="1" rowspan="1">6.0</td><td colspan="1" rowspan="1">2/90(2.2%)</td><td colspan="1" rowspan="1">88/90(97.8%)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">90/90100%</td></tr><tr><td colspan="1" rowspan="1">5.0</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">7/90(7.8%)</td><td colspan="1" rowspan="1">83/90(92.2%)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">90/90100%</td></tr><tr><td colspan="1" rowspan="1">4.0</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">6/90(6.7%)</td><td colspan="1" rowspan="1">83/90(92.2%)</td><td colspan="1" rowspan="1">1/90(1.1%)</td><td colspan="1" rowspan="1">89/9098.9%</td></tr><tr><td colspan="1" rowspan="1">3.0</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">6/90(6.7%)</td><td colspan="1" rowspan="1">84/90(93.3%)</td><td colspan="1" rowspan="1">6/906.7%</td></tr><tr><td colspan="1" rowspan="1">2.0</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">90/90(100%)</td><td colspan="1" rowspan="1">0/900.0%</td></tr><tr><td colspan="1" rowspan="1">None(No Analyte)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1800/1800(100%)</td><td colspan="1" rowspan="1">0/18000.0%</td></tr><tr><td colspan="1" rowspan="7">Staphylococcusaureussubsp. aureus(mecA/C andMREJ)ATCC 43300</td><td colspan="1" rowspan="1">7.0</td><td colspan="1" rowspan="1">90/90(100%)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">90/90100%</td></tr><tr><td colspan="1" rowspan="1">6.0</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">90/90(100%)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">90/90100%</td></tr><tr><td colspan="1" rowspan="1">5.0</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">90/90(100%)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">90/90100%</td></tr><tr><td colspan="1" rowspan="1">4.0</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">89/90(98.9%)</td><td colspan="1" rowspan="1">1/90(1.1%)</td><td colspan="1" rowspan="1">89/9098.9%</td></tr><tr><td colspan="1" rowspan="1">3.0</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">90/90(100%)</td><td colspan="1" rowspan="1">0/900.0%</td></tr><tr><td colspan="1" rowspan="1">2.0</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">90/90(100%)</td><td colspan="1" rowspan="1">0/900.0%</td></tr><tr><td colspan="1" rowspan="1">None(No Analyte)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2/1260(0.2%)</td><td colspan="1" rowspan="1">1258/1260(99.8%)</td><td colspan="1" rowspan="1">2/12600.2%</td></tr><tr><td colspan="1" rowspan="7">StreptococcusagalactiaeATCC 13813</td><td colspan="1" rowspan="1">7.8</td><td colspan="1" rowspan="1">89/90(98.9%)</td><td colspan="1" rowspan="1">1/90 (1.1%)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">90/90100%</td></tr><tr><td colspan="1" rowspan="1">6.8</td><td colspan="1" rowspan="1">89/90(98.9%)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1/90 (1.1%)</td><td colspan="1" rowspan="1">89/9098.9%</td></tr><tr><td colspan="1" rowspan="1">5.8</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">88/90(97.8%)</td><td colspan="1" rowspan="1">1/90(1.1%)</td><td colspan="1" rowspan="1">1/90(1.1%)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">90/90100%</td></tr><tr><td colspan="1" rowspan="1">4.8</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">89/90(98.9%)</td><td colspan="1" rowspan="1">1/90(1.1%)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">90/90100%</td></tr><tr><td colspan="1" rowspan="1">3.8</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">86/90(95.6%)</td><td colspan="1" rowspan="1">4/90(4.4%)</td><td colspan="1" rowspan="1">86/9095.6%</td></tr><tr><td colspan="1" rowspan="1">2.8</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">3/90(3.3%)</td><td colspan="1" rowspan="1">87/90(96.7%)</td><td colspan="1" rowspan="1">3/903.3%</td></tr><tr><td colspan="1" rowspan="1">None(No Analyte)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1800/1800(100%)</td><td colspan="1" rowspan="1">0/18000.0%</td></tr><tr><td colspan="1" rowspan="5">StreptococcuspneumoniaeATCC 6303</td><td colspan="1" rowspan="1">6.5</td><td colspan="1" rowspan="1">90/90(100%)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">90/90100%</td></tr><tr><td colspan="1" rowspan="1">5.5</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">90/90(100%)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">90/90100%</td></tr><tr><td colspan="1" rowspan="1">4.5</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">89/90(98.9%)</td><td colspan="1" rowspan="1">1/90(1.1%)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">90/90100%</td></tr><tr><td colspan="1" rowspan="1">3.5</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">89/90(98.9%)</td><td colspan="1" rowspan="1">1/90(1.1%)</td><td colspan="1" rowspan="1">89/9098.9%</td></tr><tr><td colspan="1" rowspan="1">2.5</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">90/90(100%)</td><td colspan="1" rowspan="1">0/900.0%</td></tr><tr><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="1">1.5</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">90/90(100%)</td><td colspan="1" rowspan="1">0/900.0%</td></tr><tr><td colspan="1" rowspan="1">None(No Analyte)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1800/1800(100%)</td><td colspan="1" rowspan="1">0/18000.0%</td></tr><tr><td colspan="1" rowspan="7">StreptococcuspyogenesATCC 49399</td><td colspan="1" rowspan="1">7.8</td><td colspan="1" rowspan="1">90/90(100%)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">90/90100%</td></tr><tr><td colspan="1" rowspan="1">6.8</td><td colspan="1" rowspan="1">90/90(100%)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">90/90100%</td></tr><tr><td colspan="1" rowspan="1">5.8</td><td colspan="1" rowspan="1">5/90(5.6%)</td><td colspan="1" rowspan="1">84/90(93.3%)</td><td colspan="1" rowspan="1">1/90(1.1%)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">90/90100%</td></tr><tr><td colspan="1" rowspan="1">4.8</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">3/90(3.3%)</td><td colspan="1" rowspan="1">87/90(96.7%)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">90/90100%</td></tr><tr><td colspan="1" rowspan="1">3.8</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">3/90(3.3%)</td><td colspan="1" rowspan="1">87/90(96.7%)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">90/90100%</td></tr><tr><td colspan="1" rowspan="1">2.8</td><td colspan="1" rowspan="1">.</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">.</td><td colspan="1" rowspan="1">16/90(18.9%)</td><td colspan="1" rowspan="1">74/90(81.1%)</td><td colspan="1" rowspan="1">16/9017.8%</td></tr><tr><td colspan="1" rowspan="1">None(No Analyte)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1800/1800(100%)</td><td colspan="1" rowspan="1">0/18000.0%</td></tr></table>

The precision of the antimicrobial resistance (AMR) genes was measured as the reproducibility of analyte detection on each system and overall. Results are presented in Table 8 as the percent of replicates that are detected at concentrations of the associated bacterium that are within the reportable range, or below the reportable range, as well as the percent agreement with the expected Not Detected result in unspiked samples.

Table 8: Reproducibility of FilmArray Pneumonia Panel Antimicrobial Resistance Gene Results on FilmArray, FilmArray 2.0 and FilmArray Torch   

<table><tr><td colspan="1" rowspan="3">AMR GeneOrganism</td><td colspan="1" rowspan="3">Concentrationof Organism(copies/mLlog10)</td><td colspan="1" rowspan="3">ExpectedResult</td><td colspan="4" rowspan="1">Agreement with the Expected Result</td></tr><tr><td colspan="1" rowspan="1">FilmArray</td><td colspan="1" rowspan="1">FilmArray2.0</td><td colspan="1" rowspan="1">FilmArrayTorch</td><td colspan="1" rowspan="2">All Systems/Sites[95% Cl]</td></tr><tr><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">Site C</td></tr><tr><td colspan="1" rowspan="3">CTX-MKlebsiella oxytocaGRE 1254054</td><td colspan="1" rowspan="1">ReportableRange</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">150/150100%</td><td colspan="1" rowspan="1">149/15099.3</td><td colspan="1" rowspan="1">150/150100%</td><td colspan="1" rowspan="1">449/45099.8%[98.8%-99.9%]</td></tr><tr><td colspan="1" rowspan="1">Below ReportableRange</td><td colspan="1" rowspan="1">Detected(Variable)</td><td colspan="1" rowspan="1">0/300%</td><td colspan="1" rowspan="1">1/303.3%</td><td colspan="1" rowspan="1">0/300%</td><td colspan="1" rowspan="1">1/901.1%[0.03%-6.0%]</td></tr><tr><td colspan="1" rowspan="1">None(No Analyte)</td><td colspan="1" rowspan="1">N/A or NotDetected</td><td colspan="1" rowspan="1">600/600100%</td><td colspan="1" rowspan="1">599/60099.8%</td><td colspan="1" rowspan="1">600/600100%</td><td colspan="1" rowspan="1">1799/180099.9%[99.7%-100%]</td></tr><tr><td colspan="1" rowspan="3">IMPEscherichia coliGRE 1062016</td><td colspan="1" rowspan="1">ReportableRange</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">120/120100%</td><td colspan="1" rowspan="1">120/120100%</td><td colspan="1" rowspan="1">120/120100%</td><td colspan="1" rowspan="1">360/360100%[99.0%-100%]</td></tr><tr><td colspan="1" rowspan="1">Below ReportableRange</td><td colspan="1" rowspan="1">Detected(Variable)</td><td colspan="1" rowspan="1">10/6016.7%</td><td colspan="1" rowspan="1">9/6015.0%</td><td colspan="1" rowspan="1">3/605.0%</td><td colspan="1" rowspan="1">22/18012.2%[7.8%-17.9%]</td></tr><tr><td colspan="1" rowspan="1">None(No Analyte)</td><td colspan="1" rowspan="1">N/A or NotDetected</td><td colspan="1" rowspan="1">600/600100%</td><td colspan="1" rowspan="1">600/600100%</td><td colspan="1" rowspan="1">600/600100%</td><td colspan="1" rowspan="1">1800/1800100%[99.8%-100%]</td></tr><tr><td colspan="1" rowspan="3">KPCKlebsiellapneumoniaeAR-Bank#0097</td><td colspan="1" rowspan="1">ReportableRange</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">120/120100%</td><td colspan="1" rowspan="1">119/12099.2%</td><td colspan="1" rowspan="1">120/120100%</td><td colspan="1" rowspan="1">359/36099.7%[98.5%-100%]</td></tr><tr><td colspan="1" rowspan="1">Below ReportableRange</td><td colspan="1" rowspan="1">Detected(Variable)</td><td colspan="1" rowspan="1">14/6023.3%</td><td colspan="1" rowspan="1">10/6016.7%</td><td colspan="1" rowspan="1">9/6015.0%</td><td colspan="1" rowspan="1">33/18018.3%[13.0%-24.8%]</td></tr><tr><td colspan="1" rowspan="1">None(No Analyte)</td><td colspan="1" rowspan="1">N/A or NotDetected</td><td colspan="1" rowspan="1">600/600100%</td><td colspan="1" rowspan="1">600/600100%</td><td colspan="1" rowspan="1">600/600100%</td><td colspan="1" rowspan="1">1800/1800100%[99.8%-100%]</td></tr><tr><td colspan="1" rowspan="3">mecA/C andMREJStaphylococcusaureusATCC 43300</td><td colspan="1" rowspan="1">ReportableRange</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">119/12099.2%</td><td colspan="1" rowspan="1">118/12098.3%</td><td colspan="1" rowspan="1">120/120100%</td><td colspan="1" rowspan="1">357/36099.2%[97.6%-99.8%]</td></tr><tr><td colspan="1" rowspan="1">Below ReportableRange</td><td colspan="1" rowspan="1">Detected(Variable)</td><td colspan="1" rowspan="1">0/600%</td><td colspan="1" rowspan="1">0/600%</td><td colspan="1" rowspan="1">0/600%</td><td colspan="1" rowspan="1">0/1800%[0.0%-2.0%]</td></tr><tr><td colspan="1" rowspan="1">None(No Analyte)</td><td colspan="1" rowspan="1">N/A or NotDetected</td><td colspan="1" rowspan="1">420/420100%</td><td colspan="1" rowspan="1">420/420100%</td><td colspan="1" rowspan="1">420/420100%</td><td colspan="1" rowspan="1">1260/1260100%[99.7%-100%]</td></tr><tr><td colspan="1" rowspan="3">NDMAcinetobacterbaumanniiAR-Bank#0033</td><td colspan="1" rowspan="1">ReportableRange</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">150/150100%</td><td colspan="1" rowspan="1">149/15099.3%</td><td colspan="1" rowspan="1">150/150100%</td><td colspan="1" rowspan="1">449/45099.8%[98.8%-100%]</td></tr><tr><td colspan="1" rowspan="1">Below ReportableRange</td><td colspan="1" rowspan="1">Detected(Variable)</td><td colspan="1" rowspan="1">1/303.3%</td><td colspan="1" rowspan="1">1/303.3%</td><td colspan="1" rowspan="1">0/300%</td><td colspan="1" rowspan="1">2/902.2%[0.3%-7.8%]</td></tr><tr><td colspan="1" rowspan="1">None(No Analyte)</td><td colspan="1" rowspan="1">N/A or NotDetected</td><td colspan="1" rowspan="1">599/60099.8%</td><td colspan="1" rowspan="1">600/600100%</td><td colspan="1" rowspan="1">600/600100%</td><td colspan="1" rowspan="1">1799/180099.9%[99.7%-100%]</td></tr><tr><td colspan="1" rowspan="3">OXA-48-likeSerratiamarcescens GRE1659005</td><td colspan="1" rowspan="1">ReportableRange</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">120/120100%</td><td colspan="1" rowspan="1">119/12099.2%</td><td colspan="1" rowspan="1">120/120100%</td><td colspan="1" rowspan="1">359/36099.70%[98.5%-100%]</td></tr><tr><td colspan="1" rowspan="1">Below ReportableRange</td><td colspan="1" rowspan="1">Detected(Variable)</td><td colspan="1" rowspan="1">14/6023.3%</td><td colspan="1" rowspan="1">12/6020.0%</td><td colspan="1" rowspan="1">9/6015.0%</td><td colspan="1" rowspan="1">35/18019.4%[13.9%-26.0%]</td></tr><tr><td colspan="1" rowspan="1">None(No Analyte)</td><td colspan="1" rowspan="1">N/A or NotDetected</td><td colspan="1" rowspan="1">598/60099.7%</td><td colspan="1" rowspan="1">600/600100%</td><td colspan="1" rowspan="1">600/600100%</td><td colspan="1" rowspan="1">1798/180099.9%[99.6%-100%]</td></tr><tr><td colspan="1" rowspan="1">VIM</td><td colspan="1" rowspan="1">ReportableRange</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">120/120100%</td><td colspan="1" rowspan="1">120/120100%</td><td colspan="1" rowspan="1">120/120100%</td><td colspan="1" rowspan="1">360/360100%</td></tr><tr><td colspan="1" rowspan="3">EnterobactercloacaeAR-BANK#0154</td><td colspan="1" rowspan="1">cYanmaGYellowb</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">[99.0%-100%]</td></tr><tr><td colspan="1" rowspan="1">Below ReportableRange</td><td colspan="1" rowspan="1">Detected(Variable)</td><td colspan="1" rowspan="1">10/6016.7%</td><td colspan="1" rowspan="1">9/6015.0%</td><td colspan="1" rowspan="1">3/605.0%</td><td colspan="1" rowspan="1">22/18012.2%[7.8%-17.9%]</td></tr><tr><td colspan="1" rowspan="1">None(No Analyte)</td><td colspan="1" rowspan="1">N/A or NotDetected</td><td colspan="1" rowspan="1">599/60099.8%</td><td colspan="1" rowspan="1">600/600100%</td><td colspan="1" rowspan="1">600/600100%</td><td colspan="1" rowspan="1">1799/180099.9%[99.7%-100%]</td></tr></table>

# c. Linearity/assay reportable range:

Linearity

Linearity of the FilmArray Pneumonia Panel was performed to evaluate unbinned value response to changes in concentration and to determine if an offset exists between the unbinned value and a dPCR (digital PCR) determined molecular concentration (the unbinned value is calculated using real-time amplification data for the analyte relative to a known standard (Quantified Standard Material or QSM)).

A series of contrived samples were prepared in artificial bronchoalveolar lavage (aBAL) matrix such that each bacterial analyte was tested at six different concentrations spanning $1 . 5 - 7 . 8 \log 1 0$ (copies/mL). The input concentration of each cultured bacterium was determined by digital PCR (dPCR), which served as the reference concentrations for offset determination. Once prepared, each sample was tested repeatedly at each concentration on a single day for a total of eighteen replicate measures per analyte.

The unbinned values generated by the FilmArray Pneumonia Panel assays were plotted as a function of the nominal (dPCR) input concentrations to determine the linearity of the response. All plot data were observed to be linear and this was confirmed through mathematical modeling. For each analyte, data bounding the reporting range $( \mathord { \sim } 3 . 0 - 7 . 0 \log 1 0$ (copies/mL)) were fit using linear and quadratic models. In all cases, the linear fit model was preferred as it was sufficient to describe the change in unbinned values as a function of changing inputs and the addition of curvature did not improve the fit predictions.

The best fit linear model coefficients for each bacterial FilmArray Pneumonia Panel assay were evaluated (slope, y-intercept, and coefficient of determination (Adj $\mathsf { R } ^ { 2 } )$ )). The slope provides a direct measure of how the unbinned value responds to changes in input concentration, where a slope near 1.0 demonstrates that the unbinned values change in direct proportion to the input concentration. The best fit slopes of the assays varied within the range of $0 . 9 7 5 - 1 . 1 1 4$ , demonstrating that changes in the

unbinned values display a ${ \sim } 1 { : } 1$ relationship to changes in the input concentration. The y-intercept from the linear equation represents the theoretical offset predicted to exist at a zero input concentration. The Adj $\mathtt { R } ^ { 2 }$ value provides a measure of the goodness of fit between the linear model and the data, where values of 1.0 indicate perfect agreement and values of 0 indicate random association displaying no correlation. The best fit Adj $\mathtt { R } ^ { 2 }$ values of the assays were high, ranging from 0.951 – 0.989, further demonstrating strong agreement between the data and the linear model.

The FilmArray Pneumonia Panel unbinned value was shown to respond proportionately to changes in input concentration following a linear relationship defined by slopes around 1.0. The offset of each assay was found to be within the range $0 . 0 1 - 0 . 2 7 { \mathrm { - l o g 1 0 } }$ (copies/mL) and does not change substantially as a function of concentration over the reportable range. These data demonstrate that the Pneumonia Panel unbinned value is accurate and thus that the FilmArray Pneumonia Panel reported bin results will accurately reflect the concentration of analytes in a test sample.

d. Traceability, Stability, Expected values (controls, calibrators, or methods):

# Process Controls

Two process controls are included in each pouch:

RNA Process Control: The RNA Process Control assay targets an RNA transcript from the yeast Schizosaccharomyces pombe. The yeast is present in the pouch in a freeze-dried form and becomes rehydrated when sample is loaded. The control material is carried through all stages of the test process, including lysis, nucleic acid purification, reverse transcription, PCR1, dilution, PCR2, and DNA melting. A positive RNA Process Control result indicates that all steps carried out in the FilmArray Pneumonia Panel pouch were successful.

Quantified Standard Material (QSM) Control: The QSM assay detects a quantified standard synthetic nucleic acid that is subject to all stages of the test process following sample lysis (bead beating). A positive QSM control result indicates that the expected level of QSM is present (approximately $1 0 ^ { 6 }$ copies/mL) for use in determining assay and bin results for bacterial analytes.

Both control assays must be positive for the test run to pass. If the controls fail, the sample should be retested using a new pouch.

# External Controls

External controls are not provided with the FilmArray Pneumonia Panel. However, five frozen $( - 7 0 ^ { \circ } \mathrm { C } )$ external control mixes (ECMs) (see Table below) were provided to the study sites for daily testing during the prospective clinical trial and the clinical study testing contrived specimens. FilmArray operators were required to complete a valid ECM run (correct results obtained) on each day of clinical specimen testing (tested on a rotating basis).

Table 9: External Control Mixes (EC’s) Utilized in the Clinical Evaluations   

<table><tr><td rowspan=1 colspan=1>ExternalControlMixes</td><td rowspan=1 colspan=1>Expected Calls</td></tr><tr><td rowspan=1 colspan=1>ECM1</td><td rowspan=1 colspan=1>Acinetobacter calcoaceticus-baumannii complex, Aspergillus spp., Adenovirus, Influenza A,Respiratory Syncytial Virus, Chlamydophila pneumoniae, Legionella pneumophila, Proteus spp,blanDm (Carbapenem resistance), blakpc (Carbapenem resistance)</td></tr><tr><td rowspan=1 colspan=1>ECM2</td><td rowspan=1 colspan=1>Enterobacter aerogenes/cloacae complex, Cryptococcus neoformans/gattii, Coronavirus, MiddleEastern Respiratory Syndrome Coronavirus, Haemophilus influenzae, Moraxella catarrhalis,Pseudomonas aeruginosa, Streptococcus pneumoniae, blactx-m (Extended spectrum beta-lactamase),blaOxa48-like (Carbapenem resistance)</td></tr><tr><td rowspan=1 colspan=1>ECM3</td><td rowspan=1 colspan=1>Klebsiella pneumoniae group, Escherichia coli, Human Metapneumovirus, Influenza B, KlebsiellaoxytocaSerratia marcescens, Staphylococcus aureus, blavm (Carbapenem resistance), blamp (Carbapenemresistance)</td></tr><tr><td rowspan=1 colspan=1>ECM4</td><td rowspan=1 colspan=1>Staphylococcus aureus, Stenotrophomonas maltophiliaPneumocystis jirovecii, Human Rhinovirus/EnterovirusParainfluenza Virus, Streptococcus agalactiae, Mycoplasma pneumoniae, Streptococcus pyogenesmecA/mecC and MREJ (Methicillin resistance)</td></tr><tr><td rowspan=1 colspan=1>ECM5</td><td rowspan=1 colspan=1>None</td></tr></table>

External controls should be used in accordance with laboratory protocols and the appropriate accrediting organization requirements, as applicable. Molecular grade water or saline can be used as an external negative control. Previously characterized positive samples or negative samples spiked with well characterized organisms can be used as external positive controls.

# Specimen Stability

A study was performed to assess and validate specimen storage and handling recommendations for BAL-like and sputum-like specimens that will be tested with the FilmArray Pneumonia Panel. Contrived samples were preprared with representative panel organism spiked into pools of residual clinical BAL or sputum specimens. One sample (QL) was composed of spiked RNA virus, DNA virus, and aytipical bacteria at 3x LoD, as well as the organisms that were native to the pooled clinical specimen matrix. The other sample (QT) was composed of gram-positive and gram-negative bacteria (including fastidious species and isolates carting antimicrobial resistance genes) spiked at $1 0 ^ { 4 }$ copies/mL. Samples were tested on the FilmArray Pneumonia Panel immediately after sample preparation to generate control data (no storage; D0) and were then stored in refrigerated conditions for additional testing after one (D1) and two (D2) days, based on current lower respiratory specimen storage guidelines provided by the CDC. Ten replicates were tested at each storage timepoint and results were compared to the control.

In the contrived BAL and sputum samples, detection of analytes was consistent between the stored and unstored samples out to two days of refrigerated storage (9/10 or 10/10 detected result), with no noticeable changes in amplification data associated with storage. Only the mecA/C and MREJ result in the BAL sample was detected in fewer replicates than anticipated on the second day of storage (mecA/C and MREJ was reported as Detected in 7/9 replicates $( 7 7 . 8 \% )$ with a Staphylococcus aureus Detected result), but review of the amplification data did not suggest a difference in amplification or template levels in the stored sample compared to the unstored samples. Detection of native analytes (viruses, bacteria, and AMR genes) within the same contrived BAL and sputum samples was also consistent between the stored and unstored samples out to two days of refrigerated storage. Results are presented in the Table below.

Table 10: FilmArray Pneumonia Panel Results for Stored Contrived BAL and Sputum Samples   

<table><tr><td colspan="1" rowspan="3">Analyte</td><td colspan="1" rowspan="3">Strain</td><td colspan="1" rowspan="3">Concentration</td><td colspan="6" rowspan="1">Analyte Detection</td></tr><tr><td colspan="3" rowspan="1">BAL</td><td colspan="3" rowspan="1">Sputum</td></tr><tr><td colspan="1" rowspan="1">NoStorage(D0)</td><td colspan="1" rowspan="1">Day 1(D1)</td><td colspan="1" rowspan="1">Day 2(D2)</td><td colspan="1" rowspan="1">NoStorage(D0)</td><td colspan="1" rowspan="1">Day 1(D1)</td><td colspan="1" rowspan="1">Day 2(D2)</td></tr><tr><td colspan="9" rowspan="1">Spiked Analytes</td></tr><tr><td colspan="1" rowspan="1">Adenovirus(Type B3)</td><td colspan="1" rowspan="1">Zeptometrix0810062CF</td><td colspan="1" rowspan="1">3.0E+00TCID50/mL</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">10/10</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza Virus(Type 4a)</td><td colspan="1" rowspan="1">Zeptometrix0810060CF</td><td colspan="1" rowspan="1">7.5E+02TCID50/mL</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">10/10</td></tr><tr><td colspan="1" rowspan="1">RespiratorySyncytial Virus</td><td colspan="1" rowspan="1">Zeptometrix0810040ACF</td><td colspan="1" rowspan="1">3.0E+00TCID50/mL</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">10/10</td></tr><tr><td colspan="1" rowspan="1">Legionellapneumophila</td><td colspan="1" rowspan="1">ATCC 33152</td><td colspan="1" rowspan="1">1.5E+03FU/mL</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">10/10</td></tr><tr><td colspan="1" rowspan="1">NoStorage(D0)</td><td colspan="1" rowspan="1">Day 1(1)</td><td colspan="1" rowspan="1">Day 2(D2)</td><td colspan="1" rowspan="1">NoStorage(D0)</td><td colspan="1" rowspan="1">Day 1(D1)</td><td colspan="1" rowspan="1">Day 2(D2)</td></tr><tr><td colspan="1" rowspan="1">Haemophilusinfluenzae</td><td colspan="1" rowspan="1">ATCC 10211</td><td colspan="1" rowspan="7">1.0E+04copies/mL</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">10/10</td></tr><tr><td colspan="1" rowspan="1">Klebsiella oxytoca</td><td colspan="1" rowspan="2">GRE1254054</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">10/10</td></tr><tr><td colspan="1" rowspan="1">CTX-M</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">10/10</td></tr><tr><td colspan="1" rowspan="1">Proteus mirabilis</td><td colspan="1" rowspan="1">ATCC 35659</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">10/10</td></tr><tr><td colspan="1" rowspan="1">Staphylococcusaureus</td><td colspan="1" rowspan="2">ATCC 43300</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">9/10</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">10/10</td></tr><tr><td colspan="1" rowspan="1">mecA/C &amp; MREJ</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">7/9</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">10/10</td></tr><tr><td colspan="1" rowspan="1">Streptococcuspneumoniae</td><td colspan="1" rowspan="1">ATCC 6303</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">10/10</td></tr><tr><td colspan="9" rowspan="1">NativeAnalytes</td></tr><tr><td colspan="1" rowspan="1">Human Rhinovirus/Enterovirusa</td><td colspan="2" rowspan="10">Unknown</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">9/10</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">9/10</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">10/10</td></tr><tr><td colspan="1" rowspan="1">Influenza A</td><td colspan="3" rowspan="1">-</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">10/10</td></tr><tr><td colspan="1" rowspan="1">Enterobactercloacaeccomplexa</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">10/10</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="3" rowspan="1">-</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">10/10</td></tr><tr><td colspan="1" rowspan="1">Haemophilusinfluenzae</td><td colspan="3" rowspan="1">-</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">10/10</td></tr><tr><td colspan="1" rowspan="1">Pseudomonasaeruginosaa</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">10/10</td></tr><tr><td colspan="1" rowspan="1">Streptococcusagalactiae</td><td colspan="3" rowspan="1">-</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">10/10</td></tr><tr><td colspan="1" rowspan="1">Streptococcuspneumoniae</td><td colspan="3" rowspan="1">-</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">10/10</td></tr><tr><td colspan="1" rowspan="1">Staphylococcusaureus</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">10/10</td></tr><tr><td colspan="1" rowspan="1">mecA/C and MREJ</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">9/10</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">10/10</td></tr></table>

a Human Rhinovirus/Enterovirus, Enterobacter cloacae complex and Pseudomonas aeruginosa were present as native analytes in BAL and/or sputum versions of both the QL and QT samples. Data are shown only from Sample QL (containing viruses and atypical bacteria), but each of these analytes was also detected in all replicates (10/10) per storage condition in the BAL or sputum versions of Sample QT.

A secondary stability study was performed to asses the potential impact of storage on analyte detection and bin results with clinical BAL and sputum specimens collected, stored, and tested in the intended use clinical laboratory environment.Thirty-two clinical BAL and sputum specimens were evaluated and for the majority (22/32 or $6 8 . 8 \%$ ), no difference in analyte detection or amplification data $\mathrm { C p }$ or unbinned values) was observed between the stored and unstored samples. However, in ten specimens, an increase or decrease in unbinned values of $\ge 0 . 5$ -log was observed for one or more bacterial analytes, while viral detection and $\mathrm { C p }$ values (for the same specimens, when applicable) were unaffected. Unbinned values for $P$ . aeruginosa, S. aureus or Proteus spp. increased $\ge 0 . 5$ -log after either one or two days of storage in four specimens (4/32 or $1 2 . 5 \%$ ), suggesting possible organism growth during storage of the specimen. By contrast, in six specimens (6/32 or $1 8 . 8 \%$ ) a decrease of $\ge 0 . 5$ -log in the unbinned value and/or loss of detection for one or more bacterial analytes was seen after two days of refrigerated storage. In some cases, all bacterial analytes in the specimen changed similarly over the storage interval while in others, only one analyte appeared to change while another remained stable relative to the control.

These results support the specimen storage claims in the Product Insert that BAL-like and sputum-like specimens should be tested with the FilmArray Pneumonia Panel as soon as possible after collection, or if stored in refrigerated (approximately $2 { - } 8 ^ { \circ } \mathrm { C }$ ) conditions, may be tested up to one day after collection.

# Fresh vs. Frozen Study

In order to utilize frozen clinical respiratory samples in the evaluation of FilmArray Pneumonia Panel, an analytical study was conducted to demonstrate that preservation of samples by freezing at $\mathrm { \leq } \mathrm { - } 7 0 ^ { \circ } \mathrm { C }$ does not affect the accuracy of test results compared to freshly collected or freshly prepared samples.

In this study, a contrived clinical specimen set was prepared using residual clinical samples that were pre-screened with the FilmArray Pneumonia Panel and found to be negative for the analytes of interest. The testing set was comprised of 448 specimens (223 sputum and 225 bronchoalveolar lavage (BAL)), prepared in duplicate, that were co-spiked with a select set of analytes to investigate a potential for loss of detection following freezing: gram-positive bacteria (Streptococcus pyogenes), antimicrobial resistance genes (blaKPC carbapenemase (KPC) or blaNDM New Delhi metallo- $\cdot \beta$ -lactamase (NDM) hosted by seven different gram-negative bacteria: Acinetobacter baumannii, Enterobacter cloacae, Escherichia coli, Klebsiella oxytoca, Klebsiella pneumoniae, and Proteus mirabilis), atypical bacteria (Mycoplasma pneumoniae), RNA viruses (human metapneumovirus and influenza B), and DNA viruses (adenovirus). Multiple unique strains were evaluated per analyte type.

Specimen spiking for this study and the contrived specimen study was performed at the same time, following the contrived study protocol. All specimens were prepared in duplicate (with each replicate specimen spiked separately). Specimens were spiked with a variety of different isolates/strains for each organism at concentrations spanning the clinically observed ranges, then coded and randomized so that personnel testing the specimens were blinded to the expected test results. One replicate was tested fresh (without a freeze-thaw) on the FilmArray Pneumonia Panel at BioFire. The second aliquot was frozen and tested at a clinical site according to the prospective clinical study protocol alongside clinical (non

contrived) specimens.

Additionally, 36 clinical specimens (collected during the prospective study) were tested at BioFire (by users without knowledge of the expected results) from a frozen aliquot and the results were compared to the results that were obtained when the specimens were tested fresh at the clinical study sites. The same select panel of analytes was examined in the clinical specimens as the contrived specimen set. Study results demonstrated that there is no intrinsic property of the FilmArray Pneumonia Panel analytes or LRT specimens that lead to loss of analyte integrity and supports the use of frozen specimens in Pneumonia Panel clinical studies.

# e. Analytical Reactivity (Inclusivity)

Analytical reactivity of FilmArray Pneumonia Panel assays was evaluated with a collection of more than 350 viral and bacterial isolates representing relevant species, subspecies, strains, serotypes, or genotypes as well as temporal and geographic diversity for each of the panel analytes. Each isolate was tested in triplicate at concentrations near LoD or the lowest reportable level for the analyte. In silico analyses were performed between January 2016 and January 2018 of sequence data from public databases was also used to make predictions of assay reactivity for less common strains or serotypes that were not tested but that may be detected by the FilmArray Pneumonia Panel assays. Reactivity of FilmArray Pneumonia Panel assays with genetically diverse viruses, bacteria, and antimicrobial resistance genes was evaluated via a combination of empirical (wet) testing and in-silico analysis of sequences available in public databases. Limitations on reactivity are described in the Analytical Reactivity (Inclusivity) section. There is a risk of false positive results due to non-specific amplification and/or cross-reactivity with organisms found in the respiratory tract. Observed and predicted cross-reactivity for FilmArray Pneumonia Panel is described in the Analytical Specificity section. Erroneous results due to crossreactivity with organisms that were not evaluated or new variant sequences that emerge is also possible. Positive and negative predictive values are highly dependent on prevalence. False negative test results are more likely during peak activity when prevalence of disease is high. False positive test results are more likely during periods when prevalence is moderate to low.

Atypical bacteria and viruses were tested and detected at concentrations within $3 \times \mathrm { L o D }$ (Table 12 - Table 21). Bacteria were tested at a concentration of $1 . 0 \mathrm { E } { + } 0 4$ copies/mL (based on digital PCR of a single-copy gene in the bacterial chromosome) and the majority of isolates $( 9 4 . 4 \% )$ were detected with the expected bin result (Table 22 - Table 36) and when the bacterium was detected, the appropriate associated AMR gene(s) were also detected (Table 37 - Table 43).

Limitations on assay reactivity (observed and/or predicted by in silico analysis) with specific viral and bacterial isolates or sequences and specific AMR gene types or sequences are noted (Table 11). Most limitations are associated with single-base sequence variants under one or more assay primers.

Table 11: Limitations on Analytical Reactivity of FilmArray Pneumonia Panel Assays   

<table><tr><td rowspan=1 colspan=1>Limitation</td><td rowspan=1 colspan=1>Observed/PredictedResulta</td><td rowspan=1 colspan=1>Analyte</td><td rowspan=1 colspan=1>Strain/Isolate/Variant</td></tr><tr><td rowspan=4 colspan=1>Minor</td><td rowspan=4 colspan=1>Detectedmay be under-reported by onebin (≤10-fold)</td><td rowspan=1 colspan=1>Enterobacter cloacae complex</td><td rowspan=1 colspan=1>Enterobacter hormaechei(ATCC 49162)a</td></tr><tr><td rowspan=1 colspan=1>Klebsiella pneumoniae group</td><td rowspan=1 colspan=1>Klebsiella quasipneumoniae subsp.quasipneumoniae(DSM 28211)b</td></tr><tr><td rowspan=1 colspan=1>Moraxella catarrhalis</td><td rowspan=1 colspan=1>Moraxella catarrhalis(ATCC 23246)c</td></tr><tr><td rowspan=1 colspan=1>Streptococcus pyogenes</td><td rowspan=1 colspan=1>Streptococcus pyogenes(ATCC 19615)</td></tr><tr><td rowspan=6 colspan=1>Major</td><td rowspan=3 colspan=1>Detectedmay be under-reported by twoor more bins(&gt;10-fold)</td><td rowspan=1 colspan=1>Enterobacter cloacae complex</td><td rowspan=1 colspan=1>Enterobacter asburiae(ATCC 35953, 35954, 35955, and35957)d</td></tr><tr><td rowspan=1 colspan=1>Klebsiella aerogenes</td><td rowspan=1 colspan=1>Klebsiella (Enterobacter)aerogenes(ATCC 29751)e</td></tr><tr><td rowspan=1 colspan=1>mecA/C and MREJ</td><td rowspan=1 colspan=1>MREJ type xv</td></tr><tr><td rowspan=3 colspan=1>Not Detected</td><td rowspan=1 colspan=1>Acinetobactercalcoaceticus-baumanniicomplex</td><td rowspan=1 colspan=1>Acinetobacter nosocomialis(ATCC 700472)f</td></tr><tr><td rowspan=1 colspan=1>Pseudomonas aeruginosa</td><td rowspan=1 colspan=1>Pseudomonas aeruginosa(ATCC 25619)g</td></tr><tr><td rowspan=1 colspan=1>mecA/C and MREJ</td><td rowspan=1 colspan=1>MREJ type xviiiMREJ type xix</td></tr></table>

a. Minor limitation observed for this isolate due to sequence variant under a primer. Similar limitation predicted for two $E$ . hormaechei sequences $( 3 . 0 \% )$ from public databases. b. Minor limitation observed for this isolate due to sequence variant under a primer. Similar limitation predicted for one $K$ . quasipneumoniae sequence $( 2 0 . 0 \% )$ from public databases. Additional minor or major limitation predicted for four $K$ . pneumoniae sequences $( 0 . 3 \% )$ from public databases. c. Minor limitation observed for this isolate. Additional minor or major limitation predicted for two $M$ . catarrhalis sequences $( 3 . 3 \% )$ from public databases. d. Major limitation observed or predicted for these isolates due to sequence variance under primer. Similar limitation predicted for five $E$ . asburiae sequences $( 1 0 . 9 \% )$ , eight $E$ . cloacae sequences $( 2 . 2 \% )$ , and two $E$ . ludwigii sequences $( 1 6 . 7 \% )$ ) from public databases. e. Major limitation observed for this isolate due sequence variance under primers. Similar limitation predicted for six $K$ . aerogenes sequences $( 4 . 3 \% )$ from public databases f. ATCC 700472 could not be confirmed as A. nosocomialis by sequence and may be a non-Acinetobacter calcoaceticusbaumannii complex species that has been mis-identified. g. Major limitation observed for this isolate due to sequence variant under a primer. Similar limitation predicted for one $P$ . aeruginosa sequence $( 0 . 7 \% )$ from public databases.

Table 12: Adenovirus Isolates Tested and Detected   

<table><tr><td rowspan=2 colspan=1>Organism</td><td rowspan=2 colspan=1>Species</td><td rowspan=2 colspan=1>Serotypea</td><td rowspan=2 colspan=1>Source [Strain/Location/Year]</td><td rowspan=1 colspan=2>TestConcentration</td><td rowspan=2 colspan=1>Result</td></tr><tr><td rowspan=1 colspan=1>(copies/mL)</td><td rowspan=1 colspan=1>xLoD</td></tr><tr><td rowspan=29 colspan=1>Adenovirus</td><td rowspan=3 colspan=1>A</td><td rowspan=1 colspan=1>18</td><td rowspan=1 colspan=1>ATCC VR-19 [Washington D.C./1954]</td><td rowspan=1 colspan=1>9.2E+03</td><td rowspan=1 colspan=1>1x</td><td rowspan=29 colspan=1>AdenovirusDetected</td></tr><tr><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>ATCC VR-863 [Huie/Massachusetts]</td><td rowspan=1 colspan=1>2.7E+04</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>31</td><td rowspan=1 colspan=1>Zeptometrix 0810073CF</td><td rowspan=1 colspan=1>2.7E+04</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=12 colspan=1>B</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>Zeptometrix 0810062CF</td><td rowspan=1 colspan=1>1.8E+03</td><td rowspan=1 colspan=1>1x</td></tr><tr><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>ATCC VR-7 [Gomen/California/1954]</td><td rowspan=1 colspan=1>5.3E+03</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>7A</td><td rowspan=1 colspan=1>Zeptometrix 0810021CF</td><td rowspan=1 colspan=1>5.3E+03</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>7d/d2</td><td rowspan=1 colspan=1>Univ of Iowa Research Foundation[Iowa/2001]</td><td rowspan=1 colspan=1>5.3E+03</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>7h</td><td rowspan=1 colspan=1>Univ of Iowa Research Foundation[Iowa/1999]</td><td rowspan=1 colspan=1>5.3E+03</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>ATCC VR-12 [Slobitski/Massachusetts]</td><td rowspan=1 colspan=1>5.3E+03</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>ATCC VR-15 [De Wit/Netherlands/1955]</td><td rowspan=1 colspan=1>5.3E+03</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>ATCC VR-17 [CH.79/Saudi Arabia/1955]</td><td rowspan=1 colspan=1>5.3E+03</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>ATCC VR-1833 [128/Saudi Arabia/1956]</td><td rowspan=1 colspan=1>5.3E+03</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>34</td><td rowspan=1 colspan=1>ATCC VR-716 [Compton/1972]</td><td rowspan=1 colspan=1>5.3E+03</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>35</td><td rowspan=1 colspan=1>ATCC VR-718 [Holden]</td><td rowspan=1 colspan=1>5.3E+03</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>ATCC VR-1602 [Wan/Amsterdam/1988]</td><td rowspan=1 colspan=1>5.3E+03</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=4 colspan=1>C</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>ATCC VR-846 [Adenoid 6]</td><td rowspan=1 colspan=1>7.5E+03</td><td rowspan=1 colspan=1>1x</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>Zeptometrix 0810050CF</td><td rowspan=1 colspan=1>2.3E+04</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>Zeptometrix 0810020CF</td><td rowspan=1 colspan=1>2.3E+04</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>ATCC VR-6 [Tonsil 99]</td><td rowspan=1 colspan=1>2.3E+04</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=3 colspan=1>D</td><td rowspan=1 colspan=1>37</td><td rowspan=1 colspan=1>Zeptometrix 08100119CF</td><td rowspan=1 colspan=1>5.8E+02</td><td rowspan=1 colspan=1>1x</td></tr><tr><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>Zeptometrix 0810069CF</td><td rowspan=1 colspan=1>1.7E+03</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>Zeptometrix 0810115CF</td><td rowspan=1 colspan=1>1.7E+03</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=3 colspan=1>E</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>Zeptometrix 0810070CF</td><td rowspan=1 colspan=1>1.7E+04</td><td rowspan=1 colspan=1>1x</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>ATCC VR-1572 [RI-67/Missouri/1952-1953]</td><td rowspan=1 colspan=1>1.0E+04</td><td rowspan=1 colspan=1>0.6x</td></tr><tr><td rowspan=1 colspan=1>4a</td><td rowspan=1 colspan=1>Univ of Iowa Research Foundation [SCarolina/2004]</td><td rowspan=1 colspan=1>1.0E+04</td><td rowspan=1 colspan=1>0.6x</td></tr><tr><td rowspan=4 colspan=1>F</td><td rowspan=1 colspan=1>41</td><td rowspan=1 colspan=1>ATCC VR-930 [Tak 73-3544/Netherlands/1973]</td><td rowspan=1 colspan=1>5.5E+03</td><td rowspan=1 colspan=1>1x</td></tr><tr><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>NCPV 0101141v</td><td rowspan=1 colspan=1>1.6E+04</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>Zeptometrix 0810084CF</td><td rowspan=1 colspan=1>1.6E+04</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>41</td><td rowspan=1 colspan=1>Zeptometrix 0810085CF</td><td rowspan=1 colspan=1>1.6E+04</td><td rowspan=1 colspan=1>3x</td></tr></table>

a. In silico analysis of available sequences predicts that the FilmArray Pneumonia Panel will also react with Adenovirus B55, C57, all species D serotypes, and G52.

Table 13: Coronavirus Isolates Tested and Detected   

<table><tr><td rowspan=2 colspan=1>Organism</td><td rowspan=2 colspan=1>Type</td><td rowspan=2 colspan=1>Source [Location/Year]</td><td rowspan=1 colspan=2>Test Concentration</td><td rowspan=2 colspan=1>Result</td></tr><tr><td rowspan=1 colspan=1>(copies/mL)</td><td rowspan=1 colspan=1>X LoD</td></tr><tr><td rowspan=11 colspan=1>Coronavirus</td><td rowspan=2 colspan=1>229E</td><td rowspan=1 colspan=1>ATCC VR-740</td><td rowspan=1 colspan=1>8.1E+01</td><td rowspan=1 colspan=1>1x</td><td rowspan=11 colspan=1>CoronavirusDetected</td></tr><tr><td rowspan=1 colspan=1>Zeptometrix 0810229CF</td><td rowspan=1 colspan=1>2.4E+02</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=5 colspan=1>HKU1a</td><td rowspan=1 colspan=1>Clinical Specimen [Utah/2015]</td><td rowspan=1 colspan=1>1.0E+04</td><td rowspan=1 colspan=1>1x</td></tr><tr><td rowspan=1 colspan=1>Clinical Specimen [Detroit/2010]</td><td rowspan=1 colspan=1>3.0E+04</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>Clinical Specimen [Utah/2015]</td><td rowspan=1 colspan=1>3.0E+04</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>Clinical Specimen [Utah/2015]</td><td rowspan=1 colspan=1>3.0E+04</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>Clinical Specimen [S Carolina/2010]</td><td rowspan=1 colspan=1>3.0E+04</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=2 colspan=1>NL63</td><td rowspan=1 colspan=1>BEI NR-470b [Amsterdam/2003]</td><td rowspan=1 colspan=1>2.7E+02</td><td rowspan=1 colspan=1>1x</td></tr><tr><td rowspan=1 colspan=1>Zeptometrix 0810228CF</td><td rowspan=1 colspan=1>8.0E+02</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=2 colspan=1>OC43</td><td rowspan=1 colspan=1>ATCC VR-759c</td><td rowspan=1 colspan=1>4.6E+03</td><td rowspan=1 colspan=1>1x</td></tr><tr><td rowspan=1 colspan=1>Zeptometrix 0810024CF</td><td rowspan=1 colspan=1>1.4E+04</td><td rowspan=1 colspan=1>3x</td></tr></table>

a. No cultured isolates of Coronavirus HKU1 were available for testing. Five clinical NPS specimens containing Coronavirus HKU1 were collected from different regions of the US in 2010 and 2015, quantified molecularly, and tested. b. Organism obtained through the NIH Biodefense and Emerging Infections Research Resources Repository, NIAID, NIH: Human Coronavirus NL63, NR-470. c. Discontinued part #. See ATCC VR-1558.

Table 14: Human Metapneumovirus Isolates Tested and Detected   

<table><tr><td rowspan=2 colspan=1>Organism</td><td rowspan=2 colspan=1>Genotype</td><td rowspan=2 colspan=1>Serotype</td><td rowspan=2 colspan=1>Source [Location/Year]</td><td rowspan=1 colspan=2>Test Concentration</td><td rowspan=2 colspan=1>Result</td></tr><tr><td rowspan=1 colspan=1>(TCID50/mL)</td><td rowspan=1 colspan=1>X LoD</td></tr><tr><td rowspan=9 colspan=1>HumanMetapneumovirus</td><td rowspan=2 colspan=1>A1</td><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>Zeptometrix 0810161CF[Iowa10/2003]</td><td rowspan=1 colspan=1>5.0E+01</td><td rowspan=1 colspan=1>1x</td><td rowspan=9 colspan=1>HumanMetapneumovirusDetected</td></tr><tr><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>Zeptometrix 0810160CF[Iowa3/2002]</td><td rowspan=1 colspan=1>1.5E+02</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=2 colspan=1>A2</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>Zeptometrix 0810163CF[Iowa14/2003]</td><td rowspan=1 colspan=1>1.5E+02</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>27</td><td rowspan=1 colspan=1>Zeptometrix 0810164CF[Iowa27/2004]</td><td rowspan=1 colspan=1>1.5E+02</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=2 colspan=1>B1</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>Zeptometrix0810156CF[Peru2/2002]</td><td rowspan=1 colspan=1>1.5E+02</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>Zeptometrix 0810158CF[Peru3/2003]</td><td rowspan=1 colspan=1>1.5E+02</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=3 colspan=1>B2</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>Zeptometrix 0810159CF[Peru6/2003]</td><td rowspan=1 colspan=1>1.5E+02</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>Zeptometrix 0810157CF[Peru1/2002]</td><td rowspan=1 colspan=1>1.5E+02</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>18</td><td rowspan=1 colspan=1>Zeptometrix 0810162CF[Iowa18/2003]</td><td rowspan=1 colspan=1>1.0E+02</td><td rowspan=1 colspan=1>3x</td></tr></table>

Table 15: Human Rhinovirus and Enterovirus Isolates Tested and Detected   

<table><tr><td rowspan=2 colspan=1>Species</td><td rowspan=2 colspan=1>Serotype</td><td rowspan=2 colspan=1>Source [Strain/Location/Year]</td><td rowspan=1 colspan=2>Test Concentration</td><td rowspan=2 colspan=1>Result</td></tr><tr><td rowspan=1 colspan=1>(copies/mL)</td><td rowspan=1 colspan=1>X LoD</td></tr><tr><td rowspan=1 colspan=6>Human Rhinovirusa</td></tr><tr><td rowspan=8 colspan=1>A</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>Zeptometrix 0810012CFN [1A]</td><td rowspan=1 colspan=1>2.2E+03</td><td rowspan=1 colspan=1>1x</td><td rowspan=14 colspan=1>HumanRhinovirus/EnterovirusDetected</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>ATCC VR-482 [HGP]</td><td rowspan=1 colspan=1>1.7E+03</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>ATCC VR-1601 [68-CV11]</td><td rowspan=1 colspan=1>1.7E+03</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>ATCC VR-283 [11757/Washington DC/1960]</td><td rowspan=1 colspan=1>1.7E+03</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>34</td><td rowspan=1 colspan=1>ATCC VR-507b [137-3]</td><td rowspan=1 colspan=1>1.7E+03</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>57</td><td rowspan=1 colspan=1>ATCC VR-1600 [Ch47]</td><td rowspan=1 colspan=1>1.7E+03</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>77</td><td rowspan=1 colspan=1>ATCC VR-1187 [130-63]</td><td rowspan=1 colspan=1>1.7E+03</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>85</td><td rowspan=1 colspan=1>ATCC VR-1195 [50-525-CV54]</td><td rowspan=1 colspan=1>1.7E+03</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=6 colspan=1>B</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>ATCC VR-483 [FEB]</td><td rowspan=1 colspan=1>1.7E+03</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>ATCC VR-284 [1059/S Carolina/1959]</td><td rowspan=1 colspan=1>1.7E+03</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>ATCC VR-16633 [33342/N Carolina/1959]</td><td rowspan=1 colspan=1>1.7E+03</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>27</td><td rowspan=1 colspan=1>ATCC VR-1137 [5870]</td><td rowspan=1 colspan=1>1.7E+03</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>42</td><td rowspan=1 colspan=1>ATCC VR-338 [56822]</td><td rowspan=1 colspan=1>1.7E+03</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>83</td><td rowspan=1 colspan=1>ATCC VR-1193 [Baylor 7]</td><td rowspan=1 colspan=1>1.7E+03</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=6>Enterovirus</td></tr><tr><td rowspan=2 colspan=1>A</td><td rowspan=1 colspan=1>Coxsackievirus 10</td><td rowspan=1 colspan=1>ATCC VR-168 [NY/1950]</td><td rowspan=1 colspan=1>1.7E+03</td><td rowspan=1 colspan=1>3x</td><td rowspan=11 colspan=1>HumanRhinovirus/EnterovirusDetected</td></tr><tr><td rowspan=1 colspan=1>Enterovirus 71</td><td rowspan=1 colspan=1>ATCC VR-1432 [H]</td><td rowspan=1 colspan=1>1.7E+03</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=6 colspan=1>B</td><td rowspan=1 colspan=1>Coxsackievirus A9</td><td rowspan=1 colspan=1>Zeptometrix 0810017CF</td><td rowspan=1 colspan=1>1.7E+03</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>Coxsackievirus B3</td><td rowspan=1 colspan=1>Zeptometrix 0810074CF</td><td rowspan=1 colspan=1>1.7E+03</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>Coxsackievirus B4</td><td rowspan=1 colspan=1>Zeptometrix 0810075CF</td><td rowspan=1 colspan=1>1.7E+03</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>Echovirus 6</td><td rowspan=1 colspan=1>Zeptometrix 0810076CF</td><td rowspan=1 colspan=1>5.7E+02</td><td rowspan=1 colspan=1>1x</td></tr><tr><td rowspan=1 colspan=1>Echovirus 9</td><td rowspan=1 colspan=1>Zeptometrix 0810077CF</td><td rowspan=1 colspan=1>1.7E+03</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>Echovirus 11</td><td rowspan=1 colspan=1>Zeptometrix 0810023CF</td><td rowspan=1 colspan=1>1.7E+03</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=2 colspan=1>C</td><td rowspan=1 colspan=1>CoxsackievirusA21</td><td rowspan=1 colspan=1>ATCC VR-850 [Kuykendall/California/1952]</td><td rowspan=1 colspan=1>1.7E+03</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>CoxsackievirusA24</td><td rowspan=1 colspan=1>ATCC VR-583 [DN-19/Texas/1963]</td><td rowspan=1 colspan=1>1.7E+03</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>D</td><td rowspan=1 colspan=1>Enterovirus 68</td><td rowspan=1 colspan=1>ATCC VR-1823 [US/MO/2014-18947]</td><td rowspan=1 colspan=1>1.7E+03</td><td rowspan=1 colspan=1>3x</td></tr></table>

a The concentration used for Human Rhinovirus isolate testing was based on $3 \times$ the Enterovirus LoD concentration $\mathrm { 5 . 7 E ^ { + } } 0 2$ copies $/ \mathrm { m L }$ ).

Table 16: Influenza A Isolates Tested and Detected   

<table><tr><td rowspan=2 colspan=1>Organism</td><td rowspan=2 colspan=1>Subtype</td><td rowspan=2 colspan=2>Source [Strain/Location/Year]</td><td rowspan=1 colspan=2>Test Concentration</td><td rowspan=2 colspan=1>Result</td></tr><tr><td rowspan=1 colspan=2>(copies/mL) xLoD</td></tr><tr><td rowspan=3 colspan=1>InfluenzaA</td><td rowspan=1 colspan=5>Human</td><td rowspan=3 colspan=1>Influenza ADetected</td></tr><tr><td rowspan=2 colspan=1>H1N1</td><td rowspan=1 colspan=2>ATCC VR-219 [NWS/1933]</td><td rowspan=1 colspan=1>3.1E+02</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>ATCC VR-95 [PR/8/1934a]</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1.0E+03</td><td rowspan=1 colspan=1>1.5xa</td></tr></table>

<table><tr><td colspan="1" rowspan="2">OrganismSubtype</td><td colspan="1" rowspan="2">OrganismSubtype</td><td colspan="1" rowspan="2">Source [Strain/Location/Year]</td><td colspan="2" rowspan="1">Test Concentration</td><td colspan="1" rowspan="2">Result</td></tr><tr><td colspan="1" rowspan="1">(copies/mL)</td><td colspan="1" rowspan="1">xLoD</td></tr><tr><td colspan="1" rowspan="30"></td><td colspan="1" rowspan="7">′</td><td colspan="1" rowspan="1">ATCC VR-96 [Wiess/1943]</td><td colspan="1" rowspan="1">3.1E+02</td><td colspan="1" rowspan="1">3x</td><td colspan="1" rowspan="30"></td></tr><tr><td colspan="1" rowspan="1">ATCC VR-97 [FM/1/1947]</td><td colspan="1" rowspan="1">3.1E+02</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="1">ATCC VR-98 [Mal/302/1954]</td><td colspan="1" rowspan="1">3.1E+02</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="1">ATCC VR-546 [Denver/1/1957]</td><td colspan="1" rowspan="1">3.1E+02</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="1">Zeptometrix 0810036CF [NewCaledonia/20/1999]</td><td colspan="1" rowspan="1">3.1E+02</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="1">Zeptometrix 0810036CFN [SolomonIslands/3/2006]</td><td colspan="1" rowspan="1">3.1E+02</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="1">Zeptometrix 0810244CF [Brisbane/59/2007]</td><td colspan="1" rowspan="1">3.1E+02</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="1">H1N2</td><td colspan="1" rowspan="1">BEI NR-3478b [Kilbourne F63 A/NWS/1934(HA) xA/Rockefeller Institute/5/1957 (NA))</td><td colspan="1" rowspan="1">3.1E+02</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="7">H1N1pdm09</td><td colspan="1" rowspan="1">Zeptometrix 0810249CF [SwineNY/03/2009]</td><td colspan="1" rowspan="1">6.6E+02</td><td colspan="1" rowspan="1">1x</td></tr><tr><td colspan="1" rowspan="1">Zeptometrix 0810109CFJ [Canada/6294/2009]</td><td colspan="1" rowspan="1">3.1E+02</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="1">Zeptometrix 0810165CF [California/07/2009]</td><td colspan="1" rowspan="1">3.1E+02</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="1">Zeptometrix 0810166CF [Mexico/4108/2009]</td><td colspan="1" rowspan="1">3.1E+02</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="1">BEI NR-19823c [Netherlands/2629/2009]</td><td colspan="1" rowspan="1">3.1E+02</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="1">BEI NR-42938d [Georgia/F32551/2012]</td><td colspan="1" rowspan="1">3.1E+02</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="1">BEI NR-44345e [Hong Kong/H090-761-V1(0)/2009]</td><td colspan="1" rowspan="1">1.0E+03</td><td colspan="1" rowspan="1">1.5xa</td></tr><tr><td colspan="1" rowspan="8">H3N2</td><td colspan="1" rowspan="1">ATCC VR-810 [Port Chalmers/1/1973]</td><td colspan="1" rowspan="1">1.0E+02</td><td colspan="1" rowspan="1">1x</td></tr><tr><td colspan="1" rowspan="1">ATCC VR-776 [Alice (live attenuated vaccine)]</td><td colspan="1" rowspan="1">3.1E+02</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="1">Zeptometrix 0810238CF [Texas /50/2012]</td><td colspan="1" rowspan="1">3.1E+02</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="1">ATCC VR-547 [Aichi/2/1968]</td><td colspan="1" rowspan="1">3.1E+02</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="1">ATCC VR-544 [Hong Kong/8/1968]</td><td colspan="1" rowspan="1">3.1E+02</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="1">ATCC VR-822 [Victoria/3/1975]</td><td colspan="1" rowspan="1">3.1E+02</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="1">Zeptometrix 0810252CF [Wisconsin/67/2005]</td><td colspan="1" rowspan="1">3.1E+02</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="1">Zeptometrix 0810138CF [Brisbane/10/2007]</td><td colspan="1" rowspan="1">3.1E+02</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="1">H3N2v</td><td colspan="1" rowspan="1">ODHL1 [0hio/2012]</td><td colspan="1" rowspan="1">3.1E+02</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="3" rowspan="1">Avian</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">H2N2</td><td colspan="1" rowspan="1">BEI NR-2775f [Japan/305/1957]</td><td colspan="1" rowspan="1">3.1E+02</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="1">H2N3</td><td colspan="1" rowspan="1">MRIGlobalg [Mallard/Alberta/79/2003]</td><td colspan="1" rowspan="1">3.1E+02</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="1">H5N1</td><td colspan="1" rowspan="1">MRIGlobalg [Chicken/Yunnan/1251/2003]</td><td colspan="1" rowspan="1">3.1E+02</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="1">H5N2</td><td colspan="1" rowspan="1">MRIGlobalg [Northernpintail/Washinton/40964/2014]</td><td colspan="1" rowspan="1">3.1E+02</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="1">H5N3</td><td colspan="1" rowspan="1">BEI NR-9682h [Duck/Singapore/645/97]</td><td colspan="1" rowspan="1">3.1E+02</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="2">Organism</td><td colspan="1" rowspan="2">Subtype</td><td colspan="1" rowspan="2">Source [Strain/Location/Year]</td><td colspan="2" rowspan="1">Test Concentration</td><td colspan="1" rowspan="2">Result</td></tr><tr><td colspan="1" rowspan="1">(copies/mL)</td><td colspan="1" rowspan="1">xLoD</td></tr><tr><td colspan="1" rowspan="7"></td><td colspan="1" rowspan="1">H5N8</td><td colspan="1" rowspan="1">MRIGlobalg [Gyrfalcon/Washing-ton/41088-6/2014]</td><td colspan="1" rowspan="1">3.1E+02</td><td colspan="1" rowspan="1">3x</td><td colspan="1" rowspan="7">b</td></tr><tr><td colspan="1" rowspan="1">H7N7</td><td colspan="1" rowspan="1">MRIGlobal® [Netherlands/219/2003]</td><td colspan="1" rowspan="1">3.1E+02</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="1">H7N9</td><td colspan="1" rowspan="1">MRIGlobalg [Anhui/01/2013]</td><td colspan="1" rowspan="1">3.1E+02</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="1">H10N7</td><td colspan="1" rowspan="1">BEI NR-2765i [Chicken/Germany/N/49]</td><td colspan="1" rowspan="1">3.1E+02</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="4" rowspan="1">Swine</td></tr><tr><td colspan="1" rowspan="2">H1N1 Swine</td><td colspan="1" rowspan="1">ATCC VR-333 [Swine/Iowa/15/1930]</td><td colspan="1" rowspan="1">3.1E+02</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="1">ATCC VR-99 [Swine/1976/1931]</td><td colspan="1" rowspan="1">3.1E+02</td><td colspan="1" rowspan="1">3x</td></tr></table>

a. 1.5x the LoD for Influenza A H1N1pdm09 Zeptometrix 0810109CFN [SwineNY/03/2009] ( $6 . 6 { \mathrm { E } } ^ { + 0 2 }$ copies/mL). b. Genomic RNA obtained through BEI Resources NAID, NIH: Kilbourne F63: A/NWS/1934 (HA) x A/Rockefeller Institute/5/1957 (NA) (H1N2), Reassortant NWS-F, NR-9677. c. Virus obtained through BEI Resources, NIAID, NIH: Influenza A Virus, A/Netherlands/2629/2009 (H1N1)pdm09, NR-19823 d. Virus obtained through BEI Resources, NIAID, NIH: Influenza A Virus, A/Georgia/F32551/2012 (H1N1)pdm09, NR-42938. e. Virus obtained through BEI Resources, NIAID, NIH: Influenza A Virus, A/Hong Kong/H090-761-V1(0)/2009 (H1N1)pdm09, NR44345. f. Genomic RNA obtained through BEI Resources, NIAID, NIH: Genomic RNA from Influenza A Virus, A/Japan/305/1957 (H2N2), NR2775. g. Isolate provided and tested by MRI Global, Kansas City, MO. h. Genomic RNA obtained through BEI Resources, NIAID, NIH: Genomic RNA from Influenza A Virus, A/duck/Singapore/645/1997 (H5N3), Wild Type, NR-9682. i. Genomic RNA obtained through BEI Resources, NIAID, NIH: Genomic RNA from Influenza A Virus, A/chicken/Germany/N/1949 (H10N7), NR-2765.

Table 17: Influenza B Isolates Tested and Detected   
Table 18: Parainfluenza Virus Isolates Tested and Detected   

<table><tr><td rowspan=2 colspan=1>Organism</td><td rowspan=2 colspan=1>Lineage</td><td rowspan=2 colspan=1>[Strain/Location/Year], Source</td><td rowspan=1 colspan=2>Test Concentration</td><td rowspan=2 colspan=1>ReportedResult</td></tr><tr><td rowspan=1 colspan=1>(copies/mL)</td><td rowspan=1 colspan=1>X LoD</td></tr><tr><td rowspan=13 colspan=1>InfluenzaB</td><td rowspan=6 colspan=1>N/A</td><td rowspan=1 colspan=1>ATCC VR-101 [Lee/1940]</td><td rowspan=1 colspan=1>6.3E+02</td><td rowspan=1 colspan=1>3x</td><td rowspan=13 colspan=1>Influenza BDetected</td></tr><tr><td rowspan=1 colspan=1>ATCC VR-102 [Allen/1945]</td><td rowspan=1 colspan=1>6.3E+02</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>ATCC VR-103 [GL/1739/1954]</td><td rowspan=1 colspan=1>6.3E+02</td><td rowspan=1 colspan=1>3x</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>ATCC VR-296 [1/Maryland/1959]</td><td rowspan=1 colspan=1>6.3E+02</td><td rowspan=1 colspan=1>3x</td><td rowspan=1 colspan=1>Influenza B</td></tr><tr><td rowspan=1 colspan=1>ATCC VR-295 [2/Taiwan/1962]</td><td rowspan=1 colspan=1>6.3E+02</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>ATCC VR-786 [Brigit/Russia/1969]</td><td rowspan=1 colspan=1>6.3E+02</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=3 colspan=1>Victoria</td><td rowspan=1 colspan=1>ATCC VR-823 [5/Hong Kong/1972]</td><td rowspan=1 colspan=1>6.3E+02</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>Zeptometrix 0810258CF [2506/Malaysia/2004]</td><td rowspan=1 colspan=1>6.3E+02</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>CDC 2005743348 [1/0hio/2005]</td><td rowspan=1 colspan=1>6.3E+02</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=4 colspan=1>Yamagata</td><td rowspan=1 colspan=1>Zeptometrix 0810256CF [07/Florida/2004]</td><td rowspan=1 colspan=1>6.3E+02</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>Zeptometrix 0810255CF [04/Florida/2006]</td><td rowspan=1 colspan=1>2.1E+02</td><td rowspan=1 colspan=1>1x</td></tr><tr><td rowspan=1 colspan=1>Zeptometrix 0810241CF [1/Wisconsin/2010]</td><td rowspan=1 colspan=1>6.3E+02</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>Zeptometrix 0810239CF [2/Massachusetts/2012]</td><td rowspan=1 colspan=1>6.3E+02</td><td rowspan=1 colspan=1>3x</td></tr></table>

<table><tr><td rowspan=2 colspan=1>Organism</td><td rowspan=2 colspan=1>Type</td><td rowspan=2 colspan=1>Source [Strain/Location/Year]</td><td rowspan=1 colspan=2>Test Concentration</td><td rowspan=2 colspan=1>Result</td></tr><tr><td rowspan=1 colspan=1>(copies/mL)</td><td rowspan=1 colspan=1>xLoD</td></tr><tr><td rowspan=12 colspan=1>ParainfluenzaVirus</td><td rowspan=3 colspan=1>1</td><td rowspan=1 colspan=1>Zeptometrix 0810014CF</td><td rowspan=1 colspan=1>5.2E+03</td><td rowspan=1 colspan=1>1x</td><td rowspan=12 colspan=1>ParainfluenzaVirusDetected</td></tr><tr><td rowspan=1 colspan=1>BEI NR-48680a [FRA/29221106/2009]</td><td rowspan=1 colspan=1>1.6E+04</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>ATCC VR-94 [C-35/Washington DC/1957]</td><td rowspan=1 colspan=1>1.6E+04</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=2 colspan=1>2</td><td rowspan=1 colspan=1>Zeptometrix 0810015CF</td><td rowspan=1 colspan=1>1.5E+03</td><td rowspan=1 colspan=1>1x</td></tr><tr><td rowspan=1 colspan=1>ATCC VR-92 [Greer/Ohio/1955]</td><td rowspan=1 colspan=1>8.9E+02</td><td rowspan=1 colspan=1>0.6x</td></tr><tr><td rowspan=3 colspan=1>3</td><td rowspan=1 colspan=1>Zeptometrix 0810016CF</td><td rowspan=1 colspan=1>1.9E+02</td><td rowspan=1 colspan=1>1x</td></tr><tr><td rowspan=1 colspan=1>BEI NR-3233b [NIH 47885, Wash/47885/57]</td><td rowspan=1 colspan=1>5.7E+02</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>ATCC VR-93 [C-243/Washington DC/1957]</td><td rowspan=1 colspan=1>5.7E+02</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=4 colspan=1>4</td><td rowspan=1 colspan=1>Zeptometrix 0810060CF</td><td rowspan=1 colspan=1>8.1E+03</td><td rowspan=1 colspan=1>1x</td></tr><tr><td rowspan=1 colspan=1>ATCC VR-1378 [M-25/1958]</td><td rowspan=1 colspan=1>2.4E+04</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>Zeptometrix 0810060BCF</td><td rowspan=1 colspan=1>2.4E+04</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>ATCC VR-1377 [CH-19503/Washington DC/1962]</td><td rowspan=1 colspan=1>2.4E+04</td><td rowspan=1 colspan=1>3x</td></tr></table>

a Virus obtained through BEI Resources, NIAID, NIH: Human Parainfluenza Virus 1, HPIV1/FRA/29221106/2009, NR-48680. b Virus obtained through BEI Resources, NIAID, NIH: Human Parainfluenza Virus 3, NIH 47885, NR-3233.

Table 19: Respiratory Syncytial Virus Isolates Tested and Detected   

<table><tr><td rowspan=2 colspan=1>Organism</td><td rowspan=2 colspan=1>Type</td><td rowspan=2 colspan=1>Source [Strain/Location/Year]</td><td rowspan=1 colspan=2>Test Concentration</td><td rowspan=2 colspan=1>Result</td></tr><tr><td rowspan=1 colspan=1>(copies/mL)</td><td rowspan=1 colspan=1>xLoD</td></tr><tr><td rowspan=7 colspan=1>RespiratorySyncytial Virus</td><td rowspan=3 colspan=1>A</td><td rowspan=1 colspan=1>Zeptometrix 0810040ACF [2006]</td><td rowspan=1 colspan=1>4.3E+02</td><td rowspan=1 colspan=1>1x</td><td rowspan=7 colspan=1>RespiratorySyncytialVirusDetected</td></tr><tr><td rowspan=1 colspan=1>ATCC VR-26 [Long/Maryland/1956]</td><td rowspan=1 colspan=1>1.3E+03</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>ATCC VR-1540 [A2/Melbourne/1961]</td><td rowspan=1 colspan=1>1.3E+03</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=4 colspan=1>B</td><td rowspan=1 colspan=1>Zeptometrix 0810040CF [Ch-93 (18)-18]</td><td rowspan=1 colspan=1>1.3E+03</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>ATCC VR-1400 [WV/14617/1985]</td><td rowspan=1 colspan=1>1.3E+03</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>ATCC VR-955 [9320/Massachusetts/1977]</td><td rowspan=1 colspan=1>1.3E+03</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>ATCC VR-1580 [18537/WashingtonDC/1962]</td><td rowspan=1 colspan=1>1.3E+03</td><td rowspan=1 colspan=1>3x</td></tr></table>

Table 20: Legionella pneumophila Isolates Tested and Detected   

<table><tr><td rowspan=2 colspan=1>Organism</td><td rowspan=2 colspan=1>Source [Strain]</td><td rowspan=1 colspan=2>Test Concentration</td><td rowspan=2 colspan=1>Result</td></tr><tr><td rowspan=1 colspan=1>(copies/mL)</td><td rowspan=1 colspan=1>xLoD</td></tr><tr><td rowspan=4 colspan=1>Chlamydiapneumoniae</td><td rowspan=1 colspan=1>ATCC VR-2282 [TW-183/Taiwan/1965]</td><td rowspan=1 colspan=1>6.7E+01</td><td rowspan=1 colspan=1>1x</td><td rowspan=4 colspan=1>ChlamydiapneumoniaeDetected</td></tr><tr><td rowspan=1 colspan=1>ATCC VR-1310 [CWL-029]</td><td rowspan=1 colspan=1>2.0E+02</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>ATCC VR-1360 [CM-1/Georgia]</td><td rowspan=1 colspan=1>2.0E+02</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>ATCC 53592 [AR-39/Seattle/1983]</td><td rowspan=1 colspan=1>2.0E+02</td><td rowspan=1 colspan=1>3x</td></tr></table>

Table 21: Mycoplasma pneumoniae Isolates Tested and Detected   

<table><tr><td rowspan=2 colspan=1>Organism</td><td rowspan=2 colspan=1>Type</td><td rowspan=2 colspan=1>Source [Strain]</td><td rowspan=1 colspan=2>TestConcentration</td><td rowspan=2 colspan=1>Result</td></tr><tr><td rowspan=1 colspan=1>(copies/mL)</td><td rowspan=1 colspan=1>xLoD</td></tr><tr><td rowspan=9 colspan=1>Mycoplasmapneumoniae</td><td rowspan=3 colspan=1>1</td><td rowspan=1 colspan=1>Zeptometrix 0801579 [M129]</td><td rowspan=1 colspan=1>1.2E+03</td><td rowspan=1 colspan=1>1x</td><td rowspan=4 colspan=1>Mycoplasma</td></tr><tr><td rowspan=1 colspan=1>ATCC 29342 [M129-B7]</td><td rowspan=1 colspan=1>3.5E+03</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>ATCC 29085 [PI 1428]</td><td rowspan=1 colspan=1>3.5E+03</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=2 colspan=1>2</td><td rowspan=1 colspan=1>ATCC 15531-TTR [FH strain of Eaton Agent [NCTC 10119]</td><td rowspan=1 colspan=1>3.5E+03</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>ATCC 15492 [Mac]</td><td rowspan=1 colspan=1>3.5E+03</td><td rowspan=1 colspan=1>3x</td><td rowspan=1 colspan=1>pneumoniae</td></tr><tr><td rowspan=4 colspan=1>Un-known</td><td rowspan=1 colspan=1>ATCC 15293 [M52]</td><td rowspan=1 colspan=1>3.5E+03</td><td rowspan=1 colspan=1>3x</td><td rowspan=4 colspan=1>Detected</td></tr><tr><td rowspan=1 colspan=1>ATCC 15377 [Bru]</td><td rowspan=1 colspan=1>3.5E+03</td><td rowspan=1 colspan=1>3x</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>ATCC 39505 [Mutant 22]</td><td rowspan=1 colspan=1>3.5E+03</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>ATCC 49894 [UTMB-10P]</td><td rowspan=1 colspan=1>3.5E+03</td><td rowspan=1 colspan=1>3x</td></tr></table>

Table 22: Acinetobacter calcoaceticus-baumannii complex Isolates Tested and Detected   

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Source [Strain]</td><td rowspan=1 colspan=1>Testconcentration(copies/mL)</td><td rowspan=1 colspan=1>Result</td></tr><tr><td rowspan=6 colspan=1>A. baumannii</td><td rowspan=1 colspan=1>ATCC 9955 [6-561]</td><td rowspan=1 colspan=1>1.0E+04</td><td rowspan=13 colspan=1>Acinetobactercalcoaceticus-baumannii complexDetected</td></tr><tr><td rowspan=1 colspan=1>ATCC 19606 [2208 Type strain]</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>ATCC 17961 [CDC 7788]</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>AR-Bank #0033</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>GRE 1153064</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>GRE 1062081</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=3 colspan=1>A. calcoaceticus</td><td rowspan=1 colspan=1>ATCC 51432</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>ATCC 23055 [46]</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>ATCC 14987 [H0-1]</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>A. calcoaceticus subsp. anitratus</td><td rowspan=1 colspan=1>ATCC 15308 [NCTC 7844]</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>A. pittii</td><td rowspan=1 colspan=1>ATCC 19004 [57.071.228 ]</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>A. nosocomialisa</td><td rowspan=1 colspan=1>ATCC 17903 [NCTC 8102]</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>A.seifertii</td><td rowspan=1 colspan=1>CCUG 34785</td><td rowspan=1 colspan=1>1.0E+04</td></tr></table>

a A. nosocomialis ATCC 700472 was not detected at any concentration. Sequencing suggests the isolate may be mis-identified.

Table 23: Isolates Enterobacter cloacae Complex Tested and Detected   

<table><tr><td colspan="1" rowspan="1">Organism</td><td colspan="1" rowspan="1">Source [Strain]</td><td colspan="1" rowspan="1">Test concentration(copies/mL)</td><td colspan="1" rowspan="1">Result</td></tr><tr><td colspan="1" rowspan="5">E. cloacae</td><td colspan="1" rowspan="1">ATCC 49141 [AmMs 204]</td><td colspan="1" rowspan="1">1.0E+04</td><td colspan="1" rowspan="3">Enterobacter cloacae</td></tr><tr><td colspan="1" rowspan="1">ATCC BAA-1143 [Entb 55M]</td><td colspan="1" rowspan="1">1.0E+04</td></tr><tr><td colspan="1" rowspan="1">ATCC BAA-2341 [1101152]</td><td colspan="1" rowspan="1">1.0E+04</td></tr><tr><td colspan="1" rowspan="1">AR-Bank #0154</td><td colspan="1" rowspan="1">1.0E+04</td><td colspan="1" rowspan="3">complex Detected</td></tr><tr><td colspan="1" rowspan="1">NCTC 13464</td><td colspan="1" rowspan="1">1.0E+04</td></tr><tr><td colspan="1" rowspan="1">E. cloacae subsp. cloacae</td><td colspan="1" rowspan="1">ATCC 13047 [Type Strain]</td><td colspan="1" rowspan="1">1.0E+04</td></tr><tr><td colspan="1" rowspan="1">E. cloacae subsp.dissolvens</td><td colspan="1" rowspan="1">ATCC 23373Da [ICPB ED105]</td><td colspan="1" rowspan="1">1.0E+04</td><td colspan="1" rowspan="7">−</td></tr><tr><td colspan="1" rowspan="2">E. asburiae</td><td colspan="1" rowspan="1">ATCC 35953b [CDC 1497-78 TypeStrain]</td><td colspan="1" rowspan="1">1.0E+06b</td></tr><tr><td colspan="1" rowspan="1">ATCC 35957b [CDC 570-83]</td><td colspan="1" rowspan="1">1.0E+06b</td></tr><tr><td colspan="1" rowspan="2">E. hormaechei</td><td colspan="1" rowspan="1">ATCC 49162b [CDC 992-77]</td><td colspan="1" rowspan="1">1.0E+05b</td></tr><tr><td colspan="1" rowspan="1">ATCC BAA-2082</td><td colspan="1" rowspan="1">1.0E+04</td></tr><tr><td colspan="1" rowspan="1">E. kobei</td><td colspan="1" rowspan="1">CCUG 59410</td><td colspan="1" rowspan="1">1.0E+04</td></tr><tr><td colspan="1" rowspan="1">E. ludwigii</td><td colspan="1" rowspan="1">CCUG 23050</td><td colspan="1" rowspan="1">1.0E+04</td></tr></table>

a Genomic DNA from $E$ . cloacae subsp. dissolvens. b See Table 10: Limitations on Analytical Reactivity of FilmArray Pneumonia Panel Assays

Table 24: Escherichia coli Isolates and Cross-Reactive Species Tested and Detected   

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Source [Strain]</td><td rowspan=1 colspan=1>Test concentration(copies/mL)</td><td rowspan=1 colspan=1>Result</td></tr><tr><td rowspan=12 colspan=1>E. coli</td><td rowspan=1 colspan=1>ATCC 25922 [FDA strain Seattle 1946]</td><td rowspan=1 colspan=1>1.0E+04</td><td rowspan=12 colspan=1>Escherichia coliDetected</td></tr><tr><td rowspan=1 colspan=1>ATCC 43888 [CDC B568-73]</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>AR-Bank #0061</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>AR-Bank #0086</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>AR-Bank #0137</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>AR-Bank #0150</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>AR-Bank #0162</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>GRE 1062016</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>GRE 1252008</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>GRE 1252009</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>GRE 1256018</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>Zeptometrix 0801905 [Z136]</td><td rowspan=1 colspan=1>1.0E+04</td></tr></table>

Table 25: Haemophilus influenzae Isolates Tested and Detected   

<table><tr><td colspan="1" rowspan="1">Organism</td><td colspan="1" rowspan="1">Serotype</td><td colspan="1" rowspan="1">Source [Strain/Location/Year]</td><td colspan="1" rowspan="1">Testconcentration(copies/mL)</td><td colspan="1" rowspan="1">Result</td></tr><tr><td colspan="1" rowspan="6">H. influenzaea</td><td colspan="1" rowspan="1">Type a</td><td colspan="1" rowspan="1">ATCC 9006 [AMC 36-A-3 [610, PCM 2436]]</td><td colspan="1" rowspan="1">1.0E+04</td><td colspan="1" rowspan="6">HaemophilusinfluenzaeDetected</td></tr><tr><td colspan="1" rowspan="1">Type b</td><td colspan="1" rowspan="1">ATCC 10211 [AMC 36-A-1 [572]], Biotype ]</td><td colspan="1" rowspan="1">1.0E+04</td></tr><tr><td colspan="1" rowspan="1">Type c</td><td colspan="1" rowspan="1">ATCC 49699 [C 9007]</td><td colspan="1" rowspan="1">1.0E+04</td></tr><tr><td colspan="1" rowspan="1">Type d</td><td colspan="1" rowspan="1">ATCC 9008 [AMC 36-A-6 [611]]</td><td colspan="1" rowspan="1">1.0E+04</td></tr><tr><td colspan="1" rowspan="1">Type e</td><td colspan="1" rowspan="1">ATCC 8142 [AMC 36-A-7 [595, NCTC8472]]</td><td colspan="1" rowspan="1">1.0E+04</td></tr><tr><td colspan="1" rowspan="1">Type f</td><td colspan="1" rowspan="1">ATCC 700223 [GA1264]</td><td colspan="1" rowspan="1">1.0E+04</td></tr><tr><td rowspan="2"></td><td>Biogroup aegyptius</td><td>ATCC 11116 [180-a [NCTC 8502]]</td><td>1.0E+04</td><td rowspan="2"></td></tr><tr><td>Non-typeable</td><td>ATCC 51907 [Rd [KW20]]</td><td>1.0E+04</td></tr></table>

a See Table 11: Limitations on Analytical Reactivity of FilmArray Pneumonia Panel Assays

Table 26: Klebsiella (Enterobacter) aerogenes Isolates Tested and Detected   

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Source [Strain]</td><td rowspan=1 colspan=1>Test concentration(copies/mL)</td><td rowspan=1 colspan=1>Result</td></tr><tr><td rowspan=6 colspan=1>E. aerogenes</td><td rowspan=1 colspan=1>ATCC 13048 [NCTC 10006]</td><td rowspan=1 colspan=1>1.0E+04</td><td rowspan=6 colspan=1>EnterobacteraerogenesDetected</td></tr><tr><td rowspan=1 colspan=1>AR-Bank #0062</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>AR-Bank #0074</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>AR-Bank #0161</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>GRE 1254066</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>ATCC 29751 [MULB-250]</td><td rowspan=1 colspan=1>1.0E+07a</td></tr></table>

a See Table 11: Limitations on Analytical Reactivity of FilmArray Pneumonia Panel Assays

Table 27: Klebsiella oxytoca Isolates Tested and Detected   

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Source [Strain]</td><td rowspan=1 colspan=1>Test concentration(copies/mL)</td><td rowspan=1 colspan=1>Result</td></tr><tr><td rowspan=12 colspan=1>K. oxytoca</td><td rowspan=1 colspan=1>ATCC 13182 [479-2 Type strain]</td><td rowspan=1 colspan=1>1.0E+04</td><td rowspan=12 colspan=1>Klebsiella oxytocaDetected</td></tr><tr><td rowspan=1 colspan=1>ATCC 43086 [Pasco 201]</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>ATCC 49131 [AmMS 101]</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>ATCC 700324 [LBM 90.11.033]</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>ATCC 8724 [NRRL B-199]</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>AR-Bank #0147</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>JMI 2523</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>JMI 2661</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>JMI 7818</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>JMI 10678</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>JMI 14611</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>GRE 1254054</td><td rowspan=1 colspan=1>1.0E+04</td></tr></table>

Table 28: Klebsiella pneumoniae Isolates Tested and Detected   

<table><tr><td colspan="1" rowspan="1">Organism</td><td colspan="1" rowspan="1">Source [Strain]</td><td colspan="1" rowspan="1">Test concentration(copies/mL)</td><td colspan="1" rowspan="1">Result</td></tr><tr><td colspan="1" rowspan="6">K. pneumoniae</td><td colspan="1" rowspan="1">ATCC BAA-1705 [ART2008133]</td><td colspan="1" rowspan="1">1.0E+04</td><td colspan="1" rowspan="6">KlebsiellapneumoniaeDetected</td></tr><tr><td colspan="1" rowspan="1">AR-Bank #0068</td><td colspan="1" rowspan="1">1.0E+04</td></tr><tr><td colspan="1" rowspan="1">AR-Bank #0075</td><td colspan="1" rowspan="1">1.0E+04</td></tr><tr><td colspan="1" rowspan="1">AR-Bank #0076</td><td colspan="1" rowspan="1">1.0E+04</td><td colspan="1" rowspan="2"></td></tr><tr><td colspan="1" rowspan="1">AR-Bank #0079</td><td colspan="1" rowspan="1">1.0E+04</td></tr><tr><td colspan="1" rowspan="1">AR-Bank #0080</td><td colspan="1" rowspan="1">1.0E+04</td></tr><tr><td colspan="1" rowspan="10"></td><td colspan="1" rowspan="1">AR-Bank # 0097</td><td colspan="1" rowspan="1">1.0E+04</td><td colspan="1" rowspan="16"></td></tr><tr><td colspan="1" rowspan="1">AR-Bank #0107</td><td colspan="1" rowspan="1">1.0E+04</td></tr><tr><td colspan="1" rowspan="1">AR-Bank #0153</td><td colspan="1" rowspan="1">1.0E+04</td></tr><tr><td colspan="1" rowspan="1">GRE 1062084</td><td colspan="1" rowspan="1">1.0E+04</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">GRE 1355030</td><td colspan="1" rowspan="1">1.0E+04</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">JMI 328</td><td colspan="1" rowspan="1">1.0E+04</td></tr><tr><td colspan="1" rowspan="2">JMI 766</td><td colspan="1" rowspan="2">1.0E+04</td></tr><tr></tr><tr><td colspan="1" rowspan="1">NCTC 13465</td><td colspan="1" rowspan="1">1.0E+04</td></tr><tr><td colspan="1" rowspan="1">Zeptometrix 0801886</td><td colspan="1" rowspan="1">1.0E+04</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">K. pneumoniae subsp. ozaenae</td><td colspan="1" rowspan="1">ATCC 11296 [AMC 35-E-5]</td><td colspan="1" rowspan="1">1.0E+04</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">K. pneumoniae subsp. pneumoniae</td><td colspan="1" rowspan="1">ATCC 13883[NCTC 9633]</td><td colspan="1" rowspan="1">1.0E+04</td></tr><tr><td colspan="1" rowspan="1">K. pneumoniae subsp. rhinoscleromatis</td><td colspan="1" rowspan="1">ATCC 13884 [NCTC 5046]</td><td colspan="1" rowspan="1">1.0E+04</td></tr><tr><td colspan="1" rowspan="1">K. quasipneumoniae subsp.quasipneumoniae</td><td colspan="1" rowspan="1">DSM 28211² [01A030, SB11]</td><td colspan="1" rowspan="1">1.0E+05a</td></tr><tr><td colspan="1" rowspan="1">K. quasipneumoniae subsp.simipneumoniae</td><td colspan="1" rowspan="1">DSM 28212 [07A044, SB30]</td><td colspan="1" rowspan="1">1.0E+04</td></tr><tr><td colspan="1" rowspan="1">K. variicola</td><td colspan="1" rowspan="1">ATCC BAA-830 [F2R9]</td><td colspan="1" rowspan="1">1.0E+04</td></tr></table>

a See Table 11: Limitations on Analytical Reactivity of FilmArray Pneumonia Panel Assays

Table 29: Moraxella catarrhalis Isolates Tested and Detected   

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Source [Strain]</td><td rowspan=1 colspan=1>Test concentration(copies/mL)</td><td rowspan=1 colspan=1>Result</td></tr><tr><td rowspan=5 colspan=1>M. catarrhalis</td><td rowspan=1 colspan=1>ATCC 25238 [Ne 11]</td><td rowspan=1 colspan=1>1.0E+04</td><td rowspan=5 colspan=1>Moraxella catarrhalisDetected</td></tr><tr><td rowspan=1 colspan=1>ATCC 25240 [N9]</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>ATCC 8176 [20]</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>ATCC 23246a [NCTC 4103]</td><td rowspan=1 colspan=1>1.0E+05a</td></tr><tr><td rowspan=1 colspan=1>ATCC 49143 [Am MS 116]</td><td rowspan=1 colspan=1>1.0E+04</td></tr></table>

a See Table 11: Limitations on Analytical Reactivity of FilmArray Pneumonia Panel Assays

Table 30: Proteus spp. Isolates Tested and Detected   

<table><tr><td colspan="1" rowspan="1">Organism</td><td colspan="1" rowspan="1">Source [Strain]</td><td colspan="1" rowspan="1">Test concentration(copies/mL)</td><td colspan="1" rowspan="1">Result</td></tr><tr><td colspan="1" rowspan="6">P. mirabilis</td><td colspan="1" rowspan="1">ATCC 29906 [1003]</td><td colspan="1" rowspan="1">1.0E+04</td><td colspan="1" rowspan="8">Proteus spp.Detected</td></tr><tr><td colspan="1" rowspan="1">ATCC 33583 [571101]</td><td colspan="1" rowspan="1">1.0E+04</td></tr><tr><td colspan="1" rowspan="1">ATCC 35659 [LRA 08 01 73]</td><td colspan="1" rowspan="1">1.0E+04</td></tr><tr><td colspan="1" rowspan="1">AR-Bank #0156</td><td colspan="1" rowspan="1">1.0E+04</td></tr><tr><td colspan="1" rowspan="1">AR-Bank #0159</td><td colspan="1" rowspan="1">1.0E+04</td></tr><tr><td colspan="1" rowspan="1">GRE 1254053</td><td colspan="1" rowspan="1">1.0E+04</td></tr><tr><td colspan="1" rowspan="2">P. hauseri</td><td colspan="1" rowspan="1">ATCC 13315 [NCTC 4175 Strain Lehmann]</td><td colspan="1" rowspan="1">1.0E+04</td></tr><tr><td colspan="1" rowspan="1">ATCC 700826 [CDC 1732-80]</td><td colspan="1" rowspan="1">1.0E+04</td></tr><tr><td colspan="1" rowspan="2">P. penneri</td><td colspan="1" rowspan="1">ATCC 33519 [Type Strain CDC 1808-73]</td><td colspan="1" rowspan="1">1.0E+04</td><td colspan="1" rowspan="5"></td></tr><tr><td colspan="1" rowspan="1">ATCC 35197 [CDC 1655-67]</td><td colspan="1" rowspan="1">1.0E+04</td></tr><tr><td colspan="1" rowspan="3">P. vulgaris</td><td colspan="1" rowspan="1">ATCC 29905</td><td colspan="1" rowspan="1">1.0E+04</td></tr><tr><td colspan="1" rowspan="1">ATCC 33420</td><td colspan="1" rowspan="1">1.0E+04</td></tr><tr><td colspan="1" rowspan="1">ATCC 27973 [CDC 1787-64-SC1]</td><td colspan="1" rowspan="1">1.0E+04</td></tr></table>

Table 31: Pseudomonas aeruginosa Isolates Tested and Detected   

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Source [Strain]</td><td rowspan=1 colspan=1>Test concentration(copies/mL)</td><td rowspan=1 colspan=1>Result</td></tr><tr><td rowspan=11 colspan=1>P. aeruginosaa</td><td rowspan=1 colspan=1>ATCC 10145 [ MDB strain BU 277 type strain]</td><td rowspan=1 colspan=1>1.0E+04</td><td rowspan=11 colspan=1>PseudomonasaeruginosaDetected</td></tr><tr><td rowspan=1 colspan=1>ATCC BAA-1744 [109246]</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>ATCC 19429 [NCTC 6750]</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>ATCC 27853 [Boston 41501]</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>AR-Bank #0054</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>AR-Bank #0092</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>AR-Bank #0100</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>AR-Bank #0103</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>AR-Bank #0111</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>Creighton University PS28</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>NCTC 13437</td><td rowspan=1 colspan=1>1.0E+04</td></tr></table>

$P$ . aeruginosa ATCC 25619 was not detected at any concentration tested. See Table 11: Limitations on Analytical Reactivity of FilmArray Pneumonia Panel Assays

Table 32: Serratia marcescens Isolates Tested and Detected   
Table 33: Staphylococcus aureus Isolates Tested and Detected   

<table><tr><td colspan="1" rowspan="1">Organism</td><td colspan="1" rowspan="1">Source [Strain]</td><td colspan="1" rowspan="1">Test concentration(copies/mL)</td><td colspan="1" rowspan="1">Result</td></tr><tr><td colspan="1" rowspan="7">S. marcescens</td><td colspan="1" rowspan="1">ATCC 13880 [Type strain]</td><td colspan="1" rowspan="1">1.0E+04</td><td colspan="1" rowspan="7">Serratia marcescensDetected</td></tr><tr><td colspan="1" rowspan="1">ATCC 27137 [CDC 3100-71]</td><td colspan="1" rowspan="1">1.0E+04</td></tr><tr><td colspan="1" rowspan="1">ATCC 43297 [3G]</td><td colspan="1" rowspan="1">1.0E+04</td></tr><tr><td colspan="1" rowspan="1">ATCC BAA-885 [Type strain KRED]</td><td colspan="1" rowspan="1">1.0E+04</td></tr><tr><td colspan="1" rowspan="1">GRE 1659005</td><td colspan="1" rowspan="1">1.0E+04</td></tr><tr><td colspan="1" rowspan="1">GRE 1659004</td><td colspan="1" rowspan="1">1.0E+04</td></tr><tr><td colspan="1" rowspan="1">JMI 697</td><td colspan="1" rowspan="1">1.0E+04</td></tr><tr><td colspan="2" rowspan="2">Organism</td><td colspan="1" rowspan="2">Source [Strain] (PFGE Type if applicable)</td></tr><tr><td colspan="1" rowspan="2">Result</td></tr><tr><td colspan="2" rowspan="66">S. aureus</td><td colspan="1" rowspan="2">Staphylococcus aureus representing PFGE Types USA100-USA1200</td></tr><tr><td colspan="1" rowspan="65">StaphylococcusaureusDetected</td></tr><tr><td colspan="1" rowspan="2">NARSA NRS705 [PFGE USA100]</td></tr><tr><td colspan="1" rowspan="2">1.0E+04</td></tr><tr><td colspan="1" rowspan="2">NARSA NRS701 [PFGE USA200]</td></tr><tr><td colspan="1" rowspan="2">1.0E+04</td></tr><tr><td colspan="1" rowspan="2">ATCC BAA-1717 [PFGE USA300]</td></tr><tr><td colspan="1" rowspan="2">1.0E+05a</td></tr><tr><td colspan="1" rowspan="2">NARSA NRS683 [PFGE USA300]</td></tr><tr><td colspan="1" rowspan="2">1.0E+04</td></tr><tr><td colspan="1" rowspan="2">NARSA NRS662 [PFGE USA300]</td></tr><tr><td colspan="1" rowspan="2">1.0E+04</td></tr><tr><td colspan="1" rowspan="2">NARSA NRS707 [PFGE USA300]</td></tr><tr><td colspan="1" rowspan="2">1.0E+04</td></tr><tr><td colspan="1" rowspan="2">ATCC BAA-1707 [PFGE USA400]</td></tr><tr><td colspan="1" rowspan="2">1.0E+04</td></tr><tr><td colspan="1" rowspan="2">NARSA NRS691 [PFGE USA500]</td></tr><tr><td colspan="1" rowspan="2">1.0E+04</td></tr><tr><td colspan="2" rowspan="2">NARSA NRS648 [PFGE USA600]</td></tr><tr><td colspan="1" rowspan="2">1.0E+04</td></tr><tr><td colspan="2" rowspan="2">NARSA NRS689 [PFGE USA700]</td></tr><tr><td colspan="1" rowspan="2">1.0E+04</td></tr><tr><td colspan="2" rowspan="2">NARSA NRS668 [PFGE USA800]</td></tr><tr><td colspan="1" rowspan="2">1.0E+04</td></tr><tr><td colspan="2" rowspan="2">ATCC BAA-1749 [PFGE USA900 96:308]</td></tr><tr><td colspan="1" rowspan="2">1.0E+04</td></tr><tr><td colspan="2" rowspan="2">ATCC BAA-1759 [PFGE USA900 N7129]</td></tr><tr><td colspan="1" rowspan="2">1.0E+04</td></tr><tr><td colspan="2" rowspan="2">ATCC BAA-1700 [PFGE USA1000]</td></tr><tr><td colspan="1" rowspan="2">1.0E+04</td></tr><tr><td colspan="2" rowspan="2">BEI NR-46081 [PFGE USA1100 HIP12899]</td></tr><tr><td colspan="1" rowspan="2">1.0E+04</td></tr><tr><td colspan="2" rowspan="2">ATCC BAA-1765 [PFGE USA1200 102-04]</td></tr><tr><td colspan="1" rowspan="2">1.0E+04</td></tr><tr><td colspan="2" rowspan="2">ATCC BAA-1691 [Not USA100-1100]</td></tr><tr><td colspan="1" rowspan="1">1.0E+04</td></tr><tr><td colspan="2" rowspan="1">Methicillin Sensitive Staphylococcus aureus (MSSA)</td></tr><tr><td colspan="2" rowspan="2">ATCC 10832 [W00d 46]</td></tr><tr><td colspan="1" rowspan="2">1.0E+04</td></tr><tr><td colspan="2" rowspan="2">ATCC 14154 [Rose]</td></tr><tr><td colspan="1" rowspan="2">1.0E+04</td></tr><tr><td colspan="2" rowspan="2">ATCC 12600 [NCTC Type strain]</td></tr><tr><td colspan="1" rowspan="2">1.0E+04</td></tr><tr><td colspan="2" rowspan="2">ATCC 25923 [Seattle/1945]</td></tr><tr><td colspan="1" rowspan="2">1.0E+04</td></tr><tr><td colspan="2" rowspan="2">ATCC 29213 [Wichita]</td></tr><tr><td colspan="1" rowspan="2">1.0E+04</td></tr><tr><td colspan="2" rowspan="2">ATCC BAA-2421 [Mass/2010]</td></tr><tr><td colspan="1" rowspan="2">1.0E+04</td></tr><tr><td colspan="2" rowspan="2">Rennes 1060728</td></tr><tr><td colspan="1" rowspan="2">1.0E+04</td></tr><tr><td colspan="2" rowspan="2">GRE 1062519 [SCCmec Type: III / MREJ xix]b</td></tr><tr><td colspan="1" rowspan="1">1.0E+04</td></tr><tr><td colspan="2" rowspan="1">Borderline Resistant Staphylococcus aureus (BORSA)</td></tr><tr><td colspan="2" rowspan="2">SUN1 [Sunnybrook]</td></tr><tr><td colspan="1" rowspan="1">1.0E+04</td></tr><tr><td colspan="2" rowspan="1">Methicillin Resistant Staphylococcus aureus (MRSA)</td></tr><tr><td colspan="2" rowspan="2">ATCC 43300 [F182 Kansas / SCCmec Type: II]</td></tr><tr><td colspan="1" rowspan="2">1.0E+04</td></tr><tr><td colspan="2" rowspan="2">ATCC BAA-2422 [Worcester MA/2010 / SCCmec Type: II]</td></tr><tr><td colspan="1" rowspan="2">1.0E+04</td></tr><tr><td colspan="2" rowspan="2">ATCC BAA-1720 [MRSA252 / SCCmec Type: II / PFGEUSA200]</td></tr><tr><td colspan="1" rowspan="2">1.0E+04</td></tr><tr><td colspan="2" rowspan="2">NARSA NRS745 [CA-629 / SCCmec Type: V]</td></tr><tr><td colspan="1" rowspan="2">1.0E+04</td></tr><tr></tr></table>

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Source [Strain] (PFGE Type if applicable)</td><td rowspan=1 colspan=1>Test concentration(copies/mL)</td><td rowspan=1 colspan=1>Result</td></tr><tr><td rowspan=23 colspan=1></td><td rowspan=1 colspan=1>ATCC BAA-38 [E2125 / SCCmec Type: I]</td><td rowspan=1 colspan=1>1.0E+04</td><td rowspan=23 colspan=1></td></tr><tr><td rowspan=1 colspan=1>NARSA NRS686 [MREJ type i]</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>ATCC BAA-44 [HPV107 / SCCmec Type: I / PFGE: Iberian]</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>ATCC BAA-41 [NYBK2464 / SCCmec Type: II / PFGE 100]</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>NARSA NRS385 [MREJ type ii]</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>ATCC BAA-42 [HDE288 / SCCmec: Type VI / PFGE 800]</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>ATCC BAA-39 [HUSA304 / SCCmec Type: III]</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>ATCC BAA-40 [CPS22 / SCCmec Type: III]</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>GRE 1062264 [SCCmec Type: IV / MREJ type iv]</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>GRE 1055015 [SCCmec Type: IVa / MREJ type vi]</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>GRE 0759084 [SCCmec Type: IV / MREJ type v]</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>GRE 0860042 [SCCmec Type: III / MREJ type vii]</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>GRE 1052034 [MREJ ix]</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>GRE 1151100 [SCCmec Type: IV / MREJ type xi]</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>GRE 0960006 [MREJ type xii]</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>GRE 1055017 [SCCmec Type: IVa / MREJ type xiii]</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>GRE 0759163 [MREJ type xiv]</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>GRE 1062373 [MREJ type xv]</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>GRE 1057114 [MREJ type xvii]</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>GRE 1062292 [MREJ type xvii]</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=2>Methicillin Resistant Staphylococcus aureus (MRSA) - mecC+</td></tr><tr><td rowspan=1 colspan=1>ATCC BAA-2312 [M10/0061 / SCCmec Type: XI / mecC]</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>ATCC BAA-2313 [M10/0148 / SCCmec Type: XI / mecC]</td><td rowspan=1 colspan=1>1.0E+04</td></tr></table>

a. Staphylococcus aureus ATCC BAA-1717 was not detected at $1 . 0 \mathrm { E } ^ { + 0 4 }$ copies/mL but was detected at $1 . 0 \mathrm { E } { + 0 5 }$ copies/mL with accurate bin results. No limitation on reactivity could be identified based on the isolate sequence. b. MREJ type xix characterized as MSSA. Huletsky A, and Giroux R, inventors; Sequences for Detection and Identification of Methicillin-resistant Staphylococcus aureus (MRSA) of MREJ Types XI to XX. EP1934613A4. Jan. 19, 2011.

Table 34: Streptococcus agalactiae Isolates Tested and Detected   

<table><tr><td colspan="1" rowspan="1">Organism</td><td colspan="1" rowspan="1">Source [Strain]</td><td colspan="1" rowspan="1">Test concentration(copies/mL)</td><td colspan="1" rowspan="1">Result</td></tr><tr><td colspan="1" rowspan="4">S. agalactiae</td><td colspan="1" rowspan="1">NCTC 8017 [MK 104 P]</td><td colspan="1" rowspan="1">1.0E+04</td><td colspan="1" rowspan="4">StreptococcusagalactiaeDetected</td></tr><tr><td colspan="1" rowspan="1">ATCC 13813 [Ia/c Type Strain]</td><td colspan="1" rowspan="1">1.0E+04</td></tr><tr><td colspan="1" rowspan="1">ATCC 12403 [III Typing Strain D136C]</td><td colspan="1" rowspan="1">1.0E+04</td></tr><tr><td colspan="1" rowspan="1">ATCC 12386 [Grouping strain O90R]</td><td colspan="1" rowspan="1">1.0E+04</td></tr><tr><td rowspan="3"></td><td>ATCC BAA-611 [V 2603 V/R]</td><td>1.0E+04</td><td rowspan="3"></td></tr><tr><td>ATCC BAA-2669 [VIII 5030-08]</td><td>1.0E+04</td></tr><tr><td>Clinical Isolate [Utah/2010/CI03]</td><td>1.0E+04</td></tr></table>

Table 35: Streptococcus pneumoniae Isolates Tested and Detected   

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Serotype</td><td rowspan=1 colspan=1>Source [Strain]</td><td rowspan=1 colspan=1>Test concentration(copies/mL)</td><td rowspan=1 colspan=1>Result</td></tr><tr><td rowspan=7 colspan=1>S. pneumoniae</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>ATCC 6303</td><td rowspan=1 colspan=1>1.0E+04</td><td rowspan=7 colspan=1>StreptococcuspneumoniaeDetected</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>ATCC BAA-341 [SPN1439-106]</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>11A</td><td rowspan=1 colspan=1>NCTC 11900 [Gorman]</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>ATCC 700672 [VH14]</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>19A</td><td rowspan=1 colspan=1>ATCC 700673 [Hungary19A-6]</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>Non-capsulated</td><td rowspan=1 colspan=1>ATCC BAA-255[R6]</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>unknown</td><td rowspan=1 colspan=1>ATCC BAA-1409 [62076]</td><td rowspan=1 colspan=1>1.0E+04</td></tr></table>

Table 36: Streptococcus pyogenes Isolates Tested and Detected   

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Source [Strain]</td><td rowspan=1 colspan=1>Test concentration(copies/mL)</td><td rowspan=1 colspan=1>Result</td></tr><tr><td rowspan=8 colspan=1>S. pyogenes</td><td rowspan=1 colspan=1>ATCC 12344 [Typing strain T1, NCIB 11841, SF 130]</td><td rowspan=1 colspan=1>1.0E+04</td><td rowspan=8 colspan=1>StreptococcuspyogenesDetected</td></tr><tr><td rowspan=1 colspan=1>ATCC 12348 [Typing strain S43 Type 6]</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>ATCC 12384 [Typing strain C203 Type 3]</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>ATCC 19615a [Bruno]</td><td rowspan=1 colspan=1>1.0E+06a</td></tr><tr><td rowspan=1 colspan=1>ATCC 700294 [SF370; M1 GAS [M-type 1 T-type 1]]</td><td rowspan=1 colspan=1>1.0E+05b</td></tr><tr><td rowspan=1 colspan=1>ATCC 49399 [QC A62]</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>ATCC BAA-595 [MGAS 315, serotype M3]</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>ATCC BAA-947 [MGAS 5005, serotype M1]</td><td rowspan=1 colspan=1>1.0E+04</td></tr></table>

a. See Table 10: Limitations on Analytical Reactivity of FilmArray Pneumonia Panel Assays b. Streptococcus pyogenes ATCC 700294 was detected in 3/5 replicates at $1 . 0 \mathrm { E } ^ { + 0 4 }$ copies $/ \mathrm { m L }$ with $1 0 ^ { 4 }$ copies/mL bin results and 3/3 replicates at $1 . 0 \mathrm { E } ^ { + 0 5 }$ copies $/ \mathrm { m L }$ with $1 0 ^ { 5 }$ copies $/ \mathrm { m L }$ bin results. No limitation on reactivity could be identified based on isolate sequence.

The following tables (Table 37- Table 43) describe the reactivity of the AMR genes assays with different AMR gene types in various host bacteria. Results are shown for the isolates tested as well as predictions of reactivity with untested AMR gene types based on in silico analysis of sequences retrieved from public databases from June 2016 to Sept 2016.

Table 37: Isolates Containing mecA/C and MREJ Tested and Detected   

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=2>Source [Strain]</td><td rowspan=1 colspan=1>Testconcentration(copies/mL)</td><td rowspan=1 colspan=1>Result</td></tr><tr><td rowspan=34 colspan=1>S. aureus</td><td rowspan=1 colspan=3>Methicillin Sensitive Staphylococcus aureus (MSSA) containing SCCmeccassette (non-functional mecA variant)</td><td rowspan=34 colspan=1>mecA/CandMREJ Detected</td></tr><tr><td rowspan=1 colspan=2>ATCC BAA-2421 [Mass/2010]</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=3>Methicillin Resistant Staphylococcus aureus (MRSA)(Characterized SCCmec Types)</td></tr><tr><td rowspan=1 colspan=2>NARSA NRS705 [NY-12 / SCCmec Type: II]</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=2>NARSA NRS701 [MN-082 / SCCmec Type: II]</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=2>ATCC BAA-1717 [TCH1516 / SCCmec Type: IVa]</td><td rowspan=1 colspan=1>1.0E+05a</td></tr><tr><td rowspan=1 colspan=2>NARSA NRS683 [GA-298 / SCCmec Type: IV]</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=2>NARSA NRS662 [CO-34 / SCCmec Type: IV]</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=2>NARSA NRS707 [NY-155 / SCCmec Type: IV]</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=2>ATCC BAA-1707 [MW2 /SCCmec Type: IV]</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=2>NARSA NRS691 [GA-62 /SCCmec Type: IV]</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=2>NARSA NRS648 [CA-347 /SCCmec Type:II or IV]</td><td rowspan=1 colspan=1>1.0E+05a</td></tr><tr><td rowspan=1 colspan=2>NARSA NRS689 [GA-442 / SCCmec Type: IV]</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=2>NARSA NRS668 [CO-72 / SCCmec Type: IV]</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=2>ATCC BAA-1700 [HFH-33798 / SCCmec Type: IVb]</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=2>BEI NR-46081b (NRSA NRS484) [HIP12899 / SCCmec Type: IV]</td><td rowspan=1 colspan=1>1.0E+05a</td></tr><tr><td rowspan=1 colspan=2>ATCC BAA-1691 [HFH-30137 / SCCmec Type: IV]</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=2>ATCC 43300 [F182 Kansas / SCCmec Type: II ]</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=2>ATCC BAA-2422 [Worcester MA/2010 / SCCmec Type: II]</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=2>ATCC BAA-1720 [MRSA252 / SCCmec Type: II]</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=2>NARSA NRS745 [CA-629 / SCCmec Type: IV or V]</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=3>Methicillin Resistant Staphylococcus aureus (MRSA)(Characterized MREJ Types)</td></tr><tr><td rowspan=1 colspan=2>ATCC BAA-38 [MREJ type i]</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=2>NARSA NRS686 [MREJ type i]</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=2>ATCC BAA-44 [MREJ type ii]</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=2>ATCC BAA-41 [MREJ type ii]</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=2>NARSA NRS385 [MREJ type ii]</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=2>ATCC BAA-42 [MREJ type ii]</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=2>ATCC BAA-39 [MREJ type iii]</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=2>ATCC BAA-40 [MREJ type iv]</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>GRE 1062264 [MREJ type iv]</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>GRE 1055015 [MREJ type vi]</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>GRE 0860042 [MREJ type vii]</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>GRE 1052034 [MREJ type ix]</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1.0E+04</td></tr></table>

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=3>Source [Strain]</td><td rowspan=1 colspan=1>Testconcentration(copies/mL)</td><td rowspan=1 colspan=1>Result</td></tr><tr><td rowspan=11 colspan=1></td><td rowspan=1 colspan=3>GRE 1151100 [MREJ type xi]</td><td rowspan=1 colspan=1>1.0E+04</td><td rowspan=6 colspan=1></td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>[MREJ type xii</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=3>GRE 1055017 [MREJ type xiii]</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=2>GRE 07591633 [MREJ type xiv]</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=3>GRE 1062373 [MREJ type xv]</td><td rowspan=1 colspan=1>1.0E+06b</td></tr><tr><td rowspan=1 colspan=3>GRE 1057114 [MREJ type xvii]</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=3>GRE 1062292 [MREJ type xviii]</td><td rowspan=1 colspan=1>3.3E+08</td><td rowspan=1 colspan=1>mecA/C and MREJ</td></tr><tr><td rowspan=1 colspan=3>GRE 1062519 [MREJ type xix]cd</td><td rowspan=1 colspan=1>1.0E+07</td><td rowspan=1 colspan=1>Not Detected</td></tr><tr><td rowspan=1 colspan=4>Methicillin Resistant Staphylococcus aureus (MRSA) (SCCmec Type: XI / mecC/ mecALGA251 variants)</td><td rowspan=5 colspan=1>mecA/C andMREJ Detected</td></tr><tr><td rowspan=1 colspan=3>ATCC BAA-2312 [M10/0061 / SCCmec Type: XI / mecC]</td><td rowspan=1 colspan=1>1.0E+04</td><td rowspan=1 colspan=1>−</td></tr><tr><td rowspan=1 colspan=3>ATCC BAA-2313 [M10/0148 / SCCmec:Type XI / mecC ]</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=2 colspan=1>S. argenteus</td><td rowspan=1 colspan=3>Methicillin Resistant Staphylococcus argenteus</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=3>DSM 28299 [MSHR-1132]</td><td rowspan=1 colspan=1>1.00E+05</td></tr></table>

a. mecA/C and MREJ assays positive in less than three replicates at $1 . 0 \mathrm { E } ^ { + 0 4 }$ copies/mL, no sequence-based limitation on reactivity identified. b. Bacteria obtained through NARSA for distribution by BEI Resources, NIAID, NIH: Staphylococcus aureus, Strain HIP12899, NR-46081 c. See Table 6 for limitation. d. MREJ type xix characterized MSSA. Huletsky A, and Giroux R, inventors; Sequences for Detection and Identification of Methicillinresistant Staphylococcus aureus (MRSA) of MREJ Types XI to XX. EP1934613A4. Jan. 19, 2011.

Table 38: Isolates Containing the blaCTX-M gene Tested and Detected   

<table><tr><td colspan="1" rowspan="1">CTX-M Type</td><td colspan="1" rowspan="1">Organism</td><td colspan="1" rowspan="1">Source</td><td colspan="1" rowspan="1">Test concentration(copies/mL)</td><td colspan="1" rowspan="1">Result</td></tr><tr><td colspan="1" rowspan="5">CTX-M</td><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">AR-Bank #0137a</td><td colspan="1" rowspan="1">1.0E+04</td><td colspan="1" rowspan="12">CTX-M Detected</td></tr><tr><td colspan="1" rowspan="1">K. oxytoca</td><td colspan="1" rowspan="1">GRE 1254054</td><td colspan="1" rowspan="1">1.0E+04</td></tr><tr><td colspan="1" rowspan="3">K. pneumoniae</td><td colspan="1" rowspan="1">AR-Bank #0068a</td><td colspan="1" rowspan="1">1.0E+04</td></tr><tr><td colspan="1" rowspan="1">AR-Bank #0153a</td><td colspan="1" rowspan="1">1.0E+04</td></tr><tr><td colspan="1" rowspan="1">GRE 1355030</td><td colspan="1" rowspan="1">1.0E+04</td></tr><tr><td colspan="1" rowspan="1">CTX-M-1</td><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">AR-Bank #0162</td><td colspan="1" rowspan="1">1.0E+04</td></tr><tr><td colspan="1" rowspan="1">CTX-M-2</td><td colspan="1" rowspan="1">K. pneumoniae</td><td colspan="1" rowspan="1">AR-Bank #0107</td><td colspan="1" rowspan="1">1.0E+04</td></tr><tr><td colspan="1" rowspan="1">CTX-M-8</td><td colspan="1" rowspan="1">E. aerogenes</td><td colspan="1" rowspan="1">GRE 1254066</td><td colspan="1" rowspan="1">1.0E+04</td></tr><tr><td colspan="1" rowspan="2">CTX-M-9</td><td colspan="1" rowspan="1">E.coli</td><td colspan="1" rowspan="1">AR-Bank #0086</td><td colspan="1" rowspan="1">1.0E+04</td></tr><tr><td colspan="1" rowspan="1">E.cloacae</td><td colspan="1" rowspan="1">NCTC 13464</td><td colspan="1" rowspan="1">1.0E+04</td></tr><tr><td colspan="1" rowspan="1">CTX-M-14</td><td colspan="1" rowspan="1">K. pneumoniae</td><td colspan="1" rowspan="1">AR-Bank #0079</td><td colspan="1" rowspan="1">1.0E+04</td></tr><tr><td colspan="1" rowspan="1">CTX-M-15</td><td colspan="1" rowspan="1">E.coli</td><td colspan="1" rowspan="1">Zeptometrix 0801905</td><td colspan="1" rowspan="1">1.0E+04</td></tr><tr><td colspan="1" rowspan="1">CTX-M-22</td><td colspan="1" rowspan="1">P. mirabilis</td><td colspan="1" rowspan="1">GRE 1254053</td><td colspan="1" rowspan="1">1.0E+04</td><td colspan="1" rowspan="2"></td></tr><tr><td colspan="1" rowspan="1">CTX-M-25</td><td colspan="1" rowspan="1">K. pneumoniae</td><td colspan="1" rowspan="1">NCTC 13465</td><td colspan="1" rowspan="1">1.0E+04</td></tr></table>

a. AR-Bank #0137, AR-Bank $\# 0 0 6 8$ , AR-Bank $\# 0 1 5 3$ were confirmed to carry the blaCTX-M gene by an independent assay

Table 39: Isolates Containing the blaIMP gene Tested and Detected   

<table><tr><td rowspan=1 colspan=1>IMP Type</td><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Source</td><td rowspan=1 colspan=1>Test concentration(copies/mL)</td><td rowspan=1 colspan=1>Result</td></tr><tr><td rowspan=3 colspan=1>IMP</td><td rowspan=1 colspan=1>E. aerogenes</td><td rowspan=1 colspan=1>AR-Bank #0161</td><td rowspan=1 colspan=1>1.0E+04</td><td rowspan=9 colspan=1>IMPDetected</td></tr><tr><td rowspan=1 colspan=1>E. coli</td><td rowspan=1 colspan=1>GRE 1062016</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>K. pneumoniae</td><td rowspan=1 colspan=1>AR-Bank #0080</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>IMP-1</td><td rowspan=1 colspan=1>P. aeruginosa</td><td rowspan=1 colspan=1>AR-Bank #0103</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>IMP-3</td><td rowspan=1 colspan=1>E. coli</td><td rowspan=1 colspan=1>GRE 1252008</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>IMP-4</td><td rowspan=1 colspan=1>A. baumannii</td><td rowspan=1 colspan=1>GRE 1062081</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>IMP-8</td><td rowspan=1 colspan=1>K. pneumoniae</td><td rowspan=1 colspan=1>GRE 1062084</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>IMP-9</td><td rowspan=1 colspan=1>E. coli</td><td rowspan=1 colspan=1>GRE 1252009</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>IMP-14</td><td rowspan=1 colspan=1>P. aeruginosa</td><td rowspan=1 colspan=1>AR-Bank #0092</td><td rowspan=1 colspan=1>1.0E+04</td></tr></table>

Table 40: Isolates Containing the blaKPC gene Tested and Detected   

<table><tr><td rowspan=1 colspan=1>KPC Type</td><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Source</td><td rowspan=1 colspan=1>Test concentration(copies/mL)</td><td rowspan=1 colspan=1>Result</td></tr><tr><td rowspan=6 colspan=1>KPC</td><td rowspan=1 colspan=1>E. cloacae</td><td rowspan=1 colspan=1>ATCC BAA-2341</td><td rowspan=1 colspan=1>1.0E+04</td><td rowspan=15 colspan=1>KPCDetected</td></tr><tr><td rowspan=1 colspan=1>E. hormaechi</td><td rowspan=1 colspan=1>ATCC BAA-2082</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>P. mirabilis</td><td rowspan=1 colspan=1>AR-Bank #0156</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>K. oxytoca</td><td rowspan=1 colspan=1>AR-Bank #0147</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>K. pneumoniae</td><td rowspan=1 colspan=1>AR-Bank #0097</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>K. oxytoca</td><td rowspan=1 colspan=1>JMI 2523</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=5 colspan=1>KPC-2</td><td rowspan=1 colspan=1>K. oxytoca</td><td rowspan=1 colspan=1>JMI 7818</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>K. pneumoniae</td><td rowspan=1 colspan=1>Zeptometrix 0801886</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>K. pneumoniae</td><td rowspan=1 colspan=1>JMI 328</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>K. pneumoniae</td><td rowspan=1 colspan=1>ATCC BAA-1705</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>S. marcescens</td><td rowspan=1 colspan=1>JMI 697</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=2 colspan=1>KPC-3</td><td rowspan=1 colspan=1>E. coli</td><td rowspan=1 colspan=1>AR-Bank #0061</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>K. oxytoca</td><td rowspan=1 colspan=1>JMI 2661</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>KPC-4</td><td rowspan=1 colspan=1>K. pneumoniae</td><td rowspan=1 colspan=1>JMI 766</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>KPC-5</td><td rowspan=1 colspan=1>P. aeruginosa</td><td rowspan=1 colspan=1>Creighton University PS28</td><td rowspan=1 colspan=1>1.0E+04</td></tr></table>

Table 41: Isolates Containing the blaNDM gene Tested and Detected   

<table><tr><td rowspan=1 colspan=1>NDM Type</td><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Source</td><td rowspan=1 colspan=1>Test concentration(copies/mL)</td><td rowspan=1 colspan=1>Result</td></tr><tr><td rowspan=4 colspan=1>NDM</td><td rowspan=1 colspan=1>E. coli</td><td rowspan=1 colspan=1>AR-Bank #0162</td><td rowspan=1 colspan=1>1.0E+04</td><td rowspan=8 colspan=1>NDMDetected</td></tr><tr><td rowspan=1 colspan=1>K. pneumoniae</td><td rowspan=1 colspan=1>AR-Bank #0153</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>K. pneumoniae</td><td rowspan=1 colspan=1>AR-Bank #0068</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>P. mirabilis</td><td rowspan=1 colspan=1>AR-Bank #0159</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>NDM-1</td><td rowspan=1 colspan=1>A. baumannii</td><td rowspan=1 colspan=1>AR-Bank #0033</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>NDM-2</td><td rowspan=1 colspan=1>A. baumannii</td><td rowspan=1 colspan=1>GRE 1153064</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>NDM-5</td><td rowspan=1 colspan=1>E. coli</td><td rowspan=1 colspan=1>AR-Bank #0150</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>NDM-6</td><td rowspan=1 colspan=1>E. coli</td><td rowspan=1 colspan=1>AR-Bank #0137</td><td rowspan=1 colspan=1>1.0E+04</td></tr></table>

Table 42: Isolates Containing the blaOXA-48 and like genes Tested and Detected   

<table><tr><td rowspan=1 colspan=1>OXA Typea</td><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Source</td><td rowspan=1 colspan=1>Test concentration(copies/mL)</td><td rowspan=1 colspan=1>Result</td></tr><tr><td rowspan=1 colspan=1>OXA-48</td><td rowspan=1 colspan=1>E. aerogenes</td><td rowspan=1 colspan=1>AR-Bank #0074</td><td rowspan=1 colspan=1>1.0E+04</td><td rowspan=6 colspan=1>Gram negative&amp; OXA-48-LikeDetected</td></tr><tr><td rowspan=2 colspan=1>OXA-48-like</td><td rowspan=1 colspan=1>S. marcescens</td><td rowspan=1 colspan=1>GRE 1411136</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>S. marcescens</td><td rowspan=1 colspan=1>GRE 1411137</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>OXA-162</td><td rowspan=1 colspan=1>K. pneumoniae</td><td rowspan=1 colspan=1>GRE 1355030</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>OXA-181</td><td rowspan=1 colspan=1>K. pneumoniae</td><td rowspan=1 colspan=1>AR-Bank #0068</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>OXA-232</td><td rowspan=1 colspan=1>K. pneumoniae</td><td rowspan=1 colspan=1>AR-Bank #0075</td><td rowspan=1 colspan=1>1.0E+04</td></tr></table>

a No reactivity predicted with non-carbapenemase variant types -163, -247,-405, -436, -438, and -439 based on assay design and in silico evaluation.

Table 43: Isolates Containing the blaVIM gene Tested and Detected   

<table><tr><td rowspan=1 colspan=1>VIM Type</td><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Source</td><td rowspan=1 colspan=1>Testconcentration</td><td rowspan=1 colspan=1>Result</td></tr><tr><td rowspan=3 colspan=1>VIM</td><td rowspan=1 colspan=1>E. cloacae</td><td rowspan=1 colspan=1>AR-Bank #0154</td><td rowspan=1 colspan=1>1.0E+04</td><td rowspan=7 colspan=1>VIM Detected</td></tr><tr><td rowspan=1 colspan=1>P. aeruginosa</td><td rowspan=1 colspan=1>AR-Bank #0111</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>K. pneumoniae</td><td rowspan=1 colspan=1>AR-Bank 0076</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>VIM-2</td><td rowspan=1 colspan=1>P. aeruginosa</td><td rowspan=1 colspan=1>AR-Bank #0100</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>VIM-4</td><td rowspan=1 colspan=1>P. aeruginosa</td><td rowspan=1 colspan=1>AR-Bank #0054</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>VIM-7</td><td rowspan=1 colspan=1>E. coli</td><td rowspan=1 colspan=1>GRE 1256018</td><td rowspan=1 colspan=1>1.0E+04</td></tr><tr><td rowspan=1 colspan=1>VIM-10</td><td rowspan=1 colspan=1>P. aeruginosa</td><td rowspan=1 colspan=1>NCTC 13437</td><td rowspan=1 colspan=1>1.0E+04</td></tr></table>

# $f .$ Analytical Specificity (Cross-Reactivity and Exclusivity)

There is a risk of false positive results due to non-specific amplification and/or cross-reactivity with organisms found in the respiratory tract. The potential for non-specific amplification and detection by the FilmArray Pneumonia Panel assays was evaluated by in silico analyses of available sequences and by empirical (wet) testing of high concentrations of organisms in contrived samples. The observed and predicted cross-reactivities for organisms closely related to those detected by the panel and unrelated organisms that may be present in lower respiratory specimens are summarized in Table 44. Erroneous results due to cross-reactivity with organisms that were not evaluated or new variant sequences that emerge is also possible.

On-panel organisms were tested to assess the potential for intra-panel cross-reactivity (Table 45). Offpanel organisms included species of the same genus or otherwise genetically related to organisms detected by the panel, as well as normal flora and pathogens that may be present in sputum-like and BAL-like specimens (Table 46). Antimicrobial resistance genes were also evaluated in conjunction with on and off panel host organisms.

The final concentration of analyte in the sample (typically ${ \ge } 1 . 0 \mathrm { E } { + } 0 7$ CFU/mL for bacteria and fungi and ${ \ge } 1 . 0 \mathrm { E } { + } 0 5$ TCID50/mL for viruses) represented levels approximately 100 - 100,000-fold higher than the LoD or lowest reportable level of the FilmArray Pneumonia Panel assays.

Table 44: Observed and Predicted Cross-Reactivity of FilmArray Pneumonia Panel Assays   

<table><tr><td rowspan=1 colspan=2>FilmArray Pneumonia Panel Result</td><td rowspan=1 colspan=1>Cross-Reactive Organism</td></tr><tr><td rowspan=1 colspan=3>Closely-Related Species</td></tr><tr><td rowspan=2 colspan=2>Escherichia coli</td><td rowspan=1 colspan=1>Escherichia fergusoniia</td></tr><tr><td rowspan=1 colspan=1>Shigella species (S. boydii, S. dysenteriae, S. flexneri, S.sonnei)a</td></tr><tr><td rowspan=1 colspan=2>Klebsiella oxytoca</td><td rowspan=1 colspan=1>Klebsiella michiganensisa</td></tr><tr><td rowspan=2 colspan=2>Staphylococcus aureus</td><td rowspan=1 colspan=1>Staphylococcus argenteusb</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus schweitzeric</td></tr><tr><td rowspan=1 colspan=2>Pseudomonas aeruginosa</td><td rowspan=1 colspan=1>Pseudomonas putidad</td></tr><tr><td rowspan=1 colspan=3>Unrelated Species</td></tr><tr><td rowspan=1 colspan=2>Human Rhinovirus/Enterovirus</td><td rowspan=1 colspan=1>Bordetella speciese</td></tr><tr><td rowspan=3 colspan=2>Parainfluenza Virusf</td><td rowspan=1 colspan=1>Aspergillus niger</td></tr><tr><td rowspan=1 colspan=1>Cryptococcus laurentii</td></tr><tr><td rowspan=1 colspan=1>Cryptococcus uniguttulatus</td></tr><tr><td rowspan=1 colspan=2>Adenovirus</td><td rowspan=1 colspan=1>Stenotrophomonas acidaminiphilag</td></tr><tr><td rowspan=1 colspan=2>CTX-Mh</td><td rowspan=1 colspan=1>Acinetobacter schindleri</td></tr><tr><td rowspan=4 colspan=2>Escherichia colij</td><td rowspan=1 colspan=1>Lelliottia amnigena (Enterobacter amnigenus)</td></tr><tr><td rowspan=1 colspan=1>Enterobacter kobei</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Enterobacter ludwigii</td></tr><tr><td rowspan=1 colspan=1>Enterobacter cloacae</td></tr></table>

a. Genetically or phenotypically indistinguishable and often misidentified by standard laboratory techniques. Detected at concentrations ${ \ge } 1 . 0 \mathrm { E } ^ { + 0 4 }$ copies $/ \mathrm { m L }$ .

b. Genetically or phenotypically indistinguishable and often misidentified by standard laboratory techniques. Detected at concentrations ${ \ge } 1 . 0 \mathrm { E } { + } 0 5$ copies $/ \mathrm { m L }$ .   
c. Genetically or phenotypically indistinguishable and often misidentified by standard laboratory techniques. Detected at concentrations ${ \geq } 1 . 0 \mathrm { E } { + } 0 6 $ copies/mL.   
d. Cross-reactivity possible at concentrations ${ > } 1 . 0 \mathrm { E } { + } 0 7$ copies/mL.   
e. Cross-reactivity with $B$ . pertussis confirmed at ${ \geq } 1 . 0 \mathrm { E } { + } 0 6 $ CFU/mL. Cross-reactivity with B. parapertussis and B. bronchiseptica was not observed at $1 . 0 \mathrm { E } ^ { + 0 8 }$ CFU/mL, but possible based on sequence analysis.   
f. Cross-reactivity was observed with A. niger, C. laurentii and C. uniguttulatus at concentrations ${ > } 1 . 0 \mathrm { E } { + } 0 6 $ copies/mL. Cross-reactivity with other Cryptococcus species may be possible based on sequence analysis.   
g . S. acidaminiphila has not been isolated from human clinical specimens, no cross-reactivity observed with other Stenotrophomonas species.   
h. Cross-reactive product observed only at concentrations ${ > } 4 . 5 \mathrm { E } { + } 0 7$ CFU/mL and only reported if an applicable gram-negative bacterium is also detected.   
i. If observed, results will be reported as Escherichia coli $1 0 ^ { 4 }$ copies/mL.   
j. j Based on in silico analysis, cross-reactivity is also possible at high concentrations $( > 1 . 0 \mathrm { E } { + 0 7 }$ copies $/ \mathrm { m L }$ ) of other $e$ species $E$ . hormaechei, $E$ . aerogenes, $E$ . lingolyticus), Citrobacter freundii, Citrobacter koseri, Escherichia vulneris, and Leclercia adecaboxylata.

Table 45: On Panel Organisms Tested for Evaluation of FilmArray Pneumonia Panel Analytical Specificity (Organisms with the potential for non-specific amplification are shown in bold.)   
Table 46: Off-Panel Bacteria Tested for Evaluation of FilmArray Pneumonia Panel Analytical Specificity (Note: Organisms with the potential for non-specific amplification by a FilmArray   

<table><tr><td rowspan=1 colspan=6>ON-PANEL</td></tr><tr><td rowspan=1 colspan=6>Bacteria</td></tr><tr><td rowspan=1 colspan=1>Acinetobacter baumannii</td><td rowspan=1 colspan=2>Enterobacter hormaechei</td><td rowspan=1 colspan=2>Klebsiella quasipneumoniae</td><td rowspan=1 colspan=1>Pseudomonas aeruginosa</td></tr><tr><td rowspan=1 colspan=1>Acinetobacter calcoaceticus</td><td rowspan=1 colspan=2>Enterobacter kobei</td><td rowspan=1 colspan=2>Klebsiella variicola</td><td rowspan=1 colspan=1>Serratia marcescens</td></tr><tr><td rowspan=1 colspan=1>Acinetobacter nosocomialis</td><td rowspan=1 colspan=2>Enterobacter ludwigii</td><td rowspan=1 colspan=2>Moraxella catarrhalis</td><td rowspan=1 colspan=1>Staphylococcus aureus</td></tr><tr><td rowspan=1 colspan=1>Acinetobacter pittii</td><td rowspan=1 colspan=2>Escherichia coli</td><td rowspan=1 colspan=2>Proteus hauseri</td><td rowspan=1 colspan=1>Streptococcus agalactiae</td></tr><tr><td rowspan=1 colspan=1>Enterobacter aerogenes</td><td rowspan=1 colspan=2>Haemophilus influenzae</td><td rowspan=1 colspan=2>Proteus mirabilis</td><td rowspan=1 colspan=1>Streptococcus pneumoniae</td></tr><tr><td rowspan=1 colspan=1>Enterobacter asburiae</td><td rowspan=1 colspan=2>Klebsiella oxytoca</td><td rowspan=1 colspan=2>Proteus penneri</td><td rowspan=1 colspan=1>Streptococcus pyogenes</td></tr><tr><td rowspan=1 colspan=1>Enterobacter cloacae</td><td rowspan=1 colspan=2>Klebsiella pneumoniae</td><td rowspan=1 colspan=2>Proteus vulgaris</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=6>Atypical Bacteria</td></tr><tr><td rowspan=1 colspan=2>Chlamydia pneumoniae</td><td rowspan=1 colspan=2>Legionella pneumophila</td><td rowspan=1 colspan=2>Mycoplasma pneumoniae</td></tr><tr><td rowspan=1 colspan=6>Viruses</td></tr><tr><td rowspan=1 colspan=1>Adenovirus B</td><td rowspan=1 colspan=2>Coronavirus NL63</td><td rowspan=1 colspan=2>Human Metapneumovirus</td><td rowspan=1 colspan=1>Parainfluenza Virus 2</td></tr><tr><td rowspan=1 colspan=1>Adenovirus C</td><td rowspan=1 colspan=2>Coronavirus OC43</td><td rowspan=1 colspan=2>Influenza A</td><td rowspan=1 colspan=1>Parainfluenza Virus 3</td></tr><tr><td rowspan=1 colspan=1>Adenovirus E</td><td rowspan=1 colspan=2>Enterovirus</td><td rowspan=1 colspan=2>Influenza B</td><td rowspan=1 colspan=1>Parainfluenza Virus 4</td></tr><tr><td rowspan=1 colspan=1>Coronavirus 229E</td><td rowspan=1 colspan=2>Human Rhinovirus</td><td rowspan=1 colspan=2>Parainfluenza Virus 1</td><td rowspan=1 colspan=1>Respiratory Syncytial Virus</td></tr><tr><td rowspan=1 colspan=1>Coronavirus HKU1</td><td></td><td></td><td></td><td></td><td></td></tr><tr><td rowspan=1 colspan=6>Antimicrobial Resistance Genes</td></tr><tr><td rowspan=1 colspan=1>CTX-M(Klebsiella oxytoca)</td><td rowspan=1 colspan=2>KPC(Klebsiella pneumoniae)</td><td rowspan=1 colspan=2>OXA-48-like(Serratia marcescens)</td><td rowspan=1 colspan=1>mecA and MREJ(Staphylococcus aureus)</td></tr><tr><td rowspan=1 colspan=1>IMP(Pseudomonas aeruginosa) (Acinetobacter baumannii)</td><td rowspan=1 colspan=2>NDM(Pseudomonas aeruginosa) (Acinetobacter baumannii)</td><td rowspan=1 colspan=2>VIM(Enterobacter cloacae)</td><td rowspan=1 colspan=1></td></tr></table>

# Pneumonia Panel assay are shown in bold.)

<table><tr><td colspan="4" rowspan="1">OFF-PANEL</td></tr><tr><td colspan="4" rowspan="1">Bacteria</td></tr><tr><td colspan="1" rowspan="1">Abiotrophia defectiva</td><td colspan="1" rowspan="1">Escherichia fergusonii</td><td colspan="1" rowspan="1">Mycobacterium tuberculosis</td><td colspan="1" rowspan="1">Shigella boydii</td></tr><tr><td colspan="1" rowspan="1">Achromobacter xylosoxidans</td><td colspan="1" rowspan="1">Escherichia hermanii</td><td colspan="1" rowspan="1">Mycoplasma bovis</td><td colspan="1" rowspan="1">Shigella dysenteriae</td></tr><tr><td colspan="1" rowspan="1">Acinetobacter haemolyticus</td><td colspan="1" rowspan="1">Escherichia vulneris</td><td colspan="1" rowspan="1">Mycoplasma genitalium</td><td colspan="1" rowspan="1">Shigella flexneri</td></tr><tr><td colspan="1" rowspan="1">Acinetobacter johnsonii</td><td colspan="1" rowspan="1">Fluoribacter dumoffei</td><td colspan="1" rowspan="1">Mycoplasma hominis</td><td colspan="1" rowspan="1">Shigella sonnei</td></tr><tr><td colspan="1" rowspan="1">Acinetobacter junii</td><td colspan="1" rowspan="1">Fusobacterium varium</td><td colspan="1" rowspan="1">Mycoplasma orale</td><td colspan="1" rowspan="1">Staphylococcus argenteus</td></tr><tr><td colspan="1" rowspan="1">Acinetobacter lwolfii</td><td colspan="1" rowspan="1">Gemella morbillorum</td><td colspan="1" rowspan="1">Neisseria gonorrhoeae</td><td colspan="1" rowspan="1">Staphylococcus capitis</td></tr><tr><td colspan="1" rowspan="1">Acinetobacter radioresistens</td><td colspan="1" rowspan="1">Granulicatella adiacens</td><td colspan="1" rowspan="1">Neisseria lactamica</td><td colspan="1" rowspan="1">Staphylococcus caprae</td></tr><tr><td colspan="1" rowspan="1">Acinetobacter schindleri</td><td colspan="1" rowspan="1">Haemophilus ducreyi</td><td colspan="1" rowspan="1">Neisseria meningitidis</td><td colspan="1" rowspan="1">Staphylococcus cohnii</td></tr><tr><td colspan="1" rowspan="1">Acinetobacter ursingii</td><td colspan="1" rowspan="1">Haemophilus haemolyticus</td><td colspan="1" rowspan="1">Neisseria mucosa</td><td colspan="1" rowspan="1">Staphylococcushaemolyticus</td></tr><tr><td colspan="1" rowspan="1">Actinobacillusactinomycetemcomitans</td><td colspan="1" rowspan="1">Haemophilusparahaemolyticus</td><td colspan="1" rowspan="1">Neisseria sicca</td><td colspan="1" rowspan="1">Staphylococcusepidermidis(mecA)</td></tr><tr><td colspan="1" rowspan="1">Actinobacillus hominis</td><td colspan="1" rowspan="1">Haemophilusparainfluenzae</td><td colspan="1" rowspan="1">Nocardia asteroides</td><td colspan="1" rowspan="1">Staphylococcus hominis</td></tr><tr><td colspan="1" rowspan="1">Actinobacillus ureae</td><td colspan="1" rowspan="1">Haemophilus parasuis</td><td colspan="1" rowspan="1">Nocardia brasilensis</td><td colspan="1" rowspan="1">Staphylococcusintermedius</td></tr><tr><td colspan="1" rowspan="1">Actinomyces isrealii</td><td colspan="1" rowspan="1">Hafnia alvei</td><td colspan="1" rowspan="1">Pantoea agglomerans</td><td colspan="1" rowspan="1">Staphylococcuslugdunensis</td></tr><tr><td colspan="1" rowspan="1">Actinomyces naeslundii</td><td colspan="1" rowspan="1">Hafnia paralvei</td><td colspan="1" rowspan="1">Pasteurella multocida</td><td colspan="1" rowspan="1">Staphylococcus lutrae</td></tr><tr><td colspan="1" rowspan="1">Bacillus cereus</td><td colspan="1" rowspan="1">Helicobacter pylori</td><td colspan="1" rowspan="1">Pediococcus acidilactici</td><td colspan="1" rowspan="1">Staphylococcus pasteuri</td></tr><tr><td colspan="1" rowspan="1">Bacteriodes fragilis</td><td colspan="1" rowspan="1">Kingella kingae</td><td colspan="1" rowspan="1">Peptostreptococcusanaerobius</td><td colspan="1" rowspan="1">Staphylococcuspseudointermedius</td></tr><tr><td colspan="1" rowspan="1">Bordatella bronchiseptica</td><td colspan="1" rowspan="1">Klebsiella michiganensis</td><td colspan="1" rowspan="1">Pluralibacter gergoviae</td><td colspan="1" rowspan="1">Staphylococcussaprophyticus</td></tr><tr><td colspan="1" rowspan="1">Bordatella parapertussis</td><td colspan="1" rowspan="1">Kluyvera intermedia</td><td colspan="1" rowspan="1">Porphyromonas gingivalis</td><td colspan="1" rowspan="1">Staphylococcus schleiferi</td></tr><tr><td colspan="1" rowspan="1">Bordatella pertussis</td><td colspan="1" rowspan="1">Kluyvera ascorbata</td><td colspan="1" rowspan="1">Prevotella intermedia</td><td colspan="1" rowspan="1">Staphylococcusschweitzeri</td></tr><tr><td colspan="1" rowspan="1">Burkholderia cepacia</td><td colspan="1" rowspan="1">Lactobacillus acidophilus</td><td colspan="1" rowspan="1">Prevotella melaninogenica</td><td colspan="1" rowspan="1">Staphylococcus sciuri</td></tr><tr><td colspan="1" rowspan="1">Burkholderia mallei</td><td colspan="1" rowspan="1">Leclercia adecarboxylata</td><td colspan="1" rowspan="1">Prevotella oralis</td><td colspan="1" rowspan="1">Staphylococcus warneri</td></tr><tr><td colspan="1" rowspan="1">Burkholderia multivorans</td><td colspan="1" rowspan="1">Legionella bozemanii</td><td colspan="1" rowspan="1">Propionibacterium acnes</td><td colspan="1" rowspan="1">Staphylococcus xylosus</td></tr><tr><td colspan="1" rowspan="1">Burkholderia pseudomallei</td><td colspan="1" rowspan="1">Legionella cincinnatiensis</td><td colspan="1" rowspan="1">Providencia rettgeri(OXA-48-like)</td><td colspan="1" rowspan="1">Stenotrophomonasacidaminiphila</td></tr><tr><td colspan="1" rowspan="1">Cardiobacterium hominis</td><td colspan="1" rowspan="1">Legionella feeleii</td><td colspan="1" rowspan="1">Providencia stuartii</td><td colspan="1" rowspan="1">Stenotrophomonasmaltophilia</td></tr><tr><td colspan="1" rowspan="1">Cedecea davisae</td><td colspan="1" rowspan="1">Legionella lansingensis</td><td colspan="1" rowspan="1">Pseudomonas fluorescens</td><td colspan="1" rowspan="1">Stenotrophomonasnitritireducens</td></tr><tr><td colspan="1" rowspan="1">Chlamydia trachomatis</td><td colspan="1" rowspan="1">Legionella longbeachae</td><td colspan="1" rowspan="1">Pseudomonas luteola</td><td colspan="1" rowspan="1">Stenotrophomonasrhizophila</td></tr><tr><td colspan="1" rowspan="1">Chlamydophila psittaci</td><td colspan="1" rowspan="1">Legionella micdadei</td><td colspan="1" rowspan="1">Pseudomonas nitroreducens</td><td colspan="1" rowspan="1">Streptococcus equi</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">subsp. zooepidemicus</td></tr><tr><td colspan="1" rowspan="1">Citrobacter freundii (KPC)</td><td colspan="1" rowspan="1">Legionella wadsworthii</td><td colspan="1" rowspan="1">Pseudomonas oryzihabitans</td><td colspan="1" rowspan="1">Streptococcus mitis</td></tr><tr><td colspan="1" rowspan="1">Citrobacter koseri(OXA-48-like)</td><td colspan="1" rowspan="1">Lelliottia nimipressuralis</td><td colspan="1" rowspan="1">Pseudomonas pertucinogena</td><td colspan="1" rowspan="1">Streptococcus mutans</td></tr><tr><td colspan="1" rowspan="1">Citrobacter sedlakii</td><td colspan="1" rowspan="1">Lelliottia amnigena(Enterobacter amnigenus)</td><td colspan="1" rowspan="1">Pseudomonas putida (IMP)</td><td colspan="1" rowspan="1">Streptococcus oralis</td></tr><tr><td colspan="1" rowspan="1">Citrobacter werkmanii(VIM)</td><td colspan="1" rowspan="1">Leuconostoc lactis</td><td colspan="1" rowspan="1">Pseudomonas stutzeri</td><td colspan="1" rowspan="1">Streptococcusparasanguinis</td></tr><tr><td colspan="1" rowspan="1">Clostridium difficile</td><td colspan="1" rowspan="1">Listeria monocytogenes</td><td colspan="1" rowspan="1">Ralstonia pickettii</td><td colspan="1" rowspan="1">Streptococcuspseudopneumoniae</td></tr><tr><td colspan="1" rowspan="1">Clostridium perfringens</td><td colspan="1" rowspan="1">Macrococcus caseolyticus</td><td colspan="1" rowspan="1">Raoultella ornithinolytica</td><td colspan="1" rowspan="1">Streptococcus salivarius</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium diptheriae</td><td colspan="1" rowspan="1">Micrococcus luteus</td><td colspan="1" rowspan="1">Raoultella planticola</td><td colspan="1" rowspan="1">Streptococcus sanguinis</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium genitalium</td><td colspan="1" rowspan="1">Moraxella equi</td><td colspan="1" rowspan="1">Raoultella terrigena</td><td colspan="1" rowspan="1">Streptococcus tigurinus</td></tr><tr><td colspan="1" rowspan="1">Corynebacteriumpseudodiptherticum</td><td colspan="1" rowspan="1">Moraxella lacunata</td><td colspan="1" rowspan="1">Rhodococcus equi</td><td colspan="1" rowspan="1">Streptomyces anulatus</td></tr><tr><td colspan="1" rowspan="1">Corynebacteriumurealyticum</td><td colspan="1" rowspan="1">Moraxella lincolnii</td><td colspan="1" rowspan="1">Rothia mucilaginosa</td><td colspan="1" rowspan="1">Treponema denticola</td></tr><tr><td colspan="1" rowspan="1">Cronobacter sakazakii</td><td colspan="1" rowspan="1">Moraxellanonliquiefaciens</td><td colspan="1" rowspan="1">Salmonella enterica(CTX-M)</td><td colspan="1" rowspan="1">Ureaplasma parvum</td></tr><tr><td colspan="1" rowspan="1">Eikenella corrodens</td><td colspan="1" rowspan="1">Morganella morganii(NDM)</td><td colspan="1" rowspan="1">Serratia fonticola</td><td colspan="1" rowspan="1">Ureaplasma urealyticum</td></tr><tr><td colspan="1" rowspan="1">Enterobacter cancerogenus</td><td colspan="1" rowspan="1">Mycobacteriumafricanum</td><td colspan="1" rowspan="1">Serratia liquefaciens</td><td colspan="1" rowspan="1">Vagococcus fluvialis</td></tr><tr><td colspan="1" rowspan="1">Enterobacter massiliensis</td><td colspan="1" rowspan="1">Mycobacterium bovis</td><td colspan="1" rowspan="1">Serratia odorifera</td><td colspan="1" rowspan="1">Veillonella parvula</td></tr><tr><td colspan="1" rowspan="1">Enterobacter soli</td><td colspan="1" rowspan="1">Mycobacterium caprae</td><td colspan="1" rowspan="1">Serratia plymuthica</td><td colspan="1" rowspan="1">Yersinia enterocolitica</td></tr><tr><td colspan="1" rowspan="1">Enterococcus faecium</td><td colspan="1" rowspan="1">Mycobacterium microti</td><td colspan="1" rowspan="1">Serratia rubidaea</td><td colspan="1" rowspan="1">Yersiniapseudotuberculosis</td></tr><tr><td colspan="1" rowspan="1">Enterococcus faecalis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="3" rowspan="1">Viruses</td></tr><tr><td colspan="1" rowspan="1">Bocavirus</td><td colspan="1" rowspan="1">Hantavirus</td><td colspan="1" rowspan="1">Human Papillomavirus(HPV)</td><td colspan="1" rowspan="1">Mumps virus</td></tr><tr><td colspan="1" rowspan="1">Cytomegalovirus</td><td colspan="1" rowspan="1">Herpes simplex virus 1</td><td colspan="1" rowspan="1">Influenza C</td><td colspan="1" rowspan="1">Varicella zoster virus</td></tr><tr><td colspan="1" rowspan="1">Epstein Barr virus</td><td colspan="1" rowspan="1">Human ImmunodeficiencyVirus (HIV)</td><td colspan="1" rowspan="1">Middle East RespiratorySyndrome Coronavirus(MERS- CoV)</td><td colspan="1" rowspan="1">Severe Acute RespiratorySyndrome Coronavirus(SARS-CoV)</td></tr><tr><td colspan="1" rowspan="1">German Measles Virus(Rubella)</td><td colspan="1" rowspan="1">Measles Virus (Rubeola)</td><td colspan="1" rowspan="1">Fusarium kyushense</td><td colspan="1" rowspan="1">Pneumocystis carinii</td></tr><tr><td colspan="1" rowspan="1">Aspergillus flavus</td><td colspan="1" rowspan="1">Coccidioides posadasii</td><td colspan="1" rowspan="1">Histoplasma capsulatum</td><td colspan="1" rowspan="1">Pneumocystis jirovecii</td></tr><tr><td colspan="1" rowspan="1">Aspergillus fumigatus</td><td colspan="1" rowspan="1">Cryptococcus albidus</td><td colspan="1" rowspan="1">Paecilomyces variottii</td><td colspan="1" rowspan="1">Pneumocystis murina</td></tr><tr><td colspan="1" rowspan="1">Aspergillus niger</td><td colspan="1" rowspan="1">Cryptococcus gattii</td><td colspan="1" rowspan="1">Paracoccidodes brasiliensis</td><td colspan="1" rowspan="1">Rhizopus microsporus</td></tr><tr><td colspan="1" rowspan="1">Aspergillus terreus</td><td colspan="1" rowspan="1">Cryptococcus laurentii</td><td colspan="1" rowspan="1">Penicillium chrysogenum</td><td colspan="1" rowspan="1">Scedosporium</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">apiospermum</td></tr><tr><td colspan="1" rowspan="1">Blastomyces dermatitidis</td><td colspan="1" rowspan="1">Cryptococcus neoformans</td><td colspan="1" rowspan="1">Penicillium marneffei</td><td colspan="1" rowspan="1">Scedosporium prolificans</td></tr><tr><td colspan="1" rowspan="1">Candida albicans</td><td colspan="1" rowspan="1">Cryptococcusuniguttalatus</td><td colspan="1" rowspan="1">Candida glabrata</td><td colspan="1" rowspan="1">Filobasidiumcapsuligenum</td></tr><tr><td colspan="4" rowspan="1">Antimicrobial Resistance Genes</td></tr><tr><td colspan="1" rowspan="1">AmpC(Enterobacter aerogenes)</td><td colspan="1" rowspan="1">OXA-24/40 (non-48-like)(Acinetobacter baumannii)</td><td colspan="1" rowspan="1">SME(Serratia marcescens)</td><td colspan="1" rowspan="1">TEM(Escherichia coli)</td></tr><tr><td colspan="1" rowspan="1">CMY (II)(Escherichia coli)</td><td colspan="1" rowspan="1">SHV (Klebsiellapneumoniae)</td><td colspan="1" rowspan="1">SPM(Pseudomonas aeruginosa)</td><td colspan="1" rowspan="1">VAN(Staphylococcus aureus)</td></tr><tr><td colspan="1" rowspan="1">ompK36 [SHV-12, OMPC]a(Klebsiella pneumoniae)</td><td colspan="3" rowspan="1"></td></tr></table>

# g. Assay Cut-off:

The FilmArray Pneumonia Panel is part of BioFire Diagnostics’ (BFDX) FilmArray system. The FilmArray system is designed to interpret the test data and automatically report the test results to the operator. The FilmArray system uses the results of the Melt Detector to determine each test result. The Melt Detector is part of the FilmArray Analysis Software and assigns a positive or negative result to each reaction on the array through analysis of the melt data collected during the test. These positive and negative results are combined in the FilmArray Analysis Software (using the replicate, assay and interpretation rules) to report the presence or absence of each pathogen in the panel.

The purpose of this study was to validate the use of this Melt Detector with current optimization parameters with the FilmArray Pneumonia Panel. To evaluate the Melt Detector performance, the observed sensitivity and specificity rates for the individual melt curves and assay calls are reported. These sensitivity and specificity rates are determined by comparing the FilmArray test results obtained from well-characterized samples, collected as part of the clinical evaluation and analytic testing of the Pneumonia Panel, to expert annotation. Annotations (positive and negative calls) for all melt curves and assay calls were determined by the sponsor.

For individual melt curves, the observed sensitivity and specificity, as compared to expert annotation, of the Melt Detector is $9 9 . 6 9 \%$ and $9 9 . 9 3 \%$ , respectively. For the Analysis Software, the observed sensitivity and specificity, as compared to the expert annotation, of the assay calls are greater than $9 9 . 8 2 \%$ for sensitivity and $9 9 . 9 9 \%$ specificity. The validation results met the predefined acceptance criteria of $59 5 \%$ accuracy as compared to expert annotation.

h. Interfering substances:

Potentially interfering substances that could be present in BAL-like or sputum-like specimens or that may be introduced during specimen collection and testing were evaluated for their effect on FilmArray Pneumonia Panel performance. Substances included endogenous substances that may be found in specimens at normal or elevated levels (e.g. blood, mucus/mucin, human genomic DNA), various commensal or infectious microorganisms, medications, a variety of sample processing substances and substances used to clean, decontaminate, or disinfect work areas. The performance of the FilmArray Pneumonia Panel has not been established with all potentially interfering medications for the treatment of lower respiratory tract infections. The effect of interfering substances has only been evaluated for those listed in Table 47. Interference from substances that were not evaluated could lead to erroneous results.

Each substance was added to contrived samples containing representative qualitatively reported organisms and representative organisms with bin reporting. Qualitatively reported organisms were at concentrations near $( 2 - 3 \times )$ LoD and those with bin reporting were present at $1 0 ^ { 4 }$ copies/mL (e.g. in the lowest reported bin). The concentration of substance added to the samples (Table 47) was equal to or greater than the highest level expected to be in BAL-like or Sputum-like specimens.

Four of the evaluated substances were found to interfere with the ability of the FilmArray Pneumonia Panel to report accurate analyte results; Bleach, MycoPrep, $2 \% \mathrm { N a O H }$ , and $5 \%$ Oxalic acid. Each of these substances contains chemicals known to react with nucleic acids, altering their chemical structure. The interference observed was related to the inability to detect the chemically modified nucleic acids.

Treatment of specimens with these substances prior to FilmArray Pneumonia Panel testing may result in loss of analyte detection, therefore samples that have been in contact with these substances should not be tested using the FilmArray Pneumonia Panel. None of the tested substances were shown to interfere with the FilmArray Pneumonia function.

Table 47: Evaluation of Potentially Interfering Substances on the FilmArray Pneumonia Panel   

<table><tr><td colspan="1" rowspan="1">Substance</td><td colspan="1" rowspan="1">Concentration Tested</td><td colspan="1" rowspan="1">Testing Outcome</td></tr><tr><td colspan="3" rowspan="1">Endogenous Substances</td></tr><tr><td colspan="1" rowspan="1">Blood</td><td colspan="1" rowspan="1">10% v/v</td><td colspan="1" rowspan="1">No Interference</td></tr><tr><td colspan="1" rowspan="1">Albumin</td><td colspan="1" rowspan="1">60 mg/mL</td><td colspan="1" rowspan="1">No Interference</td></tr><tr><td colspan="1" rowspan="1">HC1 (gastric acid)</td><td colspan="1" rowspan="1">5 mmol/L</td><td colspan="1" rowspan="1">No Interference</td></tr><tr><td colspan="1" rowspan="1">Hemoglobin</td><td colspan="1" rowspan="1">2 mg/mL</td><td colspan="1" rowspan="1">No Interference</td></tr><tr><td colspan="1" rowspan="1">Human Cells (K-562 cell line)</td><td colspan="1" rowspan="1">3.8E+06 cells/mL</td><td colspan="1" rowspan="1">No Interference</td></tr><tr><td colspan="1" rowspan="1">Immunoglobulins (IgG)</td><td colspan="1" rowspan="1">60 mg/mL</td><td colspan="1" rowspan="1">No Interference</td></tr><tr><td colspan="1" rowspan="1">Mucin</td><td colspan="1" rowspan="1">16 mg/mL</td><td colspan="1" rowspan="1">No Interference</td></tr><tr><td colspan="3" rowspan="1">Exogenous Substances</td></tr><tr><td colspan="1" rowspan="1">Albuterol (bronchodilator)</td><td colspan="1" rowspan="1">1.7 μmol/L</td><td colspan="1" rowspan="1">No Interference</td></tr><tr><td colspan="1" rowspan="1">Benzocaine (Orajel)</td><td colspan="1" rowspan="1">1.0 % w/v</td><td colspan="1" rowspan="1">No Interference</td></tr><tr><td colspan="1" rowspan="1">Epinephrine (hormone,</td><td colspan="1" rowspan="1">8.3 μg/mL</td><td colspan="1" rowspan="1">No Interference</td></tr><tr><td colspan="1" rowspan="1">bronchodilator)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Y</td></tr><tr><td colspan="1" rowspan="1">Galphimia glauca(Homeopathic remedy)</td><td colspan="1" rowspan="1">1.0 % w/v</td><td colspan="1" rowspan="1">No Interference</td></tr><tr><td colspan="1" rowspan="1">Guaifenesin (expectorant)</td><td colspan="1" rowspan="1">15.2 mmol/L</td><td colspan="1" rowspan="1">No Interference</td></tr><tr><td colspan="1" rowspan="1">Lidocaine</td><td colspan="1" rowspan="1">5.1 mmol/L</td><td colspan="1" rowspan="1">No Interference</td></tr><tr><td colspan="1" rowspan="1">Menthol and cetylpyridiniumchloride (Cepacol Mouthwash)</td><td colspan="1" rowspan="1">1.0% v/v</td><td colspan="1" rowspan="1">No Interference</td></tr><tr><td colspan="1" rowspan="1">Mupirocin (antibiotic)</td><td colspan="1" rowspan="1">6.0 ng/mL</td><td colspan="1" rowspan="1">No Interference</td></tr><tr><td colspan="1" rowspan="1">Nicotine</td><td colspan="1" rowspan="1">6.2 μmol/L</td><td colspan="1" rowspan="1">No Interference</td></tr><tr><td colspan="1" rowspan="1">Pentamidine (antimicrobial)</td><td colspan="1" rowspan="1">1.5 mg/mL</td><td colspan="1" rowspan="1">No Interference</td></tr><tr><td colspan="1" rowspan="1">Phenylephrine hydrochloride(decongestant)</td><td colspan="1" rowspan="1">0.3 mg/mL</td><td colspan="1" rowspan="1">No Interference</td></tr><tr><td colspan="1" rowspan="1">Tobramycin sulfate (antibiotic)</td><td colspan="1" rowspan="1">30 mg/mL</td><td colspan="1" rowspan="1">No Interference</td></tr><tr><td colspan="1" rowspan="1">Zanamivir (influenza antiviral)</td><td colspan="1" rowspan="1">426 ng/mL</td><td colspan="1" rowspan="1">No Interference</td></tr><tr><td colspan="3" rowspan="1">Competitive Microorganisms</td></tr><tr><td colspan="1" rowspan="1">Actinobacillus actinomycetecomitans</td><td colspan="1" rowspan="1">3.8E+07 CFU/mL</td><td colspan="1" rowspan="1">No Interference</td></tr><tr><td colspan="1" rowspan="1">Aspergillus fumigatus</td><td colspan="1" rowspan="1">5.5E+07 CFU/mL</td><td colspan="1" rowspan="1">No Interference</td></tr><tr><td colspan="1" rowspan="1">Burkholderia cepacia</td><td colspan="1" rowspan="1">1.7E+07 CFU/mL</td><td colspan="1" rowspan="1">No Interference</td></tr><tr><td colspan="1" rowspan="1">Cryptococcus neoformans</td><td colspan="1" rowspan="1">2.5E+05 CFU/mL</td><td colspan="1" rowspan="1">No Interference</td></tr><tr><td colspan="1" rowspan="1">Enterovirus D68</td><td colspan="1" rowspan="1">1.4E+06 copies/mL</td><td colspan="1" rowspan="1">No Interference</td></tr><tr><td colspan="1" rowspan="1">Haemophilus influenzae</td><td colspan="1" rowspan="1">1.8E+07 CFU/mL</td><td colspan="1" rowspan="1">No Interference</td></tr><tr><td colspan="1" rowspan="1">Legionella pneumophila</td><td colspan="1" rowspan="1">8.1E+06 CFU/mL</td><td colspan="1" rowspan="1">No Interference</td></tr><tr><td colspan="1" rowspan="1">Respiratory Syncytial Virus</td><td colspan="1" rowspan="1">3.5E+04 copies/mL</td><td colspan="1" rowspan="1">No Interference</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus epidermidis</td><td colspan="1" rowspan="1">1.9E+07 CFU/mL</td><td colspan="1" rowspan="1">No Interference</td></tr><tr><td colspan="1" rowspan="1">Streptococcus mutans</td><td colspan="1" rowspan="1">5.9E+06 CFU/mL</td><td colspan="1" rowspan="1">No Interference</td></tr><tr><td colspan="1" rowspan="1">Streptococcus pyogenes</td><td colspan="1" rowspan="1">5.5E+06 CFU/mL</td><td colspan="1" rowspan="1">No Interference</td></tr><tr><td colspan="1" rowspan="1">Varicella Zoster Virus</td><td colspan="1" rowspan="1">8.7E+07copies/mL</td><td colspan="1" rowspan="1">No Interference</td></tr><tr><td colspan="3" rowspan="1">Disinfection/Cleaning Substances</td></tr><tr><td colspan="1" rowspan="1">Reagent Alcohol</td><td colspan="1" rowspan="1">7.0%</td><td colspan="1" rowspan="1">No Interference</td></tr><tr><td colspan="1" rowspan="1">Bleach</td><td colspan="1" rowspan="1">1.0% v/v(600 ppm chlorine)</td><td colspan="1" rowspan="1">Interference Observeda</td></tr><tr><td colspan="3" rowspan="1">Sample Processing Materials</td></tr><tr><td colspan="1" rowspan="1">Copan Snotbuster(active ingredient DTT)</td><td colspan="1" rowspan="1">50.0% v/v</td><td colspan="1" rowspan="1">No Interference</td></tr><tr><td colspan="1" rowspan="1">Sputolysin (active ingredient DTT)</td><td colspan="1" rowspan="1">50.0% v/v</td><td colspan="1" rowspan="1">No Interference</td></tr><tr><td colspan="1" rowspan="1">SPUTASOL(active ingredient DTT + salts)</td><td colspan="1" rowspan="1">50.0% v/v</td><td colspan="1" rowspan="1">No Interference</td></tr><tr><td colspan="1" rowspan="1">MycoPrep (active ingredient NaOH+ NALC)</td><td colspan="1" rowspan="1">50.0% v/v</td><td colspan="1" rowspan="1">Interference Observeda</td></tr><tr><td colspan="1" rowspan="1">NaOH (decontaminant)</td><td colspan="1" rowspan="1">1.0%</td><td colspan="1" rowspan="1">Interference Observeda</td></tr><tr><td colspan="1" rowspan="1">Oxalic Acid (decontaminant)</td><td colspan="1" rowspan="1">2.5%</td><td colspan="1" rowspan="1">Interference Observeda</td></tr></table>

a. Pouch controls passed but Not Detected results were reported for one or more analytes after incubation of the sample with substance. Substance(s) are known to chemically interact with and damage nucleic acids (DNA and/or RNA) to prevent amplification.

# i. Competitive Interference:

To evaluate the potential for competitive inhibition between targeted microorganisms, various combinations of microorganisms were evaluated in contrived samples. The effect of several on-panel or off-panel microorganisms (fungi, viruses, and bacteria) on detection of analytes in a qualitative and quantitative sample mixtures were evaluated, with contrived samples serving as the control for analyte detection and a negative analyte sample serving as the control for whether the spiked organism/substance is or is not detected by the panel. Most microorganisms were evaluated at high concentrations, similar to the levels evaluated for potential cross-reactivity in the exclusivity study.

The following analytes were selected for evaluation for competitive inhibition based on the prevalence of each microorganism in tracheal aspirate specimens. Two sample pools were prepared with multiple targeted microorganisms at either low or high concentrations in each sample as shown in Table 48.

Table 48: Competitive Interference Microorganisms Tested   

<table><tr><td rowspan=1 colspan=2>Off-Panel</td></tr><tr><td rowspan=1 colspan=1>Cryptococcus neoformans</td><td rowspan=1 colspan=1>Streptococcus pyogenes</td></tr><tr><td rowspan=1 colspan=1>Respiratory Syncytial Virus</td><td rowspan=1 colspan=1>Streptococcus mutans</td></tr><tr><td rowspan=1 colspan=1>Varicella Zoster Virus</td><td rowspan=1 colspan=1>Staphylococcus epidermidis</td></tr><tr><td rowspan=1 colspan=1>Aspergillus fumigatus</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>On-Panel</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Haemophilus influenzae</td><td rowspan=1 colspan=1>Streptococcus pneumoniae</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus aureus</td><td rowspan=1 colspan=1>Pseudomonas aeruginosa</td></tr><tr><td rowspan=1 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>Influenza A</td></tr><tr><td rowspan=1 colspan=1>Mycoplasma pneumoniae</td><td rowspan=1 colspan=1></td></tr></table>

Results for samples spiked with high levels of on-panel and off-panel microorganisms (Cryptococcus neoformans, Streptococcus pyogenes, Respiratory Syncytial Virus, Actinobacillus actinomycetemcomitans, Streptococcus mutans, Varicella Zoster Virus, Staphylococcus epidermidis and Aspergillus fumigatus) showed no effect on pouch controls and gave the expected Detected and Not Detected results.

An unexpected Human Rhinovirus/Enterovirus Detected result was observed in replicate 1 of the negative sample spiked with a Haemophilus influenzae, but not in any of the other 8 replicates containing these high levels of this microorganism. In total, expected results were obtained in 8/9 replicates and there was no observable amplification for the Human Rhinovirus/Enterovirus assay in the other replicates, indicating no interference from or cross reactivity with this bacterium. Additional testing of two more negative sample replicates indicating the Human Rhinovirus/Enterovirus Detected result was not due to the presence of Haemophilus influenzae in the sample.

For samples spiked with Enterovirus, an unexpected Adenovirus Not Detected result was observed in replicate 2 of the qualitative sample mixture. Two additional replicates of this sample were tested and the expected Detected results were reported. In total, Adenovirus was detected in 4/5 replicates of this sample. Analyte Cp values and/or unbinned values for all assays were comparable between the control samples and those containing Enterovirus. These data indicate that the RNA virus Enterovirus does not interfere with Pneumonia Panel detection of Adenovirus or other analytes.

For samples spiked with Legionella pneumophila, unexpected Legionella pneumophila Not Detected and Respiratory Syncytial Virus Detected results were reported in replicate 3 of the qualitative sample mixture. An investigation into the cause of the unexpected results determined that the sample was mislabeled and contained Respiratory Syncytial Virus. Two additional replicates of the appropriate sample were tested and the expected results were reported.

For samples spiked with Burkholderia cepacia a control failure was observed in replicate 3 of the negative sample. Two additional replicates of this sample were retested and the control failure was not reproducible. In total, control assays passed on 10/11 tests containing this substance and the expected Detected and Not Detected results were reported for all contrived samples.

No reproducible effect on control or analyte assays due to competitive interference or cross-reactivity related to high levels of microorganisms was identified.

# $j$ . Carry-over:

A formal carry-over study in support of this regulatory submission for the FilmArray Pneumonia Panel was not performed, since carry-over studies with high positive samples followed by negative samples have been performed for other FDA-cleared FilmArray Panels (i.e., FilmArray RP, BCID, and GI) for both the FilmArray 2.0 and the FilmArray Torch systems, and no carry-over has been observed.

2. Comparison studies:

# a. Method comparison

N/A

# b. Matrix comparison:

Due to the challenge of obtaining large volumes of natural negative BAL matrix, artificial bronchoalveolar lavage (aBAL) was used to prepare samples for analytical studies. To validate that the FilmArray Pneumonia Panel results for samples in the artificial matrix are equivalent to those in the natural specimen type, contrived samples containing multiple analytes were prepared in BAL and aBAL, followed by serial dilution in 10-fold increments to span a range of at least four concentrations. The samples containing bacterial analytes were prepared by spiking each analyte into both aBAL and BAL sample matrices at a concentration intended to yield a bin result of at least $1 0 ^ { 4 }$ or $1 0 ^ { 5 }$ copies/mL, and the sample was then serially diluted in 10-fold increments to extend below the reportable range of the assay.

Detection agreement over this titration was evaluated to assess the impact of the matrices on detection at different concentrations, particularly near the detection or reporting limit of the assays. Determination of equivalency between the matrices also included an evaluation of the impact of matrix on amplification crossing point $\textstyle ( \mathrm { C p } )$ values, unbinned values for bacterial assays, and melting temperature (Tm) values for the amplified products.

A separate matrix equivalency study was performed to compare assay performance for samples prepared in artificial respiratory matrix (ARM) and samples prepared in pooled natural tracheal aspirate matrix. Tracheal aspirate test samples were prepared using pools of 4-6 individual tracheal aspirates determined to be negative for all LRT panel analytes. Both ARM and natural aspirate matrices were spiked with microorganism concentrations near the LoD for nine representative LRT analytes (between $\mathrm { 1 \bar { 0 } ^ { 4 } - 1 0 ^ { 6 } C F U / m L }$ ). Matrix equivalence was evaluated based on qualitative analysis (percent positivity) and quantitative analyses (comparison of mean assay signal).

Results demonstrate that FilmArray Pneumonia Panel amplification, detection, melting curve analysis, and reporting of viruses, bacteria, and the antimicrobial resistance genes that they carry is equivalent whether testing is performed in contrived aBAL or BAL samples. These results validate the use of aBAL as an appropriate sample matrix for use in performance evaluations of the panel.

# 3. Clinical Studies:

# Prospective Study:

The clinical performance of the FilmArray Pneumonia Panel was established during a multi-center study conducted at eight geographically distinct U.S. study sites from October 2016 to July 2017. A total of 904 residual BAL (821 BAL and 83 mini-BAL) and 925 residual sputum (478 sputum and 447 ETA) specimens were acquired for the prospective clinical study. FilmArray Pneumonia Panel performance in BAL and mini-BAL was similar, as was performance in sputum and ETA; therefore, these sample types are not stratified further in performance tables. A total of 58 BAL and 89 sputum specimens were excluded from the final data analysis. The most common reasons for specimen exclusion for both specimen types was reference culture unable to be performed, the specimen was found to not meet the inclusion criteria after the specimen had been enrolled, or the study site was unable to complete the Case Report Form (CRF). The final data set consisted of 846 BAL and 836 sputum specimens. Table 49 and Table 50 provide a summary of demographic information for the specimens included in the prospective study.

Table 49: Overall and Per Site Demographic Analysis for BAL Specimens   

<table><tr><td rowspan=1 colspan=11>BAL</td></tr><tr><td rowspan=1 colspan=2>cYanmaGYellowb</td><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>Site 1a</td><td rowspan=1 colspan=1>Site 2</td><td rowspan=1 colspan=1>Site 3</td><td rowspan=1 colspan=1>Site 4</td><td rowspan=1 colspan=1>Site 5</td><td rowspan=1 colspan=1>Site 6</td><td rowspan=1 colspan=1>Site 7</td><td rowspan=1 colspan=1>Site 8</td></tr><tr><td rowspan=4 colspan=1>S</td><td rowspan=2 colspan=1>Male</td><td rowspan=2 colspan=1>480(57%)</td><td rowspan=1 colspan=1>80</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>138</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>75</td><td rowspan=1 colspan=1>82</td><td rowspan=1 colspan=1>27</td><td rowspan=1 colspan=1>50</td></tr><tr><td rowspan=1 colspan=1>(59%)</td><td rowspan=1 colspan=1>(54%)</td><td rowspan=1 colspan=1>(55%)</td><td rowspan=1 colspan=1>(68%)</td><td rowspan=1 colspan=1>(61%)</td><td rowspan=1 colspan=1>(52%)</td><td rowspan=1 colspan=1>(61%)</td><td rowspan=1 colspan=1>(55%)</td></tr><tr><td rowspan=2 colspan=1>Female</td><td rowspan=2 colspan=1>366(43%)</td><td rowspan=1 colspan=1>55</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>113</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>48</td><td rowspan=1 colspan=1>76</td><td rowspan=1 colspan=1>17</td><td rowspan=2 colspan=1>41(45%)</td></tr><tr><td rowspan=1 colspan=1>(41%)</td><td rowspan=1 colspan=1>(46%)</td><td rowspan=1 colspan=1>(45%)</td><td rowspan=1 colspan=1>(32%)</td><td rowspan=1 colspan=1>(39%)</td><td rowspan=1 colspan=1>(48%)</td><td rowspan=1 colspan=1>(39%)</td></tr><tr><td rowspan=6 colspan=1>AE</td><td rowspan=1 colspan=1>≤ 5 years</td><td rowspan=1 colspan=1>23(3%)</td><td rowspan=1 colspan=1>0(0%)</td><td rowspan=1 colspan=1>5(38%)</td><td rowspan=1 colspan=1>0(0%)</td><td rowspan=1 colspan=1>15(48%)</td><td rowspan=1 colspan=1>0(0%)</td><td rowspan=1 colspan=1>3(2%)</td><td rowspan=1 colspan=1>0(0%)</td><td rowspan=1 colspan=1>0(0%)</td></tr><tr><td rowspan=1 colspan=1>6 - 17 years</td><td rowspan=1 colspan=1>27(3%)</td><td rowspan=1 colspan=1>0(0%)</td><td rowspan=1 colspan=1>8(62%)</td><td rowspan=1 colspan=1>0(0%)</td><td rowspan=1 colspan=1>13(42%)</td><td rowspan=1 colspan=1>0(0%)</td><td rowspan=1 colspan=1>4(3%)</td><td rowspan=1 colspan=1>1(2%)</td><td rowspan=1 colspan=1>1(1%)</td></tr><tr><td rowspan=1 colspan=1>18 - 34 years</td><td rowspan=1 colspan=1>70(8%)</td><td rowspan=1 colspan=1>18(13%)</td><td rowspan=1 colspan=1>0(0%)</td><td rowspan=1 colspan=1>17(7%)</td><td rowspan=1 colspan=1>3(10%)</td><td rowspan=1 colspan=1>10(8%)</td><td rowspan=1 colspan=1>10(6%)</td><td rowspan=1 colspan=1>5(11%)</td><td rowspan=1 colspan=1>7(8%)</td></tr><tr><td rowspan=2 colspan=1>35 - 65 years</td><td rowspan=2 colspan=1>470(56%)</td><td rowspan=2 colspan=1>78(58%)</td><td rowspan=2 colspan=1>0(0%)</td><td rowspan=1 colspan=1>152</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>70</td><td rowspan=1 colspan=1>88</td><td rowspan=1 colspan=1>27</td><td rowspan=2 colspan=1>55(60%)</td></tr><tr><td rowspan=1 colspan=1>(61%)</td><td rowspan=1 colspan=1>(0%)</td><td rowspan=1 colspan=1>(57%)</td><td rowspan=1 colspan=1>(56%)</td><td rowspan=1 colspan=1>(61%)</td></tr><tr><td rowspan=1 colspan=1>&gt; 65 years</td><td rowspan=1 colspan=1>255(30%)</td><td rowspan=1 colspan=1>38(28%)</td><td rowspan=1 colspan=1>0(0%)</td><td rowspan=1 colspan=1>82(33%)</td><td rowspan=1 colspan=1>0(0%)</td><td rowspan=1 colspan=1>43(35%)</td><td rowspan=1 colspan=1>53(34%)</td><td rowspan=1 colspan=1>11(25%)</td><td rowspan=1 colspan=1>28(31%)</td></tr><tr><td rowspan=3 colspan=1>Cea in</td><td rowspan=1 colspan=1>Hospitalized</td><td rowspan=1 colspan=1>666(79%)</td><td rowspan=1 colspan=1>116(86%)</td><td rowspan=1 colspan=1>12(92%)</td><td rowspan=1 colspan=1>223(89%)</td><td rowspan=1 colspan=1>9(29%)</td><td rowspan=1 colspan=1>82(67%)</td><td rowspan=1 colspan=1>118(75%)</td><td rowspan=1 colspan=1>25(57%)</td><td rowspan=1 colspan=1>81(89%)</td></tr><tr><td rowspan=1 colspan=1>Outpatient</td><td rowspan=1 colspan=1>159(19%)</td><td rowspan=1 colspan=1>18(13%)</td><td rowspan=1 colspan=1>0(0%)</td><td rowspan=1 colspan=1>28(11%)</td><td rowspan=1 colspan=1>22(71%)</td><td rowspan=1 colspan=1>31(25%)</td><td rowspan=1 colspan=1>39(25%)</td><td rowspan=1 colspan=1>14(32%)</td><td rowspan=1 colspan=1>7(8%)</td></tr><tr><td rowspan=1 colspan=1>Emergency</td><td rowspan=1 colspan=1>21(2%)</td><td rowspan=1 colspan=1>1(1%)</td><td rowspan=1 colspan=1>1(8%)</td><td rowspan=1 colspan=1>0(0%)</td><td rowspan=1 colspan=1>0(0%)</td><td rowspan=1 colspan=1>10(8%)</td><td rowspan=1 colspan=1>1(1%)</td><td rowspan=1 colspan=1>5(11%)</td><td rowspan=1 colspan=1>3(3%)</td></tr><tr><td rowspan=1 colspan=2>Total</td><td rowspan=1 colspan=1>846</td><td rowspan=1 colspan=1>135</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>251</td><td rowspan=1 colspan=1>31</td><td rowspan=1 colspan=1>123</td><td rowspan=1 colspan=1>158</td><td rowspan=1 colspan=1>44</td><td rowspan=1 colspan=1>91</td></tr></table>

a Subject age could not be determined for one specimen from Site 1

Table 50: Overall and Per Site Demographic Analysis for Sputum Specimens   

<table><tr><td colspan="10">Site 8</td></tr><tr><td rowspan="3">S</td><td rowspan="2"></td><td rowspan="2">Overall</td><td rowspan="2">Site 1</td><td rowspan="2">Site 2</td><td rowspan="2">Sputum Site 3</td><td rowspan="2">Site 4</td><td rowspan="2">Site 5</td><td rowspan="2">Site 6</td><td rowspan="2">Site 7</td></tr><tr><td>31</td></tr><tr><td>Male</td><td>481 (58%)</td><td>66 (59%)</td><td>54 (54%) 46</td><td>136 (56%)</td><td>97 (61%)</td><td>14 (82%)</td><td>(53%)</td><td>34 (74%)</td><td>49 (47%)</td></tr><tr><td></td><td>Female</td><td>355 (42%)</td><td>45 (41%) (46%)</td><td>105 (44%)</td><td>61 (39%)</td><td>3 (18%)</td><td>28 (47%)</td><td>12 (26%)</td><td>55 (53%)</td></tr><tr><td colspan="11" rowspan="1">Sputum</td></tr><tr><td colspan="1" rowspan="7">A0</td><td colspan="1" rowspan="2">≤ 5 years</td><td colspan="1" rowspan="2">138(17%)</td><td colspan="1" rowspan="2">0(0%)</td><td colspan="1" rowspan="2">49(49%)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">80</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">7</td></tr><tr><td colspan="1" rowspan="1">(0%)</td><td colspan="1" rowspan="1">(51%)</td><td colspan="1" rowspan="1">(0%)</td><td colspan="1" rowspan="1">(0%)</td><td colspan="1" rowspan="1">(4%)</td><td colspan="1" rowspan="1">(7%)</td></tr><tr><td colspan="1" rowspan="1">6 - 17 years</td><td colspan="1" rowspan="1">107(13%)</td><td colspan="1" rowspan="1">0(0%)</td><td colspan="1" rowspan="1">35(35%)</td><td colspan="1" rowspan="1">0(0%)</td><td colspan="1" rowspan="1">64(41%)</td><td colspan="1" rowspan="1">0(0%)</td><td colspan="1" rowspan="1">0(0%)</td><td colspan="1" rowspan="1">2(4%)</td><td colspan="1" rowspan="1">6(6%)</td></tr><tr><td colspan="1" rowspan="1">18 - 34 years</td><td colspan="1" rowspan="1">86(10%)</td><td colspan="1" rowspan="1">15(14%)</td><td colspan="1" rowspan="1">16(16%)</td><td colspan="1" rowspan="1">20(8%)</td><td colspan="1" rowspan="1">13(8%)</td><td colspan="1" rowspan="1">1(6%)</td><td colspan="1" rowspan="1">6(10%)</td><td colspan="1" rowspan="1">5(11%)</td><td colspan="1" rowspan="1">10(10%)</td></tr><tr><td colspan="1" rowspan="2">35 - 65 years</td><td colspan="1" rowspan="2">284(34%)</td><td colspan="1" rowspan="2">51(46%)</td><td colspan="1" rowspan="2">0(0%)</td><td colspan="1" rowspan="2">133(55%)</td><td colspan="1" rowspan="2">1(1%)</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="2">36(61%)</td><td colspan="1" rowspan="2">20(43%)</td><td colspan="1" rowspan="2">37(36%)</td></tr><tr><td colspan="1" rowspan="1">(35%)</td></tr><tr><td colspan="1" rowspan="1">&gt; 65 years</td><td colspan="1" rowspan="1">221(26%)</td><td colspan="1" rowspan="1">45(41%)</td><td colspan="1" rowspan="1">0(0%)</td><td colspan="1" rowspan="1">88(37%)</td><td colspan="1" rowspan="1">0(0%)</td><td colspan="1" rowspan="1">10(59%)</td><td colspan="1" rowspan="1">17(29%)</td><td colspan="1" rowspan="1">17(37%)</td><td colspan="1" rowspan="1">44(42%)</td></tr><tr><td colspan="1" rowspan="5">Ca in</td><td colspan="1" rowspan="1">Hospitalized</td><td colspan="1" rowspan="1">682(82%)</td><td colspan="1" rowspan="1">106(95%)</td><td colspan="1" rowspan="1">64(64%)</td><td colspan="1" rowspan="1">219(91%)</td><td colspan="1" rowspan="1">105(66%)</td><td colspan="1" rowspan="1">12(71%)</td><td colspan="1" rowspan="1">52(88%)</td><td colspan="1" rowspan="1">23(50%)</td><td colspan="1" rowspan="1">101(97%)</td></tr><tr><td colspan="1" rowspan="2">Outpatient</td><td colspan="1" rowspan="2">73(9%)</td><td colspan="1" rowspan="2">2(2%)</td><td colspan="1" rowspan="2">14(14%)</td><td colspan="1" rowspan="1">18</td><td colspan="1" rowspan="1">24</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">7</td><td colspan="1" rowspan="2">1(1%)</td></tr><tr><td colspan="1" rowspan="1">(7%)</td><td colspan="1" rowspan="1">(15%)</td><td colspan="1" rowspan="1">(12%)</td><td colspan="1" rowspan="1">(8%)</td><td colspan="1" rowspan="1">(15%)</td></tr><tr><td colspan="1" rowspan="2">Emergency</td><td colspan="1" rowspan="2">81(10%)</td><td colspan="1" rowspan="2">3(3%)</td><td colspan="1" rowspan="2">22(22%)</td><td colspan="1" rowspan="2">4(2%)</td><td colspan="1" rowspan="1">29</td><td colspan="1" rowspan="2">3(18%)</td><td colspan="1" rowspan="2">2(3%)</td><td colspan="1" rowspan="2">16(35%)</td><td colspan="1" rowspan="2">2(2%)</td></tr><tr><td colspan="1" rowspan="1">(18%)</td></tr><tr><td colspan="2" rowspan="1">Total</td><td colspan="1" rowspan="1">836</td><td colspan="1" rowspan="1">111</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">241</td><td colspan="1" rowspan="1">158</td><td colspan="1" rowspan="1">17</td><td colspan="1" rowspan="1">59</td><td colspan="1" rowspan="1">46</td><td colspan="1" rowspan="1">104</td></tr></table>

All specimens were evaluated with the FilmArray Pneumonia Panel at clinical study sites. Refrigerated specimen aliquots were sent to a central reference laboratory for quantitative reference culture (qRefCx) and frozen specimen aliquots were also sent to BioFire for evaluation by polymerase chain reaction (PCR)/sequencing-based comparator methods.

The reference methods used in this study were as follows:

Bacterial analytes were compared to qRefCx to evaluate sensitivity and specificity, and the method was considered positive for the presence of the organism of interest if it was recovered in culture and enumerated at a level of $1 0 ^ { { \bar { 3 } } . 5 } \mathrm { C F U / m L }$ or greater.

Bacterial analytes were also evaluated by comparison to a single PCR assay for the organism of interest followed by a quantitative molecular assay that included sequencing (qMol) to assess FilmArray bin reporting performance. Atypical bacteria and viruses were compared to two conventional PCR assays followed by bidirectional sequencing. For specimens with an applicable bacteria detected by FilmArray, AMR genes were compared to a single PCR assay (from the specimen) followed by sequencing. A specimen was considered to be positive for an analyte if bi-directional sequencing data meeting predefined quality acceptance criteria matched organism-specific sequences deposited in the NCBI GenBank database (www.ncbi.nlm.nih.gov) with acceptable E-values. When two PCR comparator assays were used, any specimen that tested negative by both of the comparator assays was considered Negative.

Positive Percent Agreement (PPA) or Sensitivity for each analyte was calculated as $100 \% \mathbf { x }$ $\left( \mathrm { T P } / \left( \mathrm { T P } + \right. \right.$ FN)). True positive (TP) indicates that both the FilmArray Pneumonia Panel and the comparator method had a positive result for this specific analyte, and false negative (FN) indicates that the FilmArray Pneumonia Panel result was negative while the comparator result was positive. Negative Percent Agreement (NPA) or Specificity was calculated as $100 \%$ x $\left( \mathrm { T N } / \left( \mathrm { T N } + \mathrm { F P } \right) \right)$ . True negative (TN) indicates that both the FilmArray Pneumonia Panel and the comparator method had negative results, and a false positive (FP) indicates that the FilmArray Pneumonia Panel result was positive but the comparator result was negative. The exact binomial two-sided $9 5 \%$ confidence interval was calculated. Samples for which false positive and/or false negative results (i.e., discrepant results) were obtained when comparing the FilmArray Pneumonia Panel results to the comparator method results were further investigated. The discrepancy investigations were primarily performed as follows: 1) for discrepancies between the Pneumonia Panel and reference culture for bacterial analytes, 2) discrepancies were first examined to see if qRefCx or FilmArray had observed the analyte but reported it as “negative” or “Not Detected” because it was below the detection threshold. If this did not resolve the discrepancy, the results of qMol testing were considered. And if these methods still did not resolve the discrepancy, it was then investigated in the same manner as other analytes that used molecular comparator (i.e. using multiple additional molecular assays followed by sequence analysis). The results of SOC (Standard of Care) testing were also considered. The prospective clinical study results are summarized in Table 51 and Table 52 for BAL and sputum specimens, respectively.

Table 51: FilmArray Pneumonia Panel Clinical Performance Summary for BAL Specimens a   

<table><tr><td colspan="8" rowspan="1">BAL</td></tr><tr><td colspan="1" rowspan="2">Analyte</td><td colspan="1" rowspan="2">ReferenceMethod</td><td colspan="3" rowspan="1">Sensitivity/PPA</td><td colspan="3" rowspan="1">Specificity/NPA</td></tr><tr><td colspan="1" rowspan="1">TP/(TP + FN)</td><td colspan="1" rowspan="1">%</td><td colspan="1" rowspan="1">95%CI</td><td colspan="1" rowspan="1">TN/(TN + FP)</td><td colspan="1" rowspan="1">%</td><td colspan="1" rowspan="1">95%CI</td></tr><tr><td colspan="7" rowspan="1">Bacteria</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Acinetobacter calcoaceticus-baumanniicompllex b</td><td colspan="1" rowspan="1">qRefCx</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">839/846</td><td colspan="1" rowspan="1">99.2</td><td colspan="1" rowspan="1">98.3-99.6%</td></tr><tr><td colspan="1" rowspan="1">Enterobacter cloacae complex </td><td colspan="1" rowspan="1">qRefCx</td><td colspan="1" rowspan="1">11/12</td><td colspan="1" rowspan="1">91.7</td><td colspan="1" rowspan="1">64.6-98.5%</td><td colspan="1" rowspan="1">822/834</td><td colspan="1" rowspan="1">98.6</td><td colspan="1" rowspan="1">97.5-99.2%</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli d</td><td colspan="1" rowspan="1">qRefCx</td><td colspan="1" rowspan="1">12/12</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">75.8-100%</td><td colspan="1" rowspan="1">826/834</td><td colspan="1" rowspan="1">99.0</td><td colspan="1" rowspan="1">98.1-99.5%</td></tr><tr><td colspan="1" rowspan="1">Haemophilus influenzae </td><td colspan="1" rowspan="1">qRefCx</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">72.2-100%</td><td colspan="1" rowspan="1">764/836</td><td colspan="1" rowspan="1">91.4</td><td colspan="1" rowspan="1">89.3-93.1%</td></tr><tr><td colspan="1" rowspan="1">Klebsiella aerogenesf</td><td colspan="1" rowspan="1">qRefCx</td><td colspan="1" rowspan="1">6/7</td><td colspan="1" rowspan="1">85.7</td><td colspan="1" rowspan="1">48.7-97.4%</td><td colspan="1" rowspan="1">832/839</td><td colspan="1" rowspan="1">99.2</td><td colspan="1" rowspan="1">98.3-99.6%</td></tr><tr><td colspan="1" rowspan="1">Klebsiella oxytoca 8</td><td colspan="1" rowspan="1">qRefCx</td><td colspan="1" rowspan="1">2/2</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">34.2-100%</td><td colspan="1" rowspan="1">835/844</td><td colspan="1" rowspan="1">98.9</td><td colspan="1" rowspan="1">98.0-99.4%</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae group h</td><td colspan="1" rowspan="1">qRefCx</td><td colspan="1" rowspan="1">15/15</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">79.6-100%</td><td colspan="1" rowspan="1">819/831</td><td colspan="1" rowspan="1">98.6</td><td colspan="1" rowspan="1">97.5-99.2%</td></tr><tr><td colspan="1" rowspan="1">Moraxella catarrhalis</td><td colspan="1" rowspan="1">qRefCx</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">817/846</td><td colspan="1" rowspan="1">96.6</td><td colspan="1" rowspan="1">95.1-97.6%</td></tr><tr><td colspan="1" rowspan="1">Proteus spp.</td><td colspan="1" rowspan="1">qRefCx</td><td colspan="1" rowspan="1">5/5</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">56.6-100%</td><td colspan="1" rowspan="1">837/841</td><td colspan="1" rowspan="1">99.5</td><td colspan="1" rowspan="1">98.8-99.8%</td></tr><tr><td colspan="1" rowspan="1">Pseudomonas aeruginosa k</td><td colspan="1" rowspan="1">qRefCx</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">90.4-100%</td><td colspan="1" rowspan="1">772/810</td><td colspan="1" rowspan="1">95.3</td><td colspan="1" rowspan="1">93.6-96.6%</td></tr><tr><td colspan="1" rowspan="1">Serratia marcescens1</td><td colspan="1" rowspan="1">qRefCx</td><td colspan="1" rowspan="1">6/6</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">61.0-100%</td><td colspan="1" rowspan="1">834/840</td><td colspan="1" rowspan="1">99.3</td><td colspan="1" rowspan="1">98.5-99.7%</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus m</td><td colspan="1" rowspan="1">qRefCx</td><td colspan="1" rowspan="1">46/47</td><td colspan="1" rowspan="1">97.9</td><td colspan="1" rowspan="1">88.9-99.6%</td><td colspan="1" rowspan="1">729/799</td><td colspan="1" rowspan="1">91.2</td><td colspan="1" rowspan="1">89.1-93.0%</td></tr><tr><td colspan="1" rowspan="1">Streptococcus agalactiae "</td><td colspan="1" rowspan="1">qRefCx</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">821/845</td><td colspan="1" rowspan="1">97.2</td><td colspan="1" rowspan="1">95.8-98.1%</td></tr><tr><td colspan="1" rowspan="1">Streptococcus pneumoniae</td><td colspan="1" rowspan="1">qRefCx</td><td colspan="1" rowspan="1">5/5</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">56.6-100%</td><td colspan="1" rowspan="1">817/841</td><td colspan="1" rowspan="1">97.1</td><td colspan="1" rowspan="1">95.8-98.1%</td></tr><tr><td colspan="1" rowspan="1">TP/(TP +FN)</td><td colspan="1" rowspan="1">%</td><td colspan="1" rowspan="1">95%CI</td><td colspan="1" rowspan="1">TN/(TN + FP)</td><td colspan="1" rowspan="1">%</td><td colspan="1" rowspan="1">95%CI</td></tr><tr><td colspan="1" rowspan="1">Streptococcus pyogenes P</td><td colspan="1" rowspan="1">qRefCx</td><td colspan="1" rowspan="1">2/2</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">34.2-100%</td><td colspan="1" rowspan="1">838/844</td><td colspan="1" rowspan="1">99.3</td><td colspan="1" rowspan="1">98.5-99.7%</td></tr><tr><td colspan="3" rowspan="1">Atypical Bacteria</td><td></td><td></td><td></td><td></td><td></td></tr><tr><td colspan="1" rowspan="1">Chlamydia pneumoniae </td><td colspan="1" rowspan="1">PCR/Seq</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">844/845</td><td colspan="1" rowspan="1">99.9</td><td colspan="1" rowspan="1">99.3-100%</td></tr><tr><td colspan="1" rowspan="1">Legionella pneumophila</td><td colspan="1" rowspan="1">PCR/Seq</td><td colspan="1" rowspan="1">2/2</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">34.2-100%</td><td colspan="1" rowspan="1">833/833</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">99.5-100%</td></tr><tr><td colspan="1" rowspan="1">Mycoplasma pneumoniae "</td><td colspan="1" rowspan="1">PCR/Seq</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">43.9-100%</td><td colspan="1" rowspan="1">841/842</td><td colspan="1" rowspan="1">99.9</td><td colspan="1" rowspan="1">99.3-100%</td></tr><tr><td></td><td></td><td colspan="1" rowspan="1">Viruses</td><td></td><td></td><td></td><td></td><td></td></tr><tr><td colspan="1" rowspan="1">Adenovirus</td><td colspan="1" rowspan="1">PCR/Seq</td><td colspan="1" rowspan="1">8/8</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">67.6-100%</td><td colspan="1" rowspan="1">837/837</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">99.5-100%</td></tr><tr><td colspan="1" rowspan="1">Coronavirus </td><td colspan="1" rowspan="1">PCR/Seq</td><td colspan="1" rowspan="1">18/21</td><td colspan="1" rowspan="1">85.7</td><td colspan="1" rowspan="1">65.4-95.0%</td><td colspan="1" rowspan="1">810/823</td><td colspan="1" rowspan="1">98.4</td><td colspan="1" rowspan="1">97.3-99.1%</td></tr><tr><td colspan="1" rowspan="1">Human Metapneumovirus t</td><td colspan="1" rowspan="1">PCR/Seq</td><td colspan="1" rowspan="1">8/8</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">67.6-100%</td><td colspan="1" rowspan="1">836/837</td><td colspan="1" rowspan="1">99.9</td><td colspan="1" rowspan="1">99.3-100%</td></tr><tr><td colspan="1" rowspan="1">Human Rhinovirus/Enterovirus "</td><td colspan="1" rowspan="1">PCR/Seq</td><td colspan="1" rowspan="1">52/54</td><td colspan="1" rowspan="1">96.3</td><td colspan="1" rowspan="1">87.5-99.0%</td><td colspan="1" rowspan="1">771/782</td><td colspan="1" rowspan="1">98.6</td><td colspan="1" rowspan="1">97.5-99.2%</td></tr><tr><td colspan="1" rowspan="1">Influenza A v</td><td colspan="1" rowspan="1">PCR/Seq</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">72.2-100%</td><td colspan="1" rowspan="1">830/833</td><td colspan="1" rowspan="1">99.6</td><td colspan="1" rowspan="1">98.9-99.9%</td></tr><tr><td colspan="1" rowspan="1">Influenza B w</td><td colspan="1" rowspan="1">PCR/Seq</td><td colspan="1" rowspan="1">5/6</td><td colspan="1" rowspan="1">83.3</td><td colspan="1" rowspan="1">43.6-97.0%</td><td colspan="1" rowspan="1">837/838</td><td colspan="1" rowspan="1">99.9</td><td colspan="1" rowspan="1">99.3-100%</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza Virus </td><td colspan="1" rowspan="1">PCR/Seq</td><td colspan="1" rowspan="1">16/18</td><td colspan="1" rowspan="1">88.9</td><td colspan="1" rowspan="1">67.2-96.9%</td><td colspan="1" rowspan="1">824/826</td><td colspan="1" rowspan="1">99.8</td><td colspan="1" rowspan="1">99.1-99.9%</td></tr><tr><td colspan="1" rowspan="1">Respiratory Syncytial Virus</td><td colspan="1" rowspan="1">PCR/Seq</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">43.9-100%</td><td colspan="1" rowspan="1">841/841</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">99.5-100%</td></tr></table>

a. The performance measures of sensitivity and specificity only refer to the bacterial analytes for which the gold-standard of qRefCx was used as the reference method. Performance measures of PPA and NPA refer to all other analytes, for which PCR/sequencing assays were used as comparator methods.

b. Evidence of ACB complex was found in all seven FP specimens; one was enumerated below $1 0 ^ { 3 . 5 }$ CFU/mL by qRefCx and six were detected by qMol.

c. $E$ . cloacae complex was observed in the single FN specimen below the $1 0 ^ { 4 }$ bin by the FilmArray Pneumonia Panel. Evidence of $E$ . cloacae complex was found in all $1 2 \mathrm { F P }$ specimens; six were enumerated below $1 0 ^ { 3 . 5 } \mathrm { C F U / m L }$ by qRefCx, five were detected by $\mathbf { q M o l }$ , and one was detected using an additional molecular method.

d. Evidence of $E$ . coli was found in all eight FP specimens; six were enumerated below $1 0 ^ { 3 . 5 } \mathrm { C F U / m L }$ by qRefCx and two were detected by qMol.

Evidence of H. influenzae was found in all 72 FP specimens; seven were enumerated below $1 0 ^ { 3 . 5 } \mathrm { C F U / m L }$ by qRefCx, 56 were detected by qMol, eight were detected using an additional molecular method, and one was identified in SOC culture.

$K .$ . aerogenes was identified in the single FN specimen in SOC culture. Evidence of $K$ . aerogenes was found in all seven FP specimens; four were enumerated below $1 0 ^ { 3 . 5 } \mathrm { C F U / m L }$ by qRefCx and three were detected by qMol.

g. Evidence of $K$ . oxytoca was found in all nine FP specimens; three were enumerated below $1 0 ^ { 3 . 5 } \mathrm { C F U / m L }$ by qRefCx, five were detected by qMol, and one was detected using an additional molecular method.

Evidence of $K$ . pneumoniae group was found in all $1 2 \mathrm { F P }$ specimens; seven were enumerated below $1 0 ^ { 3 . 5 }$ CFU/mL by qRefCx, four were detected b qMol, and one was detected using an additional molecular method.

Evidence of $M .$ . catarrhalis was found in all $2 9 \mathrm { F P }$ specimens; two were enumerated below $1 0 ^ { 3 . 5 }$ CFU/mL by qRefCx, 25 were detected by qMol, and two were detected using an additional molecular method.

j. Evidence of Proteus spp. was found in all four FP specimens; three were enumerated below $1 0 ^ { 3 . 5 }$ CFU/mL by qRefCx and one was detected by qMol.

k. Evidence of $\cdot \cal { P }$ . aeruginosa was found in all 38 FP specimens; 19 were enumerated below $1 0 ^ { 3 . 5 }$ CFU/mL by qRefCx, 16 were detected by qMol, and three were detected using an additional molecular method.

l. Evidence of S. marcescens was found in all six FP specimens; four were enumerated below $1 0 ^ { 3 . 5 }$ CFU/mL by qRefCx and two were detected by qMol.

m. S. aureus was detected in the single FN specimen using an additional molecular method. Evidence of $S$ . aureus was found in 69/70 FP specimens; 29 were enumerated below $1 0 ^ { 3 . 5 } \mathrm { C F U / m L }$ by qRefCx, 30 were detected by qMol, eight were detected using an additional molecular method, and two were identified in SOC culture.   
n. Evidence of $S _ { \cdot }$ . agalactiae was found in all $2 4 \mathrm { F P }$ specimens; seven were enumerated below $1 0 ^ { 3 . 5 }$ CFU/mL by qRefCx, 13 were detected by qMol, and four were detected using an additional molecular method.   
o. Evidence of S. pneumoniae was found in all $2 4 \mathrm { F P }$ specimens; five were enumerated below $1 0 ^ { 3 . 5 }$ CFU/mL by qRefCx, 18 were detected by qMol, and one was detected using an additional molecular method.   
p. Evidence of S. pyogenes was found in all six FP specimens; two were enumerated below $1 0 ^ { 3 . 5 }$ CFU/mL by qRefCx, three were detected by qMol, and one was detected using an additional molecular method.   
q. The single FP specimen was negative for $C$ . pneumoniae when tested with additional molecular methods during discrepancy investigation.   
r. The single FP specimen was negative for M. pneumoniae when tested with additional molecular methods during discrepancy investigation.   
s. CoV was detected in $2 / 3 \ \mathrm { F N }$ and 8/13 FP specimens using an additional molecular method.   
t. The single FP specimen was negative for hMPV when tested with additional molecular methods during discrepancy investigation.   
u. HRV/EV was detected in both FN specimens using an additional molecular method. HRV/EV was detected in 8/11 FP specimens during discrepancy investigation; seven were detected using an additional molecular method and one was detected upon FilmArray Pneumonia Panel retest.   
v. FluA was detected in 2/3 FP specimens using an additional molecular method.   
w. FluB was detected in the single FN specimen upon FilmArray Pneumonia Panel retest. FluB was detected in the single FP specimen using an additional molecular method.   
x. PIV was detected in both FN and both FP specimens using an additional molecular method.

Table 52: FilmArray Pneumonia Panel Clinical Performance Summary for Sputum Specimensa   

<table><tr><td colspan="8" rowspan="1">Sputum</td></tr><tr><td colspan="1" rowspan="2">Analyte</td><td colspan="1" rowspan="2">ReferenceMethod</td><td colspan="3" rowspan="1">Sensitivity/PPA</td><td colspan="3" rowspan="1">Specificity/NPA</td></tr><tr><td colspan="1" rowspan="1">TP/(TP + FN)</td><td colspan="1" rowspan="1">%</td><td colspan="1" rowspan="1">95%CI</td><td colspan="1" rowspan="1">TN/(TN + FP)</td><td colspan="1" rowspan="1">%</td><td colspan="1" rowspan="1">95%CI</td></tr><tr><td colspan="5" rowspan="1">Bacteria</td><td colspan="3" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Acinetobacter calcoaceticus-baumanniicomplexb</td><td colspan="1" rowspan="1">qRefCx</td><td colspan="1" rowspan="1">10/11</td><td colspan="1" rowspan="1">90.9</td><td colspan="1" rowspan="1">62.3-98.4%</td><td colspan="1" rowspan="1">807/825</td><td colspan="1" rowspan="1">97.8</td><td colspan="1" rowspan="1">96.6-98.6%</td></tr><tr><td colspan="1" rowspan="1">Enterobacter cloacae complex c</td><td colspan="1" rowspan="1">qRefCx</td><td colspan="1" rowspan="1">11/12</td><td colspan="1" rowspan="1">91.7</td><td colspan="1" rowspan="1">64.6-98.5%</td><td colspan="1" rowspan="1">803/824</td><td colspan="1" rowspan="1">97.5</td><td colspan="1" rowspan="1">96.1-98.3%</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli d</td><td colspan="1" rowspan="1">qRefCx</td><td colspan="1" rowspan="1">23/24</td><td colspan="1" rowspan="1">95.8</td><td colspan="1" rowspan="1">79.8-99.3%</td><td colspan="1" rowspan="1">787/812</td><td colspan="1" rowspan="1">96.9</td><td colspan="1" rowspan="1">95.5-97.9%</td></tr><tr><td colspan="1" rowspan="1">Haemophilus influenzae </td><td colspan="1" rowspan="1">qRefCx</td><td colspan="1" rowspan="1">16/18</td><td colspan="1" rowspan="1">88.9</td><td colspan="1" rowspan="1">67.2-96.9%</td><td colspan="1" rowspan="1">727/818</td><td colspan="1" rowspan="1">88.9</td><td colspan="1" rowspan="1">86.5-90.9%</td></tr><tr><td colspan="1" rowspan="1">Klebsiella aerogenes f</td><td colspan="1" rowspan="1">qRefCx</td><td colspan="1" rowspan="1">3/4</td><td colspan="1" rowspan="1">75.0</td><td colspan="1" rowspan="1">30.1-95.4%</td><td colspan="1" rowspan="1">823/832</td><td colspan="1" rowspan="1">98.9</td><td colspan="1" rowspan="1">98.0-99.4%</td></tr><tr><td colspan="1" rowspan="1">Klebsiella oxytoca g</td><td colspan="1" rowspan="1">qRefCx</td><td colspan="1" rowspan="1">9/9</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">70.1-100%</td><td colspan="1" rowspan="1">817/827</td><td colspan="1" rowspan="1">98.8</td><td colspan="1" rowspan="1">97.8-99.3%</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae group h</td><td colspan="1" rowspan="1">qRefCx</td><td colspan="1" rowspan="1">21/23</td><td colspan="1" rowspan="1">91.3</td><td colspan="1" rowspan="1">73.2-97.6%</td><td colspan="1" rowspan="1">769/813</td><td colspan="1" rowspan="1">94.6</td><td colspan="1" rowspan="1">92.8-95.9%</td></tr><tr><td colspan="1" rowspan="1">Moraxella catarrhalis </td><td colspan="1" rowspan="1">qRefCx</td><td colspan="1" rowspan="1">5/5</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">56.6-100%</td><td colspan="1" rowspan="1">761/831</td><td colspan="1" rowspan="1">91.6</td><td colspan="1" rowspan="1">89.5-93.3%</td></tr><tr><td colspan="1" rowspan="1">Proteus spp. </td><td colspan="1" rowspan="1">qRefCx</td><td colspan="1" rowspan="1">15/15</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">79.6-100%</td><td colspan="1" rowspan="1">813/821</td><td colspan="1" rowspan="1">99.0</td><td colspan="1" rowspan="1">98.1-99.5%</td></tr><tr><td colspan="1" rowspan="1">Pseudomonas aeruginosa *</td><td colspan="1" rowspan="1">qRefCx</td><td colspan="1" rowspan="1">103/106</td><td colspan="1" rowspan="1">97.2</td><td colspan="1" rowspan="1">92.0-99.0%</td><td colspan="1" rowspan="1">673/730</td><td colspan="1" rowspan="1">92.2</td><td colspan="1" rowspan="1">90.0-93.9%</td></tr><tr><td colspan="1" rowspan="1">Serratia marcescens</td><td colspan="1" rowspan="1">qRefCx</td><td colspan="1" rowspan="1">26/27</td><td colspan="1" rowspan="1">96.3</td><td colspan="1" rowspan="1">81.7-99.3%</td><td colspan="1" rowspan="1">782/809</td><td colspan="1" rowspan="1">96.7</td><td colspan="1" rowspan="1">95.2-97.7%</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus m</td><td colspan="1" rowspan="1">qRefCx</td><td colspan="1" rowspan="1">111/112</td><td colspan="1" rowspan="1">99.1</td><td colspan="1" rowspan="1">95.1-99.8%</td><td colspan="1" rowspan="1">631/724</td><td colspan="1" rowspan="1">87.2</td><td colspan="1" rowspan="1">84.5-89.4%</td></tr><tr><td colspan="1" rowspan="1">Streptococcus agalactiae "</td><td colspan="1" rowspan="1">qRefCx</td><td colspan="1" rowspan="1">9/9</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">70.1-100%</td><td colspan="1" rowspan="1">793/827</td><td colspan="1" rowspan="1">95.9</td><td colspan="1" rowspan="1">94.3-97.0%</td></tr><tr><td colspan="1" rowspan="1">Streptococcus pneumoniae </td><td colspan="1" rowspan="1">qRefCx</td><td colspan="1" rowspan="1">16/16</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">80.6-100%</td><td colspan="1" rowspan="1">785/820</td><td colspan="1" rowspan="1">95.7</td><td colspan="1" rowspan="1">94.1-96.9%</td></tr><tr><td colspan="1" rowspan="1">Streptococcus pyogenes P</td><td colspan="1" rowspan="1">qRefCx</td><td colspan="1" rowspan="1">6/6</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">61.0-100%</td><td colspan="1" rowspan="1">825/830</td><td colspan="1" rowspan="1">99.4</td><td colspan="1" rowspan="1">98.6-99.7%</td></tr><tr><td colspan="8" rowspan="1">Atypical Bacteria</td></tr><tr><td colspan="1" rowspan="1">Chlamydia pneumonia</td><td colspan="1" rowspan="1">PCR/Seq</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">835/835</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">99.5-100%</td></tr><tr><td colspan="1" rowspan="1">Legionella pneumophila 9</td><td colspan="1" rowspan="1">PCR/Seq</td><td colspan="1" rowspan="1">0/1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">826/826</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">99.5-100%</td></tr><tr><td colspan="1" rowspan="1">Mycoplasma pneumoniae </td><td colspan="1" rowspan="1">PCR/Seq</td><td colspan="1" rowspan="1">7/8</td><td colspan="1" rowspan="1">87.5</td><td colspan="1" rowspan="1">52.9-97.8%</td><td colspan="1" rowspan="1">827/827</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">99.5-100%</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="3" rowspan="1">Viruses</td><td colspan="3" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Adenovirus </td><td colspan="1" rowspan="1">PCR/Seq</td><td colspan="1" rowspan="1">13/17</td><td colspan="1" rowspan="1">76.5</td><td colspan="1" rowspan="1">52.7-90.4%</td><td colspan="1" rowspan="1">815/817</td><td colspan="1" rowspan="1">99.8</td><td colspan="1" rowspan="1">99.1-99.9%</td></tr><tr><td colspan="1" rowspan="1">Coronavirus </td><td colspan="1" rowspan="1">PCR/Seq</td><td colspan="1" rowspan="1">28/32</td><td colspan="1" rowspan="1">87.5</td><td colspan="1" rowspan="1">71.9-95.0%</td><td colspan="1" rowspan="1">796/802</td><td colspan="1" rowspan="1">99.3</td><td colspan="1" rowspan="1">98.4-99.7%</td></tr><tr><td colspan="1" rowspan="1">Human Metapneumovirus "</td><td colspan="1" rowspan="1">PCR/Seq</td><td colspan="1" rowspan="1">20/21</td><td colspan="1" rowspan="1">95.2</td><td colspan="1" rowspan="1">77.3-99.2%</td><td colspan="1" rowspan="1">812/813</td><td colspan="1" rowspan="1">99.9</td><td colspan="1" rowspan="1">99.3-100%</td></tr><tr><td colspan="1" rowspan="1">Human Rhinovirus/Enterovirus v</td><td colspan="1" rowspan="1">PCR/Seq</td><td colspan="1" rowspan="1">96/96</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">96.2-100%</td><td colspan="1" rowspan="1">717/730</td><td colspan="1" rowspan="1">98.2</td><td colspan="1" rowspan="1">97.0-99.0%</td></tr><tr><td colspan="1" rowspan="1">Influenza A w</td><td colspan="1" rowspan="1">PCR/Seq</td><td colspan="1" rowspan="1">13/13</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">77.2-100%</td><td colspan="1" rowspan="1">819/822</td><td colspan="1" rowspan="1">99.6</td><td colspan="1" rowspan="1">98.9-99.9%</td></tr><tr><td colspan="1" rowspan="1">Influenza B x</td><td colspan="1" rowspan="1">PCR/Seq</td><td colspan="1" rowspan="1">12/12</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">75.8-100%</td><td colspan="1" rowspan="1">821/823</td><td colspan="1" rowspan="1">99.8</td><td colspan="1" rowspan="1">99.1-99.9%</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza Virus y</td><td colspan="1" rowspan="1">PCR/Seq</td><td colspan="1" rowspan="1">28/29</td><td colspan="1" rowspan="1">96.6</td><td colspan="1" rowspan="1">82.8-99.4%</td><td colspan="1" rowspan="1">804/806</td><td colspan="1" rowspan="1">99.8</td><td colspan="1" rowspan="1">99.1-99.9%</td></tr><tr><td colspan="1" rowspan="1">Respiratory Syncytial Virus 2</td><td colspan="1" rowspan="1">PCR/Seq</td><td colspan="1" rowspan="1">43/43</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">91.8-100%</td><td colspan="1" rowspan="1">787/791</td><td colspan="1" rowspan="1">99.5</td><td colspan="1" rowspan="1">98.7-99.8%</td></tr></table>

a. The performance measures of sensitivity and specificity only refer to the bacterial analytes for which the gold-standard of qRefCx was used as the reference method. Performance measures of PPA and NPA refer to all other analytes, for which PCR/sequencing assays were used as comparator methods.

b. The isolate recovered from the single FN specimen was misidentified by qRefCx; molecular testing of the isolate identified it as Pseudomonas fluorescens during discrepancy investigation. Evidence of ACB complex was found in all $1 8 \mathrm { F P }$ specimens; 15 were detected by qMol, two were detected using an additional molecular method, and one was identified in SOC culture.

c. E. cloacae complex was detected in the single FN specimen using an additional molecular method. Evidence of $E$ . cloacae complex was found in all $2 1 \ \mathrm { F P }$ specimens; four were enumerated below $1 0 ^ { 3 . 5 } \mathrm { C F U / m L }$ by qRefCx, 16 were detected by qMol, and one was detected using an additional molecular method.

d. $E$ . coli was observed in the single FN specimen below the $1 0 ^ { 4 }$ bin by the FilmArray Pneumonia Panel. Evidence of $E$ . coli was found in all 25 FP specimens; six were enumerated below $1 0 ^ { 3 . 5 }$ CFU/mL by qRefCx, 14 were detected by qMol, and five were detected using an additional molecular method.

e. H. influenzae was detected in 1/2 FN specimens by qMol. The isolate recovered from the other FN specimen was misidentified by qRefCx; molecular testing of the isolate identified it as Haemophilus haemolyticus during discrepancy investigation. Evidence of H. influenzae was found in all 91 FP specimens; four were enumerated below $1 0 ^ { 3 . 5 } \mathrm { C F U / m L }$ by qRefCx, 78 were detected by $\mathbf { \boldsymbol { q } } \mathbf { \boldsymbol { M } } \mathbf { \boldsymbol { o l } }$ , seven were detected using an additional molecular method, and two were identified in SOC culture.

f. The isolate recovered from the single FN specimen was misidentified by qRefCx; molecular testing of the isolate identified it as Hafnia paralvei during discrepancy investigation. Evidence of $K$ . aerogenes was found in all nine FP specimens; three were enumerated below $1 0 ^ { 3 . 5 }$ $\mathrm { C F U / m L }$ by qRefCx, five were detected by qMol, and one was detected using an additional molecular method.

g. Evidence of K. oxytoca was found in all $1 0 \mathrm { F P }$ specimens; three were enumerated below $1 0 ^ { 3 . 5 }$ CFU/mL by qRefCx, five were detected by qMol, and two were detected using an additional molecular method.

h. K. pneumoniae group was detected in 1/2 FN specimens by qMol. The other FN appeared to be a result of a specimen swap at the central reference laboratory. Evidence of K. pneumoniae group was found in 43/44 FP specimens; 15 were enumerated below $1 0 ^ { 3 . 5 }$ CFU/mL by qRefCx, 21 were detected by qMol, and seven were detected using an additional molecular method.

i. Evidence of M. catarrhalis was found in all $7 0 \mathrm { F P }$ specimens; one was enumerated below $1 0 ^ { 3 . 5 } \mathrm { C F U / m L }$ by qRefCx, 63 were detected by qMol, five were detected using an additional molecular method, and one was identified in SOC culture.

j. Evidence of Proteus spp. was found in all eight FP specimens; two were enumerated below $1 0 ^ { 3 . 5 }$ CFU/mL by qRefCx, four were detected by qMol, and two were detected using an additional molecular method.

k. $P$ . aeruginosa was observed in 1/3 FN specimens below the $1 0 ^ { 4 }$ bin by the FilmArray Pneumonia Panel. The isolates recovered from the other two FN specimens were misidentified by qRefCx; molecular testing of the isolates identified one as Pseudomonas denitrificans and the other

as Pseudomonas fluorescens during discrepancy investigation. Evidence of $P$ . aeruginosa was found in all 57 FP specimens; 21 were enumerated below $1 0 ^ { 3 . 5 } \mathrm { C F U / m L }$ by qRefCx, 33 were detected by qMol, two were detected using an additional molecular method, and one was identified in SOC culture. S. marcescens was observed in the single FN specimen below the $1 0 ^ { 4 }$ bin by the FilmArray Pneumonia Panel. Evidence of S. marcescens was found in 26/27 FP specimens; seven were enumerated below $0 ^ { 3 . 5 } \mathrm { C F U / m L }$ by qRefCx, 16 were detected by $\mathbf { \boldsymbol { q } } \mathbf {  { M } } \mathbf {  { o } } \mathbf { \Lambda }$ , and three were detected using an additional molecular method. m. S. aureus was observed in the single FN specimen below the $1 0 ^ { 4 }$ bin by the FilmArray Pneumonia Panel. Evidence of S. aureus was found in all 93 FP specimens; 43 were enumerated below $1 0 ^ { 3 . 5 }$ CFU/mL by qRefCx, 43 were detected by qMol, three were detected using an additional molecular method, and four were identified in SOC culture. . Evidence of S. agalactiae was found in all $3 4 \mathrm { F P }$ specimens; five were enumerated below $1 0 ^ { 3 . 5 }$ CFU/mL by qRefCx, 24 were detected by qMol, and five were detected using an additional molecular method . Evidence of S. pneumoniae was found in all $3 5 \mathrm { F P }$ specimens; one was enumerated below $1 0 ^ { 3 . 5 }$ CFU/mL by qRefCx and 34 were detected by qMol. p. Evidence of $S _ { \cdot }$ . pyogenes was found in all five FP specimens; four were detected by qMol and one was detected using an additional molecular method. q. L. pneumophila was detected in the single FN specimen using an additional molecular method. r. The single FN specimen was negative for $M .$ . pneumoniae when tested during discrepancy investigation. s. AdV was detected in all four FN and 1/2 FP specimens using an additional molecular method. t. CoV was detected in all four FN and $3 / 6 \mathrm { F P }$ specimens using an additional molecular method. u. hMPV was detected in the single FN specimen using an additional molecular method. The single FP specimen was negative for hMPV when tested during discrepancy investigation. v. HRV/EV was detected in 12/13 FP specimens during discrepancy investigation; 11 were detected using an additional molecular method and one was detected upon FilmArray Pneumonia Panel retest. w. FluA was detected in all three FP specimens using an additional molecular method. x. Both FP specimens were negative for FluB when tested with additional molecular methods during discrepancy investigation. y. PIV was detected in the single FN and $1 / 2 \mathrm { F P }$ specimens using an additional molecular method. z. RSV was detected in all four FP specimens using an additional molecular method.

A total of 156 BAL specimens and 295 sputum specimens received a FilmArray Pneumonia Panel Detected result for at least one applicable gram-negative bacterium on the panel and reported results for CTX-M, IMP, KPC, NDM, OXA-48-like, and VIM; a total of 94 BAL specimens and 196 sputum specimens received a FilmArray Pneumonia Panel Detected result for at least one applicable gramnegative bacterium on the panel and reported results for OXA-48-like, and a total of 116 BAL specimens and 204 sputum specimens received a FilmArray Pneumonia Panel Staphylococcus aureus Detected result and reported results for mecA/C and MREJ. Performance of the Pneumonia Panel AMR gene assays was calculated by comparing results of qMol direct from these specimens and is shown in Table 53 (five BAL and four sputum specimens were excluded from qMol analysis due to invalid comparator results).

Table 53: FilmArray Pneumonia Panel Clinical Performance Summary – AMR Genes (Comparator method: qMol direct from specimen)a   

<table><tr><td rowspan=3 colspan=1>Analyte</td><td rowspan=1 colspan=6>BAL</td><td rowspan=1 colspan=6>Sputum</td></tr><tr><td rowspan=1 colspan=3>PPA</td><td rowspan=1 colspan=3>NPA</td><td rowspan=1 colspan=3>PPA</td><td rowspan=1 colspan=3>NPA</td></tr><tr><td rowspan=1 colspan=1>TP/(TP +FN</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95%CI</td><td rowspan=1 colspan=1>TN/(TN +FP</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95%CI</td><td rowspan=1 colspan=1>TP/(TP+FN</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95%C I</td><td rowspan=1 colspan=1>TN/(TN +FP</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95%CI</td></tr><tr><td rowspan=1 colspan=1>CTX-Mb</td><td rowspan=1 colspan=1>6/7</td><td rowspan=1 colspan=1>85.7</td><td rowspan=1 colspan=1>48.7-97.4%</td><td rowspan=1 colspan=1>144/144</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>97.4-100%</td><td rowspan=1 colspan=1>8/10</td><td rowspan=1 colspan=1>80.0</td><td rowspan=1 colspan=1>49.0-94.3%</td><td rowspan=1 colspan=1>280/281</td><td rowspan=1 colspan=1>99.6</td><td rowspan=1 colspan=1>98.0-99.9%</td></tr><tr><td rowspan=1 colspan=1>IMP</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>151/151</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>97.5-100%</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>291/291</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>98.7-100%</td></tr><tr><td rowspan=1 colspan=1>KPCc</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>34.2-100%</td><td rowspan=1 colspan=1>148/149</td><td rowspan=1 colspan=1>99.3</td><td rowspan=1 colspan=1>96.3-99.9%</td><td rowspan=1 colspan=1>7/7</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>64.6-100%</td><td rowspan=1 colspan=1>284/284</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>98.7-100%</td></tr><tr><td rowspan=1 colspan=1>mecA/CandMREJd</td><td rowspan=1 colspan=1>40/45</td><td rowspan=1 colspan=1>88.9</td><td rowspan=1 colspan=1>76.5-95.2%</td><td rowspan=1 colspan=1>64/70</td><td rowspan=1 colspan=1>91.4</td><td rowspan=1 colspan=1>82.5-96.0%</td><td rowspan=1 colspan=1>94/98</td><td rowspan=1 colspan=1>95.9</td><td rowspan=1 colspan=1>90.0-98.4%</td><td rowspan=1 colspan=1>91/104</td><td rowspan=1 colspan=1>87.5</td><td rowspan=1 colspan=1>79.8-92.5%</td></tr><tr><td rowspan=1 colspan=1>NDMe</td><td rowspan=1 colspan=1>0/1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>149/150</td><td rowspan=1 colspan=1>99.3</td><td rowspan=1 colspan=1>96.3-99.9%</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>291/291</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>98.7-100%</td></tr><tr><td rowspan=1 colspan=1>OXA- 48-like</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>92/92</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>96.0-100%</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>195/195</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>98.1-100%</td></tr><tr><td rowspan=1 colspan=1>VIMf</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>151/151</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>97.5-100%</td><td rowspan=1 colspan=1>1/1</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>289/290</td><td rowspan=1 colspan=1>99.7</td><td rowspan=1 colspan=1>98.1-99.9%</td></tr></table>

a. Performance in this summary table is calculated when any applicable organism is detected in the sample. b. CTX-M was detected in the single FN BAL and $1 / 2 \ \mathrm { F N }$ sputum specimens using an additional molecular method. The single FP sputum specimen was negative for CTX-M when tested during discrepancy investigation. None of the applicable isolates identified by the FilmArray or qRefCx from these specimens had evidence of ESBL activity or CTX-M presence. c. KPC was detected in the single FP BAL specimen using an additional molecular method; the isolate recovered from this specimen (A. baumannii) exhibited carbapenem resistance but did not carry KPC. d. Evidence of mecA/C and/or SCCmec cassette genetic elements was found in all five FN BAL and all four FN sputum specimens by an additional molecular method; three of these also had a MRSA isolate recovered via qRefCx or SOC culture. Evidence of mecA $/ C$ and/or SCCmec cassette genetic elements was found in 5/6 FP BAL and all $1 3 \mathrm { F P }$ sputum specimens; nine had a MRSA isolate recovered via qRefCx or SOC culture, and nine additional specimens had evidence of mecA/C and/or SCCmec cassette genetic elements by an additional molecular method. e. NDM was detected in the single FN BAL specimen using an additional molecular method; P. aeruginosa was recovered from the specimen and was resistant to carbapenems but carried only KPC. The single FP BAL specimen was negative for NDM when tested with additional molecular methods during discrepancy investigation. f. The single FP sputum specimen was negative for VIM when tested with additional molecular methods during discrepancy investigation.

qRefCx isolated one or more applicable gram-negative bacteria from 127 of the 156 BAL specimens and 230 of the 295 sputum specimens that received a FilmArray Pneumonia Panel Detected result for an applicable gram-negative bacterium for CTX-M, IMP, KPC, NDM, and VIM. The method used to assess correlation of the CTX-M, IMP, KPC, NDM, and VIM results (Table 54 and Table 55) reported in the specimen by the FilmArray Pneumonia Panel to identification of the gene in the cultured isolates from that particular specimen was one conventional PCR assay followed by bidirectional sequencing, performed directly on the isolate.

Table 54: CTX-M, IMP, KPC, NDM, and VIM Performance Table (PCR/seq on cultured isolate(s) from BAL specimens)   

<table><tr><td rowspan="2">Applicable Bacteria Result</td><td colspan="3"></td><td colspan="3">BAL</td><td colspan="2"></td><td colspan="2">OXA-48-like</td><td colspan="2">VIM</td><td colspan="2">Overall (any resistance</td></tr><tr><td>N</td><td>CTX-M PPA</td><td>NPA PPA</td><td>IMP NPA</td><td>PPA</td><td>KPC</td><td>NPA PPA</td><td>NDM</td><td>PPA</td><td>NPA</td><td>PPA</td><td>NPA</td><td>PPA</td><td>gene) NPA</td></tr><tr><td>Overall (any</td><td>4/4</td><td>121/123</td><td>0/0</td><td>127/127</td><td>1/1</td><td>124/126</td><td>0/0</td><td>NPA 127/127 (100%)</td><td>0/0</td><td>127/127</td><td>0/0</td><td>127/127</td><td>5/5 (100%)</td><td>118/122 (96.7%)</td></tr><tr><td>applicable bacteria Detected)a Acinetobacter</td><td>127 (100%)</td><td>(98.4%)</td><td>(-)</td><td>(100%)</td><td>(100%)</td><td>(98.4%)</td><td>(-)</td><td></td><td>(-)</td><td>(100%)</td><td>(-)</td><td>(100%)</td><td>56.6- 100%]</td><td>91.- 98.7%]</td></tr><tr><td>calcoaceticus- baumannii complex Enterobacter cloacae</td><td>0 0/0</td><td>- 9/9</td><td>- 0/0</td><td>- 9/9</td><td>0/0</td><td>- 9/9</td><td>- 0/0</td><td></td><td>-</td><td>-</td><td>-</td><td>-</td><td>-</td><td>- 9/9</td></tr><tr><td>complex</td><td>9 (-) 4/4</td><td>(100%) 8/8</td><td>(-) 0/0</td><td>(100%) 12/12</td><td>(-) 0/0</td><td>(100%) 12/12</td><td>(-) 0/0</td><td>9/9 (100%)</td><td>0/0 (-</td><td>9/9 (100%) 12/12</td><td>0/0 () 0/0</td><td>9/9 (100%) 12/12</td><td>0/0 (-) 4/4</td><td>(100%) 8/8</td></tr><tr><td>Escherichia coli</td><td>12 (100%) 7</td><td>(100%)</td><td>()</td><td>(100%)</td><td>(-)</td><td>(100%)</td><td>(-)</td><td>12/12 (100%)</td><td>00 </td><td>(100%)</td><td>(-</td><td>(100%)</td><td>(100%) 0/0</td><td>(100%) 7/7</td></tr><tr><td>Klebsiella aerogenes</td><td>0/0 (-)</td><td>7/7 (100%)</td><td>0/0 (-)</td><td>7/7 (100%)</td><td>0/0 (-)</td><td>7/7 (100%)</td><td>0/0 (-)</td><td>7/7 (100%)</td><td>0/0 (-)</td><td>7/7 (100%)</td><td>0/0 (-)</td><td>7/7 (100%)</td><td>()</td><td>(100%) 2/2</td></tr><tr><td>Klebsiella oxytoca</td><td>0/0 (-)</td><td>2/2 (100%)</td><td>0/0 (-)</td><td>2/2 (100%)</td><td>0/0 (-)</td><td>2/2 (100%)</td><td>0/0 (-)</td><td>2/2 (100%)</td><td>0/0 (-)</td><td>2/2 (100%)</td><td>0/0 (-)</td><td>2/2 (100%)</td><td>0/0 (-)</td><td>(100%) 14/14</td></tr><tr><td>Klebsiella pneumoniae group</td><td>0/0 (-)</td><td>14/14 (100%)</td><td>0/0 (-)</td><td>14/14 (100%)</td><td>0/0 (-)</td><td>14/14 (100%)</td><td>0/0 ()</td><td>14/14 (100%)</td><td>0/0 (</td><td>14/14 (100%)</td><td>0/0 (-)</td><td>14/14 (100%)</td><td>0/0 (-)</td><td>(100%)</td></tr><tr><td>Proteus spp.</td><td>0/0 (-)</td><td>6/6 (100%)</td><td>0/0 (-)</td><td>6/6 (100%)</td><td>0/0 (-)</td><td>6/6 (100%)</td><td>0/0 ()</td><td>6/6 (100%)</td><td>0/0 (-</td><td>6/6 (100%)</td><td>0/0 ()</td><td>6/6 (100%)</td><td>0/0 ()</td><td>6/6 (100%)</td></tr><tr><td>Pseudomonas aeruginosa</td><td>0/0 (-)</td><td>42/43 (97.7%)</td><td>0/0 (-)</td><td>43/43 (100%)</td><td>0/0 (-)</td><td>43/43 (100%)</td><td>0/0 (-)</td><td>43/43 (100%)</td><td>0/0 (-)</td><td>43/43 (100%)</td><td>0/0 (-)</td><td>43/43 (100%)</td><td>0/0 (-)</td><td>42/43 (97.7%)</td></tr><tr><td>Serratia marcescens</td><td>0/0</td><td>6/6</td><td>0/0</td><td>6/6</td><td>0/0</td><td>6/6</td><td>0/0</td><td></td><td>0/0</td><td>6/6</td><td>0/0</td><td>6/6</td><td>0/0</td><td>6/6</td></tr><tr><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>(100%)</td></tr><tr><td></td><td></td><td>(100%)</td><td>(-)</td><td>(100%)</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr><tr><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr><tr><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>6/6 (100%)</td><td></td><td></td><td></td><td></td><td></td><td></td></tr><tr><td></td><td></td><td></td><td></td><td></td><td>(-)</td><td>(100%)</td><td>(-)</td><td></td><td>(-)</td><td>(100%)</td><td>(-)</td><td>(100%)</td><td>(-)</td><td></td></tr><tr><td></td><td>(-)</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr><tr><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>1/1</td><td>24/27</td></tr><tr><td>Polymicrobial specimens</td><td>0/0</td><td>27/28b</td><td>0/0</td><td>28/28</td><td>1/1c</td><td>25/27d</td><td>0/0</td><td>28/28 (100%)</td><td>0/0</td><td>28/28</td><td>0/0</td><td>28/28</td><td></td><td></td></tr><tr><td></td><td>28</td><td></td><td>(-)</td><td>(100%)</td><td></td><td>(92.6%)</td><td>(-)</td><td></td><td>(-)</td><td>(100%)</td><td>(-)</td><td>(100%)</td><td>(100%)</td><td>(88.9%)</td></tr><tr><td></td><td>(-)</td><td>(96.4%)</td><td></td><td></td><td>(100%)</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr><tr><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr><tr><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr><tr><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr></table>

a. An additional nine specimens had no applicable bacteria detected by FilmArray, but had one or more applicable bacteria isolated by qRefCx; no resistance marker were identified in the cultured isolate(s) by PCR/seq from these specimens b. One specimen $E$ . cloacae complex and $K$ . pneumoniae group detected by FilmArray (E. cloacae complex isolated by qRefCx; CTX-M was not identified in this isolate by PCR/seq) c. E. cloacae complex and $P$ . aeruginosa detected by FilmArray and isolated by qRefCx (KPC identified from the $E$ . cloacae isolate by PCR/seq) d. One specimen $A$ . calcoaceticus-baumannii complex and egroup detected by FilmArray (A. calcoaceticus-baumannii complex isolated by qRefCx; KPC was not identified in this isolate by $\mathrm { P C R / s e q }$ ); one specimen Proteus spp. and $P$ . aeruginosa detected by FilmArray (P. aeruginosa isolated by qRefCx; KPC was not identified in this isolated by PCR/seq)

Table 55: CTX-M, IMP, KPC, NDM, and VIM Performance Table (PCR/seq on cultured isolate(s) from sputum specimens)   

<table><tr><td rowspan=1 colspan=16>Sputum</td></tr><tr><td rowspan=2 colspan=1>ApplicableBacteriaResult</td><td rowspan=2 colspan=1>N</td><td rowspan=1 colspan=2>CTX-M</td><td rowspan=1 colspan=2>IMP</td><td rowspan=1 colspan=2>KPC</td><td rowspan=1 colspan=2>NDM</td><td rowspan=1 colspan=2>OXA-48-like</td><td rowspan=1 colspan=2>VIM</td><td rowspan=1 colspan=2>Overall(anyresistancegene)</td></tr><tr><td rowspan=1 colspan=1>PPA</td><td rowspan=1 colspan=1>NPA</td><td rowspan=1 colspan=1>PPA</td><td rowspan=1 colspan=1>NPA</td><td rowspan=1 colspan=1>PPA</td><td rowspan=1 colspan=1>NPA</td><td rowspan=1 colspan=1>PPA</td><td rowspan=1 colspan=1>NPA</td><td rowspan=1 colspan=1>PPA</td><td rowspan=1 colspan=1>NPA</td><td rowspan=1 colspan=1>PPA</td><td rowspan=1 colspan=1>NPA</td><td rowspan=1 colspan=1>PPA</td><td rowspan=1 colspan=1>NPA</td></tr><tr><td rowspan=2 colspan=1>Overall(any applicablebacteriaDetected)a</td><td rowspan=2 colspan=1>230</td><td rowspan=2 colspan=1>3/4(75.0%)(97.8%)</td><td rowspan=2 colspan=1>221/226(75.0%)(97.8%)</td><td rowspan=2 colspan=1>0/0(-)</td><td rowspan=2 colspan=1>230/230(100%)</td><td rowspan=2 colspan=1>6/7(85.7%)(99.6%)</td><td rowspan=2 colspan=1>222/223(85.7%)(99.6%)</td><td rowspan=2 colspan=1>0/0(-)</td><td rowspan=2 colspan=1>230/230(100%)</td><td rowspan=2 colspan=1>0/0(-)</td><td rowspan=2 colspan=1>230/230(100%)</td><td rowspan=2 colspan=1>1/1(100%)</td><td rowspan=2 colspan=1>229/229(100%)</td><td rowspan=2 colspan=1>9/11b(81.8%)(97.7%)[52.3-94.9%]</td><td rowspan=1 colspan=1>214/219</td></tr><tr><td rowspan=1 colspan=1>[94.8-99.0%]</td></tr><tr><td rowspan=1 colspan=1>Acinetobactercalcoaceticus-baumanniicomplex</td><td rowspan=1 colspan=1>5c</td><td rowspan=1 colspan=1>0/0(-)</td><td rowspan=1 colspan=1>5/5(100%)</td><td rowspan=1 colspan=1>0/0(-)</td><td rowspan=1 colspan=1>5/5(100%)</td><td rowspan=1 colspan=1>0/0(-)</td><td rowspan=1 colspan=1>5/5(100%)</td><td rowspan=1 colspan=1>0/0(-)</td><td rowspan=1 colspan=1>5/5(100%)</td><td rowspan=1 colspan=1>0/0(-)</td><td rowspan=1 colspan=1>5/5(100%)</td><td rowspan=1 colspan=1>0/0(-)</td><td rowspan=1 colspan=1>5/5(100%)</td><td rowspan=1 colspan=1>0/0(-)</td><td rowspan=1 colspan=1>5/5(100%)</td></tr><tr><td rowspan=1 colspan=1>Enterobactercloacaecomplex</td><td rowspan=1 colspan=1>7d</td><td rowspan=1 colspan=1>0/0(-)</td><td rowspan=1 colspan=1>7/7(100%)</td><td rowspan=1 colspan=1>0/0(-)</td><td rowspan=1 colspan=1>7/7(100%)</td><td rowspan=1 colspan=1>0/0(-)</td><td rowspan=1 colspan=1>7/7(100%)</td><td rowspan=1 colspan=1>0/0(-)</td><td rowspan=1 colspan=1>7/7(100%)</td><td rowspan=1 colspan=1>0/0(-)</td><td rowspan=1 colspan=1>7/7(100%)</td><td rowspan=1 colspan=1>0/0(-)</td><td rowspan=1 colspan=1>7/7(100%)</td><td rowspan=1 colspan=1>0/0(-)</td><td rowspan=1 colspan=1>7/7(100%)</td></tr><tr><td rowspan=1 colspan=1>Escherichiacoli</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>1/1(100%)</td><td rowspan=1 colspan=1>9/9(100%)</td><td rowspan=1 colspan=1>0/0(-)</td><td rowspan=1 colspan=1>10/10(100%)</td><td rowspan=1 colspan=1>0/0(-)</td><td rowspan=1 colspan=1>10/10(100%)</td><td rowspan=1 colspan=1>0/0(-)</td><td rowspan=1 colspan=1>10/10(100%)</td><td rowspan=1 colspan=1>0/0(-)</td><td rowspan=1 colspan=1>10/10(100%)</td><td rowspan=1 colspan=1>0/0(-)</td><td rowspan=1 colspan=1>10/10(100%)</td><td rowspan=1 colspan=1>1/1(100%)</td><td rowspan=1 colspan=1>9/9(100%)</td></tr><tr><td rowspan=1 colspan=1>Klebsiellaaerogenes</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0/0(-)</td><td rowspan=1 colspan=1>3/3(100%)</td><td rowspan=1 colspan=1>0/0(-)</td><td rowspan=1 colspan=1>3/3(100%)</td><td rowspan=1 colspan=1>0/0(-)</td><td rowspan=1 colspan=1>3/3(100%)</td><td rowspan=1 colspan=1>0/0(-)</td><td rowspan=1 colspan=1>3/3(100%)</td><td rowspan=1 colspan=1>0/0(-)</td><td rowspan=1 colspan=1>3/3(100%)</td><td rowspan=1 colspan=1>0/0(-)</td><td rowspan=1 colspan=1>3/3(100%)</td><td rowspan=1 colspan=1>0/0(-)</td><td rowspan=1 colspan=1>3/3(100%)</td></tr><tr><td rowspan=2 colspan=1>Klebsiellaoxytoca</td><td rowspan=2 colspan=1>4</td><td rowspan=2 colspan=1>0/0(-)</td><td rowspan=2 colspan=1>4/4(100%)</td><td rowspan=2 colspan=1>0/0(-)</td><td rowspan=2 colspan=1>4/4(100%)</td><td rowspan=2 colspan=1>0/0(-)</td><td rowspan=2 colspan=1>4/4(100%)</td><td rowspan=2 colspan=1>0/0(-)</td><td rowspan=2 colspan=1>4/4(100%)</td><td rowspan=2 colspan=1>0/0(-)</td><td rowspan=1 colspan=1>4/4</td><td rowspan=2 colspan=1>0/0(-)</td><td rowspan=2 colspan=1>4/4(100%)</td><td rowspan=2 colspan=1>0/0(-)</td><td rowspan=2 colspan=1>4/4(100%)</td></tr><tr><td rowspan=1 colspan=1>(100%)</td></tr><tr><td rowspan=1 colspan=1>Klebsiellapneumoniaegroup</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>1/1(100%)</td><td rowspan=1 colspan=1>20/20(100%)</td><td rowspan=1 colspan=1>0/0(-)</td><td rowspan=1 colspan=1>21/21(100%)</td><td rowspan=1 colspan=1>1/1(100%)</td><td rowspan=1 colspan=1>20/20(100%)</td><td rowspan=1 colspan=1>0/0(-)</td><td rowspan=1 colspan=1>21/21(100%)</td><td rowspan=1 colspan=1>0/0(-)</td><td rowspan=1 colspan=1>21/21(100%)</td><td rowspan=1 colspan=1>0/0(-)</td><td rowspan=1 colspan=1>21/21(100%)</td><td rowspan=1 colspan=1>2/2(100%)</td><td rowspan=1 colspan=1>19/19(100%)</td></tr><tr><td rowspan=1 colspan=1>Proteus spp.</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>0/0(-)</td><td rowspan=1 colspan=1>6/6(100%)</td><td rowspan=1 colspan=1>0/0(-)</td><td rowspan=1 colspan=1>6/6(100%)</td><td rowspan=1 colspan=1>0/0(-)</td><td rowspan=1 colspan=1>6/6(100%)</td><td rowspan=1 colspan=1>0/0(-)</td><td rowspan=1 colspan=1>6/6(100%)</td><td rowspan=1 colspan=1>0/0(-)</td><td rowspan=1 colspan=1>6/6(100%)</td><td rowspan=1 colspan=1>0/0(-)</td><td rowspan=1 colspan=1>6/6(100%)</td><td rowspan=1 colspan=1>0/0(-)</td><td rowspan=1 colspan=1>6/6(100%)</td></tr><tr><td rowspan=2 colspan=1>Pseudomonasaeruginosa</td><td rowspan=2 colspan=1>68</td><td rowspan=2 colspan=1>0/1(0%)</td><td rowspan=2 colspan=1>65/67(97.0%)</td><td rowspan=2 colspan=1>0/0(-)</td><td rowspan=2 colspan=1>68/68(100%)</td><td rowspan=1 colspan=1>0/1</td><td rowspan=1 colspan=1>67/67</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>68/68</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>68/68</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>68/68</td><td rowspan=1 colspan=1>0/2</td><td rowspan=1 colspan=1>64/66</td></tr><tr><td rowspan=1 colspan=1>(0%)</td><td rowspan=1 colspan=1>(100%)</td><td rowspan=1 colspan=1>(-)</td><td rowspan=1 colspan=1>(100%)</td><td rowspan=1 colspan=1>(-)</td><td rowspan=1 colspan=1>(100%)</td><td rowspan=1 colspan=1>(-)</td><td rowspan=1 colspan=1>(100%)</td><td rowspan=1 colspan=1>(0%)</td><td rowspan=1 colspan=1>(97.0%)</td></tr><tr><td rowspan=1 colspan=1>Serratiamarcescens</td><td rowspan=1 colspan=1>14e</td><td rowspan=1 colspan=1>0/0 (-)</td><td rowspan=1 colspan=1>14/14(100%)</td><td rowspan=1 colspan=1>1/1(100%)</td><td rowspan=1 colspan=1>13/13(100%)</td><td rowspan=1 colspan=1>0/0(-)</td><td rowspan=1 colspan=1>14/14(100%)</td><td rowspan=1 colspan=1>0/0(-)</td><td rowspan=1 colspan=1>14/14(100%)</td><td rowspan=1 colspan=1>0/0(-)</td><td rowspan=1 colspan=1>14/14(100%)</td><td rowspan=1 colspan=1>0/0(-)</td><td rowspan=1 colspan=1>14/14(100%)</td><td rowspan=1 colspan=1>1/1(100%)</td><td rowspan=1 colspan=1>13/13(100%)</td></tr><tr><td rowspan=1 colspan=1>Polymicrobialspecimens</td><td rowspan=1 colspan=1>92</td><td rowspan=1 colspan=1>1/1f(100%)</td><td rowspan=1 colspan=1>88/91(96.7%)</td><td rowspan=1 colspan=1>0/0(-)</td><td rowspan=1 colspan=1>92/92(100%)</td><td rowspan=1 colspan=1>4/4g(100%)</td><td rowspan=1 colspan=1>87/88)(98.9%)</td><td rowspan=1 colspan=1>0/0(-)</td><td rowspan=1 colspan=1>92/92(100%)</td><td rowspan=1 colspan=1>0/0(-)</td><td rowspan=1 colspan=1>92/92(100%)</td><td rowspan=1 colspan=1>1/1h(100%)</td><td rowspan=1 colspan=1>91/91(100%)</td><td rowspan=1 colspan=1>5/5(100%)</td><td rowspan=1 colspan=1>84/87(96.6%)</td></tr></table>

a. An additional 19 specimens had no applicable bacteria detected by FilmArray, but had one or more applicable bacteria isolated by qRefCx; CTX-M was identified by PCR/seq in one specimen $E .$ . coli isolated by qRefCx), but no other resistance markers were identified in the cultured isolate(s) by PCR/seq from these specimens b. One specimen had presence of dual AMR genes (KPC and VIM) c. An A. calcoaceticus-baumannii isolate was not recovered by qRefCx for one specimen d. An $E$ . cloacae isolate was not recovered by qRefCx for one specimen e. A $S .$ . marcescens isolate was not recovered by qRefCx for one specimen f. $E$ . coli and $P$ . aeruginosa detected by FilmArray and isolated by qRefCx (CTX-M identified in the $E$ . coli isolate by PCR/seq) g. One specimen $A$ . calcoaceticus-baumannii complex, $K .$ . pneumoniae group, Proteus spp., and $P$ . aeruginosa detected by FilmArray (K. pneumoniae group and $P$ . aeruginosa isolated by qRefCx; CTX-M was not identified in either of these isolates by PCR/seq); one specimen E. coli, K. pneumoniae group, and $P$ . aeruginosa detected by FilmArray (E. coli and $P$ . aeruginosa isolated by qRefCx; CTX-M was not identified in either of these isolated by PCR/seq); one specimen Proteus spp. and $P$ . aeruginosa detected by FilmArray $P$ . aeruginosa isolated by qRefCx; CTX-M was not identified in this isolate by PCR/seq)

h. One specimen $E$ . coli and $e$ group detected by FilmArray and isolated by qRefCx (KPC identified in the $K .$ . pneumoniae isolate by PCR/seq); one specimen $P$ . aeruginosa and S. marcescens detected by FilmArray and isolated by qRefCx (KPC identified in the S. marcescens isolate by PCR/seq); one specimen A. calcoaceticus-baumannii complex, K. aerogenes, K. pneumoniae group, and $P$ . aeruginosa detected by FilmArray (A. calcoaceticus-baumannii, K. pneumoniae, and P. aeruginosa isolated by qRefCx; KPC identified in the $K$ . pneumoniae isolate by PCR/seq); one specimen A. calcoaceticus-baumannii complex, $K .$ . pneumoniae group, Proteus spp., and $P$ . aeruginosa detected by FilmArray $K .$ pneumoniae and $P$ . aeruginosa isolated by qRefCx; KPC identified in the K. pneumoniae isolate by PCR/seq)   
i. One specimen K. pneumoniae group and $P$ . aeruginosa detected by FilmArray (P. aeruginosa isolated by qRefCx; KPC was not identified in this isolate by PCR/seq)   
j. One specimen A. calcoaceticus-baumannii complex, K. aerogenes, K. pneumoniae group, and $P$ . aeruginosa detected by FilmArray (A. calcoaceticus-baumannii, K. pneumoniae, and P. aeruginosa isolated by qRefCx; VIM identified in the $P$ . aeruginosa isolate by PCR/seq)

qRefCx isolated one or more applicable gram-negative bacteria from 79 of the 94 BAL specimens and 131 of the 196 sputum specimens that received a FilmArray Pneumonia Panel Detected result for an applicable gram-negative bacterium for OXA-48-like. The method used to assess correlation of the OXA-48-like results (Table 56 and Table 57) reported in the specimen by the FilmArray Pneumonia Panel to identification of the gene in the cultured isolates from that particular specimen was one conventional PCR assay followed by bidirectional sequencing, performed directly on the isolate.

Table 56: OXA-48-like Performance Table (PCR/seq on cultured isolate(s) from BAL specimens)   

<table><tr><td rowspan=1 colspan=7>BAL</td></tr><tr><td rowspan=2 colspan=1>Applicable Bacteria Result</td><td rowspan=1 colspan=3>Positive Percent Agreement</td><td rowspan=1 colspan=3>Negative Percent Agreement</td></tr><tr><td rowspan=1 colspan=1>TP/(TP + FN)</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95%CI</td><td rowspan=1 colspan=1>TN/(TN + FP)</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95%CI</td></tr><tr><td rowspan=1 colspan=1>Overall(any applicable bacteria Detected)</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>79/79</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>95.4-100%</td></tr><tr><td rowspan=1 colspan=1>Enterobacter cloacae complex</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>72.2-100%</td></tr><tr><td rowspan=1 colspan=1>Escherichia coli</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>13/13</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>77.2-100%</td></tr><tr><td rowspan=1 colspan=1>Klebsiella aerogenes</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>7/7</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>64.6-100%</td></tr><tr><td rowspan=1 colspan=1>Klebsiella oxytoca</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>43.9-100%</td></tr><tr><td rowspan=1 colspan=1>Klebsiella pneumoniae group</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>15/15</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>79.6-100%</td></tr><tr><td rowspan=1 colspan=1>Proteus spp.</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>61.0-100%</td></tr><tr><td rowspan=1 colspan=1>Serratia marcescens</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>8/8</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>67.6-100%</td></tr><tr><td rowspan=1 colspan=1>Polymicrobial specimens</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>17/17</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>81.6-100%</td></tr></table>

Table 57: OXA-48-like Performance Table (PCR/seq on cultured isolate(s) from sputum specimens)   

<table><tr><td rowspan=1 colspan=7>Sputum</td></tr><tr><td rowspan=2 colspan=1>Applicable Bacteria Result</td><td rowspan=1 colspan=3>Positive Percent Agreement</td><td rowspan=1 colspan=3>Negative Percent Agreement</td></tr><tr><td rowspan=1 colspan=1>TP/(TP + FN)</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95%CI</td><td rowspan=1 colspan=1>TN/(TN + FP)</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95%CI</td></tr><tr><td rowspan=1 colspan=1>Overall(any applicable bacteria Detected)</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>131/131</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>97.2-100%</td></tr><tr><td rowspan=1 colspan=1>Enterobacter cloacae complex</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>9/9</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>70.1-100%</td></tr><tr><td rowspan=1 colspan=1>Escherichia coli</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>17/17</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>81.6-100%</td></tr><tr><td rowspan=1 colspan=1>Klebsiella aerogenes</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>4/4</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>51.0-100%</td></tr><tr><td rowspan=1 colspan=1>Klebsiella oxytoca</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>5/5</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>56.6-100%</td></tr><tr><td rowspan=1 colspan=1>Klebsiella pneumoniae group</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>25/25</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>86.7-100%</td></tr><tr><td rowspan=1 colspan=1>Proteus spp.</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>9/9</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>70.1-100%</td></tr><tr><td rowspan=1 colspan=1>Serratia marcescens</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>25/25</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>86.7-100%</td></tr><tr><td rowspan=1 colspan=1>Polymicrobial specimens</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>37/37</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>90.6-100%</td></tr></table>

qRefCx isolated S. aureus from 75 of 116 BAL specimens and 154 of 204 sputum specimens that received a FilmArray Pneumonia Panel Staphylococcus aureus Detected result. The method used to assess correlation of the mecA/C and MREJ results (Table 58 and Table 59) reported in the specimen by the FilmArray Pneumonia Panel to identification of the gene in the cultured isolates from that particular specimen was one conventional PCR assay followed by bidirectional sequencing, performed directly on the isolate.

Table 58: mecA/C and MREJ 3x3 Performance Table (qRefCx & PCR/seq on cultured isolate(s) from BAL specimens)   

<table><tr><td colspan="6" rowspan="1">BAL</td></tr><tr><td colspan="2" rowspan="2">S. aureusmecA/C and MREJ</td><td colspan="4" rowspan="1">qRefCx: S. aureus PCR/seq: mecA/C</td></tr><tr><td colspan="1" rowspan="1">Org+ / Res+</td><td colspan="1" rowspan="1">Org+ / Res-</td><td colspan="1" rowspan="1">Org -</td><td colspan="1" rowspan="1">Total</td></tr><tr><td colspan="1" rowspan="4">FilmArrayResult</td><td colspan="1" rowspan="1">Org+ /Res+</td><td colspan="1" rowspan="1">19</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">25</td><td colspan="1" rowspan="1">46</td></tr><tr><td colspan="1" rowspan="1">Org+ / Res-</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">24</td><td colspan="1" rowspan="1">45</td><td colspan="1" rowspan="1">70</td></tr><tr><td colspan="1" rowspan="1">Org -</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">729</td><td colspan="1" rowspan="1">730</td></tr><tr><td colspan="1" rowspan="1">Total</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">27</td><td colspan="1" rowspan="1">799</td><td colspan="1" rowspan="1">846</td></tr><tr><td colspan="3" rowspan="1">Performance</td><td colspan="1" rowspan="1">Agreement</td><td colspan="1" rowspan="1">%</td><td colspan="1" rowspan="1">95%CI</td></tr><tr><td colspan="3" rowspan="1">Org+ /Res+</td><td colspan="1" rowspan="1">19/20</td><td colspan="1" rowspan="1">95.0%</td><td colspan="1" rowspan="1">76.4-99.1%</td></tr><tr><td colspan="3" rowspan="1">Org+ / Res-</td><td colspan="1" rowspan="1">24/27</td><td colspan="1" rowspan="1">88.9%</td><td colspan="1" rowspan="1">71.9-96.1%</td></tr><tr><td colspan="3" rowspan="1">Org -</td><td colspan="1" rowspan="1">729/799</td><td colspan="1" rowspan="1">91.2%</td><td colspan="1" rowspan="1">89.1-93.0%</td></tr><tr><td colspan="3" rowspan="1">Interpretation</td><td colspan="1" rowspan="1">PPA</td><td colspan="1" rowspan="1">NPA</td><td colspan="1" rowspan="1">Prevalence</td></tr><tr><td colspan="3" rowspan="2">MRSA</td><td colspan="1" rowspan="1">19/20</td><td colspan="1" rowspan="1">799/826</td><td colspan="1" rowspan="1">46/846</td></tr><tr><td colspan="1" rowspan="1">(95.0%)</td><td colspan="1" rowspan="1">(96.7%)</td><td colspan="1" rowspan="1">(5.4%)</td></tr><tr><td colspan="3" rowspan="1">MSSA</td><td colspan="1" rowspan="1">24/27</td><td colspan="1" rowspan="1">773/819</td><td colspan="1" rowspan="1">70/846</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">(88.9%)</td><td colspan="1" rowspan="1">(94.4%)</td><td colspan="1" rowspan="1">(8.3%)</td></tr><tr><td colspan="1" rowspan="2">S. aureus</td><td colspan="1" rowspan="1">46/47</td><td colspan="1" rowspan="1">729/799</td><td colspan="1" rowspan="1">116/846</td></tr><tr><td colspan="1" rowspan="1">(97.9%)</td><td colspan="1" rowspan="1">(91.2%)</td><td colspan="1" rowspan="1">(13.7%)</td></tr></table>

Table 59: mecA/C and MREJ 3x3 Performance Table (qRefCx & PCR/seq on cultured isolate(s) from sputum specimens)   

<table><tr><td rowspan=1 colspan=6>Sputum</td></tr><tr><td rowspan=2 colspan=2>S. aureusmecA/C and MREJ</td><td rowspan=1 colspan=4>qRefCx: S. aureus PCR/seq: mecA/C</td></tr><tr><td rowspan=1 colspan=1>Org+ / Res+</td><td rowspan=1 colspan=1>Org+ / Res-</td><td rowspan=1 colspan=1>Org -</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=4 colspan=1>FilmArrayResult</td><td rowspan=1 colspan=1>Org+ / Res+</td><td rowspan=1 colspan=1>58</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>45</td><td rowspan=1 colspan=1>107</td></tr><tr><td rowspan=1 colspan=1>Org+ / Res-</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>49</td><td rowspan=1 colspan=1>48</td><td rowspan=1 colspan=1>97</td></tr><tr><td rowspan=1 colspan=1>Org -</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>631</td><td rowspan=1 colspan=1>632</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>58</td><td rowspan=1 colspan=1>54</td><td rowspan=1 colspan=1>724</td><td rowspan=1 colspan=1>836</td></tr><tr><td rowspan=1 colspan=3>Performance</td><td rowspan=1 colspan=1>Agreement</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95%CI</td></tr><tr><td rowspan=1 colspan=3>Org+ /Res+</td><td rowspan=1 colspan=1>58/58</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>93.8-100%</td></tr><tr><td rowspan=1 colspan=3>Org+ / Res-</td><td rowspan=1 colspan=1>49/54</td><td rowspan=1 colspan=1>90.7%</td><td rowspan=1 colspan=1>80.1-96.0%</td></tr><tr><td rowspan=1 colspan=3>Org -</td><td rowspan=1 colspan=1>631/724</td><td rowspan=1 colspan=1>87.2%</td><td rowspan=1 colspan=1>84.5-89.4%</td></tr><tr><td rowspan=1 colspan=3>Interpretation</td><td rowspan=1 colspan=1>PPA</td><td rowspan=1 colspan=1>NPA</td><td rowspan=1 colspan=1>Prevalence</td></tr><tr><td rowspan=2 colspan=3>MRSA</td><td rowspan=1 colspan=1>58/58</td><td rowspan=1 colspan=1>729/778</td><td rowspan=1 colspan=1>107/836</td></tr><tr><td rowspan=1 colspan=1>(100%)</td><td rowspan=1 colspan=1>(93.7%)</td><td rowspan=1 colspan=1>(12.8%)</td></tr><tr><td rowspan=2 colspan=3>MSSA</td><td rowspan=1 colspan=1>49/54</td><td rowspan=1 colspan=1>734/782</td><td rowspan=1 colspan=1>97/836</td></tr><tr><td rowspan=1 colspan=1>(90.7%)</td><td rowspan=1 colspan=1>(93.9%)</td><td rowspan=1 colspan=1>(11.6%)</td></tr><tr><td rowspan=2 colspan=3>S. aureus</td><td rowspan=1 colspan=1>111/112</td><td rowspan=1 colspan=1>631/724</td><td rowspan=1 colspan=1>204/836</td></tr><tr><td rowspan=1 colspan=1>(99.1%)</td><td rowspan=1 colspan=1>(87.2%)</td><td rowspan=1 colspan=1>(24.4%)</td></tr></table>

FilmArray Pneumonia Panel CTX-M reporting was also compared to the standard phenotypic extended spectrum $\beta$ -lactamase (ESBL) activity testing methods performed in conjunction with qRefCx. Standard phenotypic ESBL activity was only reported for $E$ . coli and Klebsiella spp. by the central reference laboratory.

Of the 156 BAL specimens that received a FilmArray Pneumonia Panel Detected result for at least one applicable gram-negative bacterium on the panel, 53 specimens received a Detected result for $E$ . coli, $K$ . oxytoca, and/or $K$ . pneumoniae; qRefCx isolated E. coli, K. oxytoca, and/or $K .$ . pneumoniae from 43 of these specimens. Of the 295 sputum specimens that received a FilmArray Pneumonia Panel Detected result for at least one applicable gram-negative bacterium on the panel, 114 specimens received a Detected result for E. coli, K. oxytoca, and/or K. pneumoniae; qRefCx isolated E. coli, $K .$ . oxytoca, and/or $K$ . pneumoniae from 71 of these specimens. The correlation between FilmArray Pneumonia Panel reporting of CTX-M in a particular specimen as compared to the phenotypic AST results of isolates recovered from the same specimen are stratified by each applicable associated organism in Table 60 and Table 61.

Table 60: CTX-M Performance Table (comparison to phenotypic AST methods for BAL specimens)   

<table><tr><td rowspan=1 colspan=7>BAL</td></tr><tr><td rowspan=2 colspan=1>Applicable Bacteria Result</td><td rowspan=1 colspan=3>Positive Percent Agreement</td><td rowspan=1 colspan=3>Negative Percent Agreement</td></tr><tr><td rowspan=1 colspan=1>TP/(TP + FN)</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95%CI</td><td rowspan=1 colspan=1>TN/(TN + FP)</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95%CI</td></tr><tr><td rowspan=1 colspan=1>Overall(any applicable bacteria Detected)</td><td rowspan=1 colspan=1>4/5</td><td rowspan=1 colspan=1>80.0</td><td rowspan=1 colspan=1>37.6-96.4%</td><td rowspan=1 colspan=1>38/38</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>90.8-100%</td></tr><tr><td rowspan=1 colspan=1>Escherichia coli</td><td rowspan=1 colspan=1>4/4</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>51.0-100%</td><td rowspan=1 colspan=1>11/11</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>74.1-100%</td></tr><tr><td rowspan=1 colspan=1>Klebsiella oxytoca</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>5/5</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>56.6-100%</td></tr><tr><td rowspan=1 colspan=1>Klebsiella pneumoniae group</td><td rowspan=1 colspan=1>0/1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>17/17</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>81.6-100%</td></tr><tr><td rowspan=1 colspan=1>Polymicrobial specimens</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>5/5</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>56.6-100%</td></tr></table>

Table 61: CTX-M Performance Table (comparison to phenotypic AST methods for sputum specimens)   

<table><tr><td rowspan=1 colspan=7>Sputum</td></tr><tr><td rowspan=2 colspan=1>Applicable Bacteria Result</td><td rowspan=1 colspan=3>Positive Percent Agreement</td><td rowspan=1 colspan=3>Negative Percent Agreement</td></tr><tr><td rowspan=1 colspan=1>TP/(TP + FN)</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95%CI</td><td rowspan=1 colspan=1>TN/(TN + FP)</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95%CI</td></tr><tr><td rowspan=1 colspan=1>Overall(any applicable bacteria Detected)</td><td rowspan=1 colspan=1>4/7</td><td rowspan=1 colspan=1>57.1</td><td rowspan=1 colspan=1>25.1-84.2%</td><td rowspan=1 colspan=1>63/64</td><td rowspan=1 colspan=1>98.4</td><td rowspan=1 colspan=1>91.7-99.7%</td></tr><tr><td rowspan=1 colspan=1>Escherichia coli</td><td rowspan=1 colspan=1>2/3</td><td rowspan=1 colspan=1>66.7</td><td rowspan=1 colspan=1>20.8-93.9%</td><td rowspan=1 colspan=1>17/17</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>81.6-100%</td></tr><tr><td rowspan=1 colspan=1>Klebsiella oxytoca</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>9/9</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>70.1-100%</td></tr><tr><td rowspan=1 colspan=1>Klebsiella pneumoniae group</td><td rowspan=1 colspan=1>1/2</td><td rowspan=1 colspan=1>50.0</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>26/27</td><td rowspan=1 colspan=1>96.3</td><td rowspan=1 colspan=1>81.7-99.3%</td></tr><tr><td rowspan=1 colspan=1>Polymicrobial specimens</td><td rowspan=1 colspan=1>1/2a</td><td rowspan=1 colspan=1>50.0</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>11/11</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>74.1-100%</td></tr></table>

a. One specimen $E$ . coli and $K$ . oxytoca detected by FilmArray and isolated by qRefCx (ESBL activity identified in the $E$ . coli isolate by qRefCx AST); one specimen $E$ . coli and $K$ . pneumoniae group detected by FilmArray $E$ . coli isolated by qRefCx and ESBL activity identified by qRefCx AST).

The FilmArray Pneumonia Panel carbapenem resistance gene reporting was also compared to the standard phenotypic carbapenem susceptibility testing methods performed in conjunction with qRefCx. In accordance with current CLSI guidelines, standard phenotypic ertapenem susceptibility is not reported for A. calcoaceticus-baumannii complex; therefore, carbapenem susceptibility is only based on meropenem susceptibility for this organism. Resistance or intermediate-resistance to either ertapenem or meropenem constituted carbapenem resistance for this analysis. The correlation between FilmArray Pneumonia Panel reporting of the carbapenem resistance genes in a particular specimen as compared to the phenotypic AST results of isolates recovered from the same specimen are stratified by each applicable associated organism in Table 62 through Table 65.

Table 62: IMP, KPC, NDM, and VIM Performance Table (comparison to phenotypic AST methods for BAL specimens)   

<table><tr><td rowspan=1 colspan=17>BAL</td></tr><tr><td rowspan=2 colspan=1>ApplicableBacteria Result</td><td rowspan=2 colspan=1>N</td><td rowspan=1 colspan=2>IMP</td><td rowspan=1 colspan=2>KPC</td><td rowspan=1 colspan=2>NDM</td><td rowspan=1 colspan=5>OXA-48-like</td><td rowspan=1 colspan=2>VIM</td><td rowspan=1 colspan=2>Overall(anycarbapenemresistancegene)</td></tr><tr><td rowspan=1 colspan=1>PPA</td><td rowspan=1 colspan=1>NPA</td><td rowspan=1 colspan=1>PPA</td><td rowspan=1 colspan=1>NPA</td><td rowspan=1 colspan=1>PPA</td><td rowspan=1 colspan=1>NPA</td><td rowspan=1 colspan=4>PPA</td><td rowspan=1 colspan=1>NPA</td><td rowspan=1 colspan=1>PPA</td><td rowspan=1 colspan=1>NPA</td><td rowspan=1 colspan=1>PPA</td><td rowspan=1 colspan=1>NPA</td></tr><tr><td rowspan=1 colspan=1>Overall(any applicablebacteria Detected)</td><td rowspan=1 colspan=1>126a</td><td rowspan=1 colspan=1>0/17(0%)</td><td rowspan=1 colspan=1>109/109 (100%)</td><td rowspan=1 colspan=1>3/17(17.6%)</td><td rowspan=1 colspan=1>109/109 (100%)</td><td rowspan=1 colspan=1>0/17(0%)</td><td rowspan=1 colspan=1>109/109(100%)</td><td rowspan=1 colspan=4>0/17((0%)</td><td rowspan=1 colspan=1>109/109(100%)</td><td rowspan=1 colspan=1>0/17((0%)</td><td rowspan=1 colspan=1>109/109(100%)</td><td rowspan=1 colspan=1>3/17 (17.6%)[6.2-41.0%]</td><td rowspan=1 colspan=1>109/109(100%)6.600%]</td></tr><tr><td rowspan=1 colspan=1>Acinetobactercalcoaceticus-baumanniicomplex</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=4>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=2 colspan=1>Enterobactercloacae complex</td><td rowspan=2 colspan=1>9</td><td rowspan=2 colspan=1>0/0(-)</td><td rowspan=2 colspan=1>9/9(100%)</td><td rowspan=2 colspan=1>0/0(-)</td><td rowspan=2 colspan=1>9/9(100%)</td><td rowspan=2 colspan=1>0/0(-)</td><td rowspan=1 colspan=1>9/9</td><td rowspan=1 colspan=4>0/0</td><td rowspan=1 colspan=1>9/9</td><td rowspan=2 colspan=1>0/0(-)</td><td rowspan=2 colspan=1>9/9(100%)</td><td rowspan=2 colspan=1>0/0(-)</td><td rowspan=2 colspan=1>9/9(100%)</td></tr><tr><td rowspan=1 colspan=1>(100%)</td><td rowspan=1 colspan=4>(-)</td><td rowspan=1 colspan=1>(100%)</td></tr><tr><td rowspan=1 colspan=1>Escherichia coli</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>0/0(-)</td><td rowspan=1 colspan=1>12/12(100%)</td><td rowspan=1 colspan=1>0/0(-)</td><td rowspan=1 colspan=1>12/12(100%)</td><td rowspan=1 colspan=1>0/0(-)</td><td rowspan=1 colspan=1>12/12(100%)</td><td rowspan=1 colspan=4>0/0(-)</td><td rowspan=1 colspan=1>12/12(100%)</td><td rowspan=1 colspan=1>0/0(-)</td><td rowspan=1 colspan=1>12/12(100%)</td><td rowspan=1 colspan=1>0/0(-)</td><td rowspan=1 colspan=1>12/12(100%)</td></tr><tr><td rowspan=1 colspan=1>Enterobacteraerogenes</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>0/0(-)</td><td rowspan=1 colspan=1>7/7(100%)</td><td rowspan=1 colspan=1>0/0(-)</td><td rowspan=1 colspan=1>7/7(100%)</td><td rowspan=1 colspan=1>0/0(-)</td><td rowspan=1 colspan=1>7/7(100%)</td><td rowspan=1 colspan=4>0/0(-)</td><td rowspan=1 colspan=1>7/7(100%)</td><td rowspan=1 colspan=1>0/0(-)</td><td rowspan=1 colspan=1>7/7(100%)</td><td rowspan=1 colspan=1>0/0(-)</td><td rowspan=1 colspan=1>7/7(100%)</td></tr><tr><td rowspan=2 colspan=1>Klebsiella oxytoca</td><td rowspan=2 colspan=1>2</td><td rowspan=2 colspan=1>0/0(-)</td><td rowspan=2 colspan=1>2/2(100%)</td><td rowspan=2 colspan=1>0/0(-)</td><td rowspan=2 colspan=1>2/2(100%)</td><td rowspan=2 colspan=1>0/0(-)</td><td rowspan=2 colspan=1>2/2(100%)</td><td rowspan=2 colspan=4>0/0(-)</td><td rowspan=2 colspan=1>2/2(100%)</td><td rowspan=2 colspan=1>0/0(-)</td><td rowspan=2 colspan=1>2/2(100%)</td><td rowspan=2 colspan=1>0/0(-)</td><td rowspan=2 colspan=1>2/2(100%)</td></tr><tr><td rowspan=1 colspan=1></td></tr><tr><td rowspan=2 colspan=1>Klebsiellapneumoniae group</td><td rowspan=2 colspan=1>14</td><td rowspan=2 colspan=1>0/0(-)</td><td rowspan=2 colspan=1>14/14(100%)</td><td rowspan=2 colspan=1>0/0(-)</td><td rowspan=1 colspan=1>14/14</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>14/14</td><td rowspan=2 colspan=4>0/0(-)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>14/14</td><td rowspan=2 colspan=1>0/0(-)</td><td rowspan=1 colspan=1>14/14</td><td rowspan=1 colspan=1>0/0</td></tr><tr><td rowspan=1 colspan=1>(100%)</td><td rowspan=1 colspan=1>(-)</td><td rowspan=1 colspan=1>(100%)</td><td rowspan=1 colspan=1>(100%)</td><td></td><td rowspan=1 colspan=1>(100%)</td><td rowspan=1 colspan=1>(-)</td><td rowspan=1 colspan=1>(100%)</td></tr><tr><td rowspan=1 colspan=1>Proteus spp.</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>0/0(-)</td><td rowspan=1 colspan=1>6/6(100%)</td><td rowspan=1 colspan=1>0/0(-)</td><td rowspan=1 colspan=1>6/6(100%)</td><td rowspan=1 colspan=1>0/0(-)</td><td rowspan=1 colspan=1>6/6(100%)</td><td rowspan=1 colspan=4>0/0(-)</td><td rowspan=1 colspan=1>6/6(100%)</td><td rowspan=1 colspan=1>0/0(-)</td><td rowspan=1 colspan=1>6/6(100%)</td><td rowspan=1 colspan=1>0/0(-)</td><td rowspan=1 colspan=1>6/6(100%)</td></tr><tr><td rowspan=2 colspan=1>Pseudomonasaeruginosa</td><td rowspan=2 colspan=1>42</td><td rowspan=2 colspan=1>0/12(0%)</td><td rowspan=2 colspan=1>30/30(100%)</td><td rowspan=2 colspan=1>0/12(0%)</td><td rowspan=2 colspan=1>30/30(100%)</td><td rowspan=2 colspan=1>0/12(0%)</td><td rowspan=2 colspan=1>30/30(100%)</td><td rowspan=2 colspan=4>0/12(0%)</td><td rowspan=2 colspan=1>30/30(100%)</td><td rowspan=2 colspan=1>0/12(0%)</td><td rowspan=2 colspan=1>30/30(100%)</td><td rowspan=2 colspan=1>0/12(0%)</td><td rowspan=2 colspan=1>30/30(100%)</td></tr><tr><td rowspan=1 colspan=1>)</td></tr><tr><td rowspan=2 colspan=1>Serratia marcescens</td><td rowspan=2 colspan=1>6</td><td rowspan=2 colspan=1>0/0(-)</td><td rowspan=2 colspan=1>6/6(100%)</td><td rowspan=2 colspan=1>0/0(-)</td><td rowspan=2 colspan=1>6/6(100%)</td><td rowspan=2 colspan=1>0/0(-)</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=3>0/0</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>6/6</td><td rowspan=2 colspan=1>0/0(-)</td><td rowspan=2 colspan=1>6/6(100%)</td><td rowspan=2 colspan=1>0/0(-)</td><td rowspan=2 colspan=1>6/6(100%)</td></tr><tr><td rowspan=1 colspan=1>(100%)</td><td rowspan=1 colspan=4>()</td><td rowspan=1 colspan=1>(100%)</td></tr><tr><td rowspan=2 colspan=1>Polymicrobialspecimens</td><td rowspan=2 colspan=1>28</td><td rowspan=2 colspan=1>0/5(0%)</td><td rowspan=2 colspan=1>23/23(100%)</td><td rowspan=2 colspan=1>3/5(60.0%)</td><td rowspan=2 colspan=1>23/23(100%)</td><td rowspan=2 colspan=1>0/5(0%)</td><td rowspan=1 colspan=1>23/23</td><td rowspan=2 colspan=4>0/5(0%)</td><td rowspan=1 colspan=1>23/23</td><td rowspan=1 colspan=1>0/5</td><td rowspan=1 colspan=1>23/23</td><td rowspan=2 colspan=1>3/5b(60.0%)</td><td rowspan=2 colspan=1>23/23(100%)</td></tr><tr><td rowspan=1 colspan=1>(100%)</td><td rowspan=1 colspan=1>(100%)</td><td rowspan=1 colspan=1>(0%)</td><td rowspan=1 colspan=1>(100%)</td></tr></table>

a. The isolate recovered from one specimen $P$ . aeruginosa) did not yield valid AST results on the VITEK instrument b. Of the three specimens that were concordant: one specimen A. calcoaceticus-baumannii complex and $K$ . pneumoniae group detected by FilmArray (A. calcoaceticus-baumannii isolated by qRefCx, carbapenem resistance identified by qRefCx AST, and KPC not identified in the isolate by $\mathrm { P C R / s e q }$ ); one specimen $E$ . cloacae complex and $P$ . aeruginosa detected by FilmArray and isolated by qRefCx (carbapenem resistance identified in the $E$ . cloacae isolate by qRefCx AST, and KPC identified in the $E$ . cloacae isolate by $\operatorname { P C R } / \sec _ { \mathrm { q } } )$ ; one specimen Proteus spp. and $P$ . aeruginosa detected by FilmArray $P$ . aeruginosa isolated by qRefCx, carbapenem resistance identified by qRefCx AST, and KPC not identified in the isolate by PCR/seq). Of the two specimens that were not concordant: one specimen A. calcoaceticus-baumannii complex and $P$ . aeruginosa detected by FilmArray (P. aeruginosa isolated by qRefCx, carbapenem resistance identified by qRefCx AST, and KPC not identified in the isolate by PCR/seq); one specimen E. cloacae complex and K. aerogenes detected by FilmArray (E. cloacae isolated by qRefCx, carbapenem resistance identified by qRefCx AST and KPC not identified in the isolate by PCR/seq).

Table 63: IMP, KPC, NDM, and VIM Performance Table (comparison to phenotypic AST methods for sputum specimens)   

<table><tr><td rowspan=1 colspan=15>Sputum</td></tr><tr><td rowspan=2 colspan=1>ApplicableBacteria Result</td><td rowspan=2 colspan=1>N</td><td rowspan=1 colspan=2>IMP</td><td rowspan=1 colspan=2>KPC</td><td rowspan=1 colspan=2>NDM</td><td rowspan=1 colspan=3>OXA-48-like</td><td rowspan=1 colspan=2>VIM</td><td rowspan=1 colspan=2>Overall(anycarbapenemresistancegene)</td></tr><tr><td rowspan=1 colspan=1>PPA</td><td rowspan=1 colspan=1>NPA</td><td rowspan=1 colspan=1>PPA</td><td rowspan=1 colspan=1>NPA</td><td rowspan=1 colspan=1>PPA</td><td rowspan=1 colspan=1>NPA</td><td rowspan=1 colspan=2>PPA</td><td rowspan=1 colspan=1>NPA</td><td rowspan=1 colspan=1>PPA</td><td rowspan=1 colspan=1>NPA</td><td rowspan=1 colspan=1>PPA</td><td rowspan=1 colspan=1>NPA</td></tr><tr><td rowspan=1 colspan=1>Overall(any applicablebacteria Detected)</td><td rowspan=1 colspan=1>229a</td><td rowspan=1 colspan=1>0/35(0%)</td><td rowspan=1 colspan=1>194/194(100%)</td><td rowspan=1 colspan=1>6/35(17.1%)</td><td rowspan=1 colspan=1>193/194(99.5%)</td><td rowspan=1 colspan=1>0/35(0%)</td><td rowspan=1 colspan=1>194/194(100%)</td><td rowspan=1 colspan=2>0/35(%)</td><td rowspan=1 colspan=1>194/194(100%)</td><td rowspan=1 colspan=1>1/35(2.9%)</td><td rowspan=1 colspan=1>194/194(100%)</td><td rowspan=1 colspan=1>6/35b (17.1%)[8.1-32.7%]</td><td rowspan=1 colspan=1>193/194(99.5%)[97.1-99.9%]</td></tr><tr><td rowspan=1 colspan=1>Acinetobactercalcoaceticus-baumanniicomplex</td><td rowspan=1 colspan=1>5c</td><td rowspan=1 colspan=1>0/2(0%)</td><td rowspan=1 colspan=1>3/3(100%)</td><td rowspan=1 colspan=1>0/2(0%)</td><td rowspan=1 colspan=1>3/3(100%)</td><td rowspan=1 colspan=1>0/2(0%)</td><td rowspan=1 colspan=1>3/3(100%)</td><td rowspan=1 colspan=2>0/2(0%)</td><td rowspan=1 colspan=1>3/3(100%)</td><td rowspan=1 colspan=1>0/2(0%)</td><td rowspan=1 colspan=1>3/3(100%)</td><td rowspan=1 colspan=1>0/2(0%)</td><td rowspan=1 colspan=1>3/3(100%)</td></tr><tr><td rowspan=1 colspan=1>Enterobactercloacae complex</td><td rowspan=1 colspan=1>7d</td><td rowspan=1 colspan=1>0/1(0%)</td><td rowspan=1 colspan=1>6/6(100%)</td><td rowspan=1 colspan=1>0/1(0%)</td><td rowspan=1 colspan=1>6/6(100%)</td><td rowspan=1 colspan=1>0/1(0%)</td><td rowspan=1 colspan=1>6/6(100%)</td><td rowspan=1 colspan=2>0/1(0%)</td><td rowspan=1 colspan=1>6/6(100%)</td><td rowspan=1 colspan=1>0/1(0%)</td><td rowspan=1 colspan=1>6/6(100%)</td><td rowspan=1 colspan=1>0/1(0%)</td><td rowspan=1 colspan=1>6/6(100%)</td></tr><tr><td rowspan=2 colspan=1>Escherichia coli</td><td rowspan=2 colspan=1>10</td><td rowspan=2 colspan=1>0/0(-)</td><td rowspan=2 colspan=1>10/10(100%)</td><td rowspan=2 colspan=1>0/0()</td><td rowspan=2 colspan=1>10/10(100%)</td><td rowspan=2 colspan=1>0/0(-)</td><td rowspan=2 colspan=1>10/10(100%)</td><td rowspan=2 colspan=2>0/0(-)</td><td rowspan=1 colspan=1>10/10</td><td rowspan=2 colspan=1>0/0(-)</td><td rowspan=2 colspan=1>10/10(100%)</td><td rowspan=2 colspan=1>0/0(-)</td><td rowspan=2 colspan=1>10/10(100%)</td></tr><tr><td rowspan=1 colspan=1>(100%)</td></tr><tr><td rowspan=2 colspan=1>Klebsiellaaerogenes</td><td rowspan=2 colspan=1>3</td><td rowspan=2 colspan=1>0/0(-)</td><td rowspan=2 colspan=1>3/3(100%)</td><td rowspan=2 colspan=1>0/0(-)</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=2>0/0</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>(100%)</td><td rowspan=1 colspan=1>(-)</td><td rowspan=1 colspan=1>(100%)</td><td rowspan=1 colspan=2>(-)</td><td rowspan=1 colspan=1>(100%)</td><td rowspan=1 colspan=1>(-)</td><td rowspan=1 colspan=1>(100%)</td><td rowspan=1 colspan=1>(-)</td><td rowspan=1 colspan=1>(100%)</td></tr><tr><td rowspan=2 colspan=1>Klebsiellaoxytoca</td><td rowspan=2 colspan=1>4</td><td rowspan=2 colspan=1>0/0(-)</td><td rowspan=2 colspan=1>4/4(100%)</td><td rowspan=2 colspan=1>0/0(-)</td><td rowspan=2 colspan=1>4/4(100%)</td><td rowspan=2 colspan=1>0/0(-)</td><td rowspan=1 colspan=1>4/4</td><td rowspan=1 colspan=2>0/0</td><td rowspan=1 colspan=1>4/4</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>4/4</td><td rowspan=2 colspan=1>0/0(-)</td><td rowspan=2 colspan=1>4/4(100%)</td></tr><tr><td rowspan=1 colspan=1>(100%)</td><td rowspan=1 colspan=2>(-)</td><td rowspan=1 colspan=1>(100%)</td><td rowspan=1 colspan=1>(-)</td><td rowspan=1 colspan=1>(100%)</td></tr><tr><td rowspan=2 colspan=1>Klebsiellapneumoniae group</td><td rowspan=2 colspan=1>21</td><td rowspan=2 colspan=1>0/2(0%)</td><td rowspan=2 colspan=1>19/19(100%)</td><td rowspan=2 colspan=1>1/2(50.0%)</td><td rowspan=1 colspan=1>19/19</td><td rowspan=1 colspan=1>0/2</td><td rowspan=1 colspan=1>19/19</td><td rowspan=1 colspan=2>0/2</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>19/19</td><td rowspan=1 colspan=1>0/2</td><td rowspan=1 colspan=1>19/19</td><td rowspan=1 colspan=1>1/2</td></tr><tr><td rowspan=1 colspan=1>(100%)</td><td rowspan=1 colspan=1>(0%)</td><td rowspan=1 colspan=1>(100%)</td><td rowspan=1 colspan=2>(0%)</td><td rowspan=1 colspan=1>(100%)</td><td rowspan=1 colspan=1>(0%)</td><td rowspan=1 colspan=1>(100%)</td><td rowspan=1 colspan=1>(50.0%)</td><td></td></tr><tr><td rowspan=1 colspan=1>Proteus spp.</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>0/0(-)</td><td rowspan=1 colspan=1>6/6(100%)</td><td rowspan=1 colspan=1>0/0(-)</td><td rowspan=1 colspan=1>6/6(100%)</td><td rowspan=1 colspan=1>0/0(-)</td><td rowspan=1 colspan=1>6/6(100%)</td><td rowspan=1 colspan=2>0/0()</td><td rowspan=1 colspan=1>6/6(100%)</td><td rowspan=1 colspan=1>0/0(-)</td><td rowspan=1 colspan=1>6/6(100%)</td><td rowspan=1 colspan=1>0/0(-)</td><td rowspan=1 colspan=1>6/6(100%)</td></tr><tr><td rowspan=2 colspan=1>Pseudomonasaeruginosa</td><td rowspan=2 colspan=1>67</td><td rowspan=2 colspan=1>0/16(0%)</td><td rowspan=2 colspan=1>51/51(100%)</td><td rowspan=2 colspan=1>0/16(0%)</td><td rowspan=2 colspan=1>51/51(100%)</td><td rowspan=2 colspan=1>0/16(0%)</td><td rowspan=2 colspan=1>51/51(100%)</td><td rowspan=1 colspan=2>0/16</td><td rowspan=1 colspan=1>51/51</td><td rowspan=1 colspan=1>0/16</td><td rowspan=1 colspan=1>51/51</td><td rowspan=1 colspan=1>0/16</td><td rowspan=1 colspan=1>51/51</td></tr><tr><td rowspan=1 colspan=2>(0%)</td><td rowspan=1 colspan=1>(100%)</td><td rowspan=1 colspan=1>(0%)</td><td rowspan=1 colspan=1>(100%)</td><td rowspan=1 colspan=1>(0%)</td><td rowspan=1 colspan=1>(100%)</td></tr><tr><td rowspan=1 colspan=1>Serratia marcescens</td><td rowspan=1 colspan=1>14e</td><td rowspan=1 colspan=1>0/1(0%)</td><td rowspan=1 colspan=1>13/13(100%)</td><td rowspan=1 colspan=1>1/1(100%)</td><td rowspan=1 colspan=1>13/13(100%)</td><td rowspan=1 colspan=1>0/1(0%)</td><td rowspan=1 colspan=1>13/13(100%)</td><td rowspan=1 colspan=2>0/1(0%)</td><td rowspan=1 colspan=1>13/13(100%)</td><td rowspan=1 colspan=1>0/1(0%)</td><td rowspan=1 colspan=1>13/13(100%)</td><td rowspan=1 colspan=1>1/1(100%)</td><td rowspan=1 colspan=1>13/13(100%)</td></tr><tr><td rowspan=2 colspan=1>Polymicrobialspecimens</td><td rowspan=2 colspan=1>92</td><td rowspan=2 colspan=1>0/13(0%)</td><td rowspan=2 colspan=1>79/79(100%)</td><td rowspan=2 colspan=1>4/13(30.8%)</td><td rowspan=1 colspan=1>78/79</td><td rowspan=1 colspan=1>0/13</td><td rowspan=1 colspan=1>79/79</td><td rowspan=1 colspan=2>0/13</td><td rowspan=1 colspan=1>79/79</td><td rowspan=2 colspan=1>1/13(7.7%)</td><td rowspan=2 colspan=1>79/79(100%)</td><td rowspan=2 colspan=1>4/13f(30.8%)</td><td rowspan=2 colspan=1>78/79(98.7%)</td></tr><tr><td rowspan=1 colspan=1>(98.7%)</td><td rowspan=1 colspan=1>(0%)</td><td rowspan=1 colspan=1>(100%)</td><td rowspan=1 colspan=2>(0%)</td><td rowspan=1 colspan=1>(100%)</td></tr></table>

a. The isolate recovered from one specimen $P$ . aeruginosa) did not yield valid AST results on the VITEK instrument b. One specimen had presence of dual AMR genes (KPC and VIM) c. An A. calcoaceticus-baumannii isolate was not recovered by qRefCx for one specimen d. $\boldsymbol { \mathrm { A n } E }$ . cloacae isolate was not recovered by qRefCx for one specimen e. A S. marcescens isolate was not recovered by qRefCx for one specimen f. Of the four specimens that were concordant: one specimen $A$ . calcoaceticus-baumannii complex, $K .$ . aerogenes, $K .$ pneumoniae group, and $P$ . aeruginosa detected by FilmArray (A. calcoaceticus-baumannii, $K .$ . pneumoniae, and $P$ . aeruginosa isolated by qRefCx, carbapenem resistance identified in all three isolates by qRefCx AST, KPC identified in the $K$ . pneumoniae isolate by PCR/seq, and VIM identified in the $P$ . aeruginosa isolate by PCR/seq); one specimen A. calcoaceticus-baumannii complex, $K .$ . pneumoniae group, Proteus spp., and $P$ . aeruginosa detected by FilmArray $K .$ . pneumoniae and $P$ . aeruginosa isolated by qRefCx, carbapenem resistance identified in the $K .$ . pneumoniae isolate by qRefCx AST, and KPC identified in the $K$ . pneumoniae isolate by

PCR/seq); one specimen $E$ . coli and K. pneumoniae group detected by FilmArray and isolated by qRefCx (carbapenem resistance identified in the $K .$ . pneumoniae isolate by qRefCx AST and KPC identified in the isolate by PCR/seq); one specimen $P$ . aeruginosa and S. marcescens detected by FilmArray and isolated by qRefCx (carbapenem resistance identified in the S. marcescens isolate by qRefCx AST and KPC identified in the isolate by PCR/seq). Of the nine specimens that were not concordant: one specimen $A$ . calcoaceticus-baumannii complex and Proteus spp. detected by FilmArray and isolated by qRefCx (carbapenem resistance identified in the A. calcoaceticus-baumannii isolate by qRefCx AST and KPC not identified in either isolate by PCR/seq); one specimen $E$ . cloacae complex and $P$ . aeruginosa detected by FilmArray (P. aeruginosa isolated by qRefCx, carbapenem resistance identified by qRefCx AST, and KPC not identified in the isolate by PCR/seq); one specimen $E$ . coli and $P$ . aeruginosa detected by FilmArray and isolated by qRefCx (carbapenem resistance identified in the P. aeruginosa isolate by qRefCx AST and KPC not identified in either isolate by PCR/seq); one specimen $K .$ . pneumoniae group and $P$ . aeruginosa detected by FilmArray and isolated by qRefCx (carbapenem resistance identified in the $K$ . pneumoniae isolate by qRefCx AST and KPC not identified in either isolate by $\mathrm { P C R } / \mathrm { s e q } )$ ; one specimen Proteus spp. and $P$ . aeruginosa detected by FilmArray and isolated by qRefCx (carbapenem resistance identified in the P. aeruginosa isolate by qRefCx AST and KPC not identified in either isolate by PCR/seq); one specimen $P$ . aeruginosa and S. marcescens detected by FilmArray and isolated by qRefCx (carbapenem resistance identified in the S. marcescens isolate by qRefCx AST and KPC not identified in either isolate by PCR/seq); one specimen $P$ . aeruginosa and S. marcescens detected by FilmArray $( P$ . aeruginosa isolated by qRefCx, carbapenem resistance identified by qRefCx AST, and KPC not identified in the isolate by PCR/seq); one specimen A. calcoaceticus-baumannii complex, E. coli, K. pneumoniae group, and $P$ . aeruginosa detected by FilmArray ( $E$ . coli and $P$ . aeruginosa isolated by qRefCx, carbapenem resistance identified in the $P$ . aeruginosa isolate by qRefCx AST, and KPC not identified in either isolate by PCR/seq); one specimen A. calcoaceticus-baumannii complex, Proteus spp., $P$ . aeruginosa, and S. marcescens detected by FilmArray (Proteus spp. and $P$ . aeruginosa isolated by qRefCx; carbapenem resistance identified in the $P$ . aeruginosa isolate by qRefCx AST and KPC not identified in either isolate by PCR/seq).

Table 64: OXA-48-like Performance Table (comparison to phenotypic AST methods for BAL specimens)   

<table><tr><td colspan="7" rowspan="1">BAL</td></tr><tr><td colspan="1" rowspan="2">ApplicableBacteriaResult</td><td colspan="3" rowspan="1">Positive Percent Agreement</td><td colspan="3" rowspan="1">Negative Percent Agreement</td></tr><tr><td colspan="1" rowspan="1">TP/(TP + FN)</td><td colspan="1" rowspan="1">%</td><td colspan="1" rowspan="1">95%CI</td><td colspan="1" rowspan="1">TN/(TN + FP)</td><td colspan="1" rowspan="1">%</td><td colspan="1" rowspan="1">95%CI</td></tr><tr><td colspan="1" rowspan="1">Overall(any applicablebacteriaDetected)</td><td colspan="1" rowspan="1">0/2</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">.</td><td colspan="1" rowspan="1">77/77</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">95.2-100%</td></tr><tr><td colspan="1" rowspan="1">Enterobactercloacaecomplex</td><td colspan="1" rowspan="1">0/1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">9/9</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">70.1-100%</td></tr><tr><td colspan="1" rowspan="1">Escherichiacoli</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">13/13</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">77.2-100%</td></tr><tr><td colspan="1" rowspan="1">Klebsiellaaerogenes</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">7/7</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">64.6-100%</td></tr><tr><td colspan="1" rowspan="1">Klebsiellaoxytoca</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">43.9-100%</td></tr><tr><td colspan="1" rowspan="1">Klebsiellapneumoniaegroup</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">15/15</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">79.6-100%</td></tr><tr><td colspan="1" rowspan="1">Proteus spp.</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">6/6</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">61.0-100%</td></tr><tr><td colspan="1" rowspan="1">Serratiamarcescens</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">8/8</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">67.6-100%</td></tr><tr><td colspan="1" rowspan="1">Polymicrobialspecimens</td><td colspan="1" rowspan="1">0/1a</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">16/16</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">80.6-100%</td></tr></table>

a. E. cloacae complex and $K$ . aerogenes detected by FilmArray $E _ { \cdot }$ . cloacae complex isolated by qRefCx and carbapenem resistance identified by qRefCx AST)

Table 65: OXA-48-like Performance Table (comparison to phenotypic AST methods for sputum specimens)   

<table><tr><td rowspan=1 colspan=7>Sputum</td></tr><tr><td rowspan=2 colspan=1>ApplicableBacteria Result</td><td rowspan=1 colspan=3>Positive Percent Agreement</td><td rowspan=1 colspan=3>Negative Percent Agreement</td></tr><tr><td rowspan=1 colspan=1>TP/(TP + FN)</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95%CI</td><td rowspan=1 colspan=1>TN/(TN + FP)</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95%CI</td></tr><tr><td rowspan=1 colspan=1>Overall(any applicablebacteriaDetected)</td><td rowspan=1 colspan=1>0/10</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>.</td><td rowspan=1 colspan=1>121/121</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>96.9-100%</td></tr><tr><td rowspan=1 colspan=1>Enterobactercloacaecomplex</td><td rowspan=1 colspan=1>0/1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>8/8</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>67.6-100%</td></tr><tr><td rowspan=1 colspan=1>Escherichia coli</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>17/17</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>81.6-100%</td></tr><tr><td rowspan=1 colspan=1>Klebsiellaaerogenes</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>4/4</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>51.0-100%</td></tr><tr><td rowspan=1 colspan=1>Klebsiellaoxytoca</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>5/5</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>56.6-100%</td></tr><tr><td rowspan=1 colspan=1>Klebsiellapneumoniaegroup</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>22/22</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>85.1-100%</td></tr><tr><td rowspan=1 colspan=1>Proteus spp.</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>9/9</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>70.1-100%</td></tr><tr><td rowspan=1 colspan=1>Serratiamarcescens</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>22/22</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>85.1-100%</td></tr><tr><td rowspan=1 colspan=1>Polymicrobialspecimens</td><td rowspan=1 colspan=1>0/3a</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>34/34</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>89.8-100%</td></tr></table>

a. One specimen $E$ . coli and $K$ . pneumoniae group detected by FilmArray and isolated by qRefCx (carbapenem resistance identified in the $K$ . pneumoniae group isolate by qRefCx AST); one specimen $K$ . aerogenes and $K$ . pneumoniae group detected by FilmArray $K .$ . pneumoniae group isolated by qRefCx and carbapenem resistance identified by qRefCx AST); one specimen $K .$ . pneumoniae group and Proteus spp. detected by FilmArray $K .$ . pneumoniae group isolated by qRefCx and carbapenem resistance identified by qRefCx AST)

The FilmArray Pneumonia Panel mecA/C and MREJ reporting was also compared to the standard phenotypic cefoxitin susceptibility testing methods performed in conjunction with qRefCx. The correlation between FilmArray Pneumonia Panel reporting of mecA/C and MREJ in a particular specimen as compared to the phenotypic AST results of isolates recovered from the same specimen is

Table 66: mecA/C and MREJ 3x3 Performance Table (qRefCx & phenotypic AST on cultured isolate(s) from BAL specimens)   

<table><tr><td rowspan=1 colspan=6>BAL</td></tr><tr><td rowspan=2 colspan=2>S. aureusmecA/C and MREJ</td><td rowspan=1 colspan=4>qRefCx: S. aureus PCR/seq: mecA/C</td></tr><tr><td rowspan=1 colspan=1>Org+ / Res+</td><td rowspan=1 colspan=1>Org+ / Res-</td><td rowspan=1 colspan=1>Org -</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=4 colspan=1>FilmArrayResult</td><td rowspan=1 colspan=1>Org+ / Res+</td><td rowspan=1 colspan=1>18</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>25</td><td rowspan=1 colspan=1>46</td></tr><tr><td rowspan=1 colspan=1>Org+ / Res-</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>45</td><td rowspan=1 colspan=1>70</td></tr><tr><td rowspan=1 colspan=1>Org -</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>729</td><td rowspan=1 colspan=1>730</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>19</td><td rowspan=1 colspan=1>28</td><td rowspan=1 colspan=1>799</td><td rowspan=1 colspan=1>846</td></tr><tr><td rowspan=1 colspan=3>Performance</td><td rowspan=1 colspan=1>Agreement</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95%CI</td></tr><tr><td rowspan=1 colspan=3>Org+ /Res+</td><td rowspan=1 colspan=1>18/19</td><td rowspan=1 colspan=1>94.7%</td><td rowspan=1 colspan=1>75.4-99.1%</td></tr><tr><td rowspan=1 colspan=3>Org+ / Res-</td><td rowspan=1 colspan=1>24/28</td><td rowspan=1 colspan=1>85.7%</td><td rowspan=1 colspan=1>68.5-94.3%</td></tr><tr><td rowspan=1 colspan=3>Org -</td><td rowspan=1 colspan=1>729/799</td><td rowspan=1 colspan=1>91.2%</td><td rowspan=1 colspan=1>89.1-93.0%</td></tr><tr><td rowspan=1 colspan=3>Interpretation</td><td rowspan=1 colspan=1>PPA</td><td rowspan=1 colspan=1>NPA</td><td rowspan=1 colspan=1>Prevalence</td></tr><tr><td rowspan=2 colspan=3>MRSA</td><td rowspan=1 colspan=1>18/19</td><td rowspan=1 colspan=1>799/827</td><td rowspan=1 colspan=1>46/846</td></tr><tr><td rowspan=1 colspan=1>(94.7%)</td><td rowspan=1 colspan=1>(96.6%)</td><td rowspan=1 colspan=1>(5.4%)</td></tr><tr><td rowspan=2 colspan=3>MSSA</td><td rowspan=1 colspan=1>24/28</td><td rowspan=1 colspan=1>772/818</td><td rowspan=1 colspan=1>70/846</td></tr><tr><td rowspan=1 colspan=1>(85.7%)</td><td rowspan=1 colspan=1>(94.4%)</td><td rowspan=1 colspan=1>(8.3%)</td></tr><tr><td rowspan=2 colspan=3>S. aureus</td><td rowspan=1 colspan=1>46/47</td><td rowspan=1 colspan=1>729/799</td><td rowspan=1 colspan=1>116/846</td></tr><tr><td rowspan=1 colspan=1>(97.9%)</td><td rowspan=1 colspan=1>(91.2%)</td><td rowspan=1 colspan=1>(13.7%)</td></tr></table>

Table 67: mecA/C and MREJ 3x3 Performance Table (qRefCx & phenotypic AST on cultured isolate(s) from sputum specimens)   

<table><tr><td rowspan=1 colspan=6>Sputum</td></tr><tr><td rowspan=2 colspan=2>S. aureusmecA/C and MREJ</td><td rowspan=1 colspan=4>qRefCx: S. aureus PCR/seq: mecA/C</td></tr><tr><td rowspan=1 colspan=1>Org+ / Res+</td><td rowspan=1 colspan=1>Org+ / Res-</td><td rowspan=1 colspan=1>Org -</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=4 colspan=1>FilmArrayResult</td><td rowspan=1 colspan=1>Org+ / Res+</td><td rowspan=1 colspan=1>59</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>45</td><td rowspan=1 colspan=1>107</td></tr><tr><td rowspan=1 colspan=1>Org+ / Res-</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>48</td><td rowspan=1 colspan=1>48</td><td rowspan=1 colspan=1>97</td></tr><tr><td rowspan=1 colspan=1>Org -</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>631</td><td rowspan=1 colspan=1>632</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>52</td><td rowspan=1 colspan=1>724</td><td rowspan=1 colspan=1>836</td></tr><tr><td rowspan=1 colspan=3>Performance</td><td rowspan=1 colspan=1>Agreement</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95%CI</td></tr><tr><td rowspan=1 colspan=3>Org+ / Res+</td><td rowspan=1 colspan=1>59/60</td><td rowspan=1 colspan=1>98.3%</td><td rowspan=1 colspan=1>91.1-99.7%</td></tr><tr><td rowspan=1 colspan=3>Org+ /Res-</td><td rowspan=1 colspan=1>48/52</td><td rowspan=1 colspan=1>92.3%</td><td rowspan=1 colspan=1>81.8-97.0%</td></tr><tr><td rowspan=1 colspan=3>Org -</td><td rowspan=1 colspan=1>631/724</td><td rowspan=1 colspan=1>87.2%</td><td rowspan=1 colspan=1>84.5-89.4%</td></tr><tr><td rowspan=1 colspan=3>Interpretation</td><td rowspan=1 colspan=1>PPA</td><td rowspan=1 colspan=1>NPA</td><td rowspan=1 colspan=1>Prevalence</td></tr><tr><td rowspan=2 colspan=3>MRSA</td><td rowspan=1 colspan=1>59/60</td><td rowspan=1 colspan=1>728/776</td><td rowspan=1 colspan=1>107/836</td></tr><tr><td rowspan=1 colspan=1>(98.3%)</td><td rowspan=1 colspan=1>(93.8%)</td><td rowspan=1 colspan=1>(12.8%)</td></tr><tr><td rowspan=2 colspan=3>MSSA</td><td rowspan=1 colspan=1>48/52</td><td rowspan=1 colspan=1>735/784</td><td rowspan=1 colspan=1>97/836</td></tr><tr><td rowspan=1 colspan=1>(92.3%)</td><td rowspan=1 colspan=1>(93.8%)</td><td rowspan=1 colspan=1>(11.6%)</td></tr></table>

<table><tr><td rowspan=1 colspan=4>Sputum</td></tr><tr><td rowspan=2 colspan=1>S. aureus</td><td rowspan=1 colspan=1>111/112</td><td rowspan=1 colspan=1>631/724</td><td rowspan=1 colspan=1>204/836</td></tr><tr><td rowspan=1 colspan=1>(99.1%)</td><td rowspan=1 colspan=1>(87.2%)</td><td rowspan=1 colspan=1>(24.4%)</td></tr></table>

The FilmArray Pneumonia Panel bin performance compared to the quantitative molecular assay (qMol) comparator is shown for BAL (Table 68) and sputum (Table 69). The qMol values are broken into onelog ranges correlating to the reported semi-quantitative FilmArray Pneumonia Panel bins. The relationship between qMol quantitative bins in copies $/ \mathrm { m L }$ and traditional culture quantification in CFU/mL was not established.

Table 68: FilmArray Pneumonia Panel Overall bin performance for BAL Specimens (qMol)   

<table><tr><td rowspan=1 colspan=8>BAL</td></tr><tr><td rowspan=1 colspan=2>qMol Range ofValues a(copies/mL)</td><td rowspan=1 colspan=1>Not Detected(ND) to&lt;103.5</td><td rowspan=1 colspan=1>104.0</td><td rowspan=1 colspan=1>105.0</td><td rowspan=1 colspan=1>106.0</td><td rowspan=1 colspan=1>≥107.0</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=5 colspan=1>Rm(/o)</td><td rowspan=1 colspan=1>ND</td><td rowspan=1 colspan=1>12025</td><td rowspan=1 colspan=1>35</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>12067</td></tr><tr><td rowspan=1 colspan=1>104</td><td rowspan=1 colspan=1>47</td><td rowspan=1 colspan=1>48</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>117</td></tr><tr><td rowspan=1 colspan=1>105}$</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>23</td><td rowspan=1 colspan=1>57</td><td rowspan=1 colspan=1>22</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>109</td></tr><tr><td rowspan=1 colspan=1>106</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>29</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>88</td></tr><tr><td rowspan=1 colspan=1>≥107</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>41</td><td rowspan=1 colspan=1>112</td><td rowspan=1 colspan=1>159</td></tr><tr><td rowspan=2 colspan=2>% concordant</td><td rowspan=2 colspan=1>12025/12082(99.5%)</td><td rowspan=1 colspan=1>48/109(44.0%)</td><td rowspan=1 colspan=1>57/116(49.1%)</td><td rowspan=1 colspan=1>40/103(38.8%)</td><td rowspan=1 colspan=1>112/130(86.2%)</td><td rowspan=2 colspan=1>12540</td></tr><tr><td rowspan=1 colspan=4>257/458 (56.1%)</td></tr></table>

a.Shaded cells indicate results considered concordant between the FilmArray Pneumonia Panel and qMol.

Table 69: FilmArray Pneumonia Panel Overall bin performance for Sputum Specimens (qMol)   

<table><tr><td rowspan=1 colspan=8>Sputum</td></tr><tr><td rowspan=6 colspan=2>qMol Range ofValues a(copies/mL)R          ND(!d)104$10}$106≥107</td><td rowspan=1 colspan=1>ND to&lt;103.5</td><td rowspan=1 colspan=1>104.0</td><td rowspan=1 colspan=1>105.0</td><td rowspan=1 colspan=1>106.0</td><td rowspan=1 colspan=1>≥107.0</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>ND</td><td rowspan=1 colspan=1>11392</td><td rowspan=1 colspan=1>85</td><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>11498</td></tr><tr><td rowspan=1 colspan=1>104</td><td rowspan=1 colspan=1>79</td><td rowspan=1 colspan=1>87</td><td rowspan=1 colspan=1>41</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>214</td></tr><tr><td rowspan=1 colspan=1>$10}$</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>33</td><td rowspan=1 colspan=1>104</td><td rowspan=1 colspan=1>43</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>197</td></tr><tr><td rowspan=2 colspan=1>106≥107</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>39</td><td rowspan=1 colspan=1>88</td><td rowspan=1 colspan=1>41</td><td rowspan=1 colspan=1>174</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>44</td><td rowspan=1 colspan=1>288</td><td rowspan=1 colspan=1>337</td></tr><tr><td rowspan=2 colspan=2>% concordant</td><td rowspan=2 colspan=1>11392/11489(99.2%)</td><td rowspan=1 colspan=1>87/209(41.6%)</td><td rowspan=1 colspan=1>104/202(51.5%)</td><td rowspan=1 colspan=1>88/184(47.8%)</td><td rowspan=1 colspan=1>288/336(67.9%)</td><td rowspan=2 colspan=1>12420</td></tr><tr><td rowspan=1 colspan=4>567/931 (60.9%)</td></tr></table>

a Shaded cells indicate results considered concordant between the FilmArray Pneumonia Panel and qMol.

Table 70: FilmArray Pneumonia Panel Bin Performance Summary for BAL Specimens Compared to qMol   

<table><tr><td rowspan=1 colspan=7>BAL</td></tr><tr><td rowspan=1 colspan=1>qMol Range of Values(Concordant FilmArray Bin)</td><td rowspan=1 colspan=1>103.5 to&lt;104.0(ND or 104)</td><td rowspan=1 colspan=1>104.0 to&lt;105.0(104 or 105)</td><td rowspan=1 colspan=1>105.0 to&lt;1060(105 or 106)</td><td rowspan=1 colspan=1>106.0 to&lt;107.00(106 or 107)</td><td rowspan=1 colspan=1>≥107.0 (≥107)</td><td rowspan=1 colspan=1>Overall</td></tr><tr><td rowspan=1 colspan=1>Acinetobactercalcoaceticus-baumannii complex</td><td rowspan=1 colspan=1>2/2(100%)</td><td rowspan=1 colspan=1>1/2(50.0%)</td><td rowspan=1 colspan=1>2/3(66.7%)</td><td rowspan=1 colspan=1>2/3(66.7%)</td><td rowspan=1 colspan=1>0/0(-)</td><td rowspan=1 colspan=1>7/10(70.0%)</td></tr><tr><td rowspan=1 colspan=1>Enterobacter aerogenes</td><td rowspan=1 colspan=1>1/1(100%)</td><td rowspan=1 colspan=1>2/3(66.7%)</td><td rowspan=1 colspan=1>4/4(100%)</td><td rowspan=1 colspan=1>2/3(66.7%)</td><td rowspan=1 colspan=1>3/4(75.0%)</td><td rowspan=1 colspan=1>12/15(80.0%)</td></tr><tr><td rowspan=1 colspan=1>Enterobacter cloacaecomplex</td><td rowspan=1 colspan=1>0/0(-)</td><td rowspan=1 colspan=1>3/7(42.9%)</td><td rowspan=1 colspan=1>2/3(66.7%)</td><td rowspan=1 colspan=1>5/7(71.4%)</td><td rowspan=1 colspan=1>3/4(75.0%)</td><td rowspan=1 colspan=1>13/21(61.9%)</td></tr><tr><td rowspan=1 colspan=1>Escherichia coli</td><td rowspan=1 colspan=1>1/2(50.0%)</td><td rowspan=1 colspan=1>3/3(100%)</td><td rowspan=1 colspan=1>4/5(80.0%)</td><td rowspan=1 colspan=1>4/4(100%)</td><td rowspan=1 colspan=1>5/5(100%)</td><td rowspan=1 colspan=1>17/19(89.5%)</td></tr><tr><td rowspan=1 colspan=1>Haemophilus influenzae</td><td rowspan=1 colspan=1>11/11(100%)</td><td rowspan=1 colspan=1>8/13(61.5%)</td><td rowspan=1 colspan=1>14/15(93.3%)</td><td rowspan=1 colspan=1>23/23(100%)</td><td rowspan=1 colspan=1>16/16(100%)</td><td rowspan=1 colspan=1>72/78(92.3%)</td></tr><tr><td rowspan=1 colspan=1>Klebsiella oxytoca</td><td rowspan=1 colspan=1>1/1(100%)</td><td rowspan=1 colspan=1>4/5(80.0%)</td><td rowspan=1 colspan=1>1/1(100%)</td><td rowspan=1 colspan=1>1/1(100%)</td><td rowspan=1 colspan=1>1/1(100%)</td><td rowspan=1 colspan=1>8/9(88.9%)</td></tr><tr><td rowspan=2 colspan=1>Klebsiella pneumoniaegroup</td><td rowspan=2 colspan=1>3/5(60.0%)</td><td rowspan=2 colspan=1>5/6(83.3%)</td><td rowspan=2 colspan=1>7/8(87.5%)</td><td rowspan=2 colspan=1>4/4(100%)</td><td rowspan=1 colspan=1>5/5</td><td rowspan=1 colspan=1>24/28</td></tr><tr><td rowspan=1 colspan=1>(100%)</td><td rowspan=1 colspan=1>(85.7%)</td></tr><tr><td rowspan=1 colspan=1>Moraxella catarrhalis</td><td rowspan=1 colspan=1>4/4(100%)</td><td rowspan=1 colspan=1>4/5(80.0%)</td><td rowspan=1 colspan=1>9/10(90.0%)</td><td rowspan=1 colspan=1>4/4(100%)</td><td rowspan=1 colspan=1>4/4(100%)</td><td rowspan=1 colspan=1>25/27(92.6%)</td></tr><tr><td rowspan=1 colspan=1>Proteus spp.</td><td rowspan=1 colspan=1>1/1(100%)</td><td rowspan=1 colspan=1>3/4(75.0%)</td><td rowspan=1 colspan=1>3/3(100%)</td><td rowspan=1 colspan=1>1/1(100%)</td><td rowspan=1 colspan=1>2/2(100%)</td><td rowspan=1 colspan=1>10/11(90.9%)</td></tr><tr><td rowspan=1 colspan=1>Pseudomonas aeruginosa</td><td rowspan=1 colspan=1>9/11(81.8%)</td><td rowspan=1 colspan=1>11/13(84.6%)</td><td rowspan=1 colspan=1>10/14(71.4%)</td><td rowspan=1 colspan=1>22/22(100%)</td><td rowspan=1 colspan=1>10/10(100%)</td><td rowspan=1 colspan=1>62/70(88.6%)</td></tr><tr><td rowspan=1 colspan=1>Serratia marcescens</td><td rowspan=1 colspan=1>1/1(100%)</td><td rowspan=1 colspan=1>4/4(100%)</td><td rowspan=1 colspan=1>4/4(100%)</td><td rowspan=1 colspan=1>3/3(100%)</td><td rowspan=1 colspan=1>0/0(-)</td><td rowspan=1 colspan=1>12/12(100%)</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus aureus</td><td rowspan=1 colspan=1>12/13(92.3%)</td><td rowspan=1 colspan=1>20/24(83.3%)</td><td rowspan=1 colspan=1>22/31(71.0%)</td><td rowspan=1 colspan=1>20/22(90.9%)</td><td rowspan=1 colspan=1>10/11(90.9%)</td><td rowspan=1 colspan=1>84/101(83.2%)</td></tr><tr><td rowspan=1 colspan=1>Streptococcus agalactiae</td><td rowspan=1 colspan=1>1/1(100%)</td><td rowspan=1 colspan=1>7/7(100%)</td><td rowspan=1 colspan=1>3/4(75.0%)</td><td rowspan=1 colspan=1>5/6(83.3%)</td><td rowspan=1 colspan=1>2/2(100%)</td><td rowspan=1 colspan=1>18/20(90.0%)</td></tr><tr><td rowspan=1 colspan=1>Streptococcus pneumoniae</td><td rowspan=1 colspan=1>4/4(100%)</td><td rowspan=1 colspan=1>7/8(87.5%)</td><td rowspan=1 colspan=1>6/8(75.0%)</td><td rowspan=1 colspan=1>4/4(100%)</td><td rowspan=1 colspan=1>5/6(83.3%)</td><td rowspan=1 colspan=1>26/30(86.7%)</td></tr><tr><td rowspan=1 colspan=1>Streptococcus pyogenes</td><td rowspan=1 colspan=1>2/3(66.7%)</td><td rowspan=1 colspan=1>1/1(100%)</td><td rowspan=1 colspan=1>2/2(100%)</td><td rowspan=1 colspan=1>1/1(100%)</td><td rowspan=1 colspan=1>0/0(-)</td><td rowspan=1 colspan=1>6/7(85.7%)</td></tr></table>

Table 71: FilmArray Pneumonia Panel Bin Performance Summary for Sputum Specimens Compared to qMol   

<table><tr><td colspan="7">Sputum</td></tr><tr><td>qMol Range of Values (Concordant FilmArray Bin)</td><td>103.5 to &lt;104.0 (ND or 104</td><td>104.0 to &lt;105.0 (104 or 105)</td><td>105.0 to &lt;106.0 (105 or 106)</td><td>106.0 to &lt;107.0 (106 or 107)</td><td>≥107.0 (≥107)</td><td>Overall</td></tr><tr><td>Acinetobacter calcoaceticus-</td><td>10/10 (100%)</td><td>2/8 (25.0%)</td><td>9/12 (75.0%)</td><td>2/3 (66.7%)</td><td>5/7 (71.4%)</td><td>28/40 (70.0%)</td></tr><tr><td colspan="1" rowspan="2">Enterobacter aerogenes</td><td colspan="1" rowspan="2">3/3(100%)</td><td colspan="1" rowspan="1">5/6</td><td colspan="1" rowspan="1">0/2</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="2">1/1(100%)</td><td colspan="1" rowspan="2">12/15(80.0%)</td></tr><tr><td colspan="1" rowspan="1">(83.3%)</td><td colspan="1" rowspan="1">(0%)</td><td colspan="1" rowspan="1">(100%)</td></tr><tr><td colspan="1" rowspan="2">Enterobacter cloacaecomplex</td><td colspan="1" rowspan="2">7/7(100%)</td><td colspan="1" rowspan="2">10/17(58.8%)</td><td colspan="1" rowspan="1">6/7</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">30/38</td></tr><tr><td colspan="1" rowspan="1">(85.7%)</td><td colspan="1" rowspan="1">(100%)</td><td colspan="1" rowspan="1">(100%)</td><td colspan="1" rowspan="1">(78.9%)</td></tr><tr><td colspan="1" rowspan="2">Escherichia coli</td><td colspan="1" rowspan="2">7/7(100%)</td><td colspan="1" rowspan="1">14/19</td><td colspan="1" rowspan="1">6/7</td><td colspan="1" rowspan="1">7/7</td><td colspan="1" rowspan="1">11/11</td><td colspan="1" rowspan="1">45/51</td></tr><tr><td colspan="1" rowspan="1">(73.7%)</td><td colspan="1" rowspan="1">(85.7%)</td><td colspan="1" rowspan="1">(100%)</td><td colspan="1" rowspan="1">(100%)</td><td colspan="1" rowspan="1">(88.2%)</td></tr><tr><td colspan="1" rowspan="2">Haemophilus influenzae</td><td colspan="1" rowspan="2">11/11(100%)</td><td colspan="1" rowspan="2">11/15(73.3%)</td><td colspan="1" rowspan="2">15/21(71.4%)</td><td colspan="1" rowspan="2">23/25(92.0%)</td><td colspan="1" rowspan="1">34/38</td><td colspan="1" rowspan="2">94/110(85.5%)</td></tr><tr><td colspan="1" rowspan="1">(89.5%)</td></tr><tr><td colspan="1" rowspan="1">Klebsiella oxytoca</td><td colspan="1" rowspan="1">3/3(100%)</td><td colspan="1" rowspan="1">2/4(50.0%)</td><td colspan="1" rowspan="1">7/7(100%)</td><td colspan="1" rowspan="1">2/3(66.7%)</td><td colspan="1" rowspan="1">1/1(100%)</td><td colspan="1" rowspan="1">15/18(83.3%)</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniaegroup</td><td colspan="1" rowspan="1">6/6(100%)</td><td colspan="1" rowspan="1">14/20(70.0%)</td><td colspan="1" rowspan="1">12/16(75.0%)</td><td colspan="1" rowspan="1">9/13(69.2%)</td><td colspan="1" rowspan="1">8/9(88.9%)</td><td colspan="1" rowspan="1">49/64(76.6%)</td></tr><tr><td colspan="1" rowspan="1">Moraxella catarrhalis</td><td colspan="1" rowspan="1">2/2(100%)</td><td colspan="1" rowspan="1">6/7(85.7%)</td><td colspan="1" rowspan="1">11/13(84.6%)</td><td colspan="1" rowspan="1">18/19(94.7%)</td><td colspan="1" rowspan="1">28/29(96.6%)</td><td colspan="1" rowspan="1">65/70(92.9%)</td></tr><tr><td colspan="1" rowspan="1">Proteus spp.</td><td colspan="1" rowspan="1">2/2(100%)</td><td colspan="1" rowspan="1">6/8(75.0%)</td><td colspan="1" rowspan="1">3/4(75.0%)</td><td colspan="1" rowspan="1">6/7(85.7%)</td><td colspan="1" rowspan="1">3/3(100%)</td><td colspan="1" rowspan="1">20/24(83.3%)</td></tr><tr><td colspan="1" rowspan="2">Pseudomonas aeruginosa</td><td colspan="1" rowspan="2">5/8(62.5%)</td><td colspan="1" rowspan="2">17/23(73.9%)</td><td colspan="1" rowspan="2">26/34(76.5%)</td><td colspan="1" rowspan="1">36/39</td><td colspan="1" rowspan="1">45/48</td><td colspan="1" rowspan="2">129/152(84.9%)</td></tr><tr><td colspan="1" rowspan="1">(92.3%)</td><td colspan="1" rowspan="1">(93.8%)</td></tr><tr><td colspan="1" rowspan="1">Serratia marcescens</td><td colspan="1" rowspan="1">4/5(80.0%)</td><td colspan="1" rowspan="1">15/16(93.8%)</td><td colspan="1" rowspan="1">12/13(92.3%)</td><td colspan="1" rowspan="1">11/11(100%)</td><td colspan="1" rowspan="1">1/1(100%)</td><td colspan="1" rowspan="1">43/46(93.5%)</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">15/21(71.4%)</td><td colspan="1" rowspan="1">43/53(81.1%)</td><td colspan="1" rowspan="1">33/36(91.7%)</td><td colspan="1" rowspan="1">38/45(84.4%)</td><td colspan="1" rowspan="1">37/40(92.5%)</td><td colspan="1" rowspan="1">166/195(85.1%)</td></tr><tr><td colspan="1" rowspan="1">Streptococcus agalactiae</td><td colspan="1" rowspan="1">2/2(100%)</td><td colspan="1" rowspan="1">3/6(50.0%)</td><td colspan="1" rowspan="1">12/14(85.7%)</td><td colspan="1" rowspan="1">9/9(100%)</td><td colspan="1" rowspan="1">9/9(100%)</td><td colspan="1" rowspan="1">35/40(87.5%)</td></tr><tr><td colspan="1" rowspan="1">Streptococcus pneumoniae</td><td colspan="1" rowspan="1">5/5(100%)</td><td colspan="1" rowspan="1">9/13(69.2%)</td><td colspan="1" rowspan="1">6/9(66.7%)</td><td colspan="1" rowspan="1">7/10(70.0%)</td><td colspan="1" rowspan="1">18/21(85.7%)</td><td colspan="1" rowspan="1">45/58(77.6%)</td></tr><tr><td colspan="1" rowspan="1">Streptococcus pyogenes</td><td colspan="1" rowspan="1">0/0(-)</td><td colspan="1" rowspan="1">2/2(100%)</td><td colspan="1" rowspan="1">2/2(100%)</td><td colspan="1" rowspan="1">3/3(100%)</td><td colspan="1" rowspan="1">3/3(100%)</td><td colspan="1" rowspan="1">10/10(100%)</td></tr></table>

The FilmArray Pneumonia Panel bin performance compared to the quantitative reference culture (qRefCx) comparator is shown for BAL and sputum in the Tables 72 through 83. Data is shown for overall performance, as well as for some individual organisms. In these tables, the values reported by culture are broken into ranges. A FilmArray Pneumonia Panel bin result is considered concordant if the culture value is within $0 . 5 \log$ of the bin boundary. For example, the $1 0 ^ { 5 }$ FilmArray Pneumonia Panel bin $( 1 0 ^ { 4 . 5 } – 1 0 ^ { 5 . 5 } )$ is concordant with the culture range of $1 0 ^ { 4 } – 1 0 ^ { 5 } \mathrm { C F U / m L }$ .

Table 72: FilmArray Pneumonia Panel Overall Bin Performance for BAL Specimens $\bf ( q R e f C x )$   

<table><tr><td colspan="8" rowspan="1">BAL</td></tr><tr><td colspan="2" rowspan="1">qRefCx Range ofValues(CFU/mL)(Predicted FilmArray Bin)</td><td colspan="1" rowspan="1">ND to&lt;103.5(ND)</td><td colspan="1" rowspan="1">103.5 to&lt;104.0(ND or 104)</td><td colspan="1" rowspan="1">104.0 to&lt;105.0(104 or 105)</td><td colspan="1" rowspan="1">105.0 to&lt;106.0(105 or 106)</td><td colspan="1" rowspan="1">106.0 to&lt;107.0(106 or 107)</td><td colspan="1" rowspan="1">≥107.0(≥107)</td></tr><tr><td colspan="1" rowspan="3">M       c!csnmM</td><td colspan="1" rowspan="1">ND</td><td colspan="1" rowspan="1">12202</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">104</td><td colspan="1" rowspan="1">116</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">105}$</td><td colspan="1" rowspan="1">90</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">11</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="1">106</td><td colspan="1" rowspan="1">61</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">17</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">≥107</td><td colspan="1" rowspan="1">61</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">36</td><td colspan="1" rowspan="1">32</td><td colspan="1" rowspan="1">11</td><td colspan="1" rowspan="1">12</td></tr><tr><td colspan="2" rowspan="2">% concordant</td><td colspan="1" rowspan="2">12202/12530(97.4%)</td><td colspan="1" rowspan="1">1/32 (3.1%)</td><td colspan="1" rowspan="1">14/69 (20.3%)</td><td colspan="1" rowspan="1">2/34 (5.9%)</td><td colspan="1" rowspan="1">12/13 (92.3%)</td><td colspan="1" rowspan="1">12/12 (100%)</td></tr><tr><td colspan="3" rowspan="1">42/160 (26.3%)</td><td colspan="2" rowspan="1"></td></tr></table>

Table 73: FilmArray Pneumonia Panel Overall Bin Performance for Sputum Specimens (qRefCx)   

<table><tr><td rowspan=1 colspan=8>Sputum</td></tr><tr><td rowspan=1 colspan=2>qRefCx Range ofValues(CFU/mL)(Predicted FilmArray Bin)</td><td rowspan=1 colspan=1>ND to&lt;103.5(ND)</td><td rowspan=1 colspan=1>103.5 to&lt;104.0(104)</td><td rowspan=1 colspan=1>104.0 to&lt;105.0(104 or 105)</td><td rowspan=1 colspan=1>105.0 to&lt;106.0(105 or 106)</td><td rowspan=1 colspan=1>10%.0 to&lt;107.0(106 or 107)</td><td rowspan=1 colspan=1>≥107.0(≥107)</td></tr><tr><td rowspan=5 colspan=1>Fr   (Tc/do)1</td><td rowspan=1 colspan=1>ND</td><td rowspan=1 colspan=1>11596</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>104</td><td rowspan=1 colspan=1>193</td><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>$10c}$</td><td rowspan=1 colspan=1>139</td><td rowspan=1 colspan=1>19</td><td rowspan=1 colspan=1>28</td><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>106</td><td rowspan=1 colspan=1>94</td><td rowspan=1 colspan=1>19</td><td rowspan=1 colspan=1>36</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>≥107</td><td rowspan=1 colspan=1>121</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>88</td><td rowspan=1 colspan=1>53</td><td rowspan=1 colspan=1>49</td><td rowspan=1 colspan=1>20</td></tr><tr><td rowspan=2 colspan=2>% concordant</td><td rowspan=2 colspan=1>11596/12143</td><td rowspan=1 colspan=1>14/64 (21.9%)</td><td rowspan=1 colspan=1>37/167(22.2%)</td><td rowspan=1 colspan=1>35/90 (38.9%)</td><td rowspan=1 colspan=1>53/54(98.1%)</td><td rowspan=1 colspan=1>20/22(90.9%)</td></tr><tr><td rowspan=1 colspan=2></td><td rowspan=1 colspan=3>159/397 (40.1%)</td></tr></table>

Table 74: FilmArray Pneumonia Panel H. influenzae Bin Performance for BAL Specimens (qRefCx)   

<table><tr><td rowspan=1 colspan=8>BAL</td></tr><tr><td rowspan=1 colspan=2>qRefCx Range ofValues(CFU/mL)(Predicted FilmArray Bin)</td><td rowspan=1 colspan=1>ND to&lt;103.5(ND)</td><td rowspan=1 colspan=1>103.5 to&lt;104.0(104)</td><td rowspan=1 colspan=1>104.0 to&lt;105.0(104 or 105)</td><td rowspan=1 colspan=1>105.0 to&lt;106.0(105 or 106)</td><td rowspan=1 colspan=1>10%.0 to&lt;107.0(106 or 107)</td><td rowspan=1 colspan=1>≥107.0(≥107)</td></tr><tr><td rowspan=5 colspan=1>Fra   (Tc/!do)B</td><td rowspan=1 colspan=1>ND</td><td rowspan=1 colspan=1>764</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>104</td><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>$105}$</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>106</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>≥107</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=2 colspan=2>% concordant</td><td rowspan=2 colspan=1>764/836(91.4%)</td><td rowspan=1 colspan=1>0/2 (0%)</td><td rowspan=1 colspan=1>0/2 (0%)</td><td rowspan=1 colspan=1>0/5 (0%)</td><td rowspan=1 colspan=1>0/0 (-)</td><td rowspan=1 colspan=1>1/1 (100%)</td></tr><tr><td rowspan=1 colspan=5>1/10 (10.0%)</td></tr></table>

Table 75: FilmArray Pneumonia Panel H. influenzae Bin Performance for Sputum Specimens (qRefCx)   

<table><tr><td rowspan=1 colspan=8>Sputum</td></tr><tr><td rowspan=1 colspan=2>qRefCx Range ofValues(CFU/mL)(Predicted FilmArray Bin)</td><td rowspan=1 colspan=1>ND to&lt;103.5(ND)</td><td rowspan=1 colspan=1>103.5 to&lt;104.0(104)</td><td rowspan=1 colspan=1>104.0 to&lt;105.0(104 or 105)</td><td rowspan=1 colspan=1>105.0 to&lt;10%.0(105 or 106)</td><td rowspan=1 colspan=1>106.0 to&lt;107.0(106 or 107)</td><td rowspan=1 colspan=1>≥107.0(≥107)</td></tr><tr><td rowspan=5 colspan=1>Rra  (/d)1</td><td rowspan=1 colspan=1>ND</td><td rowspan=1 colspan=1>727</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>104</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>10c}$</td><td rowspan=1 colspan=1>19</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>106</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>≥107</td><td rowspan=1 colspan=1>38</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=2 colspan=2>% concordant</td><td rowspan=2 colspan=1>727/818(88.9%)</td><td rowspan=1 colspan=1>0/0 (-)</td><td rowspan=1 colspan=1>0/12 (0%)</td><td rowspan=1 colspan=1>1/4 (25.0%)</td><td rowspan=1 colspan=1>2/2 (100%)</td><td rowspan=1 colspan=1>0/0 (-)</td></tr><tr><td rowspan=1 colspan=5>3/18 (16.7%)</td></tr></table>

Table 76: FilmArray Pneumonia Panel $P .$ aeruginosa Bin Performance for BAL Specimens (qRefCx)   

<table><tr><td rowspan=1 colspan=8>BAL</td></tr><tr><td rowspan=1 colspan=2>qRefCx Range ofValues(CFU/mL)(Predicted FilmArray Bin)</td><td rowspan=1 colspan=1>ND to&lt;103.5(ND)</td><td rowspan=1 colspan=1>103.5 to&lt;104.0(104)</td><td rowspan=1 colspan=1>104.0 to&lt;105.0(104 or 105)</td><td rowspan=1 colspan=1>105.0 to&lt;106.0(105 or 106)</td><td rowspan=1 colspan=1>106.0 to&lt;107.0(106 or 107)</td><td rowspan=1 colspan=1>≥107.0(≥107)</td></tr><tr><td rowspan=5 colspan=1>Fra   (/d)1</td><td rowspan=1 colspan=1>ND</td><td rowspan=1 colspan=1>772</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>104</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>10c}$</td><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>106</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>≥107</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>2</td></tr><tr><td rowspan=2 colspan=2>% concordant</td><td rowspan=2 colspan=1>772/810(95.3%)</td><td rowspan=1 colspan=1>0/4 (0.0%)</td><td rowspan=1 colspan=1>2/15 (13.3%)</td><td rowspan=1 colspan=1>1/13 (7.7%)</td><td rowspan=1 colspan=1>2/2 (100%)</td><td rowspan=1 colspan=1>2/2 (100%)</td></tr><tr><td rowspan=1 colspan=5>7/36 (19.4%)</td></tr></table>

Table 77: FilmArray Pneumonia Panel $P .$ . aeruginosa Bin Performance for Sputum Specimens (qRefCx)   

<table><tr><td rowspan=1 colspan=8>Sputum</td></tr><tr><td rowspan=1 colspan=2>qRefCx Range ofValues(CFU/mL)(Predicted FilmArray Bin)</td><td rowspan=1 colspan=1>ND to&lt;103.5(ND)</td><td rowspan=1 colspan=1>103.5 to&lt;104.0(104)</td><td rowspan=1 colspan=1>104.0 to&lt;105.0(104 or 105)</td><td rowspan=1 colspan=1>105.0 to&lt;106.0(105 or 106)</td><td rowspan=1 colspan=1>10%.0 to&lt;107.0(106 or 107)</td><td rowspan=1 colspan=1>≥107.02107)</td></tr><tr><td rowspan=5 colspan=1>Rra  (e/do)1</td><td rowspan=1 colspan=1>ND</td><td rowspan=1 colspan=1>673</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>104</td><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>$10c5}$</td><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>106</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>≥107</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>26</td><td rowspan=1 colspan=1>19</td><td rowspan=1 colspan=1>18</td><td rowspan=1 colspan=1>6</td></tr><tr><td rowspan=2 colspan=2>% concordant</td><td rowspan=2 colspan=1>673/730(92.2%)</td><td rowspan=1 colspan=1>3/15 (20.0%)</td><td rowspan=1 colspan=1>7/42 (16.7%)</td><td rowspan=1 colspan=1>5/24 (20.8%)</td><td rowspan=1 colspan=1>19/19 (100%)</td><td rowspan=1 colspan=1>6/6 (100%)</td></tr><tr><td rowspan=1 colspan=5>40/106 (37.7%)</td></tr></table>

Table 78: FilmArray Pneumonia Panel S. aureus Bin Performance for BAL Specimens (qRefCx)   

<table><tr><td rowspan=1 colspan=8>BAL</td></tr><tr><td rowspan=1 colspan=2>qRefCx Range ofValues(CFU/mL)(Predicted FilmArray Bin)</td><td rowspan=1 colspan=1>ND to&lt;103.5(ND)</td><td rowspan=1 colspan=1>103.5 to&lt;104.0(104)</td><td rowspan=1 colspan=1>104.0 to&lt;105.0(104 or 105)</td><td rowspan=1 colspan=1>105.0 to&lt;10%.0(105 or 106)</td><td rowspan=1 colspan=1>106.0 to&lt;107.0(106 or 107)</td><td rowspan=1 colspan=1>≥107.0(≥107)</td></tr><tr><td rowspan=5 colspan=1>Rr   (Tc/!do)</td><td rowspan=1 colspan=1>ND</td><td rowspan=1 colspan=1>729</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>104</td><td rowspan=1 colspan=1>33</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>$10}$</td><td rowspan=1 colspan=1>23</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>106</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>≥107</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>3</td></tr><tr><td rowspan=2 colspan=2>% concordant</td><td rowspan=2 colspan=1>729/799(91.2%)</td><td rowspan=1 colspan=1>0/11 (0.0%)</td><td rowspan=1 colspan=1>0/19 (0.0%)</td><td rowspan=1 colspan=1>1/9 (11.1%)</td><td rowspan=1 colspan=1>5/5 (100%)</td><td rowspan=1 colspan=1>3/3 (100%)</td></tr><tr><td rowspan=1 colspan=5>9/47 (19.1%)</td></tr></table>

Table 79: FilmArray Pneumonia Panel S. aureus Bin Performance for Sputum Specimens (qRefCx)   

<table><tr><td rowspan=1 colspan=8>Sputum</td></tr><tr><td rowspan=1 colspan=2>qRefCx Range ofValues(CFU/mL)(Predicted FilmArray Bin)</td><td rowspan=1 colspan=1>ND to&lt;103.5(ND)</td><td rowspan=1 colspan=1>103.5 to&lt;104.0(104)</td><td rowspan=1 colspan=1>104.0 to&lt;105.0(104 or 105)</td><td rowspan=1 colspan=1>105.0 to&lt;10%.0(105 or 106)</td><td rowspan=1 colspan=1>106.0 to&lt;107.0(106 or 107)</td><td rowspan=1 colspan=1>≥107.0(≥107)</td></tr><tr><td rowspan=5 colspan=1>Fra   (/do)</td><td rowspan=1 colspan=1>ND</td><td rowspan=1 colspan=1>631</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>104</td><td rowspan=1 colspan=1>39</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>105}$</td><td rowspan=1 colspan=1>33</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>106</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>≥107</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>7</td></tr><tr><td rowspan=2 colspan=2>% concordant</td><td rowspan=2 colspan=1>631/724(87.2%)</td><td rowspan=1 colspan=1>1/18 (5.6%)</td><td rowspan=1 colspan=1>11/45 (24.4%)</td><td rowspan=1 colspan=1>13/28 (46.4%)</td><td rowspan=1 colspan=1>12/13 (92.3%)</td><td rowspan=1 colspan=1>7/8 (87.5%)</td></tr><tr><td rowspan=1 colspan=5>44/112 (39.3%)</td></tr></table>

Table 80: FilmArray Pneumonia Panel S. pneumoniae Bin Performance for BAL Specimens (qRefCx)   

<table><tr><td rowspan=1 colspan=5>BAL</td><td rowspan=1 colspan=3>cYanmaGellowb</td></tr><tr><td rowspan=1 colspan=2>qRefCx Range ofValues(CFU/mL)(Predicted FilmArray Bin)</td><td rowspan=1 colspan=1>ND to&lt;103.5(ND)</td><td rowspan=1 colspan=1>103.5 to&lt;104.0(104)</td><td rowspan=1 colspan=1>104.0 to&lt;105.0(104 or 105)</td><td rowspan=1 colspan=1>105.0 to&lt;106.0(105 or 106)</td><td rowspan=1 colspan=1>10%.0 to&lt;107.0(106 or 107)</td><td rowspan=1 colspan=1>≥107.0(≥107)</td></tr><tr><td rowspan=5 colspan=1>Ra   (Tc/!do)A</td><td rowspan=1 colspan=1>ND</td><td rowspan=1 colspan=1>817</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>104</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>$105}$</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>106</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>≥107</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=2 colspan=2>% concordant</td><td rowspan=2 colspan=1>817/841(97.1%)</td><td rowspan=1 colspan=1>1/2 (50.0%)</td><td rowspan=1 colspan=1>1/2 (50.0%)</td><td rowspan=1 colspan=1>0/1 (0.0%)</td><td rowspan=1 colspan=1>0/0 (-%)</td><td rowspan=1 colspan=1>0/0 (-%)</td></tr><tr><td rowspan=1 colspan=5>2/5 (40%)</td></tr></table>

Table 81: FilmArray Pneumonia Panel S. pneumoniae Bin Performance for Sputum Specimens (qRefCx)   

<table><tr><td rowspan=1 colspan=8>Sputum</td></tr><tr><td rowspan=1 colspan=2>qRefCx Range ofValues(CFU/mL)(Predicted FilmArray Bin)</td><td rowspan=1 colspan=1>ND to&lt;103.5(ND)</td><td rowspan=1 colspan=1>103.5 to&lt;104.0(104)</td><td rowspan=1 colspan=1>104.0 to&lt;105.0(104 or 105)</td><td rowspan=1 colspan=1>105.0 to&lt;10%.0(105 or 106)</td><td rowspan=1 colspan=1>106.0 to&lt;107.0(106 or 107)</td><td rowspan=1 colspan=1>≥107.0(≥107)</td></tr><tr><td rowspan=5 colspan=1>Fra   (/d)</td><td rowspan=1 colspan=1>ND</td><td rowspan=1 colspan=1>785</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>104</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>$10f5}$</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>106</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>≥107</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=2 colspan=2>% concordant</td><td rowspan=2 colspan=1>785/820(95.7%)</td><td rowspan=1 colspan=1>2/3 (66.7%)</td><td rowspan=1 colspan=1>2/6 (33.3%)</td><td rowspan=1 colspan=1>0/2 (0.0%)</td><td rowspan=1 colspan=1>4/4 (100%)</td><td rowspan=1 colspan=1>1/1 (100%)</td></tr><tr><td rowspan=1 colspan=5>9/16 (56.3%)</td></tr></table>

Table 82: FilmArray Pneumonia Panel Bin Performance Summary for BAL Specimens compared to qRefCx   

<table><tr><td colspan="7">BAL</td></tr><tr><td>qRefCx Range of Values (Concordant FilmArray</td><td>103.5 to &lt;104.0</td><td>104.0 to &lt;105.0</td><td>105.0 to &lt;106.0</td><td>106.0 to &lt;107.0</td><td>≥107.0 (≥107)</td><td>Overall</td></tr><tr><td>Bin)</td><td>(104)</td><td>(104 or 105)</td><td>(105 or 106)</td><td>(106 r 107)</td><td></td><td></td></tr><tr><td>Acinetobacter</td><td>0/0</td><td>0/0</td><td>0/0</td><td>0/0</td><td>0/0</td><td>0/0</td></tr><tr><td>calcoaceticus-</td><td>(-)</td><td>(-)</td><td>(-)</td><td>(-)</td><td>(-)</td><td>(-)</td></tr><tr><td>baumannii complex</td><td></td><td></td><td></td><td></td><td></td><td></td></tr><tr><td>Enterobacter cloacae</td><td>0/1</td><td>1/7</td><td>0/3</td><td>0/0</td><td>1/1</td><td>2/12</td></tr><tr><td>complex</td><td>(0%)</td><td>(14.3%)</td><td>(0%)</td><td>(-)</td><td>(100%)</td><td>(16.7%)</td></tr><tr><td>Escherichia coli</td><td>0/4</td><td>1/7</td><td>0/0</td><td>0/0</td><td>1/1</td><td>2/12</td></tr><tr><td>Haemophilus</td><td>(0%)</td><td>(14.3%)</td><td>(-)</td><td>(-)</td><td>(100%)</td><td>(16.7%)</td></tr><tr><td>influenzae</td><td>0/2</td><td>0/2 (0%)</td><td>0/5</td><td>0/0</td><td>1/1</td><td>1/10</td></tr><tr><td></td><td>(0%)</td><td>1/2</td><td>(0%) 0/1</td><td>() 2/3</td><td>(100%)</td><td>(10.0%)</td></tr><tr><td>Klebsiella aerogenes</td><td>0/1</td><td>(50.0%)</td><td>(0%)</td><td>(66.7%)</td><td>0/0</td><td>3/7</td></tr><tr><td rowspan="2">Klebsiella oxytoca</td><td>(0%)</td><td>1/2</td><td>0/0</td><td></td><td>(-)</td><td>(42.9%)</td></tr><tr><td>0/0</td><td>(50.0%)</td><td>(-)</td><td>0/0</td><td>0/0</td><td>1/2</td></tr><tr><td>Klebsiella pneumoniae</td><td>(-) 0/5</td><td>3/5</td><td>0/0</td><td>(-) 2/2</td><td>(-) 3/3</td><td>(50.0%)</td></tr><tr><td>group</td><td>(0%)</td><td>(60.0%)</td><td>(-)</td><td>(100%)</td><td>(100%)</td><td>8/15</td></tr><tr><td rowspan="2">Moraxella catarrhalis</td><td></td><td>0/0</td><td>0/0</td><td>0/0</td><td></td><td>(53.3%)</td></tr><tr><td>0/0</td><td></td><td></td><td></td><td>0/0</td><td>0/0</td></tr><tr><td rowspan="2">Proteus spp.</td><td>(-)</td><td>(-)</td><td>(-)</td><td>(-)</td><td>(-)</td><td>(-)</td></tr><tr><td>0/1</td><td>1/1</td><td>0/1</td><td>1/1</td><td>1/1</td><td>3/5</td></tr><tr><td></td><td>(0%)</td><td>(100%)</td><td>(0%)</td><td>(100%)</td><td>(100%)</td><td>(60.0%)</td></tr><tr><td>Pseudomonas</td><td>0/4</td><td>2/15</td><td>1/13</td><td>2/2 (100%)</td><td>2/2</td><td>7/36</td></tr><tr><td>aeruginosa</td><td>(0%)</td><td>(13.3%)</td><td>(7.7%)</td><td></td><td>(100%)</td><td>(19.4%)</td></tr><tr><td>Serratia marcescens</td><td>0/1</td><td>1/4</td><td>0/1</td><td>0/0</td><td>0/0</td><td>1/6</td></tr></table>

<table><tr><td colspan="8">BAL</td></tr><tr><td>Staphylococcus aureus</td><td>(0%) 0/11</td><td>(25.0%) 0/19</td><td>0%) 1/9</td><td>(-) 5/5</td><td>(-) 3/3</td><td>(16.7%) 9/47</td></tr><tr><td></td><td>(0.0%)</td><td>(0%)</td><td>(11.1%) 0/0</td><td>(100%) 0/0</td><td>(100%) 0/0</td><td>(19.1%)</td></tr><tr><td>Streptococcus</td><td>0/0</td><td>0/1 (0%)</td><td>(-)</td><td>(-)</td><td>(-)</td><td>0/1 (0%)</td></tr><tr><td>agalactiae</td><td>(-)</td><td>1/2</td><td>0/1</td><td>0/0</td><td>0/0</td><td>2/5</td></tr><tr><td>Streptococcus</td><td>1/2</td><td>(50.0%)</td><td>(0%)</td><td>(-)</td><td>(-)</td><td>(40.0%)</td></tr><tr><td>pneumoniae</td><td>(50.0%)</td><td>2/2</td><td>0/0</td><td>0/0</td><td>0/0</td><td>2/2</td></tr><tr><td>Streptococcus pyogenes</td><td>0/0 (-)</td><td>(100%)</td><td>(-)</td><td>(-)</td><td>(-)</td><td>(100%)</td></tr></table>

Table 83: FilmArray Pneumonia Panel Bin Performance Summary for Sputum Specimens compared to qRefCx   

<table><tr><td rowspan=1 colspan=7>Sputum</td></tr><tr><td rowspan=1 colspan=1>qRefCx Range of Values(Concordant FilmArrayBin)</td><td rowspan=1 colspan=1>103.5 to&lt;104.0(104)</td><td rowspan=1 colspan=1>104.0 to&lt;105.0(104 or 105)</td><td rowspan=1 colspan=1>105.0 to&lt;106.0(105 or 106)</td><td rowspan=1 colspan=1>106.0 to&lt;107.0(106 or 107)</td><td rowspan=1 colspan=1>≥107.0 (≥107)</td><td rowspan=1 colspan=1>Overall</td></tr><tr><td rowspan=1 colspan=1>Acinetobactercalcoaceticus-baumannii complex</td><td rowspan=1 colspan=1>1/5(20.0%)</td><td rowspan=1 colspan=1>1/1(100%)</td><td rowspan=1 colspan=1>0/2(0%)</td><td rowspan=1 colspan=1>3/3(100%)</td><td rowspan=1 colspan=1>0/0(-)</td><td rowspan=1 colspan=1>5/11(45.5%)</td></tr><tr><td rowspan=1 colspan=1>Enterobactercloacae complex</td><td rowspan=1 colspan=1>0/1(0%)</td><td rowspan=1 colspan=1>3/6(50.0%)</td><td rowspan=1 colspan=1>1/3(33.3%)</td><td rowspan=1 colspan=1>1/1(100%)</td><td rowspan=1 colspan=1>1/1(100%)</td><td rowspan=1 colspan=1>6/12(50.0%)</td></tr><tr><td rowspan=1 colspan=1>Escherichia coli</td><td rowspan=1 colspan=1>2/3(66.7%)</td><td rowspan=1 colspan=1>1/13(7.7%)</td><td rowspan=1 colspan=1>2/5(40.0%)</td><td rowspan=1 colspan=1>1/1(100%)</td><td rowspan=1 colspan=1>2/2(100%)</td><td rowspan=1 colspan=1>8/24(33.3%)</td></tr><tr><td rowspan=1 colspan=1>Haemophilusinfluenzae</td><td rowspan=1 colspan=1>0/0(-)</td><td rowspan=1 colspan=1>0/12(0%)</td><td rowspan=1 colspan=1>1/4(25.0%)</td><td rowspan=1 colspan=1>2/2(100%)</td><td rowspan=1 colspan=1>0/0(-)</td><td rowspan=1 colspan=1>3/18(16.7%)</td></tr><tr><td rowspan=2 colspan=1>Klebsiella aerogenes</td><td rowspan=2 colspan=1>1/1(100%)</td><td rowspan=2 colspan=1>0/1(0%)</td><td rowspan=2 colspan=1>0/1(0%)</td><td rowspan=1 colspan=1>1/1</td><td rowspan=1 colspan=1>0/0</td><td rowspan=2 colspan=1>2/4(50.0%)</td></tr><tr><td rowspan=1 colspan=1>(100%)</td><td rowspan=1 colspan=1>(-)</td></tr><tr><td rowspan=1 colspan=1>Klebsiella oxytoca</td><td rowspan=1 colspan=1>1/3(33.3%)</td><td rowspan=1 colspan=1>0/3(0%)</td><td rowspan=1 colspan=1>2/2(100%)</td><td rowspan=1 colspan=1>1/1(100%)</td><td rowspan=1 colspan=1>0/0(-)</td><td rowspan=1 colspan=1>4/9(44.4%)</td></tr><tr><td rowspan=2 colspan=1>Klebsiella pneumoniaegroup</td><td rowspan=2 colspan=1>3/6(50.0%)</td><td rowspan=2 colspan=1>2/7(28.6%)</td><td rowspan=1 colspan=1>5/7</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>0/1</td><td rowspan=1 colspan=1>12/23</td></tr><tr><td rowspan=1 colspan=1>(71.4%)</td><td rowspan=1 colspan=1>(100%)</td><td rowspan=1 colspan=1>(0%)</td><td rowspan=1 colspan=1>(52.2%)</td></tr><tr><td rowspan=1 colspan=1>Moraxella catarrhalis</td><td rowspan=1 colspan=1>0/1(0%)</td><td rowspan=1 colspan=1>0/1(0%)</td><td rowspan=1 colspan=1>0/1(0%)</td><td rowspan=1 colspan=1>1/1(100%)</td><td rowspan=1 colspan=1>1/1(100%)</td><td rowspan=1 colspan=1>2/5(40.0%)</td></tr><tr><td rowspan=1 colspan=1>Proteus spp.</td><td rowspan=1 colspan=1>0/1(0%)</td><td rowspan=1 colspan=1>5/10(50.0%)</td><td rowspan=1 colspan=1>2/3(66.7%)</td><td rowspan=1 colspan=1>1/1(100%)</td><td rowspan=1 colspan=1>0/0(-)</td><td rowspan=1 colspan=1>8/15(53.3%)</td></tr><tr><td rowspan=2 colspan=1>Pseudomonasaeruginosa</td><td rowspan=2 colspan=1>3/15(20.0%)</td><td rowspan=2 colspan=1>7/42(16.7%)</td><td rowspan=2 colspan=1>5/24(20.8%)</td><td rowspan=1 colspan=1>19/19</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>40/106</td></tr><tr><td rowspan=1 colspan=1>(100%)</td><td rowspan=1 colspan=1>(100%)</td><td rowspan=1 colspan=1>(37.7%)</td></tr><tr><td rowspan=1 colspan=1>Serratia marcescens</td><td rowspan=1 colspan=1>0/4(0%)</td><td rowspan=1 colspan=1>4/12(16.7%)</td><td rowspan=1 colspan=1>3/6(50.0%)</td><td rowspan=1 colspan=1>4/4(100%)</td><td rowspan=1 colspan=1>1/1(100%)</td><td rowspan=1 colspan=1>12/27(44.4%)</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus aureus</td><td rowspan=1 colspan=1>1/18(5.6%)</td><td rowspan=1 colspan=1>11/45(24.4%)</td><td rowspan=1 colspan=1>13/28(46.4%)</td><td rowspan=1 colspan=1>12/13(92.3%)</td><td rowspan=1 colspan=1>7/8(87.5%)</td><td rowspan=1 colspan=1>44/112(39.3%)</td></tr></table>

<table><tr><td colspan="8">Sputum</td></tr><tr><td>Streptococcus agalactiae Streptococcus</td><td>0/3 (0%)</td><td>1/5 (20.0%)</td><td>0/1 (0%)</td><td>0/0 (-)</td><td>0/0 (-)</td><td>1/9 (11.1%)</td></tr><tr><td>pneumoniae Streptococcus</td><td>2/3 (66.7%)</td><td>2/6 (33.3%) 0/3</td><td>0/2 (0%) 1/1</td><td>4/4 (100%) 1/1</td><td>1/1 (100%) 1/1</td><td>9/16 (56.3%)</td></tr><tr><td>pyogenes</td><td>0/0 (-)</td><td>(0%)</td><td>(100%)</td><td>(100%)</td><td>(100%)</td><td>3/6 (50.0%)</td></tr></table>

The FilmArray Pneumonia Panel bin performance compared to standard of care culture (SOC) comparator is shown for BAL (Table 85) and sputum (Table 86). For SOC, the various values that were provided by the source lab were converted to “Few/Mod/Many” using the following rules:

Table 84: SOC values provided to source lab   

<table><tr><td rowspan=1 colspan=1>Label Classification</td></tr><tr><td rowspan=1 colspan=1>Ln Fecoy  ae    100 000 1000 1000 00000,&#x27;3000&#x27;, 4000, &#x27;4000&#x27;, 5000, 5000&#x27;, 6000, 8000, 1-10000&#x27;, &lt;10000&#x27;, 1000-10000&#x27;, &#x27;lessthan10000&#x27;]</td></tr><tr><td rowspan=1 colspan=1>Labls inModerate:[3+, 2+, +(moderate), &#x27;moderate, &#x27;moderategrowth, totalmicrobialgrowthmoderate&#x27;, &#x27;mod,10000, 1000, &gt;10000, 1000,1000, 20000, 20000, 2000, 3000, 5000, 40000, 5000, 50000, 6000, 6500075000, &#x27;80000&#x27;]</td></tr><tr><td rowspan=1 colspan=1>Labels in Many: [&#x27;4+&#x27;, &#x27;many&#x27;, &#x27;heavygrowth&#x27;, &#x27;heavy&#x27;, &#x27;4+(many), 100000, &gt;100000&#x27;]</td></tr><tr><td rowspan=1 colspan=1>Discarded Labels: [1st&#x27;, 2nd&#x27;, 3rd, &#x27;secondmostgrowth, &#x27;thirdmostgrowth&#x27;, &#x27;reportedfirst(mostgrowth),osrowth&#x27;,mostgrowtho, econmostgrowtho3, leastgrowthofthe mostrowth;2traisdentifed, </td></tr></table>

Table 85: FilmArray Pneumonia Panel Bin Performance Summary for BAL Specimens compared to SOC   

<table><tr><td colspan="5">BAL</td></tr><tr><td>SOC Range of Values</td><td>Few</td><td>Moderate</td><td>Many (106 or ≥107)</td><td>Overall</td></tr><tr><td>(Concordant FilmArray Bin)</td><td>(104 or 105) 1/2</td><td>(105 or 106) 0/0</td><td>0/0</td><td>1/2</td></tr><tr><td>Acinetobacter calcoaceticus-baumannii complex</td><td>(50.0%)</td><td>(-)</td><td>(-)</td><td>(50.0%)</td></tr><tr><td></td><td>2/2</td><td>5/13</td><td>2/2</td><td>9/17</td></tr><tr><td>Enterobacter cloacae complex</td><td>(100%) 2/5</td><td>(38.5%) 2/8</td><td>(100%) 3/4</td><td>(52.9%) 7/17</td></tr><tr><td>Escherichia coli</td><td>(40.0%) 0/0</td><td>(25.0%) 4/16</td><td>(75.0%)</td><td>(41.2%)</td></tr><tr><td>Haemophilus influenzae</td><td>(-) 0/2</td><td>(25.0%)</td><td>9/9 (100%)</td><td>13/25 (52.0%)</td></tr><tr><td>Klebsiella aerogenes</td><td>(0%)</td><td>3/5 (60.0%)</td><td>2/4 (50.0%)</td><td>5/11 (45.5%)</td></tr><tr><td>Klebsiella oxytoca</td><td>2/4 (50.0%)</td><td>0/2 (0%)</td><td>0/1 (0%)</td><td>2/7 (28.6%)</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae group</td><td colspan="1" rowspan="1">7/10(70.0%)</td><td colspan="1" rowspan="1">4/5(80.0%)</td><td colspan="1" rowspan="1">4/4(100%)</td><td colspan="1" rowspan="1">15/19(78.9%)</td></tr><tr><td colspan="1" rowspan="1">Moraxella catarrhalis</td><td colspan="1" rowspan="1">0/0(-)</td><td colspan="1" rowspan="1">1/3(33.3%)</td><td colspan="1" rowspan="1">6/6(100%)</td><td colspan="1" rowspan="1">7/9(77.8%)</td></tr><tr><td colspan="1" rowspan="1">Proteus spp.</td><td colspan="1" rowspan="1">1/1(100%)</td><td colspan="1" rowspan="1">1/2(50.0%)</td><td colspan="1" rowspan="1">2/2(100%)</td><td colspan="1" rowspan="1">4/5(80.0%)</td></tr><tr><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">7/15(46.7%)</td><td colspan="1" rowspan="1">11/23(47.8%)</td><td colspan="1" rowspan="1">14/14(100%)</td><td colspan="1" rowspan="1">32/52(61.5%)</td></tr><tr><td colspan="1" rowspan="1">Serratia marcescens</td><td colspan="1" rowspan="1">0/0()</td><td colspan="1" rowspan="1">4/7(57.1%)</td><td colspan="1" rowspan="1">1/1(100%)</td><td colspan="1" rowspan="1">5/8(62.5%)</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">12/21(57.1%)</td><td colspan="1" rowspan="1">16/36(44.4%)</td><td colspan="1" rowspan="1">18/18(100%)</td><td colspan="1" rowspan="1">46/75(61.3%)</td></tr><tr><td colspan="1" rowspan="1">Streptococcus agalactiae a</td><td colspan="1" rowspan="1">0/0(-)</td><td colspan="1" rowspan="1">0/2(0%)</td><td colspan="1" rowspan="1">2/2(100%)</td><td colspan="1" rowspan="1">2/4(50.0%)</td></tr><tr><td colspan="1" rowspan="1">Streptococcus pneumoniae</td><td colspan="1" rowspan="1">3/5(60.0%)</td><td colspan="1" rowspan="1">5/5(100%)</td><td colspan="1" rowspan="1">3/3(100%)</td><td colspan="1" rowspan="1">11/13(84.6%)</td></tr><tr><td colspan="1" rowspan="1">Streptococcus pyogenes</td><td colspan="1" rowspan="1">2/2(100%)</td><td colspan="1" rowspan="1">2/3(66.7%)</td><td colspan="1" rowspan="1">0/0()</td><td colspan="1" rowspan="1">4/5(80.0%)</td></tr></table>

a Semi-quantitative information not provided for one specimen in which S. agalactiae was isolated by SOC culture

Table 86: FilmArray Pneumonia Panel Bin Performance Summary for Sputum Specimens compared to SOC   

<table><tr><td rowspan=1 colspan=5>Sputum</td></tr><tr><td rowspan=1 colspan=1>SOC Range of Values(Concordant FilmArray Bin)</td><td rowspan=1 colspan=1>Few(104 or 105)</td><td rowspan=1 colspan=1>Moderate(105 or 10)</td><td rowspan=1 colspan=1>Many(106 or≥107)</td><td rowspan=1 colspan=1>Overall</td></tr><tr><td rowspan=1 colspan=1>Acinetobacter calcoaceticus-baumanniicomplex</td><td rowspan=1 colspan=1>4/6(66.7%)</td><td rowspan=1 colspan=1>1/4(25.0%)</td><td rowspan=1 colspan=1>2/3(66.7%)</td><td rowspan=1 colspan=1>7/13(53.8%)</td></tr><tr><td rowspan=1 colspan=1>Enterobacter cloacae complex</td><td rowspan=1 colspan=1>1/2(50.0%)</td><td rowspan=1 colspan=1>4/8(50.0%)</td><td rowspan=1 colspan=1>1/2(50.0%)</td><td rowspan=1 colspan=1>6/12(50.0%)</td></tr><tr><td rowspan=1 colspan=1>Escherichia coli</td><td rowspan=1 colspan=1>4/8(50.0%)</td><td rowspan=1 colspan=1>1/4(25.0%)</td><td rowspan=1 colspan=1>5/5(100%)</td><td rowspan=1 colspan=1>10/17(58.8%)</td></tr><tr><td rowspan=1 colspan=1>Haemophilus influenzae</td><td rowspan=1 colspan=1>2/3(66.7%)</td><td rowspan=1 colspan=1>1/0(10.0%)</td><td rowspan=1 colspan=1>15/16(93.8%)</td><td rowspan=1 colspan=1>18/29(62.1%)</td></tr><tr><td rowspan=1 colspan=1>Klebsiella aerogenes</td><td rowspan=1 colspan=1>2/2(100%)</td><td rowspan=1 colspan=1>0/3(0%)</td><td rowspan=1 colspan=1>0/0(-)</td><td rowspan=1 colspan=1>2/5(40.0%)</td></tr><tr><td rowspan=1 colspan=1>Klebsiella oxytoca</td><td rowspan=1 colspan=1>0/3(0%)</td><td rowspan=1 colspan=1>3/3(100%)</td><td rowspan=1 colspan=1>0/0(-)</td><td rowspan=1 colspan=1>3/6(50.0%)</td></tr><tr><td rowspan=1 colspan=1>Klebsiella pneumoniae group</td><td rowspan=1 colspan=1>10/14(71.4%)</td><td rowspan=1 colspan=1>4/13(30.8%)</td><td rowspan=1 colspan=1>1/1(100%)</td><td rowspan=1 colspan=1>15/28(53.6%)</td></tr><tr><td rowspan=1 colspan=1>Moraxella catarrhalis</td><td rowspan=1 colspan=1>1/3(33.3%)</td><td rowspan=1 colspan=1>1/5(20.0%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>17/23(73.9%)</td></tr><tr><td rowspan=1 colspan=1>Proteus spp.</td><td rowspan=1 colspan=1>0/2(0%)</td><td rowspan=1 colspan=1>2/3(66.7%)</td><td rowspan=1 colspan=1>0/1(0%)</td><td rowspan=1 colspan=1>2/6(33.3%)</td></tr><tr><td rowspan=1 colspan=1>Pseudomonas aeruginosa</td><td rowspan=1 colspan=1>14/37(37.8%)</td><td rowspan=1 colspan=1>9/41(22.0%)</td><td rowspan=1 colspan=1>22/25(88.0%)</td><td rowspan=1 colspan=1>45/103(43.7%)</td></tr><tr><td rowspan=1 colspan=1>Serratia marcescens</td><td rowspan=1 colspan=1>6/10(60.0%)</td><td rowspan=1 colspan=1>6/11(54.5%)</td><td rowspan=1 colspan=1>4/4(100%)</td><td rowspan=1 colspan=1>16/25(64.0%)</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus aureus</td><td rowspan=1 colspan=1>27/41(65.9%)</td><td rowspan=1 colspan=1>19/41(46.3%)</td><td rowspan=1 colspan=1>31/35(88.6%)</td><td rowspan=1 colspan=1>77/117(65.8%)</td></tr><tr><td rowspan=1 colspan=1>Streptococcus agalactiaea</td><td rowspan=1 colspan=1>2/3(6.7%)</td><td rowspan=1 colspan=1>0/1(0%)</td><td rowspan=1 colspan=1>4/4(100%)</td><td rowspan=1 colspan=1>6/8(75.0%)</td></tr><tr><td rowspan=1 colspan=1>Streptococcus pneumoniae</td><td rowspan=1 colspan=1>0/1(0%)</td><td rowspan=1 colspan=1>3/7(42.9%)</td><td rowspan=1 colspan=1>1/2(50.0%)</td><td rowspan=1 colspan=1>4/10(40.0%)</td></tr><tr><td rowspan=1 colspan=1>Streptococcus pyogenes</td><td rowspan=1 colspan=1>0/1(0%)</td><td rowspan=1 colspan=1>0/2(0%)</td><td rowspan=1 colspan=1>2/3(66.7%)</td><td rowspan=1 colspan=1>2/6(33.3%)</td></tr></table>

a. Semi-quantitative information not provided for two specimens in which $A$ . calcoaceticus-baumannii complex was isolated by SOC b. Semi-quantitative information not provided for one specimen in which $E$ . cloacae complex was isolated by SOC culture c. Semi-quantitative information not provided for five specimens in which $E$ . coli was isolated by SOC culture d. Semi-quantitative information not provided for three specimens in which $H .$ influenzae was isolated by SOC culture e. Semi-quantitative information not provided for two specimens in which $K .$ . aerogenes was isolated by SOC culture f. Semi-quantitative information not provided for two specimens in which $K .$ . oxytoca was isolated by SOC culture g. Semi-quantitative information not provided for five specimens in which $K$ . pneumoniae group was isolated by SOC culture h. Semi-quantitative information not provided for one specimen in which M. catarrhalis was isolated by SOC culture i. Semi-quantitative information not provided for one specimen in which Proteus spp. was isolated by SOC culture j. Semi-quantitative information not provided for 19 specimens in which $P$ . aeruginosa was isolated by SOC culture k. Semi-quantitative information not provided for four specimens in which S. marcescens was isolated by SOC culture

A ‘rank order’ analysis was performed on polymicrobial specimens to compare the relative abundance of each analyte within a specimen as reported by qRefCx to the order reported by the FilmArray Pneumonia Panel. In this analysis, there were 20 specimens with two or more organisms reported by qRefCx for BAL and 84 for sputum. In these specimens, the FilmArray Pneumonia Panel was in agreement with qRefCx for the most abundant organism $4 5 . 0 \%$ of the time (9/20) for BAL and $4 1 . 7 \%$ of the time (35/84) for sputum. For the second most abundant organism, the FilmArray Pneumonia Panel was in agreement with qRefCx $3 0 . 0 \%$ of the time (6/20) for BAL and $2 6 . 2 \%$ of the time (22/84) for sputum and was in agreement for the third most abundant organism $7 . 7 \%$ of the time (1/13) for BAL and $3 . 8 \%$ of the time (2/52) for sputum.

Table 87: Concordance of Abundance in Polymicrobial Specimens (as compared to qRefCx)   

<table><tr><td rowspan=2 colspan=1>Ranking Performance</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>BAL</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Sputum</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Correct</td><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>Correct</td><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>%</td></tr><tr><td rowspan=1 colspan=1>Most Abundant</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>45.0%</td><td rowspan=1 colspan=1>35</td><td rowspan=1 colspan=1>84</td><td rowspan=1 colspan=1>41.7%</td></tr><tr><td rowspan=1 colspan=1>Second Most Abundant</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>30.0%</td><td rowspan=1 colspan=1>22</td><td rowspan=1 colspan=1>84</td><td rowspan=1 colspan=1>26.2%</td></tr><tr><td rowspan=1 colspan=1>Third Most Abundant</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>7.7%</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>52</td><td rowspan=1 colspan=1>3.8%</td></tr></table>

‘Rank concordance’ analysis was performed on polymicrobial specimens to determine the ability of the FilmArray Pneumonia Panel to measure relative abundance of the nucleic acid for an organism with respect to other organisms in the specimen, as compared to qRefCx. In this analysis, the detected organisms from individual polymicrobial specimens (110 BAL and 246 sputum) were ranked in descending order based on their quantification values from qRefCx. The rank determined by the FilmArray Pneumonia Panel bin result was compared to the qRefCx ranking. False positives results were always considered discordant (i.e. only exact rank matches are considered concordant).

Table 88: Concordance of Organism Ranking in Polymicrobial Specimens (as compared to qRefCx); false positive results considered discordant   

<table><tr><td colspan="1" rowspan="2">Ranking Performance</td><td colspan="3" rowspan="1">BAL</td><td colspan="3" rowspan="1">Sputum</td></tr><tr><td colspan="1" rowspan="1">Concordant</td><td colspan="1" rowspan="1">Total</td><td colspan="1" rowspan="1">%</td><td colspan="1" rowspan="1">Concordant</td><td colspan="1" rowspan="1">Total</td><td colspan="1" rowspan="1">%</td></tr><tr><td colspan="1" rowspan="1">Acinetobacter calcoaceticus-baumannii complex</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">16.7%</td><td colspan="1" rowspan="1">7</td><td colspan="1" rowspan="1">25</td><td colspan="1" rowspan="1">28.0%</td></tr><tr><td colspan="1" rowspan="1">Enterobacter aerogenes</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">9</td><td colspan="1" rowspan="1">44.4%</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">9</td><td colspan="1" rowspan="1">44.4%</td></tr><tr><td colspan="1" rowspan="1">Enterobacter cloacae complex</td><td colspan="1" rowspan="1">9</td><td colspan="1" rowspan="1">14</td><td colspan="1" rowspan="1">64.3%</td><td colspan="1" rowspan="1">9</td><td colspan="1" rowspan="1">25</td><td colspan="1" rowspan="1">36.0%</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">13</td><td colspan="1" rowspan="1">46.2%</td><td colspan="1" rowspan="1">14</td><td colspan="1" rowspan="1">39</td><td colspan="1" rowspan="1">35.9%</td></tr><tr><td colspan="1" rowspan="1">Haemophilus influenzae</td><td colspan="1" rowspan="1">9</td><td colspan="1" rowspan="1">47</td><td colspan="1" rowspan="1">19.1%</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">62</td><td colspan="1" rowspan="1">16.1%</td></tr><tr><td colspan="1" rowspan="1">Klebsiella oxytoca</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">30.0%</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">17</td><td colspan="1" rowspan="1">29.4%</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae group</td><td colspan="1" rowspan="1">8</td><td colspan="1" rowspan="1">16</td><td colspan="1" rowspan="1">50.0%</td><td colspan="1" rowspan="1">15</td><td colspan="1" rowspan="1">43</td><td colspan="1" rowspan="1">34.9%</td></tr><tr><td colspan="1" rowspan="1">Moraxella catarrhalis</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">15</td><td colspan="1" rowspan="1">0.0%</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">59</td><td colspan="1" rowspan="1">5.1%</td></tr><tr><td colspan="1" rowspan="1">Proteus spp.</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">33.3%</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">25.0%</td></tr><tr><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">17</td><td colspan="1" rowspan="1">25</td><td colspan="1" rowspan="1">68.0%</td><td colspan="1" rowspan="1">59</td><td colspan="1" rowspan="1">107</td><td colspan="1" rowspan="1">55.1%</td></tr><tr><td colspan="1" rowspan="1">Serratia marcescens</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">50.0%</td><td colspan="1" rowspan="1">17</td><td colspan="1" rowspan="1">43</td><td colspan="1" rowspan="1">39.5%</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">30</td><td colspan="1" rowspan="1">55</td><td colspan="1" rowspan="1">54.5%</td><td colspan="1" rowspan="1">59</td><td colspan="1" rowspan="1">141</td><td colspan="1" rowspan="1">41.8%</td></tr><tr><td colspan="1" rowspan="1">Streptococcus agalactiae</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">19</td><td colspan="1" rowspan="1">31.6%</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">35</td><td colspan="1" rowspan="1">28.6%</td></tr><tr><td colspan="1" rowspan="1">Streptococcus pneumoniae</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">23</td><td colspan="1" rowspan="1">17.4%</td><td colspan="1" rowspan="1">7</td><td colspan="1" rowspan="1">37</td><td colspan="1" rowspan="1">18.9%</td></tr><tr><td colspan="1" rowspan="1">Streptococcus pyogenes</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">20.0%</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">40.0%</td></tr></table>

The overall success rate for initial specimen tests in the prospective study was $9 8 . 1 \%$ (1764/1798); 34 tests were unsuccessful (two due to an incomplete test and 32 due to control failures). Two tests (2/1798; $0 . 1 \% )$ did not complete on the initial run, resulting in an instrument success rate of $9 9 . 9 \%$ (1796/1798) for initial specimen tests. Both specimens were able to be retested and valid results were produced after a single retest.

Thirty-two (32) tests (32/1764; $1 . 8 \%$ ) did not produce valid pouch controls, resulting in a pouch control success rate of $9 8 . 2 \%$ (1764/1796) for completed runs in the initial specimen tests. Twenty-eight (28) of the 32 invalid specimens were able to be retested; 25 produced valid control results after a single retest, while the remaining three did not produce valid control results after retesting and were not able to be retested further due to insufficient specimen volume; four were not able to be retested at all due to insufficient specimen volume.

Two additional studies were also conducted to demonstrate all aspects of clinical performance (see Testing of Preselected Archived Specimens and Testing of Contrived Specimens below).

# Testing of Preselected Archived Specimens

Some of the analytes on the FilmArray Pneumonia Panel were of low prevalence and were not encountered in large enough numbers during the prospective study to adequately demonstrate system performance. To supplement the results of the prospective clinical study, an evaluation of preselected archived retrospective specimens was performed at BioFire.

A total of 171 frozen archived specimens were received for testing from external laboratories. Eighteen (18) specimens were negatives (13 BAL and 5 sputum) and 153 specimens (139 BAL and 14 sputum) contained at least one analyte of interest. Twenty-two (22) specimens contained two or more analytes of interest. The set included specimens known to be positive for: Acinetobacter calcoaceticus-baumannii complex, Klebsiella aerogenes, Klebsiella oxytoca, Proteus spp., Serratia marcescens, Streptococcus pyogenes, Chlamydia pneumoniae, Legionella pneumophila, adenovirus, human metapneumovirus, influenza A, influenza B, parainfluenza virus (PIV), respiratory syncytial virus, various gram-negative bacteria with extended-spectrum $\beta$ -lactamase (ESBL) phenotype, and various gram-negative bacteria with a carbapenem resistant phenotype.

Prior to testing with the FilmArray Pneumonia Panel, the composition/integrity of the specimens was first confirmed with confirmatory molecular methods. At the completion of testing, four specimens were excluded due to invalid confirmation tests; these specimens had insufficient volume for retesting. Results from the remaining 18 negative and 149 positive specimens (containing 173 analytes) are presented in the Tables below.

The reported analyte (as determined by the source laboratory) was confirmed for 17 analytes (107 in BAL and 10 in sputum) of the expected positive results (117/173; $6 7 . 6 \%$ ) of the expected positive results $( 1 1 6 / 1 7 3 ; 6 7 . 1 \% )$ . More than three quarters of the unconfirmed analytes were specimens previously identified as positive for gram-negative bacteria exhibiting phenotypic ESBL or carbapenamase activity $( 4 4 / 5 7 ; 7 7 . 2 \% )$ . This is expected since this phenotypic activity can be conferred by alternative mechanisms beyond the antibiotic resistance genes found on the FilmArray Pneumonia Panel. Specimens with unconfirmed (or unexpected) analytes were excluded from performance calculations for that particular analyte and FilmArray test results are reported separately.

The FilmArray Pneumonia Panel demonstrated positive percent agreement (PPA) of $100 \%$ with previous laboratory results for 11 of 14 analytes tested in BAL specimens (Table 89) and all of the analytes tested in sputum specimens (Table 90). The exceptions were Klebsiella aerogenes, Proteus spp. and RSV with a PPA of $5 0 . 0 \%$ , $8 0 . 0 \%$ , and $9 3 . 8 \%$ respectively, due to one false negative (FN) for each analyte. While PPA for most analytes was $100 \%$ , an insufficient number of specimens were tested for all but influenza A, parainfluenza virus, and respiratory syncytial virus in BAL. Contrived specimen testing was used to demonstrate performance for these analytes in an additional contrived study (see Testing of Contrived Specimens). Negative percent agreement (NPA) was $100 \%$ for all analytes except parainfluenza virus in BAL and influenza A in sputum; however, NPA was more thoroughly evaluated in the prospective clinical study.

Table 89: FilmArray Pneumonia Panel Archived BAL Specimen Performance Data Summary   

<table><tr><td colspan="1" rowspan="2">Analyte</td><td colspan="3" rowspan="1">PPA</td><td colspan="3" rowspan="1">NPA</td></tr><tr><td colspan="1" rowspan="1">TP/(TP + FN)</td><td colspan="1" rowspan="1">%</td><td colspan="1" rowspan="1">95% CI</td><td colspan="1" rowspan="1">TN/(TN + FP)</td><td colspan="1" rowspan="1">%</td><td colspan="1" rowspan="1">95% CI</td></tr><tr><td colspan="7" rowspan="1">Quantitative Bacteria</td></tr><tr><td colspan="1" rowspan="1">Acinetobacter calcoaceticus-baumanniicomplex</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">51.0-100%</td><td colspan="1" rowspan="1">53/53</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">93.2-100%</td></tr><tr><td colspan="1" rowspan="1">Klebsiella aerogenes</td><td colspan="1" rowspan="1">1/2</td><td colspan="1" rowspan="1">50.0</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">53/53</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">93.2-100%</td></tr><tr><td colspan="1" rowspan="1">Proteus spp.</td><td colspan="1" rowspan="1">4/5</td><td colspan="1" rowspan="1">80.0</td><td colspan="1" rowspan="1">37.6-96.4%</td><td colspan="1" rowspan="1">48/48</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">92.6-100%</td></tr><tr><td colspan="1" rowspan="1">Serratia marcescens</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">72.2-100%</td><td colspan="1" rowspan="1">46/46</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">92.3-100%</td></tr><tr><td colspan="1" rowspan="1">Streptococcus pyogenes</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">57/57</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">93.7-100%</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="3" rowspan="1">Antimicrobial Resistance Genes</td><td colspan="3" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">CTX-M</td><td colspan="1" rowspan="1">7/7</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">64.6-100%</td><td colspan="1" rowspan="1">29/29</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">88.3-100%</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="6" rowspan="1">Atypical Bacteria</td></tr><tr><td colspan="1" rowspan="1">Chlamydia pneumoniae</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">90/90</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">95.9-100%</td></tr><tr><td colspan="1" rowspan="1">Legionella pneumophila</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">57/57</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">93.7-100%</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="6" rowspan="1">Viruses</td></tr><tr><td colspan="1" rowspan="1">Adenovirus</td><td colspan="1" rowspan="1">8/8</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">67.6-100%</td><td colspan="1" rowspan="1">81/81</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">95.5-100%</td></tr><tr><td colspan="1" rowspan="1">Human Metapneumovirus</td><td colspan="1" rowspan="1">11/11</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">74.1-100%</td><td colspan="1" rowspan="1">77/77</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">95.2-100%</td></tr><tr><td colspan="1" rowspan="1">Influenza A</td><td colspan="1" rowspan="1">21/21</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">84.5-100%</td><td colspan="1" rowspan="1">69/69</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">94.7-100%</td></tr><tr><td colspan="1" rowspan="1">Influenza B</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">43.9-100%</td><td colspan="1" rowspan="1">86/86</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">95.7-100%</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza Virus</td><td colspan="1" rowspan="1">17/17</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">81.6-100%</td><td colspan="1" rowspan="1">68/69</td><td colspan="1" rowspan="1">99.0</td><td colspan="1" rowspan="1">92.2-99.7%</td></tr><tr><td colspan="1" rowspan="1">Respiratory Syncytial Virus</td><td colspan="1" rowspan="1">15/16</td><td colspan="1" rowspan="1">93.8</td><td colspan="1" rowspan="1">71.7-98.9%</td><td colspan="1" rowspan="1">74/74</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">95.1-100%</td></tr></table>

Table 90: FilmArray Pneumonia Panel Archived Sputum Specimen Performance Data Summary   

<table><tr><td rowspan=2 colspan=1>Analyte</td><td rowspan=1 colspan=3>PPA</td><td rowspan=1 colspan=3>NPA</td></tr><tr><td rowspan=1 colspan=1>TP/(TP + FN)</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=1>TN/(TN + FP)</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=1>Quantitative Bacteria</td><td rowspan=1 colspan=1>cYanmaGaYellowb</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Streptococcus pyogenes</td><td rowspan=1 colspan=1>7/7</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>64.6-100%</td><td rowspan=1 colspan=1>8/8</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>67.6-100%</td></tr><tr><td rowspan=1 colspan=7>Antimicrobial Resistance Genes</td></tr><tr><td rowspan=1 colspan=1>CTX-M</td><td rowspan=1 colspan=1>1/1</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>12/12</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>75.8-100%</td></tr><tr><td rowspan=1 colspan=7>Viruses</td></tr><tr><td rowspan=1 colspan=1>Influenza A</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>34.2-100%</td><td rowspan=1 colspan=1>0/1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>-</td></tr></table>

# Testing of Contrived Specimens

A prospective clinical evaluation of the FilmArray Pneumonia Panel was performed during the 2016- 2017 respiratory infection season at several geographically diverse clinical laboratories. Over 1600 specimens were analyzed from subjects whose specimens were submitted for microbial evaluation of lower respiratory tract pathogens. In the prospective study, some analytes were of insufficient prevalence to adequately demonstrate system performance and additional archived, preselected positive specimens containing rare analytes were also tested. Several analytes were so rare that both prospective and archived testing efforts were insufficient to demonstrate system performance. In this study, contrived clinical specimens were created to evaluate the sensitivity and specificity of the FilmArray Pneumonia Panel assays for these rare analytes (Table 91).

Table 91: Contrived Clinical Specimen Analytes   

<table><tr><td rowspan=2 colspan=1>Analyte</td><td rowspan=1 colspan=2>Matrix</td></tr><tr><td rowspan=1 colspan=1>BAL</td><td rowspan=1 colspan=1>Sputum</td></tr><tr><td rowspan=1 colspan=3>Bacteria</td></tr><tr><td rowspan=1 colspan=1>Acinetobacter calcoaceticus-baumanniicomplex</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Klebsiella aerogenes</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td></tr><tr><td rowspan=1 colspan=1>Klebsiella oxytoca</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Proteus spp.</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Serratia marcescens</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Streptococcus pyogenes</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td></tr><tr><td rowspan=1 colspan=3>Atypical Bacteria</td></tr><tr><td rowspan=1 colspan=1>Chlamydia pneumoniae</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td></tr><tr><td rowspan=1 colspan=1>Legionella pneumophila</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td></tr><tr><td rowspan=1 colspan=1>Mycoplasma pneumoniae</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td></tr><tr><td rowspan=1 colspan=1>Viruses</td><td rowspan=1 colspan=2>cYanmaGllowb</td></tr><tr><td rowspan=1 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td></tr><tr><td rowspan=1 colspan=1>Human Metapneumovirus</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td></tr><tr><td rowspan=1 colspan=1>Influenza A</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td></tr><tr><td rowspan=1 colspan=1>Influenza B</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td></tr><tr><td rowspan=1 colspan=2>Antibiotic Resistance Markers</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>CTX-M</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td></tr><tr><td rowspan=1 colspan=1>IMP</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td></tr><tr><td rowspan=1 colspan=1>KPC</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td></tr><tr><td rowspan=1 colspan=1>NDM</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td></tr><tr><td rowspan=1 colspan=1>OXA-48-like</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td></tr><tr><td rowspan=1 colspan=1>VIM</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td></tr></table>

Contrived specimens $\scriptstyle ( \mathrm { N = 1 } 1 2 5 )$ ) were spiked using residual clinical samples that were pre-screened with the FilmArray Pneumonia Panel and found to be negative for the analytes of interest. Specimens were spiked with a variety of different isolates/strains for each organism at concentrations spanning observed ranges in clinical specimens. Different isolates of organisms were used from those used in analytical testing when possible. Samples positive for one analyte served as negatives for other analytes.

For the majority of analytes reported qualitatively, at least 25 of the contrived positive specimens had analyte concentrations at $2 \times$ the limit of detection (LoD), while the remaining specimens were tested at additional concentrations that spanned clinically observed ranges. The “clinically observed range” was based on data from previous FilmArray Pneumonia Panel positive test results (e.g. observations from the prospective or archived studies). If a clinically observed range could not be determined for a particular analyte, specimens were spiked at various factors of LoD. If the stock concentration of organism did not allow for spiking at the highest level, the highest achievable level was used. For bacteria reported with binned values, specimens were spiked at various concentrations starting just below and then spanning the reported levels (i.e. $1 0 ^ { 3 }$ to ${ \geq } 1 0 ^ { 7 }$ copies $/ \mathrm { m L }$ ).

Specimens were prepared and randomized at BioFire such that the analyte status of each contrived specimen was unknown to the users performing the testing. BAL and sputum specimens were analyzed separately; however, the preparation and testing for both matrices were identical. Contrived specimens were frozen, then distributed to prospective study sites and tested according to the prospective clinical study protocol alongside clinical (non-contrived) specimens.

The positive percent agreement (PPA) and negative percent agreement (NPA) for the FilmArray Pneumonia Panel assays were determined using standard binomial sampling statistics. In this study, a success was defined as agreement between the known composition of the contrived specimen and the FilmArray Pneumonia Panel result; i.e., a positive FilmArray Pneumonia Panel result for spiked samples (True Positive, TP) and a negative FilmArray Pneumonia Panel result for un-spiked samples (True Negative, TN).

The results of the 1125 specimens tested in this study are summarized in Table 92 for BAL and Table 93 for sputum below.

The majority of analytes in both specimen types met the performance goals of $90 \%$ PPA with an $80 \%$ lower bound of the $9 5 \%$ CI and $9 8 \%$ NPA with a $9 5 \%$ lower bound of the $9 5 \%$ CI. The exceptions being influenza A spiked into BAL and Klebsiella aerogenes spiked into BAL and sputum. Influenza A spiked in BAL demonstrated $8 6 \%$ PPA in part due to two missed detections at $0 . 2 \times \mathrm { L o D }$ and two additional missed detections at $2 \times \mathrm { L o D }$ from a strain that may have been under-quantified. However, FilmArray performance goals for influenza A in BAL were achieved in the archived study. Klebsiella aerogenes spiked in both sample types demonstrated $8 5 . 5 \%$ PPA in part due to five missed detections in BAL and four missed detections in sputum at a range of concentrations from a Klebsiella aerogenes strain that demonstrated poor reactivity with the FilmArray Pneumonia Panel.

Samples spiked with bacterial analytes just below the $1 0 \land 4$ copies/mL binned value (i.e., near $1 . 0 0 \mathrm { E } { + 0 3 }$ ), and samples spiked with other analytes below their LoD (i.e., near $0 . 2 \times \mathrm { L o D } \mathrm { , }$ ), produced the expected unreliable detection.

Table 92: FilmArray Pneumonia Panel Performance of Contrived BAL Specimens   

<table><tr><td rowspan=2 colspan=1>Analyte</td><td rowspan=1 colspan=4>Sensitivity/PPA</td><td rowspan=1 colspan=5>Specificity/NPA</td></tr><tr><td rowspan=1 colspan=1>TP/(TP + FN)</td><td rowspan=1 colspan=2>%</td><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=2>TN/(TN + FP)</td><td rowspan=1 colspan=2>%</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=3>Bacteria</td><td></td><td></td><td></td><td></td><td></td></tr><tr><td rowspan=1 colspan=1>Acinetobacter calcoaceticus-baumannii complex</td><td rowspan=1 colspan=1>47/50</td><td rowspan=1 colspan=2>94.0</td><td rowspan=1 colspan=1>83.8-97.9%</td><td rowspan=1 colspan=2>598/598</td><td rowspan=1 colspan=2>100</td><td rowspan=1 colspan=1>99.4-100%</td></tr><tr><td rowspan=1 colspan=1>Klebsiella aerogenes a</td><td rowspan=1 colspan=1>47/55</td><td rowspan=1 colspan=2>85.5</td><td rowspan=1 colspan=1>73.8-92.4%</td><td rowspan=1 colspan=2>592/594</td><td rowspan=1 colspan=2>99.7</td><td rowspan=1 colspan=1>98.8-99.9%</td></tr><tr><td rowspan=1 colspan=1>Klebsiella oxytoca</td><td rowspan=1 colspan=1>46/50</td><td rowspan=1 colspan=2>92.0</td><td rowspan=1 colspan=1>81.2-96.8%</td><td rowspan=1 colspan=2>604/604</td><td rowspan=1 colspan=2>100</td><td rowspan=1 colspan=1>99.4-100%</td></tr><tr><td rowspan=1 colspan=1>Proteus spp.</td><td rowspan=1 colspan=1>48/50</td><td rowspan=1 colspan=2>96.0</td><td rowspan=1 colspan=1>86.5-98.9%</td><td rowspan=1 colspan=2>603/603</td><td rowspan=1 colspan=2>100</td><td rowspan=1 colspan=1>99.4-100%</td></tr><tr><td rowspan=1 colspan=1>Serratia marcescens</td><td rowspan=1 colspan=1>49/50</td><td rowspan=1 colspan=2>98.0</td><td rowspan=1 colspan=1>89.5-99.6%</td><td rowspan=1 colspan=2>604/604</td><td rowspan=1 colspan=2>100</td><td rowspan=1 colspan=1>99.4-100%</td></tr><tr><td rowspan=1 colspan=1>Streptococcus pyogenes</td><td rowspan=1 colspan=1>49/50</td><td rowspan=1 colspan=2>98.0</td><td rowspan=1 colspan=1>89.5-99.6%</td><td rowspan=1 colspan=2>597/597</td><td rowspan=1 colspan=2>100</td><td rowspan=1 colspan=1>99.4-100%</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2>Atypical</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2></td><td rowspan=1 colspan=2></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Chlamydia pneumoniae</td><td rowspan=1 colspan=1>47/50</td><td rowspan=1 colspan=2>94.0</td><td rowspan=1 colspan=1>83.8-97.9%</td><td rowspan=1 colspan=2>604/604</td><td rowspan=1 colspan=2>100</td><td rowspan=1 colspan=1>99.4-100%</td></tr><tr><td rowspan=1 colspan=1>Legionella pneumophila</td><td rowspan=1 colspan=1>50/50</td><td rowspan=1 colspan=2>100</td><td rowspan=1 colspan=1>92.9-100%</td><td rowspan=1 colspan=2>599/599</td><td rowspan=1 colspan=2>100</td><td rowspan=1 colspan=1>99.4-100%</td></tr><tr><td rowspan=1 colspan=1>Mycoplasma pneumoniae</td><td rowspan=1 colspan=1>48/50</td><td rowspan=1 colspan=2>96.0</td><td rowspan=1 colspan=1>86.5-98.9%</td><td rowspan=1 colspan=2>603/604</td><td rowspan=1 colspan=2>99.8</td><td rowspan=1 colspan=1>99.1-100%</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=8>Viruses</td></tr><tr><td rowspan=1 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>50/53</td><td rowspan=1 colspan=2>94.3</td><td rowspan=1 colspan=1>84.6-98.1%</td><td rowspan=1 colspan=1>568/569</td><td rowspan=1 colspan=2>99.8</td><td rowspan=1 colspan=2>99.0-100%</td></tr><tr><td rowspan=1 colspan=1>Human Metapneumovirus</td><td rowspan=1 colspan=1>50/50</td><td rowspan=1 colspan=2>100</td><td rowspan=1 colspan=1>92.9-100%</td><td rowspan=1 colspan=1>597/598</td><td rowspan=1 colspan=2>99.8</td><td rowspan=1 colspan=2>99.1-100%</td></tr><tr><td rowspan=1 colspan=1>Influenza Ab</td><td rowspan=1 colspan=1>43/50</td><td rowspan=1 colspan=2>86.0</td><td rowspan=1 colspan=1>73.8-93.0%</td><td rowspan=1 colspan=1>585/585</td><td rowspan=1 colspan=2>100</td><td rowspan=1 colspan=2>99.3-100%</td></tr><tr><td rowspan=1 colspan=1>Influenza Bc</td><td rowspan=1 colspan=1>47/50</td><td rowspan=1 colspan=2>94.0</td><td rowspan=1 colspan=1>83.8-97.9%</td><td rowspan=1 colspan=1>588/589</td><td rowspan=1 colspan=2>99.8</td><td rowspan=1 colspan=2>99.0-100%</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=9>Antibiotic Resistance Markers</td></tr><tr><td rowspan=1 colspan=1>CTX-M</td><td rowspan=1 colspan=2>130/130</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>97.1-100%</td><td rowspan=1 colspan=2>323/324</td><td rowspan=1 colspan=2>99.7</td><td rowspan=1 colspan=1>98.3-100%</td></tr><tr><td rowspan=1 colspan=1>IMP</td><td rowspan=1 colspan=2>45/45</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>92.1-100%</td><td rowspan=1 colspan=2>412/412</td><td rowspan=1 colspan=2>100</td><td rowspan=1 colspan=1>99.1-100%</td></tr><tr><td rowspan=1 colspan=1>KPC</td><td rowspan=1 colspan=2>53/53</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>93.2-100%</td><td rowspan=1 colspan=2>400/400</td><td rowspan=1 colspan=2>100</td><td rowspan=1 colspan=1>99.1-100%</td></tr><tr><td rowspan=1 colspan=1>NDM</td><td rowspan=1 colspan=2>53/53</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>93.2-100%</td><td rowspan=1 colspan=2>404/404</td><td rowspan=1 colspan=2>100</td><td rowspan=1 colspan=1>99.1-100%</td></tr><tr><td rowspan=1 colspan=1>OXA-48-like</td><td rowspan=1 colspan=2>53/53</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>93.2-100%</td><td rowspan=1 colspan=2>307/307</td><td rowspan=1 colspan=2>100</td><td rowspan=1 colspan=1>98.8-100%</td></tr><tr><td rowspan=1 colspan=1>VIM</td><td rowspan=1 colspan=2>58/58</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>93.8-100%</td><td rowspan=1 colspan=2>399/399</td><td rowspan=1 colspan=2>100</td><td rowspan=1 colspan=1>99.0-100%</td></tr></table>

a. Five FN specimens were spiked with an $K$ . aerogenes (previously $E$ . aerogenes) strain (ATCC 29751) that demonstrated poor reactivity with the FilmArray Pneumonia Panel (see Table 8). b. Two FN specimens were spiked with an influenza A strain that may have been under-quantified c. Three FN specimens were spiked with an influenza B strain that may have been under-quantified

Table 93: FilmArray Pneumonia Panel Performance of Contrived Sputum Specimens   

<table><tr><td colspan="1" rowspan="2">Analyte</td><td colspan="3" rowspan="1">Sensitivity/PPA</td><td colspan="4" rowspan="1">Specificity/NPA</td></tr><tr><td colspan="1" rowspan="1">TP/(TP + FN)</td><td colspan="1" rowspan="1">%</td><td colspan="1" rowspan="1">95% CI</td><td colspan="2" rowspan="1">TN/(TN + FP)</td><td colspan="1" rowspan="1">%</td><td colspan="1" rowspan="1">95% CI</td></tr><tr><td colspan="8" rowspan="1">Bacteria</td></tr><tr><td colspan="1" rowspan="1">Klebsiella aerogenesa</td><td colspan="1" rowspan="1">47/55</td><td colspan="1" rowspan="1">85.5</td><td colspan="1" rowspan="1">73.8-92.4%</td><td colspan="1" rowspan="1">513/513</td><td colspan="2" rowspan="1">100</td><td colspan="1" rowspan="1">99.3-100%</td></tr><tr><td colspan="1" rowspan="1">Streptococcus pyogenes</td><td colspan="1" rowspan="1">48/50</td><td colspan="1" rowspan="1">96.0</td><td colspan="1" rowspan="1">86.5-98.9%</td><td colspan="1" rowspan="1">516/516</td><td colspan="2" rowspan="1">100</td><td colspan="1" rowspan="1">99.3-100%</td></tr><tr><td colspan="8" rowspan="1">Atypical Bacteria</td></tr><tr><td colspan="1" rowspan="1">Chlamydia pneumoniae</td><td colspan="1" rowspan="1">49/50</td><td colspan="1" rowspan="1">98.0</td><td colspan="1" rowspan="1">89.5-99.6%</td><td colspan="1" rowspan="1">521/521</td><td colspan="2" rowspan="1">100</td><td colspan="1" rowspan="1">99.3-100%</td></tr><tr><td colspan="1" rowspan="1">Legionella pneumophila</td><td colspan="1" rowspan="1">50/50</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">92.9-100%</td><td colspan="1" rowspan="1">521/521</td><td colspan="2" rowspan="1">100</td><td colspan="1" rowspan="1">99.3-100%</td></tr><tr><td colspan="1" rowspan="1">Mycoplasma pneumoniae</td><td colspan="1" rowspan="1">48/50</td><td colspan="1" rowspan="1">96.0</td><td colspan="1" rowspan="1">86.5-98.9%</td><td colspan="1" rowspan="1">521/521</td><td colspan="2" rowspan="1">100</td><td colspan="1" rowspan="1">99.3-100%</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="2" rowspan="1">Viruses</td><td></td><td></td><td></td><td></td></tr><tr><td colspan="1" rowspan="1">Adenovirus</td><td colspan="1" rowspan="1">50/52</td><td colspan="1" rowspan="1">96.2</td><td colspan="1" rowspan="1">87.0-98.9%</td><td colspan="1" rowspan="1">494/494</td><td colspan="2" rowspan="1">100</td><td colspan="1" rowspan="1">99.2-100%</td></tr><tr><td colspan="1" rowspan="1">Human Metapneumovirus</td><td colspan="1" rowspan="1">51/51</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">93.0-100%</td><td colspan="1" rowspan="1">520/520</td><td colspan="2" rowspan="1">100</td><td colspan="1" rowspan="1">99.3-100%</td></tr><tr><td colspan="1" rowspan="1">Influenza Ab</td><td colspan="1" rowspan="1">47/50</td><td colspan="1" rowspan="1">94.0</td><td colspan="1" rowspan="1">83.8-97.9%</td><td colspan="1" rowspan="1">517/521</td><td colspan="2" rowspan="1">99.2</td><td colspan="1" rowspan="1">98.0-99.7%</td></tr><tr><td colspan="1" rowspan="1">Influenza Bc</td><td colspan="1" rowspan="1">48/51</td><td colspan="1" rowspan="1">94.1</td><td colspan="1" rowspan="1">84.1-98.0%</td><td colspan="1" rowspan="1">516/517</td><td colspan="2" rowspan="1">99.8</td><td colspan="1" rowspan="1">98.9-100%</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="7" rowspan="1">Antibiotic Resistance Markers</td></tr><tr><td colspan="1" rowspan="1">CTX-M</td><td colspan="1" rowspan="1">121/122</td><td colspan="1" rowspan="1">99.2</td><td colspan="1" rowspan="1">95.6-99.9%</td><td colspan="1" rowspan="1">289/290</td><td colspan="2" rowspan="1">99.7</td><td colspan="1" rowspan="1">98.1-99.9%</td></tr><tr><td colspan="1" rowspan="1">IMP</td><td colspan="1" rowspan="1">43/44</td><td colspan="1" rowspan="1">97.7</td><td colspan="1" rowspan="1">88.2-99.6%</td><td colspan="1" rowspan="1">381/381</td><td colspan="2" rowspan="1">100</td><td colspan="1" rowspan="1">99.0-100%</td></tr><tr><td colspan="1" rowspan="1">KPC</td><td colspan="1" rowspan="1">54/54</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">93.4-100%</td><td colspan="1" rowspan="1">360/361</td><td colspan="2" rowspan="1">99.7</td><td colspan="1" rowspan="1">98.4-100%</td></tr><tr><td colspan="1" rowspan="1">NDM</td><td colspan="1" rowspan="1">53/53</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">93.2-100%</td><td colspan="1" rowspan="1">372/372</td><td colspan="2" rowspan="1">100</td><td colspan="1" rowspan="1">99.0-100%</td></tr><tr><td colspan="1" rowspan="1">OXA-48-like</td><td colspan="1" rowspan="1">51/51</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">93.0-100%</td><td colspan="1" rowspan="1">232/232</td><td colspan="2" rowspan="1">100</td><td colspan="1" rowspan="1">98.4-100%</td></tr><tr><td colspan="1" rowspan="1">VIM</td><td colspan="1" rowspan="1">56/56</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">93.6-100%</td><td colspan="1" rowspan="1">369/369</td><td colspan="2" rowspan="1">100</td><td colspan="1" rowspan="1">99.0-100%</td></tr></table>

a. Four FN specimens were spiked with an $K$ . aerogenes (previously $E$ . aerogenes) strain (ATCC 29751) that demonstrated poor reactivity with the FilmArray Pneumonia Panel (see Table 8). b. Two FN specimens were spiked with an influenza A strain that may have been under-quantified c. Two FN specimens were spiked with an influenza B strain that may have been under-quantified

# Testing of Polymicrobial Contrived Specimens

Additionally, two sets of individual BAL $\mathrm { ( N { = } } 6 0$ ) and sputum $\scriptstyle ( \mathrm { N } = 6 0$ ) specimens were multi-spiked with randomized low ( $1 0 ^ { 4 }$ copies/mL), medium $1 0 ^ { 5 . 5 }$ copies/mL) and high ( $\mathrm { \bar { 1 0 ^ { 7 } } }$ copies/mL) relative concentrations of either $A$ . baumannii, $E$ . cloacae and $E$ . coli or K. oxytoca, $P$ . mirabilis and $S$ . marcescens. As shown in Table 94 and Table 95, the majority of the spiked organisms were reported at the expected relative low, medium, or high bin level by the FilmArray Pneumonia Panel. In four BAL specimens, $E$ . cloacae was intended to be spiked at a medium level but was reported in a in a high $( \geq 1 0 ^ { 7 } )$ bin. Also, one specimen spiked with a high level of $P$ . mirabilis was not detected (i.e. a false negative result for $P$ . mirabilis).

Table 94: Polymicrobial Sputum Specimen Results   
Table 95: Polymicrobial BAL Specimen Results   

<table><tr><td colspan="6" rowspan="1">Organism Spiked Into Sputum</td></tr><tr><td colspan="2" rowspan="2"></td><td colspan="3" rowspan="1">FilmArray Result</td><td colspan="1" rowspan="2">Total</td></tr><tr><td colspan="1" rowspan="1">Low</td><td colspan="1" rowspan="1">Medium</td><td colspan="1" rowspan="1">High</td></tr><tr><td colspan="1" rowspan="3">Sr kc</td><td colspan="1" rowspan="1">Low(104 copies/mL)</td><td colspan="1" rowspan="1">60</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">60</td></tr><tr><td colspan="1" rowspan="1">Medium(105.5 copies/mL)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">60</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">60</td></tr><tr><td colspan="1" rowspan="1">High(107 copies/mL)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">60</td><td colspan="1" rowspan="1">60</td></tr><tr><td colspan="6" rowspan="1">Organism Spiked Into BAL</td></tr><tr><td colspan="2" rowspan="2"></td><td colspan="3" rowspan="1">FilmArray Result</td><td colspan="1" rowspan="2">Total</td></tr><tr><td colspan="1" rowspan="1">Low</td><td colspan="1" rowspan="1">Medium</td><td colspan="1" rowspan="1">High</td></tr><tr><td colspan="1" rowspan="3">S kde</td><td colspan="1" rowspan="1">Low(104 copies/mL)</td><td colspan="1" rowspan="1">60</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">60</td></tr><tr><td colspan="1" rowspan="1">Medium(105.5 copies/mL)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">56</td><td colspan="1" rowspan="1">4a</td><td colspan="1" rowspan="1">60</td></tr><tr><td colspan="1" rowspan="1">High(107 copies/mL)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">59</td><td colspan="1" rowspan="1">59b</td></tr></table>

a Quantified at the bin boundary and reported $\mathsf { a s } { \mathrm { > } } { = } 1 0 ^ { 7 }$ b One false negative result

4. Clinical cut-off:

Not applicable

5. Expected values/Reference range:

In the prospective clinical evaluation of the FilmArray Pneumonia Panel, 846 BAL (including mini-BAL) and 836 sputum (including ETA) specimens, were collected and tested at eight study sites across the United States over approximately ten months (October 2016 to July 2017). Expected value (as determined by FilmArray Pneumonia Panel) summaries for BAL and sputum specimens are stratified by subject age and care setting in Tables 96 through 101.

# Table 96: Expected Value (EV) as Determined by FilmArray Pneumonia Panel: Summary by Age Group for BAL Specimens Collected from Hospitalized Subjects During the FilmArray Pneumonia Panel Prospective Clinical Evaluation (October 2016 to July 2017)

Table 97: Expected Value (EV) as Determined by FilmArray Pneumonia Panel:   

<table><tr><td rowspan=1 colspan=13>BAL</td></tr><tr><td rowspan=3 colspan=1>FilmArray Result</td><td rowspan=1 colspan=2>Overall(N=846)</td><td rowspan=1 colspan=10>Hospitalized (N=666)</td></tr><tr><td rowspan=2 colspan=1>#</td><td rowspan=2 colspan=1>EV</td><td rowspan=1 colspan=2>≤5 (N=8)</td><td rowspan=1 colspan=2>6-17 (N=18)</td><td rowspan=1 colspan=2>18-34 (N=61)</td><td rowspan=1 colspan=2>35-65 (N=366)</td><td rowspan=1 colspan=2>&gt;65 (N=212)</td></tr><tr><td rowspan=1 colspan=1>#</td><td rowspan=1 colspan=1>EV</td><td rowspan=1 colspan=1>#</td><td rowspan=1 colspan=1>EV</td><td rowspan=1 colspan=1>#</td><td rowspan=1 colspan=1>EV</td><td rowspan=1 colspan=1>#</td><td rowspan=1 colspan=1>EV</td><td rowspan=1 colspan=1>#</td><td rowspan=1 colspan=1>EV</td></tr><tr><td rowspan=1 colspan=1>Acinetobacter calcoaceticus-baumanniicomplex</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>0.8%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>1.1%</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0.9%</td></tr><tr><td rowspan=1 colspan=1>Enterobacter cloacae complex</td><td rowspan=1 colspan=1>23</td><td rowspan=1 colspan=1>2.7%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>2.7%</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>5.7%</td></tr><tr><td rowspan=1 colspan=1>Escherichia coli</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>2.4%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>4.9%</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>2.2%</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>3.3%</td></tr><tr><td rowspan=1 colspan=1>Haemophilus influenzae</td><td rowspan=1 colspan=1>82</td><td rowspan=1 colspan=1>9.7%</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>25.0%</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>33.3%</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>9.8%</td><td rowspan=1 colspan=1>38</td><td rowspan=1 colspan=1>10.4%</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>3.8%</td></tr><tr><td rowspan=1 colspan=1>Klebsiella aerogenes</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>1.5%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1.6%</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>1.1%</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>3.3%</td></tr><tr><td rowspan=1 colspan=1>Klebsiella oxytoca</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>1.3%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>12.5%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>3.3%</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>1.4%</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0.9%</td></tr><tr><td rowspan=1 colspan=1>Klebsiella pneumoniae group</td><td rowspan=1 colspan=1>27</td><td rowspan=1 colspan=1>3.2%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>12.5%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>3.3%</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>2.7%</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>4.2%</td></tr><tr><td rowspan=1 colspan=1>Moraxella catarrhalis</td><td rowspan=1 colspan=1>29</td><td rowspan=1 colspan=1>3.4%</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>37.5%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>5.6%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1.6%</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>2.7%</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0.9%</td></tr><tr><td rowspan=1 colspan=1>Proteus spp.</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>1.1%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1.6%</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0.5%</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>2.8%</td></tr><tr><td rowspan=1 colspan=1>Pseudomonas aeruginosa</td><td rowspan=1 colspan=1>74</td><td rowspan=1 colspan=1>8.7%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>12.5%</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>11.1%</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>4.9%</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>8.2%</td><td rowspan=1 colspan=1>22</td><td rowspan=1 colspan=1>10.4%</td></tr><tr><td rowspan=1 colspan=1>Serratia marcescens</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>1.4%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1.6%</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0.8%</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>1.9%</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus aureus</td><td rowspan=1 colspan=1>116</td><td rowspan=1 colspan=1>13.7%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>12.5%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>5.6%</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>21.3%</td><td rowspan=1 colspan=1>61</td><td rowspan=1 colspan=1>16.7%</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>11.3%</td></tr><tr><td rowspan=1 colspan=1>Streptococcus agalactiae</td><td rowspan=1 colspan=1>25</td><td rowspan=1 colspan=1>3.0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>6.6%</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>4.1%</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0.9%</td></tr><tr><td rowspan=1 colspan=1>Streptococcus pneumoniae</td><td rowspan=1 colspan=1>29</td><td rowspan=1 colspan=1>3.4%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>11.1%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1.6%</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>3.6%</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>2.4%</td></tr><tr><td rowspan=1 colspan=1>Streptococcus pyogenes</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>0.9%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>5.6%</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>3.3%</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0.5%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td></tr><tr><td rowspan=1 colspan=1>CTX-M</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>0.8%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>1.4%</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0.9%</td></tr><tr><td rowspan=1 colspan=1>IMP</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td></tr><tr><td rowspan=1 colspan=1>KPC</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0.4%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.3%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.5%</td></tr><tr><td rowspan=1 colspan=1>mecA/C and MREJ</td><td rowspan=1 colspan=1>46</td><td rowspan=1 colspan=1>5.4%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>12.5%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>5.6%</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>6.6%</td><td rowspan=1 colspan=1>25</td><td rowspan=1 colspan=1>6.8%</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>5.7%</td></tr><tr><td rowspan=1 colspan=1>NDM</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.1%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.3%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td></tr><tr><td rowspan=1 colspan=1>OXA-48-like</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td></tr><tr><td rowspan=1 colspan=1>VIM</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td></tr><tr><td rowspan=1 colspan=1>Chlamydia pneumoniae</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.1%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.3%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td></tr><tr><td rowspan=1 colspan=1>Legionella pneumophila</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0.2%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1.6%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.3%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td></tr><tr><td rowspan=1 colspan=1>Mycoplasma pneumoniae</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>0.5%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>5.6%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.3%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.5%</td></tr><tr><td rowspan=1 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>0.9%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>12.5%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1.6%</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>1.1%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.5%</td></tr><tr><td rowspan=1 colspan=1>Coronavirus</td><td rowspan=1 colspan=1>31</td><td rowspan=1 colspan=1>3.7%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>4.4%</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>2.8%</td></tr><tr><td rowspan=1 colspan=1>Human Metapneumovirus</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>1.1%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1.6%</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>1.1%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.5%</td></tr><tr><td rowspan=1 colspan=1>Human Rhinovirus/Enterovirus</td><td rowspan=1 colspan=1>64</td><td rowspan=1 colspan=1>7.6%</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>37.5%</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>22.2%</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>6.6%</td><td rowspan=1 colspan=1>25</td><td rowspan=1 colspan=1>6.8%</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>5.2%</td></tr><tr><td rowspan=1 colspan=1>Influenza A</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>1.8%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>1.6%</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>3.3%</td></tr><tr><td rowspan=1 colspan=1>Influenza B</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>0.8%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>5.6%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1.6%</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>1.1%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza Virus</td><td rowspan=1 colspan=1>18</td><td rowspan=1 colspan=1>2.1%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>3.3%</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>2.7%</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>2.4%</td></tr><tr><td rowspan=1 colspan=1>Respiratory Syncytial Virus</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>0.5%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.3%</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>1.4%</td></tr></table>

# Summary by Age Group for Sputum Specimens Collected from Hospitalized Subjects During the FilmArray Pneumonia Panel Prospective Clinical Evaluation (October 2016 to July 2017)

<table><tr><td rowspan=1 colspan=13>Sputum</td></tr><tr><td rowspan=3 colspan=1>FilmArray Result</td><td rowspan=1 colspan=2>Overall(N=836)</td><td rowspan=1 colspan=10>Hospitalized (N=682)</td></tr><tr><td rowspan=2 colspan=1>#</td><td rowspan=2 colspan=1>EV</td><td rowspan=1 colspan=2>≤5 (N=102</td><td rowspan=1 colspan=2>6-17 (N=64)</td><td rowspan=1 colspan=2>18-34 (N=68)</td><td rowspan=1 colspan=2>35-65 (N=252)</td><td rowspan=1 colspan=1>>65</td><td rowspan=1 colspan=1>N=196)</td></tr><tr><td rowspan=1 colspan=1>#</td><td rowspan=1 colspan=1>EV</td><td rowspan=1 colspan=1>#</td><td rowspan=1 colspan=1>EV</td><td rowspan=1 colspan=1>#</td><td rowspan=1 colspan=1>EV</td><td rowspan=1 colspan=1>#</td><td rowspan=1 colspan=1>EV</td><td rowspan=1 colspan=1>#</td><td rowspan=1 colspan=1>EV</td></tr><tr><td rowspan=1 colspan=1>Acinetobacter calcoaceticus-baumanniicomplex</td><td rowspan=1 colspan=1>28</td><td rowspan=1 colspan=1>3.3%</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>2.9%</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>4.7%</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>2.9%</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>1.6%</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>2.6%</td></tr><tr><td rowspan=1 colspan=1>Enterobacter cloacae complex</td><td rowspan=1 colspan=1>32</td><td rowspan=1 colspan=1>3.8%</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>6.9%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1.6%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1.5%</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>3.6%</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>3.6%</td></tr><tr><td rowspan=1 colspan=1>Escherichia coli</td><td rowspan=1 colspan=1>48</td><td rowspan=1 colspan=1>5.7%</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>2.9%</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>6.3%</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>10.3%</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>3.2%</td><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>8.2%</td></tr><tr><td rowspan=1 colspan=1>Haemophilus influenzae</td><td rowspan=1 colspan=1>107</td><td rowspan=1 colspan=1>12.8%</td><td rowspan=1 colspan=1>23</td><td rowspan=1 colspan=1>22.5%</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>10.9%</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>13.2%</td><td rowspan=1 colspan=1>25</td><td rowspan=1 colspan=1>9.9%</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>10.2%</td></tr><tr><td rowspan=1 colspan=1>Klebsiella aerogenes</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>1.4%</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>2.0%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1.6%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1.5%</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>1.2%</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>1.5%</td></tr><tr><td rowspan=1 colspan=1>Klebsiella oxytoca</td><td rowspan=1 colspan=1>19</td><td rowspan=1 colspan=1>2.3%</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>2.9%</td><td rowspan=1 colspan=1>−1−</td><td rowspan=1 colspan=1>1.6%</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>2.9%</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>2.0%</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>1.5%</td></tr><tr><td rowspan=1 colspan=1>Klebsiella pneumoniae group</td><td rowspan=1 colspan=1>65</td><td rowspan=1 colspan=1>7.8%</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>7.8%</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>4.7%</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>10.3%</td><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>6.3%</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>10.2%</td></tr><tr><td rowspan=1 colspan=1>Moraxella catarrhalis</td><td rowspan=1 colspan=1>75</td><td rowspan=1 colspan=1>9.0%</td><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>16.7%</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>7.8%</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>5.9%</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>3.6%</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>5.1%</td></tr><tr><td rowspan=1 colspan=1>Proteus spp.</td><td rowspan=1 colspan=1>23</td><td rowspan=1 colspan=1>2.8%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1.6%</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>4.4%</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0.8%</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>2.0%</td></tr><tr><td rowspan=1 colspan=1>Pseudomonas aeruginosa</td><td rowspan=1 colspan=1>160</td><td rowspan=1 colspan=1>19.1%</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>8.8%</td><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>21.9%</td><td rowspan=1 colspan=1>18</td><td rowspan=1 colspan=1>26.5%</td><td rowspan=1 colspan=1>32</td><td rowspan=1 colspan=1>12.7%</td><td rowspan=1 colspan=1>33</td><td rowspan=1 colspan=1>16.8%</td></tr><tr><td rowspan=1 colspan=1>Serratia marcescens</td><td rowspan=1 colspan=1>53</td><td rowspan=1 colspan=1>6.3%</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>3.9%</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>6.3%</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>7.4%</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>2.4%</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>4.1%</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus aureus</td><td rowspan=1 colspan=1>204</td><td rowspan=1 colspan=1>24.4%</td><td rowspan=1 colspan=1>23</td><td rowspan=1 colspan=1>22.5%</td><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>21.9%</td><td rowspan=1 colspan=1>18</td><td rowspan=1 colspan=1>26.5%</td><td rowspan=1 colspan=1>54</td><td rowspan=1 colspan=1>21.4%</td><td rowspan=1 colspan=1>43</td><td rowspan=1 colspan=1>21.9%</td></tr><tr><td rowspan=1 colspan=1>Streptococcus agalactiae</td><td rowspan=1 colspan=1>43</td><td rowspan=1 colspan=1>5.1%</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>2.9%</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>7.8%</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>5.9%</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>4.8%</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>2.0%</td></tr><tr><td rowspan=1 colspan=1>Streptococcus pneumoniae</td><td rowspan=1 colspan=1>51</td><td rowspan=1 colspan=1>6.1%</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>11.8%</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>3.1%</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>4.4%</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>4.4%</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>3.6%</td></tr><tr><td rowspan=1 colspan=1>Streptococcus pyogenes</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>1.3%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>6.3%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0.8%</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>1.0%</td></tr><tr><td rowspan=1 colspan=1>CTX-M</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>1.1%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1.0%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1.6%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1.5%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.4%</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>1.0%</td></tr><tr><td rowspan=1 colspan=1>IMP</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td></tr><tr><td rowspan=1 colspan=1>KPC</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>0.8%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1.5%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.4%</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>2.0%</td></tr><tr><td rowspan=1 colspan=1>mecA/C and MREJ</td><td rowspan=1 colspan=1>107</td><td rowspan=1 colspan=1>12.8%</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>5.9%</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>10.9%</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>11.8%</td><td rowspan=1 colspan=1>32</td><td rowspan=1 colspan=1>12.7%</td><td rowspan=1 colspan=1>28</td><td rowspan=1 colspan=1>14.3%</td></tr><tr><td rowspan=1 colspan=1>NDM</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td></tr><tr><td rowspan=1 colspan=1>OXA-48-like</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td></tr><tr><td rowspan=1 colspan=1>VIM</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0.2%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1.5%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.4%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td></tr><tr><td rowspan=1 colspan=1>Chlamydia pneumoniae</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td></tr><tr><td rowspan=1 colspan=1>Legionella pneumophila</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td></tr><tr><td rowspan=1 colspan=1>Mycoplasma pneumoniae</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>0.8%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1.0%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1.6%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td></tr><tr><td rowspan=1 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>1.9%</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>2.0%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1.6%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>2.4%</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>1.5%</td></tr><tr><td rowspan=1 colspan=1>Coronavirus</td><td rowspan=1 colspan=1>35</td><td rowspan=1 colspan=1>4.2%</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>2.9%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>2.9%</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>2.8%</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>5.6%</td></tr><tr><td rowspan=1 colspan=1>Human Metapneumovirus</td><td rowspan=1 colspan=1>22</td><td rowspan=1 colspan=1>2.6%</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>3.9%</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>4.7%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>2.0%</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>2.6%</td></tr><tr><td rowspan=1 colspan=1>Human Rhinovirus/Enterovirus</td><td rowspan=1 colspan=1>112</td><td rowspan=1 colspan=1>13.4%</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>20.6%</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>10.9%</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>11.8%</td><td rowspan=1 colspan=1>19</td><td rowspan=1 colspan=1>7.5%</td><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>7.1%</td></tr><tr><td rowspan=1 colspan=1>Influenza A</td><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>1.9%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1.0%</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>4.7%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.4%</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>2.0%</td></tr><tr><td rowspan=1 colspan=1>Influenza B</td><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>1.7%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1.6%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>2.0%</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>2.6%</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza Virus</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>3.6%</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>3.9%</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>6.3%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1.5%</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>3.6%</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>3.6%</td></tr><tr><td rowspan=1 colspan=1>Respiratory Syncytial Virus</td><td rowspan=1 colspan=1>48</td><td rowspan=1 colspan=1>5.7%</td><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>16.7%</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>3.1%</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>4.4%</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>2.4%</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>5.1%</td></tr><tr><td></td><td rowspan=1 colspan=2></td><td rowspan=1 colspan=2></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr></table>

# Table 98: Expected Value (EV) as Determined by FilmArray Pneumonia Panel): Summary by Age Group for BAL Specimens Collected from Outpatient Subjects During the FilmArray Pneumonia Panel Prospective Clinical Evaluation (October 2016 to July 2017)

Table 99: Expected Value (EV) as Determined by FilmArray Pneumonia Panel:   

<table><tr><td rowspan=1 colspan=13>BAL</td></tr><tr><td rowspan=3 colspan=1>FilmArray Result</td><td rowspan=1 colspan=2>Overall(N=846)</td><td rowspan=1 colspan=10>Hospitalized (N=159)</td></tr><tr><td rowspan=2 colspan=1>#</td><td rowspan=2 colspan=1>EV</td><td rowspan=1 colspan=1>≤5</td><td rowspan=1 colspan=1>(N=15)</td><td rowspan=1 colspan=2>6-17 (N=8)</td><td rowspan=1 colspan=2>18-34 (N=5)</td><td rowspan=1 colspan=2>35-65 (N=93)</td><td rowspan=1 colspan=1>>65</td><td rowspan=1 colspan=1>(N=38)</td></tr><tr><td rowspan=1 colspan=1>#</td><td rowspan=1 colspan=1>EV</td><td rowspan=1 colspan=1>#</td><td rowspan=1 colspan=1>EV</td><td rowspan=1 colspan=1>#</td><td rowspan=1 colspan=1>EV</td><td rowspan=1 colspan=1>#</td><td rowspan=1 colspan=1>EV</td><td rowspan=1 colspan=1>#</td><td rowspan=1 colspan=1>EV</td></tr><tr><td rowspan=1 colspan=1>Acinetobacter calcoaceticus-baumanniicomplex</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>0.8%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td></tr><tr><td rowspan=1 colspan=1>Enterobacter cloacae complex</td><td rowspan=1 colspan=1>23</td><td rowspan=1 colspan=1>2.7%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td></tr><tr><td rowspan=1 colspan=1>Escherichia coli</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>2.4%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td></tr><tr><td rowspan=1 colspan=1>Haemophilus influenzae</td><td rowspan=1 colspan=1>82</td><td rowspan=1 colspan=1>9.7%</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>26.7%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>12.5%</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>40.0%</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>8.6%</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>5.3%</td></tr><tr><td rowspan=1 colspan=1>Klebsiella aerogenes</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>1.5%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>2.6%</td></tr><tr><td rowspan=1 colspan=1>Klebsiella oxytoca</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>1.3%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1.1%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td></tr><tr><td rowspan=1 colspan=1>Klebsiella pneumoniae group</td><td rowspan=1 colspan=1>27</td><td rowspan=1 colspan=1>3.2%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>2.2%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>2.6%</td></tr><tr><td rowspan=1 colspan=1>Moraxella catarrhalis</td><td rowspan=1 colspan=1>29</td><td rowspan=1 colspan=1>3.4%</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>26.7%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>12.5%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>4.3%</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>5.3%</td></tr><tr><td rowspan=1 colspan=1>Proteus spp.</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>1.1%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td></tr><tr><td rowspan=1 colspan=1>Pseudomonas aeruginosa</td><td rowspan=1 colspan=1>74</td><td rowspan=1 colspan=1>8.7%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>8.6%</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>13.2%</td></tr><tr><td rowspan=1 colspan=1>Serratia marcescens</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>1.4%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>2.2%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus aureus</td><td rowspan=1 colspan=1>116</td><td rowspan=1 colspan=1>13.7%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>6.7%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>20.0%</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>6.5%</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>5.3%</td></tr><tr><td rowspan=1 colspan=1>Streptococcus agalactiae</td><td rowspan=1 colspan=1>25</td><td rowspan=1 colspan=1>3.0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>20.0%</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>2.2%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td></tr><tr><td rowspan=1 colspan=1>Streptococcus pneumoniae</td><td rowspan=1 colspan=1>29</td><td rowspan=1 colspan=1>3.4%</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>13.3%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>12.5%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>20.0%</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>3.2%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>2.6%</td></tr><tr><td rowspan=1 colspan=1>Streptococcus pyogenes</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>0.9%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>6.7%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>12.5%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td></tr><tr><td rowspan=1 colspan=1>CTX-M</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>0.8%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td></tr><tr><td rowspan=1 colspan=1>IMP</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td></tr><tr><td rowspan=1 colspan=1>KPC</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0.4%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td></tr><tr><td rowspan=1 colspan=1>mecA/C and MREJ</td><td rowspan=1 colspan=1>46</td><td rowspan=1 colspan=1>5.4%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1.1%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td></tr><tr><td rowspan=1 colspan=1>NDM</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.1%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td></tr><tr><td rowspan=1 colspan=1>OXA-48-like</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td></tr><tr><td rowspan=1 colspan=1>VIM</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td></tr><tr><td rowspan=1 colspan=1>Chlamydia pneumoniae</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.1%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td></tr><tr><td rowspan=1 colspan=1>Legionella pneumophila</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0.2%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td></tr><tr><td rowspan=1 colspan=1>Mycoplasma pneumoniae</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>0.5%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>6.7%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td></tr><tr><td rowspan=1 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>0.9%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1.1%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td></tr><tr><td rowspan=1 colspan=1>Coronavirus</td><td rowspan=1 colspan=1>31</td><td rowspan=1 colspan=1>3.7%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>7.5%</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>5.3%</td></tr><tr><td rowspan=1 colspan=1>Human Metapneumovirus</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>1.1%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>2.2%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td></tr><tr><td rowspan=1 colspan=1>Human Rhinovirus/Enterovirus</td><td rowspan=1 colspan=1>64</td><td rowspan=1 colspan=1>7.6%</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>20.0%</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>50.0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>6.5%</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>10.5%</td></tr><tr><td rowspan=1 colspan=1>Influenza A</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>1.8%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1.1%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>2.6%</td></tr><tr><td rowspan=1 colspan=1>Influenza B</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>0.8%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza Virus</td><td rowspan=1 colspan=1>18</td><td rowspan=1 colspan=1>2.1%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>2.6%</td></tr><tr><td rowspan=1 colspan=1>Respiratory Syncytial Virus</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>0.5%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td></tr></table>

Summary by Age Group for Sputum Specimens Collected from Outpatient Subjects During the FilmArray Pneumonia Panel Prospective Clinical Evaluation (October 2016 to July 2017)   

<table><tr><td rowspan=1 colspan=13>Sputum</td></tr><tr><td rowspan=3 colspan=1>FilmArray Result</td><td rowspan=1 colspan=2>Overall(N=836)</td><td rowspan=1 colspan=10>Hospitalized (N=73)</td></tr><tr><td rowspan=2 colspan=1>#</td><td rowspan=2 colspan=1>EV</td><td rowspan=1 colspan=1>≤5</td><td rowspan=1 colspan=1>≤5 (N=13)</td><td rowspan=1 colspan=2>6-17 (N=21)</td><td rowspan=1 colspan=2>18-34 (N=7)</td><td rowspan=1 colspan=2>35-65 (N=18)</td><td rowspan=1 colspan=1>>65</td><td rowspan=1 colspan=1>&gt;65 (N=14)</td></tr><tr><td rowspan=1 colspan=1>#</td><td rowspan=1 colspan=1>EV</td><td rowspan=1 colspan=1>#</td><td rowspan=1 colspan=1>EV</td><td rowspan=1 colspan=1>#</td><td rowspan=1 colspan=1>EV</td><td rowspan=1 colspan=1>#</td><td rowspan=1 colspan=1>EV</td><td rowspan=1 colspan=1>#</td><td rowspan=1 colspan=1>EV</td></tr><tr><td rowspan=1 colspan=1>Acinetobacter calcoaceticus-baumanniicomplex</td><td rowspan=1 colspan=1>28</td><td rowspan=1 colspan=1>3.3%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>7.7%</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>19.0%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>14.3%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td></tr><tr><td rowspan=1 colspan=1>Enterobacter cloacae complex</td><td rowspan=1 colspan=1>32</td><td rowspan=1 colspan=1>3.8%</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>23.1%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>4.8%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>7.1%</td></tr><tr><td rowspan=1 colspan=1>Escherichia coli</td><td rowspan=1 colspan=1>48</td><td rowspan=1 colspan=1>5.7%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>4.8%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>14.3%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td></tr><tr><td rowspan=1 colspan=1>Haemophilus influenzae</td><td rowspan=1 colspan=1>107</td><td rowspan=1 colspan=1>12.8%</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>23.1%</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>19.0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>16.7%</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>21.4%</td></tr><tr><td rowspan=1 colspan=1>Klebsiella aerogenes</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>1.4%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td></tr><tr><td rowspan=1 colspan=1>Klebsiella oxytoca</td><td rowspan=1 colspan=1>19</td><td rowspan=1 colspan=1>2.3%</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>23.1%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td></tr><tr><td rowspan=1 colspan=1>Klebsiella pneumoniae group</td><td rowspan=1 colspan=1>65</td><td rowspan=1 colspan=1>7.8%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>9.5%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>11.1%</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>14.3%</td></tr><tr><td rowspan=1 colspan=1>Moraxella catarrhalis</td><td rowspan=1 colspan=1>75</td><td rowspan=1 colspan=1>9.0%</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>46.2%</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>33.3%</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>28.6%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>5.6%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td></tr><tr><td rowspan=1 colspan=1>Proteus spp.</td><td rowspan=1 colspan=1>23</td><td rowspan=1 colspan=1>2.8%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>7.7%</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>14.3%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>5.6%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td></tr><tr><td rowspan=1 colspan=1>Pseudomonas aeruginosa</td><td rowspan=1 colspan=1>160</td><td rowspan=1 colspan=1>19.1%</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>23.1%</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>61.9%</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>57.1%</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>16.7%</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>35.7%</td></tr><tr><td rowspan=1 colspan=1>Serratia marcescens</td><td rowspan=1 colspan=1>53</td><td rowspan=1 colspan=1>6.3%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>7.7%</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>33.3%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>11.1%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus aureus</td><td rowspan=1 colspan=1>204</td><td rowspan=1 colspan=1>24.4%</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>53.8%</td><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>66.7%</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>28.6%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>5.6%</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>14.3%</td></tr><tr><td rowspan=1 colspan=1>Streptococcus agalactiae</td><td rowspan=1 colspan=1>43</td><td rowspan=1 colspan=1>5.1%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>7.7%</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>14.3%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>14.3%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>5.6%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>7.1%</td></tr><tr><td rowspan=1 colspan=1>Streptococcus pneumoniae</td><td rowspan=1 colspan=1>51</td><td rowspan=1 colspan=1>6.1%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>7.7%</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>9.5%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>14.3%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td></tr><tr><td rowspan=1 colspan=1>Streptococcus pyogenes</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>1.3%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>4.8%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td></tr><tr><td rowspan=1 colspan=1>CTX-M</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>1.1%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>4.8%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>14.3%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td></tr><tr><td rowspan=1 colspan=1>IMP</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td></tr><tr><td rowspan=1 colspan=1>KPC</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>0.8%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>5.6%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td></tr><tr><td rowspan=1 colspan=1>mecA/C and MREJ</td><td rowspan=1 colspan=1>107</td><td rowspan=1 colspan=1>12.8%</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>15.4%</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>47.6%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>14.3%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>7.1%</td></tr><tr><td rowspan=1 colspan=1>NDM</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td></tr><tr><td rowspan=1 colspan=1>OXA-48-like</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td></tr><tr><td rowspan=1 colspan=1>VIM</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0.2%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td></tr><tr><td rowspan=1 colspan=1>Chlamydia pneumoniae</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td></tr><tr><td rowspan=1 colspan=1>Legionella pneumophila</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td></tr><tr><td rowspan=1 colspan=1>Mycoplasma pneumoniae</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>0.8%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td></tr><tr><td rowspan=1 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>1.9%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>7.1%</td></tr><tr><td rowspan=1 colspan=1>Coronavirus</td><td rowspan=1 colspan=1>35</td><td rowspan=1 colspan=1>4.2%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>7.7%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>4.8%</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>57.1%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>5.6%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>7.1%</td></tr><tr><td rowspan=1 colspan=1>Human Metapneumovirus</td><td rowspan=1 colspan=1>22</td><td rowspan=1 colspan=1>2.6%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>14.3%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td></tr><tr><td rowspan=1 colspan=1>Human Rhinovirus/Enterovirus</td><td rowspan=1 colspan=1>112</td><td rowspan=1 colspan=1>13.4%</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>38.5%</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>23.8%</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>28.6%</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>11.1%</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>28.6%</td></tr><tr><td rowspan=1 colspan=1>Influenza A</td><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>1.9%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>11.1%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>7.1%</td></tr><tr><td rowspan=1 colspan=1>Influenza B</td><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>1.7%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>7.1%</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza Virus</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>3.6%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td></tr><tr><td rowspan=1 colspan=1>Respiratory Syncytial Virus</td><td rowspan=1 colspan=1>48</td><td rowspan=1 colspan=1>5.7%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>7.7%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>5.6%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td></tr></table>

# Table 100: Expected Value(EV) as Determined by FilmArray Pneumonia Panel: Summary by Age Group for BAL Specimens Collected from Emergency Department Subjects During the FilmArray Pneumonia Panel Prospective Clinical Evaluation (October 2016 to July 2017)

<table><tr><td rowspan=1 colspan=13>BAL</td></tr><tr><td rowspan=3 colspan=1>FilmArray Result</td><td rowspan=1 colspan=2>Overall(N=846)</td><td rowspan=1 colspan=10>Hospitalized (N=21)</td></tr><tr><td rowspan=2 colspan=1>#</td><td rowspan=2 colspan=1>EV</td><td rowspan=1 colspan=2>≤5(N=0)</td><td rowspan=1 colspan=2>6-17 (N=1)</td><td rowspan=1 colspan=2>18-34 (N=4)</td><td rowspan=1 colspan=2>35-65 (N=11)</td><td rowspan=1 colspan=2>&gt;65 (N=5)</td></tr><tr><td rowspan=1 colspan=1>#</td><td rowspan=1 colspan=1>EV</td><td rowspan=1 colspan=1>#</td><td rowspan=1 colspan=1>EV</td><td rowspan=1 colspan=1>#</td><td rowspan=1 colspan=1>EV</td><td rowspan=1 colspan=1>#</td><td rowspan=1 colspan=1>EV</td><td rowspan=1 colspan=1>#</td><td rowspan=1 colspan=1>EV</td></tr><tr><td rowspan=1 colspan=1>Acinetobacter calcoaceticus-baumanniicomplex</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>0.8%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>9.1%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td></tr><tr><td rowspan=1 colspan=1>Enterobacter cloacae complex</td><td rowspan=1 colspan=1>23</td><td rowspan=1 colspan=1>2.7%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>9.1%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td></tr><tr><td rowspan=1 colspan=1>Escherichia coli</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>2.4%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>25.0%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>9.1%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td></tr><tr><td rowspan=1 colspan=1>Haemophilus influenzae</td><td rowspan=1 colspan=1>82</td><td rowspan=1 colspan=1>9.7%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>25.0%</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>18.2%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>20.0%</td></tr><tr><td rowspan=1 colspan=1>Klebsiella aerogenes</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>1.5%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td></tr><tr><td rowspan=1 colspan=1>Klebsiella oxytoca</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>1.3%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td></tr><tr><td rowspan=1 colspan=1>Klebsiella pneumoniae group</td><td rowspan=1 colspan=1>27</td><td rowspan=1 colspan=1>3.2%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>9.1%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td></tr><tr><td rowspan=1 colspan=1>Moraxella catarrhalis</td><td rowspan=1 colspan=1>29</td><td rowspan=1 colspan=1>3.4%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td></tr><tr><td rowspan=1 colspan=1>Proteus spp.</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>1.1%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td></tr><tr><td rowspan=1 colspan=1>Pseudomonas aeruginosa</td><td rowspan=1 colspan=1>74</td><td rowspan=1 colspan=1>8.7%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>18.2%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>20.0%</td></tr><tr><td rowspan=1 colspan=1>Serratia marcescens</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>1.4%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>20.0%</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus aureus</td><td rowspan=1 colspan=1>116</td><td rowspan=1 colspan=1>13.7%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>50.0%</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>27.3%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td></tr><tr><td rowspan=1 colspan=1>Streptococcus agalactiae</td><td rowspan=1 colspan=1>25</td><td rowspan=1 colspan=1>3.0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>9.1%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td></tr><tr><td rowspan=1 colspan=1>Streptococcus pneumoniae</td><td rowspan=1 colspan=1>29</td><td rowspan=1 colspan=1>3.4%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td></tr><tr><td rowspan=1 colspan=1>Streptococcus pyogenes</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>0.9%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>25.0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td></tr><tr><td rowspan=1 colspan=1>CTX-M</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>0.8%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td></tr><tr><td rowspan=1 colspan=1>IMP</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td></tr><tr><td rowspan=1 colspan=1>KPC</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0.4%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>9.1%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td></tr><tr><td rowspan=1 colspan=1>mecA/C and MREJ</td><td rowspan=1 colspan=1>46</td><td rowspan=1 colspan=1>5.4%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>25.0%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>9.1%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td></tr><tr><td rowspan=1 colspan=1>NDM</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.1%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td></tr><tr><td rowspan=1 colspan=1>OXA-48-like</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td></tr><tr><td rowspan=1 colspan=1>VIM</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td></tr><tr><td rowspan=1 colspan=1>Chlamydia pneumoniae</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.1%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td></tr><tr><td rowspan=1 colspan=1>Legionella pneumophila</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0.2%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td></tr><tr><td rowspan=1 colspan=1>Mycoplasma pneumoniae</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>0.5%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td></tr><tr><td rowspan=1 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>0.9%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td></tr><tr><td rowspan=1 colspan=1>Coronavirus</td><td rowspan=1 colspan=1>31</td><td rowspan=1 colspan=1>3.7%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td></tr><tr><td rowspan=1 colspan=1>Human Metapneumovirus</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>1.1%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>9.1%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td></tr><tr><td rowspan=1 colspan=1>Human Rhinovirus/Enterovirus</td><td rowspan=1 colspan=1>64</td><td rowspan=1 colspan=1>7.6%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td></tr><tr><td rowspan=1 colspan=1>Influenza A</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>1.8%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td></tr><tr><td rowspan=1 colspan=1>Influenza B</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>0.8%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>9.1%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza Virus</td><td rowspan=1 colspan=1>18</td><td rowspan=1 colspan=1>2.1%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td></tr><tr><td rowspan=1 colspan=1>Respiratory Syncytial Virus</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>0.5%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td></tr></table>

# Table 101: Expected Value(EV) as Determined by FilmArray Pneumonia Panel: Summary by Age Group for Sputum Specimens Collected from Emergency Department Subjects During the FilmArray Pneumonia Panel Prospective Clinical Evaluation (October 2016 to July 2017)

<table><tr><td rowspan=1 colspan=13>Sputum</td></tr><tr><td rowspan=3 colspan=1>FilmArray Result</td><td rowspan=1 colspan=2>Overall(N=836)</td><td rowspan=1 colspan=10>Hospitalized (N=81)</td></tr><tr><td rowspan=2 colspan=1>#</td><td rowspan=2 colspan=1>EV</td><td rowspan=1 colspan=2>≤5 (N=23)</td><td rowspan=1 colspan=2>6-17 (N=22)</td><td rowspan=1 colspan=2>18-34 (N=11)</td><td rowspan=1 colspan=2>35-65 (N=14)</td><td rowspan=1 colspan=1>>65</td><td rowspan=1 colspan=1>&gt;65 (N=11)</td></tr><tr><td rowspan=1 colspan=1>#</td><td rowspan=1 colspan=1>EV</td><td rowspan=1 colspan=1>#</td><td rowspan=1 colspan=1>EV</td><td rowspan=1 colspan=1>#</td><td rowspan=1 colspan=1>EV</td><td rowspan=1 colspan=1>#</td><td rowspan=1 colspan=1>EV</td><td rowspan=1 colspan=1>#</td><td rowspan=1 colspan=1>EV</td></tr><tr><td rowspan=1 colspan=1>Acinetobacter calcoaceticus-baumanniicomplex</td><td rowspan=1 colspan=1>28</td><td rowspan=1 colspan=1>3.3%</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>8.7%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>4.5%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>9.1%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>9.1%</td></tr><tr><td rowspan=1 colspan=1>Enterobacter cloacae complex</td><td rowspan=1 colspan=1>32</td><td rowspan=1 colspan=1>3.8%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>4.3%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>4.5%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td></tr><tr><td rowspan=1 colspan=1>Escherichia coli</td><td rowspan=1 colspan=1>48</td><td rowspan=1 colspan=1>5.7%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>22.7%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>9.1%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>7.1%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>9.1%</td></tr><tr><td rowspan=1 colspan=1>Haemophilus influenzae</td><td rowspan=1 colspan=1>107</td><td rowspan=1 colspan=1>12.8%</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>8.7%</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>18.2%</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>18.2%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>7.1%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>9.1%</td></tr><tr><td rowspan=1 colspan=1>Klebsiella aerogenes</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>1.4%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>9.1%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>7.1%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td></tr><tr><td rowspan=1 colspan=1>Klebsiella oxytoca</td><td rowspan=1 colspan=1>19</td><td rowspan=1 colspan=1>2.3%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>4.3%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>7.1%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td></tr><tr><td rowspan=1 colspan=1>Klebsiella pneumoniae group</td><td rowspan=1 colspan=1>65</td><td rowspan=1 colspan=1>7.8%</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>8.7%</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>9.1%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>9.1%</td></tr><tr><td rowspan=1 colspan=1>Moraxella catarrhalis</td><td rowspan=1 colspan=1>75</td><td rowspan=1 colspan=1>9.0%</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>34.8%</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>22.7%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>9.1%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td></tr><tr><td rowspan=1 colspan=1>Proteus spp.</td><td rowspan=1 colspan=1>23</td><td rowspan=1 colspan=1>2.8%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>4.3%</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>18.2%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>9.1%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>7.1%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>9.1%</td></tr><tr><td rowspan=1 colspan=1>Pseudomonas aeruginosa</td><td rowspan=1 colspan=1>160</td><td rowspan=1 colspan=1>19.1%</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>39.1%</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>36.4%</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>63.6%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>7.1%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>9.1%</td></tr><tr><td rowspan=1 colspan=1>Serratia marcescens</td><td rowspan=1 colspan=1>53</td><td rowspan=1 colspan=1>6.3%</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>30.4%</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>22.7%</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>27.3%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>9.1%</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus aureus</td><td rowspan=1 colspan=1>204</td><td rowspan=1 colspan=1>24.4%</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>47.8%</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>45.5%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>14.3%</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>27.3%</td></tr><tr><td rowspan=1 colspan=1>Streptococcus agalactiae</td><td rowspan=1 colspan=1>43</td><td rowspan=1 colspan=1>5.1%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>13.6%</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>18.2%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>7.1%</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>18.2%</td></tr><tr><td rowspan=1 colspan=1>Streptococcus pneumoniae</td><td rowspan=1 colspan=1>51</td><td rowspan=1 colspan=1>6.1%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>4.3%</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>31.8%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>9.1%</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>14.3%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>9.1%</td></tr><tr><td rowspan=1 colspan=1>Streptococcus pyogenes</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>1.3%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>9.1%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td></tr><tr><td rowspan=1 colspan=1>TX-M</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>1.1%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>4.5%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td></tr><tr><td rowspan=1 colspan=1>IMP</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td></tr><tr><td rowspan=1 colspan=1>KPC</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>0.8%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td></tr><tr><td rowspan=1 colspan=1>mecA/C and MREJ</td><td rowspan=1 colspan=1>107</td><td rowspan=1 colspan=1>12.8%</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>21.7%</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>18.2%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>27.3%</td></tr><tr><td rowspan=1 colspan=1>NDM</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td></tr><tr><td rowspan=1 colspan=1>OXA-48-like</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td></tr><tr><td rowspan=1 colspan=1>VIM</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0.2%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td></tr><tr><td rowspan=1 colspan=1>Chlamydia pneumoniae</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td></tr><tr><td rowspan=1 colspan=1>Legionella pneumophila</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td></tr><tr><td rowspan=1 colspan=1>Mycoplasma pneumoniae</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>0.8%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>13.6%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>9.1%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>9.1%</td></tr><tr><td rowspan=1 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>1.9%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>4.3%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>4.5%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>9.1%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td></tr><tr><td rowspan=1 colspan=1>Coronavirus</td><td rowspan=1 colspan=1>35</td><td rowspan=1 colspan=1>4.2%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>4.3%</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>9.1%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>9.1%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td></tr><tr><td rowspan=1 colspan=1>Human Metapneumovirus</td><td rowspan=1 colspan=1>22</td><td rowspan=1 colspan=1>2.6%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>4.3%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>9.1%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>7.1%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>9.1%</td></tr><tr><td rowspan=1 colspan=1>Human Rhinovirus/Enterovirus</td><td rowspan=1 colspan=1>112</td><td rowspan=1 colspan=1>13.4%</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>52.2%</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>31.8%</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>27.3%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>7.1%</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>18.2%</td></tr><tr><td rowspan=1 colspan=1>Influenza A</td><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>1.9%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>4.5%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>14.3%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>9.1%</td></tr><tr><td rowspan=1 colspan=1>Influenza B</td><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>1.7%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>4.5%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>9.1%</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza Virus</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>3.6%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>13.6%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>14.3%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td></tr><tr><td rowspan=1 colspan=1>Respiratory Syncytial Virus</td><td rowspan=1 colspan=1>48</td><td rowspan=1 colspan=1>5.7%</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>13.0%</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>13.6%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>9.1%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>7.1%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td></tr></table>

In addition, observed multiple detections in each specimen type (as determined by FilmArray Pneumonia

Panel) during the FilmArray Pneumonia Panel prospective clinical evaluation is presented in Table 102. The FilmArray Pneumonia Panel detected at least one analyte in a total of 413 BAL specimens $( 4 8 . 8 \%$ positivity rate; 413/846) and 602 sputum specimens $( 7 2 . 0 \%$ positivity rate; 602/836). Two or more analytes were detected by the FilmArray Pneumonia Panel in $3 7 . 8 \%$ of positive BAL specimens (156/413; $1 8 . 4 \%$ of all tested BAL specimens, 156/846) and $56 . 5 \%$ of positive sputum specimens (340/602; $4 0 . 7 \%$ of all tested sputum specimens, 340/836). Up to six analytes were detected in both specimen types.

Table 102: Expected Values Multiple Detections as Determined by the FilmArray Pneumonia Panel) for the FilmArray Pneumonia Panel Clinical Evaluation (October 2016 – July 2017)   

<table><tr><td rowspan=2 colspan=1>FilmArray Result</td><td rowspan=1 colspan=2>Expected Value (as Determined byTesting of 846 Prospective BALSpecimens)</td><td rowspan=1 colspan=2>Expected Value (as Determined byTesting of 836 Prospective SputumSpecimens)</td></tr><tr><td rowspan=1 colspan=1>NumberDetectedand Reported</td><td rowspan=1 colspan=1>% of Total(% of Positives)</td><td rowspan=1 colspan=1>Number Detectedand Reported</td><td rowspan=1 colspan=1>% of Total(% of Positives)</td></tr><tr><td rowspan=1 colspan=1>Detected (at least oneresult)</td><td rowspan=1 colspan=1>413</td><td rowspan=1 colspan=1>48.8%(100%)</td><td rowspan=1 colspan=1>602</td><td rowspan=1 colspan=1>72.0%(100%)</td></tr><tr><td rowspan=1 colspan=1>One analyte result</td><td rowspan=1 colspan=1>257</td><td rowspan=1 colspan=1>30.4%(62.2%)</td><td rowspan=1 colspan=1>262</td><td rowspan=1 colspan=1>31.3%(43.5%)</td></tr><tr><td rowspan=1 colspan=1>Two analyte results</td><td rowspan=1 colspan=1>105</td><td rowspan=1 colspan=1>12.4%(25.4%)</td><td rowspan=1 colspan=1>178</td><td rowspan=1 colspan=1>21.3%(29.6%)</td></tr><tr><td rowspan=1 colspan=1>Three analyte results</td><td rowspan=1 colspan=1>28</td><td rowspan=1 colspan=1>3.3%(6.8%)</td><td rowspan=1 colspan=1>85</td><td rowspan=1 colspan=1>10.2%(14.1%)</td></tr><tr><td rowspan=1 colspan=1>Four analyte results</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>2.4%(4.8%)</td><td rowspan=1 colspan=1>42</td><td rowspan=1 colspan=1>5.0%(7.0%)</td></tr><tr><td rowspan=1 colspan=1>Five analyte results</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0.2%(0.5%)</td><td rowspan=1 colspan=1>23</td><td rowspan=1 colspan=1>2.8%(3.8%)</td></tr><tr><td rowspan=1 colspan=1>Six or more analyte results</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.1%(0.2%)</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>1.4%(2.0%)</td></tr></table>

Detection profiles, including co-detections with multiple bacteria, multiple viruses, or bacteria and virus combinations, are shown in Figure 2. Viruses and bacteria were observed together in $6 \%$ of BAL and $21 \%$ of sputum specimens. The rate of multiple bacteria detections in single specimens was higher in sputum specimens $( 1 9 \% )$ compared to BAL $( 1 2 \% )$ . Multiple viruses were observed in both specimen types at a low rate $( 1 \% )$ .

Figure 2: Detection profiles (as determined by the FilmArray Pneumonia Panel) for BAL and sputum specimens (AMR genes excluded)

![](images/4e84d6ec641ba8b9c2e71a98dcac38cfdc23e5b82f7f7168f7da361e99522468.jpg)

The FilmArray Pneumonia Panel identified 119 different co-detection combinations in 156 BAL specimens, 100 of which were unique combinations (Table 103 and Table 104). False positive results (as compared to SOC) were observed in 104 of 156 BAL specimens with co-detections. Similarly, 243 different co-detection combinations were identified in 340 sputum specimens, 194 of which were unique combinations. False positive results (as compared to SOC) were observed in 239 of 340 sputum specimens with co-detections.

able 103: Co-detections by FilmArray Pneumonia Panel with performance compared to SOC for BAL specimens   

<table><tr><td rowspan=1 colspan=6>BAL</td></tr><tr><td rowspan=1 colspan=1>Organism Co-Detections(includes viruses andbacteria)</td><td rowspan=1 colspan=1>Number of Co-DetectionCombinations</td><td rowspan=1 colspan=1>Number of Co-DetectionCombinationsObserved in onlyOne Specimen</td><td rowspan=1 colspan=1>Total Specimenswith Co-Detection</td><td rowspan=1 colspan=1>Total Specimenswith FalsePositive(s)</td><td rowspan=1 colspan=1>False PositiveAnalyte(s)/ TotalAnalytes</td></tr><tr><td rowspan=1 colspan=1>Two analyte results</td><td rowspan=1 colspan=1>69</td><td rowspan=1 colspan=1>51</td><td rowspan=1 colspan=1>105</td><td rowspan=1 colspan=1>64</td><td rowspan=1 colspan=1>78/210</td></tr><tr><td rowspan=1 colspan=1>Three analyte results</td><td rowspan=1 colspan=1>27</td><td rowspan=1 colspan=1>26</td><td rowspan=1 colspan=1>28</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>34/84</td></tr><tr><td rowspan=1 colspan=1>Four analyte results</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>44/80</td></tr><tr><td rowspan=1 colspan=1>Five analyte results</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>6/10</td></tr><tr><td rowspan=1 colspan=1>Six analyte results</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>Seven analyte results</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>2/7</td></tr><tr><td rowspan=1 colspan=1>All co-detections</td><td rowspan=1 colspan=1>119</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>156</td><td rowspan=1 colspan=1>104</td><td rowspan=1 colspan=1>168/391</td></tr></table>

Table 104: Co-detections by FilmArray Pneumonia Panel with performance compared to SOC for sputum specimens   

<table><tr><td rowspan=1 colspan=6>Sputum</td></tr><tr><td rowspan=1 colspan=1>Organism Co-Detections(includes viruses andbacteria)</td><td rowspan=1 colspan=1>Number of Co-DetectionCombinations</td><td rowspan=1 colspan=1>Number of Co-DetectionCombinationsObserved in onlyOne Specimen</td><td rowspan=1 colspan=1>Total Specimenswith Co-Detection</td><td rowspan=1 colspan=1>Total Specimenswith FalsePositive(s)</td><td rowspan=1 colspan=1>False PositiveAnalyte(s)/ TotalAnalytes</td></tr><tr><td rowspan=1 colspan=1>Two analyte results</td><td rowspan=1 colspan=1>91</td><td rowspan=1 colspan=1>52</td><td rowspan=1 colspan=1>178</td><td rowspan=1 colspan=1>104</td><td rowspan=1 colspan=1>128/356</td></tr><tr><td rowspan=1 colspan=1>Three analyte results</td><td rowspan=1 colspan=1>76</td><td rowspan=1 colspan=1>67</td><td rowspan=1 colspan=1>85</td><td rowspan=1 colspan=1>68</td><td rowspan=1 colspan=1>109/255</td></tr><tr><td rowspan=1 colspan=1>Four analyte results</td><td rowspan=1 colspan=1>41</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>42</td><td rowspan=1 colspan=1>34</td><td rowspan=1 colspan=1>79/168</td></tr><tr><td rowspan=1 colspan=1>Five analyte results</td><td rowspan=1 colspan=1>23</td><td rowspan=1 colspan=1>23</td><td rowspan=1 colspan=1>23</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>62/115</td></tr><tr><td rowspan=1 colspan=1>Six analyte results</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>31/54</td></tr><tr><td rowspan=1 colspan=1>Seven analyte results</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>14/21</td></tr><tr><td rowspan=1 colspan=1>All co-detections</td><td rowspan=1 colspan=1>243</td><td rowspan=1 colspan=1>194</td><td rowspan=1 colspan=1>340</td><td rowspan=1 colspan=1>239</td><td rowspan=1 colspan=1>423/969</td></tr></table>

SOC testing reported Normal Oral Flora (NOF) and no specific organism for 79/322 $( 2 4 . 5 \% )$ BAL and 141/510 $( 2 7 . 6 \% )$ sputum specimens for which the FilmArray Pneumonia Panel reported at least one nonatypical bacteria (Table 105 and Table 106).

Table 105: Non-Atypical Bacterial Detections (as compared to SOC) in BAL Specimens for the FilmArray Pneumonia Panel Clinical Evaluation (October 2016 – July 2017)   

<table><tr><td rowspan=1 colspan=5>BAL</td></tr><tr><td rowspan=2 colspan=1>FilmArray Pneumonia Panel Result(n=846)</td><td rowspan=2 colspan=1>Positive SOCCulture</td><td rowspan=1 colspan=2>Negative SOC Culture</td><td rowspan=2 colspan=1>SOC CultureNot Performed</td></tr><tr><td rowspan=1 colspan=1>No Growth</td><td rowspan=1 colspan=1>NOF Reported</td></tr><tr><td rowspan=1 colspan=1>Detected (n=322)</td><td rowspan=1 colspan=1>195/322(60.6%)</td><td rowspan=1 colspan=1>43/322 (13.4%)</td><td rowspan=1 colspan=1>79/322 (24.5%)</td><td rowspan=1 colspan=1>5/322 (1.6%)</td></tr><tr><td rowspan=1 colspan=1>Not Detected (n=524)</td><td rowspan=1 colspan=1>11/524 (2.1%)</td><td rowspan=1 colspan=1>268/524 (51.1%)</td><td rowspan=1 colspan=1>242/524 (46.2%)</td><td rowspan=1 colspan=1>3/524 (0.6%)</td></tr></table>

Table 106: Non-Atypical Bacterial Detections (as compared to SOC) in Sputum Specimens for the FilmArray Pneumonia Panel Clinical Evaluation (October 2016 – July 2017)   

<table><tr><td rowspan=1 colspan=5>Sputum</td></tr><tr><td rowspan=2 colspan=1>FilmArray Pneumonia Panel Result(n=836)</td><td rowspan=2 colspan=1>Positive SOCCulture</td><td rowspan=1 colspan=2>Negative SOC Culture</td><td rowspan=2 colspan=1>SOC CultureNot Performed</td></tr><tr><td rowspan=1 colspan=1>No Growth</td><td rowspan=1 colspan=1>NOFReported</td></tr><tr><td rowspan=1 colspan=1>Detected (n=510)</td><td rowspan=1 colspan=1>331/510(64.9%)</td><td rowspan=1 colspan=1>26/510 (5.1%)</td><td rowspan=1 colspan=1>141/510 (27.6%)</td><td rowspan=1 colspan=1>12/510 (2.4%)</td></tr><tr><td rowspan=1 colspan=1>Not Detected (n=326)</td><td rowspan=1 colspan=1>11/326 (3.4%)</td><td rowspan=1 colspan=1>110/326 (33.7%)201/326 (61.7%)</td><td rowspan=1 colspan=1>110/326 (33.7%)201/326 (61.7%)</td><td rowspan=1 colspan=1>4/326 (1.2%)</td></tr></table>

Two or more bacteria were detected by the FilmArray Pneumonia Panel in $4 2 . 8 \%$ (356/832) of positive specimens; $3 4 . 2 \%$ (110/322) of positive BAL specimens and $4 8 . 2 \%$ (246/510) of positive sputum specimens. The resulting co-detection combinations, as reported by the FilmArray Pneumonia Panel, are presented in Table 107 and Table 108. These tables also indicate the number of specimens with false positive results for each co-detection combination, as well as the specific analytes that were discrepant.

Table 107: Co-detections of Non-atypical Bacteria Detected by FilmArray Pneumonia Panel and Compared to qRefCxDistinct $\mathbf { C o }$ -Detection Combinations   

<table><tr><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td colspan="1" rowspan="2">TotalSpecimenswith Co-DetectionCombination</td><td colspan="1" rowspan="2">#Specimenswith FalsePositiveCo-Detections</td><td colspan="1" rowspan="2">False PositiveAnalyte(s) [FalsePositive AMRGene(s)a</td></tr><tr><td colspan="1" rowspan="1">Analyte1</td><td colspan="1" rowspan="1">Analyte2</td><td colspan="1" rowspan="1">Analyte3</td><td colspan="1" rowspan="1">Analyte4</td><td colspan="1" rowspan="1">Analyte5</td><td colspan="1" rowspan="1">Analyte6</td><td colspan="1" rowspan="1">AMRGene(s)</td></tr><tr><td colspan="1" rowspan="1">H. influenzae</td><td colspan="1" rowspan="1">K. oxytoca</td><td colspan="1" rowspan="1">M.catarrhalis</td><td colspan="1" rowspan="1">P.aeruginosa</td><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1">S.agalactiae</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">H. influenzae, K.oxytoca, M.catarrhalis, S. aureus,S. agalactiae</td></tr><tr><td colspan="1" rowspan="1">ACB complex</td><td colspan="1" rowspan="1">K.aerogenes</td><td colspan="1" rowspan="1">Proteus spp.</td><td colspan="1" rowspan="1">P.aeruginosa</td><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">mecA/C &amp;MREJ</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">ACB complex</td></tr><tr><td colspan="1" rowspan="1">ACB complex</td><td colspan="1" rowspan="1">E. cloacaecomplex</td><td colspan="1" rowspan="1">K.pneumoniaegroup</td><td colspan="1" rowspan="1">S.agalactiae</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">ACB complex, S.agalactiae</td></tr><tr><td colspan="1" rowspan="1">ACB complex</td><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">H.influenzae</td><td colspan="1" rowspan="1">P.aeruginosa</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">ACB complex, H.influenzae, P.aeruginosa</td></tr><tr><td colspan="1" rowspan="1">E. cloacaecomplex</td><td colspan="1" rowspan="1">H.influenzae</td><td colspan="1" rowspan="1">K. oxytoca</td><td colspan="1" rowspan="1">K.pneumoniaegroup</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">→</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">E. cloacae complex, H.influenzae, K. oxytoca,K. pneumoniae group</td></tr><tr><td colspan="1" rowspan="1">E. cloacaecomplex</td><td colspan="1" rowspan="1">K. oxytoca</td><td colspan="1" rowspan="1">K.pneumoniaeProteus spp.group</td><td colspan="1" rowspan="1">pneumoniaeProteus spp.</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">E. cloacae complex, K.oxytoca</td></tr><tr><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">H.influenzae</td><td colspan="1" rowspan="1">K.aerogenes</td><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">E. coli, H. influenzae,K. aerogenes, S.aureus</td></tr><tr><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">H.influenzae</td><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1">S.agalactiae</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">E. coli, H. influenzae,S. agalactiae</td></tr><tr><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">K. oxytoca</td><td colspan="1" rowspan="1">P.aeruginosa</td><td colspan="1" rowspan="1">S.marcescens</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">K. oxytoca, S.marcescens</td></tr><tr><td colspan="1" rowspan="1">H. influenzae</td><td colspan="1" rowspan="1">K.pneumoniaegroup</td><td colspan="1" rowspan="1">M.catarrhalis</td><td colspan="1" rowspan="1">S.marcescens</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">→</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">H. influenzae, M.catarrhalis</td></tr><tr><td colspan="1" rowspan="1">H. influenzae</td><td colspan="1" rowspan="1">Proteusspp.</td><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1">S. pyogenes</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">H. influenzae, Proteusspp., S. aureus, S.pyogenes</td></tr><tr><td colspan="1" rowspan="1">H. influenzae</td><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1">S.agalactiae</td><td colspan="1" rowspan="1">S.pneumoniae</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">H. influenzae, S.aureus, S. agalactiae,S. pneumoniae</td></tr><tr><td colspan="1" rowspan="1">K. oxytoca</td><td colspan="1" rowspan="1">P.aeruginosa</td><td colspan="1" rowspan="1">S.marcescens</td><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">S. marcescens</td></tr><tr><td colspan="1" rowspan="1">K.pneumoniaegroup</td><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1">S.agalactiae</td><td colspan="1" rowspan="1">S.pneumoniae</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">S. aureus, S.agalactiae, S.pneumoniae</td></tr><tr><td colspan="1" rowspan="1">ACB complex</td><td colspan="1" rowspan="1">H.influenzae</td><td colspan="1" rowspan="1">S.pneumoniae</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">ACB complex, H.influenzae, S.pneumoniae</td></tr><tr><td colspan="1" rowspan="1">E. cloacaecomplex</td><td colspan="1" rowspan="1">H.influenzae</td><td colspan="1" rowspan="1">K. oxytoca</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">H. influenzae</td></tr><tr><td colspan="1" rowspan="1">E. cloacaecomplex</td><td colspan="1" rowspan="1">H.influenzae</td><td colspan="1" rowspan="1">S.pneumoniae</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">H. influenzae, S.pneumoniae</td></tr><tr><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">H.influenzae</td><td colspan="1" rowspan="1">S.pneumoniae</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">E. coli, S. pneumoniae</td></tr><tr><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">K.pneumoniaegroup</td><td colspan="1" rowspan="1">S.pneumoniae</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">K. pneumoniae group,S. pneumoniae</td></tr><tr><td colspan="1" rowspan="1">H. influenzae</td><td colspan="1" rowspan="1">K.pneumoniaegroup</td><td colspan="1" rowspan="1">M.catarrhalis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">H. influenzae, K.pneumoniaegroup, M. catarrhalis</td></tr><tr><td colspan="1" rowspan="1">H. influenzae</td><td colspan="1" rowspan="1">K.pneumoniaegroup</td><td colspan="1" rowspan="1">P.aeruginosa</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">H. influenzae</td></tr><tr><td colspan="1" rowspan="1">H. influenzae</td><td colspan="1" rowspan="1">M.catarrhalis</td><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">H. influenzae (1), M.catarrhalis(2)</td></tr><tr><td colspan="1" rowspan="1">H. influenzae</td><td colspan="1" rowspan="1">M.catarrhalis</td><td colspan="1" rowspan="1">S.pneumoniae</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">H. influenzae (2), M.catarrhalis(2), S. pneumoniae (2)</td></tr><tr><td colspan="1" rowspan="1">H. influenzae</td><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1">S.agalactiae</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">H. influenzae, S.aureus, S. agalactiae</td></tr><tr><td colspan="1" rowspan="1">H. influenzae</td><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1">S. pyogenes</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">mecA/C &amp;MREJ</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">H. influenzae, S.aureus, S. pyogenes</td></tr><tr><td colspan="1" rowspan="1">K. aerogenes</td><td colspan="1" rowspan="1">P.aeruginosa</td><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">K. aerogenes, S. aureus</td></tr><tr><td colspan="1" rowspan="1">K.pneumoniaegroup</td><td colspan="1" rowspan="1">P.aeruginosa</td><td colspan="1" rowspan="1">S.marcescens</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">→</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">S. marcescens</td></tr><tr><td colspan="1" rowspan="1">Proteus spp.</td><td colspan="1" rowspan="1">P.aeruginosa</td><td colspan="1" rowspan="1">S.agalactiae</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">KPC</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Proteus spp., P.aeruginosa, S.agalactiae</td></tr><tr><td colspan="1" rowspan="1">P. aeruginosa</td><td colspan="1" rowspan="1">S.marcescens</td><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">P. aeruginosa</td></tr><tr><td colspan="1" rowspan="1">P. aeruginosa</td><td colspan="1" rowspan="1">S.marcescens</td><td colspan="1" rowspan="1">S.pneumoniae</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">S.marcescens</td><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1">S.agalactiae</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">S. marcescens, S.agalactiae</td></tr><tr><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1">S.agalactiae</td><td colspan="1" rowspan="1">S.pneumoniae</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">mecA/C &amp;MREJ</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">S. aureus, S.agalactiae, S.pneumoniae</td></tr><tr><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1">S.agalactiae</td><td colspan="1" rowspan="1">S.pneumoniae</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">S. agalactiae, S.pneumoniae</td></tr><tr><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1">S.pneumoniae</td><td colspan="1" rowspan="1">S. pyogenes</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">S. pneumoniae</td></tr><tr><td colspan="1" rowspan="1">ACB complex</td><td colspan="1" rowspan="1">K.pneumoniaegroup</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">KPC</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">ACB complex, K.pneumoniaegroup, [KPC]</td></tr><tr><td colspan="1" rowspan="1">ACB complex</td><td colspan="1" rowspan="1">P.aeruginosa</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">ACB complex, P.aeruginosa</td></tr><tr><td colspan="1" rowspan="1">E. cloacaecomplex</td><td colspan="1" rowspan="1">K.aerogenes</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">K. aerogenes</td></tr><tr><td colspan="1" rowspan="1">E. cloacaecomplex</td><td colspan="1" rowspan="1">K. oxytoca</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">CTX-Mb</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">K. oxytoca</td></tr><tr><td colspan="1" rowspan="1">E. cloacaecomplex</td><td colspan="1" rowspan="1">K.pneumoniaegroup</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">E. cloacae complex (2),K. pneumoniae group(2)</td></tr><tr><td colspan="1" rowspan="1">E. cloacaecomplex</td><td colspan="1" rowspan="1">P.aeruginosa</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">KPC</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">E. cloacaecomplex</td><td colspan="1" rowspan="1">P.aeruginosa</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">E. cloacae complex, P.aeruginosa</td></tr><tr><td colspan="1" rowspan="1">E. cloacaecomplex</td><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">NDM</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">E. cloacae complex,[NDM]</td></tr><tr><td colspan="1" rowspan="1">E. cloacaecomplex</td><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">E. cloacae complex, S.aureus</td></tr><tr><td colspan="1" rowspan="1">E. cloacaecomplex</td><td colspan="1" rowspan="1">S.pneumoniae</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">E. cloacae complex, S.pneumoniae</td></tr><tr><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">K.aerogenes</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">E. coli</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">K.</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">pneumoniaegroup</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">CTX-M,mecA/C &amp;MREJ</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">S. aureus</td></tr><tr><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">mecA/C &amp;MREJ</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">E. coli, S. aureus</td></tr><tr><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">E. coli (1), S. aureus(1)</td></tr><tr><td colspan="1" rowspan="1">H. influenzae</td><td colspan="1" rowspan="1">K.aerogenes</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">H. influenzae, K.aerogenes</td></tr><tr><td colspan="1" rowspan="1">H. influenzae</td><td colspan="1" rowspan="1">K.pneumoniaegroup</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">H. infuenzae</td></tr><tr><td colspan="1" rowspan="1">H. influenzae</td><td colspan="1" rowspan="1">M.catarrhalis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">H. influenzae (4), M.catarrhalis(5)</td></tr><tr><td colspan="1" rowspan="1">H. influenzae</td><td colspan="1" rowspan="1">P.aeruginosa</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">H. influenzae</td></tr><tr><td colspan="1" rowspan="1">H. influenzae</td><td colspan="1" rowspan="1">S.marcescens</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">H. influenzae</td><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">mecA/C &amp;MREJ</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">H. influenzae (3), S.aureus (1)</td></tr><tr><td colspan="1" rowspan="1">H. influenzae</td><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">H. influenzae (4), S.aureus (2)</td></tr><tr><td colspan="1" rowspan="1">H. influenzae</td><td colspan="1" rowspan="1">S.agalactiae</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">H. influenzae, S.agalactiae</td></tr><tr><td colspan="1" rowspan="1">H. influenzae</td><td colspan="1" rowspan="1">S.pneumoniae</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">7</td><td colspan="1" rowspan="1">7</td><td colspan="1" rowspan="1">H. influenzae (7), S.pneumoniae(3)</td></tr><tr><td colspan="1" rowspan="1">H. influenzae</td><td colspan="1" rowspan="1">S. pyogenes</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">H. influenzae, S.pyogenes</td></tr><tr><td colspan="1" rowspan="1">K. aerogenes</td><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">mecA/C &amp;MREJ</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">K. aerogenes (2), S.aureus (1)</td></tr><tr><td colspan="1" rowspan="1">K. aerogenes</td><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">S. aureus</td></tr><tr><td colspan="1" rowspan="1">K. oxytoca</td><td colspan="1" rowspan="1">P.aeruginosa</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">K. oxytoca</td></tr><tr><td colspan="1" rowspan="1">K. oxytoca</td><td colspan="1" rowspan="1">S.agalactiae</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">K. oxytoca (2), S.agalactiae (2)</td></tr><tr><td colspan="1" rowspan="1">K.pneumoniae</td><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">mecA/C &amp;MREJ</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">K. pneumoniae group</td></tr><tr><td colspan="1" rowspan="1">group</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">K.pneumoniaegroup</td><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">K. pneumoniae group,S. aureus</td></tr><tr><td colspan="1" rowspan="1">M. catarrhalis</td><td colspan="1" rowspan="1">Proteus spp.</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">M. catarrhalis</td></tr><tr><td colspan="1" rowspan="1">M. catarrhalis</td><td colspan="1" rowspan="1">S.pneumoniae</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">M. catarrhalis, S.pneumoniae</td></tr><tr><td colspan="1" rowspan="1">M. catarrhalis</td><td colspan="1" rowspan="1">S. pyogenes</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">M. catarrhalis, S.pyogenes</td></tr><tr><td colspan="1" rowspan="1">Proteus spp.</td><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">mecA/C &amp;MREJ</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">S. aureus, [mecA/C &amp;MREJ]</td></tr><tr><td colspan="1" rowspan="1">P. aeruginosa</td><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">mecA/C &amp;MREJ</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">P. aeruginosa (3), S.aureus (2), [mecA/C &amp;MREJ</td></tr><tr><td colspan="1" rowspan="1">P. aeruginosa</td><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">S. aureus (2)</td></tr><tr><td colspan="1" rowspan="1">P. aeruginosa</td><td colspan="1" rowspan="1">S.agalactiae</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">P. aeruginosa, S.agalactiae</td></tr><tr><td colspan="1" rowspan="1">P. aeruginosa</td><td colspan="1" rowspan="1">S.pneumoniae</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">P. aeruginosa, S.pneumoniae</td></tr><tr><td colspan="1" rowspan="1">S.marcescens</td><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">mecA/C &amp;MREJ</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">S. marcescens</td></tr><tr><td colspan="1" rowspan="1">S.marcescens</td><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">S. marcescens, S.aureus</td></tr><tr><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1">S.agalactiae</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">mecA/C &amp;MREJ</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">S. agalactiae</td></tr><tr><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1">S.agalactiae</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">S. aureus (4), S.agalactiae (3)</td></tr><tr><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1">S.pneumoniae</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">mecA/C &amp;MREJ</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">S. aureus, S.pneumoniae</td></tr><tr><td colspan="7" rowspan="1">Total Co-Detections</td><td colspan="1" rowspan="1">110</td><td colspan="1" rowspan="1">103</td><td colspan="1" rowspan="1">187C/273</td></tr><tr><td colspan="7" rowspan="1">Total Double Detections</td><td colspan="1" rowspan="1">74</td><td colspan="1" rowspan="1">68</td><td colspan="1" rowspan="1">103/148</td></tr><tr><td colspan="7" rowspan="1">Total Triple Detections</td><td colspan="1" rowspan="1">22</td><td colspan="1" rowspan="1">21</td><td colspan="1" rowspan="1">44/66</td></tr><tr><td colspan="7" rowspan="1">Total Quadruple Detections</td><td colspan="1" rowspan="1">12</td><td colspan="1" rowspan="1">12</td><td colspan="1" rowspan="1">34/48</td></tr><tr><td colspan="7" rowspan="1">Total Quintuple Detections</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1/5</td></tr><tr><td colspan="7" rowspan="1">Total Sextuple Detections</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">5/6</td></tr></table>

a. AMR genes compared to qMol b. Performance not determined c. Of the 187 discrepant analytes (out of 273 total analytes), all 187 $1 0 0 \%$ ) were observed as being present in the specimen during discrepancy investigation; 55/187 $( 2 9 . 4 \% )$ were enumerated below $1 0 ^ { 3 . 5 } \mathrm { C F U / m L }$ by qRefCx, 112/187 d. $( 5 9 . 9 \% )$ ) were detected by qMol, 17/187 $( 9 . 1 \% )$ were detected using an additional molecular method, and the remaining 3/187 $( 1 . 6 \% )$ were identified in SOC culture.

Table 108: Co-detections of Non-atypical Bacteria in Sputum Detected by FilmArray Pneumonia Panel and Compared to qRefCx   

<table><tr><td colspan="8" rowspan="1">Colpared to qkelex</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="8" rowspan="1">Distinct Co-Detection Combinations</td><td colspan="1" rowspan="2">TotalSpecimenswith Co-DetectionCombination</td><td colspan="1" rowspan="2">#Specimenswith FalsePositiveCo-Detections</td><td colspan="1" rowspan="2">False PositiveAnalyte(s) [FalsePositive AMRGene(s)a</td></tr><tr><td colspan="1" rowspan="1">Analyte1</td><td colspan="1" rowspan="1">Analyte2</td><td colspan="1" rowspan="1">Analyte3</td><td colspan="1" rowspan="1">Analyte4</td><td colspan="1" rowspan="1">Analyte5</td><td colspan="1" rowspan="1">Analyte6</td><td colspan="1" rowspan="1">Analyte7</td><td colspan="1" rowspan="1">AMRGene(s)</td></tr><tr><td colspan="1" rowspan="1">ACBcomplex</td><td colspan="1" rowspan="1">Proteus spp.</td><td colspan="1" rowspan="1">P.aeruginosa</td><td colspan="1" rowspan="1">S.marcescens</td><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1">S. agalactiae</td><td colspan="1" rowspan="1">S.pneumoniae</td><td colspan="1" rowspan="1">mecA/C &amp;MREJ</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">ACB complex, S.agalactiae</td></tr><tr><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">K.pneumoniaegroup</td><td colspan="1" rowspan="1">M.catarrhalis</td><td colspan="1" rowspan="1">Proteus spp.</td><td colspan="1" rowspan="1">P.aeruginosa</td><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1">S. agalactiae</td><td colspan="1" rowspan="1">mecA/C &amp;MREJ</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">E. coli, K.pneumoniae group,M. catarrhalis,Proteus spp., P.aeruginosa, S. aureus,S. agalactiae</td></tr><tr><td colspan="1" rowspan="1">H.influenzae</td><td colspan="1" rowspan="1">K.pneumoniaegroup</td><td colspan="1" rowspan="1">P.aeruginosa</td><td colspan="1" rowspan="1">S.marcescens</td><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1">S. agalactiae</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">H. influenzae, K.pneumoniae group</td></tr><tr><td colspan="1" rowspan="1">M.catarrhalis</td><td colspan="1" rowspan="1">Proteus spp.</td><td colspan="1" rowspan="1">P.aeruginosa</td><td colspan="1" rowspan="1">S.marcescens</td><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1">S. agalactiae</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">mecA/C &amp;MREJ</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">M. catarrhalis, P.aeruginosa, S. aureus,S. agalactiae</td></tr><tr><td colspan="1" rowspan="1">M.catarrhalis</td><td colspan="1" rowspan="1">Proteus spp.</td><td colspan="1" rowspan="1">P.aeruginosa</td><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1">S. agalactiae</td><td colspan="1" rowspan="1">S.pneumoniae</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">CTX-M,mecA/C &amp;MREJ</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">M. catarrhalis,Proteus spp., S.aureus, S. agalactiae,S. pneumoniae,[mecA/C &amp; MREJ]</td></tr><tr><td colspan="1" rowspan="1">ACBcomplex</td><td colspan="1" rowspan="1">E.cloacaecomplex</td><td colspan="1" rowspan="1">K.aerogenes</td><td colspan="1" rowspan="1">K.pneumoniaegroup</td><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">ACB complex, K.aerogenes, K.pneumoniae group, S.aureus</td></tr><tr><td colspan="1" rowspan="1">ACBcomplex</td><td colspan="1" rowspan="1">K.aerogenes</td><td colspan="1" rowspan="1">K.pneumoniaegroup</td><td colspan="1" rowspan="1">P.aeruginosa</td><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">KPC,mecA/C &amp;MREJ, VIM</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">K. aerogenes, S.aureus, [mecA/C &amp;MREJ</td></tr><tr><td colspan="1" rowspan="1">ACBcomplex</td><td colspan="1" rowspan="1">M.catarrhalis</td><td colspan="1" rowspan="1">P.aeruginosa</td><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1">S. agalactiae</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">mecA/C &amp;MREJ</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">M. catarrhalis, P.aeruginosa, S.agalactiae</td></tr><tr><td colspan="1" rowspan="1">E.cloacaecomplex</td><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">K.aerogenes</td><td colspan="1" rowspan="1">K.pneumoniaegroup</td><td colspan="1" rowspan="1">M.catarrhalis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">E. cloacae complex,K. aerogenes, K.pneumoniae group,M. catarrhalis</td></tr><tr><td colspan="1" rowspan="1">E.cloacaecomplex</td><td colspan="1" rowspan="1">K. oxytoca</td><td colspan="1" rowspan="1">K.pneumoniaegroup</td><td colspan="1" rowspan="1">S.marcescens</td><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">E. cloacae complex,K. oxytoca, K.pneumoniae group, S.marcescens</td></tr><tr><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">H. influenzae</td><td colspan="1" rowspan="1">K. oxytoca</td><td colspan="1" rowspan="1">P.aeruginosa</td><td colspan="1" rowspan="1">S. agalactiae</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">E. coli, H. influenzae,K. oxytoca, P.aeruginosa, S.agalactiae</td></tr><tr><td colspan="8" rowspan="1">Distinct Co-Detection Combinations</td><td colspan="1" rowspan="2">TotalSpecimenswith Co-DetectionCombination</td><td colspan="1" rowspan="2">#Specimenswith FalsePositiveCo-Detections</td><td colspan="1" rowspan="2">False PositiveAnalyte(s) [FalsePositive AMRGene(s)a</td></tr><tr><td colspan="1" rowspan="1">Analyte1</td><td colspan="1" rowspan="1">Analyte2</td><td colspan="1" rowspan="1">Analyte3</td><td colspan="1" rowspan="1">Analyte4</td><td colspan="1" rowspan="1">Analyte5</td><td colspan="1" rowspan="1">Analyte6</td><td colspan="1" rowspan="1">Analyte7</td><td colspan="1" rowspan="1">AMRGene(s)</td></tr><tr><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">K. oxytoca</td><td colspan="1" rowspan="1">S.marcescens</td><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1">S. agalactiae</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">mecA/C &amp;MREJ</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">K. oxytoca, S.marcescens,S. agalactiae</td></tr><tr><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">M.catarrhalis</td><td colspan="1" rowspan="1">P.aeruginosa</td><td colspan="1" rowspan="1">S. agalactiae</td><td colspan="1" rowspan="1">S.pneumoniae</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">M. catarrhalis</td></tr><tr><td colspan="1" rowspan="1">H.influenzae</td><td colspan="1" rowspan="1">M.catarrhalis</td><td colspan="1" rowspan="1">P.aeruginosa</td><td colspan="1" rowspan="1">S.marcescens</td><td colspan="1" rowspan="1">S. agalactiae</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">H. influenzae, M.catarrhalis,P. aeruginosa, S.marcescens, S.agalactiae</td></tr><tr><td colspan="1" rowspan="1">M.catarrhalis</td><td colspan="1" rowspan="1">Proteus spp.</td><td colspan="1" rowspan="1">P.aeruginosa</td><td colspan="1" rowspan="1">S.marcescens</td><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">mecA/C &amp;MREJ</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">M. catarrhalis, P.aeruginosa, S.marcescens, S. aureus</td></tr><tr><td colspan="1" rowspan="1">M.catarrhalis</td><td colspan="1" rowspan="1">P.aeruginosa</td><td colspan="1" rowspan="1">S.marcescens</td><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1">S.pneumoniae</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">mecA/C &amp;MREJ</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">M. catarrhalis (2), P.aeruginosa (2), S.marcescens (1)</td></tr><tr><td colspan="1" rowspan="1">ACBcomplex</td><td colspan="1" rowspan="1">E.cloacaecomplex</td><td colspan="1" rowspan="1">K. oxytoca</td><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">ACB complex, E.cloacaecomplex</td></tr><tr><td colspan="1" rowspan="1">ACBcomplex</td><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">K.pneumoniaegroup</td><td colspan="1" rowspan="1">P.aeruginosa</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">ACB complex, E.coli, K. pneumoniaegroup</td></tr><tr><td colspan="1" rowspan="1">ACBcomplex</td><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1">S. agalactiae</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">ACB complex, E.coli, S. aureus, S.agalactiae</td></tr><tr><td colspan="1" rowspan="1">ACBcomplex</td><td colspan="1" rowspan="1">K.pneumoniaegroup</td><td colspan="1" rowspan="1">M.catarrhalis</td><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">mecA/C &amp;MREJ</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">M. catarrhalis</td></tr><tr><td colspan="1" rowspan="1">ACBcomplex</td><td colspan="1" rowspan="1">K.pneumoniaegroup</td><td colspan="1" rowspan="1">Proteus spp.</td><td colspan="1" rowspan="1">P.aeruginosa</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">CTX-M,KPC</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">ACB complex,Proteus spp.</td></tr><tr><td colspan="1" rowspan="1">ACBcomplex</td><td colspan="1" rowspan="1">M.catarrhalis</td><td colspan="1" rowspan="1">P.aeruginosa</td><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">ACB complex, M.catarrhalis, P.aeruginosa, S. aureus</td></tr><tr><td colspan="1" rowspan="1">ACBcomplex</td><td colspan="1" rowspan="1">Proteus spp.</td><td colspan="1" rowspan="1">P.aeruginosa</td><td colspan="1" rowspan="1">S.marcescens</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">ACB complex, S.marcescens</td></tr><tr><td colspan="1" rowspan="1">E.cloacaecomplex</td><td colspan="1" rowspan="1">K. oxytoca</td><td colspan="1" rowspan="1">K.pneumoniaegroup</td><td colspan="1" rowspan="1">S.marcescens</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">E. cloacae complex,K. oxytoca, S.marcescens</td></tr><tr><td colspan="1" rowspan="1">E.cloacaecomplex</td><td colspan="1" rowspan="1">K.pneumoniaegroup</td><td colspan="1" rowspan="1">P.aeruginosa</td><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">E. cloacae complex</td></tr><tr><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">H. influenzae</td><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1">S.pneumoniae</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">H. influenzae, S.aureus, S.pneumoniae</td></tr><tr><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">K.pneumoniaegroup</td><td colspan="1" rowspan="1">M.catarrhalis</td><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">mecA/C &amp;MREJ</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">E. coli, K.pneumoniae group, M.catarrhalis, S. aureus</td></tr><tr><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">K.pneumoniaegroup</td><td colspan="1" rowspan="1">P.aeruginosa</td><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">K. pneumoniae group</td></tr><tr><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">Proteus spp.</td><td colspan="1" rowspan="1">P.aeruginosa</td><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">mecA/C &amp;MREJ</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">P.aeruginosa</td><td colspan="1" rowspan="1">S.marcescens</td><td colspan="1" rowspan="1">S.pneumoniae</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">E. coli, P. aeruginosa,S. marcescens, S.pneumoniae</td></tr><tr><td colspan="1" rowspan="1">H.influenzae</td><td colspan="1" rowspan="1">K.aerogenes</td><td colspan="1" rowspan="1">P.aeruginosa</td><td colspan="1" rowspan="1">S. pyogenes</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">K. aerogenes</td></tr><tr><td colspan="1" rowspan="1">H.influenzae</td><td colspan="1" rowspan="1">M.catarrhalis</td><td colspan="1" rowspan="1">P.aeruginosa</td><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">mecA/C &amp;MREJ</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">H. influenzae, M.catarrhalis,P. aeruginosa,[mecA/C &amp; MREJ]</td></tr><tr><td colspan="1" rowspan="1">H.influenzae</td><td colspan="1" rowspan="1">M.catarrhalis</td><td colspan="1" rowspan="1">P.aeruginosa</td><td colspan="1" rowspan="1">S.marcescens</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">M. catarrhalis</td></tr><tr><td colspan="1" rowspan="1">H.influenzae</td><td colspan="1" rowspan="1">P.aeruginosa</td><td colspan="1" rowspan="1">S.marcescens</td><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">H. influenzae, S.marcescens, S. aureus</td></tr><tr><td colspan="1" rowspan="1">H.influenzae</td><td colspan="1" rowspan="1">P.aeruginosa</td><td colspan="1" rowspan="1">S.marcescens</td><td colspan="1" rowspan="1">S.pneumoniae</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">H.influenzae</td><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1">S. agalactiae</td><td colspan="1" rowspan="1">S.pneumoniae</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">H. influenzae, S.agalactiae,S. pneumoniae</td></tr><tr><td colspan="1" rowspan="1">K.pneumoniaegroup</td><td colspan="1" rowspan="1">P.aeruginosa</td><td colspan="1" rowspan="1">S.marcescens</td><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">K. pneumoniae group,S. aureus</td></tr><tr><td colspan="1" rowspan="1">M.catarrhalis</td><td colspan="1" rowspan="1">P.aeruginosa</td><td colspan="1" rowspan="1">S.marcescens</td><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">mecA/C &amp;MREJ</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">M. catarrhalis</td></tr><tr><td colspan="1" rowspan="1">M.catarrhalis</td><td colspan="1" rowspan="1">P.aeruginosa</td><td colspan="1" rowspan="1">S.marcescens</td><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">M. catarrhalis, P.aeruginosa, S.marcescens, S. aureus</td></tr><tr><td colspan="1" rowspan="1">ACBcomplex</td><td colspan="1" rowspan="1">E.cloacaecomplex</td><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">mecA/C &amp;MREJ</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">E. cloacae complex</td></tr><tr><td colspan="1" rowspan="1">ACBcomplex</td><td colspan="1" rowspan="1">H. influenzae</td><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">ACB complex, H.influenzae, S. aureus</td></tr><tr><td colspan="1" rowspan="1">ACBcomplex</td><td colspan="1" rowspan="1">K.pneumoniaegroup</td><td colspan="1" rowspan="1">P.aeruginosa</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">ACB complex, K.pneumoniae group</td></tr><tr><td colspan="1" rowspan="1">ACBcomplex</td><td colspan="1" rowspan="1">Proteus spp.</td><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">mecA/C &amp;MREJ</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">ACBcomplex</td><td colspan="1" rowspan="1">P.aeruginosa</td><td colspan="1" rowspan="1">S.marcescens</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">ACB complex, P.aeruginosa, S.</td></tr><tr><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">EY</td></tr><tr><td colspan="5" rowspan="1">Distinct Co-Detection Combinations</td><td colspan="3" rowspan="1"></td><td colspan="1" rowspan="2">TotalSpecimenswith Co-DetectionCombination</td><td colspan="1" rowspan="2">#Specimenswith FalsePositiveCo-Detections</td><td colspan="1" rowspan="2">False PositiveAnalyte(s) [FalsePositive AMRGene(s)a</td></tr><tr><td colspan="1" rowspan="1">Analyte1</td><td colspan="1" rowspan="1">Analyte2</td><td colspan="1" rowspan="1">Analyte3</td><td colspan="1" rowspan="1">Analyte4</td><td colspan="1" rowspan="1">Analyte5</td><td colspan="1" rowspan="1">Analyte6</td><td colspan="1" rowspan="1">Analyte7</td><td colspan="1" rowspan="1">AMRGene(s)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">marcescens</td></tr><tr><td colspan="1" rowspan="1">ACBcomplex</td><td colspan="1" rowspan="1">P.aeruginosa</td><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">mecA/C &amp;MREJ</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">ACB complex (2), P.aeruginosa (1), S.aureus (1)</td></tr><tr><td colspan="1" rowspan="1">ACBcomplex</td><td colspan="1" rowspan="1">P.aeruginosa</td><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">ACB complex, S.aureus</td></tr><tr><td colspan="1" rowspan="1">ACBcomplex</td><td colspan="1" rowspan="1">P.aeruginosa</td><td colspan="1" rowspan="1">S. agalactiae</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">E.cloacaecomplex</td><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">K. oxytoca</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">E. cloacae complex,K. oxytoca</td></tr><tr><td colspan="1" rowspan="1">E.cloacaecomplex</td><td colspan="1" rowspan="1">H. influenzae</td><td colspan="1" rowspan="1">P.aeruginosa</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">H. influenzae, P.aeruginosa</td></tr><tr><td colspan="1" rowspan="1">E.cloacaecomplex</td><td colspan="1" rowspan="1">H. influenzae</td><td colspan="1" rowspan="1">M.catarrhalis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">H. influenzae, M.catarrhalis</td></tr><tr><td colspan="1" rowspan="1">E.cloacaecomplex</td><td colspan="1" rowspan="1">K.aerogenes</td><td colspan="1" rowspan="1">K.pneumoniaegroup</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">E. cloacae complex,K. pneumoniae group</td></tr><tr><td colspan="1" rowspan="1">E.cloacaecomplex</td><td colspan="1" rowspan="1">K. oxytoca</td><td colspan="1" rowspan="1">M.catarrhalis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">E. cloacae complex,M. catarrhalis</td></tr><tr><td colspan="1" rowspan="1">E.cloacaecomplex</td><td colspan="1" rowspan="1">K.pneumoniaegroup</td><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">E. cloacae complex,K. pneumoniae group,S. aureus</td></tr><tr><td colspan="1" rowspan="1">E.cloacaecomplex</td><td colspan="1" rowspan="1">M.catarrhalis</td><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">M. catarrhalis, S.aureus</td></tr><tr><td colspan="1" rowspan="1">E.cloacaecomplex</td><td colspan="1" rowspan="1">P.aeruginosa</td><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">mecA/C &amp;MREJ</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">E. cloacae complex</td></tr><tr><td colspan="1" rowspan="1">E.cloacaecomplex</td><td colspan="1" rowspan="1">P.aeruginosa</td><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">E. cloacae complex</td></tr><tr><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">H. influenzae</td><td colspan="1" rowspan="1">P.aeruginosa</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">E. coli, H. influenzae,P. aeruginosa</td></tr><tr><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">H. influenzae</td><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">CTX-M</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">H. influenzae, S.aureus</td></tr><tr><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">H. influenzaeS. agalactiae</td><td colspan="1" rowspan="1">H. influenzaeS. agalactiae</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">E. coli, H. influenzae,S. agalactiae</td></tr><tr><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">K.pneumoniaegroup</td><td colspan="1" rowspan="1">P.aeruginosa</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">CTX-M</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">K. pneumoniae group</td></tr><tr><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">K.pneumoniae aeruginosa</td><td colspan="1" rowspan="1">P.pneumoniae aeruginosa</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">E. coli, K.pneumoniae group,</td></tr><tr><td colspan="5" rowspan="1">Distinct Co-Detection Combinations</td><td colspan="3" rowspan="1"></td><td colspan="1" rowspan="2">TotalSpecimenswith Co-DetectionCombination</td><td colspan="1" rowspan="2">#Specimenswith FalsePositiveCo-Detections</td><td colspan="1" rowspan="2">False PositiveAnalyte(s) [FalsePositive AMRGene(s)a</td></tr><tr><td colspan="1" rowspan="1">Analyte1</td><td colspan="1" rowspan="1">Analyte2</td><td colspan="1" rowspan="1">Analyte3</td><td colspan="1" rowspan="1">Analyte4</td><td colspan="1" rowspan="1">Analyte5</td><td colspan="1" rowspan="1">Analyte6</td><td colspan="1" rowspan="1">Analyte7</td><td colspan="1" rowspan="1">AMRGene(s)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">group</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">P. aeruginosa</td></tr><tr><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">K.pneumoniaegroup</td><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">mecA/C &amp;MREJ</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">E. coli, K.pneumoniae group</td></tr><tr><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">K.pneumoniaegroup</td><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">S. aureus</td></tr><tr><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">M.catarrhalis</td><td colspan="1" rowspan="1">P.aeruginosa</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">E. coli, M. catarrhalis</td></tr><tr><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">Proteus spp.</td><td colspan="1" rowspan="1">P.aeruginosa</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">E. coli, P. aeruginosa</td></tr><tr><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">P.aeruginosa</td><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">mecA/C &amp;MREJ</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">P. aeruginosa</td></tr><tr><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">P.aeruginosa</td><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">H.influenzae</td><td colspan="1" rowspan="1">K.pneumoniaegroup</td><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">H. influenzae (1), K.pneumoniae groupb(1), S. aureus (2)</td></tr><tr><td colspan="1" rowspan="1">H.influenzae</td><td colspan="1" rowspan="1">M.catarrhalis</td><td colspan="1" rowspan="1">P.aeruginosa</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">H. influenzae (2), M.catarrhalis (2), P.aeruginosa(1)</td></tr><tr><td colspan="1" rowspan="1">H.influenzae</td><td colspan="1" rowspan="1">M.catarrhalis</td><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">mecA/C &amp;MREJ</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">H. influenzae, M.catarrhalis,S. aureus</td></tr><tr><td colspan="1" rowspan="1">H.influenzae</td><td colspan="1" rowspan="1">M.catarrhalis</td><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">H. influenzae, S.aureus</td></tr><tr><td colspan="1" rowspan="1">H.influenzae</td><td colspan="1" rowspan="1">M.catarrhalis</td><td colspan="1" rowspan="1">S. agalactiae</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">H. influenzae, M.catarrhalis,S. agalactiae</td></tr><tr><td colspan="1" rowspan="1">H.influenzae</td><td colspan="1" rowspan="1">M.catarrhalis</td><td colspan="1" rowspan="1">S.pneumoniae</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">H. influenzae (2), M.catarrhalis (3), S.pneumoniae (2)</td></tr><tr><td colspan="1" rowspan="1">H.influenzae</td><td colspan="1" rowspan="1">P.aeruginosa</td><td colspan="1" rowspan="1">S.marcescens</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">H. influenzae, S.marcescens</td></tr><tr><td colspan="1" rowspan="1">H.influenzae</td><td colspan="1" rowspan="1">P.aeruginosa</td><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">H. influenzae (2), P.aeruginosa (2), S.aureus (1)</td></tr><tr><td colspan="1" rowspan="1">H.influenzae</td><td colspan="1" rowspan="1">P.aeruginosa</td><td colspan="1" rowspan="1">S.pneumoniae</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">H. influenzae, P.aeruginosa, S.pneumoniae</td></tr><tr><td colspan="1" rowspan="1">H.influenzae</td><td colspan="1" rowspan="1">S.marcescens</td><td colspan="1" rowspan="1">S.pneumoniae</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">H. influenzae, S.pneumoniae</td></tr><tr><td colspan="1" rowspan="1">H.influenzae</td><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1">S. agalactiae</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">H. influenzae, S.aureus, S. agalactiae</td></tr><tr><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">EY</td></tr><tr><td colspan="5" rowspan="1">Distinct Co-Detection Combinations</td><td colspan="3" rowspan="1"></td><td colspan="1" rowspan="2">TotalSpecimenswith Co-DetectionCombination</td><td colspan="1" rowspan="2">#Specimenswith FalsePositiveCo-Detections</td><td colspan="1" rowspan="2">False PositiveAnalyte(s) [FalsePositive AMRGene(s)a</td></tr><tr><td colspan="1" rowspan="1">Analyte1</td><td colspan="1" rowspan="1">Analyte2</td><td colspan="1" rowspan="1">Analyte3</td><td colspan="1" rowspan="1">Analyte4</td><td colspan="1" rowspan="1">Analyte5</td><td colspan="1" rowspan="1">Analyte6</td><td colspan="1" rowspan="1">Analyte7</td><td colspan="1" rowspan="1">AMRGene(s)</td></tr><tr><td colspan="1" rowspan="1">H.influenzae</td><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1">S.pneumoniae</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">mecA/C &amp;MREJ</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">H. influenzae</td></tr><tr><td colspan="1" rowspan="1">H.influenzae</td><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1">S. pyogenes</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">H. influenzae</td></tr><tr><td colspan="1" rowspan="1">H.influenzae</td><td colspan="1" rowspan="1">S. agalactiae</td><td colspan="1" rowspan="1">S.pneumoniae</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">S. agalactiae</td></tr><tr><td colspan="1" rowspan="1">K.aerogenes</td><td colspan="1" rowspan="1">Proteus spp.</td><td colspan="1" rowspan="1">P.aeruginosa</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">P. aeruginosa</td></tr><tr><td colspan="1" rowspan="1">K. oxytoca</td><td colspan="1" rowspan="1">P.aeruginosa</td><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">mecA/C &amp;MREJ</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">K. oxytoca</td><td colspan="1" rowspan="1">S.marcescens</td><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">K. oxytoca</td><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1">S. agalactiae</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">mecA/C &amp;MREJ</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">K. oxytoca, S.agalactiae</td></tr><tr><td colspan="1" rowspan="1">K.pneumoniaegroup</td><td colspan="1" rowspan="1">P.aeruginosa</td><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">mecA/C &amp;MREJ</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">K. pneumoniae group(2), P. aeruginosa (1)</td></tr><tr><td colspan="1" rowspan="1">K.pneumoniaegroup</td><td colspan="1" rowspan="1">P.aeruginosa</td><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">K.pneumoniaegroup</td><td colspan="1" rowspan="1">S.marcescens</td><td colspan="1" rowspan="1">S.pneumoniae</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">K. pneumoniae group,S. marcescens, S.pneumoniae</td></tr><tr><td colspan="1" rowspan="1">K.pneumoniaegroup</td><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1">S. agalactiae</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">S. agalactiae</td></tr><tr><td colspan="1" rowspan="1">M.catarrhalis</td><td colspan="1" rowspan="1">Proteus spp.</td><td colspan="1" rowspan="1">S. agalactiae</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">M. catarrhalis,Proteus spp.</td></tr><tr><td colspan="1" rowspan="1">M.catarrhalis</td><td colspan="1" rowspan="1">P.aeruginosa</td><td colspan="1" rowspan="1">S.marcescens</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">M. catarrhalis, S.marcescens</td></tr><tr><td colspan="1" rowspan="1">M.catarrhalis</td><td colspan="1" rowspan="1">P.aeruginosa</td><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">CTX-M,mecA/C &amp;MREJ</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">M. catarrhalis, S.aureus</td></tr><tr><td colspan="1" rowspan="1">M.catarrhalis</td><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1">S.pneumoniae</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">mecA/C &amp;MREJ</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">M. catarrhalis, S.pneumoniae</td></tr><tr><td colspan="1" rowspan="1">M.catarrhalis</td><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1">S.pneumoniae</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">M. catarrhalis (2), S.aureus (1), S.pneumoniae (1)</td></tr><tr><td colspan="1" rowspan="1">Proteus spp.</td><td colspan="1" rowspan="1">P.aeruginosa</td><td colspan="1" rowspan="1">S.marcescens</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">Proteus spp. (2)</td></tr><tr><td colspan="1" rowspan="1">Proteus spp.</td><td colspan="1" rowspan="1">P.aeruginosa</td><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">mecA/C &amp;MREJ</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">S. aureus</td></tr><tr><td colspan="1" rowspan="1">Proteus spp.</td><td colspan="1" rowspan="1">P.aeruginosa</td><td colspan="1" rowspan="1">S.pneumoniae</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Proteus spp.</td></tr><tr><td colspan="1" rowspan="1">P.aeruginosa</td><td colspan="1" rowspan="1">S.marcescens</td><td colspan="1" rowspan="1">S. agalactiae</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">KPC</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">S. agalactiae</td></tr><tr><td colspan="5" rowspan="1">Distinct Co-Detection Combinations</td><td colspan="3" rowspan="1">cYanmaGntlow</td><td colspan="1" rowspan="2">TotalSpecimenswith Co-DetectionCombination</td><td colspan="1" rowspan="2">#Specimenswith FalsePositiveCo-Detections</td><td colspan="1" rowspan="2">False PositiveAnalyte(s)[FalsePositive AMRGene(s)a</td></tr><tr><td colspan="1" rowspan="1">Analyte1</td><td colspan="1" rowspan="1">Analyte2</td><td colspan="1" rowspan="1">Analyte3</td><td colspan="1" rowspan="1">Analyte4</td><td colspan="1" rowspan="1">Analyte5</td><td colspan="1" rowspan="1">Analyte6</td><td colspan="1" rowspan="1">Analyte7</td><td colspan="1" rowspan="1">AMRGene(s)</td></tr><tr><td colspan="1" rowspan="1">P.aeruginosa</td><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1">S. agalactiae</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">mecA/C &amp;MREJ</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">S. aureus (1), S.agalactiae (1)</td></tr><tr><td colspan="1" rowspan="1">P.aeruginosa</td><td colspan="1" rowspan="1">S. agalactiae</td><td colspan="1" rowspan="1">S.pneumoniae</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">S. agalactiae, S.pneumoniae</td></tr><tr><td colspan="1" rowspan="1">ACBcomplex</td><td colspan="1" rowspan="1">E.cloacaecomplex</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">ACB complex</td></tr><tr><td colspan="1" rowspan="1">ACBcomplex</td><td colspan="1" rowspan="1">P.aeruginosa</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">ACB complex</td></tr><tr><td colspan="1" rowspan="1">ACBcomplex</td><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">mecA/C &amp;MREJ</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">ACB complex (1), S.aureus (2), [mecA/C&amp; MREJ (1)]</td></tr><tr><td colspan="1" rowspan="1">E.cloacaecomplex</td><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">E. cloacae complex(1), E. coli (2)</td></tr><tr><td colspan="1" rowspan="1">E.cloacaecomplex</td><td colspan="1" rowspan="1">K. oxytoca</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">E.cloacaecomplex</td><td colspan="1" rowspan="1">K.pneumoniaegroup</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">E. cloacae complex,K. pneumoniae group</td></tr><tr><td colspan="1" rowspan="1">E.cloacaecomplex</td><td colspan="1" rowspan="1">P.aeruginosa</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">E. cloacae complex,P. aeruginosa</td></tr><tr><td colspan="1" rowspan="1">E.cloacaecomplex</td><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">mecA/C &amp;MREJ</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">E. cloacae complex(1)</td></tr><tr><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">H. influenzae</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">H. influenzae</td></tr><tr><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">K.pneumoniaegroup</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">KPC</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">K.pneumoniaegroup</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">K. pneumoniae group</td></tr><tr><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">M.catarrhalis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">E. coli, M. catarrhalis</td></tr><tr><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">Proteus spp.</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">CTX-M</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">E. coli, Proteus spp.</td></tr><tr><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">P.aeruginosa</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">CTX-M</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">P.aeruginosa</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">E. coli (1), P.aeruginosa (1)</td></tr><tr><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">mecA/C &amp;MREJ</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">E. coli (2), S. aureus(2)</td></tr><tr><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">S. aureus</td></tr><tr><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">S. pyogenes</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">S. pyogenes</td></tr><tr><td colspan="5" rowspan="1">Distinct Co-Detection Combinations</td><td colspan="3" rowspan="1"></td><td colspan="1" rowspan="2">TotalSpecimenswith Co-DetectionCombination</td><td colspan="1" rowspan="2">#Specimenswith FalsePositiveCo-Detections</td><td colspan="1" rowspan="2">False PositiveAnalyte(s) [FalsePositive AMRGene(s)a</td></tr><tr><td colspan="1" rowspan="1">Analyte1</td><td colspan="1" rowspan="1">Analyte2</td><td colspan="1" rowspan="1">Analyte3</td><td colspan="1" rowspan="1">Analyte4</td><td colspan="1" rowspan="1">Analyte5</td><td colspan="1" rowspan="1">Analyte6</td><td colspan="1" rowspan="1">Analyte7</td><td colspan="1" rowspan="1">AMRGene(s)</td></tr><tr><td colspan="1" rowspan="1">H.influenzae</td><td colspan="1" rowspan="1">M.catarrhalis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">H. influenzae (4), M.catarrhalis (6)</td></tr><tr><td colspan="1" rowspan="1">H.influenzae</td><td colspan="1" rowspan="1">Proteus spp.</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">H. influenzae</td></tr><tr><td colspan="1" rowspan="1">H.influenzae</td><td colspan="1" rowspan="1">P.aeruginosa</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">H. influenzae</td></tr><tr><td colspan="1" rowspan="1">H.influenzae</td><td colspan="1" rowspan="1">S.marcescens</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">H. influenzae</td></tr><tr><td colspan="1" rowspan="1">H.influenzae</td><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">mecA/C &amp;MREJ</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">H. influenzae (2), S.aureus (2), [mecA/C&amp; MREJ (1)</td></tr><tr><td colspan="1" rowspan="1">H.influenzae</td><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">9</td><td colspan="1" rowspan="1">H. influenzae (9), S.aureus(6)</td></tr><tr><td colspan="1" rowspan="1">H.influenzae</td><td colspan="1" rowspan="1">S. agalactiae</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">H. influenzae</td></tr><tr><td colspan="1" rowspan="1">H.influenzae</td><td colspan="1" rowspan="1">S.pneumoniae</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">H. influenzae (4), S.pneumoniae (4)</td></tr><tr><td colspan="1" rowspan="1">K.aerogenes</td><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">K. aerogenes (2), S.aureus (2)</td></tr><tr><td colspan="1" rowspan="1">K. oxytoca</td><td colspan="1" rowspan="1">K.pneumoniaegroup</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">K. oxytoca</td><td colspan="1" rowspan="1">M.catarrhalis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">K. oxytoca, M.catarrhalis</td></tr><tr><td colspan="1" rowspan="1">K. oxytoca</td><td colspan="1" rowspan="1">P.aeruginosa</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">K. oxytoca</td></tr><tr><td colspan="1" rowspan="1">K. oxytoca</td><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">K.pneumoniaegroup</td><td colspan="1" rowspan="1">P.aeruginosa</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">KPC</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">K. pneumoniae group</td></tr><tr><td colspan="1" rowspan="1">K.pneumoniaegroup</td><td colspan="1" rowspan="1">P.aeruginosa</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">K. pneumoniae group(4), P. aeruginosa (2)</td></tr><tr><td colspan="1" rowspan="1">K.pneumoniaegroup</td><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">CTX-M,mecA/C &amp;MREJ</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">K. pneumoniae group</td></tr><tr><td colspan="1" rowspan="1">K.pneumoniaegroup</td><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">mecA/C &amp;MREJ</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">K. pneumoniae group(1), S. aureus (2),[mecA/C &amp; MREJ(2)]</td></tr><tr><td colspan="1" rowspan="1">K.pneumoniae S. agalactiaegroup</td><td colspan="1" rowspan="1">pneumoniae S. agalactiae</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">K. pneumoniae group,S. agalactiae</td></tr><tr><td colspan="1" rowspan="1">M.catarrhalis</td><td colspan="1" rowspan="1">P.aeruginosa</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">M. catarrhalis (2), P.aeruginosa (2)</td></tr><tr><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">EY</td></tr><tr><td colspan="5" rowspan="1">Distinct Co-Detection Combinations</td><td colspan="3" rowspan="1"></td><td colspan="1" rowspan="2">TotalSpecimenswith Co-DetectionCombination</td><td colspan="1" rowspan="2">#Specimenswith FalsePositiveCo-Detections</td><td colspan="1" rowspan="2">False PositiveAnalyte(s) [FalsePositive AMRGene(s)a</td></tr><tr><td colspan="1" rowspan="1">Analyte1</td><td colspan="1" rowspan="1">Analyte2</td><td colspan="1" rowspan="1">Analyte3</td><td colspan="1" rowspan="1">Analyte4</td><td colspan="1" rowspan="1">Analyte5</td><td colspan="1" rowspan="1">Analyte6</td><td colspan="1" rowspan="1">Analyte7</td><td colspan="1" rowspan="1">AMRGene(s)</td></tr><tr><td colspan="1" rowspan="1">M.catarrhalis</td><td colspan="1" rowspan="1">S.marcescens</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">M. catarrhalis (2)</td></tr><tr><td colspan="1" rowspan="1">M.catarrhalis</td><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">mecA/C &amp;MREJ</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">M. catarrhalis (3), S.aureus (2)</td></tr><tr><td colspan="1" rowspan="1">M.catarrhalis</td><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">M. catarrhalis (1), S.aureus(1)</td></tr><tr><td colspan="1" rowspan="1">M.catarrhalis</td><td colspan="1" rowspan="1">S. agalactiae</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">M. catarrhalis</td></tr><tr><td colspan="1" rowspan="1">M.catarrhalis</td><td colspan="1" rowspan="1">S.pneumoniae</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">M. catarrhalis (4), S.pneumoniae (3)</td></tr><tr><td colspan="1" rowspan="1">Proteus spp.</td><td colspan="1" rowspan="1">P.aeruginosa</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Proteus spp.</td><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">mecA/C &amp;MREJ</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">P.aeruginosa</td><td colspan="1" rowspan="1">S.marcescens</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">7</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">P. aeruginosa (2), S.marcescens (2)</td></tr><tr><td colspan="1" rowspan="1">P.aeruginosa</td><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">CTX-M,mecA/C &amp;MREJ</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">[CTX-M, mecA/C &amp;MREJ]</td></tr><tr><td colspan="1" rowspan="1">P.aeruginosa</td><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">mecA/C &amp;MREJ</td><td colspan="1" rowspan="1">12</td><td colspan="1" rowspan="1">8</td><td colspan="1" rowspan="1">P. aeruginosa (4), S.aureus (8)</td></tr><tr><td colspan="1" rowspan="1">P.aeruginosa</td><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">P. aeruginosa (2), S.aureus (1)</td></tr><tr><td colspan="1" rowspan="1">P.aeruginosa</td><td colspan="1" rowspan="1">S.pneumoniae</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">P. aeruginosa, S.pneumoniae</td></tr><tr><td colspan="1" rowspan="1">S.marcescens</td><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">mecA/C &amp;MREJ</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">S. marcescens (2), S.aureus (1), [mecA/C&amp; MREJ (1)]</td></tr><tr><td colspan="1" rowspan="1">S.marcescens</td><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">S. marcescens (1), S.aureus (1)</td></tr><tr><td colspan="1" rowspan="1">S.marcescens</td><td colspan="1" rowspan="1">S. pyogenes</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">VIM</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">S. marcescensb, S.pyogenes, [VIM]</td></tr><tr><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1">S. agalactiae</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">mecA/C &amp;MREJ</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">S. aureus (2), S.agalactiae (4)</td></tr><tr><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1">. agalactiae</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">S. aureus (2), S.agalactiae (3)</td></tr><tr><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1">S.pneumoniae</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">mecA/C &amp;MREJ</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">S. pneumoniae,[mecA/C &amp; MREJ]</td></tr><tr><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1">S.pneumoniae</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">S. aureus (2), S.pneumoniae (4)</td></tr><tr><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1">S. pyogenes</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">mecA/C &amp;MREJ</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="8" rowspan="2">Total Co-DetectionsTotal Double Detections</td><td colspan="1" rowspan="1">246</td><td colspan="1" rowspan="1">209</td><td colspan="1" rowspan="1">392b/667</td></tr><tr><td colspan="1" rowspan="1">135</td><td colspan="1" rowspan="1">106</td><td colspan="1" rowspan="1">155/270</td></tr><tr><td colspan="8" rowspan="1">Distinct Co-Detection Combinations</td><td colspan="1" rowspan="2">TotalSpecimenswith Co-DetectionCombination</td><td colspan="1" rowspan="2">#Specimenswith FalsePositiveCo-Detections</td><td colspan="1" rowspan="2">False PositiveAnalyte(s) [FalsePositive AMRGene(s)a</td></tr><tr><td colspan="1" rowspan="1">Analyte1</td><td colspan="1" rowspan="1">Analyte2</td><td colspan="1" rowspan="1">Analyte3</td><td colspan="1" rowspan="1">Analyte4</td><td colspan="1" rowspan="1">Analyte5</td><td colspan="1" rowspan="1">Analyte6</td><td colspan="1" rowspan="1">Analyte7</td><td colspan="1" rowspan="1">AMRGene(s)</td></tr><tr><td colspan="8" rowspan="1">Total Triple Detections</td><td colspan="1" rowspan="1">71</td><td colspan="1" rowspan="1">65</td><td colspan="1" rowspan="1">125/213</td></tr><tr><td colspan="8" rowspan="1">Total Quadruple Detections</td><td colspan="1" rowspan="1">23</td><td colspan="1" rowspan="1">21</td><td colspan="1" rowspan="1">52/92</td></tr><tr><td colspan="8" rowspan="1">Total Quintuple Detections</td><td colspan="1" rowspan="1">12</td><td colspan="1" rowspan="1">12</td><td colspan="1" rowspan="1">40/60</td></tr><tr><td colspan="8" rowspan="1">Total Sextuple Detections</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">11/18</td></tr><tr><td colspan="8" rowspan="1">Total Septuple Detections</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">9/14</td></tr></table>

b Discrepant analyte could not be confirmed as being present in the specimen during discrepancy investigation.   
c Of the 392 discrepant analytes (out of 667 total analytes), 390 $( 9 9 . 5 \% )$ were observed as being present in the specimen during discrepancy investigation; 79/392 $( 2 0 . 2 \% )$ were enumerated below $1 0 ^ { 3 . 5 } \mathrm { C F U / m L }$ by qRefCx, 271/392 $( 6 9 . 1 \% )$ ) were detected by qMol, 34/392 $( 8 . 7 \% )$ were detected using an additional molecular method, and 6/392 $( 1 . 5 \% )$ were identifed in SOC culture.

# N. Instrument Name:

FilmArray, FilmArray 2.0, or FilmArray Torch

# O. System Descriptions:

1. Modes of Operation:

Does the applicant’s device contain the ability to transmit data to a computer, webserver, or mobile device?

Yes or No ___X_

Does the applicant’s device transmit data to a computer, webserver, or mobile device using wireless transmission? Yes or No __X_

2. Software: FDA has reviewed applicant’s Hazard Analysis and software development processes for this line of product types: Yes ___X_____ or No

Level of Concern Moderate

Software Description   
A detailed description of the FilmArray software was provided and included the function of each FilmArray System software component.

# Device Hazard Analysis:

The firm performed a risk analysis on the FilmArray Software pouch modules to identify risks and their possible causes in accordance with their Risk Analysis SOP. Risks associated with the FilmArray Pouch were categorized into two types—

• System risks with mitigations in the FilmArray Pouch Module Software, or Software risks isolated to the FilmArray Pouch Module Software

For each risk identified, the risk analysis determines if mitigation is required. Hazards were classified into four types: false positive results, false negative results, assay reporting errors, and delayed test results. For each of these hazard types, a severity of harm was defined by a medical expert with the following rationale—

<table><tr><td rowspan=1 colspan=1>Hazard</td><td rowspan=1 colspan=1>Severityof Harm</td><td rowspan=1 colspan=1>Rationale</td></tr><tr><td rowspan=1 colspan=1>FalsePositive</td><td rowspan=1 colspan=1>M</td><td rowspan=1 colspan=1>The patient is treated unnecessarily and there is delayed treatment forthe true cause. Unnecessary precautions and expenses are incurred.</td></tr><tr><td rowspan=1 colspan=1>FalseNegative</td><td rowspan=1 colspan=1>M</td><td rowspan=1 colspan=1>Risk scenarios are misjudged. The patient may not receive necessarytreatment.</td></tr><tr><td rowspan=1 colspan=1>Delayed/NoResult</td><td rowspan=1 colspan=1>L</td><td rowspan=1 colspan=1>The test must be re-run, resulting in delayed treatment.</td></tr><tr><td rowspan=1 colspan=1>ReportingError</td><td rowspan=1 colspan=1>M</td><td rowspan=1 colspan=1>Results for validated assays may not be reported, or results for un-validated assays may be reported.</td></tr></table>

$\mathbf { M } =$ medium risk, $\mathrm { L } =$ low risk

Risk scores were calculated as a function of severity, the likelihood of occurrence, and the likelihood of the hazardous situation. Mitigations for potential hazards include qualifying calls by checking the Tm and checking for potential errors associated with the melt data analysis. As part of the release of the FilmArray Respiratory Panel test, the Melt Detector was optimized on a training data set and validated against the Clinical Evaluation Data. The FilmArray Pouch Module Software was also tested and successfully executed to demonstrate compatibility with the FilmArray software.

In addition, because the panel can be used on both the FilmArray v1.5 and $\mathrm { v } 2 . 0$ Systems, the Analysis Software compares the proper .dll files as part of verification testing to ensure that the proper files are used. The FilmArray pouch module software is designed so that pouch modules compatible with the FilmArray 1.5 systems cannot be installed on the FilmArray 2.0 systems and vice versa. Note, the firm states that the FilmArray Torch software is similar to the FilmArray 2.0 system software.

The firm also stated that as part of the release of the FilmArray Respiratory Panel, a regression test was successfully executed to show that the release candidate pouch module produced the same results as those submitted for review.

Further, as detailed in the cybersecurity review, the firm has implemented measures such as encrypting database passwords to address the potential risk for modification of test data in the database that could impact listed test results.

After mitigation, all risk scores were at a Low or Immeasurable level.

Architecture Design Chart:

A detailed structure of the software used in the FilmArray System was provided.

Software Requirements Specification (SRS):   
SRS documentation was provided describing requirements and specifications for each of the software components of the FilmArray System was described.

Documentation of traceability matrix that links all product requirements, functional specifications, and verification and validation testing for the complete FilmArray system was provided.

Software Development Environment Description:

A description of FilmArray software development environment was provided and was acceptable.

Verification and Validation Testing:

The sponsor provided adequate documentation of verification and validation (V & V) testing covering all software/instrument components of the FilmArray System. V &V testing of FilmArray System software was successfully completed at the individual component and system integration levels. The normal operation and user interface of all FilmArray software components were also tested and verified.

# Revision Level History:

The firm provided a software revision level history that detailed the updates to the system software corresponding to each version.

# Unresolved Anomalies:

All major residual risks and unresolved anomalies were properly mitigated. Any remaining anomalies did not present major concerns for safety and efficacy for either the user or the patient.

# EMC Testing:

This submission does not include any new hardware and therefore there are no updates requiring EMC testing.

5. Specimen Identification: The Sample ID can be entered manually or scanned in by using the FilmArray barcode scanner.

6. Specimen Sampling and Handling: N/A

7. Calibration: N/A

6. Quality Control: See section M (d) for information on internal and external controls.

O. Other Supportive Instrument Performance Characteristics Data Not Covered In The “Performance Characteristics” Section above: Not applicable

# P. Proposed Labeling:

The labeling is sufficient and satisfies the requirements of 21 CFR parts 801 and 809 as well as the Special Controls for this type of device.

# Q. Patient Perspectives

This submission did not include specific information on patient perspectives for this device.

# R. Conclusion:

The submitted information in this premarket notification is complete and supports a substantial equivalence decision.

The potential value of “semi quantitative” bin reporting to help determine the clinical significance of the pathogens/resistance markers reported needs to be established by the individual testing site in consideration of current testing practices.

Reporting and interpretation can be patient/institution specific and guidelines should be established and local education facilitated by joint decisions between various clinical care groups (Laboratory, Infectious Diseases, Pulmonologists, Intensivists, Infection Control, Pharmacy, antibiotic stewardship committee, etc.)