GenMark Diagnostics, Incorporated   
Alan Maderazo   
VP, Quality, Regulatory & Clinical Affairs   
5964 La Place Court   
Carlsbad, California 92008

Re: K181663 Trade/Device Name: ePlex Blood Culture Identification Panel - Gram Positive (BCID-GP) Panel Regulation Number: 21 CFR 866.3365 Regulation Name: Multiplex nucleic acid assay for identification of microorganisms and resistance markers from positive blood cultures Regulatory Class: Class II Product Code: PAM, PEN, PEO Dated: June 22, 2018 Received: June 25, 2018

Dear Alan Maderazo:

We have reviewed your Section 510(k) premarket notification of intent to market the device referenced above and have determined the device is substantially equivalent (for the indications for use stated in the enclosure) to legally marketed predicate devices marketed in interstate commerce prior to May 28, 1976, the enactment date of the Medical Device Amendments, or to devices that have been reclassified in accordance with the provisions of the Federal Food, Drug, and Cosmetic Act (Act) that do not require approval of a premarket approval application (PMA). You may, therefore, market the device, subject to the general controls provisions of the Act. Although this letter refers to your product as a device, please be aware that some cleared products may instead be combination products. The 510(k) Premarket Notification Database located at https://www.accessdata.fda.gov/scripts/cdrh/cfdocs/cfpmn/pmn.cfm identifies combination product submissions. The general controls provisions of the Act include requirements for annual registration, listing of devices, good manufacturing practice, labeling, and prohibitions against misbranding and adulteration. Please note: CDRH does not evaluate information related to contract liability warranties. We remind you, however, that device labeling must be truthful and not misleading.

If your device is classified (see above) into either class II (Special Controls) or class III (PMA), it may be subject to additional controls. Existing major regulations affecting your device can be found in the Code of Federal Regulations, Title 21, Parts 800 to 898. In addition, FDA may publish further announcements concerning your device in the Federal Register.

Please be advised that FDA's issuance of a substantial equivalence determination does not mean that FDA has made a determination that your device complies with other requirements of the Act or any Federal

statutes and regulations administered by other Federal agencies. You must comply with all the Act's requirements, including, but not limited to: registration and listing (21 CFR Part 807); labeling (21 CFR Part 801 and Part 809); medical device reporting (reporting of medical device-related adverse events) (21 CFR 803) for devices or postmarketing safety reporting (21 CFR 4, Subpart B) for combination products (see https://www.fda.gov/CombinationProducts/GuidanceRegulatoryInformation/ucm597488.htm); good manufacturing practice requirements as set forth in the quality systems (QS) regulation (21 CFR Part 820) for devices or current good manufacturing practices (21 CFR 4, Subpart A) for combination products; and, if applicable, the electronic product radiation control provisions (Sections 531-542 of the Act); 21 CFR 1000- 1050.

Also, please note the regulation entitled, "Misbranding by reference to premarket notification" (21 CFR Part 807.97). For questions regarding the reporting of adverse events under the MDR regulation (21 CFR Part 803), please go to http://www.fda.gov/MedicalDevices/Safety/ReportaProblem/default.htm.

For comprehensive regulatory information about medical devices and radiation-emitting products, including information about labeling regulations, please see Device Advice   
(https://www.fda.gov/MedicalDevices/DeviceRegulationandGuidance/) and CDRH Learn   
(http://www.fda.gov/Training/CDRHLearn). Additionally, you may contact the Division of Industry and Consumer Education (DICE) to ask a question about a specific regulatory topic. See the DICE website (http://www.fda.gov/DICE) for more information or contact DICE by email (DICE@fda.hhs.gov) or phone (1-800-638-2041 or 301-796-7100).

Sincerely,

# Uwe Scherf -S

Uwe Scherf, M.Sc., Ph.D.   
Director   
Division of Microbiology Devices   
Office of In Vitro Diagnostics   
and Radiological Health   
Center for Devices and Radiological Health

Enclosure

510(k) Number (if known) K181663

Device Name ePlex Blood Culture Identification Gram-Positive (BCID-GP) Panel

Indications for Use (Describe)

The GenMark ePlex Blood Culture Identification Gram-Positive (BCID-GP) Panel is a qualitative nucleic acid multiplex in vitro diagnostic test intended for use on GenMark's ePlex Instrument for simultaneous qualitative detection and identification of multiple potentially pathogenic gram-positive bacterial organisms and select determinants associated with antimicrobial resistance in positive blood culture. In addition, the ePlex BCID-GP Panel is capable of detecting a wide variety of gram-negative bacteria (Pan Gram-Negative assay) and several Candida species (Pan Candida assay). The ePlex BCID-GP Panel is performed directly on blood culture samples identified as positive by a continuous monitoring blood culture system and which contain gram-positive organism.

The following bacterial organisms and genes associated with antibiotic resistance are identified using the ePlex BCID-GP Panel: Bacillus cereus group, Bacillus subtilis group, Corynebacterium, Cutibacterium acnes (Propionibacterium acnes), Enterococcus, Enterococcus faecalis, Enterococcus faecium, Lactobacillus, Listeria, Listeria monocytogenes, Micrococcus, Staphylococcus, Staphylococcus aureus, Staphylococcus epidermidis, Staphylococcus lugdunensis, Streptococcus, Streptococcus agalactiae (GBS), Streptococcus anginosus group, Streptococcus pneumoniae, Streptococcus pyogenes (GAS), mecA, mecC, vanA and vanB.

The ePlex BCID-GP Panel contains assays for the detection of genetic determinants associated with resistance to methicillin (mecA and mecC) and vancomycin (vanA and vanB) to aid in the identification of potentially antimicrobial resistant organisms in positive blood culture samples. The antimicrobial resistance gene detected may or may not be associated with the agent responsible for disease.

The ePlex BCID-GP Panel also contains targets designed to detect a broad range of organisms with a potentially misleading Gram stain result or organisms that may be missed by Gram staining altogether, for example in the case of co-infections. These include a broad Pan GramNegative assay as well as a Pan Candida assay, which is designed to detect four of the most prevalent Candida species: Candida albicans, Candida glabrata, Candida krusei and Candida parapsilosis.

The detection and identification of specific bacterial and fungal nucleic acids from individuals exhibiting signs and/or symptoms of bloodstream infection aids in the diagnosis of bloodstream infection when used in conjunction with other clinical information. The results from the ePlex BCID-GP Panel are intended to be interpreted in conjunction with Gram stain results and should not be used as the sole basis for diagnosis, treatment, or other patient management decisions.

Negative results in the setting of a suspected bloodstream infection may be due to infection with pathogens that are not detected by this test. Positive results do not rule out co-infection with other organisms; the organism(s) detected by the ePlex BCID-GP Panel may not be the definite cause of disease. Additional laboratory testing (e.g. sub-culturing of positive blood cultures for identification of organisms not detected by ePlex BCID-GP Panel and for susceptibility testing, differentiation of mixed growth and association of antimicrobial resistance marker genes to a specific organism) and clinical presentation must be taken into consideration in the final diagnosis of blood stream infection.

# CONTINUE ON A SEPARATE PAGE IF NEEDED.

This section applies only to requirements of the Paperwork Reduction Act of 1995.

# \*DO NOT SEND YOUR COMPLETED FORM TO THE PRA STAFF EMAIL ADDRESS BELOW.\*

The burden time for this collection of information is estimated to average 79 hours per response, including the time to review instructions, search existing data sources, gather and maintain the data needed and complete and review the collection of information. Send comments regarding this burden estimate or any other aspect of this information collection, including suggestions for reducing this burden, to:

Department of Health and Human Services Food and Drug Administration Office of Chief Information Officer Paperwork Reduction Act (PRA) Staff PRAStaff@fda.hhs.gov

“An agency may not conduct or sponsor, and a person is not required to respond to, a collection of information unless it displays a currently valid OMB number.”

# 510(k) Summary

Summary of Safety and Effectiveness

# Submitter Information

Submitter: GenMark Diagnostics, Incorporated 5964 La Place Court Carlsbad, CA 92008

Manufacturer: GenMark Diagnostics, Incorporated 5964 La Place Court Carlsbad, CA 92008

Establishment Registration Number: 3008632402

Contact:

Phone:

# Name of Device and Classification

Product Name: ePlex® Blood Culture Identification Gram-Positive (BCID-GP) Panel

Device Classification: 866.3980, Multiplex nucleic acid assay for identification of microorganisms and resistance markers from positive blood cultures, Class II

Product Code(s): PAM, PEN, PEO

# Predicate Device

Predicate:

# FilmArray Blood Culture Identification Panel; BioFire Diagnostics; K130914

# Device Description

The ePlex Blood Culture Identification Gram-Positive (BCID-GP) Panel is based on the principles of competitive nucleic acid hybridization using a sandwich assay format, wherein a single-stranded target binds concurrently to a sequence-specific solution-phase signal probe and a solid-phase electrode-bound capture probe. The test employs nucleic acid extraction, target amplification via polymerase chain reaction (PCR) or reverse transcription PCR (RT-PCR) and hybridization of target DNA. In the process, the double-stranded PCR amplicons are digested with exonuclease to generate single-stranded DNA suitable for hybridization.

Nucleic acid extraction from biological samples occurs within the cartridge via cell lysis, nucleic acid capture onto magnetic beads, and release for amplification. The nucleic acid extraction is processed through microfluidic liquid handling. Once the nucleic acid targets are captured and inhibitors are washed away, the magnetic particles are delivered to the electrowetting environment on the printed circuit board (PCB) and the targets are eluted from the particles and amplified.

During hybridization, the single-stranded target DNA binds to a complementary, single-stranded capture probe immobilized on the working gold electrode surface. Single-stranded signal probes (labeled with electrochemically active ferrocenes) bind to specific target sequence / region adjacent to the capture probe. Simultaneous hybridization of target to signal probes and capture probe is detected by alternating current voltammetry (ACV). Each working electrode on the array contains specific capture probes, and sequential analysis of each electrode allows detection of multiple analyte targets.

# Intended Use/Indications for Use

The GenMark ePlex® Blood Culture Identification Gram-Positive (BCID-GP) Panel is a qualitative nucleic acid multiplex in vitro diagnostic test intended for use on GenMark’s ePlex Instrument for simultaneous qualitative detection and identification of multiple potentially pathogenic gram-positive bacterial organisms and select determinants associated with antimicrobial resistance in positive blood culture. In addition, the ePlex BCID-GP Panel is capable of detecting a wide variety of gram-negative bacteria (Pan Gram-Negative assay) and several Candida species (Pan Candida assay). The ePlex BCID-GP Panel is performed directly on blood culture samples identified as positive by a continuous monitoring blood culture system and which contain gram-positive organism.

The following bacterial organisms and genes associated with antibiotic resistance are identified using the ePlex BCID-GP Panel: Bacillus cereus group, Bacillus subtilis group, Corynebacterium, Cutibacterium acnes (Propionibacterium acnes), Enterococcus, Enterococcus faecalis, Enterococcus faecium, Lactobacillus, Listeria, Listeria monocytogenes, Micrococcus, Staphylococcus, Staphylococcus aureus, Staphylococcus epidermidis, Staphylococcus lugdunensis, Streptococcus, Streptococcus agalactiae (GBS), Streptococcus anginosus group, Streptococcus pneumoniae, Streptococcus pyogenes (GAS), mecA, mecC, vanA and vanB.

The ePlex BCID-GP Panel contains assays for the detection of genetic determinants associated with resistance to methicillin (mecA and mecC) and vancomycin (vanA and vanB) to aid in the identification of potentially antimicrobial resistant organisms in positive blood culture samples. The antimicrobial resistance gene detected may or may not be associated with the agent responsible for disease.

The ePlex BCID-GP Panel also contains targets designed to detect a broad range of organisms with a potentially misleading Gram stain result or organisms that may be missed by Gram staining altogether, for example in the case of co-infections. These include a broad Pan GramNegative assay as well as a Pan Candida assay, which is designed to detect four of the most prevalent Candida species: Candida albicans, Candida glabrata, Candida krusei and Candida parapsilosis.

The detection and identification of specific bacterial and fungal nucleic acids from individuals exhibiting signs and/or symptoms of bloodstream infection aids in the diagnosis of bloodstream infection when used in conjunction with other clinical information. The results from the ePlex BCID-GP Panel are intended to be interpreted in conjunction with Gram stain results and should not be used as the sole basis for diagnosis, treatment, or other patient management decisions.

Negative results in the setting of a suspected bloodstream infection may be due to infection with pathogens that are not detected by this test. Positive results do not rule out co-infection with other organisms; the organism(s) detected by the ePlex BCID-GP Panel may not be the definite cause of disease. Additional laboratory testing (e.g. sub-culturing of positive blood cultures for identification of organisms not detected by ePlex BCID-GP Panel and for susceptibility testing, differentiation of mixed growth and association of antimicrobial resistance marker genes to a specific organism) and clinical presentation must be taken into consideration in the final diagnosis of blood stream infection.

# Summary of Technological Characteristics of the Device Compared to the Predicate Device

The GenMark ePlex Blood Culture Identification Gram-Positive (BCID-GP) Panel (“Subject Device”) and the legally marketed device, FilmArray Blood Culture Identification Gram-Positive Panel, K130914, (“Predicate Device”) are described below:

<table><tr><td>Characteristic</td><td>Subject Device</td><td>Predicate Device</td></tr><tr><td>Product Name</td><td>ePlex BCID-GP Panel</td><td>FilmArray BCID Panel</td></tr><tr><td>Manufacturer</td><td>GenMark Diagnostics, Inc.</td><td>BioFire Diagnostics, Inc.</td></tr><tr><td>Organisms Detected</td><td>Bacillus cereus group • Bacillus subtilis group Corynebacterium Cutibacterium acnes (P. acnes) Enterococcus Enterococcus facealis Enterococcus faecium Lactobacillus Listeria Listeria monocytogenes Micrococcus Staphylococcus Staphylococcus aureus Staphylococcus epidermidis</td><td>Enterococci, Listeria monocytogenes, commonly encountered Staphylococci (including specific differentiation of Staphylococcus aureus), commonly encountered Streptococci (with specific differentiation of Streptococcus agalactiae, Streptococcus pneumoniae, and Streptococcus pyogenes), Acinetobacter baumannii, commonly encountered Enterobacteriaceae (including specific differentiation of the Enterobacter cloacae complex, Escherichia coli, Klebsiella oxytoca,</td></tr><tr><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1">• Staphylococcus lugdunensisStreptococcusStreptococcus agalactiae (GBS)Streptococcus anginosus groupStreptococcus pneumoniaStreptococcus pyogenes (GAS)Pan Gram-NegativePan Candida</td><td colspan="1" rowspan="1">Klebsiella pneumoniae, Proteus, andSerratia marcescens), Haemophilusinfluenzae, Neisseria meningitidis(encapsulated), Pseudomonasaeruginosa, Candida albicans,Candida glabrata, Candida krusei,Candida parapsilosis, and Candidatropicalis.</td></tr><tr><td colspan="1" rowspan="1">Resistance GenesDetected</td><td colspan="1" rowspan="1">Genetic determinants of resistance tomethicillin (mecA and mecC) andvancomycin (vanA and vanB)</td><td colspan="1" rowspan="1">Genetic determinants of resistance tomethicillin (mecA), vancomycin(vanA and vanB), and carbapenems(blakpc)</td></tr><tr><td colspan="1" rowspan="1">Indication forUse</td><td colspan="1" rowspan="1">The GenMark BCID-GP panel isindicated as an aid in the diagnosis ofspecific agents of bacteremia. The useof additional laboratory testing (e.g.sub-culturing of positive blood culturesfor identification of organisms notdetected by BCID-GP Panel and forsusceptibility testing, differentiation ofmixed growth, and association ofantimicrobial resistance marker genes toa specific organism) and clinicalpresentation must be taken intoconsideration in the final diagnosis ofblood stream infection.</td><td colspan="1" rowspan="1">FilmArray BCID is indicated as an aidin the diagnosis of specific agents ofbacteremia and fungemia and resultsshould be used in conjunction withother clinical and laboratory findings.</td></tr><tr><td colspan="1" rowspan="1">Specimen Type</td><td colspan="1" rowspan="1">Gram-Positive Blood Culture</td><td colspan="1" rowspan="1">Gram-Positive &amp; Gram-NegativeBlood Culture</td></tr><tr><td colspan="1" rowspan="1">Chemistry</td><td colspan="1" rowspan="1">Reagents on cartridge include: samplelysis and nucleic acid extraction, PCRamplification and hybridization-basedelectrochemical detection reagents.</td><td colspan="1" rowspan="1">The FilmArray BCID pouch containsfreeze-dried reagents to performnucleic acid purification and nested,multiplex PCR with DNA meltanalysis.</td></tr><tr><td colspan="1" rowspan="1">Hardware</td><td colspan="1" rowspan="1">GenMark ePlex Instrument &amp; SingleUse Cartridge</td><td colspan="1" rowspan="1">FilmArray Instrument and assay pouch</td></tr><tr><td colspan="1" rowspan="1">SoftwareInterfaceResult Reporting</td><td colspan="1" rowspan="1">GenMark ePlex System Software−  GenMark ePlex BCID-GP PanelSoftware</td><td colspan="1" rowspan="1">The FilmArray Software automaticallyinterprets the results of each DNAmelt curve analysis and combines thedata with the results of the internalpouch controls to provide a test resultfor each organism and antimicrobialresistance gene on the panel.</td></tr></table>

Analysis of the similarities and differences indicate that the devices are substantially equivalent in their intended uses/indications for use, and are generally the same regarding user process, ease of use and general operator protocol. Comparison of technological similarities and differences between the proposed device and the predicate do not raise new or different questions of safety and effectiveness, and therefore render the proposed device as substantially equivalent to the predicate device.

# Summary of Performance Data

# EXPECTED VALUES

A prospective, multicenter clinical study was conducted to evaluate the clinical performance of the ePlex BCID-GP Panel in positive blood culture samples. A total of 711 samples were prospectively collected at 7 clinical sites in 2 phases from patients of all ages and genders. In the first phase from June 2014 through July 2016, 399 samples were prospectively collected and frozen; from January through February 2018, 312 samples were prospectively collected and tested fresh (never frozen). The expected values of individual analytes based on the ePlex BCIDGP Panel results in prospective samples are summarized by age group and by site in Table 1 and Table 2, respectively.

Table 1: Expected Value by Age Group (Prospective Samples)   

<table><tr><td rowspan=1 colspan=1>Target</td><td rowspan=1 colspan=1>All Ages(N=711</td><td rowspan=1 colspan=1>Age &lt;1N=27)</td><td rowspan=1 colspan=1>Age 1-17(N=42)</td><td rowspan=1 colspan=1>Age 18-44N =121)</td><td rowspan=1 colspan=1>Age 45-64(N=2500)</td><td rowspan=1 colspan=1>Age 65-84(N=217)</td><td rowspan=1 colspan=1>Age 85+(N=54)</td></tr><tr><td rowspan=1 colspan=1>Bacillus cereus group</td><td rowspan=1 colspan=1>5(0.7)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>3 (2.5)</td><td rowspan=1 colspan=1>2 (0.8)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td></tr><tr><td rowspan=1 colspan=1>Bacillus subtilis group</td><td rowspan=1 colspan=1>2 (0.3)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>1 (0.8)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>1 (1.9)</td></tr><tr><td rowspan=1 colspan=1>Corynebacterium</td><td rowspan=1 colspan=1>14 (2.0)</td><td rowspan=1 colspan=1>1 (3.7)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>4 (3.3)</td><td rowspan=1 colspan=1>5 (2.0)</td><td rowspan=1 colspan=1>4 (1.8)</td><td rowspan=1 colspan=1>0 (0.0)</td></tr><tr><td rowspan=1 colspan=1>Cutibacterium acnes (P. acnes)</td><td rowspan=1 colspan=1>8 (1.1)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>3 (2.5)</td><td rowspan=1 colspan=1>2 (0.8)</td><td rowspan=1 colspan=1>3 (1.4)</td><td rowspan=1 colspan=1>0 (0.0)</td></tr><tr><td rowspan=1 colspan=1>Enterococcus</td><td rowspan=1 colspan=1>62 (8.7)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>6 (14.3)</td><td rowspan=1 colspan=1>8 (6.6)</td><td rowspan=1 colspan=1>20 (8.0)</td><td rowspan=1 colspan=1>24 (11.1)</td><td rowspan=1 colspan=1>4 (7.4)</td></tr><tr><td rowspan=1 colspan=1>Enterococcus faecalis</td><td rowspan=1 colspan=1>50 (7.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>6 (14.3)</td><td rowspan=1 colspan=1>6 (5.0)</td><td rowspan=1 colspan=1>15 (6.0)</td><td rowspan=1 colspan=1>20 (9.2)</td><td rowspan=1 colspan=1>3 (5.6)</td></tr><tr><td rowspan=1 colspan=1>Enterococcus faecium</td><td rowspan=1 colspan=1>12 (1.7)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>1 (0.8)</td><td rowspan=1 colspan=1>6 (2.4)</td><td rowspan=1 colspan=1>5 (2.3)</td><td rowspan=1 colspan=1>0 (0.0)</td></tr><tr><td rowspan=1 colspan=1>Lactobacillus</td><td rowspan=1 colspan=1>5 (0.7)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>2 (1.7)</td><td rowspan=1 colspan=1>1 (0.4)</td><td rowspan=1 colspan=1>1 (0.5)</td><td rowspan=1 colspan=1>1 (1.9)</td></tr><tr><td rowspan=1 colspan=1>Listeria</td><td rowspan=1 colspan=1>1 (0.1)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>1 (0.5)</td><td rowspan=1 colspan=1>0 (0.0)</td></tr><tr><td rowspan=1 colspan=1>Listeria monocytogenes</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td></tr><tr><td rowspan=1 colspan=1>Micrococcus</td><td rowspan=1 colspan=1>19 (2.7)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>3 (7.1)</td><td rowspan=1 colspan=1>2 (1.7)</td><td rowspan=1 colspan=1>8 (3.2)</td><td rowspan=1 colspan=1>5 (2.3)</td><td rowspan=1 colspan=1>1 (1.9)</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus</td><td rowspan=1 colspan=1>452 (63.6)</td><td rowspan=1 colspan=1>23 (85.2)</td><td rowspan=1 colspan=1>23 (54.8)</td><td rowspan=1 colspan=1>78 (64.5)</td><td rowspan=1 colspan=1>154 (61.6)</td><td rowspan=1 colspan=1>139 (64.1)</td><td rowspan=1 colspan=1>35 (64.8)</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus aureus</td><td rowspan=1 colspan=1>162 (22.8)</td><td rowspan=1 colspan=1>8 (29.6)</td><td rowspan=1 colspan=1>4 (9.5)</td><td rowspan=1 colspan=1>37 (30.6)</td><td rowspan=1 colspan=1>69 (27.6)</td><td rowspan=1 colspan=1>38 (17.5)</td><td rowspan=1 colspan=1>6 (11.1)</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus epidermidis</td><td rowspan=1 colspan=1>182 (25.6)</td><td rowspan=1 colspan=1>10 (37.0)</td><td rowspan=1 colspan=1>11 (26.2)</td><td rowspan=1 colspan=1>26 (21.5)</td><td rowspan=1 colspan=1>54 (21.6)</td><td rowspan=1 colspan=1>62 (28.6)</td><td rowspan=1 colspan=1>19 (35.2)</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus lugdunensis</td><td rowspan=1 colspan=1>5 (0.7)</td><td rowspan=1 colspan=1>1 (3.7)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>3 (1.2)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>1 (1.9)</td></tr><tr><td rowspan=1 colspan=1>Streptococcus</td><td rowspan=1 colspan=1>110 (15.5)</td><td rowspan=1 colspan=1>5 (18.5)</td><td rowspan=1 colspan=1>9 (21.4)</td><td rowspan=1 colspan=1>16 (13.2)</td><td rowspan=1 colspan=1>40 (16.0)</td><td rowspan=1 colspan=1>31 (14.3)</td><td rowspan=1 colspan=1>9 (16.7)</td></tr><tr><td rowspan=1 colspan=1>Streptococcus agalactiae</td><td rowspan=1 colspan=1>12 (1.7)</td><td rowspan=1 colspan=1>1 (3.7)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>1 (0.8)</td><td rowspan=1 colspan=1>5 (2.0)</td><td rowspan=1 colspan=1>5 (2.3)</td><td rowspan=1 colspan=1>0 (0.0)</td></tr><tr><td rowspan=1 colspan=1>Streptococcus anginosus group</td><td rowspan=1 colspan=1>10 (1.4)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>2 (1.7)</td><td rowspan=1 colspan=1>3 (1.2)</td><td rowspan=1 colspan=1>3 (1.4)</td><td rowspan=1 colspan=1>2 (3.7)</td></tr><tr><td rowspan=1 colspan=1>Streptococcus pneumoniae</td><td rowspan=1 colspan=1>28 (3.9)</td><td rowspan=1 colspan=1>2 (7.4)</td><td rowspan=1 colspan=1>2 (4.8)</td><td rowspan=1 colspan=1>3 (2.5)</td><td rowspan=1 colspan=1>12 (4.8)</td><td rowspan=1 colspan=1>8 (3.7)</td><td rowspan=1 colspan=1>1 (1.9)</td></tr><tr><td rowspan=1 colspan=1>Streptococcus pyogenes</td><td rowspan=1 colspan=1>8 (1.1)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>1 (0.8)</td><td rowspan=1 colspan=1>5(2.0)</td><td rowspan=1 colspan=1>2 (0.9)</td><td rowspan=1 colspan=1>0 (0.0)</td></tr><tr><td rowspan=1 colspan=1>Pan Candida</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td></tr><tr><td rowspan=1 colspan=1>Pan Gram-Negative</td><td rowspan=1 colspan=1>25 (3.5)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>4 (9.5)</td><td rowspan=1 colspan=1>4 (3.3)</td><td rowspan=1 colspan=1>10 (4.0)</td><td rowspan=1 colspan=1>6 (2.8)</td><td rowspan=1 colspan=1>1 (1.9)</td></tr><tr><td rowspan=1 colspan=1>mecA (Staphylococcus)</td><td rowspan=1 colspan=1>261 (36.7)</td><td rowspan=1 colspan=1>14 (51.9)</td><td rowspan=1 colspan=1>10 (23.8)</td><td rowspan=1 colspan=1>41 (33.9)</td><td rowspan=1 colspan=1>83 (33.2)</td><td rowspan=1 colspan=1>94 (43.3)</td><td rowspan=1 colspan=1>19 (35.2)</td></tr><tr><td rowspan=1 colspan=1>mecA (S. aureus)</td><td rowspan=1 colspan=1>86 (12.1)</td><td rowspan=1 colspan=1>4 (14.8)</td><td rowspan=1 colspan=1>1 (2.4)</td><td rowspan=1 colspan=1>17 (14.0)</td><td rowspan=1 colspan=1>35 (14.0)</td><td rowspan=1 colspan=1>26 (12.0)</td><td rowspan=1 colspan=1>3 (5.6)</td></tr><tr><td rowspan=1 colspan=1>mecA (S. epidermidis)</td><td rowspan=1 colspan=1>137 (19.3)</td><td rowspan=1 colspan=1>8 (29.6)</td><td rowspan=1 colspan=1>9 (21.4)</td><td rowspan=1 colspan=1>19 (15.7)</td><td rowspan=1 colspan=1>38 (15.2)</td><td rowspan=1 colspan=1>48 (22.1)</td><td rowspan=1 colspan=1>15 (27.8)</td></tr><tr><td rowspan=1 colspan=1>mecA (S. lugdunensis)</td><td rowspan=1 colspan=1>1 (0.1)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>1 (0.4)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td></tr><tr><td rowspan=1 colspan=1>mecA (CoNS excluding S.epidermidis / S. lugdunensis)</td><td rowspan=1 colspan=1>40 (5.6)</td><td rowspan=1 colspan=1>2 (7.4)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>6 (5.0)</td><td rowspan=1 colspan=1>10 (4.0)</td><td rowspan=1 colspan=1>20 (9.2)</td><td rowspan=1 colspan=1>2 (3.7)</td></tr><tr><td rowspan=1 colspan=1>mecC (Staphylococcus)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td></tr><tr><td rowspan=1 colspan=1>vanA (Enterococcus)</td><td rowspan=1 colspan=1>9 (1.3)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>6 (2.4)</td><td rowspan=1 colspan=1>3 (1.4)</td><td rowspan=1 colspan=1>0 (0.0)</td></tr><tr><td rowspan=1 colspan=1>vanA (E. faecalis)</td><td rowspan=1 colspan=1>1 (0.1)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>1 (0.4)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td></tr><tr><td rowspan=1 colspan=1>vanA (E. faecium)</td><td rowspan=1 colspan=1>8 (1.1)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>5 (2.0)</td><td rowspan=1 colspan=1>3 (1.4)</td><td rowspan=1 colspan=1>0 (0.0)</td></tr><tr><td rowspan=1 colspan=1>vanB (Enterococcus)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td></tr><tr><td></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr></table>

Table 2: Expected Value by Collection Site (Prospective Samples)   

<table><tr><td rowspan=1 colspan=1>Target</td><td rowspan=1 colspan=1>All Sites(N=711)</td><td rowspan=1 colspan=1>Site 1(N=161)</td><td rowspan=1 colspan=1>Site 2(N=58)</td><td rowspan=1 colspan=1>Site 3(N=164)</td><td rowspan=1 colspan=1>Site 4(N=145)</td><td rowspan=1 colspan=1>Site 5(N=77)</td><td rowspan=1 colspan=1>Site 6(N=33)</td><td rowspan=1 colspan=1>Site 7(N=73</td></tr><tr><td rowspan=1 colspan=1>Bacillus cereus group</td><td rowspan=1 colspan=1>5 (0.7)</td><td rowspan=1 colspan=1>3 (1.9)</td><td rowspan=1 colspan=1>1 (1.7)</td><td rowspan=1 colspan=1>1 (0.6)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td></tr><tr><td rowspan=1 colspan=1>Bacillus subtilis group</td><td rowspan=1 colspan=1>2 (0.3)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>1 (1.7)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>1 (1.3)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td></tr><tr><td rowspan=1 colspan=1>Corynebacterium</td><td rowspan=1 colspan=1>14 (2.0)</td><td rowspan=1 colspan=1>2 (1.2)</td><td rowspan=1 colspan=1>2 (3.4)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>6 (4.1)</td><td rowspan=1 colspan=1>2 (2.6)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>2 (2.7)</td></tr><tr><td rowspan=1 colspan=1>Cutibacterium acnes (P. acnes)</td><td rowspan=1 colspan=1>8 (1.1)</td><td rowspan=1 colspan=1>1 (0.6)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>1 (0.6)</td><td rowspan=1 colspan=1>2(1.4)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>4(5.5)</td></tr><tr><td rowspan=1 colspan=1>Enterococcus</td><td rowspan=1 colspan=1>62 (8.7)</td><td rowspan=1 colspan=1>20 (12.4)</td><td rowspan=1 colspan=1>7 (12.1)</td><td rowspan=1 colspan=1>15 (9.1)</td><td rowspan=1 colspan=1>9 (6.2)</td><td rowspan=1 colspan=1>10 (13.0)</td><td rowspan=1 colspan=1>1 (3.0)</td><td rowspan=1 colspan=1>0 (0.0)</td></tr><tr><td rowspan=1 colspan=1>Enterococcus faecalis</td><td rowspan=1 colspan=1>50 (7.0)</td><td rowspan=1 colspan=1>14 (8.7)</td><td rowspan=1 colspan=1>6 (10.3)</td><td rowspan=1 colspan=1>13 (7.9)</td><td rowspan=1 colspan=1>8 (5.5)</td><td rowspan=1 colspan=1>8 (10.4)</td><td rowspan=1 colspan=1>1 (3.0)</td><td rowspan=1 colspan=1>0 (0.0)</td></tr><tr><td rowspan=1 colspan=1>Enterococcus faecium</td><td rowspan=1 colspan=1>12 (1.7)</td><td rowspan=1 colspan=1>6 (3.7)</td><td rowspan=1 colspan=1>1 (1.7)</td><td rowspan=1 colspan=1>1 (0.6)</td><td rowspan=1 colspan=1>2 (1.4)</td><td rowspan=1 colspan=1>2 (2.6)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td></tr><tr><td rowspan=1 colspan=1>Lactobacillus</td><td rowspan=1 colspan=1>5 (0.7)</td><td rowspan=1 colspan=1>1 (0.6)</td><td rowspan=1 colspan=1>1 (1.7)</td><td rowspan=1 colspan=1>2 (1.2)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>1 (1.3)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td></tr><tr><td rowspan=1 colspan=1>Listeria</td><td rowspan=1 colspan=1>1 (0.1)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>1 (0.7)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td></tr><tr><td rowspan=1 colspan=1>Listeria monocytogenes</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td></tr><tr><td rowspan=1 colspan=1>Micrococcus</td><td rowspan=1 colspan=1>19 (2.7)</td><td rowspan=1 colspan=1>2 (1.2)</td><td rowspan=1 colspan=1>2 (3.4)</td><td rowspan=1 colspan=1>5 (3.0)</td><td rowspan=1 colspan=1>6 (4.1)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>4(5.5)</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus</td><td rowspan=1 colspan=1>452 (63.6)</td><td rowspan=1 colspan=1>106 (65.8)</td><td rowspan=1 colspan=1>27 (46.6)</td><td rowspan=1 colspan=1>109 (66.5)</td><td rowspan=1 colspan=1>98 (67.6)</td><td rowspan=1 colspan=1>52 (67.5)</td><td rowspan=1 colspan=1>24 (72.7)</td><td rowspan=1 colspan=1>36 (49.3)</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus aureus</td><td rowspan=1 colspan=1>162 (22.8)</td><td rowspan=1 colspan=1>36 (22.4)</td><td rowspan=1 colspan=1>7(12.1)</td><td rowspan=1 colspan=1>56 (34.1)</td><td rowspan=1 colspan=1>27 (18.6)</td><td rowspan=1 colspan=1>18 (23.4)</td><td rowspan=1 colspan=1>6 (18.2)</td><td rowspan=1 colspan=1>12 (16.4)</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus epidermidis</td><td rowspan=1 colspan=1>182 (25.6)</td><td rowspan=1 colspan=1>41 (25.5)</td><td rowspan=1 colspan=1>14 (24.1)</td><td rowspan=1 colspan=1>34 (20.7)</td><td rowspan=1 colspan=1>44 (30.3)</td><td rowspan=1 colspan=1>23 (29.9)</td><td rowspan=1 colspan=1>13 (39.4)</td><td rowspan=1 colspan=1>13 (17.8)</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus lugdunensis</td><td rowspan=1 colspan=1>5 (0.7)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>2 (1.2)</td><td rowspan=1 colspan=1>2 (1.4)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>1 (1.4)</td></tr><tr><td rowspan=1 colspan=1>Streptococcus</td><td rowspan=1 colspan=1>110 (15.5)</td><td rowspan=1 colspan=1>18 (11.2)</td><td rowspan=1 colspan=1>9 (15.5)</td><td rowspan=1 colspan=1>28 (17.1)</td><td rowspan=1 colspan=1>20 (13.8)</td><td rowspan=1 colspan=1>11 (14.3)</td><td rowspan=1 colspan=1>5 (15.2)</td><td rowspan=1 colspan=1>19 (26.0)</td></tr><tr><td rowspan=1 colspan=1>Streptococcus agalactiae</td><td rowspan=1 colspan=1>12 (1.7)</td><td rowspan=1 colspan=1>2 (1.2)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>2 (1.2)</td><td rowspan=1 colspan=1>2 (1.4)</td><td rowspan=1 colspan=1>2 (2.6)</td><td rowspan=1 colspan=1>2 (6.1)</td><td rowspan=1 colspan=1>2 (2.7)</td></tr><tr><td rowspan=1 colspan=1>Streptococcus anginosus group</td><td rowspan=1 colspan=1>10 (1.4)</td><td rowspan=1 colspan=1>2 (1.2)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>6 (3.7)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>2 (2.6)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td></tr><tr><td rowspan=1 colspan=1>Streptococcus pneumoniae</td><td rowspan=1 colspan=1>28 (3.9)</td><td rowspan=1 colspan=1>3 (1.9)</td><td rowspan=1 colspan=1>5(8.6)</td><td rowspan=1 colspan=1>5 (3.0)</td><td rowspan=1 colspan=1>4 (2.8)</td><td rowspan=1 colspan=1>1 (1.3)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>10 (13.7)</td></tr><tr><td rowspan=1 colspan=1>Streptococcus pyogenes</td><td rowspan=1 colspan=1>8 (1.1)</td><td rowspan=1 colspan=1>1 (0.6)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>3 (1.8)</td><td rowspan=1 colspan=1>1 (0.7)</td><td rowspan=1 colspan=1>1 (1.3)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>2 (2.7)</td></tr><tr><td rowspan=1 colspan=1>Pan Candida</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td></tr><tr><td rowspan=1 colspan=1>Pan Gram-Negative</td><td rowspan=1 colspan=1>25 (3.5)</td><td rowspan=1 colspan=1>9 (5.6)</td><td rowspan=1 colspan=1>4 (6.9)</td><td rowspan=1 colspan=1>3 (1.8)</td><td rowspan=1 colspan=1>2 (1.4)</td><td rowspan=1 colspan=1>5 (6.5)</td><td rowspan=1 colspan=1>1 (3.0)</td><td rowspan=1 colspan=1>1 (1.4)</td></tr><tr><td rowspan=1 colspan=1>mecA (Staphylococcus)</td><td rowspan=1 colspan=1>261 (36.7)</td><td rowspan=1 colspan=1>69 (42.9)</td><td rowspan=1 colspan=1>17 (29.3)</td><td rowspan=1 colspan=1>68 (41.5)</td><td rowspan=1 colspan=1>55 (37.9)</td><td rowspan=1 colspan=1>25 (32.5)</td><td rowspan=1 colspan=1>13 (39.4)</td><td rowspan=1 colspan=1>14 (19.2)</td></tr><tr><td rowspan=1 colspan=1>mecA (S. aureus)</td><td rowspan=1 colspan=1>86 (12.1)</td><td rowspan=1 colspan=1>19 (11.8)</td><td rowspan=1 colspan=1>5(8.6)</td><td rowspan=1 colspan=1>28 (17.1)</td><td rowspan=1 colspan=1>19 (13.1)</td><td rowspan=1 colspan=1>7(9.1)</td><td rowspan=1 colspan=1>3 (9.1)</td><td rowspan=1 colspan=1>5(6.8)</td></tr><tr><td rowspan=1 colspan=1>mecA (S. epidermidis)</td><td rowspan=1 colspan=1>137 (19.3)</td><td rowspan=1 colspan=1>36 (22.4)</td><td rowspan=1 colspan=1>10 (17.2)</td><td rowspan=1 colspan=1>31 (18.9)</td><td rowspan=1 colspan=1>29 (20.0)</td><td rowspan=1 colspan=1>17 (22.1)</td><td rowspan=1 colspan=1>8 (24.2)</td><td rowspan=1 colspan=1>6 (8.2)</td></tr><tr><td rowspan=1 colspan=1>mecA (S. lugdunensis)</td><td rowspan=1 colspan=1>1 (0.1)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>1 (0.6)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td></tr><tr><td rowspan=1 colspan=1>mecA (CoNS excluding S.epidermidis / S. lugdunensis)</td><td rowspan=1 colspan=1>40 (5.6)</td><td rowspan=1 colspan=1>15 (9.3)</td><td rowspan=1 colspan=1>2 (3.4)</td><td rowspan=1 colspan=1>9 (5.5)</td><td rowspan=1 colspan=1>7 (4.8)</td><td rowspan=1 colspan=1>2 (2.6)</td><td rowspan=1 colspan=1>2 (6.1)</td><td rowspan=1 colspan=1>3 (4.1)</td></tr><tr><td rowspan=1 colspan=1>mecC (Staphylococcus)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td></tr><tr><td rowspan=1 colspan=1>vanA (Enterococcus)</td><td rowspan=1 colspan=1>9 (1.3)</td><td rowspan=1 colspan=1>6 (3.7)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>1 (0.6)</td><td rowspan=1 colspan=1>1 (0.7)</td><td rowspan=1 colspan=1>1 (1.3)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td></tr><tr><td rowspan=1 colspan=1>vanA (E. faecalis)</td><td rowspan=1 colspan=1>1 (0.1)</td><td rowspan=1 colspan=1>1 (0.6)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td></tr><tr><td rowspan=1 colspan=1>vanA (E. faecium)</td><td rowspan=1 colspan=1>8 (1.1)</td><td rowspan=1 colspan=1>5 (3.1)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>1 (0.6)</td><td rowspan=1 colspan=1>1 (0.7)</td><td rowspan=1 colspan=1>1 (1.3)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td></tr><tr><td rowspan=1 colspan=1>vanB (Enterococcus)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td></tr></table>

# PERFORMANCE CHARACTERISTICS

# Clinical Performance

Samples with final, valid ePlex BCID-GP Panel test results and a valid comparator result were evaluable and included in summaries and analyses of demographics, expected values (positivity rate), and performance characteristics. Evaluable samples included 312 prospective fresh and 399 prospective frozen samples as well as 586 retrospective samples and 565 contrived samples.

# Comparator Method

The performance of the ePlex BCID-GP Panel was compared to standard laboratory procedures, including traditional and automated culture, MALDI-TOF IVD, and microbiological and biochemical techniques for organism identification. Identification for samples with Corynebacterium, Staphylococcus epidermidis, Staphylococcus hominis, or Candida parapsilosis identified by standard laboratory procedures were confirmed using analytically validated PCR assays followed by bi-directional sequencing or 16S sequencing. For antibiotic resistance genes, the ePlex BCID-GP Panel was compared to analytically validated qPCR amplification assays followed by bi-directional sequencing in samples with an associated organism identified (i.e., Staphylococcus, Enterococcus).

The comparator method(s) results were used to determine the Detected / Not Detected status for each target organism on the ePlex BCID-GP Panel. The comparator methods for each target are summarized in Table 3.

Table 3: Comparator Method(s) by ePlex BCID-GP Panel Target   

<table><tr><td rowspan=1 colspan=1>Target</td><td rowspan=1 colspan=2>Comparator Method</td></tr><tr><td rowspan=1 colspan=1>Bacillus cereus group</td><td rowspan=22 colspan=2>Standard laboratory procedures for organism identification.</td></tr><tr><td rowspan=1 colspan=1>Bacillus subtilis group</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Cutibacterium acnes (Propionibacterium acnes)</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Enterococcus</td></tr><tr><td rowspan=1 colspan=1>Enterococcus faecalis</td></tr><tr><td rowspan=1 colspan=1>Enterococcus faecium</td></tr><tr><td rowspan=1 colspan=1>Lactobacillus</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=2 colspan=1>Listeria</td><td rowspan=2 colspan=1>b</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Listeria monocytogenes</td></tr><tr><td rowspan=1 colspan=1>Micrococcus</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus</td></tr><tr><td rowspan=2 colspan=1>Staphylococcus aureus</td><td rowspan=2 colspan=1>b</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1></td></tr><tr><td rowspan=2 colspan=1>Staphylococcus lugdunensis</td><td rowspan=2 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Streptococcus</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Streptococcus agalactiae</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Streptococcus anginosus group</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Streptococcus pneumoniae</td></tr><tr><td rowspan=1 colspan=1>Streptococcus pyogenes</td></tr><tr><td rowspan=1 colspan=1>Pan Gram-Negative</td></tr><tr><td rowspan=1 colspan=1>Corynebacterium</td><td rowspan=1 colspan=2>Standard laboratory procedures for organism ID.PCR/sequencing and 16S sequencing to confirm (oridentify Coryneform) or exclude Corynebacterium speciesnot included in this panel target*</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus epidermidis</td><td rowspan=1 colspan=2>Standard laboratory procedures for organism ID.PCR/sequencing to confirm S. epidermidis, S. hominis</td></tr><tr><td rowspan=1 colspan=1>Pan Candida</td><td rowspan=1 colspan=2>Standard laboratory procedures for organism ID.PCR/sequencing to confirm C. parapsilosis or identifyC. metapsilosis, C. orthopsilosis</td></tr><tr><td rowspan=1 colspan=1>mecA</td><td rowspan=2 colspan=2>qPCR/sequencing in samples with Staphylococcusidentified</td></tr><tr><td rowspan=1 colspan=1>mecC</td></tr><tr><td rowspan=1 colspan=1>vanA</td><td rowspan=2 colspan=2>qPCR/sequencing in samples with Enterococcus identified</td></tr><tr><td rowspan=1 colspan=1>vanB</td></tr></table>

\* The Corynebacterium assay is not designed to detect the following Corynebacterium species: C. amycolatum, C. argentoratense, C. bovis, C. durum, C. glucuronolyticum, C. macginleyi, C. propinquum, C. riegelii, and C. sundsvallense

# Demographics of Clinical Samples

Clinical performance was evaluated in positive blood culture samples prospectively and retrospectively collected. Prospective samples were collected at 7 clinical sites in 2 phases. From June 2014 through July 2016, 400 samples were prospectively collected and frozen; from January through February 2018, 319 samples were prospectively collected and tested fresh (never frozen) for a total of 719 samples across the 2 phases. 8 of these samples were withdrawn;

5 due to the sample coming from a patient already enrolled; 1 was collected outside of the required timeframe; 1 was not viable upon subculture and 1 was from an autopsy. Samples with final, valid ePlex BCID-GP Panel results and a valid comparator result were considered evaluable. Of the 711 prospectively-collected samples eligible for testing, all 711 were evaluable. Demographic information for prospectively-collected samples is described in Table 4. Subjects enrolled in this study were from a diverse demographic distribution and represent the intended patient population.

To supplement the number of positives for low prevalence targets in the prospective collection, 586 samples were collected retrospectively, and all 586 were evaluable. Demographic information for retrospectively-collected samples is described in Table 5.

Table 4: Demographic Data for Clinical Samples by Collection Site (Prospective Collection)   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>All SitesN = 711</td><td rowspan=1 colspan=1>Site 1N = 161</td><td rowspan=1 colspan=1>Site 2N = 58</td><td rowspan=1 colspan=1>Site 3N = 164</td><td rowspan=1 colspan=1>Site 4N = 145</td><td rowspan=1 colspan=1>Site 5N = 77</td><td rowspan=1 colspan=1>Site 6N = 33</td><td rowspan=1 colspan=1>Site 7N = 73</td></tr><tr><td rowspan=1 colspan=9>Sex</td></tr><tr><td rowspan=1 colspan=1>Male</td><td rowspan=1 colspan=1>377 (53.0)</td><td rowspan=1 colspan=1>93 (57.8)</td><td rowspan=1 colspan=1>28 (48.3)</td><td rowspan=1 colspan=1>91 (55.5)</td><td rowspan=1 colspan=1>66 (45.5)</td><td rowspan=1 colspan=1>42 (54.5)</td><td rowspan=1 colspan=1>17 (51.5)</td><td rowspan=1 colspan=1>40 (54.8)</td></tr><tr><td rowspan=1 colspan=1>Female</td><td rowspan=1 colspan=1>334 (47.0)</td><td rowspan=1 colspan=1>68 (42.2)</td><td rowspan=1 colspan=1>30 (51.7)</td><td rowspan=1 colspan=1>73 (44.5)</td><td rowspan=1 colspan=1>79 (54.5)</td><td rowspan=1 colspan=1>35 (45.5)</td><td rowspan=1 colspan=1>16 (48.5)</td><td rowspan=1 colspan=1>33 (45.2)</td></tr><tr><td rowspan=1 colspan=9>Age</td></tr><tr><td rowspan=1 colspan=1>&lt;1 yr</td><td rowspan=1 colspan=1>27 (3.8)</td><td rowspan=1 colspan=1>3 (1.9)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>8 (4.9)</td><td rowspan=1 colspan=1>10 (6.9)</td><td rowspan=1 colspan=1>4 (5.2)</td><td rowspan=1 colspan=1>2 (6.1)</td><td rowspan=1 colspan=1>0 (0.0)</td></tr><tr><td rowspan=1 colspan=1>1-17 yrs</td><td rowspan=1 colspan=1>42 (5.9)</td><td rowspan=1 colspan=1>8 (5.0)</td><td rowspan=1 colspan=1>2 (3.4)</td><td rowspan=1 colspan=1>11 (6.7)</td><td rowspan=1 colspan=1>10 (6.9)</td><td rowspan=1 colspan=1>7 (9.1)</td><td rowspan=1 colspan=1>2(6.1)</td><td rowspan=1 colspan=1>2 (2.7)</td></tr><tr><td rowspan=1 colspan=1>18-44 yrs</td><td rowspan=1 colspan=1>121 (17)</td><td rowspan=1 colspan=1>32 (19.9)</td><td rowspan=1 colspan=1>9 (15.5)</td><td rowspan=1 colspan=1>24 (14.6)</td><td rowspan=1 colspan=1>24 (16.6)</td><td rowspan=1 colspan=1>13 (16.9)</td><td rowspan=1 colspan=1>4 (12.1)</td><td rowspan=1 colspan=1>15 (20.5)</td></tr><tr><td rowspan=1 colspan=1>45-64 yrs</td><td rowspan=1 colspan=1>250 (35.2)</td><td rowspan=1 colspan=1>66 (41.0)</td><td rowspan=1 colspan=1>18 (31.0)</td><td rowspan=1 colspan=1>67 (40.9)</td><td rowspan=1 colspan=1>36 (24.8)</td><td rowspan=1 colspan=1>25 (32.5)</td><td rowspan=1 colspan=1>11 (33.3)</td><td rowspan=1 colspan=1>27 (37.0)</td></tr><tr><td rowspan=1 colspan=1>65-84 yrs</td><td rowspan=1 colspan=1>217 (30.5)</td><td rowspan=1 colspan=1>44 (27.3)</td><td rowspan=1 colspan=1>20 (34.5)</td><td rowspan=1 colspan=1>41 (25.0)</td><td rowspan=1 colspan=1>51 (35.2)</td><td rowspan=1 colspan=1>23 (29.9)</td><td rowspan=1 colspan=1>13 (39.4)</td><td rowspan=1 colspan=1>25 (34.2)</td></tr><tr><td rowspan=1 colspan=1>85+yrs</td><td rowspan=1 colspan=1>54 (7.6)</td><td rowspan=1 colspan=1>8(5.0)</td><td rowspan=1 colspan=1>9 (15.5)</td><td rowspan=1 colspan=1>13 (7.9)</td><td rowspan=1 colspan=1>14 (9.7)</td><td rowspan=1 colspan=1>5(6.5)</td><td rowspan=1 colspan=1>1 (3.0)</td><td rowspan=1 colspan=1>4(5.5)</td></tr></table>

Table 5: Demographic Data for Clinical Samples by Collection Site (Retrospective Collection)   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>All SitesN = 586</td><td rowspan=1 colspan=1>Site 1N = 80</td><td rowspan=1 colspan=1>Site 2N = 98</td><td rowspan=1 colspan=1>Site 3N = 51</td><td rowspan=1 colspan=1>Site 4N = 43</td><td rowspan=1 colspan=1>Site 5N = 3</td><td rowspan=1 colspan=1>Site 6N = 61</td><td rowspan=1 colspan=1>Site 7N = 85</td><td rowspan=1 colspan=1>Site 8N = 25</td><td rowspan=1 colspan=1>Site 9N = 46</td><td rowspan=1 colspan=1>Site 10N = 94</td></tr><tr><td rowspan=1 colspan=12>Sex</td></tr><tr><td rowspan=1 colspan=1>Male</td><td rowspan=1 colspan=1>317 (54.1)</td><td rowspan=1 colspan=1>39 (48.8)</td><td rowspan=1 colspan=1>59 (60.2)</td><td rowspan=1 colspan=1>24 (47.1)</td><td rowspan=1 colspan=1>20 (46.5)</td><td rowspan=1 colspan=1>1 (33.3)</td><td rowspan=1 colspan=1>36 (59.0)</td><td rowspan=1 colspan=1>45 (52.9)</td><td rowspan=1 colspan=1>17 (68.0)</td><td rowspan=1 colspan=1>28 (60.9)</td><td rowspan=1 colspan=1>48 (51.1)</td></tr><tr><td rowspan=1 colspan=1>Female</td><td rowspan=1 colspan=1>269 (45.9)</td><td rowspan=1 colspan=1>41 (51.3)</td><td rowspan=1 colspan=1>39 (39.8)</td><td rowspan=1 colspan=1>27 (52.9)</td><td rowspan=1 colspan=1>23 (53.5)</td><td rowspan=1 colspan=1>2 (66.7)</td><td rowspan=1 colspan=1>25 (41.0)</td><td rowspan=1 colspan=1>40 (47.1)</td><td rowspan=1 colspan=1>8 (32.0)</td><td rowspan=1 colspan=1>18 (39.1)</td><td rowspan=1 colspan=1>46 (48.9)</td></tr><tr><td rowspan=1 colspan=12>Age</td></tr><tr><td rowspan=1 colspan=1>&lt;1 yr</td><td rowspan=1 colspan=1>11 (1.9)</td><td rowspan=1 colspan=1>1 (1.3)</td><td rowspan=1 colspan=1>2 (2)</td><td rowspan=1 colspan=1>0 (0)</td><td rowspan=1 colspan=1>3 (7)</td><td rowspan=1 colspan=1>0 (0)</td><td rowspan=1 colspan=1>1 (1.6)</td><td rowspan=1 colspan=1>0 (0)</td><td rowspan=1 colspan=1>0 (0)</td><td rowspan=1 colspan=1>1 (2.2)</td><td rowspan=1 colspan=1>3 (3.2)</td></tr><tr><td rowspan=1 colspan=1>1-17 yrs</td><td rowspan=1 colspan=1>17 (2.9)</td><td rowspan=1 colspan=1>6 (7.5)</td><td rowspan=1 colspan=1>1 (1)</td><td rowspan=1 colspan=1>0 (0)</td><td rowspan=1 colspan=1>4 (9.3)</td><td rowspan=1 colspan=1>0 (0)</td><td rowspan=1 colspan=1>0 (0)</td><td rowspan=1 colspan=1>0 (0)</td><td rowspan=1 colspan=1>1 (4)</td><td rowspan=1 colspan=1>1 (2.2)</td><td rowspan=1 colspan=1>4 (4.3)</td></tr><tr><td rowspan=1 colspan=1>18-44 yrs</td><td rowspan=1 colspan=1>104 (17.7)</td><td rowspan=1 colspan=1>14 (17.5)</td><td rowspan=1 colspan=1>13 (13.3)</td><td rowspan=1 colspan=1>5 (9.8)</td><td rowspan=1 colspan=1>9 (20.9)</td><td rowspan=1 colspan=1>0 (0)</td><td rowspan=1 colspan=1>15 (24.6)</td><td rowspan=1 colspan=1>11 (12.9)</td><td rowspan=1 colspan=1>7 (28)</td><td rowspan=1 colspan=1>5 (10.9)</td><td rowspan=1 colspan=1>25 (26.6)</td></tr><tr><td rowspan=1 colspan=1>45-64 yrs</td><td rowspan=1 colspan=1>193 (32.9)</td><td rowspan=1 colspan=1>25 (31.3)</td><td rowspan=1 colspan=1>33 (33.7)</td><td rowspan=1 colspan=1>17 (33.3)</td><td rowspan=1 colspan=1>15 (34.9)</td><td rowspan=1 colspan=1>1 (33.3)</td><td rowspan=1 colspan=1>21 (34.4)</td><td rowspan=1 colspan=1>30 (35.3)</td><td rowspan=1 colspan=1>10 (40)</td><td rowspan=1 colspan=1>12 (26.1)</td><td rowspan=1 colspan=1>29 (30.9)</td></tr><tr><td rowspan=1 colspan=1>65-84 yrs</td><td rowspan=1 colspan=1>209 (35.7)</td><td rowspan=1 colspan=1>26 (32.5)</td><td rowspan=1 colspan=1>42 (42.9)</td><td rowspan=1 colspan=1>22 (43.1)</td><td rowspan=1 colspan=1>9 (20.9)</td><td rowspan=1 colspan=1>0 (0)</td><td rowspan=1 colspan=1>20 (32.8)</td><td rowspan=1 colspan=1>35 (41.2)</td><td rowspan=1 colspan=1>7 (28)</td><td rowspan=1 colspan=1>18 (39.1)</td><td rowspan=1 colspan=1>30 (31.9)</td></tr><tr><td rowspan=1 colspan=1>85+ yrs</td><td rowspan=1 colspan=1>50 (8.5)</td><td rowspan=1 colspan=1>8 (10)</td><td rowspan=1 colspan=1>7 (7.1)</td><td rowspan=1 colspan=1>7 (13.7)</td><td rowspan=1 colspan=1>3 (7)</td><td rowspan=1 colspan=1>2 (66.7)</td><td rowspan=1 colspan=1>4 (6.6)</td><td rowspan=1 colspan=1>7 (8.2)</td><td rowspan=1 colspan=1>0 (0)</td><td rowspan=1 colspan=1>9 (19.6)</td><td rowspan=1 colspan=1>3 (3.2)</td></tr><tr><td rowspan=1 colspan=1>Unknown</td><td rowspan=1 colspan=1>2 (0.3)</td><td rowspan=1 colspan=1>0 (0)</td><td rowspan=1 colspan=1>0 (0)</td><td rowspan=1 colspan=1>0 (0)</td><td rowspan=1 colspan=1>0 (0)</td><td rowspan=1 colspan=1>0 (0)</td><td rowspan=1 colspan=1>0 (0)</td><td rowspan=1 colspan=1>2 (2.4)</td><td rowspan=1 colspan=1>0 (0)</td><td rowspan=1 colspan=1>0 (0)</td><td rowspan=1 colspan=1>0 (0)</td></tr></table>

# Clinical Performance

Sensitivity or positive percent agreement (PPA) was calculated by dividing the number of true positive (TP) results by the sum of TP and false negative (FN) results, while specificity or negative percent agreement (NPA) was calculated by dividing the number of true negative (TN) results by the sum of TN and false positive (FP) results. A TP result being defined as a sample where the detected ePlex BCID-GP Panel result matched the detected comparator method result, while a TN result was one where a negative ePlex BCID-GP Panel result matched a negative comparator method result. The two-sided $9 5 \%$ confidence interval was also calculated.

A total of 711 prospectively-collected samples (312 tested fresh and 399 tested after previously frozen) and 586 retrospectively collected samples as well as 565 contrived samples were evaluated for the ePlex BCID-GP Panel targets. Contrived samples were prepared by spiking an isolate into a blood culture bottle and growing until flagged positive by a continuously monitoring blood culture system. Samples were removed from the system within 8 hours of positivity and stored frozen until the time of testing. PPA and NPA results are summarized by target in Tables 6-30 and the strains used to contrive samples are summarized in Table 31.

Table 6: Clinical Performance for Bacillus cereus group   

<table><tr><td rowspan="2">Target</td><td rowspan="2">Sample Type</td><td colspan="2">Sensitivity/PPA</td><td colspan="2">Specificity/NPA</td></tr><tr><td>TP/TP+FN</td><td>% (95% CI)</td><td>TN/TN+FP</td><td>% (95% CI)</td></tr><tr><td rowspan="7">Bacillus cereus group</td><td>Prospective (Fresh)</td><td>2/2</td><td>100 (34.2-100)</td><td>310/310</td><td>100 (98.8-100)</td></tr><tr><td>Prospective (Frozen)</td><td>3/3</td><td>100 (43.9-100)</td><td>396/396</td><td>100 (99.0-100)</td></tr><tr><td>Prospective (All)</td><td>515</td><td>100 (56.6-100)</td><td>706/706</td><td>100 (99.5-100)</td></tr><tr><td>Retrospective</td><td>6/7</td><td>85.7 (48.7-97.4)</td><td>579/579</td><td>100 (99.3-100)</td></tr><tr><td>Prospective/Retrospective</td><td>11/12</td><td>91.7 (64.6-98.5)</td><td>1765/1765</td><td>100 (99.8-100)</td></tr><tr><td>Contrived</td><td>46/46</td><td>100 (92.3-100)</td><td>519/519</td><td>100 (99.3-100)</td></tr><tr><td>Overall</td><td>57/58</td><td>98.3 (90.9-99.7)</td><td>2284/2284</td><td>100 (99.8-100)</td></tr></table>

CI= Confidence Interval

Table 7: Clinical Performance for Bacillus subtilis group   

<table><tr><td rowspan="2">Target</td><td rowspan="2">Sample Type</td><td colspan="2">Sensitivity/PPA</td><td colspan="2">Specificity/NPA</td></tr><tr><td>TP/TP+FN</td><td>% (95% CI)</td><td>TN/TN+FP</td><td>% (95% CI)</td></tr><tr><td rowspan="7">Bacillus subtilis group</td><td>Prospective (Fresh)</td><td>2/2</td><td>100 (34.2-100)</td><td>309/309</td><td>100 (98.8-100)</td></tr><tr><td>Prospective (Frozen)</td><td>0/0</td><td>---</td><td>399/399</td><td>100 (99.0-100)</td></tr><tr><td>Prospective (All)</td><td>2/2</td><td>100 (34.2-100)</td><td>708/708</td><td>100 (99.5-100)</td></tr><tr><td>Retrospective</td><td>0/0</td><td>---</td><td>586/586</td><td>100 (99.3-100)</td></tr><tr><td>Prospective/Retrospective</td><td>2/2</td><td>100 (34.2-100)</td><td>1294/1294</td><td>100 (99.7-100)</td></tr><tr><td>Contrived</td><td>50/50</td><td>100 (92.9-100)</td><td>515/515</td><td>100 (99.3-100)</td></tr><tr><td>Overall</td><td>52/52</td><td>100 (93.1-100)</td><td>1809/1809</td><td>100 (99.8-100)</td></tr></table>

Table 8: Clinical Performance for Corynebacterium   

<table><tr><td rowspan="2">Target</td><td rowspan="2">Sample Type</td><td colspan="2">Sensitivity/PPA</td><td colspan="2">Specificity/NPA</td></tr><tr><td>TP/TP+FN</td><td>% (95% CI)</td><td>TN/TN+FP</td><td>% (95% CI)</td></tr><tr><td rowspan="7">Corynebacterium</td><td>Prospective (Fresh)</td><td>5/7</td><td>71.4 (35.9-91.8)</td><td>304/305</td><td>99.7 (98.2-99.9)</td></tr><tr><td>Prospective (Frozen)</td><td>8/12</td><td>66.7 (39.1-86.2)</td><td>387/387</td><td>100 (99.0-100)</td></tr><tr><td>Prospective (All)</td><td>13/19</td><td>68.4 (46.0-84.6)</td><td>691/692</td><td>99.9 (99.2-100)</td></tr><tr><td>Retrospective</td><td>27/32</td><td>84.4 (68.2-93.1)</td><td>553/554</td><td>99.8 (99.0-100)</td></tr><tr><td>Prospective/Retrospective</td><td>40/51A</td><td>78.4 (65.4-87.5)</td><td>1244/1246B</td><td>99.8 (99.4-100)</td></tr><tr><td>Contrived</td><td>20/20</td><td>100 (83.9-100)</td><td>545/545</td><td>100 (99.3-100)</td></tr><tr><td>Overall</td><td>60/71</td><td>84.5 (74.3-91.1)</td><td>1789/1791</td><td>99.9 (99.6-100)</td></tr></table>

A. Corynebacterium was not detected in 4 of the false negative samples using PCR/sequencing, but 16S sequencing instead detected Staphylococcus pettenkoferi, Macrococcus caseolyticus, Lactobacillus fermentum, and Cutibacterium acnes, which were not identified by standard laboratory procedures. B. Corynebacterium was detected in 2/2 false positive samples using PCR/sequencing.

Table 9: Clinical Performance for Cutibacterium acnes   

<table><tr><td rowspan="2">Target</td><td rowspan="2">Sample Type</td><td colspan="2">Sensitivity/PPA</td><td colspan="2">Specificity/NPA</td></tr><tr><td>TP/TP+FN</td><td>% (95% CI)</td><td>TN/TN+FP</td><td>% (95% CI)</td></tr><tr><td rowspan="7">Cutibacterium acnes</td><td>Prospective (Fresh)</td><td>4/5</td><td>80.0 (37.6-96.4)</td><td>306/307</td><td>99.7 (98.2-99.9)</td></tr><tr><td>Prospective (Frozen)</td><td>2/2</td><td>100 (34.2-100)</td><td>396/397</td><td>99.7 (98.6-100)</td></tr><tr><td>Prospective (All)</td><td>6/7</td><td>85.7 (48.7-97.4)</td><td>702/704</td><td>99.7 (99.0-99.9)</td></tr><tr><td>Retrospective</td><td>12/13</td><td>92.3 (66.7-98.6)</td><td>573/573</td><td>100 (99.3-100)</td></tr><tr><td>Prospective/Retrospective</td><td>18/20</td><td>90.0 (69.9-97.2)</td><td>1275/1277A</td><td>99.8 (99.4-100)</td></tr><tr><td>Contrived</td><td>25/26</td><td>96.2 (81.1-99.3)</td><td>539/539</td><td>100 (99.3-100)</td></tr><tr><td>Overall</td><td>43/46</td><td>93.5 (82.5-97.8)</td><td>1814/1816</td><td>99.9 (99.6-100)</td></tr></table>

A. Cutibacterium acnes was detected in 1/2 false positive samples using PCR/sequencing.

Table 10: Clinical Performance for Enterococcus   

<table><tr><td rowspan="2">Target</td><td rowspan="2">Sample Type</td><td colspan="2">Sensitivity/PPA</td><td colspan="2">Specificity/NPA</td></tr><tr><td>TP/TP+FN</td><td>% (95% CI)</td><td>TN/TN+FP</td><td>% (95% CI)</td></tr><tr><td rowspan="7">Enterococcus</td><td>Prospective (Fresh)</td><td>25/25</td><td>100 (86.7-100)</td><td>287/287</td><td>100 (98.7-100)</td></tr><tr><td>Prospective (Frozen)</td><td>36/36</td><td>100 (90.4-100)</td><td>362/363</td><td>99.7 (98.5-100)</td></tr><tr><td>Prospective (All)</td><td>61/61</td><td>100 (94.1-100)</td><td>649/650</td><td>99.8 (99.1-100)</td></tr><tr><td>Retrospective</td><td>139/147</td><td>94.6 (89.6-97.2)</td><td>439/439</td><td>100 (99.1-100)</td></tr><tr><td>Prospective/Retrospective</td><td>200/2084</td><td>96.2 (92.6-98.0)</td><td>1088/1089B</td><td>99.9 (99.5-100)</td></tr><tr><td>Contrived</td><td>126/126</td><td>100 (97.0-100)</td><td>439/439</td><td>100 (99.1-100)</td></tr><tr><td>Overall</td><td>326/334</td><td>97.6 (95.3-98.8)</td><td>1527/1528</td><td>99.9 (99.6-100)</td></tr></table>

A. Enterococcus was not detected in 1 false negative sample, but PCR/sequencing instead detected Lactococcus lactis, which was not identified by standard laboratory procedures. B. Enterococcus was detected in 1/1 false positive samples using PCR/sequencing.

Table 11: Clinical Performance for Enterococcus faecalis   

<table><tr><td rowspan="2">Target</td><td rowspan="2">Sample Type</td><td colspan="2">Sensitivity/PPA</td><td colspan="2">Specificity/NPA</td></tr><tr><td>TP/TP+FN</td><td>% (95% CI)</td><td>TN/TN+FP</td><td>% (95% CI)</td></tr><tr><td rowspan="7">Enterococcus faecalis</td><td>Prospective (Fresh)</td><td>21/21</td><td>100 (84.5-100)</td><td>291/291</td><td>100 (98.7-100)</td></tr><tr><td>Prospective (Frozen)</td><td>28/28</td><td>100 (87.9-100)</td><td>370/370</td><td>100 (99.0-100)</td></tr><tr><td>Prospective (All)</td><td>49/49</td><td>100 (92.7-100)</td><td>661/661</td><td>100 (99.4-100)</td></tr><tr><td>Retrospective</td><td>82/90</td><td>91.1 (83.4-95.4)</td><td>496/496</td><td>100 (99.2-100)</td></tr><tr><td>Prospective/Retrospective</td><td>131/139A</td><td>94.2 (89.1-97.1)</td><td>1157/1157</td><td>100 (99.7-100)</td></tr><tr><td>Contrived</td><td>52/52</td><td>100 (93.1-100)</td><td>513/513</td><td>100 (99.3-100)</td></tr><tr><td>Overall</td><td>183/191</td><td>95.8 (92.0-97.9)</td><td>1670/1670</td><td>100 (99.8-100)</td></tr></table>

A. Enterococcus faecalis was not detected in 4 false negative samples, but PCR/sequencing instead detected Enterococcus faecium (3) and Lactococcus lactis (1), which were not identified by standard laboratory procedures.

Table 12: Clinical Performance for Enterococcus faecium   

<table><tr><td rowspan="2">Target</td><td rowspan="2">Sample Type</td><td colspan="2">Sensitivity/PPA</td><td colspan="2">Specificity/NPA</td></tr><tr><td>TP/TP+FN</td><td>% (95% CI)</td><td>TN/TN+FP</td><td>% (95% CI)</td></tr><tr><td rowspan="7">Enterococcus faecium</td><td>Prospective (Fresh)</td><td>3/3</td><td>100 (43.9-100)</td><td>309/309</td><td>100 (98.8-100)</td></tr><tr><td>Prospective (Frozen)</td><td>8/9</td><td>88.9 (56.5-98.0)</td><td>388/389</td><td>99.7 (98.6-100)</td></tr><tr><td>Prospective (All)</td><td>11/12</td><td>91.7 (64.6-98.5)</td><td>697/698</td><td>99.9 (99.2-100)</td></tr><tr><td>Retrospective</td><td>52/53</td><td>98.1 (90.1-99.7)</td><td>526/533</td><td>98.7 (97.3-99.4)</td></tr><tr><td>Prospective/Retrospective</td><td>63/65</td><td>96.9 (89.5-99.2)</td><td>1223/1231A</td><td>99.4 (98.7-99.7)</td></tr><tr><td>Contrived</td><td>60/60</td><td>100 (94.0-100)</td><td>505/505</td><td>100 (99.2-100)</td></tr><tr><td>Overall</td><td>123/125</td><td>98.4 (94.4-99.6)</td><td>1728/1736</td><td>99.5 (99.1-99.8)</td></tr></table>

A. Enterococcus faecium was detected in 5/8 false positive samples using PCR/sequencing.

Table 13: Clinical Performance for Lactobacillus   

<table><tr><td rowspan="2">Target</td><td rowspan="2">Sample Type</td><td colspan="2">Sensitivity/PPA</td><td colspan="2">Specificity/NPA</td></tr><tr><td>TP/TP+FN</td><td>% (95% CI)</td><td>TN/TN+FP</td><td>% (95% CI)</td></tr><tr><td rowspan="7">Lactobacillus</td><td>Prospective (Fresh)</td><td>2/2</td><td>100 (34.2-100)</td><td>309/310</td><td>99.7 (98.2-99.9)</td></tr><tr><td>Prospective (Frozen)</td><td>2/2</td><td>100 (34.2-100)</td><td>397/397</td><td>100 (99.0-100)</td></tr><tr><td>Prospective (All)</td><td>4/4</td><td>100 (51.0-100)</td><td>706/707</td><td>99.9 (99.2-100)</td></tr><tr><td>Retrospective</td><td>9/9</td><td>100 (70.1-100)</td><td>576/577</td><td>99.8 (99.0-100)</td></tr><tr><td>Prospective/Retrospective</td><td>13/13</td><td>100 (77.2-100)</td><td>1282/1284A</td><td>99.8 (99.4-100)</td></tr><tr><td>Contrived</td><td>32/33</td><td>97.0 (84.7-99.5)</td><td>532/532</td><td>100 (99.3-100)</td></tr><tr><td>Overall</td><td>45/46</td><td>97.8 (88.7-99.6)</td><td>1814/1816</td><td>99.9 (99.6-100)</td></tr></table>

A. Lactobacillus casei was detected in 1/2 false positive samples using PCR/sequencing.

Table 14: Clinical Performance for Listeria   

<table><tr><td rowspan="2">Target</td><td rowspan="2">Sample Type</td><td colspan="2">Sensitivity/PPA</td><td colspan="2">Specificity/NPA</td></tr><tr><td>TP/TP+FN</td><td>% (95% CI)</td><td>TN/TN+FP</td><td>% (95% CI)</td></tr><tr><td rowspan="7">Listeria</td><td>Prospective (Fresh)</td><td>0/0</td><td>---</td><td>312/312</td><td>100 (98.8-100)</td></tr><tr><td>Prospective (Frozen)</td><td>0/0</td><td>---</td><td>398/399</td><td>99.7 (98.6-100)</td></tr><tr><td>Prospective (All)</td><td>0/0</td><td>---</td><td>710/711</td><td>99.9 (99.2-100)</td></tr><tr><td>Retrospective</td><td>2/2</td><td>100 (34.2-100)</td><td>584/584</td><td>100 (99.3-100)</td></tr><tr><td>Prospective/Retrospective</td><td>2/2</td><td>100 (34.2-100)</td><td>1294/1295</td><td>99.9 (99.6-100)</td></tr><tr><td>Contrived</td><td>74/75</td><td>98.7 (92.8-99.8)</td><td>490/490</td><td>100 (99.2-100)</td></tr><tr><td>Overall</td><td>76/77</td><td>98.7 (93.0-99.8)</td><td>1784/1785</td><td>99.9 (99.7-100)</td></tr></table>

Table 15: Clinical Performance for Listeria monocytogenes   

<table><tr><td rowspan="2">Target</td><td rowspan="2">Sample Type</td><td colspan="2">Sensitivity/PPA</td><td colspan="2">Specificity/NPA</td></tr><tr><td>TP/TP+FN</td><td>% (95% CI)</td><td>TN/TN+FP</td><td>% (95% CI)</td></tr><tr><td rowspan="7">Listeria monocytogenes</td><td>Prospective (Fresh)</td><td>0/0</td><td>---</td><td>312/312</td><td>100 (98.8-100)</td></tr><tr><td>Prospective (Frozen)</td><td>0/0</td><td>---</td><td>399/399</td><td>100 (99.0-100)</td></tr><tr><td>Prospective (All)</td><td>0/0</td><td>---</td><td>711/711</td><td>100 (99.5-100)</td></tr><tr><td>Retrospective</td><td>2/2</td><td>100 (34.2-100)</td><td>584/584</td><td>100 (99.3-100)</td></tr><tr><td>Prospective/Retrospective</td><td>2/2</td><td>100 (34.2-100)</td><td>1295/1295</td><td>100 (99.7-100)</td></tr><tr><td>Contrived</td><td>46/46</td><td>100 (92.3-100)</td><td>519/519</td><td>100 (99.3-100)</td></tr><tr><td>Overall</td><td>48/48</td><td>100 (92.6-100)</td><td>1814/1814</td><td>100 (99.8-100)</td></tr></table>

Table 16: Clinical Performance for Micrococcus   

<table><tr><td rowspan="2">Target</td><td rowspan="2">Sample Type</td><td colspan="2">Sensitivity/PPA</td><td colspan="2">Specificity/NPA</td></tr><tr><td>TP/TP+FN</td><td>% (95% CI)</td><td>TN/TN+FP</td><td>% (95% CI)</td></tr><tr><td rowspan="7">Micrococcus</td><td>Prospective (Fresh)</td><td>9/10</td><td>90.0 (59.6-98.2)</td><td>302/302</td><td>100 (98.7-100)</td></tr><tr><td>Prospective (Frozen)</td><td>10/11</td><td>90.9 (62.3-98.4)</td><td>388/388</td><td>100 (99.0-100)</td></tr><tr><td>Prospective (All)</td><td>19/21</td><td>90.5 (71.1-97.3)</td><td>690/690</td><td>100 (99.4-100)</td></tr><tr><td>Retrospective</td><td>20/23</td><td>87.0 (67.9-95.5)</td><td>562/563</td><td>99.8 (99.0-100)</td></tr><tr><td>Prospective/Retrospective</td><td>39/444</td><td>88.6 (76.0-95.0)</td><td>1252/1253</td><td>99.9 (99.5-100)</td></tr><tr><td>Contrived</td><td>27/27</td><td>100 (87.5-100)</td><td>538/538</td><td>100 (99.3-100)</td></tr><tr><td>Overall</td><td>66/71</td><td>93.0 (84.6-97.0)</td><td>1790/1791</td><td>99.9 (99.7-100)</td></tr></table>

A. Micrococcus was not detected in 3 false negative samples, but PCR/sequencing instead detected Brevibacterium ravenspurgense, Nesterenkonia halotolerans, and Staphylococcus pettenkoferi, which were not identified by standard laboratory procedures.

Table 17: Clinical Performance for Staphylococcus   

<table><tr><td rowspan="2">Target</td><td rowspan="2">Sample Type</td><td colspan="2">Sensitivity/PPA</td><td colspan="2">Specificity/NPA</td></tr><tr><td>TP/TP+FN</td><td>% (95% CI)</td><td>TN/TN+FP</td><td>% (95% CI)</td></tr><tr><td rowspan="7">Staphylococcus</td><td>Prospective (Fresh)</td><td>178/182</td><td>97.8 (94.5-99.1)</td><td>127/130</td><td>97.7 (93.4-99.2)</td></tr><tr><td>Prospective (Frozen)</td><td>269/274</td><td>98.2 (95.8-99.2)</td><td>123/125</td><td>98.4 (94.4-99.6)</td></tr><tr><td>Prospective (All)</td><td>447/456</td><td>98.0 (96.3-99.0)</td><td>250/255</td><td>98.0 (95.5-99.2)</td></tr><tr><td>Retrospective</td><td>185/191</td><td>96.9 (93.3-98.6)</td><td>390/395</td><td>98.7 (97.1-99.5)</td></tr><tr><td>Prospective/Retrospective</td><td>632/647A</td><td>97.7 (96.2-98.6)</td><td>640/650B</td><td>98.5 (97.2-99.2)</td></tr><tr><td>Contrived</td><td>105/105</td><td>100 (96.5-100)</td><td>460/460</td><td>100 (99.2-100)</td></tr><tr><td>Overall</td><td>737/752</td><td>98.0 (96.7-98.8)</td><td>1100/1110</td><td>99.1 (98.3-99.5)</td></tr></table>

A. Staphylococcus was not detected in 3 false negative samples, but PCR/sequencing instead detected Escherichia coli, Klebsiella pneumoniae, and Streptococcus salivarius, which were not identified by standard laboratory procedures. B. Staphylococcus was detected in 9/10 false positive samples using PCR/sequencing.

Table 18: Clinical Performance for Staphylococcus aureus   

<table><tr><td rowspan="2">Target</td><td rowspan="2">Sample Type</td><td colspan="2">Sensitivity/PPA</td><td colspan="2">Specificity/NPA</td></tr><tr><td>TP/TP+FN</td><td>% (95% CI)</td><td>TN/TN+FP</td><td>% (95% CI)</td></tr><tr><td rowspan="7">Staphylococcus aureus</td><td>Prospective (Fresh)</td><td>62/65</td><td>95.4 (87.3-98.4)</td><td>244/245</td><td>99.6 (97.7-99.9)</td></tr><tr><td>Prospective (Frozen)</td><td>98/101</td><td>97.0 (91.6-99.0)</td><td>222/223</td><td>99.6 (97.5-99.9)</td></tr><tr><td>Prospective (All)</td><td>160/166</td><td>96.4 (92.3-98.3)</td><td>466/468</td><td>99.6 (98.5-99.9)</td></tr><tr><td>Retrospective</td><td>122/125</td><td>97.6 (93.2-99.2)</td><td>454/458</td><td>99.1 (97.8-99.7)</td></tr><tr><td>Prospective/Retrospective</td><td>282/291A</td><td>96.9 (94.2-98.4)</td><td>920/926B</td><td>99.4 (98.6-99.7)</td></tr><tr><td>Contrived</td><td>59/59</td><td>100 (93.9-100)</td><td>506/506</td><td>100 (99.2-100)</td></tr><tr><td>Overall</td><td>341/350</td><td>97.4 (95.2-98.6)</td><td>1426/1432</td><td>99.6 (99.1-99.8)</td></tr></table>

A. Staphylococcus aureus was not detected in 3 false negative samples, but PCR/sequencing instead detected Klebsiella pneumoniae, Staphylococcus simulans, and Streptococcus agalactiae, which were not identified by standard laboratory procedures. B. Staphylococcus aureus was detected in $5 / 6$ false positive samples using PCR/sequencing.

Table 19: Clinical Performance for Staphylococcus epidermidis   

<table><tr><td rowspan="2">Target</td><td rowspan="2">Sample Type</td><td colspan="2">Sensitivity/PPA</td><td colspan="2">Specificity/NPA</td></tr><tr><td>TP/TP+FN</td><td>% (95% CI)</td><td>TN/TN+FP</td><td>% (95% CI)</td></tr><tr><td rowspan="7">Staphylococcus epidermidis</td><td>Prospective (Fresh)</td><td>59/63</td><td>93.7 (84.8-97.5)</td><td>223/229</td><td>97.4 (94.4-98.8)</td></tr><tr><td>Prospective (Frozen)</td><td>56/58</td><td>96.6 (88.3-99.0)</td><td>258/265</td><td>97.4 (94.6-98.7)</td></tr><tr><td>Prospective (All)</td><td>115/121</td><td>95.0 (89.6-97.7)</td><td>481/494</td><td>97.4 (95.6-98.5)</td></tr><tr><td>Retrospective</td><td>33/38</td><td>86.8 (72.7-94.2)</td><td>539/545</td><td>98.9 (97.6-99.5)</td></tr><tr><td>Prospective/Retrospective</td><td>148/1594</td><td>93.1 (88.0-96.1)</td><td>1020/1039B</td><td>98.2 (97.2-98.8)</td></tr><tr><td>Contrived</td><td>1/1</td><td>100 (20.7-100)</td><td>564/564</td><td>100 (99.3-100)</td></tr><tr><td>Overall</td><td>149/160</td><td>93.1 (88.1-96.1)</td><td>1584/1603</td><td>98.8 (98.2-99.2)</td></tr></table>

A. Staphylococcus epidermidis was not detected in 7 false negative samples, but PCR/sequencing instead detected Staphylococcus aureus (4), Staphylococcus capitis (1), Staphylococcus pettenkoferi (1), and Escherichia coli (1), which were not identified by standard laboratory procedures. B. Staphylococcus epidermidis was detected in 3/19 false positive samples using PCR/sequencing.

Table 20: Clinical Performance for Staphylococcus lugdunensis   

<table><tr><td rowspan="2">Target</td><td rowspan="2">Sample Type</td><td colspan="2">Sensitivity/PPA</td><td colspan="2">Specificity/NPA</td></tr><tr><td>TP/TP+FN</td><td>% (95% CI)</td><td>TN/TN+FP</td><td>% (95% CI)</td></tr><tr><td rowspan="7">Staphylococcus lugdunensis</td><td>Prospective (Fresh)</td><td>1/1</td><td>100 (20.7-100)</td><td>290/291</td><td>99.7 (98.1-99.9)</td></tr><tr><td>Prospective (Frozen)</td><td>1/1</td><td>100 (20.7-100)</td><td>321/322</td><td>99.7 (98.3-99.9)</td></tr><tr><td>Prospective (All)</td><td>2/2</td><td>100 (34.2-100)</td><td>611/613</td><td>99.7 (98.8-99.9)</td></tr><tr><td>Retrospective</td><td>4/4</td><td>100 (51.0-100)</td><td>579/579</td><td>100 (99.3-100)</td></tr><tr><td>Prospective/Retrospective</td><td>6/6</td><td>100 (61.0-100)</td><td>1190/1192A</td><td>99.8 (99.4-100)</td></tr><tr><td>Contrived</td><td>45/45</td><td>100 (92.1-100)</td><td>519/520</td><td>99.8 (98.9-100)</td></tr><tr><td>Overall</td><td>51/51</td><td>100 (93.0-100)</td><td>1709/1712</td><td>99.8 (99.5-99.9)</td></tr></table>

A. Staphylococcus lugdunensis was detected in 2/2 false positive samples using PCR/sequencing.

Table 21: Clinical Performance for Streptococcus   

<table><tr><td rowspan="2">Target</td><td rowspan="2">Sample Type</td><td colspan="2">Sensitivity/PPA</td><td colspan="2">Specificity/NPA</td></tr><tr><td>TP/TP+FN</td><td>% (95% CI)</td><td>TN/TN+FP</td><td>% (95% CI)</td></tr><tr><td rowspan="7">Streptococcus</td><td>Prospective (Fresh)</td><td>54/55</td><td>98.2 (90.4-99.7)</td><td>253/257</td><td>98.4 (96.1-99.4)</td></tr><tr><td>Prospective (Frozen)</td><td>49/55</td><td>89.1 (78.2-94.9)</td><td>341/344</td><td>99.1 (97.5-99.7)</td></tr><tr><td>Prospective (All)</td><td>103/110</td><td>93.6 (87.4-96.9)</td><td>594/601</td><td>98.8 (97.6-99.4)</td></tr><tr><td>Retrospective</td><td>171/173</td><td>98.8 (95.9-99.7)</td><td>411/413</td><td>99.5 (98.3-99.9)</td></tr><tr><td>Prospective/Retrospective</td><td>274/283</td><td>96.8 (94.1-98.3)</td><td>1005/1014A</td><td>99.1 (98.3-99.5)</td></tr><tr><td>Contrived</td><td>57/57</td><td>100 (93.7-100)</td><td>508/508</td><td>100 (99.2-100)</td></tr><tr><td>Overall</td><td>331/340</td><td>97.4 (95.0-98.6)</td><td>1513/1522</td><td>99.4 (98.9-99.7)</td></tr></table>

A. Streptococcus was detected in 8/9 false positive samples using PCR/sequencing.

Table 22: Clinical Performance for Streptococcus agalactiae   

<table><tr><td rowspan="2">Target</td><td rowspan="2">Sample Type</td><td colspan="2">Sensitivity/PPA</td><td colspan="2">Specificity/NPA</td></tr><tr><td>TP/TP+FN</td><td>% (95% CI)</td><td>TN/TN+FP</td><td>% (95% CI)</td></tr><tr><td rowspan="7">Streptococcus agalactiae</td><td>Prospective (Fresh)</td><td>5/6</td><td>83.3 (43.6-97.0)</td><td>298/300</td><td>99.3 (97.6-99.8)</td></tr><tr><td>Prospective (Frozen)</td><td>5/5</td><td>100 (56.6-100)</td><td>374/374</td><td>100 (99.0-100)</td></tr><tr><td>Prospective (All)</td><td>10/11</td><td>90.9 (62.3-98.4)</td><td>672/674</td><td>99.7 (98.9-99.9)</td></tr><tr><td>Retrospective</td><td>36/37</td><td>97.3 (86.2-99.5)</td><td>548/548</td><td>100 (99.3-100)</td></tr><tr><td>Prospective/Retrospective</td><td>46/48 A</td><td>95.8 (86.0-98.8)</td><td>1220/1222B</td><td>99.8 (99.4-100)</td></tr><tr><td>Contrived</td><td>8/8</td><td>100 (67.6-100)</td><td>557/557</td><td>100 (99.3-100)</td></tr><tr><td>Overall</td><td>54/56</td><td>96.4 (87.9-99.0)</td><td>1777/1779</td><td>99.9 (99.6-100)</td></tr></table>

A. Streptococcus agalactiae was not detected in 1 false negative sample, but PCR/sequencing instead detected Streptococcus mitis, which was not identified by standard laboratory procedures. B. Streptococcus agalactiae was detected in 1/2 false positive samples using PCR/sequencing.

Table 23: Clinical Performance for Streptococcus anginosus group   

<table><tr><td rowspan="2">Target</td><td rowspan="2">Sample Type</td><td colspan="2">Sensitivity/PPA</td><td colspan="2">Specificity/NPA</td></tr><tr><td>TP/TP+FN</td><td>% (95% CI)</td><td>TN/TN+FP</td><td>% (95% CI)</td></tr><tr><td rowspan="7">Streptococcus anginosus group</td><td>Prospective (Fresh)</td><td>3/3</td><td>100 (43.9-100)</td><td>303/303</td><td>100 (98.7-100)</td></tr><tr><td>Prospective (Frozen)</td><td>1/2</td><td>50.0 (9.5-90.5)</td><td>375/377</td><td>99.5 (98.1-99.9)</td></tr><tr><td>Prospective (All)</td><td>4/5</td><td>80.0 (37.6-96.4)</td><td>678/680</td><td>99.7 (98.9-99.9)</td></tr><tr><td>Retrospective</td><td>38/40</td><td>95.0 (83.5-98.6)</td><td>544/545</td><td>99.8 (99.0-100)</td></tr><tr><td>Prospective/Retrospective</td><td>42/45A</td><td>93.3 (82.1-97.7)</td><td>1222/1225B</td><td>99.8 (99.3-99.9)</td></tr><tr><td>Contrived</td><td>23/23</td><td>100 (85.7-100)</td><td>542/542</td><td>100 (99.3-100)</td></tr><tr><td>Overall</td><td>65/68</td><td>95.6 (87.8-98.5)</td><td>1764/1767</td><td>99.8 (99.5-99.9)</td></tr></table>

A. Streptococcus anginosus group was not detected in 3 false negative samples, but PCR/sequencing instead detected Granulicatella adiacens, Streptococcus dysgalactiae, and Streptococcus lutetiensis, which were not identified by standard laboratory procedures. B. Streptococcus intermedius was detected in 1/3 false positive samples using PCR/sequencing.

Table 24: Clinical Performance for Streptococcus pneumoniae   

<table><tr><td rowspan="2">Target</td><td rowspan="2">Sample Type</td><td colspan="2">Sensitivity/PPA</td><td colspan="2">Specificity/NPA</td></tr><tr><td>TP/TP+FN</td><td>% (95% CI)</td><td>TN/TN+FP</td><td>% (95% CI)</td></tr><tr><td rowspan="7">Streptococcus pneumoniae</td><td>Prospective (Fresh)</td><td>19/19</td><td>100 (83.2-100)</td><td>286/287</td><td>99.7 (98.1-99.9)</td></tr><tr><td>Prospective (Frozen)</td><td>8/9</td><td>88.9 (56.5-98.0)</td><td>370/370</td><td>100 (99.0-100)</td></tr><tr><td>Prospective (All)</td><td>27/28</td><td>96.4 (82.3-99.4)</td><td>656/657</td><td>99.8 (99.1-100)</td></tr><tr><td>Retrospective</td><td>39/41</td><td>95.1 (83.9-98.7)</td><td>542/543</td><td>99.8 (99.0-100)</td></tr><tr><td>Prospective/Retrospective</td><td>66/694</td><td>95.7 (88.0-98.5)</td><td>1198/1200B</td><td>99.8 (99.4-100)</td></tr><tr><td>Contrived</td><td>0/0</td><td>---</td><td>565/565</td><td>100 (99.3-100)</td></tr><tr><td>Overall</td><td>66/69</td><td>95.7 (88.0-98.5)</td><td>1763/1765</td><td>99.9 (99.6-100)</td></tr></table>

A. Streptococcus pneumoniae was not detected in 3 false negative samples, but PCR/sequencing instead detected Streptococcus mitis (2) and Streptococcus anginosus (1), which were not detected by standard laboratory procedures. B. Streptococcus pneumoniae was detected in 1/2 false positive samples using PCR/sequencing.

Table 25: Clinical Performance for Streptococcus pyogenes   

<table><tr><td rowspan="2">Target</td><td rowspan="2">Sample Type</td><td colspan="2">Sensitivity/PPA</td><td colspan="2">Specificity/NPA</td></tr><tr><td>TP/TP+FN</td><td>% (95% CI)</td><td>TN/TN+FP</td><td>% (95% CI)</td></tr><tr><td rowspan="7">Streptococcus pyogenes</td><td>Prospective (Fresh)</td><td>4/4</td><td>100 (51.0-100)</td><td>302/302</td><td>100 (98.7-100)</td></tr><tr><td>Prospective (Frozen)</td><td>4/4</td><td>100 (51.0-100)</td><td>375/375</td><td>100 (99.0-100)</td></tr><tr><td>Prospective (All)</td><td>8/8</td><td>100 (67.6-100)</td><td>677/677</td><td>100 (99.4-100)</td></tr><tr><td>Retrospective</td><td>19/20</td><td>95.0 (76.4-99.1)</td><td>564/564</td><td>100 (99.3-100)</td></tr><tr><td>Prospective/Retrospective</td><td>27/28</td><td>96.4 (82.3-99.4)</td><td>1241/1241</td><td>100 (99.7-100)</td></tr><tr><td>Contrived</td><td>26/26</td><td>100 (87.1-100)</td><td>539/539</td><td>100 (99.3-100)</td></tr><tr><td>Overall</td><td>53/54</td><td>98.1 (90.2-99.7)</td><td>1780/1780</td><td>100 (99.8-100)</td></tr></table>

Table 26: Clinical Performance for mecA   

<table><tr><td rowspan="2">Target</td><td rowspan="2">Sample Type</td><td colspan="2">Sensitivity/PPA</td><td colspan="2">Specificity/NPA</td></tr><tr><td>TP/TP+FN</td><td>% (95% CI)</td><td>TN/TN+FP</td><td>% (95% CI)</td></tr><tr><td rowspan="7">mecA Staphylococcus</td><td>Prospective (Fresh)</td><td>86/89</td><td>96.6 (90.6-98.8)</td><td>85/93</td><td>91.4 (83.9-95.6)</td></tr><tr><td>Prospective (Frozen)</td><td>164/171</td><td>95.9 (91.8-98.0)</td><td>101/103</td><td>98.1 (93.2-99.5)</td></tr><tr><td>Prospective (All)</td><td>250/260</td><td>96.2 (93.1-97.9)</td><td>186/196</td><td>94.9 (90.9-97.2)</td></tr><tr><td>Retrospective</td><td>151/153</td><td>98.7 (95.4-99.6)</td><td>37/38</td><td>97.4 (86.5-99.5)</td></tr><tr><td>Prospective/Retrospective</td><td>401/413</td><td>97.1 (95.0-98.3)</td><td>223/234A</td><td>95.3 (91.8-97.4)</td></tr><tr><td>Contrived</td><td>11/11</td><td>100 (74.1-100)</td><td>94/94</td><td>100 (96.1-100)</td></tr><tr><td>Overall</td><td>412/424</td><td>97.2 (95.1-98.4)</td><td>317/328</td><td>96.6 (94.1-98.1)</td></tr><tr><td rowspan="7">mecA Staphylococcus</td><td>Prospective (Fresh)</td><td>27/28</td><td>96.4 (82.3-99.4)</td><td>34/37</td><td>91.9 (78.7-97.2)</td></tr><tr><td>Prospective (Frozen)</td><td>56/58</td><td>96.6 (88.3-99.0)</td><td>43/43</td><td>100 (91.8-100)</td></tr><tr><td>Prospective (All)</td><td>83/86</td><td>96.5 (90.2-98.8)</td><td>77/80</td><td>96.3 (89.5-98.7)</td></tr><tr><td>Retrospective</td><td>107/108</td><td>99.1 (94.9-99.8)</td><td>16/17</td><td>94.1 (73.0-99.0)</td></tr><tr><td>Prospective/Retrospective</td><td>190/194</td><td>97.9 (94.8-99.2)</td><td>93/97</td><td>95.9 (89.9-98.4)</td></tr><tr><td>Contrived</td><td>10/10</td><td>100 (72.2-100)</td><td>49/49</td><td>100 (92.7-100)</td></tr><tr><td>Overall</td><td>200/204</td><td>98.0 (95.1-99.2)</td><td>142/146</td><td>97.3 (93.2-98.9)</td></tr><tr><td rowspan="7">mecA Staphylococcus epidermidis</td><td>Prospective (Fresh)</td><td>36/36</td><td>100 (90.4-100)</td><td>24/27</td><td>88.9 (71.9-96.1)</td></tr><tr><td>Prospective (Frozen)</td><td>41/43</td><td>95.3 (84.5-98.7)</td><td>15/15</td><td>100 (79.6-100)</td></tr><tr><td>Prospective (All)</td><td>77/79</td><td>97.5 (91.2-99.3)</td><td>39/42</td><td>92.9 (81.0-97.5)</td></tr><tr><td>Retrospective</td><td>30/30</td><td>100 (88.6-100)</td><td>43289</td><td>87.5 (52.9-97.8)</td></tr><tr><td>Prospective/Retrospective</td><td>107/109</td><td>98.2 (93.6-99.5)</td><td>46/50</td><td>92.0 (81.2-96.8)</td></tr><tr><td>Contrived</td><td>1/1</td><td>100 (20.7-100)</td><td>0/0</td><td>-</td></tr><tr><td>Overall</td><td>108/110</td><td>98.2 (93.6-99.5)</td><td>46/50</td><td>92.0 (81.2-96.8)</td></tr><tr><td rowspan="7">mecA Staphylococcus</td><td>Prospective (Fresh)</td><td>0/0</td><td>---</td><td>1/1</td><td>100 (20.7-100)</td></tr><tr><td>Prospective (Frozen)</td><td>0/0</td><td>---</td><td>1/1</td><td>100 (20.7-100)</td></tr><tr><td>Prospective (All)</td><td>0/0</td><td>---</td><td>2/2</td><td>100 (34.2-100)</td></tr><tr><td>Retrospective</td><td>1/1</td><td>100 (20.7-100)</td><td>3/3</td><td>100 (43.9-100)</td></tr><tr><td>Prospective/Retrospective</td><td>1/1</td><td>100 (20.7-100)</td><td>5/5</td><td>100 (56.6-100)</td></tr><tr><td>Contrived</td><td>0/0</td><td>---</td><td>45/45</td><td>100 (92.1-100)</td></tr><tr><td>Overall</td><td>1/1</td><td>100 (20.7-100)</td><td>50/50</td><td>100 (92.9-100)</td></tr></table>

A. mecA was detected in 4 of the 7 false positive samples that were tested with an FDA-cleared multiplex assay.

Table 27: Clinical Performance for mecC   

<table><tr><td rowspan="2">Target</td><td rowspan="2">Sample Type</td><td colspan="2">Sensitivity/PPA</td><td colspan="2">Specificity/NPA</td></tr><tr><td>TP/TP+FN</td><td>% (95% CI)</td><td>TN/TN+FP</td><td>% (95% CI)</td></tr><tr><td rowspan="7">mecC Staphylococcus</td><td>Prospective (Fresh)</td><td>0/0</td><td>---</td><td>182/182</td><td>100 (97.9-100)</td></tr><tr><td>Prospective (Frozen)</td><td>0/0</td><td></td><td>274/274</td><td>100 (98.6-100)</td></tr><tr><td>Prospective (All)</td><td>0/0</td><td>---</td><td>456/456</td><td>100 (99.2-100)</td></tr><tr><td>Retrospective</td><td>0/0</td><td>---</td><td>191/191</td><td>100 (98.0-100)</td></tr><tr><td>Prospective/Retrospective</td><td>0/0</td><td>---</td><td>647/647</td><td>100 (99.4-100)</td></tr><tr><td>Contrived</td><td>49/49</td><td>100 (92.7-100)</td><td>56/56</td><td>100 (93.6-100)</td></tr><tr><td>Overall</td><td>49/49</td><td>100 (92.7-100)</td><td>703/703</td><td>100 (99.5-100)</td></tr><tr><td rowspan="7">mecC Staphylococcus aureus</td><td>Prospective (Fresh)</td><td>0/0</td><td>---</td><td>65/65</td><td>100 (94.4-100)</td></tr><tr><td>Prospective (Frozen)</td><td>0/0</td><td>---</td><td>101/101</td><td>100 (96.3-100)</td></tr><tr><td>Prospective (All)</td><td>0/0</td><td>--</td><td>166/166</td><td>100 (97.7-100)</td></tr><tr><td>Retrospective</td><td>0/0</td><td>---</td><td>125/125</td><td>100 (97.0-100)</td></tr><tr><td>Prospective/Retrospective</td><td>0/0</td><td>---</td><td>291/291</td><td>100 (98.7-100)</td></tr><tr><td>Contrived</td><td>49/49</td><td>100 (92.7-100)</td><td>10/10</td><td>100 (72.2-100)</td></tr><tr><td>Overall</td><td>49/49</td><td>100 (92.7-100)</td><td>301/301</td><td>100 (98.7-100)</td></tr><tr><td rowspan="7">mecC Staphylococcus epidermidis</td><td>Prospective (Fresh)</td><td>0/0</td><td>---</td><td>63/63</td><td>100 (94.3-100)</td></tr><tr><td>Prospective (Frozen)</td><td>0/0</td><td>---</td><td>58/58</td><td>100 (93.8-100)</td></tr><tr><td>Prospective (All)</td><td>0/0</td><td>---</td><td>121/121</td><td>100 (96.9-100)</td></tr><tr><td>Retrospective</td><td>0/0</td><td>---</td><td>38/38</td><td>100 (90.8-100)</td></tr><tr><td>Prospective/Retrospective</td><td>0/0</td><td>---</td><td>159/159</td><td>100 (97.6-100)</td></tr><tr><td>Contrived</td><td>0/0</td><td>---</td><td>1/1</td><td>100 (20.7-100)</td></tr><tr><td>Overall</td><td>0/0</td><td>--</td><td>160/160</td><td>100 (97.7-100)</td></tr><tr><td rowspan="7">mecC Staphylococcus</td><td>Prospective (Fresh)</td><td>0/0</td><td>---</td><td>1/1</td><td>100 (20.7-100)</td></tr><tr><td>Prospective (Frozen)</td><td>0/0</td><td>--</td><td>1/1</td><td>100 (20.7-100)</td></tr><tr><td>Prospective (All)</td><td>0/0</td><td>---</td><td>2/2</td><td>100 (34.2-100)</td></tr><tr><td>Retrospective</td><td>0/0</td><td>---</td><td>4/4</td><td>100 (51.0-100)</td></tr><tr><td>Prospective/Retrospective</td><td>0/0</td><td>--</td><td>6/6</td><td>100 (61.0-100)</td></tr><tr><td>Contrived</td><td>0/0</td><td>---</td><td>45/45</td><td>100 (92.1-100)</td></tr><tr><td>Overall</td><td>0/0</td><td>---</td><td>51/51</td><td>100 (93.0-100)</td></tr></table>

Table 28: Clinical Performance for vanA   

<table><tr><td rowspan="2">Target</td><td rowspan="2">Sample Type</td><td colspan="2">Sensitivity/PPA</td><td colspan="2">Specificity/NPA</td></tr><tr><td>TP/TP+FN</td><td>% (95% CI)</td><td>TN/TN+FP</td><td>% (95% CI)</td></tr><tr><td rowspan="7">vanA Enterococcus</td><td>Prospective (Fresh)</td><td>0/0</td><td>--</td><td>24/25</td><td>96.0 (80.5-99.3)</td></tr><tr><td>Prospective (Frozen)</td><td>8/8</td><td>100 (67.6-100)</td><td>28/28</td><td>100 (87.9-100)</td></tr><tr><td>Prospective (All)</td><td>8/8</td><td>100 (67.6-100)</td><td>52/53</td><td>98.1 (90.1-99.7)</td></tr><tr><td>Retrospective</td><td>53/57</td><td>93.0 (83.3-97.2)</td><td>89/90</td><td>98.9 (94.0-99.8)</td></tr><tr><td>Prospective/Retrospective</td><td>61/654</td><td>93.8 (85.2-97.6)</td><td>141/143B</td><td>98.6 (95.0-99.6)</td></tr><tr><td>Contrived</td><td>60/60</td><td>100 (94.0-100)</td><td>66/66</td><td>100 (94.5-100)</td></tr><tr><td>Overall</td><td>121/125</td><td>96.8 (92.1-98.7)</td><td>207/209</td><td>99.0 (96.6-99.7)</td></tr><tr><td rowspan="7">vanA Enterococcus faecalis</td><td>Prospective (Fresh)</td><td>0/0</td><td>---</td><td>21/21</td><td>100 (84.5-100)</td></tr><tr><td>Prospective (Frozen)</td><td>1/1</td><td>100 (20.7-100)</td><td>27/27</td><td>100 (87.5-100)</td></tr><tr><td>Prospective (All)</td><td>1/1</td><td>100 (20.7-100)</td><td>48/48</td><td>100 (92.6-100)</td></tr><tr><td>Retrospective</td><td>11/14</td><td>78.6 (52.4-92.4)</td><td>76/76</td><td>100 (95.2-100)</td></tr><tr><td>Prospective/Retrospective</td><td>12/15</td><td>80.0 (54.8-93.0)</td><td>124/124</td><td>100 (97.0-100)</td></tr><tr><td>Contrived</td><td>10/10</td><td>100 (72.2-100)</td><td>42/42</td><td>100 (91.6-100)</td></tr><tr><td>Overall</td><td>22/25</td><td>88.0 (70.0-95.8)</td><td>166/166</td><td>100 (97.7-100)</td></tr><tr><td rowspan="7">vanA Enterococcus faecium</td><td>Prospective (Fresh)</td><td>0/0</td><td>---</td><td>2/3</td><td>66.7 (20.8-93.9)</td></tr><tr><td>Prospective (Frozen)</td><td>7/7</td><td>100 (64.6-100)</td><td>2/2</td><td>100 (34.2-100)</td></tr><tr><td>Prospective (All)</td><td>7/7</td><td>100 (64.6-100)</td><td>4/5</td><td>80.0 (37.6-96.4)</td></tr><tr><td>Retrospective</td><td>44/44</td><td>100 (92.0-100)</td><td>8/9</td><td>88.9 (56.5-98.0)</td></tr><tr><td>Prospective/Retrospective</td><td>51/51</td><td>100 (93.0-100)</td><td>12/14</td><td>85.7 (60.1-96.0)</td></tr><tr><td>Contrived</td><td>50/50</td><td>100 (92.9-100)</td><td>10/10</td><td>100 (72.2-100)</td></tr><tr><td>Overall</td><td>101/101</td><td>100 (96.3-100)</td><td>22/24</td><td>91.7 (74.2-97.7)</td></tr></table>

A. vanA was not detected in 1 false negative sample using an FDA-cleared multiplex assay. B. vanA was detected in the 1 false positive sample that was tested using an FDA-cleared multiplex assay.

Table 29: Clinical Performance for vanB   

<table><tr><td rowspan="2">Target</td><td rowspan="2">Sample Type</td><td colspan="2">Sensitivity/PPA</td><td colspan="2">Specificity/NPA</td></tr><tr><td>TP/TP+FN</td><td>% (95% CI)</td><td>TN/TN+FP</td><td>% (95% CI)</td></tr><tr><td rowspan="7">vanB Enterococcus</td><td>Prospective (Fresh)</td><td>0/0</td><td>---</td><td>25/25</td><td>100 (86.7-100)</td></tr><tr><td>Prospective (Frozen)</td><td>0/0</td><td>---</td><td>36/36</td><td>100 (90.4-100)</td></tr><tr><td>Prospective (All)</td><td>0/0</td><td>---</td><td>61/61</td><td>100 (94.1-100)</td></tr><tr><td>Retrospective</td><td>1/1</td><td>100 (20.7-100)</td><td>146/146</td><td>100 (97.4-100)</td></tr><tr><td>Prospective/Retrospective</td><td>1/1</td><td>100 (20.7-100)</td><td>207/207</td><td>100 (98.2-100)</td></tr><tr><td>Contrived</td><td>52/52</td><td>100 (93.1-100)</td><td>74/74</td><td>100 (95.1-100)</td></tr><tr><td>Overall</td><td>53/53</td><td>100 (93.2-100)</td><td>281/281</td><td>100 (98.7-100)</td></tr><tr><td rowspan="7">vanB Enterococcus faecalis</td><td>Prospective (Fresh)</td><td>0/0</td><td>---</td><td>21/21</td><td>100 (84.5-100)</td></tr><tr><td>Prospective (Frozen)</td><td>0/0</td><td>---</td><td>28/28</td><td>100 (87.9-100)</td></tr><tr><td>Prospective (All)</td><td>0/0</td><td>---</td><td>49/49</td><td>100 (92.7-100)</td></tr><tr><td>Retrospective</td><td>1/1</td><td>100 (20.7-100)</td><td>89/89</td><td>100 (95.9-100)</td></tr><tr><td>Prospective/Retrospective</td><td>1/1</td><td>100 (20.7-100)</td><td>138/138</td><td>100 (97.3-100)</td></tr><tr><td>Contrived</td><td>42/42</td><td>100 (91.6-100)</td><td>10/10</td><td>100 (72.2-100)</td></tr><tr><td>Overall</td><td>43/43</td><td>100 (91.8-100)</td><td>148/148</td><td>100 (97.5-100)</td></tr><tr><td rowspan="7">vanB Enterococcus faecium</td><td>Prospective (Fresh)</td><td>0/0</td><td>---</td><td>3/3</td><td>100 (43.9-100)</td></tr><tr><td>Prospective (Frozen)</td><td>0/0</td><td>---</td><td>9/9</td><td>100 (70.1-100)</td></tr><tr><td>Prospective (All)</td><td>0/0</td><td>---</td><td>12/12</td><td>100 (75.8-100)</td></tr><tr><td>Retrospective</td><td>0/0</td><td>--</td><td>53/53</td><td>100 (93.2-100)</td></tr><tr><td>Prospective/Retrospective</td><td>0/0</td><td>---</td><td>65/65</td><td>100 (94.4-100)</td></tr><tr><td>Contrived</td><td>10/10</td><td>100 (72.2-100)</td><td>50/50</td><td>100 (92.9-100)</td></tr><tr><td>Overall</td><td>10/10</td><td>100 (72.2-100)</td><td>115/115</td><td>100 (96.8-100)</td></tr></table>

# Pan Targets

In addition to the evaluable prospective and retrospective samples that contain gram-positive organisms, the clinical performance of the Pan Candida and Pan Gram-Negative targets was evaluated by testing an additional 480 non-intended use retrospective samples with gramnegative or fungal organisms; these are denoted as Retrospective (Non-Intended Use) samples. Results from those samples are summarized in Table 30.

Table 30: Clinical Performance for Pan Targets   

<table><tr><td rowspan="2">Target</td><td rowspan="2">Sample Type</td><td colspan="2">Sensitivity/PPA</td><td colspan="2">Specificity/NPA</td></tr><tr><td>TP/TP+FN</td><td>% (95% CI)</td><td>TN/TN+FP</td><td>% (95% CI)</td></tr><tr><td rowspan="6">Pan Candida</td><td>Prospective (Fresh)</td><td>0/0</td><td>---</td><td>312/312</td><td>100 (98.8-100)</td></tr><tr><td>Prospective (Frozen)</td><td>0/0</td><td>---</td><td>399/399</td><td>100 (99.0-100)</td></tr><tr><td>Prospective (All)</td><td>0/0</td><td>---</td><td>711/711</td><td>100 (99.5-100)</td></tr><tr><td>Retrospective</td><td>7/9</td><td>77.8 (45.3-93.7)</td><td>576/577</td><td>99.8 (99.0-100)</td></tr><tr><td>Retrospective (Non-Intended Use)</td><td>90/96</td><td>93.8 (87.0-97.1)</td><td>383/3844</td><td>99.7 (98.5-100)</td></tr><tr><td>Contrived</td><td>0/0</td><td>---</td><td>565/565</td><td>100 (99.3-100)</td></tr><tr><td rowspan="6">Pan Gram- Negative</td><td>Prospective (Fresh)</td><td>10/11</td><td>90.9 (62.3-98.4)</td><td>299/301</td><td>99.3 (97.6-99.8)</td></tr><tr><td>Prospective (Frozen)</td><td>12/12</td><td>100 (75.8-100)</td><td>386/387</td><td>99.7 (98.6-100)</td></tr><tr><td>Prospective (All)</td><td>22/23</td><td>95.7 (79.0-99.2)</td><td>685/688B</td><td>99.6 (98.7-99.9)</td></tr><tr><td>Retrospective</td><td>36/43</td><td>83.7 (70.0-91.9)</td><td>540/543c</td><td>99.4 (98.4-99.8)</td></tr><tr><td>Retrospective (Non-Intended Use)</td><td>364/375</td><td>97.1 (94.8-98.4)</td><td>104/105</td><td>99.0 (94.8-99.8)</td></tr><tr><td>Contrived</td><td>0/0</td><td>---</td><td>565/565</td><td>100 (99.3-100)</td></tr></table>

A. Candida glabrata was detected in $1 / 1$ false positive samples using PCR/sequencing. B. A gram-negative organism, Klebsiella pneumoniae, was detected in 1/3 false positive samples using PCR/sequencing. C. A gram-negative organism, Escherichia coli, was detected in 1/3 false positive samples using PCR/sequencing.

Table 31: Contrived Sample Summary   

<table><tr><td colspan="1" rowspan="1">Target</td><td colspan="1" rowspan="1">Organism</td><td colspan="1" rowspan="1">Strain</td><td colspan="1" rowspan="1">Independent ContrivedSamples Tested</td></tr><tr><td colspan="1" rowspan="8">Bacillus cereusgroup</td><td colspan="1" rowspan="4">Bacillus cereus</td><td colspan="1" rowspan="1">ATCC 10876</td><td colspan="1" rowspan="1">11</td></tr><tr><td colspan="1" rowspan="1">ATCC 21769</td><td colspan="1" rowspan="1">10</td></tr><tr><td colspan="1" rowspan="1">ATCC 31430</td><td colspan="1" rowspan="1">9</td></tr><tr><td colspan="1" rowspan="1">ATCC 53522</td><td colspan="1" rowspan="1">10</td></tr><tr><td colspan="1" rowspan="3">Bacillus thuringiensis</td><td colspan="1" rowspan="1">ATCC 33679</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">ATCC 10792</td><td colspan="1" rowspan="1">2</td></tr><tr><td colspan="1" rowspan="1">ATCC 55173</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="2" rowspan="1">Bacillus cereus group total</td><td colspan="1" rowspan="1">46</td></tr><tr><td colspan="1" rowspan="13">Bacillus subtilisgroup</td><td colspan="1" rowspan="3">Bacillus amyloliquefaciens</td><td colspan="1" rowspan="1">ATCC 23350</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">ATCC 23845</td><td colspan="1" rowspan="1">4</td></tr><tr><td colspan="1" rowspan="1">ATCC 53495</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="3">Bacillus atrophaeus</td><td colspan="1" rowspan="1">ATCC 51189</td><td colspan="1" rowspan="1">4</td></tr><tr><td colspan="1" rowspan="1">ATCC 6455</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">ATCC 6537</td><td colspan="1" rowspan="1">4</td></tr><tr><td colspan="1" rowspan="3">Bacillus licheniformis</td><td colspan="1" rowspan="1">ATCC 21039</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">ATCC 21667</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">ATCC 53926</td><td colspan="1" rowspan="1">4</td></tr><tr><td colspan="1" rowspan="3">Bacillus subtilis</td><td colspan="1" rowspan="1">ATCC 15040</td><td colspan="1" rowspan="1">5</td></tr><tr><td colspan="1" rowspan="1">ATCC 15561</td><td colspan="1" rowspan="1">8</td></tr><tr><td colspan="1" rowspan="1">ATCC 55614</td><td colspan="1" rowspan="1">6</td></tr><tr><td colspan="1" rowspan="1">Bacillus subtilis group total</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">50</td></tr><tr><td colspan="1" rowspan="4">Corynebacterium</td><td colspan="1" rowspan="1">Corynebacterium coyleae</td><td colspan="1" rowspan="1">ATCC 700219</td><td colspan="1" rowspan="1">7</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium falsenii</td><td colspan="1" rowspan="1">ATCC BAA-596</td><td colspan="1" rowspan="1">9</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium striatum</td><td colspan="1" rowspan="1">ATCC BAA-1293</td><td colspan="1" rowspan="1">4</td></tr><tr><td colspan="2" rowspan="1">Corynebacterium total</td><td colspan="1" rowspan="1">20</td></tr><tr><td colspan="1" rowspan="1">EY</td><td></td><td></td><td></td></tr><tr><td colspan="1" rowspan="1">Target</td><td colspan="1" rowspan="1">Organism</td><td colspan="1" rowspan="1">Strain</td><td colspan="1" rowspan="1">Independent ContrivedSamples Tested</td></tr><tr><td colspan="1" rowspan="27">Enterococcus</td><td colspan="1" rowspan="1">Enterococcus faecalis, vanA</td><td colspan="1" rowspan="1">JMI 876745</td><td colspan="1" rowspan="1">10</td></tr><tr><td colspan="1" rowspan="4">Enterococcus faecalis, vanB</td><td colspan="1" rowspan="1">ATCC 51299</td><td colspan="1" rowspan="1">11</td></tr><tr><td colspan="1" rowspan="1">ATCC 51575</td><td colspan="1" rowspan="1">11</td></tr><tr><td colspan="1" rowspan="1">ATCC 700802</td><td colspan="1" rowspan="1">10</td></tr><tr><td colspan="1" rowspan="1">ATCC BAA-2365</td><td colspan="1" rowspan="1">10</td></tr><tr><td colspan="1" rowspan="13">Enterococcus faecium, vanA</td><td colspan="1" rowspan="1">ATCC 51559</td><td colspan="1" rowspan="1">4</td></tr><tr><td colspan="1" rowspan="1">ATCC 700221</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">ATCC BAA-2316</td><td colspan="1" rowspan="1">5</td></tr><tr><td colspan="1" rowspan="1">ATCC BAA-2317</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">ATCC BAA-2318</td><td colspan="1" rowspan="1">5</td></tr><tr><td colspan="1" rowspan="1">ATCC BAA-2319</td><td colspan="1" rowspan="1">5</td></tr><tr><td colspan="1" rowspan="1">ATCC BAA-2320</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">LMC 002867</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">LMC 003921</td><td colspan="1" rowspan="1">4</td></tr><tr><td colspan="1" rowspan="1">LMC 032261</td><td colspan="1" rowspan="1">4</td></tr><tr><td colspan="1" rowspan="1">LMC 055971</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">LMC 103676</td><td colspan="1" rowspan="1">5</td></tr><tr><td colspan="1" rowspan="1">LMC 104266</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">Enterococcus faecium, vanB</td><td colspan="1" rowspan="1">ATCC 51858</td><td colspan="1" rowspan="1">10</td></tr><tr><td colspan="1" rowspan="1">Enterococcus flavescens</td><td colspan="1" rowspan="1">ATCC 49996</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="2">Enterococcus gallinarum</td><td colspan="1" rowspan="1">ATCC 49610</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">ATCC 700425</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">Enterococcus hirae</td><td colspan="1" rowspan="1">ATCC 10541</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">Enterococcus malodoratus</td><td colspan="1" rowspan="1">ATCC 43197</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">Enterococcus raffinosus</td><td colspan="1" rowspan="1">ATCC 49464</td><td colspan="1" rowspan="1">2</td></tr><tr><td colspan="1" rowspan="1">Enterococcus saccharolyticus</td><td colspan="1" rowspan="1">ATCC 43076</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">Enterococcus total</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">126</td></tr><tr><td colspan="1" rowspan="10">Lactobacillus</td><td colspan="1" rowspan="3">Lactobacillus casei</td><td colspan="1" rowspan="1">ATCC 25598</td><td colspan="1" rowspan="1">2</td></tr><tr><td colspan="1" rowspan="1">ATCC 334</td><td colspan="1" rowspan="1">6</td></tr><tr><td colspan="1" rowspan="1">ATCC 39392</td><td colspan="1" rowspan="1">4</td></tr><tr><td colspan="1" rowspan="3">Lactobacillus paracasei</td><td colspan="1" rowspan="1">148-260 *</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">ATCC 27092</td><td colspan="1" rowspan="1">2</td></tr><tr><td colspan="1" rowspan="1">ATCC BAA-52</td><td colspan="1" rowspan="1">6</td></tr><tr><td colspan="1" rowspan="3">Lactobacillus rhamnosus</td><td colspan="1" rowspan="1">ATCC 39595</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">ATCC 53103</td><td colspan="1" rowspan="1">5</td></tr><tr><td colspan="1" rowspan="1">ATCC 55915</td><td colspan="1" rowspan="1">2</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus total</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">33</td></tr><tr><td colspan="1" rowspan="15">Listeria</td><td colspan="1" rowspan="2">Listeria innocua</td><td colspan="1" rowspan="1">ATCC 33090</td><td colspan="1" rowspan="1">4</td></tr><tr><td colspan="1" rowspan="1">NCTC 11288</td><td colspan="1" rowspan="1">5</td></tr><tr><td colspan="1" rowspan="3">Listeria ivanovii</td><td colspan="1" rowspan="1">ATCC 19119</td><td colspan="1" rowspan="1">2</td></tr><tr><td colspan="1" rowspan="1">ATCC 700402</td><td colspan="1" rowspan="1">4</td></tr><tr><td colspan="1" rowspan="1">ATCC BAA-139</td><td colspan="1" rowspan="1">4</td></tr><tr><td colspan="1" rowspan="10">Listeria monocytogenes</td><td colspan="1" rowspan="1">ATCC 13932</td><td colspan="1" rowspan="1">5</td></tr><tr><td colspan="1" rowspan="1">ATCC 19111</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">ATCC 19112</td><td colspan="1" rowspan="1">4</td></tr><tr><td colspan="1" rowspan="1">ATCC 19114</td><td colspan="1" rowspan="1">5</td></tr><tr><td colspan="1" rowspan="1">ATCC 19116</td><td colspan="1" rowspan="1">5</td></tr><tr><td colspan="1" rowspan="1">ATCC 19117</td><td colspan="1" rowspan="1">5</td></tr><tr><td colspan="1" rowspan="1">ATCC 19118</td><td colspan="1" rowspan="1">5</td></tr><tr><td colspan="1" rowspan="1">ATCC 7644</td><td colspan="1" rowspan="1">5</td></tr><tr><td colspan="1" rowspan="1">ATCC BAA-751</td><td colspan="1" rowspan="1">5</td></tr><tr><td colspan="1" rowspan="1">NCTC 10890</td><td colspan="1" rowspan="1">4</td></tr><tr><td colspan="1" rowspan="1">Target</td><td colspan="1" rowspan="1">Organism</td><td colspan="1" rowspan="1">Strain</td><td colspan="1" rowspan="1">Independent ContrivedSamples Tested</td></tr><tr><td colspan="1" rowspan="3"></td><td colspan="1" rowspan="1">Listeria seeligeri</td><td colspan="1" rowspan="1">ATCC 35967</td><td colspan="1" rowspan="1">5</td></tr><tr><td colspan="1" rowspan="1">Listeria welshimeri</td><td colspan="1" rowspan="1">ATCC 35897</td><td colspan="1" rowspan="1">5</td></tr><tr><td colspan="1" rowspan="1">Listeria total</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">75</td></tr><tr><td colspan="1" rowspan="9">Micrococcus</td><td colspan="1" rowspan="6">Micrococcus luteus</td><td colspan="1" rowspan="1">ATCC 10240</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">ATCC 19212</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">ATCC 400</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">ATCC 4698</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">ATCC 49732</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">ATCC 53598</td><td colspan="1" rowspan="1">4</td></tr><tr><td colspan="1" rowspan="1">Micrococcus lylae</td><td colspan="1" rowspan="1">ATCC 27566</td><td colspan="1" rowspan="1">4</td></tr><tr><td colspan="1" rowspan="1">Micrococcus yunnanensis</td><td colspan="1" rowspan="1">ATCC 7468</td><td colspan="1" rowspan="1">4</td></tr><tr><td colspan="1" rowspan="1">Micrococcus total</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">27</td></tr><tr><td colspan="1" rowspan="5">Cutibacteriumacnes</td><td colspan="1" rowspan="4">Cutibacterium acnes</td><td colspan="1" rowspan="1">ATCC 11827</td><td colspan="1" rowspan="1">8</td></tr><tr><td colspan="1" rowspan="1">ATCC 11828</td><td colspan="1" rowspan="1">6</td></tr><tr><td colspan="1" rowspan="1">ATCC 33179</td><td colspan="1" rowspan="1">4</td></tr><tr><td colspan="1" rowspan="1">ATCC 6919</td><td colspan="1" rowspan="1">8</td></tr><tr><td colspan="1" rowspan="1">Cutibacterium acnes total</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">26</td></tr><tr><td colspan="1" rowspan="12">Staphylococcus</td><td colspan="1" rowspan="3">Staphylococcus aureus, mecA</td><td colspan="1" rowspan="1">ATCC 33591</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">ATCC BAA-44</td><td colspan="1" rowspan="1">5</td></tr><tr><td colspan="1" rowspan="1">NCTC 12493</td><td colspan="1" rowspan="1">2</td></tr><tr><td colspan="1" rowspan="2">Staphylococcus aureus, mecC</td><td colspan="1" rowspan="1">ATCC BAA-2312</td><td colspan="1" rowspan="1">23</td></tr><tr><td colspan="1" rowspan="1">ATCC BAA-2313</td><td colspan="1" rowspan="1">26</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus epidermidis, mecA</td><td colspan="1" rowspan="1">ATCC 35984</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="5">Staphylococcus lugdunensis</td><td colspan="1" rowspan="1">ATCC 49576</td><td colspan="1" rowspan="1">9</td></tr><tr><td colspan="1" rowspan="1">NRS 878</td><td colspan="1" rowspan="1">9</td></tr><tr><td colspan="1" rowspan="1">NRS 879</td><td colspan="1" rowspan="1">9</td></tr><tr><td colspan="1" rowspan="1">NRS 880</td><td colspan="1" rowspan="1">9</td></tr><tr><td colspan="1" rowspan="1">NRS 881</td><td colspan="1" rowspan="1">9</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus total</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">105</td></tr><tr><td colspan="1" rowspan="17">Streptococcus</td><td colspan="1" rowspan="4">Streptococcus agalactiae</td><td colspan="1" rowspan="1">ATCC 12403</td><td colspan="1" rowspan="1">2</td></tr><tr><td colspan="1" rowspan="1">ATCC 12973</td><td colspan="1" rowspan="1">2</td></tr><tr><td colspan="1" rowspan="1">ATCC 13813</td><td colspan="1" rowspan="1">2</td></tr><tr><td colspan="1" rowspan="1">ATCC 27956</td><td colspan="1" rowspan="1">2</td></tr><tr><td colspan="1" rowspan="3">Streptococcus anginosus</td><td colspan="1" rowspan="1">ATCC 700231</td><td colspan="1" rowspan="1">5</td></tr><tr><td colspan="1" rowspan="1">ATCC 9895</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">NCTC 10713</td><td colspan="1" rowspan="1">5</td></tr><tr><td colspan="1" rowspan="2">Streptococcus constellatus</td><td colspan="1" rowspan="1">ATCC 27513</td><td colspan="1" rowspan="1">4</td></tr><tr><td colspan="1" rowspan="1">ATCC 27823</td><td colspan="1" rowspan="1">2</td></tr><tr><td colspan="1" rowspan="1">Streptococcus intermedius</td><td colspan="1" rowspan="1">ATCC 27335</td><td colspan="1" rowspan="1">4</td></tr><tr><td colspan="1" rowspan="6">Streptococcus pyogenes</td><td colspan="1" rowspan="1">ATCC 12344</td><td colspan="1" rowspan="1">5</td></tr><tr><td colspan="1" rowspan="1">ATCC 12384</td><td colspan="1" rowspan="1">4</td></tr><tr><td colspan="1" rowspan="1">ATCC 14289</td><td colspan="1" rowspan="1">4</td></tr><tr><td colspan="1" rowspan="1">ATCC 19615</td><td colspan="1" rowspan="1">4</td></tr><tr><td colspan="1" rowspan="1">ATCC 49399</td><td colspan="1" rowspan="1">5</td></tr><tr><td colspan="1" rowspan="1">NCIMB 13285</td><td colspan="1" rowspan="1">4</td></tr><tr><td colspan="2" rowspan="1">Streptococcus total</td><td colspan="1" rowspan="1">57</td></tr></table>

\*Derived from clinical specimen

# Genus and Group Assay Species Stratification

The ePlex BCID-GP Panel reports genus or group level results for Bacillus cereus group, Bacillus subtilis group, Corynebacterium, Enterococcus, Lactobacillus, Listeria, Micrococcus, Staphylococcus, Streptococcus, Streptococcus anginosus group, Pan Gram-Negative and Pan Candida targets. Sensitivity/PPA of these genus and group level targets for species as determined by comparator methods for all evaluable samples tested are summarized in Table 32.

Table 32: Species Detected in Genus and Group Assays by Comparator Methods   

<table><tr><td rowspan="2">Target Species detected by Comparator Method</td><td colspan="2">Sensitivity/PPA (Prospective)</td><td colspan="2">Sensitivity/PPA (Retrospective)</td><td colspan="2">Sensitivity/PPA (Contrived)</td><td colspan="2">Sensitivity/PPA (Combined)</td></tr><tr><td>TP/ TP+FN</td><td>% (95% CI)</td><td>TP/ TP+FN</td><td>% (95% CI)</td><td>TP/ TP+FN</td><td>% (95% CI)</td><td>TP/ TP+FN</td><td>% (95% CI)</td></tr><tr><td>Bacillus cereus group</td><td>5/5</td><td>100 (56.6-100)</td><td>6/7</td><td>85.7 (48.7-97.4)</td><td>46/46</td><td>100 (92.3-100)</td><td>57/58</td><td>98.3 (90.9-99.7)</td></tr><tr><td>Bacillus cereus</td><td>3/3</td><td>100 (43.9-100)</td><td>6/7</td><td>85.7 (48.7-97.4)</td><td>40/40</td><td>100 (91.2-100)</td><td>49/50</td><td>98.0 (89.5-99.6)</td></tr><tr><td>Bacillus thuringiensis</td><td>2/2</td><td>100 (34.2-100)</td><td>-</td><td></td><td>6/6</td><td>100 (61.0-100)</td><td>8/8</td><td>100 (67.6-100)</td></tr><tr><td>Bacillus subtilis group</td><td>2/2</td><td>100 (34.2-100)</td><td>-</td><td>-</td><td>50/50</td><td>100 (92.9-100)</td><td>52/52</td><td>100 (93.1-100)</td></tr><tr><td>Bacillus amyloliquefaciens</td><td>1/1</td><td>100 (20.7-100)</td><td></td><td></td><td>10/10</td><td>100 (72.2-100)</td><td>11/11</td><td>100 (74.1-100)</td></tr><tr><td>Bacillus atrophaeus</td><td>-</td><td>-</td><td>-</td><td></td><td>11/11</td><td>100 (74.1-100)</td><td>11/11</td><td>100 (74.1-100)</td></tr><tr><td>Bacillus licheniformis</td><td>-</td><td>-</td><td>-</td><td></td><td>10/10</td><td>100 (72.2-100)</td><td>10/10</td><td>100 (72.2-100)</td></tr><tr><td>Bacillus subtilis</td><td>1/1</td><td>100 (20.7-100)</td><td>-</td><td>-</td><td>19/19</td><td>100 (83.2-100)</td><td>20/20</td><td>100 (83.9-100)</td></tr><tr><td>Corynebacterium</td><td>13/19</td><td>68.4 (46.0-84.6)</td><td>27/32</td><td>84.4 (68.2-93.1)</td><td>20/20</td><td>100 (83.9-100)</td><td>60/71</td><td>84.5 (74.3-91.1)</td></tr><tr><td>Corynebacterium</td><td>4/9</td><td>44.4 (18.9-73.3)</td><td>5/7</td><td>71.4 (35.9-91.8)</td><td>-</td><td></td><td>9/16</td><td>56.3 (33.2-76.9)</td></tr><tr><td>Corynebacterium afermentans</td><td>0/1</td><td>0.0 (0.0-79.3)</td><td>3/3</td><td>100 (43.9-100)</td><td>-</td><td></td><td>3/4</td><td>75.0 (30.1-95.4)</td></tr><tr><td>Corynebacterium amycolatum</td><td>1/1</td><td>100 (20.7-100)</td><td>-</td><td>-</td><td>-</td><td>-</td><td>1/1</td><td>100 (20.7-100)</td></tr><tr><td>Corynebacterium aurimucosum</td><td>1/1</td><td>100 (20.7-100)</td><td>1/1</td><td>100 (20.7-100)</td><td>-</td><td>-</td><td>2/2</td><td>100 (34.2-100)</td></tr><tr><td>Corynebacterium casei</td><td>1/1</td><td>100 (20.7-100)</td><td>-</td><td>-</td><td>-</td><td></td><td>1/1</td><td>100 (20.7-100)</td></tr><tr><td>Corynebacterium coyleae</td><td>1/1</td><td>100 (20.7-100)</td><td>2/2</td><td>100 (34.2-100)</td><td>7/7</td><td>100 (64.6-100)</td><td>10/10</td><td>100 (72.2-100)</td></tr><tr><td>Corynebacterium falsenii</td><td>-</td><td>-</td><td>-</td><td>-</td><td>9/9</td><td>100 (70.1-100)</td><td>9/9</td><td>100 (70.1-100)</td></tr><tr><td>Corynebacterium imitans</td><td>2/2</td><td>100 (34.2-100)</td><td>2/2</td><td>100 (34.2-100)</td><td>-</td><td></td><td>4/4</td><td>100 (51.0-100)</td></tr><tr><td>Corynebacterium jeikeium</td><td>-</td><td></td><td>4/5</td><td>80.0 (37.6-96.4)</td><td></td><td></td><td>4/5</td><td>80.0 (37.6-96.4)</td></tr><tr><td>Corynebacterium kroppenstedtii</td><td>-</td><td>-</td><td>1/1</td><td>100 (20.7-100)</td><td>-</td><td>-</td><td>1/1</td><td>100 (20.7-100)</td></tr><tr><td>Corynebacterium matruchotii</td><td>-</td><td>-</td><td>1/1</td><td>100 (20.7-100)</td><td>-</td><td></td><td>1/1</td><td>100 (20.7-100)</td></tr><tr><td>Corynebacterium mucifaciens</td><td>1/1</td><td>100 (20.7-100)</td><td>2/2</td><td>100 (34.2-100)</td><td>-</td><td></td><td>3/3</td><td>100 (43.9-100)</td></tr><tr><td>Corynebacterium pseudotuberculosis</td><td>-</td><td></td><td>0/1</td><td>0.0 (0.0-79.3)</td><td>-</td><td></td><td>0/1</td><td>0.0 (0.0-79.3)</td></tr><tr><td>Corynebacterium striatum</td><td>1/1</td><td>100 (20.7-100)</td><td>6/6</td><td>100 (61.0-100)</td><td>4/4</td><td>100 (51.0-100)</td><td>11/11</td><td>100 (74.1-100)</td></tr><tr><td>Corynebacterium tuberculostearicum</td><td>1/1</td><td>100 (20.7-100)</td><td>-</td><td>-</td><td>-</td><td></td><td>1/1</td><td>100 (20.7-100)</td></tr><tr><td>Corynebacterium urealyticum</td><td>-</td><td>-</td><td>0/1</td><td>0.0 (0.0-79.3)</td><td>-</td><td>-</td><td>0/1</td><td>0.0 (0.0-79.3)</td></tr><tr><td rowspan="3">Target Species detected by Comparator Method</td><td colspan="2">Sensitivity/PPA (Prospective)</td><td colspan="2">Sensitivity/PPA (Retrospective)</td><td colspan="2">Sensitivity/PPA (Contrived)</td><td colspan="2">Sensitivity/PPA (Combined)</td></tr><tr><td>TP/ TP+FN</td><td>% (95% CI)</td><td>TP/ TP+FN</td><td>% (95% CI)</td><td>TP/ TP+FN</td><td>% (95% CI)</td><td>TP/ TP+FN</td><td>% (95% CI)</td></tr><tr><td>Enterococcus</td><td>61/61</td><td>100 (94.1-100)</td><td>139/147</td><td>94.6 (89.6-97.2)</td><td>126/126</td><td>100 (97.0-100)</td><td>326/334</td><td>97.6 (95.3-98.8)</td></tr><tr><td>Enterococcus</td><td>1/1</td><td>100 (20.7-100)</td><td>-</td><td>-</td><td>-</td><td>-</td><td>1/1</td><td>100 (20.7-100)</td></tr><tr><td>Enterococcus avium</td><td>1/1</td><td>100 (20.7-100)</td><td>2/3</td><td>66.7 (20.8-93.9)</td><td>-</td><td></td><td>3/4</td><td>75.0 (30.1-95.4)</td></tr><tr><td>Enterococcus casseliflavus</td><td>-</td><td>-</td><td>0/1</td><td>0.0 (0.0-79.3)</td><td>-</td><td></td><td>0/1</td><td>0.0 (0.0-79.3)</td></tr><tr><td>Enterococcus casseliflavus /</td><td>-</td><td>-</td><td>1/1</td><td>100 (20.7-100)</td><td>-</td><td>-</td><td>1/1</td><td>100 (20.7-100)</td></tr><tr><td>gallinarum Enterococcus faecalis</td><td>49/49</td><td>100 (92.7-100)</td><td>85/90</td><td>94.4 (87.6-97.6)</td><td>52/52</td><td>100 (93.1-100)</td><td>186/191</td><td>97.4 (94.0-98.9)</td></tr><tr><td>Enterococcus faecium</td><td>12/12</td><td>100 (75.8-100)</td><td>52/53</td><td>98.1 (90.1-99.7)</td><td>60/60</td><td>100 (94.0-100)</td><td>124/125</td><td>99.2 (95.6-99.9)</td></tr><tr><td>Enterococcus flavescens</td><td>-</td><td></td><td>-</td><td>-</td><td>3/3</td><td>100 (43.9-100)</td><td>3/3</td><td>100 (43.9-100)</td></tr><tr><td>Enterococcus gallinarum</td><td></td><td>-</td><td>2/2</td><td>100 (34.2-100)</td><td>4/4</td><td>100 (51.0-100)</td><td>6/6</td><td>100 (61.0-100)</td></tr><tr><td>Enterococcus hirae</td><td>-</td><td>-</td><td>-</td><td></td><td>1/1</td><td>100 (20.7-100)</td><td>1/1</td><td>100 (20.7-100)</td></tr><tr><td>Enterococcus malodoratus</td><td>-</td><td>-</td><td>-</td><td></td><td>3/3</td><td>100 (43.9-100)</td><td>3/3</td><td>100 (43.9-100)</td></tr><tr><td>Enterococcus raffinosus</td><td>-</td><td></td><td>-</td><td></td><td>2/2</td><td>100 (34.2-100)</td><td>2/2</td><td>100 (34.2-100)</td></tr><tr><td>Enterococcus saccharolyticus</td><td>-</td><td></td><td>-</td><td></td><td>1/1</td><td>100 (20.7-100)</td><td>1/1</td><td>100 (20.7-100)</td></tr><tr><td>Lactobacillus</td><td>4/4</td><td>100 (51.0-100)</td><td>9/9</td><td>100 (70.1-100)</td><td>32/33</td><td>97.0 (84.7-99.5)</td><td>45/46</td><td>97.8 (88.7-99.6)</td></tr><tr><td>Lactobacillus casei</td><td>-</td><td>-</td><td>1/1</td><td>100 (20.7-100)</td><td>12/12</td><td>100 (75.8-100)</td><td>13/13</td><td>100 (77.2-100)</td></tr><tr><td>Lactobacillus paracasei</td><td>1/1</td><td>100 (20.7-100)</td><td>-</td><td>-</td><td>11/11</td><td>100 (74.1-100)</td><td>12/12</td><td>100 (75.8-100)</td></tr><tr><td>Lactobacillus rhamnosus</td><td>2/2</td><td>100 (34.2-100)</td><td>8/8</td><td>100 (67.6-100)</td><td>9/10</td><td>90.0 (59.6-98.2)</td><td>19/20</td><td>95.0 (76.4-99.1)</td></tr><tr><td>Lactobacillus zeae</td><td>1/1</td><td>100 (20.7-100)</td><td>-</td><td>-</td><td>-</td><td>-</td><td>1/1</td><td>100 (20.7-100)</td></tr><tr><td>Listeria</td><td>-</td><td>-</td><td>2/2</td><td>100 (34.2-100)</td><td>74/75</td><td>98.7 (92.8-99.8)</td><td>76/77</td><td>98.7 (93.0-99.8)</td></tr><tr><td>Listeria innocua</td><td>-</td><td>-</td><td>-</td><td>-</td><td>9/9</td><td>100 (70.1-100)</td><td>9/9</td><td>100 (70.1-100)</td></tr><tr><td>Listeria ivanovii</td><td>-</td><td>-</td><td>-</td><td>-</td><td>9/10</td><td>90.0 (59.6-98.2)</td><td>9/10</td><td>90.0 (59.6-98.2)</td></tr><tr><td>Listeria monocytogenes</td><td></td><td></td><td>2/2</td><td>100 (34.2-100)</td><td>46/46</td><td>100 (92.3-100)</td><td>48/48</td><td>100 (92.6-100)</td></tr><tr><td>Listeria seeligeri</td><td>-</td><td>-</td><td>-</td><td>-</td><td>5/5</td><td>100 (56.6-100)</td><td>5/5</td><td>100 (56.6-100)</td></tr><tr><td>Listeria welshimeri</td><td>-</td><td>-</td><td>-</td><td>-</td><td>5/5</td><td>100 (56.6-100)</td><td>5/5</td><td>100 (56.6-100)</td></tr><tr><td>Micrococcus</td><td>19/21</td><td>90.5 (71.1-97.3)</td><td>20/23</td><td>87.0 (67.9-95.5)</td><td>27/27</td><td>100 (87.5-100)</td><td>66/71</td><td>93.0 (84.6-97.0)</td></tr><tr><td>Micrococcus</td><td>8/9</td><td>88.9 (56.5-98.0)</td><td>10/13</td><td>76.9 (49.7-91.8)</td><td>-</td><td>-</td><td>18/22</td><td>81.8 (61.5-92.7)</td></tr><tr><td>Micrococcus luteus</td><td>9/9</td><td>100 (70.1-100)</td><td>8/8</td><td>100 (67.6-100)</td><td>19/19</td><td>100 (83.2-100)</td><td>36/36</td><td>100 (90.4-100)</td></tr><tr><td>Micrococcus luteus/lylae</td><td>2/3</td><td>66.7 (20.8-93.9)</td><td>2/2</td><td>100 (34.2-100)</td><td>-</td><td>-</td><td>4/5</td><td>80.0 (37.6-96.4)</td></tr><tr><td>Micrococcus lylae</td><td>-</td><td>-</td><td>-</td><td></td><td>4/4</td><td>100 (51.0-100)</td><td>4/4</td><td>100 (51.0-100)</td></tr><tr><td>Micrococcus yunnanensis</td><td>-</td><td>-</td><td>-</td><td>-</td><td>4/4</td><td>100 (51.0-100)</td><td>4/4</td><td>100 (51.0-100)</td></tr><tr><td colspan="1" rowspan="2">TargetSpecies detected byComparator Method</td><td colspan="2" rowspan="1">Sensitivity/PPA(Prospective)</td><td colspan="2" rowspan="1">Sensitivity/PPA(Retrospective)</td><td colspan="2" rowspan="1">Sensitivity/PPA(Contrived)</td><td colspan="2" rowspan="1">Sensitivity/PPA(Combined)</td></tr><tr><td colspan="1" rowspan="1">TP/TP+FN</td><td colspan="1" rowspan="1">% (95% CI)</td><td colspan="1" rowspan="1">TP/TP+FN</td><td colspan="1" rowspan="1">% (95% CI)</td><td colspan="1" rowspan="1">TP/TP+FN</td><td colspan="1" rowspan="1">% (95% CI)</td><td colspan="1" rowspan="1">TP/TP+FN</td><td colspan="1" rowspan="1">% (95% CI)</td></tr><tr><td colspan="1" rowspan="5">StaphylococcusCoagulase-negativestaphylococci (CoNS)CoNS (Not S. epidermidis, S.lugdunensis)StaphylococcusStaphylococcus aureusStaphylococcus aureus subsp.aureusStaphylococcus auricularisStaphylococcus capitis</td><td colspan="1" rowspan="2">447/45618/182/274/78158/160</td><td colspan="1" rowspan="2">98.0 (96.3-99.0)100 (82.4-100)100 (34.2-100)94.9 (87.5-98.0)98.8 (95.6-99.7)</td><td colspan="1" rowspan="1">185/191--1/3</td><td colspan="1" rowspan="1">96.9 (93.3-98.6)33.3 (6.1-79.2)</td><td colspan="1" rowspan="1">105/105---</td><td colspan="1" rowspan="1">100 (96.5-100)</td><td colspan="1" rowspan="1">737/75218/182/275/81</td><td colspan="1" rowspan="1">98.0 (96.7-98.8)100 (82.4-100)100 (34.2-100)92.6 (84.8-96.6)</td></tr><tr><td colspan="1" rowspan="1">121/123</td><td colspan="1" rowspan="1">98.4 (94.3-99.6)</td><td colspan="1" rowspan="1">59/59</td><td colspan="1" rowspan="1">100 (93.9-100)</td><td colspan="1" rowspan="1">338/342</td><td colspan="1" rowspan="1">98.8 (97.0-99.5)</td></tr><tr><td colspan="1" rowspan="2">6/62/2</td><td colspan="1" rowspan="2">100 (61.0-100)100 (34.2-100)</td><td colspan="1" rowspan="1">2/2</td><td colspan="1" rowspan="1">100 (34.2-100)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="1">8/8</td><td colspan="1" rowspan="1">100 (67.6-100)</td></tr><tr><td colspan="1" rowspan="1">2/2</td><td colspan="1" rowspan="1">100 (34.2-100)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">100 (51.0-100)</td></tr><tr><td colspan="1" rowspan="1">14/14</td><td colspan="1" rowspan="1">100 (78.5-100)</td><td colspan="1" rowspan="1">7/7</td><td colspan="1" rowspan="1">100 (64.6-100)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">21/21</td><td colspan="1" rowspan="1">100 (84.5-100)</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus carnosussubsp. carnosus</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">0/1</td><td colspan="1" rowspan="1">0.0 (0.0-79.3)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">0/1</td><td colspan="1" rowspan="1">0.0 (0.0-79.3)</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus cohnii</td><td colspan="1" rowspan="1">1/2</td><td colspan="1" rowspan="1">50.0 (9.5-90.5)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1/2</td><td colspan="1" rowspan="1">50.0 (9.5-90.5)</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus epidermidis</td><td colspan="1" rowspan="1">117/121</td><td colspan="1" rowspan="1">96.7 (91.8-98.7)</td><td colspan="1" rowspan="1">37/38</td><td colspan="1" rowspan="1">97.4 (86.5-99.5)</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100 (20.7-100)</td><td colspan="1" rowspan="1">155/160</td><td colspan="1" rowspan="1">96.9 (92.9-98.7)</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus haemolyticus</td><td colspan="1" rowspan="1">6/6</td><td colspan="1" rowspan="1">100 (61.0-100)</td><td colspan="1" rowspan="1">2/2</td><td colspan="1" rowspan="1">100 (34.2-100)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">8/8</td><td colspan="1" rowspan="1">100 (67.6-100)</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus hominis</td><td colspan="1" rowspan="1">24/24</td><td colspan="1" rowspan="1">100 (86.2-100)</td><td colspan="1" rowspan="1">13/13</td><td colspan="1" rowspan="1">100 (77.2-100)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">37/37</td><td colspan="1" rowspan="1">100 (90.6-100)</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus hominis subsp.hominis</td><td colspan="1" rowspan="1">22/22</td><td colspan="1" rowspan="1">100 (85.1-100)</td><td colspan="1" rowspan="1">5/5</td><td colspan="1" rowspan="1">100 (56.6-100)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">27/27</td><td colspan="1" rowspan="1">100 (87.5-100)</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus hominis subsp.novobiosepticus</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100 (20.7-100)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100 (20.7-100)</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus lugdunensis</td><td colspan="1" rowspan="1">2/2</td><td colspan="1" rowspan="1">100 (34.2-100)</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">100 (51.0-100)</td><td colspan="1" rowspan="1">45/45</td><td colspan="1" rowspan="1">100 (92.1-100)</td><td colspan="1" rowspan="1">51/51</td><td colspan="1" rowspan="1">100 (93.0-100)</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus pettenkoferi</td><td colspan="1" rowspan="1">2/2</td><td colspan="1" rowspan="1">100 (34.2-100)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2/2</td><td colspan="1" rowspan="1">100 (34.2-100)</td></tr><tr><td colspan="1" rowspan="1">Staphylococcussaccharolyticus</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100 (20.7-100)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100 (20.7-100)</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus saprophyticus</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100 (20.7-100)</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100 (20.7-100)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2/2</td><td colspan="1" rowspan="1">100 (34.2-100)</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus schleiferi</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100 (20.7-100)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100 (20.7-100)</td></tr><tr><td colspan="1" rowspan="2">Staphylococcus simulansStaphylococcus warneri</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">100 (43.9-100)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">100 (43.9-100)</td></tr><tr><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">100 (51.0-100)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">100 (51.0-100)</td></tr><tr><td rowspan="2">Target Species detected by Comparator Method</td><td colspan="2">Sensitivity/PPA (Prospective)</td><td colspan="2">Sensitivity/PPA (Retrospective)</td><td colspan="2">Sensitivity/PPA (Contrived)</td><td colspan="2">Sensitivity/PPA (Combined)</td></tr><tr><td>TP/ TP+FN</td><td>% (95% CI)</td><td>TP/ TP+FN</td><td>% (95% CI)</td><td>TP/ TP+FN</td><td>% (95% CI)</td><td>TP/ TP+FN</td><td>% (95% CI)</td></tr><tr><td>Streptococcus</td><td>103/110</td><td>93.6 (87.4-96.9)</td><td>171/173</td><td>98.8 (95.9-99.7)</td><td>57/57</td><td>100 (93.7-100)</td><td>331/340</td><td>97.4 (95.0-98.6)</td></tr><tr><td>Alpha Hemolytic Streptococcus</td><td>1/1</td><td>100 (20.7-100)</td><td>-</td><td></td><td>-</td><td></td><td>1/1</td><td>100 (20.7-100)</td></tr><tr><td>Gamma Hemolytic</td><td>1/1</td><td>100 (20.7-100)</td><td>-</td><td></td><td>-</td><td></td><td>1/1</td><td>100 (20.7-100)</td></tr><tr><td>Streptococcus Streptococcus</td><td>5/7</td><td>71.4 (35.9-91.8)</td><td>-</td><td></td><td>-</td><td></td><td>5/7</td><td>71.4 (35.9-91.8)</td></tr><tr><td>Streptococcus (Group G)</td><td>1/1</td><td>100 (20.7-100)</td><td>-</td><td>-</td><td>-</td><td>-</td><td>1/1</td><td>100 (20.7-100)</td></tr><tr><td>Streptococcus agalactiae</td><td>10/11</td><td>90.9 (62.3-98.4)</td><td>37/37</td><td>100 (90.6-100)</td><td>8/8</td><td>100 (67.6-100)</td><td>55/56</td><td>98.2 (90.6-99.7)</td></tr><tr><td>Streptococcus anginosus</td><td>1/1</td><td>100 (20.7-100)</td><td>13/13</td><td>100 (77.2-100)</td><td>13/13</td><td>100 (77.2-100)</td><td>27/27</td><td>100 (87.5-100)</td></tr><tr><td>Streptococcus anginosus</td><td>4/4</td><td>100 (51.0-100)</td><td>22/22</td><td>100 (85.1-100)</td><td>-</td><td>-</td><td>26/26</td><td>100 (87.1-100)</td></tr><tr><td>group Streptococcus bovis</td><td>-</td><td>-</td><td>2/2</td><td>100 (34.2-100)</td><td>-</td><td></td><td>2/2</td><td>100 (34.2-100)</td></tr><tr><td>Streptococcus bovis group</td><td>1/1</td><td>100 (20.7-100)</td><td>-</td><td>-</td><td>-</td><td></td><td>1/1</td><td>100 (20.7-100)</td></tr><tr><td>Streptococcus constellatus</td><td>-</td><td>-</td><td>-</td><td>-</td><td>6/6</td><td>100 (61.0-100)</td><td>6/6</td><td>100 (61.0-100)</td></tr><tr><td>Streptococcus constellatus subsp. constellatus</td><td>-</td><td></td><td>2/2</td><td>100 (34.2-100)</td><td>-</td><td></td><td>2/2</td><td>100 (34.2-100)</td></tr><tr><td>Streptococcus constellatus subsp. pharyngis</td><td>-</td><td></td><td>1/1</td><td>100 (20.7-100)</td><td>-</td><td></td><td>1/1</td><td>100 (20.7-100)</td></tr><tr><td>Streptococcus dysgalactiae</td><td>-</td><td></td><td>2/2</td><td>100 (34.2-100)</td><td>-</td><td></td><td>2/2</td><td>100 (34.2-100)</td></tr><tr><td>Streptococcus dysgalactiae (Group G)</td><td>4/4</td><td>100 (51.0-100)</td><td>1/1</td><td>100 (20.7-100)</td><td>-</td><td></td><td>5/5</td><td>100 (56.6-100)</td></tr><tr><td>Streptococcus gallolyticus</td><td>1/1</td><td>100 (20.7-100)</td><td>-</td><td>-</td><td>-</td><td></td><td>1/1</td><td>100 (20.7-100)</td></tr><tr><td>Streptococcus gordonii</td><td>1/1</td><td>100 (20.7-100)</td><td>-</td><td>-</td><td>-</td><td></td><td>1/1</td><td>100 (20.7-100)</td></tr><tr><td>Streptococcus infantarius</td><td>-</td><td></td><td>1/1</td><td>100 (20.7-100)</td><td>-</td><td></td><td>1/1</td><td>100 (20.7-100)</td></tr><tr><td>Streptococcus intermedius</td><td>-</td><td></td><td>2/2</td><td>100 (34.2-100)</td><td>4/4</td><td>100 (51.0-100)</td><td>6/6</td><td>100 (61.0-100)</td></tr><tr><td>Streptococcus mitis</td><td>9/10</td><td>90.0 (59.6-98.2)</td><td>14/15</td><td>93.3 (70.2-98.8)</td><td>-</td><td></td><td>23/25</td><td>92.0 (75.0-97.8)</td></tr><tr><td>Streptococcus mitis group</td><td>10/10</td><td>100 (72.2-100)</td><td>-</td><td>-</td><td>-</td><td></td><td>10/10</td><td>100 (72.2-100)</td></tr><tr><td>Streptococcus mutans</td><td>-</td><td>-</td><td>1/1</td><td>100 (20.7-100)</td><td>-</td><td></td><td>1/1</td><td>100 (20.7-100)</td></tr><tr><td>Streptococcus oralis</td><td>-</td><td></td><td>3/3</td><td>100 (43.9-100)</td><td>-</td><td></td><td>3/3</td><td>100 (43.9-100)</td></tr><tr><td>Streptococcus parasanguinis</td><td>2/2</td><td>100 (34.2-100)</td><td>4/4</td><td>100 (51.0-100)</td><td>-</td><td></td><td>6/6</td><td>100 (61.0-100)</td></tr><tr><td>Streptococcus pneumoniae</td><td>28/28</td><td>100 (87.9-100)</td><td>41/41</td><td>100 (91.4-100)</td><td>-</td><td></td><td>69/69</td><td>100 (94.7-100)</td></tr><tr><td>Streptococcus pyogenes</td><td>8/8</td><td>100 (67.6-100)</td><td>19/20</td><td>95.0 (76.4-99.1)</td><td>26/26</td><td>100 (87.1-100)</td><td>53/54</td><td>98.1 (90.2-99.7)</td></tr><tr><td>Streptococcus salivarius</td><td>4/4</td><td>100 (51.0-100)</td><td>5/5</td><td>100 (56.6-100)</td><td>-</td><td>-</td><td>9/9</td><td>100 (70.1-100)</td></tr><tr><td>Streptococcus vestibularis</td><td>-</td><td></td><td>1/1</td><td>100 (20.7-100)</td><td>-</td><td></td><td>1/1</td><td>100 (20.7-100)</td></tr><tr><td>Streptococcus viridans group</td><td>14/17</td><td>82.4 (59.0-93.8)</td><td>2/2</td><td>100 (34.2-100)</td><td>-</td><td>-</td><td>16/19</td><td>84.2 (62.4-94.5)</td></tr><tr><td>Streptococcus anginosus group</td><td>4/5</td><td>80.0 (37.6-96.4)</td><td>38/40</td><td>95.0 (83.5-98.6)</td><td>23/23</td><td>100 (85.7-100)</td><td>65/68</td><td>95.6 (87.8-98.5)</td></tr><tr><td>Streptococcus anginosus</td><td>0/1</td><td>0.0 (0.0-79.3)</td><td>12/13</td><td>92.3 (66.7-98.6)</td><td>13/13</td><td>100 (77.2-100)</td><td>25/27</td><td>92.6 (76.6-97.9)</td></tr><tr><td>Streptococcus anginosus group</td><td>4/4</td><td>100 (51.0-100)</td><td>21/22</td><td>95.5 (78.2-99.2)</td><td>-</td><td></td><td>25/26</td><td>96.2 (81.1-99.3)</td></tr><tr><td>Streptococcus constellatus</td><td>-</td><td></td><td>-</td><td></td><td>6/6</td><td>100 (61.0-100)</td><td>6/6</td><td>100 (61.0-100)</td></tr><tr><td>Streptococcus constellatus spp constellatus</td><td>-</td><td></td><td>2/2</td><td>100 (34.2-100)</td><td>-</td><td>-</td><td>2/2</td><td>100 (34.2-100)</td></tr><tr><td>Streptococcus constellatus spp pharynges</td><td>-</td><td></td><td>1/1</td><td>100 (20.7-100)</td><td>-</td><td></td><td>1/1</td><td>100 (20.7-100)</td></tr><tr><td>Streptococcus intermedius</td><td></td><td></td><td>2/2</td><td>100 (34.2-100)</td><td>4/4</td><td>100 (51.0-100)</td><td>6/6</td><td>100 (61.0-100)</td></tr></table>

<table><tr><td rowspan=2 colspan=4>TargetSpecies detected byComparator Method</td><td rowspan=1 colspan=2>Sensitivity/PPA(Prospective)</td><td rowspan=1 colspan=2>Sensitivity/PPA(Retrospective)</td><td rowspan=1 colspan=3>Sensitivity/PPA(Contrived)</td><td rowspan=1 colspan=2>Sensitivity/PPA(Combined)</td></tr><tr><td rowspan=1 colspan=1>TP/TP+FN</td><td rowspan=1 colspan=1>% (95% CI)</td><td rowspan=1 colspan=1>TP/TP+FN</td><td rowspan=1 colspan=1>% (95% CI)</td><td rowspan=1 colspan=1>TP/TP+FN</td><td rowspan=1 colspan=2>% (95% CI)</td><td rowspan=1 colspan=1>TP/TP+FN</td><td rowspan=1 colspan=1>% (95% CI)</td></tr><tr><td rowspan=2 colspan=4>Pan CandidaCandida albicansCandida glabrataCandida kruseiCandida parapsilosis</td><td rowspan=2 colspan=1>----</td><td rowspan=2 colspan=1>-</td><td rowspan=2 colspan=1>7/94/41/21/11/2</td><td rowspan=2 colspan=1>77.8 (45.3-93.7)100 (51.0-100)50 (9.5-90.5)100 (20.7-100)50 (9.5-90.5)</td><td rowspan=2 colspan=1>----</td><td rowspan=2 colspan=2></td><td rowspan=2 colspan=1>7/94/41/21/11/2</td><td rowspan=1 colspan=1>77.8 (45.3-93.7)</td></tr><tr><td rowspan=1 colspan=1>100 (51.0-100)50 (9.5-90.5)100 (20.7-100)50 (9.5-90.5)</td></tr><tr><td rowspan=4 colspan=4>Pan Gram-NegativeAcinetobacter baumanniiAcinetobacter lwoffiiAeromonas caviae</td><td rowspan=1 colspan=1>22/23</td><td rowspan=1 colspan=1>95.7 (79.0-99.2)</td><td rowspan=1 colspan=1>36/43</td><td rowspan=1 colspan=1>83.7 (70.0-91.9)</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=2></td><td rowspan=1 colspan=1>58/66</td><td rowspan=1 colspan=1>87.9 (77.9-93.7)</td></tr><tr><td rowspan=3 colspan=1>3/31/1-</td><td rowspan=3 colspan=1>100 (43.9-100)100 (20.7-100)</td><td rowspan=3 colspan=1>2/4-1/1</td><td rowspan=3 colspan=1>50.0 (15.0-85.0)-100 (20.7-100)</td><td rowspan=1 colspan=1>-</td><td rowspan=2 colspan=2></td><td rowspan=1 colspan=1>5/7</td><td rowspan=1 colspan=1>71.4 (35.9-91.8)</td></tr><tr><td rowspan=1 colspan=1>-</td><td rowspan=2 colspan=2></td><td rowspan=2 colspan=1>1/11/1</td><td rowspan=1 colspan=1>100 (20.7-100)</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>100 (20.7-100)</td></tr><tr><td rowspan=2 colspan=4>Bacteroides fragilisCampylobacter gracilis</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>100 (34.2-100)</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2></td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>100 (34.2-100)</td></tr><tr><td rowspan=1 colspan=1>0/1</td><td rowspan=1 colspan=1>0.0 (0.0-79.3)</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=2></td><td rowspan=1 colspan=1>0/1</td><td rowspan=1 colspan=1>0.0 (0.0-79.3)</td></tr><tr><td rowspan=1 colspan=2>Citrobacter braakii</td><td rowspan=1 colspan=2>Citrobacter braakii</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>1/1</td><td rowspan=1 colspan=1>100 (20.7-100)</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1/1</td><td rowspan=1 colspan=1>100 (20.7-100)</td></tr><tr><td rowspan=1 colspan=3>Citrobacter freundii</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1/1</td><td rowspan=1 colspan=1>100 (20.7-100)</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=2></td><td rowspan=1 colspan=1>1/1</td><td rowspan=1 colspan=1>100 (20.7-100)</td></tr><tr><td rowspan=1 colspan=4>Citrobacter koseri</td><td rowspan=1 colspan=1>1/1</td><td rowspan=1 colspan=1>100 (20.7-100)</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=2></td><td rowspan=1 colspan=1>1/1</td><td rowspan=1 colspan=1>100 (20.7-100)</td></tr><tr><td rowspan=1 colspan=4>Enterobacter aerogenes</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1/1</td><td rowspan=1 colspan=1>100 (20.7-100)</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1/1</td><td rowspan=1 colspan=1>100 (20.7-100)</td></tr><tr><td rowspan=1 colspan=4>Enterobacter cloacae</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>4/4</td><td rowspan=1 colspan=1>100 (51.0-100)</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>4/4</td><td rowspan=1 colspan=1>100 (51.0-100)</td></tr><tr><td rowspan=1 colspan=4>Escherichia coli</td><td rowspan=1 colspan=1>4/4</td><td rowspan=1 colspan=1>100 (51.0-100)</td><td rowspan=1 colspan=1>14/14</td><td rowspan=1 colspan=1>100 (78.5-100)</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>18/18</td><td rowspan=1 colspan=1>100 (82.4-100)</td></tr><tr><td rowspan=1 colspan=4>Klebsiella oxytoca</td><td rowspan=1 colspan=1>1/1</td><td rowspan=1 colspan=1>100 (20.7-100)</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>100 (43.9-100)</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>4/4</td><td rowspan=1 colspan=1>100 (51.0-100)</td></tr><tr><td rowspan=1 colspan=4>Klebsiella pneumoniae</td><td rowspan=1 colspan=1>4/4</td><td rowspan=1 colspan=1>100 (51.0-100)</td><td rowspan=1 colspan=1>4/5</td><td rowspan=1 colspan=1>80.0 (37.6-96.4)</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>8/9</td><td rowspan=1 colspan=1>88.9 (56.5-98.0)</td></tr><tr><td rowspan=1 colspan=4>Moraxella (Branhamella)catarrhalis</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1/1</td><td rowspan=1 colspan=1>100 (20.7-100)</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1/1</td><td rowspan=1 colspan=1>100 (20.7-100)</td></tr><tr><td rowspan=1 colspan=4>Moraxella catarrhalis</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1/1</td><td rowspan=1 colspan=1>100 (20.7-100)</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1/1</td><td rowspan=1 colspan=1>100 (20.7-100)</td></tr><tr><td rowspan=1 colspan=4>Morganella morganii</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>100 (34.2-100)</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>100 (34.2-100)</td></tr><tr><td rowspan=1 colspan=4>Pediococcus pentosaceus</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1/1</td><td rowspan=1 colspan=1>100 (20.7-100)</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1/1</td><td rowspan=1 colspan=1>100 (20.7-100)</td></tr><tr><td rowspan=1 colspan=4>Proteus mirabilis</td><td rowspan=1 colspan=1>5/5</td><td rowspan=1 colspan=1>100 (56.6-100)</td><td rowspan=1 colspan=1>4/5</td><td rowspan=1 colspan=1>80.0 (37.6-96.4)</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>9/10</td><td rowspan=1 colspan=1>90.0 (59.6-98.2)</td></tr><tr><td rowspan=1 colspan=4>Proteus vulgaris</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>1/1</td><td rowspan=1 colspan=1>100 (20.7-100)</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1/1</td><td rowspan=1 colspan=1>100 (20.7-100)</td></tr><tr><td rowspan=1 colspan=4>Providencia stuartii</td><td rowspan=1 colspan=1>1/1</td><td rowspan=1 colspan=1>100 (20.7-100)</td><td rowspan=1 colspan=1>0/1</td><td rowspan=1 colspan=1>0.0 (0.0-79.3)</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1/2</td><td rowspan=1 colspan=1>50.0 (9.5-90.5)</td></tr><tr><td rowspan=1 colspan=4>Pseudomonas</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>1/1</td><td rowspan=1 colspan=1>100 (20.7-100)</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1/1</td><td rowspan=1 colspan=1>100 (20.7-100)</td></tr><tr><td rowspan=1 colspan=4>Pseudomonas aeruginosa</td><td rowspan=1 colspan=1>1/1</td><td rowspan=1 colspan=1>100 (20.7-100)</td><td rowspan=1 colspan=1>1/2</td><td rowspan=1 colspan=1>50.0 (9.5-90.5)</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>2/3</td><td rowspan=1 colspan=1>66.7 (20.8-93.9)</td></tr><tr><td rowspan=1 colspan=4>Serratia marcescens</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>100 (34.2-100)</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>100 (34.2-100)</td></tr><tr><td rowspan=2 colspan=4>Stenotrophomonas maltophiliaVeillonella species</td><td rowspan=1 colspan=1>1/1</td><td rowspan=1 colspan=1>100 (20.7-100)</td><td rowspan=1 colspan=1>1/1</td><td rowspan=1 colspan=1>100 (20.7-100)</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>100 (34.2-100)</td></tr><tr><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>0/1</td><td rowspan=1 colspan=1>0.0 (0.0-79.3)</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>0/1</td><td rowspan=1 colspan=1>0.0 (0.0-79.3)</td></tr><tr><td rowspan=1 colspan=4>Non-fermenting gram-negative bacilli</td><td rowspan=1 colspan=1>1/1</td><td rowspan=1 colspan=1>100 (20.7-100)</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=2>-</td><td rowspan=1 colspan=1>1/1</td><td rowspan=1 colspan=1>100 (20.7-100)</td></tr></table>

# Resistance Gene Assay Species Stratification

Test results for resistance genes are only reported when an associated organism assay is positive in the same sample. See Table 33 for organisms specifically associated with the four resistance markers on the ePlex BCID-GP Panel.

Table 33: Resistance Marker Organism Associations   

<table><tr><td rowspan=1 colspan=1>Resistance GeneResult</td><td rowspan=1 colspan=1>Associated Targets</td></tr><tr><td rowspan=1 colspan=1>mecA and/or mecC</td><td rowspan=1 colspan=1>Any Staphylococcus assay (Staphylococcus, S. aureus, S. epidermidis, S. lugdunensis)</td></tr><tr><td rowspan=1 colspan=1>vanA and/or vanB</td><td rowspan=1 colspan=1>Any Enterococcus assay (Enterococcus, E. faecalis, E. faecium)</td></tr></table>

# mecA/mecC

The PPA and NPA of the BCID-GP Panel mecA target stratified by the Staphylococcus species identified by comparator methods for prospective, retrospective and contrived samples are shown in Table 34.

Table 34: Clinical Performance of mecA Target by Staphylococcus Species Detected by Comparator Methods   
Confidential   

<table><tr><td colspan="2" rowspan="2">Species detected by Comparator Method</td><td colspan="2" rowspan="1">Sensitivity/PPA</td><td colspan="2" rowspan="1">Specificity/NPA</td></tr><tr><td colspan="1" rowspan="1">TP/TP+FN</td><td colspan="1" rowspan="1">% (95% CI)</td><td colspan="1" rowspan="1">TN/TN+FP</td><td colspan="1" rowspan="1">% (95% CI)</td></tr><tr><td colspan="1" rowspan="4">Coagulase-negativeStaphylococci (CoNS)</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">12/12</td><td colspan="1" rowspan="1">100 (75.8-100)</td><td colspan="1" rowspan="1">6/6</td><td colspan="1" rowspan="1">100 (61.0-100)</td></tr><tr><td colspan="1" rowspan="1">Retrospective</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Contrived</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Combined</td><td colspan="1" rowspan="1">12/12</td><td colspan="1" rowspan="1">100 (75.8-100)</td><td colspan="1" rowspan="1">6/6</td><td colspan="1" rowspan="1">100 (61.0-100)</td></tr><tr><td colspan="1" rowspan="4">CoNS (not S. epidermidis /S. lugdunensis)</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100 (20.7-100)</td><td colspan="1" rowspan="1">0/1</td><td colspan="1" rowspan="1">0.0 (0.0-79.3)</td></tr><tr><td colspan="1" rowspan="1">Retrospective</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Contrived</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Combined</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100 (20.7-100)</td><td colspan="1" rowspan="1">0/1</td><td colspan="1" rowspan="1">0.0 (0.0-79.3)</td></tr><tr><td colspan="1" rowspan="4">Staphylococcus</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">49/52</td><td colspan="1" rowspan="1">94.2 (84.4-98.0)</td><td colspan="1" rowspan="1">24/26</td><td colspan="1" rowspan="1">92.3 (75.9-97.9)</td></tr><tr><td colspan="1" rowspan="1">Retrospective</td><td colspan="1" rowspan="1">1/2</td><td colspan="1" rowspan="1">50.0 (9.5-90.5)</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100 (20.7-100)</td></tr><tr><td colspan="1" rowspan="1">Contrived</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Combined</td><td colspan="1" rowspan="1">50/54</td><td colspan="1" rowspan="1">92.6 (82.4-97.1)</td><td colspan="1" rowspan="1">25/27</td><td colspan="1" rowspan="1">92.6 (76.6-97.9)</td></tr><tr><td colspan="1" rowspan="4">Staphylococcus aureus</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">80/83</td><td colspan="1" rowspan="1">96.4 (89.9-98.8)</td><td colspan="1" rowspan="1">74/77</td><td colspan="1" rowspan="1">96.1 (89.2-98.7)</td></tr><tr><td colspan="1" rowspan="1">Retrospective</td><td colspan="1" rowspan="1">105/106</td><td colspan="1" rowspan="1">99.1 (94.8-99.8)</td><td colspan="1" rowspan="1">16/17</td><td colspan="1" rowspan="1">94.1 (73.0-99.0)</td></tr><tr><td colspan="1" rowspan="1">Contrived</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">100 (72.2-100)</td><td colspan="1" rowspan="1">49/49</td><td colspan="1" rowspan="1">100 (92.7-100)</td></tr><tr><td colspan="1" rowspan="1">Combined</td><td colspan="1" rowspan="1">195/199</td><td colspan="1" rowspan="1">98.0 (94.9-99.2)</td><td colspan="1" rowspan="1">139/143</td><td colspan="1" rowspan="1">97.2 (93.0-98.9)</td></tr><tr><td colspan="1" rowspan="4">Staphylococcus aureus subsp.aureus</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">100 (43.9-100)</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">100 (43.9-100)</td></tr><tr><td colspan="1" rowspan="1">Retrospective</td><td colspan="1" rowspan="1">2/2</td><td colspan="1" rowspan="1">100 (34.2-100)</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Contrived</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Combined</td><td colspan="1" rowspan="1">5/5</td><td colspan="1" rowspan="1">100 (56.6-100)</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">100 (43.9-100)</td></tr><tr><td colspan="1" rowspan="4">Staphylococcus auricularis</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100 (20.7-100)</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100 (20.7-100)</td></tr><tr><td colspan="1" rowspan="1">Retrospective</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">2/2</td><td colspan="1" rowspan="1">100 (34.2-100)</td></tr><tr><td colspan="1" rowspan="1">Contrived</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Combined</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100 (20.7-100)</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">100 (43.9-100)</td></tr><tr><td colspan="1" rowspan="4">Staphylococcus capitis</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">4/5</td><td colspan="1" rowspan="1">80.0 (37.6-96.4)</td><td colspan="1" rowspan="1">5/5</td><td colspan="1" rowspan="1">100 (56.6-100)</td></tr><tr><td colspan="1" rowspan="1">Retrospective</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">100 (51.0-100)</td><td colspan="1" rowspan="1">7/7</td><td colspan="1" rowspan="1">100 (64.6-100)</td></tr><tr><td colspan="1" rowspan="1">Contrived</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Combined</td><td colspan="1" rowspan="1">8/9</td><td colspan="1" rowspan="1">88.9 (56.5-98.0)</td><td colspan="1" rowspan="1">12/12</td><td colspan="1" rowspan="1">100 (75.8-100)</td></tr><tr><td colspan="1" rowspan="4">Staphylococcus carnosus subsp.carnosus</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Retrospective</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100 (20.7-100)</td></tr><tr><td colspan="1" rowspan="1">Contrived</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Combined</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100 (20.7-100)</td></tr><tr><td colspan="1" rowspan="4">Staphylococcus cohnii</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">0/1</td><td colspan="1" rowspan="1">0.0 (0.0-79.3)</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100 (20.7-100)</td></tr><tr><td colspan="1" rowspan="1">Retrospective</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Contrived</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Combined</td><td colspan="1" rowspan="1">0/1</td><td colspan="1" rowspan="1">0.0 (0.0-79.3)</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100 (20.7-100)</td></tr><tr><td colspan="1" rowspan="4">Staphylococcus epidermidis</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">77/79</td><td colspan="1" rowspan="1">97.5 (91.2-99.3)</td><td colspan="1" rowspan="1">39/42</td><td colspan="1" rowspan="1">92.9 (81.0-97.5)</td></tr><tr><td colspan="1" rowspan="1">Retrospective</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">100 (88.6-100)</td><td colspan="1" rowspan="1">7/8</td><td colspan="1" rowspan="1">87.5 (52.9-97.8)</td></tr><tr><td colspan="1" rowspan="1">Contrived</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100 (20.7-100)</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Combined</td><td colspan="1" rowspan="1">108/110</td><td colspan="1" rowspan="1">98.2 (93.6-99.5)</td><td colspan="1" rowspan="1">46/50</td><td colspan="1" rowspan="1">92.0 (81.2-96.8)</td></tr><tr><td colspan="1" rowspan="4">Staphylococcus haemolyticus</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">100 (51.0-100)</td><td colspan="1" rowspan="1">2/2</td><td colspan="1" rowspan="1">100 (34.2-100)</td></tr><tr><td colspan="1" rowspan="1">Retrospective</td><td colspan="1" rowspan="1">2/2</td><td colspan="1" rowspan="1">100 (34.2-100)</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Contrived</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Combined</td><td colspan="1" rowspan="1">6/6</td><td colspan="1" rowspan="1">100 (61.0-100)</td><td colspan="1" rowspan="1">2/2</td><td colspan="1" rowspan="1">100 (34.2-100)</td></tr><tr><td colspan="1" rowspan="4">Staphylococcus hominis</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">12/13</td><td colspan="1" rowspan="1">92.3 (66.7-98.6)</td><td colspan="1" rowspan="1">10/11</td><td colspan="1" rowspan="1">90.9 (62.3-98.4)</td></tr><tr><td colspan="1" rowspan="1">Retrospective</td><td colspan="1" rowspan="1">12/12</td><td colspan="1" rowspan="1">100 (75.8-100)</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100 (20.7-100)</td></tr><tr><td colspan="1" rowspan="1">Contrived</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Combined</td><td colspan="1" rowspan="1">24/25</td><td colspan="1" rowspan="1">96.0 (80.5-99.3)</td><td colspan="1" rowspan="1">11/12</td><td colspan="1" rowspan="1">91.7 (64.6-98.5)</td></tr><tr><td colspan="1" rowspan="4">Staphylococcus hominis subsp.hominis</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">10/11</td><td colspan="1" rowspan="1">90.9 (62.3-98.4)</td><td colspan="1" rowspan="1">11/11</td><td colspan="1" rowspan="1">100 (74.1-100)</td></tr><tr><td colspan="1" rowspan="1">Retrospective</td><td colspan="1" rowspan="1">2/2</td><td colspan="1" rowspan="1">100 (34.2-100)</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">100 (43.9-100)</td></tr><tr><td colspan="1" rowspan="1">Contrived</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Combined</td><td colspan="1" rowspan="1">12/13</td><td colspan="1" rowspan="1">92.3 (66.7-98.6)</td><td colspan="1" rowspan="1">14/14</td><td colspan="1" rowspan="1">100 (78.5-100)</td></tr><tr><td colspan="1" rowspan="4">Staphylococcus hominis subsp.novobiosepticus</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100 (20.7-100)</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Retrospective</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Contrived</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Combined</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100 (20.7-100)</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="4">Staphylococcus lugdunensis</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2/2</td><td colspan="1" rowspan="1">100 (34.2-100)</td></tr><tr><td colspan="1" rowspan="1">Retrospective</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100 (20.7-100)</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">100 (43.9-100)</td></tr><tr><td colspan="1" rowspan="1">Contrived</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">---</td><td colspan="1" rowspan="1">45/45</td><td colspan="1" rowspan="1">100 (92.1-100)</td></tr><tr><td colspan="1" rowspan="1">Combined</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100 (20.7-100)</td><td colspan="1" rowspan="1">50/50</td><td colspan="1" rowspan="1">100 (92.9-100)</td></tr><tr><td colspan="1" rowspan="4">Staphylococcus pettenkoferi</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1/2</td><td colspan="1" rowspan="1">50.0 (9.5-90.5)</td></tr><tr><td colspan="1" rowspan="1">Retrospective</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Contrived</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Combined</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1/2</td><td colspan="1" rowspan="1">50.0 (9.5-90.5)</td></tr><tr><td colspan="1" rowspan="4">Staphylococcus saccharolyticus</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100 (20.7-100)</td></tr><tr><td colspan="1" rowspan="1">Retrospective</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Contrived</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Combined</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100 (20.7-100)</td></tr><tr><td colspan="1" rowspan="4">Staphylococcus saprophyticus</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100 (20.7-100)</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Retrospective</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100 (20.7-100)</td></tr><tr><td colspan="1" rowspan="1">Contrived</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Combined</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100 (20.7-100)</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100 (20.7-100)</td></tr><tr><td colspan="1" rowspan="4">Staphylococcus schleiferi</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Retrospective</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100 (20.7-100)</td></tr><tr><td colspan="1" rowspan="1">Contrived</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Combined</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100 (20.7-100)</td></tr><tr><td colspan="1" rowspan="4">Staphylococcus simulans</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100 (20.7-100)</td><td colspan="1" rowspan="1">2/2</td><td colspan="1" rowspan="1">100 (34.2-100)</td></tr><tr><td colspan="1" rowspan="1">Retrospective</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Contrived</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Combined</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100 (20.7-100)</td><td colspan="1" rowspan="1">2/2</td><td colspan="1" rowspan="1">100 (34.2-100)</td></tr><tr><td colspan="1" rowspan="4">Staphylococcus warneri</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">100 (51.0-100)</td></tr><tr><td colspan="1" rowspan="1">Retrospective</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Contrived</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Combined</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">100 (51.0-100)</td></tr></table>

A comparison of specific Staphylococcus species and mecA identified by comparator methods versus the ePlex BCID-GP Panel results are shown in Table 35 and Table 36 for prospective and retrospective samples.

Table 35: Distribution of mecA Results in Staphylococcus aureus Prospective/Retrospective Samples   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=4>Comparator Method</td></tr><tr><td rowspan=1 colspan=1>BCID-GP</td><td rowspan=1 colspan=1>Org+/ARG+</td><td rowspan=1 colspan=1>Org+/ARG-</td><td rowspan=1 colspan=1>Org-</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Org+/ARG+</td><td rowspan=1 colspan=1>190</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>194</td></tr><tr><td rowspan=1 colspan=1>Org+/ARG-</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>88</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>94</td></tr><tr><td rowspan=1 colspan=1>Org-</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>1000</td><td rowspan=1 colspan=1>1009</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>194</td><td rowspan=1 colspan=1>97</td><td rowspan=1 colspan=1>1006</td><td rowspan=1 colspan=1>1297</td></tr><tr><td rowspan=1 colspan=5>% Agreement (95% CI) for Org+/ARG+: 190/194=97.9 (94.8-99.2)% Agreement (95% CI) for Org+/ARG-: 88/97=90.7 (83.3-95.0)% Agreement (95% CI) for Org-: 1000/1006=99.4 (98.7-99.7)</td></tr></table>

Table 36: Distribution of mecA Results in Staphylococcus Species (Excluding Known S. aureus, $s .$ epidermidis, S. lugdunensis) Prospective/Retrospective Samples   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=4>Comparator Method</td></tr><tr><td rowspan=1 colspan=1>BCID-GP</td><td rowspan=1 colspan=1>Org+/ARG+</td><td rowspan=1 colspan=1>Org+/ARG-</td><td rowspan=1 colspan=1>Org-</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Org+/ARG+</td><td rowspan=1 colspan=1>33</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>38</td></tr><tr><td rowspan=1 colspan=1>Org+/ARG-</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>45</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>56</td></tr><tr><td rowspan=1 colspan=1>Org-</td><td rowspan=1 colspan=1>10*</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>1091</td><td rowspan=1 colspan=1>1105</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>46</td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>1103</td><td rowspan=1 colspan=1>1199</td></tr><tr><td rowspan=1 colspan=5>% Agreement (95% CI) for Org+/ARG+: 33/46=71.7 (57.5-82.7)% Agreement (95% CI) for Org+/ARG-: 45/50=90.0 (78.6-95.7)% Agreement (95% CI) for Org-: 1091/1103=98.9 (98.1-99.4)</td></tr></table>

$^ { * } 1 0$ samples had a Staphylococcus species (not S. aureus, S. epidermidis, or $S$ . lugdunensis) with mecA identified by comparator methods, whereas ePlex BCID-GP Panel detected S. epidermidis with mecA.

A table for mecC is not provided because mecC was only detected in a single species, Staphylococcus aureus. In the 49 contrived samples with Staphylococcus aureus containing mecC, the resulting PPA and NPA were both $100 \%$ .

# vanA/vanB

The PPA and NPA of the BCID-GP Panel vanA target stratified by the Enterococcus species identified by comparator methods for 208 clinical prospective/retrospective samples and 126 contrived samples are shown in Table 37.

A table for vanB is not provided because vanB was only detected in 53 samples comprised of two species, $E .$ . faecalis $( n { = } 4 3 )$ and $E .$ . faecium $( n { = } 1 0 )$ ), resulting in PPA and NPA of $100 \%$ .

Table 37: Clinical Performance of vanA Target by Enterococcus Species Detected by Comparator Methods   

<table><tr><td colspan="2" rowspan="2">Species detected by Comparator Method</td><td colspan="2" rowspan="1">Sensitivity/PPA</td><td colspan="2" rowspan="1">Specificity/NPA</td></tr><tr><td colspan="1" rowspan="1">TP/TP+FN</td><td colspan="1" rowspan="1">% (95% CI)</td><td colspan="1" rowspan="1">TN/TN+FP</td><td colspan="1" rowspan="1">% (95% CI)</td></tr><tr><td colspan="1" rowspan="4">Enterococcus</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100 (20.7-100)</td></tr><tr><td colspan="1" rowspan="1">Retrospective</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">:</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Contrived</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Combined</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100 (20.7-100)</td></tr><tr><td colspan="1" rowspan="4">Enterococcus avium</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100 (20.7-100)</td></tr><tr><td colspan="1" rowspan="1">Retrospective</td><td colspan="1" rowspan="1">0/1</td><td colspan="1" rowspan="1">0.0 (0.0-79.3)</td><td colspan="1" rowspan="1">2/2</td><td colspan="1" rowspan="1">100 (34.2-100)</td></tr><tr><td colspan="1" rowspan="1">Contrived</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Combined</td><td colspan="1" rowspan="1">0/1</td><td colspan="1" rowspan="1">0.0 (0.0-79.3)</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">100 (43.9-100)</td></tr><tr><td colspan="1" rowspan="4">Enterococcus casseliflavus</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Retrospective</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">--</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100 (20.7-100)</td></tr><tr><td colspan="1" rowspan="1">Contrived</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Combined</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100 (20.7-100)</td></tr><tr><td colspan="1" rowspan="4">Enterococcus casseliflavus /E. gallinarum</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Retrospective</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100 (20.7-100)</td></tr><tr><td colspan="1" rowspan="1">Contrived</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Combined</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100 (20.7-100)</td></tr><tr><td colspan="1" rowspan="4">Enterococcus faecalis</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100 (20.7-100)</td><td colspan="1" rowspan="1">48/48</td><td colspan="1" rowspan="1">100 (92.6-100)</td></tr><tr><td colspan="1" rowspan="1">Retrospective</td><td colspan="1" rowspan="1">11/14</td><td colspan="1" rowspan="1">78.6 (52.4-92.4)</td><td colspan="1" rowspan="1">76/76</td><td colspan="1" rowspan="1">100 (95.2-100)</td></tr><tr><td colspan="1" rowspan="1">Contrived</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">100 (72.2-100)</td><td colspan="1" rowspan="1">42/42</td><td colspan="1" rowspan="1">100 (91.6-100)</td></tr><tr><td colspan="1" rowspan="1">Combined</td><td colspan="1" rowspan="1">22/25</td><td colspan="1" rowspan="1">88.0 (70.0-95.8)</td><td colspan="1" rowspan="1">166/166</td><td colspan="1" rowspan="1">100 (97.7-100)</td></tr><tr><td colspan="1" rowspan="4">Enterococcus faecium</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">7/7</td><td colspan="1" rowspan="1">100 (64.6-100)</td><td colspan="1" rowspan="1">4/5</td><td colspan="1" rowspan="1">80.0 (37.6-96.4)</td></tr><tr><td colspan="1" rowspan="1">Retrospective</td><td colspan="1" rowspan="1">44/44</td><td colspan="1" rowspan="1">100 (92.0-100)</td><td colspan="1" rowspan="1">8/9</td><td colspan="1" rowspan="1">88.9 (56.5-98.0)</td></tr><tr><td colspan="1" rowspan="1">Contrived</td><td colspan="1" rowspan="1">50/50</td><td colspan="1" rowspan="1">100 (92.9-100)</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">100 (72.2-100)</td></tr><tr><td colspan="1" rowspan="1">Combined</td><td colspan="1" rowspan="1">101/101</td><td colspan="1" rowspan="1">100 (96.3-100)</td><td colspan="1" rowspan="1">22/24</td><td colspan="1" rowspan="1">91.7 (74.2-97.7)</td></tr><tr><td colspan="1" rowspan="4">Enterococcus flavescens</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">:</td><td colspan="1" rowspan="1">:</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Retrospective</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Contrived</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">100 (43.9-100)</td></tr><tr><td colspan="1" rowspan="1">Combined</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">100 (43.9-100)</td></tr><tr><td colspan="1" rowspan="4">Enterococcus gallinarum</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">:</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Retrospective</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">2/2</td><td colspan="1" rowspan="1">100 (34.2-100)</td></tr><tr><td colspan="1" rowspan="1">Contrived</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">100 (51.0-100)</td></tr><tr><td colspan="1" rowspan="1">Combined</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">6/6</td><td colspan="1" rowspan="1">100 (61.0-100)</td></tr><tr><td colspan="1" rowspan="4">Enterococcus hirae</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Retrospective</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">:</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Contrived</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100 (20.7-100)</td></tr><tr><td colspan="1" rowspan="1">Combined</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100 (20.7-100)</td></tr><tr><td colspan="1" rowspan="4">Enterococcus malodoratus</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">:</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Retrospective</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Contrived</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">100 (43.9-100)</td></tr><tr><td colspan="1" rowspan="1">Combined</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">100 (43.9-100)</td></tr><tr><td colspan="1" rowspan="4">Enterococcus raffinosus</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Retrospective</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Contrived</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2/2</td><td colspan="1" rowspan="1">100 (34.2-100)</td></tr><tr><td colspan="1" rowspan="1">Combined</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">2/2</td><td colspan="1" rowspan="1">100 (34.2-100)</td></tr><tr><td colspan="1" rowspan="4">Enterococcus saccharolyticus</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">:</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Retrospective</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Contrived</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">---</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100 (20.7-100)</td></tr><tr><td colspan="1" rowspan="1">Combined</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100 (20.7-100)</td></tr></table>

A comparison of Enterococcus faecalis/Enterococcus faecium and vanA identified by comparator methods versus the ePlex BCID-GP Panel results is shown in Table 38 and Table 39 for prospective and retrospective samples.

Table 38: Distribution of vanA Results in Enterococcus faecalis Prospective/Retrospective Samples   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=4>Comparator Method</td></tr><tr><td rowspan=1 colspan=1>BCID-GP</td><td rowspan=1 colspan=1>Org+/ARG+</td><td rowspan=1 colspan=1>Org+/ARG-</td><td rowspan=1 colspan=1>Org-</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Org+/ARG+</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>10</td></tr><tr><td rowspan=1 colspan=1>Org+/ARG-</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>120</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>122</td></tr><tr><td rowspan=1 colspan=1>Org-</td><td rowspan=1 colspan=1>4*</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>1157</td><td rowspan=1 colspan=1>1165</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>124</td><td rowspan=1 colspan=1>1158</td><td rowspan=1 colspan=1>1297</td></tr><tr><td rowspan=1 colspan=5>% Agreement (95% CI) for Org+/ARG+: 10/15=66.7 (41.7-84.8)% Agreement (95% CI) for Org+/ARG-: 120/124=96.8 (92.0-98.7)% Agreement (95% CI) for Org-: 1157/1158=99.9 (99.5-100)</td></tr></table>

$^ { * 2 }$ of the 4 samples had $E$ . faecium with vanA detected by the ePlex BCID-GP Panel.

Table 39: Distribution of vanA Results in Enterococcus faecium Prospective/Retrospective Samples   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=4>Comparator Method</td></tr><tr><td rowspan=1 colspan=1>BCID-GP</td><td rowspan=1 colspan=1>Org+/ARG+</td><td rowspan=1 colspan=1>Org+/ARG-</td><td rowspan=1 colspan=1>Org-</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Org+/ARG+</td><td rowspan=1 colspan=1>51</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>56</td></tr><tr><td rowspan=1 colspan=1>Org+/ARG-</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>15</td></tr><tr><td rowspan=1 colspan=1>Org-</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>1224</td><td rowspan=1 colspan=1>1226</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>51</td><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>1232</td><td rowspan=1 colspan=1>1297</td></tr><tr><td rowspan=1 colspan=5>% Agreement (95% CI) for Org+/ARG+: 51/51=100.0 (93.0-100)% Agreement (95% CI) for Org+/ARG-: 10/14=71.4 (45.4-88.3)% Agreement (95% CI) for Org-: 1224/1232=99.4 (98.7-99.7)</td></tr></table>

# Co-detections in Clinical Samples

The ePlex BCID-GP Panel identified a total of 103 bacterial co-detections in 1297 clinical samples (prospective/retrospective). In the 711 prospective samples of the clinical study, 38/711 $( 5 . 3 \% )$ had double detections and 1/711 $( 0 . 1 \% )$ had a triple detection. In the 586 retrospective samples, 56/586 $( 9 . 6 \% )$ had co-detections and 8/586 $( 1 . 4 \% )$ had triple detections. Neither the prospective nor the retrospective arms of the clinical studies contained a sample with more than 3 organisms detected.

In prospective samples, the most common co-detection combination identified by the ePlex BCID-GP Panel was Enterococcus faecalis with Pan Gram-Negative which was detected in 6 samples. Pan Gram-Negative was identified in 17 co-detections, Staphylococcus was identified in 26 co-detections, Enterococcus was identified in 13 co-detections, and Streptococcus was identified in 12 co-detections.

In retrospective samples, the most common co-detection combination identified by the ePlex BCID-GP Panel was Enterococcus faecalis with Pan Gram-Negative which was detected in 9 samples, 2 of which also had vanA detected. Pan Gram-Negative was identified in 33 codetections, Staphylococcus was identified in 22 co-detections, Enterococcus was identified in 34 co-detections, and Streptococcus was identified in 24 co-detections. See Tables 40-41 below for summaries of co-detections in prospective and retrospective samples in targets as determined by the ePlex BCID-GP Panel.

Table 40: Co-Detections Identified by the ePlex BCID-GP Panel in Prospective Clinical Samples by Target   

<table><tr><td colspan="4" rowspan="1">Distinct Co-Detection Combinations Detected by the ePlex BCID-GPPanel in Prospective Clinical Samples</td><td colspan="1" rowspan="2">Number Samples(NumberDiscrepant)</td><td colspan="1" rowspan="2">DiscrepantOrganism(s) /ResistanceMarker(s)*</td></tr><tr><td colspan="1" rowspan="1">Target 1</td><td colspan="1" rowspan="1">Target 2</td><td colspan="1" rowspan="1">Target 3</td><td colspan="1" rowspan="1">ResistanceMMarker</td></tr><tr><td colspan="1" rowspan="1">C. acnes</td><td colspan="1" rowspan="1">Staphylococcus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (1)</td><td colspan="1" rowspan="1">C. acnes (1)</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium</td><td colspan="1" rowspan="1">S. epidermidis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">mecA</td><td colspan="1" rowspan="1">2 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">E. faecalis</td><td colspan="1" rowspan="1">E. faecium</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2 (1)</td><td colspan="1" rowspan="1">E. faecium (1)</td></tr><tr><td colspan="1" rowspan="1">E. faecalis</td><td colspan="1" rowspan="1">Pan GN</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">6 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">E. faecalis</td><td colspan="1" rowspan="1">S. epidermidis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">mecA</td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">E. faecalis</td><td colspan="1" rowspan="1">Staphylococcus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">E. faecalis</td><td colspan="1" rowspan="1">Staphylococcus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">mecA</td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Target 1</td><td colspan="1" rowspan="1">Target 2</td><td colspan="1" rowspan="1">Target 3</td><td colspan="1" rowspan="1">ResistanceMarker</td></tr><tr><td colspan="1" rowspan="1">E. faecium</td><td colspan="1" rowspan="1">Pan GN</td><td colspan="1" rowspan="1">Staphylococcus</td><td colspan="1" rowspan="1">mecA, vanA</td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">E. faecium</td><td colspan="1" rowspan="1">S. epidermidis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">mecA, vanA</td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Lactobacillus</td><td colspan="1" rowspan="1">Streptococcus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (1)</td><td colspan="1" rowspan="1">Lactobacillus (1)</td></tr><tr><td colspan="1" rowspan="1">Listeria</td><td colspan="1" rowspan="1">Staphylococcus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (1)</td><td colspan="1" rowspan="1">Listeria (1)</td></tr><tr><td colspan="1" rowspan="1">Pan GN</td><td colspan="1" rowspan="1">S. anginosus group</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Pan GN</td><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Pan GN</td><td colspan="1" rowspan="1">S. epidermidis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">mecA</td><td colspan="1" rowspan="1">2 (2)</td><td colspan="1" rowspan="1">Pan GN (1),S. epidermidis (1)</td></tr><tr><td colspan="1" rowspan="1">Pan GN</td><td colspan="1" rowspan="1">S. pneumoniae</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (1)</td><td colspan="1" rowspan="1">S. pneumoniae (1)</td></tr><tr><td colspan="1" rowspan="1">Pan GN</td><td colspan="1" rowspan="1">Staphylococcus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Pan GN</td><td colspan="1" rowspan="1">Staphylococcus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">mecA</td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Pan GN</td><td colspan="1" rowspan="1">Streptococcus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">S. agalactiae</td><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">S. anginosus group</td><td colspan="1" rowspan="1">Staphylococcus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2 (2)</td><td colspan="1" rowspan="1">S. anginosus gp (1),Staphylococcus (1)</td></tr><tr><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1">S. epidermidis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">mecA</td><td colspan="1" rowspan="1">2 (2)</td><td colspan="1" rowspan="1">S. epidermidis (2)</td></tr><tr><td colspan="1" rowspan="1">S. epidermidis</td><td colspan="1" rowspan="1">S. lugdunensis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (1)</td><td colspan="1" rowspan="1">S. epidermidis (1),S. lugdunensis (1)</td></tr><tr><td colspan="1" rowspan="1">S. epidermidis</td><td colspan="1" rowspan="1">S. lugdunensis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">mecA</td><td colspan="1" rowspan="1">1 (1)</td><td colspan="1" rowspan="1">S. lugdunensis (1)</td></tr><tr><td colspan="1" rowspan="1">S. epidermidis</td><td colspan="1" rowspan="1">Streptococcus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2 (1)</td><td colspan="1" rowspan="1">S. epidermidis (1)</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus</td><td colspan="1" rowspan="1">Streptococcus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Staphylococcus</td><td colspan="1" rowspan="1">Streptococcus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">mecA</td><td colspan="1" rowspan="1">1 (1)</td><td colspan="1" rowspan="1">Streptococcus (1)</td></tr></table>

\* A discrepant result is one that was detected by the ePlex BCID-GP Panel but not by the comparator method(s). 7/16 discrepant organisms were detected using PCR/sequencing as shown below: -In 1/1 false positive $E .$ . faecium samples, $E .$ . faecium was detected. -In 1/1 false positive Lactobacillus samples, Lactobacillus was detected. -In 2/5 false positive S. epidermidis samples, S. epidermidis was detected. - $. \ln { 2 / 2 }$ false positive S. lugdunensis samples, S. lugdunensis was detected. -In 1/1 false positive Streptococcus samples, Streptococcus was detected.

Table 41: Co-Detections Identified by the ePlex BCID-GP Panel in Retrospective Clinical Samples by Target   

<table><tr><td colspan="4" rowspan="1">Distinct Co-Detection Combinations Detected by the ePlex BCID-GPPanel in Retrospective Clinical Samples</td><td colspan="1" rowspan="2">Number Samples(NumberDiscrepant)</td><td colspan="1" rowspan="2">DiscrepantOrganism(s) /Resistance Marker(s)*</td></tr><tr><td colspan="1" rowspan="1">Target 1</td><td colspan="1" rowspan="1">Target 2</td><td colspan="1" rowspan="1">Target 3</td><td colspan="1" rowspan="1">ResistanceMarker</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium</td><td colspan="1" rowspan="1">S. epidermidis</td><td colspan="1" rowspan="1">S. lugdunensis</td><td colspan="1" rowspan="1">mecA</td><td colspan="1" rowspan="1">1 (1)</td><td colspan="1" rowspan="1">Corynebacterium (1)</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium</td><td colspan="1" rowspan="1">Staphylococcus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Corynebacterium</td><td colspan="1" rowspan="1">Staphylococcus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">mecA</td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">E. faecalis</td><td colspan="1" rowspan="1">E. faecium</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">4 (3)</td><td colspan="1" rowspan="1">E. faecium (3)</td></tr><tr><td colspan="1" rowspan="1">E. faecalis</td><td colspan="1" rowspan="1">E. faecium</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">vanA</td><td colspan="1" rowspan="1">3 (1)</td><td colspan="1" rowspan="1">E. faecium (1)</td></tr><tr><td colspan="4" rowspan="1">Distinct Co-Detection Combinations Detected by the ePlex BCID-GPPanel in Retrospective Clinical Samples</td><td colspan="1" rowspan="2">Number Samples(Number Discrepant)</td><td colspan="1" rowspan="2">DiscrepantOrganism(s) /Resistance Marker(s)*</td></tr><tr><td colspan="1" rowspan="1">Target 1</td><td colspan="1" rowspan="1">Target 2</td><td colspan="1" rowspan="1">Target 3</td><td colspan="1" rowspan="1">ResistanceMarker</td></tr><tr><td colspan="1" rowspan="1">E. faecalis</td><td colspan="1" rowspan="1">Pan Candida</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">E. faecalis</td><td colspan="1" rowspan="1">Pan GN</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">6 (0)</td><td colspan="1" rowspan="1">Y</td></tr><tr><td colspan="1" rowspan="1">E. faecalis</td><td colspan="1" rowspan="1">Pan GN</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">vanA</td><td colspan="1" rowspan="1">2 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">E. faecalis</td><td colspan="1" rowspan="1">Pan GN</td><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1">Y</td></tr><tr><td colspan="1" rowspan="1">E. faecalis</td><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">mecA</td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1">Y</td></tr><tr><td colspan="1" rowspan="1">E. faecalis</td><td colspan="1" rowspan="1">Staphylococcus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">vanA</td><td colspan="1" rowspan="1">1 (1)</td><td colspan="1" rowspan="1">Staphylococcus (1)</td></tr><tr><td colspan="1" rowspan="1">E. faecium</td><td colspan="1" rowspan="1">Lactobacillus</td><td colspan="1" rowspan="1">Pan GN</td><td colspan="1" rowspan="1">vanA</td><td colspan="1" rowspan="1">1 (1)</td><td colspan="1" rowspan="1">Lactobacillus (1)</td></tr><tr><td colspan="1" rowspan="1">E. faecium</td><td colspan="1" rowspan="1">Pan Candida</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">vanA</td><td colspan="1" rowspan="1">1 (1)</td><td colspan="1" rowspan="1">E. faecium (1)</td></tr><tr><td colspan="1" rowspan="1">E. faecium</td><td colspan="1" rowspan="1">Pan Candida</td><td colspan="1" rowspan="1">S. epidermidis</td><td colspan="1" rowspan="1">mecA, vanA</td><td colspan="1" rowspan="1">1 (1)</td><td colspan="1" rowspan="1">S. epidermidis (1)</td></tr><tr><td colspan="1" rowspan="1">E. faecium</td><td colspan="1" rowspan="1">Pan GN</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">3 (0)</td><td colspan="1" rowspan="1">Y</td></tr><tr><td colspan="1" rowspan="1">E. faecium</td><td colspan="1" rowspan="1">Pan GN</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">vanA</td><td colspan="1" rowspan="1">5 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">E. faecium</td><td colspan="1" rowspan="1">Pan GN</td><td colspan="1" rowspan="1">Staphylococcus</td><td colspan="1" rowspan="1">mecA, vanA</td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">E. faecium</td><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">mecA, vanA</td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1">Y</td></tr><tr><td colspan="1" rowspan="1">E. faecium</td><td colspan="1" rowspan="1">Streptococcus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">vanA</td><td colspan="1" rowspan="1">1 (1)</td><td colspan="1" rowspan="1">Streptococcus (1)</td></tr><tr><td colspan="1" rowspan="1">Enterococcus</td><td colspan="1" rowspan="1">S. anginosus group</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Lactobacillus</td><td colspan="1" rowspan="1">S. anginosus group</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Micrococcus</td><td colspan="1" rowspan="1">S. pyogenes</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (1)</td><td colspan="1" rowspan="1">Micrococcus (1)</td></tr><tr><td colspan="1" rowspan="1">Pan Candida</td><td colspan="1" rowspan="1">S. epidermidis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">mecA</td><td colspan="1" rowspan="1">2 (0)</td><td colspan="1" rowspan="1">Y</td></tr><tr><td colspan="1" rowspan="1">Pan Candida</td><td colspan="1" rowspan="1">S. pneumoniae</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1">Y</td></tr><tr><td colspan="1" rowspan="1">Pan GN</td><td colspan="1" rowspan="1">S. agalactiae</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2 (1)</td><td colspan="1" rowspan="1">Pan GN (1)</td></tr><tr><td colspan="1" rowspan="1">Pan GN</td><td colspan="1" rowspan="1">S. anginosus group</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">4 (0)</td><td colspan="1" rowspan="1">Y</td></tr><tr><td colspan="1" rowspan="1">Pan GN</td><td colspan="1" rowspan="1">S. anginosus group</td><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (1)</td><td colspan="1" rowspan="1">Pan GN (1)</td></tr><tr><td colspan="1" rowspan="1">Pan GN</td><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1">Y</td></tr><tr><td colspan="1" rowspan="1">Pan GN</td><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1">S. epidermidis</td><td colspan="1" rowspan="1">mecA</td><td colspan="1" rowspan="1">1 (1)</td><td colspan="1" rowspan="1">S. epidermidis (1)</td></tr><tr><td colspan="1" rowspan="1">Pan GN</td><td colspan="1" rowspan="1">S. pneumoniae</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Pan GN</td><td colspan="1" rowspan="1">Streptococcus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">3 (0)</td><td colspan="1" rowspan="1">Y</td></tr><tr><td colspan="1" rowspan="1">S. agalactiae</td><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">S. agalactiae</td><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">mecA</td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1">Y</td></tr><tr><td colspan="1" rowspan="1">S. agalactiae</td><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1">S. epidermidis</td><td colspan="1" rowspan="1">mecA</td><td colspan="1" rowspan="1">1 (1)</td><td colspan="1" rowspan="1">S. epidermidis (1)</td></tr><tr><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1">S. epidermidis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">mecA</td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1">Y</td></tr><tr><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1">Streptococcus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">mecA</td><td colspan="1" rowspan="1">2 (1)</td><td colspan="1" rowspan="1">Streptococcus (1)</td></tr><tr><td colspan="1" rowspan="1">S. epidermidis</td><td colspan="1" rowspan="1">Streptococcus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">mecA</td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1">Y</td></tr></table>

\* A discrepant result is one that was detected by the ePlex BCID-GP Panel but not by the comparator method(s). 6/16 discrepant organisms were detected using PCR/sequencing as shown below: -In 1/1 false positive Corynebacterium samples, Corynebacterium was detected. - $. \ln { 2 / 5 }$ false positive $E .$ . faecium samples, $E .$ . faecium was detected. -In 1/1 false positive Staphylococcus samples, Staphylococcus was detected. - $. \ln { 2 / 2 }$ false positive Streptococcus samples, Streptococcus was detected.

Additional co-detection combinations identified by comparator method(s) are summarized in Table 42 and Table 43 for prospective and retrospective samples, respectively.

Table 42: Additional Co-Detections Identified by Comparator Method(s) in Prospective Clinical Samples by Organism   

<table><tr><td colspan="5" rowspan="1">Distinct Co-Detection Combinations Detected by theComparator Methods in Prospective Clinical Samples</td><td colspan="1" rowspan="2">NumberSamples(NumberDiscrepant</td><td colspan="1" rowspan="2">DiscrepantOrganism(s)/ReistanceMarkers(s)*</td></tr><tr><td colspan="1" rowspan="1">Organism 1</td><td colspan="1" rowspan="1">Organism 2</td><td colspan="1" rowspan="1">Organism 3</td><td colspan="1" rowspan="1">Organism 4</td><td colspan="1" rowspan="1">ResistanceMarker</td></tr><tr><td colspan="1" rowspan="1">A. baumannii</td><td colspan="1" rowspan="1">E. faecium</td><td colspan="1" rowspan="1">Staphylococcus</td><td colspan="1" rowspan="1">Y</td><td colspan="1" rowspan="1">mecA, vanA</td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1">Y</td></tr><tr><td colspan="1" rowspan="1">A. baumannii</td><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">A. baumannii</td><td colspan="1" rowspan="1">Staphylococcus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">mecA</td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1">Y</td></tr><tr><td colspan="1" rowspan="1">Acinetobacter lwoffi</td><td colspan="1" rowspan="1">Staphylococcushominis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">mecA</td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Aerococcus viridansA</td><td colspan="1" rowspan="1">K. oxytoca</td><td colspan="1" rowspan="1">S. epidermidis</td><td colspan="1" rowspan="1">Staphylococcuscohni</td><td colspan="1" rowspan="1">mecA</td><td colspan="1" rowspan="1">1 (1)</td><td colspan="1" rowspan="1">S. cohnii (1),S. epidermidis (1),mecA (1)</td></tr><tr><td colspan="1" rowspan="1">Aerococcus viridans A</td><td colspan="1" rowspan="1">Staphylococcushominis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">B. fragilis</td><td colspan="1" rowspan="1">ClostridiumspeciesA</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">B. fragilis</td><td colspan="1" rowspan="1">S. anginosus gp</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">C. acnes</td><td colspan="1" rowspan="1">S. epidermidis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (1)</td><td colspan="1" rowspan="1">S. epidermidis (1)</td></tr><tr><td colspan="1" rowspan="1">C. acnes</td><td colspan="1" rowspan="1">S. lugdunensis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (1)</td><td colspan="1" rowspan="1">C. acnes (1)</td></tr><tr><td colspan="1" rowspan="1">Citrobacter freundii</td><td colspan="1" rowspan="1">K. pneumoniae</td><td colspan="1" rowspan="1">Staphylococcushominis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">mecA</td><td colspan="1" rowspan="1">1 (1)</td><td colspan="1" rowspan="1">meA (1)</td></tr><tr><td colspan="1" rowspan="1">Citrobacter koseri</td><td colspan="1" rowspan="1">E. faecalis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Corynebacterium</td><td colspan="1" rowspan="1">S. epidermidis</td><td colspan="1" rowspan="1">Streptococcus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">mecA</td><td colspan="1" rowspan="1">1 (1)</td><td colspan="1" rowspan="1">Streptococcus (1)</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium</td><td colspan="1" rowspan="1">Streptococcus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (1)</td><td colspan="1" rowspan="1">Corynebacterium (1)</td></tr><tr><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">E. faecalis</td><td colspan="1" rowspan="1">P. mirabilis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">LactococcuslactisA</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">P. mirabilis</td><td colspan="1" rowspan="1">Providenciastuartii</td><td colspan="1" rowspan="1">S. anginosus gp</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">E. faecalis</td><td colspan="1" rowspan="1">E. faecium</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Y</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (1)</td><td colspan="1" rowspan="1">E. faecium (1)</td></tr><tr><td colspan="1" rowspan="1">E. faecalis</td><td colspan="1" rowspan="1">K. pneumoniae</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Y</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2 (0)</td><td colspan="1" rowspan="1">Y</td></tr><tr><td colspan="1" rowspan="1">E. faecalis</td><td colspan="1" rowspan="1">P. mirabilis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Y</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1">Y</td></tr><tr><td colspan="1" rowspan="1">E. faecalis</td><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">mecA</td><td colspan="1" rowspan="1">1 (1)</td><td colspan="1" rowspan="1">S. aureus (1),mecA (1)</td></tr><tr><td colspan="1" rowspan="1">E. faecalis</td><td colspan="1" rowspan="1">S. marcescens</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">E. faecalis</td><td colspan="1" rowspan="1">StaphylococcusCoNSOS</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">mecA</td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">E. faecium</td><td colspan="1" rowspan="1">S. epidermidis</td><td colspan="1" rowspan="1">Staphylococcushaemolyticus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">mecA, vanA</td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">K. pneumoniae</td><td colspan="1" rowspan="1">Staphylococcushaemolyticus</td><td colspan="1" rowspan="1">non-fermentingGN bacilli</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">P. aeruginosa</td><td colspan="1" rowspan="1">P. mirabilis</td><td colspan="1" rowspan="1">Streptococcus -viridans group</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">P. mirabilis</td><td colspan="1" rowspan="1">Staphylococcus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">mecA</td><td colspan="1" rowspan="1">1 (1)</td><td colspan="1" rowspan="1">Staphylococcus (1),mecA(1)</td></tr><tr><td colspan="1" rowspan="1">PeptostreptococcusspeciesA</td><td colspan="1" rowspan="1">Staphylococcus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="5" rowspan="1">Distinct Co-Detection Combinations Detected by theComparator Methods in Prospective Clinical Samples</td><td colspan="1" rowspan="2">NumberSamples(NumberDiscrepant)</td><td colspan="1" rowspan="2">DiscrepantOrganism(s)/ResistanceMarkers(s)*</td></tr><tr><td colspan="1" rowspan="1">Organism 1</td><td colspan="1" rowspan="1">Organism 2</td><td colspan="1" rowspan="1">Organism 3</td><td colspan="1" rowspan="1">Organism 4</td><td colspan="1" rowspan="1">ResistanceMarker</td></tr><tr><td colspan="1" rowspan="1">Rothia(stomatococcus)mucilaginosusA</td><td colspan="1" rowspan="1">S. epidermidis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Rothia mucilaginosa</td><td colspan="1" rowspan="1">Streptococcus -viridans group</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">S. agalactiae</td><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1">Staphylococcus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">mecA</td><td colspan="1" rowspan="1">1 (1)</td><td colspan="1" rowspan="1">S. aureus (1),Staphylococcus (1),mecA (1)</td></tr><tr><td colspan="1" rowspan="1">S. anginosus</td><td colspan="1" rowspan="1">Streptococcusmitis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (1)</td><td colspan="1" rowspan="1">S. anginosus (1)</td></tr><tr><td colspan="1" rowspan="1">S. epidermidis</td><td colspan="1" rowspan="1">Staphylococcuscapitis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">mecA</td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">S. epidermidis</td><td colspan="1" rowspan="1">Staphylococcushominis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">S. epidermidis</td><td colspan="1" rowspan="1">Staphylococcushominis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">mecA</td><td colspan="1" rowspan="1">4 (1)</td><td colspan="1" rowspan="1">. epidermidis (1)</td></tr><tr><td colspan="1" rowspan="1">S. epidermidis</td><td colspan="1" rowspan="1">Staphylococcushominis</td><td colspan="1" rowspan="1">Staphylococcuswarneri</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">S. epidermidis</td><td colspan="1" rowspan="1">Streptococcus -viridans group</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (1)</td><td colspan="1" rowspan="1">S. epidermidis (1)</td></tr><tr><td colspan="1" rowspan="1">S. epidermidis</td><td colspan="1" rowspan="1">Streptococcusparasanguinis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">S. maltophilia</td><td colspan="1" rowspan="1">Streptococcus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (1)</td><td colspan="1" rowspan="1">Streptococcus (1)</td></tr><tr><td colspan="1" rowspan="1">S. marcescens</td><td colspan="1" rowspan="1">Streptococcusmitis group</td><td colspan="1" rowspan="1">Streptococcussalivarius</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Staphylococcus cohnii</td><td colspan="1" rowspan="1">Streptococcus -viridans group</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (1)</td><td colspan="1" rowspan="1">S. viridans group (1)</td></tr><tr><td colspan="1" rowspan="1">Staphylococcushominis</td><td colspan="1" rowspan="1">Staphylococcuspettenkoferi</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Staphylococcushominis</td><td colspan="1" rowspan="1">Streptococcusmitis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">mecA</td><td colspan="1" rowspan="1">1 (1)</td><td colspan="1" rowspan="1">mecA (1)</td></tr></table>

\* A discrepant result is one that was detected by the comparator method(s) but not by the ePlex BCID-GP Panel (excludes organisms not targeted by the ePlex BCID-GP Panel). 16 discrepant organisms were investigated using PCR/sequencing; 1 discrepant organism was not detected: -In 1/1 false negative $S _ { \ast }$ . anginosus group sample, PCR/Sequencing instead detected Streptococcus dysgalactiae.

A. Off-panel organisms not targeted by the ePlex BCID-GP Panel.

# Table 43: Additional Co-Detections Identified by Comparator Method(s) in Retrospective

Clinical Samples by Organism

<table><tr><td colspan="5" rowspan="1">Distinct Co-Detection Combinations Detected by theComparator Methods in Retrospective Clinical Samples</td><td colspan="1" rowspan="2">NumberSamples(NumberDiscrepant)</td><td colspan="1" rowspan="2">DiscrepantOrganism(s)/ResistanceMarkers(s)*</td></tr><tr><td colspan="1" rowspan="1">Organism 1</td><td colspan="1" rowspan="1">Organism 2</td><td colspan="1" rowspan="1">Organism 3</td><td colspan="1" rowspan="1">Organism 4</td><td colspan="1" rowspan="1">ResistanceMarker</td></tr><tr><td colspan="1" rowspan="1">A. baumannii</td><td colspan="1" rowspan="1">E. faecalis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">vanA</td><td colspan="1" rowspan="1">2 (2)</td><td colspan="1" rowspan="1">A. baumannii (2)</td></tr><tr><td colspan="1" rowspan="1">A. baumannii</td><td colspan="1" rowspan="1">E. faecalis</td><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">mecA</td><td colspan="1" rowspan="1">1 (1)</td><td colspan="1" rowspan="1">mecA (1)</td></tr><tr><td colspan="1" rowspan="1">A. baumannii</td><td colspan="1" rowspan="1">E. faecium</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">vanA</td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Aerococcussanguinicola A</td><td colspan="1" rowspan="1">Corynebacterium</td><td colspan="1" rowspan="1">Staphylococcussaprophyticus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="5" rowspan="1">Distinct Co-Detection Combinations Detected by theComparator Methods in Retrospective Clinical Samples</td><td colspan="1" rowspan="1">NumberSamples</td><td colspan="1" rowspan="2">DiscrepantOrganism(s)/ResistanceMarkers(s)*</td></tr><tr><td colspan="1" rowspan="1">Organism 1</td><td colspan="1" rowspan="1">Organism 2</td><td colspan="1" rowspan="1">Organism 3</td><td colspan="1" rowspan="1">Organism 4</td><td colspan="1" rowspan="1">ResistanceMarker</td><td colspan="1" rowspan="1">NumberDiscrepant</td></tr><tr><td colspan="1" rowspan="1">Aeromonas caviae</td><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">Enterococcuscasseliflavus</td><td colspan="1" rowspan="1">K. oxytoca</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (1)</td><td colspan="1" rowspan="1">E. casseliflavus(1)</td></tr><tr><td colspan="1" rowspan="1">C. acnes</td><td colspan="1" rowspan="1">Enterococcusavium</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">vanA</td><td colspan="1" rowspan="1">1 (1)</td><td colspan="1" rowspan="1">E. avium (1),vanA)</td></tr><tr><td colspan="1" rowspan="1">C. albicans</td><td colspan="1" rowspan="1">E. faecalis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">vanA</td><td colspan="1" rowspan="1">1 (1)</td><td colspan="1" rowspan="1">E. faecalis (1)</td></tr><tr><td colspan="1" rowspan="1">C. albicans</td><td colspan="1" rowspan="1">E. faecium</td><td colspan="1" rowspan="1">Staphylococcushominis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">mecA, vanA</td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">C. albicans</td><td colspan="1" rowspan="1">S. epidermidis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">mecA</td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">C. glabrata</td><td colspan="1" rowspan="1">Lactobacillusrhamnosus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (1)</td><td colspan="1" rowspan="1">C. glabrata (1)</td></tr><tr><td colspan="1" rowspan="1">C. glabrata</td><td colspan="1" rowspan="1">S. pneumoniae</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1">Y</td></tr><tr><td colspan="1" rowspan="1">C. krusei</td><td colspan="1" rowspan="1">S. epidermidis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">mecA</td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">C. parapsilosis</td><td colspan="1" rowspan="1">E. faecalis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1(0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">C. parapsilosis</td><td colspan="1" rowspan="1">E. faecalis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">vanA</td><td colspan="1" rowspan="1">1 (1)</td><td colspan="1" rowspan="1">C. parapsilosis(11)</td></tr><tr><td colspan="1" rowspan="1">Citrobacter braakii</td><td colspan="1" rowspan="1">Streptococcusoralis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">E. cloacae</td><td colspan="1" rowspan="1">E. faecalis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">PY</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1">Y</td></tr><tr><td colspan="1" rowspan="1">E. cloacae</td><td colspan="1" rowspan="1">E. faecium</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">vanA</td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1">Y</td></tr><tr><td colspan="1" rowspan="1">E. cloacae</td><td colspan="1" rowspan="1">E. faecium</td><td colspan="1" rowspan="1">Staphylococcushominis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">mecA, vanA</td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">E. cloacae</td><td colspan="1" rowspan="1">S. anginosus gp</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">E. faecalis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">3 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">E. faecalis</td><td colspan="1" rowspan="1">K. pneumoniae</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1">Y</td></tr><tr><td colspan="1" rowspan="1">E. col</td><td colspan="1" rowspan="1">E. faecalis</td><td colspan="1" rowspan="1">P. mirabilis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1">Y</td></tr><tr><td colspan="1" rowspan="1">E. col</td><td colspan="1" rowspan="1">E. faecium</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2(0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">K. oxytoca</td><td colspan="1" rowspan="1">Streptococcusinfantarius</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">E. col</td><td colspan="1" rowspan="1">S. agalactiae</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">S. anginosus gp</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1">Y</td></tr><tr><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">mecA</td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1">Y</td></tr><tr><td colspan="1" rowspan="1">E. col</td><td colspan="1" rowspan="1">S. pneumoniae</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">Streptococcusbovis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">E. faecalis</td><td colspan="1" rowspan="1">K. pneumoniae</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">vanA</td><td colspan="1" rowspan="1">1 (1)</td><td colspan="1" rowspan="1">E. faecalis (1),K. pemoniae(1)</td></tr><tr><td colspan="1" rowspan="1">E. faecalis</td><td colspan="1" rowspan="1">M. morganii</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">vanA</td><td colspan="1" rowspan="1">1(0)</td><td colspan="1" rowspan="1">Y</td></tr><tr><td colspan="1" rowspan="1">E. faecalis</td><td colspan="1" rowspan="1">M. morganii</td><td colspan="1" rowspan="1">Proteus vulgaris</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">vanA</td><td colspan="1" rowspan="1">1 (1)</td><td colspan="1" rowspan="1">E. faecalis (1),vanA(1)</td></tr><tr><td colspan="1" rowspan="1">E. faecalis</td><td colspan="1" rowspan="1">P. aeruginosa</td><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">mecA</td><td colspan="1" rowspan="1">1 (1)</td><td colspan="1" rowspan="1">E. faecalis (1),P. aeruginosa (1)</td></tr><tr><td colspan="1" rowspan="1">E. faecalis</td><td colspan="1" rowspan="1">P. mirabilis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2 (2)</td><td colspan="1" rowspan="1">E. faecalis (1),P. mirabilis (1)</td></tr><tr><td colspan="1" rowspan="1">E. faecalis</td><td colspan="1" rowspan="1">P. mirabilis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">vanA</td><td colspan="1" rowspan="1">1 (1)</td><td colspan="1" rowspan="1">E. faecalis (1),vanA (1)</td></tr><tr><td colspan="1" rowspan="1">E. faecalis</td><td colspan="1" rowspan="1">Providenciastuartii</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (1)</td><td colspan="1" rowspan="1">P. stuartii (1)</td></tr><tr><td colspan="1" rowspan="1">E. faecalis</td><td colspan="1" rowspan="1">S. maltophilia</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">vanA</td><td colspan="1" rowspan="1">1(0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">E. faecium</td><td colspan="1" rowspan="1">K. pneumoniae</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="5" rowspan="1">Distinct Co-Detection Combinations Detected by theComparator Methods in Retrospective Clinical Samples</td><td colspan="1" rowspan="2">NumberSamples(NumberDiscrepant)</td><td colspan="1" rowspan="2">DiscrepantOrganism(s)/ResistanceMarkers(s)*</td></tr><tr><td colspan="1" rowspan="1">Organism 1</td><td colspan="1" rowspan="1">Organism 2</td><td colspan="1" rowspan="1">Organism 3</td><td colspan="1" rowspan="1">Organism 4</td><td colspan="1" rowspan="1">ResistanceMarker</td></tr><tr><td colspan="1" rowspan="1">E. faecium</td><td colspan="1" rowspan="1">Moraxella(Branhamella)catarrhalis</td><td colspan="1" rowspan="1">Pediococcuspentosaceus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">vanA</td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">E. faecium</td><td colspan="1" rowspan="1">P. aeruginosa</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">vanA</td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">E. faecium</td><td colspan="1" rowspan="1">P. mirabilis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">vanA</td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">E. faecium</td><td colspan="1" rowspan="1">Pseudomonas</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">vanA</td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">E. faecium</td><td colspan="1" rowspan="1">S. epidermidis</td><td colspan="1" rowspan="1">Staphylococcushoinis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">mecA</td><td colspan="1" rowspan="1">1 (1)</td><td colspan="1" rowspan="1">E. faecium (1)</td></tr><tr><td colspan="1" rowspan="1">Enterobacteraerogenes</td><td colspan="1" rowspan="1">S. anginosus gp</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Enterococcus avium</td><td colspan="1" rowspan="1">S. anginosus gp</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">K. oxytoca</td><td colspan="1" rowspan="1">S. anginosus gp</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">K. pneumoniae</td><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2(1)</td><td colspan="1" rowspan="1">S. aureus (1)</td></tr><tr><td colspan="1" rowspan="1">L. monocytogenes</td><td colspan="1" rowspan="1">Staphylococcus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">mecA</td><td colspan="1" rowspan="1">1 (1)</td><td colspan="1" rowspan="1">Staphylococcus(1), mecA (1)</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus casei</td><td colspan="1" rowspan="1">Veillonellaspecies</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (1)</td><td colspan="1" rowspan="1">Veillonellaspecies (1)</td></tr><tr><td colspan="1" rowspan="1">Lactobacillusrhamnosus</td><td colspan="1" rowspan="1">Pediococcusacidilactici </td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Lactobacillusrhamnosus</td><td colspan="1" rowspan="1">S. anginosus gp</td><td colspan="1" rowspan="1">Staphylococcus</td><td colspan="1" rowspan="1">Streptococcus -viridans group</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (1)</td><td colspan="1" rowspan="1">Staphylococcus(1)</td></tr><tr><td colspan="1" rowspan="1">Micrococcus</td><td colspan="1" rowspan="1">Pseudoclavibacter.A</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Moraxella catarrhalis</td><td colspan="1" rowspan="1">S. pneumoniae</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">S. agalactiae</td><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (1)</td><td colspan="1" rowspan="1">S. aureus (1)</td></tr><tr><td colspan="1" rowspan="1">S. agalactiae</td><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1">Streptococcus -viridans group</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1">S. epidermidis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (1)</td><td colspan="1" rowspan="1">S. aureus (1)</td></tr><tr><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1">S. pyogenes</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">mecA</td><td colspan="1" rowspan="1">1 (1)</td><td colspan="1" rowspan="1">S. pyogenes (1)</td></tr><tr><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1">Staphylococcuscapitis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1">Streptococcusmitis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">mecA</td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">S. epidermidis</td><td colspan="1" rowspan="1">Staphylococcushominis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">S. epidermidis</td><td colspan="1" rowspan="1">Staphylococcushominis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">mecA</td><td colspan="1" rowspan="1">3 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">S. epidermidis</td><td colspan="1" rowspan="1">Staphylococcushominis</td><td colspan="1" rowspan="1">Streptococcusparasanguinis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">mecA</td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Staphylococcus capitis</td><td colspan="1" rowspan="1">Staphylococcushominis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">mecA</td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr></table>

\* A discrepant result is one that was detected by the comparator method(s) but not by the ePlex BCID-GP Panel (excludes organisms not targeted by the ePlex BCID-GP Panel). 24 discrepant organisms were investigated using PCR/sequencing; 2 discrepant organisms were not detected: - $. \mathrm { I n } 2 / 6$ false negative $E .$ . faecalis samples, PCR/Sequencing instead detected Enterococcus faecium.

# Clinical Study ePlex Instrument Performance

A total of 2354 samples (including prospective, retrospective, and contrived samples) were initially tested in the clinical evaluations. Of these, 24/2354 $( 1 . 0 \% )$ did not complete the run and the sample was retested. After repeat testing, all 2354 samples completed testing and 2246/2354 ( $9 5 . 4 \%$ , $9 5 \%$ CI: $9 4 . 5 \% - 9 6 . 2 \%$ ) generated valid results and 108/2354 ( $4 . 6 \%$ , $9 5 \%$ CI: $3 . 8 \%$ - $5 . 5 \%$ ) generated invalid results on the first completed attempt.

Upon repeat testing of the 108 samples with initially invalid results, 3/108 $( 2 . 8 \% )$ did not complete the run and the sample was retested. After repeat testing, all 108 samples completed testing and 106/108 $( 9 8 . 1 \% )$ generated valid results. Overall, after final testing, 2/2354 $( 0 . 1 \%$ , $9 5 \%$ CI: $0 . 0 \% 1 . 0 3 \%$ ) had final, invalid results, resulting in a final validity rate of 2352/2354 $( 9 9 . 9 \%$ , $9 5 \%$ CI: $9 9 . 7 \% - 1 0 0 \% )$ ).

# ANALYTICAL PERFORMANCE CHARACTERISTICS

# Limit of Detection (Analytical Sensitivity)

The limit of detection (LoD), or analytical sensitivity, was identified and verified for each assay on the BCID-GP Panel using at least two quantified reference strains in simulated blood culture sample matrix, which is defined as a whole blood with EDTA added to a blood culture bottle in the same ratio as the manufacturer recommends and incubated for 8 hours. At least 20 replicates per target were tested for each condition. The limit of detection was defined as the lowest concentration of each target that is detected in $2 9 5 \%$ of tested replicates. The confirmed LoD for each ePlex BCID-GP Panel organism is shown in Table 44.

Table 44: LoD Results Summary   
Confidential   

<table><tr><td colspan="1" rowspan="1">Target</td><td colspan="1" rowspan="1">Organism</td><td colspan="1" rowspan="1">Strain</td><td colspan="1" rowspan="1">LoD Concentration(CFU/mL)</td></tr><tr><td colspan="1" rowspan="2">B. cereus group</td><td colspan="1" rowspan="1">Bacillus cereus</td><td colspan="1" rowspan="1">ATCC 21769</td><td colspan="1" rowspan="1">1 x 105}$</td></tr><tr><td colspan="1" rowspan="1">Bacillus thuringiensis</td><td colspan="1" rowspan="1">ATCC 35646</td><td colspan="1" rowspan="1">1 x 105</td></tr><tr><td colspan="1" rowspan="2">B. subtilis group</td><td colspan="1" rowspan="1">Bacillus subtilis</td><td colspan="1" rowspan="1">ATCC 55614</td><td colspan="1" rowspan="1">1 x 106</td></tr><tr><td colspan="1" rowspan="1">Bacillus atrophaeus</td><td colspan="1" rowspan="1">ATCC 51189</td><td colspan="1" rowspan="1">1 x 106</td></tr><tr><td colspan="1" rowspan="2">Corynebacterium</td><td colspan="1" rowspan="1">Corynebacterium striatum</td><td colspan="1" rowspan="1">ATCC 43735</td><td colspan="1" rowspan="1">1 x 106</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium jeikeium</td><td colspan="1" rowspan="1">ATCC 43217</td><td colspan="1" rowspan="1">1 x 107</td></tr><tr><td colspan="1" rowspan="2">Cutibacterium acnes (P. acnes)</td><td colspan="1" rowspan="1">Cutibacterium acnes (P. acnes)</td><td colspan="1" rowspan="1">ATCC 33179</td><td colspan="1" rowspan="1">1 x 107</td></tr><tr><td colspan="1" rowspan="1">Cutibacterium acnes (P. acnes)</td><td colspan="1" rowspan="1">ATCC 6919</td><td colspan="1" rowspan="1">1 x 108</td></tr><tr><td colspan="1" rowspan="3">Enterococcus</td><td colspan="1" rowspan="1">Enterococcus faecium</td><td colspan="1" rowspan="1">ATCC BAA-2316</td><td colspan="1" rowspan="1">1 x 105</td></tr><tr><td colspan="1" rowspan="1">Enterococcus faecium</td><td colspan="1" rowspan="1">ATCC BAA-2317</td><td colspan="1" rowspan="1">1 x 106</td></tr><tr><td colspan="1" rowspan="1">Enterococcus raffinosus</td><td colspan="1" rowspan="1">ATCC 49464</td><td colspan="1" rowspan="1">1 x 106</td></tr><tr><td colspan="1" rowspan="2">Enterococcus faecium</td><td colspan="1" rowspan="1">Enterococcus faecium</td><td colspan="1" rowspan="1">ATCC BAA-2316</td><td colspan="1" rowspan="1">1 x 105</td></tr><tr><td colspan="1" rowspan="1">Enterococcus faecium</td><td colspan="1" rowspan="1">ATCC BAA-2317</td><td colspan="1" rowspan="1">1 x 106</td></tr><tr><td colspan="1" rowspan="2">Enterococcus faecalis</td><td colspan="1" rowspan="1">Enterococcus faecalis</td><td colspan="1" rowspan="1">ATCC 51575</td><td colspan="1" rowspan="1">1 x 106</td></tr><tr><td colspan="1" rowspan="1">Enterococcus faecalis</td><td colspan="1" rowspan="1">ATCC 700802</td><td colspan="1" rowspan="1">1 x 106</td></tr><tr><td colspan="1" rowspan="2">Lactobacillus</td><td colspan="1" rowspan="1">Lactobacillus paracasei</td><td colspan="1" rowspan="1">ATCC 25598</td><td colspan="1" rowspan="1">1 x 105</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus casei</td><td colspan="1" rowspan="1">ATCC 334</td><td colspan="1" rowspan="1">1 x 105</td></tr><tr><td colspan="1" rowspan="3">Listeria</td><td colspan="1" rowspan="1">Listeria seeligeri</td><td colspan="1" rowspan="1">ATCC 35967</td><td colspan="1" rowspan="1">1 x 105</td></tr><tr><td colspan="1" rowspan="1">Listeria monocytogenes</td><td colspan="1" rowspan="1">ATCC 10890</td><td colspan="1" rowspan="1">1 x 105}$</td></tr><tr><td colspan="1" rowspan="1">Listeria monocytogenes</td><td colspan="1" rowspan="1">ATCC 19111</td><td colspan="1" rowspan="1">1 x 106</td></tr><tr><td colspan="1" rowspan="2">Listeria monocytogenes</td><td colspan="1" rowspan="1">Listeria monocytogenes</td><td colspan="1" rowspan="1">ATCC 10890</td><td colspan="1" rowspan="1">1 x 105</td></tr><tr><td colspan="1" rowspan="1">Listeria monocytogenes</td><td colspan="1" rowspan="1">ATCC 19111</td><td colspan="1" rowspan="1">1 x 105</td></tr><tr><td colspan="1" rowspan="2">Micrococcus</td><td colspan="1" rowspan="1">Micrococcus luteus</td><td colspan="1" rowspan="1">ATCC 19212</td><td colspan="1" rowspan="1">1 x 106</td></tr><tr><td colspan="1" rowspan="1">Micrococcus luteus</td><td colspan="1" rowspan="1">ATCC 10240</td><td colspan="1" rowspan="1">1 x 107</td></tr><tr><td colspan="1" rowspan="7">Staphylococcus</td><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">ATCC BAA-2313</td><td colspan="1" rowspan="1">1 x 104</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">ATCC BAA-2312</td><td colspan="1" rowspan="1">1 x 105</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus epidermidis</td><td colspan="1" rowspan="1">ATCC 35983</td><td colspan="1" rowspan="1">1 x 105</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus epidermidis</td><td colspan="1" rowspan="1">ATCC 35984</td><td colspan="1" rowspan="1">1 x 105</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus lugdunensis</td><td colspan="1" rowspan="1">NRS 879</td><td colspan="1" rowspan="1">1 x 105</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus lugdunensis</td><td colspan="1" rowspan="1">ATCC 49576</td><td colspan="1" rowspan="1">1 x 106</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus haemolyticus</td><td colspan="1" rowspan="1">NRS 62</td><td colspan="1" rowspan="1">1 x 107</td></tr><tr><td colspan="1" rowspan="2">Staphylococcus aureus</td><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">ATCC BAA-2313</td><td colspan="1" rowspan="1">1 x 105$</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">ATCC BAA-2312</td><td colspan="1" rowspan="1">1 x 105</td></tr><tr><td colspan="1" rowspan="2">Staphylococcus epidermidis</td><td colspan="1" rowspan="1">Staphylococcus epidermidis</td><td colspan="1" rowspan="1">ATCC 35983</td><td colspan="1" rowspan="1">1 x 105</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus epidermidis</td><td colspan="1" rowspan="1">ATCC 35984</td><td colspan="1" rowspan="1">1 x 105</td></tr><tr><td colspan="1" rowspan="2">Staphylococcus lugdunensis</td><td colspan="1" rowspan="1">Staphylococcus lugdunensis</td><td colspan="1" rowspan="1">NRS 879</td><td colspan="1" rowspan="1">1 x 105</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus lugdunensis</td><td colspan="1" rowspan="1">ATCC 49576</td><td colspan="1" rowspan="1">1 x 105</td></tr><tr><td colspan="1" rowspan="5">Streptococus</td><td colspan="1" rowspan="1">Streptococus pneumoniae</td><td colspan="1" rowspan="1">ATCC BAA-475</td><td colspan="1" rowspan="1">1 x 105</td></tr><tr><td colspan="1" rowspan="1">Streptococus pneumoniae</td><td colspan="1" rowspan="1">ATCC 10357</td><td colspan="1" rowspan="1">1 x 105</td></tr><tr><td colspan="1" rowspan="1">Streptococus gordonii</td><td colspan="1" rowspan="1">ATCC 10558</td><td colspan="1" rowspan="1">1 x 106</td></tr><tr><td colspan="1" rowspan="1">Streptococus agalactiae</td><td colspan="1" rowspan="1">ATCC 12401</td><td colspan="1" rowspan="1">1 x 106</td></tr><tr><td colspan="1" rowspan="1">Streptococus agalactiae</td><td colspan="1" rowspan="1">ATCC 13813</td><td colspan="1" rowspan="1">1 x 107</td></tr><tr><td colspan="1" rowspan="2">Streptococus agalactiae</td><td colspan="1" rowspan="1">Streptococus agalactiae</td><td colspan="1" rowspan="1">ATCC 12401</td><td colspan="1" rowspan="1">1 x 105</td></tr><tr><td colspan="1" rowspan="1">Streptococus agalactiae</td><td colspan="1" rowspan="1">ATCC 13813</td><td colspan="1" rowspan="1">1 x 106</td></tr><tr><td colspan="1" rowspan="2">Streptococcus anginosus group</td><td colspan="1" rowspan="1">Streptococus intermedius</td><td colspan="1" rowspan="1">ATCC 27335</td><td colspan="1" rowspan="1">1 x 104</td></tr><tr><td colspan="1" rowspan="1">Streptococus anginosus</td><td colspan="1" rowspan="1">ATCC 9895</td><td colspan="1" rowspan="1">1 x 106</td></tr><tr><td colspan="1" rowspan="2">Streptococus pneumoniae</td><td colspan="1" rowspan="1">Streptococus pneumoniae</td><td colspan="1" rowspan="1">ATCC BAA-475</td><td colspan="1" rowspan="1">1 x 105</td></tr><tr><td colspan="1" rowspan="1">Streptococus pneumoniae</td><td colspan="1" rowspan="1">ATCC 10357</td><td colspan="1" rowspan="1">1 x 105</td></tr><tr><td colspan="1" rowspan="2">Streptococus pyogenes</td><td colspan="1" rowspan="1">Streptococus pyogenes</td><td colspan="1" rowspan="1">ATCC 12384</td><td colspan="1" rowspan="1">1 x 105</td></tr><tr><td colspan="1" rowspan="1">Streptococus pyogenes</td><td colspan="1" rowspan="1">ATCC 49399</td><td colspan="1" rowspan="1">1 x 105$</td></tr><tr><td colspan="1" rowspan="9">Pan Gram-Negative</td><td colspan="1" rowspan="1">Stenotrophomonas maltophilia</td><td colspan="1" rowspan="1">ATCC 13636</td><td colspan="1" rowspan="1">1 x 106</td></tr><tr><td colspan="1" rowspan="1">Enterobacter cloacae</td><td colspan="1" rowspan="1">ATCC 13047</td><td colspan="1" rowspan="1">1 x 106</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">ATCC 4157</td><td colspan="1" rowspan="1">1 x 106</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">ATCC BAA-1706</td><td colspan="1" rowspan="1">1 x 106</td></tr><tr><td colspan="1" rowspan="1">Serratia marcescens</td><td colspan="1" rowspan="1">ATCC 8100</td><td colspan="1" rowspan="1">1 x 106</td></tr><tr><td colspan="1" rowspan="1">Proteus mirabilis</td><td colspan="1" rowspan="1">ATCC 43071</td><td colspan="1" rowspan="1">1 x 106</td></tr><tr><td colspan="1" rowspan="1">Acinetobacter baumannii</td><td colspan="1" rowspan="1">NCTC13302</td><td colspan="1" rowspan="1">1 x 107</td></tr><tr><td colspan="1" rowspan="1">Neisseria meningitidis</td><td colspan="1" rowspan="1">ATCC 13113</td><td colspan="1" rowspan="1">1 x 107</td></tr><tr><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">ATCC 15442</td><td colspan="1" rowspan="1">1 x 107</td></tr><tr><td colspan="1" rowspan="2">Pan Candida</td><td colspan="1" rowspan="1">Candida albicans</td><td colspan="1" rowspan="1">ATCC 24433</td><td colspan="1" rowspan="1">1 x 106</td></tr><tr><td colspan="1" rowspan="1">Candida glabrata</td><td colspan="1" rowspan="1">ATCC 66032</td><td colspan="1" rowspan="1">1 x 106</td></tr><tr><td colspan="1" rowspan="2">mecA</td><td colspan="1" rowspan="1">Staphylococcus epidermidis</td><td colspan="1" rowspan="1">ATCC 35983</td><td colspan="1" rowspan="1">1 x 105</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus epidermidis</td><td colspan="1" rowspan="1">ATCC 35984</td><td colspan="1" rowspan="1">1 x 105</td></tr><tr><td colspan="1" rowspan="2">mecC</td><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">ATCC BAA-2313</td><td colspan="1" rowspan="1">1 x 104</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">ATCC BAA-2312</td><td colspan="1" rowspan="1">1 x 104</td></tr><tr><td colspan="1" rowspan="2">vanA</td><td colspan="1" rowspan="1">Enterococcus faecium</td><td colspan="1" rowspan="1">ATCC BAA-2316</td><td colspan="1" rowspan="1">1 x 104</td></tr><tr><td colspan="1" rowspan="1">Enterococcus faecium</td><td colspan="1" rowspan="1">ATCC BAA-2317</td><td colspan="1" rowspan="1">1 x 105}$</td></tr><tr><td colspan="1" rowspan="2">vanB</td><td colspan="1" rowspan="1">Enterococcus faecalis</td><td colspan="1" rowspan="1">ATCC 51575</td><td colspan="1" rowspan="1">1 x 105</td></tr><tr><td colspan="1" rowspan="1">Enterococcus faecalis</td><td colspan="1" rowspan="1">ATCC 700802</td><td colspan="1" rowspan="1">1 x 105</td></tr></table>

# Analytical Reactivity (Inclusivity)

A panel of 459 strains/isolates representing the genetic, temporal and geographic diversity of each target on the ePlex BCID-GP Panel was evaluated to demonstrate analytical reactivity. Each bacterial strain was tested in triplicate at $1 \mathrm { ~ x ~ } 1 0 ^ { 8 } \mathrm { C F U / m L }$ or less and each fungal strain was tested at $1 \mathrm { ~ x ~ } 1 0 ^ { 6 } \mathrm { C F U / m L }$ . In the cases where the initial testing concentration did not result in a “Detected” result, the concentration was increased to the point where detection was observed (see footnotes for concentrations of these strains). Organisms detected are shown in Table 45. Additional strains were detected as a part of the Limit of Detection (Analytical Sensitivity) study and can be found in Table 44.

Table 45: Analytical Reactivity (Inclusivity) Results   

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Strain</td></tr><tr><td rowspan=1 colspan=1>Bacillus cereus</td><td rowspan=1 colspan=1>cYanmaGellowb</td></tr><tr><td rowspan=4 colspan=1>Bacillus cereus</td><td rowspan=1 colspan=1>ATCC 21769</td></tr><tr><td rowspan=1 colspan=1>ATCC 10876</td></tr><tr><td rowspan=1 colspan=1>ATCC 31430</td></tr><tr><td rowspan=1 colspan=1>ATCC 53522</td></tr><tr><td rowspan=4 colspan=1>Bacillus thuringiensis</td><td rowspan=1 colspan=1>ATCC 35646</td></tr><tr><td rowspan=1 colspan=1>ATCC 33679</td></tr><tr><td rowspan=1 colspan=1>ATCC 55173</td></tr><tr><td rowspan=1 colspan=1>ATCC 10792</td></tr><tr><td rowspan=1 colspan=2>Bacillus subtilis</td></tr><tr><td rowspan=4 colspan=1>Bacillus amyloliquefaciens</td><td rowspan=1 colspan=1>ATCC 23845</td></tr><tr><td rowspan=1 colspan=1>ATCC 23842</td></tr><tr><td rowspan=1 colspan=1>ATCC 23350</td></tr><tr><td rowspan=1 colspan=1>ATCC 53495</td></tr><tr><td rowspan=5 colspan=1>Bacillus atrophaeus</td><td rowspan=1 colspan=1>ATCC 51189</td></tr><tr><td rowspan=1 colspan=1>ATCC 6455</td></tr><tr><td rowspan=1 colspan=1>ATCC 49337</td></tr><tr><td rowspan=1 colspan=1>ATCC 7972</td></tr><tr><td rowspan=1 colspan=1>ATCC 6537</td></tr><tr><td rowspan=4 colspan=1>Bacillus licheniformis</td><td rowspan=1 colspan=1>ATCC 53926</td></tr><tr><td rowspan=1 colspan=1>ATCC 55768</td></tr><tr><td rowspan=1 colspan=1>ATCC 21039</td></tr><tr><td rowspan=1 colspan=1>ATCC 21667</td></tr><tr><td rowspan=4 colspan=1>Bacillus subtilis</td><td rowspan=1 colspan=1>ATCC 55614</td></tr><tr><td rowspan=1 colspan=1>ATCC 15561</td></tr><tr><td rowspan=1 colspan=1>ATCC 21008</td></tr><tr><td rowspan=1 colspan=1>ATCC 15040</td></tr><tr><td rowspan=1 colspan=1>Corynebacterium</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Corynebacterium afermentanssubsp. afermentans</td><td rowspan=1 colspan=1>ATCC 51403A</td></tr><tr><td rowspan=1 colspan=1>Corynebacterium afermentanssubsp. lipophilum</td><td rowspan=1 colspan=1>ATCC 51404</td></tr><tr><td rowspan=1 colspan=1>Corynebacterium confusum</td><td rowspan=1 colspan=1>ATCC 382684</td></tr><tr><td rowspan=1 colspan=1>Corynebacterium coyleae</td><td rowspan=1 colspan=1>ATCC 700219</td></tr><tr><td rowspan=1 colspan=1>Corynebacterium diphtheriae</td><td rowspan=1 colspan=1>ATCC 13812</td></tr><tr><td rowspan=1 colspan=1>Corynebacterium falsenii</td><td rowspan=1 colspan=1>ATCC BAA-596A</td></tr><tr><td rowspan=1 colspan=1>Corynebacterium freneyi</td><td rowspan=1 colspan=1>ATCC 64424B</td></tr><tr><td rowspan=1 colspan=1>Corynebacterium imitans</td><td rowspan=1 colspan=1>ATCC 700354B</td></tr><tr><td rowspan=4 colspan=1>Corynebacterium jeikeium</td><td rowspan=1 colspan=1>ATCC 43217</td></tr><tr><td rowspan=1 colspan=1>ATCC 43216</td></tr><tr><td rowspan=1 colspan=1>ATCC 43734</td></tr><tr><td rowspan=1 colspan=1>ATCC BAA-949</td></tr><tr><td rowspan=1 colspan=1>Corynebacterium minutissimum</td><td rowspan=1 colspan=1>ATCC 23348</td></tr><tr><td rowspan=1 colspan=1>Corynebacterium resistens</td><td rowspan=1 colspan=1>CCUG 50093T</td></tr><tr><td rowspan=1 colspan=1>Corynebacterium simulans</td><td rowspan=1 colspan=1>ATCC BAA-15</td></tr><tr><td rowspan=3 colspan=1>Corynebacterium striatum</td><td rowspan=1 colspan=1>ATCC BAA-1293</td></tr><tr><td rowspan=1 colspan=1>ATCC 43735</td></tr><tr><td rowspan=1 colspan=1>ATCC 7094</td></tr></table>

<table><tr><td colspan="1" rowspan="1">Organism</td><td colspan="1" rowspan="1">Strain</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus lentus</td><td colspan="1" rowspan="1">ATCC 700403</td></tr><tr><td colspan="1" rowspan="4">Staphylococcus lugdunensis</td><td colspan="1" rowspan="1">NRS-879</td></tr><tr><td colspan="1" rowspan="1">ATCC 49576</td></tr><tr><td colspan="1" rowspan="1">NRS-878</td></tr><tr><td colspan="1" rowspan="1">NRS-880</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus muscae</td><td colspan="1" rowspan="1">ATCC 49910</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus pasteuri</td><td colspan="1" rowspan="1">ATCC 51129</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus pasteuri(mecC+)</td><td colspan="1" rowspan="1">ATCC 51128</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus pettenkoferi</td><td colspan="1" rowspan="1">DSM-19554</td></tr><tr><td colspan="1" rowspan="1">Staphylococcuspseudintermedius</td><td colspan="1" rowspan="1">ATCC 49444</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus saccharolyticus</td><td colspan="1" rowspan="1">ATCC 14953</td></tr><tr><td colspan="1" rowspan="2">Staphylococcus saprophyticus</td><td colspan="1" rowspan="1">ATCC 15305</td></tr><tr><td colspan="1" rowspan="1">ATCC 35552</td></tr><tr><td colspan="1" rowspan="2">Staphylococcus schleiferi</td><td colspan="1" rowspan="1">ATCC 49545</td></tr><tr><td colspan="1" rowspan="1">ATCC 43808</td></tr><tr><td colspan="1" rowspan="5">Staphylococcus sciuri</td><td colspan="1" rowspan="1">ATCC 29060</td></tr><tr><td colspan="1" rowspan="1">ATCC 29061</td></tr><tr><td colspan="1" rowspan="1">ATCC 29059</td></tr><tr><td colspan="1" rowspan="1">ATCC 49575</td></tr><tr><td colspan="1" rowspan="1">ATCC 29062</td></tr><tr><td colspan="1" rowspan="4">Staphylococcus simulans</td><td colspan="1" rowspan="1">ATCC 27848</td></tr><tr><td colspan="1" rowspan="1">ATCC 27850</td></tr><tr><td colspan="1" rowspan="1">ATCC 31432</td></tr><tr><td colspan="1" rowspan="1">ATCC 27851</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus species</td><td colspan="1" rowspan="1">ATCC 155</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus species(mecA+)</td><td colspan="1" rowspan="1">ATCC 27626</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus vitulinus</td><td colspan="1" rowspan="1">ATCC 51699</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">ATCC 51161</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus warneri</td><td colspan="1" rowspan="1">ATCC 27836</td></tr><tr><td colspan="1" rowspan="2">Staphylococcus xylosus</td><td colspan="1" rowspan="1">ATCC 49148</td></tr><tr><td colspan="1" rowspan="1">ATCC 29971</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="10">Staphylococcus aureus</td><td colspan="1" rowspan="1">ATCC 25923</td></tr><tr><td colspan="1" rowspan="1">ATCC 29247</td></tr><tr><td colspan="1" rowspan="1">ATCC 6538P</td></tr><tr><td colspan="1" rowspan="1">ATCC 29213</td></tr><tr><td colspan="1" rowspan="1">NR-45889</td></tr><tr><td colspan="1" rowspan="1">NR-45890</td></tr><tr><td colspan="1" rowspan="1">NR-46074</td></tr><tr><td colspan="1" rowspan="1">NR-45881</td></tr><tr><td colspan="1" rowspan="1">NR-46411</td></tr><tr><td colspan="1" rowspan="1">NR-46414</td></tr><tr><td colspan="1" rowspan="2">Staphylococcus aureus</td><td colspan="1" rowspan="1">NR-46418</td></tr><tr><td colspan="1" rowspan="1">NR-46417</td></tr><tr><td colspan="1" rowspan="2">Staphylococcus aureus(mecA+)</td><td colspan="1" rowspan="1">NR-13524</td></tr><tr><td colspan="1" rowspan="1">NR-13527</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium timonense</td><td colspan="1" rowspan="1">CCUG 64728A</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium ulcerans</td><td colspan="1" rowspan="1">ATCC 51799</td></tr><tr><td colspan="1" rowspan="3">Corynebacterium urealyticum</td><td colspan="1" rowspan="1">ATCC 43044</td></tr><tr><td colspan="1" rowspan="1">ATCC 43042</td></tr><tr><td colspan="1" rowspan="1">ATCC 43043</td></tr><tr><td colspan="1" rowspan="1">Corynebacteriumureicelerivorans</td><td colspan="1" rowspan="1">CCUG 59144</td></tr><tr><td colspan="2" rowspan="1">Cutibacterium acnes</td></tr><tr><td colspan="1" rowspan="4">Cutibacterium acnes</td><td colspan="1" rowspan="1">ATCC 11827</td></tr><tr><td colspan="1" rowspan="1">ATCC 11828</td></tr><tr><td colspan="1" rowspan="1">ATCC 33179</td></tr><tr><td colspan="1" rowspan="1">ATCC 6919</td></tr><tr><td colspan="1" rowspan="1">Enterococcus</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Enterococcus avium</td><td colspan="1" rowspan="1">ATCC 14025</td></tr><tr><td colspan="1" rowspan="3">Enterococcus casseliflavus</td><td colspan="1" rowspan="1">ATCC 700668</td></tr><tr><td colspan="1" rowspan="1">ATCC 25788</td></tr><tr><td colspan="1" rowspan="1">ATCC 700327</td></tr><tr><td colspan="1" rowspan="1">Enterococcus cecorum</td><td colspan="1" rowspan="1">ATCC 43198</td></tr><tr><td colspan="1" rowspan="1">Enterococcus dispar</td><td colspan="1" rowspan="1">ATCC 51266</td></tr><tr><td colspan="1" rowspan="1">Enterococcus durans</td><td colspan="1" rowspan="1">ATCC 11576</td></tr><tr><td colspan="1" rowspan="18">Enterococcus faecalis</td><td colspan="1" rowspan="1">ATCC 51575</td></tr><tr><td colspan="1" rowspan="1">ATCC 700802</td></tr><tr><td colspan="1" rowspan="1">ATCC 10100</td></tr><tr><td colspan="1" rowspan="1">ATCC 12399</td></tr><tr><td colspan="1" rowspan="1">ATCC 14506</td></tr><tr><td colspan="1" rowspan="1">ATCC 33186</td></tr><tr><td colspan="1" rowspan="1">ATCC 49532</td></tr><tr><td colspan="1" rowspan="1">ATCC 49533</td></tr><tr><td colspan="1" rowspan="1">ATCC 7080</td></tr><tr><td colspan="1" rowspan="1">ATCC 49474</td></tr><tr><td colspan="1" rowspan="1">ATCC 49332</td></tr><tr><td colspan="1" rowspan="1">ATCC 29200</td></tr><tr><td colspan="1" rowspan="1">ATCC BAA-2128</td></tr><tr><td colspan="1" rowspan="1">ATCC 51188</td></tr><tr><td colspan="1" rowspan="1">ATCC 49149</td></tr><tr><td colspan="1" rowspan="1">NCTC-775</td></tr><tr><td colspan="1" rowspan="1">ATCC 19433</td></tr><tr><td colspan="1" rowspan="1">ATCC 49452</td></tr><tr><td colspan="1" rowspan="1">Enterococcus faecalis (vanA+)</td><td colspan="1" rowspan="1">JMI 876745</td></tr><tr><td colspan="1" rowspan="2">Enterococcus faecalis (vanB+)</td><td colspan="1" rowspan="1">ATCC BAA-2365</td></tr><tr><td colspan="1" rowspan="1">ATCC 51299</td></tr><tr><td colspan="1" rowspan="7">Enterococcus faecium</td><td colspan="1" rowspan="1">ATCC 19434</td></tr><tr><td colspan="1" rowspan="1">ATCC 23828</td></tr><tr><td colspan="1" rowspan="1">ATCC 27273</td></tr><tr><td colspan="1" rowspan="1">ATCC BAA-2127</td></tr><tr><td colspan="1" rowspan="1">ATCC 6057</td></tr><tr><td colspan="1" rowspan="1">ATCC 49624</td></tr><tr><td colspan="1" rowspan="1">ATCC 6569</td></tr><tr><td colspan="1" rowspan="2">Enterococcus faecium</td><td colspan="1" rowspan="1">ATCC BAA-472</td></tr><tr><td colspan="1" rowspan="1">ATCC 9756</td></tr></table>

<table><tr><td colspan="1" rowspan="1">Organism</td><td colspan="1" rowspan="1">Strain</td></tr><tr><td colspan="1" rowspan="6"></td><td colspan="1" rowspan="1">NR-13525</td></tr><tr><td colspan="1" rowspan="1">NR-13526</td></tr><tr><td colspan="1" rowspan="1">NRS-123</td></tr><tr><td colspan="1" rowspan="1">NR-46805</td></tr><tr><td colspan="1" rowspan="1">NRS-662</td></tr><tr><td colspan="1" rowspan="1">ATCC BAA-1707</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">ATCC 43300</td></tr><tr><td colspan="1" rowspan="4"></td><td colspan="1" rowspan="1">NRS-383</td></tr><tr><td colspan="1" rowspan="1">NCTC-12493</td></tr><tr><td colspan="1" rowspan="1">NRS-676</td></tr><tr><td colspan="1" rowspan="1">NRS-678</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">NRS-648</td></tr><tr><td colspan="1" rowspan="26"></td><td colspan="1" rowspan="1">NRS-651</td></tr><tr><td colspan="1" rowspan="1">NRS-643</td></tr><tr><td colspan="1" rowspan="1">NRS-484</td></tr><tr><td colspan="1" rowspan="1">NRS-385</td></tr><tr><td colspan="1" rowspan="1">NRS-384</td></tr><tr><td colspan="1" rowspan="1">ATCC BAA-40</td></tr><tr><td colspan="1" rowspan="1">ATCC 700698</td></tr><tr><td colspan="1" rowspan="1">NRS-382</td></tr><tr><td colspan="1" rowspan="1">NRS-659</td></tr><tr><td colspan="1" rowspan="1">NRS-657</td></tr><tr><td colspan="1" rowspan="1">NRS-655</td></tr><tr><td colspan="1" rowspan="1">NRS-654</td></tr><tr><td colspan="1" rowspan="1">NRS-647</td></tr><tr><td colspan="1" rowspan="1">ATCC BAA-42</td></tr><tr><td colspan="1" rowspan="1">ATCC BAA-41</td></tr><tr><td colspan="1" rowspan="1">NRS-483</td></tr><tr><td colspan="1" rowspan="1">NRS-675</td></tr><tr><td colspan="1" rowspan="1">NRS-645</td></tr><tr><td colspan="1" rowspan="1">NRS-687</td></tr><tr><td colspan="1" rowspan="1">NRS-667</td></tr><tr><td colspan="1" rowspan="1">NRS-677</td></tr><tr><td colspan="1" rowspan="1">NRS-683</td></tr><tr><td colspan="1" rowspan="1">NRS-688</td></tr><tr><td colspan="1" rowspan="1">NRS-22</td></tr><tr><td colspan="1" rowspan="1">NRS-387</td></tr><tr><td colspan="1" rowspan="1">NRS-70</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus(mecA+)</td><td colspan="1" rowspan="1">ATCC 33591</td></tr><tr><td colspan="1" rowspan="2">Staphylococcus aureus(mecC+)</td><td colspan="1" rowspan="1">ATCC BAA-2312</td></tr><tr><td colspan="1" rowspan="1">ATCC BAA-2313</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus epidermidis</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="5">Staphylococcus epidermidis</td><td colspan="1" rowspan="1">ATCC 12228</td></tr><tr><td colspan="1" rowspan="1">ATCC 49134</td></tr><tr><td colspan="1" rowspan="1">NCIMB-8853</td></tr><tr><td colspan="1" rowspan="1">ATCC 700583</td></tr><tr><td colspan="1" rowspan="1">ATCC 14990</td></tr><tr><td colspan="1" rowspan="2">Staphylococcus epidermidis(mecA+)</td><td colspan="1" rowspan="1">ATCC 35983</td></tr><tr><td colspan="1" rowspan="1">ATCC 35984</td></tr><tr><td colspan="1" rowspan="5"></td><td colspan="1" rowspan="1">ATCC 49224</td></tr><tr><td colspan="1" rowspan="1">ATCC BAA-2846</td></tr><tr><td colspan="1" rowspan="1">ATCC 19953</td></tr><tr><td colspan="1" rowspan="1">ATCC 27270</td></tr><tr><td colspan="1" rowspan="1">ATCC 35667</td></tr><tr><td colspan="1" rowspan="15">Enterococcus faecium (vanA+)</td><td colspan="1" rowspan="1">ATCC BAA-2316</td></tr><tr><td colspan="1" rowspan="1">ATCC BAA-2317</td></tr><tr><td colspan="1" rowspan="1">ATCC 700221</td></tr><tr><td colspan="1" rowspan="1">ATCC BAA-2319</td></tr><tr><td colspan="1" rowspan="1">ATCC BAA-2320</td></tr><tr><td colspan="1" rowspan="1">LMC 003921</td></tr><tr><td colspan="1" rowspan="1">LMC 103676</td></tr><tr><td colspan="1" rowspan="1">LMC 089524</td></tr><tr><td colspan="1" rowspan="1">LMC 104266</td></tr><tr><td colspan="1" rowspan="1">LMC 032261</td></tr><tr><td colspan="1" rowspan="1">LMC 110371</td></tr><tr><td colspan="1" rowspan="1">LMC 002867</td></tr><tr><td colspan="1" rowspan="1">LMC 055971</td></tr><tr><td colspan="1" rowspan="1">ATCC 51559</td></tr><tr><td colspan="1" rowspan="1">ATCC BAA-2318</td></tr><tr><td colspan="1" rowspan="1">Enterococcus faecium (vanB+)</td><td colspan="1" rowspan="1">ATCC 51858</td></tr><tr><td colspan="1" rowspan="5">Enterococcus gallinarum</td><td colspan="1" rowspan="1">ATCC 49573</td></tr><tr><td colspan="1" rowspan="1">ATCC 49609</td></tr><tr><td colspan="1" rowspan="1">ATCC 49610</td></tr><tr><td colspan="1" rowspan="1">ATCC 700425</td></tr><tr><td colspan="1" rowspan="1">ATCC 49608</td></tr><tr><td colspan="1" rowspan="2">Enterococcus hirae</td><td colspan="1" rowspan="1">ATCC 49479</td></tr><tr><td colspan="1" rowspan="1">ATCC 10541</td></tr><tr><td colspan="1" rowspan="1">Enterococcus italicus</td><td colspan="1" rowspan="1">CCUG 47860</td></tr><tr><td colspan="1" rowspan="1">Enterococcus malodoratus</td><td colspan="1" rowspan="1">ATCC 43197</td></tr><tr><td colspan="1" rowspan="1">Enterococcus pseudoavium</td><td colspan="1" rowspan="1">CCUG 33310T</td></tr><tr><td colspan="1" rowspan="1">Enterococcus raffinosus</td><td colspan="1" rowspan="1">ATCC 49464</td></tr><tr><td colspan="1" rowspan="1">Enterococcus saccharolyticus</td><td colspan="1" rowspan="1">ATCC 43076</td></tr><tr><td colspan="1" rowspan="1">Enterococcus sanguinicola</td><td colspan="1" rowspan="1">DSM-21767</td></tr><tr><td colspan="1" rowspan="1">Enterococcus faecalis</td><td colspan="1" rowspan="1">cYanmaGYellowb</td></tr><tr><td colspan="1" rowspan="4">Enterococcus faecalis</td><td colspan="1" rowspan="1">ATCC 51575</td></tr><tr><td colspan="1" rowspan="1">ATCC 700802</td></tr><tr><td colspan="1" rowspan="1">ATCC 10100</td></tr><tr><td colspan="1" rowspan="1">ATCC 12399</td></tr><tr><td colspan="1" rowspan="7">Enterococcus faecalis</td><td colspan="1" rowspan="1">ATCC 14506</td></tr><tr><td colspan="1" rowspan="1">ATCC 33186</td></tr><tr><td colspan="1" rowspan="1">ATCC 49532</td></tr><tr><td colspan="1" rowspan="1">ATCC 49533</td></tr><tr><td colspan="1" rowspan="1">ATCC 7080</td></tr><tr><td colspan="1" rowspan="1">ATCC 49474</td></tr><tr><td colspan="1" rowspan="1">ATCC 49332</td></tr><tr><td colspan="1" rowspan="3">Enterococcus faecalis</td><td colspan="1" rowspan="1">ATCC 29200</td></tr><tr><td colspan="1" rowspan="1">ATCC BAA-2128</td></tr><tr><td colspan="1" rowspan="1">ATCC 51188</td></tr></table>

<table><tr><td colspan="1" rowspan="1">Organism</td><td colspan="1" rowspan="1">Strain</td></tr><tr><td colspan="1" rowspan="4"></td><td colspan="1" rowspan="1">ATCC 29887</td></tr><tr><td colspan="1" rowspan="1">ATCC 49461</td></tr><tr><td colspan="1" rowspan="1">ATCC 700565</td></tr><tr><td colspan="1" rowspan="1">ATCC 51625</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus lugdunensis</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="4">Staphylococcus lugdunensis</td><td colspan="1" rowspan="1">NRS-879</td></tr><tr><td colspan="1" rowspan="1">ATCC 49576</td></tr><tr><td colspan="1" rowspan="1">NRS-878</td></tr><tr><td colspan="1" rowspan="1">NRS-880</td></tr><tr><td colspan="1" rowspan="1">Streptococcus</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="7">Streptococcus agalactiae</td><td colspan="1" rowspan="1">ATCC 12401</td></tr><tr><td colspan="1" rowspan="1">ATCC 13813</td></tr><tr><td colspan="1" rowspan="1">ATCC 12386</td></tr><tr><td colspan="1" rowspan="1">ATCC 12973</td></tr><tr><td colspan="1" rowspan="1">ATCC 27956</td></tr><tr><td colspan="1" rowspan="1">NCTC-8017</td></tr><tr><td colspan="1" rowspan="1">ATCC 12403</td></tr><tr><td colspan="1" rowspan="3">Streptococcus anginosus</td><td colspan="1" rowspan="1">ATCC 9895</td></tr><tr><td colspan="1" rowspan="1">ATCC 33397</td></tr><tr><td colspan="1" rowspan="1">NCTC-10713</td></tr><tr><td colspan="1" rowspan="2">Streptococcus bovis</td><td colspan="1" rowspan="1">ATCC 33317</td></tr><tr><td colspan="1" rowspan="1">ATCC 35034</td></tr><tr><td colspan="1" rowspan="1">Streptococcus constellatus</td><td colspan="1" rowspan="1">ATCC 27513</td></tr><tr><td colspan="1" rowspan="1">Streptococcus criceti</td><td colspan="1" rowspan="1">ATCC 19642c</td></tr><tr><td colspan="1" rowspan="3">Streptococcus dysgalactiae</td><td colspan="1" rowspan="1">ATCC 35666</td></tr><tr><td colspan="1" rowspan="1">ATCC 43078</td></tr><tr><td colspan="1" rowspan="1">ATCC 12394</td></tr><tr><td colspan="1" rowspan="2">Streptococcus equi</td><td colspan="1" rowspan="1">ATCC 43079</td></tr><tr><td colspan="1" rowspan="1">ATCC 9528</td></tr><tr><td colspan="1" rowspan="2">Streptococcus equinus</td><td colspan="1" rowspan="1">ATCC 15351</td></tr><tr><td colspan="1" rowspan="1">ATCC 9812</td></tr><tr><td colspan="1" rowspan="3">Streptococcus gallolyticus</td><td colspan="1" rowspan="1">ATCC 49475</td></tr><tr><td colspan="1" rowspan="1">ATCC 9809</td></tr><tr><td colspan="1" rowspan="1">ATCC 43144</td></tr><tr><td colspan="1" rowspan="1">Streptococcus gordonii</td><td colspan="1" rowspan="1">ATCC 10558</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">ATCC 35557</td></tr><tr><td colspan="1" rowspan="2">Streptococcus infantarius</td><td colspan="1" rowspan="1">ATCC BAA-102</td></tr><tr><td colspan="1" rowspan="1">ATCC BAA-103</td></tr><tr><td colspan="1" rowspan="2">Streptococcus infantis</td><td colspan="1" rowspan="1">ATCC BAA-2089</td></tr><tr><td colspan="1" rowspan="1">ATCC 700779</td></tr><tr><td colspan="1" rowspan="1">Streptococcus intermedius</td><td colspan="1" rowspan="1">ATCC 27335</td></tr><tr><td colspan="1" rowspan="2">Streptococcus mitis</td><td colspan="1" rowspan="1">ATCC 15914</td></tr><tr><td colspan="1" rowspan="1">ATCC 6249</td></tr><tr><td colspan="1" rowspan="2">Streptococcus mitis</td><td colspan="1" rowspan="1">ATCC 49456</td></tr><tr><td colspan="1" rowspan="1">NCIMB-13770</td></tr><tr><td colspan="1" rowspan="2">Streptococcus oralisStreptococcus oralis</td><td colspan="1" rowspan="1">ATCC 35037</td></tr><tr><td colspan="1" rowspan="1">ATCC 55229</td></tr><tr><td colspan="1" rowspan="1">Streptococcus parasanguinis</td><td colspan="1" rowspan="1">ATCC 15909</td></tr><tr><td colspan="1" rowspan="1">Streptococcus peroris</td><td colspan="1" rowspan="1">ATCC 700780</td></tr><tr><td colspan="1" rowspan="4"></td><td colspan="1" rowspan="1">ATCC 49149</td></tr><tr><td colspan="1" rowspan="1">NCTC-775</td></tr><tr><td colspan="1" rowspan="1">ATCC 19433</td></tr><tr><td colspan="1" rowspan="1">ATCC 49452</td></tr><tr><td colspan="1" rowspan="1">Enterococcus faecalis (vanA+)</td><td colspan="1" rowspan="1">JMI 876745</td></tr><tr><td colspan="1" rowspan="2">Enterococcus faecalis (vanB+)</td><td colspan="1" rowspan="1">ATCC BAA-2365</td></tr><tr><td colspan="1" rowspan="1">ATCC 51299</td></tr><tr><td colspan="2" rowspan="1">Enterococcus faecium</td></tr><tr><td colspan="1" rowspan="14">Enterococcus faecium</td><td colspan="1" rowspan="1">ATCC 19434</td></tr><tr><td colspan="1" rowspan="1">ATCC 23828</td></tr><tr><td colspan="1" rowspan="1">ATCC 27273</td></tr><tr><td colspan="1" rowspan="1">ATCC BAA-2127</td></tr><tr><td colspan="1" rowspan="1">ATCC 6057</td></tr><tr><td colspan="1" rowspan="1">ATCC 49624</td></tr><tr><td colspan="1" rowspan="1">ATCC 6569</td></tr><tr><td colspan="1" rowspan="1">ATCC BAA-472</td></tr><tr><td colspan="1" rowspan="1">ATCC 9756</td></tr><tr><td colspan="1" rowspan="1">ATCC 49224</td></tr><tr><td colspan="1" rowspan="1">ATCC BAA-2846</td></tr><tr><td colspan="1" rowspan="1">ATCC 19953</td></tr><tr><td colspan="1" rowspan="1">ATCC 27270</td></tr><tr><td colspan="1" rowspan="1">ATCC 35667</td></tr><tr><td colspan="1" rowspan="2">Enterococcus faecium (vanA+)</td><td colspan="1" rowspan="1">ATCC BAA-2316</td></tr><tr><td colspan="1" rowspan="1">ATCC BAA-2317</td></tr><tr><td colspan="1" rowspan="11">Enterococcus faecium (vanA+)</td><td colspan="1" rowspan="1">ATCC 700221</td></tr><tr><td colspan="1" rowspan="1">ATCC BAA-2319</td></tr><tr><td colspan="1" rowspan="1">ATCC BAA-2320</td></tr><tr><td colspan="1" rowspan="1">LMC 003921</td></tr><tr><td colspan="1" rowspan="1">LMC 103676</td></tr><tr><td colspan="1" rowspan="1">LMC 089524</td></tr><tr><td colspan="1" rowspan="1">LMC 104266</td></tr><tr><td colspan="1" rowspan="1">LMC 032261</td></tr><tr><td colspan="1" rowspan="1">LMC 110371</td></tr><tr><td colspan="1" rowspan="1">LMC 002867</td></tr><tr><td colspan="1" rowspan="1">LMC 055971</td></tr><tr><td colspan="1" rowspan="2">Enterococcus faecium (vanA+)</td><td colspan="1" rowspan="1">ATCC 51559</td></tr><tr><td colspan="1" rowspan="1">ATCC BAA-2318</td></tr><tr><td colspan="1" rowspan="1">Enterococcus faecium (vanB+)</td><td colspan="1" rowspan="1">ATCC 51858</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus</td><td colspan="1" rowspan="1">cYanmaGellowb</td></tr><tr><td colspan="1" rowspan="2">Lactobacillus casei</td><td colspan="1" rowspan="1">ATCC 334</td></tr><tr><td colspan="1" rowspan="1">ATCC 55825</td></tr></table>

<table><tr><td colspan="1" rowspan="1">Organism</td><td colspan="1" rowspan="1">Strain</td></tr><tr><td colspan="1" rowspan="7">Streptococcus pneumoniae</td><td colspan="1" rowspan="1">ATCC BAA-475</td></tr><tr><td colspan="1" rowspan="1">ATCC 10357</td></tr><tr><td colspan="1" rowspan="1">ATCC 49619</td></tr><tr><td colspan="1" rowspan="1">ATCC 49136</td></tr><tr><td colspan="1" rowspan="1">ATCC BAA-659</td></tr><tr><td colspan="1" rowspan="1">ATCC 6315</td></tr><tr><td colspan="1" rowspan="1">ATCC BAA-1667</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">ATCC 700674</td></tr><tr><td colspan="1" rowspan="8"></td><td colspan="1" rowspan="1">ATCC 6321</td></tr><tr><td colspan="1" rowspan="1">ATCC 700673</td></tr><tr><td colspan="1" rowspan="1">ATCC 51916</td></tr><tr><td colspan="1" rowspan="1">ATCC 8338</td></tr><tr><td colspan="1" rowspan="1">ATCC 6301</td></tr><tr><td colspan="1" rowspan="1">ATCC 8335</td></tr><tr><td colspan="1" rowspan="1">NCIMB-13286</td></tr><tr><td colspan="1" rowspan="1">ATCC BAA-1656</td></tr><tr><td colspan="1" rowspan="6">Streptococcus pyogenes</td><td colspan="1" rowspan="1">ATCC 12384</td></tr><tr><td colspan="1" rowspan="1">ATCC 49399</td></tr><tr><td colspan="1" rowspan="1">ATCC 12344</td></tr><tr><td colspan="1" rowspan="1">ATCC 19615</td></tr><tr><td colspan="1" rowspan="1">NCIMB-13285</td></tr><tr><td colspan="1" rowspan="1">ATCC 14289</td></tr><tr><td colspan="1" rowspan="2">Streptococcus salivarius</td><td colspan="1" rowspan="1">ATCC 13419c</td></tr><tr><td colspan="1" rowspan="1">ATCC 7073</td></tr><tr><td colspan="1" rowspan="2">Streptococcus salivarius</td><td colspan="1" rowspan="1">ATCC 25975</td></tr><tr><td colspan="1" rowspan="1">ATCC 31067C</td></tr><tr><td colspan="1" rowspan="1">Streptococcus sanguinis</td><td colspan="1" rowspan="1">ATCC 10556</td></tr><tr><td colspan="1" rowspan="1">Streptococcus thoraltensis</td><td colspan="1" rowspan="1">ATCC 700865</td></tr><tr><td colspan="2" rowspan="1">Streptococcus agalactiae</td></tr><tr><td colspan="1" rowspan="7">Streptococcus agalactiae</td><td colspan="1" rowspan="1">ATCC 12401</td></tr><tr><td colspan="1" rowspan="1">ATCC 13813</td></tr><tr><td colspan="1" rowspan="1">ATCC 12386</td></tr><tr><td colspan="1" rowspan="1">ATCC 12973</td></tr><tr><td colspan="1" rowspan="1">ATCC 27956</td></tr><tr><td colspan="1" rowspan="1">NCTC-8017</td></tr><tr><td colspan="1" rowspan="1">ATCC 12403</td></tr><tr><td colspan="1" rowspan="1">Streptococcus anginosus group</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Streptococcus anginosus</td><td colspan="1" rowspan="1">ATCC 9895</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">ATCC 33397</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">NCTC-10713</td></tr><tr><td colspan="1" rowspan="1">Streptococcus constellatus</td><td colspan="1" rowspan="1">ATCC 27513</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">ATCC 39392</td></tr><tr><td colspan="1" rowspan="3">Lactobacillus paracasei</td><td colspan="1" rowspan="1">ATCC 25598</td></tr><tr><td colspan="1" rowspan="1">ATCC BAA-52</td></tr><tr><td colspan="1" rowspan="1">ATCC 27092</td></tr><tr><td colspan="1" rowspan="3">Lactobacillus rhamnosus</td><td colspan="1" rowspan="1">ATCC 39595</td></tr><tr><td colspan="1" rowspan="1">ATCC 53103</td></tr><tr><td colspan="1" rowspan="1">ATCC 55915</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus zeae</td><td colspan="1" rowspan="1">ATCC 15820</td></tr><tr><td colspan="1" rowspan="1">Listeria</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="2">Listeria innocua</td><td colspan="1" rowspan="1">NCTC-11288</td></tr><tr><td colspan="1" rowspan="1">ATCC 33090</td></tr><tr><td colspan="1" rowspan="3">Listeria ivanovii</td><td colspan="1" rowspan="1">ATCC 19119</td></tr><tr><td colspan="1" rowspan="1">ATCC BAA-139</td></tr><tr><td colspan="1" rowspan="1">ATCC 700402</td></tr><tr><td colspan="1" rowspan="9">Listeria monocytogenes</td><td colspan="1" rowspan="1">ATCC 10890</td></tr><tr><td colspan="1" rowspan="1">ATCC 19111</td></tr><tr><td colspan="1" rowspan="1">ATCC 13932</td></tr><tr><td colspan="1" rowspan="1">ATCC 19114</td></tr><tr><td colspan="1" rowspan="1">ATCC 7644</td></tr><tr><td colspan="1" rowspan="1">ATCC 19112</td></tr><tr><td colspan="1" rowspan="1">ATCC BAA-751</td></tr><tr><td colspan="1" rowspan="1">ATCC 19118</td></tr><tr><td colspan="1" rowspan="1">ATCC 19115</td></tr><tr><td colspan="1" rowspan="1">Listeria seeligeri</td><td colspan="1" rowspan="1">ATCC 35967</td></tr><tr><td colspan="1" rowspan="1">Listeria welshimeri</td><td colspan="1" rowspan="1">ATCC 35897</td></tr><tr><td colspan="1" rowspan="1">Listeria monocytogenes</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="3">Listeria monocytogenes</td><td colspan="1" rowspan="1">ATCC 10890</td></tr><tr><td colspan="1" rowspan="1">ATCC 19111</td></tr><tr><td colspan="1" rowspan="1">ATCC 13932</td></tr><tr><td colspan="1" rowspan="1">Listeria monocytogenes</td><td colspan="1" rowspan="1">ATCC 19114</td></tr><tr><td colspan="1" rowspan="5">Listeria monocytogenes</td><td colspan="1" rowspan="1">ATCC 7644</td></tr><tr><td colspan="1" rowspan="1">ATCC 19112</td></tr><tr><td colspan="1" rowspan="1">ATCC BAA-751</td></tr><tr><td colspan="1" rowspan="1">ATCC 19118</td></tr><tr><td colspan="1" rowspan="1">ATCC 19115</td></tr><tr><td colspan="2" rowspan="1">Micrococcus</td></tr><tr><td colspan="1" rowspan="6">Micrococcus luteus</td><td colspan="1" rowspan="1">ATCC 19212</td></tr><tr><td colspan="1" rowspan="1">ATCC 10240</td></tr><tr><td colspan="1" rowspan="1">ATCC 400</td></tr><tr><td colspan="1" rowspan="1">ATCC 53598</td></tr><tr><td colspan="1" rowspan="1">ATCC 49732</td></tr><tr><td colspan="1" rowspan="1">ATCC 4698</td></tr><tr><td colspan="1" rowspan="1">Micrococcus lylae</td><td colspan="1" rowspan="1">ATCC 27566</td></tr><tr><td colspan="1" rowspan="1">Micrococcus yunnanensis</td><td colspan="1" rowspan="1">ATCC 7468</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Staphylococcus arlettae</td><td colspan="1" rowspan="1">ATCC 43957</td></tr><tr><td colspan="1" rowspan="2">Staphylococcus aureus</td><td colspan="1" rowspan="1">ATCC 25923</td></tr><tr><td colspan="1" rowspan="1">ATCC 29247</td></tr></table>

<table><tr><td colspan="1" rowspan="1">Organism</td><td colspan="1" rowspan="1">Strain</td></tr><tr><td colspan="1" rowspan="1">Streptococcus intermedius</td><td colspan="1" rowspan="1">ATCC 27335</td></tr><tr><td colspan="1" rowspan="1">Streptococcus pneumoniae</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="6">Streptococcus pneumoniae</td><td colspan="1" rowspan="1">ATCC BAA-475</td></tr><tr><td colspan="1" rowspan="1">ATCC 10357</td></tr><tr><td colspan="1" rowspan="1">ATCC 49619</td></tr><tr><td colspan="1" rowspan="1">ATCC 49136</td></tr><tr><td colspan="1" rowspan="1">ATCC BAA-659</td></tr><tr><td colspan="1" rowspan="1">ATCC 6315</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">ATCC BAA-1667</td></tr><tr><td colspan="1" rowspan="9"></td><td colspan="1" rowspan="1">ATCC 700674</td></tr><tr><td colspan="1" rowspan="1">ATCC 6321</td></tr><tr><td colspan="1" rowspan="1">ATCC 700673</td></tr><tr><td colspan="1" rowspan="1">ATCC 51916</td></tr><tr><td colspan="1" rowspan="1">ATCC 8338</td></tr><tr><td colspan="1" rowspan="1">ATCC 6301</td></tr><tr><td colspan="1" rowspan="1">ATCC 8335</td></tr><tr><td colspan="1" rowspan="1">NCIMB-13286</td></tr><tr><td colspan="1" rowspan="1">ATCC BAA-1656</td></tr><tr><td colspan="1" rowspan="1">Streptococcus pyogenes</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="6">Streptococcus pyogenes</td><td colspan="1" rowspan="1">ATCC 12384</td></tr><tr><td colspan="1" rowspan="1">ATCC 49399</td></tr><tr><td colspan="1" rowspan="1">ATCC 12344</td></tr><tr><td colspan="1" rowspan="1">ATCC 19615</td></tr><tr><td colspan="1" rowspan="1">NCIMB-13285</td></tr><tr><td colspan="1" rowspan="1">ATCC 14289</td></tr><tr><td colspan="1" rowspan="1">Pan Candida</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="3">Candida albicans</td><td colspan="1" rowspan="1">ATCC 24433</td></tr><tr><td colspan="1" rowspan="1">ATCC MYA-4441</td></tr><tr><td colspan="1" rowspan="1">ATCC 14053</td></tr><tr><td colspan="1" rowspan="1">Candida albicans</td><td colspan="1" rowspan="1">ATCC 90028</td></tr><tr><td colspan="1" rowspan="4">Candida glabrata</td><td colspan="1" rowspan="1">ATCC 66032</td></tr><tr><td colspan="1" rowspan="1">ATCC 15126</td></tr><tr><td colspan="1" rowspan="1">ATCC MYA-2950</td></tr><tr><td colspan="1" rowspan="1">ATCC 2001</td></tr><tr><td colspan="1" rowspan="1">Candida krusei</td><td colspan="1" rowspan="1">ATCC 34135</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">ATCC 32196</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">ATCC 14243</td></tr><tr><td colspan="1" rowspan="3">Candida parapsilosis</td><td colspan="1" rowspan="1">ATCC 90018D</td></tr><tr><td colspan="1" rowspan="1">ATCC 58895</td></tr><tr><td colspan="1" rowspan="1">ATCC 22019</td></tr><tr><td colspan="1" rowspan="1">Pan Gram-Negative</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="2">Acinetobacter baumannii</td><td colspan="1" rowspan="1">NCTC-13302</td></tr><tr><td colspan="1" rowspan="1">NCTC-13422</td></tr><tr><td colspan="1" rowspan="1">Acinetobacter haemolyticus</td><td colspan="1" rowspan="1">ATCC 19002</td></tr><tr><td colspan="1" rowspan="1">Acinetobacter junii</td><td colspan="1" rowspan="1">ATCC 17908</td></tr><tr><td colspan="1" rowspan="1">Acinetobacter lwoffii</td><td colspan="1" rowspan="1">ATCC 15309</td></tr><tr><td colspan="1" rowspan="1">Acinetobacter ursingii</td><td colspan="1" rowspan="1">ATCC BAA-617</td></tr><tr><td colspan="1" rowspan="1">Bacteroides fragilis</td><td colspan="1" rowspan="1">NCTC-9343</td></tr><tr><td colspan="1" rowspan="10">−</td><td colspan="1" rowspan="1">ATCC 6538P</td></tr><tr><td colspan="1" rowspan="1">ATCC 29213</td></tr><tr><td colspan="1" rowspan="1">NR-45889</td></tr><tr><td colspan="1" rowspan="1">NR-45890</td></tr><tr><td colspan="1" rowspan="1">NR-46074</td></tr><tr><td colspan="1" rowspan="1">NR-45881</td></tr><tr><td colspan="1" rowspan="1">NR-46411</td></tr><tr><td colspan="1" rowspan="1">NR-46414</td></tr><tr><td colspan="1" rowspan="1">NR-46418</td></tr><tr><td colspan="1" rowspan="1">NR-46417</td></tr><tr><td colspan="1" rowspan="12">Staphylococcus aureus (mecA+)</td><td colspan="1" rowspan="1">NR-13524</td></tr><tr><td colspan="1" rowspan="1">NR-13527</td></tr><tr><td colspan="1" rowspan="1">NR-13525</td></tr><tr><td colspan="1" rowspan="1">NR-13526</td></tr><tr><td colspan="1" rowspan="1">NRS-123</td></tr><tr><td colspan="1" rowspan="1">NR-46805</td></tr><tr><td colspan="1" rowspan="1">NRS-662</td></tr><tr><td colspan="1" rowspan="1">ATCC BAA-1707</td></tr><tr><td colspan="1" rowspan="1">ATCC 43300</td></tr><tr><td colspan="1" rowspan="1">NRS-383</td></tr><tr><td colspan="1" rowspan="1">NCTC-12493</td></tr><tr><td colspan="1" rowspan="1">NRS-676</td></tr><tr><td colspan="1" rowspan="12">Staphylococcus aureus (mecA+)</td><td colspan="1" rowspan="1">NRS-678</td></tr><tr><td colspan="1" rowspan="1">NRS-648</td></tr><tr><td colspan="1" rowspan="1">NRS-651</td></tr><tr><td colspan="1" rowspan="1">NRS-643</td></tr><tr><td colspan="1" rowspan="1">NRS-484</td></tr><tr><td colspan="1" rowspan="1">NRS-385</td></tr><tr><td colspan="1" rowspan="1">NRS-384</td></tr><tr><td colspan="1" rowspan="1">ATCC BAA-40</td></tr><tr><td colspan="1" rowspan="1">ATCC 700698</td></tr><tr><td colspan="1" rowspan="1">NRS-382</td></tr><tr><td colspan="1" rowspan="1">NRS-659</td></tr><tr><td colspan="1" rowspan="1">NRS-657</td></tr><tr><td colspan="1" rowspan="7">Staphylococcus aureus (mecA+)</td><td colspan="1" rowspan="1">NRS-655</td></tr><tr><td colspan="1" rowspan="1">NRS-654</td></tr><tr><td colspan="1" rowspan="1">NRS-647</td></tr><tr><td colspan="1" rowspan="1">ATCC BAA-42</td></tr><tr><td colspan="1" rowspan="1">ATCC BAA-41</td></tr><tr><td colspan="1" rowspan="1">NRS-483</td></tr><tr><td colspan="1" rowspan="1">NRS-675</td></tr></table>

<table><tr><td colspan="1" rowspan="1">Organism</td><td colspan="1" rowspan="1">Strain</td></tr><tr><td colspan="1" rowspan="1">Burkholderia cepacia</td><td colspan="1" rowspan="1">ATCC 25416</td></tr><tr><td colspan="1" rowspan="1">Cedecea davisiae</td><td colspan="1" rowspan="1">ATCC 43025</td></tr><tr><td colspan="1" rowspan="1">Citrobacter freundii</td><td colspan="1" rowspan="1">NCTC-9750</td></tr><tr><td colspan="1" rowspan="1">Citrobacter koseri</td><td colspan="1" rowspan="1">ATCC 27156</td></tr><tr><td colspan="1" rowspan="1">Cronobacter muytjensii</td><td colspan="1" rowspan="1">ATCC 51329</td></tr><tr><td colspan="1" rowspan="1">Cronobacter sakazakii</td><td colspan="1" rowspan="1">ATCC 12868</td></tr><tr><td colspan="1" rowspan="1">Edwardsiella tarda</td><td colspan="1" rowspan="1">ATCC 15947</td></tr><tr><td colspan="1" rowspan="1">Enterobacter aerogenes</td><td colspan="1" rowspan="1">ATCC 13048</td></tr><tr><td colspan="1" rowspan="1">Enterobacter amnigenus</td><td colspan="1" rowspan="1">ATCC 51816</td></tr><tr><td colspan="1" rowspan="1">Enterobacter asburiae</td><td colspan="1" rowspan="1">ATCC 35953</td></tr><tr><td colspan="1" rowspan="4">Enterobacter cloacae</td><td colspan="1" rowspan="1">ATCC 13047</td></tr><tr><td colspan="1" rowspan="1">ATCC 23373</td></tr><tr><td colspan="1" rowspan="1">ATCC 23355</td></tr><tr><td colspan="1" rowspan="1">ATCC 35030</td></tr><tr><td colspan="1" rowspan="1">Enterobacter hormaechei</td><td colspan="1" rowspan="1">ATCC 700323</td></tr><tr><td colspan="1" rowspan="1">Erwinia aphidicola</td><td colspan="1" rowspan="1">ATCC 27991</td></tr><tr><td colspan="1" rowspan="4">Escherichia coli</td><td colspan="1" rowspan="1">ATCC 4157</td></tr><tr><td colspan="1" rowspan="1">ATCC 14948</td></tr><tr><td colspan="1" rowspan="1">ATCC 25922</td></tr><tr><td colspan="1" rowspan="1">ATCC BAA-196</td></tr><tr><td colspan="1" rowspan="1">Escherichia fergusonii</td><td colspan="1" rowspan="1">ATCC 35469</td></tr><tr><td colspan="1" rowspan="1">Escherichia hermannii</td><td colspan="1" rowspan="1">ATCC 700368</td></tr><tr><td colspan="1" rowspan="1">Escherichia vulneris</td><td colspan="1" rowspan="1">ATCC 33821</td></tr><tr><td colspan="1" rowspan="1">Fusobacterium necrophorum</td><td colspan="1" rowspan="1">ATCC 25286</td></tr><tr><td colspan="1" rowspan="1">Fusobacterium nucleatum</td><td colspan="1" rowspan="1">ATCC 25586</td></tr><tr><td colspan="1" rowspan="3">Haemophilus influenzae</td><td colspan="1" rowspan="1">ATCC 10211</td></tr><tr><td colspan="1" rowspan="1">ATCC 49144</td></tr><tr><td colspan="1" rowspan="1">ATCC 43065</td></tr><tr><td colspan="1" rowspan="1">Haemophilus influenzae</td><td colspan="1" rowspan="1">NCTC-8468</td></tr><tr><td colspan="1" rowspan="1">Haemophilus parainfluenzae</td><td colspan="1" rowspan="1">ATCC 9796</td></tr><tr><td colspan="1" rowspan="1">Hafnia alvei</td><td colspan="1" rowspan="1">ATCC 51815</td></tr><tr><td colspan="1" rowspan="3">Klebsiella oxytoca</td><td colspan="1" rowspan="1">ATCC 700324</td></tr><tr><td colspan="1" rowspan="1">ATCC 43165</td></tr><tr><td colspan="1" rowspan="1">ATCC 43863</td></tr><tr><td colspan="1" rowspan="1">Klebsiella ozaenae</td><td colspan="1" rowspan="1">ATCC 11296</td></tr><tr><td colspan="1" rowspan="4">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">ATCC BAA-1706</td></tr><tr><td colspan="1" rowspan="1">NCTC-9633</td></tr><tr><td colspan="1" rowspan="1">NCTC-13440</td></tr><tr><td colspan="1" rowspan="1">ATCC 51503</td></tr><tr><td colspan="1" rowspan="1">Kluyvera ascorbata</td><td colspan="1" rowspan="1">ATCC 14236</td></tr><tr><td colspan="1" rowspan="1">Kluyvera cryocrescens</td><td colspan="1" rowspan="1">ATCC 14240</td></tr><tr><td colspan="1" rowspan="10">−</td><td colspan="1" rowspan="1">NRS-645</td></tr><tr><td colspan="1" rowspan="1">NRS-687</td></tr><tr><td colspan="1" rowspan="1">NRS-667</td></tr><tr><td colspan="1" rowspan="1">NRS-677</td></tr><tr><td colspan="1" rowspan="1">NRS-683</td></tr><tr><td colspan="1" rowspan="1">NRS-688</td></tr><tr><td colspan="1" rowspan="1">NRS-22</td></tr><tr><td colspan="1" rowspan="1">NRS-387</td></tr><tr><td colspan="1" rowspan="1">NRS-70</td></tr><tr><td colspan="1" rowspan="1">ATCC 33591</td></tr><tr><td colspan="1" rowspan="2">Staphylococcus aureus (mecC+)</td><td colspan="1" rowspan="1">ATCC BAA-2312</td></tr><tr><td colspan="1" rowspan="1">ATCC BAA-2313</td></tr><tr><td colspan="1" rowspan="2">Staphylococcus auricularis</td><td colspan="1" rowspan="1">ATCC 33750</td></tr><tr><td colspan="1" rowspan="1">ATCC 33753</td></tr><tr><td colspan="1" rowspan="3">Staphylococcus capitis</td><td colspan="1" rowspan="1">ATCC 27840</td></tr><tr><td colspan="1" rowspan="1">NRS-866</td></tr><tr><td colspan="1" rowspan="1">ATCC 35661</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus caprae</td><td colspan="1" rowspan="1">ATCC 51548</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus carnosus</td><td colspan="1" rowspan="1">ATCC 51365</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus chromogenes</td><td colspan="1" rowspan="1">ATCC 43764</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus cohnii</td><td colspan="1" rowspan="1">ATCC 29974</td></tr><tr><td colspan="1" rowspan="5">Staphylococcus epidermidis</td><td colspan="1" rowspan="1">ATCC 12228</td></tr><tr><td colspan="1" rowspan="1">ATCC 49134</td></tr><tr><td colspan="1" rowspan="1">NCIMB-8853</td></tr><tr><td colspan="1" rowspan="1">ATCC 700583</td></tr><tr><td colspan="1" rowspan="1">ATCC 14990</td></tr><tr><td colspan="1" rowspan="6">Staphylococcus epidermidis(mecA+)</td><td colspan="1" rowspan="1">ATCC 35983</td></tr><tr><td colspan="1" rowspan="1">ATCC 35984</td></tr><tr><td colspan="1" rowspan="1">ATCC 29887</td></tr><tr><td colspan="1" rowspan="1">ATCC 49461</td></tr><tr><td colspan="1" rowspan="1">ATCC 700565</td></tr><tr><td colspan="1" rowspan="1">ATCC 51625</td></tr><tr><td colspan="1" rowspan="2">Staphylococcus gallinarum</td><td colspan="1" rowspan="1">ATCC 700401</td></tr><tr><td colspan="1" rowspan="1">ATCC 35539</td></tr><tr><td colspan="1" rowspan="2">Staphylococcus haemolyticus</td><td colspan="1" rowspan="1">ATCC 29970</td></tr><tr><td colspan="1" rowspan="1">ATCC 31874</td></tr><tr><td colspan="1" rowspan="3">Staphylococcus haemolyticus(mecA+)</td><td colspan="1" rowspan="1">NRS-62</td></tr><tr><td colspan="1" rowspan="1">ATCC BAA-1693</td></tr><tr><td colspan="1" rowspan="1">NRS-69</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus hominis</td><td colspan="1" rowspan="1">NRS-871</td></tr></table>

<table><tr><td colspan="1" rowspan="1">Organism</td><td colspan="1" rowspan="1">Strain</td></tr><tr><td colspan="1" rowspan="1">Legionella pneumophila</td><td colspan="1" rowspan="1">ATCC 33823</td></tr><tr><td colspan="1" rowspan="1">Moraxella catarrhalis</td><td colspan="1" rowspan="1">ATCC 23246</td></tr><tr><td colspan="1" rowspan="1">Moraxella nonliquefaciens</td><td colspan="1" rowspan="1">ATCC 17953</td></tr><tr><td colspan="1" rowspan="1">Moraxella osloensis</td><td colspan="1" rowspan="1">ATCC 10973</td></tr><tr><td colspan="1" rowspan="1">Morganella morganii</td><td colspan="1" rowspan="1">ATCC 25829</td></tr><tr><td colspan="1" rowspan="1">Neisseria lactamica</td><td colspan="1" rowspan="1">ATCC 23970</td></tr><tr><td colspan="1" rowspan="4">Neisseria meningitidis</td><td colspan="1" rowspan="1">ATCC 13090</td></tr><tr><td colspan="1" rowspan="1">ATCC 13113</td></tr><tr><td colspan="1" rowspan="1">ATCC 13077</td></tr><tr><td colspan="1" rowspan="1">ATCC 13102</td></tr><tr><td colspan="1" rowspan="1">Neisseria mucosa</td><td colspan="1" rowspan="1">ATCC 19695</td></tr><tr><td colspan="1" rowspan="1">Neisseria sicca</td><td colspan="1" rowspan="1">ATCC 29193</td></tr><tr><td colspan="1" rowspan="1">Pantoea agglomerans</td><td colspan="1" rowspan="1">ATCC 14537</td></tr><tr><td colspan="1" rowspan="1">Pasteurella aerogenes</td><td colspan="1" rowspan="1">ATCC 27883</td></tr><tr><td colspan="1" rowspan="3">Proteus mirabilis</td><td colspan="1" rowspan="1">ATCC 43071</td></tr><tr><td colspan="1" rowspan="1">CDC#0155</td></tr><tr><td colspan="1" rowspan="1">CDC#0159</td></tr><tr><td colspan="1" rowspan="1">Proteus penneri</td><td colspan="1" rowspan="1">ATCC 35197</td></tr><tr><td colspan="1" rowspan="1">Proteus vulgaris</td><td colspan="1" rowspan="1">ATCC 6380</td></tr><tr><td colspan="1" rowspan="1">Providencia alcalifaciens</td><td colspan="1" rowspan="1">ATCC 9886</td></tr><tr><td colspan="1" rowspan="1">Providencia rettgeri</td><td colspan="1" rowspan="1">ATCC 9919</td></tr><tr><td colspan="1" rowspan="1">Providencia stuartii</td><td colspan="1" rowspan="1">ATCC 33672</td></tr><tr><td colspan="1" rowspan="3">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">ATCC 15442</td></tr><tr><td colspan="1" rowspan="1">NCIMB-12469</td></tr><tr><td colspan="1" rowspan="1">ATCC 10145</td></tr><tr><td colspan="1" rowspan="1">Pseudomonas fluorescens</td><td colspan="1" rowspan="1">ATCC 13525</td></tr><tr><td colspan="1" rowspan="1">Pseudomonas luteola</td><td colspan="1" rowspan="1">ATCC 35970</td></tr><tr><td colspan="1" rowspan="1">Pseudomonas putida</td><td colspan="1" rowspan="1">ATCC 31483</td></tr><tr><td colspan="1" rowspan="1">Raoultella terrigena</td><td colspan="1" rowspan="1">ATCC 55553</td></tr><tr><td colspan="1" rowspan="1">Salmonella bongori</td><td colspan="1" rowspan="1">ATCC 43975</td></tr><tr><td colspan="1" rowspan="1">Salmonella enterica subsp.enterica</td><td colspan="1" rowspan="1">ATCC 51957</td></tr><tr><td colspan="1" rowspan="1">Salmonella entericahChleresuis</td><td colspan="1" rowspan="1">ATCC 12011</td></tr><tr><td colspan="1" rowspan="1">Salmonella enterica Paratyphi</td><td colspan="1" rowspan="1">FSL S5-0447</td></tr><tr><td colspan="1" rowspan="1">Salmonella typhi</td><td colspan="1" rowspan="1">ATCC 19430</td></tr><tr><td colspan="1" rowspan="1">Serratia fonticola</td><td colspan="1" rowspan="1">ATCC 29844</td></tr><tr><td colspan="1" rowspan="1">Serratia liquefaciens</td><td colspan="1" rowspan="1">ATCC 27592</td></tr><tr><td colspan="1" rowspan="3">Serratia marcescens</td><td colspan="1" rowspan="1">ATCC 8100</td></tr><tr><td colspan="1" rowspan="1">ATCC 14756</td></tr><tr><td colspan="1" rowspan="1">ATCC 13880</td></tr><tr><td colspan="1" rowspan="1">Serratia odorifera</td><td colspan="1" rowspan="1">ATCC 33077</td></tr><tr><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="1">NRS-870</td></tr><tr><td colspan="1" rowspan="1">ATCC 27844</td></tr><tr><td colspan="1" rowspan="2">Staphylococcus hominis(mecA+)</td><td colspan="1" rowspan="1">ATCC 700237</td></tr><tr><td colspan="1" rowspan="1">ATCC 700236</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus hyicus</td><td colspan="1" rowspan="1">ATCC 11249</td></tr><tr><td colspan="1" rowspan="2">Staphylococcus intermedius</td><td colspan="1" rowspan="1">ATCC 29663</td></tr><tr><td colspan="1" rowspan="1">ATCC 49052</td></tr></table>

A Detected at $2 \mathrm { ~ x ~ } 1 0 ^ { 8 } \mathrm { C F U / m L }$ B  Detected at $2 \mathrm { \times 1 0 ^ { 9 } C F U / m L }$ C  Detected at $4 \mathrm { \times 1 0 ^ { 8 } C F U / m L }$ D Detected at $1 \mathrm { ~ x ~ } 1 0 ^ { 7 } \mathrm { C F U / m L }$

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Strain</td></tr><tr><td rowspan=1 colspan=1>Serratia rubidaea</td><td rowspan=1 colspan=1>ATCC 27593</td></tr><tr><td rowspan=1 colspan=1>Shigella boydii</td><td rowspan=1 colspan=1>ATCC 9207</td></tr><tr><td rowspan=1 colspan=1>Shigella dysenteriae</td><td rowspan=1 colspan=1>ATCC 13313</td></tr><tr><td rowspan=1 colspan=1>Shigella sonnei</td><td rowspan=1 colspan=1>ATCC 29930</td></tr><tr><td rowspan=1 colspan=1>Stenotrophomonas maltophilia</td><td rowspan=1 colspan=1>ATCC 13636</td></tr><tr><td rowspan=1 colspan=1>Tatumella ptyseos</td><td rowspan=1 colspan=1>ATCC 33302</td></tr><tr><td rowspan=1 colspan=1>Yersinia enterocolitica subsp.Enterocolitica</td><td rowspan=1 colspan=1>ATCC 9610</td></tr></table>

# Predicted (in silico) Reactivity for Genus and Group Assays

Note: the performance of the ePlex BCID-GP Panel has not been established for all of the organisms listed in the tables below. See the Analytical Reactivity (Inclusivity) and Limit of Detection (Analytical Sensitivity) sections for data on organisms for which performance characteristics have been established (indicated with an asterisk in Tables 46-57). Some species were not assessed in silico due to lack of sequence data, though they may appear in the analytical sensitivity or specificity studies.

In addition to species-specific assays, the ePlex BCID-GP Panel contains a number of broader genus or group-level assays; including Bacillus cereus group, Bacillus subtilis group, Corynebacterium, Enterococcus, Lactobacillus, Listeria, Micrococcus, Staphylococcus, Streptococcus, Streptococcus anginosus group, Pan Candida and Pan Gram-Negative assays. Tables 46-57 summarize the predicted (in silico) reactivity (inclusivity) for these assay targets.

# Table 46: Predicted (in silico) Reactivity (Inclusivity) Results for Bacillus cereus group

<table><tr><td colspan="3">Detection Predicted for ≥95% of target sequences</td></tr><tr><td>Bacillus thuringiensis*</td><td>Bacillus toyonensis</td><td>Bacillus weihenstephanensis</td></tr><tr><td colspan="3">Detection Predicted for 85%-94% of target sequences</td></tr><tr><td>Bacillus cereus*</td><td></td><td></td></tr><tr><td></td><td>Detection Predicted for &lt;85% of target sequences</td><td></td></tr><tr><td colspan="3">None Identified</td></tr><tr><td></td><td>Detection Not Predicted</td><td></td></tr><tr><td>Bacillus mycoides*</td><td>Bacillus pseudomycoides</td><td></td></tr></table>

Table 47 Predicted (in silico) Reactivity (Inclusivity) Results for Bacillus subtilis group   

<table><tr><td rowspan=1 colspan=3>Detection Predicted for ≥95% of target sequences</td></tr><tr><td rowspan=1 colspan=1>Bacillus subtilis *</td><td rowspan=1 colspan=1>Bacillus licheniformis*</td><td rowspan=1 colspan=1>Bacillus siamensis</td></tr><tr><td rowspan=1 colspan=1>Bacillus amyloliquefaciens*</td><td rowspan=1 colspan=1>Bacillus methylotrophicus /Bacillus velezensis</td><td rowspan=1 colspan=1>Bacillus tequilensis</td></tr><tr><td rowspan=1 colspan=1>Bacillus atrophaeus*</td><td rowspan=1 colspan=1>Bacillus paralicheniformis</td><td rowspan=1 colspan=1>Bacillus vallismortis</td></tr><tr><td rowspan=1 colspan=3>Detection Predicted for 85%-94% of target sequences</td></tr><tr><td rowspan=1 colspan=3>None Identified</td></tr><tr><td rowspan=1 colspan=3>Detection Predicted for &lt;85% of target sequences</td></tr><tr><td rowspan=1 colspan=1>Bacillus mojavensis (77.8%)</td><td rowspan=1 colspan=1>Bacillus sonorensis (83.3%)</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=3>Detection Not Predicted</td></tr><tr><td rowspan=1 colspan=3>None Identified</td></tr></table>

Table 48: Predicted (in silico) Reactivity (Inclusivity) Results for Corynebacterium   

<table><tr><td colspan="3" rowspan="1">Detection Predicted for ≥95% of target sequences</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium afermentans</td><td colspan="1" rowspan="1">Corynebacterium glaucum</td><td colspan="1" rowspan="1">Corynebacterium renale</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium afermentanssubsp. afermentans*</td><td colspan="1" rowspan="1">Corynebacterium halotolerans</td><td colspan="1" rowspan="1">Corynebacterium simulans*</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium afermentanssubsp. lipophilium*</td><td colspan="1" rowspan="1">Corynebacterium hansenii</td><td colspan="1" rowspan="1">Corynebacterium singulare</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium appendicis</td><td colspan="1" rowspan="1">Corynebacterium humireducens</td><td colspan="1" rowspan="1">Corynebacterium sputi</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium auris</td><td colspan="1" rowspan="1">Corynebacterium imitans*</td><td colspan="1" rowspan="1">Corynebacterium striatum*</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium auriscanis</td><td colspan="1" rowspan="1">Corynebacterium lipophiloflavum</td><td colspan="1" rowspan="1">Corynebacterium suicordis</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium camporealensis</td><td colspan="1" rowspan="1">Corynebacterium lowii</td><td colspan="1" rowspan="1">Corynebacterium terpenotabidum</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium casei</td><td colspan="1" rowspan="1">Corynebacterium marinum</td><td colspan="1" rowspan="1">Corynebacterium testudinoris</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium ciconiae</td><td colspan="1" rowspan="1">Corynebacterium massiliense</td><td colspan="1" rowspan="1">Corynebacterium timonense*</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium coyleae*</td><td colspan="1" rowspan="1">Corynebacterium minutissimum*</td><td colspan="1" rowspan="1">Corynebacterium tuscaniense</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium doosanense</td><td colspan="1" rowspan="1">Corynebacterium mucifaciens</td><td colspan="1" rowspan="1">Corynebacterium ulceribovis</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium falsenii*</td><td colspan="1" rowspan="1">Corynebacterium mycetoides</td><td colspan="1" rowspan="1">Corynebacterium urealyticum*</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium felinum</td><td colspan="1" rowspan="1">Corynebacterium nuruki</td><td colspan="1" rowspan="1">Corynebacterium ureicelerivorans*</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium freneyi*</td><td colspan="1" rowspan="1">Corynebacterium oculi</td><td colspan="1" rowspan="1">Corynebacterium xerosis</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium genitalium</td><td colspan="1" rowspan="1">Corynebacterium pilosum</td><td colspan="1" rowspan="1">Y</td></tr><tr><td colspan="3" rowspan="1">Detection Predicted for 85%-94% of target sequences</td></tr><tr><td colspan="3" rowspan="1">None Identified</td></tr><tr><td colspan="3" rowspan="1">Detection Predicted for &lt;85% of target sequences</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium aurimucosum(50.0%)</td><td colspan="1" rowspan="1">Corynebacterium jeikeium(38.7%)*A</td><td colspan="1" rowspan="1">Corynebacterium variabile(75.0%)</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium diphtheriae(76.4%)*</td><td colspan="1" rowspan="1">Corynebacterium ulcerans(47.4%)*A</td><td colspan="1" rowspan="1">Corynebacterium vitaeruminis(75.0%)</td></tr><tr><td colspan="3" rowspan="1">Detection Not Predicted</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium accolens</td><td colspan="1" rowspan="1">Corynebacterium epidermidicanis</td><td colspan="1" rowspan="1">Corynebacterium propinquum</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium ammoniagenes</td><td colspan="1" rowspan="1">Corynebacterium flavescens</td><td colspan="1" rowspan="1">Corynebacterium pseudodiphtheriticum</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium amycolatum</td><td colspan="1" rowspan="1">Corynebacterium frankenforstense</td><td colspan="1" rowspan="1">Corynebacterium pseudogenitalium</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium aquilae</td><td colspan="1" rowspan="1">Corynebacterium freiburgense</td><td colspan="1" rowspan="1">Corynebacterium pseudotuberculosis</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium argentoratense</td><td colspan="1" rowspan="1">Corynebacterium glucuronolyticum</td><td colspan="1" rowspan="1">Corynebacterium pyruviciproducens</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium atypicum</td><td colspan="1" rowspan="1">Corynebacterium glutamicum</td><td colspan="1" rowspan="1">Corynebacterium resistens</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium bovis</td><td colspan="1" rowspan="1">Corynebacterium glyciniphilum</td><td colspan="1" rowspan="1">Corynebacterium riegelii</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium callunae</td><td colspan="1" rowspan="1">Corynebacterium kroppenstedtii</td><td colspan="1" rowspan="1">Corynebacterium sphenisci</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium capitovis</td><td colspan="1" rowspan="1">Corynebacterium kutscheri</td><td colspan="1" rowspan="1">Corynebacterium spheniscorum</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium caspium</td><td colspan="1" rowspan="1">Corynebacterium lactis</td><td colspan="1" rowspan="1">Corynebacterium stationis</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium confusum</td><td colspan="1" rowspan="1">Corynebacterium lubricantis</td><td colspan="1" rowspan="1">Corynebacterium sundsvallense</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium crenatum</td><td colspan="1" rowspan="1">Corynebacterium maris</td><td colspan="1" rowspan="1">Corynebacterium thomssenii</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium cystitidis</td><td colspan="1" rowspan="1">Corynebacterium mastitidis</td><td colspan="1" rowspan="1">Corynebacterium tuberculostearicum</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium deserti</td><td colspan="1" rowspan="1">Corynebacterium matruchotii</td><td colspan="1" rowspan="1">Corynebacterium uropygiale</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium durum</td><td colspan="1" rowspan="1">Corynebacterium mustelae</td><td colspan="1" rowspan="1">Corynebacterium uterequi</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium efficiens</td><td colspan="1" rowspan="1">Corynebacterium phocae</td><td colspan="1" rowspan="1"></td></tr></table>

A. $3 8 . 7 \%$ of sequences in NCBI for $C$ . jeikeium were predicted to be detected bioinformatically; $4 7 . 4 \%$ of sequences in NCBI for C. ulcerans were predicted to be detected bioinformatically. All strains tested for these species were detected as a part of the Analytical Reactivity (Inclusivity) or Limit of Detection (Analytical Sensitivity) studies.

Table 49: Predicted (in silico) Reactivity (Inclusivity) Results for Enterococcus   

<table><tr><td rowspan=1 colspan=3>Detection Predicted for ≥95% of target sequences</td></tr><tr><td rowspan=1 colspan=1>Enterococcus avium*</td><td rowspan=1 colspan=1>Enterococcus flavescens</td><td rowspan=1 colspan=1>Enterococcus raffinosus</td></tr><tr><td rowspan=1 colspan=1>Enterococcus dispar*</td><td rowspan=1 colspan=1>Enterococcus italicus*</td><td rowspan=1 colspan=1>Enterococcus saccharolyticus*</td></tr><tr><td rowspan=1 colspan=1>Enterococcus durans</td><td rowspan=1 colspan=1>Enterococcus lactis</td><td rowspan=1 colspan=1>Enterococcus thailandicus</td></tr><tr><td rowspan=1 colspan=1>Enterococcus faecalis*</td><td rowspan=1 colspan=1>Enterococcus malodoratus</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Enterococcus faecium*</td><td rowspan=1 colspan=1>Enterococcus pseudoavium</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=3>Detection Predicted for 85%-94% of target sequences</td></tr><tr><td rowspan=1 colspan=1>Enterococcus casseliflavus</td><td rowspan=1 colspan=1>Enterococcus gallinarum</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Enterococcus cecorum</td><td rowspan=1 colspan=1>Enterococcus hirae*</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=3>Detection Predicted for &lt;85% of target sequences</td></tr><tr><td rowspan=1 colspan=1>Enterococcus mundtii (9.1%)</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=2>Detection Not Predicted</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Enterococcus aquimarinus</td><td rowspan=1 colspan=1>Enterococcus hawaiiensis</td><td rowspan=1 colspan=1>Enterococcus rotai</td></tr><tr><td rowspan=1 colspan=1>Enterococcus asini</td><td rowspan=1 colspan=1>Enterococcus hermanniensis</td><td rowspan=1 colspan=1>Enterococcus silesiacus</td></tr><tr><td rowspan=1 colspan=1>Enterococcus caccae</td><td rowspan=1 colspan=1>Enterococcus pallens</td><td rowspan=1 colspan=1>Enterococcus sulfureus</td></tr><tr><td rowspan=1 colspan=1>Enterococcus camelliae</td><td rowspan=1 colspan=1>Enterococcus pernyi</td><td rowspan=1 colspan=1>Enterococcus termitis</td></tr><tr><td rowspan=1 colspan=1>Enterococcus canintestini</td><td rowspan=1 colspan=1>Enterococcus phoeniculicola</td><td rowspan=1 colspan=1>Enterococcus ureasiticus</td></tr><tr><td rowspan=1 colspan=1>Enterococcus canis</td><td rowspan=1 colspan=1>Enterococcus plantarum</td><td rowspan=1 colspan=1>Enterococcus ureilyticus</td></tr><tr><td rowspan=1 colspan=1>Enterococcus columbae</td><td rowspan=1 colspan=1>Enterococcus quebecensis</td><td rowspan=1 colspan=1>Enterococcus villorum</td></tr><tr><td rowspan=1 colspan=1>Enterococcus devriesei</td><td rowspan=1 colspan=1>Enterococcus ratti</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Enterococcus haemoperoxidus</td><td rowspan=1 colspan=1>Enterococcus rivorum</td><td rowspan=1 colspan=1></td></tr></table>

Table 50 Predicted (in silico) Reactivity (Inclusivity) Results for Lactobacillus   

<table><tr><td colspan="3" rowspan="1">Detection Predicted for ≥95% of target sequences</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus casei*</td><td colspan="1" rowspan="1">Lactobacillus rhamnosus*</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Lactobacillus paracasei*</td><td colspan="1" rowspan="1">Lactobacillus zeae*</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="3" rowspan="1">Detection Predicted for 85%-94% of target sequences</td></tr><tr><td colspan="3" rowspan="1">None Identified</td></tr><tr><td colspan="3" rowspan="1">Detection Predicted for &lt;85% of target sequences</td></tr><tr><td colspan="3" rowspan="1">None Identified</td></tr><tr><td colspan="3" rowspan="1">Detection Not Predicted</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus acetotolerans</td><td colspan="1" rowspan="1">Lactobacillus ghanensis</td><td colspan="1" rowspan="1">Lactobacillus panis</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus acidifarinae</td><td colspan="1" rowspan="1">Lactobacillus gigeriorum</td><td colspan="1" rowspan="1">Lactobacillus pantheris</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus acidipiscis</td><td colspan="1" rowspan="1">Lactobacillusginsenosidimutans</td><td colspan="1" rowspan="1">Lactobacillus parabrevis</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus acidophilus</td><td colspan="1" rowspan="1">Lactobacillus gorillae</td><td colspan="1" rowspan="1">Lactobacillus parabuchneri</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus agilis</td><td colspan="1" rowspan="1">Lactobacillus graminis</td><td colspan="1" rowspan="1">Lactobacillus paracollinoides</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus algidus</td><td colspan="1" rowspan="1">Lactobacillus hammesii</td><td colspan="1" rowspan="1">Lactobacillus parafarraginis</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus alimentarius</td><td colspan="1" rowspan="1">Lactobacillus hamsteri</td><td colspan="1" rowspan="1">Lactobacillus parakefiri</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus amylolyticus</td><td colspan="1" rowspan="1">Lactobacillus harbinensis</td><td colspan="1" rowspan="1">Lactobacillus paralimentarius</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus amylophilus</td><td colspan="1" rowspan="1">Lactobacillus hayakitensis</td><td colspan="1" rowspan="1">Lactobacillus paraplantarum</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus amylotrophicus</td><td colspan="1" rowspan="1">Lactobacillus heilongjiangensis</td><td colspan="1" rowspan="1">Lactobacillus pasteurii</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus amylovorus</td><td colspan="1" rowspan="1">Lactobacillus helsingborgensis</td><td colspan="1" rowspan="1">Lactobacillus paucivorans</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus animalis</td><td colspan="1" rowspan="1">Lactobacillus helveticus</td><td colspan="1" rowspan="1">Lactobacillus pentosus</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus antri</td><td colspan="1" rowspan="1">Lactobacillus herbarum</td><td colspan="1" rowspan="1">Lactobacillus perolens</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus apinorum</td><td colspan="1" rowspan="1">Lactobacillus hilgardii</td><td colspan="1" rowspan="1">Lactobacillus plantarum</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus apis</td><td colspan="1" rowspan="1">Lactobacillus hokkaidonensis</td><td colspan="1" rowspan="1">Lactobacillus pobuzihii</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus apodemi</td><td colspan="1" rowspan="1">Lactobacillus hominis</td><td colspan="1" rowspan="1">Lactobacillus pontis</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus aquaticus</td><td colspan="1" rowspan="1">Lactobacillus homohiochii</td><td colspan="1" rowspan="1">Lactobacillus psittaci</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus aviarius</td><td colspan="1" rowspan="1">Lactobacillus hordei</td><td colspan="1" rowspan="1">Lactobacillus rapi</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus backii</td><td colspan="1" rowspan="1">Lactobacillus iners</td><td colspan="1" rowspan="1">Lactobacillus rennini</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus bifermentans</td><td colspan="1" rowspan="1">Lactobacillus ingluviei</td><td colspan="1" rowspan="1">Lactobacillus reuteri</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus bombi</td><td colspan="1" rowspan="1">Lactobacillus intestinalis</td><td colspan="1" rowspan="1">Lactobacillus rodentium</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus bombicola</td><td colspan="1" rowspan="1">Lactobacillus jensenii</td><td colspan="1" rowspan="1">Lactobacillus rossiae</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus brantae</td><td colspan="1" rowspan="1">Lactobacillus johnsonii</td><td colspan="1" rowspan="1">Lactobacillus ruminis</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus brevis</td><td colspan="1" rowspan="1">Lactobacillus kalixensis</td><td colspan="1" rowspan="1">Lactobacillus saerimneri</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus buchneri</td><td colspan="1" rowspan="1">Lactobacillus kefiranofaciens</td><td colspan="1" rowspan="1">Lactobacillus sakei</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus cacaonum</td><td colspan="1" rowspan="1">Lactobacillus kefiri</td><td colspan="1" rowspan="1">Lactobacillus salivarius</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus camelliae</td><td colspan="1" rowspan="1">Lactobacillus kimbladii</td><td colspan="1" rowspan="1">Lactobacillus sanfranciscensis</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus capillatus</td><td colspan="1" rowspan="1">Lactobacillus kimchicus</td><td colspan="1" rowspan="1">Lactobacillus saniviri</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus ceti</td><td colspan="1" rowspan="1">Lactobacillus kimchiensis</td><td colspan="1" rowspan="1">Lactobacillus satsumensis</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus coleohominis</td><td colspan="1" rowspan="1">Lactobacillus kisonensis</td><td colspan="1" rowspan="1">Lactobacillus secaliphilus</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus collinoides</td><td colspan="1" rowspan="1">Lactobacillus kitasatonis</td><td colspan="1" rowspan="1">Lactobacillus selangorensis</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus composti</td><td colspan="1" rowspan="1">Lactobacillus koreensis</td><td colspan="1" rowspan="1">Lactobacillus senioris</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus concavus</td><td colspan="1" rowspan="1">Lactobacillus kullabergensis</td><td colspan="1" rowspan="1">Lactobacillus senmaizukei</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus coryniformis</td><td colspan="1" rowspan="1">Lactobacillus kunkeei</td><td colspan="1" rowspan="1">Lactobacillus sharpeae</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus crispatus</td><td colspan="1" rowspan="1">Lactobacillus letivazi</td><td colspan="1" rowspan="1">Lactobacillus shenzhenensis</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus crustorum</td><td colspan="1" rowspan="1">Lactobacillus lindneri</td><td colspan="1" rowspan="1">Lactobacillus silagei</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus curieae</td><td colspan="1" rowspan="1">Lactobacillus malefermentans</td><td colspan="1" rowspan="1">Lactobacillus siliginis</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus curvatus</td><td colspan="1" rowspan="1">Lactobacillus mali</td><td colspan="1" rowspan="1">Lactobacillus similis</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus delbrueckii</td><td colspan="1" rowspan="1">Lactobacillus manihotivorans</td><td colspan="1" rowspan="1">Lactobacillus spicheri</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus dextrinicus</td><td colspan="1" rowspan="1">Lactobacillus mellifer</td><td colspan="1" rowspan="1">Lactobacillus sucicola</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus diolivorans</td><td colspan="1" rowspan="1">Lactobacillus mellis</td><td colspan="1" rowspan="1">Lactobacillus suebicus</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus equi</td><td colspan="1" rowspan="1">Lactobacillus melliventris</td><td colspan="1" rowspan="1">Lactobacillus sunkii</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus equicursoris</td><td colspan="1" rowspan="1">Lactobacillus mindensis</td><td colspan="1" rowspan="1">Lactobacillus suntoryeus</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus equigenerosi</td><td colspan="1" rowspan="1">Lactobacillus mucosae</td><td colspan="1" rowspan="1">Lactobacillus taiwanensis</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus fabifermentans</td><td colspan="1" rowspan="1">Lactobacillus murinus</td><td colspan="1" rowspan="1">Lactobacillus thailandensis</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus farciminis</td><td colspan="1" rowspan="1">Lactobacillus nagelii</td><td colspan="1" rowspan="1">Lactobacillus tucceti</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus farraginis</td><td colspan="1" rowspan="1">Lactobacillus namurensis</td><td colspan="1" rowspan="1">Lactobacillus ultunensis</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus fermentum</td><td colspan="1" rowspan="1">Lactobacillus nantensis</td><td colspan="1" rowspan="1">Lactobacillus uvarum</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus floricola</td><td colspan="1" rowspan="1">Lactobacillus nasuensis</td><td colspan="1" rowspan="1">Lactobacillus vaccinostercus</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus florum</td><td colspan="1" rowspan="1">Lactobacillus nodensis</td><td colspan="1" rowspan="1">Lactobacillus vaginalis</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus fructivorans</td><td colspan="1" rowspan="1">Lactobacillus odoratitofui</td><td colspan="1" rowspan="1">Lactobacillus versmoldensis</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus frumenti</td><td colspan="1" rowspan="1">Lactobacillus oeni</td><td colspan="1" rowspan="1">Lactobacillus vini</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus fuchuensis</td><td colspan="1" rowspan="1">Lactobacillus oligofermentans</td><td colspan="1" rowspan="1">Lactobacillus wasatchensis</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus futsaii</td><td colspan="1" rowspan="1">Lactobacillus oris</td><td colspan="1" rowspan="1">Lactobacillus xiangfangensis</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus gallinarum</td><td colspan="1" rowspan="1">Lactobacillus oryzae</td><td colspan="1" rowspan="1">Lactobacillus zymae</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus gasseri</td><td colspan="1" rowspan="1">Lactobacillus otakiensis</td><td colspan="1" rowspan="1">Y</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus gastricus</td><td colspan="1" rowspan="1">Lactobacillus ozensis</td><td colspan="1" rowspan="1">Y</td></tr></table>

Table 51: Predicted (in silico) Reactivity (Inclusivity) Results for Listeria   

<table><tr><td rowspan=1 colspan=3>Detection Predicted for ≥95% of target sequences</td></tr><tr><td rowspan=1 colspan=1>Listeria monocytogenes*</td><td rowspan=1 colspan=1>Listeria marthii</td><td rowspan=1 colspan=1>Listeria welshimeri*</td></tr><tr><td rowspan=1 colspan=1>Listeria ivanovii*</td><td rowspan=1 colspan=1>Listeria seeligeri*</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=3>Detection Predicted for 85%-94% of target sequences</td></tr><tr><td rowspan=1 colspan=1>Listeria innocua*</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=3>Detection Predicted for &lt;85% of target sequences</td></tr><tr><td rowspan=1 colspan=3>None Identified</td></tr><tr><td rowspan=1 colspan=3>Detection Not Predicted</td></tr><tr><td rowspan=1 colspan=1>Listeria grayi</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr></table>

Table 52: Predicted (in silico) Reactivity (Inclusivity) Results for Micrococcus   

<table><tr><td rowspan=1 colspan=3>Detection Predicted for ≥95% of target sequences</td></tr><tr><td rowspan=1 colspan=1>Micrococcus luteus*</td><td rowspan=1 colspan=1>Micrococcus aquilus</td><td rowspan=1 colspan=1>Micrococcus yunnanensis *</td></tr><tr><td rowspan=1 colspan=1>Micrococcus alkanovora</td><td rowspan=1 colspan=1>Micrococcus flavus</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Micrococcus aloeverae</td><td rowspan=1 colspan=1>Micrococcus thailandicus</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=3>Detection Predicted for 85%-94% of target sequences</td></tr><tr><td rowspan=1 colspan=1>Micrococcus endophyticus</td><td rowspan=1 colspan=1>Micrococcus indicus</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=3>Detection Predicted for &lt;85% of target sequences</td></tr><tr><td rowspan=1 colspan=1>Micrococcus lylae (50.0%)*</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=3>Detection Not Predicted</td></tr><tr><td rowspan=1 colspan=1>Micrococcus antarcticus</td><td rowspan=1 colspan=1>Micrococcus lactis</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Micrococcus chenggongense</td><td rowspan=1 colspan=1>Micrococcus terreus</td><td rowspan=1 colspan=1></td></tr></table>

Table 53: Predicted (in silico) Reactivity (Inclusivity) Results for Staphylococcus   

<table><tr><td rowspan=1 colspan=3>Detection Predicted for ≥95% of target sequences</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus aureus*</td><td rowspan=1 colspan=1>Staphylococcus haemolyticus*</td><td rowspan=1 colspan=1>Staphylococcus pseudintermedius*</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus agnetis</td><td rowspan=1 colspan=1>Staphylococcus hominis*</td><td rowspan=1 colspan=1>Staphylococcus pseudolugdunensis</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus argensis</td><td rowspan=1 colspan=1>Staphylococcus hominis subsp.novobiosepticus</td><td rowspan=1 colspan=1>Staphylococcus pulvereri</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus argenteus</td><td rowspan=1 colspan=1>Staphylococcus hyicus*</td><td rowspan=1 colspan=1>Staphylococcus rostri</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus auricularis *</td><td rowspan=1 colspan=1>Staphylococcus jettensis</td><td rowspan=1 colspan=1>Staphylococcus saprophyticus*</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus capitis*</td><td rowspan=1 colspan=1>Staphylococcus kloosii</td><td rowspan=1 colspan=1>Staphylococcus schleiferi*</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus caprae*</td><td rowspan=1 colspan=1>Staphylococcus lentus*</td><td rowspan=1 colspan=1>Staphylococcus schweitzeri</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus carnosus*</td><td rowspan=1 colspan=1>Staphylococcus lugdunensis*</td><td rowspan=1 colspan=1>Staphylococcus sciuri*</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus chromogenes*</td><td rowspan=1 colspan=1>Staphylococcus lutrae</td><td rowspan=1 colspan=1>Staphylococcus simiae</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus cohnii*</td><td rowspan=1 colspan=1>Staphylococcus massiliensis</td><td rowspan=1 colspan=1>Staphylococcus simulans*</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus delphini</td><td rowspan=1 colspan=1>Staphylococcus microti</td><td rowspan=1 colspan=1>Staphylococcus stepanovicii</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus devriesei</td><td rowspan=1 colspan=1>Staphylococcus muscae*</td><td rowspan=1 colspan=1>Staphylococcus succinus</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus epidermidis*</td><td rowspan=1 colspan=1>Staphylococcus nepalensis</td><td rowspan=1 colspan=1>Staphylococcus vitulinus*</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus equorum</td><td rowspan=1 colspan=1>Staphylococcus pasteuri*</td><td rowspan=1 colspan=1>Staphylococcus warneri *</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus felis</td><td rowspan=1 colspan=1>Staphylococcus petrasii</td><td rowspan=1 colspan=1>Staphylococcus xylosus*</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus fleurettii</td><td rowspan=1 colspan=1>Staphylococcus pettenkoferi*</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Staphylococcus gallinarum*</td><td rowspan=1 colspan=1>Staphylococcus piscifermentans</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=3>Detection Predicted for 85%-94% of target sequences</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus arlettae</td><td rowspan=1 colspan=1>Staphylococcus intermedius*</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Staphylococcus condimenti</td><td rowspan=1 colspan=1>Staphylococcus saccharolyticus *</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=3>Detection Predicted for &lt;85% of target sequences</td></tr><tr><td rowspan=1 colspan=3>None Identified</td></tr><tr><td rowspan=1 colspan=3>Detection Not Predicted</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus caseolyticusA</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr></table>

A Data for only one strain available in NCBI.

Table 54: Predicted (in silico) Reactivity (Inclusivity) Results for Streptococcus   

<table><tr><td rowspan=1 colspan=3>Detection Predicted for ≥95% of target sequences</td></tr><tr><td rowspan=1 colspan=1>Streptococcus agalactiae*</td><td rowspan=1 colspan=1>Streptococcus infantarius*</td><td rowspan=1 colspan=1>Streptococcus phocae</td></tr><tr><td rowspan=1 colspan=1>Streptococcus alactolyticus</td><td rowspan=1 colspan=1>Streptococcus infantis *</td><td rowspan=1 colspan=1>Streptococcus pneumoniae*</td></tr><tr><td rowspan=1 colspan=1>Streptococcus anginosus*</td><td rowspan=1 colspan=1>Streptococcus intermedius*</td><td rowspan=1 colspan=1>Streptococcus porcinus</td></tr><tr><td rowspan=1 colspan=1>Streptococcus australis</td><td rowspan=1 colspan=1>Streptococcus intestinalis</td><td rowspan=1 colspan=1>Streptococcus porcorum</td></tr><tr><td rowspan=1 colspan=1>Streptococcus caballi</td><td rowspan=1 colspan=1>Streptococcus lactarius</td><td rowspan=1 colspan=1>Streptococcus pseudopneumoniae</td></tr><tr><td rowspan=1 colspan=1>Streptococcus constellatus*</td><td rowspan=1 colspan=1>Streptococcus loxodontasalivarius</td><td rowspan=1 colspan=1>Streptococcus pseudoporcinus</td></tr><tr><td rowspan=1 colspan=1>Streptococcus criceti*</td><td rowspan=1 colspan=1>Streptococcus luteciae</td><td rowspan=1 colspan=1>Streptococcus pyogenes*</td></tr></table>

Confidential

<table><tr><td rowspan=1 colspan=1>Streptococcus cristatus</td><td rowspan=1 colspan=1>Streptococcus lutetiensis</td><td rowspan=1 colspan=1>Streptococcus rifensis</td></tr><tr><td rowspan=1 colspan=1>Streptococcus danieliae</td><td rowspan=1 colspan=1>Streptococcus macedonicus</td><td rowspan=1 colspan=1>Streptococcus rubneri</td></tr><tr><td rowspan=1 colspan=1>Streptococcus dentasini</td><td rowspan=1 colspan=1>Streptococcus marimammalium</td><td rowspan=1 colspan=1>Streptococcus salivarius*</td></tr><tr><td rowspan=1 colspan=1>Streptococcus dentisani</td><td rowspan=1 colspan=1>Streptococcus massiliensis</td><td rowspan=1 colspan=1>Streptococcus saliviloxodontae</td></tr><tr><td rowspan=1 colspan=1>Streptococcus didelphis</td><td rowspan=1 colspan=1>Streptococcus mitis*</td><td rowspan=1 colspan=1>Streptococcus sanguinis*</td></tr><tr><td rowspan=1 colspan=1>Streptococcus difficilis</td><td rowspan=1 colspan=1>Streptococcus moroccensis</td><td rowspan=1 colspan=1>Streptococcus seminale</td></tr><tr><td rowspan=1 colspan=1>Streptococcus dysgalactiaesubsp. dysgalactiae</td><td rowspan=1 colspan=1>Streptococcus oligofermentans</td><td rowspan=1 colspan=1>Streptococcus sinensis</td></tr><tr><td rowspan=1 colspan=1>Streptococcus dysgalactiaesubsp. equisimilis</td><td rowspan=1 colspan=1>Streptococcus oralis*</td><td rowspan=1 colspan=1>Streptococcus suis</td></tr><tr><td rowspan=1 colspan=1>Streptococcus dysgalactiae*</td><td rowspan=1 colspan=1>Streptococcus oricebi</td><td rowspan=1 colspan=1>Streptococcus thermophilus</td></tr><tr><td rowspan=1 colspan=1>Streptococcus equi*</td><td rowspan=1 colspan=1>Streptococcus orisratti</td><td rowspan=1 colspan=1>Streptococcus thoraltensis*</td></tr><tr><td rowspan=1 colspan=1>Streptococcus equinus*</td><td rowspan=1 colspan=1>Streptococcus panodentis</td><td rowspan=1 colspan=1>Streptococcus tigurinus</td></tr><tr><td rowspan=1 colspan=1>Streptococcus fryi</td><td rowspan=1 colspan=1>Streptococcus parasanguinis*</td><td rowspan=1 colspan=1>Streptococcus troglodytae</td></tr><tr><td rowspan=1 colspan=1>Streptococcus gallolyticus*</td><td rowspan=1 colspan=1>Streptococcus parasuis</td><td rowspan=1 colspan=1>Streptococcus troglodytidis</td></tr><tr><td rowspan=1 colspan=1>Streptococcus gordonii*</td><td rowspan=1 colspan=1>Streptococcus parauberis</td><td rowspan=1 colspan=1>Streptococcus urinalis</td></tr><tr><td rowspan=1 colspan=1>Streptococcus himalayensis</td><td rowspan=1 colspan=1>Streptococcus pasteuri</td><td rowspan=1 colspan=1>Streptococcus ursoris</td></tr><tr><td rowspan=1 colspan=1>Streptococcus hongkongensis</td><td rowspan=1 colspan=1>Streptococcus pasteurianus</td><td rowspan=1 colspan=1>Streptococcus vestibularis</td></tr><tr><td rowspan=1 colspan=1>Streptococcus hyointestinalis</td><td rowspan=1 colspan=1>Streptococcus peroris*</td><td rowspan=1 colspan=1>Streptococcus waiu</td></tr><tr><td rowspan=1 colspan=3>Detection Predicted for 85%-94% of target sequences</td></tr><tr><td rowspan=1 colspan=1>Streptococcus bovis*</td><td rowspan=1 colspan=1>Streptococcus uberis</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=3>Detection Predicted for &lt;85% of target sequences</td></tr><tr><td rowspan=1 colspan=1>Streptococcus canis (15.8%)</td><td rowspan=1 colspan=1>Streptococcus henryi (10.0%)</td><td rowspan=1 colspan=1>Streptococcus ratti (75.0%)</td></tr><tr><td rowspan=1 colspan=1>Streptococcus halichoeri(66.7%)</td><td rowspan=1 colspan=1>Streptococcus iniae (6.9%)</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=3>Detection Not Predicted</td></tr><tr><td rowspan=1 colspan=1>Streptococcus acidominimus</td><td rowspan=1 colspan=1>Streptococcus entericus</td><td rowspan=1 colspan=1>Streptococcus orisasini</td></tr><tr><td rowspan=1 colspan=1>Streptococcus azizii</td><td rowspan=1 colspan=1>Streptococcus ferus</td><td rowspan=1 colspan=1>Streptococcus orisuis</td></tr><tr><td rowspan=1 colspan=1>Streptococcus cameli</td><td rowspan=1 colspan=1>Streptococcus gallinaceus</td><td rowspan=1 colspan=1>Streptococcus ovis</td></tr><tr><td rowspan=1 colspan=1>Streptococcus castoreus</td><td rowspan=1 colspan=1>Streptococcus hyovaginalis</td><td rowspan=1 colspan=1>Streptococcus pharyngis</td></tr><tr><td rowspan=1 colspan=1>Streptococcus cremoris</td><td rowspan=1 colspan=1>Streptococcus ictaluri</td><td rowspan=1 colspan=1>Streptococcus pluranimalium</td></tr><tr><td rowspan=1 colspan=1>Streptococcus criae</td><td rowspan=1 colspan=1>Streptococcus lactis</td><td rowspan=1 colspan=1>Streptococcus plurextorum</td></tr><tr><td rowspan=1 colspan=1>Streptococcus cuniculi</td><td rowspan=1 colspan=1>Streptococcus macacae</td><td rowspan=1 colspan=1>Streptococcus plutanimalium</td></tr><tr><td rowspan=1 colspan=1>Streptococcus dentapri</td><td rowspan=1 colspan=1>Streptococcus marmotae</td><td rowspan=1 colspan=1>Streptococcus porci</td></tr><tr><td rowspan=1 colspan=1>Streptococcus dentiloxodontae</td><td rowspan=1 colspan=1>Streptococcus merionis</td><td rowspan=1 colspan=1>Streptococcus rupicaprae</td></tr><tr><td rowspan=1 colspan=1>Streptococcus dentirousetti</td><td rowspan=1 colspan=1>Streptococcus milleri</td><td rowspan=1 colspan=1>Streptococcus sobrinus</td></tr><tr><td rowspan=1 colspan=1>Streptococcus devriesei</td><td rowspan=1 colspan=1>Streptococcus minor</td><td rowspan=1 colspan=1>Streptococcus tangierensis</td></tr><tr><td rowspan=1 colspan=1>Streptococcus downei</td><td rowspan=1 colspan=1>Streptococcus oriloxodontae</td><td rowspan=1 colspan=1></td></tr></table>

Table 55: Predicted (in silico) Reactivity (Inclusivity) Results for Streptococcus anginosus group   

<table><tr><td rowspan=1 colspan=3>Detection Predicted for ≥95% of target sequences</td></tr><tr><td rowspan=1 colspan=1>Streptococcus anginosus*</td><td rowspan=1 colspan=1>Streptococcus constellatus*</td><td rowspan=1 colspan=1>Streptococcus intermedius*</td></tr><tr><td rowspan=1 colspan=3>Detection Predicted for 85%-94% of target sequences</td></tr><tr><td rowspan=1 colspan=3>None Identified</td></tr><tr><td rowspan=1 colspan=3>Detection Predicted for &lt;85% of target sequences</td></tr><tr><td rowspan=1 colspan=3>None Identified</td></tr><tr><td rowspan=1 colspan=3>Detection Not Predicted</td></tr><tr><td rowspan=1 colspan=3>None Identified</td></tr></table>

Table 56: Predicted (in silico) Reactivity (Inclusivity) Results for Pan Candida   

<table><tr><td rowspan=1 colspan=4>Detection Predicted for ≥95% of target sequences</td></tr><tr><td rowspan=1 colspan=1>Candida albicans*</td><td rowspan=1 colspan=2>Candida glabrata*</td><td rowspan=1 colspan=1>Candida krusei*</td></tr><tr><td rowspan=1 colspan=1>Candida parapsilosis*</td><td rowspan=1 colspan=2></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=4>Detection Predicted for 85%-94% of target sequences</td></tr><tr><td rowspan=1 colspan=4>None Identified</td></tr><tr><td rowspan=1 colspan=4>Detection Predicted for &lt;85% of target sequences</td></tr><tr><td rowspan=1 colspan=4>None Identified</td></tr><tr><td rowspan=1 colspan=4>Detection Not Predicted</td></tr><tr><td rowspan=1 colspan=2>Candida lusitaniae*</td><td rowspan=1 colspan=1>Candida orthopsilosis*</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=2>Candida metapsilosis*</td><td rowspan=1 colspan=1>Candida tropicalis*</td><td rowspan=1 colspan=1></td></tr></table>

# Table 57: Predicted (in silico) Reactivity (Inclusivity) Results for Pan Gram-Negative

# Analytical Specificity (Cross-Reactivity and Exclusivity)

Cross-reactivity of on-panel and off-panel analytes was evaluated with the BCID-GP Panel. Bacterial targets were tested in triplicate at a concentration of $\geq 1 \mathrm { x } 1 0 ^ { 9 } \mathrm { C F U / m L }$ while fungi were tested in triplicate at a concentration of $\ge 1 \mathrm { x } 1 0 ^ { 7 } \mathrm { C F U / m L }$ . If the target concentration could not be reached, the organism was diluted 2-fold from stock for use.

No cross reactivity was observed for any of the on-panel gram-positive organisms. Three organisms showed cross-reactivity, Burkholderia cepacia cross reacts with the Corynebacterium assay at levels $\ge 1 \mathrm { x } 1 0 ^ { 7 } \mathrm { C F U / m L }$ , an unspeciated Rhodococcus strain (ATCC 49988) cross reacts with the Micrococcus assay at levels $\ge 1 \mathrm { x } 1 0 ^ { 7 } \mathrm { C F U / m L }$ and Bacillus badius cross reacts with the Bacillus subtilis group assay at $7 \mathrm { \times 1 0 ^ { 7 } C F U / m L }$ . See Table 44 and Table 45 for summaries of the on-panel strains tested as a part of the Limit of Detection (Analytical Sensitivity) and Analytical Reactivity (Inclusivity) studies and Table 58 for a summary of off-panel strains tested.

# Off Panel Exclusivity

# Table 58: Targets Assessed for Cross-Reactivity with the ePlex BCID-GP Panel (Exclusivity)

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Strain ID</td></tr><tr><td rowspan=1 colspan=1>Abiotrophia defectiva</td><td rowspan=1 colspan=1>ATCC 49176</td></tr><tr><td rowspan=1 colspan=1>Actinomyces odontolyticus</td><td rowspan=1 colspan=1>ATCC 17929</td></tr><tr><td rowspan=1 colspan=1>Aerococcus sanguinicola</td><td rowspan=1 colspan=1>ATCC BAA-465</td></tr><tr><td rowspan=1 colspan=1>Aerococcus urinae</td><td rowspan=1 colspan=1>ATCC 700306</td></tr><tr><td rowspan=1 colspan=1>Aerococcus viridans</td><td rowspan=1 colspan=1>ATCC 10400</td></tr><tr><td rowspan=1 colspan=1>Anaerococcus prevotii</td><td rowspan=1 colspan=1>ATCC 9321</td></tr><tr><td rowspan=1 colspan=1>Arcanobacterium bernardiae</td><td rowspan=1 colspan=1>ATCC 51728</td></tr><tr><td rowspan=1 colspan=1>Arcanobacterium haemolyticum</td><td rowspan=1 colspan=1>ATCC BAA-1784A</td></tr><tr><td rowspan=1 colspan=1>Arthrobacter psychrolactophilus</td><td rowspan=1 colspan=1>ATCC 700733</td></tr><tr><td rowspan=1 colspan=1>Aspergillus fumigatus</td><td rowspan=1 colspan=1>ATCC 204305B</td></tr><tr><td rowspan=1 colspan=1>Bacillus badius</td><td rowspan=1 colspan=1>ATCC 14574</td></tr><tr><td rowspan=1 colspan=1>Bacillus pumilus</td><td rowspan=1 colspan=1>ATCC 14884</td></tr><tr><td rowspan=1 colspan=1>Brochothrix thermosphacta</td><td rowspan=1 colspan=1>ATCC 11509</td></tr><tr><td rowspan=1 colspan=1>Candida lusitaniae</td><td rowspan=1 colspan=1>ATCC 34449</td></tr><tr><td rowspan=1 colspan=1>Candida metapsilosis</td><td rowspan=1 colspan=1>ATCC 96144</td></tr><tr><td rowspan=1 colspan=1>Candida orthopsilosis</td><td rowspan=1 colspan=1>ATCC 96139</td></tr><tr><td rowspan=1 colspan=1>Candida tropicalis</td><td rowspan=1 colspan=1>ATCC 1369</td></tr><tr><td rowspan=1 colspan=1>Carnobacterium divergens</td><td rowspan=1 colspan=1>ATCC 35677</td></tr><tr><td rowspan=1 colspan=1>Carnobacterium maltaromaticum</td><td rowspan=1 colspan=1>ATCC 27865C</td></tr></table>

<table><tr><td colspan="1" rowspan="1">Organism</td><td colspan="1" rowspan="1">Strain ID</td></tr><tr><td colspan="1" rowspan="1">Granulicatella elegans</td><td colspan="1" rowspan="1">ATCC 700633C</td></tr><tr><td colspan="1" rowspan="1">Kocuria kristinae</td><td colspan="1" rowspan="1">ATCC BAA-752</td></tr><tr><td colspan="1" rowspan="1">Kocuria rhizophila</td><td colspan="1" rowspan="1">ATCC 9341</td></tr><tr><td colspan="1" rowspan="1">Kytococcus sedentarius</td><td colspan="1" rowspan="1">ATCC 27575</td></tr><tr><td colspan="1" rowspan="1">Lactococcus garvieae</td><td colspan="1" rowspan="1">ATCC 43921D</td></tr><tr><td colspan="1" rowspan="1">Lactococcus lactis</td><td colspan="1" rowspan="1">ATCC 49032</td></tr><tr><td colspan="1" rowspan="1">Leuconostoc carnosum</td><td colspan="1" rowspan="1">ATCC 49367</td></tr><tr><td colspan="1" rowspan="1">Leuconostoc citreum</td><td colspan="1" rowspan="1">ATCC 13146</td></tr><tr><td colspan="1" rowspan="1">Leuconostoc mesenteroides</td><td colspan="1" rowspan="1">ATCC 8293</td></tr><tr><td colspan="1" rowspan="1">Macrococcus caseolyticus</td><td colspan="1" rowspan="1">ATCC 29750</td></tr><tr><td colspan="1" rowspan="1">Macrococcus caseolyticus</td><td colspan="1" rowspan="1">ATCC 51834</td></tr><tr><td colspan="1" rowspan="1">Mycobacterium avium complex</td><td colspan="1" rowspan="1">ATCC 15769</td></tr><tr><td colspan="1" rowspan="1">Mycobacterium fortuitum</td><td colspan="1" rowspan="1">ATCC 6841</td></tr><tr><td colspan="1" rowspan="1">Mycobacterium mucogenicum</td><td colspan="1" rowspan="1">ATCC 49650</td></tr><tr><td colspan="1" rowspan="1">Mycobacterium tuberculosis</td><td colspan="1" rowspan="1">ATCC 25177</td></tr><tr><td colspan="1" rowspan="1">Mycoplasma pneumoniae</td><td colspan="1" rowspan="1">ATCC 39505</td></tr><tr><td colspan="1" rowspan="1">Pediococcus acidilactici</td><td colspan="1" rowspan="1">ATCC 8042</td></tr><tr><td colspan="1" rowspan="1">Pediococcus pentosaceus</td><td colspan="1" rowspan="1">ATCC 33316</td></tr><tr><td colspan="1" rowspan="1">Penicillium marneffei</td><td colspan="1" rowspan="1">ATCC 200050</td></tr><tr><td colspan="1" rowspan="1">Cellulomonas turbata</td><td colspan="1" rowspan="1">ATCC 25835</td></tr><tr><td colspan="1" rowspan="1">Cellulosimicrobium cellulans</td><td colspan="1" rowspan="1">ATCC 27402</td></tr><tr><td colspan="1" rowspan="1">Clostridium clostridioforme</td><td colspan="1" rowspan="1">ATCC 25537</td></tr><tr><td colspan="1" rowspan="1">Clostridium perfringens</td><td colspan="1" rowspan="1">ATCC 13124</td></tr><tr><td colspan="1" rowspan="1">Clostridium ramosum</td><td colspan="1" rowspan="1">ATCC 25582</td></tr><tr><td colspan="1" rowspan="1">Cryptococcus gattii</td><td colspan="1" rowspan="1">ATCC 76108</td></tr><tr><td colspan="1" rowspan="1">Cryptococcus grubii</td><td colspan="1" rowspan="1">ATCC 208821</td></tr><tr><td colspan="1" rowspan="1">Cryptococcus neoformans</td><td colspan="1" rowspan="1">ATCC 14116</td></tr><tr><td colspan="1" rowspan="1">Cutibacterium granulosum</td><td colspan="1" rowspan="1">ATCC 11829</td></tr><tr><td colspan="1" rowspan="1">Erysipelothrix rhusiopathiae</td><td colspan="1" rowspan="1">ATCC 35457</td></tr><tr><td colspan="1" rowspan="1">Gemella haemolysans</td><td colspan="1" rowspan="1">ATCC 10379</td></tr><tr><td colspan="1" rowspan="1">Gemella morbillorum</td><td colspan="1" rowspan="1">ATCC 27824</td></tr><tr><td colspan="1" rowspan="1">Geotrichum capitatum</td><td colspan="1" rowspan="1">ATCC 10663</td></tr><tr><td colspan="1" rowspan="1">Granulicatella adiacens</td><td colspan="1" rowspan="1">ATCC 43205</td></tr></table>

A Final testing concentration of $4 . 0 5 { \mathrm { x } } 1 0 ^ { 8 } \mathrm { C F U / m L }$ B Final testing concentration of $2 . 5 \mathrm { x } 1 0 ^ { 6 } \mathrm { C F U / m L }$ C Final testing concentration of $3 . 6 3 \mathrm { x } 1 0 ^ { 8 } \mathrm { C F U / m L }$ D Final testing concentration of $2 . 7 8 \mathrm { x } 1 0 ^ { 8 } \mathrm { C F U / m L }$

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Strain ID</td></tr><tr><td rowspan=1 colspan=1>Peptostreptococcus anaerobius</td><td rowspan=1 colspan=1>ATCC 27337</td></tr><tr><td rowspan=1 colspan=1>Planococcus species</td><td rowspan=1 colspan=1>ATCC 35671</td></tr><tr><td rowspan=1 colspan=1>Propionibacterium propionicum</td><td rowspan=1 colspan=1>ATCC 14157</td></tr><tr><td rowspan=1 colspan=1>Rhodococcus equi</td><td rowspan=1 colspan=1>ATCC 6939</td></tr><tr><td rowspan=1 colspan=1>Rhodococcus species</td><td rowspan=1 colspan=1>ATCC 49988</td></tr><tr><td rowspan=1 colspan=1>Rhodotorula glutinis</td><td rowspan=1 colspan=1>ATCC 32766</td></tr><tr><td rowspan=1 colspan=1>Rhodotorula minuta</td><td rowspan=1 colspan=1>ATCC 36236</td></tr><tr><td rowspan=1 colspan=1>Rhodotorula mucilaginosa</td><td rowspan=1 colspan=1>ATCC 9449</td></tr><tr><td rowspan=1 colspan=1>Rothia dentocariosa</td><td rowspan=1 colspan=1>ATCC 31918</td></tr><tr><td rowspan=1 colspan=1>Rothia mucilaginosa</td><td rowspan=1 colspan=1>ATCC 25296</td></tr><tr><td rowspan=1 colspan=1>Saccharomyces cerevisiae</td><td rowspan=1 colspan=1>ATCC 18824</td></tr><tr><td rowspan=1 colspan=1>Trichosporon asahii</td><td rowspan=1 colspan=1>ATCC 201110</td></tr><tr><td rowspan=1 colspan=1>Vagococcus fluvialis</td><td rowspan=1 colspan=1>ATCC 49515</td></tr><tr><td rowspan=1 colspan=1>Weissella paramesenteroides</td><td rowspan=1 colspan=1>ATCC 33313</td></tr></table>

# Bottle Positivity

Several representative bacterial and fungal organisms were spiked into blood culture bottles along with the manufacturer’s recommended volume of human whole blood and grown to positivity in a commercially-available continuously monitoring blood culture system. Bottles were removed from the incubator within two hours of being identified as positive as well as eight hours after bottle positivity. At least two independent positive blood culture replicates were quantified for each organism on culture plates. Organisms tested and approximate bottle positivity concentrations are summarized in Table 59. Concentrations shown below represent approximate levels that may be observed in a clinical setting. All estimated bottle positivity concentrations are equivalent or greater than the established Limit of Detection (LOD) for each of the assays of the ePlex BCID-GP Panel.

Table 59: Bottle Positivity Concentrations   

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Strain ID</td><td rowspan=1 colspan=1>Mean Bottle PositivityConcentration</td><td rowspan=1 colspan=1>Mean Bottle Positivity+8 hours Concentration</td></tr><tr><td rowspan=1 colspan=4>Gram-positive Organisms</td></tr><tr><td rowspan=1 colspan=1>Corynebacterium striatum</td><td rowspan=1 colspan=1>ATCC BAA-1293</td><td rowspan=1 colspan=1>4.5 x 106 CFU/mL</td><td rowspan=1 colspan=1>2.7 x 109 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Cutibacterium acnes (P. acnes)</td><td rowspan=1 colspan=1>ATCC 6919</td><td rowspan=1 colspan=1>3.0 x 108 CFU/mL</td><td rowspan=1 colspan=1>3.5 x 108 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Enterococcus faecium</td><td rowspan=1 colspan=1>ATCC BAA-2317</td><td rowspan=1 colspan=1>4.9 x 107 CFU/mL</td><td rowspan=1 colspan=1>3.6 x 107 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Lactobacillus casei</td><td rowspan=1 colspan=1>ATCC 39392</td><td rowspan=1 colspan=1>4.8 x 107 CFU/mL</td><td rowspan=1 colspan=1>3.1 x 1011 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus aureus</td><td rowspan=1 colspan=1>NRS 483</td><td rowspan=1 colspan=1>2.8 x 107 CFU/mL</td><td rowspan=1 colspan=1>2.1 x 107 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Streptococcus anginosus</td><td rowspan=1 colspan=1>ATCC 33397</td><td rowspan=1 colspan=1>4.1 x 107 CFU/mL</td><td rowspan=1 colspan=1>4.0 x 108 CFU/mL</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=3>Gram-negative Organisms</td></tr><tr><td rowspan=1 colspan=1>Acinetobacter baumannii</td><td rowspan=1 colspan=1>NCTC 13301</td><td rowspan=1 colspan=1>4.4 x 108 CFU/mL</td><td rowspan=1 colspan=1>3.8 x 108 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Bacteroides fragilis</td><td rowspan=1 colspan=1>ATCC 700786</td><td rowspan=1 colspan=1>4.7 x 108 CFU/mL</td><td rowspan=1 colspan=1>6.7 x 109 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Enterobacter cloacae</td><td rowspan=1 colspan=1>NCTC 13464</td><td rowspan=1 colspan=1>2.8 x 108 CFU/mL</td><td rowspan=1 colspan=1>7.7 x 108 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Escherichia coli</td><td rowspan=1 colspan=1>NCTC 13476</td><td rowspan=1 colspan=1>2.3 x 108 CFU/mL</td><td rowspan=1 colspan=1>1.5 x 109 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Fusobacterium nucleatum</td><td rowspan=1 colspan=1>ATCC 31647</td><td rowspan=1 colspan=1>6.5 x 107 CFU/mL</td><td rowspan=1 colspan=1>4.9 x 108 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Haemophilus influenzae</td><td rowspan=1 colspan=1>ATCC 19418</td><td rowspan=1 colspan=1>6.9 x 108 CFU/mL</td><td rowspan=1 colspan=1>1.2 x 109 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Klebsiella oxytoca</td><td rowspan=1 colspan=1>CDC #0147</td><td rowspan=1 colspan=1>9.3 x 108 CFU/mL</td><td rowspan=1 colspan=1>1.5 x 109 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Neisseria meningitidis</td><td rowspan=1 colspan=1>ATCC 13102</td><td rowspan=1 colspan=1>3.1 x 107 CFU/mL</td><td rowspan=1 colspan=1>2.1 x 108 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Pseudomonas aeruginosa</td><td rowspan=1 colspan=1>NCTC 13476</td><td rowspan=1 colspan=1>1.6 x 108 CFU/mL</td><td rowspan=1 colspan=1>8.4 x 108 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Serratia marcescens</td><td rowspan=1 colspan=1>ATCC 14041</td><td rowspan=1 colspan=1>1.2 x 109 CFU/mL</td><td rowspan=1 colspan=1>2.2 x 109 CFU/mL</td></tr><tr><td rowspan=1 colspan=4>Fungal Organism</td></tr><tr><td rowspan=1 colspan=1>Candida albicans</td><td rowspan=1 colspan=1>ATCC 90082</td><td rowspan=1 colspan=1>1.6 x 106 CFU/mL</td><td rowspan=1 colspan=1>1.4 x 106 CFU/mL</td></tr></table>

# Reproducibility

Four positive mixes including 9 on-panel organisms and 3 antibiotic resistance genes representing 15 targets at two concentrations and one negative mix including an off-panel organism were tested. Concentrations in the positive mixes reflected those observed at time of bottle positivity plus 8 hours $( \mathrm { B P } { + } 8 )$ ) and time of bottle positivity (BP) and one mix containing an off-panel organism grown to bottle positivity, which is expected to yield a negative result. Bottle concentrations used in this study are summarized in Table 60. Testing occurred at three sites with two operators testing the mixes over six days using three cartridge lots.

The percent agreement of each target with the expected result is summarized in Tables 61-75.   
The ePlex BCID-GP Panel demonstrates a high level of agreement with the expected results.

Table 60: Bottle Positivity Concentrations   

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Bottle PositivityConcentration</td><td rowspan=1 colspan=1>Bottle Positivity +8 HoursConcentration</td></tr><tr><td rowspan=1 colspan=1>Corynebacterium striatum</td><td rowspan=1 colspan=1>4 x 106 CFU/mL</td><td rowspan=1 colspan=1>1 x 108 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Cutibacterium acnes (P. acnes)</td><td rowspan=1 colspan=1>1 x 108 CFU/mL</td><td rowspan=1 colspan=1>4.4 x 107 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Enterococcus faecium (vanA+)</td><td rowspan=1 colspan=1>1 x 107 CFU/mL</td><td rowspan=1 colspan=1>1 x 108 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Enterococcus faecalis (vanB+)</td><td rowspan=1 colspan=1>1 x 107 CFU/mL</td><td rowspan=1 colspan=1>1 x 108 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Lactobacillus casei</td><td rowspan=1 colspan=1>1 x 107 CFU/mL</td><td rowspan=1 colspan=1>1 x 108 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus aureus (mecA+)</td><td rowspan=1 colspan=1>1 x 107 CFU/mL</td><td rowspan=1 colspan=1>1 x 108 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Streptococcus anginosus</td><td rowspan=1 colspan=1>1 x 107 CFU/mL</td><td rowspan=1 colspan=1>1 x 108 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Candida albicans (Pan Candida target)</td><td rowspan=1 colspan=1>1 x 106 CFU/mL</td><td rowspan=1 colspan=1>1 x 107 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Klebsiella pneumoniae (Pan Gram-Negative target)</td><td rowspan=1 colspan=1>1 x 108 CFU/mL</td><td rowspan=1 colspan=1>1 x 109 CFU/mL</td></tr></table>

Table 61: Percent Agreement for Corynebacterium   

<table><tr><td rowspan="2">Concentration of Corynebacterium striatum</td><td rowspan="2">Site</td><td colspan="3">Agreement with Expected Results</td></tr><tr><td>Agreed / N</td><td>%</td><td>95% CI</td></tr><tr><td rowspan="4">Bottle Positive + 8 Hours (1x108 CFU/mL)</td><td>1</td><td>36/36</td><td>100</td><td>(90.4-100)</td></tr><tr><td>2</td><td>36/36</td><td>100</td><td>(90.4-100)</td></tr><tr><td>3</td><td>36/36</td><td>100</td><td>(90.4-100)</td></tr><tr><td>All</td><td>108/108</td><td>100</td><td>(96.6-100)</td></tr><tr><td rowspan="4">Bottle Positive (4x106 CFU/mL)</td><td>1</td><td>35/36</td><td>97.2</td><td>(85.8-99.5)</td></tr><tr><td>2</td><td>35/36</td><td>97.2</td><td>(85.8-99.5)</td></tr><tr><td>3</td><td>33/36</td><td>91.7</td><td>(78.2-97.1)</td></tr><tr><td>All</td><td>103/108</td><td>95.4</td><td>(89.6-98.0)</td></tr><tr><td rowspan="4">Negative</td><td>1</td><td>108/108</td><td>100</td><td>(96.6-100)</td></tr><tr><td>2</td><td>107/108</td><td>99.1</td><td>(94.9-99.8)</td></tr><tr><td>3</td><td>107/107</td><td>100</td><td>(96.5-100)</td></tr><tr><td>All</td><td>322/323</td><td>99.7</td><td>(98.3-99.9)</td></tr></table>

CI=Confidence Interval

Table 62: Percent Agreement for Cutibacterium acnes (Propionibacterium acnes)   

<table><tr><td rowspan="2">Concentration of Cutibacterium acnes</td><td rowspan="2">Site</td><td colspan="3">Agreement with Expected Results</td></tr><tr><td>Agreed / N</td><td>%</td><td>95% CI</td></tr><tr><td rowspan="4">Bottle Positive + 8 Hours (4.4x107 CFU/mL)</td><td>1</td><td>32/36</td><td>88.9</td><td>(74.7-95.6)</td></tr><tr><td>2</td><td>33/36</td><td>91.7</td><td>(78.2-97.1)</td></tr><tr><td>3</td><td>36/36</td><td>100</td><td>(90.4-100)</td></tr><tr><td>All</td><td>101/108</td><td>93.5A</td><td>(87.2-96.8)</td></tr><tr><td rowspan="4">Bottle Positive (1.1x108 CFU/mL)</td><td>1</td><td>36/36</td><td>100</td><td>(90.4-100)</td></tr><tr><td>2</td><td>33/36</td><td>91.7</td><td>(78.2-97.1)</td></tr><tr><td>3</td><td>35/36</td><td>97.2</td><td>(85.8-99.5)</td></tr><tr><td>All</td><td>104/108</td><td>96.3B</td><td>(90.9-98.6)</td></tr><tr><td rowspan="4">Negative</td><td>1</td><td>108/108</td><td>100</td><td>(96.6-100)</td></tr><tr><td>2</td><td>108/108</td><td>100</td><td>(96.6-100)</td></tr><tr><td>3</td><td>107/107</td><td>100</td><td>(96.5-100)</td></tr><tr><td>All</td><td>323/323</td><td>100</td><td>(98.8-100)</td></tr></table>

A. ${ < } 9 5 \%$ agreement is expected because the C. acnes organism concentration is more than 0.5 log below the limit of detection of this assay $( 1 . 0 \mathrm { ~ x ~ } 1 0 ^ { \bar { 8 } } \mathrm { C F U / m L } )$ ). B. $9 5 \%$ agreement is expected because the $C$ . acnes organism concentration is approximately equal to the limit of detection of this assay $( 1 . 0 \mathrm { ~ x ~ } 1 0 ^ { 8 } \mathrm { C F U / m L } )$ .

Table 63: Percent Agreement for Enterococcus   

<table><tr><td rowspan="2">Concentration of Enterococcus</td><td rowspan="2">Site</td><td colspan="3">Agreement with Expected Results</td></tr><tr><td>Agreed / N</td><td>%</td><td>95% CI</td></tr><tr><td rowspan="4">Bottle Positive + 8 Hours (1x108 CFU/mL)</td><td>1</td><td>72/72</td><td>100</td><td>(94.9-100)</td></tr><tr><td>2</td><td>72/72</td><td>100</td><td>(94.9-100)</td></tr><tr><td>3</td><td>72/72</td><td>100</td><td>(94.9-100)</td></tr><tr><td>All</td><td>216/216</td><td>100</td><td>(98.3-100)</td></tr><tr><td rowspan="4">Bottle Positive (1x107 CFU/mL)</td><td>1</td><td>72/72</td><td>100</td><td>(94.9-100)</td></tr><tr><td>2</td><td>72/72</td><td>100</td><td>(94.9-100)</td></tr><tr><td>3</td><td>72/72</td><td>100</td><td>(94.9-100)</td></tr><tr><td>All</td><td>216/216</td><td>100</td><td>(98.3-100)</td></tr><tr><td rowspan="4">Negative</td><td>1</td><td>36/36</td><td>100</td><td>(90.4-100)</td></tr><tr><td>2</td><td>36/36</td><td>100</td><td>(90.4-100)</td></tr><tr><td>3</td><td>35/35</td><td>100</td><td>(90.1-100)</td></tr><tr><td>All</td><td>107/107</td><td>100</td><td>(96.5-100)</td></tr></table>

Table 64: Percent Agreement for Enterococcus faecalis   

<table><tr><td rowspan="2">Concentration of Enterococcus faecalis</td><td rowspan="2">Site</td><td colspan="3">Agreement with Expected Results</td></tr><tr><td>Agreed / N</td><td>%</td><td>95% CI</td></tr><tr><td rowspan="4">Bottle Positive + 8 Hours (1x108 CFU/mL)</td><td>1</td><td>36/36</td><td>100</td><td>(90.4-100)</td></tr><tr><td>2</td><td>36/36</td><td>100</td><td>(90.4-100)</td></tr><tr><td>3</td><td>36/36</td><td>100</td><td>(90.4-100)</td></tr><tr><td>All</td><td>108/108</td><td>100</td><td>(96.6-100)</td></tr><tr><td rowspan="4">Bottle Positive (1x107 CFU/mL)</td><td>1</td><td>36/36</td><td>100</td><td>(90.4-100)</td></tr><tr><td>2</td><td>36/36</td><td>100</td><td>(90.4-100)</td></tr><tr><td>3</td><td>36/36</td><td>100</td><td>(90.4-100)</td></tr><tr><td>All</td><td>108/108</td><td>100</td><td>(96.6-100)</td></tr><tr><td rowspan="4">Negative</td><td>1</td><td>108/108</td><td>100</td><td>(96.6-100)</td></tr><tr><td>2</td><td>108/108</td><td>100</td><td>(96.6-100)</td></tr><tr><td>3</td><td>107/107</td><td>100</td><td>(96.5-100)</td></tr><tr><td>All</td><td>323/323</td><td>100</td><td>(98.8-100)</td></tr></table>

Table 65: Percent Agreement for Enterococcus faecium   

<table><tr><td rowspan="2">Concentration of Enterococcus faecium</td><td rowspan="2">Site</td><td colspan="3">Agreement with Expected Results</td></tr><tr><td>Agreed / N</td><td>%</td><td>95% CI</td></tr><tr><td rowspan="4">Bottle Positive + 8 Hours (1x108 CFU/mL)</td><td>1</td><td>36/36</td><td>100</td><td>(90.4-100)</td></tr><tr><td>2</td><td>36/36</td><td>100</td><td>(90.4-100)</td></tr><tr><td>3</td><td>36/36</td><td>100</td><td>(90.4-100)</td></tr><tr><td>All</td><td>108/108</td><td>100</td><td>(96.6-100)</td></tr><tr><td rowspan="4">Bottle Positive (1x107 CFU/mL)</td><td>1</td><td>36/36</td><td>100</td><td>(90.4-100)</td></tr><tr><td>2</td><td>36/36</td><td>100</td><td>(90.4-100)</td></tr><tr><td>3</td><td>36/36</td><td>100</td><td>(90.4-100)</td></tr><tr><td>All</td><td>108/108</td><td>100</td><td>(96.6-100)</td></tr><tr><td rowspan="4">Negative</td><td>1</td><td>108/108</td><td>100</td><td>(96.6-100)</td></tr><tr><td>2</td><td>108/108</td><td>100</td><td>(96.6-100)</td></tr><tr><td>3</td><td>107/107</td><td>100</td><td>(96.5-100)</td></tr><tr><td>All</td><td>323/323</td><td>100</td><td>(98.8-100)</td></tr></table>

Table 66: Percent Agreement for Lactobacillus   

<table><tr><td rowspan="2">Concentration of Lactobacillus casei</td><td rowspan="2">Site</td><td colspan="3">Agreement with Expected Results</td></tr><tr><td>Agreed / N</td><td>%</td><td>95% CI</td></tr><tr><td rowspan="4">Bottle Positive + 8 Hours (1x108 CFU/mL)</td><td>1</td><td>36/36</td><td>100</td><td>(90.1-100)</td></tr><tr><td>2</td><td>36/36</td><td>100</td><td>(89.8-100)</td></tr><tr><td>3</td><td>36/36</td><td>100</td><td>(90.4-100)</td></tr><tr><td>All</td><td>108/108</td><td>100</td><td>(96.5-100)</td></tr><tr><td rowspan="4">Bottle Positive (1x107 CFU/mL)</td><td>1</td><td>36/36</td><td>100</td><td>(89.8-100)</td></tr><tr><td>2</td><td>36/36</td><td>100</td><td>(90.4-100)</td></tr><tr><td>3</td><td>36/36</td><td>100</td><td>(90.1-100)</td></tr><tr><td>All</td><td>108/108</td><td>100</td><td>(96.5-100)</td></tr><tr><td rowspan="4">Negative</td><td>1</td><td>108/108</td><td>100</td><td>(90.4-100)</td></tr><tr><td>2</td><td>108/108</td><td>100</td><td>(89.8-100)</td></tr><tr><td>3</td><td>107/107</td><td>100</td><td>(90.1-100)</td></tr><tr><td>All</td><td>323/323</td><td>100</td><td>(96.5-100)</td></tr></table>

Table 67: Percent Agreement for Staphylococcus   

<table><tr><td rowspan="2">Concentration of Staphylococcus</td><td rowspan="2">Site</td><td colspan="3">Agreement with Expected Results</td></tr><tr><td>Agreed / N</td><td>%</td><td>95% CI</td></tr><tr><td rowspan="4">Bottle Positive + 8 Hours (1x108 CFU/mL)</td><td>1</td><td>36/36</td><td>100</td><td>(90.4-100)</td></tr><tr><td>2</td><td>36/36</td><td>100</td><td>(90.4-100)</td></tr><tr><td>3</td><td>36/36</td><td>100</td><td>(90.4-100)</td></tr><tr><td>All</td><td>108/108</td><td>100</td><td>(96.6-100)</td></tr><tr><td rowspan="4">Bottle Positive (1x107 CFU/mL)</td><td>1</td><td>36/36</td><td>100</td><td>(90.4-100)</td></tr><tr><td>2</td><td>36/36</td><td>100</td><td>(90.48-100)</td></tr><tr><td>3</td><td>36/36</td><td>100</td><td>(90.4-100)</td></tr><tr><td>All</td><td>108/108</td><td>100</td><td>(96.6-100)</td></tr><tr><td rowspan="4">Negative</td><td>1</td><td>108/108</td><td>100</td><td>(96.6-100)</td></tr><tr><td>2</td><td>106/108</td><td>98.1</td><td>(93.5-99.5)</td></tr><tr><td>3</td><td>107/107</td><td>100</td><td>(96.5-100)</td></tr><tr><td>All</td><td>321/323</td><td>99.4</td><td>(97.8-99.8)</td></tr></table>

Table 68: Percent Agreement for Staphylococcus aureus   

<table><tr><td rowspan="2">Concentration of Staphylococcus aureus</td><td rowspan="2">Site</td><td colspan="3">Agreement with Expected Results</td></tr><tr><td>Agreed / N</td><td>%</td><td>95% CI</td></tr><tr><td rowspan="4">Bottle Positive + 8 Hours (1x108 CFU/mL)</td><td>1</td><td>36/36</td><td>100</td><td>(90.4-100)</td></tr><tr><td>2</td><td>36/36</td><td>100</td><td>(90.4-100)</td></tr><tr><td>3</td><td>36/36</td><td>100</td><td>(90.4-100)</td></tr><tr><td>All</td><td>108/108</td><td>100</td><td>(96.6-100)</td></tr><tr><td rowspan="4">Bottle Positive (1x107 CFU/mL)</td><td>1</td><td>36/36</td><td>100</td><td>(90.4-100)</td></tr><tr><td>2</td><td>36/36</td><td>100</td><td>(90.4-100)</td></tr><tr><td>3</td><td>36/36</td><td>100</td><td>(90.4-100)</td></tr><tr><td>All</td><td>108/108</td><td>100</td><td>(96.6-100)</td></tr><tr><td rowspan="4">Negative</td><td>1</td><td>108/108</td><td>100</td><td>(96.6-100)</td></tr><tr><td>2</td><td>108/108</td><td>100</td><td>(96.6-100)</td></tr><tr><td>3</td><td>107/107</td><td>100</td><td>(96.5-100)</td></tr><tr><td>All</td><td>323/323</td><td>100</td><td>(98.8-100)</td></tr></table>

Table 69: Percent Agreement for Streptococcus   

<table><tr><td rowspan="2">Concentration of Streptococcus</td><td rowspan="2">Site</td><td colspan="3">Agreement with Expected Results</td></tr><tr><td>Agreed / N</td><td>%</td><td>95% CI</td></tr><tr><td rowspan="4">Bottle Positive + 8 Hours (1x108 CFU/mL)</td><td>1</td><td>36/36</td><td>100</td><td>(90.4-100)</td></tr><tr><td>2</td><td>36/36</td><td>100</td><td>(90.4-100)</td></tr><tr><td>3</td><td>36/36</td><td>100</td><td>(90.4-100)</td></tr><tr><td>All</td><td>108/108</td><td>100</td><td>(96.6-100)</td></tr><tr><td rowspan="4">Bottle Positive (1x107 CFU/mL)</td><td>1</td><td>36/36</td><td>100</td><td>(90.4-100)</td></tr><tr><td>2</td><td>35/36</td><td>97.2</td><td>(85.8-99.5)</td></tr><tr><td>3</td><td>36/36</td><td>100</td><td>(90.4-100)</td></tr><tr><td>All</td><td>107/108</td><td>99.1</td><td>(94.9-99.8)</td></tr><tr><td rowspan="4">Negative</td><td>1</td><td>108/108</td><td>100</td><td>(96.6-100)</td></tr><tr><td>2</td><td>107/108</td><td>99.1</td><td>(94.9-99.8)</td></tr><tr><td>3</td><td>107/107</td><td>100</td><td>(96.5-100)</td></tr><tr><td>All</td><td>322/323</td><td>99.7</td><td>(98.3-99.9)</td></tr></table>

Table 70: Percent Agreement for Streptococcus anginosus group   

<table><tr><td rowspan="2">Concentration of Streptococcus anginosus</td><td rowspan="2">Site</td><td colspan="3">Agreement with Expected Results</td></tr><tr><td>Agreed / N</td><td>%</td><td>95% CI</td></tr><tr><td rowspan="4">Bottle Positive + 8 Hours (1x108 CFU/mL)</td><td>1</td><td>36/36</td><td>100</td><td>(90.4-100)</td></tr><tr><td>2</td><td>36/36</td><td>100</td><td>(90.4-100)</td></tr><tr><td>3</td><td>36/36</td><td>100</td><td>(90.4-100)</td></tr><tr><td>All</td><td>108/108</td><td>100</td><td>(96.6-100)</td></tr><tr><td rowspan="4">Bottle Positive (1x107 CFU/mL)</td><td>1</td><td>36/36</td><td>100</td><td>(90.4-100)</td></tr><tr><td>2</td><td>35/36</td><td>97.2</td><td>(85.8-99.5)</td></tr><tr><td>3</td><td>36/36</td><td>100</td><td>(90.4-100)</td></tr><tr><td>All</td><td>107/108</td><td>99.1</td><td>(94.9-99.8)</td></tr><tr><td rowspan="4">Negative</td><td>1</td><td>108/108</td><td>100</td><td>(96.6-100)</td></tr><tr><td>2</td><td>108/108</td><td>100</td><td>(96.6-100)</td></tr><tr><td>3</td><td>107/107</td><td>100</td><td>(96.5-100)</td></tr><tr><td>All</td><td>323/323</td><td>100</td><td>(98.8-100)</td></tr></table>

Table 71: Percent Agreement for Pan Candida   

<table><tr><td rowspan="2">Concentration of Candida albicans</td><td rowspan="2">Site</td><td colspan="3">Agreement with Expected Results</td></tr><tr><td>Agreed / N</td><td>%</td><td>95% CI</td></tr><tr><td rowspan="4">Bottle Positive + 8 Hours (1x107 CFU/mL)</td><td>1</td><td>36/36</td><td>100</td><td>(90.4-100)</td></tr><tr><td>2</td><td>36/36</td><td>100</td><td>(90.4-100)</td></tr><tr><td>3</td><td>36/36</td><td>100</td><td>(90.4-100)</td></tr><tr><td>All</td><td>108/108</td><td>100</td><td>(96.6-100)</td></tr><tr><td rowspan="4">Bottle Positive (1x106 CFU/mL)</td><td>1</td><td>36/36</td><td>100</td><td>(90.4-100)</td></tr><tr><td>2</td><td>36/36</td><td>100</td><td>(90.4-100)</td></tr><tr><td>3</td><td>36/36</td><td>100</td><td>(90.4-100)</td></tr><tr><td>All</td><td>108/108</td><td>100</td><td>(96.6-100)</td></tr><tr><td rowspan="4">Negative</td><td>1</td><td>108/108</td><td>100</td><td>(96.6-100)</td></tr><tr><td>2</td><td>108/108</td><td>100</td><td>(96.6-100)</td></tr><tr><td>3</td><td>107/107</td><td>100</td><td>(96.5-100)</td></tr><tr><td>All</td><td>323/323</td><td>100</td><td>(98.8-100)</td></tr></table>

Table 72: Percent Agreement for Pan Gram-Negative   

<table><tr><td rowspan="2">Concentration of Klebsiella pneumoniae</td><td rowspan="2">Site</td><td colspan="3">Agreement with Expected Results</td></tr><tr><td>Agreed / N</td><td>%</td><td>95% CI</td></tr><tr><td rowspan="4">Bottle Positive + 8 Hours (1x109 CFU/mL)</td><td>1</td><td>36/36</td><td>100</td><td>(90.4-100)</td></tr><tr><td>2</td><td>36/36</td><td>100</td><td>(90.4-100)</td></tr><tr><td>3</td><td>36/36</td><td>100</td><td>(90.4-100)</td></tr><tr><td>All</td><td>108/108</td><td>100</td><td>(96.6-100)</td></tr><tr><td rowspan="4">Bottle Positive (1x108 CFU/mL)</td><td>1</td><td>36/36</td><td>100</td><td>(90.4-100)</td></tr><tr><td>2</td><td>36/36</td><td>100</td><td>(90.4-100)</td></tr><tr><td>3</td><td>36/36</td><td>100</td><td>(90.4-100)</td></tr><tr><td>All</td><td>108/108</td><td>100</td><td>(96.6-100)</td></tr><tr><td rowspan="4">Negative</td><td>1</td><td>108/108</td><td>100</td><td>(96.6-100)</td></tr><tr><td>2</td><td>107/108</td><td>99.1</td><td>(94.9-99.8)</td></tr><tr><td>3</td><td>107/107</td><td>100</td><td>(96.5-100)</td></tr><tr><td>All</td><td>322/323</td><td>99.7</td><td>(98.3-99.9)</td></tr></table>

Table 73: Percent Agreement for mecA   

<table><tr><td rowspan="2">Concentration of Staphylococcus aureus</td><td rowspan="2">Site</td><td colspan="3">Agreement with Expected Results</td></tr><tr><td>Agreed / N</td><td>%</td><td>95% CI</td></tr><tr><td rowspan="4">Bottle Positive + 8 Hours (1x108 CFU/mL)</td><td>1</td><td>36/36</td><td>100</td><td>(90.4-100)</td></tr><tr><td>2</td><td>36/36</td><td>100</td><td>(90.4-100)</td></tr><tr><td>3</td><td>36/36</td><td>100</td><td>(90.4-100)</td></tr><tr><td>All</td><td>108/108</td><td>100</td><td>(96.6-100)</td></tr><tr><td rowspan="4">Bottle Positive (1x107 CFU/mL)</td><td>1</td><td>36/36</td><td>100</td><td>(90.4-100)</td></tr><tr><td>2</td><td>36/36</td><td>100</td><td>(90.4-100)</td></tr><tr><td>3</td><td>36/36</td><td>100</td><td>(90.4-100)</td></tr><tr><td>All</td><td>108/108</td><td>100</td><td>(96.6-100)</td></tr><tr><td rowspan="4">Negative</td><td>1</td><td>108/108</td><td>100</td><td>(96.6-100)</td></tr><tr><td>2</td><td>108/108</td><td>100</td><td>(96.6-100)</td></tr><tr><td>3</td><td>107/107</td><td>100</td><td>(96.5-100)</td></tr><tr><td>All</td><td>323/323</td><td>100</td><td>(98.8-100)</td></tr></table>

Table 74: Percent Agreement for vanA   

<table><tr><td rowspan="2">Concentration of Enterococcus faecium</td><td rowspan="2">Site</td><td colspan="3">Agreement with Expected Results</td></tr><tr><td>Agreed / N</td><td>%</td><td>95% CI</td></tr><tr><td rowspan="4">Bottle Positive + 8 Hours (1x108 CFU/mL)</td><td>1</td><td>36/36</td><td>100</td><td>(90.4-100)</td></tr><tr><td>2</td><td>36/36</td><td>100</td><td>(90.4-100)</td></tr><tr><td>3</td><td>36/36</td><td>100</td><td>(90.4-100)</td></tr><tr><td>All</td><td>108/108</td><td>100</td><td>(96.6-100)</td></tr><tr><td rowspan="4">Bottle Positive (1x107 CFU/mL)</td><td>1</td><td>36/36</td><td>100</td><td>(90.4-100)</td></tr><tr><td>2</td><td>36/36</td><td>100</td><td>(90.4-100)</td></tr><tr><td>3</td><td>36/36</td><td>100</td><td>(90.4-100)</td></tr><tr><td>All</td><td>108/108</td><td>100</td><td>(96.6-100)</td></tr><tr><td rowspan="4">Negative</td><td>1</td><td>108/108</td><td>100</td><td>(96.6-100)</td></tr><tr><td>2</td><td>108/108</td><td>100</td><td>(96.6-100)</td></tr><tr><td>3</td><td>107/107</td><td>100</td><td>(96.5-100)</td></tr><tr><td>All</td><td>323/323</td><td>100</td><td>(98.8-100)</td></tr></table>

Table 75: Percent Agreement for vanB   

<table><tr><td rowspan="2">Concentration of Enterococcus faecalis</td><td rowspan="2">Site</td><td colspan="3">Agreement with Expected Results</td></tr><tr><td>Agreed / N</td><td>%</td><td>95% CI</td></tr><tr><td rowspan="4">Bottle Positive + 8 Hours (1x108 CFU/mL)</td><td>1</td><td>36/36</td><td>100</td><td>(90.4-100)</td></tr><tr><td>2</td><td>36/36</td><td>100</td><td>(90.4-100)</td></tr><tr><td>3</td><td>36/36</td><td>100</td><td>(90.4-100)</td></tr><tr><td>All</td><td>108/108</td><td>100</td><td>(96.6-100)</td></tr><tr><td rowspan="4">Bottle Positive (1x107 CFU/mL)</td><td>1</td><td>36/36</td><td>100</td><td>(90.4-100)</td></tr><tr><td>2</td><td>36/36</td><td>100</td><td>(90.4-100)</td></tr><tr><td>3</td><td>36/36</td><td>100</td><td>(90.4-100)</td></tr><tr><td>All</td><td>108/108</td><td>100</td><td>(96.6-100)</td></tr><tr><td rowspan="4">Negative</td><td>1</td><td>108/108</td><td>100</td><td>(96.6-100)</td></tr><tr><td>2</td><td>108/108</td><td>100</td><td>(96.6-100)</td></tr><tr><td>3</td><td>107/107</td><td>100</td><td>(96.5-100)</td></tr><tr><td>All</td><td>323/323</td><td>100</td><td>(98.8-100)</td></tr></table>

# Interfering Substances and Sample Matrix Equivalency (Bottle Evaluation)

Two organism mixes consisting of 9 on-panel organisms representing 14 targets and negative blood matrix were used to assess potentially interfering substances and bottle types for interference. The concentration of each organism tested is summarized in Table 76.

Table 76: Interfering Substance and Bottle Equivalency Concentrations   

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Concentration</td></tr><tr><td rowspan=1 colspan=1>Cutibacterium acnes</td><td rowspan=1 colspan=1>3 x 108 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Enterococcus faecalis (vanB+)</td><td rowspan=1 colspan=1>4 x 107 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Enterococcus faecium (vanA+)</td><td rowspan=1 colspan=1>4 x 107 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Lactobacillus casei</td><td rowspan=1 colspan=1>4 x 107 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus aureus</td><td rowspan=1 colspan=1>2 x 107 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus epidermidis</td><td rowspan=1 colspan=1>2 x 107 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Streptococcus pneumoniae</td><td rowspan=1 colspan=1>4 x 107 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Klebsiella pneumoniae (Pan Gram-Negative target)</td><td rowspan=1 colspan=1>5 x 108 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Candida albicans (Pan Candida target)</td><td rowspan=1 colspan=1>1 x 106 CFU/mL</td></tr></table>

# Interfering Substances

Eighteen substances were used to assess the ePlex BCID-GP Panel for potential interference. The organisms in Table 76 were spiked into negative blood matrix and tested in triplicate with and without each potentially interfering substance. Negative blood matrix was tested to control for potential positive interference. Potentially interfering substances are summarized in Table 77. None of the eighteen substances commonly found in blood culture specimens or as medications commonly used to treat skin or bloodstream infections were found to inhibit the ePlex BCID-GP Panel at clinically relevant concentrations. The effect of interfering substances has only been evaluated for the organisms listed in Table 76. Interference due to substances other than those described in this section can lead to erroneous results.

Table 77: Potentially Interfering Substances: Substance List   

<table><tr><td rowspan=1 colspan=1>Endogenous Substances</td><td rowspan=1 colspan=1>Testing Concentration</td></tr><tr><td rowspan=1 colspan=1>Bilirubin</td><td rowspan=1 colspan=1>60 μg/mL</td></tr><tr><td rowspan=1 colspan=1>Hemoglobin</td><td rowspan=1 colspan=1>0.6 g/L</td></tr><tr><td rowspan=1 colspan=1>Human Genomic DNA</td><td rowspan=1 colspan=1>6 x 105 copies/mL</td></tr><tr><td rowspan=1 colspan=1>Triglycerides</td><td rowspan=1 colspan=1>1000 mg/dL</td></tr><tr><td rowspan=1 colspan=1>γ-globulin</td><td rowspan=1 colspan=1>0.425 g/dL</td></tr><tr><td rowspan=1 colspan=1>Exogenous Substances</td><td rowspan=1 colspan=1>Testing Concentration</td></tr><tr><td rowspan=1 colspan=1>Amoxicillin/Clavulanate</td><td rowspan=1 colspan=1>3.5 μg/mL</td></tr><tr><td rowspan=1 colspan=1>Amphotericin B</td><td rowspan=1 colspan=1>2 μg/mL</td></tr><tr><td rowspan=1 colspan=1>Caspofungin</td><td rowspan=1 colspan=1>5 μg/mL</td></tr><tr><td rowspan=1 colspan=1>Ceftriaxone</td><td rowspan=1 colspan=1>0.23 mg/mL</td></tr><tr><td rowspan=1 colspan=1>Ciprofloxacin</td><td rowspan=1 colspan=1>3 mg/L</td></tr><tr><td rowspan=1 colspan=1>Fluconazole</td><td rowspan=1 colspan=1>25 mg/L</td></tr><tr><td rowspan=1 colspan=1>Flucytosine</td><td rowspan=1 colspan=1>90 μg/mL</td></tr><tr><td rowspan=1 colspan=1>Gentamicin sulfate</td><td rowspan=1 colspan=1>3 μg/mL</td></tr><tr><td rowspan=1 colspan=1>Heparin</td><td rowspan=1 colspan=1>0.9 U/mL</td></tr><tr><td rowspan=1 colspan=1>Imipenem</td><td rowspan=1 colspan=1>83 μg/mL</td></tr><tr><td rowspan=1 colspan=1>Sodium Polyanethol Sulfonate</td><td rowspan=1 colspan=1>0.25% w/v</td></tr><tr><td rowspan=1 colspan=1>Tetracycline</td><td rowspan=1 colspan=1>5 mg/L</td></tr><tr><td rowspan=1 colspan=1>Vancomycin</td><td rowspan=1 colspan=1>30 mg/L</td></tr></table>

# Sample Matrix Equivalency (Bottle Evaluation)

Thirteen bottle types were tested for interference with each of the organisms listed in Table 76. Five replicates of each organism were tested in each of two bottle lots. Negative blood matrix was run as a negative control. Eleven of the bottle types tested showed no interference for any of the targets tested. One lot of the BACTECTM Plus Anaerobic bottles showed false positive results for Pan Gram-Negative. The BacT/ALERT® FN Plus bottle type showed lower sensitivity for some targets (Pan Gram-Negative and $E$ . faecium with vanA). A summary of the bottle types assessed and the study outcomes is found in Table 78.

Table 78: Sample Matrix Equivalency (Bottle Evaluation) Bottle Types   

<table><tr><td rowspan=1 colspan=1>Manufacturer</td><td rowspan=1 colspan=1>Bottle Brand</td><td rowspan=1 colspan=1>Bottle Type</td><td rowspan=1 colspan=1>Study Outcome</td></tr><tr><td rowspan=1 colspan=1>BD</td><td rowspan=1 colspan=1>BACTECTM</td><td rowspan=1 colspan=1>Plus Aerobic</td><td rowspan=1 colspan=1>No interference observed</td></tr><tr><td rowspan=1 colspan=1>BD</td><td rowspan=1 colspan=1>BACTECTM</td><td rowspan=1 colspan=1>Plus Anaerobic</td><td rowspan=1 colspan=1>False positive results for PanGram-Negative target wereobserved in one lot.</td></tr><tr><td rowspan=1 colspan=1>BD</td><td rowspan=1 colspan=1>BACTECTM</td><td rowspan=1 colspan=1>Standard Aerobic</td><td rowspan=1 colspan=1>No interference observed</td></tr><tr><td rowspan=1 colspan=1>BD</td><td rowspan=1 colspan=1>BACTECTM</td><td rowspan=1 colspan=1>Standard Anaerobic</td><td rowspan=1 colspan=1>No interference observed</td></tr><tr><td rowspan=1 colspan=1>BD</td><td rowspan=1 colspan=1>BACTECTM</td><td rowspan=1 colspan=1>Peds PlusTM</td><td rowspan=1 colspan=1>No interference observed</td></tr><tr><td rowspan=1 colspan=1>BD</td><td rowspan=1 colspan=1>BACTECTM</td><td rowspan=1 colspan=1>Lytic Anaerobic</td><td rowspan=1 colspan=1>No interference observed</td></tr><tr><td rowspan=1 colspan=1>bioMérieux</td><td rowspan=1 colspan=1>BACT/ALERT</td><td rowspan=1 colspan=1>SA Standard Aerobic</td><td rowspan=1 colspan=1>No interference observed</td></tr><tr><td rowspan=1 colspan=1>bioMérieux</td><td rowspan=1 colspan=1>BACT/ALERT®</td><td rowspan=1 colspan=1>SN Standard Anaerobic</td><td rowspan=1 colspan=1>No interference observed</td></tr><tr><td rowspan=1 colspan=1>bioMérieux</td><td rowspan=1 colspan=1>BACT/ALERT</td><td rowspan=1 colspan=1>FA Plus</td><td rowspan=1 colspan=1>No interference observed</td></tr><tr><td rowspan=1 colspan=1>bioMérieux</td><td rowspan=1 colspan=1>BACT/ALERT</td><td rowspan=1 colspan=1>FN Plus</td><td rowspan=1 colspan=1>False negative results wereobserved for Pan Gram-Negativeand E.faecium/vanA targets</td></tr><tr><td rowspan=1 colspan=1>bioMérieux</td><td rowspan=1 colspan=1>BACT/ALERT®</td><td rowspan=1 colspan=1>PF Plus</td><td rowspan=1 colspan=1>No interference observed</td></tr><tr><td rowspan=1 colspan=1>Thermo Scientific™</td><td rowspan=1 colspan=1>VersaTREKT</td><td rowspan=1 colspan=1>REDOXTM 1 EZ DraW Aerobic</td><td rowspan=1 colspan=1>No interference observed</td></tr><tr><td rowspan=1 colspan=1>Thermo Scientific™M</td><td rowspan=1 colspan=1>VersaTREKTM</td><td rowspan=1 colspan=1>REDOXTM 2 EZ DraW Anaerobic</td><td rowspan=1 colspan=1>No interference observed</td></tr></table>

# Carryover and Cross-Contamination

Carryover and cross-contamination were evaluated for the ePlex BCID-GP Panel within and between runs by alternating high positive and negative samples across multiple runs over 5 rounds of testing. A high-titer mix of mecA positive Staphylococcus aureus, vanA positive Enterococcus faecium and Klebsiella pneumoniae (a Pan Gram-Negative target organism) was prepared at $1 \mathrm { ~ x ~ } 1 0 ^ { 9 } \mathrm { C F U / m L }$ each as well as Candida albicans (a Pan Candida target organism) at $1 \mathrm { { x } 1 0 ^ { 7 } C F U / m L }$ to simulate clinically relevant high positive samples for positive testing. Negative blood culture matrix was used to represent negative samples. Over 120 runs, all valid positive runs resulted in detection of Staphylococcus aureus, mecA, Enterococcus faecium, vanA, Pan Gram-Negative and Pan Candida and no false positives were detected in the negative runs.

# Competitive Inhibition Study

Competitive inhibition was evaluated for the ePlex BCID-GP Panel by pairing eight clinically relevant organisms (including a Pan Gram-Negative assay target, a Pan Candida assay target, and an off-panel gram-positive organism) in four simulated dual infection sample mixes. Each dual infection mix was tested in combination with each of the three other mixes, such that all organisms were tested at low titer (concentrations expected at bottle positivity) while in the presence of other organisms at higher titer (concentrations expected at 8 hours beyond bottle positivity, or one log higher than that expected at bottle positivity). No competitive inhibition was observed in any replicates of the twelve testing conditions. A summary of the organisms assessed and testing concentrations is found in Table 79.

Table 79: Competitive Inhibition Organisms and Concentrations Tested   

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>HighConcentration</td><td rowspan=1 colspan=1>LowConcentration</td></tr><tr><td rowspan=1 colspan=1>Enterococcus faecium</td><td rowspan=1 colspan=1>1 x 108 CFU/mL</td><td rowspan=1 colspan=1>4 x 107 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus aureus</td><td rowspan=1 colspan=1>1 x 108 CFU/mL</td><td rowspan=1 colspan=1>2 x 107 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus epidermidis</td><td rowspan=1 colspan=1>1 x 108 CFU/mL</td><td rowspan=1 colspan=1>2 x 107 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Streptococcus agalactiae</td><td rowspan=1 colspan=1>4 x 108 CFU/mL</td><td rowspan=1 colspan=1>4 x 107 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Streptococcus pneumoniae</td><td rowspan=1 colspan=1>4 x 108 CFU/mL</td><td rowspan=1 colspan=1>4 x 107 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Candida albicans</td><td rowspan=1 colspan=1>1 x 107 CFU/mL</td><td rowspan=1 colspan=1>1 x 106 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Escherichia coli</td><td rowspan=1 colspan=1>1 x 109 CFU/mL</td><td rowspan=1 colspan=1>2 x 108 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Cutibacterium granulosum4</td><td rowspan=1 colspan=1>1 x 109 CFU/mL</td><td rowspan=1 colspan=1>3 x 108 CFU/mL</td></tr></table>

A Off-panel organism