# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATION DECISION SUMMARY

A. 510(k) Number: K181427   
B. Purpose for Submission: Clearance of the Enteric Viral Panel assay on the BD MAX Syst   
C. Measurand: Norovirus, Rotavirus, Adenovirus, Sapovirus, and Astrovirus.   
D. Type of Test: Qualitative real-time polymerase chain reaction   
E. Applicant: Becton, Dickinson and Company   
F. Proprietary and Established Names: BD MAX Enteric Viral Panel

# G. Regulatory Information:

1. Regulation section: 21 CFR 866.3990 - Gastrointestinal microorganism multiplex nucleic acid based assay

2. Classification: Class II

3. Product code: PCH, OOI

4. Panel:

# Microbiology (83)

# H. Intended Use:

1. Intended use(s):

The BD MAX Enteric Viral Panel performed on the BD MAX System, is an automated in vitro diagnostic test for the direct qualitative detection and differentiation of enteric viral pathogens. The BD MAX Enteric Viral Panel detects nucleic acids from

• Norovirus GI & GII   
• Rotavirus A   
• Adenovirus F40/41   
• Sapovirus (genogroups I, II, IV, V)   
• Human Astrovirus (hAstro)

Testing is performed on unpreserved soft to diarrheal or Cary-Blair preserved stool specimens from symptomatic patients with suspected acute gastroenteritis, enteritis or colitis. The test is performed directly on the specimen, utilizing real-time polymerase chain reaction (PCR) for the amplification of relevant gene target DNA/RNA. The test utilizes fluorogenic gene-specific hybridization probes for the detection of the amplified DNA.

This test is intended for use, in conjunction with clinical presentation, laboratory findings, and epidemiological information, as an aid in the differentialdiagnosis of Norovirus GI & GII, Rotavirus A, Adenovirus F40/41, Sapovirus (genogroups I, II, IV, V), and Astrovirus infections. Results of this test should not be used as the sole basis for diagnosis, treatment, or other patient management decisions. Positive results do not rule out co-infection with other organisms that are not detected by this test, and may not be the sole or definitive cause of patient illness. Negative results in the setting of clinical illness compatible with gastroenteritis may be due to infection by pathogens that are not detected by this test or non-infectious causes such as ulcerative colitis, irritable bowel syndrome, or Crohn’s disease.

2. Indication(s) for use: Same as intended use.

3. Special conditions for use statement(s):

For prescription use only.

Limitations:

This product should only be used with the BD MAX System. The Sample Buffer Tube has not been designed to support organism viability. If culture is necessary it must be performed from the original specimen.

The performance of this test has not been evaluated for immunocompromised individuals or for patients without symptoms of gastrointestinal infection. BD MAX Enteric Viral Panel assay performance has not been evaluated in individuals who have received the Rotavirus A vaccine, which is known to react with this assay. · Adenovirus Type 1 associated with infections in humans was shown to have the potential to cross-react with BD MAX Enteric Viral Panel.

Additional limitations are noted in the device labeling.

4. Special instrument requirements:

For use with the BD MAX System.

# I. Device Description:

The BD MAX Enteric Viral Panel performed on the BD MAX System, is an automated in vitro diagnostic test for the direct qualitative detection and differentiation of enteric viral pathogens. The BD MAX Enteric Viral Panel detects nucleic acids from

• Norovirus GI & GII   
• Rotavirus A   
• Adenovirus F40/41   
• Sapovirus (genogroups I, II, IV, V)   
• Human Astrovirus (hAstro)

Testing is performed on unpreserved soft to diarrheal or Cary-Blair preserved stool specimens from symptomatic patients with suspected acute gastroenteritis, enteritis or colitis. The test is performed directly on the specimen, utilizing real-time polymerase chain reaction (PCR) for the amplification of relevant gene target DNA/RNA. The test utilizes fluorogenic gene-specific hybridization probes for the detection of the amplified DNA.

The BD MAX System and the BD MAX Enteric Viral Panel is run with the instrument, associated hardware and accessories, disposable microfluidic cartridges, master mixes, unitized reagent strips, extraction reagents, and sample buffer tubes. The instrument automates sample preparation including target lysis, DNA/RNA extraction and concentration, reagent rehydration, and target nucleic acid amplification and detection using real-time PCR. The assay includes a Sample Processing Control (SPC) that is present in the Extraction Tube. The SPC monitors DNA/RNA extraction steps, thermal cycling steps, reagent integrity and the presence of inhibitory substances. The BD MAX System software automatically interprets test results. A test result may be called as POS (Positive), NEG (Negative), or UNR (Unresolved) for each of the assay’s targets, based on the amplification status of the target and of the Sample Processing Control. IND (Indeterminate) or INC (Incomplete) results are due to BD MAX System failure.

# Materials provided in each BD MAX Enteric Viral Panel Kit:

Each kit contains sufficient reagents to test 24 samples (443985):

· BD MAX Enteric Viral Panel Master Mix (D6) · BD MAX Enteric Viral Panel Master Mix (D5) · BD MAX Enteric Viral Panel Unitized Reagent Strips · BD MAX Enteric Viral Panel Extraction Tube (D4) · BD MAX Enteric Viral Panel Ssample Buffer Tube · BD MAX Disposable Inoculation Loops Septum Caps

# Materials required but not provided:

· BD MAX PCR Cartridges   
· VWR Multi-Tube Vortexer or equivalent Vortex Genie 2 or equivalent Nalgene Cryogenic Vial Holder Rack compatible with a multi-tube vortex mixer (e.g., Cryogenic Vial Holder or equivalent) Lab coat and disposable gloves, powderless Stopwatch or timer For “Unpreserved” stool specimen type: - Dry, clean containers for collection of liquid or soft stool specimens. For preserved stool specimen type: Cary-Blair transport media $\mathrm { 1 5 ~ m L }$ .

# Interpretation of Results

Targets amplified by both mastermixes D6 and D5 are detected with hydrolysis probes (TaqMan probes) labelled at one end with a fluorescent reporter dye and at the other end with a quencher moiety. Six separate probes labelled with different reporter dyes are used to detect, in four different BD MAX System optical channels, the amplicons generated by their respective primers for MM D6 Enteric Viral Panel targets (Astrovirus, Sapovirus, and a sample processing control) and eight probes for MM D5 Enteric Viral Panel targets (Adenovirus, Norovirus, Rotavirus, and a sample processing control). The BD MAX System monitors several amplification curve metrics including: the fluorescence height at the end of the amplification curve (EP fluorescence), the location and value of the curve’s first derivative, the location and value of the curve’s second derivative, the threshold cycle value with an additional quality control applied to prevent false positives due to signal drift (Ct.Score), and measurements of system noise along the PCR amplification curve. Curve metrics are transformed into results via comparison of the metrics to cutoffs and the use of logic statements. Specifically, cutoffs are applied to the Ct.Score and EP values to determine amplification status with other metrics serving as quality controls.

Results are available on the “Results” tab in the “Results” window on the BD MAX System monitor. The BD MAX System software automatically interprets test results. Results are reported for each of the analytes and for the Sample Processing Control. A test result may be called as NEG (negative), POS (positive) or UNR (unresolved) based on the amplification status of the target and of the Sample Processing Control. IND (Indeterminate) or INC (Incomplete) results are due to BD MAX System failure and require sample repeat testing. A sample can be re-tested directly from the already prepared Sample Buffer Tube or following the preparation of a new Sample Buffer Tube inoculation. In the case of a partial UNR, where one or more targets have a POS result and all other targets have a UNR result, the targets with a UNR result will not be called NEG. This will be reported on a per Master Mix basis.

Erroneous results may occur from improper specimen collection, handling, storage, technical error, specimen mix-up or because the number of organisms in the specimen is below the analytical sensitivity of the test.

The Sample Processing Control has been added to the test to aid in the identification of samples that contain inhibitors to PCR amplification and as a control for reagent integrity and of the assay system as a whole. The Sample Processing Control does not indicate if nucleic acid has been lost due to inadequate collection, transport or storage of samples, or whether viral capsids have been adequately lysed.

# J. Substantial Equivalence Information:

1. Predicate device name(s): BioFire FilmArray Gastrointestinal (GI) Panel

2. Predicate 510(k) number(s): K143005

3. Comparison with predicate:

<table><tr><td colspan="3"></td></tr><tr><td colspan="3">Similarities</td></tr><tr><td>Item</td><td>Device: BD MAX Enteric Viral Panel (K181427)</td><td>Predicate: FilmArray Gastrointestinal Pathogen Panel (K143005)</td></tr><tr><td>Intended Use</td><td>The BD MAX Enteric Viral Panel performed on the BD MAX System, is an automated in vitro diagnostic test for the direct qualitative detection and differentiation of enteric viral pathogens. The BD MAX Enteric Viral Panel detects nucleic acids from Norovirus GI &amp; GII Rotavirus A</td><td>The FilmArray Gastrointestinal (GI) Panel is a qualitative multiplexed nucleic acid-based in vitro diagnostic test intended for use with FilmArray systems. The FilmArray GI Panel is capable of the simultaneous detection and identification of nucleic acids from multiple bacteria, viruses, and parasites directly from stool samples in Cary-Blair transport media</td></tr><tr><td></td><td>Adenovirus F40/41 •Sapovirus (genogroups I, II, IV, V) Human Astrovirus (hAstro)</td><td>obtained from individuals with signs and/or symptoms of gastrointestinal infection.</td></tr><tr><td></td><td>Testing is performed on unpreserved soft to diarrheal or Cary-Blair preserved stool specimens from symptomatic patients with</td><td>The following bacteria (including several diarrheagenic E. coli/Shigella pathotypes), parasites, and viruses are identified using the FilmArray GI Panel:</td></tr><tr><td></td><td></td><td></td></tr><tr><td></td><td>suspected acute gastroenteritis, enteritis or colitis. The test is performed directly on the specimen, utilizing real-time polymerase chain reaction (PCR) for the amplification of</td><td>•Campylobacter (C. jejuni/C. coli/C. upsaliensis)</td></tr><tr><td></td><td></td><td>Clostridium difficile (C. difficile) toxin</td></tr><tr><td></td><td>relevant gene target DNA/RNA. The test</td><td>A/B Plesiomonas shigelloides</td></tr><tr><td></td><td>utilizes fluorogenic gene-specific hybridization probes for the detection of the</td><td>Salmonella Vibrio (V. parahaemolyticus/V.</td></tr><tr><td></td><td>amplified DNA. This test is intended for use, in conjunction with clinical presentation, laboratory findings, and epidemiological information, as an aid in the differential diagnosis of Norovirus GI &amp; GII, Rotavirus A, Adenovirus F40/41,</td><td>vulnificus/V. cholerae), including specific identification of Vibrio cholerae Yersinia enterocolitica</td></tr><tr><td></td><td>Sapovirus (genogroups I, II, IV, V), and Astrovirus infections. Results of this test should not be used as the sole basis for</td><td>Enteroaggregative Escherichia coli (EAEC)</td></tr><tr><td></td><td></td><td>•Enteropathogenic Escherichia coli (EPEC) •Enterotoxigenic Escherichia coli (ETEC) lt/st</td></tr><tr><td></td><td></td><td></td></tr><tr><td></td><td></td><td>•Shiga-like toxin-producing Escherichia coli (STEC) stx1/stx2 (including specific</td></tr><tr><td></td><td>diagnosis, treatment, or other patient</td><td>identification of the E. coli O157</td></tr><tr><td></td><td>management decisions. Positive results do not rule out co-infection with other organisms that</td><td>serogroup within STEC) Shigella/Enteroinvasive Escherichia coli</td></tr><tr><td></td><td>are not detected by this test, and may not be</td><td>(EIEC)</td></tr><tr><td></td><td>the sole or definitive cause of patient illness. Negative results in the setting of clinical</td><td>Cryptosporidium •Cyclospora cayetanensis</td></tr><tr><td></td><td>illness compatible with gastroenteritis may be</td><td>Entamoeba histolytica</td></tr><tr><td></td><td>due to infection by pathogens that are not</td><td>•Giardia lamblia (also known as G.</td></tr><tr><td></td><td>detected by this test or non-infectious causes such as ulcerative colitis, irritable bowel</td><td>intestinalis and G. duodenalis) •Adenovirus F 40/41</td></tr><tr><td></td><td>syndrome, or Crohn's disease.</td><td>Astrovirus</td></tr><tr><td></td><td></td><td>Norovirus GI/GII Rotavirus A Sapovirus (Genogroups I, II, IV, and V)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">be used in conjunction with other clinical,laboratory, and epidemiological data.Positive results do not rule out co-infectionwith organisms not included in the FilmArrayGI Panel. The agent detected may not be thedefinite cause of the disease.Concomitant culture is necessary for organismrecovery and further typing of bacterialagents.This device is not intended to monitor orguide treatment for C. difficile infection.Due to the small number of positivespecimens collected for certain organismsduring the prospective clinical study,performance characteristics for E. coli O157,Plesiomonas shigelloides, Yersiniaenterocolitica, Astrovirus, and Rotavirus Awere established primarily with retrospectiveclinical specimens.Performance characteristics for Entamoebahistolytica, and Vibrio (V. parahaemolyticus,V. vulnificus, and Vibrio cholerae) wereestablished primarily using contrived clinicalspecimens.Negative FilmArray GI Panel results in thesetting of clinical illness compatible withgastroenteritis may be due to infection bypathogens that are not detected by this test ornon-infectious causes such asulcerative colitis, irritable bowel syndrome, orCrohn's disease.A gastrointestinal microorganism multiplexnucleic acid-based assay also aids in thedetection andidentification of acute gastroenteritis in thecontext of outbreaks.</td></tr><tr><td colspan="1" rowspan="1">Assay Format</td><td colspan="1" rowspan="1">Amplification: Real-time PCR</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Organisms Detected</td><td colspan="1" rowspan="1">Norovoirus GI &amp; GIIRotavirus AAdenovirus F40/41Sapovirus (genogroups I, II, IV, V)Human Astrovirus (hAstro)</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Assay Results</td><td colspan="1" rowspan="1">Qualitative</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Analyte</td><td colspan="1" rowspan="1">DNA/RNA</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Extraction</td><td colspan="1" rowspan="1">Automated by instrument</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">User Complexity</td><td colspan="1" rowspan="1">Moderate</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="3" rowspan="1">Differences</td></tr><tr><td colspan="1" rowspan="1">Item</td><td colspan="1" rowspan="1">Device: BD MAX Enteric ViralPanel (K181427)</td><td colspan="1" rowspan="1">Predicate: FilmArrayGastrointestinal Pathogen Panel(K143005)</td></tr><tr><td colspan="1" rowspan="1">Assay Format</td><td colspan="1" rowspan="1">Detection: Fluorigenic target-specific oligohybridization (TaqMan)</td><td colspan="1" rowspan="1">Detection: non-target-specific double strandedDNA fluorescent dye (LC Green Plus)</td></tr><tr><td colspan="1" rowspan="1">Assay Format</td><td colspan="1" rowspan="1">Single Real-time PCR reaction</td><td colspan="1" rowspan="1">Two-stage PCR amplification</td></tr><tr><td colspan="1" rowspan="1">Organisms Detected</td><td colspan="1" rowspan="1">See similarities above</td><td colspan="1" rowspan="1">Additional bacterial/parasitic analytes as listedin intended use statement above</td></tr><tr><td colspan="1" rowspan="1">Specimen Type</td><td colspan="1" rowspan="1">Cary-Blair preserved stool andunpreserved soft to diarrheal stool</td><td colspan="1" rowspan="1">Cary-Blair preserved stool only</td></tr><tr><td colspan="1" rowspan="1">Norovirus Target(s)</td><td colspan="1" rowspan="1">Junction RdRp and VP1 capsid gene</td><td colspan="1" rowspan="1">ORF1/2 junction</td></tr><tr><td colspan="1" rowspan="1">Adenovirus Target(s)</td><td colspan="1" rowspan="1">Hexon gene</td><td colspan="1" rowspan="1">Hexon Gene</td></tr><tr><td colspan="1" rowspan="1">Astrovirus Target(s)</td><td colspan="1" rowspan="1">RdRp gene</td><td colspan="1" rowspan="1">ORF1b</td></tr><tr><td colspan="1" rowspan="1">Rotavirus Target(s)</td><td colspan="1" rowspan="1">Non-coding sequence after non-structuralprotein 3 gene</td><td colspan="1" rowspan="1">VP1 and VP6</td></tr><tr><td colspan="1" rowspan="1">Sapovirus Target(s)</td><td colspan="1" rowspan="1">RdRp gene and Vp1 gene</td><td colspan="1" rowspan="1">ORF1 (RdRp)</td></tr><tr><td colspan="1" rowspan="1">Assay Controls</td><td colspan="1" rowspan="1">Sample Processing Control</td><td colspan="1" rowspan="1">Two controls in each pouch for sampleprocessing and both stages of PCR and meltanalysis</td></tr><tr><td colspan="1" rowspan="1">Time to result</td><td colspan="1" rowspan="1">Approximately 3 hours</td><td colspan="1" rowspan="1">Approximately 1 hour</td></tr><tr><td colspan="1" rowspan="1">Reagent Storage</td><td colspan="1" rowspan="1">2-25°C</td><td colspan="1" rowspan="1">15-25°C</td></tr></table>

# K. Standard/Guidance Document Referenced (if applicable):

Guidance for the Content of Premarket Submissions for Software Contained in Medical Devices

Molecular Diagnostic Methods for Infectious Diseases; Clinical and Laboratory Standards Institute (CLSI) – Third Edition, MM03

User Protocol for Evaluation of Qualitative Test Performance; Approved Guideline, Clinical and Laboratory Standards Institute (CLSI) – Second Edition, EP12

Guidance for Industry and Food and Drug Administration Staff - Class II Special Controls Guidance Document: Gastrointestinal Microorganism Multiplex Nucleic Acid-Based Assays for Detection and Identification of Microorganisms and Toxin Genes from Human Stool Specimens (November 2, 2015)

# L. Test Principle:

Stool specimens are collected from subjects and transported to the laboratory unpreserved in a clean container or preserved in Cary-Blair transport media. A loop is inserted to the depth of the loop into the specimen and expressed via swirling motion into a BD MAX Sample Buffer Tube included in the BD MAX Enteric Viral Panel kit. The Sample Buffer Tube is closed with a septum cap, vortexed and transferred to the BD MAX System. Once the work list is generated and the specimen is loaded on the BD MAX instrument, along with a BD

MAX Enteric Viral Panel Unitized Reagent Strip and PCR Cartridge, the run is started and no further operator intervention is required. The BD MAX System automates specimen preparation, including target organism lysis, DNA/RNA extraction and concentration, reagent rehydration, target nucleic acid sequence amplification and detection using real-time PCR. The interpretation of the signal is performed automatically by the BD MAX System. The assay also includes a Sample Processing Control that is provided in the Extraction Tube and subjected to extraction, concentration and amplification steps. The Sample Processing Control monitors for the presence of potential inhibitory substances as well as system or reagent failures. Following enzymatic viral lysis at an elevated temperature, the released nucleic acids are captured by magnetic affinity beads.

The beads, with the bound nucleic acids, are washed and the nucleic acids are eluted. Eluted DNA/RNA is neutralized and transferred to the Master Mix tubes to rehydrate the PCR reagents. After rehydration, the BD MAX System dispenses a fixed volume of PCR-ready solution into the BD MAX PCR Cartridge. Microvalves in the BD MAX PCR Cartridge are sealed by the system to prevent evaporation and amplicon contamination prior to the initiation of reverse transcriptase PCR to convert RNA to cDNA and subsequent real time PCR.

The amplified DNA targets are detected using hydrolysis (TaqMan) probes labeled at one end with a fluorescent reporter dye (fluorophore) and at the other end with a quencher moiety. Probes labeled with different fluorophores are used to detect the amplicons of the viral targets (Norovirus GI & GII, Rotavirus A, Adenovirus F40/41, Sapovirus (genogroups I, II, IV, V), and hAstro) and the Sample Processing Control amplicons in four different optical channels of the BD MAX System. When the probes are in their native state, the fluorescence of the fluorophore is quenched due to its proximity to the quencher. However, in the presence of target DNA, the probes hybridize to their complementary sequences and are hydrolyzed by the $5 ' - 3 '$ exonuclease activity of the DNA polymerase as it synthesizes the nascent strand along the cDNA template. As a result, the fluorophores are separated from the quencher molecules and fluorescence is emitted. The BD MAX System monitors these signals at each cycle and interprets the data at the end of the program to report the final results.

# M. Performance Characteristics (if/when applicable):

1. Analytical performance:

a. Precision/Reproducibility:

The reproducibility panel utilized in all of the precision/reproducibility studies consisted of 18 individual members. Three members of the panel contained no analytes. Each of the 15 remaining panel members consisted of one viral pathogen at low positive levels (2x LoD) and one viral pathogen at moderate positive levels (3x LoD) with all other viral targets absent. In total, each individual viral target of the Enteric Viral Panel was represented with 3 low positive (LP), 3 moderate positive (MP) and 12 true negative (TN) samples. However, the combination of each analyte per panel member is fixed. For example, the moderate positive Rotavirus specimens always contain low positive Norovirus. And the moderate positive Sapovirus samples always contain low positive Rotavirus. Proposed acceptance criteria included $2 9 5 \%$ agreement with expected results for all low positive specimens and $100 \%$ agreement with expected results for all moderate positive and true negative specimens. All study results met the acceptance criteria for all targets and all categories.

For the Site-to-Site Reproducibility study, a single lot of reagents was tested over the course of 5 days at three sites. Two users each completed a single run of 18 panel members each day for a total of 30 runs.

Table 1: Qualitative Site-to-Site Reproducibility Results   

<table><tr><td rowspan=3 colspan=1>Target</td><td rowspan=3 colspan=1>Category</td><td rowspan=3 colspan=1>X LoD</td><td rowspan=1 colspan=4>Agreement with Expected Results (95% CI)</td></tr><tr><td rowspan=1 colspan=3>Site</td><td rowspan=2 colspan=1>Overall</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>3</td></tr><tr><td rowspan=3 colspan=1>Norovirus</td><td rowspan=1 colspan=1>TN</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%(97.0-100%)</td><td rowspan=1 colspan=1>100%(97.0-100%)</td><td rowspan=1 colspan=1>100%(97.0-100%)</td><td rowspan=1 colspan=1>100%(99.0-100)</td></tr><tr><td rowspan=1 colspan=1>LP</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>100%(88.4-100)</td><td rowspan=1 colspan=1>100%(88.4-100)</td><td rowspan=1 colspan=1>100%(88.4-100)</td><td rowspan=1 colspan=1>100%(96.0-100)</td></tr><tr><td rowspan=1 colspan=1>MP</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>100%(88.4-100)</td><td rowspan=1 colspan=1>100%(88.4-100)</td><td rowspan=1 colspan=1>100%(88.4-100)</td><td rowspan=1 colspan=1>100%(96.0-100)</td></tr><tr><td rowspan=3 colspan=1>Rotavirus</td><td rowspan=1 colspan=1>TN</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%(97.0-100%)</td><td rowspan=1 colspan=1>100%(97.0-100%)</td><td rowspan=1 colspan=1>100%(97.0-100%)</td><td rowspan=1 colspan=1>100%(99.0-100)</td></tr><tr><td rowspan=1 colspan=1>LP</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>100%(88.4-100)</td><td rowspan=1 colspan=1>93.3%(77.9-99.2%)</td><td rowspan=1 colspan=1>100%(88.4-100)</td><td rowspan=1 colspan=1>97.8%(92.2-99.7)</td></tr><tr><td rowspan=1 colspan=1>MP</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>100%(88.4-100)</td><td rowspan=1 colspan=1>100%(88.4-100)</td><td rowspan=1 colspan=1>100%(88.4-100)</td><td rowspan=1 colspan=1>100%(96.0-100)</td></tr><tr><td rowspan=3 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>TN</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%(97.0-100%)</td><td rowspan=1 colspan=1>100%(97.0-100%)</td><td rowspan=1 colspan=1>100%(97.0-100%)</td><td rowspan=1 colspan=1>100%(99.0-100)</td></tr><tr><td rowspan=1 colspan=1>LP</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>100%(88.4-100)</td><td rowspan=1 colspan=1>100%(88.4-100)</td><td rowspan=1 colspan=1>100%(88.4-100)</td><td rowspan=1 colspan=1>100%(96.0-100)</td></tr><tr><td rowspan=1 colspan=1>MP</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>100%(88.4-100)</td><td rowspan=1 colspan=1>100%(88.4-100)</td><td rowspan=1 colspan=1>100%(88.4-100)</td><td rowspan=1 colspan=1>100%(96.0-100)</td></tr><tr><td rowspan=3 colspan=1>Sapovirus</td><td rowspan=1 colspan=1>TN</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%(97.0-100%)</td><td rowspan=1 colspan=1>100%(97.0-100%)</td><td rowspan=1 colspan=1>100%(97.0-100%)</td><td rowspan=1 colspan=1>100%(99.0-100)</td></tr><tr><td rowspan=1 colspan=1>LP</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>100%(88.4-100)</td><td rowspan=1 colspan=1>100%(88.4-100)</td><td rowspan=1 colspan=1>100%(88.4-100)</td><td rowspan=1 colspan=1>100%(96.0-100)</td></tr><tr><td rowspan=1 colspan=1>MP</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>100%(88.4-100)</td><td rowspan=1 colspan=1>90.0%(73.5-97.9%)</td><td rowspan=1 colspan=1>100%(88.4-100)</td><td rowspan=1 colspan=1>96.7%(90.6-99.3)</td></tr><tr><td rowspan=3 colspan=1>Astrovirus</td><td rowspan=1 colspan=1>TN</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%(97.0-100%)</td><td rowspan=1 colspan=1>100%(97.0-100%)</td><td rowspan=1 colspan=1>100%(97.0-100%)</td><td rowspan=1 colspan=1>100%(99.0-100)</td></tr><tr><td rowspan=1 colspan=1>LP</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>100%(88.4-100)</td><td rowspan=1 colspan=1>96.7%(82.8-99.9%)</td><td rowspan=1 colspan=1>100%(88.4-100)</td><td rowspan=1 colspan=1>98.9%(94.0-100)</td></tr><tr><td rowspan=1 colspan=1>MP</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>100%(88.4-100)</td><td rowspan=1 colspan=1>100%(88.4-100)</td><td rowspan=1 colspan=1>100%(88.4-100)</td><td rowspan=1 colspan=1>100%(96.0-100)</td></tr></table>

A quantitative analysis of site-to-site variance on the numerical outputs used by the BD MAX for sample classification (Ct.Score and Cycle EP) was also performed.

Table 2: Quantitative Site-to-Site Reproducibility Results   

<table><tr><td rowspan=2 colspan=1>Target</td><td rowspan=2 colspan=1>Metric</td><td rowspan=2 colspan=1>Sample</td><td rowspan=2 colspan=1>N</td><td rowspan=2 colspan=1>Mean(ct)</td><td rowspan=1 colspan=2>Within Run</td><td rowspan=1 colspan=2>Between RunWithin Day</td><td rowspan=1 colspan=2>Between DayWithin Site</td><td rowspan=1 colspan=2>Between Site</td><td rowspan=1 colspan=2>Total</td></tr><tr><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td></tr><tr><td rowspan=4 colspan=1>Norovirus</td><td rowspan=2 colspan=1>Ct.Score</td><td rowspan=1 colspan=1>LP</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>MP</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=2 colspan=1>Cycle EndPoint</td><td rowspan=1 colspan=1>LP</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>4553</td><td rowspan=1 colspan=1>1215</td><td rowspan=1 colspan=1>27</td><td rowspan=1 colspan=1>368</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1270</td><td rowspan=1 colspan=1>28</td></tr><tr><td rowspan=1 colspan=1>MP</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>4129</td><td rowspan=1 colspan=1>1760</td><td rowspan=1 colspan=1>43</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>178</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>1769</td><td rowspan=1 colspan=1>43</td></tr><tr><td rowspan=4 colspan=1>Rotavirus</td><td rowspan=2 colspan=1>Ct.Score</td><td rowspan=1 colspan=1>LP</td><td rowspan=1 colspan=1>88</td><td rowspan=1 colspan=1>32</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>2</td></tr><tr><td rowspan=1 colspan=1>MP</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>31</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>2</td></tr><tr><td rowspan=2 colspan=1>Cycle EndPoint</td><td rowspan=1 colspan=1>LP</td><td rowspan=1 colspan=1>88</td><td rowspan=1 colspan=1>738</td><td rowspan=1 colspan=1>238</td><td rowspan=1 colspan=1>32</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>270</td><td rowspan=1 colspan=1>37</td><td rowspan=1 colspan=1>360</td><td rowspan=1 colspan=1>49</td></tr><tr><td rowspan=1 colspan=1>MP</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>925</td><td rowspan=1 colspan=1>243</td><td rowspan=1 colspan=1>26</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>295</td><td rowspan=1 colspan=1>32</td><td rowspan=1 colspan=1>382</td><td rowspan=1 colspan=1>41</td></tr><tr><td rowspan=4 colspan=1>Adenovirus</td><td rowspan=2 colspan=1>Ct.Score</td><td rowspan=1 colspan=1>LP</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>29</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>2</td></tr><tr><td rowspan=1 colspan=1>MP</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>29</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>2</td></tr><tr><td rowspan=2 colspan=1>Cycle EndPoint</td><td rowspan=1 colspan=1>LP</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>1067</td><td rowspan=1 colspan=1>470</td><td rowspan=1 colspan=1>44</td><td rowspan=1 colspan=1>136</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>282</td><td rowspan=1 colspan=1>26</td><td rowspan=1 colspan=1>565</td><td rowspan=1 colspan=1>53</td></tr><tr><td rowspan=1 colspan=1>MP</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>1097</td><td rowspan=1 colspan=1>411</td><td rowspan=1 colspan=1>38</td><td rowspan=1 colspan=1>146</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>241</td><td rowspan=1 colspan=1>22</td><td rowspan=1 colspan=1>498</td><td rowspan=1 colspan=1>46</td></tr><tr><td rowspan=4 colspan=1>Sapovirus</td><td rowspan=2 colspan=1>Ct.Score</td><td rowspan=1 colspan=1>LP</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>29</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2</td></tr><tr><td rowspan=1 colspan=1>MP</td><td rowspan=1 colspan=1>87</td><td rowspan=1 colspan=1>29</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>2</td></tr><tr><td rowspan=2 colspan=1>Cycle EndPoint</td><td rowspan=1 colspan=1>LP</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>754</td><td rowspan=1 colspan=1>232</td><td rowspan=1 colspan=1>31</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>198</td><td rowspan=1 colspan=1>26</td><td rowspan=1 colspan=1>305</td><td rowspan=1 colspan=1>40</td></tr><tr><td rowspan=1 colspan=1>MP</td><td rowspan=1 colspan=1>87</td><td rowspan=1 colspan=1>963</td><td rowspan=1 colspan=1>271</td><td rowspan=1 colspan=1>28</td><td rowspan=1 colspan=1>44</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>25</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>152</td><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>315</td><td rowspan=1 colspan=1>33</td></tr><tr><td rowspan=4 colspan=1>Astrovirus</td><td rowspan=2 colspan=1>Ct.Score</td><td rowspan=1 colspan=1>LP</td><td rowspan=1 colspan=1>89</td><td rowspan=1 colspan=1>29</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2</td></tr><tr><td rowspan=1 colspan=1>MP</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>28</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=2 colspan=1>Cycle EndPoint</td><td rowspan=1 colspan=1>LP</td><td rowspan=1 colspan=1>89</td><td rowspan=1 colspan=1>3166</td><td rowspan=1 colspan=1>1110</td><td rowspan=1 colspan=1>35</td><td rowspan=1 colspan=1>71</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>343</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1164</td><td rowspan=1 colspan=1>37</td></tr><tr><td rowspan=1 colspan=1>MP</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>4786</td><td rowspan=1 colspan=1>1071</td><td rowspan=1 colspan=1>22</td><td rowspan=1 colspan=1>460</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>359</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>341</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>1266</td><td rowspan=1 colspan=1>27</td></tr><tr><td rowspan=2 colspan=1>SPC MM D6</td><td rowspan=1 colspan=1>Ct.Score</td><td rowspan=1 colspan=1>TN</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>27</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>Cycle EndPoint</td><td rowspan=1 colspan=1>TN</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>8582</td><td rowspan=1 colspan=1>927</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>537</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>448</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>161</td><td rowspan=1 colspan=1>14</td></tr><tr><td rowspan=2 colspan=1>SPC MM D5</td><td rowspan=1 colspan=1>Ct.Score</td><td rowspan=1 colspan=1>TN</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>28</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>Cycle EndPoint</td><td rowspan=1 colspan=1>TN</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>7668</td><td rowspan=1 colspan=1>1510</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>1010</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>707</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>1949</td><td rowspan=1 colspan=1>25</td></tr></table>

To evaluate lot-to-lot reproducibility, three lots of reagents were tested over five days at a single site. Two operators performed two runs per day for a total of 30 runs. Results from five days of the precision study were used as the final lot for analysis of lot-to-lot reproducibility. All lot-to-lot study results met the acceptance criteria for all targets and all categories.

The cycle end point fluorescence values typically have high standard deviations but also high absolute magnitudes (700-5,000) compared to the cut-off threshold for these values to make a positive call (70-200). A high degree of the observed variance is due to higher value outliers among a pool of relatively consistent datapoints that do not approach the threshold cutoff for these analytes.

Table 3: Qualitative Lot-to-Lot Reproducibility Results   

<table><tr><td rowspan=3 colspan=1>Target</td><td rowspan=3 colspan=1>Category</td><td rowspan=3 colspan=1>X LoD</td><td rowspan=1 colspan=4>Agreement with Expected Results (95% CI)</td></tr><tr><td rowspan=1 colspan=3>Lot</td><td rowspan=2 colspan=1>Overall</td></tr><tr><td rowspan=1 colspan=1>IUO3</td><td rowspan=1 colspan=1>IU04</td><td rowspan=1 colspan=1>IU05</td></tr><tr><td rowspan=3 colspan=1>Norovirus</td><td rowspan=1 colspan=1>TN</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%(97.0-100)</td><td rowspan=1 colspan=1>100%(97.0-100)</td><td rowspan=1 colspan=1>100%(97.0-100)</td><td rowspan=1 colspan=1>100%(99.0-100)</td></tr><tr><td rowspan=1 colspan=1>LP</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>100%(88.4-100)</td><td rowspan=1 colspan=1>100%(88.4-100)</td><td rowspan=1 colspan=1>100%(88.4-100)</td><td rowspan=1 colspan=1>100%(96.0-100)</td></tr><tr><td rowspan=1 colspan=1>MP</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>100%(88.4-100)</td><td rowspan=1 colspan=1>100%(88.4-100)</td><td rowspan=1 colspan=1>100%(88.4-100)</td><td rowspan=1 colspan=1>100%(96.0-100)</td></tr><tr><td rowspan=3 colspan=1>Rotavirus</td><td rowspan=1 colspan=1>TN</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>99.2%(95.4-99.9)</td><td rowspan=1 colspan=1>100%(97.0-100)</td><td rowspan=1 colspan=1>100%(97.0-100)</td><td rowspan=1 colspan=1>99.7%(98.4-100)</td></tr><tr><td rowspan=1 colspan=1>LP</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>100%(88.4-100)</td><td rowspan=1 colspan=1>93.3%(77.9-99.2%)</td><td rowspan=1 colspan=1>100%(88.4-100)</td><td rowspan=1 colspan=1>97.8%(92.2-99.7)</td></tr><tr><td rowspan=1 colspan=1>MP</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>100%(88.4-100)</td><td rowspan=1 colspan=1>100%(88.4-100)</td><td rowspan=1 colspan=1>100%(88.4-100)</td><td rowspan=1 colspan=1>100%(96.0-100)</td></tr><tr><td rowspan=3 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>TN</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%(97.0-100)</td><td rowspan=1 colspan=1>100%(97.0-100)</td><td rowspan=1 colspan=1>100%(97.0-100)</td><td rowspan=1 colspan=1>100%(99.0-100)</td></tr><tr><td rowspan=1 colspan=1>LP</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>100%(88.4-100)</td><td rowspan=1 colspan=1>100%(88.4-100)</td><td rowspan=1 colspan=1>100%(88.4-100)</td><td rowspan=1 colspan=1>100%(96.0-100)</td></tr><tr><td rowspan=1 colspan=1>MP</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>100%(88.4-100)</td><td rowspan=1 colspan=1>100%(88.4-100)</td><td rowspan=1 colspan=1>100%(88.4-100)</td><td rowspan=1 colspan=1>100%(96.0-100)</td></tr><tr><td rowspan=3 colspan=1>Sapovirus</td><td rowspan=1 colspan=1>TN</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%(97.0-100)</td><td rowspan=1 colspan=1>100%(97.0-100)</td><td rowspan=1 colspan=1>100%(97.0-100)</td><td rowspan=1 colspan=1>100%(99.0-100)</td></tr><tr><td rowspan=1 colspan=1>LP</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>96.7%(83.3-99.4)</td><td rowspan=1 colspan=1>100%(88.4-100)</td><td rowspan=1 colspan=1>100%(88.4-100)</td><td rowspan=1 colspan=1>98.9%(94.0-99.8)</td></tr><tr><td rowspan=1 colspan=1>MP</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>100%(88.4-100)</td><td rowspan=1 colspan=1>100%(88.4-100)</td><td rowspan=1 colspan=1>100%(88.4-100)</td><td rowspan=1 colspan=1>100%(96.0-100)</td></tr><tr><td rowspan=3 colspan=1>Astrovirus</td><td rowspan=1 colspan=1>TN</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%(97.0-100)</td><td rowspan=1 colspan=1>100%(97.0-100)</td><td rowspan=1 colspan=1>100%(97.0-100)</td><td rowspan=1 colspan=1>100%(99.0-100)</td></tr><tr><td rowspan=1 colspan=1>LP</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>100%(88.4-100)</td><td rowspan=1 colspan=1>100%(88.4-100)</td><td rowspan=1 colspan=1>100%(88.4-100)</td><td rowspan=1 colspan=1>100%(96.0-100)</td></tr><tr><td rowspan=1 colspan=1>MP</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>100%(88.4-100)</td><td rowspan=1 colspan=1>100%(88.4-100)</td><td rowspan=1 colspan=1>100%(88.4-100)</td><td rowspan=1 colspan=1>100%(96.0-100)</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr></table>

Table 4: Quantitative Lot-to-Lot Reproducibility Results   

<table><tr><td colspan="1" rowspan="2">Target</td><td colspan="1" rowspan="2">Metric</td><td colspan="1" rowspan="2">Sample</td><td colspan="1" rowspan="2">N</td><td colspan="1" rowspan="2">Mean</td><td colspan="2" rowspan="1">Within Run</td><td colspan="2" rowspan="1">Between RunWithin Day</td><td colspan="2" rowspan="1">Between DayWithin Site</td><td colspan="2" rowspan="1">Between Site</td><td colspan="2" rowspan="1">Total</td></tr><tr><td colspan="1" rowspan="1">SD</td><td colspan="1" rowspan="1">%CV</td><td colspan="1" rowspan="1">SD</td><td colspan="1" rowspan="1">%CV</td><td colspan="1" rowspan="1">SD</td><td colspan="1" rowspan="1">%CV</td><td colspan="1" rowspan="1">SD</td><td colspan="1" rowspan="1">%CV</td><td colspan="1" rowspan="1">SD</td><td colspan="1" rowspan="1">%CV</td></tr><tr><td colspan="1" rowspan="4">Norovirus</td><td colspan="1" rowspan="2">Ct.Score</td><td colspan="1" rowspan="1">LP</td><td colspan="1" rowspan="1">90</td><td colspan="1" rowspan="1">30</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">MP</td><td colspan="1" rowspan="1">90</td><td colspan="1" rowspan="1">30</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="2">Cycle EndPoint</td><td colspan="1" rowspan="1">LP</td><td colspan="1" rowspan="1">90</td><td colspan="1" rowspan="1">4946</td><td colspan="1" rowspan="1">1188</td><td colspan="1" rowspan="1">24</td><td colspan="1" rowspan="1">1606</td><td colspan="1" rowspan="1">33</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">705</td><td colspan="1" rowspan="1">14</td><td colspan="1" rowspan="1">2118</td><td colspan="1" rowspan="1">43</td></tr><tr><td colspan="1" rowspan="1">MP</td><td colspan="1" rowspan="1">90</td><td colspan="1" rowspan="1">4949</td><td colspan="1" rowspan="1">856</td><td colspan="1" rowspan="1">17</td><td colspan="1" rowspan="1">1702</td><td colspan="1" rowspan="1">34</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1208</td><td colspan="1" rowspan="1">24</td><td colspan="1" rowspan="1">2256</td><td colspan="1" rowspan="1">46</td></tr><tr><td colspan="1" rowspan="4">Rotavirus</td><td colspan="1" rowspan="2">Ct.Score</td><td colspan="1" rowspan="1">LP</td><td colspan="1" rowspan="1">90</td><td colspan="1" rowspan="1">32</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">2</td></tr><tr><td colspan="1" rowspan="1">MP</td><td colspan="1" rowspan="1">90</td><td colspan="1" rowspan="1">31</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="2">Cycle EndPoint</td><td colspan="1" rowspan="1">LP</td><td colspan="1" rowspan="1">90</td><td colspan="1" rowspan="1">1117</td><td colspan="1" rowspan="1">231</td><td colspan="1" rowspan="1">21</td><td colspan="1" rowspan="1">25</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">44</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">124</td><td colspan="1" rowspan="1">11</td><td colspan="1" rowspan="1">276</td><td colspan="1" rowspan="1">24</td></tr><tr><td colspan="1" rowspan="1">MP</td><td colspan="1" rowspan="1">90</td><td colspan="1" rowspan="1">1346</td><td colspan="1" rowspan="1">165</td><td colspan="1" rowspan="1">12</td><td colspan="1" rowspan="1">104</td><td colspan="1" rowspan="1">8</td><td colspan="1" rowspan="1">59</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">85</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">221</td><td colspan="1" rowspan="1">16</td></tr><tr><td colspan="1" rowspan="4">Adenovirus</td><td colspan="1" rowspan="2">Ct.Score</td><td colspan="1" rowspan="1">LP</td><td colspan="1" rowspan="1">90</td><td colspan="1" rowspan="1">28</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">2</td></tr><tr><td colspan="1" rowspan="1">MP</td><td colspan="1" rowspan="1">90</td><td colspan="1" rowspan="1">29</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">2</td></tr><tr><td colspan="1" rowspan="2">Cycle EndPoint</td><td colspan="1" rowspan="1">LP</td><td colspan="1" rowspan="1">90</td><td colspan="1" rowspan="1">1191</td><td colspan="1" rowspan="1">360</td><td colspan="1" rowspan="1">30</td><td colspan="1" rowspan="1">515</td><td colspan="1" rowspan="1">43</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">392</td><td colspan="1" rowspan="1">28</td><td colspan="1" rowspan="1">710</td><td colspan="1" rowspan="1">60</td></tr><tr><td colspan="1" rowspan="1">MP</td><td colspan="1" rowspan="1">90</td><td colspan="1" rowspan="1">1171</td><td colspan="1" rowspan="1">292</td><td colspan="1" rowspan="1">25</td><td colspan="1" rowspan="1">629</td><td colspan="1" rowspan="1">54</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">270</td><td colspan="1" rowspan="1">23</td><td colspan="1" rowspan="1">744</td><td colspan="1" rowspan="1">64</td></tr><tr><td colspan="1" rowspan="4">Sapovirus</td><td colspan="1" rowspan="2">Ct.Score</td><td colspan="1" rowspan="1">LP</td><td colspan="1" rowspan="1">89</td><td colspan="1" rowspan="1">29</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">MP</td><td colspan="1" rowspan="1">90</td><td colspan="1" rowspan="1">29</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="2">Cycle EndPoint</td><td colspan="1" rowspan="1">LP</td><td colspan="1" rowspan="1">89</td><td colspan="1" rowspan="1">1184</td><td colspan="1" rowspan="1">291</td><td colspan="1" rowspan="1">25</td><td colspan="1" rowspan="1">175</td><td colspan="1" rowspan="1">15</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">178</td><td colspan="1" rowspan="1">15</td><td colspan="1" rowspan="1">383</td><td colspan="1" rowspan="1">32</td></tr><tr><td colspan="1" rowspan="1">MP</td><td colspan="1" rowspan="1">90</td><td colspan="1" rowspan="1">1311</td><td colspan="1" rowspan="1">382</td><td colspan="1" rowspan="1">29</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">138</td><td colspan="1" rowspan="1">11</td><td colspan="1" rowspan="1">406</td><td colspan="1" rowspan="1">31</td></tr><tr><td colspan="1" rowspan="4">Astrovirus</td><td colspan="1" rowspan="2">Ct.Score</td><td colspan="1" rowspan="1">LP</td><td colspan="1" rowspan="1">90</td><td colspan="1" rowspan="1">29</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">2</td></tr><tr><td colspan="1" rowspan="1">MP</td><td colspan="1" rowspan="1">90</td><td colspan="1" rowspan="1">28</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="2">Cycle EndPoint</td><td colspan="1" rowspan="1">LP</td><td colspan="1" rowspan="1">90</td><td colspan="1" rowspan="1">3278</td><td colspan="1" rowspan="1">575</td><td colspan="1" rowspan="1">18</td><td colspan="1" rowspan="1">579</td><td colspan="1" rowspan="1">18</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">500</td><td colspan="1" rowspan="1">14</td><td colspan="1" rowspan="1">932</td><td colspan="1" rowspan="1">28</td></tr><tr><td colspan="1" rowspan="1">MP</td><td colspan="1" rowspan="1">90</td><td colspan="1" rowspan="1">4724</td><td colspan="1" rowspan="1">543</td><td colspan="1" rowspan="1">12</td><td colspan="1" rowspan="1">558</td><td colspan="1" rowspan="1">12</td><td colspan="1" rowspan="1">105</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">512</td><td colspan="1" rowspan="1">11</td><td colspan="1" rowspan="1">938</td><td colspan="1" rowspan="1">20</td></tr><tr><td colspan="1" rowspan="2">SPC MMD6</td><td colspan="1" rowspan="1">Ct.Score</td><td colspan="1" rowspan="1">TN</td><td colspan="1" rowspan="1">90</td><td colspan="1" rowspan="1">27</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">2</td></tr><tr><td colspan="1" rowspan="1">Cycle EndPoint</td><td colspan="1" rowspan="1">TN</td><td colspan="1" rowspan="1">90</td><td colspan="1" rowspan="1">8994</td><td colspan="1" rowspan="1">651</td><td colspan="1" rowspan="1">7</td><td colspan="1" rowspan="1">667</td><td colspan="1" rowspan="1">7</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">259</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">967</td><td colspan="1" rowspan="1">11</td></tr><tr><td colspan="1" rowspan="2">SPC MMD5</td><td colspan="1" rowspan="1">Ct.Score</td><td colspan="1" rowspan="1">TN</td><td colspan="1" rowspan="1">90</td><td colspan="1" rowspan="1">27</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">2</td></tr><tr><td colspan="1" rowspan="1">Cycle EndPoint</td><td colspan="1" rowspan="1">TN</td><td colspan="1" rowspan="1">90</td><td colspan="1" rowspan="1">8740</td><td colspan="1" rowspan="1">834</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">1198</td><td colspan="1" rowspan="1">14</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">565</td><td colspan="1" rowspan="1">7</td><td colspan="1" rowspan="1">1565</td><td colspan="1" rowspan="1">18</td></tr></table>

The precision study utilized only 12 of the 18 original reproducibility panel members A single lot of reagents were tested over 12 days at a single site with two users completing a single run on each day of the study. The composition of the panel included true negative (TN), low positive (LP), and moderate positive (MP) reactivities for each analyte. A total of 24 runs were performed.

Table 5: Qualitative Precision Results   

<table><tr><td rowspan=2 colspan=1>Category</td><td rowspan=1 colspan=5>Agreement with Expected Results</td></tr><tr><td rowspan=1 colspan=1>Norovirus</td><td rowspan=1 colspan=1>Rotavirus</td><td rowspan=1 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>Sapovirus</td><td rowspan=1 colspan=1>Astrovirus</td></tr><tr><td rowspan=1 colspan=1>TN</td><td rowspan=1 colspan=1>100%(98-100)</td><td rowspan=1 colspan=1>99.5%(97.1-99.9)</td><td rowspan=1 colspan=1>100%(98.0-100)</td><td rowspan=1 colspan=1>100%(98.0-100)</td><td rowspan=1 colspan=1>100%(98.0-100)</td></tr><tr><td rowspan=1 colspan=1>LP</td><td rowspan=1 colspan=1>100%(92.6-100)</td><td rowspan=1 colspan=1>100%(92.6-100)</td><td rowspan=1 colspan=1>100%(92.6-100)</td><td rowspan=1 colspan=1>100%(92.6-100)</td><td rowspan=1 colspan=1>100%(92.6-100)</td></tr><tr><td rowspan=1 colspan=1>MP</td><td rowspan=1 colspan=1>100%(92.6-100)</td><td rowspan=1 colspan=1>100%(92.6-100)</td><td rowspan=1 colspan=1>100%(92.6-100)</td><td rowspan=1 colspan=1>100%(92.6-100)</td><td rowspan=1 colspan=1>100%(92.6-100)</td></tr></table>

Again, quantitative analysis was performed on numerical PCR metrics from the precision study (Table 6) and while the Cycle EP values do sometimes exhibit high variance, especially in low positive panel members, this is usually due to the presence of higher value outliers. No values are near the cutoff threshold for each analyte.

The largest Ct.Score variability observed during the quantitative Precision study segment (among TN, LP and MP) was obtained for the Adenovirus target, which obtained a $2 . 1 \%$ overall variability for the LP and MP. Overall variability values for Ct.Score obtained for the other targets ranged between $0 . 9 \%$ and $1 . 8 \%$ . In Cycle EP, variability for the LP and MP ranged from $1 5 . 5 \%$ to $6 4 . 7 \%$ .

Table 6: Quantitative Precision Results   

<table><tr><td rowspan=2 colspan=1>Target</td><td rowspan=2 colspan=1>Metric</td><td rowspan=2 colspan=1>Sample</td><td rowspan=2 colspan=1>N</td><td rowspan=2 colspan=1>Mean</td><td rowspan=1 colspan=2>Within Run</td><td rowspan=1 colspan=2>Between RunWithin Day</td><td rowspan=1 colspan=2>Between Day</td><td rowspan=1 colspan=2>Total</td></tr><tr><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td></tr><tr><td rowspan=4 colspan=1>Norovirus</td><td rowspan=2 colspan=1>Ct.Score</td><td rowspan=1 colspan=1>LP</td><td rowspan=1 colspan=1>48</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>MP</td><td rowspan=1 colspan=1>48</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=2 colspan=1>Cycle EndPoint</td><td rowspan=1 colspan=1>LP</td><td rowspan=1 colspan=1>48</td><td rowspan=1 colspan=1>4416</td><td rowspan=1 colspan=1>911</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>1857</td><td rowspan=1 colspan=1>42</td><td rowspan=1 colspan=1>200</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>2078</td><td rowspan=1 colspan=1>47</td></tr><tr><td rowspan=1 colspan=1>MP</td><td rowspan=1 colspan=1>48</td><td rowspan=1 colspan=1>4727</td><td rowspan=1 colspan=1>803</td><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>1470</td><td rowspan=1 colspan=1>31</td><td rowspan=1 colspan=1>1051</td><td rowspan=1 colspan=1>22</td><td rowspan=1 colspan=1>1978</td><td rowspan=1 colspan=1>42</td></tr><tr><td rowspan=4 colspan=1>Rotavirus</td><td rowspan=2 colspan=1>Ct.Score</td><td rowspan=1 colspan=1>LP</td><td rowspan=1 colspan=1>48</td><td rowspan=1 colspan=1>32</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>2</td></tr><tr><td rowspan=1 colspan=1>MP</td><td rowspan=1 colspan=1>48</td><td rowspan=1 colspan=1>31</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=2 colspan=1>Cycle EndPoint</td><td rowspan=1 colspan=1>LP</td><td rowspan=1 colspan=1>48</td><td rowspan=1 colspan=1>1050</td><td rowspan=1 colspan=1>242</td><td rowspan=1 colspan=1>23</td><td rowspan=1 colspan=1>97</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>80</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>273</td><td rowspan=1 colspan=1>26</td></tr><tr><td rowspan=1 colspan=1>MP</td><td rowspan=1 colspan=1>48</td><td rowspan=1 colspan=1>1269</td><td rowspan=1 colspan=1>158</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>132</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>84</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>222</td><td rowspan=1 colspan=1>18</td></tr><tr><td rowspan=4 colspan=1>Adenovirus</td><td rowspan=2 colspan=1>Ct.Score</td><td rowspan=1 colspan=1>LP</td><td rowspan=1 colspan=1>48</td><td rowspan=1 colspan=1>28</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>2</td></tr><tr><td rowspan=1 colspan=1>MP</td><td rowspan=1 colspan=1>48</td><td rowspan=1 colspan=1>29</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>2</td></tr><tr><td rowspan=2 colspan=1>Cycle EndPoint</td><td rowspan=1 colspan=1>LP</td><td rowspan=1 colspan=1>48</td><td rowspan=1 colspan=1>1171</td><td rowspan=1 colspan=1>315</td><td rowspan=1 colspan=1>27</td><td rowspan=1 colspan=1>510</td><td rowspan=1 colspan=1>44</td><td rowspan=1 colspan=1>165</td><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>622</td><td rowspan=1 colspan=1>53</td></tr><tr><td rowspan=1 colspan=1>MP</td><td rowspan=1 colspan=1>48</td><td rowspan=1 colspan=1>1119</td><td rowspan=1 colspan=1>335</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>641</td><td rowspan=1 colspan=1>57</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>724</td><td rowspan=1 colspan=1>65</td></tr><tr><td rowspan=4 colspan=1>Sapovirus</td><td rowspan=2 colspan=1>Ct.Score</td><td rowspan=1 colspan=1>LP</td><td rowspan=1 colspan=1>48</td><td rowspan=1 colspan=1>29</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>MP</td><td rowspan=1 colspan=1>48</td><td rowspan=1 colspan=1>28</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=2 colspan=1>Cycle EndPoint</td><td rowspan=1 colspan=1>LP</td><td rowspan=1 colspan=1>48</td><td rowspan=1 colspan=1>1205</td><td rowspan=1 colspan=1>170</td><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>199</td><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>262</td><td rowspan=1 colspan=1>22</td></tr><tr><td rowspan=1 colspan=1>MP</td><td rowspan=1 colspan=1>48</td><td rowspan=1 colspan=1>1333</td><td rowspan=1 colspan=1>266</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>85</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>126</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>306</td><td rowspan=1 colspan=1>23</td></tr><tr><td rowspan=4 colspan=1>Astrovirus</td><td rowspan=2 colspan=1>Ct.Score</td><td rowspan=1 colspan=1>LP</td><td rowspan=1 colspan=1>48</td><td rowspan=1 colspan=1>29</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>2</td></tr><tr><td rowspan=1 colspan=1>MP</td><td rowspan=1 colspan=1>48</td><td rowspan=1 colspan=1>28</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=2 colspan=1>Cycle EndPoint</td><td rowspan=1 colspan=1>LP</td><td rowspan=1 colspan=1>48</td><td rowspan=1 colspan=1>3579</td><td rowspan=1 colspan=1>901</td><td rowspan=1 colspan=1>25</td><td rowspan=1 colspan=1>406</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>311</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>1036</td><td rowspan=1 colspan=1>29</td></tr><tr><td rowspan=1 colspan=1>MP</td><td rowspan=1 colspan=1>48</td><td rowspan=1 colspan=1>5716</td><td rowspan=1 colspan=1>670</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>413</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>152</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>802</td><td rowspan=1 colspan=1>16</td></tr><tr><td rowspan=2 colspan=1>SPC MM D6</td><td rowspan=1 colspan=1>Ct.Score</td><td rowspan=1 colspan=1>TN</td><td rowspan=1 colspan=1>48</td><td rowspan=1 colspan=1>27</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>Cycle EndPoint</td><td rowspan=1 colspan=1>TN</td><td rowspan=1 colspan=1>48</td><td rowspan=1 colspan=1>8666</td><td rowspan=1 colspan=1>754</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>853</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1138</td><td rowspan=1 colspan=1>13</td></tr><tr><td rowspan=2 colspan=1>SPC MM D5</td><td rowspan=1 colspan=1>Ct.Score</td><td rowspan=1 colspan=1>TN</td><td rowspan=1 colspan=1>48</td><td rowspan=1 colspan=1>27</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>Cycle EndPoint</td><td rowspan=1 colspan=1>TN</td><td rowspan=1 colspan=1>48</td><td rowspan=1 colspan=1>8421</td><td rowspan=1 colspan=1>1336</td><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>1261</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1837</td><td rowspan=1 colspan=1>22</td></tr></table>

b. Linearity/assay reportable range: Not applicable.   
c. Traceability, Stability, Expected values (controls, calibrators, or methods):

# Specimen Stability:

According to the package insert, specimens can be stored for up to 120 hours (5 days) at $2 { - } 8 ^ { \circ } \mathrm { C }$ or for up to 48 hours at $2 { - } 2 5 ^ { \circ } \mathrm { C }$ before testing. Similarly, prepared SBTs can be stored at $2 { - } 8 ^ { \circ } \mathrm { C }$ for a maximum of 120 hours (5 days) OR at $2 { - } 2 5 ^ { \circ } \mathrm { C }$ for a maximum of 48 hours. Studies to support this stability claim for both CaryBlair preserved and unpreserved stool specimens followed the below scheme (Tables 7 & 8):

Table 7: Sample Stability Testing Scheme - Round 1   

<table><tr><td rowspan=2 colspan=1>Test Point</td><td rowspan=1 colspan=2>Neat</td><td rowspan=1 colspan=2>SBT</td><td rowspan=2 colspan=1>MinimumTotal Hours</td></tr><tr><td rowspan=1 colspan=2>Storage Condition 1</td><td rowspan=1 colspan=2>Storage Condition 2</td></tr><tr><td rowspan=1 colspan=1>1 (Baseline)</td><td rowspan=1 colspan=4>N/A</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=3 colspan=1>25°C</td><td rowspan=3 colspan=1>48 hrs</td><td rowspan=1 colspan=2>N/A</td><td rowspan=1 colspan=1>48</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>25°C</td><td rowspan=1 colspan=1>24 hrs</td><td rowspan=1 colspan=1>72</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>2-8°C</td><td rowspan=1 colspan=1>120 hrs</td><td rowspan=1 colspan=1>168</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=3 colspan=1>2-8°C</td><td rowspan=3 colspan=1>120 hrs</td><td rowspan=1 colspan=2>N/A</td><td rowspan=1 colspan=1>120</td></tr><tr><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>25°C</td><td rowspan=1 colspan=1>24 hrs</td><td rowspan=1 colspan=1>144</td></tr><tr><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>2-8°C</td><td rowspan=1 colspan=1>120 hrs</td><td rowspan=1 colspan=1>240</td></tr></table>

Table 8: Sample Stability Testing Scheme - Round 2   

<table><tr><td rowspan=2 colspan=1>Test Point</td><td rowspan=1 colspan=2>Neat</td><td rowspan=1 colspan=2>SBT</td><td rowspan=2 colspan=1>MinimumTotal Hours</td></tr><tr><td rowspan=1 colspan=2>Storage Condition 1</td><td rowspan=1 colspan=2>Storage Condition 2</td></tr><tr><td rowspan=1 colspan=1>1 (Baseline)</td><td rowspan=1 colspan=4>N/A</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>25°C</td><td rowspan=1 colspan=1>48 hrs</td><td rowspan=1 colspan=1>25°C</td><td rowspan=1 colspan=1>48 hrs</td><td rowspan=1 colspan=1>96</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>2-8°C</td><td rowspan=1 colspan=1>120 hrs</td><td rowspan=1 colspan=1>25°C</td><td rowspan=1 colspan=1>48 hrs</td><td rowspan=1 colspan=1>168</td></tr></table>

A minimum of twenty valid replicates for positives and four valid replicates for negatives were required for each condition tested. Baseline testing was required to yield $100 \%$ (20/20) positive replicates and $100 \%$ (4/4) negative replicates. Each subsequent test point was required to yield $\ge 9 5 \%$ (19/20) positive replicates and $100 \%$ (4/4) negative replicates. Additionally, a trend of the quantitative PCR curve metrics for each channel over time was modeled using simple linear regression analysis. The stability break point at which the specimen was no longer considered stable was reached when the one-sided $9 5 \%$ lower confidence interval of the linear regression crossed the acceptable drift limit (“ADL”) in any channel. The ADL was determined for each target at baseline, which was derived from the mean Ct.Score endpoint thresholds.

Data from the specimen stability study supports claims in the package insert, with no observed change in sample status (Table 9 and Table 10) or significant loss of cycle EP . One run of test point 4 for unpreserved stool samples included several samples with unexpected negative results. The run was repeated to exclude sample stability issues and this data is presented in Table 10. Regression lines of quantitative PCR parameters were plotted with the $9 5 \%$ confidence interval and the slope of the regression analysis determined for each sample and condition. In no instance did the lower $9 5 \%$ confidence bound intersect the lower specification (EP min) within the duration of the study period.

Table 9: Cary-Blair Preserved Stool Sample Stability   

<table><tr><td rowspan=2 colspan=1>TestPoint</td><td rowspan=1 colspan=2>Neat</td><td rowspan=1 colspan=2>SBT</td><td rowspan=1 colspan=1>Min.</td><td rowspan=1 colspan=5>Positive Results</td><td rowspan=2 colspan=1>NegativeResults</td><td rowspan=2 colspan=1>UNR/IND/INCResults</td></tr><tr><td rowspan=1 colspan=2>StorageCondition 1</td><td rowspan=1 colspan=2>StorageCondition 2</td><td rowspan=1 colspan=1>TotalHrs</td><td rowspan=1 colspan=1>AdVF41</td><td rowspan=1 colspan=1>AdVType 4</td><td rowspan=1 colspan=1>NoV GII</td><td rowspan=1 colspan=1>RoVVa70</td><td rowspan=1 colspan=1>SaV GI</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=2></td><td rowspan=1 colspan=2></td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>4/4</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=2 colspan=1>25°</td><td rowspan=2 colspan=1>48 h</td><td rowspan=1 colspan=2>N/A</td><td rowspan=1 colspan=1>48</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>4/4</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>25°C</td><td rowspan=1 colspan=1>24 h</td><td rowspan=1 colspan=1>72</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>19/20</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>4/4</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>1a</td><td rowspan=1 colspan=2></td><td rowspan=1 colspan=2></td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>19/20</td><td rowspan=1 colspan=1>19/20</td><td rowspan=1 colspan=1>19/20</td><td rowspan=1 colspan=1>19/20</td><td rowspan=1 colspan=1>19/20</td><td rowspan=1 colspan=1>4/4</td><td rowspan=1 colspan=1>5</td></tr><tr><td rowspan=1 colspan=1>1b</td><td rowspan=1 colspan=4>Baseline</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1/1</td><td rowspan=1 colspan=1>1/1</td><td rowspan=1 colspan=1>1/1</td><td rowspan=1 colspan=1>1/1</td><td rowspan=1 colspan=1>1/1</td><td rowspan=1 colspan=1>1/1</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=2 colspan=1>25°</td><td rowspan=2 colspan=1>48 h</td><td rowspan=2 colspan=1>2-8</td><td rowspan=2 colspan=1>120h</td><td rowspan=2 colspan=1>168</td><td rowspan=1 colspan=1>19/20</td><td rowspan=1 colspan=1>19/20</td><td rowspan=1 colspan=1>19/20</td><td rowspan=1 colspan=1>19/20</td><td rowspan=1 colspan=1>19/20</td><td rowspan=1 colspan=1>4/4</td><td rowspan=1 colspan=1>5</td></tr><tr><td rowspan=1 colspan=1>4c</td><td rowspan=1 colspan=1>1/1</td><td rowspan=1 colspan=1>1/1</td><td rowspan=1 colspan=1>1/1</td><td rowspan=1 colspan=1>1/1</td><td rowspan=1 colspan=1>1/1</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=3 colspan=1>2-8</td><td rowspan=3 colspan=1>120</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>120</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>4/4</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>25°C</td><td rowspan=1 colspan=1>24 h</td><td rowspan=1 colspan=1>144</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>19/20</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>4/4</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>2-8°C</td><td rowspan=1 colspan=1>120 h</td><td rowspan=1 colspan=1>240</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>19/20</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>4/4</td><td rowspan=1 colspan=1>0</td></tr></table>

aBaseline repeated due to instrument malfunction bSample retested to obtain valid result cSample repeated due to IND result

Table 10: Unpreserved Stool Sample Stability   

<table><tr><td rowspan=2 colspan=1>TestPoint</td><td rowspan=1 colspan=2>Neat</td><td rowspan=1 colspan=2>SBT</td><td rowspan=1 colspan=1>Min.TotalHrs</td><td rowspan=1 colspan=5>Positive Results</td><td rowspan=1 colspan=1>NegativeResults</td><td rowspan=1 colspan=1>UNR/IND/INCResults</td></tr><tr><td rowspan=1 colspan=2>StorageCondition 1</td><td rowspan=1 colspan=2>StorageCondition 2</td><td rowspan=1 colspan=1>Hrs</td><td rowspan=1 colspan=1>AdVF41</td><td rowspan=1 colspan=1>AdVType 4</td><td rowspan=1 colspan=1>NoV GII</td><td rowspan=1 colspan=1>RoVVa70</td><td rowspan=1 colspan=1>SaV GI</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=2>Baseline</td><td rowspan=1 colspan=2></td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>4/4</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>2a</td><td rowspan=2 colspan=1>25°</td><td rowspan=2 colspan=1>48 h</td><td rowspan=1 colspan=2>N/A</td><td rowspan=1 colspan=1>48</td><td rowspan=1 colspan=1>38/40</td><td rowspan=1 colspan=1>40/40</td><td rowspan=1 colspan=1>38/40</td><td rowspan=1 colspan=1>38/40</td><td rowspan=1 colspan=1>40/40</td><td rowspan=1 colspan=1>8/8</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>25°C</td><td rowspan=1 colspan=1>24 h</td><td rowspan=1 colspan=1>72</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>4/4</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>4b</td><td rowspan=2 colspan=1>25°</td><td rowspan=2 colspan=1>48 h</td><td rowspan=2 colspan=1>2-8</td><td rowspan=2 colspan=1>120h</td><td rowspan=2 colspan=1>168</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>4/4</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>4b</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=3 colspan=1>2-8</td><td rowspan=3 colspan=1>120</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>120</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>4/4</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>25°C</td><td rowspan=1 colspan=1>24 h</td><td rowspan=1 colspan=1>144</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>4/4</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>2-8°C</td><td rowspan=1 colspan=1>120 h</td><td rowspan=1 colspan=1>240</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>19/20</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>19/20</td><td rowspan=1 colspan=1>4/4</td><td rowspan=1 colspan=1>0</td></tr></table>

aInstrument/reagent error caused repeat testing bRetest to ensure original results were due to instrument error

A fresh vs. frozen study was performed to demonstrate no difference in assay performance upon repeated freeze-thaw cycles and validate the use of frozen retrospective samples in the clinical studies. A total of 60 replicates for each analyte present at $1 . 9 9 \mathrm { { x } }$ LoD $( \mathrm { N } = 3 0 )$ , 4x LoD $( \Nu = 1 5 )$ ), and $1 0 \mathrm { x }$ LoD $( \mathrm { N } = 1 5 )$ ) were analyzed at baseline and after 1, 2, or 3 freeze-thaw cycles. All data agreed with the expected results (Table 11) and the data demonstrated no significant drift in Ct cycle values.

Table 11: Freeze-Thaw Study   

<table><tr><td rowspan=1 colspan=2>Condition</td><td rowspan=1 colspan=5>Proportion Positive (%)a</td></tr><tr><td rowspan=1 colspan=1>Matrix</td><td rowspan=1 colspan=1>Freeze-ThawCycles</td><td rowspan=1 colspan=1>NorovirusGII</td><td rowspan=1 colspan=1>RotavirusVa70</td><td rowspan=1 colspan=1>AdenovirusF41</td><td rowspan=1 colspan=1>AstrovirusType 4</td><td rowspan=1 colspan=1>SapovirusGI</td></tr><tr><td rowspan=4 colspan=1>Unpreserved</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>60/60(100)</td><td rowspan=1 colspan=1>60/60(100)</td><td rowspan=1 colspan=1>60/60 (100)</td><td rowspan=1 colspan=1>60/60(100)</td><td rowspan=1 colspan=1>60/60(100)</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>60/60(100)</td><td rowspan=1 colspan=1>60/60(100)</td><td rowspan=1 colspan=1>60/60 (100)</td><td rowspan=1 colspan=1>60/60(100)</td><td rowspan=1 colspan=1>60/60(100)</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>60/60(100)</td><td rowspan=1 colspan=1>60/60(100)</td><td rowspan=1 colspan=1>60/60 (100)</td><td rowspan=1 colspan=1>60/60(100)</td><td rowspan=1 colspan=1>60/60(100)</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>60/60(100)</td><td rowspan=1 colspan=1>60/60(100)</td><td rowspan=1 colspan=1>60/60 (100)</td><td rowspan=1 colspan=1>60/60(100)</td><td rowspan=1 colspan=1>60/60(100)</td></tr><tr><td rowspan=4 colspan=1>Preserved</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>60/60(100)</td><td rowspan=1 colspan=1>60/60(100)</td><td rowspan=1 colspan=1>60/60 (100)</td><td rowspan=1 colspan=1>60/60(100)</td><td rowspan=1 colspan=1>60/60(100)</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>60/60(100)</td><td rowspan=1 colspan=1>60/60(100)</td><td rowspan=1 colspan=1>60/60 (100)</td><td rowspan=1 colspan=1>60/60(100)</td><td rowspan=1 colspan=1>60/60(100)</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>60/60(100)</td><td rowspan=1 colspan=1>60/60(100)</td><td rowspan=1 colspan=1>60/60 (100)</td><td rowspan=1 colspan=1>60/60(100)</td><td rowspan=1 colspan=1>60/60(100)</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>60/60(100)</td><td rowspan=1 colspan=1>60/60(100)</td><td rowspan=1 colspan=1>60/60 (100)</td><td rowspan=1 colspan=1>60/60(100)</td><td rowspan=1 colspan=1>60/60(100)</td></tr></table>

aProportion positive rates were calculated based on sample number after exclusion of IND/UNR results

# Internal Control Effectiveness

The sample processing control is designed to flag the user to potentially disruptive errors in either sample extraction and nucleic acid purification or inhibition of the PCR reaction. Samples that are reactive in the absence of a positive SPC control signal will still be called positive; however, negative samples will be labeled unresolved and necessitate repeat testing. To validate proper performance of the SPC control, the sponsor performed 20 runs of the assay under normal conditions (control), improper extraction (I), and absence of a mastermix tube (II). These conditions were evaluated for each mastermix in the assay. All test runs agreed with expected results except for one improper extraction sample that actually exhibited unexpected SPC amplification (Table 12).

Table 12: SPC Effectiveness Testing   

<table><tr><td rowspan=2 colspan=1>Condition</td><td rowspan=2 colspan=1>#Tested</td><td rowspan=2 colspan=1>ExpectedResult</td><td rowspan=1 colspan=5>Results</td><td rowspan=2 colspan=1>%Agreement</td></tr><tr><td rowspan=1 colspan=1>IND</td><td rowspan=1 colspan=1>INC</td><td rowspan=1 colspan=1>UNR</td><td rowspan=1 colspan=1>POS</td><td rowspan=1 colspan=1>NEG</td></tr><tr><td rowspan=1 colspan=1>Control</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>IMM D6</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>Unresolved</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>19</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>95%</td></tr><tr><td rowspan=1 colspan=1>IMM D5</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>Unresolved</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>IMM D6</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>Indeterminate</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>IMM D5</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>Indeterminate</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%</td></tr></table>

# d. Detection limit:

Individual sample buffer tubes were spiked with the viral pathogens detected by the BD MAX Enteric Viral Panel assay in the presence of either unpreserved or CaryBlair preserved stool matrix. A putative LoD was determined by testing 4 replicates across a range of 5 analyte levels using 3 lots of assay reagents. Further confirmation of LoD was determined by additional testing of 20 replicates near the putative LoD levels to achieve $9 5 \%$ detection. The final LoD determination for each analyte in either Unpreserved and Cary-Blair preserved stool matrix is displayed in Table 13 and Table 14, respectively.

Table 13: BD MAX Enteric Viral Panel LoD Results in Unpreserved Stool Matrix   

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>LoD (cp/mL)</td><td rowspan=1 colspan=1>Positivity</td><td rowspan=1 colspan=1>95%ConfidenceInterval</td></tr><tr><td rowspan=2 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>F40</td><td rowspan=1 colspan=1>6.89E+04</td><td rowspan=1 colspan=1>95% (19/20)</td><td rowspan=1 colspan=1>76.4% - 99.1%</td></tr><tr><td rowspan=1 colspan=1>F41</td><td rowspan=1 colspan=1>8.16E+04</td><td rowspan=1 colspan=1>100% (20/20)</td><td rowspan=1 colspan=1>83.9% - 100%</td></tr><tr><td rowspan=2 colspan=1>Astrovirus</td><td rowspan=1 colspan=1>Type 4</td><td rowspan=1 colspan=1>1.75E+07</td><td rowspan=1 colspan=1>100% (20/20)</td><td rowspan=1 colspan=1>83.9% - 100%</td></tr><tr><td rowspan=1 colspan=1>Type 8</td><td rowspan=1 colspan=1>5.23E+06</td><td rowspan=1 colspan=1>100% (20/20)</td><td rowspan=1 colspan=1>83.9% - 100%</td></tr><tr><td rowspan=2 colspan=1>Rotavirus</td><td rowspan=1 colspan=1>WA</td><td rowspan=1 colspan=1>6.46E+03</td><td rowspan=1 colspan=1>95% (19/20)</td><td rowspan=1 colspan=1>76.4% - 99.1%</td></tr><tr><td rowspan=1 colspan=1>Va70</td><td rowspan=1 colspan=1>1.16E+04</td><td rowspan=1 colspan=1>100% (20/20)</td><td rowspan=1 colspan=1>83.9% - 100%</td></tr><tr><td rowspan=2 colspan=1>Norovirus</td><td rowspan=1 colspan=1>GI</td><td rowspan=1 colspan=1>6.28E+06</td><td rowspan=1 colspan=1>95% (19/20)</td><td rowspan=1 colspan=1>76.4% - 99.1%</td></tr><tr><td rowspan=1 colspan=1>GII</td><td rowspan=1 colspan=1>2.49E+05</td><td rowspan=1 colspan=1>95% (19/20)</td><td rowspan=1 colspan=1>76.4% - 99.1%</td></tr><tr><td rowspan=2 colspan=1>Sapovirus</td><td rowspan=1 colspan=1>GI</td><td rowspan=1 colspan=1>6.51E+07</td><td rowspan=1 colspan=1>95% (19/20)</td><td rowspan=1 colspan=1>76.4% - 99.1%</td></tr><tr><td rowspan=1 colspan=1>GII</td><td rowspan=1 colspan=1>1.94E+06</td><td rowspan=1 colspan=1>100% (20/20)</td><td rowspan=1 colspan=1>83.9% - 100%</td></tr></table>

Table 14: BD MAX Enteric Viral Panel LoD Results in Preserved Stool Matrix   

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>LoD (cp/mL)</td><td rowspan=1 colspan=1>Positivity</td><td rowspan=1 colspan=1>95%ConfidenceInterval</td></tr><tr><td rowspan=2 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>F40</td><td rowspan=1 colspan=1>2.75E+05</td><td rowspan=1 colspan=1>100% (20/20)</td><td rowspan=1 colspan=1>83.9% - 100%</td></tr><tr><td rowspan=1 colspan=1>F41</td><td rowspan=1 colspan=1>1.22E+05</td><td rowspan=1 colspan=1>100% (20/20)</td><td rowspan=1 colspan=1>83.9% - 100%</td></tr><tr><td rowspan=2 colspan=1>Astrovirus</td><td rowspan=1 colspan=1>Type 4</td><td rowspan=1 colspan=1>3.49E+07</td><td rowspan=1 colspan=1>100% (20/20)</td><td rowspan=1 colspan=1>83.9% - 100%</td></tr><tr><td rowspan=1 colspan=1>Type 8</td><td rowspan=1 colspan=1>2.09E+07</td><td rowspan=1 colspan=1>95% (19/20)</td><td rowspan=1 colspan=1>76.4% - 99.1%</td></tr><tr><td rowspan=2 colspan=1>Rotavirus</td><td rowspan=1 colspan=1>WA</td><td rowspan=1 colspan=1>1.29E+04</td><td rowspan=1 colspan=1>95% (19/20)</td><td rowspan=1 colspan=1>76.4% - 99.1%</td></tr><tr><td rowspan=1 colspan=1>Va70</td><td rowspan=1 colspan=1>5.82E+03</td><td rowspan=1 colspan=1>100% (20/20)</td><td rowspan=1 colspan=1>83.9% - 100%</td></tr><tr><td rowspan=2 colspan=1>Norovirus</td><td rowspan=1 colspan=1>GI</td><td rowspan=1 colspan=1>4.71E+06</td><td rowspan=1 colspan=1>95% (19/20)</td><td rowspan=1 colspan=1>76.4% - 99.1%</td></tr><tr><td rowspan=1 colspan=1>GII</td><td rowspan=1 colspan=1>2.49E+05</td><td rowspan=1 colspan=1>100% (20/20)</td><td rowspan=1 colspan=1>83.9% - 100%</td></tr><tr><td rowspan=2 colspan=1>Sapovirus</td><td rowspan=1 colspan=1>GI</td><td rowspan=1 colspan=1>6.51E+07</td><td rowspan=1 colspan=1>100% (20/20)</td><td rowspan=1 colspan=1>83.9% - 100%</td></tr><tr><td rowspan=1 colspan=1>GII</td><td rowspan=1 colspan=1>1.94E+06</td><td rowspan=1 colspan=1>95% (19/20)</td><td rowspan=1 colspan=1>76.4% - 99.1%</td></tr></table>

# e. Analytical specificity:

# Cross reactivity:

Other off-panel phylogenetically related microorganisms that may be present in human stool were evaluated for potential Enteric Viral Panel reactivity. Specifically, 94 bacteria. ten viruses, and eight parasites were tested in three replicates using three reagent lots and twelve BD MAX instruments (Table 15). The concentrations of bacteria/parasites were confirmed by colony/cell count while parasites were enumerated by the respective suppliers. Viruses were quantitated by their respective vendor(s) and the concentrations were documented in the Certificates of Analysis. Samples that exhibited initial reactivity or had INC or UNR results were subsequently retested (Table 16 and Table 17). Adenovirus Type 1 was the only pathogen to exhibit cross-reactivity, being consistently detected by the Enteric Viral Panel adenovirus test and thus included as a cross-reactant in the package insert. This reactivity disappeared at a tenfold lower dilution of virus.

Cross-reactivity with organisms other than those listed in the table below has not been evaluated.

Table 15: Testing of Potentially Cross-Reactive Organisms   

<table><tr><td colspan="1" rowspan="1">Microorganism</td><td colspan="1" rowspan="1">Concentration(CFU/mL)</td><td colspan="1" rowspan="1">CollectionNumber</td><td colspan="1" rowspan="1">Test Result(Positive/Total)</td><td colspan="1" rowspan="1">Initial ResultDetails</td></tr><tr><td colspan="1" rowspan="1">Abiotrophia defectiva</td><td colspan="1" rowspan="1">1.1E+08</td><td colspan="1" rowspan="1">ATCC 49176</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">All NEG</td></tr><tr><td colspan="1" rowspan="1">Acinetobacter baumannii</td><td colspan="1" rowspan="1">2.0E+08</td><td colspan="1" rowspan="1">ATCC 19606</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">All NEG</td></tr><tr><td colspan="1" rowspan="1">Acinetobacter iwoffii</td><td colspan="1" rowspan="1">2.0E+08</td><td colspan="1" rowspan="1">ATCC15309</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">All NEG</td></tr><tr><td colspan="1" rowspan="1">Aeromonas caviae</td><td colspan="1" rowspan="1">1.7E+08</td><td colspan="1" rowspan="1">ATCC 15468</td><td colspan="1" rowspan="1">0/0b</td><td colspan="1" rowspan="1">3 INC</td></tr><tr><td colspan="1" rowspan="1">Aeromonas hydrophila</td><td colspan="1" rowspan="1">1.7E+08</td><td colspan="1" rowspan="1">ATCC 7966</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">All NEG</td></tr><tr><td colspan="1" rowspan="1">Aeromonas schubertii</td><td colspan="1" rowspan="1">1.7E+08</td><td colspan="1" rowspan="1">ATCC 43700</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">All NEG</td></tr><tr><td colspan="1" rowspan="1">Aeromonas sobria</td><td colspan="1" rowspan="1">1.7E+08</td><td colspan="1" rowspan="1">ATCC BAA-237</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">All NEG</td></tr><tr><td colspan="1" rowspan="1">Aeromonas veronii</td><td colspan="1" rowspan="1">1.7E+08</td><td colspan="1" rowspan="1">ATCC 35623</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">All NEG</td></tr><tr><td colspan="1" rowspan="1">Alcaligenes faecalis subsp.faecalis</td><td colspan="1" rowspan="1">3.4E+08</td><td colspan="1" rowspan="1">ATCC 15554</td><td colspan="1" rowspan="1">1/3b</td><td colspan="1" rowspan="1">1 NoV POS2 NEG</td></tr><tr><td colspan="1" rowspan="1">Anaerococcus tetradius</td><td colspan="1" rowspan="1">1.2E+08</td><td colspan="1" rowspan="1">ATCC 35098</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">All NEG</td></tr><tr><td colspan="1" rowspan="1">Arcobacter butzleri</td><td colspan="1" rowspan="1">2.0E+08</td><td colspan="1" rowspan="1">ATCC 49616</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">All NEG</td></tr><tr><td colspan="1" rowspan="1">Bacillus cereus</td><td colspan="1" rowspan="1">8.2E+07</td><td colspan="1" rowspan="1">ATCC 13472</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">All NEG</td></tr><tr><td colspan="1" rowspan="1">Bacteroides caccae</td><td colspan="1" rowspan="1">6.8E+08</td><td colspan="1" rowspan="1">ATCC 43185</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">All NEG</td></tr><tr><td colspan="1" rowspan="1">Bacteroides stercoris</td><td colspan="1" rowspan="1">6.8E+08</td><td colspan="1" rowspan="1">ATCC 43183</td><td colspan="1" rowspan="1">0/2b</td><td colspan="1" rowspan="1">2 NEG1UNR</td></tr><tr><td colspan="1" rowspan="1">Bifidobacterium adolescentis</td><td colspan="1" rowspan="1">9.2E+08</td><td colspan="1" rowspan="1">ATCC 15703</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">All NEG</td></tr><tr><td colspan="1" rowspan="1">Bifidobacterium bifidum</td><td colspan="1" rowspan="1">9.2E+08</td><td colspan="1" rowspan="1">ATCC 29521</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">All NEG</td></tr><tr><td colspan="1" rowspan="1">Campylobacter coli</td><td colspan="1" rowspan="1">1.0 McFa</td><td colspan="1" rowspan="1">ATCC 43134</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">All NEG</td></tr><tr><td colspan="1" rowspan="1">Campylobacter jejuni</td><td colspan="1" rowspan="1">6.4E+07</td><td colspan="1" rowspan="1">NH 411</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">All NEG</td></tr><tr><td colspan="1" rowspan="1">Candida albicans</td><td colspan="1" rowspan="1">1.8E+07</td><td colspan="1" rowspan="1">ATCC 24433</td><td colspan="1" rowspan="1">0/2b</td><td colspan="1" rowspan="1">2 NEG1 INC</td></tr><tr><td colspan="1" rowspan="1">Cedecea davisae</td><td colspan="1" rowspan="1">2.4E+08</td><td colspan="1" rowspan="1">ATCC 33431</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">All NEG</td></tr><tr><td colspan="1" rowspan="1">Citrobacter freundii</td><td colspan="1" rowspan="1">2.4E+08</td><td colspan="1" rowspan="1">ATCC 8090</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">All NEG</td></tr><tr><td colspan="1" rowspan="1">Citrobacter koseri</td><td colspan="1" rowspan="1">2.4E+08</td><td colspan="1" rowspan="1">ATCC 27028</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">All NEG</td></tr><tr><td colspan="1" rowspan="1">Chlamydia trachomatisserovar D</td><td colspan="1" rowspan="1">6.84E+08 EBs/mL a</td><td colspan="1" rowspan="1">RD013117-01</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">All NEG</td></tr><tr><td colspan="1" rowspan="1">Clostridium difficile</td><td colspan="1" rowspan="1">6.1E+06</td><td colspan="1" rowspan="1">ATCC 43598</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">All NEG</td></tr><tr><td colspan="1" rowspan="1">Clostridium histolyticum</td><td colspan="1" rowspan="1">6.1E+06</td><td colspan="1" rowspan="1">ATCC 19401</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">All NEG</td></tr><tr><td colspan="1" rowspan="1">Clostridium perfringens</td><td colspan="1" rowspan="1">6.1E+06</td><td colspan="1" rowspan="1">ATCC 13124</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">All NEG</td></tr><tr><td colspan="1" rowspan="1">Clostridium sordellii</td><td colspan="1" rowspan="1">6.1E+06</td><td colspan="1" rowspan="1">ATCC 9714</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">All NEG</td></tr><tr><td colspan="1" rowspan="1">Clostridium tetani</td><td colspan="1" rowspan="1">6.1E+06</td><td colspan="1" rowspan="1">ATCC 19406</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">All NEG</td></tr><tr><td colspan="1" rowspan="1">Collinsella aerofaciens</td><td colspan="1" rowspan="1">2.2E+08</td><td colspan="1" rowspan="1">ATCC 33031</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">All NEG</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium genitalium</td><td colspan="1" rowspan="1">2.8E+08</td><td colspan="1" rowspan="1">ATCC 33031</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">All NEG</td></tr><tr><td colspan="1" rowspan="1">Desulfovibrio piger</td><td colspan="1" rowspan="1">7.5E+07</td><td colspan="1" rowspan="1">ATCC 29098</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">All NEG</td></tr><tr><td colspan="1" rowspan="1">Edwardsiella tarda</td><td colspan="1" rowspan="1">3.3E+08</td><td colspan="1" rowspan="1">ATCC 15947</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">All NEG</td></tr><tr><td colspan="1" rowspan="1">Eggerthella lenta</td><td colspan="1" rowspan="1">4.7E+08</td><td colspan="1" rowspan="1">CCRI-9926</td><td colspan="1" rowspan="1">0/2b</td><td colspan="1" rowspan="1">2 NEG1UNR</td></tr><tr><td colspan="1" rowspan="1">Enterobacter cloacae</td><td colspan="1" rowspan="1">2.6E+08</td><td colspan="1" rowspan="1">ATCC 35030</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">All NEG</td></tr><tr><td colspan="1" rowspan="1">Enterococcus faecalis</td><td colspan="1" rowspan="1">2.5E+08</td><td colspan="1" rowspan="1">ATCC 29212</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">All NEG</td></tr><tr><td colspan="1" rowspan="1">Enterococcus faecium</td><td colspan="1" rowspan="1">2.5E+08</td><td colspan="1" rowspan="1">ATCC 700221</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">All NEG</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">4.4E+08</td><td colspan="1" rowspan="1">ENF 10513</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">All NEG</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli O157: H7EHEC</td><td colspan="1" rowspan="1">4.1E+08</td><td colspan="1" rowspan="1">ATCC 43889</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">All NEG</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli ETECH10405 O78:H11 (Lt/St)</td><td colspan="1" rowspan="1">1.6E+08</td><td colspan="1" rowspan="1">ATCC 35401</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">All NEG</td></tr><tr><td colspan="1" rowspan="1">Escherichia fergusonii</td><td colspan="1" rowspan="1">2.5E+08</td><td colspan="1" rowspan="1">ATCC 35469</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">All NEG</td></tr><tr><td colspan="1" rowspan="1">Escherichia hermannii</td><td colspan="1" rowspan="1">2.5E+08</td><td colspan="1" rowspan="1">ATCC 33650</td><td colspan="1" rowspan="1">1/3</td><td colspan="1" rowspan="1">1 NoV POS2 NEG</td></tr><tr><td colspan="1" rowspan="1">Escherichia vulneris</td><td colspan="1" rowspan="1">2.5E+08</td><td colspan="1" rowspan="1">ATCC 33821</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">All NEG</td></tr><tr><td colspan="1" rowspan="1">Fusobacterium varium</td><td colspan="1" rowspan="1">3.4E+08</td><td colspan="1" rowspan="1">ATCC 8501</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">All NEG</td></tr><tr><td colspan="1" rowspan="1">Gardnerella vaginalis</td><td colspan="1" rowspan="1">2.1E+08</td><td colspan="1" rowspan="1">ATCC 14019</td><td colspan="1" rowspan="1">0/2b</td><td colspan="1" rowspan="1">2 NEG1 INC</td></tr><tr><td colspan="1" rowspan="1">Gemella morbillorum</td><td colspan="1" rowspan="1">1.3E+08</td><td colspan="1" rowspan="1">ATCC 27824</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">All NEG</td></tr><tr><td colspan="1" rowspan="1">Hafnia alvei</td><td colspan="1" rowspan="1">3.5E+08</td><td colspan="1" rowspan="1">ATCC 13337</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">All NEG</td></tr><tr><td colspan="1" rowspan="1">Helicobacter pylori</td><td colspan="1" rowspan="1">2.4E+08</td><td colspan="1" rowspan="1">ATCC 43504</td><td colspan="1" rowspan="1">0/2b</td><td colspan="1" rowspan="1">2 NEG1 INC</td></tr><tr><td colspan="1" rowspan="1">Klebsiella aerogenes</td><td colspan="1" rowspan="1">2.6E+08</td><td colspan="1" rowspan="1">ATCC 13048</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">All NEG</td></tr><tr><td colspan="1" rowspan="1">Klebsiella oxytoca</td><td colspan="1" rowspan="1">1.8E+08</td><td colspan="1" rowspan="1">ATCC 33496</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">All NEG</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">1.8E+08</td><td colspan="1" rowspan="1">ATCC 700603</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">All NEG</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus acidophilus</td><td colspan="1" rowspan="1">9.2E+07</td><td colspan="1" rowspan="1">ATCC 4356</td><td colspan="1" rowspan="1">0/2b</td><td colspan="1" rowspan="1">2 NEG1 INC</td></tr><tr><td colspan="1" rowspan="1">Lactococcus lactis</td><td colspan="1" rowspan="1">1.9E+08</td><td colspan="1" rowspan="1">ATCC 11454</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">All NEG</td></tr><tr><td colspan="1" rowspan="1">Leminorella grimontii</td><td colspan="1" rowspan="1">6.5E+08</td><td colspan="1" rowspan="1">ATCC 33999</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">All NEG</td></tr><tr><td colspan="1" rowspan="1">Listeria monocytogenes</td><td colspan="1" rowspan="1">6.3E+08</td><td colspan="1" rowspan="1">ATCC 15313</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">All NEG</td></tr><tr><td colspan="1" rowspan="1">Megamonas hypermegale</td><td colspan="1" rowspan="1">2.5E+07</td><td colspan="1" rowspan="1">ATCC 25560</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">All NEG</td></tr><tr><td colspan="1" rowspan="1">Megasphaeara elsdenii</td><td colspan="1" rowspan="1">3.5E+07</td><td colspan="1" rowspan="1">ATCC 25940</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">All NEG</td></tr><tr><td colspan="1" rowspan="1">Morganella morganii</td><td colspan="1" rowspan="1">2.8E+08</td><td colspan="1" rowspan="1">ATCC 25830</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">All NEG</td></tr><tr><td colspan="1" rowspan="1">Parabacteroides merdae</td><td colspan="1" rowspan="1">5.5E+08</td><td colspan="1" rowspan="1">ATCC 43184</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">All NEG</td></tr><tr><td colspan="1" rowspan="1">Peptoniphilusasaccharolyticus</td><td colspan="1" rowspan="1">2.7E+08</td><td colspan="1" rowspan="1">ATCC 14963</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">All NEG</td></tr><tr><td colspan="1" rowspan="1">Peptostreptococcusanaerobius</td><td colspan="1" rowspan="1">2.1E+08</td><td colspan="1" rowspan="1">ATCC 27337</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">All NEG</td></tr><tr><td colspan="1" rowspan="1">Photobacterium damselaesubsp. damselae</td><td colspan="1" rowspan="1">1.1E+08</td><td colspan="1" rowspan="1">ATCC 33539</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">All NEG</td></tr><tr><td colspan="1" rowspan="1">Plesiomonas shigelloides</td><td colspan="1" rowspan="1">1.6E+08</td><td colspan="1" rowspan="1">ATCC 14029</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">All NEG</td></tr><tr><td colspan="1" rowspan="1">Porphyromonasasaccharolytica</td><td colspan="1" rowspan="1">3.4E+07</td><td colspan="1" rowspan="1">ATCC 25260</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">All NEG</td></tr><tr><td colspan="1" rowspan="1">Prevotella melaninogenica</td><td colspan="1" rowspan="1">1.2E+09</td><td colspan="1" rowspan="1">ATCC 25845</td><td colspan="1" rowspan="1">0/2b</td><td colspan="1" rowspan="1">2 NEG1 INC</td></tr><tr><td colspan="1" rowspan="1">Proteus mirabilis</td><td colspan="1" rowspan="1">4.8E+08</td><td colspan="1" rowspan="1">ATCC 25933</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">All NEG</td></tr><tr><td colspan="1" rowspan="1">Proteus vulgaris</td><td colspan="1" rowspan="1">4.8E+08</td><td colspan="1" rowspan="1">ENF 12392</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">All NEG</td></tr><tr><td colspan="1" rowspan="1">Providencia alcalifaciens</td><td colspan="1" rowspan="1">3.1E+08</td><td colspan="1" rowspan="1">ATCC 9886</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">All NEG</td></tr><tr><td colspan="1" rowspan="1">Ruminococcus bromii</td><td colspan="1" rowspan="1">2.1E+08</td><td colspan="1" rowspan="1">ATCC 27255</td><td colspan="1" rowspan="1">1/2b</td><td colspan="1" rowspan="1">1 hAstV POS1 NEG1 INC</td></tr><tr><td colspan="1" rowspan="1">Saccharomyces cerevisiae</td><td colspan="1" rowspan="1">2.6E+07</td><td colspan="1" rowspan="1">ATCC 76527</td><td colspan="1" rowspan="1">0/2b</td><td colspan="1" rowspan="1">2 NEG1 INC</td></tr><tr><td colspan="1" rowspan="1">Salmonella bongori</td><td colspan="1" rowspan="1">2.3E+08</td><td colspan="1" rowspan="1">ATCC 43975</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">All NEG</td></tr><tr><td colspan="1" rowspan="1">Salmonella enterica subsp.arizonae</td><td colspan="1" rowspan="1">2.3E+08</td><td colspan="1" rowspan="1">7379</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">All NEG</td></tr><tr><td colspan="1" rowspan="1">Salmonella enterica subsp.Enterica serovar enteriditis</td><td colspan="1" rowspan="1">2.3E+08</td><td colspan="1" rowspan="1">ATCC 13076</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">All NEG</td></tr><tr><td colspan="1" rowspan="1">Salmonella typhimurium</td><td colspan="1" rowspan="1">4.0E+08</td><td colspan="1" rowspan="1">ATCC 14028</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">All NEG</td></tr><tr><td colspan="1" rowspan="1">Serratia liquefaciens</td><td colspan="1" rowspan="1">1.5E+08</td><td colspan="1" rowspan="1">ATCC 27592</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">All NEG</td></tr><tr><td colspan="1" rowspan="1">Serratia marcescens</td><td colspan="1" rowspan="1">1.5E+08</td><td colspan="1" rowspan="1">ATCC 13880</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">All NEG</td></tr><tr><td colspan="1" rowspan="1">Shigella boydii</td><td colspan="1" rowspan="1">1.9E+08</td><td colspan="1" rowspan="1">ATCC 9207</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">All NEG</td></tr><tr><td colspan="1" rowspan="1">Shigella dysenteriae</td><td colspan="1" rowspan="1">1.9E+08</td><td colspan="1" rowspan="1">2933</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">All NEG</td></tr><tr><td colspan="1" rowspan="1">Shigella flexneri</td><td colspan="1" rowspan="1">1.9E+08</td><td colspan="1" rowspan="1">ATCC 700930</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">All NEG</td></tr><tr><td colspan="1" rowspan="1">Shigella sonnei</td><td colspan="1" rowspan="1">1.9E+08</td><td colspan="1" rowspan="1">ENF 7142</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">All NEG</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">4.0E+08</td><td colspan="1" rowspan="1">ATCC 43300</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">All NEG</td></tr><tr><td colspan="1" rowspan="1">Microorganism</td><td colspan="1" rowspan="1">Concentration(CFU/mL)</td><td colspan="1" rowspan="1">CollectionNumber</td><td colspan="1" rowspan="1">Test Result(Positive/Total)</td><td colspan="1" rowspan="1">Initial Resultetails</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus epidermidis</td><td colspan="1" rowspan="1">4.0E+08</td><td colspan="1" rowspan="1">ATCC 14990</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">All NEG</td></tr><tr><td colspan="1" rowspan="1">Stenotrophomonas maltphilia</td><td colspan="1" rowspan="1">7.0E+07</td><td colspan="1" rowspan="1">ATCC 13637</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">All NEG</td></tr><tr><td colspan="1" rowspan="1">Streptococcus agalactiae</td><td colspan="1" rowspan="1">4.6E+08</td><td colspan="1" rowspan="1">ATCC 13813</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">All NEG</td></tr><tr><td colspan="1" rowspan="1">Streptococcus intermedius</td><td colspan="1" rowspan="1">4.6E+08</td><td colspan="1" rowspan="1">ATCC 27335</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">All NEG</td></tr><tr><td colspan="1" rowspan="1">Trabulsiella guamensis</td><td colspan="1" rowspan="1">4.3E+08</td><td colspan="1" rowspan="1">ATCC 49490</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">All NEG</td></tr><tr><td colspan="1" rowspan="1">Veillonella parvula</td><td colspan="1" rowspan="1">1.8E+08</td><td colspan="1" rowspan="1">CCUG 59474</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">All NEG</td></tr><tr><td colspan="1" rowspan="1">Vibrio cholerae</td><td colspan="1" rowspan="1">3.0E+08</td><td colspan="1" rowspan="1">ATCC 14033</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">All NEG</td></tr><tr><td colspan="1" rowspan="1">Vibrio mimicus</td><td colspan="1" rowspan="1">2.2E+08</td><td colspan="1" rowspan="1">ATCC 33653</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">All NEG</td></tr><tr><td colspan="1" rowspan="1">Vibrio parahaemolyticus</td><td colspan="1" rowspan="1">2.5E+08</td><td colspan="1" rowspan="1">ATCC 17802</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">All NEG</td></tr><tr><td colspan="1" rowspan="1">Vibrio vulnificus</td><td colspan="1" rowspan="1">2.6E+08</td><td colspan="1" rowspan="1">ATCC 27562</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">All NEG</td></tr><tr><td colspan="1" rowspan="1">Yersinia bercovieri</td><td colspan="1" rowspan="1">2.8E+08</td><td colspan="1" rowspan="1">ATCC 43970</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">All NEG</td></tr><tr><td colspan="1" rowspan="1">Yersinia enterocolitica</td><td colspan="1" rowspan="1">6.0E+07</td><td colspan="1" rowspan="1">CCUG 4588</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">All NEG</td></tr><tr><td colspan="1" rowspan="1">Yersinia enterocolitica subsp.enterocolitica</td><td colspan="1" rowspan="1">4.7E+07</td><td colspan="1" rowspan="1">ATCC 9610</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">All NEG</td></tr><tr><td colspan="1" rowspan="1">Yersinia rohdei</td><td colspan="1" rowspan="1">2.8E+08</td><td colspan="1" rowspan="1">ATCC 43380</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">All NEG</td></tr><tr><td colspan="1" rowspan="1">Blastocystic hominis</td><td colspan="1" rowspan="1">1.0E+07</td><td colspan="1" rowspan="1">ATCC 50608</td><td colspan="1" rowspan="1">0/2b</td><td colspan="1" rowspan="1">2 NEG1 INC</td></tr><tr><td colspan="1" rowspan="1">Cryptosporidium hominis</td><td colspan="1" rowspan="1">1.5E+07</td><td colspan="1" rowspan="1">IDI-P-22</td><td colspan="1" rowspan="1">0/2b</td><td colspan="1" rowspan="1">2 NEG1 INC</td></tr><tr><td colspan="1" rowspan="1">Cryptosporidium parvum</td><td colspan="1" rowspan="1">6.3E+06</td><td colspan="1" rowspan="1">Waterborne P102(lot 131219)</td><td colspan="1" rowspan="1">0/2b</td><td colspan="1" rowspan="1">2 NEG1 INC</td></tr><tr><td colspan="1" rowspan="1">Encephalitozoon intestinalis</td><td colspan="1" rowspan="1">2.08E+08 cells/mL</td><td colspan="1" rowspan="1">ATCC 50651</td><td colspan="1" rowspan="1">0/2b</td><td colspan="1" rowspan="1">2 NEG1 INC</td></tr><tr><td colspan="1" rowspan="1">Entamoeba histolytica</td><td colspan="1" rowspan="1">6.0E+05 cells/mL</td><td colspan="1" rowspan="1">ATCC 30458</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">All NEG</td></tr><tr><td colspan="1" rowspan="1">Giardia lamblia</td><td colspan="1" rowspan="1">6.3E+06</td><td colspan="1" rowspan="1">Waterborne P101</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">All NEG</td></tr><tr><td colspan="1" rowspan="1">Giardia muris</td><td colspan="1" rowspan="1">5.6E+05</td><td colspan="1" rowspan="1">Waterborne P105(lot: 130904)</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">All NEG</td></tr><tr><td colspan="1" rowspan="1">Pentatrichomonas hominis</td><td colspan="1" rowspan="1">8.5E+06</td><td colspan="1" rowspan="1">ATCC 30098</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">All NEG</td></tr><tr><td colspan="1" rowspan="1">Adenovirus Type 1</td><td colspan="1" rowspan="1">1.0E+6.58 TCID50units/mL</td><td colspan="1" rowspan="1">Zeptometrix0810050CF</td><td colspan="1" rowspan="1">3/3b</td><td colspan="1" rowspan="1">3 AdV POS</td></tr><tr><td colspan="1" rowspan="1">Adenovirus Type 18</td><td colspan="1" rowspan="1">1.0E+7.25 TCID50units/mL</td><td colspan="1" rowspan="1">ATCC VR-19</td><td colspan="1" rowspan="1">1/3b</td><td colspan="1" rowspan="1">1 hAstV POS2 NEG</td></tr><tr><td colspan="1" rowspan="1">Adenovirus Type 3</td><td colspan="1" rowspan="1">1.0E+5.23 TCID50units/mL</td><td colspan="1" rowspan="1">Zeptometrix0810062CF</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">All NEG</td></tr><tr><td colspan="1" rowspan="1">Adenovirus Type 4</td><td colspan="1" rowspan="1">1.0E+5.86 TCID50units/mL</td><td colspan="1" rowspan="1">Zeptometrix081070CF</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">All NEG</td></tr><tr><td colspan="1" rowspan="1">Adenovirus Type 8</td><td colspan="1" rowspan="1">1.0E+5.15 TCID50units/mL</td><td colspan="1" rowspan="1">Zeptometrix0810069CF</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">All NEG</td></tr><tr><td colspan="1" rowspan="1">Coxsackie A9</td><td colspan="1" rowspan="1">1.0E+4.5 TCID50units/ 0.2 mL</td><td colspan="1" rowspan="1">ATCC VR-186</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">All NEG</td></tr><tr><td colspan="1" rowspan="1">Coxsackie B1</td><td colspan="1" rowspan="1">1.0E+7.5 TCID50units/ 0.2 mL</td><td colspan="1" rowspan="1">ATCC VR-28</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">All NEG</td></tr><tr><td colspan="1" rowspan="1">Echovirus 19</td><td colspan="1" rowspan="1">1.2E+06 TCID50units/mL</td><td colspan="1" rowspan="1">CCRI-15850</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">All NEG</td></tr><tr><td colspan="1" rowspan="1">Enterovirus type 68</td><td colspan="1" rowspan="1">1.0E+5.3 TCID50units/ 0.1 mL</td><td colspan="1" rowspan="1">ATCC VR-561</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">All NEG</td></tr><tr><td colspan="1" rowspan="1">Microorganism</td><td colspan="1" rowspan="1">Concentration(CFU/mL)</td><td colspan="1" rowspan="1">CollectionNumber</td><td colspan="1" rowspan="1">Test Result(Positive/Total)</td><td colspan="1" rowspan="1">Initial ResultDetails</td></tr><tr><td colspan="1" rowspan="1">Herpes Simplex Virus 1</td><td colspan="1" rowspan="1">1.0E+4.25 TCID50units/ 0.2 mL</td><td colspan="1" rowspan="1">ATCC VR-735</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">All NEG</td></tr></table>

aConcentration in CFU/mL was unavailable for certain panels. Evaluation of the concentration of Campylobacter coli was estimated in McFarland units. Evaluation of the concentration of Chlamydia trachomatis was estimated in elementary bodies per mL. bUNR, INC, POS, results were subject to supplemental testing.

Table 16: Retesting of Analytes with Positive Initial Results from the Exclusivity Study   

<table><tr><td rowspan=1 colspan=1>Microorganism</td><td rowspan=1 colspan=1>Dilution</td><td rowspan=1 colspan=1>Concentration(CFU/mL)</td><td rowspan=1 colspan=1>NoVResult</td><td rowspan=1 colspan=1>RoVResult</td><td rowspan=1 colspan=1>AdVResult</td><td rowspan=1 colspan=1>SaVResult</td><td rowspan=1 colspan=1>hAstVResult</td></tr><tr><td rowspan=2 colspan=1>Adenovirus Type 1</td><td rowspan=1 colspan=1>1:10</td><td rowspan=1 colspan=1>1 x 105.58TCID50/mL</td><td rowspan=1 colspan=1>20/20NEG</td><td rowspan=1 colspan=1>20/20NEG</td><td rowspan=1 colspan=1>9/20POS</td><td rowspan=1 colspan=1>20/20NEG</td><td rowspan=1 colspan=1>20/20NEG</td></tr><tr><td rowspan=1 colspan=1>1:00</td><td rowspan=1 colspan=1>1 x 104.58TCID50/mL</td><td rowspan=1 colspan=1>20/20NEG</td><td rowspan=1 colspan=1>20/20NEG</td><td rowspan=1 colspan=1>20/20NEG</td><td rowspan=1 colspan=1>20/20NEG</td><td rowspan=1 colspan=1>20/20NEG</td></tr><tr><td rowspan=2 colspan=1>Adenovirus Type18</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>1 x 107.25TCID50/mL</td><td rowspan=1 colspan=1>20/20NEG</td><td rowspan=1 colspan=1>20/20NEG</td><td rowspan=1 colspan=1>20/20NEG</td><td rowspan=1 colspan=1>20/20NEG</td><td rowspan=1 colspan=1>20/20NEG</td></tr><tr><td rowspan=1 colspan=1>1:10</td><td rowspan=1 colspan=1>1 x 106.25TCID50/mL</td><td rowspan=1 colspan=1>20/20NEG</td><td rowspan=1 colspan=1>20/20NEG</td><td rowspan=1 colspan=1>20/20NEG</td><td rowspan=1 colspan=1>20/20NEG</td><td rowspan=1 colspan=1>20/20NEG</td></tr><tr><td rowspan=2 colspan=1>Alcaligenesfaecalis subsp.faecalis</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>3.4E+08</td><td rowspan=1 colspan=1>20/20NEG</td><td rowspan=1 colspan=1>20/20NEG</td><td rowspan=1 colspan=1>20/20NEG</td><td rowspan=1 colspan=1>20/20NEG</td><td rowspan=1 colspan=1>20/20NEG</td></tr><tr><td rowspan=1 colspan=1>1:10</td><td rowspan=1 colspan=1>3.4E+07</td><td rowspan=1 colspan=1>20/20NEG</td><td rowspan=1 colspan=1>20/20NEG</td><td rowspan=1 colspan=1>20/20NEG</td><td rowspan=1 colspan=1>20/20NEG</td><td rowspan=1 colspan=1>20/20NEG</td></tr><tr><td rowspan=2 colspan=1>Collinsellaaerofaciens</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>2.2E+08</td><td rowspan=1 colspan=1>20/20NEG</td><td rowspan=1 colspan=1>20/20NEG</td><td rowspan=1 colspan=1>20/20NEG</td><td rowspan=1 colspan=1>20/20NEG</td><td rowspan=1 colspan=1>20/20NEG</td></tr><tr><td rowspan=1 colspan=1>1:10</td><td rowspan=1 colspan=1>2.2E+07</td><td rowspan=1 colspan=1>20/20NEG</td><td rowspan=1 colspan=1>20/20NEG</td><td rowspan=1 colspan=1>20/20NEG</td><td rowspan=1 colspan=1>20/20NEG</td><td rowspan=1 colspan=1>20/20NEG</td></tr><tr><td rowspan=2 colspan=1>Escherichiahermannii</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>2.5E+08</td><td rowspan=1 colspan=1>20/20NEG</td><td rowspan=1 colspan=1>20/20NEG</td><td rowspan=1 colspan=1>20/20NEG</td><td rowspan=1 colspan=1>20/20NEG</td><td rowspan=1 colspan=1>20/20NEG</td></tr><tr><td rowspan=1 colspan=1>1:10</td><td rowspan=1 colspan=1>2.5E+07</td><td rowspan=1 colspan=1>19/19aNEG</td><td rowspan=1 colspan=1>19/19aNEG</td><td rowspan=1 colspan=1>19/19aNEG</td><td rowspan=1 colspan=1>19/19aNEG</td><td rowspan=1 colspan=1>19/19aNEG</td></tr><tr><td rowspan=2 colspan=1>Ruminococcusbromii</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>2.1E+08</td><td rowspan=1 colspan=1>20/20NEG</td><td rowspan=1 colspan=1>20/20NEG</td><td rowspan=1 colspan=1>20/20NEG</td><td rowspan=1 colspan=1>20/20NEG</td><td rowspan=1 colspan=1>20/20NEG</td></tr><tr><td rowspan=1 colspan=1>1:10</td><td rowspan=1 colspan=1>2.1E+07</td><td rowspan=1 colspan=1>20/20NEG</td><td rowspan=1 colspan=1>20/20NEG</td><td rowspan=1 colspan=1>20/20NEG</td><td rowspan=1 colspan=1>20/20NEG</td><td rowspan=1 colspan=1>20/20NEG</td></tr></table>

a1/20 replicates yielded an IND result and was excluded from analysis

Table 17: Retesting of Analytes with INC/UNR Results from the Exclusivity Study   

<table><tr><td colspan="1" rowspan="1">Microorganism</td><td colspan="1" rowspan="1">Concentration(CFU/mL)</td><td colspan="1" rowspan="1">CollectionNumber</td><td colspan="1" rowspan="1">Initial Result</td><td colspan="1" rowspan="1">Repeated TestResult(Positive/Total)</td><td colspan="1" rowspan="1">RepeatedResult Details</td></tr><tr><td colspan="1" rowspan="1">Aeromonascaviae</td><td colspan="1" rowspan="1">1.7E+08</td><td colspan="1" rowspan="1">ATCC 15468</td><td colspan="1" rowspan="1">INC</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">NEG</td></tr><tr><td colspan="1" rowspan="1">Bacteroidesstercoris</td><td colspan="1" rowspan="1">6.8E+08</td><td colspan="1" rowspan="1">ATCC 43183</td><td colspan="1" rowspan="1">UNR</td><td colspan="1" rowspan="1">0/1</td><td colspan="1" rowspan="1">NEG</td></tr><tr><td colspan="1" rowspan="1">Candidaalbicans</td><td colspan="1" rowspan="1">1.8E+07</td><td colspan="1" rowspan="1">ATCC 24433</td><td colspan="1" rowspan="1">INC</td><td colspan="1" rowspan="1">0/1</td><td colspan="1" rowspan="1">NEG</td></tr><tr><td colspan="1" rowspan="1">Eggerthellalenta</td><td colspan="1" rowspan="1">4.7E+08</td><td colspan="1" rowspan="1">CCRI-9926</td><td colspan="1" rowspan="1">UNR</td><td colspan="1" rowspan="1">0/1</td><td colspan="1" rowspan="1">NEG</td></tr><tr><td colspan="1" rowspan="1">Gardnerellavaginalis</td><td colspan="1" rowspan="1">2.1E+08</td><td colspan="1" rowspan="1">ATCC 14019</td><td colspan="1" rowspan="1">INC</td><td colspan="1" rowspan="1">0/1</td><td colspan="1" rowspan="1">NEG</td></tr><tr><td colspan="1" rowspan="1">Helicobacterpylori</td><td colspan="1" rowspan="1">2.4E+08</td><td colspan="1" rowspan="1">ATCC 43504</td><td colspan="1" rowspan="1">INC</td><td colspan="1" rowspan="1">0/1</td><td colspan="1" rowspan="1">NEG</td></tr><tr><td colspan="1" rowspan="1">Lactobacillusacidophilus</td><td colspan="1" rowspan="1">9.2E+07</td><td colspan="1" rowspan="1">ATCC 4356</td><td colspan="1" rowspan="1">INC</td><td colspan="1" rowspan="1">0/1</td><td colspan="1" rowspan="1">NEG</td></tr><tr><td colspan="1" rowspan="1">Prevotellamelaninogenica</td><td colspan="1" rowspan="1">1.2E+09</td><td colspan="1" rowspan="1">ATCC 25845</td><td colspan="1" rowspan="1">INC</td><td colspan="1" rowspan="1">0/1</td><td colspan="1" rowspan="1">NEG</td></tr><tr><td colspan="1" rowspan="1">Ruminococcusbromii</td><td colspan="1" rowspan="1">2.1E+08</td><td colspan="1" rowspan="1">ATCC 27255</td><td colspan="1" rowspan="1">INC</td><td colspan="1" rowspan="1">0/1</td><td colspan="1" rowspan="1">NEG</td></tr><tr><td colspan="1" rowspan="1">Saccharomycescerevisiae</td><td colspan="1" rowspan="1">2.6E+07</td><td colspan="1" rowspan="1">ATCC 76527</td><td colspan="1" rowspan="1">INC</td><td colspan="1" rowspan="1">0/1</td><td colspan="1" rowspan="1">NEG</td></tr><tr><td colspan="1" rowspan="1">Blastocystishominis</td><td colspan="1" rowspan="1">1.0E+07</td><td colspan="1" rowspan="1">ATCC 50608</td><td colspan="1" rowspan="1">INC</td><td colspan="1" rowspan="1">0/1</td><td colspan="1" rowspan="1">NEG</td></tr><tr><td colspan="1" rowspan="1">Cryptosporidiumhominis</td><td colspan="1" rowspan="1">1.5E+07</td><td colspan="1" rowspan="1">IDI-P-22</td><td colspan="1" rowspan="1">INC</td><td colspan="1" rowspan="1">0/1</td><td colspan="1" rowspan="1">NEG</td></tr><tr><td colspan="1" rowspan="1">Cryptosporidiumparvum</td><td colspan="1" rowspan="1">6.3E+06</td><td colspan="1" rowspan="1">WaterborneP102(lot:131219)</td><td colspan="1" rowspan="1">INC</td><td colspan="1" rowspan="1">0/1</td><td colspan="1" rowspan="1">NEG</td></tr><tr><td colspan="1" rowspan="1">Encephalitozoonintestinalis</td><td colspan="1" rowspan="1">2.08E+08cells/mL</td><td colspan="1" rowspan="1">ATCC 50651</td><td colspan="1" rowspan="1">INC</td><td colspan="1" rowspan="1">0/1</td><td colspan="1" rowspan="1">NEG</td></tr></table>

# Interference

Exogenous and endogenous interfering substances were evaluated for potential interference by spiking into unpreserved stool or Cary-Blair preserved stool matrix containing 2x LoD concentrations of Adenovirus, Astrovirus, Rotavirus, Norovirus, and Sapovirus. A total of 3 replicates were tested and compared to expected results with an acceptance criteria of $100 \%$ of spiked samples showing reactivity and $100 \%$ of organism-free samples being negative.

Hydrocortisone cream at concentrations of $5 0 \% \mathrm { v }$ was shown to reduce sample reactivity and was labeled as an interferant. However, repeat testing with Hydrocortisone present at $2 5 \% \mathrm { v / v }$ demonstrated $100 \%$ agreement with expected results (Table 18). The RotaTeq live Rotavirus vaccine was also picked up by the assay, as expected.

Potential interference has not been evaluated for substances other than those described in the table below. Interference by substances other than those described in the table below could lead to erroneous results.

Table 18: Interfering Substances Study Percent Agreement with Expected Results   

<table><tr><td colspan="1" rowspan="1">Pool/Individual</td><td colspan="1" rowspan="1">Substance</td><td colspan="1" rowspan="1">TestConcentration</td><td colspan="1" rowspan="1">RoV</td><td colspan="1" rowspan="1">AdV</td><td colspan="1" rowspan="1">hAstV</td><td colspan="1" rowspan="1">SaV</td><td colspan="1" rowspan="1">NoV</td><td colspan="1" rowspan="1">NoOrg.</td></tr><tr><td colspan="1" rowspan="6">Individually Tested</td><td colspan="1" rowspan="2">Hydrocortisone Cream</td><td colspan="1" rowspan="1">50% v/v</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">67%</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">25% v/v</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Benzalkonium Chloride</td><td colspan="1" rowspan="1">50% v/v</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Enema/Mineral Oil</td><td colspan="1" rowspan="1">50% v/v</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Hemorrhoidal gel</td><td colspan="1" rowspan="1">50% v/v</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Anti-fungal/itch vaginal;</td><td colspan="1" rowspan="1">50% v/v</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="4"></td><td colspan="1" rowspan="1">Nystatin</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Vagisil</td><td colspan="1" rowspan="1">50% v/v</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Polymixin B sulfate,bacitracin zinc,Neomycin Sulfate</td><td colspan="1" rowspan="1">50% v/v</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Nonoxynol-9</td><td colspan="1" rowspan="1">50% v/v</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="8">Individually Tested</td><td colspan="1" rowspan="1">Fecal Fats, Triglycerides</td><td colspan="1" rowspan="1">14% v/v</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Mucus</td><td colspan="1" rowspan="1">3.5% v/v</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Human Whole Blood</td><td colspan="1" rowspan="1">50% v/v</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Suppository</td><td colspan="1" rowspan="1">50% v/v</td><td colspan="1" rowspan="1">100%a</td><td colspan="1" rowspan="1">100%a</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%a</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">RoV Vaccine (ATCCVR-2195 and VR-2415)</td><td colspan="1" rowspan="1">2-116 x 106 IU</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Cary-Blair: Remel</td><td colspan="1" rowspan="1">75%</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Cary-Blair: Meridian:Para-Pack Enteric Plus</td><td colspan="1" rowspan="1">75%</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Cary-Blair: MeridianPara-Pack C&amp;S</td><td colspan="1" rowspan="1">75%</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="7">Antibiotic Pool</td><td colspan="1" rowspan="1">Naproxen sodium</td><td colspan="1" rowspan="1">81 mg/ml</td><td colspan="1" rowspan="7">100%</td><td colspan="1" rowspan="7">100%</td><td colspan="1" rowspan="7">100%</td><td colspan="1" rowspan="7">100%</td><td colspan="1" rowspan="7">100%</td><td colspan="1" rowspan="7">100%</td></tr><tr><td colspan="1" rowspan="1">Ceftriaxone disodium</td><td colspan="1" rowspan="1">16.0 mg/ml</td></tr><tr><td colspan="1" rowspan="1">Erythromycin</td><td colspan="1" rowspan="1">14.0 mg/ml</td></tr><tr><td colspan="1" rowspan="1">Metronidazole</td><td colspan="1" rowspan="1">60.8 mg/ml</td></tr><tr><td colspan="1" rowspan="1">Sulfamethoxazole</td><td colspan="1" rowspan="1">80.0 mg/ml</td></tr><tr><td colspan="1" rowspan="1">Tetracyclinehydrochloride</td><td colspan="1" rowspan="1">16.0 mg/ml</td></tr><tr><td colspan="1" rowspan="1">Trimethoprim</td><td colspan="1" rowspan="1">16.0 mg/ml</td></tr><tr><td colspan="1" rowspan="5">Other stoolorganisms Pool</td><td colspan="1" rowspan="1">Enterococcus faecalis</td><td colspan="1" rowspan="5">1 x 106 cells/mLeach</td><td colspan="1" rowspan="5">100%</td><td colspan="1" rowspan="5">100%</td><td colspan="1" rowspan="5">100%</td><td colspan="1" rowspan="5">100%</td><td colspan="1" rowspan="5">100%</td><td colspan="1" rowspan="5">100%</td></tr><tr><td colspan="1" rowspan="1">Escgerichia coli</td></tr><tr><td colspan="1" rowspan="1">Peptostreptococcusanaerobius</td></tr><tr><td colspan="1" rowspan="1">Proteus vulgaris</td></tr><tr><td colspan="1" rowspan="1">Salmonella typhimurium</td></tr><tr><td colspan="1" rowspan="4">OTC Pool</td><td colspan="1" rowspan="1">Laxatives (Sennosides)</td><td colspan="1" rowspan="1">47 mg/ml</td><td colspan="1" rowspan="4">100%</td><td colspan="1" rowspan="4">100%</td><td colspan="1" rowspan="4">100%</td><td colspan="1" rowspan="4">100%</td><td colspan="1" rowspan="4">100%</td><td colspan="1" rowspan="4">100%</td></tr><tr><td colspan="1" rowspan="1">Anti-diarrheal (Bismuthsubsalicylate)</td><td colspan="1" rowspan="1">8.75 mg/ml</td></tr><tr><td colspan="1" rowspan="1">Anti-diarrheal(Loperamidehydrochloride)</td><td colspan="1" rowspan="1">3.75 mg/ml</td></tr><tr><td colspan="1" rowspan="1">Antacids (CalciumCarbonate</td><td colspan="1" rowspan="1">32 mg/ml</td></tr></table>

aOne replicate resulted in partial UNR result but repeat testing resolved the issue

Mixed infection studies were carried out for each mastermix reaction with one viral pathogen being represented at $2 \mathbf { x }$ LoD concentration and the other pathogens in the same mastermix occurring at the $9 5 ^ { \mathrm { t h } }$ percentile concentration observed in the clinical study. The acceptance criteria were $100 \%$ reactivity for high concentration analytes and $2 9 5 \%$ reactivity for samples present at $2 \mathbf { x }$ LoD concentrations. All test conditions met acceptance criteria (Table 19).

19Table : Mixed Infection Study Results   

<table><tr><td colspan="1" rowspan="1">Test Condition</td><td colspan="1" rowspan="1">Organism</td><td colspan="1" rowspan="1">Concentration</td><td colspan="1" rowspan="1">Replicates</td><td colspan="1" rowspan="1">Negatives</td></tr><tr><td colspan="1" rowspan="3">1</td><td colspan="1" rowspan="1">Rotavirus</td><td colspan="1" rowspan="1">High</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">Adenovirus</td><td colspan="1" rowspan="1">High</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">Norovirus</td><td colspan="1" rowspan="1">2x LoD</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="3">2</td><td colspan="1" rowspan="1">Rotavirus</td><td colspan="1" rowspan="1">High</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">Adenovirus</td><td colspan="1" rowspan="1">2x LoD</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">Norovirus</td><td colspan="1" rowspan="1">High</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="3">3</td><td colspan="1" rowspan="1">Rotavirus</td><td colspan="1" rowspan="1">2x LoD</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">Adenovirus</td><td colspan="1" rowspan="1">High</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">Norovirus</td><td colspan="1" rowspan="1">High</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="2">4</td><td colspan="1" rowspan="1">Sapovirus</td><td colspan="1" rowspan="1">2x LoD</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">Astrovirus</td><td colspan="1" rowspan="1">High</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="2">5</td><td colspan="1" rowspan="1">Sapovirus</td><td colspan="1" rowspan="1">High</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">Astrovirus</td><td colspan="1" rowspan="1">2x LoD</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">0</td></tr></table>

# Cross-contamination (Carry-Over) Study

The sponsor analyzed a panel of alternating negative and high positive specimens to demonstrate the absence of significant carry-over effects in the BD MAX instrument during processing. Specifically, twelve high reactivity replicates of both Astrovirus and Adenovirus were tested with negative samples lacking any analyte. Three runs were performed on three instruments for a total of 108 negative and 108 high positive samples. The final analysis determined that $2 / 1 0 8$ expected negative samples were reactive demonstrating a false-reactivity rate of $1 . 8 5 \%$ due to cross-contamination effects.

# Guardrail Family D and Workflow Compatibility Study

A Guardrail Family describes the list of assays that can be run together on the BD MAX Instrument within one sample rack. Guardrail Family D follows the basic extraction workflow for an IVD assay established in Guardrail Family A with two major differences: it utilizes a 4-snap Unitized Reagent Strip (URS) and can accommodate assays of either one or two Master Mix tubes.

The purpose of this challenge testing was to ensure the Enteric Viral Panel assay can be simultaneously run with other assays utilizing the same Guardrail Family D software parameters. Two BD MAX instruments each ran 18 Enteric Viral Panel tests (9 positive and 9 negative) and 6 guardrail challenge tests. An additional instrument completed a single run of 4 Enteric Viral Panel tests (2 positive and 2 negative) and 2 guardrail challenge tests. In total, 54 samples were analyzed. All data conformed to acceptance guidelines and Enteric Viral Panel results as expected demonstrating no interference from the Guardrail challenge tests.

# Analytical Reactivity/Inclusivity Study

Inclusivity testing of the BD MAX Enteric Viral Panel included the strains listed in Table 20. Three strains did not confirm at $\ge 9 5 \%$ at $3 \mathbf { x }$ LoD. Sapovirus GI clinical strain BA0145AP required titration to 6x LoD before results were $\ge 9 5 \%$ . Sapovirus GI clinical strain BA0141AP and Rotavirus A isolate WISC2 VR-2517 required

titration to $2 0 \mathrm { x }$ LoD before results were $\ge 9 5 \%$ .

Four strains of Norovirus (GI.3, GII.P16_GII.2, GII.P16_GII.4, GII.Pe_GII.4) were unavailable for wet laboratory testing and were thus evaluated in silico for potential sequence amplification by the Enteric Viral Panel. From gene sequences available in the NCBI database, no significant primer or probe mismatches were identified that would theoretically limit the detection capability of the Enteric Viral Panel for these strains.

Table 20: Inclusivity Testing Results   

<table><tr><td colspan="1" rowspan="1">Virus</td><td colspan="1" rowspan="1">Isolate ID</td><td colspan="1" rowspan="1">Type/Strain</td><td colspan="1" rowspan="1">Spike Level(xLoD)</td><td colspan="1" rowspan="1">Percent Positive(%)</td></tr><tr><td colspan="1" rowspan="5">Adenovirus</td><td colspan="1" rowspan="1">MG10973</td><td colspan="1" rowspan="1">F40/41</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">MG08756</td><td colspan="1" rowspan="1">F40/41</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">BA0034AP</td><td colspan="1" rowspan="1">F40/41</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">BA0042AP</td><td colspan="1" rowspan="1">F40/41</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">BA0066AP</td><td colspan="1" rowspan="1">F40/41</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="5">Adenovirus</td><td colspan="1" rowspan="1">BA0313AP</td><td colspan="1" rowspan="1">F40/41</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">DLS14-07148</td><td colspan="1" rowspan="1">F40/41</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">DLS14-07137</td><td colspan="1" rowspan="1">F40/41</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">BA0002AP</td><td colspan="1" rowspan="1">F40/41</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">BA0221AP</td><td colspan="1" rowspan="1">F40/41</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="10">Astrovirus</td><td colspan="1" rowspan="1">Guix HastV-1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">Guix HastV-2</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">Guix HastV-3</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">Guix HastV-4</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">ZeptoMetrix</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">Guix HastV-5</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">Guix HastV-6</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">Guix HastV-7</td><td colspan="1" rowspan="1">7</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">Guix HastV-8</td><td colspan="1" rowspan="1">8</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">NYB488277</td><td colspan="1" rowspan="1">UNKNOWN</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="8">Rotavirus</td><td colspan="1" rowspan="1">DS-1 VR-2550</td><td colspan="1" rowspan="1">A</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">W161 VR-2551</td><td colspan="1" rowspan="1">A</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="4">WISC 2 VR02517</td><td colspan="1" rowspan="4">A</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">33</td></tr><tr><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">67</td></tr><tr><td colspan="1" rowspan="1">9</td><td colspan="1" rowspan="1">33</td></tr><tr><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">W179-4.9</td><td colspan="1" rowspan="1">A</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">DLS14-07384</td><td colspan="1" rowspan="1">A</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="5">Sapovirus</td><td colspan="1" rowspan="1">BA0355AP</td><td colspan="1" rowspan="1">GI</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">NS10653</td><td colspan="1" rowspan="1">GI</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="3">BA0141AP</td><td colspan="1" rowspan="3">GI</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">33</td></tr><tr><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">67</td></tr><tr><td colspan="1" rowspan="1">9</td><td colspan="1" rowspan="1">67</td></tr><tr><td colspan="1" rowspan="9">b</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">BA0130AP</td><td colspan="1" rowspan="1">GI</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="2">BA0145AP</td><td colspan="1" rowspan="2">GI</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">67</td></tr><tr><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">NYB470159</td><td colspan="1" rowspan="1">GII</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">BA0260AP</td><td colspan="1" rowspan="1">GIV</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">MG10206</td><td colspan="1" rowspan="1">GIV</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">NYB488186</td><td colspan="1" rowspan="1">GIV</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">NYB470205</td><td colspan="1" rowspan="1">GV</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="10">Norovirus</td><td colspan="1" rowspan="1">Guix M21.2</td><td colspan="1" rowspan="1">GI.4</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">Guix D20171A</td><td colspan="1" rowspan="1">GI.6</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">Guix V15</td><td colspan="1" rowspan="1">GII.1</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">UNP262</td><td colspan="1" rowspan="1">GII.1</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">NYB 488093</td><td colspan="1" rowspan="1">GII.2</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">NYB 470206</td><td colspan="1" rowspan="1">GII.3</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">UNP062</td><td colspan="1" rowspan="1">GII.3</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">Guix N23.1 2009</td><td colspan="1" rowspan="1">GII.4</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">(2008) N23.1</td><td colspan="1" rowspan="1">GII.4</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">NYB 470009</td><td colspan="1" rowspan="1">GII.4</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="12"></td><td colspan="1" rowspan="1">UNP088</td><td colspan="1" rowspan="1">GII.4</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">UNP178</td><td colspan="1" rowspan="1">GII.4</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">UNP193</td><td colspan="1" rowspan="1">GII.4</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">UNP297</td><td colspan="1" rowspan="1">GII.4</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">UNP300</td><td colspan="1" rowspan="1">GII.4</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">NYB470011</td><td colspan="1" rowspan="1">GII.6</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">NYB 470169</td><td colspan="1" rowspan="1">GII.6</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">UNP082</td><td colspan="1" rowspan="1">GII.6</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">Guix FII.3</td><td colspan="1" rowspan="1">GII.12</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">J95.10</td><td colspan="1" rowspan="1">GII.12</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">NYB 470155</td><td colspan="1" rowspan="1">GII.17</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">MG10737</td><td colspan="1" rowspan="1">Unknown</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">100</td></tr></table>

In silico analysis predicted that most strains of all genotypes would be detected, though some variant strains may be detected with reduced sensitivity or may not be detected due to inefficient amplification or exclusion by melt analysis. For Norovirus in silico analysis, three sequences showed more than one mismatch, two GI.3 variants and one GI.7 variant. Some Norovirus sequences showed more than two mismatches, one GII.3 variant, one GII.4 variant, one GII.6 variant and one GII.12 variant. These mutations could affect the detection of these variants. For Rotavirus A in silico analysis, there were five variants that had more than three mismatches. There were ten Rotavirus A sequences that were truncated by four nucleotides. These mutations could affect the detection of these variants. For Sapovirus GI in silico analysis, one GI.2 variant showed more than one mismatch, this mutation could affect the detection of this variant. One Sapovirus GV sequence showed two mismatches, these mutations could affect the detection of this variant.

# f. Assay cut-off:

The clinical trial data used in this analysis included testing results collected from 2148 patients (1786 prospective specimens and 362 retrospective specimes) and seven testing sites. There were 1010 Cary-Blair preserved samples and 776 unpreserved samples collected prospectively and 136 Cary-Blair preserved samples and 226 unpreserved samples collected as part of a retrospective study. For each specimen, the BD MAX Enteric Viral Panel result and a reference method result were determined. The reference method result is determined from a composite reference method and the validation of the BD MAX cutoffs is performed against the reference method.

Table 21: Cutoff Threshold Values for BD MAX Enteric Viral Panel   

<table><tr><td rowspan=1 colspan=1>Cut-off EntericViral Panel</td><td rowspan=1 colspan=1>RoV</td><td rowspan=1 colspan=1>NoV</td><td rowspan=1 colspan=1>AdV</td><td rowspan=1 colspan=1>AstV</td><td rowspan=1 colspan=1>SaV</td><td rowspan=1 colspan=1>SPC</td></tr><tr><td rowspan=1 colspan=1>Ct.Scorethreshold</td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>32</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>Ct.Score Min</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>6</td></tr><tr><td rowspan=1 colspan=1>Ct.Score Max</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>38</td><td rowspan=1 colspan=1>39</td><td rowspan=1 colspan=1>37</td><td rowspan=1 colspan=1>38</td><td rowspan=1 colspan=1>40</td></tr><tr><td rowspan=1 colspan=1>EP Min.</td><td rowspan=1 colspan=1>201</td><td rowspan=1 colspan=1>202</td><td rowspan=1 colspan=1>70</td><td rowspan=1 colspan=1>190</td><td rowspan=1 colspan=1>130</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>PCR Noise Max.</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>PCR Noise Min.</td><td rowspan=1 colspan=1>3.8</td><td rowspan=1 colspan=1>3.8</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3.8</td><td rowspan=1 colspan=1>3.8</td><td rowspan=1 colspan=1>0</td></tr></table>

2. Comparison studies:

a. Method comparison with predicate device: Not applicable.

b. Matrix comparison: Not applicable.

3. Clinical studies:

a. Clinical Sensitivity:

# Prospective Study

Six clinical sites enrolled prospective specimens and performed BD MAX testing. Five were located in the U.S. and one was located in Canada. One additional internal BD site was used as a testing site only. One internal BD site performed reference method testing (alternate PCR and bi-directional sequencing). Sites were selected

based on several criteria, such as investigator and site personnel availability, number of specimens eligible, historical prevalence rate for each target, familiarity with reference methods and/or PCR methodologies, and characteristics of their respective served patient populations.

All six geographically diverse clinical sites enrolled prospective specimens that were being collected as part of routine patient care. The collection period for Enteric Viral Panel Prospective specimens ranged from November 2015 to April 2017. The study included 1873 Prospective specimens (818 unpreserved and 1055 Cary-Blair preserved). These specimens were tested within the stability period established for the BD MAX Enteric Viral Panel Assay and reference method.

# Retrospective Study

Due to the low prevalence of the assay targets, archived (retrospective) specimens collected from November 2011 to March 2017 were utilized to support the number of positive specimens for all five targets. Retrospective specimens were collected from the USA, Canada, and Uganda. The retrospective specimens were obtained from various sources, including the participating clinical centers and commercial sources. A total of 366 retrospective specimens were enrolled in the study (230 unpreserved and 136 Cary-Blair preserved). Historical results were used as the first portion of the composite reference method for the retrospective specimens.

# Reference Method (RM)

The sponsor employed a composite reference method consisting of two validated alternate PCRs followed by bi-directional sequencing analysis for one of the primer sets. Confirmatory PCR primers were designed to target different regions of gene sequence than those targeted by the BD MAX Enteric Viral Panel assay. Importantly, while all prospective samples tested on the BD MAX Enteric Viral Panel assay were evaluated fresh, they were frozen before being tested by the reference method.

Only specimens with positive or negative RM results and positive, negative, UNR or IND MAX results were analyzed for each virus, which resulted in 1881 results for Norovirus, 1909 results for Rotavirus, 1871 results for Adenovirus, 1827 results for Sapovirus and 1847 results for Astrovirus. The primary reason for specimen exclusion was the lack of data for the RM, but other reasons included specimens that were too old, duplicate specimens that were collected from the same patient, and samples that were enrolled before IRB renewal.

The sponsor submitted results from extensive testing of analytical LoD determination and inclusivity testing for the reference PCR methods that utilized analytes prepared from stock viral culture as well as characterized clinical stool specimens. The LoD was determined for each of the 21 reference method assays. Testing was then performed on ten representative assays, covering all five viral targets, in twenty sample replicates at 1x LoD to validate viral DNA/RNA extraction from stool

samples, RT-PCR amplification and subsequent sequencing of viral genomic targets. Samples with amplification above the cutoff values were called positive; any concentration below these values was deemed negative. Non-sequencing master mixes usually used cutoffs for YmaxEP conditions. Sequencing master mix positives were based on melt temperature. However, the mixes were required to be sequenced prior to calling a positive result.

To demonstrate analytical reactivity (inclusivity) for the reference method 61 strains were tested by the non-sequencing PCR, with 41 confirmed $2 9 5 \%$ positive at $3 \mathbf { x }$ LoD. Seven more were confirmed at 30x LoD, five more were confirmed at $3 0 0 \mathrm { x } \mathrm { L o D }$ , and four more required titration to 3000x LoD. The four isolates not detected were: ID Sapovirus NYB 470159, Guix NoV GI.7 10.31750, NoV MG10737, and Guix NoV GII.12 FII.3.

The sequencing PCR method detected 39/60 tested strains at 3x LoD. The remaining samples were then titrated up to $3 0 \mathrm { x }$ LoD where seven more samples were confirmed. Nine samples confirmed at $3 0 0 \mathrm { x } \mathrm { L o D }$ , and finally two were confirmed at 3000x LoD. The remaining three samples were not detected. The three strains not detected were: Astrovirus Guix HAstV-7, Sapovirus BA0141AP, and Sapovirus NYB 470159.

# Acceptance Criteria

Performance of $2 9 0 \%$ positive agreement of a NAAT test and NAAT reference method was considered acceptable for Norovirus, Rotavirus, Adenovirus and Astrovirus compared to current clinical practice. The acceptable performance of Sapovirus was adjusted by the sponsor to a point estimate of $2 8 0 \%$ on the rationale that the infection has a reduced severity compared to the other assay targets.

# Study Demographics

The clinical study analyzed a total of 1146 Cary-Blair Preserved stool samples and 1002 Unpreserved stool samples across in-patient, out-patient, emergency, and longterm care facility patient groups. The sponsor provided the following demographic breakdown for the samples from the combined Retrospective and Prospective clinical studies.

Table 22: Combined Prospective and Retrospective Study Demographics   

<table><tr><td rowspan=1 colspan=1>Age Group</td><td rowspan=1 colspan=1>Cary-BlairPreserved</td><td rowspan=1 colspan=1>Unpreserved</td><td rowspan=1 colspan=1>Combined</td></tr><tr><td rowspan=1 colspan=1>0-1 month</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>4</td></tr><tr><td rowspan=1 colspan=1>1 month − 2 y/o</td><td rowspan=1 colspan=1>188</td><td rowspan=1 colspan=1>112</td><td rowspan=1 colspan=1>300</td></tr><tr><td rowspan=1 colspan=1>2-12 y/o</td><td rowspan=1 colspan=1>228</td><td rowspan=1 colspan=1>153</td><td rowspan=1 colspan=1>381</td></tr><tr><td rowspan=1 colspan=1>13-18 y/o</td><td rowspan=1 colspan=1>117</td><td rowspan=1 colspan=1>66</td><td rowspan=1 colspan=1>183</td></tr><tr><td rowspan=1 colspan=1>19-21 y/o</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>41</td></tr><tr><td rowspan=1 colspan=1>&gt;21 y/o</td><td rowspan=1 colspan=1>568</td><td rowspan=1 colspan=1>640</td><td rowspan=1 colspan=1>1208</td></tr><tr><td rowspan=1 colspan=1>Unknown</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>31</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>1146</td><td rowspan=1 colspan=1>1002</td><td rowspan=1 colspan=1>2148</td></tr></table>

Norovirus Detection

Among prospectively collected fresh specimens, the BD MAX Enteric Viral Panel assay achieved a PPA of $9 2 . 5 \%$ ( $9 5 \%$ CI: $8 4 . 6 { - } 9 6 . 5 \% )$ for Cary-Blair preserved stool samples and a PPA of $9 0 . 7 \%$ ( $9 5 \%$ CI: $7 8 . 4 \substack { - 9 6 . 3 \% }$ for unpreserved stool samples.

Table 23: Norovirus Clinical Performance Stratified by Fresh/Frozen Sample Status   

<table><tr><td rowspan=2 colspan=1>Specimen Type</td><td rowspan=2 colspan=1>Specimen Origin</td><td rowspan=2 colspan=1>BD MAX</td><td rowspan=1 colspan=2>RM</td><td rowspan=2 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>P</td><td rowspan=1 colspan=1>N</td></tr><tr><td rowspan=3 colspan=1>Cary-Blair Preserved</td><td rowspan=3 colspan=1>Prospective (fresh)</td><td rowspan=1 colspan=1>P</td><td rowspan=1 colspan=1>74</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>81</td></tr><tr><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>835</td><td rowspan=1 colspan=1>841</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>80</td><td rowspan=1 colspan=1>842</td><td rowspan=1 colspan=1>922</td></tr><tr><td rowspan=1 colspan=6>PPA (95% CI): 92.5% (84.6-96.5%)</td></tr><tr><td rowspan=1 colspan=6>NPA (95% CI): 99.2% (98.3-99.6%)</td></tr><tr><td rowspan=3 colspan=1>Cary-Blair Preserved</td><td rowspan=3 colspan=1>Retrospective (frozen)</td><td rowspan=1 colspan=1>P</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>7</td></tr><tr><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>105</td><td rowspan=1 colspan=1>105</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>106</td><td rowspan=1 colspan=1>112</td></tr><tr><td rowspan=1 colspan=6>PPA (95% CI): 100% (61-100%)</td></tr><tr><td rowspan=1 colspan=6>NPA (95% CI): 99.1% (94.8-99.9%)</td></tr><tr><td rowspan=3 colspan=1>Unpreserved</td><td rowspan=3 colspan=1>Prospective (fresh)</td><td rowspan=1 colspan=1>P</td><td rowspan=1 colspan=1>39</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>42</td></tr><tr><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>694</td><td rowspan=1 colspan=1>698</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>43</td><td rowspan=1 colspan=1>697</td><td rowspan=1 colspan=1>740</td></tr><tr><td rowspan=1 colspan=4>PPA (95% CI): 90.7% (78.4-96.3%)</td><td rowspan=1 colspan=2></td></tr><tr><td rowspan=1 colspan=6>NPA (95% CI): 99.6% (98.7-99.9%)</td></tr><tr><td rowspan=3 colspan=1>Unpreserved</td><td rowspan=3 colspan=1>Retrospective (frozen)</td><td rowspan=1 colspan=1>P</td><td rowspan=1 colspan=1>35</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>35</td></tr><tr><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>58</td><td rowspan=1 colspan=1>60</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>37</td><td rowspan=1 colspan=1>58</td><td rowspan=1 colspan=1>95</td></tr><tr><td rowspan=1 colspan=6>PPA (95% CI): 94.6% (82.3-98.5%)</td></tr><tr><td rowspan=1 colspan=6>NPA (95% CI): 100% (93.8-100%)</td></tr></table>

Table 24: Norovirus Clinical Performance Stratified by Stool Type Only   

<table><tr><td rowspan=2 colspan=1>Specimen Type</td><td rowspan=2 colspan=1>BD MAX</td><td rowspan=1 colspan=2>RM</td><td rowspan=2 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>P</td><td rowspan=1 colspan=1>N</td></tr><tr><td rowspan=3 colspan=1>Cary-Blair Preserved</td><td rowspan=1 colspan=1>P</td><td rowspan=1 colspan=1>80</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>88</td></tr><tr><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>940</td><td rowspan=1 colspan=1>946</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>86</td><td rowspan=1 colspan=1>948</td><td rowspan=1 colspan=1>1034</td></tr><tr><td rowspan=1 colspan=1>PPA (95% CI</td><td rowspan=1 colspan=4>PPA (95% CI): 93% (85.6-96.8%)</td></tr><tr><td rowspan=1 colspan=5>NPA (95%CI): 99.2% (98.3-99.6%)</td></tr><tr><td rowspan=3 colspan=1>Unpreserved</td><td rowspan=1 colspan=1>P</td><td rowspan=1 colspan=1>74</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>77</td></tr><tr><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>752</td><td rowspan=1 colspan=1>758</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>80</td><td rowspan=1 colspan=1>755</td><td rowspan=1 colspan=1>835</td></tr><tr><td rowspan=1 colspan=5>PPA (95% CI): 92.5% (84.6-96.5%)</td></tr><tr><td rowspan=1 colspan=5>NPA (95% CI): 99.6% (98.8-99.9%)</td></tr></table>

Historical data for 2 of BD MAX Enteric Viral Panel false negative samples indicate positive FilmArray GI Panel status. Another 6 Enteric Viral Panel false negative Cary-Blair preserved samples tested by FilmArray GI panel were negative. One other false negative sample displayed an alert code for LLS Failure. Seven false positive preserved samples available for discrepant resolution were also positive by FilmArray GI panel.

# Rotavirus Detection

Among prospectively collected fresh specimens, the BD MAX Enteric Viral Panel assay achieved a PPA of $100 \%$ ( $9 5 \%$ CI: $8 9 - 1 0 0 \%$ ) for Cary-Blair preserved stool samples and a PPA of $100 \%$ $9 5 \%$ CI: $7 4 . 1 \mathrm { - } 1 0 0 \%$ ) for unpreserved stool samples.

Table 25: Rotavirus Clinical Performance Stratified by Fresh/Frozen Sample Status   

<table><tr><td colspan="1" rowspan="2">Specimen Type</td><td colspan="1" rowspan="2">Specimen Origin</td><td colspan="1" rowspan="2">BD MAX</td><td colspan="2" rowspan="1">RM</td><td colspan="1" rowspan="2">Total</td></tr><tr><td colspan="1" rowspan="1">P</td><td colspan="1" rowspan="1">N</td></tr><tr><td colspan="1" rowspan="3">Cary-Blair Preserved</td><td colspan="1" rowspan="3">Prospective (fresh)</td><td colspan="1" rowspan="1">P</td><td colspan="1" rowspan="1">31</td><td colspan="1" rowspan="1">7</td><td colspan="1" rowspan="1">38</td></tr><tr><td colspan="1" rowspan="1">N</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">888</td><td colspan="1" rowspan="1">888</td></tr><tr><td colspan="1" rowspan="1">Total</td><td colspan="1" rowspan="1">31</td><td colspan="1" rowspan="1">895</td><td colspan="1" rowspan="1">926</td></tr><tr><td colspan="6" rowspan="1">PPA (95% CI): 100% (89.0-100%)</td></tr><tr><td colspan="6" rowspan="1">NPA (95% CI): 99.2% (98.4-99.6%)</td></tr><tr><td colspan="1" rowspan="3">Cary-Blair Preserved</td><td colspan="1" rowspan="3">Retrospective (frozen)</td><td colspan="1" rowspan="1">P</td><td colspan="1" rowspan="1">38</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">39</td></tr><tr><td colspan="1" rowspan="1">N</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">76</td><td colspan="1" rowspan="1">76</td></tr><tr><td colspan="1" rowspan="1">Total</td><td colspan="1" rowspan="1">38</td><td colspan="1" rowspan="1">77</td><td colspan="1" rowspan="1">115</td></tr><tr><td colspan="6" rowspan="1">PPA (95% CI): 100% (90.8-100%)</td></tr><tr><td colspan="6" rowspan="1">NPA (95% CI): 98.7% (93-99.8%)</td></tr><tr><td colspan="1" rowspan="3">Unpreserved</td><td colspan="1" rowspan="3">Prospective (fresh)</td><td colspan="1" rowspan="1">P</td><td colspan="1" rowspan="1">11</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">12</td></tr><tr><td colspan="1" rowspan="1">N</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">735</td><td colspan="1" rowspan="1">735</td></tr><tr><td colspan="1" rowspan="1">Total</td><td colspan="1" rowspan="1">11</td><td colspan="1" rowspan="1">736</td><td colspan="1" rowspan="1">747</td></tr><tr><td colspan="6" rowspan="1">PPA (95% CI): 100% (74.1-100%)</td></tr><tr><td colspan="6" rowspan="1">NPA (95% CI): 99.9% (99.2-100%)</td></tr><tr><td colspan="1" rowspan="3">Unpreserved</td><td colspan="1" rowspan="3">Retrospective (frozen)</td><td colspan="1" rowspan="1">P</td><td colspan="1" rowspan="1">56</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">57</td></tr><tr><td colspan="1" rowspan="1">N</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">47</td><td colspan="1" rowspan="1">47</td></tr><tr><td colspan="1" rowspan="1">Total</td><td colspan="1" rowspan="1">56</td><td colspan="1" rowspan="1">48</td><td colspan="1" rowspan="1">104</td></tr><tr><td colspan="6" rowspan="1">PPA (95% CI): 100% (93.6-100%)</td></tr><tr><td colspan="6" rowspan="1">NPA (95% CI): 97.9% (89.1-99.6%)</td></tr></table>

Table 26: Rotavirus Clinical Performance Stratified by Stool Type Only   

<table><tr><td rowspan=2 colspan=1>Specimen Type</td><td rowspan=2 colspan=1>BD MAX</td><td rowspan=1 colspan=2>RM</td><td rowspan=2 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>P</td><td rowspan=1 colspan=1>N</td></tr><tr><td rowspan=3 colspan=1>Cary-Blair Preserved</td><td rowspan=1 colspan=1>P</td><td rowspan=1 colspan=1>69</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>77</td></tr><tr><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>964</td><td rowspan=1 colspan=1>964</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>69</td><td rowspan=1 colspan=1>972</td><td rowspan=1 colspan=1>1041</td></tr><tr><td rowspan=1 colspan=5>PPA (95% CI): 100% (94.7-100%)</td></tr><tr><td rowspan=1 colspan=5>NPA (95%CI): 99.2% (98.4-99.6%)</td></tr><tr><td rowspan=3 colspan=1>Unpreserved</td><td rowspan=1 colspan=1>P</td><td rowspan=1 colspan=1>67</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>69</td></tr><tr><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>782</td><td rowspan=1 colspan=1>782</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>67</td><td rowspan=1 colspan=1>784</td><td rowspan=1 colspan=1>851</td></tr><tr><td rowspan=1 colspan=5>PPA (95% CI): 100% (94.6-100%)</td></tr><tr><td rowspan=1 colspan=5>NPA (95% CI): 99.7% (99.1-99.9%)</td></tr></table>

Seven false positive Cary-Blair preserved results available for discrepant analysis were tested with the FilmArray GI panel. $4 / 7$ of these samples were positive. Two other preserved false positive specimens had historical negative results on either the FilmArray GI panel or Luminex xTAG GPP assays.

# Adenovirus Detection

Among prospectively collected fresh specimens, the BD MAX Enteric Viral Panel assay achieved a PPA of $9 3 . 8 \%$ ( $9 5 \%$ CI: $7 1 . 7 – 9 8 . 9 \%$ for Cary-Blair preserved stool samples and a PPA of $80 \%$ ( $9 5 \%$ CI: $3 7 . 6 \mathrm { - } 9 6 . 4 \%$ for unpreserved stool samples.

Clinical performance of the BD MAX Enteric Viral Panel Adenovirus assay failed to meet specific acceptance criteria for PPA due to a low number of positive samples detected in the patient cohort. To supplement their prospective study results, the sponsor performed additional contrived testing.

Table 27: Adenovirus Clinical Performance Stratified by Fresh/Frozen Sample Status   

<table><tr><td rowspan=2 colspan=1>Specimen Type</td><td rowspan=2 colspan=1>Specimen Origin</td><td rowspan=2 colspan=1>BD MAX</td><td rowspan=1 colspan=2>RM</td><td rowspan=2 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>P</td><td rowspan=1 colspan=1>N</td></tr><tr><td rowspan=3 colspan=1>Cary-Blair Preserved</td><td rowspan=3 colspan=1>Prospective (fresh)</td><td rowspan=1 colspan=1>P</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>15</td></tr><tr><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>914</td><td rowspan=1 colspan=1>915</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>914</td><td rowspan=1 colspan=1>930</td></tr><tr><td rowspan=1 colspan=6>PPA (95% CI): 93.8% (71.7-98.9%)</td></tr><tr><td rowspan=1 colspan=6>NPA (95% CI): 100% (99.6-100%)</td></tr><tr><td rowspan=3 colspan=1>Cary-Blair Preserved</td><td rowspan=3 colspan=1>Retrospective (frozen)</td><td rowspan=1 colspan=1>P</td><td rowspan=1 colspan=1>18</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>18</td></tr><tr><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>84</td><td rowspan=1 colspan=1>84</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>18</td><td rowspan=1 colspan=1>84</td><td rowspan=1 colspan=1>102</td></tr><tr><td rowspan=1 colspan=6>PPA (95% CI): 100% (82.4-100%)</td></tr><tr><td rowspan=1 colspan=6>NPA (95% CI): 100% (95.6-100%)</td></tr><tr><td rowspan=3 colspan=1>Unpreserved</td><td rowspan=3 colspan=1>Prospective (fresh)</td><td rowspan=1 colspan=1>P</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>5</td></tr><tr><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>747</td><td rowspan=1 colspan=1>748</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>748</td><td rowspan=1 colspan=1>753</td></tr><tr><td rowspan=1 colspan=6>PPA (95% CI): 80% (37.6-96.4%)</td></tr><tr><td rowspan=1 colspan=6>NPA (95% CI): 99.9% (99.2-100%)</td></tr><tr><td rowspan=3 colspan=1>Unpreserved</td><td rowspan=3 colspan=1>Retrospective (frozen)</td><td rowspan=1 colspan=1>P</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>6</td></tr><tr><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>68</td><td rowspan=1 colspan=1>68</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>68</td><td rowspan=1 colspan=1>74</td></tr><tr><td rowspan=1 colspan=6>PPA (95% CI): 100% (61-100%)</td></tr><tr><td rowspan=1 colspan=6>NPA (95% CI): 100% (94.7-99.6%)</td></tr></table>

Table 28: Adenovirus Clinical Performance Stratified by Stool Type Only   

<table><tr><td rowspan=2 colspan=1>Specimen Type</td><td rowspan=2 colspan=1>BD MAX</td><td rowspan=1 colspan=2>RM</td><td rowspan=2 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>P</td><td rowspan=1 colspan=1>N</td></tr><tr><td rowspan=3 colspan=1>Cary-Blair Preserved</td><td rowspan=1 colspan=1>P</td><td rowspan=1 colspan=1>33</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>33</td></tr><tr><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>998</td><td rowspan=1 colspan=1>999</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>34</td><td rowspan=1 colspan=1>998</td><td rowspan=1 colspan=1>1032</td></tr><tr><td rowspan=1 colspan=5>PPA (95% CI): 97.1% (85.1-99.5%)</td></tr><tr><td rowspan=1 colspan=1>NPA (95% CI): 100%</td><td rowspan=1 colspan=2>(99.6-100%)</td><td rowspan=1 colspan=2></td></tr><tr><td rowspan=3 colspan=1>Unpreserved</td><td rowspan=1 colspan=1>P</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>11</td></tr><tr><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>815</td><td rowspan=1 colspan=1>816</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>816</td><td rowspan=1 colspan=1>827</td></tr><tr><td rowspan=1 colspan=5>PPA (95% CI): 90.9% (62.3-98.4%)</td></tr><tr><td rowspan=1 colspan=5>NPA (95% CI): 99.9% (99.3-100%)</td></tr></table>

Since the minimum number of RM positives required as per the protocol was not reached for Adenovirus among unpreserved specimens, contrived samples were also tested. Contrived samples were prepared by spiking stock Adenovirus into BD MAX SBT and adding unpreserved clinical stool matrix. A total of 48 samples were tested at two external sites for a total of 96 samples. $50 \%$ of contrived specimens were made at $1 . 9 9 \mathrm { { x } }$ LoD while the other $50 \%$ of specimens spanned the expected clinical range of pathogen concentrations. Negative controls were prepared from the same clinical matrix unspiked with virus. These results conformed to expected result with high PPA and NPA values.

Table 29: Adenovirus Contrived Clinical Sample Performance   

<table><tr><td rowspan=2 colspan=1>BD MAX</td><td rowspan=1 colspan=2>Expected Result</td><td rowspan=2 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>48</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>48</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>48</td><td rowspan=1 colspan=1>48</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>48</td><td rowspan=1 colspan=1>48</td><td rowspan=1 colspan=1>96</td></tr><tr><td rowspan=1 colspan=4>PPA (95%CI): 100%(92.6-100%)NPA (95% CI): 100% (92.6-100%)</td></tr></table>

# Sapovirus Detection

Among prospectively collected fresh specimens, the BD MAX Enteric Viral Panel assay achieved a PPA of $8 7 . 8 \%$ $9 5 \%$ CI: $7 5 . 8 – 9 4 . 3 \%$ ) for Cary-Blair preserved stool samples and a PPA of $80 \%$ ( $9 5 \%$ CI: $6 2 . 7 { - } 9 0 . 5 \% )$ for unpreserved stool samples.

BD MAX Enteric Viral Panel Sapovirus detection met the sponsor’s indicated lower acceptance criteria (a PPA $2 8 0 \%$ as opposed to $90 \%$ for other analytes).

Table 30: Sapovirus Clinical Performance Stratified by Fresh/Frozen Sample Status   

<table><tr><td rowspan=2 colspan=1>Specimen Type</td><td rowspan=2 colspan=1>Specimen Origin</td><td rowspan=2 colspan=1>BD MAX</td><td rowspan=1 colspan=2>RM</td><td rowspan=2 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>P</td><td rowspan=1 colspan=1>N</td></tr><tr><td rowspan=3 colspan=1>Cary-Blair Preserved</td><td rowspan=3 colspan=1>Prospective (fresh)</td><td rowspan=1 colspan=1>P</td><td rowspan=1 colspan=1>43</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>52</td></tr><tr><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>863</td><td rowspan=1 colspan=1>869</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>49</td><td rowspan=1 colspan=1>872</td><td rowspan=1 colspan=1>921</td></tr><tr><td rowspan=1 colspan=6>PPA (95% CI): 87.8% (75.8-94.3%)</td></tr><tr><td rowspan=1 colspan=6>NPA (95% CI): 99.0% (98.1-99.5%)</td></tr><tr><td rowspan=3 colspan=1>Cary-Blair Preserved</td><td rowspan=3 colspan=1>Retrospective (frozen)</td><td rowspan=1 colspan=1>P</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2</td></tr><tr><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>98</td><td rowspan=1 colspan=1>99</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>98</td><td rowspan=1 colspan=1>101</td></tr><tr><td rowspan=1 colspan=6>PPA (95% CI): 66.7% (20.8-93.9%)</td></tr><tr><td rowspan=1 colspan=6>NPA (95% CI): 100% (96.2-100%)</td></tr><tr><td rowspan=3 colspan=1>Unpreserved</td><td rowspan=3 colspan=1>Prospective (fresh)</td><td rowspan=1 colspan=1>P</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>25</td></tr><tr><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>720</td><td rowspan=1 colspan=1>726</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>721</td><td rowspan=1 colspan=1>751</td></tr><tr><td rowspan=1 colspan=6>PPA (95% CI): 80% (62.7-90.5%)</td></tr><tr><td rowspan=1 colspan=6>NPA (95% CI): 99.9% (99.2-100%)</td></tr><tr><td rowspan=3 colspan=1>Unpreserved</td><td rowspan=3 colspan=1>Retrospective (frozen)</td><td rowspan=1 colspan=1>P</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>5</td></tr><tr><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>39</td><td rowspan=1 colspan=1>39</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>44</td></tr><tr><td rowspan=1 colspan=6>PPA (95% CI): 100% (51-100%)</td></tr><tr><td rowspan=1 colspan=6>NPA (95% CI): 97.5% (87.1-99.6%)</td></tr></table>

Table 31: Sapovirus Clinical Performance Stratified by Stool Type Only   

<table><tr><td rowspan=2 colspan=1>Specimen Type</td><td rowspan=2 colspan=1>BD MAX</td><td rowspan=1 colspan=2>RM</td><td rowspan=2 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>P</td><td rowspan=1 colspan=1>N</td></tr><tr><td rowspan=3 colspan=1>Cary-Blair Preserved</td><td rowspan=1 colspan=1>P</td><td rowspan=1 colspan=1>45</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>54</td></tr><tr><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>961</td><td rowspan=1 colspan=1>968</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>52</td><td rowspan=1 colspan=1>970</td><td rowspan=1 colspan=1>1022</td></tr><tr><td rowspan=1 colspan=5>PPA (95% CI): 86.5% (74.7-93.3%)</td></tr><tr><td rowspan=1 colspan=5>NPA (95%CI): 99.1% (98.2-99.5%)</td></tr><tr><td rowspan=3 colspan=1>Unpreserved</td><td rowspan=1 colspan=1>P</td><td rowspan=1 colspan=1>28</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>30</td></tr><tr><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>759</td><td rowspan=1 colspan=1>765</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>34</td><td rowspan=1 colspan=1>761</td><td rowspan=1 colspan=1>795</td></tr><tr><td rowspan=1 colspan=5>PPA (95% CI): 82.4% (66.5-91.7%)</td></tr><tr><td rowspan=1 colspan=5>NPA (95% CI): 99.7% (99-99.9%)</td></tr></table>

Nine false positive Cary-Blair preserved specimens were available for discrepant testing by FilmArray GI Panel. Four out of nine of these samples tested positive on the predicate device. One other false positive specimen had a negative historical result by an unknown “other molecular method”.

Six false negative preserved samples were similarly tested by the FilmArray GI panel with four specimens testing negative for Sapovirus.

Despite meeting a less stringent acceptance criterion, the observed lower PPA between the BD MAX Enteric Viral Panel and the reference method for Sapovirus among unpreserved stool samples raised potential performance concerns. These concerns were mitigated by additional discrepant testing performed by the sponsor. Briefly, discordant unpreserved stool samples were mixed with Cary-Blair transport medium and evaluated on the BioFire FilmArray GI panel. 5/6 unpreserved false negative samples were also negative on the FilmArray GI panel. The one unpreserved BD MAX Enteric Viral Panel false positive specimen, however, was negative by BioFire FilmArray GI panel. These results support similar Sapovirus performance characteristics between the BD MAX Enteric Viral Panel assay and the predicate device.

# Astrovirus Detection

Among prospectively collected fresh specimens, the BD MAX Enteric Viral Panel assay achieved a PPA of $9 3 . 5 \%$ ( $9 5 \%$ CI: $7 9 . 3 \mathrm { - } 9 8 . 2 \% )$ for Cary-Blair preserved stool samples and a PPA of $9 3 . 3 \%$ ( $9 5 \%$ CI: $7 8 . 7 \mathrm { - } 9 8 . 2 \% )$ for unpreserved stool samples.

Table 32: Astrovirus Clinical Performance Stratified by Fresh/Frozen Sample Status   

<table><tr><td rowspan=2 colspan=1>Specimen Type</td><td rowspan=2 colspan=1>Specimen Origin</td><td rowspan=2 colspan=1>BD MAX</td><td rowspan=1 colspan=2>RM</td><td rowspan=2 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>P</td><td rowspan=1 colspan=1>N</td></tr><tr><td rowspan=3 colspan=1>Cary-Blair Preserved</td><td rowspan=3 colspan=1>Prospective (fresh)</td><td rowspan=1 colspan=1>P</td><td rowspan=1 colspan=1>29</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>30</td></tr><tr><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>899</td><td rowspan=1 colspan=1>901</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>31</td><td rowspan=1 colspan=1>900</td><td rowspan=1 colspan=1>931</td></tr><tr><td rowspan=1 colspan=6>PPA (95% CI): 93.5% (79.3-98.2%)</td></tr><tr><td rowspan=1 colspan=6>NPA (95% CI): 99.9% (99.4-100%)</td></tr><tr><td rowspan=3 colspan=1>Cary-Blair Preserved</td><td rowspan=3 colspan=1>Retrospective (frozen)</td><td rowspan=1 colspan=1>P</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>2</td></tr><tr><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>80</td><td rowspan=1 colspan=1>82</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>22</td><td rowspan=1 colspan=1>81</td><td rowspan=1 colspan=1>103</td></tr><tr><td rowspan=1 colspan=6>PPA (95% CI): 90.9% (72.2-97.5%)</td></tr><tr><td rowspan=1 colspan=6>NPA (95% CI): 98.8% (93.3-99.8%)</td></tr><tr><td rowspan=3 colspan=1>Unpreserved</td><td rowspan=3 colspan=1>Prospective (fresh)</td><td rowspan=1 colspan=1>P</td><td rowspan=1 colspan=1>28</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>30</td></tr><tr><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>722</td><td rowspan=1 colspan=1>724</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>724</td><td rowspan=1 colspan=1>754</td></tr><tr><td rowspan=1 colspan=6>PPA (95% CI): 93.3% (78.7-98.2%)</td></tr><tr><td rowspan=1 colspan=6>NPA (95% CI): 99.7% (99-99.9%)</td></tr><tr><td rowspan=3 colspan=1>Unpreserved</td><td rowspan=3 colspan=1>Retrospective (frozen)</td><td rowspan=1 colspan=1>P</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>4</td></tr><tr><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>45</td><td rowspan=1 colspan=1>45</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>46</td><td rowspan=1 colspan=1>49</td></tr><tr><td rowspan=1 colspan=6>PPA (95% CI): 100% (43.9-100%)</td></tr><tr><td rowspan=1 colspan=6>NPA (95% CI): 97.8% (88.7-99.6%)</td></tr></table>

Table 33: Astrovirus Clinical Performance Stratified by Stool Type Only   

<table><tr><td rowspan=2 colspan=1>Specimen Type</td><td rowspan=2 colspan=1>BD MAX</td><td rowspan=1 colspan=2>RM</td><td rowspan=2 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>P</td><td rowspan=1 colspan=1>N</td></tr><tr><td rowspan=3 colspan=1>Cary-Blair Preserved</td><td rowspan=1 colspan=1>P</td><td rowspan=1 colspan=1>49</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>51</td></tr><tr><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>979</td><td rowspan=1 colspan=1>983</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>53</td><td rowspan=1 colspan=1>981</td><td rowspan=1 colspan=1>1034</td></tr><tr><td rowspan=1 colspan=1>PPA (95% CI): 92.5%</td><td rowspan=1 colspan=4>(82.1-97%)</td></tr><tr><td rowspan=1 colspan=5>NPA (95%CI): 99.8%(99.3-99.9%)</td></tr><tr><td rowspan=3 colspan=1>Unpreserved</td><td rowspan=1 colspan=1>P</td><td rowspan=1 colspan=1>31</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>34</td></tr><tr><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>767</td><td rowspan=1 colspan=1>769</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>33</td><td rowspan=1 colspan=1>770</td><td rowspan=1 colspan=1>803</td></tr><tr><td rowspan=1 colspan=5>PPA (95% CI): 93.9% (80.4-98.3%)</td></tr><tr><td rowspan=1 colspan=5>NPA (95% CI): 99.6% (98.9-99.9%)</td></tr></table>

One false positive Cary-Blair preserved result was tested by FilmArray GI panel with negative results. Two other false positive specimens had historical negative Astrovirus results by “other molecular methods.”

Two false negative preserved samples tested by FilmArray GI panel were negative for Astrovirus. An additional two other false negatives had positive Astrovirus historical results from “other molecular methods.”

b. Clinical specificity: See section M.3a above.   
c. Other clinical supportive data (when a. and b. are not applicable): Not applicable.

4. Clinical cut-off: Not applicable.

5. Expected values/Reference range:

All six geographically diverse clinical sites enrolled Prospective specimens that were being collected as part of routine patient care. The collection period for Enteric Viral Panel Prospective specimens ranged from November 2015 to April 2017. The study included 1873 Prospective specimens (818 unpreserved and 1055 Cary-Blair preserved). The observed prevalence based on the composite reference method and the expected value summaries as based on BD MAX Enteric Viral Panel assay results for the prospective study are summarized in the tables below.

Table 34: Observed Prevalence based on Composite Reference Method   

<table><tr><td></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>sel vea i leva</td><td rowspan=1 colspan=1>Clce basca 0l</td><td rowspan=1 colspan=1>Conposree 1</td><td></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>SpecimenType</td><td rowspan=1 colspan=1>Site</td><td rowspan=1 colspan=1>Norovirus</td><td rowspan=1 colspan=1>Rotavirus</td><td rowspan=1 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>Sapovirus</td><td rowspan=1 colspan=1>Astrovirus</td></tr><tr><td rowspan=5 colspan=1>Cary-BlairPreserved</td><td rowspan=1 colspan=1>ALB</td><td rowspan=1 colspan=1>3.4%(2/59)</td><td rowspan=1 colspan=1>0.0%(0/59)</td><td rowspan=1 colspan=1>0.0%(0/59)</td><td rowspan=1 colspan=1>0.0%(0/59)</td><td rowspan=1 colspan=1>1.7%(1/59)</td></tr><tr><td rowspan=1 colspan=1>CIN</td><td rowspan=1 colspan=1>13.7%(53/387)</td><td rowspan=1 colspan=1>4.8%(19/392)</td><td rowspan=1 colspan=1>3.0%(12/394)</td><td rowspan=1 colspan=1>10.1%(39/387)</td><td rowspan=1 colspan=1>5.6%(22/394)</td></tr><tr><td rowspan=1 colspan=1>JHU</td><td rowspan=1 colspan=1>8.2%(15/184)</td><td rowspan=1 colspan=1>3.8%(7/185)</td><td rowspan=1 colspan=1>0.5%(1/185)</td><td rowspan=1 colspan=1>0.0%(0/184)</td><td rowspan=1 colspan=1>1.1%(2/185)</td></tr><tr><td rowspan=1 colspan=1>POR</td><td rowspan=1 colspan=1>3.9%(14/359)</td><td rowspan=1 colspan=1>1.7%(6/358)</td><td rowspan=1 colspan=1>0.8%(3/360)</td><td rowspan=1 colspan=1>3.1%(11/358)</td><td rowspan=1 colspan=1>1.9%(7/360)</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>8.5%(84/989)</td><td rowspan=1 colspan=1>3.2%(32/994)</td><td rowspan=1 colspan=1>1.6%(16/998)</td><td rowspan=1 colspan=1>5.1%(50/988)</td><td rowspan=1 colspan=1>3.2%(32/998)</td></tr><tr><td rowspan=7 colspan=1>Unpreserved</td><td rowspan=1 colspan=1>ALB</td><td rowspan=1 colspan=1>0.0%(0/1)</td><td rowspan=1 colspan=1>0.0%(0/1)</td><td rowspan=1 colspan=1>0.0%(0/1)</td><td rowspan=1 colspan=1>0.0%(0/1)</td><td rowspan=1 colspan=1>0.0%(0/1)</td></tr><tr><td rowspan=1 colspan=1>CAL</td><td rowspan=1 colspan=1>4.7%(27/578)</td><td rowspan=1 colspan=1>1.0%(6/580)</td><td rowspan=1 colspan=1>0.7%(4/585)</td><td rowspan=1 colspan=1>3.6%(21/584)</td><td rowspan=1 colspan=1>4.8%(28/585)</td></tr><tr><td rowspan=1 colspan=1>CIN</td><td rowspan=1 colspan=1>0.0%(0/2)</td><td rowspan=1 colspan=1>33.3%(1/3)</td><td rowspan=1 colspan=1>0.0%(0/3)</td><td rowspan=1 colspan=1>0.0%(0/3)</td><td rowspan=1 colspan=1>0.0%(0/3)</td></tr><tr><td rowspan=1 colspan=1>JHU</td><td rowspan=1 colspan=1>10.3%(7/68)</td><td rowspan=1 colspan=1>0.0%(0/69)</td><td rowspan=1 colspan=1>0.0%(0/69)</td><td rowspan=1 colspan=1>0.0%(0/68)</td><td rowspan=1 colspan=1>0.0%(0/69)</td></tr><tr><td rowspan=1 colspan=1>LUR</td><td rowspan=1 colspan=1>6.9%(7/101)</td><td rowspan=1 colspan=1>3.8%(4/104)</td><td rowspan=1 colspan=1>1.0%(1/104)</td><td rowspan=1 colspan=1>6.8%(7/103)</td><td rowspan=1 colspan=1>1.9%(2/104)</td></tr><tr><td rowspan=1 colspan=1>POR</td><td rowspan=1 colspan=1>16.7%(2/12)</td><td rowspan=1 colspan=1>0.0%(0/12)</td><td rowspan=1 colspan=1>0.0%(0/13)</td><td rowspan=1 colspan=1>15.4%(2/13)</td><td rowspan=1 colspan=1>0.0%(0/13)</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>5.6%(43/762)</td><td rowspan=1 colspan=1>1.4%(11/769)</td><td rowspan=1 colspan=1>0.6%(5/775)</td><td rowspan=1 colspan=1>3.9%(30/772)</td><td rowspan=1 colspan=1>3.9%(30/775)</td></tr><tr><td rowspan=7 colspan=1>Combined</td><td rowspan=1 colspan=1>ALB</td><td rowspan=1 colspan=1>3.3%(2/60)</td><td rowspan=1 colspan=1>0.0%(0/60)</td><td rowspan=1 colspan=1>0.0%(0/60)</td><td rowspan=1 colspan=1>0.0%(0/60)</td><td rowspan=1 colspan=1>1.7%(1/60)</td></tr><tr><td rowspan=1 colspan=1>CAL</td><td rowspan=1 colspan=1>4.7%(27/578)</td><td rowspan=1 colspan=1>1.0%(6/580)</td><td rowspan=1 colspan=1>0.7%(4/585)</td><td rowspan=1 colspan=1>3.6%(21/584)</td><td rowspan=1 colspan=1>4.8%(28/585)</td></tr><tr><td rowspan=1 colspan=1>CIN</td><td rowspan=1 colspan=1>13.6%(53/389)</td><td rowspan=1 colspan=1>5.1%(20/395)</td><td rowspan=1 colspan=1>3.0%(12/397)</td><td rowspan=1 colspan=1>10..0%(39/390)</td><td rowspan=1 colspan=1>5.5%(22/397)</td></tr><tr><td rowspan=1 colspan=1>JHU</td><td rowspan=1 colspan=1>8.7%(22/252)</td><td rowspan=1 colspan=1>2.8%(7/254)</td><td rowspan=1 colspan=1>0.4%(1/254)</td><td rowspan=1 colspan=1>0.0%(0/252)</td><td rowspan=1 colspan=1>0.8%(2/254)</td></tr><tr><td rowspan=1 colspan=1>LUR</td><td rowspan=1 colspan=1>6.9%(7/101)</td><td rowspan=1 colspan=1>3.8%(4/104)</td><td rowspan=1 colspan=1>1.0%(1/104)</td><td rowspan=1 colspan=1>6.8%(7/103)</td><td rowspan=1 colspan=1>1.9%(2/104)</td></tr><tr><td rowspan=1 colspan=1>POR</td><td rowspan=1 colspan=1>4.3%(16/371)</td><td rowspan=1 colspan=1>1.6%(6/370)</td><td rowspan=1 colspan=1>0.8%(3/373)</td><td rowspan=1 colspan=1>3.5%(13/371)</td><td rowspan=1 colspan=1>1.9%(7/373)</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>7.3%(127/1751)</td><td rowspan=1 colspan=1>2.4%(43/1763)</td><td rowspan=1 colspan=1>1.2%(21/1773)</td><td rowspan=1 colspan=1>4.5%(80/1760)</td><td rowspan=1 colspan=1>3.5%(62/1773)</td></tr></table>

Table 35: Expected Values based on BD MAX Enteric Viral Panel Assay   

<table><tr><td rowspan=1 colspan=1>SpecimenType</td><td rowspan=1 colspan=1>Site</td><td rowspan=1 colspan=1>Norovirus</td><td rowspan=1 colspan=1>Rotavirus</td><td rowspan=1 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>Sapovirus</td><td rowspan=1 colspan=1>Astrovirus</td></tr><tr><td rowspan=5 colspan=1>Cary-BlairPreserved</td><td rowspan=1 colspan=1>ALB</td><td rowspan=1 colspan=1>7.7%(4/52)</td><td rowspan=1 colspan=1>0.0%(0/52)</td><td rowspan=1 colspan=1>0.0 (0/52)</td><td rowspan=1 colspan=1>1.9%(1/52)</td><td rowspan=1 colspan=1>1.9%(1/52)</td></tr><tr><td rowspan=1 colspan=1>CIN</td><td rowspan=1 colspan=1>15.0%(59/394)</td><td rowspan=1 colspan=1>6.9%(27/394)</td><td rowspan=1 colspan=1>2.8%(11/394)</td><td rowspan=1 colspan=1>12.2%(48/394)</td><td rowspan=1 colspan=1>5.1%(20/394)</td></tr><tr><td rowspan=1 colspan=1>JHU</td><td rowspan=1 colspan=1>9.8%(13/132)</td><td rowspan=1 colspan=1>4.5%(6/132)</td><td rowspan=1 colspan=1>0.8%(1/132)</td><td rowspan=1 colspan=1>1.5%(2/132)</td><td rowspan=1 colspan=1>0.8%(1/132)</td></tr><tr><td rowspan=1 colspan=1>POR</td><td rowspan=1 colspan=1>3.9%(14/359)</td><td rowspan=1 colspan=1>1.9%(7/359)</td><td rowspan=1 colspan=1>0.8%(3/59)</td><td rowspan=1 colspan=1>2.5%(9/360)</td><td rowspan=1 colspan=1>2.2%(8/360)</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>9.6%(90/937)</td><td rowspan=1 colspan=1>4.3%(40/937)</td><td rowspan=1 colspan=1>1.6%(15/937)</td><td rowspan=1 colspan=1>6.4%(60/938)</td><td rowspan=1 colspan=1>3.2%(30/938)</td></tr><tr><td rowspan=7 colspan=1>Unpreserved</td><td rowspan=1 colspan=1>CAL</td><td rowspan=1 colspan=1>5.4%(31/574)</td><td rowspan=1 colspan=1>1.0%(6/574)</td><td rowspan=1 colspan=1>0.7%(4/574)</td><td rowspan=1 colspan=1>3.0%(17/575)</td><td rowspan=1 colspan=1>4.9%(28/575)</td></tr><tr><td rowspan=1 colspan=1>CIN</td><td rowspan=1 colspan=1>0.0%(0/3)</td><td rowspan=1 colspan=1>33.3%(1/3)</td><td rowspan=1 colspan=1>0.0%(0/3)</td><td rowspan=1 colspan=1>0.0%(0/3)</td><td rowspan=1 colspan=1>0.0%(0/3)</td></tr><tr><td rowspan=1 colspan=1>JHU</td><td rowspan=1 colspan=1>13.8%(9/65)</td><td rowspan=1 colspan=1>0.0%(0/65)</td><td rowspan=1 colspan=1>0.0%(0/65)</td><td rowspan=1 colspan=1>0.0%(0/63)</td><td rowspan=1 colspan=1>1.6%(1/63)</td></tr><tr><td rowspan=1 colspan=1>LUR</td><td rowspan=1 colspan=1>10.1%(10/99)</td><td rowspan=1 colspan=1>5.1%(5/99)</td><td rowspan=1 colspan=1>1.0%(1/99)</td><td rowspan=1 colspan=1>7.9%(8/101)</td><td rowspan=1 colspan=1>1.0%(1/101)</td></tr><tr><td rowspan=1 colspan=1>POR</td><td rowspan=1 colspan=1>23.1%(3/13)</td><td rowspan=1 colspan=1>7.7%(1/13)</td><td rowspan=1 colspan=1>0.0%(0/13)</td><td rowspan=1 colspan=1>15.4%(2/13)</td><td rowspan=1 colspan=1>0.0%(0/13)</td></tr><tr><td rowspan=2 colspan=1>Total</td><td rowspan=2 colspan=1>7.0%(53/754)</td><td rowspan=2 colspan=1>1.7%(13/754)</td><td rowspan=2 colspan=1>0.7%(5/754)</td><td rowspan=1 colspan=1>3.6%</td><td rowspan=2 colspan=1>4.0%(30/755)</td></tr><tr><td rowspan=1 colspan=1>(27/755)</td></tr><tr><td rowspan=7 colspan=1>Combined</td><td rowspan=1 colspan=1>ALB</td><td rowspan=1 colspan=1>7.7%(4/52)</td><td rowspan=1 colspan=1>0.0%(0/52)</td><td rowspan=1 colspan=1>0.0%(0/52)</td><td rowspan=1 colspan=1>1.9%(1/52)</td><td rowspan=1 colspan=1>1.9%(1/52)</td></tr><tr><td rowspan=1 colspan=1>CAL</td><td rowspan=1 colspan=1>5.4%(31/574)</td><td rowspan=1 colspan=1>1.0%(6/574)</td><td rowspan=1 colspan=1>0.7%(4/574)</td><td rowspan=1 colspan=1>3.0%(17/575)</td><td rowspan=1 colspan=1>4.9%(28/575)</td></tr><tr><td rowspan=1 colspan=1>CIN</td><td rowspan=1 colspan=1>14.9%(59/397)</td><td rowspan=1 colspan=1>7.1%(28/397)</td><td rowspan=1 colspan=1>2.8%(11/397)</td><td rowspan=1 colspan=1>12.1%(48/397)</td><td rowspan=1 colspan=1>5.0%(20/397)</td></tr><tr><td rowspan=1 colspan=1>JHU</td><td rowspan=1 colspan=1>11.2%(22/197)</td><td rowspan=1 colspan=1>3.0%(6/197)</td><td rowspan=1 colspan=1>0.5%(1/197)</td><td rowspan=1 colspan=1>1.0%(2/195)</td><td rowspan=1 colspan=1>1.0%(2/195)</td></tr><tr><td rowspan=1 colspan=1>LUR</td><td rowspan=1 colspan=1>10.1%(10/99)</td><td rowspan=1 colspan=1>5.1%(5/99)</td><td rowspan=1 colspan=1>1.0%(1/99)</td><td rowspan=1 colspan=1>7.9%(8/101)</td><td rowspan=1 colspan=1>1.0%(1/101)</td></tr><tr><td rowspan=1 colspan=1>POR</td><td rowspan=1 colspan=1>4.6%(17/372)</td><td rowspan=1 colspan=1>2.2%(8/372)</td><td rowspan=1 colspan=1>0.8%(3/372)</td><td rowspan=1 colspan=1>2.9%(11/373)</td><td rowspan=1 colspan=1>2.1%(8/373)</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>8.5%(143/1691)</td><td rowspan=1 colspan=1>3.1%(53/1691)</td><td rowspan=1 colspan=1>1.2%(20/1691)</td><td rowspan=1 colspan=1>5.1%(87/1693)</td><td rowspan=1 colspan=1>3.5%(60/1693)</td></tr></table>

# N. Instrument Name:

BD MAX System

O. System Descriptions:

1. Modes of Operation:

Does the applicant’s device contain the ability to transmit data to a computer, webserver, or mobile device?

Yes ____X____ or No

Does the applicant’s device transmit data to a computer, webserver, or mobile device using wireless transmission?

Yes or No _ _X

2. Software:

FDA has reviewed applicant’s Hazard Analysis and software development processes for this line of product types:

Yes ___ _X____ or No a. Level of Concern: Moderate b. Device Hazard Analysis: Acceptable

The risks associated with the BD MAX Platform and Software were reviewed as part of previous premarket notifications—

· K111860 (BD MAX Platform and Software with BD MAX GBS Assay) · DEN160001 (BD MAX Vaginal Panel and Software Updates) · K170308 (BD MAX Extended Enteric Bacterial Panel and Software Updates)

The device hazard analysis submitted for the Enteric Viral Panel assay contains redacted hazards and Risk Control Measures that, according to BD, are not associated with the BD MAX system software for the ease of review by FDA.

According to the risk region chart in the Hazard Analysis Report, BD identified 93, 14, and 0 hazards in green, yellow, and red risk regions, respectively. After accounting for appropriate mitigatins, where possible, all identified hazards are acceptable. Examples of appropriate mitigations include software changes implemented as a Risk Control Measures, proper device labeling, training of end users, and the results of clinical and analytical validation studies.

c. Architecture Design Chart: Acceptable

The architecture design chart for the BD MAX System and software was submitted as part of the original instrument clearance (K111860).

d. Software Description: Acceptable

Each new BD MAX assay is developed with an Assay Definition File (ADF) containing the assay workflow parameters to run and report results for the assay on the BD MAX System. The ADF controls sample preparation cartridge loading, barcodes, result scripts, BANCS parameters and Master Mix parameters. The BD MAX Enteric Viral Panel Assay ADF (BD MAX ENT VIR 49 Version: 1) was released on Assay Release Key V2.38. The Assay Release Key is a compilation of released BD MAX assay ADFs.

There are no unresolved anomalies associated with the ADF developed for use with the BD MAX Enteric Viral Panel.

The BD MAX System software automatically interprets test results. Results are reported for each of the analytes and for the Sample Processing Control. In the absence of positive sample processing control amplification, positive analyte results are maintained but any negative result is labeled unresolved “UNR”. Results for each analyte are determined based on pre-extablished cut-off thresholds for both Ct.Score (amplification threshold cycle) and Cycle End point (end point fluorescence of each well).

e. Software Requirements Specifications: Acceptable

Extensive documentation was provided in the original K111860 submission with assay-specific updates to software and ADFs in DEN160001, K170308, and the current submission. No fundamental changes that affect safety or effectiveness are noted.

f. Software Development Environment: Acceptable

Extensive documentation was provided in the original K111860 submission with assay-specific updates to software and ADFs in DEN160001 and K170308. No fundamental changes that affect safety or effectiveness are noted.

g. Traceability Analysis: Acceptable

Both the original K111860 submission and subsequent submissions contain adequate documentation that links together design requirements and specifications to validation testing.

h. Verification and Validation $( \mathrm { V n V } )$ Testing: Acceptable

BD provided a Software $\mathrm { V n V }$ plan that details testing and implementation procedures for the release of V4.72A as well as the Assay Release Key V2.39A Software which contains updated BD MAX Vaginal Panel (MVP) Assay Definition Files. According to BD, the software version update to 4.72A is primarily to improve robustness of the MVP assay.

The sponsor states that an equivalence comparison for the BD MAX Enteric Viral Panel ADF will be performed to support the ADF parameters between the versions used in the performance studies and the final commercial release version.

Software validation includes integration testing of new features, regression testing to ensure retention of system performance and direct testing of bug fixes. Additional testing includes stress testing and resolution of unexpected results. All completeion criteria have been met for validation testing of the BD MAX Enteric Viral Panel System and Assay software and all expected results have been addressed and documented.

j. Revision Level History: Acceptable

BD MAX Enteric Viral Panel analytical and clinical studies were completed using the functionality of BD MAX System software version 4.60A or 4.61D and with BD MAX Enteric Viral Panel Assay Definition File “BD MAX Enteric Viral Panel IUO v1”. The BD MAX Enteric Viral Panel assay kit will be distributed for use with the BD MAX System running System Software Version 4.60A (or greater). Additional enhancements, new features, and bug fixes are included in the upgrade to 4.72A.

A change to sample preparation script for the cleared BD MAX Vaginal Panel is noted in release version 4.72A. The Software change impact review provided by the sponsor suggests that this is to optimize workflow for interfering substances for version 2 of the MVP assay and no new hazards are introduced.

None of the changes incorporated into the latest software version would impact the BD MAX Enteric Viral Panel assay. The BD MAX ENT VIR 49 ADF was released on February 15, 2018 for use with the CE marked version of the BD MAX Enteric Viral Panel (CE Mark February 2018).

k. Unresolved Anomalies and Software Defects: Acceptable

In the Software Test Summary document, BD confirms passing results for all four test protocols (SP_MVP Scripting, Stress Test, IVD Regression Test, Windows Regression Test) using version 4.72A. Additionally, BD provides a list of all open Software Change Requests for this version of software release.

While no unresolved software anomalies remain to the assay-specific ADF component of the software, there is a total of 77 unresolved anomalies for the general instrument software -- 21 Serious, 41 Medium, and 15 low – that primarily affet user convenience and do not impact assay safety and effectiveness.

l. Off the Shelf Software (OTS)/Software of Unknown Pedigree (SOUP): Acceptable

The following third-party software components are used in the BD MAX instrument. Microsoft Windows 7, Microsoft Windows IoT10 64 bits, Microsoft SQL Server 2012 Express, Microsoft .NET framework, Microsoft Visual Studio 2010,

CylancePROTECT.

m. EMC Testing and Standards: Acceptable

The BD MAX System is an already cleared device and requires no further EMC testing.

3. Specimen Identification: Barcode

4. Specimen Sampling and Handling:

Collected specimens, either unpreserved stool or stool stored in $1 5 ~ \mathrm { m L }$ Cary-Blair transport media, should be kept between $2 \ { } ^ { \circ } { \bf C }$ and $2 5 ~ ^ { \circ } \mathrm { C }$ during transport. Protect against exposure to excessive heat.

Specimens can be stored for up to 120 hours (5 days) at $2 { - } 8 ~ ^ { \circ } \mathrm { C }$ or for up to 48 hours at $2 { - } 2 5 \ ^ { \circ } \mathrm { C }$ before testing.

5. Calibration:

The system is calibrated on-site as part of the installation procedure, as well as during preventive maintenance that is offered with a service contract.

6. Quality Control:

The Sample Processing Control (SPC) is contained in the extraction tube and serves as verification that the extraction, amplification, and detection processes occurred appropriately.

Users may also add external controls in order to comply with applicable regulations. No external control materials are provided by the manufacturer and nor are they used by the BD MAX system for sample result interpretation.

As an external Negative control, the sponsor recommends commercially available material or a previously characterized negative specimen. As an external positive control, commercially available viral strains are purchaseable from Zeptometrix.

P. Other Supportive Instrument Performance Characteristics Data Not Covered In The “Performance Characteristics” Section above:

Not applicable.

# Q. Proposed Labeling:

The labeling supports the finding of substantial equivalence for this device.

# R. Conclusion:

The submitted information in this premarket notification is complete and supports a substantial equivalence decision.