# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATION DECISION SUMMARY ASSAY ONLY TEMPLATE

A. 510(k) Number: K181665   
B. Purpose for Submission: Addition of the BD Phoenix CPO detect test to the BD Phoenix Gram negative ID/AST and AST only Phoenix panels.   
C. Measurand: Carbapenemase production   
D. Type of Test: Growth-based detection and algorithm using predetermined thresholds   
E. Applicant: Becton, Dickinson and Company   
F. Proprietary and Established Names: BD Phoenix Automated Microbiology System – BD Phoenix CPO detect - GN

# G. Regulatory Information:

Regulation section: 21 CFR 866.1645 Short-Term Antimicrobial Susceptibility Test System

2. Classification: Class II

3. Product code: LON System, Test, Automated, Antimicrobial Susceptibility, Short Incubation

4. Panel:

83 Microbiology

# H. Intended Use:

1. Intended use(s):

The BD Phoenix Automated Microbiology System is intended for the in vitro rapid identification (ID) and quantitative determination of antimicrobial susceptibility by minimal inhibitory concentration (MIC) of Gram Negative aerobic and facultative anaerobic bacteria belonging to the family Enterobacteriaceae and nonEnterobacteriaceae.

2. Indication(s) for use:

BD Phoenix CPO detect is a qualitative confirmatory test that uses a growth-based algorithm intended to phenotypically detect carbapenemase-production in Enterobacteriaceae, Pseudomonas aeruginosa, and Acinetobacter baumannii on Gramnegative ID/AST or AST only Phoenix panels. The test also provides the Ambler classification (Class A, Class B and Class D) of the carbapenemase produced. One of three test configurations are available per panel for carbapenemase detection with/without Ambler classification for the target organism groups. BD Phoenix CPO detect does not report multiple classes of carbapenemases from a single isolate.

3. Special conditions for use statement(s):

For Prescription Use only

For in vitro diagnostic use

Due to the small number of tested isolates identified as harboring carbapenemases from multiple Ambler classes $( \mathrm { n } { = } 7 )$ , there is limited information on the performance of the BD Phoenix CPO detect with organisms co-producing different classes of carbapenemases.   
· Carbapenemase positive isolates that are carbapenem susceptible (Enterobacteriaceae, Pseudomonas aeruginosa, or Acinetobacter baumannii) are rare, as encountered in the clinical study. Additional confirmatory testing should be conducted before reporting carbapenemase activity in a susceptible isolate.   
· Well-established phenotypic carbapenemase tests (e.g. CarbaNP and mCIM) do not include testing of Acinetobacter baumannii. End-users should consider other methods for further characterization of Acinetobacter baumannii if necessary.   
· In silico analysis of carbapenemase gene variants was performed in the validation of the PCR multiplex method for the reference method. Performance with isolates harboring specific carbapenemase gene variants expressing enzymes from various Ambler classes by BD Phoenix CPO detect was not determined.

4. Special instrument requirements: BD Phoenix Automated Microbiology System

# I. Device Description:

The Phoenix panel is a sealed and self-inoculating molded polystyrene tray, with 136 microwells containing dried reagents. The combination panel includes an ID side with dried substrates for bacterial identification and an AST side with varying concentrations of antimicrobial agents, growth, and fluorescent controls at appropriate well locations. The Phoenix system utilizes an optimized colorimetric redox indicator for AST, and a variety of colorimetric and fluorometric indicators for ID. Phoenix panels are also available in an "ID Only" format where only the 51-well side is filled with substrates and the 85-well side is empty. Phoenix panels are also available in an "AST Only" format where both the 51-well and 85-well sides are filled with antibiotics.

Microorganisms to be tested are isolated from pure culture and preliminarily identified as a Gram-negative bacillus, Gram-positive coccus or Gram-positive bacillus. For each bacterial isolate, the inoculum suspension is prepared in the Phoenix ID broth and vortexed to mix. The appropriate amount of inoculated ID Broth is transferred to the prepared AST Broth tube, so that the resulting AST inoculum contains approximately $5 \mathrm { ~ x ~ } 1 0 ^ { 5 } \mathrm { C F U / m l } .$ . The Phoenix ID broth is poured into the inoculation port of the 51-well side (left) of the Phoenix Combo panel (for ID substrates). The Phoenix AST Broth is poured into the inoculation port of the 85-well side (right) of the Phoenix Combo panel (for antimicrobials). The panel barcode is scanned to log-in the panel configuration and the panel is loaded into the Phoenix instrument. The instrument houses the inoculated panels where they are continuously incubated at $3 5 ^ { \circ } \mathrm { C }$ . The Phoenix instrument reads and records the results of the biochemical substrates and antimicrobial agents contained in the panel and interprets the reactions to give an ID of the isolate, minimal inhibitory concentration (MIC) values, and category interpretations of S, I, R or $_ \mathrm { N }$ (susceptible, intermediate, resistant or not susceptible).

BD Phoenix CPO detect is a qualitative growth-based test intended to phenotypically detect carbapenemase-production in Enterobacteriaceae, Pseudomonas aeruginosa, and Acinetobacter baumannii on the BD Phoenix Automated Microbiology System. The test also provides the Ambler classification (Class A, Class B and Class D) of the carbapenemase produced. CPO detect uses the principle of carbapenem resistance to detect the presence of the carbapenemase, and it employs the principles of Ambler-class specific beta-lactamase inhibition and Ambler-class specific antibiotic resistance to derive the Ambler class of the carbapenemase.

# J. Substantial Equivalence Information:

1. Predicate device name(s):

BD Phoenix Automated Microbiology System-Confirmatory ESBL test

2. Predicate 510(k) number(s): K033458

3. Comparison with predicate:

<table><tr><td rowspan=1 colspan=3>Similarities</td></tr><tr><td rowspan=1 colspan=1>Item</td><td rowspan=1 colspan=1>BD Phoenix CPO detect(K181665)</td><td rowspan=1 colspan=1>Phoenix ESBL test (predicate)(K033458)</td></tr><tr><td rowspan=1 colspan=1>Intended Use</td><td rowspan=1 colspan=1>The BD Phoenix AutomatedMicrobiology System is intended for thein vitro rapid identification (ID) andquantitative determination ofantimicrobial susceptibility by minimalinhibitory concentration (MIC) of GramNegative aerobic and facultativeanaerobic bacteria belonging to thefamily Enterobacteriaceae and non-Enterobacteriaceae.</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=1>Specimen</td><td rowspan=1 colspan=1>Isolated colonies from culture</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=1>Inoculum</td><td rowspan=1 colspan=1>Inoculum density of 0.5 McFarlandstandard</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=1>Incubation Time</td><td rowspan=1 colspan=1>&lt; 16 hours</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=1>Technology</td><td rowspan=1 colspan=1>Automated growth based</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=1>Panels</td><td rowspan=1 colspan=1>Panel with dried antibiotics and/orinhibitors</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=1>Instrumentation</td><td rowspan=1 colspan=1>BD Phoenix Automated MicrobiologySystem</td><td rowspan=1 colspan=1>Same</td></tr></table>

<table><tr><td colspan="3" rowspan="1">Differences</td></tr><tr><td colspan="1" rowspan="1">Item</td><td colspan="1" rowspan="1">BD Phoenix CPO detect(K181665)</td><td colspan="1" rowspan="1">Phoenix ESBL test (predicate)(K033458)</td></tr><tr><td colspan="1" rowspan="1">Indications for Use</td><td colspan="1" rowspan="1">BD Phoenix CPO detect is a qualitativeconfirmatory test that uses a growth-based algorithm intended tophenotypically detect carbapenemase-production in Enterobacteriaceae,Pseudomonas aeruginosa, andAcinetobacter baumannii on Gram-negative ID/AST or AST only Phoenixpanels. The test also provides theAmbler classification (Class A, Class Band Class D) of the carbapenemaseproduced. One of three testconfigurations are available per panelfor carbapenemase detectionwith/without Ambler classification forthe target organism groups. BDPhoenix CPO detect does not reportmultiple classes of carbapenemasesfrom a single isolate.</td><td colspan="1" rowspan="1">This submission is for the additionof the confirmatory ESBL Test toGram-negative ID/AST or AST BDPhoenix panels. The PhoenixConfirmatory ESBL Test is aconfirmatory test for the detectionof the organisms that produceextended spectrum β-lactamase(ESBL) in Escherichia coli,Klebsiella pneumoniae andKlebsiella oxytoca.</td></tr><tr><td colspan="1" rowspan="1">Test Results</td><td colspan="1" rowspan="1">Detection of carbapenemase-producingorganisms with/without thecategorization into one of the Amblerclasses</td><td colspan="1" rowspan="1">Detection of Extended Spectrum β-Lactamase (ESBL) producingorganisms</td></tr><tr><td colspan="1" rowspan="1">Reagents as part ofdetermination</td><td colspan="1" rowspan="1">Set concentrations of meropenem,doripenem, temocillin, and cloxacillinalone and in combination with variouschelators and beta-lactamase inhibitors</td><td colspan="1" rowspan="1">Set concentrations of Ceftriaxone,Cefotaxime, Ceftazidime plusscreening antibiotics used in adecision tree</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">used in a decision tree</td><td colspan="1" rowspan="1">EY</td></tr></table>

# K. Standard/Guidance Document Referenced (if applicable):

Clinical Laboratory Standards Institute: M100 “Performance Standards for Antimicrobial Susceptibility Testing.” 28th Edition (January 2018)

# L. Test Principle:

The BD Phoenix CPO detect test determines the growth or inhibition of growth with each antimicrobic agent or antimicrobic agent/beta-lactamase inhibitor combination in relation to the growth control well. The classification tree algorithms compare these relative growth rates to a series of thresholds in the decision trees. Based on the resultant profile, the system will produce a particular result. The CPO detect result does not depend on the MIC or SIR of any drug on the panel.

The test is offered to users as a choice of multiple configurations—9-well, 6-well, and 2-well (See Table 1 below). There are different classification tree algorithms for Enterobacteriaceae and non-fermenters. The Phoenix instrument calculates both of these results simultaneously and then reports the appropriate result based on the isolate identification. The wells for the CPO detect test are specific for the test and are not part of any drug MIC determination.

If the test is unable to classify the carbapenemase into an Ambler class, then it reports only the presence of a carbapenemase. When two different Ambler class carbapenemases are present, BD Phoenix CPO detect will not report multiple classes; e.g., it will not report both Class A and Class B. For bacteria that produce more than one class of carbapenemase, the Phoenix test is expected to give a result of positive with no classification.

Table 1. Interpretation of Results for the (3) Possible Configurations of the BD Phoenix CPO Detect Test   

<table><tr><td rowspan=1 colspan=1>Test Configuration</td><td rowspan=1 colspan=1>Organism Group</td><td rowspan=1 colspan=1>Possible Test Results</td></tr><tr><td rowspan=1 colspan=1>2-Well</td><td rowspan=1 colspan=1>Enterobacteriaceae,Pseudomonas aeruginosa, orAcinetobacter baumannii</td><td rowspan=1 colspan=1>Carbapenemase Producer</td></tr><tr><td rowspan=3 colspan=1>6-Well</td><td rowspan=2 colspan=1>Enterobacteriaceae</td><td rowspan=1 colspan=1>Carbapenemase Producer1</td></tr><tr><td rowspan=1 colspan=1>Carbapenemase Producer, Class A, B, or D</td></tr><tr><td rowspan=1 colspan=1>Pseudomonas aeruginosa andAcinetobacter baumannii</td><td rowspan=1 colspan=1>Carbapenemase Producer</td></tr><tr><td rowspan=2 colspan=1>9-Well</td><td rowspan=2 colspan=1>Enterobacteriaceae,Pseudomonas aeruginosa, orAcinetobacter baumannii</td><td rowspan=1 colspan=1>Carbapenemase Producer1</td></tr><tr><td rowspan=1 colspan=1>Carbapenemase Producer, Class A, B, or D</td></tr></table>

1Ambler class could not be determined; therefore, no Ambler class is reported.

# M. Performance Characteristics (if/when applicable):

1. Analytical performance:

a. Precision/Reproducibility:

Intra- and inter-site reproducibility of the BD Phoenix CPO detect test on the BD Phoenix System was evaluated at three sites using a panel of Gram-negative isolates. Thirteen gram-negative organisms were tested, where each site tested the isolates in triplicate on three different days using Gram Negative Phoenix panels containing CPO detect and associated reagents. There were two QC isolates included on the reproducibility panel. Results of the study demonstrated that the BD Phoenix CPO detect test showed acceptable reproducibility at $\ge 9 5 \%$ with the expected results for Gram-negative organisms prepared by both manual and Phoenix AP inoculations. Table 2 below presents the reproducibility of the organism panels with and without the QC organism data included in the performance calculations.

Table 2. BD Phoenix CPO Detect Reproducibility Summary (Manual and Phoenix AP Inoculation)   

<table><tr><td rowspan=3 colspan=1>Organism</td><td rowspan=1 colspan=2>Expected Result</td><td rowspan=1 colspan=4>BD Phoenix CPO Detect Result</td></tr><tr><td rowspan=2 colspan=1>CPODetection(P/N)}$</td><td rowspan=2 colspan=1>AmblerClass(A, B, D)2</td><td rowspan=1 colspan=2>Manual</td><td rowspan=1 colspan=2>BD Phoenix AP</td></tr><tr><td rowspan=1 colspan=1>CPO Detection(Observed/Expected)</td><td rowspan=1 colspan=1>Ambler Class(Observed/Expected)</td><td rowspan=1 colspan=1>CPO Detection(Observed/Expected)</td><td rowspan=1 colspan=1>Ambler Class(Observed/Expected)</td></tr><tr><td rowspan=1 colspan=1>E. coli</td><td rowspan=1 colspan=1>P</td><td rowspan=1 colspan=1>A</td><td rowspan=1 colspan=1>27/27(100%)</td><td rowspan=1 colspan=1>27/27(100%)</td><td rowspan=1 colspan=1>27/27(100%)</td><td rowspan=1 colspan=1>27/27(100%)</td></tr><tr><td rowspan=1 colspan=1>K. pneumo. spp.pneumo.</td><td rowspan=1 colspan=1>P</td><td rowspan=1 colspan=1>B</td><td rowspan=1 colspan=1>27/27(100%)</td><td rowspan=1 colspan=1>27/27(100%)</td><td rowspan=1 colspan=1>27/27(100%)</td><td rowspan=1 colspan=1>27/27(100%)</td></tr><tr><td rowspan=1 colspan=1>E. coli ATCC259223</td><td rowspan=1 colspan=1>N4</td><td rowspan=1 colspan=1>--</td><td rowspan=1 colspan=1>27/27(100%)</td><td rowspan=1 colspan=1>27/27(100%)</td><td rowspan=1 colspan=1>27/27(100%)</td><td rowspan=1 colspan=1>27/27(100%)</td></tr><tr><td rowspan=1 colspan=1>K. pneumo. spp.pneumo.</td><td rowspan=1 colspan=1>N4</td><td rowspan=1 colspan=1>--</td><td rowspan=1 colspan=1>27/27(100%)</td><td rowspan=1 colspan=1>27/27(100%)</td><td rowspan=1 colspan=1>27/27(100%)</td><td rowspan=1 colspan=1>27/27(100%)</td></tr><tr><td rowspan=1 colspan=1>K. pneumo. spp.pneumo.</td><td rowspan=1 colspan=1>P</td><td rowspan=1 colspan=1>D</td><td rowspan=1 colspan=1>27/27(100%)</td><td rowspan=1 colspan=1>25/27(92.6%)</td><td rowspan=1 colspan=1>27/27(100%)</td><td rowspan=1 colspan=1>24/27(88.9%)</td></tr><tr><td rowspan=1 colspan=1>Klebsiellaaerogenes</td><td rowspan=1 colspan=1>N4</td><td rowspan=1 colspan=1>--</td><td rowspan=1 colspan=1>27/27(100%)</td><td rowspan=1 colspan=1>27/27(100%)</td><td rowspan=1 colspan=1>26/27(96.3%)</td><td rowspan=1 colspan=1>26/27(96.3%)</td></tr><tr><td rowspan=1 colspan=1>K. pneumo. spp.pneumo. ATCCBAAA17053</td><td rowspan=1 colspan=1>P</td><td rowspan=1 colspan=1>A</td><td rowspan=1 colspan=1>27/27(100%)</td><td rowspan=1 colspan=1>27/27(100%)</td><td rowspan=1 colspan=1>27/27(100%)</td><td rowspan=1 colspan=1>27/27(100%)</td></tr><tr><td rowspan=1 colspan=1>E. coli</td><td rowspan=1 colspan=1>P</td><td rowspan=1 colspan=1>B</td><td rowspan=1 colspan=1>27/27(100%)</td><td rowspan=1 colspan=1>27/27(100%)</td><td rowspan=1 colspan=1>27/27(100%)</td><td rowspan=1 colspan=1>27/27(100%)</td></tr><tr><td rowspan=1 colspan=1>A. baumannii</td><td rowspan=1 colspan=1>P</td><td rowspan=1 colspan=1>B</td><td rowspan=1 colspan=1>27/27(100%)</td><td rowspan=1 colspan=1>27/27(100%)</td><td rowspan=1 colspan=1>27/27(100%)</td><td rowspan=1 colspan=1>27/27(100%)</td></tr><tr><td rowspan=1 colspan=1>P. aeruginosa</td><td rowspan=1 colspan=1>P</td><td rowspan=1 colspan=1>A</td><td rowspan=1 colspan=1>27/27(100%)</td><td rowspan=1 colspan=1>26/27(96.3%)</td><td rowspan=1 colspan=1>27/27(100%)</td><td rowspan=1 colspan=1>27/27(100%)</td></tr><tr><td rowspan=1 colspan=1>P. aeruginosaATCC 27853</td><td rowspan=1 colspan=1>N4</td><td rowspan=1 colspan=1>--</td><td rowspan=1 colspan=1>26/27(96.3%)</td><td rowspan=1 colspan=1>26/27(96.3%)</td><td rowspan=1 colspan=1>27/27(100%)</td><td rowspan=1 colspan=1>27/27(100%)</td></tr><tr><td rowspan=1 colspan=1>A. baumannii</td><td rowspan=1 colspan=1>P</td><td rowspan=1 colspan=1>D</td><td rowspan=1 colspan=1>27/27(100%)</td><td rowspan=1 colspan=1>27/27(100%)</td><td rowspan=1 colspan=1>27/27(100%)</td><td rowspan=1 colspan=1>26/27(96.3%)</td></tr><tr><td rowspan=1 colspan=1>P. aeruginosa</td><td rowspan=1 colspan=1>P</td><td rowspan=1 colspan=1>B</td><td rowspan=1 colspan=1>27/27(100%)</td><td rowspan=1 colspan=1>27/27(100%)</td><td rowspan=1 colspan=1>27/27(100%)</td><td rowspan=1 colspan=1>27/27(100%)</td></tr><tr><td rowspan=1 colspan=1>All (with QC isolates)</td><td rowspan=2 colspan=2></td><td rowspan=1 colspan=1>350/351(99.7%)</td><td rowspan=1 colspan=1>347/351(98.9%)</td><td rowspan=1 colspan=1>350/351(99.7%)</td><td rowspan=1 colspan=1>346/351(98.6%)</td></tr><tr><td rowspan=1 colspan=1>w/o QC Isolates</td><td rowspan=1 colspan=1>296/297(99.7%)</td><td rowspan=1 colspan=1>293/297(98.7%)</td><td rowspan=1 colspan=1>296/297(99.7%)</td><td rowspan=1 colspan=1>292/297(98.3%)</td></tr></table>

$^ { I } P$ : carbapenemase detected; N: carbapenemase not detected. The reproducibility set included nine carbapenemase producers. 2The nine carbapenemase producers covered three Class A, four Class $B$ , and two Class $D$ carbapenemases. $( - \cdot )$ represents no Ambler class reported for this organism. $^ { 3 } { Q C }$ isolates were E. coli ATCC 25922 and K. pneumo. spp. pneumo. ATCC BAA-1705. 4Numbers in table for organisms negative for carbapenemase show observed negatives/expected negatives.

b. Linearity/assay reportable range: Not Applicable

c. Traceability, Stability, Expected values (controls, calibrators, or methods):

Quality Control (QC) strains recommended for carbapenemase testing were evaluated using both the Manual and Phoenix AP Inoculation procedures. The BD Phoenix CPO detect test gave the expected results $> 9 5 \%$ of the time, demonstrating that the system can produce the expected QC results with this test (Table 3).

Table 3. QC Performance Summary   

<table><tr><td rowspan=3 colspan=1>Organism</td><td rowspan=1 colspan=2>Expected Result</td><td rowspan=1 colspan=4>BD Phoenix CPO Detect Result</td></tr><tr><td rowspan=2 colspan=1>CPODetection(P/N) 1</td><td rowspan=2 colspan=1>AmblerClass(A, B, D)2</td><td rowspan=1 colspan=2>Manual</td><td rowspan=1 colspan=2>BD Phoenix AP</td></tr><tr><td rowspan=1 colspan=1>CPO Detection(Observed/Expected)</td><td rowspan=1 colspan=1>Ambler Class(Observed/Expected)</td><td rowspan=1 colspan=1>CPO Detection(Observed/Expected)</td><td rowspan=1 colspan=1>Ambler Class(Observed/Expected)</td></tr><tr><td rowspan=1 colspan=1>E. coli ATCC25922</td><td rowspan=1 colspan=1>$№^3}$</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>118/118</td><td rowspan=1 colspan=1>118/118</td><td rowspan=1 colspan=1>71/71</td><td rowspan=1 colspan=1>71/71</td></tr><tr><td rowspan=1 colspan=1>K.pneumoniaeATCC BAA-1705</td><td rowspan=1 colspan=1>P</td><td rowspan=1 colspan=1>A</td><td rowspan=1 colspan=1>117/117</td><td rowspan=1 colspan=1>115/117</td><td rowspan=1 colspan=1>71/71</td><td rowspan=1 colspan=1>71/71</td></tr><tr><td rowspan=1 colspan=1>All</td><td rowspan=1 colspan=2></td><td rowspan=1 colspan=1>100%(235/235)</td><td rowspan=1 colspan=1>99.1%(233/235)</td><td rowspan=1 colspan=1>100%(142/142)</td><td rowspan=1 colspan=1>100%(142/142)</td></tr></table>

$^ { I } P$ : carbapenemase detected; $N .$ : carbapenemase not detected 2Ambler Class $A$ , $B$ , or $D$ expected. $( - \cdot )$ represents no Ambler Class reported for this organism. 3Numbers in table for organism negative for carbapenemase show observed negatives/expected negatives.

d. Detection limit: Not Applicable e. Analytical specificity: Not Applicable

f. Assay cut-off: Not Applicable

2. Comparison studies:

a. Method comparison with predicate device: Not Applicable

b. Matrix comparison: Not Applicable

# a. Clinical Sensitivity:

Clinical fresh and stock isolates were tested across three sites to demonstrate the performance of the BD Phoenix CPO detect test with multiple lots of BD Phoenix Gram Negative panels. Challenge isolates obtained from domestic and international sources were also tested at these sites. All studies were conducted with the 9-well configuration of the BD Phoenix CPO detect test. For the challenge isolates, Phoenix System results with the CPO detect test were compared to the expected results. Phoenix System results for clinical isolates were compared to the results obtained from the Composite Reference Method, which was based on results from three different and independent tests. The tests included the following: 1) Phenotypic testing to detect the presence of carbapenemase expression according to the CLSIrecommended modified Carbapenem Inactivation Method (mCIM), 2) carbapenem MIC results obtained by the BD Phoenix cleared system and 3) Molecular (multiplex PCR amplification) testing to assign the Ambler classification of the enzyme based on detection of gene or genes associated with each class. All strains were tested using the mCIM assay and on the BD Phoenix panel. PCR testing was performed as appropriate based on the reference decision flowchart. Under certain conditions, and to lessen the burden of extensive PCR testing, the MIC screen was applied for each organism group (Table 4) and used to determine whether an isolate would proceed to PCR testing. The possible outcomes for the Composite Reference Method are shown in Table 5.

Table 4. MIC Screen Cut-Offs Set for Each Organism Group   

<table><tr><td rowspan=1 colspan=1>Organism Group</td><td rowspan=1 colspan=1>Drug</td><td rowspan=1 colspan=1>MIC Screen Cut-Off(considered positive result)</td></tr><tr><td rowspan=1 colspan=1>Enterobacteriaceae(other than Proteus spp).</td><td rowspan=1 colspan=1>Meropenem</td><td rowspan=1 colspan=1>&gt;0.25μg/ml</td></tr><tr><td rowspan=1 colspan=1>Proteus spp.</td><td rowspan=1 colspan=1>Ertapenem</td><td rowspan=1 colspan=1>&gt;0.5μg/ml</td></tr><tr><td rowspan=1 colspan=1>Pseudomonas aeruginosa</td><td rowspan=1 colspan=1>Meropenem</td><td rowspan=1 colspan=1>&gt;4μg/ml</td></tr><tr><td rowspan=1 colspan=1>Acinetobacter baumannii</td><td rowspan=1 colspan=1>Imipenem</td><td rowspan=1 colspan=1>&gt;2μg/ml</td></tr></table>

Table 5. Possible Outcomes of Composite Reference Method   

<table><tr><td rowspan=1 colspan=1>mCIM Result</td><td rowspan=1 colspan=1>MIC Screen</td><td rowspan=1 colspan=1>PCR</td><td rowspan=1 colspan=1>Composite ReferenceMethod Result</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Positive with Class</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Positive no Class</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Positive with Class</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>Negative</td></tr><tr><td rowspan=1 colspan=1>Indeterminate</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative</td></tr><tr><td rowspan=1 colspan=1>Indeterminate</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>No Result</td></tr><tr><td rowspan=1 colspan=1>Indeterminate</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>No Result</td></tr><tr><td rowspan=1 colspan=1>Indeterminate</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Positive with Class</td></tr></table>

There were 1641 Gram-negative clinical and challenge isolates enrolled in the study. A total of 189 isolates were excluded for the following reasons:

· (30) related to isolate characterization (e.g. failed or incomplete challenge screen, mixed purity plates, organisms not in inclusion criteria)   
· (17) due to non-compliance with the Phoenix System procedure (incorrect amount of indicator, uninoculated panels, no growth)   
· (142) due to non-compliance with the reference system results (QC failure, protocol deviation, no reference/expected result)

Results from these 1452 compliant Gram-negative Phoenix and reference isolates were used to calculate the performance for BD Phoenix CPO detect test for carbapenemase detection. Of these 1452 isolates, 317 $( 2 1 . 8 \% )$ were challenge isolates and 1135 $( 7 8 . 2 \% )$ were clinical isolates. Clinical isolates were comprised of 416 fresh isolates ( $2 8 . 7 \%$ of total isolates tested) and 719 stock isolates $( 4 9 . 5 \%$ of the total isolates tested). A total of 155 isolates were used from the CDC/FDA AR Bank. The distribution of isolates according to the Reference Method and BD Phoenix CPO detect test result is displayed in Table 6. Table 7 and Table 8 show the ability of the BD Phoenix CPO detect test to identify carbapenemase-producing isolates for all organisms, and for each organism group, respectively. The classification performance of the BD Phoenix CPO detect test (Class A, Class B, or Class D) for all isolates $\scriptstyle ( \ n = 1 4 5 2 )$ is presented in Table 9.

Table 6. Distribution of Isolates for the Clinical Study Based on Test Results   

<table><tr><td rowspan=1 colspan=10>Combined (Clinical and Challenge)</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=7>Reference Method Results</td><td rowspan=2 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>A1only</td><td rowspan=1 colspan=1>Multiple,A and B2</td><td rowspan=1 colspan=1>Bonly</td><td rowspan=1 colspan=1>Multiple,B and D2</td><td rowspan=1 colspan=1>Donly</td><td rowspan=1 colspan=1>CPO,unclass3</td><td rowspan=1 colspan=1>NoCPO4</td></tr><tr><td rowspan=5 colspan=1>BDPhoenixCPODetectResults</td><td rowspan=1 colspan=1>CPO, A5</td><td rowspan=1 colspan=1>122</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>137</td></tr><tr><td rowspan=1 colspan=1>CPO, B</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>126</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>145</td></tr><tr><td rowspan=1 colspan=1>CPO, D</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>134</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>152</td></tr><tr><td rowspan=1 colspan=1>CPO,unclass.6</td><td rowspan=1 colspan=1>18</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>27</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>76</td></tr><tr><td rowspan=1 colspan=1>No CPO7</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>932</td><td rowspan=1 colspan=1>942</td></tr><tr><td rowspan=1 colspan=2>Total</td><td rowspan=1 colspan=1>146</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>161</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>151</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>966</td><td rowspan=1 colspan=1>1452</td></tr></table>

1Carbapenemase-positive with Ambler Class A gene identified by Reference Method 2Carbapenemase-positive with multiple Ambler Class genes identified by Reference Method 3Carbapenemase-positive, but no Ambler Class identified by Reference Method 4No carbapenemase detected by Reference Method 5Carbapenemase-positive with Ambler Class detected (Class A) by the BD Phoenix CPO detect test 6Carbepenemase-positve, but no Ambler Class identified by the BD Phoenix CPO detect test 7No carbapenemase detected by the BD Phoenix CPO detect test

Table 7. BD Phoenix CPO Detect Test Performance [Carbapenemase Detection] vs the Composite Reference Method.   

<table><tr><td rowspan=1 colspan=1>Isolate Type</td><td rowspan=1 colspan=1>Total #Isolates</td><td rowspan=1 colspan=1>TP</td><td rowspan=1 colspan=1>FP</td><td rowspan=1 colspan=1>TN</td><td rowspan=1 colspan=1>FN</td><td rowspan=1 colspan=1>PPA[95% CI]</td><td rowspan=1 colspan=1>NPA[95% CI]</td></tr><tr><td rowspan=1 colspan=1>Clinical</td><td rowspan=1 colspan=1>1135</td><td rowspan=1 colspan=1>245</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>864</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>97.6%(245/251)[94.9%; 98.9%]</td><td rowspan=1 colspan=1>97.7%(864/884)[96.5%; 98.5%]</td></tr><tr><td rowspan=1 colspan=1>Challenge1,2</td><td rowspan=1 colspan=1>317</td><td rowspan=1 colspan=1>231</td><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>68</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>98.3%(231/235)[95.7%; 99.3%]</td><td rowspan=1 colspan=1>82.9%(68/82)[73.4%; 89.6%]</td></tr><tr><td rowspan=1 colspan=1>Combined3</td><td rowspan=1 colspan=1>1452</td><td rowspan=1 colspan=1>476</td><td rowspan=1 colspan=1>34</td><td rowspan=1 colspan=1>932</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>97.9%(476/486)[96.3%; 98.9%]</td><td rowspan=1 colspan=1>96.5%(932/966)[95.1%; 97.4%]</td></tr></table>

1The challenge set isolates were well-characterized and compared to the expected results. 2Of the (19) challenge isolates with carbapenemase genes other than blaIMP, blaVIM, blaKPC, blaNDM, blaOXA-48, blaOXA- $^ { . 2 4 , }$ , blaOXA-23, and blaOXA-58, all (19) were reported as carbapenemase producers by the BD Phoenix CPO detect test. These included: (3) blaGES, (3) blaIMI, (12) blaSME, and (1) blaSPM. 3The specificity of the BD Phoenix CPO detect when testing isolates with known carbapenem resistance by cephalosporinase or ESBL coupled with decreased permeability through porin mutations among Enterobacteriaceae, Pseudomonas aeruginosa, and Acinetobacter baumannii has not been fully evaluated. Out of 735 carbapenem “not susceptible” isolates in the clinical study, only four were well-characterized strains with such a known resistance mechanism. All four strains were falsely reported as carbapenemase producers by the BD Phoenix CPO detect test.

Table 8. BD Phoenix CPO Detect Test Performance [Carbapenemase Detection] vs the Composite Reference Method (by Organism Group)   

<table><tr><td rowspan=1 colspan=1>Organism Group</td><td rowspan=1 colspan=1>IsolateType</td><td rowspan=1 colspan=1>Total #</td><td rowspan=1 colspan=1>TP</td><td rowspan=1 colspan=1>FP</td><td rowspan=1 colspan=1>TN</td><td rowspan=1 colspan=1>FN</td><td rowspan=1 colspan=1>PPA[95% CI]</td><td rowspan=1 colspan=1>NPA[95% CI]</td></tr><tr><td rowspan=3 colspan=1>Enterobacteriaceae</td><td rowspan=1 colspan=1>Clinical</td><td rowspan=1 colspan=1>889</td><td rowspan=1 colspan=1>189</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>691</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>99.5%(189/190)[97.1%; 99.9%]</td><td rowspan=1 colspan=1>98.9%(691/699)[97.8%; 99.4%]</td></tr><tr><td rowspan=1 colspan=1>Challenge</td><td rowspan=1 colspan=1>210</td><td rowspan=1 colspan=1>149</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>44</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>97.4%(149/153)[93.5%; 99.0%]</td><td rowspan=1 colspan=1>77.2%1(44/57)[64.8%; 86.2%]</td></tr><tr><td rowspan=1 colspan=1>Combined</td><td rowspan=1 colspan=1>1099</td><td rowspan=1 colspan=1>338</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>735</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>98.5%(338/343)[96.6%-99.4%]</td><td rowspan=1 colspan=1>97.2%(735/756)[95.8%-98.2%]</td></tr><tr><td rowspan=3 colspan=1>Acinetobacterbaumannii</td><td rowspan=1 colspan=1>Clinical</td><td rowspan=1 colspan=1>82</td><td rowspan=1 colspan=1>27</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>51</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>93.1%(27/29)[78.0%; 98.1%]</td><td rowspan=1 colspan=1>96.2%(51/53)[87.3%; 99.0%]</td></tr><tr><td rowspan=1 colspan=1>Challenge</td><td rowspan=1 colspan=1>56</td><td rowspan=1 colspan=1>41</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%(41/41)[91.4%; 100%]</td><td rowspan=1 colspan=1>100%(15/15)[79.6%; 100%]</td></tr><tr><td rowspan=1 colspan=1>Combined</td><td rowspan=1 colspan=1>138</td><td rowspan=1 colspan=1>68</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>66</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>97.1%(68/70)(90.2%-99.2%)</td><td rowspan=1 colspan=1>97.1%(66/68)(89.9%-99.2%)</td></tr><tr><td rowspan=3 colspan=1>Pseudomonasaeruginosa</td><td rowspan=1 colspan=1>Clinical</td><td rowspan=1 colspan=1>164</td><td rowspan=1 colspan=1>29</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>122</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>90.6%(29/32)[75.8%; 98.8%]</td><td rowspan=1 colspan=1>92.4%(122/132)[86.6%; 95.8%]</td></tr><tr><td rowspan=1 colspan=1>Challenge</td><td rowspan=1 colspan=1>51</td><td rowspan=1 colspan=1>41</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%(41/41)[91.4%; 100%]</td><td rowspan=1 colspan=1>90.0%(9/10)[59.6%; 98.2%]</td></tr><tr><td rowspan=1 colspan=1>Combined</td><td rowspan=1 colspan=1>215</td><td rowspan=1 colspan=1>70</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>131</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>95.9%(70/73)[88.6%-98.6%]</td><td rowspan=1 colspan=1>92.3%(131/142)[86.7%-95.6%]</td></tr></table>

1For Enterobacteriacea challenge isolates, (13) false positives were observed from 57 isolates that were negative for carbapenemaseproduction by the Reference Method. These isolates included the following organisms—(8) K. penumoniae spp, (1) P. stuartii, (1) P. mirabilis, (1) K. oxytoca, (1) E. coli, and (1). K. aerogenes.

Table 9. BD Phoenix CPO Detect Test Performance [Ambler Classification] vs the Composite Reference Method   

<table><tr><td rowspan=1 colspan=1>AmblerClass</td><td rowspan=1 colspan=1>IsolateType</td><td rowspan=1 colspan=1>Total #</td><td rowspan=1 colspan=1>TP</td><td rowspan=1 colspan=1>FP</td><td rowspan=1 colspan=1>TN</td><td rowspan=1 colspan=1>FN</td><td rowspan=1 colspan=1>PPA[95% CI]</td><td rowspan=1 colspan=1>NPA[95% CI]</td></tr><tr><td rowspan=3 colspan=1>Class A</td><td rowspan=1 colspan=1>Clinical</td><td rowspan=1 colspan=1>1135</td><td rowspan=1 colspan=1>64</td><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>1043</td><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>82.1%(64/78)[72.1%; 89.0%]</td><td rowspan=1 colspan=1>98.7%(1043/1057)[97.8%; 99.2%]</td></tr><tr><td rowspan=1 colspan=1>Challenge</td><td rowspan=1 colspan=1>317</td><td rowspan=1 colspan=1>59</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>247</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>85.5%(59/69)[75.3%; 91.9%]</td><td rowspan=1 colspan=1>99.6%(247/248)[97.8%; 99.9%]</td></tr><tr><td rowspan=1 colspan=1>Combined</td><td rowspan=1 colspan=1>1452</td><td rowspan=1 colspan=1>123</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>1290</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>83.7%(123/147)[76.9%; 88.8%]</td><td rowspan=1 colspan=1>98.9%(1290/1305)[98.1%; 99.3%]</td></tr><tr><td rowspan=3 colspan=1>Class B</td><td rowspan=1 colspan=1>Clinical</td><td rowspan=1 colspan=1>1135</td><td rowspan=1 colspan=1>52</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>1051</td><td rowspan=1 colspan=1>22</td><td rowspan=1 colspan=1>70.3%(52/74)[59.1%; 79.5%]</td><td rowspan=1 colspan=1>99.1%(1051/1061)[98.3%; 99.5%]</td></tr><tr><td rowspan=1 colspan=1>Challenge</td><td rowspan=1 colspan=1>317</td><td rowspan=1 colspan=1>76</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>214</td><td rowspan=1 colspan=1>18</td><td rowspan=1 colspan=1>80.9%(76/94)[71.8%; 87.5%]</td><td rowspan=1 colspan=1>96.0%(214/223)[92.5%; 97.9%]</td></tr><tr><td rowspan=1 colspan=1>Combined</td><td rowspan=1 colspan=1>1452</td><td rowspan=1 colspan=1>128</td><td rowspan=1 colspan=1>19</td><td rowspan=1 colspan=1>1265</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>76.2%(128/168)[69.2%; 82%]</td><td rowspan=1 colspan=1>98.5%(1265/1284)[97.7%; 99.1%]</td></tr><tr><td rowspan=3 colspan=1>Class D</td><td rowspan=1 colspan=1>Clinical</td><td rowspan=1 colspan=1>1135</td><td rowspan=1 colspan=1>65</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>1042</td><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>79.3%(65/82)[69.3%; 86.6%]</td><td rowspan=1 colspan=1>99.0%(1042/1053)[98.1%; 99.4%]</td></tr><tr><td rowspan=1 colspan=1>Challenge</td><td rowspan=1 colspan=1>317</td><td rowspan=1 colspan=1>70</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>240</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>93.3%(70/75)[85.3%; 97.1%]</td><td rowspan=1 colspan=1>99.2%(240/242)[97.0%; 99.8%]</td></tr><tr><td rowspan=1 colspan=1>Combined</td><td rowspan=1 colspan=1>1452</td><td rowspan=1 colspan=1>135</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>1282</td><td rowspan=1 colspan=1>22</td><td rowspan=1 colspan=1>86.0%(135/157)[79.7%; 90.6%]</td><td rowspan=1 colspan=1>99.0%(1282/1295)[98.3%; 99.4%]</td></tr></table>

The percent of unclassified isolates as determined by the BD Phoenix CPO detect test was $5 . 2 \%$ (76/1452). For those isolates identified as carbapenemase producers by the Reference Method, $1 3 . 2 \%$ (64/486) were not assigned to an Ambler class by BD Phoenix CPO detect. Performance of the device for isolates in which BD Phoenix CPO detect yielded a CPO detection result with Ambler classification is presented in the Table 10 $\mathrm { ( n { = } 1 } 3 5 7 \mathrm { ) }$ , where the PPA and NPA for each classification is shown with (95) isolates excluded. Using the data in Table 10 below, the Ambler class for the carbapenemase was correctly reported by the BD Phoenix CPO detect test the following $\%$ of times—Class A: $9 3 . 8 \%$ (122/130), Class B: $8 7 . 5 \%$ (126/144), and Class D: $9 4 . 4 \%$ (134/142).

Table 10. BD Phoenix CPO Detect Ambler Classification Performance vs the Composite Reference Method (Clinical and Challenge Isolates Combined with Exclusions)1   

<table><tr><td rowspan=1 colspan=1>AmblerClass</td><td rowspan=1 colspan=1>Total #</td><td rowspan=1 colspan=1>TP</td><td rowspan=1 colspan=1>FP</td><td rowspan=1 colspan=1>TN</td><td rowspan=1 colspan=1>FN</td><td rowspan=1 colspan=1>PPA</td><td rowspan=1 colspan=1>NPA</td></tr><tr><td rowspan=1 colspan=1>Class A</td><td rowspan=1 colspan=1>1357</td><td rowspan=1 colspan=1>122</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>1221</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>95.3%(122/128)[90.2%; 97.8%]</td><td rowspan=1 colspan=1>99.3%(1221/1229)[98.7%; 99.7%]</td></tr><tr><td rowspan=1 colspan=1>Class B</td><td rowspan=1 colspan=1>1357</td><td rowspan=1 colspan=1>126</td><td rowspan=1 colspan=1>18</td><td rowspan=1 colspan=1>1205</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>94.0%(126/134)[88.6%; 96.9%]</td><td rowspan=1 colspan=1>98.5%(1205/1223)[97.7%; 99.1%]</td></tr><tr><td rowspan=1 colspan=1>Class D</td><td rowspan=1 colspan=1>1357</td><td rowspan=1 colspan=1>134</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>1208</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>95.0%(134/141)[90%; 98%]</td><td rowspan=1 colspan=1>99.3%(1208/1216)[98.7%; 99.7%]</td></tr></table>

$^ { I } A$ total of 95 isolates were excluded in this analysis—(76) BD Phoenix CPO detect test unclassified isolates, (21) isolates where the Reference Method results were carbapenemase positive, but no Ambler class could be assigned, and (7) isolates containing multiple carbapenemase genes.

b. Clinical specificity: See above   
c. Other clinical supportive data (when a. and b. are not applicable): Not Applicable

4. Clinical cut-off: Not Applicable

5. Expected values/Reference range:

Of the 735 isolates included in the analysis for carbapenemase detection and reported as ‘not susceptible’ based on conventional phenotypic (AST) results, 506 isolates were determined to be carbapenemase producers by the BD Phoenix CPO detect test. For the 717 carbapenem susceptible isolates tested, 713 isolates were determined to be negative for carbapenemase production by the BD Phoenix CPO detect test. Tables 11-14 show the results of the BD Phoenix CPO detect test compared to susceptibility test results for the 1452 isolates included in the clinical study (combined and separately by organism group).

Table 11. All Samples (Enterobacteriaceae $^ +$ Pseudomonas aeruginosa $^ +$ Acinetobacter baumannii), $\mathrm { n } { = } 1 4 5 2$   

<table><tr><td rowspan=5 colspan=1>Co bcee astBard i</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=3>Phenotype (AST)</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>NS1(not susceptible)</td><td rowspan=1 colspan=1>S1(susceptible)</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>CPO Detected</td><td rowspan=1 colspan=1>506</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>510</td></tr><tr><td rowspan=1 colspan=1>CPO Not Detected</td><td rowspan=1 colspan=1>229</td><td rowspan=1 colspan=1>713</td><td rowspan=1 colspan=1>942</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>735</td><td rowspan=1 colspan=1>717</td><td rowspan=1 colspan=1>1452</td></tr></table>

1NS denotes that the isolate was ‘not susceptible’ to at least one of the carbapenems tested. S means the isolate was susceptible to all relevant carbapenems tested. For Pseudomonas aeruginosa and Acinetobacter baumannii/Acinetobacter calcoaceticus complex, ertapenem was not used to determine NS as these organisms are intrinsically resistant to ertapenem.

Table 12. Enterobacteriaceae Only, $\scriptstyle \mathrm { n = 1 0 9 9 }$   

<table><tr><td rowspan=5 colspan=1>Ceo bcee asBacn </td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=3>Phenotype (AST)</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>NS(not susceptible)</td><td rowspan=1 colspan=1>S(susceptible)</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>CPO Detected</td><td rowspan=1 colspan=1>357</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>359</td></tr><tr><td rowspan=1 colspan=1>CPO Not Detected</td><td rowspan=1 colspan=1>174</td><td rowspan=1 colspan=1>566</td><td rowspan=1 colspan=1>740</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>531</td><td rowspan=1 colspan=1>568</td><td rowspan=1 colspan=1>1099</td></tr></table>

Table 13. Pseudomonas aeruginosa Only, $\mathtt { n } { = } 2 1 5$   

<table><tr><td rowspan=5 colspan=1>Ceo bce CrBern in</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=3>Phenotype (AST)</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>NS(not susceptible)</td><td rowspan=1 colspan=1>S(susceptible)</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>CPO Detected</td><td rowspan=1 colspan=1>79</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>81</td></tr><tr><td rowspan=1 colspan=1>CPO Not Detected</td><td rowspan=1 colspan=1>44</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>134</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>123</td><td rowspan=1 colspan=1>92</td><td rowspan=1 colspan=1>215</td></tr></table>

Table 14. Acinetobacter baumannii Only, $\mathrm { n } { = } 1 3 8$   

<table><tr><td rowspan=5 colspan=1>CeoO becee CssBern in</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=3>Phenotype (AST)</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>NS(not susceptible)</td><td rowspan=1 colspan=1>S(susceptible)</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>CPO Detected</td><td rowspan=1 colspan=1>70</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>70</td></tr><tr><td rowspan=1 colspan=1>CPO Not Detected</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>57</td><td rowspan=1 colspan=1>68</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>81</td><td rowspan=1 colspan=1>57</td><td rowspan=1 colspan=1>138</td></tr></table>

# N. Proposed Labeling:

The labeling supports the finding of substantial equivalence for this device.

# O. Conclusion:

The submitted information in this premarket notification is complete and supports a substantial equivalence decision.