# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATION DECISION SUMMARY

B. Purpose for Submission:

To obtain clearance for the ePlex Blood Culture Identification Gram-Positive (BCID-GP) Panel

# C. Measurand:

Bacillus cereus group, Bacillus subtilis group, Corynebacterium, Cutibacterium acnes (P. acnes), Enterococcus, Enterococcus faecalis, Enterococcus faecium, Lactobacillus, Listeria, Listeria monocytogenes, Micrococcus, Staphylococcus, Staphylococcus aureus, Staphylococcus epidermidis, Staphylococcus lugdunensis, Streptococcus, Streptococcus agalactiae (GBS), Streptococcus anginosus group, Streptococcus pneumoniae, Streptococcus pyogenes (GAS), mecA, mecC, vanA and vanB.

# D. Type of Test:

A multiplexed nucleic acid-based test intended for use with the GenMark’s ePlex instrument for the qualitative in vitro detection and identification of multiple bacterial and yeast nucleic acids and select genetic determinants of antimicrobial resistance. The BCID-GP assay is performed directly on positive blood culture samples that demonstrate the presence of organisms as determined by Gram stain.

# E. Applicant:

GenMark Diagnostics, Incorporated

F. Proprietary and Established Names:

ePlex Blood Culture Identification Gram-Positive (BCID-GP) Panel

# G. Regulatory Information:

1. Regulation section:

21 CFR 866.3365 - Multiplex Nucleic Acid Assay for Identification of Microorganisms and Resistance Markers from Positive Blood Cultures

2. Classification:

# Class II

3. Product codes: PAM, PEN, PEO

4. Panel: 83 (Microbiology)

# H. Intended Use:

1. Intended use(s):

The GenMark ePlex Blood Culture Identification Gram-Positive (BCID-GP) Panel is a qualitative nucleic acid multiplex in vitro diagnostic test intended for use on GenMark’s ePlex Instrument for simultaneous qualitative detection and identification of multiple potentially pathogenic gram-positive bacterial organisms and select determinants associated with antimicrobial resistance in positive blood culture. In addition, the ePlex BCID-GP Panel is capable of detecting a wide variety of gram-negative bacteria (Pan Gram-Negative assay) and several Candida species (Pan Candida assay). The ePlex BCID-GP Panel is performed directly on blood culture samples identified as positive by a continuous monitoring blood culture system and which contain gram-positive organism.

The following bacterial organisms and genes associated with antibiotic resistance are identified using the ePlex BCID-GP Panel: Bacillus cereus group, Bacillus subtilis group, Corynebacterium, Cutibacterium acnes (Propionibacterium acnes), Enterococcus, Enterococcus faecalis, Enterococcus faecium, Lactobacillus, Listeria, Listeria monocytogenes, Micrococcus, Staphylococcus, Staphylococcus aureus, Staphylococcus epidermidis, Staphylococcus lugdunensis, Streptococcus, Streptococcus agalactiae (GBS), Streptococcus anginosus group, Streptococcus pneumoniae, Streptococcus pyogenes (GAS), mecA, mecC, vanA and vanB.

The ePlex BCID-GP Panel contains assays for the detection of genetic determinants associated with resistance to methicillin (mecA and mecC) and vancomycin (vanA and vanB) to aid in the identification of potentially antimicrobia l resistant organisms in positive blood culture samples. The antimicrobial resistance gene detected may or may not be associated with the agent responsible for disease.

The ePlex BCID-GP Panel also contains targets designed to detect a broad range of organisms with a potentially misleading Gram stain result or organisms that may be missed by Gram staining altogether, for example in the case of co-infections. These include a broad Pan Gram-Negative assay as well as a Pan Candida assay, which is designed to detect four of the most prevalent Candida species: Candida albicans, Candida glabrata, Candida krusei and Candida parapsilosis.

The detection and identification of specific bacterial and fungal nucleic acids from individuals exhibiting signs and/or symptoms of bloodstream infection aids in the diagnosis of bloodstream infection when used in conjunction with other clinical information. The results from the ePlex BCID-GP Panel are intended to be interpreted in conjunction with Gram stain results and should not be used as the sole basis for diagnosis, treatment, or other patient management decisions.

Negative results in the setting of a suspected bloodstream infection may be due to infection with pathogens that are not detected by this test. Positive results do not rule out co-infection with other organisms; the organism(s) detected by the ePlex BCID-GP Panel may not be the definite cause of disease. Additional laboratory testing (e.g. sub-culturing of positive blood cultures for identification of organisms not detected by ePlex BCID-GP Panel and for susceptibility testing, differentiation of mixed growth and association of antimicrobial resistance marker genes to a specific organism) and clinical presentation must be taken into consideration in the final diagnosis of blood stream infection.

2. Indication(s) for use: Same as Intended Use

3. Special conditions for use statement(s):

For prescription use only For in vitro diagnostic use only

Limitations :

For prescription use only. This test is a qualitative test and does not provide a quantitative value. This product should not be used with blood culture media that contains charcoal. False results were observed for some targets using the BacT Alert FN Plus bottle type (see the Sample Matrix Equivalency (Bottle Evaluation) section of the package insert for additional details) and with a specific lot of BD BACTECTM Plus Anaerobic bottles.   
• Bacterial and fungal nucleic acids may be present in blood culture, independent of bacterial or fungal viability. Detection of an assay target does not guarantee that the corresponding bacteria or fungi are infectious or are the causative agents for clinical symptoms.   
• There is a risk of false negative results due to the presence of sequence variants in the bacterial or fungal targets of the test. For some strains within the Corynebacterium, Streptococcus and Pan Candida results, $100 \%$ detection was not observed at concentrations expected at bottle positivity. See the Analytical Reactivity (Inclusivity) section for additional details.   
• A result of “No Targets Detected” on the ePlex BCID-GP Panel does not preclude the possibility of bacterial or fungal infection. A specimen with a result of No Targets Detected may contain an organism not targeted by the ePlex BCID-GP Panel.   
• Antimicrobial resistance can occur via multiple mechanisms. A “Not Detected” result for

the BCID-GP antimicrobial resistance gene assays does not indicate antimicrobial susceptibility. Subculturing and standard susceptibility testing of isolates is required to determine antimicrobial susceptibility.

In mixed cultures, the ePlex BCID-GP Panel may not identify all organisms in the specimen, depending upon the concentration of each target present.   
• The results of the ePlex BCID-GP Panel should not be used as the sole basis for diagnosis, treatment or other patient management decisions. Bacillus badius was shown to cross-react with the Bacillus subtilis group assay. Burkholderia cepacia was shown to cross-react with the Corynebacterium assay at levels ≥1x107 CFU/mL. An unspeciated Rhodococcus strain (ATCC 49988) was shown to cross-react with the Micrococcus assay at levels $\geq 1 \mathbf { x } 1 0 ^ { 7 }$ CFU/mL. The genus level and group assays included as a part of the BCID-GP Panel are designed to detect a broad range of species but will not necessarily detect all species within a genus or group. For species detected by these assays please refer to the analytical and in silico inclusivity sections of this package insert.   
• For genus level assays it is possible that an unspeciated target may be masked in the case of a co-infection. For example, in the event that an unspeciated Staphylococcus species is present in the same sample as a Staphylococcus epidermidis, there is no ability to determine that the unspeciated Staphylococcus species is present.

4. Special instrument requirements:

For use with the GenMark ePlex instrument

# I. Device Description:

The ePlex BCID-GP Panel is based on the principles of competitive nucleic acid hybridization using a sandwich assay format, wherein a single-stranded target binds concurrently to a sequence-specific solution-phase signal probe and a solid-phase electrode-bound capture probe. The test employs nucleic acid extraction, target amplification via polymerase chain reaction (PCR) or reverse transcription PCR (RTPCR), and hybridization of target DNA. In the process, the double-stranded PCR amplicons are digested with exonuclease to generate single-stranded DNA suitable for hybridization.

Nucleic acid extraction from biological specimens occurs within the cartridge via cell lysis, nucleic acid capture onto magnetic beads, and release for amplification. The nucleic acid extraction is processed through microfluidic liquid handling. Once the nucleic acid targets are captured and inhibitors are washed away, the magnetic particles are delivered to the electrowetting environment on the printed circuit board (PCB) and the targets are eluted from the particles and amplified.

During hybridization, the single-stranded target DNA binds to a complementary, singlestranded capture probe immobilized on the working gold electrode surface. Singlestranded signal probes (labeled with electrochemically active ferrocenes) bind to specific target sequence / region adjacent to the capture probe. Simultaneous hybridization of target to signal probes and capture probe is detected by alternating current voltammetry (ACV). Each working electrode on the array contains specific capture probes, and sequential analysis of each electrode allows detection of multiple analyte targets.

A summary of the ePlex BCID-GP Panel nucleic acid targets is presented in Table 1 below.

Table 1: Targets Detected by the ePlex BCID-GP Panel   

<table><tr><td rowspan=1 colspan=2>Bacte rial Targets</td></tr><tr><td rowspan=1 colspan=1>Bacillus cereus group</td><td rowspan=1 colspan=1>Micrococcus</td></tr><tr><td rowspan=1 colspan=1>Bacillus subtilis group</td><td rowspan=1 colspan=1>Staphylococcus</td></tr><tr><td rowspan=1 colspan=1>Corynebacterium</td><td rowspan=1 colspan=1>Staphylococcus aureus</td></tr><tr><td rowspan=1 colspan=1>Cutibacterium acnes(Propionibacterium acnes)</td><td rowspan=1 colspan=1>Staphylococcus epidermidis</td></tr><tr><td rowspan=1 colspan=1>Enterococcus</td><td rowspan=1 colspan=1>Staphylococcus lugdunensis</td></tr><tr><td rowspan=1 colspan=1>Enterococcus faecalis</td><td rowspan=1 colspan=1>Streptococcus</td></tr><tr><td rowspan=1 colspan=1>Enterococcus faecium</td><td rowspan=1 colspan=1>Streptococcus agalactiae</td></tr><tr><td rowspan=1 colspan=1>Lactobacillus</td><td rowspan=1 colspan=1>Streptococcus anginosus group</td></tr><tr><td rowspan=1 colspan=1>Listeria</td><td rowspan=1 colspan=1>Streptococcus pneumoniae</td></tr><tr><td rowspan=1 colspan=1>Listeria monocytogenes</td><td rowspan=1 colspan=1>Streptococcus pyogenes</td></tr><tr><td rowspan=1 colspan=2>Antimicrobial Resistance Markers</td></tr><tr><td rowspan=1 colspan=1>mecA (methicillin resistance)</td><td rowspan=1 colspan=1>vanA (vancomycin resistance)</td></tr><tr><td rowspan=1 colspan=1>mecC (methicillin resistance)</td><td rowspan=1 colspan=1>vanB (vancomycin resistance)</td></tr><tr><td rowspan=1 colspan=2>Pan Targets</td></tr><tr><td rowspan=1 colspan=1>Pan Gram-Negative</td><td rowspan=1 colspan=1>Pan Candida</td></tr></table>

Materials provided in each kit:   

<table><tr><td rowspan=1 colspan=1>Product</td><td rowspan=1 colspan=1>Ite m</td><td rowspan=1 colspan=1>Compone nts (quantity)</td><td rowspan=1 colspan=1>Storage</td></tr><tr><td rowspan=1 colspan=1>ePlex Blood CultureIdentification Gram-Positive (BCID-GP)</td><td rowspan=1 colspan=1>EA003012</td><td rowspan=1 colspan=1>ePlex BCID-GP Panel Cartridge(12)</td><td rowspan=1 colspan=1>28</td></tr></table>

# Materials required but not provided:

• GenMark ePlex Instrument and Software   
• Pipettes capable of delivering $5 0 \mu \mathrm { L }$   
• Printer (optional) - See ePlex Operator Manual for compatibility guidelines   
$\bullet$ Pipette tips, aerosol resistant, RNase/DNase-free   
• Disposable, powder free gloves   
• $10 \%$ bleach for appropriate surfaces   
• $70 \%$ ethanol or isopropyl alcohol (or equivalent) for appropriate surfaces   
• $1 . 5 \mathrm { m L }$ RNase/DNase-free microcentrifuge tube or equivalent (optional)

# Interpretation of Results

Results interpretation of the ePlex BCID-GP Panel is performed by the ePlex instrument, and the interpretation of results on the ePlex BCID-GP Panel Detection Report for each targeted analyte is summarized in Table 2 below.

Table 2: Interpretation of Results on the ePlex BCID-GP Panel Detection Report   

<table><tr><td rowspan=1 colspan=1>TargetResult</td><td rowspan=1 colspan=1>Explanation</td><td rowspan=1 colspan=1>Action</td></tr><tr><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>The test was completed successfully and the target hasgenerated signal above its defined threshold and theInternal Control was reported as PASS.</td><td rowspan=1 colspan=1>All results are displayed on the ePlexBCID-GP Panel Detection Report.Test is valid, report results.</td></tr><tr><td rowspan=1 colspan=1>NotDetected</td><td rowspan=1 colspan=1>The test was completed successfully and the target didnot generate signal above its defined threshold and theInternal Control was reported as PASS.</td><td rowspan=1 colspan=1>All results are displayed on the ePlexBCID-GP Panel Detection Report.Test is valid, report results.</td></tr><tr><td rowspan=1 colspan=1>Invalid</td><td rowspan=1 colspan=1>The test has not successfully completed and results forthis test are not valid. This may be due to aninstrument or software error.</td><td rowspan=1 colspan=1>No results are displayed on the ePlexBCID-GP Panel Detection Report.Test is not valid, repeat test.</td></tr></table>

# Genus and Group Assay Result Interpretation

The ePlex BCID-GP Panel Enterococcus result is based on three assays: the species-specific Enterococcus faecalis and Enterococcus faecium assays and a broad Enterococcus assay. The broad Enterococcus assay will detect Enterococcus faecalis and Enterococcus faecium, however, its primary purpose is to detect non-faecalis/faecium Enterococcus species. If all three assays are negative, the Enterococcus result will be Not Detected. If any of the three assays is positive, the Enterococcus result will be ‘Detected’. If only the Enterococcus assay is positive, an unspeciated Enterococcus species has been detected. Interpretations of results for Enterococcus are described in Table 3 below.

Table 3: Enterococcus Target Results from ePlex BCID-GP Panel Detection Report   

<table><tr><td rowspan=1 colspan=1>EnterococcusResult</td><td rowspan=1 colspan=1>EnterococcusAssay</td><td rowspan=1 colspan=1>EnterococcusfaecalisAssay</td><td rowspan=1 colspan=1>EnterococcusfaeciumAssay</td><td rowspan=1 colspan=1>Description</td></tr><tr><td rowspan=1 colspan=1>Not Detected</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>No Enterococcus species Detected</td></tr><tr><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Enterococcus faecalis Detected</td></tr><tr><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Enterococcus faecium Detected</td></tr><tr><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Enterococcus faecalis DetectedEnterococcus faecium Detected</td></tr><tr><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Unspeciated EnterococcusDetected</td></tr></table>

The ePlex BCID-GP Panel Listeria result is based on two assays: the species-specific Listeria monocytogenes assay and a broad Listeria assay. The broad Listeria assay will detect multiple Listeria species including Listeria monocytogenes; however, its primary purpose is to detect non-monocytogenes Listeria species. If either assay is positive, the Listeria result will be ‘Detected’. If only the Listeria assay is positive, an unspeciated Listeria species has been detected. Interpretations of Listeria results are described in Table 4 below.

Table 4: Listeria Target Results from ePlex BCID-GP Panel Detection Report   

<table><tr><td rowspan=1 colspan=1>Listeria Result</td><td rowspan=1 colspan=1>ListeriaAssay</td><td rowspan=1 colspan=1>ListeriamonocytogenesAssay</td><td rowspan=1 colspan=1>Description</td></tr><tr><td rowspan=1 colspan=1>Not Detected</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>No Listeria species Detected</td></tr><tr><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Listeria monocytogenesDetected</td></tr><tr><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Unspeciated Listeria Detected</td></tr></table>

The ePlex BCID-GP Panel Staphylococcus result is based on four assays: the species-specific Staphylococcus aureus, Staphylococcus epidermidis and Staphylococcus lugdunensis assays and a broad Staphylococcus assay. The broad Staphylococcus assay will detect each of the species targeted by the species-specific assays, but its primary purpose is to detect other Staphylococcus species. If all four assays are negative, the Staphylococcus result will be ‘Not Detected’. If any of the four assays is positive, the Staphylococcus result will be ‘Detected’. If only the Staphylococcus assay is positive, an unspeciated Staphylococcus has been detected. Interpretations of Staphylococcus results are described in Table 5 below.

Table 5: Staphylococcus Target Results from ePlex BCID-GP Panel Detection Report   

<table><tr><td rowspan=1 colspan=1>StaphylococcusStaphylococcusResult</td><td rowspan=1 colspan=1>StaphylococcusStaphylococcusAssay</td><td rowspan=1 colspan=1>StaphylococcusaureusAssay</td><td rowspan=1 colspan=1>StaphylococcusepidermidisAssay</td><td rowspan=1 colspan=1>StaphylococcuslugdunensisAssay</td><td rowspan=1 colspan=1>Description</td></tr><tr><td rowspan=1 colspan=1>Not Detected</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>No Staphylococcusspecies Detected</td></tr><tr><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>StaphylococcusaureusDetected</td></tr><tr><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>StaphylococcusepidermidisDetected</td></tr><tr><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Staphylococcuslugdunensisetected</td></tr><tr><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>StaphylococcusaureusDetectedStaphylococcusepidermidisDetected</td></tr><tr><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>StaphylococcusaureusDetectedStaphylococcuslugdunensisDetected</td></tr><tr><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>StaphylococcusepidermidisDetectedStaphylococcuslugdunensisDetected</td></tr><tr><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>StaphylococcusaureusDetectedStaphylococcusepidermidisDetectedStaphylococcuslugdunensisDetected</td></tr><tr><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>UnspeciatedStaphylococcusDetected</td></tr></table>

The ePlex BCID-GP Panel Streptococcus result is based on five assays: the species-specific Streptococcus agalactiae, Streptococcus anginosus group, Streptococcus pneumoniae and Streptococcus pyogenes assays and a broad Streptococcus assay. The broad Streptococcus assay will detect each of the species targeted by the species-specific assays, but its primary purpose is to detect other Streptococcus species. If all five assays are negative, the Streptococcus result will be ‘Not Detected’. If any of the five assays is positive, the Streptococcus result will be ‘Detected’. If only the Streptococcus assay is positive, an unspeciated Streptococcus species has been detected. Interpretations of Streptococcus results are described in Table 6 below.

Table 6: Streptococcus Target Results from ePlex BCID-GP Panel Detection Report   

<table><tr><td colspan="1" rowspan="1">StreptococcusStreptococcusResult</td><td colspan="1" rowspan="1">StreptococcusStreptococcusAssay</td><td colspan="1" rowspan="1">StreptococcusagalactiaeAssay</td><td colspan="1" rowspan="1">Streptococcusanginosusgroup Assay</td><td colspan="1" rowspan="1">StreptococcuspneumoniaeAssay</td><td colspan="1" rowspan="1">StreptococcuspyogenesAssay</td><td colspan="1" rowspan="1">Description</td></tr><tr><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">NoStreptococcusspeciesDetected</td></tr><tr><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">StreptococcusagalactiaeDetected</td></tr><tr><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">StreptococcusanginosusgroupDetected</td></tr><tr><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">StreptococcuspneumoniaDetected</td></tr><tr><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">StreptococcuspyogenesDetected</td></tr><tr><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">StreptococcusagalactiaeDetectedStreptococcusanginosusgroupDetected</td></tr><tr><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">StreptococcusagalactiaeDetectedStreptococcuspneumoniaeDetected</td></tr><tr><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Streptococcusagalactiae</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">DetectedStreptococcuspyogenesDetected</td></tr><tr><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">StreptococcusanginosusgroupDetectedStreptococcuspneumoniaeDetected</td></tr><tr><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">StreptococcusanginosusgroupDetectedStreptococcuspyogenesDetected</td></tr><tr><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">StreptococcuspneumoniaeDetectedStreptococcuspyogenesDetected</td></tr><tr><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">StreptococcusagalactiaeDetectedStreptococcusanginosusgroupDetectedStreptococcuspneumoniaeDetected</td></tr><tr><td colspan="1" rowspan="1">StreptococcusStreptococcusResult</td><td colspan="1" rowspan="1">StreptococcusStreptococcusAssay</td><td colspan="1" rowspan="1">StreptococcusagalactiaeAssay</td><td colspan="1" rowspan="1">Streptococcus Streptococcusanginosusgroup Assay</td><td colspan="1" rowspan="1">Streptococcus StreptococcuspneumoniaeAssay</td><td colspan="1" rowspan="1">StreptococcuspyogenesAssay</td><td colspan="1" rowspan="1">Description</td></tr><tr><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">StreptococcusagalactiaeDetectedStreptococcusanginosusgroupDetectedStreptococcuspyogenesDetected</td></tr><tr><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">StreptococcusagalactiaeDetectedStreptococcuspneumoniaeDetectedStreptococcuspyogenesDetected</td></tr><tr><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">StreptococcusanginosusgroupDetectedStreptococcuspneumoniaeDetectedStreptococcuspyogenesDetected</td></tr><tr><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">StreptococcusagalactiaeDetectedStreptococcusanginosusDetectedStreptococcuspneumoniaeDetectedStreptococcuspyogenesDetected</td></tr><tr><td colspan="1" rowspan="1">StreptococcusStreptococcusResult</td><td colspan="1" rowspan="1">StreptococcusStreptococcusAssay</td><td colspan="1" rowspan="1">StreptococcusagalactiaeAssay</td><td colspan="1" rowspan="1">StreptococcusStreptococcusanginosusgroup Assay</td><td colspan="1" rowspan="1">StreptococcusStreptococcuspneumoniaeAssay</td><td colspan="1" rowspan="1">StreptococcuspyogenesAssay</td><td colspan="1" rowspan="1">Description</td></tr><tr><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">UnspeciatedStreptococcusDetected</td></tr></table>

Test results for resistance genes are only reported when an associated organism assay is positive in the same sample. See Table 7 for organisms specifically associated with the four resistance markers on the ePlex BCID-GP Panel.

Table 7: Resistance Marker Organism Associations   

<table><tr><td rowspan=1 colspan=1>Resistance GeneResult</td><td rowspan=1 colspan=1>Associated Targets</td></tr><tr><td rowspan=1 colspan=1>mecA and/or mecC</td><td rowspan=1 colspan=1>Any Staphylococcus assay (Staphylococcus, S. aureus, S. epidermidis, S.lugdunensis)</td></tr><tr><td rowspan=1 colspan=1>vanA and/or vanB</td><td rowspan=1 colspan=1>Any Enterococcus assay (Enterococcus, E. faecalis, E. faecium)</td></tr></table>

The ePlex BCID-GP Panel Pan Gram-Negative result is based on a broad assay that covers most gram-negative organisms which include but are not limited to Acinetobacter, Bacteroides, Enterobacteriaceae, Neisseria, Pseudomonas, Serratia and Stenotrophomonas maltophilia, as shown in Table 8.

Table 8: Pan Gram-Negative Target Results from ePlex BCID-GP Panel Detection Report   

<table><tr><td rowspan=1 colspan=1>Pan Gram-NegativeResult</td><td rowspan=1 colspan=1>Description</td></tr><tr><td rowspan=1 colspan=1>Not Detected</td><td rowspan=1 colspan=1>No gram-negative organism detected.</td></tr><tr><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>One or more gram-negative organisms detected: genera include but are not limited toAcinetobacter, Bacteroides, Enterobacteriaceae, Neisseria, Pseudomonas, Serratia,Stenotrophomonas maltophilia. Additional testing for identification is recommended.</td></tr></table>

The ePlex BCID-GP Panel Pan Candida result indicates the presence of one or more of the following Candida species targets: Candida albicans, Candida glabrata, Candida krusei, or Candida parapsilosis as shown in Table 9.

Table 9: Pan Candida Target Results from ePlex BCID-GP Panel Detection Report   

<table><tr><td rowspan=1 colspan=1>Pan Candida Result</td><td rowspan=1 colspan=1>Description</td></tr><tr><td rowspan=1 colspan=1>Not Detected</td><td rowspan=1 colspan=1>No specified Candida species detected.</td></tr><tr><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>One or more of the following Candida organisms has been detected: Candidaalbicans, Candida glabrata, Candida krusei and /or Candida parapsilosis.Additional testing for identification is recommended.</td></tr></table>

# ePlex BCID-GP Panel Test Reports

Several different reports are available on the ePlex System. Results are provided in a printable format and may be viewed electronically or exported for additional analysis. Reports can be customized with account specific information such as the address, logo and institutional specific footers on each report.

# Detection Report

The ePlex BCID-GP Panel Detection Report includes the results for each individual sample run on the ePlex System. The Summary section indicates the overall test result and lists all detected targets in that sample. The Results section includes a list of all targets on the panel with an individual result for each target. Results are reported as Detected, Not Detected, or Invalid (displayed as a red x); results for the Internal Control are reported as PASS, FAIL, INVALID, or N/A.

# External Control Report

The ePlex BCID-GP Panel External Control Report is generated for an external control that has been pre-defined in the ePlex BCID-GP Panel Software. For more information on defining external controls on the ePlex System, refer to the ePlex Operator Manual.

The Summary section indicates the overall result (PASS or FAIL status) and lists all detected targets for that external control. The Results section includes a list of all panel targets with the result, expected result and PASS/FAIL status for each. Results are reported as Detected, Not Detected, or Invalid (displayed as a red x). A target is reported as PASS if the actual result matches the expected result (as defined for that control); a target is reported as FAIL if the actual result does not match the expected result. If the actual result for each target matches the expected result (all targets reported as PASS), the overall result for the external control is reported as PASS in the Summary section. If the actual result for any target does not match the expected result, the overall result for the external control is reported as FAIL in the Summary section.

# Summary Report

The Summary Report allows the operator to use searchable criteria to create customized reports, using specified targets, dates, range of dates, sample, external control, test bay, or operator.

# J. Substantial Equivalence Information:

1. Predicate device name(s):

BioFire Diagnostics; FilmArray Blood Culture Identification (BCID) Panel

2. Predicate 510(k) number(s):

3. Comparison with predicate:

<table><tr><td colspan="3" rowspan="1">Similarities</td></tr><tr><td colspan="1" rowspan="1">Ite m</td><td colspan="1" rowspan="1">Device: ePlex BCID-GP Panel(k181663)</td><td colspan="1" rowspan="1">Predicate: FilmArray BCID Panel(k130914)</td></tr><tr><td colspan="1" rowspan="1">Indication forUse</td><td colspan="1" rowspan="1">The GenMark ePlex Blood CultureIdentific ation Gram-Positive (BCID-GP)Panel is a qualitative nucleic ac id multiplexin vitro diagnostic test intended for use onGenMark's ePlex Instrument forsimultaneous qualitative detection andidentific ation of multiple potentiallypathogenic gram-positive bacterialorganisms and select determinantsassociated with antimicrobial resistance inpositive blood culture. In addition, the ePlexBCID-GP Panel is capable of detecting awide variety of gram-negative bacteria (PanGram-Negative assay) and several Candidaspecies (Pan Candida assay). The ePlexBCID-GP Panel is performed directly onblood culture samples identified as positiveby a continuous monitoring blood culturesystem and which contain gram-positiveorganism.The following bacterial organisms and genesassociated with antibiotic resistance areidentified using the ePlex BCID-GP Panel:Bacillus cereus group, Bacillus subtilisgroup, Corynebacterium, Cutibacteriumacnes (Propionibacterium acnes),Enterococcus, Enterococcus faecalis,Enterococcus faecium, Lactobacillus,Listeria, Listeria monocytogenes,Micrococcus, Staphylococcus,Staphylococcus aureus, Staphylococcusepidermidis, Staphylococcus lugdunensis,Streptococcus, Streptococcus agalactiae(GBS), Streptococcus anginosus group,Streptococcus pneumoniae, Streptococcuspyogenes (GAS), mecA, mecC, vanA andvanB.The ePlex BCID-GP Panel contains assays</td><td colspan="1" rowspan="1">The FilmArray Blood Culture Identific ation(BCID) Panel is a qualitative multiplexednucleic acid-based in vitro diagnostic testintended for use with the FilmArray Instrument.The FilmArray BCID Panel is capable ofsimultaneous detection and identific ation ofmultiple bacterial and yeast nucleic ac ids andselect genetic determinants of antimicrobialresistance. The BCID assay is performeddirectly on blood culture samples identified aspositive by a continuous monitoring bloodculture system that demonstrates the presence oforganisms as determined by Gram stain.The following gram-positive bacteria, gram-negative bacteria, and yeast are identified usingthe FilmArray BCID Panel: Enterococci,Listeria monocytogenes, commonly encounteredStaphylococci (including specific differentiationof Staphylococcus aureus), commonlyencountered Streptococci (with specificdifferentiation of Streptococcus agalactiae,Streptococcus pneumoniae, and Streptococcuspyogenes), Acinetobacter baumannii, commonlyencountered Enterobacteriaceae (includingspecific differentiation of the Enterobactercloacae complex, Escherichia coli, Klebsiellaoxytoca, Klebsiella pneumoniae, Proteus, andSerratia marcescens), Haemophilus influenzae,Neisseria meningitidis (encapsulated),Pseudomonas aeruginosa, Candida albicans,Candida glabrata, Candida krusei, Candidaparapsilosis, and Candida tropicalis.The FilmArray BCID Panel also contains assaysfor the detection of genetic determinants ofresistance to methicillin (mecA), vancomycin(vanA and vanB), and carbapenems (blakpc) toaid in the identific ation of potentiallyantimicrobial resistant organisms in positive</td></tr><tr><td colspan="3"></td></tr><tr><td colspan="3">Similarities</td></tr><tr><td>Ite m</td><td>De vice: ePlex BCID-GP Panel (k181663)</td><td>Predicate: FilmArray BCID Panel (k130914)</td></tr><tr><td></td><td>for the detection of genetic determinants associated with resistance to methic illin (mecA and mecC) and vancomycin (vanA and vanB) to aid in the identific ation of potentially antimicrobial resistant organisms in positive blood culture samples. The antimicrobial resistance gene detected may or may not be associated with the agent responsible for disease. The ePlex BCID-GP Panel also contains targets designed to detect a broad range of organisms w ith a potentially misleading Gram stain result or organisms that may be missed by Gram staining altogether, for example in the case of co-infections. These include a broad Pan Gram-Negative assay as well as a Pan Candida assay, which is designed to detect four of the most prevalent Candida species: Candida albicans, Candida glabrata, Candida krusei and Candida parapsilosis. The detection and identific ation of spec ific bacterial and fungal nucleic ac ids from individuals exhibiting signs and/or symptoms of bloodstream infection aids in the diagnosis of bloodstream infection when used in conjunction with other clinical information. The results from the ePlex BCID-GP Panel are intended to be interpreted in conjunction w ith Gram stain results and should not be used as the sole basis for diagnosis, treatment, or other patient management dec isions. Negative results in the setting of a suspected bloodstream infection may be due to infection w ith pathogens that are not detected by this test. Positive results do not rule out co-infection with other organisms; the organism(s) detected by the ePlex BCID-GP Panel may not be the definite cause of disease. Additional laboratory testing (e.g. sub-culturing of positive blood cultures for identific ation of organisms not</td><td>blood culture samples. The antimicrobial resistance gene detected may or may not be associated with the agent responsible for disease. Negative results for these select antimicrobial resistance gene assays do not indic ate susceptibility, as multiple mechanisms of resistance to methic illin, vancomycin, and carbapenems exist. FilmArray BCID is indic ated as an aid in the diagnosis of spec ific agents of bacteremia and fungemia and results should be used in conjunction with other clinical and laboratory findings. Positive FilmArray results do not rule out co-infection with organisms not inc luded in the FilmArray BCID Panel. FilmArray BCID is not intended to monitor treatment for bacteremia or fungemia. Subculturing of positive blood cultures is nec essary to recover organisms for susceptibility testing and epidemiological typing, to identify organisms in the blood culture that are not detected by the FilmArray BCID Panel, and for species determination of some Staphylococci, Enterococci, Streptococci, and Enterobacteriac eae that are not spec ific ally identified by the FilmArray BCID Panel assays.</td></tr><tr><td colspan="3" rowspan="1">Similarities</td></tr><tr><td colspan="1" rowspan="1">Ite m</td><td colspan="1" rowspan="1">Device: ePlex BCID-GP Panel(k181663)</td><td colspan="1" rowspan="1">Predicate: FilmArray BCID Panel(k130914)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">detected by ePlex BCID-GP Panel and forsusceptibility testing, differentiation ofmixed growth and association ofantimicrobial resistance marker genes to aspec ific organism) and clinical presentationmust be taken into consideration in the finaldiagnosis of blood stream infection.</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Analyte</td><td colspan="1" rowspan="1">DNA</td><td colspan="1" rowspan="1">DNA</td></tr><tr><td colspan="1" rowspan="1">SampleProcessing</td><td colspan="1" rowspan="1">Automated by instrument</td><td colspan="1" rowspan="1">Automated by instrument</td></tr><tr><td colspan="1" rowspan="1">Controls</td><td colspan="1" rowspan="1">Each cartridge includes internal controls thatmonitor performance of each step of thetesting process, including extraction,amplific ation and detection of targets.</td><td colspan="1" rowspan="1">Two controls are included in each reagent pouchto control for sample processing and both stagesof PCR and melt analysis.</td></tr></table>

<table><tr><td rowspan=1 colspan=3>Diffe rences</td></tr><tr><td rowspan=1 colspan=1>Ite m</td><td rowspan=1 colspan=1>Device: ePlex BCID-GP Panel(k181663)</td><td rowspan=1 colspan=1>Predicate: FilmArray BCID Panel(k130914)</td></tr><tr><td rowspan=1 colspan=1>Specimen Type</td><td rowspan=1 colspan=1>Gram-Positive Blood Culture</td><td rowspan=1 colspan=1>Gram-Positive &amp; Gram-Negative BloodCulture</td></tr><tr><td rowspan=1 colspan=1>Test Principles</td><td rowspan=1 colspan=1>Reagents on cartridge inc lude:sample lysis and nucleic ac idextraction, PCR amplification andhybridization-based electrochemic aldetection reagents.</td><td rowspan=1 colspan=1>The FilmArray BCID pouchcontains freeze-dried reagents to performnuc leic ac id purific ation and nested,multiplex PCR with DNA melt analysis.</td></tr><tr><td rowspan=1 colspan=1>Instrumentationand Hardware</td><td rowspan=1 colspan=1>GenMark ePlex Instrument &amp; SingleUse Cartridge</td><td rowspan=1 colspan=1>FilmArray Instrument andassay pouch</td></tr><tr><td rowspan=1 colspan=1>Software InterfaceResult Reporting</td><td rowspan=1 colspan=1>• GenMark ePlex System Software•  GenMark ePlex BCID-GP PanelSoftware</td><td rowspan=1 colspan=1>The FilmArray Software automaticallyinterprets the results of each DNA melt curveanalysis and combines the data with theresults of the internal pouch controls toprovide a test result for each organism andantimicrobial resistance gene on the panel.</td></tr></table>

# K. Standard/Guidance Document Referenced

Class II Special Controls Guideline: Multiplex Nucleic Acid Assay for Identification of Microorganisms and Resistance Markers from Positive Blood Cultures (May 2015)   
• CLSI MM17-A, Vol. 28, No. 9, Verification and Validation of Multiplex Nucleic Acid Assays   
• CLSI EP17-A2 Evaluation of Detection Capability for Clinical Laboratory Measurement Procedures; Approved Guideline – Second Edition (June 2012)   
• CLSI EP07-A2: Interference Testing in Clinical Chemistry; Approved Guideline – Second Edition (November 2005)   
• CLSP EP25-A Evaluation of Stability of In Vitro Diagnostic Reagents; Approved Guideline (May 2013)

# L. Test Principle:

The ePlex Blood Culture Identification Gram-Positive (BCID-GP) Panel is based on the principle of competitive nucleic acid hybridization using a sandwich assay format, wherein a single-stranded target binds concurrently to a sequence-specific solution-phase signal probe and a solid-phase electrode-bound capture probe. The test employs nucleic acid extraction, target amplification via polymerase chain reaction (PCR) or reverse transcription PCR (RTPCR) and hybridization of target DNA. In the process, the double-stranded PCR amplicons are digested with exonuclease to generate single-stranded DNA suitable for hybridization.

Nucleic acid extraction from biological samples occurs within the cartridge via cell lysis, nucleic acid capture onto magnetic beads, and release for amplification. The nucleic acid extraction is processed through microfluidic liquid handling. Once the nucleic acid targets are captured and inhibitors are washed away, the magnetic particles are delivered to the electrowetting environment on the printed circuit board (PCB) and the targets are eluted from the particles and amplified.

During hybridization, the single-stranded target DNA binds to a complementary, singlestranded capture probe immobilized on the working gold electrode surface. Single-stranded signal probes (labeled with electrochemically active ferrocenes) bind to specific target sequence/region adjacent to the capture probe. Simultaneous hybridization of target to signal probes and capture probe is detected by alternating current voltammetry (ACV). Each working electrode on the array contains specific capture probes, and sequential analysis of each electrode allows detection of multiple analyte targets.

# M. Performance Characteristics:

# 1. Analytical performance:

# a. Analytical Sensitivity:

The limit of detection (LoD) of the ePlex Blood Culture Identification GramPositive Panel was assessed with by testing pooled analytes at a range of concentrations (CFU/mL). Each pool was comprised of an organism mix combined together in negative blood matrix. LoD was determined using qualified stocks of organisms with documented CFU/mL that were pooled together and serially diluted to create organism mixes at the test concentrations. Each assay’s LoD was determined by testing at least two strains or species. The Pan Gram-Negative assay was evaluated with 6 species due to its broad specificity. Testing was initially done using 3 test concentrations per multi-organism mix. Testing was performed with a minimum of two ePlex consumable lots using multiple instruments and

covered a minimum of two testing days. A minimum of 20 replicates per organism mix were tested at each concentration.

A summary of the determined LoD with the overall detection rate is shown in Table 10 below. The target LoD was conservatively established based on the observed testing outcomes. The species tested for each genus level call or antibiotic resistance marker is noted in parentheses, where appropriate.

Table 10: BCID-GP Panel LoD Results Summary   

<table><tr><td rowspan=1 colspan=1>BCID-GP Panel</td><td rowspan=1 colspan=1>Organism Strain ID</td><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>LoD Conce ntration(CFU/mL)</td></tr><tr><td rowspan=2 colspan=1>B. cereus group</td><td rowspan=1 colspan=1>Bacillus cereus</td><td rowspan=1 colspan=1>ATCC 21769</td><td rowspan=1 colspan=1>1 x 105</td></tr><tr><td rowspan=1 colspan=1>Bacillus thuringiensis</td><td rowspan=1 colspan=1>ATCC 35646</td><td rowspan=1 colspan=1>1 x 105</td></tr><tr><td rowspan=2 colspan=1>B. subtilis group</td><td rowspan=1 colspan=1>Bacillus subtilis</td><td rowspan=1 colspan=1>ATCC 55614</td><td rowspan=1 colspan=1>1 x 106</td></tr><tr><td rowspan=1 colspan=1>Bacillus atrophaeus</td><td rowspan=1 colspan=1>ATCC 51189</td><td rowspan=1 colspan=1>1 x 106</td></tr><tr><td rowspan=2 colspan=1>Corynebacterium</td><td rowspan=1 colspan=1>Corynebacterium striatum</td><td rowspan=1 colspan=1>ATCC 43735</td><td rowspan=1 colspan=1>1 x 106</td></tr><tr><td rowspan=1 colspan=1>Corynebacterium jeikeium</td><td rowspan=1 colspan=1>ATCC 43217</td><td rowspan=1 colspan=1>1 x 107</td></tr><tr><td rowspan=2 colspan=1>C. acnes (P.acnes)</td><td rowspan=1 colspan=1>Cutibacterium acnes (P.acnes)</td><td rowspan=1 colspan=1>ATCC 33179</td><td rowspan=1 colspan=1>1 x 107</td></tr><tr><td rowspan=1 colspan=1>Cutibacterium acnes (P.acnes)</td><td rowspan=1 colspan=1>ATCC 6919</td><td rowspan=1 colspan=1>1 x 108</td></tr><tr><td rowspan=3 colspan=1>Enterococcus</td><td rowspan=1 colspan=1>Enterococcus faecium</td><td rowspan=1 colspan=1>ATCC BAA-2316</td><td rowspan=1 colspan=1>1 x 105</td></tr><tr><td rowspan=1 colspan=1>Enterococcus faecium</td><td rowspan=1 colspan=1>ATCC BAA-2317</td><td rowspan=1 colspan=1>1 x 106</td></tr><tr><td rowspan=1 colspan=1>Enterococcus raffinosus</td><td rowspan=1 colspan=1>ATCC 49464</td><td rowspan=1 colspan=1>1 x 106</td></tr><tr><td rowspan=2 colspan=1>E. faecium</td><td rowspan=1 colspan=1>Enterococcus faecium</td><td rowspan=1 colspan=1>ATCC BAA-2316</td><td rowspan=1 colspan=1>1 x 105</td></tr><tr><td rowspan=1 colspan=1>Enterococcus faecium</td><td rowspan=1 colspan=1>ATCC BAA-2317</td><td rowspan=1 colspan=1>1 x 106</td></tr><tr><td rowspan=2 colspan=1>E. faecalis</td><td rowspan=1 colspan=1>Enterococcus faecalis</td><td rowspan=1 colspan=1>ATCC 51575</td><td rowspan=1 colspan=1>1 x 106</td></tr><tr><td rowspan=1 colspan=1>Enterococcus faecalis</td><td rowspan=1 colspan=1>ATCC 700802</td><td rowspan=1 colspan=1>1 x 106</td></tr><tr><td rowspan=2 colspan=1>Lactobacillus</td><td rowspan=1 colspan=1>Lactobacillus paracasei</td><td rowspan=1 colspan=1>ATCC 25598</td><td rowspan=1 colspan=1>1 x 105</td></tr><tr><td rowspan=1 colspan=1>Lactobacillus casei</td><td rowspan=1 colspan=1>ATCC 334</td><td rowspan=1 colspan=1>1 x 105</td></tr><tr><td rowspan=3 colspan=1>Listeria</td><td rowspan=1 colspan=1>Listeria seeligeri</td><td rowspan=1 colspan=1>ATCC 35967</td><td rowspan=1 colspan=1>1 x 105</td></tr><tr><td rowspan=1 colspan=1>Listeria monocytogenes</td><td rowspan=1 colspan=1>ATCC 10890</td><td rowspan=1 colspan=1>1 x 105</td></tr><tr><td rowspan=1 colspan=1>Listeria monocytogenes</td><td rowspan=1 colspan=1>ATCC19111</td><td rowspan=1 colspan=1>1 x 106</td></tr><tr><td rowspan=2 colspan=1>L. monocytogenes</td><td rowspan=1 colspan=1>Listeria monocytogenes</td><td rowspan=1 colspan=1>ATCC10890</td><td rowspan=1 colspan=1>1 x 105</td></tr><tr><td rowspan=1 colspan=1>Listeria monocytogenes</td><td rowspan=1 colspan=1>ATCC19111</td><td rowspan=1 colspan=1>1 x 105</td></tr><tr><td rowspan=2 colspan=1>Micrococcus</td><td rowspan=1 colspan=1>Micrococcus luteus</td><td rowspan=1 colspan=1>ATCC19212</td><td rowspan=1 colspan=1>1 x 106</td></tr><tr><td rowspan=1 colspan=1>Micrococcus luteus</td><td rowspan=1 colspan=1>ATCC10240</td><td rowspan=1 colspan=1>1 x 107</td></tr><tr><td rowspan=7 colspan=1>Staphylococcus</td><td rowspan=1 colspan=1>Staphylococcus aureus</td><td rowspan=1 colspan=1>ATCC BAA-2313</td><td rowspan=1 colspan=1>1 x 104</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus aureus</td><td rowspan=1 colspan=1>ATCCBAA-2312</td><td rowspan=1 colspan=1>1 x 105</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus epidermidis</td><td rowspan=1 colspan=1>ATCC 35983</td><td rowspan=1 colspan=1>1 x 105</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus epidermidis</td><td rowspan=1 colspan=1>ATCC 35984</td><td rowspan=1 colspan=1>1 x 105</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus lugdunensis</td><td rowspan=1 colspan=1>NRS 879</td><td rowspan=1 colspan=1>1 x 105</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus lugdunensis</td><td rowspan=1 colspan=1>ATCC 49576</td><td rowspan=1 colspan=1>1 x 106</td></tr><tr><td rowspan=1 colspan=1>Staphylococcushaemolyticus</td><td rowspan=1 colspan=1>NRS 62</td><td rowspan=1 colspan=1>1 x 107</td></tr></table>

b. Reproducibility:   

<table><tr><td rowspan=1 colspan=1>BCID-GP Panel</td><td rowspan=1 colspan=1>Organis m Strain ID</td><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>LoD Conce ntration(CFU/mL)</td></tr><tr><td rowspan=2 colspan=1>S. aureus</td><td rowspan=1 colspan=1>Staphylococcus aureus</td><td rowspan=1 colspan=1>ATCC BAA-2313</td><td rowspan=1 colspan=1>1 x 105</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus aureus</td><td rowspan=1 colspan=1>ATCC BAA-2312</td><td rowspan=1 colspan=1>1 x 105</td></tr><tr><td rowspan=2 colspan=1>S. epidermidis</td><td rowspan=1 colspan=1>Staphylococcus epidermidis</td><td rowspan=1 colspan=1>ATCC 35983</td><td rowspan=1 colspan=1>1 x 105</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus epidermidis</td><td rowspan=1 colspan=1>ATCC 35984</td><td rowspan=1 colspan=1>1 x 105</td></tr><tr><td rowspan=2 colspan=1>S. lugdunensis</td><td rowspan=1 colspan=1>Staphylococcus lugdunensis</td><td rowspan=1 colspan=1>NRS 879</td><td rowspan=1 colspan=1>1 x 105</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus lugdunensis</td><td rowspan=1 colspan=1>ATCC 49576</td><td rowspan=1 colspan=1>1 x 105</td></tr><tr><td rowspan=5 colspan=1>Streptococcus</td><td rowspan=1 colspan=1>Streptococus pneumoniae</td><td rowspan=1 colspan=1>ATCC BAA-475</td><td rowspan=1 colspan=1>1 x 105</td></tr><tr><td rowspan=1 colspan=1>Streptococus pneumoniae</td><td rowspan=1 colspan=1>ATCC 10357</td><td rowspan=1 colspan=1>1 x 105</td></tr><tr><td rowspan=1 colspan=1>Streptococus gordonii</td><td rowspan=1 colspan=1>ATCC 10558</td><td rowspan=1 colspan=1>1 x 106</td></tr><tr><td rowspan=1 colspan=1>Streptococus agalactiae</td><td rowspan=1 colspan=1>ATCC 12401</td><td rowspan=1 colspan=1>1 x 106</td></tr><tr><td rowspan=1 colspan=1>Streptococus agalactiae</td><td rowspan=1 colspan=1>ATCC 13813</td><td rowspan=1 colspan=1>1 x 107</td></tr><tr><td rowspan=2 colspan=1>S. agalactiae</td><td rowspan=1 colspan=1>Streptococus agalactiae</td><td rowspan=1 colspan=1>ATCC 12401</td><td rowspan=1 colspan=1>1 x 105</td></tr><tr><td rowspan=1 colspan=1>Streptococus agalactiae</td><td rowspan=1 colspan=1>ATCC 13813</td><td rowspan=1 colspan=1>1 x 106</td></tr><tr><td rowspan=2 colspan=1>S. anginosus</td><td rowspan=1 colspan=1>Streptococus intermedius</td><td rowspan=1 colspan=1>ATCC 27335</td><td rowspan=1 colspan=1>1 x 104</td></tr><tr><td rowspan=1 colspan=1>Streptococus anginosus</td><td rowspan=1 colspan=1>ATCC 9895</td><td rowspan=1 colspan=1>1 x 106</td></tr><tr><td rowspan=2 colspan=1>S. pneumoniae</td><td rowspan=1 colspan=1>Streptococus pneumoniae</td><td rowspan=1 colspan=1>ATCC BAA-475</td><td rowspan=1 colspan=1>1 x 105</td></tr><tr><td rowspan=1 colspan=1>Streptococus pneumoniae</td><td rowspan=1 colspan=1>ATCC 10357</td><td rowspan=1 colspan=1>1 x 105</td></tr><tr><td rowspan=2 colspan=1>S. pyogenes</td><td rowspan=1 colspan=1>Streptococus pyogenes</td><td rowspan=1 colspan=1>ATCC 12384</td><td rowspan=1 colspan=1>1 x 105</td></tr><tr><td rowspan=1 colspan=1>Streptococus pyogenes</td><td rowspan=1 colspan=1>ATCC 49399</td><td rowspan=1 colspan=1>1 x 105</td></tr><tr><td rowspan=9 colspan=1>Pan Gram-Negative</td><td rowspan=1 colspan=1>Stenotrophomonasmaltophilia</td><td rowspan=1 colspan=1>ATCC 13636</td><td rowspan=1 colspan=1>1 x 106</td></tr><tr><td rowspan=1 colspan=1>Enterobacter cloacae</td><td rowspan=1 colspan=1>ATCC 13047</td><td rowspan=1 colspan=1>1 x 106</td></tr><tr><td rowspan=1 colspan=1>Escherichia coli</td><td rowspan=1 colspan=1>ATCC 4157</td><td rowspan=1 colspan=1>1 x 106</td></tr><tr><td rowspan=1 colspan=1>Klebsiella pneumoniae</td><td rowspan=1 colspan=1>ATCC BAA-1706</td><td rowspan=1 colspan=1>1 x 106</td></tr><tr><td rowspan=1 colspan=1>Serratia marcescens</td><td rowspan=1 colspan=1>ATCC 8100</td><td rowspan=1 colspan=1>1 x 106</td></tr><tr><td rowspan=1 colspan=1>Proteus mirabilis</td><td rowspan=1 colspan=1>ATCC 43071</td><td rowspan=1 colspan=1>1 x 106</td></tr><tr><td rowspan=1 colspan=1>Acinetobacter baumannii</td><td rowspan=1 colspan=1>NCTC13302</td><td rowspan=1 colspan=1>1 x 107</td></tr><tr><td rowspan=1 colspan=1>Neisseria meningitidis</td><td rowspan=1 colspan=1>ATCC 13113</td><td rowspan=1 colspan=1>1 x 107</td></tr><tr><td rowspan=1 colspan=1>Pseudomonas aeruginosa</td><td rowspan=1 colspan=1>ATCC 15442</td><td rowspan=1 colspan=1>1 x 107</td></tr><tr><td rowspan=2 colspan=1>Pan Candida</td><td rowspan=1 colspan=1>Candida albicans</td><td rowspan=1 colspan=1>ATCC 24433</td><td rowspan=1 colspan=1>1 x 106</td></tr><tr><td rowspan=1 colspan=1>Candida glabrata</td><td rowspan=1 colspan=1>ATCC 66032</td><td rowspan=1 colspan=1>1 x 106</td></tr><tr><td rowspan=2 colspan=1>mecA</td><td rowspan=1 colspan=1>Staphylococcus epidermidis</td><td rowspan=1 colspan=1>ATCC 35983</td><td rowspan=1 colspan=1>1 x 105</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus epidermidis</td><td rowspan=1 colspan=1>ATCC 35984</td><td rowspan=1 colspan=1>1 x 105</td></tr><tr><td rowspan=2 colspan=1>mecC</td><td rowspan=1 colspan=1>Staphylococcus aureus</td><td rowspan=1 colspan=1>ATCC BAA-2313</td><td rowspan=1 colspan=1>1 x 104</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus aureus</td><td rowspan=1 colspan=1>ATCC BAA-2312</td><td rowspan=1 colspan=1>1 x 104</td></tr><tr><td rowspan=2 colspan=1>vanA</td><td rowspan=1 colspan=1>Enterococcus faecium</td><td rowspan=1 colspan=1>ATCC BAA-2316</td><td rowspan=1 colspan=1>1 x 104</td></tr><tr><td rowspan=1 colspan=1>Enterococcus faecium</td><td rowspan=1 colspan=1>ATCC BAA-2317</td><td rowspan=1 colspan=1>1 x 105</td></tr><tr><td rowspan=2 colspan=1>vanB</td><td rowspan=1 colspan=1>Enterococcus faecalis</td><td rowspan=1 colspan=1>ATCC 51575</td><td rowspan=1 colspan=1>1 x 105</td></tr><tr><td rowspan=1 colspan=1>Enterococcus faecalis</td><td rowspan=1 colspan=1>ATCC 700802</td><td rowspan=1 colspan=1>1 x 105</td></tr></table>

A multisite reproducibility study of the ePlex BCID-GP Panel was conducted at three testing sites (two external sites and one internal site) across major potential sources of variability, such as site-to-site, lot-to-lot, day-to-day, and operator-to-operator. One ePlex instrument per study site with four towers was employed in this reproducibility study. Two operators performed testing at each site on six days (five nonconsecutive days) with three unique lots of ePlex BCID-GP Panel cartridges. A reproducibility study panel consisting of four panel members; including 9 on-panel organisms and three antibiotic resistance genes representing 15 targets at two concentrations and one negative mix including an off-panel organism were tested in triplicate. Concentrations in the positive mixes reflected those observed at time of bottle positivity plus 8 hours $( \mathrm { B P } { + } 8 )$ and time of bottle positivity (BP) and one mix containing an off-panel organism grown to bottle positivity, which is expected to yield a negative result.

Summary results for the ePlex BCID-GP Panel reproducibility study are provided in Table 11 below.

Table 11: Summary of Reproducibility Results   

<table><tr><td colspan="1" rowspan="2">BCID-GP Test Result</td><td colspan="1" rowspan="2">Organis m TestConcentration</td><td colspan="1" rowspan="2">Site</td><td colspan="3" rowspan="1">Agre ement with Expected Results</td></tr><tr><td colspan="1" rowspan="1">Agreed/N</td><td colspan="1" rowspan="1">%</td><td colspan="1" rowspan="1">95% CI</td></tr><tr><td colspan="1" rowspan="12">Corynebacteriumstriatum</td><td colspan="1" rowspan="4">Bottle Positive+ 8 Hours</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">(90.4-100)</td></tr><tr><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">(90.4-100)</td></tr><tr><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">(90.4-100)</td></tr><tr><td colspan="1" rowspan="1">All</td><td colspan="1" rowspan="1">108/108</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">(96.6-100)</td></tr><tr><td colspan="1" rowspan="4">Bottle Positive</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">35/36</td><td colspan="1" rowspan="1">97.2</td><td colspan="1" rowspan="1">(85.8-99.5)</td></tr><tr><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">35/36</td><td colspan="1" rowspan="1">97.2</td><td colspan="1" rowspan="1">(85.8-99.5)</td></tr><tr><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">33/36</td><td colspan="1" rowspan="1">91.7</td><td colspan="1" rowspan="1">(78.2-97.1)</td></tr><tr><td colspan="1" rowspan="1">All</td><td colspan="1" rowspan="1">103/108</td><td colspan="1" rowspan="1">95.4</td><td colspan="1" rowspan="1">(89.6-98.0)</td></tr><tr><td colspan="1" rowspan="4">Negative</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">108/108</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">(96.6-100)</td></tr><tr><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">107/108</td><td colspan="1" rowspan="1">99.1</td><td colspan="1" rowspan="1">(94.9-99.8)</td></tr><tr><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">107/107</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">(96.5-100)</td></tr><tr><td colspan="1" rowspan="1">All</td><td colspan="1" rowspan="1">322/323</td><td colspan="1" rowspan="1">99.7</td><td colspan="1" rowspan="1">(98.3-99.9)</td></tr><tr><td colspan="1" rowspan="12">Cutibacterium acnes(Propionibacteriumacnes)</td><td colspan="1" rowspan="4">Bottle Positive+ 8 Hours</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">32/36</td><td colspan="1" rowspan="1">88.9</td><td colspan="1" rowspan="1">(74.7-95.6)</td></tr><tr><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">33/36</td><td colspan="1" rowspan="1">91.7</td><td colspan="1" rowspan="1">(78.2-97.1)</td></tr><tr><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">(90.4-100)</td></tr><tr><td colspan="1" rowspan="1">All</td><td colspan="1" rowspan="1">101/108</td><td colspan="1" rowspan="1">93.5</td><td colspan="1" rowspan="1">(87.2-96.8)</td></tr><tr><td colspan="1" rowspan="4">Bottle Positive</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">(90.4-100)</td></tr><tr><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">33/36</td><td colspan="1" rowspan="1">91.7</td><td colspan="1" rowspan="1">(78.2-97.1)</td></tr><tr><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">35/36</td><td colspan="1" rowspan="1">97.2</td><td colspan="1" rowspan="1">(85.8-99.5)</td></tr><tr><td colspan="1" rowspan="1">All</td><td colspan="1" rowspan="1">104/108</td><td colspan="1" rowspan="1">96.3</td><td colspan="1" rowspan="1">(90.9-98.6)</td></tr><tr><td colspan="1" rowspan="4">Negative</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">108/108</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">(96.6-100)</td></tr><tr><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">108/108</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">(96.6-100)</td></tr><tr><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">107/107</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">(96.5-100)</td></tr><tr><td colspan="1" rowspan="1">All</td><td colspan="1" rowspan="1">323/323</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">(98.8-100)</td></tr><tr><td colspan="1" rowspan="12">Enterococcus</td><td colspan="1" rowspan="4">Bottle Positive+ 8 Hours</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">72/72</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">(94.9-100)</td></tr><tr><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">72/72</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">(94.9-100)</td></tr><tr><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">72/72</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">(94.9-100)</td></tr><tr><td colspan="1" rowspan="1">All</td><td colspan="1" rowspan="1">216/216</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">(98.3-100)</td></tr><tr><td colspan="1" rowspan="4">Bottle Positive</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">72/72</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">(94.9-100)</td></tr><tr><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">72/72</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">(94.9-100)</td></tr><tr><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">72/72</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">(94.9-100)</td></tr><tr><td colspan="1" rowspan="1">All</td><td colspan="1" rowspan="1">216/216</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">(98.3-100)</td></tr><tr><td colspan="1" rowspan="4">Negative</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">(90.4-100)</td></tr><tr><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">(90.4-100)</td></tr><tr><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">35/35</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">(90.1-100)</td></tr><tr><td colspan="1" rowspan="1">All</td><td colspan="1" rowspan="1">107/107</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">(96.5-100)</td></tr><tr><td colspan="1" rowspan="12">Enterococcusfaecalis</td><td colspan="1" rowspan="4">Bottle Positive+ 8 Hours</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">(90.4-100)</td></tr><tr><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">(90.4-100)</td></tr><tr><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">(90.4-100)</td></tr><tr><td colspan="1" rowspan="1">All</td><td colspan="1" rowspan="1">108/108</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">(96.6-100)</td></tr><tr><td colspan="1" rowspan="4">Bottle Positive</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">(90.4-100)</td></tr><tr><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">(90.4-100)</td></tr><tr><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">(90.4-100)</td></tr><tr><td colspan="1" rowspan="1">All</td><td colspan="1" rowspan="1">108/108</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">(96.6-100)</td></tr><tr><td colspan="1" rowspan="4">Negative</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">108/108</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">(96.6-100)</td></tr><tr><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">108/108</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">(96.6-100)</td></tr><tr><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">107/107</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">(96.5-100)</td></tr><tr><td colspan="1" rowspan="1">All</td><td colspan="1" rowspan="1">323/323</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">(98.8-100)</td></tr><tr><td colspan="1" rowspan="12">Enterococcusfaecium</td><td colspan="1" rowspan="4">Bottle Positive+ 8 Hours</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">(90.4-100)</td></tr><tr><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">(90.4-100)</td></tr><tr><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">(90.4-100)</td></tr><tr><td colspan="1" rowspan="1">All</td><td colspan="1" rowspan="1">108/108</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">(96.6-100)</td></tr><tr><td colspan="1" rowspan="4">Bottle Positive</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">(90.4-100)</td></tr><tr><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">(90.4-100)</td></tr><tr><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">107-107</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">(96.5-100)</td></tr><tr><td colspan="1" rowspan="1">All</td><td colspan="1" rowspan="1">108/108</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">(96.6-100)</td></tr><tr><td colspan="1" rowspan="4">Negative</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">108/108</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">(96.6-100)</td></tr><tr><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">108/108</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">(96.6-100)</td></tr><tr><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">107/107</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">(96.5-100)</td></tr><tr><td colspan="1" rowspan="1">All</td><td colspan="1" rowspan="1">323/323</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">(98.8-100)</td></tr><tr><td colspan="1" rowspan="9">Lactobacillus</td><td colspan="1" rowspan="4">Bottle Positive+ 8 Hours</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">(90.1-100)</td></tr><tr><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">(89.8-100)</td></tr><tr><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">(90.4-100)</td></tr><tr><td colspan="1" rowspan="1">All</td><td colspan="1" rowspan="1">108/108</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">(96.5-100)</td></tr><tr><td colspan="1" rowspan="4">Bottle Positive</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">(89.8-100)</td></tr><tr><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">(90.4-100)</td></tr><tr><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">(90.1-100)</td></tr><tr><td colspan="1" rowspan="1">All</td><td colspan="1" rowspan="1">108/108</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">(96.5-100)</td></tr><tr><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">108/108</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">(90.4-100)</td></tr><tr><td colspan="1" rowspan="3"></td><td colspan="1" rowspan="3"></td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">108/108</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">(89.8-100)</td></tr><tr><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">107/107</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">(90.1-100)</td></tr><tr><td colspan="1" rowspan="1">All</td><td colspan="1" rowspan="1">323/323</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">(96.5-100)</td></tr><tr><td colspan="1" rowspan="12">Staphylococcus</td><td colspan="1" rowspan="4">Bottle Positive+ 8 Hours</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">(90.4-100)</td></tr><tr><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">(90.4-100)</td></tr><tr><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">(90.4-100)</td></tr><tr><td colspan="1" rowspan="1">All</td><td colspan="1" rowspan="1">108/108</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">(96.6-100)</td></tr><tr><td colspan="1" rowspan="4">Bottle Positive</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">(90.4-100)</td></tr><tr><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">35/36</td><td colspan="1" rowspan="1">97.2</td><td colspan="1" rowspan="1">(85.8-99.5)</td></tr><tr><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">(90.4-100)</td></tr><tr><td colspan="1" rowspan="1">All</td><td colspan="1" rowspan="1">107/108</td><td colspan="1" rowspan="1">99.1</td><td colspan="1" rowspan="1">(94.9-99.8)</td></tr><tr><td colspan="1" rowspan="4">Negative</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">108/108</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">(96.6-100)</td></tr><tr><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">107/108</td><td colspan="1" rowspan="1">99.1</td><td colspan="1" rowspan="1">(94.9-99.8)</td></tr><tr><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">107/107</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">(96.5-100)</td></tr><tr><td colspan="1" rowspan="1">All</td><td colspan="1" rowspan="1">322/323</td><td colspan="1" rowspan="1">99.7</td><td colspan="1" rowspan="1">(98.3-99.9)</td></tr><tr><td colspan="1" rowspan="12">Staphylococcusaureus</td><td colspan="1" rowspan="4">Bottle Positive+ 8 Hours</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">(90.4-100)</td></tr><tr><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">(90.4-100)</td></tr><tr><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">(90.4-100)</td></tr><tr><td colspan="1" rowspan="1">All</td><td colspan="1" rowspan="1">108/108</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">(96.6-100)</td></tr><tr><td colspan="1" rowspan="4">Bottle Positive</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">(90.4-100)</td></tr><tr><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">(90.4-100)</td></tr><tr><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">(90.4-100)</td></tr><tr><td colspan="1" rowspan="1">All</td><td colspan="1" rowspan="1">108/108</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">(96.6-100)</td></tr><tr><td colspan="1" rowspan="4">Negative</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">108/108</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">(96.6-100)</td></tr><tr><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">108/108</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">(96.6-100)</td></tr><tr><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">107/107</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">(96.5-100)</td></tr><tr><td colspan="1" rowspan="1">All</td><td colspan="1" rowspan="1">323/323</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">(98.8-100)</td></tr><tr><td colspan="1" rowspan="12">Streptococcus</td><td colspan="1" rowspan="4">Bottle Positive+ 8 Hours</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">(90.4-100)</td></tr><tr><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">(90.4-100)</td></tr><tr><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">(90.4-100)</td></tr><tr><td colspan="1" rowspan="1">All</td><td colspan="1" rowspan="1">108/108</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">(96.6-100)</td></tr><tr><td colspan="1" rowspan="4">Bottle Positive</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">(90.4-100)</td></tr><tr><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">35/36</td><td colspan="1" rowspan="1">97.2</td><td colspan="1" rowspan="1">(85.8-99.5)</td></tr><tr><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">(90.4-100)</td></tr><tr><td colspan="1" rowspan="1">All</td><td colspan="1" rowspan="1">107/108</td><td colspan="1" rowspan="1">99.1</td><td colspan="1" rowspan="1">(94.9-99.8)</td></tr><tr><td colspan="1" rowspan="4">Negative</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">108/108</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">(96.6-100)</td></tr><tr><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">107/108</td><td colspan="1" rowspan="1">99.1</td><td colspan="1" rowspan="1">(94.9-99.8)</td></tr><tr><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">107/107</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">(96.5-100)</td></tr><tr><td colspan="1" rowspan="1">All</td><td colspan="1" rowspan="1">322/323</td><td colspan="1" rowspan="1">99.7</td><td colspan="1" rowspan="1">(98.3-99.9)</td></tr><tr><td colspan="1" rowspan="6">Streptococcus anginosusgroup</td><td colspan="1" rowspan="4">Bottle Positive+ 8 Hours</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">(90.4-100)</td></tr><tr><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">(90.4-100)</td></tr><tr><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">(90.4-100)</td></tr><tr><td colspan="1" rowspan="1">All</td><td colspan="1" rowspan="1">108/108</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">(96.6-100)</td></tr><tr><td colspan="1" rowspan="2">Bottle Positive</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">(90.4-100)</td></tr><tr><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">35/36</td><td colspan="1" rowspan="1">97.2</td><td colspan="1" rowspan="1">(85.8-99.5)</td></tr><tr><td colspan="1" rowspan="6"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">(90.4-100)</td></tr><tr><td colspan="1" rowspan="1">All</td><td colspan="1" rowspan="1">107/108</td><td colspan="1" rowspan="1">99.1</td><td colspan="1" rowspan="1">(94.9-99.8)</td></tr><tr><td colspan="1" rowspan="4">Negative</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">108/108</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">(96.6-100)</td></tr><tr><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">108/108</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">(96.6-100)</td></tr><tr><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">107/107</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">(96.5-100)</td></tr><tr><td colspan="1" rowspan="1">All</td><td colspan="1" rowspan="1">323/323</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">(98.8-100)</td></tr><tr><td colspan="1" rowspan="12">Pan Candida</td><td colspan="1" rowspan="4">Bottle Positive+ 8 Hours</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">(90.4-100)</td></tr><tr><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">(90.4-100)</td></tr><tr><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">(90.4-100)</td></tr><tr><td colspan="1" rowspan="1">All</td><td colspan="1" rowspan="1">108/108</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">(96.6-100)</td></tr><tr><td colspan="1" rowspan="4">Bottle Positive</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">(90.4-100)</td></tr><tr><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">(90.4-100)</td></tr><tr><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">(90.4-100)</td></tr><tr><td colspan="1" rowspan="1">All</td><td colspan="1" rowspan="1">108/108</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">(96.6-100)</td></tr><tr><td colspan="1" rowspan="4">Negative</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">108/108</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">(96.6-100)</td></tr><tr><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">108/108</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">(96.6-100)</td></tr><tr><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">107/107</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">(96.5-100)</td></tr><tr><td colspan="1" rowspan="1">All</td><td colspan="1" rowspan="1">323/323</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">(98.8-100)</td></tr><tr><td colspan="1" rowspan="12">Pan Gram-Negative</td><td colspan="1" rowspan="4">Bottle Positive+ 8 Hours</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">(90.4-100)</td></tr><tr><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">(90.4-100)</td></tr><tr><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">(90.4-100)</td></tr><tr><td colspan="1" rowspan="1">All</td><td colspan="1" rowspan="1">108/108</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">(96.6-100)</td></tr><tr><td colspan="1" rowspan="4">Bottle Positive</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">(90.4-100)</td></tr><tr><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">(90.4-100)</td></tr><tr><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">(90.4-100)</td></tr><tr><td colspan="1" rowspan="1">All</td><td colspan="1" rowspan="1">108/108</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">(96.6-100)</td></tr><tr><td colspan="1" rowspan="4">Negative</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">108/108</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">(96.6-100)</td></tr><tr><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">107/108</td><td colspan="1" rowspan="1">99.1</td><td colspan="1" rowspan="1">(94.9-99.8)</td></tr><tr><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">107/107</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">(96.5-100)</td></tr><tr><td colspan="1" rowspan="1">All</td><td colspan="1" rowspan="1">322/323</td><td colspan="1" rowspan="1">99.7</td><td colspan="1" rowspan="1">(98.3-99.9)</td></tr><tr><td colspan="1" rowspan="12">mecA</td><td colspan="1" rowspan="4">Bottle Positive+ 8 Hours</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">(90.4-100)</td></tr><tr><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">(90.4-100)</td></tr><tr><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">(90.4-100)</td></tr><tr><td colspan="1" rowspan="1">All</td><td colspan="1" rowspan="1">108/108</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">(96.6-100)</td></tr><tr><td colspan="1" rowspan="4">Bottle Positive</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">(90.4-100)</td></tr><tr><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">(90.4-100)</td></tr><tr><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">(90.4-100)</td></tr><tr><td colspan="1" rowspan="1">All</td><td colspan="1" rowspan="1">108/108</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">(96.6-100)</td></tr><tr><td colspan="1" rowspan="4">Negative</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">108/108</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">(96.6-100)</td></tr><tr><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">108/108</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">(96.6-100)</td></tr><tr><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">107/107</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">(96.5-100)</td></tr><tr><td colspan="1" rowspan="1">All</td><td colspan="1" rowspan="1">323/323</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">(98.8-100)</td></tr><tr><td colspan="1" rowspan="3">vanA</td><td colspan="1" rowspan="3">Bottle Positive+ 8 Hours</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">(90.4-100)</td></tr><tr><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">(90.4-100)</td></tr><tr><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">(90.4-100)</td></tr><tr><td colspan="1" rowspan="9"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">All</td><td colspan="1" rowspan="1">108/108</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">(96.6-100)</td></tr><tr><td colspan="1" rowspan="4">Bottle Positive</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">(90.4-100)</td></tr><tr><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">(90.4-100)</td></tr><tr><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">(90.4-100)</td></tr><tr><td colspan="1" rowspan="1">All</td><td colspan="1" rowspan="1">108/108</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">(96.6-100)</td></tr><tr><td colspan="1" rowspan="4">Negative</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">108/108</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">(96.6-100)</td></tr><tr><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">108/108</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">(96.6-100)</td></tr><tr><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">107/107</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">(96.5-100)</td></tr><tr><td colspan="1" rowspan="1">All</td><td colspan="1" rowspan="1">323/323</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">(98.8-100)</td></tr><tr><td colspan="1" rowspan="12">vanB</td><td colspan="1" rowspan="4">Bottle Positive+ 8 Hours</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">(90.4-100)</td></tr><tr><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">(90.4-100)</td></tr><tr><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">(90.4-100)</td></tr><tr><td colspan="1" rowspan="1">All</td><td colspan="1" rowspan="1">108/108</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">(96.6-100)</td></tr><tr><td colspan="1" rowspan="4">Bottle Positive</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">(90.4-100)</td></tr><tr><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">(90.4-100)</td></tr><tr><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">(90.4-100)</td></tr><tr><td colspan="1" rowspan="1">All</td><td colspan="1" rowspan="1">108/108</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">(96.6-100)</td></tr><tr><td colspan="1" rowspan="4">Negative</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">108/108</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">(96.6-100)</td></tr><tr><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">108/108</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">(96.6-100)</td></tr><tr><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">107/107</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">(96.5-100)</td></tr><tr><td colspan="1" rowspan="1">All</td><td colspan="1" rowspan="1">323/323</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">(98.8-100)</td></tr></table>

c. Linearity/assay reportable range:

Not applicable

d. Traceability, Stability, Expected values (controls, calibrators, or methods):

# Assay Controls

Internal Controls:   
Each ePlex BCID- GP Panel cartridge includes internal controls that monitor performance of each step of the testing process, including extraction, amplification and detection of targets.

Each amplification reaction on the cartridge has an internal control and in each reaction either the internal control or a target must generate signal above the defined threshold for a valid test result. Internal control results are interpreted by the ePlex Software and displayed on the ePlex BCID-GP Panel Reports as Internal Control with a result of PASS, FAIL, N/A or INVALID. Table 12 includes details on the interpretation of Internal Control results.

Table 12: Internal Control Results   

<table><tr><td rowspan=1 colspan=1>Inte rnal ControlResult</td><td rowspan=1 colspan=1>Explanation</td><td rowspan=1 colspan=1>Action</td></tr><tr><td rowspan=1 colspan=1>PASS</td><td rowspan=1 colspan=1>Signal above threshold has beendetected from each amplificationreaction.The test was completed andinternal controls were successful,indicating valid results weregenerated.</td><td rowspan=1 colspan=1>All results are displayed on theePlex BCID-GP PanelDetection Report.Test is valid, report results.</td></tr><tr><td rowspan=1 colspan=1>FAIL</td><td rowspan=1 colspan=1>Signal above threshold has notbeen detected from at least oneamplification reaction.The test was completed butinternal controls were notdetected, indic ating that resultsmay not be valid.</td><td rowspan=1 colspan=1>No results are displayed on theePlex BCID-GP PanelDetection Report.Test is not valid, repeat thetest using a new cartridge.</td></tr><tr><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>The internal control in everyamplification reaction does notgenerate signal above thethreshold, but a target in everyamplification reaction doesgenerate signal above thethreshold.The test was completed andinternal controls were notsuccessful, however detection ofsignal above the threshold for atarget in every amplificationreaction indic ates valid resultswere generated.</td><td rowspan=1 colspan=1>All results are displayed on theePlex BCID-GP PanelDetection Report.Test is valid, report results.</td></tr><tr><td rowspan=1 colspan=1>INVALID</td><td rowspan=1 colspan=1>An error has occurred duringprocessing that prevents analysisof signal data.The test has not successfullycompleted and results for this testare not valid. This may be due toan instrument or software error.</td><td rowspan=1 colspan=1>No results are displayed on theePlex BCID-GP PanelDetection Report.Test is not valid, repeat thetest using a new cartridge.</td></tr></table>

Recommended External Controls: External controls are not provided with the ePlex BCID Gram-Positive Panel, but are recommended in the package insert. Positive and negative external controls should be tested with each new lot of reagents or monthly, whichever occurs first. Blood culture medium can be used as the negative control. Previously characterized positive samples or blood culture medium spiked with well characterized organisms can be used as the external positive control. External controls should be run in accordance with laboratory protocols and accrediting organizations, as applicable.

# Specimen Stability

A specimen stability study was performed to confirm the stability of specimens stored under four different temperature conditions over multiple time points which vary based on storage temperature. Organism mixes whose members were at a concentration approximating bottle positivity were stored at $\mathrm { \leq } \mathrm { - } 7 0 ^ { \circ } \mathrm { C }$ , $\mathrm { \leq - 2 0 ^ { \circ } C }$ , $2 ^ { \circ } - 8 ^ { \circ } \mathrm { C }$ , and ambient temperature over various time points described in Table 13 below.

Table 13: Storage Conditions and Testing Time Points   

<table><tr><td rowspan=1 colspan=1>StorageCondition</td><td rowspan=1 colspan=8>Time Points</td></tr><tr><td rowspan=1 colspan=1>Ambient</td><td rowspan=1 colspan=1>TO</td><td rowspan=1 colspan=1>1 day</td><td rowspan=1 colspan=1>3 days</td><td rowspan=1 colspan=1>5 days</td><td rowspan=1 colspan=1>7 days</td><td rowspan=1 colspan=1>14 days</td><td rowspan=1 colspan=1>30 days</td><td rowspan=1 colspan=1>--</td></tr><tr><td rowspan=1 colspan=1>Refrigerated(2-8C)</td><td rowspan=1 colspan=1>--</td><td rowspan=1 colspan=1>1 day</td><td rowspan=1 colspan=1>3 days</td><td rowspan=1 colspan=1>5 days</td><td rowspan=1 colspan=1>7 days</td><td rowspan=1 colspan=1>14 days</td><td rowspan=1 colspan=1>30 days</td><td rowspan=1 colspan=1>--</td></tr><tr><td rowspan=1 colspan=1>Frozen(≤-20C)</td><td rowspan=1 colspan=1>1 week</td><td rowspan=1 colspan=1>2 weeks</td><td rowspan=1 colspan=1>1 mo.</td><td rowspan=1 colspan=1>3 mo.*</td><td rowspan=1 colspan=1>6 mo.</td><td rowspan=1 colspan=1>12 mo.</td><td rowspan=1 colspan=1>18 mo.</td><td rowspan=1 colspan=1>24 mo.</td></tr><tr><td rowspan=1 colspan=1>Frozen(≤-70)</td><td rowspan=1 colspan=1>1 week</td><td rowspan=1 colspan=1>2 weeks</td><td rowspan=1 colspan=1>1 mo.</td><td rowspan=1 colspan=1>3 mo.*</td><td rowspan=1 colspan=1>6 mo.</td><td rowspan=1 colspan=1>12 mo.</td><td rowspan=1 colspan=1>18 mo.</td><td rowspan=1 colspan=1>24 mo.</td></tr><tr><td rowspan=1 colspan=1>BottleIncubator</td><td rowspan=1 colspan=1>Bottlepositivity</td><td rowspan=1 colspan=1>12 hourspostpositivity</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>--</td><td rowspan=1 colspan=1>--</td><td rowspan=1 colspan=1>--</td><td rowspan=1 colspan=1>--</td><td rowspan=1 colspan=1>--</td></tr></table>

$^ { * } \mathrm { M i x } \ 1$ was tested up to 3 months; Mix 2 was tested up to 1 month.

These organism mixes represent each of the eight multiplex amplification pools on the BCID-GP Panel and include common gram-positive bacteria, drug resistance genes, bottle contaminants (Lactobacillus, Corynebacterium, Cutibacterium), gram-negative (Klebsiella pneumoniae) and fungal (Candida albicans) targets. Twenty replicates were tested at the beginning of the study when organism mixes were freshly made (time point T0). Ten replicates were tested for each additional temperature and time point.

# Ambient Storage Condition

All Mix 1 and Mix 2 analytes showed positivity rates of $5 9 5 \%$ when stored under ambient temperature for up to 1 month. The positivity rates and mean signals (nA) are summarized for Mix 1 and Mix 2 in Table 14 and Table 15, respectively.

Table 14: Mix 1 Ambient Storage Sample Stability   

<table><tr><td rowspan=2 colspan=1>TimePoint</td><td rowspan=2 colspan=1>n</td><td rowspan=2 colspan=1>Metric</td><td rowspan=1 colspan=6>Mix 1 Analytes Ambient Storage</td></tr><tr><td rowspan=1 colspan=1>cu</td><td rowspan=1 colspan=1>E!.</td><td rowspan=1 colspan=1> </td><td rowspan=1 colspan=1>ds</td><td rowspan=1 colspan=1>d s</td><td rowspan=1 colspan=1>Wun</td></tr><tr><td rowspan=2 colspan=1>Day 0</td><td rowspan=2 colspan=1>20</td><td rowspan=1 colspan=1>% Detected</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>Mean Signal (nA)</td><td rowspan=1 colspan=1>276.0</td><td rowspan=1 colspan=1>897.2</td><td rowspan=1 colspan=1>423.8</td><td rowspan=1 colspan=1>484.0</td><td rowspan=1 colspan=1>496.6</td><td rowspan=1 colspan=1>486.7</td></tr><tr><td rowspan=2 colspan=1>Day 1</td><td rowspan=2 colspan=1>10</td><td rowspan=1 colspan=1>% Detected</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>Mean Signal (nA)</td><td rowspan=1 colspan=1>309.3</td><td rowspan=1 colspan=1>794.3</td><td rowspan=1 colspan=1>396.6</td><td rowspan=1 colspan=1>492.4</td><td rowspan=1 colspan=1>361.4</td><td rowspan=1 colspan=1>447.7</td></tr><tr><td rowspan=2 colspan=1>Day 3</td><td rowspan=2 colspan=1>10</td><td rowspan=1 colspan=1>% Detected</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>Mean Signal (nA)</td><td rowspan=1 colspan=1>448.4</td><td rowspan=1 colspan=1>915.0</td><td rowspan=1 colspan=1>515.9</td><td rowspan=1 colspan=1>635.7</td><td rowspan=1 colspan=1>447.0</td><td rowspan=1 colspan=1>507.8</td></tr><tr><td rowspan=2 colspan=1>Day 5</td><td rowspan=2 colspan=1>10</td><td rowspan=1 colspan=1>% Detected</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>Mean Signal (nA)</td><td rowspan=1 colspan=1>438.1</td><td rowspan=1 colspan=1>901.5</td><td rowspan=1 colspan=1>489.7</td><td rowspan=1 colspan=1>673.0</td><td rowspan=1 colspan=1>491.7</td><td rowspan=1 colspan=1>526.0</td></tr><tr><td rowspan=2 colspan=1>Week 1</td><td rowspan=2 colspan=1>10</td><td rowspan=1 colspan=1>% Detected</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>Mean Signal (nA)</td><td rowspan=1 colspan=1>449.8</td><td rowspan=1 colspan=1>979.8</td><td rowspan=1 colspan=1>533.1</td><td rowspan=1 colspan=1>657.4</td><td rowspan=1 colspan=1>540.6</td><td rowspan=1 colspan=1>526.5</td></tr><tr><td rowspan=2 colspan=1>Week 2</td><td rowspan=2 colspan=1>10</td><td rowspan=1 colspan=1>% Detected</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>Mean Signal (nA)</td><td rowspan=1 colspan=1>295.2</td><td rowspan=1 colspan=1>996.1</td><td rowspan=1 colspan=1>479.4</td><td rowspan=1 colspan=1>624.2</td><td rowspan=1 colspan=1>520.0</td><td rowspan=1 colspan=1>452.3</td></tr><tr><td rowspan=2 colspan=1>Month 1</td><td rowspan=2 colspan=1>20</td><td rowspan=1 colspan=1>% Detected</td><td rowspan=1 colspan=1>95</td><td rowspan=1 colspan=1>95</td><td rowspan=1 colspan=1>95</td><td rowspan=1 colspan=1>95</td><td rowspan=1 colspan=1>95</td><td rowspan=1 colspan=1>95</td></tr><tr><td rowspan=1 colspan=1>Mean Signal (nA)</td><td rowspan=1 colspan=1>374.1</td><td rowspan=1 colspan=1>980.7</td><td rowspan=1 colspan=1>492.1</td><td rowspan=1 colspan=1>642.2</td><td rowspan=1 colspan=1>582.8</td><td rowspan=1 colspan=1>486.2</td></tr></table>

\* T he Enterococcus genus call is not sensitive to Enterococcus faecalis and is therefore forced by the software logic in the presence of Enterococcus faecalis signal as detected. Due to this, signal and detection rate for Enterococcus are not analyzed in this table.

Table 15: Mix 2 Ambient Storage Sample Stability   

<table><tr><td rowspan=2 colspan=1>Timepoint</td><td rowspan=2 colspan=1>n</td><td rowspan=2 colspan=1>Metric</td><td rowspan=1 colspan=9>Mix 2 Analytes Ambie nt Storage</td></tr><tr><td rowspan=1 colspan=1>Sh</td><td rowspan=1 colspan=1>En</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>sn S</td><td rowspan=1 colspan=1>s00s</td><td rowspan=1 colspan=1>onnd s</td><td rowspan=1 colspan=1>sq</td><td rowspan=1 colspan=1>oumud o</td><td rowspan=1 colspan=1>BunA</td></tr><tr><td rowspan=2 colspan=1>Day 0</td><td rowspan=2 colspan=1>20</td><td rowspan=1 colspan=1>% Detected</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>Mean Signal(nA)</td><td rowspan=1 colspan=1>230.1</td><td rowspan=1 colspan=1>548.0</td><td rowspan=1 colspan=1>511.3</td><td rowspan=1 colspan=1>458.9</td><td rowspan=1 colspan=1>766.0</td><td rowspan=1 colspan=1>669.0</td><td rowspan=1 colspan=1>281.1</td><td rowspan=1 colspan=1>135.8</td><td rowspan=1 colspan=1>741.8</td></tr><tr><td rowspan=2 colspan=1>Day 1</td><td rowspan=2 colspan=1>10</td><td rowspan=1 colspan=1>% Detected</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>Mean Signal(nA)</td><td rowspan=1 colspan=1>294.1</td><td rowspan=1 colspan=1>580.8</td><td rowspan=1 colspan=1>606.5</td><td rowspan=1 colspan=1>641.1</td><td rowspan=1 colspan=1>829.5</td><td rowspan=1 colspan=1>705.4</td><td rowspan=1 colspan=1>308.1</td><td rowspan=1 colspan=1>143.2</td><td rowspan=1 colspan=1>900.0</td></tr><tr><td rowspan=2 colspan=1>Day 3</td><td rowspan=2 colspan=1>10</td><td rowspan=1 colspan=1>% Detected</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>Mean Signal(nA)</td><td rowspan=1 colspan=1>292.9</td><td rowspan=1 colspan=1>592.0</td><td rowspan=1 colspan=1>654.8</td><td rowspan=1 colspan=1>751.2</td><td rowspan=1 colspan=1>876.3</td><td rowspan=1 colspan=1>756.4</td><td rowspan=1 colspan=1>335.4</td><td rowspan=1 colspan=1>141.2</td><td rowspan=1 colspan=1>854.5</td></tr><tr><td rowspan=2 colspan=1>Day 5</td><td rowspan=2 colspan=1>10</td><td rowspan=1 colspan=1>% Detected</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>Mean Signal(nA)</td><td rowspan=1 colspan=1>303.3</td><td rowspan=1 colspan=1>586.0</td><td rowspan=1 colspan=1>603.3</td><td rowspan=1 colspan=1>716.2</td><td rowspan=1 colspan=1>865.2</td><td rowspan=1 colspan=1>753.4</td><td rowspan=1 colspan=1>334.1</td><td rowspan=1 colspan=1>170.9</td><td rowspan=1 colspan=1>931.7</td></tr><tr><td rowspan=2 colspan=1>Week 1 10</td><td rowspan=2 colspan=1></td><td rowspan=1 colspan=1>% Detected</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>Mean Signal(nA)</td><td rowspan=1 colspan=1>322.9</td><td rowspan=1 colspan=1>608.5</td><td rowspan=1 colspan=1>722.3</td><td rowspan=1 colspan=1>537.1</td><td rowspan=1 colspan=1>813.2</td><td rowspan=1 colspan=1>739.5</td><td rowspan=1 colspan=1>358.7</td><td rowspan=1 colspan=1>177.5</td><td rowspan=1 colspan=1>950.1</td></tr><tr><td rowspan=2 colspan=1>Week 2</td><td rowspan=2 colspan=1>20</td><td rowspan=1 colspan=1>% Detected</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>95</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>Mean Signal(nA)</td><td rowspan=1 colspan=1>323.7</td><td rowspan=1 colspan=1>632.0</td><td rowspan=1 colspan=1>733.0</td><td rowspan=1 colspan=1>465.2</td><td rowspan=1 colspan=1>800.5</td><td rowspan=1 colspan=1>714.4</td><td rowspan=1 colspan=1>338.5</td><td rowspan=1 colspan=1>214.7</td><td rowspan=1 colspan=1>971.5</td></tr><tr><td rowspan=2 colspan=1>Month 120</td><td rowspan=2 colspan=1>20</td><td rowspan=1 colspan=1>% Detected</td><td rowspan=1 colspan=1>95</td><td rowspan=1 colspan=1>95</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>95</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>95</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>95</td></tr><tr><td rowspan=1 colspan=1>Mean Signal(nA)</td><td rowspan=1 colspan=1>262.4</td><td rowspan=1 colspan=1>478.0</td><td rowspan=1 colspan=1>672.9</td><td rowspan=1 colspan=1>284.0</td><td rowspan=1 colspan=1>720.3</td><td rowspan=1 colspan=1>593.0</td><td rowspan=1 colspan=1>278.2</td><td rowspan=1 colspan=1>198.8</td><td rowspan=1 colspan=1>865.7</td></tr></table>

All Mix 1 and Mix 2 analytes showed positivity rates of $2 9 5 \%$ when stored under refrigerated conditions $( 2 ^ { \circ } \mathrm { C } { - } 8 ^ { \circ } \mathrm { C } )$ for up to 1 month. The positivity rates and mean signals (nA) are summarized for Mix 1 and Mix 2 in Table 16 and Table 17, respectively.

Table 16: Mix 1, $\bf { 2 . 8 ^ { \circ } C }$ Storage Sample Stability   

<table><tr><td rowspan=2 colspan=1>TimePoint</td><td rowspan=2 colspan=1>n</td><td rowspan=2 colspan=1>Metric</td><td rowspan=1 colspan=6>Mix 1 Analytes 2°C-8°C Storage</td></tr><tr><td rowspan=1 colspan=1>cunn</td><td rowspan=1 colspan=1>*.</td><td rowspan=1 colspan=1>T </td><td rowspan=1 colspan=1>scyds</td><td rowspan=1 colspan=1>spd s</td><td rowspan=1 colspan=1>Wune</td></tr><tr><td rowspan=2 colspan=1>Day 1</td><td rowspan=2 colspan=1>20</td><td rowspan=1 colspan=1>% Detected</td><td rowspan=1 colspan=1>95</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>Mean Signal (nA)</td><td rowspan=1 colspan=1>377.9</td><td rowspan=1 colspan=1>909.5</td><td rowspan=1 colspan=1>489.8</td><td rowspan=1 colspan=1>578.7</td><td rowspan=1 colspan=1>515.3</td><td rowspan=1 colspan=1>473.4</td></tr><tr><td rowspan=2 colspan=1>Day 3</td><td rowspan=2 colspan=1>10</td><td rowspan=1 colspan=1>% Detected</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>Mean Signal (nA)</td><td rowspan=1 colspan=1>341.1</td><td rowspan=1 colspan=1>830.7</td><td rowspan=1 colspan=1>446.7</td><td rowspan=1 colspan=1>496.2</td><td rowspan=1 colspan=1>451.5</td><td rowspan=1 colspan=1>466.7</td></tr><tr><td rowspan=2 colspan=1>Day 5</td><td rowspan=2 colspan=1>10</td><td rowspan=1 colspan=1>% Detected</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>Mean Signal (nA)</td><td rowspan=1 colspan=1>427.8</td><td rowspan=1 colspan=1>900.4</td><td rowspan=1 colspan=1>500.2</td><td rowspan=1 colspan=1>566.2</td><td rowspan=1 colspan=1>461.2</td><td rowspan=1 colspan=1>493.6</td></tr><tr><td rowspan=2 colspan=1>Week 1</td><td rowspan=2 colspan=1>10</td><td rowspan=1 colspan=1>% Detected</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>Mean Signal (nA)</td><td rowspan=1 colspan=1>298.5</td><td rowspan=1 colspan=1>778.6</td><td rowspan=1 colspan=1>418.9</td><td rowspan=1 colspan=1>488.7</td><td rowspan=1 colspan=1>414.7</td><td rowspan=1 colspan=1>503.3</td></tr><tr><td rowspan=2 colspan=1>Week 2</td><td rowspan=2 colspan=1>10</td><td rowspan=1 colspan=1>% Detected</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>Mean Signal (nA)</td><td rowspan=1 colspan=1>316.8</td><td rowspan=1 colspan=1>1015.2</td><td rowspan=1 colspan=1>522.5</td><td rowspan=1 colspan=1>586.0</td><td rowspan=1 colspan=1>443.4</td><td rowspan=1 colspan=1>475.0</td></tr><tr><td rowspan=2 colspan=1>Month 1</td><td rowspan=2 colspan=1>10</td><td rowspan=1 colspan=1>% Detected</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>Mean Signal (nA)</td><td rowspan=1 colspan=1>414.8</td><td rowspan=1 colspan=1>1134.2</td><td rowspan=1 colspan=1>563.7</td><td rowspan=1 colspan=1>685.1</td><td rowspan=1 colspan=1>721.7</td><td rowspan=1 colspan=1>514.7</td></tr></table>

\* The Enterococcus genus call is not sensitive to Enterococcus faecalis and is therefore forced by the software logic in the presence of Enterococcus faecalis signal as detected. Due to this, signal and detection rate for Enterococcus are not analyzed in this table.

Table 17: Mix 2, $2 ^ { \circ } \mathbf { C } { \cdot } \mathbf { 8 } ^ { \circ } \mathbf { C }$ Storage Sample Stability   

<table><tr><td rowspan=2 colspan=1>TimePoint</td><td rowspan=2 colspan=1>n</td><td rowspan=2 colspan=1>Metric</td><td rowspan=1 colspan=9>Mix 2 Analytes 2°C-8°C Storage</td></tr><tr><td rowspan=1 colspan=1>ShccuS</td><td rowspan=1 colspan=1>enn</td><td rowspan=1 colspan=1>S</td><td rowspan=1 colspan=1>mnunnsi</td><td rowspan=1 colspan=1>Scs</td><td rowspan=1 colspan=1>ounud s</td><td rowspan=1 colspan=1>ccqS</td><td rowspan=1 colspan=1>pounndy</td><td rowspan=1 colspan=1>HunA</td></tr><tr><td rowspan=2 colspan=1>Day 1</td><td rowspan=2 colspan=1>10</td><td rowspan=1 colspan=1>% Detected</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>Mean Signal (nA)</td><td rowspan=1 colspan=1>310.7</td><td rowspan=1 colspan=1>553.0</td><td rowspan=1 colspan=1>582.6</td><td rowspan=1 colspan=1>620.7</td><td rowspan=1 colspan=1>809.1</td><td rowspan=1 colspan=1>710.4</td><td rowspan=1 colspan=1>316.9</td><td rowspan=1 colspan=1>138.2</td><td rowspan=1 colspan=1>890.5</td></tr><tr><td rowspan=2 colspan=1>Day 3</td><td rowspan=2 colspan=1>10</td><td rowspan=1 colspan=1>% Detected</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>Mean Signal (nA)</td><td rowspan=1 colspan=1>321.1</td><td rowspan=1 colspan=1>582.7</td><td rowspan=1 colspan=1>574.7</td><td rowspan=1 colspan=1>621.7</td><td rowspan=1 colspan=1>876.7</td><td rowspan=1 colspan=1>732.9</td><td rowspan=1 colspan=1>329.5</td><td rowspan=1 colspan=1>180.9</td><td rowspan=1 colspan=1>870.2</td></tr><tr><td rowspan=2 colspan=1>Day 5</td><td rowspan=2 colspan=1>10</td><td rowspan=1 colspan=1>% Detected</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>Mean Signal (nA)</td><td rowspan=1 colspan=1>302.9</td><td rowspan=1 colspan=1>589.1</td><td rowspan=1 colspan=1>597.5</td><td rowspan=1 colspan=1>623.6</td><td rowspan=1 colspan=1>869.2</td><td rowspan=1 colspan=1>753.2</td><td rowspan=1 colspan=1>334.3</td><td rowspan=1 colspan=1>119.0</td><td rowspan=1 colspan=1>863.6</td></tr><tr><td rowspan=2 colspan=1>Week 1</td><td rowspan=2 colspan=1>10</td><td rowspan=1 colspan=1>% Detected</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>Mean Signal (nA)</td><td rowspan=1 colspan=1>270.8</td><td rowspan=1 colspan=1>564.9</td><td rowspan=1 colspan=1>582.7</td><td rowspan=1 colspan=1>334.0</td><td rowspan=1 colspan=1>706.2</td><td rowspan=1 colspan=1>655.5</td><td rowspan=1 colspan=1>307.4</td><td rowspan=1 colspan=1>176.5</td><td rowspan=1 colspan=1>889.6</td></tr><tr><td rowspan=2 colspan=1>Week 2</td><td rowspan=2 colspan=1>10</td><td rowspan=1 colspan=1>% Detected</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>Mean Signal (nA)</td><td rowspan=1 colspan=1>282.7</td><td rowspan=1 colspan=1>593.2</td><td rowspan=1 colspan=1>611.2</td><td rowspan=1 colspan=1>400.0</td><td rowspan=1 colspan=1>698.3</td><td rowspan=1 colspan=1>596.1</td><td rowspan=1 colspan=1>301.5</td><td rowspan=1 colspan=1>153.5</td><td rowspan=1 colspan=1>951.0</td></tr><tr><td rowspan=2 colspan=1>Month 1</td><td rowspan=2 colspan=1>10</td><td rowspan=1 colspan=1>% Detected</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>Mean Signal (nA)</td><td rowspan=1 colspan=1>270.6</td><td rowspan=1 colspan=1>526.0</td><td rowspan=1 colspan=1>604.2</td><td rowspan=1 colspan=1>312.8</td><td rowspan=1 colspan=1>681.3</td><td rowspan=1 colspan=1>596.5</td><td rowspan=1 colspan=1>311.1</td><td rowspan=1 colspan=1>164.2</td><td rowspan=1 colspan=1>891.3</td></tr></table>

# Frozen $\left( { \leq } { - } 2 0 ^ { \circ } \mathrm { C } \right)$ Storage Condition

All Mix 1 and Mix 2 analytes showed positivity rates of $100 \%$ when stored under frozen conditions $( { \leq } \mathrm { - } 2 0 ^ { \circ } \mathrm { C } )$ for up to 1 month. The positivity rates and mean signals (nA) are summarized in Table 18 and Table 19, respectively. Mix 1 was also tested at the subsequent 3 month time point and continued to show $100 \%$ positivity for all analytes.

Table 18: Mix 1, $\mathbf { \Omega } _ { \mathbf { \Lambda } } \leq \mathbf { - } 2 \mathbf { 0 } ^ { \circ } \mathbf { C }$ Storage Sample Stability   

<table><tr><td rowspan=2 colspan=1>TimePoint</td><td rowspan=2 colspan=1>n</td><td rowspan=2 colspan=1>Metric</td><td rowspan=1 colspan=6>Mix 1 Analytes ≤-20°C Storage</td></tr><tr><td rowspan=1 colspan=1>cu</td><td rowspan=1 colspan=1>*S</td><td rowspan=1 colspan=1>s t</td><td rowspan=1 colspan=1>Sds</td><td rowspan=1 colspan=1>spd s</td><td rowspan=1 colspan=1>Aun</td></tr><tr><td rowspan=2 colspan=1>Week 1</td><td rowspan=2 colspan=1>10</td><td rowspan=1 colspan=1>% Detected</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>Mean Signal (nA)</td><td rowspan=1 colspan=1>447.0</td><td rowspan=1 colspan=1>986.2</td><td rowspan=1 colspan=1>537.1</td><td rowspan=1 colspan=1>651.5</td><td rowspan=1 colspan=1>504.6</td><td rowspan=1 colspan=1>545.1</td></tr><tr><td rowspan=2 colspan=1>Week 2</td><td rowspan=2 colspan=1>10</td><td rowspan=1 colspan=1>% Detected</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>Mean Signal (nA)</td><td rowspan=1 colspan=1>299.7</td><td rowspan=1 colspan=1>943.0</td><td rowspan=1 colspan=1>458.3</td><td rowspan=1 colspan=1>553.9</td><td rowspan=1 colspan=1>445.6</td><td rowspan=1 colspan=1>467.9</td></tr><tr><td rowspan=2 colspan=1>Month 1</td><td rowspan=2 colspan=1>10</td><td rowspan=1 colspan=1>% Detected</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>Mean Signal (nA)</td><td rowspan=1 colspan=1>373.2</td><td rowspan=1 colspan=1>934.2</td><td rowspan=1 colspan=1>490.5</td><td rowspan=1 colspan=1>581.8</td><td rowspan=1 colspan=1>661.4</td><td rowspan=1 colspan=1>474.3</td></tr><tr><td rowspan=2 colspan=1>Month 3</td><td rowspan=2 colspan=1>10</td><td rowspan=1 colspan=1>% Detected</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>Mean Signal (nA)</td><td rowspan=1 colspan=1>214.7</td><td rowspan=1 colspan=1>853.6</td><td rowspan=1 colspan=1>440.2</td><td rowspan=1 colspan=1>637.3</td><td rowspan=1 colspan=1>584.0</td><td rowspan=1 colspan=1>505.4</td></tr></table>

\* The Enterococcus genus call is not sensitive to Enterococcus faecalis and is therefore forced by the software logic in the presence of Enterococcus faecalis signal as detected. Due to this, signal and detection rate for Enterococcus are not analyzed in this table.

Table 19: Mix $2 , \leq - 2 0 ^ { \circ } \mathrm { C }$ Storage Sample Stability   

<table><tr><td rowspan=2 colspan=1>TimePoint</td><td rowspan=2 colspan=1>n</td><td rowspan=2 colspan=1>Metric</td><td rowspan=1 colspan=9>Mix 2 Analytes ≤-20°C Storage</td></tr><tr><td rowspan=1 colspan=1>shccu</td><td rowspan=1 colspan=1>Semm</td><td rowspan=1 colspan=1>scs</td><td rowspan=1 colspan=1>sun s</td><td rowspan=1 colspan=1>Sccs</td><td rowspan=1 colspan=1>nd </td><td rowspan=1 colspan=1>sqnS</td><td rowspan=1 colspan=1>pony</td><td rowspan=1 colspan=1>HunA</td></tr><tr><td rowspan=2 colspan=1>Week 1</td><td rowspan=2 colspan=1>20</td><td rowspan=1 colspan=1>% Detected</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>95</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>Mean Signal(nA)</td><td rowspan=1 colspan=1>282.8</td><td rowspan=1 colspan=1>559.4</td><td rowspan=1 colspan=1>608.2</td><td rowspan=1 colspan=1>504.0</td><td rowspan=1 colspan=1>763.9</td><td rowspan=1 colspan=1>663.5</td><td rowspan=1 colspan=1>316.8</td><td rowspan=1 colspan=1>174.9</td><td rowspan=1 colspan=1>873.6</td></tr><tr><td rowspan=2 colspan=1>Week2</td><td rowspan=2 colspan=1>10</td><td rowspan=1 colspan=1>% Detected</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>Mean Signal(nA)</td><td rowspan=1 colspan=1>282.7</td><td rowspan=1 colspan=1>589.6</td><td rowspan=1 colspan=1>558.8</td><td rowspan=1 colspan=1>445.4</td><td rowspan=1 colspan=1>807.2</td><td rowspan=1 colspan=1>671.3</td><td rowspan=1 colspan=1>305.0</td><td rowspan=1 colspan=1>197.8</td><td rowspan=1 colspan=1>951.3</td></tr><tr><td rowspan=2 colspan=1>Month 1</td><td rowspan=2 colspan=1>10</td><td rowspan=1 colspan=1>% Detected</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>Mean Signal(nA)</td><td rowspan=1 colspan=1>281.0</td><td rowspan=1 colspan=1>482.2</td><td rowspan=1 colspan=1>614.4</td><td rowspan=1 colspan=1>317.8</td><td rowspan=1 colspan=1>813.4</td><td rowspan=1 colspan=1>630.4</td><td rowspan=1 colspan=1>261.3</td><td rowspan=1 colspan=1>155.3</td><td rowspan=1 colspan=1>847.4</td></tr></table>

# Frozen $\mathrm { ( { \leq } - 7 0 ^ { \circ } C ) }$ Storage Condition

Mix 1 analytes showed positivity rates of $2 9 5 \%$ when stored under frozen conditions $( { \leq } { - } 7 0 ^ { \circ } \mathrm { C } )$ for up to 3 months with the exception of $C .$ acnes and $L$ . casei, which observed a positivity rate of $90 \%$ on the first week of storage at $\leq -$ $7 0 ^ { \circ } \mathrm { C }$ . Per the study acceptance criteria, subsequent time points were evaluated. At all other tested time points up to 3 months, $C .$ acnes and $L$ . casei showed $2 9 5 \%$ positive detection indicating that the lower detection rates at the week 1 time point were not due to specimen stability issues. The observed positivity rates and mean signals (nA) for Mix 1 analytes are summarized in Table 20 below.

Table 20: Mix 1, $\bf { \Omega } _ { 3 } { \leq } { - } 7 0 ^ { \circ } C$ Storage Sample Stability   

<table><tr><td rowspan=2 colspan=1>TimePoint</td><td rowspan=2 colspan=1>n</td><td rowspan=2 colspan=1>Metric</td><td rowspan=1 colspan=6>Mix 1 Analytes ≤-70°C Storage</td></tr><tr><td rowspan=1 colspan=1>cu</td><td rowspan=1 colspan=1>*S</td><td rowspan=1 colspan=1>s t</td><td rowspan=1 colspan=1>syds</td><td rowspan=1 colspan=1>spd s</td><td rowspan=1 colspan=1>Aun</td></tr><tr><td rowspan=2 colspan=1>Week 1</td><td rowspan=2 colspan=1>20</td><td rowspan=1 colspan=1>% Detected</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>95</td></tr><tr><td rowspan=1 colspan=1>Mean Signal(nA)</td><td rowspan=1 colspan=1>328.8</td><td rowspan=1 colspan=1>796.8</td><td rowspan=1 colspan=1>448.6</td><td rowspan=1 colspan=1>465.6</td><td rowspan=1 colspan=1>367.4</td><td rowspan=1 colspan=1>473.9</td></tr><tr><td rowspan=2 colspan=1>Week2</td><td rowspan=2 colspan=1>10</td><td rowspan=1 colspan=1>% Detected</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>Mean Signal(nA)</td><td rowspan=1 colspan=1>275.3</td><td rowspan=1 colspan=1>795.4</td><td rowspan=1 colspan=1>385.8</td><td rowspan=1 colspan=1>505.3</td><td rowspan=1 colspan=1>420.9</td><td rowspan=1 colspan=1>414.6</td></tr><tr><td rowspan=2 colspan=1>Month 1</td><td rowspan=2 colspan=1>10</td><td rowspan=1 colspan=1>% Detected</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>Mean Signal(nA)</td><td rowspan=1 colspan=1>308.3</td><td rowspan=1 colspan=1>872.4</td><td rowspan=1 colspan=1>428.5</td><td rowspan=1 colspan=1>490.3</td><td rowspan=1 colspan=1>590.9</td><td rowspan=1 colspan=1>469.8</td></tr><tr><td rowspan=2 colspan=1>Month3</td><td rowspan=2 colspan=1>20</td><td rowspan=1 colspan=1>% Detected</td><td rowspan=1 colspan=1>95</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>Mean Signal(nA)</td><td rowspan=1 colspan=1>206.3</td><td rowspan=1 colspan=1>795.0</td><td rowspan=1 colspan=1>405.4</td><td rowspan=1 colspan=1>514.7</td><td rowspan=1 colspan=1>481.8</td><td rowspan=1 colspan=1>500.2</td></tr></table>

\* The Enterococcus genus call is not sensitive to Enterococcus faecalis and is therefore forced by the software logic in the presence of Enterococcus faecalis signal as detected. Due to this, signal and detection rate for Enterococcus are not analyzed in this table.

All Mix 2 analytes showed positivity rates of $100 \%$ when stored under frozen conditions $( { \leq } { - } 7 0 ^ { \circ } \mathrm { C } )$ for up to 1 month. The positivity rates and mean signals (nA) for Mix 2 analytes are summarized in Table 21.

Table 21: Mix $2 , { \le } \mathrm { - } 7 { \bf 0 } ^ { \circ } { \bf C }$ Storage Sample Stability   

<table><tr><td rowspan=2 colspan=1>TimePoint</td><td rowspan=2 colspan=1>n</td><td rowspan=2 colspan=1>Metric</td><td rowspan=1 colspan=9>Mix 2 Analytes ≤-70°C Storage</td></tr><tr><td rowspan=1 colspan=1>Shcccu</td><td rowspan=1 colspan=1>Sn</td><td rowspan=1 colspan=1>sc0</td><td rowspan=1 colspan=1>sn S</td><td rowspan=1 colspan=1>Sh0us</td><td rowspan=1 colspan=1>oounud&#x27;s</td><td rowspan=1 colspan=1>ccqn</td><td rowspan=1 colspan=1>nuoumudI</td><td rowspan=1 colspan=1>umA</td></tr><tr><td rowspan=2 colspan=1>Week 1</td><td rowspan=2 colspan=1>10</td><td rowspan=1 colspan=1>% Detected</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>Mean Signal(nA)</td><td rowspan=1 colspan=1>295.6</td><td rowspan=1 colspan=1>587.2</td><td rowspan=1 colspan=1>591.5</td><td rowspan=1 colspan=1>466.9</td><td rowspan=1 colspan=1>733.6</td><td rowspan=1 colspan=1>677.2</td><td rowspan=1 colspan=1>318.6</td><td rowspan=1 colspan=1>143.0</td><td rowspan=1 colspan=1>893.8</td></tr><tr><td rowspan=2 colspan=1>Week2</td><td rowspan=2 colspan=1>10</td><td rowspan=1 colspan=1>% Detected</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>Mean Signal(nA)</td><td rowspan=1 colspan=1>278.8</td><td rowspan=1 colspan=1>507.3</td><td rowspan=1 colspan=1>555.2</td><td rowspan=1 colspan=1>431.2</td><td rowspan=1 colspan=1>793.7</td><td rowspan=1 colspan=1>643.4</td><td rowspan=1 colspan=1>297.5</td><td rowspan=1 colspan=1>167.0</td><td rowspan=1 colspan=1>885.2</td></tr><tr><td rowspan=2 colspan=1>Month 1</td><td rowspan=2 colspan=1>10</td><td rowspan=1 colspan=1>% Detected</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>Mean Signal(nA)</td><td rowspan=1 colspan=1>267.2</td><td rowspan=1 colspan=1>498.5</td><td rowspan=1 colspan=1>609.1</td><td rowspan=1 colspan=1>349.2</td><td rowspan=1 colspan=1>775.5</td><td rowspan=1 colspan=1>617.6</td><td rowspan=1 colspan=1>306.6</td><td rowspan=1 colspan=1>182.3</td><td rowspan=1 colspan=1>862.9</td></tr></table>

The positivity rates and mean signals were calculated for the two organisms (Staphylococcus lugdunensis and Streptococcus pyogenes) grown to bottle positivity and incubated 12 hours post positivity. The positivity rates for Staphylococcus lugdunensis and Streptococcus pyogenes at bottle ring and 12 hours post bottle ring was $100 \%$ for both organisms. Results are summarized in Table 22.

Table 22: Bottle Ring and 12 hours Post Bottle Ring   

<table><tr><td rowspan=2 colspan=1>Blood culture</td><td rowspan=2 colspan=1>Metric</td><td rowspan=1 colspan=2>Bottle ring</td><td rowspan=1 colspan=2>Bottle ring + 12 hrs</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus</td><td rowspan=1 colspan=1>S. lugdunensis</td><td rowspan=1 colspan=1>Staphylococcus</td><td rowspan=1 colspan=1>S. lugdunensis</td></tr><tr><td rowspan=2 colspan=1>Staphylococcuslugdunensis</td><td rowspan=1 colspan=1>% Detected</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Mean Signal (nA)</td><td rowspan=1 colspan=1>617.7</td><td rowspan=1 colspan=1>833.8</td><td rowspan=1 colspan=1>537.8</td><td rowspan=1 colspan=1>800.1</td></tr><tr><td rowspan=1 colspan=1>Blood culture</td><td rowspan=1 colspan=1>Metric</td><td rowspan=1 colspan=1>Streptococcus</td><td rowspan=1 colspan=1>S. pyogenes</td><td rowspan=1 colspan=1>Streptococcus</td><td rowspan=1 colspan=1>S. pyogenes</td></tr><tr><td rowspan=2 colspan=1>Streptococcuspyogenes</td><td rowspan=1 colspan=1>% Detected</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Mean Signal (nA)</td><td rowspan=1 colspan=1>591.8</td><td rowspan=1 colspan=1>936.9</td><td rowspan=1 colspan=1>666.1</td><td rowspan=1 colspan=1>1045.9</td></tr></table>

Results demonstrated that specimens can be stored up to 1 month under the following temperature conditions without adversely impacting the performance of the BCID-GP Panel:

• Ambient temperature • Refrigerated $( 2 ^ { \circ } \mathrm { C } { - } 8 ^ { \circ } \mathrm { C } )$ Frozen $( \le - 2 0 ^ { \circ } \mathrm { C }$ and $\le - 7 0 ^ { \circ } \mathrm { C }$ )

Specimens can also be tested when incubated up to 12 hours after bottle ring in a continuously monitoring blood culture device.

The specimen stability claims are summarized in the table below.

<table><tr><td rowspan=1 colspan=1>Storage Condition</td><td rowspan=1 colspan=1>Spe cime n Stability</td></tr><tr><td rowspan=1 colspan=1>Ambient</td><td rowspan=1 colspan=1>1 month</td></tr><tr><td rowspan=1 colspan=1>Refrigerated (2°C-8°C)</td><td rowspan=1 colspan=1>1 month</td></tr><tr><td rowspan=1 colspan=1>Frozen (≤-20°)</td><td rowspan=1 colspan=1>1 month</td></tr><tr><td rowspan=1 colspan=1>Frozen (≤-70)</td><td rowspan=1 colspan=1>1 month</td></tr><tr><td rowspan=1 colspan=1>Post Bottle Ring</td><td rowspan=1 colspan=1>12 hours</td></tr></table>

# Freeze-Thaw Study

A study was performed to assess the tolerance of the ePlex BCID-GP to correctly identify specimens containing bacterial and fungal organisms that have gone through one or two freeze/thaw cycles prior to testing. One hundred and ten positive clinical samples that were tested fresh during the BCID-GP Panel Clinical Performance Study were selected to best represent a variety of organisms in clinical samples. The samples were frozen at ≤-

$7 0 \mathrm { { } ^ { \circ } C }$ and then thawed once or twice prior to re-testing with the BCID-GP Panel.

The positive agreement with the fresh condition at each freeze-thaw cycle was calculated and was above $9 5 \%$ for each condition. Table 23 below summarizes the concordance at each freeze-thaw cycle with a $9 5 \%$ confidence level.

Table 23: Concordance of Results with Fresh Condition   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1x Freeze Thaw</td><td rowspan=1 colspan=1>2x Freeze Thaw</td></tr><tr><td rowspan=1 colspan=1>Concordant Samples</td><td rowspan=1 colspan=1>98</td><td rowspan=1 colspan=1>101</td></tr><tr><td rowspan=1 colspan=1>Discordant Samples</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>Total Samples Tested</td><td rowspan=1 colspan=1>102</td><td rowspan=1 colspan=1>102</td></tr><tr><td rowspan=1 colspan=1>Concordance</td><td rowspan=1 colspan=1>96.08%</td><td rowspan=1 colspan=1>99.02%</td></tr><tr><td rowspan=1 colspan=1>CI Low</td><td rowspan=1 colspan=1>90.35%</td><td rowspan=1 colspan=1>94.65%</td></tr><tr><td rowspan=1 colspan=1>CI High</td><td rowspan=1 colspan=1>98.46%</td><td rowspan=1 colspan=1>99.83%</td></tr></table>

Results demonstrate that the BCID-GP Panel can detect the same organisms in samples that were tested fresh and in samples that have been frozen and thawed up to 2 times.

# In-Cartridge Sample Stability Study

An analytical study was carried out to demonstrate performance under the following conditions:

1) Condition 1 (Open Pouch): the cartridge is stable for 2 hours after the pouch has been opened and the unloaded cartridge is exposed to the environment.   
2) Condition 2 (In-Consumable Sample Stability): the cartridge can be stored at room temperature for at least 2 hours after the sample has been loaded prior to running the cartridge in an ePlex bay.

A test mix of five organisms was used in the study. The selected organisms represent each of the eight multiplex primer pools on the BCID-GP Panel and produce nine target results. Each organism was tested at a concentration approximating bottle positivity. A description of the test mix used for this study is summarized in the Table below.

Table 24: Representative Organisms Contained in Test Mix   

<table><tr><td rowspan=1 colspan=1>Organis m in Test Mix</td><td rowspan=1 colspan=1>Conce ntration inMix (CFU/mL)</td><td rowspan=1 colspan=1>PrimerPool</td><td rowspan=1 colspan=1>ePlex BCID-GP PanelExpected Result</td></tr><tr><td rowspan=2 colspan=1>Streptococcus pneumoniae</td><td rowspan=2 colspan=1>4 x 107</td><td rowspan=2 colspan=1>5</td><td rowspan=1 colspan=1>Streptococcus pneumoniae</td></tr><tr><td rowspan=1 colspan=1>Streptococcus*</td></tr><tr><td rowspan=3 colspan=1>Enterococcus faecium/ vanA</td><td rowspan=3 colspan=1>4 x 107</td><td rowspan=3 colspan=1>1, 3, 4</td><td rowspan=1 colspan=1>Enterococcus faecium</td></tr><tr><td rowspan=1 colspan=1>Enterococcus*</td></tr><tr><td rowspan=1 colspan=1>vanA</td></tr><tr><td rowspan=1 colspan=1>Candida albicans</td><td rowspan=1 colspan=1>1 x 106</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>Pan Candida</td></tr><tr><td rowspan=1 colspan=1>Klebsiella pneumoniae</td><td rowspan=1 colspan=1>5 x 108</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>Pan Gram-Negative</td></tr><tr><td rowspan=2 colspan=1>Staphylococcus aureus</td><td rowspan=2 colspan=1>2 x 107</td><td rowspan=2 colspan=1>6,8</td><td rowspan=1 colspan=1>Staphylococcus aureus</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus*</td></tr></table>

\* Software logic defines that when a Staphylococcus, Streptococcus or Enterococcus species is detected, the corresponding genus call is detected.

The open pouch stability study compared 20 replicates of cartridges that were kept in closed pouches until the cartridges were loaded with sample vs. 20 replicates of cartridges where the pouches were opened and the cartridges were exposed to ambient room temperature conditions for up to 2 hours before being loaded with sample.

To demonstrate that the consumable can be held at room temperature for at least 2 hours after the sample has been loaded (prior to running the consumable on an ePlex bay), 20 consumables were loaded with the organism mix, within 20 minutes of opening the pouch. The consumables were kept for 2 hours at room temperature and then run in the ePlex instrument.

A control condition was run wherein 20 replicates of the organism mix were loaded within 20 minutes of pouch opening and the consumables were run on the ePlex instrument within 20 minutes of sample loading. The data from the test conditions were compared to the control condition and all cartridges were run on the ePlex instrument following packing insert instructions.

The results of the study which assessed the interim storage conditions are summarized in Table 25. All representative targets were detected in all replicates (20/20) in the control and the two test conditions (Open Pouch Stability Condition and In-Consumable Sample Stability Condition).

Table 25: Detection Rates for All Conditions Tested   

<table><tr><td rowspan=1 colspan=4>Detection Rate</td></tr><tr><td rowspan=1 colspan=1>Target</td><td rowspan=1 colspan=1>ControlCondition</td><td rowspan=1 colspan=1>Ope n PouchStabilityCondition</td><td rowspan=1 colspan=1>In-Cons umableSample StabilityCondition</td></tr><tr><td rowspan=1 colspan=1>Enterococcus</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Enterococcus faecium</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus aureus</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Streptococcus</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Streptococcus pneumoniae</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Pan Candida</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Pan Gram-Negative</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>vanA</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td></tr></table>

The mean signal, standard deviation (SD), and mean minus 2SD were calculated for each target and test condition (Table 26). Mean minus 2SD values were all well above the assigned cutoff for each target/condition.

Table 26: Signals Remain Above Assay Cut-Offs in All Test Conditions   

<table><tr><td colspan="1" rowspan="1">Target</td><td colspan="1" rowspan="1">Test Condition</td><td colspan="1" rowspan="1">MeanSignal(nA)</td><td colspan="1" rowspan="1">Std Dev(nA)</td><td colspan="1" rowspan="1">Meanminus2SD (nA)</td><td colspan="1" rowspan="1">TargetCutoff(nA)</td></tr><tr><td colspan="1" rowspan="3">Enterococcus</td><td colspan="1" rowspan="1">Control</td><td colspan="1" rowspan="1">313.80</td><td colspan="1" rowspan="1">70.76</td><td colspan="1" rowspan="1">172.29</td><td colspan="1" rowspan="3">20</td></tr><tr><td colspan="1" rowspan="1">In-Consumable SampleStability Test</td><td colspan="1" rowspan="1">297.93</td><td colspan="1" rowspan="1">85.45</td><td colspan="1" rowspan="1">127.02</td></tr><tr><td colspan="1" rowspan="1">Open Pouch Stability Test</td><td colspan="1" rowspan="1">329.48</td><td colspan="1" rowspan="1">43.90</td><td colspan="1" rowspan="1">241.68</td></tr><tr><td colspan="1" rowspan="3">Enterococcus faecium</td><td colspan="1" rowspan="1">Control</td><td colspan="1" rowspan="1">561.66</td><td colspan="1" rowspan="1">89.84</td><td colspan="1" rowspan="1">381.98</td><td colspan="1" rowspan="3">20*</td></tr><tr><td colspan="1" rowspan="1">In-Consumable Sample StabilityTest</td><td colspan="1" rowspan="1">539.41</td><td colspan="1" rowspan="1">113.24</td><td colspan="1" rowspan="1">312.92</td></tr><tr><td colspan="1" rowspan="1">Open Pouch Stability Test</td><td colspan="1" rowspan="1">574.44</td><td colspan="1" rowspan="1">74.49</td><td colspan="1" rowspan="1">425.46</td></tr><tr><td colspan="1" rowspan="3">Staphylococcus</td><td colspan="1" rowspan="1">Control</td><td colspan="1" rowspan="1">704.29</td><td colspan="1" rowspan="1">109.10</td><td colspan="1" rowspan="1">486.09</td><td colspan="1" rowspan="3">20</td></tr><tr><td colspan="1" rowspan="1">In-Consumable SampleStability Test</td><td colspan="1" rowspan="1">694.93</td><td colspan="1" rowspan="1">119.17</td><td colspan="1" rowspan="1">456.59</td></tr><tr><td colspan="1" rowspan="1">Open Pouch Stability Test</td><td colspan="1" rowspan="1">679.21</td><td colspan="1" rowspan="1">129.69</td><td colspan="1" rowspan="1">419.83</td></tr><tr><td colspan="1" rowspan="3">Staphylococcus aureus</td><td colspan="1" rowspan="1">Control</td><td colspan="1" rowspan="1">673.25</td><td colspan="1" rowspan="1">225.07</td><td colspan="1" rowspan="1">223.10</td><td colspan="1" rowspan="3">15</td></tr><tr><td colspan="1" rowspan="1">In-Consumable SampleStability Test</td><td colspan="1" rowspan="1">681.33</td><td colspan="1" rowspan="1">201.87</td><td colspan="1" rowspan="1">277.59</td></tr><tr><td colspan="1" rowspan="1">Open Pouch Stability Test</td><td colspan="1" rowspan="1">739.48</td><td colspan="1" rowspan="1">269.42</td><td colspan="1" rowspan="1">200.63</td></tr><tr><td colspan="1" rowspan="2">Streptococcus</td><td colspan="1" rowspan="1">Control</td><td colspan="1" rowspan="1">682.97</td><td colspan="1" rowspan="1">187.04</td><td colspan="1" rowspan="1">308.88</td><td colspan="1" rowspan="2">30</td></tr><tr><td colspan="1" rowspan="1">In-Consumable SampleStability Test</td><td colspan="1" rowspan="1">652.05</td><td colspan="1" rowspan="1">202.44</td><td colspan="1" rowspan="1">247.17</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Open Pouch Stability Test</td><td colspan="1" rowspan="1">785.78</td><td colspan="1" rowspan="1">132.81</td><td colspan="1" rowspan="1">520.15</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="3">Streptococcuspneumoniae</td><td colspan="1" rowspan="1">Control</td><td colspan="1" rowspan="1">715.97</td><td colspan="1" rowspan="1">118.33</td><td colspan="1" rowspan="1">479.30</td><td colspan="1" rowspan="3">20</td></tr><tr><td colspan="1" rowspan="1">In-Consumable SampleStability Test</td><td colspan="1" rowspan="1">682.68</td><td colspan="1" rowspan="1">132.63</td><td colspan="1" rowspan="1">417.41</td></tr><tr><td colspan="1" rowspan="1">Open Pouch Stability Test</td><td colspan="1" rowspan="1">757.41</td><td colspan="1" rowspan="1">90.83</td><td colspan="1" rowspan="1">575.76</td></tr><tr><td colspan="1" rowspan="3">Pan Candida</td><td colspan="1" rowspan="1">Control</td><td colspan="1" rowspan="1">350.90</td><td colspan="1" rowspan="1">68.35</td><td colspan="1" rowspan="1">214.20</td><td colspan="1" rowspan="3">40</td></tr><tr><td colspan="1" rowspan="1">In-Consumable SampleStability Test</td><td colspan="1" rowspan="1">338.80</td><td colspan="1" rowspan="1">76.92</td><td colspan="1" rowspan="1">184.96</td></tr><tr><td colspan="1" rowspan="1">Open Pouch Stability Test</td><td colspan="1" rowspan="1">347.09</td><td colspan="1" rowspan="1">68.26</td><td colspan="1" rowspan="1">210.56</td></tr><tr><td colspan="1" rowspan="3">Pan Gram-Negative</td><td colspan="1" rowspan="1">Control</td><td colspan="1" rowspan="1">261.41</td><td colspan="1" rowspan="1">88.85</td><td colspan="1" rowspan="1">83.70</td><td colspan="1" rowspan="3">40</td></tr><tr><td colspan="1" rowspan="1">In-Consumable SampleStability Test</td><td colspan="1" rowspan="1">248.32</td><td colspan="1" rowspan="1">50.92</td><td colspan="1" rowspan="1">146.49</td></tr><tr><td colspan="1" rowspan="1">Open Pouch Stability Test</td><td colspan="1" rowspan="1">296.43</td><td colspan="1" rowspan="1">66.03</td><td colspan="1" rowspan="1">164.37</td></tr><tr><td colspan="1" rowspan="3">VanA</td><td colspan="1" rowspan="1">Control</td><td colspan="1" rowspan="1">935.81</td><td colspan="1" rowspan="1">102.70</td><td colspan="1" rowspan="1">730.40</td><td colspan="1" rowspan="3">20</td></tr><tr><td colspan="1" rowspan="1">In-Consumable SampleStability Test</td><td colspan="1" rowspan="1">916.30</td><td colspan="1" rowspan="1">217.64</td><td colspan="1" rowspan="1">481.02</td></tr><tr><td colspan="1" rowspan="1">Open Pouch Stability Test</td><td colspan="1" rowspan="1">955.10</td><td colspan="1" rowspan="1">102.58</td><td colspan="1" rowspan="1">749.93</td></tr></table>

\*Enterococcus faecium cutoff is 20nA when signal for Enterococcus is $\geq 2 0 n A$ . An Enterococcus faecium signal that is ${ \ge } 4 0 0 \mathrm { n A }$ is independent of the Enterococcus signal.

# e. Growth and Detection Study

A study was performed to establish the range of expected organism concentrations present in incubated blood cultures at bottle positivity (i.e., bottle “ring) and eight hours after bottle positivity/bottle “ring.” A total of 22 representative organisms (from organism glycerol stocks) were tested in this study. A minimum of eight organism-appropriate blood culture bottles were procured and decontaminated with $10 \%$ bleach and $70 \%$ isopropyl alcohol. The volume of negative human whole blood, determined by the manufacturer’s instructions, was inoculated into each bottle and grown to bottle positivity and bottle positivity $+ 8$ hours. At the time of positivity (and/or eight hours after positivity), the blood culture was removed from the instrument for determination of organism concentration (CFU/mL using a plate count procedure) and ePlex BCID panel testing.

The following table summarizes the concentration of organism (CFU/mL) determined for a representative panel of 22 isolates. The concentration of Corynebacterium striatum at bottle positivity was observed to be the lowest of all the bacterial species tested so a second round of growth was tested to confirm the reproducibility of the counts. In addition, the Corynebacterium genus is large (at least 88 species) so additional species were cultured and counted in order to ensure a representative average. Table 28 lists the concentration for multiple species of Corynebacterium at bottle positivity to obtain an average concentration.

Table 27: Average Bottle Positive Concentration by Genus   

<table><tr><td rowspan=1 colspan=1>Analyte</td><td rowspan=1 colspan=1>Average PositiveConce ntration (CFU/mL)</td><td rowspan=1 colspan=1>Average Positive +8hrsConce ntration (CFU/mL)</td></tr><tr><td rowspan=1 colspan=1>Serratia</td><td rowspan=1 colspan=1>1.2x109</td><td rowspan=1 colspan=1>2.2 x109</td></tr><tr><td rowspan=1 colspan=1>Escherichia</td><td rowspan=1 colspan=1>2.3 x108</td><td rowspan=1 colspan=1>1.5 x109</td></tr><tr><td rowspan=1 colspan=1>Enterobacter</td><td rowspan=1 colspan=1>2.8 x108</td><td rowspan=1 colspan=1>7.7 x108</td></tr><tr><td rowspan=1 colspan=1>Neisseria</td><td rowspan=1 colspan=1>3.2 x107</td><td rowspan=1 colspan=1>2.1 x108</td></tr><tr><td rowspan=1 colspan=1>Haemophilus</td><td rowspan=1 colspan=1>6.9 x108</td><td rowspan=1 colspan=1>1.15 x109</td></tr><tr><td rowspan=1 colspan=1>Pseudomonas</td><td rowspan=1 colspan=1>1.6 x108</td><td rowspan=1 colspan=1>8.4 x108</td></tr><tr><td rowspan=1 colspan=1>Fusobacterium*</td><td rowspan=1 colspan=1>6.5 x107</td><td rowspan=1 colspan=1>4.9 x108</td></tr><tr><td rowspan=1 colspan=1>Acinetobacter</td><td rowspan=1 colspan=1>4.4 x108</td><td rowspan=1 colspan=1>3.8 x108</td></tr><tr><td rowspan=1 colspan=1>Klebsiella</td><td rowspan=1 colspan=1>9.3 x108</td><td rowspan=1 colspan=1>1.5 x109</td></tr><tr><td rowspan=1 colspan=1>Bacteroides*</td><td rowspan=1 colspan=1>4.7x108</td><td rowspan=1 colspan=1>6.7x109</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus</td><td rowspan=1 colspan=1>2.8 x107</td><td rowspan=1 colspan=1>2.1 x107</td></tr><tr><td rowspan=1 colspan=1>Lactobacillus*</td><td rowspan=1 colspan=1>4.8 x107</td><td rowspan=1 colspan=1>3.1 x1011</td></tr><tr><td rowspan=1 colspan=1>Streptococcus</td><td rowspan=1 colspan=1>4.1 x107</td><td rowspan=1 colspan=1>4.0 x108</td></tr><tr><td rowspan=1 colspan=1>Enterococcus</td><td rowspan=1 colspan=1>4.9 x107</td><td rowspan=1 colspan=1>3.6 x107</td></tr><tr><td rowspan=1 colspan=1>Corynebacterium</td><td rowspan=1 colspan=1>4.5 x106</td><td rowspan=1 colspan=1>2.7 x109</td></tr><tr><td rowspan=1 colspan=1>Propionibacterium*(Cutibacterium)</td><td rowspan=1 colspan=1>3.0 x108</td><td rowspan=1 colspan=1>3.5 x108</td></tr><tr><td rowspan=1 colspan=1>Candida</td><td rowspan=1 colspan=1>1.6 x106</td><td rowspan=1 colspan=1>1.4x106</td></tr><tr><td rowspan=1 colspan=1>Cryptococcus</td><td rowspan=1 colspan=1>1.3 x107</td><td rowspan=1 colspan=1>6.5 x107</td></tr><tr><td rowspan=1 colspan=1>Malassezia</td><td rowspan=1 colspan=1>1.0 x107</td><td rowspan=1 colspan=1>6.3 x106</td></tr><tr><td rowspan=1 colspan=1>Fusarium</td><td rowspan=1 colspan=1>9.6 x108</td><td rowspan=1 colspan=1>8.8 x108</td></tr><tr><td rowspan=1 colspan=1>Rhodotorula</td><td rowspan=1 colspan=1>1.6 x106</td><td rowspan=1 colspan=1>4.2 x106</td></tr><tr><td rowspan=1 colspan=1>Trichosporon</td><td rowspan=1 colspan=1>2.1x106</td><td rowspan=1 colspan=1>2.9 x106</td></tr></table>

\*Denotes a genus cultured under anaerobic conditions

Table 28: Average Bottle Positive Concentration for Corynebacterium   

<table><tr><td rowspan=1 colspan=1>Analyte</td><td rowspan=1 colspan=1>Strain ID</td><td rowspan=1 colspan=1>Average PositiveConce ntration (CFU/mL)</td></tr><tr><td rowspan=1 colspan=1>C. striatum (1)</td><td rowspan=1 colspan=1>ATCC BAA-1293</td><td rowspan=1 colspan=1>4.5 x106</td></tr><tr><td rowspan=1 colspan=1>C. striatum (2)</td><td rowspan=1 colspan=1>ATCC BAA-1293</td><td rowspan=1 colspan=1>8.1 x 106</td></tr><tr><td rowspan=1 colspan=1>C. minutissium</td><td rowspan=1 colspan=1>ATCC23348</td><td rowspan=1 colspan=1>1.1 x 107</td></tr><tr><td rowspan=1 colspan=1>C. timonense</td><td rowspan=1 colspan=1>CCUG64728</td><td rowspan=1 colspan=1>2.3 x 107</td></tr><tr><td rowspan=1 colspan=1>C. falsenii</td><td rowspan=1 colspan=1>ATCC BAA-596</td><td rowspan=1 colspan=1>6.4 x 107</td></tr><tr><td rowspan=1 colspan=1>C. coyleae</td><td rowspan=1 colspan=1>ATCC700219</td><td rowspan=1 colspan=1>1.0 x 107</td></tr><tr><td rowspan=1 colspan=1>EY</td><td rowspan=1 colspan=1>Average =</td><td rowspan=1 colspan=1>2.0 x 107</td></tr></table>

# f. Analytical Reactivity (Inclusivity):

The analytical reactivity of the ePlex BCID-GP Panel was evaluated with a collection of 489 bacterial and fungal organisms that represent the diversity of the ePlex BCID-GP Panel analytes, including antimicrobial resistance genes. Isolates were selected to represent relevant species or serotypes and selection with specific inclusion of more commonly encountered species and known human pathogens. When possible, in silico analysis of sequence data was used to make predictions of assay reactivity for less common species that were not tested but that may be detected by the ePlex BCID-GP Panel.

Each isolate was initially tested in blood culture matrix at a concentration consistent with the levels of organism enumerated from blood cultures at the time of positivity (see Growth and Detection section above) and were tested in triplicate. If an isolate was not detected initially, the sample was retested at higher concentrations. If detected at the higher concentration(s), the species/isolate is indicated as detected with reduced sensitivity and the concentration of organism that was detected is indicated. If not detected at the highest concentration the isolate is listed as not detected by the ePlex BCID-GP Panel. Results are provided below:

Table 29: Analytical Reactivity (Inclusivity) Results for the BCID-GP Panel   

<table><tr><td colspan="1" rowspan="1">Organism</td><td colspan="1" rowspan="1">Strain</td><td colspan="2" rowspan="1">Organism</td><td colspan="1" rowspan="1">Strain</td></tr><tr><td colspan="2" rowspan="1">Bacillus cereus</td><td colspan="2" rowspan="1">Staphylococcus lentus</td><td colspan="1" rowspan="1">ATCC 700403</td></tr><tr><td colspan="1" rowspan="4">Bacillus cereus</td><td colspan="1" rowspan="1">ATCC 21769</td><td colspan="2" rowspan="4">Staphylococcus lugdunensis</td><td colspan="1" rowspan="1">NRS-879</td></tr><tr><td colspan="1" rowspan="1">ATCC 10876</td><td colspan="1" rowspan="1">ATCC 49576</td></tr><tr><td colspan="1" rowspan="1">ATCC 31430</td><td colspan="1" rowspan="1">NRS-878</td></tr><tr><td colspan="1" rowspan="1">ATCC 53522</td><td colspan="1" rowspan="1">NRS-880</td></tr><tr><td colspan="1" rowspan="4">Bacillus thuringiensis</td><td colspan="1" rowspan="1">ATCC 35646</td><td colspan="2" rowspan="1">Staphylococcus muscae</td><td colspan="1" rowspan="1">ATCC 49910</td></tr><tr><td colspan="1" rowspan="1">ATCC 33679</td><td colspan="2" rowspan="1">Staphylococcus pasteuri</td><td colspan="1" rowspan="1">ATCC 51129</td></tr><tr><td colspan="1" rowspan="1">ATCC 55173</td><td colspan="2" rowspan="1">Staphylococcus pasteuri (mecC+)</td><td colspan="1" rowspan="1">ATCC 51128</td></tr><tr><td colspan="1" rowspan="1">ATCC 10792</td><td colspan="2" rowspan="1">Staphylococcus pettenkoferi</td><td colspan="1" rowspan="1">DSM-19554</td></tr><tr><td colspan="2" rowspan="1">Bacillus subtilis</td><td colspan="2" rowspan="1">Staphylococcus pseudintermedius</td><td colspan="1" rowspan="1">ATCC 49444</td></tr><tr><td colspan="1" rowspan="4">Bacillus amyloliquefaciens</td><td colspan="1" rowspan="1">ATCC 23845</td><td colspan="2" rowspan="1">Staphylococcus saccharolyticus</td><td colspan="1" rowspan="1">ATCC 14953</td></tr><tr><td colspan="1" rowspan="1">ATCC 23842</td><td colspan="2" rowspan="2">Staphylococcus saprophyticus</td><td colspan="1" rowspan="1">ATCC 15305</td></tr><tr><td colspan="1" rowspan="1">ATCC 23350</td><td colspan="1" rowspan="1">ATCC 35552</td></tr><tr><td colspan="1" rowspan="1">ATCC 53495</td><td colspan="2" rowspan="2">Staphylococcus schleiferi</td><td colspan="1" rowspan="1">ATCC 49545</td></tr><tr><td colspan="1" rowspan="5">Bacillus atrophaeus</td><td colspan="1" rowspan="1">ATCC 51189</td><td colspan="1" rowspan="1">ATCC 43808</td></tr><tr><td colspan="1" rowspan="1">ATCC6455</td><td colspan="2" rowspan="5">Staphylococcus sciuri</td><td colspan="1" rowspan="1">ATCC 29060</td></tr><tr><td colspan="1" rowspan="1">ATCC49337</td><td colspan="1" rowspan="1">ATCC 29061</td></tr><tr><td colspan="1" rowspan="1">ATCC 7972</td><td colspan="1" rowspan="1">St</td><td colspan="1" rowspan="1">ATCC 29059</td></tr><tr><td colspan="1" rowspan="1">ATCC6537</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">ATCC 49575</td></tr><tr><td colspan="1" rowspan="4">Bacillus licheniformis</td><td colspan="1" rowspan="1">ATCC 53926</td><td colspan="1" rowspan="1">ATCC 29062</td></tr><tr><td colspan="1" rowspan="1">ATCC55768</td><td colspan="2" rowspan="3">Staphylococcus simulans</td><td colspan="1" rowspan="1">ATCC 27848</td></tr><tr><td colspan="1" rowspan="1">ATCC 21039</td><td colspan="1" rowspan="1">St</td><td colspan="1" rowspan="1">ATCC 27850</td></tr><tr><td colspan="1" rowspan="1">ATCC 21667</td><td colspan="1" rowspan="1">ATCC 31432</td></tr><tr><td colspan="1" rowspan="3">Bacillus subtilis</td><td colspan="1" rowspan="1">ATCC 55614</td><td colspan="2" rowspan="1">Y</td><td colspan="1" rowspan="1">ATCC 27851</td></tr><tr><td colspan="1" rowspan="1">ATCC 15561</td><td colspan="2" rowspan="1">Staphylococcus species</td><td colspan="1" rowspan="1">ATCC 155</td></tr><tr><td colspan="1" rowspan="1">ATCC 21008</td><td colspan="2" rowspan="1">Staphylococcus species (mecA+)</td><td colspan="1" rowspan="1">ATCC 27626</td></tr><tr><td colspan="1" rowspan="1">Bacillus subtilis</td><td colspan="1" rowspan="1">ATCC 15040</td><td colspan="2" rowspan="2">Staphylococcus vitulinus</td><td colspan="1" rowspan="1">ATCC 51699</td></tr><tr><td colspan="2" rowspan="1">Corynebacterium</td><td colspan="1" rowspan="1">ATCC 51161</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium afermentanssubsp. afermentans</td><td colspan="1" rowspan="1">ATCC 51403A</td><td colspan="2" rowspan="1">Staphylococcus warneri</td><td colspan="1" rowspan="1">ATCC 27836</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium afermentanssubsp. lipophilum</td><td colspan="1" rowspan="1">ATCC 51404</td><td colspan="2" rowspan="2">Staphylococcus xylosus</td><td colspan="1" rowspan="1">ATCC 49148</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium confusum</td><td colspan="1" rowspan="1">ATCC 38268A</td><td colspan="1" rowspan="1">ATCC 29971</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium coyleae</td><td colspan="1" rowspan="1">ATCC 700219</td><td colspan="2" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Corynebacterium diphtheriae</td><td colspan="1" rowspan="1">ATCC 13812</td><td colspan="2" rowspan="10">Staphylococcus aureus</td><td colspan="1" rowspan="1">ATCC 25923</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium falsenii</td><td colspan="1" rowspan="1">ATCC BAA-596A</td><td colspan="1" rowspan="1">ATCC 29247</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium freneyi</td><td colspan="1" rowspan="1">ATCC 64424B</td><td colspan="1" rowspan="1">ATCC 6538P</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium imitans</td><td colspan="1" rowspan="1">ATCC 700354B</td><td colspan="1" rowspan="1">ATCC 29213</td></tr><tr><td colspan="1" rowspan="4">Corynebacterium jeikeium</td><td colspan="1" rowspan="1">ATCC 43217</td><td colspan="1" rowspan="1">NR-45889</td></tr><tr><td colspan="1" rowspan="1">ATCC 43216</td><td colspan="1" rowspan="1">NR-45890</td></tr><tr><td colspan="1" rowspan="1">ATCC 43734</td><td colspan="1" rowspan="1">NR-46074</td></tr><tr><td colspan="1" rowspan="1">ATCC BAA-949</td><td colspan="1" rowspan="1">NR-45881</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium minutissimum</td><td colspan="1" rowspan="1">ATCC 23348</td><td colspan="1" rowspan="1">NR-46411</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium resistens</td><td colspan="1" rowspan="1">CCUG 50093T</td><td colspan="1" rowspan="1">NR-46414</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium simulans</td><td colspan="1" rowspan="1">ATCC BAA-15</td><td colspan="2" rowspan="2">Staphylococcus aureus</td><td colspan="1" rowspan="1">NR-46418</td></tr><tr><td colspan="1" rowspan="3">Corynebacterium striatum</td><td colspan="1" rowspan="1">ATCC BAA-1293</td><td colspan="1" rowspan="1">NR-46417</td></tr><tr><td colspan="1" rowspan="1">ATCC 43735</td><td colspan="2" rowspan="20">Staphylococcus aureus (mecA +)</td><td colspan="1" rowspan="1">NR-13524</td></tr><tr><td colspan="1" rowspan="1">ATCC 7094</td><td colspan="1" rowspan="1">NR-13527</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium timonense</td><td colspan="1" rowspan="1">CCUG 64728A</td><td colspan="1" rowspan="1">NR-13525</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium ulcerans</td><td colspan="1" rowspan="1">ATCC 51799</td><td colspan="1" rowspan="1">NR-13526</td></tr><tr><td colspan="1" rowspan="3">Corynebacterium urealyticum</td><td colspan="1" rowspan="1">ATCC 43044</td><td colspan="1" rowspan="1">NRS-123</td></tr><tr><td colspan="1" rowspan="1">ATCC 43042</td><td colspan="1" rowspan="1">NR-46805</td></tr><tr><td colspan="1" rowspan="1">ATCC 43043</td><td colspan="1" rowspan="1">NRS-662</td></tr><tr><td colspan="1" rowspan="1">Corynebacteriumureicelerivorans</td><td colspan="1" rowspan="1">CCUG 59144</td><td colspan="1" rowspan="1">ATCC BAA-1707</td></tr><tr><td colspan="2" rowspan="1">Cutibacterium acnes</td><td colspan="1" rowspan="1">ATCC 43300</td></tr><tr><td colspan="1" rowspan="4">Cutibacterium acnes</td><td colspan="1" rowspan="1">ATCC 11827</td><td colspan="1" rowspan="1">Stan</td><td colspan="1" rowspan="1">NRS-383</td></tr><tr><td colspan="1" rowspan="1">ATCC 11828</td><td colspan="1" rowspan="1">NCTC-12493</td></tr><tr><td colspan="1" rowspan="1">ATCC 33179</td><td colspan="1" rowspan="1">NRS-676</td></tr><tr><td colspan="1" rowspan="1">ATCC 6919</td><td colspan="1" rowspan="1">NRS-678</td></tr><tr><td colspan="1" rowspan="1">Enterococcus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">NRS-648</td></tr><tr><td colspan="1" rowspan="1">Enterococcus avium</td><td colspan="1" rowspan="1">ATCC 14025</td><td colspan="1" rowspan="1">NRS-651</td></tr><tr><td colspan="1" rowspan="3">Enterococcus casseliflavus</td><td colspan="1" rowspan="1">ATCC 700668</td><td colspan="1" rowspan="1">NRS-643</td></tr><tr><td colspan="1" rowspan="1">ATCC 25788</td><td colspan="1" rowspan="1">NRS-484</td></tr><tr><td colspan="1" rowspan="1">ATCC 700327</td><td colspan="1" rowspan="1">NRS-385</td></tr><tr><td colspan="1" rowspan="1">Enterococcus cecorum</td><td colspan="1" rowspan="1">ATCC 43198</td><td colspan="1" rowspan="1">NRS-384</td></tr><tr><td colspan="1" rowspan="1">Enterococcus dispar</td><td colspan="1" rowspan="1">ATCC 51266</td><td colspan="1" rowspan="1">ATCC BAA-40</td></tr></table>

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=6>Organism</td><td rowspan=1 colspan=1>Strain</td></tr><tr><td rowspan=1 colspan=1>Enterococcus durans</td><td rowspan=1 colspan=1>ATCC 11576</td><td rowspan=7 colspan=6></td><td rowspan=1 colspan=1>ATCC 700698</td></tr><tr><td rowspan=12 colspan=1>Enterococcus faecalis</td><td rowspan=1 colspan=1>ATCC 51575</td><td rowspan=1 colspan=1>NRS-382</td></tr><tr><td rowspan=1 colspan=1>ATCC 700802</td><td rowspan=1 colspan=4>′</td><td rowspan=1 colspan=1>NRS-659</td></tr><tr><td rowspan=1 colspan=1>ATCC 10100</td><td rowspan=1 colspan=3></td><td rowspan=1 colspan=1>NRS-657</td></tr><tr><td rowspan=1 colspan=1>ATCC 12399</td><td rowspan=1 colspan=3></td><td rowspan=1 colspan=1>NRS-655</td></tr><tr><td rowspan=1 colspan=1>ATCC 14506</td><td rowspan=1 colspan=4></td><td rowspan=1 colspan=1>NRS-654</td></tr><tr><td rowspan=1 colspan=1>ATCC 33186</td><td rowspan=1 colspan=2></td><td rowspan=1 colspan=3></td><td rowspan=1 colspan=1>NRS-647</td></tr><tr><td rowspan=1 colspan=1>ATCC 49532</td><td rowspan=6 colspan=6></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>ATCC BAA-42</td></tr><tr><td rowspan=1 colspan=1>ATCC 49533</td><td rowspan=1 colspan=1>ATCC BAA-41</td></tr><tr><td rowspan=1 colspan=1>ATCC 7080</td><td rowspan=1 colspan=1>NRS-483</td></tr><tr><td rowspan=1 colspan=1>ATCC 49474</td><td rowspan=1 colspan=3></td><td rowspan=1 colspan=1>NRS-675</td></tr><tr><td rowspan=1 colspan=1>ATCC 49332</td><td rowspan=1 colspan=1>NRS-645</td></tr><tr><td rowspan=1 colspan=1>ATCC 29200</td><td rowspan=1 colspan=1>NRS-687</td></tr><tr><td rowspan=6 colspan=1>Enterococcus faecalis</td><td rowspan=1 colspan=1>ATCC BAA-2128</td><td rowspan=8 colspan=6>Staphylococcus aureus (mecA+)</td><td rowspan=1 colspan=1>NRS-667</td></tr><tr><td rowspan=1 colspan=1>ATCC 51188</td><td rowspan=1 colspan=1>NRS-677</td></tr><tr><td rowspan=1 colspan=1>ATCC 49149</td><td rowspan=1 colspan=1>NRS-683</td></tr><tr><td rowspan=1 colspan=1>NCTC-775</td><td rowspan=1 colspan=1>NRS-688</td></tr><tr><td rowspan=1 colspan=1>ATCC 19433</td><td rowspan=1 colspan=1>NRS-22</td></tr><tr><td rowspan=1 colspan=1>ATCC 49452</td><td rowspan=1 colspan=1>NRS-387</td></tr><tr><td rowspan=1 colspan=1>Enterococcus faecalis (vanA+)</td><td rowspan=1 colspan=1>JMI 876745</td><td rowspan=1 colspan=1>NRS-70</td></tr><tr><td rowspan=2 colspan=1>Enterococcus faecalis (vanB+)</td><td rowspan=1 colspan=1>ATCC BAA-2365</td><td rowspan=1 colspan=1>ATCC 33591</td></tr><tr><td rowspan=1 colspan=1>ATCC 51299</td><td rowspan=2 colspan=6>Staphylococcus aureus (mecC+)</td><td rowspan=1 colspan=1>ATCC BAA-2312</td></tr><tr><td rowspan=7 colspan=1>Enterococcus faecium</td><td rowspan=1 colspan=1>ATCC 19434</td><td rowspan=1 colspan=1>ATCC BAA-2313</td></tr><tr><td rowspan=1 colspan=1>ATCC 23828</td><td rowspan=1 colspan=7>Staphylococcus epidermidis</td></tr><tr><td rowspan=1 colspan=1>ATCC 27273</td><td rowspan=5 colspan=6>Staphylococcus epidermidis</td><td rowspan=1 colspan=1>ATCC 12228</td></tr><tr><td rowspan=1 colspan=1>ATCC BAA-2127</td><td rowspan=1 colspan=1>ATCC 49134</td></tr><tr><td rowspan=1 colspan=1>ATCC 6057</td><td rowspan=1 colspan=1>NCIMB-8853</td></tr><tr><td rowspan=1 colspan=1>ATCC 49624</td><td rowspan=1 colspan=1>ATCC 700583</td></tr><tr><td rowspan=1 colspan=1>ATCC 6569</td><td rowspan=1 colspan=1>ATCC 14990</td></tr><tr><td rowspan=7 colspan=1>Enterococcus faecium</td><td rowspan=1 colspan=1>ATCC BAA-472</td><td rowspan=3 colspan=6>Staphylococcus epidermidis</td><td rowspan=1 colspan=1>ATCC 35983</td></tr><tr><td rowspan=1 colspan=1>ATCC 9756</td><td rowspan=1 colspan=3></td><td rowspan=1 colspan=1>ATCC 35984</td></tr><tr><td rowspan=1 colspan=1>ATCC 49224</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>ATCC 29887</td></tr><tr><td rowspan=1 colspan=1>ATCC BAA-2846</td><td rowspan=3 colspan=6>(mecA+)</td><td rowspan=1 colspan=1>ATCC 49461</td></tr><tr><td rowspan=1 colspan=1>ATCC 19953</td><td rowspan=1 colspan=1>ATCC 700565</td></tr><tr><td rowspan=1 colspan=1>ATCC 27270</td><td rowspan=1 colspan=1>ATCC 51625</td></tr><tr><td rowspan=1 colspan=1>ATCC 35667</td><td rowspan=1 colspan=6>Staphylococcus lugdunensis</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=2 colspan=1>Enterococcus faecium (vanA+)</td><td rowspan=1 colspan=1>ATCC BAA-2316</td><td rowspan=2 colspan=6>Staphylococcus lugdunensis</td><td rowspan=1 colspan=1>NRS-879</td></tr><tr><td rowspan=1 colspan=1>ATCC BAA-2317</td><td rowspan=1 colspan=1>ATCC 49576</td></tr></table>

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Strain</td></tr><tr><td rowspan=13 colspan=1></td><td rowspan=1 colspan=1>ATCC 700221</td><td rowspan=2 colspan=1></td><td rowspan=1 colspan=1>NRS-878</td></tr><tr><td rowspan=1 colspan=1>ATCC BAA-2319</td><td rowspan=1 colspan=1>NRS-880</td></tr><tr><td rowspan=1 colspan=1>ATCC BAA-2320</td><td rowspan=1 colspan=2>Streptococcus</td></tr><tr><td rowspan=1 colspan=1>LMC 003921</td><td rowspan=7 colspan=1>Streptococcus agalactiae</td><td rowspan=1 colspan=1>ATCC 12401</td></tr><tr><td rowspan=1 colspan=1>LMC 103676</td><td rowspan=1 colspan=1>ATCC 13813</td></tr><tr><td rowspan=1 colspan=1>LMC 089524</td><td rowspan=1 colspan=1>ATCC 12386</td></tr><tr><td rowspan=1 colspan=1>LMC 104266</td><td rowspan=1 colspan=1>ATCC 12973</td></tr><tr><td rowspan=1 colspan=1>LMC 032261</td><td rowspan=1 colspan=1>ATCC 27956</td></tr><tr><td rowspan=1 colspan=1>LMC 110371</td><td rowspan=1 colspan=1>NCTC-8017</td></tr><tr><td rowspan=1 colspan=1>LMC 002867</td><td rowspan=1 colspan=1>ATCC 12403</td></tr><tr><td rowspan=1 colspan=1>LMC 055971</td><td rowspan=3 colspan=1>Streptococcus anginosus</td><td rowspan=1 colspan=1>ATCC 9895</td></tr><tr><td rowspan=1 colspan=1>ATCC 51559</td><td rowspan=1 colspan=1>ATCC 33397</td></tr><tr><td rowspan=1 colspan=1>ATCC BAA-2318</td><td rowspan=1 colspan=1>NCTC-10713</td></tr><tr><td rowspan=1 colspan=1>Enterococcus faecium (vanB+)</td><td rowspan=1 colspan=1>ATCC 51858</td><td rowspan=2 colspan=1>Streptococcus bovis</td><td rowspan=1 colspan=1>ATCC 33317</td></tr><tr><td rowspan=5 colspan=1>Enterococcus gallinarum</td><td rowspan=1 colspan=1>ATCC 49573</td><td rowspan=1 colspan=1>ATCC 35034</td></tr><tr><td rowspan=1 colspan=1>ATCC 49609</td><td rowspan=1 colspan=1>Streptococcus constellatus</td><td rowspan=1 colspan=1>ATCC 27513</td></tr><tr><td rowspan=1 colspan=1>ATCC 49610</td><td rowspan=1 colspan=1>Streptococcus criceti</td><td rowspan=1 colspan=1>ATCC 19642c</td></tr><tr><td rowspan=1 colspan=1>ATCC 700425</td><td rowspan=3 colspan=1>Streptococcus dysgalactiae</td><td rowspan=1 colspan=1>ATCC 35666</td></tr><tr><td rowspan=1 colspan=1>ATCC 49608</td><td rowspan=1 colspan=1>ATCC 43078</td></tr><tr><td rowspan=2 colspan=1>Enterococcus hirae</td><td rowspan=1 colspan=1>ATCC 49479</td><td rowspan=1 colspan=1>ATCC 12394</td></tr><tr><td rowspan=1 colspan=1>ATCC 10541</td><td rowspan=2 colspan=1>Streptococcus equi</td><td rowspan=1 colspan=1>ATCC 43079</td></tr><tr><td rowspan=1 colspan=1>Enterococcus italicus</td><td rowspan=1 colspan=1>CCUG 47860</td><td rowspan=1 colspan=1>ATCC 9528</td></tr><tr><td rowspan=1 colspan=1>Enterococcus malodoratus</td><td rowspan=1 colspan=1>ATCC 43197</td><td rowspan=2 colspan=1>Streptococcus equinus</td><td rowspan=1 colspan=1>ATCC 15351</td></tr><tr><td rowspan=1 colspan=1>Enterococcus pseudoavium</td><td rowspan=1 colspan=1>CCUG 33310T</td><td rowspan=1 colspan=1>ATCC 9812</td></tr><tr><td rowspan=1 colspan=1>Enterococcus raffinosus</td><td rowspan=1 colspan=1>ATCC 49464</td><td rowspan=3 colspan=1>Streptococcus gallolyticus</td><td rowspan=1 colspan=1>ATCC 49475</td></tr><tr><td rowspan=1 colspan=1>Enterococcus saccharolyticus</td><td rowspan=1 colspan=1>ATCC 43076</td><td rowspan=1 colspan=1>ATCC 9809</td></tr><tr><td rowspan=1 colspan=1>Enterococcus sanguinicola</td><td rowspan=1 colspan=1>DSM-21767</td><td rowspan=1 colspan=1>ATCC 43144</td></tr><tr><td rowspan=1 colspan=1>Enterococcus faecalis</td><td rowspan=1 colspan=1></td><td rowspan=2 colspan=1>Streptococcus gordonii</td><td rowspan=1 colspan=1>ATCC 10558</td></tr><tr><td rowspan=4 colspan=1>Enterococcus faecalis</td><td rowspan=1 colspan=1>ATCC 51575</td><td rowspan=1 colspan=1>ATCC 35557</td></tr><tr><td rowspan=1 colspan=1>ATCC 700802</td><td rowspan=2 colspan=1>Streptococcus infantarius</td><td rowspan=1 colspan=1>ATCC BAA-102</td></tr><tr><td rowspan=1 colspan=1>ATCC 10100</td><td rowspan=1 colspan=1>ATCC BAA-103</td></tr><tr><td rowspan=1 colspan=1>ATCC 12399</td><td rowspan=2 colspan=1>Streptococcus infantis</td><td rowspan=1 colspan=1>ATCC BAA-2089</td></tr><tr><td rowspan=7 colspan=1>Enterococcus faecalis</td><td rowspan=1 colspan=1>ATCC 14506</td><td rowspan=1 colspan=1>ATCC 700779</td></tr><tr><td rowspan=1 colspan=1>ATCC 33186</td><td rowspan=1 colspan=1>Streptococcus intermedius</td><td rowspan=1 colspan=1>ATCC 27335</td></tr><tr><td rowspan=1 colspan=1>ATCC 49532</td><td rowspan=2 colspan=1>Streptococcus mitis</td><td rowspan=1 colspan=1>ATCC 15914</td></tr><tr><td rowspan=1 colspan=1>ATCC 49533</td><td rowspan=1 colspan=1>ATCC 6249</td></tr><tr><td rowspan=1 colspan=1>ATCC 7080</td><td rowspan=2 colspan=1>Streptococcus mitis</td><td rowspan=1 colspan=1>ATCC 49456</td></tr><tr><td rowspan=1 colspan=1>ATCC 49474</td><td rowspan=1 colspan=1>NCIMB-13770</td></tr><tr><td rowspan=1 colspan=1>ATCC 49332</td><td rowspan=1 colspan=1>Streptococcus oralis</td><td rowspan=1 colspan=1>ATCC 35037</td></tr></table>

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=2>Organism</td><td rowspan=1 colspan=1>Strain</td></tr><tr><td rowspan=7 colspan=1>Enterococcus faecalis</td><td rowspan=1 colspan=1>ATCC 29200</td><td rowspan=1 colspan=2>Streptococcus oralis</td><td rowspan=1 colspan=1>ATCC 55229</td></tr><tr><td rowspan=1 colspan=1>ATCC BAA-2128</td><td rowspan=1 colspan=2>Streptococcus parasanguinis</td><td rowspan=1 colspan=1>ATCC 15909</td></tr><tr><td rowspan=1 colspan=1>ATCC 51188</td><td rowspan=1 colspan=2>Streptococcus peroris</td><td rowspan=1 colspan=1>ATCC 700780</td></tr><tr><td rowspan=1 colspan=1>ATCC 49149</td><td rowspan=4 colspan=2></td><td rowspan=1 colspan=1>ATCC BAA-475</td></tr><tr><td rowspan=1 colspan=1>NCTC-775</td><td rowspan=1 colspan=1>ATCC 10357</td></tr><tr><td rowspan=1 colspan=1>ATCC 19433</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>ATCC 49619</td></tr><tr><td rowspan=1 colspan=1>ATCC 49452</td><td rowspan=1 colspan=1>ATCC 49136</td></tr><tr><td rowspan=1 colspan=1>Enterococcus faecalis (vanA +)</td><td rowspan=1 colspan=1>JMI 876745</td><td rowspan=4 colspan=2>Streptococcus pneumoniae</td><td rowspan=1 colspan=1>ATCC BAA-659</td></tr><tr><td rowspan=2 colspan=1>Enterococcus faecalis (vanB+)</td><td rowspan=1 colspan=1>ATCC BAA-2365</td><td rowspan=1 colspan=1>ATCC 6315</td></tr><tr><td rowspan=1 colspan=1>ATCC 51299</td><td rowspan=1 colspan=1>ATCC BAA-1667</td></tr><tr><td rowspan=1 colspan=2>Enterococcus faecium</td><td rowspan=1 colspan=1>ATCC 700674</td></tr><tr><td rowspan=14 colspan=1>Enterococcus faecium</td><td rowspan=1 colspan=1>ATCC 19434</td><td rowspan=8 colspan=2></td><td rowspan=1 colspan=1>ATCC 6321</td></tr><tr><td rowspan=1 colspan=1>ATCC 23828</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>ATCC 700673</td></tr><tr><td rowspan=1 colspan=1>ATCC 27273</td><td rowspan=1 colspan=1>ATCC 51916</td></tr><tr><td rowspan=1 colspan=1>ATCC BAA-2127</td><td rowspan=1 colspan=1>ATCC 8338</td></tr><tr><td rowspan=1 colspan=1>ATCC 6057</td><td rowspan=1 colspan=1>ATCC 6301</td></tr><tr><td rowspan=1 colspan=1>ATCC 49624</td><td rowspan=1 colspan=1>ATCC 8335</td></tr><tr><td rowspan=1 colspan=1>ATCC 6569</td><td rowspan=1 colspan=1>NCIMB-13286</td></tr><tr><td rowspan=1 colspan=1>ATCC BAA-472</td><td rowspan=1 colspan=1>ATCC BAA-1656</td></tr><tr><td rowspan=1 colspan=1>ATCC 9756</td><td rowspan=6 colspan=2>Streptococcus pyogenes</td><td rowspan=1 colspan=1>ATCC 12384</td></tr><tr><td rowspan=1 colspan=1>ATCC 49224</td><td rowspan=1 colspan=1>ATCC 49399</td></tr><tr><td rowspan=1 colspan=1>ATCC BAA-2846</td><td rowspan=1 colspan=1>ATCC 12344</td></tr><tr><td rowspan=1 colspan=1>ATCC 19953</td><td rowspan=1 colspan=1>ATCC 19615</td></tr><tr><td rowspan=1 colspan=1>ATCC 27270</td><td rowspan=1 colspan=1>NCIMB-13285</td></tr><tr><td rowspan=1 colspan=1>ATCC 35667</td><td rowspan=1 colspan=1>ATCC 14289</td></tr><tr><td rowspan=2 colspan=1>Enterococcus faecium (vanA +)</td><td rowspan=1 colspan=1>ATCC BAA-2316</td><td rowspan=2 colspan=2>Streptococcus salivarius</td><td rowspan=1 colspan=1>ATCC 13419C</td></tr><tr><td rowspan=1 colspan=1>ATCC BAA-2317</td><td rowspan=1 colspan=1>ATCC 7073</td></tr><tr><td rowspan=3 colspan=1>Enterococcus faecium (vanA+)</td><td rowspan=1 colspan=1>ATCC 700221</td><td rowspan=2 colspan=2>Streptococcus salivarius</td><td rowspan=1 colspan=1>ATCC 25975</td></tr><tr><td rowspan=1 colspan=1>ATCC BAA-2319</td><td rowspan=1 colspan=1>ATCC 31067c</td></tr><tr><td rowspan=1 colspan=1>ATCC BAA-</td><td rowspan=1 colspan=2>Streptococcus sanguinis</td><td rowspan=1 colspan=1>ATCC 10556</td></tr></table>

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=2>Strain</td><td rowspan=1 colspan=3>Organism</td><td rowspan=1 colspan=1>Strain</td></tr><tr><td rowspan=9 colspan=1></td><td rowspan=1 colspan=2>2320</td><td rowspan=1 colspan=3></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=2>LMC 003921</td><td rowspan=1 colspan=3>Streptococcus thoraltensis</td><td rowspan=1 colspan=1>ATCC 700865</td></tr><tr><td rowspan=1 colspan=2>LMC 103676</td><td rowspan=1 colspan=4>Streptococcus agalactiae</td></tr><tr><td rowspan=1 colspan=2>LMC 089524</td><td rowspan=7 colspan=3>Streptococcus agalactiae</td><td rowspan=1 colspan=1>ATCC 12401</td></tr><tr><td rowspan=1 colspan=2>LMC 104266</td><td rowspan=1 colspan=1>ATCC 13813</td></tr><tr><td rowspan=1 colspan=2>LMC 032261</td><td rowspan=1 colspan=1>ATCC 12386</td></tr><tr><td rowspan=1 colspan=2>LMC 110371</td><td rowspan=1 colspan=1>ATCC 12973</td></tr><tr><td rowspan=1 colspan=2>LMC 002867</td><td rowspan=1 colspan=1>ATCC 27956</td></tr><tr><td rowspan=1 colspan=2>LMC 055971</td><td rowspan=1 colspan=1>NCTC-8017</td></tr><tr><td rowspan=2 colspan=1>Enterococcus faecium (vanA+)</td><td rowspan=1 colspan=2>ATCC 51559</td><td rowspan=1 colspan=1>ATCC 12403</td></tr><tr><td rowspan=1 colspan=2>ATCC BAA-2318</td><td rowspan=1 colspan=4>Streptococcus anginosus group</td></tr><tr><td rowspan=1 colspan=1>Enterococcus faecium (vanB+)</td><td rowspan=1 colspan=2>ATCC 51858</td><td rowspan=3 colspan=3>Streptococcus anginosus</td><td rowspan=1 colspan=1>ATCC 9895</td></tr><tr><td rowspan=1 colspan=1>Lactobacillus</td><td rowspan=1 colspan=2></td><td rowspan=1 colspan=2>St</td><td rowspan=1 colspan=1>ATCC 33397</td></tr><tr><td rowspan=3 colspan=1>Lactobacillus casei</td><td rowspan=1 colspan=2>ATCC 334</td><td rowspan=1 colspan=1>NCTC-10713</td></tr><tr><td rowspan=1 colspan=2>ATCC 55825</td><td rowspan=1 colspan=3>Streptococcus constellatus</td><td rowspan=1 colspan=1>ATCC 27513</td></tr><tr><td rowspan=1 colspan=2>ATCC 39392</td><td rowspan=1 colspan=3>Streptococcus intermedius</td><td rowspan=1 colspan=1>ATCC 27335</td></tr><tr><td rowspan=3 colspan=1>Lactobacillus paracasei</td><td rowspan=1 colspan=2>ATCC 25598</td><td rowspan=1 colspan=4>Streptococcus pneum oniae</td></tr><tr><td rowspan=1 colspan=2>ATCC BAA-52</td><td rowspan=17 colspan=3>Streptococcus pneumoniae</td><td rowspan=1 colspan=1>ATCC BAA-475</td></tr><tr><td rowspan=1 colspan=2>ATCC 27092</td><td rowspan=1 colspan=1>ATCC 10357</td></tr><tr><td rowspan=4 colspan=1>Lactobacillus rhamnosus</td><td rowspan=1 colspan=2>ATCC 39595</td><td rowspan=1 colspan=2></td><td rowspan=1 colspan=1>ATCC 49619</td></tr><tr><td rowspan=1 colspan=2>ATCC 53103</td><td rowspan=1 colspan=2></td><td rowspan=1 colspan=1>ATCC 49136</td></tr><tr><td rowspan=2 colspan=2>ATCC 55915</td><td rowspan=2 colspan=1>ATCC BAA-659</td></tr><tr></tr><tr><td rowspan=1 colspan=1>Lactobacillus zeae</td><td rowspan=1 colspan=2>ATCC 15820</td><td rowspan=1 colspan=1>ATCC 6315</td></tr><tr><td rowspan=1 colspan=3>Listeria</td><td rowspan=1 colspan=1>ATCC BAA-1667</td></tr><tr><td rowspan=2 colspan=1>Listeria innocua</td><td rowspan=1 colspan=2>NCTC-11288</td><td rowspan=1 colspan=1>b</td><td rowspan=1 colspan=1>ATCC 700674</td></tr><tr><td rowspan=1 colspan=2>ATCC 33090</td><td rowspan=1 colspan=2></td><td rowspan=1 colspan=1>ATCC 6321</td></tr><tr><td rowspan=3 colspan=1>Listeria ivanovii</td><td rowspan=1 colspan=2>ATCC 19119</td><td rowspan=1 colspan=1>ATCC 700673</td></tr><tr><td rowspan=1 colspan=2>ATCC BAA-139</td><td rowspan=1 colspan=1>ATCC 51916</td></tr><tr><td rowspan=1 colspan=2>ATCC 700402</td><td rowspan=1 colspan=1>ATCC 8338</td></tr><tr><td rowspan=8 colspan=1>Listeria monocytogenes</td><td rowspan=1 colspan=2>ATCC 10890</td><td rowspan=1 colspan=1>ATCC 6301</td></tr><tr><td rowspan=1 colspan=2>ATCC 19111</td><td rowspan=1 colspan=1>ATCC 8335</td></tr><tr><td rowspan=1 colspan=2>ATCC 13932</td><td rowspan=1 colspan=1>NCIMB-13286</td></tr><tr><td rowspan=1 colspan=2>ATCC 19114</td><td rowspan=1 colspan=1>ATCC BAA-1656</td></tr><tr><td rowspan=1 colspan=2>ATCC 7644</td><td rowspan=1 colspan=3>Streptococcus pyogenes</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=2>ATCC 19112</td><td rowspan=3 colspan=3>Streptococcus pyogenes</td><td rowspan=1 colspan=1>ATCC 12384</td></tr><tr><td rowspan=1 colspan=2>ATCC BAA-751</td><td rowspan=1 colspan=1>ATCC 49399</td></tr><tr><td rowspan=1 colspan=2>ATCC 19118</td><td rowspan=1 colspan=1>ATCC 12344</td></tr></table>

<table><tr><td colspan="1" rowspan="1">Organism</td><td colspan="1" rowspan="1">Strain</td><td colspan="1" rowspan="1">Organism</td><td colspan="1" rowspan="1">Strain</td></tr><tr><td colspan="1" rowspan="1">EY</td><td colspan="1" rowspan="1">ATCC 19115</td><td colspan="1" rowspan="3"></td><td colspan="1" rowspan="1">ATCC 19615</td></tr><tr><td colspan="1" rowspan="1">Listeria seeligeri</td><td colspan="1" rowspan="1">ATCC 35967</td><td colspan="1" rowspan="1">NCIMB-13285</td></tr><tr><td colspan="1" rowspan="1">Listeria welshimeri</td><td colspan="1" rowspan="1">ATCC 35897</td><td colspan="1" rowspan="1">ATCC 14289</td></tr><tr><td colspan="1" rowspan="1">Listeria monocytogenes</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Pan Candida</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="3">Listeria monocytogenes</td><td colspan="1" rowspan="1">ATCC 10890</td><td colspan="1" rowspan="3">Candida albicans</td><td colspan="1" rowspan="1">ATCC 24433</td></tr><tr><td colspan="1" rowspan="1">ATCC 19111</td><td colspan="1" rowspan="1">ATCC MYA-4441</td></tr><tr><td colspan="1" rowspan="1">ATCC 13932</td><td colspan="1" rowspan="1">ATCC 14053</td></tr><tr><td colspan="1" rowspan="1">Listeria monocytogenes</td><td colspan="1" rowspan="1">ATCC 19114</td><td colspan="1" rowspan="1">Candida albicans</td><td colspan="1" rowspan="1">ATCC 90028</td></tr><tr><td colspan="1" rowspan="5">Listeria monocytogenes</td><td colspan="1" rowspan="1">ATCC 7644</td><td colspan="1" rowspan="4">Candida glabrata</td><td colspan="1" rowspan="1">ATCC 66032</td></tr><tr><td colspan="1" rowspan="1">ATCC 19112</td><td colspan="1" rowspan="1">ATCC 15126</td></tr><tr><td colspan="1" rowspan="1">ATCC BAA-751</td><td colspan="1" rowspan="1">ATCC MYA-2950</td></tr><tr><td colspan="1" rowspan="1">ATCC 19118</td><td colspan="1" rowspan="1">ATCC 2001</td></tr><tr><td colspan="1" rowspan="1">ATCC 19115</td><td colspan="1" rowspan="3">Candida krusei</td><td colspan="1" rowspan="1">ATCC 34135</td></tr><tr><td colspan="2" rowspan="1">Micrococcus</td><td colspan="1" rowspan="1">ATCC 32196</td></tr><tr><td colspan="1" rowspan="6">Micrococcus luteus</td><td colspan="1" rowspan="1">ATCC 19212</td><td colspan="1" rowspan="1">ATCC 14243</td></tr><tr><td colspan="1" rowspan="1">ATCC 10240</td><td colspan="1" rowspan="3">Candida parapsilosis</td><td colspan="1" rowspan="1">ATCC 90018D</td></tr><tr><td colspan="1" rowspan="1">ATCC 400</td><td colspan="1" rowspan="1">ATCC 58895</td></tr><tr><td colspan="1" rowspan="1">ATCC 53598</td><td colspan="1" rowspan="1">ATCC 22019</td></tr><tr><td colspan="1" rowspan="1">ATCC 49732</td><td colspan="2" rowspan="1">Pan Gram-Negative</td></tr><tr><td colspan="1" rowspan="1">ATCC 4698</td><td colspan="1" rowspan="2">Acinetobacter baumannii</td><td colspan="1" rowspan="1">NCTC-13302</td></tr><tr><td colspan="1" rowspan="1">Micrococcus lylae</td><td colspan="1" rowspan="1">ATCC 27566</td><td colspan="1" rowspan="1">NCTC-13422</td></tr><tr><td colspan="1" rowspan="1">Micrococcus yunnanensis</td><td colspan="1" rowspan="1">ATCC 7468</td><td colspan="1" rowspan="1">Acinetobacter haemolyticus</td><td colspan="1" rowspan="1">ATCC 19002</td></tr><tr><td colspan="2" rowspan="1">Staphylococcus</td><td colspan="1" rowspan="1">Acinetobacter junii</td><td colspan="1" rowspan="1">ATCC 17908</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus arlettae</td><td colspan="1" rowspan="1">ATCC 43957</td><td colspan="1" rowspan="1">Acinetobacter lwoffii</td><td colspan="1" rowspan="1">ATCC 15309</td></tr><tr><td colspan="1" rowspan="12">Staphylococcus aureus</td><td colspan="1" rowspan="1">ATCC 25923</td><td colspan="1" rowspan="1">Acinetobacter ursingii</td><td colspan="1" rowspan="1">ATCC BAA-617</td></tr><tr><td colspan="1" rowspan="1">ATCC 29247</td><td colspan="1" rowspan="1">Bacteroides fragilis</td><td colspan="1" rowspan="1">NCTC-9343</td></tr><tr><td colspan="1" rowspan="1">ATCC 6538P</td><td colspan="1" rowspan="1">Burkholderia cepacia</td><td colspan="1" rowspan="1">ATCC 25416</td></tr><tr><td colspan="1" rowspan="1">ATCC 29213</td><td colspan="1" rowspan="1">Cedecea davisiae</td><td colspan="1" rowspan="1">ATCC 43025</td></tr><tr><td colspan="1" rowspan="1">NR-45889</td><td colspan="1" rowspan="1">Citrobacter freundii</td><td colspan="1" rowspan="1">NCTC-9750</td></tr><tr><td colspan="1" rowspan="1">NR-45890</td><td colspan="1" rowspan="1">Citrobacter koseri</td><td colspan="1" rowspan="1">ATCC 27156</td></tr><tr><td colspan="1" rowspan="1">NR-46074</td><td colspan="1" rowspan="1">Cronobacter muytjensii</td><td colspan="1" rowspan="1">ATCC 51329</td></tr><tr><td colspan="1" rowspan="1">NR-45881</td><td colspan="1" rowspan="1">Cronobacter sakazakii</td><td colspan="1" rowspan="1">ATCC 12868</td></tr><tr><td colspan="1" rowspan="1">NR-46411</td><td colspan="1" rowspan="1">Edwardsiella tarda</td><td colspan="1" rowspan="1">ATCC 15947</td></tr><tr><td colspan="1" rowspan="1">NR-46414</td><td colspan="1" rowspan="1">Enterobacter aerogenes</td><td colspan="1" rowspan="1">ATCC 13048</td></tr><tr><td colspan="1" rowspan="1">NR-46418</td><td colspan="1" rowspan="1">Enterobacter amnigenus</td><td colspan="1" rowspan="1">ATCC 51816</td></tr><tr><td colspan="1" rowspan="1">NR-46417</td><td colspan="1" rowspan="1">Enterobacter asburiae</td><td colspan="1" rowspan="1">ATCC 35953</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus (mecA+)</td><td colspan="1" rowspan="1">NR-13524</td><td colspan="1" rowspan="1">Enterobacter cloacae</td><td colspan="1" rowspan="1">ATCC 13047</td></tr><tr><td colspan="1" rowspan="8"></td><td colspan="1" rowspan="1">NR-13527</td><td colspan="1" rowspan="3"></td><td colspan="1" rowspan="1">ATCC 23373</td></tr><tr><td colspan="1" rowspan="1">NR-13525</td><td colspan="1" rowspan="1">ATCC 23355</td></tr><tr><td colspan="1" rowspan="1">NR-13526</td><td colspan="1" rowspan="1">ATCC 35030</td></tr><tr><td colspan="1" rowspan="1">NRS-123</td><td colspan="1" rowspan="1">Enterobacter hormaechei</td><td colspan="1" rowspan="1">ATCC 700323</td></tr><tr><td colspan="1" rowspan="1">NR-46805</td><td colspan="1" rowspan="1">Erwinia aphidicola</td><td colspan="1" rowspan="1">ATCC 27991</td></tr><tr><td colspan="1" rowspan="1">NRS-662</td><td colspan="1" rowspan="3">Escherichia coli</td><td colspan="1" rowspan="1">ATCC 4157</td></tr><tr><td colspan="1" rowspan="1">ATCC BAA-1707</td><td colspan="1" rowspan="1">ATCC 14948</td></tr><tr><td colspan="1" rowspan="1">ATCC 43300</td><td colspan="1" rowspan="1">ATCC 25922</td></tr><tr><td colspan="1" rowspan="3">Staphylococcus aureus (mecA+)</td><td colspan="1" rowspan="1">NRS-383</td><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">ATCC BAA-196</td></tr><tr><td colspan="1" rowspan="1">NCTC-12493</td><td colspan="1" rowspan="1">Escherichia fergusonii</td><td colspan="1" rowspan="1">ATCC 35469</td></tr><tr><td colspan="1" rowspan="1">NRS-676</td><td colspan="1" rowspan="1">Escherichia hermannii</td><td colspan="1" rowspan="1">ATCC 700368</td></tr><tr><td colspan="1" rowspan="12">Staphylococcus aureus (mecA+)</td><td colspan="1" rowspan="1">NRS-678</td><td colspan="1" rowspan="1">Escherichia vulneris</td><td colspan="1" rowspan="1">ATCC 33821</td></tr><tr><td colspan="1" rowspan="1">NRS-648</td><td colspan="1" rowspan="1">Fusobacterium necrophorum</td><td colspan="1" rowspan="1">ATCC 25286</td></tr><tr><td colspan="1" rowspan="1">NRS-651</td><td colspan="1" rowspan="1">Fusobacterium nucleatum</td><td colspan="1" rowspan="1">ATCC 25586</td></tr><tr><td colspan="1" rowspan="1">NRS-643</td><td colspan="1" rowspan="3">Haemophilus influenzae</td><td colspan="1" rowspan="1">ATCC 10211</td></tr><tr><td colspan="1" rowspan="1">NRS-484</td><td colspan="1" rowspan="1">ATCC 49144</td></tr><tr><td colspan="1" rowspan="1">NRS-385</td><td colspan="1" rowspan="1">ATCC 43065</td></tr><tr><td colspan="1" rowspan="1">NRS-384</td><td colspan="1" rowspan="1">Haemophilus influenzae</td><td colspan="1" rowspan="1">NCTC-8468</td></tr><tr><td colspan="1" rowspan="1">ATCC BAA-40</td><td colspan="1" rowspan="1">Haemophilus parainfluenzae</td><td colspan="1" rowspan="1">ATCC 9796</td></tr><tr><td colspan="1" rowspan="1">ATCC 700698</td><td colspan="1" rowspan="1">Hafnia alvei</td><td colspan="1" rowspan="1">ATCC 51815</td></tr><tr><td colspan="1" rowspan="1">NRS-382</td><td colspan="1" rowspan="3">Klebsiella oxytoca</td><td colspan="1" rowspan="1">ATCC 700324</td></tr><tr><td colspan="1" rowspan="1">NRS-659</td><td colspan="1" rowspan="1">ATCC 43165</td></tr><tr><td colspan="1" rowspan="1">NRS-657</td><td colspan="1" rowspan="1">ATCC 43863</td></tr><tr><td colspan="1" rowspan="10">Staphylococcus aureus (mecA +)</td><td colspan="1" rowspan="1">NRS-655</td><td colspan="1" rowspan="1">Klebsiella ozaenae</td><td colspan="1" rowspan="1">ATCC 11296</td></tr><tr><td colspan="1" rowspan="1">NRS-654</td><td colspan="1" rowspan="4">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">ATCC BAA-1706</td></tr><tr><td colspan="1" rowspan="1">NRS-647</td><td colspan="1" rowspan="1">NCTC-9633</td></tr><tr><td colspan="1" rowspan="1">ATCC BAA-42</td><td colspan="1" rowspan="1">NCTC-13440</td></tr><tr><td colspan="1" rowspan="1">ATCC BAA-41</td><td colspan="1" rowspan="1">ATCC 51503</td></tr><tr><td colspan="1" rowspan="1">NRS-483</td><td colspan="1" rowspan="1">Kluyvera ascorbata</td><td colspan="1" rowspan="1">ATCC 14236</td></tr><tr><td colspan="1" rowspan="1">NRS-675</td><td colspan="1" rowspan="1">Kluyvera cryocrescens</td><td colspan="1" rowspan="1">ATCC 14240</td></tr><tr><td colspan="1" rowspan="1">NRS-645</td><td colspan="1" rowspan="1">Legionella pneumophila</td><td colspan="1" rowspan="1">ATCC 33823</td></tr><tr><td colspan="1" rowspan="1">NRS-687</td><td colspan="1" rowspan="1">Moraxella catarrhalis</td><td colspan="1" rowspan="1">ATCC 23246</td></tr><tr><td colspan="1" rowspan="1">NRS-667</td><td colspan="1" rowspan="1">Moraxella nonliquefaciens</td><td colspan="1" rowspan="1">ATCC 17953</td></tr><tr><td colspan="1" rowspan="7"></td><td colspan="1" rowspan="1">NRS-677</td><td colspan="1" rowspan="1">Moraxella osloensis</td><td colspan="1" rowspan="1">ATCC 10973</td></tr><tr><td colspan="1" rowspan="1">NRS-683</td><td colspan="1" rowspan="1">Morganella morganii</td><td colspan="1" rowspan="1">ATCC 25829</td></tr><tr><td colspan="1" rowspan="1">NRS-688</td><td colspan="1" rowspan="1">Neisseria lactamica</td><td colspan="1" rowspan="1">ATCC 23970</td></tr><tr><td colspan="1" rowspan="1">NRS-22</td><td colspan="1" rowspan="4">Neisseria meningitidis</td><td colspan="1" rowspan="1">ATCC 13090</td></tr><tr><td colspan="1" rowspan="1">NRS-387</td><td colspan="1" rowspan="1">ATCC 13113</td></tr><tr><td colspan="1" rowspan="1">NRS-70</td><td colspan="1" rowspan="1">ATCC 13077</td></tr><tr><td colspan="1" rowspan="1">ATCC 33591</td><td colspan="1" rowspan="1">ATCC 13102</td></tr><tr><td colspan="1" rowspan="2">Staphylococcus aureus (mecC+)</td><td colspan="1" rowspan="1">ATCC BAA-2312</td><td colspan="1" rowspan="1">Neisseria mucosa</td><td colspan="1" rowspan="1">ATCC 19695</td></tr><tr><td colspan="1" rowspan="1">ATCC BAA-2313</td><td colspan="1" rowspan="1">Neisseria sicca</td><td colspan="1" rowspan="1">ATCC 29193</td></tr><tr><td colspan="1" rowspan="2">Staphylococcus auricularis</td><td colspan="1" rowspan="1">ATCC 33750</td><td colspan="1" rowspan="1">Pantoea agglomerans</td><td colspan="1" rowspan="1">ATCC 14537</td></tr><tr><td colspan="1" rowspan="1">ATCC 33753</td><td colspan="1" rowspan="1">Pasteurella aerogenes</td><td colspan="1" rowspan="1">ATCC 27883</td></tr><tr><td colspan="1" rowspan="3">Staphylococcus capitis</td><td colspan="1" rowspan="1">ATCC 27840</td><td colspan="1" rowspan="3">Proteus mirabilis</td><td colspan="1" rowspan="1">ATCC 43071</td></tr><tr><td colspan="1" rowspan="1">NRS-866</td><td colspan="1" rowspan="1">CDC#0155</td></tr><tr><td colspan="1" rowspan="1">ATCC 35661</td><td colspan="1" rowspan="1">CDC#0159</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus caprae</td><td colspan="1" rowspan="1">ATCC 51548</td><td colspan="1" rowspan="1">Proteus penneri</td><td colspan="1" rowspan="1">ATCC 35197</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus carnosus</td><td colspan="1" rowspan="1">ATCC 51365</td><td colspan="1" rowspan="1">Proteus vulgaris</td><td colspan="1" rowspan="1">ATCC 6380</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus chromogenes</td><td colspan="1" rowspan="1">ATCC 43764</td><td colspan="1" rowspan="1">Providencia alcalifaciens</td><td colspan="1" rowspan="1">ATCC 9886</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus cohnii</td><td colspan="1" rowspan="1">ATCC 29974</td><td colspan="1" rowspan="1">Providencia rettgeri</td><td colspan="1" rowspan="1">ATCC 9919</td></tr><tr><td colspan="1" rowspan="5">Staphylococcus epidermidis</td><td colspan="1" rowspan="1">ATCC 12228</td><td colspan="1" rowspan="1">Providencia stuartii</td><td colspan="1" rowspan="1">ATCC 33672</td></tr><tr><td colspan="1" rowspan="1">ATCC 49134</td><td colspan="1" rowspan="3">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">ATCC 15442</td></tr><tr><td colspan="1" rowspan="1">NCIMB-8853</td><td colspan="1" rowspan="1">NCIMB-12469</td></tr><tr><td colspan="1" rowspan="1">ATCC 700583</td><td colspan="1" rowspan="1">ATCC 10145</td></tr><tr><td colspan="1" rowspan="1">ATCC 14990</td><td colspan="1" rowspan="1">Pseudomonas fluorescens</td><td colspan="1" rowspan="1">ATCC 13525</td></tr><tr><td colspan="1" rowspan="6">Staphylococcus epidermidis(mecA +)</td><td colspan="1" rowspan="1">ATCC 35983</td><td colspan="1" rowspan="1">Pseudomonas luteola</td><td colspan="1" rowspan="1">ATCC 35970</td></tr><tr><td colspan="1" rowspan="1">ATCC 35984</td><td colspan="1" rowspan="1">Pseudomonas putida</td><td colspan="1" rowspan="1">ATCC 31483</td></tr><tr><td colspan="1" rowspan="1">ATCC 29887</td><td colspan="1" rowspan="1">Raoultella terigena</td><td colspan="1" rowspan="1">ATCC 55553</td></tr><tr><td colspan="1" rowspan="1">ATCC 49461</td><td colspan="1" rowspan="1">Salmonella bongori</td><td colspan="1" rowspan="1">ATCC 43975</td></tr><tr><td colspan="1" rowspan="1">ATCC 700565</td><td colspan="1" rowspan="1">Salmonella enterica subsp. enterica</td><td colspan="1" rowspan="1">ATCC 51957</td></tr><tr><td colspan="1" rowspan="1">ATCC 51625</td><td colspan="1" rowspan="1">Salmonella enterica Choleraesuis</td><td colspan="1" rowspan="1">ATCC 12011</td></tr><tr><td colspan="1" rowspan="2">Staphylococcus gallinarum</td><td colspan="1" rowspan="1">ATCC 700401</td><td colspan="1" rowspan="1">Salmonella enterica Paratyphi</td><td colspan="1" rowspan="1">FSL S5-0447</td></tr><tr><td colspan="1" rowspan="1">ATCC 35539</td><td colspan="1" rowspan="1">Salmonella typhi</td><td colspan="1" rowspan="1">ATCC 19430</td></tr><tr><td colspan="1" rowspan="2">Staphylococcus haemolyticus</td><td colspan="1" rowspan="1">ATCC 29970</td><td colspan="1" rowspan="1">Serratia fonticola</td><td colspan="1" rowspan="1">ATCC 29844</td></tr><tr><td colspan="1" rowspan="1">ATCC 31874</td><td colspan="1" rowspan="1">Serratia liquefaciens</td><td colspan="1" rowspan="1">ATCC 27592</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus haemolyticus</td><td colspan="1" rowspan="1">NRS-62</td><td colspan="1" rowspan="1">Serratia marcescens</td><td colspan="1" rowspan="1">ATCC 8100</td></tr><tr><td colspan="1" rowspan="2">(mecA+)</td><td colspan="1" rowspan="1">ATCC BAA-1693</td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="1">ATCC 14756</td></tr><tr><td colspan="1" rowspan="1">NRS-69</td><td colspan="1" rowspan="1">ATCC 13880</td></tr><tr><td colspan="1" rowspan="3">Staphylococcus hominis</td><td colspan="1" rowspan="1">NRS-871</td><td colspan="1" rowspan="1">Serratia odorifera</td><td colspan="1" rowspan="1">ATCC 33077</td></tr><tr><td colspan="1" rowspan="1">NRS-870</td><td colspan="1" rowspan="1">Serratia rubidaea</td><td colspan="1" rowspan="1">ATCC 27593</td></tr><tr><td colspan="1" rowspan="1">ATCC 27844</td><td colspan="1" rowspan="1">Shigella boydii</td><td colspan="1" rowspan="1">ATCC 9207</td></tr><tr><td colspan="1" rowspan="2">Staphylococcus hominis(mecA+)</td><td colspan="1" rowspan="1">ATCC 700237</td><td colspan="1" rowspan="1">Shigella dysenteriae</td><td colspan="1" rowspan="1">ATCC 13313</td></tr><tr><td colspan="1" rowspan="1">ATCC 700236</td><td colspan="1" rowspan="1">Shigella sonnei</td><td colspan="1" rowspan="1">ATCC 29930</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus hyicus</td><td colspan="1" rowspan="1">ATCC 11249</td><td colspan="1" rowspan="1">Stenotrophomonas maltophilia</td><td colspan="1" rowspan="1">ATCC 13636</td></tr><tr><td colspan="1" rowspan="2">Staphylococcus intermedius</td><td colspan="1" rowspan="1">ATCC 29663</td><td colspan="1" rowspan="1">Tatumella ptyseos</td><td colspan="1" rowspan="1">ATCC 33302</td></tr><tr><td colspan="1" rowspan="1">ATCC 49052</td><td colspan="1" rowspan="1">Yersinia enterocolitica subsp.Enterocolitica</td><td colspan="1" rowspan="1">ATCC 9610</td></tr></table>

A. Detected at $2 \mathrm { ~ x ~ } 1 0 ^ { 8 } \mathrm { C F U / m L }$ C. Detected at $\overline { { 4 \mathrm { ~ x ~ } 1 0 ^ { 8 } \mathrm { C F U / m L } } }$ B. Detected at $2 \mathrm { \times 1 0 ^ { 9 } C F U / m L }$ D. Detected at $1 \mathrm { ~ x ~ } 1 0 ^ { 7 } \mathrm { C F U / m L }$

Ten additional samples did not achieve $100 \%$ detection after re-testing. These samples were then tested at 10-fold, or when necessary 100-fold, higher concentration to achieve $100 \%$ detection. These samples and their test results are listed in Table 30 below. These strains are expected to have lower sensitivity on the BCID-GP Panel.

Table 30: Samples Tested at Higher Concentrations   

<table><tr><td colspan="1" rowspan="1">Organis m</td><td colspan="1" rowspan="1">Initial Dete ctionResult(3 replicates)</td><td colspan="1" rowspan="1">Retest Result(3 replicates)</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium falseniiATCCBAA-596</td><td colspan="1" rowspan="1">67% (2/3)</td><td colspan="1" rowspan="1">100% detection at 10-fold higherconc. (2x108 CFU/mL)</td></tr><tr><td colspan="1" rowspan="1">Streptococcus cricetiATCC19642</td><td colspan="1" rowspan="1">67% (2/3)</td><td colspan="1" rowspan="1">100% detection at 10-fold higherconc. (4x108 CFU/mL)</td></tr><tr><td colspan="1" rowspan="1">Candida parapsilosisATCC90018</td><td colspan="1" rowspan="1">33% (1/3)</td><td colspan="1" rowspan="1">100% detection at 10-fold higherconc. (1x107 CFU/mL)</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium afermentans subsp.afermentans ATCC51403</td><td colspan="1" rowspan="1">67% (2/3)</td><td colspan="1" rowspan="1">100% detection at 10-fold higherconc. (2x108 CFU/mL)</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium confusumATCC38268</td><td colspan="1" rowspan="1">67% (2/3)</td><td colspan="1" rowspan="1">100% detection at 10-fold higherconc. (2x108 CFU/mL)</td></tr><tr><td colspan="1" rowspan="1">Streptococcus salivariusATCC13419</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">100% detection at 10-fold higherconc. (4x108 CFU/mL)</td></tr><tr><td colspan="1" rowspan="1">Streptococcus salivariusATCC31067</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">100% detection at 10-fold higherconc. (4x108 CFU/mL)</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium timonenseCCUG64728</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">100% detection at 10-fold higherconc. (2x108 CFU/mL)</td></tr><tr><td colspan="1" rowspan="1">Organis m</td><td colspan="1" rowspan="1">Initial Dete ctionResult(3 replicates)</td><td colspan="1" rowspan="1">Retest Result(3 re plicates)</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium freneyiATCC64424</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">100% detection at 100-fold higherconc. (2x109 CFU/mL)</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium imitansATCC700354</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">100% detection at 100-fold higherconc. (2x109 CFU/mL)</td></tr></table>

In addition to species-specific assays, the ePlex BCID-GP Panel contains a number of broader genus or group-level assays; including Bacillus cereus group, Bacillus subtilis group, Corynebacterium, Enterococcus, Lactobacillus, Listeria, Micrococcus, Staphylococcus, Streptococcus, Streptococcus anginosus group, Pan Candida and Pan Gram-Negative assays. The following tables predicted (in silico) reactivity (inclusivity) for these assay targets.

Note: the performance of the ePlex BCID-GP Panel has not been established for all of the organisms listed in the tables below. See the Analytical Reactivity (Inclusivity) and Limit of Detection (Analytical Sensitivity) sections for data on organisms for which performance characteristics have been established (indicated with an asterisk in Tables 31-41). Some species were not assessed in silico due to lack of sequence data, though they may appear in the analytical sensitivity or specificity studies.

Table 31: Predicted (in silico) Reactivity (Inclusivity) Results for Bacillus cereus group   

<table><tr><td rowspan=1 colspan=4>Detection Predicted for ≥95% of target sequences</td></tr><tr><td rowspan=1 colspan=1>Bacillus thuringiensis*</td><td rowspan=1 colspan=2>Bacillus toyonensis</td><td rowspan=1 colspan=1>Bacillus weihenstephanensis</td></tr><tr><td rowspan=1 colspan=4>Detection Predicted for 85%-94% of target sequences</td></tr><tr><td rowspan=1 colspan=4>Bacillus cereus*</td></tr><tr><td rowspan=1 colspan=4>Detection Predicted for &lt;85% of target sequences</td></tr><tr><td rowspan=1 colspan=4>None Identified</td></tr><tr><td rowspan=1 colspan=4>Detection Not Predicted</td></tr><tr><td rowspan=1 colspan=2>Bacillus mycoides*</td><td rowspan=1 colspan=2>Bacillus pseudomycoides</td></tr></table>

Table 32: Predicted (in silico) Reactivity (Inclusivity) Results for Bacillus subtilis group   

<table><tr><td rowspan=1 colspan=4>Detection Predicted for ≥95% of target sequences</td></tr><tr><td rowspan=1 colspan=1>Bacillus subtilis*</td><td rowspan=1 colspan=2>Bacillus licheniformis*</td><td rowspan=1 colspan=1>Bacillus siamensis</td></tr><tr><td rowspan=1 colspan=1>Bacillus amyloliquefaciens*</td><td rowspan=1 colspan=2>Bacillus methylotrophicus /Bacillus velezensis</td><td rowspan=1 colspan=1>Bacillus tequilensis</td></tr><tr><td rowspan=1 colspan=1>Bacillus atrophaeus*</td><td rowspan=1 colspan=2>Bacillus paralicheniformis</td><td rowspan=1 colspan=1>Bacillus vallismortis</td></tr><tr><td rowspan=1 colspan=4>Detection Predicted for 85%-94% of target sequences</td></tr><tr><td rowspan=1 colspan=4>None Identified</td></tr><tr><td rowspan=1 colspan=4>Detection Predicted for &lt;85% of target sequences</td></tr><tr><td rowspan=1 colspan=2>Bacillus mojavensis (77.8%)</td><td rowspan=1 colspan=2>Bacillus sonorensis (83.3%)</td></tr><tr><td rowspan=1 colspan=4>Detection Not Predicted</td></tr><tr><td rowspan=1 colspan=4>None Identified</td></tr></table>

Table 33: Predicted (in silico) Reactivity (Inclusivity) Results for Corynebacterium   

<table><tr><td colspan="3" rowspan="1">Detection Predicted for ≥95% of target sequences</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium afermentans</td><td colspan="1" rowspan="1">Corynebacterium glaucum</td><td colspan="1" rowspan="1">Corynebacterium renale</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium afermentanssubsp. afermentans*</td><td colspan="1" rowspan="1">Corynebacterium halotolerans</td><td colspan="1" rowspan="1">Corynebacterium simulans*</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium afermentanssubsp. lipophilium*</td><td colspan="1" rowspan="1">Corynebacterium hansenii</td><td colspan="1" rowspan="1">Corynebacterium singulare</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium appendicis</td><td colspan="1" rowspan="1">Corynebacteriumhumireducens</td><td colspan="1" rowspan="1">Corynebacterium sputi</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium auris</td><td colspan="1" rowspan="1">Corynebacterium imitans*</td><td colspan="1" rowspan="1">Corynebacterium striatum*</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium auriscanis</td><td colspan="1" rowspan="1">Corynebacteriumlipophiloflavum</td><td colspan="1" rowspan="1">Corynebacterium suicordis</td></tr><tr><td colspan="1" rowspan="1">Corynebacteriumcamporealensis</td><td colspan="1" rowspan="1">Corynebacterium lowii</td><td colspan="1" rowspan="1">Corynebacteriumterpenotabidum</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium casei</td><td colspan="1" rowspan="1">Corynebacterium marinum</td><td colspan="1" rowspan="1">Corynebacterium testudinoris</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium ciconiae</td><td colspan="1" rowspan="1">Corynebacterium massiliense</td><td colspan="1" rowspan="1">Corynebacterium timonense*</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium coyleae*</td><td colspan="1" rowspan="1">Corynebacteriumminutissimum*</td><td colspan="1" rowspan="1">Corynebacterium tuscaniense</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium doosanense</td><td colspan="1" rowspan="1">Corynebacterium mucifaciens</td><td colspan="1" rowspan="1">Corynebacterium ulceribovis</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium falsenii*</td><td colspan="1" rowspan="1">Corynebacterium mycetoides</td><td colspan="1" rowspan="1">Corynebacteriumurealyticum*</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium felinum</td><td colspan="1" rowspan="1">Corynebacterium nuruki</td><td colspan="1" rowspan="1">Corynebacteriumureicelerivorans*</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium freneyi*</td><td colspan="1" rowspan="1">Corynebacterium oculi</td><td colspan="1" rowspan="1">Corynebacterium xerosis</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium genitalium</td><td colspan="1" rowspan="1">Corynebacterium pilosum</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="2" rowspan="1">Detection Predicted for 85%-94% of target sequences</td></tr><tr><td colspan="3" rowspan="1">None Identified</td></tr><tr><td colspan="3" rowspan="1">Detection Predicted for &lt;85% of target sequences</td></tr><tr><td colspan="1" rowspan="1">Corynebacteriumaurimucosum (50.0%)</td><td colspan="1" rowspan="1">Corynebacterium jeikeium(38.7%)*A</td><td colspan="1" rowspan="1">Corynebacterium variabile(75.0%)</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium diphtheriae(76.4%)*</td><td colspan="1" rowspan="1">Corynebacterium ulcerans(47.4%)*A</td><td colspan="1" rowspan="1">Corynebacterium vitaeruminis(75.0%)</td></tr><tr><td colspan="3" rowspan="1">Detection Not Predicted</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium accolens</td><td colspan="1" rowspan="1">Corynebacteriumepidermidicanis</td><td colspan="1" rowspan="1">Corynebacterium propinquum</td></tr><tr><td colspan="1" rowspan="1">Corynebacteriumammoniagenes</td><td colspan="1" rowspan="1">Corynebacterium flavescens</td><td colspan="1" rowspan="1">Corynebacteriumpseudodiphtheriticum</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium amycolatum</td><td colspan="1" rowspan="1">Corynebacteriumfrankenforstense</td><td colspan="1" rowspan="1">Corynebacteriumpseudogenitalium</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium aquilae</td><td colspan="1" rowspan="1">Corynebacterium freiburgense</td><td colspan="1" rowspan="1">Corynebacteriumpseudotuberculosis</td></tr><tr><td colspan="1" rowspan="1">Corynebacteriumargentoratense</td><td colspan="1" rowspan="1">Corynebacteriumglucuronolyticum</td><td colspan="1" rowspan="1">Corynebacteriumpyruviciproducens</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium atypicum</td><td colspan="1" rowspan="1">Corynebacterium glutamicum</td><td colspan="1" rowspan="1">Corynebacterium resistens</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium bovis</td><td colspan="1" rowspan="1">Corynebacteriumglyciniphilum</td><td colspan="1" rowspan="1">Corynebacterium riegelii</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium callunae</td><td colspan="1" rowspan="1">Corynebacteriumkroppenstedtii</td><td colspan="1" rowspan="1">Corynebacterium sphenisci</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium capitovis</td><td colspan="1" rowspan="1">Corynebacterium kutscheri</td><td colspan="1" rowspan="1">Corynebacteriumspheniscorum</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium caspium</td><td colspan="1" rowspan="1">Corynebacterium lactis</td><td colspan="1" rowspan="1">Corynebacterium stationis</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium confusum</td><td colspan="1" rowspan="1">Corynebacterium lubricantis</td><td colspan="1" rowspan="1">Corynebacteriumsundsvallense</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium crenatum</td><td colspan="1" rowspan="1">Corynebacterium maris</td><td colspan="1" rowspan="1">Corynebacterium thomssenii</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium cystitidis</td><td colspan="1" rowspan="1">Corynebacterium mastitidis</td><td colspan="1" rowspan="1">Corynebacteriumtuberculostearicum</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium deserti</td><td colspan="1" rowspan="1">Corynebacterium matruchotii</td><td colspan="1" rowspan="1">Corynebacterium uropygiale</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium durum</td><td colspan="1" rowspan="1">Corynebacterium mustelae</td><td colspan="1" rowspan="1">Corynebacterium uterequi</td></tr><tr><td colspan="1" rowspan="1">Corynebacteriumefficiens</td><td colspan="1" rowspan="1">Corynebacterium phocae</td><td colspan="1" rowspan="1"></td></tr></table>

A. $3 8 . 7 \%$ of sequences in NCBI for C. jeikeium were predicted to be detected bioinformatically; $4 7 . 4 \%$ of sequences in NCBI for C. ulcerans were predicted to be detected bioinformatically. All strains tested for these species were detected as a part of t he Analyt ical React ivit y (Inclusivit y) or Limit of Det ect ion (Analytical Sensit ivit y) st udies.

Table 34: Predicted (in silico) Reactivity (Inclusivity) Results for Enterococcus   

<table><tr><td rowspan=1 colspan=3>Detection Predicted for ≥95% of target sequences</td></tr><tr><td rowspan=1 colspan=1>Enterococcus avium*</td><td rowspan=1 colspan=1>Enterococcus flavescens</td><td rowspan=1 colspan=1>Enterococcus raffinosus</td></tr><tr><td rowspan=1 colspan=1>Enterococcus dispar*</td><td rowspan=1 colspan=1>Enterococcus italicus*</td><td rowspan=1 colspan=1>Enterococcus saccharolyticus*</td></tr><tr><td rowspan=1 colspan=1>Enterococcus durans</td><td rowspan=1 colspan=1>Enterococcus lactis</td><td rowspan=1 colspan=1>Enterococcus thailandicus</td></tr><tr><td rowspan=1 colspan=1>Enterococcus faecalis*</td><td rowspan=1 colspan=1>Enterococcus malodoratus</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Enterococcus faecium*</td><td rowspan=1 colspan=1>Enterococcus pseudoavium</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=3>Detection Predicted for 85%-94% of target sequences</td></tr><tr><td rowspan=1 colspan=1>Enterococcus casseliflavus</td><td rowspan=1 colspan=1>Enterococcus gallinarum</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Enterococcus cecorum</td><td rowspan=1 colspan=1>Enterococcus hirae*</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=3>Detection Predicted for &lt;85% of target sequences</td></tr><tr><td rowspan=1 colspan=3>Enterococcus mundtii (9.1%)</td></tr><tr><td rowspan=1 colspan=3>Detection Not Predicted</td></tr><tr><td rowspan=1 colspan=1>Enterococcus aquimarinus</td><td rowspan=1 colspan=1>Enterococcus hawaiiensis</td><td rowspan=1 colspan=1>Enterococcus rotai</td></tr><tr><td rowspan=1 colspan=1>Enterococcus asini</td><td rowspan=1 colspan=1>Enterococcus hermanniensis</td><td rowspan=1 colspan=1>Enterococcus silesiacus</td></tr><tr><td rowspan=1 colspan=1>Enterococcus caccae</td><td rowspan=1 colspan=1>Enterococcus pallens</td><td rowspan=1 colspan=1>Enterococcus sulfureus</td></tr><tr><td rowspan=1 colspan=1>Enterococcus camelliae</td><td rowspan=1 colspan=1>Enterococcus pernyi</td><td rowspan=1 colspan=1>Enterococcus termitis</td></tr><tr><td rowspan=1 colspan=1>Enterococcus canintestini</td><td rowspan=1 colspan=1>Enterococcus phoeniculicola</td><td rowspan=1 colspan=1>Enterococcus ureasiticus</td></tr><tr><td rowspan=1 colspan=1>Enterococcus canis</td><td rowspan=1 colspan=1>Enterococcus plantarum</td><td rowspan=1 colspan=1>Enterococcus ureilyticus</td></tr><tr><td rowspan=1 colspan=1>Enterococcus columbae</td><td rowspan=1 colspan=1>Enterococcus quebecensis</td><td rowspan=1 colspan=1>Enterococcus villorum</td></tr><tr><td rowspan=1 colspan=1>Enterococcus devriesei</td><td rowspan=1 colspan=1>Enterococcus ratti</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Enterococcus haemoperoxidus</td><td rowspan=1 colspan=1>Enterococcus rivorum</td><td rowspan=1 colspan=1></td></tr></table>

Table 35: Predicted (in silico) Reactivity (Inclusivity) Results for Lactobacillus   

<table><tr><td colspan="3" rowspan="1">Detection Predicted for ≥95% of target sequences</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus casei*</td><td colspan="1" rowspan="1">Lactobacillus rhamnosus*</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Lactobacillus paracasei*</td><td colspan="1" rowspan="1">Lactobacillus zeae*</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="3" rowspan="1">Detection Predicted for 85% -94% of target sequences</td></tr><tr><td colspan="3" rowspan="1">None Identified</td></tr><tr><td colspan="3" rowspan="1">Detection Predicted for &lt;85% of target sequences</td></tr><tr><td colspan="3" rowspan="1">None Identified</td></tr><tr><td colspan="3" rowspan="1">Detection Not Predicted</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus acetotolerans</td><td colspan="1" rowspan="1">Lactobacillus ghanensis</td><td colspan="1" rowspan="1">Lactobacillus panis</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus acidifarinae</td><td colspan="1" rowspan="1">Lactobacillus gigeriorum</td><td colspan="1" rowspan="1">Lactobacillus pantheris</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus acidipiscis</td><td colspan="1" rowspan="1">Lactobacillusginsenosidimutans</td><td colspan="1" rowspan="1">Lactobacillus parabrevis</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus acidophilus</td><td colspan="1" rowspan="1">Lactobacillus gorillae</td><td colspan="1" rowspan="1">Lactobacillus parabuchneri</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus agilis</td><td colspan="1" rowspan="1">Lactobacillus graminis</td><td colspan="1" rowspan="1">Lactobacillus paracollinoides</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus algidus</td><td colspan="1" rowspan="1">Lactobacillus hammesii</td><td colspan="1" rowspan="1">Lactobacillus parafarraginis</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus alimentarius</td><td colspan="1" rowspan="1">Lactobacillus hamsteri</td><td colspan="1" rowspan="1">Lactobacillus parakefiri</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus amylolyticus</td><td colspan="1" rowspan="1">Lactobacillus harbinensis</td><td colspan="1" rowspan="1">Lactobacillus paralimentarius</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus amylophilus</td><td colspan="1" rowspan="1">Lactobacillus hayakitensis</td><td colspan="1" rowspan="1">Lactobacillus paraplantarum</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus amylotrophicus</td><td colspan="1" rowspan="1">Lactobacillusheilongjiangensis</td><td colspan="1" rowspan="1">Lactobacillus pasteurii</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus amylovorus</td><td colspan="1" rowspan="1">Lactobacillushelsingborgensis</td><td colspan="1" rowspan="1">Lactobacillus paucivorans</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus animalis</td><td colspan="1" rowspan="1">Lactobacillus helveticus</td><td colspan="1" rowspan="1">Lactobacillus pentosus</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus antri</td><td colspan="1" rowspan="1">Lactobacillus herbarum</td><td colspan="1" rowspan="1">Lactobacillus perolens</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus apinorum</td><td colspan="1" rowspan="1">Lactobacillus hilgardii</td><td colspan="1" rowspan="1">Lactobacillus plantarum</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus apis</td><td colspan="1" rowspan="1">Lactobacillushokkaidonensis</td><td colspan="1" rowspan="1">Lactobacillus pobuzihii</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus apodemi</td><td colspan="1" rowspan="1">Lactobacillus hominis</td><td colspan="1" rowspan="1">Lactobacillus pontis</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus aquaticus</td><td colspan="1" rowspan="1">Lactobacillus homohiochii</td><td colspan="1" rowspan="1">Lactobacillus psittaci</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus aviarius</td><td colspan="1" rowspan="1">Lactobacillus hordei</td><td colspan="1" rowspan="1">Lactobacillus rapi</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus backii</td><td colspan="1" rowspan="1">Lactobacillus iners</td><td colspan="1" rowspan="1">Lactobacillus rennini</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus bifermentans</td><td colspan="1" rowspan="1">Lactobacillus ingluviei</td><td colspan="1" rowspan="1">Lactobacillus reuteri</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus bombi</td><td colspan="1" rowspan="1">Lactobacillus intestinalis</td><td colspan="1" rowspan="1">Lactobacillus rodentium</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus bombicola</td><td colspan="1" rowspan="1">Lactobacillus jensenii</td><td colspan="1" rowspan="1">Lactobacillus rossiae</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus brantae</td><td colspan="1" rowspan="1">Lactobacillus johnsonii</td><td colspan="1" rowspan="1">Lactobacillus ruminis</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus brevis</td><td colspan="1" rowspan="1">Lactobacillus kalixensis</td><td colspan="1" rowspan="1">Lactobacillus saerimneri</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus buchneri</td><td colspan="1" rowspan="1">Lactobacilluskefiranofaciens</td><td colspan="1" rowspan="1">Lactobacillus sakei</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus cacaonum</td><td colspan="1" rowspan="1">Lactobacillus kefiri</td><td colspan="1" rowspan="1">Lactobacillus salivarius</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus camelliae</td><td colspan="1" rowspan="1">Lactobacillus kimbladii</td><td colspan="1" rowspan="1">Lactobacillus sanfranciscensis</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus capillatus</td><td colspan="1" rowspan="1">Lactobacillus kimchicus</td><td colspan="1" rowspan="1">Lactobacillus saniviri</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus ceti</td><td colspan="1" rowspan="1">Lactobacillus kimchiensis</td><td colspan="1" rowspan="1">Lactobacillus satsumensis</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus coleohominis</td><td colspan="1" rowspan="1">Lactobacillus kisonensis</td><td colspan="1" rowspan="1">Lactobacillus secaliphilus</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus collinoides</td><td colspan="1" rowspan="1">Lactobacillus kitasatonis</td><td colspan="1" rowspan="1">Lactobacillus selangorensis</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus composti</td><td colspan="1" rowspan="1">Lactobacillus koreensis</td><td colspan="1" rowspan="1">Lactobacillus senioris</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus concavus</td><td colspan="1" rowspan="1">Lactobacillus kullabergensis</td><td colspan="1" rowspan="1">Lactobacillus senmaizukei</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus coryniformis</td><td colspan="1" rowspan="1">Lactobacillus kunkeei</td><td colspan="1" rowspan="1">Lactobacillus sharpeae</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus crispatus</td><td colspan="1" rowspan="1">Lactobacillus letivazi</td><td colspan="1" rowspan="1">Lactobacillus shenzhenensis</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus crustorum</td><td colspan="1" rowspan="1">Lactobacillus lindneri</td><td colspan="1" rowspan="1">Lactobacillus silagei</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus curieae</td><td colspan="1" rowspan="1">Lactobacillusmalefermentans</td><td colspan="1" rowspan="1">Lactobacillus siliginis</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus curvatus</td><td colspan="1" rowspan="1">Lactobacillus mali</td><td colspan="1" rowspan="1">Lactobacillus similis</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus delbrueckii</td><td colspan="1" rowspan="1">Lactobacillusmanihotivorans</td><td colspan="1" rowspan="1">Lactobacillus spicheri</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus dextrinicus</td><td colspan="1" rowspan="1">Lactobacillus mellifer</td><td colspan="1" rowspan="1">Lactobacillus sucicola</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus diolivorans</td><td colspan="1" rowspan="1">Lactobacillus mellis</td><td colspan="1" rowspan="1">Lactobacillus suebicus</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus equi</td><td colspan="1" rowspan="1">Lactobacillus melliventris</td><td colspan="1" rowspan="1">Lactobacillus sunkii</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus equicursoris</td><td colspan="1" rowspan="1">Lactobacillus mindensis</td><td colspan="1" rowspan="1">Lactobacillus suntoryeus</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus equigenerosi</td><td colspan="1" rowspan="1">Lactobacillus mucosae</td><td colspan="1" rowspan="1">Lactobacillus taiwanensis</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus fabifermentans</td><td colspan="1" rowspan="1">Lactobacillus murinus</td><td colspan="1" rowspan="1">Lactobacillus thailandensis</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus farciminis</td><td colspan="1" rowspan="1">Lactobacillus nagelii</td><td colspan="1" rowspan="1">Lactobacillus tucceti</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus farraginis</td><td colspan="1" rowspan="1">Lactobacillus namurensis</td><td colspan="1" rowspan="1">Lactobacillus ultunensis</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus fermentum</td><td colspan="1" rowspan="1">Lactobacillus nantensis</td><td colspan="1" rowspan="1">Lactobacillus uvarum</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus floricola</td><td colspan="1" rowspan="1">Lactobacillus nasuensis</td><td colspan="1" rowspan="1">Lactobacillus vaccinostercus</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus florum</td><td colspan="1" rowspan="1">Lactobacillus nodensis</td><td colspan="1" rowspan="1">Lactobacillus vaginalis</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus fructivorans</td><td colspan="1" rowspan="1">Lactobacillus odoratitofui</td><td colspan="1" rowspan="1">Lactobacillus versmoldensis</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus frumenti</td><td colspan="1" rowspan="1">Lactobacillus oeni</td><td colspan="1" rowspan="1">Lactobacillus vini</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus fuchuensis</td><td colspan="1" rowspan="1">Lactobacillusoligofermentans</td><td colspan="1" rowspan="1">Lactobacillus wasatchensis</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus futsaii</td><td colspan="1" rowspan="1">Lactobacillus oris</td><td colspan="1" rowspan="1">Lactobacillus xiangfangensis</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus gallinarum</td><td colspan="1" rowspan="1">Lactobacillus oryzae</td><td colspan="1" rowspan="1">Lactobacillus zymae</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus gasseri</td><td colspan="1" rowspan="1">Lactobacillus otakiensis</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Lactobacillus gastricus</td><td colspan="1" rowspan="1">Lactobacillus ozensis</td><td colspan="1" rowspan="1"></td></tr></table>

Table 36: Predicted (in silico) Reactivity (Inclusivity) Results for Listeria   

<table><tr><td rowspan=1 colspan=3>Detection Predicted for ≥95% of target sequences</td></tr><tr><td rowspan=1 colspan=1>Listeria monocytogenes*</td><td rowspan=1 colspan=1>Listeria marthii</td><td rowspan=1 colspan=1>Listeria welshimeri*</td></tr><tr><td rowspan=1 colspan=1>Listeria ivanovii*</td><td rowspan=1 colspan=1>Listeria seeligeri*</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=3>Detection Predicted for 85%-94% of target sequences</td></tr><tr><td rowspan=1 colspan=3>Listeria innocua*</td></tr><tr><td rowspan=1 colspan=3>Detection Predicted for &lt;85% of target sequences</td></tr><tr><td rowspan=1 colspan=3>None Identified</td></tr><tr><td rowspan=1 colspan=3>Detection Not Predicted</td></tr><tr><td rowspan=1 colspan=3>Listeria grayi</td></tr></table>

Table 37: Predicted (in silico) Reactivity (Inclusivity) Results for Micrococcus   

<table><tr><td rowspan=1 colspan=3>Detection Predicted for ≥95% of target sequences</td></tr><tr><td rowspan=1 colspan=1>Micrococcus luteus*</td><td rowspan=1 colspan=1>Micrococcus aquilus</td><td rowspan=1 colspan=1>Micrococcus yunnanensis*</td></tr><tr><td rowspan=1 colspan=1>Micrococcus alkanovora</td><td rowspan=1 colspan=1>Micrococcus flavus</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Micrococcus aloeverae</td><td rowspan=1 colspan=1>Micrococcus thailandicus</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=3>Detection Predicted for 85%-94% of target sequences</td></tr><tr><td rowspan=1 colspan=1>Micrococcus endophyticus</td><td rowspan=1 colspan=1>Micrococcus indicus</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=3>Detection Predicted for &lt;85% of target sequences</td></tr><tr><td rowspan=1 colspan=1>Micrococcus lylae (50.0%)*</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=3>Detection Not Predicted</td></tr><tr><td rowspan=1 colspan=1>Micrococcus antarcticus</td><td rowspan=1 colspan=1>Micrococcus lactis</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Micrococcus chenggongense</td><td rowspan=1 colspan=1>Micrococcus terreus</td><td rowspan=1 colspan=1></td></tr></table>

Table 38: Predicted (in silico) Reactivity (Inclusivity) Results for Staphylococcus   

<table><tr><td rowspan=1 colspan=4>Detection Predicted for ≥95% of target sequences</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus aureus*</td><td rowspan=1 colspan=2>Staphylococcus haemolyticus*</td><td rowspan=1 colspan=1>Staphylococcuspseudintermedius*</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus agnetis</td><td rowspan=1 colspan=2>Staphylococcus hominis*</td><td rowspan=1 colspan=1>Staphylococcuspseudolugdunensis</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus argensis</td><td rowspan=1 colspan=2>Staphylococcus hominissubsp. novobiosepticus</td><td rowspan=1 colspan=1>Staphylococcus pulvereri</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus argenteus</td><td rowspan=1 colspan=2>Staphylococcus hyicus*</td><td rowspan=1 colspan=1>Staphylococcus rostri</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus auricularis*</td><td rowspan=1 colspan=2>Staphylococcus jettensis</td><td rowspan=1 colspan=1>Staphylococcussaprophyticus*</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus capitis*</td><td rowspan=1 colspan=2>Staphylococcus kloosii</td><td rowspan=1 colspan=1>Staphylococcus schleiferi*</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus caprae*</td><td rowspan=1 colspan=2>Staphylococcus lentus*</td><td rowspan=1 colspan=1>Staphylococcus schweitzeri</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus carnosus*</td><td rowspan=1 colspan=2>Staphylococcus lugdunensis*</td><td rowspan=1 colspan=1>Staphylococcus sciuri*</td></tr><tr><td rowspan=1 colspan=1>Staphylococcuschromogenes*</td><td rowspan=1 colspan=2>Staphylococcus lutrae</td><td rowspan=1 colspan=1>Staphylococcus simiae</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus cohnii*</td><td rowspan=1 colspan=2>Staphylococcus massiliensis</td><td rowspan=1 colspan=1>Staphylococcus simulans*</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus delphini</td><td rowspan=1 colspan=2>Staphylococcus microti</td><td rowspan=1 colspan=1>Staphylococcus stepanovicii</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus devriesei</td><td rowspan=1 colspan=2>Staphylococcus muscae*</td><td rowspan=1 colspan=1>Staphylococcus succinus</td></tr><tr><td rowspan=1 colspan=1>Staphylococcusepidermidis*</td><td rowspan=1 colspan=2>Staphylococcus nepalensis</td><td rowspan=1 colspan=1>Staphylococcus vitulinus*</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus equorum</td><td rowspan=1 colspan=2>Staphylococcus pasteuri*</td><td rowspan=1 colspan=1>Staphylococcus warneri *</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus felis</td><td rowspan=1 colspan=2>Staphylococcus petrasii</td><td rowspan=1 colspan=1>Staphylococcus xylosus*</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus fleurettii</td><td rowspan=1 colspan=2>Staphylococcus pettenkoferi*</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Staphylococcus gallinarum*</td><td rowspan=1 colspan=2>Staphylococcuspiscifermentans</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=4>Detection Predicted for 85%-94% of target sequences</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus arlettae</td><td rowspan=1 colspan=2>Staphylococcus intermedius*</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Staphylococcus condimenti</td><td rowspan=1 colspan=2>Staphylococcussaccharolyticus*</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=4>Detection Predicted for &lt;85% of target sequences</td></tr><tr><td rowspan=1 colspan=4>None Identified</td></tr><tr><td rowspan=1 colspan=4>Detection Not Predicted</td></tr><tr><td rowspan=1 colspan=2>Staphylococcus caseolyticusA</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr></table>

Table 39: Predicted (in silico) Reactivity (Inclusivity) Results for Streptococcus   

<table><tr><td colspan="3" rowspan="1">Detection Predicted for ≥95% of target sequences</td></tr><tr><td colspan="1" rowspan="1">Streptococcus agalactiae*</td><td colspan="1" rowspan="1">Streptococcus infantarius*</td><td colspan="1" rowspan="1">Streptococcus phocae</td></tr><tr><td colspan="1" rowspan="1">Streptococcus alactolyticus</td><td colspan="1" rowspan="1">Streptococcus infantis*</td><td colspan="1" rowspan="1">Streptococcus pneumoniae*</td></tr><tr><td colspan="1" rowspan="1">Streptococcus anginosus*</td><td colspan="1" rowspan="1">Streptococcus intermedius*</td><td colspan="1" rowspan="1">Streptococcus porcinus</td></tr><tr><td colspan="1" rowspan="1">Streptococcus australis</td><td colspan="1" rowspan="1">Streptococcus intestinalis</td><td colspan="1" rowspan="1">Streptococcus porcorum</td></tr><tr><td colspan="1" rowspan="1">Streptococcus caballi</td><td colspan="1" rowspan="1">Streptococcus lactarius</td><td colspan="1" rowspan="1">Streptococcuspseudopneumoniae</td></tr><tr><td colspan="1" rowspan="1">Streptococcus constellatus*</td><td colspan="1" rowspan="1">Streptococcus loxodontasalivarius</td><td colspan="1" rowspan="1">Streptococcus pseudoporcinus</td></tr><tr><td colspan="1" rowspan="1">Streptococcus criceti*</td><td colspan="1" rowspan="1">Streptococcus luteciae</td><td colspan="1" rowspan="1">Streptococcus pyogenes*</td></tr><tr><td colspan="1" rowspan="1">Streptococcus cristatus</td><td colspan="1" rowspan="1">Streptococcus lutetiensis</td><td colspan="1" rowspan="1">Streptococcus rifensis</td></tr><tr><td colspan="1" rowspan="1">Streptococcus danieliae</td><td colspan="1" rowspan="1">Streptococcus macedonicus</td><td colspan="1" rowspan="1">Streptococcus rubneri</td></tr><tr><td colspan="1" rowspan="1">Streptococcus dentasini</td><td colspan="1" rowspan="1">Streptococcusmarimammalium</td><td colspan="1" rowspan="1">Streptococcus salivarius*</td></tr><tr><td colspan="1" rowspan="1">Streptococcus dentisani</td><td colspan="1" rowspan="1">Streptococcus massiliensis</td><td colspan="1" rowspan="1">Streptococcussaliviloxodontae</td></tr><tr><td colspan="1" rowspan="1">Streptococcus didelphis</td><td colspan="1" rowspan="1">Streptococcus mitis*</td><td colspan="1" rowspan="1">Streptococcus sanguinis*</td></tr><tr><td colspan="1" rowspan="1">Streptococcus difficilis</td><td colspan="1" rowspan="1">Streptococcus moroccensis</td><td colspan="1" rowspan="1">Streptococcus seminale</td></tr><tr><td colspan="1" rowspan="1">Streptococcus dysgalactiaesubsp. dysgalactiae</td><td colspan="1" rowspan="1">Streptococcusoligofermentans</td><td colspan="1" rowspan="1">Streptococcus sinensis</td></tr><tr><td colspan="1" rowspan="1">Streptococcus dysgalactiaesubsp. equisimilis</td><td colspan="1" rowspan="1">Streptococcus oralis*</td><td colspan="1" rowspan="1">Streptococcus suis</td></tr><tr><td colspan="1" rowspan="1">Streptococcus dysgalactiae*</td><td colspan="1" rowspan="1">Streptococcus oricebi</td><td colspan="1" rowspan="1">Streptococcus thermophilus</td></tr><tr><td colspan="1" rowspan="1">Streptococcus equi*</td><td colspan="1" rowspan="1">Streptococcus orisratti</td><td colspan="1" rowspan="1">Streptococcus thoraltensis*</td></tr><tr><td colspan="1" rowspan="1">Streptococcus equinus*</td><td colspan="1" rowspan="1">Streptococcus panodentis</td><td colspan="1" rowspan="1">Streptococcus tigurinus</td></tr><tr><td colspan="1" rowspan="1">Streptococcus fryi</td><td colspan="1" rowspan="1">Streptococcusparasanguinis*</td><td colspan="1" rowspan="1">Streptococcus troglodytae</td></tr><tr><td colspan="1" rowspan="1">Streptococcus gallolyticus*</td><td colspan="1" rowspan="1">Streptococcus parasuis</td><td colspan="1" rowspan="1">Streptococcus troglodytidis</td></tr><tr><td colspan="1" rowspan="1">Streptococcus gordonii*</td><td colspan="1" rowspan="1">Streptococcus parauberis</td><td colspan="1" rowspan="1">Streptococcus urinalis</td></tr><tr><td colspan="1" rowspan="1">Streptococcus himalayensis</td><td colspan="1" rowspan="1">Streptococcus pasteuri</td><td colspan="1" rowspan="1">Streptococcus ursoris</td></tr><tr><td colspan="1" rowspan="1">Streptococcus hongkongensis</td><td colspan="1" rowspan="1">Streptococcus pasteurianus</td><td colspan="1" rowspan="1">Streptococcus vestibularis</td></tr><tr><td colspan="1" rowspan="1">Streptococcus hyointestinalis</td><td colspan="1" rowspan="1">Streptococcus peroris*</td><td colspan="1" rowspan="1">Streptococcus waiu</td></tr><tr><td colspan="3" rowspan="1">Detection Predicted for 85%-94% of target sequences</td></tr><tr><td colspan="1" rowspan="1">Streptococcus bovis*</td><td colspan="1" rowspan="1">Streptococcus uberis</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="3" rowspan="1">Detection Predicted for &lt;85% of target sequences</td></tr><tr><td colspan="1" rowspan="1">Streptococcus canis (15.8%)</td><td colspan="1" rowspan="1">Streptococcus henryi(10.0%)</td><td colspan="1" rowspan="1">Streptococcus ratti (75.0%)</td></tr><tr><td colspan="1" rowspan="1">Streptococcus halichoeri(66.7%)</td><td colspan="1" rowspan="1">Streptococcus iniae (6.9%)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="3" rowspan="1">Detection Not Predicted</td></tr><tr><td colspan="1" rowspan="1">Streptococcus acidominimus</td><td colspan="1" rowspan="1">Streptococcus entericus</td><td colspan="1" rowspan="1">Streptococcus orisasini</td></tr><tr><td colspan="1" rowspan="1">Streptococcus azizii</td><td colspan="1" rowspan="1">Streptococcus ferus</td><td colspan="1" rowspan="1">Streptococcus orisuis</td></tr><tr><td colspan="1" rowspan="1">Streptococcus cameli</td><td colspan="1" rowspan="1">Streptococcus gallinaceus</td><td colspan="1" rowspan="1">Streptococcus ovis</td></tr><tr><td colspan="1" rowspan="1">Streptococcus castoreus</td><td colspan="1" rowspan="1">Streptococcus hyovaginalis</td><td colspan="1" rowspan="1">Streptococcus pharyngis</td></tr><tr><td colspan="1" rowspan="1">Streptococcus cremoris</td><td colspan="1" rowspan="1">Streptococcus ictaluri</td><td colspan="1" rowspan="1">Streptococcus pluranimalium</td></tr><tr><td colspan="1" rowspan="1">Streptococcus criae</td><td colspan="1" rowspan="1">Streptococcus lactis</td><td colspan="1" rowspan="1">Streptococcus plurextorum</td></tr><tr><td colspan="1" rowspan="1">Streptococcus cuniculi</td><td colspan="1" rowspan="1">Streptococcus macacae</td><td colspan="1" rowspan="1">Streptococcus plutanimalium</td></tr><tr><td colspan="1" rowspan="1">Streptococcus dentapri</td><td colspan="1" rowspan="1">Streptococcus marmotae</td><td colspan="1" rowspan="1">Streptococcus porci</td></tr><tr><td colspan="1" rowspan="1">Streptococcus dentiloxodontae</td><td colspan="1" rowspan="1">Streptococcus merionis</td><td colspan="1" rowspan="1">Streptococcus rupicaprae</td></tr><tr><td colspan="1" rowspan="1">Streptococcus dentirousetti</td><td colspan="1" rowspan="1">Streptococcus milleri</td><td colspan="1" rowspan="1">Streptococcus sobrinus</td></tr><tr><td colspan="1" rowspan="1">Streptococcus devriesei</td><td colspan="1" rowspan="1">Streptococcus minor</td><td colspan="1" rowspan="1">Streptococcus tangierensis</td></tr><tr><td colspan="1" rowspan="1">Streptococcus downei</td><td colspan="1" rowspan="1">Streptococcusoriloxodontae</td><td colspan="1" rowspan="1"></td></tr></table>

Table 40: Predicted (in silico) Reactivity (Inclusivity) Results for Streptococcus anginosus group   

<table><tr><td rowspan=1 colspan=3>Detection Predicted for ≥95% of target sequences</td></tr><tr><td rowspan=1 colspan=1>Streptococcus anginosus*</td><td rowspan=1 colspan=1>Streptococcus constellatus*</td><td rowspan=1 colspan=1>Streptococcus intermedius*</td></tr><tr><td rowspan=1 colspan=3>Detection Predicted for 85%-94% of target sequences</td></tr><tr><td rowspan=1 colspan=3>None Identified</td></tr><tr><td rowspan=1 colspan=3>Detection Predicted for &lt;85% of target sequences</td></tr><tr><td rowspan=1 colspan=3>None Identified</td></tr><tr><td rowspan=1 colspan=3>Detection Not Predicted</td></tr><tr><td rowspan=1 colspan=3>None Identified</td></tr></table>

Table 41: Predicted (in silico) Reactivity (Inclusivity) Results for Pan Candida   

<table><tr><td rowspan=1 colspan=4>Detection Predicted for ≥95% of target sequences</td></tr><tr><td rowspan=1 colspan=1>Candida albicans*</td><td rowspan=1 colspan=2>Candida glabrata*</td><td rowspan=1 colspan=1>Candida krusei*</td></tr><tr><td rowspan=1 colspan=1>Candida parapsilosis*</td><td rowspan=1 colspan=2></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=4>Detection Predicted for 85%-94% of target sequences</td></tr><tr><td rowspan=1 colspan=4>None Identified</td></tr><tr><td rowspan=1 colspan=4>Detection Predicted for &lt;85% of target sequences</td></tr><tr><td rowspan=1 colspan=4>None Identified</td></tr><tr><td rowspan=1 colspan=4>Detection Not Predicted</td></tr><tr><td rowspan=1 colspan=2>Candida lusitaniae*</td><td rowspan=1 colspan=1>Candida orthopsilosis*</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=2>Candida metapsilosis*</td><td rowspan=1 colspan=1>Candida tropicalis*</td><td rowspan=1 colspan=1></td></tr></table>

# Predicted (in silico) Reactivity (Inclusivity) Results for Pan Gram-Negative

The Pan Gram-Negative assay was designed to be broadly inclusive of the majority of gram-negative organisms.

# g. Analytical specificity (Exclusivity):

Cross-reactivity of on-panel and off-panel analytes was evaluated with the BCID-GP Panel. Bacterial targets were tested in triplicate at a concentration of ${ \geq } 1 \mathbf { x } 1 0 ^ { 9 }$ CFU/mL while fungi were tested in triplicate at a concentration of $\geq 1 \mathbf { x } 1 0 ^ { 7 }$ CFU/mL. If the target concentration could not be reached, the organism was diluted 2-fold from stock for use.

No cross reactivity was observed for any of the on-panel gram-positive organisms.

Three organisms showed cross-reactivity, Burkholderia cepacia cross reacts with the Corynebacterium assay at levels $\geq 1 \mathbf { x } 1 0 ^ { 7 }$ CFU/mL, an unspeciated Rhodococcus strain (ATCC 49988) cross reacts with the Micrococcus assay at levels $\geq 1 \mathbf { x } 1 0 ^ { 7 }$ CFU/mL and Bacillus badius cross reacts with the Bacillus subtilis group assay at $7 \mathrm { ~ x ~ } 1 0 ^ { 7 }$ CFU/mL. See Table 54 and Table 55 for summaries of the on-panel strains tested as a part of the Limit of Detection (Analytical Sensitivity) and Analytical Reactivity (Inclusivity) studies and Table 42 for a summary of off-panel strains tested.

Table 42: Targets Assessed for Cross-Reactivity with the ePlex BCID-GP Panel (Exclusivity)   

<table><tr><td colspan="1" rowspan="1">Organism</td><td colspan="1" rowspan="1">Strain ID</td><td colspan="1" rowspan="1">Organism</td><td colspan="1" rowspan="1">Strain ID</td></tr><tr><td colspan="1" rowspan="1">Abiotrophia defectiva</td><td colspan="1" rowspan="1">ATCC 49176</td><td colspan="1" rowspan="1">Granulicatella elegans</td><td colspan="1" rowspan="1">ATCC 700633c</td></tr><tr><td colspan="1" rowspan="1">Actinomyces odontolyticus</td><td colspan="1" rowspan="1">ATCC 17929</td><td colspan="1" rowspan="1">Kocuria kristinae</td><td colspan="1" rowspan="1">ATCC BAA-752</td></tr><tr><td colspan="1" rowspan="1">Aerococcus sanguinicola</td><td colspan="1" rowspan="1">ATCC BAA-465</td><td colspan="1" rowspan="1">Kocuria rhizophila</td><td colspan="1" rowspan="1">ATCC 9341</td></tr><tr><td colspan="1" rowspan="1">Aerococcus urinae</td><td colspan="1" rowspan="1">ATCC 700306</td><td colspan="1" rowspan="1">Kytococcus sedentarius</td><td colspan="1" rowspan="1">ATCC 27575</td></tr><tr><td colspan="1" rowspan="1">Aerococcus viridans</td><td colspan="1" rowspan="1">ATCC 10400</td><td colspan="1" rowspan="1">Lactococcus garvieae</td><td colspan="1" rowspan="1">ATCC 43921D</td></tr><tr><td colspan="1" rowspan="1">Anaerococcus prevotii</td><td colspan="1" rowspan="1">ATCC 9321</td><td colspan="1" rowspan="1">Lactococcus lactis</td><td colspan="1" rowspan="1">ATCC 49032</td></tr><tr><td colspan="1" rowspan="1">Arcanobacterium bernardiae</td><td colspan="1" rowspan="1">ATCC 51728</td><td colspan="1" rowspan="1">Leuconostoc carnosum</td><td colspan="1" rowspan="1">ATCC 49367</td></tr><tr><td colspan="1" rowspan="1">Arcanobacteriumhaemolyticum</td><td colspan="1" rowspan="1">ATCC BAA-1784A</td><td colspan="1" rowspan="1">Leuconostoc citreum</td><td colspan="1" rowspan="1">ATCC 13146</td></tr><tr><td colspan="1" rowspan="1">Arthrobacterpsychrolactophilus</td><td colspan="1" rowspan="1">ATCC 700733</td><td colspan="1" rowspan="1">Leuconostoc mesenteroides</td><td colspan="1" rowspan="1">ATCC 8293</td></tr><tr><td colspan="1" rowspan="1">Aspergillus fumigatus</td><td colspan="1" rowspan="1">ATCC 204305B</td><td colspan="1" rowspan="1">Macrococcus caseolyticus</td><td colspan="1" rowspan="1">ATCC 29750</td></tr><tr><td colspan="1" rowspan="1">Bacillus badius</td><td colspan="1" rowspan="1">ATCC 14574</td><td colspan="1" rowspan="1">Macrococcus caseolyticus</td><td colspan="1" rowspan="1">ATCC 51834</td></tr><tr><td colspan="1" rowspan="1">Bacillus pumilus</td><td colspan="1" rowspan="1">ATCC 14884</td><td colspan="1" rowspan="1">Mycobacterium avium complex</td><td colspan="1" rowspan="1">ATCC 15769</td></tr><tr><td colspan="1" rowspan="1">Brochothrix thermosphacta</td><td colspan="1" rowspan="1">ATCC 11509</td><td colspan="1" rowspan="1">Mycobacterium fortuitum</td><td colspan="1" rowspan="1">ATCC 6841</td></tr><tr><td colspan="1" rowspan="1">Candida lusitaniae</td><td colspan="1" rowspan="1">ATCC 34449</td><td colspan="1" rowspan="1">Mycobacterium mucogenicum</td><td colspan="1" rowspan="1">ATCC 49650</td></tr><tr><td colspan="1" rowspan="1">Candida metapsilosis</td><td colspan="1" rowspan="1">ATCC 96144</td><td colspan="1" rowspan="1">Mycobacterium tuberculosis</td><td colspan="1" rowspan="1">ATCC 25177</td></tr><tr><td colspan="1" rowspan="1">Candida orthopsilosis</td><td colspan="1" rowspan="1">ATCC 96139</td><td colspan="1" rowspan="1">Mycoplasma pneumoniae</td><td colspan="1" rowspan="1">ATCC 39505</td></tr><tr><td colspan="1" rowspan="1">Candida tropicalis</td><td colspan="1" rowspan="1">ATCC 1369</td><td colspan="1" rowspan="1">Pediococcus acidilactici</td><td colspan="1" rowspan="1">ATCC 8042</td></tr><tr><td colspan="1" rowspan="1">Carnobacterium divergens</td><td colspan="1" rowspan="1">ATCC 35677</td><td colspan="1" rowspan="1">Pediococcus pentosaceus</td><td colspan="1" rowspan="1">ATCC 33316</td></tr><tr><td colspan="1" rowspan="1">Carnobacteriummaltaromaticum</td><td colspan="1" rowspan="1">ATCC 27865c</td><td colspan="1" rowspan="1">Penicillium marneffei</td><td colspan="1" rowspan="1">ATCC 200050</td></tr><tr><td colspan="1" rowspan="1">Cellulomonas turbata</td><td colspan="1" rowspan="1">ATCC 25835</td><td colspan="1" rowspan="1">Peptostreptococcus anaerobius</td><td colspan="1" rowspan="1">ATCC 27337</td></tr><tr><td colspan="1" rowspan="1">Cellulosimicrobium cellulans</td><td colspan="1" rowspan="1">ATCC 27402</td><td colspan="1" rowspan="1">Planococcus species</td><td colspan="1" rowspan="1">ATCC 35671</td></tr><tr><td colspan="1" rowspan="1">Clostridium clostridioforme</td><td colspan="1" rowspan="1">ATCC 25537</td><td colspan="1" rowspan="1">Propionibacteriumpropionicum</td><td colspan="1" rowspan="1">ATCC 14157</td></tr><tr><td colspan="1" rowspan="1">Clostridium perfringens</td><td colspan="1" rowspan="1">ATCC 13124</td><td colspan="1" rowspan="1">Rhodococcus equi</td><td colspan="1" rowspan="1">ATCC 6939</td></tr><tr><td colspan="1" rowspan="1">Clostridium ramosum</td><td colspan="1" rowspan="1">ATCC 25582</td><td colspan="1" rowspan="1">Rhodococcus species</td><td colspan="1" rowspan="1">ATCC 49988</td></tr><tr><td colspan="1" rowspan="1">Cryptococcus gatti</td><td colspan="1" rowspan="1">ATCC 76108</td><td colspan="1" rowspan="1">Rhodotorula glutinis</td><td colspan="1" rowspan="1">ATCC 32766</td></tr><tr><td colspan="1" rowspan="1">Cryptococcus grubii</td><td colspan="1" rowspan="1">ATCC 208821</td><td colspan="1" rowspan="1">Rhodotorula minuta</td><td colspan="1" rowspan="1">ATCC 36236</td></tr><tr><td colspan="1" rowspan="1">Cryptococcus neoformans</td><td colspan="1" rowspan="1">ATCC 14116</td><td colspan="1" rowspan="1">Rhodotorula mucilaginosa</td><td colspan="1" rowspan="1">ATCC 9449</td></tr><tr><td colspan="1" rowspan="1">Cutibacterium granulosum</td><td colspan="1" rowspan="1">ATCC 11829</td><td colspan="1" rowspan="1">Rothia dentocariosa</td><td colspan="1" rowspan="1">ATCC 31918</td></tr><tr><td colspan="1" rowspan="1">Erysipelothrix rhusiopathiae</td><td colspan="1" rowspan="1">ATCC 35457</td><td colspan="1" rowspan="1">Rothia mucilaginosa</td><td colspan="1" rowspan="1">ATCC 25296</td></tr><tr><td colspan="1" rowspan="1">Gemella haemolysans</td><td colspan="1" rowspan="1">ATCC 10379</td><td colspan="1" rowspan="1">Saccharomyces cerevisiae</td><td colspan="1" rowspan="1">ATCC 18824</td></tr><tr><td colspan="1" rowspan="1">Gemella morbillorum</td><td colspan="1" rowspan="1">ATCC 27824</td><td colspan="1" rowspan="1">Trichosporon asahii</td><td colspan="1" rowspan="1">ATCC 201110</td></tr><tr><td colspan="1" rowspan="1">Geotrichum capitatum</td><td colspan="1" rowspan="1">ATCC 10663</td><td colspan="1" rowspan="1">Vagococcus fluvialis</td><td colspan="1" rowspan="1">ATCC 49515</td></tr><tr><td colspan="1" rowspan="1">Granulicatella adiacens</td><td colspan="1" rowspan="1">ATCC 43205</td><td colspan="1" rowspan="1">Weissella paramesenteroides</td><td colspan="1" rowspan="1">ATCC 33313</td></tr></table>

A. Final testing concentration of $4 . 0 5 { \mathrm { x } } 1 0 ^ { 8 } { \mathrm { C F U / m L } }$ C. Final testing concentration of $\overline { { 3 . 6 3 \mathrm { x } 1 0 ^ { 8 } \mathrm { C F U / m L } } }$ B. Final testing concentration of $2 . 5 { \mathrm { x } } 1 0 ^ { 6 } { \mathrm { C F U / m L } }$ D. Final testing concentration of $2 . 7 8 \mathrm { x } 1 0 ^ { 8 } \mathrm { C F U / m L }$

Three off-panel organisms showed cross-reactivity, Burkholderia cepacia cross reacts with the Corynebacterium assay at levels $\geq 1 \mathbf { x } 1 0 ^ { 7 }$ CFU/mL, an unspeciated Rhodococcus strain (ATCC 49988) cross reacts with the Micrococcus assay at levels $\geq 1 \mathbf { x } 1 0 ^ { 7 }$ CFU/mL and Bacillus badius cross reacts with the Bacillus subtilis group assay at $1 \mathrm { { x } 1 0 ^ { 8 } \ C F U / m L }$ .

h. Assay cut-off:

Analytical studies were conducted to establish the signal boundaries for all targets and controls of the ePlex BCID-GP Panel.

A mixture of clinical samples, contrived bottle positives, and organisms spiked at or above the analytically determined limit of detection were used as samples for the study. Positive data points from samples at or greater than the determined LoD test concentration in the Limit of Detection study were used to supplement the Cutoff study data. Negative data points for all targets were obtained from the Limit of Blank study. Expected negative data points from replicates that were positive for other targets in the LoD and Cutoff studies were also combined to increase statistical power of the ROC analysis.

For each target, the signals for positive and negative tests were analyzed. An appropriate boundary was established wherein specificity and sensitivity were maximized. The analysis was verified by ROC analysis. The final boundary set points are listed in Table 43 below.

Table 43: Summary of BCID-GP Panel Boundary Set   

<table><tr><td colspan="1" rowspan="1">Target Name</td><td colspan="1" rowspan="1">Boundary (nA)</td></tr><tr><td colspan="1" rowspan="1">Bacillus cereus group</td><td colspan="1" rowspan="1">20</td></tr><tr><td colspan="1" rowspan="1">Bacillus subtilis group</td><td colspan="1" rowspan="1">30</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium</td><td colspan="1" rowspan="1">10</td></tr><tr><td colspan="1" rowspan="1">Cutibacterium acnes/P. acnes</td><td colspan="1" rowspan="1">50</td></tr><tr><td colspan="1" rowspan="1">Enterococcus</td><td colspan="1" rowspan="1">20</td></tr><tr><td colspan="1" rowspan="1">Enterococcus faecalis</td><td colspan="1" rowspan="1">15</td></tr><tr><td colspan="1" rowspan="1">Enterococcus faecium</td><td colspan="1" rowspan="1">20-400*</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus</td><td colspan="1" rowspan="1">20</td></tr><tr><td colspan="1" rowspan="1">Listeria</td><td colspan="1" rowspan="1">20</td></tr><tr><td colspan="1" rowspan="1">Listeria monocytogenes</td><td colspan="1" rowspan="1">20</td></tr><tr><td colspan="1" rowspan="1">Micrococcus</td><td colspan="1" rowspan="1">30</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus</td><td colspan="1" rowspan="1">20</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">15</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus epidermidis</td><td colspan="1" rowspan="1">20</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus lugdunensis</td><td colspan="1" rowspan="1">20</td></tr><tr><td colspan="1" rowspan="1">Streptococcus</td><td colspan="1" rowspan="1">30</td></tr><tr><td colspan="1" rowspan="1">Streptococcus agalactiae</td><td colspan="1" rowspan="1">50</td></tr><tr><td colspan="1" rowspan="1">Streptococcus anginosus group</td><td colspan="1" rowspan="1">30</td></tr><tr><td colspan="1" rowspan="1">Streptococcus pneumoniae</td><td colspan="1" rowspan="1">20</td></tr><tr><td colspan="1" rowspan="1">Streptococcus pyogenes</td><td colspan="1" rowspan="1">30</td></tr><tr><td colspan="1" rowspan="1">Pan Candida</td><td colspan="1" rowspan="1">40</td></tr><tr><td colspan="1" rowspan="1">Stenotrophomonas maltophilia</td><td colspan="1" rowspan="1">20</td></tr><tr><td colspan="1" rowspan="1">Pan Gram-Negative</td><td colspan="1" rowspan="1">40</td></tr><tr><td colspan="1" rowspan="1">mecA</td><td colspan="1" rowspan="1">20</td></tr><tr><td colspan="1" rowspan="1">mecC</td><td colspan="1" rowspan="1">20</td></tr><tr><td colspan="1" rowspan="1">vanA</td><td colspan="1" rowspan="1">20</td></tr><tr><td colspan="1" rowspan="1">vanB</td><td colspan="1" rowspan="1">20</td></tr><tr><td colspan="1" rowspan="1">Schizosaccharomyces pombe</td><td colspan="1" rowspan="1">20</td></tr><tr><td colspan="1" rowspan="1">Synthetic Control</td><td colspan="1" rowspan="1">20</td></tr></table>

\*Enterococcus faecium is considered detected if the signal is ${ \ge } 2 0 $ nA for both Enterococcus faecium and the Enterococcus genus call. Enterococcus faecium is considered detected independent of the Enterococcus species call if the signal is above $4 0 0 \mathrm { n A }$ .

# i. Interference:

Two organism mixes consisting of 9 on-panel organisms representing 14 targets and negative blood matrix were used to assess eighteen potentially interfering substances and thirteen bottle types for interference. Potentially interfering test substances were spiked at levels predicted to be above the concentration of the substance likely to be found in a blood culture specimen. The positivity of the organisms for each potentially interfering substance at the initial testing concentration is summarized in Table 44 and 45.

Table 44: Composition of Test Mixes   

<table><tr><td colspan="1" rowspan="1">Mix</td><td colspan="1" rowspan="1">PrimerPool</td><td colspan="1" rowspan="1">Organism(Test Concentration)</td><td colspan="1" rowspan="1">BCID-GP Panel Expected Result</td></tr><tr><td colspan="1" rowspan="3">1</td><td colspan="1" rowspan="1">1, 7, 2</td><td colspan="1" rowspan="1">Enterococcus faecalis/vanB(4x107 CFU/mL)</td><td colspan="1" rowspan="1">EnterococcusEnterococcus faecalisvanB</td></tr><tr><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">Lactobacillus casei(4x107 CFU/mL)</td><td colspan="1" rowspan="1">Lactobacillus casei</td></tr><tr><td colspan="1" rowspan="1">6,8</td><td colspan="1" rowspan="1">Staphylococcus epidermidis(2x107 CFU/mL)</td><td colspan="1" rowspan="1">StaphylococcusStaphylococcus epidermidis</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">Cutibacterium acnes(3x108 CFU/mL)</td><td colspan="1" rowspan="1">Cutibacterium acnes</td></tr><tr><td colspan="1" rowspan="5">2</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">Streptococcus pneumoniae(4x107 CFU/mL)</td><td colspan="1" rowspan="1">StreptococcusStreptococcus pneumoniae</td></tr><tr><td colspan="1" rowspan="1">1, 4, 3</td><td colspan="1" rowspan="1">Enterococcus faecium/vanA(4x107 CFU/mL)</td><td colspan="1" rowspan="1">EnterococcusEnterococcus faeciumvanA</td></tr><tr><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">Candida albicans(1x106 CFU/mL)</td><td colspan="1" rowspan="1">Pan Canadida</td></tr><tr><td colspan="1" rowspan="1">7</td><td colspan="1" rowspan="1">Klebsiella pneumoniae(5x108 CFU/mL)</td><td colspan="1" rowspan="1">Pan Gram-Negative</td></tr><tr><td colspan="1" rowspan="1">6,8</td><td colspan="1" rowspan="1">Staphylococcus aureus(2x107 CFU/mL)</td><td colspan="1" rowspan="1">StaphylococcusStaphylococcus aureus</td></tr></table>

Table 45: List of Potentially Interfering Substances   

<table><tr><td rowspan=1 colspan=1>Endogenous Substances</td><td rowspan=1 colspan=1>Testing Concentration</td></tr><tr><td rowspan=1 colspan=1>Bilirubin</td><td rowspan=1 colspan=1>60 µg/mL</td></tr><tr><td rowspan=1 colspan=1>Hemoglobin</td><td rowspan=1 colspan=1>0.6 g/L</td></tr><tr><td rowspan=1 colspan=1>Human genomic DNA</td><td rowspan=1 colspan=1>6 x 105 copies/mL</td></tr><tr><td rowspan=1 colspan=1>Triglycerides</td><td rowspan=1 colspan=1>1000 mg/dL</td></tr><tr><td rowspan=1 colspan=1>γ-globulin</td><td rowspan=1 colspan=1>0.425 g/dL</td></tr><tr><td rowspan=1 colspan=1>Exogenous Substances</td><td rowspan=1 colspan=1>Testing Concentration</td></tr><tr><td rowspan=1 colspan=1>Amoxic illin/Clavulanate</td><td rowspan=1 colspan=1>3.5 μg/mL</td></tr><tr><td rowspan=1 colspan=1>Amphoteric in B</td><td rowspan=1 colspan=1>2 μg/mL^</td></tr><tr><td rowspan=1 colspan=1>Caspofungin</td><td rowspan=1 colspan=1>5 μg/mL</td></tr><tr><td rowspan=1 colspan=1>Ceftriaxone</td><td rowspan=1 colspan=1>0.23 mg/mL</td></tr><tr><td rowspan=1 colspan=1>Ciprofloxac in</td><td rowspan=1 colspan=1>3 mg/L</td></tr><tr><td rowspan=1 colspan=1>Fluconazole</td><td rowspan=1 colspan=1>25 mg/L</td></tr><tr><td rowspan=1 colspan=1>Flucytosine</td><td rowspan=1 colspan=1>90 μg/mL</td></tr><tr><td rowspan=1 colspan=1>Gentamic in sulfate</td><td rowspan=1 colspan=1>3 μg/mL</td></tr><tr><td rowspan=1 colspan=1>Heparin</td><td rowspan=1 colspan=1>0.9 U/mL</td></tr><tr><td rowspan=1 colspan=1>Imipenem</td><td rowspan=1 colspan=1>83 μg/mL</td></tr><tr><td rowspan=1 colspan=1>Sodium Polyanetholsulfonate</td><td rowspan=1 colspan=1>0.25% w/v</td></tr><tr><td rowspan=1 colspan=1>Tetracycline</td><td rowspan=1 colspan=1>5 mg/L</td></tr><tr><td rowspan=1 colspan=1>Vancomycin</td><td rowspan=1 colspan=1>30 mg/L</td></tr></table>

No false positives were detected in negative blood matrix runs without or with interfering substances.

Thirteen of 14 targets were detected in all conditions tested but $C .$ acnes was detected in only 2 of 3 runs in the presence of $\gamma$ -globulin. Upon retest with an additional 6 replicates per the rerun criteria for false negatives defined in the testing protocol, an additional C. acnes false negative was observed. Therefore, $\gamma$ -globulin was diluted and tested until all expected targets were detected.

# k. Mixed Culture Study (Microbial interference):

A study was performed to evaluate whether the presence of high levels of organisms will interfere with detection of representative organisms at bottle positivity and bottle positivity $+ 8$ hours concentrations. The study evaluated the possibility of high concentration organisms interfering with detection of lower concentration targets in the same sample. Eight clinically-relevant co-infection organisms were selected for the study. Selection of the organisms in the mixes was based on those that have been identified in peer-reviewed literature as common polymicrobial co-infections. Testing was performed in triplicate on the BCID-GP Panel

Table 46: Co-Infections Mix Concentrations   

<table><tr><td rowspan=1 colspan=1>Mix</td><td rowspan=1 colspan=1>Organis ms</td><td rowspan=1 colspan=1>Bottle PositivityConce ntration(CFU/mL)</td><td rowspan=1 colspan=1>Bottle Positivity +8 HrsConcentration(CFU/mL)</td></tr><tr><td rowspan=2 colspan=1>1</td><td rowspan=1 colspan=1>Staphylococcus epidermidis</td><td rowspan=1 colspan=1>2 x 107</td><td rowspan=1 colspan=1>1 x 108*</td></tr><tr><td rowspan=1 colspan=1>Streptococcus agalactiae</td><td rowspan=1 colspan=1>4 x 107</td><td rowspan=1 colspan=1>4 x 108</td></tr><tr><td rowspan=2 colspan=1>2</td><td rowspan=1 colspan=1>Streptococcus pneumoniae</td><td rowspan=1 colspan=1>4 x 107</td><td rowspan=1 colspan=1>4 x 108</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus aureus</td><td rowspan=1 colspan=1>2 x 107</td><td rowspan=1 colspan=1>1 x 108*</td></tr><tr><td rowspan=2 colspan=1>3</td><td rowspan=1 colspan=1>Enterococcus faecium</td><td rowspan=1 colspan=1>4 x 107</td><td rowspan=1 colspan=1>1 x 108*</td></tr><tr><td rowspan=1 colspan=1>Cutibacterium granulosum**</td><td rowspan=1 colspan=1>3 x 108</td><td rowspan=1 colspan=1>1 x 109*</td></tr><tr><td rowspan=2 colspan=1>4</td><td rowspan=1 colspan=1>Escherichia coli</td><td rowspan=1 colspan=1>2 x 108</td><td rowspan=1 colspan=1>1 x 109</td></tr><tr><td rowspan=1 colspan=1>Candida albicans</td><td rowspan=1 colspan=1>1 x 106</td><td rowspan=1 colspan=1>1 x 107*</td></tr></table>

\*Per Bottle Positivity Study, this organism concentration did not increase after 8 hours of culture post bottle ring. T he concentration at bottle positivity was rounded up to the next integer log concentration to serve as a second (high) test concentration for this study. \*\*Cutibacterium granulosum is an off-panel organism (not detected by BCID-GP Panel).

Study results demonstrated that high concentrations of the on-panel BCID-GP microorganisms spiked into blood culture samples produced positive results for the relevant assays on the BCID-GP Panel but did interfere with any expected results for other analytes. High concentrations of off-panel BCID-GP microorganisms also showed no interference with the detection of any ePlex BCID-GP organism with no unexpected false negative or false positive results observed.

# l. Testing of Additional Blood Culture Bottle Types

Thirteen different blood culture bottle types from three different blood culture systems (BacT/Alert, BACTEC and VersaTREK) were evaluated analytically with the BCID-GP Panel. Blood culture bottles/media were tested with the recommended ratio of blood to media. Testing was performed using two organism mixes whose members were present at a concentration approximating blood culture bottle positivity in a clinical sample. These organism mixes represent each of the eight multiplex amplification pools on the BCID-GP Panel and include common grampositive bacteria, drug resistance genes, bottle contaminants (Lactobacillus, Corynebacterium, Cutibacterium), gram-negative (Klebsiella pneumoniae) and fungal (Candida albicans) targets. The 13 bottle types listed in the table below were evaluated and study results demonstrated correct positive and negative BCID-GP Panel results with each bottle type.

Table 47: Sample Matrix Equivalency (Bottle Evaluation) Bottle Types   

<table><tr><td rowspan=1 colspan=1>Manufacturer</td><td rowspan=1 colspan=1>Bottle Brand</td><td rowspan=1 colspan=1>Bottle Type</td><td rowspan=1 colspan=1>Study Outcome</td></tr><tr><td rowspan=1 colspan=1>BD</td><td rowspan=1 colspan=1>BACTEC</td><td rowspan=1 colspan=1>Plus Aerobic</td><td rowspan=1 colspan=1>No interference observed</td></tr><tr><td rowspan=1 colspan=1>BD</td><td rowspan=1 colspan=1>BACTEC</td><td rowspan=1 colspan=1>Plus Anaerobic</td><td rowspan=1 colspan=1>False positive results for Pan Gram-Negative target were observed in onelot.</td></tr><tr><td rowspan=1 colspan=1>BD</td><td rowspan=1 colspan=1>BACTEC</td><td rowspan=1 colspan=1>Standard Aerobic</td><td rowspan=1 colspan=1>No interference observed</td></tr><tr><td rowspan=1 colspan=1>BD</td><td rowspan=1 colspan=1>BACTEC</td><td rowspan=1 colspan=1>Standard Anaerobic</td><td rowspan=1 colspan=1>No interference observed</td></tr><tr><td rowspan=1 colspan=1>BD</td><td rowspan=1 colspan=1>BACTEC</td><td rowspan=1 colspan=1>Peds PlusTM</td><td rowspan=1 colspan=1>No interference observed</td></tr><tr><td rowspan=1 colspan=1>BD</td><td rowspan=1 colspan=1>BACTEC</td><td rowspan=1 colspan=1>Lytic Anaerobic</td><td rowspan=1 colspan=1>No interference observed</td></tr><tr><td rowspan=1 colspan=1>bioMérieux</td><td rowspan=1 colspan=1>BACT/ALERT</td><td rowspan=1 colspan=1>SA Standard Aerobic</td><td rowspan=1 colspan=1>No interference observed</td></tr><tr><td rowspan=1 colspan=1>bioMérieux</td><td rowspan=1 colspan=1>BACT/ALERT</td><td rowspan=1 colspan=1>SN Standard Anaerobic</td><td rowspan=1 colspan=1>No interference observed</td></tr><tr><td rowspan=1 colspan=1>bioMérieux</td><td rowspan=1 colspan=1>BACT/ALERT</td><td rowspan=1 colspan=1>FA Plus</td><td rowspan=1 colspan=1>No interference observed</td></tr><tr><td rowspan=1 colspan=1>bioMérieux</td><td rowspan=1 colspan=1>BACT/ALERT</td><td rowspan=1 colspan=1>FN Plus</td><td rowspan=1 colspan=1>False negative results were observedfor Pan Gram-Negative and E.faecium/vanA targets</td></tr><tr><td rowspan=1 colspan=1>bioMérieux</td><td rowspan=1 colspan=1>BACT/ALERT</td><td rowspan=1 colspan=1>PF Plus</td><td rowspan=1 colspan=1>No interference observed</td></tr><tr><td rowspan=1 colspan=1>Thermo Scientific</td><td rowspan=1 colspan=1>VersaTREK</td><td rowspan=1 colspan=1>REDOX 1 EZ Draw Aerobic</td><td rowspan=1 colspan=1>No interference observed</td></tr><tr><td rowspan=1 colspan=1>Thermo Scientific</td><td rowspan=1 colspan=1>VersaTREK</td><td rowspan=1 colspan=1>REDOX 2 EZ Draw Anaerobic</td><td rowspan=1 colspan=1>No interference observed</td></tr></table>

Eleven of the bottle types tested showed no interference for any of the targets tested. BD Bactec Anaerobic/F had $9 5 \%$ detection of $C .$ . acnes; this organism was tested at Limit of Detection (LoD) concentration. BioMérieux BacT Alert Standard Aerobic observed false positives in the control runs in one of two tested lots. However, false positives were not detected in samples from seven bottle lots used in the clinical study. Together, this data suggest that the bottle is not inhibitory and it is considered acceptable for use with the BCID-GP Panel, but an elevated level of false positive results is possible with certain bottle lots. The appropriate limitation is included in the package insert. The BioMérieux BacT Alert FN bottle type may result in lower sensitivity for some targets. The appropriate limitation statement for this specific bottle type is included in the package insert.

The carryover/cross-contamination rate of the ePlex BCID-GP Panel and ePlex instrument was evaluated using a checkerboard approach by running high positive and negative samples interspersed in all bays of a four-tower ePlex instrument (i.e., 24 bays total) over five separate runs on five separate days. The positive sample was a multi-organism mix containing bacterial species at $1 \mathrm { x } 1 0 ^ { 9 } \mathrm { C F U / m L }$ and one fungal species at $\mathrm { 1 x 1 0 ^ { 7 } C F U / m L }$ . The organism levels were higher than typically found in positive blood cultures in order to challenge the system for potential cross contamination. On each round of testing, 24 ePlex BCID-GP Panel cartridges were evaluated. $100 \%$ of the positive sample runs resulted in detection of S. aureus, mecA, $E .$ . faecium, vanA, Pan Gram-Negative, and Pan Candida. No false positives were detected in the negative runs indicating no carryover or cross-contamination was observed between bays or within bays with the ePlex BCID-GP Panel when testing samples consecutively or in adjacent bays with an ePlex instrument.

2. Comparison studies:

a. Method comparison with predicate device: Not applicable. Refer to the Clinical Studies Section of this document.

b. Matrix comparison: Not applicable

3. Clinical studies:

The clinical performance of the ePlex BCID-GP Panel was established during multicenter clinical studies conducted at seven distinct U.S. test sites in two phases. Each test site was representative of the intended use setting (clinical laboratories) and testing was performed by trained clinical laboratory personnel. From June 2014 through July 2016, 400 samples were prospectively collected and frozen; from January through February 2018, 319 samples were prospectively collected and tested fresh (never frozen) for a total of 719 samples across the 2 phases. 8 of these samples were withdrawn; 5 due to the sample coming from a patient already enrolled; 1 was collected outside of the required timeframe; 1 was not viable upon subculture and 1 was from an autopsy. Samples with final, valid ePlex BCID-GP Panel results and valid comparator results were considered evaluable. Of the 711 prospectively-collected samples eligible for testing, all 711 were evaluable. Demographic information for prospectively collected samples is described in Table 48. Subjects enrolled in this study were from a diverse demographic distribution and represent the intended patient population.

To supplement the number of positives for low prevalence targets in the prospective collection, 586 samples were collected retrospectively and all 586 were evaluable. Demographic information for retrospectively collected samples is described in Table 49.

Table 48: Demographic Data for Clinical Samples by Collection Site (Prospective Collection)   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>All SitesN = 711</td><td rowspan=1 colspan=1>Site 1N = 161</td><td rowspan=1 colspan=1>Site 2N =58</td><td rowspan=1 colspan=1>Site 3N = 164</td><td rowspan=1 colspan=1>Site 4N = 145</td><td rowspan=1 colspan=1>Site 5N =77</td><td rowspan=1 colspan=1>Site 6N = 33</td><td rowspan=1 colspan=1>Site 7N = 73</td></tr><tr><td rowspan=1 colspan=9>Sex</td></tr><tr><td rowspan=1 colspan=1>Male</td><td rowspan=1 colspan=1>377(53.0)</td><td rowspan=1 colspan=1>93 (57.8)</td><td rowspan=1 colspan=1>28(48.3)</td><td rowspan=1 colspan=1>91 (55.5)</td><td rowspan=1 colspan=1>66 (45.5)</td><td rowspan=1 colspan=1>42 (54.5)</td><td rowspan=1 colspan=1>17(51.5)</td><td rowspan=1 colspan=1>40(54.8)</td></tr><tr><td rowspan=1 colspan=1>Female</td><td rowspan=1 colspan=1>334(47.0)</td><td rowspan=1 colspan=1>68 (42.2)</td><td rowspan=1 colspan=1>30(51.7)</td><td rowspan=1 colspan=1>73 (44.5)</td><td rowspan=1 colspan=1>79 (54.5)</td><td rowspan=1 colspan=1>35 (45.5)</td><td rowspan=1 colspan=1>16(48.5)</td><td rowspan=1 colspan=1>33(45.2)</td></tr><tr><td rowspan=1 colspan=9>Age</td></tr><tr><td rowspan=1 colspan=1>&lt;1 yr</td><td rowspan=1 colspan=1>27(3.8)</td><td rowspan=1 colspan=1>3(1.9)</td><td rowspan=1 colspan=1>0(0.0)</td><td rowspan=1 colspan=1>8(4.9)</td><td rowspan=1 colspan=1>10(6.9)</td><td rowspan=1 colspan=1>4(5.2)</td><td rowspan=1 colspan=1>2(6.1)</td><td rowspan=1 colspan=1>0(0.0)</td></tr><tr><td rowspan=1 colspan=1>1-17 yrs</td><td rowspan=1 colspan=1>42(5.9)</td><td rowspan=1 colspan=1>8(5.0)</td><td rowspan=1 colspan=1>2(3.4)</td><td rowspan=1 colspan=1>11(6.7)</td><td rowspan=1 colspan=1>10(6.9)</td><td rowspan=1 colspan=1>7(9.1)</td><td rowspan=1 colspan=1>2(6.1)</td><td rowspan=1 colspan=1>2(2.7)</td></tr><tr><td rowspan=1 colspan=1>18-44 yrs</td><td rowspan=1 colspan=1>121(17)</td><td rowspan=1 colspan=1>32(19.9)</td><td rowspan=1 colspan=1>9(15.5)</td><td rowspan=1 colspan=1>24 (14.6)</td><td rowspan=1 colspan=1>24 (16.6)</td><td rowspan=1 colspan=1>13 (16.9)</td><td rowspan=1 colspan=1>4 (12.1)</td><td rowspan=1 colspan=1>15(20.5)</td></tr><tr><td rowspan=1 colspan=1>45-64 yrs</td><td rowspan=1 colspan=1>250(35.2)</td><td rowspan=1 colspan=1>66(41.0)</td><td rowspan=1 colspan=1>18(31.0)</td><td rowspan=1 colspan=1>67 (40.9)</td><td rowspan=1 colspan=1>36 (24.8)</td><td rowspan=1 colspan=1>25 (32.5)</td><td rowspan=1 colspan=1>11(33.3)</td><td rowspan=1 colspan=1>27(37.0)</td></tr><tr><td rowspan=1 colspan=1>65-84 yrs</td><td rowspan=1 colspan=1>217(30.5)</td><td rowspan=1 colspan=1>44(27.3)</td><td rowspan=1 colspan=1>20(34.5)</td><td rowspan=1 colspan=1>41 (25.0)</td><td rowspan=1 colspan=1>51 (35.2)</td><td rowspan=1 colspan=1>23 (29.9)</td><td rowspan=1 colspan=1>13(39.4)</td><td rowspan=1 colspan=1>25(34.2)</td></tr><tr><td rowspan=1 colspan=1>85+ yrs</td><td rowspan=1 colspan=1>54(7.6)</td><td rowspan=1 colspan=1>8(5.0)</td><td rowspan=1 colspan=1>9(15.5)</td><td rowspan=1 colspan=1>13(7.9)</td><td rowspan=1 colspan=1>14(9.7)</td><td rowspan=1 colspan=1>5(6.5)</td><td rowspan=1 colspan=1>1(3.0)</td><td rowspan=1 colspan=1>4(5.5)</td></tr></table>

Table 49: Demographic Data for Clinical Samples by Collection Site (Retrospective Collection)   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>All SitesN = 586</td><td rowspan=1 colspan=1>Site 1N = 80</td><td rowspan=1 colspan=1>Site 2N = 98</td><td rowspan=1 colspan=1>Site 3N = 51</td><td rowspan=1 colspan=1>Site 4N = 43</td><td rowspan=1 colspan=1>Site 5N = 3</td><td rowspan=1 colspan=1>Site 6N = 61</td><td rowspan=1 colspan=1>Site 7N = 85</td><td rowspan=1 colspan=1>Site 8N = 25</td><td rowspan=1 colspan=1>Site 9N = 46</td><td rowspan=1 colspan=1>Site 10N = 94</td></tr><tr><td rowspan=1 colspan=12>Sex</td></tr><tr><td rowspan=1 colspan=1>Male</td><td rowspan=1 colspan=1>317(54.1)</td><td rowspan=1 colspan=1>39(48.8)</td><td rowspan=1 colspan=1>59(60.2)</td><td rowspan=1 colspan=1>24(47.1)</td><td rowspan=1 colspan=1>20(46.5)</td><td rowspan=1 colspan=1>1(33.3)</td><td rowspan=1 colspan=1>36(59.0)</td><td rowspan=1 colspan=1>45(52.9)</td><td rowspan=1 colspan=1>17(68.0)</td><td rowspan=1 colspan=1>28(60.9)</td><td rowspan=1 colspan=1>48(51.1)</td></tr><tr><td rowspan=1 colspan=1>Female</td><td rowspan=1 colspan=1>269(45.9)</td><td rowspan=1 colspan=1>41(51.3)</td><td rowspan=1 colspan=1>39(39.8)</td><td rowspan=1 colspan=1>27(52.9)</td><td rowspan=1 colspan=1>23(53.5)</td><td rowspan=1 colspan=1>2(66.7)</td><td rowspan=1 colspan=1>25(41.0)</td><td rowspan=1 colspan=1>40(47.1)</td><td rowspan=1 colspan=1>8(32.0)</td><td rowspan=1 colspan=1>18(39.1)</td><td rowspan=1 colspan=1>46(48.9)</td></tr><tr><td rowspan=1 colspan=12>Age</td></tr><tr><td rowspan=1 colspan=1>&lt;1 yr</td><td rowspan=1 colspan=1>11(1.9)</td><td rowspan=1 colspan=1>1(1.3)</td><td rowspan=1 colspan=1>2(2)</td><td rowspan=1 colspan=1>0(0)</td><td rowspan=1 colspan=1>3(7)</td><td rowspan=1 colspan=1>0(0)</td><td rowspan=1 colspan=1>1(1.6)</td><td rowspan=1 colspan=1>0(0)</td><td rowspan=1 colspan=1>0(0)</td><td rowspan=1 colspan=1>1(2.2)</td><td rowspan=1 colspan=1>3(3.2)</td></tr><tr><td rowspan=1 colspan=1>1-17 yrs</td><td rowspan=1 colspan=1>17(2.9)</td><td rowspan=1 colspan=1>6(7.5)</td><td rowspan=1 colspan=1>1(1)</td><td rowspan=1 colspan=1>0(0)</td><td rowspan=1 colspan=1>4(9.3)</td><td rowspan=1 colspan=1>0(0)</td><td rowspan=1 colspan=1>0(0)</td><td rowspan=1 colspan=1>0(0)</td><td rowspan=1 colspan=1>1(4)</td><td rowspan=1 colspan=1>1(2.2)</td><td rowspan=1 colspan=1>4(4.3)</td></tr><tr><td rowspan=1 colspan=1>18-44 yrs</td><td rowspan=1 colspan=1>104(17.7)</td><td rowspan=1 colspan=1>14(17.5)</td><td rowspan=1 colspan=1>13(13.3)</td><td rowspan=1 colspan=1>5(9.8)</td><td rowspan=1 colspan=1>9(20.9)</td><td rowspan=1 colspan=1>0(0)</td><td rowspan=1 colspan=1>15(24.6)</td><td rowspan=1 colspan=1>11(12.9)</td><td rowspan=1 colspan=1>7(28)</td><td rowspan=1 colspan=1>5(10.9)</td><td rowspan=1 colspan=1>25(26.6)</td></tr><tr><td rowspan=1 colspan=1>45-64 yrs</td><td rowspan=1 colspan=1>193(32.9)</td><td rowspan=1 colspan=1>25(31.3)</td><td rowspan=1 colspan=1>33(33.7)</td><td rowspan=1 colspan=1>17(33.3)</td><td rowspan=1 colspan=1>15(34.9)</td><td rowspan=1 colspan=1>1(33.3)</td><td rowspan=1 colspan=1>21(34.4)</td><td rowspan=1 colspan=1>30(35.3)</td><td rowspan=1 colspan=1>10(40)</td><td rowspan=1 colspan=1>12(26.1)</td><td rowspan=1 colspan=1>29(30.9)</td></tr><tr><td rowspan=1 colspan=1>65-84 yrs</td><td rowspan=1 colspan=1>209(35.7)</td><td rowspan=1 colspan=1>26(32.5)</td><td rowspan=1 colspan=1>42(42.9)</td><td rowspan=1 colspan=1>22(43.1)</td><td rowspan=1 colspan=1>9(20.9)</td><td rowspan=1 colspan=1>0(0)</td><td rowspan=1 colspan=1>20(32.8)</td><td rowspan=1 colspan=1>35(41.2)</td><td rowspan=1 colspan=1>7(28)</td><td rowspan=1 colspan=1>18(39.1)</td><td rowspan=1 colspan=1>30(31.9)</td></tr><tr><td rowspan=1 colspan=1>85+ yrs</td><td rowspan=1 colspan=1>50(8.5)</td><td rowspan=1 colspan=1>8(10)</td><td rowspan=1 colspan=1>7(7.1)</td><td rowspan=1 colspan=1>7(13.7)</td><td rowspan=1 colspan=1>3(7)</td><td rowspan=1 colspan=1>2(66.7)</td><td rowspan=1 colspan=1>4(6.6)</td><td rowspan=1 colspan=1>7(8.2)</td><td rowspan=1 colspan=1>0(0)</td><td rowspan=1 colspan=1>9(19.6)</td><td rowspan=1 colspan=1>3(3.2)</td></tr><tr><td rowspan=1 colspan=1>Unknown</td><td rowspan=1 colspan=1>2(0.3)</td><td rowspan=1 colspan=1>0(0)</td><td rowspan=1 colspan=1>0(0)</td><td rowspan=1 colspan=1>0(0)</td><td rowspan=1 colspan=1>0(0)</td><td rowspan=1 colspan=1>0(0)</td><td rowspan=1 colspan=1>0(0)</td><td rowspan=1 colspan=1>2(2.4)</td><td rowspan=1 colspan=1>0(0)</td><td rowspan=1 colspan=1>0(0)</td><td rowspan=1 colspan=1>0(0)</td></tr></table>

Clinical performance of the ePlex BCID-GP was compared to standard laboratory procedures, including traditional and automated culture, MALDI-TOF IVD, and microbiological and biochemical techniques for organism identification. Identification for samples with Corynebacterium, Staphylococcus epidermidis, Staphylococcus hominis, or Candida parapsilosis identified by standard laboratory procedures were confirmed using analytically validated PCR assays followed by bi-directional sequencing or 16S sequencing. For antibiotic resistance genes, the ePlex BCID-GP Panel was compared to analytically validated qPCR amplification assays followed by bi-directional sequencing in samples with an associated organism identified (i.e., Staphylococcus, Enterococcus).

The comparator method(s) results were used to determine the Detected/Not Detected status for each target organism on the ePlex BCID-GP Panel. The comparator methods for each target are summarized in Table 50.

Table 50: Comparator Method(s) by ePlex BCID-GP Panel Target   

<table><tr><td rowspan=1 colspan=1>Target</td><td rowspan=1 colspan=1>Comparator Method</td></tr><tr><td rowspan=1 colspan=1>Bacillus cereus group</td><td rowspan=20 colspan=1>Standard laboratory proc edures fororganism identific ation.</td></tr><tr><td rowspan=1 colspan=1>Bacillus subtilis group</td></tr><tr><td rowspan=1 colspan=1>Cutibacterium acnes(Propionibacterium acnes)</td></tr><tr><td rowspan=1 colspan=1>Enterococcus</td></tr><tr><td rowspan=1 colspan=1>Enterococcus faecalis</td></tr><tr><td rowspan=1 colspan=1>Enterococcus faecium</td></tr><tr><td rowspan=1 colspan=1>Lactobacillus</td></tr><tr><td rowspan=1 colspan=1>Listeria</td></tr><tr><td rowspan=1 colspan=1>Listeria monocytogenes</td><td rowspan=1 colspan=1>Sb</td></tr><tr><td rowspan=1 colspan=1>Micrococcus</td><td rowspan=1 colspan=1>org</td></tr><tr><td rowspan=2 colspan=1>Staphylococcus</td></tr><tr></tr><tr><td rowspan=1 colspan=1>Staphylococcus aureus</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus lugdunensis</td></tr><tr><td rowspan=1 colspan=1>Streptococcus</td></tr><tr><td rowspan=1 colspan=1>Streptococcus agalactiae</td></tr><tr><td rowspan=1 colspan=1>Streptococcus anginosus group</td></tr><tr><td rowspan=1 colspan=1>Streptococcus pneumoniae</td></tr><tr><td rowspan=1 colspan=1>Streptococcus pyogenes</td></tr><tr><td rowspan=1 colspan=1>Pan Gram-Negative</td></tr><tr><td rowspan=1 colspan=1>Corynebacterium</td><td rowspan=1 colspan=1>Standard laboratory proc edures fororganism ID. PCR/sequenc ing and 16Ssequenc ing to confirm (or identifyCoryneform) or exclude Corynebacteriumspecies not included in this panel target*</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus epidermidis</td><td rowspan=1 colspan=1>Standard laboratory procedures fororganism ID. PCR/sequenc ing to confirmS. epidermidis, S. hominis</td></tr><tr><td rowspan=1 colspan=1>Pan Candida</td><td rowspan=1 colspan=1>Standard laboratory proc edures fororganism ID. PCR/sequenc ing to confirmC. parapsilosis or identify C. metapsilosis,C. orthopsilosis</td></tr><tr><td rowspan=1 colspan=1>mecA</td><td rowspan=2 colspan=1>qPCR/sequenc ing in samples withStaphylococcus identified</td></tr><tr><td rowspan=1 colspan=1>mecC</td></tr></table>

<table><tr><td>Target</td><td>Comparator Method</td></tr><tr><td>vanA</td><td>qPCR/sequenc ing in samples w ith</td></tr><tr><td>vanB</td><td>Enterococcus identified</td></tr></table>

\*T he Corynebacterium assay is not designed to detect the following Corynebacterium species: $C$ . amycolatum, C. argentoratense, C. bovis, C. durum, C. glucuronolyticum, C. macginleyi, C. propinquum, C. riegelii, and C. sundsvallens

Sensitivity or positive percent agreement (PPA) was calculated by dividing the number of true positive (TP) results by the sum of TP and false negative (FN) results, while specificity or negative percent agreement (NPA) was calculated by dividing the number of true negative (TN) results by the sum of TN and false positive (FP) results. A TP result being defined as a sample where the detected ePlex BCID-GP Panel result matched the detected comparator method result, while a TN result was one where a negative ePlex BCID-GP Panel result matched a negative comparator method result. The two-sided $9 5 \%$ confidence interval was also calculated.

A total of 711 prospectively-collected samples (312 tested fresh and 399 tested after previously frozen) and 586 retrospectively collected samples as well as 565 contrived samples were evaluated for the ePlex BCID-GP Panel targets. Contrived samples were prepared by spiking an isolate into a blood culture bottle and growing until flagged positive by a continuously monitoring blood culture system. Samples were removed from the system within 8 hours of positivity and stored frozen until the time of testing. PPA and NPA results are summarized by target in Tables 51-74 and the strains used to contrive samples are summarized in Table 75.

Table 51: Clinical Performance for Bacillus cereus group   

<table><tr><td rowspan=2 colspan=1>Target</td><td rowspan=2 colspan=1>Sample Type</td><td rowspan=1 colspan=2>Sensitivity/PPA</td><td rowspan=1 colspan=2>Specificity/NP</td></tr><tr><td rowspan=1 colspan=1>TP/TP+FN</td><td rowspan=1 colspan=1>% (95%CI)</td><td rowspan=1 colspan=1>TN/TN+FP</td><td rowspan=1 colspan=1>% (95%CI</td></tr><tr><td rowspan=7 colspan=1>Bacillus cereusgroup</td><td rowspan=1 colspan=1>Prospective (Fresh)</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>100(34.2-100)</td><td rowspan=1 colspan=1>310/310</td><td rowspan=1 colspan=1>100(98.8-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective (Frozen)</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>100(43.9-100)</td><td rowspan=1 colspan=1>396/396</td><td rowspan=1 colspan=1>100(99.0-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective (All)</td><td rowspan=1 colspan=1>5/5</td><td rowspan=1 colspan=1>100(56.6-100)</td><td rowspan=1 colspan=1>706/706</td><td rowspan=1 colspan=1>100(99.5-100)</td></tr><tr><td rowspan=1 colspan=1>Retrospective</td><td rowspan=1 colspan=1>6/7</td><td rowspan=1 colspan=1>85.7(48.7-97.4)</td><td rowspan=1 colspan=1>579/579</td><td rowspan=1 colspan=1>100(99.3-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective/Retrospective</td><td rowspan=1 colspan=1>11/12</td><td rowspan=1 colspan=1>91.7(64.6-98.5)</td><td rowspan=1 colspan=1>1765/1765</td><td rowspan=1 colspan=1>100(99.8-100)</td></tr><tr><td rowspan=1 colspan=1>Contrived</td><td rowspan=1 colspan=1>46/46</td><td rowspan=1 colspan=1>100(92.3-100)</td><td rowspan=1 colspan=1>519/519</td><td rowspan=1 colspan=1>100(99.3-100)</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>57/58</td><td rowspan=1 colspan=1>98.3(90.9-99.7)</td><td rowspan=1 colspan=1>2284/2284</td><td rowspan=1 colspan=1>100(99.8-100)</td></tr></table>

Table 52: Clinical Performance for Bacillus subtilis group   

<table><tr><td rowspan=2 colspan=1>Target</td><td rowspan=2 colspan=1>Sample Type</td><td rowspan=1 colspan=2>Sensitivity/PPA</td><td rowspan=1 colspan=2>Specificity/NP</td></tr><tr><td rowspan=1 colspan=1>TP/TP+FN</td><td rowspan=1 colspan=1>% (95%CI)</td><td rowspan=1 colspan=1>TN/TN+FP</td><td rowspan=1 colspan=1>% (95%CII</td></tr><tr><td rowspan=7 colspan=1>Bacillus subtilisgroup</td><td rowspan=1 colspan=1>Prospectivee (Fresh)</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>100(34.2-100)</td><td rowspan=1 colspan=1>309/309</td><td rowspan=1 colspan=1>100(98.8-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective (Frozen)</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>---</td><td rowspan=1 colspan=1>399/399</td><td rowspan=1 colspan=1>100(99.0-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective (All)</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>100(34.2-100)</td><td rowspan=1 colspan=1>708/708</td><td rowspan=1 colspan=1>100(99.5-100)</td></tr><tr><td rowspan=1 colspan=1>Retrospective</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>---</td><td rowspan=1 colspan=1>586/586</td><td rowspan=1 colspan=1>100(99.3-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective/Retrospective</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>100(34.2-100)</td><td rowspan=1 colspan=1>1294/1294</td><td rowspan=1 colspan=1>100(99.7-100)</td></tr><tr><td rowspan=1 colspan=1>Contrived</td><td rowspan=1 colspan=1>50/50</td><td rowspan=1 colspan=1>100(92.9-100)</td><td rowspan=1 colspan=1>515/515</td><td rowspan=1 colspan=1>100(99.3-100)</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>52/52</td><td rowspan=1 colspan=1>100(93.1-100)</td><td rowspan=1 colspan=1>1809/1809</td><td rowspan=1 colspan=1>100(99.8-100)</td></tr></table>

Table 53: Clinical Performance for Corynebacterium   

<table><tr><td rowspan=2 colspan=1>Target</td><td rowspan=2 colspan=1>Sample Type</td><td rowspan=1 colspan=2>Sensitivity/PPA</td><td rowspan=1 colspan=2>Specificity/NP</td></tr><tr><td rowspan=1 colspan=1>TP/TP+FN</td><td rowspan=1 colspan=1>% (95%CI)</td><td rowspan=1 colspan=1>TN/TN+FP</td><td rowspan=1 colspan=1>% (95%CI)</td></tr><tr><td rowspan=7 colspan=1>Corynebacterium</td><td rowspan=1 colspan=1>Prospective (Fresh)</td><td rowspan=1 colspan=1>5/7</td><td rowspan=1 colspan=1>71.4(35.9-91.8)</td><td rowspan=1 colspan=1>304/305</td><td rowspan=1 colspan=1>99.7(98.2-99.9)</td></tr><tr><td rowspan=1 colspan=1>Prospective (Frozen)</td><td rowspan=1 colspan=1>8/12</td><td rowspan=1 colspan=1>66.7(39.1-86.2)</td><td rowspan=1 colspan=1>387/387</td><td rowspan=1 colspan=1>100(99.0-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective (All)</td><td rowspan=1 colspan=1>13/19</td><td rowspan=1 colspan=1>68.4(46.0-84.6)</td><td rowspan=1 colspan=1>691/692</td><td rowspan=1 colspan=1>99.9(99.2-100)</td></tr><tr><td rowspan=1 colspan=1>Retrospective</td><td rowspan=1 colspan=1>27/32</td><td rowspan=1 colspan=1>84.4(68.2-93.1)</td><td rowspan=1 colspan=1>553/554</td><td rowspan=1 colspan=1>99.8(99.0-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective/Retrospective</td><td rowspan=1 colspan=1>40/51</td><td rowspan=1 colspan=1>78.4(65.4-87.5)</td><td rowspan=1 colspan=1>1244/1246</td><td rowspan=1 colspan=1>99.8(99.4-100)</td></tr><tr><td rowspan=1 colspan=1>Contrived</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>100(83.9-100)</td><td rowspan=1 colspan=1>545/545</td><td rowspan=1 colspan=1>100(99.3-100)</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>60/71A</td><td rowspan=1 colspan=1>84.5(74.3-91.1)</td><td rowspan=1 colspan=1>1789/1791B</td><td rowspan=1 colspan=1>99.9(99.6-100)</td></tr></table>

A. Corynebacterium was not detected in 4 of the false negative samples using PCR/sequencing, but 16S sequencing instead detected Staphylococcus pettenkoferi, Macrococcus caseolyticus, Lactobacillus fermentum, and Cutibacterium acnes, which were not identified by standard laboratory procedures. B. Corynebacterium was detected in 2/2 false positive samples using PCR/sequencing.

Table 54: Clinical Performance for Cutibacterium acnes   

<table><tr><td rowspan=2 colspan=1>Target</td><td rowspan=2 colspan=1>Sample Type</td><td rowspan=1 colspan=2>Sensitivity/PPA</td><td rowspan=1 colspan=2>Specificity/NP</td></tr><tr><td rowspan=1 colspan=1>TP/TP+FN</td><td rowspan=1 colspan=1>% (95%CI)</td><td rowspan=1 colspan=1>TN/TN+FP</td><td rowspan=1 colspan=1>% (95%CI</td></tr><tr><td rowspan=7 colspan=1>Cutibacteriumacnes</td><td rowspan=1 colspan=1>Prospective (Fresh)</td><td rowspan=1 colspan=1>4/5</td><td rowspan=1 colspan=1>80.0(37.6-96.4)</td><td rowspan=1 colspan=1>306/307</td><td rowspan=1 colspan=1>99.7(98.2-99.9)</td></tr><tr><td rowspan=1 colspan=1>Prospective (Frozen)</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>100(34.2-100)</td><td rowspan=1 colspan=1>396/397</td><td rowspan=1 colspan=1>99.7(98.6-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective (All)</td><td rowspan=1 colspan=1>6/7</td><td rowspan=1 colspan=1>85.7(48.7-97.4)</td><td rowspan=1 colspan=1>702/704</td><td rowspan=1 colspan=1>99.7(99.0-99.9)</td></tr><tr><td rowspan=1 colspan=1>Retrospective</td><td rowspan=1 colspan=1>12/13</td><td rowspan=1 colspan=1>92.3(66.7-98.6)</td><td rowspan=1 colspan=1>573/573</td><td rowspan=1 colspan=1>100(99.3-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective/Retrospective</td><td rowspan=1 colspan=1>18/20</td><td rowspan=1 colspan=1>90.0(69.9-97.2)</td><td rowspan=1 colspan=1>1275/1277</td><td rowspan=1 colspan=1>99.8(99.4-100)</td></tr><tr><td rowspan=1 colspan=1>Contrived</td><td rowspan=1 colspan=1>25/26</td><td rowspan=1 colspan=1>96.2(81.1-99.3)</td><td rowspan=1 colspan=1>539/539</td><td rowspan=1 colspan=1>100(99.3-100)</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>43/46</td><td rowspan=1 colspan=1>93.5(82.5-97.8)</td><td rowspan=1 colspan=1>1814/18164</td><td rowspan=1 colspan=1>99.9(99.6-100)</td></tr></table>

A. Cutibacterium acnes was detected in 1/2 false positive samples using PCR/sequencing.

Table 55: Clinical Performance for Enterococcus   

<table><tr><td rowspan=2 colspan=1>Target</td><td rowspan=2 colspan=1>Sample Type</td><td rowspan=1 colspan=2>Sensitivity/PPA</td><td rowspan=1 colspan=2>Specificity/NP</td></tr><tr><td rowspan=1 colspan=1>TP/TP+FN</td><td rowspan=1 colspan=1>% (95%CI)</td><td rowspan=1 colspan=1>TN/TN+FP</td><td rowspan=1 colspan=1>% (95%CI)</td></tr><tr><td rowspan=7 colspan=1>Enterococcus</td><td rowspan=1 colspan=1>Prospective(Fresh)</td><td rowspan=1 colspan=1>25/25</td><td rowspan=1 colspan=1>100(86.7-100)</td><td rowspan=1 colspan=1>287/287</td><td rowspan=1 colspan=1>100(98.7-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective (Frozen)</td><td rowspan=1 colspan=1>36/36</td><td rowspan=1 colspan=1>100(90.4-100)</td><td rowspan=1 colspan=1>362/363</td><td rowspan=1 colspan=1>99.7(98.5-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective (All)</td><td rowspan=1 colspan=1>61/61</td><td rowspan=1 colspan=1>100(94.1-100)</td><td rowspan=1 colspan=1>649/650</td><td rowspan=1 colspan=1>99.8(99.1-100)</td></tr><tr><td rowspan=1 colspan=1>Retrospective</td><td rowspan=1 colspan=1>139/147</td><td rowspan=1 colspan=1>94.6(89.6-97.2)</td><td rowspan=1 colspan=1>439/439</td><td rowspan=1 colspan=1>100(99.1-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective/Retrospective</td><td rowspan=1 colspan=1>200/208</td><td rowspan=1 colspan=1>96.2(92.6-98.0)</td><td rowspan=1 colspan=1>1088/1089</td><td rowspan=1 colspan=1>99.9(99.5-100)</td></tr><tr><td rowspan=1 colspan=1>Contrived</td><td rowspan=1 colspan=1>126/126</td><td rowspan=1 colspan=1>100(97.0-100)</td><td rowspan=1 colspan=1>439/439</td><td rowspan=1 colspan=1>100(99.1-100)</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>326/334A</td><td rowspan=1 colspan=1>97.6(95.3-98.8)</td><td rowspan=1 colspan=1>1527/1528B</td><td rowspan=1 colspan=1>99.9(99.6-100)</td></tr></table>

A. Enterococcus was not detected in 1 false negative sample, but PCR/sequencing instead detected Lactococcus lactis, which was not identified by standard laboratory procedures. B. Enterococcus was detected in 1/1 false positive samples using PCR/sequencing.

Table 56: Clinical Performance for Enterococcus faecalis   

<table><tr><td rowspan=2 colspan=1>Target</td><td rowspan=2 colspan=1>Sample Type</td><td rowspan=1 colspan=2>Sensitivity/PPA</td><td rowspan=1 colspan=2>Specificity/NP</td></tr><tr><td rowspan=1 colspan=1>TP/TP+FN</td><td rowspan=1 colspan=1>% (95%CI)</td><td rowspan=1 colspan=1>TN/TN+FP</td><td rowspan=1 colspan=1>% (95%CI</td></tr><tr><td rowspan=7 colspan=1>Enterococcusfaecalis</td><td rowspan=1 colspan=1>Prospective(Fresh)</td><td rowspan=1 colspan=1>21/21</td><td rowspan=1 colspan=1>100(84.5-100)</td><td rowspan=1 colspan=1>291/291</td><td rowspan=1 colspan=1>100(98.7-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective (Frozen)</td><td rowspan=1 colspan=1>28/28</td><td rowspan=1 colspan=1>100(87.9-100)</td><td rowspan=1 colspan=1>370/370</td><td rowspan=1 colspan=1>100(99.0-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective (All)</td><td rowspan=1 colspan=1>49/49</td><td rowspan=1 colspan=1>100(92.7-100)</td><td rowspan=1 colspan=1>661/661</td><td rowspan=1 colspan=1>100(99.4-100)</td></tr><tr><td rowspan=1 colspan=1>Retrospective</td><td rowspan=1 colspan=1>82/90</td><td rowspan=1 colspan=1>91.1(83.4-95.4)</td><td rowspan=1 colspan=1>496/496</td><td rowspan=1 colspan=1>100(99.2-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective/Retrospective</td><td rowspan=1 colspan=1>131/139</td><td rowspan=1 colspan=1>94.2(89.1-97.1)</td><td rowspan=1 colspan=1>1157/1157</td><td rowspan=1 colspan=1>100(99.7-100)</td></tr><tr><td rowspan=1 colspan=1>Contrived</td><td rowspan=1 colspan=1>52/52</td><td rowspan=1 colspan=1>100(93.1-100)</td><td rowspan=1 colspan=1>513/513</td><td rowspan=1 colspan=1>100(99.3-100)</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>183/191A</td><td rowspan=1 colspan=1>95.8(92.0-97.9)</td><td rowspan=1 colspan=1>1670/1670</td><td rowspan=1 colspan=1>100(99.8-100)</td></tr></table>

A. Enterococcus faecalis was not detected in 4 false negative samples, but PCR/sequencing instead detected Enterococcus faecium (3) and Lactococcus lactis (1), which were not identified by standard laboratory procedures.

Table 57: Clinical Performance for Enterococcus faecium   

<table><tr><td rowspan=2 colspan=1>Target</td><td rowspan=2 colspan=1>Sample Type</td><td rowspan=1 colspan=2>Sensitivity/PPA</td><td rowspan=1 colspan=2>Specificity/NP</td></tr><tr><td rowspan=1 colspan=1>TP/TP+FN</td><td rowspan=1 colspan=1>% (95%CI)</td><td rowspan=1 colspan=1>TN/TN+FP</td><td rowspan=1 colspan=1>% (95%CI)</td></tr><tr><td rowspan=7 colspan=1>Enterococcusfaecium</td><td rowspan=1 colspan=1>Prospective (Fresh)</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>100(43.9-100)</td><td rowspan=1 colspan=1>309/309</td><td rowspan=1 colspan=1>100(98.8-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective (Frozen)</td><td rowspan=1 colspan=1>8/9</td><td rowspan=1 colspan=1>88.9(56.5-98.0)</td><td rowspan=1 colspan=1>388/389</td><td rowspan=1 colspan=1>99.7(98.6-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective (All)</td><td rowspan=1 colspan=1>11/12</td><td rowspan=1 colspan=1>91.7(64.6-98.5)</td><td rowspan=1 colspan=1>697/698</td><td rowspan=1 colspan=1>99.9(99.2-100)</td></tr><tr><td rowspan=1 colspan=1>Retrospective</td><td rowspan=1 colspan=1>52/53</td><td rowspan=1 colspan=1>98.1(90.1-99.7)</td><td rowspan=1 colspan=1>526/533</td><td rowspan=1 colspan=1>98.7 (97.3-99.4</td></tr><tr><td rowspan=1 colspan=1>Prospective/Retrospective</td><td rowspan=1 colspan=1>63/65</td><td rowspan=1 colspan=1>96.9(89.5-99.2)</td><td rowspan=1 colspan=1>1223/1231</td><td rowspan=1 colspan=1>99.4(98.7-99.7)</td></tr><tr><td rowspan=1 colspan=1>Contrived</td><td rowspan=1 colspan=1>60/60</td><td rowspan=1 colspan=1>100(94.0-100)</td><td rowspan=1 colspan=1>505/505</td><td rowspan=1 colspan=1>100(99.2-100)</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>123/125</td><td rowspan=1 colspan=1>98.4(94.4-99.6)</td><td rowspan=1 colspan=1>1728/1736A</td><td rowspan=1 colspan=1>99.5(99.1-99.8)</td></tr></table>

A. Enterococcus faecium was detected in 5/8 false positive samples using PCR/sequencing.

Table 58: Clinical Performance for Lactobacillus   

<table><tr><td rowspan=2 colspan=1>Target</td><td rowspan=2 colspan=1>Sample Type</td><td rowspan=1 colspan=2>Sensitivity/PPA</td><td rowspan=1 colspan=2>Specificity/NP</td></tr><tr><td rowspan=1 colspan=1>TP/TP+FN</td><td rowspan=1 colspan=1>% (95%CI</td><td rowspan=1 colspan=1>TN/TN+FP</td><td rowspan=1 colspan=1>% (95%CI</td></tr><tr><td rowspan=7 colspan=1>Lactobacillus</td><td rowspan=1 colspan=1>Prospective (Fresh)</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>100(34.2-100)</td><td rowspan=1 colspan=1>309/310</td><td rowspan=1 colspan=1>99.7(98.2-99.9)</td></tr><tr><td rowspan=1 colspan=1>Prospective (Frozen)</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>100(34.2-100)</td><td rowspan=1 colspan=1>397/397</td><td rowspan=1 colspan=1>100(99.0-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective (All)</td><td rowspan=1 colspan=1>4/4</td><td rowspan=1 colspan=1>100(51.0-100)</td><td rowspan=1 colspan=1>706/707</td><td rowspan=1 colspan=1>99.9(99.2-100)</td></tr><tr><td rowspan=1 colspan=1>Retrospective</td><td rowspan=1 colspan=1>9/9</td><td rowspan=1 colspan=1>100(70.1-100)</td><td rowspan=1 colspan=1>576/577</td><td rowspan=1 colspan=1>99.8(99.0-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective/Retrospective</td><td rowspan=1 colspan=1>13/13</td><td rowspan=1 colspan=1>100(77.2-100)</td><td rowspan=1 colspan=1>1282/1284</td><td rowspan=1 colspan=1>99.8(99.4-100)</td></tr><tr><td rowspan=1 colspan=1>Contrived</td><td rowspan=1 colspan=1>32/33</td><td rowspan=1 colspan=1>97.0(84.7-99.5)</td><td rowspan=1 colspan=1>532/532</td><td rowspan=1 colspan=1>100(99.3-100)</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>45/46</td><td rowspan=1 colspan=1>97.8(88.7-99.6)</td><td rowspan=1 colspan=1>1814/1816A</td><td rowspan=1 colspan=1>99.9(99.6-100)</td></tr></table>

A. Lactobacillus casei was detected in 1/2 false positive samples using PCR/sequencing.

Table 59: Clinical Performance for Listeria   

<table><tr><td rowspan=2 colspan=1>Target</td><td rowspan=2 colspan=1>Sample Type</td><td rowspan=1 colspan=2>Sensitivity/PPA</td><td rowspan=1 colspan=2>Specificity/NP</td></tr><tr><td rowspan=1 colspan=1>TP/TP+FN</td><td rowspan=1 colspan=1>% (95%CI)</td><td rowspan=1 colspan=1>TN/TN+FP</td><td rowspan=1 colspan=1>% (95%CI</td></tr><tr><td rowspan=7 colspan=1>Listeria</td><td rowspan=1 colspan=1>Prospective (Fresh)</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>---</td><td rowspan=1 colspan=1>312/312</td><td rowspan=1 colspan=1>100(98.8-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective (Frozen)</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>---</td><td rowspan=1 colspan=1>398/399</td><td rowspan=1 colspan=1>99.7(98.6-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective (All)</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>---</td><td rowspan=1 colspan=1>710/711</td><td rowspan=1 colspan=1>99.9(99.2-100)</td></tr><tr><td rowspan=1 colspan=1>Retrospective</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>100(34.2-100)</td><td rowspan=1 colspan=1>584/584</td><td rowspan=1 colspan=1>100(99.3-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective/Retrospective</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>100(34.2-100)</td><td rowspan=1 colspan=1>1294/1295</td><td rowspan=1 colspan=1>99.9(99.6-100)</td></tr><tr><td rowspan=1 colspan=1>Contrived</td><td rowspan=1 colspan=1>74/75</td><td rowspan=1 colspan=1>98.7(92.8-99.8)</td><td rowspan=1 colspan=1>490/490</td><td rowspan=1 colspan=1>100(99.2-100)</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>76/77</td><td rowspan=1 colspan=1>98.7(93.0-99.8)</td><td rowspan=1 colspan=1>1784/1785</td><td rowspan=1 colspan=1>99.9(99.7-100)</td></tr></table>

Table 60: Clinical Performance for Listeria monocytogenes   

<table><tr><td rowspan=2 colspan=1>Target</td><td rowspan=2 colspan=1>Sample Type</td><td rowspan=1 colspan=2>Sensitivity/PPA</td><td rowspan=1 colspan=2>Specificity/NP</td></tr><tr><td rowspan=1 colspan=1>TP/TP+FN</td><td rowspan=1 colspan=1>% (95%CI</td><td rowspan=1 colspan=1>TN/TN+FP</td><td rowspan=1 colspan=1>% (95%CI</td></tr><tr><td rowspan=7 colspan=1>Listeriamonocytogenes</td><td rowspan=1 colspan=1>Prospective (Fresh)</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>---</td><td rowspan=1 colspan=1>312/312</td><td rowspan=1 colspan=1>100(98.8-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective (Frozen)</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>---</td><td rowspan=1 colspan=1>399/399</td><td rowspan=1 colspan=1>100(99.0-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective (All)</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>---</td><td rowspan=1 colspan=1>711/711</td><td rowspan=1 colspan=1>100(99.5-100)</td></tr><tr><td rowspan=1 colspan=1>Retrospective</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>100(34.2-100)</td><td rowspan=1 colspan=1>584/584</td><td rowspan=1 colspan=1>100(99.3-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective/Retrospective</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>100(34.2-100)</td><td rowspan=1 colspan=1>1295/1295</td><td rowspan=1 colspan=1>100(99.7-100)</td></tr><tr><td rowspan=1 colspan=1>Contrived</td><td rowspan=1 colspan=1>46/46</td><td rowspan=1 colspan=1>100(92.3-100)</td><td rowspan=1 colspan=1>519/519</td><td rowspan=1 colspan=1>100(99.3-100)</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>48/48</td><td rowspan=1 colspan=1>100(92.6-100)</td><td rowspan=1 colspan=1>1814/1814</td><td rowspan=1 colspan=1>100(99.8-100)</td></tr></table>

Table 61: Clinical Performance for Micrococcus   

<table><tr><td rowspan=2 colspan=1>Target</td><td rowspan=2 colspan=1>Sample Type</td><td rowspan=1 colspan=2>Sensitivity/PPA</td><td rowspan=1 colspan=2>Specificity/NP</td></tr><tr><td rowspan=1 colspan=1>TP/TP+FN</td><td rowspan=1 colspan=1>% (95%CI)</td><td rowspan=1 colspan=1>TN/TN+FP</td><td rowspan=1 colspan=1>% (95%CI)</td></tr><tr><td rowspan=7 colspan=1>Micrococcus</td><td rowspan=1 colspan=1>Prospective(Fresh)</td><td rowspan=1 colspan=1>9/10</td><td rowspan=1 colspan=1>90.0 (59.6-98.2)</td><td rowspan=1 colspan=1>302/302</td><td rowspan=1 colspan=1>100 (98.7-100</td></tr><tr><td rowspan=1 colspan=1>Prospective (Frozen)</td><td rowspan=1 colspan=1>10/11</td><td rowspan=1 colspan=1>90.9 (62.3-98.4)</td><td rowspan=1 colspan=1>388/388</td><td rowspan=1 colspan=1>100 (99.0-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective (All)</td><td rowspan=1 colspan=1>19/21</td><td rowspan=1 colspan=1>90.5(71.1-97.3)</td><td rowspan=1 colspan=1>690/690</td><td rowspan=1 colspan=1>100(99.4-100)</td></tr><tr><td rowspan=1 colspan=1>Retrospective</td><td rowspan=1 colspan=1>20/23</td><td rowspan=1 colspan=1>87.0(67.9-95.5)</td><td rowspan=1 colspan=1>562/563</td><td rowspan=1 colspan=1>99.8(99.0-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective/Retrospective</td><td rowspan=1 colspan=1>39/44</td><td rowspan=1 colspan=1>88.6(76.0-95.0)</td><td rowspan=1 colspan=1>1252/1253</td><td rowspan=1 colspan=1>99.9(99.5-100)</td></tr><tr><td rowspan=1 colspan=1>Contrived</td><td rowspan=1 colspan=1>27/27</td><td rowspan=1 colspan=1>100(87.5-100)</td><td rowspan=1 colspan=1>538/538</td><td rowspan=1 colspan=1>100(99.3-100)</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>66/71A</td><td rowspan=1 colspan=1>93.0(84.6-97.0)</td><td rowspan=1 colspan=1>1790/1791</td><td rowspan=1 colspan=1>99.9(99.7-100)</td></tr></table>

A. Micrococcus was not detected in 3 false negative samples, but PCR/sequencing instead detected Brevibacterium ravenspurgense, Nesterenkonia halotolerans, and Staphylococcus pettenkoferi, which were not identified by standard laboratory procedures.

Table 62: Clinical Performance for Staphylococcus   

<table><tr><td rowspan=2 colspan=1>Target</td><td rowspan=2 colspan=1>Sample Type</td><td rowspan=1 colspan=2>Sensitivity/PPA</td><td rowspan=1 colspan=2>Specificity/NP</td></tr><tr><td rowspan=1 colspan=1>TP/TP+FN</td><td rowspan=1 colspan=1>% (95%CI</td><td rowspan=1 colspan=1>TN/TN+FP</td><td rowspan=1 colspan=1>% (95%CI</td></tr><tr><td rowspan=7 colspan=1>Staphylococcus</td><td rowspan=1 colspan=1>Prospective (Fresh)</td><td rowspan=1 colspan=1>178/182</td><td rowspan=1 colspan=1>97.8(94.5-99.1)</td><td rowspan=1 colspan=1>127/130</td><td rowspan=1 colspan=1>97.7(93.4-99.2)</td></tr><tr><td rowspan=1 colspan=1>Prospective (Frozen)</td><td rowspan=1 colspan=1>269/274</td><td rowspan=1 colspan=1>98.2(95.8-99.2)</td><td rowspan=1 colspan=1>123/125</td><td rowspan=1 colspan=1>98.4(94.4-99.6)</td></tr><tr><td rowspan=1 colspan=1>Prospective (All)</td><td rowspan=1 colspan=1>447/456</td><td rowspan=1 colspan=1>98.0(96.3-99.0)</td><td rowspan=1 colspan=1>250/255</td><td rowspan=1 colspan=1>98.0(95.5-99.2)</td></tr><tr><td rowspan=1 colspan=1>Retrospective</td><td rowspan=1 colspan=1>185/191</td><td rowspan=1 colspan=1>96.9(93.3-98.6)</td><td rowspan=1 colspan=1>390/395</td><td rowspan=1 colspan=1>98.7(97.1-99.5)</td></tr><tr><td rowspan=1 colspan=1>Prospective/Retrospective</td><td rowspan=1 colspan=1>632/647</td><td rowspan=1 colspan=1>97.7(96.2-98.6)</td><td rowspan=1 colspan=1>640/650</td><td rowspan=1 colspan=1>98.5(97.2-99.2)</td></tr><tr><td rowspan=1 colspan=1>Contrived</td><td rowspan=1 colspan=1>105/105</td><td rowspan=1 colspan=1>100(96.5-100)</td><td rowspan=1 colspan=1>460/460</td><td rowspan=1 colspan=1>100(99.2-100)</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>737/752A</td><td rowspan=1 colspan=1>98.0(96.7-98.8)</td><td rowspan=1 colspan=1>1100/1110B</td><td rowspan=1 colspan=1>99.1(98.3-99.5)</td></tr></table>

A. Staphylococcus was not detected in 3 false negative samples, but PCR/sequencing instead detected Escherichia coli, Klebsiella pneumoniae, and Streptococcus salivarius, which were not identified by standard laboratory procedures. B. Staphylococcus was detected in 9/10 false positive samples using PCR/sequencing.

Table 63: Clinical Performance for Staphylococcus aureus   

<table><tr><td rowspan=2 colspan=1>Target</td><td rowspan=2 colspan=1>Sample Type</td><td rowspan=1 colspan=2>Sensitivity/PPA</td><td rowspan=1 colspan=2>Specificity/NP</td></tr><tr><td rowspan=1 colspan=1>TP/TP+FN</td><td rowspan=1 colspan=1>% (95%CI)</td><td rowspan=1 colspan=1>TN/TN+FP</td><td rowspan=1 colspan=1>% (95%CI)</td></tr><tr><td rowspan=7 colspan=1>Staphylococcusaureus</td><td rowspan=1 colspan=1>Prospective (Fresh)</td><td rowspan=1 colspan=1>62/65</td><td rowspan=1 colspan=1>95.4(87.3-98.4)</td><td rowspan=1 colspan=1>244/245</td><td rowspan=1 colspan=1>99.6(97.7-99.9)</td></tr><tr><td rowspan=1 colspan=1>Prospective (Frozen)</td><td rowspan=1 colspan=1>98/101</td><td rowspan=1 colspan=1>97.0(91.6-99.0)</td><td rowspan=1 colspan=1>222/223</td><td rowspan=1 colspan=1>99.6(97.5-99.9)</td></tr><tr><td rowspan=1 colspan=1>Prospective (All)</td><td rowspan=1 colspan=1>160/166</td><td rowspan=1 colspan=1>96.4(92.3-98.3)</td><td rowspan=1 colspan=1>466/468</td><td rowspan=1 colspan=1>99.6(98.5-99.9)</td></tr><tr><td rowspan=1 colspan=1>Retrospective</td><td rowspan=1 colspan=1>122/125</td><td rowspan=1 colspan=1>97.6(93.2-99.2)</td><td rowspan=1 colspan=1>454/458</td><td rowspan=1 colspan=1>99.1(97.8-99.7)</td></tr><tr><td rowspan=1 colspan=1>Prospective/Retrospective</td><td rowspan=1 colspan=1>282/291</td><td rowspan=1 colspan=1>96.9(94.2-98.4)</td><td rowspan=1 colspan=1>920/926</td><td rowspan=1 colspan=1>99.4(98.6-99.7)</td></tr><tr><td rowspan=1 colspan=1>Contrived</td><td rowspan=1 colspan=1>59/59</td><td rowspan=1 colspan=1>100(93.9-100)</td><td rowspan=1 colspan=1>506/506</td><td rowspan=1 colspan=1>100(99.2-100)</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>341/3504</td><td rowspan=1 colspan=1>97.4(95.2-98.6)</td><td rowspan=1 colspan=1>1426/1432B</td><td rowspan=1 colspan=1>99.6(99.1-99.8)</td></tr></table>

A. Staphylococcus aureus was not detected in 3 false negative samples, but PCR/sequencing instead detected Klebsiella pneumoniae, Staphylococcus simulans, and Streptococcus agalactiae, which were not identified by standard laboratory procedures. B. Staphylococcus aureus was detected in $5 / 6$ false positive samples using PCR/sequencing

Table 64: Clinical Performance for Staphylococcus epidermidis   

<table><tr><td rowspan=2 colspan=1>Target</td><td rowspan=2 colspan=1>Sample Type</td><td rowspan=1 colspan=2>Sensitivity/PPA</td><td rowspan=1 colspan=2>Specificity/NP</td></tr><tr><td rowspan=1 colspan=1>TP/TP+FN</td><td rowspan=1 colspan=1>% (95%CI</td><td rowspan=1 colspan=1>TN/TN+FP</td><td rowspan=1 colspan=1>% (95%CI</td></tr><tr><td rowspan=7 colspan=1>Staphylococcusepidermidis</td><td rowspan=1 colspan=1>Prospective (Fresh)</td><td rowspan=1 colspan=1>59/63</td><td rowspan=1 colspan=1>93.7(84.8-97.5)</td><td rowspan=1 colspan=1>223/229</td><td rowspan=1 colspan=1>97.4(94.4-98.8)</td></tr><tr><td rowspan=1 colspan=1>Prospective (Frozen)</td><td rowspan=1 colspan=1>56/58</td><td rowspan=1 colspan=1>96.6(88.3-99.0)</td><td rowspan=1 colspan=1>258/265</td><td rowspan=1 colspan=1>97.4(94.6-98.7)</td></tr><tr><td rowspan=1 colspan=1>Prospective (All)</td><td rowspan=1 colspan=1>115/121</td><td rowspan=1 colspan=1>95.0(89.6-97.7)</td><td rowspan=1 colspan=1>481/494</td><td rowspan=1 colspan=1>97.4(95.6-98.5)</td></tr><tr><td rowspan=1 colspan=1>Retrospective</td><td rowspan=1 colspan=1>33/38</td><td rowspan=1 colspan=1>86.8(72.7-94.2)</td><td rowspan=1 colspan=1>539/545</td><td rowspan=1 colspan=1>98.9(97.6-99.5)</td></tr><tr><td rowspan=1 colspan=1>Prospective/Retrospective</td><td rowspan=1 colspan=1>148/159</td><td rowspan=1 colspan=1>93.1(88.0-96.1)</td><td rowspan=1 colspan=1>1020/1039</td><td rowspan=1 colspan=1>98.2(97.2-98.8)</td></tr><tr><td rowspan=1 colspan=1>Contrived</td><td rowspan=1 colspan=1>1/1</td><td rowspan=1 colspan=1>100(20.7-100)</td><td rowspan=1 colspan=1>564/564</td><td rowspan=1 colspan=1>100(99.3-100)</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>149/160A</td><td rowspan=1 colspan=1>93.1(88.1-96.1)</td><td rowspan=1 colspan=1>1584/1603B</td><td rowspan=1 colspan=1>98.8(98.2-99.2)</td></tr></table>

A. Staphylococcus epidermidis was not detected in 7 false negative samples, but PCR/sequencing instead detected Staphylococcus aureus (4), Staphylococcus capitis (1), Staphylococcus pettenkoferi (1), and Escherichia coli (1), which were not identified by standard laboratory procedures. B. Staphylococcus epidermidis was detected in 3/19 false positive samples using PCR/sequencing.

Table 65: Clinical Performance for Staphylococcus lugdunensis   

<table><tr><td rowspan=2 colspan=1>Target</td><td rowspan=2 colspan=1>Sample Type</td><td rowspan=1 colspan=2>Sensitivity/PPA</td><td rowspan=1 colspan=2>Specificity/NP</td></tr><tr><td rowspan=1 colspan=1>TP/TP+FN</td><td rowspan=1 colspan=1>% (95%CI)</td><td rowspan=1 colspan=1>TN/TN+FP</td><td rowspan=1 colspan=1>% (95%CI)</td></tr><tr><td rowspan=7 colspan=1>Staphylococcuslugdunensis</td><td rowspan=1 colspan=1>Prospective (Fresh)</td><td rowspan=1 colspan=1>1/1</td><td rowspan=1 colspan=1>100(20.7-100)</td><td rowspan=1 colspan=1>290/291</td><td rowspan=1 colspan=1>99.7(98.1-99.9)</td></tr><tr><td rowspan=1 colspan=1>Prospective (Frozen)</td><td rowspan=1 colspan=1>1/1</td><td rowspan=1 colspan=1>100(20.7-100)</td><td rowspan=1 colspan=1>321/322</td><td rowspan=1 colspan=1>99.7(98.3-99.9)</td></tr><tr><td rowspan=1 colspan=1>Prospective (All)</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>100(34.2-100)</td><td rowspan=1 colspan=1>611/613</td><td rowspan=1 colspan=1>99.7(98.8-99.9)</td></tr><tr><td rowspan=1 colspan=1>Retrospective</td><td rowspan=1 colspan=1>4/4</td><td rowspan=1 colspan=1>100(51.0-100)</td><td rowspan=1 colspan=1>579/579</td><td rowspan=1 colspan=1>100(99.3-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective/Retrospective</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>100(61.0-100)</td><td rowspan=1 colspan=1>1190/1192</td><td rowspan=1 colspan=1>99.8(99.4-100)</td></tr><tr><td rowspan=1 colspan=1>Contrived</td><td rowspan=1 colspan=1>45/45</td><td rowspan=1 colspan=1>100(92.1-100)</td><td rowspan=1 colspan=1>519/520</td><td rowspan=1 colspan=1>99.8(98.9-100)</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>51/51</td><td rowspan=1 colspan=1>100(93.0-100)</td><td rowspan=1 colspan=1>1709/1712A</td><td rowspan=1 colspan=1>99.8(99.5-99.9)</td></tr></table>

A. Staphylococcus lugdunensis was detected in 2/2 false positive samples using PCR/sequencing.

<table><tr><td rowspan=2 colspan=1>Target</td><td rowspan=2 colspan=1>Sample Type</td><td rowspan=1 colspan=2>Sensitivity/PPA</td><td rowspan=1 colspan=2>Specificity/NP</td></tr><tr><td rowspan=1 colspan=1>TP/TP+FN</td><td rowspan=1 colspan=1>% (95%CI)</td><td rowspan=1 colspan=1>TN/TN+FP</td><td rowspan=1 colspan=1>% (95%CI)</td></tr><tr><td rowspan=7 colspan=1>Streptococcus</td><td rowspan=1 colspan=1>Prospective (Fresh)</td><td rowspan=1 colspan=1>54/55</td><td rowspan=1 colspan=1>98.2(90.4-99.7)</td><td rowspan=1 colspan=1>253/257</td><td rowspan=1 colspan=1>98.4(96.1-99.4)</td></tr><tr><td rowspan=1 colspan=1>Prospective (Frozen)</td><td rowspan=1 colspan=1>49/55</td><td rowspan=1 colspan=1>89.1(78.2-94.9)</td><td rowspan=1 colspan=1>341/344</td><td rowspan=1 colspan=1>99.1(97.5-99.7)</td></tr><tr><td rowspan=1 colspan=1>Prospective (All)</td><td rowspan=1 colspan=1>103/110</td><td rowspan=1 colspan=1>93.6(87.4-96.9)</td><td rowspan=1 colspan=1>594/601</td><td rowspan=1 colspan=1>98.8(97.6-99.4)</td></tr><tr><td rowspan=1 colspan=1>Retrospective</td><td rowspan=1 colspan=1>171/173</td><td rowspan=1 colspan=1>98.8(95.9-99.7)</td><td rowspan=1 colspan=1>411/413</td><td rowspan=1 colspan=1>99.5(98.3-99.9)</td></tr><tr><td rowspan=1 colspan=1>Prospective/Retrospective</td><td rowspan=1 colspan=1>274/283</td><td rowspan=1 colspan=1>96.8(94.1-98.3)</td><td rowspan=1 colspan=1>1005/1014</td><td rowspan=1 colspan=1>99.1(98.3-99.5)</td></tr><tr><td rowspan=1 colspan=1>Contrived</td><td rowspan=1 colspan=1>57/57</td><td rowspan=1 colspan=1>100(93.7-100)</td><td rowspan=1 colspan=1>508/508</td><td rowspan=1 colspan=1>100(99.2-100)</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>331/340</td><td rowspan=1 colspan=1>97.4(95.0-98.6)</td><td rowspan=1 colspan=1>1513/1522A</td><td rowspan=1 colspan=1>99.4(98.9-99.7)</td></tr></table>

A. Streptococcus was detected in 8/9 false positive samples using PCR/sequencing.

Table 67: Clinical Performance for Streptococcus agalactiae   

<table><tr><td rowspan=2 colspan=1>Target</td><td rowspan=2 colspan=1>Sample Type</td><td rowspan=1 colspan=2>Sensitivity/PPA</td><td rowspan=1 colspan=2>Specificity/NP</td></tr><tr><td rowspan=1 colspan=1>TP/TP+FN</td><td rowspan=1 colspan=1>% (95%CI)</td><td rowspan=1 colspan=1>TN/TN+FP</td><td rowspan=1 colspan=1>% (95%CI</td></tr><tr><td rowspan=7 colspan=1>Streptococcusagalactiae</td><td rowspan=1 colspan=1>Prospectivee (Fresh)</td><td rowspan=1 colspan=1>5/6</td><td rowspan=1 colspan=1>83.3(43.6-97.0)</td><td rowspan=1 colspan=1>298/300</td><td rowspan=1 colspan=1>99.3(97.6-99.8)</td></tr><tr><td rowspan=1 colspan=1>Prospective (Frozen)</td><td rowspan=1 colspan=1>5/5</td><td rowspan=1 colspan=1>100(56.6-100)</td><td rowspan=1 colspan=1>374/374</td><td rowspan=1 colspan=1>100(99.0-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective (All)</td><td rowspan=1 colspan=1>10/11</td><td rowspan=1 colspan=1>90.9(62.3-98.4)</td><td rowspan=1 colspan=1>672/674</td><td rowspan=1 colspan=1>99.7(98.9-99.9)</td></tr><tr><td rowspan=1 colspan=1>Retrospective</td><td rowspan=1 colspan=1>36/37</td><td rowspan=1 colspan=1>97.3(86.2-99.5)</td><td rowspan=1 colspan=1>548/548</td><td rowspan=1 colspan=1>100(99.3-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective/Retrospective</td><td rowspan=1 colspan=1>46/48</td><td rowspan=1 colspan=1>95.8(86.0-98.8)</td><td rowspan=1 colspan=1>1220/1222</td><td rowspan=1 colspan=1>99.8(99.4-100)</td></tr><tr><td rowspan=1 colspan=1>Contrived</td><td rowspan=1 colspan=1>8/8</td><td rowspan=1 colspan=1>100(67.6-100)</td><td rowspan=1 colspan=1>557/557</td><td rowspan=1 colspan=1>100(99.3-100)</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>54/564</td><td rowspan=1 colspan=1>96.4(87.9-99.0)</td><td rowspan=1 colspan=1>1777/1779B</td><td rowspan=1 colspan=1>99.9(99.6-100)</td></tr></table>

A. Streptococcus agalactiae was not detected in 1 false negative sample, but PCR/sequencing instead detected Streptococcus mitis, which was not identified by standard laboratory procedures. B. Streptococcus agalactiae was detected in 1/2 false positive samples using PCR/sequencing.

Table 68: Clinical Performance for Streptococcus anginosus group   

<table><tr><td rowspan=2 colspan=1>Target</td><td rowspan=2 colspan=1>Sample Type</td><td rowspan=1 colspan=2>Sensitivity/PPA</td><td rowspan=1 colspan=2>Specificity/NP</td></tr><tr><td rowspan=1 colspan=1>TP/TP+FN</td><td rowspan=1 colspan=1>% (95%CI)</td><td rowspan=1 colspan=1>TN/TN+FP</td><td rowspan=1 colspan=1>% (95%CI</td></tr><tr><td rowspan=7 colspan=1>Streptococcusanginosus group</td><td rowspan=1 colspan=1>Prospective (Fresh)</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>100(43.9-100)</td><td rowspan=1 colspan=1>303/303</td><td rowspan=1 colspan=1>100(98.7-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective (Frozen)</td><td rowspan=1 colspan=1>1/2</td><td rowspan=1 colspan=1>50.0(9.5-90.5)</td><td rowspan=1 colspan=1>375/377</td><td rowspan=1 colspan=1>99.5(98.1-99.9)</td></tr><tr><td rowspan=1 colspan=1>Prospective (All)</td><td rowspan=1 colspan=1>4/5</td><td rowspan=1 colspan=1>80.0(37.6-96.4)</td><td rowspan=1 colspan=1>678/680</td><td rowspan=1 colspan=1>99.7(98.9-99.9)</td></tr><tr><td rowspan=1 colspan=1>Retrospective</td><td rowspan=1 colspan=1>38/40</td><td rowspan=1 colspan=1>95.0(83.5-98.6)</td><td rowspan=1 colspan=1>544/545</td><td rowspan=1 colspan=1>99.8(99.0-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective/Retrospective</td><td rowspan=1 colspan=1>42/45</td><td rowspan=1 colspan=1>93.3(82.1-97.7)</td><td rowspan=1 colspan=1>1222/1225</td><td rowspan=1 colspan=1>99.8(99.3-99.9)</td></tr><tr><td rowspan=1 colspan=1>Contrived</td><td rowspan=1 colspan=1>23/23</td><td rowspan=1 colspan=1>100(85.7-100)</td><td rowspan=1 colspan=1>542/542</td><td rowspan=1 colspan=1>100(99.3-100)</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>65/684</td><td rowspan=1 colspan=1>95.6(87.8-98.5)</td><td rowspan=1 colspan=1>1764/1767B</td><td rowspan=1 colspan=1>99.8(99.5-99.9)</td></tr></table>

A. Streptococcus anginosus group was not detected in 3 false negative samples, but PCR/sequencing instead detected Granulicatella adiacens, Streptococcus dysgalactiae, and Streptococcus lutetiensis, which were not identified by standard laboratory procedures. B. Streptococcus intermedius was detected in 1/3 false positive samples using PCR/sequencing.

Table 69: Clinical Performance for Streptococcus pneumoniae   

<table><tr><td rowspan=2 colspan=1>Target</td><td rowspan=2 colspan=1>Sample Type</td><td rowspan=1 colspan=2>Sensitivity/PPA</td><td rowspan=1 colspan=2>Specificity/NP</td></tr><tr><td rowspan=1 colspan=1>TP/TP+FN</td><td rowspan=1 colspan=1>% (95%CI)</td><td rowspan=1 colspan=1>TN/TN+FP</td><td rowspan=1 colspan=1>% (95%CI)</td></tr><tr><td rowspan=7 colspan=1>Streptococcuspneumoniae</td><td rowspan=1 colspan=1>Prospective(Fresh)</td><td rowspan=1 colspan=1>19/19</td><td rowspan=1 colspan=1>100(83.2-100)</td><td rowspan=1 colspan=1>286/287</td><td rowspan=1 colspan=1>99.7(98.1-99.9)</td></tr><tr><td rowspan=1 colspan=1>Prospective (Frozen)</td><td rowspan=1 colspan=1>8/9</td><td rowspan=1 colspan=1>88.9(56.5-98.0)</td><td rowspan=1 colspan=1>370/370</td><td rowspan=1 colspan=1>100(99.0-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective (All)</td><td rowspan=1 colspan=1>27/28</td><td rowspan=1 colspan=1>96.4(82.3-99.4)</td><td rowspan=1 colspan=1>656/657</td><td rowspan=1 colspan=1>99.8(99.1-100)</td></tr><tr><td rowspan=1 colspan=1>Retrospective</td><td rowspan=1 colspan=1>39/41</td><td rowspan=1 colspan=1>95.1(83.9-98.7)</td><td rowspan=1 colspan=1>542/543</td><td rowspan=1 colspan=1>99.8(99.0-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective/Retrospective</td><td rowspan=1 colspan=1>66/69</td><td rowspan=1 colspan=1>95.7(88.0-98.5)</td><td rowspan=1 colspan=1>1198/1200</td><td rowspan=1 colspan=1>99.8(99.4-100)</td></tr><tr><td rowspan=1 colspan=1>Contrived</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>---</td><td rowspan=1 colspan=1>565/565</td><td rowspan=1 colspan=1>10099.3-100)</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>66/694</td><td rowspan=1 colspan=1>95.7(88.0-98.5)</td><td rowspan=1 colspan=1>1763/1765B</td><td rowspan=1 colspan=1>99.9(99.6-100)</td></tr></table>

A. Streptococcus pneumoniae was not detected in 3 false negative samples, but PCR/sequencing instead detected Streptococcus mitis (2) and Streptococcus anginosus (1), which were not detected by standard laboratory procedures. B. Streptococcus pneumoniae was detected in $1 / 2$ false positive samples using PCR/sequencing.

Table 70: Clinical Performance for Streptococcus pyogenes   

<table><tr><td rowspan=2 colspan=1>Target</td><td rowspan=2 colspan=1>Sample Type</td><td rowspan=1 colspan=2>Sensitivity/PPA</td><td rowspan=1 colspan=2>Specificity/NP</td></tr><tr><td rowspan=1 colspan=1>TP/TP+FN</td><td rowspan=1 colspan=1>% (95%CI)</td><td rowspan=1 colspan=1>TN/TN+FP</td><td rowspan=1 colspan=1>% (95%CI</td></tr><tr><td rowspan=7 colspan=1>Streptococcuspyogenes</td><td rowspan=1 colspan=1>Prospective(Fresh)</td><td rowspan=1 colspan=1>4/4</td><td rowspan=1 colspan=1>100(51.0-100)</td><td rowspan=1 colspan=1>302/302</td><td rowspan=1 colspan=1>100(98.7-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective (Frozen)</td><td rowspan=1 colspan=1>4/4</td><td rowspan=1 colspan=1>100(51.0-100)</td><td rowspan=1 colspan=1>375/375</td><td rowspan=1 colspan=1>100(99.0-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective (All)</td><td rowspan=1 colspan=1>8/8</td><td rowspan=1 colspan=1>100(67.6-100)</td><td rowspan=1 colspan=1>677/677</td><td rowspan=1 colspan=1>100(99.4-100)</td></tr><tr><td rowspan=1 colspan=1>Retrospective</td><td rowspan=1 colspan=1>19/20</td><td rowspan=1 colspan=1>95.0(76.4-99.1)</td><td rowspan=1 colspan=1>564/564</td><td rowspan=1 colspan=1>100(99.3-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective/Retrospective</td><td rowspan=1 colspan=1>27/28</td><td rowspan=1 colspan=1>96.4(82.3-99.4)</td><td rowspan=1 colspan=1>1241/1241</td><td rowspan=1 colspan=1>100(99.7-100)</td></tr><tr><td rowspan=1 colspan=1>Contrived</td><td rowspan=1 colspan=1>26/26</td><td rowspan=1 colspan=1>100(87.1-100)</td><td rowspan=1 colspan=1>539/539</td><td rowspan=1 colspan=1>100(99.3-100)</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>53/54</td><td rowspan=1 colspan=1>98.1(90.2-99.7)</td><td rowspan=1 colspan=1>1780/1780</td><td rowspan=1 colspan=1>100(99.8-100)</td></tr></table>

Table 71: Clinical Performance for mecA   

<table><tr><td colspan="1" rowspan="2">Target</td><td colspan="1" rowspan="2">Sample Type</td><td colspan="2" rowspan="1">Sensitivity/PPA</td><td colspan="2" rowspan="1">Specificity/NP</td></tr><tr><td colspan="1" rowspan="1">TP/TP+FN</td><td colspan="1" rowspan="1">% (95%CI)</td><td colspan="1" rowspan="1">TN/TN+FP</td><td colspan="1" rowspan="1">% (95%CI)</td></tr><tr><td colspan="1" rowspan="7">mecCStaphylococcus</td><td colspan="1" rowspan="1">Prospective (Fresh)</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">---</td><td colspan="1" rowspan="1">182/182</td><td colspan="1" rowspan="1">100(97.9-100)</td></tr><tr><td colspan="1" rowspan="1">Prospective (Frozen)</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">274/274</td><td colspan="1" rowspan="1">100(98.6-100)</td></tr><tr><td colspan="1" rowspan="1">Prospective (All)</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">456/456</td><td colspan="1" rowspan="1">100(99.2-100)</td></tr><tr><td colspan="1" rowspan="1">Retrospective</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">191/191</td><td colspan="1" rowspan="1">100(98.0-100)</td></tr><tr><td colspan="1" rowspan="1">Prospective/Retrospective</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">---</td><td colspan="1" rowspan="1">647/647</td><td colspan="1" rowspan="1">100(99.4-100)</td></tr><tr><td colspan="1" rowspan="1">Contrived</td><td colspan="1" rowspan="1">49/49</td><td colspan="1" rowspan="1">100(92.7-100)</td><td colspan="1" rowspan="1">56/56</td><td colspan="1" rowspan="1">100(93.6-100)</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">49/49</td><td colspan="1" rowspan="1">100(92.7-100)</td><td colspan="1" rowspan="1">703/703</td><td colspan="1" rowspan="1">100(99.5-100)</td></tr><tr><td colspan="1" rowspan="5">mecCStaphylococcusaureus</td><td colspan="1" rowspan="1">Prospective (Fresh)</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">---</td><td colspan="1" rowspan="1">65/65</td><td colspan="1" rowspan="1">100(94.4-100)</td></tr><tr><td colspan="1" rowspan="1">Prospective (Frozen)</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">---</td><td colspan="1" rowspan="1">101/101</td><td colspan="1" rowspan="1">100(96.3-100)</td></tr><tr><td colspan="1" rowspan="1">Prospective (All)</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">---</td><td colspan="1" rowspan="1">166/166</td><td colspan="1" rowspan="1">100(97.7-100)</td></tr><tr><td colspan="1" rowspan="1">Retrospective</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">---</td><td colspan="1" rowspan="1">125/125</td><td colspan="1" rowspan="1">100(97.0-100)</td></tr><tr><td colspan="1" rowspan="1">Prospective/Retrospective</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">---</td><td colspan="1" rowspan="1">291/291</td><td colspan="1" rowspan="1">100(98.7-100)</td></tr><tr><td colspan="1" rowspan="1">TP/TP+FN</td><td colspan="1" rowspan="1">% (95%CI)</td><td colspan="1" rowspan="1">TN/TN+FP</td><td colspan="1" rowspan="1">% (95%CI</td></tr><tr><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="1">Contrived</td><td colspan="1" rowspan="1">49/49</td><td colspan="1" rowspan="1">100(92.7-100)</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">100(72.2-100)</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">49/49</td><td colspan="1" rowspan="1">100(92.7-100)</td><td colspan="1" rowspan="1">301/301</td><td colspan="1" rowspan="1">100(98.7-100)</td></tr><tr><td colspan="1" rowspan="7">mecCStaphylococcusepidermidis</td><td colspan="1" rowspan="1">Prospective (Fresh)</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">63/63</td><td colspan="1" rowspan="1">100 (94.3-100)</td></tr><tr><td colspan="1" rowspan="1">Prospective (Frozen)</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">58/58</td><td colspan="1" rowspan="1">100 (93.8-100)</td></tr><tr><td colspan="1" rowspan="1">Prospective (All)</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">121/121</td><td colspan="1" rowspan="1">100(96.9-100)</td></tr><tr><td colspan="1" rowspan="1">Retrospective</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">38/38</td><td colspan="1" rowspan="1">100(90.8-100)</td></tr><tr><td colspan="1" rowspan="1">Prospective/Retrospective</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">---</td><td colspan="1" rowspan="1">159/159</td><td colspan="1" rowspan="1">100(97.6-100)</td></tr><tr><td colspan="1" rowspan="1">Contrived</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100(20.7-100)</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">---</td><td colspan="1" rowspan="1">160/160</td><td colspan="1" rowspan="1">100(97.7-100)</td></tr><tr><td colspan="1" rowspan="7">mecCStaphylococcuslugdunensis</td><td colspan="1" rowspan="1">Prospective (Fresh)</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">---</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100(20.7-100)</td></tr><tr><td colspan="1" rowspan="1">Prospective (Frozen)</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100(20.7-100)</td></tr><tr><td colspan="1" rowspan="1">Prospective (All)</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2/2</td><td colspan="1" rowspan="1">100(34.2-100)</td></tr><tr><td colspan="1" rowspan="1">Retrospective</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">---</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">100(51.0-100)</td></tr><tr><td colspan="1" rowspan="1">Prospective/Retrospective</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">---</td><td colspan="1" rowspan="1">6/6</td><td colspan="1" rowspan="1">100(61.0-100)</td></tr><tr><td colspan="1" rowspan="1">Contrived</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">---</td><td colspan="1" rowspan="1">45/45</td><td colspan="1" rowspan="1">100(92.1-100)</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">---</td><td colspan="1" rowspan="1">51/51</td><td colspan="1" rowspan="1">100(93.0-100)</td></tr></table>

Table 72: Clinical Performance for vanA   

<table><tr><td rowspan=1 colspan=1>Table 72: Clinical Pe rformance for vanA</td><td rowspan=1 colspan=5>Table 72: Clinical Pe rformance for vanA</td></tr><tr><td rowspan=2 colspan=1>Target</td><td rowspan=2 colspan=1>Sample Type</td><td rowspan=1 colspan=2>Sensitivity/PPA</td><td rowspan=1 colspan=2>Specificity/NP</td></tr><tr><td rowspan=1 colspan=1>TP/TP+FN</td><td rowspan=1 colspan=1>% (95%CI)</td><td rowspan=1 colspan=1>TN/TN+FP</td><td rowspan=1 colspan=1>% (95%CII</td></tr><tr><td rowspan=7 colspan=1>vanAEnterococcus</td><td rowspan=1 colspan=1>Prospective (Fresh)</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>---</td><td rowspan=1 colspan=1>24/25</td><td rowspan=1 colspan=1>96.0(80.5-99.3)</td></tr><tr><td rowspan=1 colspan=1>Prospective (Frozen)</td><td rowspan=1 colspan=1>8/8</td><td rowspan=1 colspan=1>100(67.6-100)</td><td rowspan=1 colspan=1>28/28</td><td rowspan=1 colspan=1>100(87.9-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective (All)</td><td rowspan=1 colspan=1>8/8</td><td rowspan=1 colspan=1>100(67.6-100)</td><td rowspan=1 colspan=1>52/53</td><td rowspan=1 colspan=1>98.1(90.1-99.7)</td></tr><tr><td rowspan=1 colspan=1>Retrospective</td><td rowspan=1 colspan=1>53/57</td><td rowspan=1 colspan=1>93.0(83.3-97.2)</td><td rowspan=1 colspan=1>89/90</td><td rowspan=1 colspan=1>98.9(94.0-99.8)</td></tr><tr><td rowspan=1 colspan=1>Prospective/Retrospective</td><td rowspan=1 colspan=1>61/65</td><td rowspan=1 colspan=1>93.8(85.2-97.6)</td><td rowspan=1 colspan=1>141/143</td><td rowspan=1 colspan=1>98.6(95.0-99.6)</td></tr><tr><td rowspan=1 colspan=1>Contrived</td><td rowspan=1 colspan=1>60/60</td><td rowspan=1 colspan=1>100(94.0-100)</td><td rowspan=1 colspan=1>66/66</td><td rowspan=1 colspan=1>100(94.5-100)</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>121/125</td><td rowspan=1 colspan=1>96.8(92.1-98.7)</td><td rowspan=1 colspan=1>207/209</td><td rowspan=1 colspan=1>99.0(96.6-99.7)</td></tr><tr><td rowspan=7 colspan=1>vanAEnterococcusfaecalis</td><td rowspan=1 colspan=1>Prospective (Fresh)</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>---</td><td rowspan=1 colspan=1>21/21</td><td rowspan=1 colspan=1>100(84.5-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective (Frozen)</td><td rowspan=1 colspan=1>1/1</td><td rowspan=1 colspan=1>100(20.7-100)</td><td rowspan=1 colspan=1>27/27</td><td rowspan=1 colspan=1>100(87.5-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective (All)</td><td rowspan=1 colspan=1>1/1</td><td rowspan=1 colspan=1>100(20.7-100)</td><td rowspan=1 colspan=1>48/48</td><td rowspan=1 colspan=1>100(92.6-100)</td></tr><tr><td rowspan=1 colspan=1>Retrospective</td><td rowspan=1 colspan=1>11/14</td><td rowspan=1 colspan=1>78.6(52.4-92.4)</td><td rowspan=1 colspan=1>76/76</td><td rowspan=1 colspan=1>100(95.2-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective/Retrospective</td><td rowspan=1 colspan=1>12/15</td><td rowspan=1 colspan=1>80.0(54.8-93.0)</td><td rowspan=1 colspan=1>124/124</td><td rowspan=1 colspan=1>100(97.0-100)</td></tr><tr><td rowspan=1 colspan=1>Contrived</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>100(72.2-100)</td><td rowspan=1 colspan=1>42/42</td><td rowspan=1 colspan=1>100(91.6-100)</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>22/25</td><td rowspan=1 colspan=1>88.0(70.0-95.8)</td><td rowspan=1 colspan=1>166/166</td><td rowspan=1 colspan=1>100(97.7-100)</td></tr><tr><td rowspan=7 colspan=1>vanAEnterococcusfaecium</td><td rowspan=1 colspan=1>Prospective (Fresh)</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>---</td><td rowspan=1 colspan=1>2/3</td><td rowspan=1 colspan=1>66.7(20.8-93.9)</td></tr><tr><td rowspan=1 colspan=1>Prospective (Frozen)</td><td rowspan=1 colspan=1>7/7</td><td rowspan=1 colspan=1>100(64.6-100)</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>100(34.2-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective (All)</td><td rowspan=1 colspan=1>7/7</td><td rowspan=1 colspan=1>100(64.6-100)</td><td rowspan=1 colspan=1>4/5</td><td rowspan=1 colspan=1>80.0(37.6-96.4)</td></tr><tr><td rowspan=1 colspan=1>Retrospective</td><td rowspan=1 colspan=1>44/44</td><td rowspan=1 colspan=1>100(92.0-100)</td><td rowspan=1 colspan=1>8/9</td><td rowspan=1 colspan=1>88.9(56.5-98.0)</td></tr><tr><td rowspan=1 colspan=1>Prospective/Retrospective</td><td rowspan=1 colspan=1>51/51</td><td rowspan=1 colspan=1>100(93.0-100)</td><td rowspan=1 colspan=1>12/14</td><td rowspan=1 colspan=1>85.7(60.1-96.0)</td></tr><tr><td rowspan=1 colspan=1>Contrived</td><td rowspan=1 colspan=1>50/50</td><td rowspan=1 colspan=1>100(92.9-100)</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>100(72.2-100)</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>101/101</td><td rowspan=1 colspan=1>100(96.3-100)</td><td rowspan=1 colspan=1>22/24A</td><td rowspan=1 colspan=1>91.7(74.2-97.7)</td></tr></table>

Table 74: Clinical Performance for vanB   

<table><tr><td rowspan=2 colspan=1>Target</td><td rowspan=2 colspan=1>Sample Type</td><td rowspan=1 colspan=2>Sensitivity/PPA</td><td rowspan=1 colspan=2>Specificity/NP</td></tr><tr><td rowspan=1 colspan=1>TP/TP+FN</td><td rowspan=1 colspan=1>% (95%CI</td><td rowspan=1 colspan=1>TN/TN+FP</td><td rowspan=1 colspan=1>% (95%CI</td></tr><tr><td rowspan=7 colspan=1>vanBEnterococcus</td><td rowspan=1 colspan=1>Prospective (Fresh)</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>---</td><td rowspan=1 colspan=1>25/25</td><td rowspan=1 colspan=1>100(86.7-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective (Frozen)</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>---</td><td rowspan=1 colspan=1>36/36</td><td rowspan=1 colspan=1>100(90.4-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective (All)</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>---</td><td rowspan=1 colspan=1>61/61</td><td rowspan=1 colspan=1>100(94.1-100)</td></tr><tr><td rowspan=1 colspan=1>Retrospective</td><td rowspan=1 colspan=1>1/1</td><td rowspan=1 colspan=1>100(20.7-100)</td><td rowspan=1 colspan=1>146/146</td><td rowspan=1 colspan=1>100(97.4-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective/Retrospective</td><td rowspan=1 colspan=1>1/1</td><td rowspan=1 colspan=1>100(20.7-100)</td><td rowspan=1 colspan=1>207/207</td><td rowspan=1 colspan=1>100(98.2-100)</td></tr><tr><td rowspan=1 colspan=1>Contrived</td><td rowspan=1 colspan=1>52/52</td><td rowspan=1 colspan=1>100(93.1-100)</td><td rowspan=1 colspan=1>74/74</td><td rowspan=1 colspan=1>100(95.1-100)</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>53/53</td><td rowspan=1 colspan=1>100(93.2-100)</td><td rowspan=1 colspan=1>281/281</td><td rowspan=1 colspan=1>100(98.7-100)</td></tr><tr><td rowspan=7 colspan=1>vanBEnterococcusfaecalis</td><td rowspan=1 colspan=1>Prospective (Fresh)</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>---</td><td rowspan=1 colspan=1>21/21</td><td rowspan=1 colspan=1>100(84.5-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective (Frozen)</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>---</td><td rowspan=1 colspan=1>28/28</td><td rowspan=1 colspan=1>100(87.9-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective (All)</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>---</td><td rowspan=1 colspan=1>49/49</td><td rowspan=1 colspan=1>100(92.7-100)</td></tr><tr><td rowspan=1 colspan=1>Retrospective</td><td rowspan=1 colspan=1>1/1</td><td rowspan=1 colspan=1>100(20.7-100)</td><td rowspan=1 colspan=1>89/89</td><td rowspan=1 colspan=1>100(95.9-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective/Retrospective</td><td rowspan=1 colspan=1>1/1</td><td rowspan=1 colspan=1>100(20.7-100)</td><td rowspan=1 colspan=1>138/138</td><td rowspan=1 colspan=1>100(97.3-100)</td></tr><tr><td rowspan=1 colspan=1>Contrived</td><td rowspan=1 colspan=1>42/42</td><td rowspan=1 colspan=1>100(91.6-100)</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>100(72.2-100)</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>43/43</td><td rowspan=1 colspan=1>100(91.8-100)</td><td rowspan=1 colspan=1>148/148</td><td rowspan=1 colspan=1>100(97.5-100)</td></tr><tr><td rowspan=7 colspan=1>vanBEnterococcusfaecium</td><td rowspan=1 colspan=1>Prospective (Fresh)</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>---</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>100(43.9-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective (Frozen)</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>---</td><td rowspan=1 colspan=1>9/9</td><td rowspan=1 colspan=1>100(70.1-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective (All)</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>---</td><td rowspan=1 colspan=1>12/12</td><td rowspan=1 colspan=1>100(75.8-100)</td></tr><tr><td rowspan=1 colspan=1>Retrospective</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>---</td><td rowspan=1 colspan=1>53/53</td><td rowspan=1 colspan=1>100(93.2-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective/Retrospective</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>---</td><td rowspan=1 colspan=1>65/65</td><td rowspan=1 colspan=1>100(94.4-100)</td></tr><tr><td rowspan=1 colspan=1>Contrived</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>100(72.2-100)</td><td rowspan=1 colspan=1>50/50</td><td rowspan=1 colspan=1>100(92.9-100)</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>100(72.2-100)</td><td rowspan=1 colspan=1>115/115</td><td rowspan=1 colspan=1>100(96.8-100)</td></tr></table>

In addition to the evaluable prospective and retrospective samples that contain grampositive organisms, the clinical performance of the Pan Candida and Pan Gram-Negative targets was evaluated by testing an additional 480 non-intended use retrospective samples with gram-negative or fungal organisms; these are denoted as Retrospective (NonIntended Use) samples. Results from those samples are summarized in Table 74.

Table 74: Clinical Performance for Pan Targets   

<table><tr><td rowspan=2 colspan=1>Target</td><td rowspan=2 colspan=1>Sample Type</td><td rowspan=1 colspan=2>Sensitivity/PPA</td><td rowspan=1 colspan=2>Specificity/NP</td></tr><tr><td rowspan=1 colspan=1>TP/TP+FN</td><td rowspan=1 colspan=1>% (95%CI)</td><td rowspan=1 colspan=1>TN/TN+FP</td><td rowspan=1 colspan=1>% (95%CI</td></tr><tr><td rowspan=6 colspan=1>Pan Candida</td><td rowspan=1 colspan=1>Prospective (Fresh)</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>312/312</td><td rowspan=1 colspan=1>100(98.8-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective (Frozen)</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>399/399</td><td rowspan=1 colspan=1>100(99.0-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective (All)</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>---</td><td rowspan=1 colspan=1>711/711</td><td rowspan=1 colspan=1>100(99.5-100)</td></tr><tr><td rowspan=1 colspan=1>Retrospective</td><td rowspan=1 colspan=1>7/9</td><td rowspan=1 colspan=1>77.8(45.3-93.7)</td><td rowspan=1 colspan=1>576/577</td><td rowspan=1 colspan=1>99.8(99.0-100)</td></tr><tr><td rowspan=1 colspan=1>Retrospective(Non-Intended Use)</td><td rowspan=1 colspan=1>90/96</td><td rowspan=1 colspan=1>93.8(87.0-97.1)</td><td rowspan=1 colspan=1>383/384</td><td rowspan=1 colspan=1>99.7(98.5-100)</td></tr><tr><td rowspan=1 colspan=1>Contrived</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>---</td><td rowspan=1 colspan=1>565/565</td><td rowspan=1 colspan=1>100(99.3-100)</td></tr><tr><td rowspan=6 colspan=1>Pan Gram-Negative</td><td rowspan=1 colspan=1>Prospective (Fresh)</td><td rowspan=1 colspan=1>10/11</td><td rowspan=1 colspan=1>90.9(62.3-98.4)</td><td rowspan=1 colspan=1>299/301</td><td rowspan=1 colspan=1>99.3(97.6-99.8)</td></tr><tr><td rowspan=1 colspan=1>Prospective (Frozen)</td><td rowspan=1 colspan=1>12/12</td><td rowspan=1 colspan=1>100(75.8-100)</td><td rowspan=1 colspan=1>386/387</td><td rowspan=1 colspan=1>99.7(98.6-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective (All)</td><td rowspan=1 colspan=1>22/23</td><td rowspan=1 colspan=1>95.7(79.0-99.2)</td><td rowspan=1 colspan=1>685/688</td><td rowspan=1 colspan=1>99.6(98.7-99.9)</td></tr><tr><td rowspan=1 colspan=1>Retrospective</td><td rowspan=1 colspan=1>36/43</td><td rowspan=1 colspan=1>83.7(70.0-91.9)</td><td rowspan=1 colspan=1>540/543</td><td rowspan=1 colspan=1>99.4(98.4-99.8)</td></tr><tr><td rowspan=1 colspan=1>Retrospective(Non-Intended Use)</td><td rowspan=1 colspan=1>364/375</td><td rowspan=1 colspan=1>97.1(94.8-98.4)</td><td rowspan=1 colspan=1>104/105</td><td rowspan=1 colspan=1>99.0(94.8-99.8)</td></tr><tr><td rowspan=1 colspan=1>Contrived</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>---</td><td rowspan=1 colspan=1>565/565</td><td rowspan=1 colspan=1>100(99.3-100)</td></tr></table>

Table 75: Contrived Sample Summary   

<table><tr><td colspan="1" rowspan="1">Target</td><td colspan="1" rowspan="1">Organism</td><td colspan="1" rowspan="1">Strain</td><td colspan="1" rowspan="1">Independent ContrivedSamples Tested</td></tr><tr><td colspan="1" rowspan="8">Bacillus cereusgroup</td><td colspan="1" rowspan="4">Bacillus cereus</td><td colspan="1" rowspan="1">ATCC 10876</td><td colspan="1" rowspan="1">11</td></tr><tr><td colspan="1" rowspan="1">ATCC 21769</td><td colspan="1" rowspan="1">10</td></tr><tr><td colspan="1" rowspan="1">ATCC 31430</td><td colspan="1" rowspan="1">9</td></tr><tr><td colspan="1" rowspan="1">ATCC 53522</td><td colspan="1" rowspan="1">10</td></tr><tr><td colspan="1" rowspan="3">Bacillus thuringiensis</td><td colspan="1" rowspan="1">ATCC 33679</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">ATCC 10792</td><td colspan="1" rowspan="1">2</td></tr><tr><td colspan="1" rowspan="1">ATCC 55173</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">Bacillus cereus group total</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">46</td></tr><tr><td colspan="1" rowspan="4">Bacillus subtilisgroup</td><td colspan="1" rowspan="3">Bacillus amyloliquefaciens</td><td colspan="1" rowspan="1">ATCC 23350</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">ATCC 23845</td><td colspan="1" rowspan="1">4</td></tr><tr><td colspan="1" rowspan="1">ATCC 53495</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">Bacillus atrophaeus</td><td colspan="1" rowspan="1">ATCC 51189</td><td colspan="1" rowspan="1">4</td></tr><tr><td colspan="1" rowspan="9">−</td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="1">ATCC 6455</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">ATCC 6537</td><td colspan="1" rowspan="1">4</td></tr><tr><td colspan="1" rowspan="3">Bacillus licheniformis</td><td colspan="1" rowspan="1">ATCC 21039</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">ATCC 21667</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">ATCC 53926</td><td colspan="1" rowspan="1">4</td></tr><tr><td colspan="1" rowspan="3">Bacillus subtilis</td><td colspan="1" rowspan="1">ATCC 15040</td><td colspan="1" rowspan="1">5</td></tr><tr><td colspan="1" rowspan="1">ATCC 15561</td><td colspan="1" rowspan="1">8</td></tr><tr><td colspan="1" rowspan="1">ATCC 55614</td><td colspan="1" rowspan="1">6</td></tr><tr><td colspan="1" rowspan="1">Bacillus subtilis group total</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">50</td></tr><tr><td colspan="1" rowspan="4">Corynebacterium</td><td colspan="1" rowspan="1">Corynebacterium coyleae</td><td colspan="1" rowspan="1">ATCC 700219</td><td colspan="1" rowspan="1">7</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium falsenii</td><td colspan="1" rowspan="1">ATCC BAA-596</td><td colspan="1" rowspan="1">9</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium striatum</td><td colspan="1" rowspan="1">ATCC BAA-1293</td><td colspan="1" rowspan="1">4</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium total</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">20</td></tr><tr><td colspan="1" rowspan="27">Enterococcus</td><td colspan="1" rowspan="1">Enterococcus faecalis, vanA</td><td colspan="1" rowspan="1">JMI 876745</td><td colspan="1" rowspan="1">10</td></tr><tr><td colspan="1" rowspan="4">Enterococcus faecalis, vanB</td><td colspan="1" rowspan="1">ATCC 51299</td><td colspan="1" rowspan="1">11</td></tr><tr><td colspan="1" rowspan="1">ATCC 51575</td><td colspan="1" rowspan="1">11</td></tr><tr><td colspan="1" rowspan="1">ATCC 700802</td><td colspan="1" rowspan="1">10</td></tr><tr><td colspan="1" rowspan="1">ATCC BAA-2365</td><td colspan="1" rowspan="1">10</td></tr><tr><td colspan="1" rowspan="13">Enterococcus faecium, vanA</td><td colspan="1" rowspan="1">ATCC 51559</td><td colspan="1" rowspan="1">4</td></tr><tr><td colspan="1" rowspan="1">ATCC 700221</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">ATCC BAA-2316</td><td colspan="1" rowspan="1">5</td></tr><tr><td colspan="1" rowspan="1">ATCC BAA-2317</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">ATCC BAA-2318</td><td colspan="1" rowspan="1">5</td></tr><tr><td colspan="1" rowspan="1">ATCC BAA-2319</td><td colspan="1" rowspan="1">5</td></tr><tr><td colspan="1" rowspan="1">ATCC BAA-2320</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">LMC 002867</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">LMC 003921</td><td colspan="1" rowspan="1">4</td></tr><tr><td colspan="1" rowspan="1">LMC 032261</td><td colspan="1" rowspan="1">4</td></tr><tr><td colspan="1" rowspan="1">LMC 055971</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">LMC 103676</td><td colspan="1" rowspan="1">5</td></tr><tr><td colspan="1" rowspan="1">LMC 104266</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">Enterococcus faecium, vanB</td><td colspan="1" rowspan="1">ATCC 51858</td><td colspan="1" rowspan="1">10</td></tr><tr><td colspan="1" rowspan="1">Enterococcus flavescens</td><td colspan="1" rowspan="1">ATCC 49996</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="2">Enterococcus gallinarum</td><td colspan="1" rowspan="1">ATCC 49610</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">ATCC 700425</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">Enterococcus hirae</td><td colspan="1" rowspan="1">ATCC 10541</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">Enterococcus malodoratus</td><td colspan="1" rowspan="1">ATCC 43197</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">Enterococcus raffinosus</td><td colspan="1" rowspan="1">ATCC 49464</td><td colspan="1" rowspan="1">2</td></tr><tr><td colspan="1" rowspan="1">Enterococcus saccharolyticus</td><td colspan="1" rowspan="1">ATCC 43076</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">Enterococcus total</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">126</td></tr><tr><td colspan="1" rowspan="10">Lactobacillus</td><td colspan="1" rowspan="3">Lactobacillus casei</td><td colspan="1" rowspan="1">ATCC 25598</td><td colspan="1" rowspan="1">2</td></tr><tr><td colspan="1" rowspan="1">ATCC 334</td><td colspan="1" rowspan="1">6</td></tr><tr><td colspan="1" rowspan="1">ATCC 39392</td><td colspan="1" rowspan="1">4</td></tr><tr><td colspan="1" rowspan="3">Lactobacillus paracasei</td><td colspan="1" rowspan="1">148-260 *</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">ATCC 27092</td><td colspan="1" rowspan="1">2</td></tr><tr><td colspan="1" rowspan="1">ATCC BAA-52</td><td colspan="1" rowspan="1">6</td></tr><tr><td colspan="1" rowspan="3">Lactobacillus rhamnosus</td><td colspan="1" rowspan="1">ATCC 39595</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">ATCC 53103</td><td colspan="1" rowspan="1">5</td></tr><tr><td colspan="1" rowspan="1">ATCC 55915</td><td colspan="1" rowspan="1">2</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus total</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">33</td></tr><tr><td colspan="1" rowspan="18">Listeria</td><td colspan="1" rowspan="2">Listeria innocua</td><td colspan="1" rowspan="1">ATCC 33090</td><td colspan="1" rowspan="1">4</td></tr><tr><td colspan="1" rowspan="1">NCTC 11288</td><td colspan="1" rowspan="1">5</td></tr><tr><td colspan="1" rowspan="3">Listeria ivanovii</td><td colspan="1" rowspan="1">ATCC 19119</td><td colspan="1" rowspan="1">2</td></tr><tr><td colspan="1" rowspan="1">ATCC 700402</td><td colspan="1" rowspan="1">4</td></tr><tr><td colspan="1" rowspan="1">ATCC BAA-139</td><td colspan="1" rowspan="1">4</td></tr><tr><td colspan="1" rowspan="10">Listeria monocytogenes</td><td colspan="1" rowspan="1">ATCC 13932</td><td colspan="1" rowspan="1">5</td></tr><tr><td colspan="1" rowspan="1">ATCC 19111</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">ATCC 19112</td><td colspan="1" rowspan="1">4</td></tr><tr><td colspan="1" rowspan="1">ATCC 19114</td><td colspan="1" rowspan="1">5</td></tr><tr><td colspan="1" rowspan="1">ATCC 19116</td><td colspan="1" rowspan="1">5</td></tr><tr><td colspan="1" rowspan="1">ATCC 19117</td><td colspan="1" rowspan="1">5</td></tr><tr><td colspan="1" rowspan="1">ATCC 19118</td><td colspan="1" rowspan="1">5</td></tr><tr><td colspan="1" rowspan="1">ATCC 7644</td><td colspan="1" rowspan="1">5</td></tr><tr><td colspan="1" rowspan="1">ATCC BAA-751</td><td colspan="1" rowspan="1">5</td></tr><tr><td colspan="1" rowspan="1">NCTC 10890</td><td colspan="1" rowspan="1">4</td></tr><tr><td colspan="1" rowspan="1">Listeria seeligeri</td><td colspan="1" rowspan="1">ATCC 35967</td><td colspan="1" rowspan="1">5</td></tr><tr><td colspan="1" rowspan="1">Listeria welshimeri</td><td colspan="1" rowspan="1">ATCC 35897</td><td colspan="1" rowspan="1">5</td></tr><tr><td colspan="1" rowspan="1">Listeria total</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">75</td></tr><tr><td colspan="1" rowspan="9">Micrococcus</td><td colspan="1" rowspan="6">Micrococcus luteus</td><td colspan="1" rowspan="1">ATCC 10240</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">ATCC 19212</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">ATCC 400</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">ATCC 4698</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">ATCC 49732</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">ATCC 53598</td><td colspan="1" rowspan="1">4</td></tr><tr><td colspan="1" rowspan="1">Micrococcus lylae</td><td colspan="1" rowspan="1">ATCC 27566</td><td colspan="1" rowspan="1">4</td></tr><tr><td colspan="1" rowspan="1">Micrococcus yunnanensis</td><td colspan="1" rowspan="1">ATCC 7468</td><td colspan="1" rowspan="1">4</td></tr><tr><td colspan="1" rowspan="1">Micrococcus total</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">27</td></tr><tr><td colspan="1" rowspan="5">Cutibacteriumacnes</td><td colspan="1" rowspan="4">Cutibacterium acnes</td><td colspan="1" rowspan="1">ATCC 11827</td><td colspan="1" rowspan="1">8</td></tr><tr><td colspan="1" rowspan="1">ATCC 11828</td><td colspan="1" rowspan="1">6</td></tr><tr><td colspan="1" rowspan="1">ATCC 33179</td><td colspan="1" rowspan="1">4</td></tr><tr><td colspan="1" rowspan="1">ATCC 6919</td><td colspan="1" rowspan="1">8</td></tr><tr><td colspan="1" rowspan="1">Cutibacterium acnes total</td><td colspan="1" rowspan="1">cYanmaGYellowb</td><td colspan="1" rowspan="1">26</td></tr><tr><td colspan="1" rowspan="12">Staphylococcus</td><td colspan="1" rowspan="3">Staphylococcus aureus, mecA</td><td colspan="1" rowspan="1">ATCC 33591</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">ATCC BAA-44</td><td colspan="1" rowspan="1">5</td></tr><tr><td colspan="1" rowspan="1">NCTC 12493</td><td colspan="1" rowspan="1">2</td></tr><tr><td colspan="1" rowspan="2">Staphylococcus aureus, mecC</td><td colspan="1" rowspan="1">ATCC BAA-2312</td><td colspan="1" rowspan="1">23</td></tr><tr><td colspan="1" rowspan="1">ATCC BAA-2313</td><td colspan="1" rowspan="1">26</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus epidermidis,mecA</td><td colspan="1" rowspan="1">ATCC 35984</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="5">Staphylococcus lugdunensis</td><td colspan="1" rowspan="1">ATCC 49576</td><td colspan="1" rowspan="1">9</td></tr><tr><td colspan="1" rowspan="1">NRS 878</td><td colspan="1" rowspan="1">9</td></tr><tr><td colspan="1" rowspan="1">NRS 879</td><td colspan="1" rowspan="1">9</td></tr><tr><td colspan="1" rowspan="1">NRS 880</td><td colspan="1" rowspan="1">9</td></tr><tr><td colspan="1" rowspan="1">NRS 881</td><td colspan="1" rowspan="1">9</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus total</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">105</td></tr><tr><td colspan="1" rowspan="17">Streptococcus</td><td colspan="1" rowspan="4">Streptococcus agalactiae</td><td colspan="1" rowspan="1">ATCC 12403</td><td colspan="1" rowspan="1">2</td></tr><tr><td colspan="1" rowspan="1">ATCC 12973</td><td colspan="1" rowspan="1">2</td></tr><tr><td colspan="1" rowspan="1">ATCC 13813</td><td colspan="1" rowspan="1">2</td></tr><tr><td colspan="1" rowspan="1">ATCC 27956</td><td colspan="1" rowspan="1">2</td></tr><tr><td colspan="1" rowspan="3">Streptococcus anginosus</td><td colspan="1" rowspan="1">ATCC 700231</td><td colspan="1" rowspan="1">5</td></tr><tr><td colspan="1" rowspan="1">ATCC 9895</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">NCTC 10713</td><td colspan="1" rowspan="1">5</td></tr><tr><td colspan="1" rowspan="2">Streptococcus constellatus</td><td colspan="1" rowspan="1">ATCC 27513</td><td colspan="1" rowspan="1">4</td></tr><tr><td colspan="1" rowspan="1">ATCC 27823</td><td colspan="1" rowspan="1">2</td></tr><tr><td colspan="1" rowspan="1">Streptococcus intermedius</td><td colspan="1" rowspan="1">ATCC 27335</td><td colspan="1" rowspan="1">4</td></tr><tr><td colspan="1" rowspan="6">Streptococcus pyogenes</td><td colspan="1" rowspan="1">ATCC 12344</td><td colspan="1" rowspan="1">5</td></tr><tr><td colspan="1" rowspan="1">ATCC12384</td><td colspan="1" rowspan="1">4</td></tr><tr><td colspan="1" rowspan="1">ATCC 14289</td><td colspan="1" rowspan="1">4</td></tr><tr><td colspan="1" rowspan="1">ATCC 19615</td><td colspan="1" rowspan="1">4</td></tr><tr><td colspan="1" rowspan="1">ATCC 49399</td><td colspan="1" rowspan="1">5</td></tr><tr><td colspan="1" rowspan="1">NCIMB 13285</td><td colspan="1" rowspan="1">4</td></tr><tr><td colspan="1" rowspan="1">Streptococcus total</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">57</td></tr></table>

\*Derived from clinical specimen

The ePlex BCID-GP Panel reports genus or group level results for Bacillus cereus group, Bacillus subtilis group, Corynebacterium, Enterococcus, Lactobacillus, Listeria, Micrococcus, Staphylococcus, Streptococcus, Streptococcus anginosus group, Pan GramNegative and Pan Candida targets. Sensitivity/PPA of these genus and group level targets for species as determined by comparator methods for all evaluable samples tested are summarized in Table 76.

Table 76: Species Detected in Genus and Group Assays by Comparator Methods   

<table><tr><td colspan="1" rowspan="2">TargetSpecies detectedbyComparator Method</td><td colspan="2" rowspan="1">Sensitivity/PPA(Pros pective)</td><td colspan="2" rowspan="1">Sensitivity/PPA(Retros pective)</td><td colspan="2" rowspan="1">Sensitivity/PPA(Contrived)</td><td colspan="2" rowspan="1">Sensitivity/PPA(Combined)</td></tr><tr><td colspan="1" rowspan="1">TP/TP+FN</td><td colspan="1" rowspan="1">% (95%CI</td><td colspan="1" rowspan="1">TP/TP+FN</td><td colspan="1" rowspan="1">% (95%CII</td><td colspan="1" rowspan="1">TP/TP+FN</td><td colspan="1" rowspan="1">% (95%CI</td><td colspan="1" rowspan="1">TP/TP+FN</td><td colspan="1" rowspan="1">% (95% CI)</td></tr><tr><td colspan="1" rowspan="1">Bacilluscereus group</td><td colspan="1" rowspan="1">5/5</td><td colspan="1" rowspan="1">100(56.6-100)</td><td colspan="1" rowspan="1">6/7</td><td colspan="1" rowspan="1">85.7(48.7-97.4)</td><td colspan="1" rowspan="1">46/46</td><td colspan="1" rowspan="1">100(92.3-100)</td><td colspan="1" rowspan="1">57/58</td><td colspan="1" rowspan="1">98.3(90.9-99.7)</td></tr><tr><td colspan="1" rowspan="1">Bacillus cereus</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">100(43.9-100)</td><td colspan="1" rowspan="1">6/7</td><td colspan="1" rowspan="1">85.7(48.7-97.4)</td><td colspan="1" rowspan="1">40/40</td><td colspan="1" rowspan="1">100(91.2-100)</td><td colspan="1" rowspan="1">49/50</td><td colspan="1" rowspan="1">98.0(89.5-99.6)</td></tr><tr><td colspan="1" rowspan="1">Bacillus thuringiensis</td><td colspan="1" rowspan="1">2/2</td><td colspan="1" rowspan="1">100(34.2-100)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">6/6</td><td colspan="1" rowspan="1">100(61.0-100)</td><td colspan="1" rowspan="1">8/8</td><td colspan="1" rowspan="1">100(67.6-100)</td></tr><tr><td colspan="1" rowspan="1">Bacillus subtilis group</td><td colspan="1" rowspan="1">2/2</td><td colspan="1" rowspan="1">100(34.2-100)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">50/50</td><td colspan="1" rowspan="1">100(92.9-100)</td><td colspan="1" rowspan="1">52/52</td><td colspan="1" rowspan="1">100(93.1-100)</td></tr><tr><td colspan="1" rowspan="1">Bacillusamyloliquefaciens</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100(20.7-100)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">100(72.2-100)</td><td colspan="1" rowspan="1">11/11</td><td colspan="1" rowspan="1">100(74.1-100)</td></tr><tr><td colspan="1" rowspan="1">Bacillus atrophaeus</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">11/11</td><td colspan="1" rowspan="1">100(74.1-100)</td><td colspan="1" rowspan="1">11/11</td><td colspan="1" rowspan="1">100(74.1-100)</td></tr><tr><td colspan="1" rowspan="1">Bacillus licheniformis</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">100(72.2-100)</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">100(72.2-100)</td></tr><tr><td colspan="1" rowspan="1">Bacillus subtilis</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100(20.7-100)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">19/19</td><td colspan="1" rowspan="1">100(83.2-100)</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">100(83.9-100)</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium</td><td colspan="1" rowspan="1">13/19</td><td colspan="1" rowspan="1">68.4(46.0-84.6)</td><td colspan="1" rowspan="1">27/32</td><td colspan="1" rowspan="1">84.4(68.2-93.1)</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">100(83.9-100)</td><td colspan="1" rowspan="1">60/71</td><td colspan="1" rowspan="1">84.5(74.3-91.1)</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium</td><td colspan="1" rowspan="1">4/9</td><td colspan="1" rowspan="1">44.4(18.9-73.3)</td><td colspan="1" rowspan="1">5/7</td><td colspan="1" rowspan="1">71.4(35.9-91.8)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">9/16</td><td colspan="1" rowspan="1">56.3(33.2-76.9)</td></tr><tr><td colspan="1" rowspan="1">Corynebacteriumafermentans</td><td colspan="1" rowspan="1">0/1</td><td colspan="1" rowspan="1">0.0(0.0-79.3)</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">100(43.9-100)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">3/4</td><td colspan="1" rowspan="1">75.0(30.1-95.4)</td></tr><tr><td colspan="1" rowspan="1">Corynebacteriumamycolatum</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100(20.7-100)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100 (20.7-100)</td></tr><tr><td colspan="1" rowspan="1">Corynebacteriumaurimucosum</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100(20.7-100)</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100(20.7-100)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">2/2</td><td colspan="1" rowspan="1">100(34.2-100)</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium casei</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100(20.7-100)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100(20.7-100)</td></tr><tr><td colspan="1" rowspan="1">Corynebacteriumcoyleae</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100(20.7-100)</td><td colspan="1" rowspan="1">2/2</td><td colspan="1" rowspan="1">100(34.2-100)</td><td colspan="1" rowspan="1">7/7</td><td colspan="1" rowspan="1">100(64.6-100)</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">100(72.2-100)</td></tr><tr><td colspan="1" rowspan="1">Corynebacteriumfalsenii</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">9/9</td><td colspan="1" rowspan="1">100(70.1-100)</td><td colspan="1" rowspan="1">9/9</td><td colspan="1" rowspan="1">100(70.1-100)</td></tr><tr><td colspan="1" rowspan="1">Corynebacteriumimitans</td><td colspan="1" rowspan="1">2/2</td><td colspan="1" rowspan="1">100(34.2-100)</td><td colspan="1" rowspan="1">2/2</td><td colspan="1" rowspan="1">100(34.2-100)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">100(51.0-100)</td></tr><tr><td colspan="1" rowspan="1">Corynebacteriumjeikeium</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">4/5</td><td colspan="1" rowspan="1">80.0(37.6-96.4)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">4/5</td><td colspan="1" rowspan="1">80.0(37.6-96.4)</td></tr><tr><td colspan="1" rowspan="1">Corynebacteriumkroppenstedti</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100(20.7-100)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100(20.7-100)</td></tr><tr><td colspan="1" rowspan="1">Corynebacteriummatruchotii</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100(20.7-100)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100(20.7-100)</td></tr><tr><td colspan="1" rowspan="1">Corynebacteriummucifaciens</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100(20.7-100)</td><td colspan="1" rowspan="1">2/2</td><td colspan="1" rowspan="1">100(34.2-100)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">100(43.9-100)</td></tr><tr><td colspan="1" rowspan="1">Corynebacteriumpseudotuberculosis</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">0/1</td><td colspan="1" rowspan="1">0.0(0.0-79.3)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">0/1</td><td colspan="1" rowspan="1">0.0(0.0-79.3)</td></tr><tr><td colspan="1" rowspan="1">Corynebacteriumstriatum</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100(20.7-100)</td><td colspan="1" rowspan="1">6/6</td><td colspan="1" rowspan="1">100(61.0-100)</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">100 (51.0- 0)</td><td colspan="1" rowspan="1">11/11</td><td colspan="1" rowspan="1">100(74.1-100)</td></tr><tr><td colspan="1" rowspan="1">Corynebacteriumtuberculostearicum</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100(20.7-100)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100(20.7-100)</td></tr><tr><td colspan="1" rowspan="2">TargetSpecies detectedbyComparator Method</td><td colspan="2" rowspan="1">Sensitivity/PPA(Prospective)</td><td colspan="2" rowspan="1">Sensitivity/PPA(Retros pective)</td><td colspan="2" rowspan="1">Sensitivity/PPAContrived)</td><td colspan="2" rowspan="1">Sensitivity/PPA(Combined)</td></tr><tr><td colspan="1" rowspan="1">TP/TP+FN</td><td colspan="1" rowspan="1">% (95%C</td><td colspan="1" rowspan="1">TP/TP+FN</td><td colspan="1" rowspan="1">% (95%C</td><td colspan="1" rowspan="1">TP/TP+FN</td><td colspan="1" rowspan="1">% (95%CII</td><td colspan="1" rowspan="1">TP/TP+FN</td><td colspan="1" rowspan="1">% (95% CI)</td></tr><tr><td colspan="1" rowspan="1">Corynebacteriumurealyticum</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">0/1</td><td colspan="1" rowspan="1">0.0(0.0-79.3)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">0/1</td><td colspan="1" rowspan="1">0.0(0.0-79.3)</td></tr><tr><td colspan="1" rowspan="1">Enterococcus</td><td colspan="1" rowspan="1">61/61</td><td colspan="1" rowspan="1">100(94.1-100)</td><td colspan="1" rowspan="1">139/147</td><td colspan="1" rowspan="1">94.6(89.6-97.2)</td><td colspan="1" rowspan="1">126/126</td><td colspan="1" rowspan="1">100 (97.0-10)</td><td colspan="1" rowspan="1">326/334</td><td colspan="1" rowspan="1">97.6(95.3-98.8)</td></tr><tr><td colspan="1" rowspan="1">Enterococcus</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100(20.7-100)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100(20.7-100)</td></tr><tr><td colspan="1" rowspan="1">Enterococcus avium</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100(20.7-100)</td><td colspan="1" rowspan="1">2/3</td><td colspan="1" rowspan="1">66.7(20.8-93.9)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">3/4</td><td colspan="1" rowspan="1">75.0(30.1-95.4)</td></tr><tr><td colspan="1" rowspan="1">Enterococcuscasseliflavus</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">0/1</td><td colspan="1" rowspan="1">0.0(0.0-79.3)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">0/1</td><td colspan="1" rowspan="1">0.0(0.0-79.3)</td></tr><tr><td colspan="1" rowspan="1">Enterococcuscasseliflavus/gallinarum</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100(20.7-100)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100 (20.7-100</td></tr><tr><td colspan="1" rowspan="1">Enterococcus faecalis</td><td colspan="1" rowspan="1">49/49</td><td colspan="1" rowspan="1">100(92.7-100)</td><td colspan="1" rowspan="1">85/90</td><td colspan="1" rowspan="1">94.4(87.6-97.6)</td><td colspan="1" rowspan="1">52/52</td><td colspan="1" rowspan="1">100(93.1-100)</td><td colspan="1" rowspan="1">186/191</td><td colspan="1" rowspan="1">97.4(94.0-98.9)</td></tr><tr><td colspan="1" rowspan="1">Enterococcus faecium</td><td colspan="1" rowspan="1">12/12</td><td colspan="1" rowspan="1">100(75.8-100)</td><td colspan="1" rowspan="1">52/53</td><td colspan="1" rowspan="1">98.1(90.1-99.7)</td><td colspan="1" rowspan="1">60/60</td><td colspan="1" rowspan="1">100(94.0-100)</td><td colspan="1" rowspan="1">124/125</td><td colspan="1" rowspan="1">99.2(95.6-99.9)</td></tr><tr><td colspan="1" rowspan="1">Enterococcus flavescens</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">100(43.9-100)</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">100(43.9-100)</td></tr><tr><td colspan="1" rowspan="1">Enterococcusgallinarum</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">2/2</td><td colspan="1" rowspan="1">100(34.2-100)</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">100(51.0-100)</td><td colspan="1" rowspan="1">6/6</td><td colspan="1" rowspan="1">100(61.0-100)</td></tr><tr><td colspan="1" rowspan="1">Enterococcus hirae</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100(20.7-100)</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100(20.7-100)</td></tr><tr><td colspan="1" rowspan="1">Enterococcusmalodoratus</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">100(43.9-100)</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">100(43.9-100)</td></tr><tr><td colspan="1" rowspan="1">Enterococcus raffinosus</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">2/2</td><td colspan="1" rowspan="1">100(34.2-100)</td><td colspan="1" rowspan="1">2/2</td><td colspan="1" rowspan="1">100(34.2-100)</td></tr><tr><td colspan="1" rowspan="1">Enterococcussaccharolyticus</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100(20.7-100)</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100(20.7-100)</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">100(51.0-100)</td><td colspan="1" rowspan="1">9/9</td><td colspan="1" rowspan="1">100(70.1-100)</td><td colspan="1" rowspan="1">32/33</td><td colspan="1" rowspan="1">97.0(84.7-99.5)</td><td colspan="1" rowspan="1">45/46</td><td colspan="1" rowspan="1">97.8(88.7-99.6)</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus casei</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100(20.7-100)</td><td colspan="1" rowspan="1">12/12</td><td colspan="1" rowspan="1">100(75.8-100)</td><td colspan="1" rowspan="1">13/13</td><td colspan="1" rowspan="1">100(77.2-100)</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus paracasei</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100(20.7-100)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">11/11</td><td colspan="1" rowspan="1">100(74.1-100)</td><td colspan="1" rowspan="1">12/12</td><td colspan="1" rowspan="1">100(75.8-100)</td></tr><tr><td colspan="1" rowspan="1">Lactobacillusrhamnosus</td><td colspan="1" rowspan="1">2/2</td><td colspan="1" rowspan="1">100(34.2-100)</td><td colspan="1" rowspan="1">8/8</td><td colspan="1" rowspan="1">100(67.6-100)</td><td colspan="1" rowspan="1">9/10</td><td colspan="1" rowspan="1">90.0(59.6-98.2)</td><td colspan="1" rowspan="1">19/20</td><td colspan="1" rowspan="1">95.0(76.4-99.1)</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus zeae</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100(20.7-100)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100(20.7-100)</td></tr><tr><td colspan="1" rowspan="1">Listeria</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">2/2</td><td colspan="1" rowspan="1">100(34.2-100)</td><td colspan="1" rowspan="1">74/75</td><td colspan="1" rowspan="1">98.7(92.8-99.8)</td><td colspan="1" rowspan="1">76/77</td><td colspan="1" rowspan="1">98.7(93.0-99.8)</td></tr><tr><td colspan="1" rowspan="1">Listeria innocua</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">9/9</td><td colspan="1" rowspan="1">100(70.1-100)</td><td colspan="1" rowspan="1">9/9</td><td colspan="1" rowspan="1">100(70.1-100)</td></tr><tr><td colspan="1" rowspan="1">Listeria ivanovii</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">9/10</td><td colspan="1" rowspan="1">90.0(59.6-98.2)</td><td colspan="1" rowspan="1">9/10</td><td colspan="1" rowspan="1">90.0(59.6-98.2)</td></tr><tr><td colspan="1" rowspan="1">Listeria monocytogenes</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">2/2</td><td colspan="1" rowspan="1">100(34.2-100)</td><td colspan="1" rowspan="1">46/46</td><td colspan="1" rowspan="1">100(92.3-100)</td><td colspan="1" rowspan="1">48/48</td><td colspan="1" rowspan="1">100(92.6-100)</td></tr><tr><td colspan="1" rowspan="2">TargetSpecies detectedbyComparator Method</td><td colspan="2" rowspan="1">Sensitivity/PPA(Prospective)</td><td colspan="2" rowspan="1">Sensitivity/PPA(Retros pective)</td><td colspan="2" rowspan="1">Sensitivity/PPA(Contrived)</td><td colspan="2" rowspan="1">Sensitivity/PPA(Combined)</td></tr><tr><td colspan="1" rowspan="1">TP/TP+FN</td><td colspan="1" rowspan="1">% (95%CI</td><td colspan="1" rowspan="1">TP/TP+FN</td><td colspan="1" rowspan="1">% (95%CII</td><td colspan="1" rowspan="1">TP/TP+FN</td><td colspan="1" rowspan="1">% (95%C</td><td colspan="1" rowspan="1">TP/TP+FN</td><td colspan="1" rowspan="1">% (95% CI)</td></tr><tr><td colspan="1" rowspan="1">Listeria seeligeri</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">5/5</td><td colspan="1" rowspan="1">100(56.6-100)</td><td colspan="1" rowspan="1">5/5</td><td colspan="1" rowspan="1">100(56.6-100)</td></tr><tr><td colspan="1" rowspan="1">Listeria welshimeri</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">5/5</td><td colspan="1" rowspan="1">100(56.6-100)</td><td colspan="1" rowspan="1">5/5</td><td colspan="1" rowspan="1">100(56.6-100)</td></tr><tr><td colspan="1" rowspan="1">Micrococcus</td><td colspan="1" rowspan="1">19/21</td><td colspan="1" rowspan="1">90.5(71.1-97.3)</td><td colspan="1" rowspan="1">20/23</td><td colspan="1" rowspan="1">87.0 (67.9-95.5)</td><td colspan="1" rowspan="1">27/27</td><td colspan="1" rowspan="1">100(87.5-100)</td><td colspan="1" rowspan="1">66/71</td><td colspan="1" rowspan="1">93.0(84.6-97.0)</td></tr><tr><td colspan="1" rowspan="1">Micrococcus</td><td colspan="1" rowspan="1">8/9</td><td colspan="1" rowspan="1">88.9(56.5-98.0)</td><td colspan="1" rowspan="1">10/13</td><td colspan="1" rowspan="1">76.9(49.7-91.8)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">18/22</td><td colspan="1" rowspan="1">81.8(61.5-92.7)</td></tr><tr><td colspan="1" rowspan="1">Micrococcus luteus</td><td colspan="1" rowspan="1">9/9</td><td colspan="1" rowspan="1">100(70.1-100)</td><td colspan="1" rowspan="1">8/8</td><td colspan="1" rowspan="1">100(67.6-100)</td><td colspan="1" rowspan="1">19/19</td><td colspan="1" rowspan="1">100(83.2-100)</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">100(90.4-100)</td></tr><tr><td colspan="1" rowspan="1">Micrococcusluteus/lylae</td><td colspan="1" rowspan="1">2/3</td><td colspan="1" rowspan="1">66.7(20.8-93.9)</td><td colspan="1" rowspan="1">2/2</td><td colspan="1" rowspan="1">100(34.2-100)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">4/5</td><td colspan="1" rowspan="1">80.0(37.6-96.4)</td></tr><tr><td colspan="1" rowspan="1">Micrococcus lylae</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">100(51.0-100)</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">100(51.0-100)</td></tr><tr><td colspan="1" rowspan="1">Micrococcusyunnanensis</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">100(51.0-100)</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">100(51.0-100)</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus</td><td colspan="1" rowspan="1">447/456</td><td colspan="1" rowspan="1">98.0(96.3-99.0)</td><td colspan="1" rowspan="1">185/191</td><td colspan="1" rowspan="1">96.9(93.3-98.6)</td><td colspan="1" rowspan="1">105/105</td><td colspan="1" rowspan="1">100(96.5-100)</td><td colspan="1" rowspan="1">737/752</td><td colspan="1" rowspan="1">98.0(96.7-98.8)</td></tr><tr><td colspan="1" rowspan="1">Coagulase-negativestaphylococci (CoNS)</td><td colspan="1" rowspan="1">18/18</td><td colspan="1" rowspan="1">100(82.4-100)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">18/18</td><td colspan="1" rowspan="1">100(82.4-100)</td></tr><tr><td colspan="1" rowspan="1">CoNS (Not S.epidermidis, S.lugdunensis)</td><td colspan="1" rowspan="1">2/2</td><td colspan="1" rowspan="1">100(34.2-100)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">2/2</td><td colspan="1" rowspan="1">100(34.2-100)</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus</td><td colspan="1" rowspan="1">74/78</td><td colspan="1" rowspan="1">94.9(87.5-98.0)</td><td colspan="1" rowspan="1">1/3</td><td colspan="1" rowspan="1">33.3(6.1-79.2)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">75/81</td><td colspan="1" rowspan="1">92.6(84.8-96.6)</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">158/160</td><td colspan="1" rowspan="1">98.8(95.6-99.7)</td><td colspan="1" rowspan="1">121/123</td><td colspan="1" rowspan="1">98.4(94.3-99.6)</td><td colspan="1" rowspan="1">59/59</td><td colspan="1" rowspan="1">100(93.9-100)</td><td colspan="1" rowspan="1">338/342</td><td colspan="1" rowspan="1">98.8(97.0-99.5)</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureussubsp. aureus</td><td colspan="1" rowspan="1">6/6</td><td colspan="1" rowspan="1">100(61.0-100)</td><td colspan="1" rowspan="1">2/2</td><td colspan="1" rowspan="1">100(34.2-100)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">8/8</td><td colspan="1" rowspan="1">100(67.6-100)</td></tr><tr><td colspan="1" rowspan="1">Staphylococcusauricularis</td><td colspan="1" rowspan="1">2/2</td><td colspan="1" rowspan="1">100(34.2-100)</td><td colspan="1" rowspan="1">2/2</td><td colspan="1" rowspan="1">100(34.2-100)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">100(51.0-100)</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus capitis</td><td colspan="1" rowspan="1">14/14</td><td colspan="1" rowspan="1">100(78.5-100)</td><td colspan="1" rowspan="1">7/7</td><td colspan="1" rowspan="1">100(64.6-100)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">21/21</td><td colspan="1" rowspan="1">100(84.5-100)</td></tr><tr><td colspan="1" rowspan="1">Staphylococcuscarnosus subsp.carnosus</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">0/1</td><td colspan="1" rowspan="1">0.0(0.0-79.3)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">0/1</td><td colspan="1" rowspan="1">0.0(0.0-79.3)</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus cohnii</td><td colspan="1" rowspan="1">1/2</td><td colspan="1" rowspan="1">50.0(9.5-90.5)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1/2</td><td colspan="1" rowspan="1">50.0(9.5-90.5)</td></tr><tr><td colspan="1" rowspan="1">Staphylococcusepidermidis</td><td colspan="1" rowspan="1">117/121</td><td colspan="1" rowspan="1">96.7(91.8-98.7)</td><td colspan="1" rowspan="1">37/38</td><td colspan="1" rowspan="1">97.4(86.5-99.5)</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100(20.7-100)</td><td colspan="1" rowspan="1">155/160</td><td colspan="1" rowspan="1">96.9(92.9-98.7)</td></tr><tr><td colspan="1" rowspan="1">Staphylococcushaemolyticus</td><td colspan="1" rowspan="1">6/6</td><td colspan="1" rowspan="1">100(61.0-100)</td><td colspan="1" rowspan="1">2/2</td><td colspan="1" rowspan="1">100(34.2-100)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">8/8</td><td colspan="1" rowspan="1">100(67.6-100)</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus hominis</td><td colspan="1" rowspan="1">24/24</td><td colspan="1" rowspan="1">100(86.2-100)</td><td colspan="1" rowspan="1">13/13</td><td colspan="1" rowspan="1">100(77.2-100)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">37/37</td><td colspan="1" rowspan="1">100(90.6-100)</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus hominissubsp. hominis</td><td colspan="1" rowspan="1">22/22</td><td colspan="1" rowspan="1">100(85.1-100)</td><td colspan="1" rowspan="1">5/5</td><td colspan="1" rowspan="1">100(56.6-100)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">27/27</td><td colspan="1" rowspan="1">100(87.5-100)</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus hominissubsp. novobiosepticus</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100(20.7-100)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100(20.7-100)</td></tr><tr><td colspan="1" rowspan="1">EY</td><td colspan="2" rowspan="1">EY</td><td colspan="2" rowspan="1"></td><td colspan="2" rowspan="1"></td><td colspan="2" rowspan="1"></td></tr><tr><td colspan="1" rowspan="2">TargetSpecies detectedbyComparator Method</td><td colspan="2" rowspan="1">Sensitivity/PPA(Prospective)</td><td colspan="2" rowspan="1">Sensitivity/PPA(Retrospective)</td><td colspan="2" rowspan="1">Sensitivity/PPA(Contrived)</td><td colspan="2" rowspan="1">Sensitivity/PPA(Combined)</td></tr><tr><td colspan="1" rowspan="1">TP/TP+FN</td><td colspan="1" rowspan="1">% (95%CI</td><td colspan="1" rowspan="1">TP/TP+FN</td><td colspan="1" rowspan="1">% (95%CI</td><td colspan="1" rowspan="1">TP/TP+FN</td><td colspan="1" rowspan="1">% (95%CI</td><td colspan="1" rowspan="1">TP/TP+FN</td><td colspan="1" rowspan="1">% (95% Cl)</td></tr><tr><td colspan="1" rowspan="1">Staphylococcuslugdunensis</td><td colspan="1" rowspan="1">2/2</td><td colspan="1" rowspan="1">100(34.2-100)</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">100(51.0-100)</td><td colspan="1" rowspan="1">45/45</td><td colspan="1" rowspan="1">100(92.1-100)</td><td colspan="1" rowspan="1">51/51</td><td colspan="1" rowspan="1">100(93.0-100)</td></tr><tr><td colspan="1" rowspan="1">Staphylococcuspettenkoferi</td><td colspan="1" rowspan="1">2/2</td><td colspan="1" rowspan="1">100(34.2-100)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">2/2</td><td colspan="1" rowspan="1">100(34.2-100)</td></tr><tr><td colspan="1" rowspan="1">Staphylococcussaccharolyticus</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100(20.7-100)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100(20.7-100)</td></tr><tr><td colspan="1" rowspan="1">Staphylococcussaprophyticus</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100(20.7-100)</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100(20.7-100)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">2/2</td><td colspan="1" rowspan="1">100(34.2-100)</td></tr><tr><td colspan="1" rowspan="1">Staphylococcusschleiferi</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100(20.7-100)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100(20.7-100)</td></tr><tr><td colspan="1" rowspan="1">Staphylococcussimulans</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">100(43.9-100)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">100(43.9-100)</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus warneri</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">100(51.0-100)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">100(51.0-100)</td></tr><tr><td colspan="1" rowspan="1">Streptococcus</td><td colspan="1" rowspan="1">103/110</td><td colspan="1" rowspan="1">93.6(87.4-96.9)</td><td colspan="1" rowspan="1">171/17</td><td colspan="1" rowspan="1">98.8(95.9-99.7)</td><td colspan="1" rowspan="1">57/57</td><td colspan="1" rowspan="1">100(93.7-100)</td><td colspan="1" rowspan="1">331/340</td><td colspan="1" rowspan="1">97.4(95.0-98.6)</td></tr><tr><td colspan="1" rowspan="1">Alpha HemolyticStreptococcus</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100(20.7-100)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100(20.7-100)</td></tr><tr><td colspan="1" rowspan="1">Gamma HemolyticStreptococcus</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100(20.7-100)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100(20.7-100)</td></tr><tr><td colspan="1" rowspan="1">Streptococcus</td><td colspan="1" rowspan="1">5/7</td><td colspan="1" rowspan="1">71.4(35.9-91.8)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">5/7</td><td colspan="1" rowspan="1">71.4(35.9-91.8)</td></tr><tr><td colspan="1" rowspan="1">Streptococcus (GroupG</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100(20.7-100)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100(20.7-100)</td></tr><tr><td colspan="1" rowspan="1">Streptococcusagalactiae</td><td colspan="1" rowspan="1">10/11</td><td colspan="1" rowspan="1">90.9(62.3-98.4)</td><td colspan="1" rowspan="1">37/37</td><td colspan="1" rowspan="1">100(90.6-100)</td><td colspan="1" rowspan="1">8/8</td><td colspan="1" rowspan="1">100(67.6-100)</td><td colspan="1" rowspan="1">55/56</td><td colspan="1" rowspan="1">98.2(90.6-99.7)</td></tr><tr><td colspan="1" rowspan="1">Streptococcusanginosus</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100(20.7-100)</td><td colspan="1" rowspan="1">13/13</td><td colspan="1" rowspan="1">100(77.2-100)</td><td colspan="1" rowspan="1">13/13</td><td colspan="1" rowspan="1">100(77.2-100)</td><td colspan="1" rowspan="1">27/27</td><td colspan="1" rowspan="1">100(87.5-100)</td></tr><tr><td colspan="1" rowspan="1">Streptococcusanginosus group</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">100(51.0-100)</td><td colspan="1" rowspan="1">22/22</td><td colspan="1" rowspan="1">100(85.1-100)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">26/26</td><td colspan="1" rowspan="1">100(87.1-100)</td></tr><tr><td colspan="1" rowspan="1">Streptococcus bovis</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">2/2</td><td colspan="1" rowspan="1">100(34.2-100)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">2/2</td><td colspan="1" rowspan="1">100(34.2-100)</td></tr><tr><td colspan="1" rowspan="1">Streptococcus bovisgroup</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100(20.7-100)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100(20.7-100)</td></tr><tr><td colspan="1" rowspan="1">Streptococcusconstellatus</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">6/6</td><td colspan="1" rowspan="1">100(61.0-100)</td><td colspan="1" rowspan="1">6/6</td><td colspan="1" rowspan="1">100(61.0-100)</td></tr><tr><td colspan="1" rowspan="1">Streptococcusconstellatus subsp.constellatus</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">2/2</td><td colspan="1" rowspan="1">100(34.2-100)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">2/2</td><td colspan="1" rowspan="1">100(34.2-100)</td></tr><tr><td colspan="1" rowspan="1">Streptococcusconstellatus subsp.pharyngis</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100(20.7-100)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100(20.7-100)</td></tr><tr><td colspan="1" rowspan="1">Streptococcusdysgalactiae</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">2/2</td><td colspan="1" rowspan="1">100(34.2-100)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">2/2</td><td colspan="1" rowspan="1">100(34.2-100)</td></tr><tr><td colspan="1" rowspan="1">Streptococcusdysgalactiae (Group G)</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">100(51.0-100)</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100(20.7-100)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">5/5</td><td colspan="1" rowspan="1">100(56.6-100)</td></tr><tr><td colspan="1" rowspan="1">Streptococcusgallolyticus</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100(20.7-100)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100(20.7-100)</td></tr><tr><td colspan="1" rowspan="2">TargetSpecies detectedbyComparator Method</td><td colspan="2" rowspan="1">Sensitivity/PPA(Pros pective)</td><td colspan="2" rowspan="1">Sensitivity/PPA(Retros pective)</td><td colspan="2" rowspan="1">Sensitivity/PPA(Contrived)</td><td colspan="2" rowspan="1">Sensitivity/PPA(Combined)</td></tr><tr><td colspan="1" rowspan="1">TP/TP+FN</td><td colspan="1" rowspan="1">% (95%CI</td><td colspan="1" rowspan="1">TP/TP+FN</td><td colspan="1" rowspan="1">% (95%CI</td><td colspan="1" rowspan="1">TP/TP+FN</td><td colspan="1" rowspan="1">% (95%CI</td><td colspan="1" rowspan="1">TP/TP+FN</td><td colspan="1" rowspan="1">% (95% CI)</td></tr><tr><td colspan="1" rowspan="1">Streptococcus gordonii</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100(20.7-100)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100(20.7-100)</td></tr><tr><td colspan="1" rowspan="1">Streptococcusinfantarius</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100(20.7-100)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100(20.7-100)</td></tr><tr><td colspan="1" rowspan="1">Streptococcusintermedius</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">2/2</td><td colspan="1" rowspan="1">100(34.2-100)</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">100(51.0-100)</td><td colspan="1" rowspan="1">6/6</td><td colspan="1" rowspan="1">100(61.0-100)</td></tr><tr><td colspan="1" rowspan="1">Streptococcus mitis</td><td colspan="1" rowspan="1">9/10</td><td colspan="1" rowspan="1">90.0(59.6-98.2)</td><td colspan="1" rowspan="1">14/15</td><td colspan="1" rowspan="1">93.3(70.2-98.8)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">23/25</td><td colspan="1" rowspan="1">92.0(75.0-97.8)</td></tr><tr><td colspan="1" rowspan="1">Streptococcus mitisgroup</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">100(72.2-100)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">100(72.2-100)</td></tr><tr><td colspan="1" rowspan="1">Streptococcus mutans</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100(20.7-100)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100(20.7-100)</td></tr><tr><td colspan="1" rowspan="1">Streptococcus oralis</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">100(43.9-100)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">100(43.9-100)</td></tr><tr><td colspan="1" rowspan="1">Streptococcusparasanguinis</td><td colspan="1" rowspan="1">2/2</td><td colspan="1" rowspan="1">100(34.2-100)</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">100(51.0-100)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">6/6</td><td colspan="1" rowspan="1">100(61.0-100)</td></tr><tr><td colspan="1" rowspan="1">Streptococcuspneumoniae</td><td colspan="1" rowspan="1">28/28</td><td colspan="1" rowspan="1">100(87.9-100)</td><td colspan="1" rowspan="1">41/41</td><td colspan="1" rowspan="1">100(91.4-100)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">69/69</td><td colspan="1" rowspan="1">100(94.7-100)</td></tr><tr><td colspan="1" rowspan="1">Streptococcus pyogenes</td><td colspan="1" rowspan="1">8/8</td><td colspan="1" rowspan="1">100(67.6-100)</td><td colspan="1" rowspan="1">19/20</td><td colspan="1" rowspan="1">95.0(76.4-99.1)</td><td colspan="1" rowspan="1">26/26</td><td colspan="1" rowspan="1">100(87.1-100)</td><td colspan="1" rowspan="1">53/54</td><td colspan="1" rowspan="1">98.1(90.2-99.7)</td></tr><tr><td colspan="1" rowspan="1">Streptococcus salivarius</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">100(51.0-100)</td><td colspan="1" rowspan="1">5/5</td><td colspan="1" rowspan="1">100(56.6-100)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">9/9</td><td colspan="1" rowspan="1">100(70.1-100)</td></tr><tr><td colspan="1" rowspan="1">Streptococcusvestibularis</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100(20.7-100)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100(20.7-100)</td></tr><tr><td colspan="1" rowspan="1">Streptococcus viridansgroup</td><td colspan="1" rowspan="1">14/17</td><td colspan="1" rowspan="1">82.4(59.0-93.8)</td><td colspan="1" rowspan="1">2/2</td><td colspan="1" rowspan="1">100(34.2-100)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">16/19</td><td colspan="1" rowspan="1">84.2(62.4-94.5)</td></tr><tr><td colspan="1" rowspan="1">Streptococcusanginosus group</td><td colspan="1" rowspan="1">4/5</td><td colspan="1" rowspan="1">80.0(37.6-96.4)</td><td colspan="1" rowspan="1">38/40</td><td colspan="1" rowspan="1">95.0(83.5-98.6)</td><td colspan="1" rowspan="1">23/23</td><td colspan="1" rowspan="1">100(85.7-100)</td><td colspan="1" rowspan="1">65/68</td><td colspan="1" rowspan="1">95.6(87.8-98.5)</td></tr><tr><td colspan="1" rowspan="1">Streptococcusanginosus</td><td colspan="1" rowspan="1">0/1</td><td colspan="1" rowspan="1">0.0(0.0-79.3)</td><td colspan="1" rowspan="1">12/13</td><td colspan="1" rowspan="1">92.3(66.7-98.6)</td><td colspan="1" rowspan="1">13/13</td><td colspan="1" rowspan="1">100(77.2-100)</td><td colspan="1" rowspan="1">25/27</td><td colspan="1" rowspan="1">92.6(76.6-97.9)</td></tr><tr><td colspan="1" rowspan="1">Streptococcusanginosus group</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">100(51.0-100)</td><td colspan="1" rowspan="1">21/22</td><td colspan="1" rowspan="1">95.5(78.2-99.2)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">25/26</td><td colspan="1" rowspan="1">96.2(81.1-99.3)</td></tr><tr><td colspan="1" rowspan="1">Streptococcusconstellatus</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">6/6</td><td colspan="1" rowspan="1">100(61.0-100)</td><td colspan="1" rowspan="1">6/6</td><td colspan="1" rowspan="1">100(61.0-100)</td></tr><tr><td colspan="1" rowspan="1">Streptococcusconstellatus sppconstellatus</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">2/2</td><td colspan="1" rowspan="1">100(34.2-100)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2/2</td><td colspan="1" rowspan="1">100(34.2-100)</td></tr><tr><td colspan="1" rowspan="1">Streptococcusconstellatus spppharynges</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100(20.7-100)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100(20.7-100)</td></tr><tr><td colspan="1" rowspan="1">Streptococcusintermedius</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">2/2</td><td colspan="1" rowspan="1">100(34.2-100)</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">100(51.0-100)</td><td colspan="1" rowspan="1">6/6</td><td colspan="1" rowspan="1">100(61.0-100)</td></tr><tr><td colspan="1" rowspan="1">Pan Candida</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">7/9</td><td colspan="1" rowspan="1">77.8(45.3-93.7)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">7/9</td><td colspan="1" rowspan="1">77.8(45.3-93.7)</td></tr><tr><td colspan="1" rowspan="1">Candida albicans</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">100(51.0-100)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">100(51.0-100)</td></tr><tr><td colspan="1" rowspan="1">Candida glabrata</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1/2</td><td colspan="1" rowspan="1">50(9.5-90.5)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1/2</td><td colspan="1" rowspan="1">50(9.5-90.5)</td></tr><tr><td colspan="1" rowspan="2">TargetSpecies detectedbyComparator Method</td><td colspan="2" rowspan="1">Sensitivity/PPA(Pros pective)</td><td colspan="2" rowspan="1">Sensitivity/PPA(Retrospective)</td><td colspan="2" rowspan="1">Sensitivity/PPAContrived)</td><td colspan="2" rowspan="1">Sensitivity/PPA(Combined)</td></tr><tr><td colspan="1" rowspan="1">TP/TP+FN</td><td colspan="1" rowspan="1">% (95%CI</td><td colspan="1" rowspan="1">TP/TP+FN</td><td colspan="1" rowspan="1">% (95%CI</td><td colspan="1" rowspan="1">TP/TP+FN</td><td colspan="1" rowspan="1">% (95%CI</td><td colspan="1" rowspan="1">TP/TP+FN</td><td colspan="1" rowspan="1">% (95% Cl)</td></tr><tr><td colspan="1" rowspan="1">Candidakrusei</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100(20.7-100)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100(20.7-100)</td></tr><tr><td colspan="1" rowspan="1">Candida parapsilosis</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1/2</td><td colspan="1" rowspan="1">50(9.5-90.5)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1/2</td><td colspan="1" rowspan="1">50(9.5-90.5)</td></tr><tr><td colspan="1" rowspan="1">Pan Gram-Negative</td><td colspan="1" rowspan="1">22/23</td><td colspan="1" rowspan="1">95.7(79.0-99.2)</td><td colspan="1" rowspan="1">36/43</td><td colspan="1" rowspan="1">83.7(70.0-91.9)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">58/66</td><td colspan="1" rowspan="1">87.9(77.9-93.7)</td></tr><tr><td colspan="1" rowspan="1">Acinetobacterbaumanni</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">100(43.9-100)</td><td colspan="1" rowspan="1">2/4</td><td colspan="1" rowspan="1">50.0(15.0-85.0)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">5/7</td><td colspan="1" rowspan="1">71.4(35.9-91.8)</td></tr><tr><td colspan="1" rowspan="1">Acinetobacter lwoffii</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100(20.7-100)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100(20.7-100)</td></tr><tr><td colspan="1" rowspan="1">Aeromonas caviae</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100(20.7-100)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100(20.7-100)</td></tr><tr><td colspan="1" rowspan="1">Bacteroides fragilis</td><td colspan="1" rowspan="1">2/2</td><td colspan="1" rowspan="1">100(34.2-100)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">2/2</td><td colspan="1" rowspan="1">100(34.2-100)</td></tr><tr><td colspan="1" rowspan="1">Campylobacter gracilis</td><td colspan="1" rowspan="1">0/1</td><td colspan="1" rowspan="1">0.0(0.0-79.3)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">0/1</td><td colspan="1" rowspan="1">0.0(0.0-79.3)</td></tr><tr><td colspan="1" rowspan="1">Citrobacter braakii</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100(20.7-100)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100(20.7-100)</td></tr><tr><td colspan="1" rowspan="1">Citrobacter freundii</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100(20.7-100)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100(20.7-100)</td></tr><tr><td colspan="1" rowspan="1">Citrobacter koseri</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100(20.7-100)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100(20.7-100)</td></tr><tr><td colspan="1" rowspan="1">Enterobacter aerogenes</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100(20.7-100)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100(20.7-100)</td></tr><tr><td colspan="1" rowspan="1">Enterobacter cloacae</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">100(51.0-100)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">100(51.0-100)</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">100(51.0-100)</td><td colspan="1" rowspan="1">14/14</td><td colspan="1" rowspan="1">100(78.5-100)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">18/18</td><td colspan="1" rowspan="1">100(82.4-100)</td></tr><tr><td colspan="1" rowspan="1">Klebsiella oxytoca</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100(20.7-100)</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">100(43.9-100)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">100(51.0-100)</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">100(51.0-100)</td><td colspan="1" rowspan="1">4/5</td><td colspan="1" rowspan="1">80.0(37.6-96.4)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">8/9</td><td colspan="1" rowspan="1">88.9(56.5-98.0)</td></tr><tr><td colspan="1" rowspan="1">Moraxella(Branhamella)catarrhalis</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100(20.7-100)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100(20.7-100)</td></tr><tr><td colspan="1" rowspan="1">Moraxella catarrhalis</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100(20.7-100)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100(20.7-100)</td></tr><tr><td colspan="1" rowspan="1">Morganella morganii</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">2/2</td><td colspan="1" rowspan="1">100(34.2-100)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">2/2</td><td colspan="1" rowspan="1">100(34.2-100)</td></tr><tr><td colspan="1" rowspan="1">Pediococcuspentosaceus</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100(20.7-100)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100(20.7-100)</td></tr><tr><td colspan="1" rowspan="1">Proteus mirabilis</td><td colspan="1" rowspan="1">5/5</td><td colspan="1" rowspan="1">100(56.6-100)</td><td colspan="1" rowspan="1">4/5</td><td colspan="1" rowspan="1">80.0(37.6-96.4)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">9/10</td><td colspan="1" rowspan="1">90.0(59.6-98.2)</td></tr><tr><td colspan="1" rowspan="1">Proteus vulgaris</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100(20.7-100)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100(20.7-100)</td></tr><tr><td colspan="1" rowspan="1">Providencia stuartii</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100(20.7-100)</td><td colspan="1" rowspan="1">0/1</td><td colspan="1" rowspan="1">0.0(0.0-79.3)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1/2</td><td colspan="1" rowspan="1">50.0(9.5-90.5)</td></tr><tr><td colspan="1" rowspan="2">TargetSpecies detectedbyComparator Method</td><td colspan="2" rowspan="1">Sensitivity/PPA(Prospective)</td><td colspan="2" rowspan="1">Sensitivity/PPA(Retrospective)</td><td colspan="2" rowspan="1">Sensitivity/PPA(Contrived)</td><td colspan="2" rowspan="1">Sensitivity/PPA(Combined)</td></tr><tr><td colspan="1" rowspan="1">TP/TP+FN</td><td colspan="1" rowspan="1">% (95%CI</td><td colspan="1" rowspan="1">TP/TP+FN</td><td colspan="1" rowspan="1">% (95%CI</td><td colspan="1" rowspan="1">TP/TP+FN</td><td colspan="1" rowspan="1">% (95%CI</td><td colspan="1" rowspan="1">TP/TP+FN</td><td colspan="1" rowspan="1">% (95% CI)</td></tr><tr><td colspan="1" rowspan="1">Pseudomonas</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100 (20.7-100)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100(20.7-100)</td></tr><tr><td colspan="1" rowspan="1">Pseudomonasaeruginosa</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100(20.7-100)</td><td colspan="1" rowspan="1">1/2</td><td colspan="1" rowspan="1">50.0(9.5-90.5)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2/3</td><td colspan="1" rowspan="1">66.7(20.8-93.9)</td></tr><tr><td colspan="1" rowspan="1">Serratia marcescens</td><td colspan="1" rowspan="1">2/2</td><td colspan="1" rowspan="1">100(34.2-100)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">2/2</td><td colspan="1" rowspan="1">100(34.2-100)</td></tr><tr><td colspan="1" rowspan="1">Stenotrophomonasmaltophilia</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100(20.7-100)</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100(20.7-100)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2/2</td><td colspan="1" rowspan="1">100(34.2-100)</td></tr><tr><td colspan="1" rowspan="1">Veillonella species</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">0/1</td><td colspan="1" rowspan="1">0.0(0.0-79.3)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">0/1</td><td colspan="1" rowspan="1">0.0(0.0-79.3)</td></tr><tr><td colspan="1" rowspan="1">Non-fermenting gramnegative bacilli</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100(20.7-100)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100(20.7-100)</td></tr></table>

The PPA and NPA of the BCID-GP Panel mecA target stratified by the Staphylococcus species identified by comparator methods for 647 prospective and retrospective samples and 105 contrived samples are shown in Table 77.

Table 77: Clinical Performance of mecA Target by Staphylococcus Species Detected by Comparator Methods   

<table><tr><td colspan="2" rowspan="2">Spe cies detected by ComparatorMethod</td><td colspan="2" rowspan="1">Se ns itivity/PPA</td><td colspan="2" rowspan="1">Specificity/NPA</td></tr><tr><td colspan="1" rowspan="1">TP/TP+FN</td><td colspan="1" rowspan="1">% (95% CI)</td><td colspan="1" rowspan="1">TN/TN+FP</td><td colspan="1" rowspan="1">% (95% CI)</td></tr><tr><td colspan="1" rowspan="4">Coagulase-negativeStaphylococci (CoNS)</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">12/12</td><td colspan="1" rowspan="1">100 (75.8-100)</td><td colspan="1" rowspan="1">6/6</td><td colspan="1" rowspan="1">100 (61.0-100)</td></tr><tr><td colspan="1" rowspan="1">Retrospective</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Contrived</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Combined</td><td colspan="1" rowspan="1">12/12</td><td colspan="1" rowspan="1">100 (75.8-100</td><td colspan="1" rowspan="1">6/6</td><td colspan="1" rowspan="1">100 (61.0-100)</td></tr><tr><td colspan="1" rowspan="4">CoNS (not S.epidermidis/S. lugdunensis)</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100 (20.7-100)</td><td colspan="1" rowspan="1">0/1</td><td colspan="1" rowspan="1">0.0 (0.0-79.3)</td></tr><tr><td colspan="1" rowspan="1">Retrospective</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Contrived</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Combined</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100 (20.7-100)</td><td colspan="1" rowspan="1">0/1</td><td colspan="1" rowspan="1">0.0 (0.0-79.3)</td></tr><tr><td colspan="1" rowspan="4">Staphylococcus</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">49/52</td><td colspan="1" rowspan="1">94.2 (84.4-98.0)</td><td colspan="1" rowspan="1">24/26</td><td colspan="1" rowspan="1">92.3 (75.9-97.9)</td></tr><tr><td colspan="1" rowspan="1">Retrospective</td><td colspan="1" rowspan="1">1/2</td><td colspan="1" rowspan="1">50.0 (9.5-90.5)</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100 (20.7-100)</td></tr><tr><td colspan="1" rowspan="1">Contrived</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Combined</td><td colspan="1" rowspan="1">50/54</td><td colspan="1" rowspan="1">92.6 (82.4-97.1)</td><td colspan="1" rowspan="1">25/27</td><td colspan="1" rowspan="1">92.6 (76.6-97.9)</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">80/83</td><td colspan="1" rowspan="1">96.4 (89.9-98.8)</td><td colspan="1" rowspan="1">74/77</td><td colspan="1" rowspan="1">96.1 (89.2-98.7)</td></tr><tr><td colspan="2" rowspan="2">Species detected by ComparatorMethod</td><td colspan="2" rowspan="1">Se nsitivity/PPA</td><td colspan="2" rowspan="1">Specificity/NPA</td></tr><tr><td colspan="1" rowspan="1">TP/TP+FN</td><td colspan="1" rowspan="1">% (95% CI)</td><td colspan="1" rowspan="1">TN/TN+FP</td><td colspan="1" rowspan="1">% (95% CI)</td></tr><tr><td colspan="1" rowspan="3"></td><td colspan="1" rowspan="1">Retrospective</td><td colspan="1" rowspan="1">105/106</td><td colspan="1" rowspan="1">99.1 (94.8-99.8)</td><td colspan="1" rowspan="1">16/17</td><td colspan="1" rowspan="1">94.1 (73.0-99.0)</td></tr><tr><td colspan="1" rowspan="1">Contrived</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">100 (72.2-100)</td><td colspan="1" rowspan="1">49/49</td><td colspan="1" rowspan="1">100 (92.7-100)</td></tr><tr><td colspan="1" rowspan="1">Combined</td><td colspan="1" rowspan="1">195/199</td><td colspan="1" rowspan="1">98.0 (94.9-99.2)</td><td colspan="1" rowspan="1">139/143</td><td colspan="1" rowspan="1">97.2 (93.0-98.9)</td></tr><tr><td colspan="1" rowspan="4">Staphylococcus aureussubsp. aureus</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">100 (43.9-100)</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">100 (43.9-100)</td></tr><tr><td colspan="1" rowspan="1">Retrospective</td><td colspan="1" rowspan="1">2/2</td><td colspan="1" rowspan="1">100 (34.2-100)</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Contrived</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Combined</td><td colspan="1" rowspan="1">5/5</td><td colspan="1" rowspan="1">100 (56.6-100)</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">100 (43.9-100</td></tr><tr><td colspan="1" rowspan="4">Staphylococcusauricularis</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100 (20.7-100)</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100 (20.7-100)</td></tr><tr><td colspan="1" rowspan="1">Retrospective</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">2/2</td><td colspan="1" rowspan="1">100 (34.2-100)</td></tr><tr><td colspan="1" rowspan="1">Contrived</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Combined</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100 (20.7-100)</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">100 (43.9-100)</td></tr><tr><td colspan="1" rowspan="4">Staphylococcus capitis</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">4/5</td><td colspan="1" rowspan="1">80.0 (37.6-96.4</td><td colspan="1" rowspan="1">5/5</td><td colspan="1" rowspan="1">100 (56.6-100)</td></tr><tr><td colspan="1" rowspan="1">Retrospective</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">100 (51.0-100)</td><td colspan="1" rowspan="1">7/7</td><td colspan="1" rowspan="1">100 (64.6-100)</td></tr><tr><td colspan="1" rowspan="1">Contrived</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Combined</td><td colspan="1" rowspan="1">8/9</td><td colspan="1" rowspan="1">88.9 (56.5-98.0)</td><td colspan="1" rowspan="1">12/12</td><td colspan="1" rowspan="1">100 (75.8-100)</td></tr><tr><td colspan="1" rowspan="4">Staphylococcus carnosussubsp. carnosus</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Retrospective</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100 (20.7-100)</td></tr><tr><td colspan="1" rowspan="1">Contrived</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Combined</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100 (20.7-100)</td></tr><tr><td colspan="1" rowspan="4">Staphylococcus cohnii</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">0/1</td><td colspan="1" rowspan="1">0.0 (0.0-79.3)</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100 (20.7-100)</td></tr><tr><td colspan="1" rowspan="1">Retrospective</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Contrived</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Combined</td><td colspan="1" rowspan="1">0/1</td><td colspan="1" rowspan="1">0.0 (0.0-79.3)</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100 (20.7-100)</td></tr><tr><td colspan="2" rowspan="2">Spe cies detected by ComparatorMethod</td><td colspan="2" rowspan="1">Se nsitivity/PPA</td><td colspan="2" rowspan="1">Specificity/NPA</td></tr><tr><td colspan="1" rowspan="1">TP/TP+FN</td><td colspan="1" rowspan="1">% (95% CI)</td><td colspan="1" rowspan="1">TN/TN+FP</td><td colspan="1" rowspan="1">% (95% CI)</td></tr><tr><td colspan="1" rowspan="4">Staphylococcusepidermidis</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">77/79</td><td colspan="1" rowspan="1">97.5 (91.2-99.3)</td><td colspan="1" rowspan="1">39/42</td><td colspan="1" rowspan="1">92.9 (81.0-97.5)</td></tr><tr><td colspan="1" rowspan="1">Retrospective</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">100 (88.6-100)</td><td colspan="1" rowspan="1">7/8</td><td colspan="1" rowspan="1">87.5 (52.9-97.8)</td></tr><tr><td colspan="1" rowspan="1">Contrived</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100 (20.7-100)</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Combined</td><td colspan="1" rowspan="1">108/110</td><td colspan="1" rowspan="1">98.2 (93.6-99.5)</td><td colspan="1" rowspan="1">46/50</td><td colspan="1" rowspan="1">92.0 (81.2-96.8)</td></tr><tr><td colspan="1" rowspan="4">Staphylococcushaemolyticus</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">100 (51.0-100)</td><td colspan="1" rowspan="1">2/2</td><td colspan="1" rowspan="1">100 (34.2-100)</td></tr><tr><td colspan="1" rowspan="1">Retrospective</td><td colspan="1" rowspan="1">2/2</td><td colspan="1" rowspan="1">100 (34.2-100)</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Contrived</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Combined</td><td colspan="1" rowspan="1">6/6</td><td colspan="1" rowspan="1">100 (61.0-10</td><td colspan="1" rowspan="1">2/2</td><td colspan="1" rowspan="1">100 (34.2-100</td></tr><tr><td colspan="1" rowspan="4">Staphylococcus hominis</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">12/13</td><td colspan="1" rowspan="1">92.3 (66.7-98.6)</td><td colspan="1" rowspan="1">10/11</td><td colspan="1" rowspan="1">90.9 (62.3-98.4</td></tr><tr><td colspan="1" rowspan="1">Retrospective</td><td colspan="1" rowspan="1">12/12</td><td colspan="1" rowspan="1">100 (75.8-100)</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100 (20.7-100)</td></tr><tr><td colspan="1" rowspan="1">Contrived</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Combined</td><td colspan="1" rowspan="1">24/25</td><td colspan="1" rowspan="1">96.0 (80.5-99.3)</td><td colspan="1" rowspan="1">11/12</td><td colspan="1" rowspan="1">91.7 (64.6-98.5</td></tr><tr><td colspan="1" rowspan="4">Staphylococcus hominissubsp. hominis</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">10/11</td><td colspan="1" rowspan="1">90.9 (62.3-98.4</td><td colspan="1" rowspan="1">11/11</td><td colspan="1" rowspan="1">100 (74.1-100)</td></tr><tr><td colspan="1" rowspan="1">Retrospective</td><td colspan="1" rowspan="1">2/2</td><td colspan="1" rowspan="1">100 (34.2-100)</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">100 (43.9-100)</td></tr><tr><td colspan="1" rowspan="1">Contrived</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Combined</td><td colspan="1" rowspan="1">12/13</td><td colspan="1" rowspan="1">92.3 (66.7-98.6</td><td colspan="1" rowspan="1">14/14</td><td colspan="1" rowspan="1">100 (78.5-100</td></tr><tr><td colspan="1" rowspan="4">Staphylococcus hominissubsp. novobiosepticus</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100 (20.7-100)</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Retrospective</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Contrived</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Combined</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100 (20.7-100)</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="4">Staphylococcuslugdunensis</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2/2</td><td colspan="1" rowspan="1">100 (34.2-100)</td></tr><tr><td colspan="1" rowspan="1">Retrospective</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100 (20.7-100)</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">100 (43.9-100)</td></tr><tr><td colspan="1" rowspan="1">Contrived</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">45/45</td><td colspan="1" rowspan="1">100 (92.1-100)</td></tr><tr><td colspan="1" rowspan="1">Combined</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100 (20.7-100)</td><td colspan="1" rowspan="1">50/50</td><td colspan="1" rowspan="1">100 (92.9-100)</td></tr><tr><td colspan="2" rowspan="2">Spe cies detected by ComparatorMethod</td><td colspan="2" rowspan="1">Se ns itivity/PPA</td><td colspan="2" rowspan="1">Specificity/NPA</td></tr><tr><td colspan="1" rowspan="1">TP/TP+FN</td><td colspan="1" rowspan="1">% (95% CI)</td><td colspan="1" rowspan="1">TN/TN+FP</td><td colspan="1" rowspan="1">% (95% CI)</td></tr><tr><td colspan="1" rowspan="4">Staphylococcuspettenkoferi</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1/2</td><td colspan="1" rowspan="1">50.0 (9.5-90.5)</td></tr><tr><td colspan="1" rowspan="1">Retrospective</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Contrived</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Combined</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1/2</td><td colspan="1" rowspan="1">50.0 (9.5-90.5)</td></tr><tr><td colspan="1" rowspan="4">Staphylococcussaccharolyticus</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100 (20.7-100)</td></tr><tr><td colspan="1" rowspan="1">Retrospective</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Contrived</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Combined</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100 (20.7-100)</td></tr><tr><td colspan="1" rowspan="4">Staphylococcussaprophyticus</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100 (20.7-100)</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Retrospective</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100 (20.7-100)</td></tr><tr><td colspan="1" rowspan="1">Contrived</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Combined</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100 (20.7-100)</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100 (20.7-100)</td></tr><tr><td colspan="1" rowspan="4">Staphylococcus schleiferi</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Retrospective</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100 (20.7-100)</td></tr><tr><td colspan="1" rowspan="1">Contrived</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Combined</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100 (20.7-100)</td></tr><tr><td colspan="1" rowspan="4">Staphylococcus simulans</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100 (20.7-100)</td><td colspan="1" rowspan="1">2/2</td><td colspan="1" rowspan="1">100 (34.2-100)</td></tr><tr><td colspan="1" rowspan="1">Retrospective</td><td colspan="1" rowspan="1">:</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Contrived</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Combined</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100 (20.7-100)</td><td colspan="1" rowspan="1">2/2</td><td colspan="1" rowspan="1">100 (34.2-100)</td></tr><tr><td colspan="1" rowspan="4">Staphylococcus warneri</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">100 (51.0-100)</td></tr><tr><td colspan="1" rowspan="1">Retrospective</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Contrived</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Combined</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">100 (51.0-100)</td></tr></table>

A comparison of specific Staphylococcus species and mecA identified by comparator methods versus the ePlex BCID-GP Panel results are shown in Table 78 and Table 79 for prospective and retrospective samples.

Table 78: Distribution of mecA Results in Staphylococcus aureus Prospective/Retrospective Samples   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=4>Comparator Method</td></tr><tr><td rowspan=1 colspan=1>BCID-GP</td><td rowspan=1 colspan=1>Org+/ARG+</td><td rowspan=1 colspan=1>Org+/ARG-</td><td rowspan=1 colspan=1>Org-</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Org+/ARG+</td><td rowspan=1 colspan=1>190</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>194</td></tr><tr><td rowspan=1 colspan=1>Org+/ARG-</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>88</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>94</td></tr><tr><td rowspan=1 colspan=1>Org-</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>1000</td><td rowspan=1 colspan=1>1009</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>194</td><td rowspan=1 colspan=1>97</td><td rowspan=1 colspan=1>1006</td><td rowspan=1 colspan=1>1297</td></tr></table>

$\%$ Agreement $( 9 5 \% \mathrm { C I } )$ ) for $\mathrm { O r g ^ { + / A R G + } }$ : $1 9 0 / 1 9 4 { = } 9 7 . 9$ (94.8-99.2) $\%$ Agreement $( 9 5 \% \mathrm { C I } )$ for Org+/ARG-: $8 8 / 9 7 { = } 9 0 . 7 $ (83.3-95.0) $\%$ Agreement $( 9 5 \% \mathrm { C I } )$ for Org-: 1000/1006=99.4 (98.7-99.7)

Table 79: Distribution of mecA Results in Staphylococcus Species (Excluding Known S. aureus, $s .$ epidermidis, S. lugdunensis) Prospective/Retrospective Samples   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=4>Comparator Method</td></tr><tr><td rowspan=1 colspan=1>BCID-GP</td><td rowspan=1 colspan=1>Org+/ARG+</td><td rowspan=1 colspan=1>Org+/ARG-</td><td rowspan=1 colspan=1>Org-</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Org+/ARG+</td><td rowspan=1 colspan=1>33</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>38</td></tr><tr><td rowspan=1 colspan=1>Org+/ARG-</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>45</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>56</td></tr><tr><td rowspan=1 colspan=1>Org-</td><td rowspan=1 colspan=1>10*</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>1091</td><td rowspan=1 colspan=1>1105</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>46</td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>1103</td><td rowspan=1 colspan=1>1199</td></tr></table>

$\%$ Agreement $( 9 5 \% \mathrm { C I } )$ for Org+/ARG+: $3 3 / 4 6 { = } 7 1 . 7$ (57.5-82.7) $\%$ Agreement $( 9 5 \% \mathrm { C I } )$ for Org+/ARG-: $4 5 / 5 0 { = } 9 0 . 0 $ (78.6-95.7) $\%$ Agreement $( 9 5 \% \mathrm { C I } )$ for Org-: $1 0 9 1 / 1 1 0 3 / 9 8 . 9 $ (98.1-99.4) $^ { * } 1 0$ samples had a Staphylococcus species (not S. aureus, S. epidermidis, or S. lugdunensis) with mecA identified by comparator methods, whereas ePlex BCID-GP detected S. epidermidis with mecA.

A table for mecC is not provided because mecC was only detected in a single species, Staphylococcus aureus. In the 49 contrived samples with Staphylococcus aureus containing mecC, the resulting PPA and NPA were both $100 \%$ .

The PPA and NPA of the BCID-GP Panel vanA target stratified by the Enterococcus species identified by comparator methods for 208 clinical prospective/retrospective samples and 126 contrived samples are shown in Table 80.

Table 80: Clinical Performance of vanA Target by Enterococcus Species Detected by Comparator Methods   

<table><tr><td colspan="2" rowspan="2">Species detected by ComparatorMethod</td><td colspan="2" rowspan="1">Sensitivity/PPA</td><td colspan="2" rowspan="1">Specificity/NPA</td></tr><tr><td colspan="1" rowspan="1">TP/TP+FN</td><td colspan="1" rowspan="1">% (95% CI)</td><td colspan="1" rowspan="1">TN/TN+FP</td><td colspan="1" rowspan="1">% (95% CI)</td></tr><tr><td colspan="1" rowspan="4">Enterococcus</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100 (20.7-100)</td></tr><tr><td colspan="1" rowspan="1">Retrospective</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Contrived</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Combined</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100 (20.7-100)</td></tr><tr><td colspan="1" rowspan="4">Enterococcus avium</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100 (20.7-100)</td></tr><tr><td colspan="1" rowspan="1">Retrospective</td><td colspan="1" rowspan="1">0/1</td><td colspan="1" rowspan="1">0.0 (0.0-79.3)</td><td colspan="1" rowspan="1">2/2</td><td colspan="1" rowspan="1">100 (34.2-100)</td></tr><tr><td colspan="1" rowspan="1">Contrived</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Combined</td><td colspan="1" rowspan="1">0/1</td><td colspan="1" rowspan="1">0.0 (0.0-79.3)</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">100 (43.9-100)</td></tr><tr><td colspan="1" rowspan="4">Enterococcus casseliflavus</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Retrospective</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100 (20.7-100)</td></tr><tr><td colspan="1" rowspan="1">Contrived</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Combined</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100 (20.7-100)</td></tr><tr><td colspan="1" rowspan="4">Enterococcuscasseliflavus /E. gallinarum</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Retrospective</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100 (20.7-100)</td></tr><tr><td colspan="1" rowspan="1">Contrived</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Combined</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100 (20.7-100)</td></tr><tr><td colspan="1" rowspan="4">Enterococcus faecalis</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100 (20.7-100)</td><td colspan="1" rowspan="1">48/48</td><td colspan="1" rowspan="1">100 (92.6-100)</td></tr><tr><td colspan="1" rowspan="1">Retrospective</td><td colspan="1" rowspan="1">11/14</td><td colspan="1" rowspan="1">78.6 (52.4-92.4)</td><td colspan="1" rowspan="1">76/76</td><td colspan="1" rowspan="1">100 (95.2-100)</td></tr><tr><td colspan="1" rowspan="1">Contrived</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">100 (72.2-100)</td><td colspan="1" rowspan="1">42/42</td><td colspan="1" rowspan="1">100 (91.6-100)</td></tr><tr><td colspan="1" rowspan="1">Combined</td><td colspan="1" rowspan="1">22/25</td><td colspan="1" rowspan="1">88.0 (70.0-95.8)</td><td colspan="1" rowspan="1">166/166</td><td colspan="1" rowspan="1">100 (97.7-100)</td></tr><tr><td colspan="1" rowspan="4">Enterococcus faecium</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">7/7</td><td colspan="1" rowspan="1">100 (64.6-100)</td><td colspan="1" rowspan="1">4/5</td><td colspan="1" rowspan="1">80.0 (37.6-96.4</td></tr><tr><td colspan="1" rowspan="1">Retrospective</td><td colspan="1" rowspan="1">44/44</td><td colspan="1" rowspan="1">100 (92.0-100)</td><td colspan="1" rowspan="1">8/9</td><td colspan="1" rowspan="1">38.9 (56.5-98.0)</td></tr><tr><td colspan="1" rowspan="1">Contrived</td><td colspan="1" rowspan="1">50/50</td><td colspan="1" rowspan="1">100 (92.9-100)</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">100 (72.2-100)</td></tr><tr><td colspan="1" rowspan="1">Combined</td><td colspan="1" rowspan="1">101/101</td><td colspan="1" rowspan="1">100 (96.3-100)</td><td colspan="1" rowspan="1">22/24</td><td colspan="1" rowspan="1">1.7 (74.2-97.7)</td></tr><tr><td colspan="1" rowspan="4">Enterococcus flavescens</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Retrospective</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Contrived</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">100 (43.9-100)</td></tr><tr><td colspan="1" rowspan="1">Combined</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">100 (43.9-100)</td></tr><tr><td colspan="1" rowspan="4">Enterococcus gallinarum</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Retrospective</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">2/2</td><td colspan="1" rowspan="1">100 (34.2-100)</td></tr><tr><td colspan="1" rowspan="1">Contrived</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">100 (51.0-100)</td></tr><tr><td colspan="1" rowspan="1">Combined</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">6/6</td><td colspan="1" rowspan="1">100 (61.0-100)</td></tr><tr><td colspan="1" rowspan="4">Enterococcus hirae</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Retrospective</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Contrived</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100 (20.7-100)</td></tr><tr><td colspan="1" rowspan="1">Combined</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100 (20.7-100)</td></tr><tr><td colspan="1" rowspan="3">Enterococcus malodoratus</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Retrospective</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Contrived</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">---</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">100 (43.9-100)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Combined</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">100 (43.9-100)</td></tr><tr><td colspan="1" rowspan="4">Enterococcus raffinosus</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Retrospective</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Contrived</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">2/2</td><td colspan="1" rowspan="1">100 (34.2-100)</td></tr><tr><td colspan="1" rowspan="1">Combined</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">2/2</td><td colspan="1" rowspan="1">100 (34.2-100)</td></tr><tr><td colspan="1" rowspan="4">Enterococcussaccharolyticus</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Retrospective</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Contrived</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100 (20.7-100)</td></tr><tr><td colspan="1" rowspan="1">Combined</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100 (20.7-100)</td></tr></table>

A comparison of Enterococcus faecalis/Enterococcus faecium and vanA identified by comparator methods versus the ePlex BCID-GP Panel results is shown in Table 81 and Table 82 for prospective and retrospective samples.

Table 81: Distribution of vanA Results in Enterococcus faecalis Prospective/Retrospective Samples   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=4>Comparator Method</td></tr><tr><td rowspan=1 colspan=1>BCID-GP</td><td rowspan=1 colspan=1>Org+/ARG+</td><td rowspan=1 colspan=1>Org+/ARG-</td><td rowspan=1 colspan=1>Org-</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Org+/ARG+</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>10</td></tr><tr><td rowspan=1 colspan=1>Org+/ARG-</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>120</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>122</td></tr><tr><td rowspan=1 colspan=1>Org-</td><td rowspan=1 colspan=1>4*</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>1157</td><td rowspan=1 colspan=1>1165</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>124</td><td rowspan=1 colspan=1>1158</td><td rowspan=1 colspan=1>1297</td></tr></table>

$\%$ Agreement $( 9 5 \% \mathrm { C I } )$ for $\mathrm { O r g { + } / A R G { + } }$ : $1 0 / 1 5 { = } 6 6 . 7$ (41.7-84.8) $\%$ Agreement $( 9 5 \% \mathrm { C I } )$ for Org+/ARG-: $1 2 0 / 1 2 4 { = } 9 6 . 8 $ (92.0-98.7) $\%$ Agreement $( 9 5 \% \mathrm { C I } )$ for Org-: 1157/1158=99.9 (99.5-100) $^ { * 2 }$ of the 4 samples had $E .$ . faecium with vanA detected by the ePlex BCID-GP Panel.

Table 82: Distribution of vanA Results in Enterococcus faecium Prospective/Retrospective Samples   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=4>Comparator Method</td></tr><tr><td rowspan=1 colspan=1>BCID-GP</td><td rowspan=1 colspan=1>Org+/ARG+</td><td rowspan=1 colspan=1>Org+/ARG-</td><td rowspan=1 colspan=1>Org-</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Org+/ARG+</td><td rowspan=1 colspan=1>51</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>56</td></tr><tr><td rowspan=1 colspan=1>Org+/ARG-</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>15</td></tr><tr><td rowspan=1 colspan=1>Org-</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>1224</td><td rowspan=1 colspan=1>1226</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>51</td><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>1232</td><td rowspan=1 colspan=1>1297</td></tr></table>

$\%$ Agreement $( 9 5 \% \mathrm { C I } )$ for $\mathrm { O r g { + } / A R G { + } }$ : $5 1 / 5 1 { = } 1 0 0 . 0$ (93.0-100) $\%$ Agreement $( 9 5 \% \mathrm { C I } )$ for $\mathrm { O r g ^ { + / A R G - } }$ : 10/14=71.4 (45.4-88.3) $\%$ Agreement $( 9 5 \% \mathrm { C I } )$ for Org-: 1224/1232=99.4 (98.7-99.7)

A table for vanB is not provided because vanB was only detected in 1 clinical sample and 52 contrived samples comprised of two species, $E .$ . faecalis $( \mathrm { n } { = } 4 3 )$ and $E .$ . faecium $( \mathrm { n } { = } 1 0 )$ , resulting in PPA and NPA of $100 \%$ .

The ePlex BCID-GP Panel identified a total of 103 bacterial co-detections in 1297 clinical samples (prospective/retrospective). In the 711 prospective samples of the clinical study, 38/711 $( 5 . 3 \% )$ had double detections and 1/711 $\left( 0 . 1 \% \right)$ had a triple detection. In the 586 retrospective samples, 56/586 $( 9 . 6 \% )$ had co-detections and 8/586 $( 1 . 4 \% )$ had triple detections. Neither the prospective nor the retrospective arms of the clinical studies contained a sample with more than 3 organisms detected.

In prospective samples, the most common co-detection combination identified by the ePlex BCID-GP Panel was Enterococcus faecalis with Pan Gram-Negative which was detected in 6 samples. Pan Gram-Negative was identified in 17 co-detections, Staphylococcus was identified in 26 co-detections, Enterococcus was identified in 13 codetections, and Streptococcus was identified in 12 co-detections. Results are summarized in Table 83.

Table 83: Co-Detections Identified by the ePlex BCID-GP Panel in Prospective Clinical Samples   

<table><tr><td colspan="4" rowspan="1">Distinct Co-Detection Combinations Detected by the ePlex BCID-GP Panel in Prospective Clinical Samples</td><td colspan="1" rowspan="2">NumberSamples(NumberDiscrepant)</td><td colspan="1" rowspan="2">DiscrepantOrganism(s) /ResistanceMarker(s)*</td></tr><tr><td colspan="1" rowspan="1">Target 1</td><td colspan="1" rowspan="1">Target 2</td><td colspan="1" rowspan="1">Target 3</td><td colspan="1" rowspan="1">ResistanceMarker</td></tr><tr><td colspan="1" rowspan="1">C. acnes</td><td colspan="1" rowspan="1">Staphylococcus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1(1)</td><td colspan="1" rowspan="1">C. acnes (1)</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium</td><td colspan="1" rowspan="1">S. epidermidis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">mecA</td><td colspan="1" rowspan="1">2 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">E. faecalis</td><td colspan="1" rowspan="1">E. faecium</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2 (1)</td><td colspan="1" rowspan="1">E. faecium (1)</td></tr><tr><td colspan="1" rowspan="1">E. faecalis</td><td colspan="1" rowspan="1">Pan GN</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">6 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">E. faecalis</td><td colspan="1" rowspan="1">S. epidermidis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">mecA</td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">E. faecalis</td><td colspan="1" rowspan="1">Staphylococcus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">E. faecalis</td><td colspan="1" rowspan="1">Staphylococcus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">mecA</td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">E. faecium</td><td colspan="1" rowspan="1">Pan GN</td><td colspan="1" rowspan="1">Staphylococcus</td><td colspan="1" rowspan="1">mecA, vanA</td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">E. faecium</td><td colspan="1" rowspan="1">S. epidermidis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">mecA, vanA</td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Lactobacillus</td><td colspan="1" rowspan="1">Streptococcus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (1)</td><td colspan="1" rowspan="1">Lactobacillus (1)</td></tr><tr><td colspan="1" rowspan="1">Listeria</td><td colspan="1" rowspan="1">Staphylococcus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (1)</td><td colspan="1" rowspan="1">Listeria (1)</td></tr><tr><td colspan="1" rowspan="1">Pan GN</td><td colspan="1" rowspan="1">S. anginosusgroup</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Pan GN</td><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Pan GN</td><td colspan="1" rowspan="1">S. epidermidis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">mecA</td><td colspan="1" rowspan="1">2 (2)</td><td colspan="1" rowspan="1">Pan GN (1)</td></tr><tr><td colspan="1" rowspan="1">S. epidermidis (1)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Pan GN</td><td colspan="1" rowspan="1">S. pneumoniae</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (1)</td><td colspan="1" rowspan="1">S. pneumoniae (1)</td></tr><tr><td colspan="1" rowspan="1">Pan GN</td><td colspan="1" rowspan="1">Staphylococcus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="4" rowspan="1">Distinct Co-Detection Combinations Detected by the ePlex BCID-GP Panel in Prospective Clinical Samples</td><td colspan="1" rowspan="1">NumberSamples</td><td colspan="1" rowspan="1">DiscrepantOrganism(s) /</td></tr><tr><td colspan="1" rowspan="1">Pan GN</td><td colspan="1" rowspan="1">Staphylococcus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">mecA</td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Pan GN</td><td colspan="1" rowspan="1">Streptococcus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">S. agalactiae</td><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">S. anginosusgroup</td><td colspan="1" rowspan="1">Staphylococcus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2 (2)</td><td colspan="1" rowspan="1">S. anginosus gp (1),Staphylococcus (1)</td></tr><tr><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1">S. epidermidis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">mecA</td><td colspan="1" rowspan="1">2 (2)</td><td colspan="1" rowspan="1">S. epidermidis (2)</td></tr><tr><td colspan="1" rowspan="1">S. epidermidis</td><td colspan="1" rowspan="1">S. lugdunensis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (1)</td><td colspan="1" rowspan="1">S. epidermidis (1),S. lugdunensis (1)</td></tr><tr><td colspan="1" rowspan="1">S. epidermidis</td><td colspan="1" rowspan="1">S. lugdunensis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">mecA</td><td colspan="1" rowspan="1">1 (1)</td><td colspan="1" rowspan="1">S. lugdunensis (1)</td></tr><tr><td colspan="1" rowspan="1">S. epidermidis</td><td colspan="1" rowspan="1">Streptococcus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2 (1)</td><td colspan="1" rowspan="1">S. epidermidis (1)</td></tr></table>

\* A discrepant result is one that was detected by the ePlex BCID-GP Panel but not by the comparator method(s). 7/16 discrepant organisms were detected using PCR/sequencing as shown: In 1/1 false positive $E$ . faecium samples, $E .$ . faecium was detected. In 1/1 false positive Lactobacillus samples, Lactobacillus was detected. In 2/5 false positive $S .$ . epidermidis samples, S. epidermidis was detected. In 2/2 false positive S. lugdunensis samples, S. lugdunensis was detected. In 1/1 false positive Streptococcus samples, Streptococcus was detected.

In retrospective samples, the most common co-detection combination identified by the ePlex BCID-GP Panel was Enterococcus faecalis with Pan Gram-Negative which was detected in 9 samples, 2 of which also had vanA detected. Pan Gram-Negative was identified in 33 co-detections, Staphylococcus was identified in 22 co-detections, Enterococcus was identified in 34 co-detections, and Streptococcus was identified in 24 co-detections. Results are summarized in Table 84.

Table 84: Co-Detections Identified by the ePlex BCID-GP Panel in Retrospective Clinical Samples   

<table><tr><td colspan="4" rowspan="1">Distinct Co-Detection Combinations Detected by the ePlexBCID-GP Panel in Retrospective Clinical Samples</td><td colspan="1" rowspan="2">NumberSamples(NumberDiscrepant)</td><td colspan="1" rowspan="2">DiscrepantOrganism(s) /ResistanceMarker(s)*</td></tr><tr><td colspan="1" rowspan="1">Target 1</td><td colspan="1" rowspan="1">Target 2</td><td colspan="1" rowspan="1">Target 3</td><td colspan="1" rowspan="1">ResistanceMarker</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium</td><td colspan="1" rowspan="1">S. epidermidis</td><td colspan="1" rowspan="1">S. lugdunensis</td><td colspan="1" rowspan="1">mecA</td><td colspan="1" rowspan="1">1 (1)</td><td colspan="1" rowspan="1">Corynebacterium(1)</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium</td><td colspan="1" rowspan="1">Staphylococcus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Corynebacterium</td><td colspan="1" rowspan="1">Staphylococcus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">mecA</td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">E. faecalis</td><td colspan="1" rowspan="1">E. faecium</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">4 (3)</td><td colspan="1" rowspan="1">E. faecium (3)</td></tr><tr><td colspan="1" rowspan="1">E. faecalis</td><td colspan="1" rowspan="1">E. faecium</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">vanA</td><td colspan="1" rowspan="1">3 (1)</td><td colspan="1" rowspan="1">E. faecium (1)</td></tr><tr><td colspan="1" rowspan="1">E. faecalis</td><td colspan="1" rowspan="1">Pan Candida</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">E. faecalis</td><td colspan="1" rowspan="1">Pan GN</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">6 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">E. faecalis</td><td colspan="1" rowspan="1">Pan GN</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">vanA</td><td colspan="1" rowspan="1">2 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">E. faecalis</td><td colspan="1" rowspan="1">Pan GN</td><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="4" rowspan="1">Distinct Co-Detection Combinations Detected by the ePlexBCID-GP Panel in Retrospective Clinical Samples</td><td colspan="1" rowspan="1">NumberSamples(Number</td><td colspan="1" rowspan="1">Discrepantranism() /Resistance</td></tr><tr><td colspan="1" rowspan="1">E. faecalis</td><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">mecA</td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">E. faecalis</td><td colspan="1" rowspan="1">Staphylococcus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">vanA</td><td colspan="1" rowspan="1">1 (1)</td><td colspan="1" rowspan="1">Staphylococcus (1)</td></tr><tr><td colspan="1" rowspan="1">E. faecium</td><td colspan="1" rowspan="1">Lactobacillus</td><td colspan="1" rowspan="1">Pan GN</td><td colspan="1" rowspan="1">vanA</td><td colspan="1" rowspan="1">1(1)</td><td colspan="1" rowspan="1">Lactobacillus (1)</td></tr><tr><td colspan="1" rowspan="1">E. faecium</td><td colspan="1" rowspan="1">Pan Candida</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">vanA</td><td colspan="1" rowspan="1">1(1)</td><td colspan="1" rowspan="1">E. faecium (1)</td></tr><tr><td colspan="1" rowspan="1">E. faecium</td><td colspan="1" rowspan="1">Pan Candida</td><td colspan="1" rowspan="1">S. epidermidis</td><td colspan="1" rowspan="1">mecA,vanA</td><td colspan="1" rowspan="1">1 (1)</td><td colspan="1" rowspan="1">S. epidermidis (1)</td></tr><tr><td colspan="1" rowspan="1">E. faecium</td><td colspan="1" rowspan="1">Pan GN</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">3 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">E. faecium</td><td colspan="1" rowspan="1">Pan GN</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">vanA</td><td colspan="1" rowspan="1">5(0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">E. faecium</td><td colspan="1" rowspan="1">Pan GN</td><td colspan="1" rowspan="1">StaphylococcuS</td><td colspan="1" rowspan="1">mecA,vanA</td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">E. faecium</td><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">mecA,vanA</td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">E. faecium</td><td colspan="1" rowspan="1">Streptococcus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">vanA</td><td colspan="1" rowspan="1">1 (1)</td><td colspan="1" rowspan="1">Streptococcus (1)</td></tr><tr><td colspan="1" rowspan="1">Enterococcus</td><td colspan="1" rowspan="1">S. anginosusgroup</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Lactobacillus</td><td colspan="1" rowspan="1">S. anginosusgroup</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Micrococcus</td><td colspan="1" rowspan="1">S. pyogenes</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (1)</td><td colspan="1" rowspan="1">Micrococcus (1)</td></tr><tr><td colspan="1" rowspan="1">Pan Candida</td><td colspan="1" rowspan="1">S. epidermidis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">mecA</td><td colspan="1" rowspan="1">2 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Pan Candida</td><td colspan="1" rowspan="1">S. pneumoniae</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Pan GN</td><td colspan="1" rowspan="1">S. agalactiae</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2 (1)</td><td colspan="1" rowspan="1">Pan GN (1)</td></tr><tr><td colspan="1" rowspan="1">Pan GN</td><td colspan="1" rowspan="1">S. anginosusgroup</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">4 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Pan GN</td><td colspan="1" rowspan="1">S. anginosusgroup</td><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (1)</td><td colspan="1" rowspan="1">Pan GN (1)</td></tr><tr><td colspan="1" rowspan="1">Pan GN</td><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Pan GN</td><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1">S. epidermidis</td><td colspan="1" rowspan="1">mecA</td><td colspan="1" rowspan="1">1 (1)</td><td colspan="1" rowspan="1">S. epidermidis (1)</td></tr><tr><td colspan="1" rowspan="1">Pan GN</td><td colspan="1" rowspan="1">S. pneumoniae</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Pan GN</td><td colspan="1" rowspan="1">Streptococcus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">3 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">S. agalactiae</td><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">S. agalactiae</td><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">mecA</td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">S. agalactiae</td><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1">S. epidermidis</td><td colspan="1" rowspan="1">mecA</td><td colspan="1" rowspan="1">1 (1)</td><td colspan="1" rowspan="1">S. epidermidis (1)</td></tr><tr><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1">S. epidermidis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">mecA</td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="4" rowspan="1">Distinct Co-Detection Combinations Detected by the ePlexBCID-GP Panel in Retrospective Clinical Samples</td><td colspan="1" rowspan="1">NumberSamples(Number</td><td colspan="1" rowspan="1">DiscrepantOrganism(s) /Resistance</td></tr><tr><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1">Streptococcus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">mecA</td><td colspan="1" rowspan="1">2 (1)</td><td colspan="1" rowspan="1">Streptococcus (1)</td></tr><tr><td colspan="1" rowspan="1">S. epidermidis</td><td colspan="1" rowspan="1">Streptococcus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">mecA</td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr></table>

\* A discrepant result is one that was detected by the ePlex BCID-GP Panel but not by the comparator method(s). 6/18 discrepant organisms were detected using PCR/sequencing as shown: In 1/1 false positive Corynebacterium samples, Corynebacterium was detected. $\operatorname { I n } 2 / 5$ false positive $E$ . faecium samples, $E$ . faecium was detected. $\operatorname { I n } 1 / 1$ false positive Staphylococcus samples, Staphylococcus was detected. $\mathrm { I n } 2 / 2$ false positive Streptococcus samples, Streptococcus was detected.

Additional co-detection combinations identified by comparator method(s) are summarized in Table 85 and Table 86 for prospective and retrospective samples, respectively.

Table 85: Additional Co-Detections Identified by Comparator Method(s) in Prospective Clinical Samples by Organism   

<table><tr><td colspan="6" rowspan="1">Distinct Co-Detection Combinations Detected by theComparator Methods in Prospective Clinical Samples</td><td colspan="1" rowspan="2">NumberSamples(NumberDiscrepant</td><td colspan="1" rowspan="2">DiscrepantOrganism(s)/ResistanceMarkers(s)*</td></tr><tr><td colspan="1" rowspan="1">Organism 1</td><td colspan="2" rowspan="1">Organism 2</td><td colspan="1" rowspan="1">Organism 3</td><td colspan="1" rowspan="1">Organism 4</td><td colspan="1" rowspan="1">ResistanceMarker</td></tr><tr><td colspan="2" rowspan="1">A. baumannii</td><td colspan="1" rowspan="1">E. faecium</td><td colspan="1" rowspan="1">Staphylococcus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">mecA,vanA</td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="2" rowspan="1">A. baumannii</td><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="2" rowspan="1">A. baumannii</td><td colspan="1" rowspan="1">Staphylococcus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">mecA</td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="2" rowspan="1">Acinetobacterlwoffii</td><td colspan="1" rowspan="1">Staphylococcushominis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">mecA</td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="2" rowspan="1">AerococcusviridansA</td><td colspan="1" rowspan="1">K. oxytoca</td><td colspan="1" rowspan="1">S. epidermidis</td><td colspan="1" rowspan="1">Staphylococcuscohni</td><td colspan="1" rowspan="1">mecA</td><td colspan="1" rowspan="1">1 (1)</td><td colspan="1" rowspan="1">S. cohnii (1),S. epidermidis(1), mecA (1)</td></tr><tr><td colspan="2" rowspan="1">Aerococcusviridans A</td><td colspan="1" rowspan="1">Staphylococcushominis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="2" rowspan="1">B. fragilis</td><td colspan="1" rowspan="1">ClostridiumspeciesA</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="2" rowspan="1">B. fragilis</td><td colspan="1" rowspan="1">S. anginosus gp</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="2" rowspan="1">C. acnes</td><td colspan="1" rowspan="1">S. epidermidis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (1)</td><td colspan="1" rowspan="1">S. epidermidis(1)</td></tr><tr><td colspan="2" rowspan="1">C. acnes</td><td colspan="1" rowspan="1">S. lugdunensis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (1)</td><td colspan="1" rowspan="1">C. acnes (1)</td></tr><tr><td colspan="2" rowspan="1">Citrobacter freundi</td><td colspan="1" rowspan="1">K. pneumoniae</td><td colspan="1" rowspan="1">Staphylococcusominis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">mecA</td><td colspan="1" rowspan="1">1 (1)</td><td colspan="1" rowspan="1">mecA (1)</td></tr><tr><td colspan="2" rowspan="1">Citrobacter koseri</td><td colspan="1" rowspan="1">E. faecalis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="2" rowspan="1">Corynebacterium</td><td colspan="1" rowspan="1">S. epidermidis</td><td colspan="1" rowspan="1">Streptococcus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">mecA</td><td colspan="1" rowspan="1">1 (1)</td><td colspan="1" rowspan="1">Streptococcus(1)</td></tr><tr><td colspan="2" rowspan="1">Corynebacterium</td><td colspan="1" rowspan="1">Streptococcus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (1)</td><td colspan="1" rowspan="1">Corynebacterium (1)</td></tr><tr><td colspan="2" rowspan="1">E. coli</td><td colspan="1" rowspan="1">E. faecalis</td><td colspan="1" rowspan="1">P. mirabilis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="2" rowspan="1">E. coli</td><td colspan="1" rowspan="1">LactococcuslactisA</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="6" rowspan="1">Distinct Co-Detection Combinations Detected by theComparator Methods in Prospective Clinical Samples</td><td colspan="1" rowspan="2">NumberSamples(NumberDiscrepant)</td><td colspan="1" rowspan="2">DiscrepantOrganism(s)/ResistanceMarkers(s)*</td></tr><tr><td colspan="1" rowspan="1">Organism 1</td><td colspan="2" rowspan="1">Organism 2</td><td colspan="1" rowspan="1">Organism 3</td><td colspan="1" rowspan="1">Organism 4</td><td colspan="1" rowspan="1">ResistanceMarker</td></tr><tr><td colspan="2" rowspan="1">E. coli</td><td colspan="1" rowspan="1">P. mirabilis</td><td colspan="1" rowspan="1">Providenciastuartii</td><td colspan="1" rowspan="1">S. anginosus gp</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="2" rowspan="1">E. faecalis</td><td colspan="1" rowspan="1">E. faecium</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (1)</td><td colspan="1" rowspan="1">E. faecium (1)</td></tr><tr><td colspan="2" rowspan="1">E. faecalis</td><td colspan="1" rowspan="1">K. pneumoniae</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="2" rowspan="1">E. faecalis</td><td colspan="1" rowspan="1">P. mirabilis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="2" rowspan="1">E. faecalis</td><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">mecA</td><td colspan="1" rowspan="1">1(1)</td><td colspan="1" rowspan="1">S. aureus (1),mecA (1)</td></tr><tr><td colspan="2" rowspan="1">E. faecalis</td><td colspan="1" rowspan="1">S. marcescens</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="2" rowspan="1">E. faecalis</td><td colspan="1" rowspan="1">Staphylococcus (CONS)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">mecA</td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="2" rowspan="1">E. faecium</td><td colspan="1" rowspan="1">S. epidermidis</td><td colspan="1" rowspan="1">Staphylococcushaemolyticus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">mecA,vanA</td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="2" rowspan="1">K. pneumoniae</td><td colspan="1" rowspan="1">Staphylococcushaemolyticus</td><td colspan="1" rowspan="1">non-fermentingG bacilli</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="2" rowspan="1">P. aeruginosa</td><td colspan="1" rowspan="1">P. mirabilis</td><td colspan="1" rowspan="1">Streptococcus -viridans group</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="2" rowspan="1">P. mirabilis</td><td colspan="1" rowspan="1">Staphylococcus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">mecA</td><td colspan="1" rowspan="1">1 (1)</td><td colspan="1" rowspan="1">Staphylococcus (1), mecA(1)</td></tr><tr><td colspan="2" rowspan="1">PeptostreptococcusspeeciesA</td><td colspan="1" rowspan="1">Staphylococcus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="2" rowspan="1">Rothia(stomatococcus)mucilaginosusA</td><td colspan="1" rowspan="1">S. epidermidis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="2" rowspan="1">RothiamucilaginosaA</td><td colspan="1" rowspan="1">Streptococcus -viridans group</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="2" rowspan="1">S. agalactiae</td><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1">Staphylococcus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">mecA</td><td colspan="1" rowspan="1">1 (1)</td><td colspan="1" rowspan="1">S. aureus (1),Staphylococcus (1), mecA(1)</td></tr><tr><td colspan="2" rowspan="1">S. anginosus</td><td colspan="1" rowspan="1">Streptococcusmitis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (1)</td><td colspan="1" rowspan="1">S. anginosus(1)</td></tr><tr><td colspan="2" rowspan="1">S. epidermidis</td><td colspan="1" rowspan="1">Staphylococcuscapitis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">mecA</td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="2" rowspan="1">S. epidermidis</td><td colspan="1" rowspan="1">Staphylococcushominis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="2" rowspan="1">S. epidermidis</td><td colspan="1" rowspan="1">Staphylococcushominis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">mecA</td><td colspan="1" rowspan="1">4(1)</td><td colspan="1" rowspan="1">S. epidermidis(1)</td></tr><tr><td colspan="2" rowspan="1">S. epidermidis</td><td colspan="1" rowspan="1">Staphylococcushominis</td><td colspan="1" rowspan="1">Staphylococcuswarneri</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="2" rowspan="1">S. epidermidis</td><td colspan="1" rowspan="1">Streptococcus -viridans group</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (1)</td><td colspan="1" rowspan="1">S. epidermidis(1)</td></tr><tr><td colspan="6" rowspan="1">Distinct Co-Detection Combinations Detected by theComparator Methods in Prospective Clinical Samples</td><td colspan="1" rowspan="2">NumberSamples(NumberDiscrepant)</td><td colspan="1" rowspan="2">DiscrepantOrganism(s)ResistanceMarkers(s)*</td></tr><tr><td colspan="1" rowspan="1">Organism 1</td><td colspan="2" rowspan="1">Organism 2</td><td colspan="1" rowspan="1">Organism 3</td><td colspan="1" rowspan="1">Organism 4</td><td colspan="1" rowspan="1">ResistanceMarker</td></tr><tr><td colspan="2" rowspan="1">S. epidermidis</td><td colspan="1" rowspan="1">Streptococcusparasanguinis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="2" rowspan="1">S. maltophilia</td><td colspan="1" rowspan="1">Streptococcus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (1)</td><td colspan="1" rowspan="1">Streptococcus(1)</td></tr><tr><td colspan="2" rowspan="1">S. marcescens</td><td colspan="1" rowspan="1">Streptococcusmitis group</td><td colspan="1" rowspan="1">Streptococcussalivarius</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="2" rowspan="1">Staphylococcuscchni</td><td colspan="1" rowspan="1">Streptococcus -viridans group</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (1)</td><td colspan="1" rowspan="1">S. viridansgroup (1)</td></tr><tr><td colspan="2" rowspan="1">Staphylococcushominis</td><td colspan="1" rowspan="1">Staphylococcuspettenkoferi</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="2" rowspan="1">Staphylococcushominis</td><td colspan="1" rowspan="1">Streptococcusmitis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">mecA</td><td colspan="1" rowspan="1">1 (1)</td><td colspan="1" rowspan="1">mecA (1)</td></tr></table>

\* A discrepant result is one that was detected by the comparator method(s) but not by the ePlex BCID-GP Panel (excludes organisms not targeted by the ePlex BCID-GP Panel). 16 discrepant organisms were investigated using PCR/sequencing; 1 discrepant organism was not detected: In 1/1 false negative S. anginosus group sample, PCR/Sequencing instead detected Streptococcus dysgalactiae. A. Off-panel organisms not targeted by the ePlex BCID-GP Panel.

Table 86: Additional Co-Detections Identified by Comparator Method(s) in Retrospective Clinical Samples by Organism   

<table><tr><td colspan="5" rowspan="1">Distinct Co-Detection Combinations Detected by theComparator Methods in Retrospective Clinical Samples</td><td colspan="1" rowspan="2">NumberSamples(NumberDiscrepant)</td><td colspan="1" rowspan="2">DiscrepantOrganism(s)/ResistanceMarkers(s)*</td></tr><tr><td colspan="1" rowspan="1">Organism 1</td><td colspan="1" rowspan="1">Organism 2</td><td colspan="1" rowspan="1">Organism 3</td><td colspan="1" rowspan="1">Organism 4</td><td colspan="1" rowspan="1">ResistanceMarker</td></tr><tr><td colspan="1" rowspan="1">A. baumannii</td><td colspan="1" rowspan="1">E. faecalis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">vanA</td><td colspan="1" rowspan="1">2 (2)</td><td colspan="1" rowspan="1">A. baumannii(2)</td></tr><tr><td colspan="1" rowspan="1">A. baumannii</td><td colspan="1" rowspan="1">E. faecalis</td><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">mecA</td><td colspan="1" rowspan="1">1(1)</td><td colspan="1" rowspan="1">mecA (1)</td></tr><tr><td colspan="1" rowspan="1">A. baumannii</td><td colspan="1" rowspan="1">E. faecium</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">vanA</td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Aerococcussanguinicola A</td><td colspan="1" rowspan="1">Corynebacterium</td><td colspan="1" rowspan="1">Staphylococcussaprophyticus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Aeromonascaviae</td><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">Enterococcuscasseliflavus</td><td colspan="1" rowspan="1">K. oxytoca</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1(1)</td><td colspan="1" rowspan="1">E. casseliflavus(1)</td></tr><tr><td colspan="1" rowspan="1">C. acnes</td><td colspan="1" rowspan="1">Enterococcusavium</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">vanA</td><td colspan="1" rowspan="1">1 (1)</td><td colspan="1" rowspan="1">E. avium (1),vanA (1)</td></tr><tr><td colspan="1" rowspan="1">C. albicans</td><td colspan="1" rowspan="1">E. faecalis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">vanA</td><td colspan="1" rowspan="1">1(1)</td><td colspan="1" rowspan="1">E. faecalis (1)</td></tr><tr><td colspan="1" rowspan="1">C. albicans</td><td colspan="1" rowspan="1">E. faecium</td><td colspan="1" rowspan="1">Staphylococcushominis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">mecA,vanA</td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">C. albicans</td><td colspan="1" rowspan="1">S. epidermidis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">mecA</td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">C. glabrata</td><td colspan="1" rowspan="1">Lactobacillusrhamnosus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (1)</td><td colspan="1" rowspan="1">C. glabrata (1)</td></tr><tr><td colspan="1" rowspan="1">C. glabrata</td><td colspan="1" rowspan="1">S. pneumoniae</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1(0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">C. krusei</td><td colspan="1" rowspan="1">S. epidermidis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">mecA</td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">C. parapsilosis</td><td colspan="1" rowspan="1">E. faecalis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="5" rowspan="1">Distinct Co-Detection Combinations Detected by theComparator Methods in Retrospective Clinical Samples</td><td colspan="1" rowspan="2">NumberSaamples(NumberDiscrepant)</td><td colspan="1" rowspan="2">DiscrepantOrganism(s)/ResistanceMarkers(s)*</td></tr><tr><td colspan="1" rowspan="1">Organism 1</td><td colspan="1" rowspan="1">Organism 2</td><td colspan="1" rowspan="1">Organism 3</td><td colspan="1" rowspan="1">Organism 4</td><td colspan="1" rowspan="1">ResistanceMarker</td></tr><tr><td colspan="1" rowspan="1">C. parapsilosis</td><td colspan="1" rowspan="1">E. faecalis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">vanA</td><td colspan="1" rowspan="1">1 (1)</td><td colspan="1" rowspan="1">C. parapsilosis(1)</td></tr><tr><td colspan="1" rowspan="1">Citrobacterbraki</td><td colspan="1" rowspan="1">Streptococcusoralis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">E. cloacae</td><td colspan="1" rowspan="1">E. faecalis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">E. cloacae</td><td colspan="1" rowspan="1">E. faecium</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">vanA</td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">E. cloacae</td><td colspan="1" rowspan="1">E. faecium</td><td colspan="1" rowspan="1">Staphylococcushominis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">mecA,vanA</td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">E. cloacae</td><td colspan="1" rowspan="1">S. anginosus gp</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">E. faecalis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">3 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">E. faecalis</td><td colspan="1" rowspan="1">K. pneumoniae</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">E. faecalis</td><td colspan="1" rowspan="1">P. mirabilis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">E. faecium</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">K. oxytoca</td><td colspan="1" rowspan="1">Streptococcusinfantarius</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">S. agalactiae</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">E. col</td><td colspan="1" rowspan="1">S. anginosus gp</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">mecA</td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">S. pneumoniae</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">Streptococcusbovis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">E. faecalis</td><td colspan="1" rowspan="1">K. pneumoniae</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">vanA</td><td colspan="1" rowspan="1">1 (1)</td><td colspan="1" rowspan="1">E. faecalis (1),K. pneumoniae(1)</td></tr><tr><td colspan="1" rowspan="1">E. faecalis</td><td colspan="1" rowspan="1">M. morganii</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">vanA</td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">E. faecalis</td><td colspan="1" rowspan="1">M. morganii</td><td colspan="1" rowspan="1">Proteusvulgaris</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">vanA</td><td colspan="1" rowspan="1">1 (1)</td><td colspan="1" rowspan="1">E. faecalis (1),vanA (1)</td></tr><tr><td colspan="1" rowspan="1">E. faecalis</td><td colspan="1" rowspan="1">P. aeruginosa</td><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">mecA</td><td colspan="1" rowspan="1">1 (1)</td><td colspan="1" rowspan="1">E. faecalis (1),P. aeruginosa(1)</td></tr><tr><td colspan="1" rowspan="1">E. faecalis</td><td colspan="1" rowspan="1">P. mirabilis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2 (2)</td><td colspan="1" rowspan="1">E. faecalis (1),P. mirabilis (1)</td></tr><tr><td colspan="1" rowspan="1">E. faecalis</td><td colspan="1" rowspan="1">P. mirabilis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">vanA</td><td colspan="1" rowspan="1">1 (1)</td><td colspan="1" rowspan="1">E. faecalis (1),vanA (1)</td></tr><tr><td colspan="1" rowspan="1">E. faecalis</td><td colspan="1" rowspan="1">Providenciastuartii</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (1)</td><td colspan="1" rowspan="1">P. stuartii (1)</td></tr><tr><td colspan="1" rowspan="1">E. faecalis</td><td colspan="1" rowspan="1">S. maltophilia</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">vanA</td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">E. faecium</td><td colspan="1" rowspan="1">K. pneumoniae</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">E. faecium</td><td colspan="1" rowspan="1">Moraxella(Branhamella)catarrhalis</td><td colspan="1" rowspan="1">Pediococcuspentosaceus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">vanA</td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="5" rowspan="1">Distinct Co-Detection Combinations Detected by theComparator Methods in Retrospective Clinical Samples</td><td colspan="1" rowspan="2">NumberSamples(NumberDiscrepant)</td><td colspan="1" rowspan="2">Discrepantranism(s)/ResistanceMarkers(s)*</td></tr><tr><td colspan="1" rowspan="1">Organism 1</td><td colspan="1" rowspan="1">Organism 2</td><td colspan="1" rowspan="1">Organism 3</td><td colspan="1" rowspan="1">Organism 4</td><td colspan="1" rowspan="1">ResistanceMarker</td></tr><tr><td colspan="1" rowspan="1">E. faecium</td><td colspan="1" rowspan="1">P. aeruginosa</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">vanA</td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">E. faecium</td><td colspan="1" rowspan="1">P. mirabilis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">vanA</td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">E. faecium</td><td colspan="1" rowspan="1">Pseudomonas</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">vanA</td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">E. faecium</td><td colspan="1" rowspan="1">S. epidermidis</td><td colspan="1" rowspan="1">Staphylococcushominis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">mecA</td><td colspan="1" rowspan="1">1 (1)</td><td colspan="1" rowspan="1">E. faecium (1)</td></tr><tr><td colspan="1" rowspan="1">Enterobacteraerogenes</td><td colspan="1" rowspan="1">S. anginosus gp</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Enterococcusavium</td><td colspan="1" rowspan="1">S. anginosus gp</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">K. oxytoca</td><td colspan="1" rowspan="1">S. anginosus gp</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">K. pneumoniae</td><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2 (1)</td><td colspan="1" rowspan="1">S. aureus (1)</td></tr><tr><td colspan="1" rowspan="1">L.monocytogenes</td><td colspan="1" rowspan="1">Staphylococcus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">mecA</td><td colspan="1" rowspan="1">1 (1)</td><td colspan="1" rowspan="1">Staphylococcus(1), mecA (1)</td></tr><tr><td colspan="1" rowspan="1">Lactobacilluscasei</td><td colspan="1" rowspan="1">Veillonellaspecies</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (1)</td><td colspan="1" rowspan="1">Veillonellaspecies (1)</td></tr><tr><td colspan="1" rowspan="1">Lactobacillusrhamnosus</td><td colspan="1" rowspan="1">Pediococcusacidilactici</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Lactobacillusrhamnosus</td><td colspan="1" rowspan="1">S. anginosus gp</td><td colspan="1" rowspan="1">Staphylococcus</td><td colspan="1" rowspan="1">Streptococcusviridans group</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (1)</td><td colspan="1" rowspan="1">Staphylococcus(1)</td></tr><tr><td colspan="1" rowspan="1">Micrococcus</td><td colspan="1" rowspan="1">PseudoclavibacterA</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Moraxellacatarrhalis</td><td colspan="1" rowspan="1">S. pneumoniae</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">S. agalactiae</td><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1(1)</td><td colspan="1" rowspan="1">S. aureus (1)</td></tr><tr><td colspan="1" rowspan="1">S. agalactiae</td><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1">Streptococcusviridans group</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1">S. epidermidis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (1)</td><td colspan="1" rowspan="1">S. aureus (1)</td></tr><tr><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1">S. pyogenes</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">mecA</td><td colspan="1" rowspan="1">1(1)</td><td colspan="1" rowspan="1">S. pyogenes (1)</td></tr><tr><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1">Staphylococcuscapitis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1">Streptococcusmitis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">mecA</td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">S. epidermidis</td><td colspan="1" rowspan="1">Staphylococcushhominis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">S. epidermidis</td><td colspan="1" rowspan="1">Staphylococcushominis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">mecA</td><td colspan="1" rowspan="1">3 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">S. epidermidis</td><td colspan="1" rowspan="1">Staphylococcushhominis</td><td colspan="1" rowspan="1">Streptococcusparasanguinis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">mecA</td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Staphylococcuscapitis</td><td colspan="1" rowspan="1">Staphylococcushominis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">mecA</td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr></table>

$^ * \mathrm { \bf ~ A }$ discrepant result is one that was detected by the comparator method(s) but not by the ePlex BCID-GP Panel (excludes organisms not targeted by the ePlex BCID-GP Panel). 24 discrepant organisms were investigated using PCR/sequencing; 2 discrepant organisms were not detected: In 2/6 false negative $E$ . faecalis samples, PCR/Sequencing instead detected Enterococcus faecium. A. Off-panel organisms not targeted by the ePlex BCID-GP Panel.

External Control testing in the Clinical Study:

External controls used in the clinical study consisted of three positive controls (A, B, C) and 1 negative control. The positive controls together represent all the targets on the ePlex BCID-GP Panel. On each day of testing with the ePlex BCID-GP Panel, each site tested one positive and one negative control. Sites rotated through the 3 positive controls throughout the study.

Table 87: External Controls Utilized in the Clinical Evaluations   

<table><tr><td rowspan=1 colspan=1>Mix ID</td><td rowspan=1 colspan=1>Organisms in Mix</td><td rowspan=1 colspan=1>Organism StrainID</td><td rowspan=1 colspan=1>Concentration inMix (CFU/mL)</td></tr><tr><td rowspan=5 colspan=1>Mix 1</td><td rowspan=1 colspan=1>Enterococcus casseliflavus</td><td rowspan=1 colspan=1>ATCC25788</td><td rowspan=1 colspan=1>1x107</td></tr><tr><td rowspan=1 colspan=1>Micrococcus luteus</td><td rowspan=1 colspan=1>ATCC19212</td><td rowspan=1 colspan=1>1x107</td></tr><tr><td rowspan=1 colspan=1>Streptococcus agalactiae</td><td rowspan=1 colspan=1>ATCC12401</td><td rowspan=1 colspan=1>1x106</td></tr><tr><td rowspan=1 colspan=1>Bacillus cereus</td><td rowspan=1 colspan=1>ATCC21769</td><td rowspan=1 colspan=1>1x107</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus lugdunensis</td><td rowspan=1 colspan=1>NRS879</td><td rowspan=1 colspan=1>1x107</td></tr><tr><td rowspan=4 colspan=1>Mix 2</td><td rowspan=1 colspan=1>Staphylococcus aureus/mecC</td><td rowspan=1 colspan=1>ATCCBAA-2313</td><td rowspan=1 colspan=1>1x106</td></tr><tr><td rowspan=1 colspan=1>Candida albicans</td><td rowspan=1 colspan=1>ATCC24433</td><td rowspan=1 colspan=1>1x107</td></tr><tr><td rowspan=1 colspan=1>Cutibacterium acnes (P. acnes)</td><td rowspan=1 colspan=1>ATCC6919</td><td rowspan=1 colspan=1>1x108</td></tr><tr><td rowspan=1 colspan=1>Stenotrophomonas maltophilia</td><td rowspan=1 colspan=1>ATCC13636</td><td rowspan=1 colspan=1>1x107</td></tr><tr><td rowspan=3 colspan=1>Mix 3</td><td rowspan=1 colspan=1>Staphylococcus epidermidis/MecA</td><td rowspan=1 colspan=1>ATCC35983</td><td rowspan=1 colspan=1>1x106</td></tr><tr><td rowspan=1 colspan=1>Candida glabrata</td><td rowspan=1 colspan=1>ATCC66032</td><td rowspan=1 colspan=1>1x107</td></tr><tr><td rowspan=1 colspan=1>Enterococcus faecium/VanA</td><td rowspan=1 colspan=1>ATCCBAA-2317</td><td rowspan=1 colspan=1>1x108</td></tr><tr><td rowspan=4 colspan=1>Mix 4</td><td rowspan=1 colspan=1>Bacillus subtilis</td><td rowspan=1 colspan=1>ATCC55614</td><td rowspan=1 colspan=1>1x107</td></tr><tr><td rowspan=1 colspan=1>Listeria monocytogenes</td><td rowspan=1 colspan=1>ATCC10890</td><td rowspan=1 colspan=1>1x106</td></tr><tr><td rowspan=1 colspan=1>Streptococcus pneumoniae</td><td rowspan=1 colspan=1>ATCCBAA-475</td><td rowspan=1 colspan=1>1x106</td></tr><tr><td rowspan=1 colspan=1>Escherichia coli</td><td rowspan=1 colspan=1>ATCC4157</td><td rowspan=1 colspan=1>1x107</td></tr><tr><td rowspan=5 colspan=1>Mix 5</td><td rowspan=1 colspan=1>Enterococcus faecalis/vanB</td><td rowspan=1 colspan=1>ATCC51575</td><td rowspan=1 colspan=1>1x107</td></tr><tr><td rowspan=1 colspan=1>Streptococcus anginosus</td><td rowspan=1 colspan=1>ATCC9895</td><td rowspan=1 colspan=1>1x106</td></tr><tr><td rowspan=1 colspan=1>Lactobacillus paracasei</td><td rowspan=1 colspan=1>ATCC25598</td><td rowspan=1 colspan=1>1x107</td></tr><tr><td rowspan=1 colspan=1>Streptococcus pyogenes</td><td rowspan=1 colspan=1>ATCC12384</td><td rowspan=1 colspan=1>1x106</td></tr><tr><td rowspan=1 colspan=1>Corynebacterium striatum</td><td rowspan=1 colspan=1>ATCC43735</td><td rowspan=1 colspan=1>1x107</td></tr></table>

A total of 110 control runs (55 negative and 55 positive) were tested across multiple testing days. All valid negative control runs generated the expected result on the BCID-GP Panel;

no targets were detected. All positive controls tested resulted in the correct targets detected with the exception of one run. This run failed to detect the Cutibactierium acnes (C. acnes) target. The concentration of $C .$ . acnes in this mix was noted to be at the Limit of Detection (LoD) and below the bottle positive concentration. Therefore, this result was expected and accepted.

4. Clinical cut-off:

Not applicable

5. Expected values/Reference range:

A prospective, multicenter clinical study was conducted to evaluate the clinical performance of the ePlex BCID-GP Panel in positive blood culture samples. A total of 711 samples were prospectively collected at 7 clinical sites in 2 phases from patients of all ages and genders. In the first phase from June 2014 through July 2016, 399 samples were prospectively collected and frozen; from January through February 2018, 312 samples were prospectively collected and tested fresh (never frozen). The expected values of individual analytes based on the ePlex BCID-GP Panel results in prospective samples are summarized by age group and by site in Table 88 and Table 89, respectively.

Table 88: Expected Value by Age Group (Prospective Samples)   

<table><tr><td colspan="1" rowspan="1">Target</td><td colspan="1" rowspan="1">All AgesN=71 </td><td colspan="1" rowspan="1">Age&lt;1(N=27)</td><td colspan="1" rowspan="1">Age 1-17(N=4</td><td colspan="1" rowspan="1">Age 18-44N=121)</td><td colspan="1" rowspan="1">Age 45-64(N2500)</td><td colspan="1" rowspan="1">Age 65-84N=217)</td><td colspan="1" rowspan="1">Age 85+(N=54)</td></tr><tr><td colspan="1" rowspan="1">Bacillus cereusgroup</td><td colspan="1" rowspan="1">5 (0.7)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">3 (2.5)</td><td colspan="1" rowspan="1">2 (0.8)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td></tr><tr><td colspan="1" rowspan="1">Bacillus subtilisgroup</td><td colspan="1" rowspan="1">2 (0.3)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">1 (0.8)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">1 (1.9)</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium</td><td colspan="1" rowspan="1">14 (2.0)</td><td colspan="1" rowspan="1">1 (3.7)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">4 (3.3)</td><td colspan="1" rowspan="1">5 (2.0)</td><td colspan="1" rowspan="1">4(1.8)</td><td colspan="1" rowspan="1">0 (0.0)</td></tr><tr><td colspan="1" rowspan="1">Cutibacterium acnes(P.acnes)</td><td colspan="1" rowspan="1">8 (1.1)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">3 (2.5)</td><td colspan="1" rowspan="1">2 (0.8)</td><td colspan="1" rowspan="1">3 (1.4)</td><td colspan="1" rowspan="1">0 (0.0)</td></tr><tr><td colspan="1" rowspan="1">Enterococcus</td><td colspan="1" rowspan="1">62 (8.7)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">6 (14.3)</td><td colspan="1" rowspan="1">8 (6.6)</td><td colspan="1" rowspan="1">20 (8.0)</td><td colspan="1" rowspan="1">24 (11.1)</td><td colspan="1" rowspan="1">4 (7.4)</td></tr><tr><td colspan="1" rowspan="1">Enterococcusfaecalis</td><td colspan="1" rowspan="1">50 (7.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">6 (14.3)</td><td colspan="1" rowspan="1">6 (5.0)</td><td colspan="1" rowspan="1">15 (6.0)</td><td colspan="1" rowspan="1">20 (9.2)</td><td colspan="1" rowspan="1">3 (5.6)</td></tr><tr><td colspan="1" rowspan="1">Enterococcusfaecium</td><td colspan="1" rowspan="1">12 (1.7)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">1 (0.8)</td><td colspan="1" rowspan="1">6 (2.4)</td><td colspan="1" rowspan="1">5 (2.3)</td><td colspan="1" rowspan="1">0 (0.0)</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus</td><td colspan="1" rowspan="1">5 (0.7)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">2 (1.7)</td><td colspan="1" rowspan="1">1 (0.4)</td><td colspan="1" rowspan="1">1 (0.5)</td><td colspan="1" rowspan="1">1 (1.9)</td></tr><tr><td colspan="1" rowspan="1">Listeria</td><td colspan="1" rowspan="1">1 (0.1)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">1 (0.5)</td><td colspan="1" rowspan="1">0 (0.0)</td></tr><tr><td colspan="1" rowspan="1">Listeriamonocytogenes</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td></tr><tr><td colspan="1" rowspan="1">Micrococcus</td><td colspan="1" rowspan="1">19 (2.7)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">3 (7.1)</td><td colspan="1" rowspan="1">2 (1.7)</td><td colspan="1" rowspan="1">8 (3.2)</td><td colspan="1" rowspan="1">5 (2.3)</td><td colspan="1" rowspan="1">1 (1.9)</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus</td><td colspan="1" rowspan="1">452 (63.6)</td><td colspan="1" rowspan="1">23 (85.2)</td><td colspan="1" rowspan="1">23 (54.8)</td><td colspan="1" rowspan="1">78 (64.5)</td><td colspan="1" rowspan="1">154 (61.6)</td><td colspan="1" rowspan="1">139 (64.1)</td><td colspan="1" rowspan="1">35 (64.8)</td></tr><tr><td colspan="1" rowspan="1">Staphylococcusaureus</td><td colspan="1" rowspan="1">162 (22.8)</td><td colspan="1" rowspan="1">8 (29.6)</td><td colspan="1" rowspan="1">4 (9.5)</td><td colspan="1" rowspan="1">37 (30.6)</td><td colspan="1" rowspan="1">69 (27.6)</td><td colspan="1" rowspan="1">38 (17.5)</td><td colspan="1" rowspan="1">6 (11.1)</td></tr><tr><td colspan="1" rowspan="1">Staphylococcusepidermidis</td><td colspan="1" rowspan="1">182 (25.6)</td><td colspan="1" rowspan="1">10 (37.0)</td><td colspan="1" rowspan="1">11 (26.2)</td><td colspan="1" rowspan="1">26 (21.5)</td><td colspan="1" rowspan="1">54 (21.6)</td><td colspan="1" rowspan="1">62 (28.6)</td><td colspan="1" rowspan="1">19 (35.2)</td></tr><tr><td colspan="1" rowspan="1">Staphylococcuslugdunensis</td><td colspan="1" rowspan="1">5 (0.7)</td><td colspan="1" rowspan="1">1 (3.7)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">3 (1.2)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">1 (1.9)</td></tr><tr><td colspan="1" rowspan="1">Streptococcus</td><td colspan="1" rowspan="1">110 (15.5)</td><td colspan="1" rowspan="1">5 (18.5)</td><td colspan="1" rowspan="1">9 (21.4)</td><td colspan="1" rowspan="1">16 (13.2)</td><td colspan="1" rowspan="1">40 (16.0)</td><td colspan="1" rowspan="1">31 (14.3)</td><td colspan="1" rowspan="1">9 (16.7)</td></tr><tr><td colspan="1" rowspan="1">Target</td><td colspan="1" rowspan="1">All Ages(N=71)</td><td colspan="1" rowspan="1">Age&lt;1(N=27</td><td colspan="1" rowspan="1">Age 1-17(N=42)</td><td colspan="1" rowspan="1">Age 18-44(N=121)</td><td colspan="1" rowspan="1">Age 45-64(N=2500)</td><td colspan="1" rowspan="1">Age 65-84N=217</td><td colspan="1" rowspan="1">Age 85+(N=54)</td></tr><tr><td colspan="1" rowspan="1">Streptococcusagalactiae</td><td colspan="1" rowspan="1">12 (1.7)</td><td colspan="1" rowspan="1">1 (3.7)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">1 (0.8)</td><td colspan="1" rowspan="1">5 (2.0)</td><td colspan="1" rowspan="1">5 (2.3)</td><td colspan="1" rowspan="1">0 (0.0)</td></tr><tr><td colspan="1" rowspan="1">Streptococcusanginosus group</td><td colspan="1" rowspan="1">10 (1.4)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">2 (1.7)</td><td colspan="1" rowspan="1">3 (1.2)</td><td colspan="1" rowspan="1">3 (1.4)</td><td colspan="1" rowspan="1">2 (3.7)</td></tr><tr><td colspan="1" rowspan="1">Streptococcuspneumoniae</td><td colspan="1" rowspan="1">28 (3.9)</td><td colspan="1" rowspan="1">2 (7.4)</td><td colspan="1" rowspan="1">2 (4.8)</td><td colspan="1" rowspan="1">3 (2.5)</td><td colspan="1" rowspan="1">12 (4.8)</td><td colspan="1" rowspan="1">8 (3.7)</td><td colspan="1" rowspan="1">1 (1.9)</td></tr><tr><td colspan="1" rowspan="1">Streptococcuspyogenes</td><td colspan="1" rowspan="1">8 (1.1)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">1 (0.8)</td><td colspan="1" rowspan="1">5 (2.0)</td><td colspan="1" rowspan="1">2 (0.9)</td><td colspan="1" rowspan="1">0 (0.0)</td></tr><tr><td colspan="1" rowspan="1">Pan Candida</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td></tr><tr><td colspan="1" rowspan="1">Pan Gram-Negative</td><td colspan="1" rowspan="1">25 (3.5)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">4 (9.5)</td><td colspan="1" rowspan="1">4 (3.3)</td><td colspan="1" rowspan="1">10 (4.0)</td><td colspan="1" rowspan="1">6 (2.8)</td><td colspan="1" rowspan="1">1 (1.9)</td></tr><tr><td colspan="1" rowspan="1">mecA(Staphylococcus)</td><td colspan="1" rowspan="1">261 (36.7)</td><td colspan="1" rowspan="1">14 (51.9)</td><td colspan="1" rowspan="1">10 (23.8)</td><td colspan="1" rowspan="1">41 (33.9)</td><td colspan="1" rowspan="1">83 (33.2)</td><td colspan="1" rowspan="1">94 (43.3)</td><td colspan="1" rowspan="1">19 (35.2)</td></tr><tr><td colspan="1" rowspan="1">mecA (S.aureus)</td><td colspan="1" rowspan="1">86 (12.1)</td><td colspan="1" rowspan="1">4 (14.8)</td><td colspan="1" rowspan="1">1 (2.4)</td><td colspan="1" rowspan="1">17 (14.0)</td><td colspan="1" rowspan="1">35 (14.0)</td><td colspan="1" rowspan="1">26 (12.0)</td><td colspan="1" rowspan="1">3 (5.6)</td></tr><tr><td colspan="1" rowspan="1">mecA(S. epidermidis)</td><td colspan="1" rowspan="1">137 (19.3)</td><td colspan="1" rowspan="1">8 (29.6)</td><td colspan="1" rowspan="1">9 (21.4)</td><td colspan="1" rowspan="1">19 (15.7)</td><td colspan="1" rowspan="1">38 (15.2)</td><td colspan="1" rowspan="1">48 (22.1)</td><td colspan="1" rowspan="1">15 (27.8)</td></tr><tr><td colspan="1" rowspan="1">mecA (S.lugdunensis)</td><td colspan="1" rowspan="1">1 (0.1)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">1 (0.4)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td></tr><tr><td colspan="1" rowspan="1">mecA (CoNSexcludingS. epidermidis /S. lugdunensis)</td><td colspan="1" rowspan="1">40 (5.6)</td><td colspan="1" rowspan="1">2 (7.4)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">6 (5.0)</td><td colspan="1" rowspan="1">10 (4.0)</td><td colspan="1" rowspan="1">20 (9.2)</td><td colspan="1" rowspan="1">2 (3.7)</td></tr><tr><td colspan="1" rowspan="1">mecC(Staphylococcus)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td></tr><tr><td colspan="1" rowspan="1">vanA (Enterococcus)</td><td colspan="1" rowspan="1">9 (1.3)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">6 (2.4)</td><td colspan="1" rowspan="1">3 (1.4)</td><td colspan="1" rowspan="1">0 (0.0)</td></tr><tr><td colspan="1" rowspan="1">vanA (E.faecalis)</td><td colspan="1" rowspan="1">1 (0.1)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">1 (0.4)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td></tr><tr><td colspan="1" rowspan="1">vanA (E.faecium)</td><td colspan="1" rowspan="1">8 (1.1)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">5 (2.0)</td><td colspan="1" rowspan="1">3 (1.4)</td><td colspan="1" rowspan="1">0 (0.0)</td></tr><tr><td colspan="1" rowspan="1">vanB(Enterococcus)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td></tr></table>

Table 89: Expected Value by Collection Site (Prospective Samples)   

<table><tr><td colspan="1" rowspan="1">Target</td><td colspan="1" rowspan="1">All Sites(N=711)</td><td colspan="1" rowspan="1">Site 1(N=161)</td><td colspan="1" rowspan="1">Site 2(N=58)</td><td colspan="1" rowspan="1">Site 3(N=164)</td><td colspan="1" rowspan="1">Site 4(N=145)</td><td colspan="1" rowspan="1">Site 5(N=77)</td><td colspan="1" rowspan="1">Site 6(N=33)</td><td colspan="1" rowspan="1">Site 7(N=73)</td></tr><tr><td colspan="1" rowspan="1">Bacillus cereusgroup</td><td colspan="1" rowspan="1">5 (0.7)</td><td colspan="1" rowspan="1">3 (1.9)</td><td colspan="1" rowspan="1">1 (1.7)</td><td colspan="1" rowspan="1">1 (0.6)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td></tr><tr><td colspan="1" rowspan="1">Bacillus subtilisgroup</td><td colspan="1" rowspan="1">2 (0.3)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">1 (1.7)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">1 (1.3)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium</td><td colspan="1" rowspan="1">14 (2.0)</td><td colspan="1" rowspan="1">2 (1.2)</td><td colspan="1" rowspan="1">2 (3.4)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">6 (4.1)</td><td colspan="1" rowspan="1">2 (2.6)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">2 (2.7)</td></tr><tr><td colspan="1" rowspan="1">Cutibacteriumacnes (P.acnes)</td><td colspan="1" rowspan="1">8 (1.1)</td><td colspan="1" rowspan="1">1 (0.6)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">1 (0.6)</td><td colspan="1" rowspan="1">2 (1.4)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">4 (5.5)</td></tr><tr><td colspan="1" rowspan="1">Enterococcus</td><td colspan="1" rowspan="1">62 (8.7)</td><td colspan="1" rowspan="1">20 (12.4)</td><td colspan="1" rowspan="1">7 (12.1)</td><td colspan="1" rowspan="1">15 (9.1)</td><td colspan="1" rowspan="1">9 (6.2)</td><td colspan="1" rowspan="1">10 (13.0)</td><td colspan="1" rowspan="1">1 (3.0)</td><td colspan="1" rowspan="1">0 (0.0)</td></tr><tr><td colspan="1" rowspan="1">Enterococcusfaecalis</td><td colspan="1" rowspan="1">50 (7.0)</td><td colspan="1" rowspan="1">14 (8.7)</td><td colspan="1" rowspan="1">6 (10.3)</td><td colspan="1" rowspan="1">13 (7.9)</td><td colspan="1" rowspan="1">8 (5.5)</td><td colspan="1" rowspan="1">8 (10.4)</td><td colspan="1" rowspan="1">1 (3.0)</td><td colspan="1" rowspan="1">0 (0.0)</td></tr><tr><td colspan="1" rowspan="1">Enterococcusfaecium</td><td colspan="1" rowspan="1">12 (1.7)</td><td colspan="1" rowspan="1">6 (3.7)</td><td colspan="1" rowspan="1">1 (1.7)</td><td colspan="1" rowspan="1">1 (0.6)</td><td colspan="1" rowspan="1">2 (1.4)</td><td colspan="1" rowspan="1">2 (2.6)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus</td><td colspan="1" rowspan="1">5 (0.7)</td><td colspan="1" rowspan="1">1 (0.6)</td><td colspan="1" rowspan="1">1 (1.7)</td><td colspan="1" rowspan="1">2 (1.2)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">1 (1.3)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td></tr><tr><td colspan="1" rowspan="1">Listeria</td><td colspan="1" rowspan="1">1 (0.1)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">1 (0.7)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td></tr><tr><td colspan="1" rowspan="1">Listeriamonocytogenes</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td></tr><tr><td colspan="1" rowspan="1">Micrococcus</td><td colspan="1" rowspan="1">19 (2.7)</td><td colspan="1" rowspan="1">2 (1.2)</td><td colspan="1" rowspan="1">2 (3.4)</td><td colspan="1" rowspan="1">5 (3.0)</td><td colspan="1" rowspan="1">6 (4.1)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">4 (5.5)</td></tr><tr><td colspan="1" rowspan="1">Target</td><td colspan="1" rowspan="1">All Sites(N=711</td><td colspan="1" rowspan="1">Site 1(N=161</td><td colspan="1" rowspan="1">Site 2(N58</td><td colspan="1" rowspan="1">Site 3(N=164</td><td colspan="1" rowspan="1">Site 4(N=145)</td><td colspan="1" rowspan="1">Site 5(N=77</td><td colspan="1" rowspan="1">Site 6(N=33)</td><td colspan="1" rowspan="1">Site 7(N=73</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus</td><td colspan="1" rowspan="1">452(63.6)</td><td colspan="1" rowspan="1">106(65.8)</td><td colspan="1" rowspan="1">27 (46.6)</td><td colspan="1" rowspan="1">109(66.5)</td><td colspan="1" rowspan="1">98 (67.6)</td><td colspan="1" rowspan="1">52 (67.5)</td><td colspan="1" rowspan="1">24 (72.7)</td><td colspan="1" rowspan="1">36 (49.3)</td></tr><tr><td colspan="1" rowspan="1">Staphylococcusaureus</td><td colspan="1" rowspan="1">162(22.8)</td><td colspan="1" rowspan="1">36 (22.4)</td><td colspan="1" rowspan="1">7 (12.1)</td><td colspan="1" rowspan="1">56 (34.1)</td><td colspan="1" rowspan="1">27 (18.6)</td><td colspan="1" rowspan="1">18 (23.4)</td><td colspan="1" rowspan="1">6 (18.2)</td><td colspan="1" rowspan="1">12 (16.4)</td></tr><tr><td colspan="1" rowspan="1">Staphylococcusepiddermidis</td><td colspan="1" rowspan="1">182(25.6)</td><td colspan="1" rowspan="1">41 (25.5)</td><td colspan="1" rowspan="1">14 (24.1)</td><td colspan="1" rowspan="1">34 (20.7)</td><td colspan="1" rowspan="1">44 (30.3)</td><td colspan="1" rowspan="1">23 (29.9)</td><td colspan="1" rowspan="1">13 (39.4)</td><td colspan="1" rowspan="1">13 (17.8)</td></tr><tr><td colspan="1" rowspan="1">Staphylococcuslugdunensis</td><td colspan="1" rowspan="1">5 (0.7)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">2 (1.2)</td><td colspan="1" rowspan="1">2 (1.4)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">1 (1.4)</td></tr><tr><td colspan="1" rowspan="1">Streptococcus</td><td colspan="1" rowspan="1">110(15.5)</td><td colspan="1" rowspan="1">18 (11.2)</td><td colspan="1" rowspan="1">9 (15.5)</td><td colspan="1" rowspan="1">28 (17.1)</td><td colspan="1" rowspan="1">20 (13.8)</td><td colspan="1" rowspan="1">11 (14.3)</td><td colspan="1" rowspan="1">5 (15.2)</td><td colspan="1" rowspan="1">19 (26.0)</td></tr><tr><td colspan="1" rowspan="1">Streptococcusagalactiae</td><td colspan="1" rowspan="1">12 (1.7)</td><td colspan="1" rowspan="1">2 (1.2)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">2 (1.2)</td><td colspan="1" rowspan="1">2 (1.4)</td><td colspan="1" rowspan="1">2 (2.6)</td><td colspan="1" rowspan="1">2 (6.1)</td><td colspan="1" rowspan="1">2 (2.7)</td></tr><tr><td colspan="1" rowspan="1">Streptococcusanginosus group</td><td colspan="1" rowspan="1">10 (1.4)</td><td colspan="1" rowspan="1">2 (1.2)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">6 (3.7)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">2 (2.6)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td></tr><tr><td colspan="1" rowspan="1">Streptococcuspneumoniae</td><td colspan="1" rowspan="1">28 (3.9)</td><td colspan="1" rowspan="1">3 (1.9)</td><td colspan="1" rowspan="1">5 (8.6)</td><td colspan="1" rowspan="1">5 (3.0)</td><td colspan="1" rowspan="1">4 (2.8)</td><td colspan="1" rowspan="1">1 (1.3)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">10 (13.7)</td></tr><tr><td colspan="1" rowspan="1">Streptococcuspyogenes</td><td colspan="1" rowspan="1">8 (1.1)</td><td colspan="1" rowspan="1">1 (0.6)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">3 (1.8)</td><td colspan="1" rowspan="1">1 (0.7)</td><td colspan="1" rowspan="1">1 (1.3)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">2 (2.7)</td></tr><tr><td colspan="1" rowspan="1">Pan Candida</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td></tr><tr><td colspan="1" rowspan="1">Pan Gram-Negative</td><td colspan="1" rowspan="1">25 (3.5)</td><td colspan="1" rowspan="1">9 (5.6)</td><td colspan="1" rowspan="1">4 (6.9)</td><td colspan="1" rowspan="1">3 (1.8)</td><td colspan="1" rowspan="1">2 (1.4)</td><td colspan="1" rowspan="1">5 (6.5)</td><td colspan="1" rowspan="1">1 (3.0)</td><td colspan="1" rowspan="1">1(1.4)</td></tr><tr><td colspan="1" rowspan="1">mecA(Staphylococcus)</td><td colspan="1" rowspan="1">261(36.7)</td><td colspan="1" rowspan="1">69 (42.9)</td><td colspan="1" rowspan="1">17 (29.3)</td><td colspan="1" rowspan="1">68 (41.5)</td><td colspan="1" rowspan="1">55 (37.9)</td><td colspan="1" rowspan="1">25 (32.5)</td><td colspan="1" rowspan="1">13 (39.4)</td><td colspan="1" rowspan="1">14 (19.2)</td></tr><tr><td colspan="1" rowspan="1">mecA (S.aureus)</td><td colspan="1" rowspan="1">86 (12.1)</td><td colspan="1" rowspan="1">19 (11.8)</td><td colspan="1" rowspan="1">5(8.6)</td><td colspan="1" rowspan="1">28 (17.1)</td><td colspan="1" rowspan="1">19 (13.1)</td><td colspan="1" rowspan="1">7 (9.1)</td><td colspan="1" rowspan="1">3 (9.1)</td><td colspan="1" rowspan="1">5(6.8)</td></tr><tr><td colspan="1" rowspan="1">mecA (S.epidermidis)</td><td colspan="1" rowspan="1">137(19.3)</td><td colspan="1" rowspan="1">36 (22.4)</td><td colspan="1" rowspan="1">10 (17.2)</td><td colspan="1" rowspan="1">31 (18.9)</td><td colspan="1" rowspan="1">29 (20.0)</td><td colspan="1" rowspan="1">17 (22.1)</td><td colspan="1" rowspan="1">8 (24.2)</td><td colspan="1" rowspan="1">6 (8.2)</td></tr><tr><td colspan="1" rowspan="1">mecA (S.lugdunensis)</td><td colspan="1" rowspan="1">1 (0.1)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">1 (0.6)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td></tr><tr><td colspan="1" rowspan="1">mecA (CoNSexcludingS. epidermidis/S. lugdunensis)</td><td colspan="1" rowspan="1">40 (5.6)</td><td colspan="1" rowspan="1">15 (9.3)</td><td colspan="1" rowspan="1">2 (3.4)</td><td colspan="1" rowspan="1">9 (5.5)</td><td colspan="1" rowspan="1">7 (4.8)</td><td colspan="1" rowspan="1">2 (2.6)</td><td colspan="1" rowspan="1">2 (6.1)</td><td colspan="1" rowspan="1">3 (4.1)</td></tr><tr><td colspan="1" rowspan="1">mecC(Staphylococcus)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td></tr><tr><td colspan="1" rowspan="1">vanA(Enterococcus)</td><td colspan="1" rowspan="1">9 (1.3)</td><td colspan="1" rowspan="1">6 (3.7)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">1 (0.6)</td><td colspan="1" rowspan="1">1 (0.7)</td><td colspan="1" rowspan="1">1 (1.3)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td></tr><tr><td colspan="1" rowspan="1">vanA (E. faecalis)</td><td colspan="1" rowspan="1">1 (0.1)</td><td colspan="1" rowspan="1">1 (0.6)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td></tr><tr><td colspan="1" rowspan="1">vanA (E. faecium)</td><td colspan="1" rowspan="1">8 (1.1)</td><td colspan="1" rowspan="1">5 (3.1)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">1 (0.6)</td><td colspan="1" rowspan="1">1 (0.7)</td><td colspan="1" rowspan="1">1 (1.3)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td></tr><tr><td colspan="1" rowspan="1">vanB(Enterococcus)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td></tr><tr><td colspan="1" rowspan="1">EY</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">EPY</td></tr></table>

N. Instrument Name: GenMark’s ePlex Instrument

# O. System Descriptions:

1. Modes of Operation: See Device Description (Section I) above

# 2. Software

FDA has reviewed applicant’s Hazard Analysis and software development processes for this line of product types:

Yes X___ or No

3. Specimen Identification: The Sample ID can be entered manually or scanned in by using barcode scanner.

4. Quality Control: See Section M (1c) above

# P. Other Supportive Instrument Performance Characteristics Data Not Covered In The “Performance Characteristics” Section above:

Not Applicable

# Q. Proposed Labeling:

The labeling supports the finding of substantial equivalence for this device.

# R. Patient Perspectives:

This submission did not include specific information on patient perspectives for this device.

# S. Conclusion:

The submitted information in this premarket notification is complete and supports a substantial equivalence decision.