# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATION DECISION SUMMARY

A. 510(k) Number: K182703 B. Purpose for Submission:

To obtain clearance for the EntericBio Dx gastrointestinal microorganism multiplex nucleic acid-based assay.

# C. Measurand:

Nucleic acid sequences of the following enteric pathogens: Salmonella enterica spp., Shigella spp./enteroinvasive E. coli (EIEC),Campylobacter spp. (jejuni/coli/lari), stx1/ stx2 genes which may be present in Shiga toxin-producing $E$ . coli (STEC) as well as Shigella dysenteriae which possess a stx gene identical to stx1 gene of STEC, Vibrio spp. (cholerae and parahaemolyticus), Giardia lamblia (also known as $G .$ . intestinalis and G. duodenalis), Entamoeba histolytica

# D. Type of Test:

A multiplexed nucleic acid-based real-time PCR test for in vitro qualitative detection and identification of multiple enteric pathogens in Cary-Blair preserved fecal specimens from individuals with signs and symptoms of infectious colitis or gastroenteritis. It is intended to be performed on the ABI 7500 Fast Dx real-time instrument and analyzed using the UgenTec FastFinder Software with the EntericBio Dx plugin.

# E. Applicant:

Serosep, Ltd.

F. Proprietary and Established Names: EntericBio Dx assay

# G. Regulatory Information:

1. Regulation section: 866.3990 – Gastrointestinal microorganism nucleic acid-acid assay

2. Classification:

Class II

3. Product code: PCH, OOI, NSU

4. Panel: 83 (Microbiology)

# H. Intended Use:

1. Intended use:

The EntericBio Dx assay, performed on ABI 7500 Fast Dx real-time instrument, is an in vitro multiplexed nucleic acid test for the direct, simultaneous, qualitative detection and identification of multiple enteric pathogens in Cary-Blair preserved stool specimens from individuals with signs and symptoms of infectious colitis or gastroenteritis. The test is based on detection of nucleic acids from:

• Salmonella enterica spp. Shigella spp./ Enteroinvasive E. coli (EIEC) Campylobacter spp. (jejuni, coli and lari) STEC (Shiga-like toxin-producing E. coli), stx1/ stx2 genes Vibrio spp. (cholerae and parahaemolyticus) Giardia lamblia (also known as G. intestinalis and G. duodenalis) Entamoeba histolytica

Testing is performed on Cary-Blair preserved diarrheal specimens from symptomatic patients with suspected acute gastroenteritis, enteritis or colitis of bacterial or parasitic origin. The test is performed directly on the specimen, utilizing real-time polymerase chain reaction (PCR) for the amplification of Salmonella-specific, Campylobacterspecific, Shigella/ EIEC-specific ipaH, $s t x 1 / s t x 2$ , Vibrio-specific, Entamoeba-specific and Giardia-specific gene sequences. The test utilizes fluorogenic sequence-specific hybridization probes for the detection of the amplified DNA.

This test is intended for use, in conjunction with clinical presentation, laboratory findings and epidemiological information, as an aid in the diagnosis of Salmonella, Shigella / EIEC, Shiga-like toxin-producing E. coli, Campylobacter spp., Vibrio spp., Entamoeba histolytica and Giardia spp. infections in humans.

Results of this test should not be used as the sole basis for diagnosis, treatment, or other patient management decisions. Positive results do not rule out co-infection with other organisms that are not detected by this test and may not be the sole or definitive cause of patient illness. Negative EntericBio Dx assay results in the setting of clinical illness

compatible with gastroenteritis may be due to infection by pathogens that are not detected by this test or non-infectious causes such as ulcerative colitis, irritable bowel syndrome, or Crohn’s disease.

2. Indication(s) for use: Same as Intended Use.

3. Special conditions for use statement(s): For prescription use only

4. Special instrument requirements:

For use with the ABI 7500 Fast Dx real-time instrument with UgenTec FastFinder Software with EntericBio Dx plugin.

# I. Device Description:

The EntericBio Dx assay is a molecular in vitro diagnostic test for direct, qualitative detection and identification of the following enteric microorganisms, associated with human gastroenteritis, directly, from Cary-Blair preserved fecal samples:

Salmonella enterica spp.,   
Shigella spp./ Enteroinvasive E. coli,   
Campylobacter spp. (jejuni/coli/lari)   
Vibrio spp. (cholerae and parahaemolyticus),   
STEC (Shiga-like toxin-producing E. coli), stx1/ stx2 genes   
Giardia lamblia (also known as G. intestinalis and G. duodenalis),   
Entamoeba histolytica

The EntericBio Dx assay kit contains the following:

<table><tr><td>Content</td><td>Quantity</td></tr><tr><td>EntericBio Dx PCR strips Ten (10) sealed pouches of the EntericBio® Dx assay. Each pouch contains six (6), 12-well PCR strips with optical caps. Each strip is divided into four (4), 3-well sections into which a sample is inoculated. Each PCR well on the strip contains an all-in-one PCR master mix in a lyophilized pellet which includes target-specific hydrolysis probes and primers, Internal Amplification Control (IAC), PCR reagents and buffers necessary for real-time PCR.</td><td>240 tests</td></tr><tr><td colspan="1" rowspan="1">EntericBio Resuspension SolutionOne (1) 60 mL bottle of Resuspension Solution. Resuspension Solutionis added to the PCR wells to re-suspend the lyophilized PCR reagents(automated transfer on the EntericBio workstation) and it also acts as aNegative Control (No Template Control) for the assay</td><td colspan="1" rowspan="1">1 bottle</td></tr><tr><td colspan="1" rowspan="1">EntericBio Positive ControlPositive Control vial (red cap) contains a lyophilized pellet whichincludes DNA for each of the targets detected by the EntericBio® Dxassay.</td><td colspan="1" rowspan="1">1 vial</td></tr><tr><td colspan="1" rowspan="1">EntericBio Reconstitution SolutionReconstitution Buffer (500 µL, green cap) is used to re-suspend thelyophilized Positive Control pellet when the kit is first opened.</td><td colspan="1" rowspan="1">1 vial</td></tr><tr><td colspan="1" rowspan="1">EntericBio Stool Preparation Solution (SPS) tubesEach SPS tube comes pre-filled with 4 mL of SPS liquid into which aCary-Blair preserved fecal sample is inoculated.</td><td colspan="1" rowspan="1">240 tubes</td></tr></table>

Testing is performed directly on Cary-Blair preserved stool specimens and does not require nucleic acid extraction/purification. The PCR master mix with all the reagents required to perform each test is lyophilized into individual reaction wells on an EntericBio Dx PCR strip. Each reaction well contains an Internal Amplification Control (IAC) to monitor for PCR inhibition.

The EntericBio Dx assay is comprised of three multiplex assays that are performed in three wells of the EntericBio Dx PCR strip. Each PCR strip allows for testing of four specimens (three wells per specimen, 12 wells per strip). The following targets are evaluated in each of the three testing wells (Wells A, B, and C):

Well A: Salmonella enterica spp., Shigella spp. / Enteroinvasive E. coli (EIEC), Campylobacter spp. (C. jejuni, C. coli and C. lari) and IAC   
Well B: Shiga toxin-producing E. coli (STEC), Vibrio spp. (V. cholerae, V.   
parahaemolyticus) and IAC   
Well C: Giardia spp., E. histolytica and IAC

The following includes a listing of required materials, equipment and software that is required, but provided separately and a listing of materials required but not provided.

Materials and equipment required and provided separately   

<table><tr><td rowspan=1 colspan=1>Product Name</td><td rowspan=1 colspan=1>Catalog Number</td></tr><tr><td rowspan=1 colspan=1>EntericBio workstation with EntericBio SPSracks</td><td rowspan=1 colspan=1>(Serosep, Cat. No: 5070000450)</td></tr><tr><td rowspan=1 colspan=1>EntericBio heatstation with EntericBio blockinserts</td><td rowspan=1 colspan=1>(Serosep, Cat No: EBQPD4 SEROSEP US)</td></tr><tr><td rowspan=1 colspan=1>Eppendorf 50μl filter tips</td><td rowspan=1 colspan=1>(Eppendorf, Cat No: 0030014430)</td></tr><tr><td rowspan=1 colspan=1>Eppendorf 5430 Plate centrifuge and rotor</td><td rowspan=1 colspan=1>(Eppendorf, Cat. No: 022620596 and022654403)</td></tr><tr><td rowspan=1 colspan=1>Eppendorf MixMate</td><td rowspan=1 colspan=1>(Eppendorf, Cat. No: 022674200)</td></tr><tr><td rowspan=1 colspan=1>Copan disposable FLOQSwabs</td><td rowspan=1 colspan=1>(Copan, Cat No: 502CS01)</td></tr><tr><td rowspan=1 colspan=1>Empty SPS tubes</td><td rowspan=1 colspan=1>(Serosep, Cat. No. XPR2N011.SER)</td></tr><tr><td rowspan=1 colspan=1>Pierceable SPS Caps</td><td rowspan=1 colspan=1>(Serosep, Cat. No. 2E018N500.SER)</td></tr><tr><td rowspan=1 colspan=1>ABI 7500 PCR strip holder</td><td rowspan=1 colspan=1>(Bioplastics, Cat. No AB19805G)</td></tr><tr><td rowspan=1 colspan=1>UgenTec FastFinder software with EntericBioDx v1.3 plugin</td><td rowspan=1 colspan=1>N/A</td></tr></table>

Materials and equipment required but not provided   

<table><tr><td rowspan=1 colspan=1>Product Name</td><td rowspan=1 colspan=1>Catalog Number</td></tr><tr><td rowspan=1 colspan=1>ABI 7500 Dx Fast Cycler with SDS Dxsoftware</td><td rowspan=1 colspan=1>(ThermoFisher Scientific, Cat No. 4406984/4406985)</td></tr><tr><td rowspan=1 colspan=2>Sterile, graduated single-wrapped, transfer pipettes (min. transfer volume 4 mL)</td></tr><tr><td rowspan=1 colspan=2>Manual pipette of 100 - 1000 µl volume with suitable filter tip</td></tr></table>

# J. Substantial Equivalence Information:

1. Predicate device name: FilmArray Gastrointestinal (GI) Panel

2. Predicate 510(k) number(s): K140407

3. Comparison with predicate:

<table><tr><td rowspan=1 colspan=3>Similarities</td></tr><tr><td rowspan=1 colspan=1>Item</td><td rowspan=1 colspan=1>Device (EntericBio Dx)</td><td rowspan=1 colspan=1>Predicate (FilmArray GI Panel,K141407)</td></tr><tr><td rowspan=1 colspan=1>Type of Test</td><td rowspan=1 colspan=1>Qualitative nucleic acid test</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=1>TechnologicalPrinciples</td><td rowspan=1 colspan=1>Multiplex nucleic acid PCR</td><td rowspan=1 colspan=1>Same (see below for differences)</td></tr><tr><td rowspan=1 colspan=1>MicrorganismsDetected</td><td rowspan=1 colspan=1>Salmonella enterica spp.Shigella spp./ Enteroinvasive E.coli (EIEC)Campylobacter spp. (jejuni, coliand lari)STEC (Shiga-like toxin-producingE. coli), stx1/ stx2 genesVibrio spp. (cholerae andparahaemolyticus)Giardia lamblia (also known as G.intestinalis and G. duodenalis)Entamoeba histolytica</td><td rowspan=1 colspan=1>Same (see below for differences)</td></tr><tr><td rowspan=1 colspan=1>SpecimenTypes</td><td rowspan=1 colspan=1>Human stool (Cary-Blair preserved)</td><td rowspan=1 colspan=1>Same</td></tr></table>

<table><tr><td colspan="3" rowspan="1">Differences</td></tr><tr><td colspan="1" rowspan="1">Item</td><td colspan="1" rowspan="1">Device</td><td colspan="1" rowspan="1">Predicate</td></tr><tr><td colspan="1" rowspan="1">MicroorganismsDetected</td><td colspan="1" rowspan="1">Salmonella enterica spp.Shigella spp./ Enteroinvasive E. coli(EIEC)Campylobacter spp. (jejuni, coli andlari)STEC (Shiga-like toxin-producingE. coli), stx1/ stx2 genesVibrio spp. (cholerae andparahaemolyticus)Giardia lamblia (also known as G.intestinalis and G. duodenalis)Entamoeba histolytica</td><td colspan="1" rowspan="1">Same plus the followingmicroorganism targets: Clostridiumdifficile (C. difficile) toxin A/B,Campylobacter spp. (C. upsaliensis),Cryptosporidium (parvum, hominis),Plesiomonas shigelloides,Enteroaggregative E. coli,Enteropathogenic E. coli,Cyclospora cayetanensis, AdenovirusF40/41, Astrovirus, Norovirus GI/GII,RotavirusA, Sapovirus (GI, GII, GIVand GV).</td></tr><tr><td colspan="1" rowspan="1">TechnologicalPrinciples</td><td colspan="1" rowspan="1">Real-time multiplex RT-PCR basedonhydrolysis probe chemistry.</td><td colspan="1" rowspan="1">Nested multiplex PCR followed byhigh resolution melt analysis toconfirm identity of the PCR product.</td></tr><tr><td colspan="1" rowspan="1">Instrumentation</td><td colspan="1" rowspan="1">The Applied Biosystems 7500 FastDx Real-Time PCR instrument is areal-time nucleic acid amplificationand five color fluorescencedetection system for use with theEntericBio Dx Assay. Results areinterpreted using the EntericBioFastFinder plugin.</td><td colspan="1" rowspan="1">Detection and interpretation isautomated on the FilmArrayinstrument by analysis of the specificPCR product melting temperature.</td></tr><tr><td colspan="1" rowspan="1">Device Format</td><td colspan="1" rowspan="1">The EntericBio Dx Assay kitsprovide a PCR master mix with allthe reagents required to performeach test and are lyophilized intoindividual reaction wells. Eachreaction well contains an InternalAmplification Control. A syntheticPositive Control (containing targetsequences) is provided with eachkit.</td><td colspan="1" rowspan="1">The FilmArray GI panel provides allreagents lyophilized in a disposablepouch. Each pouch contains twocontrols which monitor sampleprocessing and both stages of PCRand melt analysis.</td></tr><tr><td colspan="1" rowspan="1">SamplePreparationMethod</td><td colspan="1" rowspan="1">Sample processed directlyfollowing heat treatment ofspecimen in a SPS tube.</td><td colspan="1" rowspan="1">Sample processing is automated in theFilmArray instrument. The sample islysed by a combination of mechanical(bead beating) and chemical meansand the liberated nucleic acid iscaptured, washed and eluted usingmagnetic bead technology.</td></tr><tr><td colspan="1" rowspan="1">Controls</td><td colspan="1" rowspan="1">The internal amplification control islyophilized within the PCR mix anincluded in each PCR reaction.</td><td colspan="1" rowspan="1">Two internal controls are included ineach reagent pouch.</td></tr></table>

# K. Standard/Guidance Document Referenced:

None

L. Test Principle:

A Copan FLOQSwab is used to transfer the Cary-Blair preserved specimen to the EntericBio Stool Preparation Solution (SPS). The SPS tube is closed with a pierceable cap and transferred to the EntericBio Heatstation for lysis at $1 0 3 ^ { \circ } \mathrm { C }$ for 30 minutes. The heat-treated specimens are then placed on the EntericBio Workstation for fully automated transfer of the processed samples directly to lyophilized PCR reaction wells. The wells are capped, vortexed, centrifuged and transferred to the ABI 7500 Fast Dx real-time PCR instrument for amplification.

The EntericBio Dx assay utilizes real-time multiplex PCR technology based on hydrolysis probe reagent chemistry. The amplified targets are detected by the ABI 7500 Fast Dx instrument using target-specific probes, labelled at one end with a fluorescent reporter dye (fluorophore) and at the other end with a quencher molecule.

The probes (for assay target analytes and IAC) are labelled with different fluorophores to allow simultaneous detection of multiple targets in one reaction, in different optical channels of the ABI 7500 Fast Dx instrument.

The fluorescence data generated by the ABI 7500 Fast Dx instrument and SDS Dx Software are automatically analyzed with the UgenTec FastFinder EntericBio Dx plugin.

# M. Performance Characteristics

1. Analytical performance:

a. Precision/Reproducibility:

A reproducibility study was performed to assess the inter-site and overall reproducibility of the EntericBio Dx assay. Reproducibility testing was performed inhouse, simulating a multi-site study using three instruments, multiple operators and panels of contrived stool samples. Positive panel members were prepared using natural negative stool matrix spiked with various combinations of Vibrio parahaemolyticus, Shigella sonnei and Giardia lamblia. These three target analytes were representative of each of the three component multiplex assays of EntericBio Dx. Each analyte was evaluated at two different concentrations (Low Positive at $1 . 5 \mathrm { x }$ LoD and Moderate Positive at 3x LoD) using three batches of EntericBio Dx assay reagents. A true negative specimen was also included in the study. Reproducibility panels containing randomized panel members were tested at each of three independent in-house testing areas by two different operators for five non-consecutive days. Sample identities were blinded to the user.

Results from the study demonstrated $100 \%$ agreement with the expected result for moderate positive samples tested at all three in-house sites. For low-positive samples, samples containing Shigella sonnei and Vibrio parahaemolyticus samples generated $100 \%$ agreement with the expected positive results and samples containing Giardia lamblia generated $9 8 \%$ agreement to the expected positive result. A summary of qualitative results (percent agreement with the expected result) for each analyte is shown in Table 1.

Table 1: Reproducibility study, qualitative results   

<table><tr><td rowspan=2 colspan=1>MicroorganismTested</td><td rowspan=2 colspan=1>ConcentrationTested</td><td rowspan=2 colspan=1>FastFinderExpectedResult</td><td rowspan=1 colspan=4>% Agreement with Expected Result</td></tr><tr><td rowspan=1 colspan=1>Serosep1</td><td rowspan=1 colspan=1>Serosep2</td><td rowspan=1 colspan=1>Serosep3</td><td rowspan=1 colspan=1>All</td></tr><tr><td rowspan=3 colspan=1>Shigella sonneiDSM 5570</td><td rowspan=1 colspan=1>ModeratePositive3x LoD</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>90/90100%</td></tr><tr><td rowspan=1 colspan=1>Low Positive1.5x LoD</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>90/90100%</td></tr><tr><td rowspan=1 colspan=1>True Negative</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>60/60100%</td><td rowspan=1 colspan=1>60/60100%</td><td rowspan=1 colspan=1>59/59*100%</td><td rowspan=1 colspan=1>179/179100%</td></tr><tr><td rowspan=3 colspan=1>VibrioparahaemolyticusCCUG 14474</td><td rowspan=1 colspan=1>ModeratePositive3x LoD</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>90/90100%</td></tr><tr><td rowspan=1 colspan=1>Low Positive1.5x LoD</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>90/90100%</td></tr><tr><td rowspan=1 colspan=1>True Negative</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>60/60100%</td><td rowspan=1 colspan=1>60/60100%</td><td rowspan=1 colspan=1>59/59*100%</td><td rowspan=1 colspan=1>179/179100%</td></tr><tr><td rowspan=3 colspan=1>Giardia lambliaP101</td><td rowspan=1 colspan=1>ModeratePositive3x LoD</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>90/90100%</td></tr><tr><td rowspan=1 colspan=1>Low Positive1.5x LoD</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>29/3095%</td><td rowspan=1 colspan=1>89/9098%</td></tr><tr><td rowspan=1 colspan=1>True Negative</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>60/60100%</td><td rowspan=1 colspan=1>60/60100%</td><td rowspan=1 colspan=1>59/59*100%</td><td rowspan=1 colspan=1>179/179100%</td></tr></table>

\* One True Negative sample was invalid on the EntericBio FastFinder plugin and subsequently removed from study

In addition, Cq values generated by the EntericBio FastFinder plugin were used to determine the mean, standard deviation and $\%$ coefficient of variation $( \% \mathrm { C V } )$ for all three testing sites and target analyte concentrations. A summary of these results can be found in Table 2.

Table 2: Reproducibility study, quantitative Cq results   

<table><tr><td rowspan=1 colspan=1>r able z. Reproducl</td><td rowspan=1 colspan=1>blley Study</td><td rowspan=1 colspan=2>qualtltative Cg</td><td rowspan=1 colspan=1></td><td></td><td></td><td></td></tr><tr><td rowspan=2 colspan=1>MicroorganismTested</td><td rowspan=2 colspan=1>Assay</td><td rowspan=2 colspan=2>ConcentrationTested</td><td rowspan=2 colspan=1>Test Site</td><td rowspan=1 colspan=3>Cq Reproducibility</td></tr><tr><td rowspan=1 colspan=1>Mean(Cq)</td><td rowspan=1 colspan=1>STDEV</td><td rowspan=1 colspan=1>%CV</td></tr><tr><td rowspan=8 colspan=1>Shigella sonneiDSM 5570</td><td rowspan=8 colspan=1>Well A</td><td rowspan=4 colspan=2>ModeratePositive3x LoD</td><td rowspan=1 colspan=1>Serosep 1</td><td rowspan=1 colspan=1>27.10</td><td rowspan=1 colspan=1>±0.68</td><td rowspan=1 colspan=1>2.51</td></tr><tr><td rowspan=1 colspan=1>Serosep 2</td><td rowspan=1 colspan=1>26.84</td><td rowspan=1 colspan=1>±0.47</td><td rowspan=1 colspan=1>1.74</td></tr><tr><td rowspan=1 colspan=1>Serosep 3</td><td rowspan=1 colspan=1>27.01</td><td rowspan=1 colspan=1>±0.79</td><td rowspan=1 colspan=1>2.91</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>26.98</td><td rowspan=1 colspan=1>±0.66</td><td rowspan=1 colspan=1>2.44</td></tr><tr><td rowspan=4 colspan=2>Low Positive1.5x LoD</td><td rowspan=1 colspan=1>Serosep 1</td><td rowspan=1 colspan=1>29.07</td><td rowspan=1 colspan=1>±0.57</td><td rowspan=1 colspan=1>1.97</td></tr><tr><td rowspan=1 colspan=1>Serosep 2</td><td rowspan=1 colspan=1>28.88</td><td rowspan=1 colspan=1>±0.53</td><td rowspan=1 colspan=1>1.85</td></tr><tr><td rowspan=1 colspan=1>Serosep 3</td><td rowspan=1 colspan=1>28.95</td><td rowspan=1 colspan=1>±0.57</td><td rowspan=1 colspan=1>1.98</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>28.97</td><td rowspan=1 colspan=1>±0.57</td><td rowspan=1 colspan=1>1.97</td></tr><tr><td rowspan=8 colspan=1>VibrioparahaemolyticusCCUG 14474</td><td rowspan=8 colspan=1>Well B</td><td rowspan=4 colspan=2>ModeratePositive3x LoD</td><td rowspan=1 colspan=1>Serosep 1</td><td rowspan=1 colspan=1>31.73</td><td rowspan=1 colspan=1>±0.42</td><td rowspan=1 colspan=1>1.34</td></tr><tr><td rowspan=1 colspan=1>e</td><td rowspan=1 colspan=1>Serosep 2</td><td rowspan=1 colspan=1>31.87</td><td rowspan=1 colspan=1>±0.24</td><td rowspan=1 colspan=1>0.74</td></tr><tr><td rowspan=1 colspan=1>Serosep 3</td><td rowspan=1 colspan=1>31.74</td><td rowspan=1 colspan=1>±0.44</td><td rowspan=1 colspan=1>1.40</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>31.79</td><td rowspan=1 colspan=1>±0.38</td><td rowspan=1 colspan=1>1.21</td></tr><tr><td rowspan=4 colspan=2>Low Positive1.5x LoD</td><td rowspan=1 colspan=1>Serosep 1</td><td rowspan=1 colspan=1>33.50</td><td rowspan=1 colspan=1>±0.48</td><td rowspan=1 colspan=1>1.44</td></tr><tr><td rowspan=1 colspan=1>Serosep 2</td><td rowspan=1 colspan=1>33.34</td><td rowspan=1 colspan=1>±0.50</td><td rowspan=1 colspan=1>1.51</td></tr><tr><td rowspan=1 colspan=1>Serosep 3</td><td rowspan=1 colspan=1>33.30</td><td rowspan=1 colspan=1>±0.79</td><td rowspan=1 colspan=1>2.37</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>33.38</td><td rowspan=1 colspan=1>±0.61</td><td rowspan=1 colspan=1>1.82</td></tr><tr><td rowspan=8 colspan=1>Giardia lambliaP101</td><td rowspan=8 colspan=1>Well C</td><td rowspan=4 colspan=2>ModeratePositive3x LoD</td><td rowspan=1 colspan=1>Serosep 1</td><td rowspan=1 colspan=1>32.96</td><td rowspan=1 colspan=1>±0.85</td><td rowspan=1 colspan=1>2.58</td></tr><tr><td rowspan=1 colspan=1>Serosep 2</td><td rowspan=1 colspan=1>33.04</td><td rowspan=1 colspan=1>±1.36</td><td rowspan=1 colspan=1>4.12</td></tr><tr><td rowspan=1 colspan=1>Serosep 3</td><td rowspan=1 colspan=1>33.30</td><td rowspan=1 colspan=1>±1.27</td><td rowspan=1 colspan=1>3.81</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>33.10</td><td rowspan=1 colspan=1>±1.17</td><td rowspan=1 colspan=1>3.53</td></tr><tr><td rowspan=4 colspan=2>Low Positive1.5x LoD</td><td rowspan=1 colspan=1>Serosep 1</td><td rowspan=1 colspan=1>33.98</td><td rowspan=1 colspan=1>±1.18</td><td rowspan=1 colspan=1>3.49</td></tr><tr><td rowspan=1 colspan=1>Serosep 2</td><td rowspan=1 colspan=1>34.40</td><td rowspan=1 colspan=1>±1.38</td><td rowspan=1 colspan=1>4.01</td></tr><tr><td rowspan=1 colspan=1>Serosep 3</td><td rowspan=1 colspan=1>33.11</td><td rowspan=1 colspan=1>±1.26</td><td rowspan=1 colspan=1>3.80</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>33.85</td><td rowspan=1 colspan=1>±1.37</td><td rowspan=1 colspan=1>4.05</td></tr></table>

# b. Linearity/assay reportable range:

Not applicable as the EntericBio Dx assay is a qualitative assay.

c. Traceability, Stability, Expected values (controls, calibrators, or methods):

Internal Amplification Control (IAC)

The IAC is lyophilized into each of reaction wells of the EntericBio Dx strip and is resuspended automatically on the EntericBio Workstation. The IAC is included to monitor for PCR inhibition and reagent/amplification failure for each specimen tested. Specimens cannot be considered as negative without a valid result for the IAC in the sample well. Specimens that are negative for assay targets and which also have a failed IAC in the corresponding well should be retested from the original Cary-Blair preserved fecal sample in accordance with the assay instructions for use. A specimens with a failed IAC is reported as invalid by the assay software.

# EntericBio Positive Control

Each box of the EntericBio Dx kit is supplied with a Positive Control. The Positive Control, containing nucleic acids for all assay target analytes, is provided in a lyophilized form and is reconstituted prior to its first use. The Positive Control is tested with each batch of specimens processed with the EntericBio Dx assay to monitor the PCR amplification process for potential reagent and/or process failures. If the Positive Control is negative for any of the assay target analytes, the EntericBio FastFinder plugin will report the result as invalid and the run must be repeated.

# Negative Control:

Each EntericBio Dx assay kit includes EntericBio Resuspension Buffer which is used to re-suspend the lyophilized PCR reagents and is also used as a negative (No Template) control for the assay. Fresh negative control is prepared for each run of the EntericBio Dx assay by transferring an aliquot of the EntericBio Resuspension Buffer to an empty EntericBio SPS tube. This negative control is used to monitor for any carry-over/contamination of the run. If the negative control is positive for any of the assay targets, the EntericBio FastFinder plugin will report the result as invalid and the run must be repeated.

# External controls:

Because the EntericBio Dx positive and negative kit controls do not undergo the heat lysis steps of the assay, external positive and negative specimen controls were also included in each test run performed during the clinical study. Each targeted analyte was included in the external control scheme with one analyte included per control and using a rotation of one positive control (analyte) per run.

Clinical study, control results

A total of 276 positive and 276 negative EntericDx kit controls were tested in the clinical study to monitor the performance of the EntericBio Dx assay. Valid expected results were observed for 272 $9 8 . 6 \%$ and 269 $( 9 7 . 5 \% )$ for the positive and negative kit controls respectively.

In addition, a total of 276 external positive controls and 276 external negative specimen controls were tested in the clinical study. Valid expected results were observed for 270 $( 9 7 . 8 \% )$ and 272 $( 9 8 . 6 \% )$ for the positive and negative external controls respectively.

# Specimen Stability

A study was conducted to validate appropriate specimen storage conditions for the EntericBio Dx assay. Samples evaluated included one strain for microorganism target and were prepared by spiking negative Cary-Blair preserved stool specimens with microorganisms at low positive concentrations. All replicates were tested for each panel such that a baseline reading, Time 0, was taken prior to storage at $2 { - } 8 ^ { \circ } \mathrm { C }$ . Sample replicates were also tested at days three, five, and six.

Results for each specific storage condition were considered acceptable if at least nine of the ten replicates were positive for the expected target analyte. As shown in Table 3 below, results of the study met the pre-determined acceptance criteria for claimed specimen storage conditions (up to 5 five days at $2 { - } 8 ^ { \circ } \mathrm { C }$ ). It was noted however that testing of samples with Giardia and Entamoeba histolytica generated higher Cq values after storage. Therefore the labeling indicates to test specimens as soon as possible after collection and if required, Cary-Blair preserved specimens can be stored for up to 5 days at $2 { - } 8 \mathrm { { ^ \circ C } }$ prior to testing. In addition, the following limitation has been added to the assay product labelling:

“Where possible, specimens should be tested immediately upon receipt in the laboratory. There is an increased potential for false negative results for Giardia lamblia and Entamoeba histolytica in specimens with low analyte concentrations if specimens are not tested immediately.”

Table 3. Specimen stability results, storage at 2-8°C   

<table><tr><td rowspan=2 colspan=1>Analyte</td><td rowspan=1 colspan=2>Day 0</td><td rowspan=1 colspan=2>Day 3</td><td rowspan=1 colspan=2>Day 5</td><td rowspan=1 colspan=2>Day 6</td></tr><tr><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>Neg</td><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>Neg</td><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>Neg</td><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>Neg</td></tr><tr><td rowspan=1 colspan=1>Salmonellaenterica</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>0/10</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>0/10</td><td rowspan=1 colspan=1>9/10</td><td rowspan=1 colspan=1>1/10</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>0/10</td></tr><tr><td rowspan=1 colspan=1>Shigella sonnei</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>0/10</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>0/10</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>0/10</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>0/10</td></tr><tr><td rowspan=1 colspan=1>Vibrioparahaemolyticus</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>0/10</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>0/10</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>0/10</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>0/10</td></tr><tr><td rowspan=1 colspan=1>Giardia lamblia</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>0/10</td><td rowspan=1 colspan=1>9/10</td><td rowspan=1 colspan=1>1/10</td><td rowspan=1 colspan=1>9/10</td><td rowspan=1 colspan=1>1/10</td><td rowspan=1 colspan=1>9/10</td><td rowspan=1 colspan=1>1/10</td></tr><tr><td rowspan=1 colspan=1>Campylobacterjejuni</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>0/10</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>0/10</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>0/10</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>0/10</td></tr><tr><td rowspan=1 colspan=1>Escherichia coliSTEC</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>0/10</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>0/10</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>0/10</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>0/10</td></tr><tr><td rowspan=1 colspan=1>Entamoebahistolytica</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>0/10</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>0/10</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>0/10</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>0/10</td></tr></table>

Storage of inoculated EntericBio SPS tubes prior to heat-treatment:

A study was conducted to evaluate specimen stability of inoculated EntericBio SPS specimens prior to heat-treatment. Three representative targeted microorganisms were spiked at low positive concentrations into pre-screened negative Cary-Blair preserved stool. Testing included a total of 10 replicates per microorganism and three temperature storage conditions. All replicates were detected for each microorganism evaluated. when inoculated SPS samples were stored at $2 { - } 8 ^ { \circ } \mathrm { C }$ , $2 0 \mathrm { { } ^ { \circ } C }$ and $3 0 \mathrm { { } ^ { \circ } C }$ for nine hours prior to heat treatment on the EntericBio Heatstation. Additionally, there were no significant differences in Cq values between initial testing and after storage of samples at either of the three storage temperatures evaluated. The study results support the package insert instructions indicating that inoculated EntericBio SPS tubes can be stored at room temperature prior to heat-treatment but must be processed within six hours of inoculation.

Storage of specimens in heat-treated EntericBio SPS tubes:

A study was conducted to evaluate specimen stability of inoculated EntericBio SPS specimens post heat-treatment and prior to loading of PCR wells using the EntericBio Workstation. Three representative targeted microorganisms were spiked at low positive concentrations into pre-screened negative Cary-Blair preserved stool. Testing included a total of 10 replicates per microorganism and three temperature storage conditions. All replicates were detected for each of the microorganisms evaluated. when inoculated heat treated SPS samples were stored at $2 { - } 8 ^ { \circ } \mathrm { C }$ , $2 0 \mathrm { { } ^ { \circ } C }$ and $3 0 \mathrm { { } ^ { \circ } C }$ for nine hours prior to loading PCR wells using the EntericBio Workstation. Additionally, there were no significant differences in $\mathrm { C q }$ values between initial testing and after storage of samples at the three temperature conditions evaluated. The study results support the package insert instructions indicating that inoculated EntericBio SPS tubes can be stored at $2 { - } 3 0 ^ { \circ } \mathrm { C }$ after heat-treatment for up to 2 hours before

processing on the EntericBio Workstation.

Stability of loaded EntericBio Dx PCR strips:

A study was conducted to evaluate the stability of inoculated PCR strips when there is a delay in loading onto the ABI 7500 Fast Dx instrument. For this study, three representative targeted microorganisms were spiked at low positive concentrations into pre-screened negative Cary-Blair preserved stool. A total of 10 replicates were tested per organism and timeframe evaluated. All targets tested were detected in $100 \%$ of replicates that were evaluated after a one hour delay on the EntericBio Dx Workstation after loading of PCR strips. Furthermore, there were no significant differences in $\mathrm { C q }$ values generated between initial testing of samples and after the one hour delay on the Workstation and prior to loading on the ABI 7500 Fast Dx instrument. The results of the study support the package insert instructions indicating that loaded PCR strips must be process on the ABI 7500 Fast Dx instrument within 20 minutes.

Heat station timing: The package insert instructions indicate that inoculated SPS tubes should be treated on the Heat Station for 30 minutes $\pm 5$ mins. A study was conducted to test the effect of heat-treatment duration for inoculated samples processed on the EntericBio Heatstation. The procedure outlined in the instructions for use was followed except inoculated SPS were heat-treated for either 20, 30 or 40 minutes at $1 0 3 \mathrm { { } ^ { \circ } C }$ before processing on the EntericBio Workstation. Samples were prepared using three representative targeted microorganisms spiked at low positive concentrations into pre-screened negative Cary-Blair preserved stool. A total of 10 replicates were tested per microorganism for each timepoint evaluated. All targets tested were detected in $100 \%$ of replicates tested for 20 minutes, 30 minutes and 40 minutes on the EntericBio Heatstation. Results of the study support the package insert instructions for heat-treatment of inoculated SPS specimens for $3 0 \mathrm { m i n s } \pm 5$ mins.

# d. Limit of Detection (LoD):

The analytical sensitivity (LoD, limit of detection) of the EntericBio Dx assay was determined using contrived samples with target analytes spiked into a negative stool matrix (Cary-Blair preserved stool) at three concentrations: greater than estimated LoD (High), estimated LoD (Medium) and less than LoD (Low). A total of 20 replicate SPS tubes were tested per level using three assay lots for a total of 60 replicates per microorganism concentration. If $100 \%$ of replicates were positive at the initially lowest concentration, additional testing was performed at a lower concentration and the LoD was determined as the lowest analyte concentration that was detected $2 9 5 \%$ of the time. A minimum of two strains were tested for each EntericBio Dx analyte. Table 4 lists the LoDs determined for the strains evaluated in the study.

Table 4: LoDs for EntericBio Dx analytes   

<table><tr><td rowspan=1 colspan=1>Microorganism</td><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>LoD</td></tr><tr><td rowspan=2 colspan=1>Salmonella spp.</td><td rowspan=1 colspan=1>Salmonella enterica serovar Enteritidis DSM 17420</td><td rowspan=1 colspan=1>8 x 104 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Salmonella enterica serovar Typhi NCTC 10787</td><td rowspan=1 colspan=1>8 x 104 CFU/mL</td></tr><tr><td rowspan=2 colspan=1>Shigellaspp./EIEC</td><td rowspan=1 colspan=1>Shigella sonnei DSM 5570</td><td rowspan=1 colspan=1>1.25 x 104 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>E. coli (ipaH) DSM 9029</td><td rowspan=1 colspan=1>1 x 104 CFU/mL</td></tr><tr><td rowspan=3 colspan=1>Campylobacterspp.</td><td rowspan=1 colspan=1>Campylobacter jejuni ATCC 33560</td><td rowspan=1 colspan=1>4 x 104 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Campylobacter coli DSM 4689</td><td rowspan=1 colspan=1>4 x 104 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Campylobacter lari DSM 11375</td><td rowspan=1 colspan=1>1 x 104 CFU/mL</td></tr><tr><td rowspan=2 colspan=1>Escherichia coli(STEC)</td><td rowspan=1 colspan=1>E. coli (stx1) NVRL 15x23 RE-008 (Serotype0111:H-</td><td rowspan=1 colspan=1>5 x 105 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>E. coli (stx2) NVRL 15x24 RE-006 (Serotype026:H11)</td><td rowspan=1 colspan=1>1 x 106 CFU/mL</td></tr><tr><td rowspan=2 colspan=1>Vibrio spp.</td><td rowspan=1 colspan=1>Vibrio parahaemolyticus CCUG 14474</td><td rowspan=1 colspan=1>1 x 104 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Vibrio cholerae NCTC 3661</td><td rowspan=1 colspan=1>1 x 104 CFU/mL</td></tr><tr><td rowspan=2 colspan=1>Giardia lamblia</td><td rowspan=1 colspan=1>Giardia intestinalis (WB) ATCC 30957</td><td rowspan=1 colspan=1>25 cells/mL</td></tr><tr><td rowspan=1 colspan=1>Giardia intestinalis (New Orleans) ATCC 50137</td><td rowspan=1 colspan=1>100 cells/mL</td></tr><tr><td rowspan=2 colspan=1>Entamoebahistolytica</td><td rowspan=1 colspan=1>Entamoeba histolytica (HM-1: IMSS) ATCC 30459</td><td rowspan=1 colspan=1>25 cells/mL</td></tr><tr><td rowspan=1 colspan=1>Entamoeba histolytica (HK-9) ATCC 30015</td><td rowspan=1 colspan=1>100 cells/mL</td></tr></table>

# e. Analytical Reactivity/Inclusivity:

The analytical reactivity (inclusivity) of the EntericBio Dx assay was evaluated with a collection of 101 microorganisms that represent the analytes detected by the assay. Samples were prepared using quantified whole microorganism preparations that were spiked into negative stool matrix (Cary-Blair preserved) at low positive concentrations of approximately $3 \mathbf { x }$ LoD for each assay target. Each sample was multi-spiked with up to three microorganisms, typically with one targeted analyte per each of the three multiplex assays of the EntericBio Dx assay. Testing was performed in triplicate.

The expected positive results were generated for all strains/replicates evaluated in the study demonstrating inclusivity of the EntericBio Dx assay. The strains evaluated are shown in Table 5. Also included in the table are four $E .$ . coli serotypes that are predicted to be positive for STEC $( s t x 1 / s t x 2 )$ based on in silico analyses.

Table 5: Strains evaluated in Inclusivity Study   

<table><tr><td colspan="1" rowspan="1">Microorganism</td><td colspan="1" rowspan="1">StrainIdentification</td><td colspan="1" rowspan="1">Microorganism</td><td colspan="1" rowspan="1">StrainIdentification</td></tr><tr><td colspan="1" rowspan="1">Salmonella enterica serovarEnteritidis</td><td colspan="1" rowspan="1">DSM 17420</td><td colspan="1" rowspan="1">Escherichia coli EIEC(serotype O144:H-) (ipaH)2</td><td colspan="1" rowspan="1">DSM 9029</td></tr><tr><td colspan="1" rowspan="1">Salmonella enterica serovar Typhi</td><td colspan="1" rowspan="1">NCTC 10787</td><td colspan="1" rowspan="1">Campylobacter jejuni</td><td colspan="1" rowspan="1">ATCC 33560</td></tr><tr><td colspan="1" rowspan="1">Salmonella enterica subsp. enterica Iserovar Cholerasuis</td><td colspan="1" rowspan="1">ATCC 7001</td><td colspan="1" rowspan="1">Campylobacter jejuni subsp.jejuni</td><td colspan="1" rowspan="1">DSM 104743</td></tr><tr><td colspan="1" rowspan="1">Salmonella enterica subsp. enterica Iserovar Paratyphi B</td><td colspan="1" rowspan="1">ATCC 8759</td><td colspan="1" rowspan="1">Campylobacter jejuni subsp.jejuni</td><td colspan="1" rowspan="1">DSM 27585</td></tr><tr><td colspan="1" rowspan="1">Salmonella enterica subsp. enterica Iserovar Paratyphi A</td><td colspan="1" rowspan="1">ATCC 9281</td><td colspan="1" rowspan="1">Campylobacter jejuni subsp.doylei</td><td colspan="1" rowspan="1">DSM 104768</td></tr><tr><td colspan="1" rowspan="1">Salmonella enterica subsp. enterica Iserovar Paratyphi C</td><td colspan="1" rowspan="1">ATCC 13428</td><td colspan="1" rowspan="1">Campylobacter jejuni subsp.doylei</td><td colspan="1" rowspan="1">NCTC 12208</td></tr><tr><td colspan="1" rowspan="1">Salmonella enterica subsp. enterica Iserovar Typhimurium</td><td colspan="1" rowspan="1">DSM 101475</td><td colspan="1" rowspan="1">Campylobacter coli</td><td colspan="1" rowspan="1">DSM 110395</td></tr><tr><td colspan="1" rowspan="1">Salmonella enterica subsp. enterica Iserovar Dublin</td><td colspan="1" rowspan="1">DSM 102345</td><td colspan="1" rowspan="1">Campylobacter coli</td><td colspan="1" rowspan="1">DSM 24155</td></tr><tr><td colspan="1" rowspan="1">Salmonella enterica subsp. enterica Iserovar Agona</td><td colspan="1" rowspan="1">DSM 102864</td><td colspan="1" rowspan="1">Campylobacter coli</td><td colspan="1" rowspan="1">DSM 24106</td></tr><tr><td colspan="1" rowspan="1">Salmonella enterica subsp. enterica Iserovar Heidelberg</td><td colspan="1" rowspan="1">DSM 9379</td><td colspan="1" rowspan="1">Campylobacter coli</td><td colspan="1" rowspan="1">DSM 24206</td></tr><tr><td colspan="1" rowspan="1">Salmonella enterica subsp. enterica Iserovar Infantis</td><td colspan="1" rowspan="1">NCTC 10679</td><td colspan="1" rowspan="1">Campylobacter coli</td><td colspan="1" rowspan="1">DSM 4689</td></tr><tr><td colspan="1" rowspan="1">Salmonella enterica subsp. enterica Iserovar Thompson</td><td colspan="1" rowspan="1">NCTC 2252</td><td colspan="1" rowspan="1">Campylobacter lari</td><td colspan="1" rowspan="1">DSM 11375</td></tr><tr><td colspan="1" rowspan="1">Salmonella enterica subsp. enterica Iserovar Oranienburg</td><td colspan="1" rowspan="1">NCTC 5743</td><td colspan="1" rowspan="1">Campylobacter lari</td><td colspan="1" rowspan="1">NCTC 12892</td></tr><tr><td colspan="1" rowspan="1">Salmonella enterica subsp. enterica Iserovar Bareilly</td><td colspan="1" rowspan="1">NCTC 5745</td><td colspan="1" rowspan="1">Campylobacter lari</td><td colspan="1" rowspan="1">NCTC 12893</td></tr><tr><td colspan="1" rowspan="1">Salmonella enterica subsp. enterica Iserovar Montevideo</td><td colspan="1" rowspan="1">NCTC 5747</td><td colspan="1" rowspan="1">Campylobacter lari</td><td colspan="1" rowspan="1">NCTC 12894</td></tr><tr><td colspan="1" rowspan="1">Salmonella enterica subsp. enterica Iserovar Braenderup</td><td colspan="1" rowspan="1">NCTC 5750</td><td colspan="1" rowspan="1">Campylobacter lari</td><td colspan="1" rowspan="1">NCTC 12895</td></tr><tr><td colspan="1" rowspan="1">Salmonella enterica subsp. enterica Iserovar Muenchen</td><td colspan="1" rowspan="1">NCTC 5755</td><td colspan="1" rowspan="1">Vibrio parahaemolyticus</td><td colspan="1" rowspan="1">DSM 101031</td></tr><tr><td colspan="1" rowspan="1">Salmonella enterica subsp. enterica Iserovar Saintpaul</td><td colspan="1" rowspan="1">NCTC 6022</td><td colspan="1" rowspan="1">Vibrio parahaemolyticus</td><td colspan="1" rowspan="1">DSM 11058</td></tr><tr><td colspan="1" rowspan="1">Salmonella enterica subsp. enterica Iserovar Mississippi</td><td colspan="1" rowspan="1">NCTC 6487</td><td colspan="1" rowspan="1">Vibrio parahaemolyticus</td><td colspan="1" rowspan="1">DSM 15477</td></tr><tr><td colspan="1" rowspan="1">Salmonella enterica subsp. enterica Iserovar Javiana</td><td colspan="1" rowspan="1">NCTC 6495</td><td colspan="1" rowspan="1">Vibrio parahaemolyticus</td><td colspan="1" rowspan="1">DSM 27657</td></tr><tr><td colspan="1" rowspan="1">Salmonella enterica subsp. enterica Iserovar Newport</td><td colspan="1" rowspan="1">NCTC 6704</td><td colspan="1" rowspan="1">Vibrio parahaemolyticus</td><td colspan="1" rowspan="1">CCUG 14474</td></tr><tr><td colspan="1" rowspan="1">Salmonella enterica subsp. enterica Iserovar Schwarzengrund</td><td colspan="1" rowspan="1">NCTC 6756</td><td colspan="1" rowspan="1">Vibrio cholerae (0:1 Ogawaclassical</td><td colspan="1" rowspan="1">NCTC 3661</td></tr><tr><td colspan="1" rowspan="1">Salmonella enterica subsp. enterica Iserovar Hadar</td><td colspan="1" rowspan="1">NCTC 9877</td><td colspan="1" rowspan="1">Vibrio cholerae O: 1 Biotype ElTor</td><td colspan="1" rowspan="1">NCTC 8457</td></tr><tr><td colspan="1" rowspan="1">Salmonella enterica subsp. enterica Iserovar 4, [5] 12:i:-</td><td colspan="1" rowspan="1">NSSLRLMs170397</td><td colspan="1" rowspan="1">Vibrio cholerae O:1 Ogawa</td><td colspan="1" rowspan="1">NCTC 8021</td></tr><tr><td colspan="1" rowspan="1">Salmonella enterica subsp. II(salame)</td><td colspan="1" rowspan="1">DSM 9220</td><td colspan="1" rowspan="1">Vibrio cholerae non-O:1, non-0139 (0:3)</td><td colspan="1" rowspan="1">NCTC 11502</td></tr><tr><td colspan="1" rowspan="1">Salmonella enterica subsp. IIIa(arizonae)</td><td colspan="1" rowspan="1">DSM 9386</td><td colspan="1" rowspan="1">Escherichia coli O157 (stx2)</td><td colspan="1" rowspan="1">NVRL 17X01RE-001</td></tr><tr><td colspan="1" rowspan="1">Salmonella enterica subsp. IIIb(diarizonae)</td><td colspan="1" rowspan="1">DSM 14847</td><td colspan="1" rowspan="1">Escherichia coli O157 (stxl &amp;stx2)</td><td colspan="1" rowspan="1">NVRL17X04RE-002</td></tr><tr><td colspan="1" rowspan="1">Salmonella enterica subsp. IV(houtenae)</td><td colspan="1" rowspan="1">DSM 9221</td><td colspan="1" rowspan="1">Escherichia coli O157 (stx2)</td><td colspan="1" rowspan="1">NVRL 17X09RE-003</td></tr><tr><td colspan="1" rowspan="1">Salmonella enterica subsp. VI(indica)</td><td colspan="1" rowspan="1">DSM 14848</td><td colspan="1" rowspan="1">Escherichia coli O157 (stx1 &amp;stx2</td><td colspan="1" rowspan="1">NVRL 17X15RE-004</td></tr><tr><td colspan="1" rowspan="1">Shigella sonnei</td><td colspan="1" rowspan="1">DSM 5570</td><td colspan="1" rowspan="1">Escherichia coli O157 (stx2)</td><td colspan="1" rowspan="1">NVRL 17S110RE-005</td></tr><tr><td colspan="1" rowspan="1">Shigella sonnei</td><td colspan="1" rowspan="1">DSM 25715</td><td colspan="1" rowspan="1">Escherichia coli O103:H2(stx1)</td><td colspan="1" rowspan="1">NVRL 17X128RE-010</td></tr><tr><td colspan="1" rowspan="1">Shigella sonnei</td><td colspan="1" rowspan="1">ATCC 11060</td><td colspan="1" rowspan="1">Escherichia coli O111:H8(stx1)</td><td colspan="1" rowspan="1">NVRL 13S5371RE-009</td></tr><tr><td colspan="1" rowspan="1">Shigella sonnei</td><td colspan="1" rowspan="1">ATCC 25931</td><td colspan="1" rowspan="1">Escherichia coli O121:H19(stx2)</td><td colspan="1" rowspan="1">NVRL 15X18RE-007</td></tr><tr><td colspan="1" rowspan="1">Shigella sonnei</td><td colspan="1" rowspan="1">ATCC 9290</td><td colspan="1" rowspan="1">Escherichia coli O157:NM</td><td colspan="1" rowspan="1">NVRL O6-CC3RE-011</td></tr><tr><td colspan="1" rowspan="1">Shigella flexneri (serotype 2a)</td><td colspan="1" rowspan="1">DSM 4782</td><td colspan="1" rowspan="1">Escherichia coli O157:H7 (stxl&amp; stx2</td><td colspan="1" rowspan="1">NCTC 12079NVRL RE-012</td></tr><tr><td colspan="1" rowspan="1">Shigella flexneri (serotype 2a)</td><td colspan="1" rowspan="1">ATCC 700930</td><td colspan="1" rowspan="1">Escherichia coli O157:H-(stx2)</td><td colspan="1" rowspan="1">NCTC 12080</td></tr><tr><td colspan="1" rowspan="1">Shigella flexneri (serotype la)</td><td colspan="1" rowspan="1">ATCC 9199</td><td colspan="1" rowspan="1">Escherichia coli O111:H-(stx1)</td><td colspan="1" rowspan="1">NVRL 15x23RE-008</td></tr><tr><td colspan="1" rowspan="1">Shigella flexneri (serotype b)</td><td colspan="1" rowspan="1">ATCC 12022</td><td colspan="1" rowspan="1">Escherichia coli O26:H11(stx2)</td><td colspan="1" rowspan="1">NVRL 15x24RE-006</td></tr><tr><td colspan="1" rowspan="1">Shigella flexneri (serotype)</td><td colspan="1" rowspan="1">ATCC 12025</td><td colspan="1" rowspan="1">Escherichia coli O1131</td><td colspan="1" rowspan="1">N/A</td></tr><tr><td colspan="1" rowspan="1">Shigella boydii (serotype 2)</td><td colspan="1" rowspan="1">DSM 7532</td><td colspan="1" rowspan="1">Escherichia coli O451</td><td colspan="1" rowspan="1">N/A</td></tr><tr><td colspan="1" rowspan="1">Shigella boydii (serotype 1)</td><td colspan="1" rowspan="1">ATCC 9207</td><td colspan="1" rowspan="1">Escherichia coli O1041</td><td colspan="1" rowspan="1">N/A</td></tr><tr><td colspan="1" rowspan="1">Shigella boydii (serotype 20)</td><td colspan="1" rowspan="1">ATCC BAA-1247</td><td colspan="1" rowspan="1">Escherichia coli O1451</td><td colspan="1" rowspan="1">N/A</td></tr><tr><td colspan="1" rowspan="1">Shigella boydii (serotype 10)</td><td colspan="1" rowspan="1">ATCC 12030</td><td colspan="1" rowspan="1">Giardia intestinalis (WB)</td><td colspan="1" rowspan="1">ATCC 30957</td></tr><tr><td colspan="1" rowspan="1">Shigella boydii (serotype 4)</td><td colspan="1" rowspan="1">ATCC 9210</td><td colspan="1" rowspan="1">Giardia intestinalis (New-Orleans-1)</td><td colspan="1" rowspan="1">ATCC 50137</td></tr><tr><td colspan="1" rowspan="1">Shigella dysenteriae (serotype 1)</td><td colspan="1" rowspan="1">NCTC 4837</td><td colspan="1" rowspan="1">Giardia intestinalis GSAssemblage B</td><td colspan="1" rowspan="1">ATCC 50581</td></tr><tr><td colspan="1" rowspan="1">Shigella dysenteriae (serotype 2)</td><td colspan="1" rowspan="1">NCTC 5109</td><td colspan="1" rowspan="1">Giardia intestinalis Portland 1</td><td colspan="1" rowspan="1">ATCC 30888</td></tr><tr><td colspan="1" rowspan="1">Shigella dysenteriae (serotype 7)</td><td colspan="1" rowspan="1">NCTC 9763</td><td colspan="1" rowspan="1">Giardia intestinalis Mario</td><td colspan="1" rowspan="1">ATCC PRA-244</td></tr><tr><td colspan="1" rowspan="1">Shigella dysenteriae (serotype 3)</td><td colspan="1" rowspan="1">NCTC 6340</td><td colspan="1" rowspan="1">Entamoeba histolytica (HM-1:IMSS)</td><td colspan="1" rowspan="1">ATCC 30459</td></tr><tr><td colspan="1" rowspan="1">Shigella dysenteriae (serotype 9)</td><td colspan="1" rowspan="1">NCTC 9347</td><td colspan="1" rowspan="1">Entamoeba histolytica (HK-9)</td><td colspan="1" rowspan="1">ATCC 30015</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli EIEC (serotypeO28ac:H-)</td><td colspan="1" rowspan="1">DSM 9025</td><td colspan="1" rowspan="1">Entamoeba histolytica HB-301:NIH</td><td colspan="1" rowspan="1">ATCC 30190</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli EIEC (serotypeO29:H10</td><td colspan="1" rowspan="1">DSM 9026</td><td colspan="1" rowspan="1">Entamoeba histolytica HU-21:AMC</td><td colspan="1" rowspan="1">ATCC 30457</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli EIEC (serotypeO136:H-)</td><td colspan="1" rowspan="1">DSM 9032</td><td colspan="1" rowspan="1">Entamoeba histolyticaIP:1182:2</td><td colspan="1" rowspan="1">ATCC PRA-357</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli EIEC (serotype01:30</td><td colspan="1" rowspan="1">DSM 9031</td><td colspan="2" rowspan="1"></td></tr></table>

1 Inclusivity predicted based on in-silico analysis 2 EIEC strain which also generates positive result for Shigella/EIEC (ipaH)

# f. Analytical specificity (exclusivity/cross-reactivity):

High concentrations of non-target microorganisms were evaluated for potential crossreactivity with the EntericBio Dx assay targets. The microorganisms evaluated consisted of those closely related to the EntericBio Dx assay targets, non-targeted microorganisms that are known enteric pathogens as well as microorganisms commonly present in human stool. For the most strains evaluated, contrived

specimens were prepared in a negative stool matrix (i.e., Cary-Blair preserved stool) by spiking microorganisms at high concentrations of greater than $1 0 ^ { \bar { 6 } } \mathrm { C F U / m L }$ or $1 0 ^ { 6 }$ cells/mL. Nucleic acid preparations were used to prepare samples at concentrations of $1 0 ^ { 6 }$ genomic copies $/ \mathrm { m L }$ for microorganisms that are difficult to culture or for which it is challenging to achieve high concentrations of whole microorganisms. Testing of each microorganism was performed in triplicate. Samples that were negative for $3 / 3$ replicates were considered to be non-cross-reactive between the assay and the microorganisms.

The majority of microorganisms listed in Table 6 generated the expected negative results for all three replicates tested. For microorganisms that were unexpectedly positive, additional in silico analysis and bi-directional sequencing was conducted to evaluate the potential reason(s) for cross-reactivity.

Microorganisms evaluated in the study are listed in Table 6 and those demonstrating cross-reactivity are listed in Table 7.

Table 6: Cross-reactivity Study, microorganisms evaluated   

<table><tr><td colspan="1" rowspan="1">Microorganism</td><td colspan="1" rowspan="1">Microorganism</td><td colspan="1" rowspan="1">Microorganism</td><td colspan="1" rowspan="1">Microorganism</td></tr><tr><td colspan="1" rowspan="1">Yersinia aldovae</td><td colspan="1" rowspan="1">Cryptosporidiummeleagridis</td><td colspan="1" rowspan="1">Citrobacter koseri</td><td colspan="1" rowspan="1">Pseudomonas putidal</td></tr><tr><td colspan="1" rowspan="1">Yersinia bercovieri</td><td colspan="1" rowspan="1">Cryptosporidium Skunkgenotypel</td><td colspan="1" rowspan="1">Clostridium difficile</td><td colspan="1" rowspan="1">Rahnella aquatilis</td></tr><tr><td colspan="1" rowspan="1">Yersiniaentomophaga</td><td colspan="1" rowspan="1">Cryptosporidiumubiquitum</td><td colspan="1" rowspan="1">Clostridium perfringens</td><td colspan="1" rowspan="1">Rotavirus A1</td></tr><tr><td colspan="1" rowspan="1">Yersinia frederiksenii</td><td colspan="1" rowspan="1">Cryptosporidiumviaorum2</td><td colspan="1" rowspan="1">Clostridium sordelli</td><td colspan="1" rowspan="1">Ruminococcus gauvreauii</td></tr><tr><td colspan="1" rowspan="1">Yersinia intermedia</td><td colspan="1" rowspan="1">Escherichia vulneris</td><td colspan="1" rowspan="1">Cronobacter sakazakii</td><td colspan="1" rowspan="1">Saccharomyces cerevisiae</td></tr><tr><td colspan="1" rowspan="1">Yersinia kristensenii</td><td colspan="1" rowspan="1">Cryptosporidiumcuniculus13</td><td colspan="1" rowspan="1">Dientamoeba fragilis</td><td colspan="1" rowspan="1">Sapovirus1</td></tr><tr><td colspan="1" rowspan="1">Yersinia massiliensis</td><td colspan="1" rowspan="1">Cryptosporidium Horsegenotype</td><td colspan="1" rowspan="1">Edwardsiella tarda</td><td colspan="1" rowspan="1">Serratia liquefaciens</td></tr><tr><td colspan="1" rowspan="1">Yersinia mollaretti</td><td colspan="1" rowspan="1">Cryptosporidium felis</td><td colspan="1" rowspan="1">Encephalitozoon cuniculil</td><td colspan="1" rowspan="1">Serratia marcescens</td></tr><tr><td colspan="1" rowspan="1">Yersinia nurmii</td><td colspan="1" rowspan="1">Cryptosporidium canis</td><td colspan="1" rowspan="1">Enterobacter aerogenes</td><td colspan="1" rowspan="1">Serratia odoriferae</td></tr><tr><td colspan="1" rowspan="1">Yersinia pekkaneneii</td><td colspan="1" rowspan="1">Cryptosporidium xiaoil 4</td><td colspan="1" rowspan="1">Enterobacter cloacae</td><td colspan="1" rowspan="1">Serratia rubidaea</td></tr><tr><td colspan="1" rowspan="1">Yersiniapseudotuberculosis</td><td colspan="1" rowspan="1">Cryptosporidiumandersonil</td><td colspan="1" rowspan="1">Enterococcus faecalis</td><td colspan="1" rowspan="1">Staphylococcus aureus</td></tr><tr><td colspan="1" rowspan="1">Yersinia rodhei</td><td colspan="1" rowspan="1">Cryptosporidium baileyil</td><td colspan="1" rowspan="1">Enterococcus faecium</td><td colspan="1" rowspan="1">Staphylococcus epidermidis</td></tr><tr><td colspan="1" rowspan="1">Yersinia ruckeri</td><td colspan="1" rowspan="1">Entamoeba dispar5</td><td colspan="1" rowspan="1">Eubacterium rectalel 6</td><td colspan="1" rowspan="1">Stenotrophomonasmaltophilia</td></tr><tr><td colspan="1" rowspan="1">Yersinia similis</td><td colspan="1" rowspan="1">Entamoeba moshkovoskii</td><td colspan="1" rowspan="1">Ewingella americana</td><td colspan="1" rowspan="1">Streptococcus agalactiae</td></tr><tr><td colspan="1" rowspan="1">Vibrio alginolyticus</td><td colspan="1" rowspan="1">Entamoeba invadens</td><td colspan="1" rowspan="1">Fusobacteriumgonidiaformans</td><td colspan="1" rowspan="1">Streptococcus bovis</td></tr><tr><td colspan="1" rowspan="1">Vibrio fluvialis</td><td colspan="1" rowspan="1">Aeromonas hydrophilia</td><td colspan="1" rowspan="1">Fusobacterium nucleatum</td><td colspan="1" rowspan="1">Streptococcus equinus</td></tr><tr><td colspan="1" rowspan="1">Vibrio furnissii</td><td colspan="1" rowspan="1">Adenovirus F401</td><td colspan="1" rowspan="1">Fusobacterium varium</td><td colspan="1" rowspan="1">Toxoplasma gondii</td></tr><tr><td colspan="1" rowspan="1">Vibrio mimicus</td><td colspan="1" rowspan="1">Alcaligenes faecalis</td><td colspan="1" rowspan="1">Hafnia alvei</td><td colspan="1" rowspan="1">Campylobacter concisus1</td></tr><tr><td colspan="1" rowspan="1">Vibrio harvei</td><td colspan="1" rowspan="1">Anaerococcushydrogenalis</td><td colspan="1" rowspan="1">Klebsiella oxytoca</td><td colspan="1" rowspan="1">Campylobacter curvus1</td></tr><tr><td colspan="1" rowspan="1">Vibrio fischeril</td><td colspan="1" rowspan="1">Anaerostipes hadrus7</td><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">Campylobacter fetus subsp.Fetus</td></tr><tr><td colspan="1" rowspan="1">Vibrio damsela</td><td colspan="1" rowspan="1">Arcobacter butzleri</td><td colspan="1" rowspan="1">Lactobacillus acidophilus</td><td colspan="1" rowspan="1">Campylobacter fetus subsp.</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Venerealis</td></tr><tr><td colspan="1" rowspan="1">Griomontia hollisae</td><td colspan="1" rowspan="1">Astrovirus1</td><td colspan="1" rowspan="1">Lactobacillus lactis</td><td colspan="1" rowspan="1">Campylobacter gracilis</td></tr><tr><td colspan="1" rowspan="1">Vibrio diazotrophicus</td><td colspan="1" rowspan="1">Bacillus cereus</td><td colspan="1" rowspan="1">Listeria monocytogenes</td><td colspan="1" rowspan="1">Campylobacter helveticus1</td></tr><tr><td colspan="1" rowspan="1">Vibrio proteolyticus</td><td colspan="1" rowspan="1">Bacillus subtilis subspsubtilis</td><td colspan="1" rowspan="1">Morganella morganii</td><td colspan="1" rowspan="1">Campylobacter hominis</td></tr><tr><td colspan="1" rowspan="1">Vibrio natrigens</td><td colspan="1" rowspan="1">Bacillus subtilis subspspizizenii</td><td colspan="1" rowspan="1">Neisseria gonorrhoeae</td><td colspan="1" rowspan="1">Campylobacterhyointestinalis</td></tr><tr><td colspan="1" rowspan="1">Vibrio pelagius1</td><td colspan="1" rowspan="1">Bacteroides fragilis</td><td colspan="1" rowspan="1">Norovirus GGI1</td><td colspan="1" rowspan="1">Campylobacter mucosalis1</td></tr><tr><td colspan="1" rowspan="1">Vibrio campbellii</td><td colspan="1" rowspan="1">Bacteroidesthetaiotaomicron</td><td colspan="1" rowspan="1">Norovirus GGII</td><td colspan="1" rowspan="1">Campylobacter rectus1</td></tr><tr><td colspan="1" rowspan="1">Vibrio vulnificus8</td><td colspan="1" rowspan="1">Bifidobacterium breve</td><td colspan="1" rowspan="1">Plesiomonas shigelloides</td><td colspan="1" rowspan="1">Campylobacter showae</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli nontoxigenic</td><td colspan="1" rowspan="1">Bifidobacterium longum</td><td colspan="1" rowspan="1">Prevotella melaninogenica</td><td colspan="1" rowspan="1">Campylobacter sputorum1 9</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli(Enteropathogenic)</td><td colspan="1" rowspan="1">Blastocystis hominis</td><td colspan="1" rowspan="1">Proteus mirabilis</td><td colspan="1" rowspan="1">Campylobacter ureolyticus</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli(Enterotoxigenic)</td><td colspan="1" rowspan="1">Candida albicans</td><td colspan="1" rowspan="1">Proteus vulgaris</td><td colspan="1" rowspan="1">Salmonella bongori</td></tr><tr><td colspan="1" rowspan="1">Escherichiahermannii</td><td colspan="1" rowspan="1">Citrobacteramalonaticusl</td><td colspan="1" rowspan="1">Providencia stuartii</td><td colspan="1" rowspan="1">Salmonella subterranea</td></tr><tr><td colspan="1" rowspan="1">Escherichia blattae</td><td colspan="1" rowspan="1">Citrobacter freundii</td><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">Campylobacter upsaliensis</td></tr><tr><td colspan="1" rowspan="1">Escherichia</td><td colspan="3" rowspan="1"></td></tr></table>

2Cryptosporidium viatorum: Shigella detected in 3/3 replicates. Bi-directional sequencing demonstrated the presence of Shigella DNA suggesting possible sample contamination with Shigella. In silico analysis indicated no cross-reactivity should occur.

3 Cryptosporidium cuniculus: Giardia detected in 2/3 replicates. Bi-directional sequencing of amplicon failed and thus it could not be confirmed empirically if the sample was contaminated with Giardia or if cross reactivity-occurred. In silico analysis indicated no cross-reactivity should occur.

4 Cryptosporidium xiaoi: Campylobacter detected in 3/3 replicates. Bi-directional sequencing demonstrated the presence of Campylobacter DNA suggesting possible sample contamination with Campylobacter. In silico analysis indicated no crossreactivity should occur.

5 Entamoeba dispar: IAC failures were observed for $5 / 6$ replicates after repeat testing. In silico analysis demonstrated homology for the primers and predicts amplification to occur and competitively inhibit the IAC, but the probe has several mismatches and is not expected to produce a detected signal.

6 Eubacterium rectale: Campylobacter detected in 3/3 replicates and Vibrio in 1/3 replicates. Bi-directional sequencing demonstrated the presence of Campylobacter and Vibrio DNA suggesting possible sample contamination with Vibrio and Campylobacter. In silico analysis indicated no cross-reactivity should occur.

Anaerostipes hadrus: Campylobacter detected in 2/3 replicates. Bi-directional sequencing demonstrated the presence of Campylobacter DNA suggesting possible sample contamination with Campylobacter. In silico analysis indicated no crossreactivity should occur.

8 Vibrio vulnificus: Evaluated by in silico analysis only. In silico analysis indicated no cross-reactivity should occur. 9 Campylobacter sputorum: Vibrio detected in 1/3 replicates. Bi-directional sequencing demonstrated the presence of Vibrio DNA suggesting possible sample contamination with Vibrio. In silico analysis indicated no cross-reactivity should occur.

Table 7: Cross-reactive microorganisms   

<table><tr><td rowspan=1 colspan=1>Cross ReactingMicroorganisms</td><td rowspan=1 colspan=1>Target Analyte for which Cross-reactivity wasobserved</td></tr><tr><td rowspan=1 colspan=1>Vibrio campbellii</td><td rowspan=1 colspan=1>Vibrio spp. (cholerae and parahaemolyticus)</td></tr><tr><td rowspan=1 colspan=1>Vibrio fluvialis</td><td rowspan=1 colspan=1>Vibrio spp. (cholerae and parahaemolyticus)</td></tr><tr><td rowspan=1 colspan=1>Vibrio furnissii</td><td rowspan=1 colspan=1>Vibrio spp. (cholerae and parahaemolyticus)</td></tr><tr><td rowspan=1 colspan=1>Vibrio mimicus</td><td rowspan=1 colspan=1>Vibrio spp. (cholerae and parahaemolyticus)</td></tr><tr><td rowspan=1 colspan=1>Vibrio fischeri</td><td rowspan=1 colspan=1>Vibrio spp. (cholerae and parahaemolyticus)</td></tr><tr><td rowspan=1 colspan=1>Vibrio natriegens</td><td rowspan=1 colspan=1>Vibrio spp. (cholerae and parahaemolyticus)</td></tr></table>

g. Assay cut-off:

The EntericBio Dx assay performed on the ABI 7500 Fast Dx instrument uses FastFinder software with EntericBio Dx plug-in to generate final results. The software uses proprietary decision algorithms to convert ABI 7500 Fast Dx raw data and translates these results into final qualitative results. The decision algorithms used to generate results are used for all controls and patient specimens. Any specimens with failed IACs or indeterminate final results are reported as invalid to the user. The final results reported to the end user are positive, negative or invalid.

# h. Microbial Interference

A study was conducted to assess non-targeted microorganisms for their potential to interfere with detection of microorganisms that are targeted by the EntericBio Dx assay. A total of 11 potentially interfering microorganisms were tested at high concentrations (i.e., $\mathrm { 1 0 ^ { 6 } C F U / m L }$ in the presence of EntericBio Dx target analytes at low concentrations. Samples were prepared using quantitated whole microorganism preparations of target (Table 8) and non-target (Table 9) microorganisms spiked into a negative stool specimen matrix. Testing of each microorganism combination was performed in triplicate. Control samples containing only potentially interfering microorganisms were also tested.

Table 8: EntericBio Dx target analytes evaluated   

<table><tr><td rowspan=1 colspan=1>Panel</td><td rowspan=1 colspan=1>Microorganism</td></tr><tr><td rowspan=1 colspan=1>Panel 1</td><td rowspan=1 colspan=1>Salmonella enterica subsp. Enteritidis</td></tr><tr><td rowspan=3 colspan=1>Panel 2</td><td rowspan=1 colspan=1>Shigella sonnei</td></tr><tr><td rowspan=1 colspan=1>Vibrio parahaemolyticus</td></tr><tr><td rowspan=1 colspan=1>Giardia intestinalis</td></tr><tr><td rowspan=2 colspan=1>Panel 3</td><td rowspan=1 colspan=1>Campylobacter jejuni</td></tr><tr><td rowspan=1 colspan=1>Entamoeba histolytica</td></tr><tr><td rowspan=1 colspan=1>Panel 4</td><td rowspan=1 colspan=1>Escherichia coli (STEC)</td></tr></table>

Table 9: Potentially interfering microorganisms evaluated   

<table><tr><td rowspan=1 colspan=1>Substance</td><td rowspan=1 colspan=1>Source/ID</td><td rowspan=1 colspan=1>Concentration tested</td></tr><tr><td rowspan=1 colspan=1>Aeromonas hydrophila</td><td rowspan=1 colspan=1>DSM 17695</td><td rowspan=1 colspan=1>106 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Bacteroides fragilis</td><td rowspan=1 colspan=1>DSM 2151</td><td rowspan=1 colspan=1>106CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus aureus</td><td rowspan=1 colspan=1>DSM 20231</td><td rowspan=1 colspan=1>106CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Escherichia coli</td><td rowspan=1 colspan=1>DSM 30083</td><td rowspan=1 colspan=1>106CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Enterococcus faecalis</td><td rowspan=1 colspan=1>DSM 20478</td><td rowspan=1 colspan=1>106CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Clostridium perfringens</td><td rowspan=1 colspan=1>DSM 798</td><td rowspan=1 colspan=1>106CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Saccharomyces cerevisiae</td><td rowspan=1 colspan=1>DSM 1848</td><td rowspan=1 colspan=1>106 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Blastocystis hominis</td><td rowspan=1 colspan=1>Clinical specimen</td><td rowspan=1 colspan=1>Not quantifiable</td></tr><tr><td rowspan=1 colspan=1>Pseudomonas aeruginosa</td><td rowspan=1 colspan=1>DSM 50071</td><td rowspan=1 colspan=1>106 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Klebsiella oxytoca</td><td rowspan=1 colspan=1>DSM 5175</td><td rowspan=1 colspan=1>106 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Candida albicans</td><td rowspan=1 colspan=1>DSM 1577</td><td rowspan=1 colspan=1>106 CFU/mL</td></tr></table>

Results from the study showed no microbial interference with any of the 11 nontargeted microorganisms evaluated as there were no false negative results for targeted microorganisms. Additionally, there were no false positive results or IAC failures observed in control samples which contained only non-targeted microorganisms.

# i. Interfering Substances

A study was conducted to evaluate the performance of the EntericBio Dx assay for samples containing 23 potentially interfering substances, at high, but clinically relevant levels that might be present in fecal specimens. Each substance was tested in the presence of each target analyte (Table 8 above) at low positive concentrations. Samples were prepared using quantified microorganism preparations and each potentially interfering substance spiked into negative stool matrix. Testing was performed in triplicate for each microorganism/substance combination. Control samples containing each potentially interfering substance in a negative stool matrix without targeted analytes were also tested.

Results of the study demonstrated that the presence of 21/23 potentially interfering substances at the concentrations listed in Table 10 below had no effect on the detection of the target analytes or the EntericBio Dx assay internal control. A false negative result for $V .$ . parahaemolyticus was observed for 1/3 sample replicates containing hemorrhoidal cream $( 1 \% \mathrm { w / v } )$ and false negative results for $C .$ . jejuni were observed for 2/3 sample replicates containing tetracycline $( 1 . 6 \% \ \mathrm { w / v ) }$ .

For the following substances, interference was observed for samples containing higher concentrations than are listed in Table 10 below.

Benzalkonium chloride (Initial testing at $5 0 \% \mathrm { v / v } )$ :

For samples without target microorganism, initial testing for samples with Benzalkonium chloride $( 5 0 \% \mathrm { v / v } )$ generated failed IACs for all three replicates. The substance was then evaluated at progressively lower

concentrations. A concentration of $1 . 0 \%$ v/v was the highest concentration for which the IAC was detected for all replicates.

For samples with targeted microorganism, the presence of this substance at a concentration greater than $0 . 1 5 \%$ v/v generated false negative results for one or more targets.

Hemorrhoidal cream (Initial testing at $5 0 \% \mathrm { v / v ) }$ :

For samples without targeted microorganism, initial testing for samples with Hemorrhoidal cream $( 5 0 \% \mathrm { v / v } )$ generated failed IACs for all three replicates. The substance was then evaluated at progressively lower concentrations. A concentration of $1 . 0 \%$ w/v was the highest concentration for which the IAC was detected for all replicates. • For samples with targeted microorganism, the presence of this substance at a concentration greater than $1 \% \mathrm { v / v }$ demonstrated false negative results for one or more targets.

Whole Human Blood: For samples with targeted microorganism, the presence of human whole blood at concentrations ${ > } 5 \% \ \mathrm { v } / \mathrm { v }$ generated false negative results for one or more targets.

Magnesium Hydroxide: For samples with targeted microorganism, the presence of magnesium hydroxide at concentrations greater than $0 . 5 \%$ w/v generated false negative results for one or more targets.

Tetracycline: For samples with targeted microorganisms, the presence of tetracycline at concentrations of ${ > } 0 . 8 \%$ w/v generated false negative results for one or more targets.

Sulfamethoxazole: For samples with targeted microorganism, the presence of sulfamethoxazole at concentrations greater than $0 . 5 \%$ w/v generated false negative results for one or more targets.

Loperamide hydrochloride: For samples with targeted microorganism, the presence of loperamide hydrochloride at concentrations greater than $4 \%$ w/v demonstrated false negative results for one or more targets.

Amoxicillin: For samples with targeted microorganism, the presence of amoxicillin at concentrations greater than $1 \% \mathrm { w / v }$ generated false negative results for one or more targets.

Table 10: Highest concentration for which most EntericBio Dx target analytes were detected: Substances for which interference was observed at the concentration listed are bolded   

<table><tr><td rowspan=1 colspan=1>Substance</td><td rowspan=1 colspan=1>Concentration</td></tr><tr><td rowspan=1 colspan=1>Amoxicillin</td><td rowspan=1 colspan=1>1% w/v</td></tr><tr><td rowspan=1 colspan=1>Benzalkonium Chloride</td><td rowspan=1 colspan=1>0.15% v/v</td></tr><tr><td rowspan=1 colspan=1>Ceftriaxone</td><td rowspan=1 colspan=1>1 % w/v</td></tr><tr><td rowspan=1 colspan=1>Cholesterol</td><td rowspan=1 colspan=1>7 % w/v</td></tr><tr><td rowspan=1 colspan=1>Ciprofloxacin</td><td rowspan=1 colspan=1>5.4% w/v</td></tr><tr><td rowspan=1 colspan=1>Erythromycin</td><td rowspan=1 colspan=1>1.5% w/v</td></tr><tr><td rowspan=1 colspan=1>Hemorrhoidal cream</td><td rowspan=1 colspan=1>1% w/v1</td></tr><tr><td rowspan=1 colspan=1>Human DNA</td><td rowspan=1 colspan=1>0.1% v/v</td></tr><tr><td rowspan=1 colspan=1>Hydrocortisone</td><td rowspan=1 colspan=1>50% w/v</td></tr><tr><td rowspan=1 colspan=1>Laxative</td><td rowspan=1 colspan=1>5% v/v</td></tr><tr><td rowspan=1 colspan=1>Loperamide Hydrochloride</td><td rowspan=1 colspan=1>0.5% w/v</td></tr><tr><td rowspan=1 colspan=1>Lubricant</td><td rowspan=1 colspan=1>50% w/v</td></tr><tr><td rowspan=1 colspan=1>Magnesium Hydroxide</td><td rowspan=1 colspan=1>0.5% w/v</td></tr><tr><td rowspan=1 colspan=1>Metronidazole</td><td rowspan=1 colspan=1>6% w/v</td></tr><tr><td rowspan=1 colspan=1>Mucin</td><td rowspan=1 colspan=1>10% w/v</td></tr><tr><td rowspan=1 colspan=1>Naproxen sodium (NSAID)</td><td rowspan=1 colspan=1>10% w/v</td></tr><tr><td rowspan=1 colspan=1>Nystatin Cream</td><td rowspan=1 colspan=1>50% w/v</td></tr><tr><td rowspan=1 colspan=1>Sudocrem (antiseptic healing cream)</td><td rowspan=1 colspan=1>50% w/v</td></tr><tr><td rowspan=1 colspan=1>Sulfamethoxazole</td><td rowspan=1 colspan=1>4% w/v</td></tr><tr><td rowspan=1 colspan=1>Tetracycline</td><td rowspan=1 colspan=1>0.8% w/v2</td></tr><tr><td rowspan=1 colspan=1>Trimethoprim</td><td rowspan=1 colspan=1>1.6% w/v</td></tr><tr><td rowspan=1 colspan=1>Vagisil</td><td rowspan=1 colspan=1>50% w/v</td></tr><tr><td rowspan=1 colspan=1>Whole Human Blood</td><td rowspan=1 colspan=1>5% v/v</td></tr></table>

1 V. parahaemolyticus failed in 1/3 replicates at this concentration $^ { 2 } C .$ . jejuni was only tested at $1 . 6 \%$ w/v and 2/3 replicates failed at this concentration

# j. Competitive Inhibition

A study was conducted to evaluate the potential for competitive inhibition between analytes targeted by the EntericBio Dx assay. Testing included samples containing two microorganisms detected in PCR Well A (Salmonella and Campylobacter) and two microorganisms detected PCR Well C (Giardia and Entamoeba). The analytes in Well B (Vibrio cholerae/parahaemolyticus and STEC) were not evaluated in this study; however, co-infections of these two analytes are expected to be rare.

The study was performed using Cary-Blair preserved negative stool samples spiked with dual EntericBio Dx analyte combinations at high and low concentrations (Table 11). Testing was performed in triplicate and generated expected results for all microorganism combinations except for samples containing Giardia lamblia and Entamoeba histolytica (bolded in Table 11). False negative results for Giardia lamblia were observed for all sample replicates containing either low or high concentrations of Entamoeba histolytica. A limitation indicating this interference is also included in the limitation section of the package insert for the EntericBio Dx assay.

Table 11: Competitive Inhibition Study, microorganism combinations   

<table><tr><td rowspan=1 colspan=1>EntericBio Dx Target Analyte Combinations</td></tr><tr><td rowspan=1 colspan=1>Salmonella High (300X LoD) &amp; Campylobacter Low (3X LoD)</td></tr><tr><td rowspan=1 colspan=1>Salmonella Low (18X LoD) &amp; Campylobacter High (50X LoD)</td></tr><tr><td rowspan=1 colspan=1>Shigella High (200X LoD) &amp; Giardia Low (3X LoD)</td></tr><tr><td rowspan=1 colspan=1>Shigella Low (12X LoD) &amp; Giardia High (50X LoD)</td></tr><tr><td rowspan=1 colspan=1>Campylobacter High (50X LoD) &amp; Giardia Low (3X LoD)</td></tr><tr><td rowspan=1 colspan=1>Campylobacter Low (3X LoD) &amp; Giardia High (50X LoD)</td></tr><tr><td rowspan=1 colspan=1>Entamoeba High (50X LoD) &amp; Giardia Low (3X LoD)1</td></tr><tr><td rowspan=1 colspan=1>Entamoeba Low (3X LoD) &amp; Giardia High (50X LoD)1</td></tr></table>

1 All replicates were negative for Giardia and positive for E. histolytica

# k. Carryover/Cross-Contamination:

A study was conducted to investigate the potential for carryover or crosscontamination when using the EntericBio Dx assay on the EntericBio Workstation. In each run, samples with a high concentration of target microorganism(s) were processed in an alternating sequence with negative samples. Contrived samples were prepared in a negative stool matrix (Cary-Blair preserved stool) with one bacterial analyte (Shigella sonnei) and one parasitic target (Giardia lamblia). Microorganisms were spiked at a high concentrations ( $\mathrm { 1 x 1 0 ^ { 7 } C F U / m L }$ and $1 \mathrm { ~ x ~ } 1 0 ^ { 5 } \mathrm { c e l l s / m L }$ for Shigella sonnei and Giardia lamblia respectively). Negative samples consisted of negative stool matrix. The study consisted of six runs with three runs containing Shigella sonnei and three runs containing Giardia lamblia samples. Each run contained 30 samples (15 positive and 15 negative). Results from the study demonstrated no false positive results and therefore no evidence of carryover/contamination within run or between runs when using the EntericBio Dx assay on the EntericBio Workstation.

2. Comparison studies:

a. Method comparison with predicate device: N/A

b. Matrix comparison:

N/A

3. Clinical studies:

Performance characteristics of the EntericBio Dx assay were established in a multi-site clinical study with testing conducted at two US sites and one non-US site.

The performance of the EntericBio Dx assay was evaluated using the following specimen cohorts:

1) Fresh, prospectively-collected, Cary-Blair preserved fecal specimens from patients presenting with symptoms of gastrointestinal infection;   
2) Fresh, selected Cary-Blair preserved fecal specimens from patients presenting with symptoms of gastrointestinal infection. Specimens were selected based on the results obtained with Standard of Care (SoC) methods in use at the study sites;   
3) Frozen, well characterized Cary-Blair preserved fecal specimens from patients presenting with symptoms of gastrointestinal infection, known to be positive for selected target analytes based on routine diagnostic methods used by the collection sites;   
4) Contrived Cary-Blair preserved fecal samples for lower prevalence targets

The performance of the EntericBio Dx assay in fresh prospectively tested fresh specimens, selected fresh specimens and archived specimens was evaluated by comparing the EntericBio Dx assay result for each target analyte with results obtained from FDA-cleared assays. For the STEC and Campylobacter analytes, a composite comparator method of three FDA-cleared assays was used for the fresh prospective and fresh selected specimen cohorts. A specimen was characterised as positive if two out of three comparator assays were positive and a specimen was characterized as negative if two out of three comparator assays were negative. For the remaining analytes (fresh specimens) and for all analytes (archived specimens), the comparator method consisted of one FDA-cleared assay.

A total of 1523 fresh Cary-Blair fecal specimens (1491 prospective, 32 select) were enrolled during the clinical trial. Of the 1491 prospective samples, 19 were excluded from the study, giving 1472 evaluable samples. The reasons for exclusion included invalid test result for the EntericBio Dx assay $( \mathrm { n } { = } 9 )$ or by the comparator method $( \mathrm { n } { = } 6 )$ , Indeterminate result by the EntericBio Dx assay $( \mathrm { n } { = } 1 )$ ), specimen not tested with comparator method within specimen stability $\scriptstyle ( \mathtt { n } = 2 )$ ) or a duplicate specimen from a previously enrolled patient $\scriptstyle ( \mathrm { n = 1 } )$ ).

Table 12 provides a summary of demographic information for the fresh specimens enrolled in the clinical study.

Table 12: Demographic summary, fresh specimens $\scriptstyle ( \mathbf { N } = \mathbf { 1 } 5 2 3 )$ )   

<table><tr><td rowspan=1 colspan=1>Gender</td><td rowspan=1 colspan=1>Number of specimens (%)</td></tr><tr><td rowspan=1 colspan=1>Male</td><td rowspan=1 colspan=1>636 (41.8%)</td></tr><tr><td rowspan=1 colspan=1>Female</td><td rowspan=1 colspan=1>880 (57.8%)</td></tr><tr><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>7 (0.4%)</td></tr><tr><td rowspan=1 colspan=1>Age Group</td><td rowspan=1 colspan=1>Number of specimens (%)</td></tr><tr><td rowspan=1 colspan=1>&lt;l year</td><td rowspan=1 colspan=1>11 (0.7%)</td></tr><tr><td rowspan=1 colspan=1>1-5 years</td><td rowspan=1 colspan=1>65 (4.3%)</td></tr><tr><td rowspan=1 colspan=1>6-12 years</td><td rowspan=1 colspan=1>27 (1.8%)</td></tr><tr><td rowspan=1 colspan=1>13-21 years</td><td rowspan=1 colspan=1>62 (4.1%)</td></tr><tr><td rowspan=1 colspan=1>22-65 years</td><td rowspan=1 colspan=1>863 (56.7%)</td></tr><tr><td rowspan=1 colspan=1>+ 65 years</td><td rowspan=1 colspan=1>489 (32.1%)</td></tr><tr><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>6 (0.4%)</td></tr><tr><td rowspan=1 colspan=1>Patient Status</td><td rowspan=1 colspan=1>Number of specimens (%)</td></tr><tr><td rowspan=1 colspan=1>Outpatient</td><td rowspan=1 colspan=1>692 (45.4%)</td></tr><tr><td rowspan=1 colspan=1>In-patient</td><td rowspan=1 colspan=1>700 (46.0%)</td></tr><tr><td rowspan=1 colspan=1>Emergency Care</td><td rowspan=1 colspan=1>94 (6.2%)</td></tr><tr><td rowspan=1 colspan=1>Long termcare</td><td rowspan=1 colspan=1>22 (1.4%)</td></tr><tr><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>15 (1.0%)</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>1523</td></tr></table>

Several analytes had low prevalence for the fresh specimen cohorts; therefore, 212 archived frozen positive specimens were included in the study. Specimens consisted of a mix of Cary-Blair preserved specimens and unpreserved specimens transferred to CaryBlair upon enrollment. Specimens were selected based on historic testing results for the desired target analytes by routine diagnostic methods used by the collection sites. Although standard of care results were used for selection of specimens for inclusion in the study, assay performance was evaluated by comparing EntericBio Dx results to results from one FDA-cleared comparator assay.

Of the 212 archived specimens enrolled into the study, three specimens were excluded, resulting in 209 evaluable specimens. The reasons for exclusion were invalid $( \mathrm { n } { = } 1 )$ ) and Indeterminate results $( \mathrm { n } { = } 2 )$ for the EntericBio Dx assay.

Archived specimens were distributed across three clinical testing sites were randomized such that the users performing the testing were blinded as to the expected test result. Table 13 provides a summary of demographic information for the archived specimens enrolled in the clinical study.

Table 13: Demographic summary, Archived specimens (N-212)   

<table><tr><td rowspan=1 colspan=1>Gender</td><td rowspan=1 colspan=1>Number of specimens (%)</td></tr><tr><td rowspan=1 colspan=1>Male</td><td rowspan=1 colspan=1>105 (49.5%)</td></tr><tr><td rowspan=1 colspan=1>Female</td><td rowspan=1 colspan=1>103 (48.6%)</td></tr><tr><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>4 (1.9%)</td></tr><tr><td rowspan=1 colspan=1>Age Group</td><td rowspan=1 colspan=1>Number of specimens (%)</td></tr><tr><td rowspan=1 colspan=1>&lt;1 year</td><td rowspan=1 colspan=1>5 (2.4%)</td></tr><tr><td rowspan=1 colspan=1>1-5 years</td><td rowspan=1 colspan=1>51 (24.1%)</td></tr><tr><td rowspan=1 colspan=1>6-12 years</td><td rowspan=1 colspan=1>21 (9.9%)</td></tr><tr><td rowspan=1 colspan=1>13-21 years</td><td rowspan=1 colspan=1>14 (6.6%)</td></tr><tr><td rowspan=1 colspan=1>22-65 years</td><td rowspan=1 colspan=1>96 (45.3%)</td></tr><tr><td rowspan=1 colspan=1>+ 65 years</td><td rowspan=1 colspan=1>21 (9.9%)</td></tr><tr><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>4 (1.9%)</td></tr><tr><td rowspan=1 colspan=1>Patient Status</td><td rowspan=1 colspan=1>Number of specimens (%)</td></tr><tr><td rowspan=1 colspan=1>Outpatient</td><td rowspan=1 colspan=1>136 (64.2%)</td></tr><tr><td rowspan=1 colspan=1>In-patient</td><td rowspan=1 colspan=1>42 (19.8%)</td></tr><tr><td rowspan=1 colspan=1>Emergency Care</td><td rowspan=1 colspan=1>27 (12.7%)</td></tr><tr><td rowspan=1 colspan=1>Long term care</td><td rowspan=1 colspan=1>1 (0.5%)</td></tr><tr><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>6 (2.8%)</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>212</td></tr></table>

To supplement the fresh and frozen specimen data for very low prevalence microorganisms (Vibrio and Entamoeba histolytica), contrived specimens $\mathrm { ( n { = } } 3 1 0 \mathrm { ) }$ ) were prepared and tested with the EntericBio Dx assay. Each contrived specimen was prepared using a unique fecal matrix from residual fresh specimens which had previously tested negative for all analytes. Contrived specimens were spiked at various levels, with the majority of specimens for each analyte having microorganism concentrations near the LoD (i.e., 1.5x-3x LoD). Contrived positive specimens were randomized with negative specimens for blinding purposes. Strains used for preparation of contrived specimens are listed in Table 14.

Table 14: Strains evaluated in contrived clinical specimens   

<table><tr><td rowspan=1 colspan=1>Assay Target</td><td rowspan=1 colspan=1>Microorganism</td><td rowspan=1 colspan=1>Strain ID</td></tr><tr><td rowspan=4 colspan=1>Vibrio</td><td rowspan=1 colspan=1>Vibrio parahaemolyticus</td><td rowspan=1 colspan=1>CCUG 14474</td></tr><tr><td rowspan=1 colspan=1>Vibrio parahaemolyticus</td><td rowspan=1 colspan=1>DSM 15477</td></tr><tr><td rowspan=1 colspan=1>Vibrio cholerae O:1 biotype El Toro</td><td rowspan=1 colspan=1>NCTC 3661</td></tr><tr><td rowspan=1 colspan=1>Vibrio cholerae O:3</td><td rowspan=1 colspan=1>NCTC 11502</td></tr><tr><td rowspan=2 colspan=1>Entamoeba histolytica</td><td rowspan=1 colspan=1>Entamoeba histolytica</td><td rowspan=1 colspan=1>ATCC 30459</td></tr><tr><td rowspan=1 colspan=1>Entamoeba histolytica</td><td rowspan=1 colspan=1>ATCC 30015</td></tr></table>

Of the 310 contrived specimens evaluated in the clinical study, there were no specimens excluded from the study. Performance was evaluated by comparing the test result for each target analyte to the expected result, based on the microorganism used for spiking.

Clinical performance of the EntericBio Dx assay is presented in Tables 15-21 with results stratified by prospective fresh (all-comers), selected fresh, and contrived specimen cohorts.

<table><tr><td rowspan=1 colspan=7>Table 15: Summary of Clinical Performance for Salmonella</td></tr><tr><td rowspan=6 colspan=1>Srone</td><td rowspan=2 colspan=3>Specimen Type</td><td rowspan=2 colspan=1>n=</td><td rowspan=1 colspan=2>% Agreement (95% CI)</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td></tr><tr><td rowspan=3 colspan=1>Cit Cican</td><td rowspan=2 colspan=1>Bress</td><td rowspan=1 colspan=1>Prospective (All-Comers)</td><td rowspan=1 colspan=1>1472</td><td rowspan=1 colspan=1>92.3%24/261(75.9-97.9)</td><td rowspan=1 colspan=1>100%1446/1446(99.7-100)</td></tr><tr><td rowspan=1 colspan=1>Select</td><td rowspan=1 colspan=1>32</td><td rowspan=1 colspan=1>90.0%9/102(59.6-98.2)</td><td rowspan=1 colspan=1>100%22/22(85.1-100)</td></tr><tr><td rowspan=1 colspan=1>Rrn</td><td rowspan=1 colspan=1>Archived</td><td rowspan=1 colspan=1>209</td><td rowspan=1 colspan=1>85.7%12/143(60.1-96.0)</td><td rowspan=1 colspan=1>100%195/195(98.1-100)</td></tr><tr><td rowspan=1 colspan=3>Simulated</td><td rowspan=1 colspan=1>310</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>99.7%309/310(98.2-99.9)</td></tr></table>

12/2 Salmonella FN observed were negative for Salmonella when tested with an alternative FDA cleared PCR assay $^ { 2 } 1 / 1$ Salmonella FN observed was positive for Salmonella when tested with an alternative FDA cleared PCR assay $^ { 3 } 1 / 2$ Salmonella FN observed was negative for Salmonella when tested with an alternative FDA cleared PCR assay. 1/2 Salmonella FN observed was positive for Salmonella when tested with an alternative FDA cleared PCR assay.

<table><tr><td rowspan=1 colspan=7>Table 16: Summary of Clinical Performance for Campylobacter</td></tr><tr><td rowspan=6 colspan=1>Cr</td><td rowspan=2 colspan=3>Specimen Type</td><td rowspan=2 colspan=1>n=</td><td rowspan=1 colspan=2>% Agreement (95% CI)</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td></tr><tr><td rowspan=3 colspan=1>C Cnn</td><td rowspan=2 colspan=1>Bre</td><td rowspan=1 colspan=1>Prospective (All-Comers)</td><td rowspan=1 colspan=1>1472</td><td rowspan=1 colspan=1>98.0%50/51(89.7-99.7)</td><td rowspan=1 colspan=1>99.8%1418/1421(99.4-99.9)</td></tr><tr><td rowspan=1 colspan=1>Select</td><td rowspan=1 colspan=1>32</td><td rowspan=1 colspan=1>100%9/9(70.1-100)</td><td rowspan=1 colspan=1>100%23/23(85.7-100)</td></tr><tr><td rowspan=1 colspan=1>Rrn</td><td rowspan=1 colspan=1>Archived</td><td rowspan=1 colspan=1>209</td><td rowspan=1 colspan=1>93.8%15/161(71.7-98.9)</td><td rowspan=1 colspan=1>100%193/193(98.0-100)</td></tr><tr><td rowspan=1 colspan=3>Simulated</td><td rowspan=1 colspan=1>310</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>100%310/310(98.8-100)</td></tr></table>

11/1 Campylobacter FN observed were negative for Campylobacter when tested with an alternative FDA cleared PCR assay

<table><tr><td rowspan=1 colspan=7>Table 17: Summary of Clinical Performance for Shigella/EIEC</td></tr><tr><td rowspan=6 colspan=1>Sh/C</td><td rowspan=2 colspan=3>Specimen Type</td><td rowspan=2 colspan=1>n=</td><td rowspan=1 colspan=2>% Agreement (95% CI)</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td></tr><tr><td rowspan=3 colspan=1>Chs Ccn</td><td rowspan=2 colspan=1>rres</td><td rowspan=1 colspan=1>Prospective (All-Comers)</td><td rowspan=1 colspan=1>1472</td><td rowspan=1 colspan=1>100%14/14(78.5-100)</td><td rowspan=1 colspan=1>100%1458/1458(99.7-100)</td></tr><tr><td rowspan=1 colspan=1>Select</td><td rowspan=1 colspan=1>32</td><td rowspan=1 colspan=1>100%3/3(43.9-100)</td><td rowspan=1 colspan=1>100%29/29(88.3-100)</td></tr><tr><td rowspan=1 colspan=1>Rr0n</td><td rowspan=1 colspan=1>Archived</td><td rowspan=1 colspan=1>209</td><td rowspan=1 colspan=1>90.9%10/111(62.3-98.4)</td><td rowspan=1 colspan=1>100%198/198(98.1-100)</td></tr><tr><td rowspan=1 colspan=3>Simulated</td><td rowspan=1 colspan=1>310</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>100%310/310(98.8-100)</td></tr></table>

1 1/1 Shigella/EIEC FN observed was negative for Shigella/EIEC when tested with an alternative FDA cleared PCR assay

<table><tr><td rowspan=1 colspan=7>Table 18: Summary of Clinical Performance for STEC</td></tr><tr><td rowspan=6 colspan=1>STEE</td><td rowspan=2 colspan=3>Specimen Type</td><td rowspan=2 colspan=1>n=</td><td rowspan=1 colspan=2>% Agreement (95% CI)</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td></tr><tr><td rowspan=3 colspan=1>Cits Cn</td><td rowspan=2 colspan=1>rres</td><td rowspan=1 colspan=1>Prospective (All-Comers)</td><td rowspan=1 colspan=1>1472</td><td rowspan=1 colspan=1>100%8/8(67.6-100)</td><td rowspan=1 colspan=1>99.9%1462/1464(99.5-99.9)</td></tr><tr><td rowspan=1 colspan=1>Select</td><td rowspan=1 colspan=1>32</td><td rowspan=1 colspan=1>100%3/3(43.9-100)</td><td rowspan=1 colspan=1>100%29/29(88.3-100)</td></tr><tr><td rowspan=1 colspan=1>Rrn</td><td rowspan=1 colspan=1>Archived</td><td rowspan=1 colspan=1>209</td><td rowspan=1 colspan=1>94.6%70/741(86.9-97.9)</td><td rowspan=1 colspan=1>100%135/135(92.7-100)</td></tr><tr><td rowspan=1 colspan=3>Simulated</td><td rowspan=1 colspan=1>310</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>100%310/310(98.8-100)</td></tr></table>

14/4 STEC FN observed were negative for STEC when tested with an alternative FDA cleared PCR assay

<table><tr><td rowspan=1 colspan=7>Table 19: Summary of Clinical Performance for Vibrio</td></tr><tr><td rowspan=6 colspan=1>H</td><td rowspan=2 colspan=3>Specimen Type</td><td rowspan=2 colspan=1>n=</td><td rowspan=1 colspan=2>% Agreement (95% CI)</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td></tr><tr><td rowspan=3 colspan=1>Ccs Can</td><td rowspan=2 colspan=1>rrss</td><td rowspan=1 colspan=1>Prospective (All-Comers)</td><td rowspan=1 colspan=1>1472</td><td rowspan=1 colspan=1>0.0%0/31(0.0-56.1)</td><td rowspan=1 colspan=1>100%1469/1469(99.7-100)</td></tr><tr><td rowspan=1 colspan=1>Select</td><td rowspan=1 colspan=1>32</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>100%32/32(89.3-100)</td></tr><tr><td rowspan=1 colspan=1>Rron</td><td rowspan=1 colspan=1>Archived</td><td rowspan=1 colspan=1>209</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>100%209/209(98.2-100)</td></tr><tr><td rowspan=1 colspan=3>Simulated</td><td rowspan=1 colspan=1>310</td><td rowspan=1 colspan=1>100%100/100(96.3-100)</td><td rowspan=1 colspan=1>100%210/210(98.2-100)</td></tr></table>

13/3 Vibrio FN observed were negative for Vibrio when tested with an alternative FDA cleared PCR assay

<table><tr><td rowspan=1 colspan=7>Table 20: Summary of Clinical Performance for Giardia lamblia</td></tr><tr><td rowspan=6 colspan=1>cm </td><td rowspan=2 colspan=3>Specimen Type</td><td rowspan=2 colspan=1>n=</td><td rowspan=1 colspan=2>% Agreement (95% CI)</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td></tr><tr><td rowspan=3 colspan=1>Cits Ccen</td><td rowspan=2 colspan=1>rres</td><td rowspan=1 colspan=1>Prospective (All-Comers)</td><td rowspan=1 colspan=1>1472</td><td rowspan=1 colspan=1>85.7%12/141(60.1-96.0)</td><td rowspan=1 colspan=1>99.9%1457/1458(99.6-99.9)</td></tr><tr><td rowspan=1 colspan=1>Select</td><td rowspan=1 colspan=1>32</td><td rowspan=1 colspan=1>100%3/3(43.9-100)</td><td rowspan=1 colspan=1>100%29/29(88.3-100)</td></tr><tr><td rowspan=1 colspan=1>Rrn</td><td rowspan=1 colspan=1>Archived</td><td rowspan=1 colspan=1>209</td><td rowspan=1 colspan=1>100%29/29(88.3-100)</td><td rowspan=1 colspan=1>100%180/180(97.9-100)</td></tr><tr><td rowspan=1 colspan=3>Simulated</td><td rowspan=1 colspan=1>310</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>100%310/310(98.8-100)</td></tr></table>

12/2 Giardia FN observed were negative for Giardia when tested with an alternative FDA cleared PCR assay

<table><tr><td rowspan=1 colspan=7>Table 21: Summary of Clinical Performance for Entamoeba histolytica</td></tr><tr><td rowspan=6 colspan=1>Bheoe qoc</td><td rowspan=2 colspan=3>Specimen Type</td><td rowspan=2 colspan=1>n=</td><td rowspan=1 colspan=2>% Agreement (95% CI)</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td></tr><tr><td rowspan=3 colspan=1>Chs Cn</td><td rowspan=2 colspan=1>rres</td><td rowspan=1 colspan=1>Prospective (All-Comers)</td><td rowspan=1 colspan=1>1472</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>100%1472/1472(99.7-100)</td></tr><tr><td rowspan=1 colspan=1>Select</td><td rowspan=1 colspan=1>32</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>100%32/32(89.3-100)</td></tr><tr><td rowspan=1 colspan=1>Rr0n</td><td rowspan=1 colspan=1>Archived</td><td rowspan=1 colspan=1>209</td><td rowspan=1 colspan=1>0.0%0/21(0.0-65.8)</td><td rowspan=1 colspan=1>100%207/207(98.2-100)</td></tr><tr><td rowspan=1 colspan=3>Simulated</td><td rowspan=1 colspan=1>310</td><td rowspan=1 colspan=1>98.6%74/75(92.3-99.8)</td><td rowspan=1 colspan=1>100%235/235(98.4-100)</td></tr></table>

12/2 Entamoeba FN observed were positive for Entamoeba histolytica when tested with an alternative FDA cleared PCR

The EntericBio Dx assay reported multiple microorganism detections (co-infections) for three specimens $( 0 . 2 0 \%$ , 3/1504). All multiple detections contained two target analytes and all were concordant with the comparator method(s) used for the respective target analytes. The summary of the multi-detections reported by the EntericBio Dx assay is presented in Table 22.

For analytes reported by the EntericBio Dx assay, the comparator assay(s) reported multiple microorganism detections for a total of five specimens $( 0 . 3 3 \%$ , 5/1504). All multiple detections contained two target analytes. For these specimens the EntericBio Dx assay did not detect a second target analyte in two specimens and none of the discordant specimens reported as co-infections by the comparator method were confirmed with an alternative FDA-cleared assay. The summary of the multi-detections reported by the comparator assay is presented in Table 23.

Table 22: Multi-Detections detected by EntericBio Dx in fresh clinical specimens $\mathbf { ( N { = } 1 5 0 4 ) }$   

<table><tr><td rowspan=2 colspan=1>Analyte_1</td><td rowspan=2 colspan=1>Analyte_2</td><td rowspan=1 colspan=2>Prevalence</td><td rowspan=2 colspan=1>Number of Discrepantspecimens (FP byEntericBio)</td><td rowspan=2 colspan=1>DiscrepantAnalyte(s)</td></tr><tr><td rowspan=1 colspan=1>No.</td><td rowspan=1 colspan=1>%</td></tr><tr><td rowspan=1 colspan=1>Campylobacter</td><td rowspan=1 colspan=1>Giardia</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.07</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>N/A</td></tr><tr><td rowspan=1 colspan=1>Shigella</td><td rowspan=1 colspan=1>Giardia</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.07</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>N/A</td></tr><tr><td rowspan=1 colspan=1>Campylobacter</td><td rowspan=1 colspan=1>STEC</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.07</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>N/A</td></tr><tr><td rowspan=1 colspan=2>Total</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0.20</td><td rowspan=1 colspan=2></td></tr></table>

Table 23: Multi-Detections detected by the comparator methods in fresh clinical specimens $\scriptstyle ( \mathbf { N = 1 5 0 4 } )$   

<table><tr><td rowspan=2 colspan=1>Analyte_1</td><td rowspan=2 colspan=1>Analyte_2</td><td rowspan=1 colspan=2>Prevalence</td><td rowspan=2 colspan=1>Number ofDiscrepant specimens(FN by EntericBio)</td><td rowspan=2 colspan=1>DiscrepantAnalyte(s)</td></tr><tr><td rowspan=1 colspan=1>No.</td><td rowspan=1 colspan=1>%</td></tr><tr><td rowspan=1 colspan=1>Campylobacter</td><td rowspan=1 colspan=1>Giardia</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0.13</td><td rowspan=1 colspan=1>1a</td><td rowspan=1 colspan=1>Giardiaa</td></tr><tr><td rowspan=1 colspan=1>Shigella</td><td rowspan=1 colspan=1>Giardia</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0.13</td><td rowspan=1 colspan=1>1b</td><td rowspan=1 colspan=1>Giardiab</td></tr><tr><td rowspan=1 colspan=1>Campylobacter</td><td rowspan=1 colspan=1>STEC</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.07</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>N/A</td></tr><tr><td rowspan=1 colspan=2>Total</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>0.33</td><td rowspan=1 colspan=2></td></tr></table>

a Specimen was negative with EntericBio Dx and an alternative FDA-cleared assay b Specimen was negative with EntericBio Dx and an alternative FDA-cleared assay

Of the 1482 prospective (fresh) specimens initially evaluated with EntericBio Dx assay, 28 specimens $( 1 . 9 \% )$ initially generated Invalid results. Repeat testing of these specimens generated valid results for 19 specimens and invalid results for six specimens. Repeat testing was not performed for the remaining three specimens. None of the 32 select (fresh) specimens were initially reported as invalid. Of the 212 retrospective specimens initially evaluated with EntericBio Dx assay, two specimens $( 0 . 9 \% )$ were initially reported as Invalid. Repeat testing of one specimen generated a valid result. Repeat testing was not performed for the other specimen. Table 24 includes the initial and final invalid rates for each specimen cohort as observed during the clinical study.

Table 24: Summary of Invalid results observed during the clinical study   

<table><tr><td rowspan=2 colspan=1></td><td rowspan=1 colspan=3>Initial Invalid</td><td rowspan=1 colspan=3>Final Invalid</td></tr><tr><td rowspan=1 colspan=1>Count</td><td rowspan=1 colspan=1>Percent</td><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=1>Count</td><td rowspan=1 colspan=1>Percent</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=1>Prospective (Fresh)</td><td rowspan=1 colspan=1>28/1482</td><td rowspan=1 colspan=1>1.9%</td><td rowspan=1 colspan=1>1.3-2.7%</td><td rowspan=1 colspan=1>9*/1482</td><td rowspan=1 colspan=1>0.6%</td><td rowspan=1 colspan=1>0.3-1.2%</td></tr><tr><td rowspan=1 colspan=1>Select (Fresh)</td><td rowspan=1 colspan=1>0/32</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0.0-10.7%</td><td rowspan=1 colspan=1>0/32</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0.0-10.7%</td></tr><tr><td rowspan=1 colspan=1>Total (Fresh)</td><td rowspan=1 colspan=1>28/1514</td><td rowspan=1 colspan=1>1.9%</td><td rowspan=1 colspan=1>1.3-2.7%</td><td rowspan=1 colspan=1>9*/1514</td><td rowspan=1 colspan=1>0.6%</td><td rowspan=1 colspan=1>0.3-1.1%</td></tr><tr><td rowspan=1 colspan=1>Retrospective (Frozen)</td><td rowspan=1 colspan=1>2/212</td><td rowspan=1 colspan=1>0.9%</td><td rowspan=1 colspan=1>0.3-3.4%</td><td rowspan=1 colspan=1>1**/212</td><td rowspan=1 colspan=1>0.5%</td><td rowspan=1 colspan=1>0.1-2.6%</td></tr><tr><td rowspan=1 colspan=1>Total (All)</td><td rowspan=1 colspan=1>30/1726</td><td rowspan=1 colspan=1>1.7%</td><td rowspan=1 colspan=1>1.2-2.5%</td><td rowspan=1 colspan=1>10/1726</td><td rowspan=1 colspan=1>0.6%</td><td rowspan=1 colspan=1>0.3-1.2%</td></tr></table>

$^ { * } 1 9 / 2 8$ initial invalids for fresh specimens were resolved upon repeat; 6/28 were invalid upon repeat and 3/28 were not repeated $^ { * * } 1 / 2$ initial Invalid results was resolved upon repeat and 1/2 was not repeated

Of the 1482 prospective (fresh) specimens initially evaluated with EntericBio Dx assay, one specimen $( 0 . 1 \% )$ was initially reported as Indeterminate. Repeat testing was not performed and the specimen remained as Indeterminate. None of the 32 select (fresh) specimens initially reported as Indeterminate. Of the 212 retrospective specimens initially evaluated with EntericBio Dx assay, two specimens $( 0 . 9 \% )$ were initially reported as Indeterminate. Repeat testing for these two specimens was not performed. Table 25 includes the initial and final Indeterminate rates for each specimen cohort as observed during the clinical study.

Table 25: Summary of Indeterminate results observed during the clinical study   

<table><tr><td rowspan=2 colspan=1></td><td rowspan=1 colspan=3>Initial Indeterminate</td><td rowspan=1 colspan=3>Final Indeterminate</td></tr><tr><td rowspan=1 colspan=1>Count</td><td rowspan=1 colspan=1>Percent</td><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=1>Count</td><td rowspan=1 colspan=1>Percent</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=1>Prospective (Fresh)</td><td rowspan=1 colspan=1>1/1482</td><td rowspan=1 colspan=1>0.1%</td><td rowspan=1 colspan=1>0.0-0.4%</td><td rowspan=1 colspan=1>1*/1482</td><td rowspan=1 colspan=1>0.1%</td><td rowspan=1 colspan=1>0.0-0.4%</td></tr><tr><td rowspan=1 colspan=1>Select (Fresh)</td><td rowspan=1 colspan=1>0/32</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0.0-10.7%</td><td rowspan=1 colspan=1>0/32</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0.0-10.7%</td></tr><tr><td rowspan=1 colspan=1>Total (Fresh)</td><td rowspan=1 colspan=1>1/1514</td><td rowspan=1 colspan=1>0.1%</td><td rowspan=1 colspan=1>0.0-0.4%</td><td rowspan=1 colspan=1>1*/1514</td><td rowspan=1 colspan=1>0.1%</td><td rowspan=1 colspan=1>0.0-0.4%</td></tr><tr><td rowspan=1 colspan=1>Retrospective (Frozen)</td><td rowspan=1 colspan=1>2/212</td><td rowspan=1 colspan=1>0.9%</td><td rowspan=1 colspan=1>0.3-3.4%</td><td rowspan=1 colspan=1>2**/212</td><td rowspan=1 colspan=1>0.9%</td><td rowspan=1 colspan=1>0.3-3.4%</td></tr><tr><td rowspan=1 colspan=1>Total (All)</td><td rowspan=1 colspan=1>3/1726</td><td rowspan=1 colspan=1>0.2%</td><td rowspan=1 colspan=1>0.1-0.5%</td><td rowspan=1 colspan=1>3/1726</td><td rowspan=1 colspan=1>0.2%</td><td rowspan=1 colspan=1>0.1-0.5%</td></tr></table>

$^ { * } 1 / 1$ initial indeterminate result was not repeated $^ { * * } 2 / 2$ initial indeterminate results were not repeated

Table 26 below includes the initial and final non-reportable rates (Invalid and Indeterminate results combined) as observed in the clinical study.

Table 26: Summary of Non-reportable results observed during the clinical study (Invalid and Indeterminate results combined)   

<table><tr><td rowspan=2 colspan=1></td><td rowspan=1 colspan=3>Initial All Non-reportable</td><td rowspan=1 colspan=3>Final All Non-reportable</td></tr><tr><td rowspan=1 colspan=1>Count</td><td rowspan=1 colspan=1>Percent</td><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=1>Count</td><td rowspan=1 colspan=1>Percent</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=1>Prospective (Fresh)</td><td rowspan=1 colspan=1>29/1482</td><td rowspan=1 colspan=1>2.0%</td><td rowspan=1 colspan=1>1.4-2.8%</td><td rowspan=1 colspan=1>10/1482</td><td rowspan=1 colspan=1>0.7%</td><td rowspan=1 colspan=1>0.4-1.2%</td></tr><tr><td rowspan=1 colspan=1>Select (Fresh)</td><td rowspan=1 colspan=1>0/32</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0.0-10.7%</td><td rowspan=1 colspan=1>0/32</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0.0-10.7%</td></tr><tr><td rowspan=1 colspan=1>Total (Fresh)</td><td rowspan=1 colspan=1>29/1514</td><td rowspan=1 colspan=1>1.9%</td><td rowspan=1 colspan=1>1.3-2.7%</td><td rowspan=1 colspan=1>10*/1514</td><td rowspan=1 colspan=1>0.7%</td><td rowspan=1 colspan=1>0.4-1.2%</td></tr><tr><td rowspan=1 colspan=1>Retrospective (Frozen)</td><td rowspan=1 colspan=1>4/212</td><td rowspan=1 colspan=1>1.9%</td><td rowspan=1 colspan=1>0.7-4.8%</td><td rowspan=1 colspan=1>3**/212</td><td rowspan=1 colspan=1>1.4%</td><td rowspan=1 colspan=1>0.5-4.1%</td></tr><tr><td rowspan=1 colspan=1>Total (All)</td><td rowspan=1 colspan=1>33/1726</td><td rowspan=1 colspan=1>1.9%</td><td rowspan=1 colspan=1>1.4-2.7%</td><td rowspan=1 colspan=1>13/1726</td><td rowspan=1 colspan=1>0.8%</td><td rowspan=1 colspan=1>0.4-1.3%</td></tr></table>

$^ { * } 1 9 / 2 9$ initial non-reportable results for fresh specimens were resolved upon repeat; $6 / 2 9$ were non reportable upon repeat and 4/29 were not repeated $^ { * * } 1 / 4$ initial non-reportable results was resolved upon repeat and $3 / 4$ were not repeated

4. Clinical cut-off:

N/A

5. Expected values/Reference range:

In the prospective study clinical evaluation of the EntericBio Dx assay, a total of 1504 eligible fresh fecal specimens in Cary-Blair medium were tested at three geographically diverse locations, two in the United States (Site 1, Site 2) and one outside of the United States (Site 3). The number and percentage of positive results for each of the seven target microorganisms as determined by the EntericBio assay are stratified by testing site and presented in Table 27 below. Overall, the EntericBio assay detected at least one microorganism in $9 . 2 \%$ (138/1504) of the fresh specimens.

Table 27: Expected Values summary for fresh clinical specimens $\mathbf { ( N { = } 1 5 0 4 ) }$ by clinical test site   

<table><tr><td rowspan=2 colspan=1>EntericBio Dx Result</td><td rowspan=1 colspan=1>Site 1</td><td rowspan=1 colspan=1>Site 2</td><td rowspan=1 colspan=1>Site 3</td><td rowspan=1 colspan=1>Overall</td></tr><tr><td rowspan=1 colspan=1>604</td><td rowspan=1 colspan=1>539</td><td rowspan=1 colspan=1>361</td><td rowspan=1 colspan=1>1504</td></tr><tr><td rowspan=1 colspan=1>Salmonella</td><td rowspan=1 colspan=1>21 (3.5%)</td><td rowspan=1 colspan=1>10 (1.9%)</td><td rowspan=1 colspan=1>2 (0.6%)</td><td rowspan=1 colspan=1>33 (2.2%)</td></tr><tr><td rowspan=1 colspan=1>Campylobacter</td><td rowspan=1 colspan=1>33 (5.5%)</td><td rowspan=1 colspan=1>6 (1.1%)</td><td rowspan=1 colspan=1>23 (6.4%)</td><td rowspan=1 colspan=1>62 (4.1%)</td></tr><tr><td rowspan=1 colspan=1>Shigella/EIEC</td><td rowspan=1 colspan=1>11 (1.8%)</td><td rowspan=1 colspan=1>2 (0.4%)</td><td rowspan=1 colspan=1>4 (1.1%)</td><td rowspan=1 colspan=1>17 (1.1%)</td></tr><tr><td rowspan=1 colspan=1>STEC</td><td rowspan=1 colspan=1>7 (1.2%)</td><td rowspan=1 colspan=1>6 (1.1%)</td><td rowspan=1 colspan=1>0 (0.0%)</td><td rowspan=1 colspan=1>13 (0.9%)</td></tr><tr><td rowspan=1 colspan=1>Vibrio</td><td rowspan=1 colspan=1>0 (0.0%)</td><td rowspan=1 colspan=1>0 (0.0%)</td><td rowspan=1 colspan=1>0 (0.0%)</td><td rowspan=1 colspan=1>0 (0.0%)</td></tr><tr><td rowspan=1 colspan=1>Giardia lamblia</td><td rowspan=1 colspan=1>11 (1.8%)</td><td rowspan=1 colspan=1>4 (0.7%)</td><td rowspan=1 colspan=1>1 (0.3%)</td><td rowspan=1 colspan=1>16 (1.1%)</td></tr><tr><td rowspan=1 colspan=1>Entamoeba histolytica</td><td rowspan=1 colspan=1>0 (0.0%)</td><td rowspan=1 colspan=1>0 (0.0%)</td><td rowspan=1 colspan=1>0 (0.0%)</td><td rowspan=1 colspan=1>0 (0.0%)</td></tr></table>

# N. Proposed Labeling:

The labeling supports the finding of substantial equivalence of the device.

# O. Conclusion:

The submitted information in this premarket notification is complete and supports a substantial equivalence decision.