# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATION DECISION SUMMARY

New device 510(k) clearance for the FluChip-8G Influenza $\mathbf { A } { + } \mathbf { B }$ Assay

# C. Measurand:

Influenza A and influenza B viral nucleic acids.

# D. Type of Test:

Qualitative multiplex one-step RT-PCR followed by downstream microarray-based hybridization, imaging, and subsequent influenza virus detection and characterization using a pattern recognition-based algorithm

# E. Applicant:

InDevR, Inc.

FluChip-8G Influenza $\mathbf { A } { + } \mathbf { B }$ Assay (FC8G assay)

# G. Regulatory Information:

1. Regulation section: 21 CFR 866.3980, Respiratory viral panel multiplex nucleic acid assay

2. Classification: Class II

3. Product code(s):

OZE - Influenza A and influenza B multiplex nucleic acid assay (Primary) NSU - Instrumentation for clinical multiplex test systems (Subsequent) OEP - Influenza A virus subtype differentiation nucleic acid assay (Subsequent) OQW - 2009 H1N1 influenza virus (swine origin), nucleic acid or antigen, detection and

# identification (Subsequent)

4. Panel:

Microbiology (83)

# H. Intended Use:

1. Intended use(s):

The FluChip-8G Influenza $\mathbf { A } { + } \mathbf { B }$ Assay is a multiplex RT-PCR in vitro diagnostic test intended for the qualitative detection and differentiation of seasonal influenza A/H3N2, seasonal influenza A/H1N1pdm09, and “non-seasonal” influenza A subtypes other than seasonal H1N1pdm09 or H3N2. The assay is also intended for the qualitative detection and differentiation of the genetic lineage of human influenza B viruses as B/Victoria or B/Yamagata. The assay is designed for use on influenza nucleic acids isolated and purified from nasopharyngeal swab and nasal swab specimens from human patients with signs and symptoms of respiratory infection in conjunction with clinical and epidemiological risk factors.

This assay amplifies the hemagglutinin (HA) gene segment, neuraminidase (NA) gene segment, matrix (M) gene segment, non-structural (NS) gene segment, and nucleoprotein (NP) gene segment for detection and discrimination of influenza A, and amplifies the hemagglutinin (HA) gene segment and neuraminidase (NA) gene segment for detection and discrimination of influenza B. This assay is not intended to detect influenza C viruses.

FluChip-8G Influenza $\mathbf { A } { + } \mathbf { B }$ Assay “non-seasonal” influenza A positive results are for the presumptive detection of influenza A subtypes other than seasonal influenza A/H1N1pdm09 or A/H3N2. The definitive identification of a “non-seasonal” influenza A case requires additional testing and confirmation procedures in consultation with the appropriate public health authorities (e.g., local or state public health departments, etc.) for whom reporting is necessary.

Negative results do not preclude influenza virus infection. FluChip-8G Influenza $\mathbf { A } { + } \mathbf { B }$ Assay “non-seasonal” influenza A negative results, even in the context of a FluChip-8G Influenza $\mathbf { A } { + } \mathbf { B }$ Assay positive result for seasonal influenza A/H1N1pdm09 or A/H3N2, or influenza B, do not preclude “non-seasonal” influenza A infection and should not be used as the sole basis for patient management decisions.

Performance characteristics of the FluChip-8G Influenza $\mathbf { A } { + } \mathbf { B }$ Assay for detecting and differentiating seasonal influenza A viruses were established when seasonal influenza A/H3N2 was the predominant influenza A virus circulating in the United States. Performance characteristics may vary with other emerging seasonal influenza A viruses. Performance characteristics of the FluChip-8G Influenza $\mathbf { A } { + } \mathbf { B }$ Assay for detecting and differentiating human influenza B genetic lineages were established when influenza

B/Victoria was the predominant influenza B virus circulating in the United States.

Due to low prevalence of “non-seasonal” influenza A viruses, performance characteristics of the FluChip-8G Influenza $\mathbf { A } { + } \mathbf { B }$ Assay for detecting “non-seasonal” influenza A viruses and distinguishing “non-seasonal” influenza A from seasonal influenza A H1N1pdm09 and H3N2 were assessed exclusively by conducting crossvalidation on a total of 759 microarray images generated from bench testing contrived samples consisting of 352 unique “non-seasonal” influenza A strains representing 62 subtypes, and by bench testing contrived samples and surrogate clinical specimens consisting of 133 unique non-seasonal influenza A strains representing 46 subtypes. FluChip-8G Influenza $\mathbf { A } { + } \mathbf { B }$ Assay performance may vary when testing “non-seasonal” influenza A strains not represented in the performance assessment.

If infection with a novel influenza A virus strain is suspected based on current clinical and epidemiological screening criteria recommended by public health authorities, specimens should be collected with appropriate infection control precautions for novel virulent influenza viruses and sent to local or state health department(s) for testing. Viral culture should not be attempted unless a BSL 3E facility is available to receive and culture specimens.

2. Indication(s) for use: Same as Intended Use(s)

3. Special conditions for use statement(s): For prescription use only

4. Special instrument requirements:

• BioRad T100 endpoint thermal cycler • FluChip-8G Imaging System with FluChip-8G Software v1.0.9.0 or newer

# I. Device Description:

# Overview

The FluChip-8G Influenza $\mathbf { A } { + } \mathbf { B }$ Assay system is a molecular assay system for the detection and differentiation of influenza viruses in which a multiplexed one-step RT-PCR amplification is coupled with downstream microarray-based hybridization, imaging, and subsequent influenza virus detection and characterization using a pattern recognition-based algorithm.

The system consists of the following:

1) A reagent kit comprising the reagents required to conduct RT-PCR and for performing post-PCR sample processing including PCR product fragmentation, microarray hybridization, washing, fluorescent labeling, and drying.

2) An accessory kit comprising a variety of laboratory equipment and supplies for facilitating execution of the assay.

3) Other general laboratory equipment and supplies required but not provided.

4) The FluChip-8G Imaging System with pre-installed FluChip-8G Software including an image processing module, an underlying neural network-based pattern recognition algorithm for identifying patterns in microarray signals representative of certain influenza target virus groups, and a user interface to facilitate data entry/imaging and provide result reports to the end user.

# Materials Provided

FluChip-8G Influenza $\mathbf { A } { + } \mathbf { B }$ Assay Reagent Kit (FC-6101)   

<table><tr><td colspan="1" rowspan="1">Reagents</td><td colspan="1" rowspan="1">Description</td><td colspan="1" rowspan="1">Units Per Kit</td><td colspan="1" rowspan="1">Assays PerUnit</td><td colspan="1" rowspan="1">VolumePer Unit</td><td colspan="1" rowspan="1">Cat. #</td></tr><tr><td colspan="5" rowspan="1">FC8G Amplification Reagents:</td><td colspan="1" rowspan="1">FC-6101A</td></tr><tr><td colspan="1" rowspan="1">FC8G Polymerase Mix</td><td colspan="1" rowspan="1">Mixture of components necessary forPCR amplification</td><td colspan="1" rowspan="1">6 vials</td><td colspan="1" rowspan="1">16</td><td colspan="1" rowspan="1">500 μL</td><td colspan="1" rowspan="1">FC-5015</td></tr><tr><td colspan="1" rowspan="1">FC8G RT Enzyme</td><td colspan="1" rowspan="1">Enzyme necessary to perform reversetranscription step</td><td colspan="1" rowspan="1">6 vials</td><td colspan="1" rowspan="1">16</td><td colspan="1" rowspan="1">35 μL</td><td colspan="1" rowspan="1">FC-5014</td></tr><tr><td colspan="1" rowspan="1">FC8G Primer Mix</td><td colspan="1" rowspan="1">Mixture of oligonucleotide primerscontaining sodium azide (0.02%) as apreservative</td><td colspan="1" rowspan="1">6 vials</td><td colspan="1" rowspan="1">16</td><td colspan="1" rowspan="1">40 μL</td><td colspan="1" rowspan="1">FC-5004</td></tr><tr><td colspan="1" rowspan="1">FC8G Biotin dUTP</td><td colspan="1" rowspan="1">Lyophilized biotinylated dUTP</td><td colspan="1" rowspan="1">6 vials</td><td colspan="1" rowspan="1">16</td><td colspan="1" rowspan="1">50 nmol</td><td colspan="1" rowspan="1">FC-5008</td></tr><tr><td colspan="5" rowspan="1">FC8G Microarray Processing Reagents:</td><td colspan="1" rowspan="1">FC-6101B</td></tr><tr><td colspan="1" rowspan="1">FluChip-8G Microarray Slide</td><td colspan="1" rowspan="1">Microarray slide with 16-well siliconechamber. Each well contains a singleoligonucleotide microarray</td><td colspan="1" rowspan="1">6 slides</td><td colspan="1" rowspan="1">16</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">FC-5006</td></tr><tr><td colspan="1" rowspan="1">Wash Buffer 1 (20x)</td><td colspan="1" rowspan="1">Concentrated wash buffer containingsodium azide (0.05%) as apreservative</td><td colspan="1" rowspan="1">2 bottles</td><td colspan="1" rowspan="1">48</td><td colspan="1" rowspan="1">50 mL</td><td colspan="1" rowspan="1">MI-5014</td></tr><tr><td colspan="1" rowspan="1">Wash Buffer 2 (20x)</td><td colspan="1" rowspan="1">Concentrated wash buffer containingsodium azide (0.05%) as apreservative</td><td colspan="1" rowspan="1">2 bottles</td><td colspan="1" rowspan="1">48</td><td colspan="1" rowspan="1">50 mL</td><td colspan="1" rowspan="1">MI-5015</td></tr><tr><td colspan="1" rowspan="1">FC8G Binding Buffer</td><td colspan="1" rowspan="1">Concentrated binding buffercontaining sodium azide (0.02%) as apreservative</td><td colspan="1" rowspan="1">6 vials</td><td colspan="1" rowspan="1">16</td><td colspan="1" rowspan="1">1850 μL</td><td colspan="1" rowspan="1">FC-5001</td></tr><tr><td colspan="1" rowspan="1">FC8G Hyb Mix</td><td colspan="1" rowspan="1">Protein-based hybridization solutioncontaining hybridization positivecontrol oligonucleotides</td><td colspan="1" rowspan="1">6 vials</td><td colspan="1" rowspan="1">16</td><td colspan="1" rowspan="1">192 μL</td><td colspan="1" rowspan="1">FC-5013</td></tr><tr><td colspan="1" rowspan="1">FC8G Label</td><td colspan="1" rowspan="1">Lyophilized fluorophore- streptavidinlabel containing sodium azide (1.5 ng)as a preservative</td><td colspan="1" rowspan="1">6 vials</td><td colspan="1" rowspan="1">16</td><td colspan="1" rowspan="1">15 μg</td><td colspan="1" rowspan="1">FC-5002</td></tr><tr><td colspan="1" rowspan="1">FC8G Diluent</td><td colspan="1" rowspan="1">Diluent containingsodium azide (0.025%) as apreservative</td><td colspan="1" rowspan="1">6 vials</td><td colspan="1" rowspan="1">16</td><td colspan="1" rowspan="1">1500 μL</td><td colspan="1" rowspan="1">FC-5012</td></tr><tr><td colspan="1" rowspan="1">Slide Drying Sheets</td><td colspan="1" rowspan="1">Absorbent sheets to remove washbuffer from the FluChip- 8GMicroarray Slide</td><td colspan="1" rowspan="1">20 sheets</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">FC-4005</td></tr><tr><td colspan="1" rowspan="1">Desiccant Pouches</td><td colspan="1" rowspan="1">Desiccant provided to aid in slidedrying steps</td><td colspan="1" rowspan="1">18 pouches</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">FC-4013</td></tr></table>

# Materials Required but Not Provided

# Plasticware and Consumables

<table><tr><td rowspan=1 colspan=1>Material</td><td rowspan=1 colspan=1>Cat. #</td></tr><tr><td rowspan=1 colspan=1>FluChip-8G Influenza A+B Assay Accessory Kit</td><td rowspan=1 colspan=1>FC-4000</td></tr><tr><td rowspan=1 colspan=1>Humidity Chamber (available in the FluChip-8G Influenza A+B Accessory Kit FC-4000)</td><td rowspan=1 colspan=1>MI-4001</td></tr><tr><td rowspan=1 colspan=1>Four (4) Wash Bins (available in the FluChip-8G Influenza A+B Accessory Kit FC-4000)</td><td rowspan=1 colspan=1>FC-4001</td></tr><tr><td rowspan=1 colspan=1>Two (2) Rinse Bottles (available in the FluChip-8G Influenza A+B Accessory Kit FC-4000)</td><td rowspan=1 colspan=1>MI-4004</td></tr><tr><td rowspan=1 colspan=1>Plastic Forceps (available in the FluChip-8G Influenza A+B Accessory Kit FC-4000)</td><td rowspan=1 colspan=1>FC-4002</td></tr><tr><td rowspan=1 colspan=1>Slide Drying Box (available in the FluChip-8G Influenza A+B Accessory Kit FC-4000)</td><td rowspan=1 colspan=1>FC-4006</td></tr><tr><td></td><td rowspan=1 colspan=1>cYanmaGYellowb</td></tr><tr><td rowspan=1 colspan=1>Sterile flocked swabs (Copan FLOQSwabs available through VWR, Fisher Healthcare, and CardinalHealthcare)</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Universal Transport Media (UTM) consisting of: Hanks&#x27; Balanced Salts, Bovine Serum Albumin, L-Cysteine, Gelatin, Sucrose, L-Glutamic Acid, HEPES Buffer, Vancomycin, Amphotericin B,Colistin, and Phenol Red at pH 7.3 ± 0.2 @ 25 °C</td><td rowspan=1 colspan=1>--</td></tr><tr><td rowspan=1 colspan=1>Bio-Rad 0.2 mL 8-tube PCR strips</td><td rowspan=1 colspan=1>TBS0201 (Bio-Rad)</td></tr><tr><td rowspan=1 colspan=1>Bio-Rad optical flat 8-cap strips</td><td rowspan=1 colspan=1>TCS0803 (Bio-Rad)</td></tr><tr><td rowspan=1 colspan=1>Qiagen Collection Tubes</td><td rowspan=1 colspan=1>19201 (QIAGEN)</td></tr><tr><td rowspan=1 colspan=1>Ice/Ice Bucket or Cold Block</td><td rowspan=1 colspan=1>--</td></tr><tr><td rowspan=1 colspan=1>Sterile RNase/DNase-free filter micropipettor tips (5  1000 µL)</td><td rowspan=1 colspan=1>--</td></tr><tr><td rowspan=1 colspan=1>Standard micropipettor tips (5  1000 µL)</td><td rowspan=1 colspan=1>--</td></tr><tr><td rowspan=1 colspan=1>Nuclease-free Polypropylene Tubes (1.5 mL)</td><td rowspan=1 colspan=1>--</td></tr><tr><td rowspan=1 colspan=1>Lint free tissue wipes</td><td rowspan=1 colspan=1>--</td></tr><tr><td rowspan=1 colspan=1>Clean 1 Liter bottles (2) with lids</td><td rowspan=1 colspan=1>--</td></tr></table>

# Reagents and Controls

<table><tr><td colspan="1" rowspan="1">Material</td><td colspan="1" rowspan="1">Cat. #</td></tr><tr><td colspan="1" rowspan="1">QIAamp DSP Virus Spin Kit</td><td colspan="1" rowspan="1">61704 (QIAGEN)</td></tr><tr><td colspan="1" rowspan="1">Purified (18 MΩ) water for preparation of wash buffers</td><td colspan="1" rowspan="1">--</td></tr><tr><td colspan="1" rowspan="1">Nuclease-free/PCR grade water to be used for no-template negative control(s)</td><td colspan="1" rowspan="1">--</td></tr><tr><td colspan="1" rowspan="1">Ethanol (96 - 100% purity)</td><td colspan="1" rowspan="1">--</td></tr><tr><td colspan="1" rowspan="1">External positive control (previously characterized influenza positive clinical specimen)</td><td colspan="1" rowspan="1">--</td></tr><tr><td>External negative specimen)</td><td></td></tr></table>

# Instrumentation/Equipment

<table><tr><td rowspan=1 colspan=1>Material</td><td rowspan=1 colspan=1>Cat. #</td></tr><tr><td rowspan=1 colspan=1>FluChip-8G Imaging System (FluChip-8G Software Package pre-installed)</td><td rowspan=1 colspan=1>FC-6000</td></tr><tr><td rowspan=1 colspan=1>T100 Thermal Cycler</td><td rowspan=1 colspan=1>1861096 (Bio-Rad)</td></tr><tr><td rowspan=1 colspan=1>Standard pipettes (5 - 1000 µL)</td><td rowspan=1 colspan=1>--</td></tr><tr><td rowspan=1 colspan=1>Orbital Shaker (at least 8.5&quot; x 7&quot; stage) - capable of no tilt horizontal movement at 80- 140 RPM</td><td rowspan=1 colspan=1>--</td></tr><tr><td rowspan=1 colspan=1>Vortex Mixer</td><td rowspan=1 colspan=1>--</td></tr><tr><td rowspan=1 colspan=1>Centrifuge with rotor for 1.5 and 2 mL tubes capable of achieving 20,000 g (14,000 rpm)</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>Microcentrifuge with rotor adaptor for 0.2 mL 8-tube strips</td><td rowspan=1 colspan=1>--</td></tr><tr><td rowspan=1 colspan=1>Heating block for lysing samples at 56°C in 1.5 and 2 ml tubes</td><td rowspan=1 colspan=1>--</td></tr><tr><td rowspan=1 colspan=1>Timer, capable of timing 1 minute intervals over the range from 1 to 30 min</td><td rowspan=1 colspan=1>--</td></tr></table>

# Optional Materials Not Provided

<table><tr><td rowspan=1 colspan=1>Material</td><td rowspan=1 colspan=1>Cat. #</td></tr><tr><td rowspan=1 colspan=1>Barcode Reader</td><td rowspan=1 colspan=1>9029247015 (Zebra)</td></tr><tr><td rowspan=1 colspan=1>Slide Storage Box</td><td rowspan=1 colspan=1>FC-4024</td></tr></table>

# Quality Control

# Assay Controls

Internal Control: Sequences corresponding to portions of the 18s rRNA gene are included on each microarray to monitor adequate specimen collection, and in the absence of influenza, also to indicate the absence of reaction inhibition. Primers to amplify the 18s internal control are included in the Primer Mix so that the sequence complement to these microarray markers is amplified during RT-PCR and subsequently hybridized to the microarray if sufficient human material is present in the collected specimen. This control is expected to fail in samples containing no human cellular material. This control is evaluated in the absence of an influenza positive result and is expected to result in “No Call: Assay Failure (Internal Control)” in samples containing no human cellular material or poorly collected specimens.

Hybridization Control: Oligonucleotides are included on each microarray to monitor the integrity of the hybridization procedure. The complementary sequences are fluorescently tagged single-stranded synthetic DNA sequences included in the Hyb Mix solution that bind specifically to the hybridization control sequences on the microarray. When the hybridization process fails, the FC8G assay result will be “No Call: Assay Failure (Hybridization Control)”.

Labeling Control: Each microarray includes a biotin-terminated capture oligonucleotide that bind to the fluorophore-coupled streptavidin solution present in the label applied during the labeling step. This control serves as an indicator of successful microarray labeling. When the labeling process fails, the FC8G assay result will be “No Call: Assay Failure (Labeling Control)”.

Image Processing Control: Direct fluorescently tagged sequences are included on each microarray at specific locations. The image processing controls serve as points of reference for microarray placement used by the FluChip-8G software during microarray image analysis. When the FluChip-8G Microarray Slide is improperly positioned in the FluChip-8G Imaging system, the FC8G assay result will be “Error: Image processing failure”. A failure of this control may indicate an issue with the microarray quality, placement of the FluChip-8G Microarray Slide in the FluChip-8G Imaging System, or a focus issue of the FluChip-8G Imaging System.

# External Quality Control (QC)

External Batch Positive Control: A previously characterized influenza-positive clinical specimen is recommended for inclusion as part of good laboratory practices. This control is user-supplied.

External Batch Negative Control: A previously characterized influenza-negative clinical specimen is recommended for inclusion as part of good laboratory practices. This control is user supplied.

Extraction Negative Control: This control is recommended for inclusion in each testing batch, and is composed of nuclease-free water added by the end user in place of specimen during the nucleic acid extraction step.

RT-PCR No Template Control: This control is recommended for inclusion in each testing batch, and is composed of nuclease-free water added by the end user in place of extracted nucleic acid template during RT-PCR reaction setup.

# Workflow

The FluChip-8G Influenza $\mathbf { A } { + } \mathbf { B }$ Assay is comprised of the following workflow:

1) Nucleic acid extraction

First, nucleic acids are extracted from human nasal swab or nasopharyngeal swab specimen(s) stabilized in Universal Transport Media (UTM) employing the QIAamp DSP Virus Spin Kit (catalog #61704). During the nucleic acid extraction step, both DNA and RNA are co-eluted from starting material, and any influenza viral RNA as well as the endogenous 18s rRNA internal control are made accessible for downstream amplification. Extracted nucleic acid is then eluted and utilized as template for downstream amplification.

2) Multiplexed reverse transcription polymerase chain reaction (RT-PCR) amplification followed by heat fragmentation

The RT-PCR amplification primers designed for the FluChip-8G Influenza $\mathbf { A } { + } \mathbf { B }$ Assay (provided as FC8G Primer Mix) provide simultaneous (multiplexed) amplification of several full-length gene segments of the influenza A virus genome (HA, NA, M, NS, and NP), two full length gene segments of the influenza B virus genome (HA, NA), and a segment of the 18s rRNA found in eukaryotic cells as an internal process control. The

FC8G Primer Mix provides amplification of all targeted gene segments for all influenza A and B viruses, regardless of subtype or lineage, within a single reaction. A biotinylated dUTP is supplied, and is used during RT-PCR to incorporate biotin molecules into the amplified product. The FC8G Biotin dUTP supplements the standard dNTPs already present in the FC8G Polymerase Mix. These biotin moieties are utilized for downstream fluorescence labeling with a streptavidin-coupled fluorophore (FC8G Label) after the microarray hybridization step. Post-amplification, RT-PCR products are heat-fragmented for $1 0 \mathrm { m i n }$ at $9 5 \mathrm { { ^ \circ C } }$ to fragment the amplicons into shorter pieces for more efficient downstream hybridization to the FluChip-8G Microarray Slide.

3) Microarray processing (pre-wash slides, hybridization, post-hybridization slide wash, labeling, and post-label wash)

The FluChip-8G Microarray Slide consists of a $7 5 \ : \mathrm { m m } \times 2 5 \ : \mathrm { m m }$ glass microscope slide onto which 16 replicate microarrays (labeled A1 through H2) are printed. The Microarray Slide is also labeled with a slide number at the top and a barcode at the bottom. Within each replicate microarray, 464 unique synthetic DNA oligonucleotides (capture oligonucleotide sequences) are printed that act as either influenza capture sequences (458, each printed in triplicate) or assay controls (six control sequences). A heat-fragmented post-amplification mixture is added to the microarray and incubated to allow for hybridization between the amplified nucleic acid and the microarray capture sequences. Once hybridization is complete, the microarray is washed to remove any unbound material. The microarray is then labeled with a streptavidin-coupled fluorophore to enable downstream fluorescence imaging.

# 4) Microarray imaging and data analysis

FluChip-8G Imaging System consists of: a) an optical imaging subsystem used to obtain fluorescence images of FluChip-8G Microarray Slides processed via the FC8G Influenza $\mathbf { A } { + } \mathbf { B }$ Assay, b) instrument control software and application programming interface (API), and c) FC8G Software that automates image acquisition, controls the settings and instrument features available to end users through the user interface (UI), and interprets the FluChip-8G Microarray Slide data using a neural network module to provide results to the end users. Image processing algorithms within the FC8G Software locate the microarray pattern within the image and determine the fluorescence intensity of individual oligonucleotide captures on the microarray. Specific captures on the microarray that are intended as controls verify that specific assay processing steps were successfully completed. Trained neural networks are used to recognize the patterns of intensities of the influenza-targeted captures on the microarray as belonging to specific influenza types or subtypes/lineages. The neural network module utilizes seven individual neural networks, with the microarray signals from the 458 influenza capture sequences utilized as inputs. The output of each neural network is a value between 0 and 1, and a cutoff (optimized subsequent to neural network training) is applied to each to determine if each is positive or negative for its trained target. A deterministic decision tree then uses the binary outputs of the controls and neural networks to generate a diagnostic output for each clinical sample. The diagnostic output is then displayed to the end user in un-editable format for printing and/or saving.

# Neural Networks Training and Verification

Neural networks training is an interactive process and includes the following elements:

a) Read in the library of results from well-characterized known samples corresponding to all target virus groups for training.   
b) Determine which result files contain sufficient signal to be used for training. Training set inclusion is based on key capture sequences identified for different target virus groups.   
c) Use the Fast Artificial Neural Network (FANN) open source library functions to train individual networks. Each individual neural network uses a single hidden layer neuron and a single output neuron trained via supervised learning using backpropagation.   
d) Determine thresholds that optimize network performance in terms of positive percent agreement (PPA) and negative percent agreement (NPA) with the known results of a clinically-representative set of naïve samples.   
e) Save network files with threshold values for verification and inclusion in FC8G Software.

The database of assay results available for training the neural networks consisted of 3372 microarray images. Inclusion criteria were applied to ensure that the influenza-positive results had detectable signals on key captures of the microarray. This prevented result files with undetectable signal levels from being trained as positives. The final count of microarray images used for training by virus target group is presented in Table 1 below. This includes:

• 478 results from RT-PCR No Template Controls and Extraction Negative Controls. • Results from 850 unique clinical specimens containing influenza, clinical specimens negative for influenza, and contrived influenza positive samples. Results from 1736 replicate analyses from the 850 unique sample strains noted above (primarily from dilution series testing).

Table 1: Neural Network Training Set Composition and Results Excluded from Training   

<table><tr><td rowspan=1 colspan=1>Category</td><td rowspan=1 colspan=1># Trained Images</td><td rowspan=1 colspan=1># ExcludedImages</td><td rowspan=1 colspan=1>Total Images</td></tr><tr><td rowspan=1 colspan=1>Influenza A, H1N1pdm09</td><td rowspan=1 colspan=1>446</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>506</td></tr><tr><td rowspan=1 colspan=1>Influenza A, Seasonal H3N2</td><td rowspan=1 colspan=1>378</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>408</td></tr><tr><td rowspan=1 colspan=1>Non-Seasonal Influenza A</td><td rowspan=1 colspan=1>759</td><td rowspan=1 colspan=1>75</td><td rowspan=1 colspan=1>834</td></tr><tr><td rowspan=1 colspan=1>Influenza B, Victoria lineage</td><td rowspan=1 colspan=1>287</td><td rowspan=1 colspan=1>112</td><td rowspan=1 colspan=1>399</td></tr><tr><td rowspan=1 colspan=1>Influenza B, Yamagata lineage</td><td rowspan=1 colspan=1>244</td><td rowspan=1 colspan=1>31</td><td rowspan=1 colspan=1>275</td></tr><tr><td rowspan=1 colspan=1>Clinical Negative</td><td rowspan=1 colspan=1>472</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>472</td></tr><tr><td rowspan=1 colspan=1>Negative Controls</td><td rowspan=1 colspan=1>478</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>478</td></tr><tr><td rowspan=1 colspan=1>Overall Totals</td><td rowspan=1 colspan=1>3064</td><td rowspan=1 colspan=1>308</td><td rowspan=1 colspan=1>3372</td></tr></table>

Regarding training the non-seasonal influenza A neural network, a total of 363 unique nonseasonal influenza A strains (representing 62 subtypes) were used to prepare contrived samples that upon testing generated a total of 834 microarray images. Of these 834 images,

759 microarray images were included in training, and 75 microarray images were excluded from training due to failure to meet the inclusion criteria. Table 2 below lists all 352 unique non-seasonal influenza A strains that were used to prepare contrived samples that upon testing generated the total of 759 microarray images that were included in the training, stratified by subtypes.

Table 2: Unique Non-Seasonal Influenza A Strains that Were Used to Prepare Contrived Samples to Generate the Microarray Images Included in Training the Non-Seasonal Influenza A Neural Network   

<table><tr><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=4>Subtype</td><td rowspan=1 colspan=3># ofUniqueStrain inSubtype</td><td rowspan=1 colspan=1># ofReplicateAnalyses perStrain</td><td rowspan=1 colspan=1># ofMicroarrayImages perSubtype</td></tr><tr><td rowspan=1 colspan=1>A/Arizona/07/2007</td><td rowspan=5 colspan=4></td><td rowspan=5 colspan=3></td><td rowspan=1 colspan=1>1</td><td rowspan=12 colspan=1>188</td></tr><tr><td rowspan=1 colspan=1>A/Virginia/01/2006</td><td rowspan=1 colspan=1>2</td></tr><tr><td rowspan=1 colspan=1>A/Santiago/5248/2008</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>A/Chelyabinsk/01/2006</td><td rowspan=1 colspan=3></td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>A/Denver/1/57</td><td rowspan=1 colspan=3></td><td rowspan=1 colspan=1>5</td></tr><tr><td rowspan=1 colspan=1>A/Texas/35/2008, E3 (03/12/2009)</td><td rowspan=7 colspan=4>H1N1 (pre 2009)</td><td rowspan=1 colspan=1></td><td rowspan=2 colspan=3>12</td><td rowspan=1 colspan=1>2</td></tr><tr><td rowspan=1 colspan=1>A/Mississippi/3/2001 wt 275H</td><td></td><td rowspan=1 colspan=2></td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>A/Mississippi/3/2001 mut H275Y</td><td rowspan=5 colspan=3></td><td rowspan=1 colspan=1>−</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>Clinical Specimen #1</td><td rowspan=1 colspan=3></td><td rowspan=1 colspan=2></td><td rowspan=1 colspan=1>5</td></tr><tr><td rowspan=1 colspan=1>Clinical Specimen #2</td><td rowspan=1 colspan=3></td><td rowspan=1 colspan=1>3</td></tr><tr><td rowspan=1 colspan=1>A/New Caledonia/20/1999</td><td rowspan=1 colspan=1>74</td></tr><tr><td rowspan=1 colspan=1>A/Brisbane/59/2007</td><td rowspan=1 colspan=1>92</td></tr><tr><td rowspan=1 colspan=1>A/Teal/Egypt/677/2004</td><td rowspan=6 colspan=4>H1N1 Avian</td><td rowspan=3 colspan=3></td><td rowspan=1 colspan=1>2</td><td rowspan=10 colspan=1>24</td></tr><tr><td rowspan=1 colspan=1>Unknown #1- from CDC 2014</td><td rowspan=1 colspan=3>b</td><td rowspan=1 colspan=1>2</td></tr><tr><td rowspan=1 colspan=1>Unknown #2- from CDC 2014</td><td rowspan=1 colspan=3></td><td rowspan=1 colspan=2></td><td rowspan=1 colspan=1>2</td></tr><tr><td rowspan=1 colspan=1>A/Mallard/Republic of Georgia/4/2010</td><td rowspan=1 colspan=3></td><td rowspan=1 colspan=3></td><td rowspan=1 colspan=1>3</td></tr><tr><td rowspan=1 colspan=1>A/Duck/Alberta/35/76</td><td rowspan=1 colspan=2>−</td><td rowspan=2 colspan=3>10</td><td rowspan=1 colspan=1>2</td></tr><tr><td rowspan=1 colspan=1>A/Mallard/Alberta/21/2014</td><td rowspan=1 colspan=2></td><td rowspan=1 colspan=1>−</td><td rowspan=1 colspan=1>3</td></tr><tr><td rowspan=1 colspan=1>A/Redheaded Duck/MN/SG-00123/2007</td><td rowspan=1 colspan=2></td><td rowspan=4 colspan=2></td><td rowspan=4 colspan=3></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>2</td></tr><tr><td rowspan=1 colspan=1>A/Gull/DE/428/2009</td><td></td><td></td><td rowspan=1 colspan=2></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>A/Ruddy Turnstone/DE/274/2009</td><td></td><td></td><td rowspan=1 colspan=1>3</td></tr><tr><td rowspan=1 colspan=1>A/Aqutic Bird/Hong Kong/D125/2002</td><td rowspan=1 colspan=2>−</td><td rowspan=1 colspan=1>4</td></tr><tr><td rowspan=1 colspan=1>A/Swine/1976/1931</td><td rowspan=3 colspan=4>H1N1 Swine</td><td rowspan=1 colspan=1></td><td rowspan=3 colspan=3>3</td><td rowspan=1 colspan=1>2</td></tr><tr><td rowspan=1 colspan=1>A/Swine/TN/26/1977</td><td rowspan=1 colspan=2></td><td rowspan=1 colspan=1></td><td></td><td rowspan=1 colspan=1>2</td><td></td></tr><tr><td rowspan=1 colspan=1>A/Swine/NC/18162/2002</td><td></td><td rowspan=1 colspan=1>1</td><td></td></tr><tr><td rowspan=1 colspan=1>Swine Origin Influenza Virus (soiv)</td><td rowspan=6 colspan=4>H1N2</td><td rowspan=6 colspan=3>5</td><td rowspan=1 colspan=1>2</td><td rowspan=2 colspan=1></td></tr><tr><td rowspan=1 colspan=1>A/Teal/Egypt/431/2003</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Unknown #3- from CDC 2014</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=2 colspan=1>A/Swine/Ohio/09SW1484E/2009</td><td rowspan=2 colspan=2></td><td rowspan=2 colspan=1>2</td><td rowspan=2 colspan=1></td></tr><tr><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>A/Swine/Ohio/09SW1477/2009</td><td rowspan=1 colspan=1>2</td></tr><tr><td rowspan=1 colspan=1>A/Shorebird/Delaware Bay/211/1994</td><td rowspan=1 colspan=4>H1N3</td><td rowspan=1 colspan=3>1</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>2</td></tr><tr><td rowspan=1 colspan=1>A/Red Knot/Delaware Bay/240/1994</td><td rowspan=1 colspan=4>H1N8</td><td rowspan=1 colspan=3>1</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>2</td></tr><tr><td rowspan=1 colspan=1>A/Japan/305/57</td><td rowspan=2 colspan=4>H2N2</td><td rowspan=2 colspan=3>2</td><td rowspan=1 colspan=1>1</td><td rowspan=2 colspan=1>3</td></tr><tr><td rowspan=1 colspan=1>Unknown #4- from CDC 2014</td><td rowspan=1 colspan=1>2</td></tr><tr><td rowspan=1 colspan=1>A/Duck/Germany/1215/73</td><td rowspan=1 colspan=4>H2N3</td><td rowspan=1 colspan=3>1</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>2</td></tr><tr><td rowspan=1 colspan=1>Blue-winged Teal Texas</td><td rowspan=2 colspan=4>H2N9</td><td rowspan=2 colspan=3>2</td><td rowspan=1 colspan=1>2</td><td rowspan=2 colspan=1>4</td></tr><tr><td rowspan=1 colspan=1>A/Pintail/ALB/293/77</td><td rowspan=1 colspan=1>2</td></tr><tr><td rowspan=1 colspan=1>A/Swine/North Carolina/88708/2000, CX/C1 (10/17/2011)</td><td rowspan=2 colspan=4>H3N2 Swine</td><td rowspan=2 colspan=3>14</td><td rowspan=1 colspan=1>2</td><td rowspan=2 colspan=1>23</td></tr><tr><td rowspan=1 colspan=1>A/Swine/North Carolina/32760/2007, CX/C1 (10/13/2011)</td><td rowspan=1 colspan=1>2</td></tr></table>

<table><tr><td colspan="1" rowspan="1">A/Swine/North Carolina/44897/2009, C2/C1 (10/13/2011)</td><td colspan="8" rowspan="4"></td><td colspan="3" rowspan="2"></td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="13"></td></tr><tr><td colspan="1" rowspan="1">A/Swine/North Carolina/52796/2006, CX/C1 (10/13/2011)</td><td colspan="1" rowspan="1">2</td></tr><tr><td colspan="1" rowspan="1">A/Swine/Ohio/09SW73E/2009</td><td colspan="3" rowspan="2"></td><td colspan="1" rowspan="1">2</td></tr><tr><td colspan="1" rowspan="1">A/Swine/Ohio/09SW83E/2009</td><td colspan="1" rowspan="1">2</td></tr><tr><td colspan="1" rowspan="2">A/Swine/Ohio/11SW87/2011</td><td colspan="7" rowspan="2"></td><td></td><td></td><td></td><td></td><td></td></tr><tr><td colspan="3" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="3" rowspan="1"></td><td colspan="1" rowspan="1">2</td></tr><tr><td colspan="1" rowspan="1">A/Swine/Ohio/09SW79M/2009</td><td colspan="8" rowspan="1">Y</td><td colspan="3" rowspan="1"></td><td colspan="1" rowspan="1">2</td></tr><tr><td colspan="1" rowspan="1">A/Swine/Texas/4199-2/1998</td><td colspan="8" rowspan="6"></td><td colspan="3" rowspan="1"></td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">A/Swine/NC/0668/2011</td><td colspan="3" rowspan="2"></td><td colspan="1" rowspan="1">2</td></tr><tr><td colspan="1" rowspan="1">A/Swine/NC/0666/2011</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">4D-0114-P29 swine swab</td><td colspan="3" rowspan="3"></td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">4D-0114-P21 swine swab</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">4D-0114-P15 swine swab</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">A/Ohio/20/2012</td><td colspan="8" rowspan="6"></td><td colspan="3" rowspan="2"></td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="14">28</td></tr><tr><td colspan="1" rowspan="1">A/Indiana/16/2012</td><td colspan="1" rowspan="1">2</td></tr><tr><td colspan="1" rowspan="1">A/Indiana/14/2012</td><td colspan="3" rowspan="2"></td><td colspan="1" rowspan="1">2</td></tr><tr><td colspan="1" rowspan="1">A/Indiana/65/2012</td><td colspan="2" rowspan="1"></td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1">2</td></tr><tr><td colspan="1" rowspan="1">A/Michigan/20/2012</td><td colspan="3" rowspan="1"></td><td colspan="2" rowspan="1"></td><td colspan="2" rowspan="1"></td><td colspan="1" rowspan="1">2</td></tr><tr><td colspan="1" rowspan="1">A/Pennsylvania/17/2012</td><td colspan="3" rowspan="1"></td><td colspan="3" rowspan="1"></td><td colspan="1" rowspan="1">2</td></tr><tr><td colspan="1" rowspan="1">A/Wisconsin/28/2012</td><td colspan="8" rowspan="8">H3N2v</td><td colspan="1" rowspan="1"></td><td colspan="3" rowspan="2">14</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">A/Ohio/44/2012</td><td colspan="3" rowspan="1">b</td><td></td><td colspan="1" rowspan="1">2</td></tr><tr><td colspan="1" rowspan="1">A/Ohio/47/2012</td><td colspan="3" rowspan="1"></td><td colspan="1" rowspan="1">2</td></tr><tr><td colspan="1" rowspan="1">A/Ohio/56/2012</td><td colspan="3" rowspan="2"></td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">A/Ohio/83/2012</td><td colspan="1" rowspan="1">2</td></tr><tr><td colspan="1" rowspan="1">A/Ohio/36/2012</td><td colspan="3" rowspan="3"></td><td colspan="1" rowspan="1">2</td></tr><tr><td colspan="1" rowspan="1">A/Indiana/8/2011</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">A/Swine/MN/3908-2/11</td><td colspan="1" rowspan="1">5</td></tr><tr><td colspan="1" rowspan="1">A/Blue-winged Teal/Texas/G77/2007</td><td colspan="8" rowspan="3">H3N6</td><td colspan="3" rowspan="3">3</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="3">6</td></tr><tr><td colspan="1" rowspan="1">A/Blue-Winged Teal/Illinois/10OS1546/2010</td><td colspan="1" rowspan="1">2</td></tr><tr><td colspan="1" rowspan="1">A/Redhead/Alberta/192/2002</td><td colspan="1" rowspan="1">2</td></tr><tr><td colspan="1" rowspan="1">A/Equine/TX/2004</td><td colspan="8" rowspan="6"></td><td colspan="3" rowspan="3"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="23">62</td></tr><tr><td colspan="1" rowspan="1">A/Equine/New York/452/2003</td><td colspan="3" rowspan="1"></td><td colspan="1" rowspan="1">2</td></tr><tr><td colspan="1" rowspan="1">A/Canine/New York/100525,1/06</td><td colspan="1" rowspan="1">2</td></tr><tr><td colspan="1" rowspan="1">A/Canine/Florida/89911,2/06</td><td colspan="3" rowspan="1"></td><td colspan="1" rowspan="1">2</td></tr><tr><td colspan="1" rowspan="1">A/Canine/VA/93653/09</td><td colspan="3" rowspan="1"></td><td colspan="1" rowspan="1">2</td></tr><tr><td colspan="1" rowspan="1">A/Canine/New York/4732/2012</td><td colspan="2" rowspan="1"></td><td colspan="5" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="3" rowspan="1"></td><td colspan="1" rowspan="1">2</td></tr><tr><td colspan="1" rowspan="1">A/Duck/Bangladesh/1293/2008</td><td colspan="3" rowspan="1"></td><td colspan="5" rowspan="2"></td><td colspan="1" rowspan="1"></td><td colspan="3" rowspan="1"></td><td colspan="1" rowspan="1">2</td></tr><tr><td colspan="1" rowspan="1">A/Duck/Bangladesh/1575/2009</td><td colspan="3" rowspan="1"></td><td colspan="3" rowspan="1"></td><td colspan="1" rowspan="1">2</td></tr><tr><td colspan="1" rowspan="1">Unknown #5- from CDC 2014</td><td colspan="2" rowspan="1"></td><td colspan="6" rowspan="2"></td><td colspan="3" rowspan="2"></td><td colspan="1" rowspan="1">2</td></tr><tr><td colspan="1" rowspan="1">Unknown #6- from CDC 2014</td><td colspan="3" rowspan="1"></td><td colspan="6" rowspan="2">H3N8</td><td colspan="2" rowspan="1"></td><td colspan="1" rowspan="1">2</td></tr><tr><td colspan="1" rowspan="1">A/Camel/Mongolia/335/2012</td><td></td><td></td><td></td><td colspan="3" rowspan="1">21</td><td colspan="1" rowspan="1">2</td></tr><tr><td colspan="1" rowspan="1">A/Equine-2/Miami/1963</td><td colspan="7" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="3" rowspan="1"></td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="3">A/Blue-Winged Teal/Iowa/100S2411/2010</td><td colspan="8" rowspan="3">Y</td><td></td><td></td><td></td><td></td></tr><tr><td colspan="2" rowspan="2"></td><td></td><td></td><td></td><td></td></tr><tr><td colspan="3" rowspan="1"></td><td colspan="1" rowspan="1">2</td></tr><tr><td colspan="1" rowspan="1">A/Equine/Pennsylvania/1/2007</td><td colspan="8" rowspan="2"></td><td colspan="3" rowspan="2"></td><td colspan="1" rowspan="1">2</td></tr><tr><td colspan="1" rowspan="1">A/Duck/Ukraine/63</td><td colspan="3" rowspan="1"></td><td colspan="1" rowspan="1">4</td></tr><tr><td colspan="1" rowspan="1">A/Duck/Chabarovsk/1610/72</td><td colspan="8" rowspan="6"></td><td colspan="3" rowspan="3"></td><td colspan="1" rowspan="1">2</td></tr><tr><td colspan="1" rowspan="1">A/Equine/KY/4/2011</td><td colspan="1" rowspan="1">5</td></tr><tr><td colspan="1" rowspan="1">A/Mallard/Alberta/435/2013</td><td colspan="1" rowspan="1">10</td></tr><tr><td colspan="1" rowspan="1">A/Mallard/Alberta/307/2012</td><td colspan="3" rowspan="3"></td><td colspan="1" rowspan="1">8</td></tr><tr><td colspan="1" rowspan="1">A/Laughing Gull/DE/45/2005</td><td colspan="1" rowspan="1">5</td></tr><tr><td colspan="1" rowspan="1">A/Songbird/Hong Kong/SB18/2001</td><td colspan="1" rowspan="1">2</td></tr><tr><td colspan="1" rowspan="1">A/Mallard/Alberta/54/1993</td><td colspan="8" rowspan="1">H3N9</td><td colspan="3" rowspan="1">1</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">2</td></tr><tr><td colspan="1" rowspan="1">A/Duck/Bangladesh/1746/2010</td><td colspan="8" rowspan="1">H4N2</td><td colspan="3" rowspan="1">1</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">2</td></tr><tr><td colspan="1" rowspan="2">A/Duck/Bangladesh/1283/2008</td><td colspan="4" rowspan="6">H4N6</td><td colspan="1" rowspan="6">5</td><td colspan="1" rowspan="2">2</td><td></td></tr><tr><td colspan="1" rowspan="5">10</td></tr><tr><td colspan="1" rowspan="1">Unknown #7- from CDC 2014</td><td colspan="1" rowspan="1">2</td></tr><tr><td colspan="1" rowspan="1">A/Blue-Winged Teal/Illinois/10OS1563/2010</td><td colspan="1" rowspan="1">2</td></tr><tr><td colspan="1" rowspan="1">A/Red Knot/Delaware/541/1988</td><td colspan="1" rowspan="1">2</td></tr><tr><td colspan="1" rowspan="1">A/Duck/Czech/56</td><td colspan="1" rowspan="1">2</td></tr><tr><td colspan="1" rowspan="1">Unknown #8- from CDC 2014</td><td colspan="4" rowspan="1">H4N8</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">2</td></tr><tr><td colspan="1" rowspan="1">A/Vietnam/JP20-2/2005</td><td colspan="4" rowspan="6"></td><td colspan="1" rowspan="3"></td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">A/Chicken/Vietnam/NCVDCDC33/2005</td><td colspan="1" rowspan="1">2</td></tr><tr><td colspan="1" rowspan="1">A/Chicken/Vietnam/NCVDCDC42/2005</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">A/Chicken/Vietnam/NCVDCDC37/2005</td><td colspan="3" rowspan="1">′</td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="1">2</td></tr><tr><td colspan="1" rowspan="1">A/Chicken/Vietnam/NCVDCDC52/2005</td><td colspan="2" rowspan="1"></td><td colspan="1" rowspan="1">2</td></tr><tr><td colspan="1" rowspan="1">A/Chicken/Vietnam/NCVDCDC36/2005</td><td colspan="2" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">A/Environment/Vietnam/NCVDCDC53/2005</td><td colspan="4" rowspan="37">H5N1</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">A/Environment/Vietnam/NCVDCDC54/2005</td><td colspan="3" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2</td></tr><tr><td colspan="1" rowspan="1">A/Anhui/2/2005</td><td colspan="1" rowspan="5"></td><td colspan="1" rowspan="1">2</td></tr><tr><td colspan="1" rowspan="1">A/Duck/Kulon Progo/Bb Vet/IX/2004</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">A/Turkey/VA/505477-18/07</td><td colspan="1" rowspan="1">2</td></tr><tr><td colspan="1" rowspan="1">A/Mallard/PA/454069-9/2006</td><td colspan="1" rowspan="1">9</td></tr><tr><td colspan="1" rowspan="1">A/Ruddy Turnstone/DE/105/2007</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">A/Duck/MN/1525/81</td><td colspan="1" rowspan="30">43</td><td colspan="1" rowspan="1">5</td></tr><tr><td colspan="1" rowspan="1">A/Chicken/Egypt/A10543A/2015</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">A/Chicken/Egypt/D10565A/2015</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">A/Chicken/Bangladesh/18247/12</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">A/Quail/Bangladesh/19250/13</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">A/Hong Kong/156/97</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">A/Vietnam/1203/2004</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">A/Japanese White-Eye/Hong Kong/1038/2006</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">A/Chicken/Hunan/2246/2006</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">A/Chicken/Qalubia-Egypt/1/2006</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">A/Turkey/Egypt/7/2007</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">A/Egret/Egypt/1162-NAMRU3/2006</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">A/Common Magpie/Hong Kong/5052/2007</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">A/Common Buzzard/Bulgaria/38WB/2010</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">A/Chicken/Egypt/Q1089E/2010</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">A/Goose/Egypt/M2794A/2011</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">A/Goose/Hong Kong/631/2009</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">A/Large-Billed Crow/Hong Kong/497/2011</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">A/Chicken/Egypt/S3806B/2011</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">A/Crested Myna/Hong Kong/8381/2012</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">A/Black Headed Gull/Hong Kong/84/2012</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">A/Turkey/Egypt/S6405C/2012</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">A/Chicken/Egypt/M7217B/2013</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">A/Chicken/Egypt/B9040A/2013</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">A/Chicken/Egypt/F9514A/2014</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">A/Chicken/Bangladesh/23974/2014</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">A/Goose/Bangladesh/25169/2015</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">A/Chicken/Egypt/A10758D/2015</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">A/Chicken/Egypt/Q10937B/2015</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">A/Shorebird/DE/472/2007</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">A/Duck/Bangladesh/1559/2009</td><td colspan="4" rowspan="4">H5N2</td><td colspan="1" rowspan="4">18</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="4">26</td></tr><tr><td colspan="1" rowspan="1">A/Mallard/MN/346250/00</td><td colspan="1" rowspan="1">2</td></tr><tr><td colspan="1" rowspan="1">Unknown #9- from CDC 2014</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">A/Mule Duck/Bulgaria/237/2011</td><td colspan="1" rowspan="1">3</td></tr></table>

<table><tr><td rowspan=1 colspan=1>A/Ruddy Turnstone/DE/431/2011</td><td rowspan=8 colspan=5></td><td rowspan=1 colspan=2></td><td rowspan=1 colspan=1>2</td><td rowspan=18 colspan=1></td></tr><tr><td rowspan=1 colspan=1>A/Mallard/AR/1C/2001</td><td rowspan=3 colspan=2></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>4</td></tr><tr><td rowspan=1 colspan=1>A/Turkey/ND/11419-1/15</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>A/Turkey/MN/11668-1/2015</td><td rowspan=1 colspan=3></td><td rowspan=2 colspan=1></td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>A/Turkey/MN/10777/15</td><td rowspan=1 colspan=2></td><td rowspan=1 colspan=1></td><td rowspan=4 colspan=2></td><td rowspan=4 colspan=2></td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=3 colspan=1>A/Turkey/MN/110915-1/15</td><td></td></tr><tr><td rowspan=1 colspan=2></td><td></td></tr><tr><td rowspan=1 colspan=3></td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>A/Northern Pintail/WA/40564/14</td><td rowspan=10 colspan=5></td><td rowspan=1 colspan=1></td><td rowspan=10 colspan=1>b</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=3 colspan=1>A/Chicken/Italy/312/97</td><td></td><td></td></tr><tr><td></td><td></td></tr><tr><td></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>A/Northern Pintail/Washington/40964/2014</td><td></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>A/Snow Goose/CC15-84B/2015</td><td></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>A/White-Faced Whistling Duck/Colombia/1/2011</td><td></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>A/Shorebird/DE/318/2011</td><td></td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>A/Northern Pintail/CA/44242-758/2006</td><td></td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>A/Mallard/OH/14OS2758/2014</td><td></td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>Unknown #10- from CDC 2014</td><td rowspan=3 colspan=5>H5N3</td><td rowspan=3 colspan=2>3</td><td rowspan=1 colspan=1>2</td><td rowspan=3 colspan=1>4</td></tr><tr><td rowspan=1 colspan=1>A/Duck/Hokkaido/69/2000</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>A/Ruddy Turnstone/DE/215/91</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>A/Mallard/Italy/80/93</td><td rowspan=2 colspan=5>H5N4</td><td rowspan=2 colspan=2>2</td><td rowspan=1 colspan=1>1</td><td rowspan=2 colspan=1>2</td></tr><tr><td rowspan=1 colspan=1>A/Shorebird/DE/230/2000</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>A/Mallard/Alberta/383/2009</td><td rowspan=1 colspan=5>H5N5</td><td rowspan=1 colspan=2>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>A/Peregrine Falcon/Hong Kong/4955/2015</td><td rowspan=1 colspan=5>H5N6</td><td rowspan=1 colspan=2>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>A/Shorebird/DE/75/2004</td><td rowspan=1 colspan=5>H5N7</td><td rowspan=1 colspan=2></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>A/Mule Duck/Bulgaria/328/2011</td><td rowspan=9 colspan=5>H5N8</td><td rowspan=9 colspan=2>7</td><td rowspan=1 colspan=1></td><td rowspan=2 colspan=1></td></tr><tr><td rowspan=1 colspan=1>A/Quail/CA/K1400794/2014</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>A/Ruddy Turnstone/DE/237/91</td><td rowspan=1 colspan=1>1</td><td rowspan=7 colspan=1>13</td></tr><tr><td rowspan=1 colspan=1>A/Shorebird/DE/192/98</td><td rowspan=1 colspan=1>4</td></tr><tr><td rowspan=3 colspan=1>A/Laughing Gull/DE/156/2004</td><td rowspan=3 colspan=1>4</td></tr><tr></tr><tr><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>A/Mallard/Korea/W452/14</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>A/Gyrfalcon/WA/41088-6/14</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>A/Chicken/Italy/9097/97</td><td rowspan=2 colspan=5>H5N9</td><td rowspan=2 colspan=2>2</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>A/Ruddy Turnstone/DE/117/2011</td><td rowspan=1 colspan=1>1</td><td></td></tr><tr><td rowspan=1 colspan=1>Unknown #1- from CDC 2013</td><td rowspan=3 colspan=5>H6N1</td><td rowspan=3 colspan=2>3</td><td rowspan=1 colspan=1>2</td><td rowspan=3 colspan=1>6</td></tr><tr><td rowspan=1 colspan=1>Unknown #11- from CDC 2014</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>A/Shorebird/Delaware Bay/230/2009</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Unknown #12- from CDC 2014</td><td rowspan=2 colspan=5>H6N2</td><td rowspan=2 colspan=2>2</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>−</td></tr><tr><td rowspan=1 colspan=1>A/Turkey/Massachusetts/3740/65</td><td rowspan=1 colspan=1>2</td><td></td></tr><tr><td rowspan=1 colspan=1>A/Mallard/Alberta/58/1989</td><td rowspan=1 colspan=5>H6N4</td><td rowspan=1 colspan=2></td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>2</td></tr><tr><td rowspan=1 colspan=1>Blue-winged Teal Texas</td><td rowspan=3 colspan=5>H6N5</td><td rowspan=3 colspan=2>3</td><td rowspan=1 colspan=1>2</td><td rowspan=3 colspan=1>6</td></tr><tr><td rowspan=1 colspan=1>A/Mallard/Alberta/203/1992</td><td rowspan=1 colspan=1>2</td></tr><tr><td rowspan=1 colspan=1>A/Shearwater/Australia/1/73</td><td rowspan=1 colspan=1>2</td></tr><tr><td rowspan=1 colspan=1>Unknown #13- from CDC 2014</td><td rowspan=1 colspan=5>H6N8</td><td rowspan=1 colspan=2></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>A/Shoveler/Egypt/597/2004</td><td rowspan=9 colspan=5>H7N1</td><td rowspan=7 colspan=2>8</td><td rowspan=1 colspan=1>2</td><td rowspan=9 colspan=1>9</td></tr><tr><td rowspan=1 colspan=1>Unknown #14- from CDC 2014</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Unknown #15- from CDC 2014</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>A/Turkey/Italy/6423-1/99</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=2 colspan=1>A/Parakeet/Netherlands/267497/94</td><td rowspan=2 colspan=1></td></tr><tr></tr><tr><td rowspan=1 colspan=1>A/Ostrich/Zimbabwe/222/96</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>A/Mallard/Alberta/34/2001</td><td rowspan=2 colspan=2></td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>A/Chicken/Italy/1285/2000</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>A/Avian/New York/11678-4/2005</td><td rowspan=4 colspan=5>H7N2</td><td rowspan=4 colspan=2>9</td><td rowspan=1 colspan=1>1</td><td rowspan=4 colspan=1>19</td></tr><tr><td rowspan=1 colspan=1>A/Shorebird/DE/282/2011</td><td rowspan=1 colspan=1>3</td></tr><tr><td rowspan=1 colspan=1>A/Chicken/NY/116124/2003</td><td rowspan=1 colspan=1>3</td></tr><tr><td rowspan=1 colspan=1>A/Pheasant/NJ/30739-9/2000</td><td rowspan=1 colspan=1>3</td></tr></table>

<table><tr><td rowspan=1 colspan=1>A/Ruddy Turnstone/DE/130/99</td><td rowspan=5 colspan=8></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>3</td><td rowspan=5 colspan=3></td><td rowspan=5 colspan=1></td></tr><tr><td rowspan=1 colspan=1>A/Australian Shelduck/Western Australia/1756/83</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>3</td></tr><tr><td rowspan=1 colspan=1>A/Mallard/Ramon/79/14T</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>A/Lesser Noddy/Western Australia/2371/83</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>A/Ruddy Turnstone/DE/282/2011</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>A/Mallard/Netherlands/12/2000</td><td rowspan=8 colspan=8></td><td rowspan=4 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=28 colspan=3>28</td><td></td></tr><tr><td rowspan=1 colspan=1>A/Sanderling/DE/280/2015</td><td rowspan=1 colspan=1></td><td></td></tr><tr><td rowspan=1 colspan=1>A/Mallard/Alberta/27/2001</td><td rowspan=1 colspan=1></td><td></td></tr><tr><td rowspan=1 colspan=1>A/Mallard/Alberta/243/2006</td><td rowspan=1 colspan=1></td><td></td></tr><tr><td rowspan=1 colspan=1>A/Shorebird/DE/552/2006</td><td rowspan=2 colspan=1></td><td rowspan=1 colspan=1>1</td><td></td></tr><tr><td rowspan=1 colspan=1>A/Dunlin/DE/281/2015</td><td rowspan=1 colspan=1>1</td><td></td></tr><tr><td rowspan=1 colspan=1>A/Red Knot/DE/269/2015</td><td rowspan=2 colspan=1></td><td rowspan=1 colspan=1>1</td><td></td></tr><tr><td rowspan=1 colspan=1>A/Ruddy Turnstone/DE/115/2011</td><td rowspan=1 colspan=4></td><td rowspan=1 colspan=1></td><td></td></tr><tr><td rowspan=1 colspan=1>A/Mallard/Alberta/341/2012</td><td rowspan=1 colspan=3></td><td rowspan=8 colspan=5>H7N3</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2></td><td rowspan=1 colspan=2></td></tr><tr><td rowspan=1 colspan=1>A/Ruddy Turnstone/DE/124/2007</td><td rowspan=1 colspan=4></td><td rowspan=2 colspan=1></td><td rowspan=1 colspan=1></td><td></td></tr><tr><td rowspan=1 colspan=1>A/Ruddy Turnstone/DE/503/2011</td><td rowspan=1 colspan=4></td><td rowspan=1 colspan=1>1</td><td></td></tr><tr><td rowspan=1 colspan=1>A/Ruddy Turnstone/DE/282/2006</td><td rowspan=1 colspan=3></td><td rowspan=5 colspan=1>28</td><td rowspan=1 colspan=1></td><td></td></tr><tr><td rowspan=1 colspan=1>A/Ruddy Turnstone/DE/108/2007</td><td></td><td></td><td></td><td rowspan=1 colspan=1></td><td></td></tr><tr><td rowspan=1 colspan=1>A/Shorebird/DE/53/2002</td><td></td><td></td><td></td><td rowspan=1 colspan=1></td><td></td></tr><tr><td rowspan=1 colspan=1>A/Mallard/Alberta/167/2010</td><td></td><td></td><td></td><td rowspan=1 colspan=1>1</td><td></td></tr><tr><td rowspan=1 colspan=1>A/Red Knot/DE/239/2015</td><td rowspan=13 colspan=8>b</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1</td><td></td></tr><tr><td rowspan=1 colspan=1>A/Turkey/CA/K1500529/2015</td><td rowspan=12 colspan=1></td><td rowspan=1 colspan=1>1</td><td></td></tr><tr><td rowspan=1 colspan=1>A/Mallard/Alberta/747/2015</td><td rowspan=1 colspan=1></td><td></td></tr><tr><td rowspan=1 colspan=1>A/Mallard/Alberta/174/2010</td><td rowspan=1 colspan=1>1</td><td></td></tr><tr><td rowspan=1 colspan=1>A/Shorebird/DE/218/2015</td><td rowspan=1 colspan=1>1</td><td></td></tr><tr><td rowspan=1 colspan=1>A/Laughing Gull/DE/22/2002</td><td rowspan=1 colspan=1>1</td><td></td></tr><tr><td rowspan=1 colspan=1>A/Chicken/Chile/176822/2002</td><td rowspan=1 colspan=1>1</td><td></td></tr><tr><td rowspan=1 colspan=1>A/Mallard/Netherlands/12/2000</td><td rowspan=1 colspan=1></td><td></td></tr><tr><td rowspan=1 colspan=1>A/Mallard/OH/14OS823/2015</td><td rowspan=1 colspan=1>1</td><td></td></tr><tr><td rowspan=1 colspan=1>A/Chicken/Chile/184240-4957/2002</td><td rowspan=1 colspan=1></td><td></td></tr><tr><td rowspan=1 colspan=1>A/Canada/RV444/2004</td><td rowspan=1 colspan=1>1</td><td></td></tr><tr><td rowspan=1 colspan=1>A/Chicken/Karachi(Pakistan)/NARC-100/2004</td><td rowspan=1 colspan=1></td><td></td></tr><tr><td rowspan=1 colspan=1>A/Chicken/Saskatchewan/HR10/2007</td><td rowspan=1 colspan=1>1</td><td></td></tr><tr><td rowspan=1 colspan=1>A/Duck/Nanchang/1944/93</td><td rowspan=2 colspan=8>H7N4</td><td rowspan=2 colspan=1>2</td><td rowspan=1 colspan=1></td><td rowspan=2 colspan=3>2</td><td rowspan=2 colspan=1></td></tr><tr><td rowspan=1 colspan=1>A/Ruddy Turnstone/DE/284/2006</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>A/Ruddy Turnstone/DE/262/2006</td><td rowspan=2 colspan=8>H7N5</td><td rowspan=2 colspan=1>2</td><td rowspan=1 colspan=1>1</td><td rowspan=2 colspan=3>2</td><td rowspan=2 colspan=1></td></tr><tr><td rowspan=1 colspan=1>A/Mallard/OH/14OS0821/2015</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>A/Chilean Teal/Chile/9/2013</td><td rowspan=1 colspan=8>H7N6</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=3>1</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>A/Teal/Egypt/835/2004</td><td rowspan=12 colspan=8>H7N7</td><td rowspan=2 colspan=1></td><td rowspan=1 colspan=1>2</td><td rowspan=2 colspan=3></td><td rowspan=12 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Unknown #16- from CDC 2014</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>A/Equine/Prague/56</td><td rowspan=2 colspan=1></td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=2></td><td rowspan=1 colspan=2></td></tr><tr><td rowspan=1 colspan=1>A/Ruddy Turnstone/NJ/AI11-1678/2011</td><td rowspan=1 colspan=4></td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=2></td><td rowspan=1 colspan=2></td></tr><tr><td rowspan=1 colspan=1>A/Ruddy Turnstone/DE/134/99</td><td rowspan=1 colspan=4></td><td rowspan=5 colspan=1>10</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=3 colspan=1>A/Duck/Potsdam/S28716/88</td><td></td></tr><tr><td></td></tr><tr><td rowspan=1 colspan=1>2</td></tr><tr><td rowspan=1 colspan=1>A/Red Knot/DE/259/94</td><td rowspan=1 colspan=1>3</td></tr><tr><td rowspan=1 colspan=1>A/Seal/MA/1/80</td><td rowspan=2 colspan=1></td><td rowspan=1 colspan=1>2</td></tr><tr><td rowspan=1 colspan=1>A/Turkey/Ireland/PV74/95</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>A/Netherlands/219/2003</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>A/Mallard/Alberta/579/2010</td><td rowspan=2 colspan=8>H7N8</td><td rowspan=2 colspan=1>2</td><td rowspan=1 colspan=1>1</td><td rowspan=2 colspan=3>2</td><td rowspan=2 colspan=1></td></tr><tr><td rowspan=1 colspan=1>A/Turkey/IN/1573-2/2016</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>A/Anhui/1/2013</td><td rowspan=3 colspan=8>H7N9</td><td rowspan=2 colspan=1>9</td><td rowspan=1 colspan=1>1</td><td rowspan=3 colspan=3>22</td><td rowspan=3 colspan=1></td></tr><tr><td rowspan=1 colspan=1>A/Turkey/MN/037767/2009</td><td rowspan=1 colspan=1>3</td></tr><tr><td rowspan=1 colspan=1>A/Mallard/Alberta/177/2004</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>3</td></tr></table>

<table><tr><td rowspan=3 colspan=1>A/Mallard/Alberta/114/99</td><td rowspan=8 colspan=4></td><td rowspan=6 colspan=2></td><td rowspan=3 colspan=1>4</td><td></td></tr><tr><td rowspan=54 colspan=1>2350</td></tr><tr><td rowspan=4 colspan=1></td></tr><tr><td rowspan=1 colspan=1>A/Shorebird/DE/28/95</td><td rowspan=1 colspan=1>3</td></tr><tr><td rowspan=1 colspan=1>A/Turkey/MN/1/88</td><td rowspan=1 colspan=1>2</td></tr><tr><td rowspan=1 colspan=1>A/Hong Kong/5942/2013</td><td rowspan=1 colspan=1>2</td></tr><tr><td rowspan=1 colspan=1>A/Taiwan/T1.4/2013</td><td rowspan=2 colspan=2></td><td rowspan=1 colspan=1>2</td><td rowspan=2 colspan=1></td></tr><tr><td rowspan=1 colspan=1>A/Shanghai/1/2013</td><td rowspan=1 colspan=1>2</td></tr><tr><td rowspan=1 colspan=1>A/Turkey/Ontario/6118/68</td><td rowspan=1 colspan=4>H8N4</td><td rowspan=1 colspan=2></td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>2</td></tr><tr><td rowspan=1 colspan=1>A/Shorebird/DE/133/2002</td><td rowspan=3 colspan=4>H9N1</td><td rowspan=3 colspan=2>3</td><td rowspan=1 colspan=1>1</td><td rowspan=3 colspan=1>3</td></tr><tr><td rowspan=1 colspan=1>A/Laughing Gull/DE/5/2003</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>A/Ruddy Turnstone/NJ/A107-283/2007</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>A/Chicken/Hong Kong/g9/1997</td><td rowspan=4 colspan=4></td><td rowspan=1 colspan=2></td><td rowspan=1 colspan=1></td><td rowspan=3 colspan=1></td></tr><tr><td rowspan=1 colspan=1>A/Hong Kong/1073/99</td><td rowspan=2 colspan=2></td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>A/Shorebird/Delaware Bay/127/2003</td><td rowspan=1 colspan=1>4</td></tr><tr><td rowspan=1 colspan=1>A/Turkey/Wisconsin/1/66</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2></td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>A/Pheasant/UAE/D1307.B/2011</td><td rowspan=2 colspan=4></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>A/Quail/Lebanon/272/2010</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2></td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>A/Laughing Gull/DE/12/2006</td><td rowspan=34 colspan=6>H9N2                      40</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>A/Environment/Bangladesh/10306/11 (quail cage)</td><td rowspan=1 colspan=2></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>A/Chicken/Bangladesh/10450/11</td><td rowspan=1 colspan=2></td><td rowspan=1 colspan=2></td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>A/Chicken/Beijing/1/94</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2></td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>A/ Quail /Hong Kong/G1/97</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>A/Duck/Hong Kong/Y280/97</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>A/Sanderling/DE/449/2006</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>A/Hong Kong/69955/2008</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>A/Quail/Lebanon/273/2010</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>A/Hong Kong/33982/2009</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>A/Shorebird/DE/464/2011</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>A/Chukar/Shantou/22116/2005</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>A/Chicken/Hong Kong/NT10/2011</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>A/Chicken/Hong Kong/CRA45/2010</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>A/Chicken/Hong Kong/YU341/2008</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>A/American Oystercatcher/Chile/C1307/2105</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>A/Chicken/Egypt/S4454B/2011</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>A/Chicken/Bangladesh/25947/2015</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>A/Chicken/Egypt/S4456B/2011</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>A/Chicken/Egypt/S5018B/2012</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>A/Chicken/Egypt/D7663A/2013</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>A/Duck/Egypt/C9787/2014</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>A/Quail/Egypt/D9842/2014</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>A/Chicken/Egypt/S10489C/2015</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>A/Chicken/Egypt/F10533D/2015</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>A/Chicken/Egypt/D10553A/2015</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>A/Chicken/Egypt/D10561/2015</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>A/Chicken/Egypt/A1093D/2015</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>A/Chicken/Egypt/S10598E/2015</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>A/Chicken/Egypt/D10705/2015</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>A/Chicken/Egypt/D10975/2015</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>A/Pheasant/Shantou/2785/2015</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>A/Chukar/Shantou/2777/2015</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>A/Pigeon/Shantou/3577/2015</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>A/Mallard/Ireland/PV46B/93</td><td rowspan=1 colspan=4>H9N3</td><td rowspan=1 colspan=2>1</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>A/Shorebird/DE/231/2003</td><td rowspan=1 colspan=4>H9N4</td><td rowspan=1 colspan=2>1</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td></td><td rowspan=1 colspan=4>H9N5</td><td rowspan=1 colspan=2></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr></table>

<table><tr><td rowspan=1 colspan=1>A/Mallard/Alberta/162/2007</td><td rowspan=1 colspan=2></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>A/Ruddy Turnstone/DE/268/2011</td><td rowspan=1 colspan=2>H9N6</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=2>1</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>A/Shorebird/Delaware Bay/31/1996</td><td rowspan=3 colspan=2>H9N7</td><td rowspan=2 colspan=1>3</td><td rowspan=1 colspan=2>2</td><td rowspan=2 colspan=1>4</td></tr><tr><td rowspan=1 colspan=1>A/Shorebird/DE/277/2000</td><td rowspan=1 colspan=2></td></tr><tr><td rowspan=1 colspan=1>A/Ruddy Turnstone/DE/253/2011</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2>1</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>A/Ruddy Turnstone/DE/116/98</td><td rowspan=1 colspan=2>H9N8</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=2>1</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>A/Pheasant/WA/373/49/85</td><td rowspan=4 colspan=2>H9N9</td><td rowspan=4 colspan=1>4</td><td rowspan=1 colspan=2>1</td><td rowspan=4 colspan=1>4</td></tr><tr><td rowspan=1 colspan=1>A/Ruddy Turnstone/Virginia/2297/1988</td><td rowspan=1 colspan=2>1</td><td rowspan=1 colspan=1>b</td></tr><tr><td rowspan=1 colspan=1>A/Shorebird/DE/141/2002</td><td rowspan=1 colspan=2>1</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>A/Shorebird/DE/554/2007</td><td rowspan=1 colspan=2>1</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>A/Teal/Egypt/0457/2003</td><td rowspan=4 colspan=2>H10N1</td><td rowspan=4 colspan=1>4</td><td rowspan=1 colspan=2>2</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Unknown #17- from CDC 2014</td><td rowspan=1 colspan=2>2</td><td rowspan=1 colspan=1>b</td></tr><tr><td rowspan=1 colspan=1>A/Mallard/Wisconsin/4230/2009</td><td rowspan=1 colspan=2>2</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>A/Shorebird/Delaware Bay/338/2009</td><td rowspan=1 colspan=2>1</td><td></td></tr><tr><td rowspan=1 colspan=1>A/Shorebird/Delaware Bay/63/1996</td><td rowspan=1 colspan=2>H10N2</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2>2</td><td rowspan=1 colspan=1>2</td></tr><tr><td rowspan=1 colspan=1>Unknown #1- from CDC 2012</td><td rowspan=8 colspan=2>H10N7</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2>1</td><td rowspan=2 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Green-winged Teal Texas</td><td rowspan=1 colspan=1></td><td rowspan=2 colspan=1></td><td rowspan=1 colspan=2>2</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>A/Shoveler/Egypt/0600/2004</td><td rowspan=1 colspan=2>2</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>A/Shoveler/Egypt/845/2004</td><td rowspan=2 colspan=1>8</td><td rowspan=1 colspan=2>2</td><td rowspan=3 colspan=1>15</td></tr><tr><td rowspan=1 colspan=1>Unknown #18- from CDC 2014</td><td rowspan=1 colspan=2>2</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Unknown #19- from CDC 2014</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2>2</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>A/Mallard/Illinois/100S4334/2010</td><td rowspan=2 colspan=1></td><td rowspan=1 colspan=2>2</td><td rowspan=2 colspan=1></td></tr><tr><td rowspan=1 colspan=1>A/Chicken/Germany/N/49</td><td rowspan=1 colspan=2>2</td></tr><tr><td rowspan=1 colspan=1>A/Quail/Italy/1117/65</td><td rowspan=1 colspan=2>H10N8</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=2>2</td><td rowspan=1 colspan=1>2</td></tr><tr><td rowspan=1 colspan=1>Unknown #20- from CDC 2014</td><td rowspan=1 colspan=2>H11N1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=2>2</td><td rowspan=1 colspan=1>2</td></tr><tr><td rowspan=1 colspan=1>A/Shorebird/Delaware Bay/216/1999</td><td rowspan=2 colspan=2>H11N2</td><td rowspan=2 colspan=1>2</td><td rowspan=1 colspan=2>2</td><td rowspan=2 colspan=1>4</td></tr><tr><td rowspan=1 colspan=1>A/Laughing Gull/Delaware Bay/94/1995</td><td rowspan=1 colspan=2>2</td></tr><tr><td rowspan=1 colspan=1>A/Duck/Bangladesh/1052/2007</td><td rowspan=2 colspan=2>H11N3</td><td rowspan=2 colspan=1>2</td><td rowspan=1 colspan=2>2</td><td rowspan=2 colspan=1>4</td></tr><tr><td rowspan=1 colspan=1>A/Ruddy Turnstone/Delaware Bay/39/1994</td><td rowspan=1 colspan=2>2</td></tr><tr><td rowspan=1 colspan=1>A/Mallard/Alberta/125/1999</td><td rowspan=2 colspan=2>H11N6</td><td rowspan=2 colspan=1>2</td><td rowspan=1 colspan=2>2</td><td rowspan=2 colspan=1>4</td></tr><tr><td rowspan=1 colspan=1>A/Duck/England/56</td><td rowspan=1 colspan=2>2</td></tr><tr><td rowspan=1 colspan=1>Unknown #21- from CDC 2014</td><td rowspan=5 colspan=2>H11N9</td><td rowspan=5 colspan=1>5</td><td rowspan=1 colspan=2>2</td><td rowspan=3 colspan=1>11</td></tr><tr><td rowspan=1 colspan=1>Unknown #22- from CDC 2014</td><td rowspan=1 colspan=2>2</td></tr><tr><td rowspan=1 colspan=1>A/American Green-Winged Teal/Mississippi/300/2010</td><td rowspan=1 colspan=2>2</td></tr><tr><td rowspan=1 colspan=1>A/Common Goldeneye/Iowa/3192/2009</td><td rowspan=1 colspan=2>2</td><td rowspan=2 colspan=1></td></tr><tr><td rowspan=1 colspan=1>A/Duck/Memphis/546/74</td><td rowspan=1 colspan=2>3</td></tr><tr><td rowspan=1 colspan=1>A/Mallard/Wisconsin/4218/2009</td><td rowspan=5 colspan=2>H12N5</td><td rowspan=5 colspan=1>5</td><td rowspan=1 colspan=2>2</td><td rowspan=5 colspan=1>10</td></tr><tr><td rowspan=1 colspan=1>A/Mallard/Ohio/1688/2009</td><td rowspan=1 colspan=2>2</td></tr><tr><td rowspan=1 colspan=1>A/Northern Shoveler/Mississippi/09OS025/2009</td><td rowspan=1 colspan=2>2</td></tr><tr><td rowspan=1 colspan=1>A/Northern Pintail/Missouri/319/2009</td><td rowspan=1 colspan=2>2</td></tr><tr><td rowspan=1 colspan=1>A/Duck/Alberta/60/76</td><td rowspan=1 colspan=2>2</td></tr><tr><td rowspan=1 colspan=1>A/Black-Legged Kittiwake/Quebec/02838-1/2009</td><td rowspan=3 colspan=2>H13N6</td><td rowspan=2 colspan=1>3</td><td rowspan=1 colspan=2>2</td><td rowspan=3 colspan=1>5</td></tr><tr><td rowspan=1 colspan=1>A/Ring-Billed Gull/Quebec/02434-1/2009</td><td rowspan=1 colspan=2>2</td></tr><tr><td rowspan=1 colspan=1>A/Gull/Maryland/704/77</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2>1</td></tr><tr><td rowspan=1 colspan=1>Unknown #23- from CDC 2014</td><td rowspan=2 colspan=2>H14N5</td><td rowspan=2 colspan=1>2</td><td rowspan=1 colspan=2>2</td><td rowspan=2 colspan=1>4</td></tr><tr><td rowspan=1 colspan=1>A/Mallard/Astrakhan/263/82</td><td rowspan=1 colspan=2>2</td></tr><tr><td rowspan=1 colspan=1>A/Shearwater/Australia/2576/79</td><td rowspan=1 colspan=2>H15N9</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=2>2</td><td rowspan=1 colspan=1>2</td></tr><tr><td rowspan=1 colspan=1>A/Shorebird/Delaware/172/2006</td><td rowspan=1 colspan=2>H16N3</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=2>2</td><td rowspan=1 colspan=1>2</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=2>62 Subtypes</td><td rowspan=1 colspan=1>352</td><td rowspan=1 colspan=2>759</td><td rowspan=1 colspan=1>759</td></tr></table>

Note: H1N1 (pre 2009), H1N1 Avian, and H1N1 Swine are counted as one subtype; H3N2 Swine and H3N2v are counted as one subtype.

Performance of the neural network algorithm was verified utilizing the technique of $\mathbf { k }$ -fold cross-validation $( \mathrm { k } { = } 1 0 )$ ) that allows assessment of how the neural network algorithm will generalize to a completely independent dataset. This mathematical approach was accomplished by dividing the training set randomly into 10 groups. All groups but one are used to train the neural network algorithm, and the algorithm is tested on the remaining group. Then, the groups used for training are rotated until all combinations of nine training groups/one test group have been used. The performance of each test group was then compiled to estimate performance expected on an independent dataset. Therefore, while viruses in the training set were not technically naïve to the training, this approach allowed an estimation of performance as if each portion of the training set had been naïve prior to validation activities.

# Postmarket Surveillance and Software Update Plan

InDevR developed a plan to monitor genetic changes in the circulating strains of influenza annually in order to assess the impact of those changes on the results generated by the FC8G assay. InDevR will test the CDC human influenza panel (includes seasonal influenza A and influenza B strains) and the CDC animal influenza panel (includes select, important nonseasonal strains with pandemic potential) annually, if appropriate and available. If the CDC human and animal influenza panels are not available or not appropriate for testing with the FC8G assay, e.g., virus samples inactivated with Betapropiolactone (BPL) are not compatible with the FC8G assay, etc. InDevR will conduct similar experiments on similar panels compiled from other sources (such as commercial sources and/or collaborators). Such a human influenza panel would consist of two strains of each target virus group, and include strains genetically similar to those included in the annual quadrivalent vaccine formulation for the four components in addition to two pre-2009 H1N1 strains. Composition of such an animal influenza panel will be based on recommendations from the CDC and other surveillance laboratories regarding selecting important non-seasonal strains with pandemic potential, and will be subject to availability limitations. In addition, if testing and reporting from CDC or other surveillance laboratory suggests that a newly emerging strain of influenza is on the rise in the US and it is not represented in the CDC panels, then InDevR will work with CDC and/or surveillance laboratories to test the newly emerging strain with the FC8G assay to determine its reactivity with the newly emerging strain.

If the annual reactivity testing indicates that the assay is not reactive to one or more of the strains tested, InDevR will further investigate by testing other similar strains to confirm that a systematic non-reactivity exists and that this failure to detect the virus was not due to an isolated assay failure or issue. If a change in reactivity is confirmed, the discovery will be reported to the FDA. An update to the training database and re-optimization of the neural network parameters would likely be necessary.

Once data have been generated that indicate an update to the training database is necessary, InDevR will obtain as many strains representing the non-reactive group as possible. If the virus(es) exhibiting low reactivity were initially emerging, it may be difficult to obtain a large number of strains. Even with a relatively low number of strains, it may be possible to create a robust collection of images by analyzing strains in replicates, as well as analyzing each additional strain at a variety of concentrations by preparing a contrived panel spiked at a variety of concentrations into pooled clinical negative material. These samples will then be analyzed to generate FC8G assay images. Once FC8G images are obtained for these samples, InDevR will utilize the same criteria currently in place for determining suitability for inclusion in the training database. Once the training database update is complete, InDevR will follow the same general procedure used during the initial software development to optimize the neural network parameters and finalize the software update. In brief, InDevR will utilize the FC8G Data Tools software developed for neural network training to optimize the neural network parameters using a $\mathrm { k \Omega }$ -fold cross-validation approach. Once the neural network parameters have been optimized, InDevR will optimize the thresholds applied to each neural network and sub-net (assay cutoff) to ensure the proper balance of sensitivity and specificity. After the assay thresholds have been optimized, the FC8G Software code would be updated accordingly. Final validation of the updated software with the new neural network parameters and cutoffs will also require bench testing of training-naïve samples of the strain(s) in question.

Once validation of the newly updated software is completed, InDevR will submit a 510(k) to the FDA for the modified device.

# Results Interpretation

# Interpretation of External Quality Control (QC) Results

External Batch Positive Control (user supplied influenza positive specimen): Interpret the External Batch Positive Control results (if included) to determine whether the expected result of influenza positive was obtained and if the batch of samples is valid. Failure to obtain the expected influenza positive result should invalidate the batch, and all samples included in that batch should be retested.

External Batch Negative Control (user supplied influenza negative clinical specimen): Interpret the External Batch Negative Control results (if included) to determine whether the expected result of “Influenza Not Detected” was obtained and if the batch of samples is valid. Failure to obtain the expected “Influenza Not Detected” result should invalidate the batch, and all samples included in that batch should be retested.

• Extraction Negative Control (nuclease-free water used in place of specimen during extraction):

Interpret the Extraction Negative Control results to determine whether the expected result of either “Influenza Not Detected” or “No Call: Assay Failure (Internal Control)” was obtained and if the batch of samples are valid. Failure to obtain either “Influenza Not Detected” or “No Call: Assay Failure (Internal Control)” result should invalidate the batch, and all samples included in that batch should be retested.

• RT-PCR No Template Control (nuclease-free water used in place of template during RT-PCR reaction setup):

Interpret the RT-PCR No Template Control results to determine whether the expected result of either “Influenza Not Detected” or “No Call: Assay Failure (Internal Control)” was obtained and if the batch of samples are valid. Failure to obtain either “Influenza Not Detected” or “No Call: Assay Failure (Internal Control)” result should invalidate the batch, and all samples included in that batch should be retested.

# Interpretation of Assay Results

The table below lists the expected results for the FluChip-8G $\mathbf { A } { + } \mathbf { B }$ Assay. The FluChip-8G Software automatically determines the sample result and qualified testing personnel should interpret the sample results as follows:

<table><tr><td colspan="1" rowspan="1">If Result is:</td><td colspan="1" rowspan="1">Explanation</td><td colspan="1" rowspan="1">Interpretation of Results</td></tr><tr><td colspan="1" rowspan="1">Influenza not Detected</td><td colspan="1" rowspan="1">Valid negative result.</td><td colspan="1" rowspan="6">Report Result</td></tr><tr><td colspan="1" rowspan="1">Influenza A Detected, Seasonal H1N1(2009)</td><td colspan="1" rowspan="4">Valid positive result.</td></tr><tr><td colspan="1" rowspan="1">Influenza A Detected, Seasonal H3N2</td></tr><tr><td colspan="1" rowspan="1">Influenza B Detected, Victoria Lineage</td></tr><tr><td colspan="1" rowspan="1">Influenza B Detected, Yamagata Lineage</td></tr><tr><td colspan="1" rowspan="1">Influenza B Detected</td><td colspan="1" rowspan="1">Valid positive result.Low virus titers can result in detection ofinfluenza B without an associated lineage.</td></tr><tr><td colspan="1" rowspan="1">Influenza A Detected: Equivocal Result(no subtyping)</td><td colspan="1" rowspan="1">Low virus titers can result in detection ofinfluenza A without an associated subtype.Detection of influenza A without a subtypecan also in rare cases indicate the presenceof a novel strain.</td><td colspan="1" rowspan="1">Retest Specimen to Confirm ResultIf the retest provides a different result, test thesample a third time to ensure the accuracy of theresult.In the rare instance that all 3 results conflict, theresult should be considered inconclusive and anew specimen should be collected and tested ifpossible.If an A/equivocal result is confirmed throughretesting described above, contact appropriatepublic health authorities for confirmatorytesting.</td></tr><tr><td colspan="1" rowspan="1">Influenza A Detected, Non-Seasonal</td><td colspan="1" rowspan="1">This result indicates the presumptivepresence of a novel influenza A virus otherthan the annually circulatingA/H1N1pdm09 and A/H3N2 viruses.The A/H1N1 virus circulating prior to theappearance of A/H1N1pdm09 is expectedto be identified as "Influenza A Detected,Non-Seasonal".</td><td colspan="1" rowspan="1">Retest Specimen to Confirm ResultIf the retest provides a different result, test thesample a third time to ensure the accuracy of theresult.In the rare instance that all 3 results conflict, theresult should be considered inconclusive and anew specimen should be collected and tested ifpossible.. If an A/non-seasonal result is confirmed throughretesting described above, contact theappropriate public health authorities forconfirmatory testing.</td></tr><tr><td colspan="1" rowspan="1">Multiple Infections Detected(any combination of the individual resultsabove)</td><td colspan="1" rowspan="1">This result indicates co-infection withmultiple influenza virus types and/orsubtypes. Multiple infections are possiblebut rare. This result may also occur inpatients that have recently received anattenuated live intranasal influenza vaccine.If one of the targets detected is 'InfluenzaA Detected, Non-Seasonal', this resultindicates the presumptive presence of anovel influenza A virus other than theannually circulating A/H1N1pdm09 andA/H3N2 viruses.</td><td colspan="1" rowspan="1">Retest Specimen to Confirm Result. If the retest provides a different result, test thesample a third time to ensure the accuracy of theresult.In the rare instance that all 3 results conflict, theresult should be considered inconclusive and anew specimen should be collected and tested ifpossible.If a confirmed multiple infection result includesan A/equivocal or A/non-seasonal result,contact appropriate public health authorities forconfirmatory testing.</td></tr><tr><td colspan="1" rowspan="1">No Call: Assay Failure (Internal Control)</td><td colspan="1" rowspan="1">This result indicates the internal controlfailed to be detected. This result typicallyindicates poor specimen collection,specimen degradation, or user error in thespecimen extraction or RT-PCR steps.</td><td colspan="1" rowspan="1">Retest SpecimenIf upon retest a valid result is generated, followthe recommended actions for that result.</td></tr><tr><td colspan="1" rowspan="1">No Call: Assay Failure (HybridizationControl)</td><td colspan="1" rowspan="1">This result is likely due to an issue in thehybridization step.</td><td colspan="1" rowspan="2">Retest SpecimenIf upon retest a valid result is generated, followthe recommended actions for that result.</td></tr><tr><td colspan="1" rowspan="1">No Call: Assay Failure (Labeling Control)</td><td colspan="1" rowspan="1">This result is likely due to an issue in thepost- hybridization labeling step.</td></tr><tr><td colspan="1" rowspan="1">Error: Image Processing Failure</td><td colspan="1" rowspan="1">This may indicate an error in microarrayprocessing (See final wash step in assayprocedure), in the placement of theFluChip-8G Microarray Slide within theFluChip-8G Imaging System, or theFluChip-8G Imaging System focus setting.</td><td colspan="1" rowspan="1">Visually inspect the back of the slide forartifacts such as dust or salt precipitates. Using adamp lint-free cloth, wipe the back of the slideand Rescan the Entire Slide.Check for correct slide and slide holderorientation, correct any orientation errors andRescan the Entire Slide.See FluChip-8G Imaging System OperationManual for instructions on confirming systemfocus, Rescan the Entire Slide.If upon rescan a valid result is generated for thesample that produced the error, follow therecommended actions for that result. Forsamples that did not produce the error uponinitial scan the result from the 1st scan shouldbe used.If upon rescan a valid result is not generated,retest the specimen.</td></tr><tr><td colspan="1" rowspan="1">Not Analyzed</td><td colspan="1" rowspan="1">Microarray designated as "EMPTY" duringsample type selection in FluChip-8Gsoftware. The microarray will not beanalyzed and no result will be displayed</td><td colspan="1" rowspan="1">If the position was unintentionally left empty,rescan the entire slide using the correct sampleID selection. For samples that did not producethe result of "Not Analyzed" in the initial scanthe result from the 1st scan should be used.</td></tr></table>

# J. Substantial Equivalence Information:

1. Predicate device name(s): CDC Human Influenza Virus Real-Time PCR Diagnostic Panel

2. Predicate 510(k) number(s): K172091

3. Comparison with predicate:

<table><tr><td colspan="3" rowspan="1">Similarities and Differences</td></tr><tr><td colspan="1" rowspan="2">Item</td><td colspan="1" rowspan="1">Device</td><td colspan="1" rowspan="1">Predicate</td></tr><tr><td colspan="1" rowspan="1">FluChip-8G Influenza A+B Assay(K182513)</td><td colspan="1" rowspan="1">CDC Human Influenza Virus Real-Time PCR DiagnosticPanel(K172091)</td></tr><tr><td colspan="1" rowspan="1">Measurand</td><td colspan="1" rowspan="1">Influenza RNA</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Viruses Detected</td><td colspan="1" rowspan="1">Influenza A and B viruses</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Influenza A subtypeDifferentiation</td><td colspan="1" rowspan="1">Seasonal influenza A/H1N1pdm09Seasonal influenza A/H3N2"Non-seasonal" influenza A</td><td colspan="1" rowspan="1">SameSameInfluenza A/H5 (Asian Lineage)</td></tr><tr><td colspan="1" rowspan="1">Influenza B lineageDifferentiation</td><td colspan="1" rowspan="1">Influenza B/VictoriaInfluenza B/Yamagata</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Assay Type</td><td colspan="1" rowspan="1">Qualitative</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">CLIA Complexity</td><td colspan="1" rowspan="1">High</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Sample Type</td><td colspan="1" rowspan="1">Nasal swab and nasopharyngeal swabspecimens</td><td colspan="1" rowspan="1">Upper respiratory tract specimens, including nasal swabs andnasopharyngeal swabs; lower respiratory tract specimens</td></tr><tr><td colspan="1" rowspan="1">Sample PreparationMethod</td><td colspan="1" rowspan="1">Nucleic Acid Extraction</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Detection Technology</td><td colspan="1" rowspan="1">Fluorescence</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">TechnologicalPrinciples</td><td colspan="1" rowspan="1">RT-PCR followed by endpoint detection bymicroarray via nucleic acid probes andfluorescence</td><td colspan="1" rowspan="1">Real-time RT-PCR via nucleic acid probes and fluorescence</td></tr><tr><td colspan="1" rowspan="1">Nucleic AcidExtraction</td><td colspan="1" rowspan="1">QIAamp DSP Virus Spin Kit</td><td colspan="1" rowspan="1">QIAamp Viral RNA Mini Kit, QiagenQIAcube with QIAamp Viral RNA Mini Kit•MagNA Pure Compact—Total Nucleic Acid Kit, Roche•MagNA Pure Compact—RNA Isolation Kit, Roche•MagNA Pure LC—RNA Isolation Kit II, RocheNucliSENS easyMAG, bioMerieux</td></tr><tr><td colspan="1" rowspan="1">Controls</td><td colspan="1" rowspan="1">Internal Control in each specimenExternal positive control processed with eachbatchExternal negative control processed witheach batch</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">AmplificationReagents</td><td colspan="1" rowspan="1">Quanta Biosciences qScript One-Step RT-PCR Kit</td><td colspan="1" rowspan="1">Quanta Biosciences qScript One-Step qRT-PCR Kit, lowROXInvitrogen SuperScript III Platinum One-Step QuantitativeRT-PCR Kit</td></tr><tr><td colspan="1" rowspan="1">Results Interpretation</td><td colspan="1" rowspan="1">Automated test interpretation via neuralnetwork-based algorithm using fluorescenceintensities</td><td colspan="1" rowspan="1">Manual test interpretation via Ct value determined fromfluorescence intensities</td></tr><tr><td colspan="1" rowspan="1">Instrumentation</td><td colspan="1" rowspan="1">FluChip-8G Imaging System</td><td colspan="1" rowspan="1">Applied Biosystems 7500 Dx Real-Time PCR Instrument</td></tr></table>

# K. Standard/Guidance Document Referenced (if applicable):

None

# L. Test Principle:

The FluChip-8G Influenza $\mathbf { A } { + } \mathbf { B }$ Assay (FC8G assay) includes a multiplex primer mix to amplify the hemagglutinin (HA), neuraminidase (NA), matrix (M), non-structural (NS), and nucleoprotein (NP) gene segments of all influenza A viruses, as well as the hemagglutinin (HA) and neuraminidase (NA) gene segments of all influenza B viruses, and a portion of the 18s gene present in eukaryotic cells as an endogenous internal control. During a one-step RTPCR reaction, the primers and RT-PCR enzymes and reagents are utilized to amplify nucleic acids extracted from human nasal specimens while incorporating biotin for downstream fluorescence labeling. The biotinylated amplified products are then heat-fragmented and applied to the FC8G microarray for hybridization, labeling, and detection using the FluChip8G Imaging System. The system software is trained to identify the patterns of the influenza target virus groups using a large database of thousands of results from viruses of known characterization, and a simple result in the form of a report is returned to the end user via the software user interface.

# M. Performance Characteristics (if/when applicable):

1. Analytical performance:

a. Precision/Reproducibility:

Two independent studies: within-laboratory repeatability and external multi-site userto-user reproducibility, were carried out to evaluate the precision and reproducibility of the FluChip-8G Influenza $\mathbf { A } { + } \mathbf { B }$ Assay.

A 45-member panel was used for the two studies. The panel was made with five viral strains that represent the five target influenza virus groups: A/New Caledonia/20/1999 (A/non-seasonal), A/California/07/2009 (A/H1N1pdm09), A/Victoria/361/2011 (A/H3N2), B/Wisconsin/01/2010 (B/Yamagata), and B/Florida/02/2006 (B/Victoria). Each strain was diluted into pooled influenza

negative clinical material at three different target levels: Moderate Positive (MP) (approximately 3xLoD), Low Positive (LP) (approximately 1xLoD), and High Negative (HN) (approximately 0.1xLoD). Influenza negative clinical material consisted of clinical nasopharyngeal swab samples stabilized in universal transport medium (UTM) that were determined to be negative for the presence of influenza virus via an FDA-cleared influenza molecular assay.

# External Multi-Site User-to-User Reproducibility Study

Two operators at each of the three separate laboratory sites performed the FluChip8G Influenza $\mathbf { A } { + } \mathbf { B }$ Assay (FC8G assay) according to the assay instructions for use. Performance was evaluated on six unique testing points at the three separate laboratory sites for a total of 18 testing points. Each testing point corresponds to a unique run/execution of the FC8G assay, where each member of the blinded 45-member panel of contrived samples was tested in triplicate. Two unique lots of FC8G assay reagents were evenly divided between the three sites. At each site, each of the two operators executed three testing points. Operators at each site alternated testing until all six test points/site were completed. Each testing point was separated by a minimum of one full day in which no FC8G assay processing for this study occurred.

A total of 810 data points (45 samples/testing point x 18 testing points) were generated in this study. Performance was evaluated by determining the agreement with the expected results, where the MP and LP concentrations were expected to be positive for the intended target, and the HN was expected to be negative.

Table 3 below shows the results of the multi-site user-to-user reproducibility study.

Table 3: Multi-Site User-to-User Reproducibility Study Results   

<table><tr><td colspan="2" rowspan="2"></td><td colspan="3" rowspan="1">Site A</td><td colspan="3" rowspan="1">Site B</td><td colspan="3" rowspan="1">Site C</td><td colspan="3" rowspan="2">Overall % Agreement and95% Confidence Intervalby Sample (%)</td></tr><tr><td colspan="2" rowspan="1">Count and% Agreement</td><td colspan="1" rowspan="1">95%ConfidenceInterval(%)</td><td colspan="2" rowspan="1">Count and% Agreement</td><td colspan="1" rowspan="1">95%ConfidenceInterval(%)</td><td colspan="2" rowspan="1">Count and% Agreement</td><td colspan="1" rowspan="1">95%ConfidenceInterval(%)</td></tr><tr><td colspan="1" rowspan="3">A/H1N1 (pre-2009)(A/non-seasonal)</td><td colspan="1" rowspan="1">MP (3x LoD)</td><td colspan="1" rowspan="1">18/18</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">82.4 - 100</td><td colspan="1" rowspan="1">18/18</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">82.4 - 100</td><td colspan="1" rowspan="1">17/18a</td><td colspan="1" rowspan="1">94%</td><td colspan="1" rowspan="1">74.2 - 99.0</td><td colspan="1" rowspan="1">53/54</td><td colspan="1" rowspan="1">98%</td><td colspan="1" rowspan="1">90.2 - 99.7</td></tr><tr><td colspan="1" rowspan="1">LP (1x LoD)</td><td colspan="1" rowspan="1">18/18</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">82.4 - 100</td><td colspan="1" rowspan="1">18/18</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">82.4 - 100</td><td colspan="1" rowspan="1">18/18</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">82.4 - 100</td><td colspan="1" rowspan="1">54/54</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">93.4 - 100</td></tr><tr><td colspan="1" rowspan="1">HN (0.1x LoD)</td><td colspan="1" rowspan="1">7/18b</td><td colspan="1" rowspan="1">39%</td><td colspan="1" rowspan="1">20.3 - 61.4</td><td colspan="1" rowspan="1">11/18c</td><td colspan="1" rowspan="1">61%</td><td colspan="1" rowspan="1">38.6 - 79.7</td><td colspan="1" rowspan="1">13/18d</td><td colspan="1" rowspan="1">72%</td><td colspan="1" rowspan="1">49.1 -87.5</td><td colspan="1" rowspan="1">31/54</td><td colspan="1" rowspan="1">57%</td><td colspan="1" rowspan="1">44.2 - 69.7</td></tr><tr><td colspan="1" rowspan="3">A/H1N1pdm09</td><td colspan="1" rowspan="1">MP (3x LoD)</td><td colspan="1" rowspan="1">18/18</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">82.4 - 100</td><td colspan="1" rowspan="1">18/18</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">82.4 - 100</td><td colspan="1" rowspan="1">18/18</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">82.4 - 100</td><td colspan="1" rowspan="1">54/54</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">93.4 - 100</td></tr><tr><td colspan="1" rowspan="1">LP (1x LoD)</td><td colspan="1" rowspan="1">18/18</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">82.4 - 100</td><td colspan="1" rowspan="1">14/18e</td><td colspan="1" rowspan="1">78%</td><td colspan="1" rowspan="1">54.8 - 91.0</td><td colspan="1" rowspan="1">17/18f</td><td colspan="1" rowspan="1">94%</td><td colspan="1" rowspan="1">74.2 - 99.0</td><td colspan="1" rowspan="1">49/54</td><td colspan="1" rowspan="1">91%</td><td colspan="1" rowspan="1">80.1 - 96.0</td></tr><tr><td colspan="1" rowspan="1">HN (0.1x LoD)</td><td colspan="1" rowspan="1">9/188</td><td colspan="1" rowspan="1">50%</td><td colspan="1" rowspan="1">29.0 - 71.0</td><td colspan="1" rowspan="1">11/18h</td><td colspan="1" rowspan="1">61%</td><td colspan="1" rowspan="1">38.6 - 79.7</td><td colspan="1" rowspan="1">7/18i</td><td colspan="1" rowspan="1">39%</td><td colspan="1" rowspan="1">20.3 - 61.4</td><td colspan="1" rowspan="1">27/54</td><td colspan="1" rowspan="1">50%</td><td colspan="1" rowspan="1">37.1 - 62.9</td></tr><tr><td colspan="1" rowspan="3">A/H3N2</td><td colspan="1" rowspan="1">MP (3x LoD)</td><td colspan="1" rowspan="1">18/18</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">82.4 - 100</td><td colspan="1" rowspan="1">18/18</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">82.4 - 100</td><td colspan="1" rowspan="1">18/18</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">82.4 - 100</td><td colspan="1" rowspan="1">54/54</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">93.4 - 100</td></tr><tr><td colspan="1" rowspan="1">LP (1x LoD)</td><td colspan="1" rowspan="1">18/18</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">82.4 - 100</td><td colspan="1" rowspan="1">18/18</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">82.4 - 100</td><td colspan="1" rowspan="1">18/18</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">82.4 - 100</td><td colspan="1" rowspan="1">54/54</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">93.4 - 100</td></tr><tr><td colspan="1" rowspan="1">HN (0.1x LoD)</td><td colspan="1" rowspan="1">17/181</td><td colspan="1" rowspan="1">94%</td><td colspan="1" rowspan="1">74.2 - 99.0</td><td colspan="1" rowspan="1">14/18m</td><td colspan="1" rowspan="1">78%</td><td colspan="1" rowspan="1">54.8 - 91.0</td><td colspan="1" rowspan="1">16/18n</td><td colspan="1" rowspan="1">89%</td><td colspan="1" rowspan="1">67.2 - 96.9</td><td colspan="1" rowspan="1">47/54</td><td colspan="1" rowspan="1">87%</td><td colspan="1" rowspan="1">75.6 - 93.6</td></tr><tr><td colspan="1" rowspan="3">B/Yamagata</td><td colspan="1" rowspan="1">MP (3x LoD)</td><td colspan="1" rowspan="1">18/18</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">82.4 - 100</td><td colspan="1" rowspan="1">18/18</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">82.4 - 100</td><td colspan="1" rowspan="1">18/18</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">82.4 - 100</td><td colspan="1" rowspan="1">54/54</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">93.4 - 100</td></tr><tr><td colspan="1" rowspan="1">LP (1x LoD)</td><td colspan="1" rowspan="1">18/18</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">82.4 - 100</td><td colspan="1" rowspan="1">17/18°</td><td colspan="1" rowspan="1">94%</td><td colspan="1" rowspan="1">74.2 - 99.0</td><td colspan="1" rowspan="1">18/18</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">82.4 - 100</td><td colspan="1" rowspan="1">53/54</td><td colspan="1" rowspan="1">98%</td><td colspan="1" rowspan="1">90.2 - 99.7</td></tr><tr><td colspan="1" rowspan="1">HN (0.1x LoD)</td><td colspan="1" rowspan="1">12/18P</td><td colspan="1" rowspan="1">67%</td><td colspan="1" rowspan="1">43.7 - 83.7</td><td colspan="1" rowspan="1">17/189</td><td colspan="1" rowspan="1">94%</td><td colspan="1" rowspan="1">74.2 - 99.0</td><td colspan="1" rowspan="1">15/18r</td><td colspan="1" rowspan="1">83%</td><td colspan="1" rowspan="1">60.8 - 94.2</td><td colspan="1" rowspan="1">44/54</td><td colspan="1" rowspan="1">81%</td><td colspan="1" rowspan="1">69.2 - 89.6</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">MP (3x LoD)</td><td colspan="1" rowspan="1">18/18</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">82.4 - 100</td><td colspan="1" rowspan="1">18/18</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">82.4 - 100</td><td colspan="1" rowspan="1">18/18</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">82.4 - 100</td><td colspan="1" rowspan="1">54/54</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">93.4 - 100</td></tr><tr><td colspan="1" rowspan="2">B/Victoria</td><td colspan="1" rowspan="1">LP (1x LoD)</td><td colspan="1" rowspan="1">17/18s</td><td colspan="1" rowspan="1">94%</td><td colspan="1" rowspan="1">74.2 - 99.0</td><td colspan="1" rowspan="1">18/18</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">82.4 - 100</td><td colspan="1" rowspan="1">18/18</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">82.4 - 100</td><td colspan="1" rowspan="1">53/54</td><td colspan="1" rowspan="1">98%</td><td colspan="1" rowspan="1">90.2 - 99.7</td></tr><tr><td colspan="1" rowspan="1">HN (0.1x LoD)</td><td colspan="1" rowspan="1">7/18t</td><td colspan="1" rowspan="1">39%</td><td colspan="1" rowspan="1">20.3 - 61.4</td><td colspan="1" rowspan="1">13/18u</td><td colspan="1" rowspan="1">72%</td><td colspan="1" rowspan="1">49.1 -87.5</td><td colspan="1" rowspan="1">6/18v</td><td colspan="1" rowspan="1">33%</td><td colspan="1" rowspan="1">16.3 - 56.3</td><td colspan="1" rowspan="1">26/54</td><td colspan="1" rowspan="1">48%</td><td colspan="1" rowspan="1">35.4 - 61.1</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="10" rowspan="1">Moderate Positive (MP) (3x LoD)Low Positive (LP) (1x LoD)High Negative (HN) (0.1x LoD)</td><td colspan="1" rowspan="1">269/270263/270175/270</td><td colspan="1" rowspan="1">100%97%65%</td><td colspan="1" rowspan="1">97.9 - 99.994.7 - 98.758.9 - 70.3</td></tr></table>

a 1/18 MP replicates was No Call: Assay Failure (Process Control)   
b 7/18 HN replicates were Flu A non-seasonal; $4 / 1 8 \mathrm { H N }$ replicates were Flu A Equivocal   
$^ { \mathrm { c } } 5 / 1 8 \mathrm { H N }$ replicates were Flu A non-seasonal; $2 / 1 8 \mathrm { H N }$ replicates were Flu A Equivocal   
$^ \mathrm { d } 4 / 1 8 \mathrm { H N }$ replicates were Flu A non-seasonal; $1 / 1 8 \mathrm { H N }$ replicates was Flu A Equivocal   
$\mathrm { ~ e ~ } _ { 3 / 1 8 }$ LP replicates were Flu A/H1N1pdm09 and Flu A non-seasonal dual positive; 1/18 LP replicates was Flu A Equivocal   
f 1/18 LP replicates were Flu A/H1N1pdm09 and Flu A non-seasonal dual positive   
$^ { \textrm { g } } 6 / 1 8 \mathrm { H N }$ replicates were Flu A/H1N1pdm09; 3/18 HN replicates were Flu A Equivocal   
h 2/18 HN replicates were Flu A/H1N1pdm09; 3/18 HN replicates were Flu A Equivocal; 2/18 HN replicates were Flu A non-seasonal   
i 5/18 HN replicates were Flu A/H1N1pdm09; 5/18 HN replicates were Flu A Equivocal; 1/18 HN replicates was Flu A non-seasonal   
$^ { 1 } 1 / 1 8 \mathrm { H N }$ replicates was Flu A/H3N2   
m 1/18 HN replicates was Flu A/H3N2; 1/18 HN replicates was Flu A Equivocal; 2/18 HN replicates were Flu A/H3N2 and Flu A non-seasonal dual positive   
n 2/18 HN replicates were Flu A/H3N2   
o 1/18 LP replicates was Influenza Not Detected   
p 2/18 HN replicates were Flu B; 2/18 HN were Flu B Yamagata; 2/18 HN were Flu B Victoria   
q 1/18 HN replicates was Flu B   
r 1/18 HN replicates was Flu B; 2/18 HN were Flu B Yamagata   
s 1/18 LP replicates was Error: Image Processing Failure   
$^ { \mathrm { t } } 1 / 1 8 \mathrm { H N }$ replicates was Flu B; 8/18 HN were Flu B Victoria; 2/18 HN were Flu B Yamagata   
$^ { \mathrm { u } } 1 / 1 8 \mathrm { H N }$ replicates was Flu B; 4/18 HN were Flu B Victoria   
$^ { \mathrm { ~ v ~ } } 1 2 / 1 8 \mathrm { { H N } }$ replicates were Flu B Victoria

In addition, site-to-site, operator-to-operator, run-to-run, and reagent lot-to-lot reproducibility performance were also assessed using the data generated from this multi-site reproducibility study. There were no statistically significant differences in the observed performance between sites, between operators, between runs, or between reagent lots, as indicated by overlapping confidence intervals for all samples expected to be positive.

# Within-Laboratory Precision Study

Two operators at a manufacturer’s internal laboratory site performed testing in the within-laboratory precision study. The same panel of 45 contrived samples described previously was utilized to conduct a total of 12 test points, with six test points conducted by each operator. Each testing point was separated by a minimum of one full day in which no FC8G assay processing for this study occurred. The same lot of FC8G reagents, including the QIAamp DSP Virus Spin Kit, and the same imaging system was utilized by both operators for testing.

A total of 540 data points (45 samples/testing point x 6 testing points/operator x 2 operators) were generated in this study. Performance was evaluated by determining the agreement with the expected results, where the MP and LP concentrations were expected to be positive for the intended target, and the HN was expected to be negative.

Table 4 below shows the results of the within-laboratory precision study.

Table 4: Within-Laboratory Precision Study Results   

<table><tr><td rowspan=2 colspan=2></td><td rowspan=1 colspan=3>Operator 1</td><td rowspan=1 colspan=3>Operator 2</td><td rowspan=2 colspan=3>Overall % Agreement and95% Confidence Intervalby Sample (%)</td></tr><tr><td rowspan=1 colspan=2>Count and% Agreement</td><td rowspan=1 colspan=1>95%ConfidenceInterval(%)</td><td rowspan=1 colspan=2>Count and% Agreement</td><td rowspan=1 colspan=1>95%ConfidenceInterval(%)</td></tr><tr><td rowspan=3 colspan=1>A/H1N1 (pre-2009)(A/non-seasonal)</td><td rowspan=1 colspan=1>MP (3x LoD)</td><td rowspan=1 colspan=1>18/18</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>82.4 - 100</td><td rowspan=1 colspan=1>18/18</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>82.4 - 100</td><td rowspan=1 colspan=1>36/36</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>90.4 - 100</td></tr><tr><td rowspan=1 colspan=1>LP (1x LoD)</td><td rowspan=1 colspan=1>18/18</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>82.4 - 100</td><td rowspan=1 colspan=1>18/18</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>82.4 - 100</td><td rowspan=1 colspan=1>36/36</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>90.4 - 100</td></tr><tr><td rowspan=1 colspan=1>HN (0.1x LoD)</td><td rowspan=1 colspan=1>7/18</td><td rowspan=1 colspan=1>39%</td><td rowspan=1 colspan=1>20.3 - 61.4</td><td rowspan=1 colspan=1>4/18</td><td rowspan=1 colspan=1>22%</td><td rowspan=1 colspan=1>9.0 - 45.2</td><td rowspan=1 colspan=1>11/36</td><td rowspan=1 colspan=1>31%</td><td rowspan=1 colspan=1>18.0 - 46.9</td></tr><tr><td rowspan=3 colspan=1>A/H1N1pdm09</td><td rowspan=1 colspan=1>MP (3x LoD)</td><td rowspan=1 colspan=1>18/18</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>82.4 - 100</td><td rowspan=1 colspan=1>18/18</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>82.4 - 100</td><td rowspan=1 colspan=1>36/36</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>90.4 - 100</td></tr><tr><td rowspan=1 colspan=1>LP (1x LoD)</td><td rowspan=1 colspan=1>18/18</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>82.4 - 100</td><td rowspan=1 colspan=1>12/18a</td><td rowspan=1 colspan=1>67%</td><td rowspan=1 colspan=1>43.7 - 83.7</td><td rowspan=1 colspan=1>30/36</td><td rowspan=1 colspan=1>83%</td><td rowspan=1 colspan=1>68.1 - 92.1</td></tr><tr><td rowspan=1 colspan=1>HN (0.1x LoD)</td><td rowspan=1 colspan=1>3/18</td><td rowspan=1 colspan=1>17%</td><td rowspan=1 colspan=1>5.8 - 39.2</td><td rowspan=1 colspan=1>10/18</td><td rowspan=1 colspan=1>56%</td><td rowspan=1 colspan=1>33.7 - 75.4</td><td rowspan=1 colspan=1>13/36b</td><td rowspan=1 colspan=1>36%</td><td rowspan=1 colspan=1>22.5 - 52.4</td></tr><tr><td rowspan=3 colspan=1>A/H3N2</td><td rowspan=1 colspan=1>MP (3x LoD)</td><td rowspan=1 colspan=1>18/18</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>82.4 - 100</td><td rowspan=1 colspan=1>18/18</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>82.4 - 100</td><td rowspan=1 colspan=1>36/36</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>90.4 - 100</td></tr><tr><td rowspan=1 colspan=1>LP (1x LoD)</td><td rowspan=1 colspan=1>18/18</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>82.4 - 100</td><td rowspan=1 colspan=1>18/18</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>82.4 - 100</td><td rowspan=1 colspan=1>36/36</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>90.4 - 100</td></tr><tr><td rowspan=1 colspan=1>HN (0.1x LoD)</td><td rowspan=1 colspan=1>10/18</td><td rowspan=1 colspan=1>56%</td><td rowspan=1 colspan=1>33.7 - 75.4</td><td rowspan=1 colspan=1>9/18</td><td rowspan=1 colspan=1>50%</td><td rowspan=1 colspan=1>29.0 - 71.0</td><td rowspan=1 colspan=1>19/36c</td><td rowspan=1 colspan=1>53%</td><td rowspan=1 colspan=1>37.0 - 68.0</td></tr><tr><td rowspan=3 colspan=1>B/Yamagata</td><td rowspan=1 colspan=1>MP (3x LoD)</td><td rowspan=1 colspan=1>18/18</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>82.4 - 100</td><td rowspan=1 colspan=1>18/18</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>82.4 - 100</td><td rowspan=1 colspan=1>36/36</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>90.4 - 100</td></tr><tr><td rowspan=1 colspan=1>LP (1x LoD)</td><td rowspan=1 colspan=1>18/18</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>82.4 - 100</td><td rowspan=1 colspan=1>17/18</td><td rowspan=1 colspan=1>94%</td><td rowspan=1 colspan=1>74.2 - 99.0</td><td rowspan=1 colspan=1>35/36</td><td rowspan=1 colspan=1>97%</td><td rowspan=1 colspan=1>85.8 - 99.5</td></tr><tr><td rowspan=1 colspan=1>HN (0.1x LoD)</td><td rowspan=1 colspan=1>9/18</td><td rowspan=1 colspan=1>50%</td><td rowspan=1 colspan=1>29.0 - 71.0</td><td rowspan=1 colspan=1>17/18</td><td rowspan=1 colspan=1>94%</td><td rowspan=1 colspan=1>74.2 - 99.0</td><td rowspan=1 colspan=1>26/36</td><td rowspan=1 colspan=1>72%</td><td rowspan=1 colspan=1>56.0 - 84.2</td></tr><tr><td rowspan=3 colspan=1>B/Victoria</td><td rowspan=1 colspan=1>MP (3x LoD)</td><td rowspan=1 colspan=1>18/18</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>82.4 - 100</td><td rowspan=1 colspan=1>18/18</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>82.4 - 100</td><td rowspan=1 colspan=1>36/36</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>90.4 - 100</td></tr><tr><td rowspan=1 colspan=1>LP (1x LoD)</td><td rowspan=1 colspan=1>18/18</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>82.4 - 100</td><td rowspan=1 colspan=1>18/18</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>82.4 - 100</td><td rowspan=1 colspan=1>36/36</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>90.4 - 100</td></tr><tr><td rowspan=1 colspan=1>HN (0.1x LoD)</td><td rowspan=1 colspan=1>13/18</td><td rowspan=1 colspan=1>72%</td><td rowspan=1 colspan=1>49.1-87.5</td><td rowspan=1 colspan=1>18/18</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>82.4-100</td><td rowspan=1 colspan=1>31/36</td><td rowspan=1 colspan=1>86%</td><td rowspan=1 colspan=1>71.3 - 93.9</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=7>Moderate Positive (MP) (3x LoD)Low Positive (LP) (1x LoD)High Negative (HN) (0.1x LoD)</td><td rowspan=1 colspan=1>180/180173/180100/180</td><td rowspan=1 colspan=1>100%97%56%</td><td rowspan=1 colspan=1>97.9 - 10092.2 - 98.148.3 - 62.6</td></tr></table>

a 6/18 LP replicates were Flu A/H1N1pdm09 and Flu A non-seasonal dual positive on initial testing. All six samples generated the expected Flu A/H1N1pdm09 positive results upon repeat testing per the Instructions for Use (IFU). b ${ \bar { 5 / 3 } } 6 \mathrm { H N }$ replicates were Flu A non-seasonal on initial testing. Upon repeat testing per the IFU, 2 of 5 samples generated the expected Flu A/H1N1pdm09 positive results, 2 of 5 samples generated Inconclusive results, and 1 of 5 samples generated an Influenza Not Detected result. $2 / 3 6 \mathrm { H N }$ replicates were Flu A/H1N1pdm09 and Flu A non-seasonal dual positive on initial testing. All two samples generated Inconclusive results upon repeat testing per the IFU. 1/36 replicates was Flu A Equivocal and Flu A non-seasonal on initial testing. This sample generated an Inconclusive result upon repeat testing per the IFU $^ { \mathrm { c } } 2 / 3 6 \mathrm { H N }$ replicates were Flu A non-seasonal on initial testing. Upon repeat testing per the IFU, 1 of 2 samples generated an Inconclusive result, and 1 of 2 samples generated an Influenza Not Detected result.

b. Linearity/assay reportable range: Not applicable   
c. Traceability, Stability, Expected values (controls, calibrators, or methods): External Quality Control (QC)

The following external controls are recommended to be included in each testing batch, but are not provided in the test kit.

External Batch Positive Control: A previously characterized influenzapositive clinical specimen is recommended for inclusion as part of good laboratory practices. This control is user-supplied.

External Batch Negative Control: A previously characterized influenzanegative clinical specimen is recommended for inclusion as part of good laboratory practices. This control is user supplied.   
• Extraction Negative Control: This control is recommended for inclusion in each testing batch, and is composed of nuclease-free water added by the end user in place of specimen during the nucleic acid extraction step.   
• RT-PCR No Template Control (NTC): This control is recommended for inclusion in each testing batch, and is composed of nuclease-free water added by the end user in place of extracted nucleic acid template during RT-PCR reaction setup.

The following language, pertaining to the external controls, is included in the Instructions for Use (IFU):

“Good laboratory practice recommends that both an external positive control and external negative control are included alongside clinical specimens in each batch of extractions. These external positive and negative controls should be treated as samples under analysis and interpreted as such. It is also recommended that an extraction negative control (consisting of nuclease-free water) and an RT-PCR No Template Control be run alongside each batch of specimens. These controls should be carried through the entire assay alongside the clinical specimens under analysis.”

External batch positive controls were prepared for inclusion in each batch of samples analyzed during the prospective clinical study testing. The following four viruses were utilized to prepare the external batch positive controls: Influenza A/Netherlands/2290/2009 (A/H1N1pdm09), Influenza A/Victoria/361/2011 (A/H3N2), Influenza B/Florida/02/2006 (B/Victoria), and Influenza B/Florida/04/2006 (B/Yamagata). Three dilutions of each of the viruses were prepared in viral transport media (VTM) in large volumes to produce twelve overall external positive controls representing high, medium, and low influenza concentrations in each sample, as determined by an influenza molecular assay. Replicate aliquots of each control were prepared and provided to the clinical sites for use in testing. During the prospective clinical study, each site included one of the four external batch positive controls in each batch of specimens tested on a rotating basis.

Replicate aliquots of pooled influenza negative clinical materials that were determined to be negative for the presence of influenza virus via an FDA-cleared influenza molecular assay were also provided to the clinical study sites. During the prospective clinical study, each site included one aliquot of the provided external batch negative control in each batch of specimens tested.

In addition, an Extraction Negative Control and a No Template Control were also included in each batch of specimens tested at each site.

During the prospective clinical study between June 2017 and July 2018 at three sites, upon initial testing, there were one failed external batch negative control, one failed extraction negative control, and one failed external batch positive control. All controls with initial failed results generated expected results upon repeat testing. The failure rate upon initial testing (including the failed and the invalid results) of the external quality control materials was $1 . 7 \%$ (3/176), with $9 5 \%$ CI: $( 0 . 6 \% - 4 . 9 \% )$ .

Nasal and Nasopharyngeal Swab in UTM Specimens Stability

An analytical study was performed to evaluate the stability of clinical nasal swab (NS) and nasopharyngeal swab (NPS) in UTM specimens, when stored at refrigerator temperature $( 2 ^ { \circ } - 8 ^ { \circ } \mathrm { C } )$ . Contrived samples consisting of one strain representing each FC8G target virus group (A/H1N1pdm09, A/H3N2, A/non-seasonal, B/Victoria, and B/Yamagata) were prepared in pooled influenza negative clinical matrix at an influenza concentration of approximately 3x LoD. Triplicate extractions of the contrived sample were performed following storage at $1 0 ^ { \circ } \mathrm { C } \pm 2 ^ { \circ } \mathrm { C }$ for each of the following time points: 0 hour, 24 hours, 36 hours, 48 hours, and 96 hours. For all storage time points tested in this analytical study, an overall agreement of $100 \%$ $9 5 \% \mathrm { C I }$ , 79.6 – 100) with the expected result was obtained. This data supports a stability claim of 72 hours at refrigerator temperature $( 2 ^ { \circ } - 8 ^ { \circ } \mathrm { C } )$ for clinical nasal swab (NS) and nasopharyngeal swab (NPS) in UTM specimens.

While a formal analytical study was not conducted to determine specimen stability at $\le - 7 0 ^ { \circ } \mathrm { C }$ , data from the external controls performed during the clinical study were analyzed to assess stability over time when stored at $\le - 7 0 ^ { \circ } \mathrm { C }$ for 427 days. The samples evaluated were those used as external batch controls executed weekly during the course of the clinical study at the InDevR clinical site. Influenza positives in the testing panel were stock influenza viruses spiked into UTM. Influenza negatives in the testing panel were pooled influenza negative clinical material. Based on the results of this analysis, it appears that the nasal swab and nasopharyngeal swab in UTM specimens are stable for up to 14 months when stored at $\le - 7 0 ^ { \circ } \mathrm { C }$ .

Nasal and Nasopharyngeal Swab in UTM Specimens Freeze and Thaw Study

An analytical study was conducted to assess the performance of the FC8G Assay on contrived samples that underwent repeated freeze/thaw cycles to establish the equivalency of samples subjected to freeze/thaw and support the clinical study design of testing archived frozen specimens. A panel of 60 contrived influenza samples were utilized in this testing, consisting of 12 unique strains for each of the five target groups (A/H1N1pdm2009, A/H3N2, A/non-seasonal, B/Yamagata, and B/Victoria). Samples were contrived by spiking stock whole virus into pooled influenza negative clinical material at a final concentration of approximately 3x LoD (ranging from 1x to 9x LoD, depending on strain), and five replicate aliquots of each prepared sample were made. The first aliquot of each strain was tested without re-freezing (immediately after preparation), with one aliquot of each strain tested after each of four consecutive freeze-thaw cycles to assess $\%$ agreement with expected result. Data from this study demonstrated that the FC8G Assay performance was not affected by up to four freeze and thaw cycles of the specimens.

An analytical evaluation was performed to evaluate the stability of extracted RNA specimens, when stored at $\le - 7 0 ^ { \circ } \mathrm { C }$ . A large volume of a representative A/H3N2 stock virus, A/Victoria/361/2011, was utilized to perform this testing by spiking stock virus into pooled influenza-negative clinical material at approximately 3x LoD and creating individual homogeneous aliquots for downstream testing. Sixteen aliquots of the prepared A/H3N2 contrived sample were extracted according to the Instructions for Use (IFU), subsequently pooled to create a homogeneous material, and aliquoted into five aliquots. Each aliquot was of sufficient volume to enable 14 replicate tests. One aliquot was utilized to assess the assay performance at “Day $0 ^ { \circ }$ , and the remaining four aliquots were stored at $\le - 7 0 ^ { \circ } \mathrm { C }$ until tested. Three of the four remaining aliquots were utilized to conduct testing at Days 14, 30, and 40 after storage. The last aliquot was utilized to test the cumulative “worst case” stability of the assay intermediates, including Extracted RNA Specimen, RT-PCR Product, Fragmented RT-PCR product, and Labeled Microarray.

Extracted RNA specimens stored at $\le - 7 0 ^ { \circ } \mathrm { C }$ showed $100 \%$ agreement with “Day $0 ^ { \circ }$ results for the duration of the testing schedule of 40 days, supporting a stability claim of 30 days at $\le - 7 0 ^ { \circ } \mathrm { C }$ .

Intact and Heat-Fragmentated RT-PCR Products Stability

An analytical evaluation was performed to evaluate the stability of intact and heatfragmented RT-PCR products when stored at either $+ 1 0 ^ { \circ } \mathrm { C }$ or $- 1 0 ^ { \circ } \mathrm { C }$ .

A large volume of a representative A/H3N2 stock virus, A/Victoria/361/2011, was utilized to perform this testing by spiking stock virus into pooled influenza-negative clinical material at approximately 3x LoD and creating individual homogeneous aliquots for downstream testing. Thirty-five (35) aliquots of the prepared A/H3N2 sample were extracted according to the IFU, and subsequently pooled to create a homogeneous material. This extracted material was then used as template in 254 replicate RT-PCR amplification reactions conducted according to the IFU. Once amplified, all of the replicates of the amplified A/H3N2 sample were pooled to create a homogeneous RT-PCR product which was subsequently aliquoted into approximately 220 individual PCR tubes for downstream processing by the remainder of the FC8G assay.

For the intact (un-fragmented) RT-PCR product testing, 14 replicate aliquots of the amplified A/H3N2 sample were processed through the remainder of the FC8G assay according to the IFU at “Day $0 ^ { \circ }$ . Eighty-four (84) replicate aliquots of the amplified A/H3N2 sample were divided evenly at two different storage temperatures: 42 aliquots at $+ 1 0 ^ { \circ } \mathrm { C }$ and 42 aliquots at $- 1 0 ^ { \circ } \mathrm { C }$ . For the $+ 1 0 ^ { \circ } \mathrm { C }$ stored products, 14 aliquots were processed through the remainder of the FC8G assay according to the IFU at Day 3, Day 7, and Day 14 after storage. For the $- 1 0 ^ { \circ } \mathrm { C }$ stored products, 14

aliquots were processed through the remainder of the assay at Days 7, 14, and 30 after storage.

Intact (un-fragmented) RT-PCR products stored at $+ 1 0 ^ { \circ } \mathrm { C }$ showed $100 \%$ agreement with “Day $0 ^ { \circ }$ results for the duration of the testing schedule of 14 days, supporting a stability claim of 7 days at $\le 8 ^ { \circ } \mathrm { C }$ . Intact (un-fragmented) RT-PCR products stored at - $1 0 ^ { \circ } \mathrm { C }$ showed $100 \%$ agreement with “Day $0 ^ { \circ }$ results for the duration of the testing schedule of 30 days, supporting a stability claim of 14 days at $\le - 1 5 ^ { \circ } \mathrm { C }$ .

For the heat-fragmented RT-PCR product testing, 14 replicate aliquots of the amplified A/H3N2 sample were heat-fragmented and processed through the remainder of the FC8G assay according to the IFU at “Day $0 ^ { \circ }$ . The remaining replicate RT-PCR product aliquots were heat-fragmented according to the IFU, and the fragmented products were then immediately pooled to create a homogeneous material. The homogeneous material was then re-aliquoted. These aliquots were evenly divided between two different storage temperatures, $+ 1 0 ^ { \circ } \mathrm { C }$ and $- 1 0 ^ { \circ } \mathrm { C }$ . For the $+ 1 0 ^ { \circ } \mathrm { C }$ stored products, aliquots were tested on Days 1, Day 3, Day 7, and Day 14 after storage. For the aliquots stored at $- 1 0 ^ { \circ } \mathrm { C }$ , aliquots were tested on Days 3, 7, 14, and 30 after storage.

Heat-fragmented RT-PCR products stored at $+ 1 0 ^ { \circ } \mathrm { C }$ showed $100 \%$ agreement with “Day $0 ^ { \circ }$ results for the duration of the testing schedule of 14 days, supporting a stability claim of 7 days at $\le 8 ^ { \circ } \mathrm { C }$ . Heat-fragmented RT-PCR products stored at $- 1 0 ^ { \circ } \mathrm { C }$ showed $100 \%$ agreement with “Day $0 ^ { \circ }$ results for the duration of the testing schedule of 30 days, supporting a stability claim of 14 days at $\le - 1 5 ^ { \circ } \mathrm { C }$ .

# Labeled Microarray Stability

An analytical evaluation was performed to evaluate the stability of labeled microarrays when stored at room temperature $( 1 8 ^ { \circ } \mathrm { C } - 2 5 ^ { \circ } \mathrm { C } )$ .

Two microarray slides processed for other aspects of the assay intermediates stability evaluations (one from the extracted RNA specimen stability evaluation, and one from the RT-PCR product stability evaluation), were imaged on the FC8G Imaging System immediately after assay completion (considered Day 0). A total of 14 replicate A/H3N2 samples were included on each of the two microarray slides. The slides were subsequently stored in a dry microarray slide box in the dark, at room temperature and rescanned using the FC8G Imaging System after Days 3, 7, and 14.

Labeled microarrays stored at room temperature showed $100 \%$ agreement with “Day $0 ^ { \circ }$ results for the duration of the testing schedule of 14 days, supporting a stability claim of 7 days at room temperature $( 1 8 ^ { \circ } \mathrm { C } - 2 5 ^ { \circ } \mathrm { C } )$ .

To investigate the cumulative “worst” case scenario in which each assay intermediate was stored for the longest recommended storage time at its highest recommended temperature in the IFU, an aliquot of pooled extracted RNA prepared for the extracted RNA specimens stability testing was separated into 14 identical aliquots, and each aliquot was RT-PCR amplified according to the IFU. The intact double-stranded amplicon RT-PCR products were then stored at $+ 1 0 ^ { \circ } \mathrm { C }$ for seven days prior to fragmentation. The products were then heat-fragmented according to the IFU, and the fragmented products were stored at $+ 1 0 ^ { \circ } \mathrm { C }$ for seven days prior to hybridization and labeling according to the IFU. After hybridization and labeling, the slide was initially imaged. The imaged slide was then stored for seven days upon which it was reimaged. All 14 aliquots processed produced the expected A/H3N2 result. The testing data supports the maximum recommended storage times and temperatures presented in the IFU.

# d. Detection limit:

The limit of detection (LoD) was determined for each FC8G assay target virus group (influenza A/H3N2, influenza A/H1N1pdm2009, influenza B/Yamagata, influenza B/Victoria, and non-seasonal influenza A) by testing serial dilutions of stock virus strains (used either as intact virus or viral genomic RNA, depending on influenza subtype) spiked into pooled influenza negative clinical material. The LoD was determined for each strain by: 1) performing a 10-fold serial dilution range-finding experiment to determine an appropriate range for testing, 2) analyzing four replicates of a 3-fold limiting dilution series spanning a concentration range both above and below the assay cutoff to approximate the LoD, and 3) confirming the LoD by determining the concentration at which $9 5 \%$ positivity is achieved from testing at least 20 replicates.

# LoD for Seasonal Influenza Target Virus Groups

The LoD was established for eight strains of seasonal influenza, including two strains each of A/H1N1pdm09, A/H3N2, B/Yamagata, and B/Victoria. All viral stock materials were intact whole virus of known titer spiked into pooled influenzanegative clinical material. The confirmed LoD in $\mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ for each strain investigated are shown in Table 5 below, along with associated concentrations in approximate genome copies/mL determined at the LoD.

Table 5: LoD for Seasonal Influenza Target Virus Groups   

<table><tr><td rowspan=1 colspan=1>Target Virus Group</td><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>LoDConcentration(TCID50/mL)</td><td rowspan=1 colspan=1>ApproximateGenomeCopies/mL at LoD</td><td rowspan=1 colspan=1>PositivityRate (%)</td></tr><tr><td rowspan=2 colspan=1>A/H1N1pdm09</td><td rowspan=1 colspan=1>A/New York/01/2009</td><td rowspan=1 colspan=1>6.2 x 100</td><td rowspan=1 colspan=1>1.4 x 103</td><td rowspan=1 colspan=1>25/25 (100%)</td></tr><tr><td rowspan=1 colspan=1>A/California/07/2009</td><td rowspan=1 colspan=1>1.9 x 10-1</td><td rowspan=1 colspan=1>1.8 x 103</td><td rowspan=1 colspan=1>20/20 (100%)</td></tr><tr><td rowspan=2 colspan=1>A/H3N2</td><td rowspan=1 colspan=1>A/Perth/16/2009</td><td rowspan=1 colspan=1>1.9 x 100</td><td rowspan=1 colspan=1>1.4 x 103</td><td rowspan=1 colspan=1>24/25 (96%)</td></tr><tr><td rowspan=1 colspan=1>A/Victoria/361/2011</td><td rowspan=1 colspan=1>2.0 x 101</td><td rowspan=1 colspan=1>1.2 x 103</td><td rowspan=1 colspan=1>25/25 (100%)</td></tr><tr><td rowspan=2 colspan=1>B/Victoria</td><td rowspan=1 colspan=1>B/Brisbane/60/2008</td><td rowspan=1 colspan=1>1.7 x 10-1</td><td rowspan=1 colspan=1>5.8 x 102</td><td rowspan=1 colspan=1>24/25 (96%)</td></tr><tr><td rowspan=1 colspan=1>B/Florida/02/2006</td><td rowspan=1 colspan=1>8.3 x 10-1</td><td rowspan=1 colspan=1>1.3 x 103</td><td rowspan=1 colspan=1>20/20 (100%)</td></tr><tr><td rowspan=2 colspan=1>B/Yamagata</td><td rowspan=1 colspan=1>B/Wisconsin/01/2010</td><td rowspan=1 colspan=1>5.6 x 10-2</td><td rowspan=1 colspan=1>6.3 x 102</td><td rowspan=1 colspan=1>25/25 (100%)</td></tr><tr><td rowspan=1 colspan=1>B/Phuket/3073/2013</td><td rowspan=1 colspan=1>1.9 x 10-1</td><td rowspan=1 colspan=1>1.7 x 103</td><td rowspan=1 colspan=1>20/20 (100%)</td></tr></table>

The LoD was established for 10 strains of non-seasonal influenza A, representing 10 different subtypes. Due to safety and availability reasons, in this study the A/H5 and A/H7 strains were tested as extracted viral genomic RNA, and the other strains were tested as whole virus. Both types of starting material were spiked into pooled influenza-negative clinical material and processed through the entire assay, including extraction. Specific procedures were followed (i.e., stabilization of RNA with protease/lysis buffer) to ensure that RNA spiked into clinical negative material did not degrade prior to testing. The confirmed LoD in either $\mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ or approximate genome copies/mL, depending on the starting viral material (i.e., whole virus vs. extracted genomic viral RNA), along with associated concentrations in approximate genome copies $/ \mathrm { m L }$ determined at the LoD for the whole viruses, are shown in Table 6 below.

Table 6: LoD for Non-Seasonal Influenza A Target Virus Group   

<table><tr><td rowspan=1 colspan=1>Subtype</td><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>LoDConcentration(TCID50/mL)</td><td rowspan=1 colspan=1>ApproximateGenomeCopies/mL at LoD</td><td rowspan=1 colspan=1>PositivityRate (%)</td></tr><tr><td rowspan=1 colspan=1>A/H1N1pre2009</td><td rowspan=1 colspan=1>A/New Caledonia/20/1999</td><td rowspan=1 colspan=1>1.2 x 102</td><td rowspan=1 colspan=1>5.6 x 103</td><td rowspan=1 colspan=1>20/20 (100%)</td></tr><tr><td rowspan=1 colspan=1>A/H1N1v</td><td rowspan=1 colspan=1>A/Swine/Illinois/4L013/2015</td><td rowspan=1 colspan=1>7.5 x 103</td><td rowspan=1 colspan=1>1.8 x 105</td><td rowspan=1 colspan=1>24/25 (96%)</td></tr><tr><td rowspan=1 colspan=1>A/H3N2v</td><td rowspan=1 colspan=1>A/Indiana/08/2011</td><td rowspan=1 colspan=1>9.7 x 101</td><td rowspan=1 colspan=1>2.9 x 103</td><td rowspan=1 colspan=1>19/20 (95%)</td></tr><tr><td rowspan=1 colspan=1>A/H3N8</td><td rowspan=1 colspan=1>A/Blue-Winged Teal/Iowa/10OS2411/2010</td><td rowspan=1 colspan=1>9.3 x 101</td><td rowspan=1 colspan=1>8.7 x 102</td><td rowspan=1 colspan=1>20/20 (100%)</td></tr><tr><td rowspan=1 colspan=1>A/H5N1</td><td rowspan=1 colspan=1>A/Anhui/01/2005*</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>3.2 x 103</td><td rowspan=1 colspan=1>20/20 (100%)</td></tr><tr><td rowspan=1 colspan=1>A/H5N2</td><td rowspan=1 colspan=1>A/Mule Duck/Bulgaria/237/2011*</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>1.5 x 103</td><td rowspan=1 colspan=1>20/20 (100%)</td></tr><tr><td rowspan=1 colspan=1>A/H5N8</td><td rowspan=1 colspan=1>A/Mule Duck/Bulgaria /328/2011*</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>4.9 x 103</td><td rowspan=1 colspan=1>20/20 (100%)</td></tr><tr><td rowspan=1 colspan=1>A/H7N7</td><td rowspan=1 colspan=1>A/Ruddy Turnstone/New Jersey/AI11- 1678/2011*</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>1.2 x 103</td><td rowspan=1 colspan=1>20/20 (100%)</td></tr><tr><td rowspan=1 colspan=1>A/H7N9</td><td rowspan=1 colspan=1>A/Mallard/Alberta/177/2004*</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>5.8 x 102</td><td rowspan=1 colspan=1>20/20 (100%)</td></tr><tr><td rowspan=1 colspan=1>A/H9N2</td><td rowspan=1 colspan=1>A/Shorebird/Delaware Bay/127/2003</td><td rowspan=1 colspan=1>1.8 x 10-1</td><td rowspan=1 colspan=1>7.7 x 102</td><td rowspan=1 colspan=1>20/20 (100%)</td></tr></table>

\* Extracted viral genomic RNA was used as starting material.

The A/H1N1v virus strain tested (A/Swine/IL/4L013/2015) was consistently misidentified as an A/H1N1pdm09 virus during the study. For samples at and above the LoD, this virus was correctly identified as non-seasonal influenza A, but was also identified incorrectly as an A/H1N1pdm09 (a dual positive result). At concentrations below the LoD, the A/H1N1v samples were misidentified as A/H1N1pdm09 only. The H3N2v virus strain tested (A/Indiana/08/2011) resulted in correct identification as non-seasonal influenza A at concentrations at and above the LoD, but was misidentified at concentrations below the LoD as an A/H1N1pdm09. The misidentification of A/H1N1v and A/H3N2v as seasonal A/H1N1pdm09 or as nonseasonal influenza A and A/H1N1pdm09 dual infections is likely due to close genetic similarities among these viruses. The assay Instructions for Use (IFU) includes limitation language indicating these possible results for variant viruses, and follow-up retesting requirement for all dual positive results.

# e. Analytical Reactivity

Analytical reactivity (inclusivity) was evaluated for 52 additional temporally and geographically diverse influenza strains representing all FC8G assay target virus groups. The FC8G assay was executed on either titered stock whole virus or extracted RNA (depending on subtype) spiked into pooled clinical negative material at close to the LoD concentration. Each strain was tested in triplicate. Any virus that was not correctly identified in triplicate was tested again at a higher concentration until detected.

# Analytical Reactivity of Seasonal Influenza Target Virus Groups

Analytical reactivity (inclusivity) was determined for 32 seasonal influenza strains near the limit of detection as shown in Table 7 below. In this study, whole viruses were utilized as starting material for the testing. The testing concentrations are shown in both $\mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ and approximate genome copies/mL. The multiple of the LoD tested was determined based on the approximate genome copies $ { \mathrm { / m L } }$ values.

Table 7: Analytical Reactivity of Seasonal Influenza Target Virus Groups   

<table><tr><td colspan="1" rowspan="1">Target Virus Group</td><td colspan="1" rowspan="1">Strain</td><td colspan="1" rowspan="1">Concentration(TCID50/mL,unless specified)</td><td colspan="1" rowspan="1">Concentration(approximate genomecopies/mL)</td><td colspan="1" rowspan="1">ApproximateMultiple of LoDDetected</td></tr><tr><td colspan="1" rowspan="11">A/H1N1pdm09</td><td colspan="1" rowspan="1">A/South Carolina/18/2009</td><td colspan="1" rowspan="1">1.5 x 100</td><td colspan="1" rowspan="1">2.5 x 103</td><td colspan="1" rowspan="1">1x</td></tr><tr><td colspan="1" rowspan="1">A/Canada/6294/2009</td><td colspan="1" rowspan="1">1.9 x 100</td><td colspan="1" rowspan="1">1.3 x 103</td><td colspan="1" rowspan="1">1x</td></tr><tr><td colspan="1" rowspan="1">A/Netherlands/2290/2009</td><td colspan="1" rowspan="1">2.9 x 100</td><td colspan="1" rowspan="1">5.0 x 103</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="1">A/New York/18/2009</td><td colspan="1" rowspan="1">1.1 x 100</td><td colspan="1" rowspan="1">9.9 x 102</td><td colspan="1" rowspan="1">1x</td></tr><tr><td colspan="1" rowspan="1">A/Utah/20/2009</td><td colspan="1" rowspan="1">1.2 x 100</td><td colspan="1" rowspan="1">1.0 x 104</td><td colspan="1" rowspan="1">6x</td></tr><tr><td colspan="1" rowspan="1">A/Massachusetts/15/2013</td><td colspan="1" rowspan="1">4.0 x 100</td><td colspan="1" rowspan="1">7.9 x 103</td><td colspan="1" rowspan="1">4x</td></tr><tr><td colspan="1" rowspan="1">A/Mexico/4108/2009</td><td colspan="1" rowspan="1">2.5 x 10-1</td><td colspan="1" rowspan="1">1.9 x 103</td><td colspan="1" rowspan="1">1x</td></tr><tr><td colspan="1" rowspan="1">A/New Hampshire/02/2010</td><td colspan="1" rowspan="1">7.7 x 100</td><td colspan="1" rowspan="1">3.2 x 103</td><td colspan="1" rowspan="1">2x</td></tr><tr><td colspan="1" rowspan="1">A/California/04/2009</td><td colspan="1" rowspan="1">1.1 x 101</td><td colspan="1" rowspan="1">4.7 x 103</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="1">A/Dominican Republic/7293/2013</td><td colspan="1" rowspan="1">2.2 x 102</td><td colspan="1" rowspan="1">3.9 x 103</td><td colspan="1" rowspan="1">2x</td></tr><tr><td colspan="1" rowspan="1">A/Michigan/45/2015</td><td colspan="1" rowspan="1">1.1 x 103(EID50/mL)</td><td colspan="1" rowspan="1">6.5 x 103</td><td colspan="1" rowspan="1">4x</td></tr><tr><td colspan="1" rowspan="10">A/H3N2</td><td colspan="1" rowspan="1">A/Stockholm/6/2014</td><td colspan="1" rowspan="1">2.6 x 100</td><td colspan="1" rowspan="1">2.4 x 103</td><td colspan="1" rowspan="1">2x</td></tr><tr><td colspan="1" rowspan="1">A/South Australia/55/2014</td><td colspan="1" rowspan="1">1.1 x 100</td><td colspan="1" rowspan="1">2.2 x 102</td><td colspan="1" rowspan="1">0.2x</td></tr><tr><td colspan="1" rowspan="1">A/Switzerland/9715293/2013</td><td colspan="1" rowspan="1">1.4 x 100</td><td colspan="1" rowspan="1">5.7 x 102</td><td colspan="1" rowspan="1">0.4x</td></tr><tr><td colspan="1" rowspan="1">A/Norway/466/2014</td><td colspan="1" rowspan="1">1.5 x 100</td><td colspan="1" rowspan="1">5.3 x 102</td><td colspan="1" rowspan="1">0.4x</td></tr><tr><td colspan="1" rowspan="1">A/Texas/71/2007</td><td colspan="1" rowspan="1">1.9 x 100</td><td colspan="1" rowspan="1">4.2 x 103</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="1">A/Uruguay/716/2007</td><td colspan="1" rowspan="1">1.2 x 100</td><td colspan="1" rowspan="1">5.2 x 103</td><td colspan="1" rowspan="1">4x</td></tr><tr><td colspan="1" rowspan="1">A/Texas/50/2012</td><td colspan="1" rowspan="1">2.3 x 100</td><td colspan="1" rowspan="1">3.9 x 103</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="1">A/Santiago/7981/2006</td><td colspan="1" rowspan="1">2.1 x 100</td><td colspan="1" rowspan="1">2.9 x 103</td><td colspan="1" rowspan="1">2x</td></tr><tr><td colspan="1" rowspan="1">A/Wisconsin/67/2005</td><td colspan="1" rowspan="1">7.9 x 100</td><td colspan="1" rowspan="1">5.3 x 102</td><td colspan="1" rowspan="1">0.4x</td></tr><tr><td colspan="1" rowspan="1">A/Hong Kong/4801/2014</td><td colspan="1" rowspan="1">4.6 x 102(EID50/mL)</td><td colspan="1" rowspan="1">2.1 x 103</td><td colspan="1" rowspan="1">2x</td></tr><tr><td colspan="1" rowspan="6">B/Victoria</td><td colspan="1" rowspan="1">B/Nevada/03/2011</td><td colspan="1" rowspan="1">3.1 x 10-1</td><td colspan="1" rowspan="1">1.3 x 103</td><td colspan="1" rowspan="1">1x</td></tr><tr><td colspan="1" rowspan="1">B/Malaysia/2506/2004</td><td colspan="1" rowspan="1">5.4 x 10-1</td><td colspan="1" rowspan="1">1.7 x 103</td><td colspan="1" rowspan="1">1x</td></tr><tr><td colspan="1" rowspan="1">B/New Jersey/01/2012</td><td colspan="1" rowspan="1">4.6 x 10-1</td><td colspan="1" rowspan="1">2.6 x 103</td><td colspan="1" rowspan="1">2x</td></tr><tr><td colspan="1" rowspan="1">B/Victoria/304/2006</td><td colspan="1" rowspan="1">1.6 x 101</td><td colspan="1" rowspan="1">4.7 x 103</td><td colspan="1" rowspan="1">4x</td></tr><tr><td colspan="1" rowspan="1">B/Hong Kong/330/2001</td><td colspan="1" rowspan="1">1.4 x 101</td><td colspan="1" rowspan="1">3.4 x 103</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="1">B/Texas/02/2013</td><td colspan="1" rowspan="1">3.8 x 103(EID50/mL)</td><td colspan="1" rowspan="1">1.6 x 103</td><td colspan="1" rowspan="1">1x</td></tr><tr><td colspan="1" rowspan="5">B/Yamagata</td><td colspan="1" rowspan="1">B/Texas/06/2011</td><td colspan="1" rowspan="1">6.9 x 10-1</td><td colspan="1" rowspan="1">3.6 x 103</td><td colspan="1" rowspan="1">2x</td></tr><tr><td colspan="1" rowspan="1">B/Pennsylvania/07/2007</td><td colspan="1" rowspan="1">1.6 x 100</td><td colspan="1" rowspan="1">3.8 x 103</td><td colspan="1" rowspan="1">2x</td></tr><tr><td colspan="1" rowspan="1">B/Massachusetts/02/2012</td><td colspan="1" rowspan="1">1.7 x 100</td><td colspan="1" rowspan="1">5.3 x 103</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="1">B/Brisbane/3/2007</td><td colspan="1" rowspan="1">5.9 x 10-1</td><td colspan="1" rowspan="1">2.8 x 103</td><td colspan="1" rowspan="1">2x</td></tr><tr><td colspan="1" rowspan="1">B/Florida/07/2004</td><td colspan="1" rowspan="1">2.3 x 101</td><td colspan="1" rowspan="1">1.2 x 103</td><td colspan="1" rowspan="1">1x</td></tr></table>

# Analytical Reactivity of Non-Seasonal Influenza A Target Virus Group

Analytical reactivity (inclusivity) was determined for 20 virus strains belonging to the non-seasonal influenza A target virus group (i.e., all influenza A subtypes other than A/H1N1pdm09 or A/H3N2). For A/H5 and A/H7 strains, due to safety and availability reasons, starting material used was extracted genomic viral RNA in this study. All other strains utilized whole virus as starting material in this study. Both types of starting material were spiked into pooled influenza-negative clinical material at close to the LoD concentrations and processed through the entire assay, including extraction. Results are shown in Table 8 below. The multiple of the LoD tested was determined based on the approximate genome copies $ { / \mathrm { m L } }$ values.

Table 8: Analytical Reactivity of Non-Seasonal Influenza A Target Virus Group   

<table><tr><td colspan="1" rowspan="1">Subtype</td><td colspan="1" rowspan="1">Strain</td><td colspan="1" rowspan="1">Concentration(EID50/mL, unlessspecified)</td><td colspan="1" rowspan="1">Concentration(approximategenome copies/mL)</td><td colspan="1" rowspan="1">ApproximateMultiple ofLoD Detected</td></tr><tr><td colspan="1" rowspan="1">A/H1N1pre2009</td><td colspan="1" rowspan="1">A/Fujian Gulou/1896/2009</td><td colspan="1" rowspan="1">7.5 x 104</td><td colspan="1" rowspan="1">3.4 x 103</td><td colspan="1" rowspan="1">1x</td></tr><tr><td colspan="1" rowspan="1">A/H1N1sW</td><td colspan="1" rowspan="1">A/Swine/1976/1931</td><td colspan="1" rowspan="1">6.2 x 103</td><td colspan="1" rowspan="1">6.9 x 103</td><td colspan="1" rowspan="1">2x</td></tr><tr><td colspan="1" rowspan="1">A/H1N1v</td><td colspan="1" rowspan="1">A/Swine/Illinois/4L036/2015</td><td colspan="1" rowspan="1">No Titer Available</td><td colspan="1" rowspan="1">6.7 x 104</td><td colspan="1" rowspan="1">23x*</td></tr><tr><td colspan="1" rowspan="1">A/H1N8</td><td colspan="1" rowspan="1">A/Red Knot/Delaware Bay/240/1994</td><td colspan="1" rowspan="1">2.9 x 105 TCID50/mL</td><td colspan="1" rowspan="1">4.5 x 103</td><td colspan="1" rowspan="1">2x</td></tr><tr><td colspan="1" rowspan="1">A/H2N2</td><td colspan="1" rowspan="1">A/Chicken/New York/SG-00425/2004</td><td colspan="1" rowspan="1">No Titer Available</td><td colspan="1" rowspan="1">5.6 x 103</td><td colspan="1" rowspan="1">2x</td></tr><tr><td colspan="1" rowspan="1">A/H3N2sW</td><td colspan="1" rowspan="1">A/Swine/Ohio/11SW87/2011</td><td colspan="1" rowspan="1">9.1 x 103 TCID50/mL</td><td colspan="1" rowspan="1">7.9 x 103</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="1">A/H3N2v</td><td colspan="1" rowspan="1">A/Minnesota/11/2010</td><td colspan="1" rowspan="1">3.0 x 104</td><td colspan="1" rowspan="1">1.0 x 104</td><td colspan="1" rowspan="1">4x</td></tr><tr><td colspan="1" rowspan="1">A/H3N8</td><td colspan="1" rowspan="1">A/Blue-Winged Teal/Alberta/221/1978</td><td colspan="1" rowspan="1">No Titer Available</td><td colspan="1" rowspan="1">1.6 x 103</td><td colspan="1" rowspan="1">1x</td></tr><tr><td colspan="1" rowspan="1">A/H3N8</td><td colspan="1" rowspan="1">A/Camel/Mongolia/335/2012</td><td colspan="1" rowspan="1">2.5 x 103</td><td colspan="1" rowspan="1">5.6 x 103</td><td colspan="1" rowspan="1">2x</td></tr><tr><td colspan="1" rowspan="1">A/H3N9</td><td colspan="1" rowspan="1">A/Mallard/Alberta/54/1993</td><td colspan="1" rowspan="1">2.9 x 104</td><td colspan="1" rowspan="1">2.1 x 103</td><td colspan="1" rowspan="1">1x</td></tr><tr><td colspan="1" rowspan="1">A/H4N6</td><td colspan="1" rowspan="1">A/Blue-Winged Teal/Illinois/100S1563/2010</td><td colspan="1" rowspan="1">4.6 x 104</td><td colspan="1" rowspan="1">5.6 x 103</td><td colspan="1" rowspan="1">2x</td></tr><tr><td colspan="1" rowspan="1">A/H5N1</td><td colspan="1" rowspan="1">A/Goose/Yunnan/5539/2005**</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">4.5 x 103</td><td colspan="1" rowspan="1">2x</td></tr><tr><td colspan="1" rowspan="1">A/H5N1</td><td colspan="1" rowspan="1">A/Chicken/Vietnam/NCVD-016/2008**</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">7.9 x 103</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="1">A/H5N2</td><td colspan="1" rowspan="1">A/Northern Pintail/California/44242-758/2006**</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">2.6 x 104</td><td colspan="1" rowspan="1">9x*</td></tr><tr><td colspan="1" rowspan="1">A/H5N8</td><td colspan="1" rowspan="1">A/Duck/England/36254/2014**</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">7.3 x 103</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="1">A/H6N4</td><td colspan="1" rowspan="1">A/Mallard/Alberta/58/1989</td><td colspan="1" rowspan="1">4.9 x 103</td><td colspan="1" rowspan="1">2.1 x 103</td><td colspan="1" rowspan="1">1x</td></tr><tr><td colspan="1" rowspan="1">A/H7N7</td><td colspan="1" rowspan="1">A/Italy/3/2013**</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">1.7 x 104</td><td colspan="1" rowspan="1">6x*</td></tr><tr><td colspan="1" rowspan="1">A/H7N9</td><td colspan="1" rowspan="1">A/Hong Kong/5942/2013**</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">6.9 x 103</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="1">A/H9N2</td><td colspan="1" rowspan="1">A/Turkey/Wisconsin/1/1966</td><td colspan="1" rowspan="1">3.8 x 103</td><td colspan="1" rowspan="1">3.0 x 103</td><td colspan="1" rowspan="1">1x</td></tr><tr><td colspan="1" rowspan="1">A/H11N2</td><td colspan="1" rowspan="1">A/Laughing Gull/Delaware Bay/94/1995</td><td colspan="1" rowspan="1">7.0 x 105</td><td colspan="1" rowspan="1">1.4 x 104</td><td colspan="1" rowspan="1">5x*</td></tr></table>

\* Not correctly detected at lower concentrations. \*\* Extracted viral genomic RNA was used as starting material.

# f. Analytical specificity

# Analytical Specificity for Influenza Viruses

Analytical specificity for all target virus groups was evaluated with the FC8G assay to determine whether strains of a known target influenza virus group were misidentified as another influenza virus group targeted by the assay when run at high concentration. High concentrations of stock influenza virus or extracted genomic viral RNA (depending on subtype) were spiked into pooled influenza-negative clinical material in triplicate and analyzed by the FC8G assay to determine if any mis-identification occurred.

Analytical Specificity for Seasonal Influenza Target Virus Groups

Analytical specificity (exclusivity) was demonstrated for 32 strains of seasonal influenza at high concentrations. Whole virus was utilized as starting material in this study and spiked into influenza-negative clinical material at concentrations ${ > } 1 0 ^ { 6 }$ approximate genome copies $/ \mathrm { m L }$ . Results of this study are shown in Table 9 below.

Table 9: Analytical Specificity of Seasonal Influenza Target Virus Groups   

<table><tr><td colspan="1" rowspan="1">Target Virus Group</td><td colspan="1" rowspan="1">Virus Strain</td><td colspan="1" rowspan="1">Concentration(TCID50/mL,unless specified)</td><td colspan="1" rowspan="1">Concentration(approximategenome copies/mL)</td><td colspan="1" rowspan="1">Mis-Identification</td></tr><tr><td colspan="1" rowspan="11">A/H1N1pdm09</td><td colspan="1" rowspan="1">A/Canada/6294/2009</td><td colspan="1" rowspan="1">9.3 x 103</td><td colspan="1" rowspan="1">3.0 x 107</td><td colspan="1" rowspan="1">Not observed</td></tr><tr><td colspan="1" rowspan="1">A/Michigan/45/2015</td><td colspan="1" rowspan="1">2.6 x 107(EID50/mL)</td><td colspan="1" rowspan="1">2.4 x 107</td><td colspan="1" rowspan="1">Not observed</td></tr><tr><td colspan="1" rowspan="1">A/DominicanRepublic/7293/2013</td><td colspan="1" rowspan="1">5.0 x 106</td><td colspan="1" rowspan="1">1.3 x 108</td><td colspan="1" rowspan="1">Not observed</td></tr><tr><td colspan="1" rowspan="1">A/New York/18/2009</td><td colspan="1" rowspan="1">4.1 x 103</td><td colspan="1" rowspan="1">1.4 x 107</td><td colspan="1" rowspan="1">Not observed</td></tr><tr><td colspan="1" rowspan="1">A/New Hampshire/02/2010</td><td colspan="1" rowspan="1">8.5 x 104</td><td colspan="1" rowspan="1">4.0 x 107</td><td colspan="1" rowspan="1">Not observed</td></tr><tr><td colspan="1" rowspan="1">A/Massachusetts/15/2013</td><td colspan="1" rowspan="1">1.4 x 104</td><td colspan="1" rowspan="1">1.5 x 107</td><td colspan="1" rowspan="1">Not observed</td></tr><tr><td colspan="1" rowspan="1">A/Mexico/4108/2009</td><td colspan="1" rowspan="1">8.1 x 103</td><td colspan="1" rowspan="1">8.0 x 106</td><td colspan="1" rowspan="1">Not observed</td></tr><tr><td colspan="1" rowspan="1">A/Utah/20/2009</td><td colspan="1" rowspan="1">4.1 x 103</td><td colspan="1" rowspan="1">1.8 x 107</td><td colspan="1" rowspan="1">Not observed</td></tr><tr><td colspan="1" rowspan="1">A/South Carolina/18/2009</td><td colspan="1" rowspan="1">2.8 x 103</td><td colspan="1" rowspan="1">1.1 x 107</td><td colspan="1" rowspan="1">Not observed</td></tr><tr><td colspan="1" rowspan="1">A/Netherlands/2290/2009</td><td colspan="1" rowspan="1">4.7 x 103</td><td colspan="1" rowspan="1">5.7 x 106</td><td colspan="1" rowspan="1">Not observed</td></tr><tr><td colspan="1" rowspan="1">A/California/04/2009</td><td colspan="1" rowspan="1">1.2 x 105</td><td colspan="1" rowspan="1">1.5 x 107</td><td colspan="1" rowspan="1">Not observed</td></tr><tr><td colspan="1" rowspan="10">A/H3N2</td><td colspan="1" rowspan="1">A/Wisconsin/67/2005</td><td colspan="1" rowspan="1">1.4 x 105</td><td colspan="1" rowspan="1">3.9 x 108</td><td colspan="1" rowspan="1">Not observed</td></tr><tr><td colspan="1" rowspan="1">A/Texas/50/2012</td><td colspan="1" rowspan="1">8.5 x 103</td><td colspan="1" rowspan="1">8.0 x 106</td><td colspan="1" rowspan="1">Not observed</td></tr><tr><td colspan="1" rowspan="1">A/Switzerland/9715293/2013</td><td colspan="1" rowspan="1">4.7 x 103</td><td colspan="1" rowspan="1">8.6 x 106</td><td colspan="1" rowspan="1">Not observed</td></tr><tr><td colspan="1" rowspan="1">A/South Australia/55/2014</td><td colspan="1" rowspan="1">1.6 x 104</td><td colspan="1" rowspan="1">2.4 x 107</td><td colspan="1" rowspan="1">Not observed</td></tr><tr><td colspan="1" rowspan="1">A/Stockholm/6/2014</td><td colspan="1" rowspan="1">1.2 x 104</td><td colspan="1" rowspan="1">8.6 x 106</td><td colspan="1" rowspan="1">Not observed</td></tr><tr><td colspan="1" rowspan="1">A/Norway/466/2014</td><td colspan="1" rowspan="1">5.9 x 103</td><td colspan="1" rowspan="1">4.6 x 106</td><td colspan="1" rowspan="1">Not observed</td></tr><tr><td colspan="1" rowspan="1">A/Texas/71/2007</td><td colspan="1" rowspan="1">1.8 x 104</td><td colspan="1" rowspan="1">3.0 x 107</td><td colspan="1" rowspan="1">Not observed</td></tr><tr><td colspan="1" rowspan="1">A/Santiago/7981/2006</td><td colspan="1" rowspan="1">1.7 x 104</td><td colspan="1" rowspan="1">1.3 x 107</td><td colspan="1" rowspan="1">Not observed</td></tr><tr><td colspan="1" rowspan="1">A/Hong Kong/4801/2014</td><td colspan="1" rowspan="1">1.6 x 106(EID50/mL)</td><td colspan="1" rowspan="1">2.8 x 107</td><td colspan="1" rowspan="1">Not observed</td></tr><tr><td colspan="1" rowspan="1">A/Uruguay/716/2007</td><td colspan="1" rowspan="1">8.5 x 103</td><td colspan="1" rowspan="1">1.3 x 107</td><td colspan="1" rowspan="1">Not observed</td></tr><tr><td colspan="1" rowspan="6">B/Victoria</td><td colspan="1" rowspan="1">B/Hong Kong/330/2001</td><td colspan="1" rowspan="1">7.2 x 104</td><td colspan="1" rowspan="1">2.1 x 107</td><td colspan="1" rowspan="1">Not observed</td></tr><tr><td colspan="1" rowspan="1">B/Nevada/03/2011</td><td colspan="1" rowspan="1">6.8 x 103</td><td colspan="1" rowspan="1">9.1 x 107</td><td colspan="1" rowspan="1">Not observed</td></tr><tr><td colspan="1" rowspan="1">B/Malaysia/2506/2004</td><td colspan="1" rowspan="1">1.9 x 103</td><td colspan="1" rowspan="1">2.1 x 107</td><td colspan="1" rowspan="1">Not observed</td></tr><tr><td colspan="1" rowspan="1">B/Victoria/304/2006</td><td colspan="1" rowspan="1">2.5 x 104</td><td colspan="1" rowspan="1">2.4 x 108</td><td colspan="1" rowspan="1">Not observed</td></tr><tr><td colspan="1" rowspan="1">B/New Jersey/01/2012</td><td colspan="1" rowspan="1">1.6x103</td><td colspan="1" rowspan="1">4.6 x 107</td><td colspan="1" rowspan="1">Not observed</td></tr><tr><td colspan="1" rowspan="1">B/Texas/02/2013</td><td colspan="1" rowspan="1">8.3x107(EID50/mL)</td><td colspan="1" rowspan="1">9.8 x 107</td><td colspan="1" rowspan="1">Not observed</td></tr><tr><td colspan="1" rowspan="5">B/Yamagata</td><td colspan="1" rowspan="1">B/Brisbane/3/2007</td><td colspan="1" rowspan="1">2.5 x 104</td><td colspan="1" rowspan="1">7.4 x 107</td><td colspan="1" rowspan="1">Not observed</td></tr><tr><td colspan="1" rowspan="1">B/Massachusetts/02/2012</td><td colspan="1" rowspan="1">1.7 x 104</td><td colspan="1" rowspan="1">6.0 x 107</td><td colspan="1" rowspan="1">Not observed</td></tr><tr><td colspan="1" rowspan="1">B/Texas/06/2011</td><td colspan="1" rowspan="1">2.3 x 103</td><td colspan="1" rowspan="1">4.6 x 107</td><td colspan="1" rowspan="1">Not observed</td></tr><tr><td colspan="1" rowspan="1">B/Florida/07/2004</td><td colspan="1" rowspan="1">7.2 x 105}$</td><td colspan="1" rowspan="1">3.2 x 107</td><td colspan="1" rowspan="1">Not observed</td></tr><tr><td colspan="1" rowspan="1">B/Pennsylvania/07/2007</td><td colspan="1" rowspan="1">2.8 x 103</td><td colspan="1" rowspan="1">8.0 x 106</td><td colspan="1" rowspan="1">Not observed</td></tr></table>

There were no misidentifications observed at the concentrations tested in this study.

Analytical Specificity for Non-Seasonal Influenza A Target Virus Group

Analytical specificity (exclusivity) was determined for 19 non-seasonal influenza A viruses. In this study the A/H5 and $\mathrm { A } / \mathrm { H } 7$ strains were tested as extracted genomic viral RNA for safety and availability reasons, and all other strains were tested as whole virus. Starting materials were spiked into pooled influenza negative clinical material at concentrations ${ > } 1 0 ^ { 6 }$ approximate genome copies/mL. Results of this study are shown in Table 10 below.

Table 10: Analytical Specificity of Non-Seasonal Influenza A Target Virus Group   

<table><tr><td colspan="1" rowspan="1">Subtype</td><td colspan="1" rowspan="1">Strain</td><td colspan="1" rowspan="1">Concentration(EID50/mL, unlessspecified)</td><td colspan="1" rowspan="1">Concentration(approximategenome copies/mL)</td><td colspan="1" rowspan="1">Mis- Identification</td></tr><tr><td colspan="1" rowspan="1">A/H1N1pre2009</td><td colspan="1" rowspan="1">A/Fujian Gulou/1896/2009</td><td colspan="1" rowspan="1">2.0 x 108</td><td colspan="1" rowspan="1">4.6 x 107</td><td colspan="1" rowspan="1">Not observed</td></tr><tr><td colspan="1" rowspan="1">A/H1N1sw</td><td colspan="1" rowspan="1">A/Swine/1976/1931</td><td colspan="1" rowspan="1">1.1 x 106</td><td colspan="1" rowspan="1">4.0 x 107</td><td colspan="1" rowspan="1">Not observed</td></tr><tr><td colspan="1" rowspan="1">A/H1N1v</td><td colspan="1" rowspan="1">A/Swine/Illinois/4L036/2015</td><td colspan="1" rowspan="1">No Titer Available</td><td colspan="1" rowspan="1">1.6 x 108</td><td colspan="1" rowspan="1">Not observed</td></tr><tr><td colspan="1" rowspan="1">A/H1N8</td><td colspan="1" rowspan="1">A/Red Knot/Delaware Bay/240/1994</td><td colspan="1" rowspan="1">1.1 x 109</td><td colspan="1" rowspan="1">8.5 x 107</td><td colspan="1" rowspan="1">Not observed</td></tr><tr><td colspan="1" rowspan="1">A/H2N2</td><td colspan="1" rowspan="1">A/Chicken/New York/SG-00425/2004</td><td colspan="1" rowspan="1">No Titer Available</td><td colspan="1" rowspan="1">1.2 x 108</td><td colspan="1" rowspan="1">Not observed</td></tr><tr><td colspan="1" rowspan="1">A/H3N2sw</td><td colspan="1" rowspan="1">A/Swine/Ohio/11SW87/2011</td><td colspan="1" rowspan="1">2.9 x 106TCID50/mL</td><td colspan="1" rowspan="1">2.0 x 107</td><td colspan="1" rowspan="1">Not observed</td></tr><tr><td colspan="1" rowspan="1">A/H3N2v</td><td colspan="1" rowspan="1">A/Minnesota/11/2010</td><td colspan="1" rowspan="1">2.8 x 107</td><td colspan="1" rowspan="1">6.4 x 107</td><td colspan="1" rowspan="1">Not observed</td></tr><tr><td colspan="1" rowspan="1">A/H3N8</td><td colspan="1" rowspan="1">A/Blue-Winged Teal/Alberta/221/1978</td><td colspan="1" rowspan="1">No Titer Available</td><td colspan="1" rowspan="1">7.0 x 106</td><td colspan="1" rowspan="1">Not observed</td></tr><tr><td colspan="1" rowspan="1">A/H3N8</td><td colspan="1" rowspan="1">A/Camel/Mongolia/335/2012</td><td colspan="1" rowspan="1">6.3 x 105</td><td colspan="1" rowspan="1">1.8 x 107</td><td colspan="1" rowspan="1">Not observed</td></tr><tr><td colspan="1" rowspan="1">A/H3N9</td><td colspan="1" rowspan="1">A/Mallard/Alberta/54/1993</td><td colspan="1" rowspan="1">1.1 x 108</td><td colspan="1" rowspan="1">8.5 x 107</td><td colspan="1" rowspan="1">Not observed</td></tr><tr><td colspan="1" rowspan="1">A/H4N6</td><td colspan="1" rowspan="1">A/Blue-Winged Teal/Illinois/100OS1563/2010</td><td colspan="1" rowspan="1">1.1 x 108</td><td colspan="1" rowspan="1">1.5 x 108</td><td colspan="1" rowspan="1">Not observed</td></tr><tr><td colspan="1" rowspan="1">A/H5N1</td><td colspan="1" rowspan="1">A/Goose/Yunnan/5539/2005*</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">3.5 x 106</td><td colspan="1" rowspan="1">Not observed</td></tr><tr><td colspan="1" rowspan="1">A/H5N2</td><td colspan="1" rowspan="1">A/Northern Pintail/Washington/ 40964/2014*</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">1.8 x 107</td><td colspan="1" rowspan="1">Not observed</td></tr><tr><td colspan="1" rowspan="1">A/H5N8</td><td colspan="1" rowspan="1">A/Duck/England/36254/2014*</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">3.8 x 106</td><td colspan="1" rowspan="1">Not observed</td></tr><tr><td colspan="1" rowspan="1">A/H6N4</td><td colspan="1" rowspan="1">A/Mallard/Alberta/58/1989</td><td colspan="1" rowspan="1">9.1 x 106</td><td colspan="1" rowspan="1">2.0 x 108</td><td colspan="1" rowspan="1">Not observed</td></tr><tr><td colspan="1" rowspan="1">A/H7N7</td><td colspan="1" rowspan="1">A/Italy/3/2013*</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">1.9 x 106</td><td colspan="1" rowspan="1">Not observed</td></tr><tr><td colspan="1" rowspan="1">A/H7N9</td><td colspan="1" rowspan="1">A/Hong Kong/5942/2013*</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">2.3 x 106</td><td colspan="1" rowspan="1">Not observed</td></tr><tr><td colspan="1" rowspan="1">A/H9N2</td><td colspan="1" rowspan="1">A/Turkey/Wisconsin/1/1966</td><td colspan="1" rowspan="1">1.1 x 108</td><td colspan="1" rowspan="1">2.8 x 108</td><td colspan="1" rowspan="1">Not observed</td></tr><tr><td colspan="1" rowspan="1">A/H11N2</td><td colspan="1" rowspan="1">A/Laughing Gull/DelawareBay/94/1995</td><td colspan="1" rowspan="1">1.4 x 109</td><td colspan="1" rowspan="1">9.8 x 107</td><td colspan="1" rowspan="1">Not observed</td></tr></table>

\* Extracted genomic viral RNA.

There were no misidentifications observed at the concentrations tested in this study.

# Analytical Specificity with Non-Influenza Respiratory Pathogens

Analytical specificity of the FC8G assay with non-influenza organisms that may be present in the respiratory tract of patients presenting with influenza-like illness was conducted testing a panel of 34 non-influenza organisms (19 bacteria isolates and 15 viruses). A stock of each organism in culture media at known titer was spiked into pooled influenza-negative clinical material at concentrations ${ \geq } 1 0 ^ { 6 } \mathrm { C F U / m L }$ for bacteria and $\geq 1 0 ^ { 5 } \mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ or $\mathrm { C C U / m L }$ for viruses and atypical bacteria, whenever possible. Each organism was extracted once and tested in triplicate. For any organisms producing a false positive result of influenza, additional replicates were tested as needed to evaluate consistency of assay results in the presence of potentially crossreactive organisms. Final results of this study are shown in Table 11 below.

Table 11: Analytical Specificity of Non-Influenza Organisms   

<table><tr><td colspan="1" rowspan="1">Organism</td><td colspan="1" rowspan="1">Concentration Tested</td><td colspan="1" rowspan="1">Cross-Reactivity</td></tr><tr><td colspan="1" rowspan="1">Adenovirus 71</td><td colspan="1" rowspan="1">8.0 x 106 TCID50/mL</td><td colspan="1" rowspan="1">Not observed</td></tr><tr><td colspan="1" rowspan="1">Adenovirus 7A</td><td colspan="1" rowspan="1">3.0 x 107 TCID50/mL</td><td colspan="1" rowspan="1">Not observed</td></tr><tr><td colspan="1" rowspan="1">Coronavirus 229E</td><td colspan="1" rowspan="1">2.5 x 105 TCID50/mL</td><td colspan="1" rowspan="1">Not observed</td></tr><tr><td colspan="1" rowspan="1">Coronavirus OC43</td><td colspan="1" rowspan="1">1.4 x 105 TCID50/mL</td><td colspan="1" rowspan="1">Not observed</td></tr><tr><td colspan="1" rowspan="1">Cytomegalovirus</td><td colspan="1" rowspan="1">1.2 x 105 TCID50/mL</td><td colspan="1" rowspan="1">Not observed</td></tr><tr><td colspan="1" rowspan="1">Enterovirus 71</td><td colspan="1" rowspan="1">8.0 x 105 TCID50/mL</td><td colspan="1" rowspan="1">Not observed</td></tr><tr><td colspan="1" rowspan="1">Epstein Barr Virus</td><td colspan="1" rowspan="1">2.2 x 108 TCID50/mL</td><td colspan="1" rowspan="1">Not observed</td></tr><tr><td colspan="1" rowspan="1">Human parainfluenza 1</td><td colspan="1" rowspan="1">9.5 x 106 TCID50/mL</td><td colspan="1" rowspan="1">Not observed</td></tr><tr><td colspan="1" rowspan="1">Human parainfluenza 2</td><td colspan="1" rowspan="1">4.5 x 105 TCID50/mL</td><td colspan="1" rowspan="1">Not observed</td></tr><tr><td colspan="1" rowspan="1">Human parainfluenza 3</td><td colspan="1" rowspan="1">2.5 x 105 TCID50/mL</td><td colspan="1" rowspan="1">Not observed</td></tr><tr><td colspan="1" rowspan="1">Measles</td><td colspan="1" rowspan="1">1.3 x 105 TCID50/mL</td><td colspan="1" rowspan="1">Not observed</td></tr><tr><td colspan="1" rowspan="1">Human metapneumovirus</td><td colspan="1" rowspan="1">1.3 x 105 TCID50/mL</td><td colspan="1" rowspan="1">Not observed</td></tr><tr><td colspan="1" rowspan="1">Mumps virus</td><td colspan="1" rowspan="1">5.0 x 105 TCID50/mL</td><td colspan="1" rowspan="1">Not observed</td></tr><tr><td colspan="1" rowspan="1">Human respiratory syncytial virus</td><td colspan="1" rowspan="1">1.3 x 105 TCID50/mL</td><td colspan="1" rowspan="1">Not observed</td></tr><tr><td colspan="1" rowspan="1">Rhinovirus 1A</td><td colspan="1" rowspan="1">7.0 x 104 TCID50/mL</td><td colspan="1" rowspan="1">Not observed</td></tr><tr><td colspan="1" rowspan="1">Bordetella pertussis</td><td colspan="1" rowspan="1">3.8 x 109 CFU/mL</td><td colspan="1" rowspan="1">Not observed</td></tr><tr><td colspan="1" rowspan="1">Chlamydophila pneumoniae</td><td colspan="1" rowspan="1">8.0 x 105 TCID50/mL</td><td colspan="1" rowspan="1">Not observed</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium diphtheriae</td><td colspan="1" rowspan="1">2.3 x 106 CFU/mL</td><td colspan="1" rowspan="1">Not observed</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">4.7 x 109 CFU/mL</td><td colspan="1" rowspan="1">Not observed</td></tr><tr><td colspan="1" rowspan="1">Hemophilus influenza</td><td colspan="1" rowspan="1">2.7 x 108 CFU/mL</td><td colspan="1" rowspan="1">Not observed</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus acidophilus</td><td colspan="1" rowspan="1">1.3 x 108 CFU/mL</td><td colspan="1" rowspan="1">Not observed</td></tr><tr><td colspan="1" rowspan="1">Legionella pneumophila</td><td colspan="1" rowspan="1">9.0 x 109 CFU/mL</td><td colspan="1" rowspan="1">Not observed</td></tr><tr><td colspan="1" rowspan="1">Moraxella catarrhalis</td><td colspan="1" rowspan="1">6.5 x 106 CFU/mL</td><td colspan="1" rowspan="1">Not observed</td></tr><tr><td colspan="1" rowspan="1">Mycobacterium tuberculosis</td><td colspan="1" rowspan="1">7.0 x 106 CFU/mL</td><td colspan="1" rowspan="1">Not observed</td></tr><tr><td colspan="1" rowspan="1">Mycoplasma pneumoniae</td><td colspan="1" rowspan="1">1.0 x 106 CCU/mL</td><td colspan="1" rowspan="1">Not observed</td></tr><tr><td colspan="1" rowspan="1">Neisseria meningitidis</td><td colspan="1" rowspan="1">1.0 x 106 CFU/mL</td><td colspan="1" rowspan="1">Not observed</td></tr><tr><td colspan="1" rowspan="1">Neisseria sicca</td><td colspan="1" rowspan="1">1.8 x 108 CFU/mL</td><td colspan="1" rowspan="1">Not observed</td></tr><tr><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">1.0 x 106 CFU/mL</td><td colspan="1" rowspan="1">Not observed</td></tr><tr><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">1.0 x 106 CFU/mL</td><td colspan="1" rowspan="1">Not observed</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">7.5 x 108 CFU/mL</td><td colspan="1" rowspan="1">Not observed</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus epidermidis</td><td colspan="1" rowspan="1">7.5 x 108 CFU/mL</td><td colspan="1" rowspan="1">Not observed</td></tr><tr><td colspan="1" rowspan="1">Streptococcus pneumoniae</td><td colspan="1" rowspan="1">3.4 x 107 CFU/mL</td><td colspan="1" rowspan="1">Not observed</td></tr><tr><td colspan="1" rowspan="1">Streptococcus pyogenes</td><td colspan="1" rowspan="1">8.5 x 106 CFU/mL</td><td colspan="1" rowspan="1">Not observed</td></tr><tr><td colspan="1" rowspan="1">Streptococcus salivarius</td><td colspan="1" rowspan="1">3.8 x 106 CFU/mL</td><td colspan="1" rowspan="1">Not observed</td></tr></table>

None of the organisms tested in this study were found to be cross-reactive with the FC8G assay at the concentrations tested.

# Microbial Interference Study

Microbial interference was assessed for the FC8G assay by testing a 10-member test panel that represented an abbreviated version of the test panel used in the Analytical Specificity Study above. The non-influenza organism at high concentration (i.e., ${ \geq } 1 0 ^ { 6 }$ CFU/mL for bacteria and $\ge 1 0 ^ { 5 } \mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ or $\mathrm { C C U / m L }$ for viruses and atypical bacteria) was mixed with an influenza virus at close to the LoD concentration in the same sample to determine if the presence of the non-target organism would interfere with the detection of the influenza analyte using the FC8G assay.

Contrived samples with one representative virus strain from each of the five target virus groups, A/California/07/2009 (A/H1N1pdm09), A/Victoria/361/2011 (A/H3N2), B/Florida/02/2006 (B/Victoria), B/Wisconsin/01/2010 (B/Yamagata), and A/New Caledonia/20/1999 (A/H1N1 pre-2009, non-seasonal influenza A), were prepared at a concentration of approximately 3x to 5x LoD. Influenza samples were prepared in pooled influenza-negative clinical material, and the diluted non-influenza organisms were added to the extraction buffer to yield the desired concentration for testing. Each contrived sample was extracted once and tested in triplicate.

Results of the Microbial Interference Study are summarized in Table 12 below.

Table 12: Summary Results of the Microbial Interference Study   

<table><tr><td rowspan=2 colspan=1>Non-Influenza Organism</td><td rowspan=2 colspan=1>ConcentrationTested</td><td rowspan=1 colspan=5># of Correct FC8G Assay Results</td><td rowspan=2 colspan=1>OverallAgreement</td><td rowspan=2 colspan=1>Interference</td></tr><tr><td rowspan=1 colspan=1>A/H1N1pdm09</td><td rowspan=1 colspan=1>A/H3N2</td><td rowspan=1 colspan=1>B/Victoria</td><td rowspan=1 colspan=1>B/Yamagata</td><td rowspan=1 colspan=1>A/H1N1(pre-2009)</td></tr><tr><td rowspan=1 colspan=1>None</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>N/A</td></tr><tr><td rowspan=1 colspan=1>Adenovirus 71</td><td rowspan=1 colspan=1>1.6 x 106 TCID50/mL</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>Not observed</td></tr><tr><td rowspan=1 colspan=1>Coronavirus 229E</td><td rowspan=1 colspan=1>1.0 x 105 TCID50/mL</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>Not observed</td></tr><tr><td rowspan=1 colspan=1>Human parainfluenza 3</td><td rowspan=1 colspan=1>1.0 x 105 TCID50/mL</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>Not observed</td></tr><tr><td rowspan=1 colspan=1>Human metapneumovirus</td><td rowspan=1 colspan=1>1.2 x 105 TCID50/mL</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>Not observed</td></tr><tr><td rowspan=1 colspan=1>Human respiratorysyncytial virus</td><td rowspan=1 colspan=1>8.5 x 106 TCID50/mL</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>Not observed</td></tr><tr><td rowspan=1 colspan=1>Rhinovirus 1A</td><td rowspan=1 colspan=1>1.1 x 105 TCID50/mL</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>Not observed</td></tr><tr><td rowspan=1 colspan=1>Hemophilus influenza</td><td rowspan=1 colspan=1>3.3 x 106 CFU/mL</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>Not observed</td></tr><tr><td rowspan=1 colspan=1>Mycoplasma pneumoniae</td><td rowspan=1 colspan=1>1.1 x 106 CCU/mL</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>Not observed</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus aureus</td><td rowspan=1 colspan=1>1.7 x 106 CFU/mL</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>Not observed</td></tr><tr><td rowspan=1 colspan=1>Streptococcus pneumoniae</td><td rowspan=1 colspan=1>6.8 x 106 CFU/mL</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>2/3*</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>93%</td><td rowspan=1 colspan=1>Not observed</td></tr></table>

\* One Influenza B Yamagata sample was identified only as influenza B only (without Yamagata lineage).

The results showed that the evaluated non-influenza organisms at the concentrations tested in this study did not interfere with the detection of influenza virus samples at close to the LoD concentrations.

# g. Influenza Competitive Interference Study

To assess performance of the FC8G assay for samples containing two different influenza viruses, samples containing one representative virus from each of two different target virus groups were spiked into pooled influenza negative clinical material and analyzed by the FC8G assay. Mock dual infection samples that represented all permutations of dual infections of the following five influenza strains, A/South Carolina/18/2009, A/Victoria/361/2011, A/Ruddy Turnstone/Delaware Bay/39/1994, B/New Jersey/01/2012, and B/Texas/06/2011, representing the five target virus groups, A/H1N1pdm09, A/H3N2, A/H11N3 (non-seasonal influenza A), B/Victoria, and B/Yamagata, respectively, at three different concentrations of each included virus (high, medium, and low, corresponding to approximately 500x, 60x, and 4x LoD, respectively, based on the assessment of Ct values generated by an FDAcleared influenza molecular assay), were analyze for a total of 90 samples, as outlined in Table 13 below.

Results of the Competitive Interference Study are summarized in Table 13 below.

Table 13: Competitive Interference Study Testing Scheme and Results   

<table><tr><td rowspan=2 colspan=1>Sample</td><td rowspan=1 colspan=4>Source 1</td><td rowspan=1 colspan=6>Source 2</td><td rowspan=1 colspan=1>Source 1</td><td rowspan=2 colspan=1>Source 2VirusIdentified</td><td rowspan=2 colspan=1>Both Sources ofViruses Identified</td></tr><tr><td rowspan=1 colspan=3>Virus</td><td rowspan=1 colspan=1>Concentration</td><td rowspan=1 colspan=5>Virus</td><td rowspan=1 colspan=1>Concentration</td><td rowspan=1 colspan=1>VirusIdentified</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=2 colspan=3></td><td rowspan=1 colspan=1>High</td><td rowspan=9 colspan=5>A/H3N2</td><td rowspan=1 colspan=1>High</td><td rowspan=1 colspan=1>No</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>No</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=2 colspan=1></td><td rowspan=1 colspan=1>High</td><td rowspan=1 colspan=1>Medium</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>No</td><td rowspan=1 colspan=1>No</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=2 colspan=3></td><td rowspan=1 colspan=1>High</td><td rowspan=1 colspan=1>Low</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>No</td><td rowspan=1 colspan=1>No</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>Medium</td><td rowspan=1 colspan=1>High</td><td rowspan=1 colspan=1>No</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>No</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=7 colspan=3>−</td><td rowspan=1 colspan=1>Medium</td><td rowspan=1 colspan=1>Medium</td><td rowspan=1 colspan=1>No</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>No</td></tr><tr><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>Medium</td><td rowspan=1 colspan=1>Low</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>No</td><td rowspan=1 colspan=1>No</td></tr><tr><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>Low</td><td rowspan=1 colspan=1>High</td><td rowspan=1 colspan=1>No</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>No</td></tr><tr><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>Low</td><td rowspan=1 colspan=1>Medium</td><td rowspan=1 colspan=1>No</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>No</td></tr><tr><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>Low</td><td rowspan=1 colspan=1>Low</td><td rowspan=1 colspan=1>No</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>No</td></tr><tr><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>High</td><td rowspan=5 colspan=5>A/H11N3</td><td rowspan=1 colspan=1>High</td><td rowspan=1 colspan=1>No</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>No</td></tr><tr><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>High</td><td rowspan=1 colspan=1>Medium</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>Yes</td></tr><tr><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1></td><td rowspan=22 colspan=2>A/H1N1pdm09</td><td rowspan=1 colspan=1>High</td><td rowspan=1 colspan=1>Low</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>No</td><td rowspan=1 colspan=1>No</td></tr><tr><td rowspan=1 colspan=1>13</td><td></td><td rowspan=1 colspan=1>Medium</td><td rowspan=1 colspan=1>High</td><td rowspan=1 colspan=1>No</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>No</td></tr><tr><td rowspan=1 colspan=1>14</td><td></td><td rowspan=1 colspan=1>Medium</td><td rowspan=1 colspan=1>Medium</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>Yes</td></tr><tr><td rowspan=1 colspan=1>15</td><td></td><td rowspan=1 colspan=1>Medium</td><td rowspan=1 colspan=4></td><td rowspan=1 colspan=4></td><td rowspan=1 colspan=1>Low</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>No</td><td rowspan=1 colspan=1>No</td></tr><tr><td rowspan=1 colspan=1>16</td><td></td><td rowspan=1 colspan=1>Low</td><td rowspan=3 colspan=5></td><td rowspan=1 colspan=1>High</td><td rowspan=1 colspan=1>No</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>No</td></tr><tr><td rowspan=1 colspan=1>17</td><td></td><td rowspan=1 colspan=1>Low</td><td rowspan=1 colspan=1>Medium</td><td rowspan=1 colspan=1>No</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>No</td></tr><tr><td rowspan=1 colspan=1>18</td><td></td><td rowspan=1 colspan=1>Low</td><td rowspan=1 colspan=1>Low</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>Yes</td></tr><tr><td rowspan=1 colspan=1>19</td><td></td><td rowspan=1 colspan=1>High</td><td rowspan=4 colspan=5></td><td rowspan=1 colspan=1>High</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>Yes</td></tr><tr><td rowspan=1 colspan=1>20</td><td></td><td rowspan=1 colspan=1>High</td><td rowspan=1 colspan=1>Medium</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>Yes</td></tr><tr><td rowspan=1 colspan=1>21</td><td></td><td rowspan=1 colspan=1>High</td><td rowspan=1 colspan=1>Low</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>No</td><td rowspan=1 colspan=1>No</td></tr><tr><td rowspan=1 colspan=1>22</td><td></td><td rowspan=1 colspan=1>Medium</td><td rowspan=1 colspan=1>High</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>Yes</td></tr><tr><td rowspan=1 colspan=1>23</td><td></td><td rowspan=1 colspan=1>Medium</td><td rowspan=1 colspan=2>B/Victoria</td><td></td><td></td><td rowspan=1 colspan=1>a</td><td rowspan=1 colspan=1>Medium</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>Yes</td></tr><tr><td rowspan=1 colspan=1>24</td><td></td><td rowspan=1 colspan=1>Medium</td><td rowspan=4 colspan=5></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Low</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>No</td><td rowspan=1 colspan=1>No</td></tr><tr><td rowspan=1 colspan=1>25</td><td></td><td rowspan=1 colspan=1>Low</td><td rowspan=1 colspan=1>High</td><td rowspan=1 colspan=1>No</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>No</td></tr><tr><td rowspan=1 colspan=1>26</td><td></td><td rowspan=1 colspan=1>Low</td><td rowspan=1 colspan=1>Medium</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>Yes</td></tr><tr><td rowspan=1 colspan=1>27</td><td></td><td rowspan=1 colspan=1>Low</td><td rowspan=1 colspan=1>Low</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>Yes</td></tr><tr><td rowspan=1 colspan=1>28</td><td></td><td rowspan=1 colspan=1>High</td><td rowspan=4 colspan=5></td><td rowspan=1 colspan=1>High</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>No</td><td rowspan=1 colspan=1>No</td></tr><tr><td rowspan=1 colspan=1>29</td><td></td><td rowspan=1 colspan=1>High</td><td rowspan=1 colspan=1>Medium</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>No</td><td rowspan=1 colspan=1>No</td></tr><tr><td rowspan=1 colspan=1>30</td><td></td><td rowspan=1 colspan=1>High</td><td rowspan=1 colspan=1>Low</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>No</td><td rowspan=1 colspan=1>No</td></tr><tr><td rowspan=1 colspan=1>31</td><td></td><td rowspan=1 colspan=1>Medium</td><td rowspan=1 colspan=1>High</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>Yes</td></tr><tr><td rowspan=1 colspan=1>32</td><td></td><td rowspan=1 colspan=1>Medium</td><td rowspan=1 colspan=5>B/Yamagata</td><td rowspan=1 colspan=1>Medium</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>Yes</td></tr><tr><td rowspan=1 colspan=1>33</td><td rowspan=2 colspan=1></td><td rowspan=1 colspan=1>Medium</td><td rowspan=2 colspan=5></td><td rowspan=1 colspan=1>Low</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>No</td><td rowspan=1 colspan=1>No</td></tr><tr><td rowspan=1 colspan=1>34</td><td rowspan=3 colspan=3></td><td rowspan=1 colspan=1>Low</td><td rowspan=3 colspan=5></td><td rowspan=1 colspan=1>High</td><td rowspan=1 colspan=1>No</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>No</td></tr><tr><td rowspan=1 colspan=1>35</td><td rowspan=1 colspan=1>Low</td><td rowspan=1 colspan=1>Medium</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>Yes</td></tr><tr><td rowspan=1 colspan=1>36</td><td rowspan=1 colspan=1>Low</td><td rowspan=1 colspan=1>Low</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>Yes</td></tr><tr><td rowspan=1 colspan=1>37</td><td rowspan=7 colspan=3>A/H3N2</td><td rowspan=1 colspan=1>High</td><td rowspan=5 colspan=5>A/H11N3</td><td rowspan=1 colspan=1>High</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>Yes</td></tr><tr><td rowspan=1 colspan=1>38</td><td rowspan=1 colspan=1>High</td><td rowspan=1 colspan=1>Medium</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>No</td><td rowspan=1 colspan=1>No</td></tr><tr><td rowspan=1 colspan=1>39</td><td rowspan=1 colspan=1>High</td><td rowspan=1 colspan=1>Low</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>No</td><td rowspan=1 colspan=1>No</td></tr><tr><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>Medium</td><td rowspan=1 colspan=1>High</td><td rowspan=1 colspan=1>No</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>No</td></tr><tr><td rowspan=1 colspan=1>41</td><td rowspan=1 colspan=1>Medium</td><td rowspan=1 colspan=4>A/H11N3</td><td rowspan=1 colspan=1>Medium</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>Yes</td></tr><tr><td rowspan=1 colspan=1>42</td><td rowspan=1 colspan=1>Medium</td><td rowspan=4 colspan=5></td><td rowspan=1 colspan=3></td><td rowspan=1 colspan=1>Low</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>No</td><td rowspan=1 colspan=1>No</td></tr><tr><td rowspan=1 colspan=1>43</td><td rowspan=1 colspan=1>Low</td><td rowspan=1 colspan=1>High</td><td rowspan=1 colspan=1>No</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>No</td></tr><tr><td rowspan=1 colspan=1>44</td><td rowspan=6 colspan=3></td><td rowspan=1 colspan=1>Low</td><td rowspan=1 colspan=1>Medium</td><td rowspan=1 colspan=1>No</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>No</td></tr><tr><td rowspan=1 colspan=1>45</td><td rowspan=1 colspan=1>Low</td><td rowspan=1 colspan=1>Low</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>Yes</td></tr><tr><td rowspan=1 colspan=1>46</td><td rowspan=1 colspan=1>High</td><td rowspan=4 colspan=5>B/Victoria</td><td rowspan=1 colspan=1>High</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>Yes</td></tr><tr><td rowspan=1 colspan=1>47</td><td rowspan=1 colspan=1>High</td><td rowspan=1 colspan=1>Medium</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>Yes</td></tr><tr><td rowspan=1 colspan=1>48</td><td rowspan=1 colspan=1>High</td><td rowspan=1 colspan=1>Low</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>No</td><td rowspan=1 colspan=1>No</td></tr><tr><td rowspan=1 colspan=1>49</td><td rowspan=1 colspan=1>Medium</td><td rowspan=1 colspan=1>High</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>Yes</td></tr></table>

<table><tr><td rowspan=1 colspan=1>50</td><td rowspan=3 colspan=7></td><td rowspan=1 colspan=1>Medium</td><td rowspan=3 colspan=4></td><td rowspan=1 colspan=1>Medium</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>Yes</td></tr><tr><td rowspan=1 colspan=1>51</td><td rowspan=1 colspan=4></td><td rowspan=1 colspan=1>Medium</td><td rowspan=1 colspan=1>Low</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>No</td><td rowspan=1 colspan=1>No</td></tr><tr><td rowspan=1 colspan=1>52</td><td rowspan=1 colspan=2></td><td rowspan=1 colspan=1>Low</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>High</td><td rowspan=1 colspan=1>No</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>No</td></tr><tr><td rowspan=1 colspan=1>53</td><td rowspan=1 colspan=1></td><td></td><td></td><td rowspan=1 colspan=4></td><td rowspan=1 colspan=1>Low</td><td rowspan=2 colspan=4></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Medium</td><td rowspan=1 colspan=1>No</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>No</td></tr><tr><td rowspan=1 colspan=1>54</td><td rowspan=1 colspan=1></td><td></td><td></td><td rowspan=1 colspan=4></td><td rowspan=1 colspan=1>Low</td><td rowspan=1 colspan=1>Low</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>Yes</td></tr><tr><td rowspan=1 colspan=1>55</td><td rowspan=1 colspan=2></td><td rowspan=7 colspan=5></td><td rowspan=1 colspan=1>High</td><td rowspan=7 colspan=4>B/Yamagata</td><td rowspan=1 colspan=1>High</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>No</td><td rowspan=1 colspan=1>No</td></tr><tr><td rowspan=3 colspan=1>56</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr><tr><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr><tr><td></td><td></td><td rowspan=1 colspan=1>High</td><td rowspan=1 colspan=1>Medium</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>No</td><td rowspan=1 colspan=1>No</td></tr><tr><td rowspan=1 colspan=1>57</td><td></td><td></td><td rowspan=1 colspan=1>High</td><td rowspan=1 colspan=1>Low</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>No</td><td rowspan=1 colspan=1>No</td></tr><tr><td rowspan=1 colspan=1>58</td><td rowspan=1 colspan=4>b</td><td rowspan=1 colspan=1>Medium</td><td rowspan=1 colspan=1>High</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>Yes</td></tr><tr><td rowspan=1 colspan=1>59</td><td rowspan=1 colspan=2></td><td rowspan=1 colspan=1>Medium</td><td rowspan=1 colspan=1>B</td><td rowspan=1 colspan=1>Medium</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>Yes</td></tr><tr><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1></td><td></td><td></td><td rowspan=1 colspan=4></td><td rowspan=1 colspan=1>Medium</td><td rowspan=1 colspan=2></td><td rowspan=1 colspan=3></td><td rowspan=1 colspan=1>Low</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>No</td><td rowspan=1 colspan=1>No</td></tr><tr><td rowspan=1 colspan=1>61</td><td rowspan=1 colspan=1>b</td><td></td><td></td><td rowspan=1 colspan=4></td><td rowspan=1 colspan=1>Low</td><td rowspan=7 colspan=4></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>High</td><td rowspan=1 colspan=1>No</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>No</td></tr><tr><td rowspan=5 colspan=1>62</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr><tr><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr><tr><td rowspan=4 colspan=7></td><td rowspan=3 colspan=1></td><td rowspan=3 colspan=1>Low</td><td></td><td></td><td></td><td></td></tr><tr><td></td><td></td><td></td><td></td></tr><tr><td rowspan=1 colspan=1>Medium</td><td rowspan=1 colspan=1>No</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>No</td></tr><tr><td rowspan=1 colspan=1>63</td><td rowspan=1 colspan=1>Low</td><td rowspan=1 colspan=1>Low</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>Yes</td></tr><tr><td rowspan=1 colspan=1>64</td><td rowspan=2 colspan=7></td><td rowspan=1 colspan=1>High</td><td rowspan=5 colspan=4>B/Victoria</td><td rowspan=1 colspan=1>High</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>Yes</td></tr><tr><td rowspan=1 colspan=1>65</td><td rowspan=1 colspan=2></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>High</td><td rowspan=1 colspan=1>Medium</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>Yes</td></tr><tr><td rowspan=1 colspan=1>66</td><td rowspan=1 colspan=1></td><td></td><td></td><td rowspan=1 colspan=3></td><td rowspan=1 colspan=2></td><td rowspan=1 colspan=1>High</td><td rowspan=1 colspan=1>Low</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>No</td><td rowspan=1 colspan=1>No</td></tr><tr><td rowspan=1 colspan=1>67</td><td rowspan=1 colspan=1></td><td></td><td></td><td rowspan=1 colspan=4></td><td rowspan=1 colspan=1>Medium</td><td rowspan=1 colspan=1>High</td><td rowspan=1 colspan=1>No</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>No</td></tr><tr><td rowspan=1 colspan=1>68</td><td rowspan=1 colspan=2></td><td></td><td rowspan=1 colspan=4></td><td rowspan=1 colspan=1>Medium</td><td rowspan=1 colspan=2>B</td><td rowspan=1 colspan=1>Medium</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>Yes</td></tr><tr><td rowspan=1 colspan=1>69</td><td rowspan=1 colspan=2></td><td rowspan=7 colspan=5>A/H11N3</td><td rowspan=1 colspan=1>Medium</td><td rowspan=6 colspan=4></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Low</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>No</td></tr><tr><td rowspan=3 colspan=1>70</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr><tr><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr><tr><td></td><td></td><td rowspan=1 colspan=1>Low</td><td rowspan=1 colspan=1>High</td><td rowspan=1 colspan=1>No</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>No</td></tr><tr><td rowspan=1 colspan=1>71</td><td></td><td></td><td rowspan=1 colspan=1>Low</td><td rowspan=1 colspan=1>Medium</td><td rowspan=1 colspan=1>No</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>No</td></tr><tr><td rowspan=1 colspan=1>72</td><td rowspan=1 colspan=4></td><td rowspan=1 colspan=1>Low</td><td rowspan=1 colspan=1>Low</td><td rowspan=1 colspan=1>No</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>No</td></tr><tr><td rowspan=1 colspan=1>73</td><td rowspan=1 colspan=3></td><td rowspan=1 colspan=1>High</td><td rowspan=5 colspan=4>B/Yamagata</td><td rowspan=1 colspan=1>High</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>Yes</td></tr><tr><td rowspan=1 colspan=1>74</td><td rowspan=1 colspan=3></td><td rowspan=2 colspan=4></td><td rowspan=1 colspan=1>High</td><td rowspan=1 colspan=1>Medium</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>Yes</td></tr><tr><td rowspan=1 colspan=1>75</td><td rowspan=1 colspan=1>b</td><td rowspan=1 colspan=3></td><td rowspan=1 colspan=2></td><td rowspan=1 colspan=1>High</td><td rowspan=1 colspan=1>Low</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>No</td><td rowspan=1 colspan=1>No</td></tr><tr><td rowspan=1 colspan=1>76</td><td rowspan=1 colspan=4></td><td rowspan=3 colspan=3></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=4>Medium</td><td rowspan=1 colspan=1>B/</td><td rowspan=1 colspan=1>High</td><td rowspan=1 colspan=1>No</td><td rowspan=1 colspan=1>Yes</td></tr><tr><td rowspan=1 colspan=1>77</td><td></td><td></td><td></td><td></td><td rowspan=1 colspan=1>Medium</td><td rowspan=1 colspan=2>B/Y</td><td rowspan=1 colspan=1>Medium</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>Yes</td></tr><tr><td rowspan=1 colspan=1>78</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=3></td><td rowspan=1 colspan=1>Medium</td><td rowspan=6 colspan=4></td><td rowspan=1 colspan=2></td><td rowspan=1 colspan=1>Low</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>No</td><td rowspan=1 colspan=1>No</td></tr><tr><td rowspan=3 colspan=1>79</td><td rowspan=3 colspan=2></td><td></td><td rowspan=3 colspan=4></td><td rowspan=3 colspan=1>Low</td><td></td><td></td><td></td><td></td></tr><tr><td></td><td></td><td></td><td></td><td></td></tr><tr><td></td><td rowspan=1 colspan=1>High</td><td rowspan=1 colspan=1>No</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>No</td></tr><tr><td rowspan=1 colspan=1>80</td><td rowspan=1 colspan=3></td><td rowspan=2 colspan=4></td><td rowspan=1 colspan=1>Low</td><td rowspan=1 colspan=1>Medium</td><td rowspan=1 colspan=1>No</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>No</td></tr><tr><td rowspan=1 colspan=1>81</td><td></td><td></td><td></td><td rowspan=1 colspan=1>Low</td><td rowspan=1 colspan=1>Low</td><td rowspan=1 colspan=1>No</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>No</td></tr><tr><td rowspan=1 colspan=1>82</td><td rowspan=5 colspan=7>B/Victoria</td><td rowspan=1 colspan=1>High</td><td rowspan=4 colspan=4></td><td rowspan=1 colspan=1>High</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>No</td><td rowspan=1 colspan=1>No</td></tr><tr><td rowspan=1 colspan=1>83</td><td rowspan=1 colspan=1>High</td><td rowspan=1 colspan=1>Medium</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>No</td><td rowspan=1 colspan=1>No</td></tr><tr><td rowspan=1 colspan=1>84</td><td rowspan=1 colspan=1>High</td><td rowspan=1 colspan=1>Low</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>No</td><td rowspan=1 colspan=1>No</td></tr><tr><td rowspan=1 colspan=1>85</td><td rowspan=1 colspan=1>Medium</td><td rowspan=1 colspan=1>High</td><td rowspan=1 colspan=1>No</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>No</td></tr><tr><td rowspan=1 colspan=1>86</td><td rowspan=1 colspan=1>Medium</td><td rowspan=1 colspan=4>B/Yamagata</td><td rowspan=1 colspan=1>Medium</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>No</td><td rowspan=1 colspan=1>No</td></tr><tr><td rowspan=1 colspan=1>87</td><td rowspan=1 colspan=7></td><td rowspan=1 colspan=1>Medium</td><td rowspan=4 colspan=4></td><td rowspan=1 colspan=1>Low</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>No</td><td rowspan=1 colspan=1>No</td></tr><tr><td rowspan=1 colspan=1>88</td><td rowspan=3 colspan=7></td><td rowspan=1 colspan=1>Low</td><td rowspan=1 colspan=1>High</td><td rowspan=1 colspan=1>No</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>No</td></tr><tr><td rowspan=1 colspan=1>89</td><td rowspan=1 colspan=1>Low</td><td rowspan=1 colspan=1>Medium</td><td rowspan=1 colspan=1>No</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>No</td></tr><tr><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>Low</td><td rowspan=1 colspan=1>Low</td><td rowspan=1 colspan=1>No</td><td rowspan=1 colspan=1>No</td><td rowspan=1 colspan=1>No</td></tr></table>

The FC8G assay resulted in a correct dual infection identification for $33 \%$ (30/90) of samples analyzed, and correctly identified at least one of the two viruses in the sample $6 6 \%$ (59/90) of the time. One sample, containing B/Victoria and B/Yamagata at low concentrations, resulted in an “influenza B detected (no lineage)” result.

Results of this mock dual infection testing, organized by relative concentrations of the two viruses, are presented in Table 14 below.

Table 14: Results of the Mock Dual Infection Testing, Organized by Relative Concentrations of the Two Viruses   

<table><tr><td rowspan=2 colspan=2>Relative Concentration</td><td rowspan=2 colspan=1>Concentration(Source 1 + Source 2)</td><td rowspan=2 colspan=1>Total # ofSamplesTested</td><td rowspan=1 colspan=2>Correctly Identified asDual Infection</td><td rowspan=1 colspan=2>Correctly Identified aSingle InfectionOnly</td><td rowspan=1 colspan=2>Identified InfluenzaType Only</td></tr><tr><td rowspan=1 colspan=1>#</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>#</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>#</td><td rowspan=1 colspan=1>%</td></tr><tr><td rowspan=3 colspan=2>Equal Viral Load</td><td rowspan=1 colspan=1>High + High</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>50%</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>50%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td></tr><tr><td rowspan=1 colspan=1>Medium + Medium</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>80%</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>20%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td></tr><tr><td rowspan=1 colspan=1>Low + Low</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>60%</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>30%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>10%</td></tr><tr><td rowspan=1 colspan=3>Equal Viral Loads Summary</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>19</td><td rowspan=1 colspan=1>63%</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>33%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>3%</td></tr><tr><td rowspan=3 colspan=2>Unequal Viral Load</td><td rowspan=1 colspan=1>High + Medium</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>45%</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>55%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Medium + Low</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>10%</td><td rowspan=1 colspan=1>18</td><td rowspan=1 colspan=1>90%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td></tr><tr><td rowspan=1 colspan=1>High + Low</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td></tr><tr><td rowspan=1 colspan=3>Different Viral Loads Summary</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>18%</td><td rowspan=1 colspan=1>49</td><td rowspan=1 colspan=1>82%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td></tr></table>

This analysis showed that samples for which both viruses were at the same concentration had a $6 3 \%$ rate of correct dual infection identification. This was significantly higher than the $1 8 \%$ rate of correct dual identification when samples contained viruses at two different concentrations. Regarding the samples that contained two viruses at different relative concentrations, $82 \%$ of samples resulted in a correctly identified single infection result by the FC8G assay only.

An additional analysis of samples with different viral concentrations that resulted in a correctly identified single infection only by the FC8G assay is presented in Table 15 below.

Table 15: Analysis of Samples with Unequal Viral Concentrations that Resulted in a Correctly Identified Single infection Only by the FC8G Assay   

<table><tr><td rowspan=1 colspan=1>Relative Concentrations</td><td rowspan=1 colspan=1># of Single InfectionsIdentified Correctly fromTable 14</td><td rowspan=1 colspan=1>&quot;High&quot; CorrectlyIdentified</td><td rowspan=1 colspan=1>&quot;Medium&quot; CorrectlyIdentified</td><td rowspan=1 colspan=1>&quot;Low&quot; CorrectlyIdentified</td></tr><tr><td rowspan=1 colspan=1>High + Medium</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>11 (100%)</td><td rowspan=1 colspan=1>0 (0%)</td><td rowspan=1 colspan=1>N/A</td></tr><tr><td rowspan=1 colspan=1>Medium + Low</td><td rowspan=1 colspan=1>18</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>18 (100%)</td><td rowspan=1 colspan=1>0 (0%)</td></tr><tr><td rowspan=1 colspan=1>High + Low</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>20 (100%)</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>0 (0%)</td></tr></table>

This analysis showed that when the samples contained two viruses at unequal relative concentrations, the virus at higher concentration was identified correctly $100 \%$ of the time.

Furthermore, an analysis of each co-infection by influenza subtype and lineage is presented in Table 16 below.

Table 16: Analysis of Co-infections by Influenza Subtype and Lineage   

<table><tr><td rowspan=1 colspan=2>Co-Infection by Subtype or Lineage</td><td rowspan=1 colspan=1>Dual InfectionCorrectly Identified</td><td rowspan=1 colspan=1>Correctly IdentifiedSingle Infection Only</td><td rowspan=1 colspan=1>IdentifiedInfluenza TypeOnly</td></tr><tr><td rowspan=4 colspan=1>Same Influenza Type orLineage</td><td rowspan=1 colspan=1>B/Victoria + B/Yamagata</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>A/H1N1pdm09 + A/H3N2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>A/H1N1pdm09 + A/H11N3</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>A/H11N3 + A/H3N2</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Same Influenza Type or Lineage</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>6 (17%)</td><td rowspan=1 colspan=1>29 (81%)</td><td rowspan=1 colspan=1>1 (3%)</td></tr><tr><td rowspan=6 colspan=1>Different InfluenzaTypes or Lineage</td><td rowspan=1 colspan=1>A/H1N1pdm09 + B/Victoria</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>A/H1N1pdm09 + B/Yamagata</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>A/H3N2 + B/Victoria</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>A/H3N2 + B/Yamagata</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>A/H11N3 + B/Victoria</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>A/H11N3 + B/Yamagata</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=2>Different Influenza Type or Lineage</td><td rowspan=1 colspan=1>24 (44%)</td><td rowspan=1 colspan=1>30 (56%)</td><td rowspan=1 colspan=1>0 (0%)</td></tr></table>

This analysis demonstrated that the FC8G assay was less likely to identify a dual infection on samples containing viruses from the same influenza type or lineage $( 1 7 \% )$ compared to a co-infection of samples including one influenza A and one influenza B virus $( 4 4 \% )$ . Further, a co-infection of both influenza B lineages (B/Victoria $^ +$ B/Yamagata), as well as a co-infection of both the seasonal influenza A viruses $\mathrm { ( A / H 1 N 1 p d m 0 9 + A / H 3 N 2 ) }$ resulted in zero dual infections correctly identified. The only dual infections identified in co-infection of the same influenza virus types were influenza A seasonal viruses mixed with an influenza A non-seasonal virus (either A/H1N1pdm09 or A/H3N2 with A/H11N3) in $33 \%$ of such samples tested.

# h. Potentially Interfering Substances Study

The effects of 17 potentially interfering substances that may be encountered in respiratory specimens were assessed in an analytical study. To evaluate the potential for interfering substances to cause false negative results in influenza positive samples, one representative virus strain from each of the five target virus groups, A/California/07/2009 (A/H1N1pdm09), A/Perth/16/2009 (A/H3N2), B/Florida/02/2006 (B/Victoria), B/Wisconsin/01/2010 (B/Yamagata), and A/Chicken/NY/SG-00425/2004 (A/H2N2, non-seasonal influenza A), was analyzed in triplicate at approximately 5x the LoD in the presence of the interferent. Samples were prepared by spiking each potentially interfering substance into a contrived specimen of pooled influenza negative clinical material containing one of the virus strains. To evaluate the potential for false positive results due to the presence of potentially interfering substances in influenza negative samples, representative influenza negative samples were analyzed in triplicate in the presence of each substance.

If a substance was found to interfere in an influenza positive sample (i.e., caused a false negative result), samples were prepared with the interferent at a lower

concentration and retested until a concentration of interferent was tested that resulted in the correct/expected result for all three replicates. If a substance was found to interfere in an influenza negative sample (i.e., caused a false positive result), the existing extracted sample was re-analyzed in triplicate to assure reproducible results. Any substance that resulted in consistent false positive or false negative results are reported as an interfering substance with the FC8G assay.

The results of the study are summarized in Table 17 below.

Table 17: Summary Results of the Interfering Substances Study   

<table><tr><td rowspan=2 colspan=1>Substance</td><td rowspan=2 colspan=1>Active Interferent</td><td rowspan=2 colspan=1>ConcentrationTested</td><td rowspan=1 colspan=6># of Correct FC8G Assay Results</td></tr><tr><td rowspan=1 colspan=1>A/H1N1pdm09</td><td rowspan=1 colspan=1>A/H3N2</td><td rowspan=1 colspan=1>B/Victoria</td><td rowspan=1 colspan=1>B/Yamagata</td><td rowspan=1 colspan=1>A/H2N2</td><td rowspan=1 colspan=1>InfluenzaNegativeSamples</td></tr><tr><td rowspan=1 colspan=1>Mucin</td><td rowspan=1 colspan=1>Mucin</td><td rowspan=1 colspan=1>0.1 mg/mL</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Blood</td><td rowspan=1 colspan=1>Blood</td><td rowspan=1 colspan=1>0.25% v/v</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Chloraseptic CoughDrops</td><td rowspan=1 colspan=1>Benzocaine,Menthol</td><td rowspan=1 colspan=1>1.0 mg/mL(w/v of drop)</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Halls Cough Drops</td><td rowspan=1 colspan=1>Menthol</td><td rowspan=1 colspan=1>1.0 mg/mL(w/v of drop)</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Neo-Synephrinenasal spray</td><td rowspan=1 colspan=1>Phenylephrine</td><td rowspan=1 colspan=1>10% v/v</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=2 colspan=1>Anefrin nasal spray</td><td rowspan=2 colspan=1>Oxymetaxoline</td><td rowspan=1 colspan=1>10% v/v</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>1/3a</td><td rowspan=1 colspan=1>0/3a</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>5% v/v</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td></tr><tr><td rowspan=1 colspan=1>Saline Nasal Spray</td><td rowspan=1 colspan=1>Sodium Chloride</td><td rowspan=1 colspan=1>10% v/v</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Tobramycin</td><td rowspan=1 colspan=1>Tobramycin</td><td rowspan=1 colspan=1>0.2 mg/mL</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Mupirocin</td><td rowspan=1 colspan=1>Mupirocin</td><td rowspan=1 colspan=1>0.1 mg/mL</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Beclomethasone</td><td rowspan=1 colspan=1>Beclomethasone</td><td rowspan=1 colspan=1>16 µg/ml</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Dexamethasone</td><td rowspan=1 colspan=1>Dexamethasone</td><td rowspan=1 colspan=1>0.1 mg/ml</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Allergy Nasal Spray</td><td rowspan=1 colspan=1>Triamcinolone</td><td rowspan=1 colspan=1>1 % v/v</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Mometasone</td><td rowspan=1 colspan=1>Mometasone</td><td rowspan=1 colspan=1>5 μg/mL</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Budesonide NasalSpray</td><td rowspan=1 colspan=1>Budesonide</td><td rowspan=1 colspan=1>25 μg/mL</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>2/3b</td></tr><tr><td rowspan=1 colspan=1>Zicam</td><td rowspan=1 colspan=1>Galphimia Glauca,Luffa Operculata,Sabadilla</td><td rowspan=1 colspan=1>5% v/v</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Zanamivir</td><td rowspan=1 colspan=1>Zanamivir</td><td rowspan=1 colspan=1>0.1 mg/mL</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>2/3c</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>FluMist</td><td rowspan=1 colspan=1>FluMist</td><td rowspan=1 colspan=1>0.1% v/v</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>0/3d</td></tr></table>

a Resulted in false negative results. b One false positive result for seasonal A/H3N2, but was not reproducible upon retesting of three additional replicates. c One dual positive call for B/Victoria and A/H1N1pdm09, but the A/H1N1pdm09 false positive result was not reproducible upon retesting of three additional replicates. $^ { \textrm { d } } 3 / 3$ false positive results for seasonal A/H1N1pdm09.

The study results showed that no interference with the FC8G assay was observed in the presence of potentially interfering substances at the concentrations tested in this study with the following exceptions:

Anefrin nasal spray interfered with the FC8G assay by resulting in false negatives on both B/Yamagata and B/Victoria samples at a concentration of $10 \%$ (v/v). The influenza B virus samples were reanalyzed in the presence of $5 \%$ (v/v) Anefrin nasal spray, and were correctly identified in all three replicates. In addition, as expected, FluMist (an attenuated live intranasal vaccine) interfered with the assay by causing false positive results in influenza negative samples. Limitations regarding these observations were developed and included in the assay Instructions for Use (IFU).

# i. Carry-Over Study

Potential sample-to-sample carry-over contamination was investigated by analyzing influenza positive samples at high concentration $( > 1 0 ^ { 6 }$ approximate genome copies/mL) next to pooled clinical negative samples in a single run, and conducting the microarray hybridization portion of the assay in a checkerboard pattern of alternating positives and negatives. Run-to-run carry-over contamination was evaluated by executing a run of the alternating positive/negative samples used for the sample-to-sample carry-over contamination testing followed by a run containing only influenza negatives to assess risk of carry-over. This cycle of testing was repeated three times. An A/H3N2 virus strain A/Uruguay/716/2007 was utilized to prepare positive samples in this testing. The influenza negative sample tested was pooled influenza negative clinical material.

In addition, as an alternative assessment of sample-to-sample contamination during hybridization, a mock hybridization study was performed. Two solutions, one containing blue dye and one without dye, were added to eight FluChip-8G Microarray Slides in an alternating, checkboard pattern (i.e., 64 wells with dye, 64 wells without dye). The FluChip-8G Microarray Slides were then placed on orbital shaker at 125- 140 RPM for one hour to mimic the hybridization step. Triplicate absorbance measurements were then taken of the solution in all wells without dye to determine whether any dye contaminated neighboring wells.

The FC8G assay results from the carry-over contamination study are in Table 18 below.

Table 18: Carry-over Contamination Study Results   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=3>Number of Correct FC8G Assay Calls</td><td rowspan=2 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Sample</td><td rowspan=1 colspan=1>Round 1</td><td rowspan=1 colspan=1>Round 2</td><td rowspan=1 colspan=1>Round 3</td></tr><tr><td rowspan=2 colspan=1>Sample-to-SampleContamination</td><td rowspan=1 colspan=1>A/H3N2</td><td rowspan=1 colspan=1>14/14</td><td rowspan=1 colspan=1>13/14*</td><td rowspan=1 colspan=1>12/14*</td><td rowspan=1 colspan=1>39/39 (100%)</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>14/14</td><td rowspan=1 colspan=1>14/14</td><td rowspan=1 colspan=1>13/14**</td><td rowspan=1 colspan=1>41/42 (98%)</td></tr><tr><td rowspan=1 colspan=1>Run-to-Run Carry-over</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>14/15**</td><td rowspan=1 colspan=1>15/15</td><td rowspan=1 colspan=1>15/15</td><td rowspan=1 colspan=1>44/45 (98%)</td></tr></table>

\* Three A/H3N2 samples resulted in "image processing failure," and were removed from final analysis. \*\* Resulted in 1 FP for A/H1N1pdm09.

Three out of $4 2 \mathrm { \ A } / \mathrm { H } 3 \mathrm { N } 2$ samples analyzed during the study resulted in image processing failures and were removed from analysis. All other 39 high positive H3N2 samples resulted in the expected result. While two pooled clinical negative samples did produce a false positive result for A/H1N1pdm09, follow up analysis of these

RNA extracts using an FDA-cleared influenza molecular assay resulted in no influenza detected indicating no evidence of cross-contamination during extraction. It is expected that these two incidents are not systematic, as this rate of false positivity is consistent with assay performance in other studies. The possibility does exist, however, that this was due to cross-contamination during the microarray processing steps.

The mock hybridization analysis resulted in none of the wells having absorbance measurements that were significantly above background, indicating no crosscontamination event during microarray hybridization.

There was no evidence of systematic sample-to-sample carryover or run-to-run cross contamination. However, any open platform assay based on PCR and endpoint detection has the potential for carry-over and/or cross-contamination if careful measures to avoid contamination are not taken. Therefore, the FC8G assay IFU includes the warnings and limitations language below which stresses the importance of good technique to obtain proper results:

“Limit use of this product to personnel trained in the techniques of PCR-based molecular diagnostic assays. PCR-based assays require effort by the operator to reduce potential for cross-contamination during all steps of the procedure, including: nucleic acid extraction, transfer of nucleic acid, and RT-PCR reaction preparation. In addition, care should be taken not to introduce cross-contamination during the microarray hybridization step.”

“Laboratory workflow should proceed in a unidirectional manner, starting with nucleic acid extraction and ending with microarray detection. Change gloves whenever contamination is suspected. Use dedicated areas and separate supplies/equipment for pre and post amplification processes.”

# j. Assay cut-off

The threshold value or cutoff for each of the neural networks was selected separately and subsequently to the neural network training and architecture lock down in order to optimize the performance of the neural network over a given set of samples. Initially, the entire training database of samples was used as the set over which performance was optimized. The thresholds were then determined for each neural network as the value which would minimize the total number of errors (false positives plus false negatives) over this dataset.

It was later observed that the dataset used to select the thresholds differed significantly from what would be expected with real clinical data. The training dataset had a much higher percentage of influenza-positive samples than influenza-negative samples. This was good for training the various influenza-positive patterns, but resulted in a higher False Negative/False Positive ratio than would be expected in real-world testing. In addition, the training dataset included many samples processed as part of dilution

series used to determine the limit of detection, and thus had a much higher percentage of influenza-positive samples below the limit of detection than would be expected in real-world data. Identifying these samples as true positive rather than false negative required lower thresholds. These biases in the dataset used to determine the original thresholds biased them toward sensitivity over specificity.

To optimize the thresholds for real-world testing, a set of 910 training naïve clinical samples, consists of 251 influenza A positive samples, 163 influenza B positive samples, and 496 influenza negative samples, was compiled from an existing in-house archive comprised of material received from various vendors. The clinical samples were processed by the FC8G assay and used to revise the thresholds. These samples were of unknown concentration, and thus considered to be clinically-representative of both influenza concentrations and human background material. This set of 910 samples was entirely independent from the clinical study data presented in the Clinical Studies section. A Receiver Operating Characteristic (ROC) method was used to optimize the thresholds.

The revised and optimized thresholds were further validated during the prospective clinical study conducted between June 2017 and July 2018.

2. Comparison studies:

a. Method comparison with predicate device: Not applicable.   
b. Matrix comparison : Not applicable.

3. Clinical Studies :

# Prospective Clinical Study

Clinical performance characteristics of the FC8G assay were evaluated in a multi-center clinical study between June 2017 and July 2018 in the U.S.

Nasal swab (NS) or nasopharyngeal swab (NPS) specimens were prospectively collected between 2013-2017 from consented patients meeting a febrile respiratory illness (FRI) case definition. Patients were enrolled at seven geographically distributed clinical sites across the United States under a standing influenza surveillance protocol. Upon specimen collection, the nasal swab and nasopharyngeal swab specimens were stabilized in Copan Universal Transport Media (UTM) and shipped frozen to a reference laboratory as part of this ongoing influenza surveillance effort. Remaining volume of each specimen was archived and stored at $\le - 7 0 ^ { \circ } \mathrm { C }$ at the reference laboratory.

To assess the clinical performance of the FC8G assay, the prospectively collected archived specimens were thawed and tested with an FDA-cleared influenza molecular assay (the comparator assay) by the reference laboratory, and then blinded and shipped to one of the three participating testing sites. Each clinical study testing site performed the FC8G assay on approximately one third of the blinded specimens. After all specimens were tested with the FC8G assay, the specimens were un-blinded and the FC8G assay result was compared to the comparator assay result for each specimen. Specimens with discordant results were reflex tested using bi-directional sequencing of the hemagglutinin (HA) and neuraminidase (NA) genes.

Of the 1065 subjects enrolled in this study, there were seven specimens excluded from the performance analyses due to deviation from the study protocol, leaving a total of 1058 prospectively collected swab specimens to be included in the evaluation of the assay performance. There were an additional 36 specimens that did not generate a final result with the FC8G assay due to insufficient sample volume as a result of an external batch control failure (user error) during initial or retesting efforts, resulting in a total of 1022 specimens with a final valid FC8G assay result.

Patient age and gender distribution for all the 1022 prospective specimens $7 0 0 \mathrm { N S }$ and 322 NPS specimens) are presented in Table 19 below.

Table 19: Patient Demographics - Prospective Clinical Study   

<table><tr><td rowspan=1 colspan=1>Age Group</td><td rowspan=1 colspan=1>Number</td><td rowspan=1 colspan=1>Percent of Patients</td></tr><tr><td rowspan=1 colspan=1>≤ 5 years</td><td rowspan=1 colspan=1>432</td><td rowspan=1 colspan=1>42.3%</td></tr><tr><td rowspan=1 colspan=1>6 to 21 years</td><td rowspan=1 colspan=1>299</td><td rowspan=1 colspan=1>29.2%</td></tr><tr><td rowspan=1 colspan=1>22 to 59 years</td><td rowspan=1 colspan=1>230</td><td rowspan=1 colspan=1>22.5%</td></tr><tr><td rowspan=1 colspan=1>≥ 60 years</td><td rowspan=1 colspan=1>61</td><td rowspan=1 colspan=1>6.0%</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>1022</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Sex</td><td rowspan=1 colspan=1>Number</td><td rowspan=1 colspan=1>Percent of Patients</td></tr><tr><td rowspan=1 colspan=1>Male</td><td rowspan=1 colspan=1>467</td><td rowspan=1 colspan=1>45.7%</td></tr><tr><td rowspan=1 colspan=1>Female</td><td rowspan=1 colspan=1>552</td><td rowspan=1 colspan=1>54.0%</td></tr><tr><td rowspan=1 colspan=1>Not Reported</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0.3%</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>1022</td><td rowspan=1 colspan=1>100%</td></tr></table>

Of the total of 1022 samples with a final valid FC8G assay result, 26 specimens resulted in a failure of the FC8G assay internal control to amplify above the threshold value, i.e., “No Call: Assay Failure (Internal Control)”, and were excluded from the performance analyses. In addition, there were 12 specimens that resulted in a comparator assay result of “inconclusive” due to internal control failures, and were also excluded from the performance analyses.

The performance of the FC8G assay testing the remaining 984 specimens (664 NS and 320 NPS specimens) against the comparator assay is presented in Table 20 below.

Table 20: FC8G Assay Performance Testing NS and NPS Specimens against the Comparator Assay – Prospective Clinical Study   

<table><tr><td rowspan=2 colspan=1>Virus Group (Analyte)</td><td rowspan=1 colspan=3>Sensitivity</td><td rowspan=1 colspan=3>Specificity</td></tr><tr><td rowspan=1 colspan=1>TP/(TP+FN)</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=1>TN/(TN+FP)</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=1>Influenza A (Overall)</td><td rowspan=1 colspan=1>51/58</td><td rowspan=1 colspan=1>87.9%</td><td rowspan=1 colspan=1>77.1 - 94.0</td><td rowspan=1 colspan=1>915/926</td><td rowspan=1 colspan=1>98.8%</td><td rowspan=1 colspan=1>97.9- 99.3</td></tr><tr><td rowspan=1 colspan=1>Influenza A/H1N1pdm09</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>43.9 - 100.0</td><td rowspan=1 colspan=1>975/981</td><td rowspan=1 colspan=1>99.4%</td><td rowspan=1 colspan=1>98.7 - 99.7</td></tr><tr><td rowspan=1 colspan=1>Influenza A/H3N2</td><td rowspan=1 colspan=1>48/55</td><td rowspan=1 colspan=1>87.2%</td><td rowspan=1 colspan=1>76.0 - 93.7</td><td rowspan=1 colspan=1>924/929</td><td rowspan=1 colspan=1>99.5%</td><td rowspan=1 colspan=1>98.7 99.8</td></tr><tr><td rowspan=1 colspan=1>Influenza A/Non-seasonal</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>--</td><td rowspan=1 colspan=1>--</td><td rowspan=1 colspan=1>983/984 *</td><td rowspan=1 colspan=1>99.9%</td><td rowspan=1 colspan=1>99.4 - 100.0</td></tr><tr><td rowspan=1 colspan=1>Influenza B(Overall)</td><td rowspan=1 colspan=1>22/22</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>85.1 - 100.0</td><td rowspan=1 colspan=1>960/962</td><td rowspan=1 colspan=1>99.8%</td><td rowspan=1 colspan=1>99.2 - 99.9</td></tr><tr><td rowspan=1 colspan=1>Influenza B/Victoria</td><td rowspan=1 colspan=1>14/14</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>78.5 - 100.0</td><td rowspan=1 colspan=1>970/970</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>99.6 - 100.0</td></tr><tr><td rowspan=1 colspan=1>Influenza B/Yamagata</td><td rowspan=1 colspan=1>8/8</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>67.6 - 100.0</td><td rowspan=1 colspan=1>976/976</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>99.6 - 100.0</td></tr></table>

\*One dual infection was detected by the FC8G assay, resulting in both an influenza A/H3N2 call and influenza A/nonseasonal call. Flu A/H3N2 only was detected by the comparator. This was the only FC8G assay false positive for nonseasonal influenza A observed during the study.

The performance of the FC8G assay against the comparator assay stratified by patient age groups are presented in Table 21 below.

Table 21: FC8G Assay Performance Testing NS and NPS Specimens against the Comparator Assay Stratified by Patient Age Groups – Prospective Clinical Study   

<table><tr><td colspan="1" rowspan="1">Influenza VirusGroup</td><td colspan="2" rowspan="1">≤ 5 Years of Age (n=417)</td><td colspan="2" rowspan="1">6 to 21 Years of Age (n=278)</td><td colspan="2" rowspan="1">≥ 22 Years of Age (n=289)</td></tr><tr><td colspan="1" rowspan="4">Influenza A(Overall)</td><td colspan="1" rowspan="1">Sensitivity95% CI</td><td colspan="1" rowspan="1">Specificity95% CI</td><td colspan="1" rowspan="1">Sensitivity95% CI</td><td colspan="1" rowspan="1">Specificity5% CI</td><td colspan="1" rowspan="1">Sensitivity95% CI</td><td colspan="1" rowspan="1">Specificity95% CI</td></tr><tr><td colspan="1" rowspan="1">90.0%</td><td colspan="1" rowspan="1">98.3%</td><td colspan="1" rowspan="1">93.3%</td><td colspan="1" rowspan="1">98.4%</td><td colspan="1" rowspan="1">77.8%</td><td colspan="1" rowspan="1">100.0%</td></tr><tr><td colspan="1" rowspan="1">(9/10)</td><td colspan="1" rowspan="1">(400/407)</td><td colspan="1" rowspan="1">(28/30)</td><td colspan="1" rowspan="1">(244/248)</td><td colspan="1" rowspan="1">(14/18)</td><td colspan="1" rowspan="1">(271/271)</td></tr><tr><td colspan="1" rowspan="1">59.6% - 98.2%</td><td colspan="1" rowspan="1">96.5% -99.2%</td><td colspan="1" rowspan="1">78.7% - 98.2%</td><td colspan="1" rowspan="1">95.9% - 99.4%</td><td colspan="1" rowspan="1">54.8% - 91.0%</td><td colspan="1" rowspan="1">98.6% - 100.0%</td></tr><tr><td colspan="1" rowspan="4">InfluenzaA/H1N1pdm09</td><td colspan="1" rowspan="1">Sensitivity95% CI</td><td colspan="1" rowspan="1">Specificity95% CI</td><td colspan="1" rowspan="1">Sensitivity95% CI</td><td colspan="1" rowspan="1">Specificity95% CI</td><td colspan="1" rowspan="1">Sensitivity95% CI</td><td colspan="1" rowspan="1">Specificity95% CI</td></tr><tr><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">99.0%</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">99.3%</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">100.0%</td></tr><tr><td colspan="1" rowspan="1">(0/0)</td><td colspan="1" rowspan="1">(413/417)</td><td colspan="1" rowspan="1">(1/1)</td><td colspan="1" rowspan="1">(275/277)</td><td colspan="1" rowspan="1">(2/2)</td><td colspan="1" rowspan="1">(287/287)</td></tr><tr><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">97.6% - 99.6%</td><td colspan="1" rowspan="1">20.7% - 100.0%</td><td colspan="1" rowspan="1">97.4% - 99.8%</td><td colspan="1" rowspan="1">34.2% -100.0%</td><td colspan="1" rowspan="1">98.7% - 100.0%</td></tr><tr><td colspan="1" rowspan="4">InfluenzaA/H3N2</td><td colspan="1" rowspan="1">Sensitivity95% CI</td><td colspan="1" rowspan="1">Specificity95% CI</td><td colspan="1" rowspan="1">Sensitivity95% CI</td><td colspan="1" rowspan="1">Specificity95% CI</td><td colspan="1" rowspan="1">Sensitivity95% CI</td><td colspan="1" rowspan="1">Specificity95% CI</td></tr><tr><td colspan="1" rowspan="1">90.0%</td><td colspan="1" rowspan="1">99.3%</td><td colspan="1" rowspan="1">93.1%</td><td colspan="1" rowspan="1">99.2%</td><td colspan="1" rowspan="1">75.0%</td><td colspan="1" rowspan="1">100.0%</td></tr><tr><td colspan="1" rowspan="1">(9/10)</td><td colspan="1" rowspan="1">(404/407)</td><td colspan="1" rowspan="1">(27/29)</td><td colspan="1" rowspan="1">(247/249)</td><td colspan="1" rowspan="1">(12/16)</td><td colspan="1" rowspan="1">(273/273)</td></tr><tr><td colspan="1" rowspan="1">59.6% - 98.2%</td><td colspan="1" rowspan="1">97.9% - 99.7%</td><td colspan="1" rowspan="1">78.0% - 98.1%</td><td colspan="1" rowspan="1">97.1% - 99.8%</td><td colspan="1" rowspan="1">50.5% - 89.8%</td><td colspan="1" rowspan="1">98.6% - 100.0%</td></tr><tr><td colspan="1" rowspan="4">InfluenzaA/Non-seasonal</td><td colspan="1" rowspan="1">Sensitivity95% CI</td><td colspan="1" rowspan="1">Specificity95% CI</td><td colspan="1" rowspan="1">Sensitivity95% CI</td><td colspan="1" rowspan="1">Specificity95% CI</td><td colspan="1" rowspan="1">Sensitivity95% CI</td><td colspan="1" rowspan="1">Specificity55% CI</td></tr><tr><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">99.6%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">100.0%</td></tr><tr><td colspan="1" rowspan="1">(0/0)</td><td colspan="1" rowspan="1">(417/417)</td><td colspan="1" rowspan="1">(0/0)</td><td colspan="1" rowspan="1">(277/278)</td><td colspan="1" rowspan="1">(0/0)</td><td colspan="1" rowspan="1">(289/289)</td></tr><tr><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">99.1% - 100.0%</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">98.0% - 99.9%</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">98.7% - 100.0%</td></tr><tr><td colspan="1" rowspan="4">Influenza B(Overall)</td><td colspan="1" rowspan="1">Sensitivity95% CI</td><td colspan="1" rowspan="1">Specificity95% CI</td><td colspan="1" rowspan="1">Sensitivity95% CI</td><td colspan="1" rowspan="1">Specificity95% CI</td><td colspan="1" rowspan="1">Sensitivity95% CI</td><td colspan="1" rowspan="1">Specificity95% CI</td></tr><tr><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">99.3%</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">100.0%</td></tr><tr><td colspan="1" rowspan="1">(7/7)</td><td colspan="1" rowspan="1">(827/827)</td><td colspan="1" rowspan="1">(10/10)</td><td colspan="1" rowspan="1">(266/268)</td><td colspan="1" rowspan="1">(5/5)</td><td colspan="1" rowspan="1">(284/284)</td></tr><tr><td colspan="1" rowspan="1">64.6% - 100.0%</td><td colspan="1" rowspan="1">99.5% - 100.0%</td><td colspan="1" rowspan="1">72.2% - 100.0%</td><td colspan="1" rowspan="1">97.3% - 99.8%</td><td colspan="1" rowspan="1">56.6% - 100.0%</td><td colspan="1" rowspan="1">98.7% - 100.0%</td></tr><tr><td colspan="1" rowspan="4">InfluenzaB/Victoria</td><td colspan="1" rowspan="1">Sensitivity95% CI</td><td colspan="1" rowspan="1">Specificity95% CI</td><td colspan="1" rowspan="1">Sensitivity95% CI</td><td colspan="1" rowspan="1">Specificity95% CI</td><td colspan="1" rowspan="1">Sensitivity95% CI</td><td colspan="1" rowspan="1">Specificity95% CI</td></tr><tr><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">100.0%</td></tr><tr><td colspan="1" rowspan="1">(6/6)</td><td colspan="1" rowspan="1">(411/411)</td><td colspan="1" rowspan="1">(6/6)</td><td colspan="1" rowspan="1">(272/272)</td><td colspan="1" rowspan="1">(2/2)</td><td colspan="1" rowspan="1">(287/287)</td></tr><tr><td colspan="1" rowspan="1">61.0% - 100.0%</td><td colspan="1" rowspan="1">99.1% - 100.0%</td><td colspan="1" rowspan="1">61.0% - 100.0%</td><td colspan="1" rowspan="1">98.6% - 100.0%</td><td colspan="1" rowspan="1">34.2% - 100.0%</td><td colspan="1" rowspan="1">98.7% - 100.0%</td></tr><tr><td colspan="1" rowspan="4">InfluenzaB/Yamagata</td><td colspan="1" rowspan="1">Sensitivity95% CI</td><td colspan="1" rowspan="1">Specificity95% CI</td><td colspan="1" rowspan="1">Sensitivity95% CI</td><td colspan="1" rowspan="1">Specificity95% CI</td><td colspan="1" rowspan="1">Sensitivity95% CI</td><td colspan="1" rowspan="1">Specificity95% CI</td></tr><tr><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">100.0%</td></tr><tr><td colspan="1" rowspan="1">(1/1)</td><td colspan="1" rowspan="1">(416/416)</td><td colspan="1" rowspan="1">(4/4)</td><td colspan="1" rowspan="1">(274/274)</td><td colspan="1" rowspan="1">(3/3)</td><td colspan="1" rowspan="1">(286/286)</td></tr><tr><td colspan="1" rowspan="1">20.7% - 100.0%</td><td colspan="1" rowspan="1">99.1% - 100.0%</td><td colspan="1" rowspan="1">51.0% - 100.0%</td><td colspan="1" rowspan="1">98.6% - 100.0%</td><td colspan="1" rowspan="1">56.6% - 100.0%</td><td colspan="1" rowspan="1">98.7% - 100.0%</td></tr></table>

Compared to the comparator assay, the performance of the FC8G assay in this prospective clinical study was also analyzed by testing site, and by specimen type (NS vs. NPS), demonstrating similar performance across the three testing sites and across the two specimen types.

The invalid rate observed during this clinical study was $5 . 9 \%$ (62/1058), $9 5 \%$ CI: $4 . 6 \%$ - $7 . 4 \%$ . This rate includes 26 specimens for which the internal control did not produce a result (a “no call” invalid result), and 36 specimens for which inadequate specimen volume existed to generate a final valid result due to an external batch control failure (user error) either during initial or recommended retesting.

One dual infection was detected by the FC8G assay during this study, resulting in both a Flu A/H3N2 call and a Flu A/non-seasonal call by the FC8G assay. A Flu A/H3 result was generated by the comparator assay. This specimen was retested in accordance with the follow-up recommendations in the FC8G assay instructions for use. Upon retest, this specimen produced the same dual infection result. Reflexive bi-directional sequencing for this specimen produced a result of Influenza A/H3N2, matching the comparator assay result.

# Supplemental Clinical Study

As the prevalence of influenza in the prospective clinical study was expected to be lower than needed to obtain sufficient numbers of influenza positive specimens to establish FC8G assay performance, archived nasopharyngeal swab (NPS) specimens that were prospectively collected from 500 subjects from November 16, 2014 to December 13, 2014 and from January 28, 2016 to April 6, 2016, were acquired from a vendor to be enrolled in a supplemental clinical study. These NPS specimens represented all specimens archived during a defined period that met the same inclusion criteria as the prospective clinical study described previously, and were available in the vendor’s possession with a sufficient volume.

The acquired archived specimens were thawed and tested with an FDA-cleared influenza molecular assay (the comparator assay) by the reference laboratory, and then blinded and shipped to one of the three participating testing sites. Each clinical study testing site performed the FC8G assay on approximately one third of the blinded specimens. After all specimens were tested with the FC8G assay, the specimens were un-blinded and the FC8G assay result was compared to the comparator assay result for each specimen. Specimens with discordant results were reflex tested using bi-directional sequencing of the hemagglutinin (HA) and neuraminidase (NA) genes.

Patient age and gender distribution for all the $5 0 0 \mathrm { N P S }$ specimens are presented in Table 22 below.

Table 22: Patient Demographics – Supplemental Clinical Study   

<table><tr><td rowspan=1 colspan=1>Age Group</td><td rowspan=1 colspan=1>Number</td><td rowspan=1 colspan=1>Percent of Patients</td></tr><tr><td rowspan=1 colspan=1>≤ 5 years</td><td rowspan=1 colspan=1>224</td><td rowspan=1 colspan=1>44.8%</td></tr><tr><td rowspan=1 colspan=1>6 to 21 years</td><td rowspan=1 colspan=1>274</td><td rowspan=1 colspan=1>54.8%</td></tr><tr><td rowspan=1 colspan=1>22 to 59 years</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0.4%</td></tr><tr><td rowspan=1 colspan=1>≥ 60 years</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>500</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Sex</td><td rowspan=1 colspan=1>Number</td><td rowspan=1 colspan=1>Percent of Patients</td></tr><tr><td rowspan=1 colspan=1>Male</td><td rowspan=1 colspan=1>235</td><td rowspan=1 colspan=1>47.0%</td></tr><tr><td rowspan=1 colspan=1>Female</td><td rowspan=1 colspan=1>265</td><td rowspan=1 colspan=1>53.0%</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>500</td><td rowspan=1 colspan=1>100%</td></tr></table>

The performance of the FC8G assay testing the $5 0 0 \mathrm { N P S }$ specimens against the comparator assay is presented in Table 23 below.

Table 23: FC8G Assay Performance Testing NPS Specimens against the Comparator Assay – Supplemental Clinical Study   

<table><tr><td rowspan=2 colspan=1>Virus Group (Analyte)</td><td rowspan=1 colspan=3>Sensitivity</td><td rowspan=1 colspan=3>Specificity</td></tr><tr><td rowspan=1 colspan=1>TP/(TP+FN)</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=1>TN/(TN+FP)</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=1>Influenza A (Overall)*</td><td rowspan=1 colspan=1>189/204</td><td rowspan=1 colspan=1>92.6%</td><td rowspan=1 colspan=1>88.2-95.5</td><td rowspan=1 colspan=1>293/296</td><td rowspan=1 colspan=1>98.9%</td><td rowspan=1 colspan=1>97.1- 99.7</td></tr><tr><td rowspan=1 colspan=1>Influenza A/H1N1pdm09</td><td rowspan=1 colspan=1>24/26</td><td rowspan=1 colspan=1>92.3%</td><td rowspan=1 colspan=1>75.9-97.9</td><td rowspan=1 colspan=1>469/474</td><td rowspan=1 colspan=1>98.9%</td><td rowspan=1 colspan=1>97.6- 99.5</td></tr><tr><td rowspan=1 colspan=1>Influenza A/H3N2</td><td rowspan=1 colspan=1>163/175</td><td rowspan=1 colspan=1>93.1%</td><td rowspan=1 colspan=1>88.4- 96.0</td><td rowspan=1 colspan=1>325/325</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>98.8 - 100.0</td></tr><tr><td rowspan=1 colspan=1>Influenza A/Non-seasonal</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>--</td><td rowspan=1 colspan=1>--</td><td rowspan=1 colspan=1>500/500</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>99.2 - 100.0</td></tr><tr><td rowspan=1 colspan=1>Influenza B (Overall)</td><td rowspan=1 colspan=1>52/53</td><td rowspan=1 colspan=1>98.1%</td><td rowspan=1 colspan=1>90.1 - 99.7</td><td rowspan=1 colspan=1>444/447</td><td rowspan=1 colspan=1>99.3%</td><td rowspan=1 colspan=1>98.0 - 99.8</td></tr><tr><td rowspan=1 colspan=1>Influenza B/Victoria</td><td rowspan=1 colspan=1>46/46</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>92.3 - 100.0</td><td rowspan=1 colspan=1>452/454</td><td rowspan=1 colspan=1>99.6%</td><td rowspan=1 colspan=1>98.4- 99.9</td></tr><tr><td rowspan=1 colspan=1>Influenza B/Yamagata</td><td rowspan=1 colspan=1>6/7</td><td rowspan=1 colspan=1>85.7%</td><td rowspan=1 colspan=1>48.7 - 97.4</td><td rowspan=1 colspan=1>492/493</td><td rowspan=1 colspan=1>99.8%</td><td rowspan=1 colspan=1>98.9 - 100.0</td></tr></table>

\* Influenza A (Overall) category includes two specimens that resulted in a FC8G assay result of “Influenza AH1N1pdm09”. The comparator assay generated a result of “Influenza A (Equivocal), No subtyping”. Both samples were correctly identified as influenza A by the FC8G assay, therefore, counted as true positives in the Influenza A (Overall) category, but were both counted as false positives for the Influenza A/H1N1pdm09 subtype.

The performance of the FC8G assay against the comparator assay in the supplemental clinical study stratified by patient age groups are presented in Table 24 below.

Table 24: FC8G Assay Performance Testing NS Specimens against the Comparator Assay Stratified by Patient Age Groups – Supplemental Clinical Study   

<table><tr><td rowspan=1 colspan=1>Influenza VirusGroup</td><td rowspan=1 colspan=2>≤ 5 Years of Age (n=224)</td><td rowspan=1 colspan=2>6 to 21 Years of Age (n=274)</td><td rowspan=1 colspan=2>≥ 22 Years of Age (n=2)</td></tr><tr><td rowspan=4 colspan=1>Influenza A(Overall)</td><td rowspan=1 colspan=1>Sensitivity95% CI</td><td rowspan=1 colspan=1>Specificity95% CI</td><td rowspan=1 colspan=1>Sensitivity95% CI</td><td rowspan=1 colspan=1>Specificity95% CI</td><td rowspan=1 colspan=1>Sensitivity95% CI</td><td rowspan=1 colspan=1>Specificity95% CI</td></tr><tr><td rowspan=1 colspan=1>92.3%</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>92.8%</td><td rowspan=1 colspan=1>97.8%</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>100.0%</td></tr><tr><td rowspan=1 colspan=1>(60/65)</td><td rowspan=1 colspan=1>(159/159)</td><td rowspan=1 colspan=1>(129/139)</td><td rowspan=1 colspan=1>(132/135)</td><td rowspan=1 colspan=1>(0/0)</td><td rowspan=1 colspan=1>(2/2)</td></tr><tr><td rowspan=1 colspan=1>83.2% - 96.7%</td><td rowspan=1 colspan=1>97.6% - 100.0%</td><td rowspan=1 colspan=1>87.3% - 96.0%</td><td rowspan=1 colspan=1>93.7% - 99.2%</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>34.2% - 100.0%</td></tr><tr><td rowspan=4 colspan=1>InfluenzaA/H1N1pdm09</td><td rowspan=1 colspan=1>Sensitivity95% CI</td><td rowspan=1 colspan=1>Specificity95 CI</td><td rowspan=1 colspan=1>Sensitivity95% CI</td><td rowspan=1 colspan=1>Specificity95% CI</td><td rowspan=1 colspan=1>Sensitivity95% CI</td><td rowspan=1 colspan=1>Specificity95% CI</td></tr><tr><td rowspan=1 colspan=1>81.8%</td><td rowspan=1 colspan=1>99.5%</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>98.5%</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>100.0%</td></tr><tr><td rowspan=1 colspan=1>(9/11)</td><td rowspan=1 colspan=1>(212/213)</td><td rowspan=1 colspan=1>(15/15)</td><td rowspan=1 colspan=1>(255/259)</td><td rowspan=1 colspan=1>(0/0)</td><td rowspan=1 colspan=1>(2/2)</td></tr><tr><td rowspan=1 colspan=1>52.3% - 94.9%</td><td rowspan=1 colspan=1>97.4% - 99.9%</td><td rowspan=1 colspan=1>79.6% - 100.0%</td><td rowspan=1 colspan=1>96.1% - 99.4%</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>34.2% - 100.0%</td></tr><tr><td rowspan=4 colspan=1>InfluenzaA/H3N2</td><td rowspan=1 colspan=1>Sensitivity95% CI</td><td rowspan=1 colspan=1>Specificity95% CI</td><td rowspan=1 colspan=1>Sensitivity95% CI</td><td rowspan=1 colspan=1>Specificity95% CI</td><td rowspan=1 colspan=1>Sensitivity95% CI</td><td rowspan=1 colspan=1>Specificity95% CI</td></tr><tr><td rowspan=1 colspan=1>94.2%</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>92.6%</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>0.0%</td></tr><tr><td rowspan=1 colspan=1>(49/52)</td><td rowspan=1 colspan=1>(172/172)</td><td rowspan=1 colspan=1>(112/121)</td><td rowspan=1 colspan=1>(153/153)</td><td rowspan=1 colspan=1>(2/2)</td><td rowspan=1 colspan=1>(0/0)</td></tr><tr><td rowspan=1 colspan=1>84.4% - 98.0%</td><td rowspan=1 colspan=1>97.8% - 100.0%</td><td rowspan=1 colspan=1>86.5% - 96.0%</td><td rowspan=1 colspan=1>97.6% - 100.0%</td><td rowspan=1 colspan=1>34.2% - 100.0%</td><td rowspan=1 colspan=1>N/A</td></tr><tr><td rowspan=4 colspan=1>InfluenzaA/Non-seasonal</td><td rowspan=1 colspan=1>Sensitivity95% CI</td><td rowspan=1 colspan=1>Specificity95% CI</td><td rowspan=1 colspan=1>Sensitivity95% CI</td><td rowspan=1 colspan=1>Specificity95% CI</td><td rowspan=1 colspan=1>Sensitivity95% CI</td><td rowspan=1 colspan=1>Specificity95% CI</td></tr><tr><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>100.0%</td></tr><tr><td rowspan=1 colspan=1>(0/0)</td><td rowspan=1 colspan=1>(224/224)</td><td rowspan=1 colspan=1>(0/0)</td><td rowspan=1 colspan=1>(274/274)</td><td rowspan=1 colspan=1>(0/0)</td><td rowspan=1 colspan=1>(2/2)</td></tr><tr><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>98.3% - 100.0%</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>98.6% - 100.0%</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>34.2% - 100.0%</td></tr><tr><td rowspan=4 colspan=1>Influenza B(Overall)</td><td rowspan=1 colspan=1>Sensitivity95% CI</td><td rowspan=1 colspan=1>Specificity9% CI</td><td rowspan=1 colspan=1>Sensitivity95% CI</td><td rowspan=1 colspan=1>Specificity95% CI</td><td rowspan=1 colspan=1>Sensitivity95% CI</td><td rowspan=1 colspan=1>Specificity95% CI</td></tr><tr><td rowspan=1 colspan=1>92.3%</td><td rowspan=1 colspan=1>99.5%</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>99.1%</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>100.0%</td></tr><tr><td rowspan=1 colspan=1>(12/13)</td><td rowspan=1 colspan=1>(210/211)</td><td rowspan=1 colspan=1>(40/40)</td><td rowspan=1 colspan=1>(232/234)</td><td rowspan=1 colspan=1>(0/0)</td><td rowspan=1 colspan=1>(2/2)</td></tr><tr><td rowspan=1 colspan=1>66.7% - 98.6%</td><td rowspan=1 colspan=1>97.4% - 99.9%</td><td rowspan=1 colspan=1>91.2% - 100.0%</td><td rowspan=1 colspan=1>96.9% - 99.8%</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>34.2% - 100.0%</td></tr><tr><td rowspan=4 colspan=1>InfluenzaB/Victoria</td><td rowspan=1 colspan=1>Sensitivity95% CI</td><td rowspan=1 colspan=1>Specificity95% CI</td><td rowspan=1 colspan=1>Sensitivity95% CI</td><td rowspan=1 colspan=1>Specificity95% CI</td><td rowspan=1 colspan=1>Sensitivity95% CI</td><td rowspan=1 colspan=1>Specificity9% CI</td></tr><tr><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>99.5%</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>99.6%</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>100.0%</td></tr><tr><td rowspan=1 colspan=1>(10/10)</td><td rowspan=1 colspan=1>(213/214)</td><td rowspan=1 colspan=1>(36/36)</td><td rowspan=1 colspan=1>(237/238)</td><td rowspan=1 colspan=1>(0/0)</td><td rowspan=1 colspan=1>(2/2)</td></tr><tr><td rowspan=1 colspan=1>72.2% - 100.0%</td><td rowspan=1 colspan=1>97.4% - 99.9%</td><td rowspan=1 colspan=1>90.4% - 100.0%</td><td rowspan=1 colspan=1>97.7% - 99.9%</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>34.2% - 100.0%</td></tr><tr><td rowspan=4 colspan=1>InfluenzaB/Yamagata</td><td rowspan=1 colspan=1>Sensitivity95% CI</td><td rowspan=1 colspan=1>Specificity95% CI</td><td rowspan=1 colspan=1>Sensitivity95% CI</td><td rowspan=1 colspan=1>Specificity95% CI</td><td rowspan=1 colspan=1>Sensitivity95% CI</td><td rowspan=1 colspan=1>Specificity95% CI</td></tr><tr><td rowspan=1 colspan=1>66.7%</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>99.6%</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>100.0%</td></tr><tr><td rowspan=1 colspan=1>(2/3)</td><td rowspan=1 colspan=1>(221/221)</td><td rowspan=1 colspan=1>(4/4)</td><td rowspan=1 colspan=1>(269/270)</td><td rowspan=1 colspan=1>(0/0)</td><td rowspan=1 colspan=1>(2/2)</td></tr><tr><td rowspan=1 colspan=1>20.8% - 93.9%</td><td rowspan=1 colspan=1>98.3% - 100.0%</td><td rowspan=1 colspan=1>51.0% - 100.0%</td><td rowspan=1 colspan=1>97.9% - 99.9%</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>34.2% - 100.0%</td></tr></table>

Compared to the comparator assay, the performance of the FC8G assay in this supplemental clinical study was also analyzed by testing site. The FC8G assay demonstrated similar performance across the three testing sites.

No invalid result or dual infection result was generated by the FC8G assay during this study.

# Retrospective Clinical Studies

A retrospective clinical study was conducted to supplement the prospective performance studies, since there were low numbers of positive specimens for A/H1N1pdm09 and

B/Yamagata in the prospectively collected cohorts. A total of 98 retrospective blinded NPS or NS specimens, collected from 2010 to 2018, were analyzed by the FC8G assay against the same comparator assay utilized in the prospective clinical studies, to increase the number of influenza positive specimens per target virus group. One specimen was removed from the performance analysis due to an internal control failure in the FC8G assay, resulting in 97 specimens being included in the performance analysis.

The performance of the FC8G assay testing the 97 NS or NPS retrospective specimens against the comparator assay is presented in Table 25 below.

Table 25: FC8G Assay Performance Testing NS or NPS Specimens against the Comparator Assay – Retrospective Clinical Study #1   

<table><tr><td rowspan=2 colspan=1>Virus Group (Analyte)</td><td rowspan=1 colspan=3>Sensitivity</td><td rowspan=1 colspan=3>Specificity</td></tr><tr><td rowspan=1 colspan=1>TP/(TP+FN)</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=1>TN/(TN+FP)</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=1>Influenza A (Overall)*</td><td rowspan=1 colspan=1>24/25</td><td rowspan=1 colspan=1>96.0%</td><td rowspan=1 colspan=1>80.5- 99.3</td><td rowspan=1 colspan=1>72/72</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>94.9 - 100.0</td></tr><tr><td rowspan=1 colspan=1>Influenza A/H1N1pdm09</td><td rowspan=1 colspan=1>24/24</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>86.2 - 100.0</td><td rowspan=1 colspan=1>73/73</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>95.0 - 100.0</td></tr><tr><td rowspan=1 colspan=1>Influenza A/H3N2</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>--</td><td rowspan=1 colspan=1>97/97</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>96.2 - 100.0</td></tr><tr><td rowspan=1 colspan=1>Influenza A/Non-seasonal</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>--</td><td rowspan=1 colspan=1>--</td><td rowspan=1 colspan=1>97/97</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>96.2 - 100.0</td></tr><tr><td rowspan=1 colspan=1>Influenza B (Overall)</td><td rowspan=1 colspan=1>63/65</td><td rowspan=1 colspan=1>96.9%</td><td rowspan=1 colspan=1>89.5- 99.2</td><td rowspan=1 colspan=1>32/32</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>89.3 - 100.0</td></tr><tr><td rowspan=1 colspan=1>Influenza B/Victoria</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>--</td><td rowspan=1 colspan=1>--</td><td rowspan=1 colspan=1>97/97</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>96.2 - 100.0</td></tr><tr><td rowspan=1 colspan=1>Influenza B/Yamagata</td><td rowspan=1 colspan=1>62/64</td><td rowspan=1 colspan=1>96.8%</td><td rowspan=1 colspan=1>89.3 - 99.1</td><td rowspan=1 colspan=1>33/33</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>89.6 - 100.0</td></tr></table>

\* Influenza A (Overall) category includes one specimen that resulted in a FC8G assay result of “Influenza B/Yamagata”. The comparator assay generated a result of “Influenza A (Equivocal), No subtyping” and “Influenza B/Yamagata”. This sample was counted as a false negative in the Influenza A (Overall) category, but was counted as a true positive for the Influenza B (Overall) and the Influenza B/Yamagata categories.

An additional retrospective clinical study was also conducted to supplement the prospective performance studies, since there were lower than expected numbers of specimens that were obtained from patients 65 years of age or older in the prospectively collected cohorts. One hundred and ten (110) NPS specimens were sourced retrospectively to increase the number of specimens in the $6 5 +$ age bracket. All these retrospective specimens for the $6 5 +$ age group were collected between 2016 and 2018, with 51 specimens collected from males and 59 collected from females. The specimens were analyzed by the FC8G assay against the same comparator assay utilized in the prospective clinical studies. Two specimens resulted in an assay failure by either the FC8G assay or the comparator assay and were not included in the performance analysis. The performance of the FC8G assay on the 108 retrospective NPS samples with a valid result for the $6 5 +$ age bracket is shown in Table 26 below.

Table 26: FC8G Assay Performance Testing NPS Specimens against the Comparator Assay – Retrospective Clinical Study #2   

<table><tr><td rowspan=2 colspan=1>Virus Group (Analyte)</td><td rowspan=1 colspan=3>Sensitivity</td><td rowspan=1 colspan=3>Specificity</td></tr><tr><td rowspan=1 colspan=1>TP/(TP+FN)</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=1>TN/(TN+FP)</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=1>Influenza A (Overall)</td><td rowspan=1 colspan=1>17/18</td><td rowspan=1 colspan=1>94.4%</td><td rowspan=1 colspan=1>74.2 - 99.0</td><td rowspan=1 colspan=1>90/90</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>95.9-100.0</td></tr><tr><td rowspan=1 colspan=1>Influenza A/H1N1pdm09</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>43.9 - 100.0</td><td rowspan=1 colspan=1>105/105</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>96.5-100.0</td></tr><tr><td rowspan=1 colspan=1>Influenza A/H3N2</td><td rowspan=1 colspan=1>14/15</td><td rowspan=1 colspan=1>93.3%</td><td rowspan=1 colspan=1>70.2- 98.8</td><td rowspan=1 colspan=1>93/93</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>96.0 -100.0</td></tr><tr><td rowspan=1 colspan=1>Influenza A/Non-seasonal</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>--</td><td rowspan=1 colspan=1>--</td><td rowspan=1 colspan=1>108/108</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>96.6-100.0</td></tr><tr><td rowspan=1 colspan=1>Influenza B (Overall)</td><td rowspan=1 colspan=1>11/12</td><td rowspan=1 colspan=1>91.6%</td><td rowspan=1 colspan=1>64.6- 98.5</td><td rowspan=1 colspan=1>96/96</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>96.2 100.0</td></tr><tr><td rowspan=1 colspan=1>Influenza B/Victoria</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>--</td><td rowspan=1 colspan=1>--</td><td rowspan=1 colspan=1>108/108</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>96.6--100.0</td></tr><tr><td rowspan=1 colspan=1>Influenza B/Yamagata</td><td rowspan=1 colspan=1>11/12</td><td rowspan=1 colspan=1>91.6%</td><td rowspan=1 colspan=1>64.6 - 98.5</td><td rowspan=1 colspan=1>96/96</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>96.2 - 100.0</td></tr></table>

# Non-Seasonal Influenza A Virus Challenge Study

Due to the rarity of non-seasonal influenza A infections in humans and therefore the lack of non-seasonal influenza A in the prospectively collected clinical study cohorts or in the retrospective specimens, a FC8G assay non-seasonal influenza A challenge study was executed.

The non-seasonal influenza A challenge study was performed testing 220 blinded surrogate clinical samples representing 41 unique influenza A subtypes, including an equal number of influenza positive and influenza negative samples. One hundred (100) surrogate clinical samples were contrived by spiking each of the 100 unique non-seasonal influenza A strains into unique individual clinical negative NS or NPS collected in VTM (i.e., influenza negative clinical specimen from a single patient). The 100 non-seasonal strains were selected to represent diversity in host-origin and subtype, with a focus on subtypes with pandemic potential. Fifty (50) of the 100 unique non-seasonal influenza A viruses were tested with the FC8G assay at concentrations from approximately $0 . 5 \mathrm { x }$ to 3x LoD, and the other 50 non-seasonal influenza A viruses were analyzed at concentrations greater than $3 \mathbf { x }$ LoD (see Table 27 below). For safety reasons, all H5 and H7 subtypes were utilized as extracted RNA and not intact whole virus. To reduce potential bias, 10 contrived surrogate clinical specimens from seasonal target virus groups, and 110 influenza-negative clinical specimens were also included in the sample set. All specimens included in this study were blinded to the analyst, and all testing occurred at a single site.

Overall, this challenge study included testing 62 non-seasonal influenza A strains representing subtypes that were included in the training set. The challenge study also included testing 38 non-seasonal influenza A strains that were not included in the training set (i.e., naïve to training) as a challenge to the assay. All the 100 non-seasonal influenza A strains tested in this challenge study were contrived in unique individual influenza negative clinical specimens, and therefore, had a unique clinical background compared to other analyses of the FC8G microarray images generated by testing the same strain that may have been included in training.

Table 27: Non-Seasonal Influenza A Virus Strains Included in the Challenge Study   

<table><tr><td colspan="1" rowspan="1">Strain</td><td colspan="1" rowspan="1">Host ofIsolation</td><td colspan="6" rowspan="1">Subtype</td><td colspan="1" rowspan="1">Material Input intoExtraction</td><td colspan="1" rowspan="1">Concentration(approximategenomecopies/mL)</td><td colspan="1" rowspan="1">ApproximateMultiples ofLoD Tested</td></tr><tr><td colspan="1" rowspan="1">A/Canvasback/Alberta/276/2005</td><td colspan="1" rowspan="1">Avian</td><td colspan="6" rowspan="3">A/H1N1</td><td colspan="1" rowspan="1">Whole Virus</td><td colspan="1" rowspan="1">8.3 x 104</td><td colspan="1" rowspan="1">28x</td></tr><tr><td colspan="1" rowspan="1">A/Redheaded Duck/Minnesota/SG-00123/2007</td><td colspan="1" rowspan="1">Avian</td><td colspan="1" rowspan="1">Whole Virus</td><td colspan="1" rowspan="1">6.9 x 103</td><td colspan="1" rowspan="1">2x</td></tr><tr><td colspan="1" rowspan="1">A/Mallard/Republic of Georgia/4/2010</td><td colspan="1" rowspan="1">Avian</td><td colspan="1" rowspan="1">Whole Virus</td><td colspan="1" rowspan="1">2.5 x 105$</td><td colspan="1" rowspan="1">84x</td></tr><tr><td colspan="1" rowspan="1">A/Duck/Alberta/35/1976</td><td colspan="1" rowspan="1">Avian</td><td colspan="6" rowspan="1"></td><td colspan="1" rowspan="1">Viral Genomic RNA</td><td colspan="1" rowspan="1">2.9 x 104</td><td colspan="1" rowspan="1">10x</td></tr><tr><td colspan="1" rowspan="1">A/South Dakota/06/2007</td><td colspan="1" rowspan="1">Human</td><td colspan="6" rowspan="1"></td><td colspan="1" rowspan="1">Whole Virus</td><td colspan="1" rowspan="1">6.4 x 103</td><td colspan="1" rowspan="1">2x</td></tr><tr><td colspan="1" rowspan="1">A/Florida/03/2006</td><td colspan="1" rowspan="1">Human</td><td colspan="6" rowspan="3">A/H1N1 pre-2009</td><td colspan="1" rowspan="1">Whole Virus</td><td colspan="1" rowspan="1">6.2 x 105</td><td colspan="1" rowspan="1">208x</td></tr><tr><td colspan="1" rowspan="1">A/Solomon Islands/3/2006</td><td colspan="1" rowspan="1">Human</td><td colspan="1" rowspan="1">Whole Virus</td><td colspan="1" rowspan="1">3.4 x 103</td><td colspan="1" rowspan="1">1x</td></tr><tr><td colspan="1" rowspan="1">A/Fukushima/141/2006</td><td colspan="1" rowspan="1">Human</td><td colspan="2" rowspan="1"></td><td colspan="1" rowspan="1">Whole Virus</td><td colspan="1" rowspan="1">4.5 x 103</td><td colspan="1" rowspan="1">2x</td></tr><tr><td colspan="1" rowspan="1">A/St. Petersburg/8/2006</td><td colspan="1" rowspan="1">Human</td><td colspan="5" rowspan="1"></td><td colspan="1" rowspan="1"></td><td></td><td colspan="1" rowspan="1">Whole Virus</td><td colspan="1" rowspan="1">5.5 x 104</td></tr><tr><td colspan="1" rowspan="1">A/Swine/Ohio/09SW1477/2009</td><td colspan="1" rowspan="1">Swine</td><td colspan="6" rowspan="2">A/H1N2</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Whole Virus</td><td colspan="1" rowspan="1">6.7 x 104</td></tr><tr><td colspan="1" rowspan="1">A/Swine/Ohio/09SW1484E/2009</td><td colspan="1" rowspan="1">Swine</td><td colspan="1" rowspan="1">Whole Virus</td><td colspan="1" rowspan="1">3.7 x 10^3</td><td colspan="1" rowspan="1">1x</td></tr><tr><td colspan="1" rowspan="1">A/Shorebird/Delaware Bay/211/1994</td><td colspan="1" rowspan="1">Avian</td><td colspan="6" rowspan="1">A/H1N3</td><td colspan="1" rowspan="1">Whole Virus</td><td colspan="1" rowspan="1">2.5 x 105</td><td colspan="1" rowspan="1">84x</td></tr><tr><td colspan="1" rowspan="1">A/Japan/305/1957</td><td colspan="1" rowspan="1">Human</td><td colspan="6" rowspan="1">A/H2N2</td><td colspan="1" rowspan="1">Viral Genomic RNA</td><td colspan="1" rowspan="1">8.9 x 103</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="1">A/Duck/Germany/1215/1973</td><td colspan="1" rowspan="1">Avian</td><td colspan="6" rowspan="1">A/H2N3</td><td colspan="1" rowspan="1">Viral Genomic RNA</td><td colspan="1" rowspan="1">8.7 x 103</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="1">A/Swine/North Carolina/32760/2007</td><td colspan="1" rowspan="1">Swine</td><td colspan="6" rowspan="1"></td><td colspan="1" rowspan="1">Viral Genomic RNA</td><td colspan="1" rowspan="1">6.3 x 103</td><td colspan="1" rowspan="1">2x</td></tr><tr><td colspan="1" rowspan="1">A/Swine/North Carolina/44897/2009</td><td colspan="1" rowspan="1">Swine</td><td colspan="6" rowspan="2">A/H3N2sW</td><td colspan="1" rowspan="1">Viral Genomic RNA</td><td colspan="1" rowspan="1">5.4 x 103</td><td colspan="1" rowspan="1">2x</td></tr><tr><td colspan="1" rowspan="1">A/Swine/North Carolina/52796/2006</td><td colspan="1" rowspan="1">Swine</td><td colspan="1" rowspan="1">Viral Genomic RNA</td><td colspan="1" rowspan="1">4.0 x 103</td><td colspan="1" rowspan="1">1x</td></tr><tr><td colspan="1" rowspan="1">A/Swine/North Carolina/88708/2000</td><td colspan="1" rowspan="1">Swine</td><td colspan="6" rowspan="1"></td><td colspan="1" rowspan="1">Viral Genomic RNA</td><td colspan="1" rowspan="1">3.7 x 103</td><td colspan="1" rowspan="1">1x</td></tr><tr><td colspan="1" rowspan="1">A/Indiana/10/2011</td><td colspan="1" rowspan="1">Avian</td><td colspan="6" rowspan="2"></td><td colspan="1" rowspan="1">Whole Virus</td><td colspan="1" rowspan="1">2.2 x 105</td><td colspan="1" rowspan="1">74x</td></tr><tr><td colspan="1" rowspan="1">A/Michigan/20/2012</td><td colspan="1" rowspan="1">Human</td><td colspan="1" rowspan="1">Viral Genomic RNA</td><td colspan="1" rowspan="1">1.1 x 104</td><td colspan="1" rowspan="1">4x</td></tr><tr><td colspan="1" rowspan="1">A/Ohio/20/2012</td><td colspan="1" rowspan="1">Human</td><td colspan="6" rowspan="1">A/H3N2v</td><td colspan="1" rowspan="1">Viral Genomic RNA</td><td colspan="1" rowspan="1">7.1 x 103</td><td colspan="1" rowspan="1">2x</td></tr><tr><td colspan="1" rowspan="1">A/Ohio/36/2012</td><td colspan="1" rowspan="1">Human</td><td colspan="6" rowspan="1"></td><td colspan="1" rowspan="1">Viral Genomic RNA</td><td colspan="1" rowspan="1">5.7 x 103</td><td colspan="1" rowspan="1">2x</td></tr><tr><td colspan="1" rowspan="1">A/Ohio/44/2012</td><td colspan="1" rowspan="1">Human</td><td colspan="2" rowspan="1"></td><td colspan="4" rowspan="1"></td><td colspan="1" rowspan="1">Viral Genomic RNA</td><td colspan="1" rowspan="1">5.5 x 103</td><td colspan="1" rowspan="1">2x</td></tr><tr><td colspan="1" rowspan="1">A/Blue-Winged Teal/Illinois/10OS1546/2010</td><td colspan="1" rowspan="1">Avian</td><td colspan="1" rowspan="1"></td><td colspan="5" rowspan="2">A/H3N6</td><td colspan="1" rowspan="1">Whole Virus</td><td colspan="1" rowspan="1">1.3 x 105</td><td colspan="1" rowspan="1">42x</td></tr><tr><td colspan="1" rowspan="1">A/Redhead/Alberta/192/2002</td><td colspan="1" rowspan="1">Avian</td><td></td><td colspan="1" rowspan="1">Whole Virus</td><td colspan="1" rowspan="1">4.8 x 103</td><td colspan="1" rowspan="1">2x</td></tr><tr><td colspan="1" rowspan="1">A/Equine/Pennsylvania/1/2007</td><td colspan="1" rowspan="1">Equine</td><td colspan="6" rowspan="3">A/H3N8</td><td colspan="1" rowspan="1">Whole Virus</td><td colspan="1" rowspan="1">1.4 x 103</td><td colspan="1" rowspan="1">0.5x</td></tr><tr><td colspan="1" rowspan="1">A/Equine/Miami/1/1963</td><td colspan="1" rowspan="1">Equine</td><td colspan="1" rowspan="1">Whole Virus</td><td colspan="1" rowspan="1">1.4 x 103</td><td colspan="1" rowspan="1">0.5x</td></tr><tr><td colspan="1" rowspan="1">A/Duck/Chabarovsk/1610/1972</td><td colspan="1" rowspan="1">Avian</td><td colspan="3" rowspan="1"></td><td colspan="1" rowspan="1">Whole Virus</td><td colspan="1" rowspan="1">1.6 x 103</td><td colspan="1" rowspan="1">0.5x</td></tr><tr><td colspan="1" rowspan="1">A/Duck/Ukraine/1963</td><td colspan="1" rowspan="1">Avian</td><td colspan="6" rowspan="1"></td><td colspan="1" rowspan="1">Whole Virus</td><td colspan="1" rowspan="1">4.1 x 105</td><td colspan="1" rowspan="1">137x</td></tr><tr><td colspan="1" rowspan="1">A/Blue-Winged Teal/Alberta/346/2007</td><td colspan="1" rowspan="1">Avian</td><td colspan="6" rowspan="1">A/H4N3</td><td colspan="1" rowspan="1">Whole Virus</td><td colspan="1" rowspan="1">3.4 x 103</td><td colspan="1" rowspan="1">1x</td></tr><tr><td colspan="1" rowspan="1">A/Mallard/Alberta/35/2001</td><td colspan="1" rowspan="1">Avian</td><td colspan="6" rowspan="1"></td><td colspan="1" rowspan="1">Whole Virus</td><td colspan="1" rowspan="1">3.9 x 103</td><td colspan="1" rowspan="1">1x</td></tr><tr><td colspan="1" rowspan="1">A/Shorebird/Delaware/309/2008</td><td colspan="1" rowspan="1">Avian</td><td colspan="6" rowspan="1">A/H4N6</td><td colspan="1" rowspan="1">Whole Virus</td><td colspan="1" rowspan="1">3.3 x 105</td><td colspan="1" rowspan="1">111x</td></tr><tr><td colspan="1" rowspan="1">A/Red Knot/Delaware/541/1988</td><td colspan="1" rowspan="1">Avian</td><td colspan="6" rowspan="1"></td><td colspan="1" rowspan="1">Whole Virus</td><td colspan="1" rowspan="1">1.3 x 105</td><td colspan="1" rowspan="1">42x</td></tr><tr><td colspan="1" rowspan="1">A/Common Magpie/Hong Kong/645/2006</td><td colspan="1" rowspan="1">Avian</td><td colspan="6" rowspan="1"></td><td colspan="1" rowspan="1">Viral Genomic RNA</td><td colspan="1" rowspan="1">3.1 x 104</td><td colspan="1" rowspan="1">11x</td></tr><tr><td colspan="1" rowspan="1">A/Vietnam/1194/2004</td><td colspan="1" rowspan="1">Avian</td><td colspan="6" rowspan="1"></td><td colspan="1" rowspan="1">Viral Genomic RNA</td><td colspan="1" rowspan="1">3.1 x 104</td><td colspan="1" rowspan="1">11x</td></tr><tr><td colspan="1" rowspan="1">A/Chicken/Egypt/M7217B/2013</td><td colspan="1" rowspan="1">Avian</td><td colspan="6" rowspan="1"></td><td colspan="1" rowspan="1">Viral Genomic RNA</td><td colspan="1" rowspan="1">3.8 x 103</td><td colspan="1" rowspan="1">1x</td></tr><tr><td colspan="1" rowspan="1">A/Chicken/Egypt/Q1089E/2010</td><td colspan="1" rowspan="1">Avian</td><td colspan="6" rowspan="2">A/H5N1</td><td colspan="1" rowspan="1">Viral Genomic RNA</td><td colspan="1" rowspan="1">8.6 x 103</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="1">A/Cambodia/X0810301/13</td><td colspan="1" rowspan="1">Human</td><td colspan="1" rowspan="1">Viral Genomic RNA</td><td colspan="1" rowspan="1">4.2 x 103</td><td colspan="1" rowspan="1">1x</td></tr><tr><td colspan="1" rowspan="1">A/Duck/Bangladesh/19097/13</td><td colspan="1" rowspan="1">Avian</td><td colspan="6" rowspan="1"></td><td colspan="1" rowspan="1">Viral Genomic RNA</td><td colspan="1" rowspan="1">4.4 x 103</td><td colspan="1" rowspan="1">2x</td></tr><tr><td colspan="1" rowspan="1">A/Egypt/N04915/2014</td><td colspan="1" rowspan="1">Human</td><td colspan="6" rowspan="2"></td><td colspan="1" rowspan="1">Viral Genomic RNA</td><td colspan="1" rowspan="1">3.3 x 103</td><td colspan="1" rowspan="1">1x</td></tr><tr><td colspan="1" rowspan="1">A/Indonesia/NIHRD11771/2011</td><td colspan="1" rowspan="1">Human</td><td colspan="1" rowspan="1"></td><td colspan="4" rowspan="1"></td><td colspan="1" rowspan="1">Viral Genomic RNA</td><td colspan="1" rowspan="1">1.0 x 104</td><td colspan="1" rowspan="1">4x</td></tr><tr><td colspan="1" rowspan="1">A/Chicken/Bangladesh/11rs1984-30/2011</td><td colspan="1" rowspan="1">Avian</td><td colspan="6" rowspan="1"></td><td colspan="1" rowspan="1">Viral Genomic RNA</td><td colspan="1" rowspan="1">3.6 x 103</td><td colspan="1" rowspan="1">1x</td></tr><tr><td colspan="1" rowspan="1">A/Duck/Pennsylvania/10218/1984</td><td colspan="1" rowspan="1">Avian</td><td colspan="6" rowspan="1">A/H5N2</td><td colspan="1" rowspan="1">Viral Genomic RNA</td><td colspan="1" rowspan="1">8.4 x 103</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="1">A/Chicken/Vietnam/NCVD-14-A324/2014</td><td colspan="1" rowspan="1">Avian</td><td colspan="6" rowspan="1">A/H5N6</td><td colspan="1" rowspan="1">Viral Genomic RNA</td><td colspan="1" rowspan="1">2.4 x 104</td><td colspan="1" rowspan="1">8x</td></tr><tr><td colspan="1" rowspan="1">A/Shorebird/Delaware/101/2004</td><td colspan="1" rowspan="1">Avian</td><td colspan="6" rowspan="1">A/H5N7</td><td colspan="1" rowspan="1">Viral Genomic RNA</td><td colspan="1" rowspan="1">3.6 x 105</td><td colspan="1" rowspan="1">124x</td></tr><tr><td colspan="1" rowspan="1">A/Shorebird/Delaware Bay/230/2009</td><td colspan="1" rowspan="1">Avian</td><td colspan="1" rowspan="1">A/H6N1</td><td colspan="1" rowspan="1">Whole Virus</td><td colspan="1" rowspan="1">3.0 x 103</td><td colspan="1" rowspan="1">1x</td></tr><tr><td colspan="1" rowspan="1">A/Ruddy Turnstone/Delaware/293/2006</td><td colspan="1" rowspan="1">Avian</td><td colspan="1" rowspan="2">A/H6N2</td><td colspan="1" rowspan="1">Whole Virus</td><td colspan="1" rowspan="1">2.5 x 105$</td><td colspan="1" rowspan="1">84x</td></tr><tr><td colspan="1" rowspan="1">A/Shorebird/Delaware/124/2001</td><td colspan="1" rowspan="1">Avian</td><td colspan="1" rowspan="1">Whole Virus</td><td colspan="1" rowspan="1">4.2 x 103</td><td colspan="1" rowspan="1">1x</td></tr><tr><td colspan="1" rowspan="1">A/Mallard/Alberta/203/1992</td><td colspan="1" rowspan="1">Avian</td><td colspan="1" rowspan="1">A/H6N5</td><td colspan="1" rowspan="1">Whole Virus</td><td colspan="1" rowspan="1">7.2 x 104</td><td colspan="1" rowspan="1">24x</td></tr><tr><td colspan="1" rowspan="1">A/Chicken/Italy/1285/2000</td><td colspan="1" rowspan="1">Avian</td><td colspan="1" rowspan="2">A/H7N1</td><td colspan="1" rowspan="1">Viral Genomic RNA</td><td colspan="1" rowspan="1">2.9 x 103</td><td colspan="1" rowspan="1">1x</td></tr><tr><td colspan="1" rowspan="1">A/Mallard/Alberta/34/2001</td><td colspan="1" rowspan="1">Avian</td><td colspan="1" rowspan="1">Viral Genomic RNA</td><td colspan="1" rowspan="1">2.0 x 105</td><td colspan="1" rowspan="1">68x</td></tr><tr><td colspan="1" rowspan="1">A/Mallard/Netherlands/12/2000</td><td colspan="1" rowspan="1">Avian</td><td colspan="1" rowspan="3">A/H7N3</td><td colspan="1" rowspan="1">Viral Genomic RNA</td><td colspan="1" rowspan="1">3.3 x 105</td><td colspan="1" rowspan="1">109x</td></tr><tr><td colspan="1" rowspan="1">A/Chicken/Jalisco/12283/12</td><td colspan="1" rowspan="1">Avian</td><td colspan="1" rowspan="1">Viral Genomic RNA</td><td colspan="1" rowspan="1">1.6 x 104</td><td colspan="1" rowspan="1">5x</td></tr><tr><td colspan="1" rowspan="1">A/Laughing Gull/Delaware Bay/42/2006</td><td colspan="1" rowspan="1">Avian</td><td colspan="1" rowspan="1">Viral Genomic RNA</td><td colspan="1" rowspan="1">3.1 x 106</td><td colspan="1" rowspan="1">1024x</td></tr><tr><td colspan="1" rowspan="1">A/Chicken/Chile/176822/2002</td><td colspan="1" rowspan="1">Avian</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Viral Genomic RNA</td><td colspan="1" rowspan="1">4.1 x 10}$</td><td colspan="1" rowspan="1">133x</td></tr><tr><td colspan="1" rowspan="1">A/Ruddy Turnstone/Delaware Bay/290/2006</td><td colspan="1" rowspan="1">Avian</td><td colspan="1" rowspan="1">A/H7N4</td><td colspan="1" rowspan="1">Viral Genomic RNA</td><td colspan="1" rowspan="1">6.1 x 103</td><td colspan="1" rowspan="1">2x</td></tr><tr><td colspan="1" rowspan="1">A/Mallard/Alberta/26/2001</td><td colspan="1" rowspan="1">Avian</td><td colspan="1" rowspan="1">A/H7, N1, N3,N5*</td><td colspan="1" rowspan="1">Viral Genomic RNA</td><td colspan="1" rowspan="1">3.9 x 103</td><td colspan="1" rowspan="1">1x</td></tr><tr><td colspan="1" rowspan="1">A/Netherlands/33/2003</td><td colspan="1" rowspan="1">Avian</td><td colspan="1" rowspan="2">A/H7N7</td><td colspan="1" rowspan="1">Viral Genomic RNA</td><td colspan="1" rowspan="1">3.6 x 105</td><td colspan="1" rowspan="1">124x</td></tr><tr><td colspan="1" rowspan="1">A/Seal/Massachusetts/1/1980</td><td colspan="1" rowspan="1">Seal</td><td colspan="1" rowspan="1">Viral Genomic RNA</td><td colspan="1" rowspan="1">6.9 x 103</td><td colspan="1" rowspan="1">2x</td></tr><tr><td colspan="1" rowspan="1">A/Shanghai/1/2013</td><td colspan="1" rowspan="1">Human</td><td colspan="1" rowspan="1">A/H7N9</td><td colspan="1" rowspan="1">Viral Genomic RNA</td><td colspan="1" rowspan="1">4.8 x 104</td><td colspan="1" rowspan="1">16x</td></tr><tr><td colspan="1" rowspan="1">A/Mallard/Alberta/194/92</td><td colspan="1" rowspan="1">Avian</td><td colspan="1" rowspan="1">A/H8N4</td><td colspan="1" rowspan="1">Whole Virus</td><td colspan="1" rowspan="1">5.2 x 103</td><td colspan="1" rowspan="1">2x</td></tr><tr><td colspan="1" rowspan="1">A/Mallard Duck/Alberta/743/1983</td><td colspan="1" rowspan="1">Avian</td><td colspan="1" rowspan="2">A/H9N1</td><td colspan="1" rowspan="1">Whole Virus</td><td colspan="1" rowspan="1">3.0 x 103</td><td colspan="1" rowspan="1">1x</td></tr><tr><td colspan="1" rowspan="1">A/Shorebird/Delaware Bay/133/2002</td><td colspan="1" rowspan="1">Avian</td><td colspan="1" rowspan="1">Whole Virus</td><td colspan="1" rowspan="1">1.4 x 105</td><td colspan="1" rowspan="1">49x</td></tr><tr><td colspan="1" rowspan="1">A/Quail/Hong Kong/G1/1997</td><td colspan="1" rowspan="1">Avian</td><td colspan="1" rowspan="8">A/H9N2</td><td colspan="1" rowspan="1">Whole Virus</td><td colspan="1" rowspan="1">2.1 x 103</td><td colspan="1" rowspan="1">1x</td></tr><tr><td colspan="1" rowspan="1">A/Quail/Lebanon/272/2010</td><td colspan="1" rowspan="1">Avian</td><td colspan="1" rowspan="1">Viral Genomic RNA</td><td colspan="1" rowspan="1">4.7 x 103</td><td colspan="1" rowspan="1">2x</td></tr><tr><td colspan="1" rowspan="1">A/Chicken/Beijing/1/1994</td><td colspan="1" rowspan="1">Avian</td><td colspan="1" rowspan="1">Viral Genomic RNA</td><td colspan="1" rowspan="1">2.2 x 104</td><td colspan="1" rowspan="1">8x</td></tr><tr><td colspan="1" rowspan="1">A/Hong Kong/33982/2009</td><td colspan="1" rowspan="1">Human</td><td colspan="1" rowspan="1">Viral Genomic RNA</td><td colspan="1" rowspan="1">5.1 x 104</td><td colspan="1" rowspan="1">17x</td></tr><tr><td colspan="1" rowspan="1">A/Chukar/Shantou/22116/2005</td><td colspan="1" rowspan="1">Avian</td><td colspan="1" rowspan="1">Viral Genomic RNA</td><td colspan="1" rowspan="1">2.1 x 104</td><td colspan="1" rowspan="1">7x</td></tr><tr><td colspan="1" rowspan="1">A/Chicken/Hong Kong/NT10/2011</td><td colspan="1" rowspan="1">Avian</td><td colspan="1" rowspan="1">Viral Genomic RNA</td><td colspan="1" rowspan="1">5.5 x 104</td><td colspan="1" rowspan="1">18x</td></tr><tr><td colspan="1" rowspan="1">A/Duck/Hong Kong/Y280/1997</td><td colspan="1" rowspan="1">Avian</td><td colspan="1" rowspan="1">Whole Virus</td><td colspan="1" rowspan="1">2.3 x 105</td><td colspan="1" rowspan="1">79x</td></tr><tr><td colspan="1" rowspan="1">A/Hong Kong/308/14</td><td colspan="1" rowspan="1">Human</td><td colspan="1" rowspan="1">Viral Genomic RNA</td><td colspan="1" rowspan="1">7.0 x 106</td><td colspan="1" rowspan="1">2320x</td></tr><tr><td colspan="1" rowspan="1">A/Shorebird/Delaware Bay/246/2003</td><td colspan="1" rowspan="1">Avian</td><td colspan="1" rowspan="1">A/H9N5</td><td colspan="1" rowspan="1">Whole Virus</td><td colspan="1" rowspan="1">6.0 x 103</td><td colspan="1" rowspan="1">2x</td></tr><tr><td colspan="1" rowspan="1">A/Ruddy Turnstone/Delaware/510/1988</td><td colspan="1" rowspan="1">Avian</td><td colspan="1" rowspan="1">A/H9N6</td><td colspan="1" rowspan="1">Whole Virus</td><td colspan="1" rowspan="1">3.3 x 105</td><td colspan="1" rowspan="1">111x</td></tr><tr><td colspan="1" rowspan="1">A/Gray Plover/Chile/C1313/2015</td><td colspan="1" rowspan="1">Avian</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Whole Virus</td><td colspan="1" rowspan="1">4.4 x 10 }$</td><td colspan="1" rowspan="1">147x</td></tr><tr><td colspan="1" rowspan="1">A/Shorebird/Delaware Bay/31/1996</td><td colspan="1" rowspan="1">Avian</td><td colspan="1" rowspan="1">A/H9N7</td><td colspan="1" rowspan="1">Whole Virus</td><td colspan="1" rowspan="1">3.9 x 103</td><td colspan="1" rowspan="1">1x</td></tr><tr><td colspan="1" rowspan="1">A/Shorebird/Delaware Bay/277/2000</td><td colspan="1" rowspan="1">Avian</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Whole Virus</td><td colspan="1" rowspan="1">4.2 x 103</td><td colspan="1" rowspan="1">1x</td></tr><tr><td colspan="1" rowspan="1">A/Ruddy Turnstone/Virginia/2297/1988</td><td colspan="1" rowspan="1">Avian</td><td colspan="1" rowspan="1">A/H9N9</td><td colspan="1" rowspan="1">Whole Virus</td><td colspan="1" rowspan="1">3.4 x 103</td><td colspan="1" rowspan="1">1x</td></tr><tr><td colspan="1" rowspan="1">A/Mallard/Wisconsin/4230/2009</td><td colspan="1" rowspan="1">Avian</td><td colspan="1" rowspan="2">A/H10N1</td><td colspan="1" rowspan="1">Whole Virus</td><td colspan="1" rowspan="1">2.9 x 105</td><td colspan="1" rowspan="1">97x</td></tr><tr><td colspan="1" rowspan="1">A/Shorebird/Delaware Bay/338/2009</td><td colspan="1" rowspan="1">Avian</td><td colspan="1" rowspan="1">Whole Virus</td><td colspan="1" rowspan="1">1.5 x 104</td><td colspan="1" rowspan="1">5x</td></tr><tr><td colspan="1" rowspan="1">A/Shorebird/Delaware Bay/63/1996</td><td colspan="1" rowspan="1">Avian</td><td colspan="1" rowspan="1">A/H10N2</td><td colspan="1" rowspan="1">Whole Virus</td><td colspan="1" rowspan="1">3.2 x 103</td><td colspan="1" rowspan="1">1x</td></tr><tr><td colspan="1" rowspan="1">A/Shorebird/Delaware/260/2000</td><td colspan="1" rowspan="1">Avian</td><td colspan="1" rowspan="1">A/H10N4</td><td colspan="1" rowspan="1">Whole Virus</td><td colspan="1" rowspan="1">1.2 x 105</td><td colspan="1" rowspan="1">39x</td></tr><tr><td colspan="1" rowspan="1">A/Chicken/Germany/N/49</td><td colspan="1" rowspan="1">Avian</td><td colspan="1" rowspan="2">A/H10N7</td><td colspan="1" rowspan="1">Whole Virus</td><td colspan="1" rowspan="1">3.0 x 103</td><td colspan="1" rowspan="1">1x</td></tr><tr><td colspan="1" rowspan="1">A/Mallard/Illinois/100S4334/2010</td><td colspan="1" rowspan="1">Avian</td><td colspan="1" rowspan="1">Whole Virus</td><td colspan="1" rowspan="1">1.5 x 104</td><td colspan="1" rowspan="1">5x</td></tr><tr><td colspan="1" rowspan="1">A/Quail/Italy/1117/1965</td><td colspan="1" rowspan="1">Avian</td><td colspan="1" rowspan="1">A/H10N8</td><td colspan="1" rowspan="1">Whole Virus</td><td colspan="1" rowspan="1">5.1 x 104</td><td colspan="1" rowspan="1">17x</td></tr><tr><td colspan="1" rowspan="1">A/Shorebird/Delaware Bay/216/1999</td><td colspan="1" rowspan="1">Avian</td><td colspan="1" rowspan="1">A/H11N2</td><td colspan="1" rowspan="1">Whole Virus</td><td colspan="1" rowspan="1">2.5 x 105$</td><td colspan="1" rowspan="1">84x</td></tr><tr><td colspan="1" rowspan="1">A/Ruddy Turnstone/Delaware Bay/39/1994</td><td colspan="1" rowspan="1">Avian</td><td colspan="1" rowspan="1">A/H11N3</td><td colspan="1" rowspan="1">Whole Virus</td><td colspan="1" rowspan="1">5.8 x 105</td><td colspan="1" rowspan="1">194x</td></tr><tr><td colspan="1" rowspan="1">A/Duck/England/56</td><td colspan="1" rowspan="1">Avian</td><td colspan="1" rowspan="2">A/H11N6</td><td colspan="1" rowspan="1">Whole Virus</td><td colspan="1" rowspan="1">3.9 x 103</td><td colspan="1" rowspan="1">1x</td></tr><tr><td colspan="1" rowspan="1">A/Mallard/Alberta/125/1999</td><td colspan="1" rowspan="1">Avian</td><td colspan="1" rowspan="1">Whole Virus</td><td colspan="1" rowspan="1">3.0 x 103</td><td colspan="1" rowspan="1">1x</td></tr><tr><td colspan="1" rowspan="1">A/Laughing Gull/Delaware/2/2002</td><td colspan="1" rowspan="1">Avian</td><td colspan="1" rowspan="4">A/H11N9</td><td colspan="1" rowspan="1">Whole Virus</td><td colspan="1" rowspan="1">4.8 x 103</td><td colspan="1" rowspan="1">2x</td></tr><tr><td colspan="1" rowspan="1">A/Shorebird/Delaware/6/2002</td><td colspan="1" rowspan="1">Avian</td><td colspan="1" rowspan="1">Whole Virus</td><td colspan="1" rowspan="1">5.4 x 105$</td><td colspan="1" rowspan="1">181x</td></tr><tr><td colspan="1" rowspan="1">A/American Green-Winged</td><td colspan="1" rowspan="1">Avian</td><td colspan="1" rowspan="1">Whole Virus</td><td colspan="1" rowspan="1">2.2 x 105</td><td colspan="1" rowspan="1">74x</td></tr><tr><td colspan="1" rowspan="1">A/Common Goldeneye/Iowa/3192/2009</td><td colspan="1" rowspan="1">Avian</td><td colspan="1" rowspan="1">Whole Virus</td><td colspan="1" rowspan="1">1.4 x 105</td><td colspan="1" rowspan="1">49x</td></tr><tr><td colspan="1" rowspan="1">A/Mallard/Ohio/1688/2009</td><td colspan="1" rowspan="1">Avian</td><td colspan="1" rowspan="2">A/H12N5</td><td colspan="1" rowspan="1">Whole Virus</td><td colspan="1" rowspan="1">3.2 x 103</td><td colspan="1" rowspan="1">1x</td></tr><tr><td colspan="1" rowspan="1">A/Mallard/Wisconsin/4218/2009</td><td colspan="1" rowspan="1">Avian</td><td colspan="1" rowspan="1">Whole Virus</td><td colspan="1" rowspan="1">7.3 x 103</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="1">A/Northern Pintail/Missouri/319/2009</td><td colspan="1" rowspan="1">Avian</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Whole Virus</td><td colspan="1" rowspan="1">3.0 x 103</td><td colspan="1" rowspan="1">1x</td></tr><tr><td colspan="1" rowspan="1">A/Northern Shoveler/Mississippi/09OS025/2009</td><td colspan="1" rowspan="1">Avian</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Whole Virus</td><td colspan="1" rowspan="1">3.7 x 103</td><td colspan="1" rowspan="1">1x</td></tr><tr><td colspan="1" rowspan="1">A/Ring-Billed Gull/Quebec/02434-1/2009</td><td colspan="1" rowspan="1">Avian</td><td colspan="1" rowspan="2">A/H13N6</td><td colspan="1" rowspan="1">Whole Virus</td><td colspan="1" rowspan="1">2.3 x 105</td><td colspan="1" rowspan="1">79x</td></tr><tr><td colspan="1" rowspan="1">A/Black-Legged Kittiwake/Quebec/02838-1/2009</td><td colspan="1" rowspan="1">Avian</td><td colspan="1" rowspan="1">Whole Virus</td><td colspan="1" rowspan="1">5.9 x 104</td><td colspan="1" rowspan="1">20x</td></tr><tr><td colspan="1" rowspan="1">A/Wedge-Tailed Shearwater/Western</td><td colspan="1" rowspan="1">Avian</td><td colspan="1" rowspan="2">A/H15N9</td><td colspan="1" rowspan="1">Whole Virus</td><td colspan="1" rowspan="1">3.7 x 103</td><td colspan="1" rowspan="1">1x</td></tr><tr><td colspan="1" rowspan="1">A/Shearwater/Australia/2576/79</td><td colspan="1" rowspan="1">Avian</td><td colspan="1" rowspan="1">Whole Virus</td><td colspan="1" rowspan="1">9.5 x 104</td><td colspan="1" rowspan="1">32x</td></tr></table>

\* This sample is a mixed infection of A/H7N1, A/H7N3, and A/H7N5.

The non-seasonal influenza A strains utilized to build contrived specimens in this study were characterized by a combined method comprised of virus characterization provided on Certificates of Analysis (COA) for commercially obtained strains, sequencing information for the hemagglutinin (HA) and the neuraminidase (NA) genes, and/or an FDA-cleared influenza molecular assay. Performance of the FC8G assay in this study was assessed against the expected results of the contrived specimens. One negative sample produced an inconclusive (internal control failure) result by the FDA-cleared influenza molecular assay, and was removed from the performance analysis. Six negative samples resulted in FC8G assay internal control failures, and were also excluded from the performance analysis.

FC8G assay performance for the remaining 213 contrived specimens, are presented in Table 28 below.

Table 28: FC8G Assay Performance Testing Contrived Specimens – Challenge Study   

<table><tr><td rowspan=2 colspan=1>Virus Group (Analyte)</td><td rowspan=1 colspan=3>Sensitivity</td><td rowspan=1 colspan=3>Specificity</td></tr><tr><td rowspan=1 colspan=1>TP/(TP+FN)</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=1>TN/(TN+FP)</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=1>Influenza A (Overall)</td><td rowspan=1 colspan=1>105/105</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>96.5 - 100</td><td rowspan=1 colspan=1>107/108</td><td rowspan=1 colspan=1>99%</td><td rowspan=1 colspan=1>94.9 - 99.8</td></tr><tr><td rowspan=1 colspan=1>Influenza A/H1N1pdm09</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>34.2 - 100</td><td rowspan=1 colspan=1>209/211</td><td rowspan=1 colspan=1>99%</td><td rowspan=1 colspan=1>96.6  99.7</td></tr><tr><td rowspan=1 colspan=1>Influenza A/H3N2</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>43.9 - 100</td><td rowspan=1 colspan=1>210/210</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>98.2 - 100</td></tr><tr><td rowspan=1 colspan=1>Influenza A/Non-seasonal</td><td rowspan=1 colspan=1>99/100</td><td rowspan=1 colspan=1>99%</td><td rowspan=1 colspan=1>94.6 - 99.8</td><td rowspan=1 colspan=1>113/113</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>96.7 - 100</td></tr><tr><td rowspan=1 colspan=1>Influenza B (Overall)</td><td rowspan=1 colspan=1>5/5</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>56.6 - 100</td><td rowspan=1 colspan=1>208/208</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>98.2 - 100</td></tr><tr><td rowspan=1 colspan=1>Influenza B/Victoria</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>34.2 - 100</td><td rowspan=1 colspan=1>211/211</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>98.2 - 100</td></tr><tr><td rowspan=1 colspan=1>Influenza B/Yamagata</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>43.9 - 100</td><td rowspan=1 colspan=1>210/210</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>98.2 - 100</td></tr></table>

The overall sensitivity of the assay for the non-seasonal influenza A target virus group was $9 9 \%$ ( $9 5 \%$ CI of $9 6 . 4 \% - 9 9 . 8 \% )$ and specificity was $100 \%$ ( $9 5 \%$ CI of $9 6 . 7 \%$ - $1 0 0 . 0 \% )$ . All 10 seasonal influenza viruses included in this study were correctly identified.

One pre-2009 H1N1 strain (A/Fukushima/141/2006) analyzed at approximately $1 . 5 \mathrm { x }$ LoD produced a false positive result for A/H1N1pdm09. Based on the results of the FDAcleared influenza molecular assay, the concentration of human cellular DNA appeared to be orders of magnitude higher than the concentration of the influenza A RNA in this specimen, indicating there might have been a competition between 18s and influenza A amplifications in this specific specimen. In addition, one negative sample resulted in a false positive for A/H1N1pdm09.

4. Clinical cut-off: Not applicable.

5. Expected values/Reference range:

In the FC8G assay prospective clinical study (described in the “Clinical Studies” section above), a total of 984 nasal or nasopharyngeal swab specimens prospectively collected from the 2013-2014 through 2016-2017 influenza seasons were determined to be evaluable. The number and percentage of influenza cases per specified age group, as determined by the FC8G assay, stratified by the virus groups, are presented in Table 29 and Table 30 below.

Table 29: Seasonal Influenza A Positives by the FC8G Assay per Patient Age Group   

<table><tr><td rowspan=1 colspan=1>Age Group</td><td rowspan=1 colspan=1>Number ofNasal orNasopharyngeal SwabSpecimens</td><td rowspan=1 colspan=1>Number ofInfluenzaA/H3N2Positives</td><td rowspan=1 colspan=1>InfluenzaA/H3N2PositivityRate</td><td rowspan=1 colspan=1>Number ofInfluenzaA/H1N1pdm09Positives</td><td rowspan=1 colspan=1>InfluenzaA/H1N1pdm09Positivity Rate</td><td rowspan=1 colspan=1>Number ofSeasonalInfluenza APositives</td><td rowspan=1 colspan=1>SeasonalInfluenza APositivity Rate</td></tr><tr><td rowspan=1 colspan=1>≤ 5 years</td><td rowspan=1 colspan=1>417</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>2.9%</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>1.0%</td><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>3.8%</td></tr><tr><td rowspan=1 colspan=1>6 to 21 years</td><td rowspan=1 colspan=1>278</td><td rowspan=1 colspan=1>29</td><td rowspan=1 colspan=1>10.4%</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>1.1%</td><td rowspan=1 colspan=1>32</td><td rowspan=1 colspan=1>11.5%</td></tr><tr><td rowspan=1 colspan=1>22 to 59 years</td><td rowspan=1 colspan=1>228</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>4.4%</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0.9%</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>5.3%</td></tr><tr><td rowspan=1 colspan=1>≥ 60 years</td><td rowspan=1 colspan=1>61</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>3.3%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>3.3%</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>984</td><td rowspan=1 colspan=1>53</td><td rowspan=1 colspan=1>5.4%</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>0.9%</td><td rowspan=1 colspan=1>62</td><td rowspan=1 colspan=1>6.3%</td></tr></table>

Table 30: Influenza B Positives by the FC8G Assay per Patient Age Group   

<table><tr><td rowspan=1 colspan=1>Age Group</td><td rowspan=1 colspan=1>Number ofNasal orNasopharyngeal SwabSpecimens</td><td rowspan=1 colspan=1>Number ofInfluenzaB/VictoriaPositives</td><td rowspan=1 colspan=1>InfluenzaB/VictoriaPositivityRate</td><td rowspan=1 colspan=1>Number ofInfluenzaB/YamagataPositives</td><td rowspan=1 colspan=1>InfluenzaB/YamagataPositivity Rate</td><td rowspan=1 colspan=1>Number ofInfluenza BPositives</td><td rowspan=1 colspan=1>Influenza BPositivity Rate</td></tr><tr><td rowspan=1 colspan=1>≤ 5 years</td><td rowspan=1 colspan=1>417</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>1.4%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.2%</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>1.7%</td></tr><tr><td rowspan=1 colspan=1>6 to 21 years</td><td rowspan=1 colspan=1>278</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>2.2%</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>1.4%</td><td rowspan=1 colspan=1>12*</td><td rowspan=1 colspan=1>4.3%</td></tr><tr><td rowspan=1 colspan=1>22 to 59 years</td><td rowspan=1 colspan=1>228</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0.9%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.4%</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>1.3%</td></tr><tr><td rowspan=1 colspan=1>≥ 60 years</td><td rowspan=1 colspan=1>61</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>3.3%</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>3.3%</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>984</td><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>1.4%</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>0.8%</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>2.4%</td></tr></table>

\* Two samples were Influenza B positive with no lineage identified by the FC8G assay.

# N. Instrument Name:

BioRad T100 endpoint thermal cycler FluChip-8G Imaging System

# O. System Descriptions:

1. Modes of Operation:

Does the applicant’s device contain the ability to transmit data to a computer, webserver, or mobile device?

Yes ___X _ or No

Does the applicant’s device transmit data to a computer, webserver, or mobile device using wireless transmission?

Yes or No __X_

2. Software:

FDA has reviewed applicant’s Hazard Analysis and software development processes for this line of product types:

Yes _X___ or No

3. Specimen Identification: Operator manually label each specimen container with patient ID.

4. Specimen Sampling and Handling: Manually handled using pipettes.

5. Calibration: To be qualified annually by InDevR.

6. Quality Control: Quality control is addressed for each specific FDA-cleared assay to be run on the instruments.

P. Other Supportive Instrument Performance Characteristics Data Not Covered in the “Performance Characteristics” Section above:

Not applicable.

# Q. Proposed Labeling:

The labeling supports the finding of substantial equivalence for this device.

# R. Conclusion:

The submitted information in this premarket notification is complete and supports a substantial equivalence decision.