# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATION DECISION MEMORANDUM

A. 510(k) Number:

K182619

# B. Purpose for Submission:

To establish performance claims and marketing clearance for the ePlex Blood Culture Identification Gram-Negative (BCID-GN) Panel

# C. Measurand:

Nucleic acid targeted sequences from Acinetobacter baumannii, Bacteroides fragilis, Citrobacter, Cronobacter sakazakii, Enterobacter cloacae complex, Enterobacter (non-cloacae complex), Escherichia coli, Fusobacterium necrophorum, Fusobacterium nucleatum, Haemophilus influenzae, Klebsiella oxytoca, Klebsiella pneumoniae group, Morganella morganii, Neisseria meningitidis, Proteus, Proteus mirabilis, Pseudomonas aeruginosa, Salmonella, Serratia, Serratia marcescens, Stenotrophomonas maltophilia, CTX-M, IMP, KPC, NDM, OXA, and VIM, Pan Gram-Positive, Pan Candida

# D. Type of Test:

A multiplexed nucleic acid test intended for use with the GenMark ePlex instrument for the simultaneous qualitative in vitro detection and identification of multiple bacterial nucleic acids from positive blood culture specimens.

# E. Applicant:

GenMark Diagnostics, Incorporated

F. Proprietary and Established Names:

ePlex Blood Culture Identification Panel Gram Negative (BCID-GN)

Common Name: ePlex BCID-GN Panel

# G. Regulatory Information:

1. Regulation section: 21 CFR 866.3365

2. Classification: Class II

3. Product code(s): PEN, PAM, PEO

# H. Intended Use:

1. Intended use(s):

The GenMark ePlex Blood Culture Identification Gram-Negative (BCID-GN) Panel is a qualitative nucleic acid multiplex in vitro diagnostic test intended for use on GenMark’s ePlex Instrument for simultaneous qualitative detection and identification of multiple potentially pathogenic gram-negative bacterial organisms and select determinants associated with antimicrobial resistance in positive blood culture. In addition, the ePlex BCID-GN Panel is capable of detecting several gram-positive bacteria (Pan Gram-Positive assay) and several Candida species (Pan Candida assay). The ePlex BCID-GN Panel is performed directly on blood culture samples identified as positive by a continuous monitoring blood culture system and which contain gram-negative organism.

The following bacterial organisms and genes associated with antibiotic resistance are identified using the ePlex BCID-GN Panel: Acinetobacter baumannii, Bacteroides fragilis, Citrobacter, Cronobacter sakazakii, Enterobacter cloacae complex, Enterobacter (noncloacae complex), Escherichia coli, Fusobacterium necrophorum, Fusobacterium nucleatum, Haemophilus influenzae, Klebsiella oxytoca, Klebsiella pneumoniae group, Morganella morganii, Neisseria meningitidis, Proteus, Proteus mirabilis, Pseudomonas aeruginosa, Salmonella, Serratia, Serratia marcescens, Stenotrophomonas maltophilia, CTX-M (blaCTX-M), IMP (blaIMP) , KPC (blaKPC) , NDM (blaNDM), OXA (blaOXA) (OXA-23 and OXA-48 groups only), and VIM (blaVIM).

The ePlex BCID-GN Panel contains assays for the detection of genetic determinants associated with resistance to antimicrobial agents including CTX-M(blaCTX-M), which is associated with resistance to extended spectrum beta-lactamase (ESBL)-mediated resistance to penicillins, cephalosporins and monobactams, as well as OXA (blaOXA) (OXA-23 and OXA-48 groups only), KPC $( b l a _ { \mathrm { K P C } } )$ , and metallo-beta-lactamases IMP $( b l a _ { \mathrm { I M P } } )$ , VIM (blaVIM), and NDM (blaNDM), which is associated with carbapenemase-mediated resistance. The antimicrobial resistance gene detected may or may not be associated with the agent responsible for disease. Negative results for these select antimicrobial resistance assays do not indicate susceptibility, as there are multiple mechanisms of resistance in gram-negative bacteria.

The ePlex BCID-GN Panel also contains targets designed to detect a broad range of organisms with a potentially misleading Gram stain result or organisms that may be missed by Gram staining altogether, for example in the case of co-infections. These include a broad Pan Gram-Positive assay (which is designed to detect Bacillus cereus group, Bacillus subtilis group, Enterococcus, Staphylococcus, and Streptococcus), as well as a Pan Candida assay, which is designed to detect four Candida species: Candida albicans, Candida glabrata, Candida krusei, and Candida parapsilosis.

The detection and identification of specific bacterial and fungal nucleic acids from individuals exhibiting signs and/or symptoms of bloodstream infection aids in the diagnosis of bloodstream infection when used in conjunction with other clinical information. The results from the ePlex BCID-GN Panel are intended to be interpreted in conjunction with Gram stain results and should not be used as the sole basis for diagnosis, treatment, or other patient management decisions.

Negative results in the setting of a suspected bloodstream infection may be due to infection with pathogens that are not detected by this test. Positive results do not rule out co-infection with other organisms; the organism(s) detected by the ePlex BCID-GN Panel may not be the definite cause of disease. Additional laboratory testing (e.g. sub-culturing of positive blood cultures for identification of organisms not detected by ePlex BCID-GN Panel and for susceptibility testing, differentiation of mixed growth, and association of antimicrobial resistance marker genes to a specific organism) and clinical presentation must be taken into consideration in the final diagnosis of bloodstream infection.

2. Indication(s) for use: Same as intended use.

3. Special conditions for use statement(s):

For prescription use only

Limitations:

This product should not be used with blood culture media that contains charcoal. Bacterial and fungal nucleic acids may be present in blood culture, independent of bacterial or fungal viability. Detection of an assay target does not guarantee that the corresponding bacteria or fungi are infectious or are the causative agents for clinical symptoms.   
• Decreased sensitivity has been observed for some targets in BD BACTEC Lytic Anaerobic blood culture bottles.   
• In mixed cultures, the ePlex BCID-GN Panel may not identify all organisms in the specimen, depending upon the concentration of each target present.   
• Antimicrobial resistance can occur via multiple mechanisms. A “Not Detected” result for the BCID-GP antimicrobial resistance gene assays does not indicate antimicrobial susceptibility. Subculturing and standard susceptibility testing of isolates is required to determine antimicrobial susceptibility.   
• The BCID-GN Pan Candida and Pan Gram-Positive assays are designed to detect Candida and gram-positive microorganisms in mixed blood cultures where these microorganisms might be missed by Gram stain. Lower than expected clinical sensitivity was observed for these analytes, which was likely due to the presence of Candida spp. or gram-positive microorganisms in mixed cultures at concentrations below the respective limits of detection for the Pan Candida and Pan GP targets. For Enterococcus saccharolyticus, Serratia odorifera, and Streptococcus thoraltensis, $100 \%$ detection was not observed at expected bottle positivity concentrations. Decreased sensitivity may be observed for these strains.   
False negative results may occur for specimens containing Fusobacterium   
necrophorum because the assay’s limit of detection for $F$ . necrophorum can be near the concentration observed at the time of bottle positivity.

4. Special instrument requirements: GenMark ePlex Instrument and Software

# I. Device Description:

The ePlex Blood Culture Identification Gram-Negative (BCID-GN) Panel is based on the principles of competitive nucleic acid hybridization using a sandwich assay format, wherein a single-stranded target binds concurrently to a sequence-specific solution-phase signal probe and a solid-phase electrode-bound capture probe. The test employs nucleic acid extraction, target amplification via polymerase chain reaction (PCR) or reverse transcription (RT-PCR) and hybridization of target DNA. In the process, the double-stranded PCR amplicons are digested with exonuclease to generate single-stranded DNA suitable for hybridization.

Nucleic acid extraction from biological specimens occurs within the cartridge via cell lysis, nucleic acid capture onto magnetic beads, and release for amplification. The nucleic acid extraction is processed through microfluidic liquid handling. Once the nucleic acid targets are captured and inhibitors are washed away, the magnetic particles are delivered to the electrowetting environment on the printed circuit board (PCB) and the targets are eluted from the particles and amplified.

During hybridization, the single-stranded target DNA binds to a complementary, single-stranded capture probe immobilized on the working gold electrode surface. Single-stranded signal proves (labeled with electrochemically active ferrocenes) bind to specific target sequence/region adjacent to the capture probe. Simultaneous hybridization of target to signal probes and capture probe is detected by alternating current voltammetry (ACV). Each working electrode on the array contains specific capture probes, and sequential analysis of each electrode allows detection of multiple analyte targets.

A summary of microorganisms and resistance markers detected by the ePlex BCID-GN Panel are presented in Table 1:

Table 1: ePlex BCID-GN Panel, organism and resistance marker targets   

<table><tr><td colspan="2" rowspan="1">Bacterial Targets</td></tr><tr><td colspan="1" rowspan="1">Acinetobacter baumannii</td><td colspan="1" rowspan="1">Klebsiella pneumoniae group</td></tr><tr><td colspan="1" rowspan="1">Bacteroides fragilis</td><td colspan="1" rowspan="1">Morganella morganii</td></tr><tr><td colspan="1" rowspan="1">Citrobacter</td><td colspan="1" rowspan="1">Neisseria meningitidis</td></tr><tr><td colspan="1" rowspan="1">Cronobacter sakazakii</td><td colspan="1" rowspan="1">Proteus</td></tr><tr><td colspan="1" rowspan="1">Enterobacter cloacae complex</td><td colspan="1" rowspan="1">Proteus mirabilis</td></tr><tr><td colspan="1" rowspan="1">Enterobacter (non-cloacae complex)</td><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">Salmonella</td></tr><tr><td colspan="1" rowspan="1">Fusobacterium necrophorum</td><td colspan="1" rowspan="1">Serratia</td></tr><tr><td colspan="1" rowspan="1">Fusobacterium nucleatum</td><td colspan="1" rowspan="1">Serratia marcescens</td></tr><tr><td colspan="1" rowspan="1">Haemophilus influenzae</td><td colspan="1" rowspan="1">Stenotrophomonas maltophilia</td></tr><tr><td colspan="1" rowspan="1">Klebsiella oxytoca</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Antimicrobial Resistance Markers</td><td colspan="1" rowspan="1">Antimicrobial Resistance Markers(See Table 5 For More Detail)</td></tr><tr><td colspan="1" rowspan="1">CTX-M (blacTX-M)</td><td colspan="1" rowspan="1">NDM (blanDM)</td></tr><tr><td colspan="1" rowspan="1">IMP (blaMP)</td><td colspan="1" rowspan="1">OXA (blaoxA)</td></tr><tr><td colspan="1" rowspan="1">KPC (blakPC)</td><td colspan="1" rowspan="1">VIM (blavIM)</td></tr><tr><td colspan="2" rowspan="1">Pan Targets</td></tr><tr><td colspan="1" rowspan="1">Pan Gram-Positive</td><td colspan="1" rowspan="1">Pan Candida</td></tr></table>

# Materials provided

ePlex Blood Culture Identification Gram-Negative Panel kit containing 12 individual test cartridges. Storage is at $2 { - } 8 ^ { \circ } \mathrm { C }$ .

# Materials needed but not provided

Equipment

• GenMark ePlex instrument and software • Pipettes calibrated to deliver $5 0 ~ \mu \mathrm { l }$ Printer (optional)

Consumables

• Pipette tips, aerosol resistant, RNase/DNase-free Disposable, powder free gloves $10 \%$ bleach for decontamination of appropriate surfaces $70 \%$ ethanol or isopropyl alcohol $1 . 5 \mathrm { m L }$ RNase/DNase-free microcentrifuge tube or equivalent

# Interpretation of Results

Results interpretation of the ePlex BCID-GP Panel is performed by the ePlex instrument, and the interpretation of results on the ePlex BCID-GP Panel Detection Report for each targeted analyte is summarized in Table 2.

Table 2: Interpretation of results, ePlex BCID-GN Panel Detection Report   

<table><tr><td rowspan=1 colspan=1>Target Result</td><td rowspan=1 colspan=1>Explanation</td><td rowspan=1 colspan=1>Action</td></tr><tr><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>The test was completed successfully and the targethas generated signal above its defined threshold andthe Internal Control was reported as PASS.</td><td rowspan=1 colspan=1>All results are displayed on theePlex BCID-GN Panel DetectionReport.Test is valid, report results.</td></tr><tr><td rowspan=1 colspan=1>Not Detected</td><td rowspan=1 colspan=1>The test was completed successfully and the targetdid not generate signal above its defined thresholdand the Internal Control was reported as PASS.</td><td rowspan=1 colspan=1>All results are displayed on theePlex BCID-GN Panel DetectionReport.Test is valid, report results.</td></tr><tr><td rowspan=1 colspan=1>Invalid</td><td rowspan=1 colspan=1>The test has not successfully completed and resultsfor this test are not valid. This may be due to aninstrument or software error.</td><td rowspan=1 colspan=1>No results are displayed on theePlex BCID-GN Panel DetectionReport.Test is not valid, repeat test.</td></tr></table>

Genus and group assay result interpretation

While many results on the ePlex BCID-GN Panel are based on a single assay, the ePlex BCID-GN Panel uses two assays for each of the Proteus and Serratia results.

The ePlex BCID-GN Proteus result is based on two assays: the species-specific Proteus mirabilis and the broad Proteus assay. The broad Proteus assay will detect Proteus mirabilis, however, its primary purpose is to detect non-mirabilis Proteus species. See Table 3 for detailed Proteus target call logic.

Table 3: Proteus target results from ePlex BCID-GN Panel Detection Report   

<table><tr><td rowspan=1 colspan=1>Proteus Result</td><td rowspan=1 colspan=1>Proteus Assay</td><td rowspan=1 colspan=1>Proteus mirabilis Assay</td><td rowspan=1 colspan=1>Description</td></tr><tr><td rowspan=1 colspan=1>Not Detected</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>No Proteus species detected</td></tr><tr><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Proteus mirabilis detected</td></tr><tr><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Unspeciated Proteus detected</td></tr></table>

The ePlex BCID-GN Serratia result is based on two assays: the species-specific Serratia marcescens and the broad Serratia assay. The broad Serratia assay will detect Serratia marcescens, however, its primary purpose is to detect Serratia species other than Serratia marcescens. See Table 4 for detailed Serratia target call logic.

Table 4 Serratia target results from ePlex BCID-GN Panel Detection Report   

<table><tr><td rowspan=1 colspan=1>Serratia Result</td><td rowspan=1 colspan=1>Serratia Assay</td><td rowspan=1 colspan=1>Serratia marcescens Assay</td><td rowspan=1 colspan=1>Description</td></tr><tr><td rowspan=1 colspan=1>Not Detected</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>No Serratia species detected</td></tr><tr><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Serratia marcescens detected</td></tr><tr><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Unspeciated Serratia detected</td></tr></table>

Test results for antimicrobial resistance genes are only reported when an associated organism is detected in the same specimen. Table 5 specifies organism targets that are reported with each the six resistance markers on the ePlex BCID Panel (indicated by X in the table).

Table 5. Resistance marker organism associations   

<table><tr><td rowspan=2 colspan=1>Organism</td><td rowspan=1 colspan=6>Resistance Gene</td></tr><tr><td rowspan=1 colspan=1>CTX-M</td><td rowspan=1 colspan=1>IMP</td><td rowspan=1 colspan=1>KPC</td><td rowspan=1 colspan=1>NDM</td><td rowspan=1 colspan=1>OXA</td><td rowspan=1 colspan=1>VIM</td></tr><tr><td rowspan=1 colspan=1>Acinetobacter baumannii</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td></tr><tr><td rowspan=1 colspan=1>Bacteroides fragilis</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Citrobacter</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td></tr><tr><td rowspan=1 colspan=1>Cronobacter sakazakii</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Enterobacter cloacae complex</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td></tr><tr><td rowspan=1 colspan=1>Enterobacter (non-cloacae complex)</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td></tr><tr><td rowspan=1 colspan=1>Escherichia coli</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td></tr><tr><td rowspan=1 colspan=1>Fusobacterium necrophorum</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Fusobacterium nucleatum</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Haemophilus influenzae</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Klebsiella oxytoca</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td></tr><tr><td rowspan=1 colspan=1>Klebsiella pneumoniae group</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td></tr><tr><td rowspan=1 colspan=1>Morganella morganii</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td></tr><tr><td rowspan=1 colspan=1>Neisseria meningitidis</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Proteus</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td></tr><tr><td rowspan=1 colspan=1>Proteus mirabilis</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td></tr><tr><td rowspan=1 colspan=1>Pseudomonas aeruginosa</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td></tr><tr><td rowspan=1 colspan=1>Salmonella</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td></tr><tr><td rowspan=1 colspan=1>Serratia</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td></tr><tr><td rowspan=1 colspan=1>Serratia marcescens</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td></tr><tr><td rowspan=1 colspan=1>Stenotrophomonas maltophilia</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr></table>

# Pan Assay Results

The ePlex BCID-GN Panel Pan Gram-Positive assay detects one or more of the following gram-positive organisms: Enterococcus, Bacillus cereus group, Bacillus subtilis group, Staphylococcus, and/or Streptococcus as shown in Table 6.

Table 6: Pan gram-positive target results from ePlex BCID-GN Panel Detection Report   

<table><tr><td rowspan=1 colspan=1>Pan Gram-Positive Result</td><td rowspan=1 colspan=1>Description</td></tr><tr><td rowspan=1 colspan=1>Not Detected</td><td rowspan=1 colspan=1>No specified gram-positive organism detected.</td></tr><tr><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>One or more of the following gram-positive organisms has been detected:Enterococcus, Bacillus cereus group, Bacillus subtilis group,Staphylococcus, and/or Streptococcus. Additional testing for speciation issuggested.</td></tr></table>

The ePlex BCID-GN Panel Pan Candida detects one or more of the following Candida species: Candida albicans, Candida glabrata, Candida krusei, and/or Candida parapsilosis as shown in Table 7.

Table 7: Pan Candida Target Results from ePlex BCID-GN Panel Detection Report   

<table><tr><td rowspan=1 colspan=1>Pan CandidaResult</td><td rowspan=1 colspan=1>Description</td></tr><tr><td rowspan=1 colspan=1>Not Detected</td><td rowspan=1 colspan=1>No specified Candida species detected.</td></tr><tr><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>One or more of the following Candida organisms has been detected:Candida albicans, Candida glabrata, Candida krusei, and/or Candidaparapsilosis. Additional testing for identification is recommended.</td></tr></table>

# ePlex BCID-GN Panel reports:

Several different reports are available on the ePlex System. Results are provided in a printable format and may be viewed electronically or exported for additional analysis. Reports can be customized with account specific information such as the address, logo and institutional specific footers on each report. For more information on ePlex Reports, refer to the ePlex Operator Manual.

# Detection Report

The ePlex BCID-GN Panel Detection Report includes the results for each individual sample run on the ePlex System. The Summary section indicates the overall test result and lists all detected targets in that sample. The Results section includes a list of all targets on the panel with an individual result for each target. Results are reported as Detected, Not Detected, or Invalid (displayed as a red $\mathrm { i }$ ); results for the Internal Control are reported as PASS, FAIL, INVALID, or N/A.

# External Control Report

The ePlex BCID-GN Panel External Control Report is generated for an external control that has been pre-defined in the ePlex BCID-GN Panel Software. For more information on defining external controls on the ePlex System, refer to the ePlex Operator Manual.

The Summary section indicates the overall result (PASS or FAIL status) and lists all detected targets for that external control. The Results section includes a list of all panel targets with the result, expected result, and PASS/FAIL status for each. Results are reported as Detected, Not Detected, or Invalid (displayed as a red x). A target is reported as PASS if the actual result matches the expected result (as defined for that control); a target is reported as FAIL if the actual result does not match the expected result. If the actual result for each target matches the expected result (all targets reported as PASS), the overall result for the external control is reported as PASS in the Summary section. If the actual result for any target does not match the expected result, the overall result for the external control is reported as FAIL in the Summary section.

# Summary Report

The Summary Report allows the operator to use searchable criteria to create customized reports, using specified targets, dates, range of dates, sample, external control, test bay, or operator. For more information on creating Summary Reports, refer to the ePlex Operator Manual.

# J. Substantial Equivalence Information:

1. Predicate device name(s): FilmArray Blood Culture Identification Panel

2. Predicate 510(k) number(s): K130914

3. Comparison with predicate:

<table><tr><td colspan="3" rowspan="1">Similarities</td></tr><tr><td colspan="1" rowspan="1">Item</td><td colspan="1" rowspan="1">New DeviceePlex Blood Culture IdentificationGram Negative Panel(K182619)</td><td colspan="1" rowspan="1">Predicate DeviceFilmArray Blood CultureIdentification Panel(K130914)</td></tr><tr><td colspan="1" rowspan="1">Classification</td><td colspan="1" rowspan="1">21 CFR 866.3365Multiplex nucleic acid assay foridentification of microorganisms andresistance markers from positive bloodcultures</td><td colspan="1" rowspan="1">21 CFR 866.3365Multiplex nucleic acid assay foridentification of microorganisms andresistance markers from positive bloodcultures</td></tr><tr><td colspan="1" rowspan="1">Device Class</td><td colspan="1" rowspan="1">Class II</td><td colspan="1" rowspan="1">Class II</td></tr><tr><td colspan="1" rowspan="1">Product Code</td><td colspan="1" rowspan="1">PEN, PAM, PEO</td><td colspan="1" rowspan="1">PPEN, PAM, PEO</td></tr><tr><td colspan="1" rowspan="1">Intended Use</td><td colspan="1" rowspan="1">The ePlex Blood Culture IdentificationGram-Negative (BCID-GN) Panel is aqualitative nucleic acid multiplex in vitrodiagnostic test intended for use onGenMark's ePlex instrument forsimultaneous detection and identificationof multiple potentially pathogenic gramnegative bacterial organisms and selectdeterminants of antimicrobial resistancein positive blood culture. In addition, theePlex Panel is capable of detectingseveral gram-positive bacteria (PanGram-Positive assay), and severalCandida species (Pan Candida assay).The ePlex BCID-GN Panel is performeddirectly on blood culture samplesidentified as positive by a continuousmonitoring blood culture system thatdemonstrates the presence of organismsas confirmed by Gram stain.</td><td colspan="1" rowspan="1">The FilmArray Blood CultureIdentification (BCID) Panel is aqualitative multiplexed nucleic acid-based in vitro diagnostic test intendedfor use with FilmArray systems. TheFilmArray BCID Panel is capable ofsimultaneous detection andidentification of multiple bacterial andyeast nucleic acids and select geneticdeterminants of antimicrobialresistance. The BCID assay isperformed directly on blood culturesamples identified as positive by acontinuous monitoring blood culturesystem that demonstrate the presence oforganisms as determined by Gram stain.</td></tr><tr><td colspan="1" rowspan="1">OrganismsDetected</td><td colspan="1" rowspan="1">Acinetobacter baumannii, Bacteroidesfragilis, Citrobacter, Cronobactersakazakii, Enterobacter cloacaecomplex, Enterobacter (non-cloacae</td><td colspan="1" rowspan="1">Enterococci, Listeria monocytogenes,Staphylococci (including specificdifferentiation of Staphylococcusaureus), Streptococci (with specific</td></tr><tr><td colspan="1" rowspan="1">′</td><td colspan="1" rowspan="1">complex), Escherichia coli,Fusobacterium necrophorum,Fusobacterium nucleatum, Haemophilusinfluenzae, Klebsiella oxytoca, Klebsiellapneumoniae, Morganella morganii,Neisseria meningitidis, Proteus, Proteusmirabilis, Pseudomonas aeruginosa,Salmonella, Serratia, Serratiamarcescens, Stenotrophomonasmaltophilia, Pan Gram-Positive (selectedorganisms), Pan Candida (selectedspecies)</td><td colspan="1" rowspan="1">differentiation of Streptococcusagalactiae, Streptococcus pneumoniae,and Streptococcus pyogenes),Acinetobacter baumannii,Enterobacteriaceae (including specificdifferentiation of the Enterobactercloacae complex, Escherichia coli,Klebsiella oxytoca, Klebsiellapneumoniae, Proteus, and Serratiamarcescens), Haemophilus influenzae,Neisseria meningitidis (encapsulated),Pseudomonas aeruginosa, Candidaalbicans, Candida glabrata, Candidakrusei, Candida parapsilosis, andCandida tropicalis</td></tr><tr><td colspan="1" rowspan="1">Analyte</td><td colspan="1" rowspan="1">DNA</td><td colspan="1" rowspan="1">DNA</td></tr><tr><td colspan="1" rowspan="1">TechnologicalPrinciples</td><td colspan="1" rowspan="1">Multiplex nucleic acid amplification test(NAAT)</td><td colspan="1" rowspan="1">Multiplex nucleic acid amplification test(NAAT)</td></tr><tr><td colspan="1" rowspan="1">Specimen Type</td><td colspan="1" rowspan="1">Blood culture samples identified aspositive by a continuous monitoringblood culture system that demonstratesthe presence of organisms (Gram-positive, Gram-negative, yeasts) asconfirmed by Gram stain</td><td colspan="1" rowspan="1">Blood culture samples identified aspositive by a continuous monitoringblood culture system that demonstratesthe presence of organisms (Gram-positive, Gram-negative, yeasts) asconfirmed by Gram stain</td></tr><tr><td colspan="1" rowspan="1">TestInterpretation</td><td colspan="1" rowspan="1">Automated test interpretation and reportgeneration. User cannot access raw data.</td><td colspan="1" rowspan="1">Automated test interpretation and reportgeneration. User cannot access raw data.</td></tr><tr><td colspan="1" rowspan="1">SamplePreparationMethod</td><td colspan="1" rowspan="1">Sample processing is automated in theePlex BCID-GN cartridge</td><td colspan="1" rowspan="1">Sample processing is automated in theFilmArray BCID pouch</td></tr><tr><td colspan="1" rowspan="1">Controls</td><td colspan="1" rowspan="1">Each cartridge includes two types ofinternal controls that together monitorperformance of each step of the testingprocess, including extraction,amplification and detection processes.</td><td colspan="1" rowspan="1">Two controls are included in eachreagent pouch to control for eachsample processing and both stages ofPCR and melt analysis.</td></tr></table>

<table><tr><td colspan="3" rowspan="1">Differences</td></tr><tr><td colspan="1" rowspan="1">Item</td><td colspan="1" rowspan="1">New DeviceePlex Blood Culture IdentificationGram Negative Panel(K182619)</td><td colspan="1" rowspan="1">Predicate DeviceFilmArray Blood CultureIdentification Panel(K130914)</td></tr><tr><td colspan="1" rowspan="1">ResistanceGenes</td><td colspan="1" rowspan="1">Assay detects ctx-M, imp, kpc, ndm,oxa, and vim</td><td colspan="1" rowspan="1">Assay detects mecA, vanA, vanB andkpc.</td></tr><tr><td colspan="1" rowspan="1">Chemistry</td><td colspan="1" rowspan="1">Reagents contained within cartridge toallow: sample lysis and nucleic acidextraction, PCR amplification, andhybridization-based electrochemicaldetection</td><td colspan="1" rowspan="1">Reagents contained with a pouch toallow: sample lysis, nucleic acidextraction, multiplex nested PCRfollowed by high resolution meltinganalysis to confirm identity of amplifiedproduct</td></tr><tr><td colspan="1" rowspan="1">Instrumentation</td><td colspan="1" rowspan="1">ePlex Instrument</td><td colspan="1" rowspan="1">FilmArray Instrument</td></tr><tr><td colspan="1" rowspan="1">ReagentStorage</td><td colspan="1" rowspan="1">Cartridge containing reagents is storedat refrigerated temperature</td><td colspan="1" rowspan="1">Reagents are stored at roomtemperature.</td></tr><tr><td colspan="1" rowspan="1">Gram Stain</td><td colspan="1" rowspan="1">Specimens tested have gram negativeorganisms based on Gram stain prior totesting.</td><td colspan="1" rowspan="1">Specimens tested have organismspresent based on Gram stain.</td></tr></table>

# K. Standard/Guidance Document Referenced:

Class II Special Controls Guideline: Multiplex Nucleic Acid Assay for Identification of   
Microorganisms and Resistance Markers from Positive Blood Cultures (May 2015)   
CLSI MM17-A, Vol 28, No. 9, Verification and Validation of Multiplex Nucleic Acid   
Assays   
CLSI EP17-A2 Evaluation of Detection Capability for Clinical Laboratory Measurement   
Procedures; Approved Guideline – Second Edition (June 2012)   
CLSI EP07-A2: Interference Testing in Clinical Chemistry; Approved Guideline – Second   
Edition (November 2005)   
CLSI EP25-A Evaluation of Stability of In Vitro Diagnostic Reagents; Approved Guideline   
(May 2013)

# L. Test Principle:

The ePlex instrument automates all aspects of sample processing and nucleic acid testing in a single cartridge associated with the ePlex BCID-GN Panel. Multiple bloodstream pathogens can be detected from a single blood culture specimen. Automation of DNA extraction, amplification, and target detection is accomplished within the cartridge using a combination of electrowetting and GenMark’s eSensor technology. Electrowetting, or digital microfluidics, uses electrical fields to directly manipulate discrete droplets on the surface of a hydrophobically coated printed circuit board (PCB). Sample and reagents are moved in a programmable fashion in the ePlex cartridge to complete all portions of the sample processing and detection. eSensor technology is based on the principles of competitive DNA hybridization and electrochemical detection, which is highly specific and is not based on fluorescent or optical detection.

A sample is loaded onto the ePlex cartridge and nucleic acids are extracted and purified from the specimen via magnetic solid phase extraction. PCR is used to create double-stranded DNA, which is treated with exonuclease to create single-stranded DNA in preparation for eSensor detection.

The target DNA is mixed with ferrocene-labeled signal probes that are complementary to the specific targets on the panel. Target DNA hybridizes to its complementary signal probe and target-specific capture probes, which are bound to gold-plated electrodes, forming a hybridization complex as shown below in Figure 1. The presence of each target is determined by voltammetry which generates specific electrical signals from the ferrocene-labeled signal probe.

![](images/530bb57e96d645e5d3062aec0bb1d7c282a2fa674683d37b2dea9df851260249.jpg)  
Figure 1

# M. Performance Characteristics (if/when applicable):

1. Analytical performance:

a. Precision/Reproducibility:

A multisite reproducibility study was conducted at three testing sites (two external sites and one internal site). The study evaluated multiple sources of variability, such as site-tosite, lot-to-lot, day-to-day, and operator-to-operator variability. One ePlex instrument with four towers was used at each study site. Two operators each tested the reproducibility panel at each site on five nonconsecutive days with three unique lots of ePlex BCID-GN Panel cartridges.

The reproducibility panel consisted of a seven-member test panel comprised of six panel members representing 17 BCID-GN analytes (including five resistance markers) and one panel member containing an off-panel organism, Cutibacterium granulosum.

Panel members were prepared by spiking cultured isolates into negative blood culture matrix (NBM). The NBM was prepared by incubating $1 0 ~ \mathrm { m L }$ of negative whole blood in BD BACTEC Standard/10 Aerobic/F blood culture bottles per the manufacturer’s instructions for a minimum of eight hours on a continuous monitoring blood culture system. After incubation and prior to use in analytical studies, the NBM was pre-screened to ensure the matrix is negative for all BCID-GN targets.

Each of the on-panel organisms was evaluated at two concentrations: (1) Bottle positivity and (2) One log higher than bottle positivity (to mimic organism concentrations present after eight hours additional incubation on the blood culture instrument).

The organisms and concentrations evaluated in the study are summarized in Table 8.

Table 8: Bottle Positivity Concentrations   

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Bottle PositivityConcentration</td><td rowspan=1 colspan=1>Bottle Positivity +8 HoursConcentration</td></tr><tr><td rowspan=1 colspan=1>Acinetobacter baumannii (OXA+)</td><td rowspan=1 colspan=1>1 x 108 CFU/mL</td><td rowspan=1 colspan=1>1 x 109 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Enterobacter cloacae (CTX-M+, KPC+)</td><td rowspan=1 colspan=1>1 x 108 CFU/mL</td><td rowspan=1 colspan=1>1 x 109 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Escherichia coli (IMP+)</td><td rowspan=1 colspan=1>1 x 108 CFU/mL</td><td rowspan=1 colspan=1>1 x 109 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Fusobacterium nucleatum</td><td rowspan=1 colspan=1>1 x 107 CFU/mL</td><td rowspan=1 colspan=1>1 x 108 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Haemophilus influenzae</td><td rowspan=1 colspan=1>1 x 108 CFU/mL</td><td rowspan=1 colspan=1>1 x 109 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Klebsiella oxytoca</td><td rowspan=1 colspan=1>1 x 108 CFU/mL</td><td rowspan=1 colspan=1>1 x 109 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Neisseria meningitidis</td><td rowspan=1 colspan=1>3 x 107 CFU/mL</td><td rowspan=1 colspan=1>3 x 108 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Pseudomonas aeruginosa (VIM+)</td><td rowspan=1 colspan=1>1 x 108 CFU/mL</td><td rowspan=1 colspan=1>1 x 109 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Serratia marcescens</td><td rowspan=1 colspan=1>1 x 108 CFU/mL</td><td rowspan=1 colspan=1>1 x 109 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Candida albicans (Pan Candida target)</td><td rowspan=1 colspan=1>1 x 106 CFU/mL</td><td rowspan=1 colspan=1>1 x 107 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus aureus (Pan Gram-Positivetarget)</td><td rowspan=1 colspan=1>1 x 107 CFU/mL</td><td rowspan=1 colspan=1>1 x 108 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Cutibacterium granulosum</td><td rowspan=1 colspan=1>Incubated on instrumentuntil bottle positivity pluseight hours</td><td rowspan=1 colspan=1>N/A</td></tr></table>

The seven panel members were tested 108 times each (excluding repeat testing): 3 replicates $_ { \textrm { X 3 } }$ sites $_ { \textrm { X 2 } }$ operators x 3 lots $_ { \textrm { X 2 } }$ days of testing/operator/lot, for a minimum number of 756 tests.

A summary of results for the ePlex BCID-GN Panel reproducibility study is provided in Table 9. For the negative panel member, $100 \%$ of replicates generated the expected negative results for all ePlex BCID-GN Panel analytes.

Table 9. Reproducibility of the ePlex BCID-GN Panel   

<table><tr><td colspan="1" rowspan="2">Analyte</td><td colspan="1" rowspan="2">ConcentrationTested</td><td colspan="1" rowspan="2">ExpectedResults</td><td colspan="4" rowspan="1">Agreement with Expected Result</td></tr><tr><td colspan="1" rowspan="1">Site 1</td><td colspan="1" rowspan="1">Site 2</td><td colspan="1" rowspan="1">Site 3</td><td colspan="1" rowspan="1">All Sites(95% CI)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="2" rowspan="1">Organisms</td><td></td><td></td><td></td><td></td></tr><tr><td colspan="1" rowspan="3">Acinetobacter baumannii</td><td colspan="1" rowspan="1">BP+8*1x109 CFU/mL</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">108/108(96.6-100)</td></tr><tr><td colspan="1" rowspan="1">BP**1x108 CFU/mL</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">108/108(96.6-100)</td></tr><tr><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">NotDetected</td><td colspan="1" rowspan="1">179/179</td><td colspan="1" rowspan="1">178/179</td><td colspan="1" rowspan="1">180/180</td><td colspan="1" rowspan="1">537/538(99.0-100)</td></tr><tr><td colspan="1" rowspan="3">Enterobacter cloacae complex</td><td colspan="1" rowspan="1">BP+81x109 CFU/mL</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">108/108(96.6-100)</td></tr><tr><td colspan="1" rowspan="1">BP1x108 CFU/mL</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">35/35</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">107/107(96.5-100)</td></tr><tr><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">NotDetected</td><td colspan="1" rowspan="1">180/180</td><td colspan="1" rowspan="1">179/179</td><td colspan="1" rowspan="1">180/180</td><td colspan="1" rowspan="1">539/539(99.3-100)</td></tr><tr><td colspan="1" rowspan="3">Escherichia coli</td><td colspan="1" rowspan="1">BP+81x109 CFU/mL</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">108/108(96.6-100)</td></tr><tr><td colspan="1" rowspan="1">BP1x108 CFU/mL</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">35/35</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">107/107(96.5-100)</td></tr><tr><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">NotDetected</td><td colspan="1" rowspan="1">179/179</td><td colspan="1" rowspan="1">180/180</td><td colspan="1" rowspan="1">180/180</td><td colspan="1" rowspan="1">539/539(99.3-100)</td></tr><tr><td colspan="1" rowspan="3">Fusobacterium nucleatum</td><td colspan="1" rowspan="1">BP+81x108 CFU/mL</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">35/35</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">107/107(96.5-100)</td></tr><tr><td colspan="1" rowspan="1">BP1x107 CFU/mL</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">108/108(96.6-100)</td></tr><tr><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">NotDetected</td><td colspan="1" rowspan="1">179/179</td><td colspan="1" rowspan="1">180/180</td><td colspan="1" rowspan="1">180/180</td><td colspan="1" rowspan="1">539/539(99.3-100)</td></tr><tr><td colspan="1" rowspan="3">Haemophilus influenzae</td><td colspan="1" rowspan="1">BP+81x109 CFU/mL</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">108/108(96.6-100)</td></tr><tr><td colspan="1" rowspan="1">BP1x108 CFU/mL</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">108/108(96.6-100)</td></tr><tr><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">NotDetected</td><td colspan="1" rowspan="1">179/179</td><td colspan="1" rowspan="1">179/179</td><td colspan="1" rowspan="1">180/180</td><td colspan="1" rowspan="1">538/538(96.6-100)</td></tr><tr><td colspan="1" rowspan="3">Klebsiella oxytoca</td><td colspan="1" rowspan="1">BP+81x109 CFU/mL</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">108/108(96.6-100)</td></tr><tr><td colspan="1" rowspan="1">BP1x108 CFU/mL</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">108/108(96.6-100)</td></tr><tr><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">NotDetected</td><td colspan="1" rowspan="1">179/179</td><td colspan="1" rowspan="1">179/179</td><td colspan="1" rowspan="1">180/180</td><td colspan="1" rowspan="1">538/538(96.6-100)</td></tr><tr><td colspan="1" rowspan="3">Neisseria meningitidis</td><td colspan="1" rowspan="1">BP+83x108 CFU/mL</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">35/35</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">107/107(96.5-100)</td></tr><tr><td colspan="1" rowspan="1">BP3x107 CFU/mL</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">108/108(96.6-100)</td></tr><tr><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">NotDetected</td><td colspan="1" rowspan="1">180/180</td><td colspan="1" rowspan="1">179/179</td><td colspan="1" rowspan="1">180/180</td><td colspan="1" rowspan="1">539/539(99.3-100)</td></tr><tr><td colspan="1" rowspan="3">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">BP+81x109 CFU/mL</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">35/35</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">107/107(96.5-100)</td></tr><tr><td colspan="1" rowspan="1">BP1x108 CFU/mL</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">108/108(96.6-100)</td></tr><tr><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">NotDetected</td><td colspan="1" rowspan="1">179/179</td><td colspan="1" rowspan="1">180/180</td><td colspan="1" rowspan="1">180/180</td><td colspan="1" rowspan="1">539/539(99.3-100)</td></tr><tr><td colspan="1" rowspan="3">Serratia</td><td colspan="1" rowspan="1">BP+81x109 CFU/mL</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">35/35</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">107/107(96.5-100)</td></tr><tr><td colspan="1" rowspan="1">BP1x108 CFU/mL</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">108/108(96.6-100)</td></tr><tr><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">NotDetected</td><td colspan="1" rowspan="1">180/180</td><td colspan="1" rowspan="1">179/179</td><td colspan="1" rowspan="1">180/180</td><td colspan="1" rowspan="1">539/539(99.3-100)</td></tr><tr><td colspan="1" rowspan="3">Serratia marcescens</td><td colspan="1" rowspan="1">BP+81x109 CFU/mL</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">35/35</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">107/107(96.5-100)</td></tr><tr><td colspan="1" rowspan="1">BP1x108 CFU/mL</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">108/108(96.6-100)</td></tr><tr><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">NotDetected</td><td colspan="1" rowspan="1">180/180</td><td colspan="1" rowspan="1">179/179</td><td colspan="1" rowspan="1">180/180</td><td colspan="1" rowspan="1">539/539(99.3-100)</td></tr><tr><td colspan="1" rowspan="3">Pan Candida</td><td colspan="1" rowspan="1">BP+81x107 CFU/m)</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">35/35</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">107/107(96.5-100)</td></tr><tr><td colspan="1" rowspan="1">BP1x106 CFU/mL</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">35/36</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">107/108(94.9-99.8)</td></tr><tr><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">NotDetected</td><td colspan="1" rowspan="1">180/180</td><td colspan="1" rowspan="1">179/179</td><td colspan="1" rowspan="1">180/180</td><td colspan="1" rowspan="1">539/539(99.3-100)</td></tr><tr><td colspan="1" rowspan="3">Pan Gram-Positive(Staphylococcus aureus)</td><td colspan="1" rowspan="1">BP+81x108 CFU/mL</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">34/36</td><td colspan="1" rowspan="1">35/35</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">105/107(93.4-99.5)</td></tr><tr><td colspan="1" rowspan="1">BP1x107 CFU/mL</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">108/108(96.6-100)</td></tr><tr><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">NotDetected</td><td colspan="1" rowspan="1">179/179</td><td colspan="1" rowspan="1">179/180</td><td colspan="1" rowspan="1">180/180</td><td colspan="1" rowspan="1">538/539(99.0-100)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Resistance Markers</td><td colspan="1" rowspan="1">s</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="3">CTX-M(Enterobacter cloacae)</td><td colspan="1" rowspan="1">BP+81x109 CFU/mL</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">108/108(96.6-100)</td></tr><tr><td colspan="1" rowspan="1">BP1x108 CFU/mL</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">35/35</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">107/107(96.5-100)</td></tr><tr><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">NotDetected</td><td colspan="1" rowspan="1">144/144</td><td colspan="1" rowspan="1">143/143</td><td colspan="1" rowspan="1">144/144</td><td colspan="1" rowspan="1">431/431(99.1-100)</td></tr><tr><td colspan="1" rowspan="3">IMP(Escherichia coli)</td><td colspan="1" rowspan="1">BP+81x109 CFU/mL</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">35/35</td><td colspan="1" rowspan="1">35/36</td><td colspan="1" rowspan="1">106/107(94.9-99.8)</td></tr><tr><td colspan="1" rowspan="1">BP1x108 CFU/mL</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">35/35</td><td colspan="1" rowspan="1">35/36</td><td colspan="1" rowspan="1">106/107(94.9-99.8)</td></tr><tr><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">NotDetected</td><td colspan="1" rowspan="1">143/143</td><td colspan="1" rowspan="1">144/144</td><td colspan="1" rowspan="1">144/144</td><td colspan="1" rowspan="1">431/431(99.1-100)</td></tr><tr><td colspan="1" rowspan="3">KPC(Enterobacter cloacae)</td><td colspan="1" rowspan="1">BP+81x109CFU/mL</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">108/108(96.6-100)</td></tr><tr><td colspan="1" rowspan="1">BP1x108 CFU/mL</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">35/35</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">107/107(96.5-100)</td></tr><tr><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">NotDetected</td><td colspan="1" rowspan="1">144/144</td><td colspan="1" rowspan="1">143/1433</td><td colspan="1" rowspan="1">144/144</td><td colspan="1" rowspan="1">431/431(99.1-100)</td></tr><tr><td colspan="1" rowspan="3">OXA(Acinetobacter baumannii)</td><td colspan="1" rowspan="1">BP+81x109CFU/mL</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">108/108(96.6-100)</td></tr><tr><td colspan="1" rowspan="1">BP1x108 CFU/mL</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">108/108(96.6-100)</td></tr><tr><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">NotDetected</td><td colspan="1" rowspan="1">143/143</td><td colspan="1" rowspan="1">143/143</td><td colspan="1" rowspan="1">144/144</td><td colspan="1" rowspan="1">430/430(99.1-100)</td></tr><tr><td colspan="1" rowspan="3">VIM(Pseudomonas aeruginosa)</td><td colspan="1" rowspan="1">BP+81x10 CFU/mL</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">35/35</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">107/107(96.5-100)</td></tr><tr><td colspan="1" rowspan="1">BP1x108 CFU/mL</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">108/108(96.6-100)</td></tr><tr><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">NotDetected</td><td colspan="1" rowspan="1">143/143</td><td colspan="1" rowspan="1">144/144</td><td colspan="1" rowspan="1">144/144</td><td colspan="1" rowspan="1">431/431(99.1-100)</td></tr></table>

\*BP +8 – Bottle Positivity $+ 8$ hours $\ast \ast _ { \mathrm { B P } }$ – Bottle Positivity

Five false positive results were observed in the BCID-GN Panel reproducibility study. Bacteroides fragilis (2), Acinetobacter baumannii, Fusobacterium necrophorum, and Pan Gram-Positive were detected in five different samples that had other organisms at positive concentrations. Four of the five samples with false positive results were observed at one testing site across two cartridge lots on four different days. On two of the days that false positive results were observed (i.e., Pan Gram-Positive and Acinetobacter baumannii targets), external controls containing those targets were also tested. These results suggest a potential site-specific root cause (e.g., site-specific contamination) rather than a systematic performance issue with the BCID-GN Panel.

During the study, 812 runs were attempted with six pre-flight check failures ( $1 . 2 \%$ preflight errors) for which a new cartridge had to be tested. The pre-flight check is an ePlex system process that tests the integrity of the cartridge/instrument interface prior to initiating sample processing. An additional 10 runs were aborted by the system $1 . 2 \%$ system abort errors) resulting in 796 completed runs, including repeat runs due to invalid results.

Of the 756 samples initially tested, 715 $( 9 4 . 6 \% )$ had valid results and 41 $( 5 . 4 \% )$ had invalid results. Of the 40 samples re-tested, 39 had valid results and 1 had a second invalid result upon re-test. The final validity rate $9 5 \%$ CI) was $9 9 . 7 \%$ $( 9 9 . 0 \% - 9 9 . 9 \% )$ .

b. Linearity/assay reportable range: Not applicable, qualitative assay.   
c. Traceability, Stability, Expected values (controls, calibrators, or methods):

# Assay Controls

Internal Controls

Each ePlex BCID-GN Panel cartridge includes internal controls that monitor performance of each step of the testing process, including extraction, amplification and detection of DNA targets.

Each of the eight amplification reactions has an internal control, and in each reaction, either the internal control or a target must generate signal above the defined threshold for a valid test result. Internal control results are interpreted by the ePlex Software and displayed on the ePlex BCID-GN Panel Reports as Internal Control with a result of PASS, FAIL, N/A, or INVALID. Table 10 includes details on the interpretation of Internal Control results.

Table 10: Internal control results   

<table><tr><td rowspan=1 colspan=1>Internal ControlResult</td><td rowspan=1 colspan=1>Explanation</td><td rowspan=1 colspan=1>Action</td></tr><tr><td rowspan=1 colspan=1>PASS</td><td rowspan=1 colspan=1>Signal above threshold has been detected from eachamplification reaction.The test was completed and internal controls weresuccessful, indicating valid results were generated.</td><td rowspan=1 colspan=1>All results are displayed on theePlex BCID-GN Panel DetectionReport.Test is valid, report results.</td></tr><tr><td rowspan=1 colspan=1>FAIL</td><td rowspan=1 colspan=1>Signal above threshold has not been detected from atleast one amplification reaction.The test was completed but internal controls were notdetected, indicating that results may not be valid.</td><td rowspan=1 colspan=1>No results are displayed on theePlex BCID-GN Panel DetectionReport.Test is not valid, repeat the testusing a new cartridge.</td></tr><tr><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>The internal control in every amplification reactiondoes not generate signal above the threshold, but atarget in every amplification reaction does generatesignal above the threshold.The test was completed and internal controls were notsuccessful, however detection of signal above thethreshold for a target in every amplification reactionindicates valid results were generated.</td><td rowspan=1 colspan=1>All results are displayed on theePlex BCID-GN Panel DetectionReport.Test is valid, report results.</td></tr><tr><td rowspan=1 colspan=1>INVALID</td><td rowspan=1 colspan=1>An error has occurred during processing that preventsanalysis of signal data.The test has not successfully completed and results forthis test are not valid. This may be due to aninstrument or software error.</td><td rowspan=1 colspan=1>No results are displayed on theePlex BCID-GN Panel DetectionReport.Test is not valid, repeat the testusing a new cartridge.</td></tr></table>

# External Controls

External controls are not provided with the ePlex BCID-GN Panel; however, the package insert includes the following recommendations for external controls:

Positive and negative external controls should be tested with each new lot of reagents or monthly, whichever occurs first. Blood culture medium can be used as the negative control. Previously characterized positive samples or blood culture medium spiked with well characterized organisms can be used as the external positive control. External controls should be run in accordance with laboratory protocols and accrediting organizations, as applicable.

During the prospective clinical evaluation of the ePlex BCID-GN Panel, each testing site was provided with positive and negative external control samples. There were four organism mixes that served as positive controls and were prepared at concentrations near or below the concentration expected at bottle positivity. Negative blood culture matrix was used as the negative control. At least one positive and one negative external control were tested each day and cycled throughout the clinical study. The control mixes are described in Table 11.

Table 11. External controls used in the BCID-GN Panel Clinical study   

<table><tr><td rowspan=1 colspan=1>External Controls</td><td rowspan=1 colspan=1>Expected Calls</td></tr><tr><td rowspan=1 colspan=1>Mix A</td><td rowspan=1 colspan=1>CTX-M-1, Staphylococcus, Escherichia coli, Proteus mirabilis,Pseudomonas aeruginosa, Salmonella enterica, Serratia marcescens, E.cloacae complex, Enterobacter aerogenes</td></tr><tr><td rowspan=1 colspan=1>Mix B</td><td rowspan=1 colspan=1>Bacillus cereus, IMP, OXA-23, Candida albicans, Citrobacter freundii,Serratia proteamaculans, Neisseria meningitidis, Enterobacter amnigenus,Proteus vulgaris</td></tr><tr><td rowspan=1 colspan=1>Mix C</td><td rowspan=1 colspan=1>Bacteroides fragilis, OXA-48, Acinetobacter baumannii, Enterococcusspp., Fusobacterium necrophorum, Morganella morganii, Klebsiellaoxytoca, VIM</td></tr><tr><td rowspan=1 colspan=1>Mix D</td><td rowspan=1 colspan=1>Cronobacter sakazakii, Stenotrophomonas maltophilia, Streptococcus ssp.,Enterobacter asburiae, Haemophilus influenzae, Klebsiella pneumoniae,NDM, Fusobacterium nucleatum, KPC</td></tr><tr><td rowspan=1 colspan=1>Negative control</td><td rowspan=1 colspan=1>Negative (Not Detected)</td></tr></table>

Over 90 days of investigational testing across 6 testing sites, there were 177 external controls tested with valid results: 25 Positive Control A, 22 Positive Control B, 21 Positive Control C, 21 Positive Control D, and 88 Negative Controls. Of these, 167/177 controls generated the expected results. The remaining 10 external control samples generated discordant results as follows:

3 external control samples had 1 or more false negative (FN) results:

• 1 Positive Control A sample had 1 FN Pseudomonas aeruginosa result. • 1 Positive Control A sample had 1 FN Enterobacter (non-cloacae complex) • result. • 1 Positive Control A sample had 2 FN results: Proteus mirabilis, Proteus.

7 external control samples had 1 false positive (FP) result:

• 1 Negative Control sample had a 1 FP Pan Gram-Positive result.   
• 2 Positive Control B samples each had 1 FP Salmonella.   
• 2 Positive Control C samples each had 1 NDM FP result.   
• 1 Positive Control C sample had 1 Enterobacter cloacae complex FP result.   
• 1 Positive Control D sample had 1 VIM FP result.

# Specimen stability

Clinical specimen stability claims were supported by an analytical study that investigated the effect of different specimen storage conditions/time periods on specimen integrity for specimens tested with the ePlex BCID-GN Panel. Three organism mixes containing representative targets from each of the eight multiplex primer pools on the ePlex BCIDGN Panel were used for this study. Organism mixes whose members were at concentrations consistent with those observed at bottle positivity were stored at $\underline { { < } } \mathrm { - } 8 0 ^ { \circ } \mathrm { C }$ , $\mathrm { \leq - 2 0 ^ { \circ } C }$ , $2 { - } 8 ^ { \circ } \mathrm { C }$ , and at ambient temperature over several time points.

Baseline condition $\left( \mathrm { T } _ { 0 } \right)$ was established with 20 replicates. Ten replicates were tested for each additional temperature and time point. All test conditions that were assessed in this study are summarized in Table 12.

Table 12. Specimen stability study design   

<table><tr><td rowspan=1 colspan=1>Storage Condition</td><td rowspan=1 colspan=8>Time Points</td></tr><tr><td rowspan=1 colspan=1>Ambient Conditions15°C - 30°</td><td rowspan=1 colspan=1>T0</td><td rowspan=1 colspan=1>1day</td><td rowspan=1 colspan=1>3days</td><td rowspan=1 colspan=1>5days</td><td rowspan=1 colspan=1>7days</td><td rowspan=1 colspan=1>14days</td><td rowspan=1 colspan=1>30days</td><td rowspan=1 colspan=1>--</td></tr><tr><td rowspan=1 colspan=1>Refrigerated Conditions28</td><td rowspan=1 colspan=1>--</td><td rowspan=1 colspan=1>1day</td><td rowspan=1 colspan=1>3days</td><td rowspan=1 colspan=1>5days</td><td rowspan=1 colspan=1>7days</td><td rowspan=1 colspan=1>14days</td><td rowspan=1 colspan=1>30days</td><td rowspan=1 colspan=1>--</td></tr><tr><td rowspan=1 colspan=1>Frozen Conditions≤-20°C and ≤-80°</td><td rowspan=1 colspan=1>1 w</td><td rowspan=1 colspan=1>2 w</td><td rowspan=1 colspan=1>1mo</td><td rowspan=1 colspan=1>3mo</td><td rowspan=1 colspan=1>6mo</td><td rowspan=1 colspan=1>12mo</td><td rowspan=1 colspan=1>18mo</td><td rowspan=1 colspan=1>24mo</td></tr><tr><td rowspan=1 colspan=1>Bottle Incubator</td><td rowspan=1 colspan=2>Bottlepositivity</td><td rowspan=1 colspan=2>12-hours postpositivity</td><td rowspan=1 colspan=4></td></tr></table>

The specimen stability criteria were as follows:

Performance at each time point will be considered acceptable if the positivity rate for each analyte (either from the first 10 replicates or from the final 20 replicates in case of additional testing) is equivalent to the positivity rate for the corresponding analyte at time point $\mathrm { T } _ { 0 }$ (baseline time point). If a time point did not meet the positivity acceptance criteria, the next time point was evaluated. If the subsequent time point met the acceptance criteria, the study was continued. If two consecutive time points failed, the specimen stability was determined to be the last time point when the acceptance criteria were met.

Summaries of the specimen stability assessment data are in Tables 12-14:

Table 12. Ambient storage sample stability (Percent positivity and mean signal)   

<table><tr><td colspan="1" rowspan="2">Target</td><td colspan="2" rowspan="1">Day 0</td><td colspan="2" rowspan="1">Day 1</td><td colspan="2" rowspan="1">Day 3</td><td colspan="2" rowspan="1">Day 5</td><td colspan="2" rowspan="1">Day 7</td><td colspan="2" rowspan="1">Day 14</td><td colspan="2" rowspan="1">Day 30</td></tr><tr><td colspan="1" rowspan="1">%Pos</td><td colspan="1" rowspan="1">MeannA</td><td colspan="1" rowspan="1">%Pos</td><td colspan="1" rowspan="1">MeannA</td><td colspan="1" rowspan="1">%Pos</td><td colspan="1" rowspan="1">MeannA</td><td colspan="1" rowspan="1">%Pos</td><td colspan="1" rowspan="1">MeannA</td><td colspan="1" rowspan="1">%Pos</td><td colspan="1" rowspan="1">MeannA</td><td colspan="1" rowspan="1">%Pos</td><td colspan="1" rowspan="1">MeannA</td><td colspan="1" rowspan="1">%Pos</td><td colspan="1" rowspan="1">MeannA</td></tr><tr><td colspan="15" rowspan="1">Mix 1</td></tr><tr><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">391.5</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">597.8</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">673.5</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">72.0</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">725.8</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">754.9</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">758</td></tr><tr><td colspan="1" rowspan="1">H. influenzae</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">243.9</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">259.3</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">281.3</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">233.0</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">246.3</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">251.5</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">231.3</td></tr><tr><td colspan="1" rowspan="1">E. cloacaeccomplex</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">230.1</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">516.3</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">723.4</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">818.4</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">886.6</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">937.3</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">919.2</td></tr><tr><td colspan="1" rowspan="1">Pan-Candida</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">233.1</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">247.0</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">217.8</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">247.0</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">184.7</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">233.7</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">245.3</td></tr><tr><td colspan="1" rowspan="1">OXA</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">735.7</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">746.1</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">726.0</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">816.5</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">795.2</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">823.0</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">824.0</td></tr><tr><td colspan="1" rowspan="1">CTX-M</td><td colspan="1" rowspan="1">95</td><td colspan="1" rowspan="1">292.9</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">333.5</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">340.2</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">310.4</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">350.4</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">337.9</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">387.8</td></tr><tr><td colspan="15" rowspan="1">Mix 2</td></tr><tr><td colspan="1" rowspan="1">K. oxytoca</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">489.9</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">513.8</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">424.9</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">418.3</td><td colspan="1" rowspan="1">100#</td><td colspan="1" rowspan="1">492.5</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">532.7</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">512.2</td></tr><tr><td colspan="1" rowspan="1">P. aeruginosa</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">586.8</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">517.3</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">739</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">781.9</td><td colspan="1" rowspan="1">100#</td><td colspan="1" rowspan="1">783.4</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">524.7</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">800.4</td></tr><tr><td colspan="1" rowspan="1">A. baumannii</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">783.1</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">788.8</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">727.8</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">760.4</td><td colspan="1" rowspan="1">100#</td><td colspan="1" rowspan="1">759.5</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">799.9</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">803.1</td></tr><tr><td colspan="1" rowspan="1">E. non-cloacaecomplex</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">566.4</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">519.7</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">633.5</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">639</td><td colspan="1" rowspan="1">100#</td><td colspan="1" rowspan="1">669.6</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">472.3</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">587.8</td></tr><tr><td colspan="1" rowspan="1">IMP</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">492</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">491.5</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">479.2</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">457.5</td><td colspan="1" rowspan="1">100#</td><td colspan="1" rowspan="1">447.2</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">292.8</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">473.8</td></tr><tr><td colspan="1" rowspan="1">N. meningitidis</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">306</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">349.9</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">307.6</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">235.6</td><td colspan="1" rowspan="1">100^</td><td colspan="1" rowspan="1">277.8</td><td colspan="1" rowspan="1">100*</td><td colspan="1" rowspan="1">264.7</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">282.9</td></tr><tr><td colspan="1" rowspan="1">S. marcescens</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">479</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">530.7</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">498.9</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">545</td><td colspan="1" rowspan="1">100^</td><td colspan="1" rowspan="1">397.8</td><td colspan="1" rowspan="1">100*</td><td colspan="1" rowspan="1">368.1</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">375.6</td></tr><tr><td colspan="1" rowspan="1">B. fragilis</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">712.6</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">730.5</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">713.2</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">682.9</td><td colspan="1" rowspan="1">100^</td><td colspan="1" rowspan="1">739.8</td><td colspan="1" rowspan="1">100*</td><td colspan="1" rowspan="1">767.3</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">903.8</td></tr><tr><td colspan="1" rowspan="1">Pan Gram-Positive</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">461.3</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">511.4</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">625.5</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">633.4</td><td colspan="1" rowspan="1">100^</td><td colspan="1" rowspan="1">590.3</td><td colspan="1" rowspan="1">100*</td><td colspan="1" rowspan="1">544.7</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">489.6</td></tr></table>

# Pan Gram-Positive false positive observed in 1 of 10 replicates ^ Mix 3; M. morganii and OXA-48 false positive observed in 1 of 20 replicates $^ { * } \operatorname { M i x } 3 ; P$ . aeruginosa false positive observed in 1 of 20 replicates

Table 13. Refrigerated $( 2 { \cdot } 8 ^ { \circ } \mathbf { C } )$ storage sample stability (Percent positivity and mean signal)   

<table><tr><td rowspan=2 colspan=1>Target</td><td rowspan=1 colspan=2>Day 0</td><td rowspan=1 colspan=2>Day 1</td><td rowspan=1 colspan=2>Day 3</td><td rowspan=1 colspan=2>Day 5</td><td rowspan=1 colspan=2>Day 7</td><td rowspan=1 colspan=2>Day 14</td><td rowspan=1 colspan=2>Day 30</td></tr><tr><td rowspan=1 colspan=1>%Pos</td><td rowspan=1 colspan=1>MeannA</td><td rowspan=1 colspan=1>%Pos</td><td rowspan=1 colspan=1>MeannA</td><td rowspan=1 colspan=1>%Pos</td><td rowspan=1 colspan=1>MeannA</td><td rowspan=1 colspan=1>%Pos</td><td rowspan=1 colspan=1>MeannA</td><td rowspan=1 colspan=1>%Pos</td><td rowspan=1 colspan=1>MeannA</td><td rowspan=1 colspan=1>%Pos</td><td rowspan=1 colspan=1>MeannA</td><td rowspan=1 colspan=1>%Pos</td><td rowspan=1 colspan=1>MeannA</td></tr><tr><td rowspan=1 colspan=15>Mix 1</td></tr><tr><td rowspan=1 colspan=1>E. coli</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>391.5</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>370.8</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>461.3</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>418.9</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>396.8</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>473</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>480.6</td></tr><tr><td rowspan=1 colspan=1>H. influenzae</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>243.9</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>226.6</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>308.2</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>279.2</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>268.7</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>285.1</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>218</td></tr><tr><td rowspan=1 colspan=1>E. cloacaecomplex</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>230.1</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>239.8</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>244.8</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>185.2</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>274.4</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>359.1</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>320.1</td></tr><tr><td rowspan=1 colspan=1>Pan-Candida</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>233.1</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>220</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>214.7</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>227.2</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>186.7</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>217.2</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>259.1</td></tr><tr><td rowspan=1 colspan=1>OXA</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>735.7</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>752.4</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>744.6</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>756.6</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>770.5</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>803.5</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>852.2</td></tr><tr><td rowspan=1 colspan=1>CTX-M</td><td rowspan=1 colspan=1>95</td><td rowspan=1 colspan=1>292.9</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>308.9</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>367.9</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>339.5</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>312.3</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>318.2</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>318.4</td></tr><tr><td rowspan=1 colspan=15>Mix 2</td></tr><tr><td rowspan=1 colspan=1>K. oxytoca</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>489.9</td><td rowspan=1 colspan=1>100^</td><td rowspan=1 colspan=1>413</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>411.9</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>360.7</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>504.6</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>518</td><td rowspan=1 colspan=1>100*</td><td rowspan=1 colspan=1>461.2</td></tr><tr><td rowspan=1 colspan=1>P. aeruginosa</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>586.8</td><td rowspan=1 colspan=1>100^</td><td rowspan=1 colspan=1>457.1</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>755.7</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>627</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>734.9</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>388.6</td><td rowspan=1 colspan=1>100*</td><td rowspan=1 colspan=1>697.1</td></tr><tr><td rowspan=1 colspan=1>A. baumannii</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>783.1</td><td rowspan=1 colspan=1>100^</td><td rowspan=1 colspan=1>763.4</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>762.9</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>655.4</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>760.7</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>814.7</td><td rowspan=1 colspan=1>100*</td><td rowspan=1 colspan=1>691.3</td></tr><tr><td rowspan=1 colspan=1>E. non-cloacaecomplex</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>566.4</td><td rowspan=1 colspan=1>100^</td><td rowspan=1 colspan=1>519.3</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>661.8</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>553.7</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>660.2</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>388.2</td><td rowspan=1 colspan=1>100*</td><td rowspan=1 colspan=1>497.3</td></tr><tr><td rowspan=1 colspan=1>IMP</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>492</td><td rowspan=1 colspan=1>100^</td><td rowspan=1 colspan=1>404.8</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>490.7</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>431.8</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>467.3</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>244.1</td><td rowspan=1 colspan=1>95*</td><td rowspan=1 colspan=1>443.4</td></tr><tr><td rowspan=1 colspan=15>Mix 3</td></tr><tr><td rowspan=1 colspan=1>N. meningitidis</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>306</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>327.9</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>305.3</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>260.1</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>300.3</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>274.4</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>316.5</td></tr><tr><td rowspan=1 colspan=1>S. marcescens</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>479</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>526.9</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>442.1</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>486</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>337.1</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>303</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>309.5</td></tr><tr><td rowspan=1 colspan=1>B. fragilis</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>712.6</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>758.4</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>719</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>714.6</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>747.5</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>739.6</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>875</td></tr><tr><td rowspan=1 colspan=1>Pan Gram-</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>461.2</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>500.4</td><td rowspan=1 colspan=1>100.</td><td rowspan=1 colspan=1>551.5</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>(12.4</td><td rowspan=1 colspan=1>100.</td><td rowspan=1 colspan=1>526 1</td><td rowspan=1 colspan=1>100*</td><td rowspan=1 colspan=1>542.5</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>500.</td></tr></table>

^ Mix 2, Proteus false positive detected in 1 of 20 replicates \* Mix 2 had 20 replicates

Table 14. Frozen $( \leq - 2 0 ^ { \circ } \mathbf { C }$ and $\mathrm { \le - 7 0 ^ { \circ } C }$ ) Storage Sample Stability (Percent Positivity and Mean Signal)   

<table><tr><td colspan="1" rowspan="2">Target</td><td colspan="2" rowspan="1">Day 0</td><td colspan="2" rowspan="1">Week 1</td><td colspan="2" rowspan="1">Week 2</td><td colspan="2" rowspan="1">Month 1</td><td colspan="2" rowspan="1">Month 3</td></tr><tr><td colspan="1" rowspan="1">%Pos</td><td colspan="1" rowspan="1">MeannA</td><td colspan="1" rowspan="1">%Pos</td><td colspan="1" rowspan="1">MeannA</td><td colspan="1" rowspan="1">%Pos</td><td colspan="1" rowspan="1">MeannA</td><td colspan="1" rowspan="1">%Pos</td><td colspan="1" rowspan="1">MeannA</td><td colspan="1" rowspan="1">%Pos</td><td colspan="1" rowspan="1">MeannA</td></tr><tr><td colspan="11" rowspan="1">≤-20°C Condition</td></tr><tr><td colspan="11" rowspan="1">Mix 1</td></tr><tr><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">391.5</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">534.7</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">478.7</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">547.2</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">377.1</td></tr><tr><td colspan="1" rowspan="1">H. influenzae</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">243.9</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">312.1</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">256.9</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">206.8</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">314.4</td></tr><tr><td colspan="1" rowspan="1">E. cloacaecomplex</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">230.1</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">382.2</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">298.5</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">285.5</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">278.2</td></tr><tr><td colspan="1" rowspan="1">Pan-Candida</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">233.1</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">182.9</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">235.1</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">246.7</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">178.9</td></tr><tr><td colspan="1" rowspan="1">OXA</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">735.7</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">804.2</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">790</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">801</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">771.5</td></tr><tr><td colspan="1" rowspan="1">CTX-M</td><td colspan="1" rowspan="1">95</td><td colspan="1" rowspan="1">292.9</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">339.6</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">293.3</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">351.5</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">268.3</td></tr><tr><td colspan="11" rowspan="1">Mix 2</td></tr><tr><td colspan="1" rowspan="1">K. oxytoca</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">489.9</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">422.3</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">525.2</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">445.9</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">505.4</td></tr><tr><td colspan="1" rowspan="1">P. aeruginosa</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">586.8</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">775.7</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">416.6</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">532.4</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">427.1</td></tr><tr><td colspan="1" rowspan="1">A. baumannii</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">783.1</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">738.2</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">826.2</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">767.6</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">529.5</td></tr><tr><td colspan="1" rowspan="1">E. non-cloacaecomplex</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">566.4</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">698.3</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">534.6</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">541.7</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">489.1</td></tr><tr><td colspan="1" rowspan="1">IMP</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">492</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">492.4</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">274.4</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">489.8</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">289.4</td></tr><tr><td colspan="11" rowspan="1">Mix 3</td></tr><tr><td colspan="1" rowspan="1">N. meningitidis</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">306</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">336.4</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">244.4</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">279.5</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">436.4</td></tr><tr><td colspan="1" rowspan="1">S. marcescens</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">479</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">307.9</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">300.7</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">310</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">499.4</td></tr><tr><td colspan="1" rowspan="1">B. fragilis</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">712.6</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">798.5</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">725</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">919.6</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">599.6</td></tr><tr><td colspan="1" rowspan="1">Pan Gram-Positive</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">461.3</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">559.5</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">486.5</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">541</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">381.2</td></tr><tr><td colspan="11" rowspan="1">≤-70° Condition</td></tr><tr><td colspan="11" rowspan="1">Mix 1</td></tr><tr><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">391.5</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">538.5</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">502.6</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">410.2</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">290.9</td></tr><tr><td colspan="1" rowspan="1">H. influenzae</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">243.9</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">304.6</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">261.5</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">232.8</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">324</td></tr><tr><td colspan="1" rowspan="1">E. cloacaecomplex</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">230.1</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">367</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">296.8</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">210.6</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">275.6</td></tr><tr><td colspan="1" rowspan="1">Pan-Candida</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">233.1</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">190.7</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">222.6</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">271.4</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">146.5</td></tr><tr><td colspan="1" rowspan="1">OXA</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">735.7</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">798.2</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">784.4</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">770.7</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">757.4</td></tr><tr><td colspan="1" rowspan="1">CTX-M</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">292.9</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">298.4</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">286.9</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">296.8</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">256.7</td></tr><tr><td colspan="11" rowspan="1">Mix 2</td></tr><tr><td colspan="1" rowspan="1">K. oxytoca</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">489.9</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">439.4</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">488.6</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">435.3</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">393.2</td></tr><tr><td colspan="1" rowspan="1">P. aeruginosa</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">586.8</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">790.9</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">403.4</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">604.8</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">315.2</td></tr><tr><td colspan="1" rowspan="1">A. baumannii</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">783.1</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">704.6</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">820.7</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">732.4</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">374.3</td></tr><tr><td colspan="1" rowspan="1">E. non-cloacaecomplex</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">566.4</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">664.6</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">501</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">528.2</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">368.8</td></tr><tr><td colspan="1" rowspan="1">IMP</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">492</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">476.1</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">268.8</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">482.1</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">304.2</td></tr><tr><td colspan="11" rowspan="1">Mix 3</td></tr><tr><td colspan="1" rowspan="1">N. meningitidis</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">306</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">353.9</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">226.2</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">271.3</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">448.5</td></tr><tr><td colspan="1" rowspan="1">S. marcescens</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">479</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">324.7</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">277</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">359.3</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">428.8</td></tr><tr><td colspan="1" rowspan="1">B. fragilis</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">712.6</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">792.1</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">681.1</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">926.2</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">587.6</td></tr><tr><td colspan="1" rowspan="1">Pan Gram-Positive</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">461.3</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">575.5</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">480.7</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">562.8</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">412</td></tr></table>

The specimen stability study demonstrated that positive blood culture specimens can be stored up to 30 days at both ambient and refrigerated $\left( 2 \mathrm { - } 8 ^ { \circ } \mathrm { C } \right)$ temperatures as well as up to three months at both $- 2 0 ^ { \circ } \mathrm { C }$ and $- 7 0 \mathrm { { } ^ { \circ } C }$ without negatively impacting ePlex BCID-GN Panel performance. Study results also demonstrated that specimens can also be tested after incubation in a blood culture instrument for up to 12 hours post-bottle positivity.

# Freeze-thaw Study

An analytical study was conducted to demonstrate the stability of clinical specimens after subjecting samples to one or two freeze-thaw cycles prior to testing on the BCID-GN Panel. One hundred and ten positive clinical samples representing 20 targeted analytes were tested fresh during the BCID-GN clinical study and selected to represent a variety of organisms present in positive blood cultures. All specimens contained gram-negative organisms on Gram stain. Samples were frozen at $\mathrm { \leq } \mathrm { - } 7 0 ^ { \circ } \mathrm { C }$ and then subjected to one or two thawing cycles, followed by testing with the BCID-GN Panel. Study results are summarized in Table 15.

Table 15. Freeze/Thaw study (Positivity and mean signal)   

<table><tr><td rowspan=2 colspan=1>Target</td><td rowspan=2 colspan=1>TotalReplicates</td><td rowspan=1 colspan=2>Fresh (T0)</td><td rowspan=1 colspan=2>1X Freeze/Thaw</td><td rowspan=1 colspan=2>2X Freeze/Thaw</td></tr><tr><td rowspan=1 colspan=1>% Pos</td><td rowspan=1 colspan=1>Mean nA</td><td rowspan=1 colspan=1>% Pos</td><td rowspan=1 colspan=1>Mean nA</td><td rowspan=1 colspan=1>% Pos</td><td rowspan=1 colspan=1>Mean nA</td></tr><tr><td rowspan=1 colspan=1>Bacteroides fragilis</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>520.9</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>606.0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>567.4</td></tr><tr><td rowspan=1 colspan=1>Citrobacter</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>364.6</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>643.3</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>537.8</td></tr><tr><td rowspan=1 colspan=1>Enterobacter (non-cloacae complex)</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>553.1</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>655.1</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>768.7</td></tr><tr><td rowspan=1 colspan=1>Enterobacter cloacaecomplexA</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>487.0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>706.2</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>654.8</td></tr><tr><td rowspan=1 colspan=1>Escherichia coli</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>883.1</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>684.4</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>712.6</td></tr><tr><td rowspan=1 colspan=1>Haemophilus influenzae</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>378.2</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>468.0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>427.3</td></tr><tr><td rowspan=1 colspan=1>Klebsiella oxytoca</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>362.8</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>514.1</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>562.0</td></tr><tr><td rowspan=1 colspan=1>Klebsiella pneumoniaeA</td><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>439.7</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>467.7</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>534.0</td></tr><tr><td rowspan=1 colspan=1>Morganella morganii</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>556.6</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>522.5</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>567.9</td></tr><tr><td rowspan=1 colspan=1>Proteus</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>133.8</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>185.7</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>140.1</td></tr><tr><td rowspan=1 colspan=1>Proteus mirabilis</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>246.7</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>290.0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>279.8</td></tr><tr><td rowspan=1 colspan=1>Pseudomonas aeruginosaA</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>609.9</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>650.5</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>645.3</td></tr><tr><td rowspan=1 colspan=1>Salmonella</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>93.1</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>94.3</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>101.0</td></tr><tr><td rowspan=1 colspan=1>Serratia</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>146.1</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>148.4</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>143.7</td></tr><tr><td rowspan=1 colspan=1>Serratia marcescens</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>206.3</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>378.4</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>389.1</td></tr><tr><td rowspan=1 colspan=1>Stenotrophomonasmaltophilia</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>367.8</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>495.5</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>568.4</td></tr><tr><td rowspan=1 colspan=1>KPC*</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>713.9</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>797.7</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>837.6</td></tr><tr><td rowspan=1 colspan=1>CTX-M</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>273.6</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>348.8</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>351.8</td></tr><tr><td rowspan=1 colspan=1>Pane CandidaA</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>162.4</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>153.4</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>188.9</td></tr><tr><td rowspan=1 colspan=1>Pan Gram-Positive</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>475.5</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>537.6</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>614.7</td></tr></table>

as only one sample tested for the KPC target. Therefore, the nA value is the actual observed value.   
our samples had false positive results but all expected targets were detected. i. Under the 1X freeze-thaw condition, one positive Enterobacter cloacae complex sample detected Enterobacter cloacae complex and OXA, making OXA a false positive result. ii. Under the 1X freeze-thaw condition, one positive Pan Candida sample detected Pan Candida and Pan Gram-Positive, making Pan Gram-Positive a false positive result.   
iii. Under the 2X freeze-thaw condition, one positive Pseudomonas aeruginosa sample detected Pseudomonas aeruginosa and Stenotrophomonas maltophilia, making Stenotrophomonas maltophilia a false positive result.   
iv. Under the 2X freeze-thaw condition, one positive Klebsiella pneumoniae sample detected Klebsiella pneumoniae and OXA, making OXA a false positive result.

Data from the freeze-thaw study support that clinical specimens are stable up to two freeze-thaw cycles and frozen specimens and previously frozen specimens are acceptable

for testing with the ePlex BCID-GN Panel.

# Open Pouch Stability Study In-Consumable Sample Stability Study

An analytical study was conducted to demonstrate that the ePlex BC-GN cartridge (after the pouch has been opened) can be exposed to ambient conditions for up to two hours before loading the sample and testing on the ePlex instrument.

A second study was conducted to demonstrate that the cartridge can be stored at room temperature $( 1 5 { - } 3 0 ^ { \circ } \mathrm { C } )$ for two hours after the sample has been loaded but prior to running the cartridge in an ePlex bay.

For these two studies, the test panel was comprised of four organisms representing six of the eight multiplex primer pools and covering six targets on the ePlex BCID-GN Panel. Each organism was tested at a concentration approximating bottle positivity.

A total of 20 control replicates were also run for each organism. Control samples were loaded within 20 minutes after pouch opening and were run on the ePlex instrument within 20 minutes of sample loading.

The open pouch stability study demonstrated that ePlex BCID-GN cartridges are stable for up to 2 hours at room temperature after the pouch has been opened and prior to testing on the ePlex instrument. Results from the study are presented in Table 16.

Table 16: Open pouch stability study results   

<table><tr><td rowspan=2 colspan=1>Target</td><td rowspan=1 colspan=2>% Positive</td></tr><tr><td rowspan=1 colspan=1>Sample tested within20min after pouchopening</td><td rowspan=1 colspan=1>Sample tested two hoursafter pouch opening</td></tr><tr><td rowspan=1 colspan=1>Escherichia coli</td><td rowspan=1 colspan=1>100% (20/20)</td><td rowspan=1 colspan=1>100% (20/20)</td></tr><tr><td rowspan=1 colspan=1>Enterobacter cloacae complex</td><td rowspan=1 colspan=1>100% (20/20)</td><td rowspan=1 colspan=1>95% (19/20)</td></tr><tr><td rowspan=1 colspan=1>Haemophilus influenzae</td><td rowspan=1 colspan=1>100% (20/20)</td><td rowspan=1 colspan=1>100% (20/20)</td></tr><tr><td rowspan=1 colspan=1>Pan-Candida</td><td rowspan=1 colspan=1>100% (20/20)</td><td rowspan=1 colspan=1>95% (19/20)</td></tr><tr><td rowspan=1 colspan=1>CTX-M</td><td rowspan=1 colspan=1>95% (19/20)</td><td rowspan=1 colspan=1>100% (20/20)</td></tr><tr><td rowspan=1 colspan=1>OXA</td><td rowspan=1 colspan=1>100% (20/20)</td><td rowspan=1 colspan=1>100% (20/20)</td></tr></table>

The in-cartridge stability study demonstrated that the BCID-GN cartridge is stable for up to 2 hours at room temperature after the sample has been loaded prior to testing with the ePlex BCID-GN Panel. Results of the study are summarized in Table 17.

Table 17: In-Cartridge sample stability study results   

<table><tr><td colspan="1" rowspan="2">Target</td><td colspan="2" rowspan="1">% Positive</td></tr><tr><td colspan="1" rowspan="1">Sample loaded in ≤20minutes</td><td colspan="1" rowspan="1">Sample loaded at 120minutes</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">100% (20/20)</td><td colspan="1" rowspan="1">100% (20/20)</td></tr><tr><td colspan="1" rowspan="1">Enterobacter cloacae complex</td><td colspan="1" rowspan="1">100% (20/20)</td><td colspan="1" rowspan="1">100% (20/20)</td></tr><tr><td colspan="1" rowspan="1">Haemophilus influenzae</td><td colspan="1" rowspan="1">100% (20/20)</td><td colspan="1" rowspan="1">100% (20/20)</td></tr><tr><td colspan="1" rowspan="1">Pan-Candida</td><td colspan="1" rowspan="1">100% (20/20)</td><td colspan="1" rowspan="1">100% (20/20)</td></tr><tr><td colspan="1" rowspan="1">CTX-M</td><td colspan="1" rowspan="1">95% (19/20)</td><td colspan="1" rowspan="1">95% (19/20)</td></tr><tr><td colspan="1" rowspan="1">OXA</td><td colspan="1" rowspan="1">100% (20/20)</td><td colspan="1" rowspan="1">100% (20/20)</td></tr></table>

# d. Bottle Positivity: Organism Concentration Study

To approximate organism concentrations that may be present in positive blood culture bottles in a clinical setting, a study was conducted to establish the range of expected organism concentrations present in incubated blood cultures at bottle positivity (i.e., bottle ring) and eight hours after bottle positivity/bottle “ring.” Representative organisms were evaluated, consisting of 14 genera that are detected by the ePlex BCID-GN Panel. The volume of negative human whole blood, determined by the manufacturer’s instructions, was inoculated into each organism-appropriate bottle and incubated to bottle positivity and bottle positivity plus eight hours on a continuously monitoring blood culture system. At the time of positivity (and eight hours after positivity), the blood culture was removed from the instrument for determination of organism concentration (CFU/mL using a standard plate counting method) and ePlex BCID Panel testing. All estimated bottle positivity concentrations are equivalent or greater than the established Limit of Detection (LOD) for each of the assays of the ePlex BCID-GP Panel. The following bottle types were used for the study: BD BACTEC Plus Aerobic/F blood culture bottle (E. faecium, S. aureus, S. anginosus, A. baumannii, E. cloacae, $E .$ . coli, H. influenzae, K. oxytoca, N. meningitidis, $P .$ . aeruginosa, and S. marcescens), and BD BACTEC Lytic/10 Anaerobic/F (B. fragilis and $F$ . nucleatum). Table 18 summarizes the concentration present at bottle positivity and bottle positivity plus eight hours for each organism evaluated.

Table 18: Bottle positivity study, organism concentrations   

<table><tr><td colspan="1" rowspan="1">Organism</td><td colspan="1" rowspan="1">Average BPConcentration(CFU/mL)</td><td colspan="1" rowspan="1">Average BP+8hrsConcentration (CFU/mL)</td></tr><tr><td colspan="1" rowspan="1">Serratia marcescens</td><td colspan="1" rowspan="1">1.2x109</td><td colspan="1" rowspan="1">2.2 x109</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">2.3 x108</td><td colspan="1" rowspan="1">1.5 x109</td></tr><tr><td colspan="1" rowspan="1">Enterobacter cloacae</td><td colspan="1" rowspan="1">2.8 x108</td><td colspan="1" rowspan="1">7.7 x108</td></tr><tr><td colspan="1" rowspan="1">Neisseria meningitidis</td><td colspan="1" rowspan="1">3.2 x107</td><td colspan="1" rowspan="1">2.1 x108</td></tr><tr><td colspan="1" rowspan="1">Haemophilus influenzae</td><td colspan="1" rowspan="1">6.9 x108</td><td colspan="1" rowspan="1">1.15 x109</td></tr><tr><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">1.6 x108</td><td colspan="1" rowspan="1">8.4 x108</td></tr><tr><td colspan="1" rowspan="1">Fusobacterium nucleatum</td><td colspan="1" rowspan="1">6.5 x107</td><td colspan="1" rowspan="1">4.9 x108</td></tr><tr><td colspan="1" rowspan="1">Acinetobacter baumannii</td><td colspan="1" rowspan="1">4.4 x108</td><td colspan="1" rowspan="1">3.8 x108</td></tr><tr><td colspan="1" rowspan="1">Klebsiella oxytoca</td><td colspan="1" rowspan="1">9.3 x108</td><td colspan="1" rowspan="1">1.5 x109</td></tr><tr><td colspan="1" rowspan="1">Bacteroides fragilis</td><td colspan="1" rowspan="1">4.7x108</td><td colspan="1" rowspan="1">6.7x109</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">2.8 x107</td><td colspan="1" rowspan="1">2.1 x107</td></tr><tr><td colspan="1" rowspan="1">Streptococcus anginosus</td><td colspan="1" rowspan="1">4.1 x107</td><td colspan="1" rowspan="1">4.0 x108</td></tr><tr><td colspan="1" rowspan="1">Enterococcus faecium</td><td colspan="1" rowspan="1">4.9 x107</td><td colspan="1" rowspan="1">3.6 x107</td></tr><tr><td colspan="1" rowspan="1">Candida albicans</td><td colspan="1" rowspan="1">1.6 x106</td><td colspan="1" rowspan="1">1.4 x106</td></tr></table>

# e. Detection limit:

The limit of detection (LoD) was determined for each analyte/group on the ePlex BCIDGN Panel using samples prepared in negative blood culture matrix consisting of EDTA human whole blood (amount consistent with the bottle manufacturer recommendations) spiked into a blood culture bottle and incubated for a minimum of eight hours in a continuously monitoring blood culture instrument. Quantitation of organism preparations was determined in CFU/mL using plating and colony count methods. Analytes were pooled together and serially diluted to create organism mixes containing no more than five organisms in negative blood/blood culture matrix. A minimum of 20 replicates per organism mix were tested to confirm the final LoD for each assay target. The limit of detection was defined as the lowest concentration at which each target is detected at least $9 5 \%$ of the time.

Results from the study demonstrated that all targets were detected in $2 9 5 \%$ of tested replicates at $\leq 1 \mathrm { ~ x ~ } 1 0 ^ { 8 } \mathrm { \dot { C } F U / m L }$ for bacterial and resistance marker targets and ${ \le } 1 \mathrm { ~ x ~ } 1 0 ^ { 6 }$ $\mathrm { C F U / m L }$ for fungal targets. It is noted that the LoD for Fusobacterium necrophorum approximates the concentration for this organism present at bottle positivity.

The confirmed LoD for ePlex BCID-GN Panel organism and resistance marker targets are shown in Table 19.

Table 19: LoD results summary   

<table><tr><td colspan="1" rowspan="1">Target</td><td colspan="1" rowspan="1">Organism</td><td colspan="1" rowspan="1">Strain</td><td colspan="1" rowspan="1">LoD Concentration(CFU/mL)</td></tr><tr><td colspan="1" rowspan="2">Acinetobacter baumannii</td><td colspan="1" rowspan="1">Acinetobacter baumannii</td><td colspan="1" rowspan="1">NCTC 13421</td><td colspan="1" rowspan="1">1 x 106</td></tr><tr><td colspan="1" rowspan="1">Acinetobacter baumannii</td><td colspan="1" rowspan="1">NCTC 13304</td><td colspan="1" rowspan="1">1 x 106</td></tr><tr><td colspan="1" rowspan="2">Bacteroides fragilis</td><td colspan="1" rowspan="1">Bacteroides fragilis</td><td colspan="1" rowspan="1">ATCC 25285</td><td colspan="1" rowspan="1">1 x 105$</td></tr><tr><td colspan="1" rowspan="1">Bacteroides fragilis</td><td colspan="1" rowspan="1">ATCC 43860</td><td colspan="1" rowspan="1">1 x 104</td></tr><tr><td colspan="1" rowspan="2">Citrobacter</td><td colspan="1" rowspan="1">Citrobacter freundii</td><td colspan="1" rowspan="1">NCTC 9750</td><td colspan="1" rowspan="1">1 x 106</td></tr><tr><td colspan="1" rowspan="1">Citrobacter koseri</td><td colspan="1" rowspan="1">ATCC 27156</td><td colspan="1" rowspan="1">1 x 106</td></tr><tr><td colspan="1" rowspan="2">Cronobacter sakazakii</td><td colspan="1" rowspan="1">Cronobacter sakazakii</td><td colspan="1" rowspan="1">ATCC 29544</td><td colspan="1" rowspan="1">1 x 105</td></tr><tr><td colspan="1" rowspan="1">Cronobacter sakazakii</td><td colspan="1" rowspan="1">ATCC 29004</td><td colspan="1" rowspan="1">1 x 106</td></tr><tr><td colspan="1" rowspan="3">Enterobacter (non-cloacaecomplex)</td><td colspan="1" rowspan="1">Enterobacter aerogenes</td><td colspan="1" rowspan="1">CDC#0074</td><td colspan="1" rowspan="1">1 x 106</td></tr><tr><td colspan="1" rowspan="1">Enterobacter aerogenes</td><td colspan="1" rowspan="1">CDC#0161</td><td colspan="1" rowspan="1">1 x 105</td></tr><tr><td colspan="1" rowspan="1">Enterobacter amnigenus</td><td colspan="1" rowspan="1">ATCC 33072</td><td colspan="1" rowspan="1">1 x 106</td></tr><tr><td colspan="1" rowspan="2">Enterobacter cloacaeComplex</td><td colspan="1" rowspan="1">Enterobacter cloacae</td><td colspan="1" rowspan="1">CDC#0154</td><td colspan="1" rowspan="1">1 x 106</td></tr><tr><td colspan="1" rowspan="1">Enterobacter asburiae</td><td colspan="1" rowspan="1">ATCC 35957</td><td colspan="1" rowspan="1">1 x 106</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Enterobacter hormaechei</td><td colspan="1" rowspan="1">ATCC BAA-2082</td><td colspan="1" rowspan="1">1 x 106</td></tr><tr><td colspan="1" rowspan="2">Escherichia coli</td><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">CDC#0118</td><td colspan="1" rowspan="1">1 x 107</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">NCTC 13441</td><td colspan="1" rowspan="1">1 x 106</td></tr><tr><td colspan="1" rowspan="2">Fusobacterium necrophorum</td><td colspan="1" rowspan="1">Fusobacterium necrophorum</td><td colspan="1" rowspan="1">ATCC 51357</td><td colspan="1" rowspan="1">1 x 108</td></tr><tr><td colspan="1" rowspan="1">Fusobacterium necrophorum</td><td colspan="1" rowspan="1">ATCC 27852</td><td colspan="1" rowspan="1">1 x 107</td></tr><tr><td colspan="1" rowspan="2">Fusobacterium nucleatum</td><td colspan="1" rowspan="1">Fusobacterium nucleatum</td><td colspan="1" rowspan="1">ATCC 25586</td><td colspan="1" rowspan="1">1 x 107</td></tr><tr><td colspan="1" rowspan="1">Fusobacterium nucleatum</td><td colspan="1" rowspan="1">ATCC 23726</td><td colspan="1" rowspan="1">1 x 105</td></tr><tr><td colspan="1" rowspan="2">Haemophilus influenzae</td><td colspan="1" rowspan="1">Haemophilus influenzae</td><td colspan="1" rowspan="1">ATCC 19418</td><td colspan="1" rowspan="1">1 x 105$</td></tr><tr><td colspan="1" rowspan="1">Haemophilus influenzae</td><td colspan="1" rowspan="1">ATCC 9006</td><td colspan="1" rowspan="1">1 x 107</td></tr><tr><td colspan="1" rowspan="2">Klebsiella oxytoca</td><td colspan="1" rowspan="1">Klebsiella oxytoca</td><td colspan="1" rowspan="1">ATCC 43165</td><td colspan="1" rowspan="1">1 x 107</td></tr><tr><td colspan="1" rowspan="1">Klebsiella oxytoca</td><td colspan="1" rowspan="1">ATCC 8724</td><td colspan="1" rowspan="1">1 x 107</td></tr><tr><td colspan="1" rowspan="2">Klebsiella pneumoniae group</td><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">CDC#0160</td><td colspan="1" rowspan="1">1 x 106</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">CDC#0107</td><td colspan="1" rowspan="1">1 x 106</td></tr><tr><td colspan="1" rowspan="2">Morganella morganii</td><td colspan="1" rowspan="1">Morganella morganii</td><td colspan="1" rowspan="1">ATCC 25829</td><td colspan="1" rowspan="1">1 x 107</td></tr><tr><td colspan="1" rowspan="1">Morganella morganii</td><td colspan="1" rowspan="1">CDC#0133</td><td colspan="1" rowspan="1">1 x 107</td></tr><tr><td colspan="1" rowspan="2">Neisseria meningitidis</td><td colspan="1" rowspan="1">Neisseria meningitidis</td><td colspan="1" rowspan="1">ATCC 13090</td><td colspan="1" rowspan="1">1 x 105</td></tr><tr><td colspan="1" rowspan="1">Neisseria meningitidis</td><td colspan="1" rowspan="1">ATCC 13102</td><td colspan="1" rowspan="1">1 x 104</td></tr><tr><td colspan="1" rowspan="2">Proteus</td><td colspan="1" rowspan="1">Proteus vulgaris</td><td colspan="1" rowspan="1">ATCC 6896</td><td colspan="1" rowspan="1">1 x 107</td></tr><tr><td colspan="1" rowspan="1">Proteus vulgaris</td><td colspan="1" rowspan="1">ATCC 6380</td><td colspan="1" rowspan="1">1 x 107</td></tr><tr><td colspan="1" rowspan="2">Proteus mirabilis</td><td colspan="1" rowspan="1">Proteus mirabilis</td><td colspan="1" rowspan="1">CDC#0159</td><td colspan="1" rowspan="1">1 x 106</td></tr><tr><td colspan="1" rowspan="1">Proteus mirabilis</td><td colspan="1" rowspan="1">ATCC 43071</td><td colspan="1" rowspan="1">1 x 106</td></tr><tr><td colspan="1" rowspan="2">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">CDC#0103</td><td colspan="1" rowspan="1">1 x 106</td></tr><tr><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">NCTC 13437</td><td colspan="1" rowspan="1">1 x 106</td></tr><tr><td colspan="1" rowspan="2">Salmonella</td><td colspan="1" rowspan="1">Salmonella bongori</td><td colspan="1" rowspan="1">ATCC 43975</td><td colspan="1" rowspan="1">1 x 105</td></tr><tr><td colspan="1" rowspan="1">Salmonella enterica</td><td colspan="1" rowspan="1">ATCC 6962</td><td colspan="1" rowspan="1">1 x 105$</td></tr><tr><td colspan="1" rowspan="2">Serratia</td><td colspan="1" rowspan="1">Serratia liquefaciens</td><td colspan="1" rowspan="1">ATCC 27592</td><td colspan="1" rowspan="1">1 x 106</td></tr><tr><td colspan="1" rowspan="1">Serratia plymuthica</td><td colspan="1" rowspan="1">ATCC 53858</td><td colspan="1" rowspan="1">1 x 107</td></tr><tr><td colspan="1" rowspan="2">Serratia marcescens</td><td colspan="1" rowspan="1">Serratia marcescens</td><td colspan="1" rowspan="1">ATCC 14041</td><td colspan="1" rowspan="1">1 x 107</td></tr><tr><td colspan="1" rowspan="1">Serratia marcescens</td><td colspan="1" rowspan="1">ATCC 14756</td><td colspan="1" rowspan="1">1 x 105$</td></tr><tr><td colspan="1" rowspan="2">Stenotrophomonas maltophilia</td><td colspan="1" rowspan="1">Stenotrophomonas maltophilia</td><td colspan="1" rowspan="1">ATCC 13637</td><td colspan="1" rowspan="1">1 x 106</td></tr><tr><td colspan="1" rowspan="1">Stenotrophomonas maltophilia</td><td colspan="1" rowspan="1">ATCC 17666</td><td colspan="1" rowspan="1">1 x 107</td></tr><tr><td colspan="1" rowspan="2">Pan Candida</td><td colspan="1" rowspan="1">Candia albicans</td><td colspan="1" rowspan="1">ATCC 10231</td><td colspan="1" rowspan="1">1 x 106</td></tr><tr><td colspan="1" rowspan="1">Candia glabrata</td><td colspan="1" rowspan="1">ATCC 15126</td><td colspan="1" rowspan="1">1 x 105</td></tr><tr><td colspan="1" rowspan="6">Pan Gram-Positive</td><td colspan="1" rowspan="1">Enterococcus faecalis</td><td colspan="1" rowspan="1">ATCC 51575</td><td colspan="1" rowspan="1">1 x 105</td></tr><tr><td colspan="1" rowspan="1">Enterococcus faecium</td><td colspan="1" rowspan="1">ATCC 31282</td><td colspan="1" rowspan="1">1 x 107</td></tr><tr><td colspan="1" rowspan="1">Bacillus subtilis</td><td colspan="1" rowspan="1">ATCC 21008</td><td colspan="1" rowspan="1">1 x 106</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">ATCC BAA-2313</td><td colspan="1" rowspan="1">1 x 105</td></tr><tr><td colspan="1" rowspan="1">Streptococcus agalactiae</td><td colspan="1" rowspan="1">ATCC 13813</td><td colspan="1" rowspan="1">1 x 106</td></tr><tr><td colspan="1" rowspan="1">Streptococcus anginosus</td><td colspan="1" rowspan="1">ATCC 33397</td><td colspan="1" rowspan="1">1 x 106</td></tr><tr><td colspan="1" rowspan="2">CTX-M</td><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">NCTC 13441</td><td colspan="1" rowspan="1">1 x 104</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">CDC#0107</td><td colspan="1" rowspan="1">1 x 105</td></tr><tr><td colspan="1" rowspan="2">IMP</td><td colspan="1" rowspan="1">Enterobacter aerogenes</td><td colspan="1" rowspan="1">CDC#0161</td><td colspan="1" rowspan="1">1 x 106</td></tr><tr><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">CDC#0103</td><td colspan="1" rowspan="1">1 x 105</td></tr><tr><td colspan="1" rowspan="2">KPC</td><td colspan="1" rowspan="1">Enterobacter hormaechei</td><td colspan="1" rowspan="1">ATCC BAA-2082</td><td colspan="1" rowspan="1">1 x 106</td></tr><tr><td colspan="1" rowspan="1">Morganella morganii</td><td colspan="1" rowspan="1">CDC#0133</td><td colspan="1" rowspan="1">1 x 106</td></tr><tr><td colspan="1" rowspan="2">NDM</td><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">CDC#0118</td><td colspan="1" rowspan="1">1 x 105}$</td></tr><tr><td colspan="1" rowspan="1">Proteus mirabilis</td><td colspan="1" rowspan="1">CDC#0159</td><td colspan="1" rowspan="1">1 x 105</td></tr><tr><td colspan="1" rowspan="4">OXA</td><td colspan="1" rowspan="1">Acinetobacter baumannii (OXA-23)</td><td colspan="1" rowspan="1">NCTC 13421</td><td colspan="1" rowspan="1">1 x 105}$</td></tr><tr><td colspan="1" rowspan="1">Acinetobacter baumannii (OXA-27)</td><td colspan="1" rowspan="1">NCTC 13304</td><td colspan="1" rowspan="1">1 x 105</td></tr><tr><td colspan="1" rowspan="1">Enterobacter aerogenes (OXA-48)</td><td colspan="1" rowspan="1">CDC#0074</td><td colspan="1" rowspan="1">1 x 106</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae (OXA-48)</td><td colspan="1" rowspan="1">CDC#0160</td><td colspan="1" rowspan="1">1 x 106</td></tr><tr><td colspan="1" rowspan="2">VIM</td><td colspan="1" rowspan="1">Enterobacter cloacae</td><td colspan="1" rowspan="1">CDC#0154</td><td colspan="1" rowspan="1">1 x 106</td></tr><tr><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">NCTC 13437</td><td colspan="1" rowspan="1">1 x 105</td></tr></table>

# $f .$ Analytical Reactivity (Inclusivity)

The analytical reactivity of the BCID-GN Panel was evaluated using a collection of 336 bacterial and fungal strains/isolates. Organisms were tested at concentrations expected at bottle positivity with bacteria tested at $\mathrm { \bar { 1 } \times 1 0 ^ { 9 } C F U / m L }$ or lower and fungal strains tested at $1 \mathrm { ~ x ~ i 0 ^ { 6 } ~ C F U / m L }$ . For organisms that were not evaluated in the bottle positivity study, test samples were prepared at $4 . 5 \mathrm { { x } 1 0 ^ { 8 } \mathrm { { C F U / m L } } }$ (i.e., average concentrations for all gramnegative organisms evaluated in the bottle positivity study). Each strain evaluated for inclusivity was tested in triplicate and if not detected in initial testing, the strain was retested at a higher concentration. If detected at the high concentration(s), the species/isolate is indicated as detected with reduced sensitivity and the concentration is indicated in footnotes under Table 20 below. All strains evaluated generated ‘detected’ results by the ePlex BCID-GN Panel with the following exceptions:

Strains of Citrobacter amalonaticus, Citrobacter farmer, Citrobacter gillenii, Citrobacter murliniae and Citrobacter sedlakii were not detected by the BCIDGN Panel at any concentration evaluated (up to $1 \mathrm { x } \ 1 0 ^ { 9 } \mathrm { C F U / m L }$ ). Enterobacter kobei and Enterobacter cancerogenous were not detected by the ePlex BCID-GN Panel. Enterobacter cowanii was not detected by the ePlex BCID-GN Enterobacter noncloacae complex assay. Instead, this species generated detected results with the Enterobacter cloacae complex. It is noted that this species is not included in the Enterobacter cloacae complex. The Pan Gram-positive assay target may have reduced sensitivity for

Streptococcus thoraltensis and Enterococcus saccharolyticus (1/6 replicates detected for both organisms at $1 \mathrm { ~ x ~ } 1 0 ^ { 7 } \mathrm { C F U / m L ) }$ . Additional testing at $4 \times 1 0 ^ { 8 }$ (Streptococcus thoraltensis) and $1 \mathrm { ~ x ~ } 1 0 ^ { 8 }$ (Enterococcus saccharolyticus) generated 3/3 replicates detected for the Pan Gram-positive target.

For one strain each of Enterobacter amnigenus and Morganella morganii, two of three initially tested replicates were detected. Additional testing for each strain generated positive results for three additional replicates for a total of five of six replicates tested.

One strain of Serratia odorifera was not detected at $1 \mathrm { ~ x ~ } 1 0 ^ { 8 } \mathrm { C F U / m L }$ and was detected in only one of three replicates at $1 \mathrm { ~ x ~ } 1 0 ^ { 9 } \mathrm { C F U / m L }$ . In silico analysis suggest that this species will not be detected by the BCID-GN Panel.

In initial testing of samples prepared at $1 \mathrm { ~ x ~ } 1 0 ^ { 6 } \mathrm { C F U / m L }$ , the Pan Candida target was detected for one of six replicate samples containing Candida krusei (ATCC 34135) and two of six replicates of Candida parapsilosis (ATCC 90018). Additional testing was performed with blood culture samples from culture bottles grown on a blood culture instrument until bottle positivity. For this additional testing, the Pan Candida target was detected for all six replicates of each Candida strain.

Organisms and associated resistance markers evaluated in the analytical reactivity study and detected by the ePlex BCID-GN Panel are shown in Table 20. Additional strains were evaluated as part of the LoD study (See Table 19).

Table 20: Analytical Reactivity (Inclusivity)   

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Strain</td></tr><tr><td rowspan=1 colspan=1>Acinetobacter baumannii</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=7 colspan=1>Acinetobacter baumannii</td><td rowspan=1 colspan=1>CDC#0052</td></tr><tr><td rowspan=1 colspan=1>NCTC 13302</td></tr><tr><td rowspan=1 colspan=1>NCTC 13303</td></tr><tr><td rowspan=1 colspan=1>NCTC13305</td></tr><tr><td rowspan=1 colspan=1>NCTC13420</td></tr><tr><td rowspan=1 colspan=1>NCTC 13422</td></tr><tr><td rowspan=1 colspan=1>NCTC 13423</td></tr><tr><td rowspan=1 colspan=1>Acinetobacter baumannii (NDM-1)</td><td rowspan=1 colspan=1>CDC#0033</td></tr><tr><td rowspan=5 colspan=1>Acinetobacter baumannii (OXA-23)</td><td rowspan=1 colspan=1>ATCC BAA-1605</td></tr><tr><td rowspan=1 colspan=1>CDC#0045</td></tr><tr><td rowspan=1 colspan=1>CDC#0056</td></tr><tr><td rowspan=1 colspan=1>NCTC 13301</td></tr><tr><td rowspan=1 colspan=1>NCTC 13424</td></tr><tr><td rowspan=1 colspan=1>Acinetobacter spp. (IMP only)</td><td rowspan=1 colspan=1>JMI4084A</td></tr><tr><td rowspan=1 colspan=1>Bacteroides fragilis</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=3 colspan=1>Bacteroides fragilis</td><td rowspan=1 colspan=1>ATCC 23745</td></tr><tr><td rowspan=1 colspan=1>ATCC 700786</td></tr><tr><td rowspan=1 colspan=1>NCTC 9343</td></tr></table>

<table><tr><td colspan="1" rowspan="1">Organism</td><td colspan="1" rowspan="1">Strain</td></tr><tr><td colspan="1" rowspan="1">Citrobacter</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="2">Citrobacter braakii</td><td colspan="1" rowspan="1">ATCC 43162</td></tr><tr><td colspan="1" rowspan="1">ATCC 51113</td></tr><tr><td colspan="1" rowspan="2">Citrobacter freundii</td><td colspan="1" rowspan="1">ATCC 6879</td></tr><tr><td colspan="1" rowspan="1">ATCC 8090</td></tr><tr><td colspan="1" rowspan="1">Citrobacter freundii (CTX)</td><td colspan="1" rowspan="1">JMI2047</td></tr><tr><td colspan="1" rowspan="1">Citrobacter freundii (KPC-2)</td><td colspan="1" rowspan="1">CDC#0116</td></tr><tr><td colspan="1" rowspan="4">Citrobacter koseri</td><td colspan="1" rowspan="1">ATCC 25409</td></tr><tr><td colspan="1" rowspan="1">ATCC 27028</td></tr><tr><td colspan="1" rowspan="1">ATCC 29225</td></tr><tr><td colspan="1" rowspan="1">ATCC 29936</td></tr><tr><td colspan="1" rowspan="1">Citrobacter species (CTX-15,NDM-1)</td><td colspan="1" rowspan="1">CDC #0157</td></tr><tr><td colspan="1" rowspan="1">Citrobacter werkmanii</td><td colspan="1" rowspan="1">ATCC 51114</td></tr><tr><td colspan="1" rowspan="1">Citrobacter youngae</td><td colspan="1" rowspan="1">ATCC 29935</td></tr><tr><td colspan="1" rowspan="1">Cronobacter sakazakii</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="3">Cronobacter sakazakii</td><td colspan="1" rowspan="1">ATCC 12868</td></tr><tr><td colspan="1" rowspan="1">ATCC BAA-894</td></tr><tr><td colspan="1" rowspan="1">FSL F6-0023</td></tr><tr><td colspan="1" rowspan="1">Enterobacter (non-cloacae complex)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="3">Enterobacter aerogenes</td><td colspan="1" rowspan="1">ATCC 13048</td></tr><tr><td colspan="1" rowspan="1">ATCC 29010</td></tr><tr><td colspan="1" rowspan="1">ATCC 51697</td></tr><tr><td colspan="1" rowspan="2">Enterobacter amnigenus</td><td colspan="1" rowspan="1">ATCC 33731</td></tr><tr><td colspan="1" rowspan="1">ATCC 51816B</td></tr><tr><td colspan="1" rowspan="2">Enterobacter gergoviae</td><td colspan="1" rowspan="1">ATCC 33028</td></tr><tr><td colspan="1" rowspan="1">ATCC 33426</td></tr><tr><td colspan="1" rowspan="1">Enterobacter cloacae complex</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="3">Enterobacter asburiae</td><td colspan="1" rowspan="1">ATCC 35954</td></tr><tr><td colspan="1" rowspan="1">ATCC 35955</td></tr><tr><td colspan="1" rowspan="1">ATCC 35956</td></tr><tr><td colspan="1" rowspan="1">Enterobacter cloacae (CTX-15)</td><td colspan="1" rowspan="1">CDC#0038</td></tr><tr><td colspan="1" rowspan="1">Enterobacter cloacae (CTX-9)</td><td colspan="1" rowspan="1">NCTC 13464</td></tr><tr><td colspan="1" rowspan="1">Enterobacter cloacae (CTX-15,KPC-2)</td><td colspan="1" rowspan="1">CDC#0163</td></tr><tr><td colspan="1" rowspan="1">Enterobacter cloacae (CTX,NDM)</td><td colspan="1" rowspan="1">JMI53571</td></tr><tr><td colspan="1" rowspan="2">Enterobacter cloacae subsp.cloacae</td><td colspan="1" rowspan="1">ATCC 23355</td></tr><tr><td colspan="1" rowspan="1">ATCC 35030</td></tr><tr><td colspan="1" rowspan="1">Enterobacter cloacae subsp.dissolvens</td><td colspan="1" rowspan="1">ATCC 23373</td></tr><tr><td colspan="1" rowspan="1">Enterobacter hormaechei</td><td colspan="1" rowspan="1">ATCC 700323</td></tr><tr><td colspan="1" rowspan="1">Enterobacter hormaechei subsp.hormaechei</td><td colspan="1" rowspan="1">ATCC 49162</td></tr><tr><td colspan="1" rowspan="1">Enterobacter hormaechei subsp.oharae</td><td colspan="1" rowspan="1">ATCC 49163</td></tr><tr><td colspan="1" rowspan="1">Enterobacter hormaechei subsp.steigerwaltii</td><td colspan="1" rowspan="1">CIP108489T</td></tr><tr><td colspan="1" rowspan="1">Enterobacter ludwigii</td><td colspan="1" rowspan="1">DSM-16688</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="21">Escherichia coli</td><td colspan="1" rowspan="1">ATCC 14948</td></tr><tr><td colspan="1" rowspan="1">ATCC 25922</td></tr><tr><td colspan="1" rowspan="1">ATCC 33605</td></tr><tr><td colspan="1" rowspan="1">ATCC 33876</td></tr><tr><td colspan="1" rowspan="1">ATCC 35150</td></tr><tr><td colspan="1" rowspan="1">ATCC 4157</td></tr><tr><td colspan="1" rowspan="1">ATCC 43888</td></tr><tr><td colspan="1" rowspan="1">ATCC 51446</td></tr><tr><td colspan="1" rowspan="1">ATCC 51755</td></tr><tr><td colspan="1" rowspan="1">ATCC 53498</td></tr><tr><td colspan="1" rowspan="1">ATCC 700728</td></tr><tr><td colspan="1" rowspan="1">NCIMB 8545</td></tr><tr><td colspan="1" rowspan="1">NCTC 8620</td></tr><tr><td colspan="1" rowspan="1">ATCC 9637</td></tr><tr><td colspan="1" rowspan="1">ATCC BAA-196</td></tr><tr><td colspan="1" rowspan="1">ATCC BAA-197</td></tr><tr><td colspan="1" rowspan="1">ATCC BAA-198</td></tr><tr><td colspan="1" rowspan="1">ATCC BAA-199</td></tr><tr><td colspan="1" rowspan="1">ATCC BAA-200</td></tr><tr><td colspan="1" rowspan="1">ATCC BAA-201</td></tr><tr><td colspan="1" rowspan="1">ATCC BAA-202</td></tr></table>

<table><tr><td colspan="1" rowspan="1">Organism</td><td colspan="1" rowspan="1">Strain</td></tr><tr><td colspan="1" rowspan="20">b</td><td colspan="1" rowspan="1">ATCC BAA-203</td></tr><tr><td colspan="1" rowspan="1">ATCC BAA-204</td></tr><tr><td colspan="1" rowspan="1">LMC 243094647</td></tr><tr><td colspan="1" rowspan="1">LMC 243098776</td></tr><tr><td colspan="1" rowspan="1">LMC 243098947</td></tr><tr><td colspan="1" rowspan="1">LMC 243108047</td></tr><tr><td colspan="1" rowspan="1">LMC 243109799</td></tr><tr><td colspan="1" rowspan="1">LMC 243112411</td></tr><tr><td colspan="1" rowspan="1">LMC 244006281</td></tr><tr><td colspan="1" rowspan="1">LMC 244006433</td></tr><tr><td colspan="1" rowspan="1">LMC 244008038</td></tr><tr><td colspan="1" rowspan="1">LMC 244012579</td></tr><tr><td colspan="1" rowspan="1">NCTC 13351</td></tr><tr><td colspan="1" rowspan="1">NCTC 10279</td></tr><tr><td colspan="1" rowspan="1">ATCC 10536</td></tr><tr><td colspan="1" rowspan="1">ATCC 10538</td></tr><tr><td colspan="1" rowspan="1">ATCC 10799</td></tr><tr><td colspan="1" rowspan="1">ATCC 11229</td></tr><tr><td colspan="1" rowspan="1">ATCC 13762</td></tr><tr><td colspan="1" rowspan="1">ATCC 14169</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli (CTX-14)</td><td colspan="1" rowspan="1">CDC#0086</td></tr><tr><td colspan="1" rowspan="5">Escherichia coli (CTX-15)</td><td colspan="1" rowspan="1">ATCC BAA-2326</td></tr><tr><td colspan="1" rowspan="1">NCTC 13353</td></tr><tr><td colspan="1" rowspan="1">NCTC 13400</td></tr><tr><td colspan="1" rowspan="1">NCTC 13450</td></tr><tr><td colspan="1" rowspan="1">NCTC 13451</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli (CTX-3)</td><td colspan="1" rowspan="1">NCTC 13452</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli (CTX-1)</td><td colspan="1" rowspan="1">NCTC 13461</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli (CTX-2)</td><td colspan="1" rowspan="1">NCTC 13462</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli (CTX-8)</td><td colspan="1" rowspan="1">NCTC 13463</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli (CTX-15, NDM-6)</td><td colspan="1" rowspan="1">CDC#0137</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli (CTX-15, NDM-7)</td><td colspan="1" rowspan="1">CDC#0162</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli (IMP)</td><td colspan="1" rowspan="1">NCTC 13476</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli (KPC)</td><td colspan="1" rowspan="1">ATCC BAA-2340</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli (NDM-5)</td><td colspan="1" rowspan="1">CDC#0150</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli (OXA)</td><td colspan="1" rowspan="1">LMC_DR00012</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli (VIM)</td><td colspan="1" rowspan="1">JMI32465</td></tr><tr><td colspan="1" rowspan="1">Fusobacterium necrophorum</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="3">Fusobacterium necrophorumsubsp. necrophorum</td><td colspan="1" rowspan="1">ATCC 25286</td></tr><tr><td colspan="1" rowspan="1">NCTC 10575</td></tr><tr><td colspan="1" rowspan="1">NCTC 10577</td></tr><tr><td colspan="1" rowspan="1">Fusobacterium nucleatum</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Fusobacterium nucleatum subsp.nucleatum</td><td colspan="1" rowspan="1">ATCC 31647</td></tr><tr><td colspan="1" rowspan="1">Fusobacterium nucleatum subsp.fusiforme</td><td colspan="1" rowspan="1">ATCC 51190</td></tr><tr><td colspan="1" rowspan="1">Fusobacterium nucleatum subsp.vincentii</td><td colspan="1" rowspan="1">ATCC 49256</td></tr><tr><td colspan="2" rowspan="1">Haemophilus influenzae</td></tr><tr><td colspan="1" rowspan="6">Haemophilus influenzae</td><td colspan="1" rowspan="1">ATCC 33930</td></tr><tr><td colspan="1" rowspan="1">ATCC 43065</td></tr><tr><td colspan="1" rowspan="1">ATCC 43163</td></tr><tr><td colspan="1" rowspan="1">NCTC 11931</td></tr><tr><td colspan="1" rowspan="1">NCTC 12699</td></tr><tr><td colspan="1" rowspan="1">NCTC 8143</td></tr><tr><td colspan="1" rowspan="1">Haemophilus influenzae Type b</td><td colspan="1" rowspan="1">ATCC 10211</td></tr><tr><td colspan="1" rowspan="1">Haemophilus influenzae Type c</td><td colspan="1" rowspan="1">ATCC 9007</td></tr><tr><td colspan="1" rowspan="1">Haemophilus influenzae Type d</td><td colspan="1" rowspan="1">ATCC 9332</td></tr><tr><td colspan="1" rowspan="1">Haemophilus influenzae Type e</td><td colspan="1" rowspan="1">NCTC 8472</td></tr><tr><td colspan="1" rowspan="1">Haemophilus influenzae Type f</td><td colspan="1" rowspan="1">ATCC 9833</td></tr><tr><td colspan="1" rowspan="1">Klebsiella oxytoca</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="5">Klebsiella oxytoca</td><td colspan="1" rowspan="1">ATCC 43086</td></tr><tr><td colspan="1" rowspan="1">ATCC 43863</td></tr><tr><td colspan="1" rowspan="1">ATCC 49131</td></tr><tr><td colspan="1" rowspan="1">ATCC 700324</td></tr><tr><td colspan="1" rowspan="1">ATCC 51817</td></tr><tr><td colspan="1" rowspan="1">Klebsiella oxytoca (KPC-3)</td><td colspan="1" rowspan="1">CDC#0147</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae group</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae (CTX-15)</td><td colspan="1" rowspan="1">CDC#0109</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae (CTX-25)</td><td colspan="1" rowspan="1">NCTC 13465</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae (CTX,KPC)</td><td colspan="1" rowspan="1">IMH-C2261309</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae (CTX,DM-1)</td><td colspan="1" rowspan="1">NCTC 13443</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae (CTX-15;NDM-1; OXA-232)</td><td colspan="1" rowspan="1">CDC#0153</td></tr><tr><td colspan="1" rowspan="2">Klebsiella pneumoniae (CTX-15,OXA-232)</td><td colspan="1" rowspan="1">CDC#0075</td></tr><tr><td colspan="1" rowspan="1">CDC#0066</td></tr><tr><td colspan="1" rowspan="4">Klebsiella pneumoniae (CTX-15,OXA-181</td><td colspan="1" rowspan="1">CDC#0039</td></tr><tr><td colspan="1" rowspan="1">CDC#0140</td></tr><tr><td colspan="1" rowspan="1">CDC#0141</td></tr><tr><td colspan="1" rowspan="1">CDC#0142</td></tr><tr><td colspan="1" rowspan="2">Klebsiella pneumoniae (IMP-4)</td><td colspan="1" rowspan="1">CDC#0034</td></tr><tr><td colspan="1" rowspan="1">CDC#0080</td></tr><tr><td colspan="1" rowspan="3">Klebsiella pneumoniae (KPC-3)</td><td colspan="1" rowspan="1">CDC#0125</td></tr><tr><td colspan="1" rowspan="1">CDC#0112</td></tr><tr><td colspan="1" rowspan="1">CDC#0113</td></tr><tr><td colspan="1" rowspan="5">Klebsiella pneumoniae (KPC)</td><td colspan="1" rowspan="1">ATCC BAA-1705</td></tr><tr><td colspan="1" rowspan="1">IMH-C2260742</td></tr><tr><td colspan="1" rowspan="1">IMH-C3151729</td></tr><tr><td colspan="1" rowspan="1">IMH-C4151728</td></tr><tr><td colspan="1" rowspan="1">IMH-C4171868</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae (OXA-48)</td><td colspan="1" rowspan="1">NCTC 13442</td></tr></table>

<table><tr><td colspan="1" rowspan="1">Organism</td><td colspan="1" rowspan="1">Strain</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae (CTX-15;VIM-27)</td><td colspan="1" rowspan="1">CDC#0040</td></tr><tr><td colspan="1" rowspan="3">Klebsiella pneumoniae (VIM-1)</td><td colspan="1" rowspan="1">CDC#0135</td></tr><tr><td colspan="1" rowspan="1">NCTC 13439</td></tr><tr><td colspan="1" rowspan="1">NCTC 13440</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae subsp.Ozaenae</td><td colspan="1" rowspan="1">ATCC 11296</td></tr><tr><td colspan="1" rowspan="4">Klebsiella pneumoniae subsp.pneumoniae</td><td colspan="1" rowspan="1">ATCC 13883</td></tr><tr><td colspan="1" rowspan="1">ATCC 27736</td></tr><tr><td colspan="1" rowspan="1">ATCC 51503</td></tr><tr><td colspan="1" rowspan="1">ATCC 51504</td></tr><tr><td colspan="1" rowspan="1">Klebsiella quasipneumoniae</td><td colspan="1" rowspan="1">ATCC 700603</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae subsp.rhinoscleromatis</td><td colspan="1" rowspan="1">ATCC 9436</td></tr><tr><td colspan="1" rowspan="1">Klebsiella variicola</td><td colspan="1" rowspan="1">ATCC BAA-830</td></tr><tr><td colspan="1" rowspan="1">Morganella morganii</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="2">Morganella morganii</td><td colspan="1" rowspan="1">ATCC 25830</td></tr><tr><td colspan="1" rowspan="1">GM148-209</td></tr><tr><td colspan="1" rowspan="1">Morganella morganii (CTX-15;ND-1</td><td colspan="1" rowspan="1">CDC#0057C</td></tr><tr><td colspan="1" rowspan="1">Neisseria meningitidis</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Neisseria meningitidis Serotype A</td><td colspan="1" rowspan="1">ATCC 13077</td></tr><tr><td colspan="1" rowspan="1">Neisseria meningitidis Serotype B</td><td colspan="1" rowspan="1">NCTC 10026</td></tr><tr><td colspan="1" rowspan="1">Neisseria meningitidis SerotypeW13</td><td colspan="1" rowspan="1">NCTC 11203</td></tr><tr><td colspan="1" rowspan="1">Neisseria meningitidis Serotype Y</td><td colspan="1" rowspan="1">ATCC 35561</td></tr><tr><td colspan="1" rowspan="1">Proteus</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Proteus hauseri</td><td colspan="1" rowspan="1">ATCC 13315</td></tr><tr><td colspan="1" rowspan="2">Proteus mirabilis</td><td colspan="1" rowspan="1">ATCC 33583</td></tr><tr><td colspan="1" rowspan="1">ATCC BAA-663</td></tr><tr><td colspan="1" rowspan="1">Proteus mirabilis (IMP)</td><td colspan="1" rowspan="1">JMI1955389</td></tr><tr><td colspan="1" rowspan="1">Proteus mirabilis (KPC-6)</td><td colspan="1" rowspan="1">CDC#0155</td></tr><tr><td colspan="1" rowspan="1">Proteus penneri</td><td colspan="1" rowspan="1">ATCC 35197</td></tr><tr><td colspan="1" rowspan="4">Proteus vulgaris</td><td colspan="1" rowspan="1">ATCC 33420</td></tr><tr><td colspan="1" rowspan="1">ATCC 49132</td></tr><tr><td colspan="1" rowspan="1">ATCC 8427</td></tr><tr><td colspan="1" rowspan="1">NCTC 4636</td></tr><tr><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Pseudomonas aeruginosa (IMP-14</td><td colspan="1" rowspan="1">CDC#0092</td></tr><tr><td colspan="1" rowspan="1">Pseudomonas aeruginosa (IMP-1)</td><td colspan="1" rowspan="1">CDC#0241</td></tr><tr><td colspan="1" rowspan="1">Pseudomonas aeruginosa (IMP)</td><td colspan="1" rowspan="1">CDC#0439</td></tr><tr><td colspan="1" rowspan="1">Pseudomonas aeruginosa (KPC-5)</td><td colspan="1" rowspan="1">CDC#0090</td></tr><tr><td colspan="1" rowspan="1">Pseudomonas aeruginosa (VIM-2)</td><td colspan="1" rowspan="1">CDC#0100</td></tr><tr><td colspan="1" rowspan="1">Pseudomonas aeruginosa (VIM-4)</td><td colspan="1" rowspan="1">CDC#0054</td></tr><tr><td colspan="1" rowspan="1">Salmonella</td><td colspan="1" rowspan="1">cYanmaGYellowb</td></tr><tr><td colspan="1" rowspan="1">Salmonella enterica serovar4,[5],12:i</td><td colspan="1" rowspan="1">FSL S5-0580</td></tr><tr><td colspan="1" rowspan="1">Salmonella enterica serovarAgona</td><td colspan="1" rowspan="1">ATCC 51957</td></tr><tr><td colspan="1" rowspan="1">Salmonella enterica serovarBaareilly</td><td colspan="1" rowspan="1">ATCC 9115</td></tr><tr><td colspan="1" rowspan="1">Salmonella enterica serovarBraenderup</td><td colspan="1" rowspan="1">ATCC 700136</td></tr><tr><td colspan="1" rowspan="1">Salmonella enterica serovarEnteritidis</td><td colspan="1" rowspan="1">ATCC BAA-708</td></tr><tr><td colspan="1" rowspan="1">Salmonella enterica serovarHadar</td><td colspan="1" rowspan="1">ATCC 51956</td></tr><tr><td colspan="1" rowspan="1">Salmonella enterica serovarHedelberg</td><td colspan="1" rowspan="1">ATCC 8326</td></tr><tr><td colspan="1" rowspan="1">Salmonella enterica serovarInfantis</td><td colspan="1" rowspan="1">ATCC BAA-1675</td></tr><tr><td colspan="1" rowspan="1">Salmonella enterica serovarJaviana</td><td colspan="1" rowspan="1">ATCC 10721</td></tr><tr><td colspan="1" rowspan="1">Salmonella enterica serovarontevideo</td><td colspan="1" rowspan="1">ATCC 8387</td></tr><tr><td colspan="1" rowspan="1">Salmonella enterica serovarMuenchen</td><td colspan="1" rowspan="1">ATCC 8388</td></tr><tr><td colspan="1" rowspan="1">Salmonella enterica serovarOranienburg</td><td colspan="1" rowspan="1">ATCC 9239</td></tr><tr><td colspan="1" rowspan="1">Salmonella enterica serovarParatyphi B</td><td colspan="1" rowspan="1">FSL S5-0447</td></tr><tr><td colspan="1" rowspan="1">Salmonella enterica serovarSaintpaul</td><td colspan="1" rowspan="1">ATCC 9712</td></tr><tr><td colspan="1" rowspan="1">Salmonella enterica serovarThompson</td><td colspan="1" rowspan="1">ATCC 8391</td></tr><tr><td colspan="1" rowspan="1">Salmonella enterica serovar Typhi</td><td colspan="1" rowspan="1">ATCC 19430</td></tr><tr><td colspan="1" rowspan="1">Salmonella enterica subsp.arizonae</td><td colspan="1" rowspan="1">ATCC 13314</td></tr><tr><td colspan="1" rowspan="1">Salmonella enterica subsp.diarizonae</td><td colspan="1" rowspan="1">ATCC 12325</td></tr><tr><td colspan="1" rowspan="1">Salmonella enterica subsp.enterica serovar Typhimurium</td><td colspan="1" rowspan="1">ATCC 14028</td></tr><tr><td colspan="1" rowspan="1">Salmonella enterica subsp.houtenae</td><td colspan="1" rowspan="1">ATCC 29834</td></tr><tr><td colspan="1" rowspan="1">Salmonella enterica subsp. indica</td><td colspan="1" rowspan="1">ATCC BAA-1578</td></tr><tr><td colspan="1" rowspan="1">Salmonella enterica subsp.salamae</td><td colspan="1" rowspan="1">ATCC 6959</td></tr><tr><td colspan="1" rowspan="1">Salmonella enterica subsp.enterica serovar Mississippi</td><td colspan="1" rowspan="1">FSL A4-0633</td></tr><tr><td colspan="1" rowspan="1">Salmonella enterica subsp.enterica serovar Schwarzengrund</td><td colspan="1" rowspan="1">FSL S5-0458</td></tr><tr><td colspan="1" rowspan="1">Serratia</td><td colspan="1" rowspan="1">cYanmaGYellowb</td></tr><tr><td colspan="1" rowspan="1">Serratia ficaria</td><td colspan="1" rowspan="1">ATCC 33105</td></tr></table>

<table><tr><td colspan="1" rowspan="1">Organism</td><td colspan="1" rowspan="1">Strain</td></tr><tr><td colspan="1" rowspan="1">Serratia fonticola</td><td colspan="1" rowspan="1">ATCC 29844</td></tr><tr><td colspan="1" rowspan="1">Serratia grimesii</td><td colspan="1" rowspan="1">ATCC 14460</td></tr><tr><td colspan="1" rowspan="3">Serratia marcescens</td><td colspan="1" rowspan="1">ATCC 13880</td></tr><tr><td colspan="1" rowspan="1">ATCC 43861</td></tr><tr><td colspan="1" rowspan="1">ATCC 43862</td></tr><tr><td colspan="1" rowspan="1">Serratia marcescens (CTX)</td><td colspan="1" rowspan="1">JMI10244</td></tr><tr><td colspan="1" rowspan="2">Serratia rubidaea</td><td colspan="1" rowspan="1">ATCC 27593</td></tr><tr><td colspan="1" rowspan="1">ATCC 29025</td></tr><tr><td colspan="1" rowspan="1">Stenotrophomonas maltophilia</td><td colspan="1" rowspan="1">ophilia</td></tr><tr><td colspan="1" rowspan="3">Stenotrophomonas maltophilia</td><td colspan="1" rowspan="1">ATCC 13636</td></tr><tr><td colspan="1" rowspan="1">GM148-207</td></tr><tr><td colspan="1" rowspan="1">GM148-208</td></tr><tr><td colspan="1" rowspan="1">Pan-Gram Positive</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Bacillus amyloliquefaciens</td><td colspan="1" rowspan="1">ATCC 23845</td></tr><tr><td colspan="1" rowspan="1">Bacillus atrophaeus</td><td colspan="1" rowspan="1">ATCC 49337</td></tr><tr><td colspan="1" rowspan="1">Bacillus cereus</td><td colspan="1" rowspan="1">ATCC 10876</td></tr><tr><td colspan="1" rowspan="1">Bacillus licheniformis</td><td colspan="1" rowspan="1">ATCC 21039</td></tr><tr><td colspan="1" rowspan="1">Bacillus thuringiensis</td><td colspan="1" rowspan="1">ATCC 35646</td></tr><tr><td colspan="1" rowspan="1">Enterococcus avium</td><td colspan="1" rowspan="1">ATCC 14025</td></tr><tr><td colspan="1" rowspan="1">Enterococcus casseliflavus</td><td colspan="1" rowspan="1">ATCC 700327</td></tr><tr><td colspan="1" rowspan="1">Enterococcus faecalis</td><td colspan="1" rowspan="1">JMI876745</td></tr><tr><td colspan="1" rowspan="1">Enterococcus gallinarum</td><td colspan="1" rowspan="1">ATCC 49573</td></tr><tr><td colspan="1" rowspan="1">Enterococcus hirae</td><td colspan="1" rowspan="1">ATCC 49479</td></tr><tr><td colspan="1" rowspan="1">Enterococcus raffinosus</td><td colspan="1" rowspan="1">ATCC 49464</td></tr><tr><td colspan="1" rowspan="1">Enterococcus saccharolyticus</td><td colspan="1" rowspan="1">ATCC 43076D</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus capitis</td><td colspan="1" rowspan="1">NRS866</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus chromogenes</td><td colspan="1" rowspan="1">ATCC 43764</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus cohnii</td><td colspan="1" rowspan="1">ATCC 29974</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus epidermidis</td><td colspan="1" rowspan="1">ATCC 35984</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus gallinarum</td><td colspan="1" rowspan="1">ATCC 700401</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus haemolyticus</td><td colspan="1" rowspan="1">ATCC 29970</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus hominis</td><td colspan="1" rowspan="1">ATCC 27844</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus hyicus</td><td colspan="1" rowspan="1">ATCC 11249</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus lentus</td><td colspan="1" rowspan="1">ATCC 700403</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus lugdunensis</td><td colspan="1" rowspan="1">ATCC 49576</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus pasteuri</td><td colspan="1" rowspan="1">ATCC 51128</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus vitulinus</td><td colspan="1" rowspan="1">ATCC 51699</td></tr><tr><td colspan="1" rowspan="1">Streptococcus constellatus</td><td colspan="1" rowspan="1">ATCC 27513</td></tr><tr><td colspan="1" rowspan="1">Streptococcus dysgalactiae</td><td colspan="1" rowspan="1">ATCC 35666</td></tr><tr><td colspan="1" rowspan="1">Streptococcus equi</td><td colspan="1" rowspan="1">ATCC 9528</td></tr><tr><td colspan="1" rowspan="1">Streptococcus gallolyticus</td><td colspan="1" rowspan="1">ATCC 9809</td></tr><tr><td colspan="1" rowspan="1">Streptococcus gordonii</td><td colspan="1" rowspan="1">ATCC 35557</td></tr><tr><td colspan="1" rowspan="1">Streptococcus infantis</td><td colspan="1" rowspan="1">ATCC 700779</td></tr><tr><td colspan="1" rowspan="1">Streptococcus intermedius</td><td colspan="1" rowspan="1">ATCC 27335</td></tr><tr><td colspan="1" rowspan="1">Streptococcus mitis</td><td colspan="1" rowspan="1">ATCC 49456</td></tr><tr><td colspan="1" rowspan="1">Streptococcus oralis</td><td colspan="1" rowspan="1">ATCC 35037</td></tr><tr><td colspan="1" rowspan="1">Streptococcus parasanguinis</td><td colspan="1" rowspan="1">ATCC 15909</td></tr><tr><td colspan="1" rowspan="1">Streptococcus pneumoniae</td><td colspan="1" rowspan="1">ATCC 8335</td></tr><tr><td colspan="1" rowspan="1">Streptococcus pyogenes</td><td colspan="1" rowspan="1">ATCC12344</td></tr><tr><td colspan="1" rowspan="1">Streptococcus salivarius</td><td colspan="1" rowspan="1">ATCC7073</td></tr><tr><td colspan="1" rowspan="1">Streptococcus thoraltensis</td><td colspan="1" rowspan="1">ATCC700865E</td></tr></table>

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Strain</td></tr><tr><td rowspan=1 colspan=2>Pan Candida</td></tr><tr><td rowspan=2 colspan=1>Candida albicans</td><td rowspan=1 colspan=1>ATCC 24433</td></tr><tr><td rowspan=1 colspan=1>ATCC 90028</td></tr><tr><td rowspan=2 colspan=1>Candida glabrata</td><td rowspan=1 colspan=1>ATCC 2001</td></tr><tr><td rowspan=1 colspan=1>ATCC66032</td></tr><tr><td rowspan=3 colspan=1>Candida krusei</td><td rowspan=1 colspan=1>ATCC14243</td></tr><tr><td rowspan=1 colspan=1>ATCC 32196</td></tr><tr><td rowspan=1 colspan=1>ATCC 34135F</td></tr><tr><td rowspan=3 colspan=1>Candida parapsilosis</td><td rowspan=1 colspan=1>ATCC22019</td></tr><tr><td rowspan=1 colspan=1>ATCC58895</td></tr><tr><td rowspan=1 colspan=1>ATCC 90018G</td></tr></table>

A. Non-target species used to evaluate resistance marker.   
B. 5/6 replicates detected at $2 . 0 \mathrm { ~ x ~ } 1 0 ^ { 8 } \mathrm { C F U / m L }$ .   
C. 5/6 replicates detected at $4 . 5 \mathrm { ~ x ~ } 1 0 ^ { 8 } \mathrm { C F U / m L }$ .   
D. Strain may have reduced sensitivity and was not $1 0 0 \%$ detected at concentrations $< 1 \mathrm { ~ x ~ } 1 0 ^ { 8 } \mathrm { C F U / m L }$ .   
E. Strain may have reduced sensitivity and was not $1 0 0 \%$ detected at concentrations $< 4 \mathrm { ~ x ~ } 1 0 ^ { 8 } \mathrm { C F U / m L }$ .   
F. In initial testing, $1 / 6$ replicates was detected at $1 \mathrm { x \ 1 0 ^ { 6 } C F U / m L }$ ; during additional testing, 3/3 replicates detected at bottle positivity.   
G. In initial testing, 2/6 replicates were detected at $1 \mathrm { ~ x ~ } 1 0 ^ { 6 } \mathrm { C F U / m L }$ ; during additional testing, $6 / 6$ replicates detected at bottle positivity.

In silico analysis was performed for the broader genus or group-level assays of the ePlex BCID-GN Panel including: Citrobacter, Enterobacter cloacae complex, Enterobacter non-cloacae complex, Proteus, Serratia, Pan Candida, and Pan Gram-Positive assays. Tables 21-28 below include results from this analysis including species for which detection is predicted, species for which detection is predicted for a percentage of strain sequences, and species for which detection is not predicted.

Note: The performance of the ePlex BCID-GN Panel has not been established for the organisms listed in the tables below with the exception of species that were evaluated in the analytical reactivity study which are indicated with an asterisk.

Table 21: Predicted (in silico) reactivity for Citrobacter   

<table><tr><td rowspan=1 colspan=3>Detection Predicted for ≥95% of target sequences</td></tr><tr><td rowspan=1 colspan=1>Citrobacter koseri*</td><td rowspan=1 colspan=1>Citrobacter intermedius</td><td rowspan=1 colspan=1>Citrobacter amalonaticusA*</td></tr><tr><td rowspan=1 colspan=3>Detection Predicted for 85%-94% of target sequences</td></tr><tr><td rowspan=1 colspan=1>Citrobacter freundii*</td><td rowspan=1 colspan=1>Citrobacter braakii*</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=3>Detection Predicted for &lt;85.0% of target sequences</td></tr><tr><td rowspan=1 colspan=1>Citrobacter werkmanii* (66.7%)</td><td rowspan=1 colspan=1>Citrobacter youngae* (50.0%)</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=3>Detection Not Predicted</td></tr><tr><td rowspan=1 colspan=1>Citrobacter europaeus</td><td rowspan=1 colspan=1>Citrobacter gillenii</td><td rowspan=1 colspan=1></td></tr></table>

A. Detection predicted in silico, however ATCCBAA-2563 was not detected in wet testing. See Analytical Reactivity (Inclusivity) Study.

Table 22: Predicted (in silico) reactivity for Enterobacter cloacae complex   

<table><tr><td rowspan=1 colspan=3>Detection Predicted for ≥95% of target sequences</td></tr><tr><td rowspan=1 colspan=1>Enterobacter cloacae*</td><td rowspan=1 colspan=1>Enterobacter asburiae*</td><td rowspan=1 colspan=1>Enterobacter hormaechei*</td></tr><tr><td rowspan=1 colspan=1>Enterobacter xiangfangensis</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=3>Detection Predicted for 85%-94% of target sequences</td></tr><tr><td rowspan=1 colspan=3>None Identified</td></tr><tr><td rowspan=1 colspan=3>Detection Predicted for &lt;85.0% of target sequences</td></tr><tr><td rowspan=1 colspan=1>Enterobacter ludwigii* (68.4%)</td><td rowspan=1 colspan=1>Enterobacter nimipressuralis (25.0%)</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=3>Detection Not Predicted</td></tr><tr><td rowspan=1 colspan=1>Enterobacter kobei</td><td rowspan=1 colspan=1>Enterobacter cancerogenus</td><td rowspan=1 colspan=1></td></tr></table>

Table 23: Predicted (in silico) reactivity for Enterobacter (non-cloacae complex)   

<table><tr><td colspan="3" rowspan="1">Detection Predicted for ≥95% of target sequences</td></tr><tr><td colspan="1" rowspan="1">Enterobacter aerogenes*</td><td colspan="1" rowspan="1">Enterobacter gergoviae*</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="3" rowspan="1">Detection Predicted for 85%-94% of target sequences</td></tr><tr><td colspan="3" rowspan="1">None Identified</td></tr><tr><td colspan="3" rowspan="1">Detection Predicted for &lt;85.0% of target sequences</td></tr><tr><td colspan="1" rowspan="1">Enterobacter amnigenus* (62.5%)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="3" rowspan="1">Detection Not Predicted</td></tr><tr><td colspan="3" rowspan="1">None Identified</td></tr></table>

Table 24: Predicted (in silico) reactivity for Klebsiella pneumoniae group   

<table><tr><td rowspan=1 colspan=3>Detection Predicted for ≥95% of target sequences</td></tr><tr><td rowspan=1 colspan=1>Klebsiella pneumoniae*</td><td rowspan=1 colspan=1>Klebsiella quasipneumoniae*</td><td rowspan=1 colspan=1>Klebsiella variicola*</td></tr><tr><td rowspan=1 colspan=3>Detection Predicted for 85%-94% of target sequences</td></tr><tr><td rowspan=1 colspan=3>None Identified</td></tr><tr><td rowspan=1 colspan=3>Detection Predicted for &lt;85.0% of target sequences</td></tr><tr><td rowspan=1 colspan=3>None Identified</td></tr><tr><td rowspan=1 colspan=3>Detection Not Predicted</td></tr><tr><td rowspan=1 colspan=3>None Identified</td></tr></table>

Table 25: Predicted (in silico) reactivity for Proteus   

<table><tr><td rowspan=1 colspan=3>Detection Predicted for ≥95% of target sequences</td></tr><tr><td rowspan=1 colspan=1>Proteus mirabilis*</td><td rowspan=1 colspan=1>Proteus penneri*</td><td rowspan=1 colspan=1>Proteus vulgaris*</td></tr><tr><td rowspan=1 colspan=1>Proteus hauseri*</td><td rowspan=1 colspan=1>Proteus cibarius</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=3>Detection Predicted for 85%-94% of target sequences</td></tr><tr><td rowspan=1 colspan=3>None Identified</td></tr><tr><td rowspan=1 colspan=3>Detection Predicted for &lt;85.0% of target sequences</td></tr><tr><td rowspan=1 colspan=3>None Identified</td></tr><tr><td rowspan=1 colspan=3>Detection Not Predicted</td></tr><tr><td rowspan=1 colspan=3>Proteus myxofaciens</td></tr></table>

Table 26: Predicted (in silico) reactivity for Serratia   

<table><tr><td rowspan=1 colspan=3>Detection Predicted for ≥95% of target sequences</td></tr><tr><td rowspan=1 colspan=1>Serratia marcescens*</td><td rowspan=1 colspan=1>Serratia grimesii*</td><td rowspan=1 colspan=1>Serratia rubidaea*</td></tr><tr><td rowspan=1 colspan=1>Serratia ficaria*</td><td rowspan=1 colspan=1>Serratia liquefaciens*</td><td rowspan=1 colspan=1>Serratia proteamaculans</td></tr><tr><td rowspan=1 colspan=1>Serratia fonticola*</td><td rowspan=1 colspan=1>Serratia plymuthica*</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=3>Detection Predicted for 85%-94% of target sequences</td></tr><tr><td rowspan=1 colspan=3>None Identified</td></tr><tr><td rowspan=1 colspan=3>Detection Predicted for &lt;85.0% of target sequences</td></tr><tr><td rowspan=1 colspan=1>Serratia quinivorans (33.3%)</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=3>Detection Not Predicted</td></tr><tr><td rowspan=1 colspan=1>Serratia nematodiphila</td><td rowspan=1 colspan=1>Serratia odoriferaA*</td><td rowspan=1 colspan=1>Serratia ureilytica</td></tr></table>

A. Not predicted in silico, however ATCC 33077 was intermittently detected in wet testing (1/3 replicates detected). See Analytical Reactivity (Inclusivity) Study.

Table 27: Predicted (in silico) reactivity for Pan Candida   

<table><tr><td rowspan=1 colspan=3>Detection Predicted for ≥95% of target sequences</td></tr><tr><td rowspan=1 colspan=1>Candida albicans*</td><td rowspan=1 colspan=1>Candida glabrata*</td><td rowspan=1 colspan=1>Candida krusei*</td></tr><tr><td rowspan=1 colspan=1>Candida parapsilosis*</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=3>Detection Predicted for 85%-94% of target sequences</td></tr><tr><td rowspan=1 colspan=3>None Identified</td></tr><tr><td rowspan=1 colspan=3>Detection Predicted for &lt;85% of target sequences</td></tr><tr><td rowspan=1 colspan=3>None Identified</td></tr><tr><td rowspan=1 colspan=3>Detection Not Predicted</td></tr><tr><td rowspan=1 colspan=3>Detection was not predicted by bioinformatic analysis for other Candida species for which sequencedata was available.</td></tr></table>

Table 28: Predicted (in silico) reactivity for Pan Gram-Positive   

<table><tr><td colspan="3" rowspan="1">Detection Predicted for ≥95% of target sequences</td></tr><tr><td colspan="3" rowspan="1">Bacillus</td></tr><tr><td colspan="1" rowspan="1">Bacillus amyloliquefaciens*</td><td colspan="1" rowspan="1">Bacillus paralicheniformis</td><td colspan="1" rowspan="1">Bacillus toyonensis</td></tr><tr><td colspan="1" rowspan="1">Bacillus atrophaeus*</td><td colspan="1" rowspan="1">Bacillus siamensis</td><td colspan="1" rowspan="1">Bacillus vallismortis</td></tr><tr><td colspan="1" rowspan="1">Bacillus bombysepticus</td><td colspan="1" rowspan="1">Bacillus subtilis*</td><td colspan="1" rowspan="1">Bacillus velezensis</td></tr><tr><td colspan="1" rowspan="1">Bacillus licheniformis*</td><td colspan="1" rowspan="1">Bacillus tequilensis</td><td colspan="1" rowspan="1">Bacillus weihenstephanensis</td></tr><tr><td colspan="1" rowspan="1">Bacillus methylotrophicus</td><td colspan="1" rowspan="1">Bacillus thuringiensis*</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="3" rowspan="1">Enterococcus</td></tr><tr><td colspan="1" rowspan="1">Enterococcus avium*</td><td colspan="1" rowspan="1">Enterococcus faecium*</td><td colspan="1" rowspan="1">Enterococcus raffinosus*</td></tr><tr><td colspan="1" rowspan="1">Enterococcus dispar</td><td colspan="1" rowspan="1">Enterococcus flavescens</td><td colspan="1" rowspan="1">Enterococcus saccharolyticus*</td></tr><tr><td colspan="1" rowspan="1">Enterococcus durans</td><td colspan="1" rowspan="1">Enterococcus malodoratus</td><td colspan="1" rowspan="1">Enterococcus thailandicus</td></tr><tr><td colspan="1" rowspan="1">Enterococcus faecalis*</td><td colspan="1" rowspan="1">Enterococcus pseudoavium</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="3" rowspan="1">Staphylococcus</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus*</td><td colspan="1" rowspan="1">Staphylococcus haemolyticus*</td><td colspan="1" rowspan="1">Staphylococcuspseudintermedius</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus agnetis</td><td colspan="1" rowspan="1">Staphylococcus hominis*</td><td colspan="1" rowspan="1">Staphylococcuspseudolugdunensis</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus argensis</td><td colspan="1" rowspan="1">Staphylococcus hominis subsp.novobiosepticus</td><td colspan="1" rowspan="1">Staphylococcus pulvereri</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus argenteus</td><td colspan="1" rowspan="1">Staphylococcus hyicus*</td><td colspan="1" rowspan="1">Staphylococcus rostri</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus auricularis</td><td colspan="1" rowspan="1">Staphylococcus jettensis</td><td colspan="1" rowspan="1">Staphylococcus saprophyticus</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus capitis*</td><td colspan="1" rowspan="1">Staphylococcus kloosii</td><td colspan="1" rowspan="1">Staphylococcus schleiferi</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus caprae</td><td colspan="1" rowspan="1">Staphylococcus lentus *</td><td colspan="1" rowspan="1">Staphylococcus schweitzeri</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus carnosus</td><td colspan="1" rowspan="1">Staphylococcus lugdunensis*</td><td colspan="1" rowspan="1">Staphylococcus sciuri</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus chromogenes*</td><td colspan="1" rowspan="1">Staphylococcus lutrae</td><td colspan="1" rowspan="1">Staphylococcus simiae</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus cohnii*</td><td colspan="1" rowspan="1">Staphylococcus massiliensis</td><td colspan="1" rowspan="1">Staphylococcus simulans</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus delphini</td><td colspan="1" rowspan="1">Staphylococcus microti</td><td colspan="1" rowspan="1">Staphylococcus stepanovicii</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus devriesei</td><td colspan="1" rowspan="1">Staphylococcus muscae</td><td colspan="1" rowspan="1">Staphylococcus succinus</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus epidermidis*</td><td colspan="1" rowspan="1">Staphylococcus nepalensis</td><td colspan="1" rowspan="1">Staphylococcus vitulinus*</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus equorum</td><td colspan="1" rowspan="1">Staphylococcus pasteuri*</td><td colspan="1" rowspan="1">Staphylococcus warneri</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus felis</td><td colspan="1" rowspan="1">Staphylococcus petrasii</td><td colspan="1" rowspan="1">Staphylococcus xylosus</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus fleurettii</td><td colspan="1" rowspan="1">Staphylococcus pettenkoferi</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Staphylococcus gallinarum*</td><td colspan="1" rowspan="1">Staphylococcus piscifermentans</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="3" rowspan="1">Streptococcus</td></tr><tr><td colspan="1" rowspan="1">Streptococcus agalactiae</td><td colspan="1" rowspan="1">Streptococcus infantarius</td><td colspan="1" rowspan="1">Streptococcus phocae</td></tr><tr><td colspan="1" rowspan="1">Streptococcus alactolyticus</td><td colspan="1" rowspan="1">Streptococcus infantis*</td><td colspan="1" rowspan="1">Streptococcus pneumoniae*</td></tr><tr><td colspan="1" rowspan="1">Streptococcus anginosus</td><td colspan="1" rowspan="1">Streptococcus intermedius*</td><td colspan="1" rowspan="1">Streptococcus porcinus</td></tr><tr><td colspan="1" rowspan="1">Streptococcus australis</td><td colspan="1" rowspan="1">Streptococcus intestinalis</td><td colspan="1" rowspan="1">Streptococcus porcorum</td></tr><tr><td colspan="1" rowspan="1">Streptococcus caballi</td><td colspan="1" rowspan="1">Streptococcus lactarius</td><td colspan="1" rowspan="1">Streptococcuspseudopneumoniae</td></tr><tr><td colspan="1" rowspan="1">Streptococcus constellatus</td><td colspan="1" rowspan="1">Streptococcusloxodontisalivarius</td><td colspan="1" rowspan="1">Streptococcus pseudoporcinus</td></tr><tr><td colspan="1" rowspan="1">Streptococcus criceti</td><td colspan="1" rowspan="1">Streptococcus luteciae</td><td colspan="1" rowspan="1">Strepotococcus pyogenes</td></tr><tr><td colspan="1" rowspan="1">Streptococcus cristatus</td><td colspan="1" rowspan="1">Streptococcus lutetiensis</td><td colspan="1" rowspan="1">Streptococcus rifensis</td></tr><tr><td colspan="1" rowspan="1">Streptococcus danieliae</td><td colspan="1" rowspan="1">Streptococcus macedonicus</td><td colspan="1" rowspan="1">Streptococcus rubneri</td></tr><tr><td colspan="1" rowspan="1">Streptococcus dentasini</td><td colspan="1" rowspan="1">Streptococcus marimammalium</td><td colspan="1" rowspan="1">Streptococcus salivarius*</td></tr><tr><td colspan="1" rowspan="1">Streptococcus dentisani</td><td colspan="1" rowspan="1">Streptococcus massiliensis</td><td colspan="1" rowspan="1">Streptococcus saliviloxodontae</td></tr><tr><td colspan="1" rowspan="1">Streptococcus didelphis</td><td colspan="1" rowspan="1">Streptococcus mitis*</td><td colspan="1" rowspan="1">Streptococcus sanguinis</td></tr><tr><td colspan="1" rowspan="1">Streptococcus difficilis</td><td colspan="1" rowspan="1">Streptococcus moroccensis</td><td colspan="1" rowspan="1">Streptococcus seminale</td></tr><tr><td colspan="1" rowspan="1">Streptococcus dysgalactiaesubsp. dysgalactiae</td><td colspan="1" rowspan="1">Streptococcus oligofermentans</td><td colspan="1" rowspan="1">Streptococcus sinensis</td></tr><tr><td colspan="1" rowspan="1">Streptococcus dysgalactiaesubsp. equisimilis</td><td colspan="1" rowspan="1">Streptococcus oralis*</td><td colspan="1" rowspan="1">Streptococcus suis</td></tr><tr><td colspan="1" rowspan="1">Streptococcus dysgalactiae*</td><td colspan="1" rowspan="1">Streptococcus oricebi</td><td colspan="1" rowspan="1">Streptococcus thermophilus</td></tr><tr><td colspan="1" rowspan="1">Streptococcus equi*</td><td colspan="1" rowspan="1">Streptococcus orisratti</td><td colspan="1" rowspan="1">Streptococcus thoraltensis*</td></tr><tr><td colspan="1" rowspan="1">Streptococcus equinus</td><td colspan="1" rowspan="1">Streptococcus panodentis</td><td colspan="1" rowspan="1">Streptococcus tigurinus</td></tr><tr><td colspan="1" rowspan="1">Streptococcus fryi</td><td colspan="1" rowspan="1">Streptococcus parasanguinis*</td><td colspan="1" rowspan="1">Streptococcus troglodytae</td></tr><tr><td colspan="1" rowspan="1">Streptococcus gallolyticus*</td><td colspan="1" rowspan="1">Streptococcus parasuis</td><td colspan="1" rowspan="1">Streptococcus troglodytidis</td></tr><tr><td colspan="1" rowspan="1">Streptococcus gordonii*</td><td colspan="1" rowspan="1">Streptococcus parauberis</td><td colspan="1" rowspan="1">Streptococcus urinalis</td></tr><tr><td colspan="1" rowspan="1">Streptococcus himalayensis</td><td colspan="1" rowspan="1">Streptococcus pasteuri</td><td colspan="1" rowspan="1">Streptococcus ursoris</td></tr><tr><td colspan="1" rowspan="1">Streptococcus hongkongensis</td><td colspan="1" rowspan="1">Streptococcus pasteurianus</td><td colspan="1" rowspan="1">Streptococcus vestibularis</td></tr><tr><td colspan="1" rowspan="1">Streptococcus hyointestinalis</td><td colspan="1" rowspan="1">Streptococcus peroris</td><td colspan="1" rowspan="1">Streptococcus waiu</td></tr><tr><td colspan="3" rowspan="1">Detection Predicted for 85%-94% of target sequences</td></tr><tr><td colspan="1" rowspan="1">Bacillus cereus *</td><td colspan="1" rowspan="1">Enterococcus hirae*</td><td colspan="1" rowspan="1">Staphylococcus saccharolyticus</td></tr><tr><td colspan="1" rowspan="1">Enterococcus casseliflavus*</td><td colspan="1" rowspan="1">Staphylococcus arlettae</td><td colspan="1" rowspan="1">Streptococcus bovis</td></tr><tr><td colspan="1" rowspan="1">Enterococcus cecorum</td><td colspan="1" rowspan="1">Staphylococcus condimenti</td><td colspan="1" rowspan="1">Streptococcus uberis</td></tr><tr><td colspan="1" rowspan="1">Enterococcus gallinarum</td><td colspan="1" rowspan="1">Staphylococcus intermedius</td><td colspan="1" rowspan="1">Y</td></tr><tr><td colspan="3" rowspan="1">Detection Predicted for &lt;85.0% of target sequences</td></tr><tr><td colspan="1" rowspan="1">Bacillus mojavensis (77.8%)</td><td colspan="1" rowspan="1">Bacillus sonorensis (83.3%)</td><td colspan="1" rowspan="1">Streptococcus halichoeri(66.7%)</td></tr><tr><td colspan="1" rowspan="1">Streptococcus ratti (75.0%)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Y</td></tr><tr><td colspan="3" rowspan="1">Detection Not Predicted</td></tr><tr><td colspan="1" rowspan="1">Bacillus pseudomycoides</td><td colspan="1" rowspan="1">Enterococcus sulfureus</td><td colspan="1" rowspan="1">Streptococcus hyovaginalis</td></tr><tr><td colspan="1" rowspan="1">Enterococcus aquimarinus</td><td colspan="1" rowspan="1">Enterococcus termitis</td><td colspan="1" rowspan="1">Streptococcus ictaluri</td></tr><tr><td colspan="1" rowspan="1">Enterococcus asini</td><td colspan="1" rowspan="1">Enterococcus ureasiticus</td><td colspan="1" rowspan="1">Streptococcus iniae</td></tr><tr><td colspan="1" rowspan="1">Enterococcus caccae</td><td colspan="1" rowspan="1">Enterococcus ureilyticus</td><td colspan="1" rowspan="1">Streptococcus lactis</td></tr><tr><td colspan="1" rowspan="1">Enterococcus camelliae</td><td colspan="1" rowspan="1">Enterococcus villorum</td><td colspan="1" rowspan="1">Streptococcus macacae</td></tr><tr><td colspan="1" rowspan="1">Enterococcus canintestini</td><td colspan="1" rowspan="1">Staphylococcus caseolyticus</td><td colspan="1" rowspan="1">Streptococcus marmotae</td></tr><tr><td colspan="1" rowspan="1">Enterococcus canis</td><td colspan="1" rowspan="1">Streptococcus acidominimus</td><td colspan="1" rowspan="1">Streptococcus merionis</td></tr><tr><td colspan="1" rowspan="1">Enterococcus columbae</td><td colspan="1" rowspan="1">Streptococcus azizii</td><td colspan="1" rowspan="1">Streptococcus milleri</td></tr><tr><td colspan="1" rowspan="1">Enterococcus devriesei</td><td colspan="1" rowspan="1">Streptococcus cameli</td><td colspan="1" rowspan="1">Streptococcus minor</td></tr><tr><td colspan="1" rowspan="1">Enterococcus haemoperoxidus</td><td colspan="1" rowspan="1">Streptococcus canis</td><td colspan="1" rowspan="1">Streptococcus oriloxodontae</td></tr><tr><td colspan="1" rowspan="1">Enterococcus hawaiiensis</td><td colspan="1" rowspan="1">Streptococcus castoreus</td><td colspan="1" rowspan="1">Streptococcus orisasini</td></tr><tr><td colspan="1" rowspan="1">Enterococcus hermanniensis</td><td colspan="1" rowspan="1">Streptococcus cremoris</td><td colspan="1" rowspan="1">Streptococcus orisuis</td></tr><tr><td colspan="1" rowspan="1">Enterococcus italicus</td><td colspan="1" rowspan="1">Streptococcus criae</td><td colspan="1" rowspan="1">Streptococcus ovis</td></tr><tr><td colspan="1" rowspan="1">Enterococcus mundtii</td><td colspan="1" rowspan="1">Streptococcus cuniculi</td><td colspan="1" rowspan="1">Streptococcus pharyngis</td></tr><tr><td colspan="1" rowspan="1">Enterococcus pallens</td><td colspan="1" rowspan="1">Streptococcus dentapri</td><td colspan="1" rowspan="1">Streptococcus pluranimalium</td></tr><tr><td colspan="1" rowspan="1">Enterococcus pernyi</td><td colspan="1" rowspan="1">Streptococcus dentiloxodontae</td><td colspan="1" rowspan="1">Streptococcus plurextorum</td></tr><tr><td colspan="1" rowspan="1">Enterococcus phoeniculicola</td><td colspan="1" rowspan="1">Streptococcus dentirousetti</td><td colspan="1" rowspan="1">Streptococcus plutanimalium</td></tr><tr><td colspan="1" rowspan="1">Enterococcus plantarum</td><td colspan="1" rowspan="1">Streptococcus devriesei</td><td colspan="1" rowspan="1">Streptococcus porci</td></tr><tr><td colspan="1" rowspan="1">Enterococcus quebecensis</td><td colspan="1" rowspan="1">Streptococcus downei</td><td colspan="1" rowspan="1">Streptococcus rupicaprae</td></tr><tr><td colspan="1" rowspan="1">Enterococcus ratti</td><td colspan="1" rowspan="1">Streptococcus entericus</td><td colspan="1" rowspan="1">Streptococcus sobrinus</td></tr><tr><td colspan="1" rowspan="1">Enterococcus rivorum</td><td colspan="1" rowspan="1">Streptococcus ferus</td><td colspan="1" rowspan="1">Streptococcus tangierensis</td></tr><tr><td colspan="1" rowspan="1">Enterococcus rotai</td><td colspan="1" rowspan="1">Streptococcus gallinaceus</td><td colspan="1" rowspan="2"></td></tr><tr><td colspan="1" rowspan="1">Enterococcus silesiacus</td><td colspan="1" rowspan="1">Streptococcus henryi</td></tr></table>

# Predicted Reactivity (In silico analysis) for resistance markers

Known variants of the six resistance markers reported by the ePlex BCID-GN Panel were each assessed for predicted reactivity using in silico analysis. BLAST searches to support in silico analyses were conducted in March 2018; therefore, any new resistance gene variants deposited into GenBank after March 2018 have not been evaluated.

Tables 29-39 below describe the predicted reactivity for CTX-M, IMP, KPC, NDM, OXA, and VIM resistance marker assays of the ePlex BCID GN Panel. Strains that were tested as part of the Inclusivity study are marked with an asterisk. Table 40 includes all variants that are not predicted to be detected by the ePlex BCID-GN Panel.

The CTX-M assay on the ePlex BCID-GN Panel is designed to detect the following CTX-M groups: CTX-M-1, CTX-M-2, CTX-M-8, CTX-M-9, and CTX-M-25. Results from in silico results for variants included in each of these CTX-M groups are presented in Table 29-33.

Table 29: Predicted (in silico) reactivity for CTX-M-1 group   

<table><tr><td rowspan=1 colspan=1>Target</td><td rowspan=1 colspan=2>AssociatedOrganism</td><td rowspan=1 colspan=2>VariantDetected</td><td rowspan=1 colspan=2></td><td rowspan=1 colspan=3>Target</td><td rowspan=1 colspan=4>AssociatedOrganism</td><td rowspan=1 colspan=1>VariantDetected</td></tr><tr><td rowspan=2 colspan=1>Acinetobacterbaumannii</td><td rowspan=2 colspan=2>Acinetobacterbaumannii</td><td rowspan=2 colspan=2>CTX-M-15</td><td rowspan=3 colspan=2></td><td rowspan=3 colspan=3></td><td rowspan=2 colspan=4></td><td rowspan=1 colspan=1>CTX-M-58</td></tr><tr><td rowspan=1 colspan=1>CTX-M-61</td></tr><tr><td rowspan=8 colspan=1>Citrobacter</td><td rowspan=6 colspan=2>Citrobacterfreundii</td><td rowspan=1 colspan=2>CTX-M-3</td><td rowspan=1 colspan=2></td><td rowspan=1 colspan=4></td><td rowspan=1 colspan=1>CTX-M-65</td></tr><tr><td rowspan=1 colspan=2>CTX-M-15*</td><td rowspan=1 colspan=1></td><td></td><td rowspan=1 colspan=3></td><td rowspan=1 colspan=4></td><td rowspan=1 colspan=1>CTX-M-69</td></tr><tr><td rowspan=3 colspan=2>CTX-M-30</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr><tr><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr><tr><td rowspan=2 colspan=2>−</td><td rowspan=2 colspan=3></td><td rowspan=1 colspan=4></td><td rowspan=1 colspan=1>CTX-M-79</td></tr><tr><td rowspan=1 colspan=2>CTX-M-55</td><td rowspan=1 colspan=4></td><td rowspan=1 colspan=1>CTX-M-82</td></tr><tr><td rowspan=2 colspan=2>Citrobacter koseri</td><td rowspan=1 colspan=2>CTX-M-3</td><td rowspan=6 colspan=2></td><td rowspan=6 colspan=3></td><td rowspan=1 colspan=4></td><td rowspan=1 colspan=1>CTX-M-101</td></tr><tr><td rowspan=1 colspan=2>CTX-M-15</td><td rowspan=1 colspan=4></td><td rowspan=1 colspan=1>CTX-M-103</td></tr><tr><td rowspan=4 colspan=1>Enterobacter(non-cloacaeComplex)</td><td rowspan=2 colspan=2>Enterobacteraerogenes</td><td rowspan=1 colspan=2>CTX-M-3</td><td rowspan=1 colspan=4></td><td rowspan=1 colspan=1>CTX-M-117</td></tr><tr><td rowspan=1 colspan=2>CTX-M-15</td><td rowspan=1 colspan=4></td><td rowspan=1 colspan=1>CTX-M-123</td></tr><tr><td rowspan=2 colspan=2>Enterobactergergoviae</td><td rowspan=2 colspan=2>CTX-M-15</td><td rowspan=2 colspan=4></td><td rowspan=1 colspan=1>CTX-M-127</td></tr><tr><td rowspan=19 colspan=5>b</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>CTX-M-132</td></tr><tr><td rowspan=12 colspan=1>Enterobactercloacae Complex</td><td rowspan=2 colspan=2>Enterobacterasburiae</td><td rowspan=2 colspan=2>CTX-M-15</td><td rowspan=2 colspan=2></td><td rowspan=2 colspan=4></td><td rowspan=1 colspan=1>CTX-M-138</td></tr><tr><td rowspan=1 colspan=1>CTX-M-139</td></tr><tr><td rowspan=6 colspan=2>Enterobactercloacae</td><td rowspan=1 colspan=2>CTX-M-3</td><td rowspan=1 colspan=4></td><td rowspan=1 colspan=1>CTX-M-142</td></tr><tr><td rowspan=1 colspan=2>CTX-M-15*</td><td rowspan=1 colspan=4></td><td rowspan=1 colspan=1>CTX-M-144</td></tr><tr><td rowspan=1 colspan=2>CTX-M-22</td><td rowspan=1 colspan=4></td><td rowspan=1 colspan=1>CTX-M-150</td></tr><tr><td rowspan=1 colspan=2>CTX-M-37</td><td rowspan=1 colspan=4></td><td rowspan=1 colspan=1>CTX-M-158</td></tr><tr><td rowspan=1 colspan=2>CTX-M-89</td><td rowspan=1 colspan=4></td><td rowspan=1 colspan=1>CTX-M-163</td></tr><tr><td rowspan=1 colspan=2>CTX-M-177</td><td rowspan=1 colspan=2></td><td rowspan=1 colspan=3></td><td rowspan=1 colspan=1>CTX-M-166</td></tr><tr><td rowspan=4 colspan=2>Enterobacterhormaechei</td><td rowspan=4 colspan=2>CTX-M-15</td><td rowspan=10 colspan=2></td><td rowspan=4 colspan=4></td><td></td></tr><tr><td rowspan=1 colspan=1></td><td></td></tr><tr><td rowspan=1 colspan=1>CTX-M-169</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>CTX-M-170</td></tr><tr><td rowspan=24 colspan=1>Escherichia coli</td><td rowspan=24 colspan=2>Escherichia coli</td><td rowspan=1 colspan=2>CTX-M-1*</td><td rowspan=1 colspan=1></td><td></td><td rowspan=1 colspan=1></td><td></td><td rowspan=1 colspan=1>CTX-M-171</td></tr><tr><td rowspan=1 colspan=2>CTX-M-3*</td><td rowspan=1 colspan=4></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>CTX-M-172</td></tr><tr><td rowspan=1 colspan=2>CTX-M-10</td><td rowspan=1 colspan=2></td><td rowspan=1 colspan=3></td><td rowspan=1 colspan=1>CTX-M-174</td></tr><tr><td rowspan=2 colspan=2>CTX-M-12</td><td rowspan=2 colspan=2></td><td></td><td></td><td></td></tr><tr><td rowspan=1 colspan=3></td><td rowspan=1 colspan=2></td><td rowspan=1 colspan=1>CTX-M-175</td></tr><tr><td rowspan=1 colspan=2>CTX-M-14*</td><td rowspan=1 colspan=2></td><td rowspan=1 colspan=3></td><td rowspan=1 colspan=1>CTX-M-179</td></tr><tr><td rowspan=1 colspan=2>CTX-M-15*</td><td rowspan=1 colspan=2></td><td rowspan=1 colspan=3></td><td rowspan=1 colspan=3></td><td rowspan=1 colspan=2></td><td rowspan=1 colspan=1>CTX-M-180</td></tr><tr><td rowspan=2 colspan=2>CTX-M-22</td><td rowspan=2 colspan=1></td><td></td><td rowspan=2 colspan=3></td><td rowspan=2 colspan=2></td><td></td><td></td><td></td></tr><tr><td></td><td rowspan=1 colspan=3></td><td rowspan=2 colspan=2></td><td rowspan=2 colspan=1>CTX-M-181</td></tr><tr><td rowspan=3 colspan=2>CTX-M-23</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr><tr><td rowspan=7 colspan=2></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr><tr><td rowspan=6 colspan=3></td><td rowspan=6 colspan=4></td><td rowspan=1 colspan=1>CTX-M-182</td></tr><tr><td rowspan=1 colspan=1>i</td><td rowspan=1 colspan=2>CTX-M-28</td><td></td></tr><tr><td></td><td></td><td></td></tr><tr><td rowspan=2 colspan=1></td><td></td><td></td><td></td></tr><tr><td></td><td></td><td rowspan=1 colspan=1>CTX-M-184</td></tr><tr><td rowspan=1 colspan=2>CTX-M-29</td><td rowspan=1 colspan=2></td><td rowspan=1 colspan=1>CTX-M-186</td></tr><tr><td rowspan=1 colspan=2>CTX-M-32</td><td rowspan=1 colspan=2></td><td rowspan=4 colspan=3>Klebsiella oxytoca</td><td rowspan=4 colspan=4>Klebsiella oxytoca</td><td rowspan=1 colspan=1>CTX-M-3</td></tr><tr><td rowspan=1 colspan=2>CTX-M-33</td><td rowspan=1 colspan=1>CTX-M-15</td></tr><tr><td rowspan=1 colspan=2>CTX-M-34</td><td rowspan=1 colspan=1>CTX-M-28</td></tr><tr><td rowspan=1 colspan=2>CTX-M-36</td><td rowspan=1 colspan=2></td><td rowspan=1 colspan=1>CTX-M-162</td></tr><tr><td rowspan=1 colspan=2>CTX-M-38</td><td rowspan=3 colspan=10></td></tr><tr><td rowspan=1 colspan=2>CTX-M-42</td></tr><tr><td rowspan=1 colspan=2>CTX-M-55</td></tr></table>

<table><tr><td rowspan=1 colspan=1>Target</td><td rowspan=1 colspan=6>AssociatedOrganism</td><td rowspan=1 colspan=1>VariantDetected</td></tr><tr><td rowspan=30 colspan=1>Klebsiellapneumoniae group</td><td rowspan=1 colspan=6></td><td rowspan=1 colspan=1>CTX-M-1</td></tr><tr><td rowspan=8 colspan=6></td><td rowspan=1 colspan=1>CTX-M-3</td></tr><tr><td rowspan=1 colspan=1>CTX-M-10</td></tr><tr><td rowspan=2 colspan=1></td><td rowspan=2 colspan=1>CTX-M-11</td></tr><tr><td rowspan=1 colspan=4></td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>CTX-M-12</td></tr><tr><td rowspan=1 colspan=1>CTX-M-15*</td></tr><tr><td rowspan=1 colspan=1>CTX-M-22</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>CTX-M-28</td></tr><tr><td rowspan=4 colspan=6></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>CTX-M-32</td></tr><tr><td rowspan=1 colspan=1>CTX-M-52</td></tr><tr><td rowspan=1 colspan=1>CTX-M-54</td></tr><tr><td rowspan=1 colspan=2></td><td rowspan=1 colspan=1>CTX-M-55</td></tr><tr><td rowspan=1 colspan=5>Klebsiella</td><td rowspan=1 colspan=2></td><td></td></tr><tr><td rowspan=4 colspan=6>pneumoniae</td><td></td></tr><tr><td rowspan=1 colspan=1></td></tr><tr><td rowspan=2 colspan=1>CTX-M-60</td></tr><tr><td rowspan=1 colspan=1></td></tr><tr><td rowspan=12 colspan=6>b</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>CTX-M-62</td></tr><tr><td rowspan=1 colspan=1>CTX-M-71</td></tr><tr><td rowspan=1 colspan=1>CTX-M-72</td></tr><tr><td rowspan=1 colspan=1>CTX-M-96</td></tr><tr><td rowspan=1 colspan=1>CTX-M-155</td></tr><tr><td rowspan=1 colspan=1>CTX-M-156</td></tr><tr><td rowspan=1 colspan=1>CTX-M-157</td></tr><tr><td rowspan=1 colspan=1>CTX-M-173</td></tr><tr><td rowspan=1 colspan=1>CTX-M-176</td></tr><tr><td rowspan=1 colspan=1>CTX-M-183</td></tr><tr><td rowspan=1 colspan=1>CTX-M-197</td></tr><tr><td rowspan=1 colspan=1>CTX-M-204</td></tr><tr><td rowspan=3 colspan=1>Morganellamorganii</td><td rowspan=3 colspan=6>Morganellamorganii</td><td rowspan=1 colspan=1>CTX-M-3</td></tr><tr><td rowspan=1 colspan=1>CTX-M-15*</td></tr><tr><td rowspan=1 colspan=1>CTX-M-55</td></tr></table>

<table><tr><td rowspan=1 colspan=1>Target</td><td rowspan=1 colspan=1>AssociatedOrganism</td><td rowspan=1 colspan=1>VariantDetected</td></tr><tr><td rowspan=9 colspan=1>Proteus mirabilis</td><td rowspan=9 colspan=1>Proteus mirabilis</td><td rowspan=1 colspan=1>CTX-M-1</td></tr><tr><td rowspan=1 colspan=1>CTX-M-3</td></tr><tr><td rowspan=1 colspan=1>CTX-M-15</td></tr><tr><td rowspan=1 colspan=1>CTX-M-32</td></tr><tr><td rowspan=1 colspan=1>CTX-M-66</td></tr><tr><td rowspan=1 colspan=1>CTX-M-116</td></tr><tr><td rowspan=1 colspan=1>CTX-M-136</td></tr><tr><td rowspan=1 colspan=1>CTX-M-164</td></tr><tr><td rowspan=1 colspan=1>CTX-M-167</td></tr><tr><td rowspan=3 colspan=1>Pseudomonasaeruginosa</td><td rowspan=3 colspan=1>Pseudomonasaeruginosa</td><td rowspan=1 colspan=1>CTX-M-15</td></tr><tr><td rowspan=1 colspan=1>CTX-M-28</td></tr><tr><td rowspan=1 colspan=1>CTX-M-32</td></tr><tr><td rowspan=9 colspan=1>Salmonella</td><td rowspan=6 colspan=1>Salmonellaenterica</td><td rowspan=1 colspan=1>CTX-M-15</td></tr><tr><td rowspan=1 colspan=1>CTX-M-53</td></tr><tr><td rowspan=1 colspan=1>CTX-M-55</td></tr><tr><td rowspan=1 colspan=1>CTX-M-57</td></tr><tr><td rowspan=1 colspan=1>CTX-M-61</td></tr><tr><td rowspan=1 colspan=1>CTX-M-88</td></tr><tr><td rowspan=1 colspan=1>Salmonella sp.</td><td rowspan=1 colspan=1>CTX-M-3</td></tr><tr><td rowspan=2 colspan=1>Salmonellatyphimurium</td><td rowspan=1 colspan=1>CTX-M-37</td></tr><tr><td rowspan=1 colspan=1>CTX-M-61</td></tr><tr><td rowspan=1 colspan=1>Serratia</td><td rowspan=1 colspan=1>Serratialiquefaciens</td><td rowspan=1 colspan=1>CTX-M-22</td></tr><tr><td rowspan=3 colspan=1>Serratiamarcescens/Serratia</td><td rowspan=3 colspan=1>Serratiamarcescens</td><td rowspan=1 colspan=1>CTX-M-3</td></tr><tr><td rowspan=1 colspan=1>CTX-M-15</td></tr><tr><td rowspan=1 colspan=1>CTX-M-55</td></tr><tr><td rowspan=1 colspan=1>Stenotrophomonasmaltophilia</td><td rowspan=1 colspan=1>Stenotrophomonasmaltophilia</td><td rowspan=1 colspan=1>CTX-M-15</td></tr></table>

Table 30: Predicted (in silico) reactivity for CTX-M-2 group   

<table><tr><td rowspan=1 colspan=1>Target</td><td rowspan=1 colspan=1>AssociatedOrganism</td><td rowspan=1 colspan=1>VariantDetected</td></tr><tr><td rowspan=4 colspan=1>Acinetobacterbaumannii</td><td rowspan=4 colspan=1>Acinetobacterbaumannii</td><td rowspan=1 colspan=1>CTX-M-2</td></tr><tr><td rowspan=1 colspan=1>CTX-M-5</td></tr><tr><td rowspan=1 colspan=1>CTX-M-43</td></tr><tr><td rowspan=1 colspan=1>CTX-M-115</td></tr><tr><td rowspan=1 colspan=1>Enterobactercloacae complex</td><td rowspan=1 colspan=1>Enterobactercloacae</td><td rowspan=1 colspan=1>CTX-M-5</td></tr><tr><td rowspan=6 colspan=1>Escherichia coli</td><td rowspan=6 colspan=1>Escherichia coli</td><td rowspan=1 colspan=1>CTX-M-2*</td></tr><tr><td rowspan=1 colspan=1>CTX-M-44</td></tr><tr><td rowspan=1 colspan=1>CTX-M-56</td></tr><tr><td rowspan=1 colspan=1>CTX-M-92</td></tr><tr><td rowspan=1 colspan=1>CTX-M-97</td></tr><tr><td rowspan=1 colspan=1>CTX-M-124</td></tr><tr><td rowspan=5 colspan=1>KlebsiellapneumoniaeGroup</td><td rowspan=5 colspan=1>Klebsiellapneumoniae</td><td rowspan=1 colspan=1>CTX-M-2</td></tr><tr><td rowspan=1 colspan=1>CTX-M-35</td></tr><tr><td rowspan=1 colspan=1>CTX-M-59</td></tr><tr><td rowspan=1 colspan=1>CTX-M-141</td></tr><tr><td rowspan=1 colspan=1>CTX-M-165</td></tr></table>

<table><tr><td rowspan=1 colspan=1>Target</td><td rowspan=1 colspan=1>AssociatedOrganism</td><td rowspan=1 colspan=1>VariantDetected</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>CTX-M-200</td></tr><tr><td rowspan=1 colspan=1>Morganellamorganii</td><td rowspan=1 colspan=1>Morganellamorganii</td><td rowspan=1 colspan=1>CTX-M-2</td></tr><tr><td rowspan=3 colspan=1>Proteusmirabilis/Proteus</td><td rowspan=3 colspan=1>Proteus mirabilis</td><td rowspan=1 colspan=1>CTX-M-2</td></tr><tr><td rowspan=1 colspan=1>CTX-M-20</td></tr><tr><td rowspan=1 colspan=1>CTX-M-171</td></tr><tr><td rowspan=2 colspan=1>Pseudomonasaeruginosa</td><td rowspan=2 colspan=1>Pseudomonasaeruginosa</td><td rowspan=1 colspan=1>CTX-1-2</td></tr><tr><td rowspan=1 colspan=1>CTX-M-2</td></tr><tr><td rowspan=9 colspan=1>Salmonella</td><td rowspan=5 colspan=1>Salmonellaenterica</td><td rowspan=1 colspan=1>CTX-M-2</td></tr><tr><td rowspan=1 colspan=1>CTX-M-4</td></tr><tr><td rowspan=1 colspan=1>CTX-M-5</td></tr><tr><td rowspan=1 colspan=1>CTX-M-6</td></tr><tr><td rowspan=1 colspan=1>CTX-M-7</td></tr><tr><td rowspan=4 colspan=1>Salmonellatyphimurium</td><td rowspan=1 colspan=1>CTX-M-2</td></tr><tr><td rowspan=1 colspan=1>CTX-M-4</td></tr><tr><td rowspan=1 colspan=1>CTX-M-5</td></tr><tr><td rowspan=1 colspan=1>CTX-M-7</td></tr></table>

Table 31: Predicted (in silico) reactivity for CTX-M-8 group   

<table><tr><td rowspan=1 colspan=1>Target</td><td rowspan=1 colspan=1>AssociatedOrganism</td><td rowspan=1 colspan=1>VariantDetected</td></tr><tr><td rowspan=1 colspan=1>Citrobacter</td><td rowspan=1 colspan=1>Citrobacteramalonaticus</td><td rowspan=1 colspan=1>CTX-M-8</td></tr><tr><td rowspan=1 colspan=1>EnterobactercloacaeComplex</td><td rowspan=1 colspan=1>Enterobactercloacae</td><td rowspan=1 colspan=1>CTX-M-8</td></tr><tr><td rowspan=1 colspan=1>Escherichia coli</td><td rowspan=1 colspan=1>Escherichiacoli</td><td rowspan=1 colspan=1>CTX-M-8*</td></tr></table>

<table><tr><td rowspan=1 colspan=1>Target</td><td rowspan=1 colspan=1>AssociatedOrganism</td><td rowspan=1 colspan=1>VariantDetected</td></tr><tr><td rowspan=2 colspan=1>KlebsiellapneumoniaeGroup</td><td rowspan=2 colspan=1>Klebsiellapneumoniae</td><td rowspan=1 colspan=1>CTX-M-8</td></tr><tr><td rowspan=1 colspan=1>CTX-M-63</td></tr><tr><td rowspan=1 colspan=1>Morganellamorganii</td><td rowspan=1 colspan=1>Morganellamorganii</td><td rowspan=1 colspan=1>CTX-M-63</td></tr><tr><td rowspan=1 colspan=1>Salmonella</td><td rowspan=1 colspan=1>Salmonellaenterica</td><td rowspan=1 colspan=1>CTX-M-8</td></tr></table>

Table 32: Predicted (in silico) reactivity for CTX-M-9 group   

<table><tr><td rowspan=1 colspan=1>Target</td><td rowspan=1 colspan=3>AssociatedOrganism</td><td rowspan=1 colspan=1>VariantDetected</td></tr><tr><td rowspan=2 colspan=1>Citrobacter</td><td rowspan=2 colspan=3>Citrobacterfreundii</td><td rowspan=1 colspan=1>CTX-M-14</td></tr><tr><td rowspan=1 colspan=1>CTX-M-65</td></tr><tr><td rowspan=1 colspan=1>Enterobacternon-cloacaeComplex</td><td rowspan=1 colspan=3>Enterobacteraerogenes</td><td rowspan=1 colspan=1>CTX-M-9</td></tr><tr><td rowspan=5 colspan=1>EnterobactercloacaeComplex</td><td rowspan=5 colspan=3>Enterobactercloacae</td><td rowspan=1 colspan=1>CTX-M-9*</td></tr><tr><td rowspan=1 colspan=1>CTX-M-13</td></tr><tr><td rowspan=1 colspan=1>CTX-M-14</td></tr><tr><td rowspan=1 colspan=1>CTX-M-64</td></tr><tr><td rowspan=1 colspan=1>CTX-M-125</td></tr><tr><td rowspan=26 colspan=1>Escherichia coli</td><td rowspan=26 colspan=3>Escherichiacoli</td><td rowspan=1 colspan=1>CTX-M-1/CTX-M-65</td></tr><tr><td rowspan=1 colspan=1>CTX-M-9</td></tr><tr><td rowspan=1 colspan=1>CTX-M-13</td></tr><tr><td rowspan=1 colspan=1>CTX-M-14</td></tr><tr><td rowspan=1 colspan=1>CTX-M-14/CTX-M-15</td></tr><tr><td rowspan=1 colspan=1>CTX-M-15</td></tr><tr><td rowspan=1 colspan=1>CTX-M-16</td></tr><tr><td rowspan=1 colspan=1>CTX-M-19</td></tr><tr><td rowspan=1 colspan=1>CTX-M-21</td></tr><tr><td rowspan=1 colspan=1>CTX-M-24</td></tr><tr><td rowspan=1 colspan=1>CTX-M-27</td></tr><tr><td rowspan=1 colspan=1>CTX-M-38</td></tr><tr><td rowspan=1 colspan=1>CTX-M-47</td></tr><tr><td rowspan=1 colspan=1>CTX-M-51</td></tr><tr><td rowspan=1 colspan=1>CTX-M-64</td></tr><tr><td rowspan=1 colspan=1>CTX-M-65</td></tr><tr><td rowspan=1 colspan=1>CTX-M-67</td></tr><tr><td rowspan=1 colspan=1>CTX-M-73</td></tr><tr><td rowspan=1 colspan=1>CTX-M-82</td></tr><tr><td rowspan=1 colspan=1>CTX-M-87</td></tr><tr><td rowspan=1 colspan=1>CTX-M-93</td></tr><tr><td rowspan=1 colspan=2></td><td rowspan=1 colspan=1>CTX-M-97</td></tr><tr><td rowspan=1 colspan=2></td><td rowspan=1 colspan=1>CTX-M-98</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>CTX-M-102</td></tr><tr><td rowspan=1 colspan=1>CTX-M-104</td></tr><tr><td rowspan=1 colspan=1>CTX-M-105</td></tr></table>

<table><tr><td colspan="1" rowspan="1">Target</td><td colspan="1" rowspan="1">AssociatedOrganism</td><td colspan="1" rowspan="1">VariantDetected</td></tr><tr><td colspan="1" rowspan="21"></td><td colspan="1" rowspan="21"></td><td colspan="1" rowspan="1">CTX-M-106</td></tr><tr><td colspan="1" rowspan="1">CTX-M-121</td></tr><tr><td colspan="1" rowspan="1">CTX-M-122</td></tr><tr><td colspan="1" rowspan="1">CTX-M-126</td></tr><tr><td colspan="1" rowspan="1">CTX-M-129</td></tr><tr><td colspan="1" rowspan="1">CTX-M-130</td></tr><tr><td colspan="1" rowspan="1">CTX-M-132</td></tr><tr><td colspan="1" rowspan="1">CTX-M-134</td></tr><tr><td colspan="1" rowspan="1">CTX-M-137</td></tr><tr><td colspan="1" rowspan="1">CTX-M-148</td></tr><tr><td colspan="1" rowspan="1">CTX-M-161</td></tr><tr><td colspan="1" rowspan="1">CTX-M-168</td></tr><tr><td colspan="1" rowspan="1">CTX-M-173</td></tr><tr><td colspan="1" rowspan="1">CTX-M-174</td></tr><tr><td colspan="1" rowspan="1">CTX-M-176</td></tr><tr><td colspan="1" rowspan="1">CTX-M-177</td></tr><tr><td colspan="1" rowspan="1">CTX-M-191</td></tr><tr><td colspan="1" rowspan="1">CTX-M-195</td></tr><tr><td colspan="1" rowspan="1">CTX-M-196</td></tr><tr><td colspan="1" rowspan="1">CTX-M-198</td></tr><tr><td colspan="1" rowspan="1">CTX-M-199</td></tr><tr><td colspan="1" rowspan="19">KlebsiellapneumoniaeGroup</td><td colspan="1" rowspan="19">Klebsiellapneumoniae</td><td colspan="1" rowspan="1">CTX-M-9</td></tr><tr><td colspan="1" rowspan="1">CTX-M-13</td></tr><tr><td colspan="1" rowspan="1">CTX-M-14</td></tr><tr><td colspan="1" rowspan="1">CTX-M-17</td></tr><tr><td colspan="1" rowspan="1">CTX-M-18</td></tr><tr><td colspan="1" rowspan="1">CTX-M-19</td></tr><tr><td colspan="1" rowspan="1">CTX-M-24</td></tr><tr><td colspan="1" rowspan="1">CTX-M-38</td></tr><tr><td colspan="1" rowspan="1">CTX-M-46</td></tr><tr><td colspan="1" rowspan="1">CTX-M-48</td></tr><tr><td colspan="1" rowspan="1">CTX-M-49</td></tr><tr><td colspan="1" rowspan="1">CTX-M-50</td></tr><tr><td colspan="1" rowspan="1">CTX-M-65</td></tr><tr><td colspan="1" rowspan="1">CTX-M-81</td></tr><tr><td colspan="1" rowspan="1">CTX-M-99</td></tr><tr><td colspan="1" rowspan="1">CTX-M-104</td></tr><tr><td colspan="1" rowspan="1">CTX-M-147</td></tr><tr><td colspan="1" rowspan="1">CTX-M-159</td></tr><tr><td colspan="1" rowspan="1">CTX-M-201</td></tr><tr><td colspan="1" rowspan="2">Proteusmirabilis/Proteus</td><td colspan="1" rowspan="2">Proteusmirabilis</td><td colspan="1" rowspan="1">CTX-M-13</td></tr><tr><td colspan="1" rowspan="1">CTX-M-14</td></tr></table>

<table><tr><td rowspan=1 colspan=1>Target</td><td rowspan=1 colspan=1>AssociatedOrganism</td><td rowspan=1 colspan=1>VariantDetected</td></tr><tr><td rowspan=3 colspan=1></td><td rowspan=3 colspan=1></td><td rowspan=1 colspan=1>CTX-M-24</td></tr><tr><td rowspan=1 colspan=1>CTX-M-65</td></tr><tr><td rowspan=1 colspan=1>CTX-M-90</td></tr><tr><td rowspan=10 colspan=1>Salmonella</td><td rowspan=10 colspan=1>Salmonellaenterica</td><td rowspan=1 colspan=1>CTX-M-9</td></tr><tr><td rowspan=1 colspan=1>CTX-M-14</td></tr><tr><td rowspan=1 colspan=1>CTX-M-25</td></tr><tr><td rowspan=1 colspan=1>CTX-M-27</td></tr><tr><td rowspan=1 colspan=1>CTX-M-65</td></tr><tr><td rowspan=1 colspan=1>CTX-M-83</td></tr><tr><td rowspan=1 colspan=1>CTX-M-84</td></tr><tr><td rowspan=1 colspan=1>CTX-M-85</td></tr><tr><td rowspan=1 colspan=1>CTX-M-86</td></tr><tr><td rowspan=1 colspan=1>CTX-M-143</td></tr><tr><td rowspan=1 colspan=1>Serratia</td><td rowspan=1 colspan=1>Serratialiquefaciens</td><td rowspan=1 colspan=1>CTX-M-14</td></tr></table>

Table 33: Predicted (in silico) reactivity for CTX-M-25 group   

<table><tr><td rowspan=1 colspan=1>Target</td><td rowspan=1 colspan=1>AssociatedOrganism</td><td rowspan=1 colspan=1>VariantDetected</td></tr><tr><td rowspan=4 colspan=1>Escherichia coli</td><td rowspan=4 colspan=1>Escherichiacoli</td><td rowspan=1 colspan=1>CTX-M-25</td></tr><tr><td rowspan=1 colspan=1>CTX-M-39</td></tr><tr><td rowspan=1 colspan=1>CTX-M-94</td></tr><tr><td rowspan=1 colspan=1>CTX-M-100</td></tr><tr><td rowspan=1 colspan=1>KlebsiellapneumoniaeGroup</td><td rowspan=1 colspan=1>KlebsiellapneumoniaeA*</td><td rowspan=1 colspan=1>CTX-M-26</td></tr></table>

<table><tr><td rowspan=1 colspan=1>Target</td><td rowspan=1 colspan=1>AssociatedOrganism</td><td rowspan=1 colspan=1>VariantDetected</td></tr><tr><td rowspan=4 colspan=1>Proteusmirabilis/Proteus</td><td rowspan=4 colspan=1>Proteusmirabilis</td><td rowspan=1 colspan=1>CTX-M-41</td></tr><tr><td rowspan=1 colspan=1>CTX-M-89</td></tr><tr><td rowspan=1 colspan=1>CTX-M-91</td></tr><tr><td rowspan=1 colspan=1>CTX-M-160</td></tr><tr><td rowspan=1 colspan=1>Salmonella</td><td rowspan=1 colspan=1>Salmonellaenterica</td><td rowspan=1 colspan=1>CTX-M-25</td></tr></table>

Table 34: Predicted (in silico) reactivity for IMP   

<table><tr><td rowspan=1 colspan=1>Target</td><td rowspan=1 colspan=1>AssociatedOrganism</td><td rowspan=1 colspan=1>VariantDetected</td></tr><tr><td rowspan=11 colspan=1>Acinetobacterbaumannii</td><td rowspan=11 colspan=1>Acinetobacterbaumannii</td><td rowspan=1 colspan=1>IMP-1</td></tr><tr><td rowspan=1 colspan=1>IMP-2</td></tr><tr><td rowspan=1 colspan=1>IMP-4</td></tr><tr><td rowspan=1 colspan=1>IMP-5</td></tr><tr><td rowspan=1 colspan=1>IMP-8</td></tr><tr><td rowspan=1 colspan=1>IMP-10</td></tr><tr><td rowspan=1 colspan=1>IMP-11</td></tr><tr><td rowspan=1 colspan=1>IMP-14</td></tr><tr><td rowspan=1 colspan=1>IMP-19</td></tr><tr><td rowspan=1 colspan=1>IMP-55</td></tr><tr><td rowspan=1 colspan=1>IMP-61</td></tr><tr><td rowspan=4 colspan=1>Citrobacter</td><td rowspan=4 colspan=1>Citrobacterfreundii</td><td rowspan=1 colspan=1>IMP-1</td></tr><tr><td rowspan=1 colspan=1>IMP-4</td></tr><tr><td rowspan=1 colspan=1>IMP-8</td></tr><tr><td rowspan=1 colspan=1>IMP-23</td></tr></table>

<table><tr><td colspan="1" rowspan="1">Target</td><td colspan="1" rowspan="1">AssociatedOrganism</td><td colspan="1" rowspan="1">VariantDetected</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">IMP-38</td></tr><tr><td colspan="1" rowspan="1">Enterobacter non-cloacae Complex</td><td colspan="1" rowspan="1">Enterobacteraerogenes</td><td colspan="1" rowspan="1">IMP-4</td></tr><tr><td colspan="1" rowspan="7">Enterobactercloacae Complex</td><td colspan="1" rowspan="7">Enterobactercloacae</td><td colspan="1" rowspan="1">IMP-1</td></tr><tr><td colspan="1" rowspan="1">IMP-4</td></tr><tr><td colspan="1" rowspan="1">IMP-8</td></tr><tr><td colspan="1" rowspan="1">IMP-11</td></tr><tr><td colspan="1" rowspan="1">IMP-26</td></tr><tr><td colspan="1" rowspan="1">IMP-34</td></tr><tr><td colspan="1" rowspan="1">IMP-60</td></tr><tr><td colspan="1" rowspan="2">Enterobactercloacae Complex</td><td colspan="1" rowspan="2">Enterobacterhormaechei</td><td colspan="1" rowspan="1">IMP-13</td></tr><tr><td colspan="1" rowspan="1">IMP-14</td></tr><tr><td colspan="1" rowspan="3">Escherichia coli</td><td colspan="1" rowspan="3">Escherichia coli</td><td colspan="1" rowspan="1">IMP-1</td></tr><tr><td colspan="1" rowspan="1">IMP-4</td></tr><tr><td colspan="1" rowspan="1">IMP-6</td></tr><tr><td colspan="1" rowspan="6"></td><td colspan="1" rowspan="6"></td><td colspan="1" rowspan="1">IMP-8</td></tr><tr><td colspan="1" rowspan="1">IMP-14</td></tr><tr><td colspan="1" rowspan="1">IMP-30</td></tr><tr><td colspan="1" rowspan="1">IMP-52</td></tr><tr><td colspan="1" rowspan="1">IMP-59</td></tr><tr><td colspan="1" rowspan="1">IMP-66</td></tr><tr><td colspan="1" rowspan="5">Klebsiella oxytoca</td><td colspan="1" rowspan="5">Klebsiella oxytoca</td><td colspan="1" rowspan="1">IMP-1</td></tr><tr><td colspan="1" rowspan="1">IMP-4</td></tr><tr><td colspan="1" rowspan="1">IMP-8</td></tr><tr><td colspan="1" rowspan="1">IMP-28</td></tr><tr><td colspan="1" rowspan="1">IMP-34</td></tr><tr><td colspan="1" rowspan="10">Klebsiellapneumoniae Group</td><td colspan="1" rowspan="10">Klebsiellapneumoniae</td><td colspan="1" rowspan="1">IMP-1</td></tr><tr><td colspan="1" rowspan="1">IMP-4*</td></tr><tr><td colspan="1" rowspan="1">IMP-6</td></tr><tr><td colspan="1" rowspan="1">IMP-8</td></tr><tr><td colspan="1" rowspan="1">IMP-10</td></tr><tr><td colspan="1" rowspan="1">IMP-13</td></tr><tr><td colspan="1" rowspan="1">IMP-19</td></tr><tr><td colspan="1" rowspan="1">IMP-26</td></tr><tr><td colspan="1" rowspan="1">IMP-32</td></tr><tr><td colspan="1" rowspan="1">IMP-38</td></tr><tr><td colspan="1" rowspan="3">Proteusmirabilis/Proteus</td><td colspan="1" rowspan="3">Proteus mirabilis</td><td colspan="1" rowspan="1">IMP-1</td></tr><tr><td colspan="1" rowspan="1">IMP-27</td></tr><tr><td colspan="1" rowspan="1">IMP-64</td></tr><tr><td colspan="1" rowspan="12">Pseudomonasaeruginosa</td><td colspan="1" rowspan="12">Pseudomonasaeruginosa</td><td colspan="1" rowspan="1">IMP-1*</td></tr><tr><td colspan="1" rowspan="1">IMP-2</td></tr><tr><td colspan="1" rowspan="1">IMP-4</td></tr><tr><td colspan="1" rowspan="1">IMP-6</td></tr><tr><td colspan="1" rowspan="1">IMP-7</td></tr><tr><td colspan="1" rowspan="1">IMP-9</td></tr><tr><td colspan="1" rowspan="1">IMP-10</td></tr><tr><td colspan="1" rowspan="1">IMP-11</td></tr><tr><td colspan="1" rowspan="1">IMP-13</td></tr><tr><td colspan="1" rowspan="1">IMP-14*</td></tr><tr><td colspan="1" rowspan="1">IMP-15</td></tr><tr><td colspan="1" rowspan="1">IMP-16</td></tr></table>

<table><tr><td rowspan=1 colspan=1>Target</td><td rowspan=1 colspan=1>AssociatedOrganism</td><td rowspan=1 colspan=1>VariantDetected</td></tr><tr><td rowspan=26 colspan=1></td><td rowspan=26 colspan=1></td><td rowspan=1 colspan=1>IMP-17</td></tr><tr><td rowspan=1 colspan=1>IMP-18</td></tr><tr><td rowspan=1 colspan=1>IMP-19</td></tr><tr><td rowspan=1 colspan=1>IMP-20</td></tr><tr><td rowspan=1 colspan=1>IMP-21</td></tr><tr><td rowspan=1 colspan=1>IMP-22</td></tr><tr><td rowspan=1 colspan=1>IMP-25</td></tr><tr><td rowspan=1 colspan=1>IMP-26</td></tr><tr><td rowspan=1 colspan=1>IMP-29</td></tr><tr><td rowspan=1 colspan=1>IMP-30</td></tr><tr><td rowspan=1 colspan=1>IMP-33</td></tr><tr><td rowspan=1 colspan=1>IMP-34</td></tr><tr><td rowspan=1 colspan=1>IMP-37</td></tr><tr><td rowspan=1 colspan=1>IMP-40</td></tr><tr><td rowspan=1 colspan=1>IMP-41</td></tr><tr><td rowspan=1 colspan=1>IMP-43</td></tr><tr><td rowspan=1 colspan=1>IMP-44</td></tr><tr><td rowspan=1 colspan=1>IMP-45</td></tr><tr><td rowspan=1 colspan=1>IMP-48</td></tr><tr><td rowspan=1 colspan=1>IMP-49</td></tr><tr><td rowspan=1 colspan=1>IMP-51</td></tr><tr><td rowspan=1 colspan=1>IMP-53</td></tr><tr><td rowspan=1 colspan=1>IMP-54</td></tr><tr><td rowspan=1 colspan=1>IMP-56</td></tr><tr><td rowspan=1 colspan=1>IMP-62</td></tr><tr><td rowspan=1 colspan=1>IMP-63</td></tr><tr><td rowspan=1 colspan=1>Salmonella</td><td rowspan=1 colspan=1>Salmonellaenterica</td><td rowspan=1 colspan=1>IMP-4</td></tr><tr><td rowspan=5 colspan=1>Serratiamarcescens/Serratia</td><td rowspan=5 colspan=1>Serratiamarcescens</td><td rowspan=1 colspan=1>IMP-1</td></tr><tr><td rowspan=1 colspan=1>IMP-2</td></tr><tr><td rowspan=1 colspan=1>IMP-6</td></tr><tr><td rowspan=1 colspan=1>IMP-8</td></tr><tr><td rowspan=1 colspan=1>IMP-24</td></tr><tr><td rowspan=1 colspan=1>Stenotrophomonasmaltophilia</td><td rowspan=1 colspan=1>Stenotrophomonasmaltophilia</td><td rowspan=1 colspan=1>IMP-25</td></tr></table>

Table 35: Predicted (in silico) reactivity for KPC   

<table><tr><td rowspan=1 colspan=1>Target</td><td rowspan=1 colspan=1>AssociatedOrganism</td><td rowspan=1 colspan=1>VariantDetected</td></tr><tr><td rowspan=3 colspan=1>Acinetobacterbaumannii</td><td rowspan=3 colspan=1>Acinetobacterbaumannii</td><td rowspan=1 colspan=1>KPC-2</td></tr><tr><td rowspan=1 colspan=1>KPC-3</td></tr><tr><td rowspan=1 colspan=1>KPC-10</td></tr><tr><td rowspan=3 colspan=1>Citrobacter</td><td rowspan=3 colspan=1>Citrobacterfreundii</td><td rowspan=1 colspan=1>KPC-2*</td></tr><tr><td rowspan=1 colspan=1>KPC-3</td></tr><tr><td rowspan=1 colspan=1>KPC-18</td></tr><tr><td rowspan=2 colspan=1>Enterobacter non-cloacae Complex</td><td rowspan=2 colspan=1>Enterobacteraerogenes</td><td rowspan=1 colspan=1>KPC-2</td></tr><tr><td rowspan=1 colspan=1>KPC-3</td></tr><tr><td rowspan=3 colspan=1>Enterobactercloacae Complex</td><td rowspan=3 colspan=1>Enterobactercloacae</td><td rowspan=1 colspan=1>KPC-2*</td></tr><tr><td rowspan=1 colspan=1>KPC-3</td></tr><tr><td rowspan=1 colspan=1>KPC-4</td></tr></table>

<table><tr><td colspan="1" rowspan="1">Target</td><td colspan="1" rowspan="1">AssociatedOrganism</td><td colspan="1" rowspan="1">VariantDetected</td></tr><tr><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="1">KPC-13</td></tr><tr><td colspan="1" rowspan="1">KPC-18</td></tr><tr><td colspan="1" rowspan="8">Escherichia coli</td><td colspan="1" rowspan="8">Escherichia coli</td><td colspan="1" rowspan="1">KPC-2</td></tr><tr><td colspan="1" rowspan="1">KPC-3</td></tr><tr><td colspan="1" rowspan="1">KPC-4</td></tr><tr><td colspan="1" rowspan="1">KPC-5</td></tr><tr><td colspan="1" rowspan="1">KPC-9</td></tr><tr><td colspan="1" rowspan="1">KPC-18</td></tr><tr><td colspan="1" rowspan="1">KPC-21</td></tr><tr><td colspan="1" rowspan="1">KPC-28</td></tr><tr><td colspan="1" rowspan="1">Klebsiella oxytoca</td><td colspan="1" rowspan="1">Klebsiella oxytoca</td><td colspan="1" rowspan="1">KPC-2</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">KPC-3*</td></tr><tr><td colspan="1" rowspan="14">Klebsiellapneumoniae Group</td><td colspan="1" rowspan="14">Klebsiellapneumoniae</td><td colspan="1" rowspan="1">KPC-1</td></tr><tr><td colspan="1" rowspan="1">KPC-2</td></tr><tr><td colspan="1" rowspan="1">KPC-3*</td></tr><tr><td colspan="1" rowspan="1">KPC-4</td></tr><tr><td colspan="1" rowspan="1">KPC-5</td></tr><tr><td colspan="1" rowspan="1">KPC-6</td></tr><tr><td colspan="1" rowspan="1">KPC-7</td></tr><tr><td colspan="1" rowspan="1">KPC-8</td></tr><tr><td colspan="1" rowspan="1">KPC-11</td></tr><tr><td colspan="1" rowspan="1">KPC-12</td></tr><tr><td colspan="1" rowspan="1">KPC-14</td></tr><tr><td colspan="1" rowspan="1">KPC-15</td></tr><tr><td colspan="1" rowspan="1">KPC-16</td></tr><tr><td colspan="1" rowspan="1">KPC-17</td></tr></table>

<table><tr><td rowspan=1 colspan=1>Target</td><td rowspan=1 colspan=1>AssociatedOrganism</td><td rowspan=1 colspan=1>VariantDetected</td></tr><tr><td rowspan=7 colspan=1></td><td rowspan=7 colspan=1></td><td rowspan=1 colspan=1>KPC-19</td></tr><tr><td rowspan=1 colspan=1>KPC-22</td></tr><tr><td rowspan=1 colspan=1>KPC-24</td></tr><tr><td rowspan=1 colspan=1>KPC-25</td></tr><tr><td rowspan=1 colspan=1>KPC-26</td></tr><tr><td rowspan=1 colspan=1>KPC-27</td></tr><tr><td rowspan=1 colspan=1>KPC-30</td></tr><tr><td rowspan=2 colspan=1>Proteusmirabilis/Proteus</td><td rowspan=2 colspan=1>Proteus mirabilis</td><td rowspan=1 colspan=1>KPC-2</td></tr><tr><td rowspan=1 colspan=1>KPC-6*</td></tr><tr><td rowspan=2 colspan=1>Pseudomonasaeruginosa</td><td rowspan=2 colspan=1>Pseudomonasaeruginosa</td><td rowspan=1 colspan=1>KPC-2</td></tr><tr><td rowspan=1 colspan=1>KPC-5*</td></tr><tr><td rowspan=1 colspan=1>Salmonella</td><td rowspan=1 colspan=1>Salmonellaenterica</td><td rowspan=1 colspan=1>KPC-2</td></tr><tr><td rowspan=2 colspan=1>Serratiamarcescens/Serratia</td><td rowspan=2 colspan=1>Serratiamarcescens</td><td rowspan=1 colspan=1>KPC-2</td></tr><tr><td rowspan=1 colspan=1>KPC-3</td></tr></table>

Table 36: Predicted (in silico) reactivity for VIM   

<table><tr><td rowspan=1 colspan=1>Target</td><td rowspan=1 colspan=1>AssociatedOrganism</td><td rowspan=1 colspan=1>VariantDetected</td></tr><tr><td rowspan=4 colspan=1>Acinetobacterbaumannii</td><td rowspan=4 colspan=1>Acinetobacterbaumannii</td><td rowspan=1 colspan=1>VIM-1</td></tr><tr><td rowspan=1 colspan=1>VIM-2</td></tr><tr><td rowspan=1 colspan=1>VIM-6</td></tr><tr><td rowspan=1 colspan=1>VIM-11</td></tr><tr><td rowspan=4 colspan=1>Citrobacter</td><td rowspan=4 colspan=1>Citrobacterfreundii</td><td rowspan=1 colspan=1>VIM-1</td></tr><tr><td rowspan=1 colspan=1>VIM-2</td></tr><tr><td rowspan=1 colspan=1>VIM-4</td></tr><tr><td rowspan=1 colspan=1>VIM-23</td></tr><tr><td rowspan=1 colspan=1>Enterobacter non-cloacae Complex</td><td rowspan=1 colspan=1>Enterobacteraerogenes</td><td rowspan=1 colspan=1>VIM-1</td></tr><tr><td rowspan=10 colspan=1>Enterobactercloacae Complex</td><td rowspan=6 colspan=1>Enterobactercloacae</td><td rowspan=1 colspan=1>VIM-1</td></tr><tr><td rowspan=1 colspan=1>VIM-2</td></tr><tr><td rowspan=1 colspan=1>VIM-4</td></tr><tr><td rowspan=1 colspan=1>VIM-23</td></tr><tr><td rowspan=1 colspan=1>VIM-31</td></tr><tr><td rowspan=1 colspan=1>VIM-40</td></tr><tr><td rowspan=3 colspan=1>Enterobacterhormaechei</td><td rowspan=1 colspan=1>VIM-1</td></tr><tr><td rowspan=1 colspan=1>VIM-4</td></tr><tr><td rowspan=1 colspan=1>VIM-23</td></tr><tr><td rowspan=1 colspan=1>Enterobacterxiangfangensis</td><td rowspan=1 colspan=1>VIM-1</td></tr><tr><td rowspan=4 colspan=1>Escherichia coli</td><td rowspan=4 colspan=1>Escherichia coliA</td><td rowspan=1 colspan=1>VIM-1</td></tr><tr><td rowspan=1 colspan=1>VIM-2</td></tr><tr><td rowspan=1 colspan=1>VIM-19</td></tr><tr><td rowspan=1 colspan=1>VIM-29</td></tr><tr><td rowspan=5 colspan=1>Klebsiella oxytoca</td><td rowspan=5 colspan=1>Klebsiella oxytoca</td><td rowspan=1 colspan=1>VIM-1</td></tr><tr><td rowspan=1 colspan=1>VIM-2</td></tr><tr><td rowspan=1 colspan=1>VIM-4</td></tr><tr><td rowspan=1 colspan=1>VIM-32</td></tr><tr><td rowspan=1 colspan=1>VIM-35</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>VIM-1*</td></tr></table>

<table><tr><td colspan="1" rowspan="1">Target</td><td colspan="1" rowspan="1">AssociatedOrganism</td><td colspan="1" rowspan="1">VariantDetected</td></tr><tr><td colspan="1" rowspan="13">Klebsiellapneumoniae Group</td><td colspan="1" rowspan="13">Klebsiellapneumoniae</td><td colspan="1" rowspan="1">VIM-2</td></tr><tr><td colspan="1" rowspan="1">VIM-4</td></tr><tr><td colspan="1" rowspan="1">VIM-12</td></tr><tr><td colspan="1" rowspan="1">VIM-19</td></tr><tr><td colspan="1" rowspan="1">VIM-24</td></tr><tr><td colspan="1" rowspan="1">VIM-26</td></tr><tr><td colspan="1" rowspan="1">VIM-27*</td></tr><tr><td colspan="1" rowspan="1">VIM-33</td></tr><tr><td colspan="1" rowspan="1">VIM-34</td></tr><tr><td colspan="1" rowspan="1">VIM-39</td></tr><tr><td colspan="1" rowspan="1">VIM-42</td></tr><tr><td colspan="1" rowspan="1">VIM-51</td></tr><tr><td colspan="1" rowspan="1">VIM-52</td></tr><tr><td colspan="1" rowspan="1">Morganellamorganii</td><td colspan="1" rowspan="1">Morganellamorganii</td><td colspan="1" rowspan="1">VIM-4</td></tr><tr><td colspan="1" rowspan="1">Proteusmirabilis/Proteus</td><td colspan="1" rowspan="1">Proteus mirabilis</td><td colspan="1" rowspan="1">VIM-1</td></tr><tr><td colspan="1" rowspan="14">Pseudomonasaeruginosa</td><td colspan="1" rowspan="14">Pseudomonasaeruginosa</td><td colspan="1" rowspan="1">VIM-1</td></tr><tr><td colspan="1" rowspan="1">VIM-2*</td></tr><tr><td colspan="1" rowspan="1">VIM-3</td></tr><tr><td colspan="1" rowspan="1">VIM-4*</td></tr><tr><td colspan="1" rowspan="1">VIM-5</td></tr><tr><td colspan="1" rowspan="1">VIM-6</td></tr><tr><td colspan="1" rowspan="1">VIM-8</td></tr><tr><td colspan="1" rowspan="1">VIM-9</td></tr><tr><td colspan="1" rowspan="1">VIM-10</td></tr><tr><td colspan="1" rowspan="1">VIM-11</td></tr><tr><td colspan="1" rowspan="1">VIM-14</td></tr><tr><td colspan="1" rowspan="1">VIM-15</td></tr><tr><td colspan="1" rowspan="1">VIM-16</td></tr><tr><td colspan="1" rowspan="1">VIM-17</td></tr><tr><td colspan="1" rowspan="10">−</td><td colspan="1" rowspan="10"></td><td colspan="1" rowspan="1">VIM-18</td></tr><tr><td colspan="1" rowspan="1">VIM-20</td></tr><tr><td colspan="1" rowspan="1">VIM-28</td></tr><tr><td colspan="1" rowspan="1">VIM-30</td></tr><tr><td colspan="1" rowspan="1">VIM-36</td></tr><tr><td colspan="1" rowspan="1">VIM-37</td></tr><tr><td colspan="1" rowspan="1">VIM-41</td></tr><tr><td colspan="1" rowspan="1">VIM-43</td></tr><tr><td colspan="1" rowspan="1">VIM-44</td></tr><tr><td colspan="1" rowspan="1">VIM-45</td></tr></table>

A. Unspecified VIM variant detected in Analytical Reactivity (Inclusivity) study.

<table><tr><td rowspan=1 colspan=1>Target</td><td rowspan=1 colspan=1>AssociatedOrganism</td><td rowspan=1 colspan=1>VariantDetected</td></tr><tr><td rowspan=3 colspan=1></td><td rowspan=3 colspan=1></td><td rowspan=1 colspan=1>VIM-46</td></tr><tr><td rowspan=1 colspan=1>VIM-48</td></tr><tr><td rowspan=1 colspan=1>VIM-50</td></tr><tr><td rowspan=1 colspan=1>Salmonella</td><td rowspan=1 colspan=1>Salmonellaenterica</td><td rowspan=1 colspan=1>VIM-1</td></tr><tr><td rowspan=3 colspan=1>Serratiamarcescens/Serratia</td><td rowspan=3 colspan=1>Serratiamarcescens</td><td rowspan=1 colspan=1>VIM-2</td></tr><tr><td rowspan=1 colspan=1>VIM-4</td></tr><tr><td rowspan=1 colspan=1>VIM-54</td></tr><tr><td rowspan=1 colspan=1>Stenotrophomonasmaltophilia</td><td rowspan=1 colspan=1>Stenotrophomonasmaltophilia</td><td rowspan=1 colspan=1>VIM-2</td></tr></table>

Table 37: Predicted (in silico) reactivity for OXA-23   

<table><tr><td rowspan=1 colspan=1>Target</td><td rowspan=1 colspan=1>AssociatedOrganism</td><td rowspan=1 colspan=1>Variant Detected</td></tr><tr><td rowspan=18 colspan=1>Acinetobacterbaumannii</td><td rowspan=18 colspan=1>Acinetobacterbaumannii</td><td rowspan=1 colspan=1>OXA-23*</td></tr><tr><td rowspan=1 colspan=1>OXA-49</td></tr><tr><td rowspan=1 colspan=1>OXA-23/OXA-104</td></tr><tr><td rowspan=1 colspan=1>OXA-23/OXA-64</td></tr><tr><td rowspan=1 colspan=1>OXA-23/OXA-66</td></tr><tr><td rowspan=1 colspan=1>OXA-23/OXA-69</td></tr><tr><td rowspan=1 colspan=1>OXA-27</td></tr><tr><td rowspan=1 colspan=1>OXA-65/OXA-239</td></tr><tr><td rowspan=1 colspan=1>OXA-68</td></tr><tr><td rowspan=1 colspan=1>OXA-146</td></tr><tr><td rowspan=1 colspan=1>OXA-165</td></tr><tr><td rowspan=1 colspan=1>OXA-166</td></tr><tr><td rowspan=1 colspan=1>OXA-167</td></tr><tr><td rowspan=1 colspan=1>OXA-168</td></tr><tr><td rowspan=1 colspan=1>OXA-169</td></tr><tr><td rowspan=1 colspan=1>OXA-170</td></tr><tr><td rowspan=1 colspan=1>OXA-171</td></tr><tr><td rowspan=1 colspan=1>OXA-183</td></tr></table>

<table><tr><td rowspan=1 colspan=1>Target</td><td rowspan=1 colspan=1>AssociatedOrganism</td><td rowspan=1 colspan=1>Variant Detected</td></tr><tr><td rowspan=12 colspan=1></td><td rowspan=12 colspan=1></td><td rowspan=1 colspan=1>OXA-225</td></tr><tr><td rowspan=1 colspan=1>OXA-366</td></tr><tr><td rowspan=1 colspan=1>OXA-398</td></tr><tr><td rowspan=1 colspan=1>OXA-422</td></tr><tr><td rowspan=1 colspan=1>OXA-423</td></tr><tr><td rowspan=1 colspan=1>OXA-435</td></tr><tr><td rowspan=1 colspan=1>OXA-440</td></tr><tr><td rowspan=1 colspan=1>OXA-469</td></tr><tr><td rowspan=1 colspan=1>OXA-481</td></tr><tr><td rowspan=1 colspan=1>OXA-482</td></tr><tr><td rowspan=1 colspan=1>OXA-483</td></tr><tr><td rowspan=1 colspan=1>OXA-565</td></tr><tr><td rowspan=1 colspan=1>Escherichiacoli</td><td rowspan=1 colspan=1>Escherichiacoli</td><td rowspan=1 colspan=1>OXA-23</td></tr><tr><td rowspan=1 colspan=1>KlebsiellapneumoniaeGroup</td><td rowspan=1 colspan=1>Klebsiellapneumoniae</td><td rowspan=1 colspan=1>OXA-73</td></tr><tr><td rowspan=1 colspan=1>Proteusmirabilis</td><td rowspan=1 colspan=1>Proteusmirabilis</td><td rowspan=1 colspan=1>OXA-23</td></tr></table>

Table 38: Predicted (in silico) reactivity for OXA-48   

<table><tr><td rowspan=1 colspan=1>Target</td><td rowspan=1 colspan=1>AssociatedOrganism</td><td rowspan=1 colspan=1>VariantDetected</td></tr><tr><td rowspan=1 colspan=1>Acinetobacterbaumannii</td><td rowspan=1 colspan=1>Acinetobacterbaumannii</td><td rowspan=1 colspan=1>OXA-48</td></tr><tr><td rowspan=2 colspan=1>Citrobacter</td><td rowspan=2 colspan=1>Citrobacterfreundii</td><td rowspan=1 colspan=1>OXA-48</td></tr><tr><td rowspan=1 colspan=1>OXA-181</td></tr><tr><td rowspan=1 colspan=1>Enterobacter non-cloacae Complex</td><td rowspan=1 colspan=1>Enterobacteraerogenes</td><td rowspan=1 colspan=1>OXA-244</td></tr><tr><td rowspan=4 colspan=1>Enterobactercloacae Complex</td><td rowspan=3 colspan=1>Enterobactercloacae</td><td rowspan=1 colspan=1>OXA-48</td></tr><tr><td rowspan=1 colspan=1>OXA-163</td></tr><tr><td rowspan=1 colspan=1>OXA-181</td></tr><tr><td rowspan=1 colspan=1>Enterobacterhormaechei</td><td rowspan=1 colspan=1>OXA-370</td></tr></table>

<table><tr><td colspan="1" rowspan="1">Target</td><td colspan="1" rowspan="1">AssociatedOrganism</td><td colspan="1" rowspan="1">VariantDetected</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Enterobacterludwigii</td><td colspan="1" rowspan="1">OXA-48</td></tr><tr><td colspan="1" rowspan="9">Escherichia coli</td><td colspan="1" rowspan="9">EscherichiacoliA</td><td colspan="1" rowspan="1">OXA-48</td></tr><tr><td colspan="1" rowspan="1">OXA-163</td></tr><tr><td colspan="1" rowspan="1">OXA-181</td></tr><tr><td colspan="1" rowspan="1">OXA-204</td></tr><tr><td colspan="1" rowspan="1">OXA-232</td></tr><tr><td colspan="1" rowspan="1">OXA-244</td></tr><tr><td colspan="1" rowspan="1">OXA-438</td></tr><tr><td colspan="1" rowspan="1">OXA-439</td></tr><tr><td colspan="1" rowspan="1">OXA-566</td></tr><tr><td colspan="1" rowspan="11">Klebsiellapneumoniae Group</td><td colspan="1" rowspan="11">Klebsiellapneumoniae</td><td colspan="1" rowspan="1">OXA-1/OXA-48</td></tr><tr><td colspan="1" rowspan="1">OXA-48*</td></tr><tr><td colspan="1" rowspan="1">OXA-10</td></tr><tr><td colspan="1" rowspan="1">OXA-162</td></tr><tr><td colspan="1" rowspan="1">OXA-181*</td></tr><tr><td colspan="1" rowspan="1">OXA-204</td></tr><tr><td colspan="1" rowspan="1">OXA-232*</td></tr><tr><td colspan="1" rowspan="1">OXA-244</td></tr><tr><td colspan="1" rowspan="1">OXA-245</td></tr><tr><td colspan="1" rowspan="1">OXA-247</td></tr><tr><td colspan="1" rowspan="1">OXA-484</td></tr></table>

A. Unspecified OXA variant detected in the Analytical Reactivity (Inclusivity) study.

<table><tr><td rowspan=1 colspan=1>Target</td><td rowspan=1 colspan=1>AssociatedOrganism</td><td rowspan=1 colspan=1>VariantDetected</td></tr><tr><td rowspan=4 colspan=1></td><td rowspan=3 colspan=1></td><td rowspan=1 colspan=1>OXA-505</td></tr><tr><td rowspan=1 colspan=1>OXA-517</td></tr><tr><td rowspan=1 colspan=1>OXA-519</td></tr><tr><td rowspan=1 colspan=1>Klebsiellavariicola</td><td rowspan=1 colspan=1>OXA-181</td></tr><tr><td rowspan=1 colspan=1>Morganellamorganii</td><td rowspan=1 colspan=1>Morganellamorganii</td><td rowspan=1 colspan=1>OXA-181</td></tr><tr><td rowspan=2 colspan=1>Proteusmirabilis/Proteus</td><td rowspan=2 colspan=1>Proteusmirabilis</td><td rowspan=1 colspan=1>OXA-48</td></tr><tr><td rowspan=1 colspan=1>OXA-244</td></tr><tr><td rowspan=2 colspan=1>Serratiamarcescens/Serratia</td><td rowspan=2 colspan=1>Serratiamarcescens</td><td rowspan=1 colspan=1>OXA-48</td></tr><tr><td rowspan=1 colspan=1>OXA-405</td></tr></table>

Table 39: Predicted (in silico) reactivity for NDM   

<table><tr><td rowspan=1 colspan=1>Target</td><td rowspan=1 colspan=1>AssociatedOrganism</td><td rowspan=1 colspan=1>VariantDetected</td></tr><tr><td rowspan=2 colspan=1>Acinetobacterbaumannii</td><td rowspan=2 colspan=1>Acinetobacterbaumannii</td><td rowspan=1 colspan=1>NDM-1*</td></tr><tr><td rowspan=1 colspan=1>NDM-2</td></tr><tr><td rowspan=1 colspan=1>Citrobacter</td><td rowspan=1 colspan=1>Citrobacterbraakii</td><td rowspan=1 colspan=1>NDM-4</td></tr><tr><td rowspan=4 colspan=1>Citrobacter</td><td rowspan=4 colspan=1>Citrobacterfreundii</td><td rowspan=1 colspan=1>NDM-1A</td></tr><tr><td rowspan=1 colspan=1>NDM-4</td></tr><tr><td rowspan=1 colspan=1>NDM-6</td></tr><tr><td rowspan=1 colspan=1>NDM-7</td></tr><tr><td rowspan=4 colspan=1>Enterobacter non-cloacae Complex</td><td rowspan=4 colspan=1>Enterobacteraerogenes</td><td rowspan=1 colspan=1>NDM-1</td></tr><tr><td rowspan=1 colspan=1>NDM-4</td></tr><tr><td rowspan=1 colspan=1>NDM-5</td></tr><tr><td rowspan=1 colspan=1>NDM-7</td></tr><tr><td rowspan=5 colspan=1>Enterobactercloacae Complex</td><td rowspan=3 colspan=1>EnterobactercloacaeB</td><td rowspan=1 colspan=1>NDM-1</td></tr><tr><td rowspan=1 colspan=1>NDM-4</td></tr><tr><td rowspan=1 colspan=1>NDM-7</td></tr><tr><td rowspan=1 colspan=1>Enterobacterhormaechei</td><td rowspan=1 colspan=1>NDM-1</td></tr><tr><td rowspan=1 colspan=1>Enterobacterludwigii</td><td rowspan=1 colspan=1>NDM-1</td></tr><tr><td rowspan=9 colspan=1>Escherichia coli</td><td rowspan=9 colspan=1>Escherichiacoli</td><td rowspan=1 colspan=1>NDM-1</td></tr><tr><td rowspan=1 colspan=1>NDM-3</td></tr><tr><td rowspan=1 colspan=1>NDM-4</td></tr><tr><td rowspan=1 colspan=1>NDM-5*</td></tr><tr><td rowspan=1 colspan=1>NDM-6*</td></tr><tr><td rowspan=1 colspan=1>NDM-7*</td></tr><tr><td rowspan=1 colspan=1>NDM-8</td></tr><tr><td rowspan=1 colspan=1>NDM-11</td></tr><tr><td rowspan=1 colspan=1>NDM-12</td></tr></table>

<table><tr><td rowspan=1 colspan=1>Target</td><td rowspan=1 colspan=1>AssociatedOrganism</td><td rowspan=1 colspan=1>VariantDetected</td></tr><tr><td rowspan=6 colspan=1></td><td rowspan=6 colspan=1></td><td rowspan=1 colspan=1>NDM-13</td></tr><tr><td rowspan=1 colspan=1>NDM-15</td></tr><tr><td rowspan=1 colspan=1>NDM-16</td></tr><tr><td rowspan=1 colspan=1>NDM-17</td></tr><tr><td rowspan=1 colspan=1>NDM-18</td></tr><tr><td rowspan=1 colspan=1>NDM-19</td></tr><tr><td rowspan=2 colspan=1>Klebsiella oxytoca</td><td rowspan=2 colspan=1>Klebsiellaoxytoca</td><td rowspan=1 colspan=1>NDM-1</td></tr><tr><td rowspan=1 colspan=1>NDM-4</td></tr><tr><td rowspan=7 colspan=1>KlebsiellapneumoniaeGroup</td><td rowspan=7 colspan=1>Klebsiellapneumoniae</td><td rowspan=1 colspan=1>NDM-1*</td></tr><tr><td rowspan=1 colspan=1>NDM-4</td></tr><tr><td rowspan=1 colspan=1>NDM-5</td></tr><tr><td rowspan=1 colspan=1>NDM-6</td></tr><tr><td rowspan=1 colspan=1>NDM-7</td></tr><tr><td rowspan=1 colspan=1>NDM-10</td></tr><tr><td rowspan=1 colspan=1>NDM-16</td></tr><tr><td rowspan=1 colspan=1>Morganellamorganii</td><td rowspan=1 colspan=1>Morganellamorganii</td><td rowspan=1 colspan=1>NDM-1C</td></tr><tr><td rowspan=1 colspan=1>Proteusmirabilis/Proteus</td><td rowspan=1 colspan=1>Proteusmirabilis</td><td rowspan=1 colspan=1>NDM-1</td></tr><tr><td rowspan=2 colspan=1>Pseudomonasaeruginosa</td><td rowspan=2 colspan=1>Pseudomonasaeruginosa</td><td rowspan=1 colspan=1>NDM-1</td></tr><tr><td rowspan=1 colspan=1>NDM-5</td></tr><tr><td rowspan=2 colspan=1>Salmonella</td><td rowspan=2 colspan=1>Salmonellaenterica</td><td rowspan=1 colspan=1>NDM-1</td></tr><tr><td rowspan=1 colspan=1>NDM-5</td></tr><tr><td rowspan=1 colspan=1>Serratiamarcescens/Serratia</td><td rowspan=1 colspan=1>Serratiamarcescens</td><td rowspan=1 colspan=1>NDM-1</td></tr><tr><td rowspan=1 colspan=1>Stenotrophomonasmaltophilia</td><td rowspan=1 colspan=1>Stenotrophomonas maltophilia</td><td rowspan=1 colspan=1>NDM-1</td></tr></table>

A. Detected in a Citrobacter species in the Analytical Reactivity (Inclusivity) study.   
B. Unspecified NDM variant detected in the Analytical Reactivity (Inclusivity) study.   
C. NDM-1 was detected in Morganella morganii in the Analytical Reactivity (Inclusivity) study but no sequences were available for in silico analysis.

Table 40: Predicted (in silico) Reactivity (Inclusivity) Results for Variants Not Detected   

<table><tr><td rowspan=1 colspan=1>Resistance Marker</td><td rowspan=1 colspan=1>VariantNotDetected</td><td rowspan=1 colspan=1>AssociatedOrganism</td><td rowspan=1 colspan=1>No. ofSequences</td></tr><tr><td rowspan=3 colspan=1>CTX-M-1</td><td rowspan=1 colspan=1>CTX-M-80</td><td rowspan=3 colspan=1>Klebsiellapneumoniae</td><td rowspan=3 colspan=1>3</td></tr><tr><td rowspan=1 colspan=1>CTX-M-15</td></tr><tr><td rowspan=1 colspan=1>NotSpecified</td></tr><tr><td rowspan=3 colspan=1>IMP</td><td rowspan=1 colspan=1>IMP-31</td><td rowspan=3 colspan=1>Pseudomonas aeruginosa</td><td rowspan=1 colspan=1>2</td></tr><tr><td rowspan=1 colspan=1>IMP-35</td><td rowspan=1 colspan=1>2</td></tr><tr><td rowspan=1 colspan=1>IMP-7</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=10 colspan=1>NDM</td><td rowspan=3 colspan=1>NDM-1</td><td rowspan=1 colspan=1>Escherichiacoli</td><td rowspan=1 colspan=1>6</td></tr><tr><td rowspan=1 colspan=1>Klebsiellavariicola</td><td rowspan=1 colspan=1>3</td></tr><tr><td rowspan=1 colspan=1>Salmonellaenterica</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>NDM-3</td><td rowspan=1 colspan=1>Acinetobacter baumannii</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>NDM-4</td><td rowspan=1 colspan=1>Escherichiacoli</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=3 colspan=1>NDM-9</td><td rowspan=1 colspan=1>Cronobactersakazakii</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>Escherichiacoli</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>Klebsiellapneumoniae</td><td rowspan=1 colspan=1>2</td></tr><tr><td rowspan=2 colspan=1>Notspecified</td><td rowspan=1 colspan=1>Escherichiacoli</td><td rowspan=1 colspan=1>2</td></tr><tr><td rowspan=1 colspan=1>Klebsiellapneumoniae</td><td rowspan=1 colspan=1>1</td></tr></table>

<table><tr><td rowspan=1 colspan=1>Resistance Marker</td><td rowspan=1 colspan=1>VariantNotDetected</td><td rowspan=1 colspan=1>AssociatedOrganism</td><td rowspan=1 colspan=1>No. ofSequences</td></tr><tr><td rowspan=2 colspan=1></td><td rowspan=2 colspan=1></td><td rowspan=1 colspan=1>Klebsiella sp</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>Pseudomonas aeruginosa</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=13 colspan=1>VIM</td><td rowspan=2 colspan=1>VIM-1</td><td rowspan=1 colspan=1>Pseudomonas aeruginosa</td><td rowspan=1 colspan=1>3</td></tr><tr><td rowspan=1 colspan=1>Providenciavermicola</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>VIM-2</td><td rowspan=1 colspan=1>Klebsiellapneumoniae</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=2 colspan=1>VIM-5</td><td rowspan=1 colspan=1>Enterobactercloacae</td><td rowspan=1 colspan=1>2</td></tr><tr><td rowspan=1 colspan=1>Klebsiellapneumoniae</td><td rowspan=1 colspan=1>3</td></tr><tr><td rowspan=1 colspan=1>VIM-7</td><td rowspan=1 colspan=1>Pseudomona</td><td rowspan=1 colspan=1>4</td></tr><tr><td rowspan=1 colspan=1>VIM-13</td><td rowspan=1 colspan=1>s aeruginosa</td><td rowspan=1 colspan=1>3</td></tr><tr><td rowspan=2 colspan=1>VIM-25</td><td rowspan=1 colspan=1>Acinetobacterbaumannii</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>Proteusmirabilis</td><td rowspan=1 colspan=1>2</td></tr><tr><td rowspan=1 colspan=1>VIM-38</td><td rowspan=4 colspan=1>Pseudomonasaeruginosa</td><td rowspan=1 colspan=1>2</td></tr><tr><td rowspan=1 colspan=1>VIM-47</td><td rowspan=1 colspan=1>2</td></tr><tr><td rowspan=1 colspan=1>VIM-49</td><td rowspan=1 colspan=1>2</td></tr><tr><td rowspan=1 colspan=1>Notspecified</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>OXA-48</td><td rowspan=1 colspan=1>OXA-232</td><td rowspan=1 colspan=1>Escherichiacoli</td><td rowspan=1 colspan=1>1</td></tr></table>

The analytical specificity of the BCID-GN Panel was assessed by examining the potential for cross-reactivity of on-panel and off-panel analytes. Cross-reactivity of on-panel analytes was evaluated using data from the analytical reactivity study (See Table 20). Bacterial and fungal analytes were diluted and tested in triplicate at bottle positive concentrations. Cross reactivity was not observed for any of the on-panel organism strain with the exception of one strain of $K .$ . pneumoniae for which NDM was unexpectedly detected in all three replicates. The root cause of the detected NDM was not confirmed with an alternate PCR assay; however, it is not likely that $K .$ . pneumoniae cross-reacts with the NDM target of the BCID-GN Panel based on results from 32 additional strains of $K .$ . pneumoniae that did not demonstrate cross-reactivity for NDM.

A total of 128 off-panel bacterial and fungal strains were tested in triplicate at ${ \sim } 1 \mathbf { x } 1 0 ^ { 9 }$ $\mathrm { C F U / m L }$ and ${ \sim } 1 \mathrm { x } 1 0 ^ { 7 } \mathrm { C F U / m L }$ , respectively. If these target concentrations could not be obtained, the organism was diluted 2-fold from the stock for use (indicated with an asterisk in Tables 41-44 below); these concentrations were at least as high as concentrations present at bottle positivity $\mathrm { 1 \times 1 0 ^ { 8 } C F U / m L }$ for bacteria and $1 \mathrm { ~ x ~ } 1 0 ^ { 6 }$ CFU/mL for fungi).

The following off-panel species demonstrated cross reactivity with ePlex BCID-GN Panel targets:

• Acinetobacter anitratus (at a concentration of $> 1 \mathrm { x } 1 0 ^ { 4 } \mathrm { C F U / m L } )$ cross-reacts with the BCID-GN Acinetobacter baumannii assay   
• Enterobacter cowanii (at a concentration of $> 1 \mathrm { x } 1 0 ^ { 8 } \mathrm { C F U / m L } )$ ) cross-reacts with the BCID-GN Enterobacter cloacae complex assay Escherichia hermanii cross-reacts with the BCID-GN Enterobacter (non-cloacae complex) assay (at a concentration of ${ \mathrm { > } } 1 \mathbf { x } 1 0 ^ { 6 }$ CFU/mL) and with the BCID-GN Serratia assay (at a concentration of $> 1 \mathrm { x } 1 0 ^ { 7 } \mathrm { C F U / m L } )$ Fusobacterium periodonticum (at a concentration of $5 \mathrm { x } 1 0 ^ { 8 } \mathrm { C F U / m L }$ ) and Fusobacterium simiae (at a concentration of $2 . 9 { \mathrm { x } } 1 0 ^ { 8 } \mathrm { C F U / m L }$ ) cross-react with the BCID-GN Fusobacterium nucleatum assay Shigella (at a concentration of $1 \mathrm { x } 1 0 ^ { 9 } \mathrm { C F U / m L ) }$ cross-reacts with the BCID-GN Escherichia coli assay.

Off-panel organisms demonstrating cross-reactivity with ePlex BCID-GN Panel targets are bolded in Tables 41-44 below.

Table 41: Cross Reactivity, Off-Panel gram-negative organisms evaluated   

<table><tr><td rowspan=1 colspan=1>Gram Negative Organism</td><td rowspan=1 colspan=1>Strain ID</td></tr><tr><td rowspan=1 colspan=1>Acinetobacter haemolyticus</td><td rowspan=1 colspan=1>ATCC 19002</td></tr><tr><td rowspan=1 colspan=1>Acinetobacter lwoffii</td><td rowspan=1 colspan=1>ATCC 15309</td></tr><tr><td rowspan=1 colspan=1>Acinetobacter junii</td><td rowspan=1 colspan=1>ATCC 17908</td></tr><tr><td rowspan=1 colspan=1>Acinetobacter anitratusA</td><td rowspan=1 colspan=1>ATCC 49139</td></tr><tr><td rowspan=1 colspan=1>Aeromonas hydrophila</td><td rowspan=1 colspan=1>JMI 938982</td></tr><tr><td rowspan=1 colspan=1>Aeromonas salmonicida</td><td rowspan=1 colspan=1>ATCC 33658</td></tr></table>

<table><tr><td colspan="1" rowspan="1">Gram Negative Organism</td><td colspan="1" rowspan="1">Strain ID</td></tr><tr><td colspan="1" rowspan="1">Aeromonas sobria</td><td colspan="1" rowspan="1">ATCC 35993</td></tr><tr><td colspan="1" rowspan="1">Bacteroides distasonis(Parabacteroides)</td><td colspan="1" rowspan="1">ATCC 8503</td></tr><tr><td colspan="1" rowspan="1">Bacteroides merdae</td><td colspan="1" rowspan="1">ATCC 43184</td></tr><tr><td colspan="1" rowspan="1">Bacteroides thetaiotaomicron</td><td colspan="1" rowspan="1">ATCC 29741</td></tr><tr><td colspan="1" rowspan="1">Bacteroides vulgatus*</td><td colspan="1" rowspan="1">ATCC 8482</td></tr><tr><td colspan="1" rowspan="1">Bacteroides caccae</td><td colspan="1" rowspan="1">ATCC 700189</td></tr><tr><td colspan="1" rowspan="1">Bacteroides eggerthii</td><td colspan="1" rowspan="1">ATCC 27754</td></tr><tr><td colspan="1" rowspan="1">Bacteroides ovatus*</td><td colspan="1" rowspan="1">ATCC BAA-1296</td></tr><tr><td colspan="1" rowspan="1">Bacteroides ureolyticus*</td><td colspan="1" rowspan="1">ATCC 33387</td></tr><tr><td colspan="1" rowspan="1">Bordetella pertussis</td><td colspan="1" rowspan="1">ATCC 9797</td></tr><tr><td colspan="1" rowspan="1">Burkholderia cepacia</td><td colspan="1" rowspan="1">ATCC 25416</td></tr><tr><td colspan="1" rowspan="1">Citrobacter amalonaticus</td><td colspan="1" rowspan="1">ATCC BAA-2563</td></tr><tr><td colspan="1" rowspan="1">Citrobacter gillenii</td><td colspan="1" rowspan="1">ATCC 51640</td></tr><tr><td colspan="1" rowspan="1">Citrobacter sedlakii</td><td colspan="1" rowspan="1">ATCC 51493</td></tr><tr><td colspan="1" rowspan="1">Citrobacter farmer</td><td colspan="1" rowspan="1">ATCC 51112</td></tr><tr><td colspan="1" rowspan="1">Citrobacter murliniae</td><td colspan="1" rowspan="1">ATCC 51642</td></tr><tr><td colspan="1" rowspan="1">Edwardsiella tarda</td><td colspan="1" rowspan="1">ATCC 15947</td></tr><tr><td colspan="1" rowspan="1">Enterobacter cowaniiB</td><td colspan="1" rowspan="1">DSM-18146</td></tr><tr><td colspan="1" rowspan="1">Escherichia albertii</td><td colspan="1" rowspan="1">DSM-17582</td></tr><tr><td colspan="1" rowspan="1">Escherichia fergusonii</td><td colspan="1" rowspan="1">ATCC 35469</td></tr><tr><td colspan="1" rowspan="1">Escherichia hermaniic</td><td colspan="1" rowspan="1">ATCC 33650</td></tr><tr><td colspan="1" rowspan="1">Ewingella americana*</td><td colspan="1" rowspan="1">ATCC 33853</td></tr><tr><td colspan="1" rowspan="1">Eikenella corrodens</td><td colspan="1" rowspan="1">ATCC BAA-1152</td></tr><tr><td colspan="1" rowspan="1">Fusobacterium naviforme*</td><td colspan="1" rowspan="1">ATCC 25832</td></tr><tr><td colspan="1" rowspan="1">Fusobacterium gonidiaformans</td><td colspan="1" rowspan="1">ATCC 25563</td></tr><tr><td colspan="1" rowspan="1">Fusobacterium necrogenes*</td><td colspan="1" rowspan="1">ATCC 25556</td></tr><tr><td colspan="1" rowspan="1">Fusobacteriumperiodonticum*D</td><td colspan="1" rowspan="1">ATCC 33693</td></tr><tr><td colspan="1" rowspan="1">Fusobacterium simiae*D</td><td colspan="1" rowspan="1">ATCC 33568</td></tr><tr><td colspan="1" rowspan="1">Fusobacterium varium</td><td colspan="1" rowspan="1">ATCC 27725</td></tr><tr><td colspan="1" rowspan="1">Fusobacterium russii*</td><td colspan="1" rowspan="1">ATCC 25533</td></tr><tr><td colspan="1" rowspan="1">Fusobacterium ulcerans</td><td colspan="1" rowspan="1">ATCC 49186</td></tr><tr><td colspan="1" rowspan="1">Haemophilus haemolyticus</td><td colspan="1" rowspan="1">ATCC 33390</td></tr><tr><td colspan="1" rowspan="1">Haemophilus parahaemolyticus</td><td colspan="1" rowspan="1">ATCC 10014</td></tr><tr><td colspan="1" rowspan="1">Hafnia alvei</td><td colspan="1" rowspan="1">ATCC 51815</td></tr><tr><td colspan="1" rowspan="1">Kingella kingae*</td><td colspan="1" rowspan="1">ATCC 23331</td></tr><tr><td colspan="1" rowspan="1">Kluyvera cochleae</td><td colspan="1" rowspan="1">ATCC 51609</td></tr><tr><td colspan="1" rowspan="1">Legionella pneumoniae</td><td colspan="1" rowspan="1">ATCC 33823</td></tr><tr><td colspan="1" rowspan="1">Leclercia adecarboxylata</td><td colspan="1" rowspan="1">ATCC 700325</td></tr><tr><td colspan="1" rowspan="1">Methylobacteriummesophilicum*</td><td colspan="1" rowspan="1">ATCC 29983</td></tr><tr><td colspan="1" rowspan="1">Neisseria gonorrhoeae</td><td colspan="1" rowspan="1">ATCC 19424</td></tr><tr><td colspan="1" rowspan="1">Neisseria mucosa</td><td colspan="1" rowspan="1">ATCC 19695</td></tr><tr><td colspan="1" rowspan="1">Neisseria sicca</td><td colspan="1" rowspan="1">ATCC 29193</td></tr><tr><td colspan="1" rowspan="1">Neisseria flavecens</td><td colspan="1" rowspan="1">ATCC 13115</td></tr><tr><td colspan="1" rowspan="1">Neisseria lactamica</td><td colspan="1" rowspan="1">ATCC 23970</td></tr><tr><td colspan="1" rowspan="1">Neisseria perflava</td><td colspan="1" rowspan="1">ATCC 14799</td></tr><tr><td colspan="1" rowspan="1">Ochrobactrum anthropi</td><td colspan="1" rowspan="1">ATCC BAA-749</td></tr><tr><td colspan="1" rowspan="1">Pantoea agglomerans</td><td colspan="1" rowspan="1">ATCC 14537</td></tr><tr><td colspan="1" rowspan="1">Pantoea ananatis</td><td colspan="1" rowspan="1">NRRL B-41502</td></tr><tr><td colspan="1" rowspan="1">Pasteurella aerogenes</td><td colspan="1" rowspan="1">ATCC 27883</td></tr><tr><td colspan="1" rowspan="1">Pasteurella multicida subspmulticide</td><td colspan="1" rowspan="1">ATCC 12945</td></tr><tr><td colspan="1" rowspan="1">Prevotella intermedia</td><td colspan="1" rowspan="1">ATCC 15032</td></tr><tr><td colspan="1" rowspan="1">Prevotella corporis*</td><td colspan="1" rowspan="1">ATCC 33547</td></tr><tr><td colspan="1" rowspan="1">Prevotella oralis*</td><td colspan="1" rowspan="1">ATCC 33269</td></tr><tr><td colspan="1" rowspan="1">Prevotella nigrescens*</td><td colspan="1" rowspan="1">ATCC 33563</td></tr></table>

<table><tr><td rowspan=1 colspan=1>Gram Negative Organism</td><td rowspan=1 colspan=1>Strain ID</td></tr><tr><td rowspan=1 colspan=1>Providencia rettgeri</td><td rowspan=1 colspan=1>ATCC 9250</td></tr><tr><td rowspan=1 colspan=1>Providencia stuartii</td><td rowspan=1 colspan=1>ATCC 33672</td></tr><tr><td rowspan=1 colspan=1>Providencia alcalifaciens</td><td rowspan=1 colspan=1>ATCC 9886</td></tr><tr><td rowspan=1 colspan=1>Pseudomonas fluorescens</td><td rowspan=1 colspan=1>ATCC 13525</td></tr><tr><td rowspan=1 colspan=1>Pseudomonas putida</td><td rowspan=1 colspan=1>ATCC 49128</td></tr><tr><td rowspan=1 colspan=1>Pseudomonas alcaligenes</td><td rowspan=1 colspan=1>ATCC 14909</td></tr><tr><td rowspan=1 colspan=1>Ralstonia insidiosa</td><td rowspan=1 colspan=1>ATCC 49129</td></tr><tr><td rowspan=1 colspan=1>Ralstonia pickettii</td><td rowspan=1 colspan=1>ATCC 27511</td></tr><tr><td rowspan=1 colspan=1>Raoultella planticola (Klebsiellaplanticola)</td><td rowspan=1 colspan=1>ATCC 31900</td></tr><tr><td rowspan=1 colspan=1>Raoultella ornithinolytica</td><td rowspan=1 colspan=1>CDC# 0134</td></tr><tr><td rowspan=1 colspan=1>Raoultella terrigena (Klebsiellaterrigena)</td><td rowspan=1 colspan=1>ATCC 55553</td></tr><tr><td rowspan=1 colspan=1>Shigella boydii</td><td rowspan=1 colspan=1>ATCC 9207</td></tr><tr><td rowspan=1 colspan=1>Shigella sonneiE</td><td rowspan=1 colspan=1>ATCC 25931</td></tr><tr><td rowspan=1 colspan=1>Shigella flexneriE</td><td rowspan=1 colspan=1>ATCC 9199</td></tr><tr><td rowspan=1 colspan=1>Vibrio furnissii</td><td rowspan=1 colspan=1>NCTC11218</td></tr><tr><td rowspan=1 colspan=1>Vibrio alginolyticus</td><td rowspan=1 colspan=1>ATCC 17749</td></tr><tr><td rowspan=1 colspan=1>Vibrio parahaemolyticus</td><td rowspan=1 colspan=1>ATCC 17802</td></tr><tr><td rowspan=1 colspan=1>Yersinia enterocolitica subspenterocolitica</td><td rowspan=1 colspan=1>ATCC 9610</td></tr><tr><td rowspan=1 colspan=1>Yersinia ruckeri</td><td rowspan=1 colspan=1>ATCC 29473</td></tr><tr><td rowspan=1 colspan=1>Yersinia kristensenii</td><td rowspan=1 colspan=1>ATCC 33639</td></tr></table>

A. Cross-reactivity seen with Acinetobacter baumannii at a concentration $> 1 \mathrm { x } 1 0 ^ { 4 } \mathrm { C F U / m L }$ .   
B. Cross-reactivity seen with Enterobacter cloacae complex at a concentration $> 1 \mathrm { x } 1 0 ^ { 8 } \mathrm { C F U / m L }$ .   
C. Cross-reactivity seen with Enterobacter (non-cloacae complex) at a concentration $> 1 \mathrm { x } 1 0 ^ { 6 } \mathrm { C F U / m L }$ and Serratia at a concentration of $\mathrm { \dot { > } 1 x 1 0 ^ { 7 } C F U / m L }$ .   
D. Cross-reactivity seen with the Fusobacterium nucleatum assay.   
E. Cross-reactivity seen with the Escherichia coli assay.

Table 42: Cross-reactivity study, Off-Panel gram-positive organisms evaluated   

<table><tr><td rowspan=1 colspan=1>Gram Positive Organisms</td><td rowspan=1 colspan=1>Strain ID</td><td rowspan=1 colspan=1>Gram PositiveOrganisms</td><td rowspan=1 colspan=1>Strain ID</td></tr><tr><td rowspan=1 colspan=1>Actinomyces odontolyticus</td><td rowspan=1 colspan=1>ATCC 17929</td><td rowspan=1 colspan=1>Lactobacillus paracasei*</td><td rowspan=1 colspan=1>ATCC 25598</td></tr><tr><td rowspan=1 colspan=1>Clostridium perfringens</td><td rowspan=1 colspan=1>ATCC 13124</td><td rowspan=1 colspan=1>Lactobacillusacidophilus*</td><td rowspan=1 colspan=1>ATCC 314</td></tr><tr><td rowspan=1 colspan=1>Corynebacterium jeikeium</td><td rowspan=1 colspan=1>ATCC BAA-949</td><td rowspan=1 colspan=1>Lactobacillus crispatus</td><td rowspan=1 colspan=1>ATCC 33197</td></tr><tr><td rowspan=1 colspan=1>Corynebacterium renale</td><td rowspan=1 colspan=1>ATCC 19412</td><td rowspan=1 colspan=1>Lactobacillus rhamnosus</td><td rowspan=1 colspan=1>ATCC 39595</td></tr><tr><td rowspan=1 colspan=1>Corynebacterium ulcerans</td><td rowspan=1 colspan=1>ATCC 51799</td><td rowspan=1 colspan=1>Lactococcus lactis</td><td rowspan=1 colspan=1>ATCC 49032</td></tr><tr><td rowspan=1 colspan=1>Corynebacterium xerosis*</td><td rowspan=1 colspan=1>ATCC 373</td><td rowspan=1 colspan=1>Listeria innocua</td><td rowspan=1 colspan=1>ATCC 33090</td></tr><tr><td rowspan=1 colspan=1>Corynebacterium durum</td><td rowspan=1 colspan=1>ATCC 33449</td><td rowspan=1 colspan=1>Listeria monocytogenes</td><td rowspan=1 colspan=1>ATCC 7644</td></tr><tr><td rowspan=1 colspan=1>Corynebacterium diphtheriae*</td><td rowspan=1 colspan=1>ATCC 13812</td><td rowspan=1 colspan=1>Micrococcus luteus</td><td rowspan=1 colspan=1>ATCC 10240</td></tr><tr><td rowspan=1 colspan=1>Corynebacteriumpseudodiphtheriticum</td><td rowspan=1 colspan=1>ATCC 10700</td><td rowspan=1 colspan=1>Peptostreptococcusanaerobius</td><td rowspan=1 colspan=1>ATCC 27337</td></tr><tr><td rowspan=1 colspan=1>Corynebacterium striatum*</td><td rowspan=1 colspan=1>ATCC 43735</td><td rowspan=1 colspan=1>Propionibacterium acnes</td><td rowspan=1 colspan=1>ATCC 11827</td></tr><tr><td rowspan=1 colspan=1>Corynebacterium urealyticum</td><td rowspan=1 colspan=1>ATCC 43044</td><td rowspan=1 colspan=1>Rothia mucilaginosa</td><td rowspan=1 colspan=1>ATCC 25296</td></tr><tr><td rowspan=1 colspan=1>Lactobacillus casei</td><td rowspan=1 colspan=1>ATCC 39392</td><td rowspan=1 colspan=2></td></tr></table>

Table 43: Cross-reactivity, Off-Panel fungal organisms evaluated   

<table><tr><td rowspan=1 colspan=1>Fungal Pathogens</td><td rowspan=1 colspan=1>Strain ID</td><td rowspan=1 colspan=1>Concentration Tested</td></tr><tr><td rowspan=1 colspan=1>Aspergillus fumigatus*</td><td rowspan=1 colspan=1>ATCC 204305</td><td rowspan=1 colspan=1>2.50 x 106 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Candida orthopsilosis</td><td rowspan=1 colspan=1>ATCC 96139</td><td rowspan=1 colspan=1>1 x 107 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Candida metapsilosis</td><td rowspan=1 colspan=1>ATCC 96144</td><td rowspan=1 colspan=1>1 x 107 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Candida tropicalis</td><td rowspan=1 colspan=1>ATCC 1369</td><td rowspan=1 colspan=1>1 x 107 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Cryptococcus grubii</td><td rowspan=1 colspan=1>ATCC 208821</td><td rowspan=1 colspan=1>1 x 107 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Cryptococcus gattii</td><td rowspan=1 colspan=1>ATCC 76108</td><td rowspan=1 colspan=1>1 x 107 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Cryptococcus neoformans</td><td rowspan=1 colspan=1>ATCC 14116</td><td rowspan=1 colspan=1>1 x 107 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Geotrichum capitatum</td><td rowspan=1 colspan=1>ATCC 10663</td><td rowspan=1 colspan=1>1 x 107 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Histoplasma capsulatum</td><td rowspan=1 colspan=1>In silico</td><td rowspan=1 colspan=1>N/A</td></tr><tr><td rowspan=1 colspan=1>Penicillium marneffei</td><td rowspan=1 colspan=1>ATCC 200050</td><td rowspan=1 colspan=1>1 x 107 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Rhodotorula glutinis</td><td rowspan=1 colspan=1>ATCC 32765</td><td rowspan=1 colspan=1>1 x 107 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Rhodotorula mucilaginosa</td><td rowspan=1 colspan=1>ATCC 9449</td><td rowspan=1 colspan=1>1 x 107 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Rhodotorula minuta</td><td rowspan=1 colspan=1>ATCC 36236</td><td rowspan=1 colspan=1>1 x 107 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Saccharomyces cervisiae*</td><td rowspan=1 colspan=1>ATCC 18824</td><td rowspan=1 colspan=1>5.55 x 106 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Trichosporon dermatis</td><td rowspan=1 colspan=1>ATCC MYA-4294</td><td rowspan=1 colspan=1>1 x 107 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Trichosporon mucoides</td><td rowspan=1 colspan=1>ATCC 90046</td><td rowspan=1 colspan=1>1 x 107 CFU/mL</td></tr></table>

Table 44: Cross-reactivity, Off-Panel resistance genes evaluated   

<table><tr><td rowspan=1 colspan=1>Antimicrobial Resistance Genes</td><td rowspan=1 colspan=1>Strain ID</td><td rowspan=1 colspan=1>Concentration Tested</td></tr><tr><td rowspan=1 colspan=1>FOX (Carried by Klebsiella oxytoca)*A</td><td rowspan=1 colspan=1>JMI 954306</td><td rowspan=1 colspan=1>8 x 108 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>MOX (Carried by Aeromonas hydrophila)</td><td rowspan=1 colspan=1>JMI 938982</td><td rowspan=1 colspan=1>1 x 109 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>SME (Carried by Serratia marcescens)A</td><td rowspan=1 colspan=1>CDC #0091</td><td rowspan=1 colspan=1>1 x 109 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>SHV (Carried by Klebsiella pneumoniae)A</td><td rowspan=1 colspan=1>CDC #0087</td><td rowspan=1 colspan=1>1 x 109 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>TEM (Carried by Escherichia coli)A</td><td rowspan=1 colspan=1>NCTC 13351</td><td rowspan=1 colspan=1>1 x 109 CFU/mL</td></tr></table>

. The on-panel organism associated with the resistance gene was detected by the BCID-GN panel as expecte

In silico analysis was performed to identify any additional off-panel organisms for potential cross-reactivity with ePlex BCID-GN Panel targets. Results for gram-negative and gram-positive organisms that are predicted to generate false positive results with BCID-GN Panel are presented in Tables 45 and 46.

Table 45: Cross-reactivity, Off-Panel gram-negative organisms evaluated by in silico analysis   

<table><tr><td rowspan=1 colspan=1>Cross-reactive Organism</td><td rowspan=1 colspan=1>ePlex BCID-GN Target</td><td rowspan=1 colspan=1>No. of Sequences</td><td rowspan=1 colspan=1>Predicted Cross-ReactiveSequences*n (%)</td></tr><tr><td rowspan=1 colspan=1>Fusobacterium hwasookii</td><td rowspan=1 colspan=1>F. nucleatum</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>5 (50%)</td></tr><tr><td rowspan=1 colspan=1>Haemophilus aegyptius</td><td rowspan=1 colspan=1>H. influenzae</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>3 (100%)</td></tr><tr><td rowspan=1 colspan=1>Klebsiella michiganensis</td><td rowspan=1 colspan=1>Klebsiella oxytoca</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>40 (100%)</td></tr><tr><td rowspan=1 colspan=1>Pseudomonas denitrificans</td><td rowspan=1 colspan=1>Pseudomonas aeruginosa</td><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>16 (94.1%)</td></tr></table>

Table 46: Cross-reactivity, Off-Panel gram-positive organisms evaluated by in silico analysis   

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Number ofSequences</td><td rowspan=1 colspan=1>Predicted Cross-Reactive Sequencesn (%)</td></tr><tr><td rowspan=1 colspan=1>Brevibacterium halotolerans</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>3 (100%)</td></tr><tr><td rowspan=1 colspan=1>Domibacillus indicus</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1 (100%)</td></tr><tr><td rowspan=1 colspan=1>Domibacillus robiginosus</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1 (100%)</td></tr><tr><td rowspan=1 colspan=1>Salinibacillus aidingensis</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>1 (50%)</td></tr><tr><td rowspan=1 colspan=1>Terribacillus aidingensis</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1 (100%</td></tr><tr><td rowspan=1 colspan=1>Terribacillus halophilus</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>1 (50%)</td></tr><tr><td rowspan=1 colspan=1>Terribacillus saccharophilus</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1 (100%)</td></tr><tr><td rowspan=1 colspan=1>Planomicrobium okeanokoites</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1 (100%)</td></tr><tr><td rowspan=1 colspan=1>Lactococcus chungangensis</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>4 (100%)</td></tr><tr><td rowspan=1 colspan=1>Lactococcus laudensis</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1 (100%)</td></tr><tr><td rowspan=1 colspan=1>Lactococcus piscium</td><td rowspan=1 colspan=1>18</td><td rowspan=1 colspan=1>18 (100%)</td></tr><tr><td rowspan=1 colspan=1>Lactococcus plantarum</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>5 (83.8%)</td></tr><tr><td rowspan=1 colspan=1>Lactococcus raffinolactis</td><td rowspan=1 colspan=1>49</td><td rowspan=1 colspan=1>46 (93.9%)</td></tr><tr><td rowspan=1 colspan=1>Okadaella gastrococcus</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>4 (100%)</td></tr></table>

# h. Interfering Substances:

At total of 18 substances that could be present in blood culture specimens or introduced during sample handling were evaluated for potential interference with performance of the BCID-GN Panel. Each endogenous or exogenous substances was added to simulated positive blood culture samples which contained simulated blood culture matrix (human whole blood that had been incubated in a blood culture bottle) and one of three different organism mixes. Each mix contained organisms inoculated at concentrations equivalent to bottle positivity. Potentially interfering substances were spiked at medically relevant concentrations and were evaluated both without organism with all testing performed in triplicate. On each day of testing, each organism mix was evaluated without any interfering substance to serve as a positive control (no interference) to which the test specimens were compared.

None of the 18 substances commonly found in blood culture specimens or as medications commonly used to treat skin or blood infections were found to inhibit the ePlex BCIDGN Panel. Two substances required further dilution of the substance under evaluation to determine the final concentration for which interference is no longer observed. Caspofungin was diluted from $5 ~ { \mu \mathrm { g / m L } }$ to $4 . 5 ~ \mu \mathrm { g / m L }$ and $\gamma$ -globulin was diluted from 1.7 $\mathrm { g / d L }$ to $0 . 8 5 \ : \mathrm { g / d L }$ . The final testing concentration of caspofungin and $\gamma \mathrm { . }$ -globulin are above the clinically relevant concentration of those individuals suspected of having a bloodstream infection.

Table 47 includes the concentrations of the twelve organisms evaluated in the study. Table 48 includes the potentially interfering substances tested as well as their respective concentrations.

Table 47: Organism mixes and test concentrations   

<table><tr><td rowspan=1 colspan=1>Mix</td><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>ResistanceMarker</td><td rowspan=1 colspan=1>Test Concentration(CFU/mL)</td></tr><tr><td rowspan=4 colspan=1>1</td><td rowspan=1 colspan=1>Escherichia coli</td><td rowspan=1 colspan=1>OXA</td><td rowspan=1 colspan=1>2 x 108</td></tr><tr><td rowspan=1 colspan=1>Haemophilus influenzae</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>6 x 108</td></tr><tr><td rowspan=1 colspan=1>Enterobacter cloacae</td><td rowspan=1 colspan=1>CTX-M</td><td rowspan=1 colspan=1>2 x 108</td></tr><tr><td rowspan=1 colspan=1>Candida albicans (Pan Candidatarget)</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1 x 106</td></tr><tr><td rowspan=4 colspan=1>2</td><td rowspan=1 colspan=1>Klebsiella oxytoca</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>9 x 108</td></tr><tr><td rowspan=1 colspan=1>Pseudomonas aeruginosa</td><td rowspan=1 colspan=1>IMP</td><td rowspan=1 colspan=1>1 x 108</td></tr><tr><td rowspan=1 colspan=1>Acinetobacter baumannii</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>4 x 108</td></tr><tr><td rowspan=1 colspan=1>Enterobacter aerogenes</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>2 x 108</td></tr><tr><td rowspan=4 colspan=1>3</td><td rowspan=1 colspan=1>Neisseria meningitidis</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>3 x 107</td></tr><tr><td rowspan=1 colspan=1>Serratia marcescens</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>1 x 109</td></tr><tr><td rowspan=1 colspan=1>Bacteroides fragilis</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>4 x 108</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus aureus (Pan Gram-Positive target)</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>2 x 107</td></tr></table>

Table 48. Interfering substances and test concentrations   

<table><tr><td colspan="1" rowspan="1">Endogenous Substances</td><td colspan="1" rowspan="1">Test Concentration</td></tr><tr><td colspan="1" rowspan="1">Hemoglobin</td><td colspan="1" rowspan="1">0.6 g/L</td></tr><tr><td colspan="1" rowspan="1">Triglycerides</td><td colspan="1" rowspan="1">1000 mg/dL</td></tr><tr><td colspan="1" rowspan="1">Bilirubin</td><td colspan="1" rowspan="1">60 μg/mL</td></tr><tr><td colspan="1" rowspan="1">γ-globulin</td><td colspan="1" rowspan="1">0.85 g/dL</td></tr><tr><td colspan="1" rowspan="1">Human genomic DNA</td><td colspan="1" rowspan="1">6 x 105 copies/mL</td></tr><tr><td colspan="1" rowspan="1">Exogenous Substances</td><td colspan="1" rowspan="1">Test Concentration</td></tr><tr><td colspan="1" rowspan="1">Fluconazole</td><td colspan="1" rowspan="1">25 mg/L</td></tr><tr><td colspan="1" rowspan="1">Vancomycin</td><td colspan="1" rowspan="1">30 mg/L</td></tr><tr><td colspan="1" rowspan="1">Ciprofloxacin</td><td colspan="1" rowspan="1">3 mg/L</td></tr><tr><td colspan="1" rowspan="1">Gentamicin sulfate</td><td colspan="1" rowspan="1">3 μg/mL</td></tr><tr><td colspan="1" rowspan="1">Imipenem</td><td colspan="1" rowspan="1">83 μg/mL</td></tr><tr><td colspan="1" rowspan="1">Ceftriaxone</td><td colspan="1" rowspan="1">0.23 mg/mL</td></tr><tr><td colspan="1" rowspan="1">Tetracycline</td><td colspan="1" rowspan="1">5 mg/L</td></tr><tr><td colspan="1" rowspan="1">Amoxicillin/Clavulanate</td><td colspan="1" rowspan="1">3.5 μg/mL</td></tr><tr><td colspan="1" rowspan="1">Heparin</td><td colspan="1" rowspan="1">0.9 U/mL</td></tr><tr><td colspan="1" rowspan="1">Sodium Polyanetholsulfonate</td><td colspan="1" rowspan="1">0.25% w/v</td></tr><tr><td colspan="1" rowspan="1">Amphotericin B</td><td colspan="1" rowspan="1">2 μg/mL</td></tr><tr><td colspan="1" rowspan="1">Flucytosine</td><td colspan="1" rowspan="1">90 μg/mL</td></tr><tr><td colspan="1" rowspan="1">Caspofungin</td><td colspan="1" rowspan="1">4.5 μg/mL</td></tr></table>

# i. Competitive Inhibition:

A competitive inhibition study was conducted to evaluate the possibility of high concentration organisms interfering with the detection of lower concentrations targets in the same blood culture specimen. Four organism mixes covering eight clinically relevant co-infection organisms including both on and off-panel analytes were selected for the study based on those identified in the peer-reviewed literature as common polymicrobial co-infections. The four mixes were used to formulate test case scenarios by pairing one mix with organisms at bottle positive concentrations with a second mix with organism at concentrations typically present at bottle positivity plus eight hours. Testing was performed in triplicate for each test condition. A summary of the organisms and test concentrations evaluated in the study is presented in Table 49. Testing generated $100 \%$ detected results for all targeted organisms. There were no false positive results for samples with the non-targeted organism, Corynebacterium striatum. The study results demonstrated the ability of the ePlex BCID-GN Panel to detect two clinically relevant coinfecting organisms in a specimen at both high and low concentrations. Detailed study results are presented in Table 50.

Table 49: Competitive Inhibition, Organisms and Concentrations Tested   

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>HighConcentration</td><td rowspan=1 colspan=1>LowConcentration</td></tr><tr><td rowspan=1 colspan=1>Klebsiella pneumoniae</td><td rowspan=1 colspan=1>1 x 109 CFU/mL</td><td rowspan=1 colspan=1>9 x 108 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Escherichia coli (CTX-M+)</td><td rowspan=1 colspan=1>1 x 109 CFU/mL</td><td rowspan=1 colspan=1>2 x 108 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Enterobacter cloacae (VIM+)</td><td rowspan=1 colspan=1>7 x 108 CFU/mL</td><td rowspan=1 colspan=1>2 x 108 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Klebsiella oxytoca (KPC+)</td><td rowspan=1 colspan=1>1 x 109 CFU/mL</td><td rowspan=1 colspan=1>9 x 108 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Pseudomonas aeruginosa (IMP+)</td><td rowspan=1 colspan=1>8 x 108 CFU/mL</td><td rowspan=1 colspan=1>1 x 108 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Serratia marcescens</td><td rowspan=1 colspan=1>2 x 109 CFU/mL</td><td rowspan=1 colspan=1>1 x 109 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus aureus</td><td rowspan=1 colspan=1>1 x 108 CFU/mL</td><td rowspan=1 colspan=1>2 x 107 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Corynebacterium striatum4</td><td rowspan=1 colspan=1>2 x 109 CFU/mL</td><td rowspan=1 colspan=1>4 x 106 CFU/mL</td></tr></table>

A. Off-panel organism

Table 50: Percent Detection of all Organisms in Each Mix   
$j$ . Specimen matrix equivalency study: Evaluation of blood culture bottle types   

<table><tr><td></td><td></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>s M Dacl T</td><td></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>MixCombination</td><td rowspan=1 colspan=1>Low TiterOrganisms(BP)</td><td rowspan=1 colspan=1>Conc.(CFU/mL)</td><td rowspan=1 colspan=1>%Detection</td><td rowspan=1 colspan=1>High TiterOrganisms (BP+8hrs)</td><td rowspan=1 colspan=1>Conc.(CFU/mL)</td><td rowspan=1 colspan=1>%Detection</td></tr><tr><td rowspan=2 colspan=1>1 &amp; 2</td><td rowspan=1 colspan=1>K. pneumoniae</td><td rowspan=1 colspan=1>9 x 108</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>E. coli(CTX-M)</td><td rowspan=1 colspan=1>1 x 109</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>C. striatum</td><td rowspan=1 colspan=1>4 x 106</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>E. cloacae(VIM)</td><td rowspan=1 colspan=1>7 x 108</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=2 colspan=1>2 &amp; 1</td><td rowspan=1 colspan=1>E. coli(CTX-M)</td><td rowspan=1 colspan=1>2 x 108</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>K. pneumoniae</td><td rowspan=1 colspan=1>1 x 109</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>E. cloacae(VIM)</td><td rowspan=1 colspan=1>2 x 108</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>C. striatum</td><td rowspan=1 colspan=1>2 x 109</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=2 colspan=1>1 &amp; 3</td><td rowspan=1 colspan=1>K. pneumoniae</td><td rowspan=1 colspan=1>9 x 108</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>K. oxytoca(KPC)</td><td rowspan=1 colspan=1>1 x 109</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>C. striatum</td><td rowspan=1 colspan=1>4 x 106</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>P. aeruginosa(IMP)</td><td rowspan=1 colspan=1>8 x 108</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=2 colspan=1>3 &amp; 1</td><td rowspan=1 colspan=1>K. oxytoca(KPC)</td><td rowspan=1 colspan=1>9 x 108</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>K. pneumoniae</td><td rowspan=1 colspan=1>1 x 109</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>P. aeruginosa(IMP)</td><td rowspan=1 colspan=1>1 x 108</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>C. striatum</td><td rowspan=1 colspan=1>2 x 109</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=2 colspan=1>1 &amp; 4</td><td rowspan=1 colspan=1>K. pneumoniae</td><td rowspan=1 colspan=1>9 x 108</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>S. aureus</td><td rowspan=1 colspan=1>1 x 108</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>C. striatum</td><td rowspan=1 colspan=1>4 x 106</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>S. marcescens</td><td rowspan=1 colspan=1>2 x 109</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=2 colspan=1>4 &amp; 1</td><td rowspan=1 colspan=1>S. aureus</td><td rowspan=1 colspan=1>2 x 107</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>K. pneumoniae</td><td rowspan=1 colspan=1>1 x 109</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>S. marcescens</td><td rowspan=1 colspan=1>1 x 109</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>C. striatum</td><td rowspan=1 colspan=1>2 x 109</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=2 colspan=1>2 &amp; 3</td><td rowspan=1 colspan=1>E. coli(CTX-M)</td><td rowspan=1 colspan=1>2 x 108</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>K. oxytoca(KPC)</td><td rowspan=1 colspan=1>1 x 109</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>E. cloacae(VIM)</td><td rowspan=1 colspan=1>2 x 108</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>P. aeruginosa(IMP)</td><td rowspan=1 colspan=1>8 x 108</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=2 colspan=1>3 &amp; 2</td><td rowspan=1 colspan=1>K. oxytoca(KPC)</td><td rowspan=1 colspan=1>9 x 108</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>E. coli(CTX-M)</td><td rowspan=1 colspan=1>1 x 109</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>P. aeruginosa(IMP)</td><td rowspan=1 colspan=1>1 x 108</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>E. cloacae(VIM)</td><td rowspan=1 colspan=1>7 x 108</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=2 colspan=1>2 &amp; 4</td><td rowspan=1 colspan=1>E. coli(CTX-M)</td><td rowspan=1 colspan=1>2 x 108</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>S. aureus</td><td rowspan=1 colspan=1>1 x 108</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>E. cloacae(VIM)</td><td rowspan=1 colspan=1>2 x 108</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>S. marcescens</td><td rowspan=1 colspan=1>2 x 109</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=2 colspan=1>4 &amp; 2</td><td rowspan=1 colspan=1>S. aureus</td><td rowspan=1 colspan=1>2 x 107</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>E. coli(CTX-M)</td><td rowspan=1 colspan=1>1 x 109</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>S. marcescens</td><td rowspan=1 colspan=1>1 x 109</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>E. cloacae(VIM)</td><td rowspan=1 colspan=1>7 x 108</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=2 colspan=1>3 &amp; 4</td><td rowspan=1 colspan=1>K. oxytoca(KPC)</td><td rowspan=1 colspan=1>9 x 108</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>S. aureus</td><td rowspan=1 colspan=1>1 x 108</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>P. aeruginosa(IMP)</td><td rowspan=1 colspan=1>1 x 108</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>S. marcescens</td><td rowspan=1 colspan=1>2 x 109</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=2 colspan=1>4 &amp; 3</td><td rowspan=1 colspan=1>S. aureus</td><td rowspan=1 colspan=1>2 x 107</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>K. oxytoca(KPC)</td><td rowspan=1 colspan=1>1 x 109</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>S. marcescens</td><td rowspan=1 colspan=1>1 x 109</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>P. aeruginosa(IMP)</td><td rowspan=1 colspan=1>8 x 108</td><td rowspan=1 colspan=1>100</td></tr></table>

Thirteen different bottle types indicated for use with three different blood culture monitoring systems (BacT/Alert, BACTEC and VersaTREK) were evaluated analytically with the ePlex BCID-GN panel. Results from this study will be used to support the analytical performance claims of the ePlex BCID-GN Panel as well as to assess for the potential presence of non-viable organisms or organism nucleic acids in blood culture media for each bottle type.

Testing was performed using three organism mixes with organism concentrations approximating bottle positivity. Each of the blood culture bottle types were tested with the manufacturer’s recommended ratio of human whole blood to media. The organisms evaluated that were also used for the competitive inhibition study, represent each of the eight multiplex primer amplification pools on the BCID-GN Panel. Two lots of the 13 different blood culture bottles were evaluated using five replicates of the three organism mixes. Three negative control (negative blood matrix, no organism mix) replicates were also tested per lot of each bottle type. The thirteen bottle types evaluated are listed in Table 51 below.

Twelve of the bottle types evaluated showed no interference for any of the targets tested. One of three lots of the BACTEC Lytic Anaerobic bottles tested showed reduced sensitivity (i.e., false negative results) in initial testing for four analytes. Additional replicates tested with a new bottle lot generated the expected positive results. The following are the results observed for BACTEC Lytic Anaerobic bottles for the initially discordant analytes:

• Pan Candida was not detected in 2/15 sample replicates • Enterobacter cloacae was not detected in 1/15 sample replicates • Escherichia coli was not detected in 1/15 sample replicates CTX-M was not detected in 2/15 sample replicates

A limitation is included in the package insert stating that the ePlex BCID-GN Panel may have reduced sensitivity with BACTEC Lytic Anaerobic bottles.

Table 51: Sample Matrix Equivalency (Bottle Evaluation) Bottle Types   

<table><tr><td rowspan=1 colspan=1>Manufacturer</td><td rowspan=1 colspan=1>Bottle Brand</td><td rowspan=1 colspan=1>Bottle Type</td><td rowspan=1 colspan=1>Study Outcome</td></tr><tr><td rowspan=1 colspan=1>BD</td><td rowspan=1 colspan=1>BACTEC</td><td rowspan=1 colspan=1>Plus Aerobic</td><td rowspan=1 colspan=1>No interference observed</td></tr><tr><td rowspan=1 colspan=1>BD</td><td rowspan=1 colspan=1>BACTEC</td><td rowspan=1 colspan=1>Plus Anaerobic</td><td rowspan=1 colspan=1>No interference observed</td></tr><tr><td rowspan=1 colspan=1>BD</td><td rowspan=1 colspan=1>BACTEC</td><td rowspan=1 colspan=1>StandardAerobic</td><td rowspan=1 colspan=1>No interference observed</td></tr><tr><td rowspan=1 colspan=1>BD</td><td rowspan=1 colspan=1>BACTEC</td><td rowspan=1 colspan=1>StandardAnaerobic</td><td rowspan=1 colspan=1>No interference observed</td></tr><tr><td rowspan=1 colspan=1>BD</td><td rowspan=1 colspan=1>BACTEC</td><td rowspan=1 colspan=1>Peds Plus</td><td rowspan=1 colspan=1>No interference observed</td></tr><tr><td rowspan=1 colspan=1>BD</td><td rowspan=1 colspan=1>BACTEC</td><td rowspan=1 colspan=1>LyticAnaerobic</td><td rowspan=1 colspan=1>False negative results were observed for PanCandida, Enterobacter cloacae, Escherichiacoli, CTX-M and OXA in one of three lots.*</td></tr><tr><td rowspan=1 colspan=1>bioMérieux</td><td rowspan=1 colspan=1>BACT/ALERT</td><td rowspan=1 colspan=1>SA StandardAerobic</td><td rowspan=1 colspan=1>No interference observed</td></tr><tr><td rowspan=1 colspan=1>bioMérieux</td><td rowspan=1 colspan=1>BACT/ALERT</td><td rowspan=1 colspan=1>SN StandardAnaerobic</td><td rowspan=1 colspan=1>No interference observed</td></tr><tr><td rowspan=1 colspan=1>bioMérieux</td><td rowspan=1 colspan=1>BACT/ALERT</td><td rowspan=1 colspan=1>FA Plus</td><td rowspan=1 colspan=1>No interference observed</td></tr><tr><td rowspan=1 colspan=1>bioMérieux</td><td rowspan=1 colspan=1>BACT/ALERT</td><td rowspan=1 colspan=1>FN Plus</td><td rowspan=1 colspan=1>No interference observed</td></tr><tr><td rowspan=1 colspan=1>bioMérieux</td><td rowspan=1 colspan=1>BACT/ALERT</td><td rowspan=1 colspan=1>PF Plus</td><td rowspan=1 colspan=1>No interference observed</td></tr><tr><td rowspan=1 colspan=1>Thermo Scientific</td><td rowspan=1 colspan=1>VersaTREK</td><td rowspan=1 colspan=1>REDOX 1 EZDraw Aerobic</td><td rowspan=1 colspan=1>No interference observed</td></tr><tr><td rowspan=1 colspan=1>Thermo Scientific</td><td rowspan=1 colspan=1>VersaTREK</td><td rowspan=1 colspan=1>REDOX 2 EZDrawAnaerobic</td><td rowspan=1 colspan=1>No interference observed</td></tr></table>

$^ { * } 2 / 1 5$ replicates were false negative for Pan Candida; 1/15 replicates was false negative for Enterobacter cloacae; 1/15 replicates was false negative for Escherichia coli (OXA-48); 2/15 replicates were false negative for CTX-M

# k. Carry-Over/Cross-contamination:

Potential run-to-run carryover or cross-contamination was evaluated by determining whether a high positive sample in one cartridge would cause false positive results in subsequently tested cartridges. It is noted that the configuration of the ePlex instrument mitigates the risk for carry-over/cross-contamination because each bay functions independently and is physically separated from other bays, therefore, not allowing cartridges to interact with one another. The carry-over/cross-contamination rate of the ePlex BCID-GN Panel and ePlex Instrument was tested in a checker board approach by running high positive and negative samples interspersed in all towers of a four-tower ePlex instrument over five consecutive runs on separate days. The high positive sample consisted of a multi-organism mix of four bacterial and one fungal species, covering eight assay targets, prepared at concentrations higher than typically found in positive blood cultures (i.e., $\mathrm { { 1 \ x 1 0 ^ { 9 } C F U / m L } }$ for bacteria and $1 \mathrm { ~ x ~ } 1 0 ^ { 7 } \mathrm { \dot { C } F U / m L }$ for fungi). Negative blood culture matrix was used for negative samples.

Testing resulted in detection of all expected targets for samples containing organism. No carry-over/cross-contamination (i.e., no false positive results) was observed when testing consecutively within bays or in adjacent bays over 120 total runs.

l. Assay cutoffs:

Assay cutoff analysis was conducted to define signal boundaries that would discriminate between positive and negative detections for each ePlex BCID-GN Panel analyte and for the internal control. Clinical specimens, contrived positive blood culture specimens, and organisms spiked at or above the determined LoD were used to establish assay-cutoffs. Contrived samples were prepared by spiking organisms at low concentrations into simulated negative blood culture matrix and grown to bottle positivity on a blood culture continuous monitoring system. An appropriate boundary was established per target wherein specificity (limit the number of potential false positives) and sensitivity (limit the number of potential false negatives) were maximized and verified by ROC analysis. The boundary set points that were determined during the cutoff study are listed in Table 52.

Table 52: Summary of BCID-GN Panel cutoffs   

<table><tr><td colspan="1" rowspan="1">Target</td><td colspan="1" rowspan="1">Boundary (nA)</td></tr><tr><td colspan="1" rowspan="1">Acinetobacter baumannii</td><td colspan="1" rowspan="1">10</td></tr><tr><td colspan="1" rowspan="1">Bacteroides fragilis</td><td colspan="1" rowspan="1">20</td></tr><tr><td colspan="1" rowspan="1">Citrobacter</td><td colspan="1" rowspan="1">20</td></tr><tr><td colspan="1" rowspan="1">Cronobacter sakazakii</td><td colspan="1" rowspan="1">20</td></tr><tr><td colspan="1" rowspan="1">Enterobacter aerogenes</td><td colspan="1" rowspan="1">20</td></tr><tr><td colspan="1" rowspan="1">Enterobacter amnigenus</td><td colspan="1" rowspan="1">10</td></tr><tr><td colspan="1" rowspan="1">Enterobacter asburiae</td><td colspan="1" rowspan="1">25</td></tr><tr><td colspan="1" rowspan="1">Enterobacter cloacae</td><td colspan="1" rowspan="1">20</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">30</td></tr><tr><td colspan="1" rowspan="1">Fusobacterium necrophorum</td><td colspan="1" rowspan="1">40</td></tr><tr><td colspan="1" rowspan="1">Fusobacterium nucleatum</td><td colspan="1" rowspan="1">20</td></tr><tr><td colspan="1" rowspan="1">Haemophilus influenzae</td><td colspan="1" rowspan="1">20</td></tr><tr><td colspan="1" rowspan="1">Klebsiella oxytoca</td><td colspan="1" rowspan="1">20</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">40</td></tr><tr><td colspan="1" rowspan="1">Morganella morganii</td><td colspan="1" rowspan="1">20</td></tr><tr><td colspan="1" rowspan="1">Neisseria meningitidis</td><td colspan="1" rowspan="1">40</td></tr><tr><td colspan="1" rowspan="1">Proteus</td><td colspan="1" rowspan="1">20</td></tr><tr><td colspan="1" rowspan="1">Proteus mirabilis</td><td colspan="1" rowspan="1">60</td></tr><tr><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">5</td></tr><tr><td colspan="1" rowspan="1">Salmonella</td><td colspan="1" rowspan="1">5</td></tr><tr><td colspan="1" rowspan="1">Serratia</td><td colspan="1" rowspan="1">20</td></tr><tr><td colspan="1" rowspan="1">Serratia marcescens</td><td colspan="1" rowspan="1">20</td></tr><tr><td colspan="1" rowspan="1">Stenotrophomonas maltophilia</td><td colspan="1" rowspan="1">50</td></tr><tr><td colspan="1" rowspan="1">CTX-M</td><td colspan="1" rowspan="1">40</td></tr><tr><td colspan="1" rowspan="1">IMP</td><td colspan="1" rowspan="1">20</td></tr><tr><td colspan="1" rowspan="1">KPC</td><td colspan="1" rowspan="1">20</td></tr><tr><td colspan="1" rowspan="1">NDM</td><td colspan="1" rowspan="1">20</td></tr><tr><td colspan="1" rowspan="1">OXA-23</td><td colspan="1" rowspan="1">50</td></tr><tr><td colspan="1" rowspan="1">OXA-48</td><td colspan="1" rowspan="1">20</td></tr><tr><td colspan="1" rowspan="1">VIM</td><td colspan="1" rowspan="1">20</td></tr><tr><td colspan="1" rowspan="1">Pan Gram-Positive Enterococcus faecalis</td><td colspan="1" rowspan="1">60</td></tr><tr><td colspan="1" rowspan="1">Pan Gram-Positive Bacillus</td><td colspan="1" rowspan="1">50</td></tr><tr><td colspan="1" rowspan="1">Pan Gram-Positive S. anginosus</td><td colspan="1" rowspan="1">15</td></tr><tr><td colspan="1" rowspan="1">Pan Gram-Positive Enterococcus</td><td colspan="1" rowspan="1">15</td></tr><tr><td colspan="1" rowspan="1">Pan Gram-Positive Strep/Staph</td><td colspan="1" rowspan="1">50</td></tr><tr><td colspan="1" rowspan="1">Pan Candida</td><td colspan="1" rowspan="1">30</td></tr><tr><td colspan="1" rowspan="1">Schizosaccharomyces pombe</td><td colspan="1" rowspan="1">20</td></tr><tr><td colspan="1" rowspan="1">Synthetic control</td><td colspan="1" rowspan="1">20</td></tr></table>

2. Comparison studies:

a. Method comparison with predicate device: N/A, Refer to the clinical studies section b. Matrix comparison: N/A

3. Clinical studies:

The clinical performance of the ePlex BCID-GN Panel was evaluated during a multi-center study with testing comprised of prospective and retrospective positive blood culture specimens that contained gram-negative or gram-variable organisms by Gram stain. All blood culture bottles were flagged positive by a continuously monitoring blood culture system and removed from the system within eight hours of positivity and tested with the ePlex BCID-GN Panel.

Bottle types evaluated in the clinical study included 13 different types from three different manufacturers as shown in Table 53:

Table 53: Bottle types evaluated, prospective and retrospective testing   

<table><tr><td rowspan=1 colspan=3>Bottle Types</td></tr><tr><td rowspan=1 colspan=1>BacT/Alert SA</td><td rowspan=1 colspan=1>BACTEC - Peds Plus /F</td><td rowspan=1 colspan=1>VersaTREK - Redox 1 EZDraw</td></tr><tr><td rowspan=1 colspan=1>BacT/Alert SN</td><td rowspan=1 colspan=1>BACTEC - Standard/10Aerobic/F</td><td rowspan=1 colspan=1>VersaTREK - Redox 2</td></tr><tr><td rowspan=1 colspan=1>BACTEC Lytic/10 Anaerobic/</td><td rowspan=1 colspan=1>BACTEC - Standard/10Anaerobic/F</td><td rowspan=1 colspan=1>VersaTREK - Redox 2 EZDraw</td></tr><tr><td rowspan=1 colspan=1>BACTEC PLUS Aerobic/F</td><td rowspan=1 colspan=1>BACTEC StandardAnaerobic/F</td><td rowspan=2 colspan=1></td></tr><tr><td rowspan=1 colspan=1>BACTEC PLUS Anaerobic/F</td><td rowspan=1 colspan=1>VersaTREK - Redox 1</td></tr></table>

Six clinical test sites, each having a 4-tower ePlex instrument (24 bays), participated in the prospective clinical evaluation of the ePlex BCID-GN Panel with testing performed by trained laboratory personnel. A total of seven ePlex BCID-GN Panel cartridge lots were used in the prospective clinical evaluation.

Prospective specimens were collected at seven clinical sites in two phases. A total of 183 specimens were prospectively collected and frozen prior to testing and 171 specimens were prospectively collected and tested fresh (never frozen) for a total of 354 specimens tested across the 2 phases. One specimen was withdrawn due to organism identification from unacceptable methods. Of the 353 prospectively-collected specimens eligible for testing, 349 were evaluable (i.e., valid ePlex BCID-GN Panel results and a valid comparator result). Four specimens were not evaluable as they did not generate valid ePlex BCID-GN Panel results and were excluded from performance evaluations.

To supplement the prospective study for low prevalence organism targets, a total of 578 positive blood culture specimens (577 evaluable) were collected from 10 collection sites and retrospectively tested. One specimen was not evaluable because it did not have a final, valid ePlex BCID-GN Panel result and was excluded from performance evaluations.

Demographic information for each collection site is described in Tables 54 and 55 for the prospectively-collected and retrospectively collected specimen cohorts.

Table 54: Demographic Data by Collection Site (Prospective Specimens)   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>All SitesN = 349n (%)</td><td rowspan=1 colspan=1>Site 1N = 88n (%)</td><td rowspan=1 colspan=1>Site 2N = 23n (%)</td><td rowspan=1 colspan=1>Site 3N = 98n (%)</td><td rowspan=1 colspan=1>Site 4N = 58n (%)</td><td rowspan=1 colspan=1>Site 5N = 46n (%)</td><td rowspan=1 colspan=1>Site 6N = 28n (%)</td><td rowspan=1 colspan=1>Site 7N = 8n (%)</td></tr><tr><td rowspan=1 colspan=8>Sex</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Male</td><td rowspan=1 colspan=1>168 (48.1)</td><td rowspan=1 colspan=1>37 (42.0)</td><td rowspan=1 colspan=1>12 (52.2)</td><td rowspan=1 colspan=1>52 (53.1)</td><td rowspan=1 colspan=1>28 (48.3)</td><td rowspan=1 colspan=1>21 (45.7)</td><td rowspan=1 colspan=1>13 (46.4)</td><td rowspan=1 colspan=1>5 (62.5)</td></tr><tr><td rowspan=1 colspan=1>Female</td><td rowspan=1 colspan=1>181 (51.9)</td><td rowspan=1 colspan=1>51 (58.0)</td><td rowspan=1 colspan=1>11 (47.8)</td><td rowspan=1 colspan=1>46 (46.9)</td><td rowspan=1 colspan=1>30 (51.7)</td><td rowspan=1 colspan=1>25 (54.3)</td><td rowspan=1 colspan=1>15 (53.6)</td><td rowspan=1 colspan=1>3 (37.5)</td></tr><tr><td rowspan=1 colspan=8>Age</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>&lt;1 yr</td><td rowspan=1 colspan=1>7 (2.0)</td><td rowspan=1 colspan=1>2 (2.3)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>4 (4.1)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>1 (2.2)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td></tr><tr><td rowspan=1 colspan=1>1-17 yrs</td><td rowspan=1 colspan=1>10 (2.9)</td><td rowspan=1 colspan=1>4 (4.5)</td><td rowspan=1 colspan=1>1 (4.3)</td><td rowspan=1 colspan=1>3 (3.1)</td><td rowspan=1 colspan=1>1 (1.7)</td><td rowspan=1 colspan=1>1 (2.2)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td></tr><tr><td rowspan=1 colspan=1>18-44 yrs</td><td rowspan=1 colspan=1>50 (14.3)</td><td rowspan=1 colspan=1>10 (11.4)</td><td rowspan=1 colspan=1>3 (13.0)</td><td rowspan=1 colspan=1>20 (20.4)</td><td rowspan=1 colspan=1>3 (5.2)</td><td rowspan=1 colspan=1>8 (17.4)</td><td rowspan=1 colspan=1>6 (21.4)</td><td rowspan=1 colspan=1>0 (0.0)</td></tr><tr><td rowspan=1 colspan=1>45-64 yrs</td><td rowspan=1 colspan=1>124 (35.5)</td><td rowspan=1 colspan=1>35 (39.8)</td><td rowspan=1 colspan=1>9 (39.1)</td><td rowspan=1 colspan=1>28 (28.6)</td><td rowspan=1 colspan=1>21 (36.2)</td><td rowspan=1 colspan=1>14 (30.4)</td><td rowspan=1 colspan=1>13 (46.4)</td><td rowspan=1 colspan=1>4 (50.0)</td></tr><tr><td rowspan=1 colspan=1>65-84 yrs</td><td rowspan=1 colspan=1>125 (35.8)</td><td rowspan=1 colspan=1>29 (33.0)</td><td rowspan=1 colspan=1>8 (34.8)</td><td rowspan=1 colspan=1>35 (35.7)</td><td rowspan=1 colspan=1>25 (43.1)</td><td rowspan=1 colspan=1>17 (37.0)</td><td rowspan=1 colspan=1>7 (25.0)</td><td rowspan=1 colspan=1>4 (50.0)</td></tr><tr><td rowspan=1 colspan=1>85+ yrs</td><td rowspan=1 colspan=1>33 (9.5)</td><td rowspan=1 colspan=1>8 (9.1)</td><td rowspan=1 colspan=1>2 (8.7)</td><td rowspan=1 colspan=1>8 (8.2)</td><td rowspan=1 colspan=1>8 (13.8)</td><td rowspan=1 colspan=1>5 (10.9)</td><td rowspan=1 colspan=1>2 (7.1)</td><td rowspan=1 colspan=1>0 (0.0)</td></tr></table>

Table 55: Demographic Data by Collection Site (Retrospective Specimens)   

<table><tr><td rowspan=2 colspan=1></td><td rowspan=2 colspan=1>AllSitesN =57n (%)</td><td rowspan=2 colspan=1>Site 1N =78n (%)</td><td rowspan=2 colspan=1>Site 2N = 73n (%)</td><td rowspan=2 colspan=1>Site 3N =31n (%)</td><td rowspan=2 colspan=1>Site 4N = 93n (%)</td><td rowspan=1 colspan=1>Site 5</td><td rowspan=2 colspan=1>Site 6N = 80n (%)</td><td rowspan=2 colspan=1>Site 7N = 67n (%)</td><td rowspan=2 colspan=1>Site 8N =48n (%)</td><td rowspan=2 colspan=1>Site 9N = 29n (%)</td><td rowspan=2 colspan=1>Site 10N =7n (%)</td></tr><tr><td rowspan=1 colspan=1>N = 1n (%)</td></tr><tr><td rowspan=1 colspan=11>Sex</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Male</td><td rowspan=1 colspan=1>307(53.2)</td><td rowspan=1 colspan=1>36(46.2)</td><td rowspan=1 colspan=1>41(56.2)</td><td rowspan=1 colspan=1>15(48.4)</td><td rowspan=1 colspan=1>49(52.7)</td><td rowspan=1 colspan=1>0(0.0)</td><td rowspan=1 colspan=1>47(58.8)</td><td rowspan=1 colspan=1>38(56.7)</td><td rowspan=1 colspan=1>29(60.4)</td><td rowspan=1 colspan=1>19(65.5)</td><td rowspan=1 colspan=1>33(42.9)</td></tr><tr><td rowspan=1 colspan=1>Female</td><td rowspan=1 colspan=1>270(46.8)</td><td rowspan=1 colspan=1>42(53.8)</td><td rowspan=1 colspan=1>32(43.8)</td><td rowspan=1 colspan=1>16(51.6)</td><td rowspan=1 colspan=1>44(47.3)</td><td rowspan=1 colspan=1>1(100)</td><td rowspan=1 colspan=1>33(41.3)</td><td rowspan=1 colspan=1>29(43.3)</td><td rowspan=1 colspan=1>19(39.6)</td><td rowspan=1 colspan=1>10(34.5)</td><td rowspan=1 colspan=1>44(57.1)</td></tr><tr><td rowspan=1 colspan=4>Age</td><td rowspan=1 colspan=2></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>&lt;1 yr</td><td rowspan=1 colspan=1>9 (1.6)</td><td rowspan=1 colspan=1>1 (1.3)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>3 (3.2)</td><td rowspan=1 colspan=1>0(0.0)</td><td rowspan=1 colspan=1>2 (2.5)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>1 (2.1)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>2 (2.6)</td></tr><tr><td rowspan=1 colspan=1>1-17 yrs</td><td rowspan=1 colspan=1>20 (3.5)</td><td rowspan=1 colspan=1>1 (1.3)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>1 (3.2)</td><td rowspan=1 colspan=1>8 (8.6)</td><td rowspan=1 colspan=1>0(0.0)</td><td rowspan=1 colspan=1>6 (7.5)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>1 (3.4)</td><td rowspan=1 colspan=1>3 (3.9)</td></tr><tr><td rowspan=1 colspan=1>18-44 yrs</td><td rowspan=1 colspan=1>78(13.5)</td><td rowspan=1 colspan=1>13(16.7)</td><td rowspan=1 colspan=1>7 (9.6)</td><td rowspan=1 colspan=1>2 (6.5)</td><td rowspan=1 colspan=1>10(10.8)</td><td rowspan=1 colspan=1>1(100)</td><td rowspan=1 colspan=1>15(18.8)</td><td rowspan=1 colspan=1>8(11.9)</td><td rowspan=1 colspan=1>8(16.7)</td><td rowspan=1 colspan=1>6(20.7)</td><td rowspan=1 colspan=1>8(10.4)</td></tr><tr><td rowspan=1 colspan=1>45-64 yrs</td><td rowspan=1 colspan=1>193(33.4)</td><td rowspan=1 colspan=1>27(34.6)</td><td rowspan=1 colspan=1>18(24.7)</td><td rowspan=1 colspan=1>13(41.9)</td><td rowspan=1 colspan=1>27(29.0)</td><td rowspan=1 colspan=1>0(0.0)</td><td rowspan=1 colspan=1>32(40.0)</td><td rowspan=1 colspan=1>27(40.3)</td><td rowspan=1 colspan=1>16(33.3)</td><td rowspan=1 colspan=1>9 (31.0)</td><td rowspan=1 colspan=1>24(31.2)</td></tr><tr><td rowspan=1 colspan=1>65-84 yrs</td><td rowspan=1 colspan=1>226(39.2)</td><td rowspan=1 colspan=1>29(37.2)</td><td rowspan=1 colspan=1>40(54.8)</td><td rowspan=1 colspan=1>11(35.5)</td><td rowspan=1 colspan=1>40(43.0)</td><td rowspan=1 colspan=1>0(0.0)</td><td rowspan=1 colspan=1>20(25.0)</td><td rowspan=1 colspan=1>24(35.8)</td><td rowspan=1 colspan=1>21(43.8)</td><td rowspan=1 colspan=1>11(37.9)</td><td rowspan=1 colspan=1>30(39.0)</td></tr><tr><td rowspan=1 colspan=1>85+ yrs</td><td rowspan=1 colspan=1>49 (8.5)</td><td rowspan=1 colspan=1>7 (9.0)</td><td rowspan=1 colspan=1>8(11.0)</td><td rowspan=1 colspan=1>4(12.9)</td><td rowspan=1 colspan=1>5 (5.4)</td><td rowspan=1 colspan=1>0(0.0)</td><td rowspan=1 colspan=1>5 (6.3)</td><td rowspan=1 colspan=1>6 (9.0)</td><td rowspan=1 colspan=1>2 (4.2)</td><td rowspan=1 colspan=1>2 (6.9)</td><td rowspan=1 colspan=1>10(13.0)</td></tr><tr><td rowspan=1 colspan=1>Unknown</td><td rowspan=1 colspan=1>2 (0.3)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0(0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>2 (3.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td></tr></table>

The performance of the ePlex BCID-GN Panel for organism targets was evaluated in comparison to results from standard laboratory procedures for identification of blood culture isolates. Comparator testing included traditional and automated identification methods, MALDI-TOF, as well as microbiological and biochemical techniques. Identification for specimens with Acinetobacter baumannii (to distinguish between other Acinetobacter species) or Candida parapsilosis (to distinguish between $C .$ . parapsilosis, $C .$ . metapsilosis and C. orthopsilosis) identified by standard laboratory procedures was confirmed using analytically validated PCR assays followed by bi-directional sequencing. For antimicrobial resistance markers, results from the ePlex BCID-GN Panel were compared to analytically validated qPCR amplification assays followed by bi-directional sequencing in specimens with an applicable organism identified by culture.

The comparator methods used in the study for each ePlex BCID-GN Panel target are summarized in Table 56.

Table 56: Comparator method(s) for ePlex BCID-GN Panel targets   

<table><tr><td rowspan=1 colspan=1>Target</td><td rowspan=1 colspan=1>Comparator Method</td></tr><tr><td rowspan=1 colspan=1>Acinetobacter baumannii</td><td rowspan=1 colspan=1>Standard laboratory procedures for organism ID.PCR/sequencing to confirm Acinetobacter baumannii andto differentiate between Acinetobacter species not targetedby the BCID-GN Panel</td></tr><tr><td rowspan=1 colspan=1>Bacteroides fragilis</td><td rowspan=21 colspan=1>Standard laboratory procedures for organism ID.</td></tr><tr><td rowspan=1 colspan=1>Citrobacter</td></tr><tr><td rowspan=1 colspan=1>Cronobacter sakazakii</td></tr><tr><td rowspan=1 colspan=1>Enterobacter cloacae complex</td></tr><tr><td rowspan=1 colspan=1>Enterobacter (non-cloacae complex)</td></tr><tr><td rowspan=1 colspan=1>Escherichia coli</td></tr><tr><td rowspan=1 colspan=1>Fusobacterium necrophorum</td></tr><tr><td rowspan=1 colspan=1>Fusobacterium nucleatum</td></tr><tr><td rowspan=1 colspan=1>Haemophilus influenzae</td></tr><tr><td rowspan=1 colspan=1>Klebsiella oxytoca</td></tr><tr><td rowspan=1 colspan=1>Klebsiella pneumoniae group</td></tr><tr><td rowspan=1 colspan=1>Morganella morganii</td></tr><tr><td rowspan=1 colspan=1>Neisseria meningitidis</td></tr><tr><td rowspan=1 colspan=1>Proteus</td></tr><tr><td rowspan=1 colspan=1>Proteus mirabilis</td></tr><tr><td rowspan=1 colspan=1>Pseudomonas aeruginosa</td></tr><tr><td rowspan=1 colspan=1>Salmonella</td></tr><tr><td rowspan=1 colspan=1>Serratia</td></tr><tr><td rowspan=1 colspan=1>Serratia marcescens</td></tr><tr><td rowspan=1 colspan=1>Stenotrophomonas maltophilia</td></tr><tr><td rowspan=1 colspan=1>Pan Gram-Negative</td></tr><tr><td rowspan=1 colspan=1>Pan Candida</td><td rowspan=1 colspan=1>Standard laboratory procedures for organism ID.PCR/sequencing to confirm C. parapsilosis or identifyC. metapsilosis, C. orthopsilosis.</td></tr><tr><td rowspan=1 colspan=1>CTX-M, IMP, KPC, NDM, OXA, VIM</td><td rowspan=1 colspan=1>qPCR/sequencing in specimens with associated organismdetected by comparator method. See Table 5 for organismassociations.</td></tr></table>

Positive percent agreement (PPA) was calculated by dividing the number of true positive (TP) results by the sum of TP and false negative (FN) results, while negative percent agreement (NPA) was calculated by dividing the number of true negative (TN) results by the sum of TN and false positive (FP) results. A TP result being defined as a specimen where the detected ePlex BCID-GN Panel result matched the identified/detected comparator method result, while a TN result was one where a negative ePlex BCID-GN Panel result matched a negative comparator method result. The two-sided $9 5 \%$ confidence interval was also calculated.

Specimens for which FP and/or FN results (i.e., discordant results) were further investigated using FDA-cleared devices, analytically validated PCR/sequencing assays or 16S sequencing to assess for the presence/absence of the target organism or resistance marker, as applicable. Results from discordant analyses are included as footnotes under the performance tables below.

A total of 777 contrived blood culture specimens were evaluated to supplement the prospective and retrospective specimen cohorts for low prevalence organism and resistance markers. Contrived specimens were prepared by spiking low concentrations of organism into blood culture bottle types appropriate for the organism being evaluated. Each bottle was also inoculated with human whole blood at volumes recommended by the bottle manufacturer. Bottles were incubated on a continuously monitoring blood culture system until flagged as positive. Contrived specimens were removed from the system within eight hours of positivity and stored frozen until the time of testing. Contrived specimens were batched together and tested along with other clinical specimens at five testing sites with all specimen identities blinded to test operators. A summary of the strains used to prepare contrived specimens as well as the numbers of specimens tested per strain are shown in Table 57.

Table 57: Contrived specimen study, organisms evaluated   

<table><tr><td colspan="1" rowspan="1">Target</td><td colspan="1" rowspan="1">Organism</td><td colspan="1" rowspan="1">Strain</td><td colspan="1" rowspan="1">Independent ContrivedSpecimens Tested</td></tr><tr><td colspan="1" rowspan="12">Acinetobacterbaumannii</td><td colspan="1" rowspan="7">Acinetobacter baumannii</td><td colspan="1" rowspan="1">ATCC19606</td><td colspan="1" rowspan="1">2</td></tr><tr><td colspan="1" rowspan="1">ATCCBAA-2093</td><td colspan="1" rowspan="1">4</td></tr><tr><td colspan="1" rowspan="1">ATCCBAA-747</td><td colspan="1" rowspan="1">5</td></tr><tr><td colspan="1" rowspan="1">NCIMB12457</td><td colspan="1" rowspan="1">2</td></tr><tr><td colspan="1" rowspan="1">NCTC13302</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">NCTC13303</td><td colspan="1" rowspan="1">4</td></tr><tr><td colspan="1" rowspan="1">NCTC13423</td><td colspan="1" rowspan="1">17</td></tr><tr><td colspan="1" rowspan="1">Acinetobacter baumannii, NDM</td><td colspan="1" rowspan="1">CDC#0033</td><td colspan="1" rowspan="1">5</td></tr><tr><td colspan="1" rowspan="3">Acinetobacter baumannii, OXA-23</td><td colspan="1" rowspan="1">ATCCBAA-1605</td><td colspan="1" rowspan="1">5</td></tr><tr><td colspan="1" rowspan="1">NCTC13304</td><td colspan="1" rowspan="1">5</td></tr><tr><td colspan="1" rowspan="1">NCTC13305</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">Acinetobacter baumannii Total</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">55</td></tr><tr><td colspan="1" rowspan="6">Bacteroides fragilis</td><td colspan="1" rowspan="5">Bacteroides fragilis</td><td colspan="1" rowspan="1">ATCC23745</td><td colspan="1" rowspan="1">8</td></tr><tr><td colspan="1" rowspan="1">ATCC25285</td><td colspan="1" rowspan="1">8</td></tr><tr><td colspan="1" rowspan="1">ATCC43860</td><td colspan="1" rowspan="1">8</td></tr><tr><td colspan="1" rowspan="1">ATCC700786</td><td colspan="1" rowspan="1">8</td></tr><tr><td colspan="1" rowspan="1">NCTC9343</td><td colspan="1" rowspan="1">8</td></tr><tr><td colspan="2" rowspan="1">Bacteroides fragilis Total</td><td colspan="1" rowspan="1">40</td></tr><tr><td colspan="1" rowspan="5">Citrobacter</td><td colspan="1" rowspan="2">Citrobacter braakii</td><td colspan="1" rowspan="1">ATCC43162</td><td colspan="1" rowspan="1">4</td></tr><tr><td colspan="1" rowspan="1">ATCC51113</td><td colspan="1" rowspan="1">4</td></tr><tr><td colspan="1" rowspan="3">Citrobacter freundii</td><td colspan="1" rowspan="1">ATCC43864</td><td colspan="1" rowspan="1">4</td></tr><tr><td colspan="1" rowspan="1">ATCC8090</td><td colspan="1" rowspan="1">4</td></tr><tr><td colspan="1" rowspan="1">NCTC8581</td><td colspan="1" rowspan="1">4</td></tr><tr><td colspan="1" rowspan="6"></td><td colspan="1" rowspan="1">EY</td><td colspan="1" rowspan="1">NCTC9750</td><td colspan="1" rowspan="1">5</td></tr><tr><td colspan="1" rowspan="1">Citrobacter freundii, CTX-M</td><td colspan="1" rowspan="1">JMI2047</td><td colspan="1" rowspan="1">6</td></tr><tr><td colspan="1" rowspan="1">Citrobacter freundii, KPC</td><td colspan="1" rowspan="1">CDC#0116</td><td colspan="1" rowspan="1">4</td></tr><tr><td colspan="1" rowspan="1">Citrobacter koseri</td><td colspan="1" rowspan="1">ATCC27156</td><td colspan="1" rowspan="1">4</td></tr><tr><td colspan="1" rowspan="1">Citrobacter youngae</td><td colspan="1" rowspan="1">ATCC29935</td><td colspan="1" rowspan="1">4</td></tr><tr><td colspan="1" rowspan="1">Citrobacter Total</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">43</td></tr><tr><td colspan="1" rowspan="14">Cronobacter sakazakii</td><td colspan="1" rowspan="13">Cronobacter sakazakii</td><td colspan="1" rowspan="1">ATCC12868</td><td colspan="1" rowspan="1">2</td></tr><tr><td colspan="1" rowspan="1">ATCC29004</td><td colspan="1" rowspan="1">2</td></tr><tr><td colspan="1" rowspan="1">ATCC29544</td><td colspan="1" rowspan="1">4</td></tr><tr><td colspan="1" rowspan="1">ATCCBAA-894</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">FSLF6-0023</td><td colspan="1" rowspan="1">4</td></tr><tr><td colspan="1" rowspan="1">FSLF6-0028</td><td colspan="1" rowspan="1">4</td></tr><tr><td colspan="1" rowspan="1">FSLF6-0029</td><td colspan="1" rowspan="1">4</td></tr><tr><td colspan="1" rowspan="1">FSLF6-0034</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">FSLF6-0035</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">FSLF6-0043</td><td colspan="1" rowspan="1">4</td></tr><tr><td colspan="1" rowspan="1">FSLF6-0049</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">FSLF6-0050</td><td colspan="1" rowspan="1">4</td></tr><tr><td colspan="1" rowspan="1">FSLF6-0051</td><td colspan="1" rowspan="1">5</td></tr><tr><td colspan="2" rowspan="1">Cronobacter sakazakii Total</td><td colspan="1" rowspan="1">45</td></tr><tr><td colspan="1" rowspan="12">Enterobacter cloacaecomplex</td><td colspan="1" rowspan="4">Enterobacter asburiae</td><td colspan="1" rowspan="1">ATCC35953</td><td colspan="1" rowspan="1">2</td></tr><tr><td colspan="1" rowspan="1">ATCC35955</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">ATCC35956</td><td colspan="1" rowspan="1">4</td></tr><tr><td colspan="1" rowspan="1">ATCC35957</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="2">Enterobacter cloacae, CTX-M</td><td colspan="1" rowspan="1">CDC#0038</td><td colspan="1" rowspan="1">4</td></tr><tr><td colspan="1" rowspan="1">NCTC13464</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">Enterobacter cloacae, CTX-M, KPC</td><td colspan="1" rowspan="1">CDC#0163</td><td colspan="1" rowspan="1">2</td></tr><tr><td colspan="1" rowspan="2">Enterobacter cloacae, CTX-M, NDM</td><td colspan="1" rowspan="1">CDC#0038</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">JMI53571</td><td colspan="1" rowspan="1">12</td></tr><tr><td colspan="1" rowspan="1">Enterobacter cloacae, VIM</td><td colspan="1" rowspan="1">CDC#0154</td><td colspan="1" rowspan="1">6</td></tr><tr><td colspan="1" rowspan="1">Enterobacter hormaechei, KPC</td><td colspan="1" rowspan="1">ATCCBAA-2082</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">Enterobacter cloacae complex Total</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">37</td></tr><tr><td colspan="1" rowspan="9">Enterobacter (non-cloacae complex)</td><td colspan="1" rowspan="3">Enterobacter aerogenes</td><td colspan="1" rowspan="1">ATCC13048</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">ATCC29010</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">ATCC51697</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">Enterobacter aerogenes, IMP</td><td colspan="1" rowspan="1">CDC#0161</td><td colspan="1" rowspan="1">5</td></tr><tr><td colspan="1" rowspan="1">Enterobacter aerogenes, OXA-48</td><td colspan="1" rowspan="1">CDC#0074</td><td colspan="1" rowspan="1">12</td></tr><tr><td colspan="1" rowspan="3">Enterobacter amnigenus</td><td colspan="1" rowspan="1">ATCC33072</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">ATCC33731</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">ATCC51816</td><td colspan="1" rowspan="1">4</td></tr><tr><td colspan="2" rowspan="1">Enterobacter (non-cloacae complex) Total</td><td colspan="1" rowspan="1">36</td></tr><tr><td colspan="1" rowspan="2">Escherichia coli</td><td colspan="1" rowspan="2">Escherichia coli, CTX-M</td><td colspan="1" rowspan="1">CDC#0086</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">NCTC13452</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="12">b</td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="1">NCTC13461</td><td colspan="1" rowspan="1">6</td></tr><tr><td colspan="1" rowspan="1">NCTC13463</td><td colspan="1" rowspan="1">2</td></tr><tr><td colspan="1" rowspan="2">Escherichia coli, CTX-M, NDM</td><td colspan="1" rowspan="1">CDC#0137</td><td colspan="1" rowspan="1">6</td></tr><tr><td colspan="1" rowspan="1">CDC#0151</td><td colspan="1" rowspan="1">2</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli, IMP</td><td colspan="1" rowspan="1">NCTC13476</td><td colspan="1" rowspan="1">7</td></tr><tr><td colspan="1" rowspan="2">Escherichia coli, KPC</td><td colspan="1" rowspan="1">ATCCBAA-2340</td><td colspan="1" rowspan="1">2</td></tr><tr><td colspan="1" rowspan="1">CDC#0114</td><td colspan="1" rowspan="1">4</td></tr><tr><td colspan="1" rowspan="3">Escherichia coli, NDM</td><td colspan="1" rowspan="1">CDC#0118</td><td colspan="1" rowspan="1">6</td></tr><tr><td colspan="1" rowspan="1">CDC#0150</td><td colspan="1" rowspan="1">5</td></tr><tr><td colspan="1" rowspan="1">CDC#0151</td><td colspan="1" rowspan="1">4</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli, VIM</td><td colspan="1" rowspan="1">JMI32465</td><td colspan="1" rowspan="1">2</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli Total</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">52</td></tr><tr><td colspan="1" rowspan="3">Fusobacteriumnecrophorum</td><td colspan="1" rowspan="1">Fusobacterium necrophorum subsp.funduliforme</td><td colspan="1" rowspan="1">ATCC51357</td><td colspan="1" rowspan="1">24</td></tr><tr><td colspan="1" rowspan="1">Fusobacterium necrophorum subsp.necrophorum</td><td colspan="1" rowspan="1">ATCC27852</td><td colspan="1" rowspan="1">24</td></tr><tr><td colspan="1" rowspan="1">Fusobacterium necrophorum Total</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">48</td></tr><tr><td colspan="1" rowspan="4">Fusobacteriumnucleatum</td><td colspan="1" rowspan="3">Fusobacterium nucleatum</td><td colspan="1" rowspan="1">ATCC23726</td><td colspan="1" rowspan="1">8</td></tr><tr><td colspan="1" rowspan="1">ATCC25586</td><td colspan="1" rowspan="1">20</td></tr><tr><td colspan="1" rowspan="1">ATCC31647</td><td colspan="1" rowspan="1">19</td></tr><tr><td colspan="1" rowspan="1">Fusobacterium nucleatum Total</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">47</td></tr><tr><td colspan="1" rowspan="6">Haemophilus influenzae</td><td colspan="1" rowspan="5">Haemophilus influenzae</td><td colspan="1" rowspan="1">ATCC10211</td><td colspan="1" rowspan="1">9</td></tr><tr><td colspan="1" rowspan="1">ATCC43065</td><td colspan="1" rowspan="1">6</td></tr><tr><td colspan="1" rowspan="1">ATCC49144</td><td colspan="1" rowspan="1">9</td></tr><tr><td colspan="1" rowspan="1">NCTC12699</td><td colspan="1" rowspan="1">9</td></tr><tr><td colspan="1" rowspan="1">NCTC8468</td><td colspan="1" rowspan="1">8</td></tr><tr><td colspan="1" rowspan="1">Haemophilus influenzae Total</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">41</td></tr><tr><td colspan="1" rowspan="7">Klebsiella oxytoca</td><td colspan="1" rowspan="6">Klebsiella oxytoca</td><td colspan="1" rowspan="1">ATCC43086</td><td colspan="1" rowspan="1">4</td></tr><tr><td colspan="1" rowspan="1">ATCC43165</td><td colspan="1" rowspan="1">4</td></tr><tr><td colspan="1" rowspan="1">ATCC43863</td><td colspan="1" rowspan="1">2</td></tr><tr><td colspan="1" rowspan="1">ATCC49131</td><td colspan="1" rowspan="1">4</td></tr><tr><td colspan="1" rowspan="1">ATCC51817</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">ATCC700324</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">Klebsiella oxytoca Total</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">20</td></tr><tr><td colspan="1" rowspan="9">Klebsiella pneumoniaegroup</td><td colspan="1" rowspan="1">Klebsiella pneumoniae, CTX-M</td><td colspan="1" rowspan="1">NCTC13465</td><td colspan="1" rowspan="1">5</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae, CTX-M, NDM</td><td colspan="1" rowspan="1">ATCCBAA-2146</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae, CTX-M, OXA</td><td colspan="1" rowspan="1">CDC#0140</td><td colspan="1" rowspan="1">12</td></tr><tr><td colspan="1" rowspan="2">Klebsiella pneumoniae, IMP</td><td colspan="1" rowspan="1">CDC#0034</td><td colspan="1" rowspan="1">8</td></tr><tr><td colspan="1" rowspan="1">CDC#0080</td><td colspan="1" rowspan="1">6</td></tr><tr><td colspan="1" rowspan="4">Klebsiella pneumoniae, KPC</td><td colspan="1" rowspan="1">CDC#0112</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">CDC#0113</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">CDC#0115</td><td colspan="1" rowspan="1">4</td></tr><tr><td colspan="1" rowspan="1">CDC#0117</td><td colspan="1" rowspan="1">4</td></tr><tr><td colspan="1" rowspan="7"></td><td colspan="1" rowspan="3"></td><td colspan="1" rowspan="1">CDC#0120</td><td colspan="1" rowspan="1">4</td></tr><tr><td colspan="1" rowspan="1">CDC#0125</td><td colspan="1" rowspan="1">4</td></tr><tr><td colspan="1" rowspan="1">CDC#0129</td><td colspan="1" rowspan="1">4</td></tr><tr><td colspan="1" rowspan="3">Klebsiella pneumoniae, VIM</td><td colspan="1" rowspan="1">LMC_DR00015</td><td colspan="1" rowspan="1">6</td></tr><tr><td colspan="1" rowspan="1">NCTC13439</td><td colspan="1" rowspan="1">5</td></tr><tr><td colspan="1" rowspan="1">NCTC13440</td><td colspan="1" rowspan="1">5</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae group Total</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">72</td></tr><tr><td colspan="1" rowspan="8">Morganella morganii</td><td colspan="1" rowspan="5">Morganella morganii</td><td colspan="1" rowspan="1">148-200</td><td colspan="1" rowspan="1">8</td></tr><tr><td colspan="1" rowspan="1">148-204</td><td colspan="1" rowspan="1">8</td></tr><tr><td colspan="1" rowspan="1">148-205</td><td colspan="1" rowspan="1">8</td></tr><tr><td colspan="1" rowspan="1">148-206</td><td colspan="1" rowspan="1">9</td></tr><tr><td colspan="1" rowspan="1">148-209</td><td colspan="1" rowspan="1">7</td></tr><tr><td colspan="1" rowspan="1">Morganella morganii, CTX-M1, NDM</td><td colspan="1" rowspan="1">CDC#0057</td><td colspan="1" rowspan="1">5</td></tr><tr><td colspan="1" rowspan="1">Morganella morganii, KPC</td><td colspan="1" rowspan="1">CDC#0133</td><td colspan="1" rowspan="1">4</td></tr><tr><td colspan="1" rowspan="1">Morganella morganii Total</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">49</td></tr><tr><td colspan="1" rowspan="7">Neisseria meningitidis</td><td colspan="1" rowspan="6">Neisseria meningitidis</td><td colspan="1" rowspan="1">ATCC13077</td><td colspan="1" rowspan="1">9</td></tr><tr><td colspan="1" rowspan="1">ATCC13090</td><td colspan="1" rowspan="1">8</td></tr><tr><td colspan="1" rowspan="1">ATCC13102</td><td colspan="1" rowspan="1">8</td></tr><tr><td colspan="1" rowspan="1">ATCC13113</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">ATCC35561</td><td colspan="1" rowspan="1">10</td></tr><tr><td colspan="1" rowspan="1">NCTC10026</td><td colspan="1" rowspan="1">6</td></tr><tr><td colspan="1" rowspan="1">Neisseria meningitidis Total</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">44</td></tr><tr><td colspan="1" rowspan="3">Proteus mirabilis</td><td colspan="1" rowspan="1">Proteus mirabilis, KPC</td><td colspan="1" rowspan="1">CDC#0155</td><td colspan="1" rowspan="1">4</td></tr><tr><td colspan="1" rowspan="1">Proteus mirabilis, NDM</td><td colspan="1" rowspan="1">CDC#0159</td><td colspan="1" rowspan="1">5</td></tr><tr><td colspan="1" rowspan="1">Proteus mirabilis Total</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">9</td></tr><tr><td colspan="1" rowspan="8">Pseudomonasaeruginosa</td><td colspan="1" rowspan="2">Pseudomonas aeruginosa, IMP</td><td colspan="1" rowspan="1">CDC#0092</td><td colspan="1" rowspan="1">5</td></tr><tr><td colspan="1" rowspan="1">CDC#0103</td><td colspan="1" rowspan="1">8</td></tr><tr><td colspan="1" rowspan="1">Pseudomonas aeruginosa, KPC</td><td colspan="1" rowspan="1">CDC#0090</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="4">Pseudomonas aeruginosa, VIM</td><td colspan="1" rowspan="1">CDC#0054</td><td colspan="1" rowspan="1">5</td></tr><tr><td colspan="1" rowspan="1">CDC#0100</td><td colspan="1" rowspan="1">4</td></tr><tr><td colspan="1" rowspan="1">CDC#0108</td><td colspan="1" rowspan="1">4</td></tr><tr><td colspan="1" rowspan="1">NCTC13437</td><td colspan="1" rowspan="1">5</td></tr><tr><td colspan="1" rowspan="1">Pseudomonas aeruginosa Total</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">32</td></tr><tr><td colspan="1" rowspan="10">Salmonella</td><td colspan="1" rowspan="1">Salmonella 4,5,12:I:-</td><td colspan="1" rowspan="1">FSLS5-0580</td><td colspan="1" rowspan="1">2</td></tr><tr><td colspan="1" rowspan="1">Salmonella Heidelberg</td><td colspan="1" rowspan="1">ATCC8326</td><td colspan="1" rowspan="1">2</td></tr><tr><td colspan="1" rowspan="1">Salmonella Infantis</td><td colspan="1" rowspan="1">ATCCBAA-1675</td><td colspan="1" rowspan="1">2</td></tr><tr><td colspan="1" rowspan="1">Salmonella Javiana</td><td colspan="1" rowspan="1">ATCC10721</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">Salmonella Montevideo</td><td colspan="1" rowspan="1">ATCC8387</td><td colspan="1" rowspan="1">8</td></tr><tr><td colspan="1" rowspan="1">Salmonella Muenchen</td><td colspan="1" rowspan="1">ATCC8388</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">Salmonella Newport</td><td colspan="1" rowspan="1">ATCC6962</td><td colspan="1" rowspan="1">6</td></tr><tr><td colspan="1" rowspan="1">Salmonella Typhimurium</td><td colspan="1" rowspan="1">ATCC13311</td><td colspan="1" rowspan="1">7</td></tr><tr><td colspan="1" rowspan="1">Salmonella enterica subspecies entericaEnteritidis (Group D1)</td><td colspan="1" rowspan="1">ATCCBAA-708</td><td colspan="1" rowspan="1">6</td></tr><tr><td colspan="2" rowspan="1">Salmonella Total</td><td colspan="1" rowspan="1">35</td></tr><tr><td colspan="1" rowspan="6">Serratia</td><td colspan="1" rowspan="1">Serratia ficaria</td><td colspan="1" rowspan="1">ATCC33105</td><td colspan="1" rowspan="1">4</td></tr><tr><td colspan="1" rowspan="1">Serratia grimesii</td><td colspan="1" rowspan="1">ATCC14460</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">Serratia plymuthica</td><td colspan="1" rowspan="1">ATCC53858</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="2">Serratia rubidaea</td><td colspan="1" rowspan="1">ATCC27593</td><td colspan="1" rowspan="1">4</td></tr><tr><td colspan="1" rowspan="1">ATCC29025</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="2" rowspan="1">Serratia Total</td><td colspan="1" rowspan="1">17</td></tr><tr><td colspan="1" rowspan="8">Serratia marcescens</td><td colspan="1" rowspan="6">Serratia marcescens</td><td colspan="1" rowspan="1">ATCC13880</td><td colspan="1" rowspan="1">2</td></tr><tr><td colspan="1" rowspan="1">ATCC14041</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">ATCC14756</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">ATCC43861</td><td colspan="1" rowspan="1">4</td></tr><tr><td colspan="1" rowspan="1">ATCC43862</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">ATCC8100</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">Serratia marcescens, IMP</td><td colspan="1" rowspan="1">LMC-DR23105</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">Serratia marcescens Total</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">19</td></tr><tr><td colspan="1" rowspan="6">Stenotrophomonasmaltophilia</td><td colspan="1" rowspan="5">Stenotrophomonas maltophilia</td><td colspan="1" rowspan="1">148-201</td><td colspan="1" rowspan="1">7</td></tr><tr><td colspan="1" rowspan="1">148-222</td><td colspan="1" rowspan="1">6</td></tr><tr><td colspan="1" rowspan="1">148-223</td><td colspan="1" rowspan="1">7</td></tr><tr><td colspan="1" rowspan="1">148-224</td><td colspan="1" rowspan="1">8</td></tr><tr><td colspan="1" rowspan="1">148-225</td><td colspan="1" rowspan="1">8</td></tr><tr><td colspan="2" rowspan="1">Stenotrophomonas maltophilia Total</td><td colspan="1" rowspan="1">36</td></tr></table>

# Clinical performance, organism targets:

Tables 58-78 include a summary of the ePlex BCID-GN Panel performance for each organism target for prospective, retrospective and contrived specimens. Performance is based on comparison of ePlex BCID-GN Panel results to results from comparator methods for prospective and retrospective specimens and based on comparison of ePlex BCID-GN Panel results to the expected result for contrived specimens.

Table 58: Clinical performance, Acinetobacter baumannii   

<table><tr><td rowspan=2 colspan=1>Target</td><td rowspan=2 colspan=1>Specimen Type</td><td rowspan=1 colspan=2>Sensitivity/PPA</td><td rowspan=1 colspan=2>Specificity/NPA</td></tr><tr><td rowspan=1 colspan=1>TP/TP+FN</td><td rowspan=1 colspan=1>% (95% CI)</td><td rowspan=1 colspan=1>TN/TN+FP</td><td rowspan=1 colspan=1>% (95% CI)</td></tr><tr><td rowspan=7 colspan=1>Acinetobacterbaumannii</td><td rowspan=1 colspan=1>Prospective (Fresh)</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>--</td><td rowspan=1 colspan=1>167/167</td><td rowspan=1 colspan=1>100 (97.8-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective (Frozen)</td><td rowspan=1 colspan=1>4/4</td><td rowspan=1 colspan=1>100 (51.0-100)</td><td rowspan=1 colspan=1>178/178</td><td rowspan=1 colspan=1>100 (97.9-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective (All)</td><td rowspan=1 colspan=1>4/4</td><td rowspan=1 colspan=1>100 (51.0-100)</td><td rowspan=1 colspan=1>345/345</td><td rowspan=1 colspan=1>100 (98.9-100)</td></tr><tr><td rowspan=1 colspan=1>Retrospective</td><td rowspan=1 colspan=1>15/15</td><td rowspan=1 colspan=1>100 (79.6-100)</td><td rowspan=1 colspan=1>560/561</td><td rowspan=1 colspan=1>99.8 (99.0-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective / Retrospective</td><td rowspan=1 colspan=1>19/19</td><td rowspan=1 colspan=1>100 (83.2-100)</td><td rowspan=1 colspan=1>905/906A</td><td rowspan=1 colspan=1>99.9 (99.4-100)</td></tr><tr><td rowspan=1 colspan=1>Contrived</td><td rowspan=1 colspan=1>55/55</td><td rowspan=1 colspan=1>100 (93.5-100)</td><td rowspan=1 colspan=1>722/722</td><td rowspan=1 colspan=1>100 (99.5-100)</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>74/74</td><td rowspan=1 colspan=1>100 (95.1-100)</td><td rowspan=1 colspan=1>1627/1628</td><td rowspan=1 colspan=1>99.9 (99.7-100)</td></tr></table>

$\mathrm { C I } =$ Confidence Interval A. Acinetobacter baumanii was detected in the 1/1 false positive specimen using PCR/sequencing.

Table 59: Clinical performance, Bacteroides fragilis   

<table><tr><td rowspan=2 colspan=1>Target</td><td rowspan=2 colspan=1>Specimen Type</td><td rowspan=1 colspan=2>Sensitivity/PPA</td><td rowspan=1 colspan=2>Specificity/NPA</td></tr><tr><td rowspan=1 colspan=1>TP/TP+FN</td><td rowspan=1 colspan=1>% (95% CI)</td><td rowspan=1 colspan=1>TN/TN+FP</td><td rowspan=1 colspan=1>% (95% CI)</td></tr><tr><td rowspan=7 colspan=1>Bacteroides fragilis</td><td rowspan=1 colspan=1>Prospective (Fresh)</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>100 (61.0-100)</td><td rowspan=1 colspan=1>161/161</td><td rowspan=1 colspan=1>100 (97.7-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective (Frozen)</td><td rowspan=1 colspan=1>5/5</td><td rowspan=1 colspan=1>100 (56.6-100)</td><td rowspan=1 colspan=1>177/177</td><td rowspan=1 colspan=1>100 (97.9-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective (All)</td><td rowspan=1 colspan=1>11/11</td><td rowspan=1 colspan=1>100 (74.1-100)</td><td rowspan=1 colspan=1>338/338</td><td rowspan=1 colspan=1>100 (98.9-100)</td></tr><tr><td rowspan=1 colspan=1>Retrospective</td><td rowspan=1 colspan=1>14/17</td><td rowspan=1 colspan=1>82.4 (59.0-93.8)</td><td rowspan=1 colspan=1>558/560</td><td rowspan=1 colspan=1>99.6 (98.7-99.9)</td></tr><tr><td rowspan=1 colspan=1>Prospective / Retrospective</td><td rowspan=1 colspan=1>25/28A</td><td rowspan=1 colspan=1>89.3 (72.8-96.3)</td><td rowspan=1 colspan=1>896/898B</td><td rowspan=1 colspan=1>99.8 (99.2-99.9)</td></tr><tr><td rowspan=1 colspan=1>Contrived</td><td rowspan=1 colspan=1>40/40</td><td rowspan=1 colspan=1>100 (91.2-100)</td><td rowspan=1 colspan=1>737/737</td><td rowspan=1 colspan=1>100 (99.5-100)</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>65/68</td><td rowspan=1 colspan=1>95.6 (87.8-98.5)</td><td rowspan=1 colspan=1>1633/1635</td><td rowspan=1 colspan=1>99.9 (99.6-100)</td></tr></table>

A. B. fragilis was not detected in 2 false negative specimens, but PCR/sequencing instead detected B. caccae and $B$ . thetaiotaomicron, which were not identified by standard laboratory procedures. B. B. fragilis was detected in 2/2 false positive specimens using PCR/sequencing.

Table 60: Clinical performance, Citrobacter   

<table><tr><td rowspan=2 colspan=1>Target</td><td rowspan=2 colspan=1>Specimen Type</td><td rowspan=1 colspan=2>Sensitivity/PPA</td><td rowspan=1 colspan=2>Specificity/NPA</td></tr><tr><td rowspan=1 colspan=1>TP/TP+FN</td><td rowspan=1 colspan=1>% (95% CI)</td><td rowspan=1 colspan=1>TN/TN+FP</td><td rowspan=1 colspan=1>% (95% CI)</td></tr><tr><td rowspan=7 colspan=1>Citrobacter</td><td rowspan=1 colspan=1>Prospective (Fresh)</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>100 (43.9-100)</td><td rowspan=1 colspan=1>163/164</td><td rowspan=1 colspan=1>99.4 (96.6-99.9)</td></tr><tr><td rowspan=1 colspan=1>Prospective (Frozen)</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>100 (34.2-100)</td><td rowspan=1 colspan=1>178/180</td><td rowspan=1 colspan=1>98.9 (96.0-99.7)</td></tr><tr><td rowspan=1 colspan=1>Prospective (All)</td><td rowspan=1 colspan=1>5/5</td><td rowspan=1 colspan=1>100 (56.6-100)</td><td rowspan=1 colspan=1>341/344</td><td rowspan=1 colspan=1>99.1 (97.5-99.7)</td></tr><tr><td rowspan=1 colspan=1>Retrospective</td><td rowspan=1 colspan=1>20/21</td><td rowspan=1 colspan=1>95.2 (77.3-99.2)</td><td rowspan=1 colspan=1>555/556</td><td rowspan=1 colspan=1>99.8 (99.0-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective / Retrospective</td><td rowspan=1 colspan=1>25/26</td><td rowspan=1 colspan=1>96.2 (81.1-99.3)</td><td rowspan=1 colspan=1>896/9004</td><td rowspan=1 colspan=1>99.6 (98.9-99.8)</td></tr><tr><td rowspan=1 colspan=1>Contrived</td><td rowspan=1 colspan=1>43/43</td><td rowspan=1 colspan=1>100 (91.8-100)</td><td rowspan=1 colspan=1>734/734</td><td rowspan=1 colspan=1>100 (99.5-100)</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>68/69</td><td rowspan=1 colspan=1>98.6 (92.2-99.7)</td><td rowspan=1 colspan=1>1630/1634</td><td rowspan=1 colspan=1>99.8 (99.4-99.9)</td></tr></table>

A. Citrobacter braakii (2) and Citrobacter freundii (2) were detected in $4 / 4$ false positive specimens using PCR/sequencing.

Table 61: Clinical performance, Cronobacter sakazakii   

<table><tr><td rowspan=2 colspan=1>Target</td><td rowspan=2 colspan=1>Specimen Type</td><td rowspan=1 colspan=2>Sensitivity/PPA</td><td rowspan=1 colspan=2>Specificity/NPA</td></tr><tr><td rowspan=1 colspan=1>TP/TP+FN</td><td rowspan=1 colspan=1>% (95% CI)</td><td rowspan=1 colspan=1>TN/TN+FP</td><td rowspan=1 colspan=1>% (95% CI)</td></tr><tr><td rowspan=7 colspan=1>Cronobactersakazakii</td><td rowspan=1 colspan=1>Prospective (Fresh)</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>---</td><td rowspan=1 colspan=1>167/167</td><td rowspan=1 colspan=1>100 (97.8-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective (Frozen)</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>--</td><td rowspan=1 colspan=1>182/182</td><td rowspan=1 colspan=1>100 (97.9-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective (All)</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>---</td><td rowspan=1 colspan=1>349/349</td><td rowspan=1 colspan=1>100 (98.9-100)</td></tr><tr><td rowspan=1 colspan=1>Retrospective</td><td rowspan=1 colspan=1>1/1</td><td rowspan=1 colspan=1>100 (20.7-100)</td><td rowspan=1 colspan=1>576/576</td><td rowspan=1 colspan=1>100 (99.3-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective / Retrospective</td><td rowspan=1 colspan=1>1/1</td><td rowspan=1 colspan=1>100 (20.7-100)</td><td rowspan=1 colspan=1>925/925</td><td rowspan=1 colspan=1>100 (99.6-100)</td></tr><tr><td rowspan=1 colspan=1>Contrived</td><td rowspan=1 colspan=1>45/45</td><td rowspan=1 colspan=1>100 (92.1-100)</td><td rowspan=1 colspan=1>732/732</td><td rowspan=1 colspan=1>100 (99.5-100)</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>46/46</td><td rowspan=1 colspan=1>100 (92.3-100)</td><td rowspan=1 colspan=1>1657/1657</td><td rowspan=1 colspan=1>100 (99.8-100)</td></tr></table>

Table 62: Clinical performance, Enterobacter cloacae complex   

<table><tr><td rowspan=2 colspan=1>Target</td><td rowspan=2 colspan=1>Specimen Type</td><td rowspan=1 colspan=2>Sensitivity/PPA</td><td rowspan=1 colspan=2>Specificity/NPA</td></tr><tr><td rowspan=1 colspan=1>TP/TP+FN</td><td rowspan=1 colspan=1>% (95% CI)</td><td rowspan=1 colspan=1>TN/TN+FP</td><td rowspan=1 colspan=1>% (95% CI)</td></tr><tr><td rowspan=7 colspan=1>Enterobactercloacae complex</td><td rowspan=1 colspan=1>Prospective (Fresh)</td><td rowspan=1 colspan=1>12/12</td><td rowspan=1 colspan=1>100 (75.8-100)</td><td rowspan=1 colspan=1>153/155</td><td rowspan=1 colspan=1>98.7 (95.4-99.6)</td></tr><tr><td rowspan=1 colspan=1>Prospective (Frozen)</td><td rowspan=1 colspan=1>7/7</td><td rowspan=1 colspan=1>100 (64.6-100)</td><td rowspan=1 colspan=1>173/175</td><td rowspan=1 colspan=1>98.9 (95.9-99.7)</td></tr><tr><td rowspan=1 colspan=1>Prospective (All)</td><td rowspan=1 colspan=1>19/19</td><td rowspan=1 colspan=1>100 (83.2-100)</td><td rowspan=1 colspan=1>326/330</td><td rowspan=1 colspan=1>98.8 (96.9-99.5)</td></tr><tr><td rowspan=1 colspan=1>Retrospective</td><td rowspan=1 colspan=1>47/50</td><td rowspan=1 colspan=1>94.0 (83.8-97.9)</td><td rowspan=1 colspan=1>526/527</td><td rowspan=1 colspan=1>99.8 (98.9-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective / Retrospective</td><td rowspan=1 colspan=1>66/694</td><td rowspan=1 colspan=1>95.7 (88.0-98.5)</td><td rowspan=1 colspan=1>852/857B</td><td rowspan=1 colspan=1>99.4 (98.6-99.8)</td></tr><tr><td rowspan=1 colspan=1>Contrived</td><td rowspan=1 colspan=1>35/37c</td><td rowspan=1 colspan=1>94.6 (82.3-98.5)</td><td rowspan=1 colspan=1>739/740</td><td rowspan=1 colspan=1>99.9 (99.2-100)</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>101/106</td><td rowspan=1 colspan=1>95.3 (89.4-98.0)</td><td rowspan=1 colspan=1>1591/1597</td><td rowspan=1 colspan=1>99.6 (99.2-99.8)</td></tr></table>

A. A species of the Enterobacter cloacae complex was not detected in 1 false negative specimen, but PCR/sequencing and MALDITOF instead detected $E .$ . coli. Standard laboratory procedures identified $E .$ . cloacae only. B. E. cloacae was detected in 2/5 false positive specimens using PCR/sequencing. C. E. cloacae complex was not detected in 2 specimens containing Enterobacter asburiae.

Table 63: Clinical performance, Enterobacter (non-cloacae complex)   

<table><tr><td rowspan=2 colspan=1>Target</td><td rowspan=2 colspan=1>Specimen Type</td><td rowspan=1 colspan=2>Sensitivity/PPA</td><td rowspan=1 colspan=2>Specificity/NPA</td></tr><tr><td rowspan=1 colspan=1>TP/TP+FN</td><td rowspan=1 colspan=1>% (95% CI)</td><td rowspan=1 colspan=1>TN/TN+FP</td><td rowspan=1 colspan=1>% (95% CI)</td></tr><tr><td rowspan=7 colspan=1>Enterobacter -non-cloacae complex</td><td rowspan=1 colspan=1>Prospective (Fresh)</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>100 (43.9-100)</td><td rowspan=1 colspan=1>163/164</td><td rowspan=1 colspan=1>99.4 (96.6-99.9)</td></tr><tr><td rowspan=1 colspan=1>Prospective (Frozen)</td><td rowspan=1 colspan=1>5/7</td><td rowspan=1 colspan=1>71.4 (35.9-91.8)</td><td rowspan=1 colspan=1>175/175</td><td rowspan=1 colspan=1>100 (97.9-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective (All)</td><td rowspan=1 colspan=1>8/10</td><td rowspan=1 colspan=1>80.0 (49.0-94.3)</td><td rowspan=1 colspan=1>338/339</td><td rowspan=1 colspan=1>99.7 (98.3-99.9)</td></tr><tr><td rowspan=1 colspan=1>Retrospective</td><td rowspan=1 colspan=1>12/12</td><td rowspan=1 colspan=1>100 (75.8-100)</td><td rowspan=1 colspan=1>565/565</td><td rowspan=1 colspan=1>100 (99.3-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective / Retrospective</td><td rowspan=1 colspan=1>20/22A</td><td rowspan=1 colspan=1>90.9 (72.2-97.5)</td><td rowspan=1 colspan=1>903/904B</td><td rowspan=1 colspan=1>99.9 (99.4-100)</td></tr><tr><td rowspan=1 colspan=1>Contrived</td><td rowspan=1 colspan=1>36/36</td><td rowspan=1 colspan=1>100 (90.4-100)</td><td rowspan=1 colspan=1>741/741</td><td rowspan=1 colspan=1>100 (99.5-100)</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>56/58</td><td rowspan=1 colspan=1>96.6 (88.3-99.0)</td><td rowspan=1 colspan=1>1644/1645</td><td rowspan=1 colspan=1>99.9 (99.7-100)</td></tr></table>

A. A species of the Enterobacter non-cloacae complex was not detected in 2 false negative specimens. Standard laboratory procedures identified $E .$ . aerogenes and PCR/sequencing detected $E$ . cloacae.,. B. A species of the Enterobacter non-cloacae complex was not detected in the false positive specimen using PCR/sequencing.

Table 64: Clinical performance, Escherichia coli   

<table><tr><td rowspan=2 colspan=1>Target</td><td rowspan=2 colspan=1>Specimen Type</td><td rowspan=1 colspan=2>Sensitivity/PPA</td><td rowspan=1 colspan=2>Specificity/NPA</td></tr><tr><td rowspan=1 colspan=1>TP/TP+FN</td><td rowspan=1 colspan=1>% (95% CI)</td><td rowspan=1 colspan=1>TN/TN+FP</td><td rowspan=1 colspan=1>% (95% CI)</td></tr><tr><td rowspan=7 colspan=1>Escherichia coli</td><td rowspan=1 colspan=1>Prospective (Fresh)</td><td rowspan=1 colspan=1>59/60</td><td rowspan=1 colspan=1>98.3 (91.1-99.7)</td><td rowspan=1 colspan=1>106/107</td><td rowspan=1 colspan=1>99.1 (94.9-99.8)</td></tr><tr><td rowspan=1 colspan=1>Prospective (Frozen)</td><td rowspan=1 colspan=1>72/73</td><td rowspan=1 colspan=1>98.6 (92.6-99.8)</td><td rowspan=1 colspan=1>109/109</td><td rowspan=1 colspan=1>100 (96.6-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective (All)</td><td rowspan=1 colspan=1>131/133</td><td rowspan=1 colspan=1>98.5 (94.7-99.6)</td><td rowspan=1 colspan=1>215/216</td><td rowspan=1 colspan=1>99.5 (97.4-99.9)</td></tr><tr><td rowspan=1 colspan=1>Retrospective</td><td rowspan=1 colspan=1>132/140</td><td rowspan=1 colspan=1>94.3 (89.1-97.1)</td><td rowspan=1 colspan=1>435/437</td><td rowspan=1 colspan=1>99.5 (98.3-99.9)</td></tr><tr><td rowspan=1 colspan=1>Prospective / Retrospective</td><td rowspan=1 colspan=1>263/273</td><td rowspan=1 colspan=1>96.3 (93.4-98.0)</td><td rowspan=1 colspan=1>650/6534</td><td rowspan=1 colspan=1>99.5 (98.7-99.8)</td></tr><tr><td rowspan=1 colspan=1>Contrived</td><td rowspan=1 colspan=1>52/52</td><td rowspan=1 colspan=1>100 (93.1-100)</td><td rowspan=1 colspan=1>725/725</td><td rowspan=1 colspan=1>100 (99.5-100)</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>315/325</td><td rowspan=1 colspan=1>96.9 (94.4-98.3)</td><td rowspan=1 colspan=1>1375/1378</td><td rowspan=1 colspan=1>99.8 (99.4-99.9)</td></tr></table>

A. E. coli was detected in 3/3 false positive specimens using PCR/sequencing.

Table 65: Clinical performance, Fusobacterium necrophorum   

<table><tr><td rowspan=2 colspan=1>Target</td><td rowspan=2 colspan=1>Specimen Type</td><td rowspan=1 colspan=2>Sensitivity/PPA</td><td rowspan=1 colspan=2>Specificity/NPA</td></tr><tr><td rowspan=1 colspan=1>TP/TP+FN</td><td rowspan=1 colspan=1>% (95% CI)</td><td rowspan=1 colspan=1>TN/TN+FP</td><td rowspan=1 colspan=1>% (95% CI)</td></tr><tr><td rowspan=7 colspan=1>Fusobacteriumnecrophorum</td><td rowspan=1 colspan=1>Prospective (Fresh)</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>---</td><td rowspan=1 colspan=1>167/167</td><td rowspan=1 colspan=1>100 (97.8-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective (Frozen)</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>---</td><td rowspan=1 colspan=1>182/182</td><td rowspan=1 colspan=1>100 (97.9-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective (All)</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>--</td><td rowspan=1 colspan=1>349/349</td><td rowspan=1 colspan=1>100 (98.9-100)</td></tr><tr><td rowspan=1 colspan=1>Retrospective</td><td rowspan=1 colspan=1>1/1</td><td rowspan=1 colspan=1>100 (20.7-100)</td><td rowspan=1 colspan=1>576/576</td><td rowspan=1 colspan=1>100 (99.3-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective / Retrospective</td><td rowspan=1 colspan=1>1/1</td><td rowspan=1 colspan=1>100 (20.7-100)</td><td rowspan=1 colspan=1>925/925</td><td rowspan=1 colspan=1>100 (99.6-100)</td></tr><tr><td rowspan=1 colspan=1>Contrived</td><td rowspan=1 colspan=1>47/48</td><td rowspan=1 colspan=1>97.9 (89.1-99.6)</td><td rowspan=1 colspan=1>729/729</td><td rowspan=1 colspan=1>100 (99.5-100)</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>48/49</td><td rowspan=1 colspan=1>98.0 (89.3-99.6)</td><td rowspan=1 colspan=1>1654/1654</td><td rowspan=1 colspan=1>100 (99.8-100)</td></tr></table>

Table 66: Clinical performance, Fusobacterium nucleatum   

<table><tr><td rowspan=2 colspan=1>Target</td><td rowspan=2 colspan=1>Specimen Type</td><td rowspan=1 colspan=2>Sensitivity/PPA</td><td rowspan=1 colspan=2>Specificity/NPA</td></tr><tr><td rowspan=1 colspan=1>TP/TP+FN</td><td rowspan=1 colspan=1>% (95% CI)</td><td rowspan=1 colspan=1>TN/TN+FP</td><td rowspan=1 colspan=1>% (95% CI)</td></tr><tr><td rowspan=7 colspan=1>Fusobacteriumnucleatum</td><td rowspan=1 colspan=1>Prospective (Fresh)</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>--</td><td rowspan=1 colspan=1>167/167</td><td rowspan=1 colspan=1>100 (97.8-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective (Frozen)</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>--</td><td rowspan=1 colspan=1>182/182</td><td rowspan=1 colspan=1>100 (97.9-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective (All)</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>--</td><td rowspan=1 colspan=1>349/349</td><td rowspan=1 colspan=1>100 (98.9-100)</td></tr><tr><td rowspan=1 colspan=1>Retrospective</td><td rowspan=1 colspan=1>5/5</td><td rowspan=1 colspan=1>100 (56.6-100)</td><td rowspan=1 colspan=1>571/572</td><td rowspan=1 colspan=1>99.8 (99.0-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective / Retrospective</td><td rowspan=1 colspan=1>5/5</td><td rowspan=1 colspan=1>100 (56.6-100)</td><td rowspan=1 colspan=1>920/9214</td><td rowspan=1 colspan=1>99.9 (99.4-100)</td></tr><tr><td rowspan=1 colspan=1>Contrived</td><td rowspan=1 colspan=1>47/47</td><td rowspan=1 colspan=1>100 (92.4-100)</td><td rowspan=1 colspan=1>730/730</td><td rowspan=1 colspan=1>100 (99.5-100)</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>52/52</td><td rowspan=1 colspan=1>100 (93.1-100)</td><td rowspan=1 colspan=1>1650/1651</td><td rowspan=1 colspan=1>99.9 (99.7-100)</td></tr></table>

A. F. nucleatum was detected in the 1/1 false positive specimen using PCR/sequencing.

Table 67: Clinical performance, Haemophilus influenzae   

<table><tr><td rowspan=2 colspan=1>Target</td><td rowspan=2 colspan=1>Specimen Type</td><td rowspan=1 colspan=2>Sensitivity/PPA</td><td rowspan=1 colspan=2>Specificity/NPA</td></tr><tr><td rowspan=1 colspan=1>TP/TP+FN</td><td rowspan=1 colspan=1>% (95% CI)</td><td rowspan=1 colspan=1>TN/TN+FP</td><td rowspan=1 colspan=1>% (95% CI)</td></tr><tr><td rowspan=7 colspan=1>Haemophilusinfluenzae</td><td rowspan=1 colspan=1>Prospective (Fresh)</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>100 (43.9-100)</td><td rowspan=1 colspan=1>164/164</td><td rowspan=1 colspan=1>100 (97.7-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective (Frozen)</td><td rowspan=1 colspan=1>4/4</td><td rowspan=1 colspan=1>100 (51.0-100)</td><td rowspan=1 colspan=1>178/178</td><td rowspan=1 colspan=1>100 (97.9-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective (All)</td><td rowspan=1 colspan=1>7/7</td><td rowspan=1 colspan=1>100 (64.6-100)</td><td rowspan=1 colspan=1>342/342</td><td rowspan=1 colspan=1>100 (98.9-100)</td></tr><tr><td rowspan=1 colspan=1>Retrospective</td><td rowspan=1 colspan=1>7/7</td><td rowspan=1 colspan=1>100 (64.6-100)</td><td rowspan=1 colspan=1>570/570</td><td rowspan=1 colspan=1>100 (99.3-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective / Retrospective</td><td rowspan=1 colspan=1>14/14</td><td rowspan=1 colspan=1>100 (78.5-100)</td><td rowspan=1 colspan=1>912/912</td><td rowspan=1 colspan=1>100 (99.6-100)</td></tr><tr><td rowspan=1 colspan=1>Contrived</td><td rowspan=1 colspan=1>41/41</td><td rowspan=1 colspan=1>100 (91.4-100)</td><td rowspan=1 colspan=1>736/736</td><td rowspan=1 colspan=1>100 (99.5-100)</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>55/55</td><td rowspan=1 colspan=1>100 (93.5-100)</td><td rowspan=1 colspan=1>1648/1648</td><td rowspan=1 colspan=1>100 (99.8-100)</td></tr></table>

Table 68: Clinical performance, Klebsiella oxytoca   

<table><tr><td rowspan=2 colspan=1>Target</td><td rowspan=2 colspan=1>Specimen Type</td><td rowspan=1 colspan=2>Sensitivity/PPA</td><td rowspan=1 colspan=2>Specificity/NPA</td></tr><tr><td rowspan=1 colspan=1>TP/TP+FN</td><td rowspan=1 colspan=1>% (95% CI)</td><td rowspan=1 colspan=1>TN/TN+FP</td><td rowspan=1 colspan=1>% (95% CI)</td></tr><tr><td rowspan=7 colspan=1>Klebsiella oxytoca</td><td rowspan=1 colspan=1>Prospective (Fresh)</td><td rowspan=1 colspan=1>4/6</td><td rowspan=1 colspan=1>66.7 (30.0-90.3)</td><td rowspan=1 colspan=1>160/161</td><td rowspan=1 colspan=1>99.4 (96.6-99.9)</td></tr><tr><td rowspan=1 colspan=1>Prospective (Frozen)</td><td rowspan=1 colspan=1>7/7</td><td rowspan=1 colspan=1>100 (64.6-100)</td><td rowspan=1 colspan=1>175/175</td><td rowspan=1 colspan=1>100 (97.9-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective (All)</td><td rowspan=1 colspan=1>11/13</td><td rowspan=1 colspan=1>84.6 (57.8-95.7)</td><td rowspan=1 colspan=1>335/336</td><td rowspan=1 colspan=1>99.7 (98.3-99.9)</td></tr><tr><td rowspan=1 colspan=1>Retrospective</td><td rowspan=1 colspan=1>29/34</td><td rowspan=1 colspan=1>85.3 (69.9-93.6)</td><td rowspan=1 colspan=1>541/543</td><td rowspan=1 colspan=1>99.6 (98.7-99.9)</td></tr><tr><td rowspan=1 colspan=1>Prospective / Retrospective</td><td rowspan=1 colspan=1>40/47A</td><td rowspan=1 colspan=1>85.1 (72.3-92.6)</td><td rowspan=1 colspan=1>876/879B</td><td rowspan=1 colspan=1>99.7 (99.0-99.9)</td></tr><tr><td rowspan=1 colspan=1>Contrived</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>100 (83.9-100)</td><td rowspan=1 colspan=1>757/757</td><td rowspan=1 colspan=1>100 (99.5-100)</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>60/67</td><td rowspan=1 colspan=1>89.6 (80.0-94.8)</td><td rowspan=1 colspan=1>1633/1636</td><td rowspan=1 colspan=1>99.8 (99.5-99.9)</td></tr></table>

A. K. oxytoca was not detected in 2 false negative specimens using PCR/sequencing, but 16S sequencing instead detected Raoultella ornithinolytica and Raoultella planticola, which were not identified by standard laboratory procedures. B. Klebsiella oxytoca was detected in 3/3 false positive specimens using PCR/sequencing.

Table 69: Clinical performance, Klebsiella pneumoniae group   

<table><tr><td rowspan=2 colspan=1>Target</td><td rowspan=2 colspan=1>Specimen Type</td><td rowspan=1 colspan=2>Sensitivity/PPA</td><td rowspan=1 colspan=2>Specificity/NPA</td></tr><tr><td rowspan=1 colspan=1>TP/TP+FN</td><td rowspan=1 colspan=1>% (95% CI)</td><td rowspan=1 colspan=1>TN/TN+FP</td><td rowspan=1 colspan=1>% (95% CI)</td></tr><tr><td rowspan=7 colspan=1>Klebsiellapneumoniae group</td><td rowspan=1 colspan=1>Prospective (Fresh)</td><td rowspan=1 colspan=1>29/30</td><td rowspan=1 colspan=1>96.7 (83.3-99.4)</td><td rowspan=1 colspan=1>136/137</td><td rowspan=1 colspan=1>99.3 (96.0-99.9)</td></tr><tr><td rowspan=1 colspan=1>Prospective (Frozen)</td><td rowspan=1 colspan=1>29/31</td><td rowspan=1 colspan=1>93.5 (79.3-98.2)</td><td rowspan=1 colspan=1>151/151</td><td rowspan=1 colspan=1>100 (97.5-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective (All)</td><td rowspan=1 colspan=1>58/61</td><td rowspan=1 colspan=1>95.1 (86.5-98.3)</td><td rowspan=1 colspan=1>287/288</td><td rowspan=1 colspan=1>99.7 (98.1-99.9)</td></tr><tr><td rowspan=1 colspan=1>Retrospective</td><td rowspan=1 colspan=1>106/108</td><td rowspan=1 colspan=1>98.1 (93.5-99.5)</td><td rowspan=1 colspan=1>466/469</td><td rowspan=1 colspan=1>99.4 (98.1-99.8)</td></tr><tr><td rowspan=1 colspan=1>Prospective / Retrospective</td><td rowspan=1 colspan=1>164/169A</td><td rowspan=1 colspan=1>97.0 (93.3-98.7)</td><td rowspan=1 colspan=1>753/757B</td><td rowspan=1 colspan=1>99.5 (98.6-99.8)</td></tr><tr><td rowspan=1 colspan=1>Contrived</td><td rowspan=1 colspan=1>72/72</td><td rowspan=1 colspan=1>100 (94.9-100)</td><td rowspan=1 colspan=1>705/705</td><td rowspan=1 colspan=1>100 (99.5-100)</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>236/241</td><td rowspan=1 colspan=1>97.9 (95.2-99.1)</td><td rowspan=1 colspan=1>1458/1462</td><td rowspan=1 colspan=1>99.7 (99.3-99.9)</td></tr></table>

A. K. pneumoniae was not detected in 1 false negative specimen, but PCR/sequencing and MALDI-TOF instead detected $K$ . oxytoca, which was not identified by standard laboratory procedures. B. K. pneumoniae was detected in 4/4 false positive specimens using PCR/sequencing.

Table 70: Clinical performance, Morganella morganii   

<table><tr><td rowspan=2 colspan=1>Target</td><td rowspan=2 colspan=1>Specimen Type</td><td rowspan=1 colspan=2>Sensitivity/PPA</td><td rowspan=1 colspan=2>Specificity/NPA</td></tr><tr><td rowspan=1 colspan=1>TP/TP+FN</td><td rowspan=1 colspan=1>% (95% CI)</td><td rowspan=1 colspan=1>TN/TN+FP</td><td rowspan=1 colspan=1>% (95% CI)</td></tr><tr><td rowspan=7 colspan=1>Morganellamorganii</td><td rowspan=1 colspan=1>Prospective (Fresh)</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>100 (43.9-100)</td><td rowspan=1 colspan=1>164/164</td><td rowspan=1 colspan=1>100 (97.7-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective (Frozen)</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>---</td><td rowspan=1 colspan=1>182/182</td><td rowspan=1 colspan=1>100 (97.9-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective (All)</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>100 (43.9-100)</td><td rowspan=1 colspan=1>346/346</td><td rowspan=1 colspan=1>100 (98.9-100)</td></tr><tr><td rowspan=1 colspan=1>Retrospective</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>100 (72.2-100)</td><td rowspan=1 colspan=1>566/567</td><td rowspan=1 colspan=1>99.8 (99.0-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective / Retrospective</td><td rowspan=1 colspan=1>13/13</td><td rowspan=1 colspan=1>100 (77.2-100)</td><td rowspan=1 colspan=1>912/913A</td><td rowspan=1 colspan=1>99.9 (99.4-100)</td></tr><tr><td rowspan=1 colspan=1>Contrived</td><td rowspan=1 colspan=1>49/49</td><td rowspan=1 colspan=1>100 (92.7-100)</td><td rowspan=1 colspan=1>728/728</td><td rowspan=1 colspan=1>100 (99.5-100)</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>62/62</td><td rowspan=1 colspan=1>100 (94.2-100)</td><td rowspan=1 colspan=1>1640/1641</td><td rowspan=1 colspan=1>99.9 (99.7-100)</td></tr></table>

A. M. morganii was detected in 1/1 false positive clinical specimens using PCR/sequencing.

Table 71: Clinical performance, Neisseria meningitidis   

<table><tr><td rowspan=2 colspan=1>Target</td><td rowspan=2 colspan=1>Specimen Type</td><td rowspan=1 colspan=2>Sensitivity/PPA</td><td rowspan=1 colspan=2>Specificity/NPA</td></tr><tr><td rowspan=1 colspan=1>TP/TP+FN</td><td rowspan=1 colspan=1>% (95% CI)</td><td rowspan=1 colspan=1>TN/TN+FP</td><td rowspan=1 colspan=1>% (95% CI)</td></tr><tr><td rowspan=7 colspan=1>Neisseria meningitidis</td><td rowspan=1 colspan=1>Prospective (Fresh)</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>167/167</td><td rowspan=1 colspan=1>100 (97.8-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective (Frozen)</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>---</td><td rowspan=1 colspan=1>182/182</td><td rowspan=1 colspan=1>100 (97.9-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective (All)</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>--</td><td rowspan=1 colspan=1>349/349</td><td rowspan=1 colspan=1>100 (98.9-100)</td></tr><tr><td rowspan=1 colspan=1>Retrospective</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>576/577</td><td rowspan=1 colspan=1>99.8 (99.0-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective / Retrospective</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>925/926A</td><td rowspan=1 colspan=1>99.9 (99.4-100)</td></tr><tr><td rowspan=1 colspan=1>Contrived</td><td rowspan=1 colspan=1>44/44</td><td rowspan=1 colspan=1>100 (92.0-100)</td><td rowspan=1 colspan=1>733/733</td><td rowspan=1 colspan=1>100 (99.5-100)</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>44/44</td><td rowspan=1 colspan=1>100 (92.0-100)</td><td rowspan=1 colspan=1>1658/1659</td><td rowspan=1 colspan=1>99.9 (99.7-100)</td></tr></table>

A. N. meningitidis was not detected in the false positive specimen using PCR/sequencing.

Table 72: Clinical performance, Proteus   

<table><tr><td rowspan=2 colspan=1>Target</td><td rowspan=2 colspan=1>Specimen Type</td><td rowspan=1 colspan=2>Sensitivity/PPA</td><td rowspan=1 colspan=2>Specificity/NPA</td></tr><tr><td rowspan=1 colspan=1>TP/TP+FN</td><td rowspan=1 colspan=1>% (95% CI)</td><td rowspan=1 colspan=1>TN/TN+FP</td><td rowspan=1 colspan=1>% (95% CI)</td></tr><tr><td rowspan=7 colspan=1>Proteus</td><td rowspan=1 colspan=1>Prospective (Fresh)</td><td rowspan=1 colspan=1>7/8</td><td rowspan=1 colspan=1>87.5 (52.9-97.8)</td><td rowspan=1 colspan=1>159/159</td><td rowspan=1 colspan=1>100 (97.6-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective (Frozen)</td><td rowspan=1 colspan=1>15/15</td><td rowspan=1 colspan=1>100 (79.6-100)</td><td rowspan=1 colspan=1>167/167</td><td rowspan=1 colspan=1>100 (97.8-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective (All)</td><td rowspan=1 colspan=1>22/23</td><td rowspan=1 colspan=1>95.7 (79.0-99.2)</td><td rowspan=1 colspan=1>326/326</td><td rowspan=1 colspan=1>100 (98.8-100)</td></tr><tr><td rowspan=1 colspan=1>Retrospective</td><td rowspan=1 colspan=1>54/55</td><td rowspan=1 colspan=1>98.2 (90.4-99.7)</td><td rowspan=1 colspan=1>522/522</td><td rowspan=1 colspan=1>100 (99.3-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective / Retrospective</td><td rowspan=1 colspan=1>76/78</td><td rowspan=1 colspan=1>97.4 (91.1-99.3)</td><td rowspan=1 colspan=1>848/848</td><td rowspan=1 colspan=1>100 (99.5-100)</td></tr><tr><td rowspan=1 colspan=1>Contrived</td><td rowspan=1 colspan=1>9/9</td><td rowspan=1 colspan=1>100 (70.1-100)</td><td rowspan=1 colspan=1>768/768</td><td rowspan=1 colspan=1>100 (99.5-100)</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>85/87</td><td rowspan=1 colspan=1>97.7 (92.0-99.4)</td><td rowspan=1 colspan=1>1616/1616</td><td rowspan=1 colspan=1>100 (99.8-100)</td></tr></table>

Table 73: Clinical performance, Proteus mirabilis   

<table><tr><td rowspan=2 colspan=1>Target</td><td rowspan=2 colspan=1>Specimen Type</td><td rowspan=1 colspan=2>Sensitivity/PPA</td><td rowspan=1 colspan=2>Specificity/NPA</td></tr><tr><td rowspan=1 colspan=1>TP/TP+FN</td><td rowspan=1 colspan=1>% (95% CI)</td><td rowspan=1 colspan=1>TN/TN+FP</td><td rowspan=1 colspan=1>% (95% CI)</td></tr><tr><td rowspan=7 colspan=1>Proteus mirabilis</td><td rowspan=1 colspan=1>Prospective (Fresh)</td><td rowspan=1 colspan=1>7/8</td><td rowspan=1 colspan=1>87.5 (52.9-97.8)</td><td rowspan=1 colspan=1>159/159</td><td rowspan=1 colspan=1>100 (97.6-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective (Frozen)</td><td rowspan=1 colspan=1>15/15</td><td rowspan=1 colspan=1>100 (79.6-100)</td><td rowspan=1 colspan=1>167/167</td><td rowspan=1 colspan=1>100 (97.8-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective (All)</td><td rowspan=1 colspan=1>22/23</td><td rowspan=1 colspan=1>95.7 (79.0-99.2)</td><td rowspan=1 colspan=1>326/326</td><td rowspan=1 colspan=1>100 (98.8-100)</td></tr><tr><td rowspan=1 colspan=1>Retrospective</td><td rowspan=1 colspan=1>50/51</td><td rowspan=1 colspan=1>98.0 (89.7-99.7)</td><td rowspan=1 colspan=1>526/526</td><td rowspan=1 colspan=1>100 (99.3-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective / Retrospective</td><td rowspan=1 colspan=1>72/74</td><td rowspan=1 colspan=1>97.3 (90.7-99.3)</td><td rowspan=1 colspan=1>852/852</td><td rowspan=1 colspan=1>100 (99.6-100)</td></tr><tr><td rowspan=1 colspan=1>Contrived</td><td rowspan=1 colspan=1>9/9</td><td rowspan=1 colspan=1>100 (70.1-100)</td><td rowspan=1 colspan=1>768/768</td><td rowspan=1 colspan=1>100 (99.5-100)</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>81/83</td><td rowspan=1 colspan=1>97.6 (91.6-99.3)</td><td rowspan=1 colspan=1>1620/1620</td><td rowspan=1 colspan=1>100 (99.8-100)</td></tr></table>

Table 74: Clinical performance, Pseudomonas aeruginosa   

<table><tr><td rowspan=2 colspan=1>Target</td><td rowspan=2 colspan=1>Specimen Type</td><td rowspan=1 colspan=2>Sensitivity/PPA</td><td rowspan=1 colspan=2>Specificity/NPA</td></tr><tr><td rowspan=1 colspan=1>TP/TP+FN</td><td rowspan=1 colspan=1>% (95% CI)</td><td rowspan=1 colspan=1>TN/TN+FP</td><td rowspan=1 colspan=1>% (95% CI)</td></tr><tr><td rowspan=7 colspan=1>Pseudomonasaeruginosa</td><td rowspan=1 colspan=1>Prospective (Fresh)</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>100 (72.2-100)</td><td rowspan=1 colspan=1>157/157</td><td rowspan=1 colspan=1>100 (97.6-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective (Frozen)</td><td rowspan=1 colspan=1>17/18</td><td rowspan=1 colspan=1>94.4 (74.2-99.0)</td><td rowspan=1 colspan=1>163/164</td><td rowspan=1 colspan=1>99.4 (96.6-99.9)</td></tr><tr><td rowspan=1 colspan=1>Prospective (All)</td><td rowspan=1 colspan=1>27/28</td><td rowspan=1 colspan=1>96.4 (82.3-99.4)</td><td rowspan=1 colspan=1>320/321</td><td rowspan=1 colspan=1>99.7 (98.3-99.9)</td></tr><tr><td rowspan=1 colspan=1>Retrospective</td><td rowspan=1 colspan=1>56/60</td><td rowspan=1 colspan=1>93.3 (84.1-97.4)</td><td rowspan=1 colspan=1>514/517</td><td rowspan=1 colspan=1>99.4 (98.3-99.8)</td></tr><tr><td rowspan=1 colspan=1>Prospective / Retrospective</td><td rowspan=1 colspan=1>83/88</td><td rowspan=1 colspan=1>94.3 (87.4-97.5)</td><td rowspan=1 colspan=1>834/8384</td><td rowspan=1 colspan=1>99.5 (98.8-99.8)</td></tr><tr><td rowspan=1 colspan=1>Contrived</td><td rowspan=1 colspan=1>32/32</td><td rowspan=1 colspan=1>100 (89.3-100)</td><td rowspan=1 colspan=1>745/745</td><td rowspan=1 colspan=1>100 (99.5-100)</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>115/120</td><td rowspan=1 colspan=1>95.8 (90.6-98.2)</td><td rowspan=1 colspan=1>1579/1583</td><td rowspan=1 colspan=1>99.7 (99.4-99.9)</td></tr></table>

A. P. aeruginosa was detected in 2/4 false positive specimens using PCR/sequencing.

Table 75: Clinical performance, Salmonella   

<table><tr><td rowspan=2 colspan=2>Target</td><td rowspan=2 colspan=1>Specimen Type</td><td rowspan=1 colspan=2>Sensitivity/PPA</td><td rowspan=1 colspan=2>Specificity/NPA</td></tr><tr><td rowspan=1 colspan=1>TP/TP+FN</td><td rowspan=1 colspan=1>% (95% CI)</td><td rowspan=1 colspan=1>TN/TN+FP</td><td rowspan=1 colspan=1>% (95% CI)</td></tr><tr><td rowspan=7 colspan=2>Salmonella</td><td rowspan=1 colspan=1>Prospective (Fresh)</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>100 (34.2-100)</td><td rowspan=1 colspan=1>165/165</td><td rowspan=1 colspan=1>100 (97.7-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective (Frozen)</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>---</td><td rowspan=1 colspan=1>182/182</td><td rowspan=1 colspan=1>100 (97.9-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective (All)</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>100 (34.2-100)</td><td rowspan=1 colspan=1>347/347</td><td rowspan=1 colspan=1>100 (98.9-100)</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Retrospective</td><td rowspan=1 colspan=1>18/19</td><td rowspan=1 colspan=1>94.7 (75.4-99.1)</td><td rowspan=1 colspan=1>558/558</td><td rowspan=1 colspan=1>100 (99.3-100)</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Prospective / Retrospective</td><td rowspan=1 colspan=1>20/21A</td><td rowspan=1 colspan=1>95.2 (77.3-99.2)</td><td rowspan=1 colspan=1>905/905</td><td rowspan=1 colspan=1>100 (99.6-100)</td></tr><tr><td rowspan=1 colspan=1>Contrived</td><td rowspan=1 colspan=1>34/35</td><td rowspan=1 colspan=1>97.1 (85.5-99.5)</td><td rowspan=1 colspan=1>742/742</td><td rowspan=1 colspan=1>100 (99.5-100)</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>54/56</td><td rowspan=1 colspan=1>96.4 (87.9-99.0)</td><td rowspan=1 colspan=1>1647/1647</td><td rowspan=1 colspan=1>100 (99.8-100)</td></tr></table>

A. Salmonella was not detected in 1 false negative specimen, but PCR/sequencing instead detected E. coli, which was not identified by standard laboratory procedures.

Table 76: Clinical performance, Serratia   

<table><tr><td rowspan=2 colspan=1>Target</td><td rowspan=2 colspan=1>Specimen Type</td><td rowspan=1 colspan=2>Sensitivity/PPA</td><td rowspan=1 colspan=2>Specificity/NPA</td></tr><tr><td rowspan=1 colspan=1>TP/TP+FN</td><td rowspan=1 colspan=1>% (95% CI)</td><td rowspan=1 colspan=1>TN/TN+FP</td><td rowspan=1 colspan=1>% (95% CI)</td></tr><tr><td rowspan=7 colspan=1>Serratia</td><td rowspan=1 colspan=1>Prospective (Fresh)</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>100 (61.0-100)</td><td rowspan=1 colspan=1>161/161</td><td rowspan=1 colspan=1>100 (97.7-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective (Frozen)</td><td rowspan=1 colspan=1>4/4</td><td rowspan=1 colspan=1>100 (51.0-100)</td><td rowspan=1 colspan=1>178/178</td><td rowspan=1 colspan=1>100 (97.9-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective (All)</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>100 (72.2-100)</td><td rowspan=1 colspan=1>339/339</td><td rowspan=1 colspan=1>100 (98.9-100)</td></tr><tr><td rowspan=1 colspan=1>Retrospective</td><td rowspan=1 colspan=1>34/34</td><td rowspan=1 colspan=1>100 (89.8-100)</td><td rowspan=1 colspan=1>542/543</td><td rowspan=1 colspan=1>99.8 (99.0-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective / Retrospective</td><td rowspan=1 colspan=1>44/44</td><td rowspan=1 colspan=1>100 (92.0-100)</td><td rowspan=1 colspan=1>881/882A</td><td rowspan=1 colspan=1>99.9 (99.4-100)</td></tr><tr><td rowspan=1 colspan=1>Contrived</td><td rowspan=1 colspan=1>36/36</td><td rowspan=1 colspan=1>100 (90.4-100)</td><td rowspan=1 colspan=1>741/741</td><td rowspan=1 colspan=1>100 (99.5-100)</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>80/80</td><td rowspan=1 colspan=1>100 (95.4-100)</td><td rowspan=1 colspan=1>1622/1623</td><td rowspan=1 colspan=1>99.9 (99.7-100)</td></tr></table>

A. S. marcescens was detected in the $1 / 1$ false positive specimen using PCR/sequencing.

Table 77: Clinical performance, Serratia marcescens   

<table><tr><td rowspan=2 colspan=1>Target</td><td rowspan=2 colspan=1>Specimen Type</td><td rowspan=1 colspan=2>Sensitivity/PPA</td><td rowspan=1 colspan=2>Specificity/NPA</td></tr><tr><td rowspan=1 colspan=1>TP/TP+FN</td><td rowspan=1 colspan=1>% (95% CI)</td><td rowspan=1 colspan=1>TN/TN+FP</td><td rowspan=1 colspan=1>% (95% CI)</td></tr><tr><td rowspan=7 colspan=1>Serratia marcescens</td><td rowspan=1 colspan=1>Prospective (Fresh)</td><td rowspan=1 colspan=1>5/5</td><td rowspan=1 colspan=1>100 (56.6-100)</td><td rowspan=1 colspan=1>162/162</td><td rowspan=1 colspan=1>100 (97.7-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective (Frozen)</td><td rowspan=1 colspan=1>4/4</td><td rowspan=1 colspan=1>100 (51.0-100)</td><td rowspan=1 colspan=1>178/178</td><td rowspan=1 colspan=1>100 (97.9-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective (All)</td><td rowspan=1 colspan=1>9/9</td><td rowspan=1 colspan=1>100 (70.1-100)</td><td rowspan=1 colspan=1>340/340</td><td rowspan=1 colspan=1>100 (98.9-100)</td></tr><tr><td rowspan=1 colspan=1>Retrospective</td><td rowspan=1 colspan=1>34/34</td><td rowspan=1 colspan=1>100 (89.8-100)</td><td rowspan=1 colspan=1>542/543</td><td rowspan=1 colspan=1>99.8 (99.0-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective / Retrospective</td><td rowspan=1 colspan=1>43/43</td><td rowspan=1 colspan=1>100 (91.8-100)</td><td rowspan=1 colspan=1>882/8834</td><td rowspan=1 colspan=1>99.9 (99.4-100)</td></tr><tr><td rowspan=1 colspan=1>Contrived</td><td rowspan=1 colspan=1>19/19</td><td rowspan=1 colspan=1>100 (83.2-100)</td><td rowspan=1 colspan=1>758/758</td><td rowspan=1 colspan=1>100 (99.5-100)</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>62/62</td><td rowspan=1 colspan=1>100 (94.2-100)</td><td rowspan=1 colspan=1>1640/1641</td><td rowspan=1 colspan=1>99.9 (99.7-100)</td></tr></table>

A. S. marcescens was detected in the 1/1 false positive specimen using PCR/sequencing.

Table 78: Clinical performance, Stenotrophomonas maltophilia   

<table><tr><td rowspan=2 colspan=1>Target</td><td rowspan=2 colspan=1>Specimen Type</td><td rowspan=1 colspan=2>Sensitivity/PPA</td><td rowspan=1 colspan=2>Specificity/NPA</td></tr><tr><td rowspan=1 colspan=1>TP/TP+FN</td><td rowspan=1 colspan=1>% (95% CI)</td><td rowspan=1 colspan=1>TN/TN+FP</td><td rowspan=1 colspan=1>% (95% CI)</td></tr><tr><td rowspan=7 colspan=1>Stenotrophomonasmaltophilia</td><td rowspan=1 colspan=1>Prospective (Fresh)</td><td rowspan=1 colspan=1>2/3</td><td rowspan=1 colspan=1>66.7 (20.8-93.9)</td><td rowspan=1 colspan=1>164/164</td><td rowspan=1 colspan=1>100 (97.7-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective (Frozen)</td><td rowspan=1 colspan=1>1/1</td><td rowspan=1 colspan=1>100 (20.7-100)</td><td rowspan=1 colspan=1>181/181</td><td rowspan=1 colspan=1>100 (97.9-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective (All)</td><td rowspan=1 colspan=1>3/4</td><td rowspan=1 colspan=1>75.0 (30.1-95.4)</td><td rowspan=1 colspan=1>345/345</td><td rowspan=1 colspan=1>100 (98.9-100)</td></tr><tr><td rowspan=1 colspan=1>Retrospective</td><td rowspan=1 colspan=1>8/10</td><td rowspan=1 colspan=1>80.0 (49.0-94.3)</td><td rowspan=1 colspan=1>566/567</td><td rowspan=1 colspan=1>99.8 (99.0-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective / Retrospective</td><td rowspan=1 colspan=1>11/14</td><td rowspan=1 colspan=1>78.6 (52.4-92.4)</td><td rowspan=1 colspan=1>911/912A</td><td rowspan=1 colspan=1>99.9 (99.4-100)</td></tr><tr><td rowspan=1 colspan=1>Contrived</td><td rowspan=1 colspan=1>36/36</td><td rowspan=1 colspan=1>100 (90.4-100)</td><td rowspan=1 colspan=1>741/741</td><td rowspan=1 colspan=1>100 (99.5-100)</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>47/50</td><td rowspan=1 colspan=1>94.0 (83.8-97.9)</td><td rowspan=1 colspan=1>1652/1653</td><td rowspan=1 colspan=1>99.9 (99.7-100)</td></tr></table>

A. S. maltophilia was detected in the 1/1 false positive specimen using PCR/sequencing.

Performance of the ePlex BCID-GN Panel for the Pan Candida and Pan Gram-Positive assay targets is summarized in Table 79 below. In addition to the evaluable prospective and retrospective specimens that contained gram-negative organisms by Gram stain (Intended use population), the clinical performance of the Pan Candida and Pan Gram-Positive targets was evaluated with an additional 741 non-intended use retrospective specimens that contained gram-positive or fungal organisms by Gram stain; these specimens are denoted as retrospective (non-intended use) specimens and were used to supplement the prospective intended use specimens.

Pan Candida target performance: For the intended use population (specimens with gramnegative organisms observed by Gram stain), there were eight specimens with targeted Candida species identified by the comparator method. All eight specimens were mixed cultures, including three specimens with false negative results for the ePlex BCID-GN Pan Candida assay.

It is noted that the Pan Candida assay is designed to detect only four Candida species (Candida albicans, C. glabrata, C. parapsilosis, and C. krusei). In the clinical study there were six specimens for which non-targeted Candida species were identified either by

comparator testing or PCR/sequencing performed during discordant analysis. The nontargeted Candida species identified in these six specimens included C. dubliniensis (1), C. lusitaniae (3), and $C .$ . tropicalis (2). The ePlex BCID-GN Panel generated the expected result for each of these specimens.

Pan Gram-Positive performance: For the intended use population (specimens with gramnegative bacteria observed by Gram stain), there were a total of 17 false negative specimens for the Pan Gram-positive assay target (i.e., PPA for prospective specimens was $7 3 . 9 \%$ (17/23); PPA for retrospective specimens was $80 \%$ (44/55)). For the six false negative prospective specimens, two specimens were determined to have Paenibacillus species based on additional testing of discrepant specimens. Three of the other four false negative prospective specimens and all 11 of the false negative retrospective specimens were mixed cultures containing gram-negative organisms that were correctly detected by the ePlex BCID-GN Panel.

The assay labeling includes the following two limitations regarding mixed cultures, with the second limitation specifying the lower sensitivity of the ePlex BCID Panel for the Pan Candida and Pan Gram-Positive assay targets.

In mixed cultures, the ePlex BCID-GN Panel may not identify all organisms in the specimen, depending upon the concentration of each target present. The BCID-GN Pan Candida and Pan Gram-Positive assays are designed to detect Candida and gram-positive microorganisms in mixed blood cultures where these microorganisms might be missed by Gram stain. Lower than expected clinical sensitivity was observed for these analytes, which was likely due to the presence of Candida spp. or gram-positive microorganisms in mixed cultures at concentrations below the respective limits of detection for the Pan Candida and Pan GP targets.

Table 79: Clinical performance, Pan Candida and Pan Gram-Positive targets   

<table><tr><td rowspan=2 colspan=1>Target</td><td rowspan=2 colspan=1>Specimen Type</td><td rowspan=1 colspan=2>Sensitivity/PPA</td><td rowspan=1 colspan=2>Specificity/NPA</td></tr><tr><td rowspan=1 colspan=1>TP/TP+FN</td><td rowspan=1 colspan=1>% (95% CI)</td><td rowspan=1 colspan=1>TN/TN+FP</td><td rowspan=1 colspan=1>% (95% CI)</td></tr><tr><td rowspan=6 colspan=1>Pan Candida</td><td rowspan=1 colspan=1>Prospective (Fresh)</td><td rowspan=1 colspan=1>1/1</td><td rowspan=1 colspan=1>100 (20.7-100)</td><td rowspan=1 colspan=1>165/166</td><td rowspan=1 colspan=1>99.4 (96.7-99.9)</td></tr><tr><td rowspan=1 colspan=1>Prospective (Frozen)</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>---</td><td rowspan=1 colspan=1>182/182</td><td rowspan=1 colspan=1>100 (97.9-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective (All)</td><td rowspan=1 colspan=1>1/1</td><td rowspan=1 colspan=1>100 (20.7-100)</td><td rowspan=1 colspan=1>347/348A</td><td rowspan=1 colspan=1>99.7 (98.4-99.9)</td></tr><tr><td rowspan=1 colspan=1>Retrospective</td><td rowspan=1 colspan=1>4/7B</td><td rowspan=1 colspan=1>57.1 (25.0-84.2)</td><td rowspan=1 colspan=1>569/570C</td><td rowspan=1 colspan=1>99.8 (99.0-100)</td></tr><tr><td rowspan=1 colspan=1>Retrospective (Non-Intended Use)</td><td rowspan=1 colspan=1>99/102D</td><td rowspan=1 colspan=1>97.1 (91.7-99.0)</td><td rowspan=1 colspan=1>638/639E</td><td rowspan=1 colspan=1>99.8 (99.1-100)</td></tr><tr><td rowspan=1 colspan=1>Contrived</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>---</td><td rowspan=1 colspan=1>777/777</td><td rowspan=1 colspan=1>100 (99.5-100)</td></tr><tr><td rowspan=6 colspan=1>Pan Gram-Positive</td><td rowspan=1 colspan=1>Prospective (Fresh)</td><td rowspan=1 colspan=1>7/8</td><td rowspan=1 colspan=1>87.5 (52.9-97.8)</td><td rowspan=1 colspan=1>155/159</td><td rowspan=1 colspan=1>97.5 (93.7-99.0)</td></tr><tr><td rowspan=1 colspan=1>Prospective (Frozen)</td><td rowspan=1 colspan=1>10/15</td><td rowspan=1 colspan=1>66.7 (41.7-84.8)</td><td rowspan=1 colspan=1>164/167</td><td rowspan=1 colspan=1>98.2 (94.9-99.4)</td></tr><tr><td rowspan=1 colspan=1>Prospective (All)</td><td rowspan=1 colspan=1>17/23F</td><td rowspan=1 colspan=1>73.9 (53.5-87.5)</td><td rowspan=1 colspan=1>319/326G</td><td rowspan=1 colspan=1>97.9 (95.6-99.0)</td></tr><tr><td rowspan=1 colspan=1>Retrospective</td><td rowspan=1 colspan=1>44/55H</td><td rowspan=1 colspan=1>80.0 (67.6-88.4)</td><td rowspan=1 colspan=1>512/5221</td><td rowspan=1 colspan=1>98.1 (96.5-99.0)</td></tr><tr><td rowspan=1 colspan=1>Retrospective (Non-Intended Use)</td><td rowspan=1 colspan=1>567/571</td><td rowspan=1 colspan=1>99.3 (98.2-99.7)</td><td rowspan=1 colspan=1>165/170&#x27;</td><td rowspan=1 colspan=1>97.1 (93.3-98.7)</td></tr><tr><td rowspan=1 colspan=1>Contrived</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>---</td><td rowspan=1 colspan=1>776/777</td><td rowspan=1 colspan=1>99.9 (99.3-100)</td></tr></table>

A. Candida glabrata was detected in the $1 / 1$ false positive specimen using PCR/sequencing.   
B. 3 of 3 $( 1 0 0 \% )$ false negative results occurred in specimens in mixed infections with bacterial organisms where the BCID-GN Panel results were correct for the other infections in these specimens.   
C. Candida albicans was detected in the 1/1 false positive specimen using PCR/sequencing.   
D. 2 of 3 $( 6 7 \% )$ false negative results occurred in specimens in mixed infections with bacterial organisms where the BCID-GN Panel results were correct for the other infections in these specimens.   
E. Candida glabrata was detected in the 1/1 false positive specimen using PCR/sequencing.   
F. Bacillus (the gram-positive organism identified by standard laboratory procedures) was not detected in 2 false negative specimens using PCR/sequencing, but 16S sequencing instead detected Paenibacillus lautus and Paenibacillus urinalis, which were not identified by standard laboratory procedures. 3 of the remaining 4 $( 7 5 \% )$ false negative results occurred in specimens in mixed infections with bacterial organisms where the BCID-GN Panel correctly detected the gram-negative organisms present.   
G. Enterococcus (1), Staphylococcus (3), or Streptococcus (2) were detected in $6 / 7$ false positive specimens using PCR/sequencing (testing was not performed for the remaining 1/7 false positive specimens).   
H. 11 of 11 $( 1 0 0 \% )$ false negative results occurred in specimens in mixed infections with bacterial organisms where the BCID-GN Panel correctly detected the gram-negative organisms present.   
I. Enterococcus (2), Staphylococcus (1), or Streptococcus (5) were detected in 8/10 false positive specimens using PCR/sequencing (testing was not performed for the remaining 2/10 false positive specimens).   
J. Bacillus (1) or Streptococcus (1) were detected in 2/5 false positive specimens using PCR/sequencing (testing was not performed for the remaining 3/5 false positive specimens).

# Performance for ePlex BCID-GN Panel for genus and group assays, stratification by species

The ePlex BCID-GN Panel reports genus or group level results for Citrobacter, Enterobacter cloacae complex, Enterobacter non-cloacae complex, Proteus, Salmonella, Serratia, Pan Candida, and Pan Gram-Positive assay targets. Clinical performance is described in Table 80 for intended use specimens (i.e., bottles containing gram-negative organisms by Gram stain). Performance is stratified by species identified by comparator methods.

Table 81 includes performance for the Pan-Candida and Pan-Gram Positive assays for nonintended use specimens (i.e., bottles with gram-positive or fungal organisms by Gram stain). This data was used to supplement the intended use specimens due to the lower prevalence of Candida spp. and gram-positive organisms observed in intended use specimens.

Table 80: Performance, genus and group assays stratified by species   

<table><tr><td rowspan="3">Target Species Detected by Comparator Method</td><td colspan="2">Prospective Specimens</td><td colspan="2">Retrospective Specimens</td><td colspan="2">Contrived Specimens</td><td colspan="2">Combined Specimens</td></tr><tr><td colspan="2">Sensitivity/PPA</td><td colspan="2">Sensitivity/PPA</td><td colspan="2">Sensitivity/PPA</td><td colspan="2">Sensitivity/PPA</td></tr><tr><td>TP/TP+ FN</td><td>% (95% CI)</td><td>TP/TP +FN</td><td>% (95% CI)</td><td>TP/T P+FN</td><td>% (95% CI)</td><td>TP/TP+ FN</td><td>% (95% CI)</td></tr><tr><td>Citrobacter</td><td>5/5</td><td>100 (56.6-100)</td><td>20/21</td><td>95.2 (77.3-99.2)</td><td>43/43</td><td>100 (91.8-100)</td><td>68/69</td><td>98.6 (92.2-99.7)</td></tr><tr><td>Citrobacter braakii</td><td></td><td></td><td>2/3</td><td>66.7 (20.8-93.9)</td><td>8/8</td><td>100 (67.6-100)</td><td>10/11</td><td>90.9 (62.3-98.4)</td></tr><tr><td>Citrobacter freundii</td><td>4/4</td><td>100 (51.0-100)</td><td>13/13</td><td>100 (77.2-100)</td><td>27/27</td><td>100 (87.5-100)</td><td>44/44</td><td>100 (92.0-100)</td></tr><tr><td>Citrobacter koseri</td><td>1/1</td><td>100 (20.7-100)</td><td>4/4</td><td>100 (51.0-100)</td><td>4/4</td><td>100 (51.0-100)</td><td>9/9</td><td>100 (70.1-100)</td></tr><tr><td>Citrobacter youngae</td><td></td><td></td><td>1/1</td><td>100 (20.7-100)</td><td>4/4</td><td>100 (51.0-100)</td><td>5/5</td><td>100 (56.6-100)</td></tr><tr><td>Enterobacter (non- cloacae complex)</td><td>8/10</td><td>80.0 (49.0- 94.3</td><td>12/12</td><td>100 (75.8-100)</td><td>36/36</td><td>100 (90.4-100)</td><td>56/58</td><td>96.6 (88.3-99.0)</td></tr><tr><td>Enterobacter aerogenes</td><td>7/9</td><td>77.8 (45.3- 93.7)</td><td>12/12</td><td>100 (75.8-100)</td><td>26/26</td><td>100 (87.1-100)</td><td>45/47</td><td>95.7 (85.8-98.8)</td></tr><tr><td>Enterobacter amnigenus</td><td></td><td></td><td></td><td></td><td>10/10</td><td>100 (72.2-100)</td><td>10/10</td><td>100 (72.2-100)</td></tr><tr><td>Enterobacter gergoviae</td><td>1/1</td><td>100 (20.7-100)</td><td></td><td></td><td></td><td></td><td>1/1</td><td>100 (20.7-100)</td></tr><tr><td>Enterobacter cloacae complex</td><td>19/19</td><td>100 (83.2-100)</td><td>47/50</td><td>94.0 (83.8-97.9)</td><td>35/37</td><td>94.6 (82.3- 98.5)</td><td>101/106</td><td>95.3 (89.4-98.0)</td></tr><tr><td>Enterobacter asburiae</td><td></td><td></td><td></td><td></td><td>6/8</td><td>75.0 (40.9- 92.9)</td><td>6/8</td><td>75.0 (40.9-92.9)</td></tr><tr><td>Enterobacter cloacae</td><td>19/19</td><td>100 (83.2-100)</td><td>46/49</td><td>93.9 (83.5-97.9)</td><td>28/28</td><td>100 (87.9-100)</td><td>93/96</td><td>96.9 (91.2-98.9)</td></tr><tr><td>Enterobacter hormaechei</td><td></td><td>95.7 (79.0-</td><td>1/1</td><td>100 (20.7-100)</td><td>1/1</td><td>100 (20.7-100)</td><td>2/2</td><td>100 (34.2-100)</td></tr><tr><td>Proteus</td><td>22/23</td><td>99.2</td><td>54/55</td><td>98.2 (90.4-99.7)</td><td>9/9</td><td>100 (70.1-100)</td><td>85/87</td><td>97.7 (92.0-99.4)</td></tr><tr><td>Proteus mirabilis</td><td>22/23</td><td>95.7 (79.0- 99.2</td><td>50/51</td><td>98.0 (89.7-99.7)</td><td>9/9</td><td>100 (70.1-100)</td><td>81/83</td><td>97.6 (91.6-99.3)</td></tr><tr><td>Proteus vulgaris</td><td></td><td></td><td>5/5</td><td>100 (56.6-100)</td><td></td><td></td><td>5/5</td><td>100 (56.6-100)</td></tr><tr><td>Salmonella</td><td>2/2</td><td>100 (34.2-100)</td><td>18/19</td><td>94.7 (75.4-99.1)</td><td>34/35</td><td>97.1 (85.5- 99.5)</td><td>54/56</td><td>96.4 (87.9-99.0)</td></tr><tr><td>Salmonella</td><td>2/2</td><td>100 (34.2-100)</td><td>15/15</td><td>100 (79.6-100)</td><td></td><td></td><td>17/17</td><td>100 (81.6-100)</td></tr><tr><td>Salmonella 4,5,12:i:-</td><td></td><td></td><td></td><td></td><td>2/2</td><td>100 (34.2-100)</td><td>2/2</td><td>100 (34.2-100)</td></tr><tr><td>Salmonella Heidelberg</td><td></td><td></td><td></td><td></td><td>2/2</td><td>100 (34.2-100)</td><td>2/2</td><td>100 (34.2-100)</td></tr><tr><td>Salmonella Infantis</td><td></td><td></td><td></td><td></td><td>2/2</td><td>100 (34.2-100)</td><td>2/2</td><td>100 (34.2-100)</td></tr><tr><td>Salmonella Javiana</td><td></td><td></td><td></td><td></td><td>1/1</td><td>100 (20.7-100)</td><td>1/1</td><td>100 (20.7-100)</td></tr><tr><td>Salmonella Montevideo</td><td></td><td></td><td></td><td></td><td>7/8</td><td>87.5 (52.9-</td><td>7/8</td><td>87.5 (52.9-97.8)</td></tr><tr><td>Salmonella Muenchen</td><td></td><td></td><td></td><td></td><td>1/1</td><td>97.8 100 (20.7-100)</td><td>1/1</td><td>100 (20.7-100)</td></tr><tr><td>Salmonella Newport</td><td></td><td></td><td></td><td></td><td>6/6</td><td>100 (61.0-100)</td><td>6/6</td><td>100 (61.0-100)</td></tr><tr><td>Salmonella Typhimurium Salmonella choleraesuis subsp. Arizonae Salmonella enterica</td><td></td><td></td><td>0/1</td><td>0.0 (0.0-79.3)</td><td>7/7</td><td>100 (64.6-100)</td><td>7/7</td><td>100 (64.6-100)</td></tr><tr><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>0/1</td><td>0.0 (0.0-79.3)</td></tr><tr><td>subsp. enterica Enteritidis (Group D1)</td><td></td><td></td><td></td><td></td><td>6/6</td><td>100 (61.0-100)</td><td>6/6</td><td>100 (61.0-100)</td></tr><tr><td>Salmonella enterica subsp. enterica serovar Typhimurium</td><td></td><td></td><td>1/1</td><td>100 (20.7-100)</td><td></td><td></td><td>1/1</td><td>100 (20.7-100)</td></tr><tr><td>Salmonella Typhi</td><td></td><td></td><td>2/2</td><td>100 (34.2-100)</td><td></td><td></td><td></td><td></td></tr><tr><td>Serratia</td><td>10/10</td><td>100 (72.2-100)</td><td>34/34</td><td>100 (89.8-100)</td><td>36/36</td><td>100 (90.4-100)</td><td>2/2</td><td>100 (34.2-100)</td></tr><tr><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>80/80</td><td>100 (95.4-100)</td></tr><tr><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr><tr><td>Serratia grimesii</td><td></td><td></td><td></td><td></td><td>3/3</td><td>100 (43.9-100)</td><td>3/3</td><td>100 (51.0-100) 100 (43.9-100)</td></tr><tr><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>4/4</td><td></td></tr><tr><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr><tr><td></td><td></td><td></td><td></td><td></td><td></td><td>100 (51.0-100)</td><td></td><td></td></tr><tr><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr><tr><td>Serratia ficaria</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr><tr><td></td><td></td><td></td><td></td><td></td><td>4/4</td><td></td><td></td><td></td></tr><tr><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr><tr><td rowspan="4">Target Species Detected by Comparator Method</td><td colspan="2">Prospective Specimens</td><td colspan="2">Retrospective Specimens</td><td colspan="2">Contrived Specimens</td><td colspan="2">Combined Specimens</td></tr><tr><td colspan="2">Sensitivity/PPA</td><td colspan="2">Sensitivity/PPA</td><td colspan="2">Sensitivity/PPA</td><td colspan="2">Sensitivity/PPA</td></tr><tr><td>TP/TP+ FN</td><td>% (95% CI)</td><td>TP/TP +FN</td><td>% (95% CI)</td><td>TP/T P+FN</td><td>% (95% CI)</td><td>TP/TP+ FN</td><td>% (95% CI)</td></tr><tr><td>Serratia liquefaciens</td><td>1/1</td><td>100 (20.7-100) 100 (70.1-100)</td><td>34/34</td><td>100 (89.8-100)</td><td>19/19</td><td>100 (83.2-100)</td><td>1/1 62/62</td><td>100 (20.7-100) 100 (94.2-100)</td></tr><tr><td>Serratia marcescens Serratia plymuthica</td><td>9/9</td><td></td><td></td><td></td><td>3/3</td><td>100 (43.9-100)</td><td>3/3</td><td>100 (43.9-100)</td></tr><tr><td>Serratia rubidaea</td><td></td><td></td><td></td><td></td><td>7/7</td><td>100 (64.6-100)</td><td>7/7</td><td>100 (64.6-100)</td></tr><tr><td>Pan Candida</td><td>1/1</td><td>100 (20.7-100) 100 (20.7-100)</td><td>4/7</td><td>57.1 (25.0-84.2)</td><td>N/A</td><td>N/A</td><td>5/8</td><td>62.5 (30.6-86.3)</td></tr><tr><td>Candida albicans</td><td>1/1</td><td></td><td>2/4</td><td>50.0 (15.0-85.0)</td><td></td><td></td><td>3/5</td><td>60.0 (23.1-88.2)</td></tr><tr><td>Candida glabrata</td><td></td><td></td><td>1/2</td><td>50.0 (9.5-90.5)</td><td></td><td></td><td>1/2</td><td>50.0 (9.5-90.5)</td></tr><tr><td>Candida krusei</td><td></td><td></td><td>1/1</td><td>100 (20.7-100)</td><td></td><td></td><td>1/1</td><td>100 (20.7-100)</td></tr><tr><td>Candida parapsilosis</td><td></td><td>73.9 (53.5-</td><td></td><td></td><td></td><td></td><td></td><td></td></tr><tr><td>Pan Gram-Positive</td><td>17/23</td><td>87.5 25.0 (4.6-69.9)</td><td>44/55</td><td>80.0 (67.6-88.4)</td><td>N/A</td><td>N/A</td><td>61/78</td><td>78.2 (67.8-85.9)</td></tr><tr><td>Bacillus (unspeciated) Enterococcus</td><td>1/4</td><td></td><td></td><td></td><td></td><td></td><td>1/4</td><td>25.0 (4.6-69.9)</td></tr><tr><td>(unspeciated) Enterococcus</td><td></td><td></td><td>0/1</td><td>0.0 (0.0-79.3)</td><td></td><td></td><td>0/1</td><td>0.0 (0.0-79.3)</td></tr><tr><td>casseliflavus</td><td></td><td>71.4 (35.9-</td><td>0/1</td><td>0.0 (0.0-79.3)</td><td></td><td></td><td>0/1</td><td>0.0 (0.0-79.3)</td></tr><tr><td>Enterococcus faecalis</td><td>5/7</td><td>91.8)</td><td>18/20</td><td>90.0 (69.9-97.2)</td><td></td><td></td><td>23/27</td><td>85.2 (67.5-94.1)</td></tr><tr><td>Enterococcus faecium</td><td>1/1</td><td>100 (20.7-100)</td><td>8/9</td><td>88.9 (56.5-98.0)</td><td></td><td></td><td>9/10</td><td>90.0 (59.6-98.2)</td></tr><tr><td>Staphylococcus (unspeciated)</td><td>3/3</td><td>100 (43.9-100)</td><td>4/6</td><td>66.7 (30.0-90.3)</td><td></td><td></td><td>7/9</td><td>77.8 (45.3-93.7)</td></tr><tr><td>Staphylococcus aureus</td><td>2/2</td><td>100 (34.2-100)</td><td>5/6</td><td>83.3 (43.6-97.0)</td><td></td><td></td><td>7/8</td><td>87.5 (52.9-97.8)</td></tr><tr><td>Staphylococcus cohnii</td><td>1/1</td><td>100 (20.7-100)</td><td></td><td></td><td></td><td></td><td>1/1</td><td>100 (20.7-100)</td></tr><tr><td>Staphylococcus epidermidis</td><td>1/1</td><td>100 (20.7-100)</td><td>2/3</td><td>66.7 (20.8-93.9)</td><td></td><td></td><td>3/4</td><td>75.0 (30.1-95.4)</td></tr><tr><td>Staphylococcus haemolyticus</td><td>2/2</td><td>100 (34.2-100)</td><td></td><td></td><td></td><td></td><td>2/2</td><td>100 (34.2-100)</td></tr><tr><td>Staphylococcus hominis</td><td>1/1</td><td>100 (20.7-100)</td><td>3/3</td><td>100 (43.9-100)</td><td></td><td></td><td>4/4</td><td>100 (51.0-100)</td></tr><tr><td>Streptococcus</td><td>0/1</td><td>0.0 (0.0-79.3)</td><td></td><td></td><td></td><td></td><td>0/1</td><td>0.0 (0.0-79.3)</td></tr><tr><td>Streptococcus - viridans group</td><td>1/1</td><td>100 (20.7-100)</td><td>0/1</td><td>0.0 (0.0-79.3)</td><td></td><td></td><td>1/2</td><td>50.0 (9.5-90.5)</td></tr><tr><td>Streptococcus anginosus</td><td>1/1</td><td>100 (20.7-100)</td><td>4/5</td><td>80.0 (37.6-96.4)</td><td></td><td></td><td>5/6</td><td>83.3 (43.6-97.0)</td></tr><tr><td>group Streptococcus infantarius</td><td></td><td></td><td>1/1</td><td>100 (20.7-100)</td><td></td><td></td><td>1/1</td><td>100 (20.7-100)</td></tr><tr><td>Streptococcus mitis group</td><td></td><td></td><td>1/1</td><td>100 (20.7-100)</td><td></td><td></td><td>1/1</td><td>100 (20.7-100)</td></tr><tr><td>Streptococcus oralis</td><td></td><td></td><td>1/1</td><td>100 (20.7-100)</td><td></td><td></td><td>1/1</td><td>100 (20.7-100)</td></tr><tr><td>Streptococcus pneumoniae</td><td></td><td></td><td>1/1</td><td>100 (20.7-100)</td><td></td><td></td><td>1/1</td><td>100 (20.7-100)</td></tr><tr><td>Streptococcus salivarius</td><td></td><td></td><td>1/1</td><td>100 (20.7-100)</td><td></td><td></td><td>1/1</td><td>100 (20.7-100)</td></tr><tr><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr></table>

Table 81: Pan Candida and Pan-Gram positive assay targets, performance stratified by species for non-intended use specimens (Specimens with gram-positive or fungal organisms by Gram stain).   

<table><tr><td rowspan="3">Target Species Detected by Comparator Method</td><td colspan="2">Retrospective (Non-Intended Use) Specimens</td></tr><tr><td colspan="2">Sensitivity/PPA</td></tr><tr><td>TP/TP+FN</td><td>% (95% CI)</td></tr><tr><td>Pan Candida</td><td>99/102</td><td>97.1 (91.7-99.0)</td></tr><tr><td>Candida albicans</td><td>47/48</td><td>97.9 (89.1-99.6)</td></tr><tr><td>Candida glabrata</td><td>37/38</td><td>97.4 (86.5-99.5)</td></tr><tr><td>Candida krusei</td><td>3/3</td><td>100 (43.9-100)</td></tr><tr><td>Candida parapsilosis</td><td>15/16</td><td>93.8 (71.7-98.9)</td></tr><tr><td>Pan Gram-Positive</td><td>567/571</td><td>99.3 (98.2-99.7)</td></tr><tr><td>Bacillus (unspeciated)</td><td>4/4</td><td>100 (51.0-100)</td></tr><tr><td>Bacillus cereus</td><td>4/5</td><td>80.0 (37.6-96.4)</td></tr><tr><td>Bacillus cereus group - not anthracis</td><td>1/1</td><td>100 (20.7-100)</td></tr><tr><td>Enterococcus</td><td>1/1</td><td>100 (20.7-100)</td></tr><tr><td>Enterococcus faecalis</td><td>34/34</td><td>100 (89.8-100)</td></tr><tr><td>Enterococcus faecium</td><td>14/14</td><td>100 (78.5-100)</td></tr><tr><td>Coagulase-negative Staphylococci (CoNS)</td><td>14/14</td><td>100 (78.5-100)</td></tr><tr><td>Staphylococcus (unspeciated)</td><td>86/86</td><td>100 (95.7-100)</td></tr><tr><td>Staphylococcus aureus</td><td>173/174</td><td>99.4 (96.8-99.9)</td></tr><tr><td>Staphylococcus auricularis</td><td>3/3</td><td>100 (43.9-100)</td></tr><tr><td>Staphylococcus capitis</td><td>7/7</td><td>100 (64.6-100)</td></tr><tr><td>Staphylococcus cohnii</td><td>1/1</td><td>100 (20.7-100)</td></tr><tr><td>Staphylococcus epidermidis</td><td>86/87</td><td>98.9 (93.8-99.8)</td></tr><tr><td>Staphylococcus haemolyticus</td><td>6/6</td><td>100 (61.0-100)</td></tr><tr><td>Staphylococcus hominis</td><td>19/19</td><td>100 (83.2-100)</td></tr><tr><td>Staphylococcus hominis ssp hominis</td><td>21/21</td><td>100 (84.5-100)</td></tr><tr><td>Staphylococcus intermedius</td><td>1/1</td><td>100 (20.7-100)</td></tr><tr><td>Staphylococcus lugdunensis</td><td>1/1</td><td>100 (20.7-100)</td></tr><tr><td>Staphylococcus saccharolyticus</td><td>1/1</td><td>100 (20.7-100)</td></tr><tr><td>Staphylococcus saprophyticus</td><td>1/1</td><td>100 (20.7-100)</td></tr><tr><td>Staphylococcus sciuri</td><td>1/1</td><td>100 (20.7-100)</td></tr><tr><td>Staphylococcus simulans</td><td>2/2</td><td>100 (34.2-100)</td></tr><tr><td>Staphylococcus warneri</td><td>4/4</td><td>100 (51.0-100)</td></tr><tr><td>Alpha Hemolytic Streptococcus</td><td>1/1</td><td>100 (20.7-100)</td></tr><tr><td>Beta Hemolytic Streptococci, Group G</td><td>1/1</td><td>100 (20.7-100)</td></tr><tr><td>Gamma Hemolytic Streptococcus</td><td>1/1</td><td>100 (20.7-100)</td></tr><tr><td>Streptococcus (unspeciated)</td><td>9/9</td><td>100 (70.1-100)</td></tr><tr><td></td><td></td><td></td></tr><tr><td>Streptococcus - viridans group</td><td>17/17</td><td>100 (81.6-100)</td></tr><tr><td>Streptococcus agalactiae Streptococcus anginosus</td><td>21/21 2/2</td><td>100 (84.5-100) 100 (34.2-100)</td></tr><tr><td>Streptococcus bovis</td><td>2/2</td><td>100 (34.2-100)</td></tr><tr><td>Streptococcus constellatus ssp constellatus</td><td>1/1</td><td>100 (20.7-100)</td></tr><tr><td>Streptococcus dysgalactiae (Group G)</td><td>4/4</td><td>100 (51.0-100)</td></tr><tr><td>Streptococcus gordonii</td><td>1/1</td><td>100 (20.7-100)</td></tr><tr><td>Streptococcus intermedius</td><td>1/1</td><td>100 (20.7-100)</td></tr><tr><td>Streptococcus mitis</td><td>11/12</td><td>91.7 (64.6-98.5)</td></tr><tr><td>Streptococcus mitis group</td><td>2/2</td><td>100 (34.2-100)</td></tr><tr><td>Streptococcus pneumoniae</td><td>22/22</td><td>100 (85.1-100)</td></tr><tr><td>Streptococcus pyogenes</td><td>9/9</td><td>100 (70.1-100)</td></tr><tr><td>Streptococcus salivarius</td><td>2/2</td><td>100 (34.2-100)</td></tr></table>

# Clinical performance, resistance marker targets:

Results for ePlex BCID Panel resistance marker targets are only reported when an associated organism is detected in the same specimen. The assay report will report ‘N/A’ for a resistance marker if an applicable organism is not detected in the specimen (See Table 5 above for associated organisms specifically reported with the six targeted resistance markers on the ePlex BCID-GN Panel).

Clinical performance for the ePlex BCID-GN Panel assays for CTX-M, KPC, IMP, NDM, OXA and VIM targets are described in Tables 82-99 below. Note that there were no prospective or retrospective specimens that harbored IMP, NDM, or VIM beta-lactamase genes; therefore, performance was established for these analytes using contrived specimens.

Clinical Performance, CTX-M:

Table 82 includes the clinical performance for detection of CTX-M for prospective, retrospective and contrived specimens that contain an applicable gram-negative organism based on the comparator method (for clinical specimens) or the inoculated organism (for contrived specimens).

Table 83 includes the clinical performance summary for detection of CTX-M and any applicable organism by the BCID-GN Panel in comparison to detection/identification by comparator methods. Results include all prospective and retrospective specimens combined.

Table 84 includes the clinical performance for detection of CTX-M for prospective, retrospective and contrived specimens stratified by organism identified by the comparator method (for clinical specimens) and based on the organism used for inoculation (for contrived specimens).

Table 82: Clinical performance, CTX-M   

<table><tr><td rowspan=2 colspan=1>Target</td><td rowspan=2 colspan=1>Specimen Type</td><td rowspan=1 colspan=2>Sensitivity/PPA</td><td rowspan=1 colspan=2>Specificity/NPA</td></tr><tr><td rowspan=1 colspan=1>TP/TP+FN</td><td rowspan=1 colspan=1>% (95% CI)</td><td rowspan=1 colspan=1>TN/TN+FP</td><td rowspan=1 colspan=1>% (95% CI)</td></tr><tr><td rowspan=7 colspan=1>CTX-M</td><td rowspan=1 colspan=1>Prospective (Fresh)</td><td rowspan=1 colspan=1>10/13</td><td rowspan=1 colspan=1>76.9 (49.7-91.8)</td><td rowspan=1 colspan=1>127/127</td><td rowspan=1 colspan=1>100 (97.1-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective (Frozen)</td><td rowspan=1 colspan=1>12/16</td><td rowspan=1 colspan=1>75.0 (50.5-89.8)</td><td rowspan=1 colspan=1>144/144</td><td rowspan=1 colspan=1>100 (97.4-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective (All)</td><td rowspan=1 colspan=1>22/29</td><td rowspan=1 colspan=1>75.9 (57.9-87.8)</td><td rowspan=1 colspan=1>271/271</td><td rowspan=1 colspan=1>100 (98.6-100)</td></tr><tr><td rowspan=1 colspan=1>Retrospective</td><td rowspan=1 colspan=1>52/56</td><td rowspan=1 colspan=1>92.9 (83.0-97.2)</td><td rowspan=1 colspan=1>483/483</td><td rowspan=1 colspan=1>100 (99.2-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective / Retrospective</td><td rowspan=1 colspan=1>74/85A</td><td rowspan=1 colspan=1>87.1 (78.3-92.6)</td><td rowspan=1 colspan=1>754/754</td><td rowspan=1 colspan=1>100 (99.5-100)</td></tr><tr><td rowspan=1 colspan=1>Contrived</td><td rowspan=1 colspan=1>75/75</td><td rowspan=1 colspan=1>100 (95.1-100)</td><td rowspan=1 colspan=1>437/437</td><td rowspan=1 colspan=1>100 (99.1-100)</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>149/160</td><td rowspan=1 colspan=1>93.1 (88.1-96.1)</td><td rowspan=1 colspan=1>1191/1191</td><td rowspan=1 colspan=1>100 (99.7-100)</td></tr></table>

A. In 3/11 false negative specimens, CTX-M signal was above the threshold for detection; however, an associated organism was not detected by the BCID-GN Panel and the CTX-M target was reported as ‘N/A’. Further testing of the 8/11 remaining false negative specimens indicated that 7 of the 8 specimens may have been contaminated during the original comparator extraction process and misidentified as having CTX-M present. Specifically, results for 7 of the 8 specimens were negative for CTX-M from the following additional testing: 1) $\mathsf { q P C R }$ testing of 2 repeat extractions from the original specimen, 2) qPCR testing of an extraction from the isolate, and 3) testing of the original specimen with an FDA-cleared multiplex assay. For these 7 specimens, the originally extracted specimen was re-tested with qPCR and CTX-M was again detected, suggesting contamination during the original extraction process. The remaining 8th specimen was positive for CTX-M from the repeat extractions, negative for CTX-M from the isolate, and negative for CTX-M when tested with an FDA-cleared multiplex assay. These inconsistent detection results suggest the 8th specimen may be a true low-copy CTX-M positive specimen.

Table 83: Distribution of CTX-M in clinical specimens   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=4>Comparator Method</td></tr><tr><td rowspan=1 colspan=1>BCID-GN</td><td rowspan=1 colspan=1>Org+/ARG+</td><td rowspan=1 colspan=1>Org+/ARG-</td><td rowspan=1 colspan=1>Org-</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Org+/ARG+</td><td rowspan=1 colspan=1>74</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>76</td></tr><tr><td rowspan=1 colspan=1>Org+/ARG-</td><td rowspan=1 colspan=1>8A</td><td rowspan=1 colspan=1>741</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>751</td></tr><tr><td rowspan=1 colspan=1>Org-</td><td rowspan=1 colspan=1>3B</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>83</td><td rowspan=1 colspan=1>99</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>85</td><td rowspan=1 colspan=1>754</td><td rowspan=1 colspan=1>87</td><td rowspan=1 colspan=1>926</td></tr></table>

$\%$ Agreement $( 9 5 \% \mathrm { C I } )$ for $\mathrm { O r g ^ { + / A R G + } }$ : $7 4 / 8 5 { = } 8 7 . 1 \%$ (78.3-92.6) $\%$ Agreement $( 9 5 \% \mathrm { C I } )$ for Org+/ARG-: $7 4 1 / 7 5 4 { = } 9 8 . 3 \%$ (97.1-99.0) $\%$ Agreement $( 9 5 \%$ CI) for Org-: $8 3 / 8 7 { = } 9 5 . 4 \%$ (88.8-98.2)

A. Further testing of these 8 false negative specimens indicated that 7 of the 8 specimens may have been contaminated during the original extraction process and misidentified as having CTX-M present. Specifically, results for 7 of the 8 specimens were negative for CTX-M from the following additional testing: 1) $\mathsf { q P C R }$ testing of 2 repeat extractions from the original specimen, 2) $\mathsf { q P C R }$ testing of an extraction from the isolate, and 3) testing of the original specimen with an FDA-cleared multiplex assay. For these 7 specimens, the originally extracted specimen was re-tested with qPCR and CTX-M was again detected, suggesting contamination during the original extraction process. The remaining 8th specimen was positive for CTX-M from the repeat extractions, negative for CTX-M from the isolate, and negative for CTX-M when tested with an FDA-cleared multiplex assay. These inconsistent detection results suggest the 8th specimen may be a true low-copy CTX-M positive specimen.

B. For these 3 specimens, CTX-M signal was above the threshold for detection; however, an associated organism was not detected by the ePlex BCID-GN Panel and the CTX-M target was reported as ‘N/A’.

Table 84: Clinical performance of CTX-M target in specimens with associated organisms detected by comparator methods   

<table><tr><td rowspan=2 colspan=2>Species Detected by Comparator Method</td><td rowspan=1 colspan=2>Sensitivity/PPA</td><td rowspan=1 colspan=3>Specificity/NPA</td></tr><tr><td rowspan=1 colspan=1>TP/TP+FN</td><td rowspan=1 colspan=1>% (95% CI)</td><td rowspan=1 colspan=1>TN/TN+FP</td><td rowspan=1 colspan=2>% (95% CI)</td></tr><tr><td rowspan=4 colspan=1>Acinetobacter baumannii</td><td rowspan=1 colspan=1>Prospective</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>4/4</td><td rowspan=1 colspan=2>100 (51.0-100)</td></tr><tr><td rowspan=1 colspan=1>Retrospective</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>---</td><td rowspan=1 colspan=1>15/15</td><td rowspan=1 colspan=2>100 (79.6-100)</td></tr><tr><td rowspan=1 colspan=1>Contrived</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>---</td><td rowspan=1 colspan=1>55/55</td><td rowspan=1 colspan=2>100 (93.5-100)</td></tr><tr><td rowspan=1 colspan=1>Combined</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>---</td><td rowspan=1 colspan=1>74/74</td><td rowspan=1 colspan=2>100 (95.1-100)</td></tr><tr><td rowspan=4 colspan=1>Citrobacter</td><td rowspan=1 colspan=1>Prospective</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>---</td><td rowspan=1 colspan=1>5/5</td><td rowspan=1 colspan=2>100 (56.6-100)</td></tr><tr><td rowspan=1 colspan=1>Retrospective</td><td rowspan=1 colspan=1>1/1</td><td rowspan=1 colspan=1>100 (20.7-100)</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=2>100 (83.9-100)</td></tr><tr><td rowspan=1 colspan=1>Contrived</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>100 (61.0-100)</td><td rowspan=1 colspan=1>37/37</td><td rowspan=1 colspan=2>100 (90.6-100)</td></tr><tr><td rowspan=1 colspan=1>Combined</td><td rowspan=1 colspan=1>7/7</td><td rowspan=1 colspan=1>100 (64.6-100)</td><td rowspan=1 colspan=1>62/62</td><td rowspan=1 colspan=2>100 (94.2-100)</td></tr><tr><td rowspan=4 colspan=1>Enterobacter (non-cloacae complex)</td><td rowspan=1 colspan=1>Prospective</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=2>100 (72.2-100)</td></tr><tr><td rowspan=1 colspan=1>Retrospective</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>---</td><td rowspan=1 colspan=1>12/12</td><td rowspan=1 colspan=2>100 (75.8-100)</td></tr><tr><td rowspan=1 colspan=1>Contrived</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>36/36</td><td rowspan=1 colspan=2>100 (90.4-100)</td></tr><tr><td rowspan=1 colspan=1>Combined</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>---</td><td rowspan=1 colspan=1>58/58</td><td rowspan=1 colspan=2>100 (93.8-100)</td></tr><tr><td rowspan=4 colspan=1>Enterobacter cloacae complex</td><td rowspan=1 colspan=1>Prospective</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>---</td><td rowspan=1 colspan=1>19/19</td><td rowspan=1 colspan=2>100 (83.2-100)</td></tr><tr><td rowspan=1 colspan=1>Retrospective</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>---</td><td rowspan=1 colspan=1>50/50</td><td rowspan=1 colspan=2>100 (92.9-100)</td></tr><tr><td rowspan=1 colspan=1>Contrived</td><td rowspan=1 colspan=1>22/22</td><td rowspan=1 colspan=1>100 (85.1-100)</td><td rowspan=1 colspan=1>15/15</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100 (79.6-100)</td></tr><tr><td rowspan=1 colspan=1>Combined</td><td rowspan=1 colspan=1>22/22</td><td rowspan=1 colspan=1>100 (85.1-100)</td><td rowspan=1 colspan=1>84/84</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100 (95.6-100)</td></tr><tr><td rowspan=4 colspan=1>Escherichia coli</td><td rowspan=1 colspan=1>Prospective</td><td rowspan=1 colspan=1>16/18</td><td rowspan=1 colspan=1>88.9 (67.2-96.9)</td><td rowspan=1 colspan=1>115/115</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100 (96.8-100)</td></tr><tr><td rowspan=1 colspan=1>Retrospective</td><td rowspan=1 colspan=1>35/37</td><td rowspan=1 colspan=1>94.6 (82.3-98.5)</td><td rowspan=1 colspan=1>103/103</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100 (96.4-100)</td></tr><tr><td rowspan=1 colspan=1>Contrived</td><td rowspan=1 colspan=1>22/22</td><td rowspan=1 colspan=1>100 (85.1-100)</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100 (88.6-100)</td></tr><tr><td rowspan=1 colspan=1>Combined</td><td rowspan=1 colspan=1>73/77</td><td rowspan=1 colspan=1>94.8 (87.4-98.0)</td><td rowspan=1 colspan=1>248/248</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100 (98.5-100)</td></tr><tr><td rowspan=4 colspan=1>Klebsiella oxytoca</td><td rowspan=1 colspan=1>Prospective</td><td rowspan=1 colspan=1>0/1</td><td rowspan=1 colspan=1>0.0 (0.0-79.3)</td><td rowspan=1 colspan=1>12/12</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100 (75.8-100)</td></tr><tr><td rowspan=1 colspan=1>Retrospective</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>34/34</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100 (89.8-100)</td></tr><tr><td rowspan=1 colspan=1>Contrived</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100 (83.9-100)</td></tr><tr><td rowspan=1 colspan=1>Combined</td><td rowspan=1 colspan=1>0/1</td><td rowspan=1 colspan=1>0.0 (0.0-79.3)</td><td rowspan=1 colspan=1>66/66</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100 (94.5-100)</td></tr><tr><td rowspan=4 colspan=1>Klebsiella pneumoniae group</td><td rowspan=1 colspan=1>Prospective</td><td rowspan=1 colspan=1>5/5</td><td rowspan=1 colspan=1>100 (56.6-100)</td><td rowspan=1 colspan=1>56/56</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100 (93.6-100)</td></tr><tr><td rowspan=1 colspan=1>Retrospective</td><td rowspan=1 colspan=1>14/15</td><td rowspan=1 colspan=1>93.3 (70.2-98.8)</td><td rowspan=1 colspan=1>93/93</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100 (96.0-100)</td></tr><tr><td rowspan=1 colspan=1>Contrived</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>100 (83.9-100)</td><td rowspan=1 colspan=1>52/52</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100 (93.1-100)</td></tr><tr><td rowspan=1 colspan=1>Combined</td><td rowspan=1 colspan=1>39/40</td><td rowspan=1 colspan=1>97.5 (87.1-99.6)</td><td rowspan=1 colspan=1>201/201</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100 (98.1-100)</td></tr><tr><td rowspan=4 colspan=1>Morganella morganii</td><td rowspan=1 colspan=1>Prospective</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100 (43.9-100)</td></tr><tr><td rowspan=1 colspan=1>Retrospective</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=2>100 (72.2-100)</td></tr><tr><td rowspan=1 colspan=1>Contrived</td><td rowspan=1 colspan=1>5/5</td><td rowspan=1 colspan=1>100 (56.6-100)</td><td rowspan=1 colspan=1>44/44</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100 (92.0-100)</td></tr><tr><td rowspan=1 colspan=1>Combined</td><td rowspan=1 colspan=1>5/5</td><td rowspan=1 colspan=1>100 (56.6-100)</td><td rowspan=1 colspan=1>57/57</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100 (93.7-100)</td></tr><tr><td rowspan=4 colspan=1>Proteus</td><td rowspan=1 colspan=1>Prospective</td><td rowspan=1 colspan=1>2/5</td><td rowspan=1 colspan=1>40.0 (11.8-76.9)</td><td rowspan=1 colspan=1>18/18</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100 (82.4-100)</td></tr><tr><td rowspan=1 colspan=1>Retrospective</td><td rowspan=1 colspan=1>2/3</td><td rowspan=1 colspan=1>66.7 (20.8-93.9)</td><td rowspan=1 colspan=1>52/52</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100 (93.1-100)</td></tr><tr><td rowspan=1 colspan=1>Contrived</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>9/9</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100 (70.1-100)</td></tr><tr><td rowspan=1 colspan=1>Combined</td><td rowspan=1 colspan=1>4/8</td><td rowspan=1 colspan=1>50.0 (21.5-78.5)</td><td rowspan=1 colspan=1>79/79</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100 (95.4-100)</td></tr><tr><td rowspan=4 colspan=1>Proteus mirabilis</td><td rowspan=1 colspan=1>Prospective</td><td rowspan=1 colspan=1>2/5</td><td rowspan=1 colspan=1>40.0 (11.8-76.9)</td><td rowspan=1 colspan=1>18/18</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100 (82.4-100)</td></tr><tr><td rowspan=1 colspan=1>Retrospective</td><td rowspan=1 colspan=1>2/3</td><td rowspan=1 colspan=1>66.7 (20.8-93.9)</td><td rowspan=1 colspan=1>48/48</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100 (92.6-100)</td></tr><tr><td rowspan=1 colspan=1>Contrived</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>9/9</td><td rowspan=1 colspan=1>−100</td><td rowspan=1 colspan=1>100 (70.1-100)</td></tr><tr><td rowspan=1 colspan=1>Combined</td><td rowspan=1 colspan=1>4/8</td><td rowspan=1 colspan=1>50.0 (21.5-78.5)</td><td rowspan=1 colspan=1>75/75</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100 (95.1-100)</td></tr><tr><td rowspan=4 colspan=1>Pseudomonas aeruginosa</td><td rowspan=1 colspan=1>Prospective</td><td rowspan=1 colspan=1>0/1</td><td rowspan=1 colspan=1>0.0 (0.0-79.3)</td><td rowspan=1 colspan=1>27/27</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100 (87.5-100)</td></tr><tr><td rowspan=1 colspan=1>Retrospective</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>60/60</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100 (94.0-100)</td></tr><tr><td rowspan=1 colspan=1>Contrived</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>32/32</td><td rowspan=1 colspan=2>100 (89.3-100)</td></tr><tr><td rowspan=1 colspan=1>Combined</td><td rowspan=1 colspan=1>0/1</td><td rowspan=1 colspan=1>0.0 (0.0-79.3)</td><td rowspan=1 colspan=1>119/119</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100 (96.9-100)</td></tr><tr><td rowspan=4 colspan=1>Salmonella</td><td rowspan=1 colspan=1>Prospective</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>---</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>−100</td><td rowspan=1 colspan=1>100 (34.2-100)</td></tr><tr><td rowspan=1 colspan=1>Retrospective</td><td rowspan=1 colspan=1>1/1</td><td rowspan=1 colspan=1>100 (20.7-100)</td><td rowspan=1 colspan=1>18/18</td><td rowspan=1 colspan=1>−100</td><td rowspan=1 colspan=1>100 (82.4-100)</td></tr><tr><td rowspan=1 colspan=1>Contrived</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>---</td><td rowspan=1 colspan=1>35/35</td><td rowspan=1 colspan=2>100 (90.1-100)</td></tr><tr><td rowspan=1 colspan=1>Combined</td><td rowspan=1 colspan=1>1/1</td><td rowspan=1 colspan=1>100 (20.7-100)</td><td rowspan=1 colspan=1>55/55</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100 (93.5-100)</td></tr><tr><td rowspan=4 colspan=1>Serratia</td><td rowspan=1 colspan=1>Prospective</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>−100</td><td rowspan=1 colspan=1>100 (72.2-100)</td></tr><tr><td rowspan=1 colspan=1>Retrospective</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>34/34</td><td rowspan=1 colspan=2>100 (89.8-100)</td></tr><tr><td rowspan=1 colspan=1>Contrived</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>36/36</td><td rowspan=1 colspan=2>100 (90.4-100)</td></tr><tr><td rowspan=1 colspan=1>Combined</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>---</td><td rowspan=1 colspan=1>80/80</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100 (95.4-100)</td></tr><tr><td rowspan=1 colspan=1>Serratia marcescens</td><td rowspan=1 colspan=1>Prospective</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>---</td><td rowspan=1 colspan=1>9/9</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>70.1-100)</td></tr></table>

<table><tr><td rowspan=2 colspan=2>Species Detected by Comparator Method</td><td rowspan=1 colspan=2>Sensitivity/PPA</td><td rowspan=1 colspan=2>Specificity/NPA</td></tr><tr><td rowspan=1 colspan=1>TP/TP+FN</td><td rowspan=1 colspan=1>% (95% CI)</td><td rowspan=1 colspan=1>TN/TN+FP</td><td rowspan=1 colspan=1>% (95% CI)</td></tr><tr><td rowspan=3 colspan=1></td><td rowspan=1 colspan=1>Retrospective</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>---</td><td rowspan=1 colspan=1>34/34</td><td rowspan=1 colspan=1>100 (89.8-100)</td></tr><tr><td rowspan=1 colspan=1>Contrived</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>--</td><td rowspan=1 colspan=1>19/19</td><td rowspan=1 colspan=1>100 (83.2-100)</td></tr><tr><td rowspan=1 colspan=1>Combined</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>---</td><td rowspan=1 colspan=1>62/62</td><td rowspan=1 colspan=1>100 (94.2-100)</td></tr><tr><td rowspan=4 colspan=1>Stenotrophomonas maltophilia</td><td rowspan=1 colspan=1>Prospective</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>---</td><td rowspan=1 colspan=1>4/4</td><td rowspan=1 colspan=1>100 (51.0-100)</td></tr><tr><td rowspan=1 colspan=1>Retrospective</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>---</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>100 (72.2-100)</td></tr><tr><td rowspan=1 colspan=1>Contrived</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>--</td><td rowspan=1 colspan=1>36/36</td><td rowspan=1 colspan=1>100 (90.4-100)</td></tr><tr><td rowspan=1 colspan=1>Combined</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>---</td><td rowspan=1 colspan=1>50/50</td><td rowspan=1 colspan=1>100 (92.9-100)</td></tr></table>

Clinical performance, IMP:

Table 85 includes the clinical performance for detection of IMP for prospective, retrospective and contrived specimens that contain an applicable gram-negative organism based on the comparator method (for clinical specimens) or the inoculated organism (for contrived specimens). No prospective or retrospective specimens were found to contain IMP by the ePlex BCID-GN Panel or by the comparator method.

Table 86 includes a performance summary for detection of IMP and any applicable organism by the ePlex BCID-GN Panel in comparison to detection/identification by comparator methods. Results include all prospective and retrospective specimens combined.

Table 87 includes the performance for detection of IMP for contrived specimens stratified by organism.

Table 85: Clinical performance, IMP   

<table><tr><td rowspan=2 colspan=1>Target</td><td rowspan=2 colspan=1>Specimen Type</td><td rowspan=1 colspan=2>Sensitivity/PPA</td><td rowspan=1 colspan=2>Specificity/NPA</td></tr><tr><td rowspan=1 colspan=1>TP/TP+FN</td><td rowspan=1 colspan=1>% (95% CI)</td><td rowspan=1 colspan=1>TN/TN+FP</td><td rowspan=1 colspan=1>% (95% CI)</td></tr><tr><td rowspan=7 colspan=1>IMP</td><td rowspan=1 colspan=1>Prospective (Fresh)</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>138/138</td><td rowspan=1 colspan=1>100 (97.3-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective (Frozen)</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>159/159</td><td rowspan=1 colspan=1>100 (97.6-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective (All)</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>---</td><td rowspan=1 colspan=1>297/297</td><td rowspan=1 colspan=1>100 (98.7-100)</td></tr><tr><td rowspan=1 colspan=1>Retrospective</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>532/532</td><td rowspan=1 colspan=1>100 (99.3-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective / Retrospective</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>---</td><td rowspan=1 colspan=1>829/829</td><td rowspan=1 colspan=1>100 (99.5-100)</td></tr><tr><td rowspan=1 colspan=1>Contrived</td><td rowspan=1 colspan=1>40/40</td><td rowspan=1 colspan=1>100 (91.2-100)</td><td rowspan=1 colspan=1>436/436</td><td rowspan=1 colspan=1>100 (99.1-100)</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>40/40</td><td rowspan=1 colspan=1>100 (91.2-100)</td><td rowspan=1 colspan=1>1265/1265</td><td rowspan=1 colspan=1>100 (99.7-100)</td></tr></table>

Table 86: Distribution of IMP in clinical specimens   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=4>Comparator Method</td></tr><tr><td rowspan=1 colspan=1>BCID-GN</td><td rowspan=1 colspan=1>Org+/ARG+</td><td rowspan=1 colspan=1>Org+/ARG-</td><td rowspan=1 colspan=1>Org-</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Org+/ARG+</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Org+/ARG-</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>812</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>816</td></tr><tr><td rowspan=1 colspan=1>Org-</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>93</td><td rowspan=1 colspan=1>110</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>829</td><td rowspan=1 colspan=1>97</td><td rowspan=1 colspan=1>926</td></tr></table>

$\%$ Agreement $( 9 5 \%$ CI) for $\mathrm { O r g ^ { + / A R G ^ { + } } }$ : $0 / 0 { = } \mathbf { N } / \mathbf { A }$ $\%$ Agreement $( 9 5 \% \mathrm { C I } )$ for Org+/ARG-: $8 1 2 / 8 2 9 { = } 9 7 . 9 \%$ (96.7-98.7) $\%$ Agreement $9 5 \%$ CI) for Org-: $9 3 / 9 7 { = } 9 5 . 9 \%$ (89.9-98.4)

Table 87: Clinical performance for IMP in contrived specimens   

<table><tr><td rowspan=2 colspan=1>Species Detected by ComparatorMethod</td><td rowspan=1 colspan=2>Sensitivity/PPA</td><td rowspan=1 colspan=2>Specificity/NPA</td></tr><tr><td rowspan=1 colspan=1>TP/TP+FN</td><td rowspan=1 colspan=1>% (95% CI)</td><td rowspan=1 colspan=1>TN/TN+FP</td><td rowspan=1 colspan=1>% (95% CI)</td></tr><tr><td rowspan=1 colspan=1>Acinetobacter baumannii</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>---</td><td rowspan=1 colspan=1>74/74</td><td rowspan=1 colspan=1>100 (95.1-100)</td></tr><tr><td rowspan=1 colspan=1>Citrobacter</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>---</td><td rowspan=1 colspan=1>69/69</td><td rowspan=1 colspan=1>100 (94.7-100)</td></tr><tr><td rowspan=1 colspan=1>Enterobacter (non-cloacae complex)</td><td rowspan=1 colspan=1>5/5</td><td rowspan=1 colspan=1>100 (56.6-100)</td><td rowspan=1 colspan=1>53/53</td><td rowspan=1 colspan=1>100 (93.2-100)</td></tr><tr><td rowspan=1 colspan=1>Enterobacter cloacae complex</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>---</td><td rowspan=1 colspan=1>106/106</td><td rowspan=1 colspan=1>100 (96.5-100)</td></tr><tr><td rowspan=1 colspan=1>Escherichia coli</td><td rowspan=1 colspan=1>7/7</td><td rowspan=1 colspan=1>100 (64.6-100)</td><td rowspan=1 colspan=1>318/318</td><td rowspan=1 colspan=1>100 (98.8-100)</td></tr><tr><td rowspan=1 colspan=1>Klebsiella oxytoca</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>---</td><td rowspan=1 colspan=1>67/67</td><td rowspan=1 colspan=1>100 (94.6-100)</td></tr><tr><td rowspan=1 colspan=1>Klebsiella pneumoniae group</td><td rowspan=1 colspan=1>14/14</td><td rowspan=1 colspan=1>100 (78.5-100)</td><td rowspan=1 colspan=1>227/227</td><td rowspan=1 colspan=1>100 (98.3-100)</td></tr><tr><td rowspan=1 colspan=1>Morganella morganii</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>---</td><td rowspan=1 colspan=1>62/62</td><td rowspan=1 colspan=1>100 (94.2-100)</td></tr><tr><td rowspan=1 colspan=1>Proteus</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>--</td><td rowspan=1 colspan=1>87/87</td><td rowspan=1 colspan=1>100 (95.8-100)</td></tr><tr><td rowspan=1 colspan=1>Proteus mirabilis</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>--</td><td rowspan=1 colspan=1>83/83</td><td rowspan=1 colspan=1>100 (95.6-100)</td></tr><tr><td rowspan=1 colspan=1>Pseudomonas aeruginosa</td><td rowspan=1 colspan=1>13/13</td><td rowspan=1 colspan=1>100 (77.2-100)</td><td rowspan=1 colspan=1>107/107</td><td rowspan=1 colspan=1>100 (96.5-100)</td></tr><tr><td rowspan=1 colspan=1>Salmonella</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>---</td><td rowspan=1 colspan=1>56/56</td><td rowspan=1 colspan=1>100 (93.6-100)</td></tr><tr><td rowspan=1 colspan=1>Serratia</td><td rowspan=1 colspan=1>1/1</td><td rowspan=1 colspan=1>100 (20.7-100)</td><td rowspan=1 colspan=1>79/79</td><td rowspan=1 colspan=1>100 (95.4-100)</td></tr><tr><td rowspan=1 colspan=1>Serratia marcescens</td><td rowspan=1 colspan=1>1/1</td><td rowspan=1 colspan=1>100 (20.7-100)</td><td rowspan=1 colspan=1>61/61</td><td rowspan=1 colspan=1>100 (94.1-100)</td></tr></table>

# Clinical performance, KPC:

Table 88 includes the clinical performance for KPC for prospective, retrospective and contrived specimens that contain an applicable gram-negative organism based on the comparator method (for clinical specimens) or the inoculated organism (for contrived specimens).

Table 89 includes the clinical performance summary for detection of KPC and any applicable organism by the BCID-GN Panel in comparison to detection/identification by comparator methods. Results include all prospective and retrospective specimens combined.

Table 90 includes the clinical performance for detection of KPC for prospective, retrospective and contrived specimens stratified by organism identified by the comparator method (for clinical specimens) and based on the organism used for inoculation (for contrived specimens).

Table 88: Clinical performance, KPC   

<table><tr><td rowspan=2 colspan=1>Target</td><td rowspan=2 colspan=1>Specimen Type</td><td rowspan=1 colspan=2>Sensitivity/PPA</td><td rowspan=1 colspan=2>Specificity/NPA</td></tr><tr><td rowspan=1 colspan=1>TP/TP+FN</td><td rowspan=1 colspan=1>% (95% CI)</td><td rowspan=1 colspan=1>TN/TN+FP</td><td rowspan=1 colspan=1>% (95% CI)</td></tr><tr><td rowspan=7 colspan=1>KPC</td><td rowspan=1 colspan=1>Prospective (Fresh)</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>100 (34.2-100)</td><td rowspan=1 colspan=1>136/136</td><td rowspan=1 colspan=1>100 (97.3-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective (Frozen)</td><td rowspan=1 colspan=1>1/1</td><td rowspan=1 colspan=1>100 (20.7-100)</td><td rowspan=1 colspan=1>158/158</td><td rowspan=1 colspan=1>100 (97.6-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective (All)</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>100 (43.9-100)</td><td rowspan=1 colspan=1>294/294</td><td rowspan=1 colspan=1>100 (98.7-100)</td></tr><tr><td rowspan=1 colspan=1>Retrospective</td><td rowspan=1 colspan=1>4/5</td><td rowspan=1 colspan=1>80.0 (37.6-96.4)</td><td rowspan=1 colspan=1>527/528</td><td rowspan=1 colspan=1>99.8 (98.9-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective / Retrospective</td><td rowspan=1 colspan=1>7/8</td><td rowspan=1 colspan=1>87.5 (52.9-97.8)</td><td rowspan=1 colspan=1>821/822</td><td rowspan=1 colspan=1>99.9 (99.3-100)</td></tr><tr><td rowspan=1 colspan=1>Contrived</td><td rowspan=1 colspan=1>44/44</td><td rowspan=1 colspan=1>100 (92.0-100)</td><td rowspan=1 colspan=1>477/477</td><td rowspan=1 colspan=1>100 (99.2-100)</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>51/52</td><td rowspan=1 colspan=1>98.1 (89.9-99.7)</td><td rowspan=1 colspan=1>1298/1299</td><td rowspan=1 colspan=1>99.9 (99.6-100)</td></tr></table>

Table 89: Distribution of KPC in clinical specimens   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=4>Comparator Method</td></tr><tr><td rowspan=1 colspan=1>BCID-GN</td><td rowspan=1 colspan=1>Org+/ARG+</td><td rowspan=1 colspan=1>Org+/ARG-</td><td rowspan=1 colspan=1>Org-</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Org+/ARG+</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>8</td></tr><tr><td rowspan=1 colspan=1>Org+/ARG-</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>804</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>809</td></tr><tr><td rowspan=1 colspan=1>Org-</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>92</td><td rowspan=1 colspan=1>109</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>822</td><td rowspan=1 colspan=1>96</td><td rowspan=1 colspan=1>926</td></tr></table>

$\%$ Agreement $( 9 5 \% \mathrm { C I } )$ for $\mathrm { O r g ^ { + / A R G ^ { + } } }$ : $7 / 8 { = } 8 7 . 5 \%$ (52.9-97.8) $\%$ Agreement $( 9 5 \% \mathrm { C I } )$ for $\textstyle \operatorname { O r g } ^ { + / \operatorname { A R G } . }$ -: $8 0 4 / 8 2 2 { = } 9 7 . 8 \%$ (96.6-98.6) $\%$ Agreement $( 9 5 \% \mathrm { C I } )$ for Org-: $9 2 / 9 6 { = } 9 5 . 8 \%$ (89.8-98.4)

Table 90: Clinical performance of KPC target in specimens with associated organisms detected by comparator methods   

<table><tr><td colspan="2" rowspan="2">Species Detected by Comparator Method</td><td colspan="2" rowspan="1">Sensitivity/PPA</td><td colspan="2" rowspan="1">Specificity/NPA</td></tr><tr><td colspan="1" rowspan="1">TP/TP+FN</td><td colspan="1" rowspan="1">% (95% CI)</td><td colspan="1" rowspan="1">TN/TN+FP</td><td colspan="1" rowspan="1">% (95% CI)</td></tr><tr><td colspan="1" rowspan="4">Acinetobacter baumannii</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">---</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">100 (51.0-100)</td></tr><tr><td colspan="1" rowspan="1">Retrospective</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">15/15</td><td colspan="1" rowspan="1">100 (79.6-100)</td></tr><tr><td colspan="1" rowspan="1">Contrived</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">---</td><td colspan="1" rowspan="1">55/55</td><td colspan="1" rowspan="1">100 (93.5-100)</td></tr><tr><td colspan="1" rowspan="1">Combined</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">---</td><td colspan="1" rowspan="1">74/74</td><td colspan="1" rowspan="1">100 (95.1-100)</td></tr><tr><td colspan="1" rowspan="4">Citrobacter</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">---</td><td colspan="1" rowspan="1">5/5</td><td colspan="1" rowspan="1">100 (56.6-100)</td></tr><tr><td colspan="1" rowspan="1">Retrospective</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">---</td><td colspan="1" rowspan="1">21/21</td><td colspan="1" rowspan="1">100 (84.5-100)</td></tr><tr><td colspan="1" rowspan="1">Contrived</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">100 (51.0-100)</td><td colspan="1" rowspan="1">39/39</td><td colspan="1" rowspan="1">100 (91.0-100)</td></tr><tr><td colspan="1" rowspan="1">Combined</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">100 (51.0-100)</td><td colspan="1" rowspan="1">65/65</td><td colspan="1" rowspan="1">100 (94.4-100)</td></tr><tr><td colspan="1" rowspan="4">Cronobacter sakazakii</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">---</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">---</td><td colspan="1" rowspan="1">---</td></tr><tr><td colspan="1" rowspan="1">Retrospective</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">---</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100 (20.7-100)</td></tr><tr><td colspan="1" rowspan="1">Contrived</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">---</td><td colspan="1" rowspan="1">45/45</td><td colspan="1" rowspan="1">100 (92.1-100)</td></tr><tr><td colspan="1" rowspan="1">Combined</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">---</td><td colspan="1" rowspan="1">46/46</td><td colspan="1" rowspan="1">100 (92.3-100)</td></tr><tr><td colspan="1" rowspan="4">Enterobacter (non-cloacae complex)</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">---</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">100 (72.2-100)</td></tr><tr><td colspan="1" rowspan="1">Retrospective</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">---</td><td colspan="1" rowspan="1">12/12</td><td colspan="1" rowspan="1">100 (75.8-100)</td></tr><tr><td colspan="1" rowspan="1">Contrived</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">---</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">100 (90.4-100)</td></tr><tr><td colspan="1" rowspan="1">Combined</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">---</td><td colspan="1" rowspan="1">58/58</td><td colspan="1" rowspan="1">100 (93.8-100)</td></tr><tr><td colspan="1" rowspan="4">Enterobacter cloacae complex</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">---</td><td colspan="1" rowspan="1">19/19</td><td colspan="1" rowspan="1">100 (83.2-100)</td></tr><tr><td colspan="1" rowspan="1">Retrospective</td><td colspan="1" rowspan="1">0/1</td><td colspan="1" rowspan="1">0.0 (0.0-79.3)</td><td colspan="1" rowspan="1">49/49</td><td colspan="1" rowspan="1">100 (92.7-100)</td></tr><tr><td colspan="1" rowspan="1">Contrived</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">100 (43.9-100)</td><td colspan="1" rowspan="1">34/34</td><td colspan="1" rowspan="1">100 (89.8-100)</td></tr><tr><td colspan="1" rowspan="1">Combined</td><td colspan="1" rowspan="1">3/4</td><td colspan="1" rowspan="1">75.0 (30.1-95.4)</td><td colspan="1" rowspan="1">102/102</td><td colspan="1" rowspan="1">100 (96.4-100)</td></tr><tr><td colspan="1" rowspan="4">Escherichia coli</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100 (20.7-100)</td><td colspan="1" rowspan="1">132/132</td><td colspan="1" rowspan="1">100 (97.2-100)</td></tr><tr><td colspan="1" rowspan="1">Retrospective</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">140/140</td><td colspan="1" rowspan="1">100 (97.3-100)</td></tr><tr><td colspan="1" rowspan="1">Contrived</td><td colspan="1" rowspan="1">6/6</td><td colspan="1" rowspan="1">100 (61.0-100)</td><td colspan="1" rowspan="1">46/46</td><td colspan="1" rowspan="1">100 (92.3-100)</td></tr><tr><td colspan="1" rowspan="1">Combined</td><td colspan="1" rowspan="1">7/7</td><td colspan="1" rowspan="1">100 (64.6-100)</td><td colspan="1" rowspan="1">318/318</td><td colspan="1" rowspan="1">100 (98.8-100)</td></tr><tr><td colspan="1" rowspan="4">Klebsiella oxytoca</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">---</td><td colspan="1" rowspan="1">13/13</td><td colspan="1" rowspan="1">100 (77.2-100)</td></tr><tr><td colspan="1" rowspan="1">Retrospective</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">---</td><td colspan="1" rowspan="1">34/34</td><td colspan="1" rowspan="1">100 (89.8-100)</td></tr><tr><td colspan="1" rowspan="1">Contrived</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">---</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">100 (83.9-100)</td></tr><tr><td colspan="1" rowspan="1">Combined</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">---</td><td colspan="1" rowspan="1">67/67</td><td colspan="1" rowspan="1">100 (94.6-100)</td></tr><tr><td colspan="1" rowspan="4">Klebsiella pneumoniae group</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">2/2</td><td colspan="1" rowspan="1">100 (34.2-100)</td><td colspan="1" rowspan="1">59/59</td><td colspan="1" rowspan="1">100 (93.9-100)</td></tr><tr><td colspan="1" rowspan="1">Retrospective</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">100 (51.0-100)</td><td colspan="1" rowspan="1">103/104</td><td colspan="1" rowspan="1">99.0 (94.8-99.8)</td></tr><tr><td colspan="1" rowspan="1">Contrived</td><td colspan="1" rowspan="1">22/22</td><td colspan="1" rowspan="1">100 (85.1-100)</td><td colspan="1" rowspan="1">50/50</td><td colspan="1" rowspan="1">100 (92.9-100)</td></tr><tr><td colspan="1" rowspan="1">Combined</td><td colspan="1" rowspan="1">28/28</td><td colspan="1" rowspan="1">100 (87.9-100)</td><td colspan="1" rowspan="1">212/213</td><td colspan="1" rowspan="1">99.5 (97.4-99.9)</td></tr><tr><td colspan="1" rowspan="4">Morganella morganii</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">100 (43.9-100)</td></tr><tr><td colspan="1" rowspan="1">Retrospective</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">---</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">100 (72.2-100)</td></tr><tr><td colspan="1" rowspan="1">Contrived</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">100 (51.0-100)</td><td colspan="1" rowspan="1">45/45</td><td colspan="1" rowspan="1">100 (92.1-100)</td></tr><tr><td colspan="1" rowspan="1">Combined</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">100 (51.0-100)</td><td colspan="1" rowspan="1">58/58</td><td colspan="1" rowspan="1">100 (93.8-100)</td></tr><tr><td colspan="1" rowspan="4">Proteus</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">---</td><td colspan="1" rowspan="1">23/23</td><td colspan="1" rowspan="1">100 (85.7-100)</td></tr><tr><td colspan="1" rowspan="1">Retrospective</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">55/55</td><td colspan="1" rowspan="1">100 (93.5-100)</td></tr><tr><td colspan="1" rowspan="1">Contrived</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">100 (51.0-100)</td><td colspan="1" rowspan="1">5/5</td><td colspan="1" rowspan="1">100 (56.6-100)</td></tr><tr><td colspan="1" rowspan="1">Combined</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">100 (51.0-100)</td><td colspan="1" rowspan="1">83/83</td><td colspan="1" rowspan="1">100 (95.6-100)</td></tr><tr><td colspan="1" rowspan="4">Proteus mirabilis</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">23/23</td><td colspan="1" rowspan="1">100 (85.7-100)</td></tr><tr><td colspan="1" rowspan="1">Retrospective</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">51/51</td><td colspan="1" rowspan="1">100 (93.0-100)</td></tr><tr><td colspan="1" rowspan="1">Contrived</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">100 (51.0-100)</td><td colspan="1" rowspan="1">5/5</td><td colspan="1" rowspan="1">100 (56.6-100)</td></tr><tr><td colspan="1" rowspan="1">Combined</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">100 (51.0-100)</td><td colspan="1" rowspan="1">79/79</td><td colspan="1" rowspan="1">100 (95.4-100)</td></tr><tr><td colspan="1" rowspan="4">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">28/28</td><td colspan="1" rowspan="1">100 (87.9-100)</td></tr><tr><td colspan="1" rowspan="1">Retrospective</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">---</td><td colspan="1" rowspan="1">60/60</td><td colspan="1" rowspan="1">100 (94.0-100)</td></tr><tr><td colspan="1" rowspan="1">Contrived</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100 (20.7-100)</td><td colspan="1" rowspan="1">31/31</td><td colspan="1" rowspan="1">100 (89.0-100)</td></tr><tr><td colspan="1" rowspan="1">Combined</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100 (20.7-100)</td><td colspan="1" rowspan="1">119/119</td><td colspan="1" rowspan="1">100 (96.9-100)</td></tr><tr><td colspan="1" rowspan="4">Salmonella</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">---</td><td colspan="1" rowspan="1">2/2</td><td colspan="1" rowspan="1">100 (34.2-100)</td></tr><tr><td colspan="1" rowspan="1">Retrospective</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">---</td><td colspan="1" rowspan="1">19/19</td><td colspan="1" rowspan="1">100 (83.2-100)</td></tr><tr><td colspan="1" rowspan="1">Contrived</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">---</td><td colspan="1" rowspan="1">35/35</td><td colspan="1" rowspan="1">100 (90.1-100)</td></tr><tr><td colspan="1" rowspan="1">Combined</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">---</td><td colspan="1" rowspan="1">56/56</td><td colspan="1" rowspan="1">100 (93.6-100)</td></tr><tr><td colspan="1" rowspan="4">Serratia</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">---</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">100 (72.2-100)</td></tr><tr><td colspan="1" rowspan="1">Retrospective</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">---</td><td colspan="1" rowspan="1">34/34</td><td colspan="1" rowspan="1">100 (89.8-100)</td></tr><tr><td colspan="1" rowspan="1">Contrived</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">---</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">100 (90.4-100)</td></tr><tr><td colspan="1" rowspan="1">Combined</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">---</td><td colspan="1" rowspan="1">80/80</td><td colspan="1" rowspan="1">100 (95.4-100)</td></tr><tr><td colspan="1" rowspan="4">Serratia marcescens</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">---</td><td colspan="1" rowspan="1">9/9</td><td colspan="1" rowspan="1">100 (70.1-100)</td></tr><tr><td colspan="1" rowspan="1">Retrospective</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">---</td><td colspan="1" rowspan="1">34/34</td><td colspan="1" rowspan="1">100 (89.8-100)</td></tr><tr><td colspan="1" rowspan="1">Contrived</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">---</td><td colspan="1" rowspan="1">19/19</td><td colspan="1" rowspan="1">100 (83.2-100)</td></tr><tr><td colspan="1" rowspan="1">Combined</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">---</td><td colspan="1" rowspan="1">62/62</td><td colspan="1" rowspan="1">100 (94.2-100)</td></tr></table>

# Clinical performance, NDM:

Table 91 includes the clinical performance for detection of NDM for prospective, retrospective and contrived specimens that contain an applicable gram-negative organism based on the comparator method (for clinical specimens) or the inoculated organism (for contrived specimens). No prospective or retrospective specimens were found to contain IMP by the ePlex BCID-GN Panel or by the comparator method.

Table 92 includes a performance summary for detection of NDM and any applicable organism by the ePlex BCID-GN Panel in comparison to detection/identification by comparator methods. Results include all prospective and retrospective specimens combined.

Table 93 includes the performance for detection of NDM for contrived specimens stratified by organism.

Table 91: Clinical performance, NDM   

<table><tr><td rowspan=2 colspan=1>Target</td><td rowspan=2 colspan=1>Specimen Type</td><td rowspan=1 colspan=2>Sensitivity/PPA</td><td rowspan=1 colspan=2>Specificity/NPA</td></tr><tr><td rowspan=1 colspan=1>TP/TP+FN</td><td rowspan=1 colspan=1>% (95% CI)</td><td rowspan=1 colspan=1>TN/TN+FP</td><td rowspan=1 colspan=1>% (95% CI)</td></tr><tr><td rowspan=7 colspan=1>NDM</td><td rowspan=1 colspan=1>Prospective (Fresh)</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>--</td><td rowspan=1 colspan=1>138/138</td><td rowspan=1 colspan=1>100 (97.3-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective (Frozen)</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>---</td><td rowspan=1 colspan=1>159/159</td><td rowspan=1 colspan=1>100 (97.6-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective (All)</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>---</td><td rowspan=1 colspan=1>297/297</td><td rowspan=1 colspan=1>100 (98.7-100)</td></tr><tr><td rowspan=1 colspan=1>Retrospective</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>532/532</td><td rowspan=1 colspan=1>100 (99.3-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective / Retrospective</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>---</td><td rowspan=1 colspan=1>829/829</td><td rowspan=1 colspan=1>100 (99.5-100)</td></tr><tr><td rowspan=1 colspan=1>Contrived</td><td rowspan=1 colspan=1>54/54</td><td rowspan=1 colspan=1>100 (93.4-100)</td><td rowspan=1 colspan=1>422/422</td><td rowspan=1 colspan=1>100 (99.1-100)</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>54/54</td><td rowspan=1 colspan=1>100 (93.4-100)</td><td rowspan=1 colspan=1>1251/1251</td><td rowspan=1 colspan=1>100 (99.7-100)</td></tr></table>

Table 92: Distribution of NDM in clinical specimens   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=4>Comparator Method</td></tr><tr><td rowspan=1 colspan=1>BCID-GN</td><td rowspan=1 colspan=1>Org+/ARG+</td><td rowspan=1 colspan=1>Org+/ARG-</td><td rowspan=1 colspan=1>Org-</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Org+/ARG+</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Org+/ARG-</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>812</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>816</td></tr><tr><td rowspan=1 colspan=1>Org-</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>93</td><td rowspan=1 colspan=1>110</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>829</td><td rowspan=1 colspan=1>97</td><td rowspan=1 colspan=1>926</td></tr></table>

$\%$ Agreement $( 9 5 \% \mathrm { C I } )$ for $\mathrm { O r g ^ { + / A R G ^ { + } } }$ : $0 / 0 { = } \mathbf { N } / \mathbf { A }$ $\%$ Agreement $( 9 5 \% \mathrm { C I } )$ for Org+/ARG-: $8 1 2 / 8 2 9 { = } 9 7 . 9 \%$ (96.7-98.7) $\%$ Agreement $( 9 5 \% \mathrm { C I } )$ for Org-: $9 3 / 9 7 { = } 9 5 . 9 \%$ (89.9-98.4)

Table 93: Clinical performance of NDM Target in contrived specimens by organism   

<table><tr><td colspan="1" rowspan="2">Species Detected by Comparator Method</td><td colspan="2" rowspan="1">Sensitivity/PPA</td><td colspan="2" rowspan="1">Specificity/NPA</td></tr><tr><td colspan="1" rowspan="1">TP/TP+FN</td><td colspan="1" rowspan="1">% (95% CI)</td><td colspan="1" rowspan="1">TN/TN+FP</td><td colspan="1" rowspan="1">% (95% CI)</td></tr><tr><td colspan="1" rowspan="1">Acinetobacter baumannii</td><td colspan="1" rowspan="1">5/5</td><td colspan="1" rowspan="1">100 (56.6-100)</td><td colspan="1" rowspan="1">69/69</td><td colspan="1" rowspan="1">100 (94.7-100)</td></tr><tr><td colspan="1" rowspan="1">Citrobacter</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">---</td><td colspan="1" rowspan="1">69/69</td><td colspan="1" rowspan="1">100 (94.7-100)</td></tr><tr><td colspan="1" rowspan="1">Enterobacter (non-cloacae complex)</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">---</td><td colspan="1" rowspan="1">58/58</td><td colspan="1" rowspan="1">100 (93.8-100)</td></tr><tr><td colspan="1" rowspan="1">Enterobacter cloacae complex</td><td colspan="1" rowspan="1">13/13</td><td colspan="1" rowspan="1">100 (77.2-100)</td><td colspan="1" rowspan="1">93/93</td><td colspan="1" rowspan="1">100 (96.0-100)</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">23/23</td><td colspan="1" rowspan="1">100 (85.7-100)</td><td colspan="1" rowspan="1">302/302</td><td colspan="1" rowspan="1">100 (98.7-100)</td></tr><tr><td colspan="1" rowspan="1">Klebsiella oxytoca</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">---</td><td colspan="1" rowspan="1">67/67</td><td colspan="1" rowspan="1">100 (94.6-100)</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae group</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">100 (43.9-100)</td><td colspan="1" rowspan="1">238/238</td><td colspan="1" rowspan="1">100 (98.4-100)</td></tr><tr><td colspan="1" rowspan="1">Morganella morganii</td><td colspan="1" rowspan="1">5/5</td><td colspan="1" rowspan="1">100 (56.6-100)</td><td colspan="1" rowspan="1">57/57</td><td colspan="1" rowspan="1">100 (93.7-100)</td></tr><tr><td colspan="1" rowspan="1">Proteus</td><td colspan="1" rowspan="1">5/5</td><td colspan="1" rowspan="1">100 (56.6-100)</td><td colspan="1" rowspan="1">82/82</td><td colspan="1" rowspan="1">100 (95.5-100)</td></tr><tr><td colspan="1" rowspan="1">Proteus mirabilis</td><td colspan="1" rowspan="1">5/5</td><td colspan="1" rowspan="1">100 (56.6-100)</td><td colspan="1" rowspan="1">78/78</td><td colspan="1" rowspan="1">100 (95.3-100)</td></tr><tr><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">120/120</td><td colspan="1" rowspan="1">100 (96.9-100)</td></tr><tr><td colspan="1" rowspan="1">Salmonella</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">56/56</td><td colspan="1" rowspan="1">100 (93.6-100)</td></tr><tr><td colspan="1" rowspan="1">Serratia</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">---</td><td colspan="1" rowspan="1">80/80</td><td colspan="1" rowspan="1">100 (95.4-100)</td></tr><tr><td colspan="1" rowspan="1">Serratia marcescens</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">---</td><td colspan="1" rowspan="1">62/62</td><td colspan="1" rowspan="1">100 (94.2-100)</td></tr></table>

Clinical Performance, OXA: Note that the OXA assay on the BCID-GN Panel is designed to detect OXA-23 and OXA-48 only.

Table 94 includes the clinical performance for OXA for prospective, retrospective and contrived specimens that contain an applicable gram-negative organism based on the comparator method (for clinical specimens) or the inoculated organism (for contrived specimens).

Table 95 includes the clinical performance summary for detection of OXA and any applicable organism by the BCID-GN Panel in comparison to detection/identification by comparator methods. Results include all prospective and retrospective specimens combined.

Table 96 includes the clinical performance for detection of OXA for prospective, retrospective and contrived specimens stratified by organism identified by the comparator method (for clinical specimens) and based on the organism used for inoculation (for contrived specimens).

Table 94: Clinical performance, OXA   

<table><tr><td rowspan=2 colspan=1>Target</td><td rowspan=2 colspan=1>Specimen Type</td><td rowspan=1 colspan=2>Sensitivity/PPA</td><td rowspan=1 colspan=2>Specificity/NPA</td></tr><tr><td rowspan=1 colspan=1>TP/TP+FN</td><td rowspan=1 colspan=1>% (95% CI)</td><td rowspan=1 colspan=1>TN/TN+FP</td><td rowspan=1 colspan=1>% (95% CI)</td></tr><tr><td rowspan=7 colspan=1>OXA</td><td rowspan=1 colspan=1>Prospective (Fresh)</td><td rowspan=1 colspan=1>0/1</td><td rowspan=1 colspan=1>0.0 (0.0-79.3)</td><td rowspan=1 colspan=1>137/137</td><td rowspan=1 colspan=1>100 (97.3-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective (Frozen)</td><td rowspan=1 colspan=1>1/1</td><td rowspan=1 colspan=1>100 (20.7-100)</td><td rowspan=1 colspan=1>158/158</td><td rowspan=1 colspan=1>100 (97.6-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective (All)</td><td rowspan=1 colspan=1>1/2</td><td rowspan=1 colspan=1>50.0 (9.5-90.5)</td><td rowspan=1 colspan=1>295/295</td><td rowspan=1 colspan=1>100 (98.7-100)</td></tr><tr><td rowspan=1 colspan=1>Retrospective</td><td rowspan=1 colspan=1>9/11</td><td rowspan=1 colspan=1>81.8 (52.3-94.9)</td><td rowspan=1 colspan=1>519/521</td><td rowspan=1 colspan=1>99.6 (98.6-99.9)</td></tr><tr><td rowspan=1 colspan=1>Prospective / Retrospective</td><td rowspan=1 colspan=1>10/13A</td><td rowspan=1 colspan=1>76.9 (49.7-91.8)</td><td rowspan=1 colspan=1>814/816</td><td rowspan=1 colspan=1>99.8 (99.1-99.9)</td></tr><tr><td rowspan=1 colspan=1>Contrived</td><td rowspan=1 colspan=1>37/37</td><td rowspan=1 colspan=1>100 (90.6-100)</td><td rowspan=1 colspan=1>439/439</td><td rowspan=1 colspan=1>100 (99.1-100)</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>47/50</td><td rowspan=1 colspan=1>94.0 (83.8-97.9)</td><td rowspan=1 colspan=1>1253/1255</td><td rowspan=1 colspan=1>99.8 (99.4-100)</td></tr></table>

A. In 1/3 false negative specimens, OXA signal was above the threshold for detection; however, an associated organism was not detected by the BCID-GN Panel and the OXA target was reported as $\mathbf { \hat { N } } / \mathbf { A } ^ { \prime }$ . One additional false negative specimen was tested with an FDAcleared multiplex assay and OXA was not detected. The isolate from the remaining false negative specimen tested negative for OXA23 and OXA-48 by qPCR.

Table 95: Distribution of OXA in clinical specimens   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=4>Comparator Method</td></tr><tr><td rowspan=1 colspan=1>BCID-GN</td><td rowspan=1 colspan=1>Org+/ARG+</td><td rowspan=1 colspan=1>Org+/ARG-</td><td rowspan=1 colspan=1>Org-</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Org+/ARG+</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>12</td></tr><tr><td rowspan=1 colspan=1>Org+/ARG-</td><td rowspan=1 colspan=1>2A</td><td rowspan=1 colspan=1>798</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>804</td></tr><tr><td rowspan=1 colspan=1>Org-</td><td rowspan=1 colspan=1>1B</td><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>93</td><td rowspan=1 colspan=1>110</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>816</td><td rowspan=1 colspan=1>97</td><td rowspan=1 colspan=1>926</td></tr></table>

$\%$ Agreement $( 9 5 \%$ CI) for $\mathrm { O r g ^ { + / A R G ^ { + } } }$ : $1 0 / 1 3 { = } 7 6 . 9 \%$ (49.7-91.8) $\%$ Agreement $( 9 5 \% \mathrm { C I } )$ for $\textstyle \operatorname { O r g } ^ { + / \operatorname { A R G } . }$ -: $7 9 8 / 8 1 6 { = } 9 7 . 8 \%$ (96.5-98.6) $\%$ Agreement $9 5 \%$ CI) for Org-: $9 3 / 9 7 { = } 9 5 . 9 \%$ (89.9-98.4)

A. One false negative specimen was tested with an FDA-cleared multiplex assay and OXA was not detected. The isolate from the remaining false negative specimen tested negative for OXA-23 and OXA-48 by qPCR. B. For this specimen, OXA signal was above the threshold for detection; however, an associated organism was not detected by the ePlex BCID-GN Panel and the OXA target was reported as ‘N/A’.

Table 96: Clinical performance of OXA target in specimens with associated organisms detected by comparator methods   

<table><tr><td colspan="2" rowspan="2">Species Detected by Comparator Method</td><td colspan="2" rowspan="1">Sensitivity/PPA</td><td colspan="2" rowspan="1">Specificity/NPA</td></tr><tr><td colspan="1" rowspan="1">TP/TP+FN</td><td colspan="1" rowspan="1">% (95% CI)</td><td colspan="1" rowspan="1">TN/TN+FP</td><td colspan="1" rowspan="1">% (95% CI)</td></tr><tr><td colspan="1" rowspan="4">Acinetobacter baumannii</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100 (20.7-100)</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">100 (43.9-100)</td></tr><tr><td colspan="1" rowspan="1">Retrospective</td><td colspan="1" rowspan="1">7/7</td><td colspan="1" rowspan="1">100 (64.6-100)</td><td colspan="1" rowspan="1">8/8</td><td colspan="1" rowspan="1">100 (67.6-100)</td></tr><tr><td colspan="1" rowspan="1">Contrived</td><td colspan="1" rowspan="1">13/13</td><td colspan="1" rowspan="1">100 (77.2-100)</td><td colspan="1" rowspan="1">42/42</td><td colspan="1" rowspan="1">100 (91.6-100)</td></tr><tr><td colspan="1" rowspan="1">Combined</td><td colspan="1" rowspan="1">21/21</td><td colspan="1" rowspan="1">100 (84.5-100)</td><td colspan="1" rowspan="1">53/53</td><td colspan="1" rowspan="1">100 (93.2-100)</td></tr><tr><td colspan="1" rowspan="4">Citrobacter</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">5/5</td><td colspan="1" rowspan="1">100 (56.6-100)</td></tr><tr><td colspan="1" rowspan="1">Retrospective</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">---</td><td colspan="1" rowspan="1">21/21</td><td colspan="1" rowspan="1">100 (84.5-100)</td></tr><tr><td colspan="1" rowspan="1">Contrived</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">---</td><td colspan="1" rowspan="1">43/43</td><td colspan="1" rowspan="1">100 (91.8-100)</td></tr><tr><td colspan="1" rowspan="1">Combined</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">---</td><td colspan="1" rowspan="1">69/69</td><td colspan="1" rowspan="1">100 (94.7-100)</td></tr><tr><td colspan="1" rowspan="4">Enterobacter (non-cloacaecomplex)</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">---</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">100 (72.2-100)</td></tr><tr><td colspan="1" rowspan="1">Retrospective</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">---</td><td colspan="1" rowspan="1">12/12</td><td colspan="1" rowspan="1">100 (75.8-100)</td></tr><tr><td colspan="1" rowspan="1">Contrived</td><td colspan="1" rowspan="1">12/12</td><td colspan="1" rowspan="1">100 (75.8-100)</td><td colspan="1" rowspan="1">24/24</td><td colspan="1" rowspan="1">100 (86.2-100)</td></tr><tr><td colspan="1" rowspan="1">Combined</td><td colspan="1" rowspan="1">12/12</td><td colspan="1" rowspan="1">100 (75.8-100)</td><td colspan="1" rowspan="1">46/46</td><td colspan="1" rowspan="1">100 (92.3-100)</td></tr><tr><td colspan="1" rowspan="4">Enterobacter cloacae complex</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">---</td><td colspan="1" rowspan="1">19/19</td><td colspan="1" rowspan="1">100 (83.2-100)</td></tr><tr><td colspan="1" rowspan="1">Retrospective</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">---</td><td colspan="1" rowspan="1">50/50</td><td colspan="1" rowspan="1">100 (92.9-100)</td></tr><tr><td colspan="1" rowspan="1">Contrived</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">--</td><td colspan="1" rowspan="1">37/37</td><td colspan="1" rowspan="1">100 (90.6-100)</td></tr><tr><td colspan="1" rowspan="1">Combined</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">---</td><td colspan="1" rowspan="1">106/106</td><td colspan="1" rowspan="1">100 (96.5-100)</td></tr><tr><td colspan="1" rowspan="4">Escherichia coli</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">0/1</td><td colspan="1" rowspan="1">0.0 (0.0-79.3)</td><td colspan="1" rowspan="1">132/132</td><td colspan="1" rowspan="1">100 (97.2-100)</td></tr><tr><td colspan="1" rowspan="1">Retrospective</td><td colspan="1" rowspan="1">1/2</td><td colspan="1" rowspan="1">50.0 (9.5-90.5)</td><td colspan="1" rowspan="1">138/138</td><td colspan="1" rowspan="1">100 (97.3-100)</td></tr><tr><td colspan="1" rowspan="1">Contrived</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">---</td><td colspan="1" rowspan="1">52/52</td><td colspan="1" rowspan="1">100 (93.1-100)</td></tr><tr><td colspan="1" rowspan="1">Combined</td><td colspan="1" rowspan="1">1/3</td><td colspan="1" rowspan="1">33.3 (6.1-79.2)</td><td colspan="1" rowspan="1">322/322</td><td colspan="1" rowspan="1">100 (98.8-100)</td></tr><tr><td colspan="1" rowspan="4">Klebsiella oxytoca</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">--</td><td colspan="1" rowspan="1">13/13</td><td colspan="1" rowspan="1">100 (77.2-100)</td></tr><tr><td colspan="1" rowspan="1">Retrospective</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">34/34</td><td colspan="1" rowspan="1">100 (89.8-100)</td></tr><tr><td colspan="1" rowspan="1">Contrived</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">---</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">100 (83.9-100)</td></tr><tr><td colspan="1" rowspan="1">Combined</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">---</td><td colspan="1" rowspan="1">67/67</td><td colspan="1" rowspan="1">100 (94.6-100)</td></tr><tr><td colspan="1" rowspan="4">Klebsiella pneumoniae group</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">---</td><td colspan="1" rowspan="1">61/61</td><td colspan="1" rowspan="1">100 (94.1-100)</td></tr><tr><td colspan="1" rowspan="1">Retrospective</td><td colspan="1" rowspan="1">0/1</td><td colspan="1" rowspan="1">0.0 (0.0-79.3)</td><td colspan="1" rowspan="1">107/107</td><td colspan="1" rowspan="1">100 (96.5-100)</td></tr><tr><td colspan="1" rowspan="1">Contrived</td><td colspan="1" rowspan="1">12/12</td><td colspan="1" rowspan="1">100 (75.8-100)</td><td colspan="1" rowspan="1">60/60</td><td colspan="1" rowspan="1">100 (94.0-100)</td></tr><tr><td colspan="1" rowspan="1">Combined</td><td colspan="1" rowspan="1">12/13</td><td colspan="1" rowspan="1">92.3 (66.7-98.6)</td><td colspan="1" rowspan="1">228/228</td><td colspan="1" rowspan="1">100 (98.3-100)</td></tr><tr><td colspan="1" rowspan="2">Morganella morganii</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">--</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">100 (43.9-100)</td></tr><tr><td colspan="1" rowspan="1">Retrospective</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">---</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">100 (72.2-100)</td></tr><tr><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="1">Contrived</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">---</td><td colspan="1" rowspan="1">49/49</td><td colspan="1" rowspan="1">100 (92.7-100)</td></tr><tr><td colspan="1" rowspan="1">Combined</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">---</td><td colspan="1" rowspan="1">62/62</td><td colspan="1" rowspan="1">100 (94.2-100)</td></tr><tr><td colspan="1" rowspan="4">Proteus</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">---</td><td colspan="1" rowspan="1">23/23</td><td colspan="1" rowspan="1">100 (85.7-100)</td></tr><tr><td colspan="1" rowspan="1">Retrospective</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100 (20.7-100)</td><td colspan="1" rowspan="1">53/54</td><td colspan="1" rowspan="1">98.1 (90.2-99.7)</td></tr><tr><td colspan="1" rowspan="1">Contrived</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">---</td><td colspan="1" rowspan="1">9/9</td><td colspan="1" rowspan="1">100 (70.1-100)</td></tr><tr><td colspan="1" rowspan="1">Combined</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100 (20.7-100)</td><td colspan="1" rowspan="1">85/86</td><td colspan="1" rowspan="1">98.8 (93.7-99.8)</td></tr><tr><td colspan="1" rowspan="4">Proteus mirabilis</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">---</td><td colspan="1" rowspan="1">23/23</td><td colspan="1" rowspan="1">100 (85.7-100)</td></tr><tr><td colspan="1" rowspan="1">Retrospective</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">---</td><td colspan="1" rowspan="1">50/51</td><td colspan="1" rowspan="1">98.0 (89.7-99.7)</td></tr><tr><td colspan="1" rowspan="1">Contrived</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">---</td><td colspan="1" rowspan="1">9/9</td><td colspan="1" rowspan="1">100 (70.1-100)</td></tr><tr><td colspan="1" rowspan="1">Combined</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">---</td><td colspan="1" rowspan="1">82/83</td><td colspan="1" rowspan="1">98.8 (93.5-99.8)</td></tr><tr><td colspan="1" rowspan="4">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">--</td><td colspan="1" rowspan="1">28/28</td><td colspan="1" rowspan="1">100 (87.9-100)</td></tr><tr><td colspan="1" rowspan="1">Retrospective</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">---</td><td colspan="1" rowspan="1">59/60</td><td colspan="1" rowspan="1">98.3 (91.1-99.7)</td></tr><tr><td colspan="1" rowspan="1">Contrived</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">--</td><td colspan="1" rowspan="1">32/32</td><td colspan="1" rowspan="1">100 (89.3-100)</td></tr><tr><td colspan="1" rowspan="1">Combined</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">---</td><td colspan="1" rowspan="1">119/120</td><td colspan="1" rowspan="1">99.2 (95.4-99.9)</td></tr><tr><td colspan="1" rowspan="4">Salmonella</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">---</td><td colspan="1" rowspan="1">2/2</td><td colspan="1" rowspan="1">100 (34.2-100)</td></tr><tr><td colspan="1" rowspan="1">Retrospective</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">---</td><td colspan="1" rowspan="1">19/19</td><td colspan="1" rowspan="1">100 (83.2-100)</td></tr><tr><td colspan="1" rowspan="1">Contrived</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">---</td><td colspan="1" rowspan="1">35/35</td><td colspan="1" rowspan="1">100 (90.1-100)</td></tr><tr><td colspan="1" rowspan="1">Combined</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">---</td><td colspan="1" rowspan="1">56/56</td><td colspan="1" rowspan="1">100 (93.6-100)</td></tr><tr><td colspan="1" rowspan="4">Serratia</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">---</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">100 (72.2-100)</td></tr><tr><td colspan="1" rowspan="1">Retrospective</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">---</td><td colspan="1" rowspan="1">34/34</td><td colspan="1" rowspan="1">100 (89.8-100)</td></tr><tr><td colspan="1" rowspan="1">Contrived</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">--</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">100 (90.4-100)</td></tr><tr><td colspan="1" rowspan="1">Combined</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">---</td><td colspan="1" rowspan="1">80/80</td><td colspan="1" rowspan="1">100 (95.4-100)</td></tr><tr><td colspan="1" rowspan="4">Serratia marcescens</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">---</td><td colspan="1" rowspan="1">9/9</td><td colspan="1" rowspan="1">100 (70.1-100)</td></tr><tr><td colspan="1" rowspan="1">Retrospective</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">---</td><td colspan="1" rowspan="1">34/34</td><td colspan="1" rowspan="1">100 (89.8-100)</td></tr><tr><td colspan="1" rowspan="1">Contrived</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">---</td><td colspan="1" rowspan="1">19/19</td><td colspan="1" rowspan="1">100 (83.2-100)</td></tr><tr><td colspan="1" rowspan="1">Combined</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">---</td><td colspan="1" rowspan="1">62/62</td><td colspan="1" rowspan="1">100 (94.2-100)</td></tr></table>

Clinical performance, VIM:

Table 97 includes the clinical performance for detection of VIM for prospective, retrospective and contrived specimens that contain an applicable gram-negative organism based on the comparator method (for clinical specimens) or the inoculated organism (for contrived specimens). No prospective or retrospective specimens were found to contain VIM by the ePlex BCID-GN Panel or by the comparator method.

Table 98 includes a performance summary for detection of VIM and any applicable organism by the ePlex BCID-GN Panel in comparison to detection/identification by comparator methods. Results include all prospective and retrospective specimens combined.

Table 99 includes the performance for detection of VIM for contrived specimens stratified by organism.

Table 97: Clinical performance, VIM   

<table><tr><td colspan="1" rowspan="2">Target</td><td colspan="1" rowspan="2">Specimen Type</td><td colspan="2" rowspan="1">Sensitivity/PPA</td><td colspan="2" rowspan="1">Specificity/NPA</td></tr><tr><td colspan="1" rowspan="1">TP/TP+FN</td><td colspan="1" rowspan="1">% (95% CI)</td><td colspan="1" rowspan="1">TN/TN+FP</td><td colspan="1" rowspan="1">% (95% CI)</td></tr><tr><td colspan="1" rowspan="1">VIM</td><td colspan="1" rowspan="1">Prospective (Fresh)</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">---</td><td colspan="1" rowspan="1">138/138</td><td colspan="1" rowspan="1">100 (97.3-100)</td></tr><tr><td colspan="1" rowspan="6"></td><td colspan="1" rowspan="1">Prospective (Frozen)</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">---</td><td colspan="1" rowspan="1">159/159</td><td colspan="1" rowspan="1">100 (97.6-100)</td></tr><tr><td colspan="1" rowspan="1">Prospective (All)</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">---</td><td colspan="1" rowspan="1">297/297</td><td colspan="1" rowspan="1">100 (98.7-100)</td></tr><tr><td colspan="1" rowspan="1">Retrospective</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">532/532</td><td colspan="1" rowspan="1">100 (99.3-100)</td></tr><tr><td colspan="1" rowspan="1">Prospective / Retrospective</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">---</td><td colspan="1" rowspan="1">829/829</td><td colspan="1" rowspan="1">100 (99.5-100)</td></tr><tr><td colspan="1" rowspan="1">Contrived</td><td colspan="1" rowspan="1">42/42</td><td colspan="1" rowspan="1">100 (91.6-100)</td><td colspan="1" rowspan="1">434/434</td><td colspan="1" rowspan="1">100 (99.1-100)</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">42/42</td><td colspan="1" rowspan="1">100 (91.6-100)</td><td colspan="1" rowspan="1">1263/1263</td><td colspan="1" rowspan="1">100 (99.7-100)</td></tr></table>

Table 98: Distribution of VIM in clinical specimens   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=4>Comparator Method</td></tr><tr><td rowspan=1 colspan=1>BCID-GN</td><td rowspan=1 colspan=1>Org+/ARG+</td><td rowspan=1 colspan=1>Org+/ARG-</td><td rowspan=1 colspan=1>Org-</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Org+/ARG+</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Org+/ARG-</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>812</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>816</td></tr><tr><td rowspan=1 colspan=1>Org-</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>93</td><td rowspan=1 colspan=1>110</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>829</td><td rowspan=1 colspan=1>97</td><td rowspan=1 colspan=1>926</td></tr></table>

$\%$ Agreement $( 9 5 \%$ CI) for $\mathrm { O r g ^ { + / A R G ^ { + } } }$ : $0 / 0 { = } \mathbf { N } / \mathbf { A }$ $\%$ Agreement $( 9 5 \% \mathrm { C I } )$ for Org+/ARG-: $8 1 2 / 8 2 9 { = } 9 7 . 9 \%$ (96.7-98.7) $\%$ Agreement $9 5 \%$ CI) for Org-: $9 3 / 9 7 { = } 9 5 . 9 \%$ (89.9-98.4)

Table 99: Clinical performance of VIM target in contrived specimens by organism   

<table><tr><td rowspan=2 colspan=1>Species Detected by Comparator Method</td><td rowspan=1 colspan=2>Sensitivity/PPA</td><td rowspan=1 colspan=2>Specificity/NPA</td></tr><tr><td rowspan=1 colspan=1>TP/TP+FN</td><td rowspan=1 colspan=1>% (95% CI)</td><td rowspan=1 colspan=1>TN/TN+FP</td><td rowspan=1 colspan=1>% (95% CI)</td></tr><tr><td rowspan=1 colspan=1>Acinetobacter baumannii</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>--</td><td rowspan=1 colspan=1>74/74</td><td rowspan=1 colspan=1>100 (95.1-100)</td></tr><tr><td rowspan=1 colspan=1>Citrobacter</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>---</td><td rowspan=1 colspan=1>69/69</td><td rowspan=1 colspan=1>100 (94.7-100)</td></tr><tr><td rowspan=1 colspan=1>Enterobacter (non-cloacae complex)</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>---</td><td rowspan=1 colspan=1>58/58</td><td rowspan=1 colspan=1>100 (93.8-100)</td></tr><tr><td rowspan=1 colspan=1>Enterobacter cloacae complex</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>100 (61.0-100)</td><td rowspan=1 colspan=1>100/100</td><td rowspan=1 colspan=1>100 (96.3-100)</td></tr><tr><td rowspan=1 colspan=1>Escherichia coli</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>100 (34.2-100)</td><td rowspan=1 colspan=1>323/323</td><td rowspan=1 colspan=1>100 (98.8-100)</td></tr><tr><td rowspan=1 colspan=1>Klebsiella oxytoca</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>---</td><td rowspan=1 colspan=1>67/67</td><td rowspan=1 colspan=1>100 (94.6-100)</td></tr><tr><td rowspan=1 colspan=1>Klebsiella pneumoniae group</td><td rowspan=1 colspan=1>16/16</td><td rowspan=1 colspan=1>100 (80.6-100)</td><td rowspan=1 colspan=1>225/225</td><td rowspan=1 colspan=1>100 (98.3-100)</td></tr><tr><td rowspan=1 colspan=1>Morganella morganii</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>---</td><td rowspan=1 colspan=1>62/62</td><td rowspan=1 colspan=1>100 (94.2-100)</td></tr><tr><td rowspan=1 colspan=1>Proteus</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>87/87</td><td rowspan=1 colspan=1>100 (95.8-100)</td></tr><tr><td rowspan=1 colspan=1>Proteus mirabilis</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>---</td><td rowspan=1 colspan=1>83/83</td><td rowspan=1 colspan=1>100 (95.6-100)</td></tr><tr><td rowspan=1 colspan=1>Pseudomonas aeruginosa</td><td rowspan=1 colspan=1>18/18</td><td rowspan=1 colspan=1>100 (82.4-100)</td><td rowspan=1 colspan=1>102/102</td><td rowspan=1 colspan=1>100 (96.4-100)</td></tr><tr><td rowspan=1 colspan=1>Salmonella</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>---</td><td rowspan=1 colspan=1>56/56</td><td rowspan=1 colspan=1>100 (93.6-100)</td></tr><tr><td rowspan=1 colspan=1>Serratia</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>---</td><td rowspan=1 colspan=1>80/80</td><td rowspan=1 colspan=1>100 (95.4-100)</td></tr><tr><td rowspan=1 colspan=1>Serratia marcescens</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>---</td><td rowspan=1 colspan=1>62/62</td><td rowspan=1 colspan=1>100 (94.2-100)</td></tr></table>

# Resistance markers, comparison to antimicrobial susceptibility testing

A supplemental comparison of the CTX-M BCID-GN Panel target versus phenotypic antimicrobial susceptibility testing (AST) results for extended spectrum beta-lactamase (ESBL) activity, Ceftazidime, Ceftriaxone, and Aztreonam, and a combination of the four results (ESBL/Combo) is provided in Table 100 for clinical isolates with available AST results. In total, 162 isolates had ESBL confirmatory testing and 770 isolates had ESBL confirmatory testing and/or AST results for one or more of Ceftazidime, Ceftriaxone, or

Aztreonam. A true positive (TP) result was defined where CTX-M was detected by the BCID-GN Panel and the isolate was positive for the ESBL confirmatory test or resistant (R) or intermediate (I) to the specific antimicrobial. A false negative (FN) result was defined similarly when CTX-M was not detected by the BCID-GN Panel. PPA was calculated as 100 x (TP/(TP+FN)). A true negative (TN) result was defined where CTX-M was not detected by the BCID-GN Panel and the isolate was negative for the ESBL confirmatory test or susceptible (S) to the specific antimicrobial. A false positive (FP) result was defined similarly when CTX-M was detected by the BCID-GN Panel and the isolate was susceptible to the specific antimicrobial. NPA was calculated as $1 0 0 \mathrm { { x } }$ (TN/(TN+FP)).

For the ESBL/Combo analysis, the phenotypic AST result was positive or negative based on the ESBL confirmatory test, if available. If the ESBL confirmatory test was not available, then the phenotypic AST result was considered positive if any of the three antimicrobials were resistant or intermediate, otherwise, if AST results were susceptible, the result was considered negative. Note: ESBL resistance can be due to mechanisms other than acquisition of the CTX-M resistance gene.

Table 100: BCID-GN Panel target CTX-M target, comparison to phenotypic AST results for ESBL confirmatory testing, Ceftazidime, Ceftriaxone, and Aztreonam   

<table><tr><td colspan="1" rowspan="2">AssociatedOrganism</td><td colspan="2" rowspan="1">ESBL confirmatory</td><td colspan="2" rowspan="1">Ceftazidime</td><td colspan="2" rowspan="1">Ceftriaxone</td><td colspan="2" rowspan="1">Aztreonam</td><td colspan="2" rowspan="1">ESBL/Combo</td></tr><tr><td colspan="1" rowspan="1">PPATP/TP+FN(%)</td><td colspan="1" rowspan="1">NPATN/TN+FP(%)</td><td colspan="1" rowspan="1">PPATP/TP+FN(%)</td><td colspan="1" rowspan="1">NPATN/TN+FP(%)</td><td colspan="1" rowspan="1">PPATP/TP+FN(%)</td><td colspan="1" rowspan="1">NPATN/TN+FP(%)</td><td colspan="1" rowspan="1">PPATP/TP+FN(%)</td><td colspan="1" rowspan="1">NPATN/TN+FP(%)</td><td colspan="1" rowspan="1">PPATP/TP+FN(%)</td><td colspan="1" rowspan="1">NPATN/TN+FP(%)</td></tr><tr><td colspan="1" rowspan="1">A. baumannii</td><td colspan="1" rowspan="1">---</td><td colspan="1" rowspan="1">---</td><td colspan="1" rowspan="1">0/10(0.0%)</td><td colspan="1" rowspan="1">3/3(100%)</td><td colspan="1" rowspan="1">0/11(0.0%)</td><td colspan="1" rowspan="1">1/1(100%)</td><td colspan="1" rowspan="1">0/3(0.0%)</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">0/15(0.0%)</td><td colspan="1" rowspan="1">2/2(100%)</td></tr><tr><td colspan="1" rowspan="1">Citrobacter</td><td colspan="1" rowspan="1">---</td><td colspan="1" rowspan="1">---</td><td colspan="1" rowspan="1">0/2(0.0%)</td><td colspan="1" rowspan="1">14/15(93.3%)</td><td colspan="1" rowspan="1">0/3(0.0%)</td><td colspan="1" rowspan="1">21/22(95.5%)</td><td colspan="1" rowspan="1">0/1(0.0%)</td><td colspan="1" rowspan="1">16/17(94.1%)</td><td colspan="1" rowspan="1">0/3(0.0%)</td><td colspan="1" rowspan="1">21/22(95.5%)</td></tr><tr><td colspan="1" rowspan="1">Enterobacter</td><td colspan="1" rowspan="1">---</td><td colspan="1" rowspan="1">---</td><td colspan="1" rowspan="1">0/2(0.0%)</td><td colspan="1" rowspan="1">12/12(100%)</td><td colspan="1" rowspan="1">0/2(0.0%)</td><td colspan="1" rowspan="1">18/18(100%)</td><td colspan="1" rowspan="1">0/1(0.0%)</td><td colspan="1" rowspan="1">9/9(100%)</td><td colspan="1" rowspan="1">0/3(0.0%)</td><td colspan="1" rowspan="1">17/17(100%)</td></tr><tr><td colspan="1" rowspan="1">E. cloacaecomplex</td><td colspan="1" rowspan="1">---</td><td colspan="1" rowspan="1">---</td><td colspan="1" rowspan="1">0/10(0.0%)</td><td colspan="1" rowspan="1">30/30(100%)</td><td colspan="1" rowspan="1">0/17(0.0%)</td><td colspan="1" rowspan="1">39/39(100%)</td><td colspan="1" rowspan="1">0/5(0.0%)</td><td colspan="1" rowspan="1">25/25(100%)</td><td colspan="1" rowspan="1">0/18(0.0%)</td><td colspan="1" rowspan="1">39/39(100%)</td></tr><tr><td colspan="1" rowspan="1">E. col</td><td colspan="1" rowspan="1">28/30(93.3%)</td><td colspan="1" rowspan="1">82/82(100%)</td><td colspan="1" rowspan="1">29/38(76.3%)</td><td colspan="1" rowspan="1">159/167(95.2%)</td><td colspan="1" rowspan="1">47/59(79.7%)</td><td colspan="1" rowspan="1">180/181(99.4%)</td><td colspan="1" rowspan="1">23/27(85.2%)</td><td colspan="1" rowspan="1">118/120(98.3%)</td><td colspan="1" rowspan="1">47/53(88.7%)</td><td colspan="1" rowspan="1">196/197(99.5%)</td></tr><tr><td colspan="1" rowspan="1">K. oxytoca</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">10/10(100%)</td><td colspan="1" rowspan="1">0/1(0.0%)</td><td colspan="1" rowspan="1">29/29(100%)</td><td colspan="1" rowspan="1">0/2(0.0%)</td><td colspan="1" rowspan="1">40/40(100%)</td><td colspan="1" rowspan="1">0/2(0.0%)</td><td colspan="1" rowspan="1">20/20(100%)</td><td colspan="1" rowspan="1">0/2(0.0%)</td><td colspan="1" rowspan="1">41/41(100%)</td></tr><tr><td colspan="1" rowspan="1">K.pneumoniaegroup</td><td colspan="1" rowspan="1">7/11(63.6%)</td><td colspan="1" rowspan="1">25/25(100%)</td><td colspan="1" rowspan="1">13/23(56.5%)</td><td colspan="1" rowspan="1">83/83(100%)</td><td colspan="1" rowspan="1">17/27(63.0%)</td><td colspan="1" rowspan="1">115/115(100%)</td><td colspan="1" rowspan="1">10/12(83.3%)</td><td colspan="1" rowspan="1">72/72(100%)</td><td colspan="1" rowspan="1">17/26(65.4%)</td><td colspan="1" rowspan="1">119/119(100%)</td></tr><tr><td colspan="1" rowspan="1">M. morganii</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">0/1(0.0%)</td><td colspan="1" rowspan="1">6/6(100%)</td><td colspan="1" rowspan="1">0/2(0.0%)</td><td colspan="1" rowspan="1">9/9(100%)</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">4/4(100%)</td><td colspan="1" rowspan="1">0/2(0.0%)</td><td colspan="1" rowspan="1">9/9(100%)</td></tr><tr><td colspan="1" rowspan="1">Proteus</td><td colspan="1" rowspan="1">3/3(100%)</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">1/2(50.0%)</td><td colspan="1" rowspan="1">46/48(95.8%)</td><td colspan="1" rowspan="1">4/7(57.1%)</td><td colspan="1" rowspan="1">59/59(100%)</td><td colspan="1" rowspan="1">2/3(66.7%)</td><td colspan="1" rowspan="1">35/36(97.2%)</td><td colspan="1" rowspan="1">4/8(50.0%)</td><td colspan="1" rowspan="1">59/59(100%)</td></tr><tr><td colspan="1" rowspan="1">P. mirabilis</td><td colspan="1" rowspan="1">3/3(100%)</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">1/2(50.0%)</td><td colspan="1" rowspan="1">43/45(95.6%)</td><td colspan="1" rowspan="1">4/5(80.0%)</td><td colspan="1" rowspan="1">56/56(100%)</td><td colspan="1" rowspan="1">2/3(66.7%)</td><td colspan="1" rowspan="1">32/33(97.0%)</td><td colspan="1" rowspan="1">4/6(66.7%)</td><td colspan="1" rowspan="1">56/56(100%)</td></tr><tr><td colspan="1" rowspan="1">P. aeruginosa</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">0/7(0.0%)</td><td colspan="1" rowspan="1">46/46(100%)</td><td colspan="1" rowspan="1">0/20(0.0%)</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">0/9(0.0%)</td><td colspan="1" rowspan="1">25/25(100%)</td><td colspan="1" rowspan="1">0/32(0.0%)</td><td colspan="1" rowspan="1">41/41(100%)</td></tr><tr><td colspan="1" rowspan="1">Salmonella</td><td colspan="1" rowspan="1">1/1(100%)</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">6/6(100%)</td><td colspan="1" rowspan="1">1/2(50.0%)</td><td colspan="1" rowspan="1">12/12(100%)</td><td colspan="1" rowspan="1">1/1(100%)</td><td colspan="1" rowspan="1">8/8(100%)</td><td colspan="1" rowspan="1">1/2(50.0%)</td><td colspan="1" rowspan="1">15/15(100%)</td></tr><tr><td colspan="1" rowspan="1">Serratia</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">0/1(0.0%)</td><td colspan="1" rowspan="1">25/25(100%)</td><td colspan="1" rowspan="1">0/4(0.0%)</td><td colspan="1" rowspan="1">32/32(100%)</td><td colspan="1" rowspan="1">0/1(0.0%)</td><td colspan="1" rowspan="1">20/20(100%)</td><td colspan="1" rowspan="1">0/4(0.0%)</td><td colspan="1" rowspan="1">34/34(100%)</td></tr><tr><td colspan="1" rowspan="1">S. marcescens</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">0/1(0.0%)</td><td colspan="1" rowspan="1">24/24(100%)</td><td colspan="1" rowspan="1">0/4(0.0%)</td><td colspan="1" rowspan="1">31/31(100%)</td><td colspan="1" rowspan="1">0/1(0.0%)</td><td colspan="1" rowspan="1">19/19(100%)</td><td colspan="1" rowspan="1">0/4(0.0%)</td><td colspan="1" rowspan="1">33/33(100%)</td></tr><tr><td colspan="1" rowspan="1">S. maltophilia</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">0/2(0.0%)</td><td colspan="1" rowspan="1">2/2(100%)</td><td colspan="1" rowspan="1">0/3(0.0%)</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">0/5(0.0%)</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">0/5(0.0%)</td><td colspan="1" rowspan="1">2/2(100%)</td></tr><tr><td colspan="1" rowspan="2">AnyOrganismCI</td><td colspan="1" rowspan="2">42/48(87.5%)(75.3-94.1)</td><td colspan="1" rowspan="2">117/117(100%)(96.8-100)</td><td colspan="1" rowspan="2">44/102(43.1%)(33.9-52.8)</td><td colspan="1" rowspan="2">528/541(97.6%)(95.9-98.6)</td><td colspan="1" rowspan="2">73/168(43.5%)(36.2-51.0)</td><td colspan="1" rowspan="2">613/615(99.7%)(98.8-99.9)</td><td colspan="1" rowspan="1">38/74(51.4%)</td><td colspan="1" rowspan="2">403/408(98.8%)(97.2-99.5)</td><td colspan="1" rowspan="2">73/183(39.9%)(33.1-47.1)</td><td colspan="1" rowspan="2">684/686(99.7%)(98.9-99.9)</td></tr><tr><td colspan="1" rowspan="1">(40.2-62.4</td></tr></table>

CI= confidence interval

A supplemental comparison of the five BCID-GN Panel carbapenemase resistance gene targets (OXA, KPC, IMP, NDM, VIM) versus phenotypic AST testing for Ertapenem, Imipenem, and Meropenem is provided in Table 101 for clinical isolates with available AST results. In total, 731 isolates had AST results for 1 or more of Ertapenem, Imipenem, or Meropenem. A true positive (TP) result was defined where OXA, KPC, IMP, NDM and/or VIM was detected by the BCID-GN Panel and the isolate was resistant (R) or intermediate (I) to Ertapenem, Imipenem, or Meropenem. A false negative (FN) result was defined similarly when OXA, KPC, IMP, NDM and/or VIM was not detected by the BCID-GN Panel and the isolate was resistant or intermediate to any of the three antimicrobials. PPA was calculated as $1 0 0 \mathrm { x }$ (TP/(TP+FN)). A true negative (TN) result was defined where OXA, KPC, IMP, NDM and/or VIM were not detected by the BCID-GN Panel and the isolate was susceptible for all carbapenems evaluated with AST. A false positive (FP) result was defined similarly when OXA, KPC, IMP, NDM and/or VIM was detected by the BCID-GN Panel and the isolate was susceptible for all carbapenems evaluated by AST. NPA was calculated as $1 0 0 \mathrm { x }$ $\mathrm { ( T N / ( T N + F P ) }$ ). Note: Resistance to carbapenems can be due to mechanisms other than acquisition of the OXA, KPC, IMP, NDM and/or VIM resistance genes.

Table 101: Clinical performance of the BCID-GN Panel resistance gene targets compared to phenotypic AST results for Ertapenem, Imipenem, and Meropenem   

<table><tr><td colspan="1" rowspan="2">AssociatedOrganism</td><td colspan="2" rowspan="1">OXA</td><td colspan="2" rowspan="1">KPC</td><td colspan="2" rowspan="1">IMP</td><td colspan="2" rowspan="1">NDM</td><td colspan="2" rowspan="1">VIM</td><td colspan="2" rowspan="1">Any ResistanceMarker</td></tr><tr><td colspan="1" rowspan="1">PPATP/TP+FN(%)</td><td colspan="1" rowspan="1">NPATN/TN+FP(%)</td><td colspan="1" rowspan="1">PPATP/TP+FN(%)</td><td colspan="1" rowspan="1">NPATN/TN+FP(%)</td><td colspan="1" rowspan="1">PPATP/TP+FN(%)</td><td colspan="1" rowspan="1">NPATN/TN+FP(%)</td><td colspan="1" rowspan="1">PPATP/TP+FN(%)</td><td colspan="1" rowspan="1">NPATN/TN+FP(%)</td><td colspan="1" rowspan="1">PPATP/TP+FN(%)</td><td colspan="1" rowspan="1">NPATN/TN+FP(%)</td><td colspan="1" rowspan="1">PPATP/TP+FN(%)</td><td colspan="1" rowspan="1">NPATN/TN+FP(%)</td></tr><tr><td colspan="1" rowspan="1">A.baumannii</td><td colspan="1" rowspan="1">8/9(88.9%)</td><td colspan="1" rowspan="1">9/9(100%)</td><td colspan="1" rowspan="1">0/9(0.0%)</td><td colspan="1" rowspan="1">9/9(100%)</td><td colspan="1" rowspan="1">0/9(0.0%)</td><td colspan="1" rowspan="1">9/9(100%)</td><td colspan="1" rowspan="1">0/9(0.0%)</td><td colspan="1" rowspan="1">9/9(100%)</td><td colspan="1" rowspan="1">0/9(0.0%)</td><td colspan="1" rowspan="1">9/9(100%)</td><td colspan="1" rowspan="1">8/9(88.9%)</td><td colspan="1" rowspan="1">9/9(100%)</td></tr><tr><td colspan="1" rowspan="1">Citrobacter</td><td colspan="1" rowspan="1">0/1(0.0%)</td><td colspan="1" rowspan="1">22/22(100%)</td><td colspan="1" rowspan="1">0/1(0.0%)</td><td colspan="1" rowspan="1">22/22(100%)</td><td colspan="1" rowspan="1">0/1(0.0%)</td><td colspan="1" rowspan="1">22/22(100%)</td><td colspan="1" rowspan="1">0/1(0.0%)</td><td colspan="1" rowspan="1">22/22(100%)</td><td colspan="1" rowspan="1">0/1(0.0%)</td><td colspan="1" rowspan="1">22/22(100%)</td><td colspan="1" rowspan="1">0/1(0.0%)</td><td colspan="1" rowspan="1">22/22(100%)</td></tr><tr><td colspan="1" rowspan="1">C. sakazakii</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">1/1(100%)</td><td colspan="1" rowspan="1">---</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">---</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">---</td><td colspan="1" rowspan="1">---</td></tr><tr><td colspan="1" rowspan="1">Enterobacter</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">15/15(100%)</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">15/15(100%)</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">15/15(100%)</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">15/15(100%)</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">15/15(100%)</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">15/15(100%)</td></tr><tr><td colspan="1" rowspan="1">E. cloacaecomplex</td><td colspan="1" rowspan="1">0/1(0.0%)</td><td colspan="1" rowspan="1">51/51(100%)</td><td colspan="1" rowspan="1">0/1(0.0%)</td><td colspan="1" rowspan="1">51/51(100%)</td><td colspan="1" rowspan="1">0/1(0.0%)</td><td colspan="1" rowspan="1">51/51(100%)</td><td colspan="1" rowspan="1">0/1(0.0%)</td><td colspan="1" rowspan="1">51/51(100%)</td><td colspan="1" rowspan="1">0/1(0.0%)</td><td colspan="1" rowspan="1">51/51(100%)</td><td colspan="1" rowspan="1">0/1(0.0%)</td><td colspan="1" rowspan="1">51/51(100%)</td></tr><tr><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">0/2(0.0%)</td><td colspan="1" rowspan="1">247/248(99.6%)</td><td colspan="1" rowspan="1">1/2(50.0%)</td><td colspan="1" rowspan="1">248/248(100%)</td><td colspan="1" rowspan="1">0/2(0.0%)</td><td colspan="1" rowspan="1">248/248(100%)</td><td colspan="1" rowspan="1">0/2(0.0%)</td><td colspan="1" rowspan="1">248/248(100%)</td><td colspan="1" rowspan="1">0/2(0.0%)</td><td colspan="1" rowspan="1">248/248(100%)</td><td colspan="1" rowspan="1">1/2(50.0%)</td><td colspan="1" rowspan="1">247/248(99.6%)</td></tr><tr><td colspan="1" rowspan="1">K. oxytoca</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">40/40(100%)</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">40/40(100%)</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">40/40(100%)</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">40/40(100%)</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">40/40(100%)</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">40/40(100%)</td></tr><tr><td colspan="1" rowspan="1">K.pneumoniaegroup</td><td colspan="1" rowspan="1">0/8(0.0%)</td><td colspan="1" rowspan="1">136/136(100%)</td><td colspan="1" rowspan="1">5/8(62.5%)</td><td colspan="1" rowspan="1">135/136(99.3%)</td><td colspan="1" rowspan="1">0/8(0.0%)</td><td colspan="1" rowspan="1">136/136(100%)</td><td colspan="1" rowspan="1">0/8(0.0%)</td><td colspan="1" rowspan="1">136/136(100%)</td><td colspan="1" rowspan="1">0/8(0.0%)</td><td colspan="1" rowspan="1">136/136(100%)</td><td colspan="1" rowspan="1">5/8(62.5%)</td><td colspan="1" rowspan="1">135/136(99.3%)</td></tr><tr><td colspan="1" rowspan="1">M. morgani</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">10/10(100%)</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">10/10(100%)</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">10/10(100%)</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">10/10(100%)</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">10/10(100%)</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">10/10(100%)</td></tr><tr><td colspan="1" rowspan="1">Proteus</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">61/62(98.4%)</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">62/62(100%)</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">62/62(100%)</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">62/62(100%)</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">62/62(100%)</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">61/62(98.4%)</td></tr><tr><td colspan="1" rowspan="1">P. mirabilis</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">57/57(100%)</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">57/57(100%)</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">57/57(100%)</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">57/57(100%)</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">57/57(100%)</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">57/57(100%)</td></tr><tr><td colspan="1" rowspan="1">P.aeruginosa</td><td colspan="1" rowspan="1">0/5(0.0%)</td><td colspan="1" rowspan="1">65/66(98.5%)</td><td colspan="1" rowspan="1">0/5(0.0%)</td><td colspan="1" rowspan="1">66/66(100%)</td><td colspan="1" rowspan="1">0/5(0.0%)</td><td colspan="1" rowspan="1">66/66(100%)</td><td colspan="1" rowspan="1">0/5(0.0%)</td><td colspan="1" rowspan="1">66/66(100%)</td><td colspan="1" rowspan="1">0/5(0.0%)</td><td colspan="1" rowspan="1">66/66(100%)</td><td colspan="1" rowspan="1">0/5(0.0%)</td><td colspan="1" rowspan="1">65/66(98.5%)</td></tr><tr><td colspan="1" rowspan="1">Salmonella</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">8/8(100%)</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">8/8(100%)</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">8/8(100%)</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">8/8(100%)</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">8/8(100%)</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">8/8(100%)</td></tr><tr><td colspan="1" rowspan="1">Serratia</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">38/38(100%)</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">38/38(100%)</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">38/38(100%)</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">38/38(100%)</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">38/38(100%)</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">38/38(100%)</td></tr><tr><td colspan="1" rowspan="1">S.marcescens</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">37/37(100%)</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">37/37(100%)</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">37/37(100%)</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">37/37(100%)</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">37/37(100%)</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">37/37(100%)</td></tr><tr><td colspan="1" rowspan="2">AnyOrganismCI</td><td colspan="1" rowspan="2">8/26(30.8%)(16.5-50.0)</td><td colspan="1" rowspan="2">796/799(99.6%)(98.999.9)</td><td colspan="1" rowspan="2">6/26(23.1%)(11.0-42.1)</td><td colspan="1" rowspan="2">799/800(99.9%)(99.3-100)</td><td colspan="1" rowspan="2">0/26(0.0%)(0.0-12.9)</td><td colspan="1" rowspan="2">799/799(100%)( 9.5100)</td><td colspan="1" rowspan="1">0/26(0.0%)</td><td colspan="1" rowspan="2">799/799(100%)(. 9.5100)</td><td colspan="1" rowspan="1">0/26(0.0%)</td><td colspan="1" rowspan="2">799/799(100%)(99.5-100)</td><td colspan="1" rowspan="2">14/26(53.8%)(35.5-71.2)</td><td colspan="1" rowspan="2">796/800(99.5%)(98.7-99.8)</td></tr><tr><td colspan="1" rowspan="1">0.0-12.9)</td><td colspan="1" rowspan="1">(0.0-12.9)</td></tr></table>

$\mathrm { C I } =$ confidence interval

Clinical Study, performance of BCID-GN Panel for Specimens with multiple analytes

The ePlex BCID-GN Panel identified multiple bacterial analytes in 103 prospective and retrospective specimens. Of the 349 prospective specimens tested in the clinical study, 320 $( 9 1 . 7 \% )$ had single detections by the ePlex BCID-GN Panel. A total of 29 $( 8 . 2 \% )$ specimens had multiple organisms detected; 22 $( 6 . 2 \% )$ specimens had double detections, and seven specimens $( 2 . 0 \% )$ had triple detections.

Of the 29 specimens with multi-detections by the BCID-GN Panel, 13 $( 4 4 . 8 \% )$ had one or more organism targets detected by BCID-GN that were not identified by the comparator method (i.e., false positive ePlex BCID-GN Panel result).

All multi-detection combinations as detected by the ePlex BCID-GN Panel during the prospective clinical evaluation are presented in Table 102. False positive BCID-GN Panel results are described as discrepant organisms in the far right column of the table.

Table 102: Multi-detections identified by the ePlex BCID-GN Panel (Prospective specimens)   

<table><tr><td rowspan=1 colspan=4>Distinct Co-Detection Combinations Detected by the ePlexBCID-GN Panel in Prospective Clinical Specimens</td><td rowspan=2 colspan=1>Number ofSpecimens(NumberDiscrepant)</td><td rowspan=2 colspan=1>DiscrepantOrganism(s) /Resistance Marker(s)A,B</td></tr><tr><td rowspan=1 colspan=1>Target 1</td><td rowspan=1 colspan=1>Target 2</td><td rowspan=1 colspan=1>Target 3</td><td rowspan=1 colspan=1>ResistanceMarker</td></tr><tr><td rowspan=1 colspan=1>A. baumannii</td><td rowspan=1 colspan=1>Pan Gram-Positive</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>2 (0)</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Citrobacter</td><td rowspan=1 colspan=1>E. cloacae complex</td><td rowspan=1 colspan=1>K. oxytoca</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>2 (2)</td><td rowspan=1 colspan=1>Citrobacter (2), E.cloacae complex (2)</td></tr><tr><td rowspan=1 colspan=1>Citrobacter</td><td rowspan=1 colspan=1>K. oxytoca</td><td rowspan=1 colspan=1>K. pneumoniaegroup</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1 (1)</td><td rowspan=1 colspan=1>Citrobacter (1)</td></tr><tr><td rowspan=1 colspan=1>Citrobacter</td><td rowspan=1 colspan=1>P. mirabilis</td><td rowspan=1 colspan=1>Pan Gram-Positive</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1 (1)</td><td rowspan=1 colspan=1>Pan Gram-Positive (1)</td></tr><tr><td rowspan=1 colspan=1>E. cloacaecomplex</td><td rowspan=1 colspan=1>E. coli</td><td rowspan=1 colspan=1>K. pneumoniaegroup</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1 (0)</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>E. cloacaecomplex</td><td rowspan=1 colspan=1>Pan Candida</td><td rowspan=1 colspan=1>Pan Gram-Positive</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1 (0)</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>E. cloacaecomplex</td><td rowspan=1 colspan=1>Pan Gram-Positive</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>2 (0)</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>E. coli</td><td rowspan=1 colspan=1>K. oxytoca</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>2 (1)</td><td rowspan=1 colspan=1>K. oxytoca (1)</td></tr><tr><td rowspan=1 colspan=1>E. coli</td><td rowspan=1 colspan=1>K. pneumoniaegroup</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>CTX-M</td><td rowspan=1 colspan=1>1 (1)</td><td rowspan=1 colspan=1>E. coli (1)</td></tr><tr><td rowspan=1 colspan=1>E. coli</td><td rowspan=1 colspan=1>Pan Gram-Positive</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>2 (1)</td><td rowspan=1 colspan=1>Pan Gram-Positive (1)</td></tr><tr><td rowspan=1 colspan=1>Enterobacter</td><td rowspan=1 colspan=1>K. pneumoniaegroup</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1 (1)</td><td rowspan=1 colspan=1>Enterobacter (1)</td></tr><tr><td rowspan=1 colspan=1>K. oxytoca</td><td rowspan=1 colspan=1>Pan Gram-Positive</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1 (0)</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>K. oxytoca</td><td rowspan=1 colspan=1>S. marcescens</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1 (0)</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>K. pneumoniaegroup</td><td rowspan=1 colspan=1>P. mirabilis</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1 (0)</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>K. pneumoniaegroup</td><td rowspan=1 colspan=1>Pan Gram-Positive</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>2 (0)</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>K. pneumoniaegroup</td><td rowspan=1 colspan=1>Pan Gram-Positive</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>CTX-M,KPC</td><td rowspan=1 colspan=1>1 (1)</td><td rowspan=1 colspan=1>Pan Gram-Positive (1)</td></tr><tr><td rowspan=1 colspan=1>M. morganii</td><td rowspan=1 colspan=1>P. mirabilis</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1 (0)</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>P. aeruginosa</td><td rowspan=1 colspan=1>P. mirabilis</td><td rowspan=1 colspan=1>Pan Gram-Positive</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1 (0)</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>P. aeruginosa</td><td rowspan=1 colspan=1>Pan Gram-Positive</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1 (0)</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>P. mirabilis</td><td rowspan=1 colspan=1>Pan Gram-Positive</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>3 (2)</td><td rowspan=1 colspan=1>Pan Gram-Positive (2)</td></tr><tr><td rowspan=1 colspan=1>P. mirabilis</td><td rowspan=1 colspan=1>Pan Gram-Positive</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>CTX-M</td><td rowspan=1 colspan=1>1 (0)</td><td rowspan=1 colspan=1></td></tr></table>

A. A discrepant organism or resistance marker is defined as one that was detected by the BCID-GN Panel but not by the comparator method(s).   
B. 12/13 false positive organisms were investigated using PCR/sequencing; the discrepant organism was detected in 11/13 and not detected in 1. One false positive Pan Gram-Positive specimen was not tested. i. In 3/3 false positive Citrobacter specimens, Citrobacter was detected. ii. In 2/2 false positive $E .$ . cloacae complex specimens, $E$ . cloacae complex was detected. iii. An Enterobacter species was not detected in the 1 false positive Enterobacter (non-cloacae complex) specimen. iv. In 1/1 false positive $E$ . coli specimen, $E .$ . coli was detected. v. In 1/1 false positive $K .$ . oxytoca specimen, $K .$ . oxytoca was detected. vi. In 4/4 false positive Pan Gram-Positive specimens, a Pan Gram-Positive organism was detected.

Of the 349 prospective specimens evaluated in the clinical study, 34 specimens $( 9 . 7 \% )$ had multiple organisms identified by culture. Of these 34 specimens, 10 specimens $( 2 9 . 4 \% )$ had one or more organisms that were not identified by the ePlex BCID-GN Panel (i.e., false negative ePlex BCID-GN Panel result).

Due to the significant percentage of false negative results for the ePlex BCID-GN Panel for blood culture specimens with multiple organisms, a limitation is included in the package insert stating the following:

• In mixed cultures, the ePlex BCID-GN Panel may not identify all organisms in the specimen, depending upon the concentration of each target present.

Table 103 includes results from the prospective clinical study for specimens with multiple organisms identified by comparator methods. False negative BCID-GN Panel results are described as discrepant organisms in the far right column of the table.

Table 103: Clinical performance, multi-detections identified by the comparator method(s) (Prospective Specimens)   

<table><tr><td colspan="5" rowspan="1">Distinct Co-Detection Combinations Detected by the ComparatorMethod(s) in Prospective Clinical Specimens</td><td colspan="1" rowspan="2">NumberSpecimensNumberDiscrepant)</td><td colspan="1" rowspan="2">DiscrepantOrganism(s) /ResistanceMarker(s)</td></tr><tr><td colspan="1" rowspan="1">Target 1</td><td colspan="1" rowspan="1">Target 2</td><td colspan="1" rowspan="1">Target 3</td><td colspan="1" rowspan="1">Target 4</td><td colspan="1" rowspan="1">ResistanceMarker</td></tr><tr><td colspan="1" rowspan="1">A. baumannii</td><td colspan="1" rowspan="1">E. faecium</td><td colspan="1" rowspan="1">Staphylococcus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">A. baumannii</td><td colspan="1" rowspan="1">Staphylococcus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Achromobacterxylosoxidans*</td><td colspan="1" rowspan="1">E. cloacae</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Acinetobacterlwoffi*</td><td colspan="1" rowspan="1">Staphylococcushominis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Acinetobacter pittii*</td><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Aerococcusviridans*</td><td colspan="1" rowspan="1">K. oxytoca</td><td colspan="1" rowspan="1">S. epidermidis</td><td colspan="1" rowspan="1">Staphylococcuscohnii</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Aerococcusviridans*</td><td colspan="1" rowspan="1">Staphylococcushominis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">B. fragilis</td><td colspan="1" rowspan="1">Clostridiumspecies*</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Bacillus</td><td colspan="1" rowspan="1">E. cloacae</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">C. acnes*</td><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">C. albicans</td><td colspan="1" rowspan="1">E. cloacae</td><td colspan="1" rowspan="1">E. faecalis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">C. freundii</td><td colspan="1" rowspan="1">P. mirabilis</td><td colspan="1" rowspan="1">Providenciastuartii*</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Candida lusitaniae*</td><td colspan="1" rowspan="1">S. liquefaciens</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Citrobacteramalonaticus*</td><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="5" rowspan="1">Distinct Co-Detection Combinations Detected by the ComparatorMethod(s) in Prospective Clinical Specimens</td><td colspan="1" rowspan="2">NumberSpecimensNumberDiscrepant)</td><td colspan="1" rowspan="2">DiscrepantOrganism(s) /ResistanceMarker((s)a</td></tr><tr><td colspan="1" rowspan="1">Target 1</td><td colspan="1" rowspan="1">Target 2</td><td colspan="1" rowspan="1">Target 3</td><td colspan="1" rowspan="1">Target 4</td><td colspan="1" rowspan="1">ResistanceMarker</td></tr><tr><td colspan="1" rowspan="1">E. aerogenes</td><td colspan="1" rowspan="1">K. oxytoca</td><td colspan="1" rowspan="1">Leclerciaadecarboxylata*</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2 (2)</td><td colspan="1" rowspan="1">E. aerogenes(2)</td></tr><tr><td colspan="1" rowspan="1">E. aerogenes</td><td colspan="1" rowspan="1">P. aeruginosa</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (1)</td><td colspan="1" rowspan="1">P. aeruginosa(1)</td></tr><tr><td colspan="1" rowspan="1">E. cloacae</td><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">K. pneumoniae</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">E. cloacae</td><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">E. faecalis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">K. pneumoniae</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (1)</td><td colspan="1" rowspan="1">E. coli (1)</td></tr><tr><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">P. mirabilis</td><td colspan="1" rowspan="1">Providenciastuartii*</td><td colspan="1" rowspan="1">S. anginosusgroup</td><td colspan="1" rowspan="1">CTX-M</td><td colspan="1" rowspan="1">1 (1)</td><td colspan="1" rowspan="1">E. coli (1)</td></tr><tr><td colspan="1" rowspan="1">E. faecalis</td><td colspan="1" rowspan="1">K. pneumoniae</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">3 (1)</td><td colspan="1" rowspan="1">E. faecalis (1)</td></tr><tr><td colspan="1" rowspan="1">E. faecalis</td><td colspan="1" rowspan="1">M. morganii</td><td colspan="1" rowspan="1">P. mirabilis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (1)</td><td colspan="1" rowspan="1">E. faecalis (1)</td></tr><tr><td colspan="1" rowspan="1">E. faecalis</td><td colspan="1" rowspan="1">Providenciastuartii*</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Enterobacteriaceae*</td><td colspan="1" rowspan="1">K. pneumoniae</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">K. pneumoniae</td><td colspan="1" rowspan="1">Staphylococcushaemolyticus</td><td colspan="1" rowspan="1">non-fermentinggram-negativebacilli*</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (1)</td><td colspan="1" rowspan="1">K. pneumoniae(1)</td></tr><tr><td colspan="1" rowspan="1">Lactococcus lactis*</td><td colspan="1" rowspan="1">P. mirabilis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Micrococcus luteus*</td><td colspan="1" rowspan="1">Sphingomonaspaucimobilis*</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">P. aeruginosa</td><td colspan="1" rowspan="1">P. mirabilis</td><td colspan="1" rowspan="1">Streptococcus -viridans group</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">P. aeruginosa</td><td colspan="1" rowspan="1">S. maltophilia</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (1)</td><td colspan="1" rowspan="1">S. maltophilia(1)</td></tr><tr><td colspan="1" rowspan="1">P. aeruginosa</td><td colspan="1" rowspan="1">Staphylococcushaemolyticus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">P. mirabilis</td><td colspan="1" rowspan="1">Staphylococcus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">S. maltophilia</td><td colspan="1" rowspan="1">Streptococcus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (1)</td><td colspan="1" rowspan="1">Streptococcus(1)</td></tr></table>

\* Indicates an off-panel organism not detected by the BCID-GN Panel. A. A discrepant organism or resistance marker is defined as one that was detected by the comparator method(s) but not by the BCID-GN Panel. Excludes organisms that are not targeted by the BCID-GN Panel.

Of the 577 retrospective specimens tested in the clinical study, 503 specimens $( 8 7 . 2 \% )$ had single detections by the BCID-GN Panel. A total of 74 specimens $( 1 2 . 8 \% )$ had multiple organisms detected; 62 $( 1 0 . 7 \% )$ specimens had two organisms detected, and 12 $( 2 . 1 \% )$ specimens had three organisms detected.

Of the 74 specimens with multi-detections by the BCID-GN Panel, 23 $( 3 1 . 1 \% )$ specimens had one or more organism targets detected by BCID-GN that were not identified by the comparator method (i.e., false positive for the BCID-GN Panel).

All multi-detection combinations as detected by the ePlex BCID-GN Panel during the prospective clinical evaluation are presented in Table 104. False positive BCID-GN Panel results are described as discrepant organisms in the far right column of the table.

Table 104: Co-Detections Identified by the ePlex BCID-GN Panel (Retrospective Specimens)   

<table><tr><td colspan="4" rowspan="1">Distinct Co-Detection Combinations Detected by the ePlexBCID-GN Panel in Retrospective Clinical Specimens</td><td colspan="1" rowspan="2">NumberSpecimens(NumberDiscrepant</td><td colspan="1" rowspan="2">DiscrepantOrganism(s) /ResistanceMarker(s)A.B</td></tr><tr><td colspan="1" rowspan="1">Target 1</td><td colspan="1" rowspan="1">Target 2</td><td colspan="1" rowspan="1">Target 3</td><td colspan="1" rowspan="1">ResistanceMarker</td></tr><tr><td colspan="1" rowspan="1">A. baumannii</td><td colspan="1" rowspan="1">K. pneumoniaegroup</td><td colspan="1" rowspan="1">Pan Gram-Positive</td><td colspan="1" rowspan="1">CTX-M,OXA</td><td colspan="1" rowspan="1">1 (1)</td><td colspan="1" rowspan="1">A. baumannii (1), K.pneumoniae group (1),Pan Gram-Positive (1)</td></tr><tr><td colspan="1" rowspan="1">A. baumannii</td><td colspan="1" rowspan="1">Pan Gram-Positive</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">A. baumannii</td><td colspan="1" rowspan="1">Pan Gram-Positive</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">OXA</td><td colspan="1" rowspan="1">4 (1)</td><td colspan="1" rowspan="1">Pan Gram-Positive (1)</td></tr><tr><td colspan="1" rowspan="1">B. fragilis</td><td colspan="1" rowspan="1">E. cloacae complex</td><td colspan="1" rowspan="1">Pan Gram-Positive</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (1)</td><td colspan="1" rowspan="1">B. fragilis (1)</td></tr><tr><td colspan="1" rowspan="1">B. fragilis</td><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2 (1)</td><td colspan="1" rowspan="1">B. fragilis (1)</td></tr><tr><td colspan="1" rowspan="1">B. fragilis</td><td colspan="1" rowspan="1">Pan Gram-Positive</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Citrobacter</td><td colspan="1" rowspan="1">E. cloacae complex</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (1)</td><td colspan="1" rowspan="1">E. cloacae complex(1)</td></tr><tr><td colspan="1" rowspan="1">Citrobacter</td><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Citrobacter</td><td colspan="1" rowspan="1">K. oxytoca</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (1)</td><td colspan="1" rowspan="1">Citrobacter (1)</td></tr><tr><td colspan="1" rowspan="1">Citrobacter</td><td colspan="1" rowspan="1">K. oxytoca</td><td colspan="1" rowspan="1">K. pneumoniaegroup</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (1)</td><td colspan="1" rowspan="1">K. oxytoca (1)</td></tr><tr><td colspan="1" rowspan="1">Citrobacter</td><td colspan="1" rowspan="1">K. pneumoniaegroup</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Citrobacter</td><td colspan="1" rowspan="1">K. pneumoniaegroup</td><td colspan="1" rowspan="1">Pan Gram-Positive</td><td colspan="1" rowspan="1">CTX-M</td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Citrobacter</td><td colspan="1" rowspan="1">M. morganii</td><td colspan="1" rowspan="1">Pan Gram-Positive</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (1)</td><td colspan="1" rowspan="1">M. morganii (1)</td></tr><tr><td colspan="1" rowspan="1">Citrobacter</td><td colspan="1" rowspan="1">Pan Gram-Positive</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">3 (2)</td><td colspan="1" rowspan="1">Pan Gram-Positive (2)</td></tr><tr><td colspan="1" rowspan="1">E. cloacaecomplex</td><td colspan="1" rowspan="1">K. pneumoniaegroup</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">E. cloacaecomplex</td><td colspan="1" rowspan="1">P. aeruginosa</td><td colspan="1" rowspan="1">Pan Gram-Positive</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (1)</td><td colspan="1" rowspan="1">P. aeruginosa (1)</td></tr><tr><td colspan="1" rowspan="1">E. cloacaecomplex</td><td colspan="1" rowspan="1">Pan Candida</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (1)</td><td colspan="1" rowspan="1">Pan Candida (1)</td></tr><tr><td colspan="1" rowspan="1">E. cloacaecomplex</td><td colspan="1" rowspan="1">Pan Gram-Positive</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2 (1)</td><td colspan="1" rowspan="1">Pan Gram-Positive (1)</td></tr><tr><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">K. oxytoca</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">K. oxytoca</td><td colspan="1" rowspan="1">Pan Gram-Positive</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">K. pneumoniaegroup</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="4" rowspan="1">Distinct Co-Detection Combinations Detected by the ePlexBCID-GN Panel in Retrospective Clinical Specimens</td><td colspan="1" rowspan="2">NumberSpecimens(NumberDiscrepant)</td><td colspan="1" rowspan="2">DiscrepantOrganism(s) /ResistanceMarker()AB</td></tr><tr><td colspan="1" rowspan="1">Target 1</td><td colspan="1" rowspan="1">Target 2</td><td colspan="1" rowspan="1">Target 3</td><td colspan="1" rowspan="1">ResistanceMarker</td></tr><tr><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">M. morganii</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">P. mirabilis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">3 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">P. mirabilis</td><td colspan="1" rowspan="1">Pan Gram-Positive</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">Pan Gram-Positive</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">8 (2)</td><td colspan="1" rowspan="1">Pan Gram-Positive (2)</td></tr><tr><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">Pan Gram-Positive</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">CTX-M</td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Enterobacter</td><td colspan="1" rowspan="1">Pan Candida</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Enterobacter</td><td colspan="1" rowspan="1">Pan Gram-Positive</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">H. influenzae</td><td colspan="1" rowspan="1">N. meningitidis</td><td colspan="1" rowspan="1">P. aeruginosa</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (1)</td><td colspan="1" rowspan="1">N. meningitidis (1), P.aeruginosa (1)</td></tr><tr><td colspan="1" rowspan="1">K. oxytoca</td><td colspan="1" rowspan="1">K. pneumoniaegroup</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2 (1)</td><td colspan="1" rowspan="1">K. pneumoniae group(1)</td></tr><tr><td colspan="1" rowspan="1">K. oxytoca</td><td colspan="1" rowspan="1">Pan Gram-Positive</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">3 (2)</td><td colspan="1" rowspan="1">Pan Gram-Positive (2)</td></tr><tr><td colspan="1" rowspan="1">K. oxytoca</td><td colspan="1" rowspan="1">S. marcescens</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (1)</td><td colspan="1" rowspan="1">S. marcescens (1)</td></tr><tr><td colspan="1" rowspan="1">K. pneumoniaegroup</td><td colspan="1" rowspan="1">Pan Gram-Positive</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">4 (1)</td><td colspan="1" rowspan="1">Pan Gram-Positive (1)</td></tr><tr><td colspan="1" rowspan="1">K. pneumoniaegroup</td><td colspan="1" rowspan="1">Pan Gram-Positive</td><td colspan="1" rowspan="1">S. marcescens</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (1)</td><td colspan="1" rowspan="1">K. pneumoniae group(1)</td></tr><tr><td colspan="1" rowspan="1">K. pneumoniaegroup</td><td colspan="1" rowspan="1">S. maltophilia</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">M. morganii</td><td colspan="1" rowspan="1">P. aeruginosa</td><td colspan="1" rowspan="1">Pan Gram-Positive</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (1)</td><td colspan="1" rowspan="1">P. aeruginosa (1)</td></tr><tr><td colspan="1" rowspan="1">M. morganii</td><td colspan="1" rowspan="1">P. mirabilis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">M. morganii</td><td colspan="1" rowspan="1">Pan Gram-Positive</td><td colspan="1" rowspan="1">Proteus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">P. aeruginosa</td><td colspan="1" rowspan="1">Pan Gram-Positive</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">P. mirabilis</td><td colspan="1" rowspan="1">Pan Gram-Positive</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">5 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Pan Candida</td><td colspan="1" rowspan="1">Pan Gram-Positive</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Pan Gram-Positive</td><td colspan="1" rowspan="1">S. maltophilia</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Pan Gram-Positive</td><td colspan="1" rowspan="1">S. marcescens</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">3 (0)</td><td colspan="1" rowspan="1"></td></tr></table>

A. A discrepant organism or resistance marker is defined as one that was detected by the BCID-GN Panel but not by the comparator method(s).   
B. 24/26 false positive organisms were investigated using PCR/sequencing; the discrepant organism was detected in 21/24, not detected in 2, and was indeterminate for one organism. i. In 1/1 false positive A.baumannii specimen, A. baumannii was detected. ii. $\mathrm { I n } 2 / 2$ false positive $B$ . fragilis specimens, $B .$ . fragilis was detected. iii. In 1/1 false positive Citrobacter specimen, Citrobacter was detected. iv. In the one false positive $E$ . cloacae complex specimen, PCR/sequencing was indeterminate. v. In 1/1 false positive K. oxytoca specimen, K. oxytoca was detected. vi. In 3/3 false positive $K .$ . pneumoniae group specimens, $K .$ . pneumoniae group was detected. vii. In 1/1 false positive specimen, $M .$ . morganii was detected. viii. N. meningitidis was not detected in the 1 N. meningitidis false positive specimen. ix. In 2/3 false positive $P$ . aeruginosa specimens, $P$ . aeruginosa was detected. $P$ . aeruginosa was not detected in the remaining specimen. x. In 1/1 false positive pan Candida specimen, Pan Candida was detected. xi. $\operatorname { I n } 8 / 8$ false positive Pan Gram-Positive specimens, a Pan Gram-Positive organism was detected.   
xii. In 1/1 false positive S. marcescens specimen, S. marcescens was detected.

Of the 577 retrospective specimens evaluated in the clinical study, 77 specimens $( 9 . 7 \% )$ had multiple organisms identified by the culture comparator. Of these 77 specimens, 28 specimens $( 3 6 . 4 \% )$ had one or more organisms that were not identified by the ePlex BCID-GN Panel (i.e., false negative).

Table 105 includes results from the retrospective clinical study for specimens with multiple organisms identified by comparator methods. False negative BCID-GN Panel results are described as discrepant organisms in the far right column of the table.

able 105: Co-Detections identified by the comparator method(s) (Retrospective Specimens)   

<table><tr><td colspan="5" rowspan="1">Distinct Co-Detection Combinations Detected by the Comparator Method(s) inRetrospective Clinical Specimens</td><td colspan="1" rowspan="2">NumberSpecimens(NumberDiscrepant)</td><td colspan="1" rowspan="2">DiscrepantOrganism(s) /ResistanceMarker(s)4</td></tr><tr><td colspan="1" rowspan="1">Target 1</td><td colspan="1" rowspan="1">Target 2</td><td colspan="1" rowspan="1">Target 3</td><td colspan="1" rowspan="1">Target 4</td><td colspan="1" rowspan="1">ResistanceMarker</td></tr><tr><td colspan="1" rowspan="1">A. baumannii</td><td colspan="1" rowspan="1">E. faecalis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">OXA</td><td colspan="1" rowspan="1">2 (0)</td><td colspan="1" rowspan="1">PY</td></tr><tr><td colspan="1" rowspan="1">A. baumannii</td><td colspan="1" rowspan="1">E. faecalis</td><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">A. baumannii</td><td colspan="1" rowspan="1">E. faecium</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">OXA</td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1">PY</td></tr><tr><td colspan="1" rowspan="1">A. baumannii</td><td colspan="1" rowspan="1">Staphylococcus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Acinetobacterradioresistens*</td><td colspan="1" rowspan="1">P. vulgaris</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">OXA</td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Aeromonascaviae*</td><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">Enterococcuscasseliflavus</td><td colspan="1" rowspan="1">K. oxytoca</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (1)</td><td colspan="1" rowspan="1">E. casseliflavus (1)</td></tr><tr><td colspan="1" rowspan="1">Aeromonasveronii*</td><td colspan="1" rowspan="1">E. cloacae</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (1)</td><td colspan="1" rowspan="1">E. cloacae (1)</td></tr><tr><td colspan="1" rowspan="1">B. fragilis</td><td colspan="1" rowspan="1">S. anginosusgroup</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">C. albicans</td><td colspan="1" rowspan="1">E. faecalis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">C. albicans</td><td colspan="1" rowspan="1">E. faecium</td><td colspan="1" rowspan="1">Staphylococcushominis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (1)</td><td colspan="1" rowspan="1">C. albicans (1)</td></tr><tr><td colspan="1" rowspan="1">C. albicans</td><td colspan="1" rowspan="1">P. aeruginosa</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (1)</td><td colspan="1" rowspan="1">C. albicans (1)</td></tr><tr><td colspan="1" rowspan="1">C. albicans</td><td colspan="1" rowspan="1">S. epidermidis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">C. braakii</td><td colspan="1" rowspan="1">E. cloacae</td><td colspan="1" rowspan="1">K. oxytoca</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (1)</td><td colspan="1" rowspan="1">C. braakii (1), K.oxytoca (1)</td></tr><tr><td colspan="1" rowspan="1">C. braakii</td><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">C. braakii</td><td colspan="1" rowspan="1">Streptococcusoralis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">C. freundii</td><td colspan="1" rowspan="1">Enterococcus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (1)</td><td colspan="1" rowspan="1">Enterococcus (1)</td></tr><tr><td colspan="1" rowspan="1">C. freundii</td><td colspan="1" rowspan="1">K. pneumoniae</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">C. freundii</td><td colspan="1" rowspan="1">K. pneumoniae</td><td colspan="1" rowspan="1">Staphylococcushominis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">CTX-M</td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="5" rowspan="1">Distinct Co-Detection Combinations Detected by the Comparator Method(s) inRetrospective Clinical Specimens</td><td colspan="1" rowspan="2"> SpecimensNumber(NumberDiscrepant)</td><td colspan="1" rowspan="2">DiscrepantOrganism(s) /ResistanceMarker(s)A</td></tr><tr><td colspan="1" rowspan="1">Target 1</td><td colspan="1" rowspan="1">Target 2</td><td colspan="1" rowspan="1">Target 3</td><td colspan="1" rowspan="1">Target 4</td><td colspan="1" rowspan="1">ResistanceMarker</td></tr><tr><td colspan="1" rowspan="1">C. glabrata</td><td colspan="1" rowspan="1">E. aerogenes</td><td colspan="1" rowspan="1">Staphylococcus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (1)</td><td colspan="1" rowspan="1">Staphylococcus (1)</td></tr><tr><td colspan="1" rowspan="1">C. glabrata</td><td colspan="1" rowspan="1">P. mirabilis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (1)</td><td colspan="1" rowspan="1">C. glabrata (1)</td></tr><tr><td colspan="1" rowspan="1">C. koseri</td><td colspan="1" rowspan="1">E. faecalis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">C. krusei</td><td colspan="1" rowspan="1">S. epidermidis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (1)</td><td colspan="1" rowspan="1">S. epidermidis (1)</td></tr><tr><td colspan="1" rowspan="1">C. youngae</td><td colspan="1" rowspan="1">K. oxytoca</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (1)</td><td colspan="1" rowspan="1">K. oxytoca (1)</td></tr><tr><td colspan="1" rowspan="1">Clostridiumperfringens*</td><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">E. aerogenes</td><td colspan="1" rowspan="1">S. anginosusgroup</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">E. cloacae</td><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (1)</td><td colspan="1" rowspan="1">E. coli (1)</td></tr><tr><td colspan="1" rowspan="1">E. cloacae</td><td colspan="1" rowspan="1">E. faecalis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (1)</td><td colspan="1" rowspan="1">E. faecalis (1)</td></tr><tr><td colspan="1" rowspan="1">E. cloacae</td><td colspan="1" rowspan="1">E. faecium</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">E. cloacae</td><td colspan="1" rowspan="1">E. faecium</td><td colspan="1" rowspan="1">Staphylococcushominis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">E. cloacae</td><td colspan="1" rowspan="1">K. pneumoniae</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">E. cloacae</td><td colspan="1" rowspan="1">M. morganii</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (1)</td><td colspan="1" rowspan="1">E. cloacae (1)</td></tr><tr><td colspan="1" rowspan="1">E. cloacae</td><td colspan="1" rowspan="1">S. anginosusgroup</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">E. cloacae</td><td colspan="1" rowspan="1">S. maltophilia</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (1)</td><td colspan="1" rowspan="1">S. maltophilia (1)</td></tr><tr><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">E. faecalis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">E. faecalis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">CTX-M</td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">E. faecalis</td><td colspan="1" rowspan="1">K. pneumoniae</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (1)</td><td colspan="1" rowspan="1">E. coli (1)</td></tr><tr><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">E. faecalis</td><td colspan="1" rowspan="1">P. mirabilis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">E. faecium</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">E. faecium</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">CTX-M</td><td colspan="1" rowspan="1">1 (1)</td><td colspan="1" rowspan="1">E. faecium (1)</td></tr><tr><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">K. oxytoca</td><td colspan="1" rowspan="1">Streptococcusinfantarius</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">P. aeruginosa</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (1)</td><td colspan="1" rowspan="1">P. aeruginosa (1)</td></tr><tr><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">P. mirabilis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (1)</td><td colspan="1" rowspan="1">E. coli (1)</td></tr><tr><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">P. mirabilis</td><td colspan="1" rowspan="1">P. vulgaris</td><td colspan="1" rowspan="1">Streptococcus- viridansgroup</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (1)</td><td colspan="1" rowspan="1">S. viridans group(1)</td></tr><tr><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">Propionibacteria*</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">S. anginosus gp</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (1)</td><td colspan="1" rowspan="1">S. anginosus group(1)</td></tr><tr><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">S. pneumoniae</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="5" rowspan="1">Distinct Co-Detection Combinations Detected by the Comparator Method(s) inRetrospective Clinical Specimens</td><td colspan="1" rowspan="2">NumberSpecimens(NumberDiscrepant)</td><td colspan="1" rowspan="2">DiscrepantOrganism(s) /ResistanceMarker(s)</td></tr><tr><td colspan="1" rowspan="1">Target 1</td><td colspan="1" rowspan="1">Target 2</td><td colspan="1" rowspan="1">Target 3</td><td colspan="1" rowspan="1">Target 4</td><td colspan="1" rowspan="1">ResistanceMarker</td></tr><tr><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">Staphylococcus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">E. faecalis</td><td colspan="1" rowspan="1">K. pneumoniae</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (1)</td><td colspan="1" rowspan="1">K. pneumoniae (1)</td></tr><tr><td colspan="1" rowspan="1">E. faecalis</td><td colspan="1" rowspan="1">M. morganii</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">E. faecalis</td><td colspan="1" rowspan="1">M. morganii</td><td colspan="1" rowspan="1">P. vulgaris</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">E. faecalis</td><td colspan="1" rowspan="1">P. aeruginosa</td><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (1)</td><td colspan="1" rowspan="1">P. aeruginosa (1)</td></tr><tr><td colspan="1" rowspan="1">E. faecalis</td><td colspan="1" rowspan="1">P. mirabilis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">3 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">E. faecalis</td><td colspan="1" rowspan="1">S. maltophilia</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (1)</td><td colspan="1" rowspan="1">E. faecalis (1)</td></tr><tr><td colspan="1" rowspan="1">E. faecalis</td><td colspan="1" rowspan="1">S. marcescens</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">E. faecium</td><td colspan="1" rowspan="1">K. pneumoniae</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">E. faecium</td><td colspan="1" rowspan="1">P. aeruginosa</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">E. faecium</td><td colspan="1" rowspan="1">P. mirabilis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">K. oxytoca</td><td colspan="1" rowspan="1">S. anginosusgroup</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">K. pneumoniae</td><td colspan="1" rowspan="1">P. aeruginosa</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (1)</td><td colspan="1" rowspan="1">P. aeruginosa (1)</td></tr><tr><td colspan="1" rowspan="1">K. pneumoniae</td><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2 (1)</td><td colspan="1" rowspan="1">S. aureus (1)</td></tr><tr><td colspan="1" rowspan="1">K. pneumoniae</td><td colspan="1" rowspan="1">Staphylococcus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (1)</td><td colspan="1" rowspan="1">Staphylococcus (1)</td></tr><tr><td colspan="1" rowspan="1">P. aeruginosa</td><td colspan="1" rowspan="1">S. maltophilia</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (1)</td><td colspan="1" rowspan="1">S. maltophilia (1)</td></tr><tr><td colspan="1" rowspan="1">P. mirabilis</td><td colspan="1" rowspan="1">Peptostreptococcus anaerobius*</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">P. mirabilis</td><td colspan="1" rowspan="1">Providenciastuartii*</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2 (1)</td><td colspan="1" rowspan="1">P. mirabilis (1)</td></tr><tr><td colspan="1" rowspan="1">P. mirabilis</td><td colspan="1" rowspan="1">Staphylococcus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Pseudomonasputida*</td><td colspan="1" rowspan="1">S. epidermidis</td><td colspan="1" rowspan="1">S. maltophilia</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1">S. marcescens</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">S. marcescens</td><td colspan="1" rowspan="1">Staphylococcus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">S. marcescens</td><td colspan="1" rowspan="1">Streptococcusmitis group</td><td colspan="1" rowspan="1">Streptococcussalivarius</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="2" rowspan="1">* Indicates an off-panel or</td><td colspan="5" rowspan="1">not detected by the BCID-GN Panel</td></tr></table>

\* Indicates an off-panel organism not detected by the BCID-GN Panel. A. A discrepant organism or resistance marker is defined as one that was detected by the comparator method(s) but not by the BCIDGN Panel. Excludes organisms not targeted by the BCID-GN Panel.

# Instrument Errors/Invalid Rates:

A total of 2460 prospective, retrospective, and contrived specimens were initially tested in the clinical study. Of these, 23/2460 $( 0 . 9 \% )$ did not complete the run and the specimen was retested according to assay instructions.

A total of 2334/2460 $( 9 4 . 9 \%$ , $9 5 \%$ CI: $9 3 . 9 \% 9 5 . 7 \%$ ) specimens generated valid results for an initial invalid rate of $5 . 1 \%$ ( $9 5 \%$ CI: $4 . 3 \% 6 . 1 \%$ )

Upon repeat testing of the 126 specimens with initially invalid results, 1/126 $( 0 . 8 \% )$ did not complete the run and the specimen was retested. After repeat testing, all 126 specimens completed testing with 114/126 $( 9 0 . 5 \% )$ specimens generating valid results for a repeat invalid rate of $9 . 5 \%$ ( $9 5 \%$ CI: $5 . 5 \% - 1 5 . 9 \%$ )

Overall, after final testing, 12/2460 $0 . 5 \%$ , $9 5 \%$ CI: $0 . 3 \% { - } 0 . 9 \%$ ) specimens had final, invalid results, resulting in a final validity rate of 2448/2460 $9 9 . 5 \%$ , $9 5 \%$ CI: $9 9 . 1 \%$ - $9 9 . 7 \%$ .

4. Clinical cut-off:

Not applicable

5. Expected values/Reference range:

A prospective, multicenter clinical study was conducted to evaluate the clinical performance of the ePlex BCID-GN Panel in positive blood culture specimens. A total of 349 specimens were prospectively collected at seven clinical sites in two phases from patients of all ages and genders. In the first phase from June 2014 through July 2016, 182 specimens were prospectively collected and frozen; from June through July 2018, 167 specimens were prospectively collected and tested fresh (never frozen). The expected values of individual analytes based on the ePlex BCID-GN Panel results in prospective specimens are summarized by age group and by site in Tables 106 and 107.

Table 106: Expected values by age group (Prospective specimens)   

<table><tr><td colspan="1" rowspan="1">Target</td><td colspan="1" rowspan="1">All Ages(N=349)n (%)</td><td colspan="1" rowspan="1">Age &lt;1(N=7)n (%)</td><td colspan="1" rowspan="1">Age 1-17 =10n (%)</td><td colspan="1" rowspan="1">Age 18-44(N=50)n (%)</td><td colspan="1" rowspan="1">Age 45-64(N=124)n (%)</td><td colspan="1" rowspan="1">Age 65-84(N=125)n (%)</td><td colspan="1" rowspan="1">Age 85+=3)n (%)</td></tr><tr><td colspan="1" rowspan="1">Acinetobacter baumannii</td><td colspan="1" rowspan="1">4(1.1)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">1 (2.0)</td><td colspan="1" rowspan="1">2 (1.6)</td><td colspan="1" rowspan="1">1(0.8)</td><td colspan="1" rowspan="1">0 (0.0)</td></tr><tr><td colspan="1" rowspan="1">Bacteroides fragilis</td><td colspan="1" rowspan="1">11 (3.2)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">2 (4.0)</td><td colspan="1" rowspan="1">4 (3.2)</td><td colspan="1" rowspan="1">2 (1.6)</td><td colspan="1" rowspan="1">3 (9.1)</td></tr><tr><td colspan="1" rowspan="1">Citrobacter</td><td colspan="1" rowspan="1">8 (2.3)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">2 (4.0)</td><td colspan="1" rowspan="1">1 (0.8)</td><td colspan="1" rowspan="1">2 (1.6)</td><td colspan="1" rowspan="1">3 (9.1)</td></tr><tr><td colspan="1" rowspan="1">Cronobacter sakazakii</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td></tr><tr><td colspan="1" rowspan="1">Enterobacter (non-cloacaecomplex)</td><td colspan="1" rowspan="1">9 (2.6)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">1 (10.0)</td><td colspan="1" rowspan="1">2 (4.0)</td><td colspan="1" rowspan="1">5 (4.0)</td><td colspan="1" rowspan="1">1 (0.8)</td><td colspan="1" rowspan="1">0 (0.0)</td></tr><tr><td colspan="1" rowspan="1">Enterobacter cloacae complex</td><td colspan="1" rowspan="1">23 (6.6)</td><td colspan="1" rowspan="1">3 (42.9)</td><td colspan="1" rowspan="1">1 (10.0)</td><td colspan="1" rowspan="1">6 (12.0)</td><td colspan="1" rowspan="1">5(4.0)</td><td colspan="1" rowspan="1">8 (6.4)</td><td colspan="1" rowspan="1">0 (0.0)</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">132 (37.8)</td><td colspan="1" rowspan="1">2 (28.6)</td><td colspan="1" rowspan="1">2 (20.0)</td><td colspan="1" rowspan="1">16 (32.0)</td><td colspan="1" rowspan="1">41 (33.1)</td><td colspan="1" rowspan="1">55 (44.0)</td><td colspan="1" rowspan="1">16 (48.5)</td></tr><tr><td colspan="1" rowspan="1">Fusobacterium necrophorum</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td></tr><tr><td colspan="1" rowspan="1">Fusobacterium nucleatum</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td></tr><tr><td colspan="1" rowspan="1">Haemophilus influenzae</td><td colspan="1" rowspan="1">7(2.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">3 (6.0)</td><td colspan="1" rowspan="1">1 (0.8)</td><td colspan="1" rowspan="1">1 (0.8)</td><td colspan="1" rowspan="1">2 (6.1)</td></tr><tr><td colspan="1" rowspan="1">Klebsiella oxytoca</td><td colspan="1" rowspan="1">12 (3.4)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">3 (6.0)</td><td colspan="1" rowspan="1">8 (6.5)</td><td colspan="1" rowspan="1">1(0.8)</td><td colspan="1" rowspan="1">0 (0.0)</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae group</td><td colspan="1" rowspan="1">59 (16.9)</td><td colspan="1" rowspan="1">1 (14.3)</td><td colspan="1" rowspan="1">1 (10.0)</td><td colspan="1" rowspan="1">10 (20.0)</td><td colspan="1" rowspan="1">26 (21.0)</td><td colspan="1" rowspan="1">17 (13.6)</td><td colspan="1" rowspan="1">4 (12.1)</td></tr><tr><td colspan="1" rowspan="1">Morganella morganii</td><td colspan="1" rowspan="1">3 (0.9)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">2 (1.6)</td><td colspan="1" rowspan="1">1(0.8)</td><td colspan="1" rowspan="1">0 (0.0)</td></tr><tr><td colspan="1" rowspan="1">Neisseria meningitidis</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td></tr><tr><td colspan="1" rowspan="1">Proteus</td><td colspan="1" rowspan="1">22 (6.3)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">6 (4.8)</td><td colspan="1" rowspan="1">13 (10.4)</td><td colspan="1" rowspan="1">3 (9.1)</td></tr><tr><td colspan="1" rowspan="1">Proteus mirabilis</td><td colspan="1" rowspan="1">22 (6.3)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">6(4.8)</td><td colspan="1" rowspan="1">13 (10.4)</td><td colspan="1" rowspan="1">3 (9.1)</td></tr><tr><td colspan="1" rowspan="1">Target</td><td colspan="1" rowspan="1">All Ages(N=349)n (%)</td><td colspan="1" rowspan="1">Age &lt;1(N=7)n (%)</td><td colspan="1" rowspan="1">Age 1-17(N=10)n (%)</td><td colspan="1" rowspan="1">Age 18-44(N=50)n (%)</td><td colspan="1" rowspan="1">Age 45-64(N=124)n (%)</td><td colspan="1" rowspan="1">Age 65-84(N=125)n (%)</td><td colspan="1" rowspan="1">Age 85+(N=33)n (%)</td></tr><tr><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">28 (8.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">2 (20.0)</td><td colspan="1" rowspan="1">3 (6.0)</td><td colspan="1" rowspan="1">12 (9.7)</td><td colspan="1" rowspan="1">10 (8.0)</td><td colspan="1" rowspan="1">1 (3.0)</td></tr><tr><td colspan="1" rowspan="1">Salmonella</td><td colspan="1" rowspan="1">2 (0.6)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">1(0.8)</td><td colspan="1" rowspan="1">1 (0.8)</td><td colspan="1" rowspan="1">0 (0.0)</td></tr><tr><td colspan="1" rowspan="1">Serratia</td><td colspan="1" rowspan="1">10 (2.9)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">1 (10.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">5(4.0)</td><td colspan="1" rowspan="1">4 (3.2)</td><td colspan="1" rowspan="1">0 (0.0)</td></tr><tr><td colspan="1" rowspan="1">Serratia marcescens</td><td colspan="1" rowspan="1">9 (2.6)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">1 (10.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">4 (3.2)</td><td colspan="1" rowspan="1">4 (3.2)</td><td colspan="1" rowspan="1">0 (0.0)</td></tr><tr><td colspan="1" rowspan="1">Stenotrophomonas maltophilia</td><td colspan="1" rowspan="1">3 (0.9)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">3 (6.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td></tr><tr><td colspan="1" rowspan="1">Pan Candida</td><td colspan="1" rowspan="1">2 (0.6)</td><td colspan="1" rowspan="1">1 (14.3)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">1 (0.8)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td></tr><tr><td colspan="1" rowspan="1">Pan Gram-Positive</td><td colspan="1" rowspan="1">24 (6.9)</td><td colspan="1" rowspan="1">1 (14.3)</td><td colspan="1" rowspan="1">2 (20.0)</td><td colspan="1" rowspan="1">5 (10.0)</td><td colspan="1" rowspan="1">7(5.6)</td><td colspan="1" rowspan="1">7 (5.6)</td><td colspan="1" rowspan="1">2 (6.1)</td></tr><tr><td colspan="1" rowspan="1">CTX-M</td><td colspan="1" rowspan="1">24 (6.9)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">2 (4.0)</td><td colspan="1" rowspan="1">7 (5.6)</td><td colspan="1" rowspan="1">12 (9.6)</td><td colspan="1" rowspan="1">3 (9.1)</td></tr><tr><td colspan="1" rowspan="1">IMP</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td></tr><tr><td colspan="1" rowspan="1">KPC</td><td colspan="1" rowspan="1">3 (0.9)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">1 (2.0)</td><td colspan="1" rowspan="1">1(0.8)</td><td colspan="1" rowspan="1">1 (0.8)</td><td colspan="1" rowspan="1">0 (0.0)</td></tr><tr><td colspan="1" rowspan="1">NDM</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td></tr><tr><td colspan="1" rowspan="1">OXA</td><td colspan="1" rowspan="1">1 (0.3)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">1 (0.8)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td></tr><tr><td colspan="1" rowspan="1">VIM</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td></tr></table>

Table 107: Expected values by collection site (Prospective specimens)   

<table><tr><td colspan="1" rowspan="1">Target</td><td colspan="1" rowspan="1">All Sites(N=349)n (%)</td><td colspan="1" rowspan="1">Site 1(N=88)n (%)</td><td colspan="1" rowspan="1">Site 2(N=23)n (%</td><td colspan="1" rowspan="1">Site 3(N=98)n (%)</td><td colspan="1" rowspan="1">Site 4(N=58)n (%)</td><td colspan="1" rowspan="1">Site 5(N=46)n (%)</td><td colspan="1" rowspan="1">Site 6(N=28)n (%</td><td colspan="1" rowspan="1">Site 7(N=8)n (%)</td></tr><tr><td colspan="1" rowspan="1">Acinetobacter baumannii</td><td colspan="1" rowspan="1">4(1.1)</td><td colspan="1" rowspan="1">3 (3.4)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">1 (1.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td></tr><tr><td colspan="1" rowspan="1">Bacteroides fragilis</td><td colspan="1" rowspan="1">11 (3.2)</td><td colspan="1" rowspan="1">2 (2.3)</td><td colspan="1" rowspan="1">3 (13.0)</td><td colspan="1" rowspan="1">3 (3.1)</td><td colspan="1" rowspan="1">2 (3.4)</td><td colspan="1" rowspan="1">1 (2.2)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td></tr><tr><td colspan="1" rowspan="1">Citrobacter</td><td colspan="1" rowspan="1">8 (2.3)</td><td colspan="1" rowspan="1">2 (2.3)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">3 (3.1)</td><td colspan="1" rowspan="1">1 (1.7)</td><td colspan="1" rowspan="1">1 (2.2)</td><td colspan="1" rowspan="1">1 (3.6)</td><td colspan="1" rowspan="1">0 (0.0)</td></tr><tr><td colspan="1" rowspan="1">Cronobacter sakazakii</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td></tr><tr><td colspan="1" rowspan="1">Enterobacter (non-cloacaecomplex)</td><td colspan="1" rowspan="1">9 (2.6)</td><td colspan="1" rowspan="1">2 (2.3)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">4 (4.1)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">1 (2.2)</td><td colspan="1" rowspan="1">2 (7.1)</td><td colspan="1" rowspan="1">0 (0.0)</td></tr><tr><td colspan="1" rowspan="1">Enterobacter cloacae complex</td><td colspan="1" rowspan="1">23 (6.6)</td><td colspan="1" rowspan="1">3 (3.4)</td><td colspan="1" rowspan="1">1 (4.3)</td><td colspan="1" rowspan="1">10(10.2)</td><td colspan="1" rowspan="1">1 (1.7)</td><td colspan="1" rowspan="1">6 (13.0)</td><td colspan="1" rowspan="1">2 (7.1)</td><td colspan="1" rowspan="1">0 (0.0)</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">132(37.8)</td><td colspan="1" rowspan="1">30(34.1)</td><td colspan="1" rowspan="1">8 (34.8)</td><td colspan="1" rowspan="1">37(37.8)</td><td colspan="1" rowspan="1">25(43.1)</td><td colspan="1" rowspan="1">17(37.0)</td><td colspan="1" rowspan="1">12(42.9)</td><td colspan="1" rowspan="1">3 (37.5)</td></tr><tr><td colspan="1" rowspan="1">Fusobacterium necrophorum</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td></tr><tr><td colspan="1" rowspan="1">Fusobacterium nucleatum</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td></tr><tr><td colspan="1" rowspan="1">Haemophilus influenzae</td><td colspan="1" rowspan="1">7 (2.0)</td><td colspan="1" rowspan="1">1 (1.1)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">2 (2.0)</td><td colspan="1" rowspan="1">2 (3.4)</td><td colspan="1" rowspan="1">1 (2.2)</td><td colspan="1" rowspan="1">1 (3.6)</td><td colspan="1" rowspan="1">0 (0.0)</td></tr><tr><td colspan="1" rowspan="1">Klebsiella oxytoca</td><td colspan="1" rowspan="1">12 (3.4)</td><td colspan="1" rowspan="1">5 (5.7)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">3 (3.1)</td><td colspan="1" rowspan="1">1 (1.7)</td><td colspan="1" rowspan="1">2 (4.3)</td><td colspan="1" rowspan="1">1 (3.6)</td><td colspan="1" rowspan="1">0 (0.0)</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae group</td><td colspan="1" rowspan="1">59 (16.9)</td><td colspan="1" rowspan="1">17(19.3)</td><td colspan="1" rowspan="1">3 (13.0)</td><td colspan="1" rowspan="1">20(20.4)</td><td colspan="1" rowspan="1">5 (8.6)</td><td colspan="1" rowspan="1">7 (15.2)</td><td colspan="1" rowspan="1">4 (14.3)</td><td colspan="1" rowspan="1">3 (37.5)</td></tr><tr><td colspan="1" rowspan="1">Morganella morganii</td><td colspan="1" rowspan="1">3 (0.9)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">1 (4.3)</td><td colspan="1" rowspan="1">2 (2.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td></tr><tr><td colspan="1" rowspan="1">Neisseria meningitidis</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td></tr><tr><td colspan="1" rowspan="1">Proteus</td><td colspan="1" rowspan="1">22 (6.3)</td><td colspan="1" rowspan="1">9 (10.2)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">5(5.1)</td><td colspan="1" rowspan="1">5 (8.6)</td><td colspan="1" rowspan="1">2 (4.3)</td><td colspan="1" rowspan="1">1 (3.6)</td><td colspan="1" rowspan="1">0 (0.0)</td></tr><tr><td colspan="1" rowspan="1">Proteus mirabilis</td><td colspan="1" rowspan="1">22 (6.3)</td><td colspan="1" rowspan="1">9 (10.2)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">5(5.1)</td><td colspan="1" rowspan="1">5(8.6)</td><td colspan="1" rowspan="1">2 (4.3)</td><td colspan="1" rowspan="1">1 (3.6)</td><td colspan="1" rowspan="1">0 (0.0)</td></tr><tr><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">28 (8.0)</td><td colspan="1" rowspan="1">5 (5.7)</td><td colspan="1" rowspan="1">2 (8.7)</td><td colspan="1" rowspan="1">10(10.2)</td><td colspan="1" rowspan="1">8 (13.8)</td><td colspan="1" rowspan="1">2 (4.3)</td><td colspan="1" rowspan="1">1 (3.6)</td><td colspan="1" rowspan="1">0 (0.0)</td></tr><tr><td colspan="1" rowspan="1">Salmonella</td><td colspan="1" rowspan="1">2 (0.6)</td><td colspan="1" rowspan="1">1 (1.1)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">1 (2.2)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td></tr><tr><td colspan="1" rowspan="1">Serratia</td><td colspan="1" rowspan="1">10 (2.9)</td><td colspan="1" rowspan="1">1 (1.1)</td><td colspan="1" rowspan="1">2 (8.7)</td><td colspan="1" rowspan="1">1 (1.0)</td><td colspan="1" rowspan="1">3 (5.2)</td><td colspan="1" rowspan="1">3 (6.5)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td></tr><tr><td colspan="1" rowspan="1">Serratia marcescens</td><td colspan="1" rowspan="1">9 (2.6)</td><td colspan="1" rowspan="1">1 (1.1)</td><td colspan="1" rowspan="1">2 (8.7)</td><td colspan="1" rowspan="1">1 (1.0)</td><td colspan="1" rowspan="1">3 (5.2)</td><td colspan="1" rowspan="1">2 (4.3)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td></tr><tr><td colspan="1" rowspan="1">Stenotrophomonas maltophilia</td><td colspan="1" rowspan="1">3 (0.9)</td><td colspan="1" rowspan="1">1 (1.1)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">1 (1.7)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">1 (3.6)</td><td colspan="1" rowspan="1">0 (0.0)</td></tr><tr><td colspan="1" rowspan="1">Pan Candida</td><td colspan="1" rowspan="1">2 (0.6)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">1 (1.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">1 (3.6)</td><td colspan="1" rowspan="1">0 (0.0)</td></tr><tr><td colspan="1" rowspan="1">Target</td><td colspan="1" rowspan="1">All Sites(N=349)n(%)</td><td colspan="1" rowspan="1">Site 1(N=88)n (%)</td><td colspan="1" rowspan="1">Site 2(N=23)n (%)</td><td colspan="1" rowspan="1">Site 3(N=98)n (%)</td><td colspan="1" rowspan="1">Site 4(N=58)n (%)</td><td colspan="1" rowspan="1">Site 5(N=46)n (%)</td><td colspan="1" rowspan="1">Site 6(N=28)n (%)</td><td colspan="1" rowspan="1">Site 7(N=8n (%)</td></tr><tr><td colspan="1" rowspan="1">Pan Gram-Positive</td><td colspan="1" rowspan="1">24 (6.9)</td><td colspan="1" rowspan="1">15(17.0)</td><td colspan="1" rowspan="1">1 (4.3)</td><td colspan="1" rowspan="1">5 (5.1)</td><td colspan="1" rowspan="1">1 (1.7)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">2 (7.1)</td><td colspan="1" rowspan="1">0 (0.0)</td></tr><tr><td colspan="1" rowspan="1">CTX-M</td><td colspan="1" rowspan="1">24 (6.9)</td><td colspan="1" rowspan="1">9 (10.2)</td><td colspan="1" rowspan="1">1 (4.3)</td><td colspan="1" rowspan="1">3 (3.1)</td><td colspan="1" rowspan="1">4 (6.9)</td><td colspan="1" rowspan="1">5 (10.9)</td><td colspan="1" rowspan="1">2 (7.1)</td><td colspan="1" rowspan="1">0 (0.0)</td></tr><tr><td colspan="1" rowspan="1">IMP</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td></tr><tr><td colspan="1" rowspan="1">KPC</td><td colspan="1" rowspan="1">3 (0.9)</td><td colspan="1" rowspan="1">1 (1.1)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">1 (1.0)</td><td colspan="1" rowspan="1">1 (1.7)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td></tr><tr><td colspan="1" rowspan="1">NDM</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td></tr><tr><td colspan="1" rowspan="1">OXA</td><td colspan="1" rowspan="1">1 (0.3)</td><td colspan="1" rowspan="1">1 (1.1)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td></tr><tr><td colspan="1" rowspan="1">VIM</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td></tr></table>

# N. Instrument Name:

ePlex Instrument

# O. System Descriptions:

1. Modes of Operation:

Does the applicant’s device contain the ability to transmit data to a computer, webserver, or mobile device?

Yes X __ or No

After specimens have been loaded in the ePlex BCID-GN Panel cartridge, the remaining processing steps are executed under control of the ePlex instrument.

Does the applicant’s device transmit data to a computer, webserver, or mobile device   
using wireless transmission?   
Yes _X or No

2. Software:

FDA has reviewed applicant’s Hazard Analysis and software development processes for   
this line of product types:   
Yes _ _X_ or No

3. Specimen Identification: Specimen identification can be entered manually or via barcode.

4. Specimen Sampling and Handling:

The operator opens an ePlex BCID-GN Panel cartridge pouch and writes the accession ID or places a barcode label with accession ID on the ePlex BCID-GN Panel cartridge. After mixing the blood culture bottle by inversion, the operator transfers sufficient volume of blood culture specimen from the bottle to a sterile tube. The operator then pipettes $5 0 ~ \mu \mathrm { l }$ of the positive blood culture specimen into the sample loading port of the ePlex BCID-GN Panel cartridge. The barcode scanner will read both the accession ID barcode and the cartridge barcode. The loaded cartridge is then inserted into the bay and the BCID-GN Panel test is initiated.

5. Calibration: N/A

6. Quality Control: See M.1.c above.

P. Other Supportive Instrument Performance Characteristics Data Not Covered In The “Performance Characteristics” Section above:

N/A

# Q. Proposed Labeling:

The labeling supports the finding of substantial equivalence for this device.

# R. Conclusion:

The submitted information in this premarket notification is complete and supports a substantial equivalence decision.