# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATION DECISION SUMMARY

A. 510(k) Number:

# B. Purpose for Submission:

To obtain clearance for the ePlex Blood Culture Identification Fungal Pathogen (BCID-FP) Panel

# C. Measurand:

Candida albicans, Candida auris, Candida dubliniensis, Candida famata, Candida glabrata, Candida guilliermondii, Candida kefyr, Candida krusei, Candida lusitaniae, Candida parapsilosis, Candida tropicalis, Cryptococcus gattii, Cryptococcus neoformans, Fusarium, and Rhodotorula.

# D. Type of Test:

A multiplexed nucleic acid-based test intended for use with the GenMark’s ePlex instrument for the qualitative in vitro detection and identification of multiple, pathogenic fungal organisms. The BCID-FP assay is performed directly on positive blood culture samples that demonstrate the presence of organisms as determined by Gram stain.

# E. Applicant:

GenMark Diagnostics, Incorporated

F. Proprietary and Established Names:

ePlex Blood Culture Identification Fungal Pathogen (BCID-FP) Panel

# G. Regulatory Information:

1. Regulation section:

21 CFR 866.3365 - Multiplex Nucleic Acid Assay for Identification of Microorganisms and Resistance Markers from Positive Blood Cultures

2. Classification: Class II

3. Product codes:

PEO

4. Panel:

83 (Microbiology)

# H. Intended Use:

1. Intended use(s):

The GenMark ePlex Blood Culture Identification Fungal Pathogen (BCID-FP) Panel is a qualitative nucleic acid multiplex in vitro diagnostic test intended for use on GenMark’s ePlex Instrument for simultaneous detection and identification of multiple potentially pathogenic fungal organisms in positive blood culture. The ePlex BCID-FP Panel is performed directly on blood culture samples identified as positive by a continuous monitoring blood culture system and which contain fungal organism.

The following fungal organisms are identified using the ePlex BCID-FP Panel: Candida albicans, Candida auris, Candida dubliniensis, Candida famata, Candida glabrata, Candida guilliermondii, Candida kefyr, Candida krusei, Candida lusitaniae, Candida parapsilosis, Candida tropicalis, Cryptococcus gattii, Cryptococcus neoformans, Fusarium and Rhodotorula.

The detection and identification of specific fungal nucleic acids from individuals exhibiting signs and/or symptoms of bloodstream infection aids in the diagnosis of bloodstream infection when used in conjunction with other clinical information. The results from the ePlex BCID-FP Panel are intended to be interpreted in conjunction with Gram stain results and should not be used as the sole basis for diagnosis, treatment, or other patient management decisions.

Negative results in the setting of a suspected bloodstream infection may be due to infection with pathogens that are not detected by this test. Positive results do not rule out co-infection with other organisms; the organism(s) detected by the ePlex BCID-FP Panel may not be the definite cause of disease. Additional laboratory testing (e.g. sub-culturing of positive blood cultures for identification of organisms not detected by ePlex BCID-FP Panel, susceptibility testing and differentiation of mixed growth) and clinical presentation must be taken into consideration in the final diagnosis of bloodstream infection.

# 2. Indication(s) for use:

Same as Intended Use

3. Special conditions for use statement(s):

For prescription use only

# Limitations :

• For prescription use only.   
• This test is a qualitative test and does not provide a quantitative value.   
• This product should not be used with blood culture media that contains charcoal.   
• False results were observed for some targets in a single lot of BACT/Alert PF Plus and BACT/Alert FA Plus bottle types (see the Sample Matrix Equivalency (Bottle Evaluation) section of the package insert for additional details). There is a risk of false negative values due to the presence of sequence variants in the fungal targets of the test. In mixed cultures, the ePlex BCID-FP Panel may not identify all organisms in the specimen, depending upon the concentration of each target present. The results of the ePlex BCID-FP Panel should not be used as the sole basis for diagnosis, treatment or other patient management decisions.   
• The genus level assays included as a part of the BCID-FP Panel (Fusarium, Rhodotorula) are designed to detect a broad range of species but will not necessarily detect all species within a genus or group.

4. Special instrument requirements:

For use with the GenMark ePlex instrument

# I. Device Description:

The ePlex Blood Culture Identification Fungal Pathogen (BCID-FP) Panel is based on the principles of competitive nucleic acid hybridization using a sandwich assay format, wherein a single-stranded target binds concurrently to a sequence-specific solution-phase signal probe and a solid-phase electrode-bound capture probe. The test employs nucleic acid extraction, target amplification via polymerase chain reaction (PCR) or reverse transcription PCR (RT-PCR) and hybridization of target DNA. In the process, the double-stranded PCR amplicons are digested with exonuclease to generate single-stranded DNA suitable for hybridization.

Nucleic acid extraction from biological samples occurs within the cartridge via cell lysis, nucleic acid capture onto magnetic beads, and release for amplification. The nucleic acid extraction is processed through microfluidic liquid handling. Once the nucleic acid targets are captured and inhibitors are washed away, the magnetic particles are delivered to the electrowetting environment on the printed circuit board (PCB) and the targets are eluted from the particles and amplified.

During hybridization, the single-stranded target DNA binds to a complementary, single-stranded capture probe immobilized on the working gold electrode surface. Single-stranded signal probes (labeled with electrochemically active ferrocenes) bind to specific target sequence / region adjacent to the capture probe. Simultaneous hybridization of target to signal probes and capture probe is detected by alternating current voltammetry (ACV). Each working electrode on the array contains specific capture probes, and sequential analysis of each electrode allows detection of multiple analyte targets.

Table 1: Materials provided in each kit:   

<table><tr><td rowspan=1 colspan=1>Product</td><td rowspan=1 colspan=1>Ite m number</td><td rowspan=1 colspan=1>Compone nts (quantity)</td><td rowspan=1 colspan=1>Storage</td></tr><tr><td rowspan=1 colspan=1>ePlex Blood CultureIdentification FungalPathogen (BCID-FP) Panel</td><td rowspan=1 colspan=1>EA005012</td><td rowspan=1 colspan=1>ePlex BCID-FP Panel Cartridge (12)</td><td rowspan=1 colspan=1>28 </td></tr></table>

# Materials required but not provided:

• GenMark ePlex Instrument and Software   
• Pipettes capable of delivering $5 0 \mu \mathrm { L }$   
• Printer (optional) - See ePlex Operator Manual for compatibility guidelines   
• Pipette tips, aerosol resistant, RNase/DNase-free   
• Disposable, powder free gloves   
• $10 \%$ bleach for appropriate surfaces   
• $70 \%$ ethanol or isopropyl alcohol (or equivalent) for appropriate surfaces   
• $1 . 5 \mathrm { m L }$ RNase/DNase-free microcentrifuge tube or equivalent (optional)

# Interpretation of Results

Results interpretation of the ePlex BCID-FP Panel is performed by the ePlex instrument, and the interpretation of results on the ePlex BCID-FP Panel Detection Report for each targeted analyte is summarized in Table 2 below.

Table 2: Interpretation of Results on the ePlex BCID-FP Panel Detection Report   

<table><tr><td colspan="1" rowspan="1">Target Result</td><td colspan="1" rowspan="1">Explanation</td><td colspan="1" rowspan="1">Action</td></tr><tr><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">The test was completed successfully and the targethas generated signal above its defined thresholdand the Internal Control was reported as PASS.</td><td colspan="1" rowspan="1">All results are displayedon the ePlex BCID-FPPanel Detection Report.Test is valid, report results.</td></tr><tr><td colspan="1" rowspan="1">Not Detected</td><td colspan="1" rowspan="1">The test was completed successfully and thetarget did not generate signal above its definedthreshold and the Internal Control was reportedas PASS.</td><td colspan="1" rowspan="1">All results are displayedon the ePlex BCID-FPPanel Detection Report.Test is valid, report results.</td></tr><tr><td>Invalid</td><td>The test has not successfully completed and results for this test are not valid. This may be due to an instrument or software error.</td><td>Panel Detection Report.</td></tr></table>

# ePlex BCID-FP Panel Test Reports

Several different reports are available on the ePlex System. Results are provided in a printable format and may be viewed electronically or exported for additional analysis. Reports can be customized with account specific information such as the address, logo and institutional specific footers on each report.

# Detection Report

The ePlex BCID-FP Panel Detection Report includes the results for each individual sample run on the ePlex System. The Summary section indicates the overall test result and lists all detected targets in that sample. The Results section includes a list of all targets on the panel with an individual result for each target. Results are reported as Detected, Not Detected, or Invalid (displayed as a red x); results for the Internal Control are reported as PASS, FAIL, or INVALID.

# External Control Report

The ePlex BCID-FP Panel External Control Report is generated for an external control that has been pre-defined in the ePlex BCID-FP Panel Software. For more information on defining external controls on the ePlex System, refer to the ePlex Operator Manual.

The Summary section indicates the overall result (PASS or FAIL status) and lists all detected targets for that external control. The Results section includes a list of all panel targets with the result, expected result and PASS/FAIL status for each. Results are reported as Detected, Not Detected, or Invalid (displayed as a red x). A target is reported as PASS if the actual result matches the expected result (as defined for that control); a target is reported as FAIL if the actual result does not match the expected result. If the actual result for each target matches the expected result (all targets reported as PASS), the overall result for the external control is reported as PASS in the Summary section. If the actual result for any target does not match the expected result, the overall result for the external control is reported as FAIL in the Summary section.

# Summary Report

The Summary Report allows the operator to use searchable criteria to create customized reports, using specified targets, dates, range of dates, sample, external control, test bay, or operator. For more information on creating Summary Reports, refer to the ePlex Operator Manual.

# J. Substantial Equivalence Information:

1. Predicate device name(s): BioFire Diagnostics; FilmArray Blood Culture Identification (BCID) Panel

2. Predicate 510(k) number(s): k130914

3. Comparison with predicate:

<table><tr><td colspan="3" rowspan="1">Similarities</td></tr><tr><td colspan="1" rowspan="1">Ite m</td><td colspan="1" rowspan="1">Device: ePlex BCID-FP Panel(K182690)</td><td colspan="1" rowspan="1">Predicate: FilmArray BCID Panel(K130914)</td></tr><tr><td colspan="1" rowspan="1">Indication forUse</td><td colspan="1" rowspan="1">The GenMark ePlex Blood CultureIdentification Fungal Pathogen (BCID-FP)Panel is a qualitative nucleic ac id multiplex invitro diagnostic test intended for use onGenMark's ePlex Instrument for simultaneousdetection and identific ation of multiplepotentially pathogenic fungal organisms inpositive blood culture. The ePlex BCID-FPPanel is performed directly on blood culturesamples identified as positive by a continuousmonitoring blood culture system and whichcontain fungal organism.The follow ing fungal organisms are identifiedusing the ePlex BCID-FP Panel: Candidaalbicans, Candida auris, Candidadubliniensis, Candida famata, Candidaglabrata, Candida guilliermondii, Candidakefyr, Candida krusei, Candida lusitaniae,Candida parapsilosis, Candida tropicalis,Cryptococcus gattii, Cryptococcusneoformans, Fusarium and Rhodotorula.The detection and identification of spec ificfungal nucleic ac ids from individualsexhibiting signs and/or symptoms ofbloodstream infection aids in the diagnosis ofbloodstream infection when used inconjunction with other clinical information.The results from the ePlex BCID-FP Panel areintended to be interpreted in conjunction withGram stain results and should not be used asthe sole basis for diagnosis, treatment, orother patient management decisions.</td><td colspan="1" rowspan="1">The FilmArray Blood Culture Identific ation(BCID) Panel is a qualitative multiplexednuc leic acid-based in vitro diagnostic testintended for use with the FilmArrayInstrument. The FilmArray BCID Panel iscapable of simultaneous detection andidentification of multiple bacterial and yeastnuc leic acids and select genetic determinantsof antimicrobial resistance. The BCID assay isperformed directly on blood culture samplesidentified as positive by a continuousmonitoring blood culture system thatdemonstrates the presence of organisms asdetermined by Gram stain.The following gram-positive bacteria, gram-negative bacteria, and yeast are identifiedusing the FilmArray BCID Panel:Enterococci, Listeria monocytogenes,commonly encountered Staphylococci(including specific differentiation ofStaphylococcus aureus), commonlyencountered Streptococci (with specificdifferentiation of Streptococcus agalactiae,Streptococcus pneumoniae, and Streptococcuspyogenes), Acinetobacter baumannii,commonly encountered Enterobacteriaceae(including spec ific differentiation of theEnterobacter cloacae complex, Escherichiacoli, Klebsiella oxytoca, Klebsiellapneumoniae, Proteus, and Serratiamarcescens), Haemophilus influenzae,Neisseria meningitidis (encapsulated),Pseudomonas aeruginosa, Candida albicans,</td></tr><tr><td></td><td>Negative results in the setting of a suspected bloodstream infection may be due to infection with pathogens that are not detected by this test. Positive results do not rule out co- infection with other organisms; the organism(s) detected by the ePlex BCID-FP Panel may not be the definite cause of disease. Additional laboratory testing (e.g. sub- culturing of positive blood cultures for identification of organisms not detected by ePlex BCID-FP Panel, susceptibility testing and differentiation of mixed growth) and c linical presentation must be taken into consideration in the final diagnosis of bloodstream infection.</td><td>Candida glabrata, Candida krusei, Candida parapsilosis, and Candida tropicalis. The FilmArray BCID Panel also contains assays for the detection of genetic determinants of resistance to methic illin (mecA), vancomycin (vanA and vanB), and carbapenems (blakpc) to aid in the identific ation of potentially antimicrobial resistant organisms in positive blood culture samples. The antimicrobial resistance gene detected may or may not be associated with the agent responsible for disease. Negative results for these select antimicrobial resistance gene assays do not indicate susceptibility, as multiple mechanisms of resistance to methicillin, vancomycin, and carbapenems exist. FilmArray BCID is indicated as an aid in the diagnosis of spec ific agents of bacteremia and fungemia and results should be used in conjunction with other clinical and laboratory findings. Positive FilmArray results do not rule out co-infection with organisms not included in the FilmArray BCID Panel. FilmArray BCID is not intended to monitor treatment for bacteremia or fungemia.</td></tr><tr><td></td><td></td><td>Subculturing of positive blood cultures is necessary to recover organisms for susceptibility testing and epidemiological typing, to identify organisms in the blood culture that are not detected by the FilmArray BCID Panel, and for species determination of some Staphylococci, Enterococci, Streptococci, and Enterobacteriaceae that are not specifically identified by the FilmArray</td></tr><tr><td>Analyte</td><td>DNA</td><td>BCID Panel assays DNA</td></tr><tr><td>Sample Processing</td><td>Automated by instrument</td><td>Automated by instrument</td></tr></table>

<table><tr><td colspan="3" rowspan="1">Differences</td></tr><tr><td colspan="1" rowspan="1">Ite m</td><td colspan="1" rowspan="1">Device: ePlex BCID-FP Panel(k182690)</td><td colspan="1" rowspan="1">Predicate: FilmArray BCID Panel(k130914)</td></tr><tr><td colspan="1" rowspan="1">Specimen Type</td><td colspan="1" rowspan="1">Gram-Fungal Blood Culture</td><td colspan="1" rowspan="1">Gram-Positive &amp; Gram-Negative Blood</td></tr><tr><td colspan="3" rowspan="1">Diffe rences</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Culture</td></tr><tr><td colspan="1" rowspan="1">Test Principles</td><td colspan="1" rowspan="1">Reagents on cartridge include: samplelysis and nucleic acid extraction, PCRamplification and hybridization-basedelectrochemical detection reagents.</td><td colspan="1" rowspan="1">The FilmArray BCID pouch containsfreeze-dried reagents to perform nucleicacid purification and nested, multiplexPCR with DNA melt analysis.</td></tr><tr><td colspan="1" rowspan="1">Ins trume ntationand Hardware</td><td colspan="1" rowspan="1">GenMark ePlex Instrument &amp; Single UseCartridge</td><td colspan="1" rowspan="1">FilmArray Instrument and assay pouch</td></tr><tr><td colspan="1" rowspan="1">SoftwareInte rfaceResultReporting</td><td colspan="1" rowspan="1">GenMark ePlex System SoftwareGenMark ePlex BCID-FP Panel Software</td><td colspan="1" rowspan="1">The FilmArray Software automaticallyinterprets the results of each DNA meltcurve analysis and combines the datawith the results of the internal pouchcontrols to provide a test result for eachorganism and antimicrobial resistancegene on the panel.</td></tr></table>

# K. Standard/Guidance Document Referenced

Class II Special Controls Guideline: Multiplex Nucleic Acid Assay for Identification of Microorganisms and Resistance Markers from Positive Blood Cultures (May 2015)   
• CLSI MM17-A, Vol. 28, No. 9, Verification and Validation of Multiplex Nucleic Acid Assays   
• CLSI EP17-A2 Evaluation of Detection Capability for Clinical Laboratory Measurement Procedures; Approved Guideline – Second Edition (June 2012)   
• CLSI EP07-A2: Interference Testing in Clinical Chemistry; Approved Guideline – Second Edition (November 2005) CLSP EP25-A Evaluation of Stability of In Vitro Diagnostic Reagents; Approved Guideline (May 2013)

# L. Test Principle:

The ePlex Blood Culture Identification Gram-Positive (BCID-FP) Panel is based on the principles of competitive nucleic acid hybridization using a sandwich assay format, wherein a single-stranded target binds concurrently to a sequence-specific solution-phase signal probe and a solid-phase electrode-bound capture probe. The test employs nucleic acid extraction, target amplification via polymerase chain reaction (PCR) or reverse transcription PCR (RT-PCR) and hybridization of target DNA. In the process, the double-stranded PCR amplicons are digested with exonuclease to generate single-stranded DNA suitable for hybridization.

Nucleic acid extraction from biological samples occurs within the cartridge via cell lysis, nucleic acid capture onto magnetic beads, and release for amplification. The nucleic acid extraction is processed through microfluidic liquid handling. Once the nucleic acid targets are captured and inhibitors are washed away, the magnetic particles are delivered to the electrowetting environment on the printed circuit board (PCB) and the targets are eluted from the particles and amplified.

During hybridization, the single-stranded target DNA binds to a complementary, single-stranded capture probe immobilized on the working gold electrode surface. Single-stranded signal probes (labeled with electrochemically active ferrocenes) bind to specific target sequence/region adjacent to the capture probe. Simultaneous hybridization of target to signal probes and capture probe is detected by alternating current voltammetry (ACV). Each working electrode on the array contains specific capture probes, and sequential analysis of each electrode allows detection of multiple analyte targets.

# M. Performance Characteristics:

# 1. Analytical performance:

# a. Analytical Sensitivity:

The limit of detection (LoD), or analytical sensitivity, was identified and verified for each assay on the BCID-FP Panel using quantified reference strains in simulated blood culture sample matrix, which is defined as the matrix from a negative blood culture bottle mixed with whole blood and EDTA in the same ratio as the manufacturer recommends for blood culture. At least 20 replicates per target were tested for each condition. The limit of detection was defined as the lowest concentration of each target that is detected in $2 9 5 \%$ of tested replicates. The confirmed LoD for each ePlex BCID-FP Panel organism is shown in Table 3 below.

Table 3: BCID-FP Panel LoD Results Summary   

<table><tr><td colspan="1" rowspan="1">Target</td><td colspan="1" rowspan="1">Organism</td><td colspan="1" rowspan="1">Strain</td><td colspan="1" rowspan="1">LODConce ntration(CFU/mL)</td></tr><tr><td colspan="1" rowspan="2">Candida albicans</td><td colspan="1" rowspan="1">Candida albicans</td><td colspan="1" rowspan="1">ATCC 14053</td><td colspan="1" rowspan="1">1 x 106</td></tr><tr><td colspan="1" rowspan="1">Candida albicans</td><td colspan="1" rowspan="1">ATCC 24433</td><td colspan="1" rowspan="1">1 x 105</td></tr><tr><td colspan="1" rowspan="2">Candida auris</td><td colspan="1" rowspan="1">Candida auris</td><td colspan="1" rowspan="1">CBS 10913</td><td colspan="1" rowspan="1">1 x 105</td></tr><tr><td colspan="1" rowspan="1">Candida auris</td><td colspan="1" rowspan="1">CBS 12766</td><td colspan="1" rowspan="1">1 x 105</td></tr><tr><td colspan="1" rowspan="2">Candida dubliniensis</td><td colspan="1" rowspan="1">Candida dubliniensis</td><td colspan="1" rowspan="1">ATCC MYA-577</td><td colspan="1" rowspan="1">1 x 104</td></tr><tr><td colspan="1" rowspan="1">Candida dubliniensis</td><td colspan="1" rowspan="1">NCPF 3949</td><td colspan="1" rowspan="1">1 x 105</td></tr><tr><td colspan="1" rowspan="2">Candida famata</td><td colspan="1" rowspan="1">Candida famata</td><td colspan="1" rowspan="1">CBS 767</td><td colspan="1" rowspan="1">1 x 103</td></tr><tr><td colspan="1" rowspan="1">Candida famata</td><td colspan="1" rowspan="1">CBS 766</td><td colspan="1" rowspan="1">1 x 104</td></tr><tr><td colspan="1" rowspan="2">Candida glabrata</td><td colspan="1" rowspan="1">Candida glabrata</td><td colspan="1" rowspan="1">ATCC 2001</td><td colspan="1" rowspan="1">1 x 106</td></tr><tr><td colspan="1" rowspan="1">Candida glabrata</td><td colspan="1" rowspan="1">ATCC 15545</td><td colspan="1" rowspan="1">1 x 106</td></tr><tr><td colspan="1" rowspan="2">Candidaguilliermondii</td><td colspan="1" rowspan="1">Candida guilliermondii</td><td colspan="1" rowspan="1">ATCC 22017</td><td colspan="1" rowspan="1">1 x 105</td></tr><tr><td colspan="1" rowspan="1">Candida guilliermondii</td><td colspan="1" rowspan="1">ATCC 6260</td><td colspan="1" rowspan="1">1 x 105</td></tr><tr><td colspan="1" rowspan="1">Target</td><td colspan="1" rowspan="1">Organis m</td><td colspan="1" rowspan="1">Strain</td><td colspan="1" rowspan="1">LODConce ntration(CFU/mL)</td></tr><tr><td colspan="1" rowspan="2">Candida kefyr</td><td colspan="1" rowspan="1">Candida kefyr</td><td colspan="1" rowspan="1">ATCC 4135</td><td colspan="1" rowspan="1">1 x 103</td></tr><tr><td colspan="1" rowspan="1">Candida kefyr</td><td colspan="1" rowspan="1">ATCC 8553</td><td colspan="1" rowspan="1">1 x 104</td></tr><tr><td colspan="1" rowspan="2">Candida krusei</td><td colspan="1" rowspan="1">Candida krusei</td><td colspan="1" rowspan="1">ATCC 22985</td><td colspan="1" rowspan="1">1 x 105</td></tr><tr><td colspan="1" rowspan="1">Candida krusei</td><td colspan="1" rowspan="1">ATCC28870</td><td colspan="1" rowspan="1">1 x 106</td></tr><tr><td colspan="1" rowspan="2">Candida lusitaniae</td><td colspan="1" rowspan="1">Candida lusitaniae</td><td colspan="1" rowspan="1">ATCC34449</td><td colspan="1" rowspan="1">1 x 106</td></tr><tr><td colspan="1" rowspan="1">Candida lusitaniae</td><td colspan="1" rowspan="1">ATCC66035</td><td colspan="1" rowspan="1">1 x 105</td></tr><tr><td colspan="1" rowspan="2">Candidaparapsilosis</td><td colspan="1" rowspan="1">Candida parapsilosis</td><td colspan="1" rowspan="1">ATCC 28474</td><td colspan="1" rowspan="1">1 x 104</td></tr><tr><td colspan="1" rowspan="1">Candida parapsilosis</td><td colspan="1" rowspan="1">ATCC 28475</td><td colspan="1" rowspan="1">1 x 105</td></tr><tr><td colspan="1" rowspan="2">Candida tropicalis</td><td colspan="1" rowspan="1">Candida tropicalis</td><td colspan="1" rowspan="1">ATCC 13803</td><td colspan="1" rowspan="1">1 x 105</td></tr><tr><td colspan="1" rowspan="1">Candida tropicalis</td><td colspan="1" rowspan="1">ATCC 1369</td><td colspan="1" rowspan="1">1 x 106</td></tr><tr><td colspan="1" rowspan="2">Cryptococcus gattii</td><td colspan="1" rowspan="1">Cryptococcus gattii</td><td colspan="1" rowspan="1">ATCCMYA-4877</td><td colspan="1" rowspan="1">1 x 103</td></tr><tr><td colspan="1" rowspan="1">Cryptococcus gattii</td><td colspan="1" rowspan="1">ATCCMYA-4138</td><td colspan="1" rowspan="1">1 x 103</td></tr><tr><td colspan="1" rowspan="2">Cryptococcusneoformans</td><td colspan="1" rowspan="1">Cryptococcus neoformans</td><td colspan="1" rowspan="1">ATCC 208821</td><td colspan="1" rowspan="1">1 x 105</td></tr><tr><td colspan="1" rowspan="1">Cryptococcus neoformans</td><td colspan="1" rowspan="1">ATCC MYA-565</td><td colspan="1" rowspan="1">1 x 105</td></tr><tr><td colspan="1" rowspan="2">Fusarium</td><td colspan="1" rowspan="1">Fusarium oxysporum</td><td colspan="1" rowspan="1">CBS 116611</td><td colspan="1" rowspan="1">1 x 106 spores/mL</td></tr><tr><td colspan="1" rowspan="1">Fusarium solani</td><td colspan="1" rowspan="1">ATCC 36301</td><td colspan="1" rowspan="1">1 x 106 spores/mL</td></tr><tr><td colspan="1" rowspan="2">Rhodotorula</td><td colspan="1" rowspan="1">Rhodotorula mucilaginosa</td><td colspan="1" rowspan="1">ATCC 4058</td><td colspan="1" rowspan="1">1 x 105</td></tr><tr><td colspan="1" rowspan="1">Rhodotorula glutinis</td><td colspan="1" rowspan="1">ATCC 32765</td><td colspan="1" rowspan="1">1 x 105</td></tr></table>

# b. Reproducibility:

A multisite reproducibility study of the ePlex BCID-FP Panel was conducted at three testing sites (two external sites and one internal site) across major potential sources of variability, such as site-to-site, lot-to-lot, day-to-day, and operator-to-operator. One ePlex instrument per study site with four towers was employed in this reproducibility study. Two operators performed testing at each site on six days (five nonconsecutive days) with three unique lots of ePlex BCID-FP Panel cartridges. A reproducibility study panel consisting of four positive panel members including 5 on-panel organisms at 2 concentrations reflecting bottle positivity plus 8 hours and bottle positivity, and 1 negative panel member, was tested in triplicate. Concentrations in the positive mixes reflected those observed at time of bottle positivity plus 8 hours $( \mathrm { B P } { + } 8 )$ and time of bottle positivity (BP) and one mix containing an off-panel organism grown to bottle positivity, which is expected to yield a negative result. Bottle concentrations used in this study are summarized in Table 4.

Table 4: Bottle Positivity Concentrations   

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Bottle PositivityConcentration</td><td rowspan=1 colspan=1>Bottle Positivity +8Hours Concentration</td></tr><tr><td rowspan=1 colspan=1>Candida albicans</td><td rowspan=1 colspan=1>1 x 106 CFU/mL</td><td rowspan=1 colspan=1>1 x 107 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Candida kefyr</td><td rowspan=1 colspan=1>1 x 106 CFU/mL</td><td rowspan=1 colspan=1>1 x 107 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Cryptococcus neoformans</td><td rowspan=1 colspan=1>1 x 107 CFU/mL</td><td rowspan=1 colspan=1>1 x 108 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Fusarium sacchari</td><td rowspan=1 colspan=1>6.5 x106 spores/mL</td><td rowspan=1 colspan=1>6.1 x106 spores/mL</td></tr><tr><td rowspan=1 colspan=1>Rhodotorula mucilaginosa</td><td rowspan=1 colspan=1>1 x 106 CFU/mL</td><td rowspan=1 colspan=1>1 x 107 CFU/mL</td></tr></table>

Summary results for the ePlex BCID-FP Panel reproducibility study are provided in Table 5 below.

Table 5: Summary of Reproducibility Results   

<table><tr><td colspan="1" rowspan="2">BCID-FP Test Result</td><td colspan="1" rowspan="2">Organism TestConcentration</td><td colspan="1" rowspan="2">Site</td><td colspan="3" rowspan="1">Agreement with ExpectedResults</td></tr><tr><td colspan="1" rowspan="1">Agreed/N</td><td colspan="1" rowspan="1">%</td><td colspan="1" rowspan="1">95% CI</td></tr><tr><td colspan="1" rowspan="12">Candida albicans</td><td colspan="1" rowspan="4">Bottle Positive+ 8 Hours(1x107 CFU/mL)</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">34/35</td><td colspan="1" rowspan="1">97.1</td><td colspan="1" rowspan="1">(85.5-99.5)</td></tr><tr><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">35/36</td><td colspan="1" rowspan="1">97.2</td><td colspan="1" rowspan="1">(85.8-99.5)</td></tr><tr><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">(90.4-100)</td></tr><tr><td colspan="1" rowspan="1">All</td><td colspan="1" rowspan="1">105/107</td><td colspan="1" rowspan="1">98.1</td><td colspan="1" rowspan="1">(93.4-99.5)</td></tr><tr><td colspan="1" rowspan="4">Bottle Positive(1x106 CFU/mL)</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">(90.4-100)</td></tr><tr><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">(90.4-100)</td></tr><tr><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">(90.4-100)</td></tr><tr><td colspan="1" rowspan="1">All</td><td colspan="1" rowspan="1">108/108</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">(96.6-100)</td></tr><tr><td colspan="1" rowspan="4">Negative</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">108/108</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">(96.6-100)</td></tr><tr><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">108/108</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">(96.6-100)</td></tr><tr><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">108/108</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">(96.6-100)</td></tr><tr><td colspan="1" rowspan="1">All</td><td colspan="1" rowspan="1">324/324</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">(98.8-100)</td></tr><tr><td colspan="1" rowspan="1">BCID-FP Test Result</td><td colspan="1" rowspan="1">Organism TestConcentration</td><td colspan="1" rowspan="1">Site</td><td colspan="3" rowspan="1">Agreement with ExpectedResults</td></tr><tr><td colspan="1" rowspan="12">Candida kefyr</td><td colspan="1" rowspan="4">Bottle Positive+ 8 Hours(1x107 CFU/mL)</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">(90.4-100)</td></tr><tr><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">(90.4-100)</td></tr><tr><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">(90.4-100)</td></tr><tr><td colspan="1" rowspan="1">All</td><td colspan="1" rowspan="1">108/108</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">(96.6-100)</td></tr><tr><td colspan="1" rowspan="4">Bottle Positive(1x106 CFU/mL)</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">(90.4-100)</td></tr><tr><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">(90.4-100)</td></tr><tr><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">(90.4-100)</td></tr><tr><td colspan="1" rowspan="1">All</td><td colspan="1" rowspan="1">108/108</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">(96.6-100)</td></tr><tr><td colspan="1" rowspan="4">Negative</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">107/107</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">(96.5-100)</td></tr><tr><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">108/108</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">(96.6-100)</td></tr><tr><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">108/108</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">(96.6-100)</td></tr><tr><td colspan="1" rowspan="1">All</td><td colspan="1" rowspan="1">323/323</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">(98.8-100)</td></tr><tr><td colspan="1" rowspan="12">Cryptococcusneoformans</td><td colspan="1" rowspan="4">Bottle Positive+8 Hours(1x108 CFU/mL)</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">35/35</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">(90.1-100)</td></tr><tr><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">(90.4-100)</td></tr><tr><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">(90.4-100)</td></tr><tr><td colspan="1" rowspan="1">All</td><td colspan="1" rowspan="1">107/107</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">(96.5-100)</td></tr><tr><td colspan="1" rowspan="4">Bottle Positive(1x107 CFU/mL)</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">(90.4-100)</td></tr><tr><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">(90.4-100)</td></tr><tr><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">(90.4-100)</td></tr><tr><td colspan="1" rowspan="1">All</td><td colspan="1" rowspan="1">108/108</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">(96.6-100)</td></tr><tr><td colspan="1" rowspan="4">Negative</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">108/108</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">(96.6-100)</td></tr><tr><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">108/108</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">(96.6-100)</td></tr><tr><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">108/108</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">(96.6-100)</td></tr><tr><td colspan="1" rowspan="1">All</td><td colspan="1" rowspan="1">324/324</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">(98.8-100)</td></tr><tr><td colspan="1" rowspan="12">Fusarium</td><td colspan="1" rowspan="4">Bottle Positive+8 Hours(6.12x106 spores/mL)</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">(90.4-100)</td></tr><tr><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">(90.4-100)</td></tr><tr><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">(90.4-100)</td></tr><tr><td colspan="1" rowspan="1">All</td><td colspan="1" rowspan="1">108/108</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">(96.6-100)</td></tr><tr><td colspan="1" rowspan="4">Bottle Positive(6.54x106 spores/mL)</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">(90.4-100)</td></tr><tr><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">(90.4-100)</td></tr><tr><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">(90.4-100)</td></tr><tr><td colspan="1" rowspan="1">All</td><td colspan="1" rowspan="1">108/108</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">(96.6-100)</td></tr><tr><td colspan="1" rowspan="4">Negative</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">107/107</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">(96.5-100)</td></tr><tr><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">108/108</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">(96.6-100)</td></tr><tr><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">108/108</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">(96.6-100)</td></tr><tr><td colspan="1" rowspan="1">All</td><td colspan="1" rowspan="1">323/323</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">(98.8-100)</td></tr><tr><td colspan="1" rowspan="1">BCID-FP Test Result</td><td colspan="1" rowspan="1">Organis m TestConcentration</td><td colspan="1" rowspan="1">Site</td><td colspan="3" rowspan="1">Agreement with ExpectedResults</td></tr><tr><td colspan="1" rowspan="12">Rhodotorula</td><td colspan="1" rowspan="4">Bottle Positive+ 8 Hours(1x107 CFU/mL)</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">(90.4-100)</td></tr><tr><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">34/36</td><td colspan="1" rowspan="1">94.4</td><td colspan="1" rowspan="1">(81.9-98.5)</td></tr><tr><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">35/36</td><td colspan="1" rowspan="1">97.2</td><td colspan="1" rowspan="1">(85.8-99.5)</td></tr><tr><td colspan="1" rowspan="1">All</td><td colspan="1" rowspan="1">105/108</td><td colspan="1" rowspan="1">97.2</td><td colspan="1" rowspan="1">(92.1-99.1)</td></tr><tr><td colspan="1" rowspan="4">Bottle Positive(1x106 CFU/mL)</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">35/36</td><td colspan="1" rowspan="1">97.2</td><td colspan="1" rowspan="1">(85.8-99.5)</td></tr><tr><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">(90.4-100)</td></tr><tr><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">35/36</td><td colspan="1" rowspan="1">97.2</td><td colspan="1" rowspan="1">(85.8-99.5)</td></tr><tr><td colspan="1" rowspan="1">All</td><td colspan="1" rowspan="1">106/108</td><td colspan="1" rowspan="1">98.1</td><td colspan="1" rowspan="1">(93.5-99.5)</td></tr><tr><td colspan="1" rowspan="4">Negative</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">107/107</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">(96.5-100)</td></tr><tr><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">108/108</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">(96.6-100)</td></tr><tr><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">108/108</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">(96.6-100)</td></tr><tr><td colspan="1" rowspan="1">All</td><td colspan="1" rowspan="1">323/323</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">(98.8-100)</td></tr></table>

The initial invalid rate was $4 . 7 \%$ . Initially invalid samples were retested resulting in a final invalid rate of $0 . 2 \%$ .

c. Linearity/assay reportable range: Not applicable

d. Traceability, Stability, Expected values (controls, calibrators, or methods):

# Assay Controls

Internal Controls:   
Each ePlex BCID- FP Panel cartridge includes internal controls that monitor performance of each step of the testing process, including extraction, amplification and detection of targets.

Each amplification reaction on the cartridge has an internal control and in each reaction either the internal control or a target must generate signal above the defined threshold for a valid test result. Internal control results are interpreted by the ePlex Software and displayed on the ePlex BCID-FP Panel Reports as Internal Control with a result of PASS, FAIL, or INVALID. Table 6 includes details on the interpretation of Internal Control results.

Table 6: Internal Control Results   

<table><tr><td rowspan=1 colspan=1>Inte rnal ControlResult</td><td rowspan=1 colspan=1>Explanation</td><td rowspan=1 colspan=1>Action</td></tr><tr><td rowspan=1 colspan=1>PASS</td><td rowspan=1 colspan=1>Signal above threshold has beendetected from each amplificationreaction.The test was completed andinternal controls were successful,indicating valid results weregenerated.</td><td rowspan=1 colspan=1>All results are displayed on theePlex BCID-FP PanelDetection Report.Test is valid, report results.</td></tr><tr><td rowspan=1 colspan=1>FAIL</td><td rowspan=1 colspan=1>Signal above threshold has notbeen detected from at least oneamplification reaction.The test was completed butinternal controls were notdetected, indicating that resultsmay not be valid.</td><td rowspan=1 colspan=1>No results are displayed on theePlex BCID-FP PanelDetection Report.Test is not valid, repeat thetest using a new cartridge.</td></tr><tr><td rowspan=1 colspan=1>INVALID</td><td rowspan=1 colspan=1>An error has occurred duringprocessing that prevents analysisof signal data.The test has not successfullycompleted and results for this testare not valid. This may be due toan instrument or software error.</td><td rowspan=1 colspan=1>No results are displayed on theePlex BCID-FP PanelDetection Report.Test is not valid, repeat thetest using a new cartridge.</td></tr></table>

Recommended External Controls: External controls are not provided with the ePlex Identification Fungal Pathogen (BCID-FP) Panel, but are recommended in the package insert. Positive and negative external controls should be tested with each new lot of reagents or monthly, whichever occurs first. Blood culture medium can be used as the negative control. Previously characterized positive samples or blood culture medium spiked with well characterized organisms can be used as the external positive control. External controls should be run in accordance with laboratory protocols and accrediting organizations, as applicable.

# Specimen Stability

A specimen stability study was performed to confirm the stability of specimens stored under four different temperature conditions over multiple time points which vary based on storage temperature. Additionally, specimen stability was compared at bottle ring and 12 hours after bottle ring.

Organism mixes whose members were at a concentration approximating bottle positivity were stored at $\mathrm { \leq } \mathrm { - } 7 0 ^ { \circ } \mathrm { C }$ , ${ \le } { - } 2 0 ^ { \circ } \mathrm { C }$ , $2 ^ { \circ }$ -

$8 ^ { \circ } \mathrm { C }$ , and ambient temperature over various time points described in Table 7 below. These two mixes include representative targets from each of the four multiplex primer pools on the BCID- FP Panel and include diverse fungal targets. A description of the test mix used for this study is summarized in Table 8.

Table 7: Storage Conditions and Testing Time Points   

<table><tr><td rowspan=1 colspan=1>StorageCondition</td><td rowspan=1 colspan=8>Time Points</td></tr><tr><td rowspan=1 colspan=1>Ambient</td><td rowspan=1 colspan=1>TO</td><td rowspan=1 colspan=1>1 day</td><td rowspan=1 colspan=1>3 days</td><td rowspan=1 colspan=1>5 days</td><td rowspan=1 colspan=1>7 days</td><td rowspan=1 colspan=1>14 days</td><td rowspan=1 colspan=1>30 days</td><td rowspan=1 colspan=1>--</td></tr><tr><td rowspan=1 colspan=1>Refrigerated(2-8°C)</td><td rowspan=1 colspan=1>--</td><td rowspan=1 colspan=1>1 day</td><td rowspan=1 colspan=1>3 days</td><td rowspan=1 colspan=1>5 days</td><td rowspan=1 colspan=1>7 days</td><td rowspan=1 colspan=1>14 days</td><td rowspan=1 colspan=1>30 days</td><td rowspan=1 colspan=1>--</td></tr><tr><td rowspan=1 colspan=1>Frozen(≤-20C)</td><td rowspan=1 colspan=1>1 week</td><td rowspan=1 colspan=1>2 weeks</td><td rowspan=1 colspan=1>1 mo.</td><td rowspan=1 colspan=1>3 mo.</td><td rowspan=1 colspan=1>6 mo.</td><td rowspan=1 colspan=1>12 mo.</td><td rowspan=1 colspan=1>18 mo.</td><td rowspan=1 colspan=1>24 mo.</td></tr><tr><td rowspan=1 colspan=1>Frozen(≤-70°C)</td><td rowspan=1 colspan=1>1 week</td><td rowspan=1 colspan=1>2 weeks</td><td rowspan=1 colspan=1>1 mo.</td><td rowspan=1 colspan=1>3 mo.</td><td rowspan=1 colspan=1>6 mo.</td><td rowspan=1 colspan=1>12 mo.</td><td rowspan=1 colspan=1>18 mo.</td><td rowspan=1 colspan=1>24 mo.</td></tr><tr><td rowspan=1 colspan=1>BottleIncubator</td><td rowspan=1 colspan=1>Bottlepositivity</td><td rowspan=1 colspan=1>12 hourspostpositivity</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>--</td></tr></table>

Table 8: Representative Organism Test Mixes   

<table><tr><td rowspan=1 colspan=1>Mix</td><td rowspan=1 colspan=1>PrimerPool</td><td rowspan=1 colspan=1>Organis m</td><td rowspan=1 colspan=1>Testing Concentration(CFU/mL)</td></tr><tr><td rowspan=3 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>Candida kefyr</td><td rowspan=1 colspan=1>1x106</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>Rhodotorula mucilaginosa</td><td rowspan=1 colspan=1>1x106</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>Fusarium sacchari</td><td rowspan=1 colspan=1>6x106 spores/mL</td></tr><tr><td rowspan=2 colspan=1>2</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>Cryptococcus neoformans</td><td rowspan=1 colspan=1>1x107</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>Candida albicans</td><td rowspan=1 colspan=1>1x106</td></tr></table>

Twenty replicates were tested at the beginning of the study when organism mixes were freshly made (time point T0). Ten replicates were tested for each additional temperature and time point. If any of the targets were not detected in 1 of 10 replicates, an additional 10 replicates were tested to determine whether $9 5 \%$ detection rate could be achieved.

# Ambient Storage Condition

The positivity rate and mean nA signals for all 5 targets for the Ambient Storage temperature were calculated (Table 9). All analytes were $2 9 5 \%$ positive at all time points at ambient temperature storage, with the exception of Rhodotorula. Detection of Rhodotorula was $8 5 \%$ at day 7, but passed the criteria of $2 9 5 \%$ detection at the subsequent two time points (14 and 30 days), which passes overall acceptance criteria for the ambient storage condition.

Table 9: Summary of Ambient Storage Sample Stability   

<table><tr><td rowspan=2 colspan=1>Time Point</td><td rowspan=2 colspan=1>Metric</td><td rowspan=1 colspan=5>Target(nA Cutoff)</td></tr><tr><td rowspan=1 colspan=1>C.albicans(10 nA)</td><td rowspan=1 colspan=1>C. kefyr (25 nA)</td><td rowspan=1 colspan=1>C.neoformans25 nA)</td><td rowspan=1 colspan=1>Fusarium(25 nA)</td><td rowspan=1 colspan=1>Rhodotorula(25 nA)</td></tr><tr><td rowspan=2 colspan=1>TOn=20</td><td rowspan=1 colspan=1>% positive</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Mean Signal (nA)</td><td rowspan=1 colspan=1>899</td><td rowspan=1 colspan=1>791</td><td rowspan=1 colspan=1>825</td><td rowspan=1 colspan=1>671</td><td rowspan=1 colspan=1>263</td></tr><tr><td rowspan=2 colspan=1>1 dayn=10</td><td rowspan=1 colspan=1>% positive</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Mean Signal (nA)</td><td rowspan=1 colspan=1>869.5</td><td rowspan=1 colspan=1>787.2</td><td rowspan=1 colspan=1>1037.6</td><td rowspan=1 colspan=1>714.1</td><td rowspan=1 colspan=1>247.1</td></tr><tr><td rowspan=2 colspan=1>3 daysn=10</td><td rowspan=1 colspan=1>% positive</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Mean Signal (nA)</td><td rowspan=1 colspan=1>818.4</td><td rowspan=1 colspan=1>867.8</td><td rowspan=1 colspan=1>921.6</td><td rowspan=1 colspan=1>585.4</td><td rowspan=1 colspan=1>265.9</td></tr><tr><td rowspan=2 colspan=1>5 daysn=10</td><td rowspan=1 colspan=1>% positive</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Mean Signal (nA)</td><td rowspan=1 colspan=1>599</td><td rowspan=1 colspan=1>388.9</td><td rowspan=1 colspan=1>552.3</td><td rowspan=1 colspan=1>631.5</td><td rowspan=1 colspan=1>282.2</td></tr><tr><td rowspan=2 colspan=1>7 daysn=20 Mix 1n=10 Mix 2</td><td rowspan=1 colspan=1>% positive</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>95%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>95%</td><td rowspan=1 colspan=1>85%</td></tr><tr><td rowspan=1 colspan=1>Mean Signal (nA)</td><td rowspan=1 colspan=1>706.6</td><td rowspan=1 colspan=1>314.6</td><td rowspan=1 colspan=1>680</td><td rowspan=1 colspan=1>358.1</td><td rowspan=1 colspan=1>144.4</td></tr><tr><td rowspan=2 colspan=1>14 daysn=20 Mix 1n=10 Mix 2</td><td rowspan=1 colspan=1>% positive</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>95%</td></tr><tr><td rowspan=1 colspan=1>Mean Signal (nA)</td><td rowspan=1 colspan=1>677.8</td><td rowspan=1 colspan=1>489.8</td><td rowspan=1 colspan=1>643.7</td><td rowspan=1 colspan=1>649.1</td><td rowspan=1 colspan=1>281.6</td></tr><tr><td rowspan=2 colspan=1>30 daysn=10</td><td rowspan=1 colspan=1>% positive</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Mean Signal (nA)</td><td rowspan=1 colspan=1>631.9</td><td rowspan=1 colspan=1>657.2</td><td rowspan=1 colspan=1>617</td><td rowspan=1 colspan=1>587</td><td rowspan=1 colspan=1>252.7</td></tr></table>

# Refrigerated $\left( 2 \small - 8 ^ { \circ } \mathrm { C } \right)$ Storage Condition

The positivity rate and mean nA signals for all 5 targets for the $2 ^ { \circ } C \cdot 8 ^ { \circ } C$ temperature condition were calculated (Table 10). All analytes were $100 \%$ positive for all time points up to 30 days of the $2 ^ { \circ } C \cdot 8 ^ { \circ } C$ condition of the study.

Table 10: Summary of $\underline { { 2 \mathbf { - } 8 ^ { \circ } } } \mathbf { C }$ Storage Sample Stability   

<table><tr><td rowspan=2 colspan=1>Time Point</td><td rowspan=2 colspan=1>Metric</td><td rowspan=1 colspan=5>Target(nA Cutoff)</td></tr><tr><td rowspan=1 colspan=1>C.albicans(10 nA)</td><td rowspan=1 colspan=1>C.kefyr (25nA)</td><td rowspan=1 colspan=1>C. neoformans(25 nA)</td><td rowspan=1 colspan=1>(25 nA)</td><td rowspan=1 colspan=1>Fusarium Rhodotorula(25 nA)</td></tr><tr><td rowspan=2 colspan=1>TOn=20</td><td rowspan=1 colspan=1>% positive</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Mean Signal (nA)</td><td rowspan=1 colspan=1>899</td><td rowspan=1 colspan=1>791</td><td rowspan=1 colspan=1>825</td><td rowspan=1 colspan=1>671</td><td rowspan=1 colspan=1>263</td></tr><tr><td rowspan=2 colspan=1>1 dayn=10</td><td rowspan=1 colspan=1>% positive</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Mean Signal (nA)</td><td rowspan=1 colspan=1>849</td><td rowspan=1 colspan=1>821.7</td><td rowspan=1 colspan=1>970.9</td><td rowspan=1 colspan=1>626.9</td><td rowspan=1 colspan=1>283.9</td></tr><tr><td rowspan=2 colspan=1>3 daysn=10</td><td rowspan=1 colspan=1>% positive</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Mean Signal (nA)</td><td rowspan=1 colspan=1>804.2</td><td rowspan=1 colspan=1>793.5</td><td rowspan=1 colspan=1>849.7</td><td rowspan=1 colspan=1>522.1</td><td rowspan=1 colspan=1>286.5</td></tr><tr><td rowspan=2 colspan=1>5 daysn=10</td><td rowspan=1 colspan=1>% positive</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Mean Signal (nA)</td><td rowspan=1 colspan=1>708.6</td><td rowspan=1 colspan=1>349.6</td><td rowspan=1 colspan=1>656.6</td><td rowspan=1 colspan=1>648.9</td><td rowspan=1 colspan=1>249</td></tr><tr><td rowspan=2 colspan=1>7 daysn=10</td><td rowspan=1 colspan=1>% positive</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Mean Signal (nA)</td><td rowspan=1 colspan=1>568.7</td><td rowspan=1 colspan=1>586.6</td><td rowspan=1 colspan=1>641.5</td><td rowspan=1 colspan=1>648.6</td><td rowspan=1 colspan=1>254.0</td></tr><tr><td rowspan=2 colspan=1>14 daysn=10</td><td rowspan=1 colspan=1>% positive</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Mean Signal (nA)</td><td rowspan=1 colspan=1>647.6</td><td rowspan=1 colspan=1>516.5</td><td rowspan=1 colspan=1>761</td><td rowspan=1 colspan=1>656.7</td><td rowspan=1 colspan=1>280.5</td></tr><tr><td rowspan=2 colspan=1>30 daysn=10</td><td rowspan=1 colspan=1>% positive</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Mean Signal (nA)</td><td rowspan=1 colspan=1>710.2</td><td rowspan=1 colspan=1>525.8</td><td rowspan=1 colspan=1>648.2</td><td rowspan=1 colspan=1>600.3</td><td rowspan=1 colspan=1>282.8</td></tr></table>

# Frozen $( \leq - 2 0 ^ { \circ } \mathrm { C } )$ Storage Condition

The positivity rate and mean nA signals for all 5 targets for the ${ } - 2 0 \ { } ^ { \circ } \mathrm { C }$ temperature storage condition were calculated (Table 11). All analytes were $100 \%$ positive at all time points up to 3 months at ${ } _ { - 2 0 } \mathrm { ~ } ^ { \circ } \mathrm { C }$ temperature storage, with the exception of Rhodotorula. Detection of Rhodotorula was $8 5 \%$ at 30 days, but passed the criteria of $2 9 5 \%$ detection at the subsequent time point (3 months), which passes overall acceptance criteria for the -20 $^ \circ \mathrm { C }$ storage condition.

Table 11: Summary $\leq - 2 0 ^ { \circ } \mathbf { C }$ Storage Sample Stability   

<table><tr><td rowspan=2 colspan=1>Time Point</td><td rowspan=2 colspan=1>Metric</td><td rowspan=1 colspan=5>Target(nA Cutoff)</td></tr><tr><td rowspan=1 colspan=1>C.albicans(10 nA)</td><td rowspan=1 colspan=1>C.kefyr(25 nA)</td><td rowspan=1 colspan=1>C.neoformans(25 nA)</td><td rowspan=1 colspan=1>Fusarium(25 nA)</td><td rowspan=1 colspan=1>Rhodotorula(25 nA)</td></tr><tr><td rowspan=2 colspan=1>TOn=20</td><td rowspan=1 colspan=1>% positive</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Mean Signal(nA)</td><td rowspan=1 colspan=1>899</td><td rowspan=1 colspan=1>791</td><td rowspan=1 colspan=1>825</td><td rowspan=1 colspan=1>671</td><td rowspan=1 colspan=1>263</td></tr><tr><td rowspan=2 colspan=1>7 daysn=10</td><td rowspan=1 colspan=1>% positive</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Mean Signal(nA)</td><td rowspan=1 colspan=1>686.6</td><td rowspan=1 colspan=1>513.6</td><td rowspan=1 colspan=1>705.6</td><td rowspan=1 colspan=1>642.4</td><td rowspan=1 colspan=1>296.6</td></tr><tr><td rowspan=2 colspan=1>14 daysn=10</td><td rowspan=1 colspan=1>% positive</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Mean Signal(nA)</td><td rowspan=1 colspan=1>620.3</td><td rowspan=1 colspan=1>486.5</td><td rowspan=1 colspan=1>637.8</td><td rowspan=1 colspan=1>737.8</td><td rowspan=1 colspan=1>349.5</td></tr><tr><td rowspan=2 colspan=1>30 daysn=20 Mix 1n =10 Mix 2</td><td rowspan=1 colspan=1>% positive</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>85%</td></tr><tr><td rowspan=1 colspan=1>Mean Signal(nA)</td><td rowspan=1 colspan=1>802</td><td rowspan=1 colspan=1>823.6</td><td rowspan=1 colspan=1>930.1</td><td rowspan=1 colspan=1>781.1</td><td rowspan=1 colspan=1>221.3</td></tr><tr><td rowspan=2 colspan=1>3 monthsn=10</td><td rowspan=1 colspan=1>% positive</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Mean Signal(nA)</td><td rowspan=1 colspan=1>756.5</td><td rowspan=1 colspan=1>847.8</td><td rowspan=1 colspan=1>896.2</td><td rowspan=1 colspan=1>730.8</td><td rowspan=1 colspan=1>308.5</td></tr></table>

# Frozen $\mathrm { ( { \leq } - 7 0 ^ { \circ } C ) }$ Storage Condition

The positivity rate and mean nA signals for all 5 targets for $\mathrm { \leq } \mathrm { - } 7 0 ^ { \circ } \mathrm { C }$ temperature storage conditions are shown in Table 12. All analytes were $2 9 5 \%$ positive when stored for up to 3 months at $\mathrm { \leq } \mathrm { - } 7 0 ^ { \circ } \mathrm { C }$ .

Table 12: Summary $\leq - 7 0 ^ { \circ } \mathrm { C }$ Storage Sample Stability   

<table><tr><td rowspan=2 colspan=1>Time Point</td><td rowspan=2 colspan=1>Metric</td><td rowspan=1 colspan=5>Target(nA Cutoff)</td></tr><tr><td rowspan=1 colspan=1>C. albicans(10 nA)</td><td rowspan=1 colspan=1>(25 nA)</td><td rowspan=1 colspan=1>(25 nA)</td><td rowspan=1 colspan=1>(25 nA)</td><td rowspan=1 colspan=1>(25 nA)</td></tr><tr><td rowspan=2 colspan=1>TOn=20</td><td rowspan=1 colspan=1>% positive</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Mean Signal (nA)</td><td rowspan=1 colspan=1>899</td><td rowspan=1 colspan=1>791</td><td rowspan=1 colspan=1>825</td><td rowspan=1 colspan=1>671</td><td rowspan=1 colspan=1>263</td></tr><tr><td rowspan=2 colspan=1>7 daysn=10</td><td rowspan=1 colspan=1>% positive</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Mean Signal (nA)</td><td rowspan=1 colspan=1>627.3</td><td rowspan=1 colspan=1>561.6</td><td rowspan=1 colspan=1>528.4</td><td rowspan=1 colspan=1>704.3</td><td rowspan=1 colspan=1>302</td></tr><tr><td rowspan=2 colspan=1>14 daysn=10</td><td rowspan=1 colspan=1>% positive</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Mean Signal (nA)</td><td rowspan=1 colspan=1>653.4</td><td rowspan=1 colspan=1>474.3</td><td rowspan=1 colspan=1>554.5</td><td rowspan=1 colspan=1>665</td><td rowspan=1 colspan=1>275.9</td></tr><tr><td rowspan=2 colspan=1>30 daysn=20 Mixn=10 Mix</td><td rowspan=1 colspan=1>% positive</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>95%</td></tr><tr><td rowspan=1 colspan=1>Mean Signal (nA)</td><td rowspan=1 colspan=1>778.1</td><td rowspan=1 colspan=1>787.1</td><td rowspan=1 colspan=1>872.9</td><td rowspan=1 colspan=1>625.1</td><td rowspan=1 colspan=1>243.5</td></tr><tr><td rowspan=1 colspan=1>3 months</td><td rowspan=1 colspan=1>% positive</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td></tr></table>

Bottle Ring and 12 Hours Post Bottle Ring   

<table><tr><td rowspan=1 colspan=1>Time Point</td><td rowspan=1 colspan=1>Metric</td><td rowspan=1 colspan=5>Target(nA Cutoff)</td></tr><tr><td rowspan=1 colspan=1>n=10</td><td rowspan=1 colspan=1>Mean Signal (nA)</td><td rowspan=1 colspan=1>808.2</td><td rowspan=1 colspan=1>851.2</td><td rowspan=1 colspan=1>812.5</td><td rowspan=1 colspan=1>716.4</td><td rowspan=1 colspan=1>277.4</td></tr></table>

The positivity rate and mean nA signals for the two representative organisms grown to bottle positivity and incubated 12 hours post positivity were calculated and included below. The positivity rate for $C .$ . albicans and C. neoformans at bottle ring and 12 hours post bottle ring was $100 \%$ for both organisms. Results are summarized in Table 13.

Table 13: Bottle Ring and 12 hours Post Bottle Ring   

<table><tr><td rowspan=1 colspan=1>Target</td><td rowspan=1 colspan=1>Metric</td><td rowspan=1 colspan=1>Bottle Ring(n=20)</td><td rowspan=1 colspan=1>12hrs Post Bottle RingC. albicans (n=10)C. neoformans(n=20)^</td></tr><tr><td rowspan=4 colspan=1>C. albicans</td><td rowspan=1 colspan=1>% positive</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Mean Signal (nA)</td><td rowspan=1 colspan=1>866.3</td><td rowspan=1 colspan=1>707.2</td></tr><tr><td rowspan=1 colspan=1>StandardDeviation</td><td rowspan=1 colspan=1>236.6</td><td rowspan=1 colspan=1>156.8</td></tr><tr><td rowspan=1 colspan=1>Mean minus2 SD</td><td rowspan=1 colspan=1>393.1</td><td rowspan=1 colspan=1>393.6</td></tr><tr><td rowspan=4 colspan=1>C.neoformans</td><td rowspan=1 colspan=1>%positive</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Mean Signal (nA)</td><td rowspan=1 colspan=1>660.2</td><td rowspan=1 colspan=1>804.2</td></tr><tr><td rowspan=1 colspan=1>StandardDeviation</td><td rowspan=1 colspan=1>173.0</td><td rowspan=1 colspan=1>193.1</td></tr><tr><td rowspan=1 colspan=1>Mean minuss 2 SD</td><td rowspan=1 colspan=1>314.2</td><td rowspan=1 colspan=1>417.9</td></tr></table>

^ $\scriptstyle \mathrm { n } = 2 0$ for $C .$ . neoformans $\mathrm { B P } { + } 1 2$ sample, Fusarium false positive was detected in 1 out of 20 replicat es.

Results demonstrated that specimens can be stored under the following temperature conditions without impacting the performance of the BCID-FP Panel:

Up to 1 month at ambient temperature and refrigerated $( 2 ^ { \circ } \mathrm { C } { - } 8 ^ { \circ } \mathrm { C } )$ temperature conditions Up to 3 months at frozen conditions $- 2 0 ^ { \circ } \mathrm { C }$ and $\le - 7 0 ^ { \circ } \mathrm { C }$ ). Specimens can also be tested when incubated up to 12 hours after bottle ring in a continuously monitoring blood culture device.

The specimen stability claims are summarized in the table below.

<table><tr><td rowspan=1 colspan=1>Storage Condition</td><td rowspan=1 colspan=1>Spe cime n Stability</td></tr><tr><td rowspan=1 colspan=1>Ambient</td><td rowspan=1 colspan=1>7 days</td></tr><tr><td rowspan=1 colspan=1>Refrigerated (2°C-8°C)</td><td rowspan=1 colspan=1>1 month</td></tr><tr><td rowspan=1 colspan=1>Frozen (≤-20°)</td><td rowspan=1 colspan=1>3 month</td></tr><tr><td rowspan=1 colspan=1>Frozen (≤-70C)</td><td rowspan=1 colspan=1>3 month</td></tr><tr><td rowspan=1 colspan=1>Post Bottle Ring</td><td rowspan=1 colspan=1>12 hours</td></tr></table>

# Freeze-Thaw Study

A study was performed to assess the tolerance of the ePlex BCID-FP to correctly identify specimens containing fungal organisms that have gone through one or two freeze/thaw cycles prior to testing. One hundred and twelve positive clinical samples that were tested fresh during the BCID-FP Panel Clinical Performance Study were selected to best represent a variety of organisms in clinical samples. The samples were frozen at $\mathrm { { \le - 7 0 ^ { \circ } C } }$ and then thawed once or twice prior to re-testing with the BCID-FP Panel.

The positive agreement with the fresh condition at each freeze-thaw cycle was calculated and was above $9 5 \%$ for each condition. Table 14 and 15 below summarizes the concordance at each freeze-thaw cycle with a $9 5 \%$ confidence level and the mean nA signal.

Table 14: Concordance of Results with Fresh Condition   

<table><tr><td rowspan=1 colspan=1>Metric</td><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>Fresh1x FreezeThaw</td><td rowspan=1 colspan=1>2x FreezeThaw</td></tr><tr><td rowspan=1 colspan=1>Total Valid Samples</td><td rowspan=1 colspan=1>112</td><td rowspan=1 colspan=1>103</td><td rowspan=1 colspan=1>108</td></tr><tr><td rowspan=1 colspan=1>Fals e Ne gative Samples</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>Concordant Samples</td><td rowspan=1 colspan=1>112</td><td rowspan=1 colspan=1>103</td><td rowspan=1 colspan=1>107</td></tr><tr><td rowspan=1 colspan=1>Positive Agreement</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>99.1%</td></tr><tr><td rowspan=1 colspan=1>CI Low</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>96.5%</td><td rowspan=1 colspan=1>93.9%</td></tr><tr><td rowspan=1 colspan=1>CI High</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>99.8%</td></tr></table>

Table 15: Mean Signal Results   

<table><tr><td rowspan=2 colspan=1>Target</td><td rowspan=2 colspan=1>Total Re plicates Mean Signal (nA)</td><td rowspan=1 colspan=3>Total Re plicates Mean Signal (nA)</td></tr><tr><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>1xFT</td><td rowspan=1 colspan=1>2xFT</td></tr><tr><td rowspan=1 colspan=1>Candida albicans</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>834.2</td><td rowspan=1 colspan=1>811.4</td><td rowspan=1 colspan=1>738.2</td></tr><tr><td rowspan=1 colspan=1>Candida auris</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>490.3</td><td rowspan=1 colspan=1>375.5</td><td rowspan=1 colspan=1>362.8</td></tr><tr><td rowspan=1 colspan=1>Candida dubliniensis</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>901.8</td><td rowspan=1 colspan=1>983.2</td><td rowspan=1 colspan=1>912.2</td></tr><tr><td rowspan=1 colspan=1>Candida famata</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>367.6</td><td rowspan=1 colspan=1>503.3</td><td rowspan=1 colspan=1>512.3</td></tr><tr><td rowspan=1 colspan=1>Candida glabrata</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>694.7</td><td rowspan=1 colspan=1>731.4</td><td rowspan=1 colspan=1>685.2</td></tr><tr><td rowspan=1 colspan=1>Candida guilliermondii</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>154.0</td><td rowspan=1 colspan=1>186.1</td><td rowspan=1 colspan=1>73.9</td></tr><tr><td rowspan=1 colspan=1>Candida kefyr</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>811.6</td><td rowspan=1 colspan=1>736.5</td><td rowspan=1 colspan=1>733.5</td></tr><tr><td rowspan=1 colspan=1>Candida krusei</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>713.6</td><td rowspan=1 colspan=1>656.4</td><td rowspan=1 colspan=1>614.6</td></tr><tr><td rowspan=1 colspan=1>Candida lusitaniae</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>716.6</td><td rowspan=1 colspan=1>648.8</td><td rowspan=1 colspan=1>763.8</td></tr><tr><td rowspan=1 colspan=1>Candida parapsilosis</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>778.6</td><td rowspan=1 colspan=1>824.1</td><td rowspan=1 colspan=1>746.7</td></tr><tr><td rowspan=1 colspan=1>Candida tropicalis</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>673.0</td><td rowspan=1 colspan=1>693.8</td><td rowspan=1 colspan=1>702.2</td></tr><tr><td rowspan=1 colspan=1>Cryptococcus gattii</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>698.2</td><td rowspan=1 colspan=1>862.6</td><td rowspan=1 colspan=1>749.7</td></tr><tr><td rowspan=1 colspan=1>Cryptococcus neoformans</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>953.4</td><td rowspan=1 colspan=1>906.6</td><td rowspan=1 colspan=1>871.1</td></tr><tr><td rowspan=1 colspan=1>Fusarium</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>536.1</td><td rowspan=1 colspan=1>700.1</td><td rowspan=1 colspan=1>533.1</td></tr><tr><td rowspan=1 colspan=1>Rhodotorula</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>175.9</td><td rowspan=1 colspan=1>267.2</td><td rowspan=1 colspan=1>263.5</td></tr></table>

Results demonstrate that the BCID-FP Panel can detect the same organisms in samples that were tested fresh and in samples that have been frozen and thawed up to 2 times.

# In-Cartridge Sample Stability Study

An analytical study was carried out to demonstrate performance under the following conditions:

1) Condition 1: The hands-on-time for pipetting the sample into the sample reservoir and starting the consumable run on the ePlex instrument is $\leq 2$ minutes.   
2) Condition 2 (Open Pouch): the cartridge is stable for 2 hours after the pouch has been opened and the unloaded cartridge is exposed to the environment.   
3) Condition 3 (In-Consumable Sample Stability): the cartridge can be stored at room temperature for at least 2 hours after the sample has been loaded prior to running the cartridge in an ePlex bay.

A test mix of four organisms was used in the study. The selected organisms represent all four of the multiplex primer pools on the BCID-FP Panel. Each organism was tested at a concentration approximating bottle positivity. A description of the test mix used for this study is summarized in Table 16 below.

Table 16: Representative Organisms Contained in Test Mix   

<table><tr><td rowspan=1 colspan=1>Organis m</td><td rowspan=1 colspan=1>Conce ntration(CFU/mL)</td><td rowspan=1 colspan=1>PrimerPool</td><td rowspan=1 colspan=1>e Plex BCID-FP PanelExpe cted Result</td></tr><tr><td rowspan=1 colspan=1>Candida albicans</td><td rowspan=1 colspan=1>1 x 106</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>Candida albicans</td></tr><tr><td rowspan=1 colspan=1>Candida kefyr</td><td rowspan=1 colspan=1>1 x 106</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>Candida kefyr</td></tr><tr><td rowspan=1 colspan=1>Rhodotorulamucilaginosa</td><td rowspan=1 colspan=1>1 x 106</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>Rhodotorula</td></tr><tr><td rowspan=1 colspan=1>Cryptococcusneoformans</td><td rowspan=1 colspan=1>1 x 107</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>Cryptococcusneoformans</td></tr></table>

The results of the study which assessed the interim storage conditions are summarized in Table 17. All targets were detected in $100 \%$ of test replicates in each condition tested (Open Pouch Stability Condition and InConsumable Sample Stability Condition)

Table 17: Detection Rates for All Conditions Tested   

<table><tr><td colspan="4">Dete ction Rate (n=20)</td></tr><tr><td>Target</td><td>Control Condition</td><td>Open Pouch Stability</td><td>In-Cons umable Sample Stability Condition</td></tr><tr><td colspan="1" rowspan="1">Candida albicans</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Candida kefyr</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Rhodotorula mucilaginosa</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Cryptococcus neoformans</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td></tr></table>

Hands-on time study results demonstrated that each consumable met the $\leq 2$ minute hands-on time requirement, with an overall average time of 27 seconds. Sample loading time averaged 9 seconds, and consumable loading time on the ePlex Instrument averaged 18 seconds.

# Cartridge Shelf-Life Stability Study

An analytical study was carried out to generate supporting stability data to establish the shelf-life of the ePlex BCID-FP Panel cartridge when stored under refrigerated conditions $\big ( 2 ^ { \mathrm { o } } \mathrm { C } \ : - 8 ^ { \mathrm { o } } \mathrm { C } \big )$ . The study design uses an approach consistent with CLSI guideline EP25-A.

A minimum of three lots of ePlex BCID-FP Panel cartridges were stored under refrigerated conditions and were tested throughout the duration of the real-time stability study. A positive test sample was comprised of a mix containing qualified organisms, each at concentrations approximating bottle positivity (Table 18). This sample mix represents all four multiplex primer pools on the BCID-FP Panel.

Table 18: Representative Organisms Contained in Test Mix   

<table><tr><td rowspan=1 colspan=1>Organis m in TestConce ntration in MixMix</td><td rowspan=1 colspan=1>Organis m in TestConce ntration in Mix(CFU/mL)</td><td rowspan=1 colspan=1>Prime rPool</td><td rowspan=1 colspan=1>ePlex BCID-FP PanelExpected Result</td></tr><tr><td rowspan=1 colspan=1>Candida albicans</td><td rowspan=1 colspan=1>1 x 106</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>Candida albicans</td></tr><tr><td rowspan=1 colspan=1>Candida kefyr</td><td rowspan=1 colspan=1>1 x 106</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>Candida kefyr</td></tr><tr><td rowspan=1 colspan=1>Rhodotorulamucilaginosa</td><td rowspan=1 colspan=1>1 x 106</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>Rhodotorula</td></tr><tr><td rowspan=1 colspan=1>Cryptococcusneoformans</td><td rowspan=1 colspan=1>1 x 107</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>Cryptococcusneoformans</td></tr></table>

On each day of time point testing, positive test samples were made to a final concentration. Positive test samples were created in negative blood matrix. A negative control consisting of negative blood matrix was also included with each time point. The following were tested at each time point; 4 replicates of negative blood matrix as negative control and 50 replicates of positive test samples.

Real-time study data support that the ePlex BCID-FP Panel cartridges continue to be stable for up to ten weeks when stored under refrigerated conditions $( 2 ^ { \mathrm { o } } \mathrm { C } \cdot 8 ^ { \mathrm { o } } \mathrm { C } )$ . The stability study is ongoing and the product will be labeled with an expiration date that is supported by the real-time stability

data from this study.

# e. Growth and Detection Study

A study was performed to establish the range of expected organism concentrations present in incubated blood cultures at bottle positivity (i.e., bottle “ring”) and eight hours after bottle positivity/bottle “ring” Several representative fungal organisms were spiked into blood culture bottles along with the manufacturer’s recommended volume of human whole blood and grown to positivity in a commercially-available continuously monitoring blood culture system. Bottles were removed from the incubator within two hours of being identified as positive as well as eight hours after bottle positivity. At least two independent positive blood culture replicates were quantified for each organism on culture plates. Organisms tested and approximate bottle positivity concentrations are summarized in Table 19. Concentrations shown below represent approximate levels that may be observed in a clinical setting. All estimated bottle positivity concentrations are equivalent or greater than the established Limit of Detection (LOD) for each of the assays of the ePlex BCID-FP Panel.

Table 19: Bottle Positivity Concentrations   

<table><tr><td rowspan=1 colspan=1>Organis m</td><td rowspan=1 colspan=1>StrainID</td><td rowspan=1 colspan=1>Mean Bottle PositivityConcentration</td><td rowspan=1 colspan=1>Mean BottlePositivity+8 hoursConce ntration</td></tr><tr><td rowspan=1 colspan=1>Candida albicans</td><td rowspan=1 colspan=1>ATCC90082</td><td rowspan=1 colspan=1>1.6 x 106 CFU/mL</td><td rowspan=1 colspan=1>1.4 x 106 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Cryptococcus neoformansvar. grubii</td><td rowspan=1 colspan=1>ATCC14116</td><td rowspan=1 colspan=1>1.3 x 107 CFU/mL</td><td rowspan=1 colspan=1>6.5 x 107 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Fusarium solani</td><td rowspan=1 colspan=1>ATCC36031</td><td rowspan=1 colspan=1>9.6 x 106 spores/mL</td><td rowspan=1 colspan=1>7.7 x 106 spores/mL</td></tr><tr><td rowspan=1 colspan=1>Rhodotorulamucilaginosa</td><td rowspan=1 colspan=1>ATCC66034</td><td rowspan=1 colspan=1>1.6 x 106 CFU/mL</td><td rowspan=1 colspan=1>4.2 x 106 CFU/mL</td></tr></table>

# f. Analytical Reactivity (Inclusivity):

The analytical reactivity of the ePlex BCID-FP Panel was evaluated with a collection of 51 fungal organisms that represent the diversity of the ePlex BCID-FP Panel analytes. Isolates were selected to represent relevant species or serotypes and selection with specific inclusion of more commonly encountered species and known human pathogens.

Each isolate was initially tested in blood culture matrix at a concentration consistent with the levels of organism enumerated from blood cultures at the time of bottle positivity $\mathrm { 1 ~ x ~ 1 0 ^ { 6 } ~ C F U / m L }$ for Candida and Rhodotorula, $1 \mathbf { x }$ $1 0 ^ { 7 } \mathrm { \ C F U / m L }$ for Cryptococcus, and $1 \mathrm { ~ x ~ } 1 0 ^ { 8 }$ spores $/ \mathrm { m L }$ for Fusarium) and were tested in triplicate. If an isolate was not detected initially, the sample was retested at higher concentrations. If detected at the higher concentration(s), the species/isolate is indicated as detected with reduced sensitivity and the concentration of organism that was detected is indicated. If not detected at the highest concentration the isolate is listed as not detected by the ePlex BCID-FP Panel. Results are provided in Table 20 below:

Table 20: Analytical Reactivity (Inclusivity) Results for the BCID-FP Panel   

<table><tr><td colspan="1" rowspan="1">Target</td><td colspan="1" rowspan="1">Organis m</td><td colspan="1" rowspan="1">Strain</td></tr><tr><td colspan="1" rowspan="3">Candida albicans</td><td colspan="1" rowspan="1">Candida albicans</td><td colspan="1" rowspan="1">ATCC 10231</td></tr><tr><td colspan="1" rowspan="1">Candida albicans</td><td colspan="1" rowspan="1">ATCC 90028</td></tr><tr><td colspan="1" rowspan="1">Candida albicans</td><td colspan="1" rowspan="1">ATCC MYA-4441</td></tr><tr><td colspan="1" rowspan="7">Candida auris</td><td colspan="1" rowspan="1">Candida auris</td><td colspan="1" rowspan="1">CDC#385</td></tr><tr><td colspan="1" rowspan="1">Candida auris</td><td colspan="1" rowspan="1">CDC#386</td></tr><tr><td colspan="1" rowspan="1">Candida auris</td><td colspan="1" rowspan="1">CDC#387</td></tr><tr><td colspan="1" rowspan="1">Candida auris</td><td colspan="1" rowspan="1">CDC#388</td></tr><tr><td colspan="1" rowspan="1">Candida auris</td><td colspan="1" rowspan="1">CDC#389</td></tr><tr><td colspan="1" rowspan="1">Candida auris</td><td colspan="1" rowspan="1">CDC#390</td></tr><tr><td colspan="1" rowspan="1">Candida auris</td><td colspan="1" rowspan="1">CBS 12766</td></tr><tr><td colspan="1" rowspan="3">Candida dubliniensis</td><td colspan="1" rowspan="1">Candida dubliniensis</td><td colspan="1" rowspan="1">ATCC MYA-578</td></tr><tr><td colspan="1" rowspan="1">Candida dubliniensis</td><td colspan="1" rowspan="1">ATCC MYA-579</td></tr><tr><td colspan="1" rowspan="1">Candida dubliniensis</td><td colspan="1" rowspan="1">ATCC MYA-582</td></tr><tr><td colspan="1" rowspan="3">Candida famata</td><td colspan="1" rowspan="1">Candida famata</td><td colspan="1" rowspan="1">ATCC 20850</td></tr><tr><td colspan="1" rowspan="1">Candida famata</td><td colspan="1" rowspan="1">CBA 1961</td></tr><tr><td colspan="1" rowspan="1">Candida famata</td><td colspan="1" rowspan="1">CBS 789</td></tr><tr><td colspan="1" rowspan="3">Candida glabrata</td><td colspan="1" rowspan="1">Candida glabrata</td><td colspan="1" rowspan="1">ATCC 15126</td></tr><tr><td colspan="1" rowspan="1">Candida glabrata</td><td colspan="1" rowspan="1">ATCC66032</td></tr><tr><td colspan="1" rowspan="1">Candida glabrata</td><td colspan="1" rowspan="1">ATCCMYA-2950</td></tr><tr><td colspan="1" rowspan="3">Candida guilliermondii</td><td colspan="1" rowspan="1">Candida guilliermondii</td><td colspan="1" rowspan="1">ATCC 90197</td></tr><tr><td colspan="1" rowspan="1">Candida guilliermondii</td><td colspan="1" rowspan="1">ATCC90198</td></tr><tr><td colspan="1" rowspan="1">Candida guilliermondii</td><td colspan="1" rowspan="1">ATCC90199</td></tr><tr><td colspan="1" rowspan="3">Candida kefyr</td><td colspan="1" rowspan="1">Candida kefyr</td><td colspan="1" rowspan="1">ATCC 204093</td></tr><tr><td colspan="1" rowspan="1">Candida kefyr</td><td colspan="1" rowspan="1">ATCC 2512</td></tr><tr><td colspan="1" rowspan="1">Candida kefyr</td><td colspan="1" rowspan="1">ATCC66028</td></tr><tr><td colspan="1" rowspan="3">Candida krusei</td><td colspan="1" rowspan="1">Candida krusei</td><td colspan="1" rowspan="1">ATCC14243</td></tr><tr><td colspan="1" rowspan="1">Candida krusei</td><td colspan="1" rowspan="1">ATCC32196</td></tr><tr><td colspan="1" rowspan="1">Candida krusei</td><td colspan="1" rowspan="1">ATCC 34135</td></tr><tr><td colspan="1" rowspan="3">Candida lusitaniae</td><td colspan="1" rowspan="1">Candida lusitaniae</td><td colspan="1" rowspan="1">ATCC 42720</td></tr><tr><td colspan="1" rowspan="1">Candida lusitaniae</td><td colspan="1" rowspan="1">ATCC MYA-766</td></tr><tr><td colspan="1" rowspan="1">Candida lusitaniae</td><td colspan="1" rowspan="1">Z010</td></tr><tr><td colspan="1" rowspan="1">Candida parapsilosis</td><td colspan="1" rowspan="1">Candida parapsilosis</td><td colspan="1" rowspan="1">ATCC 22019</td></tr><tr><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="1">Candida parapsilosis</td><td colspan="1" rowspan="1">ATCC 58895</td></tr><tr><td colspan="1" rowspan="1">Candida parapsilosis</td><td colspan="1" rowspan="1">ATCC90018</td></tr><tr><td colspan="1" rowspan="3">Candida tropicalis</td><td colspan="1" rowspan="1">Candida tropicalis</td><td colspan="1" rowspan="1">ATCC201380</td></tr><tr><td colspan="1" rowspan="1">Candida tropicalis</td><td colspan="1" rowspan="1">ATCC201381</td></tr><tr><td colspan="1" rowspan="1">Candida tropicalis</td><td colspan="1" rowspan="1">ATCC750</td></tr><tr><td colspan="1" rowspan="3">Cryptococcus gattii</td><td colspan="1" rowspan="1">Cryptococcus gattii</td><td colspan="1" rowspan="1">ATCC14248</td></tr><tr><td colspan="1" rowspan="1">Cryptococcus gattii</td><td colspan="1" rowspan="1">ATCC4560</td></tr><tr><td colspan="1" rowspan="1">Cryptococcus gattii</td><td colspan="1" rowspan="1">ATCC76108</td></tr><tr><td colspan="1" rowspan="3">Cryptococcus neoformans</td><td colspan="1" rowspan="1">Cryptococcus neoformans</td><td colspan="1" rowspan="1">ATCC14116</td></tr><tr><td colspan="1" rowspan="1">Cryptococcus neoformans</td><td colspan="1" rowspan="1">ATCC90112</td></tr><tr><td colspan="1" rowspan="1">Cryptococcus neoformans</td><td colspan="1" rowspan="1">NCPF 8299</td></tr><tr><td colspan="1" rowspan="5">Fusarium</td><td colspan="1" rowspan="1">Bisifusarium dimerum</td><td colspan="1" rowspan="1">CBS 110317</td></tr><tr><td colspan="1" rowspan="1">Fusarium moniliforme</td><td colspan="1" rowspan="1">ATCC 38159</td></tr><tr><td colspan="1" rowspan="1">Fusarium proliferatum</td><td colspan="1" rowspan="1">CBS 131570</td></tr><tr><td colspan="1" rowspan="1">Fusarium sacchari</td><td colspan="1" rowspan="1">CBS119828</td></tr><tr><td colspan="1" rowspan="1">Fusarium verticillioides</td><td colspan="1" rowspan="1">CBS100312</td></tr><tr><td colspan="1" rowspan="3">Rhodotorula</td><td colspan="1" rowspan="1">Rhodotorula glutinis</td><td colspan="1" rowspan="1">ATCC96365</td></tr><tr><td colspan="1" rowspan="1">Rhodotorula mucilaginosa</td><td colspan="1" rowspan="1">ATCC66034</td></tr><tr><td colspan="1" rowspan="1">Rhodotorula mucilaginosa</td><td colspan="1" rowspan="1">ATCC9449</td></tr></table>

In addition to species-specific assays, the ePlex BCID-FP Panel contains two broader genus-level assays: Fusarium and Rhodotorula. Table 21 and Table 22 highlight the predicted (in silico) reactivity (inclusivity) for these assay targets.

Table 21: Predicted (in silico) Reactivity (Inclusivity) Results for Fusarium   

<table><tr><td colspan="3" rowspan="1">Detection Predicted for ≥95% of target sequences</td></tr><tr><td colspan="1" rowspan="1">Fusarium acaciae-mearnsii</td><td colspan="1" rowspan="1">Fusarium cortaderiae</td><td colspan="1" rowspan="1">Fusarium napiforme</td></tr><tr><td colspan="1" rowspan="1">Fusarium acuminatum</td><td colspan="1" rowspan="1">Fusarium culmorum</td><td colspan="1" rowspan="1">Fusarium nisikadoi</td></tr><tr><td colspan="1" rowspan="1">Fusarium acutatum</td><td colspan="1" rowspan="1">Fusarium denticulatum</td><td colspan="1" rowspan="1">Fusarium nygamai</td></tr><tr><td colspan="1" rowspan="1">Fusarium aethiopicum</td><td colspan="1" rowspan="1">Bisifusarium dimerum*</td><td colspan="1" rowspan="1">Fusarium oxysporum*</td></tr><tr><td colspan="1" rowspan="1">Fusarium ananatum</td><td colspan="1" rowspan="1">Fusarium dlaminii</td><td colspan="1" rowspan="1">Fusarium palustre</td></tr><tr><td colspan="1" rowspan="1">Fusarium andiyazi</td><td colspan="1" rowspan="1">Fusarium equiseti</td><td colspan="1" rowspan="1">Fusarium phyllophilum</td></tr><tr><td colspan="1" rowspan="1">Fusarium anthophilum</td><td colspan="1" rowspan="1">Fusarium falciforme</td><td colspan="1" rowspan="1">Fusarium poae</td></tr><tr><td colspan="1" rowspan="1">Fusarium armeniacum</td><td colspan="1" rowspan="1">Fusarium foetens</td><td colspan="1" rowspan="1">Fusarium polyphialidicum</td></tr><tr><td colspan="1" rowspan="1">Fusarium asiaticum</td><td colspan="1" rowspan="1">Fusarium fujikuroi</td><td colspan="1" rowspan="1">Fusarium proliferatum*</td></tr><tr><td colspan="1" rowspan="1">Fusarium austroamericanum</td><td colspan="1" rowspan="1">Fusarium gaditjirri</td><td colspan="1" rowspan="1">Fusariumpseudoanthophilum</td></tr><tr><td colspan="1" rowspan="1">Fusarium avenaceum</td><td colspan="1" rowspan="1">Fusarium globosum</td><td colspan="1" rowspan="1">Fusarium pseudocircinatum</td></tr><tr><td colspan="1" rowspan="1">Fusarium aywerte</td><td colspan="1" rowspan="1">Fusarium guttiforme</td><td colspan="1" rowspan="1">Fusariumpseudograminearum</td></tr><tr><td colspan="1" rowspan="1">Fusarium bactridioides</td><td colspan="1" rowspan="1">Fusarium hostae</td><td colspan="1" rowspan="1">Fusarium pseudonygamai</td></tr><tr><td colspan="1" rowspan="1">Fusarium begoniae</td><td colspan="1" rowspan="1">Fusarium incarnatum</td><td colspan="1" rowspan="1">Fusarium ramigenum</td></tr><tr><td colspan="1" rowspan="1">Fusarium beomiforme</td><td colspan="1" rowspan="1">Fusarium inflexum</td><td colspan="1" rowspan="1">Fusarium sacchari*</td></tr><tr><td colspan="1" rowspan="1">Fusarium boothii</td><td colspan="1" rowspan="1">Fusarium konzum</td><td colspan="1" rowspan="1">Fusarium secorum</td></tr><tr><td colspan="1" rowspan="1">Fusarium brachygibbosum</td><td colspan="1" rowspan="1">Fusarium lacertarum</td><td colspan="1" rowspan="1">Fusarium sinensis</td></tr><tr><td colspan="1" rowspan="1">Fusarium brasilicum</td><td colspan="1" rowspan="1">Fusarium lactis</td><td colspan="1" rowspan="1">Fusarium solani*</td></tr><tr><td colspan="1" rowspan="1">Fusarium brevicatenulatum</td><td colspan="1" rowspan="1">Fusarium langsethiae</td><td colspan="1" rowspan="1">Fusarium sporotrichioides</td></tr><tr><td colspan="1" rowspan="1">Fusarium bulbicola</td><td colspan="1" rowspan="1">Fusarium lichenicola(Cylindrocarponlichenicola)</td><td colspan="1" rowspan="1">Fusarium sterilihyphosum</td></tr><tr><td colspan="1" rowspan="1">Fusarium bullatum</td><td colspan="1" rowspan="1">Fusarium louisianense</td><td colspan="1" rowspan="1">Fusarium subglutinans</td></tr><tr><td colspan="1" rowspan="1">Fusarium camptoceras</td><td colspan="1" rowspan="1">Fusarium lunulosporum</td><td colspan="1" rowspan="1">Fusarium temperatum</td></tr><tr><td colspan="1" rowspan="1">Fusarium cerealis</td><td colspan="1" rowspan="1">Fusarium mangiferae</td><td colspan="1" rowspan="1">Fusarium thapsinum</td></tr><tr><td colspan="1" rowspan="1">Fusarium circinatum</td><td colspan="1" rowspan="1">Fusarium meridionale</td><td colspan="1" rowspan="1">Fusarium udum</td></tr><tr><td colspan="1" rowspan="1">Fusarium commune</td><td colspan="1" rowspan="1">Fusarium mesoamericanum</td><td colspan="1" rowspan="1">Fusarium verticillioides *</td></tr><tr><td colspan="1" rowspan="1">Fusarium concentricum</td><td colspan="1" rowspan="1">Fusarium mexicanum</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Fusarium concolor</td><td colspan="1" rowspan="1">Fusarium musae</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="3" rowspan="1">Detection Predicted for 85%-94% of target se que nces</td></tr><tr><td colspan="1" rowspan="1">Fusarium torulosum</td><td colspan="1" rowspan="1">Fusarium xylarioides</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="3" rowspan="1">Detection Predicted for &lt;85% of target se que nces</td></tr><tr><td colspan="1" rowspan="1">Fusarium chlamydosporum(66.7%)</td><td colspan="1" rowspan="1">Fusarium graminearum(59.4%)</td><td colspan="1" rowspan="1">Fusarium longipes (25.0%)</td></tr><tr><td colspan="1" rowspan="1">Fusarium coeruleum (50.0%)</td><td colspan="1" rowspan="1">Fusarium lateritium (50.0%)</td><td colspan="1" rowspan="1">Fusarium nelsonii (16.7%)</td></tr><tr><td colspan="3" rowspan="1">Dete ction Not Predicted</td></tr><tr><td colspan="1" rowspan="1">Fusarium kyushuense</td><td colspan="1" rowspan="1">Fusarium sambucinum</td><td colspan="1" rowspan="1">Fusarium venenatum</td></tr><tr><td colspan="1" rowspan="1">Fusarium miscanthi</td><td colspan="1" rowspan="1">Fusarium stilboides</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Fusarium redolens</td><td colspan="1" rowspan="1">Fusarium succisae</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="3" rowspan="1">Detection Predicted for ≥95% of target se que nces</td></tr><tr><td colspan="1" rowspan="1">Rhodotorula araucariae</td><td colspan="1" rowspan="1">Rhodotorula graminis</td><td colspan="1" rowspan="1">Rhodotorula taiwanensis</td></tr><tr><td colspan="1" rowspan="1">Rhodotorula glutinis *</td><td colspan="1" rowspan="1">Rhodotorula mucilaginosa*</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="3" rowspan="1">Detection Predicted for 85%-94% of target seque nces</td></tr><tr><td colspan="3" rowspan="1">None Identified</td></tr><tr><td colspan="3" rowspan="1">Detection Predicted for &lt;85% of target se que nces</td></tr><tr><td colspan="3" rowspan="1">None Identified</td></tr><tr><td colspan="3" rowspan="1">Detection Not Predicted</td></tr><tr><td colspan="1" rowspan="1">Rhodotorula acheniorum</td><td colspan="1" rowspan="1">Rhodotorula fragariae</td><td colspan="1" rowspan="1">Rhodotorula marina</td></tr><tr><td colspan="1" rowspan="1">Rhodotorula acuta</td><td colspan="1" rowspan="1">Rhodotorula fujisanensis</td><td colspan="1" rowspan="1">Rhodotorula minuta</td></tr><tr><td colspan="1" rowspan="1">Rhodotorula armeniaca</td><td colspan="1" rowspan="1">Rhodotorula hinnulea</td><td colspan="1" rowspan="1">Rhodotorula muscorum</td></tr><tr><td colspan="1" rowspan="1">Rhodotorula aurantiaca</td><td colspan="1" rowspan="1">Rhodotorula hordea</td><td colspan="1" rowspan="1">Rhodotorula nothofagi</td></tr><tr><td colspan="1" rowspan="1">Rhodotorula auriculariae</td><td colspan="1" rowspan="1">Rhodotorula hylophila</td><td colspan="1" rowspan="1">Rhodotorula philyla</td></tr><tr><td colspan="1" rowspan="1">Rhodotorula bacarum</td><td colspan="1" rowspan="1">Rhodotorula ingeniosa</td><td colspan="1" rowspan="1">Rhodotorula phylloplana</td></tr><tr><td colspan="1" rowspan="1">Rhodotorula bogoriensis</td><td colspan="1" rowspan="1">Rhodotorula javanica</td><td colspan="1" rowspan="1">Rhodotorula pilati</td></tr><tr><td colspan="1" rowspan="1">Rhodotorula buffonii</td><td colspan="1" rowspan="1">Rhodotorula lactosa</td><td colspan="1" rowspan="1">Rhodotorula pustula</td></tr><tr><td colspan="1" rowspan="1">Rhodotorula ferulica</td><td colspan="1" rowspan="1">Rhodotorula lignophila</td><td colspan="1" rowspan="1">Rhodotorula sonckii</td></tr></table>

\* The performance of the ePlex BCID-FP Panel has not been established for all of the organisms listed in the tables above. See the Analytical Reactivity (Inclusivity) and Limit of Detection (Analytical Sensitivity) sections for data on organisms for which performance characteristics have been established. Some species were not assessed in silico due to lack of sequence data, though they may appear in the analytical sensitivity or specificity studies.

\* The performance of the ePlex BCID-FP Panel has not been established for all of the organisms listed in the tables above. See the Analytical Reactivity (Inclusivity) and Limit of Detection (Analytical Sensitivity) sections for data on organisms for which performance characteristics have been established. Some species were not assessed in silico due to lack of sequence data, though they may appear in the analytical sensitivity or specificity studies.

# Table 22: Predicted (in silico) Reactivity (Inclusivity) Results for Rhodotorula

# g. Analytical specificity (Exclusivity):

Cross-reactivity of on-panel and off-panel analytes was evaluated with the BCID-FP Panel. On-panel organisms were tested in triplicate at concentrations approximating bottle positivity are noted in Table 23. Offpanel organisms were tested at concentrations of ${ \geq } 1 \mathbf { x } 1 0 ^ { 9 }$ CFU/mL for bacteria and ${ \geq } 1 \mathbf { x } 1 0 ^ { 7 }$ CFU/mL or spores/mL for fungi unless otherwise noted in Table 24. If the target concentration could not be reached, the organism was diluted 2-fold from stock for use.

Table 23: On-Panel Organisms Assessed for Cross-reactivity with the ePlex BCID-FP Panel (Exclusivity)   

<table><tr><td colspan="1" rowspan="1">Organism</td><td colspan="1" rowspan="1">Strain</td><td colspan="1" rowspan="1">Organism</td><td colspan="1" rowspan="1">Strain</td></tr><tr><td colspan="1" rowspan="1">Candida albicans</td><td colspan="1" rowspan="1">ATCC 10231</td><td colspan="1" rowspan="1">Candida krusei</td><td colspan="1" rowspan="1">ATCC 32196</td></tr><tr><td colspan="1" rowspan="1">Candida albicans</td><td colspan="1" rowspan="1">ATCC 90028</td><td colspan="1" rowspan="1">Candida krusei</td><td colspan="1" rowspan="1">ATCC 34135</td></tr><tr><td colspan="1" rowspan="1">Candida albicans</td><td colspan="1" rowspan="1">ATCC MYA-4441</td><td colspan="1" rowspan="1">Candida lusitaniae</td><td colspan="1" rowspan="1">ATCC 42720</td></tr><tr><td colspan="1" rowspan="1">Candida auris</td><td colspan="1" rowspan="1">CBS 12766</td><td colspan="1" rowspan="1">Candida lusitaniae</td><td colspan="1" rowspan="1">ATCC MYA-766</td></tr><tr><td colspan="1" rowspan="1">Candida auris</td><td colspan="1" rowspan="1">CDC#385</td><td colspan="1" rowspan="1">Candida lusitaniae</td><td colspan="1" rowspan="1">Z010</td></tr><tr><td colspan="1" rowspan="1">Candida auris</td><td colspan="1" rowspan="1">CDC#386</td><td colspan="1" rowspan="1">Candida parapsilosis</td><td colspan="1" rowspan="1">ATCC 22019</td></tr><tr><td colspan="1" rowspan="1">Candida auris</td><td colspan="1" rowspan="1">CDC#387</td><td colspan="1" rowspan="1">Candida parapsilosis</td><td colspan="1" rowspan="1">ATCC 58895</td></tr><tr><td colspan="1" rowspan="1">Candida auris</td><td colspan="1" rowspan="1">CDC#388</td><td colspan="1" rowspan="1">Candida parapsilosis</td><td colspan="1" rowspan="1">ATCC 90018</td></tr><tr><td colspan="1" rowspan="1">Candida auris</td><td colspan="1" rowspan="1">CDC#389</td><td colspan="1" rowspan="1">Candida tropicalis</td><td colspan="1" rowspan="1">ATCC 201380</td></tr><tr><td colspan="1" rowspan="1">Candida auris</td><td colspan="1" rowspan="1">CDC#390</td><td colspan="1" rowspan="1">Candida tropicalis</td><td colspan="1" rowspan="1">ATCC 201381</td></tr><tr><td colspan="1" rowspan="1">Candidadubliniensis</td><td colspan="1" rowspan="1">ATCC MYA-578</td><td colspan="1" rowspan="1">Candida tropicalis</td><td colspan="1" rowspan="1">ATCC 750</td></tr><tr><td colspan="1" rowspan="1">Candidadubliniensis</td><td colspan="1" rowspan="1">ATCC MYA-579</td><td colspan="1" rowspan="1">Cryptococcus gattii</td><td colspan="1" rowspan="1">ATCC 14248</td></tr><tr><td colspan="1" rowspan="1">Candidadubliniensis</td><td colspan="1" rowspan="1">ATCC MYA-582</td><td colspan="1" rowspan="1">Cryptococcus gattii</td><td colspan="1" rowspan="1">ATCC 4560</td></tr><tr><td colspan="1" rowspan="1">Candida famata</td><td colspan="1" rowspan="1">ATCC 20850</td><td colspan="1" rowspan="1">Cryptococcus gattii</td><td colspan="1" rowspan="1">ATCC 76108</td></tr><tr><td colspan="1" rowspan="1">Candida famata</td><td colspan="1" rowspan="1">CBA 1961</td><td colspan="1" rowspan="1">Cryptococcus neoformans</td><td colspan="1" rowspan="1">ATCC 14116</td></tr><tr><td colspan="1" rowspan="1">Candida famata</td><td colspan="1" rowspan="1">CBS 789</td><td colspan="1" rowspan="1">Cryptococcus neoformans</td><td colspan="1" rowspan="1">ATCC 90112</td></tr><tr><td colspan="1" rowspan="1">Candida glabrata</td><td colspan="1" rowspan="1">ATCC 15126</td><td colspan="1" rowspan="1">Cryptococcus neoformans</td><td colspan="1" rowspan="1">NCPF 8299</td></tr><tr><td colspan="1" rowspan="1">Candida glabrata</td><td colspan="1" rowspan="1">ATCC 66032</td><td colspan="1" rowspan="1">Bisifusarium dimerum</td><td colspan="1" rowspan="1">CBS 110317</td></tr><tr><td colspan="1" rowspan="1">Candida glabrata</td><td colspan="1" rowspan="1">ATCC MYA-2950</td><td colspan="1" rowspan="1">Fusarium lichenicola(Cylindrocarponlichenicola)</td><td colspan="1" rowspan="1">ATCC 204306</td></tr><tr><td colspan="1" rowspan="1">Candidaguilliermondii</td><td colspan="1" rowspan="1">ATCC 90197</td><td colspan="1" rowspan="1">Fusarium moniliforme</td><td colspan="1" rowspan="1">ATCC 38159</td></tr><tr><td colspan="1" rowspan="1">Candidaguilliermondii</td><td colspan="1" rowspan="1">ATCC 90198</td><td colspan="1" rowspan="1">Fusarium proliferatum</td><td colspan="1" rowspan="1">CBS 131570</td></tr><tr><td colspan="1" rowspan="1">Candidaguilliermondii</td><td colspan="1" rowspan="1">ATCC 90199</td><td colspan="1" rowspan="1">Fusarium sacchari</td><td colspan="1" rowspan="1">CBS 119828</td></tr><tr><td colspan="1" rowspan="1">Candida kefyr</td><td colspan="1" rowspan="1">ATCC 204093</td><td colspan="1" rowspan="1">Fusarium verticillioides</td><td colspan="1" rowspan="1">CBS 100312</td></tr><tr><td colspan="1" rowspan="1">Candida kefyr</td><td colspan="1" rowspan="1">ATCC 2512</td><td colspan="1" rowspan="1">Rhodotorula glutinis</td><td colspan="1" rowspan="1">ATCC 96365</td></tr><tr><td colspan="1" rowspan="1">Candida kefyr</td><td colspan="1" rowspan="1">ATCC 66028</td><td colspan="1" rowspan="1">Rhodotorula mucilaginosa</td><td colspan="1" rowspan="1">ATCC 66034</td></tr><tr><td colspan="1" rowspan="1">Candida krusei</td><td colspan="1" rowspan="1">ATCC 14243</td><td colspan="1" rowspan="1">Rhodotorula mucilaginosa</td><td colspan="1" rowspan="1">ATCC 9449</td></tr></table>

Table 24: Cross-reactivity with Organisms Not Targeted by the ePlex BCID-FP Panel (Exclusivity)   

<table><tr><td colspan="1" rowspan="1">Organism</td><td colspan="1" rowspan="1">Strain</td><td colspan="1" rowspan="1">Organism</td><td colspan="1" rowspan="1">Strain</td></tr><tr><td colspan="1" rowspan="1">Acinetobacter Iwoffii</td><td colspan="1" rowspan="1">ATCC 15309</td><td colspan="1" rowspan="1">Kodamaea ohmeri</td><td colspan="1" rowspan="1">CDC#0396</td></tr><tr><td colspan="1" rowspan="1">Acremonium kiliense</td><td colspan="1" rowspan="1">ATCC 4301</td><td colspan="1" rowspan="1">Lactobacillus rhamnosus</td><td colspan="1" rowspan="1">ATCC 53103</td></tr><tr><td colspan="1" rowspan="1">Aspergillus fumigatus</td><td colspan="1" rowspan="1">ATCC204305A</td><td colspan="1" rowspan="1">Malassezia furfur</td><td colspan="1" rowspan="1">ATCC 12078</td></tr><tr><td colspan="1" rowspan="1">Bacteroides fragilis</td><td colspan="1" rowspan="1">ATCC 25285</td><td colspan="1" rowspan="1">Malassezia furfur</td><td colspan="1" rowspan="1">ATCC 14521</td></tr><tr><td colspan="1" rowspan="1">Bordetella pertussis</td><td colspan="1" rowspan="1">ATCC 9340</td><td colspan="1" rowspan="1">Malassezia furfur</td><td colspan="1" rowspan="1">CBS 7710</td></tr><tr><td colspan="1" rowspan="1">Candida bracarensis</td><td colspan="1" rowspan="1">CBS 10154</td><td colspan="1" rowspan="1">Malassezia globosa</td><td colspan="1" rowspan="1">ATCC MYA-4612</td></tr><tr><td colspan="1" rowspan="1">Candida carpophila</td><td colspan="1" rowspan="1">CBS 5256</td><td colspan="1" rowspan="1">Malassezia restricta</td><td colspan="1" rowspan="1">ATCC MYA-4611</td></tr><tr><td colspan="1" rowspan="1">Candida</td><td colspan="1" rowspan="1">CDC#394</td><td colspan="1" rowspan="1">Malassezia sympodialis</td><td colspan="1" rowspan="1">ATCC 44031</td></tr><tr><td colspan="1" rowspan="1">duobushaemulonii</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Candida haemulonii</td><td colspan="1" rowspan="1">CDC#393</td><td colspan="1" rowspan="1">Meyerozyma caribbica(Candida fermentati)</td><td colspan="1" rowspan="1">ATCC 20296</td></tr><tr><td colspan="1" rowspan="1">Candida inconspicua</td><td colspan="1" rowspan="1">ATCC 16783</td><td colspan="1" rowspan="1">Micrococcus luteus</td><td colspan="1" rowspan="1">ATCC 19212</td></tr><tr><td colspan="1" rowspan="1">Candida lambica</td><td colspan="1" rowspan="1">ATCC 24750</td><td colspan="1" rowspan="1">Morganella morganii</td><td colspan="1" rowspan="1">ATCC 25830</td></tr><tr><td colspan="1" rowspan="1">Candida lipolytica</td><td colspan="1" rowspan="1">ATCC 20177</td><td colspan="1" rowspan="1">Mucor velutinosus</td><td colspan="1" rowspan="1">ATCC MYA-4766</td></tr><tr><td colspan="1" rowspan="1">Candida metapsilosis</td><td colspan="1" rowspan="1">ATCC 96144</td><td colspan="1" rowspan="1">Penicillium marneffei</td><td colspan="1" rowspan="1">ATCC 200050</td></tr><tr><td colspan="1" rowspan="1">Candida nivariensis</td><td colspan="1" rowspan="1">CBS 9984</td><td colspan="1" rowspan="1">Proteus mirabilis</td><td colspan="1" rowspan="1">ATCC 35659</td></tr><tr><td colspan="1" rowspan="1">Candida norvegensis</td><td colspan="1" rowspan="1">ATCC 22977</td><td colspan="1" rowspan="1">Rhodotorula minuta</td><td colspan="1" rowspan="1">ATCC 36236</td></tr><tr><td colspan="1" rowspan="1">Candida orthopsilosis</td><td colspan="1" rowspan="1">ATCC 96139</td><td colspan="1" rowspan="1">Saccharomyces cerevisiae</td><td colspan="1" rowspan="1">ATCC 18824</td></tr><tr><td colspan="1" rowspan="1">Candida pelliculosa</td><td colspan="1" rowspan="1">ATCC 10262</td><td colspan="1" rowspan="1">Salmonella enterica (Typhi)</td><td colspan="1" rowspan="1">ATCC 19430</td></tr><tr><td colspan="1" rowspan="1">Candida rugosa</td><td colspan="1" rowspan="1">CBS 96275</td><td colspan="1" rowspan="1">Scedosporium prolificans</td><td colspan="1" rowspan="1">ATCC 200543</td></tr><tr><td colspan="1" rowspan="1">Candida sake</td><td colspan="1" rowspan="1">ATCC 22021</td><td colspan="1" rowspan="1">Schizosaccharomyces pombe</td><td colspan="1" rowspan="1">LPY 02387</td></tr><tr><td colspan="1" rowspan="1">Candida utilis</td><td colspan="1" rowspan="1">ATCC 9256</td><td colspan="1" rowspan="1">Serratia marcescens</td><td colspan="1" rowspan="1">ATCC 43861</td></tr><tr><td colspan="1" rowspan="1">Citrobacter freundii</td><td colspan="1" rowspan="1">ATCC 6879</td><td colspan="1" rowspan="1">Sporidiobolus salmonicolor</td><td colspan="1" rowspan="1">ATCC 24217</td></tr><tr><td colspan="1" rowspan="1">Clostridiumperfringens</td><td colspan="1" rowspan="1">ATCC 13124</td><td colspan="1" rowspan="1">Sporothrix schenckii</td><td colspan="1" rowspan="1">ATCC 18616</td></tr><tr><td colspan="1" rowspan="1">Corynebacteriumstriatum</td><td colspan="1" rowspan="1">ATCC 7094</td><td colspan="1" rowspan="1">Staphylococcus hominis</td><td colspan="1" rowspan="1">ATCC 27844</td></tr><tr><td colspan="1" rowspan="1">Enterobacteraerogenes</td><td colspan="1" rowspan="1">ATCC 29751</td><td colspan="1" rowspan="1">Staphylococcus intermedius</td><td colspan="1" rowspan="1">ATCC 29663</td></tr><tr><td colspan="1" rowspan="1">Enterobacter cloacae</td><td colspan="1" rowspan="1">ATCC 23373</td><td colspan="1" rowspan="1">Staphylococcussaprophyticus</td><td colspan="1" rowspan="1">ATCC 15305</td></tr><tr><td colspan="1" rowspan="1">Enterococcus faecium</td><td colspan="1" rowspan="1">ATCC 31282</td><td colspan="1" rowspan="1">Streptococcus agalactiae</td><td colspan="1" rowspan="1">ATCC 12401</td></tr><tr><td colspan="1" rowspan="1">Exophiala jeanselmei</td><td colspan="1" rowspan="1">ATCC 12734</td><td colspan="1" rowspan="1">Streptococcus anginosus</td><td colspan="1" rowspan="1">ATCC 9895</td></tr><tr><td colspan="1" rowspan="1">Filobasidium elegans</td><td colspan="1" rowspan="1">CBS 7637</td><td colspan="1" rowspan="1">Streptococcus pyogenes</td><td colspan="1" rowspan="1">ATCC 12384</td></tr><tr><td colspan="1" rowspan="1">Filobasidiumglobisporum</td><td colspan="1" rowspan="1">CBS 7642</td><td colspan="1" rowspan="1">Trichosporon asahii</td><td colspan="1" rowspan="1">ATCC 201110</td></tr><tr><td colspan="1" rowspan="1">Klebsiella oxytoca</td><td colspan="1" rowspan="1">ATCC 43165</td><td colspan="1" rowspan="1">Trichosporon asteroides</td><td colspan="1" rowspan="1">ATCC 90043</td></tr><tr><td colspan="1" rowspan="1">Kluyveromyces lactis</td><td colspan="1" rowspan="1">ATCC 10689</td><td colspan="1" rowspan="1">Trichosporon dermatis</td><td colspan="1" rowspan="1">ATCC 204094</td></tr></table>

A. T ested at $1 \mathrm { ~ x ~ } 1 0 ^ { 6 }$ spores/mL

No cross-reactivity was observed between any of the off-panel bacteria and fungi with the ePlex BCID-FP Panel targets.

h. Assay cut-off:

Analytical studies were conducted to establish the signal boundaries for all targets and controls of the ePlex BCID-FP Panel.

A mixture of clinical samples, contrived bottle positives, and organisms spiked at or above the analytically determined limit of detection were used as samples for the study. Positive data points from samples at or greater than the determined LoD test concentration in the Limit of Detection study were used to supplement the Cutoff study data. Negative data points for all targets were obtained from the Limit of Blank study. Expected negative data points from replicates that were positive for other targets in the LoD and Cutoff studies were also combined to increase statistical power of the ROC analysis.

For each target, the signals for positive and negative tests were analyzed. An appropriate boundary was established wherein specificity and sensitivity were maximized. The analysis was verified by ROC analysis. The final boundary set points are listed in Table 25 below.

Table 25: Summary of BCID-FP Panel Boundary Set   

<table><tr><td rowspan=1 colspan=1>Target Name</td><td rowspan=1 colspan=1>Boundary (nA)</td></tr><tr><td rowspan=1 colspan=1>Candida albicans</td><td rowspan=1 colspan=1>10</td></tr><tr><td rowspan=1 colspan=1>Candida auris</td><td rowspan=1 colspan=1>10</td></tr><tr><td rowspan=1 colspan=1>Candida dubliniensis</td><td rowspan=1 colspan=1>25</td></tr><tr><td rowspan=1 colspan=1>Candida famata</td><td rowspan=1 colspan=1>10</td></tr><tr><td rowspan=1 colspan=1>Candida glabrata</td><td rowspan=1 colspan=1>15</td></tr><tr><td rowspan=1 colspan=1>Candida guilliermondii</td><td rowspan=1 colspan=1>25</td></tr><tr><td rowspan=1 colspan=1>Candida kefyr</td><td rowspan=1 colspan=1>25</td></tr><tr><td rowspan=1 colspan=1>Candida krusei</td><td rowspan=1 colspan=1>25</td></tr><tr><td rowspan=1 colspan=1>Candida lusitaniae</td><td rowspan=1 colspan=1>20</td></tr><tr><td rowspan=1 colspan=1>Candida parapsilosis</td><td rowspan=1 colspan=1>25</td></tr><tr><td rowspan=1 colspan=1>Candida tropicalis</td><td rowspan=1 colspan=1>25</td></tr><tr><td rowspan=1 colspan=1>Cryptococcus gattii</td><td rowspan=1 colspan=1>25</td></tr><tr><td rowspan=1 colspan=1>Cryptococcus neoformans var. grubii</td><td rowspan=1 colspan=1>25</td></tr><tr><td rowspan=1 colspan=1>Cryptococcus neoformans var. neoformans</td><td rowspan=1 colspan=1>25</td></tr><tr><td rowspan=1 colspan=1>Fusarium</td><td rowspan=1 colspan=1>25</td></tr><tr><td rowspan=1 colspan=1>Rhodotorula</td><td rowspan=1 colspan=1>25</td></tr><tr><td rowspan=1 colspan=1>Schizosaccharomyces pombe (IC1-4)</td><td rowspan=1 colspan=1>50</td></tr></table>

# i. Interference:

A study was performed to investigate the effect of 18 potentially interfering substances on the BCID-FP Panel. Two organism mixes consisting of 5 onpanel organisms and negative blood matrix were used to assess potentially interfering substances for interference. Potentially interfering test substances were spiked at levels predicted to be above the concentration of the substance likely to be found in a blood culture specimen. The positivity of the organisms for each potentially interfering substance at the initial testing concentration is summarized in Table 26 and 27.

Table 26: Interfering Substance and Bottle Equivalency Concentrations   

<table><tr><td>Organism</td><td>Concentration</td></tr><tr><td>Candida albicans</td><td>1 x 106 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Candida kefyr</td><td colspan="1" rowspan="1">1 x 106 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Cryptococcus neoformans</td><td colspan="1" rowspan="1">1 x 107 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Fusarium sacchari</td><td colspan="1" rowspan="1">6.5 x 106spores/mL</td></tr><tr><td colspan="1" rowspan="1">Rhodotorula mucilaginosa</td><td colspan="1" rowspan="1">1 x 106 CFU/mL</td></tr></table>

Table 27: List of Potentially Interfering Substances   

<table><tr><td rowspan=1 colspan=1>Endogenous Substances</td><td rowspan=1 colspan=1>Testing Concentration</td></tr><tr><td rowspan=1 colspan=1>Bilirubin</td><td rowspan=1 colspan=1>60 μg/mL</td></tr><tr><td rowspan=1 colspan=1>Hemoglobin</td><td rowspan=1 colspan=1>0.6 g/L</td></tr><tr><td rowspan=1 colspan=1>Human genomic DNA</td><td rowspan=1 colspan=1>6 x 105 copies/mL</td></tr><tr><td rowspan=1 colspan=1>Triglycerides</td><td rowspan=1 colspan=1>1000 mg/dL</td></tr><tr><td rowspan=1 colspan=1>γ-globulin</td><td rowspan=1 colspan=1>0.85 g/dL</td></tr><tr><td rowspan=1 colspan=1>Exoge nous Substances</td><td rowspan=1 colspan=1>Testing Concentration</td></tr><tr><td rowspan=1 colspan=1>Amoxicillin/Clavulanate</td><td rowspan=1 colspan=1>3.5 ug/mL</td></tr><tr><td rowspan=1 colspan=1>Amphotericin B</td><td rowspan=1 colspan=1>2 μg/mL^</td></tr><tr><td rowspan=1 colspan=1>Caspofungin</td><td rowspan=1 colspan=1>5 μg/mL</td></tr><tr><td rowspan=1 colspan=1>Ceftriaxone</td><td rowspan=1 colspan=1>0.23 mg/mL</td></tr><tr><td rowspan=1 colspan=1>Ciprofloxacin</td><td rowspan=1 colspan=1>3 mg/L</td></tr><tr><td rowspan=1 colspan=1>Fluconazole</td><td rowspan=1 colspan=1>25 mg/L</td></tr><tr><td rowspan=1 colspan=1>Flucytosine</td><td rowspan=1 colspan=1>90 ug/mL</td></tr><tr><td rowspan=1 colspan=1>Gentamicin sulfate</td><td rowspan=1 colspan=1>3 μg/mL</td></tr><tr><td rowspan=1 colspan=1>Heparin</td><td rowspan=1 colspan=1>0.9 U/mL</td></tr><tr><td rowspan=1 colspan=1>Imipenem</td><td rowspan=1 colspan=1>83 μg/mL</td></tr><tr><td rowspan=1 colspan=1>Sodium Polyanetholsulfonate</td><td rowspan=1 colspan=1>0.25% w/v</td></tr><tr><td rowspan=1 colspan=1>Tetracycline</td><td rowspan=1 colspan=1>5 mg/L</td></tr><tr><td rowspan=1 colspan=1>Vancomycin</td><td rowspan=1 colspan=1>30 mg/L</td></tr></table>

No false positives were detected in negative blood matrix runs without or with interfering substances.

# j. Mixed Culture Study (Microbial interference):

Competitive inhibition was evaluated for the ePlex BCID-FP Panel by pairing twelve clinically relevant organisms (including 9 off-panel organisms) in thirteen simulated dual infection sample mixes. Candida albicans was tested at low titer (concentration expected at bottle positivity) while in the presence of other organisms at higher titer (concentrations expected at 8 hours beyond bottle positivity, or one log higher than that expected at bottle positivity). Candida glabrata and Candida parapsilosis were also tested at the concentration expected at bottle positivity in the presence of Candida albicans at a higher titer concentration. No competitive inhibition was observed in any of the sample mixes evaluated at the concentrations listed in Table 28.

Table 28: Competitive Inhibition Organisms and Concentrations Tested   

<table><tr><td rowspan=1 colspan=1>On-panel Organis ms</td><td rowspan=1 colspan=1>High Conce ntration</td><td rowspan=1 colspan=1>Low Conce ntration</td></tr><tr><td rowspan=1 colspan=1>Candida albicans</td><td rowspan=1 colspan=1>1 x 107 CFU/mL</td><td rowspan=1 colspan=1>1 x 106 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Candida glabrata</td><td rowspan=1 colspan=1>1 x 107 CFU/mL</td><td rowspan=1 colspan=1>1 x 106 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Candida parapsilosis</td><td rowspan=1 colspan=1>1 x 107 CFU/mL</td><td rowspan=1 colspan=1>1 x 106 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Off-panel Organis ms</td><td rowspan=1 colspan=2>High Conce ntration</td></tr><tr><td rowspan=1 colspan=1>Acinetobacter baumannii</td><td rowspan=1 colspan=2>1 x 109 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Cutibacterium acnes</td><td rowspan=1 colspan=2>1 x109CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Enterococcus faecalis</td><td rowspan=1 colspan=2>1 x108CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Escherichia coli</td><td rowspan=1 colspan=2>1x109CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Klebsiella pneumoniae</td><td rowspan=1 colspan=2>1x109CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus aureus</td><td rowspan=1 colspan=2>1 x108CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus epidermidis</td><td rowspan=1 colspan=2>1 x 108CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Streptococcus pneumoniae</td><td rowspan=1 colspan=2>4 x 108 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Pseudomonas aeruginosa</td><td rowspan=1 colspan=2>4 x 108 CFU/mL</td></tr></table>

Study results demonstrated that high concentrations of the on-panel BCID-FP microorganisms spiked into blood culture samples produced positive results for the relevant assays on the BCID-FP Panel but did interfere with any expected results for other analytes. High concentrations of off-panel BCID-FP microorganisms also showed no interference with the detection of any ePlex BCID-FP organism with no unexpected false negative or false positive results observed.

# k. Testing of Additional Blood Culture Bottle Types

Fifteen different blood culture bottle types from three different blood culture systems (BacT/Alert, BACTEC and VersaTREK) were evaluated analytically with the BCIDFP Panel. Blood culture bottles/media were tested with the recommended ratio of blood to media. Testing was performed using two organism mixes whose members were present at a concentration approximating blood culture bottle positivity in a clinical sample. The mixes contain representative organisms from four genera, including the most prevalent Candida organism. Each organism was present at a testing concentration approximating bottle positivity. The 15 bottle types listed in the table below were evaluated and study results demonstrated correct positive and negative BCID-FP Panel results with each bottle type.

Table 29: Sample Matrix Equivalency (Bottle Evaluation) Bottle Types   

<table><tr><td colspan="1" rowspan="1">Manufacturer</td><td colspan="1" rowspan="1">Bottle Brand</td><td colspan="1" rowspan="1">Bottle Type</td><td colspan="1" rowspan="1">Study Outcome</td></tr><tr><td colspan="1" rowspan="1">BD</td><td colspan="1" rowspan="1">BACTEC</td><td colspan="1" rowspan="1">Plus Aerobic</td><td colspan="1" rowspan="1">No interference observed</td></tr><tr><td colspan="1" rowspan="1">BD</td><td colspan="1" rowspan="1">BACTEC</td><td colspan="1" rowspan="1">Plus Anaerobic</td><td colspan="1" rowspan="1">No interference observed</td></tr><tr><td colspan="1" rowspan="1">BD</td><td colspan="1" rowspan="1">BACTEC</td><td colspan="1" rowspan="1">Standard Aerobic</td><td colspan="1" rowspan="1">No interference observed</td></tr><tr><td colspan="1" rowspan="1">BD</td><td colspan="1" rowspan="1">BACTEC</td><td colspan="1" rowspan="1">StandardAnaerobic</td><td colspan="1" rowspan="1">No interference observed</td></tr><tr><td colspan="1" rowspan="1">BD</td><td colspan="1" rowspan="1">BACTEC</td><td colspan="1" rowspan="1">Peds Plus</td><td colspan="1" rowspan="1">No interference observed</td></tr><tr><td colspan="1" rowspan="1">BD</td><td colspan="1" rowspan="1">BACTEC</td><td colspan="1" rowspan="1">Lytic Anaerobic</td><td colspan="1" rowspan="1">No interference observed</td></tr><tr><td colspan="1" rowspan="1">BD</td><td colspan="1" rowspan="1">BACTEC</td><td colspan="1" rowspan="1">Myco</td><td colspan="1" rowspan="1">No interference observed</td></tr><tr><td colspan="1" rowspan="1">bioMérieux</td><td colspan="1" rowspan="1">BACT/ALERT</td><td colspan="1" rowspan="1">SA StandardAerobic</td><td colspan="1" rowspan="1">No interference observed</td></tr><tr><td colspan="1" rowspan="1">bioMérieux</td><td colspan="1" rowspan="1">BACT/ALERT</td><td colspan="1" rowspan="1">SN StandardAnaerobic</td><td colspan="1" rowspan="1">No interference observed</td></tr><tr><td colspan="1" rowspan="1">bioMérieux</td><td colspan="1" rowspan="1">BACT/ALERT</td><td colspan="1" rowspan="1">FA Plus</td><td colspan="1" rowspan="1">A false negative result wasobserved for the Candidaalbicans target in one lot.</td></tr><tr><td colspan="1" rowspan="1">bioMérieux</td><td colspan="1" rowspan="1">BACT/ALERT</td><td colspan="1" rowspan="1">FN Plus</td><td colspan="1" rowspan="1">No interference observed</td></tr><tr><td colspan="1" rowspan="1">bioMérieux</td><td colspan="1" rowspan="1">BACT/ALERT</td><td colspan="1" rowspan="1">PF Plus</td><td colspan="1" rowspan="1">False negative results wereobserved for the Rhodotorulatarget in one lot.</td></tr><tr><td colspan="1" rowspan="1">bioMérieux</td><td colspan="1" rowspan="1">BACT/ALERT</td><td colspan="1" rowspan="1">MP Mycobacteria</td><td colspan="1" rowspan="1">No interference observed</td></tr><tr><td colspan="1" rowspan="1">ThermoScientific</td><td colspan="1" rowspan="1">VersaTREK</td><td colspan="1" rowspan="1">REDO 1 EZDraw Aerobic</td><td colspan="1" rowspan="1">No interference observed</td></tr><tr><td colspan="1" rowspan="1">ThermoScientific</td><td colspan="1" rowspan="1">VersaTREK</td><td colspan="1" rowspan="1">REDOX 2 EZDraw Anaerobic</td><td colspan="1" rowspan="1">No interference observed</td></tr></table>

Thirteen bottle types tested showed no interference for any of the targets tested. The BACT/Alert $\textsuperscript { \textregistered }$ PF Plus and BACT/Aler $\textsuperscript { \textregistered }$ FA Plus bottle types each showed lower sensitivity for one organism tested in one of the two bottle lots and may have reduced sensitivity for some targets. The following discordance was observed in the study:

BACT/Alert PF Plus

Both lots of BacT/ Alert PF Plus failed to detect Rhodotorula in one of five initial replicates. Upon retest, one of ten replicates resulted in a Rhodotorula false negative for lot 3048740. The second lot of the same bottle type (lot 3047800), detected all targets in all 10 repeat test replicates. Given the lower detection rate of Rhodotorula in one of two bottle lots tested, the sensitivity for some targets may potentially be reduced in BacT/ Alert PF Plus bottles.

# BacT/ Alert FA Plus

The BacT/ Alert FA Plus lot 3048018 had one run with a Candida parapsilosis false positive in the initial five test replicates with mix 2. Upon retest, no false positives were observed but one Candida albicans false negative was observed. Given the lower detection rate of Candida albicans in one of two bottle lots tested, the sensitivity for some targets may potentially be reduced in BacT/ Alert FA Plus bottles.

The following statements are included in the package insert:

<table><tr><td rowspan=1 colspan=1>Section ofthe Package Insert</td><td rowspan=1 colspan=1>State me nt</td></tr><tr><td rowspan=1 colspan=1>Interfering Substances andSample Matrix Equivalency(Bottle Evaluation)</td><td rowspan=1 colspan=1>Thirteen bottle types tested showed no interference for any of thetargets tested. The BACT/Alert PF Plus and BACT/Alert FA Plusbottle types each showed lower sensitivity for one organism testedin one of the two bottle lots and may have reduced sensitivity forsome targets.</td></tr><tr><td rowspan=1 colspan=1>Limitations of the Procedure</td><td rowspan=1 colspan=1>Decreased sensitivity has been observed for some targets inBACT/Alert PF Plus and BACT/Alert FA Plus bottle types.</td></tr></table>

# l. Carryover Study:

The carryover/cross-contamination rate of the ePlex BCID-FP Panel and ePlex instrument was evaluated using a checkerboard approach by running high positive and negative samples interspersed in all bays of a four-tower ePlex instrument (i.e., 24 bays total) over five separate runs on five separate days. The positive sample was a contrived blood culture sample containing Fusarium sacchari. This organism was contrived and grown to positivity in blood culture, then incubated for an additional 8 hours in a continuously monitoring blood culture instrument. The contrived $F .$ . sacchari sample was then spiked with Candida albicans at $\mathrm { 1 x 1 0 ^ { 7 } C F U / m L }$ . The organism levels were higher than typically found in positive blood cultures in order to challenge the system for potential cross contamination. No false positives were detected in the negative runs indicating no carryover or cross-contamination was observed between bays or within bays with the ePlex BCID-FP Panel when testing samples consecutively or in adjacent bays with an ePlex instrument. Two false negatives were detected in positive samples, which were repeated and produced expected results.

2. Comparison studies:

a. Method comparison with predicate device: Not applicable. Refer to the Clinical Studies Section of this document.

b. Matrix comparison: N/A

3. Clinical studies:

# Prospective Clinical Study

The clinical performance of the ePlex BCID-FP Panel was established during multi-center clinical studies conducted at six distinct U.S. test sites, with 21 evaluable samples. Each test site was representative of the intended use setting (clinical laboratories) and testing was performed by trained clinical laboratory personnel. Demographic information for prospectively collected samples is described in Table 30.

To supplement the results of the prospective collection, 120 samples were collected retrospectively from a total of 9 sites, and 725 evaluable samples were contrived for organisms with low prevalence. Demographic information for retrospectively collected samples is described in Table 31.

Table 30: Demographic Data for Clinical Samples by Collection Site (Prospective Collection)   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>All Sites(N=21)n (%)</td><td rowspan=1 colspan=1>Site 1(N=1)n (%)</td><td rowspan=1 colspan=1>Site 2(N=8)n (%)</td><td rowspan=1 colspan=1>Site 3(N=2)n (%)</td><td rowspan=1 colspan=1>Site 4(N=4n (%))</td><td rowspan=1 colspan=1>Site 5(N=4)n (%)</td><td rowspan=1 colspan=1>Site 6(N=2)n (%)</td></tr><tr><td rowspan=1 colspan=8>Sex, n (%)</td></tr><tr><td rowspan=1 colspan=1>Male</td><td rowspan=1 colspan=1>14(66.7)</td><td rowspan=1 colspan=1>1(100)</td><td rowspan=1 colspan=1>7(87.5)</td><td rowspan=1 colspan=1>1(50.0)</td><td rowspan=1 colspan=1>3(75.0)</td><td rowspan=1 colspan=1>1(25.0)</td><td rowspan=1 colspan=1>1(50.0)</td></tr><tr><td rowspan=1 colspan=1>Female</td><td rowspan=1 colspan=1>7(33.3)</td><td rowspan=1 colspan=1>0(0.0)</td><td rowspan=1 colspan=1>1(12.5)</td><td rowspan=1 colspan=1>1(50.0)</td><td rowspan=1 colspan=1>1(25.0)</td><td rowspan=1 colspan=1>3(75.0)</td><td rowspan=1 colspan=1>1(50.0)</td></tr><tr><td rowspan=1 colspan=1>Age (years)</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr><tr><td rowspan=1 colspan=1>&lt;1 yr</td><td rowspan=1 colspan=1>1(4.8)</td><td rowspan=1 colspan=1>0(0.0)</td><td rowspan=1 colspan=1>0(0.0)</td><td rowspan=1 colspan=1>0(0.0)</td><td rowspan=1 colspan=1>1(25.0)</td><td rowspan=1 colspan=1>0(0.0)</td><td rowspan=1 colspan=1>0(0.0)</td></tr><tr><td rowspan=1 colspan=1>1-17 yrs</td><td rowspan=1 colspan=1>2(9.5)</td><td rowspan=1 colspan=1>0(0.0)</td><td rowspan=1 colspan=1>0(0.0)</td><td rowspan=1 colspan=1>0(0.0)</td><td rowspan=1 colspan=1>0(0.0)</td><td rowspan=1 colspan=1>2(50.0)</td><td rowspan=1 colspan=1>0(0.0)</td></tr><tr><td rowspan=1 colspan=1>18-44 yrs</td><td rowspan=1 colspan=1>4(19.0)</td><td rowspan=1 colspan=1>0(0.0)</td><td rowspan=1 colspan=1>2(25.0)</td><td rowspan=1 colspan=1>0(0.0)</td><td rowspan=1 colspan=1>1(25.0)</td><td rowspan=1 colspan=1>1(25.0)</td><td rowspan=1 colspan=1>0(0.0)</td></tr><tr><td rowspan=1 colspan=1>45-64 yrs</td><td rowspan=1 colspan=1>11(52.4)</td><td rowspan=1 colspan=1>1(100)</td><td rowspan=1 colspan=1>4(50.0)</td><td rowspan=1 colspan=1>1(50.0)</td><td rowspan=1 colspan=1>2(50.0)</td><td rowspan=1 colspan=1>1(25.0)</td><td rowspan=1 colspan=1>2(100)</td></tr><tr><td rowspan=1 colspan=1>65-84 yrs</td><td rowspan=1 colspan=1>2(9.5)</td><td rowspan=1 colspan=1>0(0.0)</td><td rowspan=1 colspan=1>1(12.5)</td><td rowspan=1 colspan=1>1(50.0)</td><td rowspan=1 colspan=1>0(0.0)</td><td rowspan=1 colspan=1>0(0.0)</td><td rowspan=1 colspan=1>0(0.0)</td></tr><tr><td rowspan=1 colspan=1>85+ yrs</td><td rowspan=1 colspan=1>1(4.8)</td><td rowspan=1 colspan=1>0(0.0)</td><td rowspan=1 colspan=1>1(12.5)</td><td rowspan=1 colspan=1>0(0.0)</td><td rowspan=1 colspan=1>0(0.0)</td><td rowspan=1 colspan=1>0(0.0)</td><td rowspan=1 colspan=1>0(0.0)</td></tr></table>

Table 31: Demographic Summary for Retrospective Arm of ePlex BCID-FP Clinical Evaluation   

<table><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">All Sites(N=120)n (%)</td><td colspan="1" rowspan="1">Site 1(N=13)n (%)</td><td colspan="1" rowspan="1">Site 2(N=14)n (%)</td><td colspan="1" rowspan="1">Site 3(N=17)n (%)</td><td colspan="1" rowspan="1">Site 4(N=4)n (%)</td><td colspan="1" rowspan="1">Site 5(N=3)n(%)</td><td colspan="1" rowspan="1">Site 6(N=13)n (%)</td><td colspan="1" rowspan="1">Site 7(N=16)n (%)</td><td colspan="1" rowspan="1">Site 8(N=5)n (%)</td><td colspan="1" rowspan="1">Site 9(N=35)n (%)</td></tr><tr><td colspan="11" rowspan="1">Sex, n (%)</td></tr><tr><td colspan="1" rowspan="1">Male</td><td colspan="1" rowspan="1">68(56.7)</td><td colspan="1" rowspan="1">10(76.9)</td><td colspan="1" rowspan="1">8(57.1)</td><td colspan="1" rowspan="1">8(47.1)</td><td colspan="1" rowspan="1">1(25.0)</td><td colspan="1" rowspan="1">2(66.7)</td><td colspan="1" rowspan="1">8(61.5)</td><td colspan="1" rowspan="1">9(56.3)</td><td colspan="1" rowspan="1">3(60.0)</td><td colspan="1" rowspan="1">19(54.3)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">All Sites(N=120)n (%)</td><td colspan="1" rowspan="1">Site 1(N=13)n (%)</td><td colspan="1" rowspan="1">Site 2(N=14)n (%)</td><td colspan="1" rowspan="1">Site 3(N=17)n (%)</td><td colspan="1" rowspan="1">Site 4(N=4)n(%)</td><td colspan="1" rowspan="1">Site 5(N=3)n(%)</td><td colspan="1" rowspan="1">Site 6(N=13)n (%)</td><td colspan="1" rowspan="1">Site 7(N=16)n (%)</td><td colspan="1" rowspan="1">Site 8(N=5)n(%)</td><td colspan="1" rowspan="1">Site 9(N=35)n (%)</td></tr><tr><td colspan="1" rowspan="1">Female</td><td colspan="1" rowspan="1">52(43.3)</td><td colspan="1" rowspan="1">3(23.1)</td><td colspan="1" rowspan="1">6(42.9)</td><td colspan="1" rowspan="1">9(52.9)</td><td colspan="1" rowspan="1">3(75.0)</td><td colspan="1" rowspan="1">1(33.3)</td><td colspan="1" rowspan="1">5(38.5)</td><td colspan="1" rowspan="1">7(43.8)</td><td colspan="1" rowspan="1">2(40.0)</td><td colspan="1" rowspan="1">16(45.7)</td></tr><tr><td colspan="11" rowspan="1">Age (years)</td></tr><tr><td colspan="1" rowspan="1">&lt;1 yr</td><td colspan="1" rowspan="1">2(1.7)</td><td colspan="1" rowspan="1">0(0.0)</td><td colspan="1" rowspan="1">0(0.0)</td><td colspan="1" rowspan="1">0(0.0)</td><td colspan="1" rowspan="1">0(0.0)</td><td colspan="1" rowspan="1">0(0.0)</td><td colspan="1" rowspan="1">0(0.0)</td><td colspan="1" rowspan="1">0(0.0)</td><td colspan="1" rowspan="1">0(0.0)</td><td colspan="1" rowspan="1">2(5.7)</td></tr><tr><td colspan="1" rowspan="1">1-17 yrs</td><td colspan="1" rowspan="1">8(6.7)</td><td colspan="1" rowspan="1">1(7.7)</td><td colspan="1" rowspan="1">0(0.0)</td><td colspan="1" rowspan="1">0(0.0)</td><td colspan="1" rowspan="1">0(0.0)</td><td colspan="1" rowspan="1">0(0.0)</td><td colspan="1" rowspan="1">5(38.5)</td><td colspan="1" rowspan="1">0(0.0)</td><td colspan="1" rowspan="1">0(0.0)</td><td colspan="1" rowspan="1">2(5.7)</td></tr><tr><td colspan="1" rowspan="1">18-44yrs</td><td colspan="1" rowspan="1">27(22.5)</td><td colspan="1" rowspan="1">4(30.8)</td><td colspan="1" rowspan="1">2(14.3)</td><td colspan="1" rowspan="1">2(11.8)</td><td colspan="1" rowspan="1">1(25.0)</td><td colspan="1" rowspan="1">0(0.0)</td><td colspan="1" rowspan="1">3(23.1)</td><td colspan="1" rowspan="1">3(18.8)</td><td colspan="1" rowspan="1">1(20.0)</td><td colspan="1" rowspan="1">11(31.4)</td></tr><tr><td colspan="1" rowspan="1">45-64yrs</td><td colspan="1" rowspan="1">39(32.5)</td><td colspan="1" rowspan="1">2(15.4)</td><td colspan="1" rowspan="1">6(42.9)</td><td colspan="1" rowspan="1">6(35.3)</td><td colspan="1" rowspan="1">1(25.0)</td><td colspan="1" rowspan="1">2(66.7)</td><td colspan="1" rowspan="1">2(15.4)</td><td colspan="1" rowspan="1">7(43.8)</td><td colspan="1" rowspan="1">1(20.0)</td><td colspan="1" rowspan="1">12(34.3)</td></tr><tr><td colspan="1" rowspan="1">65-84yrs</td><td colspan="1" rowspan="1">39(32.5)</td><td colspan="1" rowspan="1">6(46.2)</td><td colspan="1" rowspan="1">6(42.9)</td><td colspan="1" rowspan="1">8(47.1)</td><td colspan="1" rowspan="1">2(50.0)</td><td colspan="1" rowspan="1">0(0.0)</td><td colspan="1" rowspan="1">2(15.4)</td><td colspan="1" rowspan="1">5(31.3)</td><td colspan="1" rowspan="1">2(40.0)</td><td colspan="1" rowspan="1">8(22.9)</td></tr><tr><td colspan="1" rowspan="1">85+ yrs</td><td colspan="1" rowspan="1">5(4.2)</td><td colspan="1" rowspan="1">0(0.0)</td><td colspan="1" rowspan="1">0(0.0)</td><td colspan="1" rowspan="1">1(5.9)</td><td colspan="1" rowspan="1">0(0.0)</td><td colspan="1" rowspan="1">1(33.3)</td><td colspan="1" rowspan="1">1(7.7)</td><td colspan="1" rowspan="1">1(6.3)</td><td colspan="1" rowspan="1">1(20.0)</td><td colspan="1" rowspan="1">0(0.0)</td></tr></table>

The performance of the ePlex BCID-FP Panel was compared to standard laboratory procedures, including traditional and automated culture, MALDI-TOF IVD, and microbiological and biochemical techniques. In addition, all prospective samples were tested with analytically validated PCR assays followed by bidirectional sequencing to determine the presence or absence of Candida auris, Fusarium, and Rhodotorula. Identification for samples with Candida parapsilosis identified by standard laboratory procedures was confirmed using analytically validated PCR assays followed by bi-directional sequencing.

The comparator method(s) results were used to determine the Detected / Not Detected status for each target organism on the ePlex BCID-FP Panel. The comparator methods for each target are summarized in Table 32.

Table 32: Reference/Comparator Methods used to Assess BCID-FP Performance   

<table><tr><td>BCID-FP Panel Target</td><td>Comparator Method(s)</td></tr><tr><td>Candida albicans, Candida dubliniensis, Candida famata, Candida glabrata, Candida guilliermondii, Candida kefyr, Candida krusei, Candida lusitaniae, Candida tropicalis,</td><td>Standard laboratory procedures for organism identification</td></tr><tr><td colspan="1" rowspan="1">Cryptococcusneoformans</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Candida parapsilosis</td><td colspan="1" rowspan="1">Standard laboratory procedures for organism identification.PCR/sequencing to confirmC. parapsilosis or identify C. metapsilosis,C. orthopsilosis</td></tr><tr><td colspan="1" rowspan="1">Candida auris,Fusarium, Rhodotorula</td><td colspan="1" rowspan="1">Standard laboratory procedures for organism identification.PCR/sequencing in prospective samples with yeast and/orfungal Gram stain results</td></tr></table>

Clinical sensitivity or positive percent agreement (PPA) was calculated by dividing the number of true positive (TP) results by the sum of TP and false negative (FN) results, while specificity or negative percent agreement (NPA) was calculated by dividing the number of true negative (TN) results by the sum of TN and false positive (FP) results. A TP result was defined as a sample where the detected ePlex BCID-FP Panel result matched the detected comparator method result, while a TN result was one where a negative ePlex BCID-FP Panel result matched a negative comparator method result. The two-sided $9 5 \%$ confidence interval was also calculated.

A total of 866 positive blood culture samples with a Gram stain result indicating fungal organism consisting of 11 fresh prospective, 10 frozen prospective, 120 retrospective, and 725 contrived samples were evaluated for the ePlex BCID-FP Panel targets. Contrived samples were prepared by spiking an isolate into a blood culture bottle and growing until flagged positive by a continuously monitoring blood culture system. Samples were removed from the system within 8 hours of positivity and stored frozen until the time of testing. PPA and NPA results are summarized by target in Tables 33-48 below and the strains used to contrive samples are summarized in Table 49.

Table 33: Clinical Performance for Candida albicans   
Table 34: Clinical Performance for Candida auris   

<table><tr><td rowspan=2 colspan=1>Target</td><td rowspan=2 colspan=1>Sample Type</td><td rowspan=1 colspan=2>Sensitivity/PPA</td><td rowspan=1 colspan=2>Specificity/NPA</td></tr><tr><td rowspan=1 colspan=1>TP/TP+FN</td><td rowspan=1 colspan=1>% (95% CI)</td><td rowspan=1 colspan=1>TN/TN+FP</td><td rowspan=1 colspan=1>% (95% CI)</td></tr><tr><td rowspan=7 colspan=1>Candidaalbicans</td><td rowspan=1 colspan=1>Prospective (Fresh)</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>100 (34.2-100)</td><td rowspan=1 colspan=1>9/9</td><td rowspan=1 colspan=1>100 (70.1-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective (Frozen)</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>100 (34.2-100)</td><td rowspan=1 colspan=1>8/8</td><td rowspan=1 colspan=1>100 (67.6-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective (All)</td><td rowspan=1 colspan=1>4/4</td><td rowspan=1 colspan=1>100 (51.0-100)</td><td rowspan=1 colspan=1>17/17</td><td rowspan=1 colspan=1>100 (81.6-100)</td></tr><tr><td rowspan=1 colspan=1>Retrospective</td><td rowspan=1 colspan=1>49/50</td><td rowspan=1 colspan=1>98.0 (89.5-99.6)</td><td rowspan=1 colspan=1>70/70</td><td rowspan=1 colspan=1>100 (94.8-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective/Retrospective</td><td rowspan=1 colspan=1>53/54</td><td rowspan=1 colspan=1>98.1 (90.2-99.7)</td><td rowspan=1 colspan=1>87/87</td><td rowspan=1 colspan=1>100 (95.8-100)</td></tr><tr><td rowspan=1 colspan=1>Contrived</td><td rowspan=1 colspan=1>13/14</td><td rowspan=1 colspan=1>92.9 (68.5-98.7)</td><td rowspan=1 colspan=1>710/711</td><td rowspan=1 colspan=1>99.9 (99.2-100)</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>66/68</td><td rowspan=1 colspan=1>97.1 (89.9-99.2)</td><td rowspan=1 colspan=1>797/798</td><td rowspan=1 colspan=1>99.9 (99.3-100)</td></tr></table>

CI= Confidence Interval

<table><tr><td rowspan=2 colspan=1>Target</td><td rowspan=2 colspan=1>Sample Type</td><td rowspan=1 colspan=2>Sensitivity/PPA</td><td rowspan=1 colspan=2>Specificity/NPA</td></tr><tr><td rowspan=1 colspan=1>TP/TP+FN</td><td rowspan=1 colspan=1>% (95% CI)</td><td rowspan=1 colspan=1>TN/TN+FP</td><td rowspan=1 colspan=1>% (95% CI)</td></tr><tr><td rowspan=7 colspan=1>Candidaauris</td><td rowspan=1 colspan=1>Prospective (Fresh)</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>---</td><td rowspan=1 colspan=1>11/11</td><td rowspan=1 colspan=1>100 (74.1-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective (Frozen)</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>---</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>100 (72.2-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective (All)</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>---</td><td rowspan=1 colspan=1>21/21</td><td rowspan=1 colspan=1>100 (84.5-100)</td></tr><tr><td rowspan=1 colspan=1>Retrospective</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>---</td><td rowspan=1 colspan=1>120/120</td><td rowspan=1 colspan=1>100 (96.9-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective/Retrospective</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>---</td><td rowspan=1 colspan=1>141/141</td><td rowspan=1 colspan=1>100 (97.3-100)</td></tr><tr><td rowspan=1 colspan=1>Contrived</td><td rowspan=1 colspan=1>49/49</td><td rowspan=1 colspan=1>100 (92.7-100)</td><td rowspan=1 colspan=1>676/676</td><td rowspan=1 colspan=1>100 (99.4-100)</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>49/49</td><td rowspan=1 colspan=1>100 (92.7-100)</td><td rowspan=1 colspan=1>817/817</td><td rowspan=1 colspan=1>100 (99.5-100)</td></tr></table>

Table 35: Clinical Performance for Candida dubliniensis   

<table><tr><td rowspan=2 colspan=1>Target</td><td rowspan=2 colspan=1>Sample Type</td><td rowspan=1 colspan=2>Sensitivity/PPA</td><td rowspan=1 colspan=2>Specificity/NPA</td></tr><tr><td rowspan=1 colspan=1>TP/TP+FN</td><td rowspan=1 colspan=1>% (95% CI)</td><td rowspan=1 colspan=1>TN/TN+FP</td><td rowspan=1 colspan=1>% (95% CI)</td></tr><tr><td rowspan=7 colspan=1>Candidadubliniensis</td><td rowspan=1 colspan=1>Prospective (Fresh)</td><td rowspan=1 colspan=1>1/1</td><td rowspan=1 colspan=1>100 (20.7-100)</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>100 (72.2-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective (Frozen)</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>---</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>100 (72.2-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective (All)</td><td rowspan=1 colspan=1>1/1</td><td rowspan=1 colspan=1>100 (20.7-100)</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>100 (83.9-100)</td></tr><tr><td rowspan=1 colspan=1>Retrospective</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>100 (43.9-100)</td><td rowspan=1 colspan=1>117/117</td><td rowspan=1 colspan=1>100 (96.8-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective/Retrospective</td><td rowspan=1 colspan=1>4/4</td><td rowspan=1 colspan=1>100 (51.0-100)</td><td rowspan=1 colspan=1>137/137</td><td rowspan=1 colspan=1>100 (97.3-100)</td></tr><tr><td rowspan=1 colspan=1>Contrived</td><td rowspan=1 colspan=1>48/48</td><td rowspan=1 colspan=1>100 (92.6-100)</td><td rowspan=1 colspan=1>677/677</td><td rowspan=1 colspan=1>100 (99.4-100)</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>52/52</td><td rowspan=1 colspan=1>100 (93.1-100)</td><td rowspan=1 colspan=1>814/814</td><td rowspan=1 colspan=1>100 (99.5-100)</td></tr></table>

Table 36: Clinical Performance for Candida famata   

<table><tr><td rowspan=2 colspan=1>Target</td><td rowspan=2 colspan=1>Sample Type</td><td rowspan=1 colspan=2>Sensitivity/PPA</td><td rowspan=1 colspan=2>Specificity/NPA</td></tr><tr><td rowspan=1 colspan=1>TP/TP+FN</td><td rowspan=1 colspan=1>% (95% CI)</td><td rowspan=1 colspan=1>TN/TN+FP</td><td rowspan=1 colspan=1>% (95% CI)</td></tr><tr><td rowspan=7 colspan=1>Candidafamata</td><td rowspan=1 colspan=1>Prospective (Fresh)</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>---</td><td rowspan=1 colspan=1>11/11</td><td rowspan=1 colspan=1>100 (74.1-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective (Frozen)</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>---</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>100 (72.2-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective (All)</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>--</td><td rowspan=1 colspan=1>21/21</td><td rowspan=1 colspan=1>100 (84.5-100)</td></tr><tr><td rowspan=1 colspan=1>Retrospective</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>--</td><td rowspan=1 colspan=1>120/120</td><td rowspan=1 colspan=1>100 (96.9-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective/Retrospective</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>---</td><td rowspan=1 colspan=1>141/141</td><td rowspan=1 colspan=1>100 (97.3-100)</td></tr><tr><td rowspan=1 colspan=1>Contrived</td><td rowspan=1 colspan=1>51/51</td><td rowspan=1 colspan=1>100 (93.0-100)</td><td rowspan=1 colspan=1>674/674</td><td rowspan=1 colspan=1>100 (99.4-100)</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>51/51</td><td rowspan=1 colspan=1>100 (93.0-100)</td><td rowspan=1 colspan=1>815/815</td><td rowspan=1 colspan=1>100 (99.5-100)</td></tr></table>

Table 37: Clinical Performance for Candida glabrata   

<table><tr><td colspan="1" rowspan="2">Target</td><td colspan="1" rowspan="2">Sample Type</td><td colspan="2" rowspan="1">Sensitivity/PPA</td><td colspan="2" rowspan="1">Specificity/NPA</td></tr><tr><td colspan="1" rowspan="1">TP/TP+FN</td><td colspan="1" rowspan="1">% (95% CI)</td><td colspan="1" rowspan="1">TN/TN+FP</td><td colspan="1" rowspan="1">% (95% CI)</td></tr><tr><td colspan="1" rowspan="4">Candidaglabrata</td><td colspan="1" rowspan="1">Prospective (Fresh)</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">100 (51.0-100)</td><td colspan="1" rowspan="1">7/7</td><td colspan="1" rowspan="1">100 (64.6-100)</td></tr><tr><td colspan="1" rowspan="1">Prospective (Frozen)</td><td colspan="1" rowspan="1">2/2</td><td colspan="1" rowspan="1">100 (34.2-100)</td><td colspan="1" rowspan="1">8/8</td><td colspan="1" rowspan="1">100 (67.6-100)</td></tr><tr><td colspan="1" rowspan="1">Prospective (All)</td><td colspan="1" rowspan="1">6/6</td><td colspan="1" rowspan="1">100 (61.0-100)</td><td colspan="1" rowspan="1">15/15</td><td colspan="1" rowspan="1">100 (79.6-100)</td></tr><tr><td colspan="1" rowspan="1">Retrospective</td><td colspan="1" rowspan="1">37/38</td><td colspan="1" rowspan="1">97.4 (86.5-99.5)</td><td colspan="1" rowspan="1">80/82</td><td colspan="1" rowspan="1">97.6 (91.5-99.3)</td></tr><tr><td colspan="1" rowspan="1">Prospective/Retrospective</td><td colspan="1" rowspan="1">43/44</td><td colspan="1" rowspan="1">97.7 (88.2-99.6)</td><td colspan="1" rowspan="1">95/97A</td><td colspan="1" rowspan="1">97.9 (92.8-99.4)</td></tr><tr><td colspan="1" rowspan="1">Contrived</td><td colspan="1" rowspan="1">16/16</td><td colspan="1" rowspan="1">100 (80.6-100)</td><td colspan="1" rowspan="1">709/709</td><td colspan="1" rowspan="1">100 (99.5-100)</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">59/60</td><td colspan="1" rowspan="1">98.3 (91.1-99.7)</td><td colspan="1" rowspan="1">804/806</td><td colspan="1" rowspan="1">99.8 (99.1-99.9)</td></tr></table>

A. C. glabrata was detected by PCR/sequencing in 2/2 false positive clinical samples.

Table 38: Clinical Performance for Candida guilliermondii   

<table><tr><td rowspan=2 colspan=1>Target</td><td rowspan=2 colspan=1>Sample Type</td><td rowspan=1 colspan=2>Sensitivity/PPA</td><td rowspan=1 colspan=2>Specificity/NPA</td></tr><tr><td rowspan=1 colspan=1>TP/TP+FN</td><td rowspan=1 colspan=1>% (95% CI)</td><td rowspan=1 colspan=1>TN/TN+FP</td><td rowspan=1 colspan=1>% (95% CI)</td></tr><tr><td rowspan=7 colspan=1>Candidaguilliermondii</td><td rowspan=1 colspan=1>Prospective (Fresh)</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>---</td><td rowspan=1 colspan=1>11/11</td><td rowspan=1 colspan=1>100 (74.1-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective (Frozen)</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>---</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>100 (72.2-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective (All)</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>---</td><td rowspan=1 colspan=1>21/21</td><td rowspan=1 colspan=1>100 (84.5-100)</td></tr><tr><td rowspan=1 colspan=1>Retrospective</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>---</td><td rowspan=1 colspan=1>120/120</td><td rowspan=1 colspan=1>100 (96.9-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective/Retrospective</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>---</td><td rowspan=1 colspan=1>141/141</td><td rowspan=1 colspan=1>100 (97.3-100)</td></tr><tr><td rowspan=1 colspan=1>Contrived</td><td rowspan=1 colspan=1>49/50</td><td rowspan=1 colspan=1>98.0 (89.5-99.6)</td><td rowspan=1 colspan=1>675/675</td><td rowspan=1 colspan=1>100 (99.4-100)</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>49/50</td><td rowspan=1 colspan=1>98.0 (89.5-99.6)</td><td rowspan=1 colspan=1>816/816</td><td rowspan=1 colspan=1>100 (99.5-100)</td></tr></table>

Table 39: Clinical Performance for Candida kefyr   

<table><tr><td rowspan=2 colspan=1>Target</td><td rowspan=2 colspan=1>Sample Type</td><td rowspan=1 colspan=2>Sensitivity/PPA</td><td rowspan=1 colspan=2>Specificity/NPA</td></tr><tr><td rowspan=1 colspan=1>TP/TP+FN</td><td rowspan=1 colspan=1>% (95% CI)</td><td rowspan=1 colspan=1>TN/TN+FP</td><td rowspan=1 colspan=1>% (95% CI)</td></tr><tr><td rowspan=7 colspan=1>Candidakefyr</td><td rowspan=1 colspan=1>Prospective (Fresh)</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>---</td><td rowspan=1 colspan=1>11/11</td><td rowspan=1 colspan=1>100 (74.1-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective (Frozen)</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>---</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>100 (72.2-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective (All)</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>---</td><td rowspan=1 colspan=1>21/21</td><td rowspan=1 colspan=1>100 (84.5-100)</td></tr><tr><td rowspan=1 colspan=1>Retrospective</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>---</td><td rowspan=1 colspan=1>120/120</td><td rowspan=1 colspan=1>100 (96.9-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective/Retrospective</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>---</td><td rowspan=1 colspan=1>141/141</td><td rowspan=1 colspan=1>100 (97.3-100)</td></tr><tr><td rowspan=1 colspan=1>Contrived</td><td rowspan=1 colspan=1>51/51</td><td rowspan=1 colspan=1>100 (93.0-100)</td><td rowspan=1 colspan=1>672/674</td><td rowspan=1 colspan=1>99.7 (98.9-99.9)</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>51/51</td><td rowspan=1 colspan=1>100 (93.0-100)</td><td rowspan=1 colspan=1>813/815</td><td rowspan=1 colspan=1>99.8 (99.1-99.9)</td></tr></table>

Table 40: Clinical Performance for Candida krusei   

<table><tr><td rowspan=2 colspan=1>Target</td><td rowspan=2 colspan=1>Sample Type</td><td rowspan=1 colspan=2>Sensitivity/PPA</td><td rowspan=1 colspan=2>Specificity/NPA</td></tr><tr><td rowspan=1 colspan=1>TP/TP+FN</td><td rowspan=1 colspan=1>% (95% CI)</td><td rowspan=1 colspan=1>TN/TN+FP</td><td rowspan=1 colspan=1>% (95% CI)</td></tr><tr><td rowspan=7 colspan=1>Candidakrusei</td><td rowspan=1 colspan=1>Prospective (Fresh)</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>---</td><td rowspan=1 colspan=1>11/11</td><td rowspan=1 colspan=1>100 (74.1-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective (Frozen)</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>100 (34.2-100)</td><td rowspan=1 colspan=1>8/8</td><td rowspan=1 colspan=1>100 (67.6-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective (All)</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>100 (34.2-100)</td><td rowspan=1 colspan=1>19/19</td><td rowspan=1 colspan=1>100 (83.2-100)</td></tr><tr><td rowspan=1 colspan=1>Retrospective</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>100 (34.2-100)</td><td rowspan=1 colspan=1>118/118</td><td rowspan=1 colspan=1>100 (96.8-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective/Retrospective</td><td rowspan=1 colspan=1>4/4</td><td rowspan=1 colspan=1>100 (51.0-100)</td><td rowspan=1 colspan=1>137/137</td><td rowspan=1 colspan=1>100 (97.3-100)</td></tr><tr><td rowspan=1 colspan=1>Contrived</td><td rowspan=1 colspan=1>46/46</td><td rowspan=1 colspan=1>100 (92.3-100)</td><td rowspan=1 colspan=1>679/679</td><td rowspan=1 colspan=1>100 (99.4-100)</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>50/50</td><td rowspan=1 colspan=1>100 (92.9-100)</td><td rowspan=1 colspan=1>816/816</td><td rowspan=1 colspan=1>100 (99.5-100)</td></tr></table>

Table 41: Clinical Performance for Candida lusitaniae   

<table><tr><td>Target</td><td>Sample Type</td><td>Sensitivity/PPA</td><td>Specificity/NPA</td></tr><tr><td colspan="1" rowspan="1">Target</td><td colspan="1" rowspan="1">Sample Type</td><td colspan="2" rowspan="1">Sensitivity/PPA</td><td colspan="2" rowspan="1">Specificity/NPA</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">TP/TP+FN</td><td colspan="1" rowspan="1">% (95% CI)</td><td colspan="1" rowspan="1">TN/TN+FP</td><td colspan="1" rowspan="1">% (95% CI)</td></tr><tr><td colspan="1" rowspan="7">Candidalusitaniae</td><td colspan="1" rowspan="1">Prospective (Fresh)</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">---</td><td colspan="1" rowspan="1">11/11</td><td colspan="1" rowspan="1">100 (74.1-100)</td></tr><tr><td colspan="1" rowspan="1">Prospective (Frozen)</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">---</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">100 (72.2-100)</td></tr><tr><td colspan="1" rowspan="1">Prospective (All)</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">---</td><td colspan="1" rowspan="1">21/21</td><td colspan="1" rowspan="1">100 (84.5-100)</td></tr><tr><td colspan="1" rowspan="1">Retrospective</td><td colspan="1" rowspan="1">3/4</td><td colspan="1" rowspan="1">75.0 (30.1-95.4)</td><td colspan="1" rowspan="1">116/116</td><td colspan="1" rowspan="1">100 (96.8-100)</td></tr><tr><td colspan="1" rowspan="1">Prospective/Retrospective</td><td colspan="1" rowspan="1">3/4</td><td colspan="1" rowspan="1">75.0 (30.1-95.4)</td><td colspan="1" rowspan="1">137/137</td><td colspan="1" rowspan="1">100 (97.3-100)</td></tr><tr><td colspan="1" rowspan="1">Contrived</td><td colspan="1" rowspan="1">45/45</td><td colspan="1" rowspan="1">100 (92.1-100)</td><td colspan="1" rowspan="1">679/680</td><td colspan="1" rowspan="1">99.9 (99.2-100)</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">48/49</td><td colspan="1" rowspan="1">98.0 (89.3-99.6)</td><td colspan="1" rowspan="1">816/817</td><td colspan="1" rowspan="1">99.9 (99.3-100)</td></tr></table>

Table 42: Clinical Performance for Candida parapsilosis   

<table><tr><td rowspan=2 colspan=1>Target</td><td rowspan=2 colspan=1>Sample Type</td><td rowspan=1 colspan=2>Sensitivity/PPA</td><td rowspan=1 colspan=2>Specificity/NPA</td></tr><tr><td rowspan=1 colspan=1>TP/TP+FN</td><td rowspan=1 colspan=1>% (95% CI)</td><td rowspan=1 colspan=1>TN/TN+FP</td><td rowspan=1 colspan=1>% (95% CI)</td></tr><tr><td rowspan=7 colspan=1>Candidaparapsilosis</td><td rowspan=1 colspan=1>Prospective (Fresh)</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>100 (34.2-100)</td><td rowspan=1 colspan=1>9/9</td><td rowspan=1 colspan=1>100 (70.1-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective (Frozen)</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>---</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>100 (72.2-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective (All)</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>100 (34.2-100)</td><td rowspan=1 colspan=1>19/19</td><td rowspan=1 colspan=1>100 (83.2-100)</td></tr><tr><td rowspan=1 colspan=1>Retrospective</td><td rowspan=1 colspan=1>16/17</td><td rowspan=1 colspan=1>94.1 (73.0-99.0)</td><td rowspan=1 colspan=1>102/103</td><td rowspan=1 colspan=1>99.0 (94.7-99.8)</td></tr><tr><td rowspan=1 colspan=1>Prospective/Retrospective</td><td rowspan=1 colspan=1>18/19</td><td rowspan=1 colspan=1>94.7 (75.4-99.1)</td><td rowspan=1 colspan=1>121/122A</td><td rowspan=1 colspan=1>99.2 (95.5-99.9)</td></tr><tr><td rowspan=1 colspan=1>Contrived</td><td rowspan=1 colspan=1>41/41</td><td rowspan=1 colspan=1>100 (91.4-100)</td><td rowspan=1 colspan=1>684/684</td><td rowspan=1 colspan=1>100 (99.4-100)</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>59/60</td><td rowspan=1 colspan=1>98.3 (91.1-99.7)</td><td rowspan=1 colspan=1>805/806</td><td rowspan=1 colspan=1>99.9 (99.3-100)</td></tr></table>

A. C. parapsilosis was detected by PCR/sequencing in 1/1 false positive clinical sample.

Table 43: Clinical Performance for Candida tropicalis   

<table><tr><td rowspan=2 colspan=1>Target</td><td rowspan=2 colspan=1>Sample Type</td><td rowspan=1 colspan=2>Sensitivity/PPA</td><td rowspan=1 colspan=2>Specificity/NPA</td></tr><tr><td rowspan=1 colspan=1>TP/TP+FN</td><td rowspan=1 colspan=1>% (95% CI)</td><td rowspan=1 colspan=1>TN/TN+FP</td><td rowspan=1 colspan=1>% (95% CI)</td></tr><tr><td rowspan=7 colspan=1>Candidatropicalis</td><td rowspan=1 colspan=1>Prospective (Fresh)</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>---</td><td rowspan=1 colspan=1>11/11</td><td rowspan=1 colspan=1>100 (74.1-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective (Frozen)</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>100 (34.2-100)</td><td rowspan=1 colspan=1>8/8</td><td rowspan=1 colspan=1>100 (67.6-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective (All)</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>100 (34.2-100)</td><td rowspan=1 colspan=1>19/19</td><td rowspan=1 colspan=1>100 (83.2-100)</td></tr><tr><td rowspan=1 colspan=1>Retrospective</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>100 (43.9-100)</td><td rowspan=1 colspan=1>116/117</td><td rowspan=1 colspan=1>99.1 (95.3-99.8)</td></tr><tr><td rowspan=1 colspan=1>Prospective/Retrospective</td><td rowspan=1 colspan=1>5/5</td><td rowspan=1 colspan=1>100 (56.6-100)</td><td rowspan=1 colspan=1>135/136A</td><td rowspan=1 colspan=1>99.3 (96.0-99.9)</td></tr><tr><td rowspan=1 colspan=1>Contrived</td><td rowspan=1 colspan=1>45/45</td><td rowspan=1 colspan=1>100 (92.1-100)</td><td rowspan=1 colspan=1>680/680</td><td rowspan=1 colspan=1>100 (99.4-100)</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>50/50</td><td rowspan=1 colspan=1>100 (92.9-100)</td><td rowspan=1 colspan=1>815/816</td><td rowspan=1 colspan=1>99.9 (99.3-100)</td></tr></table>

A. C. tropicalis was detected by PCR/sequencing in 2/2 false positive clinical samples.

Table 44: Clinical Performance for Cryptococcus gattii   

<table><tr><td rowspan=2 colspan=1>Target</td><td rowspan=2 colspan=1>Sample Type</td><td rowspan=1 colspan=2>Sensitivity/PPA</td><td rowspan=1 colspan=2>Specificity/NPA</td></tr><tr><td rowspan=1 colspan=1>TP/TP+FN</td><td rowspan=1 colspan=1>% (95% CI)</td><td rowspan=1 colspan=1>TN/TN+FP</td><td rowspan=1 colspan=1>% (95% CI)</td></tr><tr><td rowspan=3 colspan=1>Cryptococcusgattii</td><td rowspan=1 colspan=1>Prospective (Fresh)</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>--</td><td rowspan=1 colspan=1>11/11</td><td rowspan=1 colspan=1>100 (74.1-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective (Frozen)</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>---</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>100 (72.2-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective (All)</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>--</td><td rowspan=1 colspan=1>21/21</td><td rowspan=1 colspan=1>100 (84.5-100)</td></tr></table>

Table 45: Clinical Performance for Cryptococcus neoformans   

<table><tr><td rowspan=1 colspan=1>Retrospective</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>---</td><td rowspan=1 colspan=1>120/120</td><td rowspan=1 colspan=1>100 (96.9-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective/Retrospective</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>---</td><td rowspan=1 colspan=1>141/141</td><td rowspan=1 colspan=1>100 (97.3-100)</td></tr><tr><td rowspan=1 colspan=1>Contrived</td><td rowspan=1 colspan=1>50/50</td><td rowspan=1 colspan=1>100 (92.9-100)</td><td rowspan=1 colspan=1>675/675</td><td rowspan=1 colspan=1>100 (99.4-100)</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>50/50</td><td rowspan=1 colspan=1>100 (92.9-100)</td><td rowspan=1 colspan=1>816/816</td><td rowspan=1 colspan=1>100 (99.5-100)</td></tr></table>

<table><tr><td rowspan=2 colspan=1>Target</td><td rowspan=2 colspan=1>Sample Type</td><td rowspan=1 colspan=2>Sensitivity/PPA</td><td rowspan=1 colspan=2>Specificity/NPA</td></tr><tr><td rowspan=1 colspan=1>TP/TP+FN</td><td rowspan=1 colspan=1>% (95% CI)</td><td rowspan=1 colspan=1>TN/TN+FP</td><td rowspan=1 colspan=1>% (95% CI)</td></tr><tr><td rowspan=7 colspan=1>Cryptococcusneoformans</td><td rowspan=1 colspan=1>Prospective (Fresh)</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>---</td><td rowspan=1 colspan=1>11/11</td><td rowspan=1 colspan=1>100 (74.1-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective (Frozen)</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>---</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>100 (72.2-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective (All)</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>---</td><td rowspan=1 colspan=1>21/21</td><td rowspan=1 colspan=1>100 (84.5-100)</td></tr><tr><td rowspan=1 colspan=1>Retrospective</td><td rowspan=1 colspan=1>5/5</td><td rowspan=1 colspan=1>100 (56.6-100)</td><td rowspan=1 colspan=1>115/115</td><td rowspan=1 colspan=1>100 (96.8-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective/Retrospective</td><td rowspan=1 colspan=1>5/5</td><td rowspan=1 colspan=1>100 (56.6-100)</td><td rowspan=1 colspan=1>136/136</td><td rowspan=1 colspan=1>100 (97.3-100)</td></tr><tr><td rowspan=1 colspan=1>Contrived</td><td rowspan=1 colspan=1>52/52</td><td rowspan=1 colspan=1>100 (93.1-100)</td><td rowspan=1 colspan=1>673/673</td><td rowspan=1 colspan=1>100 (99.4-100)</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>57/57</td><td rowspan=1 colspan=1>100 (93.7-100)</td><td rowspan=1 colspan=1>809/809</td><td rowspan=1 colspan=1>100 (99.5-100)</td></tr></table>

Table 46: Clinical Performance for Fusarium   

<table><tr><td rowspan=2 colspan=1>Target</td><td rowspan=2 colspan=1>Sample Type</td><td rowspan=1 colspan=2>Sensitivity/PPA</td><td rowspan=1 colspan=2>Specificity/NPA</td></tr><tr><td rowspan=1 colspan=1>TP/TP+FN</td><td rowspan=1 colspan=1>% (95% CI)</td><td rowspan=1 colspan=1>TN/TN+FP</td><td rowspan=1 colspan=1>% (95% CI)</td></tr><tr><td rowspan=7 colspan=1>Fusarium</td><td rowspan=1 colspan=1>Prospective (Fresh)</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>---</td><td rowspan=1 colspan=1>11/11</td><td rowspan=1 colspan=1>100 (74.1-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective (Frozen)</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>---</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>100 (72.2-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective (All)</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>---</td><td rowspan=1 colspan=1>21/21</td><td rowspan=1 colspan=1>100 (84.5-100)</td></tr><tr><td rowspan=1 colspan=1>Retrospective</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>---</td><td rowspan=1 colspan=1>120/120</td><td rowspan=1 colspan=1>100 (96.9-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective/Retrospective</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>---</td><td rowspan=1 colspan=1>141/141</td><td rowspan=1 colspan=1>100 (97.3-100)</td></tr><tr><td rowspan=1 colspan=1>Contrived</td><td rowspan=1 colspan=1>69/70</td><td rowspan=1 colspan=1>98.6 (92.3-99.7)</td><td rowspan=1 colspan=1>655/655</td><td rowspan=1 colspan=1>100 (99.4-100)</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>69/70</td><td rowspan=1 colspan=1>98.6 (92.3-99.7)</td><td rowspan=1 colspan=1>796/796</td><td rowspan=1 colspan=1>100 (99.5-100)</td></tr></table>

Table 47: Clinical Performance for Rhodotorula   

<table><tr><td rowspan=2 colspan=1>Target</td><td rowspan=2 colspan=1>Sample Type</td><td rowspan=1 colspan=2>Sensitivity/PPA</td><td rowspan=1 colspan=2>Specificity/NPA</td></tr><tr><td rowspan=1 colspan=1>TP/TP+FN</td><td rowspan=1 colspan=1>% (95% CI)</td><td rowspan=1 colspan=1>TN/TN+FP</td><td rowspan=1 colspan=1>% (95% CI)</td></tr><tr><td rowspan=7 colspan=1>Rhodotorula</td><td rowspan=1 colspan=1>Prospective (Fresh)</td><td rowspan=1 colspan=1>1/1</td><td rowspan=1 colspan=1>100 (20.7-100)</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>100 (72.2-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective (Frozen)</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>---</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>100 (72.2-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective (All)</td><td rowspan=1 colspan=1>1/1</td><td rowspan=1 colspan=1>100 (20.7-100)</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>100 (83.9-100)</td></tr><tr><td rowspan=1 colspan=1>Retrospective</td><td rowspan=1 colspan=1>1/1</td><td rowspan=1 colspan=1>100 (20.7-100)</td><td rowspan=1 colspan=1>119/119</td><td rowspan=1 colspan=1>100 (96.9-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective/Retrospective</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>100 (34.2-100)</td><td rowspan=1 colspan=1>139/139</td><td rowspan=1 colspan=1>100 (97.3-100)</td></tr><tr><td rowspan=1 colspan=1>Contrived</td><td rowspan=1 colspan=1>48/50</td><td rowspan=1 colspan=1>96.0 (86.5-98.9)</td><td rowspan=1 colspan=1>674/675</td><td rowspan=1 colspan=1>99.9 (99.2-100)</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>50/52</td><td rowspan=1 colspan=1>96.2 (87.0-98.9)</td><td rowspan=1 colspan=1>813/814</td><td rowspan=1 colspan=1>99.9 (99.3-100)</td></tr></table>

The ePlex BCID-FP Panel reports genus level results for Fusarium and Rhodotorula targets. Sensitivity/PPA of these genus and group level targets for species from all clinical and contrived samples tested are summarized in Table 48 below.

Table 48: Species Detected by Genus Assays   

<table><tr><td rowspan=3 colspan=1>Target Species Detected</td><td rowspan=1 colspan=2>All Samples</td></tr><tr><td rowspan=1 colspan=2>Sensitivity/PPA</td></tr><tr><td rowspan=1 colspan=1>TP/TP+FN</td><td rowspan=1 colspan=1>% (95% CI)</td></tr><tr><td rowspan=1 colspan=1>Fusarium</td><td rowspan=1 colspan=1>69/70</td><td rowspan=1 colspan=1>98.6 (92.3-99.7)</td></tr><tr><td rowspan=1 colspan=1>Bisifusarium dimerum</td><td rowspan=1 colspan=1>14/15</td><td rowspan=1 colspan=1>93.3 (70.2-98.8)</td></tr><tr><td rowspan=1 colspan=1>Fusarium moniliforme</td><td rowspan=1 colspan=1>11/11</td><td rowspan=1 colspan=1>100 (74.1-100)</td></tr><tr><td rowspan=1 colspan=1>Fusarium oxysporum</td><td rowspan=1 colspan=1>1/1</td><td rowspan=1 colspan=1>100 (20.7-100)</td></tr><tr><td rowspan=1 colspan=1>Fusarium sacchari</td><td rowspan=1 colspan=1>21/21</td><td rowspan=1 colspan=1>100 (84.5-100)</td></tr><tr><td rowspan=1 colspan=1>Fusarium solani</td><td rowspan=1 colspan=1>11/11</td><td rowspan=1 colspan=1>100 (74.1-100)</td></tr><tr><td rowspan=1 colspan=1>Fusarium verticillioides</td><td rowspan=1 colspan=1>11/11</td><td rowspan=1 colspan=1>100 (74.1-100)</td></tr><tr><td rowspan=1 colspan=1>Rhodotorula</td><td rowspan=1 colspan=1>50/52</td><td rowspan=1 colspan=1>96.2 (87.0-98.9)</td></tr><tr><td rowspan=1 colspan=1>Rhodotorula*</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>100 (34.2-100)</td></tr><tr><td rowspan=1 colspan=1>Rhodotorula glutinis</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>100 (61.0-100)</td></tr><tr><td rowspan=1 colspan=1>Rhodotorula glutinis (Fresenius) Harrison</td><td rowspan=1 colspan=1>4/4</td><td rowspan=1 colspan=1>100 (51.0-100)</td></tr><tr><td rowspan=1 colspan=1>Rhodotorula mucilaginosa</td><td rowspan=1 colspan=1>38/40</td><td rowspan=1 colspan=1>95.0 (83.5-98.6)</td></tr></table>

\*Results are from 2 clinical samples (1 prospective, 1 retrospective). All other Fusarium and Rhodotorula samples were contrived.

Table 49: Contrived Sample Summary   

<table><tr><td colspan="1" rowspan="1">Target</td><td colspan="1" rowspan="1">Organism</td><td colspan="1" rowspan="1">Strain</td><td colspan="1" rowspan="1">Independent Contrived Samples Tested</td></tr><tr><td colspan="1" rowspan="6">Candida albicans</td><td colspan="1" rowspan="5">Candida albicans</td><td colspan="1" rowspan="1">ATCC 10231</td><td colspan="1" rowspan="1">2</td></tr><tr><td colspan="1" rowspan="1">ATCC 14053</td><td colspan="1" rowspan="1">2</td></tr><tr><td colspan="1" rowspan="1">ATCC 24433</td><td colspan="1" rowspan="1">2</td></tr><tr><td colspan="1" rowspan="1">ATCC 90028</td><td colspan="1" rowspan="1">5</td></tr><tr><td colspan="1" rowspan="1">ATCC MYA-4441</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">Candida albicans total</td><td colspan="1" rowspan="1">cYanmaGellowb</td><td colspan="1" rowspan="1">14</td></tr><tr><td colspan="1" rowspan="15">Candida auris</td><td colspan="1" rowspan="15">Candida auris</td><td colspan="1" rowspan="1">ATCC 10913</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">ATCC 12372</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">ATCC 12766</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">CBS 10913</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">CBS12372</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">CBS 12373</td><td colspan="1" rowspan="1">2</td></tr><tr><td colspan="1" rowspan="1">CBS12766</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">CBS12767</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">CBS 12768</td><td colspan="1" rowspan="1">2</td></tr><tr><td colspan="1" rowspan="1">CDC#0385</td><td colspan="1" rowspan="1">5</td></tr><tr><td colspan="1" rowspan="1">CDC#0386</td><td colspan="1" rowspan="1">5</td></tr><tr><td colspan="1" rowspan="1">CDC#0387</td><td colspan="1" rowspan="1">5</td></tr><tr><td colspan="1" rowspan="1">CDC#0388</td><td colspan="1" rowspan="1">5</td></tr><tr><td colspan="1" rowspan="1">CDC#0389</td><td colspan="1" rowspan="1">5</td></tr><tr><td colspan="1" rowspan="1">CDC#0390</td><td colspan="1" rowspan="1">5</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="2" rowspan="1">Candida auris total</td><td colspan="1" rowspan="1">49</td></tr><tr><td colspan="1" rowspan="6">Candida dubliniensis</td><td colspan="1" rowspan="5">Candida dubliniensis</td><td colspan="1" rowspan="1">ATCC MYA-577</td><td colspan="1" rowspan="1">6</td></tr><tr><td colspan="1" rowspan="1">ATCC MYA-578</td><td colspan="1" rowspan="1">12</td></tr><tr><td colspan="1" rowspan="1">ATCC MYA-579</td><td colspan="1" rowspan="1">12</td></tr><tr><td colspan="1" rowspan="1">ATCC MYA-582</td><td colspan="1" rowspan="1">13</td></tr><tr><td colspan="1" rowspan="1">NCPF3949</td><td colspan="1" rowspan="1">5</td></tr><tr><td colspan="2" rowspan="1">Candida dubliniensis total</td><td colspan="1" rowspan="1">48</td></tr><tr><td colspan="1" rowspan="4">Candida famata</td><td colspan="1" rowspan="1">Debaryomycesfabryi</td><td colspan="1" rowspan="1">CBS 789</td><td colspan="1" rowspan="1">21</td></tr><tr><td colspan="1" rowspan="1">Debaryomyceshansenii</td><td colspan="1" rowspan="1">CBS 1961</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">Debaryomycessubglobosus</td><td colspan="1" rowspan="1">CBS 1796</td><td colspan="1" rowspan="1">27</td></tr><tr><td colspan="1" rowspan="1">Candida famata total</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">51</td></tr><tr><td colspan="1" rowspan="9">Candida glabrata</td><td colspan="1" rowspan="8">Candida glabrata</td><td colspan="1" rowspan="1">128-4058</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">128-4072</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">ATCC 15126</td><td colspan="1" rowspan="1">2</td></tr><tr><td colspan="1" rowspan="1">ATCC 15545</td><td colspan="1" rowspan="1">2</td></tr><tr><td colspan="1" rowspan="1">ATCC 2001</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">ATCC 66032</td><td colspan="1" rowspan="1">4</td></tr><tr><td colspan="1" rowspan="1">ATCC 90030</td><td colspan="1" rowspan="1">2</td></tr><tr><td colspan="1" rowspan="1">ATCC MYA-2950</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">Candida glabrata total</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">16</td></tr><tr><td colspan="1" rowspan="6">Candida guilliermondii</td><td colspan="1" rowspan="2">Candidaguilliermondii</td><td colspan="1" rowspan="1">ATCC 22017</td><td colspan="1" rowspan="1">13</td></tr><tr><td colspan="1" rowspan="1">ATCC 6260</td><td colspan="1" rowspan="1">12</td></tr><tr><td colspan="1" rowspan="3">Meyerozymaguilliermondii</td><td colspan="1" rowspan="1">ATCC 90197</td><td colspan="1" rowspan="1">10</td></tr><tr><td colspan="1" rowspan="1">ATCC 90198</td><td colspan="1" rowspan="1">9</td></tr><tr><td colspan="1" rowspan="1">ATCC 90199</td><td colspan="1" rowspan="1">6</td></tr><tr><td colspan="2" rowspan="1">Candida guilliermondii total</td><td colspan="1" rowspan="1">50</td></tr><tr><td colspan="1" rowspan="6">Candida kefyr</td><td colspan="1" rowspan="5">Candida kefyr</td><td colspan="1" rowspan="1">ATCC 204093</td><td colspan="1" rowspan="1">4</td></tr><tr><td colspan="1" rowspan="1">ATCC 2512</td><td colspan="1" rowspan="1">10</td></tr><tr><td colspan="1" rowspan="1">ATCC 4135</td><td colspan="1" rowspan="1">13</td></tr><tr><td colspan="1" rowspan="1">ATCC 66028</td><td colspan="1" rowspan="1">12</td></tr><tr><td colspan="1" rowspan="1">ATCC 8553</td><td colspan="1" rowspan="1">12</td></tr><tr><td colspan="2" rowspan="1">Candida kefyr total</td><td colspan="1" rowspan="1">51</td></tr><tr><td colspan="1" rowspan="6">Candida krusei</td><td colspan="1" rowspan="5">Candida krusei</td><td colspan="1" rowspan="1">ATCC 14243</td><td colspan="1" rowspan="1">8</td></tr><tr><td colspan="1" rowspan="1">ATCC 22985</td><td colspan="1" rowspan="1">9</td></tr><tr><td colspan="1" rowspan="1">ATCC 28870</td><td colspan="1" rowspan="1">9</td></tr><tr><td colspan="1" rowspan="1">ATCC 32196</td><td colspan="1" rowspan="1">10</td></tr><tr><td colspan="1" rowspan="1">ATCC 34135</td><td colspan="1" rowspan="1">10</td></tr><tr><td colspan="1" rowspan="1">Candida krusei total</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">46</td></tr><tr><td colspan="1" rowspan="1">Candida lusitaniae</td><td colspan="1" rowspan="1">Candida lusitaniae</td><td colspan="1" rowspan="1">ATCC 26287</td><td colspan="1" rowspan="1">5</td></tr><tr><td colspan="1" rowspan="5"></td><td colspan="1" rowspan="4"></td><td colspan="1" rowspan="1">ATCC 34449</td><td colspan="1" rowspan="1">10</td></tr><tr><td colspan="1" rowspan="1">ATCC 42720</td><td colspan="1" rowspan="1">9</td></tr><tr><td colspan="1" rowspan="1">ATCC 66035</td><td colspan="1" rowspan="1">11</td></tr><tr><td colspan="1" rowspan="1">ATCC MYA-766</td><td colspan="1" rowspan="1">10</td></tr><tr><td colspan="1" rowspan="1">Candida lusitaniae total</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">45</td></tr><tr><td colspan="1" rowspan="6">Candida parapsilosis</td><td colspan="1" rowspan="5">Candidaparapsilosis</td><td colspan="1" rowspan="1">ATCC 22019</td><td colspan="1" rowspan="1">11</td></tr><tr><td colspan="1" rowspan="1">ATCC 28474</td><td colspan="1" rowspan="1">5</td></tr><tr><td colspan="1" rowspan="1">ATCC 28475</td><td colspan="1" rowspan="1">10</td></tr><tr><td colspan="1" rowspan="1">ATCC 58895</td><td colspan="1" rowspan="1">7</td></tr><tr><td colspan="1" rowspan="1">ATCC 90018</td><td colspan="1" rowspan="1">8</td></tr><tr><td colspan="2" rowspan="1">Candida parapsilosis total</td><td colspan="1" rowspan="1">41</td></tr><tr><td colspan="1" rowspan="6">Candida tropicalis</td><td colspan="1" rowspan="5">Candida tropicalis</td><td colspan="1" rowspan="1">ATCC 1369</td><td colspan="1" rowspan="1">9</td></tr><tr><td colspan="1" rowspan="1">ATCC 13803</td><td colspan="1" rowspan="1">12</td></tr><tr><td colspan="1" rowspan="1">ATCC 201380</td><td colspan="1" rowspan="1">9</td></tr><tr><td colspan="1" rowspan="1">ATCC 201381</td><td colspan="1" rowspan="1">7</td></tr><tr><td colspan="1" rowspan="1">ATCC 750</td><td colspan="1" rowspan="1">8</td></tr><tr><td colspan="1" rowspan="1">Candida tropicalis total</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">45</td></tr><tr><td colspan="1" rowspan="6">Cryptococcus gattii</td><td colspan="1" rowspan="5">Cryptococcus gattii</td><td colspan="1" rowspan="1">ATCC 14248</td><td colspan="1" rowspan="1">11</td></tr><tr><td colspan="1" rowspan="1">ATCC 76108</td><td colspan="1" rowspan="1">12</td></tr><tr><td colspan="1" rowspan="1">ATCC MYA-4138</td><td colspan="1" rowspan="1">10</td></tr><tr><td colspan="1" rowspan="1">ATCC MYA-4560</td><td colspan="1" rowspan="1">8</td></tr><tr><td colspan="1" rowspan="1">ATCC MYA-4877</td><td colspan="1" rowspan="1">9</td></tr><tr><td colspan="2" rowspan="1">Cryptococcus gattii total</td><td colspan="1" rowspan="1">50</td></tr><tr><td colspan="1" rowspan="10">Cryptococcusneoformans</td><td colspan="1" rowspan="6">Cryptococcusneoformans var.grubii</td><td colspan="1" rowspan="1">ATCC 14116</td><td colspan="1" rowspan="1">9</td></tr><tr><td colspan="1" rowspan="1">ATCC 208821(H99)</td><td colspan="1" rowspan="1">8</td></tr><tr><td colspan="1" rowspan="1">ATCC 90112</td><td colspan="1" rowspan="1">7</td></tr><tr><td colspan="1" rowspan="1">NCPF8195</td><td colspan="1" rowspan="1">7</td></tr><tr><td colspan="1" rowspan="1">NCPF8299</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">NCPF8357</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">Cryptococcusneoformans var.neoformans(Filobasidiellabacillispora Kwon-Chung, teleomorph(serotype D))</td><td colspan="1" rowspan="1">ATCC 34873</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="2">Cryptococcusneoformans var.neoformans(serotype D)</td><td colspan="1" rowspan="1">ATCC 36556</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">ATCC MYA-565</td><td colspan="1" rowspan="1">9</td></tr><tr><td colspan="2" rowspan="1">Cryptococcus neoformans total</td><td colspan="1" rowspan="1">52</td></tr><tr><td colspan="1" rowspan="2">Fusarium</td><td colspan="1" rowspan="2">Bisifusarium</td><td colspan="1" rowspan="1">CBS 108944</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">CBS 110317</td><td colspan="1" rowspan="1">9</td></tr><tr><td colspan="1" rowspan="8">−</td><td colspan="1" rowspan="1">dimerum</td><td colspan="1" rowspan="1">CBS 116520</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">Fusariummoniliforme</td><td colspan="1" rowspan="1">ATCC 38159</td><td colspan="1" rowspan="1">11</td></tr><tr><td colspan="1" rowspan="1">Fusarium oxysporum</td><td colspan="1" rowspan="1">CBS 116611</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="2">Fusarium sacchari</td><td colspan="1" rowspan="1">ATCC 24379</td><td colspan="1" rowspan="1">10</td></tr><tr><td colspan="1" rowspan="1">CBS 119828</td><td colspan="1" rowspan="1">11</td></tr><tr><td colspan="1" rowspan="1">Fusarium solani</td><td colspan="1" rowspan="1">ATCC 36031</td><td colspan="1" rowspan="1">11</td></tr><tr><td colspan="1" rowspan="1">Fusariumverticillioides</td><td colspan="1" rowspan="1">CBS 100312</td><td colspan="1" rowspan="1">11</td></tr><tr><td colspan="2" rowspan="1">Fusarium total</td><td colspan="1" rowspan="1">70</td></tr><tr><td colspan="1" rowspan="6">Rhodotorula</td><td colspan="1" rowspan="2">Rhodotorula glutinis</td><td colspan="1" rowspan="1">ATCC 32765</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">ATCC 32766</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">Rhodotorula glutinis(Fresenius) Harrison</td><td colspan="1" rowspan="1">ATCC 96365</td><td colspan="1" rowspan="1">4</td></tr><tr><td colspan="1" rowspan="2">Rhodotorulamucilaginosa</td><td colspan="1" rowspan="1">ATCC 66034</td><td colspan="1" rowspan="1">21</td></tr><tr><td colspan="1" rowspan="1">ATCC 9449</td><td colspan="1" rowspan="1">19</td></tr><tr><td colspan="2" rowspan="1">Rhodotorula total</td><td colspan="1" rowspan="1">50</td></tr></table>

# Co-detections in Clinical Samples

The ePlex BCID-FP Panel did not identify any fungal co-detections in prospective samples, and 6 fungal co-detections were identified in the retrospective samples. Of the 120 retrospective samples, 114 $( 9 5 . 0 \% )$ had single detections and 6 $( 5 . 0 \% )$ had double detections. In the 6 co-detections, 4 included 1 organism that comparator method(s) did not detect. See Table 50 below for a summary of codetections in retrospective samples.

Table 50: Co-Detections Identified by the ePlex BCID-FP Panel (Retrospective Samples)   

<table><tr><td rowspan=1 colspan=2>Distinct Organis m CombinationsDetected by the e Plex Panel inRetrospective Samples</td><td rowspan=2 colspan=1>Number ofSamples (NumberDiscre pant)</td><td rowspan=2 colspan=1>DiscrepantOrganis m(s )A,B</td></tr><tr><td rowspan=1 colspan=1>Target 1</td><td rowspan=1 colspan=1>Target 2</td></tr><tr><td rowspan=1 colspan=1>Candida albicans</td><td rowspan=1 colspan=1>Candida dubliniensis</td><td rowspan=1 colspan=1>1 (0)</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Candida albicans</td><td rowspan=1 colspan=1>Candida parapsilosis</td><td rowspan=1 colspan=1>1 (0)</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Candida dubliniensis</td><td rowspan=1 colspan=1>Candida parapsilosis</td><td rowspan=1 colspan=1>1 (1)</td><td rowspan=1 colspan=1>C. parapsilosis (1)</td></tr><tr><td rowspan=1 colspan=1>Candida dubliniensis</td><td rowspan=1 colspan=1>Candida tropicalis</td><td rowspan=1 colspan=1>1 (1)</td><td rowspan=1 colspan=1>C. tropicalis (1)</td></tr><tr><td rowspan=1 colspan=1>Candida glabrata</td><td rowspan=1 colspan=1>Candida lusitaniae</td><td rowspan=1 colspan=1>2 (2)</td><td rowspan=1 colspan=1>C. glabrata (2)</td></tr></table>

A A discrepant organism is defined as one that was detected by the BCID-FP Panel but not by the comparator method(s). B 4/4 organisms were investigated using PCR/sequencing and the discrepant organism was detected in 4/4 cases; 1/1

Summaries of additional distinct fungal co-detections detected by the comparator method in prospective and retrospective samples are provided in Tables 51 and 52. These tables include additional distinct fungal co-detections not included in the co-detections identified by the BCID-FP Panel; they have 1 or more organism not detected by the BCID-FP Panel or an off-panel fungal organism.

Table 51: Co-Detections Identified by the Comparator Method (Prospective Samples)   

<table><tr><td rowspan=1 colspan=2>Dis tinct Organis m CombinationsDetected by the ePlex Panel inPros pe ctive Samples</td><td rowspan=2 colspan=1>Number ofSamples (NumberDiscrepant)</td><td rowspan=2 colspan=1>Dis cre pantOrganis m(s )4</td></tr><tr><td rowspan=1 colspan=1>Target 1</td><td rowspan=1 colspan=1>Target 2</td></tr><tr><td rowspan=1 colspan=1>Candidametapsilosis*</td><td rowspan=1 colspan=1>Trichosporon asahii*</td><td rowspan=1 colspan=1>1 (0)</td><td rowspan=1 colspan=1></td></tr></table>

A A discrepant organism is defined as one that was detected by the comparator method(s) but not by the BCIDFP Panel (excludes organisms not targeted by the BCID-FP Panel). \* Off-panel organism not targeted by the BCID-FP Panel.

Table 52:Co-Detections Identified by the Comparator Method (Retrospective Samples)   

<table><tr><td rowspan=1 colspan=3>Dis tinct Organis m Combinations Dete cted bythe ePle x Panel in Retrospective Samples</td><td rowspan=2 colspan=1>Number ofSamples(NumberDiscrepant)</td><td rowspan=2 colspan=1>Discre pantOrganis m(s )4</td></tr><tr><td rowspan=1 colspan=1>Target 1</td><td rowspan=1 colspan=1>Target 2</td><td rowspan=1 colspan=1>Target 3</td></tr><tr><td rowspan=1 colspan=1>Candidaalbicans</td><td rowspan=1 colspan=1>Candidadubliniensis</td><td rowspan=1 colspan=1>Candidaglabrata</td><td rowspan=1 colspan=1>1 (0)</td><td rowspan=1 colspan=1>C. glabrata (1)</td></tr><tr><td rowspan=1 colspan=1>Candidaalbicans</td><td rowspan=1 colspan=1>Candidaparapsilosis</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1 (0)</td><td rowspan=1 colspan=1>C. parapsilosis (1)</td></tr></table>

A A discrepant organism is defined as one that was detected by the comparator method(s) but not by the BCID-FP Panel (excludes organisms not targeted by the BCID-FP Panel).

A total of 867 samples (including prospective, retrospective, and contrived samples) were initially tested in the clinical evaluations. Of these, 7/867 $( 0 . 8 \% )$ did not complete the run and the sample was retested. After repeat testing, all 867 samples completed testing and 839/867 $9 6 . 8 \%$ , $9 5 \%$ CI: $9 5 . 4 \% - 9 7 . 8 \%$ ) generated valid results and 28/867 $3 . 2 \%$ , $9 5 \%$ CI: $2 . 2 \% - 4 . 6 \%$ ) generated invalid results on the first completed attempt.

Upon repeat testing of the 28 samples with initially invalid results, all completed the run and 27/28 $( 9 6 . 4 \% )$ generated valid results. Overall, after final testing, 1/867 $( 0 . 1 \%$ , $9 5 \%$ CI: $0 . 0 \% 0 { - } 0 . 7 \% )$ ) had final, invalid results, resulting in a final validity rate of 866/867 $( 9 9 . 9 \%$ , $9 5 \%$ CI: $9 9 . 3 \% - 1 0 0 \%$ ).

External controls used in the clinical study consisted of 2 positive controls and 1 negative control. The positive controls together represent all the targets on the ePlex BCID-FP Panel. On each day of testing with the ePlex BCID-FP Panel, each site tested one positive and one negative control. Sites rotated through the 2 positive controls throughout the study.

Table 53: External Controls Utilized in the Clinical Evaluations   

<table><tr><td rowspan=1 colspan=1>Mix ID</td><td rowspan=1 colspan=1>Plasmids in Mix</td><td rowspan=1 colspan=1>Concentration in Mix(Plas mid Copies/mL)</td></tr><tr><td rowspan=8 colspan=1>Mix 1</td><td rowspan=1 colspan=1>Candida albicans</td><td rowspan=1 colspan=1>2 x106</td></tr><tr><td rowspan=1 colspan=1>Candida glabrata</td><td rowspan=1 colspan=1>2 x106</td></tr><tr><td rowspan=1 colspan=1>Candida kefyr</td><td rowspan=1 colspan=1>1 x106</td></tr><tr><td rowspan=1 colspan=1>Candida krusei</td><td rowspan=1 colspan=1>2 x106</td></tr><tr><td rowspan=1 colspan=1>Cryptococcus neoformans</td><td rowspan=1 colspan=1>1 x105</td></tr><tr><td rowspan=1 colspan=1>Candida famata</td><td rowspan=1 colspan=1>5 x107</td></tr><tr><td rowspan=1 colspan=1>Candida dublinensis</td><td rowspan=1 colspan=1>1 x107</td></tr><tr><td rowspan=1 colspan=1>Rhodotorula</td><td rowspan=1 colspan=1>1 x108</td></tr><tr><td rowspan=7 colspan=1>Mix 2</td><td rowspan=1 colspan=1>Candida auris</td><td rowspan=1 colspan=1>1 x107</td></tr><tr><td rowspan=1 colspan=1>Candida guillermondii</td><td rowspan=1 colspan=1>2 x105</td></tr><tr><td rowspan=1 colspan=1>Candida lusitaniae</td><td rowspan=1 colspan=1>2 x105</td></tr><tr><td rowspan=1 colspan=1>Candida parapsilosis</td><td rowspan=1 colspan=1>2 x106</td></tr><tr><td rowspan=1 colspan=1>Candida tropicalis</td><td rowspan=1 colspan=1>2 x106</td></tr><tr><td rowspan=1 colspan=1>Cryptococcus gattii</td><td rowspan=1 colspan=1>3 x105</td></tr><tr><td rowspan=1 colspan=1>Fusarium</td><td rowspan=1 colspan=1>2 x106</td></tr></table>

Over 47 days of investigational testing across 4 sites (median number of days/site was 12), there were 93 external control samples tested with valid results: 23 Positive Control A, 24 Positive Control B, and 46 Negative Control. Of these, 92/93 had the expected results. The remaining 1 external control sample had the expected results except as follows: 1 Positive Control B sample had 1 Candida glabrata false positive result.

4. Clinical cut-off: Not applicable

A prospective, multicenter clinical study was conducted to evaluate the clinical performance of the ePlex BCID-FP Panel in positive blood culture samples. A total of 447 positive blood culture samples were collected at 6 clinical sites in 2 phases from patients of all ages and genders. Samples were collected and frozen for future testing from May 2015 through July 2016. Samples were collected from July through August 2018 and tested fresh (never frozen). Of these 447 samples, 21 had a Gram stain result indicating fungal organism. The expected values of individual analytes based on the ePlex BCID-FP Panel results in the 21 prospective samples are summarized by age group and by site in Tables 54 and 55 below.

Table 54: Expected Value by Age Group (Prospective Samples)   

<table><tr><td rowspan=1 colspan=1>Target</td><td rowspan=1 colspan=1>All AgesN 45)n(%)</td><td rowspan=1 colspan=1>Age &lt;1(N1n (%)</td><td rowspan=1 colspan=1>Age 1-17(N=23n(%</td><td rowspan=1 colspan=1>Age 18-44(=70)n (%)</td><td rowspan=1 colspan=1>Age 45-64(N=158)n (%)</td><td rowspan=1 colspan=1>Age 65-84(N=145)n (%)</td><td rowspan=1 colspan=1>Age 85+N=3)n (%)</td></tr><tr><td rowspan=1 colspan=1>Candida albicans</td><td rowspan=1 colspan=1>4(19.0)</td><td rowspan=1 colspan=1>1 (100)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>2 (18.2)</td><td rowspan=1 colspan=1>1 (50.0)</td><td rowspan=1 colspan=1>0 (0.0)</td></tr><tr><td rowspan=1 colspan=1>Candida auris</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td></tr><tr><td rowspan=1 colspan=1>Candida dubliniensis</td><td rowspan=1 colspan=1>1 (4.8)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>1 (9.1)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td></tr><tr><td rowspan=1 colspan=1>Candida famata</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td></tr><tr><td rowspan=1 colspan=1>Candida glabrata</td><td rowspan=1 colspan=1>6 (28.6)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>1 (50.0)</td><td rowspan=1 colspan=1>1 (25.0)</td><td rowspan=1 colspan=1>3 (27.3)</td><td rowspan=1 colspan=1>1 (50.0)</td><td rowspan=1 colspan=1>0 (0.0)</td></tr><tr><td rowspan=1 colspan=1>Candida guilliermondii</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td></tr><tr><td rowspan=1 colspan=1>Candida kefyr</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td></tr><tr><td rowspan=1 colspan=1>Candida krusei</td><td rowspan=1 colspan=1>2(9.5)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>2 (18.2)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td></tr><tr><td rowspan=1 colspan=1>Candida lusitaniae</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td></tr><tr><td rowspan=1 colspan=1>Candida parapsilosis</td><td rowspan=1 colspan=1>2 (9.5)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>1 (50.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>1 (9.1)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td></tr><tr><td rowspan=1 colspan=1>Candida tropicalis</td><td rowspan=1 colspan=1>2 (9.5)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>1 (25.0)</td><td rowspan=1 colspan=1>1 (9.1)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td></tr><tr><td rowspan=1 colspan=1>Cryptococcus gattii</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td></tr><tr><td rowspan=1 colspan=1>Cryptococcus neoformans</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td></tr><tr><td rowspan=1 colspan=1>Fusarium</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td></tr><tr><td rowspan=1 colspan=1>Rhodotorula</td><td rowspan=1 colspan=1>1(4.8)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>1 (25.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td></tr></table>

Table 55: Expected Value by Collection Site (Prospective Samples)   

<table><tr><td colspan="1" rowspan="1">Target</td><td colspan="1" rowspan="1">All Sites(N=445)n (%)</td><td colspan="1" rowspan="1">Site 1(N=25)n(%)</td><td colspan="1" rowspan="1">Site 12(N=162)n%</td><td colspan="1" rowspan="1">Site 3(N=74)n(%)</td><td colspan="1" rowspan="1">Site 4(N=65)n (%)</td><td colspan="1" rowspan="1">Site 5(N=75)n (%)</td><td colspan="1" rowspan="1">Site 6(N=44)n (%)</td></tr><tr><td colspan="1" rowspan="1">Candida albicans</td><td colspan="1" rowspan="1">4(19.0)</td><td colspan="1" rowspan="1">1(100)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">2 (100)</td><td colspan="1" rowspan="1">1 (25.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td></tr><tr><td colspan="1" rowspan="1">Candida auris</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td></tr><tr><td colspan="1" rowspan="1">Candida dubliniensis</td><td colspan="1" rowspan="1">1(4.8)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">1 (25.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td></tr><tr><td colspan="1" rowspan="1">Candida famata</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td></tr><tr><td colspan="1" rowspan="1">Candida glabrata</td><td colspan="1" rowspan="1">6 (28.6)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">2 (25.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">1 (25.0)</td><td colspan="1" rowspan="1">2 (50.0)</td><td colspan="1" rowspan="1">1 (50.0)</td></tr><tr><td colspan="1" rowspan="1">Candida guilliermondii</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td></tr><tr><td colspan="1" rowspan="1">Candida kefyr</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td></tr><tr><td colspan="1" rowspan="1">Target</td><td colspan="1" rowspan="1">All Sites(N=445)n (%)</td><td colspan="1" rowspan="1">Site 1(N=25)n (%)</td><td colspan="1" rowspan="1">Site 12(N=162)n (%)</td><td colspan="1" rowspan="1">Site 3(N=74)n (%)</td><td colspan="1" rowspan="1">Site 4(N=65)n(%)</td><td colspan="1" rowspan="1">Site 5(N=75)n (%)</td><td colspan="1" rowspan="1">Site 6(N=44)n (%)</td></tr><tr><td colspan="1" rowspan="1">Candida krusei</td><td colspan="1" rowspan="1">2 (9.5)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">2 (25.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td></tr><tr><td colspan="1" rowspan="1">Candida lusitaniae</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td></tr><tr><td colspan="1" rowspan="1">Candida parapsilosis</td><td colspan="1" rowspan="1">2 (9.5)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">1 (25.0)</td><td colspan="1" rowspan="1">1 (50.0)</td></tr><tr><td colspan="1" rowspan="1">Candida tropicalis</td><td colspan="1" rowspan="1">2 (9.5)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">2 (25.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td></tr><tr><td colspan="1" rowspan="1">Cryptococcus gattii</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td></tr><tr><td colspan="1" rowspan="1">Cryptococcus neoformans</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td></tr><tr><td colspan="1" rowspan="1">Fusarium</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td></tr><tr><td colspan="1" rowspan="1">Rhodotorula</td><td colspan="1" rowspan="1">1 (4.8)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">1 (25.0)</td><td colspan="1" rowspan="1">0 (0.0)</td></tr></table>

N. Instrument Name: GenMark ePlex Instrument

# O. System Descriptions:

1. Modes of Operation:

After samples have been loaded in the ePlex BCID-FP Panel cartridge, the remaining processing steps are executed under control of the ePlex instrument.

2. Software

FDA has reviewed applicant’s Hazard Analysis and software development processes for this line of product types:

Yes _X___ or No

3. Specimen Identification: The Sample ID can be entered manually or scanned via barcode.

4. Specimen Sampling and Handling: N/A

6. Quality Control: See Section M (1d) above

P. Other Supportive Instrument Performance Characteristics Data Not Covered In The “Performance Characteristics” Section above:

# Q. Proposed Labeling:

The labeling supports the finding of substantial equivalence for this device.

# R. Patient Perspectives:

This submission did not include specific information on patient perspectives for this device.

# S. Conclusion:

The submitted information in this premarket notification is complete and supports a substantial equivalence decision.