# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATION DECISION SUMMARY

A. 510(k) Number: K183597

# B. Purpose for Submission:

To establish substantial equivalence to a predicate device and to obtain clearance for a new assay: the QIAstat-Dx Respiratory Panel.

# C. Measurands:

Influenza A Matrix gene (M)   
Influenza A H1 Hemagglutinin gene (HA)   
Influenza A H3 Hemagglutinin gene (HA)   
Influenza A H1 pdm09 Neuraminidase gene (NA)   
Influenza B Nucleoprotein gene (NP)   
Coronavirus 229E Membrane protein gene (M)   
Coronavirus OC43 Nucleocapsid protein gene (N)   
Coronavirus NL63 Nucleocapsid protein gene (N)   
Coronavirus HKU1 Nucleocapsid protein gene (N)   
Parainfluenza virus 1 Hemagglutinin-neuraminidase glycoprotein gene (HN)   
Parainfluenza virus 2 Hemagglutinin-neuraminidase glycoprotein gene (HN)   
Parainfluenza virus 3 Phosphoprotein gene (P)   
Parainfluenza virus 4 Nucleocapsid protein gene (N)   
Rhinovirus/Enterovirus $5 '$ -UTR region   
Adenovirus Hexon gene   
Respiratory Syncytial Virus A/B Matrix protein gene (M)   
Human metapneumovirus A/B Nucleoprotein gene (N)   
Legionella pneumophila Macrophage infectivity potentiator gene (MIP)   
Chlamydophila pneumoniae Major outer membrane protein gene (ompA)   
Mycoplasma pneumoniae Cytadhesin gene (P1)   
Bordetella pertussis Transposase Insertion element (IS481)

# D. Type of Test:

Multiplexed Real-Time reverse transcription nucleic acid amplification assay for the amplification and detection of multiple respiratory pathogen nucleic acids.

# E. Applicant:

QIAGEN GmbH

# F. Proprietary and Established Names:

QIAstat- $\mathbf { D } \mathbf { x } ^ { \mathbb { \left( \mathbf { R } \right) } }$ Respiratory Panel QIAstat- $\mathbf { D } \mathbf { X } ^ { \mathbf { \otimes } }$ Analyzer

# G. Regulatory Information:

1. Regulation section: 21 CFR 866.3980, Respiratory Viral Panel Multiplex Nucleic Acid Assay

2. Classification: Class II

3. Product code: OCC, OEM, OOU, OEP, OOI, OTG, OZX, OZY, OQW, OZZ

4. Panel: Microbiology (83)

# H. Intended Use:

1. Intended use(s):

The QIAstat-Dx Respiratory Panel is a multiplexed nucleic acid test intended for use with QIAstat-Dx system for the simultaneous qualitative detection and identification of multiple respiratory viral and bacterial nucleic acids in nasopharyngeal swabs (NPS) eluted in Universal Transport Media (UTM) obtained from individuals suspected of respiratory tract infections. The following organism types and subtypes are identified using the QIAstat-Dx Respiratory Panel: Adenovirus, Coronavirus 229E, Coronavirus HKU1, Coronavirus NL63, Coronavirus OC43, Human Metapneumovirus $\mathbf { A } { + } \mathbf { B }$ , Influenza A, Influenza A H1, Influenza A H3, Influenza A H1N1/pdm09, Influenza B, Parainfluenza Virus 1, Parainfluenza Virus 2, Parainfluenza Virus 3, Parainfluenza Virus 4, Rhinovirus/Enterovirus, Respiratory Syncytial Virus $\mathbf { A } { + } \mathbf { B }$ , Bordetella pertussis, Chlamydophila pneumoniae and Mycoplasma pneumoniae.

The detection and identification of specific viral and bacterial nucleic acids from individuals presenting with signs and symptoms of a respiratory infection aids in the diagnosis of respiratory infection if used in conjunction with other clinical and epidemiological information. The results of this test should not be used as the sole basis for diagnosis, treatment or other management decisions. Negative results in the setting of a respiratory illness may be due to infection with pathogens that are not detected by the test or lower respiratory tract infection that is not detected by a nasopharyngeal swab

specimen. Positive results do not rule out co-infection with other organisms: the agent(s) detected by the QIAstat-Dx Respiratory Panel may not be the definite cause of disease. Additional laboratory testing (e.g., bacterial and viral culture, immunofluorescence and radiography) may be necessary when evaluating a patient with possible respiratory tract infection.

Due to the small number of positive specimens collected for certain organisms during the prospective clinical study, performance characteristics for Bordetella pertussis and Parainfluenza Virus 1 were established primarily with retrospective clinical specimens. Performance characteristics for Chlamydophila pneumoniae, Parainfluenza Virus 2, Parainfluenza Virus 4, Influenza A subtype H1 and Coronavirus 229E were established primarily using contrived clinical specimens.

Due to the genetic similarity between Human Rhinovirus and Enterovirus, the QIAstatDx Respiratory Panel cannot reliably differentiate them. A positive QIAstat-Dx Respiratory Panel Rhinovirus/Enterovirus result should be followed-up using an alternate method (e.g., cell culture or sequence analysis).

Performance characteristics for Influenza A were established when Influenza A H1N1- 2009 and A H3 were the predominant Influenza A viruses in circulation. Performance of detecting Influenza A may vary if other Influenza A strains are circulating or a novel Influenza A virus emerges. If infection with a novel Influenza A virus is suspected based on current clinical and epidemiological screening criteria recommended by public health authorities, specimens should be collected with appropriate infection control precautions for novel virulent Influenza viruses and sent to state or local health departments for testing. Viral culture should not be attempted in these cases unless a BSL $^ { 3 + }$ facility is available to receive and culture specimens.

2. Indication(s) for use: Same as Intended Use

3. Special conditions for use statement(s): For Prescription Use

4. Special instrument requirements: To be used only with the QIAstat-Dx Analyzer

# I. Device Description:

# QIAstat-Dx Respiratory Panel Reagent Kit

Each QIAstat-Dx Respiratory Panel reagent kit contains six (6) QIAstat-Dx Cartridges in individually wrapped foil pouches and six (6) individually packaged transfer pipettes for dispensing liquid sample into the QIAstat-Dx Cartridge.

# QIAstat-Dx Respiratory Panel Cartridge

The QIAstat-Dx Respiratory Panel cartridge is a disposable plastic device that allows the RP assay to be performed on the QIAstat-Dx Analyzer. The cartridge includes all the necessary reagents to perform extraction, amplification, and detection of target nucleic acids from the eluted NPS specimen. All sample preparation and assay steps are performed within the cartridge.

# QIAstat-Dx Analyzer

The QIAstat-Dx Analyzer 1.0 is the unit that hosts a cartridge and, on command from the user, is able to run predefined assay protocols. Within the cartridge, multiple steps are automatically performed in sequence by using pneumatic pressure and a multiport valve to transfer sample and fluids via the Transfer Chamber to their intended destinations. Following the introduction of the sample from a disposable transfer pipette, the following assay steps occur automatically and sequentially:

Resuspension of internal control and Proteinase K enzyme; Cell lysis using mechanical (rotation) and chemical (chaotropic and isotonic) means; Mixing of the purified nucleic acid with lyophilized “Master Mix” reagents; Sequential transfer of mixed eluate/Master Mix from the Transfer Chamber to each Reaction Chamber containing the specified, air dried primers and probes;   
• Within each Reaction Chamber, real-time, multiplex PCR (“rt-PCR”) testing is performed. Increase in fluorescence (indicative of detection of each target analyte) is detected directly within each Reaction Chamber;   
• The detected signal per fluorescent marker per Reaction Chamber is then used by the system software to generate the assay result.

# Materials Required but not Provided

QIAstat-Dx Analyzer Instrument • QIAstat-Dx Analyzer User Manual

# Interpretation of Results

The QIAstat-Dx Analyzer automatically interprets and saves test results. After ejecting the cartridge, the results summary screen is automatically displayed. Detected analytes (i.e.; positive results) are displayed at the top of the list under the category ‘Detected’ in red font with a plus sign $( + )$ next to the name. Equivocal results are next listed in yellow font with a question mark (?) next to the name. The last section of the results screen shows all targets tested with either a plus sign if it was detected, a question mark if the result was equivocal, and a minus sign (-) with the name in green colored font if the analyte was tested but not detected.

# Quality Control

The QIAstat-Dx Respiratory Panel Cartridge includes Internal Control (IC), a titered lyophilized MS2 bacteriophage that provides verification that all steps of the analysis process including sample resuspension, lysis, nucleic acid purification, reverse transcription, and PCR were successful. The results screen displays a message indicating that the internal controls “Passed” when the test was run successfully. An IC message of “Failed” indicates that the internal control was not amplified; ‘Positive’ test results are still reported as positive, but all ‘Negative’ results are invalid. Positive and negative external controls are recommended by the manufacturer but are not provided.

# J. Substantial Equivalence Information:

1. Predicate device name(s): FilmArray Respiratory Panel (RP) 2. Predicate 510(k) number(s): K123620 3. Comparison with predicate:

Table 1: QIAstat-Dx Respiratory Panel – Comparison With Predicate   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>QIAstat-Dx Respiratory Panel</td><td rowspan=1 colspan=1>FilmArray Respiratory Panel</td></tr><tr><td rowspan=1 colspan=1>510(k) Number</td><td rowspan=1 colspan=1>K183597</td><td rowspan=1 colspan=1>K123620</td></tr><tr><td rowspan=1 colspan=1>Assay Targets</td><td rowspan=1 colspan=1>17 Respiratory virus targets plus 3atypical bacteria</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=1>Product Code</td><td rowspan=1 colspan=1>OCC</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=1>DeviceTechnology</td><td rowspan=1 colspan=1>Multiplex real time PCR</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=1>ResultsInterpretation</td><td rowspan=1 colspan=1>Automated</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=1>Time to Result</td><td rowspan=1 colspan=1>approximately 1 hour</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=1>SpecimenTypes</td><td rowspan=1 colspan=1>Nasopharyngeal swab (NPS) eluted inUTM</td><td rowspan=1 colspan=1>NPS</td></tr><tr><td rowspan=1 colspan=1>Instrument</td><td rowspan=1 colspan=1>QIAstat-Dx Analyzer</td><td rowspan=1 colspan=1>FilmArray Instrument</td></tr><tr><td rowspan=1 colspan=1>Intended Use</td><td rowspan=1 colspan=1>The QIAstat-Dx Respiratory Panel is amultiplexed nucleic acid test intendedfor use with QIAstat-Dx system forthe simultaneous qualitative detectionand identification of multiplerespiratory viral and bacterial nucleicacids in nasopharyngeal swabs (NPS)eluted in universal transport media(UTM) obtained from individualssuspected of respiratory tractinfections. The following organismtypes and subtypes are identified usingthe QIAstat-Dx Respiratory Panel:</td><td rowspan=1 colspan=1>FilmArray Respiratory Panel (RP) is amultiplexed nucleic acid test intendedfor use with the FilmArray Instrumentfor the simultaneous qualitativedetection and identification of multiplerespiratory viral and bacterial nucleicacids in nasopharyngeal swabs (NPS)obtained from individuals suspected ofrespiratory tract infections. Thefollowing organism types and subtypesare identified using the FilmArray RP:Adenovirus, Coronavirus 229E,Coronavirus HKU1, Coronavirus</td></tr></table>

<table><tr><td>Adenovirus, Coronavirus 229E, Coronavirus HKU1, Coronavirus NL63, Coronavirus OC43, Human Metapneumovirus A+B, Influenza A, Influenza A H1, Influenza A H3, Influenza A H1N1/pdm09, Influenza B, Parainfluenza Virus 1, Parainfluenza Virus 2, Parainfluenza Virus 3, Parainfluenza Virus 4, Rhinovirus/Enterovirus, Respiratory Syncytial Virus A+B, Bordetella pertussis, Chlamydophila pneumoniae and Mycoplasma pneumoniae. The detection and identification of specific viral and bacterial nucleic acids from individuals presenting with signs and symptoms of a respiratory infection aids in the diagnosis of respiratory infection if used in conjunction with other clinical and epidemiological information. The results of this test should not be used as the sole basis for diagnosis, treatment or other management decisions. Negative results in the setting of a respiratory illness may be due to infection with pathogens that are not detected by the test or lower respiratory tract infection that is not detected by a nasopharyngeal swab specimen. Positive results do not rule out co-infection with other organisms: the agent(s) detected by the QIAstat- Dx Respiratory Panel may not be the definite cause of disease. Additional laboratory testing (e.g., bacterial and viral culture, immunofluorescence and radiography) may be necessary when evaluating a patient with possible respiratory tract infection. Due to the small number of positive specimens collected for certain organisms during the prospective clinical study, performance characteristics for Bordetella pertussis and Parainfluenza Virus 1 were</td><td>Metapneumovirus, Influenza A, Influenza A subtype H1, Influenza A subtype H3, Influenza A subtype H1- 2009, Influenza B, Parainfluenza Virus 1, Parainfluenza Virus 2, Parainfluenza Virus 3, Parainfluenza Virus 4, Human Rhinovirus/Enterovirus, Respiratory Syncytial Virus, Bordetella pertussis, Chlamydophila pneumoniae, and Mycoplasma pneumoniae. The detection and identification of specific viral and bacterial nucleic acids from individuals exhibiting signs and symptoms of a respiratory infection aids in the diagnosis of respiratory infection if used in conjunction with other clinical and epidemiological information. The results of this test should not be used as the sole basis for diagnosis, treatment, or other management decisions. Negative results in the setting of a respiratory illness may be due to infection with pathogens that are not detected by this test or, lower respiratory tract infection that is not detected by a nasopharyngeal swab specimen. Positive results do not rule out coinfection with other organisms: the agent(s) detected by the Film Array RP may not be the definite cause of disease. Additional laboratory testing (e.g. bacterial and viral culture, immunofluorescence, and radiography) may be necessary when evaluating a patient with possible respiratory tract infection. Due to the small number of positive specimens collected for certain organisms during the prospective clinical study, performance characteristics for Bordetella pertussis, Coronavirus 229E, Coronavirus OC43, Influenza A H1, Influenza A H3, Influenza A H1-2009, Influenza B,</td></tr><tr><td colspan="1" rowspan="1">established primarily withretrospective clinical specimens.Performance characteristics forChlamydophila pneumoniae,Parainfluenza Virus 2, Parainfluenza</td><td colspan="3" rowspan="3">Mycoplasma pneumoniae,Parainfluenza Virus 1, ParainfluenzaVirus 2, and Parainfluenza Virus 4were established primarily withretrospective clinical specimens.Performance characteristics forChlamydophila pneumoniae wereestablished primarily using contrivedclinical specimens.Due to the genetic similarity betweenHuman Rhinovirus and Enterovirus, theFilmArray RP cannot reliablydifferentiate them. A positiveFilmArray RP Rhinovirus/Enterovirusresult should be followed-up using analternate method (e.g., cell culture orsequence analysis).The FilmArray RP assay forCoronavirus OC43 may cross-reactwith some isolates of CoronavirusHKU1. A dual positive result may bedue to cross-reactivity or may indicatea co-infection.Performance characteristics forInfluenza A were established whenInfluenza A H1-2009, A H1, and A H3were the predominant Influenza Aviruses in circulation. Performance ofdetecting Influenza A may vary if otherInfluenza A strains are circulating or anovel Influenza A virus emerges. Ifinfection with a novel Influenza Avirus is suspected based on currentclinical and epidemiological screeningcriteria recommended by public healthauthorities, specimens should becollected with appropriate infectioncontrol precautions for novel virulentInfluenza viruses and sent to state orlocal health departments for testing.Viral culture should not be attempted inthese cases unless a BSL 3+ facility isavailable to receive and culturespecimens.</td></tr><tr><td colspan="1" rowspan="2">Virus 4, Influenza A subtype H1 andCoronavirus 229E were establishedprimarily using contrived clinicalspecimens.Due to the genetic similarity betweenHuman Rhinovirus and Enterovirus,the QIAstat-Dx Respiratory Panelcannot reliably differentiate them. Apositive QIAstat-Dx Respiratory PanelRhinovirus/Enterovirus result shouldbe followed-up using an alternatemethod (e.g., cell culture or sequenceanalysis).Performance characteristics forInfluenza A were established whenInfluenza A H1N1-2009 and A H3were the predominant Influenza Aviruses in circulation. Performance ofdetecting Influenza A may vary ifother Influenza A strains arecirculating or a novel Influenza Avirus emerges. If infection with anovel Influenza A virus is suspectedbased on current clinical andepidemiological screening criteriarecommended by public healthauthorities, specimens should becollected with appropriate infectioncontrol precautions for novel virulentInfluenza viruses and sent to state orlocal health departments for testing.Viral culture should not be attemptedin these cases unless a BSL 3+ facilityis available to receive and culturespecimens.</td></tr><tr><td colspan="1" rowspan="1">haract</td></tr></table>

# K. Standard/Guidance Documents Referenced (if applicable):

Safety requirements for electrical equipment for measurement, control, and laboratory use – Part 1: General requirements. IEC 61010-1:2010.

Safety requirements for electrical equipment for measurement, control, and laboratory use – Part 2-101: Particular requirements for in vitro diagnostic (IVD) medical equipment. IEC 61010-2-2015.

Electrical equipment for measurement, control and laboratory use – EMC requirements – Part 1: General requirements. IEC 61326-1:2013

Electrical equipment for measurement, control and laboratory use. EMC requirements. Particular requirements. In vitro diagnostic (IVD) medical equipment. IEC 61326-2-6:2012

# L. Test Principle:

Multiplexed reverse transcription nucleic acid amplification

M. Performance Characteristics (if/when applicable):

1. Analytical performance:

a. Precision/Reproducibility:

Reproducibility Study

A reproducibility study of QIAstat-Dx Respiratory Panel was conducted by operators at three sites using panels of blinded coded specimens containing high negative, low positive, and moderate positive mixed analyte samples. A total of twelve sample mixes were prepared for the study. Nine operators from three sites (five operators from site 1, two operators each from sites 2 and 3) participated in the study. The study was conducted over five days testing four replicates per day. Samples were prepared by spiking individual pathogens into HeLa cells in UTM to final concentrations of 0.1x, 1x, or $3 \mathbf { x }$ LOD.

The percent agreement with expected results for all analytes was $2 9 5 \%$ for samples tested at 1x and $3 \mathbf { x }$ LOD. All of the sample mixes generated negative test results for analytes not included in the specific mix tested. There were no significant differences observed within run (replicates tested by one operator), between runs (five different days), between sites (three sites), or between operators (nine operators).

The Reproducibility Study site-to-site qualitative results (agreements with expected results) are presented in Tables 2, 3, and 4 below.

Table 2: QIAstat-Dx Reproducibility with Samples at 0.1x LOD   

<table><tr><td rowspan=1 colspan=2>Analyte(QIAstat-Dx Target)</td><td rowspan=1 colspan=1>Site</td><td rowspan=1 colspan=1>PositiveDetected</td><td rowspan=1 colspan=1>Percent Agreementwith ExpectedResults</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=4 colspan=2>Adenovirus(Adenovirus)</td><td rowspan=1 colspan=1>Site 1</td><td rowspan=1 colspan=1>10/20</td><td rowspan=1 colspan=1>50.0%</td><td rowspan=1 colspan=1>29.9-70.1%</td></tr><tr><td rowspan=1 colspan=1>Site 2</td><td rowspan=1 colspan=1>9/19</td><td rowspan=1 colspan=1>47.4%</td><td rowspan=1 colspan=1>27.3-68.3%</td></tr><tr><td rowspan=1 colspan=1>Site 3</td><td rowspan=1 colspan=1>10/19</td><td rowspan=1 colspan=1>52.6%</td><td rowspan=1 colspan=1>31.7-72.7%</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>29/58</td><td rowspan=1 colspan=1>50.0%</td><td rowspan=1 colspan=1>37.5-62.5%</td></tr><tr><td rowspan=4 colspan=2>B. pertussis(B. pertussis)</td><td rowspan=1 colspan=1>Site 1</td><td rowspan=1 colspan=1>9/20</td><td rowspan=1 colspan=1>45.0%</td><td rowspan=1 colspan=1>25.8-65.8%</td></tr><tr><td rowspan=1 colspan=1>Site 2</td><td rowspan=1 colspan=1>7/19</td><td rowspan=1 colspan=1>36.8%</td><td rowspan=1 colspan=1>19.1-59.0%</td></tr><tr><td rowspan=1 colspan=1>Site 3</td><td rowspan=1 colspan=1>9/20</td><td rowspan=1 colspan=1>45.0%</td><td rowspan=1 colspan=1>25.8-65.8%</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>25/59</td><td rowspan=1 colspan=1>42.4%</td><td rowspan=1 colspan=1>30.6-55.1%</td></tr><tr><td rowspan=4 colspan=2>C. pneumoniae(C. pneumoniae)</td><td rowspan=1 colspan=1>Site 1</td><td rowspan=1 colspan=1>11/20</td><td rowspan=1 colspan=1>55.0%</td><td rowspan=1 colspan=1>34.2-74.2%</td></tr><tr><td rowspan=1 colspan=1>Site 2</td><td rowspan=1 colspan=1>11/19</td><td rowspan=1 colspan=1>57.9%</td><td rowspan=1 colspan=1>36.3-76.9%</td></tr><tr><td rowspan=1 colspan=1>Site 3</td><td rowspan=1 colspan=1>14/20</td><td rowspan=1 colspan=1>70.0%</td><td rowspan=1 colspan=1>48.1-85.5%</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>36/59</td><td rowspan=1 colspan=1>61.0%</td><td rowspan=1 colspan=1>48.3-72.4%</td></tr><tr><td rowspan=4 colspan=2>Coronavirus 229E(Coronavirus 229E)</td><td rowspan=1 colspan=1>Site 1</td><td rowspan=1 colspan=1>9/20</td><td rowspan=1 colspan=1>45.0%</td><td rowspan=1 colspan=1>25.8-65.8%</td></tr><tr><td rowspan=1 colspan=1>Site 2</td><td rowspan=1 colspan=1>12/19</td><td rowspan=1 colspan=1>63.2%</td><td rowspan=1 colspan=1>41.0-80.9%</td></tr><tr><td rowspan=1 colspan=1>Site 3</td><td rowspan=1 colspan=1>5/20</td><td rowspan=1 colspan=1>25.0%</td><td rowspan=1 colspan=1>11.2-46.9%</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>26/59</td><td rowspan=1 colspan=1>44.1%</td><td rowspan=1 colspan=1>32.2-56.7%</td></tr><tr><td rowspan=4 colspan=2>Coronavirus HKU1(Coronavirus HKU1)</td><td rowspan=1 colspan=1>Site 1</td><td rowspan=1 colspan=1>17/20</td><td rowspan=1 colspan=1>85.0%</td><td rowspan=1 colspan=1>64.0-94.8%</td></tr><tr><td rowspan=1 colspan=1>Site 2</td><td rowspan=1 colspan=1>10/19</td><td rowspan=1 colspan=1>52.6%</td><td rowspan=1 colspan=1>31.7-72.7%</td></tr><tr><td rowspan=1 colspan=1>Site 3</td><td rowspan=1 colspan=1>9/20</td><td rowspan=1 colspan=1>45.0%</td><td rowspan=1 colspan=1>25.8-65.8%</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>36/59</td><td rowspan=1 colspan=1>61.0%</td><td rowspan=1 colspan=1>48.3-72.4%</td></tr><tr><td rowspan=1 colspan=2>Coronavirus NL63</td><td rowspan=1 colspan=1>Site 1</td><td rowspan=1 colspan=1>13/20</td><td rowspan=1 colspan=1>65.0%</td><td rowspan=1 colspan=1>43.3-81.9%</td></tr><tr><td rowspan=3 colspan=2>(Coronavirus NL63)</td><td rowspan=1 colspan=1>63)</td><td rowspan=1 colspan=1>Site 2</td><td rowspan=1 colspan=1>12/19</td><td rowspan=1 colspan=1>63.2%</td></tr><tr><td rowspan=1 colspan=1>Site 3</td><td rowspan=1 colspan=1>14/19</td><td rowspan=1 colspan=1>73.7%</td><td rowspan=1 colspan=1>51.2-88.2%</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>39/58</td><td rowspan=1 colspan=1>67.2%</td><td rowspan=1 colspan=1>54.4-77.9%</td></tr><tr><td rowspan=4 colspan=2>Coronavirus OC43(Coronavirus OC43)</td><td rowspan=1 colspan=1>Site 1</td><td rowspan=1 colspan=1>13/20</td><td rowspan=1 colspan=1>65.0%</td><td rowspan=1 colspan=1>43.3-81.9%</td></tr><tr><td rowspan=1 colspan=1>Site 2</td><td rowspan=1 colspan=1>15/20</td><td rowspan=1 colspan=1>75.0%</td><td rowspan=1 colspan=1>53.1-88.8%</td></tr><tr><td rowspan=1 colspan=1>Site 3</td><td rowspan=1 colspan=1>15/20</td><td rowspan=1 colspan=1>75.0%</td><td rowspan=1 colspan=1>53.1-88.8%</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>43/60</td><td rowspan=1 colspan=1>71.7%</td><td rowspan=1 colspan=1>59.2-81.5%</td></tr><tr><td rowspan=4 colspan=2>Enterovirus(Rhinovirus /Enterovirus)</td><td rowspan=1 colspan=1>Site 1</td><td rowspan=1 colspan=1>8/20</td><td rowspan=1 colspan=1>40.0%</td><td rowspan=1 colspan=1>21.9-61.3%</td></tr><tr><td rowspan=1 colspan=1>Site 2</td><td rowspan=1 colspan=1>6/19</td><td rowspan=1 colspan=1>31.6%</td><td rowspan=1 colspan=1>15.4-54.0%</td></tr><tr><td rowspan=1 colspan=1>Site 3</td><td rowspan=1 colspan=1>7/20</td><td rowspan=1 colspan=1>35.0%</td><td rowspan=1 colspan=1>18.1-56.7%</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>21/59</td><td rowspan=1 colspan=1>35.6%</td><td rowspan=1 colspan=1>24.6-48.3%</td></tr></table>

<table><tr><td rowspan=4 colspan=2>HumanMetapneumovirus(hMPV)</td><td rowspan=1 colspan=1>Site 1</td><td rowspan=1 colspan=1>6/20</td><td rowspan=1 colspan=1>30.0%</td><td rowspan=1 colspan=1>14.5-51.9%</td></tr><tr><td rowspan=1 colspan=1>Site 2</td><td rowspan=1 colspan=1>9/19</td><td rowspan=1 colspan=1>47.4%</td><td rowspan=1 colspan=1>27.3-68.3%</td></tr><tr><td rowspan=1 colspan=1>Site 3</td><td rowspan=1 colspan=1>9/20</td><td rowspan=1 colspan=1>45.0%</td><td rowspan=1 colspan=1>25.8-65.8%</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>24/59</td><td rowspan=1 colspan=1>40.7%</td><td rowspan=1 colspan=1>29.1-53.4%</td></tr><tr><td rowspan=4 colspan=2>InfluenzaA/SwineNY/03/ 2009(Influenza A)</td><td rowspan=1 colspan=1>Site 1</td><td rowspan=1 colspan=1>19/20</td><td rowspan=1 colspan=1>95.0%</td><td rowspan=1 colspan=1>76.4-99.1%</td></tr><tr><td rowspan=1 colspan=1>Site 2</td><td rowspan=1 colspan=1>18/20</td><td rowspan=1 colspan=1>90.0%</td><td rowspan=1 colspan=1>69.9-97.2%</td></tr><tr><td rowspan=1 colspan=1>Site 3</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>83.9-100%</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>57/60</td><td rowspan=1 colspan=1>95.0%</td><td rowspan=1 colspan=1>86.3-98.3%</td></tr><tr><td rowspan=4 colspan=2>Influenza A/PortChalmers/1/73(Influenza A)</td><td rowspan=1 colspan=1>Site 1</td><td rowspan=1 colspan=1>10/20</td><td rowspan=1 colspan=1>50.0%</td><td rowspan=1 colspan=1>29.9-70.1%</td></tr><tr><td rowspan=1 colspan=1>Site 2</td><td rowspan=1 colspan=1>9/19</td><td rowspan=1 colspan=1>47.4%</td><td rowspan=1 colspan=1>27.3-68.3%</td></tr><tr><td rowspan=1 colspan=1>Site 3</td><td rowspan=1 colspan=1>16/19</td><td rowspan=1 colspan=1>84.2%</td><td rowspan=1 colspan=1>62.4-94.5%</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>35/58</td><td rowspan=1 colspan=1>60.3%</td><td rowspan=1 colspan=1>47.5-71.9%</td></tr><tr><td rowspan=4 colspan=2>Influenza A/NJ/8/76(Influenza A)</td><td rowspan=1 colspan=1>Site 1</td><td rowspan=1 colspan=1>14/20</td><td rowspan=1 colspan=1>70.0%</td><td rowspan=1 colspan=1>48.1-85.5%</td></tr><tr><td rowspan=1 colspan=1>Site 2</td><td rowspan=1 colspan=1>9/19</td><td rowspan=1 colspan=1>47.4%</td><td rowspan=1 colspan=1>27.3-68.3%</td></tr><tr><td rowspan=1 colspan=1>Site 3</td><td rowspan=1 colspan=1>12/20</td><td rowspan=1 colspan=1>60.0%</td><td rowspan=1 colspan=1>38.7-78.1%</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>35/59</td><td rowspan=1 colspan=1>59.3%</td><td rowspan=1 colspan=1>46.6-70.9%</td></tr><tr><td rowspan=4 colspan=2>Influenza A/NJ/8/76(Influenza A H1)</td><td rowspan=1 colspan=1>Site 1</td><td rowspan=1 colspan=1>13/20</td><td rowspan=1 colspan=1>65.0%</td><td rowspan=1 colspan=1>43.3-81.9%</td></tr><tr><td rowspan=1 colspan=1>Site 2</td><td rowspan=1 colspan=1>13/19</td><td rowspan=1 colspan=1>68.4%</td><td rowspan=1 colspan=1>46.0-84.6%</td></tr><tr><td rowspan=1 colspan=1>Site 3</td><td rowspan=1 colspan=1>15/20</td><td rowspan=1 colspan=1>75.0%</td><td rowspan=1 colspan=1>53.1-88.8%</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>41/59</td><td rowspan=1 colspan=1>69.5%</td><td rowspan=1 colspan=1>56.9-79.7%</td></tr><tr><td rowspan=4 colspan=2>InfluenzaB/Taiwan/2/62(Influenza B)</td><td rowspan=1 colspan=1>Site 1</td><td rowspan=1 colspan=1>7/20</td><td rowspan=1 colspan=1>35.0%</td><td rowspan=1 colspan=1>18.1-56.7%</td></tr><tr><td rowspan=1 colspan=1>Site 2</td><td rowspan=1 colspan=1>9/19</td><td rowspan=1 colspan=1>47.4%</td><td rowspan=1 colspan=1>27.3-68.3%</td></tr><tr><td rowspan=1 colspan=1>Site 3</td><td rowspan=1 colspan=1>8/20</td><td rowspan=1 colspan=1>40.0%</td><td rowspan=1 colspan=1>21.9-61.3%</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>24/59</td><td rowspan=1 colspan=1>40.7%</td><td rowspan=1 colspan=1>29.1-53.4%</td></tr><tr><td rowspan=4 colspan=2>InfluenzaA/SwineNY/03/2009(Influenza A H1N1pdm09)</td><td rowspan=1 colspan=1>Site 1</td><td rowspan=1 colspan=1>14/20</td><td rowspan=1 colspan=1>70.0%</td><td rowspan=1 colspan=1>48.1-85.5%</td></tr><tr><td rowspan=1 colspan=1>Site 2</td><td rowspan=1 colspan=1>16/20</td><td rowspan=1 colspan=1>80.0%</td><td rowspan=1 colspan=1>58.4-91.9%</td></tr><tr><td rowspan=1 colspan=1>Site 3</td><td rowspan=1 colspan=1>15/20</td><td rowspan=1 colspan=1>75.0%</td><td rowspan=1 colspan=1>53.1-88.8%</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>45/60</td><td rowspan=1 colspan=1>75.0%</td><td rowspan=1 colspan=1>62.8-84.2%</td></tr><tr><td rowspan=4 colspan=2>Influenza A/PortChalmers/1/73(Influenza A H3)</td><td rowspan=1 colspan=1>Site 1</td><td rowspan=1 colspan=1>13/20</td><td rowspan=1 colspan=1>65.0%</td><td rowspan=1 colspan=1>43.3-81.9%</td></tr><tr><td rowspan=1 colspan=1>Site 2</td><td rowspan=1 colspan=1>16/19</td><td rowspan=1 colspan=1>84.2%</td><td rowspan=1 colspan=1>62.4-94.5%</td></tr><tr><td rowspan=1 colspan=1>Site 3</td><td rowspan=1 colspan=1>17/19</td><td rowspan=1 colspan=1>89.5%</td><td rowspan=1 colspan=1>68.6-97.1%</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>46/58</td><td rowspan=1 colspan=1>79.3%</td><td rowspan=1 colspan=1>67.2-87.7%</td></tr><tr><td rowspan=4 colspan=2>Mycoplasmapneumoniae(M. pneumoniae)</td><td rowspan=1 colspan=1>Site 1</td><td rowspan=1 colspan=1>13/20</td><td rowspan=1 colspan=1>65.0%</td><td rowspan=1 colspan=1>43.3-81.9%</td></tr><tr><td rowspan=1 colspan=1>Site 2</td><td rowspan=1 colspan=1>14/20</td><td rowspan=1 colspan=1>70.0%</td><td rowspan=1 colspan=1>48.1-85.5%</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Site 3</td><td rowspan=1 colspan=1>14/20</td><td rowspan=1 colspan=1>70.0%</td><td rowspan=1 colspan=1>48.1-85.5%</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>41/60</td><td rowspan=1 colspan=1>68.3%</td><td rowspan=1 colspan=1>55.8-78.7%</td></tr></table>

<table><tr><td rowspan=4 colspan=1>Parainfluenza virus 1(PIV 1)</td><td rowspan=1 colspan=1>Site 1</td><td rowspan=1 colspan=1>14/20</td><td rowspan=1 colspan=1>70.0%</td><td rowspan=1 colspan=1>48.1-85.5%</td></tr><tr><td rowspan=1 colspan=1>Site 2</td><td rowspan=1 colspan=1>12/19</td><td rowspan=1 colspan=1>63.2%</td><td rowspan=1 colspan=1>41.0-80.9%</td></tr><tr><td rowspan=1 colspan=1>Site 3</td><td rowspan=1 colspan=1>9/19</td><td rowspan=1 colspan=1>47.4%</td><td rowspan=1 colspan=1>27.3-68.3%</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>35/58</td><td rowspan=1 colspan=1>60.3%</td><td rowspan=1 colspan=1>47.5-71.9%</td></tr><tr><td rowspan=4 colspan=1>Parainfluenza virus 2(PIV 2)</td><td rowspan=1 colspan=1>Site 1</td><td rowspan=1 colspan=1>9/20</td><td rowspan=1 colspan=1>45.0%</td><td rowspan=1 colspan=1>25.8-65.8%</td></tr><tr><td rowspan=1 colspan=1>Site 2</td><td rowspan=1 colspan=1>11/19</td><td rowspan=1 colspan=1>57.9%</td><td rowspan=1 colspan=1>36.3-76.9%</td></tr><tr><td rowspan=1 colspan=1>Site 3</td><td rowspan=1 colspan=1>12/20</td><td rowspan=1 colspan=1>60.0%</td><td rowspan=1 colspan=1>38.7-78.1%</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>32/59</td><td rowspan=1 colspan=1>54.2%</td><td rowspan=1 colspan=1>41.7-66.3%</td></tr><tr><td rowspan=4 colspan=1>Parainfluenza virus 3(PIV 3)</td><td rowspan=1 colspan=1>Site 1</td><td rowspan=1 colspan=1>13/20</td><td rowspan=1 colspan=1>65.0%</td><td rowspan=1 colspan=1>43.3-81.9%</td></tr><tr><td rowspan=1 colspan=1>Site 2</td><td rowspan=1 colspan=1>17/20</td><td rowspan=1 colspan=1>85.0%</td><td rowspan=1 colspan=1>64.0-94.8%</td></tr><tr><td rowspan=1 colspan=1>Site 3</td><td rowspan=1 colspan=1>17/20</td><td rowspan=1 colspan=1>85.0%</td><td rowspan=1 colspan=1>64.0-94.8%</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>47/60</td><td rowspan=1 colspan=1>78.3%</td><td rowspan=1 colspan=1>66.4-86.9%</td></tr><tr><td rowspan=4 colspan=1>Parainfluenza virus 4(PIV 4)</td><td rowspan=1 colspan=1>Site 1</td><td rowspan=1 colspan=1>10/20</td><td rowspan=1 colspan=1>50.0%</td><td rowspan=1 colspan=1>29.9-70.1%</td></tr><tr><td rowspan=1 colspan=1>Site 2</td><td rowspan=1 colspan=1>11/19</td><td rowspan=1 colspan=1>57.9%</td><td rowspan=1 colspan=1>36.3-76.9%</td></tr><tr><td rowspan=1 colspan=1>Site 3</td><td rowspan=1 colspan=1>9/20</td><td rowspan=1 colspan=1>45.0%</td><td rowspan=1 colspan=1>25.8-65.8%</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>30/59</td><td rowspan=1 colspan=1>50.9%</td><td rowspan=1 colspan=1>38.4-63.2%</td></tr><tr><td rowspan=4 colspan=1>RSV A(RSV)</td><td rowspan=1 colspan=1>Site 1</td><td rowspan=1 colspan=1>6/20</td><td rowspan=1 colspan=1>30.0%</td><td rowspan=1 colspan=1>14.5-51.9%</td></tr><tr><td rowspan=1 colspan=1>Site 2</td><td rowspan=1 colspan=1>7/20</td><td rowspan=1 colspan=1>35.0%</td><td rowspan=1 colspan=1>18.1-56.7%</td></tr><tr><td rowspan=1 colspan=1>Site 3</td><td rowspan=1 colspan=1>9/20</td><td rowspan=1 colspan=1>45.0%</td><td rowspan=1 colspan=1>25.8-65.8%</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>22/60</td><td rowspan=1 colspan=1>36.7%</td><td rowspan=1 colspan=1>25.6-49.3%</td></tr><tr><td rowspan=4 colspan=1>RSV B(RSV)</td><td rowspan=1 colspan=1>Site 1</td><td rowspan=1 colspan=1>14/20</td><td rowspan=1 colspan=1>70.0%</td><td rowspan=1 colspan=1>48.1-85.5%</td></tr><tr><td rowspan=1 colspan=1>Site 2</td><td rowspan=1 colspan=1>15/19</td><td rowspan=1 colspan=1>79.0%</td><td rowspan=1 colspan=1>56.7-91.5%</td></tr><tr><td rowspan=1 colspan=1>Site 3</td><td rowspan=1 colspan=1>10/20</td><td rowspan=1 colspan=1>50.0%</td><td rowspan=1 colspan=1>29.9-70.1%</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>39/59</td><td rowspan=1 colspan=1>66.1%</td><td rowspan=1 colspan=1>53.4-76.9%</td></tr><tr><td rowspan=4 colspan=1>Rhinovirus(Rhinovirus /Enterovirus)</td><td rowspan=1 colspan=1>Site 1</td><td rowspan=1 colspan=1>15/20</td><td rowspan=1 colspan=1>75.0%</td><td rowspan=1 colspan=1>53.1-88.8%</td></tr><tr><td rowspan=1 colspan=1>Site 2</td><td rowspan=1 colspan=1>15/20</td><td rowspan=1 colspan=1>75.0%</td><td rowspan=1 colspan=1>53.1-88.8%</td></tr><tr><td rowspan=1 colspan=1>Site 3</td><td rowspan=1 colspan=1>18/20</td><td rowspan=1 colspan=1>90.0%</td><td rowspan=1 colspan=1>69.9-97.2%</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>48/60</td><td rowspan=1 colspan=1>80.0%</td><td rowspan=1 colspan=1>68.2-88.2%</td></tr></table>

Table 3: QIAstat-Dx Reproducibility with Samples at 1x LOD   

<table><tr><td colspan="2" rowspan="1">Analyte(QIAstat-Dx Target)</td><td colspan="1" rowspan="1">Site</td><td colspan="1" rowspan="1">PositiveDetected</td><td colspan="1" rowspan="1">Percent Agreementwith ExpectedResults</td><td colspan="1" rowspan="1">95% CI</td></tr><tr><td colspan="2" rowspan="4">Adenovirus(Adenovirus)</td><td colspan="1" rowspan="1">Site 1</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">83.9-100%</td></tr><tr><td colspan="1" rowspan="1">Site 2</td><td colspan="1" rowspan="1">18/18</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">82.4-100%</td></tr><tr><td colspan="1" rowspan="1">Site 3</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">83.9-100%</td></tr><tr><td colspan="1" rowspan="1">All Sites</td><td colspan="1" rowspan="1">58/58</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">93.8-100%</td></tr><tr><td colspan="2" rowspan="4">B. pertussis(B. pertussis)</td><td colspan="1" rowspan="1">Site 1</td><td colspan="1" rowspan="1">18/20</td><td colspan="1" rowspan="1">90%</td><td colspan="1" rowspan="1">69.9-97.2%</td></tr><tr><td colspan="1" rowspan="1">Site 2</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">83.9-100%</td></tr><tr><td colspan="1" rowspan="1">Site 3</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">83.9-100%</td></tr><tr><td colspan="1" rowspan="1">All Sites</td><td colspan="1" rowspan="1">58/60</td><td colspan="1" rowspan="1">96.7%</td><td colspan="1" rowspan="1">88.6-99.1%</td></tr><tr><td colspan="2" rowspan="4">C. pneumoniae(C. pneumoniae)</td><td colspan="1" rowspan="1">Site 1</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">83.9-100%</td></tr><tr><td colspan="1" rowspan="1">Site 2</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">83.9-100%</td></tr><tr><td colspan="1" rowspan="1">Site 3</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">83.9-100%</td></tr><tr><td colspan="1" rowspan="1">All Sites</td><td colspan="1" rowspan="1">60/60</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">94.0-100%</td></tr><tr><td colspan="2" rowspan="4">Coronavirus 229E(Coronavirus 229E)</td><td colspan="1" rowspan="1">Site 1</td><td colspan="1" rowspan="1">18/20</td><td colspan="1" rowspan="1">90%</td><td colspan="1" rowspan="1">69.9-97.2%</td></tr><tr><td colspan="1" rowspan="1">Site 2</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">83.9-100%</td></tr><tr><td colspan="1" rowspan="1">Site 3</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">83.9-100%</td></tr><tr><td colspan="1" rowspan="1">All Sites</td><td colspan="1" rowspan="1">58/60</td><td colspan="1" rowspan="1">96.7%</td><td colspan="1" rowspan="1">88.6-99.1%</td></tr><tr><td colspan="2" rowspan="4">Coronavirus HKU1(Coronavirus HKU1)</td><td colspan="1" rowspan="1">Site 1</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">83.9-100%</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Site 2</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">83.9-100%</td></tr><tr><td colspan="1" rowspan="1">Site 3</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">83.9-100%</td></tr><tr><td colspan="1" rowspan="1">All Sites</td><td colspan="1" rowspan="1">60/60</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">94.0-100%</td></tr><tr><td colspan="2" rowspan="4">Coronavirus NL63(Coronavirus NL63)</td><td colspan="1" rowspan="1">Site 1</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">83.9-100%</td></tr><tr><td colspan="1" rowspan="1">63)</td><td colspan="1" rowspan="1">Site 2</td><td colspan="1" rowspan="1">18/18</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">82.4-100%</td></tr><tr><td colspan="1" rowspan="1">Site 3</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">83.9-100%</td></tr><tr><td colspan="1" rowspan="1">All Sites</td><td colspan="1" rowspan="1">58/58</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">93.8-100%</td></tr><tr><td colspan="2" rowspan="4">Coronavirus OC43(Coronavirus OC43)</td><td colspan="1" rowspan="1">Site 1</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">83.9-100%</td></tr><tr><td colspan="1" rowspan="1">Site 2</td><td colspan="1" rowspan="1">19/19</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">83.2-100%</td></tr><tr><td colspan="1" rowspan="1">Site 3</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">83.9-100%</td></tr><tr><td colspan="1" rowspan="1">All Sites</td><td colspan="1" rowspan="1">59/59</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">93.9-100%</td></tr><tr><td colspan="2" rowspan="4">Enterovirus(Rhinovirus /Enterovirus)</td><td colspan="1" rowspan="1">Site 1</td><td colspan="1" rowspan="1">19/20</td><td colspan="1" rowspan="1">95.0%</td><td colspan="1" rowspan="1">76.4-99.1%</td></tr><tr><td colspan="1" rowspan="1">Site 2</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">83.9-100%</td></tr><tr><td colspan="1" rowspan="1">Site 3</td><td colspan="1" rowspan="1">19/20</td><td colspan="1" rowspan="1">95.0%</td><td colspan="1" rowspan="1">76.4-99.1%</td></tr><tr><td colspan="1" rowspan="1">All Sites</td><td colspan="1" rowspan="1">58/60</td><td colspan="1" rowspan="1">97.5%</td><td colspan="1" rowspan="1">88.6-99.1%</td></tr><tr><td colspan="2" rowspan="4">HumanMetapneumovirus(hMPV)</td><td colspan="1" rowspan="1">Site 1</td><td colspan="1" rowspan="1">19/20</td><td colspan="1" rowspan="1">95.0%</td><td colspan="1" rowspan="1">76.4-99.1%</td></tr><tr><td colspan="1" rowspan="1">Site 2</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">83.9-100%</td></tr><tr><td colspan="1" rowspan="1">Site 3</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">83.9-100%</td></tr><tr><td colspan="1" rowspan="1">All Sites</td><td colspan="1" rowspan="1">59/60</td><td colspan="1" rowspan="1">98.3%</td><td colspan="1" rowspan="1">91.1-99.7%</td></tr><tr><td colspan="2" rowspan="4">InfluenzaA/SwineNY/03/2009(Influenza A)</td><td colspan="1" rowspan="1">Site 1</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">83.9-100%</td></tr><tr><td colspan="1" rowspan="1">Site 2</td><td colspan="1" rowspan="1">19/19</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">83.2-100%</td></tr><tr><td colspan="1" rowspan="1">Site 3</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">83.9-100%</td></tr><tr><td colspan="1" rowspan="1">All Sites</td><td colspan="1" rowspan="1">59/59</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">93.9-100%</td></tr><tr><td colspan="2" rowspan="4">Influenza A/PortChalmers/1/73(Influenza A)</td><td colspan="1" rowspan="1">Site 1</td><td colspan="1" rowspan="1">19/20</td><td colspan="1" rowspan="1">95.0%</td><td colspan="1" rowspan="1">76.4-99.1%</td></tr><tr><td colspan="1" rowspan="1">Site 2</td><td colspan="1" rowspan="1">18/18</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">82.4-100%</td></tr><tr><td colspan="1" rowspan="1">Site 3</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">83.9-100%</td></tr><tr><td colspan="1" rowspan="1">All Sites</td><td colspan="1" rowspan="1">57/58</td><td colspan="1" rowspan="1">98.3%</td><td colspan="1" rowspan="1">90.9-99.7%</td></tr><tr><td colspan="2" rowspan="4">Influenza A/NJ/8/76(Influenza A)</td><td colspan="1" rowspan="1">Site 1</td><td colspan="1" rowspan="1">19/20</td><td colspan="1" rowspan="1">95.0%</td><td colspan="1" rowspan="1">76.4-99.1%</td></tr><tr><td colspan="1" rowspan="1">Site 2</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">83.9-100%</td></tr><tr><td colspan="1" rowspan="1">Site 3</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">83.9-100%</td></tr><tr><td colspan="1" rowspan="1">All Sites</td><td colspan="1" rowspan="1">59/60</td><td colspan="1" rowspan="1">98.3%</td><td colspan="1" rowspan="1">91.1-99.7%</td></tr><tr><td colspan="2" rowspan="4">Influenza A/NJ/8/76(Influenza A H1)</td><td colspan="1" rowspan="1">Site 1</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">83.9-100%</td></tr><tr><td colspan="1" rowspan="1">Site 2</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">83.9-100%</td></tr><tr><td colspan="1" rowspan="1">Site 3</td><td colspan="1" rowspan="1">19/20</td><td colspan="1" rowspan="1">95.0%</td><td colspan="1" rowspan="1">76.4-99.1%</td></tr><tr><td colspan="1" rowspan="1">All Sites</td><td colspan="1" rowspan="1">59/60</td><td colspan="1" rowspan="1">98.3%</td><td colspan="1" rowspan="1">91.1-99.7%</td></tr><tr><td colspan="2" rowspan="4">Influenza B/Taiwan/2/62(Influenza B)</td><td colspan="1" rowspan="1">Site 1</td><td colspan="1" rowspan="1">19/20</td><td colspan="1" rowspan="1">95.0%</td><td colspan="1" rowspan="1">76.4-99.1%</td></tr><tr><td colspan="1" rowspan="1">Site 2</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">83.9-100%</td></tr><tr><td colspan="1" rowspan="1">Site 3</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">83.9-100%</td></tr><tr><td colspan="1" rowspan="1">All Sites</td><td colspan="1" rowspan="1">59/60</td><td colspan="1" rowspan="1">98.3%</td><td colspan="1" rowspan="1">91.1-99.7%</td></tr><tr><td colspan="2" rowspan="4">InfluenzaA/SwineNY/03/2009(Influenza A H1N1pdm09)</td><td colspan="1" rowspan="1">Site 1</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">83.9-100%</td></tr><tr><td colspan="1" rowspan="1">Site 2</td><td colspan="1" rowspan="1">19/19</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">83.2-100%</td></tr><tr><td colspan="1" rowspan="1">Site 3</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">83.9-100%</td></tr><tr><td colspan="1" rowspan="1">All Sites</td><td colspan="1" rowspan="1">59/59</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">93.9-100%</td></tr><tr><td colspan="2" rowspan="4">Influenza A/PortChalmers/1/73(Influenza A H3)</td><td colspan="1" rowspan="1">Site 1</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">83.9-100%</td></tr><tr><td colspan="1" rowspan="1">Site 2</td><td colspan="1" rowspan="1">18/18</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">82.4-100%</td></tr><tr><td colspan="1" rowspan="1">Site 3</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">83.9-100%</td></tr><tr><td colspan="1" rowspan="1">All Sites</td><td colspan="1" rowspan="1">58/58</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">93.8-100%</td></tr><tr><td colspan="2" rowspan="4">Mycoplasmapneumoniae(M. pneumoniae)</td><td colspan="1" rowspan="1">Site 1</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">83.9-100%</td></tr><tr><td colspan="1" rowspan="1">Site 2</td><td colspan="1" rowspan="1">19/19</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">83.2-100%</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Site 3</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">83.9-100%</td></tr><tr><td colspan="1" rowspan="1">All Sites</td><td colspan="1" rowspan="1">59/59</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">93.9-100%</td></tr></table>

<table><tr><td rowspan=4 colspan=1>Parainfluenza virus 1(PIV 1)</td><td rowspan=1 colspan=1>Site 1</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>83.9-100%</td></tr><tr><td rowspan=1 colspan=1>Site 2</td><td rowspan=1 colspan=1>18/18</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>82.4-100%</td></tr><tr><td rowspan=1 colspan=1>Site 3</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>83.9-100%</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>58/58</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>93.8-100%</td></tr><tr><td rowspan=4 colspan=1>Parainfluenza virus 2(PIV 2)</td><td rowspan=1 colspan=1>Site 1</td><td rowspan=1 colspan=1>19/20</td><td rowspan=1 colspan=1>95.0%</td><td rowspan=1 colspan=1>76.4-99.1%</td></tr><tr><td rowspan=1 colspan=1>Site 2</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>83.9-100%</td></tr><tr><td rowspan=1 colspan=1>Site 3</td><td rowspan=1 colspan=1>19/20</td><td rowspan=1 colspan=1>95.0%</td><td rowspan=1 colspan=1>76.4-99.1%</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>58/60</td><td rowspan=1 colspan=1>96.7%</td><td rowspan=1 colspan=1>88.6-99.1%</td></tr><tr><td rowspan=4 colspan=1>Parainfluenza virus 3(PIV 3)</td><td rowspan=1 colspan=1>Site 1</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>83.9-100%</td></tr><tr><td rowspan=1 colspan=1>Site 2</td><td rowspan=1 colspan=1>19/19</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>83.2-100%</td></tr><tr><td rowspan=1 colspan=1>Site 3</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>83.9-100%</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>59/59</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>93.9-100%</td></tr><tr><td rowspan=4 colspan=1>Parainfluenza virus 4(PIV 4)</td><td rowspan=1 colspan=1>Site 1</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>83.9-100%</td></tr><tr><td rowspan=1 colspan=1>Site 2</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>83.9-100%</td></tr><tr><td rowspan=1 colspan=1>Site 3</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>83.9-100%</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>60/60</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>94.0-100%</td></tr><tr><td rowspan=4 colspan=1>RSV A(RSV)</td><td rowspan=1 colspan=1>Site 1</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>83.9-100%</td></tr><tr><td rowspan=1 colspan=1>Site 2</td><td rowspan=1 colspan=1>19/19</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>83.2-100%</td></tr><tr><td rowspan=1 colspan=1>Site 3</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>83.9-100%</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>59/59</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>93.9-100%</td></tr><tr><td rowspan=4 colspan=1>RSV B(RSV)</td><td rowspan=1 colspan=1>Site 1</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>83.9-100%</td></tr><tr><td rowspan=1 colspan=1>Site 2</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>83.9-100%</td></tr><tr><td rowspan=1 colspan=1>Site 3</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>83.9-100%</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>60/60</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>94.0-100%</td></tr><tr><td rowspan=4 colspan=1>Rhinovirus(Rhinovirus /Enterovirus</td><td rowspan=1 colspan=1>Site 1</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>83.9-100%</td></tr><tr><td rowspan=1 colspan=1>Site 2</td><td rowspan=1 colspan=1>19/19</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>83.2-100%</td></tr><tr><td rowspan=1 colspan=1>Site 3</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>83.9-100%</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>59/59</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>93.9-100%</td></tr></table>

Table 4: QIAstat-Dx Reproducibility with Samples at 3x LOD   

<table><tr><td colspan="2" rowspan="1">Analyte(QIAstat-Dx Target)</td><td colspan="1" rowspan="1">Site</td><td colspan="1" rowspan="1">Positive Detected</td><td colspan="1" rowspan="1">PercentAgreement withExpected Results</td><td colspan="1" rowspan="1">95% CI</td></tr><tr><td colspan="2" rowspan="4">Adenovirus(Adenovirus)</td><td colspan="1" rowspan="1">Site 1</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">83.9-100%</td></tr><tr><td colspan="1" rowspan="1">Site 2</td><td colspan="1" rowspan="1">19/19</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">83.2-100%</td></tr><tr><td colspan="1" rowspan="1">Site 3</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">83.9-100%</td></tr><tr><td colspan="1" rowspan="1">All Sites</td><td colspan="1" rowspan="1">59/59</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">93.9-100%</td></tr><tr><td colspan="2" rowspan="4">B. pertussis(B. pertussis)</td><td colspan="1" rowspan="1">Site 1</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">83.9-100%</td></tr><tr><td colspan="1" rowspan="1">Site 2</td><td colspan="1" rowspan="1">19/19</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">83.2-100%</td></tr><tr><td colspan="1" rowspan="1">Site 3</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">83.9-100%</td></tr><tr><td colspan="1" rowspan="1">All Sites</td><td colspan="1" rowspan="1">59/59</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">93.9-100%</td></tr><tr><td colspan="2" rowspan="4">C. pneumoniae(C. pneumoniae)</td><td colspan="1" rowspan="1">Site 1</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">83.9-100%</td></tr><tr><td colspan="1" rowspan="1">Site 2</td><td colspan="1" rowspan="1">19/20</td><td colspan="1" rowspan="1">95.0%</td><td colspan="1" rowspan="1">76.4-99.1%</td></tr><tr><td colspan="1" rowspan="1">Site 3</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">83.9-100%</td></tr><tr><td colspan="1" rowspan="1">All Sites</td><td colspan="1" rowspan="1">59/60</td><td colspan="1" rowspan="1">98.3%</td><td colspan="1" rowspan="1">91.1-99.7%</td></tr><tr><td colspan="2" rowspan="4">Coronavirus 229E(Coronavirus 229E)</td><td colspan="1" rowspan="1">Site 1</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">83.9-100%</td></tr><tr><td colspan="1" rowspan="1">Site 2</td><td colspan="1" rowspan="1">19/19</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">83.2-100%</td></tr><tr><td colspan="1" rowspan="1">Site 3</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">83.9-100%</td></tr><tr><td colspan="1" rowspan="1">All Sites</td><td colspan="1" rowspan="1">59/59</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">93.9-100%</td></tr><tr><td colspan="2" rowspan="4">Coronavirus HKU1(Coronavirus HKU1)</td><td colspan="1" rowspan="1">Site 1</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">83.9-100%</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Site 2</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">83.9-100%</td></tr><tr><td colspan="1" rowspan="1">Site 3</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">83.9-100%</td></tr><tr><td colspan="1" rowspan="1">All Sites</td><td colspan="1" rowspan="1">60/60</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">94.0-100%</td></tr><tr><td colspan="2" rowspan="4">Coronavirus NL63(Coronavirus NL63)</td><td colspan="1" rowspan="1">Site 1</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">83.9-100%</td></tr><tr><td colspan="1" rowspan="1">63)</td><td colspan="1" rowspan="1">Site 2</td><td colspan="1" rowspan="1">19/19</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">83.2-100%</td></tr><tr><td colspan="1" rowspan="1">Site 3</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">83.9-100%</td></tr><tr><td colspan="1" rowspan="1">All Sites</td><td colspan="1" rowspan="1">59/59</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">93.9-100%</td></tr><tr><td colspan="2" rowspan="4">Coronavirus OC43(Coronavirus OC43)</td><td colspan="1" rowspan="1">Site 1</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">83.9-100%</td></tr><tr><td colspan="1" rowspan="1">Site 2</td><td colspan="1" rowspan="1">19/19</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">83.2-100%</td></tr><tr><td colspan="1" rowspan="1">Site 3</td><td colspan="1" rowspan="1">19/19</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">83.2-100%</td></tr><tr><td colspan="1" rowspan="1">All Sites</td><td colspan="1" rowspan="1">58/58</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">93.8-100%</td></tr><tr><td colspan="2" rowspan="4">Enterovirus(Rhinovirus /Enterovirus)</td><td colspan="1" rowspan="1">Site 1</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">83.9-100%</td></tr><tr><td colspan="1" rowspan="1">Site 2</td><td colspan="1" rowspan="1">19/19</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">83.2-100%</td></tr><tr><td colspan="1" rowspan="1">Site 3</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">83.9-100%</td></tr><tr><td colspan="1" rowspan="1">All Sites</td><td colspan="1" rowspan="1">59/59</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">93.9-100%</td></tr><tr><td colspan="2" rowspan="4">HumanMetapneumovirus(hMPV)</td><td colspan="1" rowspan="1">Site 1</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">83.9-100%</td></tr><tr><td colspan="1" rowspan="1">Site 2</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">83.2-100%</td></tr><tr><td colspan="1" rowspan="1">Site 3</td><td colspan="1" rowspan="1">19/19</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">83.9-100%</td></tr><tr><td colspan="1" rowspan="1">All Sites</td><td colspan="1" rowspan="1">59/59</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">93.9-100%</td></tr><tr><td colspan="2" rowspan="4">InfluenzaA/SwineNY/03/2009(Influenza A)</td><td colspan="1" rowspan="1">Site 1</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">83.9-100%</td></tr><tr><td colspan="1" rowspan="1">Site 2</td><td colspan="1" rowspan="1">19/19</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">83.2-100%</td></tr><tr><td colspan="1" rowspan="1">Site 3</td><td colspan="1" rowspan="1">19/19</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">83.2-100%</td></tr><tr><td colspan="1" rowspan="1">All Sites</td><td colspan="1" rowspan="1">58/58</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">93.8-100%</td></tr><tr><td colspan="2" rowspan="4">Influenza A/PortChalmers/1/73(Influenza A)</td><td colspan="1" rowspan="1">Site 1</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">83.9-100%</td></tr><tr><td colspan="1" rowspan="1">Site 2</td><td colspan="1" rowspan="1">19/19</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">83.2-100%</td></tr><tr><td colspan="1" rowspan="1">Site 3</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">83.9-100%</td></tr><tr><td colspan="1" rowspan="1">All Sites</td><td colspan="1" rowspan="1">59/59</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">93.9-100%</td></tr><tr><td colspan="2" rowspan="4">Influenza A/NJ/8/76(Influenza A)</td><td colspan="1" rowspan="1">Site 1</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">83.9-100%</td></tr><tr><td colspan="1" rowspan="1">Site 2</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">83.9-100%</td></tr><tr><td colspan="1" rowspan="1">Site 3</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">83.9-100%</td></tr><tr><td colspan="1" rowspan="1">All Sites</td><td colspan="1" rowspan="1">60/60</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">94.0-100%</td></tr><tr><td colspan="2" rowspan="4">Influenza A/NJ/8/76(Influenza A H1)</td><td colspan="1" rowspan="1">Site 1</td><td colspan="1" rowspan="1">19/20</td><td colspan="1" rowspan="1">95.0%</td><td colspan="1" rowspan="1">76.4-99.1%</td></tr><tr><td colspan="1" rowspan="1">Site 2</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">83.9-100%</td></tr><tr><td colspan="1" rowspan="1">Site 3</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">83.9-100%</td></tr><tr><td colspan="1" rowspan="1">All Sites</td><td colspan="1" rowspan="1">59/60</td><td colspan="1" rowspan="1">98.3%</td><td colspan="1" rowspan="1">91.1-99.7%</td></tr><tr><td colspan="2" rowspan="4">Influenza B/Taiwan/2/62(Influenza B)</td><td colspan="1" rowspan="1">Site 1</td><td colspan="1" rowspan="1">19/20</td><td colspan="1" rowspan="1">95.0%</td><td colspan="1" rowspan="1">76.4-99.1%</td></tr><tr><td colspan="1" rowspan="1">Site 2</td><td colspan="1" rowspan="1">19/19</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">83.2-100%</td></tr><tr><td colspan="1" rowspan="1">Site 3</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">83.9-100%</td></tr><tr><td colspan="1" rowspan="1">All Sites</td><td colspan="1" rowspan="1">58/59</td><td colspan="1" rowspan="1">98.3%</td><td colspan="1" rowspan="1">91.0-99.7%</td></tr><tr><td colspan="2" rowspan="4">InfluenzaA/SwineNY/03/2009(Influenza A H1N1pdm09)</td><td colspan="1" rowspan="1">Site 1</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">83.9-100%</td></tr><tr><td colspan="1" rowspan="1">Site 2</td><td colspan="1" rowspan="1">19/19</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">83.2-100%</td></tr><tr><td colspan="1" rowspan="1">Site 3</td><td colspan="1" rowspan="1">19/19</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">83.2-100%</td></tr><tr><td colspan="1" rowspan="1">All Sites</td><td colspan="1" rowspan="1">58/58</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">93.8-100%</td></tr><tr><td colspan="2" rowspan="4">Influenza A/PortChalmers/1/73(Influenza A H3)</td><td colspan="1" rowspan="1">Site 1</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">83.9-100%</td></tr><tr><td colspan="1" rowspan="1">Site 2</td><td colspan="1" rowspan="1">19/19</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">83.2-100%</td></tr><tr><td colspan="1" rowspan="1">Site 3</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">83.9-100%</td></tr><tr><td colspan="1" rowspan="1">All Sites</td><td colspan="1" rowspan="1">59/59</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">93.9-100%</td></tr><tr><td colspan="2" rowspan="4">Mycoplasmapneumoniae(M. pneumoniae)</td><td colspan="1" rowspan="1">Site 1</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">83.9-100%</td></tr><tr><td colspan="1" rowspan="1">Site 2</td><td colspan="1" rowspan="1">19/19</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">83.2-100%</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Site 3</td><td colspan="1" rowspan="1">19/19</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">83.2-100%</td></tr><tr><td colspan="1" rowspan="1">All Sites</td><td colspan="1" rowspan="1">58/58</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">93.8-100%</td></tr></table>

<table><tr><td rowspan=4 colspan=1>Parainfluenza virus 1(PIV 1)</td><td rowspan=1 colspan=1>Site 1</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>83.9-100%</td></tr><tr><td rowspan=1 colspan=1>Site 2</td><td rowspan=1 colspan=1>19/19</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>83.2-100%</td></tr><tr><td rowspan=1 colspan=1>Site 3</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>83.9-100%</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>59/59</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>93.9-100%</td></tr><tr><td rowspan=4 colspan=1>Parainfluenza virus 2(PIV 2)</td><td rowspan=1 colspan=1>Site 1</td><td rowspan=1 colspan=1>19/20</td><td rowspan=1 colspan=1>95.0%</td><td rowspan=1 colspan=1>76.4-99.1%</td></tr><tr><td rowspan=1 colspan=1>Site 2</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>83.9-100%</td></tr><tr><td rowspan=1 colspan=1>Site 3</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>83.9-100%</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>59/60</td><td rowspan=1 colspan=1>98.3%</td><td rowspan=1 colspan=1>91.1-99.7%</td></tr><tr><td rowspan=4 colspan=1>Parainfluenza virus 3(PIV 3)</td><td rowspan=1 colspan=1>Site 1</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>83.9-100%</td></tr><tr><td rowspan=1 colspan=1>Site 2</td><td rowspan=1 colspan=1>19/19</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>83.2-100%</td></tr><tr><td rowspan=1 colspan=1>Site 3</td><td rowspan=1 colspan=1>19/19</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>83.2-100%</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>58/58</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>93.8-100%</td></tr><tr><td rowspan=4 colspan=1>Parainfluenza virus 4(PIV 4)</td><td rowspan=1 colspan=1>Site 1</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>83.9-100%</td></tr><tr><td rowspan=1 colspan=1>Site 2</td><td rowspan=1 colspan=1>19/19</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>83.2-100%</td></tr><tr><td rowspan=1 colspan=1>Site 3</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>83.9-100%</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>59/59</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>93.9-100%</td></tr><tr><td rowspan=4 colspan=1>RSV A(RSV)</td><td rowspan=1 colspan=1>Site 1</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>83.9-100%</td></tr><tr><td rowspan=1 colspan=1>Site 2</td><td rowspan=1 colspan=1>19/19</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>83.2-100%</td></tr><tr><td rowspan=1 colspan=1>Site 3</td><td rowspan=1 colspan=1>19/19</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>83.2-100%</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>58/58</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>93.8-100%</td></tr><tr><td rowspan=4 colspan=1>RSV B(RSV)</td><td rowspan=1 colspan=1>Site 1</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>83.9-100%</td></tr><tr><td rowspan=1 colspan=1>Site 2</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>83.9-100%</td></tr><tr><td rowspan=1 colspan=1>Site 3</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>83.9-100%</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>60/60</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>94.0-100%</td></tr><tr><td rowspan=4 colspan=1>Rhinovirus(Rhinovirus /Enterovirus)</td><td rowspan=1 colspan=1>Site 1</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>83.9-100%</td></tr><tr><td rowspan=1 colspan=1>Site 2</td><td rowspan=1 colspan=1>19/19</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>83.2-100%</td></tr><tr><td rowspan=1 colspan=1>Site 3</td><td rowspan=1 colspan=1>19/19</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>83.2-100%</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>58/58</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>93.8-100%</td></tr></table>

Table 5: Percent Variance Between Site, Day, Operator, and Instrument According to ANOVA for 1x LOD Reproducibility Study   

<table><tr><td colspan="1" rowspan="1">Analyte</td><td colspan="1" rowspan="1">N</td><td colspan="1" rowspan="1">MeanCt</td><td colspan="1" rowspan="1">BetweenSite</td><td colspan="1" rowspan="1">BetweenInstrument</td><td colspan="1" rowspan="1">BetweenDay</td><td colspan="1" rowspan="1">BetweenOperator</td><td colspan="1" rowspan="1">Residual</td><td colspan="1" rowspan="1">Total</td></tr><tr><td colspan="1" rowspan="1">Adenovirus</td><td colspan="1" rowspan="1">58</td><td colspan="1" rowspan="1">34.2</td><td colspan="1" rowspan="1">0.00%</td><td colspan="1" rowspan="1">4.30%</td><td colspan="1" rowspan="1">1.63%</td><td colspan="1" rowspan="1">10.4%</td><td colspan="1" rowspan="1">83.7%</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">B. pertussis</td><td colspan="1" rowspan="1">58</td><td colspan="1" rowspan="1">35.3</td><td colspan="1" rowspan="1">0.00%</td><td colspan="1" rowspan="1">0.00%</td><td colspan="1" rowspan="1">2.08%</td><td colspan="1" rowspan="1">0.00%</td><td colspan="1" rowspan="1">97.9%</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">C. pneumoniae</td><td colspan="1" rowspan="1">60</td><td colspan="1" rowspan="1">33.7</td><td colspan="1" rowspan="1">0.00%</td><td colspan="1" rowspan="1">0.00%</td><td colspan="1" rowspan="1">0.00%</td><td colspan="1" rowspan="1">0.00%</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">CoV 229E</td><td colspan="1" rowspan="1">58</td><td colspan="1" rowspan="1">36.1</td><td colspan="1" rowspan="1">0.00%</td><td colspan="1" rowspan="1">2.58%</td><td colspan="1" rowspan="1">0.00%</td><td colspan="1" rowspan="1">0.00%</td><td colspan="1" rowspan="1">97.4%</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">CoV HKU1</td><td colspan="1" rowspan="1">60</td><td colspan="1" rowspan="1">36.3</td><td colspan="1" rowspan="1">3.14%</td><td colspan="1" rowspan="1">0.00%</td><td colspan="1" rowspan="1">8.93%</td><td colspan="1" rowspan="1">0.00%</td><td colspan="1" rowspan="1">87.9%</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">CoV NL63</td><td colspan="1" rowspan="1">58</td><td colspan="1" rowspan="1">35.8</td><td colspan="1" rowspan="1">0.00%</td><td colspan="1" rowspan="1">0.00%</td><td colspan="1" rowspan="1">0.00%</td><td colspan="1" rowspan="1">10.5%</td><td colspan="1" rowspan="1">89.5%</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">CoV OC43</td><td colspan="1" rowspan="1">59</td><td colspan="1" rowspan="1">33.1</td><td colspan="1" rowspan="1">0.00%</td><td colspan="1" rowspan="1">0.00%</td><td colspan="1" rowspan="1">0.00%</td><td colspan="1" rowspan="1">1.64%</td><td colspan="1" rowspan="1">98.4%</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Enterovirus</td><td colspan="1" rowspan="1">58</td><td colspan="1" rowspan="1">35.8</td><td colspan="1" rowspan="1">0.00%</td><td colspan="1" rowspan="1">0.00%</td><td colspan="1" rowspan="1">0.00%</td><td colspan="1" rowspan="1">0.00%</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">hMPV</td><td colspan="1" rowspan="1">59</td><td colspan="1" rowspan="1">34.5</td><td colspan="1" rowspan="1">0.00%</td><td colspan="1" rowspan="1">0.00%</td><td colspan="1" rowspan="1">0.00%</td><td colspan="1" rowspan="1">0.00%</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Influenza A1</td><td colspan="1" rowspan="1">59</td><td colspan="1" rowspan="1">33.5</td><td colspan="1" rowspan="1">16.8%</td><td colspan="1" rowspan="1">0.00%</td><td colspan="1" rowspan="1">0.00%</td><td colspan="1" rowspan="1">0.00%</td><td colspan="1" rowspan="1">83.2%</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Influenza A2</td><td colspan="1" rowspan="1">57</td><td colspan="1" rowspan="1">36.0</td><td colspan="1" rowspan="1">0.00%</td><td colspan="1" rowspan="1">4.95%</td><td colspan="1" rowspan="1">12.0%</td><td colspan="1" rowspan="1">0.00%</td><td colspan="1" rowspan="1">83.1%</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Influenza A3</td><td colspan="1" rowspan="1">59</td><td colspan="1" rowspan="1">36.4</td><td colspan="1" rowspan="1">0.00%</td><td colspan="1" rowspan="1">0.00%</td><td colspan="1" rowspan="1">0.00%</td><td colspan="1" rowspan="1">0.00%</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Influenza A4</td><td colspan="1" rowspan="1">59</td><td colspan="1" rowspan="1">34.1</td><td colspan="1" rowspan="1">0.00%</td><td colspan="1" rowspan="1">0.00%</td><td colspan="1" rowspan="1">0.00%</td><td colspan="1" rowspan="1">4.18%</td><td colspan="1" rowspan="1">95.8%</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Influenza B</td><td colspan="1" rowspan="1">59</td><td colspan="1" rowspan="1">33.9</td><td colspan="1" rowspan="1">0.00%</td><td colspan="1" rowspan="1">2.92%</td><td colspan="1" rowspan="1">0.00%</td><td colspan="1" rowspan="1">0.00%</td><td colspan="1" rowspan="1">97.1%</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Influenza H1N1pdm095</td><td colspan="1" rowspan="1">59</td><td colspan="1" rowspan="1">33.2</td><td colspan="1" rowspan="1">21.2%</td><td colspan="1" rowspan="1">0.00%</td><td colspan="1" rowspan="1">0.00%</td><td colspan="1" rowspan="1">0.00%</td><td colspan="1" rowspan="1">78.8%</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Influenza A H36</td><td colspan="1" rowspan="1">58</td><td colspan="1" rowspan="1">34.0</td><td colspan="1" rowspan="1">2.22%</td><td colspan="1" rowspan="1">0.00%</td><td colspan="1" rowspan="1">0.00%</td><td colspan="1" rowspan="1">2.04%</td><td colspan="1" rowspan="1">95.7%</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">M. pneumoniae</td><td colspan="1" rowspan="1">59</td><td colspan="1" rowspan="1">33.3</td><td colspan="1" rowspan="1">11.3%</td><td colspan="1" rowspan="1">0.00%</td><td colspan="1" rowspan="1">0.00%</td><td colspan="1" rowspan="1">23.2%</td><td colspan="1" rowspan="1">65.5%</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">PIV 1</td><td colspan="1" rowspan="1">58</td><td colspan="1" rowspan="1">34.9</td><td colspan="1" rowspan="1">0.00%</td><td colspan="1" rowspan="1">0.00%</td><td colspan="1" rowspan="1">0.00%</td><td colspan="1" rowspan="1">0.00%</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">PIV 2</td><td colspan="1" rowspan="1">58</td><td colspan="1" rowspan="1">33.7</td><td colspan="1" rowspan="1">0.00%</td><td colspan="1" rowspan="1">0.00%</td><td colspan="1" rowspan="1">0.00%</td><td colspan="1" rowspan="1">7.88%</td><td colspan="1" rowspan="1">92.1%</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">PIV 3</td><td colspan="1" rowspan="1">59</td><td colspan="1" rowspan="1">33.3</td><td colspan="1" rowspan="1">0.00%</td><td colspan="1" rowspan="1">7.62%</td><td colspan="1" rowspan="1">0.00%</td><td colspan="1" rowspan="1">0.00%</td><td colspan="1" rowspan="1">92.4%</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">PIV 4</td><td colspan="1" rowspan="1">60</td><td colspan="1" rowspan="1">33.6</td><td colspan="1" rowspan="1">0.00%</td><td colspan="1" rowspan="1">0.00%</td><td colspan="1" rowspan="1">0.00%</td><td colspan="1" rowspan="1">0.00%</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">RSV A</td><td colspan="1" rowspan="1">59</td><td colspan="1" rowspan="1">34.2</td><td colspan="1" rowspan="1">11.3%</td><td colspan="1" rowspan="1">0.00%</td><td colspan="1" rowspan="1">0.00%</td><td colspan="1" rowspan="1">0.00%</td><td colspan="1" rowspan="1">88.7%</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">RSV B</td><td colspan="1" rowspan="1">60</td><td colspan="1" rowspan="1">33.9</td><td colspan="1" rowspan="1">0.00%</td><td colspan="1" rowspan="1">0.00%</td><td colspan="1" rowspan="1">2.45%</td><td colspan="1" rowspan="1">0.00%</td><td colspan="1" rowspan="1">97.6%</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Rhinovirus</td><td colspan="1" rowspan="1">59</td><td colspan="1" rowspan="1">33.7</td><td colspan="1" rowspan="1">0.00%</td><td colspan="1" rowspan="1">0.00%</td><td colspan="1" rowspan="1">0.00%</td><td colspan="1" rowspan="1">3.02%</td><td colspan="1" rowspan="1">97.0%</td><td colspan="1" rowspan="1">100%</td></tr></table>

1 Strain A/Swine NY/03/2009; Flu A target. 2 Strain A/Port Chalmers/1/73’ Flu A target. 3 Strain A/NJ/8/1976; Flu A target. 4 Strain A/NJ/8/1976; H1 target. 5 Strain A/Swine NY/03/2009; pdm09 target. 6 Strain A/Port Chalmers/1/73; H3 target.

Table 6: Percent Variance Between Site, Day, Operator, and Instrument According to ANOVA for 3x LOD Reproducibility Study   

<table><tr><td rowspan=1 colspan=1>Analyte</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>MeanCt</td><td rowspan=1 colspan=1>BetweenSite</td><td rowspan=1 colspan=1>BetweenInstrument</td><td rowspan=1 colspan=1>BetweenDay</td><td rowspan=1 colspan=1>BetweenOperator</td><td rowspan=1 colspan=1>Residual</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>59</td><td rowspan=1 colspan=1>33.1</td><td rowspan=1 colspan=1>0.00%</td><td rowspan=1 colspan=1>32.3%</td><td rowspan=1 colspan=1>4.56%</td><td rowspan=1 colspan=1>0.00%</td><td rowspan=1 colspan=1>63.1%</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>B. pertussis</td><td rowspan=1 colspan=1>59</td><td rowspan=1 colspan=1>34.2</td><td rowspan=1 colspan=1>0.00%</td><td rowspan=1 colspan=1>22.7%</td><td rowspan=1 colspan=1>1.23%</td><td rowspan=1 colspan=1>0.00%</td><td rowspan=1 colspan=1>76.1%</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>C. pneumoniae</td><td rowspan=1 colspan=1>59</td><td rowspan=1 colspan=1>33.2</td><td rowspan=1 colspan=1>0.00%</td><td rowspan=1 colspan=1>0.00%</td><td rowspan=1 colspan=1>2.32%</td><td rowspan=1 colspan=1>0.00%</td><td rowspan=1 colspan=1>97.7%</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>CoV 229E</td><td rowspan=1 colspan=1>59</td><td rowspan=1 colspan=1>34.6</td><td rowspan=1 colspan=1>12.4%</td><td rowspan=1 colspan=1>0.00%</td><td rowspan=1 colspan=1>3.77%</td><td rowspan=1 colspan=1>0.13%</td><td rowspan=1 colspan=1>83.7%</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>CoV HKU1</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>34.8</td><td rowspan=1 colspan=1>28.2%</td><td rowspan=1 colspan=1>7.31%</td><td rowspan=1 colspan=1>0.00%</td><td rowspan=1 colspan=1>0.32%</td><td rowspan=1 colspan=1>64.2%</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>CoV NL63</td><td rowspan=1 colspan=1>59</td><td rowspan=1 colspan=1>34.0</td><td rowspan=1 colspan=1>0.00%</td><td rowspan=1 colspan=1>7.24%</td><td rowspan=1 colspan=1>0.00%</td><td rowspan=1 colspan=1>0.00%</td><td rowspan=1 colspan=1>92.8%</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>CoV OC43</td><td rowspan=1 colspan=1>58</td><td rowspan=1 colspan=1>31.1</td><td rowspan=1 colspan=1>1.36%</td><td rowspan=1 colspan=1>0.00%</td><td rowspan=1 colspan=1>1.53%</td><td rowspan=1 colspan=1>0.00%</td><td rowspan=1 colspan=1>97.1%</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Enterovirus</td><td rowspan=1 colspan=1>59</td><td rowspan=1 colspan=1>34.1</td><td rowspan=1 colspan=1>0.00%</td><td rowspan=1 colspan=1>7.02%</td><td rowspan=1 colspan=1>0.80%</td><td rowspan=1 colspan=1>0.00%</td><td rowspan=1 colspan=1>92.2%</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>hMPV</td><td rowspan=1 colspan=1>59</td><td rowspan=1 colspan=1>33.2</td><td rowspan=1 colspan=1>1.11%</td><td rowspan=1 colspan=1>0.00%</td><td rowspan=1 colspan=1>0.00%</td><td rowspan=1 colspan=1>0.00%</td><td rowspan=1 colspan=1>98.9%</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Influenza A1</td><td rowspan=1 colspan=1>58</td><td rowspan=1 colspan=1>31.8</td><td rowspan=1 colspan=1>0.00%</td><td rowspan=1 colspan=1>3.43%</td><td rowspan=1 colspan=1>0.00%</td><td rowspan=1 colspan=1>0.15%</td><td rowspan=1 colspan=1>96.4%</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Influenza A2</td><td rowspan=1 colspan=1>59</td><td rowspan=1 colspan=1>34.4</td><td rowspan=1 colspan=1>0.00%</td><td rowspan=1 colspan=1>0.00%</td><td rowspan=1 colspan=1>0.99%</td><td rowspan=1 colspan=1>0.00%</td><td rowspan=1 colspan=1>99.0%</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Influenza A3</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>35.1</td><td rowspan=1 colspan=1>7.75%</td><td rowspan=1 colspan=1>0.00%</td><td rowspan=1 colspan=1>3.86%</td><td rowspan=1 colspan=1>0.00%</td><td rowspan=1 colspan=1>88.4%</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Influenza A4</td><td rowspan=1 colspan=1>59</td><td rowspan=1 colspan=1>32.6</td><td rowspan=1 colspan=1>0.00%</td><td rowspan=1 colspan=1>0.01%</td><td rowspan=1 colspan=1>0.00%</td><td rowspan=1 colspan=1>0.28%</td><td rowspan=1 colspan=1>99.7%</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Influenza B</td><td rowspan=1 colspan=1>58</td><td rowspan=1 colspan=1>32.2</td><td rowspan=1 colspan=1>9.20%</td><td rowspan=1 colspan=1>0.00%</td><td rowspan=1 colspan=1>0.00%</td><td rowspan=1 colspan=1>4.58%</td><td rowspan=1 colspan=1>86.2%</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Influenza H1N1pdm095</td><td rowspan=1 colspan=1>58</td><td rowspan=1 colspan=1>31.5</td><td rowspan=1 colspan=1>5.65%</td><td rowspan=1 colspan=1>1.97%</td><td rowspan=1 colspan=1>0.00%</td><td rowspan=1 colspan=1>0.35%</td><td rowspan=1 colspan=1>92.0%</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Influenza A H36</td><td rowspan=1 colspan=1>59</td><td rowspan=1 colspan=1>32.4</td><td rowspan=1 colspan=1>0.00%</td><td rowspan=1 colspan=1>1.76%</td><td rowspan=1 colspan=1>0.00%</td><td rowspan=1 colspan=1>0.00%</td><td rowspan=1 colspan=1>98.2%</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>M. pneumoniae</td><td rowspan=1 colspan=1>58</td><td rowspan=1 colspan=1>31.5</td><td rowspan=1 colspan=1>0.00%</td><td rowspan=1 colspan=1>0.00%</td><td rowspan=1 colspan=1>0.00%</td><td rowspan=1 colspan=1>0.00%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>PIV 1</td><td rowspan=1 colspan=1>59</td><td rowspan=1 colspan=1>33.6</td><td rowspan=1 colspan=1>4.70%</td><td rowspan=1 colspan=1>0.00%</td><td rowspan=1 colspan=1>0.00%</td><td rowspan=1 colspan=1>0.00%</td><td rowspan=1 colspan=1>95.3%</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>PIV 2</td><td rowspan=1 colspan=1>59</td><td rowspan=1 colspan=1>32.5</td><td rowspan=1 colspan=1>0.00%</td><td rowspan=1 colspan=1>1.96%</td><td rowspan=1 colspan=1>0.00%</td><td rowspan=1 colspan=1>0.04%</td><td rowspan=1 colspan=1>98.0%</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>PIV 3</td><td rowspan=1 colspan=1>58</td><td rowspan=1 colspan=1>31.8</td><td rowspan=1 colspan=1>0.00%</td><td rowspan=1 colspan=1>0.00%</td><td rowspan=1 colspan=1>9.47%</td><td rowspan=1 colspan=1>0.00%</td><td rowspan=1 colspan=1>90.5%</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>PIV 4</td><td rowspan=1 colspan=1>59</td><td rowspan=1 colspan=1>31.9</td><td rowspan=1 colspan=1>0.00%</td><td rowspan=1 colspan=1>0.00%</td><td rowspan=1 colspan=1>0.00%</td><td rowspan=1 colspan=1>0.00%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>RSV A</td><td rowspan=1 colspan=1>58</td><td rowspan=1 colspan=1>32.8</td><td rowspan=1 colspan=1>2.59%</td><td rowspan=1 colspan=1>0.00%</td><td rowspan=1 colspan=1>10.5%</td><td rowspan=1 colspan=1>0.00%</td><td rowspan=1 colspan=1>86.9%</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>RSV B</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>32.7</td><td rowspan=1 colspan=1>0.00%</td><td rowspan=1 colspan=1>8.47%</td><td rowspan=1 colspan=1>0.00%</td><td rowspan=1 colspan=1>0.76%</td><td rowspan=1 colspan=1>90.8%</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Rhinovirus</td><td rowspan=1 colspan=1>58</td><td rowspan=1 colspan=1>32.1</td><td rowspan=1 colspan=1>2.77%</td><td rowspan=1 colspan=1>0.00%</td><td rowspan=1 colspan=1>0.00%</td><td rowspan=1 colspan=1>0.00%</td><td rowspan=1 colspan=1>97.2%</td><td rowspan=1 colspan=1>100%</td></tr></table>

b. Linearity/assay reportable range:

Not applicable; this is a qualitative assay.

c. Traceability, Stability, Expected values (controls, calibrators, or methods):

Specimen Stability

To provide data supporting the specimen storage recommendations stated in the product package insert, an analytical study was carried out to evaluate specimen stability.

Contrived positive NPS samples (eluted NPS samples) were prepared by diluting specific panel targets into a sample medium consisting of UTM combined with HeLa cells. Sample mixes were prepared at both ${ 5 } \mathbf { x }$ LOD and 1x LOD concentrations in discrete sample mixes. Negative data was obtained from specimen mixes where the analyte was expected to be absent based on the composition of the specific mix. All prepared samples were tested at $\Nu { = } 1 0$ at each of the following three time points/conditions: Time 0 (fresh), 4 hours at 15 to $2 5 ~ ^ { \circ } \mathrm { C }$ , 72 hours at 2 to $8 \ { } ^ { \circ } { \mathrm { C } }$ , and 30 days at -15 to $- 2 5 ~ ^ { \circ } \mathrm { C }$ . Positive samples were considered stable as long as they tested positive in the QIAstat-Dx Respiratory Panel with at least a $90 \%$ detection rate. The acceptance criteria of $2 9 0 \%$ detection was achieved for all analytes at the 5x LOD concentration tested under the four conditions described above. Details regarding the detection rates are presented in the tables below.

Table 7a: Specimen Stability Study Results – 5x LOD   

<table><tr><td colspan="1" rowspan="1">Mix</td><td colspan="1" rowspan="1">Analyte</td><td colspan="1" rowspan="1">Fresh</td><td colspan="1" rowspan="1">RT 4h</td><td colspan="1" rowspan="1">2 to 8 °C 72h</td><td colspan="1" rowspan="1">-15 to -25 °C 30d</td></tr><tr><td colspan="1" rowspan="6">Mix 1</td><td colspan="1" rowspan="1">Influenza A</td><td colspan="1" rowspan="1">9/10</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">9/10</td><td colspan="1" rowspan="1">10/10</td></tr><tr><td colspan="1" rowspan="1">Influenza A H1</td><td colspan="1" rowspan="1">9/10</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">10/10</td></tr><tr><td colspan="1" rowspan="1">Cor HKU1</td><td colspan="1" rowspan="1">9/10</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">9/10</td><td colspan="1" rowspan="1">10/10</td></tr><tr><td colspan="1" rowspan="1">PIV 2</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">9/10</td><td colspan="1" rowspan="1">10/10</td></tr><tr><td colspan="1" rowspan="1">RSV B</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">10/10</td></tr><tr><td colspan="1" rowspan="1">C. pneumoniae</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">10/10</td></tr><tr><td colspan="1" rowspan="6">Mix 2</td><td colspan="1" rowspan="1">Influenza B</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">10/10</td></tr><tr><td colspan="1" rowspan="1">Cor 229E</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">10/10</td></tr><tr><td colspan="1" rowspan="1">PIV 4</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">10/10</td></tr><tr><td colspan="1" rowspan="1">Enterovirus D68</td><td colspan="1" rowspan="1">9/10</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">9/10</td><td colspan="1" rowspan="1">10/10</td></tr><tr><td colspan="1" rowspan="1">hMPV A1</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">10/10</td></tr><tr><td colspan="1" rowspan="1">B. pertussis</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">10/10</td></tr><tr><td colspan="1" rowspan="5">Mix 3</td><td colspan="1" rowspan="1">Influenza A</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">10/10</td></tr><tr><td colspan="1" rowspan="1">Influenza H1N1pdm09</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">10/10</td></tr><tr><td colspan="1" rowspan="1">Cor OC43</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">10/10</td></tr><tr><td colspan="1" rowspan="1">PIV 3</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">9/10</td><td colspan="1" rowspan="1">10/10</td></tr><tr><td colspan="1" rowspan="1">Rhinovirus A2</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">10/10</td></tr><tr><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="1">RSV A</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">10/10</td></tr><tr><td colspan="1" rowspan="1">M. pneumoniae</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">10/10</td></tr><tr><td colspan="1" rowspan="5">Mix 4</td><td colspan="1" rowspan="1">Influenza A</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">9/10</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">10/10</td></tr><tr><td colspan="1" rowspan="1">Influenza A H3</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">9/10</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">10/10</td></tr><tr><td colspan="1" rowspan="1">Cor NL63</td><td colspan="1" rowspan="1">9/10</td><td colspan="1" rowspan="1">9/10</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">10/10</td></tr><tr><td colspan="1" rowspan="1">PIV 1</td><td colspan="1" rowspan="1">9/10</td><td colspan="1" rowspan="1">9/10</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">10/10</td></tr><tr><td colspan="1" rowspan="1">Adenovirus B3</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">9/10</td><td colspan="1" rowspan="1">9/10</td><td colspan="1" rowspan="1">10/10</td></tr></table>

While some variance is expected due to the fact that $1 \mathrm { x } \mathrm { L O D }$ is essentially equivalent to a $9 5 \%$ detection rate for the analyte, the acceptance criteria was not met for all analytes and all time points/conditions when tested at the 1x LOD concentration.

Table 7b: Specimen Stability Study Results – 1x LOD   

<table><tr><td rowspan=1 colspan=1>Mix</td><td rowspan=1 colspan=1>Analyte</td><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>RT 4h</td><td rowspan=1 colspan=1>2 to 8 °C 72h</td><td rowspan=1 colspan=1>-15 to -25 °C 30d</td></tr><tr><td rowspan=6 colspan=1>Mix 5</td><td rowspan=1 colspan=1>Influenza A</td><td rowspan=1 colspan=1>6/10</td><td rowspan=1 colspan=1>7/10</td><td rowspan=1 colspan=1>9/10</td><td rowspan=1 colspan=1>9/10</td></tr><tr><td rowspan=1 colspan=1>Influenza A H1</td><td rowspan=1 colspan=1>8/10</td><td rowspan=1 colspan=1>9/10</td><td rowspan=1 colspan=1>9/10</td><td rowspan=1 colspan=1>9/10</td></tr><tr><td rowspan=1 colspan=1>Cor HKU1</td><td rowspan=1 colspan=1>8/10</td><td rowspan=1 colspan=1>9/10</td><td rowspan=1 colspan=1>9/10</td><td rowspan=1 colspan=1>10/10</td></tr><tr><td rowspan=1 colspan=1>PIV 2</td><td rowspan=1 colspan=1>9/10</td><td rowspan=1 colspan=1>9/10</td><td rowspan=1 colspan=1>9/10</td><td rowspan=1 colspan=1>9/10</td></tr><tr><td rowspan=1 colspan=1>RSV B</td><td rowspan=1 colspan=1>9/10</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>9/10</td><td rowspan=1 colspan=1>10/10</td></tr><tr><td rowspan=1 colspan=1>C. pneumoniae</td><td rowspan=1 colspan=1>9/10</td><td rowspan=1 colspan=1>8/10</td><td rowspan=1 colspan=1>7/10</td><td rowspan=1 colspan=1>10/10</td></tr><tr><td rowspan=6 colspan=1>Mix 6</td><td rowspan=1 colspan=1>Influenza B</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>9/10</td><td rowspan=1 colspan=1>8/10</td><td rowspan=1 colspan=1>10/10</td></tr><tr><td rowspan=1 colspan=1>Cor 229E</td><td rowspan=1 colspan=1>8/10</td><td rowspan=1 colspan=1>9/10</td><td rowspan=1 colspan=1>9/10</td><td rowspan=1 colspan=1>10/10</td></tr><tr><td rowspan=1 colspan=1>PIV 4</td><td rowspan=1 colspan=1>9/10</td><td rowspan=1 colspan=1>9/10</td><td rowspan=1 colspan=1>9/10</td><td rowspan=1 colspan=1>10/10</td></tr><tr><td rowspan=1 colspan=1>Enterovirus D68</td><td rowspan=1 colspan=1>7/10</td><td rowspan=1 colspan=1>9/10</td><td rowspan=1 colspan=1>6/10</td><td rowspan=1 colspan=1>9/10</td></tr><tr><td rowspan=1 colspan=1>hMPV A1</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>9/10</td><td rowspan=1 colspan=1>6/10</td><td rowspan=1 colspan=1>10/10</td></tr><tr><td rowspan=1 colspan=1>B. pertussis</td><td rowspan=1 colspan=1>7/10</td><td rowspan=1 colspan=1>8/10</td><td rowspan=1 colspan=1>9/10</td><td rowspan=1 colspan=1>10/10</td></tr><tr><td rowspan=7 colspan=1>Mix 7</td><td rowspan=1 colspan=1>Influenza A</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>10/10</td></tr><tr><td rowspan=1 colspan=1>Influenza H1N1pdm09</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>8/10</td><td rowspan=1 colspan=1>9/10</td><td rowspan=1 colspan=1>10/10</td></tr><tr><td rowspan=1 colspan=1>Cor OC43</td><td rowspan=1 colspan=1>9/10</td><td rowspan=1 colspan=1>9/10</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>10/10</td></tr><tr><td rowspan=1 colspan=1>PIV 3</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>9/10</td><td rowspan=1 colspan=1>9/10</td><td rowspan=1 colspan=1>10/10</td></tr><tr><td rowspan=1 colspan=1>Rhinovirus A2</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>9/10</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>10/10</td></tr><tr><td rowspan=1 colspan=1>RSV A</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>9/10</td><td rowspan=1 colspan=1>9/10</td><td rowspan=1 colspan=1>10/10</td></tr><tr><td rowspan=1 colspan=1>M. pneumoniae</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>8/10</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>10/10</td></tr><tr><td rowspan=5 colspan=1>Mix 8</td><td rowspan=1 colspan=1>Influenza A</td><td rowspan=1 colspan=1>8/10</td><td rowspan=1 colspan=1>9/10</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>9/10</td></tr><tr><td rowspan=1 colspan=1>Influenza A H3</td><td rowspan=1 colspan=1>9/10</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>9/10</td><td rowspan=1 colspan=1>10/10</td></tr><tr><td rowspan=1 colspan=1>Cor NL63</td><td rowspan=1 colspan=1>8/10</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>9/10</td></tr><tr><td rowspan=1 colspan=1>PIV 1</td><td rowspan=1 colspan=1>7/10</td><td rowspan=1 colspan=1>8/10</td><td rowspan=1 colspan=1>8/10</td><td rowspan=1 colspan=1>9/10</td></tr><tr><td rowspan=1 colspan=1>Adenovirus B3</td><td rowspan=1 colspan=1>4/10</td><td rowspan=1 colspan=1>7/10</td><td rowspan=1 colspan=1>9/10</td><td rowspan=1 colspan=1>9/10</td></tr></table>

Supplemental testing was performed for select analytes at select conditions at the 1x LOD concentration to further test specimen stability where acceptance criteria had not been previously met.

Table 7c: Additional Specimen Stability Study Results – 1x LOD   

<table><tr><td rowspan=1 colspan=1>Mix</td><td rowspan=1 colspan=1>Analyte</td><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>RT 4h</td><td rowspan=1 colspan=1>2 to 8 °C 72h</td><td rowspan=1 colspan=1>-15 to -25 °C 30d</td></tr><tr><td rowspan=3 colspan=1>Mix 9</td><td rowspan=1 colspan=1>Influenza A</td><td rowspan=1 colspan=1>9/10</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>nd</td><td rowspan=1 colspan=1>nd</td></tr><tr><td rowspan=1 colspan=1>Influenza A H1</td><td rowspan=1 colspan=1>7/10</td><td rowspan=1 colspan=1>9/10</td><td rowspan=1 colspan=1>nd</td><td rowspan=1 colspan=1>nd</td></tr><tr><td rowspan=1 colspan=1>Adenovirus B3</td><td rowspan=1 colspan=1>8/10</td><td rowspan=1 colspan=1>9/10</td><td rowspan=1 colspan=1>nd</td><td rowspan=1 colspan=1>nd</td></tr><tr><td rowspan=1 colspan=1>Mix 10</td><td rowspan=1 colspan=1>Enterovirus D68</td><td rowspan=1 colspan=1>9/10</td><td rowspan=1 colspan=1>nd</td><td rowspan=1 colspan=1>8/10</td><td rowspan=1 colspan=1>nd</td></tr></table>

<table><tr><td rowspan=1 colspan=1>hMPV A1</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>nd</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>nd</td></tr><tr><td rowspan=1 colspan=1>C. pneumoniae</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>nd</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>nd</td></tr></table>

nd – Testing not done.

# Shelf Life

The stated shelf life for the QIAstat-Dx Respiratory Panel is 12 months when stored at 15 to $2 5 ~ ^ { \circ } \mathrm { C }$ . Stability data to support the proposed shelf life and shipping conditions was obtained by testing three separate production lots of QIAstat-Dx cartridges using QC material under a real time stability study. The study was performed at room temperature at 30-day intervals spanning a planned 13-month period. Data generation is ongoing and is being maintained under Qiagen’s Quality Systems internal protocols.

# d. Detection limit:

The objective of the Analytical Sensitivity Study was to identify the limit of detection (LOD) of the QIAstat-Dx Respiratory Panel prepared from high-titer stocks obtained from commercial suppliers or clinical isolates for commercially unavailable target analytes. For the purposes of the study, the LOD level was defined as the concentration of analyte that produced positive QIAstat-Dx Respiratory Panel test results approximately $9 5 \%$ of the time when tested in multiple replicates.

The LOD was assessed in a two-step process for every analyte. The first step was setup of a preliminary LOD by testing serial dilutions for every pathogen in a minimum of 3 log serial dilutions around the expected LOD. For this step of the study, samples were prepared in a simulated matrix consisting of UTM plus HeLa cells and four replicates of each analyte were tested. The data obtained from the first step was then used to choose the concentration of analyte likely to provide a minimum of 19 out of 20 positive results for the LOD confirmation. Confirmed LOD concentrations prepared in simulated matrix were then verified by testing the analytes in clinical matrix whereby at least 19 out of 20 replicates were again detected by the assay (see “matrix equivalency” below). If the assay failed to verify the LOD in clinical matrix, a sample of $1 0 \mathrm { x }$ more concentrated titer was prepared and retested for LOD verification in clinical matrix. The concentrations of analytes which provided at least 19/20 positive results in clinical matrix is listed as the claimed “LOD” in table 8.

Table 8: QIAstat-Dx Respiratory Panel Limits of Detection   

<table><tr><td rowspan=1 colspan=1>Pathogen</td><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>LOD</td><td rowspan=1 colspan=1>Units</td></tr><tr><td rowspan=3 colspan=1>Influenza A H1N1</td><td rowspan=1 colspan=1>A/NJ/8/76</td><td rowspan=1 colspan=1>341</td><td rowspan=1 colspan=1>CEID50/mL</td></tr><tr><td rowspan=1 colspan=1>A/Brisbane/59/07</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>A/New Caldonia/20/99</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>TCID50/mL</td></tr><tr><td rowspan=3 colspan=1>Influenza A H3N2</td><td rowspan=1 colspan=1>A/Virginia/ATCC6/2012</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>PFU/mL</td></tr><tr><td rowspan=1 colspan=1>A/Wisconsin/67/2005</td><td rowspan=1 colspan=1>3.8</td><td rowspan=1 colspan=1>TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>A/Port Chalmers/1/73</td><td rowspan=1 colspan=1>499</td><td rowspan=1 colspan=1>CEID50/mL</td></tr><tr><td rowspan=2 colspan=1>Influenza A H1N1/pdm09</td><td rowspan=1 colspan=1>A/Virginia/ATCC1/2009</td><td rowspan=1 colspan=1>6.7</td><td rowspan=1 colspan=1>PFU/mL</td></tr><tr><td rowspan=1 colspan=1>A/SwineNY/03/2009</td><td rowspan=1 colspan=1>5.6</td><td rowspan=1 colspan=1>TCID50/mL</td></tr><tr><td rowspan=3 colspan=1>Influenza B</td><td rowspan=1 colspan=1>B/Virginia/ATCC5/2012</td><td rowspan=1 colspan=1>0.03</td><td rowspan=1 colspan=1>PFU/mL</td></tr><tr><td rowspan=1 colspan=1>B/FL/04/06</td><td rowspan=1 colspan=1>1080</td><td rowspan=1 colspan=1>CEID50/mL</td></tr><tr><td rowspan=1 colspan=1>B/Taiwan/2/62</td><td rowspan=1 colspan=1>5000</td><td rowspan=1 colspan=1>CEID50/mL</td></tr><tr><td rowspan=2 colspan=1>Coronavirus 229E</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>3.6</td><td rowspan=1 colspan=1>TCID50/mL</td></tr><tr><td rowspan=2 colspan=1>Coronavirus OC43</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Coronavirus NL63</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>0.01</td><td rowspan=1 colspan=1>TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Coronavirus HKU1</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>40,000</td><td rowspan=1 colspan=1>Copies/mL</td></tr><tr><td rowspan=2 colspan=1>Parainfluenza Virus 1</td><td rowspan=1 colspan=1>C35</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>TCID50/mL</td></tr><tr><td rowspan=2 colspan=1>Parainfluenza Virus 2</td><td rowspan=1 colspan=1>Greer</td><td rowspan=1 colspan=1>7.3</td><td rowspan=1 colspan=1>TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>1.3</td><td rowspan=1 colspan=1>TCID50/mL</td></tr><tr><td rowspan=2 colspan=1>Parainfluenza Virus 3</td><td rowspan=1 colspan=1>C 243</td><td rowspan=1 colspan=1>2.3</td><td rowspan=1 colspan=1>TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>11.5</td><td rowspan=1 colspan=1>TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza Virus 4a</td><td rowspan=1 colspan=1>M-25</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza Virus 4b</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>9.5</td><td rowspan=1 colspan=1>TCID50/mL</td></tr><tr><td rowspan=2 colspan=1>RSV A</td><td rowspan=1 colspan=1>A2</td><td rowspan=1 colspan=1>12.0</td><td rowspan=1 colspan=1>PFU/mL</td></tr><tr><td rowspan=1 colspan=1>Long</td><td rowspan=1 colspan=1>33.0</td><td rowspan=1 colspan=1>PFU/mL</td></tr><tr><td rowspan=2 colspan=1>RSV B</td><td rowspan=1 colspan=1>18537</td><td rowspan=1 colspan=1>0.03</td><td rowspan=1 colspan=1>PFU/mL</td></tr><tr><td rowspan=1 colspan=1>CH93(18)-18</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>TCID50/mL</td></tr><tr><td rowspan=4 colspan=1>Human metapneumovirus</td><td rowspan=1 colspan=1>Peru6-2003</td><td rowspan=1 colspan=1>0.01</td><td rowspan=1 colspan=1>TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>IA10-2003</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>IA14-2003</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Peru2-2002</td><td rowspan=1 colspan=1>1480</td><td rowspan=1 colspan=1>TCID50/mL</td></tr><tr><td rowspan=6 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>GB</td><td rowspan=1 colspan=1>4993</td><td rowspan=1 colspan=1>TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>RI-67</td><td rowspan=1 colspan=1>15.8</td><td rowspan=1 colspan=1>TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Adenoid 75</td><td rowspan=1 colspan=1>7331</td><td rowspan=1 colspan=1>TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Adenoid 71</td><td rowspan=1 colspan=1>69.5</td><td rowspan=1 colspan=1>TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Adenoid 6</td><td rowspan=1 colspan=1>28.1</td><td rowspan=1 colspan=1>TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Tonsil 99</td><td rowspan=1 colspan=1>88.8</td><td rowspan=1 colspan=1>TCID50/mL</td></tr><tr><td rowspan=2 colspan=1>Enterovirus</td><td rowspan=1 colspan=1>US/IL/14-18952</td><td rowspan=1 colspan=1>8.9</td><td rowspan=1 colspan=1>TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Echovirus 6</td><td rowspan=1 colspan=1>0.9</td><td rowspan=1 colspan=1>TCID50/mL</td></tr><tr><td rowspan=4 colspan=1>Rhinovirus</td><td rowspan=1 colspan=1>1059</td><td rowspan=1 colspan=1>8.9</td><td rowspan=1 colspan=1>TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>HGP</td><td rowspan=1 colspan=1>8.9</td><td rowspan=1 colspan=1>TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>11757</td><td rowspan=1 colspan=1>50.0</td><td rowspan=1 colspan=1>TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Type 1A</td><td rowspan=1 colspan=1>8.9</td><td rowspan=1 colspan=1>TCID50/mL</td></tr><tr><td rowspan=2 colspan=1>Mycoplasma pneumoniae</td><td rowspan=1 colspan=1>M129-B7</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>CCU/mL</td></tr><tr><td rowspan=1 colspan=1>PI 1428</td><td rowspan=1 colspan=1>1.0</td><td rowspan=1 colspan=1>CCU/mL</td></tr><tr><td rowspan=2 colspan=1>Chlamydophilapneumoniae</td><td rowspan=1 colspan=1>TW183</td><td rowspan=1 colspan=1>14.2</td><td rowspan=1 colspan=1>IFU/mL</td></tr><tr><td rowspan=1 colspan=1>CWL-029</td><td rowspan=1 colspan=1>120</td><td rowspan=1 colspan=1>IFU/mL</td></tr><tr><td rowspan=2 colspan=1>Bordetella pertussis</td><td rowspan=1 colspan=1>I028</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>CFU/mL</td></tr><tr><td rowspan=1 colspan=1>18323</td><td rowspan=1 colspan=1>2.6</td><td rowspan=1 colspan=1>CFU/mL</td></tr></table>

# e. Analytical reactivity:

Various respiratory panel virus and bacteria strains were tested to examine the ability of the QIAstat-Dx Respiratory Panel to detect a wide variety of analyte strains in a clinical setting. Samples were prepared in a simulated matrix consisting of UTM plus HeLa cells and tested according to the package insert.

Log dilutions of each analyte were prepared as sample mixes containing multiple analytes in each mix. All analyte dilutions were run in triplicate. Acceptance criterion for the study was $3 / 3$ positive results for the analyte being tested. If the criterion was not met, a $1 0 \mathrm { x }$ more concentrated titer of the analyte was tested in triplicate. Results in tables 9 - 20 show the analyte, strain, and concentration at which the acceptance criterion was met. The lowest level of each strain that generated positive results on all three replicates was identified as the lowest level detected by the QIAstat-Dx Respiratory Panel.

Table 9: QIAstat-Dx RP Analytical Reactivity Results for Influenza A H1   

<table><tr><td rowspan=1 colspan=1>Analyte</td><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>Concentration</td><td rowspan=1 colspan=1>Detected /Tested</td><td rowspan=1 colspan=1>QIAstat-Dx RPResult</td></tr><tr><td rowspan=10 colspan=1>Influenza A H1N1</td><td rowspan=1 colspan=1>A/Brisbane/59/07</td><td rowspan=1 colspan=1>0.4 TCID50/mL</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>Influenza A H1</td></tr><tr><td rowspan=1 colspan=1>A/New Caldonia/20/99</td><td rowspan=1 colspan=1>1.5 TCID50/mL</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>Influenza A H1</td></tr><tr><td rowspan=1 colspan=1>A/NJ/8/76</td><td rowspan=1 colspan=1>34.1 CEID50/mL</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>Influenza A H1</td></tr><tr><td rowspan=1 colspan=1>A/Denver/1/57</td><td rowspan=1 colspan=1>340 CEID50/mL</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>Influenza A H1</td></tr><tr><td rowspan=1 colspan=1>A/Mal/302/54</td><td rowspan=1 colspan=1>15.8 CEID50/mL</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>Influenza A H1</td></tr><tr><td rowspan=1 colspan=1>A/Weiss/43</td><td rowspan=1 colspan=1>28117 CEID50/mL</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>Influenza A H1</td></tr><tr><td rowspan=1 colspan=1>A/PR/8/34</td><td rowspan=1 colspan=1>390 PFU/mL</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>Influenza A H1</td></tr><tr><td rowspan=1 colspan=1>A/Fort Monmouth/1/1947</td><td rowspan=1 colspan=1>28.1 CEID50/mL</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>Influenza A H1</td></tr><tr><td rowspan=1 colspan=1>A/WS/33</td><td rowspan=1 colspan=1>15.8 TCID50/mL</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>Influenza A H1</td></tr><tr><td rowspan=1 colspan=1>A/Swine/Iowa/15/1930</td><td rowspan=1 colspan=1>889 CEID50/mL</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>Influenza A H1</td></tr></table>

Table 10: QIAstat-Dx RP Analytical Reactivity Results for Influenza A H3   

<table><tr><td rowspan=1 colspan=1>Analyte</td><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>Concentration</td><td rowspan=1 colspan=1>Detected /Tested</td><td rowspan=1 colspan=1>QIAstat-DxRP Result</td></tr><tr><td rowspan=10 colspan=1>Influenza A H3N2</td><td rowspan=1 colspan=1>A/Port Chalmers/1/73</td><td rowspan=1 colspan=1>499 CEID50/mL</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>Influenza A H3</td></tr><tr><td rowspan=1 colspan=1>A/Virginia/ATCC6/2012</td><td rowspan=1 colspan=1>0.1 PFU/mL</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>Influenza A H3</td></tr><tr><td rowspan=1 colspan=1>A/Wisconsin/67/2005</td><td rowspan=1 colspan=1>3.8 TCID50/mL</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>Influenza A H3</td></tr><tr><td rowspan=1 colspan=1>A/Wisconsin/15/2009</td><td rowspan=1 colspan=1>5.8 CEID50/mL</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>Influenza A H3</td></tr><tr><td rowspan=1 colspan=1>A/Victoria/3/75</td><td rowspan=1 colspan=1>16 CEID50/mL</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>Influenza A H3</td></tr><tr><td rowspan=1 colspan=1>A/Aichi/2/68</td><td rowspan=1 colspan=1>31 PFU/mL</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>Influenza A H3</td></tr><tr><td rowspan=1 colspan=1>A/Hong Kong/8/68</td><td rowspan=1 colspan=1>1581 TCID50/mL</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>Influenza A H3</td></tr><tr><td rowspan=1 colspan=1>A/Alice1</td><td rowspan=1 colspan=1>500 TCID50/mL</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>Influenza A H3</td></tr><tr><td rowspan=1 colspan=1>MRC-22</td><td rowspan=1 colspan=1>8891 CEID50/mL</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>Influenza A H3</td></tr><tr><td rowspan=1 colspan=1>A/Switzerland/9715293/2013</td><td rowspan=1 colspan=1>1000 CEID50/mL</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>Influenza A H3</td></tr></table>

1 Recombinant strain; carries A/England/42/72 genes 2 Recombinant strain; carries A/England/42/72 and A/PR/8/34 genes

Table 11: QIAstat-Dx RP Analytical Reactivity Results for Influenza A H1N1 pdm09   

<table><tr><td rowspan=1 colspan=1>Analyte</td><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>Concentration</td><td rowspan=1 colspan=1>Detected /Tested</td><td rowspan=1 colspan=1>QIAstat-DxRP Result</td></tr><tr><td rowspan=10 colspan=1>Influenza A H1N1pdm09</td><td rowspan=1 colspan=1>A/Virginia/ATCC1/2009</td><td rowspan=1 colspan=1>6.7 PFU/mL</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>Influenza AH1N1/pdm09</td></tr><tr><td rowspan=1 colspan=1>A/SwineNY/03/2009</td><td rowspan=1 colspan=1>5.6 TCID50/mL</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>Influenza AH1N1/pdm09</td></tr><tr><td rowspan=1 colspan=1>A/Virginia/ATCC2/2009</td><td rowspan=1 colspan=1>61 PFU/mL</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>Influenza AH1N1/pdm09</td></tr><tr><td rowspan=1 colspan=1>A/Virginia/ATCC3/2009</td><td rowspan=1 colspan=1>1800 PFU/mL</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>Influenza AH1N1/pdm09</td></tr><tr><td rowspan=1 colspan=1>Swine NY/01/2009</td><td rowspan=1 colspan=1>138 TCID50/mL</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>Influenza AH1N1/pdm09</td></tr><tr><td rowspan=1 colspan=1>Swine NY/02/2009</td><td rowspan=1 colspan=1>1.4 TCID50/mL</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>Influenza AH1N1/pdm09</td></tr><tr><td rowspan=1 colspan=1>A/California/07/2009</td><td rowspan=1 colspan=1>1400 CEID50/mL</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>Influenza AH1N1/pdm09</td></tr><tr><td rowspan=1 colspan=1>Canada/6294/09</td><td rowspan=1 colspan=1>1.7 TCID50/mL</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>Influenza AH1N1/pdm09</td></tr><tr><td rowspan=1 colspan=1>Mexico/4108/09</td><td rowspan=1 colspan=1>14.1 TCID50/mL</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>Influenza AH1N1/pdm09</td></tr><tr><td rowspan=1 colspan=1>Netherlands/2629/2009</td><td rowspan=1 colspan=1>16 TCID50/mL</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>Influenza AH1N1/pdm09</td></tr></table>

Table 12: QIAstat-Dx RP Analytical Reactivity Results for Influenza A   

<table><tr><td rowspan=1 colspan=1>Analyte</td><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>Concentration</td><td rowspan=1 colspan=1>Detected /Tested</td><td rowspan=1 colspan=1>QIAstat-DxRP Result</td></tr><tr><td rowspan=2 colspan=1>Influenza A H2N2</td><td rowspan=1 colspan=1>Japan/305/19571</td><td rowspan=1 colspan=1>3.26 x 10-3 ng/μL</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>Influenza A (nosubtype</td></tr><tr><td rowspan=1 colspan=1>Korea/426/19682</td><td rowspan=1 colspan=1>6.25 x 10-5 ng/μL</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>Influenza A (nosubtype</td></tr><tr><td rowspan=1 colspan=1>Influenza A H5N3</td><td rowspan=1 colspan=1>A/Duck/Singapore/645/19973</td><td rowspan=1 colspan=1>2.48 x 10-3 ng/μL</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>Influenza A (nosubtype</td></tr><tr><td rowspan=1 colspan=1>Influenza AH10N7</td><td rowspan=1 colspan=1>Chicken/Germany/N/493</td><td rowspan=1 colspan=1>6.80 x 10-2 ng/μL</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>Influenza A (nosubtype)</td></tr><tr><td rowspan=1 colspan=1>Influenza A H1N2</td><td rowspan=1 colspan=1>Recombinant Kilbourne F634</td><td rowspan=1 colspan=1>1.48 x 10-2 ng/μL</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>Influenza A H1</td></tr></table>

1 Nucleic acid 2 Nucleic acid; recombinant cross with A/PR/8/34 3 Nucleic acid; avian source 4 Nucleic acid; recombinant cross of A/NWS/1934 x A/Rockefeller Institute/5/1957

Table 13: QIAstat-Dx RP Analytical Reactivity Results for Influenza B   

<table><tr><td rowspan=1 colspan=1>Analyte</td><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>Concentration</td><td rowspan=1 colspan=1>Detected /Tested</td><td rowspan=1 colspan=1>QIAstat-DxRP Result</td></tr><tr><td rowspan=12 colspan=1>Influenza B</td><td rowspan=1 colspan=1>B/Virginia/ATCC5/2012</td><td rowspan=1 colspan=1>0.03 PFU/mL</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>Influenza B</td></tr><tr><td rowspan=1 colspan=1>B/FL/04/06</td><td rowspan=1 colspan=1>108 CEID50/mL</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>Influenza B</td></tr><tr><td rowspan=1 colspan=1>B/Taiwan/2/62</td><td rowspan=1 colspan=1>49.9 CEID50/mL</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>Influenza B</td></tr><tr><td rowspan=1 colspan=1>B/Allen/45</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>Negative</td></tr><tr><td rowspan=1 colspan=1>B/Hong Kong/5/72</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>Negative</td></tr><tr><td rowspan=1 colspan=1>B/Maryland/1/59</td><td rowspan=1 colspan=1>338 CEID50/mL</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>Influenza B</td></tr><tr><td rowspan=1 colspan=1>B/GL/1739/54</td><td rowspan=1 colspan=1>50.0 CEID50/mL</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>Influenza B</td></tr><tr><td rowspan=1 colspan=1>B/Wisconsin/1/2010</td><td rowspan=1 colspan=1>0.3 CEID50/mL</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>Influenza B</td></tr><tr><td rowspan=1 colspan=1>B/Massachusetts/2/2012</td><td rowspan=1 colspan=1>2300 CEID50/mL</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>Influenza B</td></tr><tr><td rowspan=1 colspan=1>B/Florida/02/06</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>1/3</td><td rowspan=1 colspan=1>Influenza B andNegative1</td></tr><tr><td rowspan=1 colspan=1>B/Brisbane/60/2008</td><td rowspan=1 colspan=1>1.8 CEID50/mL</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>Influenza B</td></tr><tr><td rowspan=1 colspan=1>B/Malaysia/2506/2004</td><td rowspan=1 colspan=1>1.58 CEID50/mL</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>Influenza B</td></tr></table>

1 Multiple dilutions and repeats failed to achieve acceptance criterion of 3/3

Table 14: QIAstat-Dx RP Analytical Reactivity Results for Coronavirus   

<table><tr><td rowspan=1 colspan=1>Analyte</td><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>Concentration</td><td rowspan=1 colspan=1>Detected /Tested</td><td rowspan=1 colspan=1>QIAstat-Dx RPResult</td></tr><tr><td rowspan=2 colspan=1>Coronavirus229E</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>3.6 TCID50/mL</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>Coronavirus 229E</td></tr><tr><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>0.2 TCID50/mL</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>Coronavirus 229E</td></tr><tr><td rowspan=2 colspan=1>CoronavirusOC43</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>0.1 TCID50/mL</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>Coronavirus OC43</td></tr><tr><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>0.1 TCID50/mL</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>Coronavirus OC43</td></tr><tr><td rowspan=2 colspan=1>CoronavirusNL63</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>0.01 TCID50/mL</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>Coronavirus NL63</td></tr><tr><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>1.6 TCID50/mL</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>Coronavirus NL63</td></tr><tr><td rowspan=4 colspan=1>CoronavirusHKU1</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>3.0 x104 copies/mL</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>Coronavirus HKU1</td></tr><tr><td rowspan=1 colspan=1>Clinical isolate</td><td rowspan=1 colspan=1>4.0 x108 copies/mL</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>Coronavirus HKU1</td></tr><tr><td rowspan=1 colspan=1>Clinical isolate</td><td rowspan=1 colspan=1>7.0 x107 copies/mL</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>Coronavirus HKU1</td></tr><tr><td rowspan=1 colspan=1>Clinical isolate</td><td rowspan=1 colspan=1>7.0 x107 copies/mL</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>Coronavirus HKU1</td></tr></table>

Table 15: QIAstat-Dx RP Analytical Reactivity Results for Parainfluenza Virus   

<table><tr><td rowspan=1 colspan=2>Analyte</td><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>Concentration</td><td rowspan=1 colspan=1>Detected /Tested</td><td rowspan=1 colspan=1>QIAstat-DxRP Result</td></tr><tr><td rowspan=3 colspan=2>ParainfluenzaVirus 1</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>0.02 TCID50/mL</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>PIV 1</td></tr><tr><td rowspan=1 colspan=1>C35</td><td rowspan=1 colspan=1>0.2 TCID50/mL</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>PIV 1</td></tr><tr><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>n/a1</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>PIV 1</td></tr><tr><td rowspan=3 colspan=2>ParainfluenzaVirus 2</td><td rowspan=1 colspan=1>Greer</td><td rowspan=1 colspan=1>2.3 TCID50/mL</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>PIV 2</td></tr><tr><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>1.3 TCID50/mL</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>PIV 2</td></tr><tr><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>1.3 TCID50/mL</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>PIV 2</td></tr><tr><td rowspan=3 colspan=2>ParainfluenzaVirus 3</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>11.5 TCID50/mL</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>PIV 3</td></tr><tr><td rowspan=1 colspan=1>C 243</td><td rowspan=1 colspan=1>2.3 TCID50/mL</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>PIV 3</td></tr><tr><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>n/a1</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>PIV 3</td></tr><tr><td rowspan=4 colspan=2>ParainfluenzaVirus 4</td><td rowspan=1 colspan=1>M-25</td><td rowspan=1 colspan=1>0.5 TCID50/mL</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>PIV 4</td></tr><tr><td rowspan=2 colspan=1></td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>9.6 TCID50/mL</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>PIV 4</td></tr><tr><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>28.2 TCID50/mL</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>PIV 4</td></tr><tr><td rowspan=1 colspan=1>CH 19503</td><td rowspan=1 colspan=1>1 TCID50/mL</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>PIV 4</td></tr></table>

1 Stock titer not available from supplier.

Table 16: QIAstat-Dx RP Analytical Reactivity Results for RSV $\mathbf { A } { + } \mathbf { B }$   

<table><tr><td rowspan=1 colspan=1>Analyte</td><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>Concentration</td><td rowspan=1 colspan=1>Detected /Tested</td><td rowspan=1 colspan=1>QIAstat-DxRP Result</td></tr><tr><td rowspan=6 colspan=1>RespiratorySyncytial VirusA+B</td><td rowspan=1 colspan=1>18537</td><td rowspan=1 colspan=1>0.03 PFU/mL</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>RSV A+B</td></tr><tr><td rowspan=1 colspan=1>A2</td><td rowspan=1 colspan=1>12 PFU/mL</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>RSV A+B</td></tr><tr><td rowspan=1 colspan=1>Long</td><td rowspan=1 colspan=1>33 PFU/mL</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>RSV A+B</td></tr><tr><td rowspan=1 colspan=1>CH93(18)-18</td><td rowspan=1 colspan=1>0.4 TCID50/mL</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>RSV A+B</td></tr><tr><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>0.3 TCID50/mL</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>RSV A+B</td></tr><tr><td rowspan=1 colspan=1>B WV/14617/85</td><td rowspan=1 colspan=1>15.8 TCID50/mL</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>RSV A+B</td></tr></table>

Table 17: QIAstat-Dx RP Analytical Reactivity Results for hMPV $\mathbf { A } { + } \mathbf { B }$   

<table><tr><td rowspan=1 colspan=1>Analyte</td><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>Concentration</td><td rowspan=1 colspan=1>Detected /Tested</td><td rowspan=1 colspan=1>QIAstat-DxRP Result</td></tr><tr><td rowspan=9 colspan=1>HumanMetapneumovirus</td><td rowspan=1 colspan=1>IA10-2003</td><td rowspan=1 colspan=1>0.5 TCID50/mL</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>hMPV A+B</td></tr><tr><td rowspan=1 colspan=1>IA14-2003</td><td rowspan=1 colspan=1>0.4 TCID50/mL</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>hMPV A+B</td></tr><tr><td rowspan=1 colspan=1>Peru2-2002</td><td rowspan=1 colspan=1>1479 TCID50/mL</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>hMPV A+B</td></tr><tr><td rowspan=1 colspan=1>Peru6-2003</td><td rowspan=1 colspan=1>0.01 TCID50/mL</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>hMPV A+B</td></tr><tr><td rowspan=1 colspan=1>IA3-2002</td><td rowspan=1 colspan=1>66 TCID50/mL</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>hMPV A+B</td></tr><tr><td rowspan=1 colspan=1>IA27-2004</td><td rowspan=1 colspan=1>1.3 TCID50/mL</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>hMPV A+B</td></tr><tr><td rowspan=1 colspan=1>Peru3-2003</td><td rowspan=1 colspan=1>31.6 TCID50/mL</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>hMPV A+B</td></tr><tr><td rowspan=1 colspan=1>IA8-2003</td><td rowspan=1 colspan=1>0.4 TCID50/mL</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>hMPV A+B</td></tr><tr><td rowspan=1 colspan=1>Peru1-2002</td><td rowspan=1 colspan=1>2188 TCID50/mL</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>hMPV A+B</td></tr></table>

Table 18: QIAstat-Dx RP Analytical Reactivity Results for Adenovirus   

<table><tr><td rowspan=1 colspan=1>Analyte</td><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>Concentration</td><td rowspan=1 colspan=1>Detected /Tested</td><td rowspan=1 colspan=1>QIAstat-DxRP Result</td></tr><tr><td rowspan=15 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>Tonsil 99</td><td rowspan=1 colspan=1>88.8 TCID50/mL</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>Adenovirus</td></tr><tr><td rowspan=1 colspan=1>GB</td><td rowspan=1 colspan=1>4993 TCID50/mL</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>Adenovirus</td></tr><tr><td rowspan=1 colspan=1>Adenoid 71</td><td rowspan=1 colspan=1>69.5 TCID50/mL</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>Adenovirus</td></tr><tr><td rowspan=1 colspan=1>Adenoid 6</td><td rowspan=1 colspan=1>28.1 TCID50/mL</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>Adenovirus</td></tr><tr><td rowspan=1 colspan=1>Adenoid 75</td><td rowspan=1 colspan=1>7331 TCID50/mL</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>Adenovirus</td></tr><tr><td rowspan=1 colspan=1>RI-67</td><td rowspan=1 colspan=1>15.8 TCID50/mL</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>Adenovirus</td></tr><tr><td rowspan=1 colspan=1>Huie</td><td rowspan=1 colspan=1>88.9 TCID50/mL</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>Adenovirus</td></tr><tr><td rowspan=1 colspan=1>Gomen</td><td rowspan=1 colspan=1>0.3 TCID50/mL</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>Adenovirus</td></tr><tr><td rowspan=1 colspan=1>Slobitski</td><td rowspan=1 colspan=1>16 TCID50/mL</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>Adenovirus</td></tr><tr><td rowspan=1 colspan=1>AV-1645 [128]</td><td rowspan=1 colspan=1>2.8 TCID50/mL</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>Adenovirus</td></tr><tr><td rowspan=1 colspan=1>Compton</td><td rowspan=1 colspan=1>0.28 TCID50/mL</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>Adenovirus</td></tr><tr><td rowspan=1 colspan=1>Holden</td><td rowspan=1 colspan=1>8.9TCID50/mL</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>Adenovirus</td></tr><tr><td rowspan=1 colspan=1>Trim</td><td rowspan=1 colspan=1>160 TCID50/mL</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>Adenovirus</td></tr><tr><td rowspan=1 colspan=1>Dugan</td><td rowspan=1 colspan=1>0.2 TCID50/mL</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>Adenovirus</td></tr><tr><td rowspan=1 colspan=1>Tak (73-3544)</td><td rowspan=1 colspan=1>28117 TCID50/mL</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>Adenovirus</td></tr></table>

Table 19: QIAstat-Dx RP Analytical Reactivity Results for Rhinovirus/Enterovirus   

<table><tr><td rowspan=1 colspan=1>Analyte</td><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>Concentration</td><td rowspan=1 colspan=1>Detected /Tested</td><td rowspan=1 colspan=1>QIAstat-Dx RP Result</td></tr><tr><td rowspan=12 colspan=1>Enterovirus</td><td rowspan=1 colspan=1>US/IL/14-18952</td><td rowspan=1 colspan=1>8.9 TCID50/mL</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>Rhinovirus/Enterovirus</td></tr><tr><td rowspan=1 colspan=1>D-1 (Cox)</td><td rowspan=1 colspan=1>0.9 TCID50/mL</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>Rhinovirus/Enterovirus</td></tr><tr><td rowspan=1 colspan=1>H</td><td rowspan=1 colspan=1>8.9 TCID50/mL</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>Rhinovirus/Enterovirus</td></tr><tr><td rowspan=1 colspan=1>M.K. (Kowalik)</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>Rhinovirus/Enterovirus</td></tr><tr><td rowspan=1 colspan=1>Gregory</td><td rowspan=1 colspan=1>889 TCID50/mL</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>Rhinovirus/Enterovirus</td></tr><tr><td rowspan=1 colspan=1>Bastianni</td><td rowspan=1 colspan=1>281 TCID50/mL</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>Rhinovirus/Enterovirus</td></tr><tr><td rowspan=1 colspan=1>Griggs</td><td rowspan=1 colspan=1>1.6 TCID50/mL</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>Rhinovirus/Enterovirus</td></tr><tr><td rowspan=1 colspan=1>Conn-5</td><td rowspan=1 colspan=1>158 TCID50/mL</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>Rhinovirus/Enterovirus</td></tr><tr><td rowspan=1 colspan=1>Ohio-1</td><td rowspan=1 colspan=1>2812 TCID50/mL</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>Rhinovirus/Enterovirus</td></tr><tr><td rowspan=1 colspan=1>Nancy</td><td rowspan=1 colspan=1>0.9 TCID50/mL</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>Rhinovirus/Enterovirus</td></tr><tr><td rowspan=1 colspan=1>CHHE-29</td><td rowspan=1 colspan=1>0.03 TCID50/mL</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>Rhinovirus/Enterovirus</td></tr><tr><td rowspan=1 colspan=1>Kuykendall</td><td rowspan=1 colspan=1>28.1 TCID50/mL</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>Rhinovirus/Enterovirus</td></tr><tr><td rowspan=6 colspan=1>Rhinovirus</td><td rowspan=1 colspan=1>1059</td><td rowspan=1 colspan=1>8.9 TCID50/mL</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>Rhinovirus/Enterovirus</td></tr><tr><td rowspan=1 colspan=1>2060</td><td rowspan=1 colspan=1>8.9 TCID50/mL</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>Rhinovirus/Enterovirus</td></tr><tr><td rowspan=1 colspan=1>HGP</td><td rowspan=1 colspan=1>8.9 TCID50/mL</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>Rhinovirus/Enterovirus</td></tr><tr><td rowspan=1 colspan=1>11757</td><td rowspan=1 colspan=1>49.9 TCID50/mL</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>Rhinovirus/Enterovirus</td></tr><tr><td rowspan=1 colspan=1>FEB</td><td rowspan=1 colspan=1>281 TCID50/mL</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>Rhinovirus/Enterovirus</td></tr><tr><td rowspan=1 colspan=1>33342</td><td rowspan=1 colspan=1>200 PFU/mL</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>Rhinovirus/Enterovirus</td></tr></table>

Table 20: QIAstat-Dx RP Analytical Reactivity Results for Mycoplasma pneumoniae, Bordetella pertussis, and Chlamydophila pneumoniae   

<table><tr><td rowspan=1 colspan=1>Analyte</td><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>Concentration</td><td rowspan=1 colspan=1>Detected /Tested</td><td rowspan=1 colspan=1>QIAstat-DxRP Result</td></tr><tr><td rowspan=3 colspan=1>M. pneumoniae</td><td rowspan=1 colspan=1>PI 1428</td><td rowspan=1 colspan=1>1 CCU/mL</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>M. pneumoniae</td></tr><tr><td rowspan=1 colspan=1>M129-B7</td><td rowspan=1 colspan=1>0.1 CCU/mL</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>M. pneumoniae</td></tr><tr><td rowspan=1 colspan=1>FH</td><td rowspan=1 colspan=1>0.2 CCU/mL</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>M. pneumoniae</td></tr><tr><td rowspan=3 colspan=1>B. pertussis</td><td rowspan=1 colspan=1>I028</td><td rowspan=1 colspan=1>0.3 CFU/mL</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>B. pertussis</td></tr><tr><td rowspan=1 colspan=1>19323</td><td rowspan=1 colspan=1>2.6 CFU/mL</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>B. pertussis</td></tr><tr><td rowspan=1 colspan=1>10-536</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>B. pertussis</td></tr><tr><td rowspan=3 colspan=1>C. pneumoniae</td><td rowspan=1 colspan=1>TW183</td><td rowspan=1 colspan=1>14.2 IFU/mL</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>C. pneumoniae</td></tr><tr><td rowspan=1 colspan=1>CWL-029</td><td rowspan=1 colspan=1>120 IFU/mL</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>C. pneumoniae</td></tr><tr><td rowspan=1 colspan=1>AR-39</td><td rowspan=1 colspan=1>29 IFU/mL</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>C. pneumoniae</td></tr></table>

# f. Analytical specificity:

To determine the analytical specificity of the QIAstat-Dx Respiratory Panel, 21 onpanel pathogens and 52 off-panel pathogens were tested for any potential to crossreact with primers and probes specific for other analytes in the assay. Viral targets were tested at $1 0 ^ { 5 }$ units/mL and bacteria/fungal targets were tested at $1 0 ^ { 6 }$ units/mL wherever possible. Two off-panel bacterial targets were tested at lower concentrations due to limits of availability from the supplier: Bordetella hinzii and Legionella feeleii were tested at $5 . 0 \mathrm { ~ x ~ } 1 0 ^ { 3 } \mathrm { C F U / m L }$ and $1 . 0 \mathrm { { x } 1 0 ^ { 4 } \mathrm { { C F U / m L } } }$ , respectively. The on-panel and off-panel testing samples were prepared by single spiking organisms into simulated NPS sample matrix (UTM $^ +$ HeLa cells). All organisms were tested in triplicate using three different lots of QIAstat-Dx RP cartridges and up to 22 different analyzers. Acceptance criteria for the on-panel pathogens required all replicates to

provide a positive result for the specific target present in the sample and a negative result for all targets absent from the sample. Tables of on-panel and off-panel organisms used in this study are presented below.

Table 21: On-Panel Targets for QIAstat-Dx RP Analytical Specificity   

<table><tr><td rowspan=1 colspan=1>Pathogen</td><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>Concentration Tested</td></tr><tr><td rowspan=1 colspan=1>Influenza A H1N1</td><td rowspan=1 colspan=1>A/NJ/8/76</td><td rowspan=1 colspan=1>1.0 x105 CEID50/mL</td></tr><tr><td rowspan=1 colspan=1>Influenza A H3N2</td><td rowspan=1 colspan=1>A/Virginia/ATCC6/2012</td><td rowspan=1 colspan=1>1.0 x105 PFU/mL</td></tr><tr><td rowspan=1 colspan=1>Influenza A/2009/H1N1</td><td rowspan=1 colspan=1>A/Virginia/ATCC1/2009</td><td rowspan=1 colspan=1>1.0 x105 PFU/mL</td></tr><tr><td rowspan=1 colspan=1>Influenza B</td><td rowspan=1 colspan=1>B/FL/04/06</td><td rowspan=1 colspan=1>1.0 x105 CEID50/mL</td></tr><tr><td rowspan=1 colspan=1>Coronavirus 229E</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>1.0 x105 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Coronavirus OC43</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>1.0 x105 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Coronavirus NL63</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>1.0 x105 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Coronavirus HKU1</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>1.0 x105 Copies/mL</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza Virus 1</td><td rowspan=1 colspan=1>C35</td><td rowspan=1 colspan=1>1.0 x105 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza Virus 2</td><td rowspan=1 colspan=1>Greer</td><td rowspan=1 colspan=1>1.0 x105 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza Virus 3</td><td rowspan=1 colspan=1>C 243</td><td rowspan=1 colspan=1>1.0 x105 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza Virus 4</td><td rowspan=1 colspan=1>PIV 4a</td><td rowspan=1 colspan=1>1.0 x105 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>RSV A</td><td rowspan=1 colspan=1>A2</td><td rowspan=1 colspan=1>1.0 x105 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>hMPV</td><td rowspan=1 colspan=1>IA10-2003</td><td rowspan=1 colspan=1>1.0 x105 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>Adenoid 71</td><td rowspan=1 colspan=1>1.0 x105 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>Gomen</td><td rowspan=1 colspan=1>1.0 x105 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Enterovirus</td><td rowspan=1 colspan=1>US/IL/14-18952</td><td rowspan=1 colspan=1>1.0 x105 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Rhinovirus</td><td rowspan=1 colspan=1>Type 1A</td><td rowspan=1 colspan=1>1.0 x105 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Mycoplasma pneumoniae</td><td rowspan=1 colspan=1>M129</td><td rowspan=1 colspan=1>1.0 x106 CCU/mL</td></tr><tr><td rowspan=1 colspan=1>Bordetella pertussis</td><td rowspan=1 colspan=1>E431</td><td rowspan=1 colspan=1>1.0 x106 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Chlamydophila pneumoniae</td><td rowspan=1 colspan=1>AR-39</td><td rowspan=1 colspan=1>1.0 x106 IFU/mL</td></tr></table>

All results from the on-panel target list met the acceptance criteria at the concentrations tested.

Table 22: Off-Panel Targets for QIAstat-Dx RP Analytical Specificity   

<table><tr><td rowspan=1 colspan=1>Bacteria</td><td rowspan=1 colspan=1>Bacteria (continued)</td><td rowspan=1 colspan=1>Viruses</td><td rowspan=1 colspan=1>Fungi</td></tr><tr><td rowspan=1 colspan=1>Acinetobacter calcoaceticus</td><td rowspan=1 colspan=1>Moraxella catarrhalis</td><td rowspan=1 colspan=1>Bocavirus2</td><td rowspan=1 colspan=1>Aspergillus flavis</td></tr><tr><td rowspan=1 colspan=1>Bordetella avium</td><td rowspan=1 colspan=1>Mycobacterium tuberculosis1</td><td rowspan=1 colspan=1>Cytomegalovirus</td><td rowspan=1 colspan=1>Aspergillusfumigatus</td></tr><tr><td rowspan=1 colspan=1>Bordetella bronchioseptica</td><td rowspan=1 colspan=1>Mycoplasma hominis</td><td rowspan=1 colspan=1>Epstein-Barr virus</td><td rowspan=1 colspan=1>Candida albicans</td></tr><tr><td rowspan=1 colspan=1>Bordetella hinzii</td><td rowspan=1 colspan=1>Mycoplasma orale</td><td rowspan=1 colspan=1>HSV-1</td><td rowspan=1 colspan=1>Cryptococcusneoformans</td></tr><tr><td rowspan=1 colspan=1>Bordetella holmesii</td><td rowspan=1 colspan=1>Neisseria elongata</td><td rowspan=1 colspan=1>HSV-2</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Bordetella parapertussis</td><td rowspan=1 colspan=1>Neisseria gonorrhoeae</td><td rowspan=1 colspan=1>Measles virus</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Chlamydia trachomatis</td><td rowspan=1 colspan=1>Neisseria meningitidis</td><td rowspan=1 colspan=1>MERS CoV3</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Corynebacterium diphtheriae</td><td rowspan=1 colspan=1>Proteus mirabilis</td><td rowspan=1 colspan=1>Mumps virus</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Enterobacter aerogenes</td><td rowspan=1 colspan=1>Pseudomonas aeruginosa</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Escherichia coli</td><td rowspan=1 colspan=1>Serratia marcescens</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Haemophilus influenzae</td><td rowspan=1 colspan=1>Staphylococcus aureus</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Klebsiella pneumoniae</td><td rowspan=1 colspan=1>Staphylococcus epidermidis</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Klebsiella oxytoca</td><td rowspan=1 colspan=1>Stenotrophomonas maltophilia</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Lactobacillus acidophilus</td><td rowspan=1 colspan=1>Streptococcus agalactiae</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Lactobacillus plantarum</td><td rowspan=1 colspan=1>Streptococcus pneumonia</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Legionella bozemanii</td><td rowspan=1 colspan=1>Streptococcus salivarus</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Legionella dumofii</td><td rowspan=1 colspan=1>Streptococcus pyogenes</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Legionella feeleii</td><td rowspan=1 colspan=1>Ureaplasma urealyticum</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Legionella longbeacheae</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Legionella micdadei</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Legionella pneumophila</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr></table>

1 Genomic DNA 2 Clinical isolate 3 Synthetic RNA

The following false positive results were observed in the off-panel testing: Positive M. pneumoniae results were observed for Enterobacter aerogenes (1/9), Streptococcus pyogenes (2/9), and Aspergillus fumigatus (1/3); a positive Rhinovirus/Enterovirus result was observed for Legionella micdadei (1/6); and positive Bordetella pertussis results were observed for Bordetella bronchioseptica (3/3) and Bordetella holmesii (3/3). The unexpected positive results for $M .$ . pneumoniae may be due to contamination of the off-panel pathogen sources with $M .$ pneumoniae analyte. Cross-reactivity observed with other Bordetella species is likely due to the insertion sequence transposon which is the molecular target for Bordetella pertussis (IS481). IS481 is found in other Bordetella species, albeit typically in fewer copy numbers. A precaution has been added to the package insert warning of the possibility of cross-reactivity of non-pertussis species of Bordetella with the QIAstatDx Respiratory Panel test.

# g. Potentially Interfering Substances:

An analytical study was performed to assess the potential interference effects of 30 substances naturally present in respiratory specimens or that may be artificially introduced into the nasal cavity/nasopharynx. Samples were tested in triplicate with and without addition of the potentially inhibitory substance for direct sample-to

sample comparison. Sample mixes were prepared in a simulated matrix consisting of UTM plus HeLa cells to an initial concentration of 10x LOD. Following addition of an equal volume of UTM (control) or interferent the final test concentration was ${ 5 } \mathrm { x }$ LOD for all panel targets. All pathogen-containing samples without spiked substances generated positive signals for all pathogens present in the sample mixes. Positive influenza signals were generated when influenza vaccine (Fluenz Tetra and FluMist) was tested as a potential interferent. In addition to causing false positive results due the presence of live influenza virus in the vaccine ‘interferent’, it was observed that Fluenz Tetra and FluMist are capable of causing false negative results for QIAstat-Dx RP targets. A precaution has been added to the package insert warning that influenza vaccine present in the patient specimen may cause erroneous results. No other potentially interfering substances tested in this study were found to affect the accuracy of target detection for the QIAstat-Dx Respiratory Panel.

Table 23: QIAstat-Dx Respiratory Panel Interfering Substances Tested   

<table><tr><td rowspan=1 colspan=1>Substance</td><td rowspan=1 colspan=1>Concentration</td></tr><tr><td rowspan=1 colspan=1>Human genomic DNA</td><td rowspan=1 colspan=1>20 ng/μL</td></tr><tr><td rowspan=1 colspan=1>Whole Blood</td><td rowspan=1 colspan=1>1% (v/v)</td></tr><tr><td rowspan=1 colspan=1>Mucin</td><td rowspan=1 colspan=1>1% (v/v)</td></tr><tr><td rowspan=1 colspan=1>Tobramycin</td><td rowspan=1 colspan=1>0.6 mg/mL</td></tr><tr><td rowspan=1 colspan=1>Mupirocin</td><td rowspan=1 colspan=1>2% (w/v)</td></tr><tr><td rowspan=1 colspan=1>Saline nasal spray</td><td rowspan=1 colspan=1>1% (v/v)</td></tr><tr><td rowspan=1 colspan=1>Afrin nasal spray</td><td rowspan=1 colspan=1>1% (v/v)</td></tr><tr><td rowspan=1 colspan=1>Petroleum jelly</td><td rowspan=1 colspan=1>1% (w/v)</td></tr><tr><td rowspan=1 colspan=1>VicksAnalgesic ointment</td><td rowspan=1 colspan=1>1% (w/v)</td></tr><tr><td rowspan=1 colspan=1>Fluenz Tetra nasal vaccine</td><td rowspan=1 colspan=1>0.00001% v/v</td></tr><tr><td rowspan=1 colspan=1>FluMist live influenza vaccine</td><td rowspan=1 colspan=1>0.00001% v/v</td></tr><tr><td rowspan=1 colspan=1>Bleach</td><td rowspan=1 colspan=1>5% (v/v)</td></tr></table>

Additional interference was tested using the same methods for specimen collection materials. For swabs, the specific type of swab was dipped in UTM prior to mixing with the pathogen mix. For transport media (VTM), the pathogen mixes were prepared in the specified media at $100 \%$ (i.e.; complete replacement of the UTM used as part of the standard setup). The materials and conditions tested are presented in the table below.

Table 24: QIAstat-Dx Respiratory Panel Collection Material Interference Tested   

<table><tr><td rowspan=1 colspan=1>Collection Material</td><td rowspan=1 colspan=1>Condition</td></tr><tr><td rowspan=1 colspan=1>Swab - Copan 168C</td><td rowspan=1 colspan=1>Swab +1mL UTM</td></tr><tr><td rowspan=1 colspan=1>Swab - Copan FloQ</td><td rowspan=1 colspan=1>Swab +1mL UTM</td></tr><tr><td rowspan=1 colspan=1>Swab - Copan 175KS01</td><td rowspan=1 colspan=1>Swab +1mL UTM</td></tr><tr><td rowspan=1 colspan=1>Swab - Puritan</td><td rowspan=1 colspan=1>Swab +1mL UTM</td></tr><tr><td rowspan=1 colspan=1>VTM Sigma Virocult</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>VTM Remel M4-RT</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>VTM Remel M4</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>VTM Remel M5</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>VTM Remel M6</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>BD Universal Viral Transport</td><td rowspan=1 colspan=1>100%</td></tr></table>

No interference was observed using the collection materials listed above.

# h. Microbial Interference:

QIAstat-Dx Respiratory Panel testing was performed in the presence of non-panel respiratory pathogens to determine whether the pathogens are capable of interfering with the detection of the panel targets. Samples were tested in triplicate with and without addition of the potentially inhibitory organism for direct sample-to-sample comparison. Sample mixes were prepared in a simulated matrix consisting of UTM plus HeLa cells to an initial concentration of $1 0 \mathrm { x }$ LOD. Following addition of an equal volume of UTM (control) or interferent the final test concentration was 5x LOD for all panel targets. All pathogen-containing samples without spiked interferent generated positive signals for all pathogens present in the sample mixes. The potentially interfering organisms did not inhibit detection of any panel targets at the concentrations tested. No false positive results occurred because of the presence of the interfering organism during testing. A list of the potential interfering organisms and their final test concentrations are listed in the table below.

Table 25: List of Potentially Interfering Organisms and Concentrations Tested   

<table><tr><td rowspan=1 colspan=1>Interferent Tested</td><td rowspan=1 colspan=1>InterferentConcentration</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus aureu</td><td rowspan=1 colspan=1>1.00 x 106CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Neisseria meningitidis</td><td rowspan=1 colspan=1>5.0 x 104CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Corynebacteriumdiphtheriae</td><td rowspan=1 colspan=1>5.0 x 10³ CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Cytomegalovirus</td><td rowspan=1 colspan=1>1.00 x 105 TCID50/mL</td></tr></table>

# i. Carry-over:

Not applicable. The QIAstat-Dx Respiratory Panel Assay consists of single-use disposable cassettes containing all the reagents, reservoirs, and reaction chambers necessary to perform the test.

# j. Assay cut-off:

Assay cut-off for the QIAstat-Dx Respiratory Panel was set according to data obtained during LOD studies. The assay software Result Call Algorithm (RCA) is the software processing fluorescence measurements during PCR cycling to produce a qualitative result. The RCA parameters were initially set during system development and later adjusted using empirical data obtained during the LOD studies. The refined RCA was then confirmed during clinical evaluation with the results of the method comparison testing.

2. Comparison studies:

# a. Method comparison with predicate device:

Not applicable. Performance of the QIAstat-Dx Respiratory Panel was evaluated against the comparator method in prospective and retrospective clinical studies and with contrived specimens where necessary.

b. Matrix comparison:

A matrix equivalency study was performed to verify that the initial LOD determinations performed in simulated matrix were comparable to the LOD values obtained in clinical matrix. In order to assess the performance in clinical matrix, a concentration of 1x LOD (as determined in simulated matrix) for at least one strain per respiratory panel (RP) pathogen was prepared in true negative clinical NPS sample matrix and tested in 20 replicates. Results of the matrix equivalency study (as determined empirically by an LOD confirmation experiment) are shown in the table below.

Table 26: Results of Matrix Equivalency Study   

<table><tr><td colspan="1" rowspan="1">Pathogen</td><td colspan="1" rowspan="1">Strain</td><td colspan="1" rowspan="1">DetectionRate1</td><td colspan="1" rowspan="1">Equivalent?</td></tr><tr><td colspan="1" rowspan="1">Influenza B</td><td colspan="1" rowspan="1">B/Florida/4/2006</td><td colspan="1" rowspan="1">18/20</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Influenza A H3N2</td><td colspan="1" rowspan="1">A/PortChalmers/1/73</td><td colspan="1" rowspan="1">Flu A: 20/20H3: 19/20</td><td colspan="1" rowspan="1">++</td></tr><tr><td colspan="1" rowspan="1">Coronavirus229E</td><td colspan="1" rowspan="1">n/a</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1">CoronavirusOC43</td><td colspan="1" rowspan="1">n/a</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1">CoronavirusNL63</td><td colspan="1" rowspan="1">n/a</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1">CoronavirusHKU1</td><td colspan="1" rowspan="1">n/a</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1">ParainfluenzaVirus</td><td colspan="1" rowspan="1">n/a</td><td colspan="1" rowspan="1">18/20</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Parainfluenzaa Virus 2</td><td colspan="1" rowspan="1">Greer</td><td colspan="1" rowspan="1">17/20</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">ParainfluenzaVirus 3</td><td colspan="1" rowspan="1">C 243</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1">ParainfluenzaVirus 4</td><td colspan="1" rowspan="1">M-25</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1">Rhinovirus</td><td colspan="1" rowspan="1">A2</td><td colspan="1" rowspan="1">19/20</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1">Enterovirus</td><td colspan="1" rowspan="1">US/IL/14-18952</td><td colspan="1" rowspan="1">19/20</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1">Adenovirus</td><td colspan="1" rowspan="1">GB</td><td colspan="1" rowspan="1">19/20</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1">RSV B</td><td colspan="1" rowspan="1">CH93(18)-18</td><td colspan="1" rowspan="1">19/20</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1">hMPV</td><td colspan="1" rowspan="1">hMPV-16</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1">Mycoplasma pneumoniae</td><td colspan="1" rowspan="1">PI 1428</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1">Bordetella pertussis</td><td colspan="1" rowspan="1">I028</td><td colspan="1" rowspan="1">19/20</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1">Chlamydophilapneumoniae</td><td colspan="1" rowspan="1">TW183</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">+</td></tr></table>

1 Detection Rate in clinical matrix at LOD concentration determined in simulated matrix.

The results of the matrix equivalency study demonstrated that the two matrices are not equivalent for all analytes detected by the QIAstat-Dx Respiratory Panel and that analytical studies performed in simulated matrix (at the LOD determined in simulated matrix) represent a more challenging analyte concentration for those studies. Claimed LOD concentrations listed in the LOD section of this summary and the package insert represent either the concentration confirmed in this study or the higher (more concentrated) analyte level as verified in clinical matrix according to minimum empirical acceptance criteria $( \geq 9 5 \%$ detection rate with a minimum of 20 replicates).

3. Clinical Studies:

# Prospective Specimens

A multi-center study was conducted at five study sites located throughout the U.S. plus one international site between December 2017 and April 2019. The QIAstat-Dx Respiratory Panel was used to evaluate fresh, prospectively collected nasopharyngeal swab specimens eluted in UTM from children and adults of all ages presenting with flulike symptoms and meeting inclusion/exclusion criteria. Retrospective (archived) samples were also included as part of the performance testing. Each study location was representative of the intended use setting for the QIAstat-Dx Respiratory Panel assay and testing was performed by trained clinical laboratory personnel. A residual NPS specimen in UTM was tested for each subject with the QIAstat-Dx Respiratory Panel and an FDA cleared multiplex respiratory pathogen panel comparator.

A total of 2341 nasopharyngeal swab specimens in UTM were enrolled in the study. Of those, 37 specimens did not meet eligibility criteria or produced invalid results upon repeat testing. A total of 2304 nasopharyngeal swab specimens were considered evaluable. Of the 2304 specimens that met eligibility criteria, 310 were retrospective (archived) specimens and 1994 were fresh prospective or frozen prospective samples. Patient age and gender distribution for the evaluable specimens is presented in tables 27 and 28 below.

Table 27: Prospective Clinical Study Participant Demographics by Age   

<table><tr><td rowspan=1 colspan=1>Age Group(Years)</td><td rowspan=1 colspan=1>Count</td><td rowspan=1 colspan=1>Percent</td></tr><tr><td rowspan=1 colspan=1>&lt;5</td><td rowspan=1 colspan=1>627</td><td rowspan=1 colspan=1>31.4%</td></tr><tr><td rowspan=1 colspan=1>6-21</td><td rowspan=1 colspan=1>239</td><td rowspan=1 colspan=1>12.0%</td></tr><tr><td rowspan=1 colspan=1>22-49</td><td rowspan=1 colspan=1>330</td><td rowspan=1 colspan=1>16.5%</td></tr><tr><td rowspan=1 colspan=1>&gt;50</td><td rowspan=1 colspan=1>798</td><td rowspan=1 colspan=1>40.0%</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>1994</td><td rowspan=1 colspan=1>100%</td></tr></table>

Table 28: Clinical Study Participant by Gender and Site   

<table><tr><td rowspan=1 colspan=1>Site</td><td rowspan=1 colspan=1>Female</td><td rowspan=1 colspan=1>Male</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>232</td><td rowspan=1 colspan=1>186</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>230</td><td rowspan=1 colspan=1>196</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>271</td><td rowspan=1 colspan=1>177</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>133</td><td rowspan=1 colspan=1>170</td></tr><tr><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>204</td><td rowspan=1 colspan=1>195</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>1070 (53.7%)</td><td rowspan=1 colspan=1>924 (46.3%)</td></tr></table>

Of the 1994 evaluable prospective specimens, $9 5 . 8 8 \%$ (1912/1994) yielded valid results on the first attempt (i.e., first loaded cartridge). Invalid or no result were obtained for the remaining 82 specimens $( 4 . 1 1 \% )$ . Forty-two (42) specimens were invalid due to cartridge internal control failure $( 2 . 1 1 \% )$ . Of these, 20 $( 1 . 0 0 \% )$ provided a result for positively detected targets and 22 $( 1 . 1 0 \% )$ provided a negative result. For 40 $( 2 . 0 0 \% )$ specimens no results were obtained due to incomplete runs. Of these, 1 specimen was aborted by users $( 0 . 0 5 \% )$ , 21 were due to instrument errors $( 1 . 0 5 \% )$ , and 18 were due to cartridge related errors $( 0 . 9 0 \% )$ . Seventy-two (72) of the 82 initially failed (no result or invalid) specimens yielded valid results after a single retesting using a new cartridge/sample. The remaining 10 specimens failed on the second attempt (2 due to cartridge failures, 1 due to instrument errors, and 7 due to internal control failures). Of these internal control failures, detected pathogens were reported for 4 specimens.

Compared to an FDA-cleared molecular assay, the performance of the QIAstat-Dx Respiratory Panel for NPS swabs eluted in UTM is presented below.

Table 29: QIAstat-Dx Respiratory Panel Eluted Nasopharyngeal Swab Performance   

<table><tr><td rowspan=1 colspan=4>Compared to FDA-clea</td><td rowspan=1 colspan=2>red Molecular Comp:</td><td rowspan=1 colspan=1>irator  Pr</td><td rowspan=1 colspan=1>ospective S</td><td rowspan=1 colspan=2></td></tr><tr><td rowspan=1 colspan=3>Analyte</td><td rowspan=1 colspan=1>Group</td><td rowspan=1 colspan=1>TP/(TP+FN)</td><td rowspan=1 colspan=1>PPA</td><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=1>TN/(TN+FP)</td><td rowspan=1 colspan=1>NPA</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=3 colspan=3>Adenovirus1</td><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>55/58</td><td rowspan=1 colspan=1>94.8%</td><td rowspan=1 colspan=1>85.9-98.2</td><td rowspan=1 colspan=1>833/839</td><td rowspan=1 colspan=1>99.3%</td><td rowspan=1 colspan=1>98.4-99.7</td></tr><tr><td rowspan=1 colspan=1>Frozen</td><td rowspan=1 colspan=1>31/32</td><td rowspan=1 colspan=1>96.9%</td><td rowspan=1 colspan=1>84.3-99.4</td><td rowspan=1 colspan=1>1047/1057</td><td rowspan=1 colspan=1>99.1%</td><td rowspan=1 colspan=1>98.3-99.5</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>86/90</td><td rowspan=1 colspan=1>95.6%</td><td rowspan=1 colspan=1>89.1-98.3</td><td rowspan=1 colspan=1>1880/1896</td><td rowspan=1 colspan=1>99.2%</td><td rowspan=1 colspan=1>98.6-99.5</td></tr><tr><td rowspan=3 colspan=3>Coronavirus229E</td><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>8/9</td><td rowspan=1 colspan=1>88.9%</td><td rowspan=1 colspan=1>56.5-98.0</td><td rowspan=1 colspan=1>886/886</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>99.6-100</td></tr><tr><td rowspan=1 colspan=1>Frozen</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>1089/1089</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>99.6-100</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>8/9</td><td rowspan=1 colspan=1>88.9%</td><td rowspan=1 colspan=1>56.5-98.0</td><td rowspan=1 colspan=1>1975/1975</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>99.8-100</td></tr><tr><td rowspan=3 colspan=3>CoronavirusHK12</td><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>43.8-100</td><td rowspan=1 colspan=1>890/892</td><td rowspan=1 colspan=1>99.8%</td><td rowspan=1 colspan=1>99.2-99.9</td></tr><tr><td rowspan=1 colspan=1>Frozen</td><td rowspan=1 colspan=1>48/49</td><td rowspan=1 colspan=1>98.0%</td><td rowspan=1 colspan=1>89.3-99.6</td><td rowspan=1 colspan=1>1035/1040</td><td rowspan=1 colspan=1>99.5%</td><td rowspan=1 colspan=1>98.9-99.8</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>51/52</td><td rowspan=1 colspan=1>98.1%</td><td rowspan=1 colspan=1>89.9-99.7</td><td rowspan=1 colspan=1>1925/1932</td><td rowspan=1 colspan=1>99.6%</td><td rowspan=1 colspan=1>99.3-99.8</td></tr><tr><td rowspan=3 colspan=3>CoronavirusNL633</td><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>4/5</td><td rowspan=1 colspan=1>80.0%</td><td rowspan=1 colspan=1>37.6-96.4</td><td rowspan=1 colspan=1>890/890</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>99.6-100</td></tr><tr><td rowspan=1 colspan=1>Frozen</td><td rowspan=1 colspan=1>36/42</td><td rowspan=1 colspan=1>85.7%</td><td rowspan=1 colspan=1>72.2-93.3</td><td rowspan=1 colspan=1>1046/1048</td><td rowspan=1 colspan=1>99.8%</td><td rowspan=1 colspan=1>99.3-99.9</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>40/47</td><td rowspan=1 colspan=1>85.1%</td><td rowspan=1 colspan=1>72.3-92.6</td><td rowspan=1 colspan=1>1936/1938</td><td rowspan=1 colspan=1>99.9%</td><td rowspan=1 colspan=1>99.6-100</td></tr><tr><td rowspan=3 colspan=3>CoronavirusC434</td><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>43.8-100</td><td rowspan=1 colspan=1>892/892</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>99.6-100</td></tr><tr><td rowspan=1 colspan=1>Frozen</td><td rowspan=1 colspan=1>23/26</td><td rowspan=1 colspan=1>88.5%</td><td rowspan=1 colspan=1>71.0-96.0</td><td rowspan=1 colspan=1>1059/1063</td><td rowspan=1 colspan=1>99.6%</td><td rowspan=1 colspan=1>99.0-99.9</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>26/29</td><td rowspan=1 colspan=1>89.7%</td><td rowspan=1 colspan=1>73.6-96.4</td><td rowspan=1 colspan=1>1 1951/1955</td><td rowspan=1 colspan=1>99.8%</td><td rowspan=1 colspan=1>99.5-99.9</td></tr><tr><td rowspan=3 colspan=3>Human5metapneumovirus</td><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>62/67</td><td rowspan=1 colspan=1>92.5%</td><td rowspan=1 colspan=1>83.7-96.8</td><td rowspan=1 colspan=1>828/829</td><td rowspan=1 colspan=1>99.9%</td><td rowspan=1 colspan=1>99.3-100</td></tr><tr><td rowspan=1 colspan=1>Frozen</td><td rowspan=1 colspan=1>53/55</td><td rowspan=1 colspan=1>96.4%</td><td rowspan=1 colspan=1>87.7-99.0</td><td rowspan=1 colspan=1>1030/1034</td><td rowspan=1 colspan=1>99.6%</td><td rowspan=1 colspan=1>99.0-99.8</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>115/122</td><td rowspan=1 colspan=1>94.3%</td><td rowspan=1 colspan=1>88.6-97.2</td><td rowspan=1 colspan=1>1858/1863</td><td rowspan=1 colspan=1>99.7%</td><td rowspan=1 colspan=1>99.4-99.9</td></tr><tr><td rowspan=3 colspan=3>Rhinovirus/Enterovirus6</td><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>144/157</td><td rowspan=1 colspan=1>91.7%</td><td rowspan=1 colspan=1>86.3-95.1</td><td rowspan=1 colspan=1>715/739</td><td rowspan=1 colspan=1>96.8%</td><td rowspan=1 colspan=1>95.2-97.8</td></tr><tr><td rowspan=1 colspan=1>Frozen</td><td rowspan=1 colspan=1>124/137</td><td rowspan=1 colspan=1>90.5%</td><td rowspan=1 colspan=1>84.4-94.4</td><td rowspan=1 colspan=1>941/953</td><td rowspan=1 colspan=1>98.7%</td><td rowspan=1 colspan=1>97.8-99.3</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>268/294</td><td rowspan=1 colspan=1>91.2%</td><td rowspan=1 colspan=1>87.4-93.9</td><td rowspan=1 colspan=1>1656/1692</td><td rowspan=1 colspan=1>97.9%</td><td rowspan=1 colspan=1>97.1-98.5</td></tr><tr><td rowspan=3 colspan=3>Influenza A7</td><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>132/133</td><td rowspan=1 colspan=1>99.2%</td><td rowspan=1 colspan=1>95.8-99.9</td><td rowspan=1 colspan=1>753/757</td><td rowspan=1 colspan=1>99.5%</td><td rowspan=1 colspan=1>98.6-99.8</td></tr><tr><td rowspan=1 colspan=1>Frozen</td><td rowspan=1 colspan=1>110/111</td><td rowspan=1 colspan=1>99.1%</td><td rowspan=1 colspan=1>95.1-99.8</td><td rowspan=1 colspan=1>972/977</td><td rowspan=1 colspan=1>99.5%</td><td rowspan=1 colspan=1>98.8-99.8</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>242/244</td><td rowspan=1 colspan=1>99.2%</td><td rowspan=1 colspan=1>97.0-99.8</td><td rowspan=1 colspan=1>1725/1734</td><td rowspan=1 colspan=1>99.5%</td><td rowspan=1 colspan=1>99.0-99.7</td></tr><tr><td rowspan=3 colspan=3>Influenza A H18</td><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>0/1</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0.0-79.3</td><td rowspan=1 colspan=1>894/894</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>99.6-100</td></tr><tr><td rowspan=1 colspan=2>H18</td><td rowspan=1 colspan=1>Frozen</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>1089/1089</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>99.6-100</td></tr><tr><td rowspan=1 colspan=2></td><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>0/1</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0.0-79.3</td><td rowspan=1 colspan=1>1983/1983</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>99.8-100</td></tr><tr><td rowspan=3 colspan=3>Influenza AH1N1/dm09</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>62/63</td><td rowspan=1 colspan=1>98.4%</td><td rowspan=1 colspan=1>91.5-99.7</td><td rowspan=1 colspan=1>826/831</td><td rowspan=1 colspan=1>99.4%</td></tr><tr><td rowspan=1 colspan=1>Frozen</td><td rowspan=1 colspan=1>18/18</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>82.4-100</td><td rowspan=1 colspan=1>1071/1071</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>99.6-100</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>80/81</td><td rowspan=1 colspan=1>98.8%</td><td rowspan=1 colspan=1>93.3-99.8</td><td rowspan=1 colspan=1>1897/1902</td><td rowspan=1 colspan=1>99.7%</td><td rowspan=1 colspan=1>99.4-99.9</td></tr><tr><td rowspan=3 colspan=3>Influenza A H310</td><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>67/67</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>94.5-100</td><td rowspan=1 colspan=1>825/826</td><td rowspan=1 colspan=1>99.9%</td><td rowspan=1 colspan=1>99.3-100</td></tr><tr><td rowspan=1 colspan=1>Frozen</td><td rowspan=1 colspan=1>89/90</td><td rowspan=1 colspan=1>98.9%</td><td rowspan=1 colspan=1>82.4-100</td><td rowspan=1 colspan=1>992/998</td><td rowspan=1 colspan=1>99.4%</td><td rowspan=1 colspan=1>98.7-99.7</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>156/157</td><td rowspan=1 colspan=1>99.4%</td><td rowspan=1 colspan=1>93.3-99.8</td><td rowspan=1 colspan=1>1817/1824</td><td rowspan=1 colspan=1>99.6%</td><td rowspan=1 colspan=1>99.2-99.8</td></tr><tr><td rowspan=3 colspan=3>Influenza B11</td><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>64/67</td><td rowspan=1 colspan=1>95.5%</td><td rowspan=1 colspan=1>87.6-98.5</td><td rowspan=1 colspan=1>827/892</td><td rowspan=1 colspan=1>99.9%</td><td rowspan=1 colspan=1>99.3-100</td></tr><tr><td rowspan=1 colspan=1>Frozen</td><td rowspan=1 colspan=1>58/62</td><td rowspan=1 colspan=1>93.5%</td><td rowspan=1 colspan=1>84.6-97.5</td><td rowspan=1 colspan=1>1026/1026</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>99.6-100</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>122/129</td><td rowspan=1 colspan=1>94.6%</td><td rowspan=1 colspan=1>89.2-97.3</td><td rowspan=1 colspan=1>1853/1854</td><td rowspan=1 colspan=1>99.9%</td><td rowspan=1 colspan=1>99.7-100</td></tr><tr><td rowspan=3 colspan=3>Parainfluenzavirus  12</td><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>43.8-100</td><td rowspan=1 colspan=1>892/892</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>99.6-100</td></tr><tr><td rowspan=1 colspan=1>Frozen</td><td rowspan=1 colspan=1>13/14</td><td rowspan=1 colspan=1>92.9%</td><td rowspan=1 colspan=1>68.5-98.7</td><td rowspan=1 colspan=1>1072/1075</td><td rowspan=1 colspan=1>99.7%</td><td rowspan=1 colspan=1>99.2-99,9</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>16/17</td><td rowspan=1 colspan=1>94.1%</td><td rowspan=1 colspan=1>73.0-99.0</td><td rowspan=1 colspan=1>1964/1967</td><td rowspan=1 colspan=1>99.8%</td><td rowspan=1 colspan=1>99.6-99.9</td></tr><tr><td rowspan=3 colspan=3>Parainfluenzavirus 2</td><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>34.2-100</td><td rowspan=1 colspan=1>893/893</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>99.6-100</td></tr><tr><td rowspan=1 colspan=1>Frozen</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>1089/1089</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>99.6-100</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>34.2-100</td><td rowspan=1 colspan=1>1982/1982</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>99.8-100</td></tr><tr><td rowspan=3 colspan=3>Parainfluenzavirus 313</td><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>102/104</td><td rowspan=1 colspan=1>98.1%</td><td rowspan=1 colspan=1>93.3-99.5</td><td rowspan=1 colspan=1>788/793</td><td rowspan=1 colspan=1>99.4%</td><td rowspan=1 colspan=1>98.5-99.7</td></tr><tr><td rowspan=1 colspan=1>Frozen</td><td rowspan=1 colspan=1>9/9</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>70.1-100</td><td rowspan=1 colspan=1>1081/1081</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>99.6-100</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>111/113</td><td rowspan=1 colspan=1>98.2%</td><td rowspan=1 colspan=1>93.8-99.5</td><td rowspan=1 colspan=1>1869/1874</td><td rowspan=1 colspan=1>99.7%</td><td rowspan=1 colspan=1>99.4-99.9</td></tr><tr><td rowspan=3 colspan=3>Parainfluenzavvius 414</td><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>43.8-100</td><td rowspan=1 colspan=1>892/892</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>99.6-100</td></tr><tr><td rowspan=1 colspan=1>Frozen</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>1087/1089</td><td rowspan=1 colspan=1>99.8%</td><td rowspan=1 colspan=1>99.3-99.9</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>43.8-100</td><td rowspan=1 colspan=1>1979/1981</td><td rowspan=1 colspan=1>99.9%</td><td rowspan=1 colspan=1>99.6-100</td></tr></table>

<table><tr><td rowspan=3 colspan=1>RespiratorySyncytial Virus15</td><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>73/76</td><td rowspan=1 colspan=1>96.1%</td><td rowspan=1 colspan=1>88.9-98.6</td><td rowspan=1 colspan=1>819/820</td><td rowspan=1 colspan=1>99.9%</td><td rowspan=1 colspan=1>99.3-100</td></tr><tr><td rowspan=1 colspan=1>Frozen</td><td rowspan=1 colspan=1>139/144</td><td rowspan=1 colspan=1>96.5%</td><td rowspan=1 colspan=1>92.1-98.5</td><td rowspan=1 colspan=1>941/945</td><td rowspan=1 colspan=1>99.6%</td><td rowspan=1 colspan=1>98.9-99.8</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>212/220</td><td rowspan=1 colspan=1>96.4%</td><td rowspan=1 colspan=1>93.0-98.1</td><td rowspan=1 colspan=1>1760/1765</td><td rowspan=1 colspan=1>99.7%</td><td rowspan=1 colspan=1>99.3-99.9</td></tr><tr><td rowspan=3 colspan=1>Bordetellapertussis16</td><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>34.2-100</td><td rowspan=1 colspan=1>893/893</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>99.6-100</td></tr><tr><td rowspan=1 colspan=1>Frozen</td><td rowspan=1 colspan=1>1/1</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>20.7-100</td><td rowspan=1 colspan=1>1082/1088</td><td rowspan=1 colspan=1>99.4%</td><td rowspan=1 colspan=1>98.8-99.7</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>43.8-100</td><td rowspan=1 colspan=1>1975/1981</td><td rowspan=1 colspan=1>99.7%</td><td rowspan=1 colspan=1>99.3-99.9</td></tr><tr><td rowspan=3 colspan=1>Chlamydophilapneumoniae17</td><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>4/4</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>51.0-100</td><td rowspan=1 colspan=1>891/891</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>99.6-100</td></tr><tr><td rowspan=1 colspan=1>Frozen</td><td rowspan=1 colspan=1>1/1</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>20.7-100</td><td rowspan=1 colspan=1>1087/1088</td><td rowspan=1 colspan=1>99.9%</td><td rowspan=1 colspan=1>99.5-100</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>5/5</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>56.6-100</td><td rowspan=1 colspan=1>1978/1979</td><td rowspan=1 colspan=1>99.9%</td><td rowspan=1 colspan=1>99.7-100</td></tr><tr><td rowspan=3 colspan=1>Mycoplasmapneumoniae18</td><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>18/18</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>82.4-100</td><td rowspan=1 colspan=1>875/877</td><td rowspan=1 colspan=1>99.8%</td><td rowspan=1 colspan=1>99.2-100</td></tr><tr><td rowspan=1 colspan=1>Frozen</td><td rowspan=1 colspan=1>1/1</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>20.7-100</td><td rowspan=1 colspan=1>1085/1088</td><td rowspan=1 colspan=1>99.7%</td><td rowspan=1 colspan=1>99.2-99.9</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>19/19</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>83.2-100</td><td rowspan=1 colspan=1>1960/1965</td><td rowspan=1 colspan=1>99.7%</td><td rowspan=1 colspan=1>99.4-99.9</td></tr></table>

ronavirus NL63 was detected in 1/2 FP specimens using an independent molecular method.

4 The 3 FN specimens were negative for Coronavirus OC43 when tested using an independent molecular method. Coronavirus OC43 was detected in 3/4 FP specimens using an independent molecular method.

5 Human metapneumovirus (hMPV) was detected in $4 / 7$ FN specimens using an independent molecula method. hMPV was detected in 3/5 FP specimens using an independent molecular method.

6 Rhinovirus was detected in 18/26 FN specimens using an independent molecular method. Rhinovirus was detected in 14/36 FP specimens using an independent molecular method.

7 Influenza A was detected in 1/2 FN specimens by an independent molecular method. Three (3) FP samples were not available for testing. Influenza A was detected in the $3 / 6$ remaining FP samples by an independent molecular method.

8 Influenza A H1 was detected in 1/1 FN specimen by an independent molecular method. Note: Pre$2 0 0 9 \mathrm { H } 1$ has not been in circulation since being replaced by $\dot { 2 0 0 9 } \mathrm { H 1 }$ and the discrepant results are likely a false positive result.

9 Influenza A $\mathbf { \dot { H } l N } 1 / \mathbf { p d m } 0 9 $ was detected in 1/1 FN by an independent molecular method. Influenza A H1 was detected in 3/5 FP specimens by an independent molecular method.

10 Influenza A H3 was detected in 1/1 FN by an independent molecular method. Influenza H3 was detected in 7/7 FP specimens by an independent molecular method.

11 Influenza B was detected in $\bar { 6 } / 6$ FN specimens available for testing by an independent molecular method; one discordant sample was not tested by an independent molecular method. Influenza B was detected in 1/1 FP specimens available for testing by an independent molecular method.

$^ { 1 2 } 1 / 1$ FN specimens tested negative for Parainfluenza virus 1 by an independent molecular method.   
Parainfluenza Virus 1 was detected in $3 / 3$ FP specimens by an independent molecular method.

The QIAstat-Dx Respiratory Panel detected a total of 191 specimens with distinctive multiple organism detections $( 9 . 6 \%$ of all specimens) in the prospective study. The three organisms most prevalent in multiple detections by the QIAstat-Dx Respiratory Panel in the prospective study were Rhinovirus/Enterovirus (108/191, $5 6 . 5 \% )$ , Respiratory Syncytial Virus (77/191, $4 0 . 8 \% )$ , and Adenovirus $( 5 3 / 1 9 1 , 2 7 . 7 \% )$ . A summary of the coinfections is described in Table 30.

Table 30: Coinfections Detected in Prospective Samples During the Clinical Study   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>DetectedCoinfections</td><td rowspan=1 colspan=1>DiscrepantCoinfections</td></tr><tr><td rowspan=1 colspan=1>Total Coinfections</td><td rowspan=1 colspan=1>191</td><td rowspan=1 colspan=1>51</td></tr><tr><td rowspan=1 colspan=1>Total DoubleCoinfections</td><td rowspan=1 colspan=1>166</td><td rowspan=1 colspan=1>42</td></tr><tr><td rowspan=1 colspan=1>Total TripleCoinfections</td><td rowspan=1 colspan=1>22</td><td rowspan=1 colspan=1>7</td></tr><tr><td rowspan=1 colspan=1>Total Quadruple Coinfections</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>2</td></tr></table>

# Archived Specimens

Some of the analytes on the QIAstat-Dx Respiratory Panel were of low prevalence and were not encountered in sufficiently large numbers during the prospective study to adequately demonstrate clinical performance. To supplement the results of the prospective clinical study, an evaluation of preselected frozen archived retrospective specimens was performed. The specimens selected for testing had previously tested positive for one of the following targets at the clinical laboratory by their standard of care method: Bordetella pertussis, Coronavirus 229E, Coronavirus HKU1, Coronavirus NL63, Coronavirus OC43, Influenza A H1N1 2009, Mycoplasma pneumoniae, Chlamydophila pneumoniae, Parainfluenza Virus 1, Parainfluenza Virus 2, and Parainfluenza Virus 4. Testing was performed by operators who were blinded as to the expected test result. A total of 310 clinical frozen archived retrospective sample were tested by both the comparator method and QIAstat-Dx Respiratory Panel. If the comparator method did not confirm the preselected target as positive, it was excluded from the data analysis for that target. Demographic information for the archived specimens is presented below. Archived samples consisted of 158 male and 152 female specimens.

Table 31: Participant Age for Archived Clinical Samples   

<table><tr><td rowspan=1 colspan=1>Age Group(Years)</td><td rowspan=1 colspan=1>Count</td><td rowspan=1 colspan=1>Percent</td></tr><tr><td rowspan=1 colspan=1>&lt;5</td><td rowspan=1 colspan=1>139</td><td rowspan=1 colspan=1>44.9%</td></tr><tr><td rowspan=1 colspan=1>6-21</td><td rowspan=1 colspan=1>85</td><td rowspan=1 colspan=1>27.4%</td></tr><tr><td rowspan=1 colspan=1>22-49</td><td rowspan=1 colspan=1>53</td><td rowspan=1 colspan=1>17.1%</td></tr><tr><td rowspan=1 colspan=1>&gt;50</td><td rowspan=1 colspan=1>33</td><td rowspan=1 colspan=1>10.7%</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>310</td><td rowspan=1 colspan=1>100%</td></tr></table>

Table 32: QIAstat-Dx Respiratory Panel Eluted Nasopharyngeal Swab Performance Compared to FDA-cleared Molecular Comparator – Archived Specimens   

<table><tr><td rowspan=1 colspan=1>Analyte</td><td rowspan=1 colspan=1>TP/(TP+FN)</td><td rowspan=1 colspan=1>PPA</td><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=1>TN/(TN+FP)</td><td rowspan=1 colspan=1>NPA</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=1>Adenovirus1</td><td rowspan=1 colspan=1>9/9</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>70.1-100</td><td rowspan=1 colspan=1>297/304</td><td rowspan=1 colspan=1>97/8%</td><td rowspan=1 colspan=1>95.4-98.9</td></tr><tr><td rowspan=1 colspan=1>Coronavirus229E</td><td rowspan=1 colspan=1>26/27</td><td rowspan=1 colspan=1>96.3%</td><td rowspan=1 colspan=1>81.7-99.3</td><td rowspan=1 colspan=1>286/286</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>98.7-100</td></tr><tr><td rowspan=1 colspan=1>CoronavirusHKU12</td><td rowspan=1 colspan=1>14/14</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>78.5-100</td><td rowspan=1 colspan=1>298/299</td><td rowspan=1 colspan=1>99.7%</td><td rowspan=1 colspan=1>98.1-99.9</td></tr><tr><td rowspan=1 colspan=1>CoronavirusNL63</td><td rowspan=1 colspan=1>24/24</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>86.2-100</td><td rowspan=1 colspan=1>286/288</td><td rowspan=1 colspan=1>99.3%</td><td rowspan=1 colspan=1>97.5-99.8</td></tr><tr><td rowspan=1 colspan=1>CoronavirusOC43</td><td rowspan=1 colspan=1>28/28</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>87.9-100</td><td rowspan=1 colspan=1>282/282</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>98.6-100</td></tr><tr><td rowspan=1 colspan=1>Human4metapneumovirus</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>34.2-100</td><td rowspan=1 colspan=1>311/311</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>98.7-100</td></tr><tr><td rowspan=1 colspan=1>Rhinovirus/Enterovirus</td><td rowspan=1 colspan=1>44/49</td><td rowspan=1 colspan=1>89.8%</td><td rowspan=1 colspan=1>78.2-95.5</td><td rowspan=1 colspan=1>254/264</td><td rowspan=1 colspan=1>96.2%</td><td rowspan=1 colspan=1>93.2-97.9</td></tr><tr><td rowspan=1 colspan=1>Influenza A</td><td rowspan=1 colspan=1>17/17</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>81.5-100</td><td rowspan=1 colspan=1>296/296</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>98.7-100</td></tr><tr><td rowspan=1 colspan=1>Influenza A H1</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>313/313</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>98.8-100</td></tr><tr><td rowspan=1 colspan=1>Influenza AH1N1/pdm095</td><td rowspan=1 colspan=1>7/8</td><td rowspan=1 colspan=1>87.5%</td><td rowspan=1 colspan=1>52.9-97.8</td><td rowspan=1 colspan=1>304/304</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>98.8-100</td></tr><tr><td rowspan=1 colspan=1>Influenza A H3</td><td rowspan=1 colspan=1>8/8</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>67.5-100</td><td rowspan=1 colspan=1>305/305</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>98.8-100</td></tr><tr><td rowspan=1 colspan=1>Influenza B</td><td rowspan=1 colspan=1>1/1</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>20.7-100</td><td rowspan=1 colspan=1>312/312</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>98.8-100</td></tr><tr><td rowspan=1 colspan=1>Parainfluenzavirus 1</td><td rowspan=1 colspan=1>40/40</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>91.2-100</td><td rowspan=1 colspan=1>267/267</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>98.6-100</td></tr><tr><td rowspan=1 colspan=1>Parainfluenzavirus 2</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>43.8-100</td><td rowspan=1 colspan=1>309/309</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>98.8-100</td></tr><tr><td rowspan=1 colspan=1>Parainfluenzavirus 36</td><td rowspan=1 colspan=1>1/4</td><td rowspan=1 colspan=1>25.0%</td><td rowspan=1 colspan=1>4.6-69.9</td><td rowspan=1 colspan=1>309/309</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>98.8-100</td></tr><tr><td rowspan=1 colspan=1>Parainfluenzavirus 47</td><td rowspan=1 colspan=1>22/24</td><td rowspan=1 colspan=1>91.7%</td><td rowspan=1 colspan=1>74.2-97.7</td><td rowspan=1 colspan=1>278/278</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>98.6-100</td></tr><tr><td rowspan=1 colspan=1>RespiratorySyncytial Virus8</td><td rowspan=1 colspan=1>11/12</td><td rowspan=1 colspan=1>91.7%</td><td rowspan=1 colspan=1>64.6-98.5</td><td rowspan=1 colspan=1>300/301</td><td rowspan=1 colspan=1>99.7%</td><td rowspan=1 colspan=1>98.4-99.9</td></tr><tr><td rowspan=1 colspan=1>Bordetellapertussis</td><td rowspan=1 colspan=1>33/33</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>89.6-100</td><td rowspan=1 colspan=1>261/261</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>98.5-100</td></tr><tr><td rowspan=1 colspan=1>Chlamydophilapneumoniae</td><td rowspan=1 colspan=1>54/61</td><td rowspan=1 colspan=1>88.5%</td><td rowspan=1 colspan=1>78.2-94.3</td><td rowspan=1 colspan=1>250/250</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>98.5-100</td></tr><tr><td rowspan=1 colspan=1>Mycoplasmapneumoniae</td><td rowspan=1 colspan=1>25/25</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>86.7-100</td><td rowspan=1 colspan=1>287/288</td><td rowspan=1 colspan=1>99.7%</td><td rowspan=1 colspan=1>98.1-99.9</td></tr></table>

1 Adenovirus was detected in 3/5 FP specimens using an independent molecular method. 2 The single FP Coronavirus HKU1 specimen was negative when tested using an independent molecular method. 3 The single FP Coronavirus NL63 specimen was negative when tested using an independent molecular method. 4 Rhinovirus was detected in $1 / 2 \mathrm { F N }$ when tested using an independent molecular method. Rhinovirus was detected in 4/10 FP specimens using an independent molecular method. 5 Influenza H1N1/pdm09 was detected in the single FN specimen. 6 Parainfluenza Virus 3 was detected in $1 / 3 \mathrm { F N }$ specimens by an independent molecular method. 7 Parainfluenza Virus 4 was detected in 1/2 FN specimens by an independent molecular method. 8 The single FN Respiratory Syncytial Virus was negative for that target by an independent molecular method. The single FP Respiratory Syncytial Virus was negative for that target by an independent molecular method. 9 Chlamydophila pneumoniae was detected in 4/5 FN specimens by an independent molecular method.

# Contrived Specimens

Despite all prospective and retrospective testing efforts, the number of specimens positive for Influenza A H1N1, Parainfluenza 2, Parainfluenza 4, Coronavirus 229E, and Chlamydophila pneumoniae were insufficient to demonstrate system performance. Therefore, contrived specimens were used as surrogate clinical specimens to supplement and test the sensitivity and specificity of the above analytes. Residual negative clinical specimens were spiked with the pathogens at 3x, 5x and 10x LOD levels (50 of each). Contrived samples were provided a unique study identification number and the individual who contrived the samples did not test them; therefore, the status of each contrived specimen was unknown at the time of testing.

Table 33: QIAstat-Dx Respiratory Panel Performance Compared to FDA-cleared Molecular Comparator – Contrived Specimens   

<table><tr><td rowspan=1 colspan=1>Analyte</td><td rowspan=1 colspan=1>xLOD</td><td rowspan=1 colspan=1>TP/(TP+FN)</td><td rowspan=1 colspan=1>PPA</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=3 colspan=1>Influenza A/H1</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>24/24</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>86.2-100</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>27/27</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>87.5-100</td></tr><tr><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>24/24</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>86.2-100</td></tr><tr><td rowspan=3 colspan=1>Coronavirus 229E</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>16/16</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>80.6-100</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>18/18</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>82.4-100</td></tr><tr><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>16/16</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>80.6-100</td></tr><tr><td rowspan=3 colspan=1>Parainfluenza Virus 2</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>16/16</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>80.6-100</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>18/18</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>82.4-100</td></tr><tr><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>16/16</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>80.6-100</td></tr><tr><td rowspan=3 colspan=1>Parainfluenza Virus 4</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>15/16</td><td rowspan=1 colspan=1>93.8%</td><td rowspan=1 colspan=1>71.7-98.9</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>18/18</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>82.4-100</td></tr><tr><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>16/16</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>80.6-100</td></tr><tr><td rowspan=3 colspan=1>Chlamydophilapneumoniae</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>16/16</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>80.6-100</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>18/18</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>82.4-100</td></tr><tr><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>16/16</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>80.6-100</td></tr></table>

4. Clinical cut-off:

Not applicable.

5. Expected values/Reference range:

In the QIAstat-Dx Respiratory Panel prospective clinical study, a total of 1994 eluted NPS specimens were evaluable by the QIAstat-Dx Respiratory Panel assay. The number and percentage of positive cases per site (Table 34) and per age group (Table 35), as determined by the QIAstat-Dx Respiratory Panel assay, are presented below. One clinical study site consisted of only archived specimens and is not included in the table below.

Table 34: QIAstat-Dx Respiratory Panel Expected Values by Site   

<table><tr><td rowspan=2 colspan=1>Organism</td><td rowspan=1 colspan=2>Overall(n=1994)</td><td rowspan=1 colspan=2>Site 1 (n=418)</td><td rowspan=1 colspan=2>Site 2 (n=426)</td><td rowspan=1 colspan=2>Site 3 (n=448)</td><td rowspan=1 colspan=2>Site 4 (n=303)</td><td rowspan=1 colspan=2>Site 5 (n=399)</td></tr><tr><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>Expectedvalue</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>Expectedvalue</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>Expectedvalue</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>Expectedvalue</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>Expectedvalue</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>Expectedvalue</td></tr><tr><td rowspan=1 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>102</td><td rowspan=1 colspan=1>5.1%</td><td rowspan=1 colspan=1>44</td><td rowspan=1 colspan=1>10.5%</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>2.1%</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>2.7%</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>9.9%</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>1.8%</td></tr><tr><td rowspan=1 colspan=1>Coronavirus 229E</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>0.4%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.2%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>2.3%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%</td></tr><tr><td rowspan=1 colspan=1>Coronavirus HKU1</td><td rowspan=1 colspan=1>58</td><td rowspan=1 colspan=1>2.9%</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>1.0%</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>2.6%</td><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>3.1%</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>4.0%</td><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>4.3%</td></tr><tr><td rowspan=1 colspan=1>Coronavirus NL63</td><td rowspan=1 colspan=1>42</td><td rowspan=1 colspan=1>2.1%</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>1.0%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.2%</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>3.3%</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>3.6%</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>2.8%</td></tr><tr><td rowspan=1 colspan=1>Coronavirus OC43</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>1.5%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>1.2%</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>1.3%</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>4.0%</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>1.8%</td></tr><tr><td rowspan=1 colspan=1>HumanMetapneumovirus</td><td rowspan=1 colspan=1>120</td><td rowspan=1 colspan=1>6.0%</td><td rowspan=1 colspan=1>42</td><td rowspan=1 colspan=1>10.0%</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>5.6%</td><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>3.1%</td><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>4.6%</td><td rowspan=1 colspan=1>26</td><td rowspan=1 colspan=1>6.5%</td></tr><tr><td rowspan=1 colspan=1>HumanRhinovirus/Enterovirus</td><td rowspan=1 colspan=1>304</td><td rowspan=1 colspan=1>15.2%</td><td rowspan=1 colspan=1>59</td><td rowspan=1 colspan=1>14.1%</td><td rowspan=1 colspan=1>78</td><td rowspan=1 colspan=1>18.3%</td><td rowspan=1 colspan=1>39</td><td rowspan=1 colspan=1>8.7%</td><td rowspan=1 colspan=1>53</td><td rowspan=1 colspan=1>17.5%</td><td rowspan=1 colspan=1>75</td><td rowspan=1 colspan=1>18.8%</td></tr><tr><td rowspan=1 colspan=1>Influenza A</td><td rowspan=1 colspan=1>251</td><td rowspan=1 colspan=1>12.6%</td><td rowspan=1 colspan=1>120</td><td rowspan=1 colspan=1>28.7%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>58</td><td rowspan=1 colspan=1>12.9%</td><td rowspan=1 colspan=1>38</td><td rowspan=1 colspan=1>12.5%</td><td rowspan=1 colspan=1>35</td><td rowspan=1 colspan=1>8.8%</td></tr><tr><td rowspan=1 colspan=1>Influenza A H1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%</td></tr><tr><td rowspan=1 colspan=1>Influenza A H1NIpdm09</td><td rowspan=1 colspan=1>85</td><td rowspan=1 colspan=1>4.3%</td><td rowspan=1 colspan=1>67</td><td rowspan=1 colspan=1>16.0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>0.9%</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>3.3%</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>1.0%</td></tr><tr><td rowspan=1 colspan=1>Influenza H3</td><td rowspan=1 colspan=1>163</td><td rowspan=1 colspan=1>8.2%</td><td rowspan=1 colspan=1>52</td><td rowspan=1 colspan=1>12.4%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>52</td><td rowspan=1 colspan=1>11.6%</td><td rowspan=1 colspan=1>28</td><td rowspan=1 colspan=1>9.2%</td><td rowspan=1 colspan=1>31</td><td rowspan=1 colspan=1>7.8%</td></tr><tr><td rowspan=1 colspan=1>Influenza B</td><td rowspan=1 colspan=1>123</td><td rowspan=1 colspan=1>6.2%</td><td rowspan=1 colspan=1>58</td><td rowspan=1 colspan=1>13.9%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>32</td><td rowspan=1 colspan=1>7.1%</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>2.3%</td><td rowspan=1 colspan=1>26</td><td rowspan=1 colspan=1>6.5%</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza virus 1</td><td rowspan=1 colspan=1>19</td><td rowspan=1 colspan=1>1.0%</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0.5%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.2%</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0.4%</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>1.3%</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>2.5%</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza virus 2</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0.1%</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0.5%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza virus 3</td><td rowspan=1 colspan=1>116</td><td rowspan=1 colspan=1>5.8%</td><td rowspan=1 colspan=1>23</td><td rowspan=1 colspan=1>5.5%</td><td rowspan=1 colspan=1>19</td><td rowspan=1 colspan=1>4.5%</td><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>3.6%</td><td rowspan=1 colspan=1>23</td><td rowspan=1 colspan=1>7.6%</td><td rowspan=1 colspan=1>35</td><td rowspan=1 colspan=1>8.8%</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza virus 4</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>0.3%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.2%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.2%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0.8%</td></tr><tr><td rowspan=1 colspan=1>Respiratory SyncytialVirus</td><td rowspan=1 colspan=1>217</td><td rowspan=1 colspan=1>10.9%</td><td rowspan=1 colspan=1>64</td><td rowspan=1 colspan=1>15.3%</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>9.4%</td><td rowspan=1 colspan=1>35</td><td rowspan=1 colspan=1>7.8%</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>13.2%</td><td rowspan=1 colspan=1>38</td><td rowspan=1 colspan=1>9.5%</td></tr><tr><td rowspan=1 colspan=1>Bordetella Pertussis</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>0.5%</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0.5%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.2%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>2.0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%</td></tr><tr><td rowspan=1 colspan=1>Chlamydophilapneumoniae</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>0.3%</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0.5%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.2%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.2%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.3%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.3%</td></tr><tr><td rowspan=1 colspan=1>Mycoplasma pneumoniae</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>1.2%</td><td rowspan=1 colspan=1>19</td><td rowspan=1 colspan=1>4.5%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0.4%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.3%</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0.5%</td></tr></table>

Table 35: QIAstat-Dx Respiratory Panel Expected Values by Age Group   

<table><tr><td colspan="1" rowspan="2">Organism</td><td colspan="2" rowspan="1">Overall (n=1994)</td><td colspan="2" rowspan="1">&lt;=5 yrs (n=626)</td><td colspan="2" rowspan="1">6-21 yrs (n=240)</td><td colspan="2" rowspan="1">22-49 yrs (n=330)</td><td colspan="2" rowspan="1">&gt;49 yrs (n=798)</td></tr><tr><td colspan="1" rowspan="1">N</td><td colspan="1" rowspan="1">Expectedvalue</td><td colspan="1" rowspan="1">N</td><td colspan="1" rowspan="1">Expectedvalue</td><td colspan="1" rowspan="1">N</td><td colspan="1" rowspan="1">Expectedvalue</td><td colspan="1" rowspan="1">N</td><td colspan="1" rowspan="1">Expectedvalue</td><td colspan="1" rowspan="1">N</td><td colspan="1" rowspan="1">Expectedvalue</td></tr><tr><td colspan="1" rowspan="1">Adenovirus</td><td colspan="1" rowspan="1">102</td><td colspan="1" rowspan="1">5.1%</td><td colspan="1" rowspan="1">78</td><td colspan="1" rowspan="1">12.4%</td><td colspan="1" rowspan="1">7</td><td colspan="1" rowspan="1">2.9%</td><td colspan="1" rowspan="1">11</td><td colspan="1" rowspan="1">3.3%</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">0.8%</td></tr><tr><td colspan="1" rowspan="1">Coronavirus 229E</td><td colspan="1" rowspan="1">8</td><td colspan="1" rowspan="1">0.4%</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">0.6%</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">1.7%</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0%</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0%</td></tr><tr><td colspan="1" rowspan="1">Coronavirus HKU1</td><td colspan="1" rowspan="1">58</td><td colspan="1" rowspan="1">2.9%</td><td colspan="1" rowspan="1">29</td><td colspan="1" rowspan="1">4.6%</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">2.1%</td><td colspan="1" rowspan="1">8</td><td colspan="1" rowspan="1">2.4%</td><td colspan="1" rowspan="1">16</td><td colspan="1" rowspan="1">2.0%</td></tr><tr><td colspan="1" rowspan="1">Coronavirus NL63</td><td colspan="1" rowspan="1">42</td><td colspan="1" rowspan="1">2.1%</td><td colspan="1" rowspan="1">25</td><td colspan="1" rowspan="1">4.0%</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">1.3%</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">1.5%</td><td colspan="1" rowspan="1">9</td><td colspan="1" rowspan="1">1.1%</td></tr><tr><td colspan="1" rowspan="1">Coronavirus OC43</td><td colspan="1" rowspan="1">30</td><td colspan="1" rowspan="1">1.5%</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">3.2%</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0.8%</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">1.2%</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">0.5%</td></tr><tr><td colspan="1" rowspan="1">HumanMetapneumovirus</td><td colspan="1" rowspan="1">120</td><td colspan="1" rowspan="1">6.0%</td><td colspan="1" rowspan="1">46</td><td colspan="1" rowspan="1">7.3%</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">1.3%</td><td colspan="1" rowspan="1">17</td><td colspan="1" rowspan="1">5.2%</td><td colspan="1" rowspan="1">54</td><td colspan="1" rowspan="1">6.8%</td></tr><tr><td colspan="1" rowspan="1">HumanRhinovirus/Enterovirus</td><td colspan="1" rowspan="1">304</td><td colspan="1" rowspan="1">15.2%</td><td colspan="1" rowspan="1">186</td><td colspan="1" rowspan="1">29.7%</td><td colspan="1" rowspan="1">35</td><td colspan="1" rowspan="1">14.6%</td><td colspan="1" rowspan="1">22</td><td colspan="1" rowspan="1">6.7%</td><td colspan="1" rowspan="1">61</td><td colspan="1" rowspan="1">7.6%</td></tr><tr><td colspan="1" rowspan="1">Influenza A</td><td colspan="1" rowspan="1">251</td><td colspan="1" rowspan="1">12.6%</td><td colspan="1" rowspan="1">47</td><td colspan="1" rowspan="1">7.5%</td><td colspan="1" rowspan="1">36</td><td colspan="1" rowspan="1">15.1%</td><td colspan="1" rowspan="1">64</td><td colspan="1" rowspan="1">19.4%</td><td colspan="1" rowspan="1">104</td><td colspan="1" rowspan="1">13.0%</td></tr><tr><td colspan="1" rowspan="2">Organism</td><td colspan="2" rowspan="1">Overall (n=1994)</td><td colspan="2" rowspan="1">&lt;=5 yrs (n=626)</td><td colspan="2" rowspan="1">6-21 yrs (n=240)22-49 yrs (n=330)</td><td colspan="2" rowspan="1">6-21 yrs (n=240)22-49 yrs (n=330)</td><td colspan="2" rowspan="1">&gt;49 yrs (n=798)</td></tr><tr><td colspan="1" rowspan="1">N</td><td colspan="1" rowspan="1">Expectedvalue</td><td colspan="1" rowspan="1">N</td><td colspan="1" rowspan="1">Expectedvalue</td><td colspan="1" rowspan="1">N</td><td colspan="1" rowspan="1">Expectedvalue</td><td colspan="1" rowspan="1">N</td><td colspan="1" rowspan="1">Expectedvalue</td><td colspan="1" rowspan="1">N</td><td colspan="1" rowspan="1">Expectedvalue</td></tr><tr><td colspan="1" rowspan="1">Influenza A H1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0%</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0%</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0%</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0%</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0%</td></tr><tr><td colspan="1" rowspan="1">Influenza A H1N1pdm09</td><td colspan="1" rowspan="1">85</td><td colspan="1" rowspan="1">4.3%</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">3.2%</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">2.5%</td><td colspan="1" rowspan="1">30</td><td colspan="1" rowspan="1">9.1%</td><td colspan="1" rowspan="1">29</td><td colspan="1" rowspan="1">3.6%</td></tr><tr><td colspan="1" rowspan="1">Influenza H3</td><td colspan="1" rowspan="1">163</td><td colspan="1" rowspan="1">8.2%</td><td colspan="1" rowspan="1">25</td><td colspan="1" rowspan="1">4.0%</td><td colspan="1" rowspan="1">30</td><td colspan="1" rowspan="1">12.6%</td><td colspan="1" rowspan="1">35</td><td colspan="1" rowspan="1">10.6%</td><td colspan="1" rowspan="1">73</td><td colspan="1" rowspan="1">9.1%</td></tr><tr><td colspan="1" rowspan="1">Influenza B</td><td colspan="1" rowspan="1">123</td><td colspan="1" rowspan="1">6.2%</td><td colspan="1" rowspan="1">11</td><td colspan="1" rowspan="1">1.8%</td><td colspan="1" rowspan="1">22</td><td colspan="1" rowspan="1">9.2%</td><td colspan="1" rowspan="1">27</td><td colspan="1" rowspan="1">8.2%</td><td colspan="1" rowspan="1">63</td><td colspan="1" rowspan="1">7.9%</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza virus 1</td><td colspan="1" rowspan="1">19</td><td colspan="1" rowspan="1">1.0%</td><td colspan="1" rowspan="1">11</td><td colspan="1" rowspan="1">1.8%</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0%</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">1.2%</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">0.5%</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza virus 2</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0.1%</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0.2%</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0%</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0%</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0.1%</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza virus 3</td><td colspan="1" rowspan="1">116</td><td colspan="1" rowspan="1">5.8%</td><td colspan="1" rowspan="1">70</td><td colspan="1" rowspan="1">11.2%</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">1.7%</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">1.8%</td><td colspan="1" rowspan="1">36</td><td colspan="1" rowspan="1">4.5%</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza virus 4</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">0.3%</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">0.6%</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0%</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0%</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0.1%</td></tr><tr><td colspan="1" rowspan="1">Respiratory SyncytialVirus</td><td colspan="1" rowspan="1">217</td><td colspan="1" rowspan="1">10.9%</td><td colspan="1" rowspan="1">135</td><td colspan="1" rowspan="1">21.5%</td><td colspan="1" rowspan="1">11</td><td colspan="1" rowspan="1">4.6%</td><td colspan="1" rowspan="1">17</td><td colspan="1" rowspan="1">5.2%</td><td colspan="1" rowspan="1">54</td><td colspan="1" rowspan="1">6.8%</td></tr><tr><td colspan="1" rowspan="1">Bordetella Pertussis</td><td colspan="1" rowspan="1">9</td><td colspan="1" rowspan="1">0.5%</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">0.8%</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0.8%</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0%</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0.3%</td></tr><tr><td colspan="1" rowspan="1">Chlamydophilapneumoniae</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">0.3%</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0.2%</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">1.3%</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0.6%</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0%</td></tr><tr><td colspan="1" rowspan="1">Mycoplasma pneumoniae</td><td colspan="1" rowspan="1">24</td><td colspan="1" rowspan="1">1.2%</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">0.6%</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">2.5%</td><td colspan="1" rowspan="1">11</td><td colspan="1" rowspan="1">3.3%</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0.4%</td></tr></table>

# N. Instrument Name:

QIAstat-Dx Analyzer 1.0

# O. System Descriptions:

1. Modes of Operation:

The QIAstat-Dx Analyzer 1.0 is a portable bench-top unit designed to perform in laboratory and point-of care environments. QIAstat-Dx Respiratory Panel cartridges are keyed to match the corresponding holder on the instrument and in only one direction. Once the cartridge has been inserted into the instrument, the test starts automatically and runs for approximately 74 minutes. When the test is finished, the cartridge is removed by the user and discarded. The QIAstat- Dx Analyzer 1.0 automatically interprets test results and displays a summary on the analyzer display screen. The results can be printed using a connected printer. The Analyzer is equipped with six optical measurement channels employing six fluorescence modules, FAM, NED, ROX, VIC, Cy5, and Cy5.5. Each optical module consists of a LED excitation source and photodiode receiver configured to the fluorescent channel. Fluorescence signals are processed by software running on an on-board computer.

Does the applicant’s device contain the ability to transmit data to a computer, webserver, or mobile device?

Does the applicant’s device transmit data to a computer, webserver, or mobile device using wireless transmission?

Yes or No ___X_

2. Software:

FDA has reviewed applicant’s Hazard Analysis and software development processes for this line of product types:

Yes ___X_ _ or No

3. Specimen Identification: Specimen ID is entered using on screen keyboard or barcode scanner.

4. Specimen Sampling and Handling: The specimens are manually inserted in the Sample Receiver in the instrument.

5. Calibration:

The QIAstat-Dx Analyzer 1.0 is provided factory calibrated and does not require user calibration. The Analyzer 1.0 includes self-check controls to verify the performance of all sensors and actuators and will alert the user in case of failure. The company will maintain calibration of the instrument through calibration checks performed during QIAGEN technical service visits

6. Quality Control:

Quality control is addressed for each specific FDA-cleared assay to be run on the instrument (separately cleared).

P. Other Supportive Instrument Performance Characteristics Data Not Covered In The “Performance Characteristics” Section above:

Not applicable.

# Q. Proposed Labeling:

The labeling supports the finding of substantial equivalence for this device.

# R. Conclusion:

The submitted information in this premarket notification is complete and supports a substantial equivalence decision.