# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATIONDECISION SUMMARYASSAY AND INSTRUMENT

I Background Information:

A 510(k) Number K190275   
B Applicant GenePOC Inc.   
C Proprietary and Established Names GenePOC Carba   
D Regulatory Information

<table><tr><td rowspan=1 colspan=1>ProductCode(s)</td><td rowspan=1 colspan=1>Classification</td><td rowspan=1 colspan=1>RegulationSection</td><td rowspan=1 colspan=1>Panel</td></tr><tr><td rowspan=1 colspan=1>PMY</td><td rowspan=1 colspan=1>Class II</td><td rowspan=1 colspan=1>21 CFR 866.1640-AntimicrobialSusceptibility Test Powder</td><td rowspan=1 colspan=1>IM - Immunology &amp;MI - Microbiology</td></tr><tr><td rowspan=1 colspan=1>0OI</td><td rowspan=1 colspan=1>Class II</td><td rowspan=1 colspan=1>21 CFR 862.2570Instrumentation forclinical multiplex testsystems</td><td rowspan=1 colspan=1>CH - Clinical Chemistry</td></tr></table>

# II Submission/Device Overview:

# A Purpose for Submission:

To obtain a substantial equivalence determination for the GenePOC Carba assay performed on the revogene instrument.

# B Measurand:

Conserved DNA sequences of the following carbapenemase genes: blaKPC, blaNDM, blaVIM, blaOXA-48-like and blaIMP.

# C Type of Test:

Qualitative real-time DNA amplification and detection assay.

# III Intended Use/Indications for Use:

# A Intended Use(s):

The GenePOC Carba assay, performed on the revogene instrument, is a qualitative in vitro diagnostic test designed for the detection and differentiation of the blaKPC, blaNDM, blaVIM, blaOXA-48-like, and blaIMP gene sequences associated with carbapenem-non-susceptible pure colonies of Enterobacteriaceae, Acinetobacter baumannii, or Pseudomonas aeruginosa, when grown on blood agar or MacConkey agar. The test utilizes automated real-time Polymerase Chain Reaction (PCR).

The GenePOC Carba assay should be used in conjunction with other laboratory tests including phenotypic antimicrobial susceptibility testing. A negative GenePOC Carba assay result does not preclude the presence of other resistance mechanisms.

The GenePOC Carba assay is intended as an aid for infection control in the detection of carbapenem-non-susceptible bacteria that colonize patients in healthcare settings. The identification of a blaIMP, blaNDM or blaVIM metallo-beta-lactamase gene (i.e., the genes that encode the IMP, NDM and VIM metallo-beta-lactamases, respectively) may be used as an aid to clinicians in determining appropriate therapeutic strategies for patients with known or suspected carbapenem non-susceptible infections.

B Indication(s) for Use: Same as Intended Use

C Special Conditions for Use Statement(s): Rx - For Prescription Use Only

Organisms should be identified and carbapenem non-susceptibility status should be determined prior to testing with the GenePOC Carba assay.

The performance of the GenePOC Carba assay with bacteria other than Enterobacteriaceae, Acinetobacter baumannii or Pseudomonas aeruginosa, has not been evaluated.

In silico predictions of the inclusivity of the GenePOC Carba primers and probes were performed using sequence data available in GenBank in 2018. Analysis of new variant sequences of the targeted carbapenemase genes deposited in GenBank after 2018 has not been performed.

# D Special Instrument Requirements:

GenePOC, Inc. revogene instrument

# A Device Description:

The GenePOC Carba assay is a qualitative in vitro diagnostic real-time PCR-based test for the detection and differentiation of five $\beta$ -lactamase genes (blaKPC, blaNDM, blaVIM, blaOXA-48-like and blaIMP) that are associated with non-susceptibility of Gram negative bacteria to carbapenem antimicrobial agents. The assay is intended for use on isolated, pure colonies of Enterobacteriaceae, Acinetobacter baumannii and Pseudomonas aeruginosa grown on solid culture media as an aid in detection of carbapenem non-susceptible organisms for the purposes of infection control and to aid in determining appropriate therapeutic strategies for patients with known or suspected infection with such organisms.

The GenePOC Carba assay kit comprises 24 individual pouches, each of which contains a singleuse, disposable microfluidic cartridge (PIE), Sample Buffer Tube (SBT) and a Disposable Transfer Tool (DTT). Each PIE contains a slurry of glass beads for sample homogenization and bacterial lysis, a Process Control $\mathrm { ( P r C ) }$ , a buffer for dilution of the lysate and primers and probes for detection of each of the targeted carbapenemase genes. The $\mathrm { P r } { \bf C }$ is used to verify the sample processing and amplification steps, including the detection of potential assay inhibitors, microfluidic movement of liquid and reagent integrity.

Each PIE is an integrated closed device in which a sample is dispensed and processed through different microfluidic chambers and channels for nucleic acid extraction and subsequent realtime PCR amplification/detection. User intervention is required to prepare a standardized 0.5 McFarland bacterial suspension from previously identified, carbapenem non-susceptible colonies, add $1 5 \mu \mathrm { L }$ of this suspension to the SBT and subsequently transfer an aliquot from the SBT to the PIE, and load the PIE into the revogene carousel. Sample preparation and real-time PCR are completed in approximately 70 minutes. Upon completion of a run, the results for each target gene are interpreted automatically by the revogene instrument and reported as Positive, Negative, Indeterminate (instrument error) or Unresolved ( $\mathrm { P r C }$ failure).

# B Principle of Operation:

The GenePOC Carba assay is for the qualitative detection and differentiation of the blaKPC, blaNDM, blaVIM, blaOXA-48-like and blaIMP genes in isolated colonies of carbapenem non-susceptible Enterobacteriaceae, Acinetobacter baumannii and Pseudomonas aeruginosa grown on sheep blood or MacConkey agar. The assay is performed on the revogene instrument (K170558) in single-use, disposable microfluidic cartridges (PIEs) that automate sample processing, microfluidic movement of liquid and real-time PCR amplification/detection. No operator intervention is necessary once the sample is loaded onto the revogene instrument.

To perform the test, a previously identified carbapenem non-susceptible bacterial strain is streaked on blood or MacConkey agar to obtain isolated colonies. A $1 0 \mu \ g$ meropenem disc is placed in the first quadrant of the plate to monitor retention of non-susceptibility to carbapenems. After incubation for 18-24 hours at $3 5 { \pm } 2 ^ { \circ } \mathrm { C }$ , isolated colonies are sampled and used to prepare a 0.5 McFarland suspension in saline. Fifteen microliters of the standardized suspension are then added to a Sample Buffer Tube (SBT) and, after mixing, a Disposable Transfer Tool (DTT) provided in the assay kit is used to transfer an aliquot of the diluted suspension to a GenePOC Carba PIE which is then sealed and loaded into the revogene instrument.

The revogene can process between one and eight samples (PIEs) in a single run. Each PIE is an integrated, self-contained device that performs sample homogenization/bacterial lysis using glass beads, sample dilution and PCR amplification/detection. Fluid movement within each PIE is controlled by centrifugation through microfluidic channels. An internal Process Control $\mathrm { ( P r C ) }$ is included in the homogenization chamber to monitor reagent and process integrity.

The results are interpreted automatically by the system from measured fluorescent signals and may be viewed, printed, transferred, and/or stored by the user.

# C Instrument Description Information:

<table><tr><td rowspan=1 colspan=1>Modes of Operation</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>No</td></tr><tr><td rowspan=1 colspan=1>Does the applicant&#x27;s device contain the ability to transmit data to a computer,webserver, or mobile device?</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>□</td></tr><tr><td rowspan=1 colspan=1>Does the applicant&#x27;s device transmit data to a computer, webserver, or mobile deviceusing wireless transmission?</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>X</td></tr><tr><td rowspan=1 colspan=3>Software</td></tr><tr><td rowspan=1 colspan=1>FDA has reviewed applicant&#x27;s Hazard Analysis and software development processesfor this line of product types.</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1></td></tr></table>

1. Instrument Name:

GenePOC revogene instrument

2. Specimen Identification:

The revogene instrument has two barcode readers to identify reagents and patient specimens. The Sample Buffer Tube (SBT) and microfluidic cartridge (PIE) are each pre-labeled with a unique barcode at the time of manufacture to identify both the specimen and assay type, providing traceability of each sample to the PIE, SBT and assay identifiers.

# 3. Specimen Sampling and Handling:

User intervention is required to prepare a standardized bacterial suspension from isolated colonies of a previously identified, carbapenem non-susceptible bacterial isolate, transfer an aliquot of the suspension to the Sample Buffer Tube (SBT) and from there to the microfluidic cartridge (PIE). All further processing is automated once the PIE is loaded into the revogene instrument and the run is initiated. Results are interpreted automatically by the system.

The revogene instrument is factory calibrated at the time of manufacture. Calibration is verified annually by a trained technician.

5. Quality Control:

Each microfluidic cartridge (PIE) contains a Process Control $\mathrm { ( P r C ) }$ in the homogenization chamber that is designed to monitor process and reagent integrity. The $\mathrm { P r } { \bf C }$ is mixed with the patient sample and undergoes simultaneous lysis, PCR amplification/detection.

Commercially available strains of bacteria may be used as External Controls for the GenePOC Carba assay in accordance with the requirements of local, state or federal regulations and accrediting agencies (refer to Section VII A(5)).

# V Substantial Equivalence Information:

A Predicate Device Name(s): Xpert Carba-R

B Predicate 510(k) Number(s): K152614

C Comparison with Predicate(s):

<table><tr><td colspan="1" rowspan="1">Device &amp; PredicateDevice(s):</td><td colspan="1" rowspan="1">K190275</td><td colspan="1" rowspan="1">K152614</td></tr><tr><td colspan="1" rowspan="1">Device Trade Name</td><td colspan="1" rowspan="1">GenePOC Carba</td><td colspan="1" rowspan="1">Xpert Carba-R Assay</td></tr><tr><td colspan="1" rowspan="1">General DeviceCharacteristicSimilarities</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Intended Use/IndicationsFor Use</td><td colspan="1" rowspan="1">The GenePOC Carbaassay, performed on therevogene instrument, is aqualitative in vitrodiagnostic test designed forthe detection anddifferentiation of theblakpc, blandm, blavIm,blaoxA-48-like, and blaIMPgene sequences associatedwith carbapenem-non-susceptible pure coloniesof Enterobacteriaceae,Acinetobacter baumannii,or Pseudomonasaeruginosa, when grownon blood agar or</td><td colspan="1" rowspan="1">The Xpert Carba-R Assay,performed on theGeneXpert InstrumentSystems, is a qualitative invitro diagnostic test for thedetection anddifferentiation of theblakpc, blandm, blavim,blaoxA-48, and blaIMPgene sequences associatedwith carbapenem-non-susceptible pure coloniesof Enterobacteriaceae,Acinetobacter baumannii,or Pseudomonasaeruginosa grown onblood agar or MacConkey</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">MacConkey agar. The testutilizes automated real-time Polymerase ChainReaction (PCR).The GenePOC Carba assayshould be used inconjunction with otherlaboratory tests includingphenotypic antimicrobialsusceptibility testing. Anegative GenePOC Carbaassay result does notpreclude the presence ofother resistancemechanisms.The GenePOC Carba assayis intended as an aid forinfection control in thedetection of carbapenem-non-susceptible bacteriathat colonize patients inhealthcare settings. Theidentification of a blamp,blaNDM or blavIm metallo-β-lactamase gene (i.e., thegenes that encode the IMP,NDM and VIM metallo-β-lactamases, respectively)may be used as an aid toclinicians in determiningappropriate therapeuticstrategies for patients withknown or suspectedcarbapenem non-susceptible infections.</td><td colspan="1" rowspan="1">agar. The test utilizesautomated real-timepolymerase chain reaction(PCR).A negative Xpert Carba-RAssay result does notpreclude the presence ofother resistancemechanisms. The XpertCarba-R Assay should beused in conjunction withother laboratory testsincluding phenotypicantimicrobial susceptibilitytesting. The Xpert Carba-RAssay is intended as an aidfor infection control indetecting anddifferentiating geneticmarkers of resistance tomonitor the spread ofcarbapenem-non-susceptible organisms inhealthcare settings. TheXpert Carba-R Assay isnot intended to guide ormonitor treatment forcarbapenem-non-susceptible bacterialinfections.</td></tr><tr><td colspan="1" rowspan="1">Technological Principles</td><td colspan="1" rowspan="1">Automated DNAextraction, nucleic acidamplification andfluorescence-based real-time Polymerase Chainreaction (PCR) detection</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Test Cartridge</td><td colspan="1" rowspan="1">Disposable, single-use,multichambered fluidiccartridge</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="2" rowspan="1">Device &amp; PredicateDevice(s):</td><td colspan="1" rowspan="1">K190275</td><td colspan="1" rowspan="1">K152614</td></tr><tr><td colspan="2" rowspan="1">Targeted CarbapenemaseGenes</td><td colspan="1" rowspan="1">blakpc, blandm, blavm,blaoxA-48 and blaMP</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="2" rowspan="1">Bacterial Species</td><td colspan="1" rowspan="1">Enterobacteriaceae,Acinetobacter baumannii,and Pseudomonasaeruginosa</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="2" rowspan="1">Sample Type</td><td colspan="1" rowspan="1">Isolated colonies ofcarbapenem non-susceptible Gram negativebacteria on blood orMacConkey agar</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="2" rowspan="1">General DeviceCharacteristicDifferences</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="2" rowspan="1">Instrument System</td><td colspan="1" rowspan="1">revogene instrument</td><td colspan="1" rowspan="1">GeneXpert InstrumentSystem (GeneXpert Dx,Infinity-48, Infinity-48sand Infinity-80)</td></tr><tr><td colspan="1" rowspan="5">Gene variantsdetected(based onanalyticalsuddies) 1</td><td colspan="1" rowspan="1">blakpC</td><td colspan="1" rowspan="1">2,3,4</td><td colspan="1" rowspan="1">2, 3, 4</td></tr><tr><td colspan="1" rowspan="1">blaNDM</td><td colspan="1" rowspan="1">1,4-7</td><td colspan="1" rowspan="1">1, 2, 4, 5</td></tr><tr><td colspan="1" rowspan="1">blaMP</td><td colspan="1" rowspan="1">1, 4, 8, 9, 11</td><td colspan="1" rowspan="1">1, 2, 4, 6, 10, 11</td></tr><tr><td colspan="1" rowspan="1">blaoxA-48</td><td colspan="1" rowspan="1">48, 181, 204, 232</td><td colspan="1" rowspan="1">48, 181</td></tr><tr><td colspan="1" rowspan="1">blavIM</td><td colspan="1" rowspan="1">1, 2, 10, 19</td><td colspan="1" rowspan="1">1, 2, 4, 10, 19</td></tr><tr><td colspan="1" rowspan="5">Additionalgene variantswhosedetection ispredictedbased on insilico analysis 1</td><td colspan="1" rowspan="1">blakpc</td><td colspan="1" rowspan="1">5-38</td><td colspan="1" rowspan="1">5-16</td></tr><tr><td colspan="1" rowspan="1">blaNDM</td><td colspan="1" rowspan="1">2, 3, 8-24</td><td colspan="1" rowspan="1">3, 6-9</td></tr><tr><td colspan="1" rowspan="1">blaiMP</td><td colspan="1" rowspan="1">2, 5, 6, 10, 17-20, 23-26,28-30, 32, 33, 37, 38, 40,42, 45, 47-49, 53-56, 59,60, 62, 66, 69-72, 74-79</td><td colspan="1" rowspan="1">3, 8, 9, 13, 19 to 22, 24,25, 27, 28, 30, 31, 33, 37,40, 42</td></tr><tr><td colspan="1" rowspan="1">blaoXA-48</td><td colspan="1" rowspan="1">162, 199, 204, 244, 245,252, 370, 484, 505, 514,515, 519, 546, 547, 566</td><td colspan="1" rowspan="1">162, 163, 204, 232, 244,245, 247</td></tr><tr><td colspan="1" rowspan="1">blavIM</td><td colspan="1" rowspan="1">3-6, 8, 9, 11, 12, 14-18, 20,23-46, 48-50, 52-55, 57,59,60</td><td colspan="1" rowspan="1">5-9, 11 to 18, 20, 23-38</td></tr></table>

Note: The variants listed reflect the respective labeling of each device and the analytical studies and in silico analyses conducted at the time of 510(k) clearance.

# VI Standards/Guidance Documents Referenced:

CLSI. Evaluation of Stability of In Vitro Diagnostic Reagents; Approved Guideline. CLSI document EP25-A. Wayne, PA: Clinical and Laboratory Standards Institute; 2009.

CLSI. Defining, Establishing, and Verifying Reference Intervals in the Clinical Laboratory; Approved Guideline - Third Edition. CLSI document EP28-A3c. Wayne, PA: Clinical and Laboratory Standards Institute; 2008.

CLSI. Performance Standards for Antimicrobial Disk Susceptibility Tests. $1 3 ^ { \mathrm { t h } }$ ed. CLSI standard M02. Wayne, PA: Clinical and Laboratory Standards Institute; 2018.

CLSI. Performance Standards for Antimicrobial Susceptibility Testing. $2 8 ^ { \mathrm { t h } }$ ed. CLSI supplement M100. Wayne, PA: Clinical and Laboratory Standards Institute; 2018.

# VII Performance Characteristics (if/when applicable):

# A Analytical Performance:

# 1. Precision/Reproducibility:

The precision and reproducibility of the GenePOC Carba assay was evaluated in a study conducted at three sites using an 11-member panel of samples comprised of representative strains of bacteria that between them carried each of the targeted carbapenemase gene markers (Table 1). Each panel member was prepared at standardized concentration of 0.5 McFarland and frozen prior to distribution to the study sites for testing.

Table 1. Panel members used to evaluate the precision and reproducibility of the GenePOC Carba assay   

<table><tr><td rowspan=1 colspan=1>Panel Member</td><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>Species</td><td rowspan=1 colspan=1>CarbapenemaseGene Variant</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>ATCC BAA-2146</td><td rowspan=1 colspan=1>Klebsiella pneumoniae</td><td rowspan=1 colspan=1>blaNDM-1</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>CCRI 23064</td><td rowspan=1 colspan=1>Escherichia coli</td><td rowspan=1 colspan=1>blaNDM-5</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>CCUG 59348</td><td rowspan=1 colspan=1>Klebsiella pneumoniae</td><td rowspan=1 colspan=1>blakpC-2</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>CCRI 21578</td><td rowspan=1 colspan=1>Enterobacter cloacae</td><td rowspan=1 colspan=1>blakpC-4</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>ATCC BAA-2523</td><td rowspan=1 colspan=1>Escherichia coli</td><td rowspan=1 colspan=1>blaoxA-48</td></tr><tr><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>CCRI 22264</td><td rowspan=1 colspan=1>Klebsiella pneumoniae</td><td rowspan=1 colspan=1>blaoxA-181</td></tr><tr><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>NCTC 13476</td><td rowspan=1 colspan=1>Escherichia coli</td><td rowspan=1 colspan=1>blaMP-1</td></tr><tr><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>CCRI 19583</td><td rowspan=1 colspan=1>Klebsiella pneumoniae</td><td rowspan=1 colspan=1>blaMP-4</td></tr><tr><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>NCTC 13440</td><td rowspan=1 colspan=1>Klebsiella pneumoniae</td><td rowspan=1 colspan=1>blavIM-1</td></tr><tr><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>CCRI 19585</td><td rowspan=1 colspan=1>Klebsiella pneumoniae</td><td rowspan=1 colspan=1>blavIM-1</td></tr><tr><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>CCRI 22760</td><td rowspan=1 colspan=1>Enterobacter cloacae</td><td rowspan=1 colspan=1>Negative</td></tr></table>

ATCC: American Type Culture Collection NCTC: National Collection of Type Cultures CCRI: Culture Collection of the Centre de Researche en Infectiologie CCUG: Culture Collection, University of Gothenburg

A nested study design was used to characterize within-laboratory precision, betweenlaboratory reproducibility as well as lot-to-lot reproducibility, whereby a subset of samples that was tested with one lot of reagents in the Lot-to-Lot Reproducibility Study also served as panel members in the Within Laboratory Precision and Between Laboratory Reproducibility Studies (Table 2). As a result, overall there was a total of 200 replicates of each positive panel member and 400 replicates of the negative panel member.

Table 2. Reproducibility and Precision Study design for the GenePOC Carba assay   

<table><tr><td rowspan=2 colspan=1>Parameter</td><td rowspan=1 colspan=3>Analysis</td></tr><tr><td rowspan=1 colspan=1>Within-LaboratoryPrecision</td><td rowspan=1 colspan=1>Lot-to-LotReproducibility</td><td rowspan=1 colspan=1>Between-LaboratoryReproducibility</td></tr><tr><td rowspan=1 colspan=1>Sites</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>3</td></tr><tr><td rowspan=1 colspan=1>Operators/Site</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>2</td></tr><tr><td rowspan=1 colspan=1>Reagent Lots</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>Days ofTesting/Reagent Lot</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>5</td></tr><tr><td rowspan=1 colspan=1>Replicates per PanelMember/Day/Operator</td><td rowspan=1 colspan=1>4 positive8 negative 1</td><td rowspan=1 colspan=1>4 positive8 negative</td><td rowspan=1 colspan=1>4 positive8 negative</td></tr><tr><td rowspan=1 colspan=1>Replicates/PanelMember/Lot</td><td rowspan=1 colspan=1>40 positive80 negative</td><td rowspan=1 colspan=1>40 positive80 negative</td><td rowspan=1 colspan=1>120 positive240 negative</td></tr><tr><td rowspan=2 colspan=1>Total ReplicatesPer Panel Member</td><td rowspan=1 colspan=1>40 positive80 negative</td><td rowspan=1 colspan=1>120 positive240 negative</td><td rowspan=1 colspan=1>120 positive240 negative</td></tr><tr><td rowspan=1 colspan=3>200 positive 2400 negative</td></tr></table>

1 Because only one carbapenemase negative strain was used in the study, 8 replicates of Panel Member #11 from Table 1 were tested on each day by each operator 2 Because of the nested study design, overall there was a total of 200 replicates of each positive panel member and 400 replicates of the negative panel member.

The qualitative test results from the GenePOC Carba assay Reproducibility and Precision Study are summarized in Tables 3 and 4. An analysis of Ct and endpoint fluorescence values obtained for each panel member at each study site is provided in Table 5.

Table 3. Summary of results from the Reproducibility and Precision Study for the GenePOC Carba assay   

<table><tr><td rowspan=2 colspan=1>Analysis</td><td rowspan=1 colspan=2>Agreement by Panel Member</td></tr><tr><td rowspan=1 colspan=1>Positive 1</td><td rowspan=1 colspan=1>Negative 2</td></tr><tr><td rowspan=1 colspan=1>Within-Laboratory Precision</td><td rowspan=1 colspan=1>40/40100% (91.2-100%) 3</td><td rowspan=1 colspan=1>80/80100% (95.4-100%)</td></tr><tr><td rowspan=1 colspan=1>Lot-to-Lot Reproducibility</td><td rowspan=1 colspan=1>120/120100% (96.9-100%)</td><td rowspan=1 colspan=1>239/24099.6% (97.7-99.9%)</td></tr><tr><td rowspan=1 colspan=1>Between-LaboratoryReproducibility</td><td rowspan=1 colspan=1>120/120100% (96.9-100%)</td><td rowspan=1 colspan=1>237/24098.8% (96.4-99.6%</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>200/200100% (98.1-100%)</td><td rowspan=1 colspan=1>396/40099.0% (98.1-100%)</td></tr></table>

1 In each study, there was $100 \%$ positive agreement for Panel Members #1-10. A total of 2000 samples were tested across all 10 positive panel members (10 panel members $\textbf { x } 2 0 0$ replicates each). 2 Panel Member #11 3 Two-sided $9 5 \%$ score confidence interval

On initial testing, 34/2400 $( 1 . 4 \% )$ samples produced Indeterminate results and a further eight (8) $( 8 / 2 4 0 0 = 0 . 3 \% )$ were reported as Unresolved. In each case, upon retesting of these samples, valid results were obtained. Eleven false positive results $1 1 / 2 4 0 0 = 0 . 5 \% )$ were observed over the course of the study (blaIMP: 8; blaNDM: 3) (Table 4). The GenePOC Carba assay exhibited acceptable reproducibility and precision within and between sites/instruments/reagent lots and days of testing.

Table 4. Summary of qualitative GenePOC Carba assay results from the Reproducibility and Precision Study, stratified by carbapenemase gene variant   

<table><tr><td rowspan=2 colspan=1>Site 1</td><td rowspan=2 colspan=2>Panel Member /bla variant 2</td><td rowspan=2 colspan=1>NumberTested</td><td rowspan=1 colspan=5>GenePOC Positive</td><td rowspan=2 colspan=1>GenePOCNegative</td><td rowspan=1 colspan=2>Agreement withExpected Result</td></tr><tr><td rowspan=1 colspan=1>NDM</td><td rowspan=1 colspan=1>KPC</td><td rowspan=1 colspan=1>OXA-48 3</td><td rowspan=1 colspan=1>IMP</td><td rowspan=1 colspan=1>VIM</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>%</td></tr><tr><td rowspan=11 colspan=1>Site 1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>NDM-1</td><td rowspan=1 colspan=1>120</td><td rowspan=1 colspan=1>120</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>119</td><td rowspan=1 colspan=1>99.2</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>NDM-5</td><td rowspan=1 colspan=1>120</td><td rowspan=1 colspan=1>120</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>120</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>KPC-2</td><td rowspan=1 colspan=1>120</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>120</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>120</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>KPC-4</td><td rowspan=1 colspan=1>120</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>120</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>119</td><td rowspan=1 colspan=1>99.2</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>OXA-48</td><td rowspan=1 colspan=1>120</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>120</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>120</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>OXA-181</td><td rowspan=1 colspan=1>120</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>120</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>120</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>IMP-1</td><td rowspan=1 colspan=1>120</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>120</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>120</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>IMP-4</td><td rowspan=1 colspan=1>120</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>120</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>120</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>VIM-14</td><td rowspan=1 colspan=1>120</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>120</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>120</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>VIM-1 5</td><td rowspan=1 colspan=1>120</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>120</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>119</td><td rowspan=1 colspan=1>99.2</td></tr><tr><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>240</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>239</td><td rowspan=1 colspan=1>239</td><td rowspan=1 colspan=1>99.6</td></tr><tr><td rowspan=33 colspan=1>Site 2Site 3AllSites</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>NDM-1</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>39</td><td rowspan=1 colspan=1>97.5</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>NDM-5</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>KPC-2</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>KPC-4</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>39</td><td rowspan=1 colspan=1>97.5</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>OXA-48</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>OXA-181</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>IMP-1</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>IMP-4</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>VIM-1 4</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>VIM-1 $5f$</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>80</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>79</td><td rowspan=1 colspan=1>79</td><td rowspan=1 colspan=1>98.8</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>NDM-1</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>39</td><td rowspan=1 colspan=1>97.5</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>NDM-5</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>KPC-2</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>KPC-4</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>39</td><td rowspan=1 colspan=1>97.5</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>OXA-48</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>OXA-181</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>IMP-1</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>IMP-4</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>VIM-1 4</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>VIM-1 4</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>80</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>78</td><td rowspan=1 colspan=1>78</td><td rowspan=1 colspan=1>97.5</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>NDM-1</td><td rowspan=1 colspan=1>200</td><td rowspan=1 colspan=1>200</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>197</td><td rowspan=1 colspan=1>98.5</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>NDM-5</td><td rowspan=1 colspan=1>200</td><td rowspan=1 colspan=1>200</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>200</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>KPC-2</td><td rowspan=1 colspan=1>200</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>200</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>200</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>KPC-4</td><td rowspan=1 colspan=1>200</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>200</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>197</td><td rowspan=1 colspan=1>98.5</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>OXA-48</td><td rowspan=1 colspan=1>200</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>200</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>200</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>OXA-181</td><td rowspan=1 colspan=1>200</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>200</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>200</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>IMP-1</td><td rowspan=1 colspan=1>200</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>200</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>200</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>IMP-4</td><td rowspan=1 colspan=1>200</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>200</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>200</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>VIM-1 4</td><td rowspan=1 colspan=1>200</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>200</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>200</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>VIM-1 4</td><td rowspan=1 colspan=1>200</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>200</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>199</td><td rowspan=1 colspan=1>99.5</td></tr><tr><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>400</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>396</td><td rowspan=1 colspan=1>396</td><td rowspan=1 colspan=1>99.0</td></tr></table>

1 Data from Site 1 include results obtained with 3 different lots of GenePOC Carba reagents and 3 revogene instruments 2 Refer to Table 1 for a detailed description of the Panel Members 3 Reported by the GenePOC Carba assay as positive for a blaOXA-48-like carbapenemase gene 4 Panel Members #9 and $\# 1 0$ were prepared from different isolates that carry blaVIM-1 (Table 1)

Table 5. Summary of Ct and endpoint fluorescence values obtained with each panel member in the GenePOC Carba Reproducibility and Precision Study   

<table><tr><td rowspan=3 colspan=2>Panel Member /bla variant 1</td><td rowspan=1 colspan=12>Ct Value</td></tr><tr><td rowspan=1 colspan=3>Site 1 (n = 120) {2</td><td rowspan=1 colspan=3>Site 2 (n = 40)</td><td rowspan=1 colspan=3>Site 3 (n = 40)</td><td rowspan=1 colspan=3>Overall</td></tr><tr><td rowspan=1 colspan=1>Mean</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>Mean</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>Mean</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>Mean</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>NDM-1</td><td rowspan=1 colspan=1>29.5</td><td rowspan=1 colspan=1>1.3</td><td rowspan=1 colspan=1>4.5</td><td rowspan=1 colspan=1>30.0</td><td rowspan=1 colspan=1>1.0</td><td rowspan=1 colspan=1>3.3</td><td rowspan=1 colspan=1>30.2</td><td rowspan=1 colspan=1>0.7</td><td rowspan=1 colspan=1>2.2</td><td rowspan=1 colspan=1>29.8</td><td rowspan=1 colspan=1>1.2</td><td rowspan=1 colspan=1>4.0</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>NDM-5</td><td rowspan=1 colspan=1>28.9</td><td rowspan=1 colspan=1>1.3</td><td rowspan=1 colspan=1>4.6</td><td rowspan=1 colspan=1>29.5</td><td rowspan=1 colspan=1>1.3</td><td rowspan=1 colspan=1>4.5</td><td rowspan=1 colspan=1>31.0</td><td rowspan=1 colspan=1>1.5</td><td rowspan=1 colspan=1>4.8</td><td rowspan=1 colspan=1>29.5</td><td rowspan=1 colspan=1>1.6</td><td rowspan=1 colspan=1>5.4</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>KPC-2</td><td rowspan=1 colspan=1>30.8</td><td rowspan=1 colspan=1>2.2</td><td rowspan=1 colspan=1>7.1</td><td rowspan=1 colspan=1>29.5</td><td rowspan=1 colspan=1>1.7</td><td rowspan=1 colspan=1>5.7</td><td rowspan=1 colspan=1>29.4</td><td rowspan=1 colspan=1>1.4</td><td rowspan=1 colspan=1>4.7</td><td rowspan=1 colspan=1>30.2</td><td rowspan=1 colspan=1>2.0</td><td rowspan=1 colspan=1>6.8</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>KPC-4</td><td rowspan=1 colspan=1>29.6</td><td rowspan=1 colspan=1>2.0</td><td rowspan=1 colspan=1>6.7</td><td rowspan=1 colspan=1>28.9</td><td rowspan=1 colspan=1>1.7</td><td rowspan=1 colspan=1>5.7</td><td rowspan=1 colspan=1>31.0</td><td rowspan=1 colspan=1>1.8</td><td rowspan=1 colspan=1>5.8</td><td rowspan=1 colspan=1>29.8</td><td rowspan=1 colspan=1>2.0</td><td rowspan=1 colspan=1>6.7</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>OXA-48</td><td rowspan=1 colspan=1>28.6</td><td rowspan=1 colspan=1>1.4</td><td rowspan=1 colspan=1>5.0</td><td rowspan=1 colspan=1>29.4</td><td rowspan=1 colspan=1>1.1</td><td rowspan=1 colspan=1>3.7</td><td rowspan=1 colspan=1>32.1</td><td rowspan=1 colspan=1>1.4</td><td rowspan=1 colspan=1>4.5</td><td rowspan=1 colspan=1>29.4</td><td rowspan=1 colspan=1>1.9</td><td rowspan=1 colspan=1>6.6</td></tr><tr><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>OXA-181</td><td rowspan=1 colspan=1>28.2</td><td rowspan=1 colspan=1>1.3</td><td rowspan=1 colspan=1>4.7</td><td rowspan=1 colspan=1>27.7</td><td rowspan=1 colspan=1>1.2</td><td rowspan=1 colspan=1>4.5</td><td rowspan=1 colspan=1>28.2</td><td rowspan=1 colspan=1>0.8</td><td rowspan=1 colspan=1>2.9</td><td rowspan=1 colspan=1>28.1</td><td rowspan=1 colspan=1>1.2</td><td rowspan=1 colspan=1>4.4</td></tr><tr><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>IMP-1</td><td rowspan=1 colspan=1>29.0</td><td rowspan=1 colspan=1>1.0</td><td rowspan=1 colspan=1>3.5</td><td rowspan=1 colspan=1>28.4</td><td rowspan=1 colspan=1>1.3</td><td rowspan=1 colspan=1>4.4</td><td rowspan=1 colspan=1>30.9</td><td rowspan=1 colspan=1>1.0</td><td rowspan=1 colspan=1>3.1</td><td rowspan=1 colspan=1>29.3</td><td rowspan=1 colspan=1>1.4</td><td rowspan=1 colspan=1>4.6</td></tr><tr><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>IMP-4</td><td rowspan=1 colspan=1>29.2</td><td rowspan=1 colspan=1>1.3</td><td rowspan=1 colspan=1>4.4</td><td rowspan=1 colspan=1>28.8</td><td rowspan=1 colspan=1>1.2</td><td rowspan=1 colspan=1>4.2</td><td rowspan=1 colspan=1>30.0</td><td rowspan=1 colspan=1>0.8</td><td rowspan=1 colspan=1>2.8</td><td rowspan=1 colspan=1>29.3</td><td rowspan=1 colspan=1>1.2</td><td rowspan=1 colspan=1>4.2</td></tr><tr><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>VIM-1 3</td><td rowspan=1 colspan=1>28.1</td><td rowspan=1 colspan=1>1.3</td><td rowspan=1 colspan=1>4.5</td><td rowspan=1 colspan=1>28.1</td><td rowspan=1 colspan=1>1.4</td><td rowspan=1 colspan=1>5.0</td><td rowspan=1 colspan=1>28.5</td><td rowspan=1 colspan=1>1.0</td><td rowspan=1 colspan=1>3.4</td><td rowspan=1 colspan=1>28.2</td><td rowspan=1 colspan=1>1.2</td><td rowspan=1 colspan=1>4.1</td></tr><tr><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>VIM-1 3</td><td rowspan=1 colspan=1>28.1</td><td rowspan=1 colspan=1>0.93</td><td rowspan=1 colspan=1>3.3</td><td rowspan=1 colspan=1>27.4</td><td rowspan=1 colspan=1>0.98</td><td rowspan=1 colspan=1>3.6</td><td rowspan=1 colspan=1>27.3</td><td rowspan=1 colspan=1>0.7</td><td rowspan=1 colspan=1>2.5</td><td rowspan=1 colspan=1>27.6</td><td rowspan=1 colspan=1>1.2</td><td rowspan=1 colspan=1>4.4</td></tr><tr><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>Negative 4, 5</td><td rowspan=1 colspan=1>33.0</td><td rowspan=1 colspan=1>2.5</td><td rowspan=1 colspan=1>7.6</td><td rowspan=1 colspan=1>33.6</td><td rowspan=1 colspan=1>1.3</td><td rowspan=1 colspan=1>3.9</td><td rowspan=1 colspan=1>35.2</td><td rowspan=1 colspan=1>1.1</td><td rowspan=1 colspan=1>3.2</td><td rowspan=1 colspan=1>33.6</td><td rowspan=1 colspan=1>2.3</td><td rowspan=1 colspan=1>6.7</td></tr><tr><td rowspan=3 colspan=2>Panel Member /bla variant 1</td><td rowspan=1 colspan=6>Endpoint Fluorescence</td><td rowspan=1 colspan=6>Endpoint Fluorescence</td></tr><tr><td rowspan=1 colspan=3>Site 1 (n = 120) 2</td><td rowspan=1 colspan=3>Site 2 (n = 40)</td><td rowspan=1 colspan=3>Site 3 (n = 40)</td><td rowspan=1 colspan=2>Overall</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Mean</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>Mean</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>Mean</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>Mean</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>NDM-1</td><td rowspan=1 colspan=1>793</td><td rowspan=1 colspan=1>104</td><td rowspan=1 colspan=1>13.1</td><td rowspan=1 colspan=1>750</td><td rowspan=1 colspan=1>78</td><td rowspan=1 colspan=1>10.4</td><td rowspan=1 colspan=1>760</td><td rowspan=1 colspan=1>104</td><td rowspan=1 colspan=1>13.7</td><td rowspan=1 colspan=1>778</td><td rowspan=1 colspan=1>101</td><td rowspan=1 colspan=1>13.0</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>NDM-5</td><td rowspan=1 colspan=1>789</td><td rowspan=1 colspan=1>85</td><td rowspan=1 colspan=1>10.8</td><td rowspan=1 colspan=1>715</td><td rowspan=1 colspan=1>80</td><td rowspan=1 colspan=1>11.1</td><td rowspan=1 colspan=1>727</td><td rowspan=1 colspan=1>88</td><td rowspan=1 colspan=1>12.1</td><td rowspan=1 colspan=1>762</td><td rowspan=1 colspan=1>91</td><td rowspan=1 colspan=1>11.9</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>KPC-2</td><td rowspan=1 colspan=1>1287</td><td rowspan=1 colspan=1>164</td><td rowspan=1 colspan=1>12.7</td><td rowspan=1 colspan=1>1231</td><td rowspan=1 colspan=1>169</td><td rowspan=1 colspan=1>13.7</td><td rowspan=1 colspan=1>1712</td><td rowspan=1 colspan=1>162</td><td rowspan=1 colspan=1>9.5</td><td rowspan=1 colspan=1>1361</td><td rowspan=1 colspan=1>241</td><td rowspan=1 colspan=1>17.7</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>KPC-4</td><td rowspan=1 colspan=1>1383</td><td rowspan=1 colspan=1>154</td><td rowspan=1 colspan=1>11.2</td><td rowspan=1 colspan=1>1313</td><td rowspan=1 colspan=1>128</td><td rowspan=1 colspan=1>9.7</td><td rowspan=1 colspan=1>1466</td><td rowspan=1 colspan=1>200</td><td rowspan=1 colspan=1>13.7</td><td rowspan=1 colspan=1>1386</td><td rowspan=1 colspan=1>166</td><td rowspan=1 colspan=1>12.0</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>OXA-48</td><td rowspan=1 colspan=1>807</td><td rowspan=1 colspan=1>96</td><td rowspan=1 colspan=1>11.9</td><td rowspan=1 colspan=1>716</td><td rowspan=1 colspan=1>92</td><td rowspan=1 colspan=1>12.8</td><td rowspan=1 colspan=1>672</td><td rowspan=1 colspan=1>131</td><td rowspan=1 colspan=1>19.5</td><td rowspan=1 colspan=1>762</td><td rowspan=1 colspan=1>118</td><td rowspan=1 colspan=1>15.4</td></tr><tr><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>OXA-181</td><td rowspan=1 colspan=1>688</td><td rowspan=1 colspan=1>109</td><td rowspan=1 colspan=1>15.8</td><td rowspan=1 colspan=1>633</td><td rowspan=1 colspan=1>112</td><td rowspan=1 colspan=1>17.7</td><td rowspan=1 colspan=1>713</td><td rowspan=1 colspan=1>125</td><td rowspan=1 colspan=1>17.5</td><td rowspan=1 colspan=1>682</td><td rowspan=1 colspan=1>115</td><td rowspan=1 colspan=1>16.9</td></tr><tr><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>IMP-1</td><td rowspan=1 colspan=1>1631</td><td rowspan=1 colspan=1>258</td><td rowspan=1 colspan=1>15.8</td><td rowspan=1 colspan=1>1558</td><td rowspan=1 colspan=1>222</td><td rowspan=1 colspan=1>14.3</td><td rowspan=1 colspan=1>1384</td><td rowspan=1 colspan=1>279</td><td rowspan=1 colspan=1>20.2</td><td rowspan=1 colspan=1>1567</td><td rowspan=1 colspan=1>272</td><td rowspan=1 colspan=1>17.3</td></tr><tr><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>IMP-4</td><td rowspan=1 colspan=1>1354</td><td rowspan=1 colspan=1>257</td><td rowspan=1 colspan=1>19.0</td><td rowspan=1 colspan=1>1243</td><td rowspan=1 colspan=1>233</td><td rowspan=1 colspan=1>18.8</td><td rowspan=1 colspan=1>1326</td><td rowspan=1 colspan=1>242</td><td rowspan=1 colspan=1>18.2</td><td rowspan=1 colspan=1>1326</td><td rowspan=1 colspan=1>252</td><td rowspan=1 colspan=1>19.0</td></tr><tr><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>VIM-1 3</td><td rowspan=1 colspan=1>1833</td><td rowspan=1 colspan=1>183</td><td rowspan=1 colspan=1>10.0</td><td rowspan=1 colspan=1>1415</td><td rowspan=1 colspan=1>162</td><td rowspan=1 colspan=1>11.5</td><td rowspan=1 colspan=1>1646</td><td rowspan=1 colspan=1>171</td><td rowspan=1 colspan=1>10.4</td><td rowspan=1 colspan=1>1712</td><td rowspan=1 colspan=1>241</td><td rowspan=1 colspan=1>14.1</td></tr><tr><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>VIM-1 3</td><td rowspan=1 colspan=1>1387</td><td rowspan=1 colspan=1>171</td><td rowspan=1 colspan=1>12.3</td><td rowspan=1 colspan=1>1256</td><td rowspan=1 colspan=1>178</td><td rowspan=1 colspan=1>14.1</td><td rowspan=1 colspan=1>1693</td><td rowspan=1 colspan=1>186</td><td rowspan=1 colspan=1>11.0</td><td rowspan=1 colspan=1>1422</td><td rowspan=1 colspan=1>227</td><td rowspan=1 colspan=1>16.0</td></tr><tr><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>Negative 4,.5</td><td rowspan=1 colspan=1>794</td><td rowspan=1 colspan=1>198</td><td rowspan=1 colspan=1>25.0</td><td rowspan=1 colspan=1>716</td><td rowspan=1 colspan=1>135</td><td rowspan=1 colspan=1>18.9</td><td rowspan=1 colspan=1>673</td><td rowspan=1 colspan=1>170</td><td rowspan=1 colspan=1>25.2</td><td rowspan=1 colspan=1>755</td><td rowspan=1 colspan=1>188</td><td rowspan=1 colspan=1>25.0</td></tr></table>

SD: Standard Deviation; $\% C V$ : Percent Coefficient of Variation 1 Refer to Table 1 for a detailed description of the Panel Members 2 Data from Site 1 include results obtained with 3 different lots of GenePOC Carba reagents and 3 revogene instruments 3 Panel Members #9 and $\# 1 0$ were prepared from different isolates that carry blaVIM-1 (Table 1) 4 Ct and endpoint values shown are for the Process Control 5 For Panel Member #11: ${ \bf n } = 2 4 0$ at Site 1 and $\mathrm { n } = 8 0$ at Sites 2 and 3

2. Linearity:

Not applicable.

# 3. Analytical Specificity/Interference:

# Analytical Specificity

The analytical specificity of the GenePOC Carba assay was evaluated by testing a panel of 50 carbapenem susceptible and non-susceptible isolates of Enterobacteriaceae, A. baumannii and $P .$ . aeruginosa. The carbapenem susceptibility phenotype of each isolate (susceptible, intermediate or resistant) was determined by disc diffusion and antimicrobial resistance gene content was characterize by whole Genome Sequencing and/or targeted PCR (Table 6). Most of the isolates carried $\beta$ -lactamase gene(s) that do not confer non-susceptibility to carbapenems. Those that were non-susceptible (i.e., intermediate or resistant) to carbapenems either harbored carbapenemase markers that are not targeted by the GenPOC Carba assay or non-susceptibility was conferred by a non-carbapenemase-mediated mechanism. Each isolate was tested with the GenePOC Carba assay in triplicate using a starting suspension equivalent to ${ \sim } 4 \mathrm { M c F }$ arland (i.e., ${ \sim } 5 – 1 0$ times higher concentration than the standard workflow for the GenePOC Carba assay which uses a starting suspension density of 0.5 McFarland). No false positive results were observed other than a single replicate of K. pneumoniae isolate CCRI59359 that yielded a positive result for blaOXA-48-like. Three additional replicates of this strain were tested and all yielded the expected negative results for each of the five bla gene targets.

Table 6. Isolates used to characterize the analytical specificity of the GenePOC Carba assay   

<table><tr><td colspan="1" rowspan="2">Species</td><td colspan="1" rowspan="2">Isolate</td><td colspan="1" rowspan="2">β-Lactamase Gene(s) Identified(100% Coverage)</td><td colspan="4" rowspan="1">Resistance Phenotype (S/I/R)</td></tr><tr><td colspan="1" rowspan="1">ETP</td><td colspan="1" rowspan="1">MEM</td><td colspan="1" rowspan="1">IMP</td><td colspan="1" rowspan="1">DOR</td></tr><tr><td colspan="1" rowspan="1">Acinetobacter baumannii</td><td colspan="1" rowspan="1">CCRI-1016</td><td colspan="1" rowspan="1">TEM-90, Mbl, BlaA2, OXA-653, Zn-dependent hydrolase</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">s</td><td colspan="1" rowspan="1">S</td><td colspan="1" rowspan="1">S</td></tr><tr><td colspan="1" rowspan="1">Acinetobacter baumannii</td><td colspan="1" rowspan="1">CCRI-1017</td><td colspan="1" rowspan="1">TEM-206, SCO-1, Mbl, BlaA2, OXA-673, Zn-dependent hydrolase</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">S</td><td colspan="1" rowspan="1">S</td><td colspan="1" rowspan="1">S</td></tr><tr><td colspan="1" rowspan="1">Enterobacter aerogenes</td><td colspan="1" rowspan="1">CCRI-3853</td><td colspan="1" rowspan="1">AmpC</td><td colspan="1" rowspan="1">S</td><td colspan="1" rowspan="1">S</td><td colspan="1" rowspan="1">S</td><td colspan="1" rowspan="1">S</td></tr><tr><td colspan="1" rowspan="1">Enterobacter aerogenes</td><td colspan="1" rowspan="1">CCRI-3879</td><td colspan="1" rowspan="1">AmpC</td><td colspan="1" rowspan="1">S</td><td colspan="1" rowspan="1">S</td><td colspan="1" rowspan="1">S</td><td colspan="1" rowspan="1">S</td></tr><tr><td colspan="1" rowspan="1">Enterobacter aerogenes 1</td><td colspan="1" rowspan="1">CCRI-19495</td><td colspan="1" rowspan="1">SHV-5, AmpC</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td></tr><tr><td colspan="1" rowspan="1">Enterobacter amnigenus 2</td><td colspan="1" rowspan="1">CCRI-22353</td><td colspan="1" rowspan="1">ACT-15</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td></tr><tr><td colspan="1" rowspan="1">Enterobacter cloacae</td><td colspan="1" rowspan="1">CCRI-3852</td><td colspan="1" rowspan="1">ACT-7</td><td colspan="1" rowspan="1">I</td><td colspan="1" rowspan="1">S</td><td colspan="1" rowspan="1">S</td><td colspan="1" rowspan="1">S</td></tr><tr><td colspan="1" rowspan="1">Enterobacter cloacae</td><td colspan="1" rowspan="1">CCRI-3854</td><td colspan="1" rowspan="1">ACT-42</td><td colspan="1" rowspan="1">I</td><td colspan="1" rowspan="1">S</td><td colspan="1" rowspan="1">I</td><td colspan="1" rowspan="1">S</td></tr><tr><td colspan="1" rowspan="1">Enterobacter cloacae</td><td colspan="1" rowspan="1">CCRI-23473</td><td colspan="1" rowspan="1">No β-Lactamase Gene Identified</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td></tr><tr><td colspan="1" rowspan="1">Enterobacter cloacae</td><td colspan="1" rowspan="1">CCRI-21536</td><td colspan="1" rowspan="1">ACT-5</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td></tr><tr><td colspan="1" rowspan="1">Enterobacter cloacae</td><td colspan="1" rowspan="1">CCRI-21540</td><td colspan="1" rowspan="1">ACT-7</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">S</td><td colspan="1" rowspan="1">I</td><td colspan="1" rowspan="1">S</td></tr><tr><td colspan="1" rowspan="1">Enterobacter cloacae</td><td colspan="1" rowspan="1">CCRI-21603</td><td colspan="1" rowspan="1">ACT-7</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td></tr><tr><td colspan="1" rowspan="1">Enterobacter cloacae</td><td colspan="1" rowspan="1">CCRI-21692</td><td colspan="1" rowspan="1">ACT-14</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">S</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">I</td></tr><tr><td colspan="1" rowspan="1">Enterobacter cloacae</td><td colspan="1" rowspan="1">CCRI-22075</td><td colspan="1" rowspan="1">ACT-7</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td></tr><tr><td colspan="1" rowspan="1">Enterobacter cloacae</td><td colspan="1" rowspan="1">CCRI-22097</td><td colspan="1" rowspan="1">ACT-16</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td></tr><tr><td colspan="1" rowspan="1">Enterobacter cloacae</td><td colspan="1" rowspan="1">CCRI-23318</td><td colspan="1" rowspan="1">CTX-M-15, TEM-206, CMH-1</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">CCRI-21970</td><td colspan="1" rowspan="1">AmpC1, AmpC2, MrdA, AmpH,Y-44</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">I</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">I</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">CCUG 58541</td><td colspan="1" rowspan="1">CTX-M-14, TEM-104, MrdA, AmpC2,AmpH</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">S</td><td colspan="1" rowspan="1">S</td><td colspan="1" rowspan="1">S</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">NCTC 13441</td><td colspan="1" rowspan="1">CTX-M-15, TEM-198, MrdA, OXA-13, AmpC2</td><td colspan="1" rowspan="1">S</td><td colspan="1" rowspan="1">S</td><td colspan="1" rowspan="1">S</td><td colspan="1" rowspan="1">S</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">CCRI- 21710</td><td colspan="1" rowspan="1">AmpC2, MrdA, CTX-M-15, AmpH,OXA-13</td><td colspan="1" rowspan="1">s</td><td colspan="1" rowspan="1">s</td><td colspan="1" rowspan="1">S</td><td colspan="1" rowspan="1">S</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">CCRI-778</td><td colspan="1" rowspan="1">AmpC2, MrdA</td><td colspan="1" rowspan="1">S</td><td colspan="1" rowspan="1">S</td><td colspan="1" rowspan="1">S</td><td colspan="1" rowspan="1">S</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">CCRI-779</td><td colspan="1" rowspan="1">TEM-206, AmpC1, MrdA, AmpC2,AmpH</td><td colspan="1" rowspan="1">S</td><td colspan="1" rowspan="1">S</td><td colspan="1" rowspan="1">S</td><td colspan="1" rowspan="1">S</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">CCRI-785</td><td colspan="1" rowspan="1">TEM-206, AmpC1, MrdA, AmpC2,AmpH</td><td colspan="1" rowspan="1">S</td><td colspan="1" rowspan="1">S</td><td colspan="1" rowspan="1">S</td><td colspan="1" rowspan="1">S</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">CCRI-878</td><td colspan="1" rowspan="1">AmpC2, MrdA, AmpH</td><td colspan="1" rowspan="1">S</td><td colspan="1" rowspan="1">S</td><td colspan="1" rowspan="1">S</td><td colspan="1" rowspan="1">S</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">CCUG 55970</td><td colspan="1" rowspan="1">CTX-M-9, AmpC2, TEM-206, MrdA,AmpC1, AmpH</td><td colspan="1" rowspan="1">S</td><td colspan="1" rowspan="1">S</td><td colspan="1" rowspan="1">S</td><td colspan="1" rowspan="1">S</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">CCUG 55971</td><td colspan="1" rowspan="1">CTX-M-15, TEM-143, AmpC2</td><td colspan="1" rowspan="1">S</td><td colspan="1" rowspan="1">s</td><td colspan="1" rowspan="1">S</td><td colspan="1" rowspan="1">S</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">CCUG 55972</td><td colspan="1" rowspan="1">CTX-M-2, AmpC1, AmpC2, AmpH</td><td colspan="1" rowspan="1">S</td><td colspan="1" rowspan="1">S</td><td colspan="1" rowspan="1">S</td><td colspan="1" rowspan="1">S</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">CCUG 58540</td><td colspan="1" rowspan="1">CTX-M-15, AmpC2, TEM-206, MrdA,AMPH, OXA-13</td><td colspan="1" rowspan="1">S</td><td colspan="1" rowspan="1">S</td><td colspan="1" rowspan="1">S</td><td colspan="1" rowspan="1">S</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">CCUG 58542</td><td colspan="1" rowspan="1">CTX-M-15, MrdA, OXA-13, AmpC2</td><td colspan="1" rowspan="1">S</td><td colspan="1" rowspan="1">S</td><td colspan="1" rowspan="1">S</td><td colspan="1" rowspan="1">S</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">CCUG 58546</td><td colspan="1" rowspan="1">SHV-44, AmpH</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">S</td><td colspan="1" rowspan="1">S</td><td colspan="1" rowspan="1">S</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">NCTC 13465</td><td colspan="1" rowspan="1">SHV-85, TEM-206, AmpH</td><td colspan="1" rowspan="1">S</td><td colspan="1" rowspan="1">S</td><td colspan="1" rowspan="1">S</td><td colspan="1" rowspan="1">S</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">CCUG 54718</td><td colspan="1" rowspan="1">CTX-M-15, TEM-33, OXA-13, AmpH</td><td colspan="1" rowspan="1">S</td><td colspan="1" rowspan="1">S</td><td colspan="1" rowspan="1">S</td><td colspan="1" rowspan="1">S</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">CCUG 59358</td><td colspan="1" rowspan="1">SHV-14, OXA-13, LAP-2</td><td colspan="1" rowspan="1">S</td><td colspan="1" rowspan="1">S</td><td colspan="1" rowspan="1">S</td><td colspan="1" rowspan="1">S</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">CCUG 59349</td><td colspan="1" rowspan="1">CTX-M-15, AmpH, OXA-13,TEM- 105, SHV-11</td><td colspan="1" rowspan="1">S</td><td colspan="1" rowspan="1">S</td><td colspan="1" rowspan="1">S</td><td colspan="1" rowspan="1">S</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">CCUG 59359 3</td><td colspan="1" rowspan="1">TEM-15, SHV-70, AmpH</td><td colspan="1" rowspan="1">S</td><td colspan="1" rowspan="1">S</td><td colspan="1" rowspan="1">S</td><td colspan="1" rowspan="1">S</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">CCUG 59360</td><td colspan="1" rowspan="1">SHV-12, TEM-168, AmpH, OXA-93</td><td colspan="1" rowspan="1">S</td><td colspan="1" rowspan="1">S</td><td colspan="1" rowspan="1">S</td><td colspan="1" rowspan="1">S</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">CCRI-784</td><td colspan="1" rowspan="1">SHV-27, AmpH</td><td colspan="1" rowspan="1">S</td><td colspan="1" rowspan="1">S</td><td colspan="1" rowspan="1">S</td><td colspan="1" rowspan="1">S</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">CCRI-1015</td><td colspan="1" rowspan="1">TEM-171, SCO-1, PER-2, OXA-93,SHV-39, AmpH</td><td colspan="1" rowspan="1">S</td><td colspan="1" rowspan="1">S</td><td colspan="1" rowspan="1">S</td><td colspan="1" rowspan="1">S</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae 4</td><td colspan="1" rowspan="1">CCRI-806</td><td colspan="1" rowspan="1">OKP-B-11</td><td colspan="1" rowspan="1">S</td><td colspan="1" rowspan="1">S</td><td colspan="1" rowspan="1">S</td><td colspan="1" rowspan="1">S</td></tr><tr><td colspan="1" rowspan="1">Proteus mirabilis</td><td colspan="1" rowspan="1">CCRI-21789</td><td colspan="1" rowspan="1">No β-Lactamase Gene Identified</td><td colspan="1" rowspan="1">S</td><td colspan="1" rowspan="1">S</td><td colspan="1" rowspan="1">I</td><td colspan="1" rowspan="1">S</td></tr><tr><td colspan="1" rowspan="1">Proteus mirabilis</td><td colspan="1" rowspan="1">CCRI-825</td><td colspan="1" rowspan="1">TEM-33, CTX-M-2, OXA-23</td><td colspan="1" rowspan="1">S</td><td colspan="1" rowspan="1">S</td><td colspan="1" rowspan="1">S</td><td colspan="1" rowspan="1">S</td></tr><tr><td colspan="1" rowspan="1">Proteus mirabilis</td><td colspan="1" rowspan="1">CCRI-826</td><td colspan="1" rowspan="1">TEM-215</td><td colspan="1" rowspan="1">S</td><td colspan="1" rowspan="1">S</td><td colspan="1" rowspan="1">S</td><td colspan="1" rowspan="1">S</td></tr><tr><td colspan="1" rowspan="1">Proteus mirabilis</td><td colspan="1" rowspan="1">CCRI-831</td><td colspan="1" rowspan="1">TEM-206, CTX-M-2, OXA-23</td><td colspan="1" rowspan="1">S</td><td colspan="1" rowspan="1">S</td><td colspan="1" rowspan="1">S</td><td colspan="1" rowspan="1">S</td></tr><tr><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">CCRI-873</td><td colspan="1" rowspan="1">OXA-503</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">S</td><td colspan="1" rowspan="1">S</td><td colspan="1" rowspan="1">S</td></tr><tr><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">CCRI-1228</td><td colspan="1" rowspan="1">OXA-503</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">S</td><td colspan="1" rowspan="1">S</td><td colspan="1" rowspan="1">S</td></tr><tr><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">C72</td><td colspan="1" rowspan="1">SPM</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td></tr><tr><td colspan="1" rowspan="1">Salmonella sp.</td><td colspan="1" rowspan="1">CCRI- 8892</td><td colspan="1" rowspan="1">CTX-M-5, TEM-166, OXA-13</td><td colspan="1" rowspan="1">S</td><td colspan="1" rowspan="1">S</td><td colspan="1" rowspan="1">S</td><td colspan="1" rowspan="1">S</td></tr><tr><td colspan="1" rowspan="1">Salmonella sp.</td><td colspan="1" rowspan="1">CCRI- 8893</td><td colspan="1" rowspan="1">CTX-M-5, TEM-95, OXA-13</td><td colspan="1" rowspan="1">S</td><td colspan="1" rowspan="1">S</td><td colspan="1" rowspan="1">S</td><td colspan="1" rowspan="1">S</td></tr><tr><td colspan="1" rowspan="1">Serratia marcescens</td><td colspan="1" rowspan="1">CCRI-21537</td><td colspan="1" rowspan="1">SRT-1</td><td colspan="1" rowspan="1">S</td><td colspan="1" rowspan="1">S</td><td colspan="1" rowspan="1">I</td><td colspan="1" rowspan="1">S</td></tr><tr><td colspan="1" rowspan="1">Serratia marcescens</td><td colspan="1" rowspan="1">CCRI-23334</td><td colspan="1" rowspan="1">SME-4, SRT-1</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td></tr></table>

ETP; ertapenem; MEM; meropenem; IMP: imipenem; DOR: doripenem; S: Susceptible; I: Intermediate; R: Resistant; N/A: Not applicable: A. baumannii and $P$ . aeruginosa are intrinsically resistant to ertapenem; NCTC: National Collection of Type Cultures; CCRI: Culture Collection of the Centre de Researche en Infectiologie; CCUG: Culture Collection, University of Gothenburg 1 Klebsiella aerogenes by DNA sequencing 2 Enterobacter cloacae by DNA sequencing 3 On initial testing, 1/3 replicates produced a false positive result for blaNDM; 3/3 additional replicates produced the expected negative result for all five target genes. 4 Klebsiella quasipneumoniae by DNA sequencing

In Silico Analysis

In addition to laboratory testing, the specificity of the GenePOC Carba assay primers and probes was evaluated in silico by using the Basic Local Alignment Search Tool (BLAST) to interrogate the GenBank nucleotide collection $( \mathrm { n r / n t } )$ database. All possible primer combinations within each PCR master mix $( b l a \mathrm { \mathrm { K P C } } / b l a \mathrm { \mathrm { N D M } } / \mathrm { P r C }$ for Master Mix 1 and blaIMP/blaOXA-48-like/blaVIM for Master Mix 2) were evaluated for the potential to amplify nonspecific products. The only potential amplification products identified were those for the targeted carbapenemase resistance genes. These results are acceptable.

# Interfering Substances Study

The potential for the culture medium used for growth of bacterial colonies to interfere with the GenePOC Carba assay was evaluated by testing isolated colonies grown on blood and MacConkey agar plates from three different manufacturers. Testing was performed with representative strains of bacteria that harbored each of the carbapenemase resistance markers targeted by the GenePOC Carba assay, as well as a negative control strain. Each isolate was grown on each culture medium and suspended to a standardized concentration of 0.5 McFarland using saline from two different suppliers, resulting in a total of 12 test conditions for each organism. Three replicates of each strain were tested under each condition ((3 types of blood agar $^ { + 3 }$ types of MacConkey agar) x 2 preparations of saline x 6 strains x 3 replicates $= 2 1 6$ samples). The expected results were obtained for 214/216 $( 9 9 . 1 \% )$ samples. The two (2) discordant results occurred with a strain of $K .$ . pneumoniae that was reported, as expected, as positive for blaNDM but which also produced positive results for blaIMP (1 replicate) and blaVIM (1 replicate) under different test conditions. Retesting of freshly prepared bacterial suspensions with the same combinations of culture media and saline produced the expected results (three (3) replicates per condition). The results of the Interfering Substances Study were determined to be acceptable.

4. Assay Reportable Range:

Not applicable.

5. Traceability, Stability, Expected Values (Controls, Calibrators, or Methods):

Process Control   
Each GenePOC Carba PIE includes an integrated Process Control $\mathrm { ( P r C ) }$ that is designed to monitor sample homogenization, dilution, amplification and detection. The $\mathrm { P r } { \bf C }$ is amplified and detected using a specific pair of PCR primers and a detector probe that are located in one of the two amplification chambers of the GenePOC Carba PIE.   
External Controls   
External Positive and Negative Controls should be tested in accordance with good laboratory practices and applicable regulations and the requirements accrediting agencies. The   
manufacturer does not provide External Controls for use with the GenePOC Carba assay but recommends use of standardized 0.5 McFarland suspensions of the strains listed in Table 7

as Positive Controls. For a Negative Control, a carbapenem non-susceptible isolate that does not carry any of the resistance genes targeted by the GenePOC carba assay is recommended.

Table 7. Strains recommended for use as external Positive Controls for the GenePOC Carba assay and summary of validation data   

<table><tr><td rowspan=1 colspan=1>Species</td><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>bla Gene Target</td><td rowspan=1 colspan=1>ExpectedResult (%)</td></tr><tr><td rowspan=1 colspan=1>K. pneumoniae</td><td rowspan=1 colspan=1>ATCC BAA-2146</td><td rowspan=1 colspan=1>NDM-1</td><td rowspan=1 colspan=1>36/36 (100)</td></tr><tr><td rowspan=1 colspan=1>K. pneumoniae</td><td rowspan=1 colspan=1>CCUG 59348</td><td rowspan=1 colspan=1>KPC-2</td><td rowspan=1 colspan=1>36/36 (100)</td></tr><tr><td rowspan=1 colspan=1>E. coli</td><td rowspan=1 colspan=1>ATCC BAA-2523</td><td rowspan=1 colspan=1>OXA-48</td><td rowspan=1 colspan=1>36/36 (100)</td></tr><tr><td rowspan=1 colspan=1>E. coli</td><td rowspan=1 colspan=1>NCTC 13476</td><td rowspan=1 colspan=1>IMP-1</td><td rowspan=1 colspan=1>36/36 (100)</td></tr><tr><td rowspan=1 colspan=1>K. pneumoniae</td><td rowspan=1 colspan=1>NCTC 13440</td><td rowspan=1 colspan=1>VIM-1</td><td rowspan=1 colspan=1>36/36 (100)</td></tr></table>

ATCC: American Type Culture Collection NCTC: National Collection of Type Cultures CCUG: Culture Collection, University of Gothenburg 1 1/36 replicates produced an Indeterminate result on initial testing but gave the expected result upon repeat from the same Sample Buffer Tube

Use of the strains shown in Table 7 was validated by testing 36 replicates of each with three lots of GenePOC Carba reagents (12 replicates/lot). All controls produced the expected results.

External Positive and Negative Controls were also tested on each day of the Method Comparison Study described in Section VII B(1). The strain used for the Positive Control was selected daily on a rotating basis so that a control for each gene target was tested at least twice at each study site. A total of 52 pairs of Positive and Negative External Controls was evaluated over the course of the study. Of these, 49 pairs $( 9 4 . 2 \% )$ produced the expected results for each of the targeted carbapenemase genes on initial testing and in each case, the controls that failed produced the expected results upon repeat.

# Sample Stability

Studies were performed to evaluate the stability of bacterial suspensions in the Sample Buffer Tube (SBT) and of the GenePOC Carba PIE after removal from the dessicant pouch and addition of sample. Bacterial suspensions in the SBT were shown to be stable for 4 days at $2 5 ^ { \circ } \mathrm { C }$ and 7 days at $2 { - } 8 ^ { \circ } \mathrm { C }$ . GenePOC Carba PIEs were found to be stable for up to 1 hour at $2 5 ^ { \circ } \mathrm { C }$ after addition of sample.

# Reagent Stability

The shelf-life of the GenePOC Carba assay reagents will be established through a Real-time Stability Study conducted at $2 { - } 8 ^ { \circ } \mathrm { C }$ and $2 5 { \pm } 3 ^ { \circ } { \mathrm { C } } / 6 0 \%$ relative humidity.

6. Detection Limit:

# Analytical Sensitivity

The analytical sensitivity of the GenePOC Carba assay was not evaluated because bacterial suspensions for testing must be standardized to a density of $0 . 5 \mathrm { M c F }$ arland, which is substantially above the limit of detection (LoD) for assays based on nucleic acid amplification.

# Inclusivity

The analytical reactivity (inclusivity) of the GenePOC Carba assay was evaluated using a panel of 58 carbapenem non-susceptible isolates that included multiple variants of each targeted gene marker. The non-susceptibility of each isolate was confirmed by disc diffusion with doripenem, ertapenem, imipenem and meropenem according to standard methods (CLSI, M02-13 and M100-S28). Genotypic characterization of carbapenemase resistance genes was performed by Whole Genome Sequencing. Each isolate was tested with the GenePOC Carba assay in triplicate. Samples that produced indeterminate or unexpected (i.e., false positive or false negative) results were retested. All the final results obtained with each of the 58 strains were as expected. A listing of isolates and gene variants used in the GenePOC Carba Inclusivity Study is shown Tables $\underline { { \mathbf { 8 } } }$ and 9. The inclusivity of the GenePOC Carba assay for detection of carbapenemase resistance markers in Enterobacteriaceae, $A$ . baumannii and $P$ . aeruginosa was determined to be acceptable.

Table 8. Isolates used to evaluate the inclusivity of the GenePOC Carba assay, stratified by carbapenemase gene   

<table><tr><td colspan="1" rowspan="1">Species</td><td colspan="1" rowspan="1">Isolate</td><td colspan="1" rowspan="1">Origin</td><td colspan="1" rowspan="1">Year</td><td colspan="1" rowspan="1">CarbapenemaseGene Variant</td></tr><tr><td colspan="1" rowspan="1">Isolates Harboring Multiple Genes (2)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">CCRI-23061</td><td colspan="1" rowspan="1">Switzerland</td><td colspan="1" rowspan="1">2015</td><td colspan="1" rowspan="1">OXA-232, NDM-1</td></tr><tr><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">ATCC BAA-2793</td><td colspan="1" rowspan="1">Chile</td><td colspan="1" rowspan="1">2014</td><td colspan="1" rowspan="1">KPC-2, VIM-2</td></tr><tr><td colspan="1" rowspan="1">Isolates Harboring blakpc (11)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">canmaGellowb</td></tr><tr><td colspan="1" rowspan="1">Enterobacter cloacae</td><td colspan="1" rowspan="1">CCRI-21578 1</td><td colspan="1" rowspan="1">Canada</td><td colspan="1" rowspan="1">2011</td><td colspan="1" rowspan="1">KPC-4</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">ATCC BAA-2340</td><td colspan="1" rowspan="1">USA</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">KPC</td></tr><tr><td colspan="1" rowspan="1">Klebsiella oxytoca</td><td colspan="1" rowspan="1">CCRI-21581</td><td colspan="1" rowspan="1">Canada</td><td colspan="1" rowspan="1">2011</td><td colspan="1" rowspan="1">KPC-3</td></tr><tr><td colspan="1" rowspan="7">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">NCTC 13438</td><td colspan="1" rowspan="1">N/A1</td><td colspan="1" rowspan="1">N/A1</td><td colspan="1" rowspan="1">KPC-3</td></tr><tr><td colspan="1" rowspan="1">ATCC BAA-1705</td><td colspan="1" rowspan="1">USA</td><td colspan="1" rowspan="1">2007</td><td colspan="1" rowspan="1">KPC-2</td></tr><tr><td colspan="1" rowspan="1">CCRI-19587</td><td colspan="1" rowspan="1">Canada</td><td colspan="1" rowspan="1">2009</td><td colspan="1" rowspan="1">KPC-3</td></tr><tr><td colspan="1" rowspan="1">CCRI-19570</td><td colspan="1" rowspan="1">USA</td><td colspan="1" rowspan="1">2003</td><td colspan="1" rowspan="1">KPC-2</td></tr><tr><td colspan="1" rowspan="1">CCUG 59413</td><td colspan="1" rowspan="1">Sweden</td><td colspan="1" rowspan="1">2010</td><td colspan="1" rowspan="1">KPC-3</td></tr><tr><td colspan="1" rowspan="1">CCUG 59348</td><td colspan="1" rowspan="1">Norway</td><td colspan="1" rowspan="1">2010</td><td colspan="1" rowspan="1">KPC-2</td></tr><tr><td colspan="1" rowspan="1">CCUG 56233</td><td colspan="1" rowspan="1">Sweden</td><td colspan="1" rowspan="1">2088</td><td colspan="1" rowspan="1">KPC-2</td></tr><tr><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">CCRI-21587</td><td colspan="1" rowspan="1">Canada</td><td colspan="1" rowspan="1">2011</td><td colspan="1" rowspan="1">KPC-2</td></tr><tr><td colspan="1" rowspan="1">Isolates Harboring blandm (14)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">canmaGellowb</td></tr><tr><td colspan="1" rowspan="1">Enterobacter cloacae</td><td colspan="1" rowspan="1">ATCC BAA-2468</td><td colspan="1" rowspan="1">USA</td><td colspan="1" rowspan="1">N/A1</td><td colspan="1" rowspan="1">NDM-1</td></tr><tr><td colspan="1" rowspan="5">Escherichia coli</td><td colspan="1" rowspan="1">CCRI-22255</td><td colspan="1" rowspan="1">France</td><td colspan="1" rowspan="1">2013</td><td colspan="1" rowspan="1">NDM-1</td></tr><tr><td colspan="1" rowspan="1">CCRI-23064</td><td colspan="1" rowspan="1">Switzerland</td><td colspan="1" rowspan="1">2015</td><td colspan="1" rowspan="1">NDM-5</td></tr><tr><td colspan="1" rowspan="1">CCRI-23464</td><td colspan="1" rowspan="1">Canada</td><td colspan="1" rowspan="1">2016</td><td colspan="1" rowspan="1">NDM-5</td></tr><tr><td colspan="1" rowspan="1">CCRI-23065</td><td colspan="1" rowspan="1">Switzerland</td><td colspan="1" rowspan="1">2015</td><td colspan="1" rowspan="1">NDM-6</td></tr><tr><td colspan="1" rowspan="1">CCRI-23066</td><td colspan="1" rowspan="1">Switzerland</td><td colspan="1" rowspan="1">2015</td><td colspan="1" rowspan="1">NDM-7</td></tr><tr><td colspan="1" rowspan="6">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">NCTC 13443</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">NDM-1</td></tr><tr><td colspan="1" rowspan="1">ATCC BAA-2146</td><td colspan="1" rowspan="1">USA</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">NDM-1</td></tr><tr><td colspan="1" rowspan="1">CCRI-21711</td><td colspan="1" rowspan="1">Canada</td><td colspan="1" rowspan="1">2011</td><td colspan="1" rowspan="1">NDM-1</td></tr><tr><td colspan="1" rowspan="1">CCRI-22199</td><td colspan="1" rowspan="1">Canada</td><td colspan="1" rowspan="1">2012</td><td colspan="1" rowspan="1">NDM-1</td></tr><tr><td colspan="1" rowspan="1">CCRI-22254</td><td colspan="1" rowspan="1">France</td><td colspan="1" rowspan="1">2013</td><td colspan="1" rowspan="1">NDM-4</td></tr><tr><td colspan="1" rowspan="1">CCUG 60138</td><td colspan="1" rowspan="1">Sweden</td><td colspan="1" rowspan="1">2010</td><td colspan="1" rowspan="1">NDM-1</td></tr><tr><td colspan="1" rowspan="1">Providencia rettgeri</td><td colspan="1" rowspan="1">CCRI-22257</td><td colspan="1" rowspan="1">France</td><td colspan="1" rowspan="1">2013</td><td colspan="1" rowspan="1">NDM-1</td></tr><tr><td colspan="1" rowspan="1">Providencia stuartii</td><td colspan="1" rowspan="1">CCRI-22256</td><td colspan="1" rowspan="1">France</td><td colspan="1" rowspan="1">2013</td><td colspan="1" rowspan="1">NDM-1</td></tr><tr><td colspan="1" rowspan="1">Isolates Harboring blaoxA-48-like (11)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Citrobacter freundii</td><td colspan="1" rowspan="1">CCRI-23374</td><td colspan="1" rowspan="1">Canada</td><td colspan="1" rowspan="1">2016</td><td colspan="1" rowspan="1">OXA-204</td></tr><tr><td colspan="1" rowspan="1">Enterobacter cloacae</td><td colspan="1" rowspan="1">CCRI-22266</td><td colspan="1" rowspan="1">France</td><td colspan="1" rowspan="1">2013</td><td colspan="1" rowspan="1">OXA-48</td></tr><tr><td colspan="1" rowspan="2">Escherichia coli</td><td colspan="1" rowspan="1">CCRI-22265</td><td colspan="1" rowspan="1">France</td><td colspan="1" rowspan="1">2013</td><td colspan="1" rowspan="1">OXA-48</td></tr><tr><td colspan="1" rowspan="1">ATCC BAA-2523</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">OXA-48</td></tr><tr><td colspan="1" rowspan="4">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">NCTC 13442</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">OXA-48</td></tr><tr><td colspan="1" rowspan="1">CCRI-22263</td><td colspan="1" rowspan="1">France</td><td colspan="1" rowspan="1">2013</td><td colspan="1" rowspan="1">OXA-48</td></tr><tr><td colspan="1" rowspan="1">CCRI-22264</td><td colspan="1" rowspan="1">France</td><td colspan="1" rowspan="1">2013</td><td colspan="1" rowspan="1">OXA-181</td></tr><tr><td colspan="1" rowspan="1">CCRI-23060</td><td colspan="1" rowspan="1">Switzerland</td><td colspan="1" rowspan="1">2015</td><td colspan="1" rowspan="1">OXA-204</td></tr><tr><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="1">ATCC BAA-2524</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">OXA-48</td></tr><tr><td colspan="1" rowspan="1">CCUG 64452</td><td colspan="1" rowspan="1">Sweden</td><td colspan="1" rowspan="1">2013</td><td colspan="1" rowspan="1">OXA-48</td></tr><tr><td colspan="1" rowspan="1">Providencia rettgeri</td><td colspan="1" rowspan="1">CCRI-22267</td><td colspan="1" rowspan="1">France</td><td colspan="1" rowspan="1">2013</td><td colspan="1" rowspan="1">OXA-181</td></tr><tr><td colspan="1" rowspan="1">Isolates Harboring blaimp (11)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">cYanmaGllow</td></tr><tr><td colspan="1" rowspan="1">Acinetobacter baumannii</td><td colspan="1" rowspan="1">CCRI-19488</td><td colspan="1" rowspan="1">Canada</td><td colspan="1" rowspan="1">2003</td><td colspan="1" rowspan="1">IMP-1</td></tr><tr><td colspan="1" rowspan="1">Citrobacter youngae</td><td colspan="1" rowspan="1">CCRI-21591</td><td colspan="1" rowspan="1">Canada</td><td colspan="1" rowspan="1">2011</td><td colspan="1" rowspan="1">IMP-4</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">NCTC 13476</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">IMP-1</td></tr><tr><td colspan="1" rowspan="5">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">CCRI-19569</td><td colspan="1" rowspan="1">Japan</td><td colspan="1" rowspan="1">2003</td><td colspan="1" rowspan="1">IMP-1</td></tr><tr><td colspan="1" rowspan="1">CCRI-19582</td><td colspan="1" rowspan="1">Turkey</td><td colspan="1" rowspan="1">2009</td><td colspan="1" rowspan="1">IMP-1</td></tr><tr><td colspan="1" rowspan="1">CCRI-19583</td><td colspan="1" rowspan="1">Taiwan</td><td colspan="1" rowspan="1">2009</td><td colspan="1" rowspan="1">IMP-4</td></tr><tr><td colspan="1" rowspan="1">CCRI-19588</td><td colspan="1" rowspan="1">Taiwan</td><td colspan="1" rowspan="1">2009</td><td colspan="1" rowspan="1">IMP-4</td></tr><tr><td colspan="1" rowspan="1">CCRI-19584</td><td colspan="1" rowspan="1">Taiwan</td><td colspan="1" rowspan="1">2009</td><td colspan="1" rowspan="1">IMP-8</td></tr><tr><td colspan="1" rowspan="2">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">CCRI-21589</td><td colspan="1" rowspan="1">Canada</td><td colspan="1" rowspan="1">2011</td><td colspan="1" rowspan="1">IMP-1</td></tr><tr><td colspan="1" rowspan="1">CCRI-21590</td><td colspan="1" rowspan="1">China</td><td colspan="1" rowspan="1">2000</td><td colspan="1" rowspan="1">IMP-9</td></tr><tr><td colspan="1" rowspan="1">Serratia marcescens</td><td colspan="1" rowspan="1">CCRI-22262</td><td colspan="1" rowspan="1">France</td><td colspan="1" rowspan="1">2013</td><td colspan="1" rowspan="1">IMP-11</td></tr><tr><td colspan="1" rowspan="1">Isolates Harboring blavim (9)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">cYanmaGellowb</td></tr><tr><td colspan="1" rowspan="5">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">NCTC 13439</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">VIM-1</td></tr><tr><td colspan="1" rowspan="1">NCTC 13440</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">VIM-1</td></tr><tr><td colspan="1" rowspan="1">CCRI-19585</td><td colspan="1" rowspan="1">France</td><td colspan="1" rowspan="1">2009</td><td colspan="1" rowspan="1">VIM-1</td></tr><tr><td colspan="1" rowspan="1">CCRI-22258</td><td colspan="1" rowspan="1">France</td><td colspan="1" rowspan="1">2013</td><td colspan="1" rowspan="1">VIM-1</td></tr><tr><td colspan="1" rowspan="1">CCRI-22259</td><td colspan="1" rowspan="1">France</td><td colspan="1" rowspan="1">2013</td><td colspan="1" rowspan="1">VIM-19</td></tr><tr><td colspan="1" rowspan="3">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">NCTC 13437</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">VIM-10</td></tr><tr><td colspan="1" rowspan="1">CCRI-21588</td><td colspan="1" rowspan="1">Canada</td><td colspan="1" rowspan="1">2011</td><td colspan="1" rowspan="1">VIM-2</td></tr><tr><td colspan="1" rowspan="1">CCRI-22720</td><td colspan="1" rowspan="1">Argentina</td><td colspan="1" rowspan="1">2014</td><td colspan="1" rowspan="1">VIM-2</td></tr><tr><td colspan="1" rowspan="1">Serratia marcescens</td><td colspan="1" rowspan="1">CCRI-22261</td><td colspan="1" rowspan="1">France</td><td colspan="1" rowspan="1">2013</td><td colspan="1" rowspan="1">VIM-2</td></tr></table>

ATCC: American Type Culture Collection CCRI: Culture Collection of the Centre de Researche en Infectiologie NCTC: National Collection of Type Cultures CCUG: Culture Collection, University of Gothenburg

Table 9. Summary of results from the GenePOC Carba Inclusivity Study stratified by carbapenemase gene variant and bacterial species   

<table><tr><td rowspan=2 colspan=1>Species</td><td rowspan=1 colspan=5>blandm Variants Detected</td><td rowspan=2 colspan=2>Total</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>7</td></tr><tr><td rowspan=1 colspan=1>Enterobacter cloacae</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td></td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>Escherichia coli</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td></td><td rowspan=1 colspan=1>5</td></tr><tr><td rowspan=1 colspan=1>Klebsiella pneumoniae</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td></td><td rowspan=1 colspan=1>6</td></tr><tr><td rowspan=1 colspan=1>Providencia rettgeri</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td></td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>Providencia stuartii</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td></td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td></td><td rowspan=1 colspan=1>14</td></tr><tr><td rowspan=2 colspan=1>Species</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>lakpc Vari</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=2 colspan=1>Total</td><td></td><td rowspan=2 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Not Known</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>4</td><td></td></tr><tr><td rowspan=1 colspan=1>Enterobacter cloacae</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>1</td><td rowspan=6 colspan=2></td></tr><tr><td rowspan=1 colspan=1>Escherichia coli</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>Klebsiella oxytoca</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>Klebsiella pneumoniae</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>7</td></tr><tr><td rowspan=1 colspan=1>Pseudomonas aeruginosa</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>11</td></tr><tr><td rowspan=2 colspan=1>Species</td><td rowspan=1 colspan=1>blaoxA-4</td><td rowspan=1 colspan=1>VariantsI</td><td rowspan=1 colspan=1></td><td rowspan=2 colspan=1>Total</td><td rowspan=8 colspan=3></td></tr><tr><td rowspan=1 colspan=1>48</td><td rowspan=1 colspan=1>181</td><td rowspan=1 colspan=1>204</td></tr><tr><td rowspan=1 colspan=1>Citrobacter freundii</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>Enterobacter cloacae</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>Escherichia coli</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2</td></tr><tr><td rowspan=1 colspan=1>Klebsiella pneumoniae</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>6</td></tr><tr><td rowspan=1 colspan=1>Providencia rettgeri</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>11</td></tr><tr><td rowspan=2 colspan=1>Species</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=4>blap Variants Detected</td><td></td><td rowspan=2 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>11</td><td></td></tr><tr><td rowspan=1 colspan=1>Acinetobacter baumannii</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td></td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>Citrobacter youngae</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td></td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>Escherichia coli</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td></td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>Klebsiella pneumoniae</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td></td><td rowspan=1 colspan=1>5</td></tr><tr><td rowspan=1 colspan=1>Pseudomonas aeruginosa</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td></td><td rowspan=1 colspan=1>2</td></tr><tr><td rowspan=1 colspan=1>Serratia marcescens</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td></td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td></td><td rowspan=1 colspan=1>11</td></tr><tr><td rowspan=2 colspan=1>Species</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>blavim Variants Detected</td><td rowspan=1 colspan=1>blavim Variants Detected</td><td rowspan=1 colspan=1></td><td rowspan=2 colspan=1>Total</td><td></td><td rowspan=6 colspan=1></td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>19</td><td></td></tr><tr><td rowspan=1 colspan=1>Klebsiella pneumoniae</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>5</td><td></td></tr><tr><td rowspan=1 colspan=1>Pseudomonas aeruginosa</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>4</td><td></td></tr><tr><td rowspan=1 colspan=1>Serratia marcescens</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td></td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>9</td><td></td></tr><tr><td rowspan=3 colspan=1>Species 2</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>bla Variants Detected</td><td rowspan=1 colspan=1>bla Variants Detected</td><td rowspan=1 colspan=1></td><td rowspan=3 colspan=1>Total</td><td></td><td rowspan=6 colspan=1></td></tr><tr><td rowspan=1 colspan=1>blakPC</td><td rowspan=1 colspan=1>blavIM</td><td rowspan=1 colspan=1>blaoxA-48</td><td rowspan=1 colspan=1>blaNDM</td><td></td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>232</td><td rowspan=1 colspan=1>1</td><td></td></tr><tr><td rowspan=1 colspan=1>Klebsiella pneumoniae</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1</td><td></td></tr><tr><td rowspan=1 colspan=1>Pseudomonas aeruginosa</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2>0</td><td rowspan=1 colspan=1>1</td><td></td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2>1</td><td rowspan=1 colspan=1>2</td><td></td></tr></table>

1 On initial testing, 3/3 replicates were positive for blaKPC and 1/3 replicates also produced a false positive result for blaNDM; upon repeat testing, 3/3 replicates were positive for blaKPC and none was positive for any other carbapenemase gene 2 Isolates with multiple carbapenemase genes

# In Silico Analysis

In addition to laboratory testing, the inclusivity of the GenePOC Carba assay was demonstrated through in silico analysis of the targeted carbapenemase genes using the Basic Local Alignment Search Tool (BLAST). Predictions of the likelihood of detecting different gene variants were made based on the sequence data available in GenBank in November, 2018. A summary of the results is presented in Table 10. The analysis showed that the GenePOC Carba assay was likely to detect most of the known clinically relevant subtypes of the targeted carbapenemase genes. A Limitation in the device labeling indicates that analysis of new variant sequences deposited in GenBank after 2018 has not been performed.

Table 10. Summary of in silico analysis of the inclusivity of the GenePOC Carba assay   

<table><tr><td rowspan=2 colspan=1>Gene</td><td rowspan=2 colspan=1>VariantSequencesAvailable</td><td rowspan=1 colspan=4>Prediction based on In Silico Analysis 1</td></tr><tr><td rowspan=1 colspan=1>Detectable 2</td><td rowspan=1 colspan=1>LikelyDetectable 3</td><td rowspan=1 colspan=1>PotentiallyDetectable 4</td><td rowspan=1 colspan=1>Not Detectable 5</td></tr><tr><td rowspan=1 colspan=1>blaNDM</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>24(1-24)</td><td rowspan=1 colspan=1>--</td><td rowspan=1 colspan=1>--</td><td rowspan=1 colspan=1>--</td></tr><tr><td rowspan=1 colspan=1>blakpc</td><td rowspan=1 colspan=1>37</td><td rowspan=1 colspan=1>37(2-38)</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>--</td></tr><tr><td rowspan=1 colspan=1>blaoxA</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>12(48, 162, 181, 199,204, 244, 245, 2526,370, 484, 505, 566)</td><td rowspan=1 colspan=1>6(232, 514 6, 515 6,519, 546 6, 547 6)</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>12(54 6, 1637, 247 7,4057, 416 6, 436,438 7, 4397, 517,535 6, 538 6, 567)</td></tr><tr><td rowspan=1 colspan=1>blaMP</td><td rowspan=1 colspan=1>73</td><td rowspan=1 colspan=1>2(16, 74)</td><td rowspan=1 colspan=1>49(1, 2, 4-6, 8-10, 13-15, 17-20, 23-26,28-30, 32, 33, 37,38, 40, 42, 45, 47-49, 53-56, 59, 60,62, 66, 69-72, 75-79)</td><td rowspan=1 colspan=1>18(3, 7, 11, 21, 22, 27,34, 41, 43, 44, 51,52, 58, 61, 64, 67,68, 73)</td><td rowspan=1 colspan=1>4(12, 31, 35, 63) 8</td></tr><tr><td rowspan=1 colspan=1>blavIM</td><td rowspan=1 colspan=1>58</td><td rowspan=1 colspan=1>52(1-6, 8-12, 14-20,23-46, 48-50, 52-55, 57, 59, 60)</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>3(51, 56, 58)</td><td rowspan=1 colspan=1>3(7 9, 13, 47)</td></tr></table>

Based on the percentage of homology of the primers and probe(s) to the target sequence and the number and location of mismatches. The number of variant sequences in each category is shown, followed by the applicable variant types in parentheses. 2 $1 0 0 \%$ homology of each primer/detector probe with the target sequence 3 $9 5 \mathrm { - } 1 0 0 \%$ homology of each primer/detector probe with the target sequence 4 ${ < } 9 5 \%$ homology of one or more primers and/or the detector probe with the target sequence but ${ \le } 2$ nucleotide mismatches over their entire length 5 Failure to fulfill the parameters to be considered “Detectable,” “Likely Detectable” or “Potentially Detectable” 6 Associated with Shewanella, a rare human pathogen Associated with a mutation that results in the absence of carbapenemase activity (Dortet L. et al. Antimicrob. Agents Chemother 2015 59 (7): 3823-3828) 8 Reported in Pseudomonas aeruginosa (types 31, 35 and 63) and Pseudomonas putida (type 12) 9 Atypical blaVIM variant (Toleman M.A., et al. Antimicrob Agents Chemother 2004 48 (1): 329-332)

# 7. Assay Cut-Off:

Cut-off parameters for each of the five target genes and the Process Control were established by Receiver Operating Characteristic (ROC) curve analysis of endpoint fluorescence values obtained from testing known positive and negative samples, in conjunction with analysis of the associated cycle threshold $\mathrm { ( C t ) }$ values.

8. Accuracy (Instrument): Not applicable.

9. Carry-Over:

Studies were performed to evaluate the likelihood of obtaining false positive results with the GenePOC Carba assay due to carry-over (run-to-run) or cross- (sample-to-sample) contamination. The potential for carry-over contamination was evaluated by performing 10 alternating runs of all positive and all negative samples. Similarly, the potential for crosscontamination was evaluated by testing alternating positive and negative samples within a revogene instrument carousel over a total of 10 runs. In each case, a single revogene instrument was used for all 10 runs. The positive samples comprised an isolate of blaKPCpositive $K$ . pneumoniae at a starting concentration of ${ \sim } 4 \mathrm { M c F }$ arland (i.e., ${ \sim } 5 – 1 0$ times higher concentration than the prescribed workflow for the GenePOC Carba assay which uses a starting suspension density of 0.5 McFarland). The negative samples comprised a carbapenem non-susceptible isolate that did not possess any of the targeted carbapenemase genes. A summary of the results is presented in Table 11. Across the two studies, a total of three (3) false positive results was observed, none of which were for blaKPC. The false positive results for blaIMP (2) and blaVIM (1) were attributed to contamination in the laboratory environment.

The device labeling contains instructions regarding the preparation and testing of External Negative controls to monitor performance and reduce the potential for reporting of erroneous results due to laboratory contamination. Additional precautions for handling previously used PIEs and for decontaminating the revogene instrument are also provided. This is acceptable.

Table 11. Summary of results from the GenePOC Carba Contamination Study   

<table><tr><td rowspan=3 colspan=1>Gene Target</td><td rowspan=1 colspan=4>Positive Results/Total Tested</td></tr><tr><td rowspan=1 colspan=2>Carry-over</td><td rowspan=1 colspan=2>Cross</td></tr><tr><td rowspan=1 colspan=1>High Positive 1</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>High Positive 1</td><td rowspan=1 colspan=1>Negative</td></tr><tr><td rowspan=1 colspan=1>blaNDM</td><td rowspan=1 colspan=1>0/40</td><td rowspan=1 colspan=1>0/40</td><td rowspan=1 colspan=1>0/40</td><td rowspan=1 colspan=1>0/40</td></tr><tr><td rowspan=1 colspan=1>blakpc</td><td rowspan=1 colspan=1>40/40</td><td rowspan=1 colspan=1>0/40</td><td rowspan=1 colspan=1>40/40</td><td rowspan=1 colspan=1>0/40</td></tr><tr><td rowspan=1 colspan=1>blaoxA-48-like</td><td rowspan=1 colspan=1>0/40</td><td rowspan=1 colspan=1>0/40</td><td rowspan=1 colspan=1>0/40</td><td rowspan=1 colspan=1>0/40</td></tr><tr><td rowspan=1 colspan=1>blaMP</td><td rowspan=1 colspan=1>1/40</td><td rowspan=1 colspan=1>0/40</td><td rowspan=1 colspan=1>0/40</td><td rowspan=1 colspan=1>1/40</td></tr><tr><td rowspan=1 colspan=1>blavIM</td><td rowspan=1 colspan=1>1/40</td><td rowspan=1 colspan=1>0/40</td><td rowspan=1 colspan=1>0/40</td><td rowspan=1 colspan=1>0/40</td></tr></table>

blaKPC-positive containing ${ \geq } 1 0 ^ { 7 }$ CFU/mL in the Sample Buffer Tube

# B Comparison Studies:

# 1. Method Comparison with Predicate Device:

The performance of the GenePOC Carba assay was evaluated in a study that was conducted at three sites (two in the U.S. and one ex-U.S.) with a mixture of prospectively collected and stock isolates of carbapenemase producing organisms. Prior to inclusion in the study, each isolate was confirmed by MALDI-TOF analysis to be either Acinetobacter baumannii,

Pseudomonas aeruginosa or a species that is a member of the Enterobacteriaceae, and to be non-susceptible to one or more carbapenems as determined by disc diffusion according to standard procedures (CLSI, M02-13 and M100-S28). The results obtained with the GenePOC Carba assay were compared to those produced by another FDA-cleared device for the detection of the targeted carbapenemase resistance markers from isolated colonies, used according to the manufacturer’s instructions. Testing with the GenePOC assay was performed with organisms grown on both blood and MacConkey agar. The comparator method was performed independently using organisms grown on blood agar.

Note: The performance of the FDA-cleared device used as the comparator for the GenePOC Carba assay was previously evaluated in comparison to PCR/bidirectional sequencing and exhibited $100 \%$ sensitivity and close to $100 \%$ specificity for detection of the targeted carbapenemase resistance markers from isolated colonies of carbapenem non-susceptible bacteria. Because the isolates included in the study were identified to the species level and their carbapenem non-susceptible status was determined by reference methods, use of this device as a reference comparator method to evaluate the sensitivity and specificity of the GenePOC Carba assay was determined to be acceptable.

Five hundred and thirty-two (532) bacterial isolates (475 stock and 57 prospective) were initially enrolled in the study. Of these, 16 were excluded from the analysis of performance for the following reasons: eleven (11) did not meet the inclusion criteria for species identification (either they were not among the targeted organisms or organism identification was inconclusive), three (3) were unavailable for analysis due to laboratory error and two (2) were found to be susceptible to all four carbapenems tested. Four (4) additional isolates were also excluded because they were associated with a Negative External Control failure on initial testing and produced Indeterminate results upon repeat. The performance of the GenePOC Carba assay was therefore evaluated using a total of 512 carbapenem nonsusceptible isolates of Enterobacteriaceae (306), Acinetobacter baumannii (99) and Pseudomonas aeruginosa (107) (Tables 12 and 13). Performance stratified by each organism group is shown in Tables 14 and 15 for colonies grown on blood and MacConkey agar, respectively.

Table 12. Performance of the GenePOC Carba assay for the detection of carbapenemase resistance markers in carbapenem non-susceptible isolates of Enterobacteriaceae, Acinetobacter baumannii and Pseudomonas aeruginosa grown on blood agar   

<table><tr><td rowspan=1 colspan=5>GenePOC Carba Performance with Colonies Grown on Blood Agar</td></tr><tr><td rowspan=2 colspan=2>blaNDM</td><td rowspan=1 colspan=3>Reference Comparator Method</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=3 colspan=1>GenePOCCarba</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>186</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>188</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>2{2$</td><td rowspan=1 colspan=1>322</td><td rowspan=1 colspan=1>324</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>188</td><td rowspan=1 colspan=1>324</td><td rowspan=1 colspan=1>512</td></tr><tr><td rowspan=1 colspan=2>Sensitivity</td><td rowspan=1 colspan=3>98.9% (186/188); 96.2-99.7%</td></tr><tr><td rowspan=1 colspan=2>Specificity</td><td rowspan=1 colspan=3>99.4% (322/324); 97.8-99.8%</td></tr><tr><td rowspan=2 colspan=2>blaκPC</td><td rowspan=1 colspan=3>Reference Comparator Method</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=3 colspan=1>GenePOCCarba</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>113</td><td rowspan=1 colspan=1>3 3</td><td rowspan=1 colspan=1>117</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>395</td><td rowspan=1 colspan=1>396</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>114</td><td rowspan=1 colspan=1>398</td><td rowspan=1 colspan=1>512</td></tr><tr><td rowspan=1 colspan=2>Sensitivity</td><td rowspan=1 colspan=3>99.1% (113/114); 95.2-99.8%</td></tr><tr><td rowspan=1 colspan=2>Specificity</td><td rowspan=1 colspan=3>99.2% (395/398); 97.8-99.7%</td></tr><tr><td rowspan=2 colspan=2>blaoxA-48</td><td rowspan=1 colspan=3>Reference Comparator Method</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=3 colspan=1>GenePOCCarba</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>65</td><td rowspan=1 colspan=1>$3\fr5}$</td><td rowspan=1 colspan=1>68</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>444</td><td rowspan=1 colspan=1>444</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>65</td><td rowspan=1 colspan=1>447</td><td rowspan=1 colspan=1>512</td></tr><tr><td rowspan=1 colspan=2>Sensitivity</td><td rowspan=1 colspan=3>100% (65/65); 94.4-100%</td></tr><tr><td rowspan=1 colspan=2>Specificity</td><td rowspan=1 colspan=3>99.3% (444/447); 98.0-99.8%</td></tr><tr><td rowspan=2 colspan=2>blaMP</td><td rowspan=1 colspan=3>Reference Comparator Method</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=3 colspan=1>GenePOCCarba</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>27</td><td rowspan=1 colspan=1>19 6</td><td rowspan=1 colspan=1>46</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>466</td><td rowspan=1 colspan=1>466</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>27</td><td rowspan=1 colspan=1>485</td><td rowspan=1 colspan=1>512</td></tr><tr><td rowspan=1 colspan=2>Sensitivity</td><td rowspan=1 colspan=3>100% (27/27); 87.5-100%</td></tr><tr><td rowspan=1 colspan=2>Specificity</td><td rowspan=1 colspan=3>96.1% (466/485); 94.0-97.5%</td></tr><tr><td rowspan=2 colspan=2>blavIM</td><td rowspan=1 colspan=3>Reference Comparator Method</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=3 colspan=1>GenePOCCarba</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>52</td><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>53</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>459</td><td rowspan=1 colspan=1>459</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>52</td><td rowspan=1 colspan=1>460</td><td rowspan=1 colspan=1>512</td></tr><tr><td rowspan=1 colspan=2>Sensitivity</td><td rowspan=1 colspan=3>100% (52/52); 93.1-100%</td></tr><tr><td rowspan=1 colspan=2>Specificity</td><td rowspan=1 colspan=3>99.8% (459/460); 98.8-100%</td></tr></table>

Sensitivity and specificity are reported with two-sided $9 5 \%$ score confidence intervals Note: Isolates with discordant results between the GenePOC Carba assay and the reference method were investigated using alternative PCR assays for each of the 5 target carbapenemase genes, followed by bidirectional sequencing. The results of this discordant analysis are summarized in the footnotes below:

1 1/2 positive for blaNDM-1   
2 1/2 positive for blaNDM-1; 1/2 positive for blaOXA-48   
3 2/3 positive for blaKPC-3/KPC-38   
4 1/1 positive for blaKPC-3/KPC-38   
5 1/3 positive for blaOXA-48   
6 17/19 positive for blaIMP (11 blaIMP-13/IMP-37; 2 blaIMP-62; 1 each of blaIMP-4, blaIMP-15, blaIMP-16 and blaIMP-27/IMP-64)   
7 1/1 Negative for blaVIM   
1 1/2 positive for blaNDM-1   
2 1/2 positive for blaNDM-1; 1/2 positive for blaOXA-48   
3 2/3 positive for blaKPC-3/KPC-38   
4 1/2 positive for blaOXA-48   
5 17/21 positive for blaIMP (11 blaIMP-13/IMP-37, 2 blaIMP-62; 1 each of blaIMP-4, blaIMP-15, blaIMP-16 and blaIMP-27/IMP-64   
6 0/1 positive for blaVIM

Table 13. Performance of the GenePOC Carba assay for the detection of carbapenemase resistance markers in carbapenem non-susceptible isolates of Enterobacteriaceae, Acinetobacter baumannii and Pseudomonas aeruginosa grown on MacConkey agar   

<table><tr><td rowspan=1 colspan=5>GenePOC Carba Performance with Colonies Grown on MacConkey Agar</td></tr><tr><td rowspan=2 colspan=2>blaNDM</td><td rowspan=1 colspan=3>Reference Comparator Method</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=3 colspan=1>GenePOCCarba</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>186</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>188</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>22$</td><td rowspan=1 colspan=1>322</td><td rowspan=1 colspan=1>324</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>188</td><td rowspan=1 colspan=1>324</td><td rowspan=1 colspan=1>512</td></tr><tr><td rowspan=1 colspan=2>Sensitivity</td><td rowspan=1 colspan=3>98.9% (186/188); 96.2-99.7%</td></tr><tr><td rowspan=1 colspan=2>Specificity</td><td rowspan=1 colspan=3>99.4% (322/324); 97.8-99.8%</td></tr><tr><td rowspan=2 colspan=2>blakPC</td><td rowspan=1 colspan=3>Reference Comparator Method</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=3 colspan=1>GenePOCCarba</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>114</td><td rowspan=1 colspan=1>3 3</td><td rowspan=1 colspan=1>117</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>395</td><td rowspan=1 colspan=1>395</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>114</td><td rowspan=1 colspan=1>398</td><td rowspan=1 colspan=1>512</td></tr><tr><td rowspan=1 colspan=2>Sensitivity</td><td rowspan=1 colspan=3>100% (114/114); 96.7-100%</td></tr><tr><td rowspan=1 colspan=2>Specificity</td><td rowspan=1 colspan=3>99.2% (395/398); 97.8-99.7%</td></tr><tr><td rowspan=2 colspan=2>blaoxA-48</td><td rowspan=1 colspan=3>Reference Comparator Method</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=3 colspan=1>GenePOCCarba</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>65</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>67</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>445</td><td rowspan=1 colspan=1>445</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>65</td><td rowspan=1 colspan=1>447</td><td rowspan=1 colspan=1>512</td></tr><tr><td rowspan=1 colspan=2>Sensitivity</td><td rowspan=1 colspan=3>100% (65/65); 94.4-100%</td></tr><tr><td rowspan=1 colspan=2>Specificity</td><td rowspan=1 colspan=3>99.6% (445/447); 98.4-99.9%</td></tr><tr><td rowspan=2 colspan=2>blaMP</td><td rowspan=1 colspan=3>Reference Comparator Method</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=3 colspan=1>GenePOCCarba</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>27</td><td rowspan=1 colspan=1>215fr}$</td><td rowspan=1 colspan=1>48</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>464</td><td rowspan=1 colspan=1>464</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>27</td><td rowspan=1 colspan=1>485</td><td rowspan=1 colspan=1>512</td></tr><tr><td rowspan=1 colspan=2>Sensitivity</td><td rowspan=1 colspan=3>100% (27/27); 87.5-100%</td></tr><tr><td rowspan=1 colspan=2>Specificity</td><td rowspan=1 colspan=3>95.7% (464/485); 93.5-97.2%</td></tr><tr><td rowspan=2 colspan=2>blavIM</td><td rowspan=1 colspan=3>Reference Comparator Method</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=3 colspan=1>GenePOCCarba</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>52</td><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>53</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>459</td><td rowspan=1 colspan=1>459</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>52</td><td rowspan=1 colspan=1>460</td><td rowspan=1 colspan=1>512</td></tr><tr><td rowspan=1 colspan=2>Sensitivity</td><td rowspan=1 colspan=3>100% (52/52); 93.1-100%</td></tr><tr><td rowspan=1 colspan=2>Specificity</td><td rowspan=1 colspan=3>99.8% (459/460); 98.8-100%</td></tr></table>

Sensitivity and specificity are reported with two-sided $9 5 \%$ score confidence intervals Note: Isolates with discordant results between the GenePOC Carba assay and the reference method were investigated using alternative PCR assays for each of the 5 target carbapenemase genes, followed by bidirectional sequencing. The results of this discordant analysis are summarized in the footnotes below:

Table 14. Performance of the GenePOC Carba assay for the detection of carbapenemase resistance markers in carbapenem non-susceptible isolates grown on blood agar, stratified by organism group   

<table><tr><td rowspan=1 colspan=4>GenePOC Carba Performance with Colonies Grown on Blood Agar 1</td></tr><tr><td rowspan=1 colspan=1>Organism Group</td><td rowspan=1 colspan=1>bla Gene Target</td><td rowspan=1 colspan=1>Sensitivity</td><td rowspan=1 colspan=1>Specificity</td></tr><tr><td rowspan=5 colspan=1>Enterobacteriaceae</td><td rowspan=1 colspan=1>NDM</td><td rowspan=1 colspan=1>98.8% (85/86)93.7-99.8% 1</td><td rowspan=1 colspan=1>99.5% (219/220)97.5-99.9%</td></tr><tr><td rowspan=1 colspan=1>KPC</td><td rowspan=1 colspan=1>99.1% (112/113)95.2-99.8%</td><td rowspan=1 colspan=1>98.4% (190/193)95.5-99.5%</td></tr><tr><td rowspan=1 colspan=1>OXA-48</td><td rowspan=1 colspan=1>100% (64/64)94.3-100%</td><td rowspan=1 colspan=1>98.8% (239/242)96.4-99.6%</td></tr><tr><td rowspan=1 colspan=1>IMP</td><td rowspan=1 colspan=1>100% (14/14)78.5-100%</td><td rowspan=1 colspan=1>98.3% (287/292)96.1-99.3%</td></tr><tr><td rowspan=1 colspan=1>VIM</td><td rowspan=1 colspan=1>100% (12/12)75.8-100%</td><td rowspan=1 colspan=1>100% (294/294)98.7-100%</td></tr><tr><td rowspan=5 colspan=1>Acinetobacterbaumannii</td><td rowspan=1 colspan=1>NDM</td><td rowspan=1 colspan=1>98.7% (75/76)92.9-99.8%</td><td rowspan=1 colspan=1>100% (23/23)85.7-100%</td></tr><tr><td rowspan=1 colspan=1>KPC</td><td rowspan=1 colspan=1>100% (1/1)20.7-100%</td><td rowspan=1 colspan=1>100% (98/98)96.2-100%</td></tr><tr><td rowspan=1 colspan=1>OXA-48</td><td rowspan=1 colspan=1>Not Applicable</td><td rowspan=1 colspan=1>100% (99/99)96.3-100%</td></tr><tr><td rowspan=1 colspan=1>IMP</td><td rowspan=1 colspan=1>100% (8/8)67.6-100%</td><td rowspan=1 colspan=1>100% (91/91)96.0-100%</td></tr><tr><td rowspan=1 colspan=1>VIM</td><td rowspan=1 colspan=1>100% (1/1)20.7-100%</td><td rowspan=1 colspan=1>99.0% (97/98)94.4-99.8%</td></tr><tr><td rowspan=5 colspan=1>Pseudomonasaeruginosa</td><td rowspan=1 colspan=1>NDM</td><td rowspan=1 colspan=1>100% (26/26)87.1-100%</td><td rowspan=1 colspan=1>98.8% (80/81)93.3-99.8%</td></tr><tr><td rowspan=1 colspan=1>KPC</td><td rowspan=1 colspan=1>Not Applicable</td><td rowspan=1 colspan=1>100% (107/107)96.5-100%</td></tr><tr><td rowspan=1 colspan=1>OXA-48</td><td rowspan=1 colspan=1>100% (1/1)20.7-100%</td><td rowspan=1 colspan=1>100% (106/106)96.5-100%</td></tr><tr><td rowspan=1 colspan=1>IMP</td><td rowspan=1 colspan=1>100% (5/5)56.6-100%</td><td rowspan=1 colspan=1>86.3% (88/102)78.3-91.6%</td></tr><tr><td rowspan=1 colspan=1>VIM</td><td rowspan=1 colspan=1>100% (39/39)91.0-100%</td><td rowspan=1 colspan=1>100% (68/68)94.7-100%</td></tr></table>

Sensitivity and specificity are reported with two-sided $9 5 \%$ score confidence intervals

Table 15. Performance of the GenePOC Carba assay for the detection of carbapenemase resistance markers in carbapenem non-susceptible isolates grown on MacConkey agar, stratified by organism group   

<table><tr><td rowspan=1 colspan=4>GenePOC Carba Performance with Colonies Grown on MacConkey Agar 1</td></tr><tr><td rowspan=1 colspan=1>Organism Group</td><td rowspan=1 colspan=1>bla Gene Target</td><td rowspan=1 colspan=1>Sensitivity</td><td rowspan=1 colspan=1>Specificity</td></tr><tr><td rowspan=5 colspan=1>Enterobacteriaceae</td><td rowspan=1 colspan=1>NDM</td><td rowspan=1 colspan=1>98.8% (85/86)93.7-99.8%</td><td rowspan=1 colspan=1>99.5% (219/220)97.5-99.9%</td></tr><tr><td rowspan=1 colspan=1>KPC</td><td rowspan=1 colspan=1>100% (113/113)96.7-100%</td><td rowspan=1 colspan=1>98.4% (190/193)95.5-99.5%</td></tr><tr><td rowspan=1 colspan=1>OXA-48</td><td rowspan=1 colspan=1>100% (64/64)94.3-100%</td><td rowspan=1 colspan=1>99.2% (240/242)97.0-99.8%</td></tr><tr><td rowspan=1 colspan=1>IMP</td><td rowspan=1 colspan=1>100% (14/14)78.5-100%</td><td rowspan=1 colspan=1>98.3% (287/292)96.1-99.3%</td></tr><tr><td rowspan=1 colspan=1>VIM</td><td rowspan=1 colspan=1>100% (12/12)75.8-100%</td><td rowspan=1 colspan=1>99.7% (293/294)98.1-99.9%</td></tr><tr><td rowspan=5 colspan=1>Acinetobacterbaumannii</td><td rowspan=1 colspan=1>NDM</td><td rowspan=1 colspan=1>98.7% (75/76)92.9-99.8%</td><td rowspan=1 colspan=1>100% (23/23)85.7-100%</td></tr><tr><td rowspan=1 colspan=1>KPC</td><td rowspan=1 colspan=1>100% (1/1)20.7-100%</td><td rowspan=1 colspan=1>100% (98/98)96.2-100%</td></tr><tr><td rowspan=1 colspan=1>OXA-48</td><td rowspan=1 colspan=1>Not Applicable</td><td rowspan=1 colspan=1>100% (99/99)96.3-100%</td></tr><tr><td rowspan=1 colspan=1>IMP</td><td rowspan=1 colspan=1>100% (8/8)67.6-100%</td><td rowspan=1 colspan=1>97.8% (89/91)92.3-99.4%</td></tr><tr><td rowspan=1 colspan=1>VIM</td><td rowspan=1 colspan=1>100% (1/1)20.7-100%</td><td rowspan=1 colspan=1>100% (98/98)96.2-100%</td></tr><tr><td rowspan=5 colspan=1>Pseudomonasaeruginosa</td><td rowspan=1 colspan=1>NDM</td><td rowspan=1 colspan=1>100% (26/26)87.1-100%</td><td rowspan=1 colspan=1>98.8% (80/81)93.3-99.8%</td></tr><tr><td rowspan=1 colspan=1>KPC</td><td rowspan=1 colspan=1>Not Applicable</td><td rowspan=1 colspan=1>100% (107/107)96.5-100%</td></tr><tr><td rowspan=1 colspan=1>OXA-48</td><td rowspan=1 colspan=1>100% (1/1)20.7-100%</td><td rowspan=1 colspan=1>100% (106/106)96.5-100%</td></tr><tr><td rowspan=1 colspan=1>IMP</td><td rowspan=1 colspan=1>100% (5/5)56.6-100%</td><td rowspan=1 colspan=1>86.3% (88/102)78.3-91.6%</td></tr><tr><td rowspan=1 colspan=1>VIM</td><td rowspan=1 colspan=1>100% (39/39)91.0-100%</td><td rowspan=1 colspan=1>100% (68/68)94.7-100%</td></tr></table>

Sensitivity and specificity are reported with two-sided $9 5 \%$ score confidence intervals

There were 22 isolates on blood agar and 24 on MacConkey agar that produce positive results for two or more carbapenemase genes (Table 16). All 21 isolates that were positive for both blaNDM and blaOXA by the reference method produced the expected results with the GenePOC Carba assay from both types of culture media.

Table 16. Summary of isolates that produced positive results for multiple carbapenemase genes with the GenePOC Carba assay   

<table><tr><td rowspan=2 colspan=1>Culture Medium1</td><td rowspan=2 colspan=1>Number</td><td rowspan=1 colspan=2>Carbapenemase Gene(s) Reported</td></tr><tr><td rowspan=1 colspan=1>ReferenceComparator</td><td rowspan=1 colspan=1>GenePOC Carba</td></tr><tr><td rowspan=2 colspan=1>Blood Agar</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>blaNDm, blaoxA-48</td><td rowspan=1 colspan=1>blaNDM, blaoxA-48-like</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>blaNDM</td><td rowspan=1 colspan=1>blandm, blavim 2</td></tr><tr><td rowspan=3 colspan=1>MacConkey Agar</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>blanDm, blaoxA-48</td><td rowspan=1 colspan=1>blaNDm, blaoxA-48-like</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>blaNDM</td><td rowspan=1 colspan=1>blandm, blaimp 2</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>blakPc</td><td rowspan=1 colspan=1>blakpc, blavim</td></tr></table>

1 All 21 isolates that were positive for $b l a _ { \mathrm { N D M } }$ and $b l a _ { \mathrm { O X A } }$ by the reference comparator method were also positive by the GenePOC Carba assay from both culture media 2 One (1) isolate that was positive for $b l a _ { \mathrm { N D M } }$ only by the reference comparator method was positive for blaNDM/VIM from blood agar and $b l a _ { \mathrm { N D M } / \mathrm { V I M } }$ from MacConkey agar

The number and percentage of Unresolved and Indeterminate results after initial and repeat testing of colonies grown on blood and MacConkey agar is shown in Table 17.

Table 17. Summary of Unresolved and Indeterminate results observed with the GenePOC Carba assay   

<table><tr><td rowspan=3 colspan=1>Culture Medium</td><td rowspan=1 colspan=4>Number (%) [n = 516] 1</td></tr><tr><td rowspan=1 colspan=2>Unresolved</td><td rowspan=1 colspan=2>Indeterminate</td></tr><tr><td rowspan=1 colspan=1>Initial</td><td rowspan=1 colspan=1>Final</td><td rowspan=1 colspan=1>Initial</td><td rowspan=1 colspan=1>Final</td></tr><tr><td rowspan=1 colspan=1>Blood agar</td><td rowspan=1 colspan=1>1 (0.2) 2</td><td rowspan=1 colspan=1>0 (0)</td><td rowspan=1 colspan=1>8 (1.6)</td><td rowspan=1 colspan=1>4 (0.8)</td></tr><tr><td rowspan=1 colspan=1>MacConkey agar</td><td rowspan=1 colspan=1>1 (0.2) 3</td><td rowspan=1 colspan=1>0 (0)</td><td rowspan=1 colspan=1>6 (1.2)</td><td rowspan=1 colspan=1>4 (0.8)</td></tr></table>

1 Denominator includes 4 isolates that were associated with a Negative Control failure on initial testing and which produced Indeterminate results upon repeat 2 Sample reported Unresolved for blaNDM, blaKPC, blaOXA-48-like and $b l a _ { \mathrm { I M P } }$ but Positive for blaVIM 3 Sample reported Unresolved for all five gene targets

# 2. Matrix Comparison:

# Fresh vs Frozen Study

A study was performed to evaluate the effect on GenePOC Carba assay performance of freezing standardized suspensions of bacteria prior to testing. Representative carbapenemase non-susceptible isolates were grown under selective conditions and isolated colonies were used to prepare $0 . 5 \mathrm { M c F }$ arland suspensions that were tested “fresh” after storage at $2 { - } 8 ^ { \circ } \mathrm { C }$ or after freezing at ${ } _ { - 2 0 ^ { \circ } \mathrm { C } }$ for ${ \ge } 2 4$ hours. All results were as expected. The results from this study supported use of frozen panel members in the Reproducibility and Precision Study described in Section VII A1.

# C Clinical Studies:

1. Clinical Sensitivity:

Refer to Section VII B(1), above.

2. Clinical Specificity: Refer to Section VII B(1), above.

3. Other Clinical Supportive Data (When 1. and 2. Are Not Applicable): Not applicable.

# D Clinical Cut-Off:

Not applicable.

# E Expected Values/Reference Range:

The performance of the GenePOC Carba assay was evaluated using a collection of prospectively collected and archived isolates of carbapenem non-susceptible bacteria. Five hundred and twelve (512) isolates of Enterobacteriaceae, A. baumannii and $P _ { - }$ . aeruginosa were included in the study, of which 427 were found to be positive for one or more carbapenemase resistance genes using an FDA-cleared reference molecular assay. In comparison, the GenePOC Carba assay gave a positive result for one or more carbapenemase genes with 449 isolates. A summary of the carbapenemase resistance genes identified by the reference method and by the GenePOC Carba assay is shown in Table 18.

Table 18. Summary of carbapenemase resistance genes identified in the Clinical Study for the GenePOC Carba assay   

<table><tr><td rowspan=3 colspan=1>CarbapenemaseResistance Gene</td><td rowspan=1 colspan=3>Number of Carbapenem Non-susceptible Isolates (%) [n = 512]1</td></tr><tr><td rowspan=2 colspan=1>ReferenceComparator</td><td rowspan=1 colspan=2>GenePOC Carba</td></tr><tr><td rowspan=1 colspan=1>Blood Agar</td><td rowspan=1 colspan=1>MacConkey Agar</td></tr><tr><td rowspan=1 colspan=1>NDM</td><td rowspan=1 colspan=1>188 (36.7)</td><td rowspan=1 colspan=1>188 (37.5)</td><td rowspan=1 colspan=1>188 (37.5)</td></tr><tr><td rowspan=1 colspan=1>KPC</td><td rowspan=1 colspan=1>114 (22.3</td><td rowspan=1 colspan=1>116 (22.7)</td><td rowspan=1 colspan=1>117 (22.9)</td></tr><tr><td rowspan=1 colspan=1>OXA</td><td rowspan=1 colspan=1>65 (12.7)</td><td rowspan=1 colspan=1>68 (13.3)</td><td rowspan=1 colspan=1>67 (13.1)</td></tr><tr><td rowspan=1 colspan=1>IMP</td><td rowspan=1 colspan=1>27 (5.3)</td><td rowspan=1 colspan=1>46 (9.0)</td><td rowspan=1 colspan=1>48 (9.4)</td></tr><tr><td rowspan=1 colspan=1>VIM</td><td rowspan=1 colspan=1>52 (10.2)</td><td rowspan=1 colspan=1>53 (10.4)</td><td rowspan=1 colspan=1>53 (10.4)</td></tr><tr><td rowspan=1 colspan=1>Multiple</td><td rowspan=1 colspan=1>21 (4.1) 2</td><td rowspan=1 colspan=1>22 (4.3) 3</td><td rowspan=1 colspan=1>24 (4.7) 4</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>87 (17.0)</td><td rowspan=1 colspan=1>63 (12.3)</td><td rowspan=1 colspan=1>63 (12.3)</td></tr></table>

1 Percentages do not add up to 100 because some isolates were positive for two or more carbapenemase resistance genes 2 21/21 positive for blaNDM and blaOXA 3 21/22 positive for blaNDM and blaOXA; 1/22 positive for blaNDM and blaVIM 4 21/24 positive for blaNDM and blaOXA; 2/24 positive for blaNDM and blaIMP; 1/24 positive for blaKPC and blaVIM

# F Other Supportive Instrument Performance Characteristics Data:

Not applicable.

# VIII Proposed Labeling:

The labeling supports the finding of substantial equivalence for this device.

# IX Conclusion:

The submitted information in this premarket notification is complete and supports a substantial equivalence decision.