# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATION DECISION SUMMARY

I. Background Information:

A. 510(k) Number K190341   
B. Applicant iCubate, Inc.   
C. Proprietary and Established Names iC-GN iC-Cassette for use on the iC-System iC-GN Assay

D. Regulatory Information

<table><tr><td rowspan=1 colspan=1>ProductCode(s)</td><td rowspan=1 colspan=1>Classification</td><td rowspan=1 colspan=1>RegulationSection</td><td rowspan=1 colspan=1>Panel</td></tr><tr><td rowspan=1 colspan=1>PEN</td><td rowspan=1 colspan=1>Class II</td><td rowspan=1 colspan=1>21 CFR 866.3365 -Multiplex Nucleic AcidAssay For IdentificationOf Microorganisms AndResistance Markers FromPositive Blood Cultures</td><td rowspan=1 colspan=1>MI - Microbiology</td></tr></table>

II. Submission/Device Overview:

A. Purpose for Submission:

The purpose of the submission is to obtain clearance for the iC-GN Assay for use on the iCSystem.

B. Measurand: Bacterial Species: Acinetobacter baumannii complex, Enterobacter cloacae complex, Escherichia coli, Klebsiella oxytoca, Klebsiella pneumoniae, Pseudomonas aeruginosa, Proteus species, and Serratia marcescens

# Resistance Markers

KPC (blaKPC)- associated with resistance to carbapenems   
NDM (blaNDM)- associated with resistance to carbapenems   
CTX-M group 1(blaCTX-M group 1)- associated with resistance to extended spectrum beta  
lactams

# C. Type of Test:

The iC-GN Assay utilizes polymerase chain reaction (PCR) for the multiplex amplification of specific targets and detects the amplified targets with microarray hybridization. Targets are detected directly from patient positive blood cultures confirmed by Gram stain to contain Gram-negative bacilli. The iC-GN Assay utilizes proprietary ARM-PCR (Amplicon Rescued Multiplex PCR) technology allowing for multiple targets to be amplified in one reaction. Testing is done in a self-contained, automated, disposable cassette using the iCubate processor (iC-Processor). After the reaction is complete, the cassette is read on the iCubate reader (iC-Reader). Results from the iC-Reader are interpreted by iC-Report software and a final report is displayed on the iMac computer. Results are qualitative.

II. Intended Use/Indications for Use:

A. Intended Use(s): See Indications for Use below.

# B. Indication(s) for Use:

The iCubate, Inc. iC-GN Assay for use on the iC-System is a qualitative, multiplexed, in vitro diagnostic test for the detection and identification of potentially pathogenic Gramnegative bacteria which may cause bloodstream infection (BSI). The iC-GN Assay is performed directly on positive blood cultures, confirmed by Gram stain to contain Gramnegative bacilli. Cultures demonstrating mixed Gram stain results should not be tested on the assay. The iC-GN Assay is validated for use with select BACTEC, BacT/ALERT and VersaTREK blood culture bottles. The iC-GN Assay is indicated for use in conjunction with other clinical and laboratory findings, such as culture, to aid in the diagnosis of bacterial bloodstream infections; however, it is not used to monitor bloodstream infections.

The iC-GN Assay detects target DNA and identifies the following:

Bacterial Genera and Species   
Acinetobacter baumannii complex, Enterobacter cloacae complex, Escherichia coli, Klebsiella oxytoca, Klebsiella pneumoniae, Pseudomonas aeruginosa, Proteus species, Serratia marcescens

# Resistance Markers

KPC (blaKPC)- associated with resistance to carbapenems, NDM (blaNDM)- associated with resistance to carbapenems, CTX-M group 1(blaCTX-M group 1)- associated with resistance to extended spectrum beta-lactams

In mixed growth, the iC-GN Assay does not specifically attribute detection of KPC, NDM, or CTX-M group 1 to a specific genera or species.

Sub-culturing of positive blood cultures is necessary to recover organisms for susceptibility testing, identification of organisms not detected by the iC-GN Assay, differentiation of mixed growth, association of antimicrobial resistance marker genes to a specific organism, or for epidemiological typing.

C. Special Conditions for Use Statement(s): Rx - For Prescription Use Only

D. Special Instrument Requirements: Requires the iC-System

III. Device/System Characteristics:

# A. Device Description:

The iC-GN Assay utilizes polymerase chain reaction (PCR) for the multiplex amplification of specific targets and detects the amplified targets with microarray hybridization. Targets are detected directly from patient positive blood cultures confirmed by Gram stain to contain Gram-negative bacilli. The iC-GN Assay utilizes proprietary ARM-PCR (Amplicon Rescued Multiplex PCR) technology allowing for multiple targets to be amplified in one reaction. Testing is done in a self-contained, automated, disposable cassette using the iCubate processor (iC-Processor). After the reaction is complete, the cassette is read on the iCubate reader (iC-Reader). Results from the iC-Reader are interpreted by iC-Report software and a final report is displayed on the iMac computer.

To operate, the user opens the iC-Cassette cap and pipettes an aliquot of the diluted positive blood culture sample into the sample/PCR well in the bottom well plate of the cassette. Once inoculated, the cassette cap is closed, and all extraction, amplification and detection processes are completed in the cassette, a closed system. Extraction, amplification and detection sequences are defined by an assay script controlled by the iC–Processor.

The processing script is defined within a barcode label positioned on the top of each iC-Cassette which communicates with the iC-Processor. To access and pierce the foil-sealed reagent wells located in the bottom well plate of the cassette, the processor manipulates the cassette to move the cassette pipette horizontally and vertically. The script directs the transfer of reagents between the wells in the bottom well plate and finally to the array within the cassette. The iC-Processor is capable of processing four (4) iC-Cassettes with random access.

Once processing is complete, the cassette is manually transferred from the iC-Processor to the iC-Reader where the microarray within the cassette is read. The iC-Reader is capable of reading up to four (4) iC-Cassettes at one time. The results are interpreted via the iC-Report software and displayed for the user on the iMac. Raw data and result interpretations are stored within the iMac; raw data is accessible to iCubate service personnel only and not to the end user.

When finished with a loaded iC-GN Cassette, it should be disposed as biohazardous waste.

# B. Principle of Operation:

Amplicon Rescued Multiplex PCR (ARM-PCR)

# C. Instrument Description Information:

<table><tr><td rowspan=1 colspan=1>Modes of Operation</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>No</td></tr><tr><td rowspan=1 colspan=1>Does the applicant&#x27;s device contain the ability to transmit data to a computer,webserver, or mobile device?</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>□</td></tr><tr><td rowspan=1 colspan=1>Does the applicant&#x27;s device transmit data to a computer, webserver, or mobile deviceusing wireless transmission?</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>X</td></tr><tr><td rowspan=1 colspan=3>Software</td></tr><tr><td rowspan=1 colspan=1>FDA has reviewed applicant&#x27;s Hazard Analysis and software development processesfor this line of product types.</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1></td></tr></table>

1. Instrument Name: iC-System which was cleared under K163390.

IV. Substantial Equivalence Information:

A. Predicate Device Name(s): Verigene Gram-Negative Blood Culture Nucleic Acid Test (bc-gn)   
B. Predicate 510(k) Number(s): K132843

C. Comparison with Predicate(s):

<table><tr><td rowspan=1 colspan=1>Device &amp; PredicateDevice(s):</td><td rowspan=1 colspan=1>K190341</td><td rowspan=1 colspan=1>K132843</td></tr><tr><td rowspan=1 colspan=1>Device Trade Name</td><td rowspan=1 colspan=1>iC-GN Assay for use on the iC-System</td><td rowspan=1 colspan=1>Verigene Gram-NegativeBlood Culture Nucleic AcidTest (CG-GN)</td></tr><tr><td rowspan=1 colspan=1>General DeviceCharacteristicSimilarities</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Intended Use/Indications for Use</td><td rowspan=1 colspan=1>The iCubate, Inc. iC-GN Assay for use onthe iC-System is a qualitative, multiplexed,in vitro diagnostic test for the detection andidentification of potentially pathogenicGram-negative bacteria, which may causebloodstream infection (BSI). The iC-GNAssay is performed directly on positiveblood cultures, confirmed by Gram stain tocontain Gram-negative bacilli. Culturesdemonstrating mixed Gram stain resultsshould not be tested on the assay. The iC-GNAssay is validated for use with select</td><td rowspan=1 colspan=1>The Verigene Gram-Negative BloodCulture Nucleic Acid Test (BC-GN),performed using the sample-to-results Verigene System, is aqualitative multiplexed in vitrodiagnostic test for the simultaneousdetection and identification ofselected gram-negative bacteria andresistance markers. BC-GN isperformed directly on blood culturemedia using blood culture bottlesidentified as positive by a continuous</td></tr></table>

BACTEC, BacT/ALERT and VersaTREK blood culture bottles. The iC-GN Assay is indicated for use in conjunction with other clinical and laboratory findings, such as culture, to aid in the diagnosis of bacterial monitor blood culture system and which contain gram-negative bacteria as determined by Gram stain. BC-GN detects and identifies the following:

<table><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">bloodstream infections; however, it is notused to monitor bloodstream infections.The iC-GN Assay detects target DNA andidentifies the following:Bacterial Genera and SpeciesAcinetobacter baumannii complex,Enterobacter cloacae complex,Escherichia coli, Klebsiella oxytoca,Klebsiella pneumoniae, Pseudomonasaeruginosa, Proteus species, SerratiamarcescensResistance MarkersKPC (blaKPC)- associated withresistance to carbapenems, NDM(blaNDM)- associated with resistanceto carbapenems, CTX-M group1(blaCTX-M group 1)- associated withresistance to extended spectrum beta-lactamsIn mixed growth, the iC-GN Assay does notspecifically attribute KPC, NDM, or CTX-Mto a specific genera or species.iC-GN is indicated for use in conjunctionwith other clinical and laboratory findings toaid in the diagnosis of bacterial bloodstreaminfections; however, is not to be used tomonitor these infections. Sub-culturing ofpositive blood cultures is necessary torecover organisms for susceptibility testing,identification of organisms not detected byiC-GN, differentiation of mixed growth,association of antimicrobial resistancemarker genes to a specific organism, or forepidemiological typing.</td><td colspan="1" rowspan="1">Acinetobacter spp.Citrobacter spp.Enterobacter spp.Proteus spp.Escherichia coliKlebsiella pneumoniaeKlebsiella oxytocaPseudomonas aeruginosaBC-GN is indicated for use inconjunction with other clinical andlaboratory findings to aid in thediagnosis of bacterial bloodstreaminfections; however, it is not to beused to monitor these infections.Sub-culturing of positive bloodcultures is necessary to recoverorganisms for susceptibility testing,identification of organisms notdetected by BC-GN, differentiationof mixed growth, association ofantimicrobial resistance markergenes to a specific organism, of forepidemiological typing.</td></tr><tr><td colspan="1" rowspan="1">Sample Type</td><td colspan="1" rowspan="1">Positive Blood Culture</td><td colspan="1" rowspan="1">Positive Blood Culture</td></tr><tr><td colspan="1" rowspan="1">General DeviceCharacteristicDifferences</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">InstrumentRequirements</td><td colspan="1" rowspan="1">iC-System</td><td colspan="1" rowspan="1">Verigene System</td></tr><tr><td colspan="1" rowspan="1">Test Principle</td><td colspan="1" rowspan="1">ARM-PCR</td><td colspan="1" rowspan="1">Gold nanoparticle probe-based PCR</td></tr><tr><td colspan="1" rowspan="1">Compatible BloodCulture Bottles</td><td colspan="1" rowspan="1">BD BACTEC Standard/10 Aerobic/FBD BACTEC Standard/10 Anaerobic/FBD BACTEC Plus Aerobic/FBD BACTEC Plus Anaerobic/F</td><td colspan="1" rowspan="1">BACTECTM Plus Aerobic/FBacT/ALERT FA FAN</td></tr><tr><td></td><td>BD BACTEC Lytic/10 Anaerobic/F BacT/Alert SA Standard Aerobic BacT/Alert SN Standard Anaerobic BacT/Alert FA Aerobic FAN BacT/Alert FN Anaerobic FAN BacT/Alert FA Plus Aerobic BacT/Alert FN Plus Anaerobic VersaTREK REDOX 1 VersaTREK REDOX 2</td><td></td></tr><tr><td>Throughput</td><td>Four (4) samples/iC-Processor</td><td>One (1) Sample/Processor</td></tr></table>

# V. Standards/Guidance Documents Referenced:

1. Guidance for the Content of Premarket Submissions for Software Contained in Medical Devices   
2. Content of Premarket Submissions for Management of Cybersecurity in Medical Devices   
3. Validation of Analytical Procedures: Test and Methodology Q2(R1), ICH Harmonised Tripartite Guideline, Oct. 1994   
4. Q2B Validation of Analytical Procedures: Methodology, Guidance for Industry, Nov. 1996

VI. Performance Characteristics (if/when applicable):

A. Analytical Performance:

1. Limit of Detection (LoD):

A study was performed to determine the limit of detection for each iC-GN Assay target, defined as the lowest concentration $\mathrm { ( C F U / m L ) }$ ) of analyte that can be detected approximately $9 5 \%$ of the time. For the eleven targets detected by the iC-GN Assay, a panel of twenty-seven representative strains were evaluated, a minimum of three per target. For complex and genus level targets, at least two representative species were evaluated. LoD testing was conducted in two phases: the first phase was to narrow the range for LoD analysis. In phase II, the approximated $9 5 \%$ performance point determined in phase I was confirmed by testing a minimum of twenty replicates on each of three unique cassette lots. Plating and subsequent colony counts were used to determine organism concentrations. The final limit of detection for each target, is provided below.

<table><tr><td rowspan=1 colspan=4>Table 1: iC-GN Assay LoD Results</td></tr><tr><td rowspan=1 colspan=1>Target</td><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>Concentration(CFU/mL)</td><td rowspan=1 colspan=1>Defined Target LoD(CFU/mL)</td></tr><tr><td rowspan=3 colspan=1>A. baumannii complex</td><td rowspan=1 colspan=1>307-0294</td><td rowspan=1 colspan=1>5.3 × 105</td><td rowspan=3 colspan=1>5.3 × 105  5.2 × 106</td></tr><tr><td rowspan=1 colspan=1>CDC-83</td><td rowspan=1 colspan=1>5.2× 106</td></tr><tr><td rowspan=1 colspan=1>ATCC 23055</td><td rowspan=1 colspan=1>9.0 × 105</td></tr><tr><td rowspan=3 colspan=1>E. cloacae complex</td><td rowspan=1 colspan=1>Z101</td><td rowspan=1 colspan=1>5.0 × 106</td><td rowspan=3 colspan=1>4.9 × 105  5.5 × 106</td></tr><tr><td rowspan=1 colspan=1>CDC-164</td><td rowspan=1 colspan=1>5.5× 106</td></tr><tr><td rowspan=1 colspan=1>ATCC 700323</td><td rowspan=1 colspan=1>4.9 × 105</td></tr><tr><td rowspan=3 colspan=1>E. coli</td><td rowspan=1 colspan=1>ATCC 43895</td><td rowspan=1 colspan=1>7.7× 10$</td><td rowspan=3 colspan=1>7.7 × 105 - 8.4 × 105</td></tr><tr><td rowspan=1 colspan=1>ATCC BAA-2326</td><td rowspan=1 colspan=1>7.9 × 10 5}$</td><td rowspan=1 colspan=1>[7.7 x</td></tr><tr><td rowspan=1 colspan=1>CDC-55</td><td rowspan=1 colspan=1>8.4× 105</td></tr><tr><td rowspan=3 colspan=1>K. oxytoca</td><td rowspan=1 colspan=1>Z115</td><td rowspan=1 colspan=1>6.2 × 105</td><td rowspan=3 colspan=1>5.4 × 105 - 1.1 × 106</td></tr><tr><td rowspan=1 colspan=1>ATCC 13182</td><td rowspan=1 colspan=1>5.4 × 105</td></tr><tr><td rowspan=1 colspan=1>CDC-147</td><td rowspan=1 colspan=1>1.1 × 106</td></tr><tr><td rowspan=5 colspan=1>K. pneumoniae</td><td rowspan=1 colspan=1>ATCC 35657</td><td rowspan=1 colspan=1>1.9 × 106</td><td rowspan=5 colspan=1>6.0 × 105  4.2 × 106</td></tr><tr><td rowspan=1 colspan=1>CDC-40</td><td rowspan=1 colspan=1>3.6 × 106</td></tr><tr><td rowspan=1 colspan=1>CDC-42</td><td rowspan=1 colspan=1>1.9 × 106</td></tr><tr><td rowspan=1 colspan=1>KPC-2</td><td rowspan=1 colspan=1>4.2 × 106</td></tr><tr><td rowspan=1 colspan=1>LACNY 11</td><td rowspan=1 colspan=1>6.0 × 105</td></tr><tr><td rowspan=4 colspan=1>Proteus species</td><td rowspan=1 colspan=1>Z050</td><td rowspan=1 colspan=1>1.1 × 106</td><td rowspan=4 colspan=1>6.9 × 105 - 1.1 × 106</td></tr><tr><td rowspan=1 colspan=1>CDC-59</td><td rowspan=1 colspan=1>9.9× 105</td></tr><tr><td rowspan=1 colspan=1>Z028</td><td rowspan=1 colspan=1>7.6 × 105}$</td></tr><tr><td rowspan=1 colspan=1>Z129</td><td rowspan=1 colspan=1>6.9 × 105$</td></tr><tr><td rowspan=3 colspan=1>P. aeruginosa</td><td rowspan=1 colspan=1>Z139</td><td rowspan=1 colspan=1>1.2 × 106</td><td rowspan=3 colspan=1>5.0 × 105 - 1.2 × 106</td></tr><tr><td rowspan=1 colspan=1>CDC-231</td><td rowspan=1 colspan=1>5.0 × 10}$</td></tr><tr><td rowspan=1 colspan=1>CDC-250</td><td rowspan=1 colspan=1>6.9 × 105</td></tr><tr><td rowspan=3 colspan=1>S. marcescens</td><td rowspan=1 colspan=1>ATCC 43297</td><td rowspan=1 colspan=1>7.2 × 10 5}$</td><td rowspan=3 colspan=1>6.4 × 105 - 8.1 × 105</td></tr><tr><td rowspan=1 colspan=1>ATCC 21212</td><td rowspan=1 colspan=1>8.1 × 105</td></tr><tr><td rowspan=1 colspan=1>CDC-91</td><td rowspan=1 colspan=1>6.4 × 105</td></tr><tr><td rowspan=3 colspan=1>CTX-M group 1</td><td rowspan=1 colspan=1>ATCC BAA-2326 (CTX-M-15)</td><td rowspan=1 colspan=1>7.9 × 105</td><td rowspan=3 colspan=1>7.9 × 105 - 2.3 × 106</td></tr><tr><td rowspan=1 colspan=1>CDC-40 (CTX-M-15)</td><td rowspan=1 colspan=1>2.3 × 106</td></tr><tr><td rowspan=1 colspan=1>CDC-42 (CTX-M-15)</td><td rowspan=1 colspan=1>1.9 × 106</td></tr><tr><td rowspan=3 colspan=1>KPC</td><td rowspan=1 colspan=1>CDC-147 (KPC-3)</td><td rowspan=1 colspan=1>2.3 × 106</td><td rowspan=3 colspan=1>1.5 × 105 - 4.2 × 106</td></tr><tr><td rowspan=1 colspan=1>KPC-2</td><td rowspan=1 colspan=1>4.2 × 106</td></tr><tr><td rowspan=1 colspan=1>CDC-231 (KPC-5)</td><td rowspan=1 colspan=1>1.5 × 10$</td></tr><tr><td rowspan=3 colspan=1>NDM</td><td rowspan=1 colspan=1>CDC-83 (NDM-1)</td><td rowspan=1 colspan=1>5.2 × 106</td><td rowspan=3 colspan=1>3.3 × 105  5.2 × 106</td></tr><tr><td rowspan=1 colspan=1>CDC-55 (NDM-1)</td><td rowspan=1 colspan=1>4.0 × 106</td><td rowspan=1 colspan=1>3.3 ×</td></tr><tr><td rowspan=1 colspan=1>CDC-250 (NDM-1)</td><td rowspan=1 colspan=1>3.3 × 10 }$</td></tr></table>

2. Bottle Ring:

A study was performed to establish the levels of each iC-GN target organism at two clinically relevant concentrations: initial bottle positivity (bottle “ring”) and eight hours beyond initial positivity. Twenty-seven representative organisms were evaluated, a minimum of three per iC-GN target. Organisms were grown in BD BACTEC Plus Aerobic blood culture bottles with human blood added and placed on the BD BACTEC System. Within two hours of initial bottle positivity, the bottles were removed for plating and subsequent colony counts to determine organism concentrations. The bottles were then returned to the incubator and approximately eight hours after initial bottle positivity, the bottles were again removed for plating and subsequent colony counts to determine organism concentrations. Three bottles were grown for each strain. The average concentrations at initial bottle positivity and eight hours beyond initial bottle positivity are provided in below. The concentrations at initial bottle positivity, representative of the lowest levels that may be observed in a clinical setting, are above the limits of detection determined for each strain. The results of the study are as expected and establish that the established LoD is such that the assay has is able to detect clinically relevant levels of bacteria.

<table><tr><td colspan="4" rowspan="1">Table 2: iC-GN Target Organism Concentrations at Bottle "Ring"</td></tr><tr><td colspan="1" rowspan="1">Organism</td><td colspan="1" rowspan="1">Strain ID</td><td colspan="1" rowspan="1">Initial Bottle PositivityAverage Concentration(CFU/mL)</td><td colspan="1" rowspan="1">Bottle Positivity + 8Average Concentration(CFU/mL)</td></tr><tr><td colspan="1" rowspan="1">Acinetobacter baumannii</td><td colspan="1" rowspan="1">307-0294</td><td colspan="1" rowspan="1">4.24 × 108</td><td colspan="1" rowspan="1">8.27× 108</td></tr><tr><td colspan="1" rowspan="1">Acinetobacter baumannii</td><td colspan="1" rowspan="1">CDC-83</td><td colspan="1" rowspan="1">3.39 × 108</td><td colspan="1" rowspan="1">7.23 × 108</td></tr><tr><td colspan="1" rowspan="1">Acinetobactercalcoaceticus</td><td colspan="1" rowspan="1">ATCC 23055</td><td colspan="1" rowspan="1">6.78 × 107</td><td colspan="1" rowspan="1">2.93 × 108</td></tr><tr><td colspan="1" rowspan="1">Enterobacter cloacae</td><td colspan="1" rowspan="1">Z101</td><td colspan="1" rowspan="1">2.17 × 108</td><td colspan="1" rowspan="1">1.97 × 109</td></tr><tr><td colspan="1" rowspan="1">Enterobacter cloacae</td><td colspan="1" rowspan="1">CDC-164</td><td colspan="1" rowspan="1">5.62 × 108</td><td colspan="1" rowspan="1">2.31 × 109</td></tr><tr><td colspan="1" rowspan="1">Enterobacter hormaechei</td><td colspan="1" rowspan="1">ATCC 700323</td><td colspan="1" rowspan="1">4.36 × 108</td><td colspan="1" rowspan="1">2.75 × 109</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">ATCC 43895</td><td colspan="1" rowspan="1">1.50 × 108</td><td colspan="1" rowspan="1">9.48 × 108</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">ATCC BAA-2326</td><td colspan="1" rowspan="1">6.23 × 108</td><td colspan="1" rowspan="1">1.52 × 109</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">CDC-55</td><td colspan="1" rowspan="1">4.93 × 108</td><td colspan="1" rowspan="1">1.51 × 109</td></tr><tr><td colspan="1" rowspan="1">Klebsiella oxytoca</td><td colspan="1" rowspan="1">Z115</td><td colspan="1" rowspan="1">5.32 × 108</td><td colspan="1" rowspan="1">2.07 × 109</td></tr><tr><td colspan="1" rowspan="1">Klebsiella oxytoca</td><td colspan="1" rowspan="1">ATCC 13182</td><td colspan="1" rowspan="1">4.16 × 108</td><td colspan="1" rowspan="1">4.52 × 109</td></tr><tr><td colspan="1" rowspan="1">Klebsiella oxytoca</td><td colspan="1" rowspan="1">CDC-147</td><td colspan="1" rowspan="1">9.67 × 108</td><td colspan="1" rowspan="1">1.31 × 109</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">ATCC 35657</td><td colspan="1" rowspan="1">9.78 × 108</td><td colspan="1" rowspan="1">1.08 × 109</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">CDC-40</td><td colspan="1" rowspan="1">2.16 × 108</td><td colspan="1" rowspan="1">1.36 × 109</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">CDC-42</td><td colspan="1" rowspan="1">2.55 × 108</td><td colspan="1" rowspan="1">1.10 × 109</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">KPC-2</td><td colspan="1" rowspan="1">7.70 × 108</td><td colspan="1" rowspan="1">1.66 × 109</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">LACNY 11</td><td colspan="1" rowspan="1">5.43 × 107</td><td colspan="1" rowspan="1">1.67 × 109</td></tr><tr><td colspan="1" rowspan="1">Proteus mirabilis</td><td colspan="1" rowspan="1">Z050</td><td colspan="1" rowspan="1">1.71 × 108</td><td colspan="1" rowspan="1">7.40 × 108</td></tr><tr><td colspan="1" rowspan="1">Proteus mirabilis</td><td colspan="1" rowspan="1">CDC-59</td><td colspan="1" rowspan="1">7.37 × 107</td><td colspan="1" rowspan="1">8.10 × 108</td></tr><tr><td colspan="1" rowspan="1">Proteus penneri</td><td colspan="1" rowspan="1">Z028</td><td colspan="1" rowspan="1">8.88 × 107</td><td colspan="1" rowspan="1">4.33 × 108</td></tr><tr><td colspan="1" rowspan="1">Proteus vulgaris</td><td colspan="1" rowspan="1">Z129</td><td colspan="1" rowspan="1">4.37 × 107</td><td colspan="1" rowspan="1">5.00 × 108</td></tr><tr><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">Z139</td><td colspan="1" rowspan="1">9.18 × 107</td><td colspan="1" rowspan="1">1.37 × 1010</td></tr><tr><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">CDC-231</td><td colspan="1" rowspan="1">3.26 × 108</td><td colspan="1" rowspan="1">7.98 × 108</td></tr><tr><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">CDC-250</td><td colspan="1" rowspan="1">1.64 × 108</td><td colspan="1" rowspan="1">8.97 × 108</td></tr><tr><td colspan="1" rowspan="1">Serratia marcescens</td><td colspan="1" rowspan="1">ATCC 43297</td><td colspan="1" rowspan="1">8.55 × 108</td><td colspan="1" rowspan="1">2.03 × 109</td></tr><tr><td colspan="1" rowspan="1">Serratia marcescens</td><td colspan="1" rowspan="1">ATCC 21212</td><td colspan="1" rowspan="1">1.07× 108</td><td colspan="1" rowspan="1">8.83 × 108</td></tr><tr><td colspan="1" rowspan="1">Serratia marcescens</td><td colspan="1" rowspan="1">CDC-91</td><td colspan="1" rowspan="1">7.28 × 108</td><td colspan="1" rowspan="1">1.67 × 109</td></tr></table>

# 3. Blood Culture Bottle Equivalency:

Commonly used blood culture bottle (BCB) media types were evaluated to demonstrate that variability in BCB media composition does not interfere with iC-GN Assay performance. Twenty-seven (27) representative iC-GN target organisms plus one nontarget organism were tested in thirteen (13) BCB media types. Target organisms were tested near LoD concentrations $( 2 { - } 3 { \times } \mathrm { L o D } )$ ). Each strain was tested in triplicate in each BCB media type. Target performance is based on all expected targets detected and no false positive targets detected. Non-target performance is based on all expected negative results. In the event of a false negative result, the strain was retested in replicates of ten. In the event of a false positive result or other failure, the strain was retested in triplicate. The results of iC-GN BCB equivalency testing are summarized in the table below. Performance in all bottle types met the acceptance criteria of $\ge 9 5 \%$ performance; all bottle types are validated for use with the iC GN Assay.

<table><tr><td colspan="6" rowspan="1">Table 3: iC-GN Assay BCB Equivalency Results</td></tr><tr><td colspan="1" rowspan="1">BCB Media Type</td><td colspan="1" rowspan="1">OverallPerformance(%)</td><td colspan="1" rowspan="1">FalseNegatives(%)</td><td colspan="1" rowspan="1">FalsePositives(%)</td><td colspan="1" rowspan="1">PCCheckFailures(%)</td><td colspan="1" rowspan="1">SystemFailures(%)</td></tr><tr><td colspan="1" rowspan="1">BACTEC Standard Aerobic</td><td colspan="1" rowspan="1">93/94(98.9%)</td><td colspan="1" rowspan="1">1/94(1.1%)</td><td colspan="1" rowspan="1">0/94(0.0%)</td><td colspan="1" rowspan="1">3/97(3.1%)</td><td colspan="1" rowspan="1">0/97(0.0%)</td></tr><tr><td colspan="1" rowspan="1">BACTEC Standard Anaerobic</td><td colspan="1" rowspan="1">85/86(98.8%)</td><td colspan="1" rowspan="1">0/86(0.0%)</td><td colspan="1" rowspan="1">1/86(1.2%)</td><td colspan="1" rowspan="1">0/87(0.0%)</td><td colspan="1" rowspan="1">1/87(1.1%)</td></tr><tr><td colspan="1" rowspan="1">BACTEC Plus Aerobic</td><td colspan="1" rowspan="1">93/94(98.9%)</td><td colspan="1" rowspan="1">1/94(1.1%)</td><td colspan="1" rowspan="1">0/94(0.0%)</td><td colspan="1" rowspan="1">2/97(2.1%)</td><td colspan="1" rowspan="1">1/97(1.1%)</td></tr><tr><td colspan="1" rowspan="1">BACTEC Plus Anaerobic</td><td colspan="1" rowspan="1">95/96(98.6%)</td><td colspan="1" rowspan="1">1/96(1.0%)</td><td colspan="1" rowspan="1">0/96(0.0%)</td><td colspan="1" rowspan="1">2/100(2.0%)</td><td colspan="1" rowspan="1">2/100(2.0%)</td></tr><tr><td colspan="1" rowspan="1">BACTEC Lytic/10 Anaerobic</td><td colspan="1" rowspan="1">81/81(100.0%)</td><td colspan="1" rowspan="1">0/81(0.0%)</td><td colspan="1" rowspan="1">0/81(0.0%)</td><td colspan="1" rowspan="1">0/81(0.0%)</td><td colspan="1" rowspan="1">0/81(0.0%)</td></tr><tr><td colspan="1" rowspan="1">BACT/ALERT SA StandardAerobic</td><td colspan="1" rowspan="1">97/99(98.0%)</td><td colspan="1" rowspan="1">1/99(1.0%)</td><td colspan="1" rowspan="1">1/99(1.0%)</td><td colspan="1" rowspan="1">4/103(3.9%)</td><td colspan="1" rowspan="1">0/103(0.0%)</td></tr><tr><td colspan="1" rowspan="1">BACT/ALERT SN StandardAnaerobic</td><td colspan="1" rowspan="1">87/88(98.9%)</td><td colspan="1" rowspan="1">0/88(0.0%)</td><td colspan="1" rowspan="1">1/88(1.1%)</td><td colspan="1" rowspan="1">2/90(2.2%)</td><td colspan="1" rowspan="1">0/90(0.0%)</td></tr><tr><td colspan="1" rowspan="1">BACT/ALERT FA Aerobic FAN</td><td colspan="1" rowspan="1">94/96(97.9%)</td><td colspan="1" rowspan="1">0/96(0.0%)</td><td colspan="1" rowspan="1">2/96(2.1%)</td><td colspan="1" rowspan="1">1/97(1.0%)</td><td colspan="1" rowspan="1">0/97(0.0%)</td></tr><tr><td colspan="1" rowspan="1">BACT/ALERT FN AnaerobicFAN</td><td colspan="1" rowspan="1">92/94(97.9%)</td><td colspan="1" rowspan="1">0/94(0.0%)</td><td colspan="1" rowspan="1">2/94(2.1%)</td><td colspan="1" rowspan="1">2/97(2.1%)</td><td colspan="1" rowspan="1">1/97(1.0%)</td></tr><tr><td colspan="1" rowspan="1">BACT/ALERT FA Plus Aerobic</td><td colspan="1" rowspan="1">94/95(98.9%)</td><td colspan="1" rowspan="1">1/95(1.1%)</td><td colspan="1" rowspan="1">0/95(0.0%)</td><td colspan="1" rowspan="1">1/97(1.1%)</td><td colspan="1" rowspan="1">1/97(1.1%)</td></tr><tr><td colspan="1" rowspan="1">BACT/ALERT FN PlusAnaerobic</td><td colspan="1" rowspan="1">87/87(100.0%)</td><td colspan="1" rowspan="1">0/87(0.0%)</td><td colspan="1" rowspan="1">0/87(0.0%)</td><td colspan="1" rowspan="1">2/90(2.2%)</td><td colspan="1" rowspan="1">1/90(1.1%)</td></tr><tr><td colspan="1" rowspan="1">VersaTREK REDOX 1</td><td colspan="1" rowspan="1">81/81(100.0%)</td><td colspan="1" rowspan="1">0/81(0.0%)</td><td colspan="1" rowspan="1">0/81(0.0%)</td><td colspan="1" rowspan="1">0/81(0.0%)</td><td colspan="1" rowspan="1">0/81(0.0%)</td></tr><tr><td colspan="1" rowspan="1">VersaTREK REDOX 1</td><td colspan="1" rowspan="1">92/93(98.9%)</td><td colspan="1" rowspan="1">1/93(1.1%)</td><td colspan="1" rowspan="1">0/93(0.0%)</td><td colspan="1" rowspan="1">1/94(1.1%)</td><td colspan="1" rowspan="1">0/94(0.0%)</td></tr></table>

4. Inclusivity:

To demonstrate the inclusivity of the iC-GN Assay, eighty-two (82) representative strains were evaluated, a minimum of ten strains for each target analyte. Strains were tested at the lowest level of bottle positivity, considered within two hours of bottle “ring.”

Organisms were grown in BD BACTEC Plus Aerobic blood culture bottles with human blood added on the BD BACTEC System. Each strain was tested in triplicate.

Performance calculations are based on all expected targets detected and no false positive targets detected. In the event of a false negative result, the strain was retested in replicates of ten. In the event of a false positive result or other failure, the strain was retested in triplicate. Two strains were not detected by the iC-GN Assay: Acinetobacter calcoaceticus ATCC 31926 was not detected as A. baumannii complex and Enterobacter kobei ATCC BAA-260 was not detected as E. cloacae complex. The results of iC-GN Inclusivity testing are summarized below. The results of the inclusivity testing support identification of the organisms and resistance markers claimed in the IFU, with the exceptions noted above. It was determined that the risks associated with failure to detect a small number of organisms included in the IFU could be mitigated via labeling. These exceptions are communicated to end users in the Limitations section of the device Package Insert.

<table><tr><td colspan="4" rowspan="1">Table 4: iC-GN Assay Inclusivity Results</td></tr><tr><td colspan="1" rowspan="1">Organism</td><td colspan="1" rowspan="1">Strain</td><td colspan="1" rowspan="1">Targets</td><td colspan="1" rowspan="1">Performance</td></tr><tr><td colspan="1" rowspan="1">Enterobacter kobei</td><td colspan="1" rowspan="1">ATCC BAA-260</td><td colspan="1" rowspan="1">ECX</td><td colspan="1" rowspan="1">0/133</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">ATCC 10536</td><td colspan="1" rowspan="1">EC</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">ATCC BAA-2469</td><td colspan="1" rowspan="1">EC, NDM-1</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">NCTC 9001</td><td colspan="1" rowspan="1">EC</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">NCTC 10538</td><td colspan="1" rowspan="1">EC</td><td colspan="1" rowspan="1">5/5</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">NCTC13476</td><td colspan="1" rowspan="1">EC</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">CDC-48</td><td colspan="1" rowspan="1">EC, CTX-M-15, NDM-1</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">CDC-61</td><td colspan="1" rowspan="1">EC, KPC-3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">CDC-104</td><td colspan="1" rowspan="1">EC, KPC-4</td><td colspan="1" rowspan="1">7/84</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">CDC-119</td><td colspan="1" rowspan="1">EC, CTX-M-15, NDM-1</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">CDC-162</td><td colspan="1" rowspan="1">EC, CTX-M-15, NDM-7</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Klebsiella oxytoca</td><td colspan="1" rowspan="1">ATCC 8724</td><td colspan="1" rowspan="1">KO</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Klebsiella oxytoca</td><td colspan="1" rowspan="1">ATCC 43086</td><td colspan="1" rowspan="1">KO</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Klebsiella oxytoca</td><td colspan="1" rowspan="1">ATCC 43165</td><td colspan="1" rowspan="1">KO</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Klebsiella oxytoca</td><td colspan="1" rowspan="1">ATCC 43863</td><td colspan="1" rowspan="1">KO</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Klebsiella oxytoca</td><td colspan="1" rowspan="1">ATCC 49134</td><td colspan="1" rowspan="1">KO</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Klebsiella oxytoca</td><td colspan="1" rowspan="1">ATCC 49334</td><td colspan="1" rowspan="1">KO</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Klebsiella oxytoca</td><td colspan="1" rowspan="1">ATCC 51817</td><td colspan="1" rowspan="1">KO</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Klebsiella oxytoca</td><td colspan="1" rowspan="1">ATCC 700324</td><td colspan="1" rowspan="1">KO</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Klebsiella oxytoca</td><td colspan="1" rowspan="1">NCTC 11686</td><td colspan="1" rowspan="1">KO</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Klebsiella oxytoca</td><td colspan="1" rowspan="1">CDC-71</td><td colspan="1" rowspan="1">KO</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">ATCC-13882</td><td colspan="1" rowspan="1">KPN</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">ATCC BAA-1705</td><td colspan="1" rowspan="1">KPN, KPC-2</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">NCTC 9633</td><td colspan="1" rowspan="1">KPN</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">NCTC 13438</td><td colspan="1" rowspan="1">KPN, KPC-3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">NCTC 13443</td><td colspan="1" rowspan="1">KPN, CTX-M-15, NDM-1</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">CDC-44</td><td colspan="1" rowspan="1">KPN, CTX-M-15</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">CDC-46</td><td colspan="1" rowspan="1">KPN, CTX-M-15</td><td colspan="1" rowspan="1">5/5</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">CDC-49</td><td colspan="1" rowspan="1">KPN, CTX-M-15, NDM-1</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">CDC-66</td><td colspan="1" rowspan="1">KPN, CTX-M-15</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae subsp.ozaenae</td><td colspan="1" rowspan="1">ATCC 11296</td><td colspan="1" rowspan="1">KPN</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Proteus mirabilis</td><td colspan="1" rowspan="1">ATCC 7002</td><td colspan="1" rowspan="1">Proteus</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Proteus mirabilis</td><td colspan="1" rowspan="1">ATCC 21100</td><td colspan="1" rowspan="1">Proteus</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Proteus mirabilis</td><td colspan="1" rowspan="1">ATCC 43071</td><td colspan="1" rowspan="1">Proteus</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Proteus mirabilis</td><td colspan="1" rowspan="1">NCIMB 13283</td><td colspan="1" rowspan="1">Proteus</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Proteus mirabilis</td><td colspan="1" rowspan="1">CDC-155</td><td colspan="1" rowspan="1">Proteus, KPC-6</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Proteus mirabilis</td><td colspan="1" rowspan="1">CDC-156</td><td colspan="1" rowspan="1">Proteus, KPC-2</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Proteus mirabilis</td><td colspan="1" rowspan="1">CDC-159</td><td colspan="1" rowspan="1">Proteus, NDM-1</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Proteus penneri</td><td colspan="1" rowspan="1">ATCC 33519</td><td colspan="1" rowspan="1">Proteus</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Proteus vulgaris</td><td colspan="1" rowspan="1">ATCC 9484</td><td colspan="1" rowspan="1">Proteus</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Proteus vulgaris</td><td colspan="1" rowspan="1">ATCC 29905</td><td colspan="1" rowspan="1">Proteus</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">ATCC 10145</td><td colspan="1" rowspan="1">PA</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">ATCC 19429</td><td colspan="1" rowspan="1">PA</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">ATCC BAA-1744</td><td colspan="1" rowspan="1">PA</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">CDC-54</td><td colspan="1" rowspan="1">PA</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">CDC-64</td><td colspan="1" rowspan="1">PA</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">CDC-90</td><td colspan="1" rowspan="1">PA, KPC-5</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">CDC-94</td><td colspan="1" rowspan="1">PA</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">CDC-105</td><td colspan="1" rowspan="1">PA</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">CDC-108</td><td colspan="1" rowspan="1">PA</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">CDC-246</td><td colspan="1" rowspan="1">PA, NDM-1</td><td colspan="1" rowspan="1">5/5</td></tr><tr><td colspan="1" rowspan="1">Serratia marcescens</td><td colspan="1" rowspan="1">ATCC 8100</td><td colspan="1" rowspan="1">SM</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Serratia marcescens</td><td colspan="1" rowspan="1">ATCC 13880</td><td colspan="1" rowspan="1">SM</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Serratia marcescens</td><td colspan="1" rowspan="1">ATCC 14041</td><td colspan="1" rowspan="1">SM</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Serratia marcescens</td><td colspan="1" rowspan="1">ATCC 14756</td><td colspan="1" rowspan="1">SM</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Serratia marcescens</td><td colspan="1" rowspan="1">ATCC 29634</td><td colspan="1" rowspan="1">SM</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Serratia marcescens</td><td colspan="1" rowspan="1">ATCC 29635</td><td colspan="1" rowspan="1">SM</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Serratia marcescens</td><td colspan="1" rowspan="1">ATCC 43861</td><td colspan="1" rowspan="1">SM</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Serratia marcescens</td><td colspan="1" rowspan="1">ATCC 43862</td><td colspan="1" rowspan="1">SM</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Serratia marcescens</td><td colspan="1" rowspan="1">NCTC 9743</td><td colspan="1" rowspan="1">SM</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Serratia marcescens</td><td colspan="1" rowspan="1">CDC-99</td><td colspan="1" rowspan="1">SM</td><td colspan="1" rowspan="1">3/3</td></tr></table>

1) 2/2 false negative ABX in initial testing. 7/9 false negative ABX in repeat testing. See limitation.   
2) 1/3 false positive ABX in initial testing. 1/3 false positive ABX in repeat testing. Strain repeated in replicates of 10, 10/10 repeats passed.   
3) 3/3 false negative ECX in initial testing. 10/10 false negative ECX in repeat testing. See limitation.   
4) 1/3 processor error in initial testing. 1/3 false positive KPN in repeat testing. Strain repeated in triplicate, 3/3 repeats passed.

An in silico analysis was performed for each resistance marker. The predicted reactivity of each resistance marker detected by the iC-GN Assay is summarized in the tables below.

<table><tr><td rowspan=1 colspan=7>Table 5: Predicted (in silico) Reactivity for CTX-M group 1</td></tr><tr><td rowspan=1 colspan=1>Associated Target Organism</td><td rowspan=1 colspan=1>Variant Detected</td><td rowspan=1 colspan=4>Associated Target Organism</td><td rowspan=1 colspan=1>Variant Detected</td></tr><tr><td rowspan=2 colspan=1>Acinetobacter baumanniicomplex</td><td rowspan=1 colspan=1>CTX-M-3</td><td rowspan=5 colspan=4>Klebsiella oxytoca</td><td rowspan=1 colspan=1>CTX-M-3</td></tr><tr><td rowspan=1 colspan=1>CTX-M-15</td><td rowspan=1 colspan=1>CTX-M-15</td></tr><tr><td rowspan=10 colspan=1>Enterobacter cloacaecomplex</td><td rowspan=1 colspan=1>CTX-M-1</td><td rowspan=1 colspan=1>CTX-M-35</td></tr><tr><td rowspan=1 colspan=1>CTX-M-3</td><td rowspan=1 colspan=1>CTX-M-36</td></tr><tr><td rowspan=1 colspan=1>CTX-M-15</td><td rowspan=1 colspan=1>CTX-M-162</td></tr><tr><td rowspan=1 colspan=1>CTX-M-22</td><td rowspan=14 colspan=4>Klebsiella pneumoniae</td><td rowspan=1 colspan=1>CTX-M-1</td></tr><tr><td rowspan=1 colspan=1>CTX-M-37</td><td rowspan=1 colspan=1>CTX-M-3</td></tr><tr><td rowspan=1 colspan=1>CTX-M-55</td><td rowspan=1 colspan=1>CTX-M-15</td></tr><tr><td rowspan=1 colspan=1>CTX-M-167</td><td rowspan=1 colspan=1>CTX-M-22</td></tr><tr><td rowspan=1 colspan=1>CTX-M-177</td><td rowspan=1 colspan=1>CTX-M-28</td></tr><tr><td rowspan=1 colspan=1>CTX-M-187</td><td rowspan=1 colspan=2></td><td rowspan=1 colspan=1>CTX-M-32</td></tr><tr><td rowspan=1 colspan=1>CTX-M-224</td><td rowspan=1 colspan=1>CTX-M-54</td></tr><tr><td rowspan=32 colspan=1>Escherichia coli</td><td rowspan=1 colspan=1>CTX-M-1</td><td rowspan=1 colspan=1>CTX-M-55</td></tr><tr><td rowspan=1 colspan=1>CTX-M-2</td><td rowspan=1 colspan=1>CTX-M-71</td></tr><tr><td rowspan=1 colspan=1>CTX-M-3</td><td rowspan=1 colspan=1>CTX-M-72</td></tr><tr><td rowspan=1 colspan=1>CTX-M-4</td><td rowspan=1 colspan=1>CTX-M-118</td></tr><tr><td rowspan=1 colspan=1>CTX-M-5</td><td rowspan=1 colspan=1>CTX-M-124</td></tr><tr><td rowspan=1 colspan=1>CTX-M-6</td><td rowspan=1 colspan=1>CTX-M-129</td></tr><tr><td rowspan=1 colspan=1>CTX-M-7</td><td rowspan=1 colspan=1>CTX-M-130</td></tr><tr><td rowspan=1 colspan=1>CTX-M-8</td><td rowspan=3 colspan=4></td><td rowspan=1 colspan=1>CTX-M-133</td></tr><tr><td rowspan=1 colspan=1>CTX-M-9</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>CTX-M-135</td></tr><tr><td rowspan=1 colspan=1>CTX-M-10</td><td rowspan=11 colspan=4>−</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>CTX-M-138</td></tr><tr><td rowspan=1 colspan=1>CTX-M-11</td><td rowspan=1 colspan=1>CTX-M-139</td></tr><tr><td rowspan=1 colspan=1>CTX-M-12</td><td rowspan=1 colspan=1>CTX-M-173</td></tr><tr><td rowspan=1 colspan=1>CTX-M-15</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>CTX-M-176</td></tr><tr><td rowspan=1 colspan=1>CTX-M-28</td><td rowspan=1 colspan=1>CTX-M-183</td></tr><tr><td rowspan=1 colspan=1>CTX-M-29</td><td rowspan=1 colspan=1>CTX-M-188</td></tr><tr><td rowspan=1 colspan=1>CTX-M-32</td><td rowspan=1 colspan=1>CTX-M-197</td></tr><tr><td rowspan=1 colspan=1>CTX-M-33</td><td rowspan=1 colspan=1>CTX-M-204</td></tr><tr><td rowspan=1 colspan=1>CTX-M-36</td><td rowspan=1 colspan=1>CTX-M-208</td></tr><tr><td rowspan=1 colspan=1>CTX-M-42</td><td rowspan=1 colspan=1>CTX-M-210</td></tr><tr><td rowspan=1 colspan=1>CTX-M-55</td><td rowspan=1 colspan=1>CTX-M-220</td></tr><tr><td rowspan=1 colspan=1>CTX-M-58</td><td rowspan=7 colspan=4>Proteus species</td><td rowspan=1 colspan=1>CTX-M-15</td></tr><tr><td rowspan=1 colspan=1>CTX-M-69</td><td rowspan=1 colspan=1>CTX-M-66</td></tr><tr><td rowspan=1 colspan=1>CTX-M-71</td><td rowspan=1 colspan=1>CTX-M-116</td></tr><tr><td rowspan=1 colspan=1>CTX-M-79</td><td rowspan=1 colspan=1>CTX-M-136</td></tr><tr><td rowspan=1 colspan=1>CTX-M-82</td><td rowspan=1 colspan=1>CTX-M-164</td></tr><tr><td rowspan=1 colspan=1>CTX-M-90</td><td rowspan=1 colspan=1>CTX-M-167</td></tr><tr><td rowspan=1 colspan=1>CTX-M-101</td><td rowspan=1 colspan=1>CTX-M-212</td></tr><tr><td rowspan=1 colspan=1>CTX-M-102</td><td rowspan=3 colspan=4>Pseudomonas aeruginosa</td><td rowspan=1 colspan=1>CTX-M-1</td></tr><tr><td rowspan=1 colspan=1>CTX-M-103</td><td rowspan=1 colspan=1>CTX-M-15</td></tr><tr><td rowspan=1 colspan=1>CTX-M-109</td><td rowspan=1 colspan=1>CTX-M-32</td></tr><tr><td rowspan=1 colspan=1>CTX-M-117</td><td rowspan=2 colspan=4></td><td rowspan=1 colspan=1>CTX-M-3</td></tr><tr><td rowspan=1 colspan=1>CTX-M-120</td><td rowspan=1 colspan=1>CTX-M-15</td></tr></table>

<table><tr><td rowspan=3 colspan=1>CTX-M-125</td><td rowspan=4 colspan=3></td><td></td><td></td></tr><tr><td rowspan=2 colspan=2></td><td></td></tr><tr><td rowspan=1 colspan=1>b</td><td rowspan=1 colspan=1>CTX-M-22</td></tr><tr><td rowspan=1 colspan=1>CTX-M-127</td><td rowspan=1 colspan=2></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>CTX-M-55</td></tr><tr><td rowspan=1 colspan=1>CTX-M-128</td><td rowspan=1 colspan=4>Serratia marcescens</td><td rowspan=1 colspan=1>CTX-M-221</td></tr><tr><td rowspan=1 colspan=1>CTX-M-131</td><td rowspan=24 colspan=5></td></tr><tr><td rowspan=1 colspan=1>CTX-M-132</td></tr><tr><td rowspan=1 colspan=1>CTX-M-134</td></tr><tr><td rowspan=1 colspan=1>CTX-M-137</td></tr><tr><td rowspan=1 colspan=1>CTX-M-138</td></tr><tr><td rowspan=1 colspan=1>CTX-M-139</td></tr><tr><td rowspan=1 colspan=1>CTX-M-140</td></tr><tr><td rowspan=1 colspan=1>CTX-M-141</td></tr><tr><td rowspan=1 colspan=1>CTX-M-142</td></tr><tr><td rowspan=1 colspan=1>CTX-M-143</td></tr><tr><td rowspan=1 colspan=1>CTX-M-146</td></tr><tr><td rowspan=1 colspan=1>CTX-M-158</td></tr><tr><td rowspan=1 colspan=1>CTX-M-163</td></tr><tr><td rowspan=1 colspan=1>CTX-M-166</td></tr><tr><td rowspan=1 colspan=1>CTX-M-167</td></tr><tr><td rowspan=1 colspan=1>CTX-M-170</td></tr><tr><td rowspan=1 colspan=1>CTX-M-172</td></tr><tr><td rowspan=1 colspan=1>CTX-M-175</td></tr><tr><td rowspan=1 colspan=1>CTX-M-178</td></tr><tr><td rowspan=1 colspan=1>CTX-M-179</td></tr><tr><td rowspan=1 colspan=1>CTX-M-180</td></tr><tr><td rowspan=1 colspan=1>CTX-M-181</td></tr><tr><td rowspan=1 colspan=1>CTX-M-182</td></tr><tr><td rowspan=1 colspan=1>CTX-M-184</td></tr></table>

<table><tr><td colspan="7" rowspan="1">Table 5a: Predicted (in silico) Reactivity for KPC</td></tr><tr><td colspan="1" rowspan="1">Associated Target Organism</td><td colspan="1" rowspan="1">Variant Detected</td><td colspan="4" rowspan="1">Associated Target Organism</td><td colspan="1" rowspan="1">Variant Detected</td></tr><tr><td colspan="1" rowspan="3">Acinetobacter baumanniicomplex</td><td colspan="1" rowspan="1">KPC-2</td><td colspan="4" rowspan="4"></td><td colspan="1" rowspan="1">KPC-1</td></tr><tr><td colspan="1" rowspan="1">KPC-3</td><td colspan="1" rowspan="1">KPC-2</td></tr><tr><td colspan="1" rowspan="1">KPC-10</td><td colspan="1" rowspan="1">KPC-3</td></tr><tr><td colspan="1" rowspan="7">Enterobacter cloacaecomplex</td><td colspan="1" rowspan="1">KPC-1</td><td colspan="1" rowspan="1">KPC-4</td></tr><tr><td colspan="1" rowspan="1">KPC-2</td><td colspan="4" rowspan="2"></td><td colspan="1" rowspan="1">KPC-5</td></tr><tr><td colspan="1" rowspan="1">KPC-3</td><td colspan="1" rowspan="1">KPC-6</td></tr><tr><td colspan="1" rowspan="1">KPC-4</td><td colspan="4" rowspan="2"></td><td colspan="1" rowspan="1">KPC-7</td></tr><tr><td colspan="1" rowspan="1">KPC-13</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">KPC-8</td></tr><tr><td colspan="1" rowspan="1">KPC-18</td><td colspan="4" rowspan="1"></td><td colspan="1" rowspan="1">KPC-11</td></tr><tr><td colspan="1" rowspan="1">KPC-47</td><td colspan="4" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">KPC-14</td></tr><tr><td colspan="1" rowspan="7">Escherichia coli</td><td colspan="1" rowspan="1">KPC-2</td><td colspan="4" rowspan="1"></td><td colspan="1" rowspan="1">KPC-15</td></tr><tr><td colspan="1" rowspan="1">KPC-3</td><td colspan="4" rowspan="4"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">KPC-12</td><td colspan="2" rowspan="1">−</td><td colspan="1" rowspan="1">KPC-17</td></tr><tr><td colspan="1" rowspan="1">KPC-18</td><td colspan="2" rowspan="1">−</td><td colspan="1" rowspan="1">KPC-19</td></tr><tr><td colspan="1" rowspan="1">KPC-20</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">KPC-22</td></tr><tr><td colspan="1" rowspan="1">KPC-21</td><td colspan="4" rowspan="2"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">KPC-23</td></tr><tr><td colspan="1" rowspan="1">KPC-28</td><td colspan="2" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">KPC-25</td></tr><tr><td colspan="1" rowspan="2">Klebsiella oxytoca</td><td colspan="1" rowspan="1">KPC-2</td><td colspan="2" rowspan="1"></td><td colspan="2" rowspan="2"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">KPC-3</td><td></td><td></td><td colspan="1" rowspan="1">KPC-27</td></tr><tr><td colspan="1" rowspan="2">Proteus species</td><td colspan="1" rowspan="1">KPC-1</td><td colspan="6" rowspan="5"></td><td colspan="1" rowspan="1">KPC-29</td></tr><tr><td colspan="1" rowspan="1">KPC-2</td><td colspan="1" rowspan="1">KPC-30</td></tr><tr><td colspan="1" rowspan="2">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">KPC-2</td><td colspan="1" rowspan="1">KPC-31</td></tr><tr><td colspan="1" rowspan="1">KPC-5</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">KPC-32</td></tr><tr><td colspan="1" rowspan="1">Serratia marcescens</td><td colspan="1" rowspan="1">KPC-2</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">KPC-33</td></tr><tr><td colspan="2" rowspan="10"></td><td colspan="6" rowspan="10"></td><td colspan="1" rowspan="2"></td></tr><tr><td colspan="1" rowspan="1">KPC-34</td></tr><tr><td colspan="1" rowspan="1">KPC-35</td></tr><tr><td colspan="1" rowspan="1">KPC-36</td></tr><tr><td colspan="1" rowspan="1">KPC-37</td></tr><tr><td colspan="1" rowspan="1">KPC-38</td></tr><tr><td colspan="1" rowspan="1">KPC-39</td></tr><tr><td colspan="1" rowspan="1">KPC-42</td></tr><tr><td colspan="1" rowspan="1">KPC-43</td></tr><tr><td colspan="1" rowspan="1">KPC-59</td></tr></table>

<table><tr><td rowspan=1 colspan=4>Table 5b: Predicted (in silico) Reactivity for NDM</td></tr><tr><td rowspan=1 colspan=1>Associated Target Organism</td><td rowspan=1 colspan=1>Variant Detected</td><td rowspan=1 colspan=1>Associated Target Organism</td><td rowspan=1 colspan=1>Variant Detected</td></tr><tr><td rowspan=7 colspan=1>Acinetobacter baumanniicomplex</td><td rowspan=1 colspan=1>NDM-1</td><td rowspan=3 colspan=1>Klebsiella oxytoca</td><td rowspan=1 colspan=1>NDM-1</td></tr><tr><td rowspan=1 colspan=1>NDM-2</td><td rowspan=1 colspan=1>NDM-3</td></tr><tr><td rowspan=1 colspan=1>NDM-3</td><td rowspan=1 colspan=1>NDM-4</td></tr><tr><td rowspan=1 colspan=1>NDM-4</td><td rowspan=5 colspan=1></td><td rowspan=1 colspan=1>NDM-1</td></tr><tr><td rowspan=1 colspan=1>NDM-5</td><td rowspan=1 colspan=1>NDM-3</td></tr><tr><td rowspan=1 colspan=1>NDM-7</td><td rowspan=1 colspan=1>NDM-4</td></tr><tr><td rowspan=1 colspan=1>NDM-14</td><td rowspan=1 colspan=1>NDM-5</td></tr><tr><td rowspan=5 colspan=1>Enterobacter cloacaecomplex</td><td rowspan=1 colspan=1>NDM-1</td><td rowspan=1 colspan=1>NDM-6</td></tr><tr><td rowspan=1 colspan=1>NDM-4</td><td rowspan=1 colspan=1>Klebsiella pneumoniae</td><td rowspan=1 colspan=1>NDM-7</td></tr><tr><td rowspan=1 colspan=1>NDM-5</td><td rowspan=5 colspan=1></td><td rowspan=1 colspan=1>NDM-9</td></tr><tr><td rowspan=1 colspan=1>NDM-7</td><td rowspan=1 colspan=1>NDM-10</td></tr><tr><td rowspan=1 colspan=1>NDM-22</td><td rowspan=1 colspan=1>NDM-16</td></tr><tr><td rowspan=19 colspan=1>Escherichia coli</td><td rowspan=1 colspan=1>NDM-1</td><td rowspan=1 colspan=1>NDM-23</td></tr><tr><td rowspan=1 colspan=1>NDM-2</td><td rowspan=1 colspan=1>NDM-28</td></tr><tr><td rowspan=1 colspan=1>NDM-3</td><td rowspan=1 colspan=1>Proteus species</td><td rowspan=1 colspan=1>NDM-1</td></tr><tr><td rowspan=1 colspan=1>NDM-4</td><td rowspan=2 colspan=1>Pseudomonas aeruginosa</td><td rowspan=1 colspan=1>NDM-1</td></tr><tr><td rowspan=1 colspan=1>NDM-5</td><td rowspan=1 colspan=1>NDM-5</td></tr><tr><td rowspan=1 colspan=1>NDM-6</td><td rowspan=3 colspan=1>Serratia marcescens</td><td rowspan=1 colspan=1>NDM-1</td></tr><tr><td rowspan=1 colspan=1>NDM-7</td><td rowspan=1 colspan=1>NDM-4</td></tr><tr><td rowspan=1 colspan=1>NDM-9</td><td rowspan=1 colspan=1>NDM-12</td></tr><tr><td rowspan=1 colspan=1>NDM-11</td><td rowspan=11 colspan=2></td></tr><tr><td rowspan=1 colspan=1>NDM-12</td></tr><tr><td rowspan=1 colspan=1>NDM-13</td></tr><tr><td rowspan=1 colspan=1>NDM-15</td></tr><tr><td rowspan=1 colspan=1>NDM-16</td></tr><tr><td rowspan=1 colspan=1>NDM-17</td></tr><tr><td rowspan=1 colspan=1>NDM-18</td></tr><tr><td rowspan=1 colspan=1>NDM-19</td></tr><tr><td rowspan=1 colspan=1>NDM-20</td></tr><tr><td rowspan=1 colspan=1>NDM-21</td></tr><tr><td rowspan=1 colspan=1>NDM-27</td></tr></table>

5. Exclusivity:

To demonstrate the exclusivity of the iC-GN Assay, a comprehensive panel of non-target organisms that may be encountered in positive blood cultures was evaluated. A total of 114 strains were tested including organisms phylogenetically related to iC-GN target organisms as well as common blood culture contaminants. Potential cross-reactivity was evaluated by testing exclusivity panel organisms at the highest possible concentrations, considered eight hours beyond initial bottle positivity or the equivalent. Organisms were grown in BD BACTEC Plus Aerobic blood culture bottles with human blood added on the BD BACTEC System. Each strain was tested in triplicate. Performance is based on the observation of all expected negative results. In the event of a false positive result or other failure, the organism was retested in replicates of three (3) or ten (10). Three (3) strains demonstrated reproducible cross-reactivity with iC-GN Assay targets:

Acinetobacter haemolyticus cross-reacted with Acinetobacter baumannii complex, Klebsiella variicola cross-reacted with Klebsiella pneumoniae, and Serratia odorifera cross-reacted with Serratia marcescens. Exclusivity results are in the table below.

<table><tr><td colspan="4" rowspan="1">Table 6: iC-GN Assay Exclusivity Results</td></tr><tr><td colspan="1" rowspan="1">Organism</td><td colspan="1" rowspan="1">Strain</td><td colspan="1" rowspan="1">Concentration(CFU/mL)</td><td colspan="1" rowspan="1">Performance</td></tr><tr><td colspan="1" rowspan="1">Acinetobacter haemolyticus</td><td colspan="1" rowspan="1">ATCC 19002</td><td colspan="1" rowspan="1">7.20 × 108</td><td colspan="1" rowspan="1">0/31</td></tr><tr><td colspan="1" rowspan="1">Acinetobacter lwoffi</td><td colspan="1" rowspan="1">Z141</td><td colspan="1" rowspan="1">2.45 × 108</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Acinetobacter radioresistens</td><td colspan="1" rowspan="1">ATCC 43998</td><td colspan="1" rowspan="1">5.20 × 107</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Acinetobacter schindleri</td><td colspan="1" rowspan="1">ATCCBAA618</td><td colspan="1" rowspan="1">3.50 × 108</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Acinetobacter ursingii</td><td colspan="1" rowspan="1">ATCCBAA617</td><td colspan="1" rowspan="1">3.80 × 108</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Aerococcus viridans</td><td colspan="1" rowspan="1">Z219</td><td colspan="1" rowspan="1">2.24 × 107</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Aeromonas hydrophila</td><td colspan="1" rowspan="1">Z161</td><td colspan="1" rowspan="1">8.10 × 108</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Alcaligenes faecalis</td><td colspan="1" rowspan="1">Z218</td><td colspan="1" rowspan="1">9.70 × 108</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Aspergillus niger</td><td colspan="1" rowspan="1">Z105</td><td colspan="1" rowspan="1">1.62 × 108</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Bacillus cereus</td><td colspan="1" rowspan="1">Z091</td><td colspan="1" rowspan="1">ND</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Bacteroides fragilis</td><td colspan="1" rowspan="1">Z029</td><td colspan="1" rowspan="1">8.40 × 109</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Brevundimonas vesicularis</td><td colspan="1" rowspan="1">ATCC 11426</td><td colspan="1" rowspan="1">3.80 × 108</td><td colspan="1" rowspan="1">5/5</td></tr><tr><td colspan="1" rowspan="1">Burkholderia cepacia</td><td colspan="1" rowspan="1">ATCC 25416</td><td colspan="1" rowspan="1">5.40 × 108</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Campylobacter coli</td><td colspan="1" rowspan="1">Z293</td><td colspan="1" rowspan="1">3.90 × 108</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Campylobacter jejuni</td><td colspan="1" rowspan="1">Z086</td><td colspan="1" rowspan="1">4.60 × 108</td><td colspan="1" rowspan="1">11/132</td></tr><tr><td colspan="1" rowspan="1">Candida albicans</td><td colspan="1" rowspan="1">Z006</td><td colspan="1" rowspan="1">ND</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Candida glabrata</td><td colspan="1" rowspan="1">Z007</td><td colspan="1" rowspan="1">3.20 × 107</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Candida krusei</td><td colspan="1" rowspan="1">Z009</td><td colspan="1" rowspan="1">1.90 × 107</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Candida parapsilosis</td><td colspan="1" rowspan="1">Z011</td><td colspan="1" rowspan="1">9.00 × 106</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Candida tropicalis</td><td colspan="1" rowspan="1">Z012</td><td colspan="1" rowspan="1">3.50 × 107</td><td colspan="1" rowspan="1">4/4</td></tr><tr><td colspan="1" rowspan="1">Cedecea davisae</td><td colspan="1" rowspan="1">ATCC 33431</td><td colspan="1" rowspan="1">6.20 × 108</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Citrobacter amalonaticus</td><td colspan="1" rowspan="1">Z051</td><td colspan="1" rowspan="1">8.4× 108</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Citrobacter braakii</td><td colspan="1" rowspan="1">ATCC 51113</td><td colspan="1" rowspan="1">4.90 × 108</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Citrobacter freundii</td><td colspan="1" rowspan="1">Z064</td><td colspan="1" rowspan="1">2.25 × 108</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Citrobacter koseri</td><td colspan="1" rowspan="1">Z039</td><td colspan="1" rowspan="1">1.14 × 109</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Citrobacter sedlakii</td><td colspan="1" rowspan="1">ATCC 51115</td><td colspan="1" rowspan="1">9.80 × 108</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">Clostridium difficile (NAP-1toxigenic)</td><td colspan="1" rowspan="1">NAP1</td><td colspan="1" rowspan="1">4.87 × 107</td><td colspan="1" rowspan="1">4/4</td></tr><tr><td colspan="1" rowspan="1">Clostridium difficile (non-toxigenic)</td><td colspan="1" rowspan="1">Z228</td><td colspan="1" rowspan="1">5.93 × 107</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Clostridium novyi*</td><td colspan="1" rowspan="1">Z179</td><td colspan="1" rowspan="1">1.14 × 107</td><td colspan="1" rowspan="1">5/5</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium amycolatum</td><td colspan="1" rowspan="1">Z284</td><td colspan="1" rowspan="1">9.26 × 108</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium genitalium</td><td colspan="1" rowspan="1">Z328</td><td colspan="1" rowspan="1">1.35 × 108</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium jeikeium</td><td colspan="1" rowspan="1">Z232</td><td colspan="1" rowspan="1">8.50 × 108</td><td colspan="1" rowspan="1">4/4</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium striatum</td><td colspan="1" rowspan="1">MCW000</td><td colspan="1" rowspan="1">2.07× 109</td><td colspan="1" rowspan="1">5/63</td></tr><tr><td colspan="1" rowspan="1">Cronobacter muytjensii</td><td colspan="1" rowspan="1">ATCC 51329</td><td colspan="1" rowspan="1">2.79 × 108</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Cronobacter sakazakii</td><td colspan="1" rowspan="1">ATCC 29544</td><td colspan="1" rowspan="1">6.90 × 108</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Cryptococcus neoformans</td><td colspan="1" rowspan="1">Serotype A</td><td colspan="1" rowspan="1">2.15× 108</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Edwardsiella tarda</td><td colspan="1" rowspan="1">Z183</td><td colspan="1" rowspan="1">8.70 × 107</td><td colspan="1" rowspan="1">4/54</td></tr><tr><td colspan="1" rowspan="1">Enterobacter aerogenes</td><td colspan="1" rowspan="1">Z052</td><td colspan="1" rowspan="1">1.77 × 109</td><td colspan="1" rowspan="1">5/5</td></tr><tr><td colspan="1" rowspan="1">Enterobacter amnigenus</td><td colspan="1" rowspan="1">ATCC 51816</td><td colspan="1" rowspan="1">7.50 × 108</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Enterococcus avium</td><td colspan="1" rowspan="1">Z171</td><td colspan="1" rowspan="1">2.58 × 108</td><td colspan="1" rowspan="1">5/65</td></tr><tr><td colspan="1" rowspan="1">Enterococcus casseliflavus</td><td colspan="1" rowspan="1">Z002</td><td colspan="1" rowspan="1">2.44 × 109</td><td colspan="1" rowspan="1">4/4</td></tr><tr><td colspan="1" rowspan="1">Enterococcus cecorum</td><td colspan="1" rowspan="1">Z208</td><td colspan="1" rowspan="1">1.03 × 109</td><td colspan="1" rowspan="1">5/66</td></tr><tr><td colspan="1" rowspan="1">Enterococcus faecalis</td><td colspan="1" rowspan="1">ATCC 51299</td><td colspan="1" rowspan="1">2.13 × 109</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Enterococcus faecium</td><td colspan="1" rowspan="1">ATCC 700221</td><td colspan="1" rowspan="1">7.20 × 108</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Enterococcus gallinarum</td><td colspan="1" rowspan="1">Z209</td><td colspan="1" rowspan="1">1.35 × 109</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Enterococcus hirae</td><td colspan="1" rowspan="1">Z193</td><td colspan="1" rowspan="1">2.37× 108</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Enterococcus raffinosus</td><td colspan="1" rowspan="1">ATCC 49427</td><td colspan="1" rowspan="1">5.40 × 108</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Escherichia fergusonii</td><td colspan="1" rowspan="1">ATCC 35469</td><td colspan="1" rowspan="1">8.70 × 108</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Escherichia hermannii</td><td colspan="1" rowspan="1">Z184</td><td colspan="1" rowspan="1">1.01 × 109</td><td colspan="1" rowspan="1">5/5</td></tr><tr><td colspan="1" rowspan="1">Escherichia vulneris</td><td colspan="1" rowspan="1">ATCC 33821</td><td colspan="1" rowspan="1">7.50 × 108</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Fusobacterium varium</td><td colspan="1" rowspan="1">Z361</td><td colspan="1" rowspan="1">2.49 × 109</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Hafnia alvei</td><td colspan="1" rowspan="1">ATCC 51815</td><td colspan="1" rowspan="1">1.37 × 109</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Haemophilus influenzae</td><td colspan="1" rowspan="1">ATCC 10211</td><td colspan="1" rowspan="1">3.09 × 109</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Haemophilus parainfluenzae</td><td colspan="1" rowspan="1">ATCC 9796</td><td colspan="1" rowspan="1">1.33 × 108</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Klebsiella variicola</td><td colspan="1" rowspan="1">ATCC 31488</td><td colspan="1" rowspan="1">4.40 × 108</td><td colspan="1" rowspan="1">0/37</td></tr><tr><td colspan="1" rowspan="1">Kluyvera ascorbata (KPC+)</td><td colspan="1" rowspan="1">CDC-0144</td><td colspan="1" rowspan="1">1.40 × 109</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Kocuria kristinae</td><td colspan="1" rowspan="1">Z250</td><td colspan="1" rowspan="1">7.20 × 107</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Kytococcus schroeteri</td><td colspan="1" rowspan="1">ATCCBAA2410</td><td colspan="1" rowspan="1">1.50 × 107</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus acidophilus</td><td colspan="1" rowspan="1">Z048</td><td colspan="1" rowspan="1">6.00 × 108</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus plantarum</td><td colspan="1" rowspan="1">17-5</td><td colspan="1" rowspan="1">5.30 × 108</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus reuteri</td><td colspan="1" rowspan="1">Z333</td><td colspan="1" rowspan="1">5.80 × 107</td><td colspan="1" rowspan="1">5/5</td></tr><tr><td colspan="1" rowspan="1">Lactococcus lactis</td><td colspan="1" rowspan="1">Z169</td><td colspan="1" rowspan="1">9.30 × 107</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Leclercia adecarboxylata</td><td colspan="1" rowspan="1">ATCC 23216</td><td colspan="1" rowspan="1">1.01 × 109</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Leminorella grimontii</td><td colspan="1" rowspan="1">Z364</td><td colspan="1" rowspan="1">4.00 × 109</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Leuconostoc mesenteroides</td><td colspan="1" rowspan="1">Z197</td><td colspan="1" rowspan="1">4.00 × 107</td><td colspan="1" rowspan="1">5/5</td></tr><tr><td colspan="1" rowspan="1">Listeria monocytogenes</td><td colspan="1" rowspan="1">ATCC 19115</td><td colspan="1" rowspan="1">2.03 × 109</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Micrococcus luteus</td><td colspan="1" rowspan="1">Z100</td><td colspan="1" rowspan="1">1.80 × 108</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Moraxella catarrhalis</td><td colspan="1" rowspan="1">ATCC 25238</td><td colspan="1" rowspan="1">1.27× 109</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Morganella morganii</td><td colspan="1" rowspan="1">ATCC 25830</td><td colspan="1" rowspan="1">1.23 × 109</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Neisseria gonorrhoeae</td><td colspan="1" rowspan="1">ATCC 19424</td><td colspan="1" rowspan="1">ND</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Neisseria lactamica</td><td colspan="1" rowspan="1">ATCC 23970</td><td colspan="1" rowspan="1">2.90 × 108</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Neisseria meningitidis</td><td colspan="1" rowspan="1">Serotype A</td><td colspan="1" rowspan="1">2.55× 108</td><td colspan="1" rowspan="1">5/5</td></tr><tr><td colspan="1" rowspan="1">Neisseria mucosa</td><td colspan="1" rowspan="1">ATCC 49233</td><td colspan="1" rowspan="1">5.80 × 108</td><td colspan="1" rowspan="1">4/4</td></tr><tr><td colspan="1" rowspan="1">Neisseria sicca</td><td colspan="1" rowspan="1">ATCC 9913</td><td colspan="1" rowspan="1">1.43 × 108</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Pantaea agglomerans</td><td colspan="1" rowspan="1">ATCC 27155</td><td colspan="1" rowspan="1">2.00 × 106</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Pasturella multocida</td><td colspan="1" rowspan="1">ATCC 12945</td><td colspan="1" rowspan="1">2.84 × 109</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">Pediococcus pentosaceus</td><td colspan="1" rowspan="1">Z226</td><td colspan="1" rowspan="1">1.91 × 108</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Planococcus citreus</td><td colspan="1" rowspan="1">ATCC 14404</td><td colspan="1" rowspan="1">1.95 × 108</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Pluralibacter gergoviae</td><td colspan="1" rowspan="1">ATCC 33028</td><td colspan="1" rowspan="1">1.27× 109</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Propionibacterium acnes</td><td colspan="1" rowspan="1">Z144</td><td colspan="1" rowspan="1">7.90 × 108</td><td colspan="1" rowspan="1">5/5</td></tr><tr><td colspan="1" rowspan="1">Providencia alcalifaciens</td><td colspan="1" rowspan="1">Z292</td><td colspan="1" rowspan="1">3.10 × 109</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Providencia rettgeri</td><td colspan="1" rowspan="1">Z370</td><td colspan="1" rowspan="1">2.20 × 109</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Providencia stuartii</td><td colspan="1" rowspan="1">Z213</td><td colspan="1" rowspan="1">1.70 × 109</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Pseudomonas fluorescens</td><td colspan="1" rowspan="1">ATCC 13525</td><td colspan="1" rowspan="1">2.43 × 108</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Pseudomonas luteola</td><td colspan="1" rowspan="1">ATCC 43273</td><td colspan="1" rowspan="1">1.09 × 108</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Pseudomonas mendocina</td><td colspan="1" rowspan="1">ATCC 25411</td><td colspan="1" rowspan="1">1.23 × 109</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Pseudomonas nitroreducens</td><td colspan="1" rowspan="1">ATCC 33634</td><td colspan="1" rowspan="1">5.30 × 108</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Pseudomonas oryzihabitans</td><td colspan="1" rowspan="1">ATCC 43272</td><td colspan="1" rowspan="1">1.70 × 107</td><td colspan="1" rowspan="1">4/58</td></tr><tr><td colspan="1" rowspan="1">Pseudomonas putida</td><td colspan="1" rowspan="1">Z030</td><td colspan="1" rowspan="1">3.30 × 108</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Pseudomonas stutzeri</td><td colspan="1" rowspan="1">ATCC 17588</td><td colspan="1" rowspan="1">6.20 × 108</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Raoultella planitcola</td><td colspan="1" rowspan="1">ATCC 33558</td><td colspan="1" rowspan="1">1.25 × 109</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Rothia mucilaginosus</td><td colspan="1" rowspan="1">Z033</td><td colspan="1" rowspan="1">5.50 × 107</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Salmonella enterica</td><td colspan="1" rowspan="1">ATCCBAA1715</td><td colspan="1" rowspan="1">2.23 × 109</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Serratia fonticola</td><td colspan="1" rowspan="1">ATCC 29844</td><td colspan="1" rowspan="1">1.18 × 109</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Serratia liquefaciens</td><td colspan="1" rowspan="1">ATCC 27592</td><td colspan="1" rowspan="1">1.24 × 109</td><td colspan="1" rowspan="1">11/129</td></tr><tr><td colspan="1" rowspan="1">Serratia odorifera</td><td colspan="1" rowspan="1">ATCC 33077</td><td colspan="1" rowspan="1">2.19 × 109</td><td colspan="1" rowspan="1">11/1310</td></tr><tr><td colspan="1" rowspan="1">Serratia rubidaea</td><td colspan="1" rowspan="1">ATCC 19278</td><td colspan="1" rowspan="1">1.54× 108</td><td colspan="1" rowspan="1">5/5</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">ATCC 700699</td><td colspan="1" rowspan="1">4.30 × 107</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus capitis</td><td colspan="1" rowspan="1">Z192</td><td colspan="1" rowspan="1">2.13 × 108</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus epidermidis</td><td colspan="1" rowspan="1">ATCC 700566</td><td colspan="1" rowspan="1">5.90 × 107</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus haemolyticus</td><td colspan="1" rowspan="1">Z067</td><td colspan="1" rowspan="1">2.70 × 107</td><td colspan="1" rowspan="1">5/5</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus hominis</td><td colspan="1" rowspan="1">Z031</td><td colspan="1" rowspan="1">9.90 × 107</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus intermedius</td><td colspan="1" rowspan="1">Z112</td><td colspan="1" rowspan="1">3.30 × 107</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus lugdunensis</td><td colspan="1" rowspan="1">Z097</td><td colspan="1" rowspan="1">2.43 × 108</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus schleiferi</td><td colspan="1" rowspan="1">Z294</td><td colspan="1" rowspan="1">2.52 × 109</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Stenotrophomonas maltophilia</td><td colspan="1" rowspan="1">ATCC BAA84</td><td colspan="1" rowspan="1">1.74 × 109</td><td colspan="1" rowspan="1">5/5</td></tr><tr><td colspan="1" rowspan="1">Streptococcus agalactiae</td><td colspan="1" rowspan="1">Z019</td><td colspan="1" rowspan="1">5.80 × 108</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Streptococcus anginosus</td><td colspan="1" rowspan="1">Z179</td><td colspan="1" rowspan="1">9.50 × 108</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Streptococcus bovis</td><td colspan="1" rowspan="1">Z167</td><td colspan="1" rowspan="1">8.00 × 108</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Streptococcus dysgalactiae</td><td colspan="1" rowspan="1">Z068</td><td colspan="1" rowspan="1">2.65 × 108</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Streptococcus intermedius</td><td colspan="1" rowspan="1">Z126</td><td colspan="1" rowspan="1">1.40 × 107</td><td colspan="1" rowspan="1">5/611</td></tr><tr><td colspan="1" rowspan="1">Streptococcus pneumoniae</td><td colspan="1" rowspan="1">ATCC 6301</td><td colspan="1" rowspan="1">3.80 × 108</td><td colspan="1" rowspan="1">5/612</td></tr><tr><td colspan="1" rowspan="1">Streptococcus pyogenes</td><td colspan="1" rowspan="1">Z018</td><td colspan="1" rowspan="1">4.80 × 107</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Veillonella parvula</td><td colspan="1" rowspan="1">Z379</td><td colspan="1" rowspan="1">6.70 × 109</td><td colspan="1" rowspan="1">5/613</td></tr></table>

1) 3/3 false positive $A$ . baumannii complex in initial testing. See limitation.   
2) 2/3 false positive $E$ . coli in initial testing. 10/10 repeats negative.   
3) 1/3 false positive S. marcescens in initial testing. $3 / 3$ repeats negative.   
4) 1/3 positive control check failure in initial testing. 1/3 false positive $S .$ . marcescens in repeat testing.   
5) 1/3 false positive S. marcescens in initial testing. 3/3 repeats negative.   
6) 1/3 false positive $E$ . coli in initial testing. $3 / 3$ repeats negative.   
7) 3/3 false positive $K .$ . pneumoniae in initial testing. See limitation.   
8) 1/3 false positive S. marcescens in initial testing. 2/2 repeats negative.   
9) 1/3 false positive S. marcescens in initial testing. 9/9 repeats negative.   
10) 1/3 false positive S. marcescens in initial testing. 1/10 false positive S. marcescens in repeat testing. See   
limitation.   
11) 1/3 false positive S. marcescens in initial testing. $3 / 3$ repeats negative.   
12) 1/3 false positive S. marcescens in initial testing. 3/3 repeats negative.   
13) 1/3 false positive $E$ . coli in initial testing. 3/3 repeats negative.

# 6. Interfering Substances:

iC-GN Assay performance was evaluated in the presence of potentially inhibiting substances that may be encountered in blood and blood culture media. Eight representative target organisms plus one non-target organism were evaluated. Organisms were tested at the lowest levels of bottle positivity, considered within two hours of bottle “ring.” Potential interferents were tested at concentrations exceeding the highest concentrations that may be encountered in blood and blood culture media (for concentrations tested see 510(k) Summary for K190341, Table 13 (p.24-25)). Target performance is based on all expected targets detected and no false positive targets detected. Non-target performance is based on all negative results. In the event of a false negative result, the organism/interferent combination was retested in replicates of ten (10). In the event of a false positive result or other failure, the organism/interferent combination was retested in triplicate. If the discordant result was observed in repeat testing, the combination was retested at a decreased inhibitor concentration. below. Interference testing was performed in BD BACTEC Plus Aerobic blood culture bottle media, which has a sodium polyanetholesulfonate (SPS) concentration of $0 . 0 5 \% \mathrm { w } / \mathrm { v }$ . Additional SPS at a concentration greater than $0 . 0 5 \% \mathrm { w } / \mathrm { v }$ was found to interfere with the performance of some iC-GN Assay targets, resulting in increased false negative results and positive control check failures. Interference results are included in the table below. It was determined that the risks associated with the interfering substance, SPS, could be mitigated via labeling. The information regarding interference associated with SPS is provided for the end user in the Package Insert.

<table><tr><td rowspan=1 colspan=3>Table 7: Interfering Substances Test Panel</td></tr><tr><td rowspan=1 colspan=1>Interference Compound</td><td rowspan=1 colspan=1>linically RelevantConcentration</td><td rowspan=1 colspan=1>TestConcentration</td></tr><tr><td rowspan=1 colspan=1>Hemoglobin</td><td rowspan=1 colspan=1>1-2 g/L</td><td rowspan=1 colspan=1>10 g/L</td></tr><tr><td rowspan=1 colspan=1>Conjugated Bilirubin</td><td rowspan=1 colspan=1>0.1-0.4 mg/dL</td><td rowspan=1 colspan=1>10 mg/dL</td></tr><tr><td rowspan=1 colspan=1>Unconjugated Bilirubin</td><td rowspan=1 colspan=1>0.1-0.8 mg/dL</td><td rowspan=1 colspan=1>10 mg/dL</td></tr><tr><td rowspan=1 colspan=1>Protein (γ-globulin + albumin)</td><td rowspan=1 colspan=1>0.7-1.7 g/dL</td><td rowspan=1 colspan=1>4 g/dL</td></tr><tr><td rowspan=1 colspan=1>Triglyceride</td><td rowspan=1 colspan=1>300-500 mg/dL</td><td rowspan=1 colspan=1>1500 mg/dL</td></tr><tr><td rowspan=1 colspan=1>Human Genomic DNA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>1 × 106 cells/mL</td></tr><tr><td rowspan=1 colspan=1>Sodium Polyanetholesulfonate (SPS)</td><td rowspan=1 colspan=1>0.02-0.05% w/v</td><td rowspan=1 colspan=1>0.1% w/v</td></tr><tr><td rowspan=1 colspan=1>Cefepime</td><td rowspan=1 colspan=1>16 μg/mL</td><td rowspan=1 colspan=1>80 μg/mL</td></tr><tr><td rowspan=1 colspan=1>Ceftriaxone</td><td rowspan=1 colspan=1>16 μg/mL</td><td rowspan=1 colspan=1>80 μg/mL</td></tr><tr><td rowspan=1 colspan=1>Fluconazole</td><td rowspan=1 colspan=1>25 μg/mL</td><td rowspan=1 colspan=1>100 μg/mL</td></tr><tr><td rowspan=1 colspan=1>Gentamicin</td><td rowspan=1 colspan=1>20 μg/mL</td><td rowspan=1 colspan=1>80 μg/mL</td></tr><tr><td rowspan=1 colspan=1>Meropenem</td><td rowspan=1 colspan=1>16 μg/mL</td><td rowspan=1 colspan=1>80 μg/mL</td></tr><tr><td rowspan=1 colspan=1>Piperacillin</td><td rowspan=1 colspan=1>32 μg/mL</td><td rowspan=1 colspan=1>160 μg/mL</td></tr><tr><td rowspan=1 colspan=1>Vancomycin</td><td rowspan=1 colspan=1>20 μg/mL</td><td rowspan=1 colspan=1>100 μg/mL</td></tr></table>

<table><tr><td colspan="13" rowspan="1">Table 7a: iC-GN Assay Interfering Substances Performance</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="12" rowspan="1">Target Performance</td></tr><tr><td colspan="1" rowspan="1">InterferenceCompound</td><td colspan="1" rowspan="1">ABX</td><td colspan="1" rowspan="1">ECX</td><td colspan="1" rowspan="1">EC</td><td colspan="1" rowspan="1">KO</td><td colspan="1" rowspan="1">KPN</td><td colspan="1" rowspan="1">PM</td><td colspan="1" rowspan="1">PA</td><td colspan="1" rowspan="1">SM</td><td colspan="1" rowspan="1">SE</td><td colspan="1" rowspan="1">KPC-2</td><td colspan="1" rowspan="1">NDM-1</td><td colspan="1" rowspan="1">CTX-M-15</td></tr><tr><td colspan="1" rowspan="1">Hemoglobin</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">14/14</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">5/5</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">14/14</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">ConjugatedBilirubin</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">UnconjugatedBilirubin</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Protein (γ-globulin+ albumin)</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Triglyceride</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Human GenomicDNA</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">SPS (0.1%)</td><td colspan="1" rowspan="1">3/121</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">2/122</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">8/123</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">SPS (0.5%)</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">--</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">--</td><td colspan="1" rowspan="1">--</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">--</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Cefepime</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Ceftriaxone</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Fluconazole</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Gentamicin</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Meropenem</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">5/5</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">5/64</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">5/5</td></tr><tr><td colspan="1" rowspan="1">Piperacillin</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr></table>

# 7. Microbial Interference:

Potential microbial interference was evaluated by testing high concentrations of Gramnegative exclusivity organisms in combination with low concentrations of iC-GN target organisms. A total of sixty (60) Gram-negative exclusivity strains were tested at the highest possible concentrations, considered eight hours beyond initial bottle positivity or the equivalent. Eight (8) representative iC-GN target organisms were tested at concentrations below the lowest levels of bottle positivity. Each organism combination was tested in triplicate. Performance was based on all expected targets detected and no false positive targets detected. In the event of a false negative result, the combination was retested in replicates of ten (10). In the event of a false positive result or other failure, the combination was retested in replicates of three (3) or ten (10). Microbial interference results are presented in the table below. A small number of organisms initially were observed to cause false negative results, however additional testing showed that the initial results were not reproducible. It was determined that the risks associated with the microbial interference could be mitigated via labeling. The organisms that showed an increased risk of microbial interference are listed in the Limitations section of the Package Insert.

<table><tr><td colspan="12" rowspan="1">Table 8: iC-GN Assay Microbial Interference Results</td></tr><tr><td colspan="1" rowspan="1">Organism</td><td colspan="1" rowspan="1">ABX</td><td colspan="1" rowspan="1">ECX</td><td colspan="1" rowspan="1">EC</td><td colspan="1" rowspan="1">KO</td><td colspan="1" rowspan="1">KPN</td><td colspan="1" rowspan="1">PM</td><td colspan="1" rowspan="1">PA</td><td colspan="1" rowspan="1">SM</td><td colspan="1" rowspan="1">CTX-M-15</td><td colspan="1" rowspan="1">KPC-2</td><td colspan="1" rowspan="1">NDM-1</td></tr><tr><td colspan="1" rowspan="1">A. lwoffi</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">A. radioresistens</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">A. schindleri</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">A. ursingii</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">A. hydrophila</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">A. faecalis</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">B. fragilis</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">4/51</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">B. vesicularis</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">B. cepacia</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">C. coli</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">C. jejuni</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">C. davisae</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">C. amalonaticus</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">C. braakii</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">C. freundii</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">2/2</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">C. koseri</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">C. sedlakii</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">C. muytjensii</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">C. sakazakii</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">E. tarda</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">5/5</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">5/5</td></tr><tr><td colspan="1" rowspan="1">E. aerogenes</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">E. amnigenus</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">5/5</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">5/5</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">E. fergusonii</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">E. hermannii</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">5/5</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">13/13</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">5/5</td><td colspan="1" rowspan="1">12/132</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">E. vulneris</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">F. varium</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">H. alvei</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">H. influenzae</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">H.parainfluenzae</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">K. ascorbata</td><td colspan="1" rowspan="1">10/133</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">11/124</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">10/124</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">L. adecarboxylata</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">L. grimontii</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">M. catarrhalis</td><td colspan="1" rowspan="1">12/135</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">M. morganii</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">5/5</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">5/5</td></tr><tr><td colspan="1" rowspan="1">N. gonorrhoeae</td><td colspan="1" rowspan="1">5/5</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">N. lactamica</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">5/5</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">5/5</td></tr><tr><td colspan="1" rowspan="1">N. meningitidis</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">5/5</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">5/5</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">N. mucosa</td><td colspan="1" rowspan="1">5/5</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">5/5</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">5/5</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">N. sicca</td><td colspan="1" rowspan="1">5/66</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">5/5</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">P. agglomerans</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">P. multocida</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">P. gergoviae</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">P. alcalifaciens</td><td colspan="1" rowspan="1">5/5</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">P. rettgeri</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">P. stuartii</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">P. fluorescens</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">P. luteola</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">P. mendocina</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">5/5</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">P. nitroreducens</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">P. oryzihabitans</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">5/5</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">5/5</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">P. putida</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">5/5</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">5/5</td></tr><tr><td colspan="1" rowspan="1">P. stutzeri</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">5/5</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">5/5</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">5/5</td></tr><tr><td colspan="1" rowspan="1">R. planitcola</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">S. enterica</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">S. fonticola</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">S. liquefaciens</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">5/5</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">12/137</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">13/13</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">S. odorifera</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">12/138</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">S. rubidaea</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">S. maltophilia</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">V. parvula</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr></table>

1) 1/3 array registration error in initial testing. 1/3 false positive $E$ . cloacae complex in repeat testing.   
2) 1/3 false negative KPC in initial testing. 10/10 repeats passed.   
3) 3/3 false negative A. baumannii complex in initial testing; concentration was determined to be below the   
target limit of detection. 10/10 repeats passed.   
4) 1/2 false negative CTX‐M in initial testing. 1/10 false positive $K .$ . pneumoniae in repeat testing.   
5) 1/3 false negative A. baumannii complex in initial testing. 10/10 repeats passed.   
6) 1/3 false positive $E .$ . coli in initial testing. 3/3 repeats passed.   
7) 1/3 false negative K. pneumoniae in initial testing. 10/10 repeats passed.   
8) 1/3 false positive S. marcescens in initial testing. 10/10 repeats passed.

# 8. Competitive Inhibition:

iC-GN Assay performance was evaluated with combinations of target analytes that may be found in mixed positive blood cultures. One target organism was prepared at the lowest level of bottle positivity, considered within two hours of bottle “ring”, while the second target organism was prepared at the highest possible concentration, considered eight hours after initial bottle positivity. All organisms were grown in BD BACTEC Plus Aerobic blood cultures bottles with human blood added on the BD BACTEC System. The organisms were combined at a ratio of one part “low” to four parts “high”. Each low concentration organism was tested in combination with each high concentration organism in triplicate. Performance was based on all expected targets detected. In the event of a false negative result, the organism combination was retested in replicates of ten (10) at the same “low” and “high” organism ratio. In the event of a reproducible false negative result, the organism combination was retested in replicates of ten (10) at a ratio of one part “low” to one part “high.” All high concentration iC-GN targets were detected. Due to competitive inhibition, low concentration targets were not detected in $1 . 7 \%$ of tests (3/178). When iC-GN target organisms were present at similar concentrations, all targets were detected. Competitive inhibition results are included in the table below. Based on the results of the competitive inhibition study, the following limitation was included in the Package Insert, “Due to competitive inhibition, target organisms present at low concentrations may not be detected by the iC-GN Assay when a second target organism is present at higher concentrations in a mixed culture”.

<table><tr><td colspan="2">Table 9: iC-GN Assay Competitive Inhibition Performance</td></tr><tr><td></td><td>Target Performance</td></tr></table>

<table><tr><td colspan="1" rowspan="1">LowOrganism</td><td colspan="1" rowspan="1">High Organism</td><td colspan="1" rowspan="1">LowOrganism</td><td colspan="1" rowspan="1">LowMarker</td><td colspan="1" rowspan="1">HighOrganism</td><td colspan="1" rowspan="1">HighMarker</td></tr><tr><td colspan="1" rowspan="7">A. baumannii</td><td colspan="1" rowspan="1">ECX</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">NA</td></tr><tr><td colspan="1" rowspan="1">EC (CTX-M-15+)</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">KO</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">NA</td></tr><tr><td colspan="1" rowspan="1">KPN (KPC-2+)</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">PM</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">NA</td></tr><tr><td colspan="1" rowspan="1">PA (NDM-1+)</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">SM</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">NA</td></tr><tr><td colspan="1" rowspan="7">E. cloacae</td><td colspan="1" rowspan="1">ABX</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">1 FP KPC</td></tr><tr><td colspan="1" rowspan="1">EC (CTX-M-15+)</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">KO</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">NA</td></tr><tr><td colspan="1" rowspan="1">KPN (KPC-2+)</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">PM</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">NA</td></tr><tr><td colspan="1" rowspan="1">PA (NDM-1+)</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">SM</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">NA</td></tr><tr><td colspan="1" rowspan="7">E. coli(CTX-M-15+)</td><td colspan="1" rowspan="1">ABX</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">NA</td></tr><tr><td colspan="1" rowspan="1">ECX</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">NA</td></tr><tr><td colspan="1" rowspan="1">KO</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">NA</td></tr><tr><td colspan="1" rowspan="1">KPN (KPC-2+)</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">PM</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">NA</td></tr><tr><td colspan="1" rowspan="1">PA (NDM-1+)</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">SM</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">NA</td></tr><tr><td colspan="1" rowspan="7">K. oxytoca</td><td colspan="1" rowspan="1">ABX</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">NA</td></tr><tr><td colspan="1" rowspan="1">ECX</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">NA</td></tr><tr><td colspan="1" rowspan="1">EC (CTX-M-15+)</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">KPN (KPC-2+)</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">PM</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">NA</td></tr><tr><td colspan="1" rowspan="1">PA (NDM-1+)</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">SM</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">NA</td></tr><tr><td colspan="1" rowspan="7">K.pneumoniae(KPC-2+)</td><td colspan="1" rowspan="1">ABX</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">NA</td></tr><tr><td colspan="1" rowspan="1">ECX</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">NA</td></tr><tr><td colspan="1" rowspan="1">EC (CTX-M-15+)</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">KO</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">NA</td></tr><tr><td colspan="1" rowspan="1">PM</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">NA</td></tr><tr><td colspan="1" rowspan="1">PA (NDM-1+)</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">SM</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">NA</td></tr><tr><td colspan="1" rowspan="7">P. mirabilis</td><td colspan="1" rowspan="1">ABX</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">NA</td></tr><tr><td colspan="1" rowspan="1">ECX</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">NA</td></tr><tr><td colspan="1" rowspan="1">EC (CTX-M-15+)</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">KO</td><td colspan="1" rowspan="1">5/5</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">5/5</td><td colspan="1" rowspan="1">NA</td></tr><tr><td colspan="1" rowspan="1">KPN (KPC-2+)</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">PA (NDM-1+)</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">SM</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">NA</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">ABX</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">NA</td></tr><tr><td colspan="1" rowspan="7">P. aeruginosa(NDM-1+)</td><td colspan="1" rowspan="1">ECX</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">NA</td></tr><tr><td colspan="1" rowspan="1">EC (CTX-M-15+)</td><td colspan="1" rowspan="1">12/13</td><td colspan="1" rowspan="1">11/13</td><td colspan="1" rowspan="1">13/13</td><td colspan="1" rowspan="1">13/13</td></tr><tr><td colspan="1" rowspan="1">EC(CTX-M-15+) 1:1</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">10/10</td></tr><tr><td colspan="1" rowspan="1">KO</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">NA</td></tr><tr><td colspan="1" rowspan="1">KPN (KPC-2+)</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">PM</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">NA</td></tr><tr><td colspan="1" rowspan="1">SM</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">NA</td></tr><tr><td colspan="1" rowspan="7">S. marcescens</td><td colspan="1" rowspan="1">KPN (KPC-2+)</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">PM</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">NA</td></tr><tr><td colspan="1" rowspan="1">PA (NDM-1+)</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">ABX</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">NA</td></tr><tr><td colspan="1" rowspan="1">ECX</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">NA</td></tr><tr><td colspan="1" rowspan="1">EC (CTX-M-15+)</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">KO</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">NA</td></tr></table>

9. Reproducibility:

To confirm the site-to-site, operator-to-operator, system-to-system, and lot-to-lot reproducibility of the iC-GN Assay, a representative panel of target organisms and one non-target organism were evaluated at two clinically relevant concentrations: initial bottle positivity and eight hours beyond initial bottle positivity. Organisms were grown to the appropriate concentrations in BD BACTEC Plus Aerobic blood culture bottles with human blood added on the BD BACTEC System. Testing was performed by two independent operators at each of three sites, two external and one internal. Each operator tested the eighteen-organism panel in triplicate across five, non-consecutive days. Testing was performed on six iC-GN Cassette lots and multiple iC-Systems. Performance is based on all expected targets detected and no false positive targets detected. The table below summarizes the reproducibility results stratified by iC-GN target and concentration. Overall Reproducibility performance was $9 9 . 3 \%$ , confirming that iC-GN Assay performance is reproducible across sites, operators, systems and lots.

<table><tr><td colspan="7" rowspan="1">Table 10 : iC-GN Assay Reproducibility Performance by Target</td></tr><tr><td colspan="1" rowspan="1">Target/Concentration</td><td colspan="1" rowspan="1">OverallPerformance</td><td colspan="1" rowspan="1">OverallPerformance %[95% CI]</td><td colspan="1" rowspan="1">FalseNegatives</td><td colspan="1" rowspan="1">FalsePositives</td><td colspan="1" rowspan="1">PC Check Failures</td><td colspan="1" rowspan="1">SystemFaailures</td></tr><tr><td colspan="1" rowspan="1">A. baumanniicomplexBottle Ring</td><td colspan="1" rowspan="1">90/90</td><td colspan="1" rowspan="1">100.0[95.91-100.0]</td><td colspan="1" rowspan="1">0/90(0.00%)</td><td colspan="1" rowspan="1">0/90(0.00%)</td><td colspan="1" rowspan="1">0/90(0.00%)</td><td colspan="1" rowspan="1">0/90(0.00%)</td></tr><tr><td colspan="1" rowspan="1">A. baumanniicomplexBottle Ring + 8hours</td><td colspan="1" rowspan="1">87/90</td><td colspan="1" rowspan="1">96.7[90.65-98.86]</td><td colspan="1" rowspan="1">0/90(0.00%)</td><td colspan="1" rowspan="1">3/90(3.33%)</td><td colspan="1" rowspan="1">0/90(0.00%)</td><td colspan="1" rowspan="1">0/90(0.00%)</td></tr><tr><td colspan="1" rowspan="1">E. cloacae complexBottle Ring</td><td colspan="1" rowspan="1">86/88</td><td colspan="1" rowspan="1">97.7[92.09-99.37]</td><td colspan="1" rowspan="1">1/88(1.14%)</td><td colspan="1" rowspan="1">1/88(1.14%)</td><td colspan="1" rowspan="1">2/90(2.22%)</td><td colspan="1" rowspan="1">0/90(0.00%)</td></tr><tr><td colspan="1" rowspan="1">E. cloacae complexBottle Ring + 8hours</td><td colspan="1" rowspan="1">90/90</td><td colspan="1" rowspan="1">100.0[95.91-100.0]</td><td colspan="1" rowspan="1">0/90(0.00%)</td><td colspan="1" rowspan="1">0/90(0.00%)</td><td colspan="1" rowspan="1">0/90(0.00%)</td><td colspan="1" rowspan="1">0/90(0.00%)</td></tr><tr><td colspan="1" rowspan="1">E. coliBottlee Ring</td><td colspan="1" rowspan="1">90/90</td><td colspan="1" rowspan="1">100.0[95.91-100.0]</td><td colspan="1" rowspan="1">0/90(0.00%)</td><td colspan="1" rowspan="1">0/90(0.00%)</td><td colspan="1" rowspan="1">0/90(0.00%)</td><td colspan="1" rowspan="1">0/90(0.00%)</td></tr><tr><td colspan="1" rowspan="1">E. coliBottle Ring + 8hours</td><td colspan="1" rowspan="1">89/89</td><td colspan="1" rowspan="1">100.0[95.86-100.0]</td><td colspan="1" rowspan="1">0/89(0.00%)</td><td colspan="1" rowspan="1">0/89(0.00%)</td><td colspan="1" rowspan="1">0/90(0.00%)</td><td colspan="1" rowspan="1">1/90(1.11%)</td></tr><tr><td colspan="1" rowspan="1">K. oxytoca Btle Ring</td><td colspan="1" rowspan="1">89/90</td><td colspan="1" rowspan="1">98.9[93.97-99.80]</td><td colspan="1" rowspan="1">0/90(0.00%)</td><td colspan="1" rowspan="1">1/90(1.11%)</td><td colspan="1" rowspan="1">0/90(0.00%)</td><td colspan="1" rowspan="1">0/90(0.00%)</td></tr><tr><td colspan="1" rowspan="1">K. oxytocaBottle Ring + 8hours</td><td colspan="1" rowspan="1">89/89</td><td colspan="1" rowspan="1">100.0[95.86-100.0]</td><td colspan="1" rowspan="1">0/89(0.00%)</td><td colspan="1" rowspan="1">0/89(0.00%)</td><td colspan="1" rowspan="1">1/90(1.11%)</td><td colspan="1" rowspan="1">0/90(0.00%)</td></tr><tr><td colspan="1" rowspan="1">K. pneumoniaeBo tle Ring</td><td colspan="1" rowspan="1">90/90</td><td colspan="1" rowspan="1">100.0[95.91-100.0]</td><td colspan="1" rowspan="1">0/90(0.00%)</td><td colspan="1" rowspan="1">0/90(0.00%)</td><td colspan="1" rowspan="1">0/90(0.00%)</td><td colspan="1" rowspan="1">0/90(0.00%)</td></tr><tr><td colspan="1" rowspan="1">K. pneumoniaeBottle Ring + 8hours</td><td colspan="1" rowspan="1">89/89</td><td colspan="1" rowspan="1">100.0[95.86-100.0]</td><td colspan="1" rowspan="1">0/89(0.00%)</td><td colspan="1" rowspan="1">0/89(0.00%)</td><td colspan="1" rowspan="1">1/90(1.11%)</td><td colspan="1" rowspan="1">0/90(0.00%)</td></tr><tr><td colspan="1" rowspan="1">Proteus speciesBottle Ring</td><td colspan="1" rowspan="1">89/89</td><td colspan="1" rowspan="1">100.0[95.86-100.0]</td><td colspan="1" rowspan="1">0/89(0.00%)</td><td colspan="1" rowspan="1">0/89(0.00%)</td><td colspan="1" rowspan="1">1/90(1.11%)</td><td colspan="1" rowspan="1">0/90(0.00%)</td></tr><tr><td colspan="1" rowspan="1">Proteus speciesBottle Ring + 8hours</td><td colspan="1" rowspan="1">88/88</td><td colspan="1" rowspan="1">100.0[95.92-100.0]</td><td colspan="1" rowspan="1">0/88(0.00%)</td><td colspan="1" rowspan="1">0/88(0.00%)</td><td colspan="1" rowspan="1">0/90(0.00%)</td><td colspan="1" rowspan="1">2/90(2.22%)</td></tr><tr><td colspan="1" rowspan="1">P. aeruginosaBottle Ring</td><td colspan="1" rowspan="1">88/89</td><td colspan="1" rowspan="1">98.9[93.91-99.80]</td><td colspan="1" rowspan="1">1/89(1.12%)</td><td colspan="1" rowspan="1">0/89(0.00%)</td><td colspan="1" rowspan="1">1/90(1.11%)</td><td colspan="1" rowspan="1">0/90(0.00%)</td></tr><tr><td colspan="1" rowspan="1">P. aeruginosaBottle Ring + 8hours</td><td colspan="1" rowspan="1">89/90</td><td colspan="1" rowspan="1">98.9[93.97-99.80]</td><td colspan="1" rowspan="1">1/90(1.11%)</td><td colspan="1" rowspan="1">0/90(0.00%)</td><td colspan="1" rowspan="1">0/90(0.00%)</td><td colspan="1" rowspan="1">0/90(0.00%)</td></tr><tr><td colspan="1" rowspan="1">S. marcescensBottle Ring</td><td colspan="1" rowspan="1">87/89</td><td colspan="1" rowspan="1">97.8[92.17-99.38]</td><td colspan="1" rowspan="1">0/89(0.00%)</td><td colspan="1" rowspan="1">2/89(2.25%)</td><td colspan="1" rowspan="1">1/90(1.11%)</td><td colspan="1" rowspan="1">0/90(0.00%)</td></tr><tr><td colspan="1" rowspan="1">S. marcescensBottle Ring + 8 hours</td><td colspan="1" rowspan="1">87/89</td><td colspan="1" rowspan="1">97.8[92.17-99.38]</td><td colspan="1" rowspan="1">0/89(0.00%)</td><td colspan="1" rowspan="1">2/89(2.25%)</td><td colspan="1" rowspan="1">1/90(1.11%)</td><td colspan="1" rowspan="1">0/90(0.00%)</td></tr><tr><td colspan="1" rowspan="1">Target/Concentration</td><td colspan="1" rowspan="1">OverallPerformance</td><td colspan="1" rowspan="1">OverallPerformance %[95% CI]</td><td colspan="1" rowspan="1">FalseNegatives</td><td colspan="1" rowspan="1">FalsePositives</td><td colspan="1" rowspan="1">PC CheckFailures</td><td colspan="1" rowspan="1">SystemFailures</td></tr><tr><td colspan="1" rowspan="1">CTX-M group 1Bottle Ring</td><td colspan="1" rowspan="1">90/90</td><td colspan="1" rowspan="1">100.0[95.91-100.0]</td><td colspan="1" rowspan="1">0/90(0.00%)</td><td colspan="1" rowspan="1">0/90(0.00%)</td><td colspan="1" rowspan="1">0/90(0.00%)</td><td colspan="1" rowspan="1">0/90(0.00%)</td></tr><tr><td colspan="1" rowspan="1">CTX-M group 1Bottle Ring + 8 hours</td><td colspan="1" rowspan="1">89/89</td><td colspan="1" rowspan="1">100.0[95.86-100.0]</td><td colspan="1" rowspan="1">0/89(0.00%)</td><td colspan="1" rowspan="1">0/89(0.00%)</td><td colspan="1" rowspan="1">0/90(0.00%)</td><td colspan="1" rowspan="1">1/90(1.11%)</td></tr><tr><td colspan="1" rowspan="1">KPCBottle Ring</td><td colspan="1" rowspan="1">90/90</td><td colspan="1" rowspan="1">100.0[95.91-100.0]</td><td colspan="1" rowspan="1">0/90(0.00%)</td><td colspan="1" rowspan="1">0/90(0.00%)</td><td colspan="1" rowspan="1">0/90(0.00%)</td><td colspan="1" rowspan="1">0/90(0.00%)</td></tr><tr><td colspan="1" rowspan="1">KPCBottle Ring + 8 hours</td><td colspan="1" rowspan="1">89/89</td><td colspan="1" rowspan="1">100.0[95.86-100.0]</td><td colspan="1" rowspan="1">0/89(0.00%)</td><td colspan="1" rowspan="1">0/89(0.00%)</td><td colspan="1" rowspan="1">1/90(1.11%)</td><td colspan="1" rowspan="1">0/90(0.00%)</td></tr><tr><td colspan="1" rowspan="1">NDMBottle Ring</td><td colspan="1" rowspan="1">89/89</td><td colspan="1" rowspan="1">100.0[95.86-100.0]</td><td colspan="1" rowspan="1">0/89(0.00%)</td><td colspan="1" rowspan="1">0/89(0.00%)</td><td colspan="1" rowspan="1">1/90(1.11%)</td><td colspan="1" rowspan="1">0/90(0.00%)</td></tr><tr><td colspan="1" rowspan="1">NDMBottle Ring + 8 hours</td><td colspan="1" rowspan="1">89/90</td><td colspan="1" rowspan="1">98.9[93.97-99.80]</td><td colspan="1" rowspan="1">1/90(1.11%)</td><td colspan="1" rowspan="1">0/90(0.00%)</td><td colspan="1" rowspan="1">0/90(0.00%)</td><td colspan="1" rowspan="1">0/90(0.00%)</td></tr></table>

# B. Comparison Studies:

1. Method Comparison with the Predicate Device:

A method comparison study was performed at five (5) geographically dispersed clinical sites. Sites tested 1002 leftover de-identified specimens from anaerobic and aerobic blood culture bottles flagged as positive by their respective continuous monitoring blood culture system. Three of the commonly used blood culture systems were included in the study: Thermo Fisher VersaTREK, BD BACTEC and BioMerieux BacT/ALERT.

Patient positive blood cultures confirmed by Gram stain to be positive for Gram-negative bacilli were enrolled in the study. Any positive blood cultures showing an initial mixed Gram stain were not enrolled or were subsequently withdrawn from the study dataset.

Final performance of the iC-GN Assay organism targets was compared to reference culture followed by MALDI identification per the study protocol. Final performance of the iC-GN Assay resistance marker targets was compared to PCR amplification followed by confirmatory bi-directional sequencing. Phenotypic antimicrobial susceptibility testing (AST) was also performed on all specimens to identify additional samples which required sequencing. Discordant samples were also sequenced.

To supplement performance of observed lower prevalence organisms, 170 contrived samples were prepared using verified strains. Contrived samples were prepared at iCubate using BD BACTEC Plus Aerobic Blood Culture Bottles with $1 0 ~ \mathrm { m L }$ of human blood added (in accordance with BACTEC instructions). Organisms were spiked into bottles at concentrations of 5-30 CFU/bottle and incubated until bottles were flagged as positive. Aliquots of samples were frozen and provided to the sites (frozen) for testing.

Of the 1107 positive blood culture specimens enrolled in the study, a total of 105 specimens were excluded/withdrawn from the study and excluded from all subsequent performance analyses. Of the 1002 specimens remaining, 976 were fresh prospective specimens and 26 $( 2 . 6 \% )$ were frozen prospective specimens. Nineteen (19) samples were excluded from the performance analysis for Proteus mirabilis due to confirmed contamination of BD BACTEC Bottles with non-viable Proteus organisms or nucleic acid, leaving a total of 983 evaluable specimens for Proteus mirabilis.

The total specimens excluded from the iC-GN Assay Method Comparison Study $\mathrm { ( n { = } 1 0 5 }$ ) are listed by site and the reasons for exclusion are noted in the 510(k) Summary for K190341, Table 12 (p.25). The most common reasons for exclusion included incomplete reference testing and repeat iC-GN errors.

Throughout the course of the study, an initial error rate of $2 . 9 \%$ (34/1181) was observed. Reasons for error included the following: Positive controls check failure (27), Array registration error (6), and Processor/System error $( l )$ . When an error was observed, repeat testing was performed with the iC-GN Assay per the protocol. Upon repeat testing, the error rate was reduced to $0 . 8 \%$ (9/1181). The 510(k) Summary for K190341, Table 13 (p.26) includes a table of the no-call results.

When performance of the iC-GN Assay was compared to reference culture followed by MALDI identification or PCR/bi-directional sequencing, there was no significant difference in performance noted between the five study sites or between the three blood culture systems. Performance for all positive bottle types/systems combined is presented in the tables below for detection of the iC-GN Assay targets as compared to culture and MALDI or PCR/bi-directional sequencing. Results are stratified by prospectively tested fresh specimens, prospectively collected/retrospectively tested frozen specimens and contrived specimens. These data are provided in the tables below.

<table><tr><td rowspan=1 colspan=5>Table 11a: iC-GN Assay Performance: Acinetobacterbaumannii complex (ppa)</td></tr><tr><td rowspan=2 colspan=2>SpecimenType</td><td rowspan=2 colspan=1>N=</td><td rowspan=1 colspan=2>Percent Agreement</td></tr><tr><td rowspan=1 colspan=1>Positive(95% CI)</td><td rowspan=4 colspan=1>Culture &amp;MALDI</td></tr><tr><td rowspan=3 colspan=1>Jroecsd</td><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>976</td><td rowspan=1 colspan=1>100%7/7(64.6-100)</td><td rowspan=1 colspan=1>99.9%968-969**(99.4-100)</td></tr><tr><td rowspan=1 colspan=1>Frozen</td><td rowspan=1 colspan=1>26</td><td rowspan=1 colspan=1>-0/0-</td><td rowspan=1 colspan=1>100%26/26(87.1-100)</td></tr><tr><td rowspan=1 colspan=1>TOTAL</td><td rowspan=1 colspan=1>1002</td><td rowspan=1 colspan=1>100%7/7(64.6-100)</td><td rowspan=1 colspan=1>99.9%994/995(99.4-100)</td></tr><tr><td rowspan=1 colspan=2>Contrived</td><td rowspan=1 colspan=1>170</td><td rowspan=1 colspan=1>100%45/45(92.1-100)</td><td rowspan=1 colspan=1>100%125/125(97.0-100)</td></tr></table>

$^ { * * } 1 / 1$ false positive observed was negative for A. baumannii complex by PCR/bi‐directional sequencing

<table><tr><td rowspan=1 colspan=5>Table 11b: iC-GN Assay Performance: Enterobactercloacae complex (ramA)</td></tr><tr><td rowspan=2 colspan=2>SpecimenType</td><td rowspan=2 colspan=1>N=</td><td rowspan=1 colspan=1>Percent Agreement</td><td rowspan=1 colspan=1>ComparatorMethod</td></tr><tr><td rowspan=1 colspan=1>Positive(95% CI)</td><td rowspan=4 colspan=1>Culture &amp;MALDI</td></tr><tr><td rowspan=3 colspan=1>Jrotd</td><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>976</td><td rowspan=1 colspan=1>94.5%52/55*(85.1-98.1)</td><td rowspan=1 colspan=1>100%921/921(99.6-100)</td></tr><tr><td rowspan=1 colspan=1>Frozen</td><td rowspan=1 colspan=1>26</td><td rowspan=1 colspan=1>100%5/5(56.6-100)</td><td rowspan=1 colspan=1>100%21/21(84.5-100)</td></tr><tr><td rowspan=1 colspan=1>TOTAL</td><td rowspan=1 colspan=1>1002</td><td rowspan=1 colspan=1>95.0%57/60(86.3-98.3)</td><td rowspan=1 colspan=1>100%942/942(99.6-100)</td></tr><tr><td rowspan=1 colspan=2>Contrived</td><td rowspan=1 colspan=1>170</td><td rowspan=1 colspan=1>100%17/17(81.6-100)</td><td rowspan=1 colspan=1>100%153/153(97.6-100)</td></tr></table>

$^ { * } 1 / 3$ false negatives observed was negative for $E$ . cloacae complex by PCR/bi‐directional sequencing; 2/3 were positive for $E$ . cloacae complex by PCR/bi‐directional sequencing

<table><tr><td rowspan=1 colspan=5>Table 11c: iC-GN Assay Performance: Escherichia coli(uidA)</td></tr><tr><td rowspan=2 colspan=2>SpecimenType</td><td rowspan=2 colspan=1>N=</td><td rowspan=1 colspan=1>Percent Agreement</td><td rowspan=1 colspan=1>ComparatorMethod</td></tr><tr><td rowspan=1 colspan=1>Positive(95% CI)</td><td rowspan=4 colspan=1>Culture &amp;MALDI</td></tr><tr><td rowspan=3 colspan=1>Jroecsd</td><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>976</td><td rowspan=1 colspan=1>98.4%480/488*(96.8-99.2)</td><td rowspan=1 colspan=1>100%488/488(99.2-100)</td></tr><tr><td rowspan=1 colspan=1>Frozen</td><td rowspan=1 colspan=1>26</td><td rowspan=1 colspan=1>100%6/6(61.0-100)</td><td rowspan=1 colspan=1>100%20/20(83.9-100)</td></tr><tr><td rowspan=1 colspan=1>TOTAL</td><td rowspan=1 colspan=1>1002</td><td rowspan=1 colspan=1>98.4%486/494(96.8-99.2)</td><td rowspan=1 colspan=1>100%508/508(99.2-100)</td></tr><tr><td rowspan=1 colspan=2>Contrived</td><td rowspan=1 colspan=1>170</td><td rowspan=1 colspan=1>100%15/15(79.6-100)</td><td rowspan=1 colspan=1>100%155/155(97.6-100)</td></tr></table>

$^ { * } 4 / 8$ false negatives observed were negative for E. coli by PCR/bi‐ directional sequencing; 3/8 were positive for E. coli by PCR/bi‐ directional sequencing; 1/8 was not available for sequencing

<table><tr><td rowspan=1 colspan=5>Table 11d: iC-GN Assay Performance: Klebsiella oxytoca(pehX)</td></tr><tr><td rowspan=2 colspan=2>SpecimenType</td><td rowspan=2 colspan=1>N=</td><td rowspan=1 colspan=1>Percent Agreement</td><td rowspan=1 colspan=1>ComparatorMethod</td></tr><tr><td rowspan=1 colspan=1>Positive(95% CI)</td><td rowspan=4 colspan=1>Culture &amp;MALDI</td></tr><tr><td rowspan=3 colspan=1>Jrod</td><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>976</td><td rowspan=1 colspan=1>95.8%23/24*(79.8-99.3)</td><td rowspan=1 colspan=1>99.7%949/952**(99.1-99.9)</td></tr><tr><td rowspan=1 colspan=1>Frozen</td><td rowspan=1 colspan=1>26</td><td rowspan=1 colspan=1>-0/0-</td><td rowspan=1 colspan=1>100%26/26(87.1-100)</td></tr><tr><td rowspan=1 colspan=1>TOTAL</td><td rowspan=1 colspan=1>1002</td><td rowspan=1 colspan=1>95.8%23/24(79.8-99.3)</td><td rowspan=1 colspan=1>99.7%975/978(99.1-99.9)</td></tr><tr><td rowspan=1 colspan=2>Contrived</td><td rowspan=1 colspan=1>170</td><td rowspan=1 colspan=1>100%30/30(88.6-100)</td><td rowspan=1 colspan=1>100%140/140(97.3-100)</td></tr></table>

$^ { * } 1 / 1$ false negative observed was negative for $K .$ . oxytoca by PCR/bi‐directional sequencing $\ast \ast 3 / 3$ false positives observed were negative for $K .$ oxytoca by PCR/bi‐directional sequencing

<table><tr><td rowspan=1 colspan=5>Table 11e: iC-GN Assay Performance: Klebsiellapneumoniae (parC)</td></tr><tr><td rowspan=2 colspan=2>SpecimenType</td><td rowspan=2 colspan=1>N=</td><td rowspan=1 colspan=1>Percent Agreement</td><td rowspan=1 colspan=1>ComparatorMethod</td></tr><tr><td rowspan=1 colspan=1>Positive(95% CI)</td><td rowspan=4 colspan=1>Culture &amp;MALDI</td></tr><tr><td rowspan=3 colspan=1>Jrocrd</td><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>976</td><td rowspan=1 colspan=1>96.8%150/155*(92.7-98.6)</td><td rowspan=1 colspan=1>99.3%815/821**(98.4-99.7)</td></tr><tr><td rowspan=1 colspan=1>Frozen</td><td rowspan=1 colspan=1>26</td><td rowspan=1 colspan=1>100%3/3 (43.9-100)</td><td rowspan=1 colspan=1>100%23/23 (85.7-100)</td></tr><tr><td rowspan=1 colspan=1>TOTAL</td><td rowspan=1 colspan=1>1002</td><td rowspan=1 colspan=1>96.8%153/158(92.8-98.6)</td><td rowspan=1 colspan=1>99.3%838/844(98.4-99.7)</td></tr><tr><td rowspan=1 colspan=2>Contrived</td><td rowspan=1 colspan=1>170</td><td rowspan=1 colspan=1>100%21/21(84.5-100)</td><td rowspan=1 colspan=1>99.3%148/149(96.3-99.9)</td></tr></table>

$\ast 3 / 5$ false negatives observed were negative for $K .$ . pneumoniae by PCR/bi‐directional sequencing; 2/3 were positive for $K .$ . pneumoniae by PCR/bi‐directional sequencing $^ { * * } 6 / 6$ false positives observed were negative for $K .$ . pneumoniae by PCR/bi‐directional sequencing

<table><tr><td rowspan=1 colspan=5>Table 11f: iC-GN Assay Performance: Proteus mirabilis(rpoB)</td></tr><tr><td rowspan=2 colspan=2>SpecimenType</td><td rowspan=2 colspan=1>N=</td><td rowspan=1 colspan=1>Percent Agreement</td><td rowspan=1 colspan=1>ComparatorMethod</td></tr><tr><td rowspan=1 colspan=1>Positive(95% CI)</td><td rowspan=4 colspan=1>Culture &amp;MALDI</td></tr><tr><td rowspan=3 colspan=1>Jrond</td><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>957</td><td rowspan=1 colspan=1>97.4%37/38*(86.5-99.5)</td><td rowspan=1 colspan=1>99.5%914/919**(98.7-99.8)</td></tr><tr><td rowspan=1 colspan=1>Frozen</td><td rowspan=1 colspan=1>26</td><td rowspan=1 colspan=1>100%9/9(70.1-100)</td><td rowspan=1 colspan=1>100%17/17(81.6-100)</td></tr><tr><td rowspan=1 colspan=1>TOTAL</td><td rowspan=1 colspan=1>983***</td><td rowspan=1 colspan=1>97.9%46/47(88.9-99.6)</td><td rowspan=1 colspan=1>99.5%931/936(98.8-99.8)</td></tr><tr><td rowspan=1 colspan=2>Contrived</td><td rowspan=1 colspan=1>170</td><td rowspan=1 colspan=1>100%12/12(75.8-100)</td><td rowspan=1 colspan=1>100%158/158(97.6-100)</td></tr></table>

$^ { * } 1 / 1$ false negative observed was positive for $P$ . mirabilis by PCR/bi‐directional sequencing $\ast \ast 3 / 5$ false positives observed were negative for $P$ . mirabilis by PCR/bi‐directional sequencing; 2/5 were not available for sequencing

\*\*\* 19 samples were excluded from Proteus mirabilis performance analysis due to confirmed Proteus contamination within the BD BACTEC Bottles, leaving a total of 983 evaluable specimens.

<table><tr><td rowspan=1 colspan=5>Table 11g: iC-GN Assay Performance: Pseudomonasaeruginosa (algD)</td></tr><tr><td rowspan=2 colspan=2>SpecimenType</td><td rowspan=2 colspan=1>N=</td><td rowspan=1 colspan=1>Percent Agreement</td><td rowspan=1 colspan=1>ComparatorMethod</td></tr><tr><td rowspan=1 colspan=1>Positive(95% CI)</td><td rowspan=4 colspan=1>Culture &amp;MALDI</td></tr><tr><td rowspan=3 colspan=1>Jroed</td><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>976</td><td rowspan=1 colspan=1>95.1%78/82*(88.1-98.1)</td><td rowspan=1 colspan=1>99.8%892/894**(99.2-99.9)</td></tr><tr><td rowspan=1 colspan=1>Frozen</td><td rowspan=1 colspan=1>26</td><td rowspan=1 colspan=1>100%1/1(20.7-100)</td><td rowspan=1 colspan=1>100%25/25(86.7-100)</td></tr><tr><td rowspan=1 colspan=1>TOTAL</td><td rowspan=1 colspan=1>1002</td><td rowspan=1 colspan=1>95.2%79/83(88.3-98.1)</td><td rowspan=1 colspan=1>99.8%917/919(99.2-99.9)</td></tr><tr><td rowspan=1 colspan=2>Contrived</td><td rowspan=1 colspan=1>170</td><td rowspan=1 colspan=1>100%10/10(72.2-100)</td><td rowspan=1 colspan=1>100%160/160(97.7-100)</td></tr></table>

$^ { * } 4 / 4$ false negatives observed were positive for $P$ . aeruginosa by PCR/bi‐directional sequencing $\ast \ast 2 / 2$ false positives observed were negative for $P$ . aeruginosa by PCR/bi‐directional sequencing

<table><tr><td rowspan=1 colspan=5>Table 11h: iC-GN Assay Performance: Serratiamarcescens(gyrB)</td></tr><tr><td rowspan=2 colspan=2>SpecimenType</td><td rowspan=2 colspan=1>N=</td><td rowspan=1 colspan=1>Percent Agreement</td><td rowspan=1 colspan=1>ComparatorMethod</td></tr><tr><td rowspan=1 colspan=1>Positive(95% CI)</td><td rowspan=4 colspan=1>Culture &amp;MALDI</td></tr><tr><td rowspan=3 colspan=1>Jrocd</td><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>976</td><td rowspan=1 colspan=1>100%29/29(88.3-100)</td><td rowspan=1 colspan=1>99.6%943/947**(98.9-99.8)</td></tr><tr><td rowspan=1 colspan=1>Frozen</td><td rowspan=1 colspan=1>26</td><td rowspan=1 colspan=1>0/0-</td><td rowspan=1 colspan=1>100%26/26(87.1-100)</td></tr><tr><td rowspan=1 colspan=1>TOTAL</td><td rowspan=1 colspan=1>1002</td><td rowspan=1 colspan=1>100%29/29(88.3-100)</td><td rowspan=1 colspan=1>99.6%969/973(98.9-99.8)</td></tr><tr><td rowspan=1 colspan=2>Contrived</td><td rowspan=1 colspan=1>170</td><td rowspan=1 colspan=1>100%20/20(83.9-100)</td><td rowspan=1 colspan=1>99.3%149/150(96.3-99.9)</td></tr></table>

$^ { * * } 1 / 4$ false positives observed was positive for $S .$ . marcescens by PCR/bi‐directional sequencing; 3/4 were negative for $S _ { \cdot }$ . marcescens by PCR/bi‐directional sequencing

<table><tr><td rowspan=1 colspan=5>Table 11I: iC-GN Assay Performance: CTX-M</td></tr><tr><td rowspan=2 colspan=2>SpecimenType</td><td rowspan=2 colspan=1>N=</td><td rowspan=1 colspan=1>Percent Agreement</td><td rowspan=1 colspan=1>ComparatorMethod</td></tr><tr><td rowspan=1 colspan=1>Positive(95% CI)</td><td rowspan=4 colspan=1>PCR/Bi-directionalsequencing</td></tr><tr><td rowspan=3 colspan=1>Jroed</td><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>976</td><td rowspan=1 colspan=1>97.0%64/66 (89.6-99.2)</td><td rowspan=1 colspan=1>99.9%909/910(99.4-100)</td></tr><tr><td rowspan=1 colspan=1>Frozen</td><td rowspan=1 colspan=1>26</td><td rowspan=1 colspan=1>100%1/1 (20.7-100)</td><td rowspan=1 colspan=1>100%25/25 (86.7-100)</td></tr><tr><td rowspan=1 colspan=1>TOTAL</td><td rowspan=1 colspan=1>1002</td><td rowspan=1 colspan=1>97.0%65/67 (89.8-99.2)</td><td rowspan=1 colspan=1>99.9%934/935(99.4-100)</td></tr><tr><td rowspan=1 colspan=2>Contrived</td><td rowspan=1 colspan=1>170</td><td rowspan=1 colspan=1>100%15/15 (79.6-100)</td><td rowspan=1 colspan=1>100%155/155(97.6-100)</td></tr></table>

<table><tr><td colspan="5" rowspan="1">Table 11J: iC-GN Assay Performance: KPC</td></tr><tr><td colspan="2" rowspan="2">SpecimenType</td><td colspan="1" rowspan="2">N=</td><td colspan="1" rowspan="1">Percent Agreement</td><td colspan="1" rowspan="1">ComparatorMethod</td></tr><tr><td colspan="1" rowspan="1">Positive(95% CI)</td><td colspan="1" rowspan="4">PCR/Bi-directionalsequencing</td></tr><tr><td colspan="1" rowspan="3">Jrocd</td><td colspan="1" rowspan="1">Fresh</td><td colspan="1" rowspan="1">976</td><td colspan="1" rowspan="1">100%1/1 (20.7-100)</td><td colspan="1" rowspan="1">99.9%974/975(99.4-100)</td></tr><tr><td colspan="1" rowspan="1">Frozen</td><td colspan="1" rowspan="1">26</td><td colspan="1" rowspan="1">-0/0-</td><td colspan="1" rowspan="1">100%26/26 (87.1-100)</td></tr><tr><td colspan="1" rowspan="1">TOTAL</td><td colspan="1" rowspan="1">1002</td><td colspan="1" rowspan="1">100%1/1 (20.7-100)</td><td colspan="1" rowspan="1">99.9%1000/1001(99.4-100)</td></tr><tr><td colspan="2" rowspan="1">Contrived</td><td colspan="1" rowspan="1">170</td><td colspan="1" rowspan="1">100%50/50(92.9-100)</td><td colspan="1" rowspan="1">99.2%119/120(95.4-99.9)</td></tr><tr><td colspan="5" rowspan="1">Table 11K: iC-GN Assay Performance: NDM</td></tr><tr><td colspan="2" rowspan="2">SpecimenType</td><td colspan="1" rowspan="2">N=</td><td colspan="2" rowspan="1">Percent Agreement</td></tr><tr><td colspan="1" rowspan="1">Positive(95% CI)</td><td colspan="1" rowspan="4">PCR/Bi-directionalsequencing</td></tr><tr><td colspan="1" rowspan="3">Jrecsd</td><td colspan="1" rowspan="1">Fresh</td><td colspan="1" rowspan="1">976</td><td colspan="1" rowspan="1">-0/0-</td><td colspan="1" rowspan="1">100%976/976(99.6-100)</td></tr><tr><td colspan="1" rowspan="1">Frozen</td><td colspan="1" rowspan="1">26</td><td colspan="1" rowspan="1">-0/0-</td><td colspan="1" rowspan="1">100%26/26(87.1-100)</td></tr><tr><td colspan="1" rowspan="1">TOTAL</td><td colspan="1" rowspan="1">1002</td><td colspan="1" rowspan="1">-0/0-</td><td colspan="1" rowspan="1">100%1002/1002(99.6-100)</td></tr><tr><td colspan="2" rowspan="1">Contrived</td><td colspan="1" rowspan="1">170</td><td colspan="1" rowspan="1">100%50/50(92.9-100)</td><td colspan="1" rowspan="1">100%120/120(96.9-100)</td></tr></table>

2. Analysis of Mixed Culture Results:

In the method comparison study, there were thirty (30) mixed culture specimens that were detected by the iC-GN Assay, culture and MALDI, or both. There were twelve (12) discrepant mixed samples for which iC-GN detected a target that was not detected by the comparator assay. There were four (4) discrepant mixed samples for which the comparator assay detected targets that were not detected by iC-GN. Due to competitive inhibition, target organisms present at low concentrations may not be detected by the iCGN Assay when a second target organism is present at higher concentrations. See the 510(k) Summary, Tables 29 and 30 (p.31-32) which includes tables that list the mixed target combinations detected by iC-GN and the comparator method in the clinical study.

<table><tr><td colspan="8" rowspan="1">Table 12: Multiple Organism Detections by iC-GN as Compared to Culture/MALDI</td></tr><tr><td colspan="5" rowspan="1">Multiple Detections by iC-GN</td><td colspan="1" rowspan="1">TotalTargetsDetected</td><td colspan="1" rowspan="2">No ofDiscrepantTargets</td><td colspan="1" rowspan="2">Discrepant Results(Targets NotDetected byculture/MALDI)</td></tr><tr><td colspan="1" rowspan="1">Site</td><td colspan="1" rowspan="1">ID</td><td colspan="1" rowspan="1">Target 1</td><td colspan="1" rowspan="1">Target 2</td><td colspan="1" rowspan="1">Target 3</td><td colspan="1" rowspan="1">by iC-GN</td></tr><tr><td colspan="1" rowspan="1">LAC</td><td colspan="1" rowspan="1">1102</td><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">K. pneumoniae</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">LAC</td><td colspan="1" rowspan="1">1118</td><td colspan="1" rowspan="1">E. col</td><td colspan="1" rowspan="1">K. pneumoniae</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">LAC</td><td colspan="1" rowspan="1">1141</td><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">K. pneumoniae</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">LAC</td><td colspan="1" rowspan="1">1220</td><td colspan="1" rowspan="1">E. cloacaecomplex</td><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">LAC</td><td colspan="1" rowspan="1">1236</td><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">K. oxytoca</td><td colspan="1" rowspan="1">K. pneumoniae</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">K. oxytoca</td></tr><tr><td colspan="1" rowspan="1">LAC</td><td colspan="1" rowspan="1">1285</td><td colspan="1" rowspan="1">K. pneumoniae</td><td colspan="1" rowspan="1">S. marcescens</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">S. marcescens</td></tr><tr><td colspan="1" rowspan="1">LAC</td><td colspan="1" rowspan="1">1307</td><td colspan="1" rowspan="1">E. col</td><td colspan="1" rowspan="1">K. oxytoca</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">LAC</td><td colspan="1" rowspan="1">1378</td><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">K. pneumoniae</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">K. pneumoniae</td></tr><tr><td colspan="1" rowspan="1">LAC</td><td colspan="1" rowspan="1">1382</td><td colspan="1" rowspan="1">K. oxytoca</td><td colspan="1" rowspan="1">K. pneumoniae</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">K. oxytoca</td></tr><tr><td colspan="1" rowspan="1">MCW</td><td colspan="1" rowspan="1">2023</td><td colspan="1" rowspan="1">E. col</td><td colspan="1" rowspan="1">K. pneumoniae</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="5" rowspan="1">Multiple Detections by iC-GN</td><td colspan="1" rowspan="2">TotalTargetsDetectedby iC-GN</td><td colspan="1" rowspan="2">No ofDiscrepantTargets</td><td colspan="1" rowspan="2">Discrepant Results(Targets NotDetected byculture/MALDI)</td></tr><tr><td colspan="1" rowspan="1">Site</td><td colspan="1" rowspan="1">ID</td><td colspan="1" rowspan="1">Target 1</td><td colspan="1" rowspan="1">Target 2</td><td colspan="1" rowspan="1">Target 3</td></tr><tr><td colspan="1" rowspan="1">MCW</td><td colspan="1" rowspan="1">2032</td><td colspan="1" rowspan="1">E. cloacaecomplex</td><td colspan="1" rowspan="1">K. oxytoca</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">MCW</td><td colspan="1" rowspan="1">2038</td><td colspan="1" rowspan="1">K. oxytoca</td><td colspan="1" rowspan="1">K. pneumoniae</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">MCW</td><td colspan="1" rowspan="1">2041</td><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">P. mirabilis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">MCW</td><td colspan="1" rowspan="1">2104</td><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">S. marcescens</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">S. marcescens</td></tr><tr><td colspan="1" rowspan="1">MCW</td><td colspan="1" rowspan="1">2193</td><td colspan="1" rowspan="1">K. pneumoniae</td><td colspan="1" rowspan="1">S. marcescens</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">K. pneumoniae, S.marcescens</td></tr><tr><td colspan="1" rowspan="1">TC</td><td colspan="1" rowspan="1">3015</td><td colspan="1" rowspan="1">K. oxytoca</td><td colspan="1" rowspan="1">P. aeruginosa</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">K. oxytoca, P.aeruginosa</td></tr><tr><td colspan="1" rowspan="1">TC</td><td colspan="1" rowspan="1">3096</td><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">K. pneumoniae</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">K. pneumoniae</td></tr><tr><td colspan="1" rowspan="1">TC</td><td colspan="1" rowspan="1">3131</td><td colspan="1" rowspan="1">E. cloacaecomplex</td><td colspan="1" rowspan="1">K. pneumoniae</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">TC</td><td colspan="1" rowspan="1">3183</td><td colspan="1" rowspan="1">K. pneumoniae</td><td colspan="1" rowspan="1">S. marcescens</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">S. marcescens</td></tr><tr><td colspan="1" rowspan="1">TGH</td><td colspan="1" rowspan="1">4031</td><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">P. mirabilis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">P. mirabilis</td></tr><tr><td colspan="1" rowspan="1">TGH</td><td colspan="1" rowspan="1">4037</td><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">P. aeruginosa</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">TGH</td><td colspan="1" rowspan="1">4124</td><td colspan="1" rowspan="1">E. cloacaecomplex</td><td colspan="1" rowspan="1">P. aeruginosa</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">P. aeruginosa</td></tr><tr><td colspan="1" rowspan="1">TGH</td><td colspan="1" rowspan="1">4132</td><td colspan="1" rowspan="1">E. cloacaecomplex</td><td colspan="1" rowspan="1">K. pneumoniae</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">IU</td><td colspan="1" rowspan="1">5025</td><td colspan="1" rowspan="1">A. baumanniicomplex</td><td colspan="1" rowspan="1">K. pneumoniae</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">A. baumanniicomplex</td></tr><tr><td colspan="1" rowspan="1">IU</td><td colspan="1" rowspan="1">5031</td><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">K. pneumoniae</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">IU</td><td colspan="1" rowspan="1">5042</td><td colspan="1" rowspan="1">E. cloacaecomplex</td><td colspan="1" rowspan="1">K. pneumoniae</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">LAC</td><td colspan="1" rowspan="1">1102</td><td colspan="1" rowspan="1">K. pneumoniae</td><td colspan="2" rowspan="1">E. coli</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">LAC</td><td colspan="1" rowspan="1">1118</td><td colspan="1" rowspan="1">K. pneumoniae</td><td colspan="2" rowspan="1">E. coli</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">LAC</td><td colspan="1" rowspan="1">1141</td><td colspan="1" rowspan="1">E. coli</td><td colspan="2" rowspan="1">K. pneumoniae</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">LAC</td><td colspan="1" rowspan="1">1220</td><td colspan="1" rowspan="1">E. cloacaecomplex</td><td colspan="2" rowspan="1">E. coli</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">LAC</td><td colspan="1" rowspan="1">1236</td><td colspan="1" rowspan="1">K. pneumoniae</td><td colspan="2" rowspan="1">E. coli</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">LAC</td><td colspan="1" rowspan="1">1268</td><td colspan="1" rowspan="1">P. aeruginosa</td><td colspan="2" rowspan="1">P. mirabilis</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">P. aeruginosa</td></tr><tr><td colspan="1" rowspan="1">LAC</td><td colspan="1" rowspan="1">1307</td><td colspan="1" rowspan="1">E. coli</td><td colspan="2" rowspan="1">K. oxytoca</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">LAC</td><td colspan="1" rowspan="1">1338</td><td colspan="1" rowspan="1">E. coli</td><td colspan="2" rowspan="1">K. pneumoniae</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">K. pneumoniae</td></tr><tr><td colspan="1" rowspan="1">MCW</td><td colspan="1" rowspan="1">2023</td><td colspan="1" rowspan="1">K. pneumoniae</td><td colspan="2" rowspan="1">E. coli</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">MCW</td><td colspan="1" rowspan="1">2032</td><td colspan="1" rowspan="1">K. oxytoca</td><td colspan="2" rowspan="1">E. cloacae complex</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">MCW</td><td colspan="1" rowspan="1">2038</td><td colspan="1" rowspan="1">K. pneumoniae</td><td colspan="2" rowspan="1">K. oxytoca</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">MCW</td><td colspan="1" rowspan="1">2041</td><td colspan="1" rowspan="1">P. mirabilis</td><td colspan="2" rowspan="1">E. coli</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">TC</td><td colspan="1" rowspan="1">3006</td><td colspan="1" rowspan="1">E. coli</td><td colspan="2" rowspan="1">K. pneumoniae</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">K. pneumoniae</td></tr><tr><td colspan="1" rowspan="1">TC</td><td colspan="1" rowspan="1">3131</td><td colspan="1" rowspan="1">E. cloacaecomplex</td><td colspan="2" rowspan="1">K. pneumoniae</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="5" rowspan="1">Multiple Detections by iC-GN</td><td colspan="1" rowspan="1">TotalTargetsDetected</td><td colspan="1" rowspan="2">No ofDiscrepantTargets</td><td colspan="1" rowspan="2">Discrepant Results(Targets NotDetected byculture/MALDI)</td></tr><tr><td colspan="1" rowspan="1">Site</td><td colspan="1" rowspan="1">ID</td><td colspan="1" rowspan="1">Target 1</td><td colspan="1" rowspan="1">Target 2</td><td colspan="1" rowspan="1">Target 3</td><td colspan="1" rowspan="1">by iC-GN</td></tr><tr><td colspan="1" rowspan="1">TGH</td><td colspan="1" rowspan="1">4007</td><td colspan="1" rowspan="1">E. coli</td><td colspan="2" rowspan="1">P. aeruginosa</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">P. aeruginosa</td></tr><tr><td colspan="1" rowspan="1">TGH</td><td colspan="1" rowspan="1">4037</td><td colspan="1" rowspan="1">E. coli</td><td colspan="2" rowspan="1">P. aeruginosa</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">TGH</td><td colspan="1" rowspan="1">4132</td><td colspan="1" rowspan="1">K. pneumoniae</td><td colspan="2" rowspan="1">E. cloacae complex</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">IU</td><td colspan="1" rowspan="1">5031</td><td colspan="1" rowspan="1">E. coli</td><td colspan="2" rowspan="1">K. pneumoniae</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">IU</td><td colspan="1" rowspan="1">5042</td><td colspan="1" rowspan="1">K. pneumoniae</td><td colspan="2" rowspan="1">E. cloacae complex</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr></table>

# C. Expected Values:

A total of 1002 prospectively collected fresh and frozen blood culture specimens were obtained from five geographically dispersed clinical sites. The number and percentage of positive cases (positivity rate) determined by the iC-GN Assay stratified by U.S. state for each of the organisms and resistance markers detected by the assay are presented below. Overall, the iC-GN Assay detected at least one organism in $89 \%$ (901/1002) prospectively collected specimens and at least one resistance marker in $6 . 8 \%$ (68/1002) prospectively collected specimens. Expected values are presented in the $5 1 0 ( \mathrm { k } )$ Summary, Table 30 (p.32- 33).

# VII. Proposed Labeling:

The labeling supports the finding of substantial equivalence for this device.

# VIII. Conclusion:

The submitted information in this premarket notification is complete and supports a substantial equivalence decision.