# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATION DECISION SUMMARY ASSAY AND INSTRUMENT COMBINATION TEMPLATE

A. 510(k) Number:

K190613

# B. Purpose for Submission:

To obtain a substantial equivalence determination for the BD MAX Check-Points CPO Assay on the BD MAX System in the qualitative detection of the blaKPC, blaNDM, blaVIM/blaIMP, and blaOXA-48 gene sequences from rectal swab specimens.

# C. Measurand:

Target DNA sequence of the following genes: blaKPC, blaNDM, blaVIM/blaIMP, and blaOXA-48

D. Type of Test: Qualitative real-time polymerase chain reaction (PCR) assay

# E. Applicant:

Check-Points Health B.V.

F. Proprietary and Established Names: BD MAX Check-Points CPO

# G. Regulatory Information:

1. Regulation section: 21 CFR 866.1640 (Antimicrobial susceptibility test powder)

2. Classification: Class II

3. Product code:

POC—System, nucleic acid amplification test, DNA, antimicrobial resistance marker, direct specimen   
OOI—Real-time nucleic acid amplification system

4. Panel:

83-Microbiology

# H. Intended Use:

1. Intended use(s):

The BD MAX Check-Points CPO Assay performed on the BD MAX System is a qualitative, automated in vitro diagnostic real-time PCR test designed for the detection and differentiation of the carbapenemase genes blaKPC, blaNDM, blaVIM/blaIMP and blaOXA48, that are associated with carbapenem non-susceptibility in Gram-negative bacteria. The assay does not distinguish between the blaVIM and blaIMP genes.

The BD MAX Check-Points CPO Assay is intended as an aid to infection control in the detection of carbapenem-non-susceptible bacteria that colonize patients in healthcare settings. The BD MAX Check-Points CPO Assay is not intended to guide or monitor treatment for carbapenem-non-susceptible bacterial infections. A negative BD MAX Check-Points CPO Assay result does not preclude the presence of other resistance mechanisms.

Testing is performed on rectal swabs from patients at risk for intestinal colonisation with carbapenem non-susceptible bacteria. This test is intended for use in conjunction with clinical presentation, laboratory findings, and epidemiological information. Results of this test should not be used as the sole basis for patient management decisions. Concomitant cultures are necessary to recover organisms for epidemiological typing, antimicrobial susceptibility testing, and for further confirmatory bacterial identification.

2. Indication(s) for use: Same as the Intended Use.

3. Special conditions for use statement(s):

For prescription use only.

The BD MAX Check-Points CPO Assay detects blaKPC, blaNDM, blaVIM/blaIMP, and blaOXA-48 gene sequences from rectal swab specimens and is not for bacterial identification or to report susceptibility status.

The BD MAX Check-Points CPO Assay is not a sub-typing tool and does not report or distinguish between variants of the blaKPC, blaNDM, blaVIM/blaIMP, and blaOXA-48 genes.

Detection of blaKPC, blaNDM, blaVIM/blaIMP, and/or blaOXA-48 gene sequences does not indicate the presence of viable organisms with these markers in the specimen.

# 4. Special instrument requirements:

The BD MAX Check-Points CPO Assay uses PCR technology on the BD MAX System, which extracts, amplifies, and detects the target DNA.

# I. Device Description:

The BD MAX Check-Points CPO Assay detects the presence of carbapenemase genes in Gram-negative bacteria. Briefly, rectal swab specimens are collected from patients using an ESwab collection and transport system. The ESwab tube is vortexed and a $5 0 \mu \mathrm { l }$ aliquot is transferred to the BD MAX Check-Points CPO Sample Buffer Tube (SBT) using a pipette with a disposable filter tip. The SBT is closed with a septum cap and vortexed. Once the worklist is generated and the clinical specimen is loaded on the BD MAX System, along with a BD MAX Check-Points CPO Reagent Strip and BD MAX PCR Cartridge, the run is started with no further operator intervention. The BD MAX System automates sample preparation, including target organism lysis, DNA extraction and concentration, reagent rehydration, target nucleic acid sequence amplification, and detection using real-time PCR. The interpretation of the signal is performed automatically by the BD MAX System. The assay also includes a Sample Processing Control (SPC) that is provided in the Extraction Tube and subjected to extraction, concentration, and amplification steps. The SPC monitors for the presence of potential inhibitory substances as well as system or reagent failures.

# J. Substantial Equivalence Information:

1. Predicate device name(s): Xpert Carba-R Assay   
2. Predicate 510(k) number(s): K160901   
3. Comparison with predicate:

<table><tr><td colspan="3"></td></tr><tr><td colspan="3">Similarities</td></tr><tr><td>Item</td><td>Subject Device BD MAX Check-Points CPO Assay (K190613)</td><td>Predicate Device Cepheid Xpert Carba-R Assay (K160901)</td></tr><tr><td>Intended Use</td><td>The BD MAX Check-Points CPO Assay performed on the BD MAX System is a qualitative, automated in vitro diagnostic real-time PCR test designed for the detection and differentiation of the carbapenemase genes blakpc, blanDM, blavm/blaMp and blaoxA-48, that are associated with carbapenem non- susceptibility in Gram-negative bacteria. The assay does not distinguish between the blavim and blaimp genes. The BD MAX Check-Points CPO Assay is intended as an aid to infection control in the detection of carbapenem-non- susceptible bacteria that colonize patients in healthcare settings. The BD MAX Check-Points CPO Assay is not intended to guide or monitor treatment for carbapenem-non-susceptible bacterial infections. A negative BD MAX Check- Points CPO Assay result does not preclude the presence of other resistance mechanisms. Testing is performed on rectal swabs from patients at risk for intestinal colonisation with carbapenem non-susceptible bacteria. This test is intended for use in conjunction with clinical presentation, laboratory findings, and epidemiological information. Results of this test should not be used as the sole basis for patient management decisions. Concomitant cultures are necessary to recover organisms for epidemiological typing, antimicrobial susceptibility testing, and for further confirmatory bacterial identification.</td><td>The Xpert Carba-R Assay, performed on the GeneXpert Instrument Systems, is a qualitative in vitro diagnostic test designed for the detection and differentiation of the blakpc, blanDm, blavim, blaoxA-48, and blamp gene sequences associated with carbapenem-non-susceptibility. The test utilizes automated real-time polymerase chain reaction (PCR). The Xpert Carba-R Assay is intended as an aid to infection control in the detection of carbapenem-non-susceptible bacteria that colonize patients in healthcare settings. The Xpert Carba-R Assay is not intended to guide or monitor treatment for carbapenem- non-susceptible bacterial infections. A negative Xpert Carba-R Assay result does not preclude the presence of other resistance mechanisms. The Xpert Carba R-Assay is for use with the following sample types: Rectal Swab Specimens The assay is performed on rectal swab specimens from patients at risk for intestinal colonization with carbapenem-non- susceptible bacteria. Concomitant cultures are necessary to recover organisms for epidemiological typing, antimicrobial susceptibility testing, and for further confirmatory bacterial identification. Pure Colonies The assay is performed on carbapenem-non- susceptible pure colonies of Enterobacteriaceae, Acinetobacter baumannii, or Pseudomonas aeruginosa, when grown on blood agar or MacConkey agar. For testing pure colonies, the Xpert Carba-R Assay should be used in conjunction with other laboratory tests</td></tr><tr><td colspan="3" rowspan="1">Similarities</td></tr><tr><td colspan="1" rowspan="2">Item</td><td colspan="1" rowspan="1">Subject Device</td><td colspan="1" rowspan="1">Predicate Device</td></tr><tr><td colspan="1" rowspan="1">BD MAX Check-Points CPO Assay(K190613)</td><td colspan="1" rowspan="1">Cepheid Xpert Carba-R Assay(K160901)</td></tr><tr><td colspan="1" rowspan="1">TechnologicalPrinciples</td><td colspan="1" rowspan="1">Same</td><td colspan="1" rowspan="1">Fully-automated nucleic acid amplification(DNA); real-time PCR</td></tr><tr><td colspan="1" rowspan="1">DetectionProbe</td><td colspan="1" rowspan="1">Same</td><td colspan="1" rowspan="1">TaqMan Probes</td></tr><tr><td colspan="1" rowspan="1">Controls</td><td colspan="1" rowspan="1">Same</td><td colspan="1" rowspan="1">Internal sample processing control</td></tr><tr><td colspan="1" rowspan="1">Assay Targets</td><td colspan="1" rowspan="1">Same</td><td colspan="1" rowspan="1">blaκpc, blanDm, blavm, blaoxA-48, and blaMPgene sequences</td></tr><tr><td colspan="1" rowspan="1">Interpretationof Test Results</td><td colspan="1" rowspan="1">Same</td><td colspan="1" rowspan="1">Diagnostic software of the InstrumentSystem</td></tr></table>

<table><tr><td rowspan=1 colspan=3>Differences</td></tr><tr><td rowspan=1 colspan=1>Item</td><td rowspan=1 colspan=1>Subject Device</td><td rowspan=1 colspan=1>Predicate Device</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>BD MAX Check-Points CPO Assay(K190613)</td><td rowspan=1 colspan=1>Cepheid Xpert Carba-R Assay(K160901)</td></tr><tr><td rowspan=1 colspan=1>Test System</td><td rowspan=1 colspan=1>BD MAX System</td><td rowspan=1 colspan=1>GeneXpert Instrument System (includesGeneXpert Dx, Infinity-48, Infinity-48s, andInfinity-80)</td></tr><tr><td rowspan=1 colspan=1>Test ReagentCartridge(s)</td><td rowspan=1 colspan=1>BD MAX Check-Points CPOReagent Strip and BD MAX PCR Cartridge,</td><td rowspan=1 colspan=1>Disposable single-use, multi-chamberedfluidic cartridge</td></tr><tr><td rowspan=1 colspan=1>Sample Type</td><td rowspan=1 colspan=1>Rectal Swab Only</td><td rowspan=1 colspan=1>Rectal Swab and Bacterial isolates fromculture</td></tr><tr><td rowspan=1 colspan=1>Organisms</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>Carbapenem non-susceptibleEnterobacteriaceae, Pseudomonasaeruginosa, Acinetobacter baumannii (forbacterial isolate claim)</td></tr></table>

# K. Standard/Guidance Document Referenced (if applicable):

CLSI M100-S27. Performance Standards for Antimicrobial Susceptibility Testing; Approved standard—Twenty-seventh Informational Supplement, 2017

CLSI M02-A12. Performance Standards for Antimicrobial Disk Susceptibility Tests; Twelfth Edition, 2015

# L. Test Principle:

The BD MAX System automates sample preparation, including target organism lysis, DNA extraction and concentration, reagent rehydration, target nucleic acid sequence amplification, and detection using real-time PCR. Following enzymatic cell lysis of the sample at an elevated temperature, the released nucleic acids are captured on magnetic affinity beads. The beads, with the bound nucleic acids, are washed and the nucleic acids are eluted. Eluted DNA is then neutralized and transferred to the Master Mix Tube to rehydrate the PCR reagents. After rehydration, the BD MAX System dispenses a fixed volume of PCR-ready solution into the BD MAX PCR Cartridge. The amplified DNA targets are detected using hydrolysis

(TaqMan) probes, labeled at one end with a fluorescent reporter dye (fluorophore) and at the other end with a quencher moiety. Probes labeled with different fluorophores are used to detect amplicons for the carbapenemase gene targets (KPC, VIM, OXA-48, NDM, IMP) and the SPC by the BD MAX System. The VIM and IMP genes are combined in one optical channel of the BD MAX system; all other genes have a separate optical channel. When the probes are in their native state, the fluorescence of the fluorophore is quenched due to its proximity to the quencher. However, in the presence of target DNA, the probes hybridize to their complementary sequences and are hydrolyzed by the DNA polymerase as it synthesizes the nascent strand along the DNA template. As a result, the fluorophores are separated from the quencher molecules and fluorescence is emitted. The BD MAX System monitors these signals at each cycle and interprets the data at the end of the program to report the final results.

# Interpretation of Results

Results are available on the <Results> tab in the <Results> window on the BD MAX System monitor. The BD MAX System software automatically interprets test results. Results are reported for each analyte and the SPC. A test result may be called NEG (Negative), POS (Positive) or UNR (Unresolved) based on the amplification status of the target and of the SPC. IND (Indeterminate), or INC (Incomplete) results are due to BD MAX System failures. An interpretation table for test results is shown in Table 1.

Table 1. Interpretation of Test Results for the BD MAX Check-Points CPO Assay   

<table><tr><td rowspan=1 colspan=1>Assay ResultReported</td><td rowspan=1 colspan=1>Interpretation of Result</td></tr><tr><td rowspan=1 colspan=1>KPC POS</td><td rowspan=1 colspan=1>KPC gene detected</td></tr><tr><td rowspan=1 colspan=1>KPC NEG</td><td rowspan=1 colspan=1>No KPC gene detected</td></tr><tr><td rowspan=1 colspan=1>VIM and/or IMP POS</td><td rowspan=1 colspan=1>VIM and/or IMP gene detected</td></tr><tr><td rowspan=1 colspan=1>VIM and/or IMP NEG</td><td rowspan=1 colspan=1>No VIM or IMP gene detected</td></tr><tr><td rowspan=1 colspan=1>OXA POS</td><td rowspan=1 colspan=1>OXA-48 gene detected</td></tr><tr><td rowspan=1 colspan=1>OXA NEG</td><td rowspan=1 colspan=1>No OXA-48 gene detected</td></tr><tr><td rowspan=1 colspan=1>NDM POS</td><td rowspan=1 colspan=1>NDM gene detected</td></tr><tr><td rowspan=1 colspan=1>NDM NEG</td><td rowspan=1 colspan=1>No NDM gene detected</td></tr><tr><td rowspan=1 colspan=1>UNR</td><td rowspan=1 colspan=1>Unresolved  inhibitory sample or reagent failure; no SampleProcessing Control amplification</td></tr><tr><td rowspan=1 colspan=1>IND</td><td rowspan=1 colspan=1>Indeterminate due to BD MAX System failure (with Warning orError Codes)</td></tr><tr><td rowspan=1 colspan=1>INC</td><td rowspan=1 colspan=1>Incomplete run (with Warning or Error Codes)</td></tr></table>

# M. Performance Characteristics (if/when applicable):

1. Analytical performance:

a. Precision/Reproducibility:

The reproducibility of the BD MAX Check-Points CPO Assay was established in a multi-center study. An 12-member panel was tested that included: 1) negative rectal swab spiked with target bacterial cells; one strain for each of the five resistance genes detected by the BD MAX Check-Points CPO Assay and 2) two negative samples for all five gene targets—negative rectal swab spiked with non-target bacterial cells (NC) and a negative rectal swab (Blank). Each organism was spiked into negative rectal swab matrix at low positive ${ \mathrm { [ } } { \mathord { \sim } } 1 . 5 \mathrm { x }$ LoD) and moderate positive levels $3 \mathbf { x }$ LoD). To measure site-to-site reproducibility, the 12-member panel was tested in duplicates each day at three sites over a five-day testing period with two operators per site. Interlot variation was analyzed by testing five different targets (with two concentrations per strain) at one site by two operators using three different lots across 5 days. The results of the Site-to-Site Reproducibility Study and Precision Study are summarized in Table 2 and Table 3 below. Table 4 shows the mean Ct values and $\%$ CV results for the Reproducibility Panel.

Table 2. Reproducibility of the 12-Member Sample Panel (Site-to-Site)   

<table><tr><td rowspan=2 colspan=1>Resistance Gene</td><td rowspan=1 colspan=3>Site 1</td><td rowspan=1 colspan=3>Site 2</td><td rowspan=1 colspan=3>Site 3</td><td rowspan=2 colspan=1>% TotalAgreementbby Sample</td></tr><tr><td rowspan=1 colspan=1>Op 1</td><td rowspan=1 colspan=1>Op 2</td><td rowspan=1 colspan=1>Site</td><td rowspan=1 colspan=1>Op 1</td><td rowspan=1 colspan=1>Op 2</td><td rowspan=1 colspan=1>Site</td><td rowspan=1 colspan=1>Op 1</td><td rowspan=1 colspan=1>Op 2</td><td rowspan=1 colspan=1>Site</td></tr><tr><td rowspan=1 colspan=1>Blank</td><td rowspan=1 colspan=1>100%a(10/10)</td><td rowspan=1 colspan=1>100%(10/10)</td><td rowspan=1 colspan=1>100%(20/20)</td><td rowspan=1 colspan=1>100%(10/10)</td><td rowspan=1 colspan=1>100%(10/10)</td><td rowspan=1 colspan=1>100%(20/20)</td><td rowspan=1 colspan=1>100%a(4/4)</td><td rowspan=1 colspan=1>100%a(6/6)</td><td rowspan=1 colspan=1>100%(10/10)</td><td rowspan=1 colspan=1>100%(50/50)</td></tr><tr><td rowspan=1 colspan=1>NC</td><td rowspan=1 colspan=1>100%b(10/10)</td><td rowspan=1 colspan=1>100%(10/10)</td><td rowspan=1 colspan=1>100%(20/20)</td><td rowspan=1 colspan=1>100%b(5/5)</td><td rowspan=1 colspan=1>100%b(5/5)</td><td rowspan=1 colspan=1>100%(10/10)</td><td rowspan=1 colspan=1>100%(10/10)</td><td rowspan=1 colspan=1>100%(10/10)</td><td rowspan=1 colspan=1>100%(20/20)</td><td rowspan=1 colspan=1>100%(50/50)</td></tr><tr><td rowspan=1 colspan=1>IMP Low Pos</td><td rowspan=1 colspan=1>80.0%(8/10)</td><td rowspan=1 colspan=1>100%(10/10)</td><td rowspan=1 colspan=1>90.0%(18/20)</td><td rowspan=1 colspan=1>100%(10/10)</td><td rowspan=1 colspan=1>100%(10/10)</td><td rowspan=1 colspan=1>100%(20/20)</td><td rowspan=1 colspan=1>100%(10/10)</td><td rowspan=1 colspan=1>100%(10/10)</td><td rowspan=1 colspan=1>100%(20/20)</td><td rowspan=1 colspan=1>96.7%(58/60)</td></tr><tr><td rowspan=1 colspan=1>IMP Mod Pos</td><td rowspan=1 colspan=1>90.0%(9/10)</td><td rowspan=1 colspan=1>100%(10/10)</td><td rowspan=1 colspan=1>95.0%(19/20)</td><td rowspan=1 colspan=1>100%(10/10)</td><td rowspan=1 colspan=1>100%(10/10)</td><td rowspan=1 colspan=1>100%(20/20)</td><td rowspan=1 colspan=1>100%(10/10)</td><td rowspan=1 colspan=1>100%(10/10)</td><td rowspan=1 colspan=1>100%(20/20)</td><td rowspan=1 colspan=1>98.3%(59/60)</td></tr><tr><td rowspan=1 colspan=1>VIM Low Pos</td><td rowspan=1 colspan=1>90.0%(9/10)</td><td rowspan=1 colspan=1>100%(10/10)</td><td rowspan=1 colspan=1>95%(19/20)</td><td rowspan=1 colspan=1>100%(10/10)</td><td rowspan=1 colspan=1>100%(10/10)</td><td rowspan=1 colspan=1>100%(20/20)</td><td rowspan=1 colspan=1>100%(10/10)</td><td rowspan=1 colspan=1>100%(10/10)</td><td rowspan=1 colspan=1>100%(20/20)</td><td rowspan=1 colspan=1>98.3%(59/60)</td></tr><tr><td rowspan=1 colspan=1>VIM Mod Pos</td><td rowspan=1 colspan=1>100%(10/10)</td><td rowspan=1 colspan=1>100%(10/10)</td><td rowspan=1 colspan=1>100%(20/20)</td><td rowspan=1 colspan=1>100%(10/10)</td><td rowspan=1 colspan=1>100%(10/10)</td><td rowspan=1 colspan=1>100%(20/20)</td><td rowspan=1 colspan=1>100%(10/10)</td><td rowspan=1 colspan=1>100%(10/10)</td><td rowspan=1 colspan=1>100%(20/20)</td><td rowspan=1 colspan=1>100%(60/60)</td></tr><tr><td rowspan=1 colspan=1>NDM Low Pos</td><td rowspan=1 colspan=1>100%(10/10)</td><td rowspan=1 colspan=1>90.0%(9/10)</td><td rowspan=1 colspan=1>95%(19/20)</td><td rowspan=1 colspan=1>100%(10/10)</td><td rowspan=1 colspan=1>100%(10/10)</td><td rowspan=1 colspan=1>100%(20/20)</td><td rowspan=1 colspan=1>100%(10/10)</td><td rowspan=1 colspan=1>100%(10/10)</td><td rowspan=1 colspan=1>100%(20/20)</td><td rowspan=1 colspan=1>98.3%(59/60)</td></tr><tr><td rowspan=1 colspan=1>NDM Mod Pos</td><td rowspan=1 colspan=1>100%(10/10)</td><td rowspan=1 colspan=1>100%(10/10)</td><td rowspan=1 colspan=1>100%(20/20)</td><td rowspan=1 colspan=1>100%(10/10)</td><td rowspan=1 colspan=1>100%(10/10)</td><td rowspan=1 colspan=1>100%(20/20)</td><td rowspan=1 colspan=1>100%(10/10)</td><td rowspan=1 colspan=1>100%(10/10)</td><td rowspan=1 colspan=1>100%(20/20)</td><td rowspan=1 colspan=1>100%(60/60)</td></tr><tr><td rowspan=1 colspan=1>KPC LowPos</td><td rowspan=1 colspan=1>100%(10/10)</td><td rowspan=1 colspan=1>100%(10/10)</td><td rowspan=1 colspan=1>100%(20/20)</td><td rowspan=1 colspan=1>100%(10/10)</td><td rowspan=1 colspan=1>100%(10/10)</td><td rowspan=1 colspan=1>100%(20/20)</td><td rowspan=1 colspan=1>100%(10/10)</td><td rowspan=1 colspan=1>100%(10/10)</td><td rowspan=1 colspan=1>100%(20/20)</td><td rowspan=1 colspan=1>100%(60/60)</td></tr><tr><td rowspan=1 colspan=1>KPC Mod Pos</td><td rowspan=1 colspan=1>100%(10/10)</td><td rowspan=1 colspan=1>100%(10/10)</td><td rowspan=1 colspan=1>100%(20/20)</td><td rowspan=1 colspan=1>100%(10/10)</td><td rowspan=1 colspan=1>100%(10/10)</td><td rowspan=1 colspan=1>100%(20/20)</td><td rowspan=1 colspan=1>100%(10/10)</td><td rowspan=1 colspan=1>100%(10/10)</td><td rowspan=1 colspan=1>100%(20/20)</td><td rowspan=1 colspan=1>100%(60/60)</td></tr><tr><td rowspan=2 colspan=1>OXA-48 Low Pos</td><td rowspan=2 colspan=1>100%(10/10)</td><td rowspan=2 colspan=1>100%(10/10)</td><td rowspan=2 colspan=1>100%(20/20)</td><td rowspan=2 colspan=1>100%(10/10)</td><td rowspan=2 colspan=1>100%(10/10)</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=2 colspan=1>100%(10/10)</td><td rowspan=2 colspan=1>100%(20/20)</td><td rowspan=2 colspan=1>100%(60/60)</td></tr><tr><td rowspan=1 colspan=1>(20/20)</td><td rowspan=1 colspan=1>(10/10)</td></tr><tr><td rowspan=1 colspan=1>OXA-48 Mod Pos</td><td rowspan=1 colspan=1>100%c(10/10)</td><td rowspan=1 colspan=1>100%(10/10)</td><td rowspan=1 colspan=1>100%(20/20)</td><td rowspan=1 colspan=1>100%c(10/10)</td><td rowspan=1 colspan=1>100%(10/10)</td><td rowspan=1 colspan=1>100%(20/20)</td><td rowspan=1 colspan=1>100%c(10/10)</td><td rowspan=1 colspan=1>100%c(10/10)</td><td rowspan=1 colspan=1>100%(20/20)</td><td rowspan=1 colspan=1>100%(60/60)</td></tr></table>

a Site 3 had fewer Blank samples tested and (1) UNR reported; At Site 1, (2) samples were excluded due to insufficient information and (1) UNR result was reported. All UNR samples were repeated with valid results. b Site 2 had fewer NC samples tested. At Site 1, (2) samples were excluded due to insufficient information. c With OXA-48 at the moderate level, multiple samples flagged positive for KPC across multiple sites (14 of 60 total samples); this was linked to contamination of the batch. No KPC positives were observed in the low positive samples of OXA-48.

Table 3. Summary of Results from the Precision Study for the BD MAX Check-Points CPO Assay.   

<table><tr><td rowspan=2 colspan=1>Resistance Gene</td><td rowspan=1 colspan=2>Lot 1</td><td rowspan=2 colspan=1>% TotalAgreement bySample</td></tr><tr><td rowspan=1 colspan=1>Op 1</td><td rowspan=1 colspan=1>Op 2</td></tr><tr><td rowspan=1 colspan=1>Blank</td><td rowspan=1 colspan=1>100%(12/12)</td><td rowspan=1 colspan=1>100%(12/12)</td><td rowspan=1 colspan=1>100%(24/24)</td></tr><tr><td rowspan=1 colspan=1>NC</td><td rowspan=1 colspan=1>100%(24/24)</td><td rowspan=1 colspan=1>100%(24/24)</td><td rowspan=1 colspan=1>100%(48/48)</td></tr><tr><td rowspan=1 colspan=1>IMP Low Pos</td><td rowspan=1 colspan=1>100%(24/24)</td><td rowspan=1 colspan=1>100%(24/24)</td><td rowspan=1 colspan=1>100%(48/48)</td></tr><tr><td rowspan=1 colspan=1>IMP Mod Pos</td><td rowspan=1 colspan=1>100%(24/24)</td><td rowspan=1 colspan=1>100%(24/24)</td><td rowspan=1 colspan=1>100%(48/48)</td></tr><tr><td rowspan=1 colspan=1>VIM Low Pos</td><td rowspan=1 colspan=1>95.8%(23/24)</td><td rowspan=1 colspan=1>100%(24/24)</td><td rowspan=1 colspan=1>97.9%(47/48)</td></tr><tr><td rowspan=1 colspan=1>VIM Mod Pos</td><td rowspan=1 colspan=1>100%(24/24)</td><td rowspan=1 colspan=1>100%(24/24)</td><td rowspan=1 colspan=1>100%(48/48)</td></tr><tr><td rowspan=1 colspan=1>NDM Low Pos</td><td rowspan=1 colspan=1>100%a(24/24)</td><td rowspan=1 colspan=1>100%(24/24)</td><td rowspan=1 colspan=1>100%(48/48)</td></tr><tr><td rowspan=1 colspan=1>NDM Mod Pos</td><td rowspan=1 colspan=1>100%(24/24)</td><td rowspan=1 colspan=1>100%(24/24)</td><td rowspan=1 colspan=1>100%(48/48)</td></tr><tr><td rowspan=1 colspan=1>KPC Low Pos</td><td rowspan=1 colspan=1>100%(24/24)</td><td rowspan=1 colspan=1>100%(24/24)</td><td rowspan=1 colspan=1>100%(48/48)</td></tr><tr><td rowspan=1 colspan=1>KPC Mod Pos</td><td rowspan=1 colspan=1>95.8%(23/24)</td><td rowspan=1 colspan=1>100%(24/24)</td><td rowspan=1 colspan=1>97.9%(47/48)</td></tr><tr><td rowspan=1 colspan=1>OXA-48 Low Pos</td><td rowspan=1 colspan=1>100%b(24/24)</td><td rowspan=1 colspan=1>95.8%(23/24)</td><td rowspan=1 colspan=1>97.9%(47/48)</td></tr><tr><td rowspan=1 colspan=1>OXA-48 Mod Pos</td><td rowspan=1 colspan=1>100%c(24/24)</td><td rowspan=1 colspan=1>100%c(24/24)</td><td rowspan=1 colspan=1>100%(48/48)</td></tr></table>

a (1) UNR reported. The sample was repeated and yielded a valid result. b (1) UNR reported. The sample was repeated and yielded a valid result. c With OXA-48 at the moderate level, multiple samples flagged positive for KPC across multiple sites (8 of 48 total samples); this was linked to contamination of the batch. No KPC positives were observed in the low positive samples of OXA-48.

Table 4. Mean Ct Values and $\%$ CV of the Reproducibility Panel Samples   

<table><tr><td colspan="1" rowspan="3">SpikedOrganism(ResistanceGene)</td><td colspan="1" rowspan="3">Site</td><td colspan="1" rowspan="3">Level</td><td colspan="1" rowspan="2">ObservedResults/</td><td colspan="4" rowspan="1">Ct Value</td></tr><tr><td colspan="2" rowspan="1">Target</td><td colspan="2" rowspan="1">SPC1</td></tr><tr><td colspan="1" rowspan="1">ExpectedResults (%)</td><td colspan="1" rowspan="1">Mean</td><td colspan="1" rowspan="1">%CV</td><td colspan="1" rowspan="1">Mean</td><td colspan="1" rowspan="1">%CV</td></tr><tr><td colspan="1" rowspan="8">K. pneumoniae(KPC)</td><td colspan="1" rowspan="2">1</td><td colspan="1" rowspan="1">Low</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">35.21</td><td colspan="1" rowspan="1">6.29</td><td colspan="1" rowspan="1">30.13</td><td colspan="1" rowspan="1">0.73</td></tr><tr><td colspan="1" rowspan="1">Moderate</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">32.85</td><td colspan="1" rowspan="1">10.02</td><td colspan="1" rowspan="1">30.22</td><td colspan="1" rowspan="1">1.68</td></tr><tr><td colspan="1" rowspan="2">2</td><td colspan="1" rowspan="1">Low</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">36.10</td><td colspan="1" rowspan="1">1.16</td><td colspan="1" rowspan="1">30.75</td><td colspan="1" rowspan="1">0.62</td></tr><tr><td colspan="1" rowspan="1">Moderate</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">29.66</td><td colspan="1" rowspan="1">14.57</td><td colspan="1" rowspan="1">31.23</td><td colspan="1" rowspan="1">1.42</td></tr><tr><td colspan="1" rowspan="2">3</td><td colspan="1" rowspan="1">Low</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">35.74</td><td colspan="1" rowspan="1">1.16</td><td colspan="1" rowspan="1">30.96</td><td colspan="1" rowspan="1">0.62</td></tr><tr><td colspan="1" rowspan="1">Moderate</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">28.97</td><td colspan="1" rowspan="1">14.57</td><td colspan="1" rowspan="1">30.80</td><td colspan="1" rowspan="1">1.42</td></tr><tr><td colspan="1" rowspan="2">Overall</td><td colspan="1" rowspan="1">Low</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">35.68</td><td colspan="1" rowspan="1">3.77</td><td colspan="1" rowspan="1">30.61</td><td colspan="1" rowspan="1">1.36</td></tr><tr><td colspan="1" rowspan="1">Moderate</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">30.49</td><td colspan="1" rowspan="1">13.49</td><td colspan="1" rowspan="1">30.75</td><td colspan="1" rowspan="1">1.90</td></tr><tr><td colspan="1" rowspan="8">E. cloacae(VIM)</td><td colspan="1" rowspan="2">1</td><td colspan="1" rowspan="1">Low</td><td colspan="1" rowspan="1">95.0%</td><td colspan="1" rowspan="1">33.62</td><td colspan="1" rowspan="1">1.93</td><td colspan="1" rowspan="1">30.18</td><td colspan="1" rowspan="1">0.66</td></tr><tr><td colspan="1" rowspan="1">Moderate</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">32.74</td><td colspan="1" rowspan="1">2.17</td><td colspan="1" rowspan="1">30.38</td><td colspan="1" rowspan="1">2.03</td></tr><tr><td colspan="1" rowspan="2">2</td><td colspan="1" rowspan="1">Low</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">33.89</td><td colspan="1" rowspan="1">1.08</td><td colspan="1" rowspan="1">30.68</td><td colspan="1" rowspan="1">0.71</td></tr><tr><td colspan="1" rowspan="1">Moderate</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">32.35</td><td colspan="1" rowspan="1">1.08</td><td colspan="1" rowspan="1">31.12</td><td colspan="1" rowspan="1">0.92</td></tr><tr><td colspan="1" rowspan="2">3</td><td colspan="1" rowspan="1">Low</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">33.36</td><td colspan="1" rowspan="1">1.17</td><td colspan="1" rowspan="1">31.10</td><td colspan="1" rowspan="1">1.04</td></tr><tr><td colspan="1" rowspan="1">Moderate</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">32.45</td><td colspan="1" rowspan="1">1.10</td><td colspan="1" rowspan="1">31.12</td><td colspan="1" rowspan="1">0.92</td></tr><tr><td colspan="1" rowspan="2">Overall</td><td colspan="1" rowspan="1">Low</td><td colspan="1" rowspan="1">98.3%</td><td colspan="1" rowspan="1">33.622</td><td colspan="1" rowspan="1">1.56</td><td colspan="1" rowspan="1">30.65</td><td colspan="1" rowspan="1">1.47</td></tr><tr><td colspan="1" rowspan="1">Moderate</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">32.51</td><td colspan="1" rowspan="1">1.60</td><td colspan="1" rowspan="1">30.87</td><td colspan="1" rowspan="1">1.77</td></tr><tr><td colspan="1" rowspan="8">E. coli(IMP)</td><td colspan="1" rowspan="2">1</td><td colspan="1" rowspan="1">Low</td><td colspan="1" rowspan="1">90.0%</td><td colspan="1" rowspan="1">32.232</td><td colspan="1" rowspan="1">2.94</td><td colspan="1" rowspan="1">30.19</td><td colspan="1" rowspan="1">0.59</td></tr><tr><td colspan="1" rowspan="1">Moderate</td><td colspan="1" rowspan="1">95.0%</td><td colspan="1" rowspan="1">31.25</td><td colspan="1" rowspan="1">3.51</td><td colspan="1" rowspan="1">30.05</td><td colspan="1" rowspan="1">0.50</td></tr><tr><td colspan="1" rowspan="2">2</td><td colspan="1" rowspan="1">Low</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">32.39</td><td colspan="1" rowspan="1">1.51</td><td colspan="1" rowspan="1">30.74</td><td colspan="1" rowspan="1">0.72</td></tr><tr><td colspan="1" rowspan="1">Moderate</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">31.08</td><td colspan="1" rowspan="1">2.28</td><td colspan="1" rowspan="1">31.01</td><td colspan="1" rowspan="1">0.98</td></tr><tr><td colspan="1" rowspan="2">3</td><td colspan="1" rowspan="1">Low</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">31.87</td><td colspan="1" rowspan="1">1.28</td><td colspan="1" rowspan="1">31.07</td><td colspan="1" rowspan="1">0.97</td></tr><tr><td colspan="1" rowspan="1">Moderate</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">30.91</td><td colspan="1" rowspan="1">1.50</td><td colspan="1" rowspan="1">31.12</td><td colspan="1" rowspan="1">0.83</td></tr><tr><td colspan="1" rowspan="2">Overall</td><td colspan="1" rowspan="1">Low</td><td colspan="1" rowspan="1">96.7%</td><td colspan="1" rowspan="1">32.152</td><td colspan="1" rowspan="1">2.09</td><td colspan="1" rowspan="1">30.67</td><td colspan="1" rowspan="1">1.42</td></tr><tr><td colspan="1" rowspan="1">Moderate</td><td colspan="1" rowspan="1">98.3%</td><td colspan="1" rowspan="1">31.072</td><td colspan="1" rowspan="1">2.55</td><td colspan="1" rowspan="1">30.72</td><td colspan="1" rowspan="1">1.77</td></tr><tr><td colspan="1" rowspan="8">K. pneumoniae(OXA-48)</td><td colspan="1" rowspan="2">1</td><td colspan="1" rowspan="1">Low</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">32.93</td><td colspan="1" rowspan="1">1.73</td><td colspan="1" rowspan="1">30.03</td><td colspan="1" rowspan="1">0.78</td></tr><tr><td colspan="1" rowspan="1">Moderate</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">31.99</td><td colspan="1" rowspan="1">1.18</td><td colspan="1" rowspan="1">30.03</td><td colspan="1" rowspan="1">0.46</td></tr><tr><td colspan="1" rowspan="2">2</td><td colspan="1" rowspan="1">Low</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">32.93</td><td colspan="1" rowspan="1">1.73</td><td colspan="1" rowspan="1">30.68</td><td colspan="1" rowspan="1">0.78</td></tr><tr><td colspan="1" rowspan="1">Moderate</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">31.57</td><td colspan="1" rowspan="1">1.18</td><td colspan="1" rowspan="1">31.09</td><td colspan="1" rowspan="1">0.46</td></tr><tr><td colspan="1" rowspan="2">3</td><td colspan="1" rowspan="1">Low</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">33.03</td><td colspan="1" rowspan="1">0.99</td><td colspan="1" rowspan="1">30.82</td><td colspan="1" rowspan="1">0.87</td></tr><tr><td colspan="1" rowspan="1">Moderate</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">31.98</td><td colspan="1" rowspan="1">1.10</td><td colspan="1" rowspan="1">30.80</td><td colspan="1" rowspan="1">1.07</td></tr><tr><td colspan="1" rowspan="2">Overall</td><td colspan="1" rowspan="1">Low</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">32.96</td><td colspan="1" rowspan="1">1.36</td><td colspan="1" rowspan="1">30.51</td><td colspan="1" rowspan="1">1.40</td></tr><tr><td colspan="1" rowspan="1">Moderate</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">31.84</td><td colspan="1" rowspan="1">1.34</td><td colspan="1" rowspan="1">30.64</td><td colspan="1" rowspan="1">1.74</td></tr><tr><td colspan="1" rowspan="8">K. pneumoniae(NDM)</td><td colspan="1" rowspan="2">−</td><td colspan="1" rowspan="1">Low</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">34.58</td><td colspan="1" rowspan="1">4.66</td><td colspan="1" rowspan="1">30.46</td><td colspan="1" rowspan="1">3.14</td></tr><tr><td colspan="1" rowspan="1">Moderate</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">33.735</td><td colspan="1" rowspan="1">3.57</td><td colspan="1" rowspan="1">30.14</td><td colspan="1" rowspan="1">0.54</td></tr><tr><td colspan="1" rowspan="2">2</td><td colspan="1" rowspan="1">Low</td><td colspan="1" rowspan="1">95.0%</td><td colspan="1" rowspan="1">35.922</td><td colspan="1" rowspan="1">1.85</td><td colspan="1" rowspan="1">30.77</td><td colspan="1" rowspan="1">1.17</td></tr><tr><td colspan="1" rowspan="1">Moderate</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">33.28</td><td colspan="1" rowspan="1">1.88</td><td colspan="1" rowspan="1">31.19</td><td colspan="1" rowspan="1">0.74</td></tr><tr><td colspan="1" rowspan="2">3</td><td colspan="1" rowspan="1">Low</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">35.57</td><td colspan="1" rowspan="1">1.26</td><td colspan="1" rowspan="1">30.80</td><td colspan="1" rowspan="1">1.04</td></tr><tr><td colspan="1" rowspan="1">Moderate</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">34.47</td><td colspan="1" rowspan="1">1.21</td><td colspan="1" rowspan="1">30.82</td><td colspan="1" rowspan="1">0.89</td></tr><tr><td colspan="1" rowspan="2">Overall</td><td colspan="1" rowspan="1">Low</td><td colspan="1" rowspan="1">98.3%</td><td colspan="1" rowspan="1">35.362</td><td colspan="1" rowspan="1">3.27</td><td colspan="1" rowspan="1">30.68</td><td colspan="1" rowspan="1">2.04</td></tr><tr><td colspan="1" rowspan="1">Moderate</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">33.83</td><td colspan="1" rowspan="1">2.79</td><td colspan="1" rowspan="1">30.72</td><td colspan="1" rowspan="1">1.61</td></tr><tr><td colspan="1" rowspan="4">E. coli(NC)</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">30.32</td><td colspan="1" rowspan="1">0.64</td></tr><tr><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">30.86</td><td colspan="1" rowspan="1">0.53</td></tr><tr><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">31.19</td><td colspan="1" rowspan="1">1.88</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">30.69</td><td colspan="1" rowspan="1">1.69</td></tr><tr><td colspan="1" rowspan="4">Blank(None)</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">30.04</td><td colspan="1" rowspan="1">0.53</td></tr><tr><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">31.02</td><td colspan="1" rowspan="1">1.27</td></tr><tr><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">31.08</td><td colspan="1" rowspan="1">0.69</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">30.79</td><td colspan="1" rowspan="1">1.74</td></tr></table>

1 If a positive result for the target is obtained, detection of the SPC is not required for a valid result to be reported. $^ 2 C t$ of target gene for specimens tested negative not taken into account to calculate average and $\% C V .$ . $C t$ of SPC for specimens tested negative taken into account to calculate average and $\% C V .$ .

The results of the study demonstrated acceptable reproducibility from site-to-site at target levels close to the limit of detection (LoD) of the assay. The Lot-to-Lot Study and the Precision Study also showed acceptable performance of the BD MAX CheckPoints CPO Assay.

b. Linearity/assay reportable range: Not Applicable   
c. Traceability, Stability, Expected values (controls, calibrators, or methods):

# External Controls

Commercially-available external controls can also be run in accordance with local, state, and federal accrediting organizations, as applicable. External processing controls consisting of an External Positive Control and an External Negative Control were included for each run. On each testing day, one negative control and one individual bacterial positive control were tested. Of the 431 external control samples run, $9 9 . 1 \%$ (427/431) gave a valid result on the first attempt. Table 5 below shows a list of the external controls prepared and tested during the clinical study.

Table 5. External Controls During Testing   

<table><tr><td rowspan=1 colspan=1>Target Gene</td><td rowspan=1 colspan=1>External Control Strain</td><td rowspan=1 colspan=1>Final Dilution from0.5 McFarland</td><td rowspan=1 colspan=1>Observed/Expected Results</td></tr><tr><td rowspan=1 colspan=1>KPC</td><td rowspan=1 colspan=1>Klebsiella pneumoniae (NCTC-13438)</td><td rowspan=1 colspan=1>1/1,000</td><td rowspan=1 colspan=1>48/49</td></tr><tr><td rowspan=1 colspan=1>VIM</td><td rowspan=1 colspan=1>Pseudomonas aeruginosa (NCTC-13437)</td><td rowspan=1 colspan=1>1/5,000</td><td rowspan=1 colspan=1>43/44a,b</td></tr><tr><td rowspan=1 colspan=1>IMP</td><td rowspan=1 colspan=1>Escherichia coli (NCTC 13476)</td><td rowspan=1 colspan=1>1/7,000</td><td rowspan=1 colspan=1>38/38</td></tr><tr><td rowspan=1 colspan=1>OXA-48</td><td rowspan=1 colspan=1>Klebsiella pneumoniae (NCTC-13442)</td><td rowspan=1 colspan=1>1/10,000</td><td rowspan=1 colspan=1>48/49</td></tr><tr><td rowspan=1 colspan=1>NDM</td><td rowspan=1 colspan=1>Klebsiella pneumoniae (NCTC-13443)</td><td rowspan=1 colspan=1>1/400</td><td rowspan=1 colspan=1>37/37</td></tr><tr><td rowspan=1 colspan=1>Negative Control</td><td rowspan=1 colspan=1>Escherichia coli (ATCC 25922)</td><td rowspan=1 colspan=1>1/10</td><td rowspan=1 colspan=1>209/210c</td></tr></table>

a (1) UNR b Enterobacter cloacae was used as quality control for VIM (1 sample) c (1) IND, (1) INC, (1) UNR observed.

# Sample Processing Control (SPC)

The BD MAX Check-Points CPO Assay also includes an SPC that is provided in the Extraction Tube and subjected to extraction, concentration, and amplification steps. The SPC is a synthetic target DNA sequence that monitors for the presence of potential inhibitory substances as well as system or reagent failures. The SPC monitors the following:

The effectiveness of DNA capture, washing, elution and neutralization during the sample processing steps. • The integrity of the PCR reagents and the presence of inhibitory substances during the amplification and detection steps. The integrity of the system as a whole, including the pipetting steps, PCR thermocycling, and signal detection.

In the absence of SPC amplification, non-positive results from a given Master Mix are identified as UNR.

# Stability after Rectal Swabs Resuspended in Sample Buffer Tube (SBT)

The purpose of this study was to determine the stability of samples in SBT to support repeat testing from the same sample. Two positive specimen pools with four organisms [Positive Set #1:KPC, VIM, OXA-48, NDM and Positive Set #2: KPC, IMP, OXA-48, NDM] were spiked into pooled negative rectal swab matrix from healthy human donors at $3 \mathbf { x }$ LoD and analyzed in replicates of 5 after sample storage in SBT at $2 { - } 8 ~ ^ { \circ } \mathrm { C }$ for 0, 8, 24, and 48 hrs and at $2 0 { - } 2 5 \ ^ { \circ } \mathrm { C }$ for 0, 24 and 48 hrs in accordance with the IFU. Due to performance errors (run-related issue), one run was rejected and repeated at room temperature for the 24 hrs time point. Only two tests resulted in UNR [(1) at $\scriptstyle { \mathrm { { t } = 0 } }$ , $2 5 ^ { \circ } \mathrm { C }$ and (1) at $\scriptstyle { \mathrm { t } } = 4 8$ hrs, $2 { - } 8 ^ { \circ } \mathrm { C } )$ ] but were repeated with valid results. External positive and negative controls were included with each run. It was concluded that acceptance criteria were met for each analyte/specimen at all time points analyzed. The sample stability in SBT was set at 48 hrs at $2 { - } 2 5 ^ { \circ } \mathrm { C }$ .

# d. Detection limit:

A study was conducted to determine the Limit of Detection (LoD) of the BD MAX Check-Points CPO Assay for organisms harboring the blaKPC, blaNDM, blaVIM, blaOXA48, and blaIMP gene sequence targets seeded into rectal swab background matrix. The LoD was defined as the lowest concentration (reported as cells/ml in SBT) of target at which 10/10 replicates gave a positive test result. Pre-screened negative rectal swab background matrix was used to prepare samples. Two strains harboring each assay target were tested. Bacterial cell suspensions of each strain were prepared from culture and quantified prior to inclusion in this study. Serial dilutions in negative rectal matrix were prepared for all strains at test concentrations expected to comprise the LoD for each carbapenemase target gene. To estimate the LoD, replicates of 10 for each test concentration were evaluated using 3 BD MAX instruments and 3 lots of reagents and consumables. The LoD (Table 6) was confirmed by testing 20 replicates for each strain at the estimated LoD. External positive and negative controls were included with each run.

In the LoD Study, $0 . 7 \%$ samples (9/1309) provided UNR or IND results with the BD MAX Check-Points CPO Assay (5 UNR, 4 IND). All samples were repeated yielding valid test results. (4) external positive controls gave negative results, which was linked to a sample preparation issue. Retesting with fresh preparations of these controls yielded the expected result.

Table 6. Estimated LoD Results for the Test Panel   

<table><tr><td rowspan=1 colspan=1>Target</td><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>Species</td><td rowspan=1 colspan=1>CFU/ml ofSBT</td><td rowspan=1 colspan=1>% Detection</td></tr><tr><td rowspan=2 colspan=1>KPC</td><td rowspan=1 colspan=1>CP254 (NCTC-134380)</td><td rowspan=1 colspan=1>Klebsiella pneumoniae</td><td rowspan=1 colspan=1>2005</td><td rowspan=1 colspan=1>95%</td></tr><tr><td rowspan=1 colspan=1>CP365</td><td rowspan=1 colspan=1>Klebsiella pneumoniae</td><td rowspan=1 colspan=1>3560</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=2 colspan=1>VIM</td><td rowspan=1 colspan=1>CP260(NCTC-1347)</td><td rowspan=1 colspan=1>Pseudomonas aeruginosa</td><td rowspan=1 colspan=1>159</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>CP433</td><td rowspan=1 colspan=1>Enterobacter cloacae</td><td rowspan=1 colspan=1>520</td><td rowspan=1 colspan=1>95%</td></tr><tr><td rowspan=2 colspan=1>IMP</td><td rowspan=1 colspan=1>CP253(NCTC-14376)</td><td rowspan=1 colspan=1>Escherichia coli</td><td rowspan=1 colspan=1>319</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>CP149</td><td rowspan=1 colspan=1>Klebsiella pneumoniae</td><td rowspan=1 colspan=1>144</td><td rowspan=1 colspan=1>95%</td></tr><tr><td rowspan=2 colspan=1>OXA-48</td><td rowspan=1 colspan=1>CP258(NCTC-13442)</td><td rowspan=1 colspan=1>Klebsiella pneumoniae</td><td rowspan=1 colspan=1>229</td><td rowspan=1 colspan=1>95%</td></tr><tr><td rowspan=1 colspan=1>CP411</td><td rowspan=1 colspan=1>Escherichia coli</td><td rowspan=1 colspan=1>902</td><td rowspan=1 colspan=1>95%</td></tr><tr><td rowspan=2 colspan=1>NDM</td><td rowspan=1 colspan=1>CP259(NCTC-13443)</td><td rowspan=1 colspan=1>Klebsiella pneumoniae</td><td rowspan=1 colspan=1>4774</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>CP184</td><td rowspan=1 colspan=1>Escherichia coli</td><td rowspan=1 colspan=1>4492</td><td rowspan=1 colspan=1>95%</td></tr></table>

# Analytical Reactivity

An Inclusivity Study was conducted to test reactivity of the BD MAX Check-Points CPO Assay with 93 well-characterized bacterial isolates. The panel consisted of bacterial isolates that harbor the following molecular resistance marker groups: (17) blaKPC isolates, (17) blaNDM isolates, (17) blaVIM isolates, (20) blaOXA-48 type isolates, (18) blaIMP isolates, and (4) strains containing two carbapenemase gene targets. Strains were seeded at $3 \mathbf { x }$ LoD and tested in pooled negative rectal swab matrix. For a list of strains tested during the Analytical Reactivity Study, please refer to Table 7 below.

The BD MAX Check-Points CPO Assay correctly identified 90 of the 93 strains tested at/near 3x LoD. Three strains—an Enterobacter cloacae with IMP-34, a Pseudomonas aeruginosa with IMP-4, and a Klebsiella pneumoniae with NDM-1—were detected when re-tested at $1 0 \mathrm { x } \ \mathrm { L o D }$ (indicated with asterisks in Table 7). All results were in accordance with in silico analysis. For a summary of these results, please refer to Table 8 below.

Table 7. Strains Tested in Analytical Reactivity Study   

<table><tr><td colspan="1" rowspan="1">Strain ID</td><td colspan="1" rowspan="1">Organism</td><td colspan="1" rowspan="1">Confirmed GeneticResistance Marker</td></tr><tr><td colspan="3" rowspan="1">KPC Isolates</td></tr><tr><td colspan="1" rowspan="1">CP-098</td><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">KPC-2</td></tr><tr><td colspan="1" rowspan="1">CP-196*</td><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">KPC-2</td></tr><tr><td colspan="1" rowspan="1">CP-197</td><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">KPC-2</td></tr><tr><td colspan="1" rowspan="1">CP-231</td><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">KPC-2</td></tr><tr><td colspan="1" rowspan="1">CP-232</td><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">KPC-2</td></tr><tr><td colspan="1" rowspan="1">CP-236</td><td colspan="1" rowspan="1">Enterobacter asburiae</td><td colspan="1" rowspan="1">KPC-2</td></tr><tr><td colspan="1" rowspan="1">CP-237</td><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">KPC-2</td></tr><tr><td colspan="1" rowspan="1">CP-238</td><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">KPC-2</td></tr><tr><td colspan="1" rowspan="1">CP-365</td><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">KPC-2</td></tr><tr><td colspan="1" rowspan="1">CP-395</td><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">KPC-2</td></tr><tr><td colspan="1" rowspan="1">CP-396</td><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">KPC-2</td></tr><tr><td colspan="1" rowspan="1">CP-254</td><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">KPC-3</td></tr><tr><td colspan="1" rowspan="1">CP-392</td><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">KPC-3</td></tr><tr><td colspan="1" rowspan="1">BD-18526</td><td colspan="1" rowspan="1">Serratia marcescens</td><td colspan="1" rowspan="1">KPC-untyped</td></tr><tr><td colspan="1" rowspan="1">BD-8443</td><td colspan="1" rowspan="1">Enterobacter aerogenes</td><td colspan="1" rowspan="1">KPC-untyped</td></tr><tr><td colspan="1" rowspan="1">BD-18019</td><td colspan="1" rowspan="1">Citrobacter freundii</td><td colspan="1" rowspan="1">KPC-untyped</td></tr><tr><td colspan="1" rowspan="1">BD-19517</td><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">KPC-untyped</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">NDM Isolates</td><td colspan="1" rowspan="1">cYanmaGellowb</td></tr><tr><td colspan="1" rowspan="1">CP-122</td><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">NDM-1</td></tr><tr><td colspan="1" rowspan="1">CP-157</td><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">NDM-1</td></tr><tr><td colspan="1" rowspan="1">CP-228</td><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">NDM</td></tr><tr><td colspan="1" rowspan="1">CP-259</td><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">NDM-1</td></tr><tr><td colspan="1" rowspan="1">CP-358</td><td colspan="1" rowspan="1">Providencia rettgeri</td><td colspan="1" rowspan="1">NDM-1</td></tr><tr><td colspan="1" rowspan="1">CP-383</td><td colspan="1" rowspan="1">Acinetobacter baumannii</td><td colspan="1" rowspan="1">NDM-1</td></tr><tr><td colspan="1" rowspan="1">CP-384</td><td colspan="1" rowspan="1">Acinetobacter pittii</td><td colspan="1" rowspan="1">NDM-1</td></tr><tr><td colspan="1" rowspan="1">CP-385</td><td colspan="1" rowspan="1">Enterobacter cloacae</td><td colspan="1" rowspan="1">NDM-1</td></tr><tr><td colspan="1" rowspan="1">CP-386</td><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">NDM-1</td></tr><tr><td colspan="1" rowspan="1">CP-387</td><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">NDM-1</td></tr><tr><td colspan="1" rowspan="1">CP-388</td><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">NDM-1</td></tr><tr><td colspan="1" rowspan="1">CP-389**</td><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">NDM-1</td></tr><tr><td colspan="1" rowspan="1">CP-390</td><td colspan="1" rowspan="1">Morganella morganii</td><td colspan="1" rowspan="1">NDM-1</td></tr><tr><td colspan="1" rowspan="1">CP-391</td><td colspan="1" rowspan="1">Providencia rettgeri</td><td colspan="1" rowspan="1">NDM-1</td></tr><tr><td colspan="1" rowspan="1">BD-18545</td><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">NDM-7</td></tr><tr><td colspan="1" rowspan="1">BD-19229</td><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">NDM-6</td></tr><tr><td colspan="1" rowspan="1">BD-19242</td><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">NDM5</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">VIM Isolates</td><td colspan="1" rowspan="1">cYanmaGellowb</td></tr><tr><td colspan="1" rowspan="1">CP-260</td><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">VIM-2</td></tr><tr><td colspan="1" rowspan="1">CP-161</td><td colspan="1" rowspan="1">Enterobacter cloacae</td><td colspan="1" rowspan="1">VIM-1</td></tr><tr><td colspan="1" rowspan="1">CP-167</td><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">VIM-19</td></tr><tr><td colspan="1" rowspan="1">CP-169</td><td colspan="1" rowspan="1">Acinetobacter junii</td><td colspan="1" rowspan="1">VIM-4</td></tr><tr><td colspan="1" rowspan="1">CP-240</td><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">VIM-26</td></tr><tr><td colspan="1" rowspan="1">CP-241</td><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">VIM-1</td></tr><tr><td colspan="1" rowspan="1">CP-244</td><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">VIM-19</td></tr><tr><td colspan="1" rowspan="1">CP-246</td><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">VIM-4</td></tr><tr><td colspan="1" rowspan="1">CP-248</td><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">VIM-1</td></tr><tr><td colspan="1" rowspan="1">CP-374</td><td colspan="1" rowspan="1">Aeromonas caviae</td><td colspan="1" rowspan="1">VIM-4</td></tr><tr><td colspan="1" rowspan="1">CP-375</td><td colspan="1" rowspan="1">Citrobacter braakii</td><td colspan="1" rowspan="1">VIM-1</td></tr><tr><td colspan="1" rowspan="1">CP-378*</td><td colspan="1" rowspan="1">Klebsiella oxytoca</td><td colspan="1" rowspan="1">VIM-1</td></tr><tr><td colspan="1" rowspan="1">CP-381</td><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">VIM-4</td></tr><tr><td colspan="1" rowspan="1">CP-165</td><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">VIM-2</td></tr><tr><td colspan="1" rowspan="1">CP-245</td><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">VIM-2</td></tr><tr><td colspan="1" rowspan="1">CP-377</td><td colspan="1" rowspan="1">Enterobacter cloacae</td><td colspan="1" rowspan="1">VIM-31</td></tr><tr><td colspan="1" rowspan="1">CP-379</td><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">VIM-27</td></tr><tr><td colspan="2" rowspan="1">OXA-48 and OXA-48 like isolates</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">CP-362</td><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">OXA-162</td></tr><tr><td colspan="1" rowspan="1">CP-363</td><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">OXA-163</td></tr><tr><td colspan="1" rowspan="1">CP-361</td><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">OXA-181</td></tr><tr><td colspan="1" rowspan="1">CP-427</td><td colspan="1" rowspan="1">Citrobacter freundii</td><td colspan="1" rowspan="1">OXA-181</td></tr><tr><td colspan="1" rowspan="1">CP-360</td><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">OXA-204</td></tr><tr><td colspan="1" rowspan="1">CP-364</td><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">OXA-232</td></tr><tr><td colspan="1" rowspan="1">CP-411</td><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">OXA-244</td></tr><tr><td colspan="1" rowspan="1">CP-158</td><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">OXA-48</td></tr><tr><td colspan="1" rowspan="1">CP-159</td><td colspan="1" rowspan="1">Enterobacter cloacae</td><td colspan="1" rowspan="1">OXA-48</td></tr><tr><td colspan="1" rowspan="1">CP-160</td><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">OXA-48</td></tr><tr><td colspan="1" rowspan="1">CP-179</td><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">OXA-48</td></tr><tr><td colspan="1" rowspan="1">CP-252</td><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">OXA-48</td></tr><tr><td colspan="1" rowspan="1">CP-258</td><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">OXA-48</td></tr><tr><td colspan="1" rowspan="1">CP-369*</td><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">OXA-48</td></tr><tr><td colspan="1" rowspan="1">CP-397</td><td colspan="1" rowspan="1">Citrobacter freundii</td><td colspan="1" rowspan="1">OXA-48</td></tr><tr><td colspan="1" rowspan="1">CP-398</td><td colspan="1" rowspan="1">Enterobacter aerogenes</td><td colspan="1" rowspan="1">OXA-48</td></tr><tr><td colspan="1" rowspan="1">CP-399</td><td colspan="1" rowspan="1">Enterobacter cloacae</td><td colspan="1" rowspan="1">OXA-48</td></tr><tr><td colspan="1" rowspan="1">CP-402</td><td colspan="1" rowspan="1">Klebsiella oxytoca</td><td colspan="1" rowspan="1">OXA-48</td></tr><tr><td colspan="1" rowspan="1">CP-403</td><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">OXA-48</td></tr><tr><td colspan="1" rowspan="1">CP-406</td><td colspan="1" rowspan="1">Enterobacter cloacae</td><td colspan="1" rowspan="1">OXA-48</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">IMP Isolates</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">CP-253</td><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">IMP-34</td></tr><tr><td colspan="1" rowspan="1">CP-149</td><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">IMP-1</td></tr><tr><td colspan="1" rowspan="1">CP-559</td><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">IMP-3</td></tr><tr><td colspan="1" rowspan="1">CP-551**</td><td colspan="1" rowspan="1">Enterobacter cloacae</td><td colspan="1" rowspan="1">IMP-34</td></tr><tr><td colspan="1" rowspan="1">CP-552</td><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">IMP-34</td></tr><tr><td colspan="1" rowspan="1">CP-553</td><td colspan="1" rowspan="1">Klebsiella oxytoca</td><td colspan="1" rowspan="1">IMP-34</td></tr><tr><td colspan="1" rowspan="1">CP-562</td><td colspan="1" rowspan="1">Citrobacter freundii</td><td colspan="1" rowspan="1">IMP-34</td></tr><tr><td colspan="1" rowspan="1">CP-563</td><td colspan="1" rowspan="1">Enterobacter cloacae</td><td colspan="1" rowspan="1">IMP-34</td></tr><tr><td colspan="1" rowspan="1">CP-151</td><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">IMP-4</td></tr><tr><td colspan="1" rowspan="1">CP-156</td><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">IMP-4</td></tr><tr><td colspan="1" rowspan="1">CP-548</td><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">IMP-26</td></tr><tr><td colspan="1" rowspan="1">CP-549</td><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">IMP-26</td></tr><tr><td colspan="1" rowspan="1">CP-550</td><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">IMP-26</td></tr><tr><td colspan="1" rowspan="1">CP-556</td><td colspan="1" rowspan="1">Enterobacter cloacae</td><td colspan="1" rowspan="1">IMP-4</td></tr><tr><td colspan="1" rowspan="1">CP-557</td><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">IMP-4</td></tr><tr><td colspan="1" rowspan="1">CP-558**</td><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">IMP-4</td></tr><tr><td colspan="1" rowspan="1">CP-153</td><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">IMP-7</td></tr><tr><td colspan="1" rowspan="1">CP-561</td><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">IMP-8</td></tr><tr><td colspan="3" rowspan="1">Isolates with More Than One Genetic Marker Target</td></tr><tr><td colspan="1" rowspan="1">CP-376</td><td colspan="1" rowspan="1">Citrobacter freundii</td><td colspan="1" rowspan="1">VIM-1/0XA-48</td></tr><tr><td colspan="1" rowspan="1">CP-555</td><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">IMP-34/VIM</td></tr><tr><td colspan="1" rowspan="1">CP-230</td><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">OXA-181/NDM</td></tr><tr><td colspan="1" rowspan="1">CP-554</td><td colspan="1" rowspan="1">Providencia stuartii</td><td colspan="1" rowspan="1">IMP-34/NDM</td></tr></table>

\* One of three tests resulted in unresolved (UNR), and was repeated from the same Sample Buffer Tube. \*\* Three bacterial strains (CP-389, $C P _ { \ast }$ -551 and CP-558) that did not result in $100 \%$ positive rate at $3 \ : x \ : L o D$ were repeated at $I 0 x L o D$ .

Table 8. Summary of Variants Detected by Wet Testing or Predicted to be Detected Based on In Silico Analysis.   

<table><tr><td rowspan=2 colspan=1>Marker(or TraditionalSubgroup)</td><td rowspan=1 colspan=3>Wet testing</td><td rowspan=2 colspan=1>Predicted to be detected based onin silico analysis</td></tr><tr><td rowspan=1 colspan=1>No. ofSampleswith Target</td><td rowspan=1 colspan=1>Type(s) Detected</td><td rowspan=1 colspan=1>Type(s) notDetected</td></tr><tr><td rowspan=1 colspan=1>KPC</td><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>KPC: 2, 3</td><td rowspan=1 colspan=1>cYanmaentaYellowb</td><td rowspan=1 colspan=1>KPC: 2-37</td></tr><tr><td rowspan=1 colspan=1>NDM</td><td rowspan=1 colspan=1>19</td><td rowspan=1 colspan=1>NDM: 1, 5, 6, 7</td><td rowspan=1 colspan=1>cYanmaentaYellowb</td><td rowspan=1 colspan=1>NDM: 1-24</td></tr><tr><td rowspan=1 colspan=1>VIM</td><td rowspan=1 colspan=1>19</td><td rowspan=1 colspan=1>VIM: 1, 2, 4, 19,26, 27, 31</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>VIM: 1-6, 8-52, 54, 56-60</td></tr><tr><td rowspan=1 colspan=1>OXA-48</td><td rowspan=1 colspan=1>22</td><td rowspan=1 colspan=1>OXA-48,162,163, 181, 204,232, 244</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>OXA-48, 162, 163, 181, 204, 232,244, 245, 370, 405, 438, 439, 484,505, 517, 519, 566</td></tr><tr><td rowspan=1 colspan=1>IMP</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>IMP: 1, 3, 4, 7, 8,26, 34</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>IMP: 1-4, 6-8, 10, 19, 20, 23-26, 30,34, 38, 40, 42, 43, 51, 52, 55, 59-61,66, 70, 73, 76-80</td></tr></table>

IMP-7 and OXA-163, which were not predicted to be detected by in silico analysis, were detected by BD MAX Check-Points CPO Assay after wet testing. Three (3) external positive controls gave negative results, which was linked to a sample

preparation issue. Retesting with fresh preparations of these controls yielded the expected result.

# e. Analytical specificity:

The analytical specificity of the BD MAX Check-Points CPO Assay was evaluated with a panel of 26 phylogenetically related species and other organisms likely to b found in rectal swab specimens. This panel included well-characterized bacteria confirmed as having:

• No carbapenemase gene   
• Different carbapenemase gene (not target of the assay)   
• Different $\beta$ -lactamase gene or Different antibiotic resistance determinant

Panel members were prepared by seeding bacterial cells into negative rectal swab matrix at a concentration ${ \geq } 1 0 ^ { 6 } \mathrm { C F U / m l }$ in triplicate before testing with the BD MAX Check-Points CPO Assay. Please refer to Table 9 below for the Analytical Specificity Study panel.

Table 9. Analytical Specificity Panel   

<table><tr><td colspan="1" rowspan="1">Strain ID</td><td colspan="1" rowspan="1">Species</td><td colspan="1" rowspan="1">ResistanceMechanism(s)</td></tr><tr><td colspan="1" rowspan="1">CP-575(CCUG-41359)</td><td colspan="1" rowspan="1">Campylobacter jejuni</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">CP-521</td><td colspan="1" rowspan="1">Citrobacter freundii</td><td colspan="1" rowspan="1">CTX-M9 ESBL</td></tr><tr><td colspan="1" rowspan="1">CP-338</td><td colspan="1" rowspan="1">Citrobacter braakii</td><td colspan="1" rowspan="1">GES Carbapenemase</td></tr><tr><td colspan="1" rowspan="1">CP-568(CCUG-37874)</td><td colspan="1" rowspan="1">Corynebacterium diphtheriae</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">CP-484</td><td colspan="1" rowspan="1">Enterobacter aerogenes</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">CP-034</td><td colspan="1" rowspan="1">Enterobacter cloacae</td><td colspan="1" rowspan="1">CTX-M9 ESBL</td></tr><tr><td colspan="1" rowspan="1">CP-573(CCUG-55879)</td><td colspan="1" rowspan="1">Enterococcus casseliflavus</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">CP-574(CCUG-9997)</td><td colspan="1" rowspan="1">Enterococcus faecalis</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">CP-048</td><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">CTX-M1 ESBL</td></tr><tr><td colspan="1" rowspan="1">CP-576(CCUG-17874)</td><td colspan="1" rowspan="1">Helicobacter pylori</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">CP-058</td><td colspan="1" rowspan="1">Klebsiella oxytoca</td><td colspan="1" rowspan="1">CTX-M9 ESBL</td></tr><tr><td colspan="1" rowspan="1">CP-012</td><td colspan="1" rowspan="1">Klebsiella pneumonia</td><td colspan="1" rowspan="1">SHV-ESBL</td></tr><tr><td colspan="1" rowspan="1">CP-570(CCUG-33548)</td><td colspan="1" rowspan="1">Listeria monocytogenes</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">CP-357</td><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">PER ESBL</td></tr><tr><td colspan="1" rowspan="1">CP-132</td><td colspan="1" rowspan="1">Salmonella typhimurium</td><td colspan="1" rowspan="1">pAmpC</td></tr><tr><td colspan="1" rowspan="1">CP-519</td><td colspan="1" rowspan="1">Raoultella sp.</td><td colspan="1" rowspan="1">SHV &amp; CTX-M9 ESBL</td></tr><tr><td colspan="1" rowspan="1">CP-571(CCUG-9128)</td><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">CP-250</td><td colspan="1" rowspan="1">Serratia marcescens</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">CP-009</td><td colspan="1" rowspan="1">Stenotrophomonas maltophilia</td><td colspan="1" rowspan="1">SHV &amp; CTX-M9 ESBL;pAmpC</td></tr><tr><td colspan="1" rowspan="1">CP-284</td><td colspan="1" rowspan="1">Acinetobacter baumannii</td><td colspan="1" rowspan="1">OXA-23 Carbapenemase</td></tr><tr><td colspan="1" rowspan="1">CP-503</td><td colspan="1" rowspan="1">Morganella morganii</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">CP-319</td><td colspan="1" rowspan="1">Providencia stuartii</td><td colspan="1" rowspan="1">VEB ESBL</td></tr><tr><td colspan="1" rowspan="1">CP-567(CCUG-6325)</td><td colspan="1" rowspan="1">Providencia alcalifaciens</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">CP-569(CCUG-29780)</td><td colspan="1" rowspan="1">Streptococcus agalactiae</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">CP-052</td><td colspan="1" rowspan="1">Proteus mirabilis</td><td colspan="1" rowspan="1">pAmpC</td></tr><tr><td colspan="1" rowspan="1">CP-440</td><td colspan="1" rowspan="1">Acinetobacter baumannii</td><td colspan="1" rowspan="1">OXA-58 Carbapenemase</td></tr></table>

Of the 26 potentially cross-reactive organisms tested, including organisms exhibiting antibiotic resistance mechanisms other than the presence of KPC, NDM, VIM, IMP and OXA-48 genes, none were detected with the BD MAX Check-Points CPO Assay. Of the 78 tests in rectal swab matrix, two $( 2 . 6 \% )$ tests yielded an UNR result. Both UNR were successfully repeated and were reported as negative for all targets as expected. All external positive and negative controls were correctly reported as expected.

A separate study was conducted to assess whether a high level of an assay target organism $\mathrm { ( > 1 0 ^ { 6 } C F U / m l ) }$ was capable of generating false positive (FP) signals using the BD MAX Check-Points CPO Assay. It was observed that the BD MAX CheckPoints Assay did not produce FP results from single target specimens tested at a high concentration.

# f. Assay cut-off:

The assay cut-off was defined by a 2-step process. Initially, an Assay Definition File (ADF) version was designed with software parameters assigned such that the defined cut-off gave optimal analytical sensitivity, i.e. increase of PCR cycle numbers did not further increase the analytical sensitivity or Limit of Detection (LoD). This was performed during development and used various dilutions of bacterial strains containing assay target genes spiked into negative rectal swab specimens. Secondly, the cut-off was assessed in a pre-clinical study comparing the performance of BD MAX Check-Points CPO Assay against ChromID followed by genotyping of each isolate. The study used 338 left-over specimens from routine surveillance at both a high and a low prevalence site in Europe. The final cut-offs for each target were then assigned for each target.

# g. Interference Study

A study was conducted to assess the inhibitory effects of substances potentially encountered in rectal swab specimens on the performance of the BD MAX CheckPoints CPO Assay. Twenty-nine biological and chemical substances were evaluated at “worst case scenario” concentrations. Six replicates of the negative specimen and three replicates of two positive specimen sets were analyzed with/without the potentially interfering substances. All substances were evaluated at a test concentration of $0 . 2 5 \% \mathrm { w } / \mathrm { v }$ $2 . 5 \mathrm { m g / m l } )$ in negative rectal swab matrix. External negative and external positive controls were included with each run. Positive sample sets were prepared from a mix of four carbapenemase-producing organisms harboring KPC, NDM, VIM (or IMP) and OXA-48 gene sequences seeded into pooled negative rectal swab matrix to give concentrations that were $2 { - } 3 \mathbf { x }$ LoD (Table 10). The list of potentially interfering substances tested is shown in Table 11.

Table 10. Organisms used to Prepare the Mixed Positive Samples   

<table><tr><td rowspan=1 colspan=1>Sample</td><td rowspan=1 colspan=1>Strain ID</td><td rowspan=1 colspan=1>Species</td><td rowspan=1 colspan=1>Target</td></tr><tr><td rowspan=4 colspan=1>Positive Set #1</td><td rowspan=1 colspan=1>CP-254</td><td rowspan=1 colspan=1>Klebsiella pneumoniae</td><td rowspan=1 colspan=1>KPC</td></tr><tr><td rowspan=1 colspan=1>CP-433</td><td rowspan=1 colspan=1>Enterobacter cloacae</td><td rowspan=1 colspan=1>VIM</td></tr><tr><td rowspan=1 colspan=1>CP-258</td><td rowspan=1 colspan=1>Klebsiella pneumoniae</td><td rowspan=1 colspan=1>OXA-48</td></tr><tr><td rowspan=1 colspan=1>CP-259</td><td rowspan=1 colspan=1>Klebsiella pneumoniae</td><td rowspan=1 colspan=1>NDM</td></tr><tr><td rowspan=4 colspan=1>Positive Set #2</td><td rowspan=1 colspan=1>CP-254</td><td rowspan=1 colspan=1>Klebsiella pneumoniae</td><td rowspan=1 colspan=1>KPC</td></tr><tr><td rowspan=1 colspan=1>CP-253</td><td rowspan=1 colspan=1>Escherichia coli</td><td rowspan=1 colspan=1>IMP</td></tr><tr><td rowspan=1 colspan=1>CP-258</td><td rowspan=1 colspan=1>Klebsiella pneumoniae</td><td rowspan=1 colspan=1>OXA-48</td></tr><tr><td rowspan=1 colspan=1>CP-259</td><td rowspan=1 colspan=1>Klebsiella pneumoniae</td><td rowspan=1 colspan=1>NDM</td></tr></table>

Table 11. Potentially Interfering Substances Tested   

<table><tr><td rowspan=1 colspan=1>Oils &amp; fatty acids</td><td rowspan=1 colspan=1>Metal salts</td><td rowspan=1 colspan=1>Antibiotics</td><td rowspan=1 colspan=1>Analgesics/Anti-inflammatory</td></tr><tr><td rowspan=1 colspan=1>Stearic acid</td><td rowspan=1 colspan=1>Ba2SO4</td><td rowspan=1 colspan=1>Cephalexin</td><td rowspan=1 colspan=1>Naproxen</td></tr><tr><td rowspan=1 colspan=1>Palmitic acid</td><td rowspan=1 colspan=1>CaCO3</td><td rowspan=1 colspan=1>Ciprofloxacin</td><td rowspan=1 colspan=1>Benzocaine</td></tr><tr><td rowspan=1 colspan=1>Mineral Oil</td><td rowspan=1 colspan=1>Al(OH)3</td><td rowspan=1 colspan=1>Polymyxin B</td><td rowspan=1 colspan=1>Phenylephrine</td></tr><tr><td rowspan=1 colspan=1>Simethicone</td><td rowspan=1 colspan=1>Mg(OH)2</td><td rowspan=1 colspan=1>Bacitracin</td><td rowspan=1 colspan=1>Bismuthsubsalicylate</td></tr><tr><td rowspan=1 colspan=1>Cholesterol</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Neomycin</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Alcohols</td><td rowspan=1 colspan=1>Histamineantagonists</td><td rowspan=1 colspan=1>Surfactants</td><td rowspan=1 colspan=1>Other Substances</td></tr><tr><td rowspan=1 colspan=1>Resorcinol</td><td rowspan=1 colspan=1>Famotidine</td><td rowspan=1 colspan=1>Nonoxynol-9</td><td rowspan=1 colspan=1>Hydrocortisone</td></tr><tr><td rowspan=1 colspan=1>Ethanol</td><td rowspan=1 colspan=1>Omeprazole</td><td rowspan=1 colspan=1>Benzalkoniumchloride</td><td rowspan=1 colspan=1>LoperamideHydrochloride</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Cimetidine</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Nystatin</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Sennosides</td></tr></table>

Results showed that assay targets were detected in the presence of all potentially interfering substances. Of the 432 tests, (7) tests provided non-reportable results (2 UNR and 5 IND). All samples yielding non-reportable results were successfully repeated with valid results (no interference observed). External controls gave the expected test result.

# Competitive Interference

To evaluate the potential competitive inhibitory effect of multiple carbapenemaseproducing organisms on the performance of the BD MAX Check-Points CPO Assay in rectal swab specimens, a study was performed by testing various combinations of carbapenemase-producing organisms seeded at high and low concentrations into natural matrix. High concentrations of organisms corresponded to ${ \sim } 1 \mathrm { x } 1 0 ^ { 6 } \mathrm { C F U / m l } .$ , and low concentrations corresponded to 2x LoD. Table 12 reports the number of target replicates that were detected in the competitive interference study with rectal swab matrix. 7/8 agreement with the expected assay results for target organisms at low concentration in the presence of other target organism(s) at high concentration was set as the acceptance criteria. All sample combinations met acceptance criteria.

Table 12. Number of Correct Results for Combinations in the Competitive Interference Study with the BD MAX Check-Points CPO Assay using a Rectal Swab Matrix   

<table><tr><td rowspan=1 colspan=1>Sample</td><td rowspan=1 colspan=1>KPC(pos. rate)</td><td rowspan=1 colspan=1>VIM/IMP(pos. rate)</td><td rowspan=1 colspan=1>OXA-48(pos. rate)</td><td rowspan=1 colspan=1>NDM(pos. rate)</td></tr><tr><td rowspan=1 colspan=1>High KPC / High VIM / Low OXA48</td><td rowspan=1 colspan=1>8/8</td><td rowspan=1 colspan=1>8/8</td><td rowspan=1 colspan=1>8/8</td><td rowspan=1 colspan=1>0/8</td></tr><tr><td rowspan=1 colspan=1>High KPC / High VIM / Low NDM</td><td rowspan=1 colspan=1>8/8</td><td rowspan=1 colspan=1>8/8</td><td rowspan=1 colspan=1>0/8</td><td rowspan=1 colspan=1>8/8</td></tr><tr><td rowspan=1 colspan=1>High KPC / Low VIM</td><td rowspan=1 colspan=1>8/8</td><td rowspan=1 colspan=1>7/8</td><td rowspan=1 colspan=1>0/8</td><td rowspan=1 colspan=1>0/8</td></tr><tr><td rowspan=1 colspan=1>High KPC / Low IMP</td><td rowspan=1 colspan=1>8/8</td><td rowspan=1 colspan=1>7/8</td><td rowspan=1 colspan=1>0/8</td><td rowspan=1 colspan=1>0/8</td></tr><tr><td rowspan=1 colspan=1>High VIM / Low KPC</td><td rowspan=1 colspan=1>8/8</td><td rowspan=1 colspan=1>8/8</td><td rowspan=1 colspan=1>0/8</td><td rowspan=1 colspan=1>0/8</td></tr><tr><td rowspan=1 colspan=1>High OXA48 / High NDM / Low KPC</td><td rowspan=1 colspan=1>8/8</td><td rowspan=1 colspan=1>0/8</td><td rowspan=1 colspan=1>8/8</td><td rowspan=1 colspan=1>8/8</td></tr><tr><td rowspan=1 colspan=1>High OXA48 / High NDM / Low VIM</td><td rowspan=1 colspan=1>0/8</td><td rowspan=1 colspan=1>7/8</td><td rowspan=1 colspan=1>8/8</td><td rowspan=1 colspan=1>8/8</td></tr><tr><td rowspan=1 colspan=1>High OXA48 / High NDM / Low IMP</td><td rowspan=1 colspan=1>0/8</td><td rowspan=1 colspan=1>8/8</td><td rowspan=1 colspan=1>8/8</td><td rowspan=1 colspan=1>8/8</td></tr><tr><td rowspan=1 colspan=1>High OXA48 / Low NDM</td><td rowspan=1 colspan=1>0/8</td><td rowspan=1 colspan=1>0/8</td><td rowspan=1 colspan=1>8/8</td><td rowspan=1 colspan=1>8/8</td></tr><tr><td rowspan=1 colspan=1>High NDM / LowOXA48</td><td rowspan=1 colspan=1>0/8</td><td rowspan=1 colspan=1>0/8</td><td rowspan=1 colspan=1>8/8</td><td rowspan=1 colspan=1>8/8</td></tr><tr><td rowspan=1 colspan=1>High IMP / LowKPC</td><td rowspan=1 colspan=1>8/8</td><td rowspan=1 colspan=1>8/8</td><td rowspan=1 colspan=1>0/8</td><td rowspan=1 colspan=1>0/8</td></tr><tr><td rowspan=1 colspan=1>High IMP / LowOXA48</td><td rowspan=1 colspan=1>0/8</td><td rowspan=1 colspan=1>8/8</td><td rowspan=1 colspan=1>8/8</td><td rowspan=1 colspan=1>0/8</td></tr><tr><td rowspan=1 colspan=1>High IMP/Low NDM</td><td rowspan=1 colspan=1>0/8</td><td rowspan=1 colspan=1>8/8</td><td rowspan=1 colspan=1>0/8</td><td rowspan=1 colspan=1>8/8</td></tr></table>

# h. Carry-Over

The purpose of the Carry-Over Study was to determine the potential for carry-over contamination when processing multiple high positive and negative samples. Two positive sample sets with a high load of 4 targets $\left( > 1 0 ^ { 6 } \mathrm { C F U / m l } \right)$ were analyzed in replicates of 84 with adjoining negative samples (168 total negative replicates) using 3 BD MAX instruments in accordance with the IFU. Positive sample sets were identical to those prepared for the Interference Study. Negative rectal swab matrix was obtained by pooling ESwab Amies liquid from the rectal swabs (ESwab) obtained from human healthy donors.

A total of 14 BD MAX runs were conducted. All samples yielded the expected results—82 Positive Set #1, 84 Positive Set #2, and 166 negative samples. In the study, two positive samples yielded IND results due to what later was identified as an instrument error. These two positive samples and their adjoining 2 negative samples were later excluded. In summary, the study showed no carry-over contamination from high positive to negative samples.

2. Comparison studies:

a. Method comparison with predicate device: Not Applicable

b. Matrix comparison: Not Applicable

3. Clinical studies:

a. Clinical Sensitivity:

In a multi-center study, five geographically diverse sites were selected (three in the United States and two in Europe) to evaluate the BD MAX Check-Points CPO Assay. Performance characteristics of BD MAX Check-Points CPO Assay were determined with leftover material (rectal swab; ESwab in Transport Medium) from normal standard of care laboratory testing of patients eligible for CPO screening according to institutional policies or meeting the recommendation by the CDC for CPO surveillance cultures. Specimens were collected using ESwab, transported to the laboratory, vortexed, and used for normal standard of care laboratory testing. Aliquots of the same Transport Medium were de-identified and used for analysis by the BD MAX Check-Points CPO Assay according the instructions in the package insert and by the Composite Reference Method. The positive percent agreement (PPA) and negative percent agreement (NPA) of the BD MAX Check-Points CPO Assay were evaluated relative to the Composite Reference Method (consisting of Reference Culture $^ +$ PCR and Sequencing of the amplification product).

For the Composite Reference Method, testing of rectal ESwab medium was conducted with three (3) different culture media in order to optimize recovery of Gram-negative organisms with reduced susceptibility to carbapenems:

• MacConkey broth enrichment ( $1 1 ~ \mathrm { m l }$ broth with a $1 0 ~ \mu \ g$ Meropenem disc) followed by plating onto MacConkey agar with a Meropenem disc   
• ChromID Carba agar   
• ChromID OXA-48 agar

A $5 0 ~ \mu \mu \mu$ aliquot of the transport medium was used to inoculate all culture media. chromID Carba and chromID OXA-48 plates were incubated for 18– 24 hours at $3 5 -$ $3 7 ^ { \circ } \mathrm { C }$ and inspected for the presence of bacterial colonies. If no colonies were found, the test result was recorded as negative. If colonies were found, the color was noted and each morphotype was subcultured onto a blood agar plate and incubated overnight at $3 5 { - } 3 7 ^ { \circ } \mathrm { C }$ . MacConkey broth tubes (with a meropenem disc added) were incubated for 16 to 18 hours at $3 5 { - } 3 7 ^ { \circ } \mathrm { C } _ { \mathrm { \scriptscriptstyle } }$ , after which $1 0 0 \mu \mathrm { l }$ of the MacConkey broth was spread onto a MacConkey agar plate. A $1 0 ~ \mu \ g$ Meropenem disc was then placed in the center of the MacConkey plate, and the plate was incubated for 16 to 18 hours at $3 5 { - } 3 7 ^ { \circ } \mathrm { C }$ . The day after, the MacConkey plates with the Meropenem disc were inspected and the zone diameter around the Meropenem disc, if present, was measured. If no growth was detected or the zone of clearing was $> 2 8 ~ \mathrm { m m }$ , the test result was recorded as negative. If growth was found within a $2 8 \mathrm { m m }$ zone including the disc, then bacteria were subcultured onto a blood agar plate.

Species identification was performed for isolates that were subcultured overnight on blood agar using a MALDI Biotyper or alternative routine methods at the clinical sites. Species ID and susceptibility towards carbapenems was assessed for bacterial isolates from any of these 3 media. For each Gram-negative bacterial isolate, the Meropenem, Imipenem and Ertapenem susceptibility was assessed using CLSI reference method M02-A12, which were interpreted the next day using interpretation criteria from CLSI M100-S27. Isolates that were susceptible for all carbapenems were scored as negative. Intermediate or resistant isolates (only Enterobacteriaceae, Acinetobacter or Pseudomonas aeruginosa) assigned to these categories using the criteria from the CLSI M100-S27 document were further tested for the presence of the BD MAX CPO target genes (KPC, VIM, IMP, OXA-48 and NDM) by reference PCR and bidirectional sequencing. Specimens from which Gram-negative isolates were cultured, but for which no susceptibility criteria were available, e.g. Stenotrophomonas maltophilia, Pseudomonas putida, Aeromonas veronii, were excluded for further analysis by the Composite Reference Method along with the BD MAX Check-Points CPO Assay test result for those samples. For validation of the Reference Culture methodology applied in the Clinical Study, strains with known susceptibility status (susceptible, intermediate or resistant) to Meropenem, Ertapenem, and Imipenem using CLSI standard test methods (M02-A12, CLSI M100- S27) and/or known resistance genes were spiked into unique rectal swabs in order to test for the recovery of the organisms.

A total of 1576 prospective rectal ESwab specimens were initially enrolled in the Clinical Study. Of the 1576 samples collected, there were eighteen (18) UNR $[ 1 . 1 \% ]$ , six (6) INC $[ 0 . 4 \% ]$ , and four (4) IND $[ 0 . 3 \% ]$ results documented with the BD MAX Check-Points CPO Assay, where 17 of the 28 non-reportable samples were re-tested. Fifteen of the seventeen samples yielded valid results after re-testing. A total of (103) samples were excluded because:

• (8) specimens were not added to SBT at the appropriate time   
• (11) specimens were in SBT $>$ the allotted time   
• (16) specimens were associated with an External Control result that was not repeated according to the protocol   
• (27) specimens were recovered with a Gram-negative species ID other than Enterobacteriaceae, Acinetobacter, Pseudomonas aeruginosa (no criteria for carbapenem non-susceptibility)   
• (21) specimens showed environmental contamination   
• (7) specimens included incomplete documentation (11) non-reportables samples (UNR and IND) did not undergo repeat testing

as required (2) specimens generated UNR even after repeat testing

In the final analysis, 1473 ESwab specimens were included for the Prospective Study. Performance of the BD MAX Check-Points CPO Assay was assessed separately for each type of resistance marker target and compared to the Composite Reference Method result. Study results of the BD MAX Check-Points CPO Assay compared to the Composite Reference Method are shown in Table 13 stratified by individual target for the Prospective and Contrived Studies.

Table 13. Clinical Performance Data for the BD MAX Check-Points CPO Assay vs. Reference Culture $^ +$ Sequencing   

<table><tr><td rowspan=1 colspan=1>Study</td><td rowspan=1 colspan=1>Target</td><td rowspan=1 colspan=1>TP</td><td rowspan=1 colspan=1>FP</td><td rowspan=1 colspan=1>TN</td><td rowspan=1 colspan=1>FN</td><td rowspan=1 colspan=1>PPA%(95 CI)</td><td rowspan=1 colspan=1>NPA%(95 CI)</td></tr><tr><td rowspan=4 colspan=1>Prospective(n=1473)</td><td rowspan=1 colspan=1>IMP/VIM</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>1454</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>75%(40.9-92.9)</td><td rowspan=1 colspan=1>99.3%(98.7-99.6)</td></tr><tr><td rowspan=1 colspan=1>NDM</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1471</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0%(20.7-100.0)</td><td rowspan=1 colspan=1>99.6%(99.6-100.0)</td></tr><tr><td rowspan=1 colspan=1>KPC</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>1436</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>88.2%(73.4-95.3)</td><td rowspan=1 colspan=1>99.8%(99.4-99.9)</td></tr><tr><td rowspan=1 colspan=1>OXA-48</td><td rowspan=1 colspan=1>25</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>1432</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>96.2%(81.1-99.3)</td><td rowspan=1 colspan=1>99.0%(98.3-99.4)</td></tr><tr><td rowspan=4 colspan=1>Contrived(n=166)</td><td rowspan=1 colspan=1>IMP*/VIM</td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>114</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>96.1%(87.0-98.4)</td><td rowspan=1 colspan=1>100.0%(96.7-100.0)</td></tr><tr><td rowspan=1 colspan=1>NDM</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>136</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0%(88.7-100.0)</td><td rowspan=1 colspan=1>100.0%(97.3-100.0)</td></tr><tr><td rowspan=1 colspan=1>KPC</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>136</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0%(88.7-100.0)</td><td rowspan=1 colspan=1>100.0%(97.3-100.0)</td></tr><tr><td rowspan=1 colspan=1>OXA-48</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>136</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0%(88.7-100.0)</td><td rowspan=1 colspan=1>100.0%(97.3-100.0)</td></tr></table>

\* A single NS organism harboring IMP-13 variant was identified as Negative by the BD MAX Check-Points CPO Assay; Since IMP-13 is not claimed for detection of the IMP variant, this sample was treated as TN.

A total of 136 target and 30 non-target strains were used for the Contrived Study. Clinical isolates and reference strains (National Collection of Type Cultures (NCTC) from an in-house collection at Check-Point (CP) were selected to represent a variety of clinical species, carbapenem susceptibilities, species with target variants, strains with carbapenemase genes other than the target genes, and strains with different resistance mechanisms. Each strain was spiked into unique negative rectal swab at approximately 1-2x LoD preparation for target specimens and $1 / 2 0 \mathrm { x } \mathrm { O D } _ { 6 0 0 } 0 . 1$ for non-target specimens. Target specimens yielding a negative result by the BD MAX Check-Points CPO Assay were re-tested at $3 \mathbf { x }$ LoD and $1 0 \mathrm { x } \ \mathrm { L o D }$ . The following

numbers of unique isolates were evaluated in the study spanning multiple gramnegative species (Table 14 below):

Table. 14 Various Species Tested in the Contrived Study by Target   

<table><tr><td colspan="1" rowspan="1">Target*(number tested)</td><td colspan="1" rowspan="1">Various Organismsincluded in the Study</td></tr><tr><td colspan="1" rowspan="9">VIM/IMP (52)</td><td colspan="1" rowspan="1">Citrobacter braakii</td></tr><tr><td colspan="1" rowspan="1">Citrobacter freundii</td></tr><tr><td colspan="1" rowspan="1">Enterobacter cloacae</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td></tr><tr><td colspan="1" rowspan="1">Klebsiella oxytoca</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae</td></tr><tr><td colspan="1" rowspan="1">Klebsiella variicola</td></tr><tr><td colspan="1" rowspan="1">Providencia stuartii</td></tr><tr><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td></tr><tr><td colspan="1" rowspan="8">NDM (30)</td><td colspan="1" rowspan="1">Acinetobacter baumannii</td></tr><tr><td colspan="1" rowspan="1">Acinetobacter spp.</td></tr><tr><td colspan="1" rowspan="1">Enterobacter cloacae</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae</td></tr><tr><td colspan="1" rowspan="1">Morganella morganii</td></tr><tr><td colspan="1" rowspan="1">Providencia rettgeri</td></tr><tr><td colspan="1" rowspan="1">Providencia stuartii</td></tr><tr><td colspan="1" rowspan="4">KPC (30)</td><td colspan="1" rowspan="1">Enterobacter asburiae</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae</td></tr><tr><td colspan="1" rowspan="1">Klebsiella variicola</td></tr><tr><td colspan="1" rowspan="5">OXA-48(30)</td><td colspan="1" rowspan="1">Citrobacter freundii</td></tr><tr><td colspan="1" rowspan="1">Enterobacter aerogenes</td></tr><tr><td colspan="1" rowspan="1">Enterobacter cloacae</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae</td></tr><tr><td colspan="1" rowspan="10">No target (30)</td><td colspan="1" rowspan="1">Acinetobacter baumannii</td></tr><tr><td colspan="1" rowspan="1">Acinetobacter spp</td></tr><tr><td colspan="1" rowspan="1">Citrobacter braakii</td></tr><tr><td colspan="1" rowspan="1">Citrobacter freundii</td></tr><tr><td colspan="1" rowspan="1">Enterobacter cloacae</td></tr><tr><td colspan="1" rowspan="1">Enterobacter aerogenes</td></tr><tr><td colspan="1" rowspan="1">Enterobacter asburiae</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td></tr><tr><td colspan="1" rowspan="1">Klebsiella oxytoca</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae</td></tr><tr><td colspan="1" rowspan="7"></td><td colspan="1" rowspan="1">Morganella morganii</td></tr><tr><td colspan="1" rowspan="1">Salmonella Heidelberg</td></tr><tr><td colspan="1" rowspan="1">Serratia marcescens</td></tr><tr><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td></tr><tr><td colspan="1" rowspan="1">Proteus mirabilis</td></tr><tr><td colspan="1" rowspan="1">Providencia alcalifaciens</td></tr><tr><td colspan="1" rowspan="1">Raoultella ornithinolytica</td></tr></table>

\*(6) strains had multiple targets.

b. Clinical specificity: See comments in 3a above.   
c. Other clinical supportive data (when a. and b. are not applicable): Not Applicable

4. Clinical cut-off: Not Applicable

5. Expected values/Reference range:

Of the 1473 rectal ESwab specimens in the study included for analysis, $4 . 5 \%$ of the specimens (67/1473) contained a carbapenem non-susceptible organism with at least one of the assay gene targets (IMP, VIM, NDM, KPC, OXA-48) that was recovered by the Composite Reference Method (Table 15).

Table 15. Detection of Non-susceptible Organisms with the Gene Targets By the Composite Reference Method (Reference Culture $^ +$ Sequencing)   

<table><tr><td rowspan=1 colspan=2>Site Information</td><td rowspan=1 colspan=2>Rectal Swab Specimens</td></tr><tr><td rowspan=1 colspan=1>Collection Site</td><td rowspan=1 colspan=1>Location</td><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1># Samples Positiveby ReferenceCulture +Sequencing(NS organism with atleast one assay target)</td></tr><tr><td rowspan=1 colspan=1>Site 1</td><td rowspan=1 colspan=1>US (East Coast)</td><td rowspan=1 colspan=1>490</td><td rowspan=1 colspan=1>3</td></tr><tr><td rowspan=1 colspan=1>Site 2</td><td rowspan=1 colspan=1>US (Southeast)</td><td rowspan=1 colspan=1>297</td><td rowspan=1 colspan=1>10</td></tr><tr><td rowspan=1 colspan=1>Site 3</td><td rowspan=1 colspan=1>US (Midwest)</td><td rowspan=1 colspan=1>156</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Site 4</td><td rowspan=1 colspan=1>Europe (Spain)</td><td rowspan=1 colspan=1>422</td><td rowspan=1 colspan=1>54</td></tr><tr><td rowspan=1 colspan=1>Site 5</td><td rowspan=1 colspan=1>Europe(Netherlands)</td><td rowspan=1 colspan=1>108</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=2>Totals</td><td rowspan=1 colspan=1>1473</td><td rowspan=1 colspan=1>67</td></tr></table>

# N. Instrument Name:

BD MAX System

# O. System Descriptions:

1. Modes of Operation:

Does the applicant’s device contain the ability to transmit data to a computer, webserver, or mobile device?

Yes _ _X __ or No

Does the applicant’s device transmit data to a computer, webserver, or mobile device using wireless transmission?

Yes or No __X_

2. Software:

FDA has reviewed applicant’s Hazard Analysis and software development processes for this line of product types:

Yes _ _X____ or No

3. Specimen Identification: Specimens are labeled with a unique bar code.

4. Specimen Sampling and Handling:

Specimens are handled through the BD MAX sample processing subsystem, which utilizes a fluid handling robot for sampling that is controlled by the BD MAX System; it does not require user intervention to operate. The specimens are loaded onto the BD MAX System where automated processing and reporting of assay results are completed.

5. Calibration:

BD MAX does not require user calibration. All maintenance procedures, other than daily and weekly cleaning procedures, are performed by qualified personnel.

6. Quality Control:

The assay includes a Specimen Processing Control (SPC) that is present in every Extraction Tube. The SPC monitors DNA extraction steps, thermal cycling steps, reagent integrity, and the presence of inhibitory substances. External Control materials are treated as if they were patient samples; the BD MAX software does not use QC results for the interpretation of test results. External controls are not provided and must be sourced by the user.

It is recommended that one External Positive Control and one External Negative Control be run at least daily until adequate process validation is achieved on the BD MAX System in each laboratory setting.

The External Positive Control is intended to monitor for substantial reagent failure. The External Negative Control is intended to detect reagent or environmental contamination (or carry-over) by target nucleic acids.

P. Other Supportive Instrument Performance Characteristics Data Not Covered In The “Performance Characteristics” Section above:

Not Applicable

# Q. Proposed Labeling:

The labeling supports the finding of substantial equivalence for this device.

# R. Conclusion:

The submitted information in this premarket notification is complete and supports a substantial equivalence decision.