# December 20, 2019

Curetis GmbH   
$\%$ Gail Radcliffe   
President   
Radcliffe Consulting, Inc.   
231 Fairbanks Street   
West Boylston, Massachusetts 01583

Re: K191967 Trade/Device Name: Unyvero LRT BAL Application Regulation Number: 21 CFR 866.3985 Regulation Name: Device To Detect And Identify Microorganisms And Associated Resistance Marker Nucleic Acids Directly In Respiratory Specimens Regulatory Class: Class II Product Code: QBH Dated: July 22, 2019 Received: July 23, 2019

Dear Gail Radcliffe:

We have reviewed your Section 510(k) premarket notification of intent to market the device referenced above and have determined the device is substantially equivalent (for the indications for use stated in the enclosure) to legally marketed predicate devices marketed in interstate commerce prior to May 28, 1976, the enactment date of the Medical Device Amendments, or to devices that have been reclassified in accordance with the provisions of the Federal Food, Drug, and Cosmetic Act (Act) that do not require approval of a premarket approval application (PMA). You may, therefore, market the device, subject to the general controls provisions of the Act. Although this letter refers to your product as a device, please be aware that some cleared products may instead be combination products. The 510(k) Premarket Notification Database located at https://www.accessdata.fda.gov/scripts/cdrh/cfdocs/cfpmn/pmn.cfm identifies combination product submissions. The general controls provisions of the Act include requirements for annual registration, listing of devices, good manufacturing practice, labeling, and prohibitions against misbranding and adulteration. Please note: CDRH does not evaluate information related to contract liability warranties. We remind you, however, that device labeling must be truthful and not misleading.

If your device is classified (see above) into either class II (Special Controls) or class III (PMA), it may be subject to additional controls. Existing major regulations affecting your device can be found in the Code of Federal Regulations, Title 21, Parts 800 to 898. In addition, FDA may publish further announcements concerning your device in the Federal Register.

Please be advised that FDA's issuance of a substantial equivalence determination does not mean that FDA has made a determination that your device complies with other requirements of the Act or any Federal

statutes and regulations administered by other Federal agencies. You must comply with all the Act's requirements, including, but not limited to: registration and listing (21 CFR Part 807); labeling (21 CFR Part 801 and Part 809); medical device reporting (reporting of medical device-related adverse events) (21 CFR 803) for devices or postmarketing safety reporting (21 CFR 4, Subpart B) for combination products (see https://www.fda.gov/combination-products/guidance-regulatory-information/postmarketing-safety-reportingcombination-products); good manufacturing practice requirements as set forth in the quality systems (QS) regulation (21 CFR Part 820) for devices or current good manufacturing practices (21 CFR 4, Subpart A) for combination products; and, if applicable, the electronic product radiation control provisions (Sections 531- 542 of the Act); 21 CFR 1000-1050.

Also, please note the regulation entitled, "Misbranding by reference to premarket notification" (21 CFR Part 807.97). For questions regarding the reporting of adverse events under the MDR regulation (21 CFR Part 803), please go to https://www.fda.gov/medical-devices/medical-device-safety/medical-device-reportingmdr-how-report-medical-device-problems.

For comprehensive regulatory information about medical devices and radiation-emitting products, including information about labeling regulations, please see Device Advice (https://www.fda.gov/medical  
devices/device-advice-comprehensive-regulatory-assistance) and CDRH Learn   
(https://www.fda.gov/training-and-continuing-education/cdrh-learn). Additionally, you may contact the Division of Industry and Consumer Education (DICE) to ask a question about a specific regulatory topic. See the DICE website (https://www.fda.gov/medical-devices/device-advice-comprehensive-regulatoryassistance/contact-us-division-industry-and-consumer-education-dice) for more information or contact DICE by email (DICE@fda.hhs.gov) or phone (1-800-638-2041 or 301-796-7100).

Sincerely,

Kristian Roth, Ph.D.   
Branch Chief   
Bacterial Multiples and Medical Countermeasures Division of Microbiology Devices   
OHT7: Office of In Vitro Diagnostics   
and Radiological Health   
Office of Product Evaluation and Quality   
Center for Devices and Radiological Health

Enclosure

510(k) Number (if known) K191967

Device Name Unyvero LRT BAL Application

Indications for Use (Describe)

The Unyvero LRT BAL Application is a qualitative nucleic acid multiplex test intended for the simultaneous detection and identification of nucleic acid sequences from the following microorganisms $( \mathrm { N } = 2 0 )$ ) and antibiotic resistance markers $( \mathrm { N } = 1 0 )$ ) in bronchoalveolar lavage (BAL)-like specimens (BAL or mini-BAL) from adult hospitalized patients with suspected lower respiratory tract infections.

[continued on page 2]

# CONTINUE ON A SEPARATE PAGE IF NEEDED.

This section applies only to requirements of the Paperwork Reduction Act of 1995.

# \*DO NOT SEND YOUR COMPLETED FORM TO THE PRA STAFF EMAIL ADDRESS BELOW.\*

The burden time for this collection of information is estimated to average 79 hours per response, including the time to review instructions, search existing data sources, gather and maintain the data needed and complete and review the collection of information. Send comments regarding this burden estimate or any other aspect of this information collection, including suggestions for reducing this burden, to:

Department of Health and Human Services Food and Drug Administration Office of Chief Information Officer Paperwork Reduction Act (PRA) Staff PRAStaff@fda.hhs.gov

“An agency may not conduct or sponsor, and a person is not required to respond to, a collection of information unless it displays a currently valid OMB number.”

[continued from Form FDA 3881, page 1]   

<table><tr><td rowspan=1 colspan=2>Microorganism                            Associated Antibiotic Resistance Marker</td></tr><tr><td rowspan=1 colspan=1>Acinetobacter spp. a</td><td rowspan=1 colspan=1>ctx-M b, kpc, ndm, oxa-23, oxa-24, oxa-58, vim</td></tr><tr><td rowspan=1 colspan=1>Chlamydia pneumoniae</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Citrobacter freundii</td><td rowspan=1 colspan=1>ctx-M b, kpc, ndm, oxa-48, vim</td></tr><tr><td rowspan=1 colspan=1>Enterobacter cloacae complex</td><td rowspan=1 colspan=1>ctx-M b, kpc, ndm, oxa-48, vim</td></tr><tr><td rowspan=1 colspan=1>Escherichia coli</td><td rowspan=1 colspan=1>ctx-M b, kpc, ndm, oxa-48, vim</td></tr><tr><td rowspan=1 colspan=1>Haemophilus influenzae</td><td rowspan=1 colspan=1>tem</td></tr><tr><td rowspan=1 colspan=1>Klebsiella oxytoca</td><td rowspan=1 colspan=1>ctx-M b, kpc, ndm, oxa-48, vim</td></tr><tr><td rowspan=1 colspan=1>Klebsiella pneumoniae </td><td rowspan=1 colspan=1>ctx-M b, kpc, ndm, oxa-48, vim</td></tr><tr><td rowspan=1 colspan=1>Klebsiella varicola</td><td rowspan=1 colspan=1>ctx-M b, kpc, ndm, oxa-48, vim</td></tr><tr><td rowspan=1 colspan=1>Legionella pneumophila</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>Moraxella catarrhalis</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Morganella morganii</td><td rowspan=1 colspan=1>ctx-M b, kpc, ndm, oxa-48, vim</td></tr><tr><td rowspan=1 colspan=1>Mycoplasma pneumoniae</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Pneumocystis jirovecii</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>Proteus spp. </td><td rowspan=1 colspan=1>ctx-M b, kpc, ndm, oxa-48, vim</td></tr><tr><td rowspan=1 colspan=1>Pseudomonas aeruginosa</td><td rowspan=1 colspan=1>ctx-M b, kpc, ndm, vim</td></tr><tr><td rowspan=1 colspan=1>Serratia marcescens</td><td rowspan=1 colspan=1>ctx-M b, kpc, ndm, oxa-48, vim</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus aureus</td><td rowspan=1 colspan=1>mecA</td></tr><tr><td rowspan=1 colspan=1>Stenotrophomonas maltophilia</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Streptococcus pneumoniae</td><td rowspan=1 colspan=1>-</td></tr></table>

a Acinetobacter spp. includes: A. baumannii, A. calcoaceticus, A. haemolyticus, A. junii, A. lwoffii, A. nosocomialis, A. parvus, A. pittii (detected by LRT BAL Application) and A. ursingii (not detected by LRT BAL Application). b ctx-M1 subgroup. c Enterobacter cloacae complex includes: $E .$ . asburiae, $E .$ . chengduensis, E. chuandaensis, E. cloacae, E. hormaechei (incl. ssp. xiangfangensis), E. kobei, E. ludwigii, E. roggenkampii, E. sichuanensis as well as $E$ . bugandensis (not yet recognized as member of the $E$ . cloacae complex). d Klebsiella pneumoniae includes two variants: K. pneumoniae (variant 1), and K. quasipneumoniae (variant 2). e Proteus spp. includes P. cibarius, P. hauseri, P. mirabilis, $P$ . penneri and P. vulgaris.

The Unyvero LRT BAL Application performed on the Unyvero System is indicated as an aid in the diagnosis of lower respiratory tract infection in adult hospitalized patients with signs and symptoms of lower respiratory infection; results should be used in conjunction with other clinical and laboratory findings. As BAL specimens may contain colonizing microorganisms, detection of Unyvero LRT BAL microbial targets does not indicate that the microorganism is the cause of the disease. Unyvero positive results do not rule out co-infection with other microorganisms. Negative results do not preclude lower respiratory infection, as the causative agent may be a microorganism not detected by this test.

A negative result for any antibiotic resistance marker does not indicate that detected microorganisms are susceptible to applicable antimicrobial agents. Detected antibiotic resistance markers cannot be definitively linked to specific microorganisms, and may be present in organisms that are not detected by the Unyvero LRT BAL Application.

Microbiology cultures of BALs should be performed to obtain isolates for species identification and antimicrobial susceptibility testing and to identify potential microorganisms not targeted by the Unyvero LRT BAL Application.

# 510(k) SUMMARY – LRT BAL Application (K191967)

This summary of 510(k) safety and effectiveness information is being submitted in accordance with the requirements of SMDA 1990 and 21 CFR 807.92. The assigned 510(k) number is K191967.

807.92 (a)(1): Name: Curetis GmbH Address: Max-Eyth-Strasse 42 Holzgerlingen, Germany 71088

Phone: +49 7031 49195-24   
Email: regulatory@curetis.com   
Contact: Karsten Mueller, Head of Quality and Regulatory Affairs

# 807.92 (a)(2): Device name - trade name and common name, and classification

# Trade name:

Unyvero LRT BAL Application

Common Name: Detection and identification of microorganisms and associated resistance marker nucleic acids directly in respiratory specimens

Classification / Product Code: 21 CFR Part 866.3985 / QBH

# 807.92 (a)(3): Identification of the legally marketed predicate devices

The Unyvero LRT BAL Application is substantially equivalent to its predecessor test system, the Unyvero LRT Application (Curetis, GmbH, Germany), granted de novo status under DEN170047 on April $3 ^ { \mathrm { r d } }$ , 2018.

# 807.92 (a)(4): Device Description

The Unyvero LRT BAL Application automates and integrates DNA purification and eight parallel multiplex endpoint PCR reactions. It provides qualitative detection of nucleic acids from multiple lower respiratory pathogens using hybridization on PCR chamber arrays in a single use cartridge from a single bronchoalveolar lavage (BAL)-like specimen (BAL or mini-BAL).

The Unyvero LRT BAL Application identifies 20 microorganisms and 10 antibiotic resistance markers as listed in the Intended Use Statement, below.

# Overview

The Unyvero LRT BAL Application uses a multiplex PCR approach following array hybridization which targets 30 individual analytes (microorganisms $\mathrm { N } = 2 0 $ ) and antibiotic resistance markers $( \Nu = 1 0$ ) divided into eight separate PCR reactions that are performed in individual reaction chambers simultaneously on a Unyvero LRT BAL Application cartridge. Multiplex compositions are designed to avoid any expected common occurrence of certain analytes within the same multiplex to largely reduce competitive PCR inhibition. Individual analyte assays of the Unyvero LRT BAL panel are designed to exhibit low or absent cross-reactivity with the relevant bronchoalveolar lavage (BAL)- like specimens (BAL or mini-BAL) sample matrix or ‘close neighbor’ strains. Array oligonucleotides are designed for similar hybridization and melting temperatures (approx. $6 5 \mathrm { ~ - ~ } 8 0 ~ ^ { \circ } \mathrm { C }$ , varying by amplicon). Hybridization and melting temperatures are used to exclude non-specific hybridization signals for improved signal specificity.

# Instrument, Cartridge, and Other Consumables

The instrumentation consists of one (or more) Unyvero L4 Lysator, one (or more) Unyvero A50 Analyzer, a Unyvero C8 Cockpit, and four single-use consumables: the Unyvero LRT BAL Cartridge, the Unyvero Sample Tube, Sample Tube Cap and the Unyvero Master Mix. A Unyvero Sample Tube Holder is supplied as accessory to simplify the sample filling step.

Unyvero LRT BAL Cartridge contains DNA isolation and purification reagents, primers, hybridization and wash buffers, and oligonucleotides for detection.   
Unyvero T1 Sample Tube and Transport Cap contains glass beads and buffers to lyse microorganisms and liquefy the sample.   
Unyvero T1 Sample Tube Cap seals the Unyvero Sample Tube and contains Proteinase K and a synthetic control gene for process monitoring.   
Unyvero M1 Master Mix Tube contains reagents for DNA amplification.

# Controls

An internal control (a synthetic gene without any homology to known sequences) is processed in every chamber in order to verify the DNA purification, amplification, array hybridization, and detection.

Other than the built-in controls, no external materials are supplied with Unyvero LRT BAL Application devices and consumables. Good laboratory practice recommends running external positive and negative controls using samples cultured in the laboratory.

# Reagents

No additional reagents are required to perform the Unyvero LRT BAL Application; all reagents are supplied within the cartridge or within the other consumables with the exception of the polymerase Master Mix, which is provided separately (frozen).

# Assay Procedures

How to perform a Unyvero LRT BAL test:

1. Remove the Unyvero Sample Tube from its packaging and slide it in the Unyvero Sample Tube Holder in the upright position with the barcoded end at the bottom.   
2. Remove the Transport Cap from the Sample Tube by pulling it upward.   
3. Pipette $1 8 0 ~ \mu \mathrm { L }$ of vortexed patient specimen into the Sample Tube and close it using the Unyvero Cap provided in the LRT BAL kit; align the small nodules on the neck of the Sample Tube with the openings on the Cap and press down to lock in place.   
4. Scan the Sample Tube and place it into the Lysator. Close the Lysator lid to start the Lysator.   
5. Remove Master Mix from freezer and thaw at room temperature $( 1 5 ~ ^ { \circ } \mathrm { C } ~ - ~ 2 5 ~ ^ { \circ } \mathrm { C } )$ for approximately 30 minutes.   
6. When lysis is complete, remove the Sample Tube from the Lysator and place it into the labeled position on the left-hand side of the Unyvero LRT BAL Cartridge.   
7. Place the thawed Master Mix into the labeled position on the right-hand side of the Cartridge.   
8. Scan the Cartridge on the Cockpit and insert it into the position indicated on the Analyzer. The software provides on-screen instructions to start the test.   
9. View results after completion of the run.

During the automated analysis (Step 8), which is entirely controlled by the A50 Analyzer, the sample is mixed with ethanol and then transferred onto the DNA purification column, where buffers purify and elute the DNA. Eluted DNA is transferred to a chamber, where mixing with the Master Mix takes place. This mixture is distributed into eight separate PCR reaction chambers each containing multiple primer pairs, consisting of one labeled and one non-labeled primer for the respective multiplex endpoint PCR. After specific amplification, PCR products are hybridized to the corresponding array probes. Each array has been manufactured with probes corresponding to the amplicons for the targeted microorganism sequences described above. A total of up to 49 spots per array allows for redundant detection with at least four spots per analyte, as well as spots for intensity calibration, and orientation markers for the image processing software. Binding of amplicons to specific probes is detected by analyzing fluorescence images of the arrays. Result data are displayed on the C8 Cockpit for visualization, printout, temporary storage and electronic data export.

A test run is completed after 4 to 5 hrs, and results for panel microorganisms and corresponding antibiotic resistance markers are displayed on the Cockpit screen. Four screens are provided:

The Result Summary screen provides a quick overview of all detected LRT BAL panel microorganisms, together with all detected corresponding antibiotic resistance markers.   
 The Microorganisms screen provides a list of all panel microorganisms grouped in Grampositive bacteria, non-fermenting bacteria, Enterobacteriaceae and other microorganisms together with the analysis result (reported as detected, not detected or invalid).   
 The Result Details screen provides a list of all analyzed microorganisms and the corresponding antibiotic resistance markers together with the analysis result (reported as detected, not detected, invalid or NA).   
 Test Details screen showing user name, lot numbers of used consumables, expiration dates of the consumables and start and stop times and dates of the test.

Results can be reviewed on the cockpit or, optionally, be printed out. All results are saved in a database on the Unyvero Cockpit for later review and printing.

# Software

The Unyvero software is designed to:

Manage analysis workflow (Cockpit) Carry out sample lysis (Lysator) Execute the analysis and generate the analytical result (Analyzer)  Manage communication among units (Cockpit, Lysator, Analyzer)  Monitor internal mechanical / electrical actuators (Lysator, Analyzer) Present analysis results (Cockpit)  Store analysis results (Cockpit)

Each device (Cockpit, Lysator, Analyzer) is a subsystem within the overall system, and each consists of hardware and software components. The different devices are interconnected by an Ethernet based communication interface, and system functionality is provided by the interaction of all three device types. Only the Cockpit presents a rich user interface and allows interaction with the operator. The Lysator and Analyzer units include a simple display for showing device status. Optional HIS/LIS connectivity allows transferring results to a hospital or laboratory information system.

# 807.92 (a)(5): Intended Use

The Unyvero LRT BAL Application is a qualitative nucleic acid multiplex test intended for the simultaneous detection and identification of nucleic acid sequences from the following microorganisms $( \mathrm { N } = 2 0 )$ ) and antibiotic resistance markers $( \mathrm { N } = 1 0 )$ ) in bronchoalveolar lavage (BAL)- like specimens (BAL or mini-BAL) from adult hospitalized patients with suspected lower respiratory tract infections.

<table><tr><td rowspan=1 colspan=1>Microorganism</td><td rowspan=1 colspan=1>Associated Antibiotic Resistance Marker(s)</td></tr><tr><td rowspan=1 colspan=1>Acinetobacter spp. a</td><td rowspan=1 colspan=1>ctx-M b, kpc, ndm, oxa-23, oxa-24, oxa-58, vim</td></tr><tr><td rowspan=1 colspan=1>Chlamydia pneumoniae</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>Citrobacter freundii</td><td rowspan=1 colspan=1>ctx-M b, kpc, ndm, oxa-48, vim</td></tr><tr><td rowspan=1 colspan=1>Enterobacter cloacae complex </td><td rowspan=1 colspan=1>ctx-M b, kpc, ndm, oxa-48, vim</td></tr><tr><td rowspan=1 colspan=1>Escherichia coli</td><td rowspan=1 colspan=1>ctx-M b, kpc, ndm, oxa-48, vim</td></tr><tr><td rowspan=1 colspan=1>Haemophilus influenzae</td><td rowspan=1 colspan=1>tem</td></tr><tr><td rowspan=1 colspan=1>Klebsiella oxytoca</td><td rowspan=1 colspan=1>ctx-M b, kpc, ndm, oxa-48, vim</td></tr><tr><td rowspan=1 colspan=1>Klebsiella pneumoniae d</td><td rowspan=1 colspan=1>ctx-M b, kpc, ndm, oxa-48, vim</td></tr><tr><td rowspan=1 colspan=1>Klebsiella variicola</td><td rowspan=1 colspan=1>ctx-M b, kpc, ndm, oxa-48, vim</td></tr><tr><td rowspan=1 colspan=1>Legionella pneumophila</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>Moraxella catarrhalis</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>Morganella morganii</td><td rowspan=1 colspan=1>ctx-M b, kpc, ndm, oxa-48, vim</td></tr><tr><td rowspan=1 colspan=1>Mycoplasma pneumoniae</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>Pneumocystis jirovecii</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>Proteus spp.</td><td rowspan=1 colspan=1>ctx-M b, kpc, ndm, oxa-48, vim</td></tr><tr><td rowspan=1 colspan=1>Pseudomonas aeruginosa</td><td rowspan=1 colspan=1>ctx-M b, kpc, ndm, vim</td></tr><tr><td rowspan=1 colspan=1>Serratia marcescens</td><td rowspan=1 colspan=1>ctx-M b, kpc, ndm, oxa-48, vim</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus aureus</td><td rowspan=1 colspan=1>mecA</td></tr><tr><td rowspan=1 colspan=1>Stenotrophomonas maltophilia</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>Streptococcus pneumoniae</td><td rowspan=1 colspan=1>-</td></tr></table>

a Acinetobacter spp. includes: A. baumannii, A. calcoaceticus, A. haemolyticus, A. junii, A. lwoffii, A. nosocomialis, A. parvus, A. pittii (detected by LRT BAL Application), and $A$ . ursingii (not detected by LRT BAL Application). b ctx-M1 subgroup. c Enterobacter cloacae complex includes: $E .$ . asburiae, $E .$ . chengduensis, $E .$ . chuandaensis, E. cloacae, E. hormaechei (incl. ssp. xiangfangensis), E. kobei, E. ludwigii, E. roggenkampii, $E .$ . sichuanensis and $E$ . bugandensis (not yet recognized as member of the $E$ . cloacae complex). d Klebsiella pneumoniae includes two variants: $K .$ . pneumoniae (variant 1), and K. quasipneumoniae (variant 2). e Proteus spp. includes: P. cibarius, P. hauseri, P. mirabilis, P. penneri, and $P .$ vulgaris.

The Unyvero LRT BAL Application performed on the Unyvero System is indicated as an aid in the diagnosis of lower respiratory tract infection in adult hospitalized patients with signs and symptoms of lower respiratory infection; results should be used in conjunction with other clinical and laboratory findings. As BAL specimens may contain colonizing microorganisms, detection of Unyvero LRT BAL microbial targets does not indicate that the microorganism is the cause of the disease. Unyvero positive results do not rule out co-infection with other microorganisms. Negative results do not preclude lower respiratory infection, as the causative agent may be a microorganism not detected by this test.

A negative result for any antibiotic resistance marker does not indicate that detected microorganisms are susceptible to applicable antimicrobial agents. Detected antibiotic resistance markers cannot be definitively linked to specific microorganisms, and may be present in organisms that are not detected by the Unyvero LRT BAL Application.

Microbiology cultures of BALs should be performed to obtain isolates for species identification and antimicrobial susceptibility testing and to identify potential microorganisms not targeted by the Unyvero LRT BAL Application.

# 807.92 (a)(6): Technological Similarities and Differences to the Predicate

Table 1: Comparison of the Unyvero LRT Application and the Unyvero LRT BAL Application   

<table><tr><td>Element</td><td>New Device: Curetis Unyvero LRT BAL Application</td><td>Predicate: Curetis Unyvero LRT Application (DEN170047)</td></tr><tr><td>Specimen Type</td><td>Bronchoalveolar lavage (BAL)-like specimens from adult hospitalized patients with suspected lower respiratory tract infections</td><td>Endotracheal aspirate specimens from adult hospitalized patients with suspected lower respiratory tract infections</td></tr><tr><td>Organisms Detected</td><td>Bacteria: Acinetobacter spp. Citrobacter freundii Enterobacter cloacae complex Escherichia coli Haemophilus influenzae Klebsiella oxytoca Klebsiella pneumoniae Klebsiella variicola Moraxella catarrhalis Morganella morganii Proteus spp. Pseudomonas aeruginosa Serratia marcescens Staphylococcus aureus Stenotrophomonas maltophilia Streptococcus pneumoniae Atypical Bacteria: Chlamydia pneumoniae Legionella pneumophila Mycoplasma pneumoniae Fungus: Pneumocystis jirovecii Antimicrobial Resistance Genes:</td><td>Bacteria: Acinetobacter spp. Citrobacter freundii Enterobacter cloacae complex Escherichia coli Haemophilus influenzae Klebsiella oxytoca Klebsiella pneumoniae Klebsiella variicola Moraxella catarrhalis Morganella morganii Proteus spp. Pseudomonas aeruginosa Serratia marcescens Staphylococcus aureus Stenotrophomonas maltophilia Streptococcus pneumoniae Atypical Bacteria: Chlamydia pneumoniae Legionella pneumophila Mycoplasma pneumoniae Antimicrobial Resistance Genes:</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">ctx-MkpcmecAndmoxa-23oxa-24oxa-48oxa-58temvim</td><td colspan="1" rowspan="1">ctx-MkpcmecAndmoxa-23oxa-24oxa-48,oxa-58,temvim</td></tr><tr><td colspan="1" rowspan="1">Analyte</td><td colspan="1" rowspan="1">DNA</td><td colspan="1" rowspan="1">DNA</td></tr><tr><td colspan="1" rowspan="1">TechnologicalPrinciples</td><td colspan="1" rowspan="1">Multiplex nucleic acid</td><td colspan="1" rowspan="1">Multiplex nucleic acid</td></tr><tr><td colspan="1" rowspan="1">Result Types</td><td colspan="1" rowspan="1">Qualitative for all analytes</td><td colspan="1" rowspan="1">Qualitative for all analytes</td></tr><tr><td colspan="1" rowspan="1">Instrumentation</td><td colspan="1" rowspan="1">Unyvero System</td><td colspan="1" rowspan="1">Unyvero System</td></tr><tr><td colspan="1" rowspan="1">Time to Result</td><td colspan="1" rowspan="1">About 4-5 hrs</td><td colspan="1" rowspan="1">About 4-5 hrs</td></tr><tr><td colspan="1" rowspan="1">Reagent Storage</td><td colspan="1" rowspan="1">Room temperature(below -20 °C, Master Mix Tube only)</td><td colspan="1" rowspan="1">Room temperature(below -20 °C, Master Mix Tube only)</td></tr><tr><td colspan="1" rowspan="1">TestInterpretation</td><td colspan="1" rowspan="1">Automated test interpretation and reportgeneration.</td><td colspan="1" rowspan="1">Automated test interpretation and reportgeneration.</td></tr><tr><td colspan="1" rowspan="1">Controls</td><td colspan="1" rowspan="1">Control included in each test to monitor forthe presence of PCR inhibitors and enablesthe system to detect failures in the testingprocess.</td><td colspan="1" rowspan="1">Control included in each test to monitor forthe presence of PCR inhibitors and enablesthe system to detect failures in the testingprocess.</td></tr><tr><td colspan="1" rowspan="1">User Complexity</td><td colspan="1" rowspan="1">Moderate</td><td colspan="1" rowspan="1">Moderate</td></tr></table>

# 807.92 (b)(1): Brief Description of Nonclinical Data

# Limits-of-Detection (LoDs)

Limits-of-Detection were determined for each LRT BAL panel analyte by serial dilutions of reference strains prepared in pooled negative native lavage specimens as test matrix. The lowest test concentration with a positivity rate of $9 5 \%$ or higher was determined as the LoD for each specific panel analyte (e.g., at least 19 of 20 positive results for at least 20 performed test replicates). Tables 2 and 3 summarize LoDs for all LRT BAL panel microorganisms or antibiotic resistance markers, respectively.

Table 2: LoDs for LRT BAL panel microorganisms.   

<table><tr><td rowspan=1 colspan=1>ReferenceStrain ID</td><td rowspan=1 colspan=1>LoD[CFU/mL]</td></tr><tr><td rowspan=1 colspan=1>Acinetobacter spp.</td><td rowspan=1 colspan=1>ATCC 19606(A. baumannii)</td><td rowspan=1 colspan=1>2.0 x 104</td></tr><tr><td rowspan=1 colspan=1>Chlamydia pneumoniae</td><td rowspan=1 colspan=1>ATCC VR-2282 a</td><td rowspan=1 colspan=1>3.2 x 102</td></tr><tr><td rowspan=1 colspan=1>Citrobacter freundii</td><td rowspan=1 colspan=1>ATCC 8090</td><td rowspan=1 colspan=1>8.0 x 104</td></tr><tr><td rowspan=1 colspan=1>Enterobacter cloacae complex</td><td rowspan=1 colspan=1>ATCC 13047(E. cloacae)</td><td rowspan=1 colspan=1>2.0 x 105</td></tr><tr><td rowspan=1 colspan=1>Escherichia coli</td><td rowspan=1 colspan=1>ATCC 11775</td><td rowspan=1 colspan=1>2.0 x 104</td></tr><tr><td rowspan=1 colspan=1>Haemophilus influenzae</td><td rowspan=1 colspan=1>ATCC 33391</td><td rowspan=1 colspan=1>2.0 x 104</td></tr><tr><td rowspan=1 colspan=1>Klebsiella oxytoca</td><td rowspan=1 colspan=1>ATCC 13182</td><td rowspan=1 colspan=1>1.0 x 104</td></tr><tr><td rowspan=1 colspan=1>Klebsiella pneumoniae (var. 1)</td><td rowspan=1 colspan=1>ATCC 13883</td><td rowspan=1 colspan=1>4.0 x 104</td></tr><tr><td rowspan=1 colspan=1>Klebsiella quasipneumoniae(K. pneumoniae, var. 2)</td><td rowspan=1 colspan=1>ATCC 700603</td><td rowspan=1 colspan=1>4.0 x 104</td></tr><tr><td rowspan=1 colspan=1>Klebsiella variicola</td><td rowspan=1 colspan=1>ATCC BAA-830</td><td rowspan=1 colspan=1>2.0 x 104</td></tr><tr><td rowspan=1 colspan=1>Legionella pneumophila</td><td rowspan=1 colspan=1>ATCC 33152</td><td rowspan=1 colspan=1>8.0 x 104</td></tr><tr><td rowspan=1 colspan=1>Moraxella catarrhalis</td><td rowspan=1 colspan=1>ATCC 25238</td><td rowspan=1 colspan=1>1.5 x 105</td></tr><tr><td rowspan=1 colspan=1>Morganella morganii</td><td rowspan=1 colspan=1>ATCC 25830</td><td rowspan=1 colspan=1>2.0 x 104</td></tr><tr><td rowspan=1 colspan=1>Mycoplasma pneumoniae</td><td rowspan=1 colspan=1>ATCC 29085 b</td><td rowspan=1 colspan=1>1.6 x 103</td></tr><tr><td rowspan=1 colspan=1>Pneumocystis jirovecii</td><td rowspan=1 colspan=1>36_0314c</td><td rowspan=1 colspan=1>5.0 x 105</td></tr><tr><td rowspan=1 colspan=1>Proteus spp.</td><td rowspan=1 colspan=1>ATCC 29906(P. mirabilis)ATCC 29905(P. vulgaris)</td><td rowspan=1 colspan=1>5.0 x 103</td></tr><tr><td rowspan=1 colspan=1>Pseudomonas aeruginosa</td><td rowspan=1 colspan=1>ATCC 10145</td><td rowspan=1 colspan=1>6.3 x 102</td></tr><tr><td rowspan=1 colspan=1>Serratia marcescens</td><td rowspan=1 colspan=1>ATCC 13880</td><td rowspan=1 colspan=1>4.0 x 104</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus aureus</td><td rowspan=1 colspan=1>ATCC 12600</td><td rowspan=1 colspan=1>1.5 x 105</td></tr><tr><td rowspan=1 colspan=1>Stenotrophomonas maltophilia</td><td rowspan=1 colspan=1>ATCC 13637</td><td rowspan=1 colspan=1>5.0 x 103</td></tr><tr><td rowspan=1 colspan=1>Streptococcus pneumoniae</td><td rowspan=1 colspan=1>ATCC 49619</td><td rowspan=1 colspan=1>2.0 x 104</td></tr></table>

a Cell supernatant (in $\mathrm { I F U / m L }$ ). b Bacterial suspension, quantified by qPCR (in copies $/ \mathrm { m L }$ ). c Positive clinical specimen, quantified by $\mathsf { q P C R }$ (in copies $/ \mathrm { m L }$ ). LoD was determined using DNA extracted from a positive clinical specimen, and then confirmed by testing dilutions of this clinical specimen at the LoD concentration (20/20 positive results).

Table 3: LoDs for LRT BAL panel antibiotic resistance markers.   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>ReferenceStrain ID</td><td rowspan=1 colspan=1>Host Microorganism</td><td rowspan=1 colspan=1>LoD[CFU/mL]</td></tr><tr><td rowspan=1 colspan=1>ctx-M</td><td rowspan=1 colspan=1>NCTC 13443</td><td rowspan=1 colspan=1>Klebsiella pneumoniae</td><td rowspan=1 colspan=1>1.0 x 104</td></tr><tr><td rowspan=1 colspan=1>kpc</td><td rowspan=1 colspan=1>NCTC 13438</td><td rowspan=1 colspan=1>Klebsiella pneumoniae</td><td rowspan=1 colspan=1>4.0 x 104</td></tr><tr><td rowspan=1 colspan=1>mecA</td><td rowspan=1 colspan=1>NCTC 12493</td><td rowspan=1 colspan=1>Staphylococcus aureus</td><td rowspan=1 colspan=1>4.0 x 105</td></tr><tr><td rowspan=1 colspan=1>ndm</td><td rowspan=1 colspan=1>NCTC 13443</td><td rowspan=1 colspan=1>Klebsiella pneumoniae</td><td rowspan=1 colspan=1>2.0 x 104</td></tr><tr><td rowspan=1 colspan=1>oxa-23</td><td rowspan=1 colspan=1>NCTC 13301</td><td rowspan=1 colspan=1>Acinetobacter baumannii</td><td rowspan=1 colspan=1>1.0 x 107</td></tr><tr><td rowspan=1 colspan=1>oxa-24</td><td rowspan=1 colspan=1>NCTC 13302</td><td rowspan=1 colspan=1>Acinetobacter baumannii</td><td rowspan=1 colspan=1>5.0 x 103</td></tr><tr><td rowspan=1 colspan=1>oxa-48</td><td rowspan=1 colspan=1>NCTC 13442</td><td rowspan=1 colspan=1>Klebsiella pneumoniae</td><td rowspan=1 colspan=1>3.0 x 105</td></tr><tr><td rowspan=1 colspan=1>oxa-58</td><td rowspan=1 colspan=1>NCTC 13305</td><td rowspan=1 colspan=1>Acinetobacter baumannii</td><td rowspan=1 colspan=1>5.0 x 104</td></tr><tr><td rowspan=1 colspan=1>tem a</td><td rowspan=1 colspan=1>NCTC 13443</td><td rowspan=1 colspan=1>Klebsiella pneumoniae</td><td rowspan=1 colspan=1>2.0 x 104</td></tr><tr><td rowspan=1 colspan=1>vim</td><td rowspan=1 colspan=1>NCTC 13437</td><td rowspan=1 colspan=1>Pseudomonas aeruginosa</td><td rowspan=1 colspan=1>2.0 x 104</td></tr></table>

a Although the LRT BAL Application reports tem results only for $H .$ . influenzae as corresponding host microorganism, LoD was determined with a tem positive $E$ . coli strain. Note that inclusivity testing was also successfully performed with tem positive H. influenzae strains.

# Inclusivity

# LRT BAL Microorganisms

Inclusivity wet testing was performed with reference strains for each microorganism analyte at concentrations near the LoD with contrived samples using pooled native negative lavage specimens as sample matrix (Table 4). Reference strains used for LoD determination are considered inclusive and were used as positive controls. Inclusivity was established near LoD $\mathrm { ( < 5 x \ L o D ) }$ for all but one of the tested reference strains. For one reference strain of $M .$ pneumoniae, tested as genomic DNA extract, inclusivity was demonstrated with a reduced analytical sensitivity at 5x LoD, although no primer or probe deviations to the corresponding LRT BAL assay were present.

Table 4: Inclusivity study results for LRT BAL panel microorganisms.   

<table><tr><td rowspan=1 colspan=1>Reference Strain</td><td rowspan=1 colspan=1>Strain ID</td><td rowspan=1 colspan=1>Test Conc.[CFU/ml]</td><td rowspan=1 colspan=1>x-fold LoD</td><td rowspan=1 colspan=1># Pos./#Exp.</td></tr><tr><td rowspan=1 colspan=5>analyte: Acinetobacter spp.</td></tr><tr><td rowspan=1 colspan=1>Acinetobacter baumannii(pos. control/LoD ref. strain)</td><td rowspan=1 colspan=1>ATCC 19606</td><td rowspan=1 colspan=1>4.0 x 104</td><td rowspan=1 colspan=1>2x</td><td rowspan=1 colspan=1>6/6</td></tr><tr><td rowspan=1 colspan=1>Acinetobacter baumannii</td><td rowspan=1 colspan=1>NCTC 13305</td><td rowspan=1 colspan=1>4.0 x 104</td><td rowspan=1 colspan=1>2x</td><td rowspan=1 colspan=1>2/2</td></tr><tr><td rowspan=1 colspan=1>Acinetobacter baumannii</td><td rowspan=1 colspan=1>NCTC 13301</td><td rowspan=1 colspan=1>4.0 x 104</td><td rowspan=1 colspan=1>2x</td><td rowspan=1 colspan=1>2/2</td></tr><tr><td rowspan=1 colspan=1>Acinetobacter baumannii</td><td rowspan=1 colspan=1>NCTC 13302</td><td rowspan=1 colspan=1>4.0 x 104</td><td rowspan=1 colspan=1>2x</td><td rowspan=1 colspan=1>2/2</td></tr><tr><td rowspan=1 colspan=1>Acinetobacter baumannii</td><td rowspan=1 colspan=1>Micromyx 6148</td><td rowspan=1 colspan=1>4.0 x 104</td><td rowspan=1 colspan=1>2x</td><td rowspan=1 colspan=1>2/2</td></tr><tr><td rowspan=1 colspan=1>Acinetobacter baumannii</td><td rowspan=1 colspan=1>Micromyx 4410</td><td rowspan=1 colspan=1>4.0 x 104</td><td rowspan=1 colspan=1>2x</td><td rowspan=1 colspan=1>2/2</td></tr><tr><td rowspan=1 colspan=1>Acinetobacter baumannii</td><td rowspan=1 colspan=1>JMI 49755</td><td rowspan=1 colspan=1>4.0 x 104</td><td rowspan=1 colspan=1>2x</td><td rowspan=1 colspan=1>2/2</td></tr><tr><td rowspan=1 colspan=1>Acinetobacter baumannii</td><td rowspan=1 colspan=1>NRZ-00449</td><td rowspan=1 colspan=1>1.0 x 104</td><td rowspan=1 colspan=1>0.5x</td><td rowspan=1 colspan=1>2/2</td></tr><tr><td rowspan=1 colspan=1>Acinetobacter baumannii</td><td rowspan=1 colspan=1>UCLA A4</td><td rowspan=1 colspan=1>1.0 x 104</td><td rowspan=1 colspan=1>0.5x</td><td rowspan=1 colspan=1>2/2</td></tr><tr><td rowspan=1 colspan=1>Acinetobacter calcoaceticus</td><td rowspan=1 colspan=1>ATCC 23055</td><td rowspan=1 colspan=1>4.0 x 104</td><td rowspan=1 colspan=1>2x</td><td rowspan=1 colspan=1>2/2</td></tr><tr><td rowspan=1 colspan=1>Acinetobacter lwoffi</td><td rowspan=1 colspan=1>ATCC 15309</td><td rowspan=1 colspan=1>4.0 x 104</td><td rowspan=1 colspan=1>2x</td><td rowspan=1 colspan=1>2/2</td></tr><tr><td rowspan=1 colspan=1>Acinetobacter haemolyticus</td><td rowspan=1 colspan=1>ATCC 17906</td><td rowspan=1 colspan=1>4.0 x 104</td><td rowspan=1 colspan=1>2x</td><td rowspan=1 colspan=1>2/2</td></tr><tr><td rowspan=1 colspan=1>Chlamydia pneumoniae</td><td rowspan=1 colspan=1>ATCC VR-2282</td><td rowspan=1 colspan=1>6.4 x 102 a</td><td rowspan=1 colspan=1>2x</td><td rowspan=1 colspan=1>4/4</td></tr></table>

510(k) Summary – LRT BAL Application K191967 Rev 3.0

<table><tr><td colspan="1" rowspan="1">Reference Strain</td><td colspan="1" rowspan="1">Strain ID</td><td colspan="1" rowspan="1">Test Conc.[CFU/ml|</td><td colspan="1" rowspan="1">x-fold LoD</td><td colspan="1" rowspan="1">#Pos./#Exp.</td></tr><tr><td colspan="1" rowspan="1">osconrol/LoD e.tr</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Chlamydia pneumoniae</td><td colspan="1" rowspan="1">ATCC VR-1310</td><td colspan="1" rowspan="1">6.4x 102 a</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="2">Chlamydia pneumoniae</td><td colspan="1" rowspan="2">ATCC 53592</td><td colspan="1" rowspan="1">6.4 x 102 a</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">0/2</td></tr><tr><td colspan="1" rowspan="1">1.0 x 103 a</td><td colspan="1" rowspan="1">3x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">Citrobacter freundii(pos.control/LoD ref. strain</td><td colspan="1" rowspan="1">ATCC 8090</td><td colspan="1" rowspan="1">1.6 x 105</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">4/4</td></tr><tr><td colspan="1" rowspan="1">Citrobacter freundii</td><td colspan="1" rowspan="1">ATCC 43864</td><td colspan="1" rowspan="1">1.6 x 105</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">Citrobacter freundii</td><td colspan="1" rowspan="1">NCTC 8581</td><td colspan="1" rowspan="1">1.6 x 105</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">Citrobacter freundii</td><td colspan="1" rowspan="1">NRZ-00452</td><td colspan="1" rowspan="1">1.6 x 105</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">Citrobacter freundii</td><td colspan="1" rowspan="1">UCLA C1</td><td colspan="1" rowspan="1">1.6  105</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">analyte:Enterobacter cloacae complex</td><td colspan="4" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Enterobacter cloacaeos.conrol/LoD efti</td><td colspan="1" rowspan="1">ATCC 13047</td><td colspan="1" rowspan="1">4.0 x 105</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">7/7</td></tr><tr><td colspan="1" rowspan="1">Enterobacter cloacae</td><td colspan="1" rowspan="1">ATCC 23355</td><td colspan="1" rowspan="1">4.0 x 10$</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">Enterobacter cloacae</td><td colspan="1" rowspan="1">ATCC 49141</td><td colspan="1" rowspan="1">4.0 x 105</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">Enterobacter cloacae</td><td colspan="1" rowspan="1">ATCC BAA-2468</td><td colspan="1" rowspan="1">4.0 x 105</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">Enterobacter cloacae</td><td colspan="1" rowspan="1">JMI 46239</td><td colspan="1" rowspan="1">4.0 x 105</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">Enterobacter cloacae</td><td colspan="1" rowspan="1">NRZ-00239</td><td colspan="1" rowspan="1">4.0 x 105</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">Enterobacter cloacae ssp. dissolvens</td><td colspan="1" rowspan="1">ATCC 23373</td><td colspan="1" rowspan="1">4.0 x 105</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">Enterobacter hormaechei</td><td colspan="1" rowspan="1">ATCC 49162</td><td colspan="1" rowspan="1">4.0 x 105</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">Enterobacter asburiae</td><td colspan="1" rowspan="1">ATCC 35953</td><td colspan="1" rowspan="1">4.0 x 10$</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli(pos.control/LoD ref. strain</td><td colspan="1" rowspan="1">ATCC 11775</td><td colspan="1" rowspan="1">4.0 x 104</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">4/4</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">ATCC 25922</td><td colspan="1" rowspan="1">4.0 x 104</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">ATCC 35218</td><td colspan="1" rowspan="1">4.0 x 104</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">4/4</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">ATCC BAA-2523</td><td colspan="1" rowspan="1">4.0 x 104</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">NCTC 13351</td><td colspan="1" rowspan="1">4.0 x 104</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">4/4</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">NCTC 13476</td><td colspan="1" rowspan="1">4.0 x 104</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">JMI 50067</td><td colspan="1" rowspan="1">4.0 x 104</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">Haemophilus influenzae(non-typeable/non-capsulated)(pos. control/LoD ref. strain)</td><td colspan="1" rowspan="1">ATCC 33391</td><td colspan="1" rowspan="1">4.0 x 104</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Haemophilus influenzae (serotype )</td><td colspan="1" rowspan="1">ATCC 9006</td><td colspan="1" rowspan="1">4.0 x 104</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">Haemophilus influenzae (seroty </td><td colspan="1" rowspan="1">ATCC 9007</td><td colspan="1" rowspan="1">4.0 x 104</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="2">Haemophilus infuenzae (o </td><td colspan="1" rowspan="2">ATCC 10211</td><td colspan="1" rowspan="1">4.0 x 104</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">1/2</td></tr><tr><td colspan="1" rowspan="1">6.0 x 104</td><td colspan="1" rowspan="1">3x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">Haemophilus influenzae (serotype b)</td><td colspan="1" rowspan="1">ATCC 49247</td><td colspan="1" rowspan="1">4.0 x 104</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">Haemophilus influenzae (serotype b)</td><td colspan="1" rowspan="1">ATCC 49766</td><td colspan="1" rowspan="1">4.0 x 104</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">Klebsiella oxytoca(pos. control/LoD ref. strain)</td><td colspan="1" rowspan="1">ATCC 13182</td><td colspan="1" rowspan="1">4.0 x 104</td><td colspan="1" rowspan="1">4x</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Klebsiella oxytoca</td><td colspan="1" rowspan="1">ATCC 43863</td><td colspan="1" rowspan="1">4.0 x 104</td><td colspan="1" rowspan="1">4x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">Klebsiella oxytoca</td><td colspan="1" rowspan="1">ATCC 8724</td><td colspan="1" rowspan="1">4.0 x 104</td><td colspan="1" rowspan="1">4x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">Klebsiella oxytoca</td><td colspan="1" rowspan="1">ATCC 49131</td><td colspan="1" rowspan="1">4.0 x 104</td><td colspan="1" rowspan="1">4x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">Klebsiella oxytoca</td><td colspan="1" rowspan="1">NCIMB 12819</td><td colspan="1" rowspan="1">4.0 x 104</td><td colspan="1" rowspan="1">4x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">Klebsiella oxytoca</td><td colspan="1" rowspan="1">NRZ-22060</td><td colspan="1" rowspan="1">4.0 x 104</td><td colspan="1" rowspan="1">4x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="5" rowspan="1">analyte: Klebsiella pneumoniae</td></tr><tr><td colspan="1" rowspan="1">Reference Strain</td><td colspan="1" rowspan="1">Strain ID</td><td colspan="1" rowspan="1">Test Conc. CFV/ml</td><td colspan="1" rowspan="1">x-fold LoD</td><td colspan="1" rowspan="1">#Pos./#Exp.</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae, variant 1(pos. control/LoD ref. strain)</td><td colspan="1" rowspan="1">ATCC 13883</td><td colspan="1" rowspan="1">8.0 x 104</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">4/4</td></tr><tr><td colspan="1" rowspan="1">Klebsiella quasipneumoniae(K. pneumoniae, variant 2)(pos. control/LoD ref. strain</td><td colspan="1" rowspan="1">ATCC 700603</td><td colspan="1" rowspan="1">8.0 x 104</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">NCTC 13439</td><td colspan="1" rowspan="1">8.0 x 104</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">NCTC 13438</td><td colspan="1" rowspan="1">8.0 x 104</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">NCTC 13442</td><td colspan="1" rowspan="1">8.0 x 104</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">NCTC 13443</td><td colspan="1" rowspan="1">2.0 x 104</td><td colspan="1" rowspan="1">0.5x</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">Micromyx 4653</td><td colspan="1" rowspan="1">8.0 x 104</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">Micromyx 4676</td><td colspan="1" rowspan="1">8.0 x 104</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">JMI 49767</td><td colspan="1" rowspan="1">2.0 x 104</td><td colspan="1" rowspan="1">0.5x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">NRZ-00002</td><td colspan="1" rowspan="1">2.0 x 104</td><td colspan="1" rowspan="1">0.5x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">NRZ-00103</td><td colspan="1" rowspan="1">8.0 x 104</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">Klebsiella variicolas.conolLo ef i</td><td colspan="1" rowspan="1">ATCC BAA-830</td><td colspan="1" rowspan="1">4.0 x 104</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Klebsiella variicola</td><td colspan="1" rowspan="1">clinical strain 1</td><td colspan="1" rowspan="1">4.0 x 104</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">Klebsiella varicola</td><td colspan="1" rowspan="1">clinical strain 2</td><td colspan="1" rowspan="1">4.0 x 104</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">Klebsiella variicola</td><td colspan="1" rowspan="1">clinical strain 3</td><td colspan="1" rowspan="1">4.0 x 104</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">Klebsiella variicola</td><td colspan="1" rowspan="1">clinical strain 4</td><td colspan="1" rowspan="1">4.0 x 104</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">Legionella pneumophila (serotype 1)(pos. control/LoD ref. strain)</td><td colspan="1" rowspan="1">ATCC 33152</td><td colspan="1" rowspan="1">1.6 x 105</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">7/7</td></tr><tr><td colspan="1" rowspan="1">Legionella pneumophila (serotype</td><td colspan="1" rowspan="1">ATCC 33154</td><td colspan="1" rowspan="1">1.6 x 105</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">Legionella pneumophila (serotype 3)</td><td colspan="1" rowspan="1">ATCC 33155</td><td colspan="1" rowspan="1">1.6 x 105</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">Legionella pneumophila (serotype )</td><td colspan="1" rowspan="1">ATCC 33215</td><td colspan="1" rowspan="1">1.6 x 105</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">Legionella pneumophila (serotype 8</td><td colspan="1" rowspan="1">ATCC 35096</td><td colspan="1" rowspan="1">1.6 x 105</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">Legionella pneumophila (serotype 10)</td><td colspan="1" rowspan="1">ATCC 43283</td><td colspan="1" rowspan="1">1.6 x 105</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">Legionella pneumophila</td><td colspan="1" rowspan="1">UCLA LI</td><td colspan="1" rowspan="1">1.6 x 105</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">Legionella pneumophila</td><td colspan="1" rowspan="1">UCLA L5</td><td colspan="1" rowspan="1">1.6 x 105</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">Legionella pneumophila</td><td colspan="1" rowspan="1">UCLA L6</td><td colspan="1" rowspan="1">1.6 x 105</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">Moraxella catarrhalis(pos. control/LoD ref. strain)</td><td colspan="1" rowspan="1">ATCC 25238</td><td colspan="1" rowspan="1">3.0 x 105</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">4/4</td></tr><tr><td colspan="1" rowspan="1">Moraxella catarrhalis</td><td colspan="1" rowspan="1">ATCC 43617</td><td colspan="1" rowspan="1">3.0 x 105</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">Moraxella catarrhalis</td><td colspan="1" rowspan="1">ATCC 8176</td><td colspan="1" rowspan="1">3.0 x 105</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">Moraxella catarrhalis</td><td colspan="1" rowspan="1">ATCC 25240</td><td colspan="1" rowspan="1">3.0 x 105</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">Moraxella catarrhalis</td><td colspan="1" rowspan="1">ATCC 23246</td><td colspan="1" rowspan="1">3.0 x 105</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Moraxella catarrhalis</td><td colspan="1" rowspan="1">ATCC 49143</td><td colspan="1" rowspan="1">3.0 x 105</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">Morganella morganii(pos.control/LoD ef. train</td><td colspan="1" rowspan="1">ATCC 25830</td><td colspan="1" rowspan="1">4.0 x 104</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/3</td></tr><tr><td colspan="1" rowspan="1">Morganella morganii</td><td colspan="1" rowspan="1">ATCC 8019</td><td colspan="1" rowspan="1">4.0 x 104</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="2">Morganella morganii</td><td colspan="1" rowspan="2">ATCC 25829</td><td colspan="1" rowspan="1">4.0 x 104</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">0/1</td></tr><tr><td colspan="1" rowspan="1">6.0 x 104</td><td colspan="1" rowspan="1">3x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="2">Morganella morganii</td><td colspan="1" rowspan="2">DSM-46262</td><td colspan="1" rowspan="1">4.0 x 104</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">1/2</td></tr><tr><td colspan="1" rowspan="1">6.0 x 104</td><td colspan="1" rowspan="1">3x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">Morganella morganiissp. son</td><td colspan="1" rowspan="1">ATCC 49948</td><td colspan="1" rowspan="1">4.0 x 104</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">Mycoplasma pneumoniae (type 1)(pos. control/LoD ref. strain)</td><td colspan="1" rowspan="1">ATCC 29085</td><td colspan="1" rowspan="1">3.2 x 103 b</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">5/7</td></tr><tr><td colspan="1" rowspan="1">Reference Strain</td><td colspan="1" rowspan="1">Strain ID</td><td colspan="1" rowspan="1">Test Conc. CV/ml</td><td colspan="1" rowspan="1">x-fold LoD</td><td colspan="1" rowspan="1"># Pos./#Exp.</td></tr><tr><td colspan="1" rowspan="1">Mycoplasma pneumoniae (type 1)</td><td colspan="1" rowspan="1">ATCC 29343</td><td colspan="1" rowspan="1">3.2 x 103 b</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="5">Mycoplasma pneumoniae (type 2)</td><td colspan="1" rowspan="5">ATCC 15531 c, d</td><td colspan="1" rowspan="1">3.2 x 103 b</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">0/2</td></tr><tr><td colspan="1" rowspan="1">5.0 x 103b</td><td colspan="1" rowspan="1">3x</td><td colspan="1" rowspan="1">1/2</td></tr><tr><td colspan="1" rowspan="1">6.4x 103 b</td><td colspan="1" rowspan="1">4x</td><td colspan="1" rowspan="1">1/2</td></tr><tr><td colspan="1" rowspan="1">7.0 x 103 b</td><td colspan="1" rowspan="1">4.4x</td><td colspan="1" rowspan="1">1/2</td></tr><tr><td colspan="1" rowspan="1">8.0 x 103 b</td><td colspan="1" rowspan="1">5x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">Mycoplasma pneumoniae (type 1)</td><td colspan="1" rowspan="1">ATCC 15293</td><td colspan="1" rowspan="1">3.2 x 103 b</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">Mycoplasma pneumoniae (type 2)</td><td colspan="1" rowspan="1">ATCC 49894</td><td colspan="1" rowspan="1">3.2 x 103 b</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">Pneumocystis jirovecii(pos. control/LoD ref. strain)</td><td colspan="1" rowspan="1">pos. specimen 1</td><td colspan="1" rowspan="1">1.0 x 106 b</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">1/1</td></tr><tr><td colspan="1" rowspan="1">Pneumocystis jirovecii</td><td colspan="1" rowspan="1">pos. specimen 2</td><td colspan="1" rowspan="1">1.0 x 106b</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">Pneumocystis jirovecii</td><td colspan="1" rowspan="1">pos. specimen 3</td><td colspan="1" rowspan="1">1.0 x 106b</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">Pneumocystis jirovecii</td><td colspan="1" rowspan="1">pos. specimen 4</td><td colspan="1" rowspan="1">1.0 x 106b</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">Pneumocystis jirovecii</td><td colspan="1" rowspan="1">pos. specimen 5</td><td colspan="1" rowspan="1">1.0 x 106b</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">analyte: Proteus spp.</td><td colspan="4" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Proteus vulgaris(pos.cool/LoD e</td><td colspan="1" rowspan="1">ATCC 29905</td><td colspan="1" rowspan="1">1.0 x 104</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">7/7</td></tr><tr><td colspan="1" rowspan="1">Proteus mirabilis(pos.cool/LoD e</td><td colspan="1" rowspan="1">ATCC 29906</td><td colspan="1" rowspan="1">1.0 x 104</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">Proteus mirabilis</td><td colspan="1" rowspan="1">ATCC 12453</td><td colspan="1" rowspan="1">1.0 x 104</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">Proteus mirabilis</td><td colspan="1" rowspan="1">ATCC 14153</td><td colspan="1" rowspan="1">1.0 x 104</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">Proteus mirabilis</td><td colspan="1" rowspan="1">ATCC 25933</td><td colspan="1" rowspan="1">1.0 x 104</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">Proteus vulgaris</td><td colspan="1" rowspan="1">ATCC 6380</td><td colspan="1" rowspan="1">1.0 x 104</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">Proteus vulgaris</td><td colspan="1" rowspan="1">ATCC 8427</td><td colspan="1" rowspan="1">1.0 x 104</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">Proteus hauseri</td><td colspan="1" rowspan="1">ATCC 700826</td><td colspan="1" rowspan="1">1.0 x 104</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">Proteus penneri</td><td colspan="1" rowspan="1">ATCC 33519</td><td colspan="1" rowspan="1">1.0 x 104</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">Pseudomonas aeruginosa(pos. control/LoD ref. strain)</td><td colspan="1" rowspan="1">ATCC 10145</td><td colspan="1" rowspan="1">1.3 x 103</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">ATCC 27853</td><td colspan="1" rowspan="1">1.3 x 103</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">NCTC 13437</td><td colspan="1" rowspan="1">1.3 x 103</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">Micromyx 2562</td><td colspan="1" rowspan="1">2.0 x 103</td><td colspan="1" rowspan="1">3x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">NRZ-00196</td><td colspan="1" rowspan="1">1.3 x 103</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">UCLA P20</td><td colspan="1" rowspan="1">1.3 x 103</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">Serratia marcescens(pos. control/LoD ref. strain)</td><td colspan="1" rowspan="1">ATCC 13880</td><td colspan="1" rowspan="1">8.0 x 104</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Serratia marcescens</td><td colspan="1" rowspan="1">ATCC 14756</td><td colspan="1" rowspan="1">8.0 x 104</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">Serratia marcescens</td><td colspan="1" rowspan="1">ATCC 15365</td><td colspan="1" rowspan="1">8.0 x 104</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">Serratia marcescens</td><td colspan="1" rowspan="1">ATCC 27117</td><td colspan="1" rowspan="1">8.0 x 104</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">Serratia marcescens</td><td colspan="1" rowspan="1">ATCC 43861</td><td colspan="1" rowspan="1">8.0 x 104</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">Serratia marcescens ssp. sakuensis</td><td colspan="1" rowspan="1">DSM-17174</td><td colspan="1" rowspan="1">8.0 x 104</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus(pos. control/LoD ref. strain</td><td colspan="1" rowspan="1">ATCC 12600</td><td colspan="1" rowspan="1">3.0 x 105</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">7/7</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">ATCC BAA-2312</td><td colspan="1" rowspan="1">3.0 x 105</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">NCTC 12493</td><td colspan="1" rowspan="1">3.0 x 105</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">ATCC 33591</td><td colspan="1" rowspan="1">3.0 x 105</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">DSM-17091</td><td colspan="1" rowspan="1">3.0 x 105</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">Reference Strain</td><td colspan="1" rowspan="1">Strain ID</td><td colspan="1" rowspan="1">Test Conc.CFU/ml</td><td colspan="1" rowspan="1">x-fold LoD</td><td colspan="1" rowspan="1"># Pos./#Exp.</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">ATCC 29213</td><td colspan="1" rowspan="1">3.0 x 105</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">ATCC 43300</td><td colspan="1" rowspan="1">3.0 x 105</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">Stenotrophomonas maltophilia(pos. control/LoD ref. strain)</td><td colspan="1" rowspan="1">ATCC 13637</td><td colspan="1" rowspan="1">1.0 x 104</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">4/4</td></tr><tr><td colspan="1" rowspan="2">Stenotrophomonas maltophilia</td><td colspan="1" rowspan="2">ATCC 13636</td><td colspan="1" rowspan="1">1.0 x 104</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">0/2</td></tr><tr><td colspan="1" rowspan="1">1.5 x 104</td><td colspan="1" rowspan="1">3x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">Stenotrophomonas maltophilia</td><td colspan="1" rowspan="1">ATCC 17666</td><td colspan="1" rowspan="1">1.0 x 104</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">Stenotrophomonas maltophilia</td><td colspan="1" rowspan="1">ATCC 49130</td><td colspan="1" rowspan="1">1.0 x 104</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">Stenotrophomonas maltophilia</td><td colspan="1" rowspan="1">DSM-50173</td><td colspan="1" rowspan="1">1.0 x 104</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="3">Stenotrophomonas maltophilia</td><td colspan="1" rowspan="3">DSM-21874 [NCIMB 9528]</td><td colspan="1" rowspan="1">1.0 x 104</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">0/2</td></tr><tr><td colspan="1" rowspan="1">1.5 x 104</td><td colspan="1" rowspan="1">3x</td><td colspan="1" rowspan="1">0/2</td></tr><tr><td colspan="1" rowspan="1">2.0 x 104</td><td colspan="1" rowspan="1">4x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">Streptococcus pneumoniaeserotype 19F(pos. control/LoD ref. strain)</td><td colspan="1" rowspan="1">ATCC 49619</td><td colspan="1" rowspan="1">4.0 x 104</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">4/7</td></tr><tr><td colspan="1" rowspan="1">Streptococcus pneumoniae serotype 3</td><td colspan="1" rowspan="1">ATCC 6303</td><td colspan="1" rowspan="1">4.0 x 104</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">Streptococcus pneumoniae serotype 5</td><td colspan="1" rowspan="1">ATCC 6305</td><td colspan="1" rowspan="1">4.0 x 104</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="2">Streptococcus pneumoniae</td><td colspan="1" rowspan="2">ATCC 49150 g</td><td colspan="1" rowspan="1">4.0 x 104</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">0/2</td></tr><tr><td colspan="1" rowspan="1">8.0 x 104</td><td colspan="1" rowspan="1">4x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="3">Streptococcus pneumoniae</td><td colspan="1" rowspan="3">ATCC 33400 h</td><td colspan="1" rowspan="1">4.0 x 104</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">1/2</td></tr><tr><td colspan="1" rowspan="1">6.0 x 104</td><td colspan="1" rowspan="1">3x</td><td colspan="1" rowspan="1">1/2</td></tr><tr><td colspan="1" rowspan="1">8.0 x 104</td><td colspan="1" rowspan="1">4x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">Streptococcus pneumoniae serotype 2</td><td colspan="1" rowspan="1">ATCC 27336</td><td colspan="1" rowspan="1">4.0 x 104</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">Streptococcus pneumoniae serotype 1</td><td colspan="1" rowspan="1">ATCC 6301</td><td colspan="1" rowspan="1">4.0 x 104</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">Streptococcus pneumoniae serotype 9V</td><td colspan="1" rowspan="1">DSM-11865</td><td colspan="1" rowspan="1">4.0 x 104</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="3">Streptococcus pneumoniae serotype 23F</td><td colspan="1" rowspan="3">DSM-11866</td><td colspan="1" rowspan="1">4.0 x 104</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">6.0 x 104</td><td colspan="1" rowspan="1">3x</td><td colspan="1" rowspan="1">1/2</td></tr><tr><td colspan="1" rowspan="1">8.0 x 104</td><td colspan="1" rowspan="1">4x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">Streptococcus pneumoniae serotype 6B</td><td colspan="1" rowspan="1">DSM-11867</td><td colspan="1" rowspan="1">4.0 x 104</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr></table>

To supplement inclusivity testing, in silico GenBank BLAST analyses of LRT BAL panel primer and probe sequences were performed for each LRT BAL panel microorganism (search performed July 2019) for all applicable strain entries. BLAST analyses identified strain entries for which detection by LRT BAL is predicted at LoD (match of relevant primer and probe sequences), only with reduced sensitivity (typically, single relevant mismatches of primer or probe sequences; detection likely at higher than LoD concentrations only), or is not predicted (multiple relevant mismatches in primer and probe sequences).

Table 5 lists microorganisms for which inclusivity was demonstrated by wet testing and are supported by in silico analysis (predicted at LoD or predicted with reduced sensitivity). Note that additional in silico analysis results are provided for strains that have not been wet tested. Such in silico results are provided as supplementary data only. Results are not intended to be a surrogate for wet testing and do not assure that specific strains will be detected.

NOTE: The performance of the Unyvero LRT BAL Application has not been established for those microorganism species that were evaluated by in silico analysis only.

Table 5: Inclusive LRT BAL panel microorganisms as demonstrated by inclusivity wet testing and/or as predicted by in silico analysis (at LoD concentration or with reduced sensitivity). For strains for which in silico analysis predicts higher than LoD concentrations for one or more applicable entries, numbers of GenBank entries are additionally listed.   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>WetTesting</td><td rowspan=1 colspan=1>In Silico:at LoD</td><td rowspan=1 colspan=1>In Silico:ReducedSensitivity</td><td rowspan=1 colspan=1>Comment</td></tr><tr><td rowspan=1 colspan=3>Acinetobacter spp.</td><td rowspan=1 colspan=2>EY</td></tr><tr><td rowspan=1 colspan=1>A. baumannii</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>A. calcoaceticus</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>A. lwoffii</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>A. haemolyticus</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>A. nosocomialis</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>A. pittii</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>A. junii</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>A. parvus</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>A. lactucae</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>A. oleivorans</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>A. schindleri</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>A. guillouiae</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>X1 entry</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>A. radioresistens</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>X3 entries</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>A. soli</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>X1 entry</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>A. ursingii</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>X1 entry</td><td rowspan=1 colspan=1>reference strain DSM-16037 testednegative at 1.5 x 107 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Chlamydia pneumoniae</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>E</td></tr><tr><td rowspan=1 colspan=1>Citrobacter freundii</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>x26 entries</td><td rowspan=1 colspan=1>X4 entries</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Enterobacter cloacae complex</td><td></td><td></td><td></td><td></td></tr><tr><td rowspan=1 colspan=1>E. cloacae</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>including ssp. dissolvens</td></tr></table>

510(k) Summary – LRT BAL Application K191967 Rev 3.0

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>WetTesting</td><td rowspan=1 colspan=1>In Silico:at LoD</td><td rowspan=1 colspan=1>In Silico:Reduced Sensitivity</td><td rowspan=1 colspan=1>Comment</td></tr><tr><td rowspan=1 colspan=1>E. hormaechei</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>including ssp. oharae, steigerwaltii,hoffmannii</td></tr><tr><td rowspan=1 colspan=1>E. hormaechei ssp.xiangfangensis</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>X7 entries</td><td rowspan=1 colspan=1>X1 entry</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>E. kobei</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>E. ludwigii</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>cYanmaGellowb</td></tr><tr><td rowspan=1 colspan=1>E. sichuanensis</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>E. chengduensis</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>E. chuandaensis</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>E. roggenkampii</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>E. asburiae</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X11 entries</td><td rowspan=1 colspan=1>X1 entry</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>E. bugandensis</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>not yet recognized as member of theE. cloacae complex</td></tr><tr><td rowspan=1 colspan=1>Escherichia coli</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>x</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Haemophilus influenzae</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X65 entries</td><td rowspan=1 colspan=1>x1 entry</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Klebsiella oxytoca</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X16 entries</td><td rowspan=1 colspan=1>x6 entries</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Klebsiella pneumoniae</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>variant 1 (K. pneumoniae)</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>variant 2 (K. quasipneumoniae)</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X11 entries</td><td rowspan=1 colspan=1>X10 entries</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Klebsiella variicola</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Legionella pneumophila</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>some entries predict detection atLoD, other entries predict a possibleslight performance reduction; strainsfor both variants were included ininclusivity wet testing and weredetected at concentrations near LoD.</td></tr><tr><td rowspan=1 colspan=1>Moraxella catarrhalis</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Morganella morganii</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Mycoplasma pneumoniae</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Pneumocystis jirovecii</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>E</td></tr><tr><td rowspan=1 colspan=1>Proteus spp.</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>P. mirabilis</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>P. vulgaris</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>P. hauseri</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>P. penneri</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>P. cibarius</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Pseudomonas aeruginosa</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Serratia marcescens</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>x</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Staphylococcus aureus</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>x&gt; 450entries</td><td rowspan=1 colspan=1>X1 entry</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Stenotrophomonas maltophilia</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Streptococcus pneumoniae</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>&gt;20$entries</td><td rowspan=1 colspan=1>X1 entry</td><td rowspan=1 colspan=1>including serotype 7F</td></tr></table>

# LRT BAL Antibiotic Resistance Markers

Similar to microorganisms, inclusivity wet testing was performed with reference strains carrying antibiotic resistance markers of the LRT BAL panel at concentrations near LoD with contrived samples using pooled negative native lavage specimens as sample matrix (Table 6). Subgroups of antibiotic resistance markers are added, if known (NA: unknown). Where phenotypic antimicrobial susceptibility testing (AST) results are available, this information is also shown (e.g., carbapenemR $^ { \downarrow } =$ resistant to carbapenems, carbapenem ${ \bf \Delta } ^ { \mathrm { S } } =$ susceptible to carbapenems).

Inclusivity was established near LoD for all tested reference strains.

Table 6: Inclusivity study results for LRT BAL panel antibiotic resistance markers.   

<table><tr><td colspan="1" rowspan="1">Analyte</td><td colspan="1" rowspan="1">Subgroup</td><td colspan="1" rowspan="1">Reference Strain</td><td colspan="1" rowspan="1">Strain ID</td><td colspan="1" rowspan="1">Test Conc.[CFU/ml]</td><td colspan="1" rowspan="1">x-fold LoD</td><td colspan="1" rowspan="1"># Pos./# Exp.</td></tr><tr><td colspan="1" rowspan="11">ctx-M</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">Klebsiella pneumoniae(pos. control/LoD ref. strain)</td><td colspan="1" rowspan="1">NCTC 13443</td><td colspan="1" rowspan="1">2.0 x 104</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">ctx-M3</td><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">NRZ-00751</td><td colspan="1" rowspan="1">2.0 x 104</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">ctx-M15</td><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">NRZ-00249</td><td colspan="1" rowspan="1">2.0 x 104</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="2">NA</td><td colspan="1" rowspan="2">Enterobacter cloacae</td><td colspan="1" rowspan="2">JMI 46239</td><td colspan="1" rowspan="1">2.0 x 104</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">1/2</td></tr><tr><td colspan="1" rowspan="1">3.0 x 104</td><td colspan="1" rowspan="1">3x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">JMI 50067</td><td colspan="1" rowspan="1">2.0 x 104</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="2">NA</td><td colspan="1" rowspan="2">Klebsiella pneumoniae</td><td colspan="1" rowspan="2">NRZ-00002</td><td colspan="1" rowspan="1">2.0 x 104</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">1/2</td></tr><tr><td colspan="1" rowspan="1">3.0 x 104</td><td colspan="1" rowspan="1">3x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">Enterobacter cloacae</td><td colspan="1" rowspan="1">ATCC BAA-2468</td><td colspan="1" rowspan="1">4.0 x 104</td><td colspan="1" rowspan="1">4x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">JMI 49831</td><td colspan="1" rowspan="1">4.0 x 104</td><td colspan="1" rowspan="1">4x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">JMI 49767</td><td colspan="1" rowspan="1">2.0 x 104</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="7">kpc</td><td colspan="1" rowspan="1">kpc-3</td><td colspan="1" rowspan="1">Klebsiella pneumoniae(pos. control/LoD ref. strain)</td><td colspan="1" rowspan="1">NCTC 13438</td><td colspan="1" rowspan="1">8.0 x 104</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">5/5</td></tr><tr><td colspan="1" rowspan="1">kpc-2</td><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">NRZ-00281</td><td colspan="1" rowspan="1">8.0 x 104</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">kpc-2</td><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">NRZ-00103</td><td colspan="1" rowspan="1">8.0 x 104</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="2">kpc-3</td><td colspan="1" rowspan="2">Escherichia coli</td><td colspan="1" rowspan="2">NRZ-00222</td><td colspan="1" rowspan="1">8.0 x 104</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">1/2</td></tr><tr><td colspan="1" rowspan="1">1.2 x 105</td><td colspan="1" rowspan="1">3x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">kpc-3</td><td colspan="1" rowspan="1">Klebsiella pneumoniae(carbapenem)</td><td colspan="1" rowspan="1">Micromyx 4653</td><td colspan="1" rowspan="1">8.0 x 104</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">4/4</td></tr><tr><td colspan="1" rowspan="1">kpc-3</td><td colspan="1" rowspan="1">Klebsiella pneumoniae(carbapenem)</td><td colspan="1" rowspan="1">Micromyx 4676</td><td colspan="1" rowspan="1">8.0 x 104</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">4/4</td></tr><tr><td colspan="1" rowspan="8">mecA</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">Staphylococcus aureus(methicillin", cefoxitin)(pos. control/LoD ref. strain)</td><td colspan="1" rowspan="1">NCTC 12493</td><td colspan="1" rowspan="1">8.0 x 105</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">SCCmecI</td><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">RKI 07-03165</td><td colspan="1" rowspan="1">8.0 x 105</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">SCCmecII</td><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">RKI 01-00694</td><td colspan="1" rowspan="1">8.0 x 105</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">SCCmecII</td><td colspan="1" rowspan="1">Staphylococcus aureus(methicillin)</td><td colspan="1" rowspan="1">ATCC 33591</td><td colspan="1" rowspan="1">8.0 x 105</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">SCCmecIV</td><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">RKI 09-00187</td><td colspan="1" rowspan="1">8.0 x 105</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">SCCmecV</td><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">RKI 08-02492</td><td colspan="1" rowspan="1">8.0 x 105</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">Staphylococcus aureus(methicillinR)</td><td colspan="1" rowspan="1">DSM-17091</td><td colspan="1" rowspan="1">8.0 x 105</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">SCCmecII</td><td colspan="1" rowspan="1">Staphylococcus aureus(methicillinR)</td><td colspan="1" rowspan="1">ATCC 43300</td><td colspan="1" rowspan="1">3.0 x 105</td><td colspan="1" rowspan="1">0.8x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">Analyte</td><td colspan="1" rowspan="1">Subgroup</td><td colspan="1" rowspan="1">Reference Strain</td><td colspan="1" rowspan="1">Strain ID</td><td colspan="1" rowspan="1">Test Conc.C/ml</td><td colspan="1" rowspan="1">x-fold LoD</td><td colspan="1" rowspan="1">#Pos./# Exp.</td></tr><tr><td colspan="1" rowspan="8">ndm</td><td colspan="1" rowspan="1">ndm-1</td><td colspan="1" rowspan="1">Klebsiella pneumoniae(pos. control/LoD ref. strain)</td><td colspan="1" rowspan="1">NCTC 13443</td><td colspan="1" rowspan="1">4.0 x 104</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">5/5</td></tr><tr><td colspan="1" rowspan="1">ndm-1</td><td colspan="1" rowspan="1">Acinetobacter baumannii</td><td colspan="1" rowspan="1">JMI 49755</td><td colspan="1" rowspan="1">4.0 x 104</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">4/4</td></tr><tr><td colspan="1" rowspan="2">ndm-1</td><td colspan="1" rowspan="2">Enterobacter cloacae (imipenemR,erttapenemR)</td><td colspan="1" rowspan="2">ATCC BAA-2468</td><td colspan="1" rowspan="1">4.0 x 104</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">1/2</td></tr><tr><td colspan="1" rowspan="1">6.0 x 104</td><td colspan="1" rowspan="1">3x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">ndm-1</td><td colspan="1" rowspan="1">Enterobacter cloacae</td><td colspan="1" rowspan="1">JMI 46239</td><td colspan="1" rowspan="1">4.0 x 104</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">4/4</td></tr><tr><td colspan="1" rowspan="1">ndm-1</td><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">JMI 50067</td><td colspan="1" rowspan="1">4.0 x 104</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">ndm-1</td><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">JMI 49767</td><td colspan="1" rowspan="1">2.0 x 104</td><td colspan="1" rowspan="1">1x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">ndm-1</td><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">JMI 49831</td><td colspan="1" rowspan="1">4.0 x 104</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="6">oxa-23</td><td colspan="1" rowspan="1">oxa-23</td><td colspan="1" rowspan="1">Acinetobacter baumannii(pos. control/LoD ref. strain)</td><td colspan="1" rowspan="1">NCTC 13301</td><td colspan="1" rowspan="1">2.0 x 107</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">Acinetobacter baumannii( carbhapenemR)</td><td colspan="1" rowspan="1">Micromyx 4410</td><td colspan="1" rowspan="1">2.0 x 107</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">Acinetobacter baumannii(carbapenem)</td><td colspan="1" rowspan="1">Micromyx 6148</td><td colspan="1" rowspan="1">2.0 x 107</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">Acinetobacter baumannii(carbapenem)</td><td colspan="1" rowspan="1">Micromyx 6149</td><td colspan="1" rowspan="1">2.0 x 107</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">Acinetobacter baumannii(carbapenem)</td><td colspan="1" rowspan="1">Micromyx 6153</td><td colspan="1" rowspan="1">2.0 x 107</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">Acinetobacter baumannii(imipenemR, meropenemR)</td><td colspan="1" rowspan="1">UCLA A5</td><td colspan="1" rowspan="1">2.0 x 107</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="5">oxa-24</td><td colspan="1" rowspan="1">oxa-25</td><td colspan="1" rowspan="1">Acinetobacter baumannii(pos. control/LoD ref. strain)</td><td colspan="1" rowspan="1">NCTC 13302</td><td colspan="1" rowspan="1">1.0 x 104</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">oxa-72</td><td colspan="1" rowspan="1">Acinetobacter baumannii</td><td colspan="1" rowspan="1">NRZ-00449</td><td colspan="1" rowspan="1">1.0 x 104</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">Acinetobacter baumannii(imipenemR, meropenem)</td><td colspan="1" rowspan="1">UCLA A4</td><td colspan="1" rowspan="1">1.0 x 104</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">Acinetobacter baumannii(imipenemR, meropenem)</td><td colspan="1" rowspan="1">clinical strain 1</td><td colspan="1" rowspan="1">1.0 x 104</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">Acinetobacter baumannii(imipenemR, meropenem)</td><td colspan="1" rowspan="1">clinical strain 2</td><td colspan="1" rowspan="1">1.0 x 104</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="7">oxa-48</td><td colspan="1" rowspan="1">oxa-48</td><td colspan="1" rowspan="1">Klebsiella pneumoniae(pos. control/LoD ref. strain)</td><td colspan="1" rowspan="1">NCTC 13442</td><td colspan="1" rowspan="1">6.0 x 105</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">oxa-48</td><td colspan="1" rowspan="1">Escherichia coli(ertapemR)</td><td colspan="1" rowspan="1">ATCC BAA-2523</td><td colspan="1" rowspan="1">6.0 x 105</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="2">oxa-48</td><td colspan="1" rowspan="2">Escherichia coli</td><td colspan="1" rowspan="2">NRZ-00176</td><td colspan="1" rowspan="1">6.0 x 105</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">1/2</td></tr><tr><td colspan="1" rowspan="1">9.0 x 105</td><td colspan="1" rowspan="1">3x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">oxa-162</td><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">NRZ-00361</td><td colspan="1" rowspan="1">6.0 x 105</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">oxa-162</td><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">NRZ-00472</td><td colspan="1" rowspan="1">6.0 x 105</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">oxa-232</td><td colspan="1" rowspan="1">Klebsiella oxytoca</td><td colspan="1" rowspan="1">NRZ-22060</td><td colspan="1" rowspan="1">6.0 x 105</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="2">oxa-58</td><td colspan="1" rowspan="1">oxa-58</td><td colspan="1" rowspan="1">Acinetobacter baumannii(pos. control/LoD ref. strain)</td><td colspan="1" rowspan="1">NCTC 13305</td><td colspan="1" rowspan="1">1.0 x 105</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/3</td></tr><tr><td colspan="1" rowspan="1">oxa-58</td><td colspan="1" rowspan="1">Acinetobacter baumannii</td><td colspan="1" rowspan="1">NRZ-00518</td><td colspan="1" rowspan="1">1.0 x 105</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="8">tem</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">Klebsiella pneumoniae(pos. control/LoD ref. strain)</td><td colspan="1" rowspan="1">NCTC 13443</td><td colspan="1" rowspan="1">4.0 x 104</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">tem-1</td><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">ATCC 35218</td><td colspan="1" rowspan="1">4.0 x 104</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">tem-3</td><td colspan="1" rowspan="1">Escherichia coli(ESSBL)</td><td colspan="1" rowspan="1">NCTC 13351</td><td colspan="1" rowspan="1">4.0 x 104</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">Citrobacter freundii</td><td colspan="1" rowspan="1">ATCC 43864</td><td colspan="1" rowspan="1">4.0 x 104</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">Enterobacter cloacae</td><td colspan="1" rowspan="1">JMI 46239</td><td colspan="1" rowspan="1">4.0 x 104</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">JMI 50067</td><td colspan="1" rowspan="1">2.0 x 104</td><td colspan="1" rowspan="1">1x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">NRZ-00751</td><td colspan="1" rowspan="1">2.0 x 104</td><td colspan="1" rowspan="1">1x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">ATCC BAA-2523</td><td colspan="1" rowspan="1">4.0 x 104</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr></table>

510(k) Summary – LRT BAL Application K191967 Rev 3.0

<table><tr><td rowspan=1 colspan=1>Analyte</td><td rowspan=1 colspan=1>Subgroup</td><td rowspan=1 colspan=1>Reference Strain</td><td rowspan=1 colspan=1>Strain ID</td><td rowspan=1 colspan=1>Test Conc.[CFU/ml|</td><td rowspan=1 colspan=1>x-fold LoD</td><td rowspan=1 colspan=1># Pos./#Exp.</td></tr><tr><td rowspan=4 colspan=1></td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>Acinetobacter baumannii</td><td rowspan=1 colspan=1>JMI 49755</td><td rowspan=1 colspan=1>4.0 x 104</td><td rowspan=1 colspan=1>2x</td><td rowspan=1 colspan=1>2/2</td></tr><tr><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>Haemophilus influenzae(ampicillinR, cefinase)</td><td rowspan=1 colspan=1>clinical strain 1</td><td rowspan=1 colspan=1>4.0 x 104</td><td rowspan=1 colspan=1>2x</td><td rowspan=1 colspan=1>2/2</td></tr><tr><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>Haemophilus influenzae(cefinaseR)</td><td rowspan=1 colspan=1>clinical strain 2</td><td rowspan=1 colspan=1>4.0 x 104</td><td rowspan=1 colspan=1>2x</td><td rowspan=1 colspan=1>2/2</td></tr><tr><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>Klebsiella pneumoniae</td><td rowspan=1 colspan=1>Micromyx 4676</td><td rowspan=1 colspan=1>8.0 x 104</td><td rowspan=1 colspan=1>4x</td><td rowspan=1 colspan=1>2/2</td></tr><tr><td rowspan=8 colspan=1>vim</td><td rowspan=1 colspan=1>vim-10</td><td rowspan=1 colspan=1>Pseudomonas aeruginosa(pos. control/LoD ref. strain)</td><td rowspan=1 colspan=1>NCTC 13437</td><td rowspan=1 colspan=1>4.0 x 104</td><td rowspan=1 colspan=1>2x</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>vim-1</td><td rowspan=1 colspan=1>Citrobacter freundii</td><td rowspan=1 colspan=1>NRZ-00452</td><td rowspan=1 colspan=1>4.0 x 104</td><td rowspan=1 colspan=1>2x</td><td rowspan=1 colspan=1>2/2</td></tr><tr><td rowspan=1 colspan=1>vim-1</td><td rowspan=1 colspan=1>Pseudomonas aeruginosa(ceftazidimeR, imipenem)</td><td rowspan=1 colspan=1>DSM-24600</td><td rowspan=1 colspan=1>4.0 x 104</td><td rowspan=1 colspan=1>2x</td><td rowspan=1 colspan=1>2/2</td></tr><tr><td rowspan=1 colspan=1>vim-1</td><td rowspan=1 colspan=1>Enterobacter cloacae</td><td rowspan=1 colspan=1>NRZ-00239</td><td rowspan=1 colspan=1>4.0 x 104</td><td rowspan=1 colspan=1>2x</td><td rowspan=1 colspan=1>2/2</td></tr><tr><td rowspan=1 colspan=1>vim-1</td><td rowspan=1 colspan=1>Klebsiella pneumoniae</td><td rowspan=1 colspan=1>NCTC 13439</td><td rowspan=1 colspan=1>4.0 x 104</td><td rowspan=1 colspan=1>2x</td><td rowspan=1 colspan=1>2/2</td></tr><tr><td rowspan=1 colspan=1>vim-1</td><td rowspan=1 colspan=1>Klebsiella pneumoniae</td><td rowspan=1 colspan=1>NCTC 13440</td><td rowspan=1 colspan=1>4.0 x 104</td><td rowspan=1 colspan=1>2x</td><td rowspan=1 colspan=1>2/2</td></tr><tr><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>Pseudomonas aeruginosa</td><td rowspan=1 colspan=1>UCLA P20</td><td rowspan=1 colspan=1>1.3 x 103</td><td rowspan=1 colspan=1>0.1x</td><td rowspan=1 colspan=1>2/2</td></tr><tr><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>Pseudomonas aeruginosa(carbapenem)</td><td rowspan=1 colspan=1>Micromyx 2562</td><td rowspan=1 colspan=1>4.0 x 104</td><td rowspan=1 colspan=1>2x</td><td rowspan=1 colspan=1>2/2</td></tr></table>

To supplement inclusivity testing for further specific antibiotic resistance marker variants, reference sequences for all available variants belonging to individual antibiotic resistance markers or marker subgroups were compiled and analyzed in silico.

Subgroups or variants for applicable LRT BAL panel antibiotic resistance markers for which detection is predicted at LoD (match of relevant primer and probe sequences), with reduced sensitivity (typically, single relevant mismatches of primer or probe sequences; detection likely at higher than LoD concentrations only), or is not predicted (multiple relevant mismatches in primer and probe sequences) are listed in Table 7. Such predictions are provided as supplementary data only. Results are not intended to be a surrogate for wet testing and do not assure that specific variant will be detected.

NOTE: The performance of the Unyvero LRT BAL Application has not been established for antibiotic resistance marker variants other than those listed in the inclusivity test results table above.

Table 7: In silico prediction for LRT BAL panel antibiotic resistance markers for applicable subgroups or variants.   

<table><tr><td rowspan=1 colspan=1>Antibiotic ResistanceMarker:Subgroup</td><td rowspan=1 colspan=1>Detection Predicted at LoD:Variant No.</td><td rowspan=1 colspan=1>Detection Predicted withReduced Sensitivity:Variant No.</td><td rowspan=1 colspan=1>Detection Not Predicted:Variant No.</td></tr><tr><td rowspan=1 colspan=1>ctx-M:ctx-M1 subgroup</td><td rowspan=1 colspan=1>1, 3, 10, 11, 12, 15, 22, 23, 28-30, 32 - 34, 36, 37, 42, 52 - 55,57, 58, 60 - 62, 66, 68, 69, 71,72, 79, 80, 82, 83, 88, 96, 101,103, 107, 108, 109, 114, 116,117, 132, 133, 136, 138, 139,142, 144, 150, 155 - 158, 162 -164, 166, 167, 169, 170, 172,173, 175 - 177, 179 - 184, 186,188 - 190, 193, 194, 197, 202 -204, 206 - 212, 216, 218, 222,224</td><td rowspan=1 colspan=1>64</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>ctx-M:ctx-M2ctx-M8ctx-M9ctx-M25ctx-M45subgroups</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>2, 4 - 9, 13, 14, 16 - 21, 24 - 27,31, 35, 38 - 41, 43 -51, 56, 59,63, 65, 67, 73 -78, 81, 84 - 87,89 - 95, 97 - 100, 102, 104 -106, 110 -113, 115, 121 - 126,129 -131, 134, 137, 141, 147,148, 152, 159 -161, 165, 168,171, 174, 185, 191, 192, 195,196, 198, 199, 201, 214, 215,217, 219, 221, 223</td></tr><tr><td rowspan=1 colspan=1>kpc</td><td rowspan=1 colspan=1>1 - 39</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>ndm</td><td rowspan=1 colspan=1>1 - 24, 27</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-Y</td></tr><tr><td rowspan=1 colspan=1>oxa:oxa-23</td><td rowspan=1 colspan=1>23, 27, 49, 73, 134, 146, 165 -171, 225, 239, 366, 398, 422,423, 435, 440, 469, 481 - 483,565, 657, 806 - 808, 810, 813 -815, 816, 818</td><td rowspan=1 colspan=1>103, 133, 809, 811, 812, 816</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>oxa:oxa-24</td><td rowspan=1 colspan=1>24 - 26, 33, 40, 72, 139, 160,207, 437, 653</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>oxa:oxa-48</td><td rowspan=1 colspan=1>48, 48b, 162, 163, 181, 199,232, 244, 245, 247, 252, 370,405, 416, 438, 439, 484, 505,514, 515, 517, 519, 538, 546,547, 566, 567, 731, 788, 793</td><td rowspan=1 colspan=1>204</td><td rowspan=1 colspan=1>54, 436, 535</td></tr><tr><td rowspan=1 colspan=1>oxa:oxa-58</td><td rowspan=1 colspan=1>58, 96, 97, 164, 397, 467, 512</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>420</td></tr><tr><td rowspan=1 colspan=1>tem</td><td rowspan=1 colspan=1>1 - 4, 6, 8 - 12, 15 - 17, 19 - 22,24, 26, 28 - 30, 32 - 36, 40, 43,45, 47 - 49, 52 - 55, 57, 60, 63,67, 68, 70 - 72, 76 - 88, 90 - 99,101, 102, 104 - 116, 120 - 139,141 - 150, 152 - 160, 162 - 164,166 - 169, 171, 176, 177, 181 -199, 201, 204 - 217, 219, 220,224 - 228, 229 - 235, 237</td><td rowspan=1 colspan=1>151</td><td rowspan=1 colspan=1>178</td></tr><tr><td rowspan=1 colspan=1>vim</td><td rowspan=1 colspan=1>1 - 6, 8 - 12, 14 - 20, 23 - 46,48 - 54, 56, 58, 60, 62</td><td rowspan=1 colspan=1>57, 59</td><td rowspan=1 colspan=1>7, 13, 47, 61</td></tr></table>

NOTE: $H .$ influenzae commonly hosts tem-1. tem will only be reported by the LRT BAL Application if $H .$ influenzae is concurrently detected.

# Exclusivity and Cross-Reactivity

For each of the LRT BAL panel analytes, cross-reactive species (clinically relevant close neighbor strains) were predicted by GenBank BLAST search. Predictions were complemented with exclusivity wet testing for close-neighbor strains or common respiratory flora microorganisms performed in duplicate at a worst-case concentration (typically: $1 . 5 \mathrm { x } 1 0 ^ { 7 } \mathrm { C F U / m L }$ for bacterial microorganisms, or as indicated) by spiking into $0 . 2 5 \mathrm { x }$ ARM (artificial respiratory matrix) surrogate as sample matrix.

Exclusivity study results are listed in

Table 8. For Haemophilus haemolyticus (ATCC 33390) and Aggregatibacter aphrophilus (ATCC 19415), cross-reactivity to Haemophilus influenzae close to LoD concentrations (for ATCC 33390) or with reduced sensitivity (for ATCC 19415), respectively, was observed. Another strain, Haemophilus parainfluenzae (ATCC 33392) cross-reacted to Haemophilus influenzae only at very high concentrations in some of the tests $( > 1 0 0 \mathrm { x } \mathrm { L o D } , > 1 0 ^ { 7 } \mathrm { C F U / m L } )$ .

Table 8: Exclusivity study results.   
510(k) Summary – LRT BAL Application K191967 Rev 3.0   

<table><tr><td colspan="1" rowspan="1">Strain</td><td colspan="1" rowspan="1">Test Concentration</td><td colspan="1" rowspan="1">Cross-Reactivity</td></tr><tr><td colspan="1" rowspan="1">respiratory flora microorganisms</td><td colspan="1" rowspan="1">[in CFU/mL]</td><td colspan="1" rowspan="1">Y</td></tr><tr><td colspan="1" rowspan="1">Actinomyces odontolyticus</td><td colspan="1" rowspan="1">1.5 x 107</td><td colspan="1" rowspan="1">no</td></tr><tr><td colspan="1" rowspan="1">Aspergillus fumigatus</td><td colspan="1" rowspan="1">1.5 x 107</td><td colspan="1" rowspan="1">no</td></tr><tr><td colspan="1" rowspan="1">Candida albicans</td><td colspan="1" rowspan="1">1.5 x 107</td><td colspan="1" rowspan="1">no</td></tr><tr><td colspan="1" rowspan="1">Candida dubliniensis</td><td colspan="1" rowspan="1">1.5 x 107</td><td colspan="1" rowspan="1">no</td></tr><tr><td colspan="1" rowspan="1">Candida glabrata</td><td colspan="1" rowspan="1">1.5 x 107</td><td colspan="1" rowspan="1">no</td></tr><tr><td colspan="1" rowspan="1">Candida krusei</td><td colspan="1" rowspan="1">1.5 x 107</td><td colspan="1" rowspan="1">no</td></tr><tr><td colspan="1" rowspan="1">Candida parapsilosis</td><td colspan="1" rowspan="1">1.5 x 107</td><td colspan="1" rowspan="1">no</td></tr><tr><td colspan="1" rowspan="1">Candida tropicalis</td><td colspan="1" rowspan="1">1.5 x 107</td><td colspan="1" rowspan="1">no</td></tr><tr><td colspan="1" rowspan="1">Cardiobacterium hominis</td><td colspan="1" rowspan="1">1.5 x 107</td><td colspan="1" rowspan="1">no</td></tr><tr><td colspan="1" rowspan="1">Eikenella corrodens</td><td colspan="1" rowspan="1">1.5 x 107</td><td colspan="1" rowspan="1">no</td></tr><tr><td colspan="1" rowspan="1">Enterococcus faecalis</td><td colspan="1" rowspan="1">1.5 x 107</td><td colspan="1" rowspan="1">no</td></tr><tr><td colspan="1" rowspan="1">Enterococcus faecium</td><td colspan="1" rowspan="1">1.5 x 107</td><td colspan="1" rowspan="1">no</td></tr><tr><td colspan="1" rowspan="1">Fusobacterium nucleatum</td><td colspan="1" rowspan="1">5.0 x 106 d</td><td colspan="1" rowspan="1">no</td></tr><tr><td colspan="1" rowspan="1">Granulicatella adiacens</td><td colspan="1" rowspan="1">1.5 x 107</td><td colspan="1" rowspan="1">no</td></tr><tr><td colspan="1" rowspan="1">Kingella kingae</td><td colspan="1" rowspan="1">1.5 x 107</td><td colspan="1" rowspan="1">no</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus acidophilus</td><td colspan="1" rowspan="1">1.5 x 107</td><td colspan="1" rowspan="1">no</td></tr><tr><td colspan="1" rowspan="1">Micrococcus luteus</td><td colspan="1" rowspan="1">1.5 x 107</td><td colspan="1" rowspan="1">no</td></tr><tr><td colspan="1" rowspan="1">Mycobacterium bovis</td><td colspan="1" rowspan="1">1.5 x 107 d</td><td colspan="1" rowspan="1">no</td></tr><tr><td colspan="1" rowspan="1">Mycoplasma orale</td><td colspan="1" rowspan="1">1.5 x 107</td><td colspan="1" rowspan="1">no</td></tr><tr><td colspan="1" rowspan="1">Neisseria lactamica</td><td colspan="1" rowspan="1">1.5 x 107 d</td><td colspan="1" rowspan="1">no</td></tr><tr><td colspan="1" rowspan="1">Neisseria sicca</td><td colspan="1" rowspan="1">1.5 x 107</td><td colspan="1" rowspan="1">no</td></tr><tr><td colspan="1" rowspan="1">Pantoae agglomerans</td><td colspan="1" rowspan="1">1.5 x 107</td><td colspan="1" rowspan="1">no</td></tr><tr><td colspan="1" rowspan="1">Peptostreptococcus stomatis</td><td colspan="1" rowspan="1">5.0 x 106 d</td><td colspan="1" rowspan="1">no</td></tr><tr><td colspan="1" rowspan="1">Porphyromonas gingivalis</td><td colspan="1" rowspan="1">1.5 x 107 d</td><td colspan="1" rowspan="1">no</td></tr><tr><td colspan="1" rowspan="1">Prevotella buccalis</td><td colspan="1" rowspan="1">1.5 x 107</td><td colspan="1" rowspan="1">no</td></tr><tr><td colspan="1" rowspan="1">Raoultella planticola</td><td colspan="1" rowspan="1">1.5 x 107</td><td colspan="1" rowspan="1">no</td></tr><tr><td colspan="1" rowspan="1">close neighbor microorganisms</td><td colspan="2" rowspan="1">[in CFU/mL]</td></tr><tr><td colspan="1" rowspan="1">Acinetobacter ursingii</td><td colspan="1" rowspan="1">1.5 x 107</td><td colspan="1" rowspan="1">no</td></tr><tr><td colspan="1" rowspan="1">Aggregatibacter actinomycetemcomitans</td><td colspan="1" rowspan="1">1.5 x 107 d</td><td colspan="1" rowspan="1">no</td></tr><tr><td colspan="1" rowspan="1">Aggregatibacter aphrophilus</td><td colspan="1" rowspan="1">1.5 x 107</td><td colspan="1" rowspan="1">yes (H. influenzae) a</td></tr><tr><td colspan="1" rowspan="1">Citrobacter koseri</td><td colspan="1" rowspan="1">1.5 x 107</td><td colspan="1" rowspan="1">no</td></tr><tr><td colspan="1" rowspan="1">Haemophilus haemolyticus</td><td colspan="1" rowspan="1">1.5 x 107</td><td colspan="1" rowspan="1">yes (H. influenzae) b</td></tr><tr><td colspan="1" rowspan="1">Haemophilus parahaemolyticus</td><td colspan="1" rowspan="1">1.5 x 107</td><td colspan="1" rowspan="1">no</td></tr><tr><td colspan="1" rowspan="1">Haemophilus parainfluenzae</td><td colspan="1" rowspan="1">1.5 x 107</td><td colspan="1" rowspan="1">yes (H. influenzae)c</td></tr><tr><td colspan="1" rowspan="1">Legionella longbeachae</td><td colspan="1" rowspan="1">1.5 x 107</td><td colspan="1" rowspan="1">no</td></tr><tr><td colspan="1" rowspan="1">Legionella/Tatlockia micdadei</td><td colspan="1" rowspan="1">1.5 x 107</td><td colspan="1" rowspan="1">no</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus capitis</td><td colspan="1" rowspan="1">1.5 x 107</td><td colspan="1" rowspan="1">no</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus epidermidis</td><td colspan="1" rowspan="1">1.5 x 107</td><td colspan="1" rowspan="1">no</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus haemolyticus</td><td colspan="1" rowspan="1">1.5 x 107</td><td colspan="1" rowspan="1">no</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus lugdunensis</td><td colspan="1" rowspan="1">1.5 x 107</td><td colspan="1" rowspan="1">no</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus saprophyticus</td><td colspan="1" rowspan="1">1.5 x 107</td><td colspan="1" rowspan="1">no</td></tr><tr><td colspan="1" rowspan="1">Streptococcus agalactiae</td><td colspan="1" rowspan="1">1.5 x 107</td><td colspan="1" rowspan="1">no</td></tr><tr><td colspan="1" rowspan="1">Streptococcus anginosus</td><td colspan="1" rowspan="1">1.5 x 107</td><td colspan="1" rowspan="1">no</td></tr><tr><td colspan="1" rowspan="1">Streptococcus dysgalactiae</td><td colspan="1" rowspan="1">1.5 x 107</td><td colspan="1" rowspan="1">no</td></tr><tr><td colspan="1" rowspan="1">Streptococcus gordonii</td><td colspan="1" rowspan="1">1.5 x 107</td><td colspan="1" rowspan="1">no</td></tr><tr><td colspan="1" rowspan="1">Streptococcus intermedius</td><td colspan="1" rowspan="1">1.5 x 107</td><td colspan="1" rowspan="1">no</td></tr><tr><td colspan="1" rowspan="1">Streptococcus mitis</td><td colspan="1" rowspan="1">1.5 x 107</td><td colspan="1" rowspan="1">no</td></tr><tr><td colspan="1" rowspan="1">Streptococcus mutans</td><td colspan="1" rowspan="1">1.5 x 107</td><td colspan="1" rowspan="1">no</td></tr><tr><td colspan="1" rowspan="1">Streptococcus oralis</td><td colspan="1" rowspan="1">1.5 x 107</td><td colspan="1" rowspan="1">no</td></tr><tr><td colspan="1" rowspan="1">Streptococcus parasanguinis</td><td colspan="1" rowspan="1">1.5 x 107</td><td colspan="1" rowspan="1">no</td></tr><tr><td colspan="1" rowspan="1">Streptococcus pseudopneumoniae</td><td colspan="1" rowspan="1">1.5 x 107</td><td colspan="1" rowspan="1">no</td></tr><tr><td colspan="1" rowspan="1">Streptococcus pyogenes</td><td colspan="1" rowspan="1">1.5 x 107</td><td colspan="1" rowspan="1">no</td></tr><tr><td colspan="1" rowspan="1">Streptococcus salivarius</td><td colspan="1" rowspan="1">1.5 x 107</td><td colspan="1" rowspan="1">no</td></tr><tr><td colspan="1" rowspan="1">Streptococcus sanguinis</td><td colspan="1" rowspan="1">1.5 x 107</td><td colspan="1" rowspan="1">no</td></tr><tr><td colspan="1" rowspan="1">Streptococcus vestibularis</td><td colspan="1" rowspan="1">1.5 x 107</td><td colspan="1" rowspan="1">no</td></tr><tr><td colspan="1" rowspan="1">respiratory viruses e</td><td colspan="1" rowspan="1">[in copies/mL]</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Adenovirus 41</td><td colspan="1" rowspan="1">1.0 x 105</td><td colspan="1" rowspan="1">no</td></tr><tr><td colspan="1" rowspan="1">Enterovirus 68</td><td colspan="1" rowspan="1">1.0 x 105</td><td colspan="1" rowspan="1">no</td></tr><tr><td colspan="1" rowspan="1">Enterovirus 71</td><td colspan="1" rowspan="1">1.0 x 105</td><td colspan="1" rowspan="1">no</td></tr><tr><td colspan="1" rowspan="1">Parainfluenzae Virus 1</td><td colspan="1" rowspan="1">1.0 x 105</td><td colspan="1" rowspan="1">no</td></tr><tr><td colspan="1" rowspan="1">Parainfluenzae Virus 2</td><td colspan="1" rowspan="1">1.0 x 105</td><td colspan="1" rowspan="1">no</td></tr><tr><td colspan="1" rowspan="1">Parainfluenzae Virus 3</td><td colspan="1" rowspan="1">1.0 x 105</td><td colspan="1" rowspan="1">no</td></tr><tr><td colspan="1" rowspan="1">Parainfluenzae Virus 4</td><td colspan="1" rowspan="1">1.0 x 105</td><td colspan="1" rowspan="1">no</td></tr><tr><td colspan="1" rowspan="1">Rhinovirus</td><td colspan="1" rowspan="1">1.0 x 105</td><td colspan="1" rowspan="1">no</td></tr><tr><td colspan="1" rowspan="1">RSV A</td><td colspan="1" rowspan="1">1.0 x 105</td><td colspan="1" rowspan="1">no</td></tr><tr><td colspan="1" rowspan="1">RSV B</td><td colspan="1" rowspan="1">1.0 x 105</td><td colspan="1" rowspan="1">no</td></tr></table>

a For A. aphrophilus, cross-reactivity to $H .$ . influenzae was observed down to a concentration of $2 . 0 \mathrm { ~ x ~ } 1 0 ^ { 5 } \mathrm { C F U / m L }$ (1/2 tests positive). For a concentration of $1 . 0 \mathrm { x } 1 0 ^ { 5 } \mathrm { C F U / m L }$ (equivalent to ${ 5 } \mathbf { x }$ LoD of $H .$ . influenzae), cross-reactivity was not observed any more. According to BLAST analysis, cross-reactivity of $A$ . aphrophilus to $H .$ influenzae is only predicted for some strains (including the tested reference strain ATCC 19415), but not for other strains. b For $H .$ . haemolyticus, cross-reactivity to $H .$ influenzae was observed down to a concentration of $4 . 0 \mathrm { ~ x ~ } 1 0 ^ { 5 } \mathrm { C F U / m L }$ $2 / 2$ tests positive, equivalent to $2 \mathbf { x }$ LoD of H. influenzae). Sequence comparison by BLAST for $H .$ . haemolyticus to $H .$ . influenzae shows a $_ { 3 } ,$ mismatch to 510(k) Summary – LRT BAL Application K191967 Rev 3.0 Page 21 of 87

one of the assay primers, while the second primer and all internal array probes show a full match. Therefore, the observed cross-reactivity is supported by BLAST analysis, although only predicted with reduced sensitivity.

c For $H .$ parainfluenzae, cross-reactivity to $H .$ influenzae was observed only at the tested worst-case concentration of $1 . 5 \mathrm { { x } 1 0 ^ { 7 } \mathrm { { C F U / m L } } }$ (2/4 tests positive, equivalent to $7 5 0 \mathrm { x }$ LoD of $H .$ influenzae). Additional tests at $2 . 0 \mathrm { ~ x ~ } 1 0 ^ { 6 }$ CFU/mL (equivalent to $1 0 0 \mathrm { x }$ LoD of H. influenzae), cross-reactivity was not observed any more.   
d Tested as DNA extract (in copies $/ \mathrm { m L }$ ).   
e Tested as DNA or RNA extract (in copies $/ \mathrm { m L }$ ).

Table 9 lists predicted possible cross-reactivity of applicable LRT BAL panel microorganisms based on in silico analysis, exclusivity wet testing and cases observed during the prospective or archived study.

Table 9: In silico cross-reactivity prediction, results of exclusivity wet testing and cross-reactive specimens observed for the prospective and archived study.   

<table><tr><td colspan="1" rowspan="1">Close Neighbor Strain a</td><td colspan="1" rowspan="1">Cross-Reactivity Prediction(in-silico analysis)</td><td colspan="1" rowspan="1">Wet Testing Result</td><td colspan="1" rowspan="1">Cross-ReactionsObserved in ClinicalStudy(N = 1,408 prosp. orarch. specimens)</td></tr><tr><td colspan="4" rowspan="1">Citrobacter freundii</td></tr><tr><td colspan="1" rowspan="1">Citrobacter braakii</td><td colspan="1" rowspan="1">Detection predicted at higher thanLoD concentrations (certainstrains) / Detection not predicted(other strains) c</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Citrobacter pasteurii</td><td colspan="1" rowspan="1">Detection predicted at higher thanLoD concentrations</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Citrobacter werkmannii</td><td colspan="1" rowspan="1">Detection predicted at higher thanLoD concentrations (certainstrains) / Detection not predicted(other strains) c</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Citrobacter youngae</td><td colspan="1" rowspan="1">Detection predicted at LoD (certainstrains) / Detection predicted athigher than LoD concentrations(other strains) c</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Kluyvera georgiana</td><td colspan="1" rowspan="1">Detection predicted at higher thanLoD concentrations</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Citrobacter koseri</td><td colspan="1" rowspan="1">Detection not predicted</td><td colspan="1" rowspan="1">negativeATCC 27156</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td><td></td><td></td><td></td></tr><tr><td colspan="1" rowspan="1">Shigella dysenteriae b</td><td colspan="1" rowspan="1">Detection predicted at LoD</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Shigella boydii b</td><td colspan="1" rowspan="1">Detection predicted at LoD</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Shigella flexneri b</td><td colspan="1" rowspan="1">Detection predicted at LoD</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Shigella sonnei b</td><td colspan="1" rowspan="1">Detection predicted at LoD</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Escherichia albertii</td><td colspan="1" rowspan="1">Detection predicted at LoD</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Escherichia fergusonii</td><td colspan="1" rowspan="1">Detection predicted at LoD</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="4" rowspan="1">Haemophilus influenzae</td></tr><tr><td colspan="1" rowspan="1">Haemophilus haemolyticus</td><td colspan="1" rowspan="1">Detection predicted at higher thanLoD concentrations</td><td colspan="1" rowspan="1">positiveATCC 33390</td><td colspan="1" rowspan="1">5</td></tr><tr><td colspan="1" rowspan="1">Haemophilus parahaemolyticus</td><td colspan="1" rowspan="1">Detection not predicted</td><td colspan="1" rowspan="1">negativeATCC 10014</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Haemophilus parainfluenzae</td><td colspan="1" rowspan="1">Detection not predicted</td><td colspan="1" rowspan="1">positive at worst-caseconcentration onlyATCC 33392</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">Haemophilus aegypticus b</td><td colspan="1" rowspan="1">Detection predicted at LoD</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Aggregatibacter actino-mycetemcomitans</td><td colspan="1" rowspan="1">Detection not predicted</td><td colspan="1" rowspan="1">negativeATCC 33384</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Aggregatibacter aphrophilus</td><td colspan="1" rowspan="1">Detection predicted at higher thanLoD concentrations (certainstrains) / Detection not predicted(other strains) c</td><td colspan="1" rowspan="1">positiveATCC 19415</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">Aggregatibacter segnis</td><td colspan="1" rowspan="1">Detection predicted at higher thanLoD concentrations</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="4" rowspan="1">Klebsiella oxytoca</td></tr><tr><td colspan="1" rowspan="1">Klebsiella michiganensis</td><td colspan="1" rowspan="1">Detection predicted at LoD (certainstrains) / Detection predicted athigher than LoD concentrations(other strains) c</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="4" rowspan="1">Klebsiella pneumoniae</td></tr><tr><td colspan="1" rowspan="1">Klebsiella quasivariicola</td><td colspan="1" rowspan="1">Detection predicted at higher thanLoD concentrations</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="3" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Staphylococcus argenteus b</td><td colspan="1" rowspan="1">Detection predicted at LoD</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">CNS:S. epidermidisS. capitisS. lugdunensisS. haemolyticusS. saprophyticus</td><td colspan="1" rowspan="1">Detection not predicted</td><td colspan="1" rowspan="1">negativeATCC 51625ATCC 27840ATCC 43809ATCC 29970ATCC 15305</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Streptococcus pneumoniae</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">other Streptococcus spp.:S. agalactiaeS. anginosusS. dysgalactiaeS. gordoniiS. intermediusS. mitisS. mutansS. oralisS. parasanguinisS. pseudopneumoniaeS. pyogenesS. salivariusS. sanguinisS. vestibularis</td><td colspan="1" rowspan="1">Detection not predicted</td><td colspan="1" rowspan="1">negativeATCC 13813ATCC 33397ATCC 43078ATCC 10558ATCC 27335ATCC 49456ATCC 25175ATCC 35037ATCC 15912ATCC BAA-960ATCC 12344ATCC 7073ATCC 10556ATCC 49124</td><td colspan="1" rowspan="1"></td></tr></table>

a For several analyte assays, soil, environmental, plant or animal derived close neighbor strains are also predicted at LoD or at higher than LoD concentrations (Citrobacter freundii:, C. portucalensis, Enterobacter cloacae complex: E. soli, $E .$ . mori, $E .$ . nickellidurans; Escherichia coli: E. marmotae; Staphylococcus aureus: $S _ { \ast }$ schweitzeri, S. simiae; Stenotrophomonas maltophilia: S. nitritireducens, $S .$ daejeonensis, S. acidaminiphila, S. koreensis, S. rhizophila, Xanthomonas spp., Pseudoxanthomonas spp.). b Clinical relevance unlikely for respiratory infections. c Predictions for available strains distribute into different categories.

# Interference Testing

Interference testing had already been established with the Unyvero LRT Application (predicate device), which relies on the same DNA extraction procedures and chemistry as the LRT BAL Application. Results are included below for reference. Possible interfering substances, such as respiratory medications, antibiotics, sample storage media, lavage media, sample liquefying agent (DTT), lysis buffer and endogenous substances blood and human DNA were tested. Pools of panel analytes were analyzed with and without interfering substances added at concentrations recommended in the CLSI guideline ‘Interference Testing in Clinical Chemistry’. Representative analytes were pooled and added to PBS as surrogate matrix containing potentially interfering substances at concentrations as indicated in Table 10. No interference was observed at tested concentrations.

Table 10: Tested possible interfering substances (M: test concentration in mol/L).   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Interferent</td><td rowspan=1 colspan=1>Test Concentration</td><td rowspan=1 colspan=1>InterferenceObserved</td></tr><tr><td rowspan=8 colspan=1>respiratory drugs</td><td rowspan=1 colspan=1>Guaifenesin</td><td rowspan=1 colspan=1>1.5 x 10-2 M</td><td rowspan=1 colspan=1>no</td></tr><tr><td rowspan=1 colspan=1>Dextromethorphan</td><td rowspan=1 colspan=1>3.7 x 10-6 M</td><td rowspan=1 colspan=1>no</td></tr><tr><td rowspan=1 colspan=1>Acetyl-Cystein</td><td rowspan=1 colspan=1>1.0 x 10-2 M</td><td rowspan=1 colspan=1>no</td></tr><tr><td rowspan=1 colspan=1>Salbutamol</td><td rowspan=1 colspan=1>4.0 x 10-6 M</td><td rowspan=1 colspan=1>no</td></tr><tr><td rowspan=1 colspan=1>Carbocystein</td><td rowspan=1 colspan=1>2.8 x 10-3 M</td><td rowspan=1 colspan=1>no</td></tr><tr><td rowspan=1 colspan=1>Ambroxol</td><td rowspan=1 colspan=1>8.0 x 10-4 M</td><td rowspan=1 colspan=1>no</td></tr><tr><td rowspan=1 colspan=1>Beclomethason</td><td rowspan=1 colspan=1>7.0 x 10-4 M</td><td rowspan=1 colspan=1>no</td></tr><tr><td rowspan=1 colspan=1>Theophyllin</td><td rowspan=1 colspan=1>2.2 x 10-4 M</td><td rowspan=1 colspan=1>no</td></tr><tr><td rowspan=9 colspan=1>antibiotics</td><td rowspan=1 colspan=1>Ampicillin</td><td rowspan=1 colspan=1>1.5 x 10-4 M</td><td rowspan=1 colspan=1>no</td></tr><tr><td rowspan=1 colspan=1>Cefuroxime</td><td rowspan=1 colspan=1>1.4 x 10-3 M</td><td rowspan=1 colspan=1>no</td></tr><tr><td rowspan=1 colspan=1>Erythromycin</td><td rowspan=1 colspan=1>8.2 x 10-5 M</td><td rowspan=1 colspan=1>no</td></tr><tr><td rowspan=1 colspan=1>Ciprofloxacin</td><td rowspan=1 colspan=1>3.0 x 10-5 M</td><td rowspan=1 colspan=1>no</td></tr><tr><td rowspan=1 colspan=1>Amikacin</td><td rowspan=1 colspan=1>1.4 x 10-4 M</td><td rowspan=1 colspan=1>no</td></tr><tr><td rowspan=1 colspan=1>Imipenem</td><td rowspan=1 colspan=1>5.0 x 10-4 M</td><td rowspan=1 colspan=1>no</td></tr><tr><td rowspan=1 colspan=1>Clindamycin</td><td rowspan=1 colspan=1>8.9 x 10-5 M</td><td rowspan=1 colspan=1>no</td></tr><tr><td rowspan=1 colspan=1>Trimethoprim</td><td rowspan=1 colspan=1>1.4 x 10-4 M</td><td rowspan=1 colspan=1>no</td></tr><tr><td rowspan=1 colspan=1>Sulfamethoxazole</td><td rowspan=1 colspan=1>1.6 x 10-3 M</td><td rowspan=1 colspan=1>no</td></tr><tr><td rowspan=2 colspan=1>lavage samplemedia</td><td rowspan=1 colspan=1>Ringer-Lactate solution</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>no</td></tr><tr><td rowspan=1 colspan=1>Ringer solution</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>no</td></tr><tr><td rowspan=1 colspan=1>inhalation agent(NaCl)</td><td rowspan=1 colspan=1>sodium chloride</td><td rowspan=1 colspan=1>5% w/v</td><td rowspan=1 colspan=1>no</td></tr><tr><td rowspan=1 colspan=1>lysis buffer</td><td rowspan=1 colspan=1>lysis buffer DTT, 90% v/v</td><td rowspan=1 colspan=1>lysis buffer: 90% v/v (final conc. inlysis tube) or 80% v/v (for addedsample), DTT: 40 mM (final conc.in lysis tube) or 35 mM (for addedsample)</td><td rowspan=1 colspan=1>no</td></tr><tr><td rowspan=4 colspan=1>samplecomponents</td><td rowspan=1 colspan=1>EDTA blood</td><td rowspan=1 colspan=1>100% v/v</td><td rowspan=1 colspan=1>no</td></tr><tr><td rowspan=1 colspan=1>human placenta DNA</td><td rowspan=1 colspan=1>1 µg/µL</td><td rowspan=1 colspan=1>no</td></tr><tr><td rowspan=1 colspan=1>fish sperm DNA</td><td rowspan=1 colspan=1>4 μg/μL</td><td rowspan=1 colspan=1>no</td></tr><tr><td rowspan=1 colspan=1>mucin (pig stomach, type II)</td><td rowspan=1 colspan=1>20 mg/mL</td><td rowspan=1 colspan=1>no</td></tr></table>

# Reproducibility

Assay reproducibility was established with artificial samples (viable strains spiked in ARM surrogate matrix) with a representative strain panel (incl. Gram-positive, Gram-negative and ‘atypical’ microorganisms) covering the following LRT BAL panel analytes: Acinetobacter spp., C. pneumoniae, C. freundii, H. influenzae, K. pneumoniae (comprising LRT BAL panel antibiotic resistance markers ctx-M, ndm, and tem), M. morganii, Proteus spp. and, S. aureus (comprising LRT BAL panel antibiotic resistance marker mecA). Reproducibility testing was performed at a moderate concentration $2 . 5 \mathrm { x } - 1 0 \mathrm { x }$ LoD of the corresponding target analyte), at a low concentration $\mathrm { 1 x \mathrm { ~ - ~ } 4 x }$ LoD of the corresponding target analyte), and with all analytes negative (surrogate matrix only) on three different Unyvero systems. Each Unyvero system mimicked a different clinical setting, consisted of one Unyvero Cockpit, three Unyvero Lysators, up to six Unyvero Analyzers, and was operated by one of three dedicated operators with different levels of work experience.

Each operator performed 3 ‘moderate’, 3 ‘low’, and 3 ‘negative’ replicates per test shift on his/her Unyvero system (maximum: 18 replicates per test day). Each operator performed a minimum of 30 replicates per concentration level (90 replicates in total per Unyvero system, 270 replicates in total for all three Unyvero systems with 90 replicates for each concentration level) distributed over a minimum of five test days with sample identities blinded to the operator. Samples on individual Analyzer slots were rotated (e.g., ‘moderate’ followed by ‘low’, followed by ‘negative’ etc.) to achieve a nonconsecutive testing schedule for each sample type between test shifts. Failed runs as well as runs with single invalid analyte results were repeated to achieve a minimum number of at least 30 valid results for each of the target analytes at all test concentration levels.

Tables 11 to 13 summarize the reproducibility results for each concentration level, along with the agreement rates with the expected result and $9 5 \%$ confidence intervals $9 5 \%$ CI, calculated using the Wilson score method).

Table 11: Reproducibility study results (agreement with expected result) for concentration level ‘moderate’, 5x LoD per test analyte or as indicated.   

<table><tr><td rowspan="2">moderate</td><td rowspan="2">x-fold LoD</td><td rowspan="2">Unyvero System/</td><td colspan="3">Agreement with Expected Result</td></tr><tr><td>#Pos. /# Exp.</td><td>%</td><td>95% CI</td></tr><tr><td rowspan="3">Acinetobacter baumannii ATCC 19606</td><td rowspan="3">5</td><td>Operator operator 1</td><td>30/31</td><td>96.8</td><td>83.8 - 99.4</td></tr><tr><td>operator 2</td><td>31/31</td><td>100.0</td><td>89.0 - 100.0</td></tr><tr><td>operator 3</td><td>30/30</td><td>100.0</td><td>88.7 - 100.0</td></tr><tr><td rowspan="3">Chlamydia pneumoniae ATCC VR-2282</td><td rowspan="3">5</td><td>total</td><td>91/92</td><td>98.9</td><td>94.1 - 99.8</td></tr><tr><td>operator 1</td><td>31/31</td><td>100.0</td><td>89.0 - 100.0</td></tr><tr><td>operator 2</td><td>31/31</td><td>100.0</td><td>89.0 - 100.0</td></tr><tr><td rowspan="3">Citrobacter freundii</td><td rowspan="3"></td><td>operator 3</td><td>30/30</td><td>100.0</td><td>88.7 - 100.0</td></tr><tr><td>total</td><td>92/92 29/30 a</td><td>100.0 96.7</td><td>96.0 - 100.0 83.3 - 99.4</td></tr><tr><td>operator 1 operator 2</td><td>31/31</td><td>100.0</td><td>89.0 - 100.0</td></tr><tr><td rowspan="3">ATCC 8090</td><td rowspan="3">5</td><td>operator 3</td><td>30/30</td><td>100.0</td><td>88.7 - 100.0</td></tr><tr><td>total</td><td>90/91</td><td>98.9</td><td>94.0 - 99.8</td></tr><tr><td>operator 1</td><td>29/31</td><td>93.5</td><td>79.3 - 98.2</td></tr><tr><td rowspan="3">Haemophilus influenzae ATCC 33391</td><td rowspan="3">5</td><td>operator 2</td><td>31/31</td><td>100.0</td><td>89.0 - 100.0</td></tr><tr><td>operaor 3</td><td>30/30</td><td>100.0</td><td>88.7 - 100.0</td></tr><tr><td>total</td><td>90/92</td><td>97.8</td><td>92.4 - 99.4</td></tr><tr><td rowspan="3">Klebsiella pneumoniae NCTC 13443</td><td rowspan="3">2.5</td><td>operator 1</td><td>31/31</td><td>100.0</td><td>89.0 - 100.0</td></tr><tr><td>opperator 2</td><td>31/31</td><td>100.0</td><td>89.0 - 100.0</td></tr><tr><td>operator 3</td><td>30/30</td><td>100.0</td><td>88.7 - 100.0</td></tr><tr><td rowspan="3">ctx-M NCTC 13443</td><td rowspan="3">10</td><td>total</td><td>92/92</td><td>100.0</td><td>96.0 - 100.0</td></tr><tr><td>operator 1</td><td>31/31</td><td>100.0</td><td>89.0 - 100.0</td></tr><tr><td>operator 2</td><td>31/31</td><td>100.0</td><td>89.0 - 100.0</td></tr><tr><td rowspan="3">ndm</td><td rowspan="3"></td><td>operator 3</td><td>30/30</td><td>100.0</td><td>88.7 - 100.0</td></tr><tr><td>total</td><td>92/92</td><td>100.0</td><td>96.0 - 100.0</td></tr><tr><td>operator 1</td><td>29/30 a</td><td>96.7</td><td>83.3 - 99.4</td></tr><tr><td rowspan="3">NCTC 13443</td><td rowspan="3">5</td><td>operator 2 operator 3</td><td>31/31 30/30</td><td>100.0 100.0</td><td>89.0 - 100.0 88.7 - 100.0</td></tr><tr><td>total</td><td>90/91</td><td>98.9</td><td>94.0 - 99.8</td></tr><tr><td>operator 1</td><td>28/31</td><td>90.3</td><td>75.1 - 96.7</td></tr><tr><td rowspan="3">tem NCTC 13443</td><td rowspan="3">5</td><td>operator 2</td><td>31/31</td><td>100.0</td><td>89.0 - 100.0</td></tr><tr><td>operator 3</td><td>30/30</td><td>100.0</td><td>88.7 - 100.0</td></tr><tr><td>total</td><td>89/92</td><td>96.7</td><td>90.8 - 98.9</td></tr><tr><td rowspan="3">Morganella morganii ATCC 25830</td><td rowspan="3">5</td><td>operator 1</td><td>31/31</td><td>100.0</td><td>89.0 - 100.0</td></tr><tr><td>operator 2</td><td>31/31</td><td>100.0</td><td>89.0 - 100.0</td></tr><tr><td>operator 3</td><td>30/30</td><td>100.0</td><td>88.7 - 100.0</td></tr><tr><td rowspan="3">Proteus vulgaris ATCC 29905</td><td rowspan="3">5</td><td>total</td><td>92/92</td><td>100.0</td><td>96.0 - 100.0</td></tr><tr><td>operator 1</td><td>31/31</td><td>100.0</td><td>89.0 - 100.0</td></tr><tr><td>operator 2</td><td>30/30 a</td><td>100.0</td><td>88.7 - 100.0</td></tr><tr><td rowspan="3">Staphylococcus aureus</td><td rowspan="3">8.3</td><td>operator 3 total</td><td>30/30 91/91</td><td>100.0 100.0</td><td>88.7 - 100.0 96.0 - 100.0</td></tr><tr><td>operator 1</td><td>31/31</td><td>100.0</td><td>89.0 - 100.0</td></tr><tr><td>operator 2</td><td>31/31</td><td>100.0</td><td>89.0 - 100.0</td></tr><tr><td rowspan="3">NCTC 12493 mecA</td><td rowspan="3"></td><td>operator 3</td><td>30/30</td><td>100.0</td><td>88.7 - 100.0</td></tr><tr><td>total</td><td>92/92</td><td>100.0</td><td>96.0 - 100.0</td></tr><tr><td>operator 1 operator 2</td><td>31/31</td><td>100.0</td><td>89.0 - 100.0</td></tr><tr><td rowspan="3">NCTC 12493</td><td rowspan="3">3</td><td></td><td>31/31</td><td>100.0</td><td>89.0 - 100.0</td></tr><tr><td>operator 3</td><td>30/30</td><td>100.0</td><td>88.7 - 100.0</td></tr><tr><td>total</td><td>92/92</td><td>100.0</td><td>96.0 - 100.0</td></tr></table>

# Table 12: Reproducibility study results (agreement with expected result) for concentration level ‘low’, 2x LoD per analyte or as indicated.

<table><tr><td rowspan="2">low</td><td rowspan="2">x-fold LoD</td><td rowspan="2">Unyvero System/ Operator</td><td colspan="3">Agreement with Expected Result</td></tr><tr><td>#Pos. /# Exp.</td><td>%</td><td>95% CI</td></tr><tr><td>Acinetobacter baumannii ATCC 19606</td><td>2</td><td>operator 1 opeator 2 operator 3 total</td><td>29/30 31/31 29/30</td><td>96.7 100.0 96.7</td><td>83.3 - 99.4 89.0 - 100.0 83.3 - 99.4</td></tr><tr><td>Chlamydia pneumoniae ATCC VR-2282</td><td>2</td><td>operator 1 operator2 operator 3 total</td><td>89/91 30/30 31/31 30/30</td><td>97.8 100.0 100.0 100.0</td><td>92.3 - 99.4 88.7 - 100.0 89.0 - 100.0 88.7 - 100.0</td></tr><tr><td>Citrobacter freundii ATCC 8090</td><td>2</td><td>operator 1 operator 2 opperator 3 total</td><td>91/91 28/30 26/31 29/30 83/91</td><td>100.0 93.3 83.9 96.7</td><td>96.0 - 100.0 78.7 - 98.2 67.4 - 92.9 83.3 - 99.4</td></tr><tr><td>Haemophilus influenzae ATCC 33391</td><td>2</td><td>operator 1 operator 2 operator 3</td><td>27/30 31/31 29/30 87/91</td><td>91.2 90.0 100.0 96.7 95.6</td><td>83.6 - 95.5 74.4 - 96.5 89.0 - 100.0 83.3 - 99.4</td></tr><tr><td>Klebsiella pneumoniae NCTC 13443</td><td>1</td><td>total operator 1 operator 2 operator 3 total</td><td>30/30 31/31 28/30 89/91</td><td>100.0 100.0 93.3 97.8</td><td>89.2 - 98.3 88.7 - 100.0 89.0 - 100.0 78.7 - 98.2</td></tr><tr><td>ctx-M NCTC 13443</td><td>4</td><td>operator 1 operator 2 operator 3 total</td><td>29/30 30/31 30/30 89/91</td><td>96.7 96.8 100.0</td><td>92.3 - 99.4 83.3 - 99.4 83.8 - 99.4 88.7 - 100.0</td></tr><tr><td>ndm NCTC 13443</td><td>2</td><td>operator 1 operator 2 operaor 3 total</td><td>29/30 30/31 30/30 89/91</td><td>97.8 96.7 96.8 100.0 97.8</td><td>92.3 - 99.4 83.3 - 99.4 83.8 - 99.4 88.7 - 100.0</td></tr><tr><td>tem NCTC 13443</td><td>2</td><td>operator 1 operator 2 operator 3 total</td><td>27/30 31/31 29/30</td><td>90.0 100.0 96.7</td><td>92.3 - 99.4 74.4 - 96.5 89.0 - 100.0 83.3 - 99.4 89.2 - 98.3</td></tr><tr><td>Morganella morganii ATCC 25830</td><td>2</td><td>operator 1 operator 2 operator 3 total</td><td>87/91 30/30 30/31 30/30</td><td>95.6 100.0 96.8 100.0</td><td>88.7 - 100.0 83.8 - 99.4 88.7 - 100.0</td></tr><tr><td>Proteus vulgaris a ATCC 29905</td><td>2</td><td>operator 1 opperator 2 operator 3 total</td><td>90/91 25/30 24/31 27/30 76/91</td><td>98.9 83.3 77.4 90.0 83.5</td><td>94.0 - 99.8 66.4 - 92.7 60.2 - 88.6 74.4 - 96.5 74.6 - 89.7</td></tr><tr><td>Staphylococcus aureus NCTC 12493</td><td>3</td><td>operator 1 operator 2 operator 3 total</td><td>29/30 30/30 b 30/30 89/90</td><td>96.7 100.0 100.0 98.9</td><td>83.3 - 99.4 88.7 - 100.0 88.7 - 100.0 94.0 - 99.8</td></tr><tr><td>mecA NCTC 12493</td><td>1.3</td><td>operator 1 operator 2 operator 3 total</td><td>30/30 31/31 29/30 90/91</td><td>100.0 100.0 96.7 98.9</td><td>88.7 - 100.0 89.0 - 100.0 83.3 - 99.4 94.0 - 99.8</td></tr></table>

a Proteus spp. demonstrated a slightly lower than expected positivity rate for the $2 \mathbf { x }$ LoD concentration with a positivity rate (for all operators combined) of $8 3 . 5 \%$ with a $9 5 \%$ CI of $7 4 . 6 \% - 8 9 . 7 \%$ . However, for an alternative test series performed in parallel for the sample stability study using the same strain and the identical cartridge lot with pooled negative BAL matrix the expected positivity rate at a $2 \mathbf { x }$ LoD concentration for Proteus spp. was confirmed (positivity rate: 92/93, $9 8 . 9 \%$ ). b Reduced number of available results due to invalid analyte results.

Table 13: Reproducibility study results (agreement with expected result) for negative samples.   

<table><tr><td rowspan="2">negative</td><td rowspan="2">Unyvero System/ Operator</td><td colspan="3">Agreement with Expected Result</td></tr><tr><td># Neg. /# Exp.</td><td>%</td><td>95% CI</td></tr><tr><td rowspan="3">Acinetobacter baumannii ATCC 19606</td><td>operator 1</td><td>31/31</td><td>100.0</td><td>89.0 - 100.0</td></tr><tr><td>operator 2</td><td>31/31</td><td>100.0</td><td>89.0 - 100.0</td></tr><tr><td>operator 3</td><td>30/30</td><td>100.0</td><td>88.7 - 100.0</td></tr><tr><td rowspan="4">Chlamydia pneumoniae ATCC VR-2282</td><td>total operator 1</td><td>92/92</td><td>100.0 100.0</td><td>96.0 - 100.0 89.0 - 100.0</td></tr><tr><td>operator 2</td><td>31/31 31/31</td><td>100.0</td><td>89.0 - 100.0</td></tr><tr><td>operator 3</td><td></td><td></td><td></td></tr><tr><td>total</td><td>30/30 92/92</td><td>100.0 100.0</td><td>88.7 - 100.0 96.0 - 100.0</td></tr><tr><td rowspan="4">Citrobacter freundii ATCC 8090</td><td>operator 1</td><td>31/31</td><td>100.0</td><td>89.0 - 100.0</td></tr><tr><td>opperator 2</td><td>31/31</td><td>100.0</td><td>89.0 - 100.0</td></tr><tr><td>opperator 3</td><td>30/30</td><td>100.0</td><td>88.7 - 100.0</td></tr><tr><td>total</td><td>92/92</td><td>100.0</td><td>96.0 - 100.0</td></tr><tr><td rowspan="4">Haemophilus influenzae ATCC 33391</td><td>operator 1</td><td>31/31</td><td>100.0</td><td>89.0 - 100.0</td></tr><tr><td>operator 2</td><td>31/31</td><td>100.0</td><td>89.0 - 100.0</td></tr><tr><td>operator 3</td><td>30/30</td><td>100.0</td><td>88.7 - 100.0</td></tr><tr><td>total</td><td>92/92</td><td>100.0</td><td>96.0 - 100.0</td></tr><tr><td rowspan="4">Klebsiella pneumoniae NCTC 13443</td><td>operator 1</td><td>31/31</td><td>100.0</td><td>89.0 - 100.0</td></tr><tr><td>operator 2</td><td>30/30 a</td><td>100.0</td><td>88.7 - 100.0</td></tr><tr><td>operator 3</td><td>30/30</td><td>100.0</td><td>88.7 - 100.0</td></tr><tr><td>total</td><td>91/91</td><td>100.0</td><td>96.0 - 100.0</td></tr><tr><td rowspan="4">ctx-M NCTC 13443</td><td>operator 1</td><td>31/31</td><td>100.0</td><td>89.0 - 100.0</td></tr><tr><td>operator 2</td><td>30/30 a</td><td>100.0</td><td>88.7 - 100.0</td></tr><tr><td>operator 3</td><td>30/30</td><td>100.0</td><td>88.7 - 100.0</td></tr><tr><td>total</td><td>91/91</td><td>100.0</td><td>96.0 - 100.0</td></tr><tr><td rowspan="4">ndm NCTC 13443</td><td>operator 1</td><td>31/31</td><td>100.0</td><td>89.0 - 100.0</td></tr><tr><td>operator 2</td><td>31/31</td><td>100.0</td><td>89.0 - 100.0</td></tr><tr><td>operator 3</td><td>30/30</td><td>100.0</td><td>88.7 - 100.0</td></tr><tr><td>total</td><td>92/92</td><td>100.0</td><td>96.0 - 100.0</td></tr><tr><td rowspan="4">tem NCTC 13443</td><td>operator 1</td><td>31/31</td><td>100.0</td><td>89.0 - 100.0</td></tr><tr><td>operator 2</td><td>31/31</td><td>100.0</td><td>89.0 - 100.0</td></tr><tr><td>operator 3</td><td>30/30</td><td>100.0</td><td>88.7 - 100.0</td></tr><tr><td>total</td><td>92/92</td><td>100.0</td><td>96.0 - 100.0</td></tr><tr><td rowspan="4">Morganella morganii ATCC 25830</td><td>operator 1</td><td>31/31</td><td>100.0</td><td>89.0 - 100.0</td></tr><tr><td>operator 2</td><td>30/30 a</td><td>100.0</td><td>88.7 - 100.0</td></tr><tr><td>operator 3</td><td>30/30</td><td>100.0</td><td>88.7 - 100.0</td></tr><tr><td>total</td><td>91/91</td><td>100.0</td><td>96.0 - 100.0</td></tr><tr><td rowspan="6">Proteus vulgaris ATCC 29905</td><td>operator 1</td><td>30/30 a</td><td>100.0</td><td>88.7 - 100.0</td></tr><tr><td>opperator 2</td><td>31/31</td><td>100.0</td><td>89.0 - 100.0</td></tr><tr><td>operator 3</td><td>30/30</td><td>100.0</td><td>88.7 - 100.0</td></tr><tr><td>total</td><td>91/91</td><td>100.0</td><td>96.0 - 100.0</td></tr><tr><td>operator 1</td><td>31/31</td><td>100.0</td><td>89.0 - 100.0</td></tr><tr><td>operator 2</td><td>31/31</td><td>100.0</td><td>89.0 - 100.0</td></tr><tr><td rowspan="4">NCTC 12493</td><td>operator 3</td><td>30/30</td><td>100.0</td><td>88.7 - 100.0</td></tr><tr><td>total</td><td>92/92</td><td>100.0</td><td>96.0 - 100.0</td></tr><tr><td>operator 1</td><td>31/31</td><td>100.0</td><td>89.0 - 100.0</td></tr><tr><td>operator 2</td><td>30/30 a</td><td>100.0</td><td>88.7 - 100.0</td></tr><tr><td rowspan="4">mecA NCTC 12493</td><td>operator 3</td><td>30/30</td><td>100.0</td><td>88.7 - 100.0</td></tr><tr><td></td><td></td><td></td><td></td></tr><tr><td></td><td></td><td></td><td></td></tr><tr><td>total</td><td>91/91</td><td>100.0</td><td>96.0 - 100.0</td></tr></table>

a Reduced number of available results due to invalid analyte results.

For three cartridge runs, unexpected positive results were reported with weak signals close to the assay thresholds on different test days and different operators/test systems (2x E. coli, for one ‘moderate’ and one ‘low’ test sample, 1x oxa-23, for one ‘moderate’ test sample).

# Fresh-Frozen Study

Assay reproducibility between fresh samples and samples stored frozen was established with artificial samples (viable strains spiked in pooled native negative BAL specimens as test matrix) with the representative strain panel as described for the reproducibility study above. Testing was performed at a moderate concentration ( $2 . 5 \mathrm { x } - 1 0 \mathrm { x }$ LoD of the corresponding target analyte, at least 10 replicates per condition), at a low concentration (1x - 4x LoD of the corresponding target analyte, at least 30 replicates per condition) and with all target analytes negative (at least 10 replicates per condition).

Pair-wise tests with the LRT BAL Application were performed for samples tested within 2 hrs after test sample preparation (Time 0) and samples tested after freezing below $\mathrm { - 7 0 ^ { \circ } C }$ for at least 1 day and thawing immediately before testing.

Tables 14 to 16 summarize the fresh-frozen test results for each of the tested concentration levels (moderate, low, negative) for both storage conditions (tested freshly, tested frozen), along with the agreement rates with the expected result and $9 5 \%$ confidence intervals $9 5 \%$ CI, calculated using the Wilson score method).

Comparable agreement rates with the expected result were obtained for all test analytes for both test conditions.

Table 14: Fresh-frozen study test results for concentration level ‘moderate’, approx. 5x LoD per test analyte.   

<table><tr><td rowspan=2 colspan=1>moderate</td><td rowspan=2 colspan=1>x-foldLoD</td><td rowspan=2 colspan=1>TestCondition</td><td rowspan=1 colspan=3>Agreement with ExpectedResult</td></tr><tr><td rowspan=1 colspan=1># Pos./# Exp.</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=1>Acinetobacter baumanniiATCC 19606</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>freshfrozen</td><td rowspan=1 colspan=1>11/1111/11</td><td rowspan=1 colspan=1>100.0100.0</td><td rowspan=1 colspan=1>74.1 - 100.074.1 - 100.0</td></tr><tr><td rowspan=1 colspan=1>Chlamydia pneumoniaeATCC VR-2282</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>freshfrozen</td><td rowspan=1 colspan=1>11/1111/11</td><td rowspan=1 colspan=1>100.0100.0</td><td rowspan=1 colspan=1>74.1 - 100.074.1 - 100.0</td></tr><tr><td rowspan=1 colspan=1>Citrobacter freundiiATCC 8090</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>freshfrozen</td><td rowspan=1 colspan=1>11/1111/11</td><td rowspan=1 colspan=1>100.0100.0</td><td rowspan=1 colspan=1>74.1 - 100.074.1 - 100.0</td></tr><tr><td rowspan=1 colspan=1>Haemophilus influenzaeATCC 33391</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>freshfrozen</td><td rowspan=1 colspan=1>10/1111/11</td><td rowspan=1 colspan=1>90.9100.0</td><td rowspan=1 colspan=1>62.3 - 98.474.1 - 100.0</td></tr><tr><td rowspan=1 colspan=1>Klebsiella pneumoniaeNCTC 13443</td><td rowspan=1 colspan=1>2.5</td><td rowspan=1 colspan=1>freshfrozen</td><td rowspan=1 colspan=1>11/1111/11</td><td rowspan=1 colspan=1>100.0100.0</td><td rowspan=1 colspan=1>74.1 - 100.074.1 - 100.0</td></tr><tr><td rowspan=1 colspan=1>ctx-MNCTC 13443</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>freshfrozen</td><td rowspan=1 colspan=1>11/1111/11</td><td rowspan=1 colspan=1>100.0100.0</td><td rowspan=1 colspan=1>74.1 - 100.074.1 - 100.0</td></tr><tr><td rowspan=1 colspan=1>ndmNCTC 13443</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>freshfrozen</td><td rowspan=1 colspan=1>11/1111/11</td><td rowspan=1 colspan=1>100.0100.0</td><td rowspan=1 colspan=1>74.1 - 100.074.1 - 100.0</td></tr><tr><td rowspan=1 colspan=1>temNCTC 13443</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>freshfrozen</td><td rowspan=1 colspan=1>10/1111/11</td><td rowspan=1 colspan=1>90.9100.0</td><td rowspan=1 colspan=1>62.3 - 98.474.1 - 100.0</td></tr><tr><td rowspan=1 colspan=1>Morganella morganiiATCC 25830</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>freshfrozen</td><td rowspan=1 colspan=1>11/1111/11</td><td rowspan=1 colspan=1>100.0100.0</td><td rowspan=1 colspan=1>74.1 - 100.074.1 - 100.0</td></tr><tr><td rowspan=1 colspan=1>Proteus vulgarisATCC 29905</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>freshfrozen</td><td rowspan=1 colspan=1>11/1111/11</td><td rowspan=1 colspan=1>100.0100.0</td><td rowspan=1 colspan=1>74.1 - 100.074.1 - 100.0</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus aureusNCTC 12493</td><td rowspan=1 colspan=1>8.3</td><td rowspan=1 colspan=1>freshfrozen</td><td rowspan=1 colspan=1>10/10 a11/11</td><td rowspan=1 colspan=1>100.0100.0</td><td rowspan=1 colspan=1>72.3 - 100.074.1 - 100.0</td></tr><tr><td rowspan=1 colspan=1>mecANCTC 12493</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>freshfrozen</td><td rowspan=1 colspan=1>11/1111/11</td><td rowspan=1 colspan=1>100.0100.0</td><td rowspan=1 colspan=1>74.1 - 100.074.1 - 100.0</td></tr><tr><td rowspan=1 colspan=2>total (all analytes)</td><td rowspan=1 colspan=1>freshfrozen</td><td rowspan=1 colspan=1>129/131132/132</td><td rowspan=1 colspan=1>98.5100.0</td><td rowspan=1 colspan=1>94.6 - 99.697.2 - 100.0</td></tr></table>

Table 15: Fresh-frozen study test results for concentration level ‘low’, approx. ${ \bf 2 x }$ LoD per test analyte.   

<table><tr><td rowspan=3 colspan=1>low</td><td></td><td></td><td></td><td></td><td></td></tr><tr><td rowspan=2 colspan=1>x-foldLoD</td><td rowspan=2 colspan=1>TestCondition</td><td rowspan=1 colspan=3>Agreement with ExpectedResult</td></tr><tr><td rowspan=1 colspan=1># Pos./# Exp.</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=1>Acinetobacter baumanniiATCC 19606</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>freshfrozen</td><td rowspan=1 colspan=1>31/3231/31</td><td rowspan=1 colspan=1>96.9100.0</td><td rowspan=1 colspan=1>84.3 - 99.489.0 - 100.0</td></tr><tr><td rowspan=1 colspan=1>Chlamydia pneumoniaeATCC VR-2282</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>freshfrozen</td><td rowspan=1 colspan=1>32/3231/31</td><td rowspan=1 colspan=1>100.0100.0</td><td rowspan=1 colspan=1>89.3 - 100.089.0 - 100.0</td></tr><tr><td rowspan=1 colspan=1>Citrobacter freundiiATCC 8090</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>freshfrozen</td><td rowspan=1 colspan=1>31/3230/31</td><td rowspan=1 colspan=1>96.996.8</td><td rowspan=1 colspan=1>84.3 - 99.483.8 - 99.4</td></tr><tr><td rowspan=1 colspan=1>Haemophilus influenzaeATCC 33391</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>freshfrozen</td><td rowspan=1 colspan=1>29/3128/30 a</td><td rowspan=1 colspan=1>93.593.3</td><td rowspan=1 colspan=1>79.3 - 98.278.7 - 98.2</td></tr><tr><td rowspan=1 colspan=1>Klebsiella pneumoniaeNCTC 13443</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>freshfrozen</td><td rowspan=1 colspan=1>31/3231/31</td><td rowspan=1 colspan=1>96.9100.0</td><td rowspan=1 colspan=1>84.3 - 99.489.0 - 100.0</td></tr><tr><td rowspan=1 colspan=1>ctx-MNCTC 13443</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>freshfrozen</td><td rowspan=1 colspan=1>32/3231/31</td><td rowspan=1 colspan=1>100.0100.0</td><td rowspan=1 colspan=1>89.3 - 100.089.0 - 100.0</td></tr><tr><td rowspan=1 colspan=1>ndmNCTC 13443</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>freshfrozen</td><td rowspan=1 colspan=1>31/3231/31</td><td rowspan=1 colspan=1>96.9100.0</td><td rowspan=1 colspan=1>84.3 - 99.489.0 - 100.0</td></tr><tr><td rowspan=1 colspan=1>temNCTC 13443</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>freshfrozen</td><td rowspan=1 colspan=1>30/3129/30 a</td><td rowspan=1 colspan=1>96.896.7</td><td rowspan=1 colspan=1>83.8 - 99.483.3 - 99.4</td></tr><tr><td rowspan=1 colspan=1>Morganella morganiiATCC 25830</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>freshfrozen</td><td rowspan=1 colspan=1>32/3231/31</td><td rowspan=1 colspan=1>100.0100.0</td><td rowspan=1 colspan=1>89.3 - 100.089.0 - 100.0</td></tr><tr><td rowspan=1 colspan=1>Proteus vulgarisATCC 29905</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>freshfrozen</td><td rowspan=1 colspan=1>32/3230/31</td><td rowspan=1 colspan=1>100.096.8</td><td rowspan=1 colspan=1>89.3 - 100.083.8 - 99.4</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus aureusNCTC 12493</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>freshfrozen</td><td rowspan=1 colspan=1>32/3231/31</td><td rowspan=1 colspan=1>100.0100.0</td><td rowspan=1 colspan=1>89.3 - 100.089.0 - 100.0</td></tr><tr><td rowspan=1 colspan=1>mecANCTC 12493</td><td rowspan=1 colspan=1>1.3</td><td rowspan=1 colspan=1>freshfrozen</td><td rowspan=1 colspan=1>32/3231/31</td><td rowspan=1 colspan=1>100.0100.0</td><td rowspan=1 colspan=1>89.3 - 100.089.0 - 100.0</td></tr><tr><td rowspan=1 colspan=2>total (all analytes)</td><td rowspan=1 colspan=1>freshfrozen</td><td rowspan=1 colspan=1>375/382365/370</td><td rowspan=1 colspan=1>98.298.6</td><td rowspan=1 colspan=1>96.3 - 99.196.9 - 99.4</td></tr></table>

a Reduced number of available results due to invalid analyte results.

Table 16: Fresh-frozen study test results for samples negative for all analytes.   

<table><tr><td rowspan="2">negative</td><td rowspan="2">Test Condition</td><td colspan="3">Agreement with Expected Result</td></tr><tr><td># Neg. /# Exp.</td><td>%</td><td>95% CI</td></tr><tr><td>Acinetobacter baumannii</td><td>fresh</td><td>10/10</td><td>100.0</td><td>72.3 - 100.0</td></tr><tr><td>ATCC 19606</td><td>frozen</td><td>11/11</td><td>100.0</td><td>74.1 - 100.0</td></tr><tr><td>Chlamydia pneumoniae</td><td>fresh</td><td>10/10</td><td>100.0</td><td>72.3 - 100.0</td></tr><tr><td>ATCC VR-2282 Citrobacter freundii</td><td>frozen</td><td>11/11</td><td>100.0</td><td>74.1 - 100.0</td></tr><tr><td>ATCC 8090</td><td>fresh frozen</td><td>10/10 11/11</td><td>100.0 100.0</td><td>72.3 - 100.0</td></tr><tr><td>Haemophilus influenzae</td><td>fresh</td><td>10/10</td><td>100.0</td><td>74.1 - 100.0</td></tr><tr><td>ATCC 33391</td><td>frozen</td><td>11/11</td><td>100.0</td><td>72.3 - 100.0 74.1 - 100.0</td></tr><tr><td>Klebsiella pneumoniae</td><td>fresh</td><td>10/10</td><td>100.0</td><td></td></tr><tr><td>NCTC 13443</td><td>frozen</td><td></td><td></td><td>72.3 - 100.0</td></tr><tr><td>ctx-M</td><td></td><td>11/11</td><td>100.0</td><td>74.1 - 100.0</td></tr><tr><td>NCTC 13443</td><td>fresh</td><td>10/10</td><td>100.0</td><td>72.3 - 100.0</td></tr><tr><td></td><td>frozen</td><td>11/11</td><td>100.0</td><td>74.1 - 100.0</td></tr><tr><td>ndm NCTC 13443</td><td>fresh</td><td>10/10</td><td>100.0</td><td>72.3 - 100.0</td></tr><tr><td></td><td>frozen</td><td>11/11</td><td>100.0</td><td>74.1 - 100.0</td></tr><tr><td>tem NCTC 13443</td><td>fresh</td><td>10/10</td><td>100.0</td><td>72.3 - 100.0</td></tr><tr><td>Morganella morganii</td><td>frozen</td><td>11/11</td><td>100.0</td><td>74.1 - 100.0</td></tr><tr><td>ATCC 25830</td><td>fresh</td><td>10/10</td><td>100.0</td><td>72.3 - 100.0</td></tr><tr><td>Proteus vulgaris</td><td>frozen</td><td>11/11</td><td>100.0</td><td>74.1 - 100.0</td></tr><tr><td>ATCC 29905</td><td>fresh frozen</td><td>10/10</td><td>100.0</td><td>72.3 - 100.0</td></tr><tr><td>Staphylococcus aureus</td><td>fresh</td><td>11/11</td><td>100.0</td><td>74.1 - 100.0</td></tr><tr><td>NCTC 12493</td><td>frozen</td><td>9/9 a</td><td>100.0</td><td>70.1 - 100.0</td></tr><tr><td>mecA</td><td>fresh</td><td>11/11</td><td>100.0</td><td>74.1 - 100.0</td></tr><tr><td>NCTC 12493</td><td>frozen</td><td>10/10 11/11</td><td>100.0 100.0</td><td>72.3 - 100.0</td></tr><tr><td></td><td>fresh</td><td>119/119</td><td></td><td>74.1 - 100.0</td></tr><tr><td rowspan="2">total (all analytes)</td><td></td><td></td><td>100.0</td><td>96.9 - 100.0</td></tr><tr><td>frozen</td><td>132/132</td><td>100.0</td><td>97.2 - 100.0</td></tr></table>

a Reduced number of available results due to invalid analyte results.

# Sample Stability Study

Assay reproducibility between fresh samples and samples stored refrigerated for at least 24 hrs was established with artificial samples using the same experimental approach as for the fresh-frozen study.

Pair-wise tests with the LRT BAL Application were performed for samples tested within 2 hrs after test sample preparation (Time 0) and samples tested after at total storage of at least 24 hrs including at least 2 hrs at a worst-case ambient temperature of $3 0 ~ ^ { \circ } \mathrm { C }$ and at least 22 hrs stored in a refrigerator.

Tables 17 to 19 summarize the sample stability test results for each of the tested concentration levels (moderate, low, negative) for both test conditions (tested freshly, tested after storage), along with the agreement rates with the expected result and $9 5 \%$ confidence intervals $9 5 \%$ CI, calculated using the

Wilson score method). Comparable agreement rates with the expected result were obtained for all test analytes for both test conditions.

Table 17: Sample stability study test results for concentration level ‘moderate’, approx. 5x LoD per test analyte, and a storage time of at least $2 4 \ \mathbf { h r s }$ .   

<table><tr><td rowspan=2 colspan=1>moderate</td><td rowspan=2 colspan=1>x-foldLoD</td><td rowspan=2 colspan=1>TestCondition</td><td rowspan=1 colspan=3>Agreement with ExpectedResult</td></tr><tr><td rowspan=1 colspan=1># Pos./# Exp.</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=1>Acinetobacter baumanniiATCC 19606</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>fresh24 hrs</td><td rowspan=1 colspan=1>10/1011/11</td><td rowspan=1 colspan=1>100.0100.0</td><td rowspan=1 colspan=1>72.3 - 100.074.1 - 100.0</td></tr><tr><td rowspan=1 colspan=1>Chlamydia pneumoniaeATCC VR-2282</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>fresh24 hrs</td><td rowspan=1 colspan=1>10/1011/11</td><td rowspan=1 colspan=1>100.0100.0</td><td rowspan=1 colspan=1>72.3 - 100.074.1 - 100.0</td></tr><tr><td rowspan=1 colspan=1>Citrobacter freundiiATCC 8090</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>fresh24 hrs</td><td rowspan=1 colspan=1>10/1011/11</td><td rowspan=1 colspan=1>100.0100.0</td><td rowspan=1 colspan=1>72.3 - 100.074.1 - 100.0</td></tr><tr><td rowspan=1 colspan=1>Haemophilus influenzaeATCC 33391</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>fresh24 hrs</td><td rowspan=1 colspan=1>10/1011/11</td><td rowspan=1 colspan=1>100.0100.0</td><td rowspan=1 colspan=1>72.3 - 100.074.1 - 100.0</td></tr><tr><td rowspan=1 colspan=1>Klebsiella pneumoniaeNCTC 13443</td><td rowspan=1 colspan=1>2.5</td><td rowspan=1 colspan=1>fresh24 hrs</td><td rowspan=1 colspan=1>10/1011/11</td><td rowspan=1 colspan=1>100.0100.0</td><td rowspan=1 colspan=1>72.3 - 100.074.1 - 100.0</td></tr><tr><td rowspan=1 colspan=1>ctx-MNCTC 13443</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>fresh24 hrs</td><td rowspan=1 colspan=1>10/1011/11</td><td rowspan=1 colspan=1>100.0100.0</td><td rowspan=1 colspan=1>72.3 - 100.074.1 - 100.0</td></tr><tr><td rowspan=1 colspan=1>ndmNCTC 13443</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>fresh24 hrs</td><td rowspan=1 colspan=1>9/1011/11</td><td rowspan=1 colspan=1>90.0100.0</td><td rowspan=1 colspan=1>59.6 - 98.274.1 - 100.0</td></tr><tr><td rowspan=1 colspan=1>temNCTC 13443</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>fresh24 hrs</td><td rowspan=1 colspan=1>10/1011/11</td><td rowspan=1 colspan=1>100.0100.0</td><td rowspan=1 colspan=1>72.3 - 100.074.1 - 100.0</td></tr><tr><td rowspan=1 colspan=1>Morganella morganiiATCC 25830</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>fresh24 hrs</td><td rowspan=1 colspan=1>10/1011/11</td><td rowspan=1 colspan=1>100.0100.0</td><td rowspan=1 colspan=1>72.3 - 100.074.1 - 100.0</td></tr><tr><td rowspan=1 colspan=1>Proteus vulgarisATCC 29905</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>fresh24 hrs</td><td rowspan=1 colspan=1>10/1011/11</td><td rowspan=1 colspan=1>100.0100.0</td><td rowspan=1 colspan=1>72.3 - 100.074.1 - 100.0</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus aureusNCTC 12493</td><td rowspan=1 colspan=1>8.3</td><td rowspan=1 colspan=1>fresh24 hrs</td><td rowspan=1 colspan=1>9/9 a11/11</td><td rowspan=1 colspan=1>100.0100.0</td><td rowspan=1 colspan=1>70.1 - 100.074.1 - 100.0</td></tr><tr><td rowspan=1 colspan=1>mecANCTC 12493</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>fresh24 hrs</td><td rowspan=1 colspan=1>10/1011/11</td><td rowspan=1 colspan=1>100.0100.0</td><td rowspan=1 colspan=1>72.3 - 100.074.1 - 100.0</td></tr><tr><td rowspan=1 colspan=2>total (all analytes)</td><td rowspan=1 colspan=1>fresh24 hrs</td><td rowspan=1 colspan=1>118/119132/132</td><td rowspan=1 colspan=1>99.2100.0</td><td rowspan=1 colspan=1>95.4 - 99.997.2 - 100.0</td></tr></table>

# Table 18: Sample stability study test results for concentration level ‘low’, approx. 2x LoD per test analyte, and a storage time of at least 24 hrs.

<table><tr><td rowspan=2 colspan=1>low</td><td rowspan=2 colspan=1>x-foldLoD</td><td rowspan=2 colspan=1>TestCondition</td><td rowspan=1 colspan=3>Agreement with ExpectedResult</td></tr><tr><td rowspan=1 colspan=1># Pos./# Exp.</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=1>Acinetobacter baumanniiATCC 19606</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>fresh24 hrs</td><td rowspan=1 colspan=1>31/3231/32</td><td rowspan=1 colspan=1>96.996.9</td><td rowspan=1 colspan=1>84.3 - 99.484.3 - 99.4</td></tr><tr><td rowspan=1 colspan=1>Chlamydia pneumoniaeATCC VR-2282</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>fresh24 hrs</td><td rowspan=1 colspan=1>32/3232/32</td><td rowspan=1 colspan=1>100.0100.0</td><td rowspan=1 colspan=1>89.3 - 100.089.3 - 100.0</td></tr><tr><td rowspan=1 colspan=1>Citrobacter freundiiATCC 8090</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>fresh24 hrs</td><td rowspan=1 colspan=1>31/3229/32</td><td rowspan=1 colspan=1>96.990.6</td><td rowspan=1 colspan=1>84.3 - 99.475.8 - 96.8</td></tr><tr><td rowspan=1 colspan=1>Haemophilus influenzaeATCC 33391</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>fresh24 hrs</td><td rowspan=1 colspan=1>29/3131/31 a</td><td rowspan=1 colspan=1>93.5100.0</td><td rowspan=1 colspan=1>79.3 - 98.289.0 - 100.0</td></tr><tr><td rowspan=1 colspan=1>Klebsiella pneumoniaeNCTC 13443</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>fresh24 hrs</td><td rowspan=1 colspan=1>31/3232/32</td><td rowspan=1 colspan=1>96.9100.0</td><td rowspan=1 colspan=1>84.3 - 99.489.3 - 100.0</td></tr><tr><td rowspan=1 colspan=1>ctx-MNCTC 13443</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>fresh24 hrs</td><td rowspan=1 colspan=1>32/3232/32</td><td rowspan=1 colspan=1>100.0100.0</td><td rowspan=1 colspan=1>89.3 - 100.089.3 - 100.0</td></tr><tr><td rowspan=1 colspan=1>ndmNCTC 13443</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>fresh24 hrs</td><td rowspan=1 colspan=1>31/3231/32</td><td rowspan=1 colspan=1>96.996.9</td><td rowspan=1 colspan=1>84.3 - 99.484.3 - 99.4</td></tr><tr><td rowspan=1 colspan=1>temNCTC 13443</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>fresh24 hrs</td><td rowspan=1 colspan=1>30/3130/31 a</td><td rowspan=1 colspan=1>96.896.8</td><td rowspan=1 colspan=1>83.8 - 99.483.8 - 99.4</td></tr><tr><td rowspan=1 colspan=1>Morganella morganiiATCC 25830</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>fresh24 hrs</td><td rowspan=1 colspan=1>32/3231/32</td><td rowspan=1 colspan=1>100.096.9</td><td rowspan=1 colspan=1>89.3 - 100.084.3 - 99.4</td></tr><tr><td rowspan=1 colspan=1>Proteus vulgarisATCC 29905</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>fresh24 hrs</td><td rowspan=1 colspan=1>32/3230/30</td><td rowspan=1 colspan=1>100.0100.0</td><td rowspan=1 colspan=1>89.3 - 100.088.7 - 100.0</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus aureusNCTC 12493</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>fresh24 hrs</td><td rowspan=1 colspan=1>32/3232/32</td><td rowspan=1 colspan=1>100.0100.0</td><td rowspan=1 colspan=1>89.3 - 100.089.3 - 100.0</td></tr><tr><td rowspan=1 colspan=1>mecANCTC 12493</td><td rowspan=1 colspan=1>1.3</td><td rowspan=1 colspan=1>fresh24 hrs</td><td rowspan=1 colspan=1>32/3232/32</td><td rowspan=1 colspan=1>100.0100.0</td><td rowspan=1 colspan=1>89.3 - 100.089.3 - 100.0</td></tr><tr><td rowspan=2 colspan=2>total (all analytes)</td><td rowspan=2 colspan=1>fresh24 hrs</td><td rowspan=2 colspan=1>375/382373/380</td><td rowspan=1 colspan=1>98.2</td><td rowspan=2 colspan=1>96.3 - 99.196.2 - 99.1</td></tr><tr><td rowspan=1 colspan=1>98.2</td></tr></table>

Table 19: Sample stability study test results for samples negative for all analytes for a total storage time of at least 24 hrs.   

<table><tr><td rowspan="2">negative</td><td rowspan="2">Test Condition</td><td colspan="3">Agreement with Expected Result</td></tr><tr><td># Neg. /# Exp.</td><td>%</td><td>95% CI</td></tr><tr><td>Acinetobacter baumannii</td><td>fresh</td><td>11/11</td><td>100.0</td><td>74.1 - 100.0</td></tr><tr><td>ATCC 19606</td><td>24 hrs</td><td>11/11</td><td>100.0</td><td>74.1 - 100.0</td></tr><tr><td>Chlamydia pneumoniae</td><td>fresh</td><td>11/11</td><td>100.0</td><td>74.1 - 100.0</td></tr><tr><td>ATCC VR-2282</td><td>24 hrs</td><td>11/11</td><td>100.0</td><td>74.1 - 100.0</td></tr><tr><td>Citrobacter freundii</td><td>fresh</td><td>11/11</td><td>100.0</td><td>74.1 - 100.0</td></tr><tr><td>ATCC 8090</td><td>24 hrs</td><td>11/11</td><td>100.0</td><td>74.1 - 100.0</td></tr><tr><td>Haemophilus influenzae</td><td>fresh</td><td>11/11</td><td>100.0</td><td>74.1 - 100.0</td></tr><tr><td>ATCC 33391</td><td>24 hrs</td><td>11/11</td><td>100.0</td><td>74.1 - 100.0</td></tr><tr><td>Klebsiella pneumoniae</td><td>fresh</td><td>11/11</td><td>100.0</td><td>74.1 - 100.0</td></tr><tr><td>NCTC 13443</td><td>24 hrs</td><td>11/11</td><td>100.0</td><td>74.1 - 100.0</td></tr><tr><td>ctx-M</td><td>fresh</td><td>10/10 a</td><td>100.0</td><td>72.3 - 100.0</td></tr><tr><td>NCTC 13443</td><td>24 hrs</td><td>11/11</td><td>100.0</td><td>74.1 - 100.0</td></tr><tr><td>ndm</td><td>fresh</td><td>11/11</td><td>100.0</td><td>74.1 - 100.0</td></tr><tr><td>NCTC 13443</td><td>24 hrs</td><td>11/11</td><td>100.0</td><td>74.1 - 100.0</td></tr><tr><td>tem</td><td>fresh</td><td>11/11</td><td>100.0</td><td>74.1 - 100.0</td></tr><tr><td>NCTC 13443</td><td>24 hrs</td><td>11/11</td><td>100.0</td><td>74.1 - 100.0</td></tr><tr><td>Morganella morganii</td><td>fresh</td><td>10/10 a</td><td>100.0</td><td>72.3 - 100.0</td></tr><tr><td>ATCC 25830</td><td>24 hrs</td><td>11/11</td><td>100.0</td><td>74.1 - 100.0</td></tr><tr><td>Proteus vulgaris</td><td>fresh</td><td>11/11</td><td>100.0</td><td>74.1 - 100.0</td></tr><tr><td>ATCC 29905</td><td>24 hrs</td><td>11/11</td><td>100.0</td><td>74.1 - 100.0</td></tr><tr><td>Staphylococcus aureus NCTC 12493</td><td>fresh</td><td>11/11</td><td>100.0</td><td>74.1 - 100.0</td></tr><tr><td></td><td>24 hrs</td><td>11/11</td><td>100.0</td><td>74.1 - 100.0</td></tr><tr><td>mecA</td><td>fresh</td><td>11/11</td><td>100.0</td><td>74.1 - 100.0</td></tr><tr><td>NCTC 12493</td><td>24 hrs</td><td>11/11</td><td>100.0</td><td>74.1 - 100.0</td></tr><tr><td rowspan="2">total (all analytes)</td><td>fresh</td><td>130/130</td><td>100.0</td><td>97.1 - 100.0</td></tr><tr><td>24 hrs</td><td>132/132</td><td>100.0</td><td>97.2 - 100.0</td></tr></table>

# 807.92 (b)(2): Brief Description of Clinical Data

# Introduction

Clinical performance of the LRT BAL Application was established by a prospective study cohort using 1,016 lavage specimens that had been collected at nine US clinical sites from hospitalized patients suspected for lower respiratory infections, by an archived study comprising 392 confirmed positive lavage specimens predominantly collected at US clinical sites, as well as by a contrived study using artificial positive samples to augment for rare LRT BAL panel analytes.

Performance characteristics of LRT BAL antibiotic resistance markers was evaluated for the prospective study cohort. Furthermore, antibiotic resistance markers detected by LRT BAL were correlated to resistance phenotypes of corresponding strain isolates using antimicrobial susceptibility test (AST) results collected for the prospective and archived study cohorts. Finally, rare antibiotic resistance markers were also included for contrived study testing.

# A. Prospective Study with Frozen Specimens

Specimen Cohort: Inclusion/Exclusion Criteria, Demographics

BAL and mini-BAL specimens from hospitalized patients, age 18 or older, suspected of a lower respiratory infection, were eligible for the prospective study. Specimens had been collected at nine US clinical sites between June 2015 and July 2016 for a previous clinical study. Specimens were stored frozen within 24 hrs after arrival of the specimen in the lab.

Demographic information for prospective specimens included into the LRT BAL prospective study is described in Table 20.

Table 20: Demographic patient data for the LRT BAL prospective study.   

<table><tr><td rowspan=1 colspan=1>Included Specimens</td><td rowspan=1 colspan=1>1,016</td></tr><tr><td rowspan=1 colspan=2>Sex</td></tr><tr><td rowspan=1 colspan=1>male</td><td rowspan=1 colspan=1>598</td></tr><tr><td rowspan=1 colspan=1>female</td><td rowspan=1 colspan=1>418</td></tr><tr><td rowspan=1 colspan=2>Age</td></tr><tr><td rowspan=1 colspan=1>18 - 30 years</td><td rowspan=1 colspan=1>69</td></tr><tr><td rowspan=1 colspan=1>31 - 60 years</td><td rowspan=1 colspan=1>367</td></tr><tr><td rowspan=1 colspan=1>&gt; 60 years</td><td rowspan=1 colspan=1>580</td></tr><tr><td rowspan=1 colspan=2>Clinical Setting</td></tr><tr><td rowspan=1 colspan=1>ICU</td><td rowspan=1 colspan=1>505</td></tr><tr><td rowspan=1 colspan=1>ward</td><td rowspan=1 colspan=1>511</td></tr></table>

# Reference Methods – Microorganisms

For ‘typical’ microorganisms, LRT BAL Application results were compared to standard-of-care (SoC) culture results as the reference. SoC culture results had been reported either qualitatively or quantitatively. For quantitative cultures, a reporting threshold of $1 0 ^ { 3 } \mathrm { C F U / m L }$ or higher for mini-BAL specimens and of $1 0 ^ { 4 } \mathrm { C F U / m L }$ or higher for BAL specimens was applied, following recommendations by the Infectious Diseases Society of America (IDSA) and the American Thoracic Society (ATS). For ‘atypical’ microorganisms Chlamydia pneumoniae, Mycoplasma pneumoniae, Legionella pneumophila and Pneumocystis jirovecii, no common SoC reference test was performed. Instead, SoC testing had only been performed on special request by the physician for a limited subset of patients.

In addition, LRT BAL Application results for ‘typical’ microorganisms were compared to a composite comparator as a second reference method. This composite comparator consists of the results of SoC culture combined with a molecular multiplex PCR comparator assay for which all positive PCR results were followed by bi-directional sequencing. For the composite comparator, any specimen that was positive by either SoC culture and/or positive by PCR and bi-directional sequencing was considered positive for the respective microorganism. Any specimen that was negative for both SoC culture and PCR was considered negative (Table 21A). Validated comparator PCR assays with analytical sensitivities in a similar range to the corresponding LRT BAL assays were included into the multiplex PCR comparator assay. These primers target different genetic loci than those used for the LRT BAL Application.

For ‘atypical’ microorganisms, a combination of two different validated PCR comparator assays was used as composite comparator, each targeting different genetic loci compared to the corresponding LRT BAL assays. As shown in Table 21B, specimens that were positive for either of the PCR/sequencing comparator assays were considered positive and specimens that were negative for both PCR/sequencing assays were considered negative.

Table 21: Result determination algorithms for the composite comparator: (A) for ‘typical’ microorganisms, (B) for ‘atypical’ microorganisms.   

<table><tr><td rowspan=2 colspan=2>Acomposite comparator result(&#x27;typical&#x27; microorganisms)</td><td rowspan=1 colspan=2>Comparator 2:PCR assay(included in multiplex PCR)</td></tr><tr><td rowspan=1 colspan=1>positiveresult</td><td rowspan=1 colspan=1>negativeresult</td></tr><tr><td rowspan=2 colspan=1>Comparator 1:SoC (culture)</td><td rowspan=1 colspan=1>positiveresult</td><td rowspan=1 colspan=1>composite result:positive</td><td rowspan=1 colspan=1>composite result:positive</td></tr><tr><td rowspan=1 colspan=1>negativeresult</td><td rowspan=1 colspan=1>composite results:positive</td><td rowspan=1 colspan=1>composite result:negative</td></tr></table>

<table><tr><td rowspan=2 colspan=2>Bcomposite comparator result(&#x27;atypical&#x27; microorganisms)</td><td rowspan=1 colspan=2>Comparator 2:PCR assay 2(included in multiplex PCR)</td></tr><tr><td rowspan=1 colspan=1>positiveresult</td><td rowspan=1 colspan=1>negativeresult</td></tr><tr><td rowspan=2 colspan=1>Comparator 1:PCR assay 1(included inmultiplex PCR)</td><td rowspan=1 colspan=1>positiveresult</td><td rowspan=1 colspan=1>composite result:positive</td><td rowspan=1 colspan=1>composite result:positive</td></tr><tr><td rowspan=1 colspan=1>negativeresult</td><td rowspan=1 colspan=1>composite results:positive</td><td rowspan=1 colspan=1>composite result:negative</td></tr></table>

# Reference Methods – Antibiotic Resistance Markers

PCR assays corresponding to each LRT BAL antibiotic resistance marker assay were included into the multiplex PCR comparator assay as a (single) molecular reference. Positive PCRs were followed by bi-directional sequencing.

Cultured isolates had been collected for positive prospective specimens whenever possible. Isolates were regrown and evaluated by MALDI-TOF to confirm strain identities and by whole genome sequencing using a next generation sequencing (NGS) approach to screen for presence or absence of LRT BAL panel antibiotic resistance markers. A few isolates that failed to grow were evaluated by PCR/bi-directional sequencing from frozen isolate stocks to confirm identity and to screen for presence or absence of such markers.

Phenotypic AST results for positive specimens as reported by SoC culture were collected to correlate antibiotic resistance markers detected by the LRT BAL Application to resistance phenotypes.

# Discrepant Result Analysis

False positive discrepant results were analyzed by performing singleplex PCRs/bi-directional sequencing using primer pairs targeting different genetic loci compared to the corresponding LRT BAL assays on specimen DNA extracts for each discrepant LRT BAL analyte.

Microorganism Results – Comparison to SoC Culture Reference, PPA/NPA

For the prospective clinical study cohort, 1,016 previously collected specimens were tested with the LRT BAL Application. For ‘typical’ microorganisms, LRT BAL results were compared to the SoC culture result as reference (Table 22) to determine true positives (TP), false negatives (FN), false positives (FP) and true negatives (TN), as well as positive percent agreements (PPA, TP / $( \mathrm { T P } + \mathrm { F N } ) )$ , negative percent agreements (NPA, TN / $( \mathrm { T N } + \mathrm { F P } ) )$ , positive predictive values (PPV, TP $\prime \left( \mathrm { T P + F P } \right) )$ and negative predictive values (NP $\mathrm { V , T N / ( T N + F N ) ) }$ , calculated together with their two-sided $9 5 \%$ confidence intervals $9 5 \%$ CI), determined according to the Wilson score method.

Table 22: Prospective study $\mathbf { N } = \mathbf { 1 } , \mathbf { 0 1 6 }$ specimens a ), comparison to SoC microbiology culture results.   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>TP</td><td rowspan=1 colspan=1>FN</td><td rowspan=1 colspan=1>FPc, d</td><td rowspan=1 colspan=1>TN</td><td rowspan=1 colspan=1>PPA [%](95% CI)</td><td rowspan=1 colspan=1>NPA [%](95% CI)</td><td rowspan=1 colspan=1>PPV [%](95% CI)</td><td rowspan=1 colspan=1>NPV [%](5% CI)</td></tr><tr><td rowspan=1 colspan=1>Acinetobacter spp.</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>993</td><td rowspan=1 colspan=1>90.9(62.3 - 98.4)</td><td rowspan=1 colspan=1>98.9(98.0 - 99.4)</td><td rowspan=1 colspan=1>47.6(28.3 - 67.6)</td><td rowspan=1 colspan=1>99.9(99.4 - 100.0)</td></tr><tr><td rowspan=1 colspan=1>Citrobacter freundii</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>1,011</td><td rowspan=1 colspan=1>100.0(20.7 - 100.0)</td><td rowspan=1 colspan=1>99.7(99.1 - 99.9)</td><td rowspan=1 colspan=1>25.0(4.6 - 69.9)</td><td rowspan=1 colspan=1>100.0(99.6 - 100.0)</td></tr><tr><td rowspan=1 colspan=1>Enterobacter cloacaecomplex</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>991</td><td rowspan=1 colspan=1>76.5(52.7 - 90.4)</td><td rowspan=1 colspan=1>99.3(98.6 - 99.7)</td><td rowspan=1 colspan=1>65.0(43.3 - 81.9)</td><td rowspan=1 colspan=1>99.6(99.0 - 99.8)</td></tr><tr><td rowspan=1 colspan=1>Escherichia coli</td><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>968</td><td rowspan=1 colspan=1>94.4(74.2 - 99.0)</td><td rowspan=1 colspan=1>97.0(95.7 - 97.9)</td><td rowspan=1 colspan=1>36.2(24.0 - 50.5)</td><td rowspan=1 colspan=1>99.9(99.4 - 100.0)</td></tr><tr><td rowspan=1 colspan=1>Haemophilusinfluenzae</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>48</td><td rowspan=1 colspan=1>958</td><td rowspan=1 colspan=1>88.9(56.5 - 98.0)</td><td rowspan=1 colspan=1>95.2(93.7 - 96.4)</td><td rowspan=1 colspan=1>14.3(7.4 - 25.7)</td><td rowspan=1 colspan=1>99.9(99.4 - 100.0)</td></tr><tr><td rowspan=1 colspan=1>Klebsiella oxytoca</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>1,001</td><td rowspan=1 colspan=1>85.7(48.7 - 97.4)</td><td rowspan=1 colspan=1>99.2(98.4 - 99.6)</td><td rowspan=1 colspan=1>42.9(21.4 - 67.4)</td><td rowspan=1 colspan=1>99.9(99.4 - 100.0)</td></tr><tr><td rowspan=1 colspan=1>Klebsiellapneumoniae b</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>4f</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>982</td><td rowspan=1 colspan=1>83.3(64.1 - 93.3)</td><td rowspan=1 colspan=1>99.0(98.2 - 99.5)</td><td rowspan=1 colspan=1>66.7(48.8 - 80.8)</td><td rowspan=1 colspan=1>99.6(99.0 - 99.8)</td></tr><tr><td rowspan=1 colspan=1>Klebsiellavariicola b</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>1,012</td><td rowspan=1 colspan=1>0.0(0.0 - 65.8)</td><td rowspan=1 colspan=1>99.8(99.3 - 99.9)</td><td rowspan=1 colspan=1>0.0(0.0 - 65.8)</td><td rowspan=1 colspan=1>99.8(99.3 - 99.9)</td></tr><tr><td rowspan=1 colspan=1>Moraxellacatarrhalis</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>997</td><td rowspan=1 colspan=1>100.0(34.2 - 100.0)</td><td rowspan=1 colspan=1>98.7(97.8 - 99.2)</td><td rowspan=1 colspan=1>13.3(3.7 - 37.9)</td><td rowspan=1 colspan=1>100.0(99.6 - 100.0)</td></tr><tr><td rowspan=1 colspan=1>Morganella morganii</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>1,009</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>99.7(99.1 - 99.9)</td><td rowspan=1 colspan=1>0.0(0.0 - 56.1)</td><td rowspan=1 colspan=1>100.0(99.6 - 100.0)</td></tr><tr><td rowspan=1 colspan=1>Proteus spp.</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>1,006</td><td rowspan=1 colspan=1>100.0(51.0 - 100.0)</td><td rowspan=1 colspan=1>99.4(98.7 - 99.7)</td><td rowspan=1 colspan=1>40.0(16.8 - 68.7)</td><td rowspan=1 colspan=1>100.0(99.6 - 100.0)</td></tr><tr><td rowspan=1 colspan=1>Pseudomonasaeruginosa</td><td rowspan=1 colspan=1>69</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>43</td><td rowspan=1 colspan=1>900</td><td rowspan=1 colspan=1>95.8(88.5 - 98.6)</td><td rowspan=1 colspan=1>95.4(93.9 - 96.6)</td><td rowspan=1 colspan=1>61.6(52.4 - 70.1)</td><td rowspan=1 colspan=1>99.7(99.0 - 99.9)</td></tr><tr><td rowspan=1 colspan=1>Serratia marcescens</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>998</td><td rowspan=1 colspan=1>100.0(75.8 - 100.0)</td><td rowspan=1 colspan=1>99.5(98.8 - 99.8)</td><td rowspan=1 colspan=1>70.6(46.9 - 86.7)</td><td rowspan=1 colspan=1>100.0(99.6 - 100.0)</td></tr><tr><td rowspan=1 colspan=1>Staphylococcusaureus</td><td rowspan=1 colspan=1>63</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>41</td><td rowspan=1 colspan=1>904</td><td rowspan=1 colspan=1>88.7(79.3 - 94.2)</td><td rowspan=1 colspan=1>95.7(94.2 - 96.8)</td><td rowspan=1 colspan=1>60.6(51.0 - 69.4)</td><td rowspan=1 colspan=1>99.1(98.3 - 99.6)</td></tr><tr><td rowspan=1 colspan=1>Stenotrophomonasmaltophilia</td><td rowspan=1 colspan=1>19</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>22</td><td rowspan=1 colspan=1>972</td><td rowspan=1 colspan=1>90.5(71.1 - 97.4)</td><td rowspan=1 colspan=1>97.8(96.7 - 98.5)</td><td rowspan=1 colspan=1>46.3(32.1 - 61.3)</td><td rowspan=1 colspan=1>99.8(99.3 - 99.9)</td></tr><tr><td rowspan=1 colspan=1>Streptococcuspneumoniae</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>1,003</td><td rowspan=1 colspan=1>100.0(43.9 - 100.0)</td><td rowspan=1 colspan=1>99.0(98.2 - 99.5)</td><td rowspan=1 colspan=1>23.1(8.2 - 50.3)</td><td rowspan=1 colspan=1>100.0(99.6 - 100.0)</td></tr></table>

a Observed invalid LRT BAL analyte results: Acinetobacter spp. (1), C. freundii (1), $E$ . cloacae complex (1), H. influenzae (1), $M .$ . catarrhalis (4), M. morganii (4), $P .$ . aeruginosa (1), S. marcescens (1) and $S _ { \ast }$ . maltophilia (1).

$^ { \mathfrak { b } } K .$ . variicola is typically reported by culture as $K .$ . pneumoniae. To discriminate $K .$ . variicola from $K .$ . pneumoniae, $K .$ . pneumoniae strain isolates were sequenced. For a few cases, for which no isolates were provided, sequencing was performed from specimen DNA extracts instead. Two of 26 cases were identified as $K .$ . variicola and confirmed by both isolate and DNA extract sequencing. Results for $K .$ . pneumoniae and $K .$ . variicola performance are calculated based on the species identified by sequencing.

c Note that many of the FP detections are confirmed when comparing against the composite comparator reference that includes molecular reference assays (see section ‘Comparison to Composite Comparator Reference’).

d Specimens with false positive LRT BAL results were analyzed with molecular assays (PCR/bi-directional sequencing) using specimen DNA extracts for presence or absence of the corresponding microorganism: presence was confirmed in 11 of 11 cases for Acinetobacter spp., 3 of 3 cases for $C .$ . freundii, 7 of 7 cases for $E$ . cloacae complex, 18 of 30 cases for $E$ . coli, 42 of 48 cases for $H .$ influenzae, 7 of 8 cases for $K .$ . oxytoca, 5 of 10 cases for $K .$ . pneumoniae, 2 of 2 cases for $K .$ . variicola, 13 of 13 cases for M. catarrhalis, 3 of 3 cases for M. morganii, 6 of 6 cases for Proteus spp., 41 of 43 cases for $P$ . aeruginosa, 4 of 5 cases for S. marcescens, 31 of 41 cases for S. aureus, 21 of 22 cases for S. maltophilia, and 10 of 10 cases for S. pneumoniae. For cases that were not confirmed for H. influenzae, sequencing results identified $H .$ haemolyticus $( 3 \mathbf { x } )$ , H. parainfluenzae (1x) and Aggregatibacter aphrophilus (1x). For all other cases, PCRs did not amplify sufficient amounts for sequencing.

e Two of the four specimens that were FN for $E .$ . cloacae complex contained multiple host microorganisms identified by culture, and/o LRT BAL. For one specimen, Acinetobacter spp. was additionally reported by both LRT BAL and culture. For the other specimen, S. maltophilia was additionally reported by LRT BAL only.

f Two of the four specimens that were FN for $K .$ . pneumoniae contained multiple host microorganisms identified by culture and/or the molecular reference testing. For one FN specimen, $P .$ . aeruginosa was additionally reported by both LRT BAL and culture. For the other specimen, $E$ . cloacae complex was additionally reported by both LRT BAL and culture and S. maltophilia was additionally reported by LRT BAL only.

As no common SoC reference method was performed for the ‘atypical’ microorganisms reported by the LRT BAL panel, Table 23 lists positive and negative detections by the LRT BAL Application, compared to the subset of cases for which a SoC test was performed on special request by the physician. For $C .$ . pneumoniae, only one SoC test with a negative result was performed. For the three other ‘atypical’ microorganisms, the following PPA/NPA estimates when compared against $\mathrm { S o C }$ tests were observed: $L$ . pneumophila PPA $0 / 1$ $( 0 \% )$ , NPA 237/237 $( 1 0 0 \% )$ ; $M .$ pneumoniae PPA 0/0, NPA 28/28 $( 1 0 0 \% )$ ; $P .$ jirovecii PPA 5/5 $( 1 0 0 \% )$ , NPA 99/100 $( 9 9 \% )$ .

Table 23: Summary of LRT BAL results for 'atypical' microorganisms for the prospective study.   

<table><tr><td rowspan=2 colspan=1></td><td rowspan=2 colspan=2>SOC Test(tested on request only)</td><td rowspan=1 colspan=2>LRT BAL result</td></tr><tr><td rowspan=1 colspan=1>positive</td><td rowspan=1 colspan=1>negative</td></tr><tr><td rowspan=4 colspan=1>Chlamydia pneumoniae</td><td rowspan=1 colspan=1>not tested</td><td rowspan=1 colspan=1>1,015 (99.9%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1,015</td></tr><tr><td rowspan=3 colspan=1>testedpositive resultnegative result</td><td rowspan=1 colspan=1>1 (0.1%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=4 colspan=1>Legionella pneumophila</td><td rowspan=1 colspan=1>not tested</td><td rowspan=1 colspan=1>778 (76.6%)</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>776 </td></tr><tr><td rowspan=3 colspan=1>testedpositive resultnegative result</td><td rowspan=1 colspan=1>238 (23.4%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>238</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1 b</td></tr><tr><td rowspan=1 colspan=1>237</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>237</td></tr><tr><td rowspan=4 colspan=1>Mycoplasma pneumoniae</td><td rowspan=1 colspan=1>not tested</td><td rowspan=1 colspan=1>988 (97.2%)</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>982</td></tr><tr><td rowspan=3 colspan=1>testedpositive resultnegative result</td><td rowspan=1 colspan=1>28 (2.8%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>28</td></tr><tr><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>28</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>28</td></tr><tr><td rowspan=4 colspan=1>Pneumocystis jirovecii</td><td rowspan=1 colspan=1>not tested</td><td rowspan=1 colspan=1>911 (89.7%)</td><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>895</td></tr><tr><td rowspan=3 colspan=1>testedpositive resultnegative result</td><td rowspan=1 colspan=1>105 (10.3%)</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>99</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>5d</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>1e</td><td rowspan=1 colspan=1>99</td></tr></table>

a Reduced number of available LRT BAL results due to one invalid analyte result. $^ { \mathrm { ~ b ~ } } L$ . pneumophila was only reported positive by culture after 13 days. c Applied Pneumocystis SoC methods for 105 specimens were DFA (63 specimens), IFA (29 specimens), and PCR (13 specimens). d Positive by DFA (three specimens) or IFA (two specimens). e Negative by DFA, confirmed positive by molecular reference tests.

Microorganisms - Stratification by Qualitative and Quantitative Culture Result For all prospective specimens with positive culture results, true positive and false negative LRT BAL results were stratified by the semi-quantitative result reported by qualitative SoC culture (categories: rare, few, moderate, numerous) or by quantitative SoC culture (categories: $1 0 3 \mathrm { ~ - ~ } < 1 0 4$ , $1 0 4 -- 1 0 5$ , $1 0 5 \mathrm { ~ - ~ } < 1 0 6 , > 1 0 6$ , in $\mathrm { C F U / m L }$ ) as shown in Table 24.

Table 24: Prospective study, comparison to SoC culture, stratification by qualitative or quantitative culture result.   

<table><tr><td rowspan=2 colspan=1></td><td rowspan=1 colspan=5>Qualitative Culture Result</td></tr><tr><td rowspan=1 colspan=1>Category</td><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>TP</td><td rowspan=1 colspan=1>FN</td><td rowspan=1 colspan=1>PPA [%]</td></tr><tr><td rowspan=4 colspan=1>Acinetobacter spp.</td><td rowspan=1 colspan=1>rare</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=1>few</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0</td></tr><tr><td rowspan=1 colspan=1>moderate</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.0</td></tr><tr><td rowspan=1 colspan=1>numerous</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0</td></tr><tr><td rowspan=4 colspan=1>Citrobacter freundii</td><td rowspan=1 colspan=1>rare</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=1>few</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0</td></tr><tr><td rowspan=1 colspan=1>moderate</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=1>numerous</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=4 colspan=1>Enterobactercloacae complex</td><td rowspan=1 colspan=1>rare</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.0</td></tr><tr><td rowspan=1 colspan=1>few</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0</td></tr><tr><td rowspan=1 colspan=1>moderate</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>83.3</td></tr><tr><td rowspan=1 colspan=1>numerous</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0</td></tr><tr><td rowspan=4 colspan=1>Escherichia coli</td><td rowspan=1 colspan=1>rare</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>75.0</td></tr><tr><td rowspan=1 colspan=1>few</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0</td></tr><tr><td rowspan=1 colspan=1>moderate</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0</td></tr><tr><td rowspan=1 colspan=1>numerous</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0</td></tr><tr><td rowspan=4 colspan=1>Haemophilusinfluenzae</td><td rowspan=1 colspan=1>rare</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=1>few</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=1>moderate</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0</td></tr><tr><td rowspan=1 colspan=1>numerous</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=4 colspan=1>Klebsiella oxytoca</td><td rowspan=1 colspan=1>rare</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=1>few</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0</td></tr><tr><td rowspan=1 colspan=1>moderate</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0</td></tr><tr><td rowspan=1 colspan=1>numerous</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=4 colspan=1>Klebsiellapneumoniae</td><td rowspan=1 colspan=1>rare</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.0</td></tr><tr><td rowspan=1 colspan=1>few</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>75.0</td></tr><tr><td rowspan=1 colspan=1>moderate</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.0</td></tr><tr><td rowspan=1 colspan=1>numerous</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0</td></tr><tr><td rowspan=4 colspan=1>Klebsiella variicola</td><td rowspan=1 colspan=1>rare</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=1>few</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=1>moderate</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=1>numerous</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.0</td></tr><tr><td rowspan=4 colspan=1>Moraxellacatarrhalis</td><td rowspan=1 colspan=1>rare</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=1>few</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=1>moderate</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=1>numerous</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0</td></tr><tr><td rowspan=4 colspan=1>Morganellamorganii</td><td rowspan=1 colspan=1>rare</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=1>few</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=1>moderate</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=1>numerous</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=1>Proteus spp.</td><td rowspan=1 colspan=1>rare</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>NA</td></tr></table>

<table><tr><td rowspan=1 colspan=5>Quantitative Culture Result</td></tr><tr><td rowspan=1 colspan=1>Category</td><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>TP</td><td rowspan=1 colspan=1>FN</td><td rowspan=1 colspan=1>PPA [%]</td></tr><tr><td rowspan=1 colspan=1>103- &lt;104</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0</td></tr><tr><td rowspan=1 colspan=1>104 - &lt;10^}$</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0</td></tr><tr><td rowspan=1 colspan=1>$10^{}06$</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0</td></tr><tr><td rowspan=1 colspan=1>&gt; 106</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=1>103- &lt;104</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=1>104-&lt;10^s}$</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=1>$10^5 - &lt;1066</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=1>&gt;106</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=1>10^3- &lt;104</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.0</td></tr><tr><td rowspan=1 colspan=1>104-&lt;10}$</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>83.3</td></tr><tr><td rowspan=1 colspan=1>$10^{}06$</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0</td></tr><tr><td rowspan=1 colspan=1>&gt;106</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=1>103 - &lt;104</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=1>104- &lt;10\$</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0</td></tr><tr><td rowspan=1 colspan=1>$1 0^5 - &lt;106</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0</td></tr><tr><td rowspan=1 colspan=1>&gt;106</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=1>10^3- &lt;104</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=1>104-&lt;10^}$</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>83.3</td></tr><tr><td rowspan=1 colspan=1>$1 0^5 - &lt; 106</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0</td></tr><tr><td rowspan=1 colspan=1>&gt;106</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=1>103-&lt;104</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0</td></tr><tr><td rowspan=1 colspan=1>104-&lt;10^s}$</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>66.7</td></tr><tr><td rowspan=1 colspan=1>$1 0^{ &lt;1066$</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=1>&gt;106</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=1>$10^{3- &lt;104</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>75.0</td></tr><tr><td rowspan=1 colspan=1>104 - &lt;10^5}$</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0</td></tr><tr><td rowspan=1 colspan=1>$10^5 - &lt;106</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0</td></tr><tr><td rowspan=1 colspan=1>&gt; 106</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=1>103-&lt;104</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=1>104-&lt;10$</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.0</td></tr><tr><td rowspan=1 colspan=1>$1 0^{} &lt;16$</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=1>&gt;106</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=1>103 - &lt;104</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=1>104-&lt;10s}$</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=1>$10^{ &lt;16$</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=1>&gt; 106</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=1>103 - &lt;104</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=1>104- &lt;10^s}$</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=1>$10^5 - &lt;106</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=1>&gt; 106</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=1>103-&lt;104</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0</td></tr></table>

<table><tr><td rowspan=2 colspan=1></td><td rowspan=1 colspan=5>Qualitative Culture Result</td></tr><tr><td rowspan=1 colspan=1>Category</td><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>TP</td><td rowspan=1 colspan=1>FN</td><td rowspan=1 colspan=1>PPA [%]</td></tr><tr><td rowspan=3 colspan=1></td><td rowspan=1 colspan=1>few</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=1>moderate</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0</td></tr><tr><td rowspan=1 colspan=1>numerous</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=4 colspan=1>Pseudomonasaeruginosa a</td><td rowspan=1 colspan=1>rare</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>71.4</td></tr><tr><td rowspan=1 colspan=1>few</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>92.3</td></tr><tr><td rowspan=1 colspan=1>moderate</td><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0</td></tr><tr><td rowspan=1 colspan=1>numerous</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0</td></tr><tr><td rowspan=4 colspan=1>Serratia marcescens</td><td rowspan=1 colspan=1>rare</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=1>few</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0</td></tr><tr><td rowspan=1 colspan=1>moderate</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0</td></tr><tr><td rowspan=1 colspan=1>numerous</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=4 colspan=1>Staphylococcusaureus a</td><td rowspan=1 colspan=1>rare</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>50.0</td></tr><tr><td rowspan=1 colspan=1>few</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>90.9</td></tr><tr><td rowspan=1 colspan=1>moderate</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>90.9</td></tr><tr><td rowspan=1 colspan=1>numerous</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0</td></tr><tr><td rowspan=4 colspan=1>Stenotrophomonasmaltophilia</td><td rowspan=1 colspan=1>rare</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0</td></tr><tr><td rowspan=1 colspan=1>few</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0</td></tr><tr><td rowspan=1 colspan=1>moderate</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0</td></tr><tr><td rowspan=1 colspan=1>numerous</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>83.3</td></tr><tr><td rowspan=4 colspan=1>Streptococcuspneumoniae</td><td rowspan=1 colspan=1>rare</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=1>few</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=1>moderate</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0</td></tr><tr><td rowspan=1 colspan=1>numerous</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>NA</td></tr></table>

<table><tr><td rowspan=1 colspan=5>Quantitative Culture Result</td></tr><tr><td rowspan=1 colspan=1>Category</td><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>TP</td><td rowspan=1 colspan=1>FN</td><td rowspan=1 colspan=1>PPA [%]</td></tr><tr><td rowspan=1 colspan=1>104 - &lt; 105</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=1>105 - &lt;106</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=1>&gt; 106</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=1>103-&lt;104</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0</td></tr><tr><td rowspan=1 colspan=1>104-&lt;105</td><td rowspan=1 colspan=1>19</td><td rowspan=1 colspan=1>19</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0</td></tr><tr><td rowspan=1 colspan=1>$10^5 - &lt;106</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0</td></tr><tr><td rowspan=1 colspan=1>&gt; 106</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=1>10³ -&lt; 104</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0</td></tr><tr><td rowspan=1 colspan=1>104- &lt;10}$</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0</td></tr><tr><td rowspan=1 colspan=1>105 - &lt; 106</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0</td></tr><tr><td rowspan=1 colspan=1>&gt; 106</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=1>103- &lt;104</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>80.0</td></tr><tr><td rowspan=1 colspan=1>104-&lt;10$</td><td rowspan=1 colspan=1>18</td><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>94.4</td></tr><tr><td rowspan=1 colspan=1>105 - &lt; 106</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0</td></tr><tr><td rowspan=1 colspan=1>&gt; 106</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=1>10³ - &lt; 104</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.0</td></tr><tr><td rowspan=1 colspan=1>104 - &lt;105</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0</td></tr><tr><td rowspan=1 colspan=1>105 - &lt; 106</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=1>&gt; 106</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=1>10³-&lt;104</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=1>104- &lt;10}$</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=1>$10^{ - &lt; 106</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0</td></tr><tr><td rowspan=1 colspan=1>&gt; 106</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>NA</td></tr></table>

A One $P .$ aeruginosa and one $S _ { \cdot }$ . aureus case were reported positive by SoC culture without any qualitative or quantitative information, and were therefore not included in this table.

Microorganism Results – Comparison to SoC Culture Reference, Concordance Rates for the Prospective Study

For 212 specimens, at least one LRT BAL panel microorganism was reported by both the LRT BAL Application and SoC; for 21 specimens, a positive SoC result was missed by LRT BAL. For 632 specimens, both the LRT BAL Application and SoC reported a negative result (no growth or normal/mixed flora result). For 151 specimens, the LRT BAL Application reported a positive result while SoC was negative. For 65 of these 151 specimens, SoC reported normal/mixed flora, while for 81 of 151 specimens, no growth was reported (Table 25).

Table 25: Comparison of positive and negative SoC results to LRT BAL.   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Positive SoC Result(N = 233)</td><td rowspan=1 colspan=4>Negative SoC Results(N = 783)</td></tr><tr><td rowspan=1 colspan=1>LRT BALResult</td><td rowspan=1 colspan=1>Microorganism(s)Reported</td><td rowspan=1 colspan=1>No Growth</td><td rowspan=1 colspan=1>Normal/MixedFlora</td><td rowspan=1 colspan=1>NA a</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>any positive(N =363)</td><td rowspan=1 colspan=1>212</td><td rowspan=1 colspan=1>81</td><td rowspan=1 colspan=1>65</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>151</td></tr><tr><td rowspan=1 colspan=1>all negative(N = 653)</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>358</td><td rowspan=1 colspan=1>258</td><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>632</td></tr></table>

a Presence or absence of flora not reported.

For 1,016 prospective specimens, the LRT BAL Application reported 363 specimens with at least one LRT BAL panel microorganism. For 113 specimens, multi-detections with two or more LRT BAL panel microorganisms were reported. SoC culture reported 233 specimens with at least one LRT BAL panel microorganism. For 40 specimens, multi-detections with two or more LRT BAL panel microorganisms were reported (Table 26).

Table 26: Numbers of detected LRT BAL panel microorganisms as reported by LRT BAL or SoC.   

<table><tr><td rowspan="2">Number of Detected Microorganisms</td><td colspan="2">LRT BAL</td><td colspan="2">SoC</td></tr><tr><td># specimens</td><td>[%]</td><td># specimens</td><td>[%]</td></tr><tr><td>0</td><td>653</td><td>64.3</td><td>783</td><td>77.1</td></tr><tr><td>any positive</td><td>363</td><td>35.7</td><td>233</td><td>22.9</td></tr><tr><td>1</td><td>250</td><td>24.6</td><td>194</td><td>19.1</td></tr><tr><td>2</td><td>78</td><td>7.7</td><td>31</td><td>3.1</td></tr><tr><td>3</td><td>19</td><td>1.9</td><td>5</td><td>0.5</td></tr><tr><td>4</td><td>7</td><td>0.7</td><td>3</td><td>0.3</td></tr><tr><td>5</td><td>7</td><td>0.7</td><td>0</td><td>0.0</td></tr><tr><td>6</td><td>2</td><td>0.2</td><td>0</td><td>0.0</td></tr></table>

Table 27 lists all LRT BAL results compared to SoC culture results stratified into the following categories:

category 1: concordant result for both LRT BAL and SoC category 1a: both negative, category 1b: both positive with a fully concordant result, ategory 2: LRT BAL reported additional microorganisms compared to SoC category 2a: for specimens with a negative SoC result, category 2b: for specimens with a positive SoC result, category 3: LRT BAL reported fewer microorganisms compared to SoC  category 3a: for specimens with a negative LRT BAL result, category 3b: for specimens with a positive LRT BAL result, ategory 4: LRT BAL and SoC report discordant results with different microorganisms  category 4a: LRT BAL and SoC reportings share at least one concordant microorganism,  category 4b: LRT BAL and SoC results are fully discordant.

When comparing LRT BAL to SoC culture results, an overall concordance rate of $7 5 . 9 \%$ (632 negative specimens $( 6 2 . 2 \% )$ , 139 positive specimens $( 1 3 . 7 \% )$ ) was observed; discrepancies were mostly due to LRT BAL detection of additional microorganisms. For 214 of all specimens $( 2 1 . 1 \% )$ , LRT BAL reported one or more additional microorganisms compared to SoC. 151 of these specimens $( 1 4 . 9 \% )$ were reported fully negative by SoC, while for 63 specimens $( 6 . 2 \% )$ at least one LRT BAL panel microorganism was reported by SoC. For 25 of all specimens $( 2 . 5 \% )$ , one or more positive SoC results were not reported by LRT BAL. 21 of these specimens $( 2 . 1 \% )$ were reported fully negative by LRT BAL, while four specimens $( 0 . 4 \% )$ were reported with at least one positive LRT BAL result. For $0 . 4 \%$ of all specimens, LRT BAL and $\mathrm { S o C }$ reported different microorganisms. Two of these specimens $( 0 . 2 \% )$ shared at least one concordant microorganism result, while for four specimens $( 0 . 4 \% )$ results were fully discordant.

Table 27: Comparison of LRT BAL and SoC results, full listing.   

<table><tr><td rowspan=1 colspan=1>LRT BAL Result</td><td rowspan=1 colspan=1>SoC Result</td><td rowspan=1 colspan=1># Cases</td><td rowspan=1 colspan=1>%</td></tr><tr><td rowspan=1 colspan=2>Category 1: concordant LRT BAL and SoC results</td><td rowspan=1 colspan=2>concordant results:75.9%</td></tr><tr><td rowspan=1 colspan=2>Category 1a: both negative</td><td rowspan=1 colspan=1>632</td><td rowspan=1 colspan=1>62.2</td></tr><tr><td rowspan=1 colspan=2>Category 1b: concordant positive results</td><td rowspan=1 colspan=1>139</td><td rowspan=1 colspan=1>13.7</td></tr><tr><td rowspan=1 colspan=1>Acinetobacter spp.</td><td rowspan=1 colspan=1>Acinetobacter spp.</td><td rowspan=1 colspan=1>4</td><td rowspan=12 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Enterobacter cloacae complex</td><td rowspan=1 colspan=1>Enterobacter cloacae complex</td><td rowspan=1 colspan=1>4</td></tr><tr><td rowspan=1 colspan=1>Escherichia coli</td><td rowspan=1 colspan=1>Escherichia coli</td><td rowspan=1 colspan=1>10</td></tr><tr><td rowspan=1 colspan=1>Haemophilus influenzae</td><td rowspan=1 colspan=1>Haemophilus influenzae</td><td rowspan=1 colspan=1>5</td></tr><tr><td rowspan=1 colspan=1>Klebsiella pneumoniae</td><td rowspan=1 colspan=1>Klebsiella pneumoniae</td><td rowspan=1 colspan=1>6</td></tr><tr><td rowspan=1 colspan=1>Moraxella catarrhalis</td><td rowspan=1 colspan=1>Moraxella catarrhalis</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>Pneumocystis jirovecii</td><td rowspan=1 colspan=1>Pneumocystis jirovecii</td><td rowspan=1 colspan=1>5</td></tr><tr><td rowspan=1 colspan=1>Pseudomonas aeruginosa</td><td rowspan=1 colspan=1>Pseudomonas aeruginosa</td><td rowspan=1 colspan=1>31</td></tr><tr><td rowspan=1 colspan=1>Serratia marcescens</td><td rowspan=1 colspan=1>Serratia marcescens</td><td rowspan=1 colspan=1>5</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus aureus</td><td rowspan=1 colspan=1>Staphylococcus aureus</td><td rowspan=1 colspan=1>39</td></tr><tr><td rowspan=1 colspan=1>Stenotrophomonas maltophilia</td><td rowspan=1 colspan=1>Stenotrophomonas maltophilia</td><td rowspan=1 colspan=1>8</td></tr><tr><td rowspan=1 colspan=1>Streptococcus pneumoniae</td><td rowspan=1 colspan=1>Streptococcus pneumoniae</td><td rowspan=1 colspan=1>1</td></tr></table>

510(k) Summary – LRT BAL Application K191967 Rev 3.0

510(k) Summary – LRT BAL Application K191967 Rev 3.0   

<table><tr><td colspan="1" rowspan="1">LRT BAL Result</td><td colspan="1" rowspan="1">SoC Result</td><td colspan="1" rowspan="1"># Cases</td><td colspan="1" rowspan="1">%</td></tr><tr><td colspan="1" rowspan="1">Acinetobacter spp.,Citrobacter freundii</td><td colspan="1" rowspan="1">Acinetobacter spp.,Citrobacter freundii</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="12"></td></tr><tr><td colspan="1" rowspan="1">Acinetobacter spp.,Stenotrophomonas maltophilia</td><td colspan="1" rowspan="1">Acinetobacter spp.,Stenotrophomonas maltophilia</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">Enterobacter cloacae complex,Staphylococcus aureus</td><td colspan="1" rowspan="1">Enterobacter cloacae complex,Staphylococcus aureus</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli,K lebsiella oxytoca</td><td colspan="1" rowspan="1">Escherichia coli,Klebsiella oxytoca</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">Klebsiella oxytoca,Staphylococcus aureus</td><td colspan="1" rowspan="1">Klebsiella oxytoca,Staphylococcus aureus</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae,Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">Klebsiella pneumoniae,Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">Proteus spp.,Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">Proteus spp.,Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">Pseudomonas aeruginosa,Staphylococcus aureus</td><td colspan="1" rowspan="1">Pseudomonas aeruginosa,Staphylococcus aureus</td><td colspan="1" rowspan="1">5</td></tr><tr><td colspan="1" rowspan="1">Pseudomonas aeruginosa,Stenotrophomonas maltophilia</td><td colspan="1" rowspan="1">Pseudomonas aeruginosa,Stenotrophomonas maltophilia</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">Serratia marcescens,Staphylococcus aureus</td><td colspan="1" rowspan="1">Serratia marcescens,Staphylococcus aureus</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus,Stenotrophomonas maltophilia</td><td colspan="1" rowspan="1">Staphylococcus aureus,Stenotrophomonas maltophilia</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">Proteus spp.,Pseudomonas aeruginosa,Staphylococcus aureus</td><td colspan="1" rowspan="1">Proteus spp.,Pseudomonas aeruginosa,Staphylococcus aureus</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="2" rowspan="1">Category 2: LRT BAL detects additional microorganisms</td><td colspan="2" rowspan="1">additional detections:21.1 %</td></tr><tr><td colspan="1" rowspan="1">Category 2a: negative SoC result</td><td colspan="1" rowspan="1">cYanmaGellowb</td><td colspan="1" rowspan="1">151</td><td colspan="1" rowspan="1">14.9</td></tr><tr><td colspan="1" rowspan="1">Acinetobacter spp.</td><td colspan="1" rowspan="1">negative</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="23"></td></tr><tr><td colspan="1" rowspan="1">Citrobacter freundii</td><td colspan="1" rowspan="1">negative</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">Enterobacter cloacae complex</td><td colspan="1" rowspan="1">negative</td><td colspan="1" rowspan="1">4</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">negative</td><td colspan="1" rowspan="1">14</td></tr><tr><td colspan="1" rowspan="1">Haemophilus influenzae</td><td colspan="1" rowspan="1">negative</td><td colspan="1" rowspan="1">24</td></tr><tr><td colspan="1" rowspan="1">Klebsiella oxytoca</td><td colspan="1" rowspan="1">negative</td><td colspan="1" rowspan="1">2</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">negative</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">Legionella pneumophila</td><td colspan="1" rowspan="1">negative</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">Moraxella catarrhalis</td><td colspan="1" rowspan="1">negative</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">Mycoplasma pneumoniae</td><td colspan="1" rowspan="1">negative</td><td colspan="1" rowspan="1">4</td></tr><tr><td colspan="1" rowspan="1">Pneumocystis jirovecii</td><td colspan="1" rowspan="1">negative</td><td colspan="1" rowspan="1">9</td></tr><tr><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">negative</td><td colspan="1" rowspan="1">27</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">negative</td><td colspan="1" rowspan="1">21</td></tr><tr><td colspan="1" rowspan="1">Stenotrophomonas maltophilia</td><td colspan="1" rowspan="1">negative</td><td colspan="1" rowspan="1">2</td></tr><tr><td colspan="1" rowspan="1">Streptococcus pneumoniae</td><td colspan="1" rowspan="1">negative</td><td colspan="1" rowspan="1">6</td></tr><tr><td colspan="1" rowspan="1">Enterobacter cloacae complex,Staphylococcus aureus</td><td colspan="1" rowspan="1">negative</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli,Klebsiella pneumoniae</td><td colspan="1" rowspan="1">negative</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli,Serratia marcescens</td><td colspan="1" rowspan="1">negative</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">Haemophilus influenzae,Moraxella catarrhalis</td><td colspan="1" rowspan="1">negative</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">Haemophilus influenzae,Mycoplasma pneumoniae</td><td colspan="1" rowspan="1">negative</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">Haemophilus influenzae,Staphylococcus aureus</td><td colspan="1" rowspan="1">negative</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">Haemophilus influenzae,Streptococcus pneumoniae</td><td colspan="1" rowspan="1">negative</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">Klebsiella oxytoca,</td><td colspan="1" rowspan="1">negative</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">Proteus spp.</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="11"></td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae,Staphylococcus aureus</td><td colspan="1" rowspan="1">negative</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">Moraxella catarrhalis,Pneumocystis jirovecii</td><td colspan="1" rowspan="1">negative</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">Moraxella catarrhalis,Streptococcus pneumoniae</td><td colspan="1" rowspan="1">negative</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">Pneumocystis jirovecii,Staphylococcus aureus</td><td colspan="1" rowspan="1">negative</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">Pseudomonas aeruginosa,Staphylococcus aureus</td><td colspan="1" rowspan="1">negative</td><td colspan="1" rowspan="1">6</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus,Stenotrophomonas maltophilia</td><td colspan="1" rowspan="1">negative</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">Enterobacter cloacae complex,Haemophilus influenzae,Moraxella catarrhalis</td><td colspan="1" rowspan="1">negative</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli,Haemophilus influenzae,Staphylococcus aureus</td><td colspan="1" rowspan="1">negative</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">Klebsiella oxytoca,Pseudomonas aeruginosa,Staphylococcus aureus</td><td colspan="1" rowspan="1">negative</td><td colspan="1" rowspan="1">−|</td></tr><tr><td colspan="1" rowspan="1">Enterobacter cloacae complex,Haemophilus influenzae,Moraxella catarrhalis,Morganella morganii,Staphylococcus aureus,Stenotrophomonas maltophilia</td><td colspan="1" rowspan="1">negative</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">Category 2b: positive result for both LRT BAL and SoC</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">63</td><td colspan="1" rowspan="1">6.2</td></tr><tr><td colspan="1" rowspan="1">Acinetobacter spp.,Klebsiella pneumoniae</td><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="15"></td></tr><tr><td colspan="1" rowspan="1">Acinetobacter spp.,Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">Enterobacter cloacae complex,Mycoplasma pneumoniae</td><td colspan="1" rowspan="1">Enterobacter cloacae complex</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli,Haemophilus influenzae</td><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli,Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli,Serratia marcescens</td><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli,Staphylococcus aureus</td><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli,Staphylococcus aureus</td><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">Haemophilus influenzae,Klebsiella oxytoca</td><td colspan="1" rowspan="1">Klebsiella oxytoca</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">Haemophilus influenzae,Moraxella catarrhalis</td><td colspan="1" rowspan="1">Haemophilus influenzae</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">Haemophilus influenzae,Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">Haemophilus influenzae,Staphylococcus aureus</td><td colspan="1" rowspan="1">Haemophilus influenzae</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">Haemophilus influenzae,Staphylococcus aureus</td><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">2</td></tr><tr><td colspan="1" rowspan="1">Klebsiella oxytoca,Klebsiella pneumoniae</td><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">Klebsiella oxytoca,</td><td colspan="1" rowspan="1">Klebsiella oxytoca</td><td colspan="1" rowspan="1">1</td></tr></table>

<table><tr><td rowspan=1 colspan=1>LRT BAL Result</td><td rowspan=1 colspan=1>SoC Result</td><td rowspan=1 colspan=1># Cases</td></tr><tr><td rowspan=1 colspan=1>Klebsiella oxytoca,Stenotrophomonas maltophilia</td><td rowspan=1 colspan=1>Klebsiella oxytoca</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>Klebsiella pneumoniae,Pseudomonas aeruginosa</td><td rowspan=1 colspan=1>Klebsiella pneumoniae</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>Klebsiella pneumoniae,Staphylococcus aureus</td><td rowspan=1 colspan=1>Klebsiella pneumoniae</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>Klebsiella pneumoniae,Staphylococcus aureus</td><td rowspan=1 colspan=1>Staphylococcus aureus</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>Pneumocystis jirovecii,Pseudomonas aeruginosa</td><td rowspan=1 colspan=1>Pseudomonas aeruginosa</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>Pneumocystis jirovecii,Stenotrophomonas maltophilia</td><td rowspan=1 colspan=1>Stenotrophomonas maltophilia</td><td rowspan=1 colspan=1>2</td></tr><tr><td rowspan=1 colspan=1>Proteus spp.,Staphylococcus aureus</td><td rowspan=1 colspan=1>Proteus spp.</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>Pseudomonas aeruginosa,Serratia marcescens</td><td rowspan=1 colspan=1>Pseudomonas aeruginosa</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>Pseudomonas aeruginosa,Staphylococcus aureus</td><td rowspan=1 colspan=1>Pseudomonas aeruginosa</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>Pseudomonas aeruginosa,Staphylococcus aureus</td><td rowspan=1 colspan=1>Staphylococcus aureus</td><td rowspan=1 colspan=1>2</td></tr><tr><td rowspan=1 colspan=1>Pseudomonas aeruginosa,Stenotrophomonas maltophilia</td><td rowspan=1 colspan=1>Pseudomonas aeruginosa</td><td rowspan=1 colspan=1>3</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus aureus,Stenotrophomonas maltophilia</td><td rowspan=1 colspan=1>Staphylococcus aureus</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>Acinetobacter spp.,Klebsiella pneumoniae,Staphylococcus aureus</td><td rowspan=1 colspan=1>Klebsiella pneumoniae,Staphylococcus aureus</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>Acinetobacter spp.,Klebsiella pneumoniae,Stenotrophomonas maltophilia</td><td rowspan=1 colspan=1>Klebsiella pneumoniae</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>Acinetobacter spp.,Pseudomonas aeruginosa,Stenotrophomonas maltophilia</td><td rowspan=1 colspan=1>Acinetobacter spp.,Pseudomonas aeruginosa</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Enterobacter cloacae complex,Klebsiella pneumoniae,Moraxella catarrhalis</td><td rowspan=1 colspan=1>Enterobacter cloacae complex</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Enterobacter cloacae complex,Klebsiella pneumoniae,Pseudomonas aeruginosa</td><td rowspan=1 colspan=1>Enterobacter cloacae complex</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>Escherichia coli,Morganella morganii,Pseudomonas aeruginosa</td><td rowspan=1 colspan=1>Pseudomonas aeruginosa</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>Escherichia coli,Pneumocystis jirovecii,Pseudomonas aeruginosa</td><td rowspan=1 colspan=1>Escherichia coli</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>Escherichia coli,Pseudomonas aeruginosa,Serratia marcescens</td><td rowspan=1 colspan=1>Pseudomonas aeruginosa,Serratia marcescens</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>Escherichia coli,Pseudomonas aeruginosa,Staphylococcus aureus</td><td rowspan=1 colspan=1>Staphylococcus aureus</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>Haemophilus influenzae,Moraxella catarrhalis,Streptococcus pneumoniae</td><td rowspan=1 colspan=1>Moraxella catarrhalis,Streptococcus pneumoniae</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Klebsiella oxytoca,Pseudomonas aeruginosa,Stenotrophomonas maltophilia</td><td rowspan=1 colspan=1>Pseudomonas aeruginosa</td><td rowspan=1 colspan=1>−1</td></tr><tr><td rowspan=1 colspan=1>Klebsiella pneumoniae,Pseudomonas aeruginosa,Stenotrophomonas maltophilia</td><td rowspan=1 colspan=1>Klebsiella pneumoniae,Pseudomonas aeruginosa</td><td rowspan=1 colspan=1>1</td></tr></table>

510(k) Summary – LRT BAL Application K191967 Rev 3.0   

<table><tr><td rowspan=1 colspan=1>LRT BAL Result</td><td rowspan=1 colspan=1>SoC Result</td><td rowspan=1 colspan=1># Cases</td></tr><tr><td rowspan=1 colspan=1>Pseudomonas aeruginosa,Serratia marcescens,Stenotrophomonas maltophilia</td><td rowspan=1 colspan=1>Pseudomonas aeruginosa</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>Pseudomonas aeruginosa,Serratia marcescens,Stenotrophomonas maltophilia</td><td rowspan=1 colspan=1>Pseudomonas aeruginosa,Serratia marcescens</td><td rowspan=1 colspan=1>−1</td></tr><tr><td rowspan=1 colspan=1>Acinetobacter spp.,Escherichia coli,Klebsiella pneumoniae,Pseudomonas aeruginosa</td><td rowspan=1 colspan=1>Klebsiella pneumoniae,Pseudomonas aeruginosa</td><td rowspan=1 colspan=1>−</td></tr><tr><td rowspan=1 colspan=1>Acinetobacter spp.,Escherichia coli,Klebsiella pneumoniae,Stenotrophomonas maltophilia</td><td rowspan=1 colspan=1>Acinetobacter spp.</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Acinetobacter spp.,Escherichia coli,Proteus spp.,Pseudomonas aeruginosa</td><td rowspan=1 colspan=1>Pseudomonas aeruginosa</td><td rowspan=1 colspan=1>−</td></tr><tr><td rowspan=1 colspan=1>Enterobacter cloacae complex,Escherichia coli,Klebsiella oxytoca,Stenotrophomonas maltophilia</td><td rowspan=1 colspan=1>Enterobacter cloacae complex,Escherichia coli,Klebsiella oxytoca</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>Escherichia coli,Haemophilus influenzae,Pneumocystis jirovecii,Staphylococcus aureus</td><td rowspan=1 colspan=1>Staphylococcus aureus</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Haemophilus influenzae,Proteus spp.,Pseudomonas aeruginosa,Stenotrophomonas maltophilia</td><td rowspan=1 colspan=1>Pseudomonas aeruginosa</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Pseudomonas aeruginosa,Serratia marcescens,Staphylococcus aureus,Stenotrophomonas maltophilia</td><td rowspan=1 colspan=1>Pseudomonas aeruginosa,Serratia marcescens,Staphylococcus aureus</td><td rowspan=1 colspan=1>−</td></tr><tr><td rowspan=1 colspan=1>Acinetobacter spp.,Enterobacter cloacae complex,Haemophilus influenzae,Klebsiella pneumoniae,Streptococcus pneumoniae</td><td rowspan=1 colspan=1>Enterobacter cloacae complex,Klebsiella pneumoniae,Streptococcus pneumoniae</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>Citrobacter freundii,Escherichia coli,Klebsiella pneumoniae,Pseudomonas aeruginosa,Serratia marcescens</td><td rowspan=1 colspan=1>Escherichia coli,Klebsiella pneumoniae,Pseudomonas aeruginosa,Serratia marcescens</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Enterobacter cloacae complex,Escherichia coli,Klebsiella oxytoca,Klebsiella variicola,Proteus spp.</td><td rowspan=1 colspan=1>Enterobacter cloacae complex</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Escherichia coli,Klebsiella pneumoniae,Pseudomonas aeruginosa,Serratia marcescens,Stenotrophomonas maltophilia</td><td rowspan=1 colspan=1>Klebsiella pneumoniae,Pseudomonas aeruginosa,Serratia marcescens,Stenotrophomonas maltophilia</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>Escherichia coli,Morganella morganii,Proteus spp.,Pseudomonas aeruginosa,Stenotrophomonas maltophilia</td><td rowspan=1 colspan=1>Pseudomonas aeruginosa</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Haemophilus influenzae,Klebsiella pneumoniae,</td><td rowspan=1 colspan=1>Haemophilus influenzae,Klebsiella pneumoniae,</td><td rowspan=1 colspan=1>1</td></tr></table>

<table><tr><td rowspan=1 colspan=1>LRT BAL Result</td><td rowspan=1 colspan=1>SoC Result</td><td rowspan=1 colspan=1># Cases</td><td rowspan=1 colspan=1>%</td></tr><tr><td rowspan=1 colspan=1>Klebsiella variicola,Proteus spp.,Staphylococcus aureus</td><td rowspan=1 colspan=1>Proteus spp.,Staphylococcus aureus</td><td rowspan=1 colspan=1></td><td rowspan=3 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Haemophilus influenzae,Klebsiella pneumoniae,Moraxella catarrhalis,Pneumocystis jirovecii,Serratia marcescens</td><td rowspan=1 colspan=1>Serratia marcescens</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Acinetobacter spp.,Escherichia coli,Klebsiella pneumoniae,Proteus spp.,Pseudomonas aeruginosa,Staphylococcus aureus</td><td rowspan=1 colspan=1>Klebsiella pneumoniae</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=2>Category 3: LRT BAL detects fewer microorganisms</td><td rowspan=1 colspan=2>fewer detections:2.5 %</td></tr><tr><td rowspan=1 colspan=2>Category 3a: LRT BAL negative</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>2.1</td></tr><tr><td rowspan=1 colspan=1>negative</td><td rowspan=1 colspan=1>Enterobacter cloacae complex</td><td rowspan=1 colspan=1>2</td><td rowspan=10 colspan=1></td></tr><tr><td rowspan=1 colspan=1>negative</td><td rowspan=1 colspan=1>Escherichia coli</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>negative</td><td rowspan=1 colspan=1>Klebsiella oxytoca</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>negative</td><td rowspan=1 colspan=1>Klebsiella pneumoniae</td><td rowspan=1 colspan=1>2</td></tr><tr><td rowspan=1 colspan=1>negative</td><td rowspan=1 colspan=1>Klebsiella variicola</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>negative</td><td rowspan=1 colspan=1>Legionella pneumophila</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>negative</td><td rowspan=1 colspan=1>Pneumocystis jirovecii</td><td rowspan=1 colspan=1>3</td></tr><tr><td rowspan=1 colspan=1>negative</td><td rowspan=1 colspan=1>Pseudomonas aeruginosa</td><td rowspan=1 colspan=1>3</td></tr><tr><td rowspan=1 colspan=1>negative</td><td rowspan=1 colspan=1>Staphylococcus aureus</td><td rowspan=1 colspan=1>5</td></tr><tr><td rowspan=1 colspan=1>negative</td><td rowspan=1 colspan=1>Stenotrophomonas maltophilia</td><td rowspan=1 colspan=1>2</td></tr><tr><td rowspan=1 colspan=1>Category 3b: positive result for both LRT BAL and SoC</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>0.4</td></tr><tr><td rowspan=1 colspan=1>Acinetobacter spp.</td><td rowspan=1 colspan=1>Acinetobacter spp.,Enterobacter cloacae complex</td><td rowspan=1 colspan=1>1</td><td rowspan=4 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Pseudomonas aeruginosa</td><td rowspan=1 colspan=1>Klebsiella pneumoniae,Pseudomonas aeruginosa</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>Stenotrophomonas maltophilia</td><td rowspan=1 colspan=1>Haemophilus influenzae,Stenotrophomonas maltophilia</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>Acinetobacter spp.,Enterobacter cloacae complex</td><td rowspan=1 colspan=1>Acinetobacter spp.,Enterobacter cloacae complex,Klebsiella variicola</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=2>Category 4 LRT BAL and SoC detect different microorganisms</td><td rowspan=1 colspan=2>discordantmicroorganism results:0.6 %</td></tr><tr><td rowspan=1 colspan=2>Category 4a: partially concordant positive results</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0.2</td></tr><tr><td rowspan=1 colspan=1>Citrobacter freundii,Pseudomonas aeruginosa</td><td rowspan=1 colspan=1>Pseudomonas aeruginosa,Staphylococcus aureus</td><td rowspan=1 colspan=1>1</td><td rowspan=2 colspan=1>−</td></tr><tr><td rowspan=1 colspan=1>Enterobacter cloacae complex,Klebsiella oxytoca,Stenotrophomonas maltophilia</td><td rowspan=1 colspan=1>Enterobacter cloacae complex,Klebsiella pneumoniae</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>Category 4b: fully discordant results</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>0.4</td></tr><tr><td rowspan=1 colspan=1>Haemophilus influenzae</td><td rowspan=1 colspan=1>Staphylococcus aureus</td><td rowspan=1 colspan=1>1</td><td rowspan=4 colspan=1>−</td></tr><tr><td rowspan=1 colspan=1>Pseudomonas aeruginosa</td><td rowspan=1 colspan=1>Acinetobacter spp.</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>Pseudomonas aeruginosa</td><td rowspan=1 colspan=1>Staphylococcus aureus</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>Stenotrophomonas maltophilia</td><td rowspan=1 colspan=1>Enterobacter cloacae complex</td><td rowspan=1 colspan=1>1</td></tr></table>

# Microorganism Results – Comparison to Composite Comparator Reference

LRT BAL results from the prospective clinical study were also compared to a composite comparator reference consisting of SoC culture results combined with a validated molecular test (PCR/sequencing, being part of a multiplex PCR assay). For ‘atypical’ microorganisms (for which SoC testing was not performed for all specimens), the composite comparator reference consists of two validated molecular tests for each microorganism (PCR/sequencing, also being part of the multiplex PCR assay). Table 28 summarizes comparison results together with their $9 5 \%$ confidence intervals.

Table 28: Prospective study $\mathbf { N } = \mathbf { 1 } , \mathbf { 0 1 6 }$ specimens a ), comparison to composite comparator.   

<table><tr><td rowspan=1 colspan=1>TP</td><td rowspan=1 colspan=1>FN</td><td rowspan=1 colspan=1>FP</td><td rowspan=1 colspan=1>TN</td><td rowspan=1 colspan=1>PPA [%](95% CI)</td><td rowspan=1 colspan=1>NPA [%](95% CI)</td><td rowspan=1 colspan=1>PPV [%](95% CI)</td><td rowspan=1 colspan=1>NPV [%](95% CI)</td></tr><tr><td rowspan=1 colspan=1>Acinetobacter spp.</td><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>992</td><td rowspan=1 colspan=1>88.9(67.2 - 96.9)</td><td rowspan=1 colspan=1>99.5(98.8 - 99.8)</td><td rowspan=1 colspan=1>76.2(54.9 - 89.4)</td><td rowspan=1 colspan=1>99.8(99.3 - 99.9)</td></tr><tr><td rowspan=1 colspan=1>Chlamydiapneumoniae b</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1,016</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>100.0(99.6 - 100.0)</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>100.0(99.6 - 100.0)</td></tr><tr><td rowspan=1 colspan=1>Citrobacter freundii</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>1,011</td><td rowspan=1 colspan=1>100.0(20.7 - 100.0)</td><td rowspan=1 colspan=1>99.7(99.1 - 99.9)</td><td rowspan=1 colspan=1>25.0(4.6 - 69.9)</td><td rowspan=1 colspan=1>100.0(99.6 - 100.0)</td></tr><tr><td rowspan=1 colspan=1>Enterobacter cloacaecomplex</td><td rowspan=1 colspan=1>18</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>991</td><td rowspan=1 colspan=1>81.8(61.5 - 92.7)</td><td rowspan=1 colspan=1>99.8(99.3 - 99.9)</td><td rowspan=1 colspan=1>90.0(69.9 - 97.2)</td><td rowspan=1 colspan=1>99.6(99.0 - 99.8)</td></tr><tr><td rowspan=1 colspan=1>Escherichia coli</td><td rowspan=1 colspan=1>28</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>19</td><td rowspan=1 colspan=1>963</td><td rowspan=1 colspan=1>82.4(66.5 - 91.7)</td><td rowspan=1 colspan=1>98.1(97.0 - 98.8)</td><td rowspan=1 colspan=1>59.6(45.3 - 72.4)</td><td rowspan=1 colspan=1>99.4(98.7 - 99.7)</td></tr><tr><td rowspan=1 colspan=1>Haemophilusinfluenzae</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>36</td><td rowspan=1 colspan=1>957</td><td rowspan=1 colspan=1>90.9(72.2 - 97.5)</td><td rowspan=1 colspan=1>96.4(95.0 - 97.4)</td><td rowspan=1 colspan=1>35.7(24.5 - 48.8)</td><td rowspan=1 colspan=1>99.8(99.2 - 99.9)</td></tr><tr><td rowspan=1 colspan=1>Klebsiella oxytoca</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>1,000</td><td rowspan=1 colspan=1>80.0(49.0 - 94.3)</td><td rowspan=1 colspan=1>99.4(98.7 - 99.7)</td><td rowspan=1 colspan=1>57.1(32.6 - 78.6)</td><td rowspan=1 colspan=1>99.8(99.3 - 99.9)</td></tr><tr><td rowspan=1 colspan=1>Klebsiellapneumoniae a, c</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>981</td><td rowspan=1 colspan=1>84.0(65.3 - 93.6)</td><td rowspan=1 colspan=1>99.1(98.3 - 99.5)</td><td rowspan=1 colspan=1>70.0(52.1 - 83.3)</td><td rowspan=1 colspan=1>99.6(99.0 - 99.8)</td></tr><tr><td rowspan=1 colspan=1>Klebsiellavariicola a, c</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1,012</td><td rowspan=1 colspan=1>0.0(0.0 - 65.8)</td><td rowspan=1 colspan=1>99.9(99.4 - 100.0)</td><td rowspan=1 colspan=1>0.0(0.0 - 79.3)</td><td rowspan=1 colspan=1>99.8(99.3 - 99.9)</td></tr><tr><td rowspan=1 colspan=1>Legionellapneumophilab</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1,014</td><td rowspan=1 colspan=1>100.0(20.7 - 100.0)</td><td rowspan=1 colspan=1>100.0(99.6 - 100.0)</td><td rowspan=1 colspan=1>100.0(20.7 - 100.0)</td><td rowspan=1 colspan=1>100.0(99.6 - 100.0)</td></tr><tr><td rowspan=1 colspan=1>Moraxellacatarrhalis</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>993</td><td rowspan=1 colspan=1>75.0(50.5 - 89.8)</td><td rowspan=1 colspan=1>99.7(99.1 - 99.9)</td><td rowspan=1 colspan=1>80.0(54.8 - 93.0)</td><td rowspan=1 colspan=1>99.6(99.0 - 99.8)</td></tr><tr><td rowspan=1 colspan=1>Morganella morganii</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1,009</td><td rowspan=1 colspan=1>100.0(43.9 - 100.0)</td><td rowspan=1 colspan=1>100.0(99.6 - 100.0)</td><td rowspan=1 colspan=1>100.0(43.9 - 100.0)</td><td rowspan=1 colspan=1>100.0(99.6 - 100.0)</td></tr><tr><td rowspan=1 colspan=1>Mycoplasmapneumoniae b</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>1,010</td><td rowspan=1 colspan=1>100.0(43.9 - 100.0)</td><td rowspan=1 colspan=1>99.7(99.1 - 99.9)</td><td rowspan=1 colspan=1>50.0(18.8 - 81.2)</td><td rowspan=1 colspan=1>100.0(99.6 - 100.0)</td></tr><tr><td rowspan=1 colspan=1>Pneumocystisjirovecii b</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>5d</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>989</td><td rowspan=1 colspan=1>80.0(60.9 - 91.1)</td><td rowspan=1 colspan=1>99.8(99.3 - 99.9)</td><td rowspan=1 colspan=1>90.9(72.2 -97.5)</td><td rowspan=1 colspan=1>99.5(98.8 - 99.8)</td></tr><tr><td rowspan=1 colspan=1>Proteus spp.</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1,006</td><td rowspan=1 colspan=1>100.0(72.3 - 100.0)</td><td rowspan=1 colspan=1>100.0(99.6 - 100.0)</td><td rowspan=1 colspan=1>100.0(72.3 - 100.0)</td><td rowspan=1 colspan=1>100.0(99.6 - 100.0)</td></tr><tr><td rowspan=1 colspan=1>Pseudomonasaeruginosa</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>896</td><td rowspan=1 colspan=1>93.5(87.1 - 96.8)</td><td rowspan=1 colspan=1>98.7(97.7 - 99.2)</td><td rowspan=1 colspan=1>89.3(82.2 - 93.8)</td><td rowspan=1 colspan=1>99.2(98.4 - 99.6)</td></tr><tr><td rowspan=1 colspan=1>Serratia marcescens</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>997</td><td rowspan=1 colspan=1>93.8(71.7 - 98.9)</td><td rowspan=1 colspan=1>99.8(99.3 - 99.9)</td><td rowspan=1 colspan=1>88.2(65.7 - 96.7)</td><td rowspan=1 colspan=1>99.9(99.4 - 100.0)</td></tr><tr><td rowspan=1 colspan=1>Staphylococcusaureus</td><td rowspan=1 colspan=1>80</td><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>896</td><td rowspan=1 colspan=1>83.3(74.6 - 89.5)</td><td rowspan=1 colspan=1>97.4(96.1 - 98.2)</td><td rowspan=1 colspan=1>76.9(68.0 - 84.0)</td><td rowspan=1 colspan=1>98.2(97.2 - 98.9)</td></tr><tr><td rowspan=1 colspan=1>Stenotrophomonasmaltophilia</td><td rowspan=1 colspan=1>34</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>967</td><td rowspan=1 colspan=1>82.9(68.7 - 91.5)</td><td rowspan=1 colspan=1>99.3(98.5 - 99.7)</td><td rowspan=1 colspan=1>82.9(68.7 - 91.5)</td><td rowspan=1 colspan=1>99.3(98.5 - 99.7)</td></tr><tr><td rowspan=1 colspan=1>Streptococcuspneumoniae</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>1,003</td><td rowspan=1 colspan=1>100.0(74.1 - 100.0)</td><td rowspan=1 colspan=1>99.8(99.3 - 99.9)</td><td rowspan=1 colspan=1>84.6(57.8 - 95.7)</td><td rowspan=1 colspan=1>100.0(99.6 - 100.0)</td></tr></table>

# Antibiotic Resistance Marker Results – Comparison to Molecular Reference

Performance characteristics of LRT BAL panel antibiotic resistance markers were evaluated for the prospective study (1,016 specimens) by comparing to corresponding molecular reference assays to determine PPAs $\left( \mathrm { T P } / \left( \mathrm { T P } + \mathrm { F N } \right) \right)$ and NPAs $\left( \mathrm { F N } / \left( \mathrm { F N + F P } \right) \right)$ together with $9 5 \%$ confidence intervals. For each antibiotic resistance marker, one PCR assay was performed followed by bi-directional sequencing.

Antibiotic resistance marker results are only reported by the LRT BAL Application if at least one corresponding host microorganism is simultaneously detected. Results for antibiotic resistance markers detected without a positive corresponding host microorganism result are masked on the results screen. Table 29 reports antibiotic resistance markers as displayed on the LRT BAL results screen, i.e., all antibiotic resistance markers for which a corresponding host microorganism was simultaneously detected. Note that PPA/NPA performance data are driven by LRT BAL microorganism results.

Table 29: Comparison of antibiotic resistance markers to corresponding molecular reference assays (PCR/sequencing) for the prospective study ( $\mathbf { N } = \mathbf { 1 } , \mathbf { 0 1 6 }$ specimens) for markers as displayed and reported on the LRT BAL results screen (number of detected host microorganisms for each marker in brackets).   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>TP</td><td rowspan=1 colspan=1>FN</td><td rowspan=1 colspan=1>FP c, d</td><td rowspan=1 colspan=1>TN</td><td rowspan=1 colspan=1>PPA [%](95% CI)</td><td rowspan=1 colspan=1>NPA [%](95% CI)</td></tr><tr><td rowspan=1 colspan=1>ctx-M(N = 208)</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>1 b</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>197</td><td rowspan=1 colspan=1>88.9(56.5 - 98.0)</td><td rowspan=1 colspan=1>99.5(97.2 - 99.9)</td></tr><tr><td rowspan=1 colspan=1>kpc(N = 208)</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>204</td><td rowspan=1 colspan=1>100.0(43.9 - 100.0)</td><td rowspan=1 colspan=1>99.5(97.3 - 99.9)</td></tr><tr><td rowspan=1 colspan=1>mecA(N = 104)</td><td rowspan=1 colspan=1>22</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>25 e</td><td rowspan=1 colspan=1>57</td><td rowspan=1 colspan=1>100.0(85.1 - 100.0)</td><td rowspan=1 colspan=1>69.5(58.9 - 78.4)</td></tr><tr><td rowspan=1 colspan=1>ndm(N = 208)</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>207</td><td rowspan=1 colspan=1>100.0(20.7 - 100.0)</td><td rowspan=1 colspan=1>100.0(98.2 - 100.0)</td></tr><tr><td rowspan=1 colspan=1>oxa-23(N = 21)</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>18</td><td rowspan=1 colspan=1>100.0(43.9 - 100.0)</td><td rowspan=1 colspan=1>100.0(82.4 - 100.0)</td></tr><tr><td rowspan=1 colspan=1>oxa-24(N = 21)</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>100.0(43.9 - 100.0)</td><td rowspan=1 colspan=1>94.1(73.0 - 99.0)</td></tr><tr><td rowspan=1 colspan=1>oxa-48(N = 112)</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>111</td><td rowspan=1 colspan=1>100.0(20.7 - 100.0)</td><td rowspan=1 colspan=1>100.0(96.7 - 100.0)</td></tr><tr><td rowspan=1 colspan=1>oxa-58(N = 21)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>100.0(84.5 - 100.0)</td></tr><tr><td rowspan=1 colspan=1>tem(N = 56)</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>7f</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>100.0(70.1 - 100.0)</td><td rowspan=1 colspan=1>85.1(72.3 - 92.6)</td></tr><tr><td rowspan=1 colspan=1>vim(N = 208)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>207</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>99.5(97.3 - 99.9)</td></tr></table>

a Observed invalid LRT BAL analyte results: ctx-M (1) and oxa-24 (1).   
b One false negative result belongs to the $_ { c t x }$ -M9 subgroup that is not covered by the ctx-M assay of the LRT BAL Application (targeted against ctx-M1 subgroup).

c Note that molecular reference assays use cutoffs to achieve analytical sensitivities in a similar range to LRT BAL. Negative results by molecular reference assays do not preclude the presence of the antibiotic resistance marker in the specimen at a low concentration, nor its possible clinical relevance. Please refer to section ‘Correlation of Detected Antibiotic Resistance Markers to Strain Genotypes and Phenotypes’, p. 68 for genotypic and phenotypic correlation of reported antibiotic resistance markers.

d Specimens with false positive LRT BAL results were analyzed with additional molecular assays (PCR/bi-directional sequencing) using specimen DNA extracts for presence or absence of antibiotic resistance markers. Presence of antibiotic resistance markers was confirmed in 0 of 1 case for ctx-M, 1 of 1 case of kpc, 10 of 25 cases for mecA, 1 of 1 case for oxa-24, 6 of 7 cases for tem and 1 of 1 case for vim. Cross-reactivity to other off-panel analytes has not been observed. For all not confirmed cases, PCRs did not amplify sufficient amounts for sequencing.

e Note that Staphylococci other than $S .$ . aureus commonly harbor mecA. Positive mecA results can, therefore, be due to the presence of this marker in such microorganisms.

f Although tem is reported by LRT BAL for $H .$ . influenzae only, other Gram-negative microorganisms (e.g., Enterobacteriaceae, Acinetobacter spp., $P .$ aeruginosa) can harbor tem. Positive tem results can, therefore, be due to the presence of this marker in such microorganisms.

Antibiotic Resistance Marker Results – Agreement Rates to Positive and Negative Microorganisms as Detected by Culture/Isolate Sequencing or Composite Comparator Testing

In order to determine agreement rates of antibiotic resistance markers reported by LRT BAL to positive and negative reference results, a comparison to the following references was performed: (A) SoC culture result for microorganisms, combined with sequencing results/marker screenings of provided strain isolates, and (B) composite comparator result for microorganisms and molecular reference results (PCR/sequencing) for antibiotic resistance markers. Note that an isolate was not provided for all SoC positive microorganisms, thereby limiting the dataset for comparison (A). Comparison (B) evaluates the entire specimen cohort, including all positive LRT BAL results that were negative by 510(k) Summary – LRT BAL Application K191967 Rev 3.0

SoC culture. Molecular reference assays use cutoffs that were set to achieve an analytical assay sensitivity in a comparable range to LRT BAL.

For both comparisons, the following agreement rates are evaluated for ctx-M, kpc, ndm, and vim (for Enterobacteriaceae, Acinetobacter spp. and/or $P .$ aeruginosa as possible host microorganisms), for oxa-48 (for Enterobacteriaceae), for tem (for $H .$ influenzae), and for mecA (for S. aureus) in Tables 30 - 38:

1. Agreement of one (or more) detected host microorganisms in combination with an antibiotic resistance marker $( \mathbf { O r g } { + } / \mathbf { R e s } { + } )$ between LRT BAL and reference methods (A) or $\mathbf { ( B ) }$ .   
2. Agreement of one (or more) detected host microorganisms without reported antibiotic resistance marker $( \bf { O r g } + / \bf { R e s } - )$ between LRT BAL and reference methods (A) or $\mathbf { ( B ) }$ .   
3. Agreement of a negative result for one (or more) host microorganisms (Org-) between LRT BAL and reference methods (A) and (B).

As presence of mecA is expected to directly correlate with cefoxitin/oxacillin resistance in S. aureus strains (MRSA, methicillin-resistant S. aureus), while the absence of mecA is expected to correlate with susceptible $S .$ . aureus strains (MSSA, methicillin-sensitive S. aureus), mecA was furthermore compared to the MRSA/MSSA status of the isolate as reported by SoC culture as a third reference (C) in Table 38:

1. Agreement of a detected $S .$ . aureus in combination with a detected mecA as reported by LRT BAL $( \mathbf { O r g } { + } / \mathbf { R e s } { + } )$ to $S .$ . aureus reported with an MRSA phenotype by SoC culture $( \mathbf { O r g } { + } / \mathbf { M R S A } )$ .   
2. Agreement of a detected S. aureus together with a negative result for mecA $( \bf { O r g } { + } / \bf { R e s } { - } )$ as reported by LRT BAL to $S .$ aureus reported with an MSSA phenotype by SoC culture $( \mathbf { O r g } { + } / \mathbf { M S S A } )$ .   
3. Agreement of a negative result for S. aureus between LRT BAL and SoC culture.

No agreement tables are included for oxa-58 for which positive results were reported neither by the LRT BAL Application, nor by the reference methods.

A

Table 30: Agreement rates for oxa-23 between LRT BAL and reference methods A (culture) and B (composite comparator) for Acinetobacter spp. as host microorganism.   

<table><tr><td rowspan=2 colspan=2>Acinetobacter spp.0xa-23</td><td rowspan=1 colspan=5>Culture Result for Acinetobacter spp. +Isolate Sequencing/Marker Screening for oxa-23</td></tr><tr><td rowspan=1 colspan=1>Org+/Res+</td><td rowspan=1 colspan=1>Org+/Res-</td><td rowspan=1 colspan=1>Org+/No Isolate</td><td rowspan=1 colspan=1>Org-</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=4 colspan=1>LRT BALResult</td><td rowspan=1 colspan=1>Org+/Res+</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>3</td></tr><tr><td rowspan=1 colspan=1>Org+/Res-</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>18</td></tr><tr><td rowspan=1 colspan=1>Org-</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>993</td><td rowspan=1 colspan=1>994</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>1,004</td><td rowspan=1 colspan=1>1,015 a</td></tr><tr><td rowspan=1 colspan=3>Acinetobacter spp/oxa-23</td><td rowspan=1 colspan=2>rate [%]</td><td rowspan=1 colspan=1>positivity</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=3>Agreement (Org+/Res+)</td><td rowspan=1 colspan=2>100.0</td><td rowspan=1 colspan=1>1/1</td><td rowspan=1 colspan=1>20.7 - 100.0</td></tr><tr><td rowspan=1 colspan=3>Agreement (Org+/Res-)</td><td rowspan=1 colspan=2>100.0</td><td rowspan=1 colspan=1>7/7</td><td rowspan=1 colspan=1>64.6 - 100.0</td></tr><tr><td rowspan=1 colspan=3>Agreement (Org-)</td><td rowspan=1 colspan=2>98.9</td><td rowspan=1 colspan=1>993/1,004</td><td rowspan=1 colspan=1>98.0 - 99.4</td></tr></table>

B

<table><tr><td rowspan=2 colspan=2>Acinetobacter spp.0xa-23</td><td rowspan=1 colspan=4>Composite Comparator Result for Acinetobacter spp. +Mol. Reference (PCR/Seq.) for oxa-23</td></tr><tr><td rowspan=1 colspan=1>Org+/Res+</td><td rowspan=1 colspan=1>Org+/Res-</td><td rowspan=1 colspan=1>Org-</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=4 colspan=1>LRT BALResult</td><td rowspan=1 colspan=1>Org+/Res+</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3</td></tr><tr><td rowspan=1 colspan=1>Org+/Res-</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>18</td></tr><tr><td rowspan=1 colspan=1>Org-</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>992</td><td rowspan=1 colspan=1>994</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>997</td><td rowspan=1 colspan=1>1,015 a</td></tr><tr><td rowspan=1 colspan=3>Acinetobacter spp/oxa-23</td><td rowspan=1 colspan=1>rate [%]</td><td rowspan=1 colspan=1>positivity</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=3>Agreement (Org+/Res+)</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>43.9 - 100.0</td></tr><tr><td rowspan=1 colspan=3>Agreement (Org+/Res-)</td><td rowspan=1 colspan=1>86.7</td><td rowspan=1 colspan=1>13/15</td><td rowspan=1 colspan=1>62.1 - 96.3</td></tr><tr><td rowspan=1 colspan=3>Agreement (Org-)</td><td rowspan=1 colspan=1>99.5</td><td rowspan=1 colspan=1>992/997</td><td rowspan=1 colspan=1>98.8 - 99.8</td></tr></table>

A

Table 31: Agreement rates for oxa-24 between LRT BAL and reference methods A (culture) and B (composite comparator) for Acinetobacter spp. as host microorganism.   

<table><tr><td rowspan=2 colspan=2>Acinetobacter spp.oxa-24</td><td rowspan=1 colspan=5>Culture Result for Acinetobacter spp. +Isolate Sequencing/Marker Screening for oxa-24</td></tr><tr><td rowspan=1 colspan=1>Org+/Res+</td><td rowspan=1 colspan=1>Org+/Res-</td><td rowspan=1 colspan=1>Org+/No Isolate</td><td rowspan=1 colspan=1>Org-</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=4 colspan=1>LRT BALResult</td><td rowspan=1 colspan=1>Org+/Res+</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>4</td></tr><tr><td rowspan=1 colspan=1>Org+/Res-</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>16</td></tr><tr><td rowspan=1 colspan=1>Org-</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>993</td><td rowspan=1 colspan=1>994</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>1,004</td><td rowspan=1 colspan=1>1,014 a</td></tr><tr><td rowspan=1 colspan=3>Acinetobacter spp/oxa-24</td><td rowspan=1 colspan=2>rate [%]</td><td rowspan=1 colspan=1>positivity</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=3>Agreement (Org+/Res+)</td><td rowspan=1 colspan=2>100.0</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>43.9 - 100.0</td></tr><tr><td rowspan=1 colspan=3>Agreement (Org+/Res-)</td><td rowspan=1 colspan=2>100.0</td><td rowspan=1 colspan=1>4/4</td><td rowspan=1 colspan=1>51.0 - 100.0</td></tr><tr><td rowspan=1 colspan=3>Agreement (Org-)</td><td rowspan=1 colspan=2>98.9</td><td rowspan=1 colspan=1>993/1,004</td><td rowspan=1 colspan=1>98.0 - 99.4</td></tr></table>

B

<table><tr><td rowspan=2 colspan=2>Acinetobacter spp.oxa-24</td><td rowspan=1 colspan=4>Composite Comparator Result for Acinetobacter spp. +Mol. Reference (PCR/Seq.) for oxa-24</td></tr><tr><td rowspan=1 colspan=1>Org+/Res+</td><td rowspan=1 colspan=1>Org+/Res-</td><td rowspan=1 colspan=1>Org-</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=4 colspan=1>LRT BALResult</td><td rowspan=1 colspan=1>Org+/Res+</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>4</td></tr><tr><td rowspan=1 colspan=1>Org+/Res-</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>16</td></tr><tr><td rowspan=1 colspan=1>Org-</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>992</td><td rowspan=1 colspan=1>994</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>997</td><td rowspan=1 colspan=1>1,014 a</td></tr><tr><td rowspan=1 colspan=3>Acinetobacter spp/oxa-24</td><td rowspan=1 colspan=1>rate [%]</td><td rowspan=1 colspan=1>positivity</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=3>Agreement (Org+/Res+)</td><td rowspan=1 colspan=1>75.0</td><td rowspan=1 colspan=1>3/4</td><td rowspan=1 colspan=1>30.1 - 95.4</td></tr><tr><td rowspan=1 colspan=3>Agreement (Org+/Res-)</td><td rowspan=1 colspan=1>92.3</td><td rowspan=1 colspan=1>12/13</td><td rowspan=1 colspan=1>66.7 - 98.6</td></tr><tr><td rowspan=1 colspan=3>Agreement (Org-)</td><td rowspan=1 colspan=1>99.5</td><td rowspan=1 colspan=1>992/997</td><td rowspan=1 colspan=1>98.8 - 99.8</td></tr></table>

A

Table 32: Agreement rates for ctx-M between LRT BAL and reference methods A (culture) and B (composite comparator) for combined possible host microorganisms: Enterobacteriaceae, Acinetobacter spp., P. aeruginosa.   

<table><tr><td rowspan=2 colspan=2>Combined Hostsctx-M</td><td rowspan=1 colspan=5>Culture Result for Host Microorganism(s) +Isolate Sequencing/Marker Screening for ctx-M</td></tr><tr><td rowspan=1 colspan=1>Org+/Res+</td><td rowspan=1 colspan=1>Org+/Res-</td><td rowspan=1 colspan=1>Org+/No Isolate</td><td rowspan=1 colspan=1>Org-</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=4 colspan=1>LRT BALResult</td><td rowspan=1 colspan=1>Org+/Res+</td><td rowspan=1 colspan=1>3d</td><td rowspan=1 colspan=1>4 b, e</td><td rowspan=1 colspan=1>1 b, f</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>9</td></tr><tr><td rowspan=1 colspan=1>Org+/Res-</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>92</td><td rowspan=1 colspan=1>32</td><td rowspan=1 colspan=1>74</td><td rowspan=1 colspan=1>198</td></tr><tr><td rowspan=1 colspan=1>Org-</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>794</td><td rowspan=1 colspan=1>805</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>3 a</td><td rowspan=1 colspan=1>99</td><td rowspan=1 colspan=1>41</td><td rowspan=1 colspan=1>869</td><td rowspan=1 colspan=1>1,012 c</td></tr><tr><td rowspan=1 colspan=3>Combined Hosts/ctx-M</td><td rowspan=1 colspan=2>rate [%]</td><td rowspan=1 colspan=1>positivity</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=3>Agreement (Org+/Res+)</td><td rowspan=1 colspan=2>100.0</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>43.9 - 100.0</td></tr><tr><td rowspan=1 colspan=3>Agreement (Org+/Res-)</td><td rowspan=1 colspan=2>92.9</td><td rowspan=1 colspan=1>92/99</td><td rowspan=1 colspan=1>86.1 - 96.5</td></tr><tr><td rowspan=1 colspan=3>Agreement (Org-)</td><td rowspan=1 colspan=2>91.4</td><td rowspan=1 colspan=1>794/869</td><td rowspan=1 colspan=1>89.3 - 93.1</td></tr></table>

B

<table><tr><td rowspan=10 colspan=1>8</td><td rowspan=2 colspan=2>Combined Hostsctx-M</td><td rowspan=1 colspan=4>Composite Comparator Result for Host Microorganism(s) +Mol. Reference (PCR/Seq.) for ctx-M</td></tr><tr><td rowspan=1 colspan=1>Org+/Res+</td><td rowspan=1 colspan=1>Org+/Res-</td><td rowspan=1 colspan=1>Org-</td><td rowspan=1 colspan=1>total</td></tr><tr><td rowspan=4 colspan=1>LRT BALResult</td><td rowspan=1 colspan=1>Org+/Res+</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>9</td></tr><tr><td rowspan=1 colspan=1>Org+/Res-</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>161</td><td rowspan=1 colspan=1>36</td><td rowspan=1 colspan=1>198</td></tr><tr><td rowspan=1 colspan=1>Org-</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>785</td><td rowspan=1 colspan=1>805</td></tr><tr><td rowspan=1 colspan=1>total</td><td rowspan=1 colspan=1>9 a</td><td rowspan=1 colspan=1>182</td><td rowspan=1 colspan=1>821</td><td rowspan=1 colspan=1>1,012 c</td></tr><tr><td rowspan=1 colspan=3>Combined Hosts/ctx-M</td><td rowspan=1 colspan=1>rate [%]</td><td rowspan=1 colspan=1>positivity</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=3>Agreement (Org+/Res+)</td><td rowspan=1 colspan=1>88.9</td><td rowspan=1 colspan=1>8/9</td><td rowspan=1 colspan=1>56.5 - 98.0</td></tr><tr><td rowspan=1 colspan=3>Agreement (Org+/Res-)</td><td rowspan=1 colspan=1>88.5</td><td rowspan=1 colspan=1>161/182</td><td rowspan=1 colspan=1>83.0 - 92.3</td></tr><tr><td rowspan=1 colspan=3>Agreement (Org-)</td><td rowspan=1 colspan=1>95.6</td><td rowspan=1 colspan=1>785/821</td><td rowspan=1 colspan=1>94.0 - 96.8</td></tr></table>

a Specimens with multiple detected possible host microorganisms by culture: 1 of 3, by the composite comparator: 6 of 9. b For one $\mathrm { \partial ^ { \cdot } O r g ^ { + } / R e s ^ { - } }$ and for one $\cdot _ { \mathrm { O r g } + / \mathrm { N } _ { 0 } }$ Isolate’ case, presence of the ctx-M gene was confirmed for an off-panel isolate of Providencia stuartii. c Four invalid LRT BAL ctx-M results. d SoC reported the following possible corresponding host microorganisms for three $\mathrm { O r g ^ { + / R e s ^ { + } } }$ cases: $E .$ . coli, K. pneumoniae, K. pneumoniae/P. aeruginosa. e SoC reported the following possible corresponding host microorganisms for four Org+/Res- cases: E. coli, P. aeruginosa $( 2 \mathbf { x } )$ , K. pneumoniae/P. aeruginosa/S. marcescens. f SoC reported the following possible corresponding host microorganisms for one $\mathrm { O r g ^ { + / N o } }$ Isolate case: K. pneumoniae.

A

Table 33: Agreement rates for kpc between LRT BAL and reference methods A (culture) and B (composite comparator) for combined possible host microorganisms: Enterobacteriaceae, Acinetobacter spp., $P .$ aeruginosa.   

<table><tr><td rowspan=2 colspan=2>Combined Hostskpc</td><td rowspan=1 colspan=5>Culture Result for Host Microorganism(s) +Isolate Sequencing/Marker Screening for kpc</td></tr><tr><td rowspan=1 colspan=1>Org+/Res+</td><td rowspan=1 colspan=1>Org+/Res-</td><td rowspan=1 colspan=1>Org+/No Isolate</td><td rowspan=1 colspan=1>Org-</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=4 colspan=1>LRT BALResult</td><td rowspan=1 colspan=1>Org+/Res+</td><td rowspan=1 colspan=1>1 b</td><td rowspan=1 colspan=1>2c</td><td rowspan=1 colspan=1>1d</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>4</td></tr><tr><td rowspan=1 colspan=1>Org+/Res-</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>96</td><td rowspan=1 colspan=1>32</td><td rowspan=1 colspan=1>76</td><td rowspan=1 colspan=1>204</td></tr><tr><td rowspan=1 colspan=1>Org-</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>797</td><td rowspan=1 colspan=1>808</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>1 a</td><td rowspan=1 colspan=1>101</td><td rowspan=1 colspan=1>41</td><td rowspan=1 colspan=1>873</td><td rowspan=1 colspan=1>1,016</td></tr><tr><td rowspan=1 colspan=3>Combined Hosts/kpc</td><td rowspan=1 colspan=2>rate [%]</td><td rowspan=1 colspan=1>positivity</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=3>Agreement (Org+/Res+)</td><td rowspan=1 colspan=2>100.0</td><td rowspan=1 colspan=1>1/1</td><td rowspan=1 colspan=1>20.7 - 100.0</td></tr><tr><td rowspan=1 colspan=3>Agreement (Org+/Res-)</td><td rowspan=1 colspan=2>95.0</td><td rowspan=1 colspan=1>96/101</td><td rowspan=1 colspan=1>88.9 - 97.9</td></tr><tr><td rowspan=1 colspan=3>Agreement (Org-)</td><td rowspan=1 colspan=2>91.3</td><td rowspan=1 colspan=1>797/873</td><td rowspan=1 colspan=1>89.2 - 93.0</td></tr></table>

B

<table><tr><td rowspan=10 colspan=1>3</td><td rowspan=2 colspan=2>Combined Hostskpc</td><td rowspan=1 colspan=4>Composite Comparator Result for Host Microorganism(s) +Mol. Reference (PCR/Seq.) for kpc</td></tr><tr><td rowspan=1 colspan=1>Org+/Res+</td><td rowspan=1 colspan=1>Org+/Res-</td><td rowspan=1 colspan=1>Org-</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=4 colspan=1>LRT BALResult</td><td rowspan=1 colspan=1>Org+/Res+</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>4</td></tr><tr><td rowspan=1 colspan=1>Org+/Res-</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>167</td><td rowspan=1 colspan=1>37</td><td rowspan=1 colspan=1>204</td></tr><tr><td rowspan=1 colspan=1>Org-</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>788</td><td rowspan=1 colspan=1>808</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>3 a</td><td rowspan=1 colspan=1>188</td><td rowspan=1 colspan=1>825</td><td rowspan=1 colspan=1>1,016</td></tr><tr><td rowspan=1 colspan=3>Combined Hosts/kpc</td><td rowspan=1 colspan=1>rate [%]</td><td rowspan=1 colspan=1>positivity</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=3>Agreement (Org+/Res+)</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>43.9 - 100.0</td></tr><tr><td rowspan=1 colspan=3>Agreement (Org+/Res-)</td><td rowspan=1 colspan=1>88.8</td><td rowspan=1 colspan=1>167/188</td><td rowspan=1 colspan=1>83.5 - 92.6</td></tr><tr><td rowspan=1 colspan=3>Agreement (Org-)</td><td rowspan=1 colspan=1>95.5</td><td rowspan=1 colspan=1>788/825</td><td rowspan=1 colspan=1>93.9 - 96.7</td></tr></table>

a Specimens with multiple detected possible host microorganisms by culture: 0 of 1, by the composite comparator: 2 of 3. b SoC reported the following possible corresponding host microorganisms for one $\mathrm { O r g ^ { + / R e s ^ { + } } }$ case: $E .$ . cloacae complex. c SoC reported the following possible corresponding host microorganisms for two $\mathrm { O r g ^ { + } / R e s ^ { - } }$ cases: Acinetobacter spp., $K .$ . pneumoniae/P. aeruginosa. d SoC reported the following possible corresponding host microorganisms for one $\mathrm { O r g ^ { + / N o } }$ Isolate case: $K .$ . pneumoniae.

A

Table 34: Agreement rates for ndm between LRT BAL and reference methods A (culture) and B (composite comparator) for combined possible host microorganisms: Enterobacteriaceae, Acinetobacter spp., $P .$ aeruginosa.   

<table><tr><td rowspan=2 colspan=2>Combined Hostsndm</td><td rowspan=1 colspan=5>Culture Result for Host Microorganism(s) +Isolate Sequencing/Marker Screening for ndm</td></tr><tr><td rowspan=1 colspan=1>Org+/Res+</td><td rowspan=1 colspan=1>Org+/Res-</td><td rowspan=1 colspan=1>Org+/No Isolate</td><td rowspan=1 colspan=1>Org-</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=4 colspan=1>LRT BALResult</td><td rowspan=1 colspan=1>Org+/Res+</td><td rowspan=1 colspan=1>1c</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>Org+/Res-</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>98</td><td rowspan=1 colspan=1>33</td><td rowspan=1 colspan=1>76</td><td rowspan=1 colspan=1>207</td></tr><tr><td rowspan=1 colspan=1>Org-</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>796</td><td rowspan=1 colspan=1>807</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>1 a</td><td rowspan=1 colspan=1>101</td><td rowspan=1 colspan=1>41</td><td rowspan=1 colspan=1>872</td><td rowspan=1 colspan=1>1,015 b</td></tr><tr><td rowspan=1 colspan=3>Combined Hosts/ndm</td><td rowspan=1 colspan=2>rate [%]</td><td rowspan=1 colspan=1>positivity</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=3>Agreement (Org+/Res+)</td><td rowspan=1 colspan=2>100.0</td><td rowspan=1 colspan=1>1/1</td><td rowspan=1 colspan=1>20.7 - 100.0</td></tr><tr><td rowspan=1 colspan=3>Agreement (Org+/Res-)</td><td rowspan=1 colspan=2>97.0</td><td rowspan=1 colspan=1>98/101</td><td rowspan=1 colspan=1>91.6 - 99.0</td></tr><tr><td rowspan=1 colspan=3>Agreement (Org-)</td><td rowspan=1 colspan=2>91.3</td><td rowspan=1 colspan=1>796/872</td><td rowspan=1 colspan=1>89.2 - 93.0</td></tr></table>

B

<table><tr><td rowspan=2 colspan=2>Combined Hostsndm</td><td rowspan=1 colspan=4>Composite Comparator Result for Host Microorganism(s) +Mol. Reference (PCR/Seq.) for ndm</td></tr><tr><td rowspan=1 colspan=1>Org+/Res+</td><td rowspan=1 colspan=1>Org+/Res-</td><td rowspan=1 colspan=1>Org-</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=4 colspan=1>LRT BALResult</td><td rowspan=1 colspan=1>Org+/Res+</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>Org+/Res-</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>170</td><td rowspan=1 colspan=1>37</td><td rowspan=1 colspan=1>207</td></tr><tr><td rowspan=1 colspan=1>Org-</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>787</td><td rowspan=1 colspan=1>807</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>1a</td><td rowspan=1 colspan=1>190</td><td rowspan=1 colspan=1>824</td><td rowspan=1 colspan=1>1,015 b</td></tr><tr><td rowspan=1 colspan=3>Combined Hosts/ndm</td><td rowspan=1 colspan=1>rate [%]</td><td rowspan=1 colspan=1>positivity</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=3>Agreement (Org+/Res+)</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>1/1</td><td rowspan=1 colspan=1>20.7 - 100.0</td></tr><tr><td rowspan=1 colspan=3>Agreement (Org+/Res-)</td><td rowspan=1 colspan=1>89.5</td><td rowspan=1 colspan=1>170/190</td><td rowspan=1 colspan=1>84.3 - 93.1</td></tr><tr><td rowspan=1 colspan=3>Agreement (Org-)</td><td rowspan=1 colspan=1>95.5</td><td rowspan=1 colspan=1>787/824</td><td rowspan=1 colspan=1>93.9 - 96.7</td></tr></table>

a Specimens with multiple detected possible host microorganisms by culture: 1 of 1, by the composite comparator: 1 of 1. b One invalid LRT BAL result for ndm. c SoC reported the following possible corresponding host microorganisms for one $\mathrm { O r g ^ { + / R e s ^ { + } } }$ case: K. pneumoniae/P. aeruginosa.

A

Table 35: Agreement rates for vim between LRT BAL and reference methods A (culture) and B (composite comparator) for combined possible host microorganisms: Enterobacteriaceae, Acinetobacter spp., $P .$ aeruginosa.   

<table><tr><td rowspan=2 colspan=2>Combined Hostsvim</td><td rowspan=1 colspan=5>Culture Result for Host Microorganism(s) +Isolate Sequencing/Marker Screening for vim</td></tr><tr><td rowspan=1 colspan=1>Org+/Res+</td><td rowspan=1 colspan=1>Org+/Res-</td><td rowspan=1 colspan=1>Org+/No Isolate</td><td rowspan=1 colspan=1>Org-</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=4 colspan=1>LRT BALResult</td><td rowspan=1 colspan=1>Org+/Res+</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1 a</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>Org+/Res-</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>98</td><td rowspan=1 colspan=1>33</td><td rowspan=1 colspan=1>76</td><td rowspan=1 colspan=1>207</td></tr><tr><td rowspan=1 colspan=1>Org-</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>797</td><td rowspan=1 colspan=1>808</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>102</td><td rowspan=1 colspan=1>41</td><td rowspan=1 colspan=1>873</td><td rowspan=1 colspan=1>1,016</td></tr><tr><td rowspan=1 colspan=3>Combined Hosts/vim</td><td rowspan=1 colspan=2>rate [%]</td><td rowspan=1 colspan=1>positivity</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=3>Agreement (Org+/Res+)</td><td rowspan=1 colspan=2>NA</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=3>Agreement (Org+/Res-)</td><td rowspan=1 colspan=2>96.1</td><td rowspan=1 colspan=1>98/102</td><td rowspan=1 colspan=1>90.3 - 98.5</td></tr><tr><td rowspan=1 colspan=3>Agreement (Org-)</td><td rowspan=1 colspan=2>91.3</td><td rowspan=1 colspan=1>797/873</td><td rowspan=1 colspan=1>89.2 - 93.0</td></tr></table>

B

<table><tr><td rowspan=2 colspan=2>Combined Hostsvim</td><td rowspan=1 colspan=4>Composite Comparator Result for Host Microorganism(s) +Mol. Reference (PCR/Seq.) for vim</td></tr><tr><td rowspan=1 colspan=1>Org+/Res+</td><td rowspan=1 colspan=1>Org+/Res-</td><td rowspan=1 colspan=1>Org-</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=4 colspan=1>LRT BALResult</td><td rowspan=1 colspan=1>Org+/Res+</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>Org+/Res-</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>170</td><td rowspan=1 colspan=1>37</td><td rowspan=1 colspan=1>207</td></tr><tr><td rowspan=1 colspan=1>Org-</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>788</td><td rowspan=1 colspan=1>808</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>191</td><td rowspan=1 colspan=1>825</td><td rowspan=1 colspan=1>1,016</td></tr><tr><td rowspan=1 colspan=3>Combined Hosts/vim</td><td rowspan=1 colspan=1>rate [%]</td><td rowspan=1 colspan=1>positivity</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=3>Agreement (Org+/Res+)</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=3>Agreement (Org+/Res-)</td><td rowspan=1 colspan=1>89.0</td><td rowspan=1 colspan=1>170/191</td><td rowspan=1 colspan=1>83.8 - 92.7</td></tr><tr><td rowspan=1 colspan=3>Agreement (Org-)</td><td rowspan=1 colspan=1>95.5</td><td rowspan=1 colspan=1>788/825</td><td rowspan=1 colspan=1>93.9 - 96.7</td></tr></table>

a SoC reported the following possible corresponding host microorganisms for one $\mathrm { O r g ^ { + } / R e s ^ { - } }$ case: $P .$ aeruginosa.

A

Table 36: Agreement rates for oxa-48 between LRT BAL and reference methods A (culture) and B (composite comparator) for combined possible host microorganisms: Enterobacteriaceae.   

<table><tr><td rowspan=2 colspan=2>Combined Hostsoxa-48</td><td rowspan=1 colspan=5>Culture Result for Host Microorganism(s) +Isolate Sequencing/Marker Screening for oxa-48</td></tr><tr><td rowspan=1 colspan=1>Org+/Res+</td><td rowspan=1 colspan=1>Org+/Res-</td><td rowspan=1 colspan=1>Org+/No Isolate</td><td rowspan=1 colspan=1>Org-</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=4 colspan=1>LRT BALResult</td><td rowspan=1 colspan=1>Org+/Res+</td><td rowspan=1 colspan=1>1c</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>Org+/Res-</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>51</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>47</td><td rowspan=1 colspan=1>111</td></tr><tr><td rowspan=1 colspan=1>Org-</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>890</td><td rowspan=1 colspan=1>900</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>1 a</td><td rowspan=1 colspan=1>55</td><td rowspan=1 colspan=1>19</td><td rowspan=1 colspan=1>937</td><td rowspan=1 colspan=1>1,012 b</td></tr><tr><td rowspan=1 colspan=3>Combined Hosts/oxa-48</td><td rowspan=1 colspan=2>rate [%]</td><td rowspan=1 colspan=1>positivity</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=3>Agreement (Org+/Res+)</td><td rowspan=1 colspan=2>100.0</td><td rowspan=1 colspan=1>1/1</td><td rowspan=1 colspan=1>20.7 - 100.0</td></tr><tr><td rowspan=1 colspan=3>Agreement (Org+/Res-)</td><td rowspan=1 colspan=2>92.7</td><td rowspan=1 colspan=1>51/55</td><td rowspan=1 colspan=1>82.7 - 97.1</td></tr><tr><td rowspan=1 colspan=3>Agreement (Org-)</td><td rowspan=1 colspan=2>95.0</td><td rowspan=1 colspan=1>890/937</td><td rowspan=1 colspan=1>93.4 - 96.2</td></tr></table>

B

<table><tr><td rowspan=2 colspan=2>Combined Hostsoxa-48</td><td rowspan=1 colspan=4>Composite Comparator Result for Host Microorganism(s) +Mol. Reference (PCR/Seq.) for oxa-48</td></tr><tr><td rowspan=1 colspan=1>Org+/Res+</td><td rowspan=1 colspan=1>Org+/Res-</td><td rowspan=1 colspan=1>Org-</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=4 colspan=1>LRT BALResult</td><td rowspan=1 colspan=1>Org+/Res+</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>Org+/Res-</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>84</td><td rowspan=1 colspan=1>27</td><td rowspan=1 colspan=1>111</td></tr><tr><td rowspan=1 colspan=1>Org-</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>886</td><td rowspan=1 colspan=1>900</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>1 a</td><td rowspan=1 colspan=1>98</td><td rowspan=1 colspan=1>913</td><td rowspan=1 colspan=1>1,012 b</td></tr><tr><td rowspan=1 colspan=3>Combined Hosts/oxa-48</td><td rowspan=1 colspan=1>rate [%]</td><td rowspan=1 colspan=1>positivity</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=3>Agreement (Org+/Res+)</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>1/1</td><td rowspan=1 colspan=1>20.7 - 100.0</td></tr><tr><td rowspan=1 colspan=3>Agreement (Org+/Res-)</td><td rowspan=1 colspan=1>85.7</td><td rowspan=1 colspan=1>84/98</td><td rowspan=1 colspan=1>77.4 - 91.3</td></tr><tr><td rowspan=1 colspan=3>Agreement (Org-)</td><td rowspan=1 colspan=1>97.0</td><td rowspan=1 colspan=1>886/913</td><td rowspan=1 colspan=1>95.7 - 98.0</td></tr></table>

a Specimens with multiple detected possible host microorganisms by culture: 0 of 1, by the composite comparator: 0 of 1. b Four invalid LRT BAL results for oxa-48. c SoC reported the following possible corresponding host microorganisms for one $\mathrm { O r g ^ { + / R e s ^ { + } } }$ case: K. pneumoniae/P. aeruginosa.

A

Table 37: Agreement rates for tem between LRT BAL and reference methods A (culture) and B (composite comparator) for $H .$ influenzae as host microorganism.   

<table><tr><td rowspan=2 colspan=2>H. influenzaetem</td><td rowspan=1 colspan=5>Culture Result for H. influenzae +Isolate Sequencing/Marker Screening for tem</td></tr><tr><td rowspan=1 colspan=1>Org+/Res+</td><td rowspan=1 colspan=1>Org+/Res-</td><td rowspan=1 colspan=1>Org+/No Isolate</td><td rowspan=1 colspan=1>Org-</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=4 colspan=1>LRT BALResult</td><td rowspan=1 colspan=1>Org+/Res+</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>16</td></tr><tr><td rowspan=1 colspan=1>Org+/Res-</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>35</td><td rowspan=1 colspan=1>40</td></tr><tr><td rowspan=1 colspan=1>Org-</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>958</td><td rowspan=1 colspan=1>959</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>1,006</td><td rowspan=1 colspan=1>1,015 a</td></tr><tr><td rowspan=1 colspan=3>H. influenzae/tem</td><td rowspan=1 colspan=2>rate [%]</td><td rowspan=1 colspan=1>positivity</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=3>Agreement (Org+/Res+)</td><td rowspan=1 colspan=2>100.0</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>34.2 - 100.0</td></tr><tr><td rowspan=1 colspan=3>Agreement (Org+/Res-)</td><td rowspan=1 colspan=2>100.0</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>43.9 - 100.0</td></tr><tr><td rowspan=1 colspan=3>Agreement (Org-)</td><td rowspan=1 colspan=2>95.2</td><td rowspan=1 colspan=1>958/1,006</td><td rowspan=1 colspan=1>93.7 - 96.4</td></tr></table>

B

<table><tr><td rowspan=2 colspan=2>H. influenzaetem</td><td rowspan=1 colspan=4>Composite Comparator Result for H. influenzae +Mol. Reference (PCR/Seq.) for tem</td></tr><tr><td rowspan=1 colspan=1>Org+/Res+</td><td rowspan=1 colspan=1>Org+/Res-</td><td rowspan=1 colspan=1>Org-</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=4 colspan=1>LRT BALResult</td><td rowspan=1 colspan=1>Org+/Res+</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>16</td></tr><tr><td rowspan=1 colspan=1>Org+/Res-</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>26</td><td rowspan=1 colspan=1>40</td></tr><tr><td rowspan=1 colspan=1>Org-</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>957</td><td rowspan=1 colspan=1>959</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>993</td><td rowspan=1 colspan=1>1,015 a</td></tr><tr><td rowspan=1 colspan=3>H. influenzae/tem</td><td rowspan=1 colspan=1>rate [%]</td><td rowspan=1 colspan=1>positivity</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=3>Agreement (Org+/Res+)</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>61.0 - 100.0</td></tr><tr><td rowspan=1 colspan=3>Agreement (Org+/Res-)</td><td rowspan=1 colspan=1>87.5</td><td rowspan=1 colspan=1>14/16</td><td rowspan=1 colspan=1>64.0 - 96.5</td></tr><tr><td rowspan=1 colspan=3>Agreement (Org-)</td><td rowspan=1 colspan=1>96.4</td><td rowspan=1 colspan=1>957/993</td><td rowspan=1 colspan=1>95.0 - 97.4</td></tr></table>

a One LRT BAL run with an invalid result for both $H .$ . influenzae and tem.

A

Table 38: Agreement rates for mecA between LRT BAL and reference methods A (culture) and B (composite comparator) and C (phenotypic resistance as reported by AST for the culture isolate) for S. aureus as host microorganism.   

<table><tr><td rowspan=2 colspan=2>S. aureusmecA</td><td rowspan=1 colspan=5>Culture Result for S. aureus +Isolate Sequencing/Marker Screening for mecA</td></tr><tr><td rowspan=1 colspan=1>Org+/Res+</td><td rowspan=1 colspan=1>Org+/Res-</td><td rowspan=1 colspan=1>Org+/No Isolate</td><td rowspan=1 colspan=1>Org-</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=4 colspan=1>LRT BALResult</td><td rowspan=1 colspan=1>Org+/Res+</td><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>47</td></tr><tr><td rowspan=1 colspan=1>Org+/Res-</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>23</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>57</td></tr><tr><td rowspan=1 colspan=1>Org-</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>904</td><td rowspan=1 colspan=1>912</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>29</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>945</td><td rowspan=1 colspan=1>1,016</td></tr><tr><td rowspan=1 colspan=3>S. aureus/mecA</td><td rowspan=1 colspan=2>rate [%]</td><td rowspan=1 colspan=1>positivity</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=3>Agreement (Org+/Res+)</td><td rowspan=1 colspan=2>81.0</td><td rowspan=1 colspan=1>17/21</td><td rowspan=1 colspan=1>60.0 - 92.3</td></tr><tr><td rowspan=1 colspan=3>Agreement (Org+/Res-)</td><td rowspan=1 colspan=2>79.3</td><td rowspan=1 colspan=1>23/29</td><td rowspan=1 colspan=1>61.6 - 90.2</td></tr><tr><td rowspan=1 colspan=3>Agreement (Org-)</td><td rowspan=1 colspan=2>95.7</td><td rowspan=1 colspan=1>904/945</td><td rowspan=1 colspan=1>94.2 - 96.8</td></tr></table>

B

<table><tr><td rowspan=2 colspan=2>S. aureusmecA</td><td rowspan=1 colspan=4>Composite Comparator Result for S. aureus +Mol. Reference (PCR/Seq.) for mecA</td></tr><tr><td rowspan=1 colspan=1>Org+/Res+</td><td rowspan=1 colspan=1>Org+/Res-</td><td rowspan=1 colspan=1>Org-</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=4 colspan=1>LRT BALResult</td><td rowspan=1 colspan=1>Org+/Res+</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>18 a</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>47</td></tr><tr><td rowspan=1 colspan=1>Org+/Res-</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>42</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>57</td></tr><tr><td rowspan=1 colspan=1>Org-</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>896</td><td rowspan=1 colspan=1>912</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>25</td><td rowspan=1 colspan=1>71</td><td rowspan=1 colspan=1>920</td><td rowspan=1 colspan=1>1,016</td></tr><tr><td rowspan=1 colspan=3>S. aureus/mecA</td><td rowspan=1 colspan=1>rate [%]</td><td rowspan=1 colspan=1>positivity</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=3>Agreement (Org+/Res+)</td><td rowspan=1 colspan=1>80.0</td><td rowspan=1 colspan=1>20/25</td><td rowspan=1 colspan=1>60.9 - 91.1</td></tr><tr><td rowspan=1 colspan=3>Agreement (Org+/Res-)</td><td rowspan=1 colspan=1>59.2</td><td rowspan=1 colspan=1>42/71</td><td rowspan=1 colspan=1>47.5 - 69.8</td></tr><tr><td rowspan=1 colspan=3>Agreement (Org-)</td><td rowspan=1 colspan=1>97.4</td><td rowspan=1 colspan=1>896/920</td><td rowspan=1 colspan=1>96.1 - 98.2</td></tr></table>

C

<table><tr><td rowspan=2 colspan=2>S. aureusmecA</td><td rowspan=1 colspan=5>Culture Result for S. aureus +and Cefoxitin/Oxacillin AST Results</td></tr><tr><td rowspan=1 colspan=1>Org+/MRSA</td><td rowspan=1 colspan=1>Org+/MSSA</td><td rowspan=1 colspan=1>Org+/No AST</td><td rowspan=1 colspan=1>Org-</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=4 colspan=1>LRT BALResult</td><td rowspan=1 colspan=1>Org+/Res+</td><td rowspan=1 colspan=1>23 b</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>47</td></tr><tr><td rowspan=1 colspan=1>Org+/Res-</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>27</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>57</td></tr><tr><td rowspan=1 colspan=1>Org-</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>904</td><td rowspan=1 colspan=1>912</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>27</td><td rowspan=1 colspan=1>37</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>945</td><td rowspan=1 colspan=1>1,016</td></tr><tr><td rowspan=1 colspan=3>S. aureus/MRSA status</td><td rowspan=1 colspan=2>rate [%]</td><td rowspan=1 colspan=1>positivity</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=3>Agreement (Org+/Res+)</td><td rowspan=1 colspan=2>85.1</td><td rowspan=1 colspan=1>23/27</td><td rowspan=1 colspan=1>67.5 - 94.1</td></tr><tr><td rowspan=1 colspan=3>Agreement (Org+/Res-)</td><td rowspan=1 colspan=2>73.0</td><td rowspan=1 colspan=1>27/37</td><td rowspan=1 colspan=1>57.0 - 84.6</td></tr><tr><td rowspan=1 colspan=3>Agreement (Org-)</td><td rowspan=1 colspan=2>95.7</td><td rowspan=1 colspan=1>904/945</td><td rowspan=1 colspan=1>94.2 - 96.8</td></tr></table>

a 18 cases were determined as $\mathrm { O r g ^ { + } / R e s ^ { - } }$ negative by the molecular reference assay using cutoffs set to achieve an analytical sensitivit in a comparable range to LRT BAL. For 10 of these 18 cases, presence of mecA was confirmed by alternate molecular test (PCR/sequencing).

b One strain was reported by the clinical site as MRSA, although this was not supported by the provided oxacillin Kirby-Bauer zone diameter. Independent cefoxitin AST confirmed the MRSA phenotype for this strain.

# B. Archived Study

Specimen Cohort: Inclusion/Exclusion Criteria, Demographics

Lavage specimens from patients, age 18 or older, suspected of a lower respiratory infection, with at least one LRT BAL panel microorganism reported positive by standard-of-care (SoC) were eligible for the archived study. Specimens were collected at 11 US clinical sites between 2015 and 2019, complemented with few specimens collected from other sites. In contrast to the prospective study, where specimens were only from hospitalized patients (inpatients), specimens from outpatients or from patients with an unknown status were also accepted. Confirmatory testing (PCR/bi-directional sequencing using two independent PCR assays per analyte) to exclude specimens that may have degraded during storage was performed for the reported SoC results prior to the study. Only specimens with at least one confirmed SoC result were included.

The archived cohort included 197 specimens from a previous study. These specimens were supplemented with 195 specimens meeting inclusion criteria that had been collected over time to achieve a total set of 392 archived study specimens.

Demographic information for specimens included into the LRT BAL archived study is described in Table 39.

Table 39: Demographic patient data for the LRT BAL archived study.   

<table><tr><td rowspan=1 colspan=1>Included Specimens</td><td rowspan=1 colspan=1>392</td></tr><tr><td rowspan=1 colspan=2>Sex a</td></tr><tr><td rowspan=1 colspan=1>male</td><td rowspan=1 colspan=1>198</td></tr><tr><td rowspan=1 colspan=1>female</td><td rowspan=1 colspan=1>122</td></tr><tr><td rowspan=1 colspan=2>Age b</td></tr><tr><td rowspan=1 colspan=1>18 - 30 years</td><td rowspan=1 colspan=1>18</td></tr><tr><td rowspan=1 colspan=1>31 - 60 years</td><td rowspan=1 colspan=1>106</td></tr><tr><td rowspan=1 colspan=1>&gt; 60 years</td><td rowspan=1 colspan=1>190</td></tr><tr><td rowspan=1 colspan=2>Clinical Setting c</td></tr><tr><td rowspan=1 colspan=1>ICU</td><td rowspan=1 colspan=1>112</td></tr><tr><td rowspan=1 colspan=1>ward</td><td rowspan=1 colspan=1>101</td></tr><tr><td rowspan=1 colspan=1>in-patient (not specified)</td><td rowspan=1 colspan=1>69</td></tr><tr><td rowspan=1 colspan=1>out-patient</td><td rowspan=1 colspan=1>32</td></tr></table>

a Not reported: 72 patients. b Not reported: 78 patients. c Not reported: 78 patients.

# Reference Methods

For all archived specimens, LRT BAL Application microorganism results were compared to confirmed positive results from SoC microbiology culture (for ‘typical’ microorganism) or other SoC methods (for ‘atypical’ microorganisms) as reference. SoC culture results were reported either qualitatively or quantitatively. For quantitative cultures, a reporting threshold of $1 0 ^ { 3 } \mathrm { C F U / m L }$ or higher for mini-BAL specimens and of $1 0 ^ { 4 } \mathrm { C F U / m L }$ or higher for BAL specimens was applied, following recommendations by the Infectious Diseases Society of America (IDSA) and the American Thoracic Society (ATS).

Evaluation of collected isolates and AST results, as well as discrepant result resolution was performed as for the prospective study.

Microorganism Results – Comparison to confirmed SoC reference, PPA/NPA LRT BAL Application results were compared to confirmed SoC results as reference (Tables 40 and 41) to determine true positives (TP), false negatives (FN), false positives (FP) and true negatives (TN) and to calculate positive percent agreements (PPA, TP / $( \mathrm { T P } + \mathrm { F N } ) )$ and negative percent agreements (NPA, TN / $( \mathrm { T N } + \mathrm { F P } ) ,$ ), together with their two-sided $9 5 \%$ confidence intervals $9 5 \%$ CI), determined according to the Wilson score method.

Note that SoC testing for ‘atypical’ microorganisms was only performed on special request by the physician. Other specimens besides the ones collected because of a positive SoC result for any of the ‘atypical’ microorganisms, were typically not tested, and, therefore, the NPA cannot be determined.

Table 40: Archived study $( \mathbf { N } = 3 9 2$ ), comparison to confirmed SoC results, ‘typical’ microorganisms.   
510(k) Summary – LRT BAL Application K191967 Rev 3.0   

<table><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">TP</td><td colspan="1" rowspan="1">FN</td><td colspan="1" rowspan="1">FP b</td><td colspan="1" rowspan="1">TN</td><td colspan="1" rowspan="1">PPA [%](95% CI)</td><td colspan="1" rowspan="1">NPA [%](95% CI)</td></tr><tr><td colspan="1" rowspan="1">Acinetobacter spp.</td><td colspan="1" rowspan="1">18</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">371</td><td colspan="1" rowspan="1">100.0(82.4 - 100.0)</td><td colspan="1" rowspan="1">99.2(97.7 - 99.7)</td></tr><tr><td colspan="1" rowspan="1">Citrobacter freundii</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">382</td><td colspan="1" rowspan="1">100.0(56.6 - 100.0)</td><td colspan="1" rowspan="1">98.7(97.0 - 99.4)</td></tr><tr><td colspan="1" rowspan="1">Enterobacter cloacaecomplex</td><td colspan="1" rowspan="1">15</td><td colspan="1" rowspan="1">4c</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">373</td><td colspan="1" rowspan="1">78.9(56.7 - 91.5)</td><td colspan="1" rowspan="1">100.0(99.0 - 100.0)</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">46</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">17</td><td colspan="1" rowspan="1">326</td><td colspan="1" rowspan="1">93.9(83.5 - 97.9)</td><td colspan="1" rowspan="1">95.0(92.2 - 96.9)</td></tr><tr><td colspan="1" rowspan="1">Haemophilusinfluenzae</td><td colspan="1" rowspan="1">50</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">21</td><td colspan="1" rowspan="1">321</td><td colspan="1" rowspan="1">100.0(92.9 - 100.0)</td><td colspan="1" rowspan="1">93.9(90.8 - 95.9)</td></tr><tr><td colspan="1" rowspan="1">Klebsiella oxytoca</td><td colspan="1" rowspan="1">16</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">369</td><td colspan="1" rowspan="1">94.1(73.0 - 99.0)</td><td colspan="1" rowspan="1">98.4(96.6 - 99.3)</td></tr><tr><td colspan="1" rowspan="1">Klebsiellapneumoniae a</td><td colspan="1" rowspan="1">29</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">351</td><td colspan="1" rowspan="1">93.5(79.3 - 98.2)</td><td colspan="1" rowspan="1">97.2(95.0 - 98.5)</td></tr><tr><td colspan="1" rowspan="1">Klebsiellavariicola a</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">386</td><td colspan="1" rowspan="1">100.0(34.2 - 100.0)</td><td colspan="1" rowspan="1">99.0(97.4 - 99.6)</td></tr><tr><td colspan="1" rowspan="1">Moraxellacatarrhalis</td><td colspan="1" rowspan="1">21</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">9</td><td colspan="1" rowspan="1">362</td><td colspan="1" rowspan="1">100.0(84.5 - 100.0)</td><td colspan="1" rowspan="1">97.6(95.5 - 98.7)</td></tr><tr><td colspan="1" rowspan="1">Morganella morganii</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">391</td><td colspan="1" rowspan="1">100.0(20.7 - 100.0)</td><td colspan="1" rowspan="1">100.0(99.0 - 100.0)</td></tr><tr><td colspan="1" rowspan="1">Proteus spp.</td><td colspan="1" rowspan="1">15</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">7</td><td colspan="1" rowspan="1">370</td><td colspan="1" rowspan="1">100.0(79.6 - 100.0)</td><td colspan="1" rowspan="1">98.1(96.2 - 99.1)</td></tr><tr><td colspan="1" rowspan="1">Pseudomonasaeruginosa</td><td colspan="1" rowspan="1">54</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">334</td><td colspan="1" rowspan="1">96.4(87.9 - 99.0)</td><td colspan="1" rowspan="1">99.4(97.9 - 99.8)</td></tr><tr><td colspan="1" rowspan="1">Serratia marcescens</td><td colspan="1" rowspan="1">23</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">364</td><td colspan="1" rowspan="1">92.0(75.0 - 97.8)</td><td colspan="1" rowspan="1">99.2(97.6 - 99.7)</td></tr><tr><td colspan="1" rowspan="1">Staphylococcusaureus</td><td colspan="1" rowspan="1">56</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">21</td><td colspan="1" rowspan="1">313</td><td colspan="1" rowspan="1">96.6(88.3 - 99.1)</td><td colspan="1" rowspan="1">93.7(90.6 - 95.9)</td></tr><tr><td colspan="1" rowspan="1">Stenotrophomonasmaltophilia</td><td colspan="1" rowspan="1">37</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">342</td><td colspan="1" rowspan="1">92.5(80.1 - 97.4)</td><td colspan="1" rowspan="1">97.2(94.9 - 98.4)</td></tr><tr><td colspan="1" rowspan="1">Streptococcuspneumoniae</td><td colspan="1" rowspan="1">34</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">352</td><td colspan="1" rowspan="1">97.1(85.5 - 99.5)</td><td colspan="1" rowspan="1">98.6(96.8 - 99.4)</td></tr></table>

a K. variicola is typically reported by culture as $K .$ . pneumoniae. To discriminate $K .$ . variicola from $K .$ . pneumoniae, $K .$ pneumoniae strain isolates, if available, were sequenced. If no isolate was provided, sequencing was performed from specimen DNA extracts instead. Results for $K .$ . pneumoniae and $K .$ variicola performance are calculated based on the species identified by sequencing.

b Specimens with false positive LRT BAL results were analyzed with molecular assays (PCR/bi-directional sequencing) using specimen DNA extracts for presence or absence of microorganisms. Presence of microorganisms was confirmed in 3 of 3 cases for Acinetobacter spp., 4 of 5 cases for $C .$ . freundii (one confirmed case was reported as $C .$ . youngae by SoC), 15 of 17 cases for E. coli, 17 of 21 cases for $H .$ . influenzae, 6 of 6 cases for $K .$ . oxytoca, 7 of 10 cases for $K .$ . pneumoniae, 2 of 4 cases for $K .$ . variicola, 8 of 9 cases for $M .$ . catarrhalis, 6 of 7 cases for Proteus spp., 2 of 2 cases for $P .$ aeruginosa, 1 of 3 cases for $S _ { \cdot }$ . marcescens, 18 of 21 cases for S. aureus, 10 of 10 cases for S. maltophilia, and 4 of 5 cases for $S .$ . pneumoniae. For cases that were not confirmed for H. influenzae, sequencing results identified $H .$ . haemolyticus $( 2 x )$ , A. aphropilus (2x). For all other cases, PCRs did not amplify sufficient amounts for sequencing.

c Three of the four specimens that were FN for $E$ . cloacae complex contained multiple host microorganisms identified by culture, and/or LRT BAL. For one specimen, Klebsiella oxytoca was additionally reported by both LRT BAL and culture. For the two other specimens, Proteus spp. or $E$ . coli, respectively, were additionally reported by LRT BAL only.

Table 41: Archived study $( \mathbf { N } = 3 9 2$ ), comparison to confirmed SoC results, ‘atypical’ microorganisms.   
a Specimens with additional positive (‘add. pos.’) LRT BAL results were analyzed with molecular assays (PCR/bi-directional sequencing) using specimen DNA extracts for presence or absence of microorganisms: presence was confirmed in 2 of 3 cases for M. pneumoniae, and 10 of 13 cases for $P .$ jirovecii. For all other cases, PCRs did not amplify sufficient amounts for sequencing.   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>TP</td><td rowspan=1 colspan=1>FN</td><td rowspan=1 colspan=1>add.pos. a</td><td rowspan=1 colspan=1>add.neg.</td><td rowspan=1 colspan=1>PPA [%](95% CI)</td></tr><tr><td rowspan=1 colspan=1>Chlamydia pneumoniae</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>392</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=1>Legionella pneumophila</td><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>373</td><td rowspan=1 colspan=1>89.5(68.6 - 97.1)</td></tr><tr><td rowspan=1 colspan=1>Mycoplasma pneumoniae</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>383</td><td rowspan=1 colspan=1>83.3(43.7 - 97.0)</td></tr><tr><td rowspan=1 colspan=1>Pneumocystis jirovecii</td><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>360</td><td rowspan=1 colspan=1>84.2(62.4 - 94.5)</td></tr></table>

For all archived specimens, true positive and false negative LRT BAL results were stratified by the semi-quantitative result reported by qualitative SoC culture (categories: rare, few, moderate, numerous) or by quantitative SoC culture (categories: $1 0 3 \textrm { -- } 1 0 4 , 1 0 4 \textrm { -- } 1 0 5 , 1 0 5 \textrm { -- } 1 0 6 , > 1 0 6 .$ in $\mathrm { C F U / m L }$ ) as shown in Table 42.

<table><tr><td rowspan=2 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>itative</td><td rowspan=1 colspan=3>Qualitative Culture Result a</td></tr><tr><td rowspan=1 colspan=1>Category</td><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>TP</td><td rowspan=1 colspan=1>FN</td><td rowspan=1 colspan=1>PPA [%]</td></tr><tr><td rowspan=4 colspan=1>Acinetobacter spp.</td><td rowspan=1 colspan=1>rare</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=1>few</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0</td></tr><tr><td rowspan=1 colspan=1>moderate</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0</td></tr><tr><td rowspan=1 colspan=1>numerous</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0</td></tr><tr><td rowspan=4 colspan=1>Citrobacter freundii</td><td rowspan=1 colspan=1>rare</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=1>few</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=1>moderate</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0</td></tr><tr><td rowspan=1 colspan=1>numerous</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=4 colspan=1>Enterobactercloacae complex</td><td rowspan=1 colspan=1>rare</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=1>few</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0</td></tr><tr><td rowspan=1 colspan=1>moderate</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>66.7</td></tr><tr><td rowspan=1 colspan=1>numerous</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0</td></tr><tr><td rowspan=4 colspan=1>Escherichia coli</td><td rowspan=1 colspan=1>rare</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>85.7</td></tr><tr><td rowspan=1 colspan=1>few</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0</td></tr><tr><td rowspan=1 colspan=1>moderate</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0</td></tr><tr><td rowspan=1 colspan=1>numerous</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0</td></tr><tr><td rowspan=4 colspan=1>Haemophilusinfluenzae</td><td rowspan=1 colspan=1>rare</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0</td></tr><tr><td rowspan=1 colspan=1>few</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0</td></tr><tr><td rowspan=1 colspan=1>moderate</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0</td></tr><tr><td rowspan=1 colspan=1>numerous</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0</td></tr><tr><td rowspan=4 colspan=1>Klebsiella oxytoca</td><td rowspan=1 colspan=1>rare</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0</td></tr><tr><td rowspan=1 colspan=1>few</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>80.0</td></tr><tr><td rowspan=1 colspan=1>moderate</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=1>numerous</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0</td></tr><tr><td rowspan=4 colspan=1>Klebsiellapneumoniae</td><td rowspan=1 colspan=1>rare</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.0</td></tr><tr><td rowspan=1 colspan=1>few</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0</td></tr><tr><td rowspan=1 colspan=1>moderate</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>88.9</td></tr><tr><td rowspan=1 colspan=1>numerous</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0</td></tr><tr><td rowspan=4 colspan=1>Klebsiella variicola</td><td rowspan=1 colspan=1>rare</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=1>few</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0</td></tr><tr><td rowspan=1 colspan=1>moderate</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0</td></tr><tr><td rowspan=1 colspan=1>numerous</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=4 colspan=1>Moraxellacatarrhalis</td><td rowspan=1 colspan=1>rare</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=1>few</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0</td></tr><tr><td rowspan=1 colspan=1>moderate</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0</td></tr><tr><td rowspan=1 colspan=1>numerous</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0</td></tr><tr><td rowspan=4 colspan=1>Morganellamorganii</td><td rowspan=1 colspan=1>rare</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=1>few</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=1>moderate</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=1>numerous</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=4 colspan=1>Proteus spp.</td><td rowspan=1 colspan=1>rare</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0</td></tr><tr><td rowspan=1 colspan=1>few</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0</td></tr><tr><td rowspan=1 colspan=1>moderate</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0</td></tr><tr><td rowspan=1 colspan=1>numerous</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0</td></tr><tr><td rowspan=4 colspan=1>Pseudomonasaeruginosa</td><td rowspan=1 colspan=1>rare</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>66.7</td></tr><tr><td rowspan=1 colspan=1>few</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0</td></tr><tr><td rowspan=1 colspan=1>moderate</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0</td></tr><tr><td rowspan=1 colspan=1>numerous</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0</td></tr><tr><td rowspan=1 colspan=1>Serratia marcescens</td><td rowspan=1 colspan=1>rare</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0</td></tr></table>

Table 42: Archived study, comparison to confirmed SoC results, stratification by qualitative or quantitative SoC culture result.   

<table><tr><td rowspan=1 colspan=4>Quantitative Culture Result a</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Category</td><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>TP</td><td rowspan=1 colspan=1>FN</td><td rowspan=1 colspan=1>PPA [%]</td></tr><tr><td rowspan=1 colspan=1>10³3-&lt;104</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0</td></tr><tr><td rowspan=1 colspan=1>104-&lt;10^5}$</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0</td></tr><tr><td rowspan=1 colspan=1>$105- &lt;16</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0</td></tr><tr><td rowspan=1 colspan=1>&gt;106</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=1>10^{3- &lt;104</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=1>104-&lt;10^}$</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0</td></tr><tr><td rowspan=1 colspan=1>$105- &lt;106</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0</td></tr><tr><td rowspan=1 colspan=1>&gt; 106</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=1>103-&lt;104</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.0</td></tr><tr><td rowspan=1 colspan=1>104-&lt;10^s}$</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>50.0</td></tr><tr><td rowspan=1 colspan=1>$105- &lt;106$</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0</td></tr><tr><td rowspan=1 colspan=1>&gt;106</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=1>103- &lt;104</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=1>$104-&lt;10\}$</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0</td></tr><tr><td rowspan=1 colspan=1>$105- 16$</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0</td></tr><tr><td rowspan=1 colspan=1>&gt;106</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=1>103 - &lt;104</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=1>104-&lt;10^5}$</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0</td></tr><tr><td rowspan=1 colspan=1>$10^-&lt;106$</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0</td></tr><tr><td rowspan=1 colspan=1>&gt;106</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0</td></tr><tr><td rowspan=1 colspan=1>103- &lt;104</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0</td></tr><tr><td rowspan=1 colspan=1>104-&lt;10^}$</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0</td></tr><tr><td rowspan=1 colspan=1>$10^{-&lt;106$</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0</td></tr><tr><td rowspan=1 colspan=1>&gt;106</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=1>103- &lt;104</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=1>104-&lt;10s}$</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0</td></tr><tr><td rowspan=1 colspan=1>$10^{}-6$</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0</td></tr><tr><td rowspan=1 colspan=1>≥ 106</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=1>10^{3-= &lt;104</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=1>104-&lt;105$</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=1>$10^-&lt;1066$</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=1>&gt;106</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=1>10^3- &lt;104</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0</td></tr><tr><td rowspan=1 colspan=1>104-&lt;10^}$</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0</td></tr><tr><td rowspan=1 colspan=1>$106$</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0</td></tr><tr><td rowspan=1 colspan=1>&gt;106</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0</td></tr><tr><td rowspan=1 colspan=1>103-&lt;104</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0</td></tr><tr><td rowspan=1 colspan=1>104-&lt;10^\}$</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=1>$10s- &lt;16$</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=1>&gt;106</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=1>10^{3- &lt;104</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=1>104 - &lt;10^}$</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0</td></tr><tr><td rowspan=1 colspan=1>$10^{} &lt; 6$</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0</td></tr><tr><td rowspan=1 colspan=1>&gt; 106</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=1>103-&lt;104</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0</td></tr><tr><td rowspan=1 colspan=1>$104-&lt;10}$</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0</td></tr><tr><td rowspan=1 colspan=1>105-&lt;106</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0</td></tr><tr><td rowspan=1 colspan=1>&gt;106</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=1>103&lt;104</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0</td></tr></table>

<table><tr><td rowspan=2 colspan=1></td><td rowspan=1 colspan=5>Qualitative Culture Result a</td></tr><tr><td rowspan=1 colspan=1>Category</td><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>TP</td><td rowspan=1 colspan=1>FN</td><td rowspan=1 colspan=1>PPA [%]</td></tr><tr><td rowspan=3 colspan=1></td><td rowspan=1 colspan=1>few</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>75.0</td></tr><tr><td rowspan=1 colspan=1>moderate</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0</td></tr><tr><td rowspan=1 colspan=1>numerous</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0</td></tr><tr><td rowspan=4 colspan=1>Staphylococcusaureus</td><td rowspan=1 colspan=1>rare</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>83.3</td></tr><tr><td rowspan=1 colspan=1>few</td><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>92.9</td></tr><tr><td rowspan=1 colspan=1>moderate</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0</td></tr><tr><td rowspan=1 colspan=1>numerous</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0</td></tr><tr><td rowspan=4 colspan=1>Stenotrophomonasmaltophilia</td><td rowspan=1 colspan=1>rare</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>66.7</td></tr><tr><td rowspan=1 colspan=1>few</td><td rowspan=1 colspan=1>18</td><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>88.9</td></tr><tr><td rowspan=1 colspan=1>moderate</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0</td></tr><tr><td rowspan=1 colspan=1>numerous</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0</td></tr><tr><td rowspan=4 colspan=1>Streptococcuspneumoniae</td><td rowspan=1 colspan=1>rare</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>50.0</td></tr><tr><td rowspan=1 colspan=1>few</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0</td></tr><tr><td rowspan=1 colspan=1>moderate</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0</td></tr><tr><td rowspan=1 colspan=1>numerous</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0</td></tr></table>

a One Acinetobacter spp., $4 E .$ . cloacae complex, 7. E. coli, $4 H .$ influenzae, $4 K .$ . oxytoca, $5 K .$ pneumoniae, $1 M .$ . catarrhalis, 2 Proteus spp., $1 1 P .$ aeruginosa, $4 S .$ . marcescens, $9 \ : S .$ . aureus, $3 S .$ maltophilia and $1 S .$ . pneumoniae case were reported as positive by SoC culture without qualitative or quantitative results, and were therefore not included in this table.

<table><tr><td rowspan=1 colspan=5>Quantitative Culture Result a</td></tr><tr><td rowspan=1 colspan=1>Category</td><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>TP</td><td rowspan=1 colspan=1>FN</td><td rowspan=1 colspan=1>PPA [%]</td></tr><tr><td rowspan=1 colspan=1>104 - &lt;10^$</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0</td></tr><tr><td rowspan=1 colspan=1>105 - &lt; 106</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0</td></tr><tr><td rowspan=1 colspan=1>&gt; 106</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=1>103 - &lt;104</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=1>104- &lt;10^5$</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0</td></tr><tr><td rowspan=1 colspan=1>$10^5- &lt;106</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0</td></tr><tr><td rowspan=1 colspan=1>&gt; 106</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0</td></tr><tr><td rowspan=1 colspan=1>103³ - &lt; 104</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0</td></tr><tr><td rowspan=1 colspan=1>104 - &lt; 105</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0</td></tr><tr><td rowspan=1 colspan=1>105 - &lt; 106</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0</td></tr><tr><td rowspan=1 colspan=1>&gt; 106</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=1>103 - &lt;104</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=1>$104 - &lt;10}$</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0</td></tr><tr><td rowspan=1 colspan=1>105 - &lt; 106</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0</td></tr><tr><td rowspan=1 colspan=1>&gt; 106</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0</td></tr></table>

# Antibiotic Resistance Markers

Antibiotic resistance markers reported by LRT BAL for the archived study are listed in Table 43. Note that antibiotic resistance markers are only reported and displayed on the Unyvero results screen, if a corresponding host microorganism was simultaneously detected. Results for ctx-M, kpc, ndm, and vim are reported for LRT BAL positive specimens for Enterobacteriaceae, Acinetobacter spp., and $P .$ aeruginosa $( \mathrm { N } = 2 1 2 )$ ; results for oxa-48 are reported for LRT BAL positive specimens for Enterobacteriaceae $( \mathrm { N } = 1 6 6 )$ ); results for oxa-23, oxa-24, and oxa-58 are reported for LRT BAL positive specimens for Acinetobacter spp. $( \ N = 2 1$ ); results for tem are reported for LRT BAL positive specimens for $H .$ . influenzae $\mathrm { ( N } = 7 1 $ ); and results for mecA are reported for LRT BAL positive specimens for $S .$ aureus $( \mathrm { N } = 7 7$ ). Only specimens with antibiotic resistance markers reported positive by LRT BAL were tested for such markers by molecular reference assays (PCR/sequencing).

Table 43: Archived study $\mathbf { N } = 3 9 2$ ), listing of antibiotic resistance marker results as reported by LRT BAL.   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Positive a</td><td rowspan=1 colspan=1>Negative</td></tr><tr><td rowspan=1 colspan=1>ctx-M(N = 212)</td><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>198</td></tr><tr><td rowspan=1 colspan=1>kpc(N = 212)</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>210</td></tr><tr><td rowspan=1 colspan=1>mecA(N = 77)</td><td rowspan=1 colspan=1>44</td><td rowspan=1 colspan=1>33</td></tr><tr><td rowspan=1 colspan=1>ndm(N = 212)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>212</td></tr><tr><td rowspan=1 colspan=1>oxa-23(N= 21)</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>17</td></tr><tr><td rowspan=1 colspan=1>oxa-24(N =21)</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>18</td></tr><tr><td rowspan=1 colspan=1>oxa-48(N = 166)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>166</td></tr><tr><td rowspan=1 colspan=1>oxa-58(N = 21)</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>20</td></tr><tr><td rowspan=1 colspan=1>tem(N = 71)</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>47</td></tr><tr><td rowspan=1 colspan=1>vim(N = 212)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>212</td></tr></table>

a Specimens with false positive LRT BAL results were analyzed with molecular assays (PCR/bi-directional sequencing) using specimen DNA extracts for presence or absence of microorganisms: presence of microorganisms was confirmed in 14 of 14 cases for ctx-M, 2 of 2 cases for kpc, 34 of 44 cases for mecA, 4 of 4 cases for oxa-23, 3 of 3 cases for oxa-24, 1 of 1 case for oxa-58, and 22 of 24 cases for tem. For all other cases, PCRs did not amplify sufficient amounts for sequencing.

# Correlation of Detected Antibiotic Resistance Markers to Strain Genotypes and Phenotypes

As described in the previous sections, isolates that had been collected for most of the prospective and part of archived study specimens were screened for presence or absence of antibiotic resistance markers of the LRT BAL panel. Furthermore, corresponding AST results were collected to predict resistance phenotypes of such isolates using one or more of the drug assays listed in Table 44. AST results were reported as MIC values or zone diameters (for Kirby-Bauer tests). Strain phenotypes across all test sites were standardized according to breakpoints listed in CLSI guidance M100S (Performance Standards for Antimicrobial Susceptibility Testing, 26th Edition 2016). ‘Intermediate’ calls were regarded as ‘resistant’ calls for this study. For the analysis, any strain was regarded as ‘resistant’ if at least one of the corresponding drugs resulted in an ‘intermediate’ or ‘resistant’ call. Any strain was regarded as ‘susceptible’ if a ‘sensitive’ call was obtained for all applicable tested drugs.

Table 44: AST assays used for evaluation of antibiotic resistance markers detected by LRT BAL.   

<table><tr><td rowspan=1 colspan=1>Antibiotic Resistance Marker(s)</td><td rowspan=1 colspan=1>Associated Resistance</td><td rowspan=1 colspan=1>AST Assay</td></tr><tr><td rowspan=1 colspan=1>tem</td><td rowspan=1 colspan=1>penicillins</td><td rowspan=1 colspan=1>H. influenzae:ampicillin, cefinase</td></tr><tr><td rowspan=1 colspan=1>ctx-M</td><td rowspan=1 colspan=1>3rdgeneration cephalosporins,cefepime</td><td rowspan=1 colspan=1>Enterobacteriaceae:cefotaxime, ceftazidime, ceftriaxoneAcinetobacter spp.:cefotaxime, ceftazidime, ceftriaxone, cefepimeP. aeruginosa:ceftazidime, cefepime</td></tr><tr><td rowspan=1 colspan=1>kpc, ndm, oxa-48, vim</td><td rowspan=1 colspan=1>carbapenems</td><td rowspan=1 colspan=1>Enterobacteriaceae:meropenem, ertapenem, imipenem</td></tr><tr><td rowspan=1 colspan=1>kpc, ndm, oxa-23, oxa-24, oxa-58,vim</td><td rowspan=1 colspan=1>carbapenems</td><td rowspan=1 colspan=1>Acinetobacter spp.meropenem, imipenem</td></tr><tr><td rowspan=1 colspan=1>kpc, ndm, vim</td><td rowspan=1 colspan=1>carbapenems</td><td rowspan=1 colspan=1>P. aeruginosa:meropenem, imipenem</td></tr><tr><td rowspan=1 colspan=1>mecA</td><td rowspan=1 colspan=1>oxacillin/cefoxitin</td><td rowspan=1 colspan=1>S. aureus:oxacillin, cefoxitin</td></tr></table>

Antibiotic resistance markers reported by LRT BAL were correlated to available isolates for:

1. Genotypic agreement of reported antibiotic resistance markers to presence of such markers in cultured isolates for a specific specimen.   
2. Phenotypic agreement of reported antibiotic resistance markers with associated phenotypic culture results to provided isolates as determined by antimicrobial susceptibility tests (AST).

Note that antibiotic resistance marker results are only reported by the LRT BAL Application if a corresponding host microorganism is simultaneously reported. Otherwise, such results are masked on the Unyvero results screen. Antibiotic resistance marker results reported together with a corresponding host microorganism may not be linked to this organism, but may have originated from other on-panel or off-panel microorganisms, e.g., mecA may not only originate from S. aureus, but also originate from coagulase-negative Staphylococci present in respiratory flora; tem may not only originate from H. influenzae, but also from other on-panel or off-panel Enterobacteriaceae. If multiple corresponding microorganisms are simultaneously reported for a certain antibiotic resistance marker, one or more than one of these microorganisms could be the host of such marker.

Genotypic and phenotypic agreements were evaluated for all relevant resistance phenotypes including all applicable specimens from the prospective and archived study combined (Tables 45 to 49).

Table 45: Correlation of positive mecA results reported by LRT BAL for the prospective (‘p’) or archived (‘a’) study to available S. aureus isolates (genotypic agreement, ‘yes’: presence of mecA confirmed by sequencing of isolate, ‘no’: mecA not confirmed, ‘not prov.’: no isolate provided) and to AST results indicating oxacillin/cefoxitin resistance for isolates with confirmed mecA (phenotypic agreement; R: resistant phenotype, S: sensitive phenotype, NA: AST results not reported).

<table><tr><td rowspan=1 colspan=8>Genotypic Agreement for Specimens with Available Isolate: 21/25, 84.0% (95% CI: 65.3 - 93.6)</td></tr><tr><td rowspan=1 colspan=8>Phenotypic Agreement for Isolates with Confirmed Marker: 21/21, 100.0% (95% CI: 84.5 - 100.0)</td></tr><tr><td rowspan=1 colspan=3># Cases</td><td rowspan=2 colspan=1>LRT BAL Result(relevant hosts only)</td><td rowspan=2 colspan=1>SoC Result(relevant hosts only)</td><td rowspan=2 colspan=1>GenotypicAgreement(for availableisolates)</td><td rowspan=2 colspan=1>PhenotypicAgreement(for isolateswith conf.mecA)</td><td rowspan=2 colspan=1>Comment</td></tr><tr><td rowspan=1 colspan=1>total</td><td rowspan=1 colspan=1>p</td><td rowspan=1 colspan=1>a</td></tr><tr><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>S. aureus</td><td rowspan=1 colspan=1>S. aureus</td><td rowspan=1 colspan=1>yes</td><td rowspan=1 colspan=1>Ra</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>S. aureus</td><td rowspan=1 colspan=1>S. aureus</td><td rowspan=1 colspan=1>no</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr></table>

a One prospective strain was reported by the clinical site as MRSA, although this was not supported by the provided oxacillin KirbyBauer zone diameter. An independent cefoxitin AST confirmed the MRSA phenotype for this strain. For another archived strain, no AST results were reported by the clinical site. The MRSA phenotype was determined by an independent cefoxitin AST instead.

# Penicillin Resistance for H. influenzae (tem)

Table 46: Correlation of positive tem results reported by LRT BAL for the prospective $( \mathbf { \hat { p } } )$ or archived $( { \bf \ ' a \prime } )$ study to available H. influenzae isolates (genotypic agreement, ‘yes’: presence of tem confirmed by sequencing of isolate, ‘no’: tem not confirmed, ‘not prov.’: no isolate provided) and to AST results indicating penicillin resistance for isolates with confirmed tem (phenotypic agreement; R: resistant phenotype, S: sensitive phenotype, NA: AST results not reported).

<table><tr><td rowspan=1 colspan=8>Genotypic Agreement for Specimens with Available Isolate: 4/4, 100.0% (95% CI: 51.0 - 100.0)</td></tr><tr><td rowspan=1 colspan=8>Phenotypic Agreement for Isolates with Confirmed Marker: 4/4, 100.0% (95% CI: 51.0 - 100.0)</td></tr><tr><td rowspan=1 colspan=3># Cases</td><td rowspan=2 colspan=1>LRT BAL Result(relevant hosts only)</td><td rowspan=2 colspan=1>SoC Result(relevant hosts only)</td><td rowspan=2 colspan=1>GenotypicAgreement(for availableisolates)</td><td rowspan=2 colspan=1>PhenotypicAgreement(for isolateswith conf. tem)</td><td rowspan=2 colspan=1>Comment</td></tr><tr><td rowspan=1 colspan=1>total</td><td rowspan=1 colspan=1>p</td><td rowspan=1 colspan=1>a</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>H. influenzae</td><td rowspan=1 colspan=1>H. influenzae</td><td rowspan=1 colspan=1>yes</td><td rowspan=1 colspan=1>R</td><td rowspan=1 colspan=1>-</td></tr></table>

Third Generation Cephalosporin Resistance for Enterobacteriaceae, Acinetobacter spp., and P. aeruginosa (ctx-M)

Table 47: Correlation of positive ctx-M results reported by LRT BAL for the prospective $( \mathbf { \hat { p } } )$ or archived $( { \bf \ ' a \prime } )$ study to available isolates of Enterobacteriaceae, Acinetobacter spp., or $\boldsymbol { P } .$ aeruginosa (genotypic agreement, ‘yes’: presence of ctx-M confirmed by sequencing of isolate, ‘no’: ctx-M not confirmed, ‘not prov.’: no isolate provided) and to AST results indicating third generation cephalosporin resistance for isolates with confirmed ctx-M (phenotypic agreement; R: resistant phenotype, S: sensitive phenotype, NA: AST results not reported).

<table><tr><td rowspan=1 colspan=7>Genotypic Agreement for Specimens with at Least One Available Isolate: 5/8, 62.5% (95% CI: 30.6 - 86.3)</td></tr><tr><td rowspan=1 colspan=7>Phenotypic Agreement for Isolates with Confirmed Marker: 4/4, 100.0% (95% CI: 51.0 - 100.0)</td></tr><tr><td rowspan=1 colspan=1>Study</td><td rowspan=1 colspan=1>LRT BAL Result(relevant hosts only)</td><td rowspan=1 colspan=1>SoC Result(relevant hosts only)</td><td rowspan=1 colspan=1>GenotypicAgrrement(for availableisolates)</td><td rowspan=1 colspan=2>PhenotypicAgreement(for isolateswith conf. ctx-M)</td><td rowspan=1 colspan=1>Comment</td></tr><tr><td rowspan=4 colspan=1>p</td><td rowspan=1 colspan=1>Acinetobacter spp.</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2></td><td rowspan=4 colspan=1></td></tr><tr><td rowspan=1 colspan=1>E. coli</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=2></td></tr><tr><td rowspan=2 colspan=1>K. pneumoniaeP. aeruginosa</td><td rowspan=2 colspan=1>K. pneumoniaeP. aeruginosa</td><td rowspan=1 colspan=1>yes</td><td rowspan=1 colspan=2>R</td></tr><tr><td rowspan=1 colspan=1>no</td><td rowspan=1 colspan=2>-</td></tr><tr><td rowspan=2 colspan=1>p</td><td rowspan=2 colspan=1>K. pneumoniaeP. aeruginosa</td><td rowspan=1 colspan=1>K. pneumoniae</td><td rowspan=1 colspan=1>yes</td><td rowspan=1 colspan=2>R</td><td rowspan=2 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=2>-</td></tr><tr><td rowspan=2 colspan=1>b</td><td rowspan=2 colspan=1>E. coliP. aeruginosa</td><td rowspan=1 colspan=1>E. coli</td><td rowspan=1 colspan=1>yes</td><td rowspan=1 colspan=2>R</td><td rowspan=2 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=2>-</td></tr><tr><td rowspan=2 colspan=1>p</td><td rowspan=1 colspan=1>E. coli</td><td rowspan=1 colspan=1>E. coli</td><td rowspan=1 colspan=1>no</td><td rowspan=1 colspan=2>-</td><td rowspan=2 colspan=1></td></tr><tr><td rowspan=1 colspan=1>[off-panel]</td><td rowspan=1 colspan=1>Providencia stuartii</td><td rowspan=1 colspan=1>yes</td><td rowspan=1 colspan=2>R</td></tr><tr><td rowspan=6 colspan=1>p</td><td rowspan=1 colspan=1>Acinetobacter spp.</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=2>-</td><td rowspan=6 colspan=1></td></tr><tr><td rowspan=1 colspan=1>E. coli</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2></td></tr><tr><td rowspan=1 colspan=1>K. pneumoniae</td><td rowspan=1 colspan=1>K. pneumoniae</td><td rowspan=1 colspan=1>(not. prov.)</td><td rowspan=1 colspan=2></td></tr><tr><td rowspan=1 colspan=1>Proteus spp.</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2></td></tr><tr><td rowspan=1 colspan=1>P. aeruginosa</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=2>-</td></tr><tr><td rowspan=1 colspan=1>[off-panel]</td><td rowspan=1 colspan=1>Providencia stuartii</td><td rowspan=1 colspan=1>yes</td><td rowspan=1 colspan=2>NA</td></tr><tr><td rowspan=4 colspan=1>p</td><td rowspan=4 colspan=1>E. coliK. pneumoniaeP. aeruginosaS. marcescens</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=2>-</td><td rowspan=4 colspan=1></td></tr><tr><td rowspan=1 colspan=1>K. pneumoniae</td><td rowspan=1 colspan=1>(not. prov.)</td><td rowspan=2 colspan=2></td></tr><tr><td rowspan=1 colspan=1>P. aeruginosa</td><td rowspan=1 colspan=1>no</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>S. marcescens</td><td rowspan=1 colspan=1>(not. prov.)</td><td rowspan=1 colspan=2></td></tr><tr><td rowspan=2 colspan=1>p</td><td rowspan=2 colspan=1>E. coliP. aeruginosa</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2></td><td rowspan=2 colspan=1></td></tr><tr><td rowspan=1 colspan=1>P. aeruginosa</td><td rowspan=1 colspan=1>no</td><td rowspan=1 colspan=2></td></tr><tr><td rowspan=4 colspan=1>p</td><td rowspan=1 colspan=1>E. coli</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=2></td><td rowspan=4 colspan=1></td></tr><tr><td rowspan=2 colspan=1>M. morganiiProteus spp.</td><td rowspan=2 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=2 colspan=2></td></tr><tr><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>P. aeruginosa</td><td rowspan=1 colspan=1>P. aeruginosa</td><td rowspan=1 colspan=1>no</td><td rowspan=1 colspan=2></td></tr></table>

Carbapenem Resistance for Enterobacteriaceae, Acinetobacter spp., and P. aeruginosa (kpc, ndm, oxa-48, vim)

Table 48: Correlation of positive results for kpc, ndm, vim reported by LRT BAL for the prospective (‘p’) or archived $( { \bf \ ' a \prime } )$ study to available isolates of Enterobacteriaceae, Acinetobacter spp., or P. aeruginosa, or positive results for oxa-48 to available Enterobacteriaceae isolates (genotypic agreement, ‘yes’: presence of any of these markers confirmed by sequencing, ‘no’: presence not confirmed, ‘not prov.’: no isolate provided) and to AST results indicating carbapenem resistance for isolates with confirmed marker (phenotypic agreement; R: resistant phenotype, S: sensitive phenotype, NA: AST results not reported).

<table><tr><td rowspan=1 colspan=7>Genotypic Agreement for Specimens with at Least One Available Isolate: 3/5, 60.0% (95% CI: 23.1 - 88.2)</td></tr><tr><td rowspan=1 colspan=7>Phenotypic Agreement for Isolates with Confirmed Marker: 2/2, 100.0% (95% CI: 34.2 - 100.0)</td></tr><tr><td rowspan=1 colspan=1>Study</td><td rowspan=1 colspan=1>Marker</td><td rowspan=1 colspan=1>LRT BAL Result(relevant hosts only)</td><td rowspan=1 colspan=1>SoC Result(relevant hosts only)</td><td rowspan=1 colspan=1>GenotypicAgreement(for availableisolates)</td><td rowspan=1 colspan=1>PhenotypicAgreement(for isolateswith conf. marker)</td><td rowspan=1 colspan=1>Comment</td></tr><tr><td rowspan=3 colspan=1>p</td><td rowspan=3 colspan=1>kpc</td><td rowspan=1 colspan=1>E. cloacae complex</td><td rowspan=1 colspan=1>E. cloacae complex</td><td rowspan=1 colspan=1>yes</td><td rowspan=1 colspan=1>R</td><td rowspan=3 colspan=1></td></tr><tr><td rowspan=2 colspan=1>K. pneumoniaeP. aeruginosa</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=4 colspan=1>p</td><td rowspan=4 colspan=1>ndm,oxa-48</td><td rowspan=4 colspan=1>Acinetobacter spp.E. coliK. pneumoniaeP. aeruginosa</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=4 colspan=1></td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>K. pneumoniae</td><td rowspan=1 colspan=1>yes (both)</td><td rowspan=1 colspan=1>R</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>P. aeruginosa</td><td rowspan=1 colspan=1>no</td><td rowspan=1 colspan=1>,</td></tr><tr><td rowspan=3 colspan=1>p</td><td rowspan=3 colspan=1>kpc</td><td rowspan=2 colspan=1>Acinetobacter spp.E. coli</td><td rowspan=1 colspan=1>Acinetobacter spp.</td><td rowspan=1 colspan=1>no</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>cYanmaGlowb</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>K. pneumoniae</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>yes</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>subclinical isolate a</td></tr><tr><td rowspan=2 colspan=1>b</td><td rowspan=2 colspan=1>kpc</td><td rowspan=2 colspan=1>K. pneumoniaeP. aeruginosa</td><td rowspan=1 colspan=1>K. pneumoniae</td><td rowspan=1 colspan=1>(not prov.)</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>AST: resistant</td></tr><tr><td rowspan=1 colspan=1>P. aeruginosa</td><td rowspan=1 colspan=1>no</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=2 colspan=1>p</td><td rowspan=2 colspan=1>vim</td><td rowspan=2 colspan=1>Proteus spp.P. aeruginosa</td><td rowspan=1 colspan=1>-Y</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=2 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>P. aeruginosa</td><td rowspan=1 colspan=1>no</td><td rowspan=1 colspan=1>-</td></tr></table>

a For K. pneumoniae, SoC reported a concentration $< 1 0 ^ { 3 }$ CFU/mL (considered a negative, ‘subclinical’ result below the reporting threshold for mini-BAL specimens).

Table 49: Correlation of positive results for oxa-23, oxa-24, oxa-58, reported by LRT BAL for the prospective $( \mathbf { \hat { p } } )$ or archived (‘a’) study to available Acinetobacter spp. isolates (genotypic agreement, ‘yes’: presence of any of these markers confirmed by sequencing, ‘no’: presence not confirmed, ‘not prov.’: no isolate provided) and to AST results indicating carbapenem resistance for isolates with confirmed marker (phenotypic agreement; R: resistant phenotype, S: sensitive phenotype, NA: AST results not reported).   

<table><tr><td rowspan=1 colspan=7>Genotypic Agreement for Specimens with at Least One Available Isolate: 6/6, 100.0% (95% CI: 61.0 - 100.0)</td></tr><tr><td rowspan=1 colspan=7>Phenotypic Agreement for Isolates with Confirmed Marker: 6/6, 100.0% (95% CI: 61.0 - 100.0)</td></tr><tr><td rowspan=1 colspan=1>Study</td><td rowspan=1 colspan=1>Marker</td><td rowspan=1 colspan=1>LRT BAL Result(relevant hosts only)</td><td rowspan=1 colspan=1>SoC Result(relevant hosts only)</td><td rowspan=1 colspan=1>GenotypicAgreement(for availableisolates)</td><td rowspan=1 colspan=1>PhenotypicAgreement(isolateswith conf. marker)</td><td rowspan=1 colspan=1>Comment</td></tr><tr><td rowspan=1 colspan=1>p</td><td rowspan=1 colspan=1>oxa-23</td><td rowspan=1 colspan=1>Acinetobacter spp.</td><td rowspan=1 colspan=1>Acinetobacter spp.</td><td rowspan=1 colspan=1>yes</td><td rowspan=1 colspan=1>R</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>a</td><td rowspan=1 colspan=1>oxa-23</td><td rowspan=1 colspan=1>Acinetobacter spp.</td><td rowspan=1 colspan=1>Acinetobacter spp.</td><td rowspan=1 colspan=1>yes</td><td rowspan=1 colspan=1>R</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>p</td><td rowspan=1 colspan=1>oxa-24</td><td rowspan=1 colspan=1>Acinetobacter spp.</td><td rowspan=1 colspan=1>Acinetobacter spp.</td><td rowspan=1 colspan=1>yes</td><td rowspan=1 colspan=1>R</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>p</td><td rowspan=1 colspan=1>oxa-24</td><td rowspan=1 colspan=1>Acinetobacter spp.</td><td rowspan=1 colspan=1>Acinetobacter spp.</td><td rowspan=1 colspan=1>yes</td><td rowspan=1 colspan=1>R a</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>a</td><td rowspan=1 colspan=1>oxa-23</td><td rowspan=1 colspan=1>Acinetobacter spp.</td><td rowspan=1 colspan=1>Acinetobacter spp.</td><td rowspan=1 colspan=1>yes</td><td rowspan=1 colspan=1>R</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>p</td><td rowspan=1 colspan=1>oxa-24</td><td rowspan=1 colspan=1>Acinetobacter spp.</td><td rowspan=1 colspan=1>Acinetobacter spp.</td><td rowspan=1 colspan=1>yes</td><td rowspan=1 colspan=1>R</td><td rowspan=1 colspan=1>-</td></tr></table>

a For one prospective strain, a resistant phenotype to meropenem was reported by the clinical site, although this was not supported by the provided Kirby-Bauer zone diameter. Independent AST for meropenem and imipenem confirmed resistance to carbapenems for this strain.

# C. Contrived Study

The clinical performance evaluation of rare LRT BAL analytes was supplemented by a contrived study, i.e., by testing artificial clinical specimens that were prepared by spiking pooled microorganisms at concentrations close to their respective analyte LoD or at clinically relevant concentrations into unique negative lavage specimens. LRT BAL microorganisms, for which, due to limited specimen numbers, a PPA of $90 \%$ or higher with a lower bound of the two-sided $9 5 \%$ confidence interval of $80 \%$ or higher was not demonstrated for the clinical performance evaluation using archived specimens, were selected for contrived specimen testing (C. pneumoniae, C. freundii, $E$ . cloacae complex, $K .$ . oxytoca, $K .$ . pneumoniae, $K .$ . variicola, L. pneumophila, M. morganii, M. pneumoniae, $P .$ jirovecii, Proteus spp., S. marcescens). In addition, all low prevalence LRT BAL antibiotic resistance markers (ctx-M, kpc, ndm, oxa-23, oxa-24, oxa-48, oxa-58, vim) were also included. For each target analyte a total of 60 contrived specimens with up to five different spiked reference strains was tested at a low concentration $2 \mathbf { x }$ LoD, 30 specimens) and at two moderate concentrations $( 3 . 5 \mathrm { x }$ LoD and 5x LoD, 15 specimens each) spanning the clinically relevant concentration range. For $P .$ jirovecii, a positive clinical specimen was used for spiking, and all 60 contrived specimens were tested at a $2 \mathbf { x }$ LoD concentration.

Agreement rates to the expected results were determined for each target analyte and stratified by actual test concentrations Tables 50 and 51 show agreement rates for expected positive results (positive results / expected positive results) and expected negative results (negative results / number of expected negative results) together with two-sided $9 5 \%$ confidence intervals ( $9 5 \%$ CI) for included LRT BAL microorganisms or antibiotic resistance markers, respectively.

Table 50: Contrived study results for targeted LRT BAL microorganism analytes.   

<table><tr><td rowspan="2"></td><td colspan="6">Agreement with Expected Results</td></tr><tr><td># Pos./ # Exp.</td><td>%</td><td>95% CI</td><td># Neg./ # Exp.</td><td>%</td><td>95% CI</td></tr><tr><td>Chlamydia pneumoniae</td><td>57/60</td><td>95.0</td><td>86.3 - 98.3</td><td>180/180</td><td>100.0</td><td>97.9 - 100.0</td></tr><tr><td>6.4 x 102/2x LoD</td><td>29/30</td><td>96.7</td><td>83.3 - 99.4</td><td></td><td></td><td></td></tr><tr><td>ATCC 53592</td><td>5/6</td><td colspan="2"></td><td colspan="3"></td></tr><tr><td>ATCC VR-1310</td><td>12/12</td><td></td><td></td><td></td><td></td><td></td></tr><tr><td>ATCC VR-2282</td><td>12/12</td><td></td><td></td><td></td><td></td><td></td></tr><tr><td>1.1 x 103/3.5x LoD</td><td>15/15</td><td>100.0</td><td>79.6 - 100.0</td><td></td><td></td><td></td></tr><tr><td>ATCC 53592</td><td>3/3</td><td></td><td></td><td></td><td></td><td></td></tr><tr><td>ATCC VR-1310</td><td>6/6</td><td></td><td></td><td></td><td></td><td></td></tr><tr><td>ATCC VR-2282</td><td>6/6</td><td></td><td></td><td></td><td></td><td></td></tr><tr><td>1.6 x 103/5x LoD</td><td>13/15</td><td>86.7</td><td>62.1 - 96.3</td><td></td><td></td><td></td></tr><tr><td>ATCC 53592 ATCC VR-1310</td><td>2/3 5/6</td><td></td><td></td><td></td><td></td><td></td></tr><tr><td>ATCC VR-2282</td><td>6/6</td><td></td><td></td><td></td><td></td><td></td></tr><tr><td></td><td>59/60</td><td></td><td></td><td></td><td></td><td></td></tr><tr><td>Citrobacter freundii 1.6 x 105/2x LoD</td><td>29/30</td><td>98.3 96.7</td><td>91.1 - 99.7 83.3 - 99.4</td><td>168/168</td><td>100.0</td><td>97.8 - 100.0</td></tr><tr><td>ATCC 43864</td><td>6/6</td><td></td><td></td><td></td><td></td><td></td></tr><tr><td rowspan="6"></td><td>6/6</td><td></td><td></td><td></td><td></td><td></td></tr><tr><td>ATCC 8090</td><td></td><td></td><td></td><td></td><td></td></tr><tr><td>NCTC 8581 5/6</td><td></td><td></td><td></td><td></td><td></td></tr><tr><td>NRZ-00452 6/6</td><td></td><td></td><td></td><td></td><td></td></tr><tr><td>UCLA C1 6/6</td><td></td><td></td><td></td><td></td><td></td></tr><tr><td></td><td>15/15</td><td>100.0 79.6 - 100.0</td><td></td><td></td><td></td></tr><tr><td>ATCC 43864</td><td>3/3 3/3</td><td></td><td></td><td></td><td></td></tr><tr><td>ATCC 8090</td><td></td><td></td><td></td><td></td><td></td></tr><tr><td>NCTC 8581</td><td>3/3</td><td></td><td></td><td></td><td></td></tr><tr><td>NRZ-00452</td><td>3/3</td><td></td><td></td><td></td><td></td></tr><tr><td>UCLA C1</td><td>3/3 15/15</td><td>100.0</td><td>79.6 - 100.0</td><td></td><td></td></tr><tr><td>4.0 x 105/5x LoD ATCC 43864</td><td>3/3</td><td></td><td></td><td></td><td></td><td></td></tr><tr><td>ATCC 8090</td><td>3/3</td><td></td><td></td><td></td><td></td><td></td></tr><tr><td>NCTC 8581</td><td></td><td></td><td></td><td></td><td></td><td></td></tr><tr><td></td><td>3/3</td><td></td><td></td><td></td><td></td><td></td></tr><tr><td>NRZ-00452</td><td>3/3</td><td></td><td></td><td></td><td></td><td></td></tr><tr><td>UCLA C1 Enterobacter cloacae complex</td><td>3/3 58/60</td><td>96.7</td><td>88.6 - 99.1</td><td>156/156</td><td>100.0</td><td>97.6 - 100.0</td></tr><tr><td>4.0 x 105/2x LoD</td><td>28/30</td><td>93.3</td><td>78.7 - 98.2</td><td></td><td></td><td></td></tr><tr><td>ATCC 13047</td><td>5/6</td><td></td><td></td><td></td><td></td><td></td></tr><tr><td>ATCC 23373</td><td>6/6</td><td></td><td></td><td></td><td></td><td></td></tr><tr><td>ATCC 49141</td><td>5/6</td><td></td><td></td><td></td><td></td><td></td></tr><tr><td>ATCC BAA-2468</td><td>6/6</td><td></td><td></td><td></td><td></td><td></td></tr><tr><td>NRZ-00239</td><td>6/6</td><td></td><td></td><td></td><td></td><td></td></tr><tr><td>7.0 x 105/3.5x LoD</td><td>15/15</td><td>100.0</td><td>79.6 - 100.0</td><td></td><td></td><td></td></tr><tr><td>ATCC 13047</td><td>3/3</td><td></td><td></td><td></td><td></td><td></td></tr><tr><td>ATCC 23373</td><td>3/3</td><td></td><td></td><td></td><td></td><td></td></tr><tr><td>ATCC 49141</td><td>3/3</td><td></td><td></td><td></td><td></td><td></td></tr><tr><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr><tr><td>ATCC BAA-2468</td><td>3/3</td><td></td><td></td><td></td><td></td><td></td></tr><tr><td>NRZ-00239</td><td>3/3</td><td></td><td></td><td></td><td></td><td></td></tr><tr><td>1.0 x 106/5x LoD</td><td>15/15</td><td>100.0</td><td>79.6 - 100.0</td><td></td><td></td><td></td></tr><tr><td>ATCC 13047 ATCC 23373</td><td>3/3 3/3</td><td></td><td></td><td></td><td></td><td></td></tr></table>

510(k) Summary – LRT BAL Application K191967 Rev 3.0

<table><tr><td rowspan="2"></td><td colspan="6">Agreement with Expected Results</td></tr><tr><td># Pos./</td><td>%</td><td>95% CI</td><td>#Neg./</td><td>%</td><td>95% CI</td></tr><tr><td rowspan="2">ATCC 49141 ATCC BAA-2468</td><td># Exp. 3/3</td><td></td><td></td><td>#Exp.</td><td></td><td></td></tr><tr><td>3/3</td><td></td><td></td><td></td><td></td><td></td></tr><tr><td>NRZ-00239</td><td>3/3</td><td></td><td></td><td></td><td></td><td></td></tr><tr><td>Klebsiella oxytoca</td><td>56/60</td><td>93.3</td><td>84.1 - 97.4</td><td>168/168</td><td>100.0</td><td>97.8 - 100.0</td></tr><tr><td>2.0 x 104/2x LoD</td><td>26/30</td><td>86.7</td><td>70.3 - 94.7</td><td></td><td></td><td></td></tr><tr><td rowspan="13"></td><td>ATCC 13182 4/6</td><td></td><td></td><td></td><td></td><td></td></tr><tr><td>ATCC 43863 6/6</td><td></td><td></td><td></td><td></td><td></td></tr><tr><td>ATCC 49131</td><td>6/6</td><td></td><td></td><td></td><td></td></tr><tr><td>ATCC 8724</td><td>5/6</td><td></td><td></td><td></td><td></td></tr><tr><td>NCIMB 12819</td><td>5/6</td><td></td><td></td><td></td><td></td></tr><tr><td></td><td>15/15</td><td>100.0 79.6 - 100.0</td><td></td><td></td><td></td></tr><tr><td>3.5 x 104/3.5x LoD ATCC 13182</td><td>3/3</td><td></td><td></td><td></td><td></td></tr><tr><td>ATCC 43863</td><td>3/3</td><td></td><td></td><td></td><td></td></tr><tr><td>ATCC 49131</td><td>3/3</td><td></td><td></td><td></td><td></td><td></td></tr><tr><td>ATCC 8724</td><td>3/3</td><td></td><td></td><td></td><td></td><td></td></tr><tr><td>NCIMB 12819</td><td>3/3</td><td></td><td></td><td></td><td></td><td></td></tr><tr><td>5.0 x 104/5x LoD</td><td>15/15</td><td>100.0</td><td>79.6 - 100.0</td><td></td><td></td><td></td></tr><tr><td>ATCC 13182</td><td>3/3</td><td></td><td></td><td></td><td></td><td></td></tr><tr><td>ATCC 43863</td><td>3/3</td><td></td><td></td><td></td><td></td><td></td></tr><tr><td>ATCC 49131</td><td>3/3</td><td></td><td></td><td></td><td></td><td></td></tr><tr><td>ATCC 8724</td><td>3/3</td><td></td><td></td><td></td><td></td><td></td></tr><tr><td>NCIMB 12819</td><td>3/3</td><td>100.0</td><td>94.0 - 100.0</td><td>96/96</td><td>100.0</td><td></td></tr><tr><td>Klebsiella pneumoniae 8.0 x 104/2x LoD</td><td>60/60 30/30</td><td>100.0</td><td>88.7 - 100.0</td><td></td><td></td><td>96.2 - 100.0</td></tr><tr><td>ATCC 13883</td><td></td><td></td><td></td><td></td><td></td><td></td></tr><tr><td></td><td>6/6</td><td></td><td></td><td></td><td></td><td></td></tr><tr><td>Micromyx 4653</td><td>6/6</td><td></td><td></td><td></td><td></td><td></td></tr><tr><td>NCTC 13438 NCTC 13439</td><td>6/6</td><td></td><td></td><td></td><td></td><td></td></tr><tr><td>NCTC 13442</td><td>6/6 6/6</td><td></td><td></td><td></td><td></td><td></td></tr><tr><td>1.4 x 105/3.5x LoD ATCC 13883 Micromyx 4653</td><td>15/15</td><td>100.0</td><td>79.6 - 100.0</td><td></td><td></td><td></td></tr><tr><td></td><td>3/3</td><td></td><td></td><td></td><td></td><td></td></tr><tr><td></td><td>3/3</td><td></td><td></td><td></td><td></td><td></td></tr><tr><td>NCTC 13438 NCTC 13439</td><td>3/3</td><td></td><td></td><td></td><td></td><td></td></tr><tr><td></td><td>3/3</td><td></td><td></td><td></td><td></td><td></td></tr><tr><td></td><td>NCTC 13442 3/3</td><td></td><td></td><td></td><td></td><td></td></tr><tr><td>2.0 x 105/5x LoD</td><td>15/15</td><td>100.0</td><td>79.6 - 100.0</td><td></td><td></td><td></td></tr><tr><td>ATCC 13883</td><td>3/3</td><td></td><td></td><td></td><td></td><td></td></tr><tr><td>Micromyx 4653</td><td>3/3</td><td></td><td></td><td></td><td></td><td></td></tr><tr><td>NCTC 13438</td><td>3/3</td><td></td><td></td><td></td><td></td><td></td></tr><tr><td>NCTC 13439</td><td>3/3</td><td></td><td></td><td></td><td></td><td></td></tr><tr><td>NCTC 13442</td><td>3/3</td><td></td><td>88.6 - 99.1</td><td>180/180</td><td>100.0</td><td>97.9 - 100.0</td></tr><tr><td>Klebsiella variicola 4.0 x 104/2x LoD</td><td>58/60 29/30</td><td>96.7 96.7</td><td>83.3 - 99.4</td><td></td><td></td><td></td></tr><tr><td></td><td>ATCC BAA-830 6/6</td><td></td><td></td><td></td><td></td><td></td></tr><tr><td></td><td>clinical isolate 1 6/6</td><td></td><td></td><td></td><td></td><td></td></tr><tr><td></td><td>clinical isolate 2 6/6</td><td></td><td></td><td></td><td></td><td></td></tr><tr><td>clinical isolate 4</td><td>clinical isolate 3 6/6 5/6</td><td></td><td></td><td></td><td></td><td></td></tr><tr><td>7.0 x 104/3.5x LoD</td><td>15/15</td><td>100.0</td><td>79.6 - 100.0</td><td></td><td></td><td></td></tr><tr><td>clinical isolate 1</td><td>ATCC BAA-830 3/3 3/3</td><td></td><td></td><td></td><td></td><td></td></tr><tr><td></td><td>clinical isolate 2 3/3 clinical isolate 3 3/3</td><td></td><td></td><td></td><td></td><td></td></tr><tr><td></td><td>clinical isolate 4</td><td>3/3 14/15</td><td></td><td></td><td></td><td></td></tr><tr><td>1.0 x 105/5x LoD</td><td></td><td>93.3</td><td>70.2 - 98.8</td><td></td><td></td><td></td></tr><tr><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr><tr><td>ATCC BAA-830</td><td>3/3</td><td></td><td></td><td></td><td></td><td></td></tr><tr><td>clinical isolate 2 clinical isolate 3</td><td>3/3 2/3</td><td></td><td></td><td></td><td></td><td></td></tr><tr><td></td><td>clinical isolate 1</td><td></td><td></td><td></td><td></td></table>

510(k) Summary – LRT BAL Application K191967 Rev 3.0

<table><tr><td rowspan="2"></td><td colspan="6">Agreement with Expected Results</td></tr><tr><td># Pos./ #Exp.</td><td>%</td><td>95% CI</td><td>#Neg./ #Exp.</td><td>%</td><td>95% CI</td></tr><tr><td></td><td>3/3</td><td></td><td></td><td></td><td></td><td></td></tr><tr><td>clinical isolate 4 Legionella pneumophila</td><td>57/60</td><td>95.0</td><td>86.3 - 98.3</td><td>180/180</td><td>100.0</td><td>97.9 - 100.0</td></tr><tr><td>1.6 x 105/2x LoD</td><td>28/30</td><td>93.3</td><td>78.7 - 98.2</td><td></td><td></td><td></td></tr><tr><td rowspan="10"></td><td>6/6</td><td></td><td></td><td></td><td></td><td></td></tr><tr><td>ATCC 33152 ATCC 33154</td><td>5/6</td><td></td><td></td><td></td><td></td></tr><tr><td></td><td>5/6</td><td></td><td></td><td></td><td></td></tr><tr><td>ATCC 33155</td><td></td><td></td><td></td><td></td><td></td></tr><tr><td>ATCC 33215</td><td>6/6 6/6</td><td></td><td></td><td></td><td></td></tr><tr><td>ATCC 35096</td><td></td><td>100.0 79.6 - 100.0</td><td></td><td></td><td></td></tr><tr><td>2.8 x 105/3.5x LoD</td><td>15/15</td><td></td><td></td><td></td><td></td></tr><tr><td>ATCC 33152</td><td>3/3 3/3</td><td></td><td></td><td></td><td></td></tr><tr><td>ATCC 33154</td><td></td><td></td><td></td><td></td><td></td></tr><tr><td>ATCC 33155 3/3 ATCC 33215</td><td></td><td></td><td></td><td></td><td></td></tr><tr><td></td><td>3/3 ATCC 35096 3/3 14/15</td><td></td><td></td><td></td><td></td><td></td></tr><tr><td rowspan="6">4.0 x 105/5x LoD</td><td></td><td>93.3</td><td></td><td>70.2 - 98.8</td><td></td><td></td></tr><tr><td>ATCC 33152</td><td></td><td></td><td></td><td></td><td></td></tr><tr><td>2/3</td><td></td><td></td><td></td><td></td><td></td></tr><tr><td>ATCC 33154 3/3 ATCC 33155</td><td></td><td></td><td></td><td></td><td></td></tr><tr><td>3/3 ATCC 33215 3/3</td><td></td><td></td><td></td><td></td><td></td></tr><tr><td>ATCC 35096</td><td>3/3</td><td></td><td></td><td></td><td></td></tr><tr><td rowspan="10">Morganella morganii 4.0 x 104/2x LoD</td><td></td><td>86.7</td><td></td><td>75.8 - 93.1</td><td>179/179</td><td></td><td>97.9 - 100.0</td></tr><tr><td></td><td>52/60 23/30</td><td>76.7</td><td>59.1 - 88.2</td><td></td><td>100.0</td><td></td></tr><tr><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr><tr><td>ATCC 25829 a</td><td>3/6</td><td></td><td></td><td></td><td></td><td></td></tr><tr><td>ATCC 25830</td><td>6/6 6/6</td><td></td><td></td><td></td><td></td><td></td></tr><tr><td>ATCC 49948</td><td>6/6</td><td></td><td></td><td></td><td></td><td></td></tr><tr><td>ATCC 8019</td><td>2/6</td><td></td><td></td><td></td><td></td><td></td></tr><tr><td>DSM-46262 a 7.0 x 104/3.5x LoD</td><td>15/15</td><td>100.0</td><td>79.6 - 100.0</td><td></td><td></td><td></td></tr><tr><td></td><td>3/3</td><td></td><td></td><td></td><td></td><td></td></tr><tr><td>ATCC 25829 ATCC 25830</td><td>3/3</td><td></td><td></td><td></td><td></td><td></td></tr><tr><td>ATCC 49948</td><td>3/3</td><td></td><td></td><td></td><td></td><td></td></tr><tr><td>ATCC 8019</td><td>3/3</td><td></td><td></td><td></td><td></td><td></td></tr><tr><td>DSM-46262</td><td>3/3</td><td></td><td></td><td></td><td></td><td></td></tr><tr><td></td><td>14/15</td><td>93.3</td><td>70.2 - 98.8</td><td></td><td></td><td></td></tr><tr><td>ATCC 25829</td><td>3/3</td><td></td><td></td><td></td><td></td><td></td></tr><tr><td rowspan="6"></td><td></td><td>3/3</td><td></td><td></td><td></td><td></td><td></td></tr><tr><td>ATCC 25830</td><td>3/3</td><td></td><td></td><td></td><td></td><td></td></tr><tr><td>ATCC 49948</td><td></td><td></td><td></td><td></td><td></td><td></td></tr><tr><td>ATCC 8019 DSM-46262</td><td>3/3 2/3</td><td></td><td></td><td></td><td></td><td></td></tr><tr><td>Mycoplasma pneumoniae</td><td>52/60</td><td>86.7</td><td>75.8 - 93.1</td><td>180/180</td><td>100.0</td><td>97.9 - 100.0</td></tr><tr><td>ATCC 15293</td><td>23/30</td><td>76.7</td><td>59.1 - 88.2</td><td></td><td></td><td></td></tr><tr><td rowspan="8">5.6 x 103/3.5x LoD</td><td></td><td>6/6</td><td></td><td></td><td></td><td></td><td></td></tr><tr><td>ATCC 29085</td><td>8/12</td><td></td><td></td><td></td><td></td><td></td></tr><tr><td>ATCC 49894</td><td>9/12</td><td></td><td>70.2 - 98.8</td><td></td><td></td><td></td></tr><tr><td>ATCC 15293</td><td>14/15</td><td>93.3</td><td></td><td></td><td></td><td></td></tr><tr><td>ATCC 29085</td><td>2/3 6/6</td><td></td><td></td><td></td><td></td><td></td></tr><tr><td>ATCC 49894</td><td>6/6</td><td></td><td></td><td></td><td></td><td></td></tr><tr><td>8.0 x 103/5x LoD</td><td>15/15</td><td>100.0</td><td>79.6 - 100.0</td><td></td><td></td><td></td></tr><tr><td>ATCC 15293</td><td>3/3</td><td></td><td></td><td></td><td></td><td></td></tr><tr><td></td><td>ATCC 29085</td><td>6/6</td><td></td><td></td><td></td><td></td><td></td></tr><tr><td></td><td>ATCC 49894</td><td>6/6</td><td></td><td></td><td>180/180</td><td></td><td></td></tr><tr><td>Pneumocystis jirovecii</td><td></td><td>59/60</td><td>98.3</td><td>91.1 - 99.7</td><td></td><td>100.0</td><td>97.9 - 100.0</td></tr><tr><td>1.0 x 106/2x LoD positive clinical specimen</td><td></td><td>59/60 59/60</td><td>98.3</td><td>91.1 - 99.7</td><td>100.0</td><td></td></table>

510(k) Summary – LRT BAL Application K191967 Rev 3.0

<table><tr><td rowspan=2 colspan=1></td><td rowspan=1 colspan=6>Agreement with Expected Results</td></tr><tr><td rowspan=1 colspan=1># Pos./# Exp.</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=1># Neg./# Exp.</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=1>ATCC 25933ATCC 29906ATCC 6380</td><td rowspan=1 colspan=1>6/66/66/6</td><td rowspan=1 colspan=2></td><td rowspan=1 colspan=3></td></tr><tr><td rowspan=1 colspan=1>1.8 x 104/3.5x LoD</td><td rowspan=1 colspan=1>15/15</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>79.6 - 100.0</td><td rowspan=2 colspan=3></td></tr><tr><td rowspan=1 colspan=1>ATCC 12453ATCC 14153ATCC 25933ATCC 29906ATCC 6380</td><td rowspan=1 colspan=1>3/33/33/33/33/3</td><td rowspan=1 colspan=2></td></tr><tr><td rowspan=1 colspan=1>2.5 x 104/5x LoD</td><td rowspan=1 colspan=1>15/15</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>79.6 - 100.0</td><td rowspan=2 colspan=3></td></tr><tr><td rowspan=1 colspan=1>ATCC 12453ATCC 14153ATCC 25933ATCC 29906ATCC 6380</td><td rowspan=1 colspan=1>3/33/33/33/33/3</td><td rowspan=1 colspan=2></td></tr><tr><td rowspan=1 colspan=1>Serratia marcescens</td><td rowspan=1 colspan=1>59/60</td><td rowspan=1 colspan=1>98.3</td><td rowspan=1 colspan=1>91.1 - 99.7</td><td rowspan=1 colspan=3>180/180    100.0   97.9 - 100.0</td></tr><tr><td rowspan=1 colspan=1>8.0 x 104/2x LoD</td><td rowspan=1 colspan=1>29/30</td><td rowspan=1 colspan=1>96.7</td><td rowspan=1 colspan=1>83.3 - 99.4</td><td rowspan=2 colspan=3></td></tr><tr><td rowspan=1 colspan=1>ATCC 13880ATCC 14756ATCC 15365ATCC 27117DSM-17174</td><td rowspan=1 colspan=1>6/66/66/66/65/6</td><td rowspan=1 colspan=2></td></tr><tr><td rowspan=1 colspan=1>1.4 x 105/3.5x LoD</td><td rowspan=1 colspan=1>15/15</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>79.6 - 100.0</td><td rowspan=2 colspan=3></td></tr><tr><td rowspan=1 colspan=1>ATCC 13880ATCC 14756ATCC 15365ATCC 27117DSM-17174</td><td rowspan=1 colspan=1>3/33/33/33/33/3</td><td rowspan=1 colspan=2></td></tr><tr><td rowspan=1 colspan=1>2.0 x 105/5x LoD</td><td rowspan=1 colspan=1>15/15</td><td rowspan=1 colspan=2>100.0   79.6 - 100.0</td><td rowspan=2 colspan=3></td></tr><tr><td rowspan=1 colspan=1>ATCC 13880ATCC 14756ATCC 15365ATCC 27117DSM-17174</td><td rowspan=1 colspan=1>3/33/33/33/33/3</td><td rowspan=1 colspan=2></td></tr></table>

a Morganella morganii strains ATCC 25829 and DSM-46262 strains were retested with pooled negative BAL matrix at 2x LoD both showing 10/10 results. b Mycoplasma pneumoniae strains ATCC 29085 and ATCC 49894 were retested with pooled negative BAL matrix at $2 \mathbf { x }$ LoD showing 19/20 or 18/20 positive results, respectively. For three of four unexpected negative cases of ATCC 29085, used unique matrices correlated with reduced signals or false negative results also for other analytes and therefore such failures may be matrix-related.

Table 51: Contrived study results for targeted LRT BAL antibiotic resistance marker analytes.   

<table><tr><td rowspan="2"></td><td colspan="6">Agreement with Expected Results</td></tr><tr><td># Pos./ # Exp.</td><td>%</td><td>95 %CI</td><td># Neg./ # Exp.</td><td>%</td><td>95 %CI</td></tr><tr><td>ctx-M</td><td>59/60</td><td>98.3</td><td>91.1 - 99.7</td><td>120/120</td><td>100.0</td><td>96.9 - 100.0</td></tr><tr><td>2.0 x 104/2x LoD</td><td>30/30</td><td>100.0</td><td>88.7 - 100.0</td><td></td><td></td><td></td></tr><tr><td>JMI 49767</td><td>6/6</td><td></td><td></td><td></td><td></td><td></td></tr><tr><td>JMI 50067</td><td>6/6</td><td></td><td></td><td></td><td></td><td></td></tr><tr><td>NCTC 13443</td><td>6/6</td><td></td><td></td><td></td><td></td><td></td></tr><tr><td>NRZ-00249</td><td>6/6</td><td></td><td></td><td></td><td></td><td></td></tr><tr><td>NRZ-00751</td><td>6/6</td><td></td><td></td><td></td><td></td><td></td></tr><tr><td>3.5 x 104/3.5x LoD</td><td>14/15</td><td>93.3</td><td>70.2 - 98.8</td><td></td><td></td><td></td></tr><tr><td>JMI 49767</td><td>3/3</td><td colspan="2"></td><td></td><td colspan="2"></td></tr><tr><td>JMI 50067</td><td>2/3</td><td colspan="2" rowspan="3"></td><td></td><td rowspan="12"></td><td rowspan="3"></td></tr><tr><td>NCTC 13443</td><td>3/3</td><td></td></tr><tr><td>NRZ-00249</td><td>3/3</td><td></td></tr><tr><td># Pos./ #Exp.</td><td>%</td><td>95 %CI</td><td># Neg./ #Exp.</td><td>%</td><td>95 %CI</td></tr><tr><td>5.0 x 104/5x LoD</td><td>NRZ-00751 JMI 49767</td><td>3/3 15/15 100.0 3/3</td><td>79.6 - 100.0</td><td></td><td></td><td></td></tr><tr><td></td><td>JMI 50067 NCTC 13443 NRZ-00249</td><td>3/3 3/3</td><td></td><td></td><td></td><td></td></tr><tr><td></td><td>NRZ-00751</td><td>3/3 3/3</td><td>96.7 88.6 - 99.1</td><td></td><td></td><td></td></tr><tr><td>kpc 8.0 x 104/2x LoD</td><td></td><td>58/60 29/30</td><td>96.7 83.3 - 99.4</td><td>156/156</td><td>100.0</td><td>97.6 - 100.0</td></tr><tr><td></td><td>Micromyx 4653 NCTC 13438 NRZ-00103</td><td>6/6 6/6 6/6</td><td></td><td></td><td></td><td></td></tr><tr><td>1.4 x 105/3.5x LoD</td><td>NRZ-00222 NRZ-00281</td><td>5/ 6/6 14/15</td><td>93.3 70.2 - 98.8</td><td></td><td></td><td></td></tr><tr><td></td><td>Micromyx 4653 NCTC 13438</td><td>3/3 3/3</td><td></td><td></td><td></td><td></td></tr><tr><td></td><td>NRZ-00103 NRZ-00222 NRZ-00281</td><td>2/3 3/3 3/3</td><td></td><td></td><td></td><td></td></tr><tr><td>2.0 x 105/5x LoD</td><td>Micromyx 4653 NCTC 13438</td><td>15/15 3/3 </td><td>100.0 79.6 - 100.0</td><td></td><td></td><td></td></tr><tr><td></td><td>NRZ-00103 NRZ-00222 NRZ-00281</td><td>3/3 3/3 3/3</td><td></td><td></td><td></td><td></td></tr><tr><td>ndm 4.0 x 104/2x LoD</td><td></td><td>59/60 29/30</td><td>98.3 91.1 - 99.7 96.7 83.3 - 99.4</td><td>132/132</td><td>100.0</td><td>97.2 - 100.0</td></tr><tr><td></td><td>ATCC BAA-2468 JMI 46239 JMI 49755</td><td>6/6 6/6 5/6</td><td></td><td></td><td></td><td></td></tr><tr><td>7.0 x 104/3.5x LoD</td><td>JMI 49831 NCTC 13443</td><td>6/6 6/6</td><td>79.6 - 100.0</td><td></td><td></td><td></td></tr><tr><td></td><td>ATCC BAA-2468 JMI 46239</td><td></td><td></td><td></td><td></td><td></td></tr><tr><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr><tr><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr><tr><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr><tr><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr><tr><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr><tr><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr><tr><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr><tr><td></td><td>15/15</td><td>100.0</td><td></td><td></td><td></td><td></td></tr><tr><td></td><td>3/3</td><td></td><td></td><td></td><td></td><td></td></tr><tr><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr><tr><td></td><td>3/3</td><td></td><td></td><td></td><td></td><td></td></tr><tr><td>JMI 49755</td><td></td><td></td><td></td><td></td><td></td><td></td></tr><tr><td></td><td>3/3</td><td></td><td></td><td></td><td></td><td></td></tr><tr><td>JMI 49831</td><td>3/3</td><td></td><td></td><td></td><td></td><td></td></tr><tr><td>NCTC 13443</td><td></td><td></td><td></td><td></td><td></td><td></td></tr><tr><td>1.0 x 105/5x LoD</td><td>3/3</td><td>100.0</td><td>79.6 - 100.0</td><td></td><td></td><td></td></tr><tr><td>ATCC BAA-2468</td><td>15/15</td><td></td><td></td><td></td><td></td><td></td></tr><tr><td>JMI 46239</td><td>3/3</td><td></td><td></td><td></td><td></td><td></td></tr><tr><td>JMI 49755</td><td>3/3 3/3</td><td></td><td></td><td></td><td></td><td></td></tr><tr><td>JMI 49831</td><td>3/3</td><td></td><td></td><td></td><td></td><td></td></tr><tr><td>NCTC 13443</td><td>3/3</td><td></td><td>94.0 - 100.0</td><td>180/180</td><td></td><td></td></tr><tr><td>oxa-23 2.0 x 107/2x LoD</td><td></td><td>60/60</td><td>100.0 100.0 88.7 - 100.0</td><td></td><td>100.0</td><td>97.9 - 100.0</td></tr><tr><td></td><td>Micromyx 4410</td><td>30/30 6/6</td><td></td><td></td><td></td><td></td></tr><tr><td></td><td>Micromyx 6148 Micromyx 6149</td><td>6/6 6/6</td><td></td><td></td><td></td><td></td></tr><tr><td></td><td>NCTC 13301 UCLA A5</td><td>6/6 6/6</td><td></td><td></td><td></td><td></td></tr><tr><td>3.5 x 107/3.5x LoD</td><td></td><td>15/15</td><td>100.0 79.6 - 100.0</td><td></td><td></td><td></td></tr><tr><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr><tr><td></td><td>Micromyx 4410</td><td>3/3</td><td></td><td></td><td></td><td></td></tr><tr><td></td><td>Micromyx 6148</td><td>3/3</td><td></td><td></td><td></td><td></td></tr><tr><td>Micromyx 6149 NCTC 13301 UCLA A5</td><td></td><td>3/3 3/3</td><td></td><td></td><td></td><td></td></tr></table>

510(k) Summary – LRT BAL Application K191967 Rev 3.0

<table><tr><td rowspan="2"></td><td colspan="6">Agreement with Expected Results</td></tr><tr><td># Pos./ #Exp. 3/3</td><td>%</td><td>95 %CI</td><td>#Neg./ #Exp.</td><td>%</td><td>95 %CI</td></tr><tr><td></td><td>Micromyx 4410 Micromyx 6148 3/3 Micromyx 6149 3/3</td><td></td><td></td><td></td><td></td><td></td></tr><tr><td>NCTC 13301</td><td>3/3 UCLA A5 3/3</td><td></td><td></td><td></td><td></td><td></td></tr><tr><td>oxa-24 1.0 x 104/2x LOD</td><td>59/60 29/30</td><td>98.3 96.7</td><td>91.1 - 99.7 83.3 - 99.4</td><td>180/180</td><td>100.0</td><td>97.9 - 100.0</td></tr><tr><td>clinical isolate 1 clinical isolate 2</td><td>6/6 6/6 6/6</td><td></td><td></td><td></td><td></td><td></td></tr><tr><td>1.8 x 104/3.5x LOD</td><td>NCTC 13302 NRZ-00449 5/6 UCLA A4 6/6 15/15</td><td>100.0</td><td>79.6 - 100.0</td><td></td><td></td><td></td></tr><tr><td>clinical isolate 2</td><td>clinical isolate 1 3/3 NCTC 13302</td><td></td><td></td><td></td><td></td><td></td></tr><tr><td>NRZ-00449 UCLA A4</td><td>3/3 3/3</td><td></td><td></td><td></td><td></td><td></td></tr><tr><td></td><td>3/3</td><td></td><td></td><td></td><td></td><td></td></tr><tr><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr><tr><td></td><td>3/3</td><td></td><td></td><td></td><td></td><td></td></tr><tr><td>2.5 x 104/5x LOD</td><td></td><td></td><td></td><td></td><td></td><td></td></tr><tr><td></td><td>15/15</td><td></td><td></td><td></td><td></td><td></td></tr><tr><td>clinical isolate 1</td><td></td><td>100.0</td><td>79.6 - 100.0</td><td></td><td></td><td></td></tr><tr><td></td><td>3/3</td><td></td><td></td><td></td><td></td><td></td></tr><tr><td>clinical isolate 2</td><td></td><td></td><td></td><td></td><td></td><td></td></tr><tr><td>NCTC 13302</td><td>3/3</td><td></td><td></td><td></td><td></td><td></td></tr><tr><td></td><td>3/3</td><td></td><td></td><td></td><td></td><td></td></tr><tr><td>NRZ-00449</td><td>3/3</td><td></td><td></td><td></td><td></td><td></td></tr><tr><td>UCLA A4</td><td>3/3</td><td></td><td></td><td></td><td></td><td></td></tr><tr><td>oxa-48</td><td>59/60</td><td>98.3</td><td>91.1- 99.7</td><td></td><td></td><td></td></tr><tr><td>6.0 x 105/2x LoD</td><td>29/30</td><td>96.7</td><td>83.3 - 99.4</td><td>167/167</td><td>100.0</td><td>97.8 - 100.0</td></tr><tr><td>ATCC BAA-2523 NCTC 13442</td><td>5/6</td><td></td><td></td><td></td><td></td><td></td></tr><tr><td></td><td>6/6</td><td></td><td></td><td></td><td></td><td></td></tr><tr><td>NRZ-00176</td><td>6/6</td><td></td><td></td><td></td><td></td><td></td></tr><tr><td>NRZ-00472</td><td>6/6</td><td></td><td></td><td></td><td></td><td></td></tr><tr><td>NRZ-22060</td><td>6/6</td><td></td><td></td><td></td><td></td><td></td></tr><tr><td>1.1 x 106/3.5x LoD</td><td>15/15</td><td>100.0</td><td>79.6 - 100.0</td><td></td><td></td><td></td></tr><tr><td>ATCC BAA-2523</td><td></td><td></td><td></td><td></td><td></td><td></td></tr><tr><td>NCTC 13442</td><td>3/3</td><td></td><td></td><td></td><td></td><td></td></tr><tr><td>NRZ-00176</td><td>3/3 3/3</td><td></td><td></td><td></td><td></td><td></td></tr><tr><td>NRZ-00472</td><td>3/3</td><td></td><td></td><td></td><td></td><td></td></tr><tr><td>NRZ-22060 1.5 x 106/5x LoD</td><td>3/3 15/15</td><td>100.0</td><td>79.6 - 100.0</td><td></td><td></td><td></td></tr><tr><td>ATCC BAA-2523 NCTC 13442 NRZ-00176</td><td>3/3 3/3</td><td></td><td></td><td></td><td></td><td></td></tr><tr><td>NRZ-00472</td><td>3/3 3/3</td><td></td><td></td><td></td><td></td><td></td></tr><tr><td>NRZ-22060 oxa-58</td><td>3/3</td><td></td><td></td><td></td><td></td><td></td></tr><tr><td>1.0 x 105/2x LoD</td><td>59/60 30/30</td><td>98.3</td><td>91.1-99.7</td><td>180/180</td><td>100.0</td><td>97.9 - 100.0</td></tr><tr><td>NCTC 13305</td><td>18/18</td><td>100.0</td><td>88.7 - 100.0</td><td></td><td></td><td></td></tr><tr><td>NRZ-00518</td><td>12/12</td><td></td><td></td><td></td><td></td><td></td></tr><tr><td>1.8 x 105/3.5x LoD</td><td>14/15</td><td>93.3</td><td>70.2 - 98.8</td><td></td><td></td><td></td></tr><tr><td>NCTC 13305 NRZ-00518</td><td>8/9</td><td></td><td></td><td></td><td></td><td></td></tr><tr><td>2.5 x 105/5x LoD</td><td>6/6 15/15</td><td>100.0</td><td>79.6 - 100.0</td><td></td><td></td><td></td></tr><tr><td>NCTC 13305</td><td>9/9</td><td></td><td></td><td></td><td></td><td></td></tr><tr><td>NRZ-00518</td><td>6/6</td><td></td><td></td><td>144/144</td><td></td><td></td></tr><tr><td>vim 4.0 x 104/2x LoD</td><td>60/60 30/30</td><td>100.0 100.0</td><td>94.0 - 100.0 88.7 - 100.0</td><td></td><td>100.0</td><td>97.4 - 100.0</td></tr><tr><td>DSM-24600 NCTC 13437</td><td>6/6 6/6</td><td></td><td></td><td></td><td></td><td></td></tr></table>

510(k) Summary – LRT BAL Application K191967 Rev 3.0

<table><tr><td colspan="2"></td><td colspan="6">Agreement with Expected Results</td></tr><tr><td colspan="2"></td><td># Pos./</td><td>%</td><td>95 %CI</td><td># Neg./ # Exp.</td><td>%</td><td>95 %CI</td></tr><tr><td colspan="2"></td><td># Exp. 6/6</td><td></td><td></td><td></td><td></td><td></td></tr><tr><td colspan="2">NRZ-00452 7.0 x 104/3.5x LoD</td><td>15/15</td><td>100.0</td><td>79.6 - 100.0</td><td colspan="3"></td></tr><tr><td colspan="2">DSM-24600</td><td>3/3</td><td colspan="2"></td><td></td><td></td><td></td></tr><tr><td colspan="2">NCTC 13437</td><td>3/3</td><td colspan="2"></td><td></td><td></td><td></td></tr><tr><td></td><td>NCTC 13439</td><td>3/3</td><td colspan="2"></td><td></td><td></td><td></td></tr><tr><td></td><td></td><td></td><td colspan="2"></td><td></td><td></td><td></td></tr><tr><td></td><td>NCTC 13440</td><td>3/3 3/3</td><td colspan="2"></td><td></td><td></td><td></td></tr><tr><td>1.0 x 105/5x LoD</td><td>NRZ-00452</td><td>15/15</td><td colspan="2">100.0 79.6 - 100.0</td><td></td><td></td><td></td></tr><tr><td></td><td>DSM-24600</td><td>3/3</td><td colspan="2"></td><td></td><td></td><td></td></tr><tr><td></td><td>NCTC 13437</td><td>3/3</td><td colspan="2"></td><td></td><td></td><td></td></tr><tr><td></td><td>NCTC 13439</td><td></td><td colspan="2"></td><td></td><td></td><td></td></tr><tr><td></td><td>NCTC 13440</td><td>3/3 3/3</td><td colspan="2"></td><td></td><td></td><td></td></tr><tr><td></td><td></td><td></td><td colspan="2"></td><td></td><td></td><td></td></tr><tr><td>NRZ-00452</td><td>3/3</td><td></td><td colspan="2"></td><td></td><td></td><td></td></tr></table>

For other LRT BAL panel analytes not evaluated in the contrived study, the following agreement rates with the expected negative result were observed: 120/120 ( $1 0 0 . 0 \%$ , $9 5 \%$ CI $9 6 . 9 ~ - ~ 1 0 0 . 0 $ for Acinetobacter spp., 200/200 $1 0 0 . 0 \%$ , $9 5 \%$ C $\mid 9 8 . 1 \ - \ 1 0 0 . 0 )$ for $E .$ . coli, 228/228 $1 0 0 . 0 \%$ , $9 5 \%$ CI 98.3 - 100.0) for $H .$ influenzae, 232/232 $1 0 0 . 0 \%$ , $9 5 \%$ CI 98.4 - 100.0) for mecA, 239/239 ( $1 0 0 . 0 \%$ , $9 5 \%$ CI 98.4 - 100.0) for $M .$ catarrhalis, 208/208 ( $1 0 0 . 0 \%$ , $9 5 \%$ CI 98.2 - 100.0) for $P .$ aeruginosa, 179/180 $( 9 9 . 4 ~ \%$ , $9 5 \%$ CI 96.9 - 99.9) for $S .$ . aureus, 240/240 ( $1 0 0 . 0 \%$ , $9 5 \%$ CI 98.4 - 100.0) for $S .$ . maltophilia, 239/240 $( 9 9 . 6 \%$ , $9 5 \%$ CI 97.7 - 99.9) for S. pneumoniae, and 82/82 ( $100 \%$ , $9 5 \%$ CI 95.5 - 100.0) for tem.

# Expected Values

Expected values were determined for the prospective study cohort with 1,016 evaluable BAL specimens collected from hospitalized patients suspected for lower respiratory tract infection at nine US sites between June 2015 and July 2016. The LRT BAL Application reported 363 specimens with at least one positive LRT BAL panel microorganism, including 113 specimens with multi-detections of two or more LRT BAL panel microorganisms.

Table 52 lists expected values for microorganisms; Tables 53 and 54 list expected values for antibiotic resistance markers of the LRT BAL panel (for all specimens and stratified for each host microorganism). Note that antibiotic resistance markers are reported by LRT BAL only for specimens positive for a corresponding host microorganism.

Table 52: Expected values of LRT BAL panel microorganisms as reported for the prospective study stratified by all specimens $\mathbf { ( N = 1 , 0 1 6 ) }$ , as well as by single-detection $( \mathbf { N } = 2 5 \mathbf { 0 } )$ or multi-detection specimens $\mathbf { \delta N } = \mathbf { 1 1 } 3$ ). Expected value rates are given as percentage of 1,016 prospective study specimens.   

<table><tr><td colspan="9">Expected Values (N = 1,016 Prospective Specimens)</td></tr><tr><td>Microorganism</td><td></td><td># Spec.</td><td>%</td><td>Microorganism</td><td></td><td># Spec.</td><td>%</td></tr><tr><td rowspan="4">Acinetobacter spp.</td><td>all</td><td>21</td><td>2.1%</td><td rowspan="4">Moraxella catarrhalis</td><td>all</td><td>15</td><td>1.5%</td></tr><tr><td>single</td><td>8</td><td>0.8%</td><td>single</td><td>4</td><td>0.4%</td></tr><tr><td>multi</td><td>13</td><td>1.3%</td><td>multi</td><td>11</td><td>1.1%</td></tr><tr><td>all</td><td>0</td><td>0.0%</td><td>all</td><td>3</td><td>0.3%</td></tr><tr><td rowspan="3">Chlamydia pneumoniae</td><td>single</td><td>0</td><td>0.0%</td><td rowspan="3">Morganella morganii</td><td>single</td><td>0</td><td>0.0%</td></tr><tr><td>multi</td><td>0</td><td>0.0%</td><td>multi</td><td>3</td><td>0.3%</td></tr><tr><td>all</td><td>4</td><td>0.4%</td><td>all</td><td>6</td><td>0.6%</td></tr><tr><td rowspan="3">Citrobacter freundii</td><td>single</td><td>1</td><td>0.1%</td><td rowspan="3">Mycoplasma pneumoniae</td><td>single</td><td>4</td><td>0.4%</td></tr><tr><td>multi</td><td>3</td><td>0.3%</td><td>multi</td><td>2</td><td>0.2%</td></tr><tr><td>all</td><td>20</td><td>2.0%</td><td>all</td><td>22</td><td>2.2%</td></tr><tr><td rowspan="3">Enterobacter cloacae complex</td><td>single</td><td>8</td><td>0.8%</td><td rowspan="3">Pneumocystis jirovecii</td><td>single</td><td>14</td><td>1.4%</td></tr><tr><td>multi</td><td>12</td><td>1.2%</td><td></td><td>8</td><td>0.8%</td></tr><tr><td>all</td><td>47</td><td>4.6%</td><td>multi all</td><td>10</td><td>1.0%</td></tr><tr><td rowspan="3">Escherichia coli</td><td>single</td><td>24</td><td>2.4%</td><td rowspan="3">Proteus spp.</td><td>single</td><td>0</td><td>0.0%</td></tr><tr><td>multi</td><td>23</td><td>2.3%</td><td>multi</td><td>10</td><td>1.0%</td></tr><tr><td>all</td><td>56</td><td>5.5%</td><td>all</td><td>112</td><td>11.0%</td></tr><tr><td rowspan="3">Haemophilus influenzae</td><td>single</td><td>30</td><td>3.0%</td><td rowspan="3">Pseudomonas aeruginosa</td><td>single</td><td>61</td><td>6.0%</td></tr><tr><td>multi</td><td>26</td><td>2.6%</td><td></td><td></td><td>5.0%</td></tr><tr><td>all</td><td>14</td><td></td><td>multi</td><td>51 17</td><td>1.7%</td></tr><tr><td rowspan="3">Klebsiella oxytoca</td><td></td><td>2</td><td>1.4%</td><td rowspan="3">Serratia marcescens</td><td>all single</td><td>5</td><td>0.5%</td></tr><tr><td>single multi</td><td>12</td><td>0.2%</td><td></td><td></td><td></td></tr><tr><td>all</td><td></td><td>1.2%</td><td>multi</td><td>12</td><td>1.2%</td></tr><tr><td rowspan="3">Klebsiella pneumoniae</td><td></td><td>30</td><td>3.0%</td><td rowspan="3">Staphylococcus aureus</td><td>all single</td><td>104</td><td>10.2%</td></tr><tr><td>single</td><td>9</td><td>0.9%</td><td></td><td>60</td><td>5.9%</td></tr><tr><td>multi</td><td>21</td><td>2.1%</td><td>multi</td><td>44</td><td>4.3%</td></tr><tr><td rowspan="3">Klebsiella variicola</td><td>all</td><td>2</td><td>0.2%</td><td rowspan="3">Stenotrophomonas maltophilia</td><td>all single</td><td>41</td><td>4.0%</td></tr><tr><td>single</td><td>0</td><td>0.0%</td><td></td><td>12</td><td>1.2%</td></tr><tr><td>multi</td><td>2</td><td>0.2%</td><td>multi</td><td>29</td><td>2.9%</td></tr><tr><td rowspan="3">Legionella pneumophila</td><td>all</td><td>1</td><td>0.1%</td><td rowspan="3">Streptococcus pneumoniae</td><td>all single</td><td>13 7</td><td>1.3% 0.7%</td></tr><tr><td>single multi</td><td>1</td><td>0.1%</td><td>multi</td><td>6</td><td>0.6%</td></tr><tr><td></td><td>0</td><td>0.0%</td><td></td><td></td><td></td></tr></table>

Table 53: Expected values of LRT BAL panel antibiotic resistance markers as determined by the LRT BAL Application for the prospective study.   

<table><tr><td rowspan=2 colspan=1></td><td rowspan=1 colspan=2>Expected Values, AllSpecimens (N = 1,016)</td></tr><tr><td rowspan=1 colspan=1># Spec.</td><td rowspan=1 colspan=1>%</td></tr><tr><td rowspan=1 colspan=1>ctx-M</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>0.9%</td></tr><tr><td rowspan=1 colspan=1>kpc</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>0.4%</td></tr><tr><td rowspan=1 colspan=1>mecA</td><td rowspan=1 colspan=1>47</td><td rowspan=1 colspan=1>4.6%</td></tr><tr><td rowspan=1 colspan=1>ndm</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.1%</td></tr><tr><td rowspan=1 colspan=1>oxa-23</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0.3%</td></tr><tr><td rowspan=1 colspan=1>oxa-24</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>0.4%</td></tr><tr><td rowspan=1 colspan=1>oxa-48</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.1%</td></tr><tr><td rowspan=1 colspan=1>oxa-58</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%</td></tr><tr><td rowspan=1 colspan=1>tem</td><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>1.6%</td></tr><tr><td rowspan=1 colspan=1>vim</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.1%</td></tr></table>

Table 54: Expected values of LRT BAL panel antibiotic resistance markers, stratified by reported hos microorganism.   

<table><tr><td rowspan=3 colspan=1>total reportedmulti host detections</td><td rowspan=1 colspan=1>ctx-M</td><td rowspan=1 colspan=1>kpc</td><td rowspan=1 colspan=1>ndm</td><td rowspan=1 colspan=1>oxa-23</td><td rowspan=1 colspan=1>oxa-24</td><td rowspan=1 colspan=1>oxa-48</td><td rowspan=1 colspan=1>oxa-58</td><td rowspan=1 colspan=1>vim</td><td rowspan=1 colspan=1>tem</td><td rowspan=1 colspan=1>mecA</td></tr><tr><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>47</td></tr><tr><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>Acinetobacter spp.(N =21)</td><td rowspan=1 colspan=1>2/219.5%</td><td rowspan=1 colspan=1>2/219.5%</td><td rowspan=1 colspan=1>1/214.8%</td><td rowspan=1 colspan=1>3/2114.3%</td><td rowspan=1 colspan=1>4/2119.0%</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Citrobacter freundii(N = 4)</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>,</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Enterobacter cloacaecomplex (N = 20)</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>1/205.0%</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Escherichia coli(N = 47)</td><td rowspan=1 colspan=1>8/4717.0%</td><td rowspan=1 colspan=1>1/472.1%</td><td rowspan=1 colspan=1>1/472.1%</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1/472.1%</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Klebsiella oxytoca(N = 14)</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>&#x27;</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Klebsiella pneumoniae(N = 30)</td><td rowspan=1 colspan=1>4/3013.3%</td><td rowspan=1 colspan=1>4/3013.3%</td><td rowspan=1 colspan=1>1/303.3%</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1/303.3%</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Klebsiella variicola(N=2)</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Morganella morganii(N = 3)</td><td rowspan=1 colspan=1>1/333.3%</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Proteus spp.(N = 10)</td><td rowspan=1 colspan=1>2/1020.0%</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1/1010.0%</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Pseudomonasaeruginosa (N = 112)</td><td rowspan=1 colspan=1>7/1126.3%</td><td rowspan=1 colspan=1>2/1121.8%</td><td rowspan=1 colspan=1>1/1120.9%</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1/1120.9%</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Serratia marcescens(N = 17)</td><td rowspan=1 colspan=1>2/1711.8%</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Haemophilus influenzae(N = 56)</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>16/5628.6%</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Staphylococcus aureus(N = 104)</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>47/10445.2%</td></tr></table>

Specimens reported as negative, as single detection, or as multi-detection, together with reported antibiotic resistance markers are listed in Table 55. For example, for eight specimens, LRT BAL reported a single detection of Acinetobacter spp. For four of these specimens $( 5 0 \% )$ , an antibiotic resistance marker indicating a possible carbapenem resistance was simultaneously reported, while for four other specimens, no corresponding antibiotic resistance marker was reported. Similarly, for 60 specimens, LRT BAL reported a single detection of S. aureus. For 32 of these specimens $( 5 3 \% )$ , a mecA was simultaneously reported, while for the remaining 28 specimens, mecA was not reported.

Table 55: Rates of negative, single detection and multi-detection specimens together with reported antibiotic resistance markers as detected by the LRT BAL Application for the prospective study $\mathbf { N } = \mathbf { 1 } , \mathbf { 0 1 6 }$ specimens). Note that resistance marker oxa-58 was not detected in the prospective study cohort.   

<table><tr><td rowspan=1 colspan=12>Antibiotic Resistance Marker Result</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=6>Single Marker Detected</td><td rowspan=1 colspan=5>Multiple Markers Detected</td></tr><tr><td rowspan=1 colspan=1>Microorganism Result</td><td rowspan=1 colspan=1># cases</td><td rowspan=1 colspan=1>none</td><td rowspan=1 colspan=1>tem</td><td rowspan=1 colspan=1>ctx-M</td><td rowspan=1 colspan=1>kpc</td><td rowspan=1 colspan=1>oxa-23</td><td rowspan=1 colspan=1>oxa-24</td><td rowspan=1 colspan=1>mecA</td><td rowspan=1 colspan=1>ctx-M,mecA</td><td rowspan=1 colspan=1>tem,vim</td><td rowspan=1 colspan=1>ctx-M,mecA,oxa-23</td><td rowspan=1 colspan=1>ctx-M,ndm,,oxa-48</td><td rowspan=1 colspan=1>kpc,mecA,oxa-24</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>653(64.3%)</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>Single Organism Detected</td><td rowspan=1 colspan=1>250(24.6%)</td><td rowspan=1 colspan=12></td></tr><tr><td rowspan=1 colspan=1>Acinetobacter spp.</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>−1</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>Citrobacter freundii</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>Enterobacter cloacae complex</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>Escherichia coli</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>Haemophilus influenzae</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>26</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>Klebsiella oxytoca</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>Klebsiella pneumoniae</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>Legionella pneumophila</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>Moraxella catarrhalis</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>Mycoplasma pneumoniae</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>Pneumocystis jirovecii</td><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>Pseudomonas aeruginosa</td><td rowspan=1 colspan=1>61</td><td rowspan=1 colspan=1>61</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>Serratia marcescens</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus aureus</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>32</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>28</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>Stenotrophomonas maltophilia</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>Streptococcus pneumoniae</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>Two Organisms Detected</td><td rowspan=1 colspan=1>78(7.7%)</td><td rowspan=1 colspan=12></td></tr><tr><td rowspan=1 colspan=1>Acinetobacter spp.,Citrobacter freundii</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>Acinetobacter spp., Enterobactercloacae complex</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>Acinetobacter spp.,Klebsiella pneumoniae</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>Acinetobacter spp., Pseudomonasaeruginosa</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Acinetobacter spp.,Stenotrophomonas maltophilia</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>.</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>Citrobacter freundii, Pseudomonasaeruginosa</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>.</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>.</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>Enterobacter cloacae complex,Mycoplasma pneumoniae</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>.</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>.</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>Enterobacter cloacae complex,Staphylococcus aureus</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>Escherichia coli,Haemophilus influenzae</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Escherichia coli,Klebsiella oxytoca</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td></tr></table>

510(k) Summary – LRT BAL Application K191967 Rev 3.0

<table><tr><td rowspan=1 colspan=12>Antibiotic Resistance Marker Result</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=6>Single Marker Detected</td><td rowspan=1 colspan=5>Multiple Markers Detected</td></tr><tr><td rowspan=1 colspan=1>Microorganism Result</td><td rowspan=1 colspan=1># cases</td><td rowspan=1 colspan=1>none</td><td rowspan=1 colspan=1>tem</td><td rowspan=1 colspan=1>ctx-M</td><td rowspan=1 colspan=1>kpc</td><td rowspan=1 colspan=1>oxa-23</td><td rowspan=1 colspan=1>oxa-24</td><td rowspan=1 colspan=1>mecA</td><td rowspan=1 colspan=1>ctx-M,mecA</td><td rowspan=1 colspan=1>tem,vim</td><td rowspan=1 colspan=1>ctx-M,mecA,oxa-23</td><td rowspan=1 colspan=1>ctx-M,ndm,oxa-48</td><td rowspan=1 colspan=1>kpc,mecA,oxa-24</td></tr><tr><td rowspan=1 colspan=1>Escherichia coli,Klebsiella pneumoniae</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>.</td></tr><tr><td rowspan=1 colspan=1>Escherichia coli,Pseudomonas aeruginosa</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>|-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>.</td></tr><tr><td rowspan=1 colspan=1>Escherichia coli,Serratia marcescens</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>−1</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>|-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>.</td></tr><tr><td rowspan=1 colspan=1>Escherichia coli,Staphylococcus aureus</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Haemophilus influenzae,Klebsiella oxytoca</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Haemophilus influenzae,Moraxella catarrhalis</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>.</td></tr><tr><td rowspan=1 colspan=1>Haemophilus influenzae,Mycoplasma pneumoniae</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>Haemophilus influenzae,Pseudomonas aeruginosa</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>Haemophilus influenzae,Staphylococcus aureus</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>Haemophilus influenzae,Streptococcus pneumoniae</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>Klebsiella oxytoca,Klebsiella pneumoniae</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>Klebsiella oxytoca,Proteus spp.</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>Klebsiella oxytoca,Serratia marcescens</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>Klebsiella oxytoca, Staphylococcusaureus</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>.</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>.</td></tr><tr><td rowspan=1 colspan=1>Klebsiella oxytoca,Stenotrophomonas maltophilia</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Klebsiella pneumoniae,Pseudomonas aeruginosa</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>−1</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>.</td></tr><tr><td rowspan=1 colspan=1>Klebsiella pneumoniae,Staphylococcus aureus</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Moraxella catarrhalis,Pneumocystis jirovecii</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>.</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>.</td></tr><tr><td rowspan=1 colspan=1>Moraxella catarrhalis,Streptococcus pneumoniae</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Pneumocystis jirovecii,Pseudomonas aeruginosa</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>.</td></tr><tr><td rowspan=1 colspan=1>Pneumocystis jirovecii,Staphylococcus aureus</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>Pneumocystis jirovecii,Stenotrophomonas maltophilia</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Proteus spp.,Pseudomonas aeruginosa</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>Proteus spp.,Staphylococcus aureus</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>Pseudomonas aeruginosa, Serratiamarcescens</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>Pseudomonas aeruginosa,Staphylococcus aureus</td><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Pseudomonas aeruginosa,Stenotrophomonas maltophilia</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>.</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus aureus</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus aureus,Stenotrophomonas maltophilia</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>.</td></tr><tr><td rowspan=1 colspan=1>Three Organisms Detected</td><td rowspan=1 colspan=1>19(1.9%)</td><td rowspan=1 colspan=12></td></tr><tr><td rowspan=1 colspan=1>Acinetobacter spp.,Klebsiella pneumoniae,Staphylococcus aureus</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>.</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1</td></tr></table>

510(k) Summary – LRT BAL Application K191967 Rev 3.0

<table><tr><td rowspan=2 colspan=2></td><td rowspan=1 colspan=12>Antibiotic Resistance Marker Result</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=6>Single Marker Detected</td><td rowspan=1 colspan=5>Multiple Markers Detected</td><td rowspan=1 colspan=1>|</td></tr><tr><td rowspan=1 colspan=1>Microorganism Result</td><td rowspan=1 colspan=1># cases</td><td rowspan=1 colspan=1>none</td><td rowspan=1 colspan=1>tem</td><td rowspan=1 colspan=1>ctx-M</td><td rowspan=1 colspan=1>kpc</td><td rowspan=1 colspan=1>oxa-23</td><td rowspan=1 colspan=1>oxa-24</td><td rowspan=1 colspan=1>mecA</td><td rowspan=1 colspan=1>ctx-M,mecA</td><td rowspan=1 colspan=1>tem,vim</td><td rowspan=1 colspan=1>ctx-M,mecCA,oxa-23</td><td rowspan=1 colspan=1>ctx-M,dm,oxa-48</td><td rowspan=1 colspan=1>kpc,ecA,oxa-24</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Acinetobacter spp.,Klebsiella pneumoniae,Stenotrophomonas maltophilia</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>−</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Acinetobacter spp., Pseudomonasaeruginosa, Stenotrophomonasmaltophilia</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>.</td><td rowspan=1 colspan=1>−</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>.</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Enterobacter cloacae complex,Haemophilus influenzae,Moraxella catarrhalis</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Enterobacter cloacae complex,Klebsiella oxytoca,Stenotrophomonas maltophilia</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>−</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Enterobacter cloacae complex,Klebsiella pneumoniae, Moraxellacatarrhalis</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>−</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Enterobacter cloacae complex,Klebsiella pneumoniae,Pseudomonas aeruginosa</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>−</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Escherichia coli,Haemophilus influenzae,Staphylococcus aureus</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>−</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Escherichia coli,Morganella morganii,Pseudomonas aeruginosa</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>.</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>−</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Escherichia coli,Pneumocystis jirovecii,Pseudomonas aeruginosa</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>−</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Escherichia coli,Pseudomonas aeruginosa, Serratiamarcescens</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>−</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Escherichia coli,Pseudomonas aeruginosa,Staphylococcus aureus</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>−</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td></td></tr><tr><td rowspan=1 colspan=1>Haemophilus influenzae,Moraxella catarrhalis,Streptococcus pneumoniae</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>.</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>−</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td></td></tr><tr><td rowspan=1 colspan=1>Klebsiella oxytoca, Pseudomonasaeruginosa, Staphylococcusaureus</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>.</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>−</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td></td></tr><tr><td rowspan=1 colspan=1>Klebsiella oxytoca, Pseudomonasaeruginosa, Stenotrophomonasmaltophilia</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>−</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td></td></tr><tr><td rowspan=1 colspan=1>Klebsiella pneumoniae,Pseudomonas aeruginosa,Stenotrophomonas maltophilia</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>−</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td></td></tr><tr><td rowspan=1 colspan=1>Proteus spp.,Pseudomonas aeruginosa,Staphylococcus aureus</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td></td></tr><tr><td rowspan=1 colspan=1>Pseudomonas aeruginosa, Serratiamarcescens, Stenotrophomonasmaltophilia</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td></td></tr><tr><td rowspan=1 colspan=1>Four Organisms Detected</td><td rowspan=1 colspan=1>7(0.7%)</td><td rowspan=1 colspan=12></td><td></td></tr><tr><td rowspan=1 colspan=1>Acinetobacter spp.,Escherichia coli,Klebsiella pneumoniae,Pseudomonas aeruginosa</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>.</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>.</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td></td></tr><tr><td rowspan=1 colspan=1>Acinetobacter spp.,Escherichia coli,Klebsiella pneumoniae,Stenotrophomonas maltophilia</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>.</td><td rowspan=1 colspan=1>.</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td></td></tr><tr><td rowspan=1 colspan=1>Acinetobacter spp.,Escherichia coli,</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>−-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>.</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>−-</td><td></td></tr></table>

<table><tr><td rowspan=1 colspan=12>Antibiotic Resistance Marker Result</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=6>Single Marker Detected</td><td rowspan=1 colspan=5>Multiple Markers Detected</td></tr><tr><td rowspan=1 colspan=1>Microorganism Result</td><td rowspan=1 colspan=1># cases</td><td rowspan=1 colspan=1>none</td><td rowspan=1 colspan=1>tem</td><td rowspan=1 colspan=1>ctx-M</td><td rowspan=1 colspan=1>kpc</td><td rowspan=1 colspan=1>oxa-23</td><td rowspan=1 colspan=1>oxa-24</td><td rowspan=1 colspan=1>mecA</td><td rowspan=1 colspan=1>ctx-M,mecA</td><td rowspan=1 colspan=1>tem,vim</td><td rowspan=1 colspan=1>ctx-M,mecA,oxa-23</td><td rowspan=1 colspan=1>ctx-M,ndm,oxa-48</td><td rowspan=1 colspan=1>kpc,mecA,oxa-24</td></tr><tr><td rowspan=1 colspan=1>Proteus spp.,Pseudomonas aeruginosa</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>→</td><td rowspan=1 colspan=1>→</td><td rowspan=1 colspan=1>→</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Enterobacter cloacae complex,Escherichia coli,Klebsiella oxytoca,Stenotrophomonas maltophilia</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>.</td><td rowspan=1 colspan=1>.</td><td rowspan=1 colspan=1>.</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Escherichia coli,Haemophilus influenzae,Pneumocystis jirovecii,Staphylococcus aureus</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>.</td><td rowspan=1 colspan=1>.</td><td rowspan=1 colspan=1>.</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>.</td></tr><tr><td rowspan=1 colspan=1>Haemophilus influenzae, Proteusspp.,Pseudomonas aeruginosa,Stenotrophomonas maltophilia</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>.</td><td rowspan=1 colspan=1>.</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Pseudomonas aeruginosa, Serratiamarcescens, Staphylococcusaureus, Stenotrophomonasmaltophilia</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>.</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Five Organisms Detected</td><td rowspan=1 colspan=1>7(0.7%)</td><td rowspan=1 colspan=12></td></tr><tr><td rowspan=1 colspan=1>Acinetobacter spp., Enterobactercloacae complex, Haemophilusinfluenzae, Klebsiella pneumoniae,Streptococcus pneumoniae</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>.</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Citrobacter freundii,Escherichia coli,Klebsiella pneumoniae,Pseudomonas aeruginosa, Serratiamarcescens</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Enterobacter cloacae complex,Escherichia coli,Klebsiella oxytoca,Klebsiella variicola,Protus spp.</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Escherichia coli,Klebsiella pneumoniae,Pseudomonas aeruginosa, Serratiamarcescens, Stenotrophomonasmaltophilia</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Escherichia coli,Morganella morganii,Proteus spp.,Pseudomonas aeruginosa,Stenotrophomonas maltophilia</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Haemophilus influenzae,Klebsella pneumoniae, Klebsiellavariicola,Proteus spp.,Staphylococcus aureus</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Haemophilus influenzae,Klebsiella pneumoniae, Moraxellacatarrhalis, Pneumocystisjirovecii,Serratia marcescens</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>.</td><td rowspan=1 colspan=1>.</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Six Organisms Detected</td><td rowspan=1 colspan=1>2(0.2%)</td><td rowspan=1 colspan=7></td><td rowspan=1 colspan=5></td></tr><tr><td rowspan=1 colspan=1>Acinetobacter spp.,Escherichia coli,Klebsiella pneumoniae,Proteus spp.,Pseudomonas aeruginosa,Staphylococcus aureus</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Enterobacter cloacae complex,</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr></table>

510(k) Summary – LRT BAL Application K191967 Rev 3.0

<table><tr><td rowspan=2 colspan=2></td><td rowspan=1 colspan=12>Antibiotic Resistance Marker Result</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=6>Single Marker Detected</td><td rowspan=1 colspan=5>Multiple Markers Detected</td></tr><tr><td rowspan=1 colspan=1>Microorganism Result</td><td rowspan=1 colspan=1># cases</td><td rowspan=1 colspan=1>none</td><td rowspan=1 colspan=1>tem</td><td rowspan=1 colspan=1>ctx-M</td><td rowspan=1 colspan=1>kpc</td><td rowspan=1 colspan=1>oxa-23</td><td rowspan=1 colspan=1>oxa-24</td><td rowspan=1 colspan=1>mecA</td><td rowspan=1 colspan=1>ctx-M,mecA</td><td rowspan=1 colspan=1>tem,vim</td><td rowspan=1 colspan=1>ctx-M,mecA,oxa-23</td><td rowspan=1 colspan=1>ctx-M,ndm,oxa-48</td><td rowspan=1 colspan=1>kpc,mecA,oxa-24</td></tr><tr><td rowspan=1 colspan=1>Haemophilus influenzae,Moraxella catarrhalis, Morganellamorganii, Staphylococcus aureus,Stenotrophomonas maltophilia</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>−</td><td rowspan=1 colspan=1>−</td><td rowspan=1 colspan=1>−</td><td rowspan=1 colspan=1>b</td><td rowspan=1 colspan=1>b</td><td rowspan=1 colspan=1>b</td><td rowspan=1 colspan=1>−</td><td rowspan=1 colspan=1>−</td><td rowspan=1 colspan=1>−</td><td rowspan=1 colspan=1>−</td><td rowspan=1 colspan=1>−</td><td rowspan=1 colspan=1>−</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>1,016</td><td rowspan=1 colspan=1>938</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>44</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td></tr></table>

# 807.92 (b)(3): Conclusions from Nonclinical and Clinical Data

The conclusions drawn from the analytical and clinical data demonstrate that the device is safe and effective for its intended use.