# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATION DECISION SUMMARY ASSAY ONLY

I Background Information:

K191742

# B Applicant

Luminex Corporation

C Proprietary and Established Names

ARIES MRSA Assay

# D Regulatory Information

<table><tr><td rowspan=1 colspan=1>ProductCode(s)</td><td rowspan=1 colspan=1>Classification</td><td rowspan=1 colspan=1>RegulationSection</td><td rowspan=1 colspan=1>Panel</td></tr><tr><td rowspan=1 colspan=1>NQX</td><td rowspan=1 colspan=1>Class II</td><td rowspan=1 colspan=1>21 CFR 866.1640 -AntimicrobialSusceptibility Test Powder</td><td rowspan=1 colspan=1>MI - Microbiology</td></tr></table>

# II Submission/Device Overview:

A Purpose for Submission:

To obtain a Substantial Equivalence Determination for the ARIES MRSA Assay

# B Measurand:

Target DNA sequences for   
(1) mecA and mecC genes for methicillin resistance (2) orfX complex gene of Staphylococcus aureus (3) SCCmec junction region

# C Type of Test:

Qualitative Real Time Polymerase Chain Reaction (PCR)

# III Intended Use/Indications for Use:

# A Intended Use(s):

The ARIES MRSA Assay is an integrated, real-time, polymerase chain reaction (PCR) based qualitative in vitro diagnostic test for the direct detection of methicillin–resistant Staphylococcus aureus (MRSA) DNA from nasal swabs in patients at risk for nasal colonization.

The ARIES MRSA Assay is intended to aid in the prevention and control of MRSA infections in healthcare settings.

The assay is not intended to guide, diagnose, or monitor treatment for MRSA infections. It is not intended to provide results of susceptibility to oxacillin/methicillin. A negative result does not preclude MRSA nasal colonization. Concomitant cultures are necessary to recover organisms for epidemiological typing or for further susceptibility testing.

The ARIES MRSA Assay is indicated for use with ARIES Systems.

B Indication(s) for Use: Same as Intended Use

C Special Conditions for Use Statement(s): Rx - For Prescription Use Only

D Special Instrument Requirements: For use with ARIES Systems

# IV Device/System Characteristics:

# A Device Description:

The ARIES MRSA Assay is a polymerase chain reaction (PCR)-based qualitative in vitro diagnostic test system which consists of the ARIES System or the ARIES M1 System with their included ARIES Software, a sample processing tube, an assay-specific cassette, and an assayspecific protocol file. The ARIES MRSA Assay cassette is a disposable, single-use cassette containing nucleic acid purification reagents, internal sample process control (SPC), and an assay-specific master mix capable of performing the designated assay on one sample. The ARIES MRSA Assay cassette directly detects methicillin-resistant Staphylococcus aureus (MRSA) from nasal swabs in patients at risk of nasal colonization. Specifically, the ARIES MRSA Assay cassette detects the methicillin resistance genes (mecA and mecC), Staphylococcus aureus orfX gene, the SCCmec junction region, and a DNA Sample Processing Control.

Nasal swab specimens are collected using the Liquid Amies Elution Swab (ESwab) Collection and Transport System, or equivalent. A portion of the sample is transferred to the provided 2 mL ARIES MRSA Sample Processing Tube and vortexed. The processed sample is then transferred to the ARIES MRSA Assay cassette.

The specimen is lysed and nucleic acid is extracted using the ARIES System. An extractable sample processing control (SPC) target present in the ARIES MRSA Assay cassette is processed with the specimen. The SPC controls for recovery of extracted nucleic acid, for inhibitory substances and for PCR reagent and instrument integrity. The Ct value of the SPC is designed to verify nucleic acid extraction, to identify PCR inhibition, if any, and verify proper function of the extraction system and real-time instrument.

The extracted nucleic acid and SPC are transferred via magnetic beads through the cassette to the ARIES MRSA Assay lyophilized PCR reagents in the PCR tube that contain primer pairs and probes specific to mecA/mecC, orfX, SCCmec and the SPC sequence. Each probe is labeled with a distinct fluorophore and detected in a distinct channel of an ARIES System. PCR amplification is performed and assay fluorescence is monitored. Hybridization of a fluorescently labeled probe to the amplified target results in the release of quenching and generation of fluorescence signal that is indicative of PCR generated amplicon. Following amplification, the reaction is heated to separate the fluorescent labeled probe from the amplified target, a process that results in a decrease in the fluorescence signal. The reaction fluorescence is measured during this process and the temperature at which the change in fluorescence is the maximum is the $\mathrm { T } _ { \mathrm { m } }$ of the amplicon. The instrument fluorescence output is analyzed and test results are determined using the ARIES System software and the ARIES MRSA Assay protocol and run files. ARIES MRSA Assay results may be reported from the ARIES Software or from the optional SYNCT Software.

# B Principle of Operation:

The ARIES MRSA Assay PCR amplification and detection reagents contain primers and probes specific to the methicillin resistance genes (mecA and mecC), a Staphylococcus aureus complex gene (orfX), the SCCmec region, and the Sample Processing Control (SPC) sequence. Each of the probes are labeled with a distinct fluorophore and detected in distinct channels of the ARIES System. The probes contain a fluorophore on the $_ 5 \cdot$ end of the oligonucleotide sequence and a quencher at the $_ { 3 } ,$ end of the oligonucleotide sequence such that in random coil state when the probe is not hybridized to the target sequence, the fluorescent signal is quenched. PCR amplification is performed and assay fluorescence is monitored. Hybridization of a fluorescently labeled probe to the amplified target results in the release of quenching and generation of fluorescence signal that is indicative of PCR generated amplicon.

# V Substantial Equivalence Information:

A Predicate Device Name(s): Bd Max Mrsa Xt, Bd Max Instrument   
B Predicate 510(k) Number(s): K133605   
C Comparison with Predicate(s):

<table><tr><td colspan="3" rowspan="1">Similarities</td></tr><tr><td colspan="1" rowspan="2">Item</td><td colspan="1" rowspan="1">Device (K191742)</td><td colspan="1" rowspan="1">Predicate (K133605)</td></tr><tr><td colspan="1" rowspan="1">ARIES MRSA Assay</td><td colspan="1" rowspan="1">BD Max MRSA XT</td></tr><tr><td colspan="1" rowspan="1">Intended Use</td><td colspan="1" rowspan="1">The ARIES MRSA Assay is anintegrated, real-time,polymerase chain reaction(PCR) based qualitative in vitrodiagnostic test for the directdetection of methicillin—resistant Staphylococcus aureus(MRSA) DNA from nasalswabs in patients at risk fornasal colonization.The ARIES MRSA Assay isintended to aid in theprevention and control ofMRSA infections in healthcaresettings.The assay is not intended toguide, diagnose, or monitortreatment for MRSA infections.It is not intended to provideresults of susceptibility tooxacillin/methicillin. Anegative result does notpreclude MRSA nasalcolonization. Concomitantcultures are necessary torecover organisms forepidemiological typing or forfurther susceptibility testing.The ARIES MRSA Assay isindicated for use with ARIESSystems.</td><td colspan="1" rowspan="1">The BD MAX MRSA XT assayperformed on the BD MAXSystem is an automatedqualitative in vitro diagnostic testfor the direct detection ofmethicillin-resistantStaphylococcus aureus (MRSA)DNA from nasal swabs in patientsat risk for nasal colonization. Thetest utilizes real-time polymerasechain reaction (PCR) for theamplification of MRSA DNA andfluorogenic target specifichybridization probes for thedetection of the amplified DNA.The BD MAX MRSA XT assayis intended to aid in theprevention and control of MRSAinfections in healthcare settings. Itis not intended to diagnoseMRSA infections nor guide ormonitor treatment for MRSAinfections. A negative result doesnot preclude nasal colonization.Concomitant cultures arenecessary to recover organismsfor epidemiological typing or forfurther susceptibility testing.</td></tr><tr><td colspan="1" rowspan="1">Assay Targets</td><td colspan="1" rowspan="1">(1) mecA and mecC genes formethicillin resistance(2) orfX complex gene of S.aureus(3) SCCmec junction region</td><td colspan="1" rowspan="1">(1) SCCmec/orfX junction area ofmethicillin-resistantStaphylococcus aureus (i.e.,MREJ for SCCmec RightExtremity Junction). The BDMAX MRSA XT assay isdesigned to detect MREJ types i,ii, iii, v, v, vi, ii, ix, xiii, xiv,and xxi; and(2) mecA and mecC genes formethicillin resistance.</td></tr><tr><td colspan="1" rowspan="1">Sample Types</td><td colspan="1" rowspan="1">Nasal Swabs</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Assay Type</td><td colspan="1" rowspan="1">Real-time PCR</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Assay Results</td><td colspan="1" rowspan="1">Qualitative</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Assay Controls</td><td colspan="1" rowspan="1">Sample Processing Control(SPC)</td><td colspan="1" rowspan="1">Specimen Processing Control(SPC)</td></tr><tr><td colspan="3" rowspan="1">Differences</td></tr><tr><td colspan="1" rowspan="2">Item</td><td colspan="1" rowspan="1">Device (K191742)</td><td colspan="1" rowspan="1">Predicate (K133605)</td></tr><tr><td colspan="1" rowspan="1">ARIES MRSA Assay</td><td colspan="1" rowspan="1">BD Max MRSA XT</td></tr><tr><td colspan="1" rowspan="1">Extraction Method</td><td colspan="1" rowspan="1">Automated by the ARIESSystems</td><td colspan="1" rowspan="1">Automated by the BD MAXSystem</td></tr><tr><td colspan="1" rowspan="1">Assay Instrument</td><td colspan="1" rowspan="1">ARIES Systems</td><td colspan="1" rowspan="1">BD MAX System</td></tr><tr><td colspan="1" rowspan="1">Detection</td><td colspan="1" rowspan="1">Fluorescent reporter probes foreach target, and melting curveanalysis</td><td colspan="1" rowspan="1">Fluorogenic target-specifichybridization</td></tr></table>

# VI Standards/Guidance Documents Referenced:

CLSI. Evaluation of Precision of Quantitative Measurement Procedures; Approved Guideline - Third Edition. CLSI document EP05-A3. Wayne, PA: Clinical and Laboratory Standards Institute; 2014.

CLSI. Interference Testing in Clinical Chemistry; Approved Guideline - Second Edition. CLSI document EP07-A2. Wayne, PA: Clinical and Laboratory Standards Institute; 2005.

CLSI. User Protocol for Evaluation of Qualitative Test Performance; Approved Guideline - Second Edition. CLSI document EP12-A2. Wayne, PA: Clinical and Laboratory Standards Institute; 2008.

CLSI. User Verification of Precision and Estimation of Bias; Approved Guideline – Third Edition. CLSI document EP15-A3. Wayne, PA: Clinical and Laboratory Standards Institute; 2014.

CLSI. Evaluation of Detection Capability for Clinical Laboratory Measurement Procedures; Approved Guideline - Second Edition. CLSI document EP17-A2. Wayne, PA: Clinical and Laboratory Standards Institute; 2012.

CLSI. Assessment of the Diagnostic Accuracy of Laboratory Tests Using Receiver Operating Characteristic Curves; Approved Guideline - Second Edition. CLSI document EP24-A2. Wayne, PA: Clinical and Laboratory Standards Institute; 2011.

CLSI. Evaluation of Stability of In Vitro Diagnostic Reagents; Approved Guideline. CLSI document EP25-A. Wayne, PA: Clinical and Laboratory Standards Institute; 2009.

CLSI. Supplemental Tables for Interference Testing in Clinical Chemistry – First Edition. CLSI document EP37. Wayne, PA: Clinical and Laboratory Standards Institute; 2018.

CLSI. Methods for Dilution Antimicrobial Susceptibility Tests for Bacteria that Grow Aerobically – $I I ^ { t h }$ Edition. CLSI document M07. Wayne, PA: Clinical and Laboratory Standards Institute; 2018.

CLSI. Abbreviated Identification of Bacteria and Yeast: Approved Guideline - Second Edition.   
CLSI document M35-A2. Wayne, PA: Clinical and Laboratory Standards Institute; 2008.

CLSI. Performance Standards for Antimicrobial Susceptibility Testing – $2 8 ^ { t h }$ Edition. CLSI document M100. Wayne, PA: Clinical and Laboratory Standards Institute; 2018.

# VII Performance Characteristics (if/when applicable):

# A Analytical Performance:

1. Precision/Reproducibility:

Site-to-Site Reproducibility

The reproducibility of the ARIES MRSA Assay between sites was evaluated in a study performed by two operators at each of three sites over a period of five days. Each operator tested a blinded panel of five samples in triplicate: two methicillin-resistant Staphylococcus aureus (MRSA) strains each spiked at 1x and ${ 5 } \mathrm { x }$ Limit of Detection (LoD) in simulated nasal matrix (SNM) in modified Liquid Amies (LA) (spiked SNM+LA) and a negative sample (unspiked $\mathrm { S N M + L A } )$ using the same lot of reagents (3 sites X 2 operators X 5 days X 3 replicates $= 9 0$ data points per panel member). The results of the study demonstrated acceptable reproducibility from site-to-site at target levels close to the limit of detection (LoD) of the assay. The reproducibility study results are presented in Table 1.

Table 1. Summary of results from the ARIES MRSA Assay Site-to-Site Reproducibility Study, stratified by site and overall   

<table><tr><td rowspan=2 colspan=1>Strain/Level</td><td rowspan=1 colspan=4>Positive/Number (%)</td></tr><tr><td rowspan=1 colspan=1>Site 1</td><td rowspan=1 colspan=1>Site 2</td><td rowspan=1 colspan=1>Site 3</td><td rowspan=1 colspan=1>Overall</td></tr><tr><td rowspan=1 colspan=1>MRSA mecA+Moderate Positive5x LoD1</td><td rowspan=1 colspan=1>30/30(100)</td><td rowspan=1 colspan=1>30/30(100)</td><td rowspan=1 colspan=1>30/30(100)</td><td rowspan=1 colspan=1>90/90(100)</td></tr><tr><td rowspan=1 colspan=1>MRSA mecA+Low Positive1x LoD</td><td rowspan=1 colspan=1>30/30(100)</td><td rowspan=1 colspan=1>30/30(100)</td><td rowspan=1 colspan=1>30/30(100)</td><td rowspan=1 colspan=1>90/90(100)</td></tr><tr><td rowspan=1 colspan=1>MRSA mecC+Moderate Positive5x LoD</td><td rowspan=1 colspan=1>30/30(100)</td><td rowspan=1 colspan=1>30/30(100)</td><td rowspan=1 colspan=1>30/302(100)</td><td rowspan=1 colspan=1>90/90(100)</td></tr><tr><td rowspan=1 colspan=1>MRSA mecC+Low Positive1x LoD</td><td rowspan=1 colspan=1>30/302(100)</td><td rowspan=1 colspan=1>30/30(100)</td><td rowspan=1 colspan=1>30/30(100)</td><td rowspan=1 colspan=1>90/90(100)</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>0/30(0.0)</td><td rowspan=1 colspan=1>0/303(0.0)</td><td rowspan=1 colspan=1>0/30(0.0)</td><td rowspan=1 colspan=1>0/90(0.0)</td></tr></table>

1 LoD; Limit of detection for MRSA Strains BEI NR-46232 and BAA-2313 5x LoD MRSA mecA $^ +$ strain BEI NR-46232: $9 . 7 5 \mathrm { E } \mathrm { + } 0 4$ CFU/mL; 1x LoD MRSA mecA $^ +$ BEI NR46232: $1 . 9 5 \mathrm { E } { + } 0 4$ CFU/mL; 5x LoD MRSA strain mec $C +$ ATCC BAA-2313: $3 . 8 8 \mathrm { E } { + 0 5 }$ CFU/mL; 1x LoD MRSA mec $C +$ BAA-2313: $7 . 7 5 \mathrm { E } { + } 0 4 $ CFU/mL $^ 2 \mathrm { A }$ single sample was reported as Invalid on initial testing; reported as Positive upon repeat $^ 3 \mathrm { A }$ single sample was reported as Invalid on initial testing; reported as Negative upon repeat

# Within Laboratory Precision/Repeatability

Within laboratory precision/repeatability of the ARIES MRSA Assay was evaluated by two operators who tested a panel of samples in triplicate on a single ARIES instrument over a period of 5 days (2 operators X 3 replicates X 5 days $= 3 0$ replicates per panel member). The panel members were the same as those used in the Site-to-Site Reproducibility Study, above. The precision/repeatability study results are presented in Table 2. The results demonstrated acceptable repeatability and precision from day-to-day with target levels close to the LoD of the assay.

Table 2. Summary of results from the Within Laboratory Precision/Repeatability Study for the AIRES MRSA Assay   

<table><tr><td rowspan=1 colspan=1>Strain/Level</td><td rowspan=1 colspan=1>Positive/Tested (%)</td></tr><tr><td rowspan=1 colspan=1>MRSA mecA+Moderate Positive5x LoD1</td><td rowspan=1 colspan=1>30/30(100)</td></tr><tr><td rowspan=1 colspan=1>MRSA mecA+Low Positive1x LoD</td><td rowspan=1 colspan=1>30/30(100)</td></tr><tr><td rowspan=1 colspan=1>MRSA mecC+Moderate Positive5x LoD</td><td rowspan=1 colspan=1>30/302(100)</td></tr><tr><td rowspan=1 colspan=1>MRSA mecC+Low Positive1x LoD</td><td rowspan=1 colspan=1>30/30(100)</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>0/30(0.0)</td></tr></table>

1 LoD; Limit of detection for MRSA Strains BEI NR-46232 and BAA-2313 5x LoD MRSA mecA $^ +$ strain BEI NR-46232: $9 . 7 5 \mathrm { E } \mathrm { + } 0 4$ CFU/mL; 1x LoD MRSA mec $A +$ BEI NR46232: $1 . 9 5 \mathrm { E } { + } 0 4$ CFU/mL; ${ 5 } \mathrm { x }$ LoD MRSA strain mec $C +$ ATCC BAA-2313: $3 . 8 8 \mathrm { E } { + 0 5 }$ CFU/mL; 1x LoD MRSA mec $C +$ BAA-2313: $7 . 7 5 \mathrm { E } { + } 0 4 $ CFU/mL $^ 2 \mathrm { A }$ single sample was reported as Invalid on initial testing; reported as Positive upon repeat

2. Linearity:

Not applicable.

# 3. Analytical Specificity/Interference:

Cross-reactivity Study

The analytical specificity of the ARIES MRSA Assay was evaluated by testing a panel of 99 organisms that may be found in nasal swab specimens (Table 3). Each organism was tested in triplicate in simulated nasal matrix in Modified Liquid Amies at $1 0 ^ { 6 } \mathrm { C F U / m L }$ for non-viral organisms, $1 0 ^ { 5 } \mathrm { T C I D } 5 0 / \mathrm { m L }$ for viruses and $5 ~ { \mu \mathrm { g / m L } }$ for human genomic DNA. No false positive were obtained.

Table 3. Organisms tested for potential cross-reaction in the ARIES MRSA Assay   

<table><tr><td colspan="2" rowspan="1">Non-Staphylococcal organisms</td></tr><tr><td colspan="1" rowspan="1">Acinetobacter baumannii</td><td colspan="1" rowspan="1">Listeria monocytogenes</td></tr><tr><td colspan="1" rowspan="1">Acinetobacter haemolyticus</td><td colspan="1" rowspan="1">Legionella pneumophila</td></tr><tr><td colspan="1" rowspan="1">Bacillus cereus</td><td colspan="1" rowspan="1">Moraxella catarrhalis</td></tr><tr><td colspan="1" rowspan="1">Bordetella pertussis</td><td colspan="1" rowspan="1">Micrococcus luteus</td></tr><tr><td colspan="1" rowspan="1">Candida albicans</td><td colspan="1" rowspan="1">Mycoplasma pneumoniae</td></tr><tr><td colspan="1" rowspan="1">Citrobacter freundii</td><td colspan="1" rowspan="1">Mycobacterium tuberculosis avirulent</td></tr><tr><td colspan="1" rowspan="1">Candida glabrata</td><td colspan="1" rowspan="1">Neisseria meningitidis</td></tr><tr><td colspan="1" rowspan="1">Citrobacter koseri</td><td colspan="1" rowspan="1">Listeria monocytogenes</td></tr><tr><td colspan="1" rowspan="1">Chlamydia pneumoniae</td><td colspan="1" rowspan="1">Pasteurella aerogenes</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium bovis</td><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium flavescens</td><td colspan="1" rowspan="1">Pseudomonas fluorescens1</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium genitalium</td><td colspan="1" rowspan="1">Proteus mirabilis</td></tr><tr><td colspan="1" rowspan="1">Cryptococcus neoformans</td><td colspan="1" rowspan="1">Providencia stuartii</td></tr><tr><td colspan="1" rowspan="1">Enterobacter aerogenes</td><td colspan="1" rowspan="1">Proteus vulgaris</td></tr><tr><td colspan="1" rowspan="1">Enterobacter cloacae1</td><td colspan="1" rowspan="1">Salmonella enterica subsp. Enterica</td></tr><tr><td colspan="1" rowspan="1">Enterococcus faecalis</td><td colspan="1" rowspan="1">Serratia marcescens</td></tr><tr><td colspan="1" rowspan="1">Enterococcus faecium</td><td colspan="1" rowspan="1">Streptococcus agalactiae</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli (0157:H7)</td><td colspan="1" rowspan="1">Streptococcus anginosus</td></tr><tr><td colspan="1" rowspan="1">Enterococcus flavescens</td><td colspan="1" rowspan="1">Streptococcus mitis</td></tr><tr><td colspan="1" rowspan="1">Enterococcus gallinarum</td><td colspan="1" rowspan="1">Streptococcus mutans</td></tr><tr><td colspan="1" rowspan="1">Enterococcus hirae</td><td colspan="1" rowspan="1">Streptococcus pneumoniae</td></tr><tr><td colspan="1" rowspan="1">Haemophilus influenzae</td><td colspan="1" rowspan="1">Shigella sonnei</td></tr><tr><td colspan="1" rowspan="1">Klebsiella oxytoca</td><td colspan="1" rowspan="1">Streptococcus pyogenes</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae (ESBL-producing)</td><td colspan="1" rowspan="1">Streptococcus salivarius</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae (KPC-producing)</td><td colspan="1" rowspan="1">Streptococcus sanguinis</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus crispatus</td><td colspan="1" rowspan="1">Streptococcus suis</td></tr><tr><td></td><td colspan="1" rowspan="1">Yersinia enterocolitica</td></tr><tr><td colspan="1" rowspan="1">Viruses</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Adenovirus Type 40</td><td colspan="1" rowspan="1">Measles virus</td></tr><tr><td colspan="1" rowspan="1">Adenovirus Type 7</td><td colspan="1" rowspan="1">Mumps virus</td></tr><tr><td colspan="1" rowspan="1">Coronavirus 229E</td><td colspan="1" rowspan="1">Parainfluenza 1</td></tr><tr><td colspan="1" rowspan="1">Coronavirus OC43</td><td colspan="1" rowspan="1">Parainfluenza 2</td></tr><tr><td colspan="1" rowspan="1">Cytomegalovirus</td><td colspan="1" rowspan="1">Parainfluenza 3</td></tr><tr><td colspan="1" rowspan="1">Epstein Barr Virus</td><td colspan="1" rowspan="1">Rhinovirus type 1A</td></tr><tr><td colspan="1" rowspan="1">Influenza A</td><td colspan="1" rowspan="1">RSV A</td></tr><tr><td colspan="1" rowspan="1">Influenza B</td><td colspan="1" rowspan="1">RSV B</td></tr><tr><td colspan="1" rowspan="1">Human metapneumovirus</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="2" rowspan="1">Coagulase Negative Staphylococci (CoNS)</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus arlettae</td><td colspan="1" rowspan="1">Staphylococcus equorum</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus captis</td><td colspan="1" rowspan="1">Staphylococcus felis</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus carnosus</td><td colspan="1" rowspan="1">Staphylococcus gallinarum</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus chromogenes</td><td colspan="1" rowspan="1">Staphylococcus haemolyticus Z067</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus epidermidis (255-01B)</td><td colspan="1" rowspan="1">Staphylococcus kloosii</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus epidermidis (RP12 CIP 106510)</td><td colspan="1" rowspan="1">Staphylococcus lentus</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus epidermidis (MRSE;RP62A)</td><td colspan="1" rowspan="1">Staphylococcus pulvereri</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus epidermidis (CCF 15990)</td><td colspan="1" rowspan="1">Staphylococcus simulans Z032</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus epidermidis (CCF 16471)</td><td colspan="1" rowspan="1">Staphylococcus warneri Z113</td></tr><tr><td colspan="1" rowspan="1">Coagulase Positive Staphylococci</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Staphylococcus pseudointermedius</td><td colspan="1" rowspan="1">Staphylococcus delphini</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus scheleiferi subsp. Coagulans</td><td colspan="1" rowspan="1">Staphylococcus intermedius</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus scheleiferi subsp. Schleiferi</td><td colspan="1" rowspan="1">Staphylococcus lutrae</td></tr><tr><td colspan="2" rowspan="1">Methicillin-susceptible Staphylococcus aureus (MSSA)</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus BAA-1749</td><td colspan="1" rowspan="1">Staphylococcus aureus 29213</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus BAA-1765</td><td colspan="1" rowspan="1">Staphylococcus aureus 0801675</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus BAA-1718</td><td colspan="1" rowspan="1">EY</td></tr><tr><td colspan="2" rowspan="1">Other</td></tr><tr><td colspan="1" rowspan="1">Human genomic DNA</td><td colspan="1" rowspan="1">EY</td></tr></table>

1 One replicate was reported as Invalid on initial testing; reported as Negative upon repeat

# Bioinformatic Analysis

In silico analysis was performed to evaluate the potential for cross-reaction of the ARIES MRSA Assay primers and probes. Non-specific amplification and detection of Staphylococcus argenteus by the assay oligos was identified. An investigation into this species indicates high sequence homology between S. argenteus and S. aureus in the orfX, SCCmec and mecA/mecC regions, and that S. argenteus is a member of the S. aureus complex. Due to the high sequence homology in the orfX, SCCmec and mecA/mecC regions between S. argenteus and S. aureus, it would not be possible for the assay oligos to distinguish between methicillin resistant S. argenteus and methicillin resistant $S .$ . aureus.

# Carry-over/Contamination Study

The potential for false-positive results with the ARIES MRSA Assay due to within run or between run cross-contamination was evaluated by testing an alternating series of MRSA “high positive” and negative samples in 20 successive instrument runs using two ARIES instruments. The high positive samples contained MRSA at a concentration of $1 0 ^ { 7 } \mathrm { C F U / m L }$ in simulated nasal matrix in modified Liquid Amies $( { \mathrm { S N M } } \mathrm { + L A } )$ ). Negative samples comprised $\mathrm { S N M + L A }$ alone. One replicate was reported Invalid on initial testing; reported as Positive upon repeat testing. The expected results were obtained for all MRSA positive and negative samples (60/60 each).

# Potentially Interfering Substances Study

The potential for interference with the ARIES MRSA Assay was evaluated with endogenous and exogenous substances that may be present in nasal swab specimens (Table 4). Each substance was tested in triplicate in the presence and absence of one strain of mecA $^ +$ MRSA (NR-46232) and one strain of mec $C +$ MRSA (BAA-2313) at $3 \mathbf { x }$ LoD prepared in simulated nasal matrix in Modified Liquid Amies $( { \mathrm { S N M } } { + } \mathrm { L A } )$ ). No false positives or false negatives were obtained.

Table 4. Substances evaluated for potential interference with the ARIES MRSA Assay   

<table><tr><td colspan="1" rowspan="1">Substance</td><td colspan="1" rowspan="1">Test Concentration</td></tr><tr><td colspan="1" rowspan="1">Whole Blood</td><td colspan="1" rowspan="1">5% (v/v)</td></tr><tr><td colspan="1" rowspan="1">Mucin</td><td colspan="1" rowspan="1">5 mg/mL</td></tr><tr><td colspan="1" rowspan="1">Phenylephrine</td><td colspan="1" rowspan="1">0.03 μg/mL</td></tr><tr><td colspan="1" rowspan="1">Drixoral (Oxymetazoline)</td><td colspan="1" rowspan="1">10% (v/v)</td></tr><tr><td colspan="1" rowspan="1">Benzalkonium chloride</td><td colspan="1" rowspan="1">0.12%</td></tr><tr><td colspan="1" rowspan="1">Propylene glycol</td><td colspan="1" rowspan="1">20% (v/v)</td></tr><tr><td colspan="1" rowspan="1">Sorbitol</td><td colspan="1" rowspan="1">6.45%</td></tr><tr><td colspan="1" rowspan="1">Benzyl alcohol1</td><td colspan="1" rowspan="1">0.5% (v/v)</td></tr><tr><td colspan="1" rowspan="1">Hypromellose</td><td colspan="1" rowspan="1">0.10%</td></tr><tr><td colspan="1" rowspan="1">Phosphoric acid</td><td colspan="1" rowspan="1">1.282 mg/mL</td></tr><tr><td colspan="1" rowspan="1">Beclomethasone</td><td colspan="1" rowspan="1">8.4 μg/mL</td></tr><tr><td colspan="1" rowspan="1">Dexamethasone</td><td colspan="1" rowspan="1">12 μg/mL</td></tr><tr><td colspan="1" rowspan="1">Flunisolide</td><td colspan="1" rowspan="1">5 μg/mL</td></tr><tr><td colspan="1" rowspan="1">Triamcinolone</td><td colspan="1" rowspan="1">22 μg/mL</td></tr><tr><td colspan="1" rowspan="1">Budesonide</td><td colspan="1" rowspan="1">6.30E-03 μg/mL</td></tr><tr><td colspan="1" rowspan="1">Mometasone</td><td colspan="1" rowspan="1">4.50E-04 µg/mL</td></tr><tr><td colspan="1" rowspan="1">Flonase (Fluticasone)</td><td colspan="1" rowspan="1">1.26E-03 μg/mL</td></tr><tr><td colspan="1" rowspan="1">ZICAM2 (Galphimia glauca, Histaminum hydrochloricum)</td><td colspan="1" rowspan="1">10% (v/v)</td></tr><tr><td colspan="1" rowspan="1">Benzocaine</td><td colspan="1" rowspan="1">75 μg/mL</td></tr><tr><td colspan="1" rowspan="1">Menthol</td><td colspan="1" rowspan="1">0.5 mg/mL</td></tr><tr><td colspan="1" rowspan="1">Zanamivir3</td><td colspan="1" rowspan="1">1 mg/mL</td></tr><tr><td colspan="1" rowspan="1">Mupirocin</td><td colspan="1" rowspan="1">50 μg/mL</td></tr><tr><td colspan="1" rowspan="1">Tobramycin</td><td colspan="1" rowspan="1">33 μg/mL</td></tr><tr><td colspan="1" rowspan="1">FluMist (Live intranasal influenza virus vaccine)</td><td colspan="1" rowspan="1">10% (v/v)</td></tr></table>

1 One replicate of SNM+LA negative matrix was reported as Invalid on initial testing; reported as Negative upon repeat 2 One replicate containing mecA $^ +$ MRSA strain was reported as Invalid on initial testing; reported as Positive upon repeat 3 One replicate containing mec $C +$ MRSA strain was reported as Invalid on initial testing; reported as Positive upon repeat

# Microbial Interference Study

The potential for interference with the ARIES MRSA Assay by organisms that may be present in nasal swab specimens was investigated using the same list of species that was evaluated for potential cross-reactivity (refer to Table 3). Testing was performed in triplicate with each potentially interfering species in the presence of one strain of $m e c A { + }$ MRSA (NR46232) or one strain of mec $C +$ MRSA (BAA-2313) at $3 \mathbf { x }$ LoD. The potentially interfering species were tested in triplicate in simulated nasal matrix in Modified Liquid Amies at $1 0 ^ { \bar { 6 } }$ CFU/mL for non-viral organisms, $1 0 ^ { 5 } \mathrm { T C I D } 5 0 / \mathrm { m L }$ for viruses and $5 ~ { \mu \mathrm { g / m L } }$ for human genomic DNA. One replicate containing mecA $^ +$ MRSA was reported as Invalid for $S .$ gallinarum on initial testing but was MRSA positive upon repeat testing. One replicate containing mec $C +$ MRSA strain was reported as Invalid for the specimens containing $A$ . baumannii, Rhinovirus type 1A and S. warneri Z113; however, each specimen was reported as MRSA positive upon repeat testing. These results are acceptable.

# Competitive Interference Study

Competitive interference was tested with methicillin-resistant Staphylococcus aureus (MRSA) at 1x the limit of detection (LoD) and the co-infecting agent, methicillin-susceptible Staphylococcus aureus (MSSA) or methicillin-resistant coagulase-negative Staphylococci (MRCoNS), at increasing concentrations. Each combination was tested in triplicate. The results showed that the ARIES MRSA Assay detected MRSA at 1x LoD in the presence of high concentration of MSSA $- 2 \mathrm { E } { + } 0 4 \mathrm { x }$ LoD) or MRCoNS $( \sim 1 \mathrm { E } { + } 0 5 \mathrm { x }$ LoD). No competitive interference in the ARIES MRSA Assay was observed for co-infections of MRSA with MSSA or MRCoNS.

4. Assay Reportable Range:

Not applicable.

5. Traceability, Stability, Expected Values (Controls, Calibrators, or Methods):

# Specimen Stability

The stability of nasal swabs for use with the ARIES MRSA Assay was evaluated analytically by testing six replicates of each of two methicillin-resistant Staphylococcus aureus (MRSA) strains prepared at $3 \mathbf { x }$ and $1 0 \mathrm { x }$ Limit of Detection (LoD) in Natural Nasal Matrix (NNM) and stored under different conditions. Unseeded negative sample (NNM only) was included to assess the effect of specimen storage on the performance of the Sample Processing Control (SPC). Specimens stored at $2 { \pm } 2 ^ { \circ } \mathrm { C }$ were tested at 5 time points over 10 days and specimens stored at $3 0 { \pm } 1 ^ { \circ } \mathrm { C }$ were tested at 7 time points over 10 days. The results showed that MRSA specimens stored at both temperatures $2 ^ { \circ } \mathrm { C }$ and $3 0 ^ { \circ } \mathrm { C }$ ) generated $100 \%$ expected MRSA positive results across all time points tested. An overall cassette invalid rate of $0 . 5 3 \%$ (4/757) was observed for the testing of MRSA strains and NNM samples in this study. The results of these studies support the stability of nasal swabs for use with the ARIES MRSA Assay when collected using the Liquid Amies Elution Swab (Eswab) Collection and Transport system for up to 10 days at $2 { - } 3 0 ^ { \circ } \mathrm { C }$ .

# Reagent Stability

The shelf-life of the ARIES MRSA Assay cassettes was evaluated in a real-time stability study performed on three lots of reagents that were stored either refrigerated $( 2 { - } 8 ^ { \circ } \mathrm { C } )$ or at room temperature $1 5 { - } 3 0 ^ { \circ } \mathrm { C } )$ . The results from the study support assignment of an expiration date 19 months from the day of manufacture for the assay cassettes when stored under the recommended conditions.

Cassette open box stability evaluated the performance of the ARIES MRSA Assay Cassettes after removal from the cassette pouch and exposed to ambient temperatures, humidity and light for 10 hours. Over the course of 10 hours, all three lots of cassettes produced expected results, showing that ARIES MRSA Assay Cassettes are stable in ambient temperatures for up to 10 hours after they have been removed from the storage pouch.

Sample Processing Control

Each ARIES MRSA Assay cassette contains a Sample Processing Control (SPC) that is designed to verify nucleic acid extraction, and proper reagent, cassette, ARIES System, and assay protocol performance. The SPC has a known melting temperature $\left( \mathrm { { T _ { m } } } \right)$ range and Ct range. Each time an assay is run, the system measures the temperature and fluorescence intensity of the SPC control to ensure the thermal and optical subsystems have remained in calibration.

# External Controls

External controls should be tested according to guidelines or requirements of local, provincial and/or federal regulations or accreditation organizations. A reference methicillinresistant Staphylococcus aureus strain or well characterized methicillin-resistant Staphylococcus aureus clinical isolates may be used as positive controls. The ARIES MRSA Assay Kit does not include external positive or negative controls.

External Positive and Negative Controls were tested on a daily basis during the prospective Clinical Study using a total of five ARIES systems and three ARIES MRSA Assay cassette lots. The Positive External Control comprised a standardized suspension of a strain of MRSA at $1 . 9 5 \mathrm { E } { + 0 5 }$ CFU/mL (10x LoD). The Negative External Control comprised a standardized suspension of a strain of $S .$ . epidermis at $4 . 2 3 \mathrm { E } { + } 0 4$ CFU/mL (10x LoD). On initial testing, 182/183 $( 9 9 . 5 \% )$ Positive and 178/183 $( 9 7 . 3 \% )$ Negative External Controls produced the expected results. Upon repeat testing, all controls produced the expected results.

# 6. Detection Limit:

The Limit of Detection (LoD) of the ARIES MRSA Assay was estimated for two strains of MRSA by testing various dilutions of enumerated cell stocks in natural nasal matrix (NNM). The LoD for each strain was then confirmed by testing a further 20 replicates at the lowest target level that produced $100 \%$ positive results. The LoD was defined as the lowest concentration tested at which $2 9 5 \%$ of assay replicates produced positive results. For MRSA mecA $\cdot ^ { + }$ BAA-2312, the LoD was determined to be $1 . 5 5 \mathrm { { x } } 1 0 ^ { 4 } \mathrm { { C F U / m L } }$ and for MRSA mec $C +$ NRA-46232, it was $7 . 7 5 \mathrm { ~ x ~ } 1 0 ^ { 4 } \mathrm { C F U / m L }$ .

# Inclusivity (Analytical Reactivity)

The inclusivity of the ARIES MRSA Assay was evaluated by testing 55 strains of MRSA in simulated nasal matrix in Modified Liquid Amies $\mathrm { ( S N M + L A ) }$ , which include those tested in the LoD Study. All 55 strains produced 3/3 positive results at 3x LoD $2 . 3 2 \mathrm { ~ x ~ } 1 0 ^ { 5 } \mathrm { C F U / m L }$ for mec $C +$ strains BAA-2312 and BAA-2313; $4 . 6 5 \mathrm { ~ x ~ } 1 0 ^ { 4 } \mathrm { C F U / m L }$ for all other strains). Analytical reactivity study results are presented in Table 5.

Table 5. MRSA strains used to evaluate the inclusivity of the ARIES MRSA Assay   

<table><tr><td colspan="1" rowspan="1">StrainDescription</td><td colspan="1" rowspan="1">Source</td><td colspan="1" rowspan="1">Catalog #</td><td colspan="1" rowspan="1">Strain/Location</td><td colspan="1" rowspan="1">PFGEType</td><td colspan="1" rowspan="1">SCCmecType</td></tr><tr><td colspan="1" rowspan="20">MRSAmecA+</td><td colspan="1" rowspan="1">BEI</td><td colspan="1" rowspan="1">NR-46232(LoD strain)</td><td colspan="1" rowspan="1">NRS703/Minnesota</td><td colspan="1" rowspan="1">USA300</td><td colspan="1" rowspan="1">IV</td></tr><tr><td colspan="1" rowspan="1">ATCC</td><td colspan="1" rowspan="1">BAA-38</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">I</td></tr><tr><td colspan="1" rowspan="1">ATCC</td><td colspan="1" rowspan="1">BAA-1686</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">I</td></tr><tr><td colspan="1" rowspan="1">ATCC</td><td colspan="1" rowspan="1">BAA-16871</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">I</td></tr><tr><td colspan="1" rowspan="1">ATCC</td><td colspan="1" rowspan="1">BAA-1692</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">USA100</td><td colspan="1" rowspan="1">I</td></tr><tr><td colspan="1" rowspan="1">ATCC</td><td colspan="1" rowspan="1">BAA-1681</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">USA100</td><td colspan="1" rowspan="1">I</td></tr><tr><td colspan="1" rowspan="1">ATCC</td><td colspan="1" rowspan="1">BAA-1682</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">USA100</td><td colspan="1" rowspan="1">I</td></tr><tr><td colspan="1" rowspan="1">ATCC</td><td colspan="1" rowspan="1">BAA-1699</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">USA100</td><td colspan="1" rowspan="1">I</td></tr><tr><td colspan="1" rowspan="1">BEI</td><td colspan="1" rowspan="1">NR-46250</td><td colspan="1" rowspan="1">NRS721/Oregon</td><td colspan="1" rowspan="1">USA100</td><td colspan="1" rowspan="1">I</td></tr><tr><td colspan="1" rowspan="1">ATCC</td><td colspan="1" rowspan="1">BAA-1761</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">USA100</td><td colspan="1" rowspan="1">I</td></tr><tr><td colspan="1" rowspan="1">ATCC</td><td colspan="1" rowspan="1">BAA-1750</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">USA200</td><td colspan="1" rowspan="1">I</td></tr><tr><td colspan="1" rowspan="1">BEI</td><td colspan="1" rowspan="1">NR-46251</td><td colspan="1" rowspan="1">NRS722/Oregon</td><td colspan="1" rowspan="1">USA200</td><td colspan="1" rowspan="1">I</td></tr><tr><td colspan="1" rowspan="1">ATCC</td><td colspan="1" rowspan="1">BAA-39</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">III</td></tr><tr><td colspan="1" rowspan="1">ATCC</td><td colspan="1" rowspan="1">BAA-40</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">IIIa</td></tr><tr><td colspan="1" rowspan="1">ATCC</td><td colspan="1" rowspan="1">BAA-1717</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">USA300</td><td colspan="1" rowspan="1">IVa</td></tr><tr><td colspan="1" rowspan="1">ATCC</td><td colspan="1" rowspan="1">BAA-1762</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">USA300</td><td colspan="1" rowspan="1">IVb</td></tr><tr><td colspan="1" rowspan="1">ATCC</td><td colspan="1" rowspan="1">BAA-1680</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">USA300</td><td colspan="1" rowspan="1">IVa</td></tr><tr><td colspan="1" rowspan="1">BEI</td><td colspan="1" rowspan="1">NR-46070</td><td colspan="1" rowspan="1">NRS384/Mississippi</td><td colspan="1" rowspan="1">USA300</td><td colspan="1" rowspan="1">IV</td></tr><tr><td colspan="1" rowspan="1">ATCC</td><td colspan="1" rowspan="1">BAA-1683</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">USA400</td><td colspan="1" rowspan="1">IVa</td></tr><tr><td colspan="1" rowspan="1">ATCC</td><td colspan="1" rowspan="1">BAA-1707</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">USA400</td><td colspan="1" rowspan="1">IV</td></tr><tr><td colspan="1" rowspan="33"></td><td colspan="1" rowspan="1">ATCC</td><td colspan="1" rowspan="1">BAA-1752</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">USA400</td><td colspan="1" rowspan="1">IV</td></tr><tr><td colspan="1" rowspan="1">ATCC</td><td colspan="1" rowspan="1">BAA-1757</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">USA400</td><td colspan="1" rowspan="1">IV</td></tr><tr><td colspan="1" rowspan="1">ATCC</td><td colspan="1" rowspan="1">BAA-1696</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">USA400</td><td colspan="1" rowspan="1">IVa</td></tr><tr><td colspan="1" rowspan="1">BEI</td><td colspan="1" rowspan="1">NR-46207</td><td colspan="1" rowspan="1">NRS678/Connecticut</td><td colspan="1" rowspan="1">USA500</td><td colspan="1" rowspan="1">IV</td></tr><tr><td colspan="1" rowspan="1">ATCC</td><td colspan="1" rowspan="1">BAA-1689</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">USA500</td><td colspan="1" rowspan="1">IV</td></tr><tr><td colspan="1" rowspan="1">BEI</td><td colspan="1" rowspan="1">NR-46220</td><td colspan="1" rowspan="1">NRS691/Georgia</td><td colspan="1" rowspan="1">USA500</td><td colspan="1" rowspan="1">IV</td></tr><tr><td colspan="1" rowspan="1">ATCC</td><td colspan="1" rowspan="1">BAA-1688</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">V</td></tr><tr><td colspan="1" rowspan="1">ATCC</td><td colspan="1" rowspan="1">BAA-42</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">VI</td></tr><tr><td colspan="1" rowspan="1">BEI</td><td colspan="1" rowspan="1">NR-46177</td><td colspan="1" rowspan="1">NRS648/California</td><td colspan="1" rowspan="1">USA600</td><td colspan="1" rowspan="1">I</td></tr><tr><td colspan="1" rowspan="1">ATCC</td><td colspan="1" rowspan="1">BAA-1751</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">USA600</td><td colspan="1" rowspan="1">I</td></tr><tr><td colspan="1" rowspan="1">BEI</td><td colspan="1" rowspan="1">NR-46218</td><td colspan="1" rowspan="1">NRS689/Georgia</td><td colspan="1" rowspan="1">USA700</td><td colspan="1" rowspan="1">IV</td></tr><tr><td colspan="1" rowspan="1">ATCC</td><td colspan="1" rowspan="1">BAA-1766</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">USA700</td><td colspan="1" rowspan="1">V</td></tr><tr><td colspan="1" rowspan="1">BEI</td><td colspan="1" rowspan="1">NR-46221</td><td colspan="1" rowspan="1">NRS692/Georgia</td><td colspan="1" rowspan="1">USA800</td><td colspan="1" rowspan="1">IV</td></tr><tr><td colspan="1" rowspan="1">ATCC</td><td colspan="1" rowspan="1">BAA-1771</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">USA800</td><td colspan="1" rowspan="1">IV</td></tr><tr><td colspan="1" rowspan="1">BEI</td><td colspan="1" rowspan="1">NR-46197</td><td colspan="1" rowspan="1">NRS668/Colorado</td><td colspan="1" rowspan="1">USA800</td><td colspan="1" rowspan="1">N/A</td></tr><tr><td colspan="1" rowspan="1">ATCC</td><td colspan="1" rowspan="1">BAA-1747</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">USA1000</td><td colspan="1" rowspan="1">IV</td></tr><tr><td colspan="1" rowspan="1">ATCC</td><td colspan="1" rowspan="1">BAA-1769</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">USA1000</td><td colspan="1" rowspan="1">IV</td></tr><tr><td colspan="1" rowspan="1">ATCC</td><td colspan="1" rowspan="1">BAA-1764</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">USA1100</td><td colspan="1" rowspan="1">IV</td></tr><tr><td colspan="1" rowspan="1">ATCC</td><td colspan="1" rowspan="1">BAA-1767</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">USA1100</td><td colspan="1" rowspan="1">IV</td></tr><tr><td colspan="1" rowspan="1">ATCC</td><td colspan="1" rowspan="1">33592</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">III</td></tr><tr><td colspan="1" rowspan="1">ATCC</td><td colspan="1" rowspan="1">33593</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">III</td></tr><tr><td colspan="1" rowspan="1">ATCC</td><td colspan="1" rowspan="1">43300</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">I</td></tr><tr><td colspan="1" rowspan="1">ATCC</td><td colspan="1" rowspan="1">700698</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">I</td></tr><tr><td colspan="1" rowspan="1">ATCC</td><td colspan="1" rowspan="1">700699</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">I</td></tr><tr><td colspan="1" rowspan="1">ATCC</td><td colspan="1" rowspan="1">700787</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">I</td></tr><tr><td colspan="1" rowspan="1">ATCC</td><td colspan="1" rowspan="1">700788</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">I</td></tr><tr><td colspan="1" rowspan="1">ATCC</td><td colspan="1" rowspan="1">700789</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">I</td></tr><tr><td colspan="1" rowspan="1">ATCC</td><td colspan="1" rowspan="1">BAA-41</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">I</td></tr><tr><td colspan="1" rowspan="1">ATCC</td><td colspan="1" rowspan="1">BAA-43</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">IIIa</td></tr><tr><td colspan="1" rowspan="1">ATCC</td><td colspan="1" rowspan="1">BAA-44</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">I</td></tr><tr><td colspan="1" rowspan="1">ATCC</td><td colspan="1" rowspan="1">BAA-1720</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">I</td></tr><tr><td colspan="1" rowspan="1">ATCC</td><td colspan="1" rowspan="1">BAA-2094</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">V</td></tr><tr><td colspan="1" rowspan="1">ATCC</td><td colspan="1" rowspan="1">BAA-2096</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">IV</td></tr><tr><td colspan="1" rowspan="2">MRSAmecC+</td><td colspan="1" rowspan="1">ATCC</td><td colspan="1" rowspan="1">BAA-2313(LoD strain)</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">XI</td></tr><tr><td colspan="1" rowspan="1">ATCC</td><td colspan="1" rowspan="1">BAA-2312</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">XI</td></tr></table>

1One replicate was reported as Invalid on initial testing; reported as Positive upon repeat

# Bioinformatic Analysis

The inclusivity of the ARIES MRSA primers and probes for the targeted regions of the genome was analyzed in silico using the Basic Local Alignment Search Tool (BLAST). The region was shown to be well conserved, with nearly $100 \%$ of sequences with $\ge 8 5 \%$ identity for orfX and mecA/mecC and approximately $9 3 \%$ of sequences with $\ge 8 5 \%$ identity for SCCmec junction. These results are acceptable.

# Challenge Study

An additional analytical study was carried out to evaluate the analytical performance of the ARIES MRSA Assay using a panel of challenge strains (Table 6). The challenge panel

included 16 methicillin-resistant Staphylococcus aureus (MRSA) strains with high minimum inhibitory concentration (MIC) values o $\mathrm { \Delta } \mathrm { f } \geq 1 6 ~ \mathrm { \mu g / m L }$ oxacillin and 17 MRSA strains with low MIC values of $\leq 8 ~ \mu \mathrm { g / m L }$ oxacillin, four borderline oxacillin-resistant Staphylococcus aureus (BORSA) strains, 16 empty cassette variants of Staphylococcus aureus strains, and one methicillin-resistant Staphylococcus epidermidis (MRSE) strain. The strains were tested in triplicate at $3 \mathbf { x }$ LoD (MRSA strains) or $1 0 ^ { 6 } \mathrm { C F U / m L }$ (all other strains). Two strains of MRSA with high MIC values and one strain of MRSA with low MIC value did not generate the expected MRSA positive results when tested at $3 \mathbf { x }$ LoD, and were re-tested at 5x LoD and generated the expected MRSA positive results ( $100 \%$ MRSA Positive). The four BORSA strains, the MRSE strain, and all empty cassette variants of Staphylococcus aureus generated the expected $0 \%$ MRSA positive results ( $100 \%$ MRSA Negative). Analytical challenge study results are presented in Table 6.

Table 6. ARIES MRSA Assay Analytical Challenge Study Results   

<table><tr><td></td><td></td><td colspan="1" rowspan="1">tlcal Chalcnge Stt</td><td></td></tr><tr><td colspan="1" rowspan="1">Target Description</td><td colspan="1" rowspan="1">Source</td><td colspan="1" rowspan="1">Strain ID</td><td colspan="1" rowspan="1">ARIES MRSAPositivity (%)</td></tr><tr><td colspan="1" rowspan="16">MRSA withHigh OxacillinMIC</td><td colspan="1" rowspan="1">ARLG</td><td colspan="1" rowspan="1">ARLG-1643</td><td colspan="1" rowspan="1">3/3 (100)</td></tr><tr><td colspan="1" rowspan="1">ARLG</td><td colspan="1" rowspan="1">ARLG-1644</td><td colspan="1" rowspan="1">3/3 (100)</td></tr><tr><td colspan="1" rowspan="1">ARLG</td><td colspan="1" rowspan="1">ARLG-1645</td><td colspan="1" rowspan="1">3/3 (100)</td></tr><tr><td colspan="1" rowspan="1">ARLG</td><td colspan="1" rowspan="1">ARLG-1646</td><td colspan="1" rowspan="1">3/3 (100)</td></tr><tr><td colspan="1" rowspan="1">ARLG</td><td colspan="1" rowspan="1">ARLG-1662</td><td colspan="1" rowspan="1">3/3 (100)</td></tr><tr><td colspan="1" rowspan="1">ARLG</td><td colspan="1" rowspan="1">ARLG-1669</td><td colspan="1" rowspan="1">0/31 (0)</td></tr><tr><td colspan="1" rowspan="1">ARLG</td><td colspan="1" rowspan="1">ARLG-1604</td><td colspan="1" rowspan="1">3/3 (100)</td></tr><tr><td colspan="1" rowspan="1">ARLG</td><td colspan="1" rowspan="1">ARLG-1613</td><td colspan="1" rowspan="1">2/31 (66.7)</td></tr><tr><td colspan="1" rowspan="1">CDC AR Bank</td><td colspan="1" rowspan="1">AR-0215</td><td colspan="1" rowspan="1">3/3 (100)</td></tr><tr><td colspan="1" rowspan="1">CDC AR Bank</td><td colspan="1" rowspan="1">AR-0218</td><td colspan="1" rowspan="1">3/3 (100)</td></tr><tr><td colspan="1" rowspan="1">CDC AR Bank</td><td colspan="1" rowspan="1">AR-0219</td><td colspan="1" rowspan="1">3/3 (100)</td></tr><tr><td colspan="1" rowspan="1">CDC AR Bank</td><td colspan="1" rowspan="1">AR-0220</td><td colspan="1" rowspan="1">3/3 (100)</td></tr><tr><td colspan="1" rowspan="1">CDC AR Bank</td><td colspan="1" rowspan="1">AR-0223</td><td colspan="1" rowspan="1">3/3 (100)</td></tr><tr><td colspan="1" rowspan="1">CDC AR Bank</td><td colspan="1" rowspan="1">AR-0224</td><td colspan="1" rowspan="1">3/3 (100)</td></tr><tr><td colspan="1" rowspan="1">CDC AR Bank</td><td colspan="1" rowspan="1">AR-0227</td><td colspan="1" rowspan="1">3/3 (100)</td></tr><tr><td colspan="1" rowspan="1">CDC AR Bank</td><td colspan="1" rowspan="1">AR-0228</td><td colspan="1" rowspan="1">3/3 (100)</td></tr><tr><td colspan="1" rowspan="17">MRSA withLow Oxacillin MIC</td><td colspan="1" rowspan="1">ARLG</td><td colspan="1" rowspan="1">ARLG-1642</td><td colspan="1" rowspan="1">2/31 (66.7)</td></tr><tr><td colspan="1" rowspan="1">Lyon University</td><td colspan="1" rowspan="1">20130237</td><td colspan="1" rowspan="1">3/3 (100)</td></tr><tr><td colspan="1" rowspan="1">Lyon University</td><td colspan="1" rowspan="1">20130524</td><td colspan="1" rowspan="1">3/3 (100)</td></tr><tr><td colspan="1" rowspan="1">CDC AR Bank</td><td colspan="1" rowspan="1">AR-0216</td><td colspan="1" rowspan="1">3/3 (100)</td></tr><tr><td colspan="1" rowspan="1">CDC AR Bank</td><td colspan="1" rowspan="1">AR-0217</td><td colspan="1" rowspan="1">3/3 (100)</td></tr><tr><td colspan="1" rowspan="1">CDC AR Bank</td><td colspan="1" rowspan="1">AR-0221</td><td colspan="1" rowspan="1">3/3 (100)</td></tr><tr><td colspan="1" rowspan="1">CDC AR Bank</td><td colspan="1" rowspan="1">AR-0225</td><td colspan="1" rowspan="1">3/3 (100)</td></tr><tr><td colspan="1" rowspan="1">CDC AR Bank</td><td colspan="1" rowspan="1">AR-0226</td><td colspan="1" rowspan="1">3/3 (100)</td></tr><tr><td colspan="1" rowspan="1">ATCC</td><td colspan="1" rowspan="1">BAA-1688</td><td colspan="1" rowspan="1">3/3 (100)</td></tr><tr><td colspan="1" rowspan="1">CDC AR Bank</td><td colspan="1" rowspan="1">AR-472</td><td colspan="1" rowspan="1">3/3 (100)</td></tr><tr><td colspan="1" rowspan="1">CDC AR Bank</td><td colspan="1" rowspan="1">AR-473</td><td colspan="1" rowspan="1">3/3 (100)</td></tr><tr><td colspan="1" rowspan="1">CDC AR Bank</td><td colspan="1" rowspan="1">AR-474</td><td colspan="1" rowspan="1">3/3 (100)</td></tr><tr><td colspan="1" rowspan="1">CDC AR Bank</td><td colspan="1" rowspan="1">AR-475</td><td colspan="1" rowspan="1">3/3 (100)</td></tr><tr><td colspan="1" rowspan="1">CDC AR Bank</td><td colspan="1" rowspan="1">AR-476</td><td colspan="1" rowspan="1">3/3 (100)</td></tr><tr><td colspan="1" rowspan="1">CDC AR Bank</td><td colspan="1" rowspan="1">AR-477</td><td colspan="1" rowspan="1">3/3 (100)</td></tr><tr><td colspan="1" rowspan="1">CDC AR Bank</td><td colspan="1" rowspan="1">AR-478</td><td colspan="1" rowspan="1">3/3 (100)</td></tr><tr><td colspan="1" rowspan="1">CDC AR Bank</td><td colspan="1" rowspan="1">AR-479</td><td colspan="1" rowspan="1">3/3 (100)</td></tr><tr><td colspan="1" rowspan="4">BORSA</td><td colspan="1" rowspan="1">CDC AR Bank</td><td colspan="1" rowspan="1">AR-0489</td><td colspan="1" rowspan="1">0/3 (0)</td></tr><tr><td colspan="1" rowspan="1">CDC AR Bank</td><td colspan="1" rowspan="1">AR-0490</td><td colspan="1" rowspan="1">0/3 (0)</td></tr><tr><td colspan="1" rowspan="1">CDC AR Bank</td><td colspan="1" rowspan="1">AR-0491</td><td colspan="1" rowspan="1">0/3 (0)</td></tr><tr><td colspan="1" rowspan="1">CDC AR Bank</td><td colspan="1" rowspan="1">AR-0492</td><td colspan="1" rowspan="1">0/3 (0)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Lyon University</td><td colspan="1" rowspan="1">20101270</td><td colspan="1" rowspan="1">0/3 (0)</td></tr><tr><td colspan="1" rowspan="15">Empty CassetteVariant of SA</td><td colspan="1" rowspan="1">Lyon University</td><td colspan="1" rowspan="1">20112896</td><td colspan="1" rowspan="1">0/3 (0)</td></tr><tr><td colspan="1" rowspan="1">LyonUniversity</td><td colspan="1" rowspan="1">20112911</td><td colspan="1" rowspan="1">0/3 (0)</td></tr><tr><td colspan="1" rowspan="1">LyonUniversity</td><td colspan="1" rowspan="1">20120556</td><td colspan="1" rowspan="1">0/3 (0)</td></tr><tr><td colspan="1" rowspan="1">LyonUniversity</td><td colspan="1" rowspan="1">20120844</td><td colspan="1" rowspan="1">0/3 (0)</td></tr><tr><td colspan="1" rowspan="1">LyonUniversity</td><td colspan="1" rowspan="1">20120871</td><td colspan="1" rowspan="1">0/32 (0)</td></tr><tr><td colspan="1" rowspan="1">LyonUniversity</td><td colspan="1" rowspan="1">20120984</td><td colspan="1" rowspan="1">0/3 (0)</td></tr><tr><td colspan="1" rowspan="1">LyonUniversity</td><td colspan="1" rowspan="1">20121469</td><td colspan="1" rowspan="1">0/3 (0)</td></tr><tr><td colspan="1" rowspan="1">LyonUniversity</td><td colspan="1" rowspan="1">20121544</td><td colspan="1" rowspan="1">0/32 (0)</td></tr><tr><td colspan="1" rowspan="1">LyonUniversity</td><td colspan="1" rowspan="1">20121635</td><td colspan="1" rowspan="1">0/3 (0)</td></tr><tr><td colspan="1" rowspan="1">LyonUniversity</td><td colspan="1" rowspan="1">20121891</td><td colspan="1" rowspan="1">0/3 (0)</td></tr><tr><td colspan="1" rowspan="1">LyonUniversity</td><td colspan="1" rowspan="1">20130769</td><td colspan="1" rowspan="1">0/3 (0)</td></tr><tr><td colspan="1" rowspan="1">LyonUniversity</td><td colspan="1" rowspan="1">20131190</td><td colspan="1" rowspan="1">0/3 (0)</td></tr><tr><td colspan="1" rowspan="1">LyonUniversity</td><td colspan="1" rowspan="1">20131273</td><td colspan="1" rowspan="1">0/3 (0)</td></tr><tr><td colspan="1" rowspan="1">LyonUniversity</td><td colspan="1" rowspan="1">20131727</td><td colspan="1" rowspan="1">0/3 (0)</td></tr><tr><td colspan="1" rowspan="1">LyonUniversity</td><td colspan="1" rowspan="1">20140852</td><td colspan="1" rowspan="1">0/3 (0)</td></tr><tr><td colspan="1" rowspan="1">MRSE</td><td colspan="1" rowspan="1">ATCC</td><td colspan="1" rowspan="1">51625</td><td colspan="1" rowspan="1">0/3 (0)</td></tr></table>

1 Repeat testing at 5x LoD reported as 3/3 $( 1 0 0 \% )$ MRSA Positive 2 One replicate was reported as Invalid on initial testing; reported as Negative upon repeat

# 7. Assay Cut-Off:

For the ARIES MRSA Assay, each target (mecA/mecC, orfX, and SCCmec) has a Ct cut-off, $\mathrm { T } _ { \mathrm { m } }$ window, and $\mathrm { T } _ { \mathrm { m } }$ Peak Threshold. In addition, the internal sample processing control (SPC) also has a corresponding Ct cut-off, $\mathrm { T } _ { \mathrm { m } }$ window, and $\mathrm { T } _ { \mathrm { m } }$ Peak Threshold. Collectively, the cut-off values compose the assay protocol file parameters, which are used to determine the assay result for the detection of the target as Positive, Negative, or Invalid. These values are hard-coded into the ARIES MRSA Assay Protocol File and are not modifiable. The Assay Protocol File parameters were determined, and their performance in the ARIES MRSA Assay were evaluated according to the following general procedure:

Initial Assay Protocol File parameters were set during internal optimization and benchmarking studies. The final Assay Protocol File parameters were then established during internal verification studies using data from optimization, benchmarking and verification. The selected Assay Protocol File parameter values were utilized in the determination of assay performance in the multi-site clinical trial conducted for the ARIES MRSA Assay.

# B Comparison Studies:

1. Method Comparison with Predicate Device: Not applicable.

2. Matrix Comparison:

Comparison of Performance with Natural and Simulated Matrices   
To provide a sufficient quantity of material for testing, a simulated nasal matrix was used for the majority of Analytical Studies. Testing with simulated matrix was performed in   
accordance with the standard assay procedure by transferring $2 0 0 \mu \mathrm { L }$ of the sample to the ARIES MRSA Assay cassette.

The suitability of the simulated matrix for use in analytical testing was evaluated in a comparison study with natural clinical matrix. The two matrices were tested in parallel as part of the LoD Study using MRSA strain NR-46232. The results demonstrated similar analytical sensitivity in both matrices and there were negligible differences in LoD and MRSA targets. The study therefore provided acceptable evidence to support the use of simulated matrix in the Analytical Studies to characterize the performance of the ARIES MRSA Assay.

# Nasal Swab Comparison Study

A nasal swab equivalency study was performed to evaluate the reproducibility of the ARIES MRSA Assay with two different nasal swab types, Regular Nylon Flocked Swab (Copan Catalog Number: 480C) and Flexible Minitip Nylon Flocked Swab (Copan Catalog Number: 482C). The swabs were evaluated using one strain of methicillin-resistant Staphylococcus aureus (MRSA) mecA $^ +$ (NR-46232) at three concentrations, as well as a negative sample (Simulated nasal matrix in Modified Liquid Amies, $\mathrm { S N M + L A }$ ). Samples at intermediate concentrations prepared in $\mathrm { S N M + L A }$ were transferred to Modified Liquid Amies using each of the two nasal swab types to reach the final testing concentrations at $3 \mathbf { x }$ LoD, 5x LoD and 10x LoD, respectively, and then tested on the ARIES MRSA Assay. The test results demonstrated that both swab types generated $100 \%$ expected MRSA positivity for each strain of the concentrations tested. Both swab types also generated $0 \%$ positivity ( $100 \%$ negativity) for negative samples. Swab equivalency study results are presented in Table 7.

Table 7. ARIES MRSA Assay Nasal Swab Equivalency Results   

<table><tr><td rowspan=2 colspan=1>AssayTarget</td><td rowspan=2 colspan=1>PartNumber</td><td rowspan=2 colspan=1>Test Concentration(CFU/mL)</td><td rowspan=1 colspan=2>MRSA Positivity (%)</td></tr><tr><td rowspan=1 colspan=1>Regular swab(Copan 480C)</td><td rowspan=1 colspan=1>Flexible mini-tip swab(Copan 482C)</td></tr><tr><td rowspan=3 colspan=1>MRSAmecA+</td><td rowspan=3 colspan=1>NR-46232</td><td rowspan=1 colspan=1>3x LoD (5.85E+04)</td><td rowspan=1 colspan=1>6/6 (100)</td><td rowspan=1 colspan=1>6/6 (100)</td></tr><tr><td rowspan=1 colspan=1>5x LoD (9.75E+04)</td><td rowspan=1 colspan=1>6/6 (100)</td><td rowspan=1 colspan=1>6/6 (100)</td></tr><tr><td rowspan=1 colspan=1>10x LoD (1.95E+05)</td><td rowspan=1 colspan=1>6/6 (100)</td><td rowspan=1 colspan=1>6/6 (100)</td></tr><tr><td rowspan=1 colspan=1>Negative(SNM+ LA)</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>0/6 (0)</td><td rowspan=1 colspan=1>0/6 (0)</td></tr></table>

# C Clinical Studies:

# 1. Clinical Sensitivity:

Clinical performance of the ARIES MRSA Assay for nasal swab specimens collected from patients at risk for methicillin-resistant Staphylococcus aureus (MRSA) colonization was established through a clinical study.

Performance of the ARIES MRSA Assay was evaluated prospectively from August 2018 to February 2019 at four (4) geographically distinct clinical sites within the United States using the ARIES System. Specimens included in the clinical study consisted of excess leftover deidentified, nasal clinical specimens collected using the Liquid Amies Elution Swab (Eswab) Collection and Transport system, or equivalent, from patients at risk for nasal colonization. All eligible clinical specimens were tested by both the reference method (direct and enriched bacterial culture) and ARIES MRSA Assay and the results compared. Reference method

testing was performed at a centralized testing facility while ARIES MRSA Assay testing was performed at each clinical site on their own clinical specimens.

A total of 2254 nasal swab specimens from subjects at risk for MRSA nasal colonization were collected. Of these 2,254 specimens, 472 were excluded from the study based on inclusion/exclusion criteria leaving a total of 1,782 unique specimens that met the predetermined eligibility criteria for inclusion in the study.

In total, 1,782 specimens were enrolled in the study and tested for methicillin-resistant Staphylococcus aureus by both the reference method and the ARIES MRSA Assay. There were 20 specimens that when tested with ARIES MRSA Assay yielded an invalid result due to run failure or instrument error giving an invalid rate of $1 . 1 \%$ (20/1,782). None of these specimens were re-tested due to insufficient specimen volume.

For the 1,762 eligible specimens that were included in the device performance calculations, clinical sensitivity of the ARIES MRSA Assay against direct and enriched bacterial culture (Table 8), was $9 3 . 3 \%$ (97/104) with a lower bound $9 5 \%$ confidence interval of $8 7 \%$ . Clinical specificity of the ARIES MRSA Assay was $9 3 . 5 \%$ (1550/1658) with a lower bound $9 5 \%$ confidence interval of $92 \%$ . Of the 108 specimens that were MRSA-negative by culture but MRSA-positive by the ARIES MRSA Assay, culture showed that 63 specimens were $S .$ . aureus and 45 were negative (no growth).

When ARIES MRSA Assay was compared to Direct Culture only (Table 9), the positive percent agreement of the ARIES MRSA Assay was $9 3 . 5 \%$ (87/103) with a lower bound $9 5 \%$ confidence interval of $8 7 \%$ . Negative percent agreement of the ARIES MRSA Assay was $9 2 . 9 \%$ (1550/1658) with a lower bound $9 5 \%$ confidence interval of $9 2 \%$ . Overall, performance was determined to be acceptable.

Table 8: ARIES MRSA Assay Performance Compared to Direct and Enriched Culture   

<table><tr><td rowspan=1 colspan=1>ARIES MRSAAssay</td><td rowspan=1 colspan=3>Direct and Enriched Bacterial Culture for MRSA</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>TOTAL</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>97</td><td rowspan=1 colspan=1>108</td><td rowspan=1 colspan=1>205</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>1550</td><td rowspan=1 colspan=1>1557</td></tr><tr><td rowspan=1 colspan=1>TOTAL</td><td rowspan=1 colspan=1>104</td><td rowspan=1 colspan=1>1658</td><td rowspan=1 colspan=1>1762</td></tr><tr><td rowspan=1 colspan=1>Sensitivity (95% CI)</td><td rowspan=1 colspan=3>93.3% (87% - 97%)</td></tr><tr><td rowspan=1 colspan=1>Specificity (95% CI)</td><td rowspan=1 colspan=3>93.5% (92% - 95%)</td></tr></table>

Table 9: ARIES MRSA Assay Performance Compared to Direct Culture (Informational Only)   

<table><tr><td rowspan=1 colspan=1>ARIES MRSA Assay</td><td rowspan=1 colspan=3>Direct Bacterial Culture for MRSA</td></tr><tr><td rowspan=1 colspan=1>canmaGllowb</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>TOTAL</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>87</td><td rowspan=1 colspan=1>118</td><td rowspan=1 colspan=1>205</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>1551</td><td rowspan=1 colspan=1>1557</td></tr><tr><td rowspan=1 colspan=1>TOTAL</td><td rowspan=1 colspan=1>93</td><td rowspan=1 colspan=1>1669</td><td rowspan=1 colspan=1>1762</td></tr><tr><td rowspan=1 colspan=1>Positive Percent Agreement(95% CI)</td><td rowspan=1 colspan=3>93.5% (87% - 97%)</td></tr><tr><td rowspan=1 colspan=1>Negative Percent Agreement(95% CI)</td><td rowspan=1 colspan=3>92.9% (92% - 94%)</td></tr></table>

2. Clinical Specificity: See Clinical Sensitivity above.

3. Other Clinical Supportive Data (When 1. and 2. Are Not Applicable): Not applicable.

# D Clinical Cut-Off:

Not applicable.

# E Expected Values/Reference Range:

The performance of the ARIES MRSA Assay was evaluated in a prospective Clinical Study conducted at four (4) sites in the US. The overall prevalence of MRSA in nasal swab specimens was $5 . 9 \%$ (104/1762) as determined by culture (direct plus enriched) and $1 1 . 6 \%$ (205/1762) as determined by the ARIES assay. In Table 10, the prevalence of MRSA as determined by the ARIES assay is stratified by the age and gender of the subjects.

Table 10. Prevalence of MRSA positive subjects stratified by age and gender   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Number of Subjects</td><td rowspan=1 colspan=1>ARIES MRSAAssay Positive</td><td rowspan=1 colspan=1>% Prevalence1</td></tr><tr><td rowspan=1 colspan=1>Gender</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Male</td><td rowspan=1 colspan=1>954</td><td rowspan=1 colspan=1>122</td><td rowspan=1 colspan=1>12.8</td></tr><tr><td rowspan=1 colspan=1>Female</td><td rowspan=1 colspan=1>808</td><td rowspan=1 colspan=1>83</td><td rowspan=1 colspan=1>10.3</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>1762</td><td rowspan=1 colspan=1>205</td><td rowspan=1 colspan=1>11.6</td></tr><tr><td rowspan=1 colspan=1>Age (yrs)</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>&lt;2</td><td rowspan=1 colspan=1>405</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>5.2</td></tr><tr><td rowspan=1 colspan=1>2-11</td><td rowspan=1 colspan=1>251</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>8.4</td></tr><tr><td rowspan=1 colspan=1>12-21</td><td rowspan=1 colspan=1>189</td><td rowspan=1 colspan=1>19</td><td rowspan=1 colspan=1>10.1</td></tr><tr><td rowspan=1 colspan=1>22-59</td><td rowspan=1 colspan=1>481</td><td rowspan=1 colspan=1>74</td><td rowspan=1 colspan=1>15.4</td></tr><tr><td rowspan=1 colspan=1>≥60</td><td rowspan=1 colspan=1>436</td><td rowspan=1 colspan=1>70</td><td rowspan=1 colspan=1>16.1</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>1762</td><td rowspan=1 colspan=1>205</td><td rowspan=1 colspan=1>11.6</td></tr></table>

1 As determined by the ARIES MRSA Assay

# VIII Proposed Labeling:

The labeling supports the finding of substantial equivalence for this device.

# IX Conclusion:

The submitted information in this premarket notification is complete and supports a substantial equivalence decision.