# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATION DECISION SUMMARY ASSAY ONLY

I Background Information:

A 510(k) Number K191889   
B Applicant NG Biotech   
C Proprietary and Established Names NG-Test CARBA 5   
D Regulatory Information

<table><tr><td rowspan=1 colspan=1>ProductCode(s)</td><td rowspan=1 colspan=1>Classification</td><td rowspan=1 colspan=1>RegulationSection</td><td rowspan=1 colspan=1>Panel</td></tr><tr><td rowspan=1 colspan=1>PTJ</td><td rowspan=1 colspan=1>Class I</td><td rowspan=1 colspan=1>21 CFR 866.1640 -AntimicrobialSusceptibility Test Powder</td><td rowspan=1 colspan=1>MI - Microbiology</td></tr></table>

# II Submission/Device Overview:

A Purpose for Submission:

To obtain a substantial equivalence determination for the qualitative detection of carbapenemase enzymes in carbapenem non-susceptible pure colonies of Enterobacteriaceae and Pseudomonas aeruginosa

# B Measurand:

Carbapenemase enzymes (KPC, OXA-48-like, VIM, IMP, NDM)

# C Type of Test:

Qualitative multiplex immunochromatographic assay (lateral flow)

# III Intended Use/Indications for Use:

A Intended Use(s): See Indications for Use below.

# B Indication(s) for Use:

NG-Test CARBA 5 is an in vitro rapid and visual multiplex immunochromatographic assay for the qualitative detection and differentiation of five common carbapenemases (KPC, OXA-48- like, VIM, IMP and NDM) from carbapenem non-susceptible pure bacterial colonies when grown on the following media:

$5 \%$ sheep blood agar or MacConkey agar (16-24 hours) for testing Enterobacteriaceae and Pseudomonas aeruginosa

HardyCHROM CRE agar (18-24 hours) for testing $E .$ . coli and KES (Klebsiella aerogenes, Klebsiella oxytoca, Klebsiella pneumoniae, Enterobacter cloacae complex and Serratia marcescens).

The NG-Test CARBA 5 is intended as an aid for infection control in the detection of carbapenemase-producing Enterobacteriaceae and Pseudomonas aeruginosa in healthcare settings. NG-Test CARBA 5 is not intended to guide or monitor treatment for carbapenem nonsusceptible bacterial infections. A positive or negative NG-Test CARBA 5 test result does not rule out the presence of other mechanisms of antibiotic resistance. NG-Test CARBA 5 should be used in conjunction with other laboratory tests including phenotypic antimicrobial susceptibility testing.

# C Special Conditions for Use Statement(s):

• Rx - For Prescription Use Only

The performance of NG-Test CARBA 5 was established with colonies from blood agar, MacConkey agar and HardyCHROM CRE agar. Performance with other culture media has not been evaluated and is therefore unknown.

• The performance of NG-Test CARBA 5 with bacteria other than Enterobacteriaceae and Pseudomonas aeruginosa has not been evaluated.

• Organism identification and elevated carbapenem MICs should be determined prior to testing with NG-Test CARBA 5.

D Special Instrument Requirements: None (results are read manually)

# A Device Description:

NG-Test CARBA 5 (Figure 1) is an in vitro rapid and visual multiplex immunochromatographic assay that detects one or more of the five common types of carbapenemase enzymes (KPC, OXA-48-like, IMP, VIM, NDM) in bacterial colonies of species usually associated with harboring these resistance markers. The assay consists of a sample port, sample pad and nitrocellulose test strip which are contained within a plastic cassette, in addition to reagents for liquid extraction.

![](images/e74de5aa61d1c05ebd8fdadef07e3acebbcf35ecbc883a6adad8659dd7ec5d54.jpg)  
Figure 1. NG-Test CABA 5

# B Principle of Operation:

Monoclonal antibodies that individually recognize each of the five carbapenemases are immobilized on a nitrocellulose membrane. Free monoclonal antibodies are present in the sample pad and labelled with colloidal gold. Colonies are mixed with extraction buffer to lyse the bacteria then added to the sample pad. Capillary action of the nitrocellulose draws the sample through the mobile antibodies and immobile antibodies on the test strip. The immobilized control antibodies capture any mobile antibodies that run through the sample pad and nitrocellulose without binding to other test lines. After addition of the bacterial suspension to the sample port, a result can be read after 15 minutes. Possible results include Positive (for one or more target), Negative or Invalid. A positive result occurs when a red line appears on the control region (marked “C”) and one or more lines appear in the test regions (marked “K”, “O”, “V”, “I”, or “N”). This result indicates that the sample contains one or more carbapenemases. A negative result occurs when only the control line is observed and indicates that the sample does not contain any of the 5 carbapenemases. If the control line does not appear, the test result is invalid.

# V Substantial Equivalence Information:

A Predicate Device Name(s): Rapidec Carba Np   
B Predicate 510(k) Number(s): K162385   
C Comparison with Predicate(s):

<table><tr><td colspan="1" rowspan="1">Device &amp; Predicate Device(s):</td><td colspan="1" rowspan="1">Device:K191889</td><td colspan="1" rowspan="1">Predicate:K162385</td></tr><tr><td colspan="1" rowspan="1">Device Trade Name</td><td colspan="1" rowspan="1">NG-Test CARBA 5</td><td colspan="1" rowspan="1">RAPIDEC CARBA NP</td></tr><tr><td colspan="3" rowspan="1">General Device Characteristic Similarities</td></tr><tr><td colspan="1" rowspan="1">Intended Use/Indications forUse</td><td colspan="1" rowspan="1">NG-Test CARBA 5 is an in vitrorapid and visual multipleximmunochromatographic assayfor the qualitative detection anddifferentiation of five commoncarbapenemases (KPC, OXA-48-like, VIM, IMP and NDM) fromcarbapenem non-susceptible purebacterial colonies when grown onthe following media:5% sheep blood agar orMacConkey agar (16-24hours) for testingEnterobacteriaceae andPseudomonas aeruginosaHardyCHROM CRE agar(18-24 hours) for testing E.coli and KES (Klebsiellaaerogenes, Klebsiellaoxytoca, Klebsiellapneumoniae, Enterobactercloacae complex and Serratiamarcescens).The NG-Test CARBA 5 isintended as an aid for infectioncontrol in the detection ofcarbapenemase-producingEnterobacteriaceae andPseudomonas aeruginosa inhealthcare settings. NG-TestCARBA 5 is not intended toguide or monitor treatment forcarbapenem non-susceptiblebacterial infections. A positive ornegative NG-Test CARBA 5 testresult does not rule out thepresence of other mechanisms ofantibiotic resistance. NG-TestCARBA 5 should be used inconjunction with other laboratorytests including phenotypicantimicrobial susceptibilitytesting.</td><td colspan="1" rowspan="1">RAPIDEC CARBA NP is aphenotypic (colorimetric) invitro diagnostic test for thequalitative detection ofcarbapenemase enzymes inEnterobacteriaceae andPseudomonas aeruginosacolonies that have elevatedMIC values to any carbapenem.RAPIDEC CARBA NP isperformed on pure coloniesgrown on non-selective sheepblood agar culture media.RAPIDEC CARBA NP isintended as an aid in theprevention and control ofinfection caused bycarbapenemase-producingEnterobacteriaceae andPseudomonas aeruginosa.RAPIDEC CARBA NP is notintended to guide or monitorthe treatment for these bacterialinfections. A negative resultdoes not preclude the presenceof carbapenemase enzymes.The ability of RAPIDECCARBA NP to detectcarbapenemase enzymesencoded by genetic markersother than KPC, NDM, OXA-48, VIM, and IMP has not beenestablished.RAPIDEC CARBA NP testingshould be used in conjunctionwith other laboratory testsincluding antimicrobialsusceptibility testing.</td></tr><tr><td colspan="1" rowspan="1">Inoculum Preparation</td><td colspan="1" rowspan="1">Touching three well-isolatedcolonies with a loop</td><td colspan="1" rowspan="1">Touching well-isolatedcolonies with an applicatorstick</td></tr><tr><td colspan="1" rowspan="1">Sample Type</td><td colspan="1" rowspan="1">Bacterial colonies</td><td colspan="1" rowspan="1">same</td></tr><tr><td colspan="1" rowspan="1">Interpretation</td><td colspan="1" rowspan="1">Visual (manual)</td><td colspan="1" rowspan="1">same</td></tr><tr><td colspan="1" rowspan="1">Controls</td><td colspan="1" rowspan="1">Built-in procedural control inevery test strip</td><td colspan="1" rowspan="1">same</td></tr><tr><td colspan="3" rowspan="1">General Device Characteristic Differences</td></tr><tr><td colspan="1" rowspan="1">Intended Culture Media</td><td colspan="1" rowspan="1">5% sheep blood agar, MacConkeyagar, HardyCHROM CRE agar</td><td colspan="1" rowspan="1">Non-selective sheep blood agar</td></tr></table>

# VI Standards/Guidance Documents Referenced:

CLSI M100, 28th ed., "Performance Standards for Antimicrobial Susceptibility Testing;   
Twenty-Eighth Informational Supplement, January 2018".

VII Performance Characteristics (if/when applicable):

A Analytical Performance:

# 1. Precision/Reproducibility:

The reproducibility of NG-Test CARBA 5 was evaluated in a study conducted with four lots of tests at three sites. The reproducibility panel of organisms comprised eight wellcharacterized strains of Enterobacteriaceae and $P .$ . aeruginosa that produce at least one of the targeted carbapenemases (Table 1.1) and two negative control organisms (Table 1.2).

Two of the eight carbapenemase-positive organisms co-produced two carbapenemases so that a total of ten positive test results (two for each of the five carbapenemase targets) was expected per panel. Each panel member was sub-cultured, in duplicate, onto blood agar or MacConkey agar and incubated at $3 5 ^ { \circ } \mathrm { C }$ for 16 to 24 hours. After growth, colonies from each plate were tested each day by two operators over five days at each site (3 sites x 2 operators x 5 days $_ { \textrm { X 2 } }$ targets $_ { \textrm { X 2 } }$ replicates $= 1 2 0$ positive samples per target carbapenemase per medium type).

Table 1.1. Strains Used in Reproducibility Study: Carbapenemase-positive Organisms   

<table><tr><td rowspan=2 colspan=1>Organism</td><td rowspan=2 colspan=1>Carbapenemase</td><td rowspan=1 colspan=3>Carbapenem Phenotype</td><td rowspan=2 colspan=1>ExpectedPositive Result</td><td rowspan=2 colspan=1>Expected # ofNegative Results</td></tr><tr><td rowspan=1 colspan=1>ETP</td><td rowspan=1 colspan=1>IPM</td><td rowspan=1 colspan=1>MEM</td></tr><tr><td rowspan=1 colspan=1>Pseudomonas aeruginosaCDC 0441</td><td rowspan=1 colspan=1>KPC</td><td rowspan=1 colspan=1>R1</td><td rowspan=1 colspan=1>R</td><td rowspan=1 colspan=1>R</td><td rowspan=1 colspan=1>KPC [K]</td><td rowspan=1 colspan=1>4</td></tr><tr><td rowspan=1 colspan=1>Escherichia coliCDC 0150</td><td rowspan=1 colspan=1>NDM-5</td><td rowspan=1 colspan=1>R</td><td rowspan=1 colspan=1>R</td><td rowspan=1 colspan=1>R</td><td rowspan=1 colspan=1>NDM [N]</td><td rowspan=1 colspan=1>4</td></tr><tr><td rowspan=1 colspan=1>Klebsiella pneumoniaeIHMA 1035778</td><td rowspan=1 colspan=1>IMP-4</td><td rowspan=1 colspan=1>I</td><td rowspan=1 colspan=1>R</td><td rowspan=1 colspan=1>R</td><td rowspan=1 colspan=1>IMP [1]</td><td rowspan=1 colspan=1>4</td></tr><tr><td rowspan=1 colspan=1>Enterobacter cloacaeIHMA 889980</td><td rowspan=1 colspan=1>OXA-48, VIM-31</td><td rowspan=1 colspan=1>I</td><td rowspan=1 colspan=1>R</td><td rowspan=1 colspan=1>S</td><td rowspan=1 colspan=1>OXA [O],VIM [V]</td><td rowspan=1 colspan=1>3</td></tr><tr><td rowspan=1 colspan=1>Enterobacter cloacaeJMI 325859</td><td rowspan=1 colspan=1>KPC-4</td><td rowspan=1 colspan=1>R</td><td rowspan=1 colspan=1>R</td><td rowspan=1 colspan=1>R</td><td rowspan=1 colspan=1>KPC [K]</td><td rowspan=1 colspan=1>4</td></tr><tr><td rowspan=1 colspan=1>Pseudomonas aeruginosaCDC 0444</td><td rowspan=1 colspan=1>VIM</td><td rowspan=1 colspan=1>R1</td><td rowspan=1 colspan=1>R</td><td rowspan=1 colspan=1>R</td><td rowspan=1 colspan=1>VIM [V]</td><td rowspan=1 colspan=1>4</td></tr><tr><td rowspan=1 colspan=1>Klebsiella pneumoniaeCDC 0153</td><td rowspan=1 colspan=1>NDM, OXA-232</td><td rowspan=1 colspan=1>R</td><td rowspan=1 colspan=1>R</td><td rowspan=1 colspan=1>I</td><td rowspan=1 colspan=1>NDM [N],OXA [0]</td><td rowspan=1 colspan=1>3</td></tr><tr><td rowspan=1 colspan=1>Pseudomonas aeruginosaIHMA 855945</td><td rowspan=1 colspan=1>IMP-26</td><td rowspan=1 colspan=1>R1</td><td rowspan=1 colspan=1>R</td><td rowspan=1 colspan=1>R</td><td rowspan=1 colspan=1>IMP [1]</td><td rowspan=1 colspan=1>4</td></tr></table>

ETP: ertapenem, IPM: imipenem, MEM: meropenem S: susceptible, I: intermediate, R: resistant Intrinsically resistant to ertapenem

The design of the strip also allows for interpretation of negative results, resulting in multiple expected negative results per strip. Both carbapenemase-negative organisms were expected to generate five negative results per test strip (Table 1.2). The six carbapenemase-positive organisms that produced a single carbapenemase were expected to produce four negative results (Table 1.1). The two carbapenemase-positive organisms that co-produced two carbapenemases were expected to generate three negative results (Table 1.1). Therefore, a total of 2400 negative results were expected from all organisms combined:

(2 carbapenemase-negative organisms x 5 negative tests) $^ +$ (6 carbapenemase-positive organisms $\mathtt { x 4 }$ negative results) $^ +$ (2 carbapenemase-positive organisms $\textbf { X } 3$ negative results)] $_ { \textrm { X 3 } }$ sites $_ { \textrm { X 2 } }$ operators $\texttt { X 5 }$ days $_ { \textrm { X 2 } }$ replicates $= 2 4 0 0$ negative results.

Table 1.2. Strains Used in Reproducibility Study: Carbapenemase-negative Organisms   

<table><tr><td rowspan=2 colspan=1>Organism</td><td rowspan=2 colspan=1>Carbapenemase</td><td rowspan=1 colspan=3>Carbapenem Phenotype</td><td rowspan=2 colspan=1>Expected # ofNegative Results</td></tr><tr><td rowspan=1 colspan=1>ETP</td><td rowspan=1 colspan=1>IPM</td><td rowspan=1 colspan=1>MEM</td></tr><tr><td rowspan=1 colspan=1>Escherichia coliATCC 25922</td><td rowspan=1 colspan=1>None</td><td rowspan=1 colspan=1>S</td><td rowspan=1 colspan=1>S</td><td rowspan=1 colspan=1>S</td><td rowspan=1 colspan=1>5</td></tr><tr><td rowspan=1 colspan=1>Pseudomonas aeruginosaCDC 0353</td><td rowspan=1 colspan=1>GES-1 2</td><td rowspan=1 colspan=1>R1</td><td rowspan=1 colspan=1>R</td><td rowspan=1 colspan=1>R</td><td rowspan=1 colspan=1>5</td></tr></table>

ETP: ertapenem, IPM: imipenem, MEM: meropenem S: susceptible, I: intermediate, R: resistant Intrinsically resistant to ertapenem 2 The organism harbors GES-1, which is not a target and is not detected with the NG-Test CARBA 5

All positive and negative results for each targeted carbapenemase were as expected (i.e., 120/120 $[ 1 0 0 \% ]$ positive results for each carbapenemase target and 2400/2400 $[ 1 0 0 \% ]$ negative results). The reproducibility of NG-Test CARBA 5 was determined to be acceptable.

2. Linearity: Not applicable

# 3. Analytical Specificity/Cross Reactivity:

The analytical specificity of the NG-Test CARBA 5 was evaluated using organisms that were carbapenem-susceptible or carbapenem non-susceptible with antibiotic resistance mechanisms other than those targeted by the NG-Test CARBA 5. A panel of 81 isolates were tested after growth on blood agar and MacConkey agar (54 Enterobacteriaceae, 20 Pseudomonas aeruginosa and 7 phylogenetically related organisms). A panel of 16 select Enterobacteriaceae (i.e., E. coli and KES [Klebsiella aerogenes, Klebsiella oxytoca, Klebsiella pneumoniae, Enterobacter cloacae complex and Serratia marcescens]) isolates were tested after being seeded in raw stool or stool in C&S Cary Blair Transport Media, followed by sub-culture and growth on HardyCHROM CRE agar.

The organism groups and their relevant molecular characteristics used in the analytical crossreactivity study is provided in Table 2. All organisms tested from blood and MacConkey agar (81/81, $100 \%$ and from HardyCHROM CRE agar (16/16, $100 \%$ yielded a negative NG-Test CARBA 5 result. The cross-reactivity of the NG-Test CARBA 5 was determined to be acceptable.

Table 2. Resistant mechanisms evaluated for Cross-Reactivity   

<table><tr><td rowspan=2 colspan=1>Organism Group</td><td rowspan=1 colspan=2>Resistant mechanisms evaluated</td></tr><tr><td rowspan=1 colspan=1>Blood &amp; MacConkey agar</td><td rowspan=1 colspan=1>HardyCHROM CRE agar</td></tr><tr><td rowspan=1 colspan=1>Enterobacteriaceae</td><td rowspan=1 colspan=1>ACT-type, ACT-2, AmpC,CTX-M [1, 3, 8, 9, 14, 15,22, 24, 30, 40, 55, 74, 75, 79,124], DHA-1, ESBL, IMI,mrc-1, OmpK35, OmpK37,OXA [1, 2, 30], SHV[11(2b), 12(2be), 18, 28, 31,89(2b), 108(u), 154, 179(u),180(u), 182(u), OSBL(2b)],SME, SME-2, TEM [1,1(2b), 11(2be), 63(2be),93(2be), 210(u), OSBL(2b)],tet(A), tet(B)</td><td rowspan=1 colspan=1>ACT-2, AmpC, CTX-M[9, 14, 30], DHA-1, IMI,MIR-8, OXA, SME, TEM-129(2be), tet(A)</td></tr><tr><td rowspan=1 colspan=1>Pseudomonas aeruginosa</td><td rowspan=1 colspan=1>aadA6, aadB, aph(3&#x27;)-IIb,catB7, GES-1, GES-5(c),OXA [10, 50], PA0, PDC [1,5, 19, 35], PER-1, strA, strB,sulI, tet(c), VEB-1, inducibleAmpC</td><td rowspan=1 colspan=1>N/A</td></tr><tr><td rowspan=1 colspan=1>Other</td><td rowspan=1 colspan=1>VanA</td><td rowspan=1 colspan=1>N/A</td></tr></table>

N/A: not applicable, HardyCHROM CRE agar is only intended for use with $E$ . coli and KES organisms.

4. Assay Reportable Range: Not applicable

The ability of the NG-Test CARBA 5 to consistently provide correct results over the range of recommended incubation times was evaluated using carbapenem nonsusceptible isolates previously characterized to have a target carbapenemase. Twenty-two strains were tested after growth on blood and MacConkey agar every two hours from 16 – 24 hours of incubation. Fifteen of the 22 organisms were also tested after growth on HardyCHROM CRE every two hours from 18 – 24 hours of incubation. All organisms tested produced the expected result on NG-Test CARBA 5 at each time point tested.

# b) Refrigerated Storage Study

To evaluate whether organism cultures (plated on agar media) stored in the refrigerator could be used with NG-Test CARBA 5, strains were evaluated each day for three days after refrigeration. Twelve strains were first tested after growth on blood and MacConkey agar. Ten of the 12 strains were also tested after growth on HardyCHROM CRE agar. All organisms tested produced the expected result on NG-Test CARBA 5 for each day of refrigeration.

# c) Quality Control

Quality control testing was performed each day of the clinical study. The QC panel consisted of one positive control organism for each target carbapenemase and one negative control organism (Table 3). The QC testing gave the expected results each day of testing.

Table 3. QC Strains used for NG-Test CARBA 5 evaluation   

<table><tr><td rowspan=1 colspan=1>QC Strain</td><td rowspan=1 colspan=1>Expected Results 1</td></tr><tr><td rowspan=1 colspan=1>Klebsiella pneumoniae ATCC BAA-1705 2</td><td rowspan=1 colspan=1>Positive KPC (K) Line</td></tr><tr><td rowspan=1 colspan=1>Klebsiella pneumoniae NCTC13442</td><td rowspan=1 colspan=1>PositiveOXA-48 (O) Line</td></tr><tr><td rowspan=1 colspan=1>Klebsiella pneumoniae NCTC 13439</td><td rowspan=1 colspan=1>PositiveVIM (V) Line</td></tr><tr><td rowspan=1 colspan=1>Escherichia coli NCTC 13476</td><td rowspan=1 colspan=1>Positivee IMP (I) Line</td></tr><tr><td rowspan=1 colspan=1>Klebsiella pneumoniae ATCC BAA-2146</td><td rowspan=1 colspan=1>Positive NDM (N) Line</td></tr><tr><td rowspan=1 colspan=1>Klebsiella pneumoniae ATCC BAA-1706</td><td rowspan=1 colspan=1>No Positive Test Lines</td></tr></table>

1 Positive Control Line (C) is expected for results to be considered valid. 2 According to CLSI document M100, Klebsiella pneumoniae ATCC BAA-1705 may undergo spontaneous loss of the plasmid encoding the carbapenemase, leading to false-negative QC results. This strain should be maintained on a carbapenem-containing medium or with a selective antimicrobial disk on non-selective agar prior to QC testing.

# 6. Detection Limit:

Analytical Reactivity/Inclusivity The analytical reactivity (inclusivity) of NG-Test CARBA 5 was evaluated using a panel of 92 isolates previously characterized to have a targeted carbapenemase (Table 4): 66 Enterobacteriaceae isolates (expected to provide 68 positive results) and $2 6 P .$ . aeruginosa isolates (expected to provide 26 positive results). All 92 isolates were tested in triplicate after growth on blood agar and MacConkey agar. Forty-two select Enterobacteriaceae (i.e., $E .$ . coli and KES) isolates of the panel (expected to provide 44 positive results) were also tested in triplicate after being seeded in raw stool or stool in C&S Cary Blair Transport Media and cultured on HardyCHROM CRE agar. Organisms that yielded a negative NG-Test CARBA 5 result were further analyzed by modified Carbapenemase Inactivation Method (mCIM, as described in CLSI M100, $2 9 ^ { \mathrm { t h } }$ Edition). Negative NG-Test CARBA 5 results that yielded negative mCIM results were determined to be true negatives. Negative NG-Test CARBA 5 results that yielded positive mCIM results were determined to be false negatives.

Table 4. Targets and variants evaluated in Analytical Reactivity Study   

<table><tr><td rowspan=1 colspan=1>Organism Group</td><td rowspan=1 colspan=1>TargetDetected</td><td rowspan=1 colspan=1>Number oftargetstested onBlood &amp;MacConkey</td><td rowspan=1 colspan=2>Number oftargets testedon HardyCHROM CRE</td><td rowspan=1 colspan=1>Variants tested</td><td rowspan=1 colspan=1>Variantsnotdetected</td></tr><tr><td rowspan=7 colspan=1>Enterobacteriaceae</td><td rowspan=1 colspan=1>KPC</td><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=2>8</td><td rowspan=1 colspan=1>2, 3, 4, 6, 12</td><td rowspan=10 colspan=1></td></tr><tr><td rowspan=1 colspan=1>OXA-48-like</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=2>7</td><td rowspan=1 colspan=1>48, 181, 163, 232 (48 type)</td></tr><tr><td rowspan=1 colspan=1>VIM</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=2>9</td><td rowspan=1 colspan=1>1, 4, 5, 6, 23, 27, 31</td></tr><tr><td rowspan=1 colspan=1>IMP</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=2>7</td><td rowspan=1 colspan=1>4, 8/471, 262,</td></tr><tr><td rowspan=1 colspan=1>NDM</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=2>11</td><td rowspan=1 colspan=1>12, 5, 6, 7</td></tr><tr><td rowspan=1 colspan=1>none 3</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=2>2</td><td rowspan=2 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>68</td><td rowspan=1 colspan=2>44</td></tr><tr><td rowspan=7 colspan=1>Pseudomonasaeruginosa</td><td rowspan=1 colspan=1>KPC</td><td rowspan=1 colspan=1>5</td><td rowspan=3 colspan=2></td><td rowspan=1 colspan=1>2,5</td></tr><tr><td rowspan=1 colspan=1>OXA-48-like</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>VIM</td><td rowspan=1 colspan=1>13</td><td rowspan=2 colspan=1></td><td rowspan=1 colspan=1>2, 11</td></tr><tr><td rowspan=1 colspan=1>IMP</td><td rowspan=1 colspan=1>6</td><td rowspan=4 colspan=2></td><td rowspan=1 colspan=1>1, 7, 14, 18, 19, 26</td><td rowspan=1 colspan=1>14, 18</td></tr><tr><td rowspan=1 colspan=1>NDM</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>1</td><td rowspan=3 colspan=1></td></tr><tr><td rowspan=1 colspan=1>none 3</td><td rowspan=1 colspan=1>0</td><td rowspan=2 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>26</td></tr></table>

IMP-8 and IMP-47 determined to be the same protein based on sequence analysis by the Beta-Lactamase Database. 2 Proteus mirabilis targets not detected from blood agar but detected from MacConkey agar. 3 Isolates have targeted carbapenemase resistance genes but were negative by mCIM making them true negatives for carbapenemase production. They were also negative by NG-Test CARBA 5.

A summary of the analytical reactivity data is provided in Table 5. NDM-1 and IMP-26 from two $P _ { - }$ . mirabilis isolates were detected after bacterial growth on MacConkey agar but not on blood agar. The following statement is included as a footnote to the performance table in the device labeling:

NDM-1 and IMP-26 not detected in P. mirabilis growth from blood agar, but yielded positive results from MacConkey agar.

Of the variants tested, $P$ . aeruginosa harboring IMP-14 and IMP-18 were not detected by the NG-Test CARBA 5. The following statement is included as a limitation in the device labeling:

False negative results have been observed with IMP-14 and IMP-18.

The analytical reactivity of the NG-Test CARBA 5 for detection of carbapenemases in Enterobacteriaceae and $P .$ . aeruginosa was determined to be acceptable.

Table 5. Summary of Analytical Reactivity of the NG-Test CARBA 51   

<table><tr><td rowspan=1 colspan=1>Organism Group</td><td rowspan=1 colspan=1>Media</td><td rowspan=1 colspan=1>Agreement (%)2</td></tr><tr><td rowspan=3 colspan=1>Enterobacteriaceae</td><td rowspan=1 colspan=1>Blood Agar</td><td rowspan=1 colspan=1>64/66 (97.0)3</td></tr><tr><td rowspan=1 colspan=1>MacConkey Agar</td><td rowspan=1 colspan=1>66/66 (100)</td></tr><tr><td rowspan=1 colspan=1>HardyCHROM CRE agar</td><td rowspan=1 colspan=1>41/41 (100)</td></tr><tr><td rowspan=2 colspan=1>Pseudomonas aeruginosa</td><td rowspan=1 colspan=1>Blood Agar</td><td rowspan=1 colspan=1>24/26 (92.3)4</td></tr><tr><td rowspan=1 colspan=1>MacConkey Agar</td><td rowspan=1 colspan=1>24/26 (92.3)4</td></tr></table>

1 Triplicate results were all identical. Data from a single representative replicate. 2 Percent Agreement calculated from number of organisms evaluated after mCIM analysis 3 Proteus mirabilis; NDM and IMP targets not detected from blood agar but detected form MacConkey agar. 4 P. aeruginosa false negative isolates were characterized as IMP-14 and IMP-18 variants.

7. Assay Cut-Off: Not applicable

# B Comparison Studies:

1. Method Comparison with Predicate Device: Not applicable

2. Matrix Comparison: Not applicable

# C Clinical Studies:

# 1. Clinical Sensitivity:

The performance of the NG-Test CARBA 5 was initially evaluated in a Clinical Study that was conducted at three U.S. sites using 150 prospectively collected isolates (recovered fresh or less than 6 months prior to testing) and 149 retrospectively collected isolates (stock, recovered more than 6 months prior to testing) of Enterobacteriaceae and Pseudomonas aeruginosa. These isolates were used to evaluate performance of NG-Test CARBA 5 using bacterial colonies grown on blood and MacConkey agar. A subset of these isolates was selected based on the intended use of the HardyCHROM CRE agar and used in an internal study to evaluate performance of the NG-Test CARBA 5 after bacterial growth on this medium.

To be enrolled in the study and included in the analysis of performance, retrospective isolates must have been (i) identified to the species level and (ii) determined to be non-susceptible to at least one carbapenem antimicrobial agent by an FDA-cleared susceptibility test method or harbor any carbapenem-resistance marker. All prospective isolates were enrolled regardless of carbapenem susceptibility.

Ertapenem disks were routinely used to maintain selective pressure for isolated colonies of retrospective Enterobacteriaceae isolates while no selective pressure was used for isolated colonies of retrospective $P _ { - }$ . aeruginosa isolates. Carbapenem susceptibility of each isolate was verified by disk diffusion using ertapenem (Enterobacteriaceae only), imipenem and meropenem while carbapenemase production was evaluated using the modified

Carbapenemase Inactivation Method (mCIM), as described in CLSI M100. In addition, the carbapenemase genetic marker of each isolate was determined using an FDA-cleared PCR assay for the blaKPC, blaNDM, blaOXA-48, blaIMP and blaVIM markers. The NG-Test CARBA 5 results for each isolate were interpreted using the composite reference algorithm as depicted in Table 6.

Table 6. Composite reference algorithm for interpretation of NG Test CARBA 5 test results   

<table><tr><td rowspan=1 colspan=1>PCR</td><td rowspan=1 colspan=1>mCIM</td><td rowspan=1 colspan=1>CompositeResult</td><td rowspan=1 colspan=1>NG-TestCARBA 5</td><td rowspan=1 colspan=1>NG-Test CARBA 5Interpretation</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>True positive 1</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>False Negative</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>False positive</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>False positive</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>True Negative 2</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>True Negative 3</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>False Positive</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>True Negative</td></tr></table>

mCIM: modified Carbapenemase Inactivation Method The carbapenemase(s) identified by NG-Test CARBA 5 and resistance marker(s) identified by PCR must correspond (e.g., positive for KPC by NG-Test CARBA 5 and $b l a _ { \mathrm { K P C } }$ positive by PCR) 2 Gene for a carbapenemase targeted by NG-Test CARBA 5 present but not expressed (or expressed at levels below the limit of detection of NG-Test CARBA 5) 3 Isolate positive for a carbapenemase that is not targeted by the NG-Test CARBA 5

Performance of NG-Test CARBA 5 after bacterial growth on blood and MacConkey agar with the 240 Enterobacteriaceae isolates that provided 244 results (four isolates co-produced two carbapenemase targets) is summarized in Table 7.1. Performance after bacterial growth on blood and MacConkey agar with the $6 9 P .$ . aeruginosa isolates that provided 69 results is summarized in Table 7.2. One isolate that was initially enrolled was excluded from the analysis because it was a species of Pseudomonas other than $P .$ . aeruginosa. There were no NDM or OXA-48-like $P .$ . aeruginosa isolates enrolled in the study. This is addressed in the following statement that is included in the performance section in the device labeling:

For $P .$ . aeruginosa, there were no OXA-48-like or NDM enrolled. However, $P _ { - }$ aeruginosa with NDM $( n { = } 2 )$ were evaluated analytically in the bench testing.

Table 7.1. Agreement of NG-Test CARBA 5 with the composite reference when testing Enterobacteriaceae   

<table><tr><td rowspan=2 colspan=2>Enterobacteriaceae</td><td rowspan=1 colspan=3>Composite Reference Method</td></tr><tr><td rowspan=1 colspan=1>Positive 1</td><td rowspan=1 colspan=1>Negative 2</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=3 colspan=1>NG-Test CARBA 5</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>156</td><td rowspan=1 colspan=1>4 3,4</td><td rowspan=1 colspan=1>160</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>84</td><td rowspan=1 colspan=1>84</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>156</td><td rowspan=1 colspan=1>88</td><td rowspan=1 colspan=1>244</td></tr><tr><td rowspan=1 colspan=2>Positive Percent Agreement (PPA)</td><td rowspan=1 colspan=3>156/156 = 100% (95% CI: 97.6-100%)</td></tr><tr><td rowspan=1 colspan=2>Negative Percent Agreement (NPA)</td><td rowspan=1 colspan=3>84/88 = 95.5% (95% CI: 88.9-98.2%)</td></tr></table>

1 Defined as positive by mCIM and an FDA-cleared PCR assay for blaKPC, blaNDM, blaOXA-48, blaIMP and blaVIM 2 Defined as negative by mCIM and/or an FDA-cleared PCR assay for blaKPC, blaNDM, blaOXA-48, blaIMP and blaVIM 3 An alternative PCR assay showed that the NDM false positive isolate harbored a blaNDM -1 variant. Isolate was positive by mCIM. 4 An alternative PCR assay and bidirectional sequencing showed that the three IMP false positive isolates harbored blaIMP -8/-47 variant that is predicted by in silico analysis but not analytically demonstrated to be detected by the assay. Isolates were positive by mCIM.

Table 7.2. Agreement of NG-Test CARBA 5 with the composite reference when testing $P .$ . aeruginosa   

<table><tr><td rowspan=2 colspan=2>P. aeruginosa</td><td rowspan=1 colspan=3>Composite Reference Method</td></tr><tr><td rowspan=1 colspan=1>Positive 1</td><td rowspan=1 colspan=1>Negative 2</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=3 colspan=1>NG-Test CARBA 5</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>3 3</td><td rowspan=1 colspan=1>16</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>53</td><td rowspan=1 colspan=1>53</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>56</td><td rowspan=1 colspan=1>69</td></tr><tr><td rowspan=1 colspan=2>Positive Percent Agreement (PPA)</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2>13/13 = 100% (95% CI: 77.2-100%)</td></tr><tr><td rowspan=1 colspan=2>Negative Percent Agreement (NPA)</td><td rowspan=1 colspan=3>53/56 = 94.6% (95% CI: 85.4-98.2%)</td></tr></table>

Defined as positive by mCIM and an FDA-cleared PCR assay for blaKPC, blaNDM, blaOXA-48, blaIMP and $b l a _ { \mathrm { V I M } }$ 2 Defined as negative by mCIM and/or an FDA-cleared PCR assay for blaKPC, $b l a _ { \mathrm { N D M } }$ , blaOXA-48, $b l a _ { \mathrm { I M P } }$ and blaVIM 3 An alternative PCR assay and bidirectional sequencing showed that the three IMP false positive isolates harbored $b l a _ { I M P }$ variants that (i) are not detected by the FDA-cleared PCR assay $b l a _ { I M P }$ variant $^ { - 7 }$ ), (ii) are predicted by in silico analysis but not analytically demonstrated to be detected by the assay $( b l a _ { I M P } - 1 9 )$ , or (iii) the reactivity of the assay is unknown $B l a _ { I M P }$ variant -15). Isolates were positive by mCIM.

Performance of NG-Test CARBA 5 with carbapenemase-positive isolates stratified by carbapenemase target is shown in Table 8. The same results were obtained regardless of whether testing colonies grown on blood agar or from colonies grown on MacConkey agar. For isolates positive by the composite reference method, the results obtained with the NGTest CARBA 5 after growth on blood or MacConkey agar were in agreement [Enterobacteriaceae: 156/156 $( 1 0 0 \% )$ ; $P$ . aeruginosa: 13/13 $( 1 0 0 \% ) ]$ . For isolates negative by the composite reference method, the agreement obtained with the NG-Test CARBA 5 after growth on blood and MacConkey agar was: Enterobacteriaceae: 84/88 $( 9 5 . 5 \% )$ ; $P$ . aeruginosa: 53/56 $( 9 4 . 6 \% )$ .

Table 8. Performance of NG-Test CARBA 5 vs Composite Reference with Carbapenemasepositive isolates stratified by each target   

<table><tr><td rowspan=1 colspan=2>Comparator PCR Result</td><td rowspan=1 colspan=4>NG-Test CARBA 5 Result</td><td rowspan=1 colspan=2>Agreement %(95% CI)</td></tr><tr><td rowspan=1 colspan=1>Organism Group</td><td rowspan=1 colspan=1>Target</td><td rowspan=1 colspan=1>TP</td><td rowspan=1 colspan=1>FP</td><td rowspan=1 colspan=1>TN</td><td rowspan=1 colspan=1>FN</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td></tr><tr><td rowspan=5 colspan=1>Enterobacteriaceae 5[n = 244]</td><td rowspan=1 colspan=1>KPC</td><td rowspan=1 colspan=1>84</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>160</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100(95.6-100)</td><td rowspan=1 colspan=1>100(97.7-100)</td></tr><tr><td rowspan=1 colspan=1>NDM</td><td rowspan=1 colspan=1>37</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>206</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100(90.6-100)</td><td rowspan=1 colspan=1>99.5(97.3-99.9)</td></tr><tr><td rowspan=1 colspan=1>OXA</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>224</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100(83.9-100)</td><td rowspan=1 colspan=1>100(98.3-100)</td></tr><tr><td rowspan=1 colspan=1>IMP</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>32</td><td rowspan=1 colspan=1>237</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100(51.0-100)</td><td rowspan=1 colspan=1>98.8(96.4-99.6)</td></tr><tr><td rowspan=1 colspan=1>VIM</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>233</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100(74.1-100)</td><td rowspan=1 colspan=1>100(98.4-100)</td></tr><tr><td rowspan=5 colspan=1>P. aeruginosa[n = 69]</td><td rowspan=1 colspan=1>KPC</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>67</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100(34.2-100)</td><td rowspan=1 colspan=1>100(94.6-100)</td></tr><tr><td rowspan=1 colspan=1>NDM 4</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>69</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>100(94.7-100)</td></tr><tr><td rowspan=1 colspan=1>OXA 4</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>69</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>100(94.7-100)</td></tr><tr><td rowspan=1 colspan=1>IMP</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>33</td><td rowspan=1 colspan=1>64</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100(34.2-100)</td><td rowspan=1 colspan=1>95.5(87.6-98.5)</td></tr><tr><td rowspan=1 colspan=1>VIM</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100(70.1-100)</td><td rowspan=1 colspan=1>100(94.0-100)</td></tr></table>

To evaluate the performance of NG-Test CARBA 5 after bacterial growth on HardyCHROM CRE agar, was evaluated using a subset of 186 isolates that were eligible in accordance with the intended use of this agar medium (i.e., E. coli and KES isolates). One isolate was excluded because it was unavailable for testing. Six isolates were excluded from the raw stool analysis due to the lack of or poor growth on HardyCHROM CRE agar (post-seeding into raw stool). Two additional isolates (8 total) were excluded from the C&S Cary Blair stool analysis due to the lack of or poor growth on HardyCHROM CRE agar (post-seeding into C&S Cary Blair stool). In total, 180 isolates (184 results, shown in Table 9.1) were reported in the raw stool analysis and 179 isolates (182 results, shown in Table 9.2) were reported in the C&S Cary Blair stool analysis. Results stratified by carbapenemase target are shown in Table 10).

Table 9.1. Agreement of NG-Test CARBA 5 with the composite reference after bacterial growth on HardyCHROM CRE agar (post-seeding into Raw Stool)   

<table><tr><td rowspan=2 colspan=2>Raw Stool</td><td rowspan=1 colspan=3>Composite Reference Method</td></tr><tr><td rowspan=1 colspan=1>Positive 1</td><td rowspan=1 colspan=1>Negative 2</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=3 colspan=1>NG-Test CARBA 5</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>143</td><td rowspan=1 colspan=1>4 3,4</td><td rowspan=1 colspan=1>147</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>37</td><td rowspan=1 colspan=1>37</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>143</td><td rowspan=1 colspan=1>41</td><td rowspan=1 colspan=1>184</td></tr><tr><td rowspan=1 colspan=2>Positive Percent Agreement (PPA)</td><td rowspan=1 colspan=3>143/143 = 100% (95% CI: 97.4-100%)</td></tr><tr><td rowspan=1 colspan=2>Negative Percent Agreement (NPA)</td><td rowspan=1 colspan=3>37/41 = 90.2% (95% CI: 77.5-96.1%)</td></tr></table>

1 Defined as positive by mCIM and an FDA-cleared PCR assay for blaKPC, blaNDM, blaOXA-48, blaIMP and blaVIM 2 Defined as negative by mCIM and/or an FDA-cleared PCR assay for blaKPC, blaNDM, blaOXA-48, blaIMP and blaVIM 3 An alternative PCR assay showed that the NDM false positive isolate harbored a blaNDM -1 variant. Isolate was positive by mCIM. 4 An alternative PCR assay and bidirectional sequencing showed that the three IMP false positive isolates harbored blaIMP -8/-47 variant that is predicted by in silico analysis but not analytically demonstrated to be detected by the assay. Isolates were positive by mCIM.

Table 9.2. Agreement of NG-Test CARBA 5 with the composite reference after bacterial growth on HardyCHROM CRE agar (post-seeding into C&S Cary Blair Stool)   

<table><tr><td rowspan=2 colspan=2>C&amp;S Cary Blair Stool</td><td rowspan=1 colspan=3>Composite Reference Method</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative 2</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=3 colspan=1>NG-Test CARBA 5</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>141</td><td rowspan=1 colspan=1>4 3,4</td><td rowspan=1 colspan=1>145</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>37</td><td rowspan=1 colspan=1>37</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>141</td><td rowspan=1 colspan=1>41</td><td rowspan=1 colspan=1>182</td></tr><tr><td rowspan=1 colspan=2>Positive Percent Agreement (PPA)</td><td rowspan=1 colspan=3>141/141 = 100% (95% CI: 97.3-100%)</td></tr><tr><td rowspan=1 colspan=2>Negative Percent Agreement (NPA)</td><td rowspan=1 colspan=3>37/41 = 90.2% (95% CI: 77.5-96.1%)</td></tr></table>

Defined as positive by mCIM and an FDA-cleared PCR assay for blaKPC, blaNDM, blaOXA-48, $b l a _ { \mathrm { I M P } }$ and $b l a _ { \mathrm { V I M } }$ 2 Defined as negative by mCIM and/or an FDA-cleared PCR assay for blaKPC, blaNDM, blaOXA-48, blaIMP and blaVIM 3 An alternative PCR assay showed that the NDM false positive isolate harbored a blaNDM -1 variant. Isolate was positive by mCIM. 4 An alternative PCR assay and bidirectional sequencing showed that the three IMP false positive isolates harbored blaIMP -8/-47 variant that is predicted by in silico analysis but not analytically demonstrated to be detected by the assay. Isolates were positive by mCIM.

For isolates positive by the composite reference method, the results obtained with the NGTest CARBA 5 after growth on HardyCHROM CRE agar were in agreement [Raw Stool: 143/143 $( 1 0 0 \% )$ ; C&S Cary Blair Stool: 141/141 $( 1 0 0 \% )$ , Table 10]. For isolates negative by the composite reference method, the agreement obtained with the NG-Test CARBA 5 after growth on HardyCHROM CRE agar was the same for Raw Stool and C&S Cary Blair Stool: 37/41 $( 9 0 . 2 \% )$ .

Table 10. Overall performance of NG-Test CARBA 5 with Carbapenemase-positive $E .$ . coli and KES isolates1 grown on HardyCHROM CRE agar   

<table><tr><td rowspan=1 colspan=2>Comparator PCR Result</td><td rowspan=1 colspan=4>NG-Test CARBA 5 Result</td><td rowspan=1 colspan=2>Agreement %(95% CI)</td></tr><tr><td rowspan=1 colspan=1>Organism Group</td><td rowspan=1 colspan=1>Target</td><td rowspan=1 colspan=1>TP</td><td rowspan=1 colspan=1>FP</td><td rowspan=1 colspan=1>TN</td><td rowspan=1 colspan=1>FN</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td></tr><tr><td rowspan=5 colspan=1>Raw Stool[n = 184 results]</td><td rowspan=1 colspan=1>KPC</td><td rowspan=1 colspan=1>76</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>108</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100(95.2-100)</td><td rowspan=1 colspan=1>100(96.6-100)</td></tr><tr><td rowspan=1 colspan=1>NDM</td><td rowspan=1 colspan=1>36</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>147</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100(90.4-100)</td><td rowspan=1 colspan=1>(99.3(96.3-99.9)</td></tr><tr><td rowspan=1 colspan=1>OXA</td><td rowspan=1 colspan=1>18</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>166</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100(82.4-100)</td><td rowspan=1 colspan=1>100(98.3-100)</td></tr><tr><td rowspan=1 colspan=1>IMP</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>33</td><td rowspan=1 colspan=1>177</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100(51.0-100)</td><td rowspan=1 colspan=1>98.3(95.2-99.4)</td></tr><tr><td rowspan=1 colspan=1>VIM</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>175</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100(70.1-100)</td><td rowspan=1 colspan=1>100(97.9-100)</td></tr><tr><td rowspan=5 colspan=1>C&amp;S Cary Blair Stool[n = 182 results]</td><td rowspan=1 colspan=1>KPC</td><td rowspan=1 colspan=1>75</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>107</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100(95.1-100)</td><td rowspan=1 colspan=1>100(96.5-100)</td></tr><tr><td rowspan=1 colspan=1>NDM</td><td rowspan=1 colspan=1>36</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>145</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100(90.4-100)</td><td rowspan=1 colspan=1>99.3(96.2-99.9)</td></tr><tr><td rowspan=1 colspan=1>OXA</td><td rowspan=1 colspan=1>18</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>164</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100(82.4-100)</td><td rowspan=1 colspan=1>100(97.7-100)</td></tr><tr><td rowspan=1 colspan=1>IMP</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>33</td><td rowspan=1 colspan=1>175</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100(51.0-100)</td><td rowspan=1 colspan=1>98.3(95.2-99.4)</td></tr><tr><td rowspan=1 colspan=1>VIM</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>174</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100(67.6-100)</td><td rowspan=1 colspan=1>100(97.8-100)</td></tr></table>

Isolates included: $E$ . asburiae (2), $E$ . cloacae (24), E. cloacae complex (3), $E$ . coli (45), K. aerogenes (8), $K .$ oxytoca (11), $K .$ pneumoniae (80), S. marcescens (7) 2 An alternative PCR assay showed that this isolate harbored a blaNDM -1 variant. 3 An alternative PCR assay and bidirectional sequencing showed that the three IMP false positive isolates harbored blaIMP -8/-47 variant that is predicted by in silico analysis but not analytically demonstrated to be detected by the assay. Isolates were positive by mCIM.

One K. oxytoca isolate was positive for NDM by NG-Test CARBA 5 but negative by the comparator PCR assay. Investigation of the discordant false positive result using an alternative PCR method confirmed the presence of $b l a _ { N D M } - 1$ variant. Six isolates (3 Enterobacteriaceae and $3 P .$ . aeruginosa) were positive for IMP by NG-Test CARBA 5 but negative by the comparator PCR assay. Investigation of the discordant false positive results using an alternative PCR assay and bidirectional sequencing showed that these isolates harbored blaIMP variants that (i) are not detected by the FDA-cleared PCR assay (blaIMP variant -7, $\mathrm { n } { = } 1$ ), (ii) are predicted by in silico analysis but not analytically demonstrated to be detected by the assay (blaIMP variant -8/-47, $\mathrm { n } { = } 3$ and blaIMP variant -19, $\mathrm { \Omega } _ { \mathrm { n } } { = } 1$ ), or (iii) the reactivity of the assay is unknown (blaIMP variant -15, $\mathrm { n } { = } 1$ ).

The results were determined to be acceptable.

The address the indications for use, the following statements are included in the Limitations section of the device labeling:

The performance of NG-Test CARBA 5 was established with colonies from blood agar, MacConkey agar and HardyCHROM CRE agar. Performance with other culture media has not been evaluated and is therefore unknown.

The performance of NG-Test CARBA 5 with bacteria other than Enterobacteriaceae and Pseudomonas aeruginosa has not been evaluated.

Organism identification and elevated carbapenem MICs should be determined prior to testing with NG-Test CARBA 5.

2. Clinical Specificity: Refer to Section VII C(1) above.

3. Other Clinical Supportive Data (When 1. and 2. Are Not Applicable): Not applicable

D Clinical Cut-Off: Not applicable

# E Expected Values/Reference Range:

The performance of the NG-Test CARBA 5 was evaluated using a collection of prospectively collected and archived isolates of carbapenem non-susceptible bacteria. When testing for the presence of target carbapenemases, the 309 isolates of Enterobacteriaceae (240) and $P .$ . aeruginosa (69) tested in the study generated 169 positive results using a composite reference method (FDA-cleared PCR assay and mCIM method); 176 positive results were generated using the NG-Test CARBA 5. A summary of the carbapenemase targets detected by the composite reference method and by the NG-Test CARBA 5 is shown in Table 11.

Table 11. Summary of carbapenemase targets identified in the Clinical Study for the NGTest CARBA 5   

<table><tr><td rowspan=2 colspan=1>Organism Family</td><td rowspan=2 colspan=1>CarbapenemaseTarget</td><td rowspan=1 colspan=2>Number of Targets Detected (% of combined)</td></tr><tr><td rowspan=1 colspan=1>Composite Reference</td><td rowspan=1 colspan=1>NG-Test CARBA 5</td></tr><tr><td rowspan=6 colspan=1>Enterobacteriaceae</td><td rowspan=1 colspan=1>KPC</td><td rowspan=1 colspan=1>84 (53.8)</td><td rowspan=1 colspan=1>84 (52.5)</td></tr><tr><td rowspan=1 colspan=1>NDM</td><td rowspan=1 colspan=1>37 (23.7)</td><td rowspan=1 colspan=1>38 (23.8) 1</td></tr><tr><td rowspan=1 colspan=1>OXA</td><td rowspan=1 colspan=1>20 (12.8)</td><td rowspan=1 colspan=1>20 (12.5)</td></tr><tr><td rowspan=1 colspan=1>IMP</td><td rowspan=1 colspan=1>4 (2.6)</td><td rowspan=1 colspan=1>7 (4.4) {2</td></tr><tr><td rowspan=1 colspan=1>VIM</td><td rowspan=1 colspan=1>11 (7.1)</td><td rowspan=1 colspan=1>11 (6.9)</td></tr><tr><td rowspan=1 colspan=1>Combined</td><td rowspan=1 colspan=1>156</td><td rowspan=1 colspan=1>160</td></tr><tr><td rowspan=6 colspan=1>P. aeruginosa</td><td rowspan=1 colspan=1>KPC</td><td rowspan=1 colspan=1>2 (15.4)</td><td rowspan=1 colspan=1>2 (12.5)</td></tr><tr><td rowspan=1 colspan=1>NDM</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>n/a</td></tr><tr><td rowspan=1 colspan=1>OXA</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>n/a</td></tr><tr><td rowspan=1 colspan=1>IMP</td><td rowspan=1 colspan=1>2 (15.4)</td><td rowspan=1 colspan=1>5 (31.3) 3</td></tr><tr><td rowspan=1 colspan=1>VIM</td><td rowspan=1 colspan=1>9 (69.2)</td><td rowspan=1 colspan=1>9 (56.3)</td></tr><tr><td rowspan=1 colspan=1>Combined</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>16</td></tr></table>

# VIII Proposed Labeling:

The labeling supports the finding of substantial equivalence for this device.

# IX Conclusion:

The submitted information in this premarket notification is complete and supports a substantial equivalence decision.