# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATIONDECISION SUMMARYASSAY AND INSTRUMENT

I Background Information:

A 510(k) Number K191288   
B Applicant OpGen, Inc.   
C Proprietary and Established Names Acuitas AMR Gene Panel   
D Regulatory Information

<table><tr><td rowspan=1 colspan=1>ProductCode(s)</td><td rowspan=1 colspan=1>Classification</td><td rowspan=1 colspan=1>RegulationSection</td><td rowspan=1 colspan=1>Panel</td></tr><tr><td rowspan=1 colspan=1>PMY</td><td rowspan=1 colspan=1>Class II</td><td rowspan=1 colspan=1>21 CFR 866.1640 -AntimicrobialSusceptibility Test Powder</td><td rowspan=1 colspan=1>MI - Microbiology</td></tr><tr><td rowspan=1 colspan=1>OOI</td><td rowspan=1 colspan=1>Class II</td><td rowspan=1 colspan=1>21 CFR 862.2570 -Instrumentation forclinical multiplex testsystems</td><td rowspan=1 colspan=1>CH - Clinical Chemistry</td></tr></table>

# II Submission/Device Overview:

# A Purpose for Submission:

The purpose of the submission is to obtain a substantial equivalence determination for the Acuitas AMR Gene Panel run on the OpGen Qualified QuantStudio 5 Real-time PCR System for the qualitative detection of gene sequences for resistance markers from bacterial colonies.

# B Measurand:

The following markers, associated with resistance, are claimed for reporting in the appropriate species:

AAC, AAD, ANT, APH, armA, CMY, CTX-M-1, CTX-M-2, CTX-M-9, DFR, DHA, IMP, KPC, MCR-1, NDM, OXA-1, OXA-48, OXA-9, PER, RMT, SHV, SUL1, SUL2, TEM, vanA, VEB, VIM, P. aeruginosa gyrA mutant, E. coli gyrA mutant

# C Type of Test:

A multiplexed nucleic acid-based test intended for use with the OpGen Qualified QuantStudio 5 Real-Time PCR System for the qualitative detection and differentiation of select genetic determinants of antimicrobial resistance. The Acuitas AMR Gene Panel is performed with pure bacterial colonies.

III Intended Use/Indications for Use:

A Intended Use(s):

See Indications for Use below.

# B Indication(s) for Use:

The Acuitas AMR Gene Panel, performed on the QIAGEN EZ1 Advanced XL System and the OpGen Qualified QuantStudio 5 Real-Time PCR System, is a qualitative nucleic acid-based multiplex in vitro diagnostic test for detection and differentiation of antibiotic resistance markers to one or more antimicrobial agents. The test utilizes real‐time polymerase chain reaction (PCR) and is performed on isolated colonies of Pseudomonas aeruginosa, Enterococcus faecalis, or members of Enterobacterales grown in pure culture on blood agar or MacConkey agar.

Organism identification results must be available prior to reporting results for the Acuitas AMR Gene Panel. Antimicrobial resistance gene results are reported by the Acuitas AMR Gene Panel for the combinations of bacterial pathogens and associated genetic resistance markers indicated in Table 1 below.

Table 1 - Antimicrobial Resistance Gene Markers (Genetic Determinants) Associated with Bacterial Species   

<table><tr><td colspan="1" rowspan="1">Organism</td><td colspan="1" rowspan="1">Reported AMR Gene Marker</td></tr><tr><td colspan="1" rowspan="1">Citrobacter freundii complex </td><td colspan="1" rowspan="1">CTX-M-1, CTX-M-9, KPC, NDM, OXA-48</td></tr><tr><td colspan="1" rowspan="1">Citrobacter koseri</td><td colspan="1" rowspan="1">KPC, OXA-48</td></tr><tr><td colspan="1" rowspan="1">Enterobacter cloacae complex b</td><td colspan="1" rowspan="1">CTX-M-1, CTX-M-9, KPC, TEM d</td></tr><tr><td colspan="1" rowspan="1">Enterococcus faecalis</td><td colspan="1" rowspan="1">vanA</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">AAC, ANT, CMY, CTX-M-1, CTX-M-2, CTX-M-9, DFR, gyrA Mutant c, KPC, MCR-1c, OXA-1,OXA-9, SHVd, Sul1, Sul2, TEMd</td></tr><tr><td colspan="1" rowspan="1">Klebsiella aerogenes</td><td colspan="1" rowspan="1">CTX-M-1, CTX-M-9, KPC, NDM, OXA-48</td></tr><tr><td colspan="1" rowspan="1">Klebsiella michiganensis</td><td colspan="1" rowspan="1">CTX-M-1, CTX-M-9, KPC, NDM, OXA-48</td></tr><tr><td colspan="1" rowspan="1">Klebsiella oxytoca</td><td colspan="1" rowspan="1">CTX-M-1, CTX-M-9, KPC, NDM, OXA-48</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">AAC, AAD, APH, CMY, CTX-M-1, CTX-M- 9, DFR, DHA, IMP, KPC, NDM, OXA-1, OXA-9,OXA-48, RMT, Sul1, Sul2, TEMd</td></tr><tr><td colspan="1" rowspan="1">Klebsiella quasipneumoniae</td><td colspan="1" rowspan="1">CTX-M-1, CTX-M-9, KPC, NDM, OXA-48</td></tr><tr><td colspan="1" rowspan="1">Klebsiella variicola</td><td colspan="1" rowspan="1">CTX-M-1, CTX-M-9, KPC, NDM, OXA-48</td></tr><tr><td colspan="1" rowspan="1">Morganella morganii</td><td colspan="1" rowspan="1">CTX-M-1, KPC, NDM, OXA-48</td></tr><tr><td colspan="1" rowspan="1">Proteus mirabilis</td><td colspan="1" rowspan="1">AAC, ANT, APH, armA, CMY, CTX-M-1, CTX-M-2, CTX-M-9, DFR, KPC, NDM, OXA-1, OXA-9, OXA-48, Sul2, TEM , VEB, VIM</td></tr><tr><td colspan="1" rowspan="1">Providencia rettgeri</td><td colspan="1" rowspan="1">NDM</td></tr><tr><td colspan="1" rowspan="1">Providencia stuartii</td><td colspan="1" rowspan="1">NDM</td></tr><tr><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">AAC, ANT, CTX-M-1, CTX-M-2, gyrA Mutant, KPC, NDM, OXA-1, PER, SHVd, TEM, VEB,VIM</td></tr><tr><td colspan="1" rowspan="1">Raoultella ornithinolytica</td><td colspan="1" rowspan="1">KPC, NDM, OXA-48</td></tr><tr><td colspan="1" rowspan="1">Raoultella planticola</td><td colspan="1" rowspan="1">KPC</td></tr><tr><td>Serratia marcescens</td><td>CTX-M-1, CTX-M-9, KPC, NDM, OXA-48</td></tr><tr><td colspan="2">aCitrobacter freundii complex = C. freundii, C. braakii, C. werkmanii and C. youngae. b Enterobacter cloacae complex = E. asburiae, E. cloacae, E. hormaechei, E. kobei and E. ludwigii.</td></tr><tr><td colspan="2">cPCRassays associated with fluoroquinoloneresistance detect and differentiate wild typeand mutant variants gyraseA at amino acid position 87 for E. coli and position 83 for P. aeruginosa.</td></tr><tr><td colspan="2">dPCR assays for SHV and TEM detect several sequence variants for the two genes, respectively, at amino acid positions 156 and 104 associated with wild type penicillin resistance and mutations associated with ESBL</td></tr><tr><td colspan="2">phenotypes. eThe panel includes an assay for the detection of the mobilized colistin genetic determinant MCR-l in E. coli.</td></tr></table>

The Acuitas AMR Gene Panel includes assays for the detection and reporting of genetic resistance markers associated with resistance to select drugs in the following antibiotic groups: aminoglycosides, carbapenems, cephalosporins, fluoroquinolones, penicillins, sulfonamides, trimethoprim, and vancomycin, to aid in the identification of potentially antimicrobial-resistant organisms. The panel includes an assay for the detection of the mobilized colistin genetic determinant MCR-1, a marker of public health importance associated with reduced inhibitory activity of polymyxins.

The results of the Acuitas AMR Gene Panel for detection and identification of genetic determinants associated with antimicrobial resistance are used along with the Acuitas AMR Gene Panel Electronic User Guide (EUG). In certain cases, this information may be used as an aid to clinicians in the management of patients with known or suspected antibiotic nonsusceptible or resistant bacterial infections. The EUG contains information on the appropriateness of reporting resistance markers detected by the Acuitas AMR Gene Panel for claimed organisms based on the strength of the collective, totality of scientific evidence delineating the level of association between molecular marker detection with phenotypic, clinical resistance. Test results are not conclusive or prescriptive for labeled use of any specific antimicrobial drug product, and therefore, this test cannot be used in place of or to postpone or delay phenotypic antimicrobial susceptibility testing.

A “Detected” or “Not Detected” result does not rule out the presence of other antimicrobial resistance markers not detected by the Acuitas AMR Gene Panel. A "Not Detected" result for a genetic marker of antimicrobial resistance does not indicate susceptibility to associated antimicrobial drugs or drug classes, as multiple mechanisms of resistance may exist.

C Special Conditions for Use Statement(s):

Rx - For Prescription Use Only

D Special Instrument Requirements:

For use with the OpGen Qualified QuantStudio 5 Real-Time PCR System and the QIAGEN EZ1 Advanced XL System

IV Device/System Characteristics:

A Device Description:

The Acuitas AMR Gene Panel is a qualitative nucleic acid-based in vitro diagnostic PCR test that is capable of simultaneous detection and identification of select genetic determinants of antimicrobial resistance. The test is performed on pure colonies of select Enterobacterales and Pseudomonas aeruginosa grown on blood agar or MacConkey agar and Enterococcus faecalis grown on blood agar obtained from clinical specimens in order to detect genetic determinants associated with resistance to drugs in the following antibiotic classes: Aminoglycosides, Carbapenems, Cephalosporins, Fluoroquinolones, Penicillins, Sulfonamides, Trimethoprim, and Vancomycin. In addition, the test detects an emerging genetic marker of public health importance associated with reduced inhibitory activity of polymyxins. An assessment of the levels of evidence, obtained through clinical studies, analytical studies, literature, databases, and in silico analyses was used to assess the association between the molecular (genotypic) marker detected and the resistance phenotype for certain bacterial pathogens.

# B Principle of Operation:

The Acuitas AMR Gene Panel assay is performed on the OpGen Qualified QuantStudio 5 RealTime PCR System (OpGen Qualified QS5 instrument) using the Acuitas AMR Gene Panel Kit. The Acuitas AMR Gene Panel employs automated DNA extraction on the QIAGEN EZ1 Advanced XL and multiplex real-time PCR on the OpGen Qualified QS5 instrument. The test kit includes PCR plates (96-well) with dried primers and probes for analysis of DNA from four (4) isolates (24 wells per isolate). Each Acuitas AMR Gene Panel test kit includes two PCR plates for analysis of eight isolate samples along with tubes of Master Mix containing reagents and enzymes for PCR.

From bacterial isolates, the DNA is extracted on the QIAGEN EZ1 Advanced XL system according to manufacturer instructions, which includes incorporating the Assay Control within the extraction process. A sample of extracted DNA eluate is transferred to a Reagent Reservoir Trough to which Master Mix is added. Extracted DNA/Master Mix is then transferred to each of 24 wells on the PCR plate per test sample. The contents of each well are mixed, and the plate is sealed and transferred to the OpGen Qualified QS5 instrument for real-time multiplex reaction and detection. Data are exported from the PCR instrument and imported into the Acuitas AMR Gene Analysis Software—a spreadsheet application that analyzes the data and generates a report for viewing and printing. Each test report indicates detection of applicable antimicrobial resistance genes as “Detected, ”“Not Detected,” or “NA/NR.” A one-page report will be generated if the results are invalid, with a summary of the error(s) listed.

The Acuitas AMR Gene Analysis Software generates a report for each sample on the PCR plate. For a valid sample, there are two (2) report pages:

1. Page 1 of the report contains the following information: o Run/Report Information: the top of the report provides information on the date the test was performed and the date that the report was generated. The Sample ID and the associated Plate ID are reported, as well as any errors detected during the analysis of the data. Control Results list the status of the Assay Control for a “Valid” Control. o Class/Gene Table: the results for each antimicrobial resistance gene detected by the Acuitas AMR Gene Panel are reported in this section. Antimicrobial resistance genes are grouped by antibiotic class. Table 1 below summarizes the possible results reported for each resistance gene:

Table 1 - Results Reported for Antimicrobial Resistance Genes   

<table><tr><td rowspan=1 colspan=1>Reported Result</td><td rowspan=1 colspan=1>Interpretation</td></tr><tr><td rowspan=1 colspan=1>&quot;Detected&quot;</td><td rowspan=1 colspan=1>Resistance gene target is detected. There is PCRamplification of the target above the threshold at a Ct valuebelow the assay cut-off.</td></tr><tr><td rowspan=1 colspan=1>&quot;Not Detected</td><td rowspan=1 colspan=1>Resistance gene target is not detected. There is no PCRamplification of the target above the threshold at a Ct valuebelow the assay cutoff.</td></tr><tr><td rowspan=1 colspan=1>&quot;NA/NR&quot;</td><td rowspan=1 colspan=1>Resistance gene is either Not Associated (NA) with theorganism or Non-Reportable (NR) for the organism.</td></tr><tr><td rowspan=1 colspan=1>&quot;Invalid&quot;</td><td rowspan=1 colspan=1>Resistance gene results are not reported if the Assay Controlis invalid. A mutant gyrase gene result is also reportedInvalid&quot; if the gyrase Mutant result is not consistent with theuser-selected organism.</td></tr></table>

2. Page 2 of the assay report contains Supplemental Information, which includes two (2) tables:

o Table A: table lists the reportable antimicrobial resistance genes on the Acuitas AMR Gene Panel associated for a given species or group. This table only lists corresponding antibiotics for organism/marker combinations with an established combined clinical device performance PPV (Positive Predictive Value) near or above $80 \%$ , i.e., for which the detected presence of the AMR Gene in the combined clinical device performance evaluation associated with phenotypic resistance against the antibiotic at/near $80 \%$ of those isolates with gene detection.

PPV: Rate of positive genotypic test results for an AMR Gene associated with an isolate in agreement with Antimicrobial Susceptibility Testing (AST) result indicating phenotypic resistance of the isolate against a specific antibiotic. Actual PPVs for the reported AMR Gene – isolate combinations of the Acuitas AMR Gene Panel observed during the performance evaluation can be queried in the Electronic User Guide (EUG) by selecting a specific AMR gene, organism, and antibiotic combination from the drop-down menu.

o Table $B$ : table lists the antimicrobial resistance genes on the Acuitas AMR Gene Panel that are Not Associated (NA) or Non-Reportable (NR) for a given species. NA (Not Associated) $=$ Gene not associated with the organism. NR (Non-Reportable) $=$ Minimum performance not demonstrated or not evaluated.

Additional information can be found in the Acuitas AMR Gene Panel EUG at the company website. The Acuitas AMR Gene Panel EUG is an extension of the Acuitas AMR Gene Panel package insert, and it provides additional supporting information about the performance characteristics and interpretation of results for the Acuitas AMR Gene Panel

# C Instrument Description Information:

1. Instrument Name:

The following instruments are needed to generate results for the Acuitas AMR Gene Panel:

OpGen Qualified QuantStudio 5 Real-Time PCR System. Also referred to as “OpGen Qualified QS5 instrument.”

QIAGEN EZ1 Advanced XL System

# 2. Sample Identification:

Prior to loading the Acuitas AMR Gene Panel PCR Plate into the instrument, the sample information must be entered. A barcode scanner is available for automatic entry of the PCR plate. After launching the Acuitas AMR Gene Analysis Software, if a sample is present, use the drop-down menu to select the correct species for the isolate that was tested for generating the appropriate report per isolate.

# 3. Specimen Sampling and Handling:

Extracted DNA from pure colonies is added to the Acuitas AMR Gene Panel PCR Plate and inserted into the instrument manually.

# 4. Calibration:

The OpGen Qualified QuantStudio 5 Real-Time PCR System is factory calibrated and does not require calibration at installation. To ensure optimal performance, the Operator’s manual recommends performing calibrations at the specified frequency.

# 5. Quality Control:

External controls are not provided with the Acuitas AMR Gene Panel, but quality control strains are recommended in the package insert. External Controls should be used in accordance with laboratory protocols and the appropriate accrediting organization requirements, as applicable.

This medical device product has functions subject to FDA premarket review as well as functions that are not subject to FDA premarket review. For this application, if the product has functions that are not subject to FDA premarket review, FDA assessed those functions only to the extent that they either could adversely impact the safety and effectiveness of the functions subject to FDA premarket review or they are included as a labeled positive impact that was considered in the assessment of the functions subject to FDA premarket review.

V Substantial Equivalence Information:

A Predicate Device Name(s): Xpert Carba-R   
B Predicate 510(k) Number(s): K152614   
C Comparison with Predicate(s):

<table><tr><td>Device &amp; Predicate Device(s):</td><td>Device: K191288</td><td>Predicate: K152614</td></tr><tr><td colspan="3"></td></tr><tr><td colspan="3">General Device Characteristic Similarities</td></tr><tr><td>Device Trade Name</td><td>Acuitas AMR Gene Panel</td><td>Xpert Carba-R</td></tr><tr><td>Intended Use/Indications For Use</td><td>The Acuitas AMR Gene Panel, performed on the QIAGEN EZ1 Advanced XL</td><td>The Xpert Carba-R Assay, performed on the GeneXpert Instrument Systems, is a</td></tr></table>

does not preclude the presence of other resistance mechanisms. The Xpert Carba-R Assay should be used in conjunction with other laboratory tests including phenotypic antimicrobial susceptibility testing. The Xpert CarbaR Assay is intended as an aid for infection control in detecting and differentiating genetic markers of resistance to monitor the spread of carbapenem-non-susceptible organisms in healthcare settings. The Xpert CarbaR Assay is not intended to guide or monitor treatment for carbapenem-nonsusceptible bacterial infections.

<table><tr><td></td><td>System and the OpGen Qualified QuantStudio 5 Real-Time PCR System, is a qualitative nucleic acid-based multiplex in vitro diagnostic test for detection and differentiation of antibiotic resistance markers to one or more antimicrobial agents. The test utilizes real-time polymerase chain reaction (PCR) and is performed on isolated colonies of Pseudomonas aeruginosa, Enterococcus faecalis, or members of Enterobacterales grown in pure culture on blood agar or MacConkey agar. Organism identification results must be available prior to reporting results for the Acuitas AMR Gene Panel. Antimicrobial resistance gene results are reported by the Acuitas AMR Gene Panel for the combinations of bacterial pathogens and associated genetic resistance markers indicated below.</td></tr><tr><td></td><td>Organism (Reported AMR Gene Marker) Citrobacter freundii complex (CTX-M-1, CTX-M-9, KPC, NDM, OXA-48); Citrobacter koseri (KPC, OXA-48); Enterobacter cloacae complex (CTX-M- 1, CTX-M-9, KPC, TEM); Enterococcus faecalis (vanA); Escherichia coli (AAC, ANT, CMY, CTX- M-1, CTX-M-2, CTX-M-9, DFR, gyrA Mutant c, KPC, MCR-1, OXA-1, OXA-9, SHV, Sul1, Sul2, TEM); Klebsiella aerogenes (CTX-M-1, CTX-M-9, KPC, NDM, OXA-48); Klebsiella michiganensis (CTX-M-1, CTX-M-9, KPC, NDM, OXA-48); Klebsiella oxytoca (CTX-M-1, CTX-M-9, KPC, NDM, OXA-48); Klebsiella pneumoniae (AAC, AAD, APH, CMY, CTX-M-1, CTX-M- 9, DFR, DHA, IMP, KPC, NDM, OXA-1, OXA-9, OXA-48, quasipneumoniae (CTX-M-1, CTX-M-9, KPC, NDM, OXA-48); Klebsiella variicola (CTX-M-1,</td></tr><tr><td>potentially antimicrobial-resistant</td><td>RMT, Sul1, Sul2, TEM); Klebsiella CTX-M-9, KPC, NDM, OXA-48); Morganella morganii (CTX-M-1, KPC, NDM, OXA-48); Proteus mirabilis (AAC, ANT, APH, armA, CMY, CTX-M- 1, CTX-M-2, CTX-M-9, DFR, KPC, NDM, OXA-1, OXA-9, OXA-48, Sul2, TEM, VEB, VIM); Providencia rettgeri (NDM); Providencia stuartii (NDM); Pseudomonas aeruginosa (AAC, ANT, CTX-M-1, CTX-M-2, gyrA Mutant, KPC, NDM, OXA-1, PER, SHV, TEM, VEB, VIM); Raoultella ornithinolytica (KPC, NDM, OXA-48); Raoultella planticola (KPC); Serratia marcescens (CTX-M-1, CTX-M-9, KPC, NDM, OXA-48). The Acuitas AMR Gene Panel includes assays for the detection and reporting of genetic resistance markers associated with resistance to select drugs in the following antibiotic groups: aminoglycosides, carbapenems, cephalosporins, fluoroquinolones, penicillins, sulfonamides, trimethoprim, and vancomycin, to aid in the identification of</td></tr></table>

<table><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">genetic determinant MCR-1, a marker ofpublic health importance associated withreduced inhibitory activity of polymyxins.The results of the Acuitas AMR GenePanel for detection and identification ofgenetic determinants associated withantimicrobial resistance are used alongwith the Acuitas AMR Gene PanelElectronic User Guide (EUG). In certaincases, this information may be used as anaid to clinicians in the management ofpatients with known or suspectedantibiotic non-susceptible or resistantbacterial infections. The EUG containsinformation on the appropriateness ofreporting resistance markers detected bythe Acuitas AMR Gene Panel for claimedorganisms based on the strength of thecollective, totality of scientific evidencedelineating the level of associationbetween molecular marker detection withphenotypic, clinical resistance. Test resultsare not conclusive or prescriptive forlabeled use of any specific antimicrobialdrug product, and therefore, this testcannot be used in place of or to postponeor delay phenotypic antimicrobialsusceptibility testing.A "Detected" or "Not Detected" resultdoes not rule out the presence of otherantimicrobial resistance markers notdetected by the Acuitas AMR Gene Panel.A "Not Detected" result for a geneticmarker of antimicrobial resistance doesnot indicate susceptibility to associatedantimicrobial drugs or drug classes, asmultiple mechanisms of resistance mayexist.</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Technological Principles</td><td colspan="1" rowspan="1">Qualitative, real-time PCR</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Sample Type</td><td colspan="1" rowspan="1">Bacterial Isolates</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Detection Probes</td><td colspan="1" rowspan="1">TaqMan Probes</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Interpretation of Test Results</td><td colspan="1" rowspan="1">Diagnostic Software</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="2" rowspan="1">General Device Characteristic Differences</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Targets</td><td colspan="1" rowspan="1">Twenty-eight (28) gene sequences arereported, spanning different antibioticclassesAAC, AAD, ANT, APH, armA CMY,CTX-M-1, CTX-M-2, CTX-M-9, DFR,DHA, IMP, KPC, MCR-1, NDM, OXA-1, OXA-48, OXA-9, PER, RMT, SHV,SUL1, SUL2, TEM, vanA, VEB, VIM, P.</td><td colspan="1" rowspan="1">Detects blakpc, blandm, blavim, blaoxa-48, and blamp gene sequences5 gene targets associated with resistanceto one antibiotic class</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">aeruginosa gyrA Mutant (83-M), E. coligyrA Mutant (87-M1, 87-M2, 87-M3)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Technological Principles</td><td colspan="1" rowspan="1">Extraction with the QIAGEN EZ1Advanced XL System and nucleic acidamplification and target detection with theOpGen Qualified QS5 instrument with theAcuitas AMR Gene Panel Kit.</td><td colspan="1" rowspan="1">Fully-automated DNA extraction,nucleic acid amplification (DNA), andtarget detection</td></tr><tr><td colspan="1" rowspan="1">Reaction platform</td><td colspan="1" rowspan="1">Disposable 96-well plates</td><td colspan="1" rowspan="1">Disposable, single-use multi-chamberedfluidic cartridge</td></tr><tr><td colspan="1" rowspan="1">Controls</td><td colspan="1" rowspan="1">One Assay Control</td><td colspan="1" rowspan="1">Internal sample processing control(SPC) and probe check control (PCC)External controls available</td></tr><tr><td colspan="1" rowspan="1">Time to obtain Test Results</td><td colspan="1" rowspan="1">&lt; 3 hours</td><td colspan="1" rowspan="1">Approximately 50 minutes to results</td></tr><tr><td colspan="1" rowspan="1">Organisms</td><td colspan="1" rowspan="1">Select Enterobacterales, Pseudomonasaeruginosa, Enterococcus faecalis</td><td colspan="1" rowspan="1">Enterobacteriaceae, Pseudomonasaeruginosa, Acinetobacter baumannii</td></tr><tr><td colspan="1" rowspan="1">Instrument System</td><td colspan="1" rowspan="1">Data generated with the OpGen QualifiedQS5 instrument and analyzed with theAcuitas AMR Gene Analysis Software</td><td colspan="1" rowspan="1">GeneXpert Instrument System (includesGeneXpert Dx, Infinity-48, Infinity-48s,and Infinity-80)</td></tr></table>

# VI Standards/Guidance Documents Referenced:

• CLSI EP05-A3 Evaluation of Precision of Quantitative Measurement Procedures; Approved Guideline - Third Edition; FDA Recognition Number 7-251 CLSI EP24-A2 Assessment of the Diagnostic Accuracy of Laboratory Tests Using Receiver Operating Characteristic Curves; Approved Guideline – Second Edition; FDA Recognition Number 7-234   
• CLSI M100-S28. Performance Standard for Antimicrobial Susceptibility Testing; Approved standard—Twenty-eighth Informational Supplement, 2018; FDA Recognition Number 7-281   
• CLSI EP25-A, Evaluation of Stability of In Vitro Diagnostic Reagents; Approved Guideline - Volume 29, 2009; FDA Recognition Number 7-235 IEC 62304:2006/A1:2016 Medical Device Software - Software Lifecycle Processes; FDA Recognition Number 13-79 EMC-60601-1-2:2014 , Medical electrical equipment -- Part 1-2: General requirements for basic safety and essential performance--Collateral Standard: Electromagnetic disturbances-- Requirements and tests

# VII Performance Characteristics (if/when applicable):

# A Analytical Performance:

# 1. Precision/Reproducibility:

The Acuitas AMR Gene Panel Reproducibility study was performed at three sites—two external and one internal site. The study utilized two operators per site, alternating testing over twenty days. Each site performed testing on two QS5 instruments and one EZ1 Advanced XL instrument. At least three lots of all reagents/materials required to perform the Acuitas AMR Gene Panel were used throughout the reproducibility study. Reproducibility panels included bacterial suspensions prepared at an 0.5 McFarland concentration, blinded by OpGen personnel, frozen at ${ \bf - } 2 0 ^ { \circ } \mathrm { C }$ , and shipped to external testing sites. In total, each site tested 300 samples [composed of unique panel isolates, positive controls, negative controls, and no template controls]. During the execution of the Reproducibility Study, a total of (62)

positive external controls, (62) negative external controls, and (182) no template controls were tested. In all cases the controls passed by producing the expected results. The results are summarized in Table 2.

Overall results for each isolate tested in the reproducibility study (Table 3) are summarized in Table 4 to Table 13 below which show the number of tests and the percentage of correct results obtained (compared to expected). Reproducibility study results were acceptable. Table 14 summarizes the total false negatives and false positives observed during the Reproducibility Study, as well as a breakdown by organisms exhibiting these results. The Reproducibility Study also included a quantitative analysis of variance components, i.e., average Ct values and calculated SD and $\% \mathrm { C V }$ , which were determined to be acceptable for the assay.

Table 2. Daily QC Panela   

<table><tr><td rowspan=1 colspan=1>Control ID</td><td rowspan=1 colspan=1>Organism(Resistance Gene Markers)</td><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>% Agreementwith ExpectedResult</td><td rowspan=1 colspan=1>Lot Number</td></tr><tr><td rowspan=1 colspan=1>POSControl B</td><td rowspan=1 colspan=1>P. mirabilis(AAC, , armA, CMY, NDM,Sul2,TEM)</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>100% (30/30)</td><td rowspan=1 colspan=1>MN70061</td></tr><tr><td rowspan=1 colspan=1>POSControl C</td><td rowspan=1 colspan=1>K. pneumoniae(AAC, AAD, CTX-M-1, Sul1,Sul2, TEM)</td><td rowspan=1 colspan=1>32</td><td rowspan=1 colspan=1>100% (32/32)</td><td rowspan=1 colspan=1>MN70062</td></tr><tr><td rowspan=1 colspan=1>NegControl</td><td rowspan=1 colspan=1>S. aureus(No gene targets detected)</td><td rowspan=1 colspan=1>62</td><td rowspan=1 colspan=1>100% (62/62)</td><td rowspan=1 colspan=1>MN90133</td></tr><tr><td rowspan=1 colspan=1>NTC(no templatecontrol</td><td rowspan=1 colspan=1>None (Negative for all targets)b</td><td rowspan=1 colspan=1>182</td><td rowspan=1 colspan=1>100% (182/182)</td><td rowspan=1 colspan=1>N/A</td></tr></table>

a All external positive and negative controls gave the expected Acuitas AMR Gene Panel results on all days of testing at each testing site. b Contains PCR Certified Water added at PCR Set‐Up

Table 3. Isolates Tested in Reproducibility Panel   

<table><tr><td colspan="1" rowspan="1">SampleNumber</td><td colspan="1" rowspan="1">Organism</td><td colspan="1" rowspan="1">AMR Markers Present</td></tr><tr><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">ANT, DFR, gyraseA-MT, KPC, Sul1, Sul2, TEM</td></tr><tr><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">Sul2, TEM</td></tr><tr><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">DFR, DHA, Sul1, Sul2, TEM</td></tr><tr><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">K. pneumoniae</td><td colspan="1" rowspan="1">AAC, AAD, CMY, CTX-M-1, NDM, OXA-48, Sul1,Sul2, TEM</td></tr><tr><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">K. pneumoniae</td><td colspan="1" rowspan="1">AAC, CTX-M-1, IMP, OXA-1, Sul1, Sul2, TEM</td></tr><tr><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">P. aeruginosa</td><td colspan="1" rowspan="1">gyraseA-MT, PER, VIM</td></tr><tr><td colspan="1" rowspan="1">7</td><td colspan="1" rowspan="1">P. mirabilis</td><td colspan="1" rowspan="1">AAC, ANT, CMY, Sul2, TEM, VEB</td></tr><tr><td colspan="1" rowspan="1">8</td><td colspan="1" rowspan="1">P. mirabilis</td><td colspan="1" rowspan="1">AAC, APH, CTX-M-9, DFR, OXA-1, Sul2, TEM</td></tr><tr><td colspan="1" rowspan="1">9</td><td colspan="1" rowspan="1">E. faecalis</td><td colspan="1" rowspan="1">vanA</td></tr><tr><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1">NONE</td></tr></table>

Table 4. Reproducibility Results-E. coli (Sample 1)   

<table><tr><td rowspan=1 colspan=12>E. coli (Sample 1)</td></tr><tr><td rowspan=2 colspan=1>Targeta</td><td rowspan=2 colspan=1>ExpectedResult</td><td rowspan=1 colspan=3>Site 1</td><td rowspan=1 colspan=3>Site 2</td><td rowspan=1 colspan=3>Site 3</td><td rowspan=2 colspan=1>AllSites</td></tr><tr><td rowspan=1 colspan=1>Op1</td><td rowspan=1 colspan=1>Op2</td><td rowspan=1 colspan=1>Site 1</td><td rowspan=1 colspan=1>Op1</td><td rowspan=1 colspan=1>Op2</td><td rowspan=1 colspan=1>Site 2</td><td rowspan=1 colspan=1>Op1</td><td rowspan=1 colspan=1>Op2</td><td rowspan=1 colspan=1>Site 3</td></tr><tr><td rowspan=1 colspan=1>ANT</td><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>30/30(100%)</td><td rowspan=1 colspan=1>14/15(93%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>29/30(97%)</td><td rowspan=1 colspan=1>15/15100%</td><td rowspan=1 colspan=1>14/15(93%)</td><td rowspan=1 colspan=1>29/30(97%)</td><td rowspan=1 colspan=1>88/90(98%)</td></tr><tr><td rowspan=1 colspan=1>DFR</td><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>30/30(100%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>30/30(100%)</td><td rowspan=1 colspan=1>90/90(100%)</td></tr><tr><td rowspan=1 colspan=1>E. coli gyrAMT</td><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>14/15(93%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>29/30(97%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>14/15(93%)</td><td rowspan=1 colspan=1>29/30(97%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>30/30(100%)</td><td rowspan=1 colspan=1>88/90(989%)</td></tr><tr><td rowspan=1 colspan=1>KPC</td><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>30/30(100%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>30/30(100%)</td><td rowspan=1 colspan=1>90/90(100%)</td></tr><tr><td rowspan=1 colspan=1>Sul1</td><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>30/30(100%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>30/30(100%)</td><td rowspan=1 colspan=1>90/90(100%)</td></tr><tr><td rowspan=1 colspan=1>Sul2</td><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>30/30(100%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>30/30(100%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>30/30(100%)</td><td rowspan=1 colspan=1>90/90(100%)</td></tr><tr><td rowspan=1 colspan=1>TEM</td><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>15/15 (100%)</td><td rowspan=1 colspan=1>30/30(100%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>30/30(100%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>30/30(100%)</td><td rowspan=1 colspan=1>90/90(100%)</td></tr></table>

a No false positive results were observed for other Acuitas AMR Gene Targets to be reported with this organism.

Table 5. Reproducibility Results-E. coli (Sample 2)   

<table><tr><td rowspan=1 colspan=12>E. coli (Sample 2)</td></tr><tr><td rowspan=2 colspan=1>Targeta,b</td><td rowspan=2 colspan=1>ExpectedResult</td><td rowspan=1 colspan=3>Site 1</td><td rowspan=1 colspan=3>Site 2</td><td rowspan=1 colspan=3>Site 3</td><td rowspan=2 colspan=1>AllSites</td></tr><tr><td rowspan=1 colspan=1>Op1</td><td rowspan=1 colspan=1>Op2</td><td rowspan=1 colspan=1>Site 1</td><td rowspan=1 colspan=1>Op1</td><td rowspan=1 colspan=1>Op2</td><td rowspan=1 colspan=1>Site 2</td><td rowspan=1 colspan=1>Op1</td><td rowspan=1 colspan=1>Op2</td><td rowspan=1 colspan=1>Site 3</td></tr><tr><td rowspan=1 colspan=1>Sul2</td><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>14/14(100%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>29/29(100%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>30/30(100%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>30/30(10%)</td><td rowspan=1 colspan=1>89/89(100%)</td></tr><tr><td rowspan=1 colspan=1>TEM</td><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>14/14(100%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>29/29(100%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>30/30(100%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>30/30(100%)</td><td rowspan=1 colspan=1>89/89(100%)</td></tr><tr><td rowspan=1 colspan=1>CMY</td><td rowspan=1 colspan=1>NotDetected</td><td rowspan=1 colspan=1>13/14(93%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>28/29(97%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>30/30(100%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>30/30(100%)</td><td rowspan=1 colspan=1>88/89(9%)</td></tr></table>

a Except for CMY, all other Acuitas AMR Gene Panel Targets were not detected (did not generate FP results). b One replicate had invalid results due to an Assay Control Failure. The replicate was repeated using the same lot, instrument, and operator as the original testing event. The repeat data had $100 \%$ agreement with the expected results.

Table 6. Reproducibility Results- $E .$ . coli (Sample 3)   

<table><tr><td colspan="12" rowspan="1">E. coli (Sample 3)</td></tr><tr><td colspan="1" rowspan="2">Targeta</td><td colspan="1" rowspan="2">ExpectedResult</td><td colspan="3" rowspan="1">Site 1</td><td colspan="3" rowspan="1">Site 2</td><td colspan="3" rowspan="1">Site 3</td><td colspan="1" rowspan="2">AllSites</td></tr><tr><td colspan="1" rowspan="1">Op1</td><td colspan="1" rowspan="1">Op2</td><td colspan="1" rowspan="1">Site 1</td><td colspan="1" rowspan="1">Op1</td><td colspan="1" rowspan="1">Op2</td><td colspan="1" rowspan="1">Site 2</td><td colspan="1" rowspan="1">Op1</td><td colspan="1" rowspan="1">Op2</td><td colspan="1" rowspan="1">Site 3</td></tr><tr><td colspan="1" rowspan="1">DFR</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">15/15(100%)</td><td colspan="1" rowspan="1">15/15(100%)</td><td colspan="1" rowspan="1">30/30(100%)</td><td colspan="1" rowspan="1">15/15(100%)</td><td colspan="1" rowspan="1">15/15(100%)</td><td colspan="1" rowspan="1">30/30(100%)</td><td colspan="1" rowspan="1">15/15(100%)</td><td colspan="1" rowspan="1">15/15(100%)</td><td colspan="1" rowspan="1">30/30(100%)</td><td colspan="1" rowspan="1">90/90(100%)</td></tr><tr><td colspan="1" rowspan="1">DHA</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">15/15(100%)</td><td colspan="1" rowspan="1">15/15(100%)</td><td colspan="1" rowspan="1">30/30(100%)</td><td colspan="1" rowspan="1">15/15(100%)</td><td colspan="1" rowspan="1">15/15(100%)</td><td colspan="1" rowspan="1">30/30(100%)</td><td colspan="1" rowspan="1">15/15(100%)</td><td colspan="1" rowspan="1">15/15(100%)</td><td colspan="1" rowspan="1">30/30(100%)</td><td colspan="1" rowspan="1">90/90(100%)</td></tr><tr><td colspan="1" rowspan="1">Sull</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">15/15(100%)</td><td colspan="1" rowspan="1">15/15(100%)</td><td colspan="1" rowspan="1">30/30(100%)</td><td colspan="1" rowspan="1">15/15(100%)</td><td colspan="1" rowspan="1">15/15(100%)</td><td colspan="1" rowspan="1">30/30(100%)</td><td colspan="1" rowspan="1">15/15(100%)</td><td colspan="1" rowspan="1">15/15(100%)</td><td colspan="1" rowspan="1">30/30(100%)</td><td colspan="1" rowspan="1">90/90(100%)</td></tr><tr><td colspan="1" rowspan="1">Sul2</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">15/15(100%)</td><td colspan="1" rowspan="1">15/15(100%)</td><td colspan="1" rowspan="1">30/30(100%)</td><td colspan="1" rowspan="1">15/15(100%)</td><td colspan="1" rowspan="1">15/15(100%)</td><td colspan="1" rowspan="1">30/30(100%)</td><td colspan="1" rowspan="1">15/15(100%)</td><td colspan="1" rowspan="1">15/15(100%)</td><td colspan="1" rowspan="1">30/30(100%)</td><td colspan="1" rowspan="1">90/90(100%)</td></tr><tr><td colspan="1" rowspan="1">TEM</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">15/15(100%)</td><td colspan="1" rowspan="1">15/15(100%)</td><td colspan="1" rowspan="1">30/30(100%)</td><td colspan="1" rowspan="1">15/15(100%)</td><td colspan="1" rowspan="1">15/15(100%)</td><td colspan="1" rowspan="1">30/30(100%)</td><td colspan="1" rowspan="1">15/15(100%)</td><td colspan="1" rowspan="1">15/15(100%)</td><td colspan="1" rowspan="1">30/30(100%)</td><td colspan="1" rowspan="1">90/90(100%)</td></tr><tr><td colspan="1" rowspan="1">ANT</td><td colspan="1" rowspan="1">NotDetected</td><td colspan="1" rowspan="1">15/15(100%</td><td colspan="1" rowspan="1">15/15(100%)</td><td colspan="1" rowspan="1">30/30(100%)</td><td colspan="1" rowspan="1">15/15(100%)</td><td colspan="1" rowspan="1">14/15(93%)</td><td colspan="1" rowspan="1">29/30(97%)</td><td colspan="1" rowspan="1">15/15(100%)</td><td colspan="1" rowspan="1">15/15(100%)</td><td colspan="1" rowspan="1">30/30(100%)</td><td colspan="1" rowspan="1">89/90(99%)</td></tr><tr><td colspan="12" rowspan="1">"Except for ANT, all other Acuitas AMR Gene Panel Markers did not generate any false positive results.</td></tr></table>

Table 7. Reproducibility Results-K. pneumoniae (Sample 4)   

<table><tr><td rowspan=1 colspan=12>K. pneumoniae (Sample 4)</td></tr><tr><td rowspan=2 colspan=1>Targeta</td><td rowspan=2 colspan=1>ExpectedResult</td><td rowspan=1 colspan=3>Site 1</td><td rowspan=1 colspan=3>Site 2</td><td rowspan=1 colspan=3>Site 3</td><td rowspan=2 colspan=1>AllSites</td></tr><tr><td rowspan=1 colspan=1>Op1</td><td rowspan=1 colspan=1>Op2</td><td rowspan=1 colspan=1>Site 1</td><td rowspan=1 colspan=1>Op1</td><td rowspan=1 colspan=1>Op2</td><td rowspan=1 colspan=1>Site 2</td><td rowspan=1 colspan=1>Op1</td><td rowspan=1 colspan=1>Op2</td><td rowspan=1 colspan=1>Site 3</td></tr><tr><td rowspan=1 colspan=1>AAC</td><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>15/15100%)</td><td rowspan=1 colspan=1>30/30(100%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>30/30(100%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>30/30(100%)</td><td rowspan=1 colspan=1>90/90(100%)</td></tr><tr><td rowspan=1 colspan=1>AAD</td><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>30/30(100%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>30/30(100%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>30/30(100%)</td><td rowspan=1 colspan=1>90/90(100%)</td></tr><tr><td rowspan=1 colspan=1>CMY</td><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>30/30(100%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>30/30(100%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>30/30(100%)</td><td rowspan=1 colspan=1>90/90(100%)</td></tr><tr><td rowspan=1 colspan=1>CTX-M-1</td><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>30/30(100%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>30/30(100%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>30/30(100%)</td><td rowspan=1 colspan=1>90/90(100%)</td></tr><tr><td rowspan=1 colspan=1>NDM</td><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>30/30(100%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>30/30(100%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>30/30(100%)</td><td rowspan=1 colspan=1>90/90(100%)</td></tr><tr><td rowspan=1 colspan=1>OXA-48</td><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>30/30(100%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>30/30(100%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>30/30(100%)</td><td rowspan=1 colspan=1>90/90(100%)</td></tr><tr><td rowspan=1 colspan=1>Sul1</td><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>30/30(100%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>30/30(100%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>30/30(100%)</td><td rowspan=1 colspan=1>90/90(100%)</td></tr><tr><td rowspan=1 colspan=1>Sul2</td><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>30/30(100%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>30/30(100%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>30/30(100%)</td><td rowspan=1 colspan=1>90/90(100%)</td></tr><tr><td rowspan=1 colspan=1>TEM</td><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>30/30(100%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>14/15(93%)</td><td rowspan=1 colspan=1>29/30(97%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>30/30(100%)</td><td rowspan=1 colspan=1>89/90(99%)</td></tr><tr><td rowspan=1 colspan=1>IMP</td><td rowspan=1 colspan=1>NotDetected</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>15/15(10)</td><td rowspan=1 colspan=1>30/30(100%)</td><td rowspan=1 colspan=1>15/15</td><td rowspan=1 colspan=1>15/15</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>14/15(93%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>29/30(7%)</td><td rowspan=1 colspan=1>89/90( %)</td></tr></table>

a Except for IMP, all other Acuitas AMR Gene Panel Markers did not generate any false positive results.

Table 8. Reproducibility Results-K. pneumoniae (Sample 5)   

<table><tr><td colspan="12" rowspan="1">K. pneumoniae (Sample 5)</td></tr><tr><td colspan="1" rowspan="2">Targeta</td><td colspan="1" rowspan="2">ExpectedResult</td><td colspan="3" rowspan="1">Site 1</td><td colspan="3" rowspan="1">Site 2</td><td colspan="3" rowspan="1">Site 3</td><td colspan="1" rowspan="2">AllSites</td></tr><tr><td colspan="1" rowspan="1">Op1</td><td colspan="1" rowspan="1">Op2</td><td colspan="1" rowspan="1">Site 1</td><td colspan="1" rowspan="1">Op1</td><td colspan="1" rowspan="1">Op2</td><td colspan="1" rowspan="1">Site 2</td><td colspan="1" rowspan="1">Op1</td><td colspan="1" rowspan="1">Op2</td><td colspan="1" rowspan="1">Site 3</td></tr><tr><td colspan="1" rowspan="2">AAC</td><td colspan="1" rowspan="2">Detected</td><td colspan="1" rowspan="2">15/15(100%)</td><td colspan="1" rowspan="2">15/15(100%)</td><td colspan="1" rowspan="2">30/30(100%)</td><td colspan="1" rowspan="2">15/15(100%)</td><td colspan="1" rowspan="2">15/15(100%)</td><td colspan="1" rowspan="2">30/30(100%)</td><td colspan="1" rowspan="2">15/15(100%)</td><td colspan="1" rowspan="2">15/15(100%)</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">90/90</td></tr><tr><td colspan="1" rowspan="1">(100%)</td><td colspan="1" rowspan="1">(100%)</td></tr><tr><td colspan="1" rowspan="1">CTX-M-1</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">15/15(100%)</td><td colspan="1" rowspan="1">15/15(100%)</td><td colspan="1" rowspan="1">30/30(100%)</td><td colspan="1" rowspan="1">15/15(100%)</td><td colspan="1" rowspan="1">15/15(100%)</td><td colspan="1" rowspan="1">30/30(100%)</td><td colspan="1" rowspan="1">15/15(100%)</td><td colspan="1" rowspan="1">15/15(100%)</td><td colspan="1" rowspan="1">30/30(100%)</td><td colspan="1" rowspan="1">90/90(100%)</td></tr><tr><td colspan="1" rowspan="1">IMP</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">15/15(100%)</td><td colspan="1" rowspan="1">15/15(100%)</td><td colspan="1" rowspan="1">30/30(100%)</td><td colspan="1" rowspan="1">15/15(100%)</td><td colspan="1" rowspan="1">15/15(100%)</td><td colspan="1" rowspan="1">30/30(100%)</td><td colspan="1" rowspan="1">15/15(100%)</td><td colspan="1" rowspan="1">15/15(100%)</td><td colspan="1" rowspan="1">30/30(100%)</td><td colspan="1" rowspan="1">90/90(100%)</td></tr><tr><td colspan="1" rowspan="1">OXA-1</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">15/15(100%)</td><td colspan="1" rowspan="1">15/15(100%)</td><td colspan="1" rowspan="1">30/30(100%)</td><td colspan="1" rowspan="1">15/15(100%)</td><td colspan="1" rowspan="1">15/15(100%)</td><td colspan="1" rowspan="1">30/30(100%)</td><td colspan="1" rowspan="1">15/15(100%)</td><td colspan="1" rowspan="1">15/15(100%)</td><td colspan="1" rowspan="1">30/30(100%)</td><td colspan="1" rowspan="1">90/90(100%)</td></tr><tr><td colspan="1" rowspan="2">Sul1</td><td colspan="1" rowspan="2">Detected</td><td colspan="1" rowspan="2">15/15(100%)</td><td colspan="1" rowspan="2">15/15(100%)</td><td colspan="1" rowspan="2">30/30(100%)</td><td colspan="1" rowspan="2">15/15(100%)</td><td colspan="1" rowspan="2">15/15(100%)</td><td colspan="1" rowspan="2">30/30(100%)</td><td colspan="1" rowspan="1">15/15</td><td colspan="1" rowspan="1">15/15</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">90/90</td></tr><tr><td colspan="1" rowspan="1">(100%)</td><td colspan="1" rowspan="1">(100%)</td><td colspan="1" rowspan="1">(100%)</td><td colspan="1" rowspan="1">(100%)</td></tr><tr><td colspan="1" rowspan="1">Sul2</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">15/15(100%)</td><td colspan="1" rowspan="1">15/15(100%)</td><td colspan="1" rowspan="1">30/30(100%)</td><td colspan="1" rowspan="1">15/15(100%)</td><td colspan="1" rowspan="1">15/15(100%)</td><td colspan="1" rowspan="1">30/30(100%)</td><td colspan="1" rowspan="1">15/15(100%)</td><td colspan="1" rowspan="1">15/15(100%)</td><td colspan="1" rowspan="1">30/30(100%)</td><td colspan="1" rowspan="1">90/90(100%)</td></tr><tr><td colspan="1" rowspan="1">TEM</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">15/15(100%)</td><td colspan="1" rowspan="1">15/15(100%)</td><td colspan="1" rowspan="1">30/30(100%)</td><td colspan="1" rowspan="1">15/15(100%)</td><td colspan="1" rowspan="1">15/15(100%)</td><td colspan="1" rowspan="1">30/30(100%)</td><td colspan="1" rowspan="1">15/15(100%)</td><td colspan="1" rowspan="1">15/15(100%)</td><td colspan="1" rowspan="1">30/30(100%)</td><td colspan="1" rowspan="1">90/90(100%)</td></tr><tr><td colspan="1" rowspan="1">ANT</td><td colspan="1" rowspan="1">NotDetected</td><td colspan="1" rowspan="1">15/15(100%)</td><td colspan="1" rowspan="1">15/15(100%)</td><td colspan="1" rowspan="1">30/30(100%)</td><td colspan="1" rowspan="1">15/15(100%)</td><td colspan="1" rowspan="1">15/15(100%)</td><td colspan="1" rowspan="1">30/30(100%)</td><td colspan="1" rowspan="1">14/15(93%)</td><td colspan="1" rowspan="1">15/15(100%)</td><td colspan="1" rowspan="1">29/30(97%)</td><td colspan="1" rowspan="1">89/90(99%)</td></tr><tr><td colspan="1" rowspan="2">CTX-M-9</td><td colspan="1" rowspan="2">NotDetected</td><td colspan="1" rowspan="2">15/15(100%)</td><td colspan="1" rowspan="2">15/15</td><td colspan="1" rowspan="2">30/30</td><td colspan="1" rowspan="2">15/15</td><td colspan="1" rowspan="2">14/15(93%)</td><td colspan="1" rowspan="2">29/30</td><td colspan="1" rowspan="1">15/15</td><td colspan="1" rowspan="1">15/15</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">89/90</td></tr><tr><td colspan="1" rowspan="1">(100%)</td><td colspan="1" rowspan="1">(100%)</td><td colspan="1" rowspan="1">(100%)</td><td colspan="1" rowspan="1">(99%)</td></tr><tr><td colspan="12" rowspan="1">"Except for ANT and CTX-M-9, all other Acuitas AMR Gene Panel Markers did not generate any false positiveresults.</td></tr></table>

Table 9. Reproducibility Results-P.aeruginosa (Sample 6)   

<table><tr><td rowspan=1 colspan=12>P. aeruginosa (Sample 6)</td></tr><tr><td rowspan=2 colspan=1>Targeta</td><td rowspan=2 colspan=1>ExpectedResult</td><td rowspan=1 colspan=3>Site 1</td><td rowspan=1 colspan=3>Site 2</td><td rowspan=1 colspan=3>Site 3</td><td rowspan=2 colspan=1>AllSites</td></tr><tr><td rowspan=1 colspan=1>Op1</td><td rowspan=1 colspan=1>Op2</td><td rowspan=1 colspan=1>Site 1</td><td rowspan=1 colspan=1>Op1</td><td rowspan=1 colspan=1>Op2</td><td rowspan=1 colspan=1>Site 2</td><td rowspan=1 colspan=1>Op1</td><td rowspan=1 colspan=1>Op2</td><td rowspan=1 colspan=1>Site 3</td></tr><tr><td rowspan=1 colspan=1>P.aeruginosagyrA MT</td><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>14/1593%)</td><td rowspan=1 colspan=1>29/30 97%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>30/30(100%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>14/15(93%)</td><td rowspan=1 colspan=1>29/3097%)</td><td rowspan=1 colspan=1>88/90(98%)</td></tr><tr><td rowspan=1 colspan=1>PER</td><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>30/30(100%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>30/30(100%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>30/30(100%)</td><td rowspan=1 colspan=1>90/90(100%)</td></tr><tr><td rowspan=1 colspan=1>VIM</td><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>30/30(100%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>30/30(100%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>30/30(100%)</td><td rowspan=1 colspan=1>90/90(100%)</td></tr><tr><td rowspan=1 colspan=1>ANT</td><td rowspan=1 colspan=1>NotDetected</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>30/30(100%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>15/15</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>14/15(93%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>29/30(97%)</td><td rowspan=1 colspan=1>89/90(99%)</td></tr><tr><td rowspan=1 colspan=12>&quot;Except for ANT, all other Acuitas AMR Gene Panel Markers did not generate any false positive results.</td></tr></table>

Table 10. Reproducibility Results-P.mirabilis (Sample 7)   

<table><tr><td rowspan=1 colspan=12>P. mirabilis (Sample 7)</td></tr><tr><td rowspan=2 colspan=1>Targeta</td><td rowspan=2 colspan=1>ExpectedResult</td><td rowspan=1 colspan=3>Site 1</td><td rowspan=1 colspan=3>Site 2</td><td rowspan=1 colspan=3>Site 3</td><td rowspan=2 colspan=1>AllSites</td></tr><tr><td rowspan=1 colspan=1>Op1</td><td rowspan=1 colspan=1>Op2</td><td rowspan=1 colspan=1>Site 1</td><td rowspan=1 colspan=1>Op1</td><td rowspan=1 colspan=1>Op2</td><td rowspan=1 colspan=1>Site 2</td><td rowspan=1 colspan=1>Op1</td><td rowspan=1 colspan=1>Op2</td><td rowspan=1 colspan=1>Site 3</td></tr><tr><td rowspan=1 colspan=1>AAC</td><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>30/30(100%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>30/30(100%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>30/30(100%)</td><td rowspan=1 colspan=1>90/90(100%)</td></tr><tr><td rowspan=1 colspan=1>ANT</td><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>30/30(100%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>30/30(100%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>14/15(93%)</td><td rowspan=1 colspan=1>29/30(97%)</td><td rowspan=1 colspan=1>89/90(99%)</td></tr><tr><td rowspan=1 colspan=1>CMY</td><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>30/30(100%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>30/30(100%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>30/30(100%)</td><td rowspan=1 colspan=1>90/90(100%)</td></tr><tr><td rowspan=1 colspan=1>Sul2</td><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>30/30(100%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>14/15(93%)</td><td rowspan=1 colspan=1>29/30(97%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>30/30(100%)</td><td rowspan=1 colspan=1>89/90(99%)</td></tr><tr><td rowspan=1 colspan=1>TEM</td><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>30/30(100%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>30/30(100%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>30/30(100%)</td><td rowspan=1 colspan=1>90/90(100%)</td></tr><tr><td rowspan=1 colspan=1>VEB</td><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>30/30(100%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>30/30(100%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>30/30(100%)</td><td rowspan=1 colspan=1>90/90(100%)</td></tr><tr><td rowspan=1 colspan=3>"All other Acuitas AMR Gene Pane</td><td rowspan=1 colspan=9>&quot;All other Acuitas AMR Gene Panel Markers did not generate any false positive results.</td></tr></table>

Table 11. Reproducibility Results-P.mirabilis (Sample 8)   

<table><tr><td rowspan=1 colspan=12>P. mirabilis (Sample 8)</td></tr><tr><td rowspan=2 colspan=1>Targeta,b</td><td rowspan=2 colspan=1>ExpectedResult</td><td rowspan=1 colspan=3>Site 1</td><td rowspan=1 colspan=3>Site 2</td><td rowspan=1 colspan=3>Site 3</td><td rowspan=2 colspan=1>AllSites</td></tr><tr><td rowspan=1 colspan=1>Op1</td><td rowspan=1 colspan=1>Op2</td><td rowspan=1 colspan=1>Site 1</td><td rowspan=1 colspan=1>Op1</td><td rowspan=1 colspan=1>Op2b</td><td rowspan=1 colspan=1>Site 2</td><td rowspan=1 colspan=1>Op1</td><td rowspan=1 colspan=1>Op2</td><td rowspan=1 colspan=1>Site 3</td></tr><tr><td rowspan=1 colspan=1>AAC</td><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>30/30(100%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>14/14(100%)</td><td rowspan=1 colspan=1>29/29(100%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>30/30(100%)</td><td rowspan=1 colspan=1>89/89(100%)</td></tr><tr><td rowspan=1 colspan=1>APH</td><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>30/30(100%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>14/14(100%)</td><td rowspan=1 colspan=1>29/29(100%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>30/30(100%)</td><td rowspan=1 colspan=1>89/89(100%)</td></tr><tr><td rowspan=1 colspan=1>CTX-M-9</td><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>30/30(100%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>14/14(100%)</td><td rowspan=1 colspan=1>29/29(100%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>30/30(100%)</td><td rowspan=1 colspan=1>89/89(100%)</td></tr><tr><td rowspan=1 colspan=1>DFR</td><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>30/30(100%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>14/14(100%)</td><td rowspan=1 colspan=1>29/29(100%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>30/30(100%)</td><td rowspan=1 colspan=1>89/89(100%)</td></tr><tr><td rowspan=1 colspan=1>OXA-1</td><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>30/30(100%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>14/14(100%)</td><td rowspan=1 colspan=1>29/29(100%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>30/30(100%)</td><td rowspan=1 colspan=1>89/89(100%)</td></tr><tr><td rowspan=1 colspan=1>Sul2</td><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>30/30(100%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>14/14(100%)</td><td rowspan=1 colspan=1>29/29(100%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>30/30(100%)</td><td rowspan=1 colspan=1>89/89(100%)</td></tr><tr><td rowspan=1 colspan=1>TEM</td><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>30/30(100%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>14/14(100%)</td><td rowspan=1 colspan=1>29/29(100%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>30/30(100%)</td><td rowspan=1 colspan=1>89/89(100%)</td></tr><tr><td rowspan=1 colspan=1>OXA-48</td><td rowspan=1 colspan=1>NotDetected</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>30/30(100%)</td><td rowspan=1 colspan=1>14/15(93%)</td><td rowspan=1 colspan=1>14/14(100%)</td><td rowspan=1 colspan=1>28/29(97%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>30/30(100%)</td><td rowspan=1 colspan=1>88/89(99%)</td></tr></table>

a Except for OXA-48, all other Acuitas AMR Gene Panel Targets did not generate any false positive results. b One replicate had invalid results due to an Assay Control Failure. The replicate was repeated using the same lot, instrument, and operator as the original testing event. The repeat data had $100 \%$ agreement with the expected results.

Table 12. Reproducibility Results-P.mirabilis (Sample 9)   

<table><tr><td colspan="10">E. faecalis (Sample 9)</td></tr><tr><td rowspan="2">Targeta</td><td rowspan="2">Expected Result</td><td colspan="3">Site 1</td><td colspan="3">Site 2</td><td colspan="3">Site 3</td><td rowspan="2">All Sites</td></tr><tr><td>Op1</td><td>Op2</td><td>Site 1</td><td>Op1</td><td>Op2</td><td>Site 2</td><td>Op1</td><td>Op2</td><td>Site 3</td></tr><tr><td rowspan="2">vanA</td><td rowspan="2">Detected</td><td>15/15</td><td>15/15</td><td>30/30</td><td>15/15</td><td>15/15</td><td>30/30</td><td>15/15</td><td>15/15</td><td>30/30</td><td>90/90</td></tr><tr><td>(100%)</td><td>(100%)</td><td>(100%)</td><td>(100%)</td><td>(100%)</td><td>(100%)</td><td>(100%)</td><td>(100%)</td><td>(100%)</td><td>(100%)</td></tr><tr><td colspan="10">aNo other Acuitas AMR Gene Panel Marker generated a false positive result.</td></tr></table>

Table 13. Reproducibility Results-S. aureus (Sample 10)   

<table><tr><td rowspan=1 colspan=12>S. aureus (Sample 10)</td></tr><tr><td rowspan=2 colspan=1>Targeta</td><td rowspan=2 colspan=1>ExpectedResult</td><td rowspan=1 colspan=3>Site 1</td><td rowspan=1 colspan=3>Site 2</td><td rowspan=1 colspan=3>Site 3</td><td rowspan=2 colspan=1>AllSites</td></tr><tr><td rowspan=1 colspan=1>Op1</td><td rowspan=1 colspan=1>Op2</td><td rowspan=1 colspan=1>Site 1</td><td rowspan=1 colspan=1>Op1</td><td rowspan=1 colspan=1>Op2</td><td rowspan=1 colspan=1>Site 2</td><td rowspan=1 colspan=1>Op1</td><td rowspan=1 colspan=1>Op2</td><td rowspan=1 colspan=1>Site 3</td></tr><tr><td rowspan=2 colspan=1>AAC</td><td rowspan=2 colspan=1>NotDetected</td><td rowspan=2 colspan=1>15/15(100%)</td><td rowspan=2 colspan=1>15/15(100%)</td><td rowspan=2 colspan=1>30/30(100%)</td><td rowspan=2 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>15/15</td><td rowspan=2 colspan=1>30/30(100%)</td><td rowspan=2 colspan=1>15/15(100%)</td><td rowspan=2 colspan=1>14/15(93%)</td><td rowspan=2 colspan=1>29/30(97%)</td><td rowspan=2 colspan=1>89/90(99%)</td></tr><tr><td rowspan=1 colspan=1>(100%)</td></tr><tr><td rowspan=1 colspan=12>xpt orAC,al other cuits AMR Gene Panel were not detected in this srain. This strain serves as aativecontrol for the assay.</td></tr></table>

Table 14. Summary of all Acuitas AMR Gene Panel Reproducibility False Positive (FP) and False Negative (FN) results   

<table><tr><td colspan="1" rowspan="2">AMR Gene</td><td colspan="3" rowspan="1">False Positives</td><td colspan="3" rowspan="1">False Negatives</td></tr><tr><td colspan="1" rowspan="1">TotalFP</td><td colspan="1" rowspan="1">Species w/ FP</td><td colspan="1" rowspan="1"># FPperSpecies</td><td colspan="1" rowspan="1">TotalFN</td><td colspan="1" rowspan="1">Species w/FN</td><td colspan="1" rowspan="1"># FN perSpecies</td></tr><tr><td colspan="1" rowspan="1">AAC</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td></tr><tr><td colspan="1" rowspan="1">AAD</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td></tr><tr><td colspan="1" rowspan="3">ANT</td><td colspan="1" rowspan="3">3</td><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="3">3</td><td colspan="1" rowspan="2">E. coli</td><td colspan="1" rowspan="2">2</td></tr><tr><td colspan="1" rowspan="1">K. pneumoniae</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">P. aeruginosa</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">P. mirabilis</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">CMY</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td></tr><tr><td colspan="1" rowspan="1">CTX-M-9</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">K. pneumoniae</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td></tr><tr><td colspan="1" rowspan="1">IMP</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">K. pneumoniae</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td></tr><tr><td colspan="1" rowspan="1">OXA-48</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">P. mirabilis</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td></tr><tr><td colspan="1" rowspan="1">Sul2</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">P. mirabilis</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">TEM</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">K. pneumoniae</td><td colspan="1" rowspan="1">1</td></tr></table>

2. Linearity:

N/A

# 3. Analytical Specificity/Interference:

a. Cross-Reactivity

Potential cross-reactivity was investigated for the Acuitas AMR Gene Panel with a total of four hundred and twenty-three (423) isolates. Cross-reactivity was also evaluated for the test’s species identification assays for $E$ . coli and $P .$ . aeruginosa, which are used in conjunction with the mutant gyrase assays in reporting the mutant gyrase gene for only these two organisms. The Acuitas AMR Gene Panel results were compared with species identification by well-established automated species identification methods and for AMR gene detection by Whole Genome Sequencing (WGS).

Each isolate was tested in duplicate from a single $0 . 5 \mathrm { M c F }$ arland bacterial suspension. All testing results were compared to corresponding WGS results in order to assess crossreactivity for those resistance genes harbored by a given isolate. In the event of any observed cross-reactivity, serial dilutions were performed, and the isolate was retested until cross-reactivity was no longer observed. Organisms evaluated during the CrossReactivity Study are shown in Table 15 below. Appropriate controls were performed during the execution of the Cross-Reactivity Study. A total of 41 Positive External Controls and 40 Negative External Controls were tested. In total, there were six External Control failures: Three (3) Positive Control A failures, one (1) Positive Control C failure, and two (2) Negative Control failures. One Positive Control C failure was not repeated, but all other failures were re-tested and produced expected results.

Table 15. Cross-Reactivity Panel   

<table><tr><td colspan="3" rowspan="1">Bacterial Species</td></tr><tr><td colspan="1" rowspan="1">Achromobacter xylosoxidans</td><td colspan="1" rowspan="1">Enterococcus gallinarum</td><td colspan="1" rowspan="1">Raoultella planticola</td></tr><tr><td colspan="1" rowspan="1">Acinetobacter baumanniicomplex</td><td colspan="1" rowspan="1">Enterococcus gilvus</td><td colspan="1" rowspan="1">Salmonella enterica</td></tr><tr><td colspan="1" rowspan="1">Acinetobacter ursingii</td><td colspan="1" rowspan="1">Enterococcus hiraec</td><td colspan="1" rowspan="1">Salmonella species</td></tr><tr><td colspan="1" rowspan="1">Aeromonas hydrophila</td><td colspan="1" rowspan="1">Enterococcus italicus</td><td colspan="1" rowspan="1">Serratia marcescens</td></tr><tr><td colspan="1" rowspan="1">Candida albicans</td><td colspan="1" rowspan="1">Enterococcus pseudoavium</td><td colspan="1" rowspan="1">Serratia plymuthica</td></tr><tr><td colspan="1" rowspan="1">Citrobacter braakii</td><td colspan="1" rowspan="1">Enterococcus raffinosus</td><td colspan="1" rowspan="1">Shigella boydiid</td></tr><tr><td colspan="1" rowspan="1">Citrobacter freundii complex</td><td colspan="1" rowspan="1">Hafnia alvei</td><td colspan="1" rowspan="1">Shigella dysenteriaed</td></tr><tr><td colspan="1" rowspan="1">Citrobacter koseri</td><td colspan="1" rowspan="1">Klebsiella oxytoca</td><td colspan="1" rowspan="1">Sphingomonas paucimobilis</td></tr><tr><td colspan="1" rowspan="1">Citrobacter youngae</td><td colspan="1" rowspan="1">Klebsiella oxytoca ESBL</td><td colspan="1" rowspan="1">Staphylococcus aureus</td></tr><tr><td colspan="1" rowspan="1">Clostridioides (Clostridium)difficile</td><td colspan="1" rowspan="1">Leclercia adecarboxylata</td><td colspan="1" rowspan="1">Staphylococcus capitis</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium diphtheriae</td><td colspan="1" rowspan="1">Moraxella catarrhalisc</td><td colspan="1" rowspan="1">Staphylococcus epidermidis</td></tr><tr><td colspan="1" rowspan="1">Escherichia fergusoniia</td><td colspan="1" rowspan="1">Morganella morganii sspmorganii</td><td colspan="1" rowspan="1">Staphylococcus haemolyticus</td></tr><tr><td colspan="1" rowspan="1">Escherichia hermanniib</td><td colspan="1" rowspan="1">Proteus vulgaris</td><td colspan="1" rowspan="1">Staphylococcus hominis</td></tr><tr><td colspan="1" rowspan="1">Enterobacter aerogenes</td><td colspan="1" rowspan="1">Providencia rettgeri</td><td colspan="1" rowspan="1">Staphylococcus lugdunensis</td></tr><tr><td colspan="1" rowspan="1">Enterobacter cloacae</td><td colspan="1" rowspan="1">Providencia stuartii</td><td colspan="1" rowspan="1">Staphylococcus saprophyticus</td></tr><tr><td colspan="1" rowspan="1">Enterobacter cloacae complex</td><td colspan="1" rowspan="1">Pseudomonas fluorescens</td><td colspan="1" rowspan="1">Staphylococcus warneri</td></tr><tr><td colspan="1" rowspan="1">Enterobacter hormaecheic</td><td colspan="1" rowspan="1">Pseudomonas luteola</td><td colspan="1" rowspan="1">Streptococcus agalactiae</td></tr><tr><td colspan="1" rowspan="1">Enterococcus disparc</td><td colspan="1" rowspan="1">Pseudomonas oryzihabitans</td><td colspan="1" rowspan="1">Streptococcus pyogenes</td></tr><tr><td colspan="1" rowspan="1">Enterococcus durans</td><td colspan="1" rowspan="1">Pseudomonas putida</td><td colspan="1" rowspan="1">Yersinia pseudotuberculosis</td></tr><tr><td colspan="1" rowspan="1">Enterococcus faecium</td><td colspan="1" rowspan="1">Pseudomonas stutzeri</td><td colspan="1" rowspan="1"></td></tr></table>

a One isolate was identified as Escherichia fergusonii by ATCC and by WGS, whereas automated species identification methods identified it as $E$ . coli. The Acuitas AMR Gene Panel reported true negative for $P .$ . aeruginosa species $I D$ and false negative for $E$ . coli species $I D$ with this isolate based on automated species identification methods designation as $E .$ . coli. This isolate is included as $E .$ . fergusonii based on ATCC and WGS designation. b One isolate was identified as Escherichia hermannii by ATCC and by WGS. Automated species identification methods identified the isolate as E. coli. The Acuitas AMR Gene Panel reported true negative for P. aeruginosa species $I D$ and false negative for $E .$ . coli species $I D$ with this isolate based on Automated species identification methods designation as E. coli. This isolate is included as E. hermannii based on ATCC and WGS designation. c These isolates are included based on species designations by ATCC and/or WGS.   
d Some but not all isolates of Shigella boydii and Shigella dysenteriae cross-react with the E. coli species ID assay of the Acuitas AMR Gene Panel.

Table 16 below shows false positive (FP) results for organisms tested in the CrossReactivity Study without inclusion of results from the Analytical Reactivity (Inclusivity) Study.

Table 16. False Positives Observed in Cross-Reactivity Study for Acuitas AMR Gene Targets   

<table><tr><td rowspan=2 colspan=1>Marker</td><td rowspan=1 colspan=2>Cross Reactivity Study Isolates</td><td rowspan=2 colspan=1>Species WhereFP Observed</td></tr><tr><td rowspan=1 colspan=1># Times PositiveReported / TotalSamples where Targetwas not Expected</td><td rowspan=1 colspan=1>% FP</td></tr><tr><td rowspan=1 colspan=1>ANT</td><td rowspan=1 colspan=1>1/227</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>C. koseri</td></tr><tr><td rowspan=1 colspan=1>CMY</td><td rowspan=1 colspan=1>2/229</td><td rowspan=1 colspan=1>0.9</td><td rowspan=1 colspan=1>C. braakii</td></tr><tr><td rowspan=1 colspan=1>OXA-9</td><td rowspan=1 colspan=1>1/227</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>S. marcescens</td></tr><tr><td rowspan=1 colspan=1>OXA-48</td><td rowspan=1 colspan=1>1/221</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>A. hydrophila</td></tr><tr><td rowspan=1 colspan=1>SUL1</td><td rowspan=1 colspan=1>4/207</td><td rowspan=1 colspan=1>1.9</td><td rowspan=1 colspan=1>P. vulgaris</td></tr><tr><td rowspan=1 colspan=1>VIM</td><td rowspan=1 colspan=1>1/229</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>S. marcescens</td></tr></table>

Organisms were also evaluated for cross-reacting with $E .$ . coli and $P _ { - }$ . aeruginosa species identification assays (used only in algorithm for gyrase mutant identification) of the Acuitas AMR Gene Panel. No FP were observed, except for Shigella boydii, Shigella dysenteriae, Shigella flexneri and Shigella sonnei, which cross-reacted with the $E .$ . coli ID assay on the Acuitas AMR Gene Panel as expected from in silico analysis. One isolate of Shigella boydii (ATCC 9207) and one isolate of Shigella dysenteriae (ATCC 49347) did not cross-react with the $E .$ . coli ID assay. All Shigella strains demonstrating crossreactivity with the E. coli ID assay had wild-type gyrase sequences. Shigella species are not claimed for the assay and will not be reported for the test system. This information was included as a limitation in the package insert.

Cross-reactivity with other genes associated with antimicrobial resistance was not observed for any AMR gene assays on the Acuitas AMR Gene Panel.

4. Assay Reportable Range:

N/A

5. Traceability, Stability, Expected Values (Controls, Calibrators, or Methods): a. External Controls

External processing controls consisting of external positive control and negative control organisms were included during analytical and clinical studies evaluating the Acuitas AMR Gene Panel. On each testing day, one negative control and one individual bacterial positive control were tested. Table 17 below shows a list of the external controls prepared and tested during the clinical study (stock and prospective study).

Table 17. Performance of External Controls (Clinical Study)   

<table><tr><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Target Genes</td><td rowspan=1 colspan=1>Observed/Expected</td></tr><tr><td rowspan=1 colspan=1>Positive Control A</td><td rowspan=1 colspan=1>E. coli</td><td rowspan=1 colspan=1>AAD, CMY</td><td rowspan=1 colspan=1>15/16</td></tr><tr><td rowspan=1 colspan=1>Positive Control B</td><td rowspan=1 colspan=1>P. mirabilis</td><td rowspan=1 colspan=1>AAC, AAD,armA, CMY,NDM, SUL-1,SUL-2, TEM</td><td rowspan=1 colspan=1>16/16</td></tr><tr><td rowspan=1 colspan=1>Positive Control C</td><td rowspan=1 colspan=1>K. pneumoniae</td><td rowspan=1 colspan=1>AAC, AAD,CTX-M-1,SHV, SUL-1,SUL-2,TEM</td><td rowspan=1 colspan=1>14/15</td></tr><tr><td rowspan=1 colspan=1>Negative Control</td><td rowspan=1 colspan=1>S. aureus</td><td rowspan=1 colspan=1>Negative forall targets</td><td rowspan=1 colspan=1>61/63</td></tr></table>

For the analytical performance evaluation, sets of positive external controls representing a variety of organisms and AR Gene targets and one negative control (negative for all targets) were utilized. Individual analytical performance protocols specified the use of these daily external controls, requiring at least one positive external and one negative external control be tested per testing event with valid results in order for data to be included in a given study. Of the 431 external control samples run, $9 9 . 1 \%$ (427/431) gave a valid result on the first attempt. Only sample test results with valid external controls were included in the analysis of assay performance.

# b. Assay Controls

The kit also includes eight (8) tubes of dried internal amplification control (Assay Control). The Assay Control for the Acuitas AMR Gene Panel test is Bacillus globigii (ATCC 31028), and it monitors DNA extraction efficiency and PCR inhibition for all pathogens and resistance genes. Since the Bacillus Assay Control test is in a PCR well by itself (one-plex assay), it should be positive independent of the other PCR assays. The Assay Control provided with the Acuitas AMR Gene Panel Test is added to each sample at the start of the extraction process.

c. Stability

(i) Sample DNA Stability

A study was conducted to evaluate the stability of DNA extracted from pure bacterial colonies using the QIAGEN EZ1 Advanced XL System with the Acuitas AMR Gene Panel after storage at either $1 5 { - } 2 5 ^ { \circ } \mathrm { C }$ or $2 { - } 8 ^ { \circ } \mathrm { C }$ . Testing was performed by one operator across two QS 5 instruments. Three replicates of each isolate (5 isolates) were evaluated. Test results demonstrated that extracted DNA from a 0.5 McFarland suspension for use with Acuitas AMR Gene Panel is stable for up to 6 hours at 15- $2 5 \mathrm { { ^ \circ C } }$ (room temperature), up to 7 days at $4 ^ { \circ } \mathrm { C }$ $\left( 2 \mathrm { - } 8 ^ { \circ } \mathrm { C } \right)$ , and up to 6 hours at room temperature followed by up to 7 days at $2 { - } 8 ~ ^ { \circ } \mathrm { C }$ . The study panel consisted of the following organisms and target gene panel members as shown in Table 18 below.

Table 18. Organism and Resistance Marker Panel for Sample DNA Stability Study   

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Gene Marker</td></tr><tr><td rowspan=1 colspan=1>E. coli</td><td rowspan=1 colspan=1>AAC, CTX-M-9, DFR, EC-gyrA-MT, SUL1, SUL2,TEM</td></tr><tr><td rowspan=1 colspan=1>K. pneumoniae</td><td rowspan=1 colspan=1>AAC, AAD, armA, DHA, KP-gyrA-MT, NDM, SHV,SUL1, SUL2</td></tr><tr><td rowspan=1 colspan=1>P. aeruginosa</td><td rowspan=1 colspan=1>PA-gyr83-WTa</td></tr><tr><td rowspan=1 colspan=1>P. mirabilis</td><td rowspan=1 colspan=1>AAC, CTX-M-2, SUL1, SUL2, TEM</td></tr><tr><td rowspan=1 colspan=1>E. faecalis</td><td rowspan=1 colspan=1>APHb</td></tr></table>

a gyr83-WT not reported for $P$ . aeruginosa. b APH not reported for $E .$ . faecalis.

During the execution of the Sample Stability Study, a total of four (4) positive external controls, four (4) negative external controls, and twelve (12) no template controls were tested. In all cases the controls passed by producing the expected results.

(ii) Prepared Acuitas AMR Gene Panel Assay Plate ( with Extracted DNA)

A study was conducted to determine the stability of extracted DNA after being added to the Acuitas AMR Gene Panel Assay plates along with Acuitas AMR Gene Panel Master Mix. This was designated as a "prepared plate." The prepared plate stability protocol evaluated the performance of the prepared assay plate after storage at $2 { - } 8 ^ { \circ } \mathrm { C }$ for 6 hrs with a set of five (5) isolates containing a diverse subset of antimicrobial resistance genes detected by the Acuitas AMR Gene Panel assay (See Table 18 above for list of the organism/marker panel tested). Results from prepared plates were

compared to plates without storage. Three (3) replicates of each isolate were evaluated for each testing event. Overall, this study showed that prepared plates with extracted DNA were stable for up to 6 hours at $2 { - } 8 ^ { \circ } \mathrm { C }$ with all targets detected.

During the execution of the Sample Stability-Prepared Plate Study, a total of two (2) positive external controls, two (2) negative external controls, and six (6) no template controls were tested. In all cases, the controls passed as the produced the expected results.

# (iii) Fresh vs Frozen Bacterial Suspension Study

To evaluate the equivalence of fresh bacterial suspensions and frozen bacterial suspensions, a Fresh vs. Frozen Study was conducted to determine the performance of the Acuitas AMR Gene Panel with fresh $0 . 5 \mathrm { M c F }$ arland bacterial suspensions vs. frozen $0 . 5 \mathrm { M c F }$ arland bacterial suspensions. Nine (9) isolates [(3) $E$ . coli, $( 2 ) K$ . pneumoniae, (1) $P .$ . aeruginosa, (2) $P .$ . mirabilis, and (1) $E .$ . faecalis) were evaluated. Ten replicates of each isolate were tested by three operators before and 56 days after freezing at $- 2 0 \mathrm { { ^ \circ C } }$ . All resistance markers exhibited $100 \%$ agreement when comparing detection before and 56 days after freezing, except for ANT, DFR, and DHA. For ANT, a single FP result was observed for the $P .$ . aeruginosa isolate at day 56. This isolate was retested ten times and demonstrated only the expected results during repeat testing. For DFR and DHA, a single FP result for each marker was obtained for a single replicate with a $P .$ . mirabilis isolate before freezing. Retesting of ten replicates of this isolate demonstrated only the expected results. The results of this study support that 0.5 McFarland bacterial suspensions can be stored for up to 56 days at $- 2 0 \mathrm { { ^ \circ C } }$ before testing with the Acuitas AMR Gene Panel.

During the execution of the Fresh vs. Frozen Isolate Study, a total of (27) positive external controls, (27) negative external controls, and (12) no template controls were tested. In all but one case, the controls passed by producing the expected results. One Positive Control A failed initial testing, but upon repeat testing produced expected results. The study panel consisted of the following organisms and target gene panel members as shown in Table 19 below. These results support the claim that 0.5 McFarland suspensions stored at $- 2 0 ^ { \circ } \mathrm { C }$ are stable up to 56 days and are acceptable for use in conducting the Reproducibility Study.

Table 19. Organism and Resistance Gene Marker Panel   

<table><tr><td colspan="1" rowspan="1">Organism</td><td colspan="1" rowspan="1">Resistance Gene Markers</td></tr><tr><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">EC-gyrA-MT, AADa, ANT, DFR, KPC, SUL1,SUL2, TEM</td></tr><tr><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">SUL2, TEM</td></tr><tr><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">EC-gyr87-WTa, DFR, DHA, SUL1, SUL2, TEM</td></tr><tr><td colspan="1" rowspan="1">K. pneumoniae</td><td colspan="1" rowspan="1">KP-gyrA-MTb, AAC, armAb, CMY, AAD, NDM,OXA-48, SHVb, CTX-M-1, TEM</td></tr><tr><td colspan="1" rowspan="1">K. pneumoniae</td><td colspan="1" rowspan="1">KP-gyr83-WTb, SUL1, SUL2, AAC, SHVb,TEM, OXA-1, IMP</td></tr><tr><td colspan="1" rowspan="1">P. aeruginosa</td><td colspan="1" rowspan="1">PA-gyrA-MT, SUL1c, PER, VIM</td></tr><tr><td colspan="1" rowspan="1">P. mirabilis</td><td colspan="1" rowspan="1">SUL1d, SUL2, AAC, ANT, rmt, TEM, VEB,CMY</td></tr><tr><td colspan="1" rowspan="1">P. mirabilis</td><td colspan="1" rowspan="1">SUL1d, SUL2, AAC, AADd, APH, CTX-M-9,DFR, OXA-1, TEM</td></tr><tr><td colspan="1" rowspan="1">E. faecalis</td><td colspan="1" rowspan="1">vanA</td></tr></table>

a AAD and gyr87-WT are not reported for $E$ . coli. b gyrA-MT, gyr83-WT, armA, and SHV not reported for K. pneumoniae. c SUL1 not reported for $P .$ . aeruginosa. ${ } ^ { d } A A D$ and SUL1 not reported for $P$ . mirabilis.

(iv)Frozen DNA Stability

A study was conducted to evaluate the stability of DNA extracted from pure bacterial colonies using the QIAGEN EZ1 Advanced XL instrument with the Acuitas AMR Gene Panel after storage at $- 1 5 ^ { \circ } \mathrm { C }$ to $- 2 5 ^ { \circ } \mathrm { C }$ for 30 days. The same organism panel was used as shown in Table 18 above under Sample DNA Stability. Three replicates of each isolate exhibiting the majority of antimicrobial resistance genes detected by the Acuitas AMR Gene Panel assay were evaluated during the Frozen DNA Stability study. Extracted DNA from 0.5 McFarland suspensions was stored under the following conditions: $- 1 5 ^ { \circ } \mathrm { C }$ to $- 2 5 ^ { \circ } \mathrm { C }$ for 14 days and for 30 days. The study results in the summary report showed that there was $100 \%$ agreement between $\mathrm { T } { = } 0$ days, $\mathrm { T } { = } 1 4$ , and $\mathrm { T } { = } 3 0$ days for extracted DNA stored at -15 to $- 2 5 ^ { \circ } \mathrm { C }$ . Based on the results provided, frozen DNA is stable for use with the Acuitas AMR Gene Panel assay for up to 30 days.

During the execution of the Sample Stability Frozen DNA, a total of (5) positive external controls, (5) negative external controls, and (14) no template controls were tested. The controls passed by producing the expected results.

# (v) Acuitas AMR Gene Panel Stability

Three independent lots of the Acuitas AMR Gene Panel will be evaluated for stability in a Real-Time Study at time points established for the respective material after storage under the conditions specified for the plate kit $( 2 0 – 2 5 ^ { \circ } \mathrm { C } )$ and reagent kit (2- $8 ^ { \circ } \mathrm { C }$ ). Stability studies support two months plate kit storage at $2 0 – 2 5 ^ { \circ } \mathrm { C }$ and reagent kit storage at $2 { - } 8 ^ { \circ } \mathrm { C }$ . Real-time stability studies for determining shelf-life and expiration dating are ongoing.

6. Detection Limit:

a. LoD

N/A. The extracted DNA from a 0.5 McFarland bacterial suspension is used for the assay.

b. Analytical Reactivity

Two hundred ninety-eight (298) isolates covering all resistance genes detected by the Acuitas AMR Gene Panel test were assessed for reactivity in this study. Each isolate was tested once, except for isolates harboring rare resistance genes, which were tested in replicates to achieve at least 18 positive data points per gene target assessed. Table 20 shows the detection of various AMR genes during wet testing. The Acuitas AMR Gene Panel results were compared with species identification by well-established automated species identification methods and AMR gene detection by WGS. The genes predicted to be detected by in silico analysis are listed in the last column according to the following categories: “Detectable,” “Likely Detectable,” and “Potentially Detectable.”

During the execution of the Analytical Reactivity study, a total of 36 positive external controls and 37 negative external controls were tested. There were a total of two (2) External Control failures—one (1) Positive Control A failure and one (1) Negative Control failure. All failures were retested and produced expected results.

Table 20. Summary of AMR Gene Variants Tested for Inclusivity/Reactivity on the Acuitas AMR Gene Panel a,e   

<table><tr><td colspan="1" rowspan="2">AMR Gene</td><td colspan="4" rowspan="1">Acuitas AMR Gene Panel</td><td colspan="1" rowspan="2">Other AMR Gene Variants Predicted to be Detectedby Acuitas AMR Gene Panel Based on In Silico Analysisd</td></tr><tr><td colspan="1" rowspan="1">Number ofSamplesPositive forGene by WGS</td><td colspan="1" rowspan="1">Number ofUnique IsolatesPositive forGene byWGS</td><td colspan="1" rowspan="1">AMR Gene(s)Detected</td><td colspan="1" rowspan="1">AMR Gene(s)Not Detected</td></tr><tr><td colspan="1" rowspan="1">AAC</td><td colspan="1" rowspan="1">198</td><td colspan="1" rowspan="1">162</td><td colspan="1" rowspan="1">aac(3)-IIa, aac(3)-IId,aac(3)-IVa, aac(6')-Ib,aac(6')Ib-cr, aacA4-8,aacA4</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Detectable: aac(3)-IIc, aac(3)-IIe, aac(3)-Ib-aac(6')-Ib, ant(3")-Ih-aac(6')-IId</td></tr><tr><td colspan="1" rowspan="1">AAD</td><td colspan="1" rowspan="1">70</td><td colspan="1" rowspan="1">62</td><td colspan="1" rowspan="1">aadA1, aadA2</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Detectable: aadA13, aadA3, aadA8, aadA8b, aadA7b, aadA17Likely Detectable: aadA12, aadA21, aadA22, aadA23</td></tr><tr><td colspan="1" rowspan="1">ANT</td><td colspan="1" rowspan="1">63</td><td colspan="1" rowspan="1">40</td><td colspan="1" rowspan="1">aadB</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">APH</td><td colspan="1" rowspan="1">15</td><td colspan="1" rowspan="1">14</td><td colspan="1" rowspan="1">aph(4)-Ia</td><td colspan="1" rowspan="1">aph(4)-Ia_V01499</td><td></td></tr><tr><td colspan="1" rowspan="1">armA</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">armA</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">CMY</td><td colspan="1" rowspan="1">26</td><td colspan="1" rowspan="1">25</td><td colspan="1" rowspan="1">blaCMY-16, blaCMY-2, blaCMY-4,blaCMY-42, blaCMY-6, blaCMY-60</td><td colspan="1" rowspan="1">blaCMY-16_FJ855437</td><td colspan="1" rowspan="1">Detectable: blaBIL-1, blaCMY-0, blaCMY-102, blaCMY-108,blaCMY-110,blaCMY-111, blaCMY-112,blaCMY-113,blaCMY-114, blaCMY-115, blaCMY-118, blaCMY-12, blaCMY-14, blaCMY-15, blaCMY-17, blaCMY-18, blaCMY-20, blaCMY-21, blaCMY-22, blaCMY-23, blaCMY-24, blaCMY-25, blaCMY-27, blaCMY-28, blaCMY-29, blaCMY-3, blaCMY-30, blaCMY-31, blaCMY-32, blaCMY-33, blaCMY-34, blaCMY-35, blaCMY-36, blaCMY-38, blaCMY-39, blaCMY-41, blaCMY-43, blaCMY-44, blaCMY-45, blaCMY-47, blaCMY-48, blaCMY-5, blaCMY-50, blaCMY-51, blaCMY-54, blaCMY-55, blaCMY-56, blaCMY-57, blaCMY-58, blaCMY-59, blaCMY-61, blaCMY-62, blaCMY-63, blaCMY-65, blaCMY-66, blaCMY-67, blaCMY-68, blaCMY-69, blaCMY-7, blaCMY-71, blaCMY-72, blaCMY-75, blaCMY-76, blaCMY-77, blaCMY-78, blaCMY-80, blaCMY-81, blaCMY-84, blaCMY-87, blaCMY-90, blaCMY-94, blaCMY-95, blaCMY-99, blaLAT-1, blaCMY-103, blaCMY-117, blaCMY-79Likely Detectable: blaCMY-13, blaCMY-26, blaCMY-37,blaCMY-49, blaCMY-73, blaCMY-79, blaCMY-116, blaCMY-117,Potentially Detectable: blaCMY-40, blaCMY-53</td></tr><tr><td colspan="1" rowspan="1">CTX-M-1</td><td colspan="1" rowspan="1">72</td><td colspan="1" rowspan="1">62</td><td colspan="1" rowspan="1">blaCTX-M-1, blaCTX-M-15, blaCTX-M-55,blaCTX-M-64</td><td colspan="1" rowspan="1">blaCTX-M-15_DQ302097</td><td colspan="1" rowspan="1">Detectable: blaCTX-M, blaCTX-M-10, blaCTX-M-101, blaCTX-M-103, blaCTX-M-107, blaCTX-M-108, blaCTX-M-109,blaCTX-M-11, blaCTX-M-114, blaCTX-M-117, blaCTX-M-12,blaCTX-M-123, blaCTX-M-132, blaCTX-M-136, blaCTX-M-139,blaCTX-M-144, blaCTX-M-22, blaCTX-M-28, blaCTX-M-29,blaCTX-M-3,blaCTX-M-30, blaCTX-M-32,blaCTX-M-33,blaCTX-M-34,blaCTX-M-36,blaCTX-M-37,blaCTX-M-38,blaCTX-M-42,blaCTX-M-52,blaCTX-M-53,blaCTX-M-54,blaCTX-M-58,blaCTX-M-60,blaCTX-M-61,blaCTX-M-62,blaCTX-M-66,blaCTX-M-68,blaCTX-M-69,blaCTX-M-71,blaCTX-M-72,blaCTX-M-79,blaCTX-M-82,blaCTX-M-88,blaCTX-M-89, blaCTX-M-96</td></tr><tr><td colspan="1" rowspan="2">AMR Gene</td><td colspan="2" rowspan="1"></td><td colspan="2" rowspan="1">Acuitas AMR Gene Panel</td><td colspan="1" rowspan="2">Other AMR Gene Variants Predicted to be Detectedby Acuitas AMR Gene Panel Based on In Silico Analysis</td></tr><tr><td colspan="1" rowspan="1">Number ofSamplesPositive forGene by WGS</td><td colspan="1" rowspan="1">Number ofUniqueI solatesPositive forGene byGS</td><td colspan="1" rowspan="1">AMR Gene(s)Detected</td><td colspan="1" rowspan="1">AMR Gene(s)Not Detected</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Likely Detectable: blaCTX-M-116, blaCTX-M-142, blaCTX-M-23, blaCTX-M-80</td></tr><tr><td colspan="1" rowspan="1">CTX-M-2</td><td colspan="1" rowspan="1">14</td><td colspan="1" rowspan="1">12</td><td colspan="1" rowspan="1">blaCTX-M-131,bllaCTX-M-2</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Detectable: blaCTX-M-141,blaCTX-M-20,blaCTX-M-31,blaCTX-M-35, blaCTX-M-43, blaCTX-M-44, blaCTX-M-5,blaCTX-M-56, blaCTX-M-59, blaCTX-M-76, blaCTX-M-77,blaCTX-M-92, blaCTX-M-95, blaCTX-M-97Likely Detectable: blaCTX-M-115, blaCTX-M-124</td></tr><tr><td colspan="1" rowspan="1">CTX-M-9</td><td colspan="1" rowspan="1">34</td><td colspan="1" rowspan="1">32</td><td colspan="1" rowspan="1">blaCTX-M-14,blaCTX-M-14b,blaCTX-M-27,blaCTX-M-64,blaCTX-M-65,blaCTX-M-9, blaCTX-M-90</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Detectable: blaCTX-M-104, blaCTX-M-106, blaCTX-M-110,blaCTX-M-111, blaCTX-M-112, blaCTX-M-113, blaCTX-M-121,blaCTX-M-122, blaCTX-M-123, blaCTX-M-125, blaCTX-M-126,blaCTX-M-129, blaCTX-M-13, blaCTX-M-130, blaCTX-M-134,blaCTX-M-147, blaCTX-M-148, blaCTX-M-159, blaCTX-M-16,blaCTX-M-17,blaCTX-M-19,blaCTX-M-21,blaCTX-M-24,blaCTX-M-38,blaCTX-M-46,blaCTX-M-47blaCTX-M-48,blaCTX-M-49,blaCTX-M-50,blaCTX-M-51,blaCTX-M-67,blaCTX-M-81,blaCTX-M-83,blaCTX-M-84,blaCTX-M-85,blaCTX-M-86,blaCTX-M-87,blaCTX-M-93,blaCTX-M-98,blaCTX-M-99</td></tr><tr><td colspan="1" rowspan="1">DFR</td><td colspan="1" rowspan="1">42</td><td colspan="1" rowspan="1">37</td><td colspan="1" rowspan="1">dfrA17, dfrA5</td><td colspan="1" rowspan="1"></td><td></td></tr><tr><td colspan="1" rowspan="1">DHA</td><td colspan="1" rowspan="1">16</td><td colspan="1" rowspan="1">13</td><td colspan="1" rowspan="1">blaDHA-1</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Detectable: blaDHA-10, blaDHA-13, blaDHA-14, blaDHA-15,blaDHA-17, blaDHA-18, blaDHA-19, blaDHA-2, blaDHA-20,blaDHA-21, blaDHA-22, blaDHA-3, blaDHA-5, blaDHA-6,blaDHA-7, blaDHA-9, blaMOR-2</td></tr><tr><td colspan="1" rowspan="1">IMP</td><td colspan="1" rowspan="1">25</td><td colspan="1" rowspan="1">15</td><td colspan="1" rowspan="1">blaIMP-1, blaIMP-13,blaIMP-18, blaIMP-26,blaIMP-34, blaIMP-4,blalMP-6</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Detectable: blaIMP-10, blaIMP-25, blaIMP-3, blaIMP-40, blaMP-42, blaIMP-52, blaIMP-14, blaIMP-14a, blaIMP-19, blaIMP-2,blaIMP-20, blaIMP-24, blaIMP-32, blaIMP-48, blaIMP-8, blaIMP-28, blaIMP-5Likely Detectable: blaIMP-15, blalMP-29Potentially Detectable: blalMP-38</td></tr><tr><td colspan="1" rowspan="1">KPC</td><td colspan="1" rowspan="1">23</td><td colspan="1" rowspan="1">22</td><td colspan="1" rowspan="1">blaKPC-2, blaKPC-3</td><td colspan="1" rowspan="1">blaKPC-2_AY034847</td><td colspan="1" rowspan="1">Detectable:blaKPC-1,blaKPC-10,blaKPC-11,blaKPC-12,blaKPC-13, blaKPC-14, blaKPC-15, blaKPC-16, blaKPC-17,blaKPC-19, blaKPC-22, blaKPC-4, blaKPC-5, blaKPC-6, blaKPC-8, blaKPC-9</td></tr><tr><td colspan="1" rowspan="1">MCR-1</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">13</td><td colspan="1" rowspan="1">MCR-1</td><td colspan="1" rowspan="1"></td><td></td></tr><tr><td colspan="1" rowspan="1">NDM</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">blaNDM-1, blaNDM-7</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Detectable: blaNDM-12, blaNDM-2, blaNDM-3, blaNDM-4,blaNDM-5, blaNDM-6, blaNDM-8, blaNDM-9Likely Detectable: blaNDM-10</td></tr><tr><td colspan="1" rowspan="1">OXA-1</td><td colspan="1" rowspan="1">78</td><td colspan="1" rowspan="1">56</td><td colspan="1" rowspan="1">blaOXA-1, blaOXA-4</td><td colspan="1" rowspan="1">blaOXA-1_J02967</td><td colspan="1" rowspan="1">Detectable: blaOXA-224, blaOXA-31, blaOXA-320Likely Detectable: blaOXA-47</td></tr><tr><td colspan="1" rowspan="1">OXA-48</td><td colspan="1" rowspan="1">19</td><td colspan="1" rowspan="1">19</td><td colspan="1" rowspan="1">blaOXA-181, blaOXA-232, blaOXA-370,blaOXA-48</td><td colspan="1" rowspan="1">blaOXA-48_AY236073</td><td colspan="1" rowspan="1">Detectable: blaOXA-162, blaOXA-163, blaOXA-199, blaOXA-204, blaOXA-244, blaOXA-245, blaOXA-247</td></tr><tr><td colspan="1" rowspan="1">OXA-9</td><td colspan="1" rowspan="1">26</td><td colspan="1" rowspan="1">26</td><td colspan="1" rowspan="1">blaOXA-9</td><td colspan="1" rowspan="1">blaOXA-9_JF703130</td><td colspan="1" rowspan="1">Y</td></tr><tr><td colspan="1" rowspan="1">PER</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">19</td><td colspan="1" rowspan="1">blaPER-1, blaPER-3</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Detectable: blaPER-4, blaPER-5, blaPER-7, blaPER-8</td></tr><tr><td colspan="1" rowspan="1">RMT</td><td colspan="1" rowspan="1">11</td><td colspan="1" rowspan="1">9</td><td colspan="1" rowspan="1">rmtB</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Detectable: rmtFLikely Detectable: rmtB2</td></tr><tr><td colspan="1" rowspan="1">SHV</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">blaSHV-12, blaSHV-2a</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Detectable: blaSHV-1, blaSHV-100, blaSHV-101, blaSHV-102,blaSHV-103, blaSHV-104, blaSHV-106, blaSHV-107, blaSHV-108, blaSHV-109, blaSHV-11, blaSHV-119, blaSHV-120,blaSHV-121, blaSHV-122, blaSHV-128, blaSHV-129, blaSHV-13,blaSHV-132, blaSHV-133, blaSHV-135, blaSHV-137, blaSHV-14,blaSHV-140, blaSHV-141, blaSHV-142, blaSHV-143, blaSHV-144, blaSHV-145, blaSHV-147, blaSHV-148, blaSHV-149,blaSHV-15, blaSHV-150, blaSHV-151, blaSHV-152, blaSHV-153,blaSHV-154, blaSHV-155, blaSHV-156, blaSHV-157, blaSHV-158, blaSHV-159, blaSHV-16, blaSHV-160, blaSHV-161,blaSHV-162, blaSHV-163, blaSHV-164, blaSHV-165, blaSHV-167, blaSHV-168, blaSHV-172, blaSHV-173, blaSHV-178,blaSHV-179, blaSHV-18, blaSHV-183, blaSHV-2, blaSHV-24,blaSHV-25, blaSHV-26, blaSHV-28, blaSHV-29, blaSHV-30,blaSHV-31, blaSHV-33, blaSHV-34, blaSHV-35, blaSHV-36,blaSHV-38, blaSHV-40, blaSHV-41, blaSHV-42, blaSHV-44,</td></tr><tr><td colspan="1" rowspan="2">AMR Gene</td><td colspan="4" rowspan="1">Acuitas AMR Gene Panel</td><td colspan="1" rowspan="2">Other AMR Gene Variants Predicted to be Detectedby Acuitas AMR Gene Panel Based on In Silico Analysisd</td></tr><tr><td colspan="1" rowspan="1">Number ofSamples Positive forGene by WGS</td><td colspan="1" rowspan="1">Number ofUniqueIsolatesPositive forGene byWGS</td><td colspan="1" rowspan="1">AMR Gene(s)Detected</td><td colspan="1" rowspan="1">AMR Gene(s)Not Detected</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">b</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">blaSHV-46,blaSHV-48, blaSHV-49, blaSHV-5,blaSHV-50,blaSHV-51,blaSHV-52,blaSHV-55,blaSHV-56,blaSHV-57,blaSHV-59,blaSHV-60,blaSHV-61,blaSHV-62,blaSHV-63,blaSHV-64,blaSHV-65,blaSHV-66, blaSHV-67,blaSHV-69,blaSHV-7,blaSHV-70,blaSHV-71,blaSHV-72,blaSHV-73,blaSHV-74,blaSHV-75,blaSHV-76,blaSHV-77,blaSHV-78,blaSHV-79,blaSHV-8,blaSHV-80,blaSHV-81,blaSHV-82,blaSHV-83,blaSHV-85,blaSHV-86, blaSHV-89,blaSHV-92,blaSHV-94, blaSHV-95, blaSHV-96, blaSHV-97,blaSHV-98,blaSHV-99, blaOKP-A, blaSHV-105, blaSHV-110, blaSHV-27,blaSHV-45, blaSHV-93Likely Detectable: blaSHV-119, blaSHV-137, blaSHV-144,blaSHV-167, blaSHV-168, blaSHV-38, blaSHV-51, blaSHV-70,blaSHV-71, blaSHV-72, blaSHV-73, blaSHV-80, blaSHV-81</td></tr><tr><td colspan="1" rowspan="1">Sull</td><td colspan="1" rowspan="1">137</td><td colspan="1" rowspan="1">103</td><td colspan="1" rowspan="1">Sull</td><td colspan="1" rowspan="1">Sul1_AY224185</td><td colspan="1" rowspan="1">Sul3 (AY047357)c, Sul3 (AB281183)cLikely Detectable: Sul1 (AM746675)Potentially Detectable: Sul1 (AY260546)</td></tr><tr><td colspan="1" rowspan="1">Sul2</td><td colspan="1" rowspan="1">133</td><td colspan="1" rowspan="1">99</td><td colspan="1" rowspan="1">Sul2</td><td colspan="1" rowspan="1">Sul2_GQ421466</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">TEM</td><td colspan="1" rowspan="1">173</td><td colspan="1" rowspan="1">132</td><td colspan="1" rowspan="1">blaTEM-1, blaTEM-117, blaTEM-143,blaTEM-176, blaTEM-192, blaTEM-1A,blaTEM-1B, blaTEM-1C, blaTEM-1D,blaTEM-2, blaTEM-33</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Detectable: blaTEM-10, blaTEM-101, blaTEM-102, blaTEM-104,blaTEM-105, blaTEM-108, blaTEM-11, blaTEM-110, blaTEM-112, blaTEM-114, blaTEM-115, blaTEM-116, blaTEM-118,blaTEM-12, blaTEM-120, blaTEM-122, blaTEM-126, blaTEM-127, blaTEM-128, blaTEM-129, blaTEM-132, blaTEM-135,blaTEM-136, blaTEM-137, blaTEM-141, blaTEM-144, blaTEM-145, blaTEM-147, blaTEM-148, blaTEM-150, blaTEM-151,blaTEM-152, blaTEM-154, blaTEM-155, blaTEM-156, blaTEM-157, blaTEM-158, blaTEM-159, blaTEM-160, blaTEM-163,blaTEM-164, blaTEM-166, blaTEM-168, blaTEM-169, blaTEM-171, blaTEM-178, blaTEM-182, blaTEM-183, blaTEM-185,blaTEM-186, blaTEM-187, blaTEM-188, blaTEM-189, blaTEM-190, blaTEM-193, blaTEM-194, blaTEM-195, blaTEM-198,blaTEM-20, blaTEM-201, blaTEM-206, blaTEM-207, blaTEM-209, blaTEM-216, blaTEM-217, blaTEM-28, blaTEM-29,blaTEM-30, blaTEM-34, blaTEM-45, blaTEM-47, blaTEM-48,blaTEM-49, blaTEM-53, blaTEM-54, blaTEM-55, blaTEM-57,blaTEM-67, blaTEM-68, blaTEM-70, blaTEM-71, blaTEM-72,blaTEM-75, blaTEM-76, blaTEM-77, blaTEM-78, blaTEM-79,blaTEM-80, blaTEM-81, blaTEM-82, blaTEM-83, blaTEM-84,blaTEM-85, blaTEM-86, blaTEM-91, blaTEM-93, blaTEM-95,blaTEM-96, blaTEM-97, blaTEM-98, blaTEM-99, blaTEM-106,blaTEM-107, blaTEM-109, blaTEM-111, blaTEM-113, blaTEM-121, blaTEM-123, blaTEM-124, blaTEM-130, blaTEM-131,blaTEM-133, blaTEM-134, blaTEM-138, blaTEM-139, blaTEM-142, blaTEM-149, blaTEM-15, blaTEM-153, blaTEM-16,blaTEM-167, blaTEM-177, blaTEM-184, blaTEM-197, blaTEM-199, blaTEM-205, blaTEM-21, blaTEM-211, blaTEM-22,blaTEM-24, blaTEM-3, blaTEM-43, blaTEM-52, blaTEM-52B,blaTEM-52C, blaTEM-6, blaTEM-60, blaTEM-63, blaTEM-8,blaTEM-87, blaTEM-88, blaTEM-89, blaTEM-92, blaTEM-94Likely Detectable: blaTEM-17, blaTEM-125, blaTEM-146,blaTEM-90</td></tr><tr><td colspan="1" rowspan="1">vanA</td><td colspan="1" rowspan="1">21</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">VanA-A</td><td colspan="1" rowspan="1"></td><td></td></tr><tr><td colspan="1" rowspan="1">VEB</td><td colspan="1" rowspan="1">14</td><td colspan="1" rowspan="1">13</td><td colspan="1" rowspan="1">blaVEB-1, blaVEB-5,blaVEB-6</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Detectable: blaVEB-2, blaVEB-3, blaVEB-4, blaVEB-7, blaVEB-8</td></tr><tr><td colspan="1" rowspan="1">VIM</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">18</td><td colspan="1" rowspan="1">blaVIM-1, blaVIM-2,blaVIM-20</td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1">Detectable: blaVIM-12,blaVIM-13,blaVIM-14, blaVIM-19,blaVIM-26, blaVIM-27,blaVIM-28, blaVIM-29, blaVIM-32,blaVIM-33, blaVIM-34,blaVIM-35,blaVIM-37,blaVIM-39,blaVIM-4, blaVIM-42,blaVIM-43,blaVIM-10,bllaVM-11,blaVIM-15, blaVIM-16,blaVIM-17,blaVIM-23,blaVIM-24,blaVIM-30, blaVIM-31,blaVIM-8,blaVIM-9,blaVIM-18,blaVIM-3,blaVIM-36,blaVIM-6,blaVIM-25,blaVIM-38,blaVIM-5</td></tr></table>

a Results include combinations of AMR gene variants across various organisms tested in the study.

b One gene variant of aadA7 (NCBI Accession AF224733) is not consistently detected by the Acuitas AMR Gene Panel due to 4 mismatches with the reverse PCR primer and one mismatch with the PCR probe of the AAD assay. Other aadA variants lack the mismatches and are expected to be detected.

c Only these two Sul3 sequences are predicted to be detected by the Acuitas AMR Gene Panel. Other Sul3 accession numbers are not predicted to be detected.

d Detectable indicates $100 \%$ homology of each primer/detector probe(s) with the target sequence. Likely Detectable indicates $< I O O \%$ homology of one or both primers with the target sequence (one mismatch in one or both primers) and $100 \%$ homology of detector probe(s). Potentially Detectable indicates $< 9 5 \%$ homology of one or more primers with the target sequence but ${ \le } 2$ nucleotide mismatches over their entire length along with $100 \%$ homology of detector probe(s). e The test panel for this study was composed of (5) Enterococcus faecalis,(69) Escherichia coli, (3) Klebsiella pneumoniae ssp ozaenae, (89) Klebsiella pneumoniae ssp pneumoniae, (34) Proteus mirabilis, and (98) Pseudomonas aeruginosa strains.

Table 21 shows a summary of false positives and false negatives observed during assessment of reactivity on the Acuitas AMR Gene Panel reported along with a breakdown of species in which these results were observed.

Table 21. Summary of False Positives and False Negatives Observed During Assessment of Reactivity of the Acuitas AMR Gene Panel   

<table><tr><td rowspan=2 colspan=1>AMR Genea</td><td rowspan=1 colspan=3>False Positives</td><td rowspan=1 colspan=3>False Negatives</td></tr><tr><td rowspan=1 colspan=1>TotalFP</td><td rowspan=1 colspan=1>Species w/ FP</td><td rowspan=1 colspan=1># FP perSpecies</td><td rowspan=1 colspan=1>TotalFN</td><td rowspan=1 colspan=1>Species w/ FN</td><td rowspan=1 colspan=1># FN perSpecies</td></tr><tr><td rowspan=4 colspan=1>AAC</td><td rowspan=4 colspan=1>8</td><td rowspan=1 colspan=1>E. coli</td><td rowspan=1 colspan=1>1</td><td rowspan=4 colspan=1>3</td><td rowspan=4 colspan=1>K. pneumoniae</td><td rowspan=4 colspan=1>3</td></tr><tr><td rowspan=1 colspan=1>K. pneumoniae</td><td rowspan=1 colspan=1>5</td></tr><tr><td rowspan=1 colspan=1>P. aeruginosa</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>P. mirabilis</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>AAD</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>K. pneumoniae</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td></tr><tr><td rowspan=2 colspan=1>ANT</td><td rowspan=2 colspan=1>2</td><td rowspan=1 colspan=1>P. aeruginosa</td><td rowspan=1 colspan=1>1</td><td rowspan=2 colspan=1>0</td><td rowspan=2 colspan=1>N/A</td><td rowspan=2 colspan=1>N/A</td></tr><tr><td rowspan=1 colspan=1>P. mirabilis</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>APH</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>K. pneumoniae</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>K. pneumoniae</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=2 colspan=1>CMY</td><td rowspan=2 colspan=1>10</td><td rowspan=1 colspan=1>E. coli</td><td rowspan=1 colspan=1>2</td><td rowspan=2 colspan=1>1</td><td rowspan=2 colspan=1>K. pneumoniae</td><td rowspan=2 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>K. pneumoniae</td><td rowspan=1 colspan=1>8</td></tr><tr><td rowspan=2 colspan=1>CTX-M-1</td><td rowspan=2 colspan=1>3</td><td rowspan=1 colspan=1>E. coli</td><td rowspan=1 colspan=1>2</td><td rowspan=2 colspan=1>1</td><td rowspan=2 colspan=1>K. pneumoniae</td><td rowspan=2 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>K. pneumoniae</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>DFR</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>E. coli</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td></tr><tr><td rowspan=2 colspan=1>KPC</td><td rowspan=2 colspan=1>2</td><td rowspan=1 colspan=1>E. coli</td><td rowspan=1 colspan=1>1</td><td rowspan=2 colspan=1>1</td><td rowspan=2 colspan=1>K. pneumoniae</td><td rowspan=2 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>K. pneumoniae</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>OXA-1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>E. coli</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>OXA-48</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>K. pneumoniae</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>K. pneumoniae</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=2 colspan=1>OXA-9</td><td rowspan=2 colspan=1>3</td><td rowspan=1 colspan=1>E. coli</td><td rowspan=1 colspan=1>1</td><td rowspan=2 colspan=1>1</td><td rowspan=2 colspan=1>K. pneumoniae</td><td rowspan=2 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>K. pneumoniae</td><td rowspan=1 colspan=1>2</td></tr><tr><td rowspan=2 colspan=1>Sul1</td><td rowspan=2 colspan=1>6</td><td rowspan=1 colspan=1>E. coli</td><td rowspan=1 colspan=1>1</td><td rowspan=2 colspan=1>1</td><td rowspan=2 colspan=1>K. pneumoniae</td><td rowspan=2 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>K. pneumoniae</td><td rowspan=1 colspan=1>5</td></tr><tr><td rowspan=1 colspan=1>Sul2</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>K. pneumoniae</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>K. pneumoniae</td><td rowspan=1 colspan=1>2</td></tr><tr><td rowspan=1 colspan=1>TEM</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>K. pneumoniae</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td></tr></table>

# 7. Assay Cut-Off:

The assay cut-offs for the Acuitas AMR Gene Panel were initially established and validated using pre-clinical data. The results from the four hundred thirty-nine (439) isolates were evaluated. For each assay target, Ct, Cq. Conf, and Delta Rn values were determined from which ROC curves were generated to show sensitivity and specificity over a range of Ct values for each assay target.

8. Accuracy (Instrument):

N/A

9. Carry-Over:

A panel of twelve isolates harboring a variety of targeted resistance genes was utilized for the study (See Table 22 below). Two operators performed testing, where a panel of six positive samples and six negative samples (S. aureus isolate negative for all resistance markers) were prepared and blinded to the other operator. Samples were sub-cultured onto blood agar and used to prepare a 0.5 McFarland suspension. DNA was extracted and the twelve samples were tested in alternating fashion with twelve negative controls to determine the rate of carry-over/cross contamination across multiple plates. During the execution of the Carry Over Cross Contamination Study, no positive external controls were evaluated. The negative external controls passed by producing the expected results (420/420). Isolates included in the test panel also generated expected results (171/171 results).

Table 22. Organism/Markers Included in the Carry-Over Study   

<table><tr><td rowspan=1 colspan=1>Organism ID</td><td rowspan=1 colspan=1>Resistance Genes</td></tr><tr><td rowspan=1 colspan=1>E. coli</td><td rowspan=1 colspan=1>AAC, , E. coli gyrA-MT, OXA-1, SHV, Sul1</td></tr><tr><td rowspan=1 colspan=1>E. coli</td><td rowspan=1 colspan=1>AAC, , Sul1, TEM</td></tr><tr><td rowspan=1 colspan=1>K. pneumoniae</td><td rowspan=1 colspan=1>AAC, AAD, APH, , KPC, Sul1, TEM</td></tr><tr><td rowspan=1 colspan=1>P. aeruginosa</td><td rowspan=1 colspan=1>AAC, ANT, CTX-M-2, P. aeruginosa gyrA-MT</td></tr><tr><td rowspan=1 colspan=1>P. mirabilis</td><td rowspan=1 colspan=1>AAC, APH, CTX-M-9, DFR, OXA-1, Sul2, TEM</td></tr><tr><td rowspan=1 colspan=1>E. faecalis (2)</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>E. coli</td><td rowspan=1 colspan=1>AAC, CMY, E. coli gyrA-MT, SHV, Sul1, Sul2, TEM</td></tr><tr><td rowspan=1 colspan=1>K. pneumoniae</td><td rowspan=1 colspan=1>AAC, AAD, CTX-M-1, DFR, OXA-9, , Sul1, Sul2, TEM</td></tr><tr><td rowspan=1 colspan=1>K. pneumoniae</td><td rowspan=1 colspan=1>AAC, AAD, CTX-M-1, , NDM, RMT, Sul2, TEM,</td></tr><tr><td rowspan=1 colspan=1>P. aeruginosa</td><td rowspan=1 colspan=1>P. aeruginosa gyrA-MT, VIM</td></tr><tr><td rowspan=1 colspan=1>P. mirabilis</td><td rowspan=1 colspan=1>AAC, , armA, CTX-M-1, Sul2, TEM</td></tr></table>

No cross-contamination was observed indicating that the open-plate format of the Acuitas AMR Gene Panel did not produce carry-over/cross-contamination between test samples.

# 10. Media Equivalency Study

Fifty-two isolates [(13 ) E. coli, (13) $K .$ . pneumoniae, (13) $P .$ . aeruginosa, and $( 1 3 ) P$ . mirabilis), each harboring at least one indicated resistance marker, were tested in a Media Equivalency Study to assess growth of isolates on blood agar and MacConkey agar. The Acuitas AMR Gene Panel results were evaluated for percent agreement with the two comparator methods (automated species identification methods and WGS) for each combination of organism, AMR gene, and agar media. Table 23 summarizes the results below, which includes original results without repeat results. All indicated resistance markers were included in the study, except for DHA, MCR-1, and vanA. E. faecalis was not included as it does not grow on MacConkey agar.

Isolates were sub-cultured onto blood agar and MacConkey agar, and pure colonies were selected and used to generate 0.5 MacFarland suspensions for testing with the Acuitas AMR Gene Panel. Each isolate was tested once during the study, except for the following organisms, which were tested in duplicate: $E .$ . coli, $K .$ . pneumoniae, and $P .$ . mirabilis. Study results were considered acceptable if there was $2 9 8 \%$ agreement (i.e., PPA and NPA) between Acuitas AMR Gene Panel results for the two medias. All organism/resistant marker combinations met the above outlined criteria, except for those highlighted in grey cells within Table 23 below. These isolates were subjected to retesting. Upon retesting, all isolates yielded the expected results as described in footnotes to the table. This study did not uncover evidence of a media effect between blood and MacConkey agar—both of which are suitable for the Acuitas AMR Gene Panel. During the execution of the Media Equivalency Study, a total of (11) positive external controls, (11) negative external controls, and (19) no template controls were tested. In all cases the controls produced the expected results.

Table 23. Acuitas AMR Gene Panel Media Equivalency Study Performance   

<table><tr><td colspan="1" rowspan="2">Organism</td><td colspan="1" rowspan="2">ResistanceMarker</td><td colspan="1" rowspan="2">Number ofTestingEvents</td><td colspan="2" rowspan="1">Blood Agar</td><td colspan="2" rowspan="1">MacConkey Agar</td></tr><tr><td colspan="1" rowspan="1">PPA(TP/(TP+FN))(95% CI)</td><td colspan="1" rowspan="1">NPA(TN/(TN+FP))(95% CI)</td><td colspan="1" rowspan="1">PPA(TP/(TP+FN))(95% CI)</td><td colspan="1" rowspan="1">NPA(TN/(TN+FP))(95% CI)</td></tr><tr><td colspan="1" rowspan="16">E. coli</td><td colspan="1" rowspan="1">AAC</td><td colspan="1" rowspan="1">13</td><td colspan="1" rowspan="1">100% (7/7)(64.57 - 100)</td><td colspan="1" rowspan="1">100% (6/6)(60.97 - 100)</td><td colspan="1" rowspan="1">100% (7/7)(64.57 - 100)</td><td colspan="1" rowspan="1">100% (6/6)(60.97 - 100)</td></tr><tr><td colspan="1" rowspan="1">ANT</td><td colspan="1" rowspan="1">13</td><td colspan="1" rowspan="1">100% (2/2)(34.24 - 100)</td><td colspan="1" rowspan="1">100% (11/11)(74.12 - 100)</td><td colspan="1" rowspan="1">100% (2/2)(34.24 - 100)</td><td colspan="1" rowspan="1">100% (11/11)(74.12 - 100)</td></tr><tr><td colspan="1" rowspan="1">CMY</td><td colspan="1" rowspan="1">13</td><td colspan="1" rowspan="1">100% (2/2)(34.24 - 100)</td><td colspan="1" rowspan="1">100% (11/11)(74.12 - 100)</td><td colspan="1" rowspan="1">100% (2/2)(34.24 - 100)</td><td colspan="1" rowspan="1">100% (11/11)(74.12 - 100)</td></tr><tr><td colspan="1" rowspan="1">CTX-M-1</td><td colspan="1" rowspan="1">13</td><td colspan="1" rowspan="1">100% (6/6)(60.97 - 100)</td><td colspan="1" rowspan="1">100% (7/7)(64.57 - 100)</td><td colspan="1" rowspan="1">100% (6/6)(60.97 - 100)</td><td colspan="1" rowspan="1">100% (7/7)(64.57 - 100)</td></tr><tr><td colspan="1" rowspan="1">CTX-M-2</td><td colspan="1" rowspan="1">13</td><td colspan="1" rowspan="1">-i)</td><td colspan="1" rowspan="1">100% (13/13)(77.19 - 100)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">100% (13/13)(77.19 - 100)</td></tr><tr><td colspan="1" rowspan="1">CTX-M-9</td><td colspan="1" rowspan="1">13</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">100% (13/13)(77.19 - 100)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">100% (13/13)(77.19 - 100)</td></tr><tr><td colspan="1" rowspan="1">DFR</td><td colspan="1" rowspan="1">13</td><td colspan="1" rowspan="1">100% (4/4)(51.01 - 100)</td><td colspan="1" rowspan="1">100% (9/9)(70.08 - 100)</td><td colspan="1" rowspan="1">100% (4/4)(51.01 - 100)</td><td colspan="1" rowspan="1">100% (9/9)(70.08 - 100)</td></tr><tr><td colspan="1" rowspan="1">E. coli gyrA-MT</td><td colspan="1" rowspan="1">13</td><td colspan="1" rowspan="1">100% (7/7)(64.57 - 100)</td><td colspan="1" rowspan="1">100% (6/6)(60.97 - 100)</td><td colspan="1" rowspan="1">100% (7/7)(64.57 - 100)</td><td colspan="1" rowspan="1">100% (6/6)(60.97 - 100)</td></tr><tr><td colspan="1" rowspan="1">KPC</td><td colspan="1" rowspan="1">13</td><td colspan="1" rowspan="1">100% (2/2)(34.24 - 100)</td><td colspan="1" rowspan="1">100% (11/11)(74.12 - 100)</td><td colspan="1" rowspan="1">100% (2/2)(34.24 - 100)</td><td colspan="1" rowspan="1">100% (11/11)(74.12 - 100)</td></tr><tr><td colspan="1" rowspan="1">MCR-1</td><td colspan="1" rowspan="1">13</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">100% (13/13)(77.19 - 100)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">100% (13/13)(77.19 - 100)</td></tr><tr><td colspan="1" rowspan="1">OXA-1</td><td colspan="1" rowspan="1">13</td><td colspan="1" rowspan="1">100% (4/4)(51.01- 100)</td><td colspan="1" rowspan="1">100% (9/9)(70.08 - 100)</td><td colspan="1" rowspan="1">100% (4/4)(51.01- 100)</td><td colspan="1" rowspan="1">100% (9/9)(70.08 - 100)</td></tr><tr><td colspan="1" rowspan="1">OXA-9</td><td colspan="1" rowspan="1">13</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">100% (13/13)(77.19 - 100)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">100% (13/13)(77.19 - 100)</td></tr><tr><td colspan="1" rowspan="1">SHV</td><td colspan="1" rowspan="1">13</td><td colspan="1" rowspan="1">100% (2/2)(34.24 - 100)</td><td colspan="1" rowspan="1">100% (11/11)(74.12 - 100)</td><td colspan="1" rowspan="1">100% (2/2)(34.24 - 100)</td><td colspan="1" rowspan="1">100% (11/11)(74.12 - 100)</td></tr><tr><td colspan="1" rowspan="1">Sul1</td><td colspan="1" rowspan="1">13</td><td colspan="1" rowspan="1">100% (4/4)(51.01 - 100)</td><td colspan="1" rowspan="1">100% (9/9)(70.08 - 100)</td><td colspan="1" rowspan="1">100% (4/4)(51.01 - 100)</td><td colspan="1" rowspan="1">100% (9/9)(70.08 - 100)</td></tr><tr><td colspan="1" rowspan="1">Sul2</td><td colspan="1" rowspan="1">13</td><td colspan="1" rowspan="1">100% (8/8)(67.56 - 100)</td><td colspan="1" rowspan="1">100% (5/5)(56.55 - 100)</td><td colspan="1" rowspan="1">100% (8/8)(67.56 - 100)</td><td colspan="1" rowspan="1">100% (5/5)(56.55 - 100)</td></tr><tr><td colspan="1" rowspan="1">TEM</td><td colspan="1" rowspan="1">13</td><td colspan="1" rowspan="1">100% (9/9)(70.08 - 100)</td><td colspan="1" rowspan="1">100% (4/4)(51.01 - 100)</td><td colspan="1" rowspan="1">100% (9/9)(70.08 - 100)</td><td colspan="1" rowspan="1">100% (4/4)(51.01 - 100)</td></tr><tr><td colspan="1" rowspan="2">K. pneumoniae</td><td colspan="1" rowspan="1">AAC</td><td colspan="1" rowspan="1">13</td><td colspan="1" rowspan="1">100% (8/8)(67.56 - 100)</td><td colspan="1" rowspan="1">100% (5/5)(56.55 - 100)</td><td colspan="1" rowspan="1">100% (8/8)(67.56 - 100)</td><td colspan="1" rowspan="1">100% (5/5)(56.55 - 100)</td></tr><tr><td colspan="1" rowspan="1">AAD a</td><td colspan="1" rowspan="1">13</td><td colspan="1" rowspan="1">100% (4/4)(51.01 - 100)</td><td colspan="1" rowspan="1">100% (9/9)(70.08 - 100)</td><td colspan="1" rowspan="1">75% (3/4)(30.06 - 95.44)</td><td colspan="1" rowspan="1">100% (9/9)(70.08 - 100)</td></tr><tr><td colspan="1" rowspan="1">PPA(TP/(TP+FN))(95% C)</td><td colspan="1" rowspan="1">NPA(TN/(TN+FP))(5% C)</td><td colspan="1" rowspan="1">PPA(TP/(TP+FN))(5% C)</td><td colspan="1" rowspan="1">NPA(TN/(TN+FP))(5% CI)</td></tr><tr><td colspan="1" rowspan="16"></td><td colspan="1" rowspan="1">APH</td><td colspan="1" rowspan="1">13</td><td colspan="1" rowspan="1">100% (1/1)(20.65 - 100)</td><td colspan="1" rowspan="1">100% (12/12)(75.75 - 100)</td><td colspan="1" rowspan="1">100% (1/1)(20.65 - 100)</td><td colspan="1" rowspan="1">100% (12/12)(75.75 - 100)</td></tr><tr><td colspan="1" rowspan="1">CMY</td><td colspan="1" rowspan="1">13</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">100% (13/13)(77.19 - 100)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">100% (13/13)(77.19 - 100)</td></tr><tr><td colspan="1" rowspan="1">CTX-M-1 b</td><td colspan="1" rowspan="1">13</td><td colspan="1" rowspan="1">85.7% (6/7)(48.69 - 97.43)</td><td colspan="1" rowspan="1">100% (6/6)(60.97 - 00)</td><td colspan="1" rowspan="1">100% (7/7)(64.57 - 100)</td><td colspan="1" rowspan="1">100% (6/6)(60.97 - 100)</td></tr><tr><td colspan="1" rowspan="1">CTX-M-9</td><td colspan="1" rowspan="1">13</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">100% (13/13)(77.19 - 100)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">100% (13/13)(77.19 - 100)</td></tr><tr><td colspan="1" rowspan="1">DFR</td><td colspan="1" rowspan="1">13</td><td colspan="1" rowspan="1">100% (1/1)(20.65 - 100)</td><td colspan="1" rowspan="1">100% (12/12)(75.75 - 100)</td><td colspan="1" rowspan="1">100% (1/1)(20.65 - 100)</td><td colspan="1" rowspan="1">100% (12/12)(75.75 - 100)</td></tr><tr><td colspan="1" rowspan="1">DHA</td><td colspan="1" rowspan="1">13</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">100% (13/13)(77.19 - 100</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">100% (13/13)(77.19 - 100)</td></tr><tr><td colspan="1" rowspan="1">IMP</td><td colspan="1" rowspan="1">13</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">100% (13/13)(77.19 - 100)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">100% (13/13)(77.19 - 100)</td></tr><tr><td colspan="1" rowspan="1">KPC</td><td colspan="1" rowspan="1">13</td><td colspan="1" rowspan="1">100% (3/3)(43.85 - 100)</td><td colspan="1" rowspan="1">100% (10/10)(72.25 - 100)</td><td colspan="1" rowspan="1">100% (3/3)(43.85 - 100)</td><td colspan="1" rowspan="1">100% (10/10)(72.25 - 100)</td></tr><tr><td colspan="1" rowspan="1">NDM</td><td colspan="1" rowspan="1">13</td><td colspan="1" rowspan="1">100% (1/1)(20.65 - 100)</td><td colspan="1" rowspan="1">100% (12/12)(75.75 - 100)</td><td colspan="1" rowspan="1">100% (1/1)(20.65 - 100</td><td colspan="1" rowspan="1">100% (12/12)(75.75 - 100)</td></tr><tr><td colspan="1" rowspan="1">OXA-1</td><td colspan="1" rowspan="1">13</td><td colspan="1" rowspan="1">100% (4/4)(51.01 - 100)</td><td colspan="1" rowspan="1">100% (9/9)(70.08 - 100)</td><td colspan="1" rowspan="1">100% (4/4)(51.01 -100)</td><td colspan="1" rowspan="1">100% (9/9)(70.08 - 100)</td></tr><tr><td colspan="1" rowspan="1">OXA-9</td><td colspan="1" rowspan="1">13</td><td colspan="1" rowspan="1">100% (1/1)(20.65 - 100</td><td colspan="1" rowspan="1">100% (12/12)(75.75 - 100)</td><td colspan="1" rowspan="1">100% (1/1)(20.65 - 100)</td><td colspan="1" rowspan="1">100% (12/12)(75.75 - 100)</td></tr><tr><td colspan="1" rowspan="1">OXA-48</td><td colspan="1" rowspan="1">13</td><td colspan="1" rowspan="1">100% (3/3)(43.85 - 100)</td><td colspan="1" rowspan="1">100% (10/10)(72.25 - 100)</td><td colspan="1" rowspan="1">100% (3/3)(43.85 - 100)</td><td colspan="1" rowspan="1">100% (10/10)(72.25 - 100)</td></tr><tr><td colspan="1" rowspan="1">RMTb</td><td colspan="1" rowspan="1">13</td><td colspan="1" rowspan="1">50% (1/2)(9.45 - 90.55)</td><td colspan="1" rowspan="1">100% (11/11)(74.12 - 100)</td><td colspan="1" rowspan="1">100% (2/2)(34.24 - 100)</td><td colspan="1" rowspan="1">100% (11/11)(74.12 - 100)</td></tr><tr><td colspan="1" rowspan="1">Sull a</td><td colspan="1" rowspan="1">13</td><td colspan="1" rowspan="1">100% (5/5)(56.55 - 100)</td><td colspan="1" rowspan="1">100% (8/8)(67.56 - 100)</td><td colspan="1" rowspan="1">80% (4/5)(37.55 - 96.38)</td><td colspan="1" rowspan="1">100% (8/8)(67.56 - 100)</td></tr><tr><td colspan="1" rowspan="1">Sul2</td><td colspan="1" rowspan="1">13</td><td colspan="1" rowspan="1">100% (2/2)(34.24 - 100)</td><td colspan="1" rowspan="1">100% (11/11)(74.12 - 100)</td><td colspan="1" rowspan="1">100% (2/2)(34.24 - 100)</td><td colspan="1" rowspan="1">100% (11/11)(74.12 - 100)</td></tr><tr><td colspan="1" rowspan="1">TEM b</td><td colspan="1" rowspan="1">13</td><td colspan="1" rowspan="1">83.3% (5/6)(43.65 - 96.99)</td><td colspan="1" rowspan="1">100% (7/7)(64.57 - 100)</td><td colspan="1" rowspan="1">100% (6/6)(60.97 - 100)</td><td colspan="1" rowspan="1">100% (7/7)(64.57 - 100)</td></tr><tr><td colspan="1" rowspan="12">P. aeruginosa</td><td colspan="1" rowspan="1">AAC</td><td colspan="1" rowspan="1">13</td><td colspan="1" rowspan="1">85.7% (6/7)(48.69 - 97.43)</td><td colspan="1" rowspan="1">100% (6/6)(60.97 - 100)</td><td colspan="1" rowspan="1">85.7% (6/7)(48.69 - 97.43)</td><td colspan="1" rowspan="1">100% (6/6)(60.97 - 100)</td></tr><tr><td colspan="1" rowspan="1">ANT</td><td colspan="1" rowspan="1">13</td><td colspan="1" rowspan="1">100% (1/1)(20.65 - 100)</td><td colspan="1" rowspan="1">100% (12/12)(75.75 - 100)</td><td colspan="1" rowspan="1">100% (1/1)(20.65 - 100)</td><td colspan="1" rowspan="1">100% (12/12)(75.75 - 100)</td></tr><tr><td colspan="1" rowspan="2">CTX-M-1</td><td colspan="1" rowspan="2">13</td><td colspan="1" rowspan="1">0% (0/1)</td><td colspan="1" rowspan="1">100% (12/12)</td><td colspan="1" rowspan="1">0% (0/1)</td><td colspan="1" rowspan="1">100% (12/12)</td></tr><tr><td colspan="1" rowspan="1">(0 - 79.35)</td><td colspan="1" rowspan="1">(75.75 - 100)</td><td colspan="1" rowspan="1">(0 - 79.35)</td><td colspan="1" rowspan="1">(75.75 - 100)</td></tr><tr><td colspan="1" rowspan="1">CTX-M-2</td><td colspan="1" rowspan="1">13</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">100% (13/13)(77.19 - 100)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">100% (13/13)(77.19 - 100)</td></tr><tr><td colspan="1" rowspan="1">P. aeruginosagyrA-MT</td><td colspan="1" rowspan="1">12 d</td><td colspan="1" rowspan="1">100% (7/7)(64.57 - 100)</td><td colspan="1" rowspan="1">100% (5/5)(56.55 - 100)</td><td colspan="1" rowspan="1">85.7% (6/7)(48.69 - 97.43)</td><td colspan="1" rowspan="1">100% (5/5)(56.55 - 100)</td></tr><tr><td colspan="1" rowspan="1">KPC</td><td colspan="1" rowspan="1">13</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">100% (13/13)(77.19 - 100)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">100% (13/13)(77.19 - 100)</td></tr><tr><td colspan="1" rowspan="1">NDM</td><td colspan="1" rowspan="1">13</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">100% (13/13)(77.19 - 100)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">100% (13/13)(77.19 - 100)</td></tr><tr><td colspan="1" rowspan="1">OXA-1</td><td colspan="1" rowspan="1">13</td><td colspan="1" rowspan="1">50% (1/2)(9.45 - 90.55)</td><td colspan="1" rowspan="1">100% (11/11)(74.12 - 100)</td><td colspan="1" rowspan="1">50% (1/2)(9.45 - 90.55)</td><td colspan="1" rowspan="1">100% (11/11)(74.12 - 100)</td></tr><tr><td colspan="1" rowspan="1">PER</td><td colspan="1" rowspan="1">13</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">100% (13/13)(77.19 - 100)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">100% (13/13)(77.19 - 100)</td></tr><tr><td colspan="1" rowspan="1">SHV</td><td colspan="1" rowspan="1">13</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">100% (13/13)(77.19 - 100)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">100% (13/13)(77.19 - 100)</td></tr><tr><td colspan="1" rowspan="1">TEM</td><td colspan="1" rowspan="1">13</td><td colspan="1" rowspan="1">0% (0/1)(0 - 79.35)</td><td colspan="1" rowspan="1">100% (12/12)(75.75 - 100)</td><td colspan="1" rowspan="1">0% (0/1)(0 - 79.35)</td><td colspan="1" rowspan="1">100% (12/12)(75.75 - 100)</td></tr><tr><td colspan="1" rowspan="1">PPA(TP/(TP+FN))(95% CI)</td><td colspan="1" rowspan="1">NPA(TN/(TN+FP))(95% CI)</td><td colspan="1" rowspan="1">PPA(TP/(TP+FN))(95% CI)</td><td colspan="1" rowspan="1">NPA(TN/(TN+FP))(95% CI)</td></tr><tr><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="1">VEB d</td><td colspan="1" rowspan="1">13</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">92.3% (12/13)(66.69 - 98.63)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">92.3% (12/13)(66.69 - 98.63)</td></tr><tr><td colspan="1" rowspan="1">VIM</td><td colspan="1" rowspan="1">13</td><td colspan="1" rowspan="1">100% (4/4)(51.01 - 100)</td><td colspan="1" rowspan="1">100% (9/9)(70.08 - 100)</td><td colspan="1" rowspan="1">100% (4/4)(51.01 - 100)</td><td colspan="1" rowspan="1">100% (9/9)(70.08 - 100)</td></tr><tr><td colspan="1" rowspan="18">P. mirabilis</td><td colspan="1" rowspan="1">AAC e,f</td><td colspan="1" rowspan="1">13</td><td colspan="1" rowspan="1">80% (4/5)(37.55 - 96.38)</td><td colspan="1" rowspan="1">75% (6/8)(40.93 - 92.85)</td><td colspan="1" rowspan="1">100% (5/5)(56.55 - 100)</td><td colspan="1" rowspan="1">100% (8/8)(67.56 - 100)</td></tr><tr><td colspan="1" rowspan="1">ANT</td><td colspan="1" rowspan="1">13</td><td colspan="1" rowspan="1">100% (1/1)(20.65 - 100)</td><td colspan="1" rowspan="1">100% (12/12)(75.75 - 100)</td><td colspan="1" rowspan="1">100% (1/1)(20.65 - 100)</td><td colspan="1" rowspan="1">100% (12/12)(75.75 - 100)</td></tr><tr><td colspan="1" rowspan="1">APH</td><td colspan="1" rowspan="1">13</td><td colspan="1" rowspan="1">100% (2/2)(34.24 - 100)</td><td colspan="1" rowspan="1">100% (11/11)(74.12 - 100)</td><td colspan="1" rowspan="1">100% (2/2)(34.24 - 100)</td><td colspan="1" rowspan="1">100% (11/11)(74.12 - 100)</td></tr><tr><td colspan="1" rowspan="1">armA g</td><td colspan="1" rowspan="1">13</td><td colspan="1" rowspan="1">100% (1/1)(20.65 - 100)</td><td colspan="1" rowspan="1">91.7% (11/12)(64.61 - 98.51)</td><td colspan="1" rowspan="1">100% (1/1)(20.65 - 100)</td><td colspan="1" rowspan="1">91.7% (11/12)(64.61 - 98.51)</td></tr><tr><td colspan="1" rowspan="1">CMY</td><td colspan="1" rowspan="1">13</td><td colspan="1" rowspan="1">100% (2/2)(34.24 - 100)</td><td colspan="1" rowspan="1">100% (11/11)(74.12 - 100)</td><td colspan="1" rowspan="1">100% (2/2)(34.24 - 100)</td><td colspan="1" rowspan="1">100% (11/11)(74.12 - 100)</td></tr><tr><td colspan="1" rowspan="1">CTX-M-1 h</td><td colspan="1" rowspan="1">13</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">92.3% (12/13)(66.69 - 98.63)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">100% (13/13)(77.19 - 100)</td></tr><tr><td colspan="1" rowspan="1">CTX-M-2 i</td><td colspan="1" rowspan="1">13</td><td colspan="1" rowspan="1">0% (0/1)(0 - 79.35)</td><td colspan="1" rowspan="1">91.7% (11/12)(64.61 - 98.51)</td><td colspan="1" rowspan="1">100% (1/1)(20.65 - 100)</td><td colspan="1" rowspan="1">100% (12/12)(75.75 - 100)</td></tr><tr><td colspan="1" rowspan="1">CTX-M-9</td><td colspan="1" rowspan="1">13</td><td colspan="1" rowspan="1">100% (2/2)(34.24 - 100)</td><td colspan="1" rowspan="1">100% (11/11)(74.12 - 100)</td><td colspan="1" rowspan="1">100% (2/2)(34.24 - 100)</td><td colspan="1" rowspan="1">100% (11/11)(74.12 - 100)</td></tr><tr><td colspan="1" rowspan="1">DFR</td><td colspan="1" rowspan="1">13</td><td colspan="1" rowspan="1">100% (2/2)(34.24 - 100)</td><td colspan="1" rowspan="1">100% (11/11)(74.12 - 100)</td><td colspan="1" rowspan="1">100% (2/2)(34.24 - 100)</td><td colspan="1" rowspan="1">100% (11/11)(74.12 - 100)</td></tr><tr><td colspan="1" rowspan="1">KPC</td><td colspan="1" rowspan="1">13</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">100% (13/13)(77.19 - 100)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">100% (13/13)(77.19 - 100)</td></tr><tr><td colspan="1" rowspan="1">NDM</td><td colspan="1" rowspan="1">13</td><td colspan="1" rowspan="1">100% (1/1)(20.65 - 100)</td><td colspan="1" rowspan="1">100% (12/12)(75.75 - 100)</td><td colspan="1" rowspan="1">100% (1/1)(20.65 - 100)</td><td colspan="1" rowspan="1">100% (12/12)(75.75 - 100)</td></tr><tr><td colspan="1" rowspan="1">OXA-1 h</td><td colspan="1" rowspan="1">13</td><td colspan="1" rowspan="1">100% (2/2)(34.24 - 100)</td><td colspan="1" rowspan="1">90.9% (10/11)(62.26 - 98.38)</td><td colspan="1" rowspan="1">100% (2/2)(34.24 - 100)</td><td colspan="1" rowspan="1">100% (11/11)(74.12 - 100)</td></tr><tr><td colspan="1" rowspan="1">OXA-9 f</td><td colspan="1" rowspan="1">13</td><td colspan="1" rowspan="1">0% (0/1)(0 - 79.35)</td><td colspan="1" rowspan="1">100% (12/12)(75.75 - 100)</td><td colspan="1" rowspan="1">100% (1/1)(20.65 - 100)</td><td colspan="1" rowspan="1">100% (12/12)(75.75 - 100)</td></tr><tr><td colspan="1" rowspan="1">OXA-48 h</td><td colspan="1" rowspan="1">13</td><td colspan="1" rowspan="1">.</td><td colspan="1" rowspan="1">92.3% (12/13)(66.69 - 98.63)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">100% (13/13)(77.19 - 100)</td></tr><tr><td colspan="1" rowspan="1">Sul2 </td><td colspan="1" rowspan="1">13</td><td colspan="1" rowspan="1">83.3% (5/6)(43.65 - 96.99)</td><td colspan="1" rowspan="1">100% (7/7)(64.57 - 100)</td><td colspan="1" rowspan="1">100% (6/6)(60.97 - 100)</td><td colspan="1" rowspan="1">100% (7/7)(64.57 - 100)</td></tr><tr><td colspan="1" rowspan="1">TEM</td><td colspan="1" rowspan="1">13</td><td colspan="1" rowspan="1">83.3% (5/6)(43.65 - 96.99)</td><td colspan="1" rowspan="1">100% (7/7)(64.57 - 100)</td><td colspan="1" rowspan="1">100% (6/6)(60.97 - 100)</td><td colspan="1" rowspan="1">100% (7/7)(64.57 - 100)</td></tr><tr><td colspan="1" rowspan="1">VEB</td><td colspan="1" rowspan="1">13</td><td colspan="1" rowspan="1">.</td><td colspan="1" rowspan="1">100% (13/13)(77.19 - 100)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">100% (13/13)(77.19 - 100)</td></tr><tr><td colspan="1" rowspan="1">VIM</td><td colspan="1" rowspan="1">13</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">100% (13/13)(77.19 - 100)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">100% (13/13)(77.19 - 100)</td></tr></table>

a False negatives were obtained with one isolate on MacConkey agar for AAD and Sul1. This isolate was repeated on both media, and true positives were obtained for AAD and Sul1 on both media.   
b False negatives were obtained with one isolate on blood agar for CTX-M-1, RMT and TEM. Repeat testing of this isolate on both media produced false negatives for CTX-M-1, RMT and TEM on both media, which is not consistent with a media effect.

$^ c A$ false negative was obtained with one isolate on MacConkey agar for $P$ . aeruginosa gyrA mutant. This isolate was repeated on both media, and true positives were obtained for P. aeruginosa gyrA mutant on both media.

d One isolate was not evaluated for P. aeruginosa gyrA due to a mismatch in species ID from WGS and the expected species. E. coli was reported by whole genome sequencing, but the isolate was expected to be P. aeruginosa. Results of re-sequencing for this isolate agree with the Acuitas AMR Gene panel results for the isolate producing true positives for $P$ . aeruginosa gyrase mutant and VEB with true negatives for all other AMR gene variants on both media.

e False positives were obtained with two isolates on blood agar for AAC. The two isolates were repeated on both media, and true negatives were obtained for AAC for both isolates on both media.

f False negatives were obtained with one isolate on blood agar for AAC, CTX-M-2, OXA-9, Sul2 and TEM. This isolate was repeated on both media, and true positives were obtained for AAC, CTX-M-2, OXA-9, Sul2 and TEM on both media.

g False positives were obtained with one isolate on both media for armA. Substituting original results with retest results would improve isolate (testing event) agreement across all AMR genes to $92 \%$ (12/13) for P. mirabilis on both media. The lack of $100 \%$ agreement is due to the false positives on both media for armA, which is not consistent with a media effect.

h False positives were obtained with one isolate on blood agar for CTX-M-1, OXA-1 and OXA-48. This isolate was repeated on both media, and true negatives were obtained for CTX-M-1, OXA-1 and OXA-48 on both media. $^ i A$ false positive was obtained with one isolate on blood agar for CTX-M-2. This isolate was repeated on both media, and true negatives were obtained for CTX-M-2 on both media.   
$^ j ( - )$ indicates the resistance gene was not reported in the organism.

# 11. Non-Reportable Results Summary

Non-reportable results for the Acuitas AMR Gene Panel may be due to an Assay Control failure, for which the Acuitas AMR Gene Analysis Software indicates the error “Control Failed: Ct is Undetermined”, “Control Failed: Ct is above maximum”, or “Control Failed”, and does not report any gene targets. Table 24 below summarizes instances where Assay Control failures resulted in invalid/non-reportable results.

Table 24. Acuitas AMR Gene Panel Assay Control Failures per Study   

<table><tr><td rowspan=1 colspan=1>Study</td><td rowspan=1 colspan=1>InvalidResults/Total</td><td rowspan=1 colspan=1>Percentageof InvalidResults</td></tr><tr><td rowspan=1 colspan=1>Stock Bacterial Isolate Clinical Study</td><td rowspan=1 colspan=1>1/1249</td><td rowspan=1 colspan=1>0.08%</td></tr><tr><td rowspan=1 colspan=1>Prospective Bacterial Isolate Clinical Study</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.00%</td></tr><tr><td rowspan=1 colspan=1>Reproducibility Study</td><td rowspan=1 colspan=1>2/902</td><td rowspan=1 colspan=1>0.22%</td></tr><tr><td rowspan=1 colspan=1>Carry-Over/Cross Contamination Study</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.00%</td></tr><tr><td rowspan=1 colspan=1>Media Equivalency Study</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.00%</td></tr><tr><td rowspan=1 colspan=1>Analytical Reactivity</td><td rowspan=1 colspan=1>7/441</td><td rowspan=1 colspan=1>1.59%</td></tr><tr><td rowspan=1 colspan=1>Analytical Specificity (Cross- Reactivity)</td><td rowspan=1 colspan=1>22/351</td><td rowspan=1 colspan=1>6.27%</td></tr><tr><td rowspan=1 colspan=1>Fresh vs. Frozen Isolate Study</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.00%</td></tr><tr><td rowspan=1 colspan=1>Sample Stability Study</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.00%</td></tr><tr><td rowspan=1 colspan=1>Sample Stability Frozen DNA</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.00%</td></tr><tr><td rowspan=1 colspan=1>Sample Stability-Prepared Plate</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.00%</td></tr><tr><td rowspan=1 colspan=1>Overall Invalid Results</td><td rowspan=1 colspan=1>32/3604</td><td rowspan=1 colspan=1>0.89%</td></tr></table>

# B Comparison Studies:

1. Method Comparison with Predicate Device: N/A

2. Matrix Comparison: N/A

# C Clinical Studies:

# 1. Clinical Sensitivity:

The performance characteristics of the Acuitas AMR Gene Panel with bacterial isolates were determined in a multi-site investigational clinical study by comparing the Acuitas AMR Gene Panel with WGS performed on the Illumina HiSeq 4000 with 2X150 bp paired-end sequencing, species identification by MALDI-TOF MS and Antimicrobial Susceptibility Testing by broth microdilution. Four (4) geographically diverse sites participated in the testing of isolates either prospectively collected or stocked and de-identified for use in the Clinical Performance Evaluation. To be included in the study, isolates must have been previously identified as Enterobacterales, Pseudomonas aeruginosa and Enterococcus faecalis. Organism identification was confirmed for the Gram-negative organisms (e.g., Enterobacteriaceae and Pseudomonas aeruginosa) in the clinical study using a cleared platform. Organism identification for Gram-positive organisms (e.g., Enterococcus faecalis) was performed via MALDI-TOF MS. Study samples included bacterial isolates grown on blood agar. For testing with the Acuitas AMR Gene Panel, well-isolated pure colonies that grew on blood agar were diluted to a 0.5 McFarland standard equivalent suspension using the direct colony suspension method per CLSI M07 Approved Standard. Detection of AR genes in the clinical study isolates by the Acuitas AMR Gene Panel was confirmed against WGS results for each of the isolate using a validated WGS pipeline.

The Clinical Performance Evaluation was performed between September 11, 2018 and May 2, 2019. A total of 1,307 isolate samples (1,224 clinical stock isolate samples and 83 prospective isolate samples) were enrolled and evaluated in the Clinical Performance Evaluation. There were 27 isolates/testing events excluded from evaluation (22 clinical stock isolate samples and 5 prospective isolate samples).

Twenty-one total isolates (17 clinical stock isolates and 4 prospective isolate samples) were excluded as these isolates included organisms that did not meet the acceptance criteria of the study   
Five clinical stock isolates were excluded due to a testing error on a single day which resulted in no evaluable results   
one prospective isolate sample was excluded due to duplicate testing.

There were a total of eight (8) repeated samples across the studies (7 clinical stock isolate samples and 1 prospective isolate sample) with one clinical stock isolate that produced an invalid result during the study due to a failing Assay Control. Repeat tested yielded valid results. A total of 57 positive external controls and 57 negative external controls were tested, where 55/57 $( 9 6 . 5 \% )$ of both controls returned expected results.

Results and performance of the Acuitas AMR Gene Panel for detection of AMR genes versus WGS results in the Clinical Evaluation are detailed in Table 25 below, which summarizes the Acuitas AMR Gene Panel assays for applicable organisms according to the intended use. Positive Percent Agreement (PPA) and Negative Percent Agreement (NPA) for AMR genes ranged from $9 4 . 4 \%$ to $100 \%$ and 96.5 to $100 \%$ , respectively, for the combined stock isolates and prospective isolates.

Table 25. Clinical Performance for the Acuitas AMR Gene Panel (PCR/WGS)   

<table><tr><td rowspan=2 colspan=1>AMR GeneTarget</td><td rowspan=1 colspan=3>PPA</td><td rowspan=1 colspan=3>NPA</td><td rowspan=2 colspan=1>TotalUniqueStrains(n)</td><td rowspan=2 colspan=1>TotalReplicates((n)r</td><td rowspan=1 colspan=2>Positive for AMRTargets by WGS</td><td rowspan=1 colspan=2>Negative for AMRTarget by WGS</td></tr><tr><td rowspan=1 colspan=1>TP/(TP+FN)</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=1>TN/(TN+FP)</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=1>Unique Strains(n)</td><td rowspan=1 colspan=1>Replicatesp(()r</td><td rowspan=1 colspan=1>Unique Strains(n)</td><td rowspan=1 colspan=1>Replicatesp(n)r</td></tr><tr><td rowspan=1 colspan=1>AAC</td><td rowspan=1 colspan=1>610/(622)a-c</td><td rowspan=1 colspan=1>98.1</td><td rowspan=1 colspan=1>96.66-98.89</td><td rowspan=1 colspan=1>536/(545)d-e</td><td rowspan=1 colspan=1>98.3</td><td rowspan=1 colspan=1>96.89-99.13</td><td rowspan=1 colspan=1>577</td><td rowspan=1 colspan=1>732</td><td rowspan=1 colspan=1>315</td><td rowspan=1 colspan=1>386</td><td rowspan=1 colspan=1>262</td><td rowspan=1 colspan=1>346</td></tr><tr><td rowspan=1 colspan=1>AAD</td><td rowspan=1 colspan=1>128/(130)</td><td rowspan=1 colspan=1>98.5</td><td rowspan=1 colspan=1>94.56-99.58</td><td rowspan=1 colspan=1>192/(199)f</td><td rowspan=1 colspan=1>96.5</td><td rowspan=1 colspan=1>92.92-98.29</td><td rowspan=1 colspan=1>185</td><td rowspan=1 colspan=1>189</td><td rowspan=1 colspan=1>98</td><td rowspan=1 colspan=1>54</td><td rowspan=1 colspan=1>87</td><td rowspan=1 colspan=1>135</td></tr><tr><td rowspan=1 colspan=1>ANT</td><td rowspan=1 colspan=1>203/(205)</td><td rowspan=1 colspan=1>99.0</td><td rowspan=1 colspan=1>96.51 -99.73</td><td rowspan=1 colspan=1>628/(633)g</td><td rowspan=1 colspan=1>99.2</td><td rowspan=1 colspan=1>98.16 - 99.66</td><td rowspan=1 colspan=1>392</td><td rowspan=1 colspan=1>543</td><td rowspan=1 colspan=1>64</td><td rowspan=1 colspan=1>168</td><td rowspan=1 colspan=1>328</td><td rowspan=1 colspan=1>375</td></tr><tr><td rowspan=1 colspan=1>APH</td><td rowspan=1 colspan=1>39/(40)</td><td rowspan=1 colspan=1>97.5</td><td rowspan=1 colspan=1>87.12-99.56</td><td rowspan=1 colspan=1>443/(444)</td><td rowspan=1 colspan=1>99.8</td><td rowspan=1 colspan=1>98.74-99.96</td><td rowspan=1 colspan=1>263</td><td rowspan=1 colspan=1>280</td><td rowspan=1 colspan=1>31</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>232</td><td rowspan=1 colspan=1>268</td></tr><tr><td rowspan=1 colspan=1>armA</td><td rowspan=1 colspan=1>8/(8)</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>67.56-100.00</td><td rowspan=1 colspan=1>147/(147)</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>97.45-100.00</td><td rowspan=1 colspan=1>78</td><td rowspan=1 colspan=1>91</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>74</td><td rowspan=1 colspan=1>83</td></tr><tr><td rowspan=1 colspan=1>CMY</td><td rowspan=1 colspan=1>126/(128)h</td><td rowspan=1 colspan=1>98.4</td><td rowspan=1 colspan=1>94.48-99.57</td><td rowspan=1 colspan=1>688/(691)i</td><td rowspan=1 colspan=1>99.6</td><td rowspan=1 colspan=1>98.73-99.85</td><td rowspan=1 colspan=1>422</td><td rowspan=1 colspan=1>489</td><td rowspan=1 colspan=1>55</td><td rowspan=1 colspan=1>84</td><td rowspan=1 colspan=1>367</td><td rowspan=1 colspan=1>405</td></tr><tr><td rowspan=1 colspan=1>CTX-M-1</td><td rowspan=1 colspan=1>264/(273)</td><td rowspan=1 colspan=1>96.7</td><td rowspan=1 colspan=1>93.85-98.26</td><td rowspan=1 colspan=1>929/(938)</td><td rowspan=1 colspan=1>99.0</td><td rowspan=1 colspan=1>98.19-99.49</td><td rowspan=1 colspan=1>621</td><td rowspan=1 colspan=1>732</td><td rowspan=1 colspan=1>162</td><td rowspan=1 colspan=1>143</td><td rowspan=1 colspan=1>459</td><td rowspan=1 colspan=1>589</td></tr><tr><td rowspan=1 colspan=1>CTX-M-2</td><td rowspan=1 colspan=1>35/(35)</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>90.11-100.00</td><td rowspan=1 colspan=1>801/(803)</td><td rowspan=1 colspan=1>99.8</td><td rowspan=1 colspan=1>99.10-99.93</td><td rowspan=1 colspan=1>392</td><td rowspan=1 colspan=1>543</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>27</td><td rowspan=1 colspan=1>371</td><td rowspan=1 colspan=1>516</td></tr><tr><td rowspan=1 colspan=1>CTX-M-9</td><td rowspan=1 colspan=1>73/(74)</td><td rowspan=1 colspan=1>98.6</td><td rowspan=1 colspan=1>92.73-99.76</td><td rowspan=1 colspan=1>781/(782)</td><td rowspan=1 colspan=1>99.9</td><td rowspan=1 colspan=1>99.28-99.98</td><td rowspan=1 colspan=1>459</td><td rowspan=1 colspan=1>489</td><td rowspan=1 colspan=1>58</td><td rowspan=1 colspan=1>23</td><td rowspan=1 colspan=1>401</td><td rowspan=1 colspan=1>466</td></tr><tr><td rowspan=1 colspan=1>DFR</td><td rowspan=1 colspan=1>167/(169)</td><td rowspan=1 colspan=1>98.8</td><td rowspan=1 colspan=1>95.79-99.67</td><td rowspan=1 colspan=1>646/(650)</td><td rowspan=1 colspan=1>99.4</td><td rowspan=1 colspan=1>98.43-99.76</td><td rowspan=1 colspan=1>422</td><td rowspan=1 colspan=1>489</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>96</td><td rowspan=1 colspan=1>332</td><td rowspan=1 colspan=1>393</td></tr><tr><td rowspan=1 colspan=1>DHA</td><td rowspan=1 colspan=1>36/(36)</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>90.36-1000.00</td><td rowspan=1 colspan=1>293/(293)</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>98.71-100.00</td><td rowspan=1 colspan=1>185</td><td rowspan=1 colspan=1>189</td><td rowspan=1 colspan=1>33</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>152</td><td rowspan=1 colspan=1>183</td></tr><tr><td rowspan=1 colspan=1>E. coli gyrAMutant</td><td rowspan=1 colspan=1>160/(163)</td><td rowspan=1 colspan=1>98.2</td><td rowspan=1 colspan=1>94.73-99.37</td><td rowspan=1 colspan=1>167/(168)</td><td rowspan=1 colspan=1>99.4</td><td rowspan=1 colspan=1>96.71-99.89</td><td rowspan=1 colspan=1>155</td><td rowspan=1 colspan=1>209</td><td rowspan=1 colspan=1>81</td><td rowspan=1 colspan=1>101</td><td rowspan=1 colspan=1>74</td><td rowspan=1 colspan=1>108</td></tr><tr><td rowspan=1 colspan=1>IMP</td><td rowspan=1 colspan=1>72/(72)</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>94.93-100.00</td><td rowspan=1 colspan=1>257/(257)</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>98.53-100.00</td><td rowspan=1 colspan=1>185</td><td rowspan=1 colspan=1>189</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>71</td><td rowspan=1 colspan=1>180</td><td rowspan=1 colspan=1>118</td></tr><tr><td rowspan=1 colspan=1>KPC</td><td rowspan=1 colspan=1>75/(77)</td><td rowspan=1 colspan=1>97.4</td><td rowspan=1 colspan=1>91.02-99.28</td><td rowspan=1 colspan=1>1130/(1134)</td><td rowspan=1 colspan=1>99.6</td><td rowspan=1 colspan=1>99.10-99.86</td><td rowspan=1 colspan=1>621</td><td rowspan=1 colspan=1>732</td><td rowspan=1 colspan=1>63</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>558</td><td rowspan=1 colspan=1>711</td></tr><tr><td rowspan=1 colspan=1>MCR-1</td><td rowspan=1 colspan=1>51/(54)k</td><td rowspan=1 colspan=1>94.4</td><td rowspan=1 colspan=1>84.89-98.09</td><td rowspan=1 colspan=1>281/(281)</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>98.65-100.00</td><td rowspan=1 colspan=1>159</td><td rowspan=1 colspan=1>209</td><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>48</td><td rowspan=1 colspan=1>145</td><td rowspan=1 colspan=1>161</td></tr><tr><td rowspan=1 colspan=1>NDM</td><td rowspan=1 colspan=1>56/(57)</td><td rowspan=1 colspan=1>98.2</td><td rowspan=1 colspan=1>90.71-99.69</td><td rowspan=1 colspan=1>801/(805)</td><td rowspan=1 colspan=1>99.5</td><td rowspan=1 colspan=1>98.73-99.81</td><td rowspan=1 colspan=1>448</td><td rowspan=1 colspan=1>523</td><td rowspan=1 colspan=1>47</td><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>401</td><td rowspan=1 colspan=1>506</td></tr><tr><td rowspan=1 colspan=1>OXA-1</td><td rowspan=1 colspan=1>240/(249)</td><td rowspan=1 colspan=1>96.4</td><td rowspan=1 colspan=1>93.27 -98.09</td><td rowspan=1 colspan=1>910/(918)</td><td rowspan=1 colspan=1>99.1</td><td rowspan=1 colspan=1>98.29 - 99.56</td><td rowspan=1 colspan=1>577</td><td rowspan=1 colspan=1>732</td><td rowspan=1 colspan=1>112</td><td rowspan=1 colspan=1>161</td><td rowspan=1 colspan=1>465</td><td rowspan=1 colspan=1>571</td></tr><tr><td rowspan=1 colspan=1>OXA-9</td><td rowspan=1 colspan=1>58/(58)</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>93.79-100.00</td><td rowspan=1 colspan=1>760/(761)</td><td rowspan=1 colspan=1>99.9</td><td rowspan=1 colspan=1>99.26-99.98</td><td rowspan=1 colspan=1>422</td><td rowspan=1 colspan=1>489</td><td rowspan=1 colspan=1>47</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>375</td><td rowspan=1 colspan=1>468</td></tr><tr><td rowspan=1 colspan=1>OXA-48</td><td rowspan=1 colspan=1>59/(62)</td><td rowspan=1 colspan=1>95.2</td><td rowspan=1 colspan=1>86.71-98.34</td><td rowspan=1 colspan=1>448/(452)</td><td rowspan=1 colspan=1>99.1</td><td rowspan=1 colspan=1>97.75-99.66</td><td rowspan=1 colspan=1>293</td><td rowspan=1 colspan=1>280</td><td rowspan=1 colspan=1>48</td><td rowspan=1 colspan=1>27</td><td rowspan=1 colspan=1>245</td><td rowspan=1 colspan=1>253</td></tr><tr><td rowspan=1 colspan=1>PER</td><td rowspan=1 colspan=1>81/(82)</td><td rowspan=1 colspan=1>98.8</td><td rowspan=1 colspan=1>93.41-99.78</td><td rowspan=1 colspan=1>265/(266)</td><td rowspan=1 colspan=1>99.6</td><td rowspan=1 colspan=1>97.90-99.93</td><td rowspan=1 colspan=1>155</td><td rowspan=1 colspan=1>243</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>81</td><td rowspan=1 colspan=1>146</td><td rowspan=1 colspan=1>162</td></tr><tr><td rowspan=1 colspan=1>P. aeruginosagyrA Mutant</td><td rowspan=1 colspan=1>265/(279)m</td><td rowspan=1 colspan=1>95.0</td><td rowspan=1 colspan=1>91.75-96.99</td><td rowspan=1 colspan=1>67/(68)</td><td rowspan=1 colspan=1>98.5</td><td rowspan=1 colspan=1>92.13-99.74</td><td rowspan=1 colspan=1>154</td><td rowspan=1 colspan=1>243</td><td rowspan=1 colspan=1>103</td><td rowspan=1 colspan=1>216</td><td rowspan=1 colspan=1>51</td><td rowspan=1 colspan=1>27</td></tr><tr><td rowspan=1 colspan=1>RMT</td><td rowspan=1 colspan=1>31/(32)</td><td rowspan=1 colspan=1>96.9</td><td rowspan=1 colspan=1>84.26-99.45</td><td rowspan=1 colspan=1>297/(297)</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>98.72-100.00</td><td rowspan=1 colspan=1>185</td><td rowspan=1 colspan=1>189</td><td rowspan=1 colspan=1>27</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>158</td><td rowspan=1 colspan=1>179</td></tr><tr><td rowspan=1 colspan=1>SHV</td><td rowspan=1 colspan=1>12/(12)</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>75.75-100.00</td><td rowspan=1 colspan=1>668/(671)</td><td rowspan=1 colspan=1>99.6</td><td rowspan=1 colspan=1>98.69 - 99.85</td><td rowspan=1 colspan=1>314</td><td rowspan=1 colspan=1>452</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>304</td><td rowspan=1 colspan=1>448</td></tr><tr><td rowspan=1 colspan=1>Sull</td><td rowspan=1 colspan=1>420/(424)</td><td rowspan=1 colspan=1>99.1</td><td rowspan=1 colspan=1>97.60-99.63</td><td rowspan=1 colspan=1>232/(240)n</td><td rowspan=1 colspan=1>96.7</td><td rowspan=1 colspan=1>93.56-98.30</td><td rowspan=1 colspan=1>344</td><td rowspan=1 colspan=1>398</td><td rowspan=1 colspan=1>226</td><td rowspan=1 colspan=1>249</td><td rowspan=1 colspan=1>118</td><td rowspan=1 colspan=1>149</td></tr><tr><td rowspan=1 colspan=1>Sul2</td><td rowspan=1 colspan=1>489/(501)</td><td rowspan=1 colspan=1>97.6</td><td rowspan=1 colspan=1>95.86-98.62</td><td rowspan=1 colspan=1>307/(318)°</td><td rowspan=1 colspan=1>96.5</td><td rowspan=1 colspan=1>93.91-98.06</td><td rowspan=1 colspan=1>422</td><td rowspan=1 colspan=1>489</td><td rowspan=1 colspan=1>212</td><td rowspan=1 colspan=1>331</td><td rowspan=1 colspan=1>210</td><td rowspan=1 colspan=1>158</td></tr><tr><td rowspan=1 colspan=1>TEM</td><td rowspan=1 colspan=1>600/(609)p</td><td rowspan=1 colspan=1>98.5</td><td rowspan=1 colspan=1>97.22 -99.22</td><td rowspan=1 colspan=1>559/(572)9</td><td rowspan=1 colspan=1>97.7</td><td rowspan=1 colspan=1>96.15 - 98.67</td><td rowspan=1 colspan=1>591</td><td rowspan=1 colspan=1>732</td><td rowspan=1 colspan=1>277</td><td rowspan=1 colspan=1>391</td><td rowspan=1 colspan=1>314</td><td rowspan=1 colspan=1>341</td></tr><tr><td rowspan=1 colspan=1>vanA</td><td rowspan=1 colspan=1>57(57)</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>93.69-100.00</td><td rowspan=1 colspan=1>36/(36)</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>90.36-100.00</td><td rowspan=1 colspan=1>43</td><td rowspan=1 colspan=1>54</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>52</td><td rowspan=1 colspan=1>35</td><td rowspan=1 colspan=1>2</td></tr><tr><td rowspan=1 colspan=1>VEB</td><td rowspan=1 colspan=1>89/(89)</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>95.86-100.00</td><td rowspan=1 colspan=1>411/(414)</td><td rowspan=1 colspan=1>99.3</td><td rowspan=1 colspan=1>97.89-99.75</td><td rowspan=1 colspan=1>233</td><td rowspan=1 colspan=1>334</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>72</td><td rowspan=1 colspan=1>209</td><td rowspan=1 colspan=1>262</td></tr><tr><td rowspan=1 colspan=1>VIM</td><td rowspan=1 colspan=1>91/(93)</td><td rowspan=1 colspan=1>97.8</td><td rowspan=1 colspan=1>92.49-99.41</td><td rowspan=1 colspan=1>409/(410)</td><td rowspan=1 colspan=1>99.8</td><td rowspan=1 colspan=1>98.63-99.96</td><td rowspan=1 colspan=1>233</td><td rowspan=1 colspan=1>334</td><td rowspan=1 colspan=1>22</td><td rowspan=1 colspan=1>80</td><td rowspan=1 colspan=1>211</td><td rowspan=1 colspan=1>254</td></tr></table>

a One (1) FN result attributed to the presence of an aac(3)-IIa gene variant that had no valid alignment with the primers/probe of the AAC assay harbored by a single K. pneumoniae unique isolate. b Two (2) FN results due to testing of two (2) replicates of a single unique $E .$ . coli isolate. c Two (2) FN results due to testing of two (2) replicates of a single unique $P$ . aeruginosa isolate. d One (1) FP result attributed to the presence of a truncated aac(3)-II gene harbored by a single unique E. coli isolate.

e One (1) FP result attributed to the presence of a truncated aac(3)-Ib gene in a single unique K. pneumoniae isolate. $f$ Three (3) FP results attributed to an aadA15 gene variant harbored by three (3) K. pneumoniae isolates with high numbers $( \geq 3 )$ of mismatches in the reverse primer of the AAD assay, with two (2) isolates tested as replicates of a single unique strain. Two (2) additional FP results from K. pneumoniae isolates demonstrated alignment of the AAD assay primers/probe with high numbers of mismatches $( \geq 3 )$ in the reverse primer, but no attributable gene variant was detected in the AR database used for analysis.

g One (1) FP result attributed to high PCR baseline drift and not true amplification of the ANT target assay in one (1) unique P. aeruginosa isolate.

h Two (2) FN results attributed to the presence of CMY gene variants with high numbers of mismatches $( \geq 3 )$ to the primers of the CMY assay in two (2) E. coli isolates. One (1) isolate harbored a blaCMY-2 gene variant and one (1) isolate harbored a blaCMY-42 gene variant.

i Four (4) FP results attributed to the presence of a CMY gene variant with high numbers of mismatches $\geq 3 ,$ to the primers of the CMY assay in four (4) E. coli isolates. These four (4) E. coli isolates represented two (2) unique strains, each tested in two (2) replicates. Both isolates harbored a blaCMY-4 gene variant. Seven (7) FP results attributed to the presence of a blaCMY-16 gene variant with high numbers of mismatches $( \geq 3 )$ to the reverse primer of the CMY assay in six (6) unique K. pneumoniae isolates, with one isolate tested in two (2) replicates. One (1) FP result attributed to the presence of a blaCMY-16 gene variant with high numbers of mismatches $( \geq 3 )$ to the reverse primer of the CMY assay in one (1) P. mirabilis isolate.

j Seven (7) FP results attributed to the presence of a blaCTX-M-27 gene variant with perfect alignment to the primers/probe of the CTX-M-1 assay for 7 replicates of (1) unique E. coli isolate that was not originally identified by WGS analysis. k Three (3) FN results due to testing of three (3) replicates of a single E. coli isolate.

l Four (4) FP results due to testing of four (4) replicates of a single K. pneumoniae isolate.   
m One (1) FN result attributed to a negative result for the P. aeruginosa ID assay for a single unique P. aeruginosa isol Amplification of the P. aeruginosa gyrA Mutant assay was present for this isolate.   
n Two (2) FP results due to testing of two (2) replicates of one unique E. coli isolate.   
o Four (4) FP results due to testing for four (4) replicates of a single unique K. pneumoniae isolate.   
p Two (2) FN results due to testing of two (2) replicates of a single unique E. coli isolate.   
q Four (4) FP results due to testing for four (4) replicates of a single unique K. pneumoniae isolate.   
r Replicates are the total number of samples for unique isolates tested multiple times.

A breakdown of the AMR gene detection results versus WGS per reportable species (or group) and the association with non-susceptibility are shown in Table 26. Resistance marker/organism combinations were evaluated based on the strength of the collective, totality of scientific evidence delineating the level of association between molecular marker detection with phenotypic, clinical resistance. Assessing resistance for certain organism/marker combinations included an evaluation of:

Organism/marker combinations with positive predictive value (PPV) observed near $80 \%$ with a specific antimicrobial   
Organism/marker combinations with well-established association between the marker and resistance, previous reports/literature of associated resistance related to the marker, and/or supporting data of potential resistance established across related bacterial group with PPV near $80 \%$ for a specific antimicrobial.

Table 26 includes a break-down of the reported PPV percentages observed during the clinical study and for OpGen’s SMART Study per marker/organism/drug combination, PPA and NPA values for marker detection by the Acuitas AMR Gene Panel, and information on organism/marker detection in NCBI (2019). PPV percentages show the resistance (nonsusceptible) of isolates when the marker was detected. Additional information concerning AMR gene detection and AST results was provided from an OpGen study performed with clinical isolates collected through a Study for Monitoring Antimicrobial Resistance Trends (SMART) including $E .$ . coli $( \mathtt { n } { = } 2 , 9 1 9$ ), $K .$ pneumoniae $( \mathrm { n } { = } 1 , 9 7 4$ ), $P .$ . mirabilis $( \mathrm { n } { = } 1 , 1 5 0 )$ ) and $P$ . aeruginosa $( \mathrm { n } { = } 1 , 4 8 4 )$ species. Organism/drug/resistance marker combinations for which association with

resistance was demonstrated are bolded and are highlighted in grey rows in Table 26 shown on pages 33-110. Where applicable, results of molecular resistance subtypes are included.

Table 26. Association with Non-Susceptibility or Resistance   

<table><tr><td colspan="1" rowspan="2">AMR GeneTarget</td><td colspan="1" rowspan="2">Organism</td><td colspan="2" rowspan="1">Detection of AMR GeneTarget by AMR GenePanel versus WholeGenome SequencingCombined Clinical DevicePerformance -Stock and ProspectiveIsolates (from K191288)</td><td colspan="1" rowspan="2">Antimicrobial</td><td colspan="4" rowspan="1">Detection of AMR Gene Target by AMR GenePanel vs AST Non-SusceptibilityCombined Clinical Device Performance - Stock andProspective Isolates (from K191288) includingonly unique isolates (no replicates)</td><td colspan="5" rowspan="1">Detection of AMR Gene Target versus AST Non-Susceptibility inOpGen's SMART Study</td><td colspan="1" rowspan="2">%Organismswith AMRGeneTarget inNCBIPathogenDatabase(2019)</td></tr><tr><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1">PPV[95% CI]</td><td colspan="1" rowspan="1">NPV[95% CI]</td><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1">PPV[95% CI]</td><td colspan="1" rowspan="1">NPV[95% CI]</td><td colspan="1" rowspan="1">% Organismswith AMRGene Target</td></tr><tr><td colspan="1" rowspan="1">AAC</td><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">97.3%(107/110)[92.29-99.07]</td><td colspan="1" rowspan="1">98.2%(221/225)[95.52-999.31]</td><td colspan="1" rowspan="1">Amikacin</td><td colspan="1" rowspan="1">AAC:75.7%(28/37)[59.88-86.64]</td><td colspan="1" rowspan="1">AAC:59.0%(72/122)[50.14-67.34]</td><td colspan="1" rowspan="1">AAC:35.9%(28/78)[26.15-46.97]</td><td colspan="1" rowspan="1">AAC:88.9%(72/81)[80.21-94.04]</td><td colspan="1" rowspan="1">aac(3)-II(a-): 40.8%(87/213)[34.46-47.55]aac(3)-IV:0.9%(2/213)[0.26-3.36]aac(6')-lb/lb-cr:81.7%(174/213)[75.95-86.31]</td><td colspan="1" rowspan="1">aac(3)-II(a-):71.3%(1959/2747) [69.59-72.97]aac(3)-IV:99.2%(2726/2747) [98.83-99.50]ac(6')-Ib/lb-cr:71.4%(1961/2747) [69.67-73.05]</td><td colspan="1" rowspan="1">aac(3)-I(a-): 9.9%(87/875)[8.13-12.10]aac(3)-IV:8.7%(2/23)[2.42-26.80]aac(6')-Ib/lb-cr:18.1%(174/960)[15.82-20.69]</td><td colspan="1" rowspan="1">aac(3)-II(a-):94.0%(1959/2085) (92.85-94.90]aac(3)-IV:92.8%(2726/2937) [91.82-93.69]aac(6')-Ib/lb-cr:98.0%(1961/200)97.35-98.57]</td><td colspan="1" rowspan="1">aac(3)-I(a-): 29.6%(875/2960)aac(3)-IV:0.8%(23/2960)aac(6')-Ib/lb-cr: 32.4%(960/2960)</td><td colspan="1" rowspan="1">aac(3)-I(a-):7.1% of19017aac(3)-IV:2.2% of19017aac(6')-lb/b-cr:6.2% of19017</td></tr><tr><td colspan="1" rowspan="1">AAC</td><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">97.3%(107/110)[92.29-99.07]</td><td colspan="1" rowspan="1">98.2%(221/225)[95.52-99.31]</td><td colspan="1" rowspan="1">Gentamicin</td><td colspan="1" rowspan="1">AAC:78.6%(66/84)[68.65-85.99]</td><td colspan="1" rowspan="1">AAC:84.0%(63/75)[74.08-90.60]</td><td colspan="1" rowspan="1">AAC:84.6%(66/78)[75.01-90.97]</td><td colspan="1" rowspan="1">AAC:77.8%(63/81)[67.58-85.46]</td><td colspan="1" rowspan="1">aac(3)-II(a-d): 81.8%(814/995)[79.29-84.08]aac(3)-IV:1.9%(19/995)[1.23-2.96]aac(6')-lb/lb-cr:52.7%(524/995)[49.56-5.75]</td><td colspan="1" rowspan="1">aac(3)-lI(a-d): 96.9%(1904/1965) [96.03-97.58]aac(3)-IV:99.8%(1961/1965) [99.48-9.92]aac(6')-lb/b-cr:77.8%(1529/1965) [75.92-79.59]</td><td colspan="1" rowspan="1">aac(3)-II(a-d): 93.0%(814/875)[91.15-94.53]aac(3)-IV:82.6%(19/23)[62.86-93.02]aac(6')-lb/lb-cr:54.6%(524/960)[51.42-57.71]</td><td colspan="1" rowspan="1">aac(3)-lI(a-d): 91.3%(1904/2085) [90.03-92.45]aac(3)-IV:66.8%(1961/2937) [65.04-68.45]aac(6')-lb/lb-cr:76.4%(1529/2000) [74.54-78.26]</td><td colspan="1" rowspan="1">aac(3)-II(a-d): 29.6%(875/2960)aac(3)-IV:0.8%(23/2960)aac(6')-Ib/lb-cr: 32.4%(960/2960)</td><td colspan="1" rowspan="1">aac(3)-II(a-d): 7.1% of19017aac(3)-IV:2.2% of19017aac(6')-lb/b-cr:6.2% of19017</td></tr><tr><td colspan="1" rowspan="1">AAC</td><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">97.3%(107/110)[92.29-9.07]</td><td colspan="1" rowspan="1">98.2%(221/225)[95.52-9.31]</td><td colspan="1" rowspan="1">Tobramycin</td><td colspan="1" rowspan="1">AAC:82.2%(74/90)[73.06-88.75]</td><td colspan="1" rowspan="1">AAC:94.2%(65/69)[86.02-97.72]</td><td colspan="1" rowspan="1">AAC:94.9%(74/78)[87.54-97.99]</td><td colspan="1" rowspan="1">AAC:80.2%(65/81)[70.30-87.46]</td><td colspan="1" rowspan="1">aac(3)-Il(a-d): 59.0%(0/1362)[56.40-61.61]aac(3)-IV:1.5%(20/1362)[0.95-2.26]aac(6')-lbb/b-cr:64.9%(884/1362)[62.33-67.39]</td><td colspan="1" rowspan="1">aac(3)-Il(a-d): 95.6%(15277/1598) [94.43-96.46]aac(3)-IV:99.8%(1595/1598) [99.45-99.94]aac(6')-lb/b-cr:95.2%(1522/1598) [94.09-96.18]</td><td colspan="1" rowspan="1">aac(3)-Il(a-d): 91.9%804/875)[89.89-93.52]aac(3)-IV:87.0%(20/23)[67.87-95.46]ac(6')-lb/lb-cr:92.1%(884/960)[90.20-93.63]</td><td colspan="1" rowspan="1">aac(3)-I(a-d): 73.2%(1527/2085) [71.30-75.09]aac(3)-IV:54.3%(1595/2937) [52.50-566.10]aac(6')-lb/b-cr:76.1%(1522/2000) [74.18-77.92]</td><td colspan="1" rowspan="1">aac(3)-II(a-d): 29.6%(875/2960)aac(3)-IV:0.8%(23/2960)aac(6')-Ib/lb-cr: 32.4%(960/2960)</td><td colspan="1" rowspan="1">aac(3)-lI(a-d):7.1% of19017aac(3)-IV:2.% of19017aac(6')-lb/lb-cr:6.2% of19017</td></tr><tr><td colspan="1" rowspan="1">AAC</td><td colspan="1" rowspan="1">K. pneumoniae</td><td colspan="1" rowspan="1">99.6%(241/242)[97.70-99.93]</td><td colspan="1" rowspan="1">97.7%(85/87)[92.00-99.37]</td><td colspan="1" rowspan="1">Amikacin</td><td colspan="1" rowspan="1">AAC:86.7%(78/90)[78.13-92.21]</td><td colspan="1" rowspan="1">AAC:35.8%(34/95)26.845.81]</td><td colspan="1" rowspan="1">AAC:56.1%(78/139)47.81-64.09]</td><td colspan="1" rowspan="1">AAC:73.9%(34/46)[59.74-84.40]</td><td colspan="1" rowspan="1">aac(3)-I(a-):34.9%(218/624)[31.30-38.76]aac(3)-IV:4.5%(28/624)[3.12-6.41]aac(6')-lb/b-cr:76.6%(478/624)[73.12-79.76]</td><td colspan="1" rowspan="1">aac(3)-I(a-d):57.5%(808/1404)[54.95-60.11]aac(3)-IV:97.6%(1371/1404) [96.72-98.32]aac(6')-Ib/lb-cr:36.9%(518/1404)[34.41-39.45]</td><td colspan="1" rowspan="1">aac(3)-I(a-): 26.8%(218/814)[23.85-29.93]aac(3)-IV:45.9%(28/61)[34.01-58.28]aac(6')-Ib/lb-cr:35.0%(478/1364)[32.56-37.61]</td><td colspan="1" rowspan="1">aac(3)-I(a-d): 66.6%(808/1214)[63.85-69.15]aac(3)-IV:69.7%(1371/1967) [67.63-71.69]aac(6')-Ib/lb-cr:78.0%(518/664)[74.71-81.00]</td><td colspan="1" rowspan="1">aac(3)-I(a-): 40.1%(814/2028)aac(3)-IV: 3%(61/2028)aac(6')-Ib/lb-cr: 67.3%(1364/2028)</td><td colspan="1" rowspan="1">aac(3)-I(a-): 25.0%of 7979aac(3)-IV:3.2% of7979aac(6')-lb/lb-cr:51.1% of7979</td></tr><tr><td colspan="1" rowspan="2">AMR GeneTarget</td><td colspan="1" rowspan="2">Organism</td><td colspan="2" rowspan="1">Detection of AMR GeneTarget by AMR GenePanel versus WholeGenome SequencingCombined Clinical DevicePerformance -Stock and ProspectiveIsolates (from K191288)</td><td colspan="1" rowspan="2">Antimicrobial</td><td colspan="4" rowspan="1">Detection of AMR Gene Target by AMR GenePanel vs AST Non-SusceptibilityCombined Clinical Device Performance - Stock andvaonly unique isolates (no replicates)</td><td colspan="5" rowspan="1">Detection of AMR Gene Target versus AST Non-Susceptibility inOpGen's SMART Study</td><td colspan="1" rowspan="2">%Organismswith AMRGeneTarget inNCBIPathogenDatabase(2019)</td></tr><tr><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1">PPV[95% CI]</td><td colspan="1" rowspan="1">NPV[95% CI]</td><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1">PPV[95% CI]</td><td colspan="1" rowspan="1">NPV[95% CI]</td><td colspan="1" rowspan="1">% Organismswith AMRGene Target</td></tr><tr><td colspan="1" rowspan="1">AAC</td><td colspan="1" rowspan="1">K. pneumoniae</td><td colspan="1" rowspan="1">99.6%(241/242)[97.70-99.93]</td><td colspan="1" rowspan="1">97.7%(85/87)[92.00-99.37]</td><td colspan="1" rowspan="1">Gentamicin</td><td colspan="1" rowspan="1">AAC:92.6%(125/135)[86.90-95.93]</td><td colspan="1" rowspan="1">AAC:72.0%(36/50)[58.33-82.53]</td><td colspan="1" rowspan="1">AAC:89.9%(125/139)[83.80-93.91]</td><td colspan="1" rowspan="1">AAC:78.3%(36/46)[64.43-87.74]</td><td colspan="1" rowspan="1">aac(3)-Il(a-d): 65.0%(765/1177)[62.23-67.67]aac(3)-IV:4.7%(55/1177)[3.61-6.03]aac(6')-lb/lb-cr:78.2%(920/1177)[75.72-80.43]</td><td colspan="1" rowspan="1">aac(3)-II(a-d): 94.2%(802/851)[92.47-95.62]aac(3)-IV:99.3%(845/851)[98.47-99.68]aac(6')-lb/lb-cr:47.8%(407/851)[44.49-51.18]</td><td colspan="1" rowspan="1">aac(3)-II(a-d): 94.0%(765/814)[92.13-95.42]aac(3)-IV:90.2%(55/61)[80.16-9541aac(6')-lb/lb-cr:67.4%(920/1364)[64.92-6988]</td><td colspan="1" rowspan="1">aac(3)-lI(a-d): 66.1%(802/1214)[63.35-68.67]aac(3)-IV:43.0%(845/1967)[40.79-45.16]aac(6')-lb/lb-cr:61.3%(407/664)[57.54-64.92]</td><td colspan="1" rowspan="1">aac(3)-II(a-d): 40.1%(814/2028)aac(3)-IV: 3%(61/2028)aac(6')-Ib/lb-cr: 673%(1364/2028)</td><td colspan="1" rowspan="1">aac(3)-lI(a-d): 25.0%of 7979aac(3)-IV:3.2% of7979aac(6')-lb/b-cr:51.1% of7979</td></tr><tr><td colspan="1" rowspan="1">AAC</td><td colspan="1" rowspan="1">K. pneumoniae</td><td colspan="1" rowspan="1">99.6%(241/242)[97.70-99.93]</td><td colspan="1" rowspan="1">97.7%(85/87)[92.00-99.37]</td><td colspan="1" rowspan="1">Tobramycin</td><td colspan="1" rowspan="1">AAC:92.5%(136/147)[87.10-5.77]</td><td colspan="1" rowspan="1">AAC:92.1%(35/38)[79.20-97.28]</td><td colspan="1" rowspan="1">AAC:97.8%(136/139)[93.85-99.26]</td><td colspan="1" rowspan="1">AAC:76.1%(35/46)[62.06-86.09]</td><td colspan="1" rowspan="1">aac(3)-lI(a-d): 48.8%(755/1546)[46.35-51.33]aac(3)-IV:3.8%(59/1546)[2.97-4.89]aac(6')-lb/lb-cr:84.3%(1303/1546) [82.38-86.01]</td><td colspan="1" rowspan="1">aac(3)-II(a-d): 87.8%(423/482)[84.53-90.39]aac(3)-IV:99.6%(480/482)[98.50-99.89]ac(6')-lb/lb-cr:87.3%(421/482)[84.08-90.02]</td><td colspan="1" rowspan="1">aac(3)-Il(a-d): 92.8%(755/814)[90.76-94.34]aac(3)-IV:96.7%(59/61)[88.81-99.10]aac(6')-lb/lb-cr:95.5%(1303/1364) [94.30-96.50]</td><td colspan="1" rowspan="1">aac(3)-II(a-d): 34.8%(423/1214)[32.21-37.57]aac(3)-IV:24.4%(480/1967)[22.56-26.35]aac(6')-lb/lb-cr:63.4%(421/664)[59.67-66.98]</td><td colspan="1" rowspan="1">aac(3)-Il(a-d): 40.1%(814/2028)aac(3)-IV: 3%(61/2028)aac(6')-Ib/lb-cr: 67.3%(1364/2028)</td><td colspan="1" rowspan="1">aac(3)-II(a-d): 25.0%of 7979aac(3)-IV:3.2% of7979aac(6')-lb//b-cr:51.1% of7979</td></tr><tr><td colspan="1" rowspan="2">AMR GeneTarget</td><td colspan="1" rowspan="2">Organism</td><td colspan="2" rowspan="1">Detection of AMR GeneTarget by AMR GenePanel versus WholeGenome SequencingCombined Clinical DevicePerformance -Stock and ProspectiveIsolates (from K191288)</td><td colspan="1" rowspan="2">Antimicrobial</td><td colspan="4" rowspan="1">Detection of AMR Gene Target by AMR GenePanel vs AST Non-SusceptibilityCombined Clinical Device Performance - Stock andProspective Isolates (from K191288) includingonly unique isolates (no replicates)</td><td colspan="5" rowspan="1">Detection of AMR Gene Target versus AST Non-Susceptibility inOpGen's SMART Study</td><td colspan="1" rowspan="2">%Organismswith AMRGeneTarget inNCBIPathogenDatabase(2019)</td></tr><tr><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1">PPV[95% CI]</td><td colspan="1" rowspan="1">NPV[95% CI]</td><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1">PPV[95% CI]</td><td colspan="1" rowspan="1">NPV[95% CI]</td><td colspan="1" rowspan="1">% Organismswith AMRGene Target</td></tr><tr><td colspan="1" rowspan="1">AAC</td><td colspan="1" rowspan="1">P. mirabilis</td><td colspan="1" rowspan="1">100.0%(114/114)[96.74-10.00]</td><td colspan="1" rowspan="1">92.7%(38/41)[80.57-97.48]</td><td colspan="1" rowspan="1">Amikacin</td><td colspan="1" rowspan="1">AAC:77.8%(14/18)[54.79-91.00]</td><td colspan="1" rowspan="1">AAC:45.0%(27/60)33.09-57.51]</td><td colspan="1" rowspan="1">AAC:29.8%(14/47)18.65-43.98]</td><td colspan="1" rowspan="1">AAC:87.1%(27/31)[71.15-94.87]</td><td colspan="1" rowspan="1">aac(3)-I(a-: 26.1%(30/115)[18.93-34.79]aac(3)-IV:1.7%(2/115)[0.48-6.12]ac(6')-lb/lb-cr:74.8%(86/115)[66.13-81.83]</td><td colspan="1" rowspan="1">aac(3)-I(a-:82.1%(856/1042)[79.71-84.36]aac(3)-IV:93.9%(978/1042)[92.23-95.16]ac(6')-lb/lb-cr:79.9%(833/1042)[77.40-82.26]</td><td colspan="1" rowspan="1">aac(3)-I(a-): 13.9%30/216)[9.91-19.13]aac(3)-IV:3.0%(2/66)[0.83-10.39]aac(6')-lb/b-cr:29.2%(86/295)[24.26-34.58]</td><td colspan="1" rowspan="1">aac(3)-I(a-):91.0%(856/941)[88.97-92.64]aac(3)-IV:89.6%(978/1091)[87.69-91.31]aac(6')-lb/lb-cr:96.6%(833/862)[95.2-97.65]</td><td colspan="1" rowspan="1">aac(3)-I(a-): 18.7%(216/1157)aac(3)-IV:5.7%(66/1157)aac(6')-Ib/lb-cr: 25.5%(295/1157)</td><td colspan="1" rowspan="1">aac(3)-I(a-:22.2% f 144aac(3)-IV:9.7% of144aac(6')-lb/lb-cr:13.9% of144</td></tr><tr><td colspan="1" rowspan="1">AAC</td><td colspan="1" rowspan="1">P. mirabilis</td><td colspan="1" rowspan="1">100.0%(114/114)[96.74-100.00]</td><td colspan="1" rowspan="1">92.7%(38/41)[80.57-97.48]</td><td colspan="1" rowspan="1">Gentamicin</td><td colspan="1" rowspan="1">AAC:89.1%(41/46)[76.96-95.27]</td><td colspan="1" rowspan="1">AAC:81.2%(26/32)[64.69-91.11]</td><td colspan="1" rowspan="1">AAC:87.2%(41/47)[74.83-94.02]</td><td colspan="1" rowspan="1">AAC:83.9%(26/31)[67.37-92.91]</td><td colspan="1" rowspan="1">aac(3)-I(a-d): 44.5%(208/467)[40.09-4.07]aac(3)-IV:13.1%(61/467)[10.30-16.42]aac(6')-lb/lb-cr:46.0%(215/467)[41.57-50.57]</td><td colspan="1" rowspan="1">aac(3)-Il(a-d): 98.8%(682/690)[97.73-99.41]aac(3)-IV:99.3%(685/690)[98.32-99.69]aac(6')-lb/lb-cr:88.4%(610/690)[85.80-90.58]</td><td colspan="1" rowspan="1">aac(3)-I(a-d): 96.3%(208/216)[92.86-98.11]aac(3)-IV:92.4%(61/66)[83.46-96.72]aac(6')-lb/lb-cr:72.9%(215/295)[67.54-77.64]</td><td colspan="1" rowspan="1">aac(3)-Il(a-d): 72.5%(682/941)[69.54-75.23]aac(3)-IV:62.8%(685/1091)[59.88-665.61]aac(6')-lb//b-cr:70.8%(610/862)[67.64-73.70]</td><td colspan="1" rowspan="1">aac(3)-Il(a-d): 18.7%(216/1157)aac(3)-IV:5.7%(66/1157)aac(6')-Ib/lb-cr: 25.5%(295/1157)</td><td colspan="1" rowspan="1">aac(3)-II(a-d): 22.2%of 144aac(3)-IV:9.7%of144aac(6')-lb/lb-cr:13.9% of144</td></tr><tr><td colspan="1" rowspan="2">AMR GeneTarget</td><td colspan="1" rowspan="2">Organism</td><td colspan="2" rowspan="1">Detection of AMR GeneTarget by AMR GenePanel versus WholeGenome SequencingCombined Clinical DevicePerformance-Stock and ProspectiveIsolates (from K191288)</td><td colspan="1" rowspan="2">Antimicrobial</td><td colspan="4" rowspan="1">Detection of AMR Gene Target by AMR GenePanel vs AST Non-SusceptibilityCombined Clinical Device Performance - Stock andivaonly unique isolates (no replicates)</td><td colspan="5" rowspan="1">Detection of AMR Gene Target versus AST Non-Susceptibility inOpGen's SMART Study</td><td colspan="1" rowspan="2">%Organismswith AMRGeneTarget inNCBIPathogenDatabase(2019)</td></tr><tr><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1">PPV[95% CI]</td><td colspan="1" rowspan="1">NPV[95% CI]</td><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1">PPV[95% CI]</td><td colspan="1" rowspan="1">NPV[95% CI]</td><td colspan="1" rowspan="1">% Organismswith AMRGene Target</td></tr><tr><td colspan="1" rowspan="1">AAC</td><td colspan="1" rowspan="1">P. mirabilis</td><td colspan="1" rowspan="1">100.0%(114/114)[96.74-100.00]</td><td colspan="1" rowspan="1">92.7%(38/41)[80.57-97.48]</td><td colspan="1" rowspan="1">Tobramycin</td><td colspan="1" rowspan="1">AAC:89.8%(44/49)[78.24-95.56]</td><td colspan="1" rowspan="1">AAC:89.7%(26/29)[73.61-996.42]</td><td colspan="1" rowspan="1">AAC:93.6%(44/47)[82.84-97.81]</td><td colspan="1" rowspan="1">AAC:83.9%(26/31)[67.37-92.91]</td><td colspan="1" rowspan="1">aac(3)-Il(a-d): 33.1%(147/444)[28.89-37.61]aac(3)-IV:14.2%(63/444)[11.25-17.74]aac(6')-lb/lb-cr:61.0%(271/444)[56.42-65.46]</td><td colspan="1" rowspan="1">aac(3)-II(a-d): 90.3%(644/713)[87.93-92.28]aac(3)-IV:99.6%(710/713)[98.77-99.86]aac(6')-lb/lb-cr:96.6%(689/713)[95.04-97.73]</td><td colspan="1" rowspan="1">aac(3)-II(a-d): 68.1%(147/216)[61.57-73.91]aac(3)-IV:95.5%(63/66)[87.47-98.44aac(6')-lb/lb-cr:91.9%(271/295)[88.18-94.47]</td><td colspan="1" rowspan="1">aac(3)-II(a-d): 68.4%(644/941)[65.40-71.33]aac(3)-IV:65.1%(710/1091)[62.20-67.85]aac(6')-lb/lb-cr:79.9%(689/862)[77.13-82.47]</td><td colspan="1" rowspan="1">aac(3)-II(a-d): 18.7%(216/1157)aac(3)-IV:5.7%(66/1157)aac(6')-Ib/lb-cr: 25.5%(295/1157)</td><td colspan="1" rowspan="1">aac(3)-II(a-): 22.2%of 144aac(3)-IV:9.7% of144aac(6')-lb/lb-cr:13.9% of144</td></tr><tr><td colspan="1" rowspan="1">AAC</td><td colspan="1" rowspan="1">P. aeruginosa</td><td colspan="1" rowspan="1">94.9%(148/156)[90.21-97.38]</td><td colspan="1" rowspan="1">100.0%(192/192)[98.04-100.00]</td><td colspan="1" rowspan="1">Amikacin</td><td colspan="1" rowspan="1">AAC:46.0%(46/100)[36.56-55.74]</td><td colspan="1" rowspan="1">AAC:83.6%(46/55)[71.74-91.14]</td><td colspan="1" rowspan="1">AAC:83.6%(46/55)[71.74-91.14]</td><td colspan="1" rowspan="1">AAC:46.0%(46/100)[36.56-5.74]</td><td colspan="1" rowspan="1">aac(3)-Il(a-d): 2.2%(13/600)[1.27-3.67]aac(3)-IV:0.2%(1/600)[0.03-0.94]aac(6')-lb/b-cr:33.7%(202/600)[30.00-37.54]</td><td colspan="1" rowspan="1">aac(3)-I(a-d): 98.7%(938/950)[97.81-99.28]aac(3)-IV:100.0%(950/950)[99.60-100.00]aac(6')-lb/lb-cr:87.2%(828/950)[84.88-89.14]</td><td colspan="1" rowspan="1">aac(3)-I(a-d): 52.0%(13/25)[33.50-69.97]aac(3)-IV:100.0%(1/1)[20.65-100.00]aac(6')-lb/lb-cr:62.3%(202/324)[56.95-67.45]</td><td colspan="1" rowspan="1">aac(3)-I(a-d): 61.5%(938/1525)[59.04-63.92]aac(3)-IV:61.3%(950/1549)[58.88-63.72]aac(6')-lb/lb-cr:67.5%(828/1226)[64.86-70.10]</td><td colspan="1" rowspan="1">aac(3)-Il(a-d): 1.6%(25/1550)aac(3)-IV:0.1%(1/1550)aac(6')-Ib/lb-cr: 20.9%(324/1550)</td><td colspan="1" rowspan="1">aac(3)-lIl(a-d): 0.3% of4706aac(3)-IV:0% of706aac(6')-lb/lb-cr:8.2% of4706</td></tr><tr><td colspan="1" rowspan="2">AMR GeneTarget</td><td colspan="1" rowspan="2">Organism</td><td colspan="2" rowspan="1">Detection of AMR GeneTarget by AMR GenePanel versus WholeGenome SequencingCombined Clinical DevicePerformance-Stock and ProspectiveIsolates (from K191288)</td><td colspan="1" rowspan="2">Antimicrobial</td><td colspan="4" rowspan="1">Detection of AMR Gene Target by AMR GenePanel vs AST Non-SusceptibilityCombined Clinical Device Performance - Stock andProspective Isolates (from K191288) includingonly unique isolates (no replicates)</td><td colspan="5" rowspan="1">Detection of AMR Gene Target versus AST Non-Susceptibility inOpGen's SMART Study</td><td colspan="1" rowspan="2">%Organismswith AMRGeneTarget inNCBIPathogenDatabase(2019)</td></tr><tr><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1">PPV[95% CI]</td><td colspan="1" rowspan="1">NPV[95% CI]</td><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1">PPV[95% CI]</td><td colspan="1" rowspan="1">NPV[95% CI]</td><td colspan="1" rowspan="1">% Organismswith AMRGene Target</td></tr><tr><td colspan="1" rowspan="1">AAC</td><td colspan="1" rowspan="1">P. aeruginosa</td><td colspan="1" rowspan="1">94.9%(148/156)[90.21-97.38]</td><td colspan="1" rowspan="1">100.0%(192/192)[98.04-10.00]</td><td colspan="1" rowspan="1">Gentamicin</td><td colspan="1" rowspan="1">AAC:46.2%(54/117)[37.39-5.17]</td><td colspan="1" rowspan="1">AAC:97.4%(37/38)[86.51-99.53]</td><td colspan="1" rowspan="1">AAC:98.2%(54/55)[90.39-99.68]</td><td colspan="1" rowspan="1">AAC:37.0%(37/100)[28.18-46.78]</td><td colspan="1" rowspan="1">aac(3)-I(a-d): 2.6%(24/932)[1.74-3.80]aac(3)-IV:0.1%(1/932)[0.02-0.61]aac(6')-lb/b-cr:33.3%(310/932)[30.31-36.35]</td><td colspan="1" rowspan="1">aac(3)-I(a-d): 99.8%(617/618)[99.09-9.97]aac(3)-IV:10.0%(618/618)[99.38-0.00]aac(6')-lb/lb-cr:97.7%(604/618)[96.23-98.65]</td><td colspan="1" rowspan="1">aac(3)-I(a-d): 96.0%(24/25)[80.46-9.29]aac(3)-IV:100.0%(1/1)[20.65-1000.00]ac(6')-lb/lb-cr:95.7%(310/324)[92.88-97.41]</td><td colspan="1" rowspan="1">aac(3)-I(a-d): 40.5%(617/1525)[38.02-42.94]aac(3)-IV:39.9%(618/1549)[37.49-42.36]ac(6')-lb/lb-cr:49.3%(604/1226)[46.47-52.06]</td><td colspan="1" rowspan="1">aac(3)-I(a-d): 1.6%(25/1550)aac(3)-IV:0.1%(1/1550)aac(6')-Ib/lb-cr: 0.9%(324/1550)</td><td colspan="1" rowspan="1">aac(3)-lI(a-d): 0.3% of4706aac(3)-IV:0% of4706aac(6')-lb/lb-cr:8.2% of4706</td></tr><tr><td colspan="1" rowspan="1">AAC</td><td colspan="1" rowspan="1">P. aeruginosa</td><td colspan="1" rowspan="1">94.9%(148/156)[90.21-97.38]</td><td colspan="1" rowspan="1">100.0%(192/192)[98.04-0.00]</td><td colspan="1" rowspan="1">Tobramycin</td><td colspan="1" rowspan="1">AAC:48.7%(55/113)[39.65-57.78]</td><td colspan="1" rowspan="1">AAC:100.0%(42/42)[91.62-100.00]</td><td colspan="1" rowspan="1">AAC:100.0%(55/55)[93.47-100.00]</td><td colspan="1" rowspan="1">AAC:42.0%(42/100)[32.80-51.79]</td><td colspan="1" rowspan="1">aac(3)-l(a-d): 3.0%(24/800)[2.02-4.43]aac(3)-IV:0.1%(1/800)[0.02-0.70]aac(6')-lb/b-cr:38.6%(309/800)[35.31-42.05]</td><td colspan="1" rowspan="1">aac(3)-l(a-d): 99.9%(749/750)[99.25-99.98]aac(3)-IV:100.0%(750/750)[99.49-100.00]aac(6')-lb/lb-cr:98.0%(735/750)[96.73-98.78]</td><td colspan="1" rowspan="1">aac(3)-I(a-d): 96.0%(24/25)[80.46-99.29]aac(3)-IV:100.0%(1/1)[20.65-100.00]aac(6')-lb/lb-cr:95.4%(309/324)[92.50-97.17]</td><td colspan="1" rowspan="1">aac(3)-l(a-d): 49.1%(749/1525)[46.61-51.62]aac(3)-IV:48.4%(750/1549)[45.94-50.91]aac(6')-lb/lb-cr:60.0%(735/1226)[57.18-62.66]</td><td colspan="1" rowspan="1">aac(3)-II(a-d): 1.6%(25/1550)aac(3)-IV:0.1%(1/1550)aac(6')-Ib/lb-cr: 20.9%(324/1550)</td><td colspan="1" rowspan="1">aac(3)-II(a-): 0.3% of4706aac(3)-IV:0% of706aac(6')-lb/lb-cr:8.2% of4706</td></tr><tr><td colspan="1" rowspan="1">AAD</td><td colspan="1" rowspan="1">K. pneumoniae</td><td colspan="1" rowspan="1">98.5%(128/130)[94.56-99.58]</td><td colspan="1" rowspan="1">96.5%(192/199)[92.92-98.29]</td><td colspan="1" rowspan="1">Amikacin</td><td colspan="1" rowspan="1">AAD:71.1%(64/90)[61.04-79.46]</td><td colspan="1" rowspan="1">AAD:61.1%(58/95)[51.00-70.25]</td><td colspan="1" rowspan="1">AAD:63.4%(64/101)[53.64-72.11]</td><td colspan="1" rowspan="1">AAD:69.0%(58/84)[58.51-77.92]</td><td colspan="1" rowspan="1">aadA3/A8:77.9%(486/624)[74.46-80.96]</td><td colspan="1" rowspan="1">aadA3/A8:63.1%(886/1404)[60.55-65.59]</td><td colspan="1" rowspan="1">aadA3/A8:48.4%(486/1004)[45.33-51.50]</td><td colspan="1" rowspan="1">aadA3/A8:86.5%(886/1024)[84.29-88.48]</td><td colspan="1" rowspan="1">aadA3/A8:49.5%(1004/2028)</td><td colspan="1" rowspan="1">aadA3/A8:53.3% of7979</td></tr><tr><td colspan="1" rowspan="2">AMR GeneTarget</td><td colspan="1" rowspan="2">Organism</td><td colspan="2" rowspan="1">Detection of AMR GeneTarget by AMR GenePanel versus WholeGenome SequencingCombined Clinical DevicePerformance-Stock and ProspectiveIsolates (from K191288)</td><td colspan="1" rowspan="2">Antimicrobial</td><td colspan="4" rowspan="1">Detection of AMR Gene Target by AMR GenePanel vs AST Non-SusceptibilityCombined Clinical Device Performance - Stock andProspective Isolates (rom K191288)includingonly unique isolates (no replicates)</td><td colspan="5" rowspan="1">Detection of AMR Gene Target versus AST Non-Susceptibility inOpGen's SMART Study</td><td colspan="1" rowspan="2">%Organismswith AMRGeneTarget inNCBIPathogenDatabase(2019)</td></tr><tr><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1">PPV[95% CI]</td><td colspan="1" rowspan="1">NPV[95% CI]</td><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1">PPV[95% CI]</td><td colspan="1" rowspan="1">NPV[95% CI]</td><td colspan="1" rowspan="1">% Organismswith AMRGene Target</td></tr><tr><td colspan="1" rowspan="1">AAD</td><td colspan="1" rowspan="1">K. pneumoniae</td><td colspan="1" rowspan="1">98.5%(128/130)[94.56-99.58]</td><td colspan="1" rowspan="1">96.5%(192/199)[92.92-8.29]</td><td colspan="1" rowspan="1">Gentamicin</td><td colspan="1" rowspan="1">AAD:63.7%(86/135)55.32-71.33]</td><td colspan="1" rowspan="1">AAD:70.0%(35/50)56.25580.90]</td><td colspan="1" rowspan="1">AAD:85.1%(86/101)[76.93-90.79]</td><td colspan="1" rowspan="1">AAD:41.7%(35/84)[31.71-52.35]</td><td colspan="1" rowspan="1">aadA3/A8:54.5%(641/1177)[51.61-57.29]</td><td colspan="1" rowspan="1">aadA3/A8:57.3%(488/851)[54.00-60.63]</td><td colspan="1" rowspan="1">aadA3/A8:63.8%(641/1004)[60.83-66.76]</td><td colspan="1" rowspan="1">aadA3/A8:47.7%(488/1024)[44.61-50.72]</td><td colspan="1" rowspan="1">aadA3/A8:49.5%(1004/2028)</td><td colspan="1" rowspan="1">aadA3/A8:5.3% of7979</td></tr><tr><td colspan="1" rowspan="1">AAD</td><td colspan="1" rowspan="1">K. pneumoniae</td><td colspan="1" rowspan="1">98.5%(128/130)[94.56-9.58]</td><td colspan="1" rowspan="1">96.5%(192/199)[92.92-98.29]</td><td colspan="1" rowspan="1">Tobramycin</td><td colspan="1" rowspan="1">AAD:61.2%(90/147)[53.16-68.72]</td><td colspan="1" rowspan="1">AAD:71.1%(27/38)[55.24-83.00]</td><td colspan="1" rowspan="1">AAD:89.1%(90/101)[81.54-93.81]</td><td colspan="1" rowspan="1">AAD:32.1%(27/84)[23.13-42.721</td><td colspan="1" rowspan="1">aadA3/A8:58.1%(898/1546)[55.61-60.2</td><td colspan="1" rowspan="1">aadA3/A8:78.0%(376/482)[74.10-81.48]</td><td colspan="1" rowspan="1">aadA3/A8:89.4%(898/1004)[87.39-91.20]</td><td colspan="1" rowspan="1">aadA3/A8:36.7%(376/1024)[33.82-39.72]</td><td colspan="1" rowspan="1">aadA3/A8:49.5%(1004/2028)</td><td colspan="1" rowspan="1">aadA3/A8:5.3% of7979</td></tr><tr><td colspan="1" rowspan="1">ANT</td><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">98.4%(62/63)[91.54-99.72]</td><td colspan="1" rowspan="1">100.0%(272/272)[98.61-1000]</td><td colspan="1" rowspan="1">Amikacin</td><td colspan="1" rowspan="1">ANT:10.8%(4/37)4.2924.71]</td><td colspan="1" rowspan="1">ANT:95.9%(117/122)[90.76-98.24]</td><td colspan="1" rowspan="1">ANT:44.4%(4/9)18.88-73.33]</td><td colspan="1" rowspan="1">ANT:78.0%(117/150)[70.72-83.88]</td><td colspan="1" rowspan="1">ant(2")-la:2.3%(5/213)[1.01-5.38]</td><td colspan="1" rowspan="1">ant(2")-la:98.7%(2710/2747))98.1599.02]</td><td colspan="1" rowspan="1">ant(2")-la:11.9%(5/42)5.19- 25.00]</td><td colspan="1" rowspan="1">ant(2")-la:92..9%(2710/2918) [91.88-93.75]</td><td colspan="1" rowspan="1">ant(2")-la:1.4%(42/2960)</td><td colspan="1" rowspan="1">ant(2")-la:0.6% of19017</td></tr><tr><td colspan="1" rowspan="1">ANT</td><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">98.4%(62/63)[91.54-99.72]</td><td colspan="1" rowspan="1">100.0%(272/272)[98.61-100.00]</td><td colspan="1" rowspan="1">Gentamicin</td><td colspan="1" rowspan="1">ANT:9.5%(8/84)[4.91-17.68]</td><td colspan="1" rowspan="1">ANT:98.7%(74/75)[92.83-99.76]</td><td colspan="1" rowspan="1">ANT:88.9%(8//9[56.50-98.01]</td><td colspan="1" rowspan="1">ANT:49.3%/150)[41.45-57.25]</td><td colspan="1" rowspan="1">ant(2")-la:3.6%((3/995)[2.62-4.97]</td><td colspan="1" rowspan="1">ant(2")-la:99.7%(1959/1965) (99.4-99.86]</td><td colspan="1" rowspan="1">ant(2")-la:85.7%(36/42)[72.16-93.28]</td><td colspan="1" rowspan="1">ant(2")-la:67.1%959/2918) [65.41-68.82</td><td colspan="1" rowspan="1">ant(2")-la:1.4%(42/2960)</td><td colspan="1" rowspan="1">ant(2")-la:0.6% of19017</td></tr><tr><td colspan="1" rowspan="1">ANT</td><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">98.4%(62/63)[91.54-99.72]</td><td colspan="1" rowspan="1">100.0%(272/272)[98.61-1000]</td><td colspan="1" rowspan="1">Tobramycin</td><td colspan="1" rowspan="1">ANT:10.0%(9//90)[5.35-17.92]</td><td colspan="1" rowspan="1">ANT:10.0%69/69)[94.73-0.00]</td><td colspan="1" rowspan="1">ANT:100.0%(9/9)[70.09-100.00]</td><td colspan="1" rowspan="1">ANT:46.0%(/150)38.22-53.98]</td><td colspan="1" rowspan="1">ant(2")-la:2.8%(38/1362)[2.04-3.81]</td><td colspan="1" rowspan="1">ant(2")-la:99.7%(1594/1598) [99.36-99.901</td><td colspan="1" rowspan="1">ant(2")-la:90.5%(38/42)[77.93-96.23]</td><td colspan="1" rowspan="1">ant(2")-la:54.6%(1594/2918) [52.82-56.43]</td><td colspan="1" rowspan="1">ant(2")-la:1.4%(42/2960)</td><td colspan="1" rowspan="1">ant(2")-la:0.6% of19017</td></tr><tr><td colspan="1" rowspan="1">ANT</td><td colspan="1" rowspan="1">P. mirabilis</td><td colspan="1" rowspan="1">100.0%(78/78)[95.31-100.00]</td><td colspan="1" rowspan="1">98.7%(76/77)[93.00-99.77]</td><td colspan="1" rowspan="1">Amikacin</td><td colspan="1" rowspan="1">ANT:61.1%(11/18)[38.62-79.69]</td><td colspan="1" rowspan="1">ANT:90.0%(54/60)79.85-95.34]</td><td colspan="1" rowspan="1">ANT:64.7%(11/17)41.30-82.69]</td><td colspan="1" rowspan="1">ANT:88.5%(54/61)[78.16-94.33]</td><td colspan="1" rowspan="1">ant(2")-Ila:18.3%(21/115)[12.27-26.31]</td><td colspan="1" rowspan="1">ant(2")-la:91.9%(958/1042)90.13-93.44]</td><td colspan="1" rowspan="1">ant(2")-la:20.0%(21/105)[13.47-28.65]</td><td colspan="1" rowspan="1">ant(2")-la:91.1%(958/1052)[89.19-92.64]</td><td colspan="1" rowspan="1">ant(2")-la:9.1%(105/1157)</td><td colspan="1" rowspan="1">ant(2")-Ila:5.6% of144</td></tr><tr><td colspan="1" rowspan="1">ANT</td><td colspan="1" rowspan="1">P. mirabilis</td><td colspan="1" rowspan="1">100.0%(78/78)[95.31-100.00]</td><td colspan="1" rowspan="1">98.7%(76/77)[93.00-99.77]</td><td colspan="1" rowspan="1">Gentamicin</td><td colspan="1" rowspan="1">ANT:32.6%(15/46)20.87-47.03]</td><td colspan="1" rowspan="1">ANT:93.8%(30/32)[79.85-98.27]</td><td colspan="1" rowspan="1">ANT:88.2%(15/17)[65.66-96.71]</td><td colspan="1" rowspan="1">ANT:49.2%(30/61)[37.06-61.40]</td><td colspan="1" rowspan="1">ant(2")-la:19.9% (3/467)[16.54-23.78]</td><td colspan="1" rowspan="1">ant(2")-la:98.3%(678/690)[96.98-99.00]</td><td colspan="1" rowspan="1">ant(2")-la:88.6%(93/105)[81.08-93.34]</td><td colspan="1" rowspan="1">ant(2")-la:64.4%(678/1052)[61.51-67.28]</td><td colspan="1" rowspan="1">ant(2")-la:9.1%(105/1157)</td><td colspan="1" rowspan="1">ant(2")-la:.6% of144</td></tr><tr><td colspan="1" rowspan="2">AMR GeneTarget</td><td colspan="1" rowspan="2">Organism</td><td colspan="2" rowspan="1">Detection of AMR GeneTarget by AMR GenePanel versus WholeGenome SequencingCombined Clinical DevicePerformance -Stock and ProspectiveIsolates (from K191288)</td><td colspan="1" rowspan="2">Antimicrobial</td><td colspan="4" rowspan="1">Detection of AMR Gene Target by AMR GenePanel vs AST Non-SusceptibilityCombined Clinical Device Performance - Stock andProspective Isolates (rom K191288)includingonly unique isolates (no replicates)</td><td colspan="5" rowspan="1">Detection of AMR Gene Target versus AST Non-Susceptibility inOpGen's SMART Study</td><td colspan="1" rowspan="2">%Organismswith AMRGeneTarget inNCBIPathogenDatabase(2019)</td></tr><tr><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1">PPV[95% CI]</td><td colspan="1" rowspan="1">NPV[95% CI]</td><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1">PPV[95% CI]</td><td colspan="1" rowspan="1">NPV[95% CI]</td><td colspan="1" rowspan="1">% Organismswith AMRGene Target</td></tr><tr><td colspan="1" rowspan="1">ANT</td><td colspan="1" rowspan="1">P. mirabilis</td><td colspan="1" rowspan="1">100.0%(78/78)[95.31-1000]</td><td colspan="1" rowspan="1">98.7%(76/77)[93.00-9.77]</td><td colspan="1" rowspan="1">Tobramycin</td><td colspan="1" rowspan="1">ANT:32.7%(16/49)21.21-46.62]</td><td colspan="1" rowspan="1">ANT:966.6%(28/29)[82.82-99.39]</td><td colspan="1" rowspan="1">ANT:94.%(16/17)[73.02-98.95]</td><td colspan="1" rowspan="1">ANT:45.9%(28/61)[34.01-58.28]</td><td colspan="1" rowspan="1">ant(2")-la:20.3%(90/444)16.79-24.26]</td><td colspan="1" rowspan="1">ant(2")-la:97.9%(698/713)966.56-98.72]</td><td colspan="1" rowspan="1">ant(2")-la:85.7%(90/105)[77.76-91.15]</td><td colspan="1" rowspan="1">ant(2")-la:66.3%(698/1052)[63.44-69.14]</td><td colspan="1" rowspan="1">ant(2")-la:9.1%(105/1157)</td><td colspan="1" rowspan="1">ant(2")-la:56% of144</td></tr><tr><td colspan="1" rowspan="1">ANT</td><td colspan="1" rowspan="1">P. aeruginosa</td><td colspan="1" rowspan="1">98.4%(63/64)91.67-99.72]</td><td colspan="1" rowspan="1">98.6%(280/284)[96.44-99.45]</td><td colspan="1" rowspan="1">Amikacin</td><td colspan="1" rowspan="1">ANT:36.0%(36/100)[27.27-45.76]</td><td colspan="1" rowspan="1">ANT:94.5%(52/55)[85.15-98.13]</td><td colspan="1" rowspan="1">ANT:92.3%(36/39)[79.68-97.35]</td><td colspan="1" rowspan="1">ANT:44.8%52/116)[36.09-53.901]</td><td colspan="1" rowspan="1">ant(2")-la:22.2%(133/600)[19.03-25.6]</td><td colspan="1" rowspan="1">ant(2")-la:92.1%(875/950)[90.22-93.6]</td><td colspan="1" rowspan="1">ant(2")-la:63.9%(133/208)[57.22-70.16]</td><td colspan="1" rowspan="1">ant(2")-la:65.2%(875/1342)[62.61-67.70]</td><td colspan="1" rowspan="1">ant(2")-la:13.4%(208/1550)</td><td colspan="1" rowspan="1">ant(2")-la:.6% of4706</td></tr><tr><td colspan="1" rowspan="1">ANT</td><td colspan="1" rowspan="1">P. aeruginosa</td><td colspan="1" rowspan="1">98.4%(63/64)91.67-9.72]</td><td colspan="1" rowspan="1">98.6%(280/284)[96.44-9.45]</td><td colspan="1" rowspan="1">Gentamicin</td><td colspan="1" rowspan="1">ANT:33.3%(39/117)[25.44-42.28]</td><td colspan="1" rowspan="1">ANT:100.0%(38/38)90.820.00]</td><td colspan="1" rowspan="1">ANT:100.0%(39/39)91.0310.00]</td><td colspan="1" rowspan="1">ANT:32.8%(38/116)4.89-41.73]</td><td colspan="1" rowspan="1">ant(2")-la:21.8%(203/932)[19.25-24.54]</td><td colspan="1" rowspan="1">ant(2")-la:99.2%(613/618)[98.12-99.65]</td><td colspan="1" rowspan="1">ant(2")-la:97.6%(203/208)[94.50-98.97]</td><td colspan="1" rowspan="1">ant(2")-la:45.%%(613/1342)[43.03-48.35]</td><td colspan="1" rowspan="1">ant(2")-la:13.4%(208/1550)</td><td colspan="1" rowspan="1">ant(2")-la:56% of470</td></tr><tr><td colspan="1" rowspan="1">ANT</td><td colspan="1" rowspan="1">P. aeruginosa</td><td colspan="1" rowspan="1">98.4%(63/64)91.67-99.72]</td><td colspan="1" rowspan="1">98.6%(280/284)[96.44-99.45]</td><td colspan="1" rowspan="1">Tobramycin</td><td colspan="1" rowspan="1">ANT:33.6%(38/113)25.58-42.75]</td><td colspan="1" rowspan="1">ANT:97.6%(41/42)[87.68-99.58]</td><td colspan="1" rowspan="1">ANT:97.4%(38/39)86.82-99.55]</td><td colspan="1" rowspan="1">ANT:35.3%(41/116)[27.24-44.39]</td><td colspan="1" rowspan="1">ant(2")-la:25.0%(200/800)[22.12-28.12]</td><td colspan="1" rowspan="1">ant(2")-la:98.9%(742/750)[97.91-99.46]</td><td colspan="1" rowspan="1">ant(2")-Ila:96.2%(0/208)[92.60-98.04]</td><td colspan="1" rowspan="1">ant(2")-la:55.3%(742/1342)[52.62-57.93]</td><td colspan="1" rowspan="1">ant(2")-la:13.4%(208/1550)</td><td colspan="1" rowspan="1">ant(2")-Ila:5.6% of4706</td></tr><tr><td colspan="1" rowspan="1">APH</td><td colspan="1" rowspan="1">K. pneumoniae</td><td colspan="1" rowspan="1">96.4%(27/28)[82.29-99.37]</td><td colspan="1" rowspan="1">99.7%(300/301)[98.14-99.94]</td><td colspan="1" rowspan="1">Amikacin</td><td colspan="1" rowspan="1">APH:11.1%(10/90)6.15-19.26]</td><td colspan="1" rowspan="1">APH:89.5%(85/95)[81.70-94.18]</td><td colspan="1" rowspan="1">APH:50.0%(10/20)[29.93-70.07]</td><td colspan="1" rowspan="1">APH:51.5%(85/165)43.94-59.02]</td><td colspan="1" rowspan="1">aph(4)-la:4.6%(29/624)[3.26-6.59]</td><td colspan="1" rowspan="1">aph(4)-la:97.5%(1369/140) [96.55-98.201</td><td colspan="1" rowspan="1">aph(4)-la:45.3%(29/64)[33.73-57.43]</td><td colspan="1" rowspan="1">aph(4)-la:69.7%(1369/196) [67.64-71.70]</td><td colspan="1" rowspan="1">aph(4)-la:3.2%(64/2028)</td><td colspan="1" rowspan="1">aph(4)-la:3.2% of7979</td></tr><tr><td colspan="1" rowspan="1">APH</td><td colspan="1" rowspan="1">K. pneumoniae</td><td colspan="1" rowspan="1">96.4%(27/28)[82.29-99.37]</td><td colspan="1" rowspan="1">99.7%(300/301)98.14-99.94]</td><td colspan="1" rowspan="1">Gentamicin</td><td colspan="1" rowspan="1">APH:14.1%(19/135)[9.20-20.94]</td><td colspan="1" rowspan="1">APH:98.0%(49/50)[89.50-99.65]</td><td colspan="1" rowspan="1">APH:95.0%(19/20)76.39-99.11]</td><td colspan="1" rowspan="1">APH:29.7%(49/165)[23.25-37.07]</td><td colspan="1" rowspan="1">aph(4)-la:4.8%(57/1177)[3.76-6.22]</td><td colspan="1" rowspan="1">aph(4)-la:99.2%(844/851)[98.31-99.60]</td><td colspan="1" rowspan="1">aph(4)-la:89.1%57/64)[79.10-94.60]</td><td colspan="1" rowspan="1">aph(4)-la:43.0%(844/1964)[40.80-45.17]</td><td colspan="1" rowspan="1">aph(4)-la:3.2%(64/2028)</td><td colspan="1" rowspan="1">aph(4)-la:3.2% of7979</td></tr><tr><td colspan="1" rowspan="1">APH</td><td colspan="1" rowspan="1">K. pneumoniae</td><td colspan="1" rowspan="1">96.4%(27/28)[82.29-99.37]</td><td colspan="1" rowspan="1">99.7%(300/301)[98.14-99.94]</td><td colspan="1" rowspan="1">Tobramycin</td><td colspan="1" rowspan="1">APH:13.6%(20/147)[8.98-20.08]</td><td colspan="1" rowspan="1">APH:100.0%(38/38)90.82-100.00]</td><td colspan="1" rowspan="1">APH:100.0%(20/20)[83.89-100.00]</td><td colspan="1" rowspan="1">APH:23.0%(38/165)[17.26-30.02]</td><td colspan="1" rowspan="1">aph(4)-la:3.9%(61/1546)[3.08-5.04]</td><td colspan="1" rowspan="1">aph(4)-la:99.4%(479/482)98.19-99.79]</td><td colspan="1" rowspan="1">aph(4)-la:95.3%(61/64)[87.10-98.39]</td><td colspan="1" rowspan="1">aph(4)-la:24.4%(479/1964)22.54-26.34]</td><td colspan="1" rowspan="1">aph(4)-la:3.2%(64/2028)</td><td colspan="1" rowspan="1">aph(4)-la:3.% of7979</td></tr><tr><td colspan="1" rowspan="2">AMR GeneTarget</td><td colspan="1" rowspan="2">Organism</td><td colspan="2" rowspan="1">Detection of AMR GeneTarget by AMR GenePanel versus WholeGenome SequencingCombined Clinical DevicePerformance-Stock and ProspectiveIsolates (from K191288)</td><td colspan="1" rowspan="2">Antimicrobial</td><td colspan="4" rowspan="1">Detection of AMR Gene Target by AMR GenePanel vs AST Non-SusceptibilityCombined Clinical Device Performance - Stock andProspective Isolates (from K191288) includingonly unique isolates (no replicates)</td><td colspan="5" rowspan="1">Detection of AMR Gene Target versus AST Non-Susceptibility inOpGen's SMART Study</td><td colspan="1" rowspan="2">%Organismswith AMRGeneTarget inNCBIPathogenDatabase(2019)</td></tr><tr><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1">PPV[95% CI]</td><td colspan="1" rowspan="1">NPV[95% CI]</td><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1">PPV[95% CI]</td><td colspan="1" rowspan="1">NPV[95% CI]</td><td colspan="1" rowspan="1">% Organismswith AMRGene Target</td></tr><tr><td colspan="1" rowspan="1">APH</td><td colspan="1" rowspan="1">P. mirabilis</td><td colspan="1" rowspan="1">100.0%(12/12)[75.75-100.00]</td><td colspan="1" rowspan="1">100.0%(143/143)[97.38-100.00]</td><td colspan="1" rowspan="1">Amikacin</td><td colspan="1" rowspan="1">APH:0.0%(0/18)[0.00-17.59]</td><td colspan="1" rowspan="1">APH:80.0%(48/60)[68.22-88.17]</td><td colspan="1" rowspan="1">APH:0.0%(0/12)[0.00-24.25]</td><td colspan="1" rowspan="1">APH:72.7%(48/66)60.9682.00]</td><td colspan="1" rowspan="1">aph(4)-la:0..9%(1/115)[0.15-4.76]</td><td colspan="1" rowspan="1">aph(4)-la:93.7%(976/1042)[92.02-94.99]</td><td colspan="1" rowspan="1">aph(4)-la:1.5%(1/67)[0.26-7.98]</td><td colspan="1" rowspan="1">aph(4)-la:899.5%(976/1090).5891.22]</td><td colspan="1" rowspan="1">aph(4)-la:55.8%(67/1157)</td><td colspan="1" rowspan="1">aph(4)-la:0% f 144</td></tr><tr><td colspan="1" rowspan="1">APH</td><td colspan="1" rowspan="1">P. mirabilis</td><td colspan="1" rowspan="1">100.0%(12/12)75.75-100.00]</td><td colspan="1" rowspan="1">100.0%(143/143)[97.38-100.00]</td><td colspan="1" rowspan="1">Gentamicin</td><td colspan="1" rowspan="1">APH:26.1%(12/46)15.60-40.26]</td><td colspan="1" rowspan="1">APH:100.0%(32/32)[89.28-100.00]</td><td colspan="1" rowspan="1">APH:100.0%(12/12)75.75-100.00]</td><td colspan="1" rowspan="1">APH:48.5%(32/66)36.85-60.29]</td><td colspan="1" rowspan="1">aph(4)-la:12.8%0/467)[10.11-16.19]</td><td colspan="1" rowspan="1">aph(4)-la:999.0%(683/690)[97.92-99.51]</td><td colspan="1" rowspan="1">aph(4)-la:89.6%60/67)[79.97-94.85]</td><td colspan="1" rowspan="1">aph(4)-la:62.7%(683/1090)59.75-65.48]</td><td colspan="1" rowspan="1">aph(4)-la:5.8%(67/1157)</td><td colspan="1" rowspan="1">aph(4)-la:0% of 144</td></tr><tr><td colspan="1" rowspan="1">APH</td><td colspan="1" rowspan="1">P. mirabilis</td><td colspan="1" rowspan="1">100.0%(12/12)[75.75-10.00]</td><td colspan="1" rowspan="1">100.0%(143/143)[97.38-10.00]</td><td colspan="1" rowspan="1">Tobramycin</td><td colspan="1" rowspan="1">APH:22.4%(11/49)13.02-35.88]</td><td colspan="1" rowspan="1">APH:96.6%(28/29)[82.82-99.39]</td><td colspan="1" rowspan="1">APH:91.7%(11/12)64.61-98.51]</td><td colspan="1" rowspan="1">APH:42.4%(28/66)[31.24-54.44]</td><td colspan="1" rowspan="1">aph(4)-la:14.4%(64/444)11.45-17.99]</td><td colspan="1" rowspan="1">aph(4)-la:99.6%(710/713)[98.77-99.86]</td><td colspan="1" rowspan="1">aph(4)-la:95.5%(64/67)[87.64-98.47]</td><td colspan="1" rowspan="1">aph(4)-la:65.1%(710/1090)[62.26-67.91]</td><td colspan="1" rowspan="1">aph(4)-la:5.8%(67/1157)</td><td colspan="1" rowspan="1">aph(4)-la:0%of 144</td></tr><tr><td colspan="1" rowspan="1">armA</td><td colspan="1" rowspan="1">P. mirabilis</td><td colspan="1" rowspan="1">100.0%(8/8)[67.56-100.00]</td><td colspan="1" rowspan="1">100.0%(147/147)[97.45-100.00]</td><td colspan="1" rowspan="1">Amikacin</td><td colspan="1" rowspan="1">armA:22.2%(4/18)[9.00-45.21]</td><td colspan="1" rowspan="1">armA:100.0%0/60)[93.98-100.00]</td><td colspan="1" rowspan="1">armA:100.0%(4/4)[51.01-100.00]</td><td colspan="1" rowspan="1">armA:81.1%(60/74)[70.71-88.38]</td><td colspan="1" rowspan="1">armA:52.2%(60/115)[43.12-61.08]</td><td colspan="1" rowspan="1">armA:100.0%(1042/1042) [99.63-10.00]</td><td colspan="1" rowspan="1">armA:100.0%(60/60)[93.98-100.00]</td><td colspan="1" rowspan="1">armA:95.0%(1042/1097) [93.53-96.13]</td><td colspan="1" rowspan="1">armA: 5.2%(60/1157)</td><td colspan="1" rowspan="1">armA:3.5% of144</td></tr><tr><td colspan="1" rowspan="1">armA</td><td colspan="1" rowspan="1">P. mirabilis</td><td colspan="1" rowspan="1">100.0%(8/8)67.56-100]</td><td colspan="1" rowspan="1">100.0%(147/147)[97.45-10.00]</td><td colspan="1" rowspan="1">Gentamicin</td><td colspan="1" rowspan="1">armA:8.7%(4/46)[3.43-20.32]</td><td colspan="1" rowspan="1">armA:100.0%(32/32)89280.00]</td><td colspan="1" rowspan="1">armA:100.0%(4/4)51.1-100.00]</td><td colspan="1" rowspan="1">armA:43.2%(32/74)[32.57-54.59]</td><td colspan="1" rowspan="1">armA:12.8%(60/467)[10.11-16.19]</td><td colspan="1" rowspan="1">armA:100.0%(690/690)[99.45-10.00]</td><td colspan="1" rowspan="1">armA:100.0%(60/60)[93.98-0.00]</td><td colspan="1" rowspan="1">armA:62.9%(690/1097)60.00-65.71]</td><td colspan="1" rowspan="1">armA: 5.2%(60/1157)</td><td colspan="1" rowspan="1">armA:3.5% of144</td></tr><tr><td colspan="1" rowspan="1">armA</td><td colspan="1" rowspan="1">P. mirabilis</td><td colspan="1" rowspan="1">100.0%(8/8)67. 56-100.00]</td><td colspan="1" rowspan="1">100.0%(147/147)[97.45-100.00]</td><td colspan="1" rowspan="1">Tobramycin</td><td colspan="1" rowspan="1">armA:8.2%(4/49)[3.22-19.19]</td><td colspan="1" rowspan="1">armA:100.0%(29/9)[88.30-10.00]</td><td colspan="1" rowspan="1">armA:100.0%(4/4)[51.01-100.00]</td><td colspan="1" rowspan="1">armA:39.2%(29/74)[28.86- 50.58]</td><td colspan="1" rowspan="1">armA:13.5%(60/444)[10.64-17.01]</td><td colspan="1" rowspan="1">armA:100.0%(713/713)[99.46-0.00]</td><td colspan="1" rowspan="1">armA:100.0%(60/60)93.98 10.00]</td><td colspan="1" rowspan="1">armA:65.0%((13/1097)[62.12-67.76]</td><td colspan="1" rowspan="1">armA: 5.2%(60/1157)</td><td colspan="1" rowspan="1">armA:3.5% of144</td></tr><tr><td colspan="1" rowspan="2">AMR GeneTarget</td><td colspan="1" rowspan="2">Organism</td><td colspan="2" rowspan="1">Detection of AMR GeneTarget by AMR GenePanel versus WholeGenome SequencingCombined Clinical DevicePerformance -Stock and ProspectiveIsolates (from K191288)</td><td colspan="1" rowspan="2">Antimicrobial</td><td colspan="4" rowspan="1">Detection of AMR Gene Target by AMR GenePanel vs AST Non-SusceptibilityCombined Clinical Device Performance - Stock andProspective Isolates (from K191288) includingonly unique isolates (no replicates)</td><td colspan="5" rowspan="1">Detection of AMR Gene Target versus AST Non-Susceptibility inOpGen's SMART Study</td><td colspan="1" rowspan="2">%Orgganismswith AMRGeneTarget inNCBIPathogenDatabase(2019)</td></tr><tr><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1">PPV[95% CI]</td><td colspan="1" rowspan="1">NPV[95% CI]</td><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1">PPV[95% CI]</td><td colspan="1" rowspan="1">NPV[95% CI]</td><td colspan="1" rowspan="1">% Organismswith AMRGene Target</td></tr><tr><td colspan="1" rowspan="1">CMY</td><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">96.4%(53/5)[87.68-99.00]</td><td colspan="1" rowspan="1">99.6%(279/280)[98.01-99.4]</td><td colspan="1" rowspan="1">Cefazolin</td><td colspan="1" rowspan="1">NE1</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">CMY-2:11.4%(252/2209)[10.15-12.80]CMY-41:10.4%(229/2209)[9.16-11.71]</td><td colspan="1" rowspan="1">CMY-2:98.1%(737/751)[96.90-98.89]CMY-41:95.3%(716/751)[93.59-96.63]</td><td colspan="1" rowspan="1">CMY-2:94.7%(252/266)[91.36-96.84]CMY-41:86.7%(229/264)[82.12-90.31]</td><td colspan="1" rowspan="1">CMY-2:27.4%(737/2694)[25.71-29.07]CMY-41:26.6%(716/2696)[24.92-28.26]</td><td colspan="1" rowspan="1">CMY-2: 9%(266/2960)CMY-41:8.9%(264/2960)</td><td colspan="1" rowspan="1">CMY-2:3.5% of19017CMY-41:3.4% of19017</td></tr><tr><td colspan="1" rowspan="1">CMY</td><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">96.4%(53/55)[87.68-99.00]</td><td colspan="1" rowspan="1">99.6%(279/280)[98.01-99.94]</td><td colspan="1" rowspan="1">Cefuroxime</td><td colspan="1" rowspan="1">CMY:30.1%(41/136)[23.07-38.32]</td><td colspan="1" rowspan="1">CMY:95.7%(22/23)[79.01-99.23]</td><td colspan="1" rowspan="1">CMY:97.6%(41/42)[87.68-99.58]</td><td colspan="1" rowspan="1">CMY:18.8%(22/117)[12.76-26.83]</td><td colspan="1" rowspan="1">CMY-2:11.4%(249/2187)[10.12-12.78]CMY-41:10.4%(227/2187)[9.17-11.73]</td><td colspan="1" rowspan="1">CMY-2:97.8%(756/773)[96.51-98.62]CMY-41:95.2%(736/773)[93.47-96.51]</td><td colspan="1" rowspan="1">CMY-2:93.6%(24/266)[90.00-5.97]CMY-41:86.0%(227/264)[81.28-89.66]</td><td colspan="1" rowspan="1">CMY-2:28.1%(756/2694)[26.40-29.79]CMY-41:27.3%(736/2696)[25.65-29.01]</td><td colspan="1" rowspan="1">CMY-2: 9%(266/2960)CMY-41:8.9%(264/2960)</td><td colspan="1" rowspan="1">CMY-2:3.5% of19017CMY-41:3.4% of19017</td></tr><tr><td colspan="1" rowspan="1">CMY</td><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">96.4%(53/55)[87.68-99.00]</td><td colspan="1" rowspan="1">99.6%(279/280)[98.01-99.94]</td><td colspan="1" rowspan="1">Cefepime</td><td colspan="1" rowspan="1">CMY:17.8%(19/107)[11.67-26.08]</td><td colspan="1" rowspan="1">CMY:55.8%(29/52)[42.34-68.40]</td><td colspan="1" rowspan="1">CMY:45.2%(19/42)[31.22-60.05]</td><td colspan="1" rowspan="1">CMY:24.8%(29/117)[17.85-33.33]</td><td colspan="1" rowspan="1">CMY-2:9.3%(187/2009)[8.11-10.66]CMY-41:8.1%(162/2009)[6.95-9.34]</td><td colspan="1" rowspan="1">CMY-2:91.7%(872/951)[89.77-93.28]CMY-41:89.3%(49/951)[87.15-91.09]</td><td colspan="1" rowspan="1">CMY-2:70.3%(187/266)[64.55-75.47]CMY-41:61.4%(162/264)[55.37-67.03]</td><td colspan="1" rowspan="1">CMY-2:32.4%(872/2694)[30.63-34.16]CMY-41:31.5%(849/2696)[29.77-33.27]</td><td colspan="1" rowspan="1">CMY-2: 9%(266/2960)CMY-41:8.9%(264/2960)</td><td colspan="1" rowspan="1">CMY-2:3.5% of19017CMY-41:3.4% of19017</td></tr><tr><td colspan="1" rowspan="2">AMR GeneTarget</td><td colspan="1" rowspan="2">Organism</td><td colspan="2" rowspan="1">Detection of AMR GeneTarget by AMR GenePanel versus WholeGenome SequencingCombined Clinical DevicePerformance -Stock and ProspectiveIsolates (from K191288)</td><td colspan="1" rowspan="2">Antimicrobial</td><td colspan="4" rowspan="1">Detection of AMR Gene Target by AMR GenePanel vs AST Non-SusceptibilityCombined Clinical Device Performance - Stock andonly unique isolates (no replicates)</td><td colspan="5" rowspan="1">Detection of AMR Gene Target versus AST Non-Susceptibility inOpGen's SMART Study</td><td colspan="1" rowspan="2">%Organismswith AMRGeneTarget inNCBIPathogenDatabase(2019)</td></tr><tr><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1">PPV[95% CI]</td><td colspan="1" rowspan="1">NPV[95% CI]</td><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1">PPV[95% CI]</td><td colspan="1" rowspan="1">NPV[95% CI]</td><td colspan="1" rowspan="1">% Organismswith AMRGene Target</td></tr><tr><td colspan="1" rowspan="1">CMY</td><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">96.4%(53/55)[87.68-99.00]</td><td colspan="1" rowspan="1">99.6%(279/280)[98.01-99.94]</td><td colspan="1" rowspan="1">Ceftazidime</td><td colspan="1" rowspan="1">CMY:37.7%(40/106)[29.09-47.24]</td><td colspan="1" rowspan="1">CMY:96.2%(51/53)[87.25-98.96]</td><td colspan="1" rowspan="1">CMY:95.2%(40/42)[84.21-98.68]</td><td colspan="1" rowspan="1">CMY:43.6%(51/117)[34.95-52.64]</td><td colspan="1" rowspan="1">CMY-2:14.2%(235/1651)[12.63-16.00]CMY-41:13.2%(218/1651)[11.66-14.92]</td><td colspan="1" rowspan="1">CMY-2:97.6%(1278/1309) [96.66-98.33]CMY-41:96.5%(1263/1309) [95.34-97.36]</td><td colspan="1" rowspan="1">CMY-2:88.3%(235/266)[83.93-91.67]CMY-41:82.6%(218/264)[77.54-86.68]</td><td colspan="1" rowspan="1">CMY-2:47.4%(1278/2694) [45.56-49.3]CMY-41:46.8%(1263/2696) [44.97-48.73]</td><td colspan="1" rowspan="1">CMY-2: 9%(266/2960)CMY-41:8.9%(264/2960)</td><td colspan="1" rowspan="1">CMY-2:3.5% of19017CMY-41:3.4% of19017</td></tr><tr><td colspan="1" rowspan="1">CMY</td><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">96.4%(53/55)[87.68-99.00]</td><td colspan="1" rowspan="1">99.6%(279/280)[98.01-9.94]</td><td colspan="1" rowspan="1">Ceftriaxone</td><td colspan="1" rowspan="1">CMY:31.5%(41/130)[24.18-39.96]</td><td colspan="1" rowspan="1">CMY:96.6%(28/29)[82.82-9.39]</td><td colspan="1" rowspan="1">CMY:97.6%(41/42)[87.68-99.58]</td><td colspan="1" rowspan="1">CMY:23.9%(28/117)[17.11-32.41]</td><td colspan="1" rowspan="1">CMY-2:11.7%(248/2113)[10.43-13.18]CMY-41:10.6%(225/2113)[9.40-12.04]</td><td colspan="1" rowspan="1">CMY-2:97.9%(829/847)[96.67-98.65]CMY-41:95.4%(808/847)[93.77-96.61]</td><td colspan="1" rowspan="1">CMY-2:93.2%(248/266)[89.56-5.68]CMY-41:85.2%(225/264)[80.44-89.00]</td><td colspan="1" rowspan="1">CMY-2:30.8%(829/2694)[29.06-32.54]CMY-41:30.0%(808/2696)[28.27-31.73]</td><td colspan="1" rowspan="1">CMY-2: 9%(266/2960)CMY-41:8.9%(264/2960)</td><td colspan="1" rowspan="1">CMY-2:3.5% of19017CMY-41:3.4% of19017</td></tr><tr><td colspan="1" rowspan="1">CMY</td><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">96.4%(53/55)[87.68-99.00]</td><td colspan="1" rowspan="1">99.6%(279/280)[98.01-99.94]</td><td colspan="1" rowspan="1">Imipenem</td><td colspan="1" rowspan="1">CMY:25.6%(11/43)[14.93-40.24]</td><td colspan="1" rowspan="1">CMY:73.3%(85/116)[64.57-80.49]</td><td colspan="1" rowspan="1">CMY:26.2%(11/42)[15.30-41.07]</td><td colspan="1" rowspan="1">CMY:72.6%(85/117)[63.95-79.91]</td><td colspan="1" rowspan="1">CMY-2:32.8%(76/232)[27.04-39.04]CMY-41:33.6%(78/232)[27.85-39.92]</td><td colspan="1" rowspan="1">CMY-2:93.0%(2538/2728) [92.02-93.93]CMY-41:93.2%(2542/2728) [92.17-94.07]</td><td colspan="1" rowspan="1">CMY-2:28.6%(76/266)[23.48-34.28]CMY-41:29.5%(78/264)[24.37-35.31]</td><td colspan="1" rowspan="1">CMY-2:94.2%(2538/2694) [93.26-95.03]CMY-41:94.3%(2542/2696) [93.35-95.10]</td><td colspan="1" rowspan="1">CMY-2: 9%(266/2960)CMY-41:8.9%(264/2960)</td><td colspan="1" rowspan="1">CMY-2:3.5% of19017CMY-41:3.4% of19017</td></tr><tr><td colspan="1" rowspan="2">AMR GeneTarget</td><td colspan="1" rowspan="2">Organism</td><td colspan="2" rowspan="1">Detection of AMR GeneTarget by AMR GenePanel versus WholeGenome SequencingCombined Clinical DevicePerformance-Stock and ProspectiveIsolates (from K191288)</td><td colspan="1" rowspan="2">Antimicrobial</td><td colspan="4" rowspan="1">Detection of AMR Gene Target by AMR GenePanel vs AST Non-SusceptibilityCombined Clinical Device Performance - Stock andivaonly unique isolates (no replicates)</td><td colspan="5" rowspan="1">Detection of AMR Gene Target versus AST Non-Susceptibility inOpGen's SMART Study</td><td colspan="1" rowspan="2">%Organismswith AMRGeneTarget inNCBIPathogenDatabase(2019)</td></tr><tr><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1">PPV[95% CI]</td><td colspan="1" rowspan="1">NPV[95% CI]</td><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1">PPV[95% CI]</td><td colspan="1" rowspan="1">NPV[95% CI]</td><td colspan="1" rowspan="1">% Organismswith AMRGene Target</td></tr><tr><td colspan="1" rowspan="1">CMY</td><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">96.4%(53/55)[87.68-99.00]</td><td colspan="1" rowspan="1">99.6%(279/280)[98.01-99.94]</td><td colspan="1" rowspan="1">Ertapenem</td><td colspan="1" rowspan="1">CMY:30.9%(17/55)[20.28-44.03]</td><td colspan="1" rowspan="1">CMY:76.0%(79/104)[66.92-83.15]</td><td colspan="1" rowspan="1">CMY:40.5%(17/42)[27.04-5.51]</td><td colspan="1" rowspan="1">CMY:67.5%(79/117)[58.60-75.33]</td><td colspan="1" rowspan="1">CMY-2:32.6%(126/386)[28.16-37.47]CMY-41:33.4%(129/386)[28.90-38.27]</td><td colspan="1" rowspan="1">CMY-2:94.6%(2434/2574) [93.62-5.37]CMY-41:94.8%(2439/2574) [93.83-95.55]</td><td colspan="1" rowspan="1">CMY-2:47.4%(126/266)[41.45-53.36]CMY-41:48.9%(129/264)[42.89-54.87]</td><td colspan="1" rowspan="1">CMY-2:90.3%(2434/2694) [89.18-91.41]CMY-41:90.5%(2439/2696) [89.30-91.52]</td><td colspan="1" rowspan="1">CMY-2: 9%(266/2960)CMY-41:8.9%(264/2960)</td><td colspan="1" rowspan="1">CMY-2:3.5% of19017CMY-41:3.4% of19017</td></tr><tr><td colspan="1" rowspan="1">CMY</td><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">96.4%(53/55)[87.68-99.00]</td><td colspan="1" rowspan="1">99.6%(279/280)998.01-99.94]</td><td colspan="1" rowspan="1">Meropenem</td><td colspan="1" rowspan="1">CMY:21.6%(8/37)[11.39-37.20]</td><td colspan="1" rowspan="1">CMY:72.1%(88/122)[63.59-79.32]</td><td colspan="1" rowspan="1">CMY:19.0%(8/42)[9.98-33.30]</td><td colspan="1" rowspan="1">CMY:75.2%(88/117)[66.67-82.15]</td><td colspan="1" rowspan="1">CMY-2:32.7%(67/205)[26.63-39.37]CMY-41:33.7%(69/205)[27.54-40.38]</td><td colspan="1" rowspan="1">CMY-2:92.8%(2556/2755) [91.75-93.69]CMY-41:92.9%(2560/2755) [91.90-93.82]</td><td colspan="1" rowspan="1">CMY-2:25.2%(67/266)[20.35-30.73]CMY-41:26.1%(69/264)[21.21-31.75]</td><td colspan="1" rowspan="1">CMY-2:94.9%(2556/2694) [93.98-5.65]CMY-41:95.0%(2560/2696) [94.06-95.72]</td><td colspan="1" rowspan="1">CMY-2: 9%(266/2960)CY-41:8.9%(264/2960)</td><td colspan="1" rowspan="1">CMY-2:3.5% of19017CMY-41:3.4% of19017</td></tr><tr><td colspan="1" rowspan="1">CMY</td><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">96.4%(53/55)[87.68-99.00]</td><td colspan="1" rowspan="1">99.6%(279/280)[98.01-99.94]</td><td colspan="1" rowspan="1">Amox.K.Clav</td><td colspan="1" rowspan="1">CMY:36.8%(42/114)[28.55-45.99]</td><td colspan="1" rowspan="1">CMY:100.0%(45/45)[92.13-100.00]</td><td colspan="1" rowspan="1">CMY:100.0%(42/42)[91.62-100]</td><td colspan="1" rowspan="1">CMY:38.5%(45/117)[30.15-47.51]</td><td colspan="1" rowspan="1">CMY-2:13.8%(250/1810)[12.30-15.48]CMY-41:13.0%(235/1810)[11.51-14.61]</td><td colspan="1" rowspan="1">CMY-2:98.6%(1134/1150) [97.75-99.14]CMY-41:97.5%(1121/1150) [96.40-98.24]</td><td colspan="1" rowspan="1">CMY-2:94.0%(250/266)[90.45-96.26]CMY-41:89.0%(235/264)[84.67-92.24]</td><td colspan="1" rowspan="1">CMY-2:42.1%(1134/2694) [40.24-43.97]CMY-41:41.6%(1121/2696) [39.73-43.45]</td><td colspan="1" rowspan="1">CMY-2: 9%(266/2960)CMY-41:8.9%(264/2960)</td><td colspan="1" rowspan="1">CMY-2:3.5% of19017CMY-41:3.4% of19017</td></tr><tr><td colspan="1" rowspan="2">AMR GeneTarget</td><td colspan="1" rowspan="2">Organism</td><td colspan="2" rowspan="1">Detection of AMR GeneTarget by AMR GenePanel versus WholeGenome SequencingCombined Clinical DevicePerformance -Stock and ProspectiveIsolates (from K191288)</td><td colspan="1" rowspan="2">Antimicrobial</td><td colspan="4" rowspan="1">Detection of AMR Gene Target by AMR GenePanel vs AST Non-SusceptibilityCombined Clinical Device Performance - Stock andProspective Isolates (from K191288) includingonly unique isolates (no replicates)</td><td colspan="5" rowspan="1">Detection of AMR Gene Target versus AST Non-Susceptibility inOpGen's SMART Study</td><td colspan="1" rowspan="2">%Orgganismswith AMRGeneTarget inNCBIPathogenDatabase(2019)</td></tr><tr><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1">PPV[95% CI]</td><td colspan="1" rowspan="1">NPV[95% CI]</td><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1">PPV[95% CI]</td><td colspan="1" rowspan="1">NPV[95% CI]</td><td colspan="1" rowspan="1">% Organismswith AMRGene Target</td></tr><tr><td colspan="1" rowspan="1">CMY</td><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">96.4%(53/55)[87.68-99.00]</td><td colspan="1" rowspan="1">99.6%(279/280)[98.01-9.94]</td><td colspan="1" rowspan="1">Amp.Sulbactam</td><td colspan="1" rowspan="1">CMY:27.4%(40/146)[20.81-35.14]</td><td colspan="1" rowspan="1">CMY:84.6%(11/13)[57.77-95.67]</td><td colspan="1" rowspan="1">CMY:95.2%(40/42)[84.21-98.68]</td><td colspan="1" rowspan="1">CMY:9.4%(11/117)[5.33-16.05]</td><td colspan="1" rowspan="1">CMY-2:10.3%(255/2477)[9.16-11.55]CMY-41:10.0%(247/2477)[8.85-11.21]</td><td colspan="1" rowspan="1">CMY-2:97.7%(472/483)[95.97-98.72]CMY-41:96.5%(466/483)[94.44-97.79]</td><td colspan="1" rowspan="1">CMY-2:95.9%(255/266)[92.75-97.68]CMY-41:93.6%(247/264)[89.93-95.94]</td><td colspan="1" rowspan="1">CMY-2:17.5%(472/2694)[16.13-19.00]CMY-41:17.3%(466/2696)[15.90-18.76]</td><td colspan="1" rowspan="1">CMY-2: 9%(266/2960)CMY-41:8.9%(264/2960)</td><td colspan="1" rowspan="1">CMY-2:3.5% of19017CMY-41:3.4% of19017</td></tr><tr><td colspan="1" rowspan="1">CMY</td><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">96.4%(53/55)[87.68-99.00]</td><td colspan="1" rowspan="1">99.6%(279/280)9.01-99.94]</td><td colspan="1" rowspan="1">Pip.Tazo</td><td colspan="1" rowspan="1">CMY:34.8%(24/69)[24.62-46.55]</td><td colspan="1" rowspan="1">CMY:80.0%(72/90)[70.59-86.96]</td><td colspan="1" rowspan="1">CMY:57.1%(24/42)[42.21-70.88]</td><td colspan="1" rowspan="1">CMY:61.5%(72/117)[52.49-69.85]</td><td colspan="1" rowspan="1">CMY-2:22.0%(151/687)[19.04-25.23]CMY-41:20.5%(141/687)[17.67-23.70]</td><td colspan="1" rowspan="1">CMY-2:94.9%(2158/2273) [93.96-95.77]CMY-41:94.6%(2150/2273) [93.58-95.45]</td><td colspan="1" rowspan="1">CMY-2:56.8%(151/266)[50.76-62.58]CMY-41:53.4%(141/264)[47.39-59.33]</td><td colspan="1" rowspan="1">CMY-2:80.1%(2158/2694) [78.55-81.57]CMY-41:79.7%(2150/2696) [78.19-81.22]</td><td colspan="1" rowspan="1">CMY-2: 9%(266/2960)CMY-41:8.9%(264/2960)</td><td colspan="1" rowspan="1">CMY-2:3.5% of19017CMY-41:3.4% of19017</td></tr><tr><td colspan="1" rowspan="1">CMY</td><td colspan="1" rowspan="1">K. pneumoniae</td><td colspan="1" rowspan="1">100.0%(10/10)[72.25-100.00]</td><td colspan="1" rowspan="1">99.7%(318/319)[98.25-99.94]</td><td colspan="1" rowspan="1">Cefazolin</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">CMY-2:2.6%(47/1824)[1.94-3.41]CMY-41:2.7%(49/1824)[2.04-3.53]</td><td colspan="1" rowspan="1">CMY-2:100.0%(204/204)[98.15-100.00]CMY-41:100.0%(204/204)[98.15-10.00]</td><td colspan="1" rowspan="1">CMY-2:100.0%(47/47)[92.44-100.00]CMY-41:100.0%(49/49)[92.73-100.00]</td><td colspan="1" rowspan="1">CMY-2:10.3%(204/1981)[9.04-11.71]CMY-41:10.3%(204/1979)[9.04-11.73]</td><td colspan="1" rowspan="1">CMY-2: 2.3%(47/2028)CMY-41:2.4%(49/2028)</td><td colspan="1" rowspan="1">CMY-2:1.4% of7979CMY-41:0.7% of7979</td></tr><tr><td colspan="1" rowspan="2">AMR GeneTarget</td><td colspan="1" rowspan="2">Organism</td><td colspan="2" rowspan="1">Detection of AMR GeneTarget by AMR GenePanel versus WholeGenome SequencingCombined Clinical DevicePerformance-Stock and ProspectiveIsolates (from K191288)</td><td colspan="1" rowspan="2">Antimicrobial</td><td colspan="4" rowspan="1">Detection of AMR Gene Target by AMR GenePanel vs AST Non-SusceptibilityCombined Clinical Device Performance - Stock andProspective Isolates (from K191288) includingonly unique isolates (no replicates)</td><td colspan="5" rowspan="1">Detection of AMR Gene Target versus AST Non-Susceptibility inOpGen's SMART Study</td><td colspan="1" rowspan="2">%Organismswith AMRGeneTarget inNCBIPathogenDatabase(2019)</td></tr><tr><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1">PPV[95% CI]</td><td colspan="1" rowspan="1">NPV[95% CI]</td><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1">PPV[95% CI]</td><td colspan="1" rowspan="1">NPV[95% CI]</td><td colspan="1" rowspan="1">% Organismswith AMRGene Target</td></tr><tr><td colspan="1" rowspan="1">CMY</td><td colspan="1" rowspan="1">K. pneumoniae</td><td colspan="1" rowspan="1">100.0%(10/10)[72.25-100.00]</td><td colspan="1" rowspan="1">99.7%(318/319)[98.25-99.94]</td><td colspan="1" rowspan="1">Cefuroxime</td><td colspan="1" rowspan="1">CMY:4.7%(8/170)[2.40-9.01]</td><td colspan="1" rowspan="1">CMY:93.3%(14/15)[70.18-98.81]</td><td colspan="1" rowspan="1">CMY:88.9%(8/9)[56.50-98.01]</td><td colspan="1" rowspan="1">CMY:8.0%(14/176)[4.80-12.91]</td><td colspan="1" rowspan="1">CMY-2:2.6%(47/1832)[1.93-3.39]CMY-41:2.7%(49/1832)[2.03-3.52]</td><td colspan="1" rowspan="1">CMY-2:100.0%(196/196)[98.08-0.00]CMY-41:100.0%(196/196)[98.08-100.00]</td><td colspan="1" rowspan="1">CMY-2:100.0%(47/47)[92.44-100.00]CMY-41:100.0%(49/49)[92.73-100.00]</td><td colspan="1" rowspan="1">CMY-2:9.9%(196/1981)[8.66-11.29]CMY-41:9.9%(196/1979)[8.66-11.30]</td><td colspan="1" rowspan="1">CMY-2: 2.3%(47/2028)CMY-41:2.4%(49/2028)</td><td colspan="1" rowspan="1">CMY-2:1.4% of7979CMY-41:0.7% of7979</td></tr><tr><td colspan="1" rowspan="1">CMY</td><td colspan="1" rowspan="1">K. pneumoniae</td><td colspan="1" rowspan="1">100.0%(10/10)72.25-00]</td><td colspan="1" rowspan="1">99.7%(318/319)[98.25-99.94]</td><td colspan="1" rowspan="1">Cefepime</td><td colspan="1" rowspan="1">CMY:5.0%(8/160)[2.56-9.56]</td><td colspan="1" rowspan="1">CMY:96.0%(24/25)[80.46-99.29]</td><td colspan="1" rowspan="1">CMY:88.9%(8/9)[56.50-98.01]</td><td colspan="1" rowspan="1">CMY:13.6%(24/176)[9.34-19.49]</td><td colspan="1" rowspan="1">CMY-2:2.6%45/1711)[1.97-3.50]CMY-41:2.7%(47/1711)[2.07-3.63]</td><td colspan="1" rowspan="1">CMY-2:99.4%(315/317)[97.73-99.83]CMY-41:99.4%(315/317)[97.73-99.83]</td><td colspan="1" rowspan="1">CMY-2:95.7%(45/47)[85.75-98.83]CMY-41:95.9%(47/49)[86.29-98.87]</td><td colspan="1" rowspan="1">CMY-2:15.9%(315/1981)[14.36-17.58]CMY-41:15.9%(315/1979)[14.37-17.59]</td><td colspan="1" rowspan="1">CMY-2: 2.3%(47/2028)CMY-41:2.4%(49/2028)</td><td colspan="1" rowspan="1">CMY-2:1.4% of7979CMY-41:0.7% of7979</td></tr><tr><td colspan="1" rowspan="1">CMY</td><td colspan="1" rowspan="1">K. pneumoniae</td><td colspan="1" rowspan="1">100.0%(10/10)[72.25-100.00]</td><td colspan="1" rowspan="1">99.7%(318/319)[98.25-99.94]</td><td colspan="1" rowspan="1">Ceftazidime</td><td colspan="1" rowspan="1">CMY:4.3%(7/163)[2.10-8.60]</td><td colspan="1" rowspan="1">CMY:90.9%(20/22)[72.19-97.47]</td><td colspan="1" rowspan="1">CMY:77.8%(7/9)[45.26-93.68]</td><td colspan="1" rowspan="1">CMY:11.4%(20/176)[7.48-16.90]</td><td colspan="1" rowspan="1">CMY-2:2.6%(45/1710)[1.97-3.50]CMY-41:2.7%(47/1710)[2.07-3.64]</td><td colspan="1" rowspan="1">CMY-2:99.4%(316/318)[97.74-99.83]CMY-41:99.4%(316/318)[97.74-99.83]</td><td colspan="1" rowspan="1">CMY-2:95.7%(45/47)[85.75-98.83]CMY-41:95.9%(47/49)[86.29-98.87]</td><td colspan="1" rowspan="1">CMY-2:16.0%(316/1981)[14.41-17.63]CMY-41:16.0%(316/1979)[14.42-17.65]</td><td colspan="1" rowspan="1">CMY-2: 2.3%(47/2028)CY-41:2.4%(49/2028)</td><td colspan="1" rowspan="1">CMY-2:1.4% of7979CMY-41:0.7% of7979</td></tr><tr><td colspan="1" rowspan="2">AMR GeneTarget</td><td colspan="1" rowspan="2">Organism</td><td colspan="2" rowspan="1">Detection of AMR GeneTarget by AMR GenePanel versus WholeGenome SequencingCombined Clinical DevicePerformance -Stock and ProspectiveIsolates (from K191288)</td><td colspan="1" rowspan="2">Antimicrobial</td><td colspan="4" rowspan="1">Detection of AMR Gene Target by AMR GenePanel vs AST Non-SusceptibilityCombined Clinical Device Performance - Stock andProspective Isolates (from K191288) includingonly unique isolates (no replicates)</td><td colspan="5" rowspan="1">Detection of AMR Gene Target versus AST Non-Susceptibility inOpGen's SMART Study</td><td colspan="1" rowspan="2">%Organiswith AMRGeneTarget inNCBIPathogenDatabase(2019)</td></tr><tr><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1">PPV[95% CI]</td><td colspan="1" rowspan="1">NPV[95% CI]</td><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1">PPV[95% CI]</td><td colspan="1" rowspan="1">NPV[95% CI]</td><td colspan="1" rowspan="1">% Organismswith AMRGene Target</td></tr><tr><td colspan="1" rowspan="1">CMY</td><td colspan="1" rowspan="1">K. pneumoniae</td><td colspan="1" rowspan="1">100.0%(10/10)[72.25-100.00]</td><td colspan="1" rowspan="1">99.7%(318/319)[98.25-99.94]</td><td colspan="1" rowspan="1">Ceftriaxone</td><td colspan="1" rowspan="1">CMY:4.8%(8/167)[2.45-9.17]</td><td colspan="1" rowspan="1">CMY:94.4%(17/18)[74.24-99.01]</td><td colspan="1" rowspan="1">CMY:88.9%(8/9)[56.50-98.01]</td><td colspan="1" rowspan="1">CMY:9.7%(17/176)[6.12-14.92]</td><td colspan="1" rowspan="1">CMY-2:2.7%(47/1765)[2.01-3.52]CMY-41:2.8%(49/1765)[2.11-3.65]</td><td colspan="1" rowspan="1">CMY-2:100.0%(263/263)[98.56-00]CMY-41:100.0%(263/263)[98.56-100.00]</td><td colspan="1" rowspan="1">CMY-2:100.0%(47/47)[92.44-100.00]CMY-41:100.0%(49/49)[92.73-100.00]</td><td colspan="1" rowspan="1">CMY-2:13.3%(263/1981)[11.85-14.84]CMY-41:13.3%(263/1979)[11.86-14.86]</td><td colspan="1" rowspan="1">CMY-2: 2.3%(47/2028)CMY-41:2.4%(49/2028)</td><td colspan="1" rowspan="1">CMY-2:1.4% of7979CMY-41:0.7% of7979</td></tr><tr><td colspan="1" rowspan="1">CMY</td><td colspan="1" rowspan="1">K. pneumoniae</td><td colspan="1" rowspan="1">100.0%(10/10)[72.25-100.00]</td><td colspan="1" rowspan="1">99.7%(318/319)[98.25-99.94]</td><td colspan="1" rowspan="1">Imipenem</td><td colspan="1" rowspan="1">CMY:5.7%(6/106)[2.62-11.80]</td><td colspan="1" rowspan="1">CMY:96.2%(76/79)[89.42-98.70]</td><td colspan="1" rowspan="1">CMY:66.7%(6/9)[35.42-87.94]</td><td colspan="1" rowspan="1">CMY:43.2%(76/176)[36.09-50.57]</td><td colspan="1" rowspan="1">CMY-2:4.0%(33/818)[2.89-5.61]CMY-41:4.0%(33/818)[2.89-5.61]</td><td colspan="1" rowspan="1">CMY-2:98.8%(1196/1210) [98.07-9.31]CMY-41:98.7%(1194/121) [97.86-99.18]</td><td colspan="1" rowspan="1">CMY-2:70.2%(33/47)[56.02-81.35]CMY-41:67.3%(33/49)[53.38-78.79]</td><td colspan="1" rowspan="1">CMY-2:60.4%(1196/1981) [58.20-62.51]CMY-41:60.3%(1194/1979) [58.16-62.47]</td><td colspan="1" rowspan="1">CMY-2: 2.3%(47/2028)CMY-41:2.4%(49/2028)</td><td colspan="1" rowspan="1">CMY-2:1.4% of7979CMY-41:0.7% of7979</td></tr><tr><td colspan="1" rowspan="1">CMY</td><td colspan="1" rowspan="1">K. pneumoniae</td><td colspan="1" rowspan="1">100.0%(10/10)[72.25-100.00]</td><td colspan="1" rowspan="1">99.7%(318/319)[98.25-99.94]</td><td colspan="1" rowspan="1">Ertapenem</td><td colspan="1" rowspan="1">CMY:4.8%(6/126)[2.20-10.00]</td><td colspan="1" rowspan="1">CMY:94.9%(56/59)[86.08-98.26]</td><td colspan="1" rowspan="1">CMY:66.7%(6/9)[35.42-87.94]</td><td colspan="1" rowspan="1">CMY:31.8%(56/176)[25.39-39.02]</td><td colspan="1" rowspan="1">CMY-2:3.6%(38/1053)[2.64-4.91]CMY-41:3.6%(38/1053)[2.64-4.91]</td><td colspan="1" rowspan="1">CMY-2:99.1%(966/975)[98.26-99.51]CMY-41:98.9%(964/975)[97.99-999.37]</td><td colspan="1" rowspan="1">CMY-2:80.9%(38/47)[67.46-89.58]CMY-41:77.6%(38/49)[64.12-86.98]</td><td colspan="1" rowspan="1">CMY-2:48.8%(966/1981)[46.57-50.96]CMY-41:48.7%(964/1979)[46.51-50.91]</td><td colspan="1" rowspan="1">CMY-2: 2.3%(47/2028)CMY-41:2.4%(49/2028)</td><td colspan="1" rowspan="1">CMY-2:1.4% of7979CMY-41:0.7% of7979</td></tr><tr><td colspan="1" rowspan="2">AMR GeneTarget</td><td colspan="1" rowspan="2">Organism</td><td colspan="2" rowspan="1">Detection of AMR GeneTarget by AMR GenePanel versus WholeGenome SequencingCombined Clinical DevicePerformance -Stock and ProspectiveIsolates (from K191288)</td><td colspan="1" rowspan="2">Antimicrobial</td><td colspan="4" rowspan="1">Detection of AMR Gene Target by AMR GenePanel vs AST Non-SusceptibilityCombined Clinical Device Performance - Stock andivaonly unique isolates (no replicates)</td><td colspan="5" rowspan="1">Detection of AMR Gene Target versus AST Non-Susceptibility inOpGen's SMART Study</td><td colspan="1" rowspan="2">%Organismswith AMRGeneTarget inNCBIPathogenDatabase(2019)</td></tr><tr><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1">PPV[95% CI]</td><td colspan="1" rowspan="1">NPV[95% CI]</td><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1">PPV[95% CI]</td><td colspan="1" rowspan="1">NPV[95% CI]</td><td colspan="1" rowspan="1">% Organismswith AMRGene Target</td></tr><tr><td colspan="1" rowspan="1">CMY</td><td colspan="1" rowspan="1">K. pneumoniae</td><td colspan="1" rowspan="1">100.0%(10/10)[72.25-100.00]</td><td colspan="1" rowspan="1">99.7%(318/319)[98.25-99.94]</td><td colspan="1" rowspan="1">Meropenem</td><td colspan="1" rowspan="1">CMY:6.1%(6/99)[2.81-12.60]</td><td colspan="1" rowspan="1">CMY:96.5%(83/86)[90.24-98.81]</td><td colspan="1" rowspan="1">CMY:66.7%(6/9)[35.42-87.94]</td><td colspan="1" rowspan="1">CMY:47.2%(83/176)[39.92-54.52]</td><td colspan="1" rowspan="1">CMY-2:3.8%(32/842)[2.70-5.32]CMY-41:3.8%(32/842)[2.70-5.32]</td><td colspan="1" rowspan="1">CMY-2:98.7%(1171/1186) [97.92-99.23]CMY-41:98.6%(1169/1186) [97.72-99.10]</td><td colspan="1" rowspan="1">CMY-2:68.1%(32/47)[53.83-79.60]CMY-41:65.3%(32/49)[51.31-77.08]</td><td colspan="1" rowspan="1">CMY-2:59.1%(1171/1981) [56.93-61.26]CMY-41:59.1%(1169/1979) [56.89-61.22]</td><td colspan="1" rowspan="1">CMY-2: 2.3%(47/2028)CMY-41:2.4%(49/2028)</td><td colspan="1" rowspan="1">CMY-2:1.4% of7979CMY-41:0.7% of7979</td></tr><tr><td colspan="1" rowspan="1">CMY</td><td colspan="1" rowspan="1">K. pneumoniae</td><td colspan="1" rowspan="1">100.0%(10/10)72.25-100.00]</td><td colspan="1" rowspan="1">99.7%(318/319)98.25-99.94]</td><td colspan="1" rowspan="1">Amox.K.Clav</td><td colspan="1" rowspan="1">CMY:4.9%(/163)[2.51-9.38]</td><td colspan="1" rowspan="1">CMY:95.5%(21/22)[78.20-9.19]</td><td colspan="1" rowspan="1">CMY:88.9%(8/9)[56.50-98.01]</td><td colspan="1" rowspan="1">CMY:11.9%(21/176)[7.94-17.55]</td><td colspan="1" rowspan="1">CMY-2:2.7%(47/1716)[2.07-3.62]CMY-41:2.9%(49/1716)[2.17-3.75]</td><td colspan="1" rowspan="1">CMY-2:100.0%(312/312)[98.78-100.00]CY-41:100.0%(312/312)[98.78-100.00]</td><td colspan="1" rowspan="1">CMY-2:100.0%(47/47)[92.44-10.00]CMY-41:100.0%(49/49)[92.73-100.00]</td><td colspan="1" rowspan="1">CMY-2:15.7%(312/1981)[14.21-17.42]CMY-41:15.8%(312/1979)[14.23-17.44]</td><td colspan="1" rowspan="1">CMY-2: 2.3%(47/2028)CMMY-41:2.4%(49/2028)</td><td colspan="1" rowspan="1">CMY-2:1.4% of979CMY-41:0.7% of7979</td></tr><tr><td colspan="1" rowspan="1">CMY</td><td colspan="1" rowspan="1">K. pneumoniae</td><td colspan="1" rowspan="1">100.0%(10/10)[72.25-100.00]</td><td colspan="1" rowspan="1">99.7%(318/319)[98.25-99.94]</td><td colspan="1" rowspan="1">Amp.Sulbactam</td><td colspan="1" rowspan="1">CMY:5.1%(9/177)[2.70-9.38]</td><td colspan="1" rowspan="1">CMY:100.0%(8/8)[67.56-10.00]</td><td colspan="1" rowspan="1">CMY:100.0%(9/9)[70.09-100]</td><td colspan="1" rowspan="1">CMY:4.5%(8/176)[2.32-8.71]</td><td colspan="1" rowspan="1">CMY-2:2.5%(47/1909)[1.86-3.26]CMY-41:2.6%(49/1909)[1.95-3.38]</td><td colspan="1" rowspan="1">CMY-2:100.0%(119/119)[96.87-100.00]CMY-41:100.0%(119/119)[96.87-100.00]</td><td colspan="1" rowspan="1">CMY-2:100.0%(47/47)[92.44-100.00]CMY-41:100.0%(49/49)[92.73-100.00]</td><td colspan="1" rowspan="1">CMY-2:6.0%(119/1981)[5.04-7.14]CMY-41:6.0%(119/1979)[5.05-7.15]</td><td colspan="1" rowspan="1">CMY-2: 2.3%(47/2028)CMY-41:2.4%(49/2028)</td><td colspan="1" rowspan="1">CMY-2:1.4% of7979CMY-41:0.7% of7979</td></tr><tr><td colspan="1" rowspan="2">AMR GeneTarget</td><td colspan="1" rowspan="2">Organism</td><td colspan="2" rowspan="1">Detection of AMR GeneTarget by AMR GenePanel versus WholeGenome SequencingCombined Clinical DevicePerformance -Stock and ProspectiveIsolates (from K191288)</td><td colspan="1" rowspan="2">Antimicrobial</td><td colspan="4" rowspan="1">Detection of AMR Gene Target by AMR GenePanel vs AST Non-SusceptibilityCombined Clinical Device Performance - Stock andivonly unique isolates (no replicates)</td><td colspan="5" rowspan="1">Detection of AMR Gene Target versus AST Non-Susceptibility inOpGen's SMART Study</td><td colspan="1" rowspan="2">%Organismswith AMRGeneTarget inNCBIPathogenDatabase(2019)</td></tr><tr><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1">PPV[95% CI]</td><td colspan="1" rowspan="1">NPV[95% CI]</td><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1">PPV[95% CI]</td><td colspan="1" rowspan="1">NPV[95% CI]</td><td colspan="1" rowspan="1">% Organismswith AMRGene Target</td></tr><tr><td colspan="1" rowspan="1">CMY</td><td colspan="1" rowspan="1">K. pneumoniae</td><td colspan="1" rowspan="1">100.0%(10/10)[72.25-0.00]</td><td colspan="1" rowspan="1">99.7%(318/319)[98.25-99.94]</td><td colspan="1" rowspan="1">Pip.Tazo</td><td colspan="1" rowspan="1">CMY:4.1%(6/147)[1.88-8.62]</td><td colspan="1" rowspan="1">CMY:92.1%(35/38)[79.20-97.28]</td><td colspan="1" rowspan="1">CMY:66.7%(6/9)[35.42-87.94]</td><td colspan="1" rowspan="1">CMY:19.9%(35/176)[14.66-26.40]</td><td colspan="1" rowspan="1">CMY-2:2.9%(41/1400)[2.17-3.95]CMY-41:3.1%(43/1400)[2.29-4.11]</td><td colspan="1" rowspan="1">CMY-2:999.0%(622/628)[97.93-99.56]CMY-41:99.%(622/628)[97.93-99.56]</td><td colspan="1" rowspan="1">CMY-2:87.2%(41/47)[74.83-94.02]CMY-41:87.8%(43/49)[75.76-94.27]</td><td colspan="1" rowspan="1">CMY-2:31.4%(622/1981)[29.39-33.48]CMY-41:31.4%(622/1979)[29.42-33.51]</td><td colspan="1" rowspan="1">CMY-2: 2.3%(47/2028)CMY-41:2.4%(49/2028)</td><td colspan="1" rowspan="1">CMY-2:1.4% of7979CMY-41:0.7% of7979</td></tr><tr><td colspan="1" rowspan="1">CMY</td><td colspan="1" rowspan="1">P. mirabilis</td><td colspan="1" rowspan="1">100.0%(63/63)[94.25-100.00]</td><td colspan="1" rowspan="1">98.9%(91/92)[94.10-99.81]</td><td colspan="1" rowspan="1">Cefazolin</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">CMY-2:31.3%(179/571)[27.68-35.27]CMY-41:31.3%(179/571)[27.68-35.27]</td><td colspan="1" rowspan="1">CMY-2:99.7%(584/586)[98.76-99.91]CMY-41:99.7%(584/586)[98.76-99.91]</td><td colspan="1" rowspan="1">CMY-2:98.9%(179/181)[96.06-99.70]CMY-41:98.9%(179/181)[96.06-99.70]</td><td colspan="1" rowspan="1">CMY-2:59.8%(584/976)[56.73-62.87]CMY-41:59.8%(584/976)[56.73-62.87]</td><td colspan="1" rowspan="1">CMY-2:15.6%(181/1157)CMY-41:15.6%(181/1157)</td><td colspan="1" rowspan="1">CMY-2:9.7% of144CMY-41:4.9% of144</td></tr><tr><td colspan="1" rowspan="1">CMY</td><td colspan="1" rowspan="1">P. mirabilis</td><td colspan="1" rowspan="1">100.0%(63/63)[94.25-100.00]</td><td colspan="1" rowspan="1">98.9%(91/92)[94.10-99.81]</td><td colspan="1" rowspan="1">Cefuroxime</td><td colspan="1" rowspan="1">CMY:6.7%(3/45)[2.29-17.86]</td><td colspan="1" rowspan="1">CMY:93.9%(31/33)[80.39-98.]</td><td colspan="1" rowspan="1">CMY:60.0%(3/5)[23.07-88.24]</td><td colspan="1" rowspan="1">CMY:42.5%(31/73)[31.78-5.90]</td><td colspan="1" rowspan="1">CMY-2:31.5%(156/495)[27.58-35.74]CMY-41:31.5%(156/495)[27.58-35.74]</td><td colspan="1" rowspan="1">CMY-2:96.2%(637/662)[94.48-97.43]CMY-41:96.2%(637/662)[94.48-97.43]</td><td colspan="1" rowspan="1">CMY-2:86.2%(156/181)[80.41-90.47]CMY-41:86.2%(156/181)[80.41-90.47]</td><td colspan="1" rowspan="1">CMY-2:65.3%(637/976)[62.22-68.19]CMY-41:65.3%(637/976)[62.22-68.19]</td><td colspan="1" rowspan="1">CMY-2:15.6%(181/1157)CMY-41:15.6%(181/1157)</td><td colspan="1" rowspan="1">CMY-2:9.7% of144CMY-41:4.9% of144</td></tr><tr><td colspan="1" rowspan="2">AMR GeneTarget</td><td colspan="1" rowspan="2">Organism</td><td colspan="2" rowspan="1">Detection of AMR GeneTarget by AMR GenePanel versus WholeGenome SequencingCombined Clinical DevicePerformance -Stock and ProspectiveIsolates (from K191288)</td><td colspan="1" rowspan="2">Antimicrobial</td><td colspan="4" rowspan="1">Detection of AMR Gene Target by AMR GenePanel vs AST Non-SusceptibilityCombined Clinical Device Performance - Stock andivaonly unique isolates (no replicates)</td><td colspan="5" rowspan="1">Detection of AMR Gene Target versus AST Non-Susceptibility inOpGen's SMART Study</td><td colspan="1" rowspan="2">%Organismswith AMRGeneTarget inNCBIPathogenDatabase(2019)</td></tr><tr><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1">PPV[95% CI]</td><td colspan="1" rowspan="1">NPV[95% CI]</td><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1">PPV[95% CI]</td><td colspan="1" rowspan="1">NPV[95% CI]</td><td colspan="1" rowspan="1">% Organismswith AMRGene Target</td></tr><tr><td colspan="1" rowspan="1">CMY</td><td colspan="1" rowspan="1">P. mirabilis</td><td colspan="1" rowspan="1">100.0%(63/63)[94.25-100.00]</td><td colspan="1" rowspan="1">98.9%(91/92)[94.10-99.81]</td><td colspan="1" rowspan="1">Cefepime</td><td colspan="1" rowspan="1">CMY:5.0%(2/40)[1.38-16.50]</td><td colspan="1" rowspan="1">CMY:92.1%(35/38)[79.20-97.28]</td><td colspan="1" rowspan="1">CMY:40.0%(2/5)[11.76-76.93]</td><td colspan="1" rowspan="1">CMY:47.9%(35/73)[36.88-59.22]</td><td colspan="1" rowspan="1">CMY-2:12.6%(42/334)[9.44-16.56]CMY-41:12.6%(42/334)[9.44-16.56]</td><td colspan="1" rowspan="1">CMY-2:83.1%(684/823)[80.40-85.52]CMY-41:83.1%(684/823)[80.40-85.52]</td><td colspan="1" rowspan="1">CMY-2:23.2%(42/181)[17.65-29.87]CMY-41:23.2%(42/181)[17.65-29.87]</td><td colspan="1" rowspan="1">CMY-2:70.1%(684/976)[67.14-72.87]CMY-41:70.1%(684/976)[67.14-72.87]</td><td colspan="1" rowspan="1">CMY-2:15.6%(181/1157)CMY-41:15.6%(181/1157)</td><td colspan="1" rowspan="1">CMY-2:9.7% of144CMY-41:4.9% of144</td></tr><tr><td colspan="1" rowspan="1">CMY</td><td colspan="1" rowspan="1">P. mirabilis</td><td colspan="1" rowspan="1">100.0%(63/63)[94.25-100.00]</td><td colspan="1" rowspan="1">98.9%(91/92)[94.10-99.81]</td><td colspan="1" rowspan="1">Ceftazidime</td><td colspan="1" rowspan="1">CMY:8.7%(2/23)[2.42-26.80]</td><td colspan="1" rowspan="1">CMY:94.5%(52/55)[85.15-98.13]</td><td colspan="1" rowspan="1">CMY:40.0%(2/5)[11.76-76.93]</td><td colspan="1" rowspan="1">CMY:71.2%(52/73)[59.99-80.35]</td><td colspan="1" rowspan="1">CMY-2:52.1%(135/259)[46.05-58.13]CMY-41:52.1%(135/259)[46.05-58.13]</td><td colspan="1" rowspan="1">CMY-2:94.9%(852/898)[93.23-6.14]CMY-41:94.9%%(852/898)[93.23-96.14]</td><td colspan="1" rowspan="1">CMY-2:74.6%(135/181)[67.78-800.37]CMY-41:74.6%(135/181)[67.78-80.37]</td><td colspan="1" rowspan="1">CMY-2:87.3%(852/976)[85.06-89.24]CMY-41:87.3%(852/976)[85.06-89.24]</td><td colspan="1" rowspan="1">CMY-2:15.6%(181/1157)CMY-41:15.6%(181/1157)</td><td colspan="1" rowspan="1">CMY-2:9.7% of144CMY-41:4.9% of144</td></tr><tr><td colspan="1" rowspan="1">CMY</td><td colspan="1" rowspan="1">P. mirabilis</td><td colspan="1" rowspan="1">100.0%(63/63)[94.25-100.00]</td><td colspan="1" rowspan="1">98.9%(91/92)[94.10-99.81]</td><td colspan="1" rowspan="1">Ceftriaxone</td><td colspan="1" rowspan="1">CMY:7.1%(3/42)[2.46-19.01]</td><td colspan="1" rowspan="1">CMY:94.4%(34/36)[81.86-98.46]</td><td colspan="1" rowspan="1">CMY:60.0%(3/5)[23.07-88.24]</td><td colspan="1" rowspan="1">CMY:46.6%(34/73)[35.59-57.90]</td><td colspan="1" rowspan="1">CMY-2:33.9%(162/478)[29.79-38.25]CMY-41:33.9%(162/478)[29.79-38.25]</td><td colspan="1" rowspan="1">CMY-2:97.2%(660/679)[95.67-98.20]CMY-41:97.2%(660/679)[95.67-98.20]</td><td colspan="1" rowspan="1">CMY-2:89.5%(162/181)[84.19-93.18]CMY-41:89.5%(162/181)[84.19-93.18]</td><td colspan="1" rowspan="1">CMY-2:67.6%(660/976)[64.62-70.48]CMY-41:67.6%(660/976)[64.62-70.48]</td><td colspan="1" rowspan="1">CMY-2:15.6%(181/1157)CY-41:15.6%(181/1157)</td><td colspan="1" rowspan="1">CMY-2:9.7% of144CMY-41:4.9% of144</td></tr><tr><td colspan="1" rowspan="2">AMR GeneTarget</td><td colspan="1" rowspan="2">Organism</td><td colspan="2" rowspan="1">Detection of AMR GeneTarget by AMR GenePanel versus WholeGenome SequencingCombined Clinical DevicePerformance-Stock and ProspectiveIsolates (from K191288)</td><td colspan="1" rowspan="2">Antimicrobial</td><td colspan="4" rowspan="1">Detection of AMR Gene Target by AMR GenePanel vs AST Non-SusceptibilityCombined Clinical Device Performance - Stock andivaonly unique isolates (no replicates)</td><td colspan="5" rowspan="1">Detection of AMR Gene Target versus AST Non-Susceptibility inOpGen's SMART Study</td><td colspan="1" rowspan="2">%Organismswith AMRGeneTarget inNCBIPathogenDatabase(2019)</td></tr><tr><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1">PPV[95% CI]</td><td colspan="1" rowspan="1">NPV[95% CI]</td><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1">PPV[95% CI]</td><td colspan="1" rowspan="1">NPV[95% CI]</td><td colspan="1" rowspan="1">% Organismswith AMRGene Target</td></tr><tr><td colspan="1" rowspan="1">CMY</td><td colspan="1" rowspan="1">P. mirabilis</td><td colspan="1" rowspan="1">100.0%(63/63)[94.25-100.00]</td><td colspan="1" rowspan="1">98.9%(91/92)[94.10-99.81]</td><td colspan="1" rowspan="1">Ertapenem</td><td colspan="1" rowspan="1">CMY:0.0%(0/4)[0.00-48.99]</td><td colspan="1" rowspan="1">CMY:93.2%(69/74)[85.14-97.08]</td><td colspan="1" rowspan="1">CMY:0.0%(0/5)[0.00-43.45]</td><td colspan="1" rowspan="1">CMY:94.5%(69/73)[86.74-97.85]</td><td colspan="1" rowspan="1">CMY-2:37.5%(6/16)[18.48-61.36]CMY-41:37.5%(6/16)[18.48-61.36]</td><td colspan="1" rowspan="1">CMY-2:84.7%(966/1141)[82.46-86.64]CMY-41:84.7%(966/1141)[82.46-86.64]</td><td colspan="1" rowspan="1">CMY-2:3.3%(6/181)[1.53-7.04]CMY-41:3.3%(6/181)[1.53-7.04]</td><td colspan="1" rowspan="1">CMY-2:99.0%(966/976)[98.12-99.4]CMY-41:99.0%(966/976)[98.12-99.44]</td><td colspan="1" rowspan="1">CMY-2:15.6%(181/1157)CMY-41:15.6%(181/1157)</td><td colspan="1" rowspan="1">CMY-2:9.7% of144CMY-41:4.9% of144</td></tr><tr><td colspan="1" rowspan="1">CMY</td><td colspan="1" rowspan="1">P. mirabilis</td><td colspan="1" rowspan="1">100.0%(63/63)[94.25-100.00]</td><td colspan="1" rowspan="1">98.9%(91/92)[94.10-99.81]</td><td colspan="1" rowspan="1">Meropenem</td><td colspan="1" rowspan="1">CMY:0.0%(0/3)[0.00-56.15]</td><td colspan="1" rowspan="1">CMY:93.3%(70/75)[85.32-97.12]</td><td colspan="1" rowspan="1">CMY:0.0%0/5)[0.00-43.45]</td><td colspan="1" rowspan="1">CMY:95.9%(70/73)[88.60-98.59]</td><td colspan="1" rowspan="1">CMY-2:41.2%(7/17)[21.61-63.99]CMY-41:41.2%(7/17)[21.61-63.99]</td><td colspan="1" rowspan="1">CMY-2:84.7%66/1140)[82.53-86.71]CMY-41:84.7%(966/1140)[82.53-86.71]</td><td colspan="1" rowspan="1">CMY-2:3.9%(7/181)[1.89-7.77]CMY-41:3.9%(7/181)[1.89-7.77]</td><td colspan="1" rowspan="1">CMY-2:99.0%(966/976)[98.12-99.44]CMY-41:99.0%(966/976)[98.12-99.44]</td><td colspan="1" rowspan="1">CMY-2:15.6%(181/1157)CMY-41:15.6%(181/1157)</td><td colspan="1" rowspan="1">CMY-2:9.7% of144CMY-41:4.9% of144</td></tr><tr><td colspan="1" rowspan="1">CMY</td><td colspan="1" rowspan="1">P. mirabilis</td><td colspan="1" rowspan="1">100.0%(63/63)[94.25-100.00]</td><td colspan="1" rowspan="1">98.9%(91/92)[94.10-99.81]</td><td colspan="1" rowspan="1">Amox.K.Clav</td><td colspan="1" rowspan="1">CMY:15.4%(4/26)[6.15-33.53]</td><td colspan="1" rowspan="1">CMY:98.1%(51/52)[89.88-99.66]</td><td colspan="1" rowspan="1">CMY:80.0%(4/5)[37.55-96.38]</td><td colspan="1" rowspan="1">CMY:69.9%(51/73)[58.56-79.18]</td><td colspan="1" rowspan="1">CMY-2:49.6%(178/359)[44.44-54.73]CMY-41:49.6%(178/359)[44.44-54.73]</td><td colspan="1" rowspan="1">CMY-2:99.6%(795/798)[98.90-99.87]CMY-41:99.6%(795/798)[98.90-9.87]</td><td colspan="1" rowspan="1">CMY-2:98.3%(178/181)[95.24-99.43]CMY-41:98.3%(178/181)[95.24-99.43]</td><td colspan="1" rowspan="1">CMY-2:81.5%(795/976)[78.89-83.77]CMY-41:81.5%(795/976)[78.89-83.77]</td><td colspan="1" rowspan="1">CMY-2:15.6%(181/1157)CMY-41:15.6%(181/1157)</td><td colspan="1" rowspan="1">CMY-2:9.7% of144CMY-41:4.9% of144</td></tr><tr><td colspan="1" rowspan="2">AMR GeneTarget</td><td colspan="1" rowspan="2">Organism</td><td colspan="2" rowspan="1">Detection of AMR GeneTarget by AMR GenePanel versus WholeGenome SequencingCombined Clinical DevicePerformance -Stock and ProspectiveIsolates (from K191288)</td><td colspan="1" rowspan="2">Antimicrobial</td><td colspan="4" rowspan="1">Detection of AMR Gene Target by AMR GenePanel vs AST Non-SusceptibilityCombined Clinical Device Performance - Stock andonly unique isolates (no replicates)</td><td colspan="5" rowspan="1">Detection of AMR Gene Target versus AST Non-Susceptibility inOpGen's SMART Study</td><td colspan="1" rowspan="2">%Organismswith AMRGeneTarget inNCBIPathogenDatabase(2019)</td></tr><tr><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1">PPV[95% CI]</td><td colspan="1" rowspan="1">NPV[95% CI]</td><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1">PPV[95% CI]</td><td colspan="1" rowspan="1">NPV[95% CI]</td><td colspan="1" rowspan="1">% Organismswith AMRGene Target</td></tr><tr><td colspan="1" rowspan="1">CMY</td><td colspan="1" rowspan="1">P. mirabilis</td><td colspan="1" rowspan="1">100.0%(63/63)[94.25-100.00]</td><td colspan="1" rowspan="1">98.9%(91/92)[94.10-99.81]</td><td colspan="1" rowspan="1">Amp.Sulbactam</td><td colspan="1" rowspan="1">CMY:9.5%(4/42)[3.77-22.07]</td><td colspan="1" rowspan="1">CMY:97.2%(35/36)[85.83-9.51]</td><td colspan="1" rowspan="1">CMY:80.0%(4/5)[37.55-96.38]</td><td colspan="1" rowspan="1">CMY:47.9%(35/73)[36.88-9.22]</td><td colspan="1" rowspan="1">CMY-2:30.4%(177/582)[26.81-34.27]CMY-41:30.4%(177/582)[26.81-34.27]</td><td colspan="1" rowspan="1">CMY-2:99.3%(571/575)[98.23-99.73]CMY-41:99.3%(571/575)[98.23-99.73]</td><td colspan="1" rowspan="1">CMY-2:97.8%(177/181)[94.46-99.14]CMY-41:97.8%(177/181)[94.46-99.14]</td><td colspan="1" rowspan="1">CMY-2:58.5%(571/976)[55.39-61.56]CMY-41:58.5%(571/976)[55.39-61.56]</td><td colspan="1" rowspan="1">CMY-2:15.6%(181/1157)CMY-41:15.6%(181/1157)</td><td colspan="1" rowspan="1">CMY-2:9.7% of144CMY-41:4.9% of144</td></tr><tr><td colspan="1" rowspan="1">CMY</td><td colspan="1" rowspan="1">P. mirabilis</td><td colspan="1" rowspan="1">100.0%(63/63)[94.25-100.00]</td><td colspan="1" rowspan="1">98.9%(91/92)[94.10-99.81]</td><td colspan="1" rowspan="1">Pip.Tazo</td><td colspan="1" rowspan="1">CMY:0.0%(0/4)[0.00-48.99]</td><td colspan="1" rowspan="1">CMY:93.2%(69/74)[85.14-97.08]</td><td colspan="1" rowspan="1">CMY:0.0%%(/5)[0.00-4.45]</td><td colspan="1" rowspan="1">CMY:94.5%(69/73)[86.74-997.85]</td><td colspan="1" rowspan="1">CMY-2:32.3%(10/31)[18.57-49.86]CMY-41:32.3%(10/31)[18.57-49.86]</td><td colspan="1" rowspan="1">CMY-2:84.8%(955/1126)[82.60-86.79]CMY-41:84.8%(955/1126)[82.60-86.79]</td><td colspan="1" rowspan="1">CMY-2:5.5%(10/181)[3.03-9.87]CMY-41:5.5%(10/181)[3.03-9.87]</td><td colspan="1" rowspan="1">CMY-2:97.8%(955/976)[96.73-98.59]CMY-41:97.8%(955/976)[96.73-98.59]</td><td colspan="1" rowspan="1">CMY-2:15.6%(181/1157)CMY-41:15.6%(181/1157)</td><td colspan="1" rowspan="1">CMY-2:9.7% of144CMY-41:4.9% of144</td></tr><tr><td colspan="1" rowspan="1">CTX-M-1</td><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">97.0%(65/67)[89.75-99.18]</td><td colspan="1" rowspan="1">97.0%(260/268)[94.22-98.48]</td><td colspan="1" rowspan="1">Cefazolin</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">CTX-M-1:54.1%(1194/2209) [51.97-56.12]</td><td colspan="1" rowspan="1">CTX-M-1:94.7%11/751)[92.83-96.06]</td><td colspan="1" rowspan="1">CTX-M-1:96.8%(1194/1234) [95.62-97.61]</td><td colspan="1" rowspan="1">CTX-M-1:41.2%(711/1726)[38.89-43.53]</td><td colspan="1" rowspan="1">CTX-M-1:41.7%(1234/2960)</td><td colspan="1" rowspan="1">CTX-M-1:11.8% of19017</td></tr><tr><td colspan="1" rowspan="1">CTX-M-1</td><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">97.0%65/67)[89.75-99.18]</td><td colspan="1" rowspan="1">97.0%(260/268)[94.22-98.48]</td><td colspan="1" rowspan="1">Cefuroxime</td><td colspan="1" rowspan="1">CTX-M-1:34.6%(47/136)[27.09-42.88]</td><td colspan="1" rowspan="1">CTX-M-1:100.0%(23/23)[85.69-100.00]</td><td colspan="1" rowspan="1">CTX-M-1:100.0%(47/47)[92.44-100.00]</td><td colspan="1" rowspan="1">CTX-M-1:20.5%(23/112)[14.09-28.93]</td><td colspan="1" rowspan="1">CTX-M-1:54.6%(1195/2187) [52.55-56.72]</td><td colspan="1" rowspan="1">CTX-M-1:95.0%(734/773)[93.18-966.29]</td><td colspan="1" rowspan="1">CTX-M-1:96.8%(1195/1234) [95.71-97.68]</td><td colspan="1" rowspan="1">CTX-M-1:42.5%(734/1726)[40.21-44.87]</td><td colspan="1" rowspan="1">CTX-M-1:41.7%(1234/2960)</td><td colspan="1" rowspan="1">CTX-M-1:11.8% of19017</td></tr><tr><td colspan="1" rowspan="1">CTX-M-1</td><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">97.0%(65/67)89.759.18]</td><td colspan="1" rowspan="1">97.0%(260/268)[94.22-98.48]</td><td colspan="1" rowspan="1">Cefepime</td><td colspan="1" rowspan="1">CTX-M-1:43.0%(46/107)[34.01-52.45]</td><td colspan="1" rowspan="1">CTX-M-1:98.1%(51/52)[89.8899.66]</td><td colspan="1" rowspan="1">CTX-M-1:97.9%(46/47)[88.8999.62]</td><td colspan="1" rowspan="1">CTX-M-1:45.5%(51/1)[36.61-54.75]</td><td colspan="1" rowspan="1">CTX-M-1:58.1%(1168/200) [5.97-60.28]</td><td colspan="1" rowspan="1">CTX-M-1:93.1%(85/951)[91.27-94.51]</td><td colspan="1" rowspan="1">CTX-M-1:94.7%(1168/1234) [93.25-95.7]</td><td colspan="1" rowspan="1">CTX-M-1:51.3%(885/1726)[48.92-53.63]</td><td colspan="1" rowspan="1">CTX-M-1:41.7%(1234/2960)</td><td colspan="1" rowspan="1">CTX-M-1:11.8% of19017</td></tr><tr><td colspan="1" rowspan="2">AMR GeneTarget</td><td colspan="1" rowspan="2">Organism</td><td colspan="2" rowspan="1">Detection of AMR GeneTarget by AMR GenePanel versus WholeGenome SequencingCombined Clinical DevicePerformance -Stock and ProspectiveIsolates (from K191288)</td><td colspan="1" rowspan="2">Antimicrobial</td><td colspan="4" rowspan="1">Detection of AMR Gene Target by AMR GenePanel vs AST Non-SusceptibilityCombined Clinical Device Performance - Stock andProspective Isolates (from K191288) includingonly unique isolates (no replicates)</td><td colspan="5" rowspan="1">Detection of AMR Gene Target versus AST Non-Susceptibility inOpGen's SMART Study</td><td colspan="1" rowspan="2">%Organismswith AMRGeneTarget inNCBIPathogenDatabase(2019)</td></tr><tr><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1">PPV[95% CI]</td><td colspan="1" rowspan="1">NPV[95% CI]</td><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1">PPV[95% CI]</td><td colspan="1" rowspan="1">NPV[95% CI]</td><td colspan="1" rowspan="1">% Organismswith AMRGene Target</td></tr><tr><td colspan="1" rowspan="1">CTX-M-1</td><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">97.0%(65/67)[89.75-9.18]</td><td colspan="1" rowspan="1">97.0%(260/268)[94.22-98.48]</td><td colspan="1" rowspan="1">Ceftazidime</td><td colspan="1" rowspan="1">CTX-M-1:41.5%(44/106)[32.59-51.03]</td><td colspan="1" rowspan="1">CTX-M-1:94.3%(50/53)[84.63-98.06]</td><td colspan="1" rowspan="1">CTX-M-1:93.6%(44/47)[82.84-97.81]</td><td colspan="1" rowspan="1">CTX-M-1:44.6%(50/112)[35.7753.87]</td><td colspan="1" rowspan="1">CTX-M-1:65.2%(1077/1651) [62.0-67.49]</td><td colspan="1" rowspan="1">CTX-M-1:88.0%(1152/1309) 86.13-89.66]</td><td colspan="1" rowspan="1">CTX-M-1:87.3%(1077/1234) (85.30-9.021]</td><td colspan="1" rowspan="1">CTX-M-1:66.7%(1152/1726) (6.49-68.93]</td><td colspan="1" rowspan="1">CTX-M-1:41.7%(1234/2960)</td><td colspan="1" rowspan="1">CTX-M-1:11.8% of19017</td></tr><tr><td colspan="1" rowspan="1">CTX-M-1</td><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">97.0%(65/67)[89.75-99.18]</td><td colspan="1" rowspan="1">97.0%(260/268)[94.22-98.48]</td><td colspan="1" rowspan="1">Ceftriaxone</td><td colspan="1" rowspan="1">CTX-M-1:36.2%(47/130)[28.40-44.70]</td><td colspan="1" rowspan="1">CTX-M-1:100.0%(29/29)[88.30-0.00]</td><td colspan="1" rowspan="1">CTX-M-1:100.0%(47/47)[92.44-100.00]</td><td colspan="1" rowspan="1">CTX-M-1:25.9%(29/112)[18.68-34.71]</td><td colspan="1" rowspan="1">CTX-M-1:56.5%(1194/2113) [54.38-58.61]</td><td colspan="1" rowspan="1">CTX-M-1:95.3%(807/847)[93.63-6.51]</td><td colspan="1" rowspan="1">CTX-M-1:96.8%(1194/1234) [95.62-7.61]</td><td colspan="1" rowspan="1">CTX-M-1:46.8%(807/1726)[44.41-49.1]</td><td colspan="1" rowspan="1">CTX-M-1:41.7%(1234/2960)</td><td colspan="1" rowspan="1">CTX-M-1:11.8% of19017</td></tr><tr><td colspan="1" rowspan="1">CTX-M-1</td><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">97.0%(65/67)[89.599.18]</td><td colspan="1" rowspan="1">97.0%(260/268)[94.22-98.48]</td><td colspan="1" rowspan="1">Amox.K.Clav</td><td colspan="1" rowspan="1">CTX-M-1:35.1%(40/114)[26.94-44.20]</td><td colspan="1" rowspan="1">CTX-M-1:84.4%(38/45)[71.2292.25]</td><td colspan="1" rowspan="1">CTX-M-1:85.1%(40/47)[72.31-92.59]</td><td colspan="1" rowspan="1">CTX-M-1:33.9%(38/112)25.8243.10]</td><td colspan="1" rowspan="1">CTX-M-1:51.3%(928/1810)[48.97-53.57]</td><td colspan="1" rowspan="1">CTX-M-1:73.4%(844/1150)[70.76-75.86]</td><td colspan="1" rowspan="1">CTX-M-1:75.2%(928/1234)[72.72-77.53]</td><td colspan="1" rowspan="1">CTX-M-1:48.9%(844/1726)46.55-51.26]</td><td colspan="1" rowspan="1">CTX-M-1:41.7%(1234/2960)</td><td colspan="1" rowspan="1">CTX-M-1:11.8% of19017</td></tr><tr><td colspan="1" rowspan="1">CTX-M-1</td><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">97.0%(65/67)[89.75-99.18]</td><td colspan="1" rowspan="1">97.0%(260/268)[94.22-98.48]</td><td colspan="1" rowspan="1">Amp.Sulbactam</td><td colspan="1" rowspan="1">CTX-M-1:30.8%(45/146)[23.90-38.72]</td><td colspan="1" rowspan="1">CTX-M-1:84.6%(11/13)[57.77-95.67]</td><td colspan="1" rowspan="1">CTX-M-1:95.7%(45/47)[85.75-98.83]</td><td colspan="1" rowspan="1">CTX-M-1:9.8%(11/112)[5.57-16.73]</td><td colspan="1" rowspan="1">CTX-M-1:46.9%(1162/2477) [44.95-48.8</td><td colspan="1" rowspan="1">CTX-M-1:85.1%(411/483)[81.64-87.99]</td><td colspan="1" rowspan="1">CTX-M-1:94.2%(1162/1234) [92.72-95.34]</td><td colspan="1" rowspan="1">CTX-M-1:23.8%(411/1726)[21.86-25.88]</td><td colspan="1" rowspan="1">CTX-M-1:41.7%(1234/2960)</td><td colspan="1" rowspan="1">CTX-M-1:11.8% of19017</td></tr><tr><td colspan="1" rowspan="1">CTX-M-1</td><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">97.0%(65/67)[89.599.18]</td><td colspan="1" rowspan="1">97.0%(260/268)[94.22-98.48]</td><td colspan="1" rowspan="1">Pip.Tazo</td><td colspan="1" rowspan="1">CTX-M-1:39.1%(27/69)[28.48-50.93]</td><td colspan="1" rowspan="1">CTX-M-1:77.8%(70/90)6.1585.13]</td><td colspan="1" rowspan="1">CTX-M-1:57.4%(27/47)[43.28-70.49]</td><td colspan="1" rowspan="1">CTX-M-1:62.5%(70/112)[53.26-70.91]</td><td colspan="1" rowspan="1">CTX-M-1:53.7%(369/687)49.97-57.41]</td><td colspan="1" rowspan="1">CTX-M-1:61.9%(1408/227) [59.93-63.92</td><td colspan="1" rowspan="1">CTX-M-1:29.9%(369/1234)[27.41-32.52]</td><td colspan="1" rowspan="1">CTX-M-1:81.6%(1408/172) (79.68-83.33]</td><td colspan="1" rowspan="1">CTX-M-1:41.7%(1234/2960)</td><td colspan="1" rowspan="1">CTX-M-1:11.8% of19017</td></tr><tr><td colspan="1" rowspan="1">CTX-M-1</td><td colspan="1" rowspan="1">K. pneumoniae</td><td colspan="1" rowspan="1">96.1%(174/181)[92.23-98.11]</td><td colspan="1" rowspan="1">100.0%(148/148)[97.47-100.00]</td><td colspan="1" rowspan="1">Cefazolin</td><td colspan="1" rowspan="1">NotEvaluated</td><td colspan="1" rowspan="1">NotEvaluated</td><td colspan="1" rowspan="1">NotEvaluated</td><td colspan="1" rowspan="1">NotEvaluated</td><td colspan="1" rowspan="1">CTX-M-1:60.6%(1106/182) [58.37-62.85]</td><td colspan="1" rowspan="1">CTX-M-1:95.6%(195/204)[91.83-97.66]</td><td colspan="1" rowspan="1">CTX-M-1:99.2%(1106/1115) (98.47-99.7]</td><td colspan="1" rowspan="1">CTX-M-1:21.4%(195/913)[18.82-24.13]</td><td colspan="1" rowspan="1">CTX-M-1:55%(1115/2028)</td><td colspan="1" rowspan="1">CTX-M-1:34.6% of7979</td></tr><tr><td colspan="1" rowspan="1">CTX-M-1</td><td colspan="1" rowspan="1">K. pneumoniae</td><td colspan="1" rowspan="1">96.1%(174/181)[92.23-98.11]</td><td colspan="1" rowspan="1">100.0%(148/148)[97.47-00]</td><td colspan="1" rowspan="1">Cefuroxime</td><td colspan="1" rowspan="1">CTX-M-1:51.8%(88/170)[44.30-59.15]</td><td colspan="1" rowspan="1">CTX-M-1:100.0%(15/15)79.61-100.00]</td><td colspan="1" rowspan="1">CTX-M-1:100.0%(88/88)95.82100.00]</td><td colspan="1" rowspan="1">CTX-M-1:15.5%(15/97)[9.60-23.96]</td><td colspan="1" rowspan="1">CTX-M-1:60.3%(1104/183) [58.0-62.48]</td><td colspan="1" rowspan="1">CTX-M-1:94.4%(185/196)90.2396.84]</td><td colspan="1" rowspan="1">CTX-M-1:99.0%(1104/1115)) 98.24-99.45]</td><td colspan="1" rowspan="1">CTX-M-1:20.3%(185/913)[17.78-22.99]</td><td colspan="1" rowspan="1">CTX-M-1:55%(1115/2028)</td><td colspan="1" rowspan="1">CTX-M-1:34.6% of7979</td></tr><tr><td colspan="1" rowspan="2">AMR GeneTarget</td><td colspan="1" rowspan="2">Organism</td><td colspan="2" rowspan="1">Detection of AMR GeneTarget by AMR GenePanel versus WholeGenome SequencingCombined Clinical DevicePerformance -Stock and ProspectiveIsolates (from K191288)</td><td colspan="1" rowspan="1">Antimicrobial</td><td colspan="4" rowspan="1">Detection of AMR Gene Target by AMR GenePanel vs AST Non-SusceptibilityCombined Clinical Device Performance - Stock andProspective Isolates (from K191288) includingonly nique isolates (no replicates)</td><td colspan="5" rowspan="1">Detection of AMR Gene Target versus AST Non-Susceptibility inOpGen's SMART Study</td><td colspan="1" rowspan="1">%Organismswith AMRGeneTarget inNCBIPathogen</td></tr><tr><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1">PPV[95% CI]</td><td colspan="1" rowspan="1">NPV[95% CI]</td><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1">PPV[95% CI]</td><td colspan="1" rowspan="1">NPV[95% CI]</td><td colspan="1" rowspan="1">% Organismswith AMRGene Target</td><td colspan="1" rowspan="1">Database(2019)</td></tr><tr><td colspan="1" rowspan="1">CTX-M-1</td><td colspan="1" rowspan="1">K. pneumoniae</td><td colspan="1" rowspan="1">96.1%(174/181)[92.23-98.11]</td><td colspan="1" rowspan="1">100.0%(148/148)[97.47-100.00]</td><td colspan="1" rowspan="1">Cefepime</td><td colspan="1" rowspan="1">CTX-M-1:55.0%(88/160)[47.26-62.50]</td><td colspan="1" rowspan="1">CTX-M-1:100.0%(25/25)[86.68-100.00]</td><td colspan="1" rowspan="1">CTX-M-1:100.0%(88/88)95.82-100.00]</td><td colspan="1" rowspan="1">CTX-M-1:25.8%(25/97)[18.11-35.28]</td><td colspan="1" rowspan="1">CTX-M-1:63.8%(1091/1711) [61.46-6.01]</td><td colspan="1" rowspan="1">CTX-M-1:92.4%(293/317)[88.98-94.86]</td><td colspan="1" rowspan="1">CTX-M-1:97.8%(1091/1115) ([96.82-98.55]</td><td colspan="1" rowspan="1">CTX-M-1:32.1%(293/913)[29.14-35.19]</td><td colspan="1" rowspan="1">CTX-M-1:55%(1115/2028)</td><td colspan="1" rowspan="1">CTX-M-1:34.6% of7979</td></tr><tr><td colspan="1" rowspan="1">CTX-M-1</td><td colspan="1" rowspan="1">K. pneumoniae</td><td colspan="1" rowspan="1">96.1%(174/181)[92.23-98.11]</td><td colspan="1" rowspan="1">100.0%(148/148)[97.47-10.00]</td><td colspan="1" rowspan="1">Ceftazidime</td><td colspan="1" rowspan="1">CTX-M-1:54.0%(88/163)46.33-61.46]</td><td colspan="1" rowspan="1">CTX-M-1:100.0%(22/22)[85.13-0.00]</td><td colspan="1" rowspan="1">CTX-M-1:100.0%(88/88)[95.82-0.00]</td><td colspan="1" rowspan="1">CTX-M-1:22.7%(2/97)[15.48-31.96]</td><td colspan="1" rowspan="1">CTX-M-1:62.8%(1074/1710) [60.49-665.07]</td><td colspan="1" rowspan="1">CTX-M-1:87.1%(277/318)[82.98-90.35]</td><td colspan="1" rowspan="1">CTX-M-1:96.3%(1074/1115) [95.05-97.28]</td><td colspan="1" rowspan="1">CTX-M-1:30.3%(277/913)[27.45-33.40]</td><td colspan="1" rowspan="1">CTX-M-1:55%(1115/2028)</td><td colspan="1" rowspan="1">CTX-M-1:34.6% of7979</td></tr><tr><td colspan="1" rowspan="1">CTX-M-1</td><td colspan="1" rowspan="1">K. pneumoniae</td><td colspan="1" rowspan="1">96.1%(174/181)[92.23-98.11]</td><td colspan="1" rowspan="1">100.0%(148/148)[97.47-100.00]</td><td colspan="1" rowspan="1">Ceftriaxone</td><td colspan="1" rowspan="1">CTX-M-1:52.7%(88/167)45.1560.12]</td><td colspan="1" rowspan="1">CTX-M-1:0.0%(18/18)82.41-100.00]</td><td colspan="1" rowspan="1">CTX-M-1:100.0%(88/88)95.8100.00]</td><td colspan="1" rowspan="1">CTX-M-1:18.6%(18/97)12.07-27.44]</td><td colspan="1" rowspan="1">CTX-M-1:62.2%(1098/17655) 59.92-64.44]</td><td colspan="1" rowspan="1">CTX-M-1:93.5%(246/263)[89.8995.93]</td><td colspan="1" rowspan="1">CTX-M-1:98.5%(1098/1115)[97.5799.05]</td><td colspan="1" rowspan="1">CTX-M-1:26.9%(246/913)[24.17-29.91]</td><td colspan="1" rowspan="1">CTX-M-1:55%(1115/2028)</td><td colspan="1" rowspan="1">CTX-M-1:34.6% of7979</td></tr><tr><td colspan="1" rowspan="1">CTX-M-1</td><td colspan="1" rowspan="1">K. pneumoniae</td><td colspan="1" rowspan="1">96.1%(174/181)[92.23-98.11]</td><td colspan="1" rowspan="1">100.0%(148/148)[97.47-100.00]</td><td colspan="1" rowspan="1">Amox.K.Clav</td><td colspan="1" rowspan="1">CTX-M-1:50.9%(83/163)[43.31-58.48]</td><td colspan="1" rowspan="1">CTX-M-1:77.3%(17/22)[56.56-89.88]</td><td colspan="1" rowspan="1">CTX-M-1:94.3%(83/88)[87.38-97.55]</td><td colspan="1" rowspan="1">CTX-M-1:17.5%(17/97)[11.24-26.29]</td><td colspan="1" rowspan="1">CTX-M-1:59.7%25/1716) [57.39-2.03]</td><td colspan="1" rowspan="1">CTX-M-1:71.2%(222/312)[65.89-75.90]</td><td colspan="1" rowspan="1">CTX-M-1:91.9%(1025/1115) (90.18-93.39]</td><td colspan="1" rowspan="1">CTX-M-1:24.3%(222/913)[21.64-27.20]</td><td colspan="1" rowspan="1">CTX-M-1:55%(1115/2028)</td><td colspan="1" rowspan="1">CTX-M-1:34.6% of7979</td></tr><tr><td colspan="1" rowspan="1">CTX-M-1</td><td colspan="1" rowspan="1">K. pneumoniae</td><td colspan="1" rowspan="1">96.1%(174/181)[92.23-8.11]</td><td colspan="1" rowspan="1">100.0%(148/148)[97.47-100]</td><td colspan="1" rowspan="1">Amp.Sulbactam</td><td colspan="1" rowspan="1">CTX-M-1:49.7%(88/177)[42.44-57.01]</td><td colspan="1" rowspan="1">CTX-M-1:100.0%(8/8)67.56-100.00]</td><td colspan="1" rowspan="1">CTX-M-1:100.0%(88/88)95.82-100.00]</td><td colspan="1" rowspan="1">CTX-M-1:8.2%(8/97)[4.24-15.44]</td><td colspan="1" rowspan="1">CTX-M-1:58.0%(1108/190) [555.81-60.24]</td><td colspan="1" rowspan="1">CTX-M-1:94.1%(112/119)[88.35-97.12]</td><td colspan="1" rowspan="1">CTX-M-1:99.4%(1108/1115) [98.71-99.701]</td><td colspan="1" rowspan="1">CTX-M-1:12.3%(112/913)[10.30-14.55]</td><td colspan="1" rowspan="1">CTX-M-1:55%(1115/2028)</td><td colspan="1" rowspan="1">CTX-M-1:34.6% of7979</td></tr><tr><td colspan="1" rowspan="1">CTX-M-1</td><td colspan="1" rowspan="1">K. pneumoniae</td><td colspan="1" rowspan="1">96.1%(174/181)[92.23-98.11]</td><td colspan="1" rowspan="1">100.0%(148/148)[97.47-100.00]</td><td colspan="1" rowspan="1">Pip.Tazo</td><td colspan="1" rowspan="1">CTX-M-1:51.7%(76/147)[43.68-59.63]</td><td colspan="1" rowspan="1">CTX-M-1:68.4%(26/38)[52.54-80.92]</td><td colspan="1" rowspan="1">CTX-M-1:86.4%(76/88)77. 6-92.02]</td><td colspan="1" rowspan="1">CTX-M-1:26.8%(26/97)[19.00-36.38]</td><td colspan="1" rowspan="1">CTX-M-1:56.4%(789/1400)[53.75-58.93]</td><td colspan="1" rowspan="1">CTX-M-1:48.1%(302/628)[44.20-52.00]</td><td colspan="1" rowspan="1">CTX-M-1:70.8%(789/1115)[68.02-73.36]</td><td colspan="1" rowspan="1">CTX-M-1:33.1%(302/913)[30.10-36.19]</td><td colspan="1" rowspan="1">CTX-M-1:55%(1115/2028)</td><td colspan="1" rowspan="1">CTX-M-1:34.6% of7979</td></tr><tr><td colspan="1" rowspan="1">CTX-M-1</td><td colspan="1" rowspan="1">P. mirabilis</td><td colspan="1" rowspan="1">100.0%(12/12)[75.75-100.00]</td><td colspan="1" rowspan="1">100.0%(143/143)[97.38-10000]</td><td colspan="1" rowspan="1">Cefazolin</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">CTX-M-1:14.7%(84/571)[12.04-17.85]</td><td colspan="1" rowspan="1">CTX-M-1:99.8%(585/586)99.04-99.97]</td><td colspan="1" rowspan="1">CTX-M-1:98.8%(84/85)[93.63-99.79]</td><td colspan="1" rowspan="1">CTX-M-1:54.6%(585/1072)[51.58-57.53]</td><td colspan="1" rowspan="1">CTX-M-1:7.3%(85/1157)</td><td colspan="1" rowspan="1">CTX-M-1:2.1% of144</td></tr><tr><td colspan="1" rowspan="2">AMR GeneTarget</td><td colspan="1" rowspan="2">Organism</td><td colspan="2" rowspan="1">Detection of AMR GeneTarget by AMR GenePanel versus WholeGenome SequencingCombined Clinical DevicePerformance -Stock and ProspectiveIsolates (from K191288)</td><td colspan="1" rowspan="2">Antimicrobial</td><td colspan="4" rowspan="1">Detection of AMR Gene Target by AMR GenePanel vs AST Non-SusceptibilityCombined Clinical Device Performance - Stock andivonly unique isolates (no replicates)</td><td colspan="5" rowspan="1">Detection of AMR Gene Target versus AST Non-Susceptibility inOpGen's SMART Study</td><td colspan="1" rowspan="2">%Organismswith AMRGeneTarget inNCBIPathogenDatabase(2019)</td></tr><tr><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1">PPV[95% CI]</td><td colspan="1" rowspan="1">NPV[95% CI]</td><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1">PPV[95% CI]</td><td colspan="1" rowspan="1">NPV[95% CI]</td><td colspan="1" rowspan="1">% Organismswith AMRGene Target</td></tr><tr><td colspan="1" rowspan="1">CTX-M-1</td><td colspan="1" rowspan="1">P. mirabilis</td><td colspan="1" rowspan="1">100.0%(12/12)[75.75-100.00]</td><td colspan="1" rowspan="1">100.0%(143/143)[97.38-100.00]</td><td colspan="1" rowspan="1">Cefuroxime</td><td colspan="1" rowspan="1">CTX-M-1:24.4%(11/45)[14.24-38.67]</td><td colspan="1" rowspan="1">CTX-M-1:100.0%(33/33)[89.57-10.00]</td><td colspan="1" rowspan="1">CTX-M-1:100.0%(11/11)[74.12-100.00]</td><td colspan="1" rowspan="1">CTX-M-1:49.3%(33/67)[37.65-60.4]</td><td colspan="1" rowspan="1">CTX-M-1:17.0%(84/495)[13.92-20.53]</td><td colspan="1" rowspan="1">CTX-M-1:99.8%(661/662)[99.15-99.97]</td><td colspan="1" rowspan="1">CTX-M-1:98.8%(84/85)[93.63-99.79]</td><td colspan="1" rowspan="1">CTX-M-1:61.7%(661/1072)[58.71-64.52]</td><td colspan="1" rowspan="1">CTX-M-1:7.3%(85/1157)</td><td colspan="1" rowspan="1">CTX-M-1:2.1% of144</td></tr><tr><td colspan="1" rowspan="1">CTX-M-1</td><td colspan="1" rowspan="1">P. mirabilis</td><td colspan="1" rowspan="1">100.0%(12/12)[75.75-10.00]</td><td colspan="1" rowspan="1">100.0%(143/143)[97.38-100.00]</td><td colspan="1" rowspan="1">Cefepime</td><td colspan="1" rowspan="1">CTX-M-1:27.5%(11/40)[16.11-42.83]</td><td colspan="1" rowspan="1">CTX-M-1:100.0%(38/38)90.82-10.00]</td><td colspan="1" rowspan="1">CTX-M-1:100.0%(11/11)[74.12-100.00]</td><td colspan="1" rowspan="1">CTX-M-1:56.7%(38/67)44.8167.90]</td><td colspan="1" rowspan="1">CTX-M-1:23.4%(78/334)[19.13-28.18</td><td colspan="1" rowspan="1">CTX-M-1:99.1%(816/823)[98.25-99.59]</td><td colspan="1" rowspan="1">CTX-M-1:91.8%(78/85)[83.96-5.95]</td><td colspan="1" rowspan="1">CTX-M-1:76.1%(816/1072)[73.48-78.581</td><td colspan="1" rowspan="1">CTX-M-1:7.3%(85/1157)</td><td colspan="1" rowspan="1">CTX-M-1:2.1% of144</td></tr><tr><td colspan="1" rowspan="1">CTX-M-1</td><td colspan="1" rowspan="1">P. mirabilis</td><td colspan="1" rowspan="1">100.0%(12/12)[75.75-00]</td><td colspan="1" rowspan="1">100.0%(143/143)[97.38-0.00]</td><td colspan="1" rowspan="1">Ceftazidime</td><td colspan="1" rowspan="1">CTX-M-1:17.4%(4/23)[6.98-37.14]</td><td colspan="1" rowspan="1">CTX-M-1:87.3%(48/55)[75.98-93.701</td><td colspan="1" rowspan="1">CTX-M-1:36.4%(4/11)[15.17-64.62]</td><td colspan="1" rowspan="1">CTX-M-1:71.6%(48/67)59.91-81.03]</td><td colspan="1" rowspan="1">CTX-M-1:7.3%(19/259)[4.75-11.17]</td><td colspan="1" rowspan="1">CTX-M-1:92.7%(832/898)[90.76-94.18]</td><td colspan="1" rowspan="1">CTX-M-1:22.4%(19/85)14.80-32.291</td><td colspan="1" rowspan="1">CTX-M-1:77.6%(832/1072)75.02-80.01]</td><td colspan="1" rowspan="1">CTX-M-1:7.3%(85/1157)</td><td colspan="1" rowspan="1">CTX-M-1:2.1% of144</td></tr><tr><td colspan="1" rowspan="1">CTX-M-1</td><td colspan="1" rowspan="1">P. mirabilis</td><td colspan="1" rowspan="1">100.0%(12/12)75.75-100.00]</td><td colspan="1" rowspan="1">100.0%(143/143)[97.38-100.00]</td><td colspan="1" rowspan="1">Ceftriaxone</td><td colspan="1" rowspan="1">CTX-M-1:26.2%(11/42)15.30-41.07]</td><td colspan="1" rowspan="1">CTX-M-1:100.0%(36/36)[90.36-10.00]</td><td colspan="1" rowspan="1">CTX-M-1:100.0%(11/11)[74.12-100.00]</td><td colspan="1" rowspan="1">CTX-M-1:53.7%(36/67)41.92-65.14]</td><td colspan="1" rowspan="1">CTX-M-1:17.6%(84/478)[14.42-21.24]</td><td colspan="1" rowspan="1">CTX-M-1:99.9%(678/679)[99.17-99.9</td><td colspan="1" rowspan="1">CTX-M-1:98.8%(84/85)[93.63-99.79]</td><td colspan="1" rowspan="1">CTX-M-1:63.2%(678/1072)[60.32-66.08]</td><td colspan="1" rowspan="1">CTX-M-1:7.3%(85/1157)</td><td colspan="1" rowspan="1">CTX-M-1:2.1% of144</td></tr><tr><td colspan="1" rowspan="1">CTX-M-1</td><td colspan="1" rowspan="1">P. mirabilis</td><td colspan="1" rowspan="1">100.0%(12/12)[75.75-00]</td><td colspan="1" rowspan="1">100.0%(143/143)[97.38-00]</td><td colspan="1" rowspan="1">Amox.K.Clav</td><td colspan="1" rowspan="1">CTX-M-1:15.4%(4/26)[6.15-33.53]</td><td colspan="1" rowspan="1">CTX-M-1:86.5%(45/52)[74.73-93.32]</td><td colspan="1" rowspan="1">CTX-M-1:36.4%(4/11)[15.17-64.62]</td><td colspan="1" rowspan="1">CTX-M-1:67.2%(45/67)[55.26-77.21]</td><td colspan="1" rowspan="1">CTX-M-1:8.9%(32/359)[6.38-12.31]</td><td colspan="1" rowspan="1">CTX-M-1:93.4%(745/798)91.41-94.89]</td><td colspan="1" rowspan="1">CTX-M-1:37.6%(32/85)28.09-48.27]</td><td colspan="1" rowspan="1">CTX-M-1:69.5%(745/1072)[66.67-72.18]</td><td colspan="1" rowspan="1">CTX-M-1:7.3%(85/1157)</td><td colspan="1" rowspan="1">CTX-M-1:2.1% of144</td></tr><tr><td colspan="1" rowspan="1">CTX-M-1</td><td colspan="1" rowspan="1">P. mirabilis</td><td colspan="1" rowspan="1">100.0%(12/12)75.75-100.00]</td><td colspan="1" rowspan="1">100.0%(143/143)[97.38-100.00]</td><td colspan="1" rowspan="1">Amp.Sulbactam</td><td colspan="1" rowspan="1">CTX-M-1:19.0%(8/42)[9.98-33.30]</td><td colspan="1" rowspan="1">CTX-M-1:91.7%(33/36)[78.17-97.13]</td><td colspan="1" rowspan="1">CTX-M-1:72.7%(8/11))[43.44-90.25]</td><td colspan="1" rowspan="1">CTX-M-1:49.3%(33/67)[37.65-60.94]</td><td colspan="1" rowspan="1">CTX-M-1:11.3%(66/82)[9.01-14.17]</td><td colspan="1" rowspan="1">CTX-M-1:96.7%(556/575)[94.90-97.87]</td><td colspan="1" rowspan="1">CTX-M-1:77.6%66/85)[67.71-85.20]</td><td colspan="1" rowspan="1">CTX-M-1:51.9%(56/1072)[48.87-54.84]</td><td colspan="1" rowspan="1">CTX-M-1:7.3%(85/1157)</td><td colspan="1" rowspan="1">CTX-M-1:2.1% of144</td></tr><tr><td colspan="1" rowspan="1">CTX-M-1</td><td colspan="1" rowspan="1">P. mirabilis</td><td colspan="1" rowspan="1">100.0%(12/12)[75.75-100.00]</td><td colspan="1" rowspan="1">100.0%(143/143)[97.38-100.00]</td><td colspan="1" rowspan="1">Pip.Tazo</td><td colspan="1" rowspan="1">CTX-M-1:50.0%(2/4)[15.00-85.00]</td><td colspan="1" rowspan="1">CTX-M-1:87.8%65/74)[78.47-93.47]</td><td colspan="1" rowspan="1">CTX-M-1:18.2%(2/11)5.14-47.70]</td><td colspan="1" rowspan="1">CTX-M-1:97.0%(65/67)89.599.18]</td><td colspan="1" rowspan="1">CTX-M-1:16.1%(5/31)7.09-32.63]</td><td colspan="1" rowspan="1">CTX-M-1:92.9%(1046/1126) [91.24-94.25]</td><td colspan="1" rowspan="1">CTX-M-1:5.9%(/85)[2.54-13.04]</td><td colspan="1" rowspan="1">CTX-M-1:97.6%(1046/1072) [96.47-98.34]</td><td colspan="1" rowspan="1">CTX-M-1:7.3%(85/1157)</td><td colspan="1" rowspan="1">CTX-M-1:2.1% of144</td></tr><tr><td colspan="1" rowspan="2">AMR GeneTarget</td><td colspan="1" rowspan="2">Organism</td><td colspan="2" rowspan="1">Detection of AMR GeneTarget by AMR GenePanel versus WholeGenome SequencingCombined Clinical DevicePerformance -Stock and ProspectiveIsolates (from K191288)</td><td colspan="1" rowspan="2">Antimicrobial</td><td colspan="4" rowspan="1">Detection of AMR Gene Target by AMR GenePanel vs AST Non-SusceptibilityCombined Clinical Device Performance - Stock andProspective Isolates (from K191288) includingonly unique isolates (no replicates)</td><td colspan="5" rowspan="1">Detection of AMR Gene Target versus AST Non-Susceptibility inOpGen's SMART Study</td><td colspan="1" rowspan="2">%Organismswith AMRGeneTarget inNCBIPathogenDatabase(2019)</td></tr><tr><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1">PPV[95% CI]</td><td colspan="1" rowspan="1">NPV[95% CI]</td><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1">PPV[95% CI]</td><td colspan="1" rowspan="1">NPV[95% CI]</td><td colspan="1" rowspan="1">% Organismswith AMRGene Target</td></tr><tr><td colspan="1" rowspan="1">CTX-M-1</td><td colspan="1" rowspan="1">P. aeruginosa</td><td colspan="1" rowspan="1">100.0%(2/2)[34.24-100.00]</td><td colspan="1" rowspan="1">99.7%(345/346)[98.38-9.95]</td><td colspan="1" rowspan="1">Cefepime</td><td colspan="1" rowspan="1">CTX-M-1:1.6%(2/123)[0.45-5.74]</td><td colspan="1" rowspan="1">CTX-M-1:100.0%(32/32)[89.28-100]</td><td colspan="1" rowspan="1">CTX-M-1:100.0%(2/2)[34.24-100.00]</td><td colspan="1" rowspan="1">CTX-M-1:20.9%(32/153)[15.22-28.03]</td><td colspan="1" rowspan="1">CTX-M-1:0.4%(4/1068)[0.15-0.96]</td><td colspan="1" rowspan="1">CTX-M-1:100.0%(482/482)[99.21-0.00]</td><td colspan="1" rowspan="1">CTX-M-1:100.0%(4/4)[51.01-10.00]</td><td colspan="1" rowspan="1">CTX-M-1:31.2%(482/1546)[28.92-33.53]</td><td colspan="1" rowspan="1">CTX-M-1:0.3%(4/1550)</td><td colspan="1" rowspan="1">CTX-M-1:0% of4706</td></tr><tr><td colspan="1" rowspan="1">CTX-M-1</td><td colspan="1" rowspan="1">P. aeruginosa</td><td colspan="1" rowspan="1">100.0%(2/2)[34.24-100.00]</td><td colspan="1" rowspan="1">99.7%(345/346)[98.38-99.95]</td><td colspan="1" rowspan="1">Ceftazidime</td><td colspan="1" rowspan="1">CTX-M-1:1.8%((2/114)[0.48-6.17]</td><td colspan="1" rowspan="1">CTX-M-1:100.0%(41/41)[91.43-10.00]</td><td colspan="1" rowspan="1">CTX-M-1:100.0%(2/2)[34.24-10.00]</td><td colspan="1" rowspan="1">CTX-M-1:26.8%1/153)[20.41-34.32]</td><td colspan="1" rowspan="1">CTX-M-1:0.4%(4/959)[0.16-1.07]</td><td colspan="1" rowspan="1">CTX-M-1:100.0%(591/591)[99.35-10.00]</td><td colspan="1" rowspan="1">CTX-M-1:100.0%(4/4)[51.01-10.00]</td><td colspan="1" rowspan="1">CTX-M-1:38.2%(591/1546)[35.84-40.68]</td><td colspan="1" rowspan="1">CTX-M-1:0.3%(41550)</td><td colspan="1" rowspan="1">CTX-M-1:0% of4706</td></tr><tr><td colspan="1" rowspan="1">CTX-M-1</td><td colspan="1" rowspan="1">P. aeruginosa</td><td colspan="1" rowspan="1">100.0%(2/2)[34.24-10.00]</td><td colspan="1" rowspan="1">99.7%(345/346)[98.38-9.95]</td><td colspan="1" rowspan="1">Pip.Tazo</td><td colspan="1" rowspan="1">CTX-M-1:1.9%(2/104)[0.53-6.74]</td><td colspan="1" rowspan="1">CTX-M-1:100.0%(51/51)93.00-0.00]</td><td colspan="1" rowspan="1">CTX-M-1:100.0%(2/2)[34.24-0.00]</td><td colspan="1" rowspan="1">CTX-M-1:33.3%(51/153)26.35-41.13]</td><td colspan="1" rowspan="1">CTX-M-1:0.3%(3/941)[0.11-0.93]</td><td colspan="1" rowspan="1">CTX-M-1:99.8%(608/609)9.08-99.97]</td><td colspan="1" rowspan="1">CTX-M-1:75.0%(3/4)30.06-95.44]</td><td colspan="1" rowspan="1">CTX-M-1:39.3%(608/1546)[36.92-41.79]</td><td colspan="1" rowspan="1">CTX-M-1:0.3%(4/1550)</td><td colspan="1" rowspan="1">CTX-M-1:0% of4706</td></tr><tr><td colspan="1" rowspan="1">CTX-M-1</td><td colspan="1" rowspan="1">K. aerogenes</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">Cefuroxime</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">CTX-M-1:3.1% of223</td></tr><tr><td colspan="1" rowspan="1">CTX-M-1</td><td colspan="1" rowspan="1">K. aerogenes</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">Cefepime</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">CTX-M-1:3.1% of223</td></tr><tr><td colspan="1" rowspan="1">CTX-M-1</td><td colspan="1" rowspan="1">K. aerogenes</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">Ceftazidime</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">CTX-M-1:3.1% of223</td></tr><tr><td colspan="1" rowspan="1">CTX-M-1</td><td colspan="1" rowspan="1">K. aerogenes</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">Ceftriaxone</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">CTX-M-1:3.1% of223</td></tr><tr><td colspan="1" rowspan="1">CTX-M-1</td><td colspan="1" rowspan="1">K. aerogenes</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">Pip.Tazo</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">CTX-M-1:3.1% of223</td></tr><tr><td colspan="1" rowspan="1">CTX-M-1</td><td colspan="1" rowspan="1">K.michiganensis</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">Cefazolin</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">CTX-M-1:1.1% of 89</td></tr><tr><td colspan="1" rowspan="1">CTX-M-1</td><td colspan="1" rowspan="1">K.michiganensis</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">Cefuroxime</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">CTX-M-1:1.1% of 89</td></tr><tr><td colspan="1" rowspan="1">CTX-M-1</td><td colspan="1" rowspan="1">K.michiganensis</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">Cefepime</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">CTX-M-1:1.1% of 89</td></tr><tr><td colspan="1" rowspan="2">AMR GeneTarget</td><td colspan="1" rowspan="2">Organism</td><td colspan="2" rowspan="1">Detection of AMR GeneTarget by AMR GenePanel versus WholeGenome SequencingCombined Clinical DevicePerformanceStock and ProspectiveIsolates (from K191288)</td><td colspan="1" rowspan="2">Antimicrobial</td><td colspan="4" rowspan="1">Detection of AMR Gene Target by AMR GenePanel vs AST Non-SusceptibilityCombined Clinical Device Performance - Stock andProspective Isolates (from K191288) includingonly unique isolates (no replicates)</td><td colspan="5" rowspan="1">Detection of AMR Gene Target versus AST Non-Susceptibility inOpGen's SMART Study</td><td colspan="1" rowspan="2">%Organisswith AMRGeneTarget inNCBIPathogenDatabase(2019)</td></tr><tr><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1">PPV[95% CI]</td><td colspan="1" rowspan="1">NPV[95% CI]</td><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1">PPV[95% CI]</td><td colspan="1" rowspan="1">NPV[95% CI]</td><td colspan="1" rowspan="1">% Organismswith AMRGene Target</td></tr><tr><td colspan="1" rowspan="1">CTX-M-1</td><td colspan="1" rowspan="1">K.michiganensis</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">Ceftazidime</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">CTX-M-1:1.1% of 89</td></tr><tr><td colspan="1" rowspan="1">CTX-M-1</td><td colspan="1" rowspan="1">K.michiganensis</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">Ceftriaxone</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">CTX-M-1:1.1% of 89</td></tr><tr><td colspan="1" rowspan="1">CTX-M-1</td><td colspan="1" rowspan="1">K.michiganensis</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">Amox.K.Clav</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">CTX-M-1:1.1% of 89</td></tr><tr><td colspan="1" rowspan="1">CTX-M-1</td><td colspan="1" rowspan="1">K.michiganensis</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">Amp.Sulbactam</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">CTX-M-1:1.1% of 89</td></tr><tr><td colspan="1" rowspan="1">CTX-M-1</td><td colspan="1" rowspan="1">K.michiganensis</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">Pip.Tazo</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">CTX-M-1:1.1% of 89</td></tr><tr><td colspan="1" rowspan="1">CTX-M-1</td><td colspan="1" rowspan="1">K. oxytoca</td><td colspan="1" rowspan="1">100.0%(2/2)[34.24-100.0</td><td colspan="1" rowspan="1">100.0%(3/3)[43.85-100.00]</td><td colspan="1" rowspan="1">Cefazolin</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">CTX-M-1:0.7% of273</td></tr><tr><td colspan="1" rowspan="1">CTX-M-1</td><td colspan="1" rowspan="1">K. oxytoca</td><td colspan="1" rowspan="1">100.0%(2/[34.24-00]</td><td colspan="1" rowspan="1">100.0%(3/3)43.85-100.00]</td><td colspan="1" rowspan="1">Cefuroxime</td><td colspan="1" rowspan="1">CTX-M-1:50.0%(2/415.0-85.00]</td><td colspan="1" rowspan="1">CTX-M-1:100.0%(1/1))20.5-100.00]</td><td colspan="1" rowspan="1">CTX-M-1:100.0%(2/2)[34.24-100.00]</td><td colspan="1" rowspan="1">CTX-M-1:33.3%(1/3)[6.15-79.23]</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">CTX-M-1:0.7% of273</td></tr><tr><td colspan="1" rowspan="1">CTX-M-1</td><td colspan="1" rowspan="1">K. oxytoca</td><td colspan="1" rowspan="1">100.0%(2/2)34.24-100.00]</td><td colspan="1" rowspan="1">100.0%(3/3)43.5-100.00]</td><td colspan="1" rowspan="1">Cefepime</td><td colspan="1" rowspan="1">CTX-M-1:0.0%(2/4)[15.00-85.00]</td><td colspan="1" rowspan="1">CTX-M-1:100.0%(1/1)[20.65-100.00]</td><td colspan="1" rowspan="1">CTX-M-1:100.0%(2/2)[34.24-100.00]</td><td colspan="1" rowspan="1">CTX-M-1:33.3%(1/3)[6.15-79.23]</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">CTX-M-1:0.7% of273</td></tr><tr><td colspan="1" rowspan="1">CTX-M-1</td><td colspan="1" rowspan="1">K. oxytoca</td><td colspan="1" rowspan="1">100.0%(2/2)[34.24-10.00]</td><td colspan="1" rowspan="1">100.0%(3/3)43.8-0.00]</td><td colspan="1" rowspan="1">Ceftazidime</td><td colspan="1" rowspan="1">CTX-M-1:50.0%(1/2)[9.45-90.55]</td><td colspan="1" rowspan="1">CTX-M-1:66.7%(2/3)[20.77-93.85]</td><td colspan="1" rowspan="1">CTX-M-1:50.0%(1/2)[9.45-90.55]</td><td colspan="1" rowspan="1">CTX-M-1:66.7%(2/3)[20.77-93.85]</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">CTX-M-1:0.7% of273</td></tr><tr><td colspan="1" rowspan="1">CTX-M-1</td><td colspan="1" rowspan="1">K. oxytoca</td><td colspan="1" rowspan="1">100.0%(2/2)[34.24-100.00]</td><td colspan="1" rowspan="1">100.0%(3/3)[43.85-100.00]</td><td colspan="1" rowspan="1">Ceftriaxone</td><td colspan="1" rowspan="1">CTX-M-1:50.0%(2/4)[15.00-85.00]</td><td colspan="1" rowspan="1">CTX-M-1:100.0%(1/1)[20.65-100.00]</td><td colspan="1" rowspan="1">CTX-M-1:100.0%(2//2)[34.24-100.00]</td><td colspan="1" rowspan="1">CTX-M-1:33.3%(1/3)[6.15-79.23]</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">CTX-M-1:0.7% of273</td></tr><tr><td colspan="1" rowspan="2">AMR GeneTarget</td><td colspan="1" rowspan="2">Organism</td><td colspan="2" rowspan="1">Detection of AMR GeneTarget by AMR GenePanel versus WholeGenome SequencingCombined Clinical DevicePerformance -Stock and ProspectiveIsolates (from K191288)</td><td colspan="1" rowspan="2">Antimicrobial</td><td colspan="4" rowspan="1">Detection of AMR Gene Target by AMR GenePanel vs AST Non-SusceptibilityCombined Clinical Device Performance - Stock andProspective Isolates (from K191288) includingonly unique isolates (no replicates)</td><td colspan="5" rowspan="1">Detection of AMR Gene Target versus AST Non-Susceptibility inOpGen's SMART Study</td><td colspan="1" rowspan="2">%Organismswith AMRGeneTarget inNCBIPathogenDatabase(2019)</td></tr><tr><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1">PPV[95% CI]</td><td colspan="1" rowspan="1">NPV[95% CI]</td><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1">PPV[95% CI]</td><td colspan="1" rowspan="1">NPV[95% CI]</td><td colspan="1" rowspan="1">% Organismswith AMRGene Target</td></tr><tr><td colspan="1" rowspan="1">CTX-M-1</td><td colspan="1" rowspan="1">K. oxytoca</td><td colspan="1" rowspan="1">100.0%(2/2)[34.24-100.00]</td><td colspan="1" rowspan="1">100.0%(3/3)43.85-100.00]</td><td colspan="1" rowspan="1">Amox.K.Clav</td><td colspan="1" rowspan="1">CTX-M-1:50.0%(2/4)15.00-85.00]</td><td colspan="1" rowspan="1">CTX-M-1:100.0%(1/1)[20.60.00]</td><td colspan="1" rowspan="1">CTX-M-1:100.0%(2/2)[34.24-10.00]</td><td colspan="1" rowspan="1">CTX-M-1:33.3%(1/3)[6.15-79.23]</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">CTX-M-1:0.7% of273</td></tr><tr><td colspan="1" rowspan="1">CTX-M-1</td><td colspan="1" rowspan="1">K. oxytoca</td><td colspan="1" rowspan="1">100.0%(2/2)[34.24-100.00]</td><td colspan="1" rowspan="1">100.0%(3/3)43.85-100.00]</td><td colspan="1" rowspan="1">Amp.Sulbactam</td><td colspan="1" rowspan="1">CTX-M-1:40.0%(2/5)[11.76-76.93]</td><td colspan="1" rowspan="1">CTX-M-1:NaN²</td><td colspan="1" rowspan="1">CTX-M-1:100.0%(2/2)[34.24-100.00]</td><td colspan="1" rowspan="1">CTX-M-1:0.0%(0//3)[0.0o-56.15]</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">CTX-M-1:0.7% of273</td></tr><tr><td colspan="1" rowspan="1">CTX-M-1</td><td colspan="1" rowspan="1">K. oxytoca</td><td colspan="1" rowspan="1">100.0%(2/2)[34.24-100.00]</td><td colspan="1" rowspan="1">100.0%(3/3)43.85-100.00]</td><td colspan="1" rowspan="1">Pip.Tazo</td><td colspan="1" rowspan="1">CTX-M-1:0.0%(0/2)[0.00-65.76]</td><td colspan="1" rowspan="1">CTX-M-1:33.3%(1/3)[6.15-79.23]</td><td colspan="1" rowspan="1">CTX-M-1:0.0%(0/2)0.00-65.76]</td><td colspan="1" rowspan="1">CTX-M-1:33.3%(1/3)[6.15-79.23]</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">CTX-M-1:0.7% of273</td></tr><tr><td colspan="1" rowspan="1">CTX-M-1</td><td colspan="1" rowspan="1">K. variicola</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">Cefazolin</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">CTX-M-1:11.4% of272</td></tr><tr><td colspan="1" rowspan="1">CTX-M-1</td><td colspan="1" rowspan="1">K. variicola</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">Cefuroxime</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">CTX-M-1:11.4% of272</td></tr><tr><td colspan="1" rowspan="1">CTX-M-1</td><td colspan="1" rowspan="1">K. variicola</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">Cefepime</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">CTX-M-1:11.4% of272</td></tr><tr><td colspan="1" rowspan="1">CTX-M-1</td><td colspan="1" rowspan="1">K. variicola</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">Ceftazidime</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">CTX-M-1:11.4% of272</td></tr><tr><td colspan="1" rowspan="1">CTX-M-1</td><td colspan="1" rowspan="1">K. variicola</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">Ceftriaxone</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">CTX-M-1:11.4% of272</td></tr><tr><td colspan="1" rowspan="1">CTX-M-1</td><td colspan="1" rowspan="1">K. variicola</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">Amox.K.Clav</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">CTX-M-1:11.4% of272</td></tr><tr><td colspan="1" rowspan="1">CTX-M-1</td><td colspan="1" rowspan="1">K. variicola</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">Amp.Sulbactam</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">CTX-M-1:11.4% of272</td></tr><tr><td colspan="1" rowspan="2">AMR GeneTarget</td><td colspan="1" rowspan="2">Organism</td><td colspan="2" rowspan="1">Detection of AMR GeneTarget by AMR GenePanel versus WholeGenome SequencingCombined Clinical DevicePerformance -Stock and ProspectiveIsolates (from K191288)</td><td colspan="1" rowspan="2">Antimicrobial</td><td colspan="4" rowspan="1">Detection of AMR Gene Target by AMR GenePanel vs AST Non-SusceptibilityCombined Clinical Device Performance - Stock andProspective Isolates (rom K191288)includingonly unique isolates (no replicates)</td><td colspan="5" rowspan="1">Detection of AMR Gene Target versus AST Non-Susceptibility inOpGen's SMART Study</td><td colspan="1" rowspan="2">%Organismswith AMRGeneTarget inNCBIPathogenDatabase(2019)</td></tr><tr><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1">PPV[95% CI]</td><td colspan="1" rowspan="1">NPV[95% CI]</td><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1">PPV[95% CI]</td><td colspan="1" rowspan="1">NPV[95% CI]</td><td colspan="1" rowspan="1">% Organismswith AMRGene Target</td></tr><tr><td colspan="1" rowspan="1">CTX-M-1</td><td colspan="1" rowspan="1">K. variicola</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">Pip.Tazo</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">CTX-M-1:11.4% of272</td></tr><tr><td colspan="1" rowspan="1">CTX-M-1</td><td colspan="1" rowspan="1">K.quasipneumoniae</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">Cefazolin</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">CTX-M-1:32% of 259</td></tr><tr><td colspan="1" rowspan="1">CTX-M-1</td><td colspan="1" rowspan="1">K.quasipneumoniae</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">Cefuroxime</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">CTX-M-1:32% of259</td></tr><tr><td colspan="1" rowspan="1">CTX-M-1</td><td colspan="1" rowspan="1">K.quasipneumoniae</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">Cefepime</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">CTX-M-1:32% of259</td></tr><tr><td colspan="1" rowspan="1">CTX-M-1</td><td colspan="1" rowspan="1">K.quasipneumoniae</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">Ceftazidime</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">CTX-M-1:32% of259</td></tr><tr><td colspan="1" rowspan="1">CTX-M-1</td><td colspan="1" rowspan="1">K.quasipneumoniae</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">Ceftriaxone</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">CTX-M-1:32% of259</td></tr><tr><td colspan="1" rowspan="1">CTX-M-1</td><td colspan="1" rowspan="1">K.quasipneumoniae</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">Amox.K.Clav</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">CTX-M-1:32% of259</td></tr><tr><td colspan="1" rowspan="1">CTX-M-1</td><td colspan="1" rowspan="1">K.quasipneumoniae</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">Amp.Sulbactam</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">CTX-M-1:32% of259</td></tr><tr><td colspan="1" rowspan="1">CTX-M-1</td><td colspan="1" rowspan="1">K.quasipneumoniae</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">Pip.Tazo</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">CTX-M-1:32% of 259</td></tr><tr><td colspan="1" rowspan="1">CTX-M-1</td><td colspan="1" rowspan="1">E. asburiae</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">Cefepime</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">CTX-M-1:2.7% of 73</td></tr><tr><td colspan="1" rowspan="1">CTX-M-1</td><td colspan="1" rowspan="1">E. asburiae</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">Ceftazidime</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">CTX-M-1:2.7% of 73</td></tr><tr><td colspan="1" rowspan="1">CTX-M-1</td><td colspan="1" rowspan="1">E. asburiae</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">Ceftriaxone</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">CTX-M-1:2.7% of 73</td></tr><tr><td colspan="1" rowspan="1">CTX-M-1</td><td colspan="1" rowspan="1">E. asburiae</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">Pip.Tazo</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">CTX-M-1:2.7% of 73</td></tr><tr><td colspan="1" rowspan="2">AMR GeneTarget</td><td colspan="1" rowspan="2">Organism</td><td colspan="2" rowspan="1">Detection of AMR GeneTarget by AMR GenePanel versus WholeGenome SequencingCombined Clinical DevicePerformance -Stock and ProspectiveIsolates (from K191288)</td><td colspan="1" rowspan="2">Antimicrobial</td><td colspan="4" rowspan="1">Detection of AMR Gene Target by AMR GenePanel vs AST Non-SusceptibilityCombined Clinical Device Performance - Stock andProspective Isolates (from K191288) includingonly unique isolates (no replicates)</td><td colspan="5" rowspan="1">Detection of AMR Gene Target versus AST Non-Susceptibility inOpGen's SMART Study</td><td colspan="1" rowspan="2">%Organismswith AMRGeneTarget inNCBIPathogenDatabase(2019)</td></tr><tr><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1">PPV[95% CI]</td><td colspan="1" rowspan="1">NPV[95% CI]</td><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1">PPV[95% CI]</td><td colspan="1" rowspan="1">NPV[95% CI]</td><td colspan="1" rowspan="1">% Organismswith AMRGene Target</td></tr><tr><td colspan="1" rowspan="1">CTX-M-1</td><td colspan="1" rowspan="1">E. cloacaecomplex</td><td colspan="1" rowspan="1">100.0%(5/5)[56.55-100.00]</td><td colspan="1" rowspan="1">100.0%(9/9)[70.09-100.00]</td><td colspan="1" rowspan="1">Cefepime</td><td colspan="1" rowspan="1">CTX-M-1:35.7%(5/14)[16.34-61.24]</td><td colspan="1" rowspan="1">CTX-M-1:NaN</td><td colspan="1" rowspan="1">CTX-M-1:100.0%(5/5)56.5-100.00]</td><td colspan="1" rowspan="1">CTX-M-1:0.0%(0/9)[0.00-29.91]</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">CTX-M-1:6.6% of563</td></tr><tr><td colspan="1" rowspan="1">CTX-M-1</td><td colspan="1" rowspan="1">E. cloacaecomplex</td><td colspan="1" rowspan="1">100.0%(5/5)56.55-100.00]</td><td colspan="1" rowspan="1">100.0%(9/9)[70.09-100.00]</td><td colspan="1" rowspan="1">Ceftazidime</td><td colspan="1" rowspan="1">CTX-M-1:35.7%(5/14)[16.34-61.24]</td><td colspan="1" rowspan="1">CTX-M-1:NaN</td><td colspan="1" rowspan="1">CTX-M-1:100.0%(5/5)[56.55-100.00]</td><td colspan="1" rowspan="1">CTX-M-1:0.0%(0/9)[0.0o-29.91]</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">CTX-M-1:6.6% of563</td></tr><tr><td colspan="1" rowspan="1">CTX-M-1</td><td colspan="1" rowspan="1">E. cloacaecomplex</td><td colspan="1" rowspan="1">100.0%(5/5)[56.55-10.00]</td><td colspan="1" rowspan="1">100.0%(9/9)[70.09-10.00]</td><td colspan="1" rowspan="1">Ceftriaxone</td><td colspan="1" rowspan="1">CTX-M-1:35.7%(5/14)[16.34-61.24]</td><td colspan="1" rowspan="1">CTX-M-1:NaN</td><td colspan="1" rowspan="1">CTX-M-1:100.0%(5/5)56.5-100.00]</td><td colspan="1" rowspan="1">CTX-M-1:0.0%(0/9)[0.00-29.91]</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">CTX-M-1:6.6% of563</td></tr><tr><td colspan="1" rowspan="1">CTX-M-1</td><td colspan="1" rowspan="1">E. cloacaecomplex</td><td colspan="1" rowspan="1">100.0%(5/5)56.5-0.00]</td><td colspan="1" rowspan="1">100.0%(9/9)70.09-100.00]</td><td colspan="1" rowspan="1">Pip.Tazo</td><td colspan="1" rowspan="1">CTX-M-1:35.7%(5/14)[16.34-61.24]</td><td colspan="1" rowspan="1">CTX-M-1:NaN</td><td colspan="1" rowspan="1">CTX-M-1:100.0%(5/5[56.55-100.00]</td><td colspan="1" rowspan="1">CTX-M-1:0.0%(0/9)[0.00-29.91]</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">CTX-M-1:6.6% of563</td></tr><tr><td colspan="1" rowspan="1">CTX-M-1</td><td colspan="1" rowspan="1">E. hormaechei</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">Cefepime</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">CTX-M-1:16.2% of678</td></tr><tr><td colspan="1" rowspan="1">CTX-M-1</td><td colspan="1" rowspan="1">E. hormaechei</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">Ceftazidime</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">CTX-M-1:16.2% of678</td></tr><tr><td colspan="1" rowspan="1">CTX-M-1</td><td colspan="1" rowspan="1">E. hormaechei</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">Ceftriaxone</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">CTX-M-1:16.2% of678</td></tr><tr><td colspan="1" rowspan="1">CTX-M-1</td><td colspan="1" rowspan="1">E. hormaechei</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">Pip.Tazo</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">CTX-M-1:16.2% of678</td></tr><tr><td colspan="1" rowspan="1">CTX-M-1</td><td colspan="1" rowspan="1">E. ludwigii</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">Cefepime</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">CTX-M-1:0% of 31</td></tr><tr><td colspan="1" rowspan="1">CTX-M-1</td><td colspan="1" rowspan="1">E. ludwigii</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">Ceftazidime</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">CTX-M-1:0% of 31</td></tr><tr><td colspan="1" rowspan="1">CTX-M-1</td><td colspan="1" rowspan="1">E. ludwigii</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">Ceftriaxone</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">CTX-M-1:0% of 31</td></tr><tr><td colspan="1" rowspan="1">CTX-M-1</td><td colspan="1" rowspan="1">E. ludwigii</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">Pip.Tazo</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">CTX-M-1:0% of 31</td></tr><tr><td colspan="1" rowspan="1">CTX-M-1</td><td colspan="1" rowspan="1">E. kobei</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">Cefepime</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">CTX-M-1:20% of 70</td></tr><tr><td colspan="1" rowspan="1">CTX-M-1</td><td colspan="1" rowspan="1">E. kobei</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">Ceftazidime</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">CTX-M-1:20% of 70</td></tr><tr><td colspan="1" rowspan="1">CTX-M-1</td><td colspan="1" rowspan="1">E. kobei</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">Ceftriaxone</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">CTX-M-1:20% of 70</td></tr><tr><td colspan="1" rowspan="1">CTX-M-1</td><td colspan="1" rowspan="1">E. kobei</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">Pip.Tazo</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">CTX-M-1:20% of 70</td></tr><tr><td colspan="1" rowspan="1">CTX-M-1</td><td colspan="1" rowspan="1">C. freundii</td><td colspan="1" rowspan="1">100.0%(1/1)[20.65-100.00]</td><td colspan="1" rowspan="1">100.0%(13/13)77.19-100.00]</td><td colspan="1" rowspan="1">Cefepime</td><td colspan="1" rowspan="1">CTX-M-1:12.5%(1/8)[2.24-47.09]</td><td colspan="1" rowspan="1">CTX-M-1:100.0%(6/6)[60.97-0.00]</td><td colspan="1" rowspan="1">CTX-M-1:100.0%(1/1)[20.65-0.00]</td><td colspan="1" rowspan="1">CTX-M-1:46.2%(6/13)[23.21-70.86]</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">CTX-M-1:12.3% of179</td></tr><tr><td colspan="1" rowspan="1">CTX-M-1</td><td colspan="1" rowspan="1">C. freundii</td><td colspan="1" rowspan="1">100.0%(1/1)[20.65-100.00]</td><td colspan="1" rowspan="1">100.0%(13/13)[77.19-100.00]</td><td colspan="1" rowspan="1">Ceftazidime</td><td colspan="1" rowspan="1">CTX-M-1:11.1%(1/9)1.9943.50]</td><td colspan="1" rowspan="1">CTX-M-1:100.0%(5/5) 56.5-100.00]</td><td colspan="1" rowspan="1">CTX-M-1:100.0%(1/1)[20.65-100.00]</td><td colspan="1" rowspan="1">CTX-M-1:38.5%(5/13)[17.71-64.48]</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">CTX-M-1:12.3% of179</td></tr><tr><td colspan="1" rowspan="1">CTX-M-1</td><td colspan="1" rowspan="1">C. freundii</td><td colspan="1" rowspan="1">100.0%(1/1)[20.65-100.00]</td><td colspan="1" rowspan="1">100.0%(13/13)[77.19-100.00]</td><td colspan="1" rowspan="1">Ceftriaxone</td><td colspan="1" rowspan="1">CTX-M-1:10.0%(1/10)[1.79-40.42]</td><td colspan="1" rowspan="1">CTX-M-1:100.0%(4/4)[51.01-100.00]</td><td colspan="1" rowspan="1">CTX-M-1:100.0%(1/1)[20.65-100.00]</td><td colspan="1" rowspan="1">CTX-M-1:30.8%(4/13)[12.68-57.63]</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">CTX-M-1:12.3% of179</td></tr><tr><td colspan="1" rowspan="1">CTX-M-1</td><td colspan="1" rowspan="1">C. freundii</td><td colspan="1" rowspan="1">100.0%(1/1)[20.65-100.00]</td><td colspan="1" rowspan="1">100.0%(13/13)[77.19-100.00]</td><td colspan="1" rowspan="1">Pip.Tazo</td><td colspan="1" rowspan="1">CTX-M-1:14.3%(1/7)2.57-51.31]</td><td colspan="1" rowspan="1">CTX-M-1:100.0%(7/7)[64.57-100.00]</td><td colspan="1" rowspan="1">CTX-M-1:100.0%(1/1)[20.65-100.00]</td><td colspan="1" rowspan="1">CTX-M-1:53.8%(7/13)[29.14-76.79]</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">CTX-M-1:12.3% of179</td></tr><tr><td colspan="1" rowspan="1">CTX-M-1</td><td colspan="1" rowspan="1">C. braakii</td><td colspan="1" rowspan="1">NaN</td><td colspan="1" rowspan="1">100.0%(1/1)[20.65-0.00]</td><td colspan="1" rowspan="1">Cefepime</td><td colspan="1" rowspan="1">CTX-M-1:NaN</td><td colspan="1" rowspan="1">CTX-M-1:100.0%(1/1)</td><td colspan="1" rowspan="1">CTX-M-1:NaN</td><td colspan="1" rowspan="1">CTX-M-1:100.0%(1/1)</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">CTX-M-1:0% of 30</td></tr><tr><td colspan="1" rowspan="2">AMR GeneTarget</td><td colspan="1" rowspan="2">Organism</td><td colspan="2" rowspan="1">Detection of AMR GeneTarget by AMR GenePanel versus WholeGenome SequencingCombined Clinical DevicePerformance -Stock and ProspectiveIsolates (from K191288)</td><td colspan="1" rowspan="1">Antimicrobial</td><td colspan="4" rowspan="1">Detection of AMR Gene Target by AMR GenePanel vs AST Non-SusceptibilityCombined Clinical Device Performance - Stock andProspective Isolates (from K191288) includingonly unique isolates (no replicates)</td><td colspan="5" rowspan="1">Detection of AMR Gene Target versus AST Non-Susceptibility inOpGen's SMART Study</td><td colspan="1" rowspan="1">%Organismswith AMRGeneTarget inNCBIPathogen</td></tr><tr><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1">PPV[95% CI]</td><td colspan="1" rowspan="1">NPV[95% CI]</td><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1">PPV[95% CI]</td><td colspan="1" rowspan="1">NPV[95% CI]</td><td colspan="1" rowspan="1">% Organismswith AMRGene Target</td><td colspan="1" rowspan="1">Database(2019)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">[20.65- 100.00]</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">[20.65- 100.00]</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">CTX-M-1</td><td colspan="1" rowspan="1">C. braakii</td><td colspan="1" rowspan="1">NaN</td><td colspan="1" rowspan="1">100.0%(1/1)20.65-1000]</td><td colspan="1" rowspan="1">Ceftazidime</td><td colspan="1" rowspan="1">CTX-M-1:0.0% 0/1)0.00-9.35]</td><td colspan="1" rowspan="1">CTX-M-1:NaN</td><td colspan="1" rowspan="1">CTX-M-1:NaN</td><td colspan="1" rowspan="1">CTX-M-1:0.0%(0/1)[0.0-79.35]</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">CTX-M-1:0% of 30</td></tr><tr><td colspan="1" rowspan="1">CTX-M-1</td><td colspan="1" rowspan="1">C. braakii</td><td colspan="1" rowspan="1">NaN</td><td colspan="1" rowspan="1">100.0%(1/1)20.65-100.00]</td><td colspan="1" rowspan="1">Ceftriaxone</td><td colspan="1" rowspan="1">CTX-M-1:0.0%(0/1)[0.00-79.35]</td><td colspan="1" rowspan="1">CTX-M-1:NaN</td><td colspan="1" rowspan="1">CTX-M-1:NaN</td><td colspan="1" rowspan="1">CTX-M-1:0.0%(0/1)[0.00-79.35]</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">CTX-M-1:0% of 30</td></tr><tr><td colspan="1" rowspan="1">CTX-M-1</td><td colspan="1" rowspan="1">C. braakii</td><td colspan="1" rowspan="1">NaN</td><td colspan="1" rowspan="1">100.0%(1/1)[20.65-10.00]</td><td colspan="1" rowspan="1">Pip.Tazo</td><td colspan="1" rowspan="1">CTX-M-1:0.0%(0/1)0.00-79.35]</td><td colspan="1" rowspan="1">CTX-M-1:NaN</td><td colspan="1" rowspan="1">CTX-M-1:NaN</td><td colspan="1" rowspan="1">CTX-M-1:0.0%(0/1)[0.00-79.35]</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">CTX-M-1:0% of 30</td></tr><tr><td colspan="1" rowspan="1">CTX-M-1</td><td colspan="1" rowspan="1">C. werkmanii</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">Cefepime</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">CTX-M-1:30.0% of10</td></tr><tr><td colspan="1" rowspan="1">CTX-M-1</td><td colspan="1" rowspan="1">C. werkmanii</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">Ceftazidime</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">CTX-M-1:30.0% of10</td></tr><tr><td colspan="1" rowspan="1">CTX-M-1</td><td colspan="1" rowspan="1">C. werkmanii</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">Ceftriaxone</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">CTX-M-1:30.0% of10</td></tr><tr><td colspan="1" rowspan="1">CTX-M-1</td><td colspan="1" rowspan="1">C. werkmanii</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">Pip.Tazo</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">CTX-M-1:30.0% of10</td></tr><tr><td colspan="1" rowspan="1">CTX-M-1</td><td colspan="1" rowspan="1">C. youngae</td><td colspan="1" rowspan="1">100.0%(1/1)[20.65-10000]</td><td colspan="1" rowspan="1">NaN</td><td colspan="1" rowspan="1">Cefepime</td><td colspan="1" rowspan="1">CTX-M-1:100.0%(1/1)[20.65-10.00]</td><td colspan="1" rowspan="1">CTX-M-1:NaN</td><td colspan="1" rowspan="1">CTX-M-1:100.0%(1/1)[20.65-100.00]</td><td colspan="1" rowspan="1">CTX-M-1:NaN</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">CTX-M-1:0% of 7</td></tr><tr><td colspan="1" rowspan="2">AMR GeneTarget</td><td colspan="1" rowspan="2">Organism</td><td colspan="2" rowspan="1">Detection of AMR GeneTarget by AMR GenePanel versus WholeGenome SequencingCombined Clinical DevicePerformance -Stock and ProspectiveIsolates (from K191288)</td><td colspan="1" rowspan="2">Antimicrobial</td><td colspan="4" rowspan="1">Detection of AMR Gene Target by AMR GenePanel vs AST Non-SusceptibilityCombined Clinical Device Performance - Stock andProspective Isolates (from K191288) includingonly unique isolates (no replicates)</td><td colspan="5" rowspan="1">Detection of AMR Gene Target versus AST Non-Susceptibility inOpGen's SMART Study</td><td colspan="1" rowspan="2">%Organismswith AMRGeneTarget inNCBIPathogenDatabase(2019)</td></tr><tr><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1">PPV[95% CI]</td><td colspan="1" rowspan="1">NPV[95% CI]</td><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1">PPV[95% CI]</td><td colspan="1" rowspan="1">NPV[95% CI]</td><td colspan="1" rowspan="1">% Organismswith AMRGene Target</td></tr><tr><td colspan="1" rowspan="1">CTX-M-1</td><td colspan="1" rowspan="1">C. youngae</td><td colspan="1" rowspan="1">100.0%(1/1)[20.65-100.00]</td><td colspan="1" rowspan="1">NaN</td><td colspan="1" rowspan="1">Ceftazidime</td><td colspan="1" rowspan="1">CTX-M-1:00.0%(1/1)[20.65-100.00]</td><td colspan="1" rowspan="1">CTX-M-1:NaN</td><td colspan="1" rowspan="1">CTX-M-1:100.0%(1/1)[20.65-100.00]</td><td colspan="1" rowspan="1">CTX-M-1:NaN</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">CTX-M-1:0% of 7</td></tr><tr><td colspan="1" rowspan="1">CTX-M-1</td><td colspan="1" rowspan="1">C. youngae</td><td colspan="1" rowspan="1">100.0%(1/1)20.65-100.00]</td><td colspan="1" rowspan="1">NaN</td><td colspan="1" rowspan="1">Ceftriaxone</td><td colspan="1" rowspan="1">CTX-M-1:100.0%(1/1)[20.65-100.00]</td><td colspan="1" rowspan="1">CTX-M-1:NaN</td><td colspan="1" rowspan="1">CTX-M-1:100.0%(1/1)[20.65-100.00]</td><td colspan="1" rowspan="1">CTX-M-1:NaN</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">CTX-M-1:0% of 7</td></tr><tr><td colspan="1" rowspan="1">CTX-M-1</td><td colspan="1" rowspan="1">C. youngae</td><td colspan="1" rowspan="1">100.0%(1/1)[20.65-100.00]</td><td colspan="1" rowspan="1">NaN</td><td colspan="1" rowspan="1">Pip.Tazo</td><td colspan="1" rowspan="1">CTX-M-1:00.0%(1/1)[20.65-100.00]</td><td colspan="1" rowspan="1">CTX-M-1:NaN</td><td colspan="1" rowspan="1">CTX-M-1:100.0%(1/1)[20.65-100.00]</td><td colspan="1" rowspan="1">CTX-M-1:NaN</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">CTX-M-1:0 of 7</td></tr><tr><td colspan="1" rowspan="1">CTX-M-1</td><td colspan="1" rowspan="1">S. marcescens</td><td colspan="1" rowspan="1">100.0%(1/1)20.65-100.00]</td><td colspan="1" rowspan="1">100.0%(1/1)20.65-100.00]</td><td colspan="1" rowspan="1">Cefepime</td><td colspan="1" rowspan="1">CTX-M-1:50.0%(1/2)[9.45-90.5]</td><td colspan="1" rowspan="1">CTX-M-1:NaN</td><td colspan="1" rowspan="1">CTX-M-1:100.0%(1/1)[20.65-00]</td><td colspan="1" rowspan="1">CTX-M-1:0.0%(0/)[0.0--79.35]</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">CTX-M-1:4.2% of496</td></tr><tr><td colspan="1" rowspan="1">CTX-M-1</td><td colspan="1" rowspan="1">S. marcescens</td><td colspan="1" rowspan="1">100.0%(1/1)[20.65-100.00]</td><td colspan="1" rowspan="1">100.0%(1/1)20.65-100.0]</td><td colspan="1" rowspan="1">Ceftazidime</td><td colspan="1" rowspan="1">CTX-M-1:50.0%(1/2)[9.45-90.55]</td><td colspan="1" rowspan="1">CTX-M-1:NaN</td><td colspan="1" rowspan="1">CTX-M-1:100.0%(1/1)[20.65-100.00]</td><td colspan="1" rowspan="1">CTX-M-1:0.0%(0/1)[0.0-79.35]</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">CTX-M-1:4.2% of496</td></tr><tr><td colspan="1" rowspan="1">CTX-M-1</td><td colspan="1" rowspan="1">S. marcescens</td><td colspan="1" rowspan="1">100.0%(1/1)[20.65-100.00]</td><td colspan="1" rowspan="1">100.0%(1/1)[20.65-100.00]</td><td colspan="1" rowspan="1">Ceftriaxone</td><td colspan="1" rowspan="1">CTX-M-1:50.0%((1/2)[9.45-9055]</td><td colspan="1" rowspan="1">CTX-M-1:NaN</td><td colspan="1" rowspan="1">CTX-M-1:100.0%(1/1)[20.65-0.00]</td><td colspan="1" rowspan="1">CTX-M-1:0.0%(0/1)[0.00-79.35]</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">CTX-M-1:4.2% of496</td></tr><tr><td colspan="1" rowspan="1">CTX-M-1</td><td colspan="1" rowspan="1">S. marcescens</td><td colspan="1" rowspan="1">100.0%(1/1)[20.65-100.00]</td><td colspan="1" rowspan="1">100.0%(1/1)[20.65-100.00]</td><td colspan="1" rowspan="1">Pip.Tazo</td><td colspan="1" rowspan="1">CTX-M-1:50.0%(1/2)[9.45-90.55]</td><td colspan="1" rowspan="1">CTX-M-1:NaN</td><td colspan="1" rowspan="1">CTX-M-1:100.0%(1/1)[20.65-100.00]</td><td colspan="1" rowspan="1">CTX-M-1:0.0%(0/1)[0.00-79.35]</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">CTX-M-1:4.2% of496</td></tr><tr><td colspan="1" rowspan="2">AMR GeneTarget</td><td colspan="1" rowspan="2">Organism</td><td colspan="2" rowspan="1">Detection of AMR GeneTarget by AMR GenePanel versus WholeGenome SequencingCombined Clinical DevicePerformance -Stock and ProspectiveIsolates (from K191288)</td><td colspan="1" rowspan="2">Antimicrobial</td><td colspan="4" rowspan="1">Detection of AMR Gene Target by AMR GenePanel vs AST Non-SusceptibilityCombined Clinical Device Performance - Stock andProspective Isolates (from K191288) includingonly unique isolates (no replicates)</td><td colspan="5" rowspan="1">Detection of AMR Gene Target versus AST Non-Susceptibility inOpGen's SMART Study</td><td colspan="1" rowspan="1">%Organismswith AMRGeneTarget inNCBIPathogen</td></tr><tr><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1">PPV[95% CI]</td><td colspan="1" rowspan="1">NPV[95% CI]</td><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1">PPV[95% CI]</td><td colspan="1" rowspan="1">NPV[95% CI]</td><td colspan="1" rowspan="1">% Organismswith AMRGene Target</td><td colspan="1" rowspan="1">Database(2019)</td></tr><tr><td colspan="1" rowspan="1">CTX-M-1</td><td colspan="1" rowspan="1">M. morganii</td><td colspan="1" rowspan="1">100.0%(1/1)[20.65-100.00]</td><td colspan="1" rowspan="1">100.0%(6/6)[60.97-100.00]</td><td colspan="1" rowspan="1">Cefepime</td><td colspan="1" rowspan="1">CTX-M-1:25.0%(/4)[4.56-69.94]</td><td colspan="1" rowspan="1">CTX-M-1:100.0%(3/3)[43.85-100.00]</td><td colspan="1" rowspan="1">CTX-M-1:100.0%(1/1)[20.65-100.00]</td><td colspan="1" rowspan="1">CTX-M-1:50.0%(3/6)[18.76-81.24]</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">CTX-M-1:7% of 57</td></tr><tr><td colspan="1" rowspan="1">CTX-M-1</td><td colspan="1" rowspan="1">M. morganii</td><td colspan="1" rowspan="1">100.0%(1/1)20.65-100.00]</td><td colspan="1" rowspan="1">100.0%(6/6)60.97-10.00]</td><td colspan="1" rowspan="1">Ceftazidime</td><td colspan="1" rowspan="1">CTX-M-1:16.7%(1/6)[3.01-556.35]</td><td colspan="1" rowspan="1">CTX-M-1:100.0%(1/1)[20.65-100.00]</td><td colspan="1" rowspan="1">CTX-M-1:100.0%(1/1)[20.65-100.00]</td><td colspan="1" rowspan="1">CTX-M-1:16.7%(1/6)[3.01-56.35]</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">CTX-M-1:7% of 57</td></tr><tr><td colspan="1" rowspan="1">CTX-M-1</td><td colspan="1" rowspan="1">M. morganii</td><td colspan="1" rowspan="1">100.0%(1/1)[20.65-100.00]</td><td colspan="1" rowspan="1">100.0%(/6[60.97-100]</td><td colspan="1" rowspan="1">Ceftriaxone</td><td colspan="1" rowspan="1">CTX-M-1:20.0%(1//5)[3.62-62.45]</td><td colspan="1" rowspan="1">CTX-M-1:1000%(2/2)[34.24-100.00]</td><td colspan="1" rowspan="1">CTX-M-1:00.0%(1/1)[20.65-100.00]</td><td colspan="1" rowspan="1">CTX-M-1:33.3%(2/6)[9.68-70.00]</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">CTX-M-1:7% of 57</td></tr><tr><td colspan="1" rowspan="1">CTX-M-1</td><td colspan="1" rowspan="1">M. morganii</td><td colspan="1" rowspan="1">100.0%(1/1)[20.65-00]</td><td colspan="1" rowspan="1">100.0%(6/6)[60.97-00]</td><td colspan="1" rowspan="1">Amp.Sulbactam</td><td colspan="1" rowspan="1">CTX-M-1:14.3%(1/7)[2.57-51.31]</td><td colspan="1" rowspan="1">CTX-M-1:NaN</td><td colspan="1" rowspan="1">CTX-M-1:100.0%(1/1)[20.65-0.00]</td><td colspan="1" rowspan="1">CTX-M-1:0.0%((0/6)0.00-39.03]</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">CTX-M-1:7% of 57</td></tr><tr><td colspan="1" rowspan="1">CTX-M-1</td><td colspan="1" rowspan="1">M. morganii</td><td colspan="1" rowspan="1">100.0%(1/1)[20.65-100.00]</td><td colspan="1" rowspan="1">100.0%(6/6)[60.97-100.00]</td><td colspan="1" rowspan="1">Pip.Tazo</td><td colspan="1" rowspan="1">CTX-M-1:33.3%(1/3)[6.15-79.23]</td><td colspan="1" rowspan="1">CTX-M-1:100.0%(4/4)[51.01-100.00]</td><td colspan="1" rowspan="1">CTX-M-1:100.0%(1/1)20.65-100.00]</td><td colspan="1" rowspan="1">CTX-M-1:66.7%(4/6)[30.00-90.32]</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">CTX-M-1:7% of 57</td></tr><tr><td colspan="1" rowspan="1">CTX-M-2</td><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">100.0%(8/8)[67.56-00]</td><td colspan="1" rowspan="1">99.7%(326/327)[98.29-99.95]</td><td colspan="1" rowspan="1">Cefazolin</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">CTX-M-2:0.4%(8/2209)[0.18-0.71]</td><td colspan="1" rowspan="1">CTX-M-2:100.0%(751/751)[99.49-100.00]</td><td colspan="1" rowspan="1">CTX-M-2:100.0%(8/8)67.56-100.00]</td><td colspan="1" rowspan="1">CTX-M-2:25.4%(751/2952)[23.90-27.04]</td><td colspan="1" rowspan="1">CTX-M-2:0.3%(8/2960)</td><td colspan="1" rowspan="1">CTX-M-2:0.2% of19017</td></tr><tr><td colspan="1" rowspan="1">CTX-M-2</td><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">100.0%(8/8)[67.56-00]</td><td colspan="1" rowspan="1">99.7%(326/327)[98.29-9.95]</td><td colspan="1" rowspan="1">Cefuroxime</td><td colspan="1" rowspan="1">CTX-M-2:3.7%(5/136)1.5-8.32]</td><td colspan="1" rowspan="1">CTX-M-2:100.0%(23/23)85.690.00]</td><td colspan="1" rowspan="1">CTX-M-2:100.0%(5/5)556.5-10.00]</td><td colspan="1" rowspan="1">CTX-M-2:14.9%(23/154)10.16-21.41]</td><td colspan="1" rowspan="1">CTX-M-2:0.3%(7/2187)[0.16-0.66]</td><td colspan="1" rowspan="1">CTX-M-2:99.9%(772/773)99.27-99.98]</td><td colspan="1" rowspan="1">CTX-M-2:87.5%(7/8)552.91-97.76]</td><td colspan="1" rowspan="1">CTX-M-2:26.2%(772/2952)[24.60-27.77]</td><td colspan="1" rowspan="1">CTX-M-2:0.3%(8/2960)</td><td colspan="1" rowspan="1">CTX-M-2:0.2% of19017</td></tr><tr><td colspan="1" rowspan="2">AMR GeneTarget</td><td colspan="1" rowspan="2">Organism</td><td colspan="2" rowspan="1">Detection of AMR GeneTarget by AMR GenePanel versus WholeGenome SequencingCombined Clinical DevicePerformance -Stock and ProspectiveIsolates (from K191288)</td><td colspan="1" rowspan="1">Antimicrobial</td><td colspan="4" rowspan="1">Detection of AMR Gene Target by AMR GenePanel vs AST Non-SusceptibilityCombined Clinical Device Performance - Stock andProspective Isolates (from K191288) includingonly unique isolates (no replicates)</td><td colspan="5" rowspan="1">Detection of AMR Gene Target versus AST Non-Susceptibility inOpGen's SMART Study</td><td colspan="1" rowspan="1">%Organismswith AMRGeneTarget inNCBIPathogen</td></tr><tr><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1">PPV[95% CI]</td><td colspan="1" rowspan="1">NPV[95% CI]</td><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1">PPV[95% CI]</td><td colspan="1" rowspan="1">NPV[95% CI]</td><td colspan="1" rowspan="1">% Organismswith AMRGene Target</td><td colspan="1" rowspan="1">Database(2019)</td></tr><tr><td colspan="1" rowspan="1">CTX-M-2</td><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">100.0%(8/8)[67.56-10.00]</td><td colspan="1" rowspan="1">99.7%(326/327)[98.29-99.95]</td><td colspan="1" rowspan="1">Cefepime</td><td colspan="1" rowspan="1">CTX-M-2:4.7%(5/107)2.01-10.48]</td><td colspan="1" rowspan="1">CTX-M-2:100.0%(52/52)[93.12-100.00]</td><td colspan="1" rowspan="1">CTX-M-2:100.0%(5/5)[56.55-100.00]</td><td colspan="1" rowspan="1">CTX-M-2:33.8%(2/154)[26.77-41.55]</td><td colspan="1" rowspan="1">CTX-M-2:0.3%(7/2009)[0.17-0.72]</td><td colspan="1" rowspan="1">CTX-M-2:99.9%(950/951)[99.41-99.98]</td><td colspan="1" rowspan="1">CTX-M-2:87.5%(7/8)[52.91-9776]</td><td colspan="1" rowspan="1">CTX-M-2:32.2%(950/2952)[30.52-33.89]</td><td colspan="1" rowspan="1">CTX-M-2:0.3%(8/2960)</td><td colspan="1" rowspan="1">CTX-M-2:0.2% of19017</td></tr><tr><td colspan="1" rowspan="1">CTX-M-2</td><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">100.0%(8/8)[67.56-00]</td><td colspan="1" rowspan="1">99.7%(326/327)[98.29-99.95]</td><td colspan="1" rowspan="1">Ceftazidime</td><td colspan="1" rowspan="1">CTX-M-2:3.8%(4/106)[1.48-9.30]</td><td colspan="1" rowspan="1">CTX-M-2:98.1%(52/53)[90.06-99.67]</td><td colspan="1" rowspan="1">CTX-M-2:80.0%(4/5)[37.55-96.38]</td><td colspan="1" rowspan="1">CTX-M-2:33.8%(52/154)[26.77-41.55]</td><td colspan="1" rowspan="1">CTX-M-2:0.4%(6/1651)[0.17-0.79]</td><td colspan="1" rowspan="1">CTX-M-2:99.8%(1307/1309) [99.44-99.96]</td><td colspan="1" rowspan="1">CTX-M-2:75.0%(6/8)[40.93-92.85]</td><td colspan="1" rowspan="1">CTX-M-2:44.3%(1307/2952) [42.49-46.07]</td><td colspan="1" rowspan="1">CTX-M-2:0.3%(8/2960)</td><td colspan="1" rowspan="1">CTX-M-2:0.2% of19017</td></tr><tr><td colspan="1" rowspan="1">CTX-M-2</td><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">100.0%(8/8)[67.56-100.00]</td><td colspan="1" rowspan="1">99.7%(326/327)[98.29-9.95]</td><td colspan="1" rowspan="1">Ceftriaxone</td><td colspan="1" rowspan="1">CTX-M-2:3.8%(/130)[1.65-8.69]</td><td colspan="1" rowspan="1">CTX-M-2:0.0%(29/29)[88.30100.00]</td><td colspan="1" rowspan="1">CTX-M-2:10.0%(5/5 56.5-100.00]</td><td colspan="1" rowspan="1">CTX-M-2:18.8%(29/154)[13.44-25.74]</td><td colspan="1" rowspan="1">CTX-M-2:0.3%(7/2113)[0.16-0.68]</td><td colspan="1" rowspan="1">CTX-M-2:99.9%(846/847)99.33-99.98]</td><td colspan="1" rowspan="1">CTX-M-2:87.5%(7/8) 52.91-97.76]</td><td colspan="1" rowspan="1">CTX-M-2:28.7%(846/2952)27.630.32]</td><td colspan="1" rowspan="1">CTX-M-2:0.3%(8/2960)</td><td colspan="1" rowspan="1">CTX-M-2:0.2% of19017</td></tr><tr><td colspan="1" rowspan="1">CTX-M-2</td><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">100.0%(8/8)[67.56-10.00]</td><td colspan="1" rowspan="1">99.7%(326/327)[98.29-99.95]</td><td colspan="1" rowspan="1">Amox.K.Clav</td><td colspan="1" rowspan="1">CTX-M-2:3.5%(4/114)[1.37-8.68]</td><td colspan="1" rowspan="1">CTX-M-2:97.8%(44/45)[88.43-9.61</td><td colspan="1" rowspan="1">CTX-M-2:80.0%(4//5[37.55-96.38]</td><td colspan="1" rowspan="1">CTX-M-2:28.6%4/154)[22.03-36.16]</td><td colspan="1" rowspan="1">CTX-M-2:0.4%(8/1810)[0.22-0.87]</td><td colspan="1" rowspan="1">CTX-M-2:100.0%(1150/1150) (99.67-100.00]</td><td colspan="1" rowspan="1">CTX-M-2:100.0%(8/8)67.56-100.00]</td><td colspan="1" rowspan="1">CTX-M-2:39.0%(1150/2952) [37.21-40.73]</td><td colspan="1" rowspan="1">CTX-M-2:0.3%(8/2960)</td><td colspan="1" rowspan="1">CTX-M-2:0.2% of19017</td></tr><tr><td colspan="1" rowspan="1">CTX-M-2</td><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">100.0%(8/8)[67.56-100]</td><td colspan="1" rowspan="1">99.7%(326/327)[98.29-9.95]</td><td colspan="1" rowspan="1">Amp.Sulbactam</td><td colspan="1" rowspan="1">CTX-M-2:3.4%(5/146)1.47-7.77]</td><td colspan="1" rowspan="1">CTX-M-2:100.0%(13/13)[77.19-100.00]</td><td colspan="1" rowspan="1">CTX-M-2:100.0%(5/5)5 56.5-100.00]</td><td colspan="1" rowspan="1">CTX-M-2:8.4%(13/154)[5.00-13.91]</td><td colspan="1" rowspan="1">CTX-M-2:0.3%(8/2477)[0.16-0.64]</td><td colspan="1" rowspan="1">CTX-M-2:100.0%(483/483)[99.21-100.00]</td><td colspan="1" rowspan="1">CTX-M-2:100.0%(8/8)67.56-100.00]</td><td colspan="1" rowspan="1">CTX-M-2:16.4%(483/2952)15.07-17.74]</td><td colspan="1" rowspan="1">CTX-M-2:0.3%(8/2960)</td><td colspan="1" rowspan="1">CTX-M-2:0.2% of19017</td></tr><tr><td colspan="1" rowspan="1">CTX-M-2</td><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">100.0%(8/8)[67.56-100.00]</td><td colspan="1" rowspan="1">99.7%(326/327)[98.29-99.95]</td><td colspan="1" rowspan="1">Pip.Tazo</td><td colspan="1" rowspan="1">CTX-M-2:4.3%(3/69)[1.49-12.02]</td><td colspan="1" rowspan="1">CTX-M-2:97.8%(88/90)[92.26-99.39]</td><td colspan="1" rowspan="1">CTX-M-2:60.0%(3/5)[23.07-88.24]</td><td colspan="1" rowspan="1">CTX-M-2:57.1%(88/154)[49.25-64.69]</td><td colspan="1" rowspan="1">CTX-M-2:0.4%(3/687)[0.15-1.28]</td><td colspan="1" rowspan="1">CTX-M-2:99.8%(2268/2273) [99.49-99.91]</td><td colspan="1" rowspan="1">CTX-M-2:37.5%(3/8)[13.68-69.43]</td><td colspan="1" rowspan="1">CTX-M-2:76.8%(2268/2952) [75.27-78.32]</td><td colspan="1" rowspan="1">CTX-M-2:0.3%(8/2960)</td><td colspan="1" rowspan="1">CTX-M-2:0.2% of19017</td></tr><tr><td colspan="1" rowspan="1">CTX-M-2</td><td colspan="1" rowspan="1">P. mirabilis</td><td colspan="1" rowspan="1">100.0%(10/10)[72.25-100.00]</td><td colspan="1" rowspan="1">100.0%(145/145)[97.42-100.00]</td><td colspan="1" rowspan="1">Cefazolin</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">CTX-M-2:13.0%(74/571)[10.45-15.96]</td><td colspan="1" rowspan="1">CTX-M-2:99.5%(583/586)998.51-99.83]</td><td colspan="1" rowspan="1">CTX-M-2:96.1%(74/77)[89.16-98.67]</td><td colspan="1" rowspan="1">CTX-M-2:54.0%(583/1080)[51.00-56.93]</td><td colspan="1" rowspan="1">CTX-M-2:6.7%(77/1157)</td><td colspan="1" rowspan="1">CTX-M-2:3.5% of144</td></tr><tr><td colspan="1" rowspan="2">AMR GeneTarget</td><td colspan="1" rowspan="2">Organism</td><td colspan="2" rowspan="1">Detection of AMR GeneTarget by AMR GenePanel versus WholeGenome SequencingCombined Clinical DevicePerformance -Stock and ProspectiveIsolates (from K191288)</td><td colspan="1" rowspan="2">Antimicrobial</td><td colspan="4" rowspan="1">Detection of AMR Gene Target by AMR GenePanel vs AST Non-SusceptibilityCombined Clinical Device Performance - Stock andvonly unique isolates (no replicates)</td><td colspan="5" rowspan="1">Detection of AMR Gene Target versus AST Non-Susceptibility inOpGen's SMART Study</td><td colspan="1" rowspan="2">%Organismswith AMRGeneTarget inNCBIPathogenDatabase(2019)</td></tr><tr><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1">PPV[95% CI]</td><td colspan="1" rowspan="1">NPV[95% CI]</td><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1">PPV[95% CI]</td><td colspan="1" rowspan="1">NPV[95% CI]</td><td colspan="1" rowspan="1">% Organismswith AMRGene Target</td></tr><tr><td colspan="1" rowspan="1">CTX-M-2</td><td colspan="1" rowspan="1">P. mirabilis</td><td colspan="1" rowspan="1">100.0%(10/10)[72.25-100.00]</td><td colspan="1" rowspan="1">100.0%(145/145)[97.42-100.00]</td><td colspan="1" rowspan="1">Cefuroxime</td><td colspan="1" rowspan="1">CTX-M-2:13.3%(6/45)[6.26-26.18]</td><td colspan="1" rowspan="1">CTX-M-2:100.0%(33/33)[89.57-10.00]</td><td colspan="1" rowspan="1">CTX-M-2:100.0%(6/6)[60.97-100.00]</td><td colspan="1" rowspan="1">CTX-M-2:45.8%(33/72)[34.83-57.26]</td><td colspan="1" rowspan="1">CTX-M-2:14.9%(74/495)[12.08-18.36</td><td colspan="1" rowspan="1">CTX-M-2:99.5%(659/662)[98.68-99.85]</td><td colspan="1" rowspan="1">CTX-M-2:96.1%(74/77)[89.16-98.67]</td><td colspan="1" rowspan="1">CTX-M-2:61.0%(659/1080)[58.08-63.88</td><td colspan="1" rowspan="1">CTX-M-2:6.7%777/1157)</td><td colspan="1" rowspan="1">CTX-M-2:3.5% of144</td></tr><tr><td colspan="1" rowspan="1">CTX-M-2</td><td colspan="1" rowspan="1">P. mirabilis</td><td colspan="1" rowspan="1">100.0%(10/10)[72.25-100.00]</td><td colspan="1" rowspan="1">100.0%(145/145)[97.42-100.00]</td><td colspan="1" rowspan="1">Cefepime</td><td colspan="1" rowspan="1">CTX-M-2:15.0%(6/40)[7.06-29.07]</td><td colspan="1" rowspan="1">CTX-M-2:100.0%(38/38)[90.82-10.00]</td><td colspan="1" rowspan="1">CTX-M-2:100.0%(6/6)[60.97-10.00]</td><td colspan="1" rowspan="1">CTX-M-2:52.8%(38/72)[41.40-63.87]</td><td colspan="1" rowspan="1">CTX-M-2:21.0%(70/334)[16.94-25.4]</td><td colspan="1" rowspan="1">CTX-M-2:99.1%(816/823)[98.25-99.59]</td><td colspan="1" rowspan="1">CTX-M-2:90.9%(70/77)[82.40-95.53]</td><td colspan="1" rowspan="1">CTX-M-2:75.6%(816/1080)[72.90-78.03]</td><td colspan="1" rowspan="1">CTX-M-2:6.7%(77/1157)</td><td colspan="1" rowspan="1">CTX-M-2:3.5% of144</td></tr><tr><td colspan="1" rowspan="1">CTX-M-2</td><td colspan="1" rowspan="1">P. mirabilis</td><td colspan="1" rowspan="1">100.0%(10/10)[72.25-10.00]</td><td colspan="1" rowspan="1">100.0%(145/145)[97.42-0.00]</td><td colspan="1" rowspan="1">Ceftazidime</td><td colspan="1" rowspan="1">CTX-M-2:0.0%(0/23)[0.00-14.31]</td><td colspan="1" rowspan="1">CTX-M-2:89.1%(49/55)[78.17-94.901</td><td colspan="1" rowspan="1">CTX-M-2:0.0%(0/6)[0.00-39.03]</td><td colspan="1" rowspan="1">CTX-M-2:68.1%(49/72)56.61-77.67]</td><td colspan="1" rowspan="1">CTX-M-2:0.8%(2/259)[0.21-2.77]</td><td colspan="1" rowspan="1">CTX-M-2:91.6%(823/898)[89.66-93.29]</td><td colspan="1" rowspan="1">CTX-M-2:2.6%(2/77)[0.72-8.98]</td><td colspan="1" rowspan="1">CTX-M-2:76.2%(823/1080)[73.57-78.65]</td><td colspan="1" rowspan="1">CTX-M-2:6.7%(77/1157)</td><td colspan="1" rowspan="1">CTX-M-2:3.5% of144</td></tr><tr><td colspan="1" rowspan="1">CTX-M-2</td><td colspan="1" rowspan="1">P. mirabilis</td><td colspan="1" rowspan="1">100.0%(10/10)72.25-100.00]</td><td colspan="1" rowspan="1">100.0%(145/145)[97.42-100.00]</td><td colspan="1" rowspan="1">Ceftriaxone</td><td colspan="1" rowspan="1">CTX-M-2:14.3%(6/42)[6.72-27.84]</td><td colspan="1" rowspan="1">CTX-M-2:100.0%(36/36)90.36-100]</td><td colspan="1" rowspan="1">CTX-M-2:100.0%6/6)[60.97-00]</td><td colspan="1" rowspan="1">CTX-M-2:50.0%(36/72)[38.75-61.25]</td><td colspan="1" rowspan="1">CTX-M-2:15.3%(73/478)[12.33-18.77]</td><td colspan="1" rowspan="1">CTX-M-2:99.4%(675/679)[98.50-99.7</td><td colspan="1" rowspan="1">CTX-M-2:94.8%(73/77)[87.39-97.96]</td><td colspan="1" rowspan="1">CTX-M-2:62.5%(675/1080)[59.57-65.34]</td><td colspan="1" rowspan="1">CTX-M-2:6.7%(77/1157)</td><td colspan="1" rowspan="1">CTX-M-2:3.5% of144</td></tr><tr><td colspan="1" rowspan="1">CTX-M-2</td><td colspan="1" rowspan="1">P. mirabilis</td><td colspan="1" rowspan="1">100.0%(10/10)[72.25-10.00]</td><td colspan="1" rowspan="1">100.0%(145/145)[97.42-100.00]</td><td colspan="1" rowspan="1">Amox.K.Clav</td><td colspan="1" rowspan="1">CTX-M-2:11.5%(3/26)4.00-28.98]</td><td colspan="1" rowspan="1">CTX-M-2:94.2%(49/52)[84.36-98.02]</td><td colspan="1" rowspan="1">CTX-M-2:500.0%(3/6)18.76-81.24]</td><td colspan="1" rowspan="1">CTX-M-2:68.1%(49/72)[56.61-77.67]</td><td colspan="1" rowspan="1">CTX-M-2:6.4%(23/359)[4.31-9.43]</td><td colspan="1" rowspan="1">CTX-M-2:93.2%(744/798)[91.28-94.78]</td><td colspan="1" rowspan="1">CTX-M-2:29.9%(23/77)[20.80-40.85]</td><td colspan="1" rowspan="1">CTX-M-2:68.9%(744/1080)[66.07-71.58]</td><td colspan="1" rowspan="1">CTX-M-2:6.7%(77/1157)</td><td colspan="1" rowspan="1">CTX-M-2:3.5% of144</td></tr><tr><td colspan="1" rowspan="1">CTX-M-2</td><td colspan="1" rowspan="1">P. mirabilis</td><td colspan="1" rowspan="1">100.0%(10/10)72.25-100.00]</td><td colspan="1" rowspan="1">100.0%(145/145)[97.42-100.00]</td><td colspan="1" rowspan="1">Amp.Sulbactam</td><td colspan="1" rowspan="1">CTX-M-2:11.9%(5/42)[5.19-25.00]</td><td colspan="1" rowspan="1">CTX-M-2:97.2%(35/36)[85.83-99.51]</td><td colspan="1" rowspan="1">CTX-M-2:83.3%(/6[43.65-96.99]</td><td colspan="1" rowspan="1">CTX-M-2:48.6%(35/72)[37.43-59.93]</td><td colspan="1" rowspan="1">CTX-M-2:11.9%(69/582)[9.48-14.74]</td><td colspan="1" rowspan="1">CTX-M-2:98.6%5667/575)[97.28-99.29]</td><td colspan="1" rowspan="1">CTX-M-2:89.6%(69/7)[80.82-94.64]</td><td colspan="1" rowspan="1">CTX-M-2:52.5%(67/1080)[49.52-55.46]</td><td colspan="1" rowspan="1">CTX-M-2:6.7% 77/1157)</td><td colspan="1" rowspan="1">CTX-M-2:3.5% of144</td></tr><tr><td colspan="1" rowspan="1">CTX-M-2</td><td colspan="1" rowspan="1">P. mirabilis</td><td colspan="1" rowspan="1">100.0%(10/10)[72.25-100.00]</td><td colspan="1" rowspan="1">100.0%(145/145)[97.42-100.00]</td><td colspan="1" rowspan="1">Pip.Tazo</td><td colspan="1" rowspan="1">CTX-M-2:0.0%(0/4)[0.0-48.99]</td><td colspan="1" rowspan="1">CTX-M-2:91.9%(68/74)[83.42-96.23]</td><td colspan="1" rowspan="1">CTX-M-2:0.0%(0/)[0.00--39.03]</td><td colspan="1" rowspan="1">CTX-M-2:94.4%(68/72)86.57-97.82]</td><td colspan="1" rowspan="1">CTX-M-2:0.0%(0/31)[0.00-11.03]</td><td colspan="1" rowspan="1">CTX-M-2:93.2%(1049/1126) [91.54-94.49]</td><td colspan="1" rowspan="1">CTX-M-2:0.0%(0/77)[0.00-4.75]</td><td colspan="1" rowspan="1">CTX-M-2:97.1%(1049/1080) [95.95-97.97]</td><td colspan="1" rowspan="1">CTX-M-2:6.7%(77/1157)</td><td colspan="1" rowspan="1">CTX-M-2:3.5% of144</td></tr><tr><td colspan="1" rowspan="2">AMR GeneTarget</td><td colspan="1" rowspan="2">Organism</td><td colspan="2" rowspan="1">Detection of AMR GeneTarget by AMR GenePanel versus WholeGenome SequencingCombined Clinical DevicePerformance -Stock and ProspectiveIsolates (from K191288)</td><td colspan="1" rowspan="2">Antimicrobial</td><td colspan="4" rowspan="1">Detection of AMR Gene Target by AMR GenePanel vs AST Non-SusceptibilityCombined Clinical Device Performance - Stock andonly unique isolates (no replicates)</td><td colspan="5" rowspan="1">Detection of AMR Gene Target versus AST Non-Susceptibility inOpGen's SMART Study</td><td colspan="1" rowspan="2">%Organismswith AMRGeneTarget inNCBIPathogenDatabase(2019)</td></tr><tr><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1">PPV[95% CI]</td><td colspan="1" rowspan="1">NPV[95% CI]</td><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1">PPV[95% CI]</td><td colspan="1" rowspan="1">NPV[95% CI]</td><td colspan="1" rowspan="1">% Organismswith AMRGene Target</td></tr><tr><td colspan="1" rowspan="1">CTX-M-2</td><td colspan="1" rowspan="1">P. aeruginosa</td><td colspan="1" rowspan="1">100.0%(17/17)[81.57-10.00]</td><td colspan="1" rowspan="1">99.7%(30/331)[98.31-9.95]</td><td colspan="1" rowspan="1">Cefepime</td><td colspan="1" rowspan="1">CTX-M-2:8.9%(11/123)[5.07-15.31]</td><td colspan="1" rowspan="1">CTX-M-2:100.0%(32/32)[89.28-10.00]</td><td colspan="1" rowspan="1">CTX-M-2:100.0%(11/11)[74.12-100.00]</td><td colspan="1" rowspan="1">CTX-M-2:22.2%(32/144)[16.20-29.68]</td><td colspan="1" rowspan="1">CTX-M-2:0.9%(10/1068)[0.51-1.71]</td><td colspan="1" rowspan="1">CTX-M-2:100.0%(482/482)[99.21-100]</td><td colspan="1" rowspan="1">CTX-M-2:00.0%(10/10)[72.25-10.00]</td><td colspan="1" rowspan="1">CTX-M-2:31.3%(482/1540)[29.03-33.66]</td><td colspan="1" rowspan="1">CTX-M-2:0.6%(10/1550)</td><td colspan="1" rowspan="1">CTX-M-2:0.1% of4706</td></tr><tr><td colspan="1" rowspan="1">CTX-M-2</td><td colspan="1" rowspan="1">P. aeruginosa</td><td colspan="1" rowspan="1">100.0%(17/17)[81.57-100.00]</td><td colspan="1" rowspan="1">99.7%(330/331)[98.31-99.95]</td><td colspan="1" rowspan="1">Ceftazidime</td><td colspan="1" rowspan="1">CTX-M-2:5.3%(6/114)[2.43-11.01]</td><td colspan="1" rowspan="1">CTX-M-2:87.8%(36/41)[74.46-94.68]</td><td colspan="1" rowspan="1">CTX-M-2:54.5%(6/11)[28.01-78.73]</td><td colspan="1" rowspan="1">CTX-M-2:25.0%(36/144)[18.64-32.66]</td><td colspan="1" rowspan="1">CTX-M-2:0.5%(5/959)[0.22-1.21]</td><td colspan="1" rowspan="1">CTX-M-2:99.2%(586/591)[98.03-99.64]</td><td colspan="1" rowspan="1">CTX-M-2:50.0%(5/10)[23.66-76.34]</td><td colspan="1" rowspan="1">CTX-M-2:38.1%(586/1540)[35.66-40.50]</td><td colspan="1" rowspan="1">CTX-M-2:0.6%(10/1550)</td><td colspan="1" rowspan="1">CTX-M-2:0.1% of4706</td></tr><tr><td colspan="1" rowspan="1">CTX-M-2</td><td colspan="1" rowspan="1">P. aeruginosa</td><td colspan="1" rowspan="1">100.0%(17/17)[81.57-100.00]</td><td colspan="1" rowspan="1">99.7%(330/331)[98.31-9.95]</td><td colspan="1" rowspan="1">Pip.Tazo</td><td colspan="1" rowspan="1">CTX-M-2:9.6%(10/104)[5.31-16.80]</td><td colspan="1" rowspan="1">CTX-M-2:98.0%(50/1)[89.099.65]</td><td colspan="1" rowspan="1">CTX-M-2:90.9%(10/11)62.26-98.38]</td><td colspan="1" rowspan="1">CTX-M-2:34.7%(50/144)[27.43-42.80]</td><td colspan="1" rowspan="1">CTX-M-2:1.0%(9/941)[0.50-1.81]</td><td colspan="1" rowspan="1">CTX-M-2:99.8%(608/609)[99.08-99.97]</td><td colspan="1" rowspan="1">CTX-M-2:90.0%(9/ /10)59. 898.21]</td><td colspan="1" rowspan="1">CTX-M-2:39.5%(608/1540)[37.07-41.95]</td><td colspan="1" rowspan="1">CTX-M-2:0.6%(10/1550)</td><td colspan="1" rowspan="1">CTX-M-2:0.1% of4706</td></tr><tr><td colspan="1" rowspan="1">CTX-M-9</td><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">97.7%(42/43)[87.94-99.59]</td><td colspan="1" rowspan="1">100.0%(292/292)[98.70-100.00]</td><td colspan="1" rowspan="1">Cefazolin</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">CTX-M-9:26.2%(578/2209)[24.38-28.04]</td><td colspan="1" rowspan="1">CTX-M-9:99.3%(746/751)[98.45-99.72]</td><td colspan="1" rowspan="1">CTX-M-9:99.1%(578/583)[98.01-99.63]</td><td colspan="1" rowspan="1">CTX-M-9:31.4%(746/2377)[29.55-33.28]</td><td colspan="1" rowspan="1">CTX-M-9:19.7%(583/2960)</td><td colspan="1" rowspan="1">CTX-M-9:4.9% of19017</td></tr><tr><td colspan="1" rowspan="1">CTX-M-9</td><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">97.7%(42/43)[87.94-99.59]</td><td colspan="1" rowspan="1">100.0%(292/292)[98.70-100.00]</td><td colspan="1" rowspan="1">Cefuroxime</td><td colspan="1" rowspan="1">CTX-M-9:22.1%(30/136)[15.91-29.74]</td><td colspan="1" rowspan="1">CTX-M-9:100.0%(23/23)[85.69-100.00]</td><td colspan="1" rowspan="1">CTX-M-9:100.0%(30/30)[88.65-100.00]</td><td colspan="1" rowspan="1">CTX-M-9:17.8%(23/129)[12.18-25.33]</td><td colspan="1" rowspan="1">CTX-M-9:26.4%(578/2187)[24.62-28.32]</td><td colspan="1" rowspan="1">CTX-M-9:99.4%(768/773)[98.49-99.72]</td><td colspan="1" rowspan="1">CTX-M-9:99.1%(578/583)[98.01-99.63]</td><td colspan="1" rowspan="1">CTX-M-9:32.3%(768/2377)[30.46-34.22]</td><td colspan="1" rowspan="1">CTX-M-9:19.7%(583/2960)</td><td colspan="1" rowspan="1">CTX-M-9:4.9% of19017</td></tr><tr><td colspan="1" rowspan="1">CTX-M-9</td><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">97.7%(42/43)[87.94-99.59]</td><td colspan="1" rowspan="1">100.0%(292/292)[98.70-100.00]</td><td colspan="1" rowspan="1">Cefepime</td><td colspan="1" rowspan="1">CTX-M-9:28.0%(30/107)[20.40-37.20]</td><td colspan="1" rowspan="1">CTX-M-9:100.0%(52/52)[93.12-100.00]</td><td colspan="1" rowspan="1">CTX-M-9:100.0%(30/30)[88.6510.00]</td><td colspan="1" rowspan="1">CTX-M-9:40.3%(52/129)[32.24-48.94]</td><td colspan="1" rowspan="1">CTX-M-9:28.6%(574/2009)[26.64-300.59]</td><td colspan="1" rowspan="1">CTX-M-9:99.1%(942/951)[98.21-99.50]</td><td colspan="1" rowspan="1">CTX-M-9:98.5%(574/583)[97.09-9.19]</td><td colspan="1" rowspan="1">CTX-M-9:39.6%(942/2377)[37.68-41.61]</td><td colspan="1" rowspan="1">CTX-M-9:19.7%(583/2960)</td><td colspan="1" rowspan="1">CTX-M-9:4.9% of19017</td></tr><tr><td colspan="1" rowspan="1">CTX-M-9</td><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">97.7%(42/43)[87.94-99.59]</td><td colspan="1" rowspan="1">100.0%(292/292)[98.70-100.00]</td><td colspan="1" rowspan="1">Ceftazidime</td><td colspan="1" rowspan="1">CTX-M-9:10.4%(11/106)[5.89-17.63]</td><td colspan="1" rowspan="1">CTX-M-9:64.2%(34/53)[50.69-75.70]</td><td colspan="1" rowspan="1">CTX-M-9:36.7%(11/30)[21.87-54.49]</td><td colspan="1" rowspan="1">CTX-M-9:26.4%(34/129)[19.52-34.56]</td><td colspan="1" rowspan="1">CTX-M-9:16.7%(276/1651)[14.99-18.59]</td><td colspan="1" rowspan="1">CTX-M-9:76.5%(1002/1309) [74.18-78.76]</td><td colspan="1" rowspan="1">CTX-M-9:47.3%(276/583)[43.32-51.40]</td><td colspan="1" rowspan="1">CTX-M-9:42.2%(1002/2377) [40.18-44.15]</td><td colspan="1" rowspan="1">CTX-M-9:19.7%(583/2960)</td><td colspan="1" rowspan="1">CTX-M-9:4.9% of19017</td></tr><tr><td colspan="1" rowspan="2">AMR GeneTarget</td><td colspan="1" rowspan="2">Organism</td><td colspan="2" rowspan="1">Detection of AMR GeneTarget by AMR GenePanel versus WholeGenome SequencingCombined Clinical DevicePerformance -Stock and ProspectiveIsolates (from K191288)</td><td colspan="1" rowspan="2">Antimicrobial</td><td colspan="4" rowspan="1">Detection of AMR Gene Target by AMR GenePanel vs AST Non-SusceptibilityCombined Clinical Device Performance - Stock andProspective Isolates (from K191288) includingonly unique isolates (no replicates)</td><td colspan="5" rowspan="1">Detection of AMR Gene Target versus AST Non-Susceptibility inOpGen's SMART Study</td><td colspan="1" rowspan="2">%Organismswith AMRGeneTarget inNCBIPathogenDatabase(2019)</td></tr><tr><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1">PPV[95% CI]</td><td colspan="1" rowspan="1">NPV[95% CI]</td><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1">PPV[95% CI]</td><td colspan="1" rowspan="1">NPV[95% CI]</td><td colspan="1" rowspan="1">% Organismswith AMRGene Target</td></tr><tr><td colspan="1" rowspan="1">CTX-M-9</td><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">97.7%(42/43)[87.94-9.59]</td><td colspan="1" rowspan="1">100.0%(292/292)[98.70-0.00]</td><td colspan="1" rowspan="1">Ceftriaxone</td><td colspan="1" rowspan="1">CTX-M-9:23.1%(30/130)16.67-31.03]</td><td colspan="1" rowspan="1">CTX-M-9:100.0%(29/29)88.30100.00]</td><td colspan="1" rowspan="1">CTX-M-9:100.0%(30/30)88.5100.00]</td><td colspan="1" rowspan="1">CTX-M-9:22.5%(29/129)16.13-30.42]</td><td colspan="1" rowspan="1">CTX-M-9:27.4%(578/2113)25.50-9.30]</td><td colspan="1" rowspan="1">CTX-M-9:99.4%(842/847)98.63-99.75]</td><td colspan="1" rowspan="1">CTX-M-9:99.1%(578/583)[98.01-9.63]</td><td colspan="1" rowspan="1">CTX-M-9:35.4%(842/2377)[33.53-37.37]</td><td colspan="1" rowspan="1">CTX-M-9:19.7%(53/2960)</td><td colspan="1" rowspan="1">CTX-M-9:4.9% of19017</td></tr><tr><td colspan="1" rowspan="1">CTX-M-9</td><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">97.7%(42/43)[87.94-99.59]</td><td colspan="1" rowspan="1">100.0%(292/292)[98.70-100.00]</td><td colspan="1" rowspan="1">Amox.K.Clav</td><td colspan="1" rowspan="1">CTX-M-9:12.3%(14/114)[7.46-19.56]</td><td colspan="1" rowspan="1">CTX-M-9:64.4%(29/45)[49.84-76.78]</td><td colspan="1" rowspan="1">CTX-M-9:46.7%(14/30)[30.23-63.86]</td><td colspan="1" rowspan="1">CTX-M-9:22.5%(29/129)[16.13-30.42]</td><td colspan="1" rowspan="1">CTX-M-9:13.1%(238/1810)[11.67-14.78]</td><td colspan="1" rowspan="1">CTX-M-9:70.0%(805/1150)[67.29-72.58]</td><td colspan="1" rowspan="1">CTX-M-9:40.8%(238/583)[36.91-44.86]</td><td colspan="1" rowspan="1">CTX-M-9:33.9%(805/2377)[31.99-35.79]</td><td colspan="1" rowspan="1">CTX-M-9:19.7%(583/2960)</td><td colspan="1" rowspan="1">CTX-M-9:4.9% of19017</td></tr><tr><td colspan="1" rowspan="1">CTX-M-9</td><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">97.7%(42/43)[87.94-9.59]</td><td colspan="1" rowspan="1">100.0%(292/292)[98.70-1000]</td><td colspan="1" rowspan="1">Amp.Sulbactam</td><td colspan="1" rowspan="1">CTX-M-9:19.9%(29/146)14.20-27.07]</td><td colspan="1" rowspan="1">CTX-M-9:92.3%(12/13)66.6998.63]</td><td colspan="1" rowspan="1">CTX-M-9:96.7%(29/30)[83.33-99.41]</td><td colspan="1" rowspan="1">CTX-M-9:9.3%(12/129)5.40-15.56]</td><td colspan="1" rowspan="1">CTX-M-9:18.3%(453/2477)[16.82-19.86]</td><td colspan="1" rowspan="1">CTX-M-9:73.1%(353/483)[68.96-76.85]</td><td colspan="1" rowspan="1">CTX-M-9:77.7%(453/583)[74.15-80.89]</td><td colspan="1" rowspan="1">CTX-M-9:14.9%(353/2377)13.48-16.34]</td><td colspan="1" rowspan="1">CTX-M-9:19.7%(583/2960)</td><td colspan="1" rowspan="1">CTX-M-9:4.9% of19017</td></tr><tr><td colspan="1" rowspan="1">CTX-M-9</td><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">97.7%(42/43)[87.94-99.59]</td><td colspan="1" rowspan="1">100.0%(292/292)[98.70-100.00</td><td colspan="1" rowspan="1">Pip.Tazo</td><td colspan="1" rowspan="1">CTX-M-9:10.1%(7/69)[5.00-19.49]</td><td colspan="1" rowspan="1">CTX-M-9:74.4%(67/90)[64.56-82.33]</td><td colspan="1" rowspan="1">CTX-M-9:23.3%(7/30)[11.79-40.93]</td><td colspan="1" rowspan="1">CTX-M-9:51.9%(67/129)[43.39-60.38]</td><td colspan="1" rowspan="1">CTX-M-9:8.3%(57/687)[6.46-10.60]</td><td colspan="1" rowspan="1">CTX-M-9:76.9%(1747/2273) [75.08-78.55]</td><td colspan="1" rowspan="1">CTX-M-9:9.8%(57/583)[7.62-12.46]</td><td colspan="1" rowspan="1">CTX-M-9:73.5%(1747/2377) [71.68-75.23]</td><td colspan="1" rowspan="1">CTX-M-9:19.7%(583/2960)</td><td colspan="1" rowspan="1">CTX-M-9:4.9% of19017</td></tr><tr><td colspan="1" rowspan="1">CTX-M-9</td><td colspan="1" rowspan="1">K. pneumoniae</td><td colspan="1" rowspan="1">100.0%(24/24)[86.20-0.00]</td><td colspan="1" rowspan="1">99.7%(304/305)[98.17-99.94]</td><td colspan="1" rowspan="1">Cefazolin</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">CTX-M-9:5.9%(108/1824)[4.93-7.10]</td><td colspan="1" rowspan="1">CTX-M-9:99.5%(203/204)[97.28-99.91]</td><td colspan="1" rowspan="1">CTX-M-9:99.1%(108/109)[94.99-99.84]</td><td colspan="1" rowspan="1">CTX-M-9:10.6%(203/1919)9.28-12.03]</td><td colspan="1" rowspan="1">CTX-M-9:5.4%(109/2028)</td><td colspan="1" rowspan="1">CTX-M-9:9% of 7979</td></tr><tr><td colspan="1" rowspan="1">CTX-M-9</td><td colspan="1" rowspan="1">K. pneumoniae</td><td colspan="1" rowspan="1">100.0%(24/24)[86.20-00]</td><td colspan="1" rowspan="1">99.7%(304/305)[98.17-99.94]</td><td colspan="1" rowspan="1">Cefuroxime</td><td colspan="1" rowspan="1">CTX-M-9:11.8%(20/170)[7.75-17.47]</td><td colspan="1" rowspan="1">CTX-M-9:100.0%(15/15)[79.61-100]</td><td colspan="1" rowspan="1">CTX-M-9:100.0%(20/20)[83.89-0.00]</td><td colspan="1" rowspan="1">CTX-M-9:9.1%(15/165)[5.59-14.46]</td><td colspan="1" rowspan="1">CTX-M-9:5.8%(107/1832)[4.86-7.01]</td><td colspan="1" rowspan="1">CTX-M-9:99.0%(194/196)[96.36-999.72]</td><td colspan="1" rowspan="1">CTX-M-9:98.2%(107/109)[93.56-99.50]</td><td colspan="1" rowspan="1">CTX-M-9:10.1%(194/1919)[8.84-11.54]</td><td colspan="1" rowspan="1">CTX-M-9:5.4%(109/2028)</td><td colspan="1" rowspan="1">CTX-M-9:9% of7979</td></tr><tr><td colspan="1" rowspan="1">CTX-M-9</td><td colspan="1" rowspan="1">K. pneumoniae</td><td colspan="1" rowspan="1">100.0%(24/24)[86.20-100.00]</td><td colspan="1" rowspan="1">99.7%(304/305)[98.17-99.94]</td><td colspan="1" rowspan="1">Cefepime</td><td colspan="1" rowspan="1">CTX-M-9:12.5%(20/160)[8.24-18.52]</td><td colspan="1" rowspan="1">CTX-M-9:100.0%(25/25)[86.68-100.00]</td><td colspan="1" rowspan="1">CTX-M-9:100.0%(20/20)[83.89-100.00]</td><td colspan="1" rowspan="1">CTX-M-9:15.2%(25/165)10.48-21.41]</td><td colspan="1" rowspan="1">CTX-M-9:6.1%(105/1711)[5.09-7.38]</td><td colspan="1" rowspan="1">CTX-M-9:98.7%(313/317)96.0-99.51]</td><td colspan="1" rowspan="1">CTX-M-9:96.3%(105/109)90.94-98.56]</td><td colspan="1" rowspan="1">CTX-M-9:16.3%(313/1919)14.73-18.03]</td><td colspan="1" rowspan="1">CTX-M-9:5.4%(109/2028)</td><td colspan="1" rowspan="1">CTX-M-9:9% of7979</td></tr><tr><td colspan="1" rowspan="1">CTX-M-9</td><td colspan="1" rowspan="1">K. pneumoniae</td><td colspan="1" rowspan="1">100.0%(24/24)[86.20-000]</td><td colspan="1" rowspan="1">99.7%(304/305)[98.17-99.94]</td><td colspan="1" rowspan="1">Ceftazidime</td><td colspan="1" rowspan="1">CTX-M-9:11.7%(19/163)[7.59-17.49]</td><td colspan="1" rowspan="1">CTX-M-9:95.5%(21/22)78.20-99.19]</td><td colspan="1" rowspan="1">CTX-M-9:95.0%(19/20)76.39-99.11]</td><td colspan="1" rowspan="1">CTX-M-9:12.7%(21/165)[8.48-18.67]</td><td colspan="1" rowspan="1">CTX-M-9:5.0%(85/1710)[4.04-6.11]</td><td colspan="1" rowspan="1">CTX-M-9:92.5%(294/318)[89.02-94.88]</td><td colspan="1" rowspan="1">CTX-M-9:78.0%(85/109)[69.32-84.73]</td><td colspan="1" rowspan="1">CTX-M-9:15.3%(294/1919)[13.78-17.00]</td><td colspan="1" rowspan="1">CTX-M-9:5.4%(109/2028)</td><td colspan="1" rowspan="1">CTX-M-9:9% of7979</td></tr><tr><td colspan="1" rowspan="1">CTX-M-9</td><td colspan="1" rowspan="1">K. pneumoniae</td><td colspan="1" rowspan="1">100.0%(24/24)[86.20-0.00]</td><td colspan="1" rowspan="1">99.7%(304/305)[98.17-99.94]</td><td colspan="1" rowspan="1">Ceftriaxone</td><td colspan="1" rowspan="1">CTX-M-9:12.0%(20/167)[7.89-17.77]</td><td colspan="1" rowspan="1">CTX-M-9:100.0%(18/18)[82.41-0.00]</td><td colspan="1" rowspan="1">CTX-M-9:100.0%(20/20)[83.89-0.00]</td><td colspan="1" rowspan="1">CTX-M-9:10.9%(18/165)[7.01-6.58]</td><td colspan="1" rowspan="1">CTX-M-9:6.0%(106/1765)[4.99-7.21]</td><td colspan="1" rowspan="1">CTX-M-9:98.9%(260/263)[96.70-99.61</td><td colspan="1" rowspan="1">CTX-M-9:97.2%(106/109)[92.22-99.06]</td><td colspan="1" rowspan="1">CTX-M-9:13.5%(260/1919)[12.09-15.15]</td><td colspan="1" rowspan="1">CTX-M-9:5.4%(109/2028)</td><td colspan="1" rowspan="1">CTX-M-9:9% of7979</td></tr><tr><td colspan="1" rowspan="1">CTX-M-9</td><td colspan="1" rowspan="1">K. pneumoniae</td><td colspan="1" rowspan="1">100.0%(24/24)[86.20-100.00]</td><td colspan="1" rowspan="1">99.7%(304/305)[98.17-99.94]</td><td colspan="1" rowspan="1">Amox.K.Clav</td><td colspan="1" rowspan="1">CTX-M-9:12.3%(20/163)[8.09-18.19]</td><td colspan="1" rowspan="1">CTX-M-9:100.0%(22/22)85.13-0.00]</td><td colspan="1" rowspan="1">CTX-M-9:100.0%(20/20)[83.89100.00]</td><td colspan="1" rowspan="1">CTX-M-9:13.3%(22/165)[8.97-19.36]</td><td colspan="1" rowspan="1">CTX-M-9:5.2%(89/1716)[4.23-6.34]</td><td colspan="1" rowspan="1">CTX-M-9:93.6%(292/312)[90.31-95.81]</td><td colspan="1" rowspan="1">CTX-M-9:81.7%(89/109)[73.35-87.80]</td><td colspan="1" rowspan="1">CTX-M-9:15.2%(292/1919)[13.68-16.89]</td><td colspan="1" rowspan="1">CTX-M-9:5.4%(109/2028)</td><td colspan="1" rowspan="1">CTX-M-9:9% of7979</td></tr><tr><td colspan="1" rowspan="1">CTX-M-9</td><td colspan="1" rowspan="1">K. pneumoniae</td><td colspan="1" rowspan="1">100.0%(24/24)[86.20-100.00]</td><td colspan="1" rowspan="1">99.7%(304/305)[98.17-99.94]</td><td colspan="1" rowspan="1">Amp.Sulbactam</td><td colspan="1" rowspan="1">CTX-M-9:11.3%(20/177)[7.43-16.81]</td><td colspan="1" rowspan="1">CTX-M-9:100.0%(8/8)67.56-100]</td><td colspan="1" rowspan="1">CTX-M-9:100.0%(20/20)[83.89-10.00]</td><td colspan="1" rowspan="1">CTX-M-9:4.8%(8/165)[2.48-9.27]</td><td colspan="1" rowspan="1">CTX-M-9:5.7%(108/1909)[4.71-6.79]</td><td colspan="1" rowspan="1">CTX-M-9:99.2%(118/119)[95.39-99.85]</td><td colspan="1" rowspan="1">CTX-M-9:99.1%(108/109)[94.99-99.84]</td><td colspan="1" rowspan="1">CTX-M-9:6.1%(118/1919)[5.16-7.31]</td><td colspan="1" rowspan="1">CTX-M-9:5.4%(109/2028)</td><td colspan="1" rowspan="1">CTX-M-9:9% of7979</td></tr><tr><td colspan="1" rowspan="1">CTX-M-9</td><td colspan="1" rowspan="1">K. pneumoniae</td><td colspan="1" rowspan="1">100.0%(24/24)[86.20-000]</td><td colspan="1" rowspan="1">99.7%(304/305)[98.17-99.94]</td><td colspan="1" rowspan="1">Pip.Tazo</td><td colspan="1" rowspan="1">CTX-M-9:12.2%(18/147)[7.89-18.53]</td><td colspan="1" rowspan="1">CTX-M-9:94.7%(36/38)82.71-98.54]</td><td colspan="1" rowspan="1">CTX-M-9:90.0%(18/20)69.90-97.21]</td><td colspan="1" rowspan="1">CTX-M-9:21.8%(36/165)[16.20-28.72]</td><td colspan="1" rowspan="1">CTX-M-9:5.4%(75/1400)[4.30-6.66]</td><td colspan="1" rowspan="1">CTX-M-9:94.6%(594/628)[92.53-96.10]</td><td colspan="1" rowspan="1">CTX-M-9:68.8%(75/109)5.60-76.74]</td><td colspan="1" rowspan="1">CTX-M-9:31.0%(594/1919)[28.92-33.06]</td><td colspan="1" rowspan="1">CTX-M-9:5.4%(109/2028)</td><td colspan="1" rowspan="1">CTX-M-9:9% of7979</td></tr><tr><td colspan="1" rowspan="1">CTX-M-9</td><td colspan="1" rowspan="1">P. mirabilis</td><td colspan="1" rowspan="1">100.0%(6/6)[60.97-100.00]</td><td colspan="1" rowspan="1">100.0%(149/149)[97.49-100.00]</td><td colspan="1" rowspan="1">Cefazolin</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">CTX-M-9:6.0%(34/571)[4.29-8.21]</td><td colspan="1" rowspan="1">CTX-M-9:99.5%(583/586)[98.51-99.83]</td><td colspan="1" rowspan="1">CTX-M-9:91.9%(34/37)[78.70-97.20]</td><td colspan="1" rowspan="1">CTX-M-9:52.1%(583/1120)[49.13-54.97]</td><td colspan="1" rowspan="1">CTX-M-9:3.2%(7/1157)</td><td colspan="1" rowspan="1">CTX-M-9:6.9% of144</td></tr><tr><td colspan="1" rowspan="1">CTX-M-9</td><td colspan="1" rowspan="1">P. mirabilis</td><td colspan="1" rowspan="1">100.0%(6/6)[60.97-00]</td><td colspan="1" rowspan="1">100.0%(149/149)[97.49-1000]</td><td colspan="1" rowspan="1">Cefuroxime</td><td colspan="1" rowspan="1">CTX-M-9:13.3%(6/45)6.26-26.18]</td><td colspan="1" rowspan="1">CTX-M-9:100.0%(33/33)[89.57-100.00]</td><td colspan="1" rowspan="1">CTX-M-9:100.0%(6/6)60.97-100.00]</td><td colspan="1" rowspan="1">CTX-M-9:45.8%(33/72)[34.83-57.26]</td><td colspan="1" rowspan="1">CTX-M-9:6.9%(34/495)[4.96-9.45]</td><td colspan="1" rowspan="1">CTX-M-9:99.5%(659/662)[98.68-99.]</td><td colspan="1" rowspan="1">CTX-M-9:91.9%(34/37)78.70-97.20]</td><td colspan="1" rowspan="1">CTX-M-9:58.8%(659/1120)[55.93-61.69]</td><td colspan="1" rowspan="1">CTX-M-9:3.2%(37/1157)</td><td colspan="1" rowspan="1">CTX-M-9:6.9% of144</td></tr><tr><td colspan="1" rowspan="2">AMR GeneTarget</td><td colspan="1" rowspan="2">Organism</td><td colspan="2" rowspan="1">Detection of AMR GeneTarget by AMR GenePanel versus WholeGenome SequencingCombined Clinical DevicePerformance -Stock and ProspectiveIsolates (from K191288)</td><td colspan="1" rowspan="2">Antimicrobial</td><td colspan="4" rowspan="1">Detection of AMR Gene Target by AMR GenePanel vs AST Non-SusceptibilityCombined Clinical Device Performance - Stock andProspective Isolates (from K191288) includingonly unique isolates (no replicates)</td><td colspan="5" rowspan="1">Detection of AMR Gene Target versus AST Non-Susceptibility inOpGen's SMART Study</td><td colspan="1" rowspan="1">%Organismswith AMRGeneTarget inNCBIPathogen</td></tr><tr><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1">PPV[95% CI]</td><td colspan="1" rowspan="1">NPV[95% CI]</td><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1">PPV[95% CI]</td><td colspan="1" rowspan="1">NPV[95% CI]</td><td colspan="1" rowspan="1">% Organismswith AMRGene Target</td><td colspan="1" rowspan="1">Database(2019)</td></tr><tr><td colspan="1" rowspan="1">CTX-M-9</td><td colspan="1" rowspan="1">P. mirabilis</td><td colspan="1" rowspan="1">100.0%(6/6[60.97-100.00]</td><td colspan="1" rowspan="1">100.0%(149/149)[97.49-100.00]</td><td colspan="1" rowspan="1">Cefepime</td><td colspan="1" rowspan="1">CTX-M-9:15.0%(6/40)[7.06-29.07]</td><td colspan="1" rowspan="1">CTX-M-9:100.0%(38/38)[90.82-100]</td><td colspan="1" rowspan="1">CTX-M-9:100.0%(6/6)60.97-100.00]</td><td colspan="1" rowspan="1">CTX-M-9:52.8%(38/72)41.40-63.87]</td><td colspan="1" rowspan="1">CTX-M-9:9.3%(31/33)[6.62-12.87]</td><td colspan="1" rowspan="1">CTX-M-9:99.3%(817/823)[98.42-99.67]</td><td colspan="1" rowspan="1">CTX-M-9:83.8%(31/37)[68.86-92.35]</td><td colspan="1" rowspan="1">CTX-M-9:72.9%(817/1120)70.27-75.47]</td><td colspan="1" rowspan="1">CTX-M-9:3.2%(37/1157)</td><td colspan="1" rowspan="1">CTX-M-9:6.9% of144</td></tr><tr><td colspan="1" rowspan="1">CTX-M-9</td><td colspan="1" rowspan="1">P. mirabilis</td><td colspan="1" rowspan="1">100.0%(6/6)60.97-100.00]</td><td colspan="1" rowspan="1">100.0%(149/149)[97.49-100.00]</td><td colspan="1" rowspan="1">Ceftazidime</td><td colspan="1" rowspan="1">CTX-M-9:0.0%(0/23)0.00-14.31]</td><td colspan="1" rowspan="1">CTX-M-9:89.1%(49/55)[78.17-94.90]</td><td colspan="1" rowspan="1">CTX-M-9:0.0%(0/6)[0.0-39.03]</td><td colspan="1" rowspan="1">CTX-M-9:68.1%(49/72)56.61-77.67]</td><td colspan="1" rowspan="1">CTX-M-9:0.8%(2/259)[0.21-2.77]</td><td colspan="1" rowspan="1">CTX-M-9:96.1%(863/898)[94.63-97.18]</td><td colspan="1" rowspan="1">CTX-M-9:5.4%(/37)[1.50-17.70]</td><td colspan="1" rowspan="1">CTX-M-9:77.1%(863/1120)[74.50-79.42]</td><td colspan="1" rowspan="1">CTX-M-9:3.2%(37/1157)</td><td colspan="1" rowspan="1">CTX-M-9:6.9% of144</td></tr><tr><td colspan="1" rowspan="1">CTX-M-9</td><td colspan="1" rowspan="1">P. mirabilis</td><td colspan="1" rowspan="1">100.0%(6/6)[60.97-100.00]</td><td colspan="1" rowspan="1">100.0%(149/149)[97.49-100.00]</td><td colspan="1" rowspan="1">Ceftriaxone</td><td colspan="1" rowspan="1">CTX-M-9:14.3%(6/42)[6.72-27.84]</td><td colspan="1" rowspan="1">CTX-M-9:100.0%(36/36)90.36-100.00]</td><td colspan="1" rowspan="1">CTX-M-9:100.0%(6/6)[60.97-100.00]</td><td colspan="1" rowspan="1">CTX-M-9:500.0%(36/72)38.75-61.25]</td><td colspan="1" rowspan="1">CTX-M-9:7.1%(34/478)[5.13-9.78]</td><td colspan="1" rowspan="1">CTX-M-9:99.6%(676/679)[98.71-99.85]</td><td colspan="1" rowspan="1">CTX-M-9:91.9%(34/37)[78.70-97.20]</td><td colspan="1" rowspan="1">CTX-M-9:60.4%(676/1120)[57.46-63.18</td><td colspan="1" rowspan="1">CTX-M-9:3.2%(37/1157)</td><td colspan="1" rowspan="1">CTX-M-9:6.9% of144</td></tr><tr><td colspan="1" rowspan="1">CTX-M-9</td><td colspan="1" rowspan="1">P. mirabilis</td><td colspan="1" rowspan="1">100.0%(6/6)60.97-100.00]</td><td colspan="1" rowspan="1">100.0%(149/149)[97.49100.00]</td><td colspan="1" rowspan="1">Amox.K.Clav</td><td colspan="1" rowspan="1">CTX-M-9:7.7%(2/26)[2.14-24.14]</td><td colspan="1" rowspan="1">CTX-M-9:92.3%(48/52)[81.83-96.97]</td><td colspan="1" rowspan="1">CTX-M-9:33.3%(2/6)[9.68-70.00]</td><td colspan="1" rowspan="1">CTX-M-9:66.7%(48/72)55.18 76.47]</td><td colspan="1" rowspan="1">CTX-M-9:2.2%(8/359)[1.13-4.33]</td><td colspan="1" rowspan="1">CTX-M-9:96.4%(769/798)[94.83-97.46]</td><td colspan="1" rowspan="1">CTX-M-9:21.6%(8/37)11. 937.20]</td><td colspan="1" rowspan="1">CTX-M-9:68.7%(769/1120)[65.88-71.31]</td><td colspan="1" rowspan="1">CTX-M-9:3.2%(37/157)</td><td colspan="1" rowspan="1">CTX-M-9:6.9% of144</td></tr><tr><td colspan="1" rowspan="1">CTX-M-9</td><td colspan="1" rowspan="1">P. mirabilis</td><td colspan="1" rowspan="1">100.0%(6/6)[60.97-100.00]</td><td colspan="1" rowspan="1">100.0%(149/149)[97.49-100.00]</td><td colspan="1" rowspan="1">Amp.Sulbactam</td><td colspan="1" rowspan="1">CTX-M-9:11.9%(5/42)[5.19-25.00]</td><td colspan="1" rowspan="1">CTX-M-9:97.2%(35/36)[85.83-99.51]</td><td colspan="1" rowspan="1">CTX-M-9:83.3%(5/6)43.65-96.99]</td><td colspan="1" rowspan="1">CTX-M-9:48.6%(35/72)[37.43-59..93]</td><td colspan="1" rowspan="1">CTX-M-9:4.6%(27/582)[3.21-6.67]</td><td colspan="1" rowspan="1">CTX-M-9:98.3%565/575)[96.83-99.05]</td><td colspan="1" rowspan="1">CTX-M-9:73.0%(27/37)[57.02-84.60]</td><td colspan="1" rowspan="1">CTX-M-9:50.4%(65/1120)[47.52-53.37]</td><td colspan="1" rowspan="1">CTX-M-9:3.2%(37/1157)</td><td colspan="1" rowspan="1">CTX-M-9:6.9% of144</td></tr><tr><td colspan="1" rowspan="1">CTX-M-9</td><td colspan="1" rowspan="1">P. mirabilis</td><td colspan="1" rowspan="1">100.0%(6/6)60.97-10.00]</td><td colspan="1" rowspan="1">100.0%(149/149)[97.49-100.00]</td><td colspan="1" rowspan="1">Pip.Tazo</td><td colspan="1" rowspan="1">CTX-M-9:0.0%(0/4)[0.00-48.99]</td><td colspan="1" rowspan="1">CTX-M-9:91.9%(68/74)[83.42-96.23]</td><td colspan="1" rowspan="1">CTX-M-9:0.0%(0/6)[0.00-39.03]</td><td colspan="1" rowspan="1">CTX-M-9:94.4%(68/72)[86.57-97.82]</td><td colspan="1" rowspan="1">CTX-M-9:0.0%(0/31)[0.00-11.03]</td><td colspan="1" rowspan="1">CTX-M-9:96.7%(1089/1126) [95.50-97.61]</td><td colspan="1" rowspan="1">CTX-M-9:0.0%(0/37)[0.00-9.41]</td><td colspan="1" rowspan="1">CTX-M-9:97.2%(1089/1120) [96.10-9804]</td><td colspan="1" rowspan="1">CTX-M-9:3.2%(37/1157)</td><td colspan="1" rowspan="1">CTX-M-9:6.9% of144</td></tr><tr><td colspan="1" rowspan="1">CTX-M-9</td><td colspan="1" rowspan="1">K. aerogenes</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">Cefuroxime</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">CTX-M-9:0.4% of223</td></tr><tr><td colspan="1" rowspan="1">CTX-M-9</td><td colspan="1" rowspan="1">K. aerogenes</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">Cefepime</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">CTX-M-9:0.4% of223</td></tr><tr><td colspan="1" rowspan="2">AMR GeneTarget</td><td colspan="1" rowspan="2">Organism</td><td colspan="2" rowspan="1">Detection of AMR GeneTarget by AMR GenePanel versus WholeGenome SequencingCombined Clinical DevicePerformanceStock and ProspectiveIsolates (from K191288)</td><td colspan="1" rowspan="2">Antimicrobial</td><td colspan="4" rowspan="1">Detection of AMR Gene Target by AMR GenePanel vs AST Non-SusceptibilityCombined Clinical Device Performance - Stock andProspective Isolates (from K191288) includingonly unique isolates (no replicates)</td><td colspan="5" rowspan="1">Detection of AMR Gene Target versus AST Non-Susceptibility inOpGen's SMART Study</td><td colspan="1" rowspan="2">%Organismswith AMRGeneTarget inNCBIPathogenDatabase(2019)</td></tr><tr><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1">PPV[95% CI]</td><td colspan="1" rowspan="1">NPV[95% CI]</td><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1">PPV[95% CI]</td><td colspan="1" rowspan="1">NPV[95% CI]</td><td colspan="1" rowspan="1">% Organismswith AMRGene Target</td></tr><tr><td colspan="1" rowspan="1">CTX-M-9</td><td colspan="1" rowspan="1">K. aerogenes</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">Ceftazidime</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">CTX-M-9:0.4% of223</td></tr><tr><td colspan="1" rowspan="1">CTX-M-9</td><td colspan="1" rowspan="1">K. aerogenes</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">Ceftriaxone</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">CTX-M-9:0.4% of223</td></tr><tr><td colspan="1" rowspan="1">CTX-M-9</td><td colspan="1" rowspan="1">K. aerogenes</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">Pip.Tazo</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">CTX-M-9:0.4% of223</td></tr><tr><td colspan="1" rowspan="1">CTX-M-9</td><td colspan="1" rowspan="1">K.michiganensis</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">Cefazolin</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">CTX-M-9:0% of 89</td></tr><tr><td colspan="1" rowspan="1">CTX-M-9</td><td colspan="1" rowspan="1">K.michiganensis</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">Cefuroxime</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">CTX-M-9:0% of 89</td></tr><tr><td colspan="1" rowspan="1">CTX-M-9</td><td colspan="1" rowspan="1">K.michiganensis</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">Cefepime</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">CTX-M-9:0% of 89</td></tr><tr><td colspan="1" rowspan="1">CTX-M-9</td><td colspan="1" rowspan="1">K.michiganensis</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">Ceftazidime</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">CTX-M-9:0% of 89</td></tr><tr><td colspan="1" rowspan="1">CTX-M-9</td><td colspan="1" rowspan="1">K.michiganensis</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">Ceftriaxone</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">CTX-M-9:0% of 89</td></tr><tr><td colspan="1" rowspan="1">CTX-M-9</td><td colspan="1" rowspan="1">K.michiganensis</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">Amox.K.Clav</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">CTX-M-9:0% of 89</td></tr><tr><td colspan="1" rowspan="1">CTX-M-9</td><td colspan="1" rowspan="1">K.michiganensis</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">Amp.Sulbactam</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">CTX-M-9:0% of 89</td></tr><tr><td colspan="1" rowspan="1">CTX-M-9</td><td colspan="1" rowspan="1">K.michiganensis</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">Pip.Tazo</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">CTX-M-9:0% of 89</td></tr><tr><td colspan="1" rowspan="1">CTX-M-9</td><td colspan="1" rowspan="1">K. oxytoca</td><td colspan="1" rowspan="1">NaN</td><td colspan="1" rowspan="1">100.0%(5/5)[56.55-100.00]</td><td colspan="1" rowspan="1">Cefazolin</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">CTX-M-9:0.7% of273</td></tr><tr><td colspan="1" rowspan="1">CTX-M-9</td><td colspan="1" rowspan="1">K. oxytoca</td><td colspan="1" rowspan="1">NaN</td><td colspan="1" rowspan="1">100.0%(5/5)[56.55-100.00]</td><td colspan="1" rowspan="1">Cefuroxime</td><td colspan="1" rowspan="1">CTX-M-9:0.0%(0/4)[0.00-48.99]</td><td colspan="1" rowspan="1">CTX-M-9:100.0%(1/1)[20.65-100.00]</td><td colspan="1" rowspan="1">CTX-M-9:NaN</td><td colspan="1" rowspan="1">CTX-M-9:20.0%(1//5)[3.62-62.45]</td><td colspan="1" rowspan="1">NotEvaluated</td><td colspan="1" rowspan="1">NotEvaluated</td><td colspan="1" rowspan="1">NotEvaluated</td><td colspan="1" rowspan="1">NotEvaluated</td><td colspan="1" rowspan="1">NotEvaluated</td><td colspan="1" rowspan="1">CTX-M-9:0.7% of273</td></tr><tr><td colspan="1" rowspan="2">AMR GeneTarget</td><td colspan="1" rowspan="2">Organism</td><td colspan="2" rowspan="1">Detection of AMR GeneTarget by AMR GenePanel versus WholeGenome SequencingCombined Clinical DevicePerformance -Stock and ProspectiveIsolates (from K191288)</td><td colspan="1" rowspan="2">Antimicrobial</td><td colspan="4" rowspan="1">Detection of AMR Gene Target by AMR GenePanel vs AST Non-SusceptibilityCombined Clinical Device Performance - Stock andiva92only unique isolates (no replicates)</td><td colspan="5" rowspan="1">Detection of AMR Gene Target versus AST Non-Susceptibility inOpGen's SMART Study</td><td colspan="1" rowspan="2">%Organismswith AMRGeneTarget inNCBIPathogenDatabase2019)</td></tr><tr><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1">PPV[95% CI]</td><td colspan="1" rowspan="1">NPV[95% CI]</td><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1">PPV[95% CI]</td><td colspan="1" rowspan="1">NPV[95% CI]</td><td colspan="1" rowspan="1">% Organismswith AMRGene Target</td></tr><tr><td colspan="1" rowspan="1">CTX-M-9</td><td colspan="1" rowspan="1">K. oxytoca</td><td colspan="1" rowspan="1">NaN</td><td colspan="1" rowspan="1">100.0%(5//5)[56.55-100.00]</td><td colspan="1" rowspan="1">Cefepime</td><td colspan="1" rowspan="1">CTX-M-9:0.0%((0/4)[0.00-48.99]</td><td colspan="1" rowspan="1">CTX-M-9:100.0%(1/1)20.65-100]</td><td colspan="1" rowspan="1">CTX-M-9:NaN</td><td colspan="1" rowspan="1">CTX-M-9:20.0%(1/5)[3.62-62.45]</td><td colspan="1" rowspan="1">NotEvaluated</td><td colspan="1" rowspan="1">NotEvaluated</td><td colspan="1" rowspan="1">NotEvaluated</td><td colspan="1" rowspan="1">NotEvaluated</td><td colspan="1" rowspan="1">NotEvaluated</td><td colspan="1" rowspan="1">CTX-M-9:0.7% of273</td></tr><tr><td colspan="1" rowspan="1">CTX-M-9</td><td colspan="1" rowspan="1">K. oxytoca</td><td colspan="1" rowspan="1">NaN</td><td colspan="1" rowspan="1">100.0%(5/5)[56.55-100.00]</td><td colspan="1" rowspan="1">Ceftazidime</td><td colspan="1" rowspan="1">CTX-M-9:0.0%(0/2)[0.00-65.76]</td><td colspan="1" rowspan="1">CTX-M-9:100.0%(3/3)[43.85-100.00]</td><td colspan="1" rowspan="1">CTX-M-9:NaN</td><td colspan="1" rowspan="1">CTX-M-9:60.0%(3/5)[23.07-88.24]</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">CTX-M-9:0.7% of273</td></tr><tr><td colspan="1" rowspan="1">CTX-M-9</td><td colspan="1" rowspan="1">K. oxytoca</td><td colspan="1" rowspan="1">NaN</td><td colspan="1" rowspan="1">100.0%(5/5)[56.55-100]</td><td colspan="1" rowspan="1">Ceftriaxone</td><td colspan="1" rowspan="1">CTX-M-9:0.0%(00/4)[0.00-48.99]</td><td colspan="1" rowspan="1">CTX-M-9:100.0%(1/1)[20.65-100.00]</td><td colspan="1" rowspan="1">CTX-M-9:NaN</td><td colspan="1" rowspan="1">CTX-M-9:20.0%(1/5)[3.62-62.45]</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">CTX-M-9:0.7% of273</td></tr><tr><td colspan="1" rowspan="1">CTX-M-9</td><td colspan="1" rowspan="1">K. oxytoca</td><td colspan="1" rowspan="1">NaN</td><td colspan="1" rowspan="1">100.0%(5/5)56.55-100.00]</td><td colspan="1" rowspan="1">Amox.K.Clav</td><td colspan="1" rowspan="1">CTX-M-9:0.0%(0/4)[0.00-48.99]</td><td colspan="1" rowspan="1">CTX-M-9:100.0%(1/1)[20.65-100.00]</td><td colspan="1" rowspan="1">CTX-M-9:NaN</td><td colspan="1" rowspan="1">CTX-M-9:20.0%(1/5)[3.62-62.45]</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">CTX-M-9:0.7% of273</td></tr><tr><td colspan="1" rowspan="1">CTX-M-9</td><td colspan="1" rowspan="1">K. oxytoca</td><td colspan="1" rowspan="1">NaN</td><td colspan="1" rowspan="1">100.0%(5/5)[56.55-100.00]</td><td colspan="1" rowspan="1">Amp.Sulbactam</td><td colspan="1" rowspan="1">CTX-M-9:0.0%(0/5)0.00-43.45]</td><td colspan="1" rowspan="1">CTX-M-9:NaN</td><td colspan="1" rowspan="1">CTX-M-9:NaN</td><td colspan="1" rowspan="1">CTX-M-9:0.0%(0/5)[0.00-43.45]</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">CTX-M-9:0.7% of273</td></tr><tr><td colspan="1" rowspan="1">CTX-M-9</td><td colspan="1" rowspan="1">K. oxytoca</td><td colspan="1" rowspan="1">NaN</td><td colspan="1" rowspan="1">100.0%(5/5)56.55-10.00]</td><td colspan="1" rowspan="1">Pip.Tazo</td><td colspan="1" rowspan="1">CTX-M-9:0.0%(0/2)[0.00-65.76]</td><td colspan="1" rowspan="1">CTX-M-9:100.0%(3/3)[43.85-100.00]</td><td colspan="1" rowspan="1">CTX-M-9:NaN</td><td colspan="1" rowspan="1">CTX-M-9:60.0%(3/5)[23.07-88.24]</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">CTX-M-9:0.7% of273</td></tr><tr><td colspan="1" rowspan="1">CTX-M-9</td><td colspan="1" rowspan="1">K. variicola</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">Cefazolin</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">CTX-M-9:1.5% of272</td></tr><tr><td colspan="1" rowspan="2">AMR GeneTarget</td><td colspan="1" rowspan="2">Organism</td><td colspan="2" rowspan="1">Detection of AMR GeneTarget by AMR GenePanel versus WholeGenome SequencingCombined Clinical DevicePerformance -Stock and ProspectiveIsolates (from K191288)</td><td colspan="1" rowspan="2">Antimicrobial</td><td colspan="4" rowspan="1">Detection of AMR Gene Target by AMR GenePanel vs AST Non-SusceptibilityCombined Clinical Device Performance - Stock andProspective Isolates (from K191288) includingonly unique isolates (no replicates)</td><td colspan="5" rowspan="1">Detection of AMR Gene Target versus AST Non-Susceptibility inOpGen's SMART Study</td><td colspan="1" rowspan="2">%Organismswith AMRGeneTarget inNCBIPathogenDatabase(2019)</td></tr><tr><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1">PPV[95% CI]</td><td colspan="1" rowspan="1">NPV[95% CI]</td><td colspan="1" rowspan="1">PPA[95% Cl]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1">PPV[95% CI]</td><td colspan="1" rowspan="1">NPV[95% CI]</td><td colspan="1" rowspan="1">% Organismswith AMRGene Target</td></tr><tr><td colspan="1" rowspan="1">CTX-M-9</td><td colspan="1" rowspan="1">K. variicola</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">Cefuroxime</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">CTX-M-9:1.5% of272</td></tr><tr><td colspan="1" rowspan="1">CTX-M-9</td><td colspan="1" rowspan="1">K. variicola</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">Cefepime</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">CTX-M-9:1.5% of272</td></tr><tr><td colspan="1" rowspan="1">CTX-M-9</td><td colspan="1" rowspan="1">K. variicola</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">Ceftazidime</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">CTX-M-9:1.5% of272</td></tr><tr><td colspan="1" rowspan="1">CTX-M-9</td><td colspan="1" rowspan="1">K. variicola</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">Ceftriaxone</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">CTX-M-9:1.5% of272</td></tr><tr><td colspan="1" rowspan="1">CTX-M-9</td><td colspan="1" rowspan="1">K. variicola</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">Amox.K.Clav</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">CTX-M-9:1.5% of272</td></tr><tr><td colspan="1" rowspan="1">CTX-M-9</td><td colspan="1" rowspan="1">K. variicola</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">Amp.Sulbactam</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">CTX-M-9:1.5% of272</td></tr><tr><td colspan="1" rowspan="1">CTX-M-9</td><td colspan="1" rowspan="1">K. variicola</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">Pip.Tazo</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">CTX-M-9:1.5% of272</td></tr><tr><td colspan="1" rowspan="1">CTX-M-9</td><td colspan="1" rowspan="1">K.quasipneumoniae</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">Cefazolin</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">CTX-M-9:3.5% of259</td></tr><tr><td colspan="1" rowspan="1">CTX-M-9</td><td colspan="1" rowspan="1">K.quasipneumoniae</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">Cefuroxime</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">CTX-M-9:3.5% of259</td></tr><tr><td colspan="1" rowspan="1">CTX-M-9</td><td colspan="1" rowspan="1">K.quasipneumoniae</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">Cefepime</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">CTX-M-9:3.5% of259</td></tr><tr><td colspan="1" rowspan="1">CTX-M-9</td><td colspan="1" rowspan="1">K.quasipneumoniae</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">Ceftazidime</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">CTX-M-9:3.5% of259</td></tr><tr><td colspan="1" rowspan="1">CTX-M-9</td><td colspan="1" rowspan="1">K.quasipneumoniae</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">Ceftriaxone</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">CTX-M-9:3.5% of259</td></tr><tr><td colspan="1" rowspan="2">AMR GeneTarget</td><td colspan="1" rowspan="2">Organism</td><td colspan="2" rowspan="1">Detection of AMR GeneTarget by AMR GenePanel versus WholeGenome SequencingCombined Clinical DevicePerformanceStock and ProspectiveIsolates (from K191288)</td><td colspan="1" rowspan="2">Antimicrobial</td><td colspan="4" rowspan="1">Detection of AMR Gene Target by AMR GenePanel vs AST Non-SusceptibilityCombined Clinical Device Performance - Stock andProspective Isolates fromK191288)incudingonly unique isolates (no replicates)</td><td colspan="5" rowspan="1">Detection of AMR Gene Target versus AST Non-Susceptibility inOpGen's SMART Study</td><td colspan="1" rowspan="2">%Organismswith AMRGeneTarget inNCBIPathogenDatabase(2019)</td></tr><tr><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1">PPV[95% CI]</td><td colspan="1" rowspan="1">NPV[95% CI]</td><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1">PPV[95% CI]</td><td colspan="1" rowspan="1">NPV[95% CI]</td><td colspan="1" rowspan="1">% Organismswith AMRGene Target</td></tr><tr><td colspan="1" rowspan="1">CTX-M-9</td><td colspan="1" rowspan="1">K.quasipneumoniae</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">Amox.K.Clav</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">CTX-M-9:3.5% of259</td></tr><tr><td colspan="1" rowspan="1">CTX-M-9</td><td colspan="1" rowspan="1">K.quasipneumoniae</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">Amp.Sulbactam</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">CTX-M-9:3.5% of259</td></tr><tr><td colspan="1" rowspan="1">CTX-M-9</td><td colspan="1" rowspan="1">K.quasipneumoniae</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">Pip.Tazo</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">CTX-M-9:3.5% of259</td></tr><tr><td colspan="1" rowspan="1">CTX-M-9</td><td colspan="1" rowspan="1">E. asburiae</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">Cefepime</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">CTX-M-9:1.4% of 73</td></tr><tr><td colspan="1" rowspan="1">CTX-M-9</td><td colspan="1" rowspan="1">E. asburiae</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">Ceftazidime</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">CTX-M-9:1.4% of 73</td></tr><tr><td colspan="1" rowspan="1">CTX-M-9</td><td colspan="1" rowspan="1">E. asburiae</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">Ceftriaxone</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">CTX-M-9:1.4% of 73</td></tr><tr><td colspan="1" rowspan="1">CTX-M-9</td><td colspan="1" rowspan="1">E. asburiae</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">Pip.Tazo</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">CTX-M-9:1.4% of 73</td></tr><tr><td colspan="1" rowspan="1">CTX-M-9</td><td colspan="1" rowspan="1">E. cloacaecomplex</td><td colspan="1" rowspan="1">100.0%(1/1)20.5-100.00]</td><td colspan="1" rowspan="1">100.0%(13/13)[77.19-100.00]</td><td colspan="1" rowspan="1">Cefepime</td><td colspan="1" rowspan="1">CTX-M-9:7.1%(1/14)[1.27-31.47]</td><td colspan="1" rowspan="1">CTX-M-9:NaN</td><td colspan="1" rowspan="1">CTX-M-9:100.0%(1/1)[20.65-100.00]</td><td colspan="1" rowspan="1">CTX-M-9:0.0%0/13)[0.00-22.81]</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">CTX-M-9:4.6% of563</td></tr><tr><td colspan="1" rowspan="1">CTX-M-9</td><td colspan="1" rowspan="1">E. cloacaecomplex</td><td colspan="1" rowspan="1">100.0%(1/1)[20.65-00]</td><td colspan="1" rowspan="1">100.0%(13/13)[77.19-100]</td><td colspan="1" rowspan="1">Ceftazidime</td><td colspan="1" rowspan="1">CTX-M-9:7.1%(1/14)[1.27-31.47]</td><td colspan="1" rowspan="1">CTX-M-9:NaN</td><td colspan="1" rowspan="1">CTX-M-9:100.0%(1/1)20.65-100.00]</td><td colspan="1" rowspan="1">CTX-M-9:0.0%(0/13)00.00-22.81]</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">CTX-M-9:4.6% of563</td></tr><tr><td colspan="1" rowspan="1">CTX-M-9</td><td colspan="1" rowspan="1">E. cloacaecomplex</td><td colspan="1" rowspan="1">100.0%(1/1)[20.65-100.00]</td><td colspan="1" rowspan="1">100.0%(13/13)[77.19-100.00]</td><td colspan="1" rowspan="1">Ceftriaxone</td><td colspan="1" rowspan="1">CTX-M-9:7.1%(1/14)[1.27-31.47]</td><td colspan="1" rowspan="1">CTX-M-9:NaN</td><td colspan="1" rowspan="1">CTX-M-9:100.0%(1/1)[20.65-100.00]</td><td colspan="1" rowspan="1">CTX-M-9:0.0%(0/13)[0.00-22.81]</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">CTX-M-9:4.6% of563</td></tr><tr><td colspan="1" rowspan="2">AMR GeneTarget</td><td colspan="1" rowspan="2">Organism</td><td colspan="2" rowspan="1">Detection of AMR GeneTarget by AMR GenePanel versus WholeGenome SequencingCombined Clinical DevicePerformance-Stock and ProspectiveIsolates (from K191288)</td><td colspan="1" rowspan="2">Antimicrobial</td><td colspan="4" rowspan="1">Detection of AMR Gene Target by AMR GenePanel vs AST Non-SusceptibilityCombined Clinical Device Performance - Stock andProspective Isolates (from K191288) includingonly unique isolates (no replicates)</td><td colspan="5" rowspan="1">Detection of AMR Gene Target versus AST Non-Susceptibility inOpGen's SMART Study</td><td colspan="1" rowspan="2">%Organismswith AMRGeneTarget inNCBIPathogenDatabase(2019)</td></tr><tr><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1">PPV[95% CI]</td><td colspan="1" rowspan="1">NPV[95% CI]</td><td colspan="1" rowspan="1">PPA[95% Cl]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1">PPV[95% CI]</td><td colspan="1" rowspan="1">NPV[95% CI]</td><td colspan="1" rowspan="1">% Organismswith AMRGene Target</td></tr><tr><td colspan="1" rowspan="1">CTX-M-9</td><td colspan="1" rowspan="1">E. cloacaecomplex</td><td colspan="1" rowspan="1">100.0%(1/1)[20.65-100.00]</td><td colspan="1" rowspan="1">100.0%(13/13)[77.19-100.00]</td><td colspan="1" rowspan="1">Pip.Tazo</td><td colspan="1" rowspan="1">CTX-M-9:7.1%(1/14)[1.27-31.47]</td><td colspan="1" rowspan="1">CTX-M-9:NaN</td><td colspan="1" rowspan="1">CTX-M-9:100.0%(1/1)20.65-100.00]</td><td colspan="1" rowspan="1">CTX-M-9:0.0%0/13).0-22.81]</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">CTX-M-9:4.6% of563</td></tr><tr><td colspan="1" rowspan="1">CTX-M-9</td><td colspan="1" rowspan="1">E. hormaechei</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">Cefepime</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">CTX-M-9:4.3% of678</td></tr><tr><td colspan="1" rowspan="1">CTX-M-9</td><td colspan="1" rowspan="1">E. hormaechei</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">Ceftazidime</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">CTX-M-9:4.3% of678</td></tr><tr><td colspan="1" rowspan="1">CTX-M-9</td><td colspan="1" rowspan="1">E. hormaechei</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">Ceftriaxone</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">CTX-M-9:4.3% of678</td></tr><tr><td colspan="1" rowspan="1">CTX-M-9</td><td colspan="1" rowspan="1">E. hormaechei</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">Pip.Tazo</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">CTX-M-9:4.3% of678</td></tr><tr><td colspan="1" rowspan="1">CTX-M-9</td><td colspan="1" rowspan="1">E. ludwigii</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">Cefepime</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">CTX-M-9:0% of 31</td></tr><tr><td colspan="1" rowspan="1">CTX-M-9</td><td colspan="1" rowspan="1">E. Iudwigii</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">Ceftazidime</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">CTX-M-9:0% of 31</td></tr><tr><td colspan="1" rowspan="1">CTX-M-9</td><td colspan="1" rowspan="1">E. ludwigii</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">Ceftriaxone</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">CTX-M-9:0% of 31</td></tr><tr><td colspan="1" rowspan="1">CTX-M-9</td><td colspan="1" rowspan="1">E. ludwigii</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">Pip.Tazo</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">CTX-M-9:0% of 31</td></tr><tr><td colspan="1" rowspan="1">CTX-M-9</td><td colspan="1" rowspan="1">E. kobei</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">Cefepime</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">CTX-M-9:7.1% of 70</td></tr><tr><td colspan="1" rowspan="1">CTX-M-9</td><td colspan="1" rowspan="1">E. kobei</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">Ceftazidime</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">CTX-M-9:7.1% of 70</td></tr><tr><td colspan="1" rowspan="1">CTX-M-9</td><td colspan="1" rowspan="1">E. kobei</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">Ceftriaxone</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">CTX-M-9:7.1% of 70</td></tr><tr><td colspan="1" rowspan="1">CTX-M-9</td><td colspan="1" rowspan="1">E. kobei</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">Pip.Tazo</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">CTX-M-9:7.1% of 70</td></tr><tr><td colspan="1" rowspan="1">CTX-M-9</td><td colspan="1" rowspan="1">C. freundii</td><td colspan="1" rowspan="1">NaN</td><td colspan="1" rowspan="1">100.0%14/14)</td><td colspan="1" rowspan="1">Cefepime</td><td colspan="1" rowspan="1">CTX-M-9:0.0%(0/8)</td><td colspan="1" rowspan="1">CTX-M-9:100.0%(6/6)</td><td colspan="1" rowspan="1">CTX-M-9:NaN</td><td colspan="1" rowspan="1">CTX-M-9:42.9%(6/14)</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">CTX-M-9:2.8% of179</td></tr><tr><td colspan="1" rowspan="2">AMR GeneTarget</td><td colspan="1" rowspan="2">Organism</td><td colspan="2" rowspan="1">Detection of AMR GeneTarget by AMR GenePanel versus WholeGenome SequencingCombined Clinical DevicePerformance -Stock and ProspectiveIsolates (from K191288)</td><td colspan="1" rowspan="2">Antimicrobial</td><td colspan="4" rowspan="1">Detection of AMR Gene Target by AMR GenePanel vs AST Non-SusceptibilityCombined Clinical Device Performance - Stock andProspective Isolates (from K191288) includingonly unique isolates (no replicates)</td><td colspan="5" rowspan="1">Detection of AMR Gene Target versus AST Non-Susceptibility inOpGen's SMART Study</td><td colspan="1" rowspan="1">%Organismswith AMRGeneTarget inNCBIPathogen</td></tr><tr><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1">PPV[95% CI]</td><td colspan="1" rowspan="1">NPV[95% CI]</td><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1">PPV[95% CI]</td><td colspan="1" rowspan="1">NPV[95% CI]</td><td colspan="1" rowspan="1">% Organismswith AMRGene Target</td><td colspan="1" rowspan="1">Database(2019)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">[78.47-00]</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">[0.00-32.44]</td><td colspan="1" rowspan="1">[60.97-100]</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">[21.38-67.41]</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">CTX-M-9</td><td colspan="1" rowspan="1">C. freundii</td><td colspan="1" rowspan="1">NaN</td><td colspan="1" rowspan="1">100.0%(14/14)[78.47-100.00]</td><td colspan="1" rowspan="1">Ceftazidime</td><td colspan="1" rowspan="1">CTX-M-9:0.0%(0/9)[0.0-29.91]</td><td colspan="1" rowspan="1">CTX-M-9:100.0%(5/5)56.5-100.00]</td><td colspan="1" rowspan="1">CTX-M-9:NaN</td><td colspan="1" rowspan="1">CTX-M-9:35.7%(5/14)[16.34-61.24]</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">CTX-M-9:2.8% of179</td></tr><tr><td colspan="1" rowspan="1">CTX-M-9</td><td colspan="1" rowspan="1">C. freundii</td><td colspan="1" rowspan="1">NaN</td><td colspan="1" rowspan="1">100.0%(14/14)[78.47-100.00]</td><td colspan="1" rowspan="1">Ceftriaxone</td><td colspan="1" rowspan="1">CTX-M-9:0.0%0/10)[0.00-27.75]</td><td colspan="1" rowspan="1">CTX-M-9:100.0%(4/4)[51.01-0.00]</td><td colspan="1" rowspan="1">CTX-M-9:NaN</td><td colspan="1" rowspan="1">CTX-M-9:28.6%(4/14)[11.72-54.65]</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">CTX-M-9:2.8% of179</td></tr><tr><td colspan="1" rowspan="1">CTX-M-9</td><td colspan="1" rowspan="1">C. freundii</td><td colspan="1" rowspan="1">NaN</td><td colspan="1" rowspan="1">100.0%(14/14)[78.47-100.00]</td><td colspan="1" rowspan="1">Pip.Tazo</td><td colspan="1" rowspan="1">CTX-M-9:0.0%(0/7)[0.00-35.43]</td><td colspan="1" rowspan="1">CTX-M-9:100.0%(7/7)[64.57-100.00]</td><td colspan="1" rowspan="1">CTX-M-9:NaN</td><td colspan="1" rowspan="1">CTX-M-9:50.0%(7/14)[26.80-73.20]</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">CTX-M-9:2.8% of179</td></tr><tr><td colspan="1" rowspan="1">CTX-M-9</td><td colspan="1" rowspan="1">C. braakii</td><td colspan="1" rowspan="1">NaN</td><td colspan="1" rowspan="1">100.0%(1/1)[20.65-100.00]</td><td colspan="1" rowspan="1">Cefepime</td><td colspan="1" rowspan="1">CTX-M-9:NaN</td><td colspan="1" rowspan="1">CTX-M-9:100.0%(1/1)[20.65-100.00]</td><td colspan="1" rowspan="1">CTX-M-9:NaN</td><td colspan="1" rowspan="1">CTX-M-9:100.0%(1/1)[20.65-00]</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">CTX-M-9:3.3% of 30</td></tr><tr><td colspan="1" rowspan="1">CTX-M-9</td><td colspan="1" rowspan="1">C. braakii</td><td colspan="1" rowspan="1">NaN</td><td colspan="1" rowspan="1">100.0%(1/1)[20.65-100.00]</td><td colspan="1" rowspan="1">Ceftazidime</td><td colspan="1" rowspan="1">CTX-M-9:0.0%(0/1)[0.0 0-79.35]</td><td colspan="1" rowspan="1">CTX-M-9:NaN</td><td colspan="1" rowspan="1">CTX-M-9:NaN</td><td colspan="1" rowspan="1">CTX-M-9:0.0%(0/1)[0.0-79.35]</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">CTX-M-9:3.3% of 30</td></tr><tr><td colspan="1" rowspan="1">CTX-M-9</td><td colspan="1" rowspan="1">C. braakii</td><td colspan="1" rowspan="1">NaN</td><td colspan="1" rowspan="1">100.0%(1/1)[20.65-10.00]</td><td colspan="1" rowspan="1">Ceftriaxone</td><td colspan="1" rowspan="1">CTX-M-9:0.0%(0/1)[0.00-79.35]</td><td colspan="1" rowspan="1">CTX-M-9:NaN</td><td colspan="1" rowspan="1">CTX-M-9:NaN</td><td colspan="1" rowspan="1">CTX-M-9:0.0%(0/1)[0.0o-79.35]</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">CTX-M-9:3.3% of 30</td></tr><tr><td colspan="1" rowspan="1">CTX-M-9</td><td colspan="1" rowspan="1">C. braakii</td><td colspan="1" rowspan="1">NaN</td><td colspan="1" rowspan="1">100.0%(1/1)</td><td colspan="1" rowspan="1">Pip.Tazo</td><td colspan="1" rowspan="1">CTX-M-9:0.0%(0/1)</td><td colspan="1" rowspan="1">CTX-M-9:NaN</td><td colspan="1" rowspan="1">CTX-M-9:NaN</td><td colspan="1" rowspan="1">CTX-M-9:0.0%(0/1)</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">CTX-M-9:3.3% of 30</td></tr><tr><td colspan="1" rowspan="2">AMR GeneTarget</td><td colspan="1" rowspan="2">Organism</td><td colspan="2" rowspan="1">Detection of AMR GeneTarget by AMR GenePanel versus WholeGenome SequencingCombined Clinical DevicePerformance-Stock and ProspectiveIsolates (from K191288)</td><td colspan="1" rowspan="2">Antimicrobial</td><td colspan="4" rowspan="1">Detection of AMR Gene Target by AMR GenePanel vs AST Non-SusceptibilityCombined Clinical Device Performance - Stock andProspective Isolates (from K191288) includingonly unique isolates (no replicates)</td><td colspan="5" rowspan="1">Detection of AMR Gene Target versus AST Non-Susceptibility inOpGen's SMART Study</td><td colspan="1" rowspan="2">%Organismswith AMRGeneTarget inNCBIPathogenDatabase(2019)</td></tr><tr><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1">PPV[95% CI]</td><td colspan="1" rowspan="1">NPV[95% CI]</td><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1">PPV[95% CI]</td><td colspan="1" rowspan="1">NPV[95% CI]</td><td colspan="1" rowspan="1">% Organismswith AMRGene Target</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">[20.65-100.00]</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">[0.00-79.35]</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">[0.00-79.35]</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">CTX-M-9</td><td colspan="1" rowspan="1">C. werkmanii</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">Cefepime</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">CTX-M-9:0% of 10</td></tr><tr><td colspan="1" rowspan="1">CTX-M-9</td><td colspan="1" rowspan="1">C. werkmanii</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">Ceftazidime</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">CTX-M-9:0% of 10</td></tr><tr><td colspan="1" rowspan="1">CTX-M-9</td><td colspan="1" rowspan="1">C. werkmanii</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">Ceftriaxone</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">CTX-M-9:0% of 10</td></tr><tr><td colspan="1" rowspan="1">CTX-M-9</td><td colspan="1" rowspan="1">C. werkmanii</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">Pip.Tazo</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">CTX-M-9:0% of 10</td></tr><tr><td colspan="1" rowspan="1">CTX-M-9</td><td colspan="1" rowspan="1">C. youngae</td><td colspan="1" rowspan="1">NaN</td><td colspan="1" rowspan="1">100.0%(1/1)[20.65-100.00]</td><td colspan="1" rowspan="1">Cefepime</td><td colspan="1" rowspan="1">CTX-M-9:0.0%(0/1)[0.00-79.35]</td><td colspan="1" rowspan="1">CTX-M-9:NaN</td><td colspan="1" rowspan="1">CTX-M-9:NaN</td><td colspan="1" rowspan="1">CTX-M-9:0.0%(/1)[0.00-79.35]</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">CTX-M-9:0% of 7</td></tr><tr><td colspan="1" rowspan="1">CTX-M-9</td><td colspan="1" rowspan="1">C. youngae</td><td colspan="1" rowspan="1">NaN</td><td colspan="1" rowspan="1">100.0%(1/1)[20.65-100.00]</td><td colspan="1" rowspan="1">Ceftazidime</td><td colspan="1" rowspan="1">CTX-M-9:0.0%(/1)[0.00-79.35]</td><td colspan="1" rowspan="1">CTX-M-9:NaN</td><td colspan="1" rowspan="1">CTX-M-9:NaN</td><td colspan="1" rowspan="1">CTX-M-9:0.0%(0/1)[0.00-79.35]</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">CTX-M-9:0% of 7</td></tr><tr><td colspan="1" rowspan="1">CTX-M-9</td><td colspan="1" rowspan="1">C. youngae</td><td colspan="1" rowspan="1">NaN</td><td colspan="1" rowspan="1">100.0%(1/1)[20.65-100.00]</td><td colspan="1" rowspan="1">Ceftriaxone</td><td colspan="1" rowspan="1">CTX-M-9:0.0%(0/1)0.0-79.35]</td><td colspan="1" rowspan="1">CTX-M-9:NaN</td><td colspan="1" rowspan="1">CTX-M-9:NaN</td><td colspan="1" rowspan="1">CTX-M-9:0.0%(0/1)[0.00-79.35]</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">CTX-M-9:0% of 7</td></tr><tr><td colspan="1" rowspan="1">CTX-M-9</td><td colspan="1" rowspan="1">C. youngae</td><td colspan="1" rowspan="1">NaN</td><td colspan="1" rowspan="1">100.0%(1/1)[20.65-100.00]</td><td colspan="1" rowspan="1">Pip.Tazo</td><td colspan="1" rowspan="1">CTX-M-9:0.0%((0/1)[0.00-79.35]</td><td colspan="1" rowspan="1">CTX-M-9:NaN</td><td colspan="1" rowspan="1">CTX-M-9:NaN</td><td colspan="1" rowspan="1">CTX-M-9:0.0%(0/1)[0.00-79.35]</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">CTX-M-9:0% of 7</td></tr><tr><td colspan="1" rowspan="1">CTX-M-9</td><td colspan="1" rowspan="1">S. marcescens</td><td colspan="1" rowspan="1">NaN</td><td colspan="1" rowspan="1">100.0%(2/2)[34.24-100.00]</td><td colspan="1" rowspan="1">Cefepime</td><td colspan="1" rowspan="1">CTX-M-9:0.0%(0/2)0.0-65.76]</td><td colspan="1" rowspan="1">CTX-M-9:NaN</td><td colspan="1" rowspan="1">CTX-M-9:NaN</td><td colspan="1" rowspan="1">CTX-M-9:0.0%(0/2)[0.00-65.76]</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">CTX-M-9:0.4% of496</td></tr><tr><td colspan="1" rowspan="2">AMR GeneTarget</td><td colspan="1" rowspan="2">Organism</td><td colspan="2" rowspan="1">Detection of AMR GeneTarget by AMR GenePanel versus WholeGenome SequencingCombined Clinical DevicePerformance -Stock and ProspectiveIsolates (from K191288)</td><td colspan="1" rowspan="2">Antimicrobial</td><td colspan="4" rowspan="1">Detection of AMR Gene Target by AMR GenePanel vs AST Non-SusceptibilityCombined Clinical Device Performance - Stock andProspective Isolates (from K191288) includingonly unique isolates (no replicates)</td><td colspan="5" rowspan="1">Detection of AMR Gene Target versus AST Non-Susceptibility inOpGen's SMART Study</td><td colspan="1" rowspan="1">%Organismswith AMRGeneTarget inNCBIPathogen</td></tr><tr><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1">PPV[95% CI]</td><td colspan="1" rowspan="1">NPV[95% CI]</td><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1">PPV[95% CI]</td><td colspan="1" rowspan="1">NPV[95% CI]</td><td colspan="1" rowspan="1">% Organismswith AMRGene Target</td><td colspan="1" rowspan="1">Database(2019)</td></tr><tr><td colspan="1" rowspan="1">CTX-M-9</td><td colspan="1" rowspan="1">S. marcescens</td><td colspan="1" rowspan="1">NaN</td><td colspan="1" rowspan="1">100.0%(2/2)[34.24-00]</td><td colspan="1" rowspan="1">Ceftazidime</td><td colspan="1" rowspan="1">CTX-M-9:0.0%(0/2)[0.00-65.76]</td><td colspan="1" rowspan="1">CTX-M-9:NaN</td><td colspan="1" rowspan="1">CTX-M-9:NaN</td><td colspan="1" rowspan="1">CTX-M-9:0.0%(0/2)[0.00--65.76]</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">CTX-M-9:0.4% of496</td></tr><tr><td colspan="1" rowspan="1">CTX-M-9</td><td colspan="1" rowspan="1">S. marcescens</td><td colspan="1" rowspan="1">NaN</td><td colspan="1" rowspan="1">100.0%(2/2)[34.24-100.00]</td><td colspan="1" rowspan="1">Ceftriaxone</td><td colspan="1" rowspan="1">CTX-M-9:0.0%((0/2)[0.00-65.76]</td><td colspan="1" rowspan="1">CTX-M-9:NaN</td><td colspan="1" rowspan="1">CTX-M-9:NaN</td><td colspan="1" rowspan="1">CTX-M-9:0.0%(/2)[0.00-65.76]</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">CTX-M-9:0.4% of496</td></tr><tr><td colspan="1" rowspan="1">CTX-M-9</td><td colspan="1" rowspan="1">S. marcescens</td><td colspan="1" rowspan="1">NaN</td><td colspan="1" rowspan="1">100.0%(2/2)[34.24-100.00]</td><td colspan="1" rowspan="1">Pip.Tazo</td><td colspan="1" rowspan="1">CTX-M-9:0.0%(0/2)[0.00-65.76]</td><td colspan="1" rowspan="1">CTX-M-9:NaN</td><td colspan="1" rowspan="1">CTX-M-9:NaN</td><td colspan="1" rowspan="1">CTX-M-9:0.0%(0/2)[0.00-65.76]</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">CTX-M-9:0.4% of496</td></tr><tr><td colspan="1" rowspan="1">DFR</td><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">98.6%(142/144)[95.08-999.62]</td><td colspan="1" rowspan="1">99.5%(190/191)[97.09-99.91]</td><td colspan="1" rowspan="1">Trimeth.Sulfa</td><td colspan="1" rowspan="1">DFR:58.8%(67/114)[49.59-67.38]</td><td colspan="1" rowspan="1">DFR:95.6%(43/45)[85.17-98.77]</td><td colspan="1" rowspan="1">DFR:97.1%(67/69)[90.03-99.20]</td><td colspan="1" rowspan="1">DFR:47.8%(43/90)[37.76-57.98]</td><td colspan="1" rowspan="1">dfrA5:8.5%(148/1732)[7.32-9.96]dfrA17:67.3%(1166/1732) [65.08-69.49]</td><td colspan="1" rowspan="1">dfrA5:98.3%(1207/1228) [97.40-98.88]dfrA17:95.4%(1171/1228) [94.03-96.40]</td><td colspan="1" rowspan="1">dfrA5:87.6%(148/169)[81.75-91.73]dfrA17:95.3%(1166/1223) [94.01-96.39]</td><td colspan="1" rowspan="1">dfrA5:43.2%(1207/2791) [41.42-45.09]dfrA17:67.4%(1171/1737) [65.17-69.58]</td><td colspan="1" rowspan="1">dfrA5: 5.7%(169/2960)dfrA17:41.3%(1223/2960)</td><td colspan="1" rowspan="1">dfrA5: 2%of 19017dfrA17:9.4% of19017</td></tr><tr><td colspan="1" rowspan="1">DFR</td><td colspan="1" rowspan="1">K. pneumoniae</td><td colspan="1" rowspan="1">100.0%(9/9)[70.09-00]</td><td colspan="1" rowspan="1">99.4%(318/320)[97.75-9.83]</td><td colspan="1" rowspan="1">Trimeth.Sulfa</td><td colspan="1" rowspan="1">DFR:6.0%(9/150)3.1911.01]</td><td colspan="1" rowspan="1">DFR:100.0%(35/35)0.11-100.00]</td><td colspan="1" rowspan="1">DFR:100.0%(9/9)0.09-100.00]</td><td colspan="1" rowspan="1">DFR:19.9%(35/176)[14.66-26.40]</td><td colspan="1" rowspan="1">dfrA5:39.6%(603/1523)[37.17-42.07]dfrA17:2.5%(38/1523)[1.82-3.41]</td><td colspan="1" rowspan="1">dfrA5:93.3%(471/505)[90.74-95.14]dfrA17:99.8%(504/505)[98.89-99.97]</td><td colspan="1" rowspan="1">dfrA5:94.7%(603/637)[92.63-96.16]dfrA17:97.4%(38/39)[86.82-9.5]</td><td colspan="1" rowspan="1">dfrA5:33.9%(471/1391)[31.42-36.39]dfrA17:25.3%(504/1989)[23.48-27.30</td><td colspan="1" rowspan="1">dfrA5: 31.4%(637/2028)dfrA17: 1.9%(39/2028)</td><td colspan="1" rowspan="1">dfrA5:1.3% of7979dfrA17:1.3% of7979</td></tr><tr><td colspan="1" rowspan="2">AMR GeneTarget</td><td colspan="1" rowspan="2">Organism</td><td colspan="2" rowspan="1">Detection of AMR GeneTarget by AMR GenePanel versus WholeGenome SequencingCombined Clinical DevicePerformance-Stock and ProspectiveIsolates (from K191288)</td><td colspan="1" rowspan="2">Antimicrobial</td><td colspan="4" rowspan="1">Detection of AMR Gene Target by AMR GenePanel vs AST Non-SusceptibilityCombined Clinical Device Performance - Stock andProspective Isolates (from K191288) includingonly unique isolates (no replicates)</td><td colspan="5" rowspan="1">Detection of AMR Gene Target versus AST Non-Susceptibility inOpGen's SMART Study</td><td colspan="1" rowspan="2">%Organismswith AMRGeneTarget inNCBIPathogenDatabase(2019)</td></tr><tr><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1">PPV[95% CI]</td><td colspan="1" rowspan="1">NPV[95% CI]</td><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1">PPV[95% CI]</td><td colspan="1" rowspan="1">NPV[95% CI]</td><td colspan="1" rowspan="1">% Organismswith AMRGene Target</td></tr><tr><td colspan="1" rowspan="1">DFR</td><td colspan="1" rowspan="1">P. mirabilis</td><td colspan="1" rowspan="1">100.0%(16/16)[80.64-10.00]</td><td colspan="1" rowspan="1">99.3%(138/139)[96.04-999.87]</td><td colspan="1" rowspan="1">Trimeth.Sulfa</td><td colspan="1" rowspan="1">DFR:19.4%(13/67)[11.71-30.42]</td><td colspan="1" rowspan="1">DFR:100.0%(11/11)[74.12-00]</td><td colspan="1" rowspan="1">DFR:100.0%(13/13)[77.19-0.00]</td><td colspan="1" rowspan="1">DFR:16.9%(11/65)[9.72-27.82]</td><td colspan="1" rowspan="1">dfrA5:2.3%(17/727)[1.47-3.71]dfrA17:7.3%(53/727)[5.62-9.41]</td><td colspan="1" rowspan="1">dfrA5:99.8%(429/430)[98.69-99.96]dfrA17:99.3%(427/430)[97.97-99.76]</td><td colspan="1" rowspan="1">dfrA5:94.4%(17/18)[74.24-99.01]dfrA17:94.6%(53/56)[85.39-98.16]</td><td colspan="1" rowspan="1">dfrA5:37.7%(429/1139)[34.90-440.52]dfrA17:38.8%(427/1101)[35.95-41.70]</td><td colspan="1" rowspan="1">dfrA5: 1.6%(18/1157)dfrA17: 4.8%(56/1157)</td><td colspan="1" rowspan="1">dfrA5: 0%of 144dfrA17:4.9% of144</td></tr><tr><td colspan="1" rowspan="1">DHA</td><td colspan="1" rowspan="1">K. pneumoniae</td><td colspan="1" rowspan="1">100.0%(36/36)[90.36-100.00]</td><td colspan="1" rowspan="1">100.0%(29/293)[98.71-100.00]</td><td colspan="1" rowspan="1">Cefazolin</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">DHA: 5.5%(101/1824)[4.58-6.68]</td><td colspan="1" rowspan="1">DHA:99.5%(203/204)[97.28-99.91]</td><td colspan="1" rowspan="1">DHA:99.0%(101/102)[94.65-99.83]</td><td colspan="1" rowspan="1">DHA:10.5%(203/1926)[9.25-11.99]</td><td colspan="1" rowspan="1">DHA: 5%(102/2028)</td><td colspan="1" rowspan="1">DHA: 2.5%of 7979</td></tr><tr><td colspan="1" rowspan="1">DHA</td><td colspan="1" rowspan="1">K. pneumoniae</td><td colspan="1" rowspan="1">100.0%(36/36)[90.36-100.00]</td><td colspan="1" rowspan="1">100.0%(293/293)[98.71-1000]</td><td colspan="1" rowspan="1">Cefuroxime</td><td colspan="1" rowspan="1">DHA:18.8%(32/170)13.66-25.36]</td><td colspan="1" rowspan="1">DHA:93.3%(14/15)[70.18-98.81]</td><td colspan="1" rowspan="1">DHA:97.0%(32/33)84.68-99.46]</td><td colspan="1" rowspan="1">DHA:9.2%(14/152)[5.57-14.87]</td><td colspan="1" rowspan="1">DHA: 5.5%(101/1832)[4.56-6.65]</td><td colspan="1" rowspan="1">DHA:99.5%(195/196)[97.17-99.91</td><td colspan="1" rowspan="1">DHA:99.0%(101/102)[94.65-999.83]</td><td colspan="1" rowspan="1">DHA:10.1%(195/1926)[8.86-11.55]</td><td colspan="1" rowspan="1">DHA: 5%(102/2028)</td><td colspan="1" rowspan="1">DHA: 2.5%of 7979</td></tr><tr><td colspan="1" rowspan="1">DHA</td><td colspan="1" rowspan="1">K. pneumoniae</td><td colspan="1" rowspan="1">100.0%(36/36)[90.36-100.00]</td><td colspan="1" rowspan="1">100.0%(293/293)[98.71-10.00]</td><td colspan="1" rowspan="1">Cefepime</td><td colspan="1" rowspan="1">DHA:16.2%(26/160)[11.34-22.75]</td><td colspan="1" rowspan="1">DHA:72.0%(18/25)[52.42-85.72]</td><td colspan="1" rowspan="1">DHA:78.8%(/33)[62.25-89.32]</td><td colspan="1" rowspan="1">DHA:11.8%(18/152)[7.62-17.94]</td><td colspan="1" rowspan="1">DHA: 4.4%(76/1711)[3.56-5.52]</td><td colspan="1" rowspan="1">DHA:91.8%(291/317)[88.25-94.34]</td><td colspan="1" rowspan="1">DHA:74.5%(76/102)[65.27-81.97]</td><td colspan="1" rowspan="1">DHA:15.1%(291/1926)[13.58-16.78]</td><td colspan="1" rowspan="1">DHA: 5%(102/2028)</td><td colspan="1" rowspan="1">DHA: 2.5%of 7979</td></tr><tr><td colspan="1" rowspan="1">DHA</td><td colspan="1" rowspan="1">K. pneumoniae</td><td colspan="1" rowspan="1">100.0%(36/36)[90.36-00]</td><td colspan="1" rowspan="1">100.0%(293/293)[98.71-100.00]</td><td colspan="1" rowspan="1">Ceftazidime</td><td colspan="1" rowspan="1">DHA:19.6%(32/163)[14.26-26.40]</td><td colspan="1" rowspan="1">DHA:95.5%(21//22)[78.20-99.19]</td><td colspan="1" rowspan="1">DHA:97.0%(32/33)[84.68-99.46]</td><td colspan="1" rowspan="1">DHA:13.8%(21/152)[9.22-20.20]</td><td colspan="1" rowspan="1">DHA: 5.8%(100/1710)[4.83-7.06]</td><td colspan="1" rowspan="1">DHA:99.4%(316/318)[97.74-99.83]</td><td colspan="1" rowspan="1">DHA:98.0%(100/102)[93.13-99.46]</td><td colspan="1" rowspan="1">DHA:16.4%(316/1926)[14.82-18.13]</td><td colspan="1" rowspan="1">DHA: 5%(102/2028)</td><td colspan="1" rowspan="1">DHA: 2.5%of 7979</td></tr><tr><td colspan="1" rowspan="1">DHA</td><td colspan="1" rowspan="1">K. pneumoniae</td><td colspan="1" rowspan="1">100.0%(36/36)90.36-0.00]</td><td colspan="1" rowspan="1">100.0%(293/293)[98.71-00]</td><td colspan="1" rowspan="1">Ceftriaxone</td><td colspan="1" rowspan="1">DHA:19.2%(32/167)[13.91-25.80]</td><td colspan="1" rowspan="1">DHA:94.4%(17/18)[74.24-99.01]</td><td colspan="1" rowspan="1">DHA:97.0%(32/33)[84.68-99.46]</td><td colspan="1" rowspan="1">DHA:11.2%(17/152)[7.10-17.18]</td><td colspan="1" rowspan="1">DHA: 5.6%(98/1765)[4.58-6.72]</td><td colspan="1" rowspan="1">DHA:98.5%(259/263)[96.16-9.41]</td><td colspan="1" rowspan="1">DHA:96.1%(98/102)[90.3598.46]</td><td colspan="1" rowspan="1">DHA:13.4%(259/1926)12.00-15.04]</td><td colspan="1" rowspan="1">DHA: 5%(102/2028)</td><td colspan="1" rowspan="1">DHA: 2.5%of 7979</td></tr><tr><td colspan="1" rowspan="2">AMR GeneTarget</td><td colspan="1" rowspan="2">Organism</td><td colspan="2" rowspan="1">Detection of AMR GeneTarget by AMR GenePanel versus WholeGenome SequencingCombined Clinical DevicePerformance -Stock and ProspectiveIsolates (from K191288)</td><td colspan="1" rowspan="1">Antimicrobial</td><td colspan="4" rowspan="1">Detection of AMR Gene Target by AMR GenePanel vs AST Non-SusceptibilityCombined Clinical Device Performance - Stock andProspective Isolates (from K191288) includingonly unique isolates (no replicates)</td><td colspan="5" rowspan="1">Detection of AMR Gene Target versus AST Non-Susceptibility inOpGen's SMART Study</td><td colspan="1" rowspan="1">%Organismswith AMRGeneTarget inNCBIPathogen</td></tr><tr><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1">PPV[95% CI]</td><td colspan="1" rowspan="1">NPV[95% CI]</td><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1">PPV[95% CI]</td><td colspan="1" rowspan="1">NPV[95% CI]</td><td colspan="1" rowspan="1">% Organismswith AMRGene Target</td><td colspan="1" rowspan="1">Database(2019)</td></tr><tr><td colspan="1" rowspan="1">DHA</td><td colspan="1" rowspan="1">K. pneumoniae</td><td colspan="1" rowspan="1">100.0%(36/36)[90.36-100.00]</td><td colspan="1" rowspan="1">100.0%(293/293)[98.71-0.00]</td><td colspan="1" rowspan="1">Imipenem</td><td colspan="1" rowspan="1">DHA:9.4%(10/106)[5.21-16.50]</td><td colspan="1" rowspan="1">DHA:70.9%(56/79)60.09-79.75]</td><td colspan="1" rowspan="1">DHA:30.3%(10/33)[17.38-47.34]</td><td colspan="1" rowspan="1">DHA:36.8%(556/152)[29.59-44.75]</td><td colspan="1" rowspan="1">DHA: 4.0%(33/818)[2.89-5.61]</td><td colspan="1" rowspan="1">DHA:94.3%(1141/1210) [92.85-95.47]</td><td colspan="1" rowspan="1">DHA:32.4%(33/102)[24.06-41.93]</td><td colspan="1" rowspan="1">DHA:59.2%(1141/1926) [57.03-61.42]</td><td colspan="1" rowspan="1">DHA: 5%(102/2028)</td><td colspan="1" rowspan="1">DHA: 2.5%f 7979</td></tr><tr><td colspan="1" rowspan="1">DHA</td><td colspan="1" rowspan="1">K. pneumoniae</td><td colspan="1" rowspan="1">100.0%(36/36)[90.36-10.00]</td><td colspan="1" rowspan="1">100.0%(293/293)[98.71-100.00]</td><td colspan="1" rowspan="1">Ertapenem</td><td colspan="1" rowspan="1">DHA:14.3%(18/126)[9.23-21.45]</td><td colspan="1" rowspan="1">DHA:74.6%(44/59)[62.20-83.94]</td><td colspan="1" rowspan="1">DHA:54.5%(18/33)[37.99-70.16]</td><td colspan="1" rowspan="1">DHA:28.9%(44/152)[22.33-36.61]</td><td colspan="1" rowspan="1">DHA: 4.8%(51/1053)[3.70-6.31]</td><td colspan="1" rowspan="1">DHA:94.8%(924/975)[93.19-96.00]</td><td colspan="1" rowspan="1">DHA:50.0%(51/102)[40.47-59.53]</td><td colspan="1" rowspan="1">DHA:48.0%(924/1926)[45.75-50.21]</td><td colspan="1" rowspan="1">DHA: 5%(102/2028)</td><td colspan="1" rowspan="1">DHA: 2.5%of 7979</td></tr><tr><td colspan="1" rowspan="1">DHA</td><td colspan="1" rowspan="1">K. pneumoniae</td><td colspan="1" rowspan="1">100.0%(36/36)[90.36-10.00]</td><td colspan="1" rowspan="1">100.0%(293/293)[98.71-1000]</td><td colspan="1" rowspan="1">Meropenem</td><td colspan="1" rowspan="1">DHA:4.0%(4/99)[1.58-9.931</td><td colspan="1" rowspan="1">DHA:66.3%(57/86)[55.78-75.38]</td><td colspan="1" rowspan="1">DHA:12.1%(4/33)[4.82-27.33]</td><td colspan="1" rowspan="1">DHA:37.5%(57/152)[30.20-45.42]</td><td colspan="1" rowspan="1">DHA: 1.9%(16/842)[1.17-3.06]</td><td colspan="1" rowspan="1">DHA:92.7%(1100/118) (91.13-94.09]</td><td colspan="1" rowspan="1">DHA:15.7%(16/102)[9.89-23.97]</td><td colspan="1" rowspan="1">DHA:57.1%(1100/1926) [54.89-59.31]</td><td colspan="1" rowspan="1">DHA: 5%(102/2028)</td><td colspan="1" rowspan="1">DHA: 2.5%of 7979</td></tr><tr><td colspan="1" rowspan="1">DHA</td><td colspan="1" rowspan="1">K. pneumoniae</td><td colspan="1" rowspan="1">100.0%(36/36)90.36-100.00]</td><td colspan="1" rowspan="1">100.0%(3/293)[98.71-100.00]</td><td colspan="1" rowspan="1">Amox.K.Clav</td><td colspan="1" rowspan="1">DHA:19.6%(32/163)[14.26-26.40]</td><td colspan="1" rowspan="1">DHA:95.5%(21/22)[78.20-99.19]</td><td colspan="1" rowspan="1">DHA:97.0%(32/33)[84.68-99.46]</td><td colspan="1" rowspan="1">DHA:13.8%21/152)[9.22-20.20]</td><td colspan="1" rowspan="1">DHA: 5.9%(101/1716)[4.87-7.10]</td><td colspan="1" rowspan="1">DHA:99.7%(311/312)[98.21-99.94]</td><td colspan="1" rowspan="1">DHA:99.0%(101/102)[94.65-99.83]</td><td colspan="1" rowspan="1">DHA:16.1%(311/1926)[14.57-17.86]</td><td colspan="1" rowspan="1">DHA: 5%(102/2028)</td><td colspan="1" rowspan="1">DHA: 2.5%of 7979</td></tr><tr><td colspan="1" rowspan="1">DHA</td><td colspan="1" rowspan="1">K. pneumoniae</td><td colspan="1" rowspan="1">100.0%(36/36)90.36-10.00]</td><td colspan="1" rowspan="1">100.0%(293/293)[98.71-0.00]</td><td colspan="1" rowspan="1">Amp.Sulbactam</td><td colspan="1" rowspan="1">DHA:18.1%(32/177)13.11-24.41]</td><td colspan="1" rowspan="1">DHA:87.5%(7/8)[52.91-97.76]</td><td colspan="1" rowspan="1">DHA:97.0%(32/33)[84.68-99.46]</td><td colspan="1" rowspan="1">DHA:4.6%(7/152)[2.25-9.20]</td><td colspan="1" rowspan="1">DHA: 5.3%(101/1909)[4.37-6.39]</td><td colspan="1" rowspan="1">DHA:99.2%(118/119)[95.39-99.85]</td><td colspan="1" rowspan="1">DHA:99.0%(101/102)[94.65-99.83]</td><td colspan="1" rowspan="1">DHA: 6.1%(118/1926)[5.14-7.29]</td><td colspan="1" rowspan="1">DHA: 5%(102/2028)</td><td colspan="1" rowspan="1">DHA: 2.5%of 7979</td></tr><tr><td colspan="1" rowspan="1">DHA</td><td colspan="1" rowspan="1">K. pneumoniae</td><td colspan="1" rowspan="1">100.0%(36/36)90.36-100.00]</td><td colspan="1" rowspan="1">100.0%(293/293)98.71-100.00]</td><td colspan="1" rowspan="1">Pip.Tazo</td><td colspan="1" rowspan="1">DHA:18.4%(27/147)[12.94-25.40]</td><td colspan="1" rowspan="1">DHA:84.2%(32/38)[69.58-92.56]</td><td colspan="1" rowspan="1">DHA:81.8%(27/33)[65.61-911.39]</td><td colspan="1" rowspan="1">DHA:21.1%(32/152)[15.33-28.21]</td><td colspan="1" rowspan="1">DHA: 5.6%(78/1400)[4.49-6.90]</td><td colspan="1" rowspan="1">DHA:96.2%(604/28)[94.38-97.42]</td><td colspan="1" rowspan="1">DHA:76.5%(78/102)[67.37-83.65]</td><td colspan="1" rowspan="1">DHA:31.4%(604/1926)[29.33-33.47]</td><td colspan="1" rowspan="1">DHA: 5%(102/2028)</td><td colspan="1" rowspan="1">DHA: 2.5%of 7979</td></tr><tr><td colspan="1" rowspan="1">IMP</td><td colspan="1" rowspan="1">K. pneumoniae</td><td colspan="1" rowspan="1">100.0%(72/72)[94.93-10.00]</td><td colspan="1" rowspan="1">100.0%(257/257)[98.53-00]</td><td colspan="1" rowspan="1">Imipenem</td><td colspan="1" rowspan="1">IMP:4.7%(5/106)[2.03-10.57]</td><td colspan="1" rowspan="1">IMP:100.0%(79/79)95.3610.00]</td><td colspan="1" rowspan="1">IMP:100.0%(5/5)[56.55-100.00]</td><td colspan="1" rowspan="1">IMP:43.9%(79/180)[36.84-551.19]</td><td colspan="1" rowspan="1">IMP: 1.0%(8/818)[0.50-1.92]</td><td colspan="1" rowspan="1">IMP:99.3%(1202/1210) [98.70-99.6]</td><td colspan="1" rowspan="1">IMP:50.0%(8/16)28.072.00]</td><td colspan="1" rowspan="1">IMP:59.7%(1202/2012) [57.58-1.86]</td><td colspan="1" rowspan="1">IMP: 0.8%(16/2028)</td><td colspan="1" rowspan="1">IMP: 0.8%of 7979</td></tr><tr><td colspan="1" rowspan="2">AMR GeneTarget</td><td colspan="1" rowspan="2">Organism</td><td colspan="2" rowspan="1">Detection of AMR GeneTarget by AMR GenePanel versus WholeGenome SequencingCombined Clinical DevicePerformance-Stock and ProspectiveIsolates (from K191288)</td><td colspan="1" rowspan="2">Antimicrobial</td><td colspan="4" rowspan="1">Detection of AMR Gene Target by AMR GenePanel vs AST Non-SusceptibilityCombined Clinical Device Performance - Stock andProspective Isolates (from K191288) includingonly unique isolates (no replicates)</td><td colspan="5" rowspan="1">Detection of AMR Gene Target versus AST Non-Susceptibility inOpGen's SMART Study</td><td colspan="1" rowspan="2">%Organismswith AMRGeneTarget inNCBIPathogenDatabase(2019)</td></tr><tr><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1">PPV[95% CI]</td><td colspan="1" rowspan="1">NPV[95% CI]</td><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1">PPV[95% CI]</td><td colspan="1" rowspan="1">NPV[95% CI]</td><td colspan="1" rowspan="1">% Organismswith AMRGene Target</td></tr><tr><td colspan="1" rowspan="1">IMP</td><td colspan="1" rowspan="1">K. pneumoniae</td><td colspan="1" rowspan="1">100.0%(72/72)[94.93-00]</td><td colspan="1" rowspan="1">100.0%(257/257)[98.53-00]</td><td colspan="1" rowspan="1">Ertapenem</td><td colspan="1" rowspan="1">IMP:4.0%(5/126)[1.71-8.95]</td><td colspan="1" rowspan="1">IMP:100.0%(59/59)93.890.00]</td><td colspan="1" rowspan="1">IMP:100.0%(5/5)[56.55-00]</td><td colspan="1" rowspan="1">IMP:32.8%(59/180)[26.34-39.93]</td><td colspan="1" rowspan="1">IMP: 1.1%(12/1053)[0.65-1.98]</td><td colspan="1" rowspan="1">IMP:99.6%971/975)[98.95-99.84]</td><td colspan="1" rowspan="1">IMP:75.0%(12/16)[50.50-89.82]</td><td colspan="1" rowspan="1">IMP:48.3%(971/2012)[46.08-50.45]</td><td colspan="1" rowspan="1">IMP: 0.8%(16/2028)</td><td colspan="1" rowspan="1">IMP: 0.8%of 7979</td></tr><tr><td colspan="1" rowspan="1">IMP</td><td colspan="1" rowspan="1">K. pneumoniae</td><td colspan="1" rowspan="1">100.0%(72/72)[94.93-0.00]</td><td colspan="1" rowspan="1">100.0%(257/257)[98.53-100.00]</td><td colspan="1" rowspan="1">Meropenem</td><td colspan="1" rowspan="1">IMP:5.1%(/9)[2.18-11.28]</td><td colspan="1" rowspan="1">IMP:100.0%86/86)[95.72-100.00]</td><td colspan="1" rowspan="1">IMP:100.0%(5/5)56.5-10.00]</td><td colspan="1" rowspan="1">IMP:47.8%(86/180)[40.60-55.04]</td><td colspan="1" rowspan="1">IMP: 1.1%(9/842)[0.56-2.02]</td><td colspan="1" rowspan="1">IMP:99.4%(1179/1186) [98.79-9.71]</td><td colspan="1" rowspan="1">IMP:56.2%(/16)[33.18-76.90]</td><td colspan="1" rowspan="1">IMP:58.6%(1179/2012) [56.43-60.73]</td><td colspan="1" rowspan="1">IMP: 0.8%(16/2028)</td><td colspan="1" rowspan="1">IMP: 0.8%of 7979</td></tr><tr><td colspan="1" rowspan="1">KPC</td><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">94.1%(16/17)[73.02-98.95]</td><td colspan="1" rowspan="1">99.7%(317/318)[98.24-9.94]</td><td colspan="1" rowspan="1">Imipenem</td><td colspan="1" rowspan="1">KPC:30.2%(13/43)18.60-45.11]</td><td colspan="1" rowspan="1">KPC:100.0%(116/116)[96.79-00]</td><td colspan="1" rowspan="1">KPC:100.0%(13/13)[77.19-100.00]</td><td colspan="1" rowspan="1">KPC:79.5%(116/146)[72.18-85.21]</td><td colspan="1" rowspan="1">KPC: 25.9%(60/232)[20.65-31.86]</td><td colspan="1" rowspan="1">KPC: 99.7%(2719/2728) [99.37-9.83]</td><td colspan="1" rowspan="1">KPC: 87.0%(60/69)[77.03-92.98]</td><td colspan="1" rowspan="1">KPC: 94.1%(2719/2891) [93.13-94.86]</td><td colspan="1" rowspan="1">KPC: 2.3%(69/2960)</td><td colspan="1" rowspan="1">KPC: 0.6%of 19017</td></tr><tr><td colspan="1" rowspan="1">KPC</td><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">94.1%(16/17)[73.02-98.95]</td><td colspan="1" rowspan="1">99.7%(317/318)[98.24-99.94]</td><td colspan="1" rowspan="1">Ertapenem</td><td colspan="1" rowspan="1">KPC:23.6%(13/55)[14.37-36.35]</td><td colspan="1" rowspan="1">KPC:100.0%(104/104)[96.44-0.00]</td><td colspan="1" rowspan="1">KPC:100.0%(13/13)[77.19-100.00]</td><td colspan="1" rowspan="1">KPC:71.2%(104/146)[63.42-77.96]</td><td colspan="1" rowspan="1">KPC: 16.1%(62/386)[12.74-20.06]</td><td colspan="1" rowspan="1">KPC: 99.7%(2567/2574) [99.44-99.87]</td><td colspan="1" rowspan="1">KPC: 89.9%(62/69)[80.51-95.00]</td><td colspan="1" rowspan="1">KPC: 88.8%(2567/2891) [87.59-89.89]</td><td colspan="1" rowspan="1">KPC: 2.3%(69/2960)</td><td colspan="1" rowspan="1">KPC: 0.6%of 19017</td></tr><tr><td colspan="1" rowspan="1">KPC</td><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">94.1%(16/17)[73.02-8.95]</td><td colspan="1" rowspan="1">99.7%(317/318)[98.24-99.94]</td><td colspan="1" rowspan="1">Meropenem</td><td colspan="1" rowspan="1">KPC:35.1%(13/37)[21.83-51.24]</td><td colspan="1" rowspan="1">KPC:1000%(122/122)[96.95-10.00]</td><td colspan="1" rowspan="1">KPC:100.0%(13/13)[77.19-10.00]</td><td colspan="1" rowspan="1">KPC:83.6%(122/146)[76.70-88.70]</td><td colspan="1" rowspan="1">KPC: 26.3%54/205)[20.79-32.77]</td><td colspan="1" rowspan="1">KPC: 99.5%(2740/2755) [99.10-9.67]</td><td colspan="1" rowspan="1">KPC: 78.3%54/69)[67.18-86.36]</td><td colspan="1" rowspan="1">KPC: 94.8%(2740/2891) [93.90-5.53]</td><td colspan="1" rowspan="1">KPC: 2.3%(69/2960)</td><td colspan="1" rowspan="1">KPC: 0.6%of 19017</td></tr><tr><td colspan="1" rowspan="1">KPC</td><td colspan="1" rowspan="1">K. pneumoniae</td><td colspan="1" rowspan="1">97.2%(35/36)85.83-99.51]</td><td colspan="1" rowspan="1">99.7%(292/293)[98.09-99.94]</td><td colspan="1" rowspan="1">Imipenem</td><td colspan="1" rowspan="1">KPC:27.4%(29/106)[19.78-36.52]</td><td colspan="1" rowspan="1">KPC:100.0%%(79/79)[95.36-100.00]</td><td colspan="1" rowspan="1">KPC:100.0%(29/29)[88.30-100.00]</td><td colspan="1" rowspan="1">KPC:50.6%(79/156)[42.87-58.38]</td><td colspan="1" rowspan="1">KPC: 39.2%(321/818)[35.95-42.63]</td><td colspan="1" rowspan="1">KPC: 96.2%(1164/1210) [94.97-97.14]</td><td colspan="1" rowspan="1">KPC: 87.5%(321/367)[83.69-90.47]</td><td colspan="1" rowspan="1">KPC: 70.1%(1164/1661) (67.83-72.23]</td><td colspan="1" rowspan="1">KPC: 18.1%(367/2028)</td><td colspan="1" rowspan="1">KPC:32.7% of7979</td></tr><tr><td colspan="1" rowspan="1">KPC</td><td colspan="1" rowspan="1">K. pneumoniae</td><td colspan="1" rowspan="1">97.2%(35/36)[85.83-99.51]</td><td colspan="1" rowspan="1">99.7%(292/293)[98.09-999.94]</td><td colspan="1" rowspan="1">Ertapenem</td><td colspan="1" rowspan="1">KPC:23.0%(29/126)[16.53-31.10]</td><td colspan="1" rowspan="1">KPC:100.0%(59/59)93.89-100.00]</td><td colspan="1" rowspan="1">KPC:100.0%(29/29)[88.30-100.00]</td><td colspan="1" rowspan="1">KPC:37.8%(59/156)[30.59-45.64]</td><td colspan="1" rowspan="1">KPC: 31.3%(330/1053)[28.61-34.20]</td><td colspan="1" rowspan="1">KPC: 96.2%(938/975)[94.81-97.23]</td><td colspan="1" rowspan="1">KPC: 89.9%(330/367)[86.41-92.60]</td><td colspan="1" rowspan="1">KPC: 56.5%(938/1661)[54.08-58.84]</td><td colspan="1" rowspan="1">KPC: 18.1%(367/2028)</td><td colspan="1" rowspan="1">KPC:32.7% of7979</td></tr><tr><td colspan="1" rowspan="2">AMR GeneTarget</td><td colspan="1" rowspan="2">Organism</td><td colspan="2" rowspan="1">Detection of AMR GeneTarget by AMR GenePanel versus WholeGenome SequencingCombined Clinical DevicePerformance -Stock and ProspectiveIsolates (from K191288)</td><td colspan="1" rowspan="1">Antimicrobial</td><td colspan="4" rowspan="1">Detection of AMR Gene Target by AMR GenePanel vs AST Non-SusceptibilityCombined Clinical Device Performance - Stock andivaonly unique isolates (no replicates)</td><td colspan="5" rowspan="1">Detection of AMR Gene Target versus AST Non-Susceptibility inOpGen's SMART Study</td><td colspan="1" rowspan="1">%Organismswith AMRGeneTarget inNCBIPathogen</td></tr><tr><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1">PPV[95% CI]</td><td colspan="1" rowspan="1">NPV[95% CI]</td><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1">PPV[95% CI]</td><td colspan="1" rowspan="1">NPV[95% CI]</td><td colspan="1" rowspan="1">% Organismswith AMRGene Target</td><td colspan="1" rowspan="1">Database(2019)</td></tr><tr><td colspan="1" rowspan="1">KPC</td><td colspan="1" rowspan="1">K. pneumoniae</td><td colspan="1" rowspan="1">97.2%(35/36)[85.83-9.51]</td><td colspan="1" rowspan="1">99.7%(292/293)[98.09-99.94]</td><td colspan="1" rowspan="1">Meropenem</td><td colspan="1" rowspan="1">KPC:29.3%(29/99)[21.24-38.90]</td><td colspan="1" rowspan="1">KPC:100.0%(86/86)95.72-100]</td><td colspan="1" rowspan="1">KPC:100.0%(29/29)[88.30-0.00]</td><td colspan="1" rowspan="1">KPC:55.1%(86/156)[47.29-62.72]</td><td colspan="1" rowspan="1">KPC: 38.2%(322/842)[35.02-41.57]</td><td colspan="1" rowspan="1">KPC: 96.2%(1141/1186) [94.96-7.15]</td><td colspan="1" rowspan="1">KPC: 87.7%(322/367)[83.99-90.71]</td><td colspan="1" rowspan="1">KPC: 68.7%(1141/1661) [66.42-0.88]</td><td colspan="1" rowspan="1">KPC: 18.1%(367/2028)</td><td colspan="1" rowspan="1">KPC:32.7% of7979</td></tr><tr><td colspan="1" rowspan="1">KPC</td><td colspan="1" rowspan="1">P. mirabilis</td><td colspan="1" rowspan="1">100.0%(1/1)20.65-100.00]</td><td colspan="1" rowspan="1">100.0%(154/154)[97.57-100.00]</td><td colspan="1" rowspan="1">Ertapenem</td><td colspan="1" rowspan="1">KPC:25.0%(1/4)[4.56-69.94]</td><td colspan="1" rowspan="1">KPC:100.0%(74/74)[95.07-0.00]</td><td colspan="1" rowspan="1">KPC:100.0%(1/1)[20.65-10.00]</td><td colspan="1" rowspan="1">KPC:96.1%(74/77)[89.16-98.67]</td><td colspan="1" rowspan="1">KPC: 25.0%(4/16)10.18-49.50]</td><td colspan="1" rowspan="1">KPC: 99.9%(1140/1141) [99.51-99.98]</td><td colspan="1" rowspan="1">KPC: 80.0%(4/5)[37.55-96.38]</td><td colspan="1" rowspan="1">KPC: 99.0%(1140/115) [98.19-99.40]</td><td colspan="1" rowspan="1">KPC: 0.4%(5/1157)</td><td colspan="1" rowspan="1">KPC: 4.9%of 144</td></tr><tr><td colspan="1" rowspan="1">KPC</td><td colspan="1" rowspan="1">P. mirabilis</td><td colspan="1" rowspan="1">100.0%(1/1)[20.65-100.00]</td><td colspan="1" rowspan="1">100.0%(154/154)[97.57-10.00]</td><td colspan="1" rowspan="1">Meropenem</td><td colspan="1" rowspan="1">KPC:33.3%(1/3)[6.15-79.23]</td><td colspan="1" rowspan="1">KPC:100.0%(75/)95.13-0.00]</td><td colspan="1" rowspan="1">KPC:100.0%(1/1)[20.65-0.00]</td><td colspan="1" rowspan="1">KPC:97.4%(75/77)[91.02-99.28]</td><td colspan="1" rowspan="1">KPC: 23.5%(4/7)[9.56-47.26]</td><td colspan="1" rowspan="1">KPC: 99.9%(1139/1140) [99.50-9.98]</td><td colspan="1" rowspan="1">KPC: 80.0%(4/5)[37.55-96.38]</td><td colspan="1" rowspan="1">KPC: 98.9%(1139/1152) [98.08-99.34]</td><td colspan="1" rowspan="1">KPC: 0.4%(5/1157)</td><td colspan="1" rowspan="1">KPC: 4.9%of 144</td></tr><tr><td colspan="1" rowspan="1">KPC</td><td colspan="1" rowspan="1">P. aeruginosa</td><td colspan="1" rowspan="1">100.0%(9/9)70.09-100.00]</td><td colspan="1" rowspan="1">99.4%(337/339)[97.87-99.84]</td><td colspan="1" rowspan="1">Imipenem</td><td colspan="1" rowspan="1">KPC: 7.6%(9/119)[4.03-13.75]</td><td colspan="1" rowspan="1">KPC:100.0%(36/36)[90.36-100.00]</td><td colspan="1" rowspan="1">KPC:100.0%(9/9)[70.09-100.00]</td><td colspan="1" rowspan="1">KPC:24.7%(36/146)[18.38-32.24]</td><td colspan="1" rowspan="1">KPC: 1.7%(17/1002)[1.06-2.70]</td><td colspan="1" rowspan="1">KPC:100.0%(548/548)[99.30-10.00]</td><td colspan="1" rowspan="1">KPC:100.0%(17/17)[81.57-100.00]</td><td colspan="1" rowspan="1">KPC: 35.7%(548/1533)[33.39-38.18]</td><td colspan="1" rowspan="1">KPC: 1.1%(17/1550)</td><td colspan="1" rowspan="1">KPC: 0.2%of 4706</td></tr><tr><td colspan="1" rowspan="1">KPC</td><td colspan="1" rowspan="1">P. aeruginosa</td><td colspan="1" rowspan="1">100.0%(9/9)[70.09-100.00]</td><td colspan="1" rowspan="1">99.4%(337/339)[97.87-99.84]</td><td colspan="1" rowspan="1">Meropenem</td><td colspan="1" rowspan="1">KPC: 7.6%(9/118)[4.06-13.]</td><td colspan="1" rowspan="1">KPC:100.0%(37/37)[90.59-100.00]</td><td colspan="1" rowspan="1">KPC:100.0%(9/9)[70.09-100.00]</td><td colspan="1" rowspan="1">KPC:25.3%(37/146)[18.98-32.97]</td><td colspan="1" rowspan="1">KPC: 1.7%(17/986)[1.08-2.74]</td><td colspan="1" rowspan="1">KPC:100.0%(564/564)[99.32-100.00]</td><td colspan="1" rowspan="1">KPC:100.0%(17/17)[81.57-100.00]</td><td colspan="1" rowspan="1">KPC: 36.8%(564/1533)[34.41-39.23]</td><td colspan="1" rowspan="1">KPC: 1.1%(17/1550)</td><td colspan="1" rowspan="1">KPC: 0.2%of 4706</td></tr><tr><td colspan="1" rowspan="1">KPC</td><td colspan="1" rowspan="1">K. aerogenes</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">Imipenem</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">KPC: 8.5%of 223</td></tr><tr><td colspan="1" rowspan="1">KPC</td><td colspan="1" rowspan="1">K. aerogenes</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">Ertapenem</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">KPC: 8.5%of 223</td></tr><tr><td colspan="1" rowspan="1">KPC</td><td colspan="1" rowspan="1">K. aerogenes</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">Meropenem</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">KPC: 8.5%of 223</td></tr><tr><td colspan="1" rowspan="1">KPC</td><td colspan="1" rowspan="1">K.michiganensis</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">Imipenem</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">KPC:16.9% of89</td></tr><tr><td colspan="1" rowspan="1"></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="2">AMR GeneTarget</td><td colspan="1" rowspan="2">Organism</td><td colspan="2" rowspan="1">Detection of AMR GeneTarget by AMR GenePanel versus WholeGenome SequencingCombined Clinical DevicePerformance -Stock and ProspectiveIsolates (from K191288)</td><td colspan="1" rowspan="2">Antimicrobial</td><td colspan="4" rowspan="1">Detection of AMR Gene Target by AMR GenePanel vs AST Non-SusceptibilityCombined Clinical Device Performance - Stock andivoa 12only unique isolates (no replicates)</td><td colspan="5" rowspan="1">Detection of AMR Gene Target versus AST Non-Susceptibility inOpGen's SMART Study</td><td colspan="1" rowspan="2">%Organismswith AMRGeneTarget inNCBIPathogenDatabase(2019)</td></tr><tr><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1">PPV[95% CI]</td><td colspan="1" rowspan="1">NPV[95% CI]</td><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1">PPV[95% CI]</td><td colspan="1" rowspan="1">NPV[95% CI]</td><td colspan="1" rowspan="1">% Organismswith AMRGene Target</td></tr><tr><td colspan="1" rowspan="1">KPC</td><td colspan="1" rowspan="1">K.michiganensis</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">Ertapenem</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">KPC:16.9% of89</td></tr><tr><td colspan="1" rowspan="1">KPC</td><td colspan="1" rowspan="1">K.michiganensis</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">Meropenem</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">KPC:16.9% of89</td></tr><tr><td colspan="1" rowspan="1">KPC</td><td colspan="1" rowspan="1">K. oxytoca</td><td colspan="1" rowspan="1">100.0%(2/2)[34.24-100.00]</td><td colspan="1" rowspan="1">100.0%(3/3)[43.85-100.00]</td><td colspan="1" rowspan="1">Imipenem</td><td colspan="1" rowspan="1">KPC:100.0%(2/2)[34.24-100.00]</td><td colspan="1" rowspan="1">KPC:100.0%(3/3)[43.85-100.00]</td><td colspan="1" rowspan="1">KPC:100.0%(2/2)34.24-100.00]</td><td colspan="1" rowspan="1">KPC:100.0%(3/3)[43.85-0.00]</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">KPC: 5.5%of 273</td></tr><tr><td colspan="1" rowspan="1">KPC</td><td colspan="1" rowspan="1">K. oxytoca</td><td colspan="1" rowspan="1">100.0%(2/2)[34.24-100.00]</td><td colspan="1" rowspan="1">100.0%(3/3)43.85-100.00]</td><td colspan="1" rowspan="1">Ertapenem</td><td colspan="1" rowspan="1">KPC:100.0%(2/2)[34.24-10.00]</td><td colspan="1" rowspan="1">KPC:100.0%(3/3)[43.85-10.00]</td><td colspan="1" rowspan="1">KPC:100.0%(2/2)[34.24-10.00]</td><td colspan="1" rowspan="1">KPC:100.0%(3/3)[43.85-100]</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">KPC: 5.5%of 273</td></tr><tr><td colspan="1" rowspan="1">KPC</td><td colspan="1" rowspan="1">K. oxytoca</td><td colspan="1" rowspan="1">100.0%(2/2)[34.24-100.00]</td><td colspan="1" rowspan="1">100.0%(3/3)[43.85-100.00]</td><td colspan="1" rowspan="1">Meropenem</td><td colspan="1" rowspan="1">KPC:100.0%(2/2).[34.24-100.00]</td><td colspan="1" rowspan="1">KPC:100.0%(3/3)[43.85-100.00]</td><td colspan="1" rowspan="1">KPC:100.0%(2/2)[34.24-100.00]</td><td colspan="1" rowspan="1">KPC:100.0%(3/3)[43.85-100.00]</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">KPC: 5.5%of 273</td></tr><tr><td colspan="1" rowspan="1">KPC</td><td colspan="1" rowspan="1">K. variicola</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">Imipenem</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">KPC: 4.8%of 272</td></tr><tr><td colspan="1" rowspan="1">KPC</td><td colspan="1" rowspan="1">K. variicola</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">Ertapenem</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">KPC: 4.8%of 272</td></tr><tr><td colspan="1" rowspan="1">KPC</td><td colspan="1" rowspan="1">K. variicola</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">Meropenem</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">KPC: 4.8%of 272</td></tr><tr><td colspan="1" rowspan="1">KPC</td><td colspan="1" rowspan="1">K.quasipneumoniae</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">Imipenem</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">KPC: 5.4%of 259</td></tr><tr><td colspan="1" rowspan="1">KPC</td><td colspan="1" rowspan="1">K.quasipneumoniae</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">Ertapenem</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">KPC: 5.4%of 259</td></tr><tr><td colspan="1" rowspan="1">KPC</td><td colspan="1" rowspan="1">K.quasipneumoniae</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">Meropenem</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">KPC: 5.4%of 259</td></tr><tr><td colspan="1" rowspan="2">AMR GeneTarget</td><td colspan="1" rowspan="2">Organism</td><td colspan="2" rowspan="1">Detection of AMR GeneTarget by AMR GenePanel versus WholeGenome SequencingCombined Clinical DevicePerformance -Stock and ProspectiveIsolates (from K191288)</td><td colspan="1" rowspan="2">Antimicrobial</td><td colspan="4" rowspan="1">Detection of AMR Gene Target by AMR GenePanel vs AST Non-SusceptibilityCombined Clinical Device Performance - Stock andProspective Isolates (from K191288) includingonly unique isolates (no replicates)</td><td colspan="5" rowspan="1">Detection of AMR Gene Target versus AST Non-Susceptibility inOpGen's SMART Study</td><td colspan="1" rowspan="2">%Organismswith AMRGeneTarget inNCBIPathogenDatabase2019)</td></tr><tr><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1">PPV[95% CI]</td><td colspan="1" rowspan="1">NPV[95% CI]</td><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1">PPV[95% CI]</td><td colspan="1" rowspan="1">NPV[95% CI]</td><td colspan="1" rowspan="1">% Organismswith AMRGene Target</td></tr><tr><td colspan="1" rowspan="1">KPC</td><td colspan="1" rowspan="1">E. asburiae</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">Imipenem</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">KPC: 9.6%of 73</td></tr><tr><td colspan="1" rowspan="1">KPC</td><td colspan="1" rowspan="1">E. asburiae</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">Ertapenem</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">KPC: 9.6%of 73</td></tr><tr><td colspan="1" rowspan="1">KPC</td><td colspan="1" rowspan="1">E. asburiae</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">Meropenem</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">KPC: 9.6%of 73</td></tr><tr><td colspan="1" rowspan="1">KPC</td><td colspan="1" rowspan="1">E. cloacaecomplex</td><td colspan="1" rowspan="1">100.0%(9/9)[70.09-100.00]</td><td colspan="1" rowspan="1">100.0%(5/5)[56.55-0.00]</td><td colspan="1" rowspan="1">Imipenem</td><td colspan="1" rowspan="1">KPC:64.3%(9/14[38.76-83.66]</td><td colspan="1" rowspan="1">KPC: NaN</td><td colspan="1" rowspan="1">KPC:100.0%(9/9)[70.09-00]</td><td colspan="1" rowspan="1">KPC: 0.0%(0/5)[0.00-43.45]</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">KPC: 7.6%of 563</td></tr><tr><td colspan="1" rowspan="1">KPC</td><td colspan="1" rowspan="1">E. cloacaecomplex</td><td colspan="1" rowspan="1">100.0%(9/9)[70.09-100.00]</td><td colspan="1" rowspan="1">100.0%(5/5)56.55-00]</td><td colspan="1" rowspan="1">Ertapenem</td><td colspan="1" rowspan="1">KPC:64.3%(9/14)[38.76-83.66]</td><td colspan="1" rowspan="1">KPC: NaN</td><td colspan="1" rowspan="1">KPC:100.0%(9/9)[70.09-10.00]</td><td colspan="1" rowspan="1">KPC: 0.0%(0/5)0.00-43.45]</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">KPC: 7.6%of 563</td></tr><tr><td colspan="1" rowspan="1">KPC</td><td colspan="1" rowspan="1">E. cloacaecomplex</td><td colspan="1" rowspan="1">100.0%(9/9)[70.09-100.00]</td><td colspan="1" rowspan="1">100.0%(5/5)[56.55-100.00]</td><td colspan="1" rowspan="1">Meropenem</td><td colspan="1" rowspan="1">KPC:64.3%(9/14)[38.76-83.66]</td><td colspan="1" rowspan="1">KPC: NaN</td><td colspan="1" rowspan="1">KPC:100.0%(9/9)[70.09-100.00]</td><td colspan="1" rowspan="1">KPC: 0.0%(0/5)0.00-43.45]</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">KPC: 7.6%of 563</td></tr><tr><td colspan="1" rowspan="1">KPC</td><td colspan="1" rowspan="1">E. hormaechei</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">Imipenem</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">KPC:27.4% of678</td></tr><tr><td colspan="1" rowspan="1">KPC</td><td colspan="1" rowspan="1">E. hormaechei</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">Ertapenem</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">KPC:27.4% of678</td></tr><tr><td colspan="1" rowspan="1">KPC</td><td colspan="1" rowspan="1">E. hormaechei</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">Meropenem</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">KPC:27.4% of678</td></tr><tr><td colspan="1" rowspan="1">KPC</td><td colspan="1" rowspan="1">E. ludwigii</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">Imipenem</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">KPC: 0% of31</td></tr><tr><td colspan="1" rowspan="1">KPC</td><td colspan="1" rowspan="1">E. ludwigii</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">Ertapenem</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">KPC: 0% of31</td></tr><tr><td colspan="1" rowspan="1">KPC</td><td colspan="1" rowspan="1">E. ludwigii</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">Meropenem</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">KPC: 0% of31</td></tr><tr><td colspan="1" rowspan="2">AMR GeneTarget</td><td colspan="1" rowspan="2">Organism</td><td colspan="2" rowspan="1">Detection of AMR GeneTarget by AMR GenePanel versus WholeGenome SequencingCombined Clinical DevicePerformance -Stock and ProspectiveIsolates (from K191288)</td><td colspan="1" rowspan="2">Antimicrobial</td><td colspan="4" rowspan="1">Detection of AMR Gene Target by AMR GenePanel vs AST Non-SusceptibilityCombined Clinical Device Performance - Stock andivonly unique isolates (no replicates)</td><td colspan="5" rowspan="1">Detection of AMR Gene Target versus AST Non-Susceptibility inOpGen's SMART Study</td><td colspan="1" rowspan="2">%Organismswith AMRGeneTarget inNCBIPathogenDatabase(2019)</td></tr><tr><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1">PPV[95% CI]</td><td colspan="1" rowspan="1">NPV[95% CI]</td><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1">PPV[95% CI]</td><td colspan="1" rowspan="1">NPV[95% CI]</td><td colspan="1" rowspan="1">% Organismswith AMRGene Target</td></tr><tr><td colspan="1" rowspan="1">KPC</td><td colspan="1" rowspan="1">E. kobei</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">Imipenem</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">KPC: 7.1%of 70</td></tr><tr><td colspan="1" rowspan="1">KPC</td><td colspan="1" rowspan="1">E. kobei</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">Ertapenem</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">KPC: 7.1%of 70</td></tr><tr><td colspan="1" rowspan="1">KPC</td><td colspan="1" rowspan="1">E. kobei</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">Meropenem</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">KPC: 7.1%of 70</td></tr><tr><td colspan="1" rowspan="1">KPC</td><td colspan="1" rowspan="1">C. freundii</td><td colspan="1" rowspan="1">100.0%(3/3)[43.85-00]</td><td colspan="1" rowspan="1">100.0%(11/11)[74.12-100.00]</td><td colspan="1" rowspan="1">Imipenem</td><td colspan="1" rowspan="1">KPC:50.0%((3/6)18.76-81.24]</td><td colspan="1" rowspan="1">KPC:100.0%(8/8)67.56-10.00]</td><td colspan="1" rowspan="1">KPC:100.0%(3/3)[43.85-0.00]</td><td colspan="1" rowspan="1">KPC:72.7%(8/11)[43.44-90.25]</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">KPC:11.2% of179</td></tr><tr><td colspan="1" rowspan="1">KPC</td><td colspan="1" rowspan="1">C. freundii</td><td colspan="1" rowspan="1">100.0%(3/3)[43.85-100.00]</td><td colspan="1" rowspan="1">100.0%(11/11)[74.12-100.00]</td><td colspan="1" rowspan="1">Ertapenem</td><td colspan="1" rowspan="1">KPC:75.0%(3/4)[30.06-95.44]</td><td colspan="1" rowspan="1">KPC:100.0%(10/10)72.25100.00]</td><td colspan="1" rowspan="1">KPC:100.0%(3/3)4.85-0.00]</td><td colspan="1" rowspan="1">KPC:90.9%(10/11)[62.26-98.38]</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">KPC:11.2% of179</td></tr><tr><td colspan="1" rowspan="1">KPC</td><td colspan="1" rowspan="1">C. freundii</td><td colspan="1" rowspan="1">100.0%(3/3)[43.85-100.00]</td><td colspan="1" rowspan="1">100.0%(11/11)[74.12-100.00]</td><td colspan="1" rowspan="1">Meropenem</td><td colspan="1" rowspan="1">KPC:75.0%(3/4)[30.06-95.44]</td><td colspan="1" rowspan="1">KPC:100.0%110)[72.25-100.00]</td><td colspan="1" rowspan="1">KPC:100.0%(3/3)[43.85-100.00]</td><td colspan="1" rowspan="1">KPC:90.9%(10/11)[62.26-98.38]</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">KPC:11.2% of179</td></tr><tr><td colspan="1" rowspan="1">KPC</td><td colspan="1" rowspan="1">C. braakii</td><td colspan="1" rowspan="1">NaN</td><td colspan="1" rowspan="1">100.0%(1/1)[20.65-100.00]</td><td colspan="1" rowspan="1">Imipenem</td><td colspan="1" rowspan="1">KPC: 0.0%(0/1)[0.00-79.35]</td><td colspan="1" rowspan="1">KPC: NaN</td><td colspan="1" rowspan="1">KPC: NaN</td><td colspan="1" rowspan="1">KPC: 0.0%(0/1)[0.0-79.35]</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">KPC: 6.7%of 30</td></tr><tr><td colspan="1" rowspan="1">KPC</td><td colspan="1" rowspan="1">C. braakii</td><td colspan="1" rowspan="1">NaN</td><td colspan="1" rowspan="1">100.0%(1/1)[20.65-100.00]</td><td colspan="1" rowspan="1">Ertapenem</td><td colspan="1" rowspan="1">KPC: NaN</td><td colspan="1" rowspan="1">KPC:100.0%(1/1)[20.65-100.00]</td><td colspan="1" rowspan="1">KPC: NaN</td><td colspan="1" rowspan="1">KPC:100.0%(1/1)[20.65-0.00]</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">KPC: 6.7%of 30</td></tr><tr><td colspan="1" rowspan="1">KPC</td><td colspan="1" rowspan="1">C. braakii</td><td colspan="1" rowspan="1">NaN</td><td colspan="1" rowspan="1">100.0%(1/1)[20.65-100.00]</td><td colspan="1" rowspan="1">Meropenem</td><td colspan="1" rowspan="1">KPC: NaN</td><td colspan="1" rowspan="1">KPC:100.0%(1/1)200.65-0.00]</td><td colspan="1" rowspan="1">KPC: NaN</td><td colspan="1" rowspan="1">KPC:100.0%(1/1)20.5-10.00]</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">KPC: 6.7%of 30</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="2" rowspan="1"></td><td colspan="1" rowspan="1"></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">−</td></tr><tr><td colspan="1" rowspan="2">AMR GeneTarget</td><td colspan="1" rowspan="2">Organism</td><td colspan="2" rowspan="1">Detection of AMR GeneTarget by AMR GenePanel versus WholeGenome SequencingCombined Clinical DevicePerformance -Stock and ProspectiveIsolates (from K191288)</td><td colspan="1" rowspan="2">Antimicrobial</td><td colspan="4" rowspan="1">Detection of AMR Gene Target by AMR GenePanel vs AST Non-SusceptibilityCombined Clinical Device Performance - Stock andProspective Isolates (from K191288) includingonly unique isolates (no replicates)</td><td colspan="5" rowspan="1">Detection of AMR Gene Target versus AST Non-Susceptibility inOpGen's SMART Study</td><td colspan="1" rowspan="2">%Organismswith AMRGeneTarget inNCBIPathogenDatabase(2019)</td><td></td></tr><tr><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1">PPV[95% CI]</td><td colspan="1" rowspan="1">NPV[95% CI]</td><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1">PPV[95% CI]</td><td colspan="1" rowspan="1">NPV[95% CI]</td><td colspan="1" rowspan="1">% Organismswith AMRGene Target</td><td></td></tr><tr><td colspan="1" rowspan="1">KPC</td><td colspan="1" rowspan="1">C. werkmanii</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">Imipenem</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">KPC: 0% of10</td><td></td></tr><tr><td colspan="1" rowspan="1">KPC</td><td colspan="1" rowspan="1">C. werkmanii</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">Ertapenem</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">KPC: 0% of10</td><td></td></tr><tr><td colspan="1" rowspan="1">KPC</td><td colspan="1" rowspan="1">C. werkmanii</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">Meropenem</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">KPC: 0% of10</td><td></td></tr><tr><td colspan="1" rowspan="1">KPC</td><td colspan="1" rowspan="1">C. youngae</td><td colspan="1" rowspan="1">NaN</td><td colspan="1" rowspan="1">100.0%(1/1)[20.65-100.00]</td><td colspan="1" rowspan="1">Imipenem</td><td colspan="1" rowspan="1">KPC: 0.0%(0/1)[0.00-79.35]</td><td colspan="1" rowspan="1">KPC: NaN</td><td colspan="1" rowspan="1">KPC: NaN</td><td colspan="1" rowspan="1">KPC: 0.0%(0/1)[0.00-79.35]</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">KPC: 0% of7</td><td></td></tr><tr><td colspan="1" rowspan="1">KPC</td><td colspan="1" rowspan="1">C. youngae</td><td colspan="1" rowspan="1">NaN</td><td colspan="1" rowspan="1">100.0%(1/1)20.65-100.00]</td><td colspan="1" rowspan="1">Ertapenem</td><td colspan="1" rowspan="1">KPC: 0.0%(0/1)0.00-79.35]</td><td colspan="1" rowspan="1">KPC: NaN</td><td colspan="1" rowspan="1">KPC: NaN</td><td colspan="1" rowspan="1">KPC: 0.0%(0/1)[0.00-79.35]</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">KPC: 0% of7</td><td></td></tr><tr><td colspan="1" rowspan="1">KPC</td><td colspan="1" rowspan="1">C. youngae</td><td colspan="1" rowspan="1">NaN</td><td colspan="1" rowspan="1">100.0%(1/1)[20.65-100.00]</td><td colspan="1" rowspan="1">Meropenem</td><td colspan="1" rowspan="1">KPC: 0.0%(0/1)[0.00-79.35]</td><td colspan="1" rowspan="1">KPC: NaN</td><td colspan="1" rowspan="1">KPC: NaN</td><td colspan="1" rowspan="1">KPC: 0.0%(/1)[0.0-79.35]</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">KPC: 0% of7</td><td></td></tr><tr><td colspan="1" rowspan="1">KPC</td><td colspan="1" rowspan="1">C. koseri</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">Imipenem</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">KPC: 9.7%of 31</td><td></td></tr><tr><td colspan="1" rowspan="1">KPC</td><td colspan="1" rowspan="1">C. koseri</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">Ertapenem</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">KPC: 9.7%of 31</td><td></td></tr><tr><td colspan="1" rowspan="1">KPC</td><td colspan="1" rowspan="1">C. koseri</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">Meropenem</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">KPC: 9.7%of 31</td><td></td></tr><tr><td colspan="1" rowspan="1">KPC</td><td colspan="1" rowspan="1">S. marcescens</td><td colspan="1" rowspan="1">NaN</td><td colspan="1" rowspan="1">100.0%(2/2)34.24-100.00]</td><td colspan="1" rowspan="1">Imipenem</td><td colspan="1" rowspan="1">KPC: 0.0%(0/2)[00.00-65.76]</td><td colspan="1" rowspan="1">KPC: NaN</td><td colspan="1" rowspan="1">KPC: NaN</td><td colspan="1" rowspan="1">KPC: 0.0%(0/2)[0.00-65.76]</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">KPC: 6.5%of 496</td><td></td></tr><tr><td colspan="1" rowspan="1">KPC</td><td colspan="1" rowspan="1">S. marcescens</td><td colspan="1" rowspan="1">NaN</td><td colspan="1" rowspan="1">100.0%(2/2)[34.24-100.00]</td><td colspan="1" rowspan="1">Ertapenem</td><td colspan="1" rowspan="1">KPC: 0.0%(0/2)[0.00-65.76]</td><td colspan="1" rowspan="1">KPC: NaN</td><td colspan="1" rowspan="1">KPC: NaN</td><td colspan="1" rowspan="1">KPC: 0.0%(02)[0.00-65.76]</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">KPC: 6.5%of 496</td><td></td></tr><tr><td colspan="1" rowspan="1">KPC</td><td colspan="1" rowspan="1">S. marcescens</td><td colspan="1" rowspan="1">NaN</td><td colspan="1" rowspan="1">100.0%(2/2)[34.24-00]</td><td colspan="1" rowspan="1">Meropenem</td><td colspan="1" rowspan="1">KPC: 0.0%(0/2)[0.00-665.76]</td><td colspan="1" rowspan="1">KPC: NaN</td><td colspan="1" rowspan="1">KPC: NaN</td><td colspan="1" rowspan="1">KPC: 0.0%(0/2)[0.00-65.76]</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">KPC: 6.5%of 496</td><td></td></tr><tr><td colspan="1" rowspan="2">AMR GeneTarget</td><td colspan="1" rowspan="2">Organism</td><td colspan="2" rowspan="1">Detection of AMR GeneTarget by AMR GenePanel versus WholeGenome SequencingCombined Clinical DevicePerformance-Stock and ProspectiveIsolates (from K191288)</td><td colspan="1" rowspan="2">Antimicrobial</td><td colspan="4" rowspan="1">Detection of AMR Gene Target by AMR GenePanel vs AST Non-SusceptibilityCombined Clinical Device Performance - Stock andProspective Isolates (from K191288) includingonly unique isolates (no replicates)</td><td colspan="5" rowspan="1">Detection of AMR Gene Target versus AST Non-Susceptibility inOpGen's SMART Study</td><td colspan="1" rowspan="2">%Organismswith AMRGeneTTaarget inNCBIPathogenDatabase(2019)</td></tr><tr><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1">PPV[95% CI]</td><td colspan="1" rowspan="1">NPV[95% CI]</td><td colspan="1" rowspan="1">PPA[95% Cl]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1">PPV[95% CI]</td><td colspan="1" rowspan="1">NPV[95% Cl]</td><td colspan="1" rowspan="1">% Organismswith AMRGene Target</td></tr><tr><td colspan="1" rowspan="1">KPC</td><td colspan="1" rowspan="1">M. morganii</td><td colspan="1" rowspan="1">NaN</td><td colspan="1" rowspan="1">100.0%(7/7)[64.57-100.00]</td><td colspan="1" rowspan="1">Ertapenem</td><td colspan="1" rowspan="1">KPC: 0.0%(/2)[0.0-665.76]</td><td colspan="1" rowspan="1">KPC:100.0%(5/5)[56.55-00]</td><td colspan="1" rowspan="1">KPC: NaN</td><td colspan="1" rowspan="1">KPC:71.4%(5/7)[35.89-911.78]</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">KPC: 1.8%of 57</td></tr><tr><td colspan="1" rowspan="1">KPC</td><td colspan="1" rowspan="1">M. morganii</td><td colspan="1" rowspan="1">NaN</td><td colspan="1" rowspan="1">100.0%(7/7)[64.57-100.00]</td><td colspan="1" rowspan="1">Meropenem</td><td colspan="1" rowspan="1">KPC: 0.0%(0/2)[0.00-65.76]</td><td colspan="1" rowspan="1">KPC:100.0%(5//5[56.55-00]</td><td colspan="1" rowspan="1">KPC: NaN</td><td colspan="1" rowspan="1">KPC:71.4%(5/7)[35.89-91.78]</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">KPC: 1.8%of 57</td></tr><tr><td colspan="1" rowspan="1">KPC</td><td colspan="1" rowspan="1">R.ornithinolytica</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">Imipenem</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">KPC:10.7% of56</td></tr><tr><td colspan="1" rowspan="1">KPC</td><td colspan="1" rowspan="1">R.ornithinolytica</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">Ertapenem</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">KPC:10.7% of56</td></tr><tr><td colspan="1" rowspan="1">KPC</td><td colspan="1" rowspan="1">R.ornithinolytica</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">Meropenem</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">KPC:10.7% of56</td></tr><tr><td colspan="1" rowspan="1">KPC</td><td colspan="1" rowspan="1">R. planticola</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">Imipenem</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">KPC:15.6% of32</td></tr><tr><td colspan="1" rowspan="1">KPC</td><td colspan="1" rowspan="1">R. planticola</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">Ertapenem</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">KPC:15.6% of32</td></tr><tr><td colspan="1" rowspan="1">KPC</td><td colspan="1" rowspan="1">R. planticola</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">Meropenem</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">KPC:15.6% of32</td></tr><tr><td colspan="1" rowspan="1">NDM</td><td colspan="1" rowspan="1">K. pneumoniae</td><td colspan="1" rowspan="1">97.5% (339/40)[87.12-9.56]</td><td colspan="1" rowspan="1">98.6%(285/289)[96.50-99.46]</td><td colspan="1" rowspan="1">Imipenem</td><td colspan="1" rowspan="1">NDM:31.1%(33/106)[23.11-40.48]</td><td colspan="1" rowspan="1">NDM:98.7%(78/79)[93.17-99.78]</td><td colspan="1" rowspan="1">NDM:97.1%(33/34)[85.08-99.48]</td><td colspan="1" rowspan="1">NDM:51.7%(78/151)[43.74-59.49]</td><td colspan="1" rowspan="1">NDM:22.4%(183/818)[19.65-25.35]</td><td colspan="1" rowspan="1">NDM:99.7%(1206/1210) [99.15-99.87]</td><td colspan="1" rowspan="1">NDM:97.9%(183/187)[94.63-99.17]</td><td colspan="1" rowspan="1">NDM:65.5%(1206/1841) [63.31-67.64]</td><td colspan="1" rowspan="1">NDM: 9.2%(187/2028)</td><td colspan="1" rowspan="1">NDM:7.3% of7979</td></tr><tr><td colspan="1" rowspan="2">AMR GeneTarget</td><td colspan="1" rowspan="2">Organism</td><td colspan="2" rowspan="1">Detection of AMR GeneTarget by AMR GenePanel versus WholeGenome SequencingCombined Clinical DevicePerformance -Stock and ProspectiveIsolates (from K191288)</td><td colspan="1" rowspan="2">Antimicrobial</td><td colspan="4" rowspan="1">Detection of AMR Gene Target by AMR GenePanel vs AST Non-SusceptibilityCombined Clinical Device Performance - Stock andProspective Isolates (from K191288) includingonly unique isolates (no replicates)</td><td colspan="5" rowspan="1">Detection of AMR Gene Target versus AST Non-Susceptibility inOpGen's SMART Study</td><td colspan="1" rowspan="2">%Organismswith AMRGeneTarget inNCBIPathogenDatabase(2019)</td></tr><tr><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1">PPV[95% CI]</td><td colspan="1" rowspan="1">NPV[95% CI]</td><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1">PPV[95% CI]</td><td colspan="1" rowspan="1">NPV[95% CI]</td><td colspan="1" rowspan="1">% Organismswith AMRGene Target</td></tr><tr><td colspan="1" rowspan="1">NDM</td><td colspan="1" rowspan="1">K. pneumoniae</td><td colspan="1" rowspan="1">97.5%(39/40)[87.12-99.56]</td><td colspan="1" rowspan="1">98.6%(285/289)[96.50-99.46]</td><td colspan="1" rowspan="1">Ertapenem</td><td colspan="1" rowspan="1">NDM:27.0%(34/126)[20.00-35.33]</td><td colspan="1" rowspan="1">NDM:100.0%(59/59)93.89-0.00]</td><td colspan="1" rowspan="1">NDM:100.0%(34/34)89.5100.00]</td><td colspan="1" rowspan="1">NDM:39.1%(59/151)[31.65-47.03]</td><td colspan="1" rowspan="1">NDM:17.3%(182/1053)[15.12-19.69]</td><td colspan="1" rowspan="1">NDM:99.5%970/975)[98.81-99.78]</td><td colspan="1" rowspan="1">NDM:97.3%(182/187)[93.89-8.85]</td><td colspan="1" rowspan="1">NDM:52.7%(970/1841)[50.40-54.96]</td><td colspan="1" rowspan="1">NDM: 9.2%(187/2028)</td><td colspan="1" rowspan="1">NDM:7.3% of7979</td></tr><tr><td colspan="1" rowspan="1">NDM</td><td colspan="1" rowspan="1">K. pneumoniae</td><td colspan="1" rowspan="1">97.5%(39/40)[87.12-99.56]</td><td colspan="1" rowspan="1">98.6%(285/289)[96.50-99.46]</td><td colspan="1" rowspan="1">Meropenem</td><td colspan="1" rowspan="1">NDM:33.3%(33/99)[24.82-43.09]</td><td colspan="1" rowspan="1">NDM:98.8%(85/86)[93.70-99.79]</td><td colspan="1" rowspan="1">NDM:97.1%(33/34)[85.08-99.48]</td><td colspan="1" rowspan="1">NDM:56.3%(85/151)[48.32-63.95]</td><td colspan="1" rowspan="1">NDM:21.6%(182/842)[18.97-24.52]</td><td colspan="1" rowspan="1">NDM:99.6%(1181/1186) [99.02-9.82]</td><td colspan="1" rowspan="1">NDM:97.3%(182/187)[93.89-98.85]</td><td colspan="1" rowspan="1">NDM:64.1%(1181/1841) [61.93-66.31]</td><td colspan="1" rowspan="1">NDM: 9.2%(187/2028)</td><td colspan="1" rowspan="1">NDM:7.3% of7979</td></tr><tr><td colspan="1" rowspan="1">NDM</td><td colspan="1" rowspan="1">P. mirabilis</td><td colspan="1" rowspan="1">NaN</td><td colspan="1" rowspan="1">100.0%(155/155)[97.58-100.00]</td><td colspan="1" rowspan="1">Ertapenem</td><td colspan="1" rowspan="1">NDM:0.0%(0/4)[0.00-48.99]</td><td colspan="1" rowspan="1">NDM:100.0%(74/74)[95.07-100.00]</td><td colspan="1" rowspan="1">NDM:NaN</td><td colspan="1" rowspan="1">NDM:94.9%(74/78)[87.54-97.99]</td><td colspan="1" rowspan="1">NDM:31.2%(5/16)[14.16-55.60]</td><td colspan="1" rowspan="1">NDM:99.8%(1139/1141) [99.36-99.95]</td><td colspan="1" rowspan="1">NDM:71.4%(5/7)[35.89-91.78]</td><td colspan="1" rowspan="1">NDM:99.0%(1139/1150) [98.30-99.47]</td><td colspan="1" rowspan="1">NDM: 0.6%(7/1157)</td><td colspan="1" rowspan="1">NDM:5.6% of144</td></tr><tr><td colspan="1" rowspan="1">NDM</td><td colspan="1" rowspan="1">P. mirabilis</td><td colspan="1" rowspan="1">NaN</td><td colspan="1" rowspan="1">100.0%(155/155)[97.58-0.00]</td><td colspan="1" rowspan="1">Meropenem</td><td colspan="1" rowspan="1">NDM:0.0%(00/3)[0.00-56.15]</td><td colspan="1" rowspan="1">NDM:100.0%(7/75)[95.13-100.00]</td><td colspan="1" rowspan="1">NDM:NaN</td><td colspan="1" rowspan="1">NDM:96.2%(75/78)[89.29-98.68]</td><td colspan="1" rowspan="1">NDM:35.3%6/17)[17.31-58.0]</td><td colspan="1" rowspan="1">NDM:99.9%(1139/1140) [99.50-99.98]</td><td colspan="1" rowspan="1">NDM:85.7%(6/7)[48.69-97.43]</td><td colspan="1" rowspan="1">NDM:99.0%(1139/1150) [98.30-99.47]</td><td colspan="1" rowspan="1">NDM: 0.6%(7/1157)</td><td colspan="1" rowspan="1">NDM:5.6% of144</td></tr><tr><td colspan="1" rowspan="1">NDM</td><td colspan="1" rowspan="1">P. aeruginosa</td><td colspan="1" rowspan="1">100.0%(15/15)[79.61-10.00]</td><td colspan="1" rowspan="1">100.0%(333/333)[98.86-10.00]</td><td colspan="1" rowspan="1">Imipenem</td><td colspan="1" rowspan="1">NDM:9.2%(11/119)[5.24-15.80]</td><td colspan="1" rowspan="1">NDM:100.0%(36/36)90.36-100.00]</td><td colspan="1" rowspan="1">NDM:100.0%(11/11)[74.12-100.00]</td><td colspan="1" rowspan="1">NDM:25.0%(36/144)[18.64-32.66]</td><td colspan="1" rowspan="1">NDM: 1.0%(10/1002)[0.54-1.83]</td><td colspan="1" rowspan="1">NDM:100.0%(548/548)[99.30-100.00]</td><td colspan="1" rowspan="1">NDM:100.0%(10/10)[72.25-100.00]</td><td colspan="1" rowspan="1">NDM:35.6%(548/1540)[33.23-38.01]</td><td colspan="1" rowspan="1">NDM: 0.6%(10/1550)</td><td colspan="1" rowspan="1">NDM:0.6% of4706</td></tr><tr><td colspan="1" rowspan="1">NDM</td><td colspan="1" rowspan="1">P. aeruginosa</td><td colspan="1" rowspan="1">100.0%(15/15)[79.61-100.00]</td><td colspan="1" rowspan="1">100.0%(333/333)[98.86-100.00]</td><td colspan="1" rowspan="1">Meropenem</td><td colspan="1" rowspan="1">NDM:9.3%(11/118)[5.29-15.92]</td><td colspan="1" rowspan="1">NDM:100.0%(37/37)[90.59-0.00]</td><td colspan="1" rowspan="1">NDM:100.0%(11/11)[74.12-100.00]</td><td colspan="1" rowspan="1">NDM:25.7%(37/144)[19.25-33.40]</td><td colspan="1" rowspan="1">NDM: 1.0%(10/986)[0.55-1.86]</td><td colspan="1" rowspan="1">NDM:100.0%(564/564)[99.32-100.00]</td><td colspan="1" rowspan="1">NDM:10.0%(10/10)[72.25-100.00]</td><td colspan="1" rowspan="1">NDM:36.6%(564/1540)[34.25-39.06]</td><td colspan="1" rowspan="1">NDM: 0.6%(10/1550)</td><td colspan="1" rowspan="1">NDM:0.6% of4706</td></tr><tr><td colspan="1" rowspan="1">NDM</td><td colspan="1" rowspan="1">K. aerogenes</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NotEvaluated</td><td colspan="1" rowspan="1">Imipenem</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NDM:1.3% of223</td></tr><tr><td colspan="1" rowspan="1">NDM</td><td colspan="1" rowspan="1">K. aerogenes</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NotEvaluated</td><td colspan="1" rowspan="1">Ertapenem</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NDM:1.3% of223</td></tr><tr><td colspan="1" rowspan="1">NDM</td><td colspan="1" rowspan="1">K. aerogenes</td><td colspan="1" rowspan="1">NotEvaluated</td><td colspan="1" rowspan="1">NotEvaluated</td><td colspan="1" rowspan="1">Meropenem</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NDM:1.3% of223</td></tr><tr><td colspan="1" rowspan="1">NDM</td><td colspan="1" rowspan="1">K.michiganensis</td><td colspan="1" rowspan="1">NotEvaluated</td><td colspan="1" rowspan="1">NotEvaluated</td><td colspan="1" rowspan="1">Imipenem</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NDM: 9%of 89</td></tr><tr><td colspan="1" rowspan="1">NDM</td><td colspan="1" rowspan="1">K.michiganensis</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">Ertapenem</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NDM: 9%of 89</td></tr><tr><td colspan="1" rowspan="1">NDM</td><td colspan="1" rowspan="1">K.michiganensis</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">Meropenem</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NDM: 9%of 89</td></tr><tr><td colspan="1" rowspan="1">NDM</td><td colspan="1" rowspan="1">K. oxytoca</td><td colspan="1" rowspan="1">NaN</td><td colspan="1" rowspan="1">100.0%(5/5)556.55-100]</td><td colspan="1" rowspan="1">Imipenem</td><td colspan="1" rowspan="1">NDM:0.0%(0/0/2)[0.00-65.76]</td><td colspan="1" rowspan="1">NDM:100.0%(3/3)[43.85-10.00]</td><td colspan="1" rowspan="1">NDM:NaN</td><td colspan="1" rowspan="1">NDM:60.0%(3/5) .07-88.24]</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NDM:1.1% of273</td></tr><tr><td colspan="1" rowspan="1">NDM</td><td colspan="1" rowspan="1">K. oxytoca</td><td colspan="1" rowspan="1">NaN</td><td colspan="1" rowspan="1">100.0%(5/5)[56.55- 100.00]</td><td colspan="1" rowspan="1">Ertapenem</td><td colspan="1" rowspan="1">NDM:0.0%(0//2)[0.00-65.76]</td><td colspan="1" rowspan="1">NDM:100.0%(3/3)[43.85-100.00]</td><td colspan="1" rowspan="1">NDM:NaN</td><td colspan="1" rowspan="1">NDM:60.0%(3/5)[23.07-88.24]</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NDM:1.1% of273</td></tr><tr><td colspan="1" rowspan="1">NDM</td><td colspan="1" rowspan="1">K. oxytoca</td><td colspan="1" rowspan="1">NaN</td><td colspan="1" rowspan="1">100.0%(5/5)5 6.5-100.00]</td><td colspan="1" rowspan="1">Meropenem</td><td colspan="1" rowspan="1">NDM:0.0%(0/2)[0.00-65.76]</td><td colspan="1" rowspan="1">NDM:100.0%(3/3)[43.85-0.00]</td><td colspan="1" rowspan="1">NDM:NaN</td><td colspan="1" rowspan="1">NDM:60.0%(3/5)[23.07-88.24]</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NDM:1.1% of273</td></tr><tr><td colspan="1" rowspan="1">NDM</td><td colspan="1" rowspan="1">K. variicola</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">Imipenem</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NDM:3.3% of272</td></tr><tr><td colspan="1" rowspan="1">NDM</td><td colspan="1" rowspan="1">K. variicola</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">Ertapenem</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NDM:3.3% of272</td></tr><tr><td colspan="1" rowspan="1">NDM</td><td colspan="1" rowspan="1">K. variicola</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">Meropenem</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NDM:3.3% of272</td></tr><tr><td colspan="1" rowspan="1">NDM</td><td colspan="1" rowspan="1">K.quasipneumoniae</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">Imipenem</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NDM:20.5% of259</td></tr><tr><td colspan="1" rowspan="2">AMR GeneTarget</td><td colspan="1" rowspan="2">Organism</td><td colspan="2" rowspan="1">Detection of AMR GeneTarget by AMR GenePanel versus WholeGenome SequencingCombined Clinical DevicePerformance-Stock and ProspectiveIsolates (from K191288)</td><td colspan="1" rowspan="2">Antimicrobial</td><td colspan="4" rowspan="1">Detection of AMR Gene Target by AMR GenePanel vs AST Non-SusceptibilityCombined Clinical Device Performance - Stock andProspective Isolates (from K191288) includingonly unique isolates (no replicates)</td><td colspan="5" rowspan="1">Detection of AMR Gene Target versus AST Non-Susceptibility inOpGen's SMART Study</td><td colspan="1" rowspan="2">%Organismswith AMRGeneTarget inNCBIPathogenDatabase(2019)</td></tr><tr><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1">PPV[95% CI]</td><td colspan="1" rowspan="1">NPV[95% CI]</td><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1">PPV[95% CI]</td><td colspan="1" rowspan="1">NPV[95% CI]</td><td colspan="1" rowspan="1">% Organismswith AMRGene Target</td></tr><tr><td colspan="1" rowspan="1">NDM</td><td colspan="1" rowspan="1">K.quasipneumoniae</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">Ertapenem</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NDM:20.5% of259</td></tr><tr><td colspan="1" rowspan="1">NDM</td><td colspan="1" rowspan="1">K.quasipneumoniae</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">Meropenem</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NDM:20.5% of259</td></tr><tr><td colspan="1" rowspan="1">NDM</td><td colspan="1" rowspan="1">C. freundii</td><td colspan="1" rowspan="1">NaN</td><td colspan="1" rowspan="1">100.0%(14/14)[78.47-100.00]</td><td colspan="1" rowspan="1">Imipenem</td><td colspan="1" rowspan="1">NDM:0.0%(0/6)[0.0-39.03]</td><td colspan="1" rowspan="1">NDM:100.0%(8/8)67.56-100.00]</td><td colspan="1" rowspan="1">NDM:NaN</td><td colspan="1" rowspan="1">NDM:57.1%(8/14)[32.59-78.62]</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NDM: 5%of 179</td></tr><tr><td colspan="1" rowspan="1">NDM</td><td colspan="1" rowspan="1">C. freundii</td><td colspan="1" rowspan="1">NaN</td><td colspan="1" rowspan="1">100.0%(14/14)[78.47-100.00]</td><td colspan="1" rowspan="1">Ertapenem</td><td colspan="1" rowspan="1">NDM:0.0%(0/4)[0.00-48.99]</td><td colspan="1" rowspan="1">NDM:100.0%(10/10)[72.25-0.00]</td><td colspan="1" rowspan="1">NDM:NaN</td><td colspan="1" rowspan="1">NDM:71.4%(10/14)[45.35-88.28]</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NDM: 5%of 179</td></tr><tr><td colspan="1" rowspan="1">NDM</td><td colspan="1" rowspan="1">C. freundii</td><td colspan="1" rowspan="1">NaN</td><td colspan="1" rowspan="1">100.0%(14/14)[78.47-100.00]</td><td colspan="1" rowspan="1">Meropenem</td><td colspan="1" rowspan="1">NDM:0.0%(0/4)[0.00-48.99]</td><td colspan="1" rowspan="1">NDM:100.0%(10/10)[72.25-100.00]</td><td colspan="1" rowspan="1">NDM:NaN</td><td colspan="1" rowspan="1">NDM:71.4%(10/14)[45.35-88.28]</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NDM: 5%of 179</td></tr><tr><td colspan="1" rowspan="1">NDM</td><td colspan="1" rowspan="1">C. braakii</td><td colspan="1" rowspan="1">NaN</td><td colspan="1" rowspan="1">100.0%(1/1)[20.65-100.00]</td><td colspan="1" rowspan="1">Imipenem</td><td colspan="1" rowspan="1">NDM:0.0%(0/1)0.00-79.35]</td><td colspan="1" rowspan="1">NDM:NaN</td><td colspan="1" rowspan="1">NDM:NaN</td><td colspan="1" rowspan="1">NDM:0.0%(0/1)[0.00-79.35]</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NDM: 0%of 30</td></tr><tr><td colspan="1" rowspan="1">NDM</td><td colspan="1" rowspan="1">C. braakii</td><td colspan="1" rowspan="1">NaN</td><td colspan="1" rowspan="1">100.0%(1/1)[20.65-100.00]</td><td colspan="1" rowspan="1">Ertapenem</td><td colspan="1" rowspan="1">NDM:NaN</td><td colspan="1" rowspan="1">NDM:100.0%(1/1)[20.65-0.00]</td><td colspan="1" rowspan="1">NDM:NaN</td><td colspan="1" rowspan="1">NDM:100.0%(1/1)[20.65-0.00]</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NDM: 0%of 30</td></tr><tr><td colspan="1" rowspan="1">NDM</td><td colspan="1" rowspan="1">C. braakii</td><td colspan="1" rowspan="1">NaN</td><td colspan="1" rowspan="1">100.0%(1/1)[20.65-100.00]</td><td colspan="1" rowspan="1">Meropenem</td><td colspan="1" rowspan="1">NDM:NaN</td><td colspan="1" rowspan="1">NDM:100.0%(1/1)[20.65-10.00]</td><td colspan="1" rowspan="1">NDM:NaN</td><td colspan="1" rowspan="1">NDM:100.0%(1/1)[20.65-10.00]</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NDM: 0%of 30</td></tr><tr><td colspan="1" rowspan="2">AMR GeneTarget</td><td colspan="1" rowspan="2">Organism</td><td colspan="2" rowspan="1">Detection of AMR GeneTarget by AMR GenePanel versus WholeGenome SequencingCombined Clinical DevicePerformance -Stock and ProspectiveIsolates (from K191288)</td><td colspan="1" rowspan="2">Antimicrobial</td><td colspan="4" rowspan="1">Detection of AMR Gene Target by AMR GenePanel vs AST Non-SusceptibilityCombined Clinical Device Performance - Stock andProspective Isolates (from K191288) includingonly unique isolates (no replicates)</td><td colspan="5" rowspan="1">Detection of AMR Gene Target versus AST Non-Susceptibility inOpGen's SMART Study</td><td colspan="1" rowspan="2">%Organismswith AMRGeneTarget inNCBIPathogenDatabase2019)</td></tr><tr><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1">PPV[95% CI]</td><td colspan="1" rowspan="1">NPV[95% CI]</td><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1">PPV[95% CI]</td><td colspan="1" rowspan="1">NPV[95% CI]</td><td colspan="1" rowspan="1">% Organismswith AMRGene Target</td></tr><tr><td colspan="1" rowspan="1">NDM</td><td colspan="1" rowspan="1">C. werkmanii</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">Imipenem</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NDM:20.0% of10</td></tr><tr><td colspan="1" rowspan="1">NDM</td><td colspan="1" rowspan="1">C. werkmanii</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">Ertapenem</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NDM:20.0% of10</td></tr><tr><td colspan="1" rowspan="1">NDM</td><td colspan="1" rowspan="1">C. werkmanii</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">Meropenem</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NDM:20.0% of10</td></tr><tr><td colspan="1" rowspan="1">NDM</td><td colspan="1" rowspan="1">C. youngae</td><td colspan="1" rowspan="1">100.0%(1/1)[20.65-100.00]</td><td colspan="1" rowspan="1">NaN</td><td colspan="1" rowspan="1">Imipenem</td><td colspan="1" rowspan="1">NDM:100.0%(1/1)[20.65-100.00]</td><td colspan="1" rowspan="1">NDM:NaN</td><td colspan="1" rowspan="1">NDM:100.0%(1/1)[20.65-00]</td><td colspan="1" rowspan="1">NDM:NaN</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NDM: 0%of 7</td></tr><tr><td colspan="1" rowspan="1">NDM</td><td colspan="1" rowspan="1">C. youngae</td><td colspan="1" rowspan="1">100.0%(1/1)20.5-100.00]</td><td colspan="1" rowspan="1">NaN</td><td colspan="1" rowspan="1">Ertapenem</td><td colspan="1" rowspan="1">NDM:100.0%(1/1)[20.65-00]</td><td colspan="1" rowspan="1">NDM:NaN</td><td colspan="1" rowspan="1">NDM:100.0%(1/1)[20.65-0.00]</td><td colspan="1" rowspan="1">NDM:NaN</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NDM: 0%of 7</td></tr><tr><td colspan="1" rowspan="1">NDM</td><td colspan="1" rowspan="1">C. youngae</td><td colspan="1" rowspan="1">100.0%(1/1)[20.65-100.00]</td><td colspan="1" rowspan="1">NaN</td><td colspan="1" rowspan="1">Meropenem</td><td colspan="1" rowspan="1">NDM:100.0%(1/1)[20.65-100.00]</td><td colspan="1" rowspan="1">NDM:NaN</td><td colspan="1" rowspan="1">NDM:100.0%(1/1)[20.65-100.00]</td><td colspan="1" rowspan="1">NDM:NaN</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NDM: 0%of 7</td></tr><tr><td colspan="1" rowspan="1">NDM</td><td colspan="1" rowspan="1">S. marcescens</td><td colspan="1" rowspan="1">NaN</td><td colspan="1" rowspan="1">100.0%(2/2)[34.24-100.00]</td><td colspan="1" rowspan="1">Imipenem</td><td colspan="1" rowspan="1">NDM:0.0%(0/2)[0.00-65.76]</td><td colspan="1" rowspan="1">NDM:NaN</td><td colspan="1" rowspan="1">NDM:NaN</td><td colspan="1" rowspan="1">NDM:0.0%(0/2)[0.00-65.76]</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NDM:1.6% of496</td></tr><tr><td colspan="1" rowspan="1">NDM</td><td colspan="1" rowspan="1">S. marcescens</td><td colspan="1" rowspan="1">NaN</td><td colspan="1" rowspan="1">100.0%(2/2)[34.24-100.00]</td><td colspan="1" rowspan="1">Ertapenem</td><td colspan="1" rowspan="1">NDM:0.0%(0/2)[0.00-65.76]</td><td colspan="1" rowspan="1">NDM:NaN</td><td colspan="1" rowspan="1">NDM:NaN</td><td colspan="1" rowspan="1">NDM:0.0%(0/2)[0.00-665.76]</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NDM:1.6% of496</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="2" rowspan="1"></td><td colspan="1" rowspan="1"></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">−</td></tr><tr><td colspan="1" rowspan="2">AMR GeneTarget</td><td colspan="1" rowspan="2">Organism</td><td colspan="2" rowspan="1">Detection of AMR GeneTarget by AMR GenePanel versus WholeGenome SequencingCombined Clinical DevicePerformance-Stock and ProspectiveIsolates (from K191288)</td><td colspan="1" rowspan="2">Antimicrobial</td><td colspan="4" rowspan="1">Detection of AMR Gene Target by AMR GenePanel vs AST Non-SusceptibilityCombined Clinical Device Performance - Stock andProspective Isolates (from K191288) includingonly unique isolates (no replicates)</td><td colspan="5" rowspan="1">Detection of AMR Gene Target versus AST Non-Susceptibility inOpGen's SMART Study</td><td colspan="1" rowspan="2">%Organismswith AMRGeneTarget inNCBIPathogenDatabase(2019)</td><td></td></tr><tr><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1">PPV[95% CI]</td><td colspan="1" rowspan="1">NPV[95% CI]</td><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1">PPV[95% CI]</td><td colspan="1" rowspan="1">NPV[95% CI]</td><td colspan="1" rowspan="1">% Organismswith AMRGene Target</td><td></td></tr><tr><td colspan="1" rowspan="1">NDM</td><td colspan="1" rowspan="1">S. marcescens</td><td colspan="1" rowspan="1">NaN</td><td colspan="1" rowspan="1">100.0%(2/2)[34.24-100.00]</td><td colspan="1" rowspan="1">Meropenem</td><td colspan="1" rowspan="1">NDM:0.0%(0/2)[0.00-65.76]</td><td colspan="1" rowspan="1">NDM:NaN</td><td colspan="1" rowspan="1">NDM:NaN</td><td colspan="1" rowspan="1">NDM:0.0%(0/2)0.00-665.76]</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NDM:1.6% of496</td><td></td></tr><tr><td colspan="1" rowspan="1">NDM</td><td colspan="1" rowspan="1">M. morganii</td><td colspan="1" rowspan="1">100.0%(1/1)20.5-100.00]</td><td colspan="1" rowspan="1">100.0%(6/6)60.97-0.00]</td><td colspan="1" rowspan="1">Ertapenem</td><td colspan="1" rowspan="1">NDM:50.0%(1/2)[9.45-900.55]</td><td colspan="1" rowspan="1">NDM:100.0%(5/5)[56.55-0.00]</td><td colspan="1" rowspan="1">NDM:100.0%(1/1)[20.65-0.00]</td><td colspan="1" rowspan="1">NDM:83.3%(5/6)[43.65-96.99]</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NDM:12.3% of57</td><td></td></tr><tr><td colspan="1" rowspan="1">NDM</td><td colspan="1" rowspan="1">M. morganii</td><td colspan="1" rowspan="1">100.0%(1/1)[20.65-100.00]</td><td colspan="1" rowspan="1">100.0%(6/6)[60.97-100.00]</td><td colspan="1" rowspan="1">Meropenem</td><td colspan="1" rowspan="1">NDM:50.0%(1/2)[9.45-90.55]</td><td colspan="1" rowspan="1">NDM:100.0%(5/5)[56.55-100.00]</td><td colspan="1" rowspan="1">NDM:100.0%(1/1)20.65-100.00]</td><td colspan="1" rowspan="1">NDM:83.3%(5/6)[43.65-96.99]</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NDM:12.3% of57</td><td></td></tr><tr><td colspan="1" rowspan="1">NDM</td><td colspan="1" rowspan="1">P. rettgeri</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">Imipenem</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NDM:41.7% of24</td><td></td></tr><tr><td colspan="1" rowspan="1">NDM</td><td colspan="1" rowspan="1">P. rettgeri</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">Ertapenem</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NDM:41.7% of24</td><td></td></tr><tr><td colspan="1" rowspan="1">NDM</td><td colspan="1" rowspan="1">P. rettgeri</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">Meropenem</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NDM:41.7% of24</td><td></td></tr><tr><td colspan="1" rowspan="1">NDM</td><td colspan="1" rowspan="1">P. stuartii</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">Imipenem</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NDM:9.1% of 22</td><td></td></tr><tr><td colspan="1" rowspan="1">NDM</td><td colspan="1" rowspan="1">P. stuartii</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">Ertapenem</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NDM:9.1% of 22</td><td></td></tr><tr><td colspan="1" rowspan="1">NDM</td><td colspan="1" rowspan="1">P. stuartii</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">Meropenem</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NDM:9.1% of 22</td><td></td></tr><tr><td colspan="1" rowspan="1">NDM</td><td colspan="1" rowspan="1">R.ornithinolytica</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">Imipenem</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NDM:5.4% of 56</td><td></td></tr><tr><td colspan="1" rowspan="1">NDM</td><td colspan="1" rowspan="1">R.ornithinolytica</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">Ertapenem</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NDM:5.4% of 56</td><td></td></tr><tr><td colspan="1" rowspan="1">NDM</td><td colspan="1" rowspan="1">R.ornithinolytica</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">Meropenem</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NDM:5.4% of 56</td><td></td></tr></table>

<table><tr><td colspan="1" rowspan="2">AMR GeneTarget</td><td colspan="1" rowspan="2">Organism</td><td colspan="2" rowspan="1">Detection of AMR GeneTarget by AMR GenePanel versus WholeGenome SequencingCombined Clinical DevicePerformance -Stock and ProspectiveIsolates (from K191288)</td><td colspan="1" rowspan="1">Antimicrobial</td><td colspan="4" rowspan="1">Detection of AMR Gene Target by AMR GenePanel vs AST Non-SusceptibilityCombined Clinical Device Performance - Stock andProspective Isolates (from K191288) includingonly unique isolates (no replicates)</td><td colspan="5" rowspan="1">Detection of AMR Gene Target versus AST Non-Susceptibility inOpGen's SMART Study</td><td colspan="1" rowspan="2">%Organismswith AMRGeneTarget inNCBIPathogenDatabase(2019)</td></tr><tr><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1">PPV[95% CI]</td><td colspan="1" rowspan="1">NPV[95% CI]</td><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1">PPV[95% CI]</td><td colspan="1" rowspan="1">NPV[95% CI]</td><td colspan="1" rowspan="1">% Organismswith AMRGene Target</td></tr><tr><td colspan="1" rowspan="1">OXA-1</td><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">96.6%(84/87)[90.35-98.82]</td><td colspan="1" rowspan="1">100.0%248/248)[98.47-10.00]</td><td colspan="1" rowspan="1">Ampicillin</td><td colspan="1" rowspan="1">OXA-1:18.2%(28/154)[12.89-25.02]</td><td colspan="1" rowspan="1">OXA-1:100.0%(5/5) 56.5-10.00]</td><td colspan="1" rowspan="1">OXA-1:100.0%(28/28)[87.94-100.00]</td><td colspan="1" rowspan="1">OXA-1:3.8%((5/131)[1.64-8.62]</td><td colspan="1" rowspan="1">OXA-1:32.9%(853/2594)[31.10-34.72]</td><td colspan="1" rowspan="1">OXA-1:96.4%(353/366)[94.02-97.1]</td><td colspan="1" rowspan="1">OXA-1:98.5%(853/866)[97.45-99.12]</td><td colspan="1" rowspan="1">OXA-1:16.9%(353/2094)[15.32-18.52]</td><td colspan="1" rowspan="1">OXA-1:29.3%(866/2960)</td><td colspan="1" rowspan="1">OXA-1:5.1% of19017</td></tr><tr><td colspan="1" rowspan="1">OXA-1</td><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">96.6%(84/87)[90.35-98.82]</td><td colspan="1" rowspan="1">100.0%(248/248)[98.47-10.00]</td><td colspan="1" rowspan="1">Amox.K.Clav</td><td colspan="1" rowspan="1">OXXA-1:23.7%(27/114)[16.82-32.27]</td><td colspan="1" rowspan="1">OXA-1:97.8%(44/45)[88.43-99.61]</td><td colspan="1" rowspan="1">OXA-1:96.4%(27/28)[82.29-99.37]</td><td colspan="1" rowspan="1">OXA-1:33.6%(44/131)[26.07-42.04]</td><td colspan="1" rowspan="1">OXA-1:44.2%(0/1810)[41.93-46.50]</td><td colspan="1" rowspan="1">OXA-1:94.3%(1084/1150) [92.76-95.46]</td><td colspan="1" rowspan="1">OXA-1:92.4%(800/866)[90.42-93.96]</td><td colspan="1" rowspan="1">OXA-1:51.8%(1084/2094) [49.63-53.901]</td><td colspan="1" rowspan="1">OXA-1:29.3%(866/2960)</td><td colspan="1" rowspan="1">OXA-1:5.1% of19017</td></tr><tr><td colspan="1" rowspan="1">OXA-1</td><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">96.6%(84/87)[90.35-98.82]</td><td colspan="1" rowspan="1">100.0%(248/248)[98.47-100.00]</td><td colspan="1" rowspan="1">Amp.Sulbactam</td><td colspan="1" rowspan="1">OXA-1:18.5%(27/146)13.03-25.57]</td><td colspan="1" rowspan="1">OXA-1:92.3%(12/13)66.6998.63]</td><td colspan="1" rowspan="1">OXA-1:96.4%(27/28)[82.29-999.37]</td><td colspan="1" rowspan="1">OXA-1:9.2%(12/131)[5.32-15.3]</td><td colspan="1" rowspan="1">OXA-1:33.8%(836/2477)[31.91-35.64]</td><td colspan="1" rowspan="1">OXA-1:93.8%(453/483)91.27-5.61]</td><td colspan="1" rowspan="1">OXA-1:96.5%(836/866)[9510-97.56]</td><td colspan="1" rowspan="1">OXA-1:21.6%(453/2094)19.92-23.45]</td><td colspan="1" rowspan="1">OXA-1:29.3%(866/2960)</td><td colspan="1" rowspan="1">OXA-1:5.1% of19017</td></tr><tr><td colspan="1" rowspan="1">OXA-1</td><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">96.6%(84/87)90.35-98.82]</td><td colspan="1" rowspan="1">100.0%(24/248)[98.47-100.00]</td><td colspan="1" rowspan="1">Pip.Tazo</td><td colspan="1" rowspan="1">OXA-1:27.5%(19/69)[18.39-39.05]</td><td colspan="1" rowspan="1">OXA-1:90.0%(81/90)[82.08-94.65]</td><td colspan="1" rowspan="1">OXA-1:67.9%(19/28)[49.34-82.07]</td><td colspan="1" rowspan="1">OXA-1:61.8%(1/131)[53.29-69.70]</td><td colspan="1" rowspan="1">OXA-1:49.5%(340/687)[45.77-53.22]</td><td colspan="1" rowspan="1">OXA-1:76.9%(1747/2273) [75.08-78.55]</td><td colspan="1" rowspan="1">OXA-1:39.3%(340/866)[36.06-42.55]</td><td colspan="1" rowspan="1">OXA-1:83.4%(1747/2094) [81.78-84.96]</td><td colspan="1" rowspan="1">OXA-1:29.3%(866/2960)</td><td colspan="1" rowspan="1">OXA-1:5.1% of19017</td></tr><tr><td colspan="1" rowspan="1">OXA-1</td><td colspan="1" rowspan="1">K. pneumoniae</td><td colspan="1" rowspan="1">95.7%(135/141)[91.03-98.04]</td><td colspan="1" rowspan="1">96.3%(181/188)[92.52-98.18]</td><td colspan="1" rowspan="1">Amox.K.Clav</td><td colspan="1" rowspan="1">OXA-1:37.4%(61/163)30.36-45.06]</td><td colspan="1" rowspan="1">OXA-1:86.4%(19/22)[66.67-95.25]</td><td colspan="1" rowspan="1">OXA-1:95.3%(61/64)[87.10-98.39]</td><td colspan="1" rowspan="1">OXA-1:15.7%(19/121)[10.29-23.23]</td><td colspan="1" rowspan="1">OXA-1:46.9%(0/1716)[44.56-49.28]</td><td colspan="1" rowspan="1">OXA-1:95.8%(299/312)[93.00-97.55]</td><td colspan="1" rowspan="1">OXA-1:98.4%(805/818)[97.30-99.07]</td><td colspan="1" rowspan="1">OXA-1:24.7%(299/1210)[22.36-27.22]</td><td colspan="1" rowspan="1">OXA-1:40.3%(818/2028)</td><td colspan="1" rowspan="1">OXA-1:21.7% of7979</td></tr><tr><td colspan="1" rowspan="1">OXA-1</td><td colspan="1" rowspan="1">K. pneumoniae</td><td colspan="1" rowspan="1">95.7%(135/141)91.03-98.04]</td><td colspan="1" rowspan="1">96.3%(181/188)[92.52-98.18]</td><td colspan="1" rowspan="1">Amp.Sulbactam</td><td colspan="1" rowspan="1">OXA-1:35.6%(63/177)[28.91-42.88]</td><td colspan="1" rowspan="1">OXA-1:87.5%(7/8)[52.91-97.76]</td><td colspan="1" rowspan="1">OXA-1:98.4%(63/64)91.67-99.72]</td><td colspan="1" rowspan="1">OXA-1:5.8%(7/121)[2.83-11.46]</td><td colspan="1" rowspan="1">OXA-1:42.7%(816/1909)[40.54-44.98]</td><td colspan="1" rowspan="1">OXA-1:98.3%(117/119)[94.08-99.54]</td><td colspan="1" rowspan="1">OXA-1:99.8%(816/818)[991-99.93]</td><td colspan="1" rowspan="1">OXA-1:9.7%(117/1210)[8.13-11.46]</td><td colspan="1" rowspan="1">OXA-1:40.3%(818/2028)</td><td colspan="1" rowspan="1">OXA-1:21.7% of7979</td></tr><tr><td colspan="1" rowspan="1">OXA-1</td><td colspan="1" rowspan="1">K. pneumoniae</td><td colspan="1" rowspan="1">95.7%(135/141)[91.03-98.04]</td><td colspan="1" rowspan="1">96.3%(181/188)[92.52-98.18]</td><td colspan="1" rowspan="1">Pip.Tazo</td><td colspan="1" rowspan="1">OXA-1:36.7%(54/147)29.37-44.77]</td><td colspan="1" rowspan="1">OXA-1:73.7%(28/38)57.9985.03]</td><td colspan="1" rowspan="1">OXA-1:84.4%(54/64)73.57-91.29]</td><td colspan="1" rowspan="1">OXA-1:23.1%(28/121)[16.52-31.41]</td><td colspan="1" rowspan="1">OXA-1:42.3%(592/1400)[39.72-44.891]</td><td colspan="1" rowspan="1">OXA-1:64.0%(402/628)[60.18-67.67]</td><td colspan="1" rowspan="1">OXA-1:72.4%(592/818)[69.21-75.33]</td><td colspan="1" rowspan="1">OXA-1:33.2%(402/1210)[30.63-35.93]</td><td colspan="1" rowspan="1">OXA-1:40.3%(818/2028)</td><td colspan="1" rowspan="1">OXA-1:21.7% of7979</td></tr><tr><td colspan="1" rowspan="2">AMR GeneTarget</td><td colspan="1" rowspan="2">Organism</td><td colspan="2" rowspan="1">Detection of AMR GeneTarget by AMR GenePanel versus WholeGenome SequencingCombined Clinical DevicePerformance -Stock and ProspectiveIsolates (from K191288)</td><td colspan="1" rowspan="2">Antimicrobial</td><td colspan="4" rowspan="1">Detection of AMR Gene Target by AMR GenePanel vs AST Non-SusceptibilityCombined Clinical Device Performance - Stock andonly unique isolates (no replicates)</td><td colspan="5" rowspan="1">Detection of AMR Gene Target versus AST Non-Susceptibility inOpGen's SMART Study</td><td colspan="1" rowspan="2">%Organismswith AMRGeneTarget inNCBIPathogenDatabase(2019)</td></tr><tr><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1">PPV[95% CI]</td><td colspan="1" rowspan="1">NPV[95% CI]</td><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1">PPV[95% CI]</td><td colspan="1" rowspan="1">NPV[95% CI]</td><td colspan="1" rowspan="1">% Organismswith AMRGene Target</td></tr><tr><td colspan="1" rowspan="1">OXA-1</td><td colspan="1" rowspan="1">P. mirabilis</td><td colspan="1" rowspan="1">100.0%(12/12)[75.75-100]</td><td colspan="1" rowspan="1">100.0%(143/143)[97.38-100]</td><td colspan="1" rowspan="1">Ampicillin</td><td colspan="1" rowspan="1">OXA-1:18.8%(12/64)[11.06-29.97]</td><td colspan="1" rowspan="1">OXA-1:100.0%(14/14)[78.47-10.00]</td><td colspan="1" rowspan="1">OXA-1:100.0%(12/12)75.75-100.00]</td><td colspan="1" rowspan="1">OXA-1:21.2%(14/66)[13.08-32.51]</td><td colspan="1" rowspan="1">OXA-1:8.4%(70/838)[6.66-10.42]</td><td colspan="1" rowspan="1">OXA-1:100.0%(319/319)[98.81-100]</td><td colspan="1" rowspan="1">OXA-1:100.0%(70/70)[94.80-0.00]</td><td colspan="1" rowspan="1">OXA-1:29.3%(319/1087)[26.72-32.12]</td><td colspan="1" rowspan="1">OXA-1: 6.1%(70/1157)</td><td colspan="1" rowspan="1">OXA-1:7.6% of144</td></tr><tr><td colspan="1" rowspan="1">OXA-1</td><td colspan="1" rowspan="1">P. mirabilis</td><td colspan="1" rowspan="1">100.0%(12/12)[75.75-100.00]</td><td colspan="1" rowspan="1">100.0%(143/143)[97.38-00]</td><td colspan="1" rowspan="1">Amox.K.Clav</td><td colspan="1" rowspan="1">OXA-1:26.9%(7/26)[13.70-46.08]</td><td colspan="1" rowspan="1">OXA-1:90.4%(47/52)[79.39-95.82]</td><td colspan="1" rowspan="1">OXA-1:58.3%(7/12)[31.95-80.67]</td><td colspan="1" rowspan="1">OXA-1:71.2%(47/66)59.36-80.73]</td><td colspan="1" rowspan="1">OXA-1:10.6%(38/359)[7.81-14.20]</td><td colspan="1" rowspan="1">OXA-1:96.0%(766/798)[94.39-97.15]</td><td colspan="1" rowspan="1">OXA-1:54.3%(38/70)[42.70-65.43]</td><td colspan="1" rowspan="1">OXA-1:70.5%(766/1087)[67.69-73.11]</td><td colspan="1" rowspan="1">OXA-1: 6.1%(70/1157)</td><td colspan="1" rowspan="1">OXA-1:7.6% of144</td></tr><tr><td colspan="1" rowspan="1">OXA-1</td><td colspan="1" rowspan="1">P. mirabilis</td><td colspan="1" rowspan="1">100.0%(12/12)[75.75-100.00]</td><td colspan="1" rowspan="1">100.0%(143/143)[97.38-100.00]</td><td colspan="1" rowspan="1">Amp.Sulbactam</td><td colspan="1" rowspan="1">OXA-1:26.2%(11/42)15.30-41.07]</td><td colspan="1" rowspan="1">OXA-1:97.2%(35/36)[85.839.51]</td><td colspan="1" rowspan="1">OXA-1:91.7%(11/12)6.61-8.51]</td><td colspan="1" rowspan="1">OXA-1:53.0%(35/66)4116-64.57]</td><td colspan="1" rowspan="1">OXA-1:11.3%(66/582).01-14.17]</td><td colspan="1" rowspan="1">OXA-1:999.3%(571/575)[98.23-99.73]</td><td colspan="1" rowspan="1">OXA-1:94.3%(66/70)86.21-97.76]</td><td colspan="1" rowspan="1">OXA-1:52.5%(571/1087) 9.5655.48]</td><td colspan="1" rowspan="1">OXA-1: 6.1%(70/1157)</td><td colspan="1" rowspan="1">OXA-1:7.6% of144</td></tr><tr><td colspan="1" rowspan="1">OXA-1</td><td colspan="1" rowspan="1">P. mirabilis</td><td colspan="1" rowspan="1">100.0%(12/12)75.75-100.00]</td><td colspan="1" rowspan="1">100.0%(143/143)[97.38-100.00]</td><td colspan="1" rowspan="1">Pip.Tazo</td><td colspan="1" rowspan="1">OXA-1:25.0%(1/4)[4.56-69.94]</td><td colspan="1" rowspan="1">OXA-1:85.1%(6/74)[75.31-91.49]</td><td colspan="1" rowspan="1">OXA-1:8.3%(1/12)[1.49-35.39]</td><td colspan="1" rowspan="1">OXA-1:95.5%(63/6)[87.47-98.44]</td><td colspan="1" rowspan="1">OXA-1:9.7%(3/31)[3.35-24.90]</td><td colspan="1" rowspan="1">OXA-1:94.0%(1059/1126) [92.51-95.29]</td><td colspan="1" rowspan="1">OXA-1:4.3%(3/70)[1.47-11.86]</td><td colspan="1" rowspan="1">OXA-1:97.4%(1059/1087) [96.30-98.21]</td><td colspan="1" rowspan="1">OXA-1: 6.1%(70/1157)</td><td colspan="1" rowspan="1">OXA-1:7.6% of144</td></tr><tr><td colspan="1" rowspan="1">OXA-1</td><td colspan="1" rowspan="1">P. aeruginosa</td><td colspan="1" rowspan="1">100.0%(9/9)[70.09-1000.00]</td><td colspan="1" rowspan="1">99.7%(338/339)[98.35-9.95]</td><td colspan="1" rowspan="1">Pip.Tazo</td><td colspan="1" rowspan="1">OXA-1:4.8%(5/104)[2.07-10.76]</td><td colspan="1" rowspan="1">OXA-1:100.0%((1/51)[93.00-100.00]</td><td colspan="1" rowspan="1">OXA-1:100.0%(5/5)[56.55-100.00]</td><td colspan="1" rowspan="1">OXA-1:34.0%(51/150)[26.90-41.90]</td><td colspan="1" rowspan="1">OXA-1:5.8%(55/941)[4.52-7.53]</td><td colspan="1" rowspan="1">OXA-1:97.5%(594/609)[95.98-98.50]</td><td colspan="1" rowspan="1">OXA-1:78.6%(55/70)67.61-86.56]</td><td colspan="1" rowspan="1">OXA-1:40.1%(594/1480)[37.67-42.65]</td><td colspan="1" rowspan="1">OXA-1: 4.5%(70/1550)</td><td colspan="1" rowspan="1">OXA-1:1.1% of4706</td></tr><tr><td colspan="1" rowspan="1">OXA-9</td><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">100.0%(14/14)[78.47-100.00]</td><td colspan="1" rowspan="1">99.7%(320/321)[98.26-99.94]</td><td colspan="1" rowspan="1">Cefazolin</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">OXA-9:1.2%(27/2209)[0.84-1.77]</td><td colspan="1" rowspan="1">OXA-9:99.5%(747/751)[98.64-99.79]</td><td colspan="1" rowspan="1">OXA-9:87.1%(27/31)[71.15-94.87]</td><td colspan="1" rowspan="1">OXA-9:25.5%(747/2929)[23.96-27.11]</td><td colspan="1" rowspan="1">OXA-9: 1%(31/2960)</td><td colspan="1" rowspan="1">OXA-9:0.2% of19017</td></tr><tr><td colspan="1" rowspan="1">OXA-9</td><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">100.0%(14/14)[78.47-100.00]</td><td colspan="1" rowspan="1">99.7%(320/321)[98.26-99.94]</td><td colspan="1" rowspan="1">Cefuroxime</td><td colspan="1" rowspan="1">OXA-9:8.8% (112/136)[5.12-14.79]</td><td colspan="1" rowspan="1">OXA-9:100.0%(23/23)[85.69-100.00]</td><td colspan="1" rowspan="1">OXA-9:100.0%(12/12)[75.75-100.00]</td><td colspan="1" rowspan="1">OXA-9:15.6%(23/147)[10.66-22.38]</td><td colspan="1" rowspan="1">OXA-9:1.2%(27/2187)[0.851.79]</td><td colspan="1" rowspan="1">OXA-9:999.5%(769/7)[98.68-99.80]</td><td colspan="1" rowspan="1">OXA-9:87.1%(27/31)[71.15-94.87]</td><td colspan="1" rowspan="1">OXA-9:26.3%(769/2929)[24.69-27.88]</td><td colspan="1" rowspan="1">OXA-9: 1%(/2960)</td><td colspan="1" rowspan="1">OXA-9:0.2% of19017</td></tr><tr><td colspan="1" rowspan="2">AMR GeneTarget</td><td colspan="1" rowspan="2">Organism</td><td colspan="2" rowspan="1">Detection of AMR GeneTarget by AMR GenePanel versus WholeGenome SequencingCombined Clinical DevicePerformance -Stock and ProspectiveIsolates (from K191288)</td><td colspan="1" rowspan="2">Antimicrobial</td><td colspan="4" rowspan="1">Detection of AMR Gene Target by AMR GenePanel vs AST Non-SusceptibilityCombined Clinical Device Performance - Stock andProspective Isolates (from K191288) includingonly unique isolates (no replicates)</td><td colspan="5" rowspan="1">Detection of AMR Gene Target versus AST Non-Susceptibility inOpGen's SMART Study</td><td colspan="1" rowspan="1">%Organismswith AMRGeneTarget inNCBIPathogen</td></tr><tr><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1">PPV[95% CI]</td><td colspan="1" rowspan="1">NPV[95% CI]</td><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1">PPV[95% CI]</td><td colspan="1" rowspan="1">NPV[95% CI]</td><td colspan="1" rowspan="1">% Organismswith AMRGene Target</td><td colspan="1" rowspan="1">Database(2019)</td></tr><tr><td colspan="1" rowspan="1">OXA-9</td><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">100.0%(14/14)[78.47-100.00]</td><td colspan="1" rowspan="1">99.7%(320/321)[98.26-99.94]</td><td colspan="1" rowspan="1">Cefepime</td><td colspan="1" rowspan="1">OXA-9:11.2%(12/107)[6.53-18.59]</td><td colspan="1" rowspan="1">OXA-9:100.0%(52/52)[93.12-100.00]</td><td colspan="1" rowspan="1">OXA-9:100.0%(12/12)[75.75-100.00]</td><td colspan="1" rowspan="1">OXA-9:35.4%(52/147)[28.11-43.39]</td><td colspan="1" rowspan="1">OXA-9:1.3%(27/2009)[0.93-1.95]</td><td colspan="1" rowspan="1">OXA-9:99.6%(947/951)[98.92-99.84]</td><td colspan="1" rowspan="1">OXA-9:87.1%(27/31)[71.15-94.87]</td><td colspan="1" rowspan="1">OXA-9:32.3%(947/2929)[30.66-34.05]</td><td colspan="1" rowspan="1">OXA-9: 1%(31/2960)</td><td colspan="1" rowspan="1">OXA-9:0.2% of19017</td></tr><tr><td colspan="1" rowspan="1">OXA-9</td><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">100.0%(14/14)[78.47-10.00]</td><td colspan="1" rowspan="1">99.7%(320/321)[98.26-99.94]</td><td colspan="1" rowspan="1">Ceftazidime</td><td colspan="1" rowspan="1">OXA-9:11.3%2/106)[6.60-18.75]</td><td colspan="1" rowspan="1">OXA-9:100.0%(53/53)[93.24-0.00]</td><td colspan="1" rowspan="1">OXA-9:100.0%(12/12)[75.75-0.00]</td><td colspan="1" rowspan="1">OXA-9:36.1%(553/147)[28.74-44.08]</td><td colspan="1" rowspan="1">OXA-9:1.6%(26/1651)[1.08-2.30]</td><td colspan="1" rowspan="1">OXA-9:99.6%(1304/1309) [99.11-99.84]</td><td colspan="1" rowspan="1">OXA-9:83.9%(26/31)[67.37-92.91]</td><td colspan="1" rowspan="1">OXA-9:44.5%(1304/2929) [42.73-46.33]</td><td colspan="1" rowspan="1">OXA-9: 1%(31/2960)</td><td colspan="1" rowspan="1">OXA-9:0.2% of19017</td></tr><tr><td colspan="1" rowspan="1">OXA-9</td><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">100.0%(14/14)[78.47-100.00]</td><td colspan="1" rowspan="1">99.7%(320/321)[98.26-99.94]</td><td colspan="1" rowspan="1">Ceftriaxone</td><td colspan="1" rowspan="1">OXA-9:9.2%(12/130)[5.36-15.44]</td><td colspan="1" rowspan="1">OXA-9:100.0%(29/29)8830-100.00]</td><td colspan="1" rowspan="1">OXA-9:100.0%(12/12)75.75-100.00]</td><td colspan="1" rowspan="1">OXA-9:19.7%(29/147)[14.10-26.90]</td><td colspan="1" rowspan="1">OXA-9:1.3%(27/2113)[0.88-1.85]</td><td colspan="1" rowspan="1">OXA-9:999.5%(843/847)[98.79-99.82]</td><td colspan="1" rowspan="1">OXA-9:87.1%(27/31)71.15-94.87]</td><td colspan="1" rowspan="1">OXA-9:28.8%(843/2929)[27.17-30.45]</td><td colspan="1" rowspan="1">OXA-9: 1%(31/2960)</td><td colspan="1" rowspan="1">OXA-9:0.2% of19017</td></tr><tr><td colspan="1" rowspan="1">OXA-9</td><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">100.0%(14/14)[78.47-100.00]</td><td colspan="1" rowspan="1">99.7%(320/321)[98.26-99.94]</td><td colspan="1" rowspan="1">Amox.K.Clav</td><td colspan="1" rowspan="1">OXA-9:10.5%(12/114)[6.12-17.50]</td><td colspan="1" rowspan="1">OXA-9:100.0%(45/45)[92.13-100.00]</td><td colspan="1" rowspan="1">OXA-9:100.0%(12/12)[75.75-100.00]</td><td colspan="1" rowspan="1">OXA-9:30.6%(45/147)[23.73-38.48]</td><td colspan="1" rowspan="1">OXA-9:1.5%(28810)[1.07-2.23]</td><td colspan="1" rowspan="1">OXA-9:99.7%(1147/1150) [99.24-99.91]</td><td colspan="1" rowspan="1">OXA-9:90.3%(28/31)[75.10-96.65]</td><td colspan="1" rowspan="1">OXA-9:39.2%(1147/2929) [37.41-40.94]</td><td colspan="1" rowspan="1">OXA-9: 1%(31/2960)</td><td colspan="1" rowspan="1">OXA-9:0.2% of19017</td></tr><tr><td colspan="1" rowspan="1">OXA-9</td><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">100.0%(14/14)[78.47-100.00]</td><td colspan="1" rowspan="1">99.7%(320/321)[98.26-99.94]</td><td colspan="1" rowspan="1">Amp.Sulbactam</td><td colspan="1" rowspan="1">OXA-9:8.2%(12/146)[4.76-13.82]</td><td colspan="1" rowspan="1">OXA-9:100.0%(13/13)[77.19-100.00]</td><td colspan="1" rowspan="1">OXA-9:100.0%(12/12)75.75-100.00]</td><td colspan="1" rowspan="1">OXA-9:8.8%(13/147)[5.24-14.54]</td><td colspan="1" rowspan="1">OXA-9:1.2%(30/2477)[0.85-1.72]</td><td colspan="1" rowspan="1">OXA-9:99.8%(482/483)[98.84-99.96]</td><td colspan="1" rowspan="1">OXA-9:96.8%(30/31)83.8199.43]</td><td colspan="1" rowspan="1">OXA-9:16.5%(482/2929)[15.16-17.84]</td><td colspan="1" rowspan="1">OXA-9: 1%(31/260)</td><td colspan="1" rowspan="1">OXA-9:0.2% of19017</td></tr><tr><td colspan="1" rowspan="1">OXA-9</td><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">100.0%(14/14)[78.47-100.00]</td><td colspan="1" rowspan="1">99.7%(320/321)[98.26-99.94]</td><td colspan="1" rowspan="1">Pip.Tazo</td><td colspan="1" rowspan="1">OXA-9:13.0%(9/69)[7.02-22.97]</td><td colspan="1" rowspan="1">OXA-9:96.7%(87/90)[90.65-98.86]</td><td colspan="1" rowspan="1">OXA-9:75.0%(9/12)[46.77-91.11]</td><td colspan="1" rowspan="1">OXA-9:59.2%(87/147)[51.10-66.80]</td><td colspan="1" rowspan="1">OXA-9:3.3%(23/687)[2.24-4.97]</td><td colspan="1" rowspan="1">OXA-9:99.6%(2265/2273) [99.31-99.82]</td><td colspan="1" rowspan="1">OXA-9:74.2%(23/31)[56.75-86.30]</td><td colspan="1" rowspan="1">OXA-9:77.3%(2265/2929) [75.78-78.81]</td><td colspan="1" rowspan="1">OXA-9: 1%(31/2960)</td><td colspan="1" rowspan="1">OXA-9:0.2% of19017</td></tr><tr><td colspan="1" rowspan="1">OXA-9</td><td colspan="1" rowspan="1">K. pneumoniae</td><td colspan="1" rowspan="1">100.0%(31/31)[88.97-0.00]</td><td colspan="1" rowspan="1">100.0%(298/298)[98.73-100.00]</td><td colspan="1" rowspan="1">Cefazolin</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">OXA-9:20.2%(369/1824)[18.45-22.14]</td><td colspan="1" rowspan="1">OXA-9:96.6%(197/204)[93.09-988.33]</td><td colspan="1" rowspan="1">OXA-9:98.1%(369/376)[96.21-99.10]</td><td colspan="1" rowspan="1">OXA-9:11.9%(197/1652)[10.45-13.58]</td><td colspan="1" rowspan="1">OXA-9:18.5%(376/2028)</td><td colspan="1" rowspan="1">OXA-9:9.8% of7979</td></tr><tr><td colspan="1" rowspan="2">AMR GeneTarget</td><td colspan="1" rowspan="2">Organism</td><td colspan="2" rowspan="1">Detection of AMR GeneTarget by AMR GenePanel versus WholeGenome SequencingCombined Clinical DevicePerformance -Stock and ProspectiveIsolates (from K191288)</td><td colspan="1" rowspan="1">Antimicrobial</td><td colspan="4" rowspan="1">Detection of AMR Gene Target by AMR GenePanel vs AST Non-SusceptibilityCombined Clinical Device Performance - Stock andProspective Isolates (from K191288) includingonly unique isolates (no replicates)</td><td colspan="5" rowspan="1">Detection of AMR Gene Target versus AST Non-Susceptibility inOpGen's SMART Study</td><td colspan="1" rowspan="2">%Organismswith AMRGeneTarget inNCBIPathogenDatabase(2019)</td></tr><tr><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1">PPV[95% CI]</td><td colspan="1" rowspan="1">NPV[95% CI]</td><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1">PPV[95% CI]</td><td colspan="1" rowspan="1">NPV[95% CI]</td><td colspan="1" rowspan="1">% Organismswith AMRGene Target</td></tr><tr><td colspan="1" rowspan="1">OXA-9</td><td colspan="1" rowspan="1">K. pneumoniae</td><td colspan="1" rowspan="1">100.0%(31/31)[88.97-0.00]</td><td colspan="1" rowspan="1">100.0% 998/298)[98.73-0.00]</td><td colspan="1" rowspan="1">Cefuroxime</td><td colspan="1" rowspan="1">OXA-9:14.7%(25/170)[10.16-20.1]</td><td colspan="1" rowspan="1">OXA-9:100.0%(15/15)[79.61-100.00]</td><td colspan="1" rowspan="1">OXA-9:100.0%(25/25)[86.68-100.00]</td><td colspan="1" rowspan="1">OXA-9:9.4%(15/160)[5.76-14.89]</td><td colspan="1" rowspan="1">OXA-9:20.1%(368/1832)[18.32-21.98]</td><td colspan="1" rowspan="1">OXA-9:95.9%(188/196)[92.15-97.92]</td><td colspan="1" rowspan="1">OXA-9:97.9%(368/376)[95.86-98.92]</td><td colspan="1" rowspan="1">OXA-9:11.4%(188/1652)[9.94-13.00]</td><td colspan="1" rowspan="1">OXA-9:18.5%(376/2028)</td><td colspan="1" rowspan="1">OXA-9:9.8% of7979</td></tr><tr><td colspan="1" rowspan="1">OXA-9</td><td colspan="1" rowspan="1">K. pneumoniae</td><td colspan="1" rowspan="1">100.0%(31/31)[88.97-00]</td><td colspan="1" rowspan="1">100.0%(298/298)[98.73-10.00]</td><td colspan="1" rowspan="1">Cefepime</td><td colspan="1" rowspan="1">OXA-9:15.6%(25/160)[10.81-22.05]</td><td colspan="1" rowspan="1">OXA-9:100.0%(25/25)[86.68-10.00]</td><td colspan="1" rowspan="1">OXA-9:100.0%(25/25)[86.68-0.00]</td><td colspan="1" rowspan="1">OXA-9:15.6%(25/160)[10.81-22.05]</td><td colspan="1" rowspan="1">OXA-9:21.3%(365/1711)[19.46-23.34]</td><td colspan="1" rowspan="1">OXA-9:96.5%(306/317)[93.89-98.05]</td><td colspan="1" rowspan="1">OXA-9:97.1%(365/376)[94.84-98.36]</td><td colspan="1" rowspan="1">OXA-9:18.5%(306/1652)[16.72-20.47]</td><td colspan="1" rowspan="1">OXA-9:18.5%(376/2028)</td><td colspan="1" rowspan="1">OXA-9:9.8% of7979</td></tr><tr><td colspan="1" rowspan="1">OXA-9</td><td colspan="1" rowspan="1">K. pneumoniae</td><td colspan="1" rowspan="1">100.0%(31/31)[88.97-10.00]</td><td colspan="1" rowspan="1">100.0%(298/298)[98.73-100.00]</td><td colspan="1" rowspan="1">Ceftazidime</td><td colspan="1" rowspan="1">OXA-9:15.3%(25/163)10.61-21.66]</td><td colspan="1" rowspan="1">OXA-9:100.0%(22/22)[85.13-0.00]</td><td colspan="1" rowspan="1">OXA-9:100.0%(25/25)[86.68-100.00]</td><td colspan="1" rowspan="1">OXA-9:13.8%(22/160)[9.26-19.94]</td><td colspan="1" rowspan="1">OXA-9:21.3%(365/1710)[19.47-23.35]</td><td colspan="1" rowspan="1">OXA-9:96.5%(307/318)93.1-98.06]</td><td colspan="1" rowspan="1">OXA-9:97.1%(365/376)[94.84-98.36]</td><td colspan="1" rowspan="1">OXA-9:18.6%(307/1652)16.78-20.53]</td><td colspan="1" rowspan="1">OXA-9:18.5%(376/2028)</td><td colspan="1" rowspan="1">OXA-9:9.8% of7979</td></tr><tr><td colspan="1" rowspan="1">OXA-9</td><td colspan="1" rowspan="1">K. pneumoniae</td><td colspan="1" rowspan="1">100.0%(31/31)[88.97-100.00]</td><td colspan="1" rowspan="1">100.0%(298/298)[98.73-100.00]</td><td colspan="1" rowspan="1">Ceftriaxone</td><td colspan="1" rowspan="1">OXA-9:15.0%(25/167)[10.35-21.17]</td><td colspan="1" rowspan="1">OXA-9:100.0%(18/18)[82.41-100.00]</td><td colspan="1" rowspan="1">OXA-9:100.0%(25/25)[86.68-100.00]</td><td colspan="1" rowspan="1">OXA-9:11.2%(18/160)[7.24-17.08]</td><td colspan="1" rowspan="1">OXA-9:20.8%(367/1765)[18.96-22.75]</td><td colspan="1" rowspan="1">OXA-9:96.6%(254/263)[93.63-98.19]</td><td colspan="1" rowspan="1">OXA-9:97.6%(367/376)[95.51-98.74]</td><td colspan="1" rowspan="1">OXA-9:15.4%(254/1652)[13.72-17.19]</td><td colspan="1" rowspan="1">OXA-9:18.5%(376/2028)</td><td colspan="1" rowspan="1">OXA-9:9.8% of7979</td></tr><tr><td colspan="1" rowspan="1">OXA-9</td><td colspan="1" rowspan="1">K. pneumoniae</td><td colspan="1" rowspan="1">100.0%(31/31)[88.97-0.00]</td><td colspan="1" rowspan="1">100.0%(298/298)[98.73-0.00]</td><td colspan="1" rowspan="1">Amox.K.Clav</td><td colspan="1" rowspan="1">OXA-9:14.1%(23/163)[9.59-20.28]</td><td colspan="1" rowspan="1">OXA-9:90.9%(20/22)[72.19-97.47]</td><td colspan="1" rowspan="1">OXA-9:92.0%(23/25)[75.03-97.78]</td><td colspan="1" rowspan="1">OXA-9:12.5%(20/160)[8.24-18.52]</td><td colspan="1" rowspan="1">OXA-9:21.5%(369/1716)[19.62-23.51]</td><td colspan="1" rowspan="1">OXA-9:97.8%(305/312)[95.44-98.91]</td><td colspan="1" rowspan="1">OXA-9:98.1%(369/376)[96.21-999.10]</td><td colspan="1" rowspan="1">OXA-9:18.5%(305/1652)[16.67-20.41]</td><td colspan="1" rowspan="1">OXA-9:18.5%(376/2028)</td><td colspan="1" rowspan="1">OXA-9:9.8% of7979</td></tr><tr><td colspan="1" rowspan="1">OXA-9</td><td colspan="1" rowspan="1">K. pneumoniae</td><td colspan="1" rowspan="1">100.0%(31/31)[88.97-00]</td><td colspan="1" rowspan="1">100.0%(298/298)[98.73-100.00]</td><td colspan="1" rowspan="1">Amp.Sulbactam</td><td colspan="1" rowspan="1">OXA-9:14.1%(2/17)[9.75-20.02]</td><td colspan="1" rowspan="1">OXA-9:100.0%(8/8)[67.56-100]</td><td colspan="1" rowspan="1">OXA-9:100.0%(25/25)[86.68-0.00]</td><td colspan="1" rowspan="1">OXA-9:5.0%(8/160)[2.56-9.56]</td><td colspan="1" rowspan="1">OXA-9:19.7%(376/1909)[17.97-21.54]</td><td colspan="1" rowspan="1">OXA-9:100.0%(119/119)[96.87-0.00]</td><td colspan="1" rowspan="1">OXA-9:100.0%(376/376)[98.99- 100.00]</td><td colspan="1" rowspan="1">OXA-9:7.2%(119/1652)[6.05-8.55]</td><td colspan="1" rowspan="1">OXA-9:18.5%(376/2028)</td><td colspan="1" rowspan="1">OXA-9:9.8% of7979</td></tr><tr><td colspan="1" rowspan="1">OXA-9</td><td colspan="1" rowspan="1">K. pneumoniae</td><td colspan="1" rowspan="1">100.0%(31/31)[88.97-100.00]</td><td colspan="1" rowspan="1">100.0%(298/298)[98.73-1000]</td><td colspan="1" rowspan="1">Pip.Tazo</td><td colspan="1" rowspan="1">OXA-9:15.6%(23/147)10.66-22.38]</td><td colspan="1" rowspan="1">OXA-9:94.7%(36/38)[82.71-98.54]</td><td colspan="1" rowspan="1">OXA-9:92.0%(23/25)75.03-97.78]</td><td colspan="1" rowspan="1">OXA-9:22.5%(36/160)16.72-29.57]</td><td colspan="1" rowspan="1">OXA-9:24..9%(348/1400)[22.66-27.19]</td><td colspan="1" rowspan="1">OXA-9:95.5%(600/628)[93.63-96.901]</td><td colspan="1" rowspan="1">OXA-9:92.6%(348/376)[89.45-94.80]</td><td colspan="1" rowspan="1">OXA-9:36.3%(600/1652)[34.03-38.67]</td><td colspan="1" rowspan="1">OXA-9:18.5%(376/2028)</td><td colspan="1" rowspan="1">OXA-9:9.8% of7979</td></tr><tr><td colspan="1" rowspan="2">AMR GeneTarget</td><td colspan="1" rowspan="2">Organism</td><td colspan="2" rowspan="1">Detection of AMR GeneTarget by AMR GenePanel versus WholeGenome SequencingCombined Clinical DevicePerformance-Stock and ProspectiveIsolates (from K191288)</td><td colspan="1" rowspan="2">Antimicrobial</td><td colspan="4" rowspan="1">Detection of AMR Gene Target by AMR GenePanel vs AST Non-SusceptibilityCombined Clinical Device Performance - Stock andProspective Isolates (from K191288) includingonly unique isolates (no replicates)</td><td colspan="5" rowspan="1">Detection of AMR Gene Target versus AST Non-Susceptibility inOpGen's SMART Study</td><td colspan="1" rowspan="2">%Organismswith AMRGeneTarget inNCBIPathogenDatabase2019)</td></tr><tr><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1">PPV[95% CI]</td><td colspan="1" rowspan="1">NPV[95% CI]</td><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1">PPV[95% CI]</td><td colspan="1" rowspan="1">NPV[95% CI]</td><td colspan="1" rowspan="1">% Organismswith AMRGene Target</td></tr><tr><td colspan="1" rowspan="1">OXA-9</td><td colspan="1" rowspan="1">P. mirabilis</td><td colspan="1" rowspan="1">100.0%(13/13)[77.19-100.00]</td><td colspan="1" rowspan="1">100.0%(142/142)[97.37-1000]</td><td colspan="1" rowspan="1">Cefazolin</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">OXA-9:8.2%(47/571)[6.25-10.77]</td><td colspan="1" rowspan="1">OXA-9:99.5%(583/586)[98.51-99.83]</td><td colspan="1" rowspan="1">OXA-9:94.0%(47/50)[83.78-97.94]</td><td colspan="1" rowspan="1">OXA-9:52.7%(583/1107)[49.72-55.59]</td><td colspan="1" rowspan="1">OXA-9: 4.3%(50/1157)</td><td colspan="1" rowspan="1">OXA-9: 0%of 144</td></tr><tr><td colspan="1" rowspan="1">OXA-9</td><td colspan="1" rowspan="1">P. mirabilis</td><td colspan="1" rowspan="1">100.0%(13/13)77.19-10.00]</td><td colspan="1" rowspan="1">100.0%(142/142)[97.37-100.00]</td><td colspan="1" rowspan="1">Cefuroxime</td><td colspan="1" rowspan="1">OXA-9:24.4%1/45)[14.24-38.67]</td><td colspan="1" rowspan="1">OXA-9:100.0%(33/33)89.57-0.00]</td><td colspan="1" rowspan="1">OXA-9:100.0%(11/11)[74.12-100.00]</td><td colspan="1" rowspan="1">OXA-9:49.3%(3/67)[37.65-60.94]</td><td colspan="1" rowspan="1">OXA-9:10.1%(50/495)[7.75-13.07]</td><td colspan="1" rowspan="1">OXA-9:100.0%(662/662)[99.42-10.00]</td><td colspan="1" rowspan="1">OXA-9:100.0%(50/50)[92.87-00]</td><td colspan="1" rowspan="1">OXA-9:59.8%(662/1107)[56.88-62.65]</td><td colspan="1" rowspan="1">OXA-9: 4.3%(50/1157)</td><td colspan="1" rowspan="1">OXA-9: 0%of 144</td></tr><tr><td colspan="1" rowspan="1">OXA-9</td><td colspan="1" rowspan="1">P. mirabilis</td><td colspan="1" rowspan="1">100.0%(13/13)[77.19-100.00]</td><td colspan="1" rowspan="1">100.0%(142/142)[97.37-100.00]</td><td colspan="1" rowspan="1">Cefepime</td><td colspan="1" rowspan="1">OXA-9:27.5%(11/40)16.1-42.83]</td><td colspan="1" rowspan="1">OXA-9:100.0%(38/38)[90.82-100.00]</td><td colspan="1" rowspan="1">OXA-9:100.0%(11/11)[74.12-100.00]</td><td colspan="1" rowspan="1">OXA-9:56.7%(38/67)44.81-67.90]</td><td colspan="1" rowspan="1">OXA-9:13.8%(46/334)[10.49-17.88]</td><td colspan="1" rowspan="1">OXA-9:999.5%(819/823)[98.76-99.81]</td><td colspan="1" rowspan="1">OXA-9:92.0%(46/50)[81.16-96.85]</td><td colspan="1" rowspan="1">OXA-9:74.0%(819/1107)[71.32-76.48]</td><td colspan="1" rowspan="1">OXA-9: 4.3%(50/1157)</td><td colspan="1" rowspan="1">OXA-9: 0%o f 144</td></tr><tr><td colspan="1" rowspan="1">OXA-9</td><td colspan="1" rowspan="1">P. mirabilis</td><td colspan="1" rowspan="1">100.0%(13/13)[77.19-0.00]</td><td colspan="1" rowspan="1">100.0%(142/142)[97.37-0.00]</td><td colspan="1" rowspan="1">Ceftazidime</td><td colspan="1" rowspan="1">OXA-9:30.4%(7/23)[15.60-50.87]</td><td colspan="1" rowspan="1">OXA-9:92.7%(1/25)[82.74-97.14]</td><td colspan="1" rowspan="1">OXA-9:63.6%(7/1)[35.38-84.83]</td><td colspan="1" rowspan="1">OXA-9:76.1%(51/67)[64.67-84.73]</td><td colspan="1" rowspan="1">OXA-9:9.3%(24/259)[6.31-13.42]</td><td colspan="1" rowspan="1">OXA-9:97.1%(872/898)[95.79-98.02]</td><td colspan="1" rowspan="1">OXA-9:48.0%(24/50)[34.80-6.49]</td><td colspan="1" rowspan="1">OXA-9:78.8%(872/1107)[76.27-81.08]</td><td colspan="1" rowspan="1">OXA-9: 4.3%(50/1157)</td><td colspan="1" rowspan="1">OXA-9: 0%of 144</td></tr><tr><td colspan="1" rowspan="1">OXA-9</td><td colspan="1" rowspan="1">P. mirabilis</td><td colspan="1" rowspan="1">100.0%(13/13)[77.19-100.00]</td><td colspan="1" rowspan="1">100.0%(142/142)[97.37-100.00]</td><td colspan="1" rowspan="1">Ceftriaxone</td><td colspan="1" rowspan="1">OXA-9:26.2%(11/42)15.30-41.07]</td><td colspan="1" rowspan="1">OXA-9:100.0%(36/36)90.36-100.00]</td><td colspan="1" rowspan="1">OXA-9:100.0%(1/1)[74.12-100.00]</td><td colspan="1" rowspan="1">OXA-9:53.7%(36/67)41.265.14]</td><td colspan="1" rowspan="1">OXA-9:10.0%(48/478)[7.66-13.06]</td><td colspan="1" rowspan="1">OXA-9:99.7%(677/679)[98.93-99.92]</td><td colspan="1" rowspan="1">OXA-9:96.0%(48/50)[86.54-98.90]</td><td colspan="1" rowspan="1">OXA-9:61.2%(677/1107)[58.25-63.98]</td><td colspan="1" rowspan="1">OXA-9: 4.3%0/157)</td><td colspan="1" rowspan="1">OXA-9: 0% f 144</td></tr><tr><td colspan="1" rowspan="1">OXA-9</td><td colspan="1" rowspan="1">P. mirabilis</td><td colspan="1" rowspan="1">100.0%(13/13)[77.19-100.00]</td><td colspan="1" rowspan="1">100.0%(142/142)[97.37-100.00]</td><td colspan="1" rowspan="1">Amox.K.Clav</td><td colspan="1" rowspan="1">OXA-9:15.4%(4/26)[6.15-33.53]</td><td colspan="1" rowspan="1">OXA-9:86.5%(45/52)[74.73-93.32]</td><td colspan="1" rowspan="1">OXA-9:36.4%(4/11)[15.17-64.62]</td><td colspan="1" rowspan="1">OXA-9:67.2%(45/67)[55.26-77.21]</td><td colspan="1" rowspan="1">OXA-9:4.5%(16/359)[2.76-7.12]</td><td colspan="1" rowspan="1">OXA-9:95.7%(764/798)[94.11-96.94]</td><td colspan="1" rowspan="1">OXA-9:32.0%(16/50)[20.76-45.81]</td><td colspan="1" rowspan="1">OXA-9:69.0%(764/1107)[66.23-71.67]</td><td colspan="1" rowspan="1">OXA-9: 4.3%(50/1157)</td><td colspan="1" rowspan="1">OXA-9: 0%of 144</td></tr><tr><td colspan="1" rowspan="1">OXA-9</td><td colspan="1" rowspan="1">P. mirabilis</td><td colspan="1" rowspan="1">100.0%(13/13)[77.19-100.00]</td><td colspan="1" rowspan="1">100.0%(142/142)[97.37-00]</td><td colspan="1" rowspan="1">Amp.Sulbactam</td><td colspan="1" rowspan="1">OXA-9:19.0%(8/42)[9.98-33.30]</td><td colspan="1" rowspan="1">OXA-9:91.7%(33/36)[78.17-97.13]</td><td colspan="1" rowspan="1">OXA-9:72.7%(8/11)[43.44-90.25]</td><td colspan="1" rowspan="1">OXA-9:49.3%(33/67)37.65-60.94]</td><td colspan="1" rowspan="1">OXA-9:7.6%(44/582)[5.68-10.00]</td><td colspan="1" rowspan="1">OXA-9:99.0%(569/575)[97.74-99.52]</td><td colspan="1" rowspan="1">OXA-9:88.0%(444/50)[76.20-94.38]</td><td colspan="1" rowspan="1">OXA-9:51.4%(569/1107)[48.46-54.33]</td><td colspan="1" rowspan="1">OXA-9: 4.3%(50/1157)</td><td colspan="1" rowspan="1">OXA-9: 0%of 144</td></tr><tr><td colspan="1" rowspan="2">AMR GeneTarget</td><td colspan="1" rowspan="2">Organism</td><td colspan="2" rowspan="1">Detection of AMR GeneTarget by AMR GenePanel versus WholeGenome SequencingCombined Clinical DevicePerformance -Stock and ProspectiveIsolates (from K191288)</td><td colspan="1" rowspan="2">Antimicrobial</td><td colspan="4" rowspan="1">Detection of AMR Gene Target by AMR GenePanel vs AST Non-SusceptibilityCombined Clinical Device Performance - Stock andProspective Isolates (from K191288) includingonly unique isolates (no replicates)</td><td colspan="5" rowspan="1">Detection of AMR Gene Target versus AST Non-Susceptibility inOpGen's SMART Study</td><td colspan="1" rowspan="2">%Organismswith AMRGeneTarget inNCBIPathogenDatabase(2019)</td></tr><tr><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1">PPV[95% CI]</td><td colspan="1" rowspan="1">NPV[95% CI]</td><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1">PPV[95% CI]</td><td colspan="1" rowspan="1">NPV[95% CI]</td><td colspan="1" rowspan="1">% Organismswith AMRGene Target</td></tr><tr><td colspan="1" rowspan="1">OXA-9</td><td colspan="1" rowspan="1">P. mirabilis</td><td colspan="1" rowspan="1">100.0%(13/13)[77.19-10.00]</td><td colspan="1" rowspan="1">100.0%(142/142)[97.37-100.00]</td><td colspan="1" rowspan="1">Pip.Tazo</td><td colspan="1" rowspan="1">OXA-9:25.0%(1/4)[4.56-69.94]</td><td colspan="1" rowspan="1">OXA-9:86.5%(64/74)76.88-92.49]</td><td colspan="1" rowspan="1">OXA-9:9.1%(1/11)[1.62-37.74]</td><td colspan="1" rowspan="1">OXA-9:95.5%(64/67)[87.64-98.47]</td><td colspan="1" rowspan="1">OXA-9:6.5%(2/31)[1.79-20.72]</td><td colspan="1" rowspan="1">OXA-9:95.7%(1078/112) (94.39-96.77]</td><td colspan="1" rowspan="1">OXA-9:4.0%(2/50)[1.10-13.46]</td><td colspan="1" rowspan="1">OXA-9:97.4%(1078/1107) (96.26-98.17]</td><td colspan="1" rowspan="1">OXA-9: 4.3%(50/1157)</td><td colspan="1" rowspan="1">OXA-9: 0%of 144</td></tr><tr><td colspan="1" rowspan="1">OXA-48</td><td colspan="1" rowspan="1">K. pneumoniae</td><td colspan="1" rowspan="1">95.1%(58/61)86.51-98.31]</td><td colspan="1" rowspan="1">99.3%(266/268)[97.32-99.80]</td><td colspan="1" rowspan="1">Imipenem</td><td colspan="1" rowspan="1">OXA-48:34.9%(37/106)[26.50-44.36]</td><td colspan="1" rowspan="1">OXA-48:88.6%(70/9)[79.75-93.89]</td><td colspan="1" rowspan="1">OXA-48:80.4%(37/46)[66.83-89..35]</td><td colspan="1" rowspan="1">OXA-48:50.4%(70/139)[42.15-58.55]</td><td colspan="1" rowspan="1">OXA-48:27.0%1/818)[24.09-30.16]</td><td colspan="1" rowspan="1">OXA-48:93.1%(1126/1210) [91.49-94.36]</td><td colspan="1" rowspan="1">OXA-48:72.5%(221/305)[67.19-77.17]</td><td colspan="1" rowspan="1">OXA-48:65.4%(1126/1723) [63.07-66.56]</td><td colspan="1" rowspan="1">OXA-48: 15%(305/2028)</td><td colspan="1" rowspan="1">OXA-48:10.4% of7979</td></tr><tr><td colspan="1" rowspan="1">OXA-48</td><td colspan="1" rowspan="1">K. pneumoniae</td><td colspan="1" rowspan="1">95.1%(58/61)86.51-98.31]</td><td colspan="1" rowspan="1">99.3%(266/268)[97.32-99.80]</td><td colspan="1" rowspan="1">Ertapenem</td><td colspan="1" rowspan="1">OXA-48:34.9%(44/126)[27.15-43.58]</td><td colspan="1" rowspan="1">OXA-48:96.6%(57/59)88.699.07]</td><td colspan="1" rowspan="1">OXA-48:95.7%(4/46)[85.47-98.80]</td><td colspan="1" rowspan="1">OXA-48:41.0%(57/139)[33.18-49.32]</td><td colspan="1" rowspan="1">OXA-48:27.8%(293/1053)25.20-30.61]</td><td colspan="1" rowspan="1">OXA-48:98.8%(963/975)[97.86-99.29]</td><td colspan="1" rowspan="1">OXA-48:96.1%(293/305)[93.25-97.74]</td><td colspan="1" rowspan="1">OXA-48:55.9%(963/1723)[53.54-58.22]</td><td colspan="1" rowspan="1">OXA-48: 15%(305/2028)</td><td colspan="1" rowspan="1">OXA-48:10.4% of7979</td></tr><tr><td colspan="1" rowspan="1">OXA-48</td><td colspan="1" rowspan="1">K. pneumoniae</td><td colspan="1" rowspan="1">95.1%(58/61)86.51-98.31]</td><td colspan="1" rowspan="1">99.3%(266/268)[97.32-99.80]</td><td colspan="1" rowspan="1">Meropenem</td><td colspan="1" rowspan="1">OXA-48:37.4%(37/99)[28.48-47.21]</td><td colspan="1" rowspan="1">OXA-48:89.5%(77/86)[81.29-94.40]</td><td colspan="1" rowspan="1">OXA-48:80.4%(37/46)66.83-89.35</td><td colspan="1" rowspan="1">OXA-48:55.4%(77/139)[47.10-63.40]</td><td colspan="1" rowspan="1">OXA-48:28.6%(241/842)[25.67-31.77]</td><td colspan="1" rowspan="1">OXA-48:94.6%(1122/1186) [93.17-95.75]</td><td colspan="1" rowspan="1">OXA-48:79.0%(241/305)[74.10-83.21]</td><td colspan="1" rowspan="1">OXA-48:65.1%(1122/1723) [62.84-67.33]</td><td colspan="1" rowspan="1">OXA-48: 15%(305/2028)</td><td colspan="1" rowspan="1">OXA-48:10.4% of7979</td></tr><tr><td colspan="1" rowspan="1">OXA-48</td><td colspan="1" rowspan="1">P. mirabilis</td><td colspan="1" rowspan="1">NaN</td><td colspan="1" rowspan="1">98.7%(153/155)[95.42-99.65]</td><td colspan="1" rowspan="1">Ertapenem</td><td colspan="1" rowspan="1">OXA-48:0.0%(0/4)[0.00-48.99]</td><td colspan="1" rowspan="1">OXA-48:100.0%(74/74)[95.07-100.00]</td><td colspan="1" rowspan="1">OXA-48:NaN</td><td colspan="1" rowspan="1">OXA-48:94.9%(74/78)[87.54-97.99]</td><td colspan="1" rowspan="1">OXA-48:0.0%(0/16)[0.00-19.36]</td><td colspan="1" rowspan="1">OXA-48:99.8%(1139/1141) [99.36-99.95]</td><td colspan="1" rowspan="1">OXA-48:0.0% (0/2)[0.00-665.76]</td><td colspan="1" rowspan="1">OXA-48:98.6%(1139/155) [97.76-99.15]</td><td colspan="1" rowspan="1">OXA-48:0.2%(2/1157)</td><td colspan="1" rowspan="1">OXA-48:0.7% of144</td></tr><tr><td colspan="1" rowspan="1">OXA-48</td><td colspan="1" rowspan="1">P. mirabilis</td><td colspan="1" rowspan="1">NaN</td><td colspan="1" rowspan="1">98.7%(153/155)[95.42-99.65]</td><td colspan="1" rowspan="1">Meropenem</td><td colspan="1" rowspan="1">OXA-48:0.0%((0/3)[0.00-56.15]</td><td colspan="1" rowspan="1">OXA-48:100.0%((/75)[95.13-100.00]</td><td colspan="1" rowspan="1">OXA-48:NaN</td><td colspan="1" rowspan="1">OXA-48:96.2%(75/78)[89.29-98.68]</td><td colspan="1" rowspan="1">OXA-48:0.0%(/17)[0.00-18.43]</td><td colspan="1" rowspan="1">OXA-48:99.8%(1138/1140) [99.36-99.951]</td><td colspan="1" rowspan="1">OXA-48:0.0% (0/2)[0.00-65.76]</td><td colspan="1" rowspan="1">OXA-48:98.5%(1138/1155) [97.66-99.08]</td><td colspan="1" rowspan="1">OXA-48:0.2%(2/157)</td><td colspan="1" rowspan="1">OXA-48:0.7% of144</td></tr><tr><td colspan="1" rowspan="1">OXA-48</td><td colspan="1" rowspan="1">K. aerogenes</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">Imipenem</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">OXA-48:2.2% of223</td></tr><tr><td colspan="1" rowspan="1">OXA-48</td><td colspan="1" rowspan="1">K. aerogenes</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">Ertapenem</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">OXA-48:2.2% of223</td></tr><tr><td colspan="1" rowspan="1">OXA-48</td><td colspan="1" rowspan="1">K. aerogenes</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">Meropenem</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">OXA-48:2.2% of223</td></tr><tr><td colspan="1" rowspan="1">OXA-48</td><td colspan="1" rowspan="1">K.michiganensis</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">Imipenem</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">OXA-48:1.1% of 89</td></tr><tr><td colspan="1" rowspan="1">OXA-48</td><td colspan="1" rowspan="1">K.michiganensis</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">Ertapenem</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">OXA-48:1.1% of 89</td></tr><tr><td colspan="1" rowspan="1">OXA-48</td><td colspan="1" rowspan="1">K.michiganensis</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">Meropenem</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">OXA-48:1.1% of 89</td></tr><tr><td colspan="1" rowspan="1">OXA-48</td><td colspan="1" rowspan="1">K. oxytoca</td><td colspan="1" rowspan="1">NaN</td><td colspan="1" rowspan="1">100.0%(5/5)556.55-100]</td><td colspan="1" rowspan="1">Imipenem</td><td colspan="1" rowspan="1">OXA-48:0.0%(0/0/2)0.00-665.76]</td><td colspan="1" rowspan="1">OXA-48:100.0%(3/3)[43.85-100.00]</td><td colspan="1" rowspan="1">OXA-48:NaN</td><td colspan="1" rowspan="1">OXA-48:60.0%(3/5) .07-88.24]</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">OXA-48:0.4% of273</td></tr><tr><td colspan="1" rowspan="1">OXA-48</td><td colspan="1" rowspan="1">K. oxytoca</td><td colspan="1" rowspan="1">NaN</td><td colspan="1" rowspan="1">100.0%(5/5)[56.55- 100.00]</td><td colspan="1" rowspan="1">Ertapenem</td><td colspan="1" rowspan="1">OXA-48:0.0%(0//2)[0.00-65.76]</td><td colspan="1" rowspan="1">OXA-48:100.0%(3/3)[43.85-100.00]</td><td colspan="1" rowspan="1">OXA-48:NaN</td><td colspan="1" rowspan="1">OXA-48:60.0%(3/5)[23.07-88.24]</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">OXA-48:0.4% of273</td></tr><tr><td colspan="1" rowspan="1">OXA-48</td><td colspan="1" rowspan="1">K. oxytoca</td><td colspan="1" rowspan="1">NaN</td><td colspan="1" rowspan="1">100.0%(5/5)5 6.5-100.00]</td><td colspan="1" rowspan="1">Meropenem</td><td colspan="1" rowspan="1">OXA-48:0.0%(0/2)[0.00-65.76]</td><td colspan="1" rowspan="1">OXA-48:100.0%(3/3)[43.85-0.00]</td><td colspan="1" rowspan="1">OXA-48:NaN</td><td colspan="1" rowspan="1">OXA-48:60.0%(3/5)[23.07-88.24]</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">OXA-48:0.4% of273</td></tr><tr><td colspan="1" rowspan="1">OXA-48</td><td colspan="1" rowspan="1">K. variicola</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">Imipenem</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">OXA-48:0.4% of272</td></tr><tr><td colspan="1" rowspan="1">OXA-48</td><td colspan="1" rowspan="1">K. variicola</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">Ertapenem</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">OXA-48:0.4% of272</td></tr><tr><td colspan="1" rowspan="1">OXA-48</td><td colspan="1" rowspan="1">K. variicola</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">Meropenem</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">OXA-48:0.4% of272</td></tr><tr><td colspan="1" rowspan="1">OXA-48</td><td colspan="1" rowspan="1">K.quasipneumoniae</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">Imipenem</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">OXA-48:1.2% of259</td></tr><tr><td colspan="1" rowspan="2">AMR GeneTarget</td><td colspan="1" rowspan="2">Organism</td><td colspan="2" rowspan="1">Detection of AMR GeneTarget by AMR GenePanel versus WholeGenome SequencingCombined Clinical DevicePerformance-Stock and ProspectiveIsolates (from K191288)</td><td colspan="1" rowspan="2">Antimicrobial</td><td colspan="4" rowspan="1">Detection of AMR Gene Target by AMR GenePanel vs AST Non-SusceptibilityCombined Clinical Device Performance - Stock andProspective Isolates (from K191288) includingonly unique isolates (no replicates)</td><td colspan="5" rowspan="1">Detection of AMR Gene Target versus AST Non-Susceptibility inOpGen's SMART Study</td><td colspan="1" rowspan="2">%Organismswith AMRGeneTarget inNCBIPathogenDatabase(2019)</td></tr><tr><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1">PPV[95% CI]</td><td colspan="1" rowspan="1">NPV[95% CI]</td><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1">PPV[95% CI]</td><td colspan="1" rowspan="1">NPV[95% CI]</td><td colspan="1" rowspan="1">% Organismswith AMRGene Target</td></tr><tr><td colspan="1" rowspan="1">OXA-48</td><td colspan="1" rowspan="1">K.quasipneumoniae</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">Ertapenem</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">OXA-48:1.2% of259</td></tr><tr><td colspan="1" rowspan="1">OXA-48</td><td colspan="1" rowspan="1">K.quasipneumoniae</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">Meropenem</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">OXA-48:1.2% of259</td></tr><tr><td colspan="1" rowspan="1">OXA-48</td><td colspan="1" rowspan="1">C. freundii</td><td colspan="1" rowspan="1">NaN</td><td colspan="1" rowspan="1">100.0%(14/14)[78.47-100.00]</td><td colspan="1" rowspan="1">Imipenem</td><td colspan="1" rowspan="1">OXA-48:0.0%(0/)[0.00-39.03]</td><td colspan="1" rowspan="1">OXA-48:100.0%(8/8)[67.56-100.00]</td><td colspan="1" rowspan="1">OXA-48:NaN</td><td colspan="1" rowspan="1">OXA-48:57.1%(8/14)[32.59-78.62]</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">OXA-48:12.3% of179</td></tr><tr><td colspan="1" rowspan="1">OXA-48</td><td colspan="1" rowspan="1">C. freundii</td><td colspan="1" rowspan="1">NaN</td><td colspan="1" rowspan="1">100.0%(14/14)78.47-100.00]</td><td colspan="1" rowspan="1">Ertapenem</td><td colspan="1" rowspan="1">OXA-48:0.0%(0/4)[0.00-48.99]</td><td colspan="1" rowspan="1">OXA-48:100.0%(10/10)[72.25-100.00]</td><td colspan="1" rowspan="1">OXA-48:NaN</td><td colspan="1" rowspan="1">OXA-48:71.4%(10/14)[45.35-88.28]</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">OXA-48:12.3% of179</td></tr><tr><td colspan="1" rowspan="1">OXA-48</td><td colspan="1" rowspan="1">C. freundii</td><td colspan="1" rowspan="1">NaN</td><td colspan="1" rowspan="1">100.0%(14/14)[78.47-100.00]</td><td colspan="1" rowspan="1">Meropenem</td><td colspan="1" rowspan="1">OXA-48:0.0%((0/4)0.00-48.99]</td><td colspan="1" rowspan="1">OXA-48:100.0%(10/10)[72.25-100.00]</td><td colspan="1" rowspan="1">OXA-48:NaN</td><td colspan="1" rowspan="1">OXA-48:71.4%(10/14)[45.35-88.28]</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">OXA-48:12.3% of179</td></tr><tr><td colspan="1" rowspan="1">OXA-48</td><td colspan="1" rowspan="1">C. braakii</td><td colspan="1" rowspan="1">NaN</td><td colspan="1" rowspan="1">100.0%(1/1)[20.65-100.00]</td><td colspan="1" rowspan="1">Imipenem</td><td colspan="1" rowspan="1">OXA-48:0.0%(0/)[0.00-79.35]</td><td colspan="1" rowspan="1">OXA-48:NaN</td><td colspan="1" rowspan="1">OXA-48:NaN</td><td colspan="1" rowspan="1">OXA-48:0.0%(0/1))[0.00-79.35]</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">OXA-48:0% of 30</td></tr><tr><td colspan="1" rowspan="1">OXA-48</td><td colspan="1" rowspan="1">C. braakii</td><td colspan="1" rowspan="1">NaN</td><td colspan="1" rowspan="1">100.0%(1/1)2065-100.00]</td><td colspan="1" rowspan="1">Ertapenem</td><td colspan="1" rowspan="1">OXA-48:NaN</td><td colspan="1" rowspan="1">OXA-48:100.0%(1/1))[20.65-100.00]</td><td colspan="1" rowspan="1">OXA-48:NaN</td><td colspan="1" rowspan="1">OXA-48:100.0%(1/1)[20.65-0.00]</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">OXA-48:0% of 30</td></tr><tr><td colspan="1" rowspan="1">OXA-48</td><td colspan="1" rowspan="1">C. braakii</td><td colspan="1" rowspan="1">NaN</td><td colspan="1" rowspan="1">100.0%(1/1)[20.65-100.00]</td><td colspan="1" rowspan="1">Meropenem</td><td colspan="1" rowspan="1">OXA-48:NaN</td><td colspan="1" rowspan="1">OXA-48:100.0%(1/1)[20.65-100.00]</td><td colspan="1" rowspan="1">OXA-48:NaN</td><td colspan="1" rowspan="1">OXA-48:100.0%(1/1)[20.65-10.00]</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">OXA-48:0% of 30</td></tr><tr><td colspan="1" rowspan="2">AMR GeneTarget</td><td colspan="1" rowspan="2">Organism</td><td colspan="2" rowspan="1">Detection of AMR GeneTarget by AMR GenePanel versus WholeGenome SequencingCombined Clinical DevicePerformance -Stock and ProspectiveIsolates (from K191288)</td><td colspan="1" rowspan="2">Antimicrobial</td><td colspan="4" rowspan="1">Detection of AMR Gene Target by AMR GenePanel vs AST Non-SusceptibilityCombined Clinical Device Performance - Stock andProspective Isolates (from K191288) includingonly unique isolates (no replicates)</td><td colspan="5" rowspan="1">Detection of AMR Gene Target versus AST Non-Susceptibility inOpGen's SMART Study</td><td colspan="1" rowspan="2">%Organismswith AMRGeneTarget inNCBIPathogenDatabase(2019)</td></tr><tr><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1">PPV[95% CI]</td><td colspan="1" rowspan="1">NPV[95% CI]</td><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1">PPV[95% CI]</td><td colspan="1" rowspan="1">NPV[95% CI]</td><td colspan="1" rowspan="1">% Organismswith AMRGene Target</td></tr><tr><td colspan="1" rowspan="1">OXA-48</td><td colspan="1" rowspan="1">C. werkmanii</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">Imipenem</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">OXA-48:10.0% of10</td></tr><tr><td colspan="1" rowspan="1">OXA-48</td><td colspan="1" rowspan="1">C. werkmanii</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">Ertapenem</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">OXA-48:10.0% of10</td></tr><tr><td colspan="1" rowspan="1">OXA-48</td><td colspan="1" rowspan="1">C. werkmanii</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">Meropenem</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">OXA-48:10.0% of10</td></tr><tr><td colspan="1" rowspan="1">OXA-48</td><td colspan="1" rowspan="1">C. youngae</td><td colspan="1" rowspan="1">NaN</td><td colspan="1" rowspan="1">100.0%(1/1)[20.65-100.00]</td><td colspan="1" rowspan="1">Imipenem</td><td colspan="1" rowspan="1">OXA-48:0.0%(0/1)[0.00-79.35]</td><td colspan="1" rowspan="1">OXA-48:NaN</td><td colspan="1" rowspan="1">OXA-48:NaN</td><td colspan="1" rowspan="1">OXA-48:0.0%(0/1)0.-79.35]</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">OXA-48:0% of 7</td></tr><tr><td colspan="1" rowspan="1">OXA-48</td><td colspan="1" rowspan="1">C. youngae</td><td colspan="1" rowspan="1">NaN</td><td colspan="1" rowspan="1">100.0%(1/1)[20.65-100.00]</td><td colspan="1" rowspan="1">Ertapenem</td><td colspan="1" rowspan="1">OXA-48:0.0%(0/1)[0.00-79.35]</td><td colspan="1" rowspan="1">OXA-48:NaN</td><td colspan="1" rowspan="1">OXA-48:NaN</td><td colspan="1" rowspan="1">OXA-48:0.0%(/1)[0.00-79.35]</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">OXA-48:0% of 7</td></tr><tr><td colspan="1" rowspan="1">OXA-48</td><td colspan="1" rowspan="1">C. youngae</td><td colspan="1" rowspan="1">NaN</td><td colspan="1" rowspan="1">100.0%(1/1)[20.65-100.00]</td><td colspan="1" rowspan="1">Meropenem</td><td colspan="1" rowspan="1">OXA-48:0.0%(0/ /1)[0.00-79.35]</td><td colspan="1" rowspan="1">OXA-48:NaN</td><td colspan="1" rowspan="1">OXA-48:NaN</td><td colspan="1" rowspan="1">OXA-48:0.0%(//1)[0.0-79.35]</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">OXA-48:0% of 7</td></tr><tr><td colspan="1" rowspan="1">OXA-48</td><td colspan="1" rowspan="1">C. koseri</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">Imipenem</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">OXA-48:9.7% of 31</td></tr><tr><td colspan="1" rowspan="1">OXA-48</td><td colspan="1" rowspan="1">C. koseri</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">Ertapenem</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">OXA-48:9.7% of 31</td></tr><tr><td colspan="1" rowspan="1">OXA-48</td><td colspan="1" rowspan="1">C. koseri</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">Meropenem</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">OXA-48:9.7% of 31</td></tr><tr><td colspan="1" rowspan="1">OXA-48</td><td colspan="1" rowspan="1">S. marcescens</td><td colspan="1" rowspan="1">100.0%(1/1)[20.65-1000]</td><td colspan="1" rowspan="1">100.0%(1/1)20.65-100.00]</td><td colspan="1" rowspan="1">Imipenem</td><td colspan="1" rowspan="1">OXA-48:50.0%(1/2)[9.45-900.55]</td><td colspan="1" rowspan="1">OXA-48:NaN</td><td colspan="1" rowspan="1">OXA-48:100.0%(1/1)[20.65-100.00]</td><td colspan="1" rowspan="1">OXA-48:0.0%(0/)[0.0o-79.35]</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">OXA-48:2.2% of496</td></tr><tr><td colspan="1" rowspan="2">AMR GeneTarget</td><td colspan="1" rowspan="2">Organism</td><td colspan="2" rowspan="1">Detection of AMR GeneTarget by AMR GenePanel versus WholeGenome SequencingCombined Clinical DevicePerformance -Stock and ProspectiveIsolates (from K191288)</td><td colspan="1" rowspan="2">Antimicrobial</td><td colspan="4" rowspan="1">Detection of AMR Gene Target by AMR GenePanel vs AST Non-SusceptibilityCombined Clinical Device Performance - Stock andProspective Isolates (from K191288) includingonly unique isolates (no replicates)</td><td colspan="5" rowspan="1">Detection of AMR Gene Target versus AST Non-Susceptibility inOpGen's SMART Study</td><td colspan="1" rowspan="2">%Organismswith AMRGeneTarget inNCBIPathogenDatabase2019)</td></tr><tr><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1">PPV[95% CI]</td><td colspan="1" rowspan="1">NPV[95% CI]</td><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1">PPV[95% CI]</td><td colspan="1" rowspan="1">NPV[95% CI]</td><td colspan="1" rowspan="1">% Organismswith AMRGene Target</td></tr><tr><td colspan="1" rowspan="1">OXA-48</td><td colspan="1" rowspan="1">S. marcescens</td><td colspan="1" rowspan="1">100.0%(1/1)[20.65-100]</td><td colspan="1" rowspan="1">100.0%(1/1)[20.65-100]</td><td colspan="1" rowspan="1">Ertapenem</td><td colspan="1" rowspan="1">OXA-48:50.0%(1/2).45-900.55]</td><td colspan="1" rowspan="1">OXA-48:NaN</td><td colspan="1" rowspan="1">OXA-48:100.0%(1/1)20.65-0.00]</td><td colspan="1" rowspan="1">OXA-48:0.0%(0/1)[0.0-79.35]</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">OXA-48:2.2% of496</td></tr><tr><td colspan="1" rowspan="1">OXA-48</td><td colspan="1" rowspan="1">S. marcescens</td><td colspan="1" rowspan="1">100.0%(1/1)20.65-100.0]</td><td colspan="1" rowspan="1">100.0%(1/1)20.65-100.00]</td><td colspan="1" rowspan="1">Meropenem</td><td colspan="1" rowspan="1">OXA-48:50.0%(1/2)[9.45-0.55]</td><td colspan="1" rowspan="1">OXA-48:NaN</td><td colspan="1" rowspan="1">OXA-48:100.0%(1/1)[20.65-0.00]</td><td colspan="1" rowspan="1">OXA-48:0.0%(/1)[0.00-7.35]</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">OXA-48:2.2% of496</td></tr><tr><td colspan="1" rowspan="1">OXA-48</td><td colspan="1" rowspan="1">M. morganii</td><td colspan="1" rowspan="1">NaN</td><td colspan="1" rowspan="1">100.0%(7/7)[64.57-100.00]</td><td colspan="1" rowspan="1">Ertapenem</td><td colspan="1" rowspan="1">OXA-48:0.0%(0/2)[0.00-65.76]</td><td colspan="1" rowspan="1">OXA-48:100.0%(5/5)[56.55-100.00]</td><td colspan="1" rowspan="1">OXA-48:NaN</td><td colspan="1" rowspan="1">OXA-48:71.4%(5/7)[35.89-91.78]</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">OXA-48:1.8% of 57</td></tr><tr><td colspan="1" rowspan="1">OXA-48</td><td colspan="1" rowspan="1">M. morganii</td><td colspan="1" rowspan="1">NaN</td><td colspan="1" rowspan="1">100.0%(7/7)[64.57-100.00]</td><td colspan="1" rowspan="1">Meropenem</td><td colspan="1" rowspan="1">OXA-48:0.0%(0/2)[0.00-65.76]</td><td colspan="1" rowspan="1">OXA-48:100.0%(5/5)[56.55-100.00]</td><td colspan="1" rowspan="1">OXA-48:NaN</td><td colspan="1" rowspan="1">OXA-48:71.4%(5/7)[35.89-91.78]</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">OXA-48:1.8% of 57</td></tr><tr><td colspan="1" rowspan="1">OXA-48</td><td colspan="1" rowspan="1">R.ornithinolytica</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">Imipenem</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">OXA-48:12.5% of56</td></tr><tr><td colspan="1" rowspan="1">OXA-48</td><td colspan="1" rowspan="1">R.ornithinolytica</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">Ertapenem</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">OXA-48:12.5% of56</td></tr><tr><td colspan="1" rowspan="1">OXA-48</td><td colspan="1" rowspan="1">R.ornithinolytica</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">Meropenem</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">OXA-48:12.5% of56</td></tr><tr><td colspan="1" rowspan="1">PER</td><td colspan="1" rowspan="1">P. aeruginosa</td><td colspan="1" rowspan="1">98.8%(81/82)[93.41-99.78]</td><td colspan="1" rowspan="1">99.6%(265/266)[97.90-9.93]</td><td colspan="1" rowspan="1">Imipenem</td><td colspan="1" rowspan="1">PER: 5.9%7/119)[2.88-1.65]</td><td colspan="1" rowspan="1">PER:94.4%(34/36)81.86-98.46]</td><td colspan="1" rowspan="1">PER:77.8%(7/9)45.26-93.68]</td><td colspan="1" rowspan="1">PER:23.3%(34/146)[17.17-30.77]</td><td colspan="1" rowspan="1">PER: 3.2%(32/1002)[2.27-4.47]</td><td colspan="1" rowspan="1">PER: 98.4%(539/548)[96.91-99.13]</td><td colspan="1" rowspan="1">PER: 78.0%(32/41)[63.29-88.00]</td><td colspan="1" rowspan="1">PER: 35.7%(539/509)[33.34-38.17]</td><td colspan="1" rowspan="1">PER: 2.6%(41/1550)</td><td colspan="1" rowspan="1">PER: 0.3%of 4706</td></tr><tr><td colspan="1" rowspan="2">AMR GeneTarget</td><td colspan="1" rowspan="2">Organism</td><td colspan="2" rowspan="1">Detection of AMR GeneTarget by AMR GenePanel versus WholeGenome SequencingCombined Clinical DevicePerformance -Stock and ProspectiveIsolates (from K191288)</td><td colspan="1" rowspan="1">Antimicrobial</td><td colspan="4" rowspan="1">Detection of AMR Gene Target by AMR GenePanel vs AST Non-SusceptibilityCombined Clinical Device Performance - Stock andProspective Isolates (from K191288) includingonly unique isolates (no replicates)</td><td colspan="5" rowspan="1">Detection of AMR Gene Target versus AST Non-Susceptibility inOpGen's SMART Study</td><td colspan="1" rowspan="2">%Organismswith AMRGeneTarget inNCBIPathogenDatabase(2019)</td></tr><tr><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1">PPV[95% CI]</td><td colspan="1" rowspan="1">NPV[95% CI]</td><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1">PPV[95% CI]</td><td colspan="1" rowspan="1">NPV[95% CI]</td><td colspan="1" rowspan="1">% Organismswith AMRGene Target</td></tr><tr><td colspan="1" rowspan="1">PER</td><td colspan="1" rowspan="1">P. aeruginosa</td><td colspan="1" rowspan="1">98.8%(81/82)[93.41-99.78]</td><td colspan="1" rowspan="1">99.6%(265/266)[97.90-9.93]</td><td colspan="1" rowspan="1">Meropenem</td><td colspan="1" rowspan="1">PER: 5.9%(7/118)2.90-11.74]</td><td colspan="1" rowspan="1">PER:94.6%(35/37)[82.30-98.50]</td><td colspan="1" rowspan="1">PER:77.8%(7/9)45.26-93.68]</td><td colspan="1" rowspan="1">PER:24.0%(35/146)[17.77-31.51]</td><td colspan="1" rowspan="1">PER: 3.1%(31/986)[2.22-4.43]</td><td colspan="1" rowspan="1">PER: 98.2%54/564)[96.77-99.03]</td><td colspan="1" rowspan="1">PER: 75.6%(31/41)[60.66-86.17]</td><td colspan="1" rowspan="1">PER: 36.7%(5/1509)[34.32-39.18]</td><td colspan="1" rowspan="1">PER: 2.6%(41/1550)</td><td colspan="1" rowspan="1">PER: 0.3%of 4706</td></tr><tr><td colspan="1" rowspan="1">RMT</td><td colspan="1" rowspan="1">K. pneumoniae</td><td colspan="1" rowspan="1">96.9%(31/32)[84.26-99.45]</td><td colspan="1" rowspan="1">100.0%(297/297)[98.72-10.00]</td><td colspan="1" rowspan="1">Amikacin</td><td colspan="1" rowspan="1">RMT:28.9%(26/90)[20.54-38.96]</td><td colspan="1" rowspan="1">RMT:100.0%(95/95)[96.11-10.00]</td><td colspan="1" rowspan="1">RMT:100.0%(26/26)[87.13-00]</td><td colspan="1" rowspan="1">RMT:59.7%(95/159)[51.98-67.05]</td><td colspan="1" rowspan="1">rmtB/rmtF: 17.1%(107/624)[14.39-200.30]</td><td colspan="1" rowspan="1">rmtB/rmtF: 99.6%(1399/1404) [99.17-9.85]</td><td colspan="1" rowspan="1">rmtB/rmtF: 95.5%(107/112)[89.97-98.08]</td><td colspan="1" rowspan="1">rmtB/rmtF:73.0%(1399/191) (70.98-74.96]</td><td colspan="1" rowspan="1">rmtB/rmtF:5.5%(112/2028)</td><td colspan="1" rowspan="1">rmtB/rmtF: 7.3% of7979</td></tr><tr><td colspan="1" rowspan="1">RMT</td><td colspan="1" rowspan="1">K. pneumoniae</td><td colspan="1" rowspan="1">96.9%(31/32)[84.26-99.45]</td><td colspan="1" rowspan="1">100.0%(297/297)[98.72-10.00]</td><td colspan="1" rowspan="1">Gentamicin</td><td colspan="1" rowspan="1">RMT:19.3%(26/135)13.50-26.72]</td><td colspan="1" rowspan="1">RMT:100.0%(50/50)[92.87-0.00]</td><td colspan="1" rowspan="1">RMT:100.0%(26/26)[87.13-100.00]</td><td colspan="1" rowspan="1">RMT:31.4%(50/159)[24.74-39.03]</td><td colspan="1" rowspan="1">rmtB/rmtF: 9.3%(110/1177)[7.81-11.14]</td><td colspan="1" rowspan="1">rmtB/rmtF: 99.8%(849/851)99.15-99.94]</td><td colspan="1" rowspan="1">rmtB/rmtF: 98.2%(110/112)[93.72-99.51]</td><td colspan="1" rowspan="1">rmtB/rmtF: 44.3%(849/1916)[42.10-46.54]</td><td colspan="1" rowspan="1">rmtB/rmtF:5.5%(112/2028)</td><td colspan="1" rowspan="1">rmtB/rmtF:7.3% of7979</td></tr><tr><td colspan="1" rowspan="1">RMT</td><td colspan="1" rowspan="1">K. pneumoniae</td><td colspan="1" rowspan="1">96.9%(31/32)[84.26-99.45]</td><td colspan="1" rowspan="1">100.0%97/297)[98.72-100.00]</td><td colspan="1" rowspan="1">Tobramycin</td><td colspan="1" rowspan="1">RMT:17.7%(26/147)[12.37-24.65]</td><td colspan="1" rowspan="1">RMT:100.0%(38/38)[90.82-00]</td><td colspan="1" rowspan="1">RMT:100.0%(26/26)[87.13-100.00]</td><td colspan="1" rowspan="1">RMT:23.9%(38/159)[17.94-31.09]</td><td colspan="1" rowspan="1">rmtB/rmtF:7.2%(112/1546)[6.06-8.65]</td><td colspan="1" rowspan="1">rmtB/rmtF: 100.0%(482/482)[99.21-00]</td><td colspan="1" rowspan="1">rmtB/rmtF: 100.0%(112/112)[96.68-10.00]</td><td colspan="1" rowspan="1">rmtB/rmtF: 25.2%(482/1916)[23.26-27.15]</td><td colspan="1" rowspan="1">rmtB/rmtF:5.5%(112/2028)</td><td colspan="1" rowspan="1">rmtB/rmtF:7.3% of7979</td></tr><tr><td colspan="1" rowspan="1">SHV</td><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">100.0%(11/11)[74.12-10.00]</td><td colspan="1" rowspan="1">99.7%(323/324)[98.27-9.95]</td><td colspan="1" rowspan="1">Ampicillin</td><td colspan="1" rowspan="1">SHV:6.5%(10/154)3.57-11.54]</td><td colspan="1" rowspan="1">SHV:100.0%(5/5)556.5-100.00]</td><td colspan="1" rowspan="1">SHV:100.0%(10/10)[72.25-10000]</td><td colspan="1" rowspan="1">SHV:3.4%(5/149)[1.44-7.61]</td><td colspan="1" rowspan="1">SHV: 2.5%(64/2594)[1.94-3.14]</td><td colspan="1" rowspan="1">SHV:99.7%(365/366)[98.47-99.95]</td><td colspan="1" rowspan="1">SHV:98.5%(64/5)91.79-99.73]</td><td colspan="1" rowspan="1">SHV:12.6%(365/2895)[11.45-13.87]</td><td colspan="1" rowspan="1">SHV: 2.2%(65/2960)</td><td colspan="1" rowspan="1">SHV: 0.6%of 19017</td></tr><tr><td colspan="1" rowspan="1">SHV</td><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">100.0%(11/11)74.12-100.00]</td><td colspan="1" rowspan="1">99.7%(323/324)98.27-9.95]</td><td colspan="1" rowspan="1">Amox.K.Clav</td><td colspan="1" rowspan="1">SHV:7.9%(9/114)[4.21-14.33]</td><td colspan="1" rowspan="1">SHV:97.8%(44/45)[88.43-9.61]</td><td colspan="1" rowspan="1">SHV:90.0%(9/10)[59.58-98.21]</td><td colspan="1" rowspan="1">SHV:29.5%(44/149)[22.79-37.30]</td><td colspan="1" rowspan="1">SHV: 2.5%(46/1810)[1.91-3.37]</td><td colspan="1" rowspan="1">SHV:98.3%(1131/1150) [97.43-98.94]</td><td colspan="1" rowspan="1">SHV:70.8%(46/65)58.8080.42]</td><td colspan="1" rowspan="1">SHV:39.1%(1131/2895) [37.31-40.86]</td><td colspan="1" rowspan="1">SHV: 2.2%(65/2960)</td><td colspan="1" rowspan="1">SHV: 0.6%of 19017</td></tr><tr><td colspan="1" rowspan="1">SHV</td><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">100.0%(11/11)[74.12-100.00]</td><td colspan="1" rowspan="1">99.7%(323/324)[98.27-999.95]</td><td colspan="1" rowspan="1">Amp.Sulbactam</td><td colspan="1" rowspan="1">SHV:6.2%(9/146)[3.28-111.30]</td><td colspan="1" rowspan="1">SHV:92.3%(12/13)[66.69-98.63]</td><td colspan="1" rowspan="1">SHV:90.0%(9/10)59.8-988.21]</td><td colspan="1" rowspan="1">SHV:8.1%(12/149)[4.67-13.55]</td><td colspan="1" rowspan="1">SHV: 2.5%(61/2477)[1.92-3.15]</td><td colspan="1" rowspan="1">SHV:99.2%(479/483)[97.89-99.68]</td><td colspan="1" rowspan="1">SHV:93.8%(61/65)[85.22-97.58]</td><td colspan="1" rowspan="1">SHV:16.5%(479/2895)[15.24-17.94]</td><td colspan="1" rowspan="1">SHV: 2.2%(65/2960)</td><td colspan="1" rowspan="1">SHV: 0.6%of 19017</td></tr><tr><td colspan="1" rowspan="2">AMR GeneTarget</td><td colspan="1" rowspan="2">Organism</td><td colspan="2" rowspan="1">Detection of AMR GeneTarget by AMR GenePanel versus WholeGenome SequencingCombined Clinical DevicePerformance-Stock and ProspectiveIsolates (from K191288)</td><td colspan="1" rowspan="1">Antimicrobial</td><td colspan="4" rowspan="1">Detection of AMR Gene Target by AMR GenePanel vs AST Non-SusceptibilityCombined Clinical Device Performance - Stock andProspective Isolates (from K191288)includingonly unique isolates (no replicates)</td><td colspan="5" rowspan="1">Detection of AMR Gene Target versus AST Non-Susceptibility inOpGen's SMART Study</td><td colspan="1" rowspan="2">%Organismswith AMRGeneTarget inNCBIPathogenDatabase(2019)</td></tr><tr><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1">PPV[95% CI]</td><td colspan="1" rowspan="1">NPV[95% CI]</td><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1">PPV[95% CI]</td><td colspan="1" rowspan="1">NPV[95% CI]</td><td colspan="1" rowspan="1">% Organismswith AMRGene Target</td></tr><tr><td colspan="1" rowspan="1">SHV</td><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">100.0%(11/11)[74.12-0.00]</td><td colspan="1" rowspan="1">99.7%(323/324)[98.27-9.95]</td><td colspan="1" rowspan="1">Pip.Tazo</td><td colspan="1" rowspan="1">SHV:11.6%(8/69)[5.99-21.25]</td><td colspan="1" rowspan="1">SHV:97.8%(88/90)92.26-99.39]</td><td colspan="1" rowspan="1">SHV:80.0%(8/10)49.02-94.33]</td><td colspan="1" rowspan="1">SHV:59.1%(88/149)[51.03-66.63]</td><td colspan="1" rowspan="1">SHV: 5.5%(38/687)[4.06-7.50]</td><td colspan="1" rowspan="1">SHV:98.8%(2246/2273) [98.28-99.18]</td><td colspan="1" rowspan="1">SHV:58.5%(38/65)[46.34-69.64]</td><td colspan="1" rowspan="1">SHV:77.6%(2246/289) 76.03-79.06]</td><td colspan="1" rowspan="1">SHV: 2.2%(65/2960)</td><td colspan="1" rowspan="1">SHV: 0.6% f 19017</td></tr><tr><td colspan="1" rowspan="1">SHV</td><td colspan="1" rowspan="1">P. aeruginosa</td><td colspan="1" rowspan="1">100.0%(1/1)[20.65-10.00]</td><td colspan="1" rowspan="1">99.4%(345/347)[97.92-99.84]</td><td colspan="1" rowspan="1">Pip.Tazo</td><td colspan="1" rowspan="1">SHV:1.9%(2/104)[0.53-6.74]</td><td colspan="1" rowspan="1">SHV:100.0%(5151)[93.00-100.00]</td><td colspan="1" rowspan="1">SHV:100.0%(2/2)[34.24-100.00]</td><td colspan="1" rowspan="1">SHV:33.3%(51/153)[26.35-41.13]</td><td colspan="1" rowspan="1">SHV: 0.6%(6/941)[0.29-1.38]</td><td colspan="1" rowspan="1">SHV:99.7%(607/609)[98.81-9.91</td><td colspan="1" rowspan="1">SHV:75.0%(6/8)[40.93-92.85]</td><td colspan="1" rowspan="1">SHV:39.4%(607/1542)[36.96-41.83]</td><td colspan="1" rowspan="1">SHV: 0.5%(8/1550)</td><td colspan="1" rowspan="1">SHV: 0% of4706</td></tr><tr><td colspan="1" rowspan="1">Sul1</td><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">99.5%(189/190)[97.08-9.91]</td><td colspan="1" rowspan="1">97.2%(141/145)[93.12-98.92]</td><td colspan="1" rowspan="1">Trimeth.Sulfa</td><td colspan="1" rowspan="1">Sul1:74.6%(85/114)65.86-81.66]</td><td colspan="1" rowspan="1">Sul1:86.7%(39/45)[73.82-93.74]</td><td colspan="1" rowspan="1">Sul1:93.4%(85/91)86.596.94]</td><td colspan="1" rowspan="1">Sul1:57.4%(39/68)45.5268.40]</td><td colspan="1" rowspan="1">Sul1:76.7%(1328/173) [74.62-78.61]</td><td colspan="1" rowspan="1">Sul1:91.0%(1117/1228) [89.23-92.44]</td><td colspan="1" rowspan="1">Sul1:92.3%(1328/1439)90.79-93.56]</td><td colspan="1" rowspan="1">Sul1:73.4%(1117/1521) 71.16-75.60]</td><td colspan="1" rowspan="1">Sul1: 48.6%(1439/2960)</td><td colspan="1" rowspan="1">Sul1: 20%of 19017</td></tr><tr><td colspan="1" rowspan="1">Sul1</td><td colspan="1" rowspan="1">K. pneumoniae</td><td colspan="1" rowspan="1">98.7%(231/234)[96.30-99.56]</td><td colspan="1" rowspan="1">95.8%(91/95)[89.67-98.35]</td><td colspan="1" rowspan="1">Trimeth.Sulfa</td><td colspan="1" rowspan="1">Sul1:80.0%(120/150)[72.89-85.62]</td><td colspan="1" rowspan="1">Sul1:60.0%(21/)43.57-74.45]</td><td colspan="1" rowspan="1">Sul1:89.6%(120/134)[83.23-93.67]</td><td colspan="1" rowspan="1">Sul1:41.2%(21/)[28.75-54.3]</td><td colspan="1" rowspan="1">Sul1:65.5%(997/1523)[63.04-67.81]</td><td colspan="1" rowspan="1">Sul1:82.4%(416/505)[78.81-85.45]</td><td colspan="1" rowspan="1">Sul1:91.8%(997/1086)[90.02-93.29]</td><td colspan="1" rowspan="1">Sul1:44.2%(416/942)[41.02-47.35]</td><td colspan="1" rowspan="1">Sul1: 53.6%(1086/2028)</td><td colspan="1" rowspan="1">Sul1:45.9% of7979</td></tr><tr><td colspan="1" rowspan="1">Sul2</td><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">97.3%(218/224)[94.28-98.77]</td><td colspan="1" rowspan="1">95.5%(106/111)[89.89-98.06]</td><td colspan="1" rowspan="1">Trimeth.Sulfa</td><td colspan="1" rowspan="1">Sul2:64.9%(74/114)[55.80-73.06]</td><td colspan="1" rowspan="1">Sul2:82.2%(37/45)[68.67-90.71]</td><td colspan="1" rowspan="1">Sul2:90.2%(74/82)[81.91-94.97]</td><td colspan="1" rowspan="1">Sul2:48.1%(37/77)[37.25-59.04]</td><td colspan="1" rowspan="1">Sul2:56.5%(979/1732)[54.18-58.84]</td><td colspan="1" rowspan="1">Sul2:87.2%(1071/1228) [85.23-88.97]</td><td colspan="1" rowspan="1">Sul2:86.2%(979/1136)[84.05-88.06]</td><td colspan="1" rowspan="1">Sul2:58.7%(1071/1824) [56.44-60.96]</td><td colspan="1" rowspan="1">Sul2: 38.4%(1136/2960)</td><td colspan="1" rowspan="1">Sul2:27.8% of19017</td></tr><tr><td colspan="1" rowspan="1">Sul2</td><td colspan="1" rowspan="1">K. pneumoniae</td><td colspan="1" rowspan="1">97.1%(166/171)[93.34-98.74]</td><td colspan="1" rowspan="1">97.5%(154/158)[93.67-99.01]</td><td colspan="1" rowspan="1">Trimeth.Sulfa</td><td colspan="1" rowspan="1">Sul2:56.7%(85/150)[48.67-64.33]</td><td colspan="1" rowspan="1">Sul2:88.6%(31/35)[74.05-95.46]</td><td colspan="1" rowspan="1">Sul2:95.5%(85/89)[89.01-98.24]</td><td colspan="1" rowspan="1">Sul2:32.3%(31/96)[23.78-42.17]</td><td colspan="1" rowspan="1">Sul2:45.3%(690/1523)[42.82-47.81]</td><td colspan="1" rowspan="1">Sul2:91.9%(464/505)[89.17-93.96]</td><td colspan="1" rowspan="1">Sul2:94.4%(690/731)[92.48-95.84]</td><td colspan="1" rowspan="1">Sul2:35.8%(464/1297)[33.21-38.42]</td><td colspan="1" rowspan="1">Sul2: 36%(731/2028)</td><td colspan="1" rowspan="1">Sul2:30.7% of7979</td></tr><tr><td colspan="1" rowspan="1">Sul2</td><td colspan="1" rowspan="1">P. mirabilis</td><td colspan="1" rowspan="1">99.1%(105/106)[94.85-99.83]</td><td colspan="1" rowspan="1">95.9%(47/49)[86.29-98.87]</td><td colspan="1" rowspan="1">Trimeth.Sulfa</td><td colspan="1" rowspan="1">Sul2:59.7%(40/67)[47.74-70.61]</td><td colspan="1" rowspan="1">Sul2:90.9%(10/11)62.26-98.38]</td><td colspan="1" rowspan="1">Sul2:97.6%(40/41)[87.40-999.57]</td><td colspan="1" rowspan="1">Sul2:27.0%(10/37)[15.40-42.98]</td><td colspan="1" rowspan="1">Sul2:59.7%(434/727)[56.09-63.20]</td><td colspan="1" rowspan="1">Sul2:92.1%(396/430)[89.15-94.29]</td><td colspan="1" rowspan="1">Sul2:92.7%(434/468)[90.02-94.75]</td><td colspan="1" rowspan="1">Sul2:57.5%(396/689)[53.75-61.11]</td><td colspan="1" rowspan="1">Sul2: 40.4%6/1157)</td><td colspan="1" rowspan="1">Sul2:45.1% of144</td></tr><tr><td colspan="1" rowspan="2">AMR GeneTarget</td><td colspan="1" rowspan="2">Organism</td><td colspan="2" rowspan="1">Detection of AMR GeneTarget by AMR GenePanel versus WholeGenome SequencingCombined Clinical DevicePerformance-Stock and ProspectiveIsolates (from K191288)</td><td colspan="1" rowspan="1">Antimicrobial</td><td colspan="4" rowspan="1">Detection of AMR Gene Target by AMR GenePanel vs AST Non-SusceptibilityCombined Clinical Device Performance - Stock andProspective Isolates (fromK191288)includinonly unique isolates (no replicates)</td><td colspan="5" rowspan="1">Detection of AMR Gene Target versus AST Non-Susceptibility inOpGen's SMART Study</td><td colspan="1" rowspan="2">%Organismswith AMRGeneTarget inNCBIPathogenDatabase(2019)</td></tr><tr><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1">PPV[95% CI]</td><td colspan="1" rowspan="1">NPV[95% CI]</td><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1">PPV[95% CI]</td><td colspan="1" rowspan="1">NPV[95% CI]</td><td colspan="1" rowspan="1">% Organismswith AMRGene Target</td></tr><tr><td colspan="1" rowspan="1">TEM</td><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">97.7%(255/261)[95.08-98.94]</td><td colspan="1" rowspan="1">95.9%((1/74)[88.75-98.61]</td><td colspan="1" rowspan="1">Ampicillin</td><td colspan="1" rowspan="1">TEM:63.6%(98/154)55.79-70.82]</td><td colspan="1" rowspan="1">TEM:100.0%(5/5)556.55-00]</td><td colspan="1" rowspan="1">TEM:100.0%(98/98)[96.23-100]</td><td colspan="1" rowspan="1">TEM:8.2%(5/61)[3.55-17.79]</td><td colspan="1" rowspan="1">TEM:52.1%(1351/2594) [50.16-54.00]</td><td colspan="1" rowspan="1">TEM:93.7%(343/366)[90.75-95.78]</td><td colspan="1" rowspan="1">TEM:98.3%(1351/1374) [97.50-98.88]</td><td colspan="1" rowspan="1">TEM:21.6%(343/1586)19.67-23.72]</td><td colspan="1" rowspan="1">TEM: 46.4%(1374/2960)</td><td colspan="1" rowspan="1">TEM:28.1% of19017</td></tr><tr><td colspan="1" rowspan="1">TEM</td><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">97.7%(255/261)[95.08-98.4]</td><td colspan="1" rowspan="1">95.9%(71/74)[88.75-98.61]</td><td colspan="1" rowspan="1">Amox.K.Clav</td><td colspan="1" rowspan="1">TEM:59.6%(68/114)[50.47-68.20]</td><td colspan="1" rowspan="1">TEM:33.3%(15/45)[21.36-47.93]</td><td colspan="1" rowspan="1">TEM:69.4%(68/98)[59.68-77.64]</td><td colspan="1" rowspan="1">TEM:24.6%15/6115.51-36.68]</td><td colspan="1" rowspan="1">TEM:52.6%(952/1810)[50.29-54.89]</td><td colspan="1" rowspan="1">TEM:63.3%(728/1150)[60.48-66.04]</td><td colspan="1" rowspan="1">TEM:69.3%(952/1374)[66.80-71.67]</td><td colspan="1" rowspan="1">TEM:45.9%(728/1586)[43.46-48.36]</td><td colspan="1" rowspan="1">TEM: 46.4%(1374/2960)</td><td colspan="1" rowspan="1">TEM:28.1% of19017</td></tr><tr><td colspan="1" rowspan="1">TEM</td><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">97.7%(255/261)[95.08-98.94]</td><td colspan="1" rowspan="1">95.9%(71/74)[88.75-98.61]</td><td colspan="1" rowspan="1">Amp.Sulbactam</td><td colspan="1" rowspan="1">TEM:65.8%(96/146)57.74-72.96]</td><td colspan="1" rowspan="1">TEM:84.6%(11/3)7.7 95.67]</td><td colspan="1" rowspan="1">TEM:98.0%(96/98)92.8699.44]</td><td colspan="1" rowspan="1">TEM:18.0%(11/61)10.3-29.7]</td><td colspan="1" rowspan="1">TEM:53.3%(1320/247) 51.32-55.25]</td><td colspan="1" rowspan="1">TEM:88.8%(429/483)85.70-91.33]</td><td colspan="1" rowspan="1">TEM:96.1%(1320/1374) (94.91-96.98]</td><td colspan="1" rowspan="1">TEM:27.0%(429/1586)[24.92-29.29]</td><td colspan="1" rowspan="1">TEM: 46.4%(1374/2960)</td><td colspan="1" rowspan="1">TEM:28.1% of19017</td></tr><tr><td colspan="1" rowspan="1">TEM</td><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">97.7%(255/261)[95.08-98.94]</td><td colspan="1" rowspan="1">95.9%(71/74)[88.75-988.61]</td><td colspan="1" rowspan="1">Pip.Tazo</td><td colspan="1" rowspan="1">TEM:65.2%(45/69)[53.45-75.38]</td><td colspan="1" rowspan="1">TEM:41.1%(37/90)[31.51-51.44]</td><td colspan="1" rowspan="1">TEM:45.9%(45/98)[36.39-55.75]</td><td colspan="1" rowspan="1">TEM:60.7%(37/61)[48.12-71.93]</td><td colspan="1" rowspan="1">TEM:54.0%(371/687)[50.26-57.70]</td><td colspan="1" rowspan="1">TEM:55.9%(1270/2273) [53.82-557.90]</td><td colspan="1" rowspan="1">TEM:27.0%(371/1374)[24.72-29.41]</td><td colspan="1" rowspan="1">TEM:80.1%(1270/158) [78.04-81.97]</td><td colspan="1" rowspan="1">TEM: 46.4%(1374/2960)</td><td colspan="1" rowspan="1">TEM:28.1% of19017</td></tr><tr><td colspan="1" rowspan="1">TEM</td><td colspan="1" rowspan="1">K. pneumoniae</td><td colspan="1" rowspan="1">98.6%(214/217)[96.01-99.53]</td><td colspan="1" rowspan="1">95.5%(107/112)[89.97-98.08]</td><td colspan="1" rowspan="1">Amox.K.Clav</td><td colspan="1" rowspan="1">TEM:69.3%(113/163)[61.87-75.89]</td><td colspan="1" rowspan="1">TEM:68.2%(15/22)[47.32-83.64]</td><td colspan="1" rowspan="1">TEM:94.2%(113/120)[88.45-97.15]</td><td colspan="1" rowspan="1">TEM:23.1%(15/65)[14.51-34.64]</td><td colspan="1" rowspan="1">TEM:68.7%(1179/1716) [66.47-70.86]</td><td colspan="1" rowspan="1">TEM:74.7%(233/312)[69.57-79.18]</td><td colspan="1" rowspan="1">TEM:93.7%(1179/1258) [92.24-94.93]</td><td colspan="1" rowspan="1">TEM:30.3%(233/770)[27.12-3360]</td><td colspan="1" rowspan="1">TEM: 62%(1258/2028)</td><td colspan="1" rowspan="1">TEM:49.7% of7979</td></tr><tr><td colspan="1" rowspan="1">TEM</td><td colspan="1" rowspan="1">K. pneumoniae</td><td colspan="1" rowspan="1">98.6%(214/217)96.01-99.53]</td><td colspan="1" rowspan="1">95.5%(107/112)[89.97-98.08]</td><td colspan="1" rowspan="1">Amp.Sulbactam</td><td colspan="1" rowspan="1">TEM:67.8%(120/177)[60.60-74.24]</td><td colspan="1" rowspan="1">TEM:100.0%(8/8)67.56-10.00]</td><td colspan="1" rowspan="1">TEM:100.0%(120/120)[96.90-0.00]</td><td colspan="1" rowspan="1">TEM:12.3%(8/65)[6.37-22.45]</td><td colspan="1" rowspan="1">TEM:65.6%(1252/1909) [63.42-67.68]</td><td colspan="1" rowspan="1">TEM:95.0%(113/119)[89.44-97.67]</td><td colspan="1" rowspan="1">TEM:99.5%(1252/1258) [98.96-99.78]</td><td colspan="1" rowspan="1">TEM:14.7%(113/770)[12.35-1.35]</td><td colspan="1" rowspan="1">TEM: 62%(1258/2028)</td><td colspan="1" rowspan="1">TEM:49.7% of7979</td></tr><tr><td colspan="1" rowspan="1">TEM</td><td colspan="1" rowspan="1">K. pneumoniae</td><td colspan="1" rowspan="1">98.6%(214/217)[96.01-9.53]</td><td colspan="1" rowspan="1">95.5%(107/112)[89.97-98.08]</td><td colspan="1" rowspan="1">Pip.Tazo</td><td colspan="1" rowspan="1">TEM:70.1%(103/147)[62.23-76.88]</td><td colspan="1" rowspan="1">TEM:55.3%(21/38)[39.71-69.85]</td><td colspan="1" rowspan="1">TEM:85.8%(103/120)[78.48-90.96]</td><td colspan="1" rowspan="1">TEM:32.3%(21/65)[22.20-44.39]</td><td colspan="1" rowspan="1">TEM:67.5%(945/1400)[65.00-69.90]</td><td colspan="1" rowspan="1">TEM:50.2%(315/628)[46.26-54.06]</td><td colspan="1" rowspan="1">TEM:75.1%(945/1258)[72.66-77.43]</td><td colspan="1" rowspan="1">TEM:40.9%(315/770)[37.49-44.42]</td><td colspan="1" rowspan="1">TEM: 62%(1258/2028)</td><td colspan="1" rowspan="1">TEM:49.7% of7979</td></tr><tr><td colspan="1" rowspan="2">AMR GeneTarget</td><td colspan="1" rowspan="2">Organism</td><td colspan="2" rowspan="1">Detection of AMR GeneTarget by AMR GenePanel versus WholeGenome SequencingCombined Clinical DevicePerformance -Stock and ProspectiveIsolates (from K191288)</td><td colspan="1" rowspan="2">Antimicrobial</td><td colspan="4" rowspan="1">Detection of AMR Gene Target by AMR GenePanel vs AST Non-SusceptibilityCombined Clinical Device Performance - Stock andProspective Isolates (from K191288) includingonly unique isolates (no replicates)</td><td colspan="5" rowspan="1">Detection of AMR Gene Target versus AST Non-Susceptibility inOpGen's SMART Study</td><td colspan="1" rowspan="2">%Organismswith AMRGeneTarget inNCBIPathogenDatabase2019)</td></tr><tr><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1">PPV[95% CI]</td><td colspan="1" rowspan="1">NPV[95% CI]</td><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1">PPV[95% CI]</td><td colspan="1" rowspan="1">NPV[95% CI]</td><td colspan="1" rowspan="1">% Organismswith AMRGene Target</td></tr><tr><td colspan="1" rowspan="1">TEM</td><td colspan="1" rowspan="1">P. mirabilis</td><td colspan="1" rowspan="1">100.0%(115/115)[96.77-100.00]</td><td colspan="1" rowspan="1">95.0%(38/40)[83.50-9.62]</td><td colspan="1" rowspan="1">Ampicillin</td><td colspan="1" rowspan="1">TEM:67.2%(43/64)[55.00-77.43]</td><td colspan="1" rowspan="1">TEM:92.9%(13/14)[68.53-98.73]</td><td colspan="1" rowspan="1">TEM:97.7%(43/44)[88.19-99.601]</td><td colspan="1" rowspan="1">TEM:38.2%(13/34)[23.90-54.96]</td><td colspan="1" rowspan="1">TEM:72.0%(603/838)[68.82-74.89]</td><td colspan="1" rowspan="1">TEM:97.5%(311/319)[95.13-98.72]</td><td colspan="1" rowspan="1">TEM:98.7%(603/61)[97.44-99.34]</td><td colspan="1" rowspan="1">TEM:57.0%(311/546)[52.77-61.05]</td><td colspan="1" rowspan="1">TEM: 52.8%(611/1157)</td><td colspan="1" rowspan="1">TEM:18.8% of144</td></tr><tr><td colspan="1" rowspan="1">TEM</td><td colspan="1" rowspan="1">P. mirabilis</td><td colspan="1" rowspan="1">100.0%(115/115)[96.77-100.00]</td><td colspan="1" rowspan="1">95.0%(38/40)[83.50-98.62]</td><td colspan="1" rowspan="1">Amox.K.Clav</td><td colspan="1" rowspan="1">TEM:69.2%(18/26)[50.01-83.50]</td><td colspan="1" rowspan="1">TEM:50.0%(26/52)[36.89-63.11]</td><td colspan="1" rowspan="1">TEM:40.9%(18/44)[27.69-5.59]</td><td colspan="1" rowspan="1">TEM:76.5%(26/34)60.00-8.56]</td><td colspan="1" rowspan="1">TEM:74.9%(269/359)[70.20-79.13]</td><td colspan="1" rowspan="1">TEM:57.1%(456/798)[53.68-60.531</td><td colspan="1" rowspan="1">TEM:44.0%(269/611)[40.14-47.99]</td><td colspan="1" rowspan="1">TEM:83.5%(456/546)[80.17-86.39]</td><td colspan="1" rowspan="1">TEM: 52.8%(611/1157)</td><td colspan="1" rowspan="1">TEM:18.8% of144</td></tr><tr><td colspan="1" rowspan="1">TEM</td><td colspan="1" rowspan="1">P. mirabilis</td><td colspan="1" rowspan="1">100.0%(115/115)[96.77-0.00]</td><td colspan="1" rowspan="1">95.0%(38/40)[83.50-98.62]</td><td colspan="1" rowspan="1">Amp.Sulbactam</td><td colspan="1" rowspan="1">TEM:73.8%(31/42)58.93-84.70]</td><td colspan="1" rowspan="1">TEM:63.9%(23/36)47.58-77.52]</td><td colspan="1" rowspan="1">TEM:70.5%(31/44)[55.7881.84]</td><td colspan="1" rowspan="1">TEM:67.6%(23/34)50.84-80.87]</td><td colspan="1" rowspan="1">TEM:74.7%(435/582)[71.06-78.10]</td><td colspan="1" rowspan="1">TEM:69.4%(399/575)[65.51-73.02]</td><td colspan="1" rowspan="1">TEM:71.2%(435/611)[67.48-74.64]</td><td colspan="1" rowspan="1">TEM:73.1%(399/546)[69.20-76.63]</td><td colspan="1" rowspan="1">TEM: 52.8%(611/1157)</td><td colspan="1" rowspan="1">TEM:18.8% of144</td></tr><tr><td colspan="1" rowspan="1">TEM</td><td colspan="1" rowspan="1">P. mirabilis</td><td colspan="1" rowspan="1">100.0%(115/115)[96.77-100.00]</td><td colspan="1" rowspan="1">95.0%(38/40)[83.50-98.62]</td><td colspan="1" rowspan="1">Pip.Tazo</td><td colspan="1" rowspan="1">TEM:75.0%(3/4)[30.0-95.44]</td><td colspan="1" rowspan="1">TEM:44.6%(33/74)[33.82-55.91]</td><td colspan="1" rowspan="1">TEM:6.8%(3/44)[2.35-18.23]</td><td colspan="1" rowspan="1">TEM:97.1%(33/34)85. 899.48]</td><td colspan="1" rowspan="1">TEM:77.4%(24/31)60.19-88.60]</td><td colspan="1" rowspan="1">TEM:47.9%(539/1126)[44.96-50.79]</td><td colspan="1" rowspan="1">TEM: 3.9%(24/611)[2.65-5.78]</td><td colspan="1" rowspan="1">TEM:98.7%(539/546)[97.38-99.38]</td><td colspan="1" rowspan="1">TEM: 52.8%(611/1157)</td><td colspan="1" rowspan="1">TEM:18.8% of144</td></tr><tr><td colspan="1" rowspan="1">TEM</td><td colspan="1" rowspan="1">P. aeruginosa</td><td colspan="1" rowspan="1">100.0%(4/4)[51.01-100.00]</td><td colspan="1" rowspan="1">99.1%(341/344)[97.47-9.70]</td><td colspan="1" rowspan="1">Pip.Tazo</td><td colspan="1" rowspan="1">TEM:4.8%(5/104)[2.07-10.76]</td><td colspan="1" rowspan="1">TEM:100.0%(5/51)93.00-100.00</td><td colspan="1" rowspan="1">TEM:100.0%(5/556.55-100.00]</td><td colspan="1" rowspan="1">TEM:34.0%(1/150)26.041.901</td><td colspan="1" rowspan="1">TEM: 1.7%(16/941)[1.05-2.74]</td><td colspan="1" rowspan="1">TEM:99.8%(608/609)99.08-99.9</td><td colspan="1" rowspan="1">TEM:94.1%(16/17)[73.02-98.95]</td><td colspan="1" rowspan="1">TEM:39.7%(608/1533)[37.24-42.13]</td><td colspan="1" rowspan="1">TEM: 1.1%(17/1550)</td><td colspan="1" rowspan="1">TEM: 0.4%of 4706</td></tr><tr><td colspan="1" rowspan="1">TEM</td><td colspan="1" rowspan="1">E. asburiae</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">Pip.Tazo</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">TEM:13.7% of73</td></tr><tr><td colspan="1" rowspan="1">TEM</td><td colspan="1" rowspan="1">E. cloacaecomplex</td><td colspan="1" rowspan="1">100.0%(12/12)75.75-10.00]</td><td colspan="1" rowspan="1">100.0%(2/2)[34.24-0.00]</td><td colspan="1" rowspan="1">Pip.Tazo</td><td colspan="1" rowspan="1">TEM:85.7%(12/14)60.0695.99]</td><td colspan="1" rowspan="1">TEM:NaN</td><td colspan="1" rowspan="1">TEM:100.0%(12/12)75.75-100.00]</td><td colspan="1" rowspan="1">TEM:0.0%(0/)[0.0-65.76]</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">TEM:27.2% of563</td></tr><tr><td colspan="1" rowspan="1">TEM</td><td colspan="1" rowspan="1">E. hormaechei</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">Pip.Tazo</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">TEM: 46%of 678</td></tr><tr><td colspan="1" rowspan="2">AMR GeneTarget</td><td colspan="1" rowspan="2">Organism</td><td colspan="2" rowspan="1">Detection of AMR GeneTarget by AMR GenePanel versus WholeGenome SequencingCombined Clinical DevicePerformance -Stock and ProspectiveIsolates (from K191288)</td><td colspan="1" rowspan="1">Antimicrobial</td><td colspan="4" rowspan="1">Detection of AMR Gene Target by AMR GenePanel vs AST Non-SusceptibilityCombined Clinical Device Performance - Stock andProspective Isolates (from K191288) includingonly unique isolates (no replicates)</td><td colspan="5" rowspan="1">Detection of AMR Gene Target versus AST Non-Susceptibility inOpGen's SMART Study</td><td colspan="1" rowspan="2">%Organismswith AMRGeneTarget inNCBIPathogenDatabase(2019)</td></tr><tr><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1">PPV[95% CI]</td><td colspan="1" rowspan="1">NPV[95% CI]</td><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1">PPV[95% CI]</td><td colspan="1" rowspan="1">NPV[95% CI]</td><td colspan="1" rowspan="1">% Organismswith AMRGene Target</td></tr><tr><td colspan="1" rowspan="1">TEM</td><td colspan="1" rowspan="1">E. ludwigii</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">Pip.Tazo</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">TEM: 3.2%of 31</td></tr><tr><td colspan="1" rowspan="1">TEM</td><td colspan="1" rowspan="1">E. kobei</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">Pip.Tazo</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">TEM:25.7% of70</td></tr><tr><td colspan="1" rowspan="1">VEB</td><td colspan="1" rowspan="1">P. mirabilis</td><td colspan="1" rowspan="1">100.0%(73/73)[95.00-00]</td><td colspan="1" rowspan="1">96.3%(79/82)[89.79-98.75]</td><td colspan="1" rowspan="1">Cefazolin</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">NE</td><td colspan="1" rowspan="1">VEB: 8.6%(49/571)[6.55-11.16]</td><td colspan="1" rowspan="1">VEB: 99.5%(583/586)[98.51-9.83]</td><td colspan="1" rowspan="1">VEB: 94.2%(49/52)[84.36-98.02]</td><td colspan="1" rowspan="1">VEB: 52.8%(583/1105)[49.81-5.69]</td><td colspan="1" rowspan="1">VEB: 4.5%(52/1157)</td><td colspan="1" rowspan="1">VEB: 0% of144</td></tr><tr><td colspan="1" rowspan="1">VEB</td><td colspan="1" rowspan="1">P. mirabilis</td><td colspan="1" rowspan="1">100.0%(73/73)[95.00-10.00]</td><td colspan="1" rowspan="1">96.3%(79/82)[89.79-98.75]</td><td colspan="1" rowspan="1">Cefuroxime</td><td colspan="1" rowspan="1">VEB:28.9%(13/45)17.7343.37]</td><td colspan="1" rowspan="1">VEB:97.0%(32/33)[84.68-99.46]</td><td colspan="1" rowspan="1">VEB:92.9%(13/14)68.53-98.73]</td><td colspan="1" rowspan="1">VEB:50.0%(32/64)[38.10-61.90]</td><td colspan="1" rowspan="1">VEB: 10.3%(51/495)[7.92-13.29]</td><td colspan="1" rowspan="1">VEB: 99.8%(661/662)[99.15-9.97]</td><td colspan="1" rowspan="1">VEB: 98.1%(51/52)[89.88-9.66]</td><td colspan="1" rowspan="1">VEB: 59.8%(661/1105)[56.90-2.67]</td><td colspan="1" rowspan="1">VEB: 4.5%(52/1157)</td><td colspan="1" rowspan="1">VEB: 0% of144</td></tr><tr><td colspan="1" rowspan="1">VEB</td><td colspan="1" rowspan="1">P. mirabilis</td><td colspan="1" rowspan="1">100.0%(73/73)95.00-100.00]</td><td colspan="1" rowspan="1">96.3%(79/82)8.79-98.75]</td><td colspan="1" rowspan="1">Cefepime</td><td colspan="1" rowspan="1">VEB:32.5%((13/40)0.0847.98]</td><td colspan="1" rowspan="1">VEB:97.4%(37/38)[86.51-99.53]</td><td colspan="1" rowspan="1">VEB:92.9%(13/14)68.53-98.73]</td><td colspan="1" rowspan="1">VEB:57.8%(37/64)45.61-69.13]</td><td colspan="1" rowspan="1">VEB: 15.0%(50/334)11.54-19.19]</td><td colspan="1" rowspan="1">VEB: 99.8%(821/823)992-99.93]</td><td colspan="1" rowspan="1">VEB: 96.2%(50/52)87.02-98.94]</td><td colspan="1" rowspan="1">VEB: 74.3%(821/1105)[71.64-76.79]</td><td colspan="1" rowspan="1">VEB: 4.5%(52/1157)</td><td colspan="1" rowspan="1">VEB: 0% of144</td></tr><tr><td colspan="1" rowspan="1">VEB</td><td colspan="1" rowspan="1">P. mirabilis</td><td colspan="1" rowspan="1">100.0%(73/73)[95.00-0.00]</td><td colspan="1" rowspan="1">96.3%(79/82)[89.79-8.75]</td><td colspan="1" rowspan="1">Ceftazidime</td><td colspan="1" rowspan="1">VEB:56.5%(13/23)[36.81-74.37]</td><td colspan="1" rowspan="1">VEB:98.2%(4/55)90.3999.68]</td><td colspan="1" rowspan="1">VEB:92.9%(13/14)[68.53-98.73]</td><td colspan="1" rowspan="1">VEB:84.4%(54/64)73.57-91.29]</td><td colspan="1" rowspan="1">VEB: 19.3%(50/259)[14.96-24.55]</td><td colspan="1" rowspan="1">VEB: 99.8%(896/898)[99.19-9.94]</td><td colspan="1" rowspan="1">VEB: 96.2%(50/52)[87.02-98.94]</td><td colspan="1" rowspan="1">VEB: 81.1%(896/1105)[78.67-83.29]</td><td colspan="1" rowspan="1">VEB: 4.5%(52/1157)</td><td colspan="1" rowspan="1">VEB: 0% of144</td></tr><tr><td colspan="1" rowspan="1">VEB</td><td colspan="1" rowspan="1">P. mirabilis</td><td colspan="1" rowspan="1">100.0%(73/73)[95.00-00]</td><td colspan="1" rowspan="1">96.3%(79/82)[89.79-98.75]</td><td colspan="1" rowspan="1">Ceftriaxone</td><td colspan="1" rowspan="1">VEB:31.0%(13/42)19.07-46.0]</td><td colspan="1" rowspan="1">VEB:97.2%(35/36)[85.83-99.51]</td><td colspan="1" rowspan="1">VEB:92.9%(13/14)[68.53-98.73]</td><td colspan="1" rowspan="1">VEB:54.7%(35/64)[42.57-66.27]</td><td colspan="1" rowspan="1">VEB: 10.3%(49/478)[7.84-13.29]</td><td colspan="1" rowspan="1">VEB: 99.6%(676/679)[98.71-99.85]</td><td colspan="1" rowspan="1">VEB: 94.2%(49/52)[84.36-98.02]</td><td colspan="1" rowspan="1">VEB: 61.2%(676/1105)[58.27-64.01]</td><td colspan="1" rowspan="1">VEB: 4.5%(52/1157)</td><td colspan="1" rowspan="1">VEB: 0% of144</td></tr><tr><td colspan="1" rowspan="1">VEB</td><td colspan="1" rowspan="1">P. mirabilis</td><td colspan="1" rowspan="1">100.0%(73/73)[95.00-100.00]</td><td colspan="1" rowspan="1">96.3%(79/82)[89.79-98.75]</td><td colspan="1" rowspan="1">Amox.K.Clav</td><td colspan="1" rowspan="1">VEB:19.2%(5/26)[8.51-37.88]</td><td colspan="1" rowspan="1">VEB:82.7%(43/52)[70.27-90.62]</td><td colspan="1" rowspan="1">VEB:35.7%(5/14)[16.34-61.24]</td><td colspan="1" rowspan="1">VEB:67.2%(43/64)[55.00-77.43]</td><td colspan="1" rowspan="1">VEB: 4.2%(15/359)[2.55-6.78]</td><td colspan="1" rowspan="1">VEB: 95.4%(761/798)[93.67-96.62]</td><td colspan="1" rowspan="1">VEB: 28.8%(15/52)[18.33-42.27]</td><td colspan="1" rowspan="1">VEB: 68.9%(761/1105)[66.08-71.53]</td><td colspan="1" rowspan="1">VEB: 4.5%(52/1157)</td><td colspan="1" rowspan="1">VEB: 0% of144</td></tr><tr><td colspan="1" rowspan="2">AMR GeneTarget</td><td colspan="1" rowspan="2">Organism</td><td colspan="2" rowspan="1">Detection of AMR GeneTarget by AMR GenePanel versus WholeGenome SequencingCombined Clinical DevicePerformance -Stock and ProspectiveIsolates (from K191288)</td><td colspan="1" rowspan="1">Antimicrobial</td><td colspan="4" rowspan="1">Detection of AMR Gene Target by AMR GenePanel vs AST Non-SusceptibilityCombined Clinical Device Performance - Stock andProspective Isolates (rom K191288)includingonly unique isolates (no replicates)</td><td colspan="5" rowspan="1">Detection of AMR Gene Target versus AST Non-Susceptibility inOpGen's SMART Study</td><td colspan="1" rowspan="2">%Organismswith AMRGeneTarget inNCBIPathogenDatabase(2019)</td></tr><tr><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1">PPV[95% CI]</td><td colspan="1" rowspan="1">NPV[95% CI]</td><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1">PPV[95% CI]</td><td colspan="1" rowspan="1">NPV[95% CI]</td><td colspan="1" rowspan="1">% Organismswith AMRGene Target</td></tr><tr><td colspan="1" rowspan="1">VEB</td><td colspan="1" rowspan="1">P. mirabilis</td><td colspan="1" rowspan="1">100.0%(73/73)[95.00-00]</td><td colspan="1" rowspan="1">96.3%(79/82)[89.79-98.75]</td><td colspan="1" rowspan="1">Amp.Sulbactam</td><td colspan="1" rowspan="1">VEB:21.4%(9/42)11.71-35.94]</td><td colspan="1" rowspan="1">VEB:86.1%(31/36)[71.34-93.92]</td><td colspan="1" rowspan="1">VEB:64.3%(9/14)[38.76-83.66]</td><td colspan="1" rowspan="1">VEB:48.4%(31/64)[36.63-60.42]</td><td colspan="1" rowspan="1">VEB: 4.5%(26/582)[3.07-6.47]</td><td colspan="1" rowspan="1">VEB: 95.5%(549/575)[93.46-96.90]</td><td colspan="1" rowspan="1">VEB: 50.0%(26/52)[36.89-63.11]</td><td colspan="1" rowspan="1">VEB: 49.7%(549/1105)[46.74-52.63]</td><td colspan="1" rowspan="1">VEB: 4.5%(52/1157)</td><td colspan="1" rowspan="1">VEB: 0% of144</td></tr><tr><td colspan="1" rowspan="1">VEB</td><td colspan="1" rowspan="1">P. mirabilis</td><td colspan="1" rowspan="1">100.0%(73/73)[95.00-100.00]</td><td colspan="1" rowspan="1">96.3%(79/82)[89.79-98.75]</td><td colspan="1" rowspan="1">Pip.Tazo</td><td colspan="1" rowspan="1">VEB: 0.0%(0/4)[0.00-4.99]</td><td colspan="1" rowspan="1">VEB:81.1%(60/74)[70.71-88.38]</td><td colspan="1" rowspan="1">VEB: 0.0%(0/14)[0.00-21.53]</td><td colspan="1" rowspan="1">VEB:93.8%(60/64)85 00-97.54]</td><td colspan="1" rowspan="1">VEB: 6.5%(2/31)[1.79-20.72]</td><td colspan="1" rowspan="1">VEB: 95.6%(1076/1126) [94.19-96.62]</td><td colspan="1" rowspan="1">VEB: 3.8%(2/52)[1.06-12.98]</td><td colspan="1" rowspan="1">VEB: 97.4%(1076/1105) [96.26-98.17]</td><td colspan="1" rowspan="1">VEB: 4.5%(52/1157)</td><td colspan="1" rowspan="1">VEB: 0% of144</td></tr><tr><td colspan="1" rowspan="1">VEB</td><td colspan="1" rowspan="1">P. aeruginosa</td><td colspan="1" rowspan="1">100.0%(16/16)[80.64-100]</td><td colspan="1" rowspan="1">100.0%(332/332)[98.86-100.00]</td><td colspan="1" rowspan="1">Cefepime</td><td colspan="1" rowspan="1">VEB:10.6%(13/123)[6.28-17.25]</td><td colspan="1" rowspan="1">VEB:100.0%(32/32)89.80.00]</td><td colspan="1" rowspan="1">VEB:100.0%(13/13)[77.100.00]</td><td colspan="1" rowspan="1">VEB:22.5%(32/142)(16.44-30.08]</td><td colspan="1" rowspan="1">VEB: 3.1%(33/1068)[2.21-4.31]</td><td colspan="1" rowspan="1">VEB: 99.8%(481/482)[98.83-9.96]</td><td colspan="1" rowspan="1">VEB: 97.1%(33/34)[85.08-99.48]</td><td colspan="1" rowspan="1">VEB: 31.7%(481/1516)[29.43-34.11]</td><td colspan="1" rowspan="1">VEB: 2.2%(34/1550)</td><td colspan="1" rowspan="1">VEB: 0.1%of 4706</td></tr><tr><td colspan="1" rowspan="1">VEB</td><td colspan="1" rowspan="1">P. aeruginosa</td><td colspan="1" rowspan="1">100.0%(16/16)[8064-100.00]</td><td colspan="1" rowspan="1">100.0%(332/332)[98.86-100.00]</td><td colspan="1" rowspan="1">Ceftazidime</td><td colspan="1" rowspan="1">VEB:11.4%(13/114)[6.79-18.54]</td><td colspan="1" rowspan="1">VEB:100.0%(41/41)[91.43-100.00]</td><td colspan="1" rowspan="1">VEB:100.0%(13/13)[77.19-1000.00]</td><td colspan="1" rowspan="1">VEB:28.9%(41/142)[22.05-36.81]</td><td colspan="1" rowspan="1">VEB: 3.4%(33/959)[2.46-4.79]</td><td colspan="1" rowspan="1">VEB: 99.8%(590/591)99.05-99.97]</td><td colspan="1" rowspan="1">VEB: 97.1%(33/34)85.08-99.48]</td><td colspan="1" rowspan="1">VEB: 38.9%(590/1516)[36.49-41.40]</td><td colspan="1" rowspan="1">VEB: 2.2%(34/1550)</td><td colspan="1" rowspan="1">VEB: 0.1%of 4706</td></tr><tr><td colspan="1" rowspan="1">VEB</td><td colspan="1" rowspan="1">P. aeruginosa</td><td colspan="1" rowspan="1">100.0%(16/16)[80.64-100.00]</td><td colspan="1" rowspan="1">100.0%(332/332)[98.86-10.00]</td><td colspan="1" rowspan="1">Pip.Tazo</td><td colspan="1" rowspan="1">VEB: 6.7%(7/104)[3.30-13.25]</td><td colspan="1" rowspan="1">VEB:88.2%(45/51)[76.62-94.49]</td><td colspan="1" rowspan="1">VEB:53.8%(7/13)[29.14-76.79]</td><td colspan="1" rowspan="1">VEB:31.7%(45/142)[24.61-39.74]</td><td colspan="1" rowspan="1">VEB: 2.3%(22/941)[1.55-3.51]</td><td colspan="1" rowspan="1">VEB: 98.0%(597/609)[96.59-8.87]</td><td colspan="1" rowspan="1">VEB: 64.7%(22/34)[47.91-78.51]</td><td colspan="1" rowspan="1">VEB: 39.4%(597/1516)[36.95-411.86]</td><td colspan="1" rowspan="1">VEB: 2.2%(34/1550)</td><td colspan="1" rowspan="1">VEB: 0.1%of 4706</td></tr><tr><td colspan="1" rowspan="1">VIM</td><td colspan="1" rowspan="1">P. mirabilis</td><td colspan="1" rowspan="1">NaN</td><td colspan="1" rowspan="1">100.0%(155/155)[97.58-10.00]</td><td colspan="1" rowspan="1">Ertapenem</td><td colspan="1" rowspan="1">VIM:0.0%(0/4)0.00-48.99]</td><td colspan="1" rowspan="1">VIM:100.0%74/74)[95.07-0.00]</td><td colspan="1" rowspan="1">VIM: NaN</td><td colspan="1" rowspan="1">VIM:94.9%(74/78)[87.54-97.99]</td><td colspan="1" rowspan="1">VIM: 6.2%(1/16)[1.11-28.33]</td><td colspan="1" rowspan="1">VIM:99.7%(1138/1141) [99.23-99.91]</td><td colspan="1" rowspan="1">VIM:25.0%(1/4)[4.56-69.94]</td><td colspan="1" rowspan="1">VIM:98.7%(1138/1153) [97.86-99.21]</td><td colspan="1" rowspan="1">VIM: 0.3%(4/1157)</td><td colspan="1" rowspan="1">VIM: 0% of144</td></tr><tr><td colspan="1" rowspan="1">VIM</td><td colspan="1" rowspan="1">P. mirabilis</td><td colspan="1" rowspan="1">NaN</td><td colspan="1" rowspan="1">100.0%(155/155)[97.58-100.00]</td><td colspan="1" rowspan="1">Meropenem</td><td colspan="1" rowspan="1">VIM:0.0%(0/3)0.00-56.15]</td><td colspan="1" rowspan="1">VIM:100.0% (75/75)9513-10.00]</td><td colspan="1" rowspan="1">VIM: NaN</td><td colspan="1" rowspan="1">VIM:96.2%(75/78)89.2998.68]</td><td colspan="1" rowspan="1">VIM: 5.9%(1/17)[1.05-26.98]</td><td colspan="1" rowspan="1">VIM:99.7%(1137/114) (99.23-99.91]</td><td colspan="1" rowspan="1">VIM:25.0%(1/4)[4.56-69.94]</td><td colspan="1" rowspan="1">VIM:98.6%(1137/115) (97.76-99.14</td><td colspan="1" rowspan="1">VIM: 0.3%(4/1157)</td><td colspan="1" rowspan="1">VIM: 0% of144</td></tr><tr><td colspan="1" rowspan="2">AMR GeneTarget</td><td colspan="1" rowspan="2">Organism</td><td colspan="2" rowspan="1">Detection of AMR GeneTarget by AMR GenePanel versus WholeGenome SequencingCombined Clinical DevicePerformance -Stock and ProspectiveIsolates (from K191288)</td><td colspan="1" rowspan="2">Antimicrobial</td><td colspan="4" rowspan="1">Detection of AMR Gene Target by AMR GenePanel vs AST Non-SusceptibilityCombined Clinical Device Performance - Stock andProspective Isolates (from K191288) includingonly unique isolates (no replicates)</td><td colspan="5" rowspan="1">Detection of AMR Gene Target versus AST Non-Susceptibility inOpGen's SMART Study</td><td colspan="1" rowspan="2">%Organismswith AMRGeneTarget inNCBIPathogenDatabase(2019)</td></tr><tr><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1">PPV[95% CI]</td><td colspan="1" rowspan="1">NPV[95% CI]</td><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1">PPV[95% CI]</td><td colspan="1" rowspan="1">NPV[95% CI]</td><td colspan="1" rowspan="1">% Organismswith AMRGene Target</td></tr><tr><td colspan="1" rowspan="1">VIM</td><td colspan="1" rowspan="1">P. aeruginosa</td><td colspan="1" rowspan="1">97.8%(91/93)[92.49-99.41]</td><td colspan="1" rowspan="1">99.6%(254/255)[97.81-99.93]</td><td colspan="1" rowspan="1">Imipenem</td><td colspan="1" rowspan="1">VIM:18.5%(22/119)[12.54-26.41]</td><td colspan="1" rowspan="1">VIM:100.0%(36/36)90.6-100.00]</td><td colspan="1" rowspan="1">VIM:100.0%(22/22)[85.13-100.00]</td><td colspan="1" rowspan="1">VIM:27.1%(36/133)[20.24-35.18]</td><td colspan="1" rowspan="1">VIM:19.0%(190/1002)[16.66-21.51]</td><td colspan="1" rowspan="1">VIM:99.6%(546/548)[98.68-99.90]</td><td colspan="1" rowspan="1">VIM:99.0%(190/192)[96.28-99.71]</td><td colspan="1" rowspan="1">VIM:40.2%(546/1358)[37.63-42.84]</td><td colspan="1" rowspan="1">VIM: 12.4%(192/1550)</td><td colspan="1" rowspan="1">VIM: 9.5%of 4706</td></tr><tr><td colspan="1" rowspan="1">VIM</td><td colspan="1" rowspan="1">P. aeruginosa</td><td colspan="1" rowspan="1">97.8%(91/93)[92.49-99.41]</td><td colspan="1" rowspan="1">99.6%(254/255)[97.81-99.93]</td><td colspan="1" rowspan="1">Meropenem</td><td colspan="1" rowspan="1">VIM:18.6%(22/118)[12.65-26.62]</td><td colspan="1" rowspan="1">VIM:100.0%(37/37)[90.59-0.00]</td><td colspan="1" rowspan="1">VIM:100.0%(22/22)[85.13-100.00]</td><td colspan="1" rowspan="1">VIM:27.8%(37/133)[20.91-35.98]</td><td colspan="1" rowspan="1">VIM:19.2%(189/986)[16.83-21.74]</td><td colspan="1" rowspan="1">VIM:99.5%(561/564)[98.45-99.82]</td><td colspan="1" rowspan="1">VIM:98.4%(189/192)[95.51-9947]</td><td colspan="1" rowspan="1">VIM:41.3%(561/1358)[38.72-43.95]</td><td colspan="1" rowspan="1">VIM: 12.4%(192/1550)</td><td colspan="1" rowspan="1">VIM: 9.5%of 4706</td></tr><tr><td colspan="1" rowspan="1">GyrAmutant</td><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">98.2%(160/163)[94.73-9.37]</td><td colspan="1" rowspan="1">99.4%(167/168)[96.71-99.89]</td><td colspan="1" rowspan="1">Ciprofloxacin</td><td colspan="1" rowspan="1">E. coligyrAMutant:90.7%(78/86)[82.70-95.21]</td><td colspan="1" rowspan="1">E. coligyrAMutant:98.6%(68/69)[92.24-99.74]</td><td colspan="1" rowspan="1">E. coligyrAMutant:98.7%(78/79)[93.17-99.78]</td><td colspan="1" rowspan="1">E. coligyrAMutant:89.5%(68/76)[80.58-4.57]</td><td colspan="1" rowspan="1">gyrAMutant:89.5%(1808/202) [88.09-90.77]</td><td colspan="1" rowspan="1">gyrAMutant:93.7%(881/940)[91.99-95.10]</td><td colspan="1" rowspan="1">gyrAMutant:96.8%(1808/1867) [95.95-97.54]</td><td colspan="1" rowspan="1">gyrAMutant:80.6%(881/1093)[78.15-82.84]</td><td colspan="1" rowspan="1">gyrAMutant:63.1%(1867/2960)</td><td colspan="1" rowspan="1">_3</td></tr><tr><td colspan="1" rowspan="1">GyrAmutant</td><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">98.2%(160/163)[94.73-99.37]</td><td colspan="1" rowspan="1">99.4%(167/168)[96.71-99.89]</td><td colspan="1" rowspan="1">Levofloxacin</td><td colspan="1" rowspan="1">E. coligyrAMutant:92.8%(77/83)[85.11-96.64]</td><td colspan="1" rowspan="1">E. coligyrAMutant:97.2%(70/72)[90.43-99.23]</td><td colspan="1" rowspan="1">E. coligyrAMutant:97.5%(77/79)[91.23-99.30]</td><td colspan="1" rowspan="1">E. coligyrAMutant:92.1%(70/76)[83.83-96.33]</td><td colspan="1" rowspan="1">gyrAMutant:90.6%(1803/198) [89.29-91.85]</td><td colspan="1" rowspan="1">gyrAMutant:93.4%(907/971)[91.67-94.80]</td><td colspan="1" rowspan="1">gyrAMutant:96.6%(1803/1867) [95.65-97.31]</td><td colspan="1" rowspan="1">gyrAMutant:83.0%(907/1093)[80.64-8509]</td><td colspan="1" rowspan="1">gyrAMutant:63.1%(1867/2960)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">GyrAmutant</td><td colspan="1" rowspan="1">P. aeruginosa</td><td colspan="1" rowspan="1">95.0%(265/279)[91.75-96.99]</td><td colspan="1" rowspan="1">98.5%(67/68)[92.13-999.74]</td><td colspan="1" rowspan="1">Ciprofloxacin</td><td colspan="1" rowspan="1">P.aeruginosa gyrAMutant:81.3%(100/123)[73.50-87.20]</td><td colspan="1" rowspan="1">P.aeruginosa gyrAMutant:100.0%(31/31)[88.97-100.00]</td><td colspan="1" rowspan="1">P.aeruginosa gyrAMutant:100.0%(100/100)[96.30-100.00]</td><td colspan="1" rowspan="1">P.aeruginosa gyrAMutant:57.4%(31/54)[44.16-69.67]</td><td colspan="1" rowspan="1">gyrAMutant:79.9%(834/1044)[77.35-82.21]</td><td colspan="1" rowspan="1">gyrAMtant:85.0%(430/506)[81.60-87.83]</td><td colspan="1" rowspan="1">gyrAMutant:91.6%(834/910)[89.67-93.28]</td><td colspan="1" rowspan="1">gyrAMMutant:67.2%(430/640)[63.46-70.71]</td><td colspan="1" rowspan="1">gyrAMuant:58.7%(910/1550)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="2">AMR GeneTarget</td><td colspan="1" rowspan="2">Organism</td><td colspan="2" rowspan="1">Detection of AMR GeneTarget by AMR GenePanel versus WholeGenome SequencingCombined Clinical DevicePerformance -Stock and ProspectiveIsolates (from K191288)</td><td colspan="1" rowspan="1">Antimicrobial</td><td colspan="4" rowspan="1">Detection of AMR Gene Target by AMR GenePanel vs AST Non-SusceptibilityCombined Clinical Device Performance - Stock andProspective Isolates (from K191288) includingonly unique isolates (no replicates)</td><td colspan="5" rowspan="1">Detection of AMR Gene Target versus AST Non-Susceptibility inOpGen's SMART Study</td><td colspan="1" rowspan="2">%Organismswith AMRGeneTarget inNCBIPathogenDatabase(2019)</td></tr><tr><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1">PPV[95% CI]</td><td colspan="1" rowspan="1">NPV[95% CI]</td><td colspan="1" rowspan="1">PPA[95% CI]</td><td colspan="1" rowspan="1">NPA[95% CI]</td><td colspan="1" rowspan="1">PPV[95% CI]</td><td colspan="1" rowspan="1">NPV[95% CI]</td><td colspan="1" rowspan="1">% Organismswith AMRGene Target</td></tr><tr><td colspan="1" rowspan="1">GyrAmutant</td><td colspan="1" rowspan="1">P. aeruginosa</td><td colspan="1" rowspan="1">95.0%(265/279)[91.75-96.99]</td><td colspan="1" rowspan="1">98.5%(67/68)92139974]</td><td colspan="1" rowspan="1">Levofloxacin</td><td colspan="1" rowspan="1">P.aeruginosa gyrAMutant:82.0%(100/122)[74.20-87.78]</td><td colspan="1" rowspan="1">P.aeruginosa gyrAMutant:100.0%(32/32)[89.28-100.00]</td><td colspan="1" rowspan="1">P.aeruginosa gyrAMutant:100.0%(100/100)[96.30-100.00]</td><td colspan="1" rowspan="1">P.aeruginosa gyrAMMutant:59.3%(32/54)[45.97-71.32]</td><td colspan="1" rowspan="1">gyrAMutant:77.7%(835/1074)[75.16-80.13]</td><td colspan="1" rowspan="1">gyrAMutant:84.2%(401/476)[80.70-87.24]</td><td colspan="1" rowspan="1">gyrAMutant:91.8%(835/910)[89.79-93.37]</td><td colspan="1" rowspan="1">gyrAMutant:62.7%(401/640)[58.84-66.32]</td><td colspan="1" rowspan="1">gyrAMutant:58.7%(910/1550)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">vanA</td><td colspan="1" rowspan="1">E. faecalis</td><td colspan="1" rowspan="1">100.0%(57/57)[93.69-0.00]</td><td colspan="1" rowspan="1">100.0%(36/36)[90.36-100.00]</td><td colspan="1" rowspan="1">Vancomycin</td><td colspan="1" rowspan="1">vanA:66.7%(8/12)[39.06-86.19]</td><td colspan="1" rowspan="1">vanA:100.0%(31/31)[88.97-100.00]</td><td colspan="1" rowspan="1">vanA:100.0%(8/8)[67.56-100.00]</td><td colspan="1" rowspan="1">vanA:88.6%(31/35)[74.05-95.46]</td><td colspan="1" rowspan="1">NotEvaluated</td><td colspan="1" rowspan="1">NotEvaluated</td><td colspan="1" rowspan="1">NotEvaluated</td><td colspan="1" rowspan="1">NotEvaluated</td><td colspan="1" rowspan="1">NotEvaluated</td><td colspan="1" rowspan="1">.</td></tr></table>

1NE means not evaluated. 2NaN indicates not-a-number/divide-by-zero. 3 (-) indicates not assessed for NCBI database.

2. Clinical Specificity: See Clinical Sensitivity above. 3. Other Clinical Supportive Data (When 1. and 2. Are Not Applicable): N/A   
D Clinical Cut-Off: N/A   
E Expected Values/Reference Range: N/A   
F Other Supportive Instrument Performance Characteristics Data: N/A

# VIII Proposed Labeling:

The labeling supports the finding of substantial equivalence for this device.

Considering the collective evidence from various sources (e.g., clinical studies, analytical studies, literature, databases, and in silico analyses), the Acuitas AMR Gene Panel Electronic User Guide (EUG) provides additional supporting information about the performance characteristics and interpretation of results for the Acuitas AMR Gene Panel. A breakdown of AMR gene results versus WGS per reportable species and association with resistance are available in the Acuitas AMR Gene Panel EUG, which is an extension of the Acuitas AMR Gene Panel package insert. The Acuitas AMR Gene Panel EUG can be accessed at the company’s website.

# IX Conclusion:

The submitted information in this premarket notification is complete and supports a substantial equivalence decision.