# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATION DECISION SUMMARY

I Background Information:

# B Applicant

# C Proprietary and Established Names

Unyvero Lower Respiratory Tract (LRT) BAL Application

# D Regulatory Information

<table><tr><td rowspan=1 colspan=1>ProductCode(s)</td><td rowspan=1 colspan=1>Classification</td><td rowspan=1 colspan=1>RegulationSection</td><td rowspan=1 colspan=1>Panel</td></tr><tr><td rowspan=1 colspan=1>QBH</td><td rowspan=1 colspan=1>Class II</td><td rowspan=1 colspan=1>21 CFR 866.3985 - DeviceTo Detect And IdentifyMicroorganisms AndAssociated ResistanceMarker Nucleic AcidsDirectly In RespiratorySpecimens</td><td rowspan=1 colspan=1>MI - Microbiology</td></tr></table>

# II Submission/Device Overview:

# A Purpose for Submission:

Clearance of the Unyvero Lower Respiratory Tract (LRT) BAL Application for use on the Unyvero System.

# B Measurand:

Nucleic acid sequences from the following microorganisms and associated resistance markers.

<table><tr><td rowspan=1 colspan=1>Microorganism Targets</td><td rowspan=1 colspan=1>Antibiotic Resistance Marker Targets</td></tr><tr><td rowspan=1 colspan=1>Acinetobacter spp.</td><td rowspan=1 colspan=1>ctx-M (blactx-M, subgroup 1 only)</td></tr><tr><td rowspan=1 colspan=1>Chlamydia pneumoniae</td><td rowspan=1 colspan=1>kpc (blakpc)</td></tr><tr><td rowspan=1 colspan=1>Citrobacter freundii</td><td rowspan=1 colspan=1>mecA</td></tr><tr><td rowspan=1 colspan=1>Escherichia coli</td><td rowspan=1 colspan=1>ndm (blaNDM)</td></tr><tr><td rowspan=1 colspan=1>Enterobacter cloacae complex</td><td rowspan=1 colspan=1>oxa-23 (blaoxA-23)</td></tr><tr><td rowspan=1 colspan=1>Haemophilus influenzae</td><td rowspan=1 colspan=1>oxa-24 (blaoxA-24)</td></tr><tr><td rowspan=1 colspan=1>Klebsiella oxytoca</td><td rowspan=1 colspan=1>oxa-48 (blaoxA-48)</td></tr><tr><td rowspan=1 colspan=1>Klebsiella pneumoniae</td><td rowspan=1 colspan=1>oxa-58 (blaoxA-58)</td></tr><tr><td rowspan=1 colspan=1>Klebsiella variicola</td><td rowspan=1 colspan=1>tem (blaTEM)</td></tr><tr><td rowspan=1 colspan=1>Legionella pneumophila</td><td rowspan=1 colspan=1>vim (blavIM)</td></tr><tr><td rowspan=1 colspan=1>Moraxella catarrhalis</td><td rowspan=9 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Morganella morganii</td></tr><tr><td rowspan=1 colspan=1>Mycoplasma pneumoniae</td></tr><tr><td rowspan=1 colspan=1>Proteus spp.</td></tr><tr><td rowspan=1 colspan=1>Pseudomonas aeruginosa</td></tr><tr><td rowspan=1 colspan=1>Serratia marcescens</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus aureus</td></tr><tr><td rowspan=1 colspan=1>Stenotrophomonas maltophilia</td></tr><tr><td rowspan=1 colspan=1>Streptococcus pneumoniae</td></tr></table>

# C Type of Test:

Multiplex molecular assay for detection of lower respiratory pathogens and associated resistance markers.

# III Intended Use/Indications for Use:

# A Intended Use:

The Unyvero LRT BAL Application is a qualitative nucleic acid multiplex test intended for the simultaneous detection and identification of nucleic acid sequences from the following microorganisms $( \mathrm { N } = 2 0 )$ ) and antibiotic resistance markers $( \mathrm { N } = 1 0 )$ ) in bronchoalveolar lavage (BAL)-like specimens (BAL or mini-BAL) from adult hospitalized patients with suspected lower respiratory tract infections.

<table><tr><td rowspan=1 colspan=1>Microorganism</td><td rowspan=1 colspan=1>Associated Antibiotic Resistance Marker</td></tr><tr><td rowspan=1 colspan=1>Acinetobacter spp. a</td><td rowspan=1 colspan=1>ctx-M b, kpc, ndm, oxa-23, oxa-24, oxa-58, vim</td></tr><tr><td rowspan=1 colspan=1>Chlamydia pneumoniae</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>Citrobacter freundii</td><td rowspan=1 colspan=1>ctx-M b, kpc, ndm, oxa-48, vim</td></tr><tr><td rowspan=1 colspan=1>Enterobacter cloacae complex </td><td rowspan=1 colspan=1>ctx-M b, kpc, ndm, oxa-48, vim</td></tr><tr><td rowspan=1 colspan=1>Escherichia coli</td><td rowspan=1 colspan=1>ctx-M b, kpc, ndm, oxa-48, vim</td></tr><tr><td rowspan=1 colspan=1>Haemophilus influenzae</td><td rowspan=1 colspan=1>tem</td></tr><tr><td rowspan=1 colspan=1>Klebsiella oxytoca</td><td rowspan=1 colspan=1>ctx-M b, kpc, ndm, oxa-48, vim</td></tr><tr><td rowspan=1 colspan=1>Klebsiella pneumoniae d</td><td rowspan=1 colspan=1>ctx-M b, kpc, ndm, oxa-48, vim</td></tr><tr><td rowspan=1 colspan=1>Klebsiella variicola</td><td rowspan=1 colspan=1>ctx-M b, kpc, ndm, oxa-48, vim</td></tr><tr><td rowspan=1 colspan=1>Legionella pneumophila</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>Moraxella catarrhalis</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>Morganella morganii</td><td rowspan=1 colspan=1>ctx-M b, kpc, ndm, oxa-48, vim</td></tr><tr><td rowspan=1 colspan=1>Mycoplasma pneumoniae</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>Pneumocystis jirovecii</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>Proteus spp. </td><td rowspan=1 colspan=1>ctx-M b, kpc, ndm, oxa-48, vim</td></tr><tr><td rowspan=1 colspan=1>Pseudomonas aeruginosa</td><td rowspan=1 colspan=1>ctx-M b, kpc, ndm, vim</td></tr><tr><td rowspan=1 colspan=1>Serratia marcescens</td><td rowspan=1 colspan=1>ctx-M b, kpc, ndm, oxa-48, vim</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus aureus</td><td rowspan=1 colspan=1>mecA</td></tr><tr><td rowspan=1 colspan=1>Stenotrophomonas maltophilia</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>Streptococcus pneumoniae</td><td rowspan=1 colspan=1>-</td></tr></table>

a Acinetobacter spp. includes: A. baumannii, A. calcoaceticus, A. haemolyticus, A. junii, A. lwoffii, A. nosocomialis, A. parvus, A. pittii (detected by LRT BAL Application) and $A$ . ursingii (not detected by LRT BAL Application). b ctx-M1 subgroup. c Enterobacter cloacae complex includes: $E$ . asburiae, $E$ . chengduensis, $E$ . chuandaensis, $E$ . cloacae, E. hormaechei (incl. ssp. xiangfangensis), E. kobei, E. ludwigii, $E$ . roggenkampii, $E .$ . sichuanensis as well as $E .$ . bugandensis (not yet recognized as member of the $E$ . cloacae complex). d Klebsiella pneumoniae includes two variants: $K .$ . pneumoniae (variant 1), and $K .$ . quasipneumoniae (variant 2). e Proteus spp. includes $P$ . cibarius, P. hauseri, P. mirabilis, P. penneri and $P$ . vulgaris.

The Unyvero LRT BAL Application performed on the Unyvero System is indicated as an aid in the diagnosis of lower respiratory tract infection in adult hospitalized patients with signs and symptoms of lower respiratory infection; results should be used in conjunction with other clinical and laboratory findings. As BAL specimens may contain colonizing microorganisms, detection of Unyvero LRT BAL microbial targets does not indicate that the microorganism is the cause of the disease. Unyvero positive results do not rule out co-infection with other microorganisms. Negative results do not preclude lower respiratory infection, as the causative agent may be a microorganism not detected by this test.

A negative result for any antibiotic resistance marker does not indicate that detected microorganisms are susceptible to applicable antimicrobial agents. Detected antibiotic resistance markers cannot be definitively linked to specific microorganisms, and may be present in organisms that are not detected by the Unyvero LRT BAL Application.

Microbiology cultures of BALs should be performed to obtain isolates for species identification and antimicrobial susceptibility testing and to identify potential microorganisms not targeted by the Unyvero LRT BAL Application.

B Indication(s) for Use: Same as Intended Use   
C Special Conditions for Use Statement(s): Rx - For Prescription Use Only   
D Special Instrument Requirements: Unyvero System, including the Unyvero Lysator, Analyzer and Cockpit

# IV Device/System Characteristics:

# A Device Description:

The Unyvero LRT BAL Application integrates DNA purification and eight parallel multiplex endpoint PCR reactions, each followed by array hybridization with all reactions performed simultaneously in a single use cartridge. The assay is performed with bronchial alveolar lavage (BAL or mini-BAL) specimens and provides qualitative detection of nucleic acids from 20 lower respiratory pathogens and 10 antibiotic resistance markers. Multiplex reagent compositions are designed to limit the occurrence of certain analytes within the same multiplex assay and to reduce the potential for competitive inhibition. Individual analyte assays of the Unyvero LRT Array oligonucleotides are designed for similar hybridization and melting temperatures (approx. $6 5 - 8 0 ^ { \circ } \mathrm { C }$ , varying by amplicon). Hybridization and melting temperatures are used to exclude non-specific hybridization signals for improved signal specificity.

The Unyvero instrumentation consists of one (or more) Unyvero Lysators, one (or more) Unyvero Analyzers, a Unyvero Cockpit. Four single-use consumables are required for testing: the Unyvero LRT BAL Cartridge, the Unyvero Sample Tube, Sample Tube Cap and the Unyvero Master Mix. A Unyvero Sample Tube Holder is also supplied for use in the specimen inoculation step. The four assay consumables consist of the following:

Unyvero LRT BAL Cartridge contains DNA isolation and purification reagents, primers, hybridization and wash buffers, and oligonucleotides for detection. Unyvero T1 Sample Tube, contains glass beads and buffers to lyse microorganisms and liquefy the specimen.

Unyvero T1 Sample Tube Cap seals the Unyvero Sample Tube and contains Proteinase K and a synthetic control gene for process monitoring.

• Unyvero M1 Master Mix Tube contains reagents for DNA amplification.

# Controls

An internal control (a synthetic gene with no known homology to assay target sequences) is processed in each of the eight PCR chambers. Results are used to verify adequate DNA purification, amplification, array hybridization, and detection.

Other than the built-in controls, external control materials are not supplied with Unyvero LRT BAL Application devices and consumables. The assay labeling recommends running external positive and negative controls regularly. Suggested positive controls included characterized specimens inoculated with well-characterized microorganisms.

# Reagents

No additional reagents are required to perform the Unyvero LRT BAL Application; all reagents are supplied within the cartridge or within the other consumables with the exception of the polymerase Master Mix, which is provided separately (frozen).

# B Principle of Operation:

The Unyvero LRT BAL Application uses a multiplex PCR approach followed by array hybridization. The Unyvero LRT Application is a qualitative PCR-based assay that detects DNA sequences of microorganisms and associated resistance markers in BAL specimens. The assay includes specimen processing (lysis), DNA extraction and isolation, multiplex PCR with eight parallel multiplex endpoint PCR reactions, and qualitative detection of amplified DNA products using hybridization arrays.

A BAL specimen is first pipetted into the Unyvero Sample Tube and closed with the Unyvero Sample Tube Cap. Closing the sample tube automatically adds the lysis reagent and the internal control gene template to the specimen. The sample tube which fits into the Unyvero Lysator only if closed with the Unyvero sample tube cap is then placed on the Lysator. After the specimen is lysed in the Lysator, the Sample Tube and Master Mix are loaded into the Unyvero LRT Cartridge which is then inserted into the position assigned by the Unyvero Analyzer for automated processing and analysis.

In the Unyvero LRT Cartridge, the remaining testing steps are automated by the Unyvero Analyzer. The lysed specimen is further processed and then transferred onto a DNA purification column for nucleic acid extraction and elution of genomic DNA. Eluted DNA is transferred to a chamber, where mixing with the Master Mix takes place. This mixture is distributed into eight separate PCR reaction chambers each containing multiple primer pairs. After amplification, PCR products are hybridized to the corresponding array probes. Multiple spots per array allow for redundant detection with at least four spots per analyte, as well as spots for intensity calibration, and orientation markers for the imaging software. Binding of amplicons to specific probes is detected by analyzing fluorescence images of each separate array. Result data are transferred to the Unyvero Cockpit for visualization and result printout. A test run is completed after approximately 4.5 hours, and results for panel microorganisms and associated antibiotic resistance markers are displayed on the Unyvero Cockpit screen.

# C Instrument Description Information:

<table><tr><td rowspan=1 colspan=1>Modes of Operation</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>No</td></tr><tr><td rowspan=1 colspan=1>Does the applicant&#x27;s device contain the ability to transmit data to a computer,webserver, or mobile device?</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Does the applicant&#x27;s device transmit data to a computer, webserver, or mobile deviceusing wireless transmission?</td><td rowspan=1 colspan=1>□</td><td rowspan=1 colspan=1>X</td></tr><tr><td rowspan=1 colspan=1>Software</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>FDA has reviewed applicant&#x27;s Hazard Analysis and software development processesfor this line of product types.</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>□</td></tr></table>

1. Instrument Name: Unyvero System

2. Specimen Identification: Specimen Identification information can be manually entered or automatically entered using the integrated barcode reader.

3. Specimen Sampling and Handling:

Before starting the test, the user scans the clinical identification (barcode) from the primary specimen container using the built-in barcode reader of the Unyvero Cockpit or the information may be entered manually on the cockpit on-screen keyboard.

The specimen is initially vortexed and then manually pipetted into the Unyvero Sample Tube. After the specimen is placed in the Sample Tube, the user then places the Unyvero Cap on the Sample Tube, scans the Sample Tube barcode and places it in the Unyvero Lysator. After processing on the Lysator, the user places the Sample Tube and thawed Mastermix into the Unyvero LRT Cartridge, scans the Cartridge barcode and places it into the indicated position in the Unyvero Analyzer. The Unyvero software then instructs the user to start the test which is fully automated until completion.

4. Calibration: Calibration is not required by the user.

5. Quality Control:

See section L1(c) for information on internal and external controls.

V Substantial Equivalence Information:

A Predicate Device Name(s): Curetis Unyvero LRT Application B Predicate 510(k) Number(s): DEN170047 C Comparison with Predicate:

<table><tr><td colspan="1" rowspan="1">Device &amp; PredicateDevice(s):</td><td colspan="1" rowspan="1">K191967</td><td colspan="1" rowspan="1">DEN170047</td></tr><tr><td colspan="1" rowspan="1">Device Trade Name</td><td colspan="1" rowspan="1">Unyvero LowerRespiratory TractBAL Application</td><td colspan="1" rowspan="1">Unyvero LRTApplication</td></tr><tr><td colspan="1" rowspan="1">General DeviceCharacteristic Similarities</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Microorganisms Detected</td><td colspan="1" rowspan="1">Typical' Bacteria:Acinetobacter spp.Citrobacter freundiiEnterobacter cloacaecomplexEscherichia coliHaemophilus influenzaeKlebsiella oxytocaKlebsiella pneumoniaeKlebsiella variicolaMoraxella catarrhalisMorganella morganiiProteus spp.PseudomonasaeruginosaSerratia marcescensStaphylococcus aureusStenotrophomonasmaltophiliaStreptococcuspneumoniaeAtypical' Bacteria:Chlamydia pneumoniaeLegionella pneumophilaMycoplasma</td><td colspan="1" rowspan="1">'Typical' Bacteria:Acinetobacter spp.Citrobacter freundiiEnterobacter cloacaecomplexEscherichia coliHaemophilus influenzaeKlebsiella oxytocaKlebsiella pneumoniaeKlebsiella variicolaMoraxella catarrhalisMorganella morganiiProteus spp.PseudomonasaeruginosaSerratia marcescensStaphylococcus aureusStenotrophomonasmaltophiliaStreptococcuspneumoniae'Atypical' Bacteria:Chlamydia pneumoniaeLegionella pneumophilaMycoplasma</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">pneumoniaeFungus: Pneumocystisjirovecii</td><td colspan="1" rowspan="1">pneumoniae</td></tr><tr><td colspan="1" rowspan="1">Antimicrobial ResistanceMarkers Detected</td><td colspan="1" rowspan="1">ctx-MkpcmecAndmoxa-23oxa-24oxa-48oxa-58temvim</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Technology</td><td colspan="1" rowspan="1">Multiplex PCRfollowed by probe array</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Result Type</td><td colspan="1" rowspan="1">Qualitative</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Instrumentation</td><td colspan="1" rowspan="1">Unyvero System</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Test Interpretation</td><td colspan="1" rowspan="1">Automated testinterpretation and reportgeneration</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">General DeviceCharacteristic Differences</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Specimen Type</td><td colspan="1" rowspan="1">BAL specimens</td><td colspan="1" rowspan="1">Endotracheal aspiratespecimens</td></tr></table>

# VI Standards/Guidance Documents Referenced:

None

VII Performance Characteristics (if/when applicable):

# A Analytical Performance:

Sample matrix: Unless otherwise specified, analytical study samples were prepared using a simulated matrix (ARM, Artificial Respiratory Matrix) which was adapted from a composition published by Dinesh et al. ARM was diluted to an appropriate consistency as a surrogate for BAL matrix $( 2 5 \% \mathrm { ( v / v ) }$ ARM, $60 \%$ (v/v) PBS and $1 5 \%$ (v/v) glycerol). The final $0 . 2 5 \mathrm { x }$ ARM also includes $5 \mathrm { m g / m L }$ mucin type $\mathrm { I I }$ from pig stomach and $1 \mathrm { m g / m L }$ fish sperm DNA.

# 1. Reproducibility:

Assay reproducibility was established with contrived samples that were prepared using viable microorganism strain suspensions inoculated into 0.25 ARM (artificial respiratory matrix). Representative target organisms evaluated included Gram-positive, Gram-negative and atypical microorganisms. The following LRT BAL Application analytes were included in the study: Acinetobacter spp., C. pneumoniae, C. freundii, H. influenzae, K. pneumoniae (carrying LRT BAL antibiotic resistance markers ctx-M, ndm, and tem), M. morganii, Proteus spp. and, S. aureus (carrying LRT BAL antibiotic resistance marker mecA). Reproducibility testing included samples prepared with moderate microorganism concentrations $2 . 5 \mathrm { x } - 1 0 \mathrm { x } \mathrm { L } { } $ oD), samples prepared at low concentrations ( $\mathrm { 1 x - 4 x ~ L o D ) }$ , and negative samples (surrogate matrix only (0.25 ARM)). Three different Unyvero systems were used in the study with each system consisting of one Unyvero Cockpit, three Unyvero Lysators, up to six Unyvero Analyzers, Testing was performed by three operators.

Each operator performed 3 replicates each of moderate, low and negative samples per test shift using one Unyvero system (maximum: 18 replicates per test day). Each operator performed a minimum of 30 replicates per concentration level (90 total replicates per Unyvero system, 270 replicates in total for all three Unyvero systems with 90 replicates of each concentration level). Testing was performed over a minimum of five test days. Failed runs as well as runs with single invalid analyte results were repeated to achieve a minimum number of at least 30 valid results for each of the target analytes and test concentration.

The study demonstrated acceptable test reproducibility by the LRT BAL Application for all analytes and concentrations evaluated. Three cartridge runs performed on different test days and with different operators/test systems generated unexpected positive results with weak signals close to assay thresholds (2x E. coli, for one ‘moderate’ and one ‘low’ test sample, 1x oxa-23, for one ‘moderate’ test sample).

Tables 1 to 3 summarize the study results for the microorganism and resistance marker target evaluated in the study.

Table 1: Reproducibility Study, Moderate Positive Samples   

<table><tr><td colspan="1" rowspan="2">Microorganism target</td><td colspan="1" rowspan="2">x-foldLoD</td><td colspan="1" rowspan="2">Unyvero System/Operator</td><td colspan="3" rowspan="1">Agreement with Expected Result</td></tr><tr><td colspan="1" rowspan="1"># Pos./# Exp.</td><td colspan="1" rowspan="1">%</td><td colspan="1" rowspan="1">95% CI</td></tr><tr><td colspan="1" rowspan="4">Acinetobacter baumanniiATCC 19606</td><td colspan="1" rowspan="4">5</td><td colspan="1" rowspan="1">operator 1</td><td colspan="1" rowspan="1">30/31</td><td colspan="1" rowspan="1">96.8</td><td colspan="1" rowspan="1">83.8 - 99.4</td></tr><tr><td colspan="1" rowspan="1">operator 2</td><td colspan="1" rowspan="1">31/31</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">89.0 - 100.0</td></tr><tr><td colspan="1" rowspan="1">operator 3</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">88.7 - 100.0</td></tr><tr><td colspan="1" rowspan="1">total</td><td colspan="1" rowspan="1">91/92</td><td colspan="1" rowspan="1">98.9</td><td colspan="1" rowspan="1">94.1 - 99.8</td></tr><tr><td colspan="1" rowspan="4">Chlamydia pneumoniaeATCC VR-2282</td><td colspan="1" rowspan="4">5</td><td colspan="1" rowspan="1">operator 1</td><td colspan="1" rowspan="1">31/31</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">89.0 - 100.0</td></tr><tr><td colspan="1" rowspan="1">operator 2</td><td colspan="1" rowspan="1">31/31</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">89.0 - 100.0</td></tr><tr><td colspan="1" rowspan="1">operator 3</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">88.7 - 100.0</td></tr><tr><td colspan="1" rowspan="1">total</td><td colspan="1" rowspan="1">92/92</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">96.0 - 100.0</td></tr><tr><td colspan="1" rowspan="4">Citrobacter freundiiATCC 8090</td><td colspan="1" rowspan="4">5</td><td colspan="1" rowspan="1">operator 1</td><td colspan="1" rowspan="1">29/30 a</td><td colspan="1" rowspan="1">96.7</td><td colspan="1" rowspan="1">83.3 - 99.4</td></tr><tr><td colspan="1" rowspan="1">operator 2</td><td colspan="1" rowspan="1">31/31</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">89.0 - 100.0</td></tr><tr><td colspan="1" rowspan="1">operator 3</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">88.7 - 100.0</td></tr><tr><td colspan="1" rowspan="1">total</td><td colspan="1" rowspan="1">90/91</td><td colspan="1" rowspan="1">98.9</td><td colspan="1" rowspan="1">94.0 - 99.8</td></tr><tr><td colspan="1" rowspan="4">Haemophilus influenzaeATCC 33391</td><td colspan="1" rowspan="4">5</td><td colspan="1" rowspan="1">operator 1</td><td colspan="1" rowspan="1">29/31</td><td colspan="1" rowspan="1">93.5</td><td colspan="1" rowspan="1">79.3 - 98.2</td></tr><tr><td colspan="1" rowspan="1">operator 2</td><td colspan="1" rowspan="1">31/31</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">89.0 - 100.0</td></tr><tr><td colspan="1" rowspan="1">operator 3</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">88.7 - 100.0</td></tr><tr><td colspan="1" rowspan="1">total</td><td colspan="1" rowspan="1">90/92</td><td colspan="1" rowspan="1">97.8</td><td colspan="1" rowspan="1">92.4 - 99.4</td></tr><tr><td colspan="1" rowspan="3">Klebsiella pneumoniaeNCTC 13443</td><td colspan="1" rowspan="3">2.5</td><td colspan="1" rowspan="1">operator 1</td><td colspan="1" rowspan="1">31/31</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">89.0 - 100.0</td></tr><tr><td colspan="1" rowspan="1">operator 2</td><td colspan="1" rowspan="1">31/31</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">89.0 - 100.0</td></tr><tr><td colspan="1" rowspan="1">operator 3</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">88.7 - 100.0</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">total</td><td colspan="1" rowspan="1">92/92</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">96.0 - 100.0</td></tr><tr><td colspan="1" rowspan="4">ctx-MNCTC 13443</td><td colspan="1" rowspan="4">10</td><td colspan="1" rowspan="1">operator 1</td><td colspan="1" rowspan="1">31/31</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">89.0 - 100.0</td></tr><tr><td colspan="1" rowspan="1">operator 2</td><td colspan="1" rowspan="1">31/31</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">89.0 - 100.0</td></tr><tr><td colspan="1" rowspan="1">operator 3</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">88.7 - 100.0</td></tr><tr><td colspan="1" rowspan="1">total</td><td colspan="1" rowspan="1">92/92</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">96.0 - 100.0</td></tr><tr><td colspan="1" rowspan="4">ndmNCTC 13443</td><td colspan="1" rowspan="4">5</td><td colspan="1" rowspan="1">operator 1</td><td colspan="1" rowspan="1">29/30 a</td><td colspan="1" rowspan="1">96.7</td><td colspan="1" rowspan="1">83.3 - 99.4</td></tr><tr><td colspan="1" rowspan="1">operator 2</td><td colspan="1" rowspan="1">31/31</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">89.0 - 100.0</td></tr><tr><td colspan="1" rowspan="1">operator 3</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">88.7 - 100.0</td></tr><tr><td colspan="1" rowspan="1">total</td><td colspan="1" rowspan="1">90/91</td><td colspan="1" rowspan="1">98.9</td><td colspan="1" rowspan="1">94.0 - 99.8</td></tr><tr><td colspan="1" rowspan="4">temNCTC 13443</td><td colspan="1" rowspan="4">5</td><td colspan="1" rowspan="1">operator 1</td><td colspan="1" rowspan="1">28/31</td><td colspan="1" rowspan="1">90.3</td><td colspan="1" rowspan="1">75.1 - 96.7</td></tr><tr><td colspan="1" rowspan="1">operator 2</td><td colspan="1" rowspan="1">31/31</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">89.0 - 100.0</td></tr><tr><td colspan="1" rowspan="1">operator 3</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">88.7 - 100.0</td></tr><tr><td colspan="1" rowspan="1">total</td><td colspan="1" rowspan="1">89/92</td><td colspan="1" rowspan="1">96.7</td><td colspan="1" rowspan="1">90.8 - 98.9</td></tr><tr><td colspan="1" rowspan="4">Morganella morganiiATCC 25830</td><td colspan="1" rowspan="4">5</td><td colspan="1" rowspan="1">operator 1</td><td colspan="1" rowspan="1">31/31</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">89.0 - 100.0</td></tr><tr><td colspan="1" rowspan="1">operator 2</td><td colspan="1" rowspan="1">31/31</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">89.0 - 100.0</td></tr><tr><td colspan="1" rowspan="1">operator 3</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">88.7 - 100.0</td></tr><tr><td colspan="1" rowspan="1">total</td><td colspan="1" rowspan="1">92/92</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">96.0 - 100.0</td></tr><tr><td colspan="1" rowspan="4">Proteus vulgarisATCC 29905</td><td colspan="1" rowspan="4">5</td><td colspan="1" rowspan="1">operator 1</td><td colspan="1" rowspan="1">31/31</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">89.0 - 100.0</td></tr><tr><td colspan="1" rowspan="1">operator 2</td><td colspan="1" rowspan="1">30/30 </td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">88.7 - 100.0</td></tr><tr><td colspan="1" rowspan="1">operator 3</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">88.7 - 100.0</td></tr><tr><td colspan="1" rowspan="1">total</td><td colspan="1" rowspan="1">91/91</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">96.0 - 100.0</td></tr><tr><td colspan="1" rowspan="4">Staphylococcus aureusNCTC 12493</td><td colspan="1" rowspan="4">8.3</td><td colspan="1" rowspan="1">operator 1</td><td colspan="1" rowspan="1">31/31</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">89.0 - 100.0</td></tr><tr><td colspan="1" rowspan="1">operator 2</td><td colspan="1" rowspan="1">31/31</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">89.0 - 100.0</td></tr><tr><td colspan="1" rowspan="1">operator 3</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">88.7 - 100.0</td></tr><tr><td colspan="1" rowspan="1">total</td><td colspan="1" rowspan="1">92/92</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">96.0 - 100.0</td></tr><tr><td colspan="1" rowspan="4">mecANCTC 12493</td><td colspan="1" rowspan="4">3</td><td colspan="1" rowspan="1">operator 1</td><td colspan="1" rowspan="1">31/31</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">89.0 - 100.0</td></tr><tr><td colspan="1" rowspan="1">operator 2</td><td colspan="1" rowspan="1">31/31</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">89.0 - 100.0</td></tr><tr><td colspan="1" rowspan="1">operator 3</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">88.7 - 100.0</td></tr><tr><td colspan="1" rowspan="1">total</td><td colspan="1" rowspan="1">92/92</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">96.0 - 100.0</td></tr></table>

a Reduced number of available results due to invalid analyte results

Table 2: Reproducibility study results, Low positive samples   

<table><tr><td colspan="1" rowspan="2">Microorganism Target</td><td colspan="1" rowspan="2">x-foldLoD</td><td colspan="1" rowspan="2">Unyvero System/Operator</td><td colspan="3" rowspan="1">Agreement with Expected Result</td></tr><tr><td colspan="1" rowspan="1"># Pos./# Exp.</td><td colspan="1" rowspan="1">%</td><td colspan="1" rowspan="1">95% CI</td></tr><tr><td colspan="1" rowspan="4">Acinetobacter baumanniiATCC 19606</td><td colspan="1" rowspan="4">2</td><td colspan="1" rowspan="1">operator 1</td><td colspan="1" rowspan="1">29/30</td><td colspan="1" rowspan="1">96.7</td><td colspan="1" rowspan="1">83.3 - 99.4</td></tr><tr><td colspan="1" rowspan="1">operator 2</td><td colspan="1" rowspan="1">31/31</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">89.0 - 100.0</td></tr><tr><td colspan="1" rowspan="1">operator 3</td><td colspan="1" rowspan="1">29/30</td><td colspan="1" rowspan="1">96.7</td><td colspan="1" rowspan="1">83.3 - 99.4</td></tr><tr><td colspan="1" rowspan="1">total</td><td colspan="1" rowspan="1">89/91</td><td colspan="1" rowspan="1">97.8</td><td colspan="1" rowspan="1">92.3 - 99.4</td></tr><tr><td colspan="1" rowspan="4">Chlamydia pneumoniaeATCC VR-2282</td><td colspan="1" rowspan="4">2</td><td colspan="1" rowspan="1">operator 1</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">88.7 - 100.0</td></tr><tr><td colspan="1" rowspan="1">operator 2</td><td colspan="1" rowspan="1">31/31</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">89.0 - 100.0</td></tr><tr><td colspan="1" rowspan="1">operator 3</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">88.7 - 100.0</td></tr><tr><td colspan="1" rowspan="1">total</td><td colspan="1" rowspan="1">91/91</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">96.0 - 100.0</td></tr><tr><td colspan="1" rowspan="4">Citrobacter freundiiATCC 8090</td><td colspan="1" rowspan="4">2</td><td colspan="1" rowspan="1">operator 1</td><td colspan="1" rowspan="1">28/30</td><td colspan="1" rowspan="1">93.3</td><td colspan="1" rowspan="1">78.7 - 98.2</td></tr><tr><td colspan="1" rowspan="1">operator 2</td><td colspan="1" rowspan="1">26/31</td><td colspan="1" rowspan="1">83.9</td><td colspan="1" rowspan="1">67.4 - 92.9</td></tr><tr><td colspan="1" rowspan="1">operator 3</td><td colspan="1" rowspan="1">29/30</td><td colspan="1" rowspan="1">96.7</td><td colspan="1" rowspan="1">83.3 - 99.4</td></tr><tr><td colspan="1" rowspan="1">total</td><td colspan="1" rowspan="1">83/91</td><td colspan="1" rowspan="1">91.2</td><td colspan="1" rowspan="1">83.6 - 95.5</td></tr><tr><td colspan="1" rowspan="4">Haemophilus influenzaeATCC 33391</td><td colspan="1" rowspan="4">2</td><td colspan="1" rowspan="1">operator 1</td><td colspan="1" rowspan="1">27/30</td><td colspan="1" rowspan="1">90.0</td><td colspan="1" rowspan="1">74.4 - 96.5</td></tr><tr><td colspan="1" rowspan="1">operator 2</td><td colspan="1" rowspan="1">31/31</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">89.0 - 100.0</td></tr><tr><td colspan="1" rowspan="1">operator 3</td><td colspan="1" rowspan="1">29/30</td><td colspan="1" rowspan="1">96.7</td><td colspan="1" rowspan="1">83.3 - 99.4</td></tr><tr><td colspan="1" rowspan="1">total</td><td colspan="1" rowspan="1">87/91</td><td colspan="1" rowspan="1">95.6</td><td colspan="1" rowspan="1">89.2 - 98.3</td></tr><tr><td colspan="1" rowspan="4">Klebsiella pneumoniaeNCTC 13443</td><td colspan="1" rowspan="4">1</td><td colspan="1" rowspan="1">operator 1</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">88.7 - 100.0</td></tr><tr><td colspan="1" rowspan="1">operator 2</td><td colspan="1" rowspan="1">31/31</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">89.0 - 100.0</td></tr><tr><td colspan="1" rowspan="1">operator 3</td><td colspan="1" rowspan="1">28/30</td><td colspan="1" rowspan="1">93.3</td><td colspan="1" rowspan="1">78.7 - 98.2</td></tr><tr><td colspan="1" rowspan="1">total</td><td colspan="1" rowspan="1">89/91</td><td colspan="1" rowspan="1">97.8</td><td colspan="1" rowspan="1">92.3 - 99.4</td></tr><tr><td colspan="1" rowspan="4">ctx-MNCTC 13443</td><td colspan="1" rowspan="4">4</td><td colspan="1" rowspan="1">operator 1</td><td colspan="1" rowspan="1">29/30</td><td colspan="1" rowspan="1">96.7</td><td colspan="1" rowspan="1">83.3 - 99.4</td></tr><tr><td colspan="1" rowspan="1">operator 2</td><td colspan="1" rowspan="1">30/31</td><td colspan="1" rowspan="1">96.8</td><td colspan="1" rowspan="1">83.8 - 99.4</td></tr><tr><td colspan="1" rowspan="1">operator 3</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">88.7 - 100.0</td></tr><tr><td colspan="1" rowspan="1">total</td><td colspan="1" rowspan="1">89/91</td><td colspan="1" rowspan="1">97.8</td><td colspan="1" rowspan="1">92.3 - 99.4</td></tr><tr><td colspan="1" rowspan="4">ndmNCTC 13443</td><td colspan="1" rowspan="4">2</td><td colspan="1" rowspan="1">operator 1</td><td colspan="1" rowspan="1">29/30</td><td colspan="1" rowspan="1">96.7</td><td colspan="1" rowspan="1">83.3 - 99.4</td></tr><tr><td colspan="1" rowspan="1">operator 2</td><td colspan="1" rowspan="1">30/31</td><td colspan="1" rowspan="1">96.8</td><td colspan="1" rowspan="1">83.8 - 99.4</td></tr><tr><td colspan="1" rowspan="1">operator 3</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">88.7 - 100.0</td></tr><tr><td colspan="1" rowspan="1">total</td><td colspan="1" rowspan="1">89/91</td><td colspan="1" rowspan="1">97.8</td><td colspan="1" rowspan="1">92.3 - 99.4</td></tr><tr><td colspan="1" rowspan="4">temNCTC 13443</td><td colspan="1" rowspan="4">2</td><td colspan="1" rowspan="1">operator 1</td><td colspan="1" rowspan="1">27/30</td><td colspan="1" rowspan="1">90.0</td><td colspan="1" rowspan="1">74.4 - 96.5</td></tr><tr><td colspan="1" rowspan="1">operator 2</td><td colspan="1" rowspan="1">31/31</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">89.0 - 100.0</td></tr><tr><td colspan="1" rowspan="1">operator 3</td><td colspan="1" rowspan="1">29/30</td><td colspan="1" rowspan="1">96.7</td><td colspan="1" rowspan="1">83.3 - 99.4</td></tr><tr><td colspan="1" rowspan="1">total</td><td colspan="1" rowspan="1">87/91</td><td colspan="1" rowspan="1">95.6</td><td colspan="1" rowspan="1">89.2 - 98.3</td></tr><tr><td colspan="1" rowspan="4">Morganella morganiiATCC 25830</td><td colspan="1" rowspan="4">2</td><td colspan="1" rowspan="1">operator 1</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">88.7 - 100.0</td></tr><tr><td colspan="1" rowspan="1">operator 2</td><td colspan="1" rowspan="1">30/31</td><td colspan="1" rowspan="1">96.8</td><td colspan="1" rowspan="1">83.8 - 99.4</td></tr><tr><td colspan="1" rowspan="1">operator 3</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">88.7 - 100.0</td></tr><tr><td colspan="1" rowspan="1">total</td><td colspan="1" rowspan="1">90/91</td><td colspan="1" rowspan="1">98.9</td><td colspan="1" rowspan="1">94.0 - 99.8</td></tr><tr><td colspan="1" rowspan="4">Proteus vulgaris aATCC 29905</td><td colspan="1" rowspan="4">2</td><td colspan="1" rowspan="1">operator 1</td><td colspan="1" rowspan="1">25/30</td><td colspan="1" rowspan="1">83.3</td><td colspan="1" rowspan="1">66.4 - 92.7</td></tr><tr><td colspan="1" rowspan="1">operator 2</td><td colspan="1" rowspan="1">24/31</td><td colspan="1" rowspan="1">77.4</td><td colspan="1" rowspan="1">60.2 - 88.6</td></tr><tr><td colspan="1" rowspan="1">operator 3</td><td colspan="1" rowspan="1">27/30</td><td colspan="1" rowspan="1">90.0</td><td colspan="1" rowspan="1">74.4 - 96.5</td></tr><tr><td colspan="1" rowspan="1">total</td><td colspan="1" rowspan="1">76/91</td><td colspan="1" rowspan="1">83.5</td><td colspan="1" rowspan="1">74.6 - 89.7</td></tr><tr><td colspan="1" rowspan="4">Staphylococcus aureusNCTC 12493</td><td colspan="1" rowspan="4">3</td><td colspan="1" rowspan="1">operator 1</td><td colspan="1" rowspan="1">29/30</td><td colspan="1" rowspan="1">96.7</td><td colspan="1" rowspan="1">83.3 - 99.4</td></tr><tr><td colspan="1" rowspan="1">operator 2</td><td colspan="1" rowspan="1">30/30 b</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">88.7 - 100.0</td></tr><tr><td colspan="1" rowspan="1">operator 3</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">88.7 - 100.0</td></tr><tr><td colspan="1" rowspan="1">total</td><td colspan="1" rowspan="1">89/90</td><td colspan="1" rowspan="1">98.9</td><td colspan="1" rowspan="1">94.0 - 99.8</td></tr><tr><td colspan="1" rowspan="4">mecANCTC 12493</td><td colspan="1" rowspan="4">1.3</td><td colspan="1" rowspan="1">operator 1</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">88.7 - 100.0</td></tr><tr><td colspan="1" rowspan="1">operator 2</td><td colspan="1" rowspan="1">31/31</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">89.0 - 100.0</td></tr><tr><td colspan="1" rowspan="1">operator 3</td><td colspan="1" rowspan="1">29/30</td><td colspan="1" rowspan="1">96.7</td><td colspan="1" rowspan="1">83.3 - 99.4</td></tr><tr><td colspan="1" rowspan="1">total</td><td colspan="1" rowspan="1">90/91</td><td colspan="1" rowspan="1">98.9</td><td colspan="1" rowspan="1">94.0 - 99.8</td></tr></table>

a Proteus spp. demonstrated a slightly lower than expected positivity rate for the 2x LoD concentration with a positivity rate (for all operators combined) of $8 3 . 5 \%$ with a $9 5 \%$ CI of $7 4 . 6 \% - 8 9 . 7 \%$ . However, for an alternative test series performed in parallel for the sample stability study using the same strain and the identical cartridge lot with pooled negative BAL matrix the expected positivity rate at a $2 \mathbf { x }$ LoD concentration for Proteus spp. was confirmed (positivity rate: 92/93, $9 8 . 9 \%$ ). b Reduced number of available results due to invalid analyte results.

Table 3: Reproducibility Study Results, Negative Samples   

<table><tr><td colspan="1" rowspan="2">Microorganism target</td><td colspan="1" rowspan="2">Unyvero System/Operator</td><td colspan="3" rowspan="1">Agreement with Expected Result</td></tr><tr><td colspan="1" rowspan="1"># Neg./# Exp.</td><td colspan="1" rowspan="1">%</td><td colspan="1" rowspan="1">95% CI</td></tr><tr><td colspan="1" rowspan="4">Acinetobacter baumanniiATCC 19606</td><td colspan="1" rowspan="1">operator 1</td><td colspan="1" rowspan="1">31/31</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">89.0 - 100.0</td></tr><tr><td colspan="1" rowspan="1">operator 2</td><td colspan="1" rowspan="1">31/31</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">89.0 - 100.0</td></tr><tr><td colspan="1" rowspan="1">operator 3</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">88.7 - 100.0</td></tr><tr><td colspan="1" rowspan="1">total</td><td colspan="1" rowspan="1">92/92</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">96.0 - 100.0</td></tr><tr><td colspan="1" rowspan="4">Chlamydia pneumoniaeATCC VR-2282</td><td colspan="1" rowspan="1">operator 1</td><td colspan="1" rowspan="1">31/31</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">89.0 - 100.0</td></tr><tr><td colspan="1" rowspan="1">operator 2</td><td colspan="1" rowspan="1">31/31</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">89.0 - 100.0</td></tr><tr><td colspan="1" rowspan="1">operator 3</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">88.7 - 100.0</td></tr><tr><td colspan="1" rowspan="1">total</td><td colspan="1" rowspan="1">92/92</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">96.0 - 100.0</td></tr><tr><td colspan="1" rowspan="1">Citrobacter freundii</td><td colspan="1" rowspan="1">operator 1</td><td colspan="1" rowspan="1">31/31</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">89.0 - 100.0</td></tr><tr><td colspan="1" rowspan="3">ATCC 8090</td><td colspan="1" rowspan="1">operator 2</td><td colspan="1" rowspan="1">31/31</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">89.0 - 100.0</td></tr><tr><td colspan="1" rowspan="1">operator 3</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">88.7 - 100.0</td></tr><tr><td colspan="1" rowspan="1">total</td><td colspan="1" rowspan="1">92/92</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">96.0 - 100.0</td></tr><tr><td colspan="1" rowspan="4">Haemophilus influenzaeATCC 33391</td><td colspan="1" rowspan="1">operator 1</td><td colspan="1" rowspan="1">31/31</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">89.0 - 100.0</td></tr><tr><td colspan="1" rowspan="1">operator 2</td><td colspan="1" rowspan="1">31/31</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">89.0 - 100.0</td></tr><tr><td colspan="1" rowspan="1">operator 3</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">88.7 - 100.0</td></tr><tr><td colspan="1" rowspan="1">total</td><td colspan="1" rowspan="1">92/92</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">96.0 - 100.0</td></tr><tr><td colspan="1" rowspan="4">Klebsiella pneumoniaeNCTC 13443</td><td colspan="1" rowspan="1">operator 1</td><td colspan="1" rowspan="1">31/31</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">89.0 - 100.0</td></tr><tr><td colspan="1" rowspan="1">operator 2</td><td colspan="1" rowspan="1">30/30 a</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">88.7 - 100.0</td></tr><tr><td colspan="1" rowspan="1">operator 3</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">88.7 - 100.0</td></tr><tr><td colspan="1" rowspan="1">total</td><td colspan="1" rowspan="1">91/91</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">96.0 - 100.0</td></tr><tr><td colspan="1" rowspan="4">ctx-MNCTC 13443</td><td colspan="1" rowspan="1">operator 1</td><td colspan="1" rowspan="1">31/31</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">89.0 - 100.0</td></tr><tr><td colspan="1" rowspan="1">operator 2</td><td colspan="1" rowspan="1">30/30 a</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">88.7 - 100.0</td></tr><tr><td colspan="1" rowspan="1">operator 3</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">88.7 - 100.0</td></tr><tr><td colspan="1" rowspan="1">total</td><td colspan="1" rowspan="1">91/91</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">96.0 - 100.0</td></tr><tr><td colspan="1" rowspan="4">ndmNCTC 13443</td><td colspan="1" rowspan="1">operator 1</td><td colspan="1" rowspan="1">31/31</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">89.0 - 100.0</td></tr><tr><td colspan="1" rowspan="1">operator 2</td><td colspan="1" rowspan="1">31/31</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">89.0 - 100.0</td></tr><tr><td colspan="1" rowspan="1">operator 3</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">88.7 - 100.0</td></tr><tr><td colspan="1" rowspan="1">total</td><td colspan="1" rowspan="1">92/92</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">96.0 - 100.0</td></tr><tr><td colspan="1" rowspan="4">temNCTC 13443</td><td colspan="1" rowspan="1">operator 1</td><td colspan="1" rowspan="1">31/31</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">89.0 - 100.0</td></tr><tr><td colspan="1" rowspan="1">operator 2</td><td colspan="1" rowspan="1">31/31</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">89.0 - 100.0</td></tr><tr><td colspan="1" rowspan="1">operator 3</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">88.7 - 100.0</td></tr><tr><td colspan="1" rowspan="1">total</td><td colspan="1" rowspan="1">92/92</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">96.0 - 100.0</td></tr><tr><td colspan="1" rowspan="4">Morganella morganiiATCC 25830</td><td colspan="1" rowspan="1">operator 1</td><td colspan="1" rowspan="1">31/31</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">89.0 - 100.0</td></tr><tr><td colspan="1" rowspan="1">operator 2</td><td colspan="1" rowspan="1">30/30 a</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">88.7 - 100.0</td></tr><tr><td colspan="1" rowspan="1">operator 3</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">88.7 - 100.0</td></tr><tr><td colspan="1" rowspan="1">total</td><td colspan="1" rowspan="1">91/91</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">96.0 - 100.0</td></tr><tr><td colspan="1" rowspan="4">Proteus vulgarisATCC 29905</td><td colspan="1" rowspan="1">operator 1</td><td colspan="1" rowspan="1">30/30 a</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">88.7 - 100.0</td></tr><tr><td colspan="1" rowspan="1">operator 2</td><td colspan="1" rowspan="1">31/31</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">89.0 - 100.0</td></tr><tr><td colspan="1" rowspan="1">operator 3</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">88.7 - 100.0</td></tr><tr><td colspan="1" rowspan="1">total</td><td colspan="1" rowspan="1">91/91</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">96.0 - 100.0</td></tr><tr><td colspan="1" rowspan="4">Staphylococcus aureusNCTC 12493</td><td colspan="1" rowspan="1">operator 1</td><td colspan="1" rowspan="1">31/31</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">89.0 - 100.0</td></tr><tr><td colspan="1" rowspan="1">operator 2</td><td colspan="1" rowspan="1">31/31</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">89.0 - 100.0</td></tr><tr><td colspan="1" rowspan="1">operator 3</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">88.7 - 100.0</td></tr><tr><td colspan="1" rowspan="1">total</td><td colspan="1" rowspan="1">92/92</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">96.0 - 100.0</td></tr><tr><td colspan="1" rowspan="4">mecANCTC 12493</td><td colspan="1" rowspan="1">operator 1</td><td colspan="1" rowspan="1">31/31</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">89.0 - 100.0</td></tr><tr><td colspan="1" rowspan="1">operator 2</td><td colspan="1" rowspan="1">30/30 a</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">88.7 - 100.0</td></tr><tr><td colspan="1" rowspan="1">operator 3</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">88.7 - 100.0</td></tr><tr><td colspan="1" rowspan="1">total</td><td colspan="1" rowspan="1">91/91</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">96.0 - 100.0</td></tr></table>

a Reduced number of available results due to invalid analyte results

2. Linearity:

Not Applicable

# 3. Analytical Reactivity (Inclusivity):

The analytical reactivity of the Unyvero LRT BAL Application was evaluated with reference strains for each microorganism target. Samples were prepared with microorganism concentrations near their respective LoDs in pooled negative BAL specimen matrix. Reference strains tested in the LoD study were also included in the study as control strai

Assay inclusivity was established near LoD $( < 5 \mathrm { x } \ \mathrm { L o D } )$ for all but one of the tested reference strains. For one strain of $M .$ . pneumoniae (ATCC 15531) that was evaluated using genomic DNA extract, inclusivity was demonstrated only at 5x LoD, although in silico analysis demonstrated no primer or probe mismatches to the corresponding LRT BAL assay targets. Results from the study are presented in Table 4.

Table 4: Inclusivity Study, Microorganism Targets   

<table><tr><td colspan="1" rowspan="1">Reference Strain</td><td colspan="1" rowspan="1">Strain ID</td><td colspan="1" rowspan="1">Test Conc.FU/mlor otheras notedbyfootnotel</td><td colspan="1" rowspan="1">x-foldLoD</td><td colspan="1" rowspan="1"># Pos./ #Exp.</td></tr><tr><td colspan="1" rowspan="1">Acinetobacter baumannii(pos. control/LoD ref. strain)</td><td colspan="1" rowspan="1">ATCC 19606</td><td colspan="1" rowspan="1">4.0 x 104</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">6/6</td></tr><tr><td colspan="1" rowspan="1">Acinetobacter baumannii</td><td colspan="1" rowspan="1">NCTC 13305</td><td colspan="1" rowspan="1">4.0 x 104</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">Acinetobacter baumannii</td><td colspan="1" rowspan="1">NCTC 13301</td><td colspan="1" rowspan="1">4.0 x 104</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">Acinetobacter baumannii</td><td colspan="1" rowspan="1">NCTC 13302</td><td colspan="1" rowspan="1">4.0 x 104</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">Acinetobacter baumannii</td><td colspan="1" rowspan="1">Micromyx 6148</td><td colspan="1" rowspan="1">4.0 x 104</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">Acinetobacter baumannii</td><td colspan="1" rowspan="1">Micromyx 4410</td><td colspan="1" rowspan="1">4.0 x 104</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">Acinetobacter baumannii</td><td colspan="1" rowspan="1">JMI 49755</td><td colspan="1" rowspan="1">4.0 x 104</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">Acinetobacter baumannii</td><td colspan="1" rowspan="1">NRZ-00449</td><td colspan="1" rowspan="1">1.0 x 104</td><td colspan="1" rowspan="1">0.5x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">Acinetobacter baumannii</td><td colspan="1" rowspan="1">UCLA A4</td><td colspan="1" rowspan="1">1.0 x 104</td><td colspan="1" rowspan="1">0.5x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">Acinetobacter calcoaceticus</td><td colspan="1" rowspan="1">ATCC 23055</td><td colspan="1" rowspan="1">4.0 x 104</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">Acinetobacter lwoffi</td><td colspan="1" rowspan="1">ATCC 15309</td><td colspan="1" rowspan="1">4.0 x 104</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">Acinetobacter haemolyticus</td><td colspan="1" rowspan="1">ATCC 17906</td><td colspan="1" rowspan="1">4.0 x 104</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">Chlamydia pneumoniae(pos. control/LoD ref. strain)</td><td colspan="1" rowspan="1">ATCC VR-2282</td><td colspan="1" rowspan="1">6.4 x 102 a</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">4/4</td></tr><tr><td colspan="1" rowspan="1">Chlamydia pneumoniae</td><td colspan="1" rowspan="1">ATCC VR-1310</td><td colspan="1" rowspan="1">6.4 x 102 a</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="2">Chlamydia pneumoniae</td><td colspan="1" rowspan="2">ATCC 53592</td><td colspan="1" rowspan="1">6.4 x 102 a</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">0/2</td></tr><tr><td colspan="1" rowspan="1">1.0 x 103 a</td><td colspan="1" rowspan="1">3x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">Citrobacter freundii(pos. control/LoD ref. strain)</td><td colspan="1" rowspan="1">ATCC 8090</td><td colspan="1" rowspan="1">1.6 x 105</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">4/4</td></tr><tr><td colspan="1" rowspan="1">Citrobacter freundii</td><td colspan="1" rowspan="1">ATCC 43864</td><td colspan="1" rowspan="1">1.6 x 105</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">Citrobacter freundii</td><td colspan="1" rowspan="1">NCTC 8581</td><td colspan="1" rowspan="1">1.6 x 105</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">Citrobacter freundii</td><td colspan="1" rowspan="1">NRZ-00452</td><td colspan="1" rowspan="1">1.6 x 105</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">Citrobacter freundii</td><td colspan="1" rowspan="1">UCLA C1</td><td colspan="1" rowspan="1">1.6 x 105</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">Enterobacter cloacae(pos. control/LoD ref. strain)</td><td colspan="1" rowspan="1">ATCC 13047</td><td colspan="1" rowspan="1">4.0 x 105</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">7/7</td></tr><tr><td colspan="1" rowspan="1">Enterobacter cloacae</td><td colspan="1" rowspan="1">ATCC 23355</td><td colspan="1" rowspan="1">4.0 x 10 5</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">Enterobacter cloacae</td><td colspan="1" rowspan="1">ATCC 49141</td><td colspan="1" rowspan="1">4.0 x 10$</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">Enterobacter cloacae</td><td colspan="1" rowspan="1">ATCC BAA-2468</td><td colspan="1" rowspan="1">4.0 x 105</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">Enterobacter cloacae</td><td colspan="1" rowspan="1">JMI 46239</td><td colspan="1" rowspan="1">4.0 x 105</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">Enterobacter cloacae</td><td colspan="1" rowspan="1">NRZ-00239</td><td colspan="1" rowspan="1">4.0 x 10 5$</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">Enterobacter cloacae ssp. dissolvens</td><td colspan="1" rowspan="1">ATCC 23373</td><td colspan="1" rowspan="1">4.0 x 105</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">Enterobacter hormaechei</td><td colspan="1" rowspan="1">ATCC 49162</td><td colspan="1" rowspan="1">4.0 x 105</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">Reference Strain</td><td colspan="1" rowspan="1">Strain ID</td><td colspan="1" rowspan="1">Test Conc. CFU/ml or otheras noted byfoototel</td><td colspan="1" rowspan="1">x-fold LoOD</td><td colspan="1" rowspan="1"># Pos./ #Exp.</td></tr><tr><td colspan="1" rowspan="1">Enterobacter asburiae</td><td colspan="1" rowspan="1">ATCC 35953</td><td colspan="1" rowspan="1">4.0 x 105$</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli(pos. control/LoD ref. strain</td><td colspan="1" rowspan="1">ATCC 11775</td><td colspan="1" rowspan="1">4.0 x 104</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">4/4</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">ATCC 25922</td><td colspan="1" rowspan="1">4.0 x 104</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">ATCC 35218</td><td colspan="1" rowspan="1">4.0 x 104</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">4/4</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">ATCC BAA-2523</td><td colspan="1" rowspan="1">4.0 x 104</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">NCTC 13351</td><td colspan="1" rowspan="1">4.0 x 104</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">4/4</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">NCTC 13476</td><td colspan="1" rowspan="1">4.0 x 104</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">JMI 50067</td><td colspan="1" rowspan="1">4.0 x 104</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">Haemophilus influenzae(non-typeable/non-capsulated)(pos. control/LoD ref. strain)</td><td colspan="1" rowspan="1">ATCC 33391</td><td colspan="1" rowspan="1">4.0 x 104</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Reference Strain</td><td colspan="1" rowspan="1">Strain ID</td><td colspan="1" rowspan="1">Test Conc. CFV/ml</td><td colspan="1" rowspan="1">x-fold</td><td colspan="1" rowspan="1">#Pos./# Exp.</td></tr><tr><td colspan="1" rowspan="1">Haemophilus influenzae (serotype a)</td><td colspan="1" rowspan="1">ATCC 9006</td><td colspan="1" rowspan="1">4.0 x 104</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">Haemophilus influenzae (serotype c)</td><td colspan="1" rowspan="1">ATCC 9007</td><td colspan="1" rowspan="1">4.0 x 104</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="2">Haemophilus influenzae (serotype b)</td><td colspan="1" rowspan="2">ATCC 10211</td><td colspan="1" rowspan="1">4.0 x 104</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">1/2</td></tr><tr><td colspan="1" rowspan="1">6.0 x 104</td><td colspan="1" rowspan="1">3x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">Haemophilus influenzae (serotype b)</td><td colspan="1" rowspan="1">ATCC 49247</td><td colspan="1" rowspan="1">4.0 x 104</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">Haemophilus influenzae (serotype b)</td><td colspan="1" rowspan="1">ATCC 49766</td><td colspan="1" rowspan="1">4.0 x 104</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">Klebsiella oxytocapsool/Lo i</td><td colspan="1" rowspan="1">ATCC 13182</td><td colspan="1" rowspan="1">4.0 x 104</td><td colspan="1" rowspan="1">4x</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Klebsiella oxytoca</td><td colspan="1" rowspan="1">ATCC 43863</td><td colspan="1" rowspan="1">4.0 x 104</td><td colspan="1" rowspan="1">4x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">Klebsiella oxytoca</td><td colspan="1" rowspan="1">ATCC 8724</td><td colspan="1" rowspan="1">4.0 x 104</td><td colspan="1" rowspan="1">4x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">Klebsiella oxytoca</td><td colspan="1" rowspan="1">ATCC 49131</td><td colspan="1" rowspan="1">4.0 x 104</td><td colspan="1" rowspan="1">4x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">Klebsiella oxytoca</td><td colspan="1" rowspan="1">NCIMB 12819</td><td colspan="1" rowspan="1">4.0 x 104</td><td colspan="1" rowspan="1">4x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">Klebsiella oxytoca</td><td colspan="1" rowspan="1">NRZ-22060</td><td colspan="1" rowspan="1">4.0 x 104</td><td colspan="1" rowspan="1">4x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae, variant 1(pos. control/LoD ref. strain)</td><td colspan="1" rowspan="1">ATCC 13883</td><td colspan="1" rowspan="1">8.0 x 104</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">4/4</td></tr><tr><td colspan="1" rowspan="1">Klebsiella quasipneumoniae(K. pneumoniae, variant 2)(pos. control/LoD ref. strain)</td><td colspan="1" rowspan="1">ATCC 700603</td><td colspan="1" rowspan="1">8.0 x 104</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">NCTC 13439</td><td colspan="1" rowspan="1">8.0 x 104</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">NCTC 13438</td><td colspan="1" rowspan="1">8.0 x 104</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">NCTC 13442</td><td colspan="1" rowspan="1">8.0 x 104</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">NCTC 13443</td><td colspan="1" rowspan="1">2.0 x 104</td><td colspan="1" rowspan="1">0.5x</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">Micromyx 4653</td><td colspan="1" rowspan="1">8.0 x 104</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">Micromyx 4676</td><td colspan="1" rowspan="1">8.0 x 104</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">JMI 49767</td><td colspan="1" rowspan="1">2.0 x 104</td><td colspan="1" rowspan="1">0.5x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">NRZ-00002</td><td colspan="1" rowspan="1">2.0 x 104</td><td colspan="1" rowspan="1">0.5x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">NRZ-00103</td><td colspan="1" rowspan="1">8.0 x 104</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">Klebsiella variicolaos. control/LoD ref.train</td><td colspan="1" rowspan="1">ATCC BAA-830</td><td colspan="1" rowspan="1">4.0 x 104</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Klebsiella variicola</td><td colspan="1" rowspan="1">clinical strain 1</td><td colspan="1" rowspan="1">4.0 x 104</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">Reference Strain</td><td colspan="1" rowspan="1">Strain ID</td><td colspan="1" rowspan="1">Test Conc.CFU/ml or othera s notedbbyfootnotel</td><td colspan="1" rowspan="1">x-foldLoOD</td><td colspan="1" rowspan="1"># Pos./ #Exp..</td></tr><tr><td colspan="1" rowspan="1">Klebsiella variicola</td><td colspan="1" rowspan="1">clinical strain 2</td><td colspan="1" rowspan="1">4.0 x 104</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">Klebsiella variicola</td><td colspan="1" rowspan="1">clinical strain 3</td><td colspan="1" rowspan="1">4.0 x 104</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">Klebsiella variicola</td><td colspan="1" rowspan="1">clinical strain 4</td><td colspan="1" rowspan="1">4.0 x 104</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">Legionella pneumophila (serotype 1)(pos. control/LoD ref. strain)</td><td colspan="1" rowspan="1">ATCC 33152</td><td colspan="1" rowspan="1">1.6 x 105</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">7/7</td></tr><tr><td colspan="1" rowspan="1">Legionella pneumophila (serotype 2)</td><td colspan="1" rowspan="1">ATCC 33154</td><td colspan="1" rowspan="1">1.6 x 105</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">Legionella pneumophila (serotype 3)</td><td colspan="1" rowspan="1">ATCC 33155</td><td colspan="1" rowspan="1">1.6 x 105$</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">Legionella pneumophila (serotype 6)</td><td colspan="1" rowspan="1">ATCC 33215</td><td colspan="1" rowspan="1">1.6 x 105$</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">Legionella pneumophila (serotype 8)</td><td colspan="1" rowspan="1">ATCC 35096</td><td colspan="1" rowspan="1">1.6 x 105</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">Legionella pneumophila (serotype 10)</td><td colspan="1" rowspan="1">ATCC 43283</td><td colspan="1" rowspan="1">1.6 x 105</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">Legionella pneumophila</td><td colspan="1" rowspan="1">UCLA L1</td><td colspan="1" rowspan="1">1.6 x 105</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">Legionella pneumophila</td><td colspan="1" rowspan="1">UCLA L5</td><td colspan="1" rowspan="1">1.6 x 105</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">Legionella pneumophila</td><td colspan="1" rowspan="1">UCLA L6</td><td colspan="1" rowspan="1">1.6 x 105</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">Moraxella catarrhalis(pos. control/LoD ref. strain)</td><td colspan="1" rowspan="1">ATCC 25238</td><td colspan="1" rowspan="1">3.0 x 105</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">4/4</td></tr><tr><td colspan="1" rowspan="1">Moraxella catarrhalis</td><td colspan="1" rowspan="1">ATCC 43617</td><td colspan="1" rowspan="1">3.0 x 10 5</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">Moraxella catarrhalis</td><td colspan="1" rowspan="1">ATCC 8176</td><td colspan="1" rowspan="1">3.0 x 105</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">Moraxella catarrhalis</td><td colspan="1" rowspan="1">ATCC 25240</td><td colspan="1" rowspan="1">3.0 x 10 5</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">Moraxella catarrhalis</td><td colspan="1" rowspan="1">ATCC 23246</td><td colspan="1" rowspan="1">3.0 x 105</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Moraxella catarrhalis</td><td colspan="1" rowspan="1">ATCC 49143</td><td colspan="1" rowspan="1">3.0 x 105</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">Morganella morganii(pos. control/LoD ref. strain)</td><td colspan="1" rowspan="1">ATCC 25830</td><td colspan="1" rowspan="1">4.0 x 104</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/3</td></tr><tr><td colspan="1" rowspan="1">Morganella morganii</td><td colspan="1" rowspan="1">ATCC 8019</td><td colspan="1" rowspan="1">4.0 x 104</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="2">Morganella morganii</td><td colspan="1" rowspan="2">ATCC 25829</td><td colspan="1" rowspan="1">4.0 x 104</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">0/1</td></tr><tr><td colspan="1" rowspan="1">6.0 x 104</td><td colspan="1" rowspan="1">3x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="2">Morganella morganii</td><td colspan="1" rowspan="2">DSM-46262</td><td colspan="1" rowspan="1">4.0 x 104</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">1/2</td></tr><tr><td colspan="1" rowspan="1">6.0 x 104</td><td colspan="1" rowspan="1">3x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">Morganella morganii ssp. sibonii</td><td colspan="1" rowspan="1">ATCC 49948</td><td colspan="1" rowspan="1">4.0 x 104</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">Mycoplasma pneumoniae(pos. control/LoD ref. strain)</td><td colspan="1" rowspan="1">ATCC 29085</td><td colspan="1" rowspan="1">3.2 x 103 b</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">5/7</td></tr><tr><td colspan="1" rowspan="1">Mycoplasma pneumoniae</td><td colspan="1" rowspan="1">ATCC 29343</td><td colspan="1" rowspan="1">3.2 x 103 b</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="5">Mycoplasma pneumoniae</td><td colspan="1" rowspan="5">ATCC 15531c,d</td><td colspan="1" rowspan="1">3.2 x 103 b</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">0/2</td></tr><tr><td colspan="1" rowspan="1">5.0 x 103 b</td><td colspan="1" rowspan="1">3x</td><td colspan="1" rowspan="1">1/2</td></tr><tr><td colspan="1" rowspan="1">6.4 x 103 b</td><td colspan="1" rowspan="1">4x</td><td colspan="1" rowspan="1">1/2</td></tr><tr><td colspan="1" rowspan="1">7.0 x 103 b</td><td colspan="1" rowspan="1">4.4x</td><td colspan="1" rowspan="1">1/2</td></tr><tr><td colspan="1" rowspan="1">8.0 x 103 b</td><td colspan="1" rowspan="1">5x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">Mycoplasma pneumoniae</td><td colspan="1" rowspan="1">ATCC 15293</td><td colspan="1" rowspan="1">3.2 x 103 b</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">Mycoplasma pneumoniae</td><td colspan="1" rowspan="1">ATCC 49894</td><td colspan="1" rowspan="1">3.2 x 103 b</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">Pneumocystis jirovecii(pos. control/LoD ref. strain)</td><td colspan="1" rowspan="1">pos. specimen 1</td><td colspan="1" rowspan="1">1.0 x 106 b</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">1/1</td></tr><tr><td colspan="1" rowspan="1">Pneumocystis jirovecii</td><td colspan="1" rowspan="1">pos. specimen 2</td><td colspan="1" rowspan="1">1.0 x 106 b</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">Pneumocystis jirovecii</td><td colspan="1" rowspan="1">pos. specimen 3</td><td colspan="1" rowspan="1">1.0 x 106 b</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">Pneumocystis jirovecii</td><td colspan="1" rowspan="1">pos. specimen 4</td><td colspan="1" rowspan="1">1.0 x 106 b</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">Pneumocystis jirovecii</td><td colspan="1" rowspan="1">pos. specimen 5</td><td colspan="1" rowspan="1">1.0 x 106 b</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">Reference Strain</td><td colspan="1" rowspan="1">Strain ID</td><td colspan="1" rowspan="1">Test Conc.CFU/mloor otheras notedbyfootnotel</td><td colspan="1" rowspan="1">x-foldLoOD</td><td colspan="1" rowspan="1"># Pos./ #Exp.</td></tr><tr><td colspan="1" rowspan="1">Proteus vulgaris(pos. control/LoD ref. strain)</td><td colspan="1" rowspan="1">ATCC 29905</td><td colspan="1" rowspan="1">1.0 x 104</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">7/7</td></tr><tr><td colspan="1" rowspan="1">Proteus mirabilis(pos. control/LoD ref. strain)</td><td colspan="1" rowspan="1">ATCC 29906</td><td colspan="1" rowspan="1">1.0 x 104</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">Proteus mirabilis</td><td colspan="1" rowspan="1">ATCC 12453</td><td colspan="1" rowspan="1">1.0 x 104</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">Proteus mirabilis</td><td colspan="1" rowspan="1">ATCC 14153</td><td colspan="1" rowspan="1">1.0 x 104</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">Proteus mirabilis</td><td colspan="1" rowspan="1">ATCC 25933</td><td colspan="1" rowspan="1">1.0 x 104</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">Proteus vulgaris</td><td colspan="1" rowspan="1">ATCC 6380</td><td colspan="1" rowspan="1">1.0 x 104</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">Proteus vulgaris</td><td colspan="1" rowspan="1">ATCC 8427</td><td colspan="1" rowspan="1">1.0 x 104</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">Proteus hauseri</td><td colspan="1" rowspan="1">ATCC 700826</td><td colspan="1" rowspan="1">1.0 x 104</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">Proteus penneri</td><td colspan="1" rowspan="1">ATCC 33519</td><td colspan="1" rowspan="1">1.0 x 104</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">Pseudomonas aeruginosa(pos. control/LoD ref. strain)</td><td colspan="1" rowspan="1">ATCC 10145</td><td colspan="1" rowspan="1">1.3 x 103</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">ATCC 27853</td><td colspan="1" rowspan="1">1.3 x 103</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">NCTC 13437</td><td colspan="1" rowspan="1">1.3 x 103</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">Micromyx 2562</td><td colspan="1" rowspan="1">2.0 x 103</td><td colspan="1" rowspan="1">3x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">NRZ-00196</td><td colspan="1" rowspan="1">1.3 x 103</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">UCLA P20</td><td colspan="1" rowspan="1">1.3 x 103</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">Serratia marcescens(pos. control/LoD ref. strain)</td><td colspan="1" rowspan="1">ATCC 13880</td><td colspan="1" rowspan="1">8.0 x 104</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Serratia marcescens</td><td colspan="1" rowspan="1">ATCC 14756</td><td colspan="1" rowspan="1">8.0 x 104</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">Serratia marcescens</td><td colspan="1" rowspan="1">ATCC 15365</td><td colspan="1" rowspan="1">8.0 x 104</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">Serratia marcescens</td><td colspan="1" rowspan="1">ATCC 27117</td><td colspan="1" rowspan="1">8.0 x 104</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">Serratia marcescens</td><td colspan="1" rowspan="1">ATCC 43861</td><td colspan="1" rowspan="1">8.0 x 104</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">Serratia marcescens ssp. sakuensis</td><td colspan="1" rowspan="1">DSM-17174</td><td colspan="1" rowspan="1">8.0 x 104</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus(pos. control/LoD ref. strain)</td><td colspan="1" rowspan="1">ATCC 12600</td><td colspan="1" rowspan="1">3.0 x 105</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">7/7</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">ATCC BAA-2312</td><td colspan="1" rowspan="1">3.0 x 105</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">NCTC 12493</td><td colspan="1" rowspan="1">3.0 x 105$</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">ATCC 33591</td><td colspan="1" rowspan="1">3.0 x 105</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">DSM-17091</td><td colspan="1" rowspan="1">3.0 x 105</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">ATCC 29213</td><td colspan="1" rowspan="1">3.0 x 105</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">ATCC 43300</td><td colspan="1" rowspan="1">3.0 x 105</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">Stenotrophomonas maltophilia(pos. control/LoD ref. strain)</td><td colspan="1" rowspan="1">ATCC 13637</td><td colspan="1" rowspan="1">1.0 x 104</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">4/4</td></tr><tr><td colspan="1" rowspan="2">Stenotrophomonas maltophilia</td><td colspan="1" rowspan="2">ATCC 13636</td><td colspan="1" rowspan="1">1.0 x 104</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">0/2</td></tr><tr><td colspan="1" rowspan="1">1.5 x 104</td><td colspan="1" rowspan="1">3x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">Stenotrophomonas maltophilia</td><td colspan="1" rowspan="1">ATCC 17666</td><td colspan="1" rowspan="1">1.0 x 104</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">Stenotrophomonas maltophilia</td><td colspan="1" rowspan="1">ATCC 49130</td><td colspan="1" rowspan="1">1.0 x 104</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">Stenotrophomonas maltophilia</td><td colspan="1" rowspan="1">DSM-50173</td><td colspan="1" rowspan="1">1.0 x 104</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="3">Stenotrophomonas maltophilia</td><td colspan="1" rowspan="3">DSM-21874[NCIMB 9528]</td><td colspan="1" rowspan="1">1.0 x 104</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">0/2</td></tr><tr><td colspan="1" rowspan="1">1.5 x 104</td><td colspan="1" rowspan="1">3x</td><td colspan="1" rowspan="1">0/2</td></tr><tr><td colspan="1" rowspan="1">2.0 x 104</td><td colspan="1" rowspan="1">4x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">Reference Strain</td><td colspan="1" rowspan="1">Strain ID</td><td colspan="1" rowspan="1">Test Conc.ICFU/mlor otheras notedbyfootnote</td><td colspan="1" rowspan="1">x-foldLoD</td><td colspan="1" rowspan="1"># Pos./ #xp.</td></tr><tr><td colspan="1" rowspan="1">Streptococcus pneumoniaeserotype 19F(pos. control/LoD ref. strain)</td><td colspan="1" rowspan="1">ATCC 49619</td><td colspan="1" rowspan="1">4.0 x 104</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">4/7f</td></tr><tr><td colspan="1" rowspan="1">Streptococcus pneumoniae serotype 3</td><td colspan="1" rowspan="1">ATCC 6303</td><td colspan="1" rowspan="1">4.0 x 104</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">Streptococcus pneumoniae serotype 5</td><td colspan="1" rowspan="1">ATCC 6305</td><td colspan="1" rowspan="1">4.0 x 104</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="2">Streptococcus pneumoniae</td><td colspan="1" rowspan="2">ATCC 49150g</td><td colspan="1" rowspan="1">4.0 x 104</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">0/2</td></tr><tr><td colspan="1" rowspan="1">8.0 x 104</td><td colspan="1" rowspan="1">4x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="3">Streptococcus pneumoniae</td><td colspan="1" rowspan="3">ATCC 33400g</td><td colspan="1" rowspan="1">4.0 x 104</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">1/2</td></tr><tr><td colspan="1" rowspan="1">6.0 x 104</td><td colspan="1" rowspan="1">3x</td><td colspan="1" rowspan="1">1/2</td></tr><tr><td colspan="1" rowspan="1">8.0 x 104</td><td colspan="1" rowspan="1">4x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">Streptococcus pneumoniae serotype 2</td><td colspan="1" rowspan="1">ATCC 27336</td><td colspan="1" rowspan="1">4.0 x 104</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">Streptococcus pneumoniae serotype 1</td><td colspan="1" rowspan="1">ATCC 6301</td><td colspan="1" rowspan="1">4.0 x 104</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">Streptococcus pneumoniae serotype 9V</td><td colspan="1" rowspan="1">DSM-11865</td><td colspan="1" rowspan="1">4.0 x 104</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="3">Streptococcus pneumoniae serotype 23F</td><td colspan="1" rowspan="3">DSM-11866</td><td colspan="1" rowspan="1">4.0 x 104</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">6.0 x 104</td><td colspan="1" rowspan="1">3x</td><td colspan="1" rowspan="1">1/2</td></tr><tr><td colspan="1" rowspan="1">8.0 x 104</td><td colspan="1" rowspan="1">4x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">Streptococcus pneumoniae serotype 6B</td><td colspan="1" rowspan="1">DSM-11867</td><td colspan="1" rowspan="1">4.0 x 104</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr></table>

a $\mathrm { I n \ I F U / m L }$ for C. pneumoniae. b In copies $/ \mathrm { m L }$ for $M .$ . pneumoniae and $P .$ . jirovecii. c Quantified DNA extract (reference material) from a commercial provider. d For $M .$ . pneumoniae ATCC 15531, a positivity rate of 2/2 was obtained at a ${ 5 } \mathbf { x }$ LoD concentration. Sequencing did not reveal any mismatches to primer and probe binding sites and the observed slightly reduced sensitivity is likely due to the different source material (DNA extract instead of a cell suspension). $^ \mathrm { e } S .$ maltophilia strain DSM-21874 was positive at a $4 \mathbf { x }$ LoD concentration. Sequencing did not reveal any mismatches to primer or probe sequences. f Initial testing S. pneumoniae LoD reference strain ATCC 46916 generated inconsistent detection at $2 \mathbf { x }$ LoD. Repeat testing using a freshly prepared counted culture stock generated 8/8 correct results at a $2 \mathbf { x }$ LoD concentration. $\mathrm { ~ g ~ } _ { S }$ . pneumoniae strain ATCC 49150 was positive at $_ { 4 \mathrm { X } }$ LoD. Sequencing did not reveal any mismatches to primer or probe sequences. ${ \textrm { h } } _ { S }$ . pneumoniae strain ATCC 33400 was consistently positive at 4x LoD. Sequencing did not reveal any mismatches to primer or probe sequences.

To supplement inclusivity testing, in silico GenBank BLAST analyses of LRT BAL primer and probe sequences were performed for each LRT BAL microorganism (search performed July 2019) for all applicable strain entries. BLAST analyses identified strain entries for which detection by LRT BAL is predicted at LoD (match of relevant primer and probe sequences), strains predicted to be detected with reduced sensitivity (typically, single relevant mismatches of primer or probe sequences; strains for which detection is likely at higher than LoD concentrations only), or strains for which detection is not predicted (multiple relevant mismatches in primer and probe sequences).

Table 5 lists microorganisms for which inclusivity was demonstrated by wet testing and are supported by in silico analysis (predicted at LoD or predicted with reduced sensitivity). Also presented are in silico predictions for strains/species for which in silico analysis that were not evaluated by wet-testing. These in silico results are provided to the user as supplementary data only. Results are not intended to be a surrogate for wet testing and do not assure that specific strains will be detected.

NOTE: The performance of the Unyvero LRT BAL Application has not been established for those microorganism species that were evaluated by in silico analysis only.

Table 5: Inclusivity, Wet Testing and/or Predicted Detection by In silico Analysis   

<table><tr><td colspan="1" rowspan="1">Analyte</td><td colspan="1" rowspan="1">Wet Testing</td><td colspan="1" rowspan="1">in silico:at LoD*</td><td colspan="1" rowspan="1">in silico:Reduced sensivity*</td><td colspan="1" rowspan="1">Comment</td></tr><tr><td colspan="1" rowspan="1">Acinetobacter spp.</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">A. baumannii</td><td colspan="1" rowspan="1">X</td><td colspan="1" rowspan="1">X</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Y</td></tr><tr><td colspan="1" rowspan="1">A. calcoaceticus</td><td colspan="1" rowspan="1">X</td><td colspan="1" rowspan="1">X</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">A. lwoffi</td><td colspan="1" rowspan="1">X</td><td colspan="1" rowspan="1">X</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Y</td></tr><tr><td colspan="1" rowspan="1">A. haemolyticus</td><td colspan="1" rowspan="1">X</td><td colspan="1" rowspan="1">X</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">A. nosocomialis</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">X</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Y</td></tr><tr><td colspan="1" rowspan="1">A. pittii</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">X</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">A. junii</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">X</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">A. parvus</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">X</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">Y</td></tr><tr><td colspan="1" rowspan="1">A. lactucae</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">X</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">A. oleivorans</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">x</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">A. schindleri</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">X</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">Y</td></tr><tr><td colspan="1" rowspan="1">A. guillouiae</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">X1 entry</td><td colspan="1" rowspan="1">Y</td></tr><tr><td colspan="1" rowspan="1">A. radioresistens</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">X3 entries</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">A. soli</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">X1 entry</td><td colspan="1" rowspan="1">Y</td></tr><tr><td colspan="1" rowspan="1">A. ursingii</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">X1 entry</td><td colspan="1" rowspan="1">reference strain DSM-16037 testednegative at 1.5 x 107 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Chlamydia pneumoniae</td><td colspan="1" rowspan="1">X</td><td colspan="1" rowspan="1">X</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Citrobacter freundii</td><td colspan="1" rowspan="1">X</td><td colspan="1" rowspan="1">X26 entries</td><td colspan="1" rowspan="1">X4 entries</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Enterobacter cloacae complex</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">cYanmaGellowb</td></tr><tr><td colspan="1" rowspan="1">E. cloacae</td><td colspan="1" rowspan="1">X</td><td colspan="1" rowspan="1">X</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">including ssp. dissolvens</td></tr><tr><td colspan="1" rowspan="1">E. hormaechei</td><td colspan="1" rowspan="1">X</td><td colspan="1" rowspan="1">X</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">including ssp. oharae, steigerwaltii,hoffmannii</td></tr><tr><td colspan="1" rowspan="1">E. hormaechei ssp.xiangfangensis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">X7 entries</td><td colspan="1" rowspan="1">X1 entry</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">E. kobei</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">X</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">Y</td></tr><tr><td colspan="1" rowspan="1">E. ludwigii</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">X</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">E. sichuanensis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">X</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">E. chuandaensis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">X</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">E. roggenkampii</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">X</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">E. asburiae</td><td colspan="1" rowspan="1">X</td><td colspan="1" rowspan="1">X11 entries</td><td colspan="1" rowspan="1">X1 entry</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">X</td><td colspan="1" rowspan="1">X</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Haemophilus influenzae</td><td colspan="1" rowspan="1">X</td><td colspan="1" rowspan="1">X65 entries</td><td colspan="1" rowspan="1">X1 entry</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Klebsiella oxytoca</td><td colspan="1" rowspan="1">X</td><td colspan="1" rowspan="1">X16 entries</td><td colspan="1" rowspan="1">X6 entries</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">variant 1 (K. pneumoniae)</td><td colspan="1" rowspan="1">X</td><td colspan="1" rowspan="1">X</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">variant 2 (K. quasi-pneumoniae)</td><td colspan="1" rowspan="1">X</td><td colspan="1" rowspan="1">X11 entries</td><td colspan="1" rowspan="1">X10 entries</td><td colspan="1" rowspan="1">Y</td></tr><tr><td colspan="1" rowspan="1">Klebsiella variicola</td><td colspan="1" rowspan="1">X</td><td colspan="1" rowspan="1">X</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Analyte</td><td colspan="1" rowspan="1">Wet Testing</td><td colspan="1" rowspan="1">in silico:at LoD*</td><td colspan="1" rowspan="1">in silico:Reduced enstivity*</td><td colspan="1" rowspan="1">Comment</td></tr><tr><td colspan="1" rowspan="1">Legionella pneumophila</td><td colspan="1" rowspan="1">X</td><td colspan="1" rowspan="1">X</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">some entries predict detection atLoD, other entries predict apossible slight performancereduction; strains for both variantswere included in inclusivity wettesting and were detected atconcentrations near LoD.</td></tr><tr><td colspan="1" rowspan="1">Moraxella catarrhalis</td><td colspan="1" rowspan="1">X</td><td colspan="1" rowspan="1">X</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">EY</td></tr><tr><td colspan="1" rowspan="1">Morganella morganii</td><td colspan="1" rowspan="1">X</td><td colspan="1" rowspan="1">X</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Mycoplasma pneumoniae</td><td colspan="1" rowspan="1">X</td><td colspan="1" rowspan="1">X</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">EY</td></tr><tr><td colspan="1" rowspan="1">Pneumocystis jirovecii</td><td colspan="1" rowspan="1">X</td><td colspan="1" rowspan="1">X</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Proteus spp.</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">cYanmaGellow</td></tr><tr><td colspan="1" rowspan="1">P. mirabilis</td><td colspan="1" rowspan="1">X</td><td colspan="1" rowspan="1">X</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">EY</td></tr><tr><td colspan="1" rowspan="1">P. vulgaris</td><td colspan="1" rowspan="1">X</td><td colspan="1" rowspan="1">X</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">EY</td></tr><tr><td colspan="1" rowspan="1">P. hauseri</td><td colspan="1" rowspan="1">X</td><td colspan="1" rowspan="1">X</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">P. penneri</td><td colspan="1" rowspan="1">X</td><td colspan="1" rowspan="1">X</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">P. cibarius</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">X</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">X</td><td colspan="1" rowspan="1">X</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Serratia marcecens</td><td colspan="1" rowspan="1">X</td><td colspan="1" rowspan="1">X</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">X</td><td colspan="1" rowspan="1">X&gt; 450 entries</td><td colspan="1" rowspan="1">X1 entry</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Stenotrophomonasmaltophilia</td><td colspan="1" rowspan="1">X</td><td colspan="1" rowspan="1">X</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Streptococcuspneumoniae</td><td colspan="1" rowspan="1">X</td><td colspan="1" rowspan="1">X&gt; 240 entries</td><td colspan="1" rowspan="1">X1 entry</td><td colspan="1" rowspan="1">including serotype 7F</td></tr></table>

\* When in silico analyses predicts detection of strains at higher than LoD concentrations for one or more applicable entries, numbers of GenBank entries are listed.

Similar to microorganism testing, inclusivity wet testing was performed with reference strains carrying antibiotic resistance markers targeted by the LRT BAL Application. Contrived samples were prepared in pooled negative clinical BAL matrix with organism concentrations near the LoD for each resistance marker target.

Inclusivity of the LRT BAL Application for detection of targeted resistance markers was demonstrated for all strains evaluated in the study. Results are presented in Table 6 below with resistance marker variants/subgroups designated, if known. If available, results from phenotypic antibiotic susceptibility testing (AST) results are also presented (e.g.,

CarbapenemR $^ { \dot { \iota } } =$ resistant to carbapenems, Carbapenem $^ \mathrm { S } =$ susceptible to carbapenems).

Table 6: Inclusivity Study, Antibiotic Resistance Markers   

<table><tr><td colspan="1" rowspan="1">Analyte</td><td colspan="1" rowspan="1">Subgroup</td><td colspan="1" rowspan="1">Reference Strain</td><td colspan="1" rowspan="1">Strain ID</td><td colspan="1" rowspan="1">TestConc.[CFU/ml]</td><td colspan="1" rowspan="1">x-foldLoD</td><td colspan="1" rowspan="1"># Pos./ #Exp.</td></tr><tr><td colspan="1" rowspan="11">ctx-M</td><td colspan="1" rowspan="1">NA*</td><td colspan="1" rowspan="1">Klebsiella pneumoniae(pos. control/LoD ref.strain)</td><td colspan="1" rowspan="1">NCTC 13443</td><td colspan="1" rowspan="1">2.0 x 104</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">ctx-M3</td><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">NRZ-00751</td><td colspan="1" rowspan="1">2.0 x 104</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">ctx-M15</td><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">NRZ-00249</td><td colspan="1" rowspan="1">2.0 x 104</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="2">NA</td><td colspan="1" rowspan="2">Enterobacter cloacae</td><td colspan="1" rowspan="2">JMI 46239</td><td colspan="1" rowspan="1">2.0 x 104</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">1/2</td></tr><tr><td colspan="1" rowspan="1">3.0 x 104</td><td colspan="1" rowspan="1">3x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">JMI 50067</td><td colspan="1" rowspan="1">2.0 x 104</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="2">NA</td><td colspan="1" rowspan="2">Klebsiella pneumoniae</td><td colspan="1" rowspan="2">NRZ-00002</td><td colspan="1" rowspan="1">2.0 x 104</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">1/2</td></tr><tr><td colspan="1" rowspan="1">3.0 x 104</td><td colspan="1" rowspan="1">3x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">Enterobacter cloacae</td><td colspan="1" rowspan="1">ATCC BAA-2468</td><td colspan="1" rowspan="1">4.0 x 104</td><td colspan="1" rowspan="1">4x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">JMI 49831</td><td colspan="1" rowspan="1">4.0 x 104</td><td colspan="1" rowspan="1">4x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">JMI 49767</td><td colspan="1" rowspan="1">2.0 x 104</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="7">kpc</td><td colspan="1" rowspan="1">kpc-3</td><td colspan="1" rowspan="1">Klebsiella pneumoniae(pos. control/LoD ref.strain)</td><td colspan="1" rowspan="1">NCTC 13438</td><td colspan="1" rowspan="1">8.0 x 104</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">5/5</td></tr><tr><td colspan="1" rowspan="1">kpc-2</td><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">NRZ-00281</td><td colspan="1" rowspan="1">8.0 x 104</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">kpc-2</td><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">NRZ-00103</td><td colspan="1" rowspan="1">8.0 x 104</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="2">kpc-3</td><td colspan="1" rowspan="2">Escherichia coli</td><td colspan="1" rowspan="2">NRZ-00222</td><td colspan="1" rowspan="1">8.0 x 104</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">1/2</td></tr><tr><td colspan="1" rowspan="1">1.2 x 105$</td><td colspan="1" rowspan="1">3x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">kpc-3</td><td colspan="1" rowspan="1">Klebsiella pneumoniae(Carbapenem)</td><td colspan="1" rowspan="1">Micromyx 4653</td><td colspan="1" rowspan="1">8.0 x 104</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">4/4</td></tr><tr><td colspan="1" rowspan="1">kpc-3</td><td colspan="1" rowspan="1">Klebsiella pneumoniae(Carbapenem)</td><td colspan="1" rowspan="1">Micromyx 4676</td><td colspan="1" rowspan="1">8.0 x 104</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">4/4</td></tr><tr><td colspan="1" rowspan="8">mecA</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">Staphylococcus aureus(Methicillin", Cefoxitin*)(pos. control/LoD ref.strain)</td><td colspan="1" rowspan="1">NCTC 12493</td><td colspan="1" rowspan="1">8.0 x 105</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">SCCmecI</td><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">RKI 07-03165</td><td colspan="1" rowspan="1">8.0 x 105</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">SCCmecII</td><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">RKI 01-00694</td><td colspan="1" rowspan="1">8.0 x 105</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">SCCmecIII</td><td colspan="1" rowspan="1">Staphylococcus aureus(Methicillin)</td><td colspan="1" rowspan="1">ATCC 33591</td><td colspan="1" rowspan="1">8.0 x 105</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">SCCmecIV</td><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">RKI 09-00187</td><td colspan="1" rowspan="1">8.0 x 105</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">SCCmecV</td><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">RKI 08-02492</td><td colspan="1" rowspan="1">8.0 x 105</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">Staphylococcus aureus(MethicillinR)</td><td colspan="1" rowspan="1">DSM-17091</td><td colspan="1" rowspan="1">8.0 x 105</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">SCCmecII</td><td colspan="1" rowspan="1">Staphylococcus aureus(Methicillin)</td><td colspan="1" rowspan="1">ATCC 43300</td><td colspan="1" rowspan="1">3.0 x 105</td><td colspan="1" rowspan="1">0.8x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="8">ndm</td><td colspan="1" rowspan="1">ndm-1</td><td colspan="1" rowspan="1">Klebsiella pneumoniae(pos. control/LoD ref.strain)</td><td colspan="1" rowspan="1">NCTC 13443</td><td colspan="1" rowspan="1">4.0 x 104</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">5/5</td></tr><tr><td colspan="1" rowspan="1">ndm-1</td><td colspan="1" rowspan="1">Acinetobacter baumannii</td><td colspan="1" rowspan="1">JMI 49755</td><td colspan="1" rowspan="1">4.0 x 104</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">4/4</td></tr><tr><td colspan="1" rowspan="2">ndm-1</td><td colspan="1" rowspan="2">Enterobacter cloacae(ImipenemR, Ertapenem)</td><td colspan="1" rowspan="2">ATCC BAA-2468</td><td colspan="1" rowspan="1">4.0 x 104</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">1/2</td></tr><tr><td colspan="1" rowspan="1">6.0 x 104</td><td colspan="1" rowspan="1">3x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">ndm-1</td><td colspan="1" rowspan="1">Enterobacter cloacae</td><td colspan="1" rowspan="1">JMI 46239</td><td colspan="1" rowspan="1">4.0 x 104</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">4/4</td></tr><tr><td colspan="1" rowspan="1">ndm-1</td><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">JMI 50067</td><td colspan="1" rowspan="1">4.0 x 104</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">ndm-1</td><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">JMI 49767</td><td colspan="1" rowspan="1">2.0 x 104</td><td colspan="1" rowspan="1">1x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">ndm-1</td><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">JMI 49831</td><td colspan="1" rowspan="1">4.0 x 104</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="6">oxa-23</td><td colspan="1" rowspan="1">oxa-23</td><td colspan="1" rowspan="1">Acinetobacter baumannii(pos. control/LoD ref.strain)</td><td colspan="1" rowspan="1">NCTC 13301</td><td colspan="1" rowspan="1">2.0 x 107</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">Acinetobacter baumannii(Carbapenem)</td><td colspan="1" rowspan="1">Micromyx 4410</td><td colspan="1" rowspan="1">2.0 x 107</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">Acinetobacter baumannii(Carbapenem)</td><td colspan="1" rowspan="1">Micromyx 6148</td><td colspan="1" rowspan="1">2.0 x 107</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">Acinetobacter baumannii(Carbapenem)</td><td colspan="1" rowspan="1">Micromyx 6149</td><td colspan="1" rowspan="1">2.0 x 107</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">Acinetobacter baumannii(Carbapenem)</td><td colspan="1" rowspan="1">Micromyx 6153</td><td colspan="1" rowspan="1">2.0 x 107</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">Acinetobacter baumannii(ImipenemR, MeropenemR)</td><td colspan="1" rowspan="1">UCLA A5</td><td colspan="1" rowspan="1">2.0 x 107</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="5">oxa-24</td><td colspan="1" rowspan="1">oxa-25</td><td colspan="1" rowspan="1">Acinetobacter baumannii(pos. control/LoD ref.strain)</td><td colspan="1" rowspan="1">NCTC 13302</td><td colspan="1" rowspan="1">1.0 x 104</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">oxa-72</td><td colspan="1" rowspan="1">Acinetobacter baumannii</td><td colspan="1" rowspan="1">NRZ-00449</td><td colspan="1" rowspan="1">1.0 x 104</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">Acinetobacter baumannii(ImipenemR, MeropenemR)</td><td colspan="1" rowspan="1">UCLA A4</td><td colspan="1" rowspan="1">1.0 x 104</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">Acinetobacter baumannii(ImipenemR, Meropenem)</td><td colspan="1" rowspan="1">clinical strain 1</td><td colspan="1" rowspan="1">1.0 x 104</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">Acinetobacter baumannii(ImipenemR, MeropenemR)</td><td colspan="1" rowspan="1">clinical strain 2</td><td colspan="1" rowspan="1">1.0 x 104</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="7">oxa-48</td><td colspan="1" rowspan="1">oxa-48</td><td colspan="1" rowspan="1">Klebsiella pneumoniae(pos. control/LoD ref.strain)</td><td colspan="1" rowspan="1">NCTC 13442</td><td colspan="1" rowspan="1">6.0 x 105</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">oxa-48</td><td colspan="1" rowspan="1">Escherichia coli(Ertapenem)</td><td colspan="1" rowspan="1">ATCC BAA-2523</td><td colspan="1" rowspan="1">6.0 x 105</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="2">oxa-48</td><td colspan="1" rowspan="2">Escherichia coli</td><td colspan="1" rowspan="2">NRZ-00176</td><td colspan="1" rowspan="1">6.0 x 105</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">1/2</td></tr><tr><td colspan="1" rowspan="1">9.0 x 105$</td><td colspan="1" rowspan="1">3x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">oxa-162</td><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">NRZ-00361</td><td colspan="1" rowspan="1">6.0 x 105</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">oxa-162</td><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">NRZ-00472</td><td colspan="1" rowspan="1">6.0 x 105</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">oxa-232</td><td colspan="1" rowspan="1">Klebsiella oxytoca</td><td colspan="1" rowspan="1">NRZ-22060</td><td colspan="1" rowspan="1">6.0 x 105</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="2">oxa-58</td><td colspan="1" rowspan="1">oxa-58</td><td colspan="1" rowspan="1">Acinetobacter baumannii(pos. control/LoD ref.strain)</td><td colspan="1" rowspan="1">NCTC 13305</td><td colspan="1" rowspan="1">1.0 x 105</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/3</td></tr><tr><td colspan="1" rowspan="1">oxa-58</td><td colspan="1" rowspan="1">Acinetobacter baumannii</td><td colspan="1" rowspan="1">NRZ-00518</td><td colspan="1" rowspan="1">1.0 x 105</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="12">tem</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">Klebsiella pneumoniae(pos. control/LoD ref.strain)</td><td colspan="1" rowspan="1">NCTC 13443</td><td colspan="1" rowspan="1">4.0 x 104</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">tem-1</td><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">ATCC 35218</td><td colspan="1" rowspan="1">4.0 x 104</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">tem-3</td><td colspan="1" rowspan="1">Escherichia coli(ESBL)</td><td colspan="1" rowspan="1">NCTC 13351</td><td colspan="1" rowspan="1">4.0 x 104</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">Citrobacter freundii</td><td colspan="1" rowspan="1">ATCC 43864</td><td colspan="1" rowspan="1">4.0 x 104</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">Enterobacter cloacae</td><td colspan="1" rowspan="1">JMI 46239</td><td colspan="1" rowspan="1">4.0 x 104</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">JMI 50067</td><td colspan="1" rowspan="1">2.0 x 104</td><td colspan="1" rowspan="1">1x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">NRZ-00751</td><td colspan="1" rowspan="1">2.0 x 104</td><td colspan="1" rowspan="1">1x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">ATCC BAA-2523</td><td colspan="1" rowspan="1">4.0 x 104</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">Acinetobacter baumannii</td><td colspan="1" rowspan="1">JMI 49755</td><td colspan="1" rowspan="1">4.0 x 104</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">Haemophilus influenzae(AmpicillinR, Cefinase)</td><td colspan="1" rowspan="1">clinical strain 1</td><td colspan="1" rowspan="1">4.0 x 104</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">Haemophilus influenzae(Cefinase)</td><td colspan="1" rowspan="1">clinical strain 2</td><td colspan="1" rowspan="1">4.0 x 104</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">Micromyx 4676</td><td colspan="1" rowspan="1">8.0 x 104</td><td colspan="1" rowspan="1">4x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="8">vim</td><td colspan="1" rowspan="1">vim-10</td><td colspan="1" rowspan="1">Pseudomonas aeruginosa(pos. control/LoD ref.strain)</td><td colspan="1" rowspan="1">NCTC 13437</td><td colspan="1" rowspan="1">4.0 x 104</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">vim-1</td><td colspan="1" rowspan="1">Citrobacter freundii</td><td colspan="1" rowspan="1">NRZ-00452</td><td colspan="1" rowspan="1">4.0 x 104</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">vim-1</td><td colspan="1" rowspan="1">Pseudomonas aeruginosa(CeftazidimeR, Imipenem)</td><td colspan="1" rowspan="1">DSM-24600</td><td colspan="1" rowspan="1">4.0 x 104</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">vim-1</td><td colspan="1" rowspan="1">Enterobacter cloacae</td><td colspan="1" rowspan="1">NRZ-00239</td><td colspan="1" rowspan="1">4.0 x 104</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">vim-1</td><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">NCTC 13439</td><td colspan="1" rowspan="1">4.0 x 104</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">vim-1</td><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">NCTC 13440</td><td colspan="1" rowspan="1">4.0 x 104</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">UCLA P20</td><td colspan="1" rowspan="1">1.3 x 103</td><td colspan="1" rowspan="1">0.1x</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">Pseudomonas aeruginosa(CarbapenemR)</td><td colspan="1" rowspan="1">Micromyx 2562</td><td colspan="1" rowspan="1">4.0 x 104</td><td colspan="1" rowspan="1">2x</td><td colspan="1" rowspan="1">2/2</td></tr></table>

$^ { * } \mathrm { { N } } / \mathrm { { A } } =$ variant status unknown

To supplement inclusivity testing specifically for antibiotic resistance marker variants, reference sequences for all available variants belonging to individual antibiotic resistance markers or marker subgroups were evaluated using in silico analysis.

Table 7 below includes a summary of subgroups or variants for applicable LRT BAL antibiotic resistance markers for which detection is predicted at LoD (match of relevant primer and probe sequences), detection is predicted with reduced sensitivity (typically, single relevant mismatches of primer or probe sequences; detection likely at higher than LoD concentrations only), or detection is not predicted (multiple relevant mismatches in primer and probe sequences). In silico data are provide in the test labeling as supplementary data; however results are not intended to be a surrogate for wet testing and do not assure that specific variant will be detected.

NOTE: The performance of the Unyvero LRT BAL Application has not been established for antibiotic resistance marker variants other than those listed in Table 6 above.

Table 7: In silico Analysis: Predicted Detection of Resistance Marker Subgroups and Variants.   

<table><tr><td colspan="1" rowspan="1">AntibioticResistanceMarker:Subgroup</td><td colspan="1" rowspan="1">Detection Predicted atLoD:Variant No.</td><td colspan="1" rowspan="1">Detection PredictedwithReduced Sensitivity:Variant No.</td><td colspan="1" rowspan="1">Detection NotPredicted: Variant No.</td></tr><tr><td colspan="1" rowspan="1">ctx-M:ctx-M1 subgroup</td><td colspan="1" rowspan="1">1, 3, 10, 11, 12, 15, 22, 23,28 - 30, 32 - 34, 36, 37, 42,52 - 55, 57, 58, 60 - 62, 66,68, 69, 71, 72, 79, 80, 82,83, 88, 96, 101, 103, 107,108, 109, 114, 116, 117,132, 133, 136, 138, 139,142, 144, 150, 155- 158,162 - 164, 166, 167, 169,170, 172, 173, 175 - 177,179 - 184, 186, 188 - 190,193, 194, 197, 202 - 204,206' - 212, 216, 218, 222,224</td><td colspan="1" rowspan="1">64</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">ctx-M:ctxM2ctx-M8ctx-M9ctx-M25ctx-M45subgroups</td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2, 4 - 9, 13, 14, 16 - 21, 24-27, 31, 35, 38 - 41, 43 -51, 56, 59, 63, 65, 67, 73 -78, 81, 84 - 87, 89 - 95, 97100, 102, 104 - 106, 110 -113, 115, 121 - 126, 129 -131, 134, 137, 141, 147,148, 152, 159-161, 165,168, 171, 174, 185, 191,192, 195, 196, 198, 199,201, 214, 215, 217, 219,221, 223'</td></tr><tr><td colspan="1" rowspan="1">kpc</td><td colspan="1" rowspan="1">1 - 39</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">ndm</td><td colspan="1" rowspan="1">1 - 24, 27</td><td colspan="1" rowspan="1">-Y</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">oxa:oxa-23</td><td colspan="1" rowspan="1">23, 27, 49, 73, 134, 146,165 - 171, 225, 239, 366,398, 422, 423, 435, 440,469, 481- 483, 565, 657,806 - 808, 810, 813 - 815,816, 818</td><td colspan="1" rowspan="1">103, 133, 809, 811, 812,816</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">oxa:oxa-24</td><td colspan="1" rowspan="1">24 - 26, 33, 40, 72, 139,160, 207, 437, 653</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">oxa:oxa-48</td><td colspan="1" rowspan="1">48, 48b, 162, 163, 181,199, 232, 244, 245, 247,252, 370, 405, 416, 438,439, 484, 505, 514, 515,517, 519, 538, 546, 547,566, 567, 731, 788, 793</td><td colspan="1" rowspan="1">204</td><td colspan="1" rowspan="1">54, 436, 535</td></tr><tr><td colspan="1" rowspan="1">oxa:oxa-58</td><td colspan="1" rowspan="1">58, 96, 97, 164, 397, 467,512</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">420</td></tr><tr><td colspan="1" rowspan="1">tem1</td><td colspan="1" rowspan="1">1 - 4, 6, 8 - 12, 15 - 17, 1922, 24, 26, 28 - 30, 32 - 36,40, 43, 45, 47 - 49, 52 - 55,57, 60, 63, 67, 68, 70 - 72,76 - 88, 90 - 99, 101, 102,104 - 116, 120 - 139, 141'150, 152 - 160, 162 - 164,166 - 169, 171, 176, 177,181 - 199, 201, 204 - 217,219, 220, 224 - 228, 229 -235, 237</td><td colspan="1" rowspan="1">151</td><td colspan="1" rowspan="1">178</td></tr><tr><td colspan="1" rowspan="1">vim</td><td colspan="1" rowspan="1">1 - 6, 8 - 12, 14 - 20, 23 -46, 48 - 54, 56, 58, 60, 62</td><td colspan="1" rowspan="1">57, 59</td><td colspan="1" rowspan="1">7, 13, 47, 61</td></tr></table>

1 NOTE: H. influenzae commonly hosts tem-1. tem will only be reported if $H .$ . influenzae is concurrently detected.

# 4. Analytical Specificity/Cross-reactivity:

An analytical specificity study was conducted to evaluate LRT BAL Application targets for potential cross-reactivity with common respiratory flora microorganisms and species closely related to LRT BAL analytes. Testing was performed in duplicate using samples prepared at a worst case concentration (typically: $1 . 5 \mathrm { x } \ \mathrm { 1 0 ^ { 7 } C F U / m L }$ for bacteria and $1 . 0 \bar { \mathrm { ~ x ~ } } 1 0 ^ { 7 }$ copies/mL for viruses) by spiking microorganisms into $0 . 2 5 \mathrm { x }$ ARM (artificial respiratory matrix). Study results are presented in Table 8.

Cross-reactivity was observed with the following microorganisms with the Haemophilus influenzae assay of the LRT BAL Application:

Haemophilus haemolyticus (ATCC 33390) – at LoD Aggregatibacter aphrophilus (ATCC 19415) - at 10x LoD Haemophilus parainfluenzae (ATCC 33392) - at $> 1 0 0 \mathrm { x }$ LoD (> 107 CFU/mL).

Table 8: Analytical Specificity   

<table><tr><td colspan="1" rowspan="1">Strain</td><td colspan="1" rowspan="1">TestConcentration</td><td colspan="1" rowspan="1">Cross-Reactivity</td></tr><tr><td colspan="1" rowspan="1">Respiratory Flora Microorganisms</td><td colspan="1" rowspan="1">[in CFU/mLor other, asspecified infootnotes]</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Actinomyces odontolyticus</td><td colspan="1" rowspan="1">1.5 x 107</td><td colspan="1" rowspan="1">no</td></tr><tr><td colspan="1" rowspan="1">Aspergillus fumigatus</td><td colspan="1" rowspan="1">1.5 x 107</td><td colspan="1" rowspan="1">no</td></tr><tr><td colspan="1" rowspan="1">Candida albicans</td><td colspan="1" rowspan="1">1.5 x 107</td><td colspan="1" rowspan="1">no</td></tr><tr><td colspan="1" rowspan="1">Candida dubliniensis</td><td colspan="1" rowspan="1">1.5 x 107</td><td colspan="1" rowspan="1">no</td></tr><tr><td colspan="1" rowspan="1">Candida glabrata</td><td colspan="1" rowspan="1">1.5 x 107</td><td colspan="1" rowspan="1">no</td></tr><tr><td colspan="1" rowspan="1">Candida krusei</td><td colspan="1" rowspan="1">1.5 x 107</td><td colspan="1" rowspan="1">no</td></tr><tr><td colspan="1" rowspan="1">Candida parapsilosis</td><td colspan="1" rowspan="1">1.5 x 107</td><td colspan="1" rowspan="1">no</td></tr><tr><td colspan="1" rowspan="1">Candida tropicalis</td><td colspan="1" rowspan="1">1.5 x 107</td><td colspan="1" rowspan="1">no</td></tr><tr><td colspan="1" rowspan="1">Cardiobacterium hominis</td><td colspan="1" rowspan="1">1.5 x 107</td><td colspan="1" rowspan="1">no</td></tr><tr><td colspan="1" rowspan="1">Eikenella corrodens</td><td colspan="1" rowspan="1">1.5 x 107</td><td colspan="1" rowspan="1">no</td></tr><tr><td colspan="1" rowspan="1">Enterococcus faecalis</td><td colspan="1" rowspan="1">1.5 x 107</td><td colspan="1" rowspan="1">no</td></tr><tr><td colspan="1" rowspan="1">Enterococcus faecium</td><td colspan="1" rowspan="1">1.5 x 107</td><td colspan="1" rowspan="1">no</td></tr><tr><td colspan="1" rowspan="1">Fusobacterium nucleatum</td><td colspan="1" rowspan="1">5.0 x 106d</td><td colspan="1" rowspan="1">no</td></tr><tr><td colspan="1" rowspan="1">Granulicatella adiacens</td><td colspan="1" rowspan="1">1.5 x 107</td><td colspan="1" rowspan="1">no</td></tr><tr><td colspan="1" rowspan="1">Kingella kingae</td><td colspan="1" rowspan="1">1.5 x 107</td><td colspan="1" rowspan="1">no</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus acidophilus</td><td colspan="1" rowspan="1">1.5 x 107</td><td colspan="1" rowspan="1">no</td></tr><tr><td colspan="1" rowspan="1">Micrococcus luteus</td><td colspan="1" rowspan="1">1.5 x 107</td><td colspan="1" rowspan="1">no</td></tr><tr><td colspan="1" rowspan="1">Mycobacterium bovis</td><td colspan="1" rowspan="1">1.5 x 107d</td><td colspan="1" rowspan="1">no</td></tr><tr><td colspan="1" rowspan="1">Mycoplasma orale</td><td colspan="1" rowspan="1">1.5 x 107</td><td colspan="1" rowspan="1">no</td></tr><tr><td colspan="1" rowspan="1">Neisseria lactamica</td><td colspan="1" rowspan="1">1.5 x 107 d</td><td colspan="1" rowspan="1">no</td></tr><tr><td colspan="1" rowspan="1">Neisseria sicca</td><td colspan="1" rowspan="1">1.5 x 107</td><td colspan="1" rowspan="1">no</td></tr><tr><td colspan="1" rowspan="1">Pantoae agglomerans</td><td colspan="1" rowspan="1">1.5 x 107</td><td colspan="1" rowspan="1">no</td></tr><tr><td colspan="1" rowspan="1">Peptostreptococcus stomatis</td><td colspan="1" rowspan="1">5.0 x 106 d</td><td colspan="1" rowspan="1">no</td></tr><tr><td colspan="1" rowspan="1">Porphyromonas gingivalis</td><td colspan="1" rowspan="1">1.5 x 107d</td><td colspan="1" rowspan="1">no</td></tr><tr><td colspan="1" rowspan="1">Prevotella buccalis</td><td colspan="1" rowspan="1">1.5 x 107</td><td colspan="1" rowspan="1">no</td></tr><tr><td colspan="1" rowspan="1">Raoultella planticola</td><td colspan="1" rowspan="1">1.5 x 107</td><td colspan="1" rowspan="1">no</td></tr><tr><td colspan="1" rowspan="1">Close Neighbor Microorganisms</td><td colspan="1" rowspan="1">[in CFU/mL]</td><td colspan="1" rowspan="1">cYanmaGlb</td></tr><tr><td colspan="1" rowspan="1">Acinetobacter ursingii</td><td colspan="1" rowspan="1">1.5 x 107</td><td colspan="1" rowspan="1">no</td></tr><tr><td colspan="1" rowspan="1">Aggregatibacteractinomycetemcomitans</td><td colspan="1" rowspan="1">1.5 x 107 d</td><td colspan="1" rowspan="1">no</td></tr><tr><td colspan="1" rowspan="1">Aggregatibacter aphrophilus</td><td colspan="1" rowspan="1">1.5 x 107</td><td colspan="1" rowspan="1">yes (H. influenzae) a</td></tr><tr><td colspan="1" rowspan="1">Citrobacter koseri</td><td colspan="1" rowspan="1">1.5 x 107</td><td colspan="1" rowspan="1">no</td></tr><tr><td colspan="1" rowspan="1">Haemophilus haemolyticus</td><td colspan="1" rowspan="1">1.5 x 107</td><td colspan="1" rowspan="1">yes (H. influenzae) </td></tr><tr><td colspan="1" rowspan="1">Haemophilus parahaemolyticus</td><td colspan="1" rowspan="1">1.5 x 107</td><td colspan="1" rowspan="1">no</td></tr><tr><td colspan="1" rowspan="1">Haemophilus parainfluenzae</td><td colspan="1" rowspan="1">1.5 x 107</td><td colspan="1" rowspan="1">yes (H. influenzae)c</td></tr><tr><td colspan="1" rowspan="1">Legionella longbeachae</td><td colspan="1" rowspan="1">1.5 x 107</td><td colspan="1" rowspan="1">no</td></tr><tr><td colspan="1" rowspan="1">Legionella/Tatlockia micdadei</td><td colspan="1" rowspan="1">1.5 x 107</td><td colspan="1" rowspan="1">no</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus capitis</td><td colspan="1" rowspan="1">1.5 x 107</td><td colspan="1" rowspan="1">no</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus epidermidis</td><td colspan="1" rowspan="1">1.5 x 107</td><td colspan="1" rowspan="1">no</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus haemolyticus</td><td colspan="1" rowspan="1">1.5 x 107</td><td colspan="1" rowspan="1">no</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus lugdunensis</td><td colspan="1" rowspan="1">1.5 x 107</td><td colspan="1" rowspan="1">no</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus saprophyticus</td><td colspan="1" rowspan="1">1.5 x 107</td><td colspan="1" rowspan="1">no</td></tr><tr><td colspan="1" rowspan="1">Streptococcus agalactiae</td><td colspan="1" rowspan="1">1.5 x 107</td><td colspan="1" rowspan="1">no</td></tr><tr><td colspan="1" rowspan="1">Streptococcus anginosus</td><td colspan="1" rowspan="1">1.5 x 107</td><td colspan="1" rowspan="1">no</td></tr><tr><td colspan="1" rowspan="1">Streptococcus dysgalactiae</td><td colspan="1" rowspan="1">1.5 x 107</td><td colspan="1" rowspan="1">no</td></tr><tr><td colspan="1" rowspan="1">Streptococcus gordonii</td><td colspan="1" rowspan="1">1.5 x 107</td><td colspan="1" rowspan="1">no</td></tr><tr><td colspan="1" rowspan="1">Streptococcus intermedius</td><td colspan="1" rowspan="1">1.5 x 107</td><td colspan="1" rowspan="1">no</td></tr><tr><td colspan="1" rowspan="1">Streptococcus mitis</td><td colspan="1" rowspan="1">1.5 x 107</td><td colspan="1" rowspan="1">no</td></tr><tr><td colspan="1" rowspan="1">Streptococcus mutans</td><td colspan="1" rowspan="1">1.5 x 107</td><td colspan="1" rowspan="1">no</td></tr><tr><td colspan="1" rowspan="1">Streptococcus oralis</td><td colspan="1" rowspan="1">1.5 x 107</td><td colspan="1" rowspan="1">no</td></tr><tr><td colspan="1" rowspan="1">Streptococcus parasanguinis</td><td colspan="1" rowspan="1">1.5 x 107</td><td colspan="1" rowspan="1">no</td></tr><tr><td colspan="1" rowspan="1">Streptococcus pseudopneumoniae</td><td colspan="1" rowspan="1">1.5 x 107</td><td colspan="1" rowspan="1">no</td></tr><tr><td colspan="1" rowspan="1">Streptococcus pyogenes</td><td colspan="1" rowspan="1">1.5 x 107</td><td colspan="1" rowspan="1">no</td></tr><tr><td colspan="1" rowspan="1">Streptococcus salivarius</td><td colspan="1" rowspan="1">1.5 x 107</td><td colspan="1" rowspan="1">no</td></tr><tr><td colspan="1" rowspan="1">Streptococcus sanguinis</td><td colspan="1" rowspan="1">1.5 x 107</td><td colspan="1" rowspan="1">no</td></tr><tr><td colspan="1" rowspan="1">Streptococcus vestibularis</td><td colspan="1" rowspan="1">1.5 x 107</td><td colspan="1" rowspan="1">no</td></tr><tr><td colspan="1" rowspan="1">Respiratory Viruses</td><td colspan="2" rowspan="1">[in copies/mL]</td></tr><tr><td colspan="1" rowspan="1">Adenovirus 41</td><td colspan="1" rowspan="1">1.0 x 105</td><td colspan="1" rowspan="1">no</td></tr><tr><td colspan="1" rowspan="1">Enterovirus 68</td><td colspan="1" rowspan="1">1.0 x 105</td><td colspan="1" rowspan="1">no</td></tr><tr><td colspan="1" rowspan="1">Enterovirus 71</td><td colspan="1" rowspan="1">1.0 x 105</td><td colspan="1" rowspan="1">no</td></tr><tr><td colspan="1" rowspan="1">Parainfluenzae Virus 1</td><td colspan="1" rowspan="1">1.0 x 105</td><td colspan="1" rowspan="1">no</td></tr><tr><td colspan="1" rowspan="1">Parainfluenzae Virus 2</td><td colspan="1" rowspan="1">1.0 x 105</td><td colspan="1" rowspan="1">no</td></tr><tr><td colspan="1" rowspan="1">Parainfluenzae Virus 3</td><td colspan="1" rowspan="1">1.0 x 105</td><td colspan="1" rowspan="1">no</td></tr><tr><td colspan="1" rowspan="1">Parainfluenzae Virus 4</td><td colspan="1" rowspan="1">1.0 x 105</td><td colspan="1" rowspan="1">no</td></tr><tr><td colspan="1" rowspan="1">Rhinovirus</td><td colspan="1" rowspan="1">1.0 x 105</td><td colspan="1" rowspan="1">no</td></tr><tr><td colspan="1" rowspan="1">RSV A</td><td colspan="1" rowspan="1">1.0 x 105</td><td colspan="1" rowspan="1">no</td></tr><tr><td colspan="1" rowspan="1">RSVB</td><td colspan="1" rowspan="1">1.0 x 105</td><td colspan="1" rowspan="1">no</td></tr></table>

a For $A$ . aphrophilus, cross-reactivity to $H .$ . influenzae was observed down to a concentration of 2.0 $\mathrm { ~ x ~ } 1 0 ^ { 5 } \mathrm { C F U / m L }$ (1/2 tests positive). For a concentration of $1 . 0 \mathrm { ~ x ~ } 1 0 ^ { 5 } \mathrm { C F U / m L }$ (equivalent to ${ 5 } \mathbf { x }$ LoD of $H .$ . influenzae), cross-reactivity was no longer observed. According to BLAST analysis, cross-reactivity of A. aphrophilus to $H .$ influenzae is only predicted for some strains (including the tested reference strain ATCC 19415), but not for other strains. b For $H .$ . haemolyticus, cross-reactivity to $H .$ . influenzae was observed down to a concentration of $4 . 0 \mathrm { ~ x ~ } 1 0 ^ { 4 }$ CFU/mL (2/2 tests positive, equivalent to $2 \mathbf { x }$ LoD of $H .$ . influenzae). Sequence comparison by BLAST for $H .$ . haemolyticus to $H .$ . influenzae shows a $_ { 3 } ,$ mismatch to one of the assay primers, while the second primer and all internal array probes show a full match. Therefore, the observed cross-reactivity is supported by BLAST analysis. c For $H .$ parainfluenzae, cross-reactivity to H. influenzae was observed only at the tested worstcase concentration of $1 . 5 \mathrm { ~ x ~ } 1 0 ^ { 7 } \mathrm { C F U / m L }$ (2/4 tests positive, equivalent to $7 5 0 \mathrm { x }$ LoD of $H .$ . influenzae). Additional tests at $2 . 0 \mathrm { ~ x ~ } 1 0 ^ { 6 } \mathrm { C F U / m L }$ (equivalent to $1 0 0 \mathrm { x }$ LoD of $H .$ . influenzae), cross-reactivity was no longer observed. d Tested as DNA extract (in copies $/ \mathrm { m L }$ ). e Tested as DNA or RNA extract (in copies $/ \mathrm { m L }$ ).

Additionally, in silico analysis (GenBank BLAST) was performed to assess for potential cross-reactivity of non-target microorganisms that may be present in BAL specimens. Table 9 includes predicted cross-reactivity of applicable LRT BAL microorganisms based on in silico analysis, exclusivity wet testing and false positive results observed during the prospective and archived studies.

Table 9: In silico Prediction of Cross-reactivity, Wet Testing, Cross-reactivity Observed in Clinical Study   

<table><tr><td colspan="1" rowspan="1">Close Neighbor Strain a</td><td colspan="1" rowspan="1">Cross-Reactivity Prediction(in silico analysis)</td><td colspan="1" rowspan="1">Wet Testing Result</td><td colspan="1" rowspan="1">Cross-ReactionsObserved in ClinicalStudy(N = 1,408 prosp. orarch. specimens)</td></tr><tr><td colspan="4" rowspan="1">Citrobacter freundii</td></tr><tr><td colspan="1" rowspan="1">Citrobacter braakii</td><td colspan="1" rowspan="1">Detection predicted at higherthan LoD concentrations(certain strains) /</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Detection not predicted (otherstrains)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Citrobacter pasteurii</td><td colspan="1" rowspan="1">Detection predicted at higherthan LoD concentrations</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Citrobacter werkmannii</td><td colspan="1" rowspan="1">Detection predicted at higherthan LoD concentrations(certain strains) /Detection not predicted (otherstrains)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Citrobacter youngae</td><td colspan="1" rowspan="1">Detection predicted at LoD(certain strains) /Detection predicted at higherthan LoD concentrations(other strains) c</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">−</td></tr><tr><td colspan="1" rowspan="1">Kluyvera georgiana</td><td colspan="1" rowspan="1">Detection predicted at higherthan LoD concentrations</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Citrobacter koseri</td><td colspan="1" rowspan="1">Detection not predicted</td><td colspan="1" rowspan="1">negativeATCC 27156</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="4" rowspan="1">Escherichia coli</td></tr><tr><td colspan="1" rowspan="1">Shigella dysenteriae b</td><td colspan="1" rowspan="1">Detection predicted at LoD</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Shigella boydii b</td><td colspan="1" rowspan="1">Detection predicted at LoD</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Shigella flexneri b</td><td colspan="1" rowspan="1">Detection predicted at LoD</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Shigella sonnei </td><td colspan="1" rowspan="1">Detection predicted at LoD</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Escherichia albertii</td><td colspan="1" rowspan="1">Detection predicted at LoD</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Escherichia fergusonii</td><td colspan="1" rowspan="1">Detection predicted at LoD</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="4" rowspan="1">Haemophilus influenzae</td></tr><tr><td colspan="1" rowspan="1">Haemophilus haemolyticus</td><td colspan="1" rowspan="1">Detection predicted at higherthan LoD concentrations</td><td colspan="1" rowspan="1">positiveATCC 33390</td><td colspan="1" rowspan="1">5</td></tr><tr><td colspan="1" rowspan="1">Haemophilus parahaemolyticus</td><td colspan="1" rowspan="1">Detection not predicted</td><td colspan="1" rowspan="1">negativeATCC 10014</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Haemophilus parainfluenzae</td><td colspan="1" rowspan="1">Detection not predicted</td><td colspan="1" rowspan="1">positive at worst-caseconcentration onlyATCC 33392</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Haemophilus aegyptius </td><td colspan="1" rowspan="1">Detection predicted at LoD</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Aggregatibacter actino-mycetemcomitans</td><td colspan="1" rowspan="1">Detection not predicted</td><td colspan="1" rowspan="1">negativeATCC 33384</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Aggregatibacter aphrophilus</td><td colspan="1" rowspan="1">Detection predicted at higherthan LoD concentrations(certain strains) /Detection not predicted (otherstrains)</td><td colspan="1" rowspan="1">positiveATCC 19415</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">Aggregatibacter segnis</td><td colspan="1" rowspan="1">Detection predicted at higherthan LoD concentrations</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="4" rowspan="1">Klebsiella oxytoca</td></tr><tr><td colspan="1" rowspan="1">Klebsiella michiganensis</td><td colspan="1" rowspan="1">Detection predicted at LoD(certain strains) /</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Close Neighbor Strain a</td><td colspan="1" rowspan="1">Cross-Reactivity Prediction(in silico analysis)</td><td colspan="2" rowspan="1">Wet Testing Result</td><td colspan="1" rowspan="1">Cross-ReactionsObserved in ClinicalStudy(N = 1,408 prosp. orarch. specimens)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Detection predicted at higherthan LoD concentrations(other strains) c</td><td colspan="2" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="5" rowspan="1">Klebsiella pneumoniae</td></tr><tr><td colspan="1" rowspan="1">Klebsiella quasivariicola</td><td colspan="1" rowspan="1">Detection predicted at higherthan LoD concentrations</td><td colspan="2" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="5" rowspan="1">Staphylococcus aureus</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus argenteus</td><td colspan="1" rowspan="1">Detection predicted at LoD</td><td colspan="2" rowspan="1">-</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">CNS:S. epidermidisS. capitisS. lugdunensisS. haemolyticusS. saprophyticus</td><td colspan="1" rowspan="1">Detection not predicted</td><td colspan="2" rowspan="1">negativeATCC 51625ATCC 27840ATCC 43809ATCC 29970ATCC 15305</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="5" rowspan="1">Streptococcus pneumoniae</td></tr><tr><td colspan="1" rowspan="2">Other Streptococcus spp.:S. agalactiaeS. anginosusS. dysgalactiaeS. gordoniiS. intermediusS. mitisS. mutansS. oralisS. parasanguinisS. pseudopneumoniaeS. pyogenesS. salivariusS. sanguinisS. vestibularis</td><td colspan="1" rowspan="2">Detection not predicted</td><td colspan="2" rowspan="2">negativeATCC 13813ATCC 33397ATCC 43078ATCC 10558ATCC 27335ATCC 49456ATCC 25175ATCC 35037ATCC 15912ATCC BAA-960ATCC 12344ATCC 7073ATCC 10556ATCC 49124</td><td colspan="1" rowspan="2"></td></tr><tr><td colspan="1" rowspan="1"></td></tr></table>

a For several analyte assays, soil, environmental, plant or animal derived close neighbor strains are also predicted at LoD or at higher than LoD concentrations (Citrobacter freundii:, C. portucalensis, Enterobacter cloacae complex: E. soli, E. mori, E. nickellidurans; Escherichia coli: $E$ . marmotae; Staphylococcus aureus: S. schweitzeri, S. simiae; Stenotrophomonas maltophilia: S. nitritireducens, S. daejeonensis, S. acidaminiphila, S. koreensis, S. rhizophila, Xanthomonas spp., Pseudoxanthomonas spp.). b Clinical relevance unlikely for respiratory infections c Predictions for available strains distribute into different categories.

5. Interference Testing:

Interference testing was performed previously with the Unyvero LRT Application which relies on the same DNA extraction procedures and chemistry and targets the same microorganism and resistance marker sequences as the LRT BAL Application. Refer to DEN170047 for study results.

6. Traceability, Stability, Expected Values (Controls, Calibrators, or Methods): Sample Stability:

A sample stability study was conducted to evaluated assay performance differences between fresh samples and samples stored at $2 { - } 8 ^ { \circ } \mathrm { C }$ for at least 24 hours. Samples for the study were prepared at low and moderate organism concentrations inoculated into pooled natural negative BAL matrix. Representative targeted microorganisms were evaluated in the study. Four targeted resistance markers carried by selected strains were also evaluated.

Pair-wise tests with the LRT BAL Application were performed for samples tested within 2 hours after test sample preparation (Time 0) and samples tested after a total storage of at least 24 hours including at least 2 hours at a worst-case ambient temperature of $3 0 \mathrm { { } ^ { \circ } C }$ and a minimum of 22 hour storage at $2 { - } 8 ^ { \circ } \mathrm { C }$ .

Tables 10 to 12 summarize the sample stability test results for each concentration level (moderate, low, negative) for both test conditions (tested freshly, tested after storage), together with the agreement rates based on comparison to expected results.

Results of the study support the claimed specimen storage and transport conditions of $2 { - } 8 ^ { \circ } \mathrm { C }$ for up to 24 hours prior to testing with the LRT BAL Application.

Table 10: Sample Stability, Moderate Positive (\~5x LoD), 24 hour storage at ${ \bf 2 { - } 8 ^ { \circ } C }$   
Table 11: Sample Stability, Low Positive ( ${ \sim } 2 \mathbf { x }$ LoD), 24 hr storage at ${ \bf 2 { - } 8 ^ { \circ } C }$   

<table><tr><td rowspan=2 colspan=1>Moderate</td><td rowspan=2 colspan=1>x-foldLoD</td><td rowspan=2 colspan=1>TestCondition</td><td rowspan=1 colspan=3>Agreement with ExpectedResult</td></tr><tr><td rowspan=1 colspan=1># Pos.# Exp</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=1>Acinetobacter baumanniiATCC 19606</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>fresh24 hrs</td><td rowspan=1 colspan=1>10/1011/11</td><td rowspan=1 colspan=1>100.0100.0</td><td rowspan=1 colspan=1>72.3 - 100.074.1 - 100.0</td></tr><tr><td rowspan=1 colspan=1>Chlamydia pneumoniaeATCC VR-2282</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>fresh24 hrs</td><td rowspan=1 colspan=1>10/1011/11</td><td rowspan=1 colspan=1>100.0100.0</td><td rowspan=1 colspan=1>72.3 - 100.074.1 - 100.0</td></tr><tr><td rowspan=1 colspan=1>Citrobacter freundiiATCC 8090</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>fresh24 hrs</td><td rowspan=1 colspan=1>10/1011/11</td><td rowspan=1 colspan=1>100.0100.0</td><td rowspan=1 colspan=1>72.3 - 100.074.1 - 100.0</td></tr><tr><td rowspan=1 colspan=1>Haemophilus influenzaeATCC 33391</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>fresh24 hrs</td><td rowspan=1 colspan=1>10/1011/11</td><td rowspan=1 colspan=1>100.0100.0</td><td rowspan=1 colspan=1>72.3 - 100.074.1 - 100.0</td></tr><tr><td rowspan=1 colspan=1>Klebsiella pneumoniaeNCTC 13443</td><td rowspan=1 colspan=1>2.5</td><td rowspan=1 colspan=1>fresh24 hrs</td><td rowspan=1 colspan=1>10/1011/11</td><td rowspan=1 colspan=1>100.0100.0</td><td rowspan=1 colspan=1>72.3 - 100.074.1 - 100.0</td></tr><tr><td rowspan=1 colspan=1>ctx-MNCTC 13443</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>fresh24 hrs</td><td rowspan=1 colspan=1>10/1011/11</td><td rowspan=1 colspan=1>100.0100.0</td><td rowspan=1 colspan=1>72.3 - 100.074.1 - 100.0</td></tr><tr><td rowspan=1 colspan=1>ndmNCTC 13443</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>fresh24 hrs</td><td rowspan=1 colspan=1>9/1011/11</td><td rowspan=1 colspan=1>90.0100.0</td><td rowspan=1 colspan=1>59.6 - 98.274.1 - 100.0</td></tr><tr><td rowspan=1 colspan=1>temNCTC 13443</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>fresh24 hrs</td><td rowspan=1 colspan=1>10/1011/11</td><td rowspan=1 colspan=1>100.0100.0</td><td rowspan=1 colspan=1>72.3 - 100.074.1 - 100.0</td></tr><tr><td rowspan=1 colspan=1>Morganella morganiiATCC 25830</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>fresh24 hrs</td><td rowspan=1 colspan=1>10/1011/11</td><td rowspan=1 colspan=1>100.0100.0</td><td rowspan=1 colspan=1>72.3 - 100.074.1 - 100.0</td></tr><tr><td rowspan=1 colspan=1>Proteus vulgarisATCC 29905</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>fresh24 hrs</td><td rowspan=1 colspan=1>10/1011/11</td><td rowspan=1 colspan=1>100.0100.0</td><td rowspan=1 colspan=1>72.3 - 100.074.1 - 100.0</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus aureusNCT 12493</td><td rowspan=1 colspan=1>8.3</td><td rowspan=1 colspan=1>fresh24 hrs</td><td rowspan=1 colspan=1>9/9 *11/11</td><td rowspan=1 colspan=1>100.0100.0</td><td rowspan=1 colspan=1>70.1 - 100.074.1 - 100.0</td></tr><tr><td rowspan=1 colspan=1>mecANCTC 12493</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>fresh24 hrs</td><td rowspan=1 colspan=1>10/1011/11</td><td rowspan=1 colspan=1>100.0100.0</td><td rowspan=1 colspan=1>72.3 - 100.074.1 - 100.0</td></tr><tr><td rowspan=1 colspan=2>total (all analytes)</td><td rowspan=1 colspan=1>fresh24 hrs</td><td rowspan=1 colspan=1>118/119132/132</td><td rowspan=1 colspan=1>99.2100.0</td><td rowspan=1 colspan=1>95.4 - 99.997.2 - 100.0</td></tr></table>

$^ { \mathfrak { s } } =$ reduced number of available results due to invalid analyte results.

<table><tr><td rowspan=2 colspan=1>Low</td><td rowspan=2 colspan=1>x-foldLoD</td><td rowspan=2 colspan=1>TestCondition</td><td rowspan=1 colspan=3>Agreement with ExpectedResult</td></tr><tr><td rowspan=1 colspan=1># Pos.# xp.</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=1>Acinetobacter baumanniiATCC 19606</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>fresh24 hrs</td><td rowspan=1 colspan=1>31/3231/32</td><td rowspan=1 colspan=1>96.996.9</td><td rowspan=1 colspan=1>84.3 - 99.484.3 - 99.4</td></tr><tr><td rowspan=1 colspan=1>Chlamydia pneumoniaeATCC VR-2282</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>fresh24 hrs</td><td rowspan=1 colspan=1>32/3232/32</td><td rowspan=1 colspan=1>100.0100.0</td><td rowspan=1 colspan=1>89.3 - 100.089.3 - 100.0</td></tr><tr><td rowspan=1 colspan=1>Citrobacter freundiiATCC 8090</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>fresh24 hrs</td><td rowspan=1 colspan=1>31/3229/32</td><td rowspan=1 colspan=1>96.990.6</td><td rowspan=1 colspan=1>84.3 - 99.475.8 - 96.8</td></tr><tr><td rowspan=1 colspan=1>Haemophilus influenzaeATCC 33391</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>fresh24 hrs</td><td rowspan=1 colspan=1>29/3131/31 *</td><td rowspan=1 colspan=1>93.5100.0</td><td rowspan=1 colspan=1>79.3 - 98.289.0 - 100.0</td></tr><tr><td rowspan=1 colspan=1>Klebsiella pneumoniaeNCTC 13443</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>fresh24 hrs</td><td rowspan=1 colspan=1>31/3232/32</td><td rowspan=1 colspan=1>96.9100.0</td><td rowspan=1 colspan=1>84.3 - 99.489.3 - 100.0</td></tr><tr><td rowspan=1 colspan=1>ctx-MNCTC 13443</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>fresh24 hrs</td><td rowspan=1 colspan=1>32/3232/32</td><td rowspan=1 colspan=1>100.0100.0</td><td rowspan=1 colspan=1>89.3 - 100.089.3 - 100.0</td></tr><tr><td rowspan=1 colspan=1>ndmNCTC 13443</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>fresh24 hrs</td><td rowspan=1 colspan=1>31/3231/32</td><td rowspan=1 colspan=1>96.996.9</td><td rowspan=1 colspan=1>84.3 - 99.484.3 - 99.4</td></tr><tr><td rowspan=1 colspan=1>temNCTC 13443</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>fresh24 hrs</td><td rowspan=1 colspan=1>30/3130/31 *</td><td rowspan=1 colspan=1>96.896.8</td><td rowspan=1 colspan=1>83.8 - 99.483.8 - 99.4</td></tr><tr><td rowspan=1 colspan=1>Morganella morganiiATCC 25830</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>fresh24 hrs</td><td rowspan=1 colspan=1>32/3231/32</td><td rowspan=1 colspan=1>100.096.9</td><td rowspan=1 colspan=1>89.3 - 100.084.3 - 99.4</td></tr><tr><td rowspan=1 colspan=1>Proteus vulgarisATCC 29905</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>fresh24 hrs</td><td rowspan=1 colspan=1>32/3230/30</td><td rowspan=1 colspan=1>100.0100.0</td><td rowspan=1 colspan=1>89.3 - 100.088.7 - 100.0</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus aureusNCTC 12493</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>fresh24 hrs</td><td rowspan=1 colspan=1>32/3232/32</td><td rowspan=1 colspan=1>100.0100.0</td><td rowspan=1 colspan=1>89.3 - 100.089.3 - 100.0</td></tr><tr><td rowspan=1 colspan=1>mecANCTC 12493</td><td rowspan=1 colspan=1>1.3</td><td rowspan=1 colspan=1>fresh24 hrs</td><td rowspan=1 colspan=1>32/3232/32</td><td rowspan=1 colspan=1>100.0100.0</td><td rowspan=1 colspan=1>89.3 - 100.089.3 - 100.0</td></tr><tr><td rowspan=1 colspan=2>total (all analytes)</td><td rowspan=1 colspan=1>fresh24 hrs</td><td rowspan=1 colspan=1>375/382373/380</td><td rowspan=1 colspan=1>98.298.2</td><td rowspan=1 colspan=1>96.3 - 99.196.2 - 99.1</td></tr></table>

$^ { * } =$ reduced number of available results due to invalid analyte results.

Table 12: Sample Stability, Negative Samples, 24 hr storage at $\bf { 2 . 8 ^ { \circ } C }$   

<table><tr><td colspan="1" rowspan="2">Negative</td><td colspan="1" rowspan="2">TestCondition</td><td colspan="3" rowspan="1">Agreement with ExpectedResult</td></tr><tr><td colspan="1" rowspan="1"># Neg. Exp.</td><td colspan="1" rowspan="1">%</td><td colspan="1" rowspan="1">95% CI</td></tr><tr><td colspan="1" rowspan="1">Acinetobacter baumanniiATCC 19606</td><td colspan="1" rowspan="1">fresh24 hrs</td><td colspan="1" rowspan="1">11/1111/11</td><td colspan="1" rowspan="1">100.0100.0</td><td colspan="1" rowspan="1">74.1 - 100.074.1 - 100.0</td></tr><tr><td colspan="1" rowspan="1">Chlamydia pneumoniaeATCC VR-2282</td><td colspan="1" rowspan="1">fresh24 hrs</td><td colspan="1" rowspan="1">11/1111/11</td><td colspan="1" rowspan="1">100.0100.0</td><td colspan="1" rowspan="1">74.1 - 100.074.1 - 100.0</td></tr><tr><td colspan="1" rowspan="1">Citrobacter freundiiATCC 8090</td><td colspan="1" rowspan="1">fresh24 hrs</td><td colspan="1" rowspan="1">11/1111/11</td><td colspan="1" rowspan="1">100.0100.0</td><td colspan="1" rowspan="1">74.1 - 100.074.1 - 100.0</td></tr><tr><td colspan="1" rowspan="1">Haemophilus influenzaeATCC 33391</td><td colspan="1" rowspan="1">fresh24 hrs</td><td colspan="1" rowspan="1">11/1111/11</td><td colspan="1" rowspan="1">100.0100.0</td><td colspan="1" rowspan="1">74.1 - 100.074.1 - 100.0</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniaeNCTC 13443</td><td colspan="1" rowspan="1">fresh24 hrs</td><td colspan="1" rowspan="1">11/1111/11</td><td colspan="1" rowspan="1">100.0100.0</td><td colspan="1" rowspan="1">74.1 - 100.074.1 - 100.0</td></tr><tr><td colspan="1" rowspan="1">ctx-MNCTC 13443</td><td colspan="1" rowspan="1">fresh24 hrs</td><td colspan="1" rowspan="1">10/10 *11/11</td><td colspan="1" rowspan="1">100.0100.0</td><td colspan="1" rowspan="1">72.3 - 100.074.1 - 100.0</td></tr><tr><td colspan="1" rowspan="1">ndmNCTC 13443</td><td colspan="1" rowspan="1">fresh24 hrs</td><td colspan="1" rowspan="1">11/1111/11</td><td colspan="1" rowspan="1">100.0100.0</td><td colspan="1" rowspan="1">74.1 - 100.074.1 - 100.0</td></tr><tr><td colspan="1" rowspan="1">temNCTC 13443</td><td colspan="1" rowspan="1">fresh24 hrs</td><td colspan="1" rowspan="1">11/1111/11</td><td colspan="1" rowspan="1">100.0100.0</td><td colspan="1" rowspan="1">74.1 - 100.074.1 - 100.0</td></tr><tr><td colspan="1" rowspan="1">Morganella morganiiATCC 25830</td><td colspan="1" rowspan="1">fresh24 hrs</td><td colspan="1" rowspan="1">10/10 *11/11</td><td colspan="1" rowspan="1">100.0100.0</td><td colspan="1" rowspan="1">72.3 - 100.074.1 - 100.0</td></tr><tr><td colspan="1" rowspan="1">Proteus vulgarisATCC 29905</td><td colspan="1" rowspan="1">fresh24 hrs</td><td colspan="1" rowspan="1">11/1111/11</td><td colspan="1" rowspan="1">100.0100.0</td><td colspan="1" rowspan="1">74.1 - 100.074.1 - 100.0</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureusNCT 12493</td><td colspan="1" rowspan="1">fresh24 hrs</td><td colspan="1" rowspan="1">11/1111/11</td><td colspan="1" rowspan="1">100.0100.0</td><td colspan="1" rowspan="1">74.1 - 100.074.1 - 100.0</td></tr><tr><td colspan="1" rowspan="1">mecANCTC12493</td><td colspan="1" rowspan="1">fresh24 hrs</td><td colspan="1" rowspan="1">11/1111/11</td><td colspan="1" rowspan="1">100.0100.0</td><td colspan="1" rowspan="1">74.1 - 100.074.1 - 100.0</td></tr><tr><td colspan="1" rowspan="1">total (all analytes)</td><td colspan="1" rowspan="1">fresh24 hrs</td><td colspan="1" rowspan="1">130/130132/132</td><td colspan="1" rowspan="1">100.0100.0</td><td colspan="1" rowspan="1">97.1 - 100.097.2 - 100.0</td></tr></table>

$^ { * } =$ reduced number of available results due to invalid analyte results.

# Sample Stability/Post-lysis

Package insert instructions indicate the user to load the Unyvero LRT BAL Cartridge immediately after lysis on the Unyvero Lysator or if needed, within one hour post-lysis. An analytical study was conducted to assess any differences in performance between immediate testing and after a one hour delay. Study samples were prepared with representative analytes close to their respective LoD concentrations. Results were compared for samples tested immediately after lysis and samples tested after a 1 hour or 4 hours delay between lysis completion and Analyzer start. A total of six replicates were evaluated per analyte and storage condition. The study demonstrated equivalent results for all test conditions.

# External Controls

Quantified stored frozen microorganism stocks comprising LoD reference strains for all LRT BAL analytes were used for preparation of positive controls that were regularly tested throughout clinical and analytical studies. A minimum of one strain for each microorganism target was included in the positive control scheme. All targeted resistance marker targets were also represented. Positive controls were prepared as six different microorganism pools with each analyte present at ${ \sim } 3 \mathrm { X }$ LoD.

In the clinical study, testing was performed in duplicate prior to the study, for every new cartridge lot and bi-weekly throughout the study. Negative controls consisted of pooled native negative BAL specimen matrix and were tested once per week throughout the clinical study. Of the 192 expected positive results from control testing conducted in the prospective clinical study, positive controls generated the expected positive results with the exception of two false negative S. pneumoniae results (out of 12 control run), one false positive $E$ . coli result and one invalid control run. All negative controls generated the expected negative result for all analytes.

7. Limit of Detection:

The limit of detection (LoD) for the Unyvero LRT BAL Application was established for each analyte targeted by the assay. The LoD was estimated by serial dilutions of reference strains prepared in pooled negative native BAL matrix. The LoD for each analyte was determined to be the lowest test concentration with a positivity rate of $9 5 \%$ or higher with at least 19 of 20 positive test results. Tables 13 and 14 summarize the LoD for each LRT BAL panel microorganism and antibiotic resistance marker target.

Table 13: Limit of Detection (LoD), Microorganism Targets   

<table><tr><td rowspan=1 colspan=1>Analyte</td><td rowspan=1 colspan=1>Reference Strain ID</td><td rowspan=1 colspan=1>LoD [CFU/mL]</td></tr><tr><td rowspan=1 colspan=1>Acinetobacter spp.</td><td rowspan=1 colspan=1>ATCC 19606(A. baumannii)</td><td rowspan=1 colspan=1>2.0 x 104</td></tr><tr><td rowspan=1 colspan=1>Chlamydia pneumoniae</td><td rowspan=1 colspan=1>ATCC VR-2282a</td><td rowspan=1 colspan=1>3.2 x 102</td></tr><tr><td rowspan=1 colspan=1>Citrobacter freundii</td><td rowspan=1 colspan=1>ATCC 8090</td><td rowspan=1 colspan=1>8.0 x 104</td></tr><tr><td rowspan=1 colspan=1>Enterobacter cloacae complex</td><td rowspan=1 colspan=1>ATCC 13047(E. cloacae)</td><td rowspan=1 colspan=1>2.0 x 105</td></tr><tr><td rowspan=1 colspan=1>Escherichia coli</td><td rowspan=1 colspan=1>ATCC 11775</td><td rowspan=1 colspan=1>2.0 x 104</td></tr><tr><td rowspan=1 colspan=1>Haemophilus influenzae</td><td rowspan=1 colspan=1>ATCC 33391</td><td rowspan=1 colspan=1>2.0 x 104</td></tr><tr><td rowspan=1 colspan=1>Klebsiella oxytoca</td><td rowspan=1 colspan=1>ATCC 13182</td><td rowspan=1 colspan=1>1.0 x 104</td></tr><tr><td rowspan=1 colspan=1>Klebsiella pneumoniae (var. 1)</td><td rowspan=1 colspan=1>ATCC 13883</td><td rowspan=1 colspan=1>4.0 x 104</td></tr><tr><td rowspan=1 colspan=1>Klebsiella quasipneumoniae(K. pneumoniae, var. 2)</td><td rowspan=1 colspan=1>ATCC 700603</td><td rowspan=1 colspan=1>4.0 x 104</td></tr><tr><td rowspan=1 colspan=1>Klebsiella variicola</td><td rowspan=1 colspan=1>ATCC BAA-830</td><td rowspan=1 colspan=1>2.0 x 104</td></tr><tr><td rowspan=1 colspan=1>Legionella pneumophila</td><td rowspan=1 colspan=1>ATCC 33152</td><td rowspan=1 colspan=1>8.0 x 104</td></tr><tr><td rowspan=1 colspan=1>Moraxella catarrhalis</td><td rowspan=1 colspan=1>ATCC 25238</td><td rowspan=1 colspan=1>1.5 x 105$</td></tr><tr><td rowspan=1 colspan=1>Morganella morganii</td><td rowspan=1 colspan=1>ATCC 25830</td><td rowspan=1 colspan=1>2.0 x 104</td></tr><tr><td rowspan=1 colspan=1>Mycoplasma pneumoniae</td><td rowspan=1 colspan=1>ATCC 29085b</td><td rowspan=1 colspan=1>1.6 x 103</td></tr><tr><td rowspan=1 colspan=1>Pneumocystis jirovecii</td><td rowspan=1 colspan=1>36_031</td><td rowspan=1 colspan=1>5.0 x 105</td></tr><tr><td rowspan=1 colspan=1>Proteus spp.</td><td rowspan=1 colspan=1>ATCC 29906(P. mirabilis)ATCC 29905(P. vulgaris)</td><td rowspan=1 colspan=1>5.0 x 103</td></tr><tr><td rowspan=1 colspan=1>Pseudomonas aeruginosa</td><td rowspan=1 colspan=1>ATCC 10145</td><td rowspan=1 colspan=1>6.3 x 102</td></tr><tr><td rowspan=1 colspan=1>Serratia marcescens</td><td rowspan=1 colspan=1>ATCC 13880</td><td rowspan=1 colspan=1>4.0 x 104</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus aureus</td><td rowspan=1 colspan=1>ATCC 12600</td><td rowspan=1 colspan=1>1.5 x 105</td></tr><tr><td rowspan=1 colspan=1>Stenotrophomonas maltophilia</td><td rowspan=1 colspan=1>ATCC 13637</td><td rowspan=1 colspan=1>5.0 x 103</td></tr><tr><td rowspan=1 colspan=1>Streptococcus pneumoniae</td><td rowspan=1 colspan=1>ATCC 49619</td><td rowspan=1 colspan=1>2.0 x 104</td></tr></table>

a Cell supernatant (in $\mathrm { I F U / m L }$ ). b Bacterial suspension, quantified by qPCR (in copies/mL). c Positive clinical specimen, quantified by qPCR (in copies $/ \mathrm { m L }$ ). LoD was determined using DNA extracted from a positive clinical specimen, and then confirmed by testing dilutions of this clinical specimen at the LoD concentration (20/20 positive results).

Table 14: Limit of Detection (LoD), Antibiotic Resistance Marker Targets   

<table><tr><td rowspan=1 colspan=1>ResistantMarker</td><td rowspan=1 colspan=1>ReferenceStrain ID</td><td rowspan=1 colspan=1>Host Microorganism</td><td rowspan=1 colspan=1>LoD[CFU/mL]</td></tr><tr><td rowspan=1 colspan=1>ctx-M</td><td rowspan=1 colspan=1>NCTC 13443</td><td rowspan=1 colspan=1>Klebsiella pneumoniae</td><td rowspan=1 colspan=1>1.0 x 104</td></tr><tr><td rowspan=1 colspan=1>kpc</td><td rowspan=1 colspan=1>NCTC 13438</td><td rowspan=1 colspan=1>Klebsiella pneumoniae</td><td rowspan=1 colspan=1>4.0 x 104</td></tr><tr><td rowspan=1 colspan=1>mecA</td><td rowspan=1 colspan=1>NCTC 12493</td><td rowspan=1 colspan=1>Staphylococcus aureus</td><td rowspan=1 colspan=1>4.0 x 105</td></tr><tr><td rowspan=1 colspan=1>ndm</td><td rowspan=1 colspan=1>NCTC 13443</td><td rowspan=1 colspan=1>Klebsiella pneumoniae</td><td rowspan=1 colspan=1>2.0 x 104</td></tr><tr><td rowspan=1 colspan=1>oxa-23</td><td rowspan=1 colspan=1>NCTC 13301</td><td rowspan=1 colspan=1>Acinetobacter baumannii</td><td rowspan=1 colspan=1>1.0 x 107</td></tr><tr><td rowspan=1 colspan=1>oxa-24</td><td rowspan=1 colspan=1>NCTC 13302</td><td rowspan=1 colspan=1>Acinetobacter baumannii</td><td rowspan=1 colspan=1>5.0 x 103</td></tr><tr><td rowspan=1 colspan=1>oxa-48</td><td rowspan=1 colspan=1>NCTC 13442</td><td rowspan=1 colspan=1>Klebsiella pneumoniae</td><td rowspan=1 colspan=1>3.0 x 105</td></tr><tr><td rowspan=1 colspan=1>oxa-58</td><td rowspan=1 colspan=1>NCTC 13305</td><td rowspan=1 colspan=1>Acinetobacter baumannii</td><td rowspan=1 colspan=1>5.0 x 104</td></tr><tr><td rowspan=1 colspan=1>tema</td><td rowspan=1 colspan=1>NCTC 13443</td><td rowspan=1 colspan=1>Klebsiella pneumoniae</td><td rowspan=1 colspan=1>2.0 x 104</td></tr><tr><td rowspan=1 colspan=1>vim</td><td rowspan=1 colspan=1>NCTC 13437</td><td rowspan=1 colspan=1>Pseudomonas aeruginosa</td><td rowspan=1 colspan=1>2.0 x 104</td></tr></table>

a Although the LRT BAL Application reports tem results only for $H .$ . influenzae as corresponding host microorganism, LoD was determined with a tem positive $E$ . coli strain. Note that inclusivity testing was also successfully performed with tem positive $H .$ . influenzae strains prepared as low positive samples.

# 8. Fresh versus Frozen Study:

To support the use of frozen specimens tested in clinical studies, an analytical study was conducted to assess any performance differences between fresh and frozen specimens when tested with the LRT BAL Application. Study samples were prepared with representative microorganisms inoculated into pooled native negative BAL specimen matrix. Testing was performed with both moderate positive (target analyte at $2 . 5 \mathrm { x } - 1 0 \mathrm { x }$ LoD, minimum 10 samples per storage condition) and low positive (per target analyte at $1 \mathrm { x } - 4 \mathrm { x }$ LoD, minimum of 30 samples storage condition) samples. Ten negative sample replicates were also evaluated.

Paired tests with the LRT BAL Application was performed for samples tested within 2 hours after test sample preparation (Time 0) and samples tested after freezing below $- 7 0 ~ ^ { \circ } \mathrm { C }$ for at least one day and thawing immediately before testing.

Tables 15-17 summarize the fresh versus frozen study results for each concentration level storage conditions (e.g., fresh, frozen). Results of the study demonstrated equivalent performance for fresh and frozen BAL specimens.

Table 15: Fresh Versus Frozen Study, Moderate Positive (\~5x LoD)   

<table><tr><td colspan="1" rowspan="2">BAL LRT Target</td><td colspan="1" rowspan="2">x-foldLoD</td><td colspan="1" rowspan="2">Test Condition</td><td colspan="3" rowspan="1">Agreement with ExpectedResult</td></tr><tr><td colspan="1" rowspan="1"># Pos./# Exp.</td><td colspan="1" rowspan="1">%</td><td colspan="1" rowspan="1">95% CI</td></tr><tr><td colspan="1" rowspan="1">Acinetobacter baumanniiATCC 19606</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">freshfrozen</td><td colspan="1" rowspan="1">11/1111/11</td><td colspan="1" rowspan="1">100.0100.0</td><td colspan="1" rowspan="1">74.1 - 100.074.1 - 100.0</td></tr><tr><td colspan="1" rowspan="1">Chlamydia pneumoniaeATCC VR-2282</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">freshfrozen</td><td colspan="1" rowspan="1">11/1111/11</td><td colspan="1" rowspan="1">100.0100.0</td><td colspan="1" rowspan="1">74.1 - 100.074.1 - 100.0</td></tr><tr><td colspan="1" rowspan="1">Citrobacter freundii</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">fresh</td><td colspan="1" rowspan="1">11/11</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">74.1 - 100.0</td></tr><tr><td colspan="1" rowspan="1">ATCC 8090</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">frozen</td><td colspan="1" rowspan="1">11/11</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">74.1 - 100.0</td></tr><tr><td colspan="1" rowspan="1">Haemophilus influenzaeATCC 33391</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">freshfrozen</td><td colspan="1" rowspan="1">10/1111/11</td><td colspan="1" rowspan="1">90.9100.0</td><td colspan="1" rowspan="1">62.3 - 98.474.1 - 100.0</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniaeNCTC 13443</td><td colspan="1" rowspan="1">2.5</td><td colspan="1" rowspan="1">freshfrozen</td><td colspan="1" rowspan="1">11/1111/11</td><td colspan="1" rowspan="1">100.0100.0</td><td colspan="1" rowspan="1">74.1 - 100.074.1 - 100.0</td></tr><tr><td colspan="1" rowspan="1">ctx-MNCTC 13443</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">freshfrozen</td><td colspan="1" rowspan="1">11/1111/11</td><td colspan="1" rowspan="1">100.0100.0</td><td colspan="1" rowspan="1">74.1 - 100.074.1 - 100.0</td></tr><tr><td colspan="1" rowspan="1">ndmNCTC 13443</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">freshfrozen</td><td colspan="1" rowspan="1">11/1111/11</td><td colspan="1" rowspan="1">100.0100.0</td><td colspan="1" rowspan="1">74.1 - 100.074.1 - 100.0</td></tr><tr><td colspan="1" rowspan="1">temNCTC 13443</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">freshfrozen</td><td colspan="1" rowspan="1">10/1111/11</td><td colspan="1" rowspan="1">90.9100.0</td><td colspan="1" rowspan="1">62.3 - 98.474.1 - 100.0</td></tr><tr><td colspan="1" rowspan="1">Morganella morganiiATCC 25830</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">freshfrozen</td><td colspan="1" rowspan="1">11/1111/11</td><td colspan="1" rowspan="1">100.0100.0</td><td colspan="1" rowspan="1">74.1 - 100.074.1 - 100.0</td></tr><tr><td colspan="1" rowspan="1">Proteus vulgarisATCC 29905</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">freshfrozen</td><td colspan="1" rowspan="1">11/1111/11</td><td colspan="1" rowspan="1">100.0100.0</td><td colspan="1" rowspan="1">74.1 - 100.074.1 - 100.0</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureusNCTC 12493</td><td colspan="1" rowspan="1">8.3</td><td colspan="1" rowspan="1">freshfrozen</td><td colspan="1" rowspan="1">10/10 *11/11</td><td colspan="1" rowspan="1">100.0100.0</td><td colspan="1" rowspan="1">72.3 - 100.074.1 - 100.0</td></tr><tr><td colspan="1" rowspan="1">mecANCTC 12493</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">freshfrozen</td><td colspan="1" rowspan="1">11/1111/11</td><td colspan="1" rowspan="1">100.0100.0</td><td colspan="1" rowspan="1">74.1 - 100.074.1 - 100.0</td></tr><tr><td colspan="2" rowspan="1">total (all analytes)</td><td colspan="1" rowspan="1">freshfrozen</td><td colspan="1" rowspan="1">129/131132/132</td><td colspan="1" rowspan="1">98.5100.0</td><td colspan="1" rowspan="1">94.6 - 99.697.2 - 100.0</td></tr></table>

\*Reduced number of available results due to invalid analyte results).

Table 16: Fresh Versus Frozen Study, Low Positive $\left( { \sim } 2 \mathbf { x } \ \mathbf { L o D } \right)$   

<table><tr><td colspan="1" rowspan="2">LRT BAL Target</td><td colspan="1" rowspan="2">x-fold LoD</td><td colspan="1" rowspan="2">TestCondition</td><td colspan="3" rowspan="1">Agreement with ExpectedResult</td></tr><tr><td colspan="1" rowspan="1"># Pos./# Exp.</td><td colspan="1" rowspan="1">%</td><td colspan="1" rowspan="1">95% CI</td></tr><tr><td colspan="1" rowspan="1">Acinetobacter baumanniiATCC 19606</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">freshfrozen</td><td colspan="1" rowspan="1">31/3231/31</td><td colspan="1" rowspan="1">96.9100.0</td><td colspan="1" rowspan="1">84.3 - 99.489.0 - 100.0</td></tr><tr><td colspan="1" rowspan="1">Chlamydia pneumoniaeATCC VR-2282</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">freshfrozen</td><td colspan="1" rowspan="1">32/3231/31</td><td colspan="1" rowspan="1">100.0100.0</td><td colspan="1" rowspan="1">89.3 - 100.089.0 - 100.0</td></tr><tr><td colspan="1" rowspan="1">Citrobacter freundiiATCC 8090</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">freshfrozen</td><td colspan="1" rowspan="1">31/3230/31</td><td colspan="1" rowspan="1">96.996.8</td><td colspan="1" rowspan="1">84.3 - 99.483.8 - 99.4</td></tr><tr><td colspan="1" rowspan="1">Haemophilus influenzaeATCC 33391</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">freshfrozen</td><td colspan="1" rowspan="1">29/3128/30 *</td><td colspan="1" rowspan="1">93.593.3</td><td colspan="1" rowspan="1">79.3 - 98.278.7 - 98.2</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniaeNCTC 13443</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">freshfrozen</td><td colspan="1" rowspan="1">31/3231/31</td><td colspan="1" rowspan="1">96.9100.0</td><td colspan="1" rowspan="1">84.3 - 99.489.0 - 100.0</td></tr><tr><td colspan="1" rowspan="1">ctx-MNCTC 13443</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">freshfrozen</td><td colspan="1" rowspan="1">32/3231/31</td><td colspan="1" rowspan="1">100.0100.0</td><td colspan="1" rowspan="1">89.3 - 100.089.0 - 100.0</td></tr><tr><td colspan="1" rowspan="1">ndmNCTC 13443</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">freshfrozen</td><td colspan="1" rowspan="1">31/3231/31</td><td colspan="1" rowspan="1">96.9100.0</td><td colspan="1" rowspan="1">84.3 - 99.489.0 - 100.0</td></tr><tr><td colspan="1" rowspan="1">temNCTC 13443</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">freshfrozen</td><td colspan="1" rowspan="1">30/3129/30 *</td><td colspan="1" rowspan="1">96.896.7</td><td colspan="1" rowspan="1">83.8 - 99.483.3 - 99.4</td></tr><tr><td colspan="1" rowspan="1">Morganella morganii</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">fresh</td><td colspan="1" rowspan="1">32/32</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">89.3 - 100.0</td></tr><tr><td colspan="1" rowspan="1">ATCC 25830</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">frozen</td><td colspan="1" rowspan="1">31/31</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">89.0 - 100.0</td></tr><tr><td colspan="1" rowspan="1">Proteus vulgarisATCC 29905</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">freshfrozen</td><td colspan="1" rowspan="1">32/3230/31</td><td colspan="1" rowspan="1">100.096.8</td><td colspan="1" rowspan="1">89.3 - 100.083.8 - 99.4</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureusNCTC 12493</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">freshfrozen</td><td colspan="1" rowspan="1">32/3231/31</td><td colspan="1" rowspan="1">100.0100.0</td><td colspan="1" rowspan="1">89.3 - 100.089.0 - 100.0</td></tr><tr><td colspan="1" rowspan="1">mecANCTC 12493</td><td colspan="1" rowspan="1">1.3</td><td colspan="1" rowspan="1">freshfrozen</td><td colspan="1" rowspan="1">32/3231/31</td><td colspan="1" rowspan="1">100.0100.0</td><td colspan="1" rowspan="1">89.3 - 100.089.0 - 100.0</td></tr><tr><td colspan="2" rowspan="1">total (all analytes)</td><td colspan="1" rowspan="1">freshfrozen</td><td colspan="1" rowspan="1">375/382365/370</td><td colspan="1" rowspan="1">98.298.6</td><td colspan="1" rowspan="1">96.3 - 99.196.9 - 99.4</td></tr></table>

\*Reduced number of available results due to invalid analyte results).

Table 17: Fresh versus Frozen Study, Negative Samples   

<table><tr><td rowspan="2">LRT BAL Target</td><td rowspan="2">Test Condition</td><td colspan="3">Agreement with Expected Result</td></tr><tr><td># Neg. /# Exp.</td><td>%</td><td>95% CI</td></tr><tr><td>Acinetobacter baumannii ATCC 19606</td><td>fresh frozen</td><td>10/10 11/11</td><td>100.0 100.0</td><td>72.3 - 100.0 74.1 - 100.0</td></tr><tr><td>Chlamydia pneumoniae</td><td>fresh</td><td>10/10</td><td>100.0</td><td>72.3 - 100.0</td></tr><tr><td>ATCC VR-2282 Citrobacter freundii</td><td>frozen fresh</td><td>11/11 10/10</td><td>100.0 100.0</td><td>74.1 - 100.0 72.3 - 100.0</td></tr><tr><td>ATCC 8090 Haemophilus influenzae</td><td>frozen fresh</td><td>11/11 10/10</td><td>100.0 100.0</td><td>74.1 - 100.0 72.3 - 100.0</td></tr><tr><td>ATCC 33391 Klebsiella pneumoniae</td><td>frozen fresh</td><td>11/11 10/10</td><td>100.0</td><td>74.1 - 100.0</td></tr><tr><td>NCTC 13443</td><td>frozen</td><td>11/11</td><td>100.0 100.0</td><td>72.3 - 100.0 74.1 - 100.0</td></tr><tr><td>ctx-M</td><td>fresh</td><td>10/10</td><td>100.0</td><td>72.3 - 100.0</td></tr><tr><td>NCTC 13443</td><td>frozen</td><td>11/11</td><td>100.0</td><td>74.1 - 100.0</td></tr><tr><td>ndm</td><td>fresh</td><td>10/10</td><td>100.0</td><td>72.3 - 100.0</td></tr><tr><td>NCTC 13443</td><td>frozen</td><td>11/11</td><td></td><td></td></tr><tr><td>tem</td><td></td><td></td><td>100.0</td><td>74.1 - 100.0</td></tr><tr><td>NCTC 13443</td><td>fresh</td><td>10/10</td><td>100.0</td><td>72.3 - 100.0</td></tr><tr><td></td><td>frozen</td><td>11/11</td><td>100.0</td><td>74.1 - 100.0</td></tr><tr><td>Morganella morganii</td><td>fresh</td><td>10/10</td><td>100.0</td><td>72.3 - 100.0</td></tr><tr><td>ATCC 25830</td><td>frozen</td><td>11/11</td><td>100.0</td><td></td></tr><tr><td>Proteus vulgaris</td><td>fresh</td><td></td><td></td><td>74.1 - 100.0</td></tr><tr><td>ATCC 29905</td><td></td><td>10/10</td><td>100.0</td><td>72.3 - 100.0</td></tr><tr><td>Staphylococcus aureus</td><td>frozen</td><td>11/11</td><td>100.0</td><td>74.1 - 100.0</td></tr><tr><td>NCTC 12493</td><td>fresh</td><td>9/9 *</td><td>100.0</td><td>70.1 - 100.0</td></tr><tr><td></td><td>frozen</td><td>11/11</td><td>100.0</td><td>74.1 - 100.0</td></tr><tr><td>mecA</td><td>fresh</td><td>10/10</td><td>100.0</td><td>72.3 - 100.0</td></tr><tr><td>NCTC 12493</td><td>frozen</td><td>11/11</td><td>100.0</td><td>74.1 - 100.0</td></tr><tr><td>total (all analytes)</td><td>fresh</td><td>119/119</td><td>100.0</td><td>96.9 - 100.0</td></tr><tr><td></td><td>frozen</td><td>132/132</td><td>100.0</td><td>97.2 - 100.0</td></tr></table>

\*Reduced number of available results due to invalid analyte results).

# 9. Assay Cut-Off:

The Unyvero LRT Application is comprised of eight individual multiplex PCR assays and hybridization arrays located in separate reaction chambers in the LRT Cartridge. After hybridization of PCR products, fluorescent array signals are captured with a fluorescent camera system over a defined temperature range. After correction for background values, a signal threshold is applied for probes of individual analytes. In addition to the signal thresholds, individual cutoffs are defined to reduce background signals for certain analytes that can be present in host flora of healthy individuals. Higher assay cutoffs were chosen for some analytes to exclude some non-specific low cross-reactivity background.

10. Accuracy (Instrument): Not Applicable

# 11. Carry-Over/Cross-contamination:

Please refer to the de novo Premarket Notification, DEN170047, for Carryover/crosscontamination study results performed with the Unyvero LRT Application (for tracheal aspirate specimens) and the Unyvero System.

# B Comparison Studies:

1. Method Comparison with Predicate Device:

Not Applicable

2. Matrix Comparison:

Artificial Respiratory Medium (ARM) was used as a matrix for preparation of samples for several analytical studies. A matrix equivalency study was performed to compare assay performance for samples prepared in ARM and samples prepared in pooled natural BAL matrix. BAL test samples were prepared by pooling BAL specimens that were determined to be negative for all LRT BAL Application analytes. Both ARM and natural BAL matrices were inoculated with representative microorganisms at various concentrations near the LoD (moderate positive: 5x LoD, low positive: 2x LoD and high negative: $0 . 1 \mathbf { x }$ LoD). Matrix equivalence was evaluated based on qualitative (percent positivity) and quantitative results (comparison of mean assay signal). The study results generated comparable positivity rates and assay signals between matrices for all target analytes evaluated, thereby supporting the use of ARM for preparation of analytical study samples.

# C Clinical Studies:

1. Clinical Study, Prospectively Collected Specimens:

Clinical performance of the LRT BAL Application was established using 1,051 BAL and mini-BAL specimens that were prospectively collected for a previous clinical study in June 2015-July 2016. These specimens were collected at nine US clinical sites from hospitalized patients 18 years or older and suspected of lower respiratory infections. All specimens were stored frozen within 24 hours after arrival of the specimen in the lab and remained frozen prior to inclusion in the LRT BAL Application clinical study. A total of 35 specimens $( 3 . 3 \% )$ of the original prospective cohort were excluded due to insufficient remaining

specimen volume. All other specimens (1,016, $9 6 . 7 \%$ ) were tested with the LRT BAL Application without any further pre-selection.

Table 18 includes the demographic information for the specimens tested in the LRT BAL prospective study.

Table 18: Prospective Clinical Study, Patient Demographics   

<table><tr><td rowspan=1 colspan=1>Included Specimens</td><td rowspan=1 colspan=1>1,016</td></tr><tr><td rowspan=1 colspan=2>Sex</td></tr><tr><td rowspan=1 colspan=1>male</td><td rowspan=1 colspan=1>598</td></tr><tr><td rowspan=1 colspan=1>female</td><td rowspan=1 colspan=1>418</td></tr><tr><td rowspan=1 colspan=2>Age</td></tr><tr><td rowspan=1 colspan=1>18 - 30 years</td><td rowspan=1 colspan=1>69</td></tr><tr><td rowspan=1 colspan=1>31 - 60 years</td><td rowspan=1 colspan=1>367</td></tr><tr><td rowspan=1 colspan=1>&gt; 60 years</td><td rowspan=1 colspan=1>580</td></tr><tr><td rowspan=1 colspan=2>Clinical Setting</td></tr><tr><td rowspan=1 colspan=1>ICU</td><td rowspan=1 colspan=1>505</td></tr><tr><td rowspan=1 colspan=1>ward</td><td rowspan=1 colspan=1>511</td></tr></table>

Clinical testing was performed using six Unyvero Systems and multiple operators with frozen BAL specimens thawed and tested with the LRT BAL Application. Runs with fully valid or partially valid results were eligible for data analysis. Fully invalid and partially invalid runs were repeated once whenever sufficient specimen volume for retesting was available. Results for repeat runs were used in the final performance analyses.

# Prospective Clinical Study Reference Methods – Microorganism Targets

‘Typical’ microorganism targets: Clinical performance of the LRT BAL Application was evaluated in comparison to standard-of-care (SoC) culture results which were reported either qualitatively or quantitatively. For quantitative cultures, a minimum reporting threshold of $\mathrm { 1 0 ^ { 3 } C F U / m L }$ or higher for mini-BAL specimens and $1 0 ^ { 4 } \mathrm { C F U / m L }$ or higher for BAL specimens was required to characterize a specimen as positive, following recommendations by the Infectious Diseases Society of America (IDSA) and the American Thoracic Society (ATS) for quantitative cultures.

In addition, LRT BAL Application results for each ‘typical’ microorganism target were compared to a composite comparator consisting of SoC culture and one PCR assay (with positive PCR results followed by bi-directional sequencing). Validated comparator PCR/sequencing assays were confirmed to have analytical sensitivities in a similar range to the LRT BAL Application and to target different genetic loci from the LRT BAL Application. Using the composite comparator algorithm, a specimen was characterized as positive for a specific analyte if either culture or PCR/sequencing were positive. Likewise, specimens that were negative by both SoC culture and PCR/sequencing were characterized as negative for the applicable analyte.

‘Atypical’ microorganism targets: For each targeted atypical microorganism (Chlamydia pneumoniae, Mycoplasma pneumoniae, Legionella pneumophila and Pneumocystis

jirovecii), a composite comparator consisting of two different PCR assays (each followed by bi-directional sequencing if PCR-positive) was performed for each BAL specimen. For each analyte, comparator PCR/sequencing assays targeted different genetic loci from each other as well as from LRT BAL Application target sequences. Validated comparator assays were confirmed to have analytical sensitivities similar to the LRT BAL Application for each applicable analyte. For the composite comparator, any specimen that was positive by either PCR and confirmed with bi-directional sequencing was characterized as positive for the respective microorganism target. Likewise, a specimen was characterized as negative if both respective PCR comparator assays were negative for the analyte.

For atypical analytes, assay performance was also evaluated based on a comparison to SoC testing for a subset of specimens that were tested based on clinician request. This SoC testing consisted of several methods (i.e., IFA, DFA, PCR).

# Prospective Study, Clinical Performance for ‘Typical’ Analytes

Results from the prospective study cohort using SoC culture results as the reference method are shown in Table 19 below. Additional evaluation of performance stratified by qualitative and quantitative culture results is presented in Table 20 below.

Table 19: Prospective Study $\scriptstyle ( \mathbf { N = 1 } , \mathbf { 0 1 6 }$ specimensa), Comparison to SoC   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>TP</td><td rowspan=1 colspan=1>TP FN</td><td rowspan=1 colspan=1>FP</td><td rowspan=1 colspan=1>TN</td><td rowspan=1 colspan=1>PPA [%] (95%C</td><td rowspan=1 colspan=1>NPA [%] (95%CI</td><td rowspan=1 colspan=1>PPV [%] (95%CI</td><td rowspan=1 colspan=1>NPV [%] (95%CI</td></tr><tr><td rowspan=1 colspan=1>Acinetobacter spp.</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>10 1</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>993</td><td rowspan=1 colspan=1>90.9 (62.3 - 98.4)</td><td rowspan=1 colspan=1>98.9 (98.0 - 99.4)</td><td rowspan=1 colspan=1>47.6 (28.3 - 67.6)</td><td rowspan=1 colspan=1>99.9 (99.4 -100.</td></tr><tr><td rowspan=1 colspan=1>C. freundii</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>1,011</td><td rowspan=1 colspan=1>100 (20.7 - 100.0)</td><td rowspan=1 colspan=1>99.7 (99.1 - 99.9)</td><td rowspan=1 colspan=1>25.0 (4.6 - 69.9)</td><td rowspan=1 colspan=1>100 (99.6 - 100.0)</td></tr><tr><td rowspan=1 colspan=1>E. cloacae complex</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>4 </td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>991</td><td rowspan=1 colspan=1>76.5 (52.7 - 90.4)</td><td rowspan=1 colspan=1>99.3 (98.6 - 99.7)</td><td rowspan=1 colspan=1>65.0 (43.3 - 81.9)</td><td rowspan=1 colspan=1>99.6 (99.0 - 99.8)</td></tr><tr><td rowspan=1 colspan=1>E. coli</td><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>[30</td><td rowspan=1 colspan=1>968</td><td rowspan=1 colspan=1>94.4 (74.2 - 99.0)</td><td rowspan=1 colspan=1>97.0 (95.7 - 97.9)</td><td rowspan=1 colspan=1>36.2 (24.0 - 50.5)</td><td rowspan=1 colspan=1>99.9 (99.4 -100.0</td></tr><tr><td rowspan=1 colspan=1>H. influenzae</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>48</td><td rowspan=1 colspan=1>958</td><td rowspan=1 colspan=1>88.9 (56.5 - 98.0)</td><td rowspan=1 colspan=1>95.2 (93.7 - 96.4)</td><td rowspan=1 colspan=1>14.3 (7.4 - 25.7)</td><td rowspan=1 colspan=1>99.9 (99.4 -100.0)</td></tr><tr><td rowspan=1 colspan=1>K. oxytoca</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>1,001</td><td rowspan=1 colspan=1>85.7 (48.7 - 97.4)</td><td rowspan=1 colspan=1>99.2 (98.4 - 99.6)</td><td rowspan=1 colspan=1>42.9 (21.4 - 67.4)</td><td rowspan=1 colspan=1>99.9 (99.4 -100.0)</td></tr><tr><td rowspan=1 colspan=1>K. pneumoniae b</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>4f</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>982</td><td rowspan=1 colspan=1>83.3 (64.1 - 93.3)</td><td rowspan=1 colspan=1>99.0 (98.2 - 99.5)</td><td rowspan=1 colspan=1>66.7 (48.8 - 80.8)</td><td rowspan=1 colspan=1>99.6 (99.0 - 99.8)</td></tr><tr><td rowspan=1 colspan=1>K. variicola b</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>1,012</td><td rowspan=1 colspan=1>0.0 (0.0 - 65.8)</td><td rowspan=1 colspan=1>99.8 (99.3 - 99.9)</td><td rowspan=1 colspan=1>0.0 (0.0 - 65.8)</td><td rowspan=1 colspan=1>99.8 (99.3 - 99.9)</td></tr><tr><td rowspan=1 colspan=1>M. catarrhalis</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>997</td><td rowspan=1 colspan=1>100 (34.2 - 100.0)</td><td rowspan=1 colspan=1>98.7 (97.8 - 99.2)</td><td rowspan=1 colspan=1>13.3 (3.7 - 37.9)</td><td rowspan=1 colspan=1>100 (99.6 - 100.0)</td></tr><tr><td rowspan=1 colspan=1>M. morganii</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>1,009</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>99.7 (99.1 - 99.9)</td><td rowspan=1 colspan=1>0.0 (0.0 - 56.1)</td><td rowspan=1 colspan=1>100 (99.6 - 100.0)</td></tr><tr><td rowspan=1 colspan=1>Proteus spp.</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>1,006</td><td rowspan=1 colspan=1>100 (51.0 - 100.0)</td><td rowspan=1 colspan=1>99.4 (98.7 - 99.7)</td><td rowspan=1 colspan=1>40.0 (16.8 - 68.7)</td><td rowspan=1 colspan=1>100 (99.6 - 100.0)</td></tr><tr><td rowspan=1 colspan=1>P. aeruginosa</td><td rowspan=1 colspan=1>69</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>43</td><td rowspan=1 colspan=1>900</td><td rowspan=1 colspan=1>95.8 (88.5 - 98.6)</td><td rowspan=1 colspan=1>95.4 (93.9 - 96.6)</td><td rowspan=1 colspan=1>61.6 (52.4 - 70.1)</td><td rowspan=1 colspan=1>99.7 (99.0 - 99.9)</td></tr><tr><td rowspan=1 colspan=1>S. marcescens</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>998</td><td rowspan=1 colspan=1>100 (75.8 - 100.0)</td><td rowspan=1 colspan=1>99.5 (98.8 - 99.8)</td><td rowspan=1 colspan=1>70.6 (46.9 - 86.7)</td><td rowspan=1 colspan=1>100 (99.6 - 100.0)</td></tr><tr><td rowspan=1 colspan=1>S. aureus</td><td rowspan=1 colspan=1>63</td><td rowspan=1 colspan=1>638</td><td rowspan=1 colspan=1>41</td><td rowspan=1 colspan=1>904</td><td rowspan=1 colspan=1>88.7 (79.3 - 94.2)</td><td rowspan=1 colspan=1>95.7 (94.2 - 96.8)</td><td rowspan=1 colspan=1>60.6 (51.0 - 69.4)</td><td rowspan=1 colspan=1>99.1 (98.3 - 99.6)</td></tr><tr><td rowspan=1 colspan=1>S. maltophilia</td><td rowspan=1 colspan=1>19</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>22</td><td rowspan=1 colspan=1>972</td><td rowspan=1 colspan=1>90.5 (71.1 - 97.4)</td><td rowspan=1 colspan=1>97.8 (96.7 - 98.5)</td><td rowspan=1 colspan=1>46.3 (32.1 - 61.3)</td><td rowspan=1 colspan=1>99.8 (99.3 - 99.9)</td></tr><tr><td rowspan=1 colspan=1>S. pneumoniae</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>1,003</td><td rowspan=1 colspan=1>100 (43.9 - 100.0)</td><td rowspan=1 colspan=1>99.0 (98.2 - 99.5)</td><td rowspan=1 colspan=1>23.1 (8.2 - 50.3)</td><td rowspan=1 colspan=1>100 (99.6 - 100.0)</td></tr></table>

a Observed invalid LRT BAL analyte results: Acinetobacter spp. (1), C. freundii (1), E. cloacae complex (1), H. influenzae (1), M. catarrhalis (4), M. morganii (4), $P .$ . aeruginosa (1), S. marcescens (1) and $S .$ . maltophilia (1). ${ \textbf { b } } _ { K }$ . variicola is typically reported by culture as $K$ . pneumoniae. To discriminate $K .$ . variicola from K. pneumoniae, K. pneumoniae strain isolates were sequenced. For a few cases, for which no isolates were provided, sequencing was performed from specimen DNA extracts instead. Two of 26 cases were identified as $K .$ . variicola and confirmed by both isolate and DNA

extract sequencing. Results for K. pneumoniae and K. variicola performance are calculated based on the species identified by sequencing.

c Note that many of the FP detections are confirmed when comparing against the composite comparator that includes molecular reference assays (see Table 21).

d Specimens with false positive LRT BAL results were analyzed with molecular assays (PCR/bi-directional sequencing) using specimen DNA extracts for presence or absence of the corresponding microorganism: presence was confirmed in 11 of 11 cases for Acinetobacter spp., 3 of 3 cases for C. freundii, 7 of 7 cases for $E$ . cloacae complex, 18 of 30 cases for $E$ . coli, 42 of 48 cases for $H .$ . influenzae, 7 of 8 cases for K. oxytoca, 5 of 10 cases for $K .$ . pneumoniae, 2 of 2 cases for $K .$ . variicola, 13 of 13 cases for $M .$ . catarrhalis, 3 of 3 cases for $M .$ morganii, 6 of 6 cases for Proteus spp., 41 of 43 cases for $P$ . aeruginosa, 4 of 5 cases for S. marcescens, 31 of 41 cases for S. aureus, 21 of 22 cases for S. maltophilia, and 10 of 10 cases for $S _ { \cdot }$ . pneumoniae. For cases that were not confirmed for $H .$ . influenzae, sequencing results identified $H .$ haemolyticus (3x), $H .$ parainfluenzae (1x) and Aggregatibacter aphrophilus (1x). For all other cases, PCRs did not amplify sufficient amounts for sequencing.

e Two of the four specimens that were FN for $E$ . cloacae complex contained multiple host microorganisms identified by culture, and/or LRT BAL. For one specimen, Acinetobacter spp. was additionally reported by both LRT BAL and culture. For the other specimen, S. maltophilia was additionally reported by LRT BAL only.

f Two of the four specimens that were FN for $K .$ pneumoniae contained multiple host microorganisms identified by culture and/or the molecular reference testing. For one FN specimen, $P .$ . aeruginosa was additionally reported by both LRT BAL and culture. For the other specimen, $E .$ . cloacae complex was additionally reported by both LRT BAL and culture and S. maltophilia was additionally reported by LRT BAL only.

Because of the observed lower than expected PPA/sensitivity compared to SoC culture for specimens containing S. aureus, the following limitation is included in the package insert notifying the user how to interpret negative results by the LRT BAL Application.

In clinical testing, S. aureus was not reported by the LRT BAL Application for 8/71 prospective specimens that were positive for S. aureus by SoC culture. Six of the eight false negative specimens only contained low microorganism loads (i.e., rare, $1 0 ^ { \mathrm { { 4 } } } \mathrm { C F U / m L }$ or lower). As the assay LoD for $S .$ . aureus is $\bar { 1 . 5 } \mathrm { x } 1 0 ^ { 5 } \mathrm { C F U / m L }$ , and bacterial concentrations of $\geq 1 0 ^ { 4 } \mathrm { C F U / m L }$ are generally considered significant, negative results for $S .$ aureus by the LRT BAL Application must be interpreted in conjunction with results from SoC culture.

Due to the high risk associated with lower respiratory infections involving S. aureus/MRSA, additional testing results for specimens with false negative results for S. aureus are listed in Table 20 below and also included in the package insert.

Table 20: Additional Details, Specimens False Negative for S. aureus 1   

<table><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">SpecimenType</td><td colspan="1" rowspan="1">SoC Result:Concentration,Phenotype,Gram Strain,Respiratory Flora</td><td colspan="1" rowspan="1">LRT BAL Result</td><td colspan="1" rowspan="1">CompositeComparator:MultiplexPCR/Sequencing</td><td colspan="1" rowspan="1">IsolateSequencing</td><td colspan="1" rowspan="1">OtherMicroorganismsDetected byLRT BALand/or SoC</td></tr><tr><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">mini-BAL</td><td colspan="1" rowspan="1">4x103 CFU/mL,MRSA,Gram+/Gram-,flora reported</td><td colspan="1" rowspan="1">S. aureus: negativemecA: negative</td><td colspan="1" rowspan="1">S. aureus: negativemecA: negative</td><td colspan="1" rowspan="1">mecAdetected</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">BAL</td><td colspan="1" rowspan="1">rare,MSSA,Gram stain negative,no flora reported</td><td colspan="1" rowspan="1">S. aureus: negativemecA: negative</td><td colspan="1" rowspan="1">S. aureus: negativemecA: negative</td><td colspan="1" rowspan="1">no isolatecollected</td><td colspan="1" rowspan="1">P. aeruginosa,C. freundii</td></tr><tr><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">BAL</td><td colspan="1" rowspan="1">few,MSSA,Gram stain negative,flora reported</td><td colspan="1" rowspan="1">S. aureus: negativemecA: negative</td><td colspan="1" rowspan="1">S. aureus: negativemecA: negative</td><td colspan="1" rowspan="1">no isolatecollected</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">BAL</td><td colspan="1" rowspan="1">moderate,MRSA,Gram+/Gram-,</td><td colspan="1" rowspan="1">S. aureus: negativemecA: positive(masked)</td><td colspan="1" rowspan="1">S. aureus: positivemecA: positive</td><td colspan="1" rowspan="1">mecAdetected</td><td colspan="1" rowspan="1">H. influenzae</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">flora reported</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">BAL</td><td colspan="1" rowspan="1">rare,MSSA,Gram stain negative,flora reported</td><td colspan="1" rowspan="1">S. aureus: negativemecA: negative</td><td colspan="1" rowspan="1">S. aureus: negativemecA: negative</td><td colspan="1" rowspan="1">mecA notdetected</td><td colspan="1" rowspan="1">P. aeruginosa</td></tr><tr><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">BAL</td><td colspan="1" rowspan="1">rare,AST not reported,Gram stain negative,flora reported</td><td colspan="1" rowspan="1">S. aureus: negativemecA: negative</td><td colspan="1" rowspan="1">S. aureus: negativemecA: negative</td><td colspan="1" rowspan="1">no isolatecollected</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">7</td><td colspan="1" rowspan="1">BAL</td><td colspan="1" rowspan="1">rare,MSSA,Gram stain negative,no flora reported</td><td colspan="1" rowspan="1">S. aureus: negativemecA: negative</td><td colspan="1" rowspan="1">S. aureus: negativemecA: negative</td><td colspan="1" rowspan="1">mecA notdetected</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">8</td><td colspan="1" rowspan="1">BAL</td><td colspan="1" rowspan="1">1x104 CFU/mL,MRSA,Gram stain negative,no flora reported</td><td colspan="1" rowspan="1">S. aureus: negativemecA: positive(masked)</td><td colspan="1" rowspan="1">S. aureus: negativemecA: negative</td><td colspan="1" rowspan="1">mecAdetected</td><td colspan="1" rowspan="1"></td></tr></table>

1SoC concentration in CFU/mL for quantitative culture, or few/rare/moderate/numerous for qualitative culture, Gram+: Gram positive microorganisms detected, Gram-: Gram negative microorganisms detected, MRSA: methicillin resistant S. aureus, MSSA: methicillin sensitive S. aureus, AST: antibiotic susceptibility test.

For other ‘typical’ analytes with lower than expected PPA/sensitivity compared to SoC culture ( $E .$ . cloacae complex, $K .$ . pneumoniae, $K .$ . variicola), separate limitations are included in the package insert notifying the user the need to interpret negative results by the LRT BAL Application in conjunction with results from SoC culture. The following limitations are included in the package insert.

In clinical testing, $E$ . cloacae complex was not reported by the LRT BAL Application for 4/17 prospective and $4 / 1 9$ archived specimens that were positive for $E$ . cloacae complex by SoC culture. Five of the eight false negative specimens only contained low microorganism loads (i.e., rare, $1 0 ^ { 4 } \mathrm { \bar { C } F U / m L }$ or lower). As the assay LoD for $E$ . cloacae complex is $2 \mathrm { x } 1 0 ^ { 5 } \mathrm { C F U / m L }$ , and bacterial concentrations $\mathrm { { o f } \geq 1 0 ^ { 4 } \ C F U / m L }$ are generally considered significant, negative results for $E$ . cloacae complex by the LRT BAL Application must be interpreted in conjunction with results from SoC culture.

In clinical testing, $K .$ pneumoniae was not reported by the LRT BAL Application for 4/24 prospective specimens that were positive for $K .$ . pneumoniae by SoC culture. Two of the four false negative specimens only contained low microorganism loads (i.e., rare, $1 0 ^ { 4 } \mathrm { C F U / m L }$ or lower). As the assay LoD for $K .$ . pneumoniae is $4 \mathrm { x } 1 0 ^ { 4 } \mathrm { C F U / m L }$ , and bacterial concentrations of $\geq 1 0 ^ { 4 } \mathrm { C F U / m L }$ are generally considered significant, negative results for $K .$ . pneumoniae by the Unyvero LRT Application must be interpreted in conjunction with SoC culture.

Due to the observed low prevalence and low sensitivity/positive percent agreement of the LRT BAL Application for detection of $K .$ . variicola in the prospective clinical study, negative results must be interpreted in conjunction with results from SoC culture.

Performance for ‘Typical’ analytes, Comparison to Composite Comparator Reference

For each ‘typical’ analyte, LRT BAL results from the prospective clinical study were also compared to a composite comparator reference consisting of SoC culture results combined with a validated molecular test (PCR/sequencing). Specimens were considered positive by the composite comparator for a targeted analyte if either culture or PCR/sequencing generated positive results. Likewise, specimens were considered negative if both culture and PCR/sequencing were negative. Performance is presented in Table 21.

Table 21: Prospective Study, $\mathbf { \Delta N = 1 0 1 6 }$ specimens a), Comparison to Composite Comparator.   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>TP</td><td rowspan=1 colspan=1>FN</td><td rowspan=1 colspan=1>FP</td><td rowspan=1 colspan=1>TN</td><td rowspan=1 colspan=1>PPA [%] (95%C</td><td rowspan=1 colspan=1>NPA [%] (95%C</td><td rowspan=1 colspan=1>PPV [%] (95%C</td><td rowspan=1 colspan=1>NPV [%] (95%CI</td></tr><tr><td rowspan=1 colspan=1>Acinetobacter spp.</td><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>992</td><td rowspan=1 colspan=1>88.9 (67.2 - 96.9)</td><td rowspan=1 colspan=1>99.5 (98.8 - 99.8)</td><td rowspan=1 colspan=1>76.2 (54.9 - 89.4)</td><td rowspan=1 colspan=1>99.8 (99.3 - 99.9)</td></tr><tr><td rowspan=1 colspan=1>C. freundii</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>1,011</td><td rowspan=1 colspan=1>100 (20.7 -100.0)</td><td rowspan=1 colspan=1>99.7 (99.1 - 99.9)</td><td rowspan=1 colspan=1>25.0 (4.6 - 69.9)</td><td rowspan=1 colspan=1>100 (99.6 - 100.0)</td></tr><tr><td rowspan=1 colspan=1>E. cloacae complex</td><td rowspan=1 colspan=1>18</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>991</td><td rowspan=1 colspan=1>81.8 (61.5 - 92.7)</td><td rowspan=1 colspan=1>99.8 (99.3 - 99.9)</td><td rowspan=1 colspan=1>90.0 (69.9 - 97.2)</td><td rowspan=1 colspan=1>99.6 (99.0 - 99.8)</td></tr><tr><td rowspan=1 colspan=1>E. coli</td><td rowspan=1 colspan=1>28</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>19</td><td rowspan=1 colspan=1>963</td><td rowspan=1 colspan=1>82.4 (66.5 - 91.7)</td><td rowspan=1 colspan=1>98.1 (97.0 - 98.8)</td><td rowspan=1 colspan=1>59.6 (45.3 - 72.4)</td><td rowspan=1 colspan=1>99.4 (98.7 - 99.7)</td></tr><tr><td rowspan=1 colspan=1>H. influenzae</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>36</td><td rowspan=1 colspan=1>957</td><td rowspan=1 colspan=1>90.9 (72.2 - 97.5)</td><td rowspan=1 colspan=1>96.4 (95.0 - 97.4)</td><td rowspan=1 colspan=1>35.7 (24.5 - 48.8)</td><td rowspan=1 colspan=1>99.8 (99.2 - 99.9)</td></tr><tr><td rowspan=1 colspan=1>K. oxytoca</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>1,000</td><td rowspan=1 colspan=1>80.0 (49.0 - 94.3)</td><td rowspan=1 colspan=1>99.4 (98.7 - 99.7)</td><td rowspan=1 colspan=1>57.1 (32.6 - 78.6)</td><td rowspan=1 colspan=1>99.8 (99.3 - 99.9)</td></tr><tr><td rowspan=1 colspan=1>K. pneumoniae b</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>981</td><td rowspan=1 colspan=1>84.0 (65.3 - 93.6)</td><td rowspan=1 colspan=1>99.1 (98.3 - 99.5)</td><td rowspan=1 colspan=1>70.0 (52.1 - 83.3)</td><td rowspan=1 colspan=1>99.6 (99.0 - 99.8)</td></tr><tr><td rowspan=1 colspan=1>K. variicola b</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1,012</td><td rowspan=1 colspan=1>0.0 (0.0 - 65.8)</td><td rowspan=1 colspan=1>99.9 (99.4 - 100)</td><td rowspan=1 colspan=1>0.0 (0.0 - 79.3)</td><td rowspan=1 colspan=1>99.8 (99.3 - 99.9)</td></tr><tr><td rowspan=1 colspan=1>M. catarrhalis</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>993</td><td rowspan=1 colspan=1>75.0 (50.5 - 89.8)</td><td rowspan=1 colspan=1>99.7 (99.1 - 99.9)</td><td rowspan=1 colspan=1>80.0 (54.8 - 93.0)</td><td rowspan=1 colspan=1>99.6 (99.0 - 99.8)</td></tr><tr><td rowspan=1 colspan=1>M. morganii</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1,009</td><td rowspan=1 colspan=1>100 (43.9 -100.0)</td><td rowspan=1 colspan=1>100 (99.6 - 100.0)</td><td rowspan=1 colspan=1>100 (43.9 - 100.0)</td><td rowspan=1 colspan=1>100 (99.6 - 100.0)</td></tr><tr><td rowspan=1 colspan=1>Proteus spp.</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>01</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1,006</td><td rowspan=1 colspan=1>100 (72.3 -100.0)</td><td rowspan=1 colspan=1>100 (99.6 - 100.0)</td><td rowspan=1 colspan=1>100 (72.3 - 100.0)</td><td rowspan=1 colspan=1>100 (99.6 - 100.0)</td></tr><tr><td rowspan=1 colspan=1>P. aeruginosa</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>896</td><td rowspan=1 colspan=1>93.5 (87.1 - 96.8)</td><td rowspan=1 colspan=1>98.7 (97.7 - 99.2)</td><td rowspan=1 colspan=1>89.3 (82.2 - 93.8)</td><td rowspan=1 colspan=1>99.2 (98.4 - 99.6)</td></tr><tr><td rowspan=1 colspan=1>S. marcescens</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>997</td><td rowspan=1 colspan=1>93.8 (71.7 - 98.9)</td><td rowspan=1 colspan=1>99.8 (99.3 - 99.9)</td><td rowspan=1 colspan=1>88.2 (65.7 - 96.7)</td><td rowspan=1 colspan=1>99.9 (99.4 - 100)</td></tr><tr><td rowspan=1 colspan=1>S. aureus</td><td rowspan=1 colspan=1>80</td><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>896</td><td rowspan=1 colspan=1>83.3 (74.6 - 89.5)</td><td rowspan=1 colspan=1>97.4 (96.1 - 98.2)</td><td rowspan=1 colspan=1>76.9 (68.0 - 84.0)</td><td rowspan=1 colspan=1>98.2 (97.2 - 98.9)</td></tr><tr><td rowspan=1 colspan=1>S. maltophilia</td><td rowspan=1 colspan=1>34</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>967</td><td rowspan=1 colspan=1>82.9 (68.7 - 91.5)</td><td rowspan=1 colspan=1>99.3 (98.5 - 99.7)</td><td rowspan=1 colspan=1>82.9 (68.7 - 91.5)</td><td rowspan=1 colspan=1>99.3 (98.5 - 99.7)</td></tr><tr><td rowspan=1 colspan=1>S. pneumoniae</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>1,003</td><td rowspan=1 colspan=1>100.0 (74.1 -100.0)</td><td rowspan=1 colspan=1>99.8 (99.3 - 99.9)</td><td rowspan=1 colspan=1>84.6 (57.8 - 95.7)</td><td rowspan=1 colspan=1>100.0 (99.6 -100.0)</td></tr></table>

a Specimens with observed invalid LRT BAL analyte results were positive by the composite comparator: Acinetobacter spp. (1), C. freundii (1), $E .$ . cloacae complex (1), $H .$ influenzae (1), $M .$ . catarrhalis (4), $M .$ morganii (4), $P$ . aeruginosa (1), S. marcescens (1) and $S .$ . maltophilia (1). For one SoC negative specimen, $^ { \mathfrak { b } } K .$ . pneumoniae and $K .$ . variicola were simultaneously reported by the molecular test of the composite comparator, but were negative by SoC culture. As it was not feasible to resolve whether these analytes exceeded the molecular reporting threshold, both were set to ‘invalid’. For this specimen, LRT BAL had reported a positive result for $K .$ . variicola and a negative result for $K .$ . pneumoniae.

${ \mathbf { ^ { b } } } K .$ . variicola is typically reported by culture as $K .$ . pneumoniae. To discriminate $K .$ . variicola from $K .$ . pneumoniae, provided $K .$ . pneumoniae strain isolates were sequenced. For a few cases, no isolate was provided and sequencing was performed from specimen DNA extracts instead. Two of 27 cases were identified as $K .$ . variicola and confirmed by both isolate and DNA extract sequencing. Results for $K .$ . pneumoniae and $K .$ . variicola performance are calculated based on the species identified by sequencing.

# Clinical Performance, Prospective study, Atypical analytes

Results from the prospective study cohort using a composite comparator of two independent PCR assays followed by bidirectional sequencing are shown in Table 22 below.

Table 22: Prospective Study, Atypical Analytes, Comparison to Composite Comparator   

<table><tr><td rowspan=1 colspan=1>MicroorganismTarget</td><td rowspan=1 colspan=1>TP</td><td rowspan=1 colspan=1>FN</td><td rowspan=1 colspan=1>FP</td><td rowspan=1 colspan=1>TN</td><td rowspan=1 colspan=1>PPA [%](95% CI)</td><td rowspan=1 colspan=1>NPA [%](95% CI)</td><td rowspan=1 colspan=1>PPV [%](95% CI)</td><td rowspan=1 colspan=1>NPV [%](95% CI)</td></tr><tr><td rowspan=1 colspan=1>Chlamydiapneumoniae</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1,016</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>100.0(99.6 - 100.0)</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>100.0(99.6 - 100.0)</td></tr><tr><td rowspan=1 colspan=1>Legionellapneumophila</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1,014</td><td rowspan=1 colspan=1>100.0(20.7 - 100.0)</td><td rowspan=1 colspan=1>100.0(99.6 - 100.0)</td><td rowspan=1 colspan=1>100.0(20.7 - 100.0)</td><td rowspan=1 colspan=1>100.0(99.6 - 100.0)</td></tr><tr><td rowspan=1 colspan=1>Mycoplasmapneumoniae</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>1,010</td><td rowspan=1 colspan=1>100.0(43.9 - 100.0)</td><td rowspan=1 colspan=1>99.7(99.1 - 99.9)</td><td rowspan=1 colspan=1>50.0(18.8 - 81.2)</td><td rowspan=1 colspan=1>100.0(99.6 - 100.0)</td></tr><tr><td rowspan=1 colspan=1>Pneumocystisjirovecii</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>5a</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>989</td><td rowspan=1 colspan=1>80.0(60.9 - 91.1)</td><td rowspan=1 colspan=1>99.8(99.3 - 99.9)</td><td rowspan=1 colspan=1>90.9(72.2 - 97.5)</td><td rowspan=1 colspan=1>99.5(98.8 - 99.8)</td></tr></table>

a Initial comparator testing was performed prior to specimen storage. For two of five FN $P .$ . jirovecii cases, repeat comparator testing indicated specimen degradation during prolonged storage as likely reason for observed failures.

Performance for LRT BAL ‘atypical’ analytes was also evaluated in comparison to results from SoC testing for the subset of cases for which a SoC test was performed. SoC testing included DFA, IFA or PCR. Only one specimen had $\mathrm { S o C }$ testing ordered for $C .$ . pneumoniae; results for this specimen were negative for both SoC and the LRT BAL Application. For the three other ‘atypical’ microorganism targets, the following performance was observed for the LRT BAL Application when compared to SoC testing. Additional details are presented in Table 23.

L. pneumophila: $\mathrm { P P A } = 0 \%$ (0/1) $\mathrm { N P A } = 1 0 0 \%$ (237/237) M. pneumoniae: $\mathrm { P P A } = \mathrm { N } / \mathrm { A } ( ( $ 0/0) $\mathrm { N P A } = 1 0 0 \%$ (28/28) P. jirovecii: $\mathrm { P P A } = 1 0 0 \%$ (5/5) $\mathrm { N P A } = 9 9 \%$ (99/100)

Table 23: Prospective Study, Atypical analytes, Compared to SoC   

<table><tr><td rowspan=2 colspan=1></td><td rowspan=2 colspan=2>SOC Test(tested on request only)</td><td rowspan=1 colspan=2>LRT BAL result</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td></tr><tr><td rowspan=4 colspan=1>Chlamydia pneumoniae</td><td rowspan=1 colspan=1>not tested</td><td rowspan=1 colspan=1>1,015 (99.9%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1,015</td></tr><tr><td rowspan=1 colspan=1>tested</td><td rowspan=1 colspan=1>1 (0.1%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>positive result</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>negative result</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=4 colspan=1>Legionella pneumophila</td><td rowspan=1 colspan=1>not tested</td><td rowspan=1 colspan=1>778 (76.6%)</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>776 a</td></tr><tr><td rowspan=1 colspan=1>tested</td><td rowspan=1 colspan=1>238 (23.4%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>238</td></tr><tr><td rowspan=1 colspan=1>positive result</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1b</td></tr><tr><td rowspan=1 colspan=1>negative result</td><td rowspan=1 colspan=1>237</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>237</td></tr><tr><td rowspan=4 colspan=1>Mycoplasma pneumoniae</td><td rowspan=1 colspan=1>not tested</td><td rowspan=1 colspan=1>988 (97.2%)</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>982</td></tr><tr><td rowspan=1 colspan=1>tested</td><td rowspan=1 colspan=1>28 (2.8%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>28</td></tr><tr><td rowspan=1 colspan=1>positive result</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>negative result</td><td rowspan=1 colspan=1>28</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>28</td></tr><tr><td rowspan=4 colspan=1>Pneumocystis jirovecii</td><td rowspan=1 colspan=1>not tested</td><td rowspan=1 colspan=1>911 (89.7%)</td><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>895</td></tr><tr><td rowspan=1 colspan=1>tested</td><td rowspan=1 colspan=1>105 (10.3%)</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>99</td></tr><tr><td rowspan=1 colspan=1>positive result</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>5d</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>negative result</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>1e</td><td rowspan=1 colspan=1>99</td></tr></table>

a Reduced number of available LRT BAL results due to one invalid analyte result. $^ { \flat } \mathrm { ~ L ~ }$ . pneumophila was reported positive by culture c Pneumocystis SoC methods for 105 specimens were DFA (63 specimens), IFA (29 specimens), and PCR (13 specimens). d SoC Positive by DFA (three specimens) or IFA (two specimens). e SoC Negative by DFA, discordant testing generated positive results by alternate molecular comparator tests

Due to the lower than expected clinical performance for detection of Pneumocystis jirovecii, the following limitations are included in the labeling:

For $P _ { - }$ . jirovecii, an LoD of $5 \mathrm { ~ x ~ } 1 0 ^ { 5 }$ copies/mL was determined. Clinically relevant levels of $P .$ . jirovecii DNA have been reported to be in a range of $1 0 ^ { 4 } - 1 0 ^ { 5 }$ copies/mL or above in patients with PJP (Pneumocystis jirovecii pneumonia). The LRT BAL Application should be used in conjunction with SoC tests for $P$ . jirovecii (e.g., fluorescence staining tests or highly sensitive PCR assays). Additional testing for $P .$ . jirovecii is recommended for patients who are suspected of PJP and have negative $P .$ . jirovecii results by the LRT BAL Application.

Prospective study specimens with positive reference culture results were further evaluated based on culture quantitation. The analysis included evaluation of specimens with semiquantitative result reported by qualitative SoC culture (categories: rare, few, moderate, numerous) or by quantitative SoC culture (categories: $1 0 ^ { 3 } - < 1 0 ^ { 4 }$ , $1 0 ^ { 4 } \mathrm { - } < 1 0 ^ { 5 }$ , $1 0 ^ { 5 } - < 1 0 ^ { 6 } , >$ $1 0 ^ { 6 }$ , in $\mathrm { C F U / m L }$ ) as shown in Table 24.

Table 24: Prospective Study, Comparison to Qualitative or Quantitative SoC Culture   

<table><tr><td colspan="1" rowspan="1"></td><td colspan="5" rowspan="1">Qualitative Culture Result</td><td colspan="1" rowspan="1"></td><td colspan="5" rowspan="1">Quantitative Culture Result</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Category</td><td colspan="1" rowspan="1">Total</td><td colspan="1" rowspan="1">TP</td><td colspan="1" rowspan="1">FN</td><td colspan="1" rowspan="1">PPA [%]</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Category</td><td colspan="1" rowspan="1">Total</td><td colspan="1" rowspan="1">TP</td><td colspan="1" rowspan="1">FN</td><td colspan="1" rowspan="1">PPA[%]</td></tr><tr><td colspan="1" rowspan="4">Acinetobacter spp.</td><td colspan="1" rowspan="1">rare</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">na</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">103 - &lt; 104</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.0</td></tr><tr><td colspan="1" rowspan="1">few</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">104 - &lt;105</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.0</td></tr><tr><td colspan="1" rowspan="1">moderate</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">10^5- &lt;106</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.0</td></tr><tr><td colspan="1" rowspan="1">numerous</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">b</td><td colspan="1" rowspan="1">&gt; 106</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">NA</td></tr><tr><td colspan="1" rowspan="4">Citrobacter freundii</td><td colspan="1" rowspan="1">rare</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">b</td><td colspan="1" rowspan="1">103 - &lt;104</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">NA</td></tr><tr><td colspan="1" rowspan="1">few</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">b</td><td colspan="1" rowspan="1">104 - &lt;105</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">NA</td></tr><tr><td colspan="1" rowspan="1">moderate</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">b</td><td colspan="1" rowspan="1">$10^{5 - &lt;106$</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">NA</td></tr><tr><td colspan="1" rowspan="1">numerous</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">&gt; 106</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">NA</td></tr><tr><td colspan="1" rowspan="4">Enterobacter cloacae complex</td><td colspan="1" rowspan="1">rare</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">b</td><td colspan="1" rowspan="1">103 - &lt;104</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0.0</td></tr><tr><td colspan="1" rowspan="1">few</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">104- &lt;105</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">83.3</td></tr><tr><td colspan="1" rowspan="1">moderate</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">83.3</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">105 - &lt;106</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.0</td></tr><tr><td colspan="1" rowspan="1">numerous</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">&gt;106</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">NA</td></tr><tr><td colspan="1" rowspan="4">Escherichia coli</td><td colspan="1" rowspan="1">rare</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">75.0</td><td colspan="1" rowspan="1">b</td><td colspan="1" rowspan="1">103- &lt;104</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">NA</td></tr><tr><td colspan="1" rowspan="1">few</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">b</td><td colspan="1" rowspan="1">104 - &lt;105</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.0</td></tr><tr><td colspan="1" rowspan="1">moderate</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">b</td><td colspan="1" rowspan="1">10^5- &lt;106</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.0</td></tr><tr><td colspan="1" rowspan="1">numerous</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">b</td><td colspan="1" rowspan="1">&gt; 106</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">NA</td></tr><tr><td colspan="1" rowspan="4">Haemophilus influenzae</td><td colspan="1" rowspan="1">rare</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">b</td><td colspan="1" rowspan="1">103 - &lt; 104</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">NA</td></tr><tr><td colspan="1" rowspan="1">few</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">b</td><td colspan="1" rowspan="1">104- &lt;105</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">83.3</td></tr><tr><td colspan="1" rowspan="1">moderate</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">b</td><td colspan="1" rowspan="1">105- &lt;106</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.0</td></tr><tr><td colspan="1" rowspan="1">numerous</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">b</td><td colspan="1" rowspan="1">&gt; 106</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">NA</td></tr><tr><td colspan="1" rowspan="4">Klebsiella oxytoca</td><td colspan="1" rowspan="1">rare</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">b</td><td colspan="1" rowspan="1">103- &lt;104</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.0</td></tr><tr><td colspan="1" rowspan="1">few</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">b</td><td colspan="1" rowspan="1">104 - &lt;105</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">66.7</td></tr><tr><td colspan="1" rowspan="1">moderate</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">b</td><td colspan="1" rowspan="1">$10^5}- &lt;106</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">NA</td></tr><tr><td colspan="1" rowspan="1">numerous</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">b</td><td colspan="1" rowspan="1">&gt;106</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">NA</td></tr><tr><td colspan="1" rowspan="4">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">rare</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">b</td><td colspan="1" rowspan="1">103 - &lt; 104</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">75.0</td></tr><tr><td colspan="1" rowspan="1">few</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">75.0</td><td colspan="1" rowspan="1">b</td><td colspan="1" rowspan="1">104 - &lt; 105</td><td colspan="1" rowspan="1">11</td><td colspan="1" rowspan="1">11</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.0</td></tr><tr><td colspan="1" rowspan="1">moderate</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">b</td><td colspan="1" rowspan="1">105 - &lt; 106</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.0</td></tr><tr><td colspan="1" rowspan="1">numerous</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">b</td><td colspan="1" rowspan="1">&gt; 106</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">NA</td></tr><tr><td colspan="1" rowspan="4">Klebsiella variicola</td><td colspan="1" rowspan="1">rare</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">103 - &lt;104</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">NA</td></tr><tr><td colspan="1" rowspan="1">few</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">104 - &lt;105</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0.0</td></tr><tr><td colspan="1" rowspan="1">moderate</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">105 - &lt;106</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">NA</td></tr><tr><td colspan="1" rowspan="1">numerous</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">&gt; 106</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">NA</td></tr><tr><td colspan="1" rowspan="4">Moraxella catarrhalis</td><td colspan="1" rowspan="1">rare</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">103 - &lt; 104</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">NA</td></tr><tr><td colspan="1" rowspan="1">few</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">104 - &lt;105</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">NA</td></tr><tr><td colspan="1" rowspan="1">moderate</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 05- &lt;106</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">NA</td></tr><tr><td colspan="1" rowspan="1">numerous</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">&gt;106</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">NA</td></tr><tr><td colspan="1" rowspan="4">Morganella morganii</td><td colspan="1" rowspan="1">rare</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">103- &lt;104</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">NA</td></tr><tr><td colspan="1" rowspan="1">few</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">104-&lt;105</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">NA</td></tr><tr><td colspan="1" rowspan="1">moderate</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 05- &lt;106</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">NA</td></tr><tr><td colspan="1" rowspan="1">numerous</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">&gt; 106</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">NA</td></tr><tr><td colspan="1" rowspan="4">Proteus spp.</td><td colspan="1" rowspan="1">rare</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">103 - &lt;104</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.0</td></tr><tr><td colspan="1" rowspan="1">few</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">b</td><td colspan="1" rowspan="1">104 - &lt;105</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">NA</td></tr><tr><td colspan="1" rowspan="1">moderate</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">b</td><td colspan="1" rowspan="1">105- &lt;106</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">NA</td></tr><tr><td colspan="1" rowspan="1">numerous</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">&gt; 106</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">NA</td></tr><tr><td colspan="1" rowspan="4">Pseudomonas aeruginosa a</td><td colspan="1" rowspan="1">rare</td><td colspan="1" rowspan="1">7</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">71.4</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">103 - &lt;104</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.0</td></tr><tr><td colspan="1" rowspan="1">few</td><td colspan="1" rowspan="1">13</td><td colspan="1" rowspan="1">12</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">92.3</td><td colspan="1" rowspan="1">b</td><td colspan="1" rowspan="1">104 - &lt; 105</td><td colspan="1" rowspan="1">19</td><td colspan="1" rowspan="1">19</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.0</td></tr><tr><td colspan="1" rowspan="1">moderate</td><td colspan="1" rowspan="1">16</td><td colspan="1" rowspan="1">16</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">$10^5- &lt;106</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.0</td></tr><tr><td colspan="1" rowspan="1">numerous</td><td colspan="1" rowspan="1">7</td><td colspan="1" rowspan="1">7</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">&gt; 106</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">NA</td></tr><tr><td colspan="1" rowspan="4">Serratia marcescens</td><td colspan="1" rowspan="1">rare</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">103 - &lt; 104</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.0</td></tr><tr><td colspan="1" rowspan="1">few</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">104- &lt;105</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.0</td></tr><tr><td colspan="1" rowspan="1">moderate</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">10^5- &lt;106</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.0</td></tr><tr><td colspan="1" rowspan="1">numerous</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">&gt; 106</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">NA</td></tr><tr><td colspan="1" rowspan="4">Staphylococcus aureus a</td><td colspan="1" rowspan="1">rare</td><td colspan="1" rowspan="1">8</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">50.0</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">103 - &lt;104</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">80.0</td></tr><tr><td colspan="1" rowspan="1">few</td><td colspan="1" rowspan="1">11</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">90.9</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">104 - &lt; 105</td><td colspan="1" rowspan="1">18</td><td colspan="1" rowspan="1">17</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">94.4</td></tr><tr><td colspan="1" rowspan="1">moderate</td><td colspan="1" rowspan="1">11</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">90.9</td><td colspan="1" rowspan="1">b</td><td colspan="1" rowspan="1">10^5- &lt;106</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.0</td></tr><tr><td colspan="1" rowspan="1">numerous</td><td colspan="1" rowspan="1">7</td><td colspan="1" rowspan="1">7</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">&gt; 106</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">NA</td></tr><tr><td colspan="1" rowspan="4">Stenotrophomonas maltophilia</td><td colspan="1" rowspan="1">rare</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">103 - &lt; 104</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0.0</td></tr><tr><td colspan="1" rowspan="1">few</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">104-&lt;105</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.0</td></tr><tr><td colspan="1" rowspan="1">moderate</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">105- &lt;106</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">NA</td></tr><tr><td colspan="1" rowspan="1">numerous</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">83.3</td><td colspan="1" rowspan="1">b</td><td colspan="1" rowspan="1">&gt; 106</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">NA</td></tr><tr><td colspan="1" rowspan="4">Streptococcus pneumoniae</td><td colspan="1" rowspan="1">rare</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">103³ - &lt; 104</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">NA</td></tr><tr><td colspan="1" rowspan="1">few</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">104-&lt;105</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">NA</td></tr><tr><td colspan="1" rowspan="1">moderate</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">105- &lt;106</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.0</td></tr><tr><td colspan="1" rowspan="1">numerous</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">b</td><td colspan="1" rowspan="1">&gt; 106</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">NA</td></tr></table>

a One specimen with $P$ . aeruginosa and one specimen with $S .$ . aureus were reported positive by SoC culture without any qualitative or quantitative information, and were therefore not included in this table.

For ‘typical’ analytes, Table 25 includes clinical performance of the LRT BAL Application when compared to positive SoC culture results (any microorganism reported) and negative SoC results (no growth or normal/mixed flora results).

Table 25: Prospective Study, ‘Typical’ Analytes, Compared to SoC Culture   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Positive SoC Result (N=233)</td><td rowspan=1 colspan=4>Negative SoC Results (N=783)</td></tr><tr><td rowspan=1 colspan=1>LRT BAL Result</td><td rowspan=1 colspan=1>Microorganism(s) Reported</td><td rowspan=1 colspan=1>No Growth</td><td rowspan=1 colspan=1>Normal/Mixed Flora</td><td rowspan=1 colspan=1>NA a</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>any positive (N=363)</td><td rowspan=1 colspan=1>212</td><td rowspan=1 colspan=1>81</td><td rowspan=1 colspan=1>65</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>151</td></tr><tr><td rowspan=1 colspan=1>all negative (N=653)</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>358</td><td rowspan=1 colspan=1>258</td><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>632</td></tr></table>

a Presence or absence of flora not reported.

In the prospective study, the LRT BAL Application reported 363 specimens with at least one LRT BAL panel microorganism. For 113 specimens, multi-detections were reported for two or more LRT BAL panel microorganisms. SoC culture reported 233 specimens with at least one targeted ‘typical’ microorganism for 233 specimens. For 40 specimens, multidetections were reported with two or more targeted microorganisms for 40 specimens. Table 26 includes details on numbers of reported analytes by LRT BAL and SoC.

Table 26: Prospective Study, ‘Typical’ analytes, Comparison of Positivity   

<table><tr><td rowspan=2 colspan=1>Number of DetectedMicroorganisms</td><td rowspan=1 colspan=2>LRT BAL</td><td rowspan=1 colspan=2>SoC</td></tr><tr><td rowspan=1 colspan=1># specimens</td><td rowspan=1 colspan=1>[%]</td><td rowspan=1 colspan=1># specimens</td><td rowspan=1 colspan=1>[%]</td></tr><tr><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>653</td><td rowspan=1 colspan=1>64.3</td><td rowspan=1 colspan=1>783</td><td rowspan=1 colspan=1>77.1</td></tr><tr><td rowspan=1 colspan=1>any positive</td><td rowspan=1 colspan=1>363</td><td rowspan=1 colspan=1>35.7</td><td rowspan=1 colspan=1>233</td><td rowspan=1 colspan=1>22.9</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>250</td><td rowspan=1 colspan=1>24.6</td><td rowspan=1 colspan=1>194</td><td rowspan=1 colspan=1>19.1</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>78</td><td rowspan=1 colspan=1>7.7</td><td rowspan=1 colspan=1>31</td><td rowspan=1 colspan=1>3.1</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>19</td><td rowspan=1 colspan=1>1.9</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>0.5</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>0.7</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0.3</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>0.7</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0</td></tr><tr><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0</td></tr></table>

When comparing LRT BAL to $\mathrm { S o C }$ culture results for the prospective clinical study $\scriptstyle \left( \mathrm { N = 1 0 1 6 } \right)$ , 632 negative specimens $( 6 2 . 2 \% )$ and 139 positive specimens $( 1 3 . 7 \% )$ were concordant; the majority of discordant results were due to LRT BAL detection of additional microorganisms. Table 27 includes a detailed analysis of LRT BAL results compared to SoC culture with details on multi-detections and discordant results, stratified into the following categories:

category 1: concordant result for both LRT BAL and SoC

• category 1a: both negative, category 1b: both positive with a fully concordant result,   
category 2: LRT BAL reported additional microorganisms compared to SoC • category 2a: for specimens with a negative SoC result, • category 2b: for specimens with a positive SoC result,   
category 3: LRT BAL reported fewer microorganisms compared to SoC • category 3a: for specimens with a negative LRT BAL result, • category 3b: for specimens with a positive LRT BAL result,   
category 4: LRT BAL and SoC report discordant results with different microorganisms

category 4a: LRT BAL and SoC reportings share at least one concordant microorganism, category 4b: LRT BAL and SoC results are fully discordant.

Table 27: Prospective Study, ‘Typical’ Analytes, Analysis of Multi-Detections   

<table><tr><td colspan="1" rowspan="1">LRT BAL Result</td><td colspan="1" rowspan="1">SoC Result</td><td colspan="1" rowspan="1"># Cases</td><td colspan="1" rowspan="1">%</td></tr><tr><td colspan="2" rowspan="1">Category 1: concordant LRT BAL and SoC results</td><td colspan="2" rowspan="1">Overall concordantresults:75.9%</td></tr><tr><td colspan="2" rowspan="1">Category 1a: both negative</td><td colspan="1" rowspan="1">632</td><td colspan="1" rowspan="1">62.2</td></tr><tr><td colspan="2" rowspan="1">Category 1b: concordant positive results</td><td colspan="1" rowspan="1">139</td><td colspan="1" rowspan="1">13.7</td></tr><tr><td colspan="1" rowspan="1">Acinetobacter spp.</td><td colspan="1" rowspan="1">Acinetobacter spp.</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Enterobacter cloacae complex</td><td colspan="1" rowspan="1">Enterobacter cloacae complex</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Haemophilus influenzae</td><td colspan="1" rowspan="1">Haemophilus influenzae</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Moraxella catarrhalis</td><td colspan="1" rowspan="1">Moraxella catarrhalis</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Pneumocystis jirovecii</td><td colspan="1" rowspan="1">Pneumocystis jirovecii</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">31</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Serratia marcescens</td><td colspan="1" rowspan="1">Serratia marcescens</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">39</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Stenotrophomonas maltophilia</td><td colspan="1" rowspan="1">Stenotrophomonas maltophilia</td><td colspan="1" rowspan="1">8</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Streptococcus pneumoniae</td><td colspan="1" rowspan="1">Streptococcus pneumoniae</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Acinetobacter spp.,Citrobacter freundii</td><td colspan="1" rowspan="1">Acinetobacter spp.,Citrobacter freundii</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Acinetobacter spp., Stenotrophomonasmaltophilia</td><td colspan="1" rowspan="1">Acinetobacter spp.,Stenotrophomonas maltophilia</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Enterobacter cloacae complex,Staphylococcus aureus</td><td colspan="1" rowspan="1">Enterobacter cloacae complex,Staphylococcus aureus</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Escherichia coli,Klebsiella oxytoca</td><td colspan="1" rowspan="1">Escherichia coli,Klebsiella oxytoca</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Klebsiella oxytoca,Staphylococcus aureus</td><td colspan="1" rowspan="1">Klebsiella oxytoca,Staphylococcus aureus</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae,Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">Klebsiella pneumoniae,Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Proteus spp.,Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">Proteus spp.,Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Pseudomonas aeruginosa, Staphylococcusaureus</td><td colspan="1" rowspan="1">Pseudomonas aeruginosa, Staphylococcusaureus</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Pseudomonas aeruginosa, Stenotrophomonasmaltophilia</td><td colspan="1" rowspan="1">Pseudomonas aeruginosa, Stenotrophomonasmaltophilia</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Serratia marcescens,Staphylococcus aureus</td><td colspan="1" rowspan="1">Serratia marcescens,Staphylococcus aureus</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus, Stenotrophomonasmaltophilia</td><td colspan="1" rowspan="1">Staphylococcus aureus, Stenotrophomonasmaltophilia</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Proteus spp.,Pseudomonas aeruginosa, Staphylococcusaureus</td><td colspan="1" rowspan="1">Proteus spp.,Pseudomonas aeruginosa, Staphylococcusaureus</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="2" rowspan="1">Category 2: LRT BAL detects additional microorganisms</td><td colspan="2" rowspan="1">additional detections:21.1 %</td></tr><tr><td colspan="2" rowspan="1">Category 2a: negative SoC result</td><td colspan="1" rowspan="1">151</td><td colspan="1" rowspan="1">14.9</td></tr><tr><td colspan="1" rowspan="1">Acinetobacter spp.</td><td colspan="1" rowspan="1">negative</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Citrobacter freundii</td><td colspan="1" rowspan="1">negative</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Enterobacter cloacae complex</td><td colspan="1" rowspan="1">negative</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">negative</td><td colspan="1" rowspan="1">14</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Haemophilus influenzae</td><td colspan="1" rowspan="1">negative</td><td colspan="1" rowspan="1">24</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Klebsiella oxytoca</td><td colspan="1" rowspan="1">negative</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">negative</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Legionella pneumophila</td><td colspan="1" rowspan="1">negative</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Moraxella catarrhalis</td><td colspan="1" rowspan="1">negative</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Mycoplasma pneumoniae</td><td colspan="1" rowspan="1">negative</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Pneumocystis jirovecii</td><td colspan="1" rowspan="1">negative</td><td colspan="1" rowspan="1">9</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">negative</td><td colspan="1" rowspan="1">27</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">negative</td><td colspan="1" rowspan="1">21</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Stenotrophomonas maltophilia</td><td colspan="1" rowspan="1">negative</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Streptococcus pneumoniae</td><td colspan="1" rowspan="1">negative</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Enterobacter cloacae complex,Staphylococcus aureus</td><td colspan="1" rowspan="1">negative</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Escherichia coli,Klebsiella pneumoniae</td><td colspan="1" rowspan="1">negative</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Escherichia coli,Serratia marcescens</td><td colspan="1" rowspan="1">negative</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Haemophilus influenzae,Moraxella catarrhalis</td><td colspan="1" rowspan="1">negative</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Haemophilus influenzae,Mycoplasma pneumoniae</td><td colspan="1" rowspan="1">negative</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Haemophilus influenzae, Staphylococcusaureus</td><td colspan="1" rowspan="1">negative</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Haemophilus influenzae, Streptococcuspneumoniae</td><td colspan="1" rowspan="1">negative</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Klebsiella oxytoca,Proteus spp.</td><td colspan="1" rowspan="1">negative</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae, Staphylococcusaureus</td><td colspan="1" rowspan="1">negative</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Moraxella catarrhalis,Pneumocystis jirovecii</td><td colspan="1" rowspan="1">negative</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Moraxella catarrhalis,Streptococcus pneumoniae</td><td colspan="1" rowspan="1">negative</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Pneumocystis jirovecii, Staphylococcusaureus</td><td colspan="1" rowspan="1">negative</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Pseudomonas aeruginosa, Staphylococcusaureus</td><td colspan="1" rowspan="1">negative</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus, Stenotrophomonasmaltophilia</td><td colspan="1" rowspan="1">negative</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Enterobacter cloacae complex, Haemophilusinfluenzae,Moraxella catarrhalis</td><td colspan="1" rowspan="1">negative</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Escherichia coli,Haemophilus influenzae, Staphylococcusaureus</td><td colspan="1" rowspan="1">negative</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Klebsiella oxytoca,Pseudomonas aeruginosa, Staphylococcusaureus</td><td colspan="1" rowspan="1">negative</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Enterobacter cloacae complex, Haemophilusinfluenzae,Moraxella catarrhalis,Morganella morganii,Saphylococcus aureus, Stenotrophomonasmaltophilia</td><td colspan="1" rowspan="1">negative</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">−</td></tr><tr><td colspan="2" rowspan="1">Category 2b: positive result for both LRT BAL and SoC</td><td colspan="1" rowspan="1">63</td><td colspan="1" rowspan="1">6.2</td></tr><tr><td colspan="1" rowspan="1">Acinetobacter spp.,Klebsiella pneumoniae</td><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Acinetobacter spp.,Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Enterobacter cloacae complex, Mycoplasmapneumoniae</td><td colspan="1" rowspan="1">Enterobacter cloacae complex</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Escherichia coli,Haemophilus influenzae</td><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Escherichia coli,Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Escherichia coli,Serratia marcescens</td><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Escherichia coli,Staphylococcus aureus</td><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Escherichia coli,Staphylococcus aureus</td><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Haemophilus influenzae,Klebsiella oxytoca</td><td colspan="1" rowspan="1">Klebsiella oxytoca</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Haemophilus influenzae,Moraxella catarrhalis</td><td colspan="1" rowspan="1">Haemophilus influenzae</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Haemophilus influenzae, Pseudomonasaeruginosa</td><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Haemophilus influenzae, Staphylococcusaureus</td><td colspan="1" rowspan="1">Haemophilus influenzae</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Haemophilus influenzae, Staphylococcusaureus</td><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Klebsiella oxytoca,Klebsiella pneumoniae</td><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Klebsiella oxytoca,Serratia marcescens</td><td colspan="1" rowspan="1">Klebsiella oxytoca</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Klebsiella oxytoca,Stenotrophomonas maltophilia</td><td colspan="1" rowspan="1">Klebsiella oxytoca</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae,Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae, Staphylococcusaureus</td><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae, Staphylococcusaureus</td><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Pneumocystis jirovecii,Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">cstis jiroveci, Stenotohomonasmaltophilia</td><td colspan="1" rowspan="1">Stenotrophomonas maltophilia</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Proteus spp.,Staphylococcus aureus</td><td colspan="1" rowspan="1">Proteus spp.</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Pseudomonas aeruginosa,Serratia marcescens</td><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Pseudomonas aeruginosa, Staphylococcusaureus</td><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Pseudomonas aeruginosa, Staphylococcusaureus</td><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Pseudomonas aeruginosa, Stenotrophomonasmaltophilia</td><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus, Stenotrophomonasmaltophilia</td><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Acinetobacter spp.,Klebsiella pneumoniae, Staphylococcusaureus</td><td colspan="1" rowspan="1">Klebsiella pneumoniae, Staphylococcusaureus</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Acinetobacter spp.,Klebsiella pneumoniae, Stenotrophomonasmaltophilia</td><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Acinetobacter spp.,a ruginosa Senotrohoonasmaltophilia</td><td colspan="1" rowspan="1">Acinetobacter spp.,Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Enterobacter cloacae complex, Klebsiellapneumoniae,Moraxella catarrhalis</td><td colspan="1" rowspan="1">Enterobacter cloacae complex</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Enterobacter cloacae complex, Klebsiellapneumoniae,Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">Enterobacter cloacae complex</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Escherichia coli,Morganella morganii,Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Escherichia coli,Pneumocystis jirovecii,Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Escherichia coli,Pseudomonas aeruginosa,Serratia marcescens</td><td colspan="1" rowspan="1">Pseudomonas aeruginosa,Serratia marcescens</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Escherichia coli,Pseudomonas aeruginosa,Staphylococcus aureus</td><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Haemophilus influenzae,Moraxella catarrhalis,Streptococcus pneumoniae</td><td colspan="1" rowspan="1">Moraxella catarrhalis,Streptococcus pneumoniae</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Klebsiella oxytoca,Pseudomonas aeruginosa, Stenotrophomonasmaltophilia</td><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae,Pseudomonas aeruginosa, Stenotrophomonasmaltophilia</td><td colspan="1" rowspan="1">Klebsiella pneumoniae,Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Pseudomonas aeruginosa,Serratia marcescens, Stenotrophomonasmaltophilia</td><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Pseudomonas aeruginosa,Serratia marcescens, Stenotrophomonasmaltophilia</td><td colspan="1" rowspan="1">Pseudomonas aeruginosa,Serratia marcescens</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Acinetobacter spp.,Escherichia coli,Klebsiella pneumoniae,Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">Klebsiella pneumoniae,Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Acinetobacter spp.,Escherichia coli,Klebsiella pneumoniae, Stenotrophomonasmaltophilia</td><td colspan="1" rowspan="1">Acinetobacter spp.</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Acinetobacter spp.,Escherichia coli,Proteus spp.,Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Enterobacter cloacae complex, Escherichiacoli,Klebsiella oxytoca,Stenotrophomonas maltophilia</td><td colspan="1" rowspan="1">Enterobacter cloacae complex, Escherichiacoli,Klebsiella oxytoca</td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Escherichia coli,Haemophilus influenzae, Pneumocystisjirovecii, Staphylococcus aureus</td><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">−1</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Haemophilus influenzae,Proteus spp.,Pseudomonas aeruginosa, Stenotrophomonasmaltophilia</td><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Pseudomonas aeruginosa,Serratia marcescens,Staphylococcus aureus, Stenotrophomonasmaltophilia</td><td colspan="1" rowspan="1">Pseudomonas aeruginosa,Serratia marcescens,Staphylococcus aureus</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Acinetobacter spp.,Enterobacter cloacae complex, Haemophilusinfluenzae,Klebsiella pneumoniae,Streptococcus pneumoniae</td><td colspan="1" rowspan="1">Enterobacter cloacae complex, Klebsiellapneumoniae,Streptococcus pneumoniae</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Citrobacter freundii,Escherichia coli,Klebsiella pneumoniae,Pseudomonas aeruginosa,Serratia marcescens</td><td colspan="1" rowspan="1">Escherichia coli,Klebsiella pneumoniae,Pseudomonas aeruginosa,Serratia marcescens</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Enterobacter cloacae complex, Escherichiacoli,Klebsiella oxytoca,Klebsiella variicola,Proteus spp.</td><td colspan="1" rowspan="1">Enterobacter cloacae complex</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Escherichia coli,Klebsiella pneumoniae,Pseudomonas aeruginosa,Serratia marcescens, Stenotrophomonasmaltophilia</td><td colspan="1" rowspan="1">Klebsiella pneumoniae,Pseudomonas aeruginosa,Serratia marcescens, Stenotrophomonasmaltophilia</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Escherichia coli,Morganella morganii,Proteus spp.,Pseudomonas aeruginosa, Stenotrophomonasmaltophilia</td><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Haemophilus influenzae,Klebsiella pneumoniae,Klebsiella variicola,Proteus spp.,Staphylococcus aureus</td><td colspan="1" rowspan="1">Haemophilus influenzae,Klebsiella pneumoniae,Proteus spp.,Staphylococcus aureus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Haemophilus influenzae,Klebsiella pneumoniae,Moraxella catarrhalis,Pneumocystis jirovecii,Serratia marcescens</td><td colspan="1" rowspan="1">Serratia marcescens</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Acinetobacter spp.,Escherichia coli,Klebsiella pneumoniae,Proteus spp.,Pseudomonas aeruginosa, Staphylococcusaureus</td><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">−</td></tr><tr><td colspan="2" rowspan="1">Category 3: LRT BAL detects fewer microorganisms</td><td colspan="2" rowspan="1">fewer detections:2.5 %</td></tr><tr><td colspan="2" rowspan="1">Category 3a: LRT BAL negative</td><td colspan="1" rowspan="1">21</td><td colspan="1" rowspan="1">2.1</td></tr><tr><td colspan="1" rowspan="1">negative</td><td colspan="1" rowspan="1">Enterobacter cloacae complex</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">negative</td><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">negative</td><td colspan="1" rowspan="1">Klebsiella oxytoca</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">negative</td><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">negative</td><td colspan="1" rowspan="1">Klebsiella variicola</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">negative</td><td colspan="1" rowspan="1">Legionella pneumophila</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">negative</td><td colspan="1" rowspan="1">Pneumocystis jirovecii</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">negative</td><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">negative</td><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">negative</td><td colspan="1" rowspan="1">Stenotrophomonas maltophilia</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="2" rowspan="1">Category 3b: positive result for both LRT BAL and SoC</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">0.4</td></tr><tr><td colspan="1" rowspan="1">Acinetobacter spp.</td><td colspan="1" rowspan="1">Acinetobacter spp.,Enterobacter cloacae complex</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">Klebsiella pneumoniae,Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Stenotrophomonas maltophilia</td><td colspan="1" rowspan="1">Haemophilus influenzae, Stenotrophomonasmaltophilia</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Acinetobacter spp.,Enterobacter cloacae complex</td><td colspan="1" rowspan="1">Acinetobacter spp.,Enterobacter cloacae complex, Klebsiellavariicola</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="2" rowspan="1">Category 4 LRT BAL and SoC detect different microorganisms</td><td colspan="2" rowspan="1">discordant microorganismresults:0.6 %</td></tr><tr><td colspan="2" rowspan="1">Category 4a: partially concordant positive results</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0.2</td></tr><tr><td colspan="1" rowspan="1">Citrobacter freundii,Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">Pseudomonas aeruginosa, Staphylococcusaureus</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Enterobacter cloacae complex, Klebsiellaoxytoca,Stenotrophomonas maltophilia</td><td colspan="1" rowspan="1">Enterobacter cloacae complex, Klebsiellapneumoniae</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="2" rowspan="1">Category 4b: fully discordant results</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">0.4</td></tr><tr><td colspan="1" rowspan="1">Haemophilus influenzae</td><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">Acinetobacter spp.</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Stenotrophomonas maltophilia</td><td colspan="1" rowspan="1">Enterobacter cloacae complex</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1"></td></tr></table>

# 2. Clinical Study, Archived Specimens:

The LRT BAL Application was further evaluated using selected archived frozen BAL specimens from patients, age 18 or older, suspected of lower respiratory infection, and with at least one LRT BAL microorganisms reported positive by SoC testing. Specimens were collected at eleven US clinical sites, and supplemented with other specimens from additional sites. In contrast to the prospective study, where specimens were only from hospitalized patients (inpatients), specimens from outpatients or from patients with an unknown status were also accepted. Specimens were selected based on positivity by SoC culture (for ‘typical’ analytes) or positivity by other SoC methods (for atypical analytes) such as IFA, DFA, or molecular assays. For analytes with positive SoC results, confirmatory testing (PCR/bi-directional sequencing using two independent PCR assays per applicable analyte) was performed to assess for potential specimen degradation during storage. Specimens were included in the archived study if either one of the PCR/sequencing confirmatory assays was positive for applicable analyte reported as positive by SoC. Likewise, specimens were excluded only if both confirmatory PCR assays were negative. The archived study cohort included 220 selected specimens from a previous study with 23 specimens $( 1 0 . 5 \% )$ excluded because of insufficient specimen volume. The remaining 197 specimens $( 8 9 . 5 \% )$ were tested with the LRT BAL Application without any further preselection. An additional 195 archived specimens that met inclusion criteria were also evaluated for a total set of 392 archived study specimens. Table 28 includes patient demographic information for LRT BAL archived study.

Table 28: Archived Study, Patient Demographics   

<table><tr><td rowspan=1 colspan=1>Included Specimens</td><td rowspan=1 colspan=1>392</td></tr><tr><td rowspan=1 colspan=2>Sex a</td></tr><tr><td rowspan=1 colspan=1>male</td><td rowspan=1 colspan=1>198</td></tr><tr><td rowspan=1 colspan=1>female</td><td rowspan=1 colspan=1>122</td></tr><tr><td rowspan=1 colspan=2>Age b</td></tr><tr><td rowspan=1 colspan=1>18 - 30 years</td><td rowspan=1 colspan=1>18</td></tr><tr><td rowspan=1 colspan=1>31 - 60 years</td><td rowspan=1 colspan=1>106</td></tr><tr><td rowspan=1 colspan=1>&gt; 60 years</td><td rowspan=1 colspan=1>190</td></tr><tr><td rowspan=1 colspan=2>Clinical Setting c</td></tr><tr><td rowspan=1 colspan=1>ICU</td><td rowspan=1 colspan=1>112</td></tr><tr><td rowspan=1 colspan=1>ward</td><td rowspan=1 colspan=1>101</td></tr><tr><td rowspan=1 colspan=1>in-patient (not specified)</td><td rowspan=1 colspan=1>69</td></tr><tr><td rowspan=1 colspan=1>out-patient</td><td rowspan=1 colspan=1>32</td></tr></table>

a Not reported: 72 patients. b Not reported: 78 patients. c Not reported: 78 patients.

For archived specimens, LRT BAL Application microorganism target results were compared to ‘confirmed’ results from SoC microbiology culture (for ‘typical’ microorganism) which were reported either qualitatively or quantitatively. For quantitative cultures, a reporting threshold of $1 0 ^ { 3 } \mathrm { C F U / m L }$ or higher for mini-BAL specimens and of $1 0 ^ { 4 }$ CFU/mL or higher for BAL specimens was applied, following recommendations by the Infectious Diseases Society of America (IDSA) and the American Thoracic Society (ATS). Results from the archived study for ‘typical’ microorganism targeted are presented in Table 29.

Table 29: Archived Study, Performance, ‘Typical’ Microorganisms   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>TP</td><td rowspan=1 colspan=1>FN</td><td rowspan=1 colspan=1>FP b</td><td rowspan=1 colspan=1>TN</td><td rowspan=1 colspan=1>PPA [%] (95% CI)</td><td rowspan=1 colspan=1>NPA [%] (95% CI)</td></tr><tr><td rowspan=1 colspan=1>Acinetobacter spp.</td><td rowspan=1 colspan=1>18</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>371</td><td rowspan=1 colspan=1>100.0 (82.4 - 100.0)</td><td rowspan=1 colspan=1>99.2 (97.7 - 99.7)</td></tr><tr><td rowspan=1 colspan=1>Citrobacter freundii</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>382</td><td rowspan=1 colspan=1>100.0 (56.6 - 100.0)</td><td rowspan=1 colspan=1>98.7 (97.0 - 99.4)</td></tr><tr><td rowspan=1 colspan=1>Enterobacter cloacae complex</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>4c</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>373</td><td rowspan=1 colspan=1>78.9 (56.7 - 91.5)</td><td rowspan=1 colspan=1>100.0 (99.0 - 100.0)</td></tr><tr><td rowspan=1 colspan=1>Escherichia coli</td><td rowspan=1 colspan=1>46</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>326</td><td rowspan=1 colspan=1>93.9 (83.5 - 97.9)</td><td rowspan=1 colspan=1>95.0 (92.2 - 96.9)</td></tr><tr><td rowspan=1 colspan=1>Haemophilus influenzae</td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>321</td><td rowspan=1 colspan=1>100.0 (92.9 - 100.0)</td><td rowspan=1 colspan=1>93.9 (90.8 - 95.9)</td></tr><tr><td rowspan=1 colspan=1>Klebsiella oxytoca</td><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>369</td><td rowspan=1 colspan=1>94.1 (73.0 - 99.0)</td><td rowspan=1 colspan=1>98.4 (96.6 - 99.3)</td></tr><tr><td rowspan=1 colspan=1>Klebsiella pneumoniae </td><td rowspan=1 colspan=1>29</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>351</td><td rowspan=1 colspan=1>93.5 (79.3 - 98.2)</td><td rowspan=1 colspan=1>97.2 (95.0 - 98.5)</td></tr><tr><td rowspan=1 colspan=1>Klebsiella variicola </td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>386</td><td rowspan=1 colspan=1>100.0 (34.2 - 100.0)</td><td rowspan=1 colspan=1>99.0 (97.4 - 99.6)</td></tr><tr><td rowspan=1 colspan=1>Moraxella catarrhalis</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>362</td><td rowspan=1 colspan=1>100.0 (84.5 - 100.0)</td><td rowspan=1 colspan=1>97.6 (95.5 - 98.7)</td></tr><tr><td rowspan=1 colspan=1>Morganella morganii</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>391</td><td rowspan=1 colspan=1>100.0 (20.7 - 100.0)</td><td rowspan=1 colspan=1>100.0 (99.0 - 100.0)</td></tr><tr><td rowspan=1 colspan=1>Proteus spp.</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>370</td><td rowspan=1 colspan=1>100.0 (79.6 - 100.0)</td><td rowspan=1 colspan=1>98.1 (96.2 - 99.1)</td></tr><tr><td rowspan=1 colspan=1>Pseudomonas aeruginosa</td><td rowspan=1 colspan=1>54</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>334</td><td rowspan=1 colspan=1>96.4 (87.9 - 99.0)</td><td rowspan=1 colspan=1>99.4 (97.9 - 99.8)</td></tr><tr><td rowspan=1 colspan=1>Serratia marcescens</td><td rowspan=1 colspan=1>23</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>364</td><td rowspan=1 colspan=1>92.0 (75.0 - 97.8)</td><td rowspan=1 colspan=1>99.2 (97.6 - 99.7)</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus aureus</td><td rowspan=1 colspan=1>56</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>313</td><td rowspan=1 colspan=1>96.6 (88.3 - 99.1)</td><td rowspan=1 colspan=1>93.7 (90.6 - 95.9)</td></tr><tr><td rowspan=1 colspan=1>Stenotrophomonas maltophilia</td><td rowspan=1 colspan=1>37</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>342</td><td rowspan=1 colspan=1>92.5 (80.1 - 97.4)</td><td rowspan=1 colspan=1>97.2 (94.9 - 98.4)</td></tr><tr><td rowspan=1 colspan=1>Streptococcus pneumoniae</td><td rowspan=1 colspan=1>34</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>352</td><td rowspan=1 colspan=1>97.1 (85.5 - 99.5)</td><td rowspan=1 colspan=1>98.6 (96.8 - 99.4)</td></tr></table>

a K. variicola is typically reported by culture as $K .$ . pneumoniae. To discriminate $K .$ . variicola from $K .$ . pneumoniae, K. pneumoniae strain isolates, if available, were sequenced. If no isolate was provided, sequencing was performed from specimen DNA extracts instead. Performance for $K .$ . pneumoniae and $K .$ variicola are calculated based on the species identified by sequencing.

b Specimens with false positive LRT BAL results were analyzed with molecular assays (PCR/bi-directional sequencing) using specimen DNA extracts for presence or absence of microorganisms. Presence of microorganisms was confirmed in 3 of 3 cases for Acinetobacter spp., 4 of 5 cases for C. freundii (one confirmed case was reported as $C .$ . youngae by SoC), 15 of 17 cases for E. coli, 17 of 21 cases for $H .$ . influenzae, 6 of 6 cases for $K .$ . oxytoca, 7 of 10 cases for $K .$ . pneumoniae, 2 of 4 cases for K. variicola, 8 of 9 cases for M. catarrhalis, 6 of 7 cases for Proteus spp., 2 of 2 cases for $P$ . aeruginosa, 1 of 3 cases for S. marcescens, 18 of 21 cases for S. aureus, 10 of 10 cases for S. maltophilia, and 4 of 5 cases for $S _ { \cdot }$ . pneumoniae. For cases that were not confirmed for $H .$ . influenzae, sequencing identified $H .$ haemolyticus (2x), A. aphropilus (2x). For all other cases, PCRs did not amplify sufficient amounts for sequencing.

c Three of the four specimens that were FN for $E$ . cloacae complex contained multiple host microorganisms identified by culture, and/or LRT BAL. For one specimen, Klebsiella oxytoca was additionally reported by both LRT BAL and culture. For the two other specimens, Proteus spp. or E. coli, respectively, were additionally reported by LRT BAL only.

For ‘atypical’ microorganism targets, SoC results were available for only a subset of specimens. For these specimens, comparator testing was not performed for other analytes; therefore negative percent agreement was not evaluated. Results for positive percent agreement are shown in Table 30.

Table 30: Archived study, Performance, Atypical analytes   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>TP</td><td rowspan=1 colspan=1>FN</td><td rowspan=1 colspan=1>add. pos. a</td><td rowspan=1 colspan=1>add. neg.</td><td rowspan=1 colspan=1>PPA [%] (95% CI)</td></tr><tr><td rowspan=1 colspan=1>Chlamydia pneumoniae</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>392</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=1>Legionella pneumophila</td><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>373</td><td rowspan=1 colspan=1>89.5 (68.6 - 97.1)</td></tr><tr><td rowspan=1 colspan=1>Mycoplasma pneumoniae</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>383</td><td rowspan=1 colspan=1>83.3 (43.7 - 97.0)</td></tr><tr><td rowspan=1 colspan=1>Pneumocystis jirovecii</td><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>360</td><td rowspan=1 colspan=1>84.2 (62.4 - 94.5)</td></tr></table>

a Specimens with additional positive (‘add. pos.’) LRT BAL results were analyzed with molecular assays (PCR/bi-directional sequencing) using specimen DNA extracts for presence or absence of microorganisms: presence was confirmed in 2 of 3 cases for $M .$ . pneumoniae, and 10 of 13 cases for $P .$ . jirovecii. For all other cases, PCRs did not amplify sufficient amounts for sequencing.

For the archived specimen cohort, an additional analysis of LRT BAL Application results stratified by semi-quantitative or quantitative SoC culture results are present in Table 31. Semi-quantitative result categories are rare, few, moderate, numerous and quantitative SoC culture categories are $1 0 ^ { 3 } \mathrm { ~ - } < 1 0 ^ { 4 }$ , $1 0 ^ { 4 } \mathrm { ~ - ~ } 1 0 ^ { 5 }$ , $1 0 ^ { 5 } - < 1 0 ^ { 6 } , > 1 0 ^ { 6 }$ , in $\mathrm { C F U / m L }$ .

Table 31: Archived Study Performance Compared to Semi-quantitative or Quantitative SoC Culture   

<table><tr><td colspan="1" rowspan="1"></td><td colspan="5" rowspan="1">Qualitative Culture Result a</td><td colspan="5" rowspan="1">Quantitative Culture Result a</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Category</td><td colspan="1" rowspan="1">Total</td><td colspan="1" rowspan="1">TP</td><td colspan="1" rowspan="1">FN</td><td colspan="1" rowspan="1">PPA [%]</td><td colspan="1" rowspan="1">Category</td><td colspan="1" rowspan="1">Total</td><td colspan="1" rowspan="1">TP</td><td colspan="1" rowspan="1">FN</td><td colspan="1" rowspan="1">PPA[%]</td></tr><tr><td colspan="1" rowspan="4">Acinetobacter spp.</td><td colspan="1" rowspan="1">rare</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">103-&lt;104</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.0</td></tr><tr><td colspan="1" rowspan="1">few</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">104-&lt;105</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.0</td></tr><tr><td colspan="1" rowspan="1">moderate</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">105 - &lt; 106</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.0</td></tr><tr><td colspan="1" rowspan="1">numerous</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">&gt;106</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">NA</td></tr><tr><td colspan="1" rowspan="4">Citrobacter freundii</td><td colspan="1" rowspan="1">rare</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">103-&lt;104</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">NA</td></tr><tr><td colspan="1" rowspan="1">few</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">104- &lt;105</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.0</td></tr><tr><td colspan="1" rowspan="1">moderate</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">10 5- &lt;106</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.0</td></tr><tr><td colspan="1" rowspan="1">numerous</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">&gt;106</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">NA</td></tr><tr><td colspan="1" rowspan="4">Enterobacter cloacaecomplex</td><td colspan="1" rowspan="1">rare</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">103- &lt;104</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0.0</td></tr><tr><td colspan="1" rowspan="1">few</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">104-&lt;105</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">50.0</td></tr><tr><td colspan="1" rowspan="1">moderate</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">66.7</td><td colspan="1" rowspan="1">105- &lt;106</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.0</td></tr><tr><td colspan="1" rowspan="1">numerous</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">&gt;106</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">NA</td></tr><tr><td colspan="1" rowspan="4">Escherichia coli</td><td colspan="1" rowspan="1">rare</td><td colspan="1" rowspan="1">7</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">85.7</td><td colspan="1" rowspan="1">103- &lt;104</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">NA</td></tr><tr><td colspan="1" rowspan="1">few</td><td colspan="1" rowspan="1">12</td><td colspan="1" rowspan="1">12</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">104 - &lt;105</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.0</td></tr><tr><td colspan="1" rowspan="1">moderate</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">105- &lt;106</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.0</td></tr><tr><td colspan="1" rowspan="1">numerous</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">&gt;106</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">NA</td></tr><tr><td colspan="1" rowspan="4">Haemophilus influenzae</td><td colspan="1" rowspan="1">rare</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">103- &lt;104</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">NA</td></tr><tr><td colspan="1" rowspan="1">few</td><td colspan="1" rowspan="1">8</td><td colspan="1" rowspan="1">8</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">104-&lt;105</td><td colspan="1" rowspan="1">8</td><td colspan="1" rowspan="1">8</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.0</td></tr><tr><td colspan="1" rowspan="1">moderate</td><td colspan="1" rowspan="1">7</td><td colspan="1" rowspan="1">7</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">105- &lt;106</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.0</td></tr><tr><td colspan="1" rowspan="1">numerous</td><td colspan="1" rowspan="1">13</td><td colspan="1" rowspan="1">13</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">&gt;106</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.0</td></tr><tr><td colspan="1" rowspan="4">Klebsiella oxytoca</td><td colspan="1" rowspan="1">rare</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">103- &lt;104</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.0</td></tr><tr><td colspan="1" rowspan="1">few</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">80.0</td><td colspan="1" rowspan="1">104-&lt;105</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.0</td></tr><tr><td colspan="1" rowspan="1">moderate</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">105 - &lt; 106</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.0</td></tr><tr><td colspan="1" rowspan="1">numerous</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">&gt;106</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">NA</td></tr><tr><td colspan="1" rowspan="4">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">rare</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">103- &lt;104</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">NA</td></tr><tr><td colspan="1" rowspan="1">few</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">104 - &lt; 105</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.0</td></tr><tr><td colspan="1" rowspan="1">moderate</td><td colspan="1" rowspan="1">9</td><td colspan="1" rowspan="1">8</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">88.9</td><td colspan="1" rowspan="1">105- &lt;106</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.0</td></tr><tr><td colspan="1" rowspan="1">numerous</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">&gt;106</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">NA</td></tr><tr><td colspan="1" rowspan="4">Klebsiella variicola</td><td colspan="1" rowspan="1">rare</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">103-&lt;104</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">NA</td></tr><tr><td colspan="1" rowspan="1">few</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">104-&lt;105</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">NA</td></tr><tr><td colspan="1" rowspan="1">moderate</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">105 - &lt; 106</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">NA</td></tr><tr><td colspan="1" rowspan="1">numerous</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">&gt;106</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">NA</td></tr><tr><td colspan="1" rowspan="4">Moraxella catarrhalis</td><td colspan="1" rowspan="1">rare</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">103- &lt;104</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.0</td></tr><tr><td colspan="1" rowspan="1">few</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">104-&lt;105</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.0</td></tr><tr><td colspan="1" rowspan="1">moderate</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">10 5- &lt;106</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.0</td></tr><tr><td colspan="1" rowspan="1">numerous</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">&gt;106</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.0</td></tr><tr><td colspan="1" rowspan="4">Morganella morganii</td><td colspan="1" rowspan="1">rare</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">103-&lt;104</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.0</td></tr><tr><td colspan="1" rowspan="1">few</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">104-&lt;105</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">NA</td></tr><tr><td colspan="1" rowspan="1">moderate</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">105- &lt;106</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">NA</td></tr><tr><td colspan="1" rowspan="1">numerous</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">&gt;106</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">NA</td></tr><tr><td colspan="1" rowspan="4">Proteus spp.</td><td colspan="1" rowspan="1">rare</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">103- &lt;104</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">NA</td></tr><tr><td colspan="1" rowspan="1">few</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">104-&lt;105</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.0</td></tr><tr><td colspan="1" rowspan="1">moderate</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">105 - &lt;106</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.0</td></tr><tr><td colspan="1" rowspan="1">numerous</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">&gt;106</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">NA</td></tr><tr><td colspan="1" rowspan="4">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">rare</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">66.7</td><td colspan="1" rowspan="1">103 - &lt;104</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.0</td></tr><tr><td colspan="1" rowspan="1">few</td><td colspan="1" rowspan="1">7</td><td colspan="1" rowspan="1">7</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">104- &lt;105</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.0</td></tr><tr><td colspan="1" rowspan="1">moderate</td><td colspan="1" rowspan="1">12</td><td colspan="1" rowspan="1">12</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">105- &lt;106</td><td colspan="1" rowspan="1">7</td><td colspan="1" rowspan="1">7</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.0</td></tr><tr><td colspan="1" rowspan="1">numerous</td><td colspan="1" rowspan="1">7</td><td colspan="1" rowspan="1">7</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">&gt;106</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">NA</td></tr><tr><td colspan="1" rowspan="4">Serratia marcescens</td><td colspan="1" rowspan="1">rare</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">103 - &lt;104</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.0</td></tr><tr><td colspan="1" rowspan="1">few</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">75.0</td><td colspan="1" rowspan="1">104-&lt;105</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.0</td></tr><tr><td colspan="1" rowspan="1">moderate</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">105- &lt;106</td><td colspan="1" rowspan="1">8</td><td colspan="1" rowspan="1">8</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.0</td></tr><tr><td colspan="1" rowspan="1">numerous</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">&gt; 106</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">NA</td></tr><tr><td colspan="1" rowspan="4">Staphylococcus aureus</td><td colspan="1" rowspan="1">rare</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">83.3</td><td colspan="1" rowspan="1">103- &lt;104</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">NA</td></tr><tr><td colspan="1" rowspan="1">few</td><td colspan="1" rowspan="1">14</td><td colspan="1" rowspan="1">13</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">92.9</td><td colspan="1" rowspan="1">104-&lt;105</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.0</td></tr><tr><td colspan="1" rowspan="1">moderate</td><td colspan="1" rowspan="1">7</td><td colspan="1" rowspan="1">7</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">10^5- &lt;106</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.0</td></tr><tr><td colspan="1" rowspan="1">numerous</td><td colspan="1" rowspan="1">9</td><td colspan="1" rowspan="1">9</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">&gt;106</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.0</td></tr><tr><td colspan="1" rowspan="4">Stenotrophomonas maltophilia</td><td colspan="1" rowspan="1">rare</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">66.7</td><td colspan="1" rowspan="1">103- &lt; 104</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.0</td></tr><tr><td colspan="1" rowspan="1">few</td><td colspan="1" rowspan="1">18</td><td colspan="1" rowspan="1">16</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">88.9</td><td colspan="1" rowspan="1">104-&lt;105</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.0</td></tr><tr><td colspan="1" rowspan="1">moderate</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">105- &lt;106</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.0</td></tr><tr><td colspan="1" rowspan="1">numerous</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">&gt; 106</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">NA</td></tr><tr><td colspan="1" rowspan="4">Streptococcus pneumoniae</td><td colspan="1" rowspan="1">rare</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">50.0</td><td colspan="1" rowspan="1">103 - &lt;104</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">NA</td></tr><tr><td colspan="1" rowspan="1">few</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">104 -&lt;105</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.0</td></tr><tr><td colspan="1" rowspan="1">moderate</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">105- &lt;106</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.0</td></tr><tr><td colspan="1" rowspan="1">numerous</td><td colspan="1" rowspan="1">11</td><td colspan="1" rowspan="1">11</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">&gt;106</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.0</td></tr></table>

a Positive SoC specimens with no quantitation provided (not included in table): One Acinetobacter spp., 4 E. cloacae complex, 7 E. coli, $4 H .$ influenzae, $4 K .$ oxytoca, $5 K .$ . pneumoniae, 1 M. catarrhalis, 2 Proteus spp., $1 1 P$ . aeruginosa, 4 S. marcescens, 9 S. aureus, $3 \ : S .$ maltophilia and $1 \ S .$ . pneumoniae

# 3. Clinical Study, Contrived Specimens

For low prevalence LRT BAL analytes, the prospective and archived clinical studies were supplemented with contrived specimens which were prepared by spiking pooled microorganisms at concentrations close to their respective analyte LoD or at clinically relevant concentrations into unique negative BAL specimens. The following LRT BAL microorganism targets were evaluated using contrived specimens: C. pneumoniae, C. freundii, E. cloacae complex, K. oxytoca, K. pneumoniae, K. variicola, L. pneumophila, M. morganii, M. pneumoniae, $P .$ . jirovecii, Proteus spp., S. marcescens. In addition, all low prevalence LRT BAL antibiotic resistance markers (ctx-M, kpc, ndm, oxa-23, oxa-24, oxa48, oxa-58, vim) were included in the contrived study (See below for section titled, Clinical Study, Resistance Markers) . For each target analyte, 60 contrived specimens using up to five different reference strains were tested at a low concentration (2x LoD, 30 specimens) and at two moderate concentrations (3.5x LoD and 5x LoD, 15 specimens each) spanning the clinically relevant concentration range. For $P$ . jirovecii, a positive clinical specimen was used for spiking, and all 60 contrived specimens were tested at a $2 \mathbf { x }$ LoD concentration.

In Tables 32, performance of the LRT BAL Application is presented as compared to expected positive results (#positive results / #expected positive results) and expected negative results (negative results / number of expected negative results) for included LRT BAL microorganisms or antibiotic resistance markers, respectively.

Table 32: Performance, Contrived Study   

<table><tr><td colspan="1" rowspan="2">Assay Targets, Strains and Concentrations</td><td colspan="6" rowspan="1">Agreement with Expected Results</td></tr><tr><td colspan="1" rowspan="1"># Pos./# Exp.</td><td colspan="1" rowspan="1">%</td><td colspan="1" rowspan="1">95% CI</td><td colspan="1" rowspan="1"># Neg./# Exp.</td><td colspan="1" rowspan="1">%</td><td colspan="1" rowspan="1">95% CI</td></tr><tr><td colspan="1" rowspan="1">Chlamydia pneumoniae</td><td colspan="1" rowspan="1">57/60</td><td colspan="1" rowspan="1">95.0</td><td colspan="1" rowspan="1">86.3 - 98.3</td><td colspan="1" rowspan="1">180/180</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">97.9 - 100.0</td></tr><tr><td colspan="1" rowspan="1">6.4 x 102/2x LoD</td><td colspan="1" rowspan="1">29/30</td><td colspan="1" rowspan="1">96.7</td><td colspan="1" rowspan="1">83.3 - 99.4</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">ATCC 53592</td><td colspan="1" rowspan="1">5/6</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">ATCC VR-1310</td><td colspan="1" rowspan="1">12/12</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">ATCC VR-2282</td><td colspan="1" rowspan="1">12/12</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">1.1 x 103/3.5x LoD</td><td colspan="1" rowspan="1">15/15</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">79.6 - 100.0</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">ATCC 53592</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">ATCC VR-1310</td><td colspan="1" rowspan="1">6/6</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">ATCC VR-2282</td><td colspan="1" rowspan="1">6/6</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">1.6 x 103/5x LoD</td><td colspan="1" rowspan="1">13/15</td><td colspan="1" rowspan="1">86.7</td><td colspan="1" rowspan="1">62.1 - 96.3</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">ATCC 53592</td><td colspan="1" rowspan="1">2/3</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">ATCC VR-1310</td><td colspan="1" rowspan="1">5/6</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">ATCC VR-2282</td><td colspan="1" rowspan="1">6/6</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Citrobacter freundii</td><td colspan="1" rowspan="1">59/60</td><td colspan="1" rowspan="1">98.3</td><td colspan="1" rowspan="1">91.1 - 99.7</td><td colspan="1" rowspan="1">168/168</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">97.8 - 100.0</td></tr><tr><td colspan="1" rowspan="1">1.6 x 105/2x LoD</td><td colspan="1" rowspan="1">29/30</td><td colspan="1" rowspan="1">96.7</td><td colspan="1" rowspan="1">83.3 - 99.4</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">ATCC 43864</td><td colspan="1" rowspan="1">6/6</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">ATCC 8090</td><td colspan="1" rowspan="1">6/6</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">NCTC 8581</td><td colspan="1" rowspan="1">5/6</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">NRZ-00452</td><td colspan="1" rowspan="1">6/6</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">UCLA C1</td><td colspan="1" rowspan="1">6/6</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">2.8 x 105/3.5x LoD</td><td colspan="1" rowspan="1">15/15</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">79.6 - 100.0</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">ATCC 43864</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">ATCC 8090</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">NCTC 8581</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">NRZ-00452</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">UCLA C1</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">4.0 x 105/5x LoD</td><td colspan="1" rowspan="1">15/15</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">79.6 - 100.0</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">ATCC 43864</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">ATCC 8090</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">NCTC 8581</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">NRZ-00452</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">UCLA C1</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Enterobacter cloacae complex</td><td colspan="1" rowspan="1">58/60</td><td colspan="1" rowspan="1">96.7</td><td colspan="1" rowspan="1">88.6 - 99.1</td><td colspan="1" rowspan="1">156/156</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">97.6 - 100.0</td></tr><tr><td colspan="1" rowspan="1">4.0 x 105/2x LoD</td><td colspan="1" rowspan="1">28/30</td><td colspan="1" rowspan="1">93.3</td><td colspan="1" rowspan="1">78.7 - 98.2</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">ATCC 13047</td><td colspan="1" rowspan="1">5/6</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">ATCC 23373</td><td colspan="1" rowspan="1">6/6</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">ATCC 49141</td><td colspan="1" rowspan="1">5/6</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">ATCC BAA-2468</td><td colspan="1" rowspan="1">6/6</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">NRZ-00239</td><td colspan="1" rowspan="1">6/6</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">7.0 x 105/3.5x LoD</td><td colspan="1" rowspan="1">15/15</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">79.6 - 100.0</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">ATCC 13047</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">ATCC 23373</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1"># Pos./#Exp.</td><td colspan="1" rowspan="1">%</td><td colspan="1" rowspan="1">95% CI</td><td colspan="1" rowspan="1"># Neg./#Exp.</td><td colspan="1" rowspan="1">%</td><td colspan="1" rowspan="1">95% CI</td></tr><tr><td colspan="1" rowspan="1">ATCC 49141</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">ATCC BAA-2468</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">NRZ-00239</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">1.0 x 106/5x LoD</td><td colspan="1" rowspan="1">15/15</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">79.6 - 100.0</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">ATCC 13047</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">ATCC 23373</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">ATCC 49141</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">ATCC BAA-2468</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">NRZ-00239</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Klebsiella oxytoca</td><td colspan="1" rowspan="1">56/60</td><td colspan="1" rowspan="1">93.3</td><td colspan="1" rowspan="1">84.1 - 97.4</td><td colspan="1" rowspan="1">168/168</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">97.8 - 100.0</td></tr><tr><td colspan="1" rowspan="1">2.0 x 104/2x LoD</td><td colspan="1" rowspan="1">26/30</td><td colspan="1" rowspan="1">86.7</td><td colspan="1" rowspan="1">70.3 - 94.7</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">ATCC 13182</td><td colspan="1" rowspan="1">4/6</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">ATCC 43863</td><td colspan="1" rowspan="1">6/6</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">ATCC 49131</td><td colspan="1" rowspan="1">6/6</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">ATCC 8724</td><td colspan="1" rowspan="1">5/6</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">NCIMB 12819</td><td colspan="1" rowspan="1">5/6</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">3.5 x 104/3.5x LoD</td><td colspan="1" rowspan="1">15/15</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">79.6 - 100.0</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">ATCC 13182</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">ATCC 43863</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">ATCC 49131</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">ATCC 8724</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">NCIMB 12819</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">5.0 x 104/5x LoD</td><td colspan="1" rowspan="1">15/15</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">79.6 - 100.0</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">ATCC 13182</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">ATCC 43863</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">ATCC 49131</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">ATCC 8724</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">NCIMB 12819</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">60/60</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">94.0 - 100.0</td><td colspan="1" rowspan="1">96/96</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">96.2 - 100.0</td></tr><tr><td colspan="1" rowspan="1">8.0 x 104/2x LoD</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">88.7 - 100.0</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">ATCC 13883</td><td colspan="1" rowspan="1">6/6</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Micromyx 4653</td><td colspan="1" rowspan="1">6/6</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">NCTC 13438</td><td colspan="1" rowspan="1">6/6</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">NCTC 13439</td><td colspan="1" rowspan="1">6/6</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">NCTC 13442</td><td colspan="1" rowspan="1">6/6</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">1.4 x 105/3.5x LoD</td><td colspan="1" rowspan="1">15/15</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">79.6 - 100.0</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">ATCC 13883</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Micromyx 4653</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">NCTC 13438</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">NCTC 13439</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">NCTC 13442</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">2.0 x 105/5x LoD</td><td colspan="1" rowspan="1">15/15</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">79.6 - 100.0</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">ATCC 13883</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Micromyx 4653</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">NCTC 13438</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">NCTC 13439</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">NCTC 13442</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Klebsiella variicola</td><td colspan="1" rowspan="1">58/60</td><td colspan="1" rowspan="1">96.7</td><td colspan="1" rowspan="1">88.6 - 99.1</td><td colspan="1" rowspan="1">180/180</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">97.9 - 100.0</td></tr><tr><td colspan="1" rowspan="1">4.0 x 104/2x LoD</td><td colspan="1" rowspan="1">29/30</td><td colspan="1" rowspan="1">96.7</td><td colspan="1" rowspan="1">83.3 - 99.4</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">ATCC BAA-830</td><td colspan="1" rowspan="1">6/6</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">clinical isolate 1</td><td colspan="1" rowspan="1">6/6</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1"># Pos./# Exp.</td><td colspan="1" rowspan="1">%</td><td colspan="1" rowspan="1">95% CI</td><td colspan="1" rowspan="1"># Neg./# Exp.</td><td colspan="1" rowspan="1">%</td><td colspan="1" rowspan="1">95% CI</td></tr><tr><td colspan="1" rowspan="1">clinical isolate 2</td><td colspan="1" rowspan="1">6/6</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">clinical isolate 3</td><td colspan="1" rowspan="1">6/6</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">clinical isolate 4</td><td colspan="1" rowspan="1">5/6</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">7.0 x 104/3.5x LoD</td><td colspan="1" rowspan="1">15/15</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">79.6 - 100.0</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">ATCC BAA-830</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">clinical isolate 1</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">clinical isolate 2</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">clinical isolate 3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">clinical isolate 4</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">1.0 x 105/5x LoD</td><td colspan="1" rowspan="1">14/15</td><td colspan="1" rowspan="1">93.3</td><td colspan="1" rowspan="1">70.2 - 98.8</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">ATCC BAA-830</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">clinical isolate 1</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">clinical isolate 2</td><td colspan="1" rowspan="1">2/3</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">clinical isolate 3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">clinical isolate 4</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Legionella pneumophila</td><td colspan="1" rowspan="1">57/60</td><td colspan="1" rowspan="1">95.0</td><td colspan="1" rowspan="1">86.3 - 98.3</td><td colspan="1" rowspan="1">180/180</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">97.9 - 100.0</td></tr><tr><td colspan="1" rowspan="1">1.6 x 105/2x LoD</td><td colspan="1" rowspan="1">28/30</td><td colspan="1" rowspan="1">93.3</td><td colspan="1" rowspan="1">78.7 - 98.2</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">ATCC 33152</td><td colspan="1" rowspan="1">6/6</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">ATCC 33154</td><td colspan="1" rowspan="1">5/6</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">ATCC 33155</td><td colspan="1" rowspan="1">5/6</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">ATCC 33215</td><td colspan="1" rowspan="1">6/6</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">ATCC 35096</td><td colspan="1" rowspan="1">6/6</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">2.8 x 105/3.5x LoD</td><td colspan="1" rowspan="1">15/15</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">79.6 - 100.0</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">ATCC 33152</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">ATCC 33154</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">ATCC 33155</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">ATCC 33215</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">ATCC 35096</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">4.0 x 105/5x LoD</td><td colspan="1" rowspan="1">14/15</td><td colspan="1" rowspan="1">93.3</td><td colspan="1" rowspan="1">70.2 - 98.8</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">ATCC 33152</td><td colspan="1" rowspan="1">2/3</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">ATCC 33154</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">ATCC 33155</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">ATCC 33215</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">ATCC 35096</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Morganella morganii</td><td colspan="1" rowspan="1">52/60</td><td colspan="1" rowspan="1">86.7</td><td colspan="1" rowspan="1">75.8 - 93.1</td><td colspan="1" rowspan="1">179/179</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">97.9 - 100.0</td></tr><tr><td colspan="1" rowspan="1">4.0 x 104/2x LoD</td><td colspan="1" rowspan="1">23/30</td><td colspan="1" rowspan="1">76.7</td><td colspan="1" rowspan="1">59.1 - 88.2</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">ATCC 25829 </td><td colspan="1" rowspan="1">3/6</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">ATCC 25830</td><td colspan="1" rowspan="1">6/6</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">ATCC 49948</td><td colspan="1" rowspan="1">6/6</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">ATCC 8019</td><td colspan="1" rowspan="1">6/6</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">DSM-46262 </td><td colspan="1" rowspan="1">2/6</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">7.0 x 104/3.5x LoD</td><td colspan="1" rowspan="1">15/15</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">79.6 - 100.0</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">ATCC 25829</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">ATCC 25830</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">ATCC 49948</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">ATCC 8019</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">DSM-46262</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">1.0 x 105/5x LoD</td><td colspan="1" rowspan="1">14/15</td><td colspan="1" rowspan="1">93.3</td><td colspan="1" rowspan="1">70.2 - 98.8</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">ATCC 25829</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">ATCC 25830</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">ATCC 49948</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1"># Pos./#Exp.</td><td colspan="1" rowspan="1">%</td><td colspan="1" rowspan="1">95% CI</td><td colspan="1" rowspan="1"># Neg./#Exp.</td><td colspan="1" rowspan="1">%</td><td colspan="1" rowspan="1">95% CI</td></tr><tr><td colspan="1" rowspan="1">ATCC 8019</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">DSM-46262</td><td colspan="1" rowspan="1">2/3</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Mycoplasma pneumoniae</td><td colspan="1" rowspan="1">52/60</td><td colspan="1" rowspan="1">86.7</td><td colspan="1" rowspan="1">75.8 - 93.1</td><td colspan="1" rowspan="1">180/180</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">97.9 - 100.0</td></tr><tr><td colspan="1" rowspan="1">3.2 x 103/2x LoD</td><td colspan="1" rowspan="1">23/30</td><td colspan="1" rowspan="1">76.7</td><td colspan="1" rowspan="1">59.1 - 88.2</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">ATCC 15293</td><td colspan="1" rowspan="1">6/6</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">ATCC 29085 b</td><td colspan="1" rowspan="1">8/12</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">ATCC 49894 b</td><td colspan="1" rowspan="1">9/12</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">EY</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">5.6 x 103/3.5x LoD</td><td colspan="1" rowspan="1">14/15</td><td colspan="1" rowspan="1">93.3</td><td colspan="1" rowspan="1">70.2 - 98.8</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">ATCC 15293</td><td colspan="1" rowspan="1">2/3</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">ATCC 29085</td><td colspan="1" rowspan="1">6/6</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">ATCC 49894</td><td colspan="1" rowspan="1">6/6</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">8.0 x 103/5x LoD</td><td colspan="1" rowspan="1">15/15</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">79.6 - 100.0</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">ATCC 15293</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">ATCC 29085</td><td colspan="1" rowspan="1">6/6</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">ATCC 49894</td><td colspan="1" rowspan="1">6/6</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Pneumocystis jirovecii</td><td colspan="1" rowspan="1">59/60</td><td colspan="1" rowspan="1">98.3</td><td colspan="1" rowspan="1">91.1 - 99.7</td><td colspan="1" rowspan="1">180/180</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">97.9 - 100.0</td></tr><tr><td colspan="1" rowspan="1">1.0 x 106/2x LoD</td><td colspan="1" rowspan="1">59/60</td><td colspan="1" rowspan="1">98.3</td><td colspan="1" rowspan="1">91.1 - 99.7</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">positive clinical specimen</td><td colspan="1" rowspan="1">59/60</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Proteus spp.</td><td colspan="1" rowspan="1">60/60</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">94.0 - 100.0</td><td colspan="1" rowspan="1">180/180</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">97.9 - 100.0</td></tr><tr><td colspan="1" rowspan="1">1.0 x 104/2x LoD</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">88.7 - 100.0</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">ATCC 12453</td><td colspan="1" rowspan="1">6/6</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">ATCC 14153</td><td colspan="1" rowspan="1">6/6</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">ATCC 25933</td><td colspan="1" rowspan="1">6/6</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">ATCC 29906</td><td colspan="1" rowspan="1">6/6</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">ATCC 6380</td><td colspan="1" rowspan="1">6/6</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">1.8 x 104/3.5x LoD</td><td colspan="1" rowspan="1">15/15</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">79.6 - 100.0</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">ATCC 12453</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">ATCC 14153</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">ATCC 25933</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">ATCC 29906</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">ATCC 6380</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">2.5 x 104/5x LoD</td><td colspan="1" rowspan="1">15/15</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">79.6 - 100.0</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">ATCC 12453</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">ATCC 14153</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">ATCC 25933</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">ATCC 29906</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">ATCC 6380</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Serratia marcescens</td><td colspan="1" rowspan="1">59/60</td><td colspan="1" rowspan="1">98.3</td><td colspan="1" rowspan="1">91.1 - 99.7</td><td colspan="1" rowspan="1">180/180</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">97.9 - 100.0</td></tr><tr><td colspan="1" rowspan="1">8.0 x 104/2x LoD</td><td colspan="1" rowspan="1">29/30</td><td colspan="1" rowspan="1">96.7</td><td colspan="1" rowspan="1">83.3 - 99.4</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">ATCC 13880</td><td colspan="1" rowspan="1">6/6</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">ATCC 14756</td><td colspan="1" rowspan="1">6/6</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">ATCC 15365</td><td colspan="1" rowspan="1">6/6</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">ATCC 27117</td><td colspan="1" rowspan="1">6/6</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">DSM-17174</td><td colspan="1" rowspan="1">5/6</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">1.4 x 105/3.5x LoD</td><td colspan="1" rowspan="1">15/15</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">79.6 - 100.0</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">ATCC 13880</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">ATCC 14756</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">ATCC 15365</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">ATCC 27117</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">DSM-17174</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1"># Pos./# Exp.</td><td colspan="1" rowspan="1">%</td><td colspan="1" rowspan="1">95% CI</td><td colspan="1" rowspan="1"># Neg./# Exp.</td><td colspan="1" rowspan="1">%</td><td colspan="1" rowspan="1">95% CI</td></tr><tr><td colspan="1" rowspan="1">2.0 x 105/5x LoD</td><td colspan="1" rowspan="1">15/15</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">79.6 - 100.0</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">ATCC 13880</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">ATCC 14756</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">ATCC 15365</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">ATCC 27117</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">DSM-17174</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr></table>

a Morganella morganii strains ATCC 25829 and DSM-46262 strains were retested with pooled negative BAL matrix at 2x LoD both showing 10/10 results.

b Mycoplasma pneumoniae strains ATCC 29085 and ATCC 49894 were retested with pooled negative BAL matrix at 2x LoD showing 19/20 or 18/20 positive results, respectively. For three of four unexpected negative cases of ATCC 29085, unique matrices used for specimen preparation demonstrated reduced signals or false negative results for other (co-spiked) analytes; therefore, three false negative results for $M .$ . pneumoniae may be matrix-related.

# 4. Clinical Study, Performance for Resistance Marker Targets

The LRT BAL Application was evaluated in the prospective clinical study $\mathrm { ( N } \mathrm { = } 1 , 0 1 6$ specimens) for reported antibiotic resistance marker targets. For each resistance marker, LRT BAL results were compared to one PCR assay followed by bi-directional sequencing with testing performed directly with BAL specimens. Since resistance marker results are only reported by the LRT BAL Application when at least one applicable target microorganism is simultaneously detected (i.e., otherwise, detected resistance markers are masked from reporting). Table 33 includes performance for resistance markers with masking applied, i.e., includes specimens with applicable microorganism targets reported by LRT BAL.

Table 33: Clinical Performance, Resistance Marker, Prospective Study   

<table><tr><td rowspan=1 colspan=1>Resistance Marker(Number of specimens with applicableMicroorganism detected by LRT BAL)</td><td rowspan=1 colspan=1>TP</td><td rowspan=1 colspan=1>FN</td><td rowspan=1 colspan=1>FP c, d</td><td rowspan=1 colspan=1>TN</td><td rowspan=1 colspan=1>PPA [%] (95% CI)</td><td rowspan=1 colspan=1>NPA [%] (95% CI)</td></tr><tr><td rowspan=1 colspan=1>ctx-M (N = 208) a</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>1b</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>197</td><td rowspan=1 colspan=1>88.9 (56.5 - 98.0)</td><td rowspan=1 colspan=1>99.5 (97.2 - 99.9)</td></tr><tr><td rowspan=1 colspan=1>kpc (N = 208)</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>204</td><td rowspan=1 colspan=1>100.0 (43.9 - 100.0)</td><td rowspan=1 colspan=1>99.5 (97.3 - 99.9)</td></tr><tr><td rowspan=1 colspan=1>mecA (N = 104)</td><td rowspan=1 colspan=1>22</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>25</td><td rowspan=1 colspan=1>57</td><td rowspan=1 colspan=1>100.0 (85.1 - 100.0)</td><td rowspan=1 colspan=1>69.5 (58.9 - 78.4)</td></tr><tr><td rowspan=1 colspan=1>ndm (N = 208)</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>207</td><td rowspan=1 colspan=1>100.0 (20.7 - 100.0)</td><td rowspan=1 colspan=1>100.0 (98.2 - 100.0)</td></tr><tr><td rowspan=1 colspan=1>oxa-23 (N = 21)</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>18</td><td rowspan=1 colspan=1>100.0 (43.9 - 100.0)</td><td rowspan=1 colspan=1>100.0 (82.4 - 100.0)</td></tr><tr><td rowspan=1 colspan=1>oxa-24 (N = 21) a</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>100.0 (43.9 - 100.0)</td><td rowspan=1 colspan=1>94.1 (73.0 - 99.0)</td></tr><tr><td rowspan=1 colspan=1>oxa-48 (N = 112)</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>111</td><td rowspan=1 colspan=1>100.0 (20.7 - 100.0)</td><td rowspan=1 colspan=1>100.0 (96.7 - 100.0)</td></tr><tr><td rowspan=1 colspan=1>oxa-58 (N = 21)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>100.0 (84.5 - 100.0)</td></tr><tr><td rowspan=1 colspan=1>tem (N = 56)</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>7f</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>100.0 (70.1 - 100.0)</td><td rowspan=1 colspan=1>85.1 (72.3 - 92.6)</td></tr><tr><td rowspan=1 colspan=1>vim (N = 208)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>207</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>99.5 (97.3 - 99.9)</td></tr></table>

a Observed invalid LRT BAL analyte results: ctx-M (1) and oxa-24 (1). b One false negative result belongs to the $c t x$ -M9 subgroup that is not covered by the ctx-M assay of the LRT BAL Application (targeted against ctx-M1 subgroup).

c Note that molecular reference assays use cutoffs to achieve analytical sensitivities in a similar range to LRT BAL. Negative results by molecular reference assays do not preclude the presence of the antibiotic resistance marker in the specimen at a low concentration, nor its possible clinical relevance.

d Specimens with false positive LRT BAL results were analyzed with additional molecular assays (PCR/bi-directional sequencing) using specimen DNA extracts for presence or absence of antibiotic resistance markers. Targeted resistance markers were confirmed in 0 of 1 case for ctx-M, 1 of 1 case of kpc, 10 of 25 cases for mecA, 1 of 1 case for oxa-24, 6 of 7 cases for tem and 1 of 1 case for vim. Cross-reactivity to other off-panel analytes has not been observed.

Several targeted resistance markers (ctx-M, kpc, ndm, oxa-48) are reported for more than one corresponding host microorganism for the LRT BAL Application. Performance for these markers is presented in Table 34 stratified by target microoorganism (C. freundii, $E$ . cloacae complex, E. coli, K. oxytoca, K. pneumoniae, K. variicola, M. morganii, Proteus spp., S. marcescens, Acinetobacter spp., $P .$ aeruginosa). The comparator used in the analysis is one PCR/sequencing assay for each resistance marker target. Note that the performance estimates are not based on definitive linkage information of detected antibiotic resistance markers to culture isolates or specific microorganisms.

Table 34: Performance, Prospective Study, Resistance Markers, Compared to PCR/sequencing, Stratified by Applicable Microorganism Targets Reported by LRT BAL   

<table><tr><td rowspan=1 colspan=1>Citrobacter freundii(N = 4)</td><td rowspan=1 colspan=1>TP</td><td rowspan=1 colspan=1>FN</td><td rowspan=1 colspan=1>FP</td><td rowspan=1 colspan=1>TN</td><td rowspan=1 colspan=1>PPA [%]</td><td rowspan=1 colspan=1>NPA [%]</td></tr><tr><td rowspan=1 colspan=1>ctx-M</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>100.0</td></tr><tr><td rowspan=1 colspan=1>kpc</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>100.0</td></tr><tr><td rowspan=1 colspan=1>ndm</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>100.0</td></tr><tr><td rowspan=1 colspan=1>oxa-48</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>100.0</td></tr><tr><td rowspan=1 colspan=1>vim</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>100.0</td></tr></table>

<table><tr><td rowspan=1 colspan=1>Enterobacter cloacaecomplex (N = 20)</td><td rowspan=1 colspan=1>TP</td><td rowspan=1 colspan=1>FN</td><td rowspan=1 colspan=1>FP</td><td rowspan=1 colspan=1>TN</td><td rowspan=1 colspan=1>PPA [%]</td><td rowspan=1 colspan=1>NPA [%]</td></tr><tr><td rowspan=1 colspan=1>ctx-M</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>100.0</td></tr><tr><td rowspan=1 colspan=1>kpc</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>19</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>100.0</td></tr><tr><td rowspan=1 colspan=1>ndm</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>100.0</td></tr><tr><td rowspan=1 colspan=1>oxa-48</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>100.0</td></tr><tr><td rowspan=1 colspan=1>vim</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>100.0</td></tr></table>

<table><tr><td rowspan=1 colspan=1>Escherichia coli(N = 47)</td><td rowspan=1 colspan=1>TP</td><td rowspan=1 colspan=1>FN</td><td rowspan=1 colspan=1>FP</td><td rowspan=1 colspan=1>TN</td><td rowspan=1 colspan=1>PPA [%]</td><td rowspan=1 colspan=1>NPA [%]</td></tr><tr><td rowspan=1 colspan=1>ctx-M</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>38</td><td rowspan=1 colspan=1>87.5</td><td rowspan=1 colspan=1>97.4</td></tr><tr><td rowspan=1 colspan=1>kpc</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>46</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>100.0</td></tr><tr><td rowspan=1 colspan=1>ndm</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>46</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>100.0</td></tr><tr><td rowspan=1 colspan=1>oxa-48</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>46</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>100.0</td></tr><tr><td rowspan=1 colspan=1>vim</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>47</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>100.0</td></tr></table>

<table><tr><td rowspan=1 colspan=1>Klebsiella oxytoca(N = 14)</td><td rowspan=1 colspan=1>TP</td><td rowspan=1 colspan=1>FN</td><td rowspan=1 colspan=1>FP</td><td rowspan=1 colspan=1>TN</td><td rowspan=1 colspan=1>PPA [%]</td><td rowspan=1 colspan=1>NPA [%]</td></tr><tr><td rowspan=1 colspan=1>ctx-M</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>100.0</td></tr><tr><td rowspan=1 colspan=1>kpc</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>100.0</td></tr><tr><td rowspan=1 colspan=1>ndm</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>100.0</td></tr><tr><td rowspan=1 colspan=1>oxa-48</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>100.0</td></tr><tr><td rowspan=1 colspan=1>vim</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>100.0</td></tr></table>

<table><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae(N = 30)</td><td colspan="1" rowspan="1">TP</td><td colspan="1" rowspan="1">FN</td><td colspan="1" rowspan="1">FP</td><td colspan="1" rowspan="1">TN</td><td colspan="1" rowspan="1">PPA [%]</td><td colspan="1" rowspan="1">NPA [%]</td></tr><tr><td colspan="1" rowspan="1">ctx-M</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">26</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">100.0</td></tr><tr><td colspan="1" rowspan="1">kpc</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">26</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">96.3</td></tr><tr><td colspan="1" rowspan="1">ndm</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">29</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">100.0</td></tr><tr><td colspan="1" rowspan="1">oxa-48</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">29</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">100.0</td></tr><tr><td colspan="1" rowspan="1">vim</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">30</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">100.0</td></tr></table>

<table><tr><td rowspan=1 colspan=1>Klebsiella variicola(N = 2)</td><td rowspan=1 colspan=1>TP</td><td rowspan=1 colspan=1>FN</td><td rowspan=1 colspan=1>FP</td><td rowspan=1 colspan=1>TN</td><td rowspan=1 colspan=1>PPA [%]</td><td rowspan=1 colspan=1>NPA [%]</td></tr><tr><td rowspan=1 colspan=1>ctx-M</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>100.0</td></tr><tr><td rowspan=1 colspan=1>kpc</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>100.0</td></tr><tr><td rowspan=1 colspan=1>ndm</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>100.0</td></tr><tr><td rowspan=1 colspan=1>oxa-48</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>100.0</td></tr><tr><td rowspan=1 colspan=1>vim</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>100.0</td></tr></table>

<table><tr><td rowspan=1 colspan=1>Morganella morganii(N = 3)</td><td rowspan=1 colspan=1>TP</td><td rowspan=1 colspan=1>FN</td><td rowspan=1 colspan=1>FP</td><td rowspan=1 colspan=1>TN</td><td rowspan=1 colspan=1>PPA [%]</td><td rowspan=1 colspan=1>NPA [%]</td></tr><tr><td rowspan=1 colspan=1>ctx-M</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>100.0</td></tr><tr><td rowspan=1 colspan=1>kpc</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>100.0</td></tr><tr><td rowspan=1 colspan=1>ndm</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>100.0</td></tr><tr><td rowspan=1 colspan=1>oxa-48</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>100.0</td></tr><tr><td rowspan=1 colspan=1>vim</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>100.0</td></tr></table>

<table><tr><td rowspan=1 colspan=1>Proteus spp.(N = 10)</td><td rowspan=1 colspan=1>TP</td><td rowspan=1 colspan=1>FN</td><td rowspan=1 colspan=1>FP</td><td rowspan=1 colspan=1>TN</td><td rowspan=1 colspan=1>PPA [%]</td><td rowspan=1 colspan=1>NPA [%]</td></tr><tr><td rowspan=1 colspan=1>ctx-M</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>100.0</td></tr><tr><td rowspan=1 colspan=1>kpc</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>100.0</td></tr><tr><td rowspan=1 colspan=1>ndm</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>100.0</td></tr><tr><td rowspan=1 colspan=1>oxa-48</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>100.0</td></tr><tr><td rowspan=1 colspan=1>vim</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>90.0</td></tr></table>

<table><tr><td rowspan=1 colspan=1>Serratia marcescens(N = 17)</td><td rowspan=1 colspan=1>TP</td><td rowspan=1 colspan=1>FN</td><td rowspan=1 colspan=1>FP</td><td rowspan=1 colspan=1>TN</td><td rowspan=1 colspan=1>PPA [%]</td><td rowspan=1 colspan=1>NPA [%]</td></tr><tr><td rowspan=1 colspan=1>ctx-M</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>100.0</td></tr><tr><td rowspan=1 colspan=1>kpc</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>100.0</td></tr><tr><td rowspan=1 colspan=1>ndm</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>100.0</td></tr><tr><td rowspan=1 colspan=1>oxa-48</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>100.0</td></tr><tr><td rowspan=1 colspan=1>vim</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>100.0</td></tr></table>

<table><tr><td rowspan=1 colspan=1>Acinetobacter spp.(N = 21)</td><td rowspan=1 colspan=1>TP</td><td rowspan=1 colspan=1>FN</td><td rowspan=1 colspan=1>FP</td><td rowspan=1 colspan=1>TN</td><td rowspan=1 colspan=1>PPA [%]</td><td rowspan=1 colspan=1>NPA [%]</td></tr><tr><td rowspan=1 colspan=1>ctx-M</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>19</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>100.0</td></tr><tr><td rowspan=1 colspan=1>kpc</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>19</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>100.0</td></tr><tr><td rowspan=1 colspan=1>ndm</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>100.0</td></tr><tr><td rowspan=1 colspan=1>vim</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>100.0</td></tr></table>

<table><tr><td rowspan=1 colspan=1>Pseudomonasaeruginosa (N = 112)</td><td rowspan=1 colspan=1>TP</td><td rowspan=1 colspan=1>FN</td><td rowspan=1 colspan=1>FP</td><td rowspan=1 colspan=1>TN</td><td rowspan=1 colspan=1>PPA [%]</td><td rowspan=1 colspan=1>NPA [%]</td></tr><tr><td rowspan=1 colspan=1>ctx-M</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>104</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>99.0</td></tr><tr><td rowspan=1 colspan=1>kpc</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>110</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>99.1</td></tr><tr><td rowspan=1 colspan=1>ndm</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>111</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>100.0</td></tr><tr><td rowspan=1 colspan=1>vim</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>111</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>99.1</td></tr></table>

To further assess prospective study performance for resistance markers reported by LRT BAL, a comparison to the following reference/comparator methods was performed: (A) SoC culture result for microorganisms, combined with sequencing results / marker screenings of provided strain isolates, and $\mathbf { ( B ) }$ composite comparator result for microorganisms and molecular comparator results (PCR/sequencing) for resistance markers. Note that an isolate was not provided for all positive SoC cultures, thereby limiting the dataset for comparison (A). Comparison (B) evaluates the entire specimen cohort, including all positive LRT BAL results that were negative by SoC culture.

For both comparisons, the following analyses were performed for ctx-M, kpc, ndm, and vim (for targeted Enterobacteriaceae, Acinetobacter spp. and/or $P$ . aeruginosa as possible host microorganisms), for oxa-48 (for targeted Enterobacteriaceae), for tem (for $H .$ influenzae), and for mecA (for S. aureus) in Tables 35-43:

Agreement of one (or more) detected host microorganisms in combination with an antibiotic resistance marker $( \mathbf { O r g } { + } / \mathbf { R e s } { + } )$ between LRT BAL and reference methods (A) or (B). Agreement of one (or more) detected host microorganisms without reported antibiotic resistance marker $( \bf { O r g } + / \bf { R e s } - )$ between LRT BAL and reference methods (A) or $\mathbf { ( B ) }$ .   
Agreement of a negative result for one (or more) host microorganisms (Org-) between LRT BAL and reference methods (A) and (B).

As presence of mecA is expected to directly correlate with cefoxitin/oxacillin resistance in S. aureus strains (MRSA, methicillin-resistant S. aureus), while the absence of mecA is expected to correlate with susceptible $S .$ aureus strains (MSSA, methicillin-sensitive $S _ { \cdot }$ . aureus), mecA was furthermore compared to the MRSA/MSSA status of culture isolates as reported by SoC culture (C) in Table 43. The following evaluations were performed:

Agreement of a detected S. aureus in combination with a detected mecA as reported by LRT BAL $( \mathbf { O r g } { + } / \mathbf { R e s } { + } )$ to $S .$ aureus reported with an MRSA phenotype by SoC culture $( \mathbf { O r g } { + } / \mathbf { M R S A } )$ . Agreement of a detected $S .$ aureus together with a negative result for mecA $\mathbf { ( O r g + / R e s ) }$ as reported by LRT BAL to S. aureus reported with an MSSA phenotype by SoC culture $( \mathbf { O r g } { + } / \mathbf { M S S A } )$ .   
• Agreement of a negative result for S. aureus between LRT BAL and SoC culture.

Evaluation of oxa-58 is not provided as positive results were not observed from the LRT BAL Application, nor by comparator methods.

Table 35: Clinical performance, oxa-23/Acinetobacter spp.: Comparator methods A (culture for microorganism identification; isolate sequencing for oxa-23) and B (composite comparator for microorganism identification; PCR/sequencing for oxa-23)   

<table><tr><td rowspan=10 colspan=1>A</td><td rowspan=2 colspan=2>Acinetobacter spp.0xa-23</td><td rowspan=1 colspan=5>Culture Result for Acinetobacter spp. +Isolate Sequencing / Marker Screening for oxa-23</td></tr><tr><td rowspan=1 colspan=1>Org+/Res+</td><td rowspan=1 colspan=1>Org+/Res-</td><td rowspan=1 colspan=1>Org+/No Isolate</td><td rowspan=1 colspan=1>Org-</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=4 colspan=1>LRTBALResult</td><td rowspan=1 colspan=1>Org+/Res+</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>3</td></tr><tr><td rowspan=1 colspan=1>Org+/Res-</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>18</td></tr><tr><td rowspan=1 colspan=1>Org-</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>993</td><td rowspan=1 colspan=1>994</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>1,004</td><td rowspan=1 colspan=1>1,015 a</td></tr><tr><td rowspan=1 colspan=3>Acinetobacter spp. / oxa-23</td><td rowspan=1 colspan=1>rate [%]</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>positivity</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=3>Agreement (Org+/Res+)</td><td rowspan=1 colspan=2>100.0</td><td rowspan=1 colspan=1>1/1</td><td rowspan=1 colspan=1>20.7 - 100.0</td></tr><tr><td rowspan=1 colspan=3>Agreement (Org+/Res-)</td><td rowspan=1 colspan=2>100.0</td><td rowspan=1 colspan=1>7/7</td><td rowspan=1 colspan=1>64.6 - 100.0</td></tr><tr><td rowspan=1 colspan=3>Agreement (Org-)</td><td rowspan=1 colspan=2>98.9</td><td rowspan=1 colspan=1>993/1,004</td><td rowspan=1 colspan=1>98.0 - 99.4</td></tr></table>

<table><tr><td rowspan=10 colspan=1>B</td><td rowspan=2 colspan=2>Acinetobacter spp.oxa-23</td><td rowspan=1 colspan=4>Composite Comparator Result for Acinetobacter spp. +Mol. Reference (PCR/Seq.) for oxa-23</td></tr><tr><td rowspan=1 colspan=1>Org+/Res+</td><td rowspan=1 colspan=1>Org+/Res-</td><td rowspan=1 colspan=1>Org-</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=4 colspan=1>LRTBALResult</td><td rowspan=1 colspan=1>Org+/Res+</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3</td></tr><tr><td rowspan=1 colspan=1>Org+/Res-</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>18</td></tr><tr><td rowspan=1 colspan=1>Org-</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>992</td><td rowspan=1 colspan=1>994</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>997</td><td rowspan=1 colspan=1>1,015a</td></tr><tr><td rowspan=1 colspan=3>Acinetobacter spp. / oxa-23</td><td rowspan=1 colspan=1>rate [%]</td><td rowspan=1 colspan=1>positivity</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=3>Agreement (Org+/Res+)</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>43.9 - 100.0</td></tr><tr><td rowspan=1 colspan=3>Agreement (Org+/Res-)</td><td rowspan=1 colspan=1>86.7</td><td rowspan=1 colspan=1>13/15</td><td rowspan=1 colspan=1>62.1 - 96.3</td></tr><tr><td rowspan=1 colspan=3>Agreement (Org-)</td><td rowspan=1 colspan=1>99.5</td><td rowspan=1 colspan=1>992/997</td><td rowspan=1 colspan=1>98.8 - 99.8</td></tr></table>

a One invalid LRT BAL result for Acinetobacter spp.

Table 36: Clinical Performance, oxa-24/Acinetobacter spp.: Comparator methods A (culture for microorganism identification; isolate sequencing for oxa-24) and B (composite comparator for microorganism identification; PCR/sequencing for oxa-24)   

<table><tr><td rowspan=10 colspan=1>A</td><td rowspan=2 colspan=2>Acinetobacter spp.oxa-24</td><td rowspan=1 colspan=5>Culture Result for Acinetobacter spp. +Isolate Sequencing / Marker Screening for oxa-24</td></tr><tr><td rowspan=1 colspan=1>Org+/Res+</td><td rowspan=1 colspan=1>Org+/Res-</td><td rowspan=1 colspan=1>Org+/No Isolate</td><td rowspan=1 colspan=1>Org-</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=4 colspan=1>LRTBALResult</td><td rowspan=1 colspan=1>Org+/Res+</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>4</td></tr><tr><td rowspan=1 colspan=1>Org+/Res-</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>16</td></tr><tr><td rowspan=1 colspan=1>Org-</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>993</td><td rowspan=1 colspan=1>994</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>1,004</td><td rowspan=1 colspan=1>1,014 a</td></tr><tr><td rowspan=1 colspan=3>Acinetobacter spp. / oxa-24</td><td rowspan=1 colspan=1>rate [%]</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>positivity</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=3>Agreement (Org+/Res+)</td><td rowspan=1 colspan=2>100.0</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>43.9 - 100.0</td></tr><tr><td rowspan=1 colspan=3>Agreement (Org+/Res-)</td><td rowspan=1 colspan=2>100.0</td><td rowspan=1 colspan=1>4/4</td><td rowspan=1 colspan=1>51.0 - 100.0</td></tr><tr><td rowspan=1 colspan=3>Agreement (Org-)</td><td rowspan=1 colspan=2>98.9</td><td rowspan=1 colspan=1>993/1,004</td><td rowspan=1 colspan=1>98.0 - 99.4</td></tr></table>

<table><tr><td rowspan=10 colspan=1>B</td><td rowspan=2 colspan=2>Acinetobacter spp.oxa-24</td><td rowspan=1 colspan=4>Composite Comparator Result for Acinetobacter spp. +Mol. Reference (PCR/Seq.) for oxa-24</td></tr><tr><td rowspan=1 colspan=1>Org+/Res+</td><td rowspan=1 colspan=1>Org+/Res-</td><td rowspan=1 colspan=1>Org-</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=4 colspan=1>LRTBALResult</td><td rowspan=1 colspan=1>Org+/Res+</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>4</td></tr><tr><td rowspan=1 colspan=1>Org+/Res-</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>16</td></tr><tr><td rowspan=1 colspan=1>Org-</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>992</td><td rowspan=1 colspan=1>994</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>997</td><td rowspan=1 colspan=1>1,014a</td></tr><tr><td rowspan=1 colspan=3>Acinetobacter spp. / oxa-24</td><td rowspan=1 colspan=1>rate [%]</td><td rowspan=1 colspan=1>positivity</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=3>Agreement (Org+/Res+)</td><td rowspan=1 colspan=1>75.0</td><td rowspan=1 colspan=1>3/4</td><td rowspan=1 colspan=1>30.1 - 95.4</td></tr><tr><td rowspan=1 colspan=3>Agreement (Org+/Res-)</td><td rowspan=1 colspan=1>92.3</td><td rowspan=1 colspan=1>12/13</td><td rowspan=1 colspan=1>66.7 - 98.6</td></tr><tr><td rowspan=1 colspan=3>Agreement (Org-)</td><td rowspan=1 colspan=1>99.5</td><td rowspan=1 colspan=1>992/997</td><td rowspan=1 colspan=1>98.8 - 99.8</td></tr></table>

a One invalid LRT BAL result for Acinetobacter spp. and one for oxa-24.

Table 37: Clinical Performance, ctx-M/Applicable gram-negative targets: Comparator methods A (culture for microorganism identification; isolate sequencing for $^ { c t x - \mathbf { M } } .$ ) and B (composite comparator for microorganism identification; PCR/sequencing for ctx-M)   

<table><tr><td rowspan=10 colspan=1>A</td><td rowspan=2 colspan=2>Combined Hostsctx-M</td><td rowspan=1 colspan=5>Culture Result for Host Microorganism(s) +Isolate Sequencing / Marker Screening for ctx-M</td></tr><tr><td rowspan=1 colspan=1>Org+/Res+</td><td rowspan=1 colspan=1>Org+/Res-</td><td rowspan=1 colspan=1>Org+/No Isolate</td><td rowspan=1 colspan=1>Org-</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=4 colspan=1>LRTBALResult</td><td rowspan=1 colspan=1>Org+/Res+</td><td rowspan=1 colspan=1>3d</td><td rowspan=1 colspan=1>4 b, e</td><td rowspan=1 colspan=1>1 b, f</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>9</td></tr><tr><td rowspan=1 colspan=1>Org+/Res-</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>92</td><td rowspan=1 colspan=1>32</td><td rowspan=1 colspan=1>74</td><td rowspan=1 colspan=1>198</td></tr><tr><td rowspan=1 colspan=1>Org-</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>794</td><td rowspan=1 colspan=1>805</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>3 a</td><td rowspan=1 colspan=1>99</td><td rowspan=1 colspan=1>41</td><td rowspan=1 colspan=1>869</td><td rowspan=1 colspan=1>1,012 c</td></tr><tr><td rowspan=1 colspan=3>Combined Hosts / ctx-M</td><td rowspan=1 colspan=1>rate [%]</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>positivity</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=3>Agreement (Org+/Res+)</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>43.9 - 100.0</td></tr><tr><td rowspan=1 colspan=3>Agreement (Org+/Res-)</td><td rowspan=1 colspan=1>92.9</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>92/99</td><td rowspan=1 colspan=1>86.1 - 96.5</td></tr><tr><td rowspan=1 colspan=3>Agreement (Org-)</td><td rowspan=1 colspan=2>91.4</td><td rowspan=1 colspan=1>794/869</td><td rowspan=1 colspan=1>89.3 - 93.1</td></tr></table>

<table><tr><td rowspan=10 colspan=1>B</td><td rowspan=2 colspan=2>Combined Hostsctx-M</td><td rowspan=1 colspan=4>Composite Comparator Result for Host Microorganism(s) +Mol. Reference (PCR/Seq.) for ctx-M</td></tr><tr><td rowspan=1 colspan=1>Org+/Res+</td><td rowspan=1 colspan=1>Org+/Res-</td><td rowspan=1 colspan=1>Org-</td><td rowspan=1 colspan=1>total</td></tr><tr><td rowspan=4 colspan=1>LRTBALResult</td><td rowspan=1 colspan=1>Org+/Res+</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>9</td></tr><tr><td rowspan=1 colspan=1>Org+/Res-</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>161</td><td rowspan=1 colspan=1>36</td><td rowspan=1 colspan=1>198</td></tr><tr><td rowspan=1 colspan=1>Org-</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>785</td><td rowspan=1 colspan=1>805</td></tr><tr><td rowspan=1 colspan=1>total</td><td rowspan=1 colspan=1>9 a</td><td rowspan=1 colspan=1>182</td><td rowspan=1 colspan=1>821</td><td rowspan=1 colspan=1>1,012c</td></tr><tr><td rowspan=1 colspan=3>Combined Hosts / ctx-M</td><td rowspan=1 colspan=1>rate [%]</td><td rowspan=1 colspan=1>positivity</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=3>Agreement (Org+/Res+)</td><td rowspan=1 colspan=1>88.9</td><td rowspan=1 colspan=1>8/9</td><td rowspan=1 colspan=1>56.5 - 98.0</td></tr><tr><td rowspan=1 colspan=3>Agreement (Org+/Res-)</td><td rowspan=1 colspan=1>88.5</td><td rowspan=1 colspan=1>161/182</td><td rowspan=1 colspan=1>83.0 - 92.3</td></tr><tr><td rowspan=1 colspan=3>Agreement (Org-)</td><td rowspan=1 colspan=1>95.6</td><td rowspan=1 colspan=1>785/821</td><td rowspan=1 colspan=1>94.0 - 96.8</td></tr></table>

a Specimens with multiple detected possible host microorganisms by culture: 1 of 3, by the composite comparator: 6 of 9. b For one $\mathrm { \partial ^ { \circ } { O r g ^ { + } } / R e s ^ { - } }$ and for one $\mathrm { \cdot } \mathrm { O r g } { + } / \mathrm { N } _ { 0 }$ Isolate’ case, presence of the $c t x$ -M gene was confirmed for an off-panel isolate of Providencia stuartii. c Four invalid LRT BAL ctx-M results. $\mathbf { d } _ { \mathsf { \tiny ~ S o C } }$ reported the following possible corresponding host microorganisms for three $\mathrm { O r g ^ { + / R e s ^ { + } } }$ cases: E. coli, K. pneumoniae, $K .$ pneumoniae / P. aeruginosa. e SoC reported the following possible corresponding host microorganisms for four Org+/Res- cases: E. coli, P. aeruginosa (2x), $K .$ . pneumoniae / P. aeruginosa / S. marcescens. f SoC reported the following possible corresponding host microorganisms for one Org+/No Isolate case: K. pneumoniae.

Table 38: Clinical Performance, kpc/Applicable gram-negative targets: Comparator methods A (culture for microorganism identification; isolate sequencing for kpc) and B (composite comparator for microorganism identification; PCR/sequencing for kpc)   

<table><tr><td rowspan=10 colspan=1>A</td><td rowspan=2 colspan=2>Combined Hostskpc</td><td rowspan=1 colspan=5>Culture Result for Host Microorganism(s) +Isolate Sequencing / Marker Screening for kpc</td></tr><tr><td rowspan=1 colspan=1>Org+/Res+</td><td rowspan=1 colspan=1>Org+/Res-</td><td rowspan=1 colspan=1>Org+/No Isolate</td><td rowspan=1 colspan=1>Org-</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=4 colspan=1>LRTBALResult</td><td rowspan=1 colspan=1>Org+/Res+</td><td rowspan=1 colspan=1>1b</td><td rowspan=1 colspan=1>2c</td><td rowspan=1 colspan=1>1 d</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>4</td></tr><tr><td rowspan=1 colspan=1>Org+/Res-</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>96</td><td rowspan=1 colspan=1>32</td><td rowspan=1 colspan=1>76</td><td rowspan=1 colspan=1>204</td></tr><tr><td rowspan=1 colspan=1>Org-</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>797</td><td rowspan=1 colspan=1>808</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>1 a</td><td rowspan=1 colspan=1>101</td><td rowspan=1 colspan=1>41</td><td rowspan=1 colspan=1>873</td><td rowspan=1 colspan=1>1,016</td></tr><tr><td rowspan=1 colspan=3>Combined Hosts / kpc</td><td rowspan=1 colspan=1>rate [%]</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>positivity</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=3>Agreement (Org+/Res+)</td><td rowspan=1 colspan=2>100.0</td><td rowspan=1 colspan=1>1/1</td><td rowspan=1 colspan=1>20.7 - 100.0</td></tr><tr><td rowspan=1 colspan=3>Agreement (Org+/Res-)</td><td rowspan=1 colspan=2>95.0</td><td rowspan=1 colspan=1>96/101</td><td rowspan=1 colspan=1>88.9 - 97.9</td></tr><tr><td rowspan=1 colspan=3>Agreement (Org-)</td><td rowspan=1 colspan=2>91.3</td><td rowspan=1 colspan=1>797/873</td><td rowspan=1 colspan=1>89.2 - 93.0</td></tr></table>

<table><tr><td rowspan=10 colspan=1>B</td><td rowspan=2 colspan=2>Combined Hostskpc</td><td rowspan=1 colspan=4>Composite Comparator Result for Host Microorganism(s) +Mol. Reference (PCR/Seq.) for kpc</td></tr><tr><td rowspan=1 colspan=1>Org+/Res+</td><td rowspan=1 colspan=1>Org+/Res-</td><td rowspan=1 colspan=1>Org-</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=4 colspan=1>LRTBALResult</td><td rowspan=1 colspan=1>Org+/Res+</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>4</td></tr><tr><td rowspan=1 colspan=1>Org+/Res-</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>167</td><td rowspan=1 colspan=1>37</td><td rowspan=1 colspan=1>204</td></tr><tr><td rowspan=1 colspan=1>Org-</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>788</td><td rowspan=1 colspan=1>808</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>3 a</td><td rowspan=1 colspan=1>188</td><td rowspan=1 colspan=1>825</td><td rowspan=1 colspan=1>1,016</td></tr><tr><td rowspan=1 colspan=3>Combined Hosts / kpc</td><td rowspan=1 colspan=1>rate [%]</td><td rowspan=1 colspan=1>positivity</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=3>Agreement (Org+/Res+)</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>43.9 - 100.0</td></tr><tr><td rowspan=1 colspan=3>Agreement (Org+/Res-)</td><td rowspan=1 colspan=1>88.8</td><td rowspan=1 colspan=1>167/188</td><td rowspan=1 colspan=1>83.5 - 92.6</td></tr><tr><td rowspan=1 colspan=3>Agreement (Org-)</td><td rowspan=1 colspan=1>95.5</td><td rowspan=1 colspan=1>788/825</td><td rowspan=1 colspan=1>93.9 - 96.7</td></tr></table>

a Specimens with multiple detected possible host microorganisms by culture: 0 of 1, by the composite comparator: 2 of 3. b SoC reported the following possible corresponding host microorganisms for one $\mathrm { O r g ^ { + / R e s ^ { + } } }$ case: $E .$ . cloacae complex. c SoC reported the following possible corresponding host microorganisms for two $\mathrm { O r g ^ { + } / R e s ^ { - } }$ cases: Acinetobacter spp., $K .$ . pneumoniae / P. aeruginosa. d SoC reported the following possible corresponding host microorganisms for one $\mathrm { O r g ^ { + / N o } }$ Isolate case: K. pneumoniae.

Table 39: Clinical Performance, ndm/Applicable Gram-negative targets: Comparator methods A (culture for microorganism identification; isolate sequencing for ndm) and B (composite comparator for microorganism identification; PCR/sequencing for ndm)   

<table><tr><td rowspan=10 colspan=1>A</td><td rowspan=2 colspan=2>Combined Hostsndm</td><td rowspan=1 colspan=5>Culture Result for Host Microorganism(s) +Isolate Sequencing / Marker Screening for ndm</td></tr><tr><td rowspan=1 colspan=1>Org+/Res+</td><td rowspan=1 colspan=1>Org+/Res-</td><td rowspan=1 colspan=1>Org+/No Isolate</td><td rowspan=1 colspan=1>Org-</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=4 colspan=1>LRTBALResult</td><td rowspan=1 colspan=1>Org+/Res+</td><td rowspan=1 colspan=1>1c</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>Org+/Res-</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>98</td><td rowspan=1 colspan=1>33</td><td rowspan=1 colspan=1>76</td><td rowspan=1 colspan=1>207</td></tr><tr><td rowspan=1 colspan=1>Org-</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>796</td><td rowspan=1 colspan=1>807</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>1 a</td><td rowspan=1 colspan=1>101</td><td rowspan=1 colspan=1>41</td><td rowspan=1 colspan=1>872</td><td rowspan=1 colspan=1>1,015b</td></tr><tr><td rowspan=1 colspan=3>Combined Hosts / ndm</td><td rowspan=1 colspan=1>rate [%]</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>positivity</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=3>Agreement (Org+/Res+)</td><td rowspan=1 colspan=2>100.0</td><td rowspan=1 colspan=1>1/1</td><td rowspan=1 colspan=1>20.7 - 100.0</td></tr><tr><td rowspan=1 colspan=3>Agreement (Org+/Res-)</td><td rowspan=1 colspan=2>97.0</td><td rowspan=1 colspan=1>98/101</td><td rowspan=1 colspan=1>91.6 - 99.0</td></tr><tr><td rowspan=1 colspan=3>Agreement (Org-)</td><td rowspan=1 colspan=2>91.3</td><td rowspan=1 colspan=1>796/872</td><td rowspan=1 colspan=1>89.2 - 93.0</td></tr></table>

<table><tr><td rowspan=10 colspan=1>B</td><td rowspan=2 colspan=2>Combined Hostsndm</td><td rowspan=1 colspan=4>Composite Comparator Result for Host Microorganism(s) +Mol. Reference (PCR/Seq.) for ndm</td></tr><tr><td rowspan=1 colspan=1>Org+/Res+</td><td rowspan=1 colspan=1>Org+/Res-</td><td rowspan=1 colspan=1>Org-</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=4 colspan=1>LRTBALResult</td><td rowspan=1 colspan=1>Org+/Res+</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>Org+/Res-</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>170</td><td rowspan=1 colspan=1>37</td><td rowspan=1 colspan=1>207</td></tr><tr><td rowspan=1 colspan=1>Org-</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>787</td><td rowspan=1 colspan=1>807</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>1 a</td><td rowspan=1 colspan=1>190</td><td rowspan=1 colspan=1>824</td><td rowspan=1 colspan=1>1,015 b</td></tr><tr><td rowspan=1 colspan=3>Combined Hosts / ndm</td><td rowspan=1 colspan=1>rate [%]</td><td rowspan=1 colspan=1>positivity</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=3>Agreement (Org+/Res+)</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>1/1</td><td rowspan=1 colspan=1>20.7 - 100.0</td></tr><tr><td rowspan=1 colspan=3>Agreement (Org+/Res-)</td><td rowspan=1 colspan=1>89.5</td><td rowspan=1 colspan=1>170/190</td><td rowspan=1 colspan=1>84.3 - 93.1</td></tr><tr><td rowspan=1 colspan=3>Agreement (Org-)</td><td rowspan=1 colspan=1>95.5</td><td rowspan=1 colspan=1>787/824</td><td rowspan=1 colspan=1>93.9 - 96.7</td></tr></table>

a Specimens with multiple detected possible host microorganisms by culture: 1 of 1, by the composite comparator: 1 of 1. b One invalid LRT BAL result for ndm. c SoC reported the following possible corresponding host microorganisms for one $\mathrm { O r g ^ { + / R e s ^ { + } } }$ case: K. pneumoniae/P. aeruginosa.

Table 40: Clinical Performance, vim/Applicable Gram-negative targets: Comparator methods A (culture for microorganism identification; isolate sequencing for vim) and B (composite comparator for microorganism identification; PCR/sequencing for vim)   

<table><tr><td rowspan=10 colspan=1>A</td><td rowspan=2 colspan=2>Combined Hostsvim</td><td rowspan=1 colspan=5>Culture Result for Host Microorganism(s) +Isolate Sequencing / Marker Screening for vim</td></tr><tr><td rowspan=1 colspan=1>Org+/Res+</td><td rowspan=1 colspan=1>Org+/Res-</td><td rowspan=1 colspan=1>Org+/No Isolate</td><td rowspan=1 colspan=1>Org-</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=4 colspan=1>LRTBALResult</td><td rowspan=1 colspan=1>Org+/Res+</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1 a</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>Org+/Res-</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>98</td><td rowspan=1 colspan=1>33</td><td rowspan=1 colspan=1>76</td><td rowspan=1 colspan=1>207</td></tr><tr><td rowspan=1 colspan=1>Org-</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>797</td><td rowspan=1 colspan=1>808</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>102</td><td rowspan=1 colspan=1>41</td><td rowspan=1 colspan=1>873</td><td rowspan=1 colspan=1>1,016</td></tr><tr><td rowspan=1 colspan=3>Combined Hosts / vim</td><td rowspan=1 colspan=1>rate [%]</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>positivity</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=3>Agreement (Org+/Res+)</td><td rowspan=1 colspan=2>NA</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=3>Agreement (Org+/Res-)</td><td rowspan=1 colspan=2>96.1</td><td rowspan=1 colspan=1>98/102</td><td rowspan=1 colspan=1>90.3 - 98.5</td></tr><tr><td rowspan=1 colspan=3>Agreement (Org-)</td><td rowspan=1 colspan=2>91.3</td><td rowspan=1 colspan=1>797/873</td><td rowspan=1 colspan=1>89.2 - 93.0</td></tr></table>

<table><tr><td rowspan=10 colspan=1>B</td><td rowspan=2 colspan=2>Combined Hostsvim</td><td rowspan=1 colspan=4>Composite Comparator Result for Host Microorganism(s) +Mol. Reference (PCR/Seq.) for vim</td></tr><tr><td rowspan=1 colspan=1>Org+/Res+</td><td rowspan=1 colspan=1>Org+/Res-</td><td rowspan=1 colspan=1>Org-</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=4 colspan=1>LRTBALResult</td><td rowspan=1 colspan=1>Org+/Res+</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>Org+/Res-</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>170</td><td rowspan=1 colspan=1>37</td><td rowspan=1 colspan=1>207</td></tr><tr><td rowspan=1 colspan=1>Org-</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>788</td><td rowspan=1 colspan=1>808</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>191</td><td rowspan=1 colspan=1>825</td><td rowspan=1 colspan=1>1,016</td></tr><tr><td rowspan=1 colspan=3>Combined Hosts / vim</td><td rowspan=1 colspan=1>rate [%]</td><td rowspan=1 colspan=1>positivity</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=3>Agreement (Org+/Res+)</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=3>Agreement (Org+/Res-)</td><td rowspan=1 colspan=1>89.0</td><td rowspan=1 colspan=1>170/191</td><td rowspan=1 colspan=1>83.8 - 92.7</td></tr><tr><td rowspan=1 colspan=3>Agreement (Org-)</td><td rowspan=1 colspan=1>95.5</td><td rowspan=1 colspan=1>788/825</td><td rowspan=1 colspan=1>93.9 - 96.7</td></tr></table>

a SoC reported the following possible corresponding host microorganisms for one Org+/Res- case: P. aeruginosa.

Table 41: Clinical Performance, oxa-48-M/Applicable Gram-negative targets: Comparator methods A (culture for microorganism identification; isolate sequencing for oxa-48) and B (composite comparator for microorganism identification; PCR/sequencing for oxa-48)   

<table><tr><td rowspan=10 colspan=1>A</td><td rowspan=2 colspan=2>Combined Hostsoxa-48</td><td rowspan=1 colspan=5>Culture Result for Host Microorganism(s) +Isolate Sequencing / Marker Screening for oxa-48</td></tr><tr><td rowspan=1 colspan=1>Org+/Res+</td><td rowspan=1 colspan=1>Org+/Res-</td><td rowspan=1 colspan=1>Org+/No Isolate</td><td rowspan=1 colspan=1>Org-</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=4 colspan=1>LRTBALResult</td><td rowspan=1 colspan=1>Org+/Res+</td><td rowspan=1 colspan=1>1c</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>Org+/Res-</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>51</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>47</td><td rowspan=1 colspan=1>111</td></tr><tr><td rowspan=1 colspan=1>Org-</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>890</td><td rowspan=1 colspan=1>900</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>1 a</td><td rowspan=1 colspan=1>55</td><td rowspan=1 colspan=1>19</td><td rowspan=1 colspan=1>937</td><td rowspan=1 colspan=1>1,012 b</td></tr><tr><td rowspan=1 colspan=3>Combined Hosts / oxa-48</td><td rowspan=1 colspan=1>rate [%]</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>positivity</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=3>Agreement (Org+/Res+)</td><td rowspan=1 colspan=2>100.0</td><td rowspan=1 colspan=1>1/1</td><td rowspan=1 colspan=1>20.7 - 100.0</td></tr><tr><td rowspan=1 colspan=3>Agreement (Org+/Res-)</td><td rowspan=1 colspan=2>92.7</td><td rowspan=1 colspan=1>51/55</td><td rowspan=1 colspan=1>82.7 - 97.1</td></tr><tr><td rowspan=1 colspan=3>Agreement (Org-)</td><td rowspan=1 colspan=2>95.0</td><td rowspan=1 colspan=1>890/937</td><td rowspan=1 colspan=1>93.4 - 96.2</td></tr></table>

<table><tr><td rowspan=10 colspan=1>B</td><td rowspan=2 colspan=2>Combined Hostsoxa-48</td><td rowspan=1 colspan=4>Composite Comparator Result for Host Microorganism(s) +Mol. Reference (PCR/Seq.) for oxa-48</td></tr><tr><td rowspan=1 colspan=1>Org+/Res+</td><td rowspan=1 colspan=1>Org+/Res-</td><td rowspan=1 colspan=1>Org-</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=4 colspan=1>LRTBALResult</td><td rowspan=1 colspan=1>Org+/Res+</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>Org+/Res-</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>84</td><td rowspan=1 colspan=1>27</td><td rowspan=1 colspan=1>111</td></tr><tr><td rowspan=1 colspan=1>Org-</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>886</td><td rowspan=1 colspan=1>900</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>1 a</td><td rowspan=1 colspan=1>98</td><td rowspan=1 colspan=1>913</td><td rowspan=1 colspan=1>1,012b</td></tr><tr><td rowspan=1 colspan=3>Combined Hosts / oxa-48</td><td rowspan=1 colspan=1>rate [%]</td><td rowspan=1 colspan=1>positivity</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=3>Agreement (Org+/Res+)</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>1/1</td><td rowspan=1 colspan=1>20.7 - 100.0</td></tr><tr><td rowspan=1 colspan=3>Agreement (Org+/Res-)</td><td rowspan=1 colspan=1>85.7</td><td rowspan=1 colspan=1>84/98</td><td rowspan=1 colspan=1>77.4 - 91.3</td></tr><tr><td rowspan=1 colspan=3>Agreement (Org-)</td><td rowspan=1 colspan=1>97.0</td><td rowspan=1 colspan=1>886/913</td><td rowspan=1 colspan=1>95.7 - 98.0</td></tr></table>

a Specimens with multiple detected possible host microorganisms by culture: 0 of 1, by the composite comparator: 0 of 1. b Four invalid LRT BAL results for oxa-48. c SoC reported the following possible corresponding host microorganisms for one $\mathrm { O r g ^ { + / R e s ^ { + } } }$ case: K. pneumoniae / $P$ . aeruginosa.

Table 42: Clinical Performance, tem/H. haemophilus: Comparator methods A (culture for microorganism identification; isolate sequencing for tem) and B (composite comparator for microorganism identification; PCR/sequencing for tem)   

<table><tr><td rowspan=10 colspan=1>A</td><td rowspan=2 colspan=2>H. influenzaetem</td><td rowspan=1 colspan=5>Culture Result for H. influenzae +Isolate Sequencing / Marker Screening for tem</td></tr><tr><td rowspan=1 colspan=1>Org+/Res+</td><td rowspan=1 colspan=1>Org+/Res-</td><td rowspan=1 colspan=1>Org+/No Isolate</td><td rowspan=1 colspan=1>Org-</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=4 colspan=1>LRTBALResult</td><td rowspan=1 colspan=1>Org+/Res+</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>16</td></tr><tr><td rowspan=1 colspan=1>Org+/Res-</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>35</td><td rowspan=1 colspan=1>40</td></tr><tr><td rowspan=1 colspan=1>Org-</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>958</td><td rowspan=1 colspan=1>959</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>1,006</td><td rowspan=1 colspan=1>1,015 a</td></tr><tr><td rowspan=1 colspan=3>H. influenzae / tem</td><td rowspan=1 colspan=1>rate [%]</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>positivity</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=3>Agreement (Org+/Res+)</td><td rowspan=1 colspan=2>100.0</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>34.2 - 100.0</td></tr><tr><td rowspan=1 colspan=3>Agreement (Org+/Res-)</td><td rowspan=1 colspan=2>100.0</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>43.9 - 100.0</td></tr><tr><td rowspan=1 colspan=3>Agreement (Org-)</td><td rowspan=1 colspan=2>95.2</td><td rowspan=1 colspan=1>958/1,006</td><td rowspan=1 colspan=1>93.7 - 96.4</td></tr></table>

<table><tr><td rowspan=10 colspan=1>B</td><td rowspan=2 colspan=2>H. influenzaetem</td><td rowspan=1 colspan=4>Composite Comparator Result for H. influenzae +Mol. Reference (PCR/Seq.) for tem</td></tr><tr><td rowspan=1 colspan=1>Org+/Res+</td><td rowspan=1 colspan=1>Org+/Res-</td><td rowspan=1 colspan=1>Org-</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=4 colspan=1>LRTBALResult</td><td rowspan=1 colspan=1>Org+/Res+</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>16</td></tr><tr><td rowspan=1 colspan=1>Org+/Res-</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>26</td><td rowspan=1 colspan=1>40</td></tr><tr><td rowspan=1 colspan=1>Org-</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>957</td><td rowspan=1 colspan=1>959</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>993</td><td rowspan=1 colspan=1>1,015a</td></tr><tr><td rowspan=1 colspan=3>H. influenzae / tem</td><td rowspan=1 colspan=1>rate [%]</td><td rowspan=1 colspan=1>positivity</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=3>Agreement (Org+/Res+)</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>61.0 - 100.0</td></tr><tr><td rowspan=1 colspan=3>Agreement (Org+/Res-)</td><td rowspan=1 colspan=1>87.5</td><td rowspan=1 colspan=1>14/16</td><td rowspan=1 colspan=1>64.0 - 96.5</td></tr><tr><td rowspan=1 colspan=3>Agreement (Org-)</td><td rowspan=1 colspan=1>96.4</td><td rowspan=1 colspan=1>957/993</td><td rowspan=1 colspan=1>95.0 - 97.4</td></tr></table>

a One LRT BAL run with an invalid result for both $H .$ . influenzae and tem.

Table 43: Clinical Performance, mecA/S. aureus: Comparator methods A (culture for microorganism identification; isolate sequencing for mecA), B (composite comparator for microorganism identification; PCR/sequencing for mecA) and C (phenotypic AST results for $S .$ . aureus isolates)   

<table><tr><td rowspan=10 colspan=1>A</td><td rowspan=2 colspan=2>S. aureusmecA</td><td rowspan=1 colspan=5>Culture Result for S. aureus +Isolate Sequencing / Marker Screening for mecA</td></tr><tr><td rowspan=1 colspan=1>Org+/Res+</td><td rowspan=1 colspan=1>Org+/Res-</td><td rowspan=1 colspan=1>Org+/No Isolate</td><td rowspan=1 colspan=1>Org-</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=4 colspan=1>LRTBALResult</td><td rowspan=1 colspan=1>Org+/Res+</td><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>47</td></tr><tr><td rowspan=1 colspan=1>Org+/Res-</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>23</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>57</td></tr><tr><td rowspan=1 colspan=1>Org-</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>904</td><td rowspan=1 colspan=1>912</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>29</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>945</td><td rowspan=1 colspan=1>1,016</td></tr><tr><td rowspan=1 colspan=3>S. aureus / mecA</td><td rowspan=1 colspan=1>rate [%]</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>positivity</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=3>Agreement (Org+/Res+)</td><td rowspan=1 colspan=1>81.0</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>17/21</td><td rowspan=1 colspan=1>60.0 - 92.3</td></tr><tr><td rowspan=1 colspan=3>Agreement (Org+/Res-)</td><td rowspan=1 colspan=2>79.3</td><td rowspan=1 colspan=1>23/29</td><td rowspan=1 colspan=1>61.6 - 90.2</td></tr><tr><td rowspan=1 colspan=3>Agreement (Org-)</td><td rowspan=1 colspan=2>95.7</td><td rowspan=1 colspan=1>904/945</td><td rowspan=1 colspan=1>94.2 - 96.8</td></tr></table>

<table><tr><td rowspan=10 colspan=1>B</td><td rowspan=2 colspan=2>S. aureusmecA</td><td rowspan=1 colspan=4>Composite Comparator Result for S. aureus +Mol. Reference (PCR/Seq.) for mecA</td></tr><tr><td rowspan=1 colspan=1>Org+/Res+</td><td rowspan=1 colspan=1>Org+/Res-</td><td rowspan=1 colspan=1>Org-</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=4 colspan=1>LRTBALResult</td><td rowspan=1 colspan=1>Org+/Res+</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>18 a</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>47</td></tr><tr><td rowspan=1 colspan=1>Org+/Res-</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>42</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>57</td></tr><tr><td rowspan=1 colspan=1>Org-</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>896</td><td rowspan=1 colspan=1>912</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>25</td><td rowspan=1 colspan=1>71</td><td rowspan=1 colspan=1>920</td><td rowspan=1 colspan=1>1,016</td></tr><tr><td rowspan=1 colspan=3>S. aureus / mecA</td><td rowspan=1 colspan=1>rate [%]</td><td rowspan=1 colspan=1>positivity</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=3>Agreement (Org+/Res+)</td><td rowspan=1 colspan=1>80.0</td><td rowspan=1 colspan=1>20/25</td><td rowspan=1 colspan=1>60.9 - 91.1</td></tr><tr><td rowspan=1 colspan=3>Agreement (Org+/Res-)</td><td rowspan=1 colspan=1>59.2</td><td rowspan=1 colspan=1>42/71</td><td rowspan=1 colspan=1>47.5 - 69.8</td></tr><tr><td rowspan=1 colspan=3>Agreement (Org-)</td><td rowspan=1 colspan=1>97.4</td><td rowspan=1 colspan=1>896/920</td><td rowspan=1 colspan=1>96.1 - 98.2</td></tr></table>

<table><tr><td rowspan=10 colspan=1>C</td><td rowspan=2 colspan=2>S. aureusmecA</td><td rowspan=1 colspan=5>Culture Result for S. aureus +and Cefoxitin/Oxacillin AST Results</td></tr><tr><td rowspan=1 colspan=1>Org+/MRSA</td><td rowspan=1 colspan=1>Org+/MSSA</td><td rowspan=1 colspan=1>Org+/No AST</td><td rowspan=1 colspan=1>Org-</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=4 colspan=1>LRTBALResult</td><td rowspan=1 colspan=1>Org+/Res+</td><td rowspan=1 colspan=1>23 b</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>47</td></tr><tr><td rowspan=1 colspan=1>Org+/Res-</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>27</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>57</td></tr><tr><td rowspan=1 colspan=1>Org-</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>904</td><td rowspan=1 colspan=1>912</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>27</td><td rowspan=1 colspan=1>37</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>945</td><td rowspan=1 colspan=1>1,016</td></tr><tr><td rowspan=1 colspan=2></td><td rowspan=1 colspan=1>S. aureus / MRSA status</td><td rowspan=1 colspan=1>rate [%]</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>positivity</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=3>Agreement (Org+/Res+)</td><td rowspan=1 colspan=2>85.2</td><td rowspan=1 colspan=1>23/27</td><td rowspan=1 colspan=1>67.5 - 94.1</td></tr><tr><td rowspan=1 colspan=3>Agreement (Org+/Res-)</td><td rowspan=1 colspan=2>73.0</td><td rowspan=1 colspan=1>27/37</td><td rowspan=1 colspan=1>57.0 - 84.6</td></tr><tr><td rowspan=1 colspan=3>Agreement (Org-)</td><td rowspan=1 colspan=2>95.7</td><td rowspan=1 colspan=1>904/945</td><td rowspan=1 colspan=1>94.2 - 96.8</td></tr></table>

a 18 cases were determined as Org+/Res- negative by the molecular reference assay using cutoffs set to achieve an analytical sensitivity in a comparable range to LRT BAL. For 10 of these 18 cases, presence of mecA was confirmed by alternate molecular tests (PCR/sequencing). b One strain was reported by the clinical site as MRSA, although this was not supported by the provided oxacillin Kirby-Bauer zone diameter. Independent cefoxitin AST confirmed the MRSA phenotype for this strain.

# Clinical Performance, Resistance Markers, Archived Study

Results observed for LRT BAL resistance marker targets are listed in Table 44 below. Since resistance marker results are reported by the LRT BAL Application when an applicable microorganism is simultaneously detected, the results presented include a subset of specimens tested (i.e., those specimens with an applicable microorganism detected for each resistance marker target).

Table 44: Resistance marker results reported by LRT BAL, Archived Study   

<table><tr><td rowspan=1 colspan=1>Resistance Marker(# Applicablespecimens)</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td></tr><tr><td rowspan=1 colspan=1>ctx-M (N = 212)</td><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>198</td></tr><tr><td rowspan=1 colspan=1>kpc (N = 212)</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>210</td></tr><tr><td rowspan=1 colspan=1>mecA (N = 77)</td><td rowspan=1 colspan=1>44</td><td rowspan=1 colspan=1>33</td></tr><tr><td rowspan=1 colspan=1>ndm (N = 212)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>212</td></tr><tr><td rowspan=1 colspan=1>oxa-23 (N = 21)</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>17</td></tr><tr><td rowspan=1 colspan=1>oxa-24 (N = 21)</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>18</td></tr><tr><td rowspan=1 colspan=1>oxa-48 N = 166)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>166</td></tr><tr><td rowspan=1 colspan=1>oxa-58 (N = 21)</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>20</td></tr><tr><td rowspan=1 colspan=1>tem (N = 71)</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>47</td></tr><tr><td rowspan=1 colspan=1>vim (N = 212)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>212</td></tr></table>

Applicable molecular comparator testing (PCR/sequencing) was performed for all specimens with resistance marker targets reported as positive by LRT BAL Application. LRT BAL resistance marker detections were confirmed by sequencing in 14 of 14 cases for ctx-M, 2 of 2 cases for kpc, 34 of 44 cases for mecA, 4 of 4 cases for oxa-23, 3 of 3 cases for oxa-24, 1 of 1 case for oxa-58, and 23 of 24 cases for tem. For all other cases, PCRs did not amplify sufficient amounts for sequencing.

# Correlation of Detected Antibiotic Resistance Markers to Strain Genotypes and Phenotypes:

Available isolates collected from SoC cultures for the majority of the prospective and a portion of archived study BAL specimens were screened for presence or absence of antibiotic resistance markers targeted by the LRT BAL Application. Furthermore, corresponding antimicrobial susceptibility testing (AST) results were collected for SoC culture isolates for one or more of the antimicrobials listed in Table 45. AST results were reported as MIC values or zone diameters (for Kirby-Bauer tests). Strain phenotypes across all test sites were standardized according to breakpoints listed in CLSI guidance M100S (Performance Standards for Antimicrobial Susceptibility Testing, $2 6 ^ { \mathrm { t h } }$ Edition).

‘Intermediate’ calls were regarded as ‘resistant’ calls for this study. For the analysis, any strain was regarded as ‘resistant’ if at least one of the corresponding drugs resulted in an ‘intermediate’ or ‘resistant’ call. Any strain was regarded as ‘susceptible’ if a ‘sensitive’ call was obtained for all applicable tested drugs.

Table 45: Antimicrobial Agents Evaluated by AST for LRT BAL Resistance Marker Targets   

<table><tr><td rowspan=1 colspan=1>Antibiotic Resistance Marker(s)</td><td rowspan=1 colspan=1>Associated Resistance</td><td rowspan=1 colspan=1>AST</td></tr><tr><td rowspan=1 colspan=1>tem</td><td rowspan=1 colspan=1>penicillins</td><td rowspan=1 colspan=1>H. influenzae:ampicillin, cefinase</td></tr><tr><td rowspan=1 colspan=1>ctx-M</td><td rowspan=1 colspan=1>3rd generation cephalosporins,cefepime</td><td rowspan=1 colspan=1>Enterobacteriaceae:cefotaxime, ceftazidime, ceftriaxoneAcinetobacter spp.:cefotaxime, ceftazidime, ceftriaxone,cefepimeP. aeruginosa:ceftazidime, cefepime</td></tr><tr><td rowspan=1 colspan=1>kpc, ndm, oxa-48, vim</td><td rowspan=1 colspan=1>carbapenems</td><td rowspan=1 colspan=1>Enterobacteriaceae:meropenem, ertapenem, imipenem</td></tr><tr><td rowspan=1 colspan=1>kpc, ndm, oxa-23, oxa-24, oxa-58,vim</td><td rowspan=1 colspan=1>carbapenems</td><td rowspan=1 colspan=1>Acinetobacter spp.meropenem, imipenem</td></tr><tr><td rowspan=1 colspan=1>kpc, ndm, vim</td><td rowspan=1 colspan=1>carbapenems</td><td rowspan=1 colspan=1>P. aeruginosa:meropenem, imipenem</td></tr><tr><td rowspan=1 colspan=1>mecA</td><td rowspan=1 colspan=1>oxacillin/cefoxitin</td><td rowspan=1 colspan=1>S. aureus:oxacillin, cefoxitin</td></tr></table>

Antibiotic resistance markers reported by LRT BAL were correlated to available isolates for:

Genotypic agreement of reported antibiotic resistance markers to presence of such markers in cultured isolates for a specific specimen. Phenotypic agreement of reported antibiotic resistance markers with associated phenotypic culture results to provided isolates as determined by antimicrobial susceptibility tests (AST).

Note that antibiotic resistance marker results are reported by the LRT BAL Application only if a corresponding host microorganism is simultaneously reported. Otherwise, such results are masked on the Unyvero results screen. Antibiotic resistance marker results reported together with a corresponding host microorganism may not be linked to this organism, but may have originated from other on-panel or off-panel microorganisms, e.g., mecA may not only originate from S. aureus, but also originate from coagulase-negative Staphylococci present in respiratory flora; tem may not only originate from $H .$ influenzae, but also from other on-panel or off-panel gram negative species. If multiple corresponding microorganisms are simultaneously reported for a certain antibiotic resistance marker, one or more than one of these microorganisms could be the host of such marker.

For available isolates collected from reference culture in the prospective and archived study, evaluation of genotypic and phenotypic agreement of LRT BAL results compared to PCR/sequencing and AST respectively are presented in Tables 46 to 50.

# Table 46: Oxacillin/Cefoxitin Resistance for S. aureus (mecA)

Correlation of positive mecA results reported by LRT BAL for the prospective (‘p’) or archived (‘a’) study to available S. aureus isolates (genotypic agreement, ‘yes’: presence of mecA confirmed by sequencing of isolate, ‘no’: mecA not confirmed, ‘not prov.’: no isolate provided) and to AST results indicating oxacillin/cefoxitin resistance for isolates with confirmed mecA (phenotypic agreement; R: resistant phenotype, S: sensitive phenotype, NA: AST results not reported).

<table><tr><td rowspan=1 colspan=8>Genotypic Agreement for Specimens with Available Isolate: 21/25, 84.0% (95% CI: 65.3 - 93.6)</td></tr><tr><td rowspan=1 colspan=8>Phenotypic Agreement for Isolates with Confirmed Marker: 21/21, 100.0% (95% CI: 84.5 - 100.0)</td></tr><tr><td rowspan=1 colspan=3># Cases</td><td rowspan=2 colspan=1>LRT BAL Result(relevant hosts only)</td><td rowspan=2 colspan=1>SoC Result(relevant hosts only)</td><td rowspan=2 colspan=1>GenotypicAgreement(for availableisolates)</td><td rowspan=2 colspan=1>PhenotypicAgreement(for isolateswith conf. mecA)</td><td rowspan=2 colspan=1>Comment</td></tr><tr><td rowspan=1 colspan=1>total</td><td rowspan=1 colspan=2>p  a</td></tr><tr><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>S. aureus</td><td rowspan=1 colspan=1>S. aureus</td><td rowspan=1 colspan=1>yes</td><td rowspan=1 colspan=1>R a</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>S. aureus</td><td rowspan=1 colspan=1>S. aureus</td><td rowspan=1 colspan=1>no</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=8>Other specimens without available S. aureus isolate:</td></tr><tr><td rowspan=1 colspan=1>26</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>S. aureus</td><td rowspan=1 colspan=1>S. aureus</td><td rowspan=1 colspan=1>(not prov.)</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>AST: resistant</td></tr><tr><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>S. aureus</td><td rowspan=1 colspan=1>S. aureus</td><td rowspan=1 colspan=1>(not prov.)</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>AST: sensitive</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>S. aureus</td><td rowspan=1 colspan=1>S. aureus</td><td rowspan=1 colspan=1>(not prov.)</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>AST: NA</td></tr><tr><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>S. aureus</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr></table>

# Table 47: Penicillin Resistance for H. influenzae (tem)

Correlation of positive tem results reported by LRT BAL for the prospective (‘p’) or archived (‘a’) study to available $H .$ influenzae isolates (genotypic agreement, ‘yes’: presence of tem confirmed by sequencing of isolate, ‘no’: tem not confirmed, ‘not prov.’: no isolate provided) and to AST results indicating penicillin resistance for isolates with confirmed tem (phenotypic agreement; R: resistant phenotype, S: sensitive phenotype, NA: AST results not reported).

<table><tr><td rowspan=1 colspan=8>Genotypic Agreement for Specimens with Available Isolate: 4/4, 100.0% (95% CI: 51.0 - 100.0)</td></tr><tr><td rowspan=1 colspan=8>Phenotypic Agreement for Isolates with Confirmed Marker: 4/4, 100.0% (95% CI: 51.0 - 100.0)</td></tr><tr><td rowspan=1 colspan=3># Cases</td><td rowspan=2 colspan=1>LRT BAL Result(relevant hosts only)</td><td rowspan=2 colspan=1>SoC Result(relevant hosts only)</td><td rowspan=2 colspan=1>GenotypicAgreement(for availableisolates)</td><td rowspan=2 colspan=1>PhenotypicAgreement(for isolateswith conf. tem)</td><td rowspan=2 colspan=1>Comment</td></tr><tr><td rowspan=1 colspan=1>total</td><td rowspan=1 colspan=2>p  a</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>H. influenzae</td><td rowspan=1 colspan=1>H. influenzae</td><td rowspan=1 colspan=1>yes</td><td rowspan=1 colspan=1>R</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=8>other applicable specimens without available H. influenzae isolate:</td></tr><tr><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>H. influenzae</td><td rowspan=1 colspan=1>H. influenzae</td><td rowspan=1 colspan=1>(not prov.)</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>AST: resistant</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>H. influenzae</td><td rowspan=1 colspan=1>H. influenzae</td><td rowspan=1 colspan=1>(not prov.)</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>AST: sensitive</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>H. influenzae</td><td rowspan=1 colspan=1>H. influenzae</td><td rowspan=1 colspan=1>(not prov.)</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>AST: NA</td></tr><tr><td rowspan=1 colspan=1>18</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>H. influenzae</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr></table>

# Table 48: Third Generation Cephalosporin Resistance for targeted Enterobacteriaceae, Acinetobacter spp., and $P .$ aeruginosa $( c t x \mathbf { - } \mathbf { M } )$

Correlation of positive ctx-M results reported by LRT BAL for the prospective $( ^ { \bullet } \mathfrak { p } ^ { \prime } )$ or archived (‘a’) study to available isolates of Enterobacteriaceae, Acinetobacter spp., or $P .$ . aeruginosa (genotypic agreement, ‘yes’: presence of ctx-M confirmed by sequencing of isolate, ‘no’: ctx-M not confirmed, ‘not prov.’: no isolate provided) and to AST results indicating third generation cephalosporin resistance for isolates with confirmed ctx-M (phenotypic agreement; R: resistant phenotype, S: sensitive phenotype, NA: AST results not reported).

Genotypic Agreement for Specimens with at Least One Available Isolate: 5/8, $6 2 . 5 \%$ $9 5 \%$ CI: 30.6 - 86.3)

Phenotypic Agreement for Isolates with Confirmed Marker: 4/4, $1 0 0 . 0 \%$ ( $9 5 \%$ CI: 51.0 - 100.0)

<table><tr><td colspan="1" rowspan="1">Study</td><td colspan="1" rowspan="1">LRT BAL Result(relevant hosts only)</td><td colspan="1" rowspan="1">SoC Result(relevant hosts only)</td><td colspan="1" rowspan="1">GenotypicAgreement(for availableisolates)</td><td colspan="1" rowspan="1">PhenotypicAgreement(for isolateswith conf. ctx-M)</td><td colspan="1" rowspan="1">Comment</td></tr><tr><td colspan="1" rowspan="4">p</td><td colspan="1" rowspan="1">Acinetobacter spp.</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">K. pneumoniae</td><td colspan="1" rowspan="1">K. pneumoniae</td><td colspan="1" rowspan="1">yes</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">P. aeruginosa</td><td colspan="1" rowspan="1">P. aeruginosa</td><td colspan="1" rowspan="1">no</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="2">b</td><td colspan="1" rowspan="1">K. pneumoniae</td><td colspan="1" rowspan="1">K. pneumoniae</td><td colspan="1" rowspan="1">yes</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">P. aeruginosa</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="2">p</td><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">E. col</td><td colspan="1" rowspan="1">yes</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">P. aeruginosa</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="2">p</td><td colspan="1" rowspan="1">E. col</td><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">no</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">[off-panel]</td><td colspan="1" rowspan="1">Providencia stuartii</td><td colspan="1" rowspan="1">yes</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="3">p</td><td colspan="1" rowspan="1">Acinetobacter spp.</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">K. pneumoniae</td><td colspan="1" rowspan="1">K. pneumoniae</td><td colspan="1" rowspan="1">(not. prov.)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="3">−</td><td colspan="1" rowspan="1">Proteus spp.</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">P. aeruginosa</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">[off-panel]</td><td colspan="1" rowspan="1">Providencia stuartii</td><td colspan="1" rowspan="1">yes</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="4">p</td><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">K. pneumoniae</td><td colspan="1" rowspan="1">K. pneumoniae</td><td colspan="1" rowspan="1">(not. prov.)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">P. aeruginosa</td><td colspan="1" rowspan="1">P. aeruginosa</td><td colspan="1" rowspan="1">no</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">S. marcescens</td><td colspan="1" rowspan="1">S. marcescens</td><td colspan="1" rowspan="1">(not. prov.)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="2">p</td><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">P. aeruginosa</td><td colspan="1" rowspan="1">P. aeruginosa</td><td colspan="1" rowspan="1">no</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="4">p</td><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">M. morganii</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Proteus spp.</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">P. aeruginosa</td><td colspan="1" rowspan="1">P. aeruginosa</td><td colspan="1" rowspan="1">no</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr></table>

Other applicable specimens without available isolates:   

<table><tr><td rowspan=1 colspan=1>a</td><td rowspan=1 colspan=1>K. oxytoca</td><td rowspan=1 colspan=1>K. oxytoca</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>AST: resistant</td></tr><tr><td rowspan=2 colspan=1>a</td><td rowspan=1 colspan=1>Proteus spp.</td><td rowspan=1 colspan=1>Proteus spp.</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>AST: NA</td></tr><tr><td rowspan=1 colspan=1>P. aeruginosa</td><td rowspan=1 colspan=1>P. aeruginosa</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>AST: resistant</td></tr><tr><td rowspan=1 colspan=1>a</td><td rowspan=1 colspan=1>E. coli</td><td rowspan=1 colspan=1>E. col</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>AST: resistant</td></tr><tr><td rowspan=1 colspan=1>a</td><td rowspan=1 colspan=1>E. coli</td><td rowspan=1 colspan=1>E. coli</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>AST: resistant</td></tr><tr><td rowspan=1 colspan=1>a</td><td rowspan=1 colspan=1>K. pneumoniae</td><td rowspan=1 colspan=1>K. pneumoniae</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>AST: resistant</td></tr><tr><td rowspan=2 colspan=1>a</td><td rowspan=1 colspan=1>K. pneumoniae</td><td rowspan=1 colspan=1>K. pneumoniae</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>AST: resistant</td></tr><tr><td rowspan=1 colspan=1>P. aeruginosa</td><td rowspan=1 colspan=1>P. aeruginosa</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>AST: resistant</td></tr><tr><td rowspan=1 colspan=1>a</td><td rowspan=1 colspan=1>E. col</td><td rowspan=1 colspan=1>E. coli</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>AST: resistant</td></tr><tr><td rowspan=3 colspan=1>a</td><td rowspan=1 colspan=1>E. coli</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>K. pneumoniae</td><td rowspan=1 colspan=1>K. pneumoniae</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>AST: resistant</td></tr><tr><td rowspan=1 colspan=1>P. aeruginosa</td><td rowspan=1 colspan=1>P. aeruginosa</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>AST: sensitive</td></tr><tr><td rowspan=1 colspan=1>a</td><td rowspan=1 colspan=1>E. coli</td><td rowspan=1 colspan=1>E. col</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>AST: resistant</td></tr><tr><td rowspan=1 colspan=1>a</td><td rowspan=1 colspan=1>K. pneumoniae</td><td rowspan=1 colspan=1>K. pneumoniae</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>AST: resistant</td></tr><tr><td rowspan=1 colspan=1>a</td><td rowspan=1 colspan=1>K. pneumoniae</td><td rowspan=1 colspan=1>K. pneumoniae</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>AST: resistant</td></tr><tr><td rowspan=1 colspan=1>a</td><td rowspan=1 colspan=1>E. coli</td><td rowspan=1 colspan=1>E. coli</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>AST: resistant</td></tr><tr><td rowspan=2 colspan=1>a</td><td rowspan=1 colspan=1>C. freundii</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>K. pneumoniae</td><td rowspan=1 colspan=1>K. pneumoniae</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>AST: resistant</td></tr><tr><td rowspan=3 colspan=1>a</td><td rowspan=1 colspan=1>Acinetobacter spp.</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>Proteus spp.</td><td rowspan=1 colspan=1>Proteus spp.</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>AST: NA</td></tr><tr><td rowspan=1 colspan=1>P. aeruginosa</td><td rowspan=1 colspan=1>P. aeruginosa</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>AST: sensitive</td></tr><tr><td rowspan=2 colspan=1>p</td><td rowspan=1 colspan=1>E. coli</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>S. marcescens</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr></table>

# Table 49: Carbapenem Resistance for targeted Enterobacteriaceae, Acinetobacter spp., and $\boldsymbol { P } .$ aeruginosa (kpc, ndm, oxa-48, vim)

Correlation of positive results for kpc, ndm, vim reported by LRT BAL for the prospective (‘p’) or archived (‘a’) study to available isolates of Enterobacteriaceae, Acinetobacter spp., or $P _ { - }$ . aeruginosa, or positive results for oxa-48 to available Enterobacteriaceae isolates (genotypic agreement, ‘yes’: presence of any of these markers confirmed by sequencing, ‘no’: presence not confirmed, ‘not prov.’: no isolate provided) and to AST results indicating carbapenem resistance for isolates with confirmed marker (phenotypic agreement; R: resistant phenotype, S: sensitive phenotype, NA: AST results not reported).

<table><tr><td rowspan=1 colspan=7>Genotypic Agreement for Specimens with at Least One Available Isolate: 3/5, 60.0% (95% CI: 23.1 - 88.2)</td></tr><tr><td rowspan=1 colspan=7>Phenotypic Agreement for Isolates with Confirmed Marker: 2/2, 100.0% (95% CI: 34.2 - 100.0)</td></tr><tr><td rowspan=1 colspan=1>Study</td><td rowspan=1 colspan=1>Marker</td><td rowspan=1 colspan=1>LRT BAL Result(relevant hosts only)</td><td rowspan=1 colspan=1>SoC Result(relevant hosts only)</td><td rowspan=1 colspan=1>GenotypicAgreement(foravailableisolates)</td><td rowspan=1 colspan=1>PhenotypicAgreement(for isolateswith conf.marker)</td><td rowspan=1 colspan=1>Comment</td></tr><tr><td rowspan=3 colspan=1>p</td><td rowspan=3 colspan=1>kpc</td><td rowspan=1 colspan=1>E. cloacae complex</td><td rowspan=1 colspan=1>E. cloacae complex</td><td rowspan=1 colspan=1>yes</td><td rowspan=1 colspan=1>R</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>K. pneumoniae</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>P. aeruginosa</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=4 colspan=1>p</td><td rowspan=4 colspan=1>ndm,oxa-48</td><td rowspan=1 colspan=1>Acinetobacter spp.</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>E. coli</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>K. pneumoniae</td><td rowspan=1 colspan=1>K. pneumoniae</td><td rowspan=1 colspan=1>yes (both)</td><td rowspan=1 colspan=1>R</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>P. aeruginosa</td><td rowspan=1 colspan=1>P. aeruginosa</td><td rowspan=1 colspan=1>no</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=3 colspan=1>p</td><td rowspan=3 colspan=1>kpc</td><td rowspan=1 colspan=1>Acinetobacter spp.</td><td rowspan=1 colspan=1>Acinetobacter spp.</td><td rowspan=1 colspan=1>no</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>E. coli</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>K. pneumoniae</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>yes</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>subclinical isolate a</td></tr><tr><td rowspan=2 colspan=1>b</td><td rowspan=2 colspan=1>kpc</td><td rowspan=1 colspan=1>K. pneumoniae</td><td rowspan=1 colspan=1>K. pneumoniae</td><td rowspan=1 colspan=1>(not prov.)</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>AST: resistant</td></tr><tr><td rowspan=1 colspan=1>P. aeruginosa</td><td rowspan=1 colspan=1>P. aeruginosa</td><td rowspan=1 colspan=1>no</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=2 colspan=1>p</td><td rowspan=2 colspan=1>vim</td><td rowspan=1 colspan=1>Proteus spp.</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>P. aeruginosa</td><td rowspan=1 colspan=1>P. aeruginosa</td><td rowspan=1 colspan=1>no</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr></table>

Other applicable specimens without available isolates:   

<table><tr><td rowspan=2 colspan=1>p</td><td rowspan=2 colspan=1>kpc</td><td rowspan=1 colspan=1>Acinetobacter spp.</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>K. pneumoniae</td><td rowspan=1 colspan=1>K. pneumoniae</td><td rowspan=1 colspan=1>(not prov.)</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>AST: resistant</td></tr><tr><td rowspan=2 colspan=1>a</td><td rowspan=2 colspan=1>kpc</td><td rowspan=1 colspan=1>Acinetobacter spp.</td><td rowspan=1 colspan=1>Acinetobacter spp.</td><td rowspan=1 colspan=1>(not prov.)</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>AST: sensitive</td></tr><tr><td rowspan=1 colspan=1>K. pneumoniae</td><td rowspan=1 colspan=1>K. pneumoniae</td><td rowspan=1 colspan=1>(not prov.)</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>AST: resistant</td></tr><tr><td rowspan=2 colspan=1>a</td><td rowspan=2 colspan=1>kpc</td><td rowspan=1 colspan=1>C. freundii</td><td rowspan=1 colspan=1>C. freundii</td><td rowspan=1 colspan=1>(not prov.)</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>AST: NA</td></tr><tr><td rowspan=1 colspan=1>P. aeruginosa</td><td rowspan=1 colspan=1>P. aeruginosa</td><td rowspan=1 colspan=1>(not prov.)</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>AST: NA</td></tr></table>

For K. pneumoniae, SoC reported a concentration $< 1 0 ^ { 3 }$ CFU/mL (considered a negative, ‘subclinical’ result below the applied reference threshold for mini-BAL specimens).

Table 50: Carbapenem Resistance for Acinetobacter spp. (oxa-23, oxa-24, oxa-58) Correlation of positive results for oxa-23, oxa-24, oxa-58, reported by LRT BAL for the prospective (‘p’) or archived (‘a’) study to available Acinetobacter spp. isolates (genotypic agreement, ‘yes’: presence of any of these markers confirmed by sequencing, ‘no’: presence not confirmed, ‘not prov.’: no isolate provided) and to AST results indicating carbapenem resistance for isolates with confirmed marker (phenotypic agreement; R: resistant phenotype, S: sensitive phenotype, NA: AST results not reported).

Genotypic Agreement for Specimens with at Least One Available Isolate: 6/6, $1 0 0 . 0 \%$ $9 5 \%$ CI: 61.0 - 100.0)

Phenotypic Agreement for Isolates with Confirmed Marker: 6/6, $1 0 0 . 0 \%$ ( $9 5 \%$ CI: 61.0 - 100.0)

<table><tr><td rowspan=1 colspan=1>Study</td><td rowspan=1 colspan=1>Marker</td><td rowspan=1 colspan=1>LRT BAL Result(relevant hosts only)</td><td rowspan=1 colspan=1>SoC Result(relevant hosts only)</td><td rowspan=1 colspan=1>GenotypicAgreement(for availableisolates)</td><td rowspan=1 colspan=1>PhenotypicAgreement(isolateswith conf.marker)</td><td rowspan=1 colspan=1>Comment</td></tr><tr><td rowspan=1 colspan=1>p</td><td rowspan=1 colspan=1>oxa-23</td><td rowspan=1 colspan=1>Acinetobacter spp.</td><td rowspan=1 colspan=1>Acinetobacter spp.</td><td rowspan=1 colspan=1>yes</td><td rowspan=1 colspan=1>R</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>a</td><td rowspan=1 colspan=1>oxa-23</td><td rowspan=1 colspan=1>Acinetobacter spp.</td><td rowspan=1 colspan=1>Acinetobacter spp.</td><td rowspan=1 colspan=1>yes</td><td rowspan=1 colspan=1>R</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>p</td><td rowspan=1 colspan=1>oxa-24</td><td rowspan=1 colspan=1>Acinetobacter spp.</td><td rowspan=1 colspan=1>Acinetobacter spp.</td><td rowspan=1 colspan=1>yes</td><td rowspan=1 colspan=1>R</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>b</td><td rowspan=1 colspan=1>oxa-24</td><td rowspan=1 colspan=1>Acinetobacter spp.</td><td rowspan=1 colspan=1>Acinetobacter spp.</td><td rowspan=1 colspan=1>yes</td><td rowspan=1 colspan=1>R a</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>a</td><td rowspan=1 colspan=1>oxa-23</td><td rowspan=1 colspan=1>Acinetobacter spp.</td><td rowspan=1 colspan=1>Acinetobacter spp.</td><td rowspan=1 colspan=1>yes</td><td rowspan=1 colspan=1>R</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>b</td><td rowspan=1 colspan=1>oxa-24</td><td rowspan=1 colspan=1>Acinetobacter spp.</td><td rowspan=1 colspan=1>Acinetobacter spp.</td><td rowspan=1 colspan=1>yes</td><td rowspan=1 colspan=1>R</td><td rowspan=1 colspan=1>-</td></tr></table>

Other applicable specimens without available Acinetobacter spp. isolate:   

<table><tr><td rowspan=1 colspan=1>a</td><td rowspan=1 colspan=1>oxa-24</td><td rowspan=1 colspan=1>Acinetobacter spp.</td><td rowspan=1 colspan=1>Acinetobacter spp.</td><td rowspan=1 colspan=1>(not prov.)</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>AST: resistant</td></tr><tr><td rowspan=1 colspan=1>a</td><td rowspan=1 colspan=1>oxa-24</td><td rowspan=1 colspan=1>Acinetobacter spp.</td><td rowspan=1 colspan=1>Acinetobacter spp.</td><td rowspan=1 colspan=1>(not prov.)</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>AST: resistant</td></tr><tr><td rowspan=1 colspan=1>a</td><td rowspan=1 colspan=1>oxa-24</td><td rowspan=1 colspan=1>Acinetobacter spp.</td><td rowspan=1 colspan=1>Acinetobacter spp.</td><td rowspan=1 colspan=1>(not prov.)</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>AST: sensitive</td></tr><tr><td rowspan=1 colspan=1>a</td><td rowspan=1 colspan=1>oxa-23</td><td rowspan=1 colspan=1>Acinetobacter spp.</td><td rowspan=1 colspan=1>Acinetobacter spp.</td><td rowspan=1 colspan=1>(not prov.)</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>AST: NA</td></tr><tr><td rowspan=1 colspan=1>a</td><td rowspan=1 colspan=1>oxa-23</td><td rowspan=1 colspan=1>Acinetobacter spp.</td><td rowspan=1 colspan=1>Acinetobacter spp.</td><td rowspan=1 colspan=1>(not prov.)</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>AST: NA</td></tr><tr><td rowspan=1 colspan=1>b</td><td rowspan=1 colspan=1>oxa-23</td><td rowspan=1 colspan=1>Acinetobacter spp.</td><td rowspan=1 colspan=1>Acinetobacter spp.</td><td rowspan=1 colspan=1>(not prov.)</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>AST: NA</td></tr><tr><td rowspan=1 colspan=1>a</td><td rowspan=1 colspan=1>oxa-58</td><td rowspan=1 colspan=1>Acinetobacter spp.</td><td rowspan=1 colspan=1>Acinetobacter spp.</td><td rowspan=1 colspan=1>(not prov.)</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>AST: NA</td></tr><tr><td rowspan=1 colspan=1>p</td><td rowspan=1 colspan=1>oxa-23</td><td rowspan=1 colspan=1>Acinetobacter spp.</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>p</td><td rowspan=1 colspan=1>oxa-24</td><td rowspan=1 colspan=1>Acinetobacter spp.</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr></table>

a For one prospective strain, a resistant phenotype to meropenem was reported by the clinical site, although this was not supported by the provided Kirby-Bauer zone diameter. Independent AST for meropenem and imipenem confirmed resistance to carbapenems for this strain.

# 5. Unyvero System Performance: Invalid Results and Instrument Errors

For the prospective, archived and contrived studies, a total of 1,648 clinical BAL specimens (1,016 prospective, 392 archived, 240 contrived) were processed. Results for all successful runs (reporting fully valid or partially valid analyte results) were evaluated; however, runs with partially valid result were repeated once, if sufficient specimen volume was still available in order to achieve a complete dataset. Results for repeat runs replaced the results of the predecessor run for data evaluation. Furthermore, all failed runs or runs, for which all analytes were reported invalid, were repeated whenever possible.

The invalid rate for all clinical studies combined was $3 . 9 \%$ (64 of 1,648 specimens). A total of eight of 1,648 runs generated invalid results for all analytes $( 0 . 5 \% )$ ; seven of the initially invalid specimens generated valid results upon repeat testing. For one specimen, all analytes were reported as invalid from the repeat test. A total of 56 of 1,648 runs generated partially invalid results $( 3 . 4 \% )$ ; 55 specimens generated a single failed PCR/hybridization chamber of the LRT BAL Cartridge. For one specimen, the invalid result was due to multiple chambers failures. Repeat runs were performed for 48 of invalid runs resulting in valid results for all repeat test runs.

Another 14 runs failed due to instrument error $( 0 . 8 \% )$ . All 14 runs were successfully repeated.

# D Clinical Cut-Off:

Not Applicable

# E Expected Values/Reference Range:

In the prospective study cohort of 1,016 evaluable BAL specimens, the LRT BAL Application reported 363 specimens $( 3 5 . 7 \% )$ with at least one positive LRT BAL microorganism target, including 250 specimens $( 2 4 . 6 \% )$ with single detections and 113 specimens $( 1 1 . 1 \% )$ with multidetections of two or more LRT BAL microorganism targets. Table 51 includes the positivity rate for each microorganism target observed in the prospective clinical study. Tables 52 and 53 include the positivity rate for antibiotic resistance marker targets as reported as positive by LRT BAL Application during the prospective study.

Table 51: Expected Values, Prospective Study $\mathbf { ( N = 1 , 0 1 6 ) }$ , Microorganism targets, by single-detection $( \mathbf { N } = 2 5 \mathbf { 0 } )$ or multi-detection specimens $( \mathbf { N } = \mathbf { 1 1 3 } )$ .   

<table><tr><td rowspan=1 colspan=9>Expected Values (N = 1,016 Prospective Specimens)</td></tr><tr><td rowspan=1 colspan=1>Microorganism</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1># Spec.</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=2>Microorganism</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1># Spec.</td><td rowspan=1 colspan=1>%</td></tr><tr><td rowspan=3 colspan=1>Acinetobacter spp.</td><td rowspan=1 colspan=1>all</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>2.1%</td><td rowspan=3 colspan=2>Moraxella catarrhalis</td><td rowspan=1 colspan=1>all</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>1.5%</td></tr><tr><td rowspan=1 colspan=1>single</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>0.8%</td><td rowspan=1 colspan=1>single</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>0.4%</td></tr><tr><td rowspan=1 colspan=1>multi</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>1.3%</td><td rowspan=1 colspan=1>multi</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>1.1%</td></tr><tr><td rowspan=3 colspan=1>Chlamydia pneumoniae</td><td rowspan=1 colspan=1>all</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=3 colspan=2>Morganella morganii</td><td rowspan=1 colspan=1>all</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0.3%</td></tr><tr><td rowspan=1 colspan=1>single</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>single</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%</td></tr><tr><td rowspan=1 colspan=1>multi</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>multi</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0.3%</td></tr><tr><td rowspan=3 colspan=1>Citrobacter freundii</td><td rowspan=1 colspan=1>all</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>0.4%</td><td rowspan=2 colspan=2>Mycoplasma pneumoniae</td><td rowspan=1 colspan=1>all</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>0.6%</td></tr><tr><td rowspan=1 colspan=1>single</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.1%</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>single</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>0.4%</td></tr><tr><td rowspan=1 colspan=1>multi</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0.3%</td><td rowspan=1 colspan=2></td><td rowspan=1 colspan=1>multi</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0.2%</td></tr><tr><td rowspan=3 colspan=1>Enterobacter cloacae complex</td><td rowspan=1 colspan=1>all</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>2.0%</td><td rowspan=3 colspan=2>Pneumocystis jirovecii</td><td rowspan=1 colspan=1>all</td><td rowspan=1 colspan=1>22</td><td rowspan=1 colspan=1>2.2%</td></tr><tr><td rowspan=1 colspan=1>single</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>0.8%</td><td rowspan=1 colspan=1>single</td><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>1.4%</td></tr><tr><td rowspan=1 colspan=1>multi</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>1.2%</td><td rowspan=1 colspan=1>multi</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>0.8%</td></tr><tr><td rowspan=3 colspan=1>Escherichia coli</td><td rowspan=1 colspan=1>all</td><td rowspan=1 colspan=1>47</td><td rowspan=1 colspan=1>4.6%</td><td rowspan=3 colspan=2>Proteus spp.</td><td rowspan=1 colspan=1>all</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>1.0%</td></tr><tr><td rowspan=1 colspan=1>single</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>2.4%</td><td rowspan=1 colspan=1>single</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%</td></tr><tr><td rowspan=1 colspan=1>multi</td><td rowspan=1 colspan=1>23</td><td rowspan=1 colspan=1>2.3%</td><td rowspan=1 colspan=1>multi</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>1.0%</td></tr><tr><td rowspan=3 colspan=1>Haemophilus influenzae</td><td rowspan=1 colspan=1>all</td><td rowspan=1 colspan=1>56</td><td rowspan=1 colspan=1>5.5%</td><td rowspan=3 colspan=2>Pseudomonas aeruginosa</td><td rowspan=1 colspan=1>all</td><td rowspan=1 colspan=1>112</td><td rowspan=1 colspan=1>11.0%</td></tr><tr><td rowspan=1 colspan=1>single</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>3.0%</td><td rowspan=1 colspan=1>single</td><td rowspan=1 colspan=1>61</td><td rowspan=1 colspan=1>6.0%</td></tr><tr><td rowspan=1 colspan=1>multi</td><td rowspan=1 colspan=1>26</td><td rowspan=1 colspan=1>2.6%</td><td rowspan=1 colspan=1>multi</td><td rowspan=1 colspan=1>51</td><td rowspan=1 colspan=1>5.0%</td></tr><tr><td rowspan=3 colspan=1>Klebsiella oxytoca</td><td rowspan=1 colspan=1>all</td><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>1.4%</td><td rowspan=3 colspan=2>Serratia marcescens</td><td rowspan=1 colspan=1>all</td><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>1.7%</td></tr><tr><td rowspan=1 colspan=1>single</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0.2%</td><td rowspan=1 colspan=1>single</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>0.5%</td></tr><tr><td rowspan=1 colspan=1>multi</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>1.2%</td><td rowspan=1 colspan=1>multi</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>1.2%</td></tr><tr><td rowspan=3 colspan=1>Klebsiella pneumoniae</td><td rowspan=1 colspan=1>all</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>3.0%</td><td rowspan=3 colspan=2>Staphylococcus aureus</td><td rowspan=1 colspan=1>all</td><td rowspan=1 colspan=1>104</td><td rowspan=1 colspan=1>10.2%</td></tr><tr><td rowspan=1 colspan=1>single</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>0.9%</td><td rowspan=1 colspan=1>single</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>5.9%</td></tr><tr><td rowspan=1 colspan=1>multi</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>2.1%</td><td rowspan=1 colspan=1>multi</td><td rowspan=1 colspan=1>44</td><td rowspan=1 colspan=1>4.3%</td></tr><tr><td rowspan=3 colspan=1>Klebsiella variicola</td><td rowspan=1 colspan=1>all</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0.2%</td><td rowspan=3 colspan=2>Stenotrophomonasmaltophilia</td><td rowspan=1 colspan=1>all</td><td rowspan=1 colspan=1>41</td><td rowspan=1 colspan=1>4.0%</td></tr><tr><td rowspan=1 colspan=1>single</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>single</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>1.2%</td></tr><tr><td rowspan=1 colspan=1>multi</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0.2%</td><td rowspan=1 colspan=1>multi</td><td rowspan=1 colspan=1>29</td><td rowspan=1 colspan=1>2.9%</td></tr><tr><td rowspan=3 colspan=1>Legionella pneumophila</td><td rowspan=1 colspan=1>all</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.1%</td><td rowspan=3 colspan=2>Streptococcus pneumoniae</td><td rowspan=1 colspan=1>all</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>1.3%</td></tr><tr><td rowspan=1 colspan=1>single</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.1%</td><td rowspan=1 colspan=1>single</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>0.7%</td></tr><tr><td rowspan=1 colspan=1>multi</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>multi</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>0.6%</td></tr></table>

Table 52: Expected values, Prospective Study, Resistance Markers   

<table><tr><td rowspan=1 colspan=1>ResistanceMarker</td><td rowspan=1 colspan=2>Expected Values, AllSpecimens (N = 1,016)</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1># Spec.</td><td rowspan=1 colspan=1>%</td></tr><tr><td rowspan=1 colspan=1>ctx-M</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>0.9%</td></tr><tr><td rowspan=1 colspan=1>kpc</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>0.4%</td></tr><tr><td rowspan=1 colspan=1>mecA</td><td rowspan=1 colspan=1>47</td><td rowspan=1 colspan=1>4.6%</td></tr><tr><td rowspan=1 colspan=1>ndm</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.1%</td></tr><tr><td rowspan=1 colspan=1>oxa-23</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0.3%</td></tr><tr><td rowspan=1 colspan=1>oxa-24</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>0.4%</td></tr><tr><td rowspan=1 colspan=1>oxa-48</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.1%</td></tr><tr><td rowspan=1 colspan=1>oxa-58</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%</td></tr><tr><td rowspan=1 colspan=1>tem</td><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>1.6%</td></tr><tr><td rowspan=1 colspan=1>vim</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.1%</td></tr></table>

Table 53: Expected values, Prospective Study, Resistance Markers Stratified by Detected Microorganisms   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>ctx-M</td><td rowspan=1 colspan=1>kpc</td><td rowspan=1 colspan=1>ndm</td><td rowspan=1 colspan=1>0xa-23</td><td rowspan=1 colspan=1>oxa-24</td><td rowspan=1 colspan=1>oxa-48</td><td rowspan=1 colspan=1>oxa-58</td><td rowspan=1 colspan=1>vim</td><td rowspan=1 colspan=1>tem</td><td rowspan=1 colspan=1>mecA</td></tr><tr><td rowspan=1 colspan=1>total reported</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>47</td></tr><tr><td rowspan=1 colspan=1>multi host detections</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>Acinetobacter spp. (N = 21)</td><td rowspan=1 colspan=1>2/219.5%</td><td rowspan=1 colspan=1>2/219.5%</td><td rowspan=1 colspan=1>1/214.8%</td><td rowspan=1 colspan=1>3/2114.3%</td><td rowspan=1 colspan=1>4/2119.0%</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>C. freundii (N = 4)</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>E. cloacae complex (N = 20)</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>1/205.0%</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>E. coli (N = 47)</td><td rowspan=1 colspan=1>8/4717.0%</td><td rowspan=1 colspan=1>1/472.1%</td><td rowspan=1 colspan=1>1/472.1%</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1/472.1%</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>K. oxytoca (N = 14)</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>K. pneumoniae (N = 30)</td><td rowspan=1 colspan=1>4/3013.3%</td><td rowspan=1 colspan=1>4/3013.3%</td><td rowspan=1 colspan=1>1/303.3%</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1/303.3%</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>K. variicola (N = 2)</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>M. morganii (N = 3)</td><td rowspan=1 colspan=1>1/333.3%</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Proteus spp. (N = 10)</td><td rowspan=1 colspan=1>2/1020.0%</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1/1010.0%</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>P. aeruginosa (N = 112)</td><td rowspan=1 colspan=1>7/1126.3%</td><td rowspan=1 colspan=1>2/1121.8%</td><td rowspan=1 colspan=1>1/1120.9%</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1/1120.9%</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>S. marcescens (N = 17)</td><td rowspan=1 colspan=1>2/1711.8%</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>H. influenzae (N = 56)</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>16/5628.6%</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>S. aureus (N = 104)</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>47/10445.2%</td></tr></table>

Specimens reported by the Unyvero LRT BAL Application as negative, positive for one microorganism target or positive for multiple microorganism targets, together with reported resistance markers are listed in Table 54.

Table 54: Expected values, Resistance Marker Targets, Stratified by Single and Multiple Detections of Microorganism Targets   

<table><tr><td colspan="2" rowspan="2"></td><td colspan="12" rowspan="1">Antibiotic Resistance Marker Result*</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="6" rowspan="1">Single Marker Detected</td><td colspan="5" rowspan="1">Multiple Markers Detected</td></tr><tr><td colspan="1" rowspan="1">MicroorganismResult</td><td colspan="1" rowspan="1"># cases</td><td colspan="1" rowspan="1">none</td><td colspan="1" rowspan="1">tem</td><td colspan="1" rowspan="1">ctx-M</td><td colspan="1" rowspan="1">kpc</td><td colspan="1" rowspan="1">oxa-23</td><td colspan="1" rowspan="1">oxa-24</td><td colspan="1" rowspan="1">mecA</td><td colspan="1" rowspan="1">ctx-M,mecA</td><td colspan="1" rowspan="1">tem,vim</td><td colspan="1" rowspan="1">ctx-M,mecA,oxa-23</td><td colspan="1" rowspan="1">ctx-M,ndm,oxa-48</td><td colspan="1" rowspan="1">kpc,mecA,oxa-24</td></tr><tr><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">653(64.3%)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Single OrganismDetected</td><td colspan="1" rowspan="1">250(24.6%)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Acinetobacter spp.</td><td colspan="1" rowspan="1">8</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">C. freundii</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">E. cloacae complex</td><td colspan="1" rowspan="1">8</td><td colspan="1" rowspan="1">8</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">24</td><td colspan="1" rowspan="1">24</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">H. influenzae</td><td colspan="1" rowspan="1">30</td><td colspan="1" rowspan="1">26</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">K. oxytoca</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">K. pneumoniae</td><td colspan="1" rowspan="1">9</td><td colspan="1" rowspan="1">9</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">L. pneumophila</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">M. catarrhalis</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">M. pneumoniae</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">P. jirovecii</td><td colspan="1" rowspan="1">14</td><td colspan="1" rowspan="1">14</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">P. aeruginosa</td><td colspan="1" rowspan="1">61</td><td colspan="1" rowspan="1">61</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">S. marcescens</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">:</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1">60</td><td colspan="1" rowspan="1">32</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">28</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">S. maltophilia</td><td colspan="1" rowspan="1">12</td><td colspan="1" rowspan="1">12</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">S. pneumoniae</td><td colspan="1" rowspan="1">7</td><td colspan="1" rowspan="1">7</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Two OrganismsDetected</td><td colspan="1" rowspan="1">78(7.7%)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Acinetobacter spp.,</td><td colspan="1" rowspan="2">1</td><td colspan="1" rowspan="2">1</td><td colspan="1" rowspan="2">.</td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2">.</td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2">-</td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2">.</td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2">-</td></tr><tr><td colspan="1" rowspan="1">C. freundii</td></tr><tr><td colspan="1" rowspan="1">Acinetobacter spp.,</td><td colspan="1" rowspan="2">1</td><td colspan="1" rowspan="2">1</td><td colspan="1" rowspan="2">-</td><td colspan="1" rowspan="2">-</td><td colspan="1" rowspan="2">.</td><td colspan="1" rowspan="2">-</td><td colspan="1" rowspan="2">-</td><td colspan="1" rowspan="2">-</td><td colspan="1" rowspan="2">-</td><td colspan="1" rowspan="2">-</td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2">-</td></tr><tr><td colspan="1" rowspan="1">E. cloacae complex</td></tr><tr><td colspan="1" rowspan="1">Acinetobacter spp.,</td><td colspan="1" rowspan="2">1</td><td colspan="1" rowspan="2">1</td><td colspan="1" rowspan="2">-</td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2">-</td><td colspan="1" rowspan="2">-</td><td colspan="1" rowspan="2">-</td><td colspan="1" rowspan="2">-</td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2">-</td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2">-</td></tr><tr><td colspan="1" rowspan="1">K. pneumoniae</td></tr><tr><td colspan="1" rowspan="1">Acinetobacter spp.,</td><td colspan="1" rowspan="2">1</td><td colspan="1" rowspan="2">1</td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2">-</td><td colspan="1" rowspan="2">-</td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2">-</td></tr><tr><td colspan="1" rowspan="1">P. aeruginosa</td></tr><tr><td colspan="1" rowspan="1">Acinetobacter spp.,</td><td colspan="1" rowspan="2">1</td><td colspan="1" rowspan="2">1</td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2">-</td><td colspan="1" rowspan="2">-</td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2">-</td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2">-</td></tr><tr><td colspan="1" rowspan="1">S. maltophilia</td></tr><tr><td colspan="1" rowspan="1">C. freundii,</td><td colspan="1" rowspan="2">1</td><td colspan="1" rowspan="2">1</td><td colspan="1" rowspan="2">-</td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2">.</td><td colspan="1" rowspan="2">-</td><td colspan="1" rowspan="2">-</td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2">-</td></tr><tr><td colspan="1" rowspan="1">P. aeruginosa</td></tr><tr><td colspan="1" rowspan="1">E. cloacae complex,</td><td colspan="1" rowspan="2">1</td><td colspan="1" rowspan="2">1</td><td colspan="1" rowspan="2">-</td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2">-</td><td colspan="1" rowspan="2">-</td><td colspan="1" rowspan="2">-</td><td colspan="1" rowspan="2">-</td><td colspan="1" rowspan="2">-</td><td colspan="1" rowspan="2">-</td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2">-</td><td colspan="1" rowspan="2">-</td></tr><tr><td colspan="1" rowspan="1">M. pneumoniae</td></tr><tr><td colspan="1" rowspan="1">E. cloacae complex,</td><td colspan="1" rowspan="2">2</td><td colspan="1" rowspan="2">2</td><td colspan="1" rowspan="2">-</td><td colspan="1" rowspan="2">-</td><td colspan="1" rowspan="2">-</td><td colspan="1" rowspan="2">-</td><td colspan="1" rowspan="2">-</td><td colspan="1" rowspan="2">-</td><td colspan="1" rowspan="2">-</td><td colspan="1" rowspan="2">-</td><td colspan="1" rowspan="2">-</td><td colspan="1" rowspan="2">-</td><td colspan="1" rowspan="2">-</td></tr><tr><td colspan="1" rowspan="1">S. aureus</td></tr><tr><td colspan="1" rowspan="1">E. coli,</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">H. influenzae</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">E. coli,</td><td colspan="1" rowspan="2">1</td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2">-</td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2">-</td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2">-</td></tr><tr><td colspan="1" rowspan="1">K. oxytoca</td></tr><tr><td colspan="1" rowspan="1">E. coli,</td><td colspan="1" rowspan="2">1</td><td colspan="1" rowspan="2">1</td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2">-</td><td colspan="1" rowspan="2">-</td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2">-</td><td colspan="1" rowspan="2">-</td></tr><tr><td colspan="1" rowspan="1">K. pneumoniae</td></tr><tr><td colspan="1" rowspan="1">E. coli,</td><td colspan="1" rowspan="2">1</td><td colspan="1" rowspan="2">0</td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2">-</td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2">-</td></tr><tr><td colspan="1" rowspan="1">P. aeruginosa</td></tr><tr><td colspan="1" rowspan="1">E. coli,</td><td colspan="1" rowspan="2">2</td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2">-</td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2">-</td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2">-</td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td></tr><tr><td colspan="1" rowspan="1">S. marcescens</td></tr><tr><td colspan="1" rowspan="1">E. coli,</td><td colspan="1" rowspan="2">2</td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2">-</td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2">-</td></tr><tr><td colspan="1" rowspan="1">S. aureus</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">H. influenzae,</td><td colspan="1" rowspan="2">1</td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td></tr><tr><td colspan="1" rowspan="1">K. oxytoca</td></tr><tr><td colspan="1" rowspan="1">H. influenzae,</td><td colspan="1" rowspan="2">4</td><td colspan="1" rowspan="2">3</td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2">-</td><td colspan="1" rowspan="2">−</td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2">-</td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td></tr><tr><td colspan="1" rowspan="1">M. catarrhalis</td></tr><tr><td colspan="1" rowspan="1">H. influenzae,</td><td colspan="1" rowspan="2">1</td><td colspan="1" rowspan="2">0</td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2">-</td></tr><tr><td colspan="1" rowspan="1">M. pneumoniae</td></tr><tr><td colspan="1" rowspan="1">H. influenzae,</td><td colspan="1" rowspan="2">3</td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2">2</td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2">-</td></tr><tr><td colspan="1" rowspan="1">P. aeruginosa</td></tr><tr><td colspan="1" rowspan="1">H. influenzae,</td><td colspan="1" rowspan="2">4</td><td colspan="1" rowspan="2">2</td><td colspan="1" rowspan="2">2</td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2">-</td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2">-</td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2">-</td><td colspan="1" rowspan="2">-</td><td colspan="1" rowspan="2">-</td><td colspan="1" rowspan="2">-</td></tr><tr><td colspan="1" rowspan="1">S. aureus</td></tr><tr><td colspan="1" rowspan="1">H. influenzae,</td><td colspan="1" rowspan="2">3</td><td colspan="1" rowspan="2">1</td><td colspan="1" rowspan="2">2</td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td></tr><tr><td colspan="1" rowspan="1">S. pneumoniae</td></tr><tr><td colspan="1" rowspan="1">K. oxytoca,</td><td colspan="1" rowspan="2">1</td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td></tr><tr><td colspan="1" rowspan="1">K. pneumoniae</td></tr><tr><td colspan="1" rowspan="1">K. oxytoca,</td><td colspan="1" rowspan="2">1</td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td></tr><tr><td colspan="1" rowspan="1">Proteus spp.</td></tr><tr><td colspan="1" rowspan="1">K. oxytoca,</td><td colspan="1" rowspan="2">1</td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2">-</td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2">-</td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2">-</td><td colspan="1" rowspan="2">-</td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2">-</td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2">-</td></tr><tr><td colspan="1" rowspan="1">S. marcescens</td></tr><tr><td colspan="1" rowspan="1">K. oxytoca,</td><td colspan="1" rowspan="2">1</td><td colspan="1" rowspan="2">1</td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2">-</td></tr><tr><td colspan="1" rowspan="1">S. aureus</td></tr><tr><td colspan="1" rowspan="1">K. oxytoca,</td><td colspan="1" rowspan="2">1</td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2">-</td></tr><tr><td colspan="1" rowspan="1">S. maltophilia</td></tr><tr><td colspan="1" rowspan="1">K. pneumoniae,</td><td colspan="1" rowspan="2">2</td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td></tr><tr><td colspan="1" rowspan="1">P. aeruginosa</td></tr><tr><td colspan="1" rowspan="1">K. pneumoniae,</td><td colspan="1" rowspan="2">3</td><td colspan="1" rowspan="2">2</td><td colspan="1" rowspan="2">-</td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2">-</td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2">1</td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2">-</td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2">-</td></tr><tr><td colspan="1" rowspan="1">S. aureus</td></tr><tr><td colspan="1" rowspan="1">M. catarrhalis,</td><td colspan="1" rowspan="2">1</td><td colspan="1" rowspan="2">1</td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2">-</td></tr><tr><td colspan="1" rowspan="1">P. jirovecii</td></tr><tr><td colspan="1" rowspan="1">M. catarrhalis,</td><td colspan="1" rowspan="2">1</td><td colspan="1" rowspan="2">1</td><td colspan="1" rowspan="2">-</td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2">-</td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2">-</td><td colspan="1" rowspan="2">-</td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2">-</td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2">-</td><td colspan="1" rowspan="2">-</td></tr><tr><td colspan="1" rowspan="1">S. pneumoniae</td></tr><tr><td colspan="1" rowspan="1">P. jirovecii,</td><td colspan="1" rowspan="2">1</td><td colspan="1" rowspan="2">1</td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2">-</td><td colspan="1" rowspan="2">-</td><td colspan="1" rowspan="2">-</td><td colspan="1" rowspan="2">-</td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2">-</td><td colspan="1" rowspan="2">-</td></tr><tr><td colspan="1" rowspan="1">P. aeruginosa</td></tr><tr><td colspan="1" rowspan="1">P. jirovecii,</td><td colspan="1" rowspan="2">1</td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2">-</td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2">-</td><td colspan="1" rowspan="2">−</td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2">-</td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2">-</td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2">-</td><td colspan="1" rowspan="2">-</td></tr><tr><td colspan="1" rowspan="1">S. aureus</td></tr><tr><td colspan="1" rowspan="1">P. jirovecii,</td><td colspan="1" rowspan="2">2</td><td colspan="1" rowspan="2">2</td><td colspan="1" rowspan="2">-</td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2">−</td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2">-</td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2">-</td><td colspan="1" rowspan="2">-</td></tr><tr><td colspan="1" rowspan="1">S. maltophilia</td></tr><tr><td colspan="1" rowspan="1">Proteus spp.,</td><td colspan="1" rowspan="2">1</td><td colspan="1" rowspan="2">1</td><td colspan="1" rowspan="2">-</td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2">-</td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2">-</td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2">-</td></tr><tr><td colspan="1" rowspan="1">P. aeruginosa</td></tr><tr><td colspan="1" rowspan="1">Proteus spp.,</td><td colspan="1" rowspan="2">1</td><td colspan="1" rowspan="2">0</td><td colspan="1" rowspan="2">-</td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2">−</td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2">1</td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2">-</td><td colspan="1" rowspan="2"></td></tr><tr><td colspan="1" rowspan="1">S. aureus</td></tr><tr><td colspan="1" rowspan="1">P. aeruginosa,</td><td colspan="1" rowspan="2">1</td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2">-</td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2">-</td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2">-</td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td></tr><tr><td colspan="1" rowspan="1">S. marcescens</td></tr><tr><td colspan="1" rowspan="1">P. aeruginosa,</td><td colspan="1" rowspan="2">14</td><td colspan="1" rowspan="2">4</td><td colspan="1" rowspan="2">-</td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2">−</td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2">10</td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2">-</td><td colspan="1" rowspan="2"></td></tr><tr><td colspan="1" rowspan="1">S. aureus</td></tr><tr><td colspan="1" rowspan="1">P. aeruginosa,</td><td colspan="1" rowspan="2">6</td><td colspan="1" rowspan="2">6</td><td colspan="1" rowspan="2">-</td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2">-</td><td colspan="1" rowspan="2">−</td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2">-</td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2">-</td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2">-</td><td colspan="1" rowspan="2"></td></tr><tr><td colspan="1" rowspan="1">S. maltophilia</td></tr><tr><td colspan="1" rowspan="1">S. marcescens,</td><td colspan="1" rowspan="2">1</td><td colspan="1" rowspan="2">0</td><td colspan="1" rowspan="2">-</td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2">-</td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2">1</td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td></tr><tr><td colspan="1" rowspan="1">S. aureus</td></tr><tr><td colspan="1" rowspan="1">S. aureus,</td><td colspan="1" rowspan="2">5</td><td colspan="1" rowspan="2">3</td><td colspan="1" rowspan="2">-</td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2">-</td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2">2</td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td></tr><tr><td colspan="1" rowspan="1">S. maltophilia</td></tr><tr><td colspan="1" rowspan="1">Three OrganismsDetected</td><td colspan="1" rowspan="1">19(1.9%)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Acinetobacter spp.,</td><td colspan="1" rowspan="3">1</td><td colspan="1" rowspan="3">0</td><td colspan="1" rowspan="3">.</td><td colspan="1" rowspan="3"></td><td colspan="1" rowspan="3"></td><td colspan="1" rowspan="3"></td><td colspan="1" rowspan="3"></td><td colspan="1" rowspan="3"></td><td colspan="1" rowspan="3"></td><td colspan="1" rowspan="3"></td><td colspan="1" rowspan="3"></td><td colspan="1" rowspan="3"></td><td colspan="1" rowspan="3">1</td></tr><tr><td colspan="1" rowspan="1">K. pneumoniae,</td></tr><tr><td colspan="1" rowspan="1">S. aureus</td></tr><tr><td colspan="1" rowspan="1">Acinetobacter spp.,</td><td colspan="1" rowspan="3">1</td><td colspan="1" rowspan="3"></td><td colspan="1" rowspan="3">.</td><td colspan="1" rowspan="3">.</td><td colspan="1" rowspan="3"></td><td colspan="1" rowspan="3"></td><td colspan="1" rowspan="3"></td><td colspan="1" rowspan="3"></td><td colspan="1" rowspan="3"></td><td colspan="1" rowspan="3"> -</td><td colspan="1" rowspan="3">.</td><td colspan="1" rowspan="3"></td><td colspan="1" rowspan="3">.</td></tr><tr><td colspan="1" rowspan="1">K. pneumoniae,</td></tr><tr><td colspan="1" rowspan="1">S. maltophilia</td></tr><tr><td colspan="1" rowspan="1">Acinetobacter spp.,</td><td colspan="1" rowspan="3">1</td><td colspan="1" rowspan="3">0</td><td colspan="1" rowspan="3">.</td><td colspan="1" rowspan="3">.</td><td colspan="1" rowspan="3"></td><td colspan="1" rowspan="3"></td><td colspan="1" rowspan="3"></td><td colspan="1" rowspan="3"></td><td colspan="1" rowspan="3">.</td><td colspan="1" rowspan="3"></td><td colspan="1" rowspan="3">.</td><td colspan="1" rowspan="3"></td><td colspan="1" rowspan="3"></td></tr><tr><td colspan="1" rowspan="1">P. aeruginosa,</td></tr><tr><td colspan="1" rowspan="1">S. maltophilia</td></tr><tr><td colspan="1" rowspan="1">E. cloacae complex,</td><td colspan="1" rowspan="3">1</td><td colspan="1" rowspan="3"></td><td colspan="1" rowspan="3"></td><td colspan="1" rowspan="3"></td><td colspan="1" rowspan="3"></td><td colspan="1" rowspan="3"></td><td colspan="1" rowspan="3"></td><td colspan="1" rowspan="3"></td><td colspan="1" rowspan="3"></td><td colspan="1" rowspan="3"> </td><td colspan="1" rowspan="3">.</td><td colspan="1" rowspan="3"></td><td colspan="1" rowspan="3"></td></tr><tr><td colspan="1" rowspan="1">H. influenzae,</td></tr><tr><td colspan="1" rowspan="1">M. catarrhalis</td></tr><tr><td colspan="1" rowspan="1">E. cloacae complex,</td><td colspan="1" rowspan="3">1</td><td colspan="1" rowspan="3"></td><td colspan="1" rowspan="3"></td><td colspan="1" rowspan="3"></td><td colspan="1" rowspan="3">.</td><td colspan="1" rowspan="3"></td><td colspan="1" rowspan="3"></td><td colspan="1" rowspan="3"></td><td colspan="1" rowspan="3"></td><td colspan="1" rowspan="3"></td><td colspan="1" rowspan="3">.</td><td colspan="1" rowspan="3"></td><td colspan="1" rowspan="3"></td></tr><tr><td colspan="1" rowspan="1">K. oxytoca,</td></tr><tr><td colspan="1" rowspan="1">S. maltophilia</td></tr><tr><td colspan="1" rowspan="1">E. cloacae complex,</td><td colspan="1" rowspan="3">1</td><td colspan="1" rowspan="3"></td><td colspan="1" rowspan="3">.</td><td colspan="1" rowspan="3">.</td><td colspan="1" rowspan="3"></td><td colspan="1" rowspan="3"></td><td colspan="1" rowspan="3"></td><td colspan="1" rowspan="3"></td><td colspan="1" rowspan="3">.</td><td colspan="1" rowspan="3"></td><td colspan="1" rowspan="3">.</td><td colspan="1" rowspan="3"></td><td colspan="1" rowspan="3">.</td></tr><tr><td colspan="1" rowspan="1">K. pneumoniae,</td></tr><tr><td colspan="1" rowspan="1">M. catarrhalis</td></tr><tr><td colspan="1" rowspan="1">E. cloacae complex,</td><td colspan="1" rowspan="3">1</td><td colspan="1" rowspan="3">0</td><td colspan="1" rowspan="3">.</td><td colspan="1" rowspan="3">.</td><td colspan="1" rowspan="3"></td><td colspan="1" rowspan="3"></td><td colspan="1" rowspan="3"></td><td colspan="1" rowspan="3"></td><td colspan="1" rowspan="3"></td><td colspan="1" rowspan="3"></td><td colspan="1" rowspan="3">.</td><td colspan="1" rowspan="3"></td><td colspan="1" rowspan="3"></td></tr><tr><td colspan="1" rowspan="1">K. pneumoniae,</td></tr><tr><td colspan="1" rowspan="1">P. aeruginosa</td></tr><tr><td colspan="1" rowspan="1">E. coli,</td><td colspan="1" rowspan="3">1</td><td colspan="1" rowspan="3"></td><td colspan="1" rowspan="3">.</td><td colspan="1" rowspan="3"></td><td colspan="1" rowspan="3"></td><td colspan="1" rowspan="3">-</td><td colspan="1" rowspan="3">-</td><td colspan="1" rowspan="3">-</td><td colspan="1" rowspan="3"></td><td colspan="1" rowspan="3"></td><td colspan="1" rowspan="3"></td><td colspan="1" rowspan="3"></td><td colspan="1" rowspan="3">-</td></tr><tr><td colspan="1" rowspan="1">H. influenzae,</td></tr><tr><td colspan="1" rowspan="1">S. aureus</td></tr><tr><td colspan="1" rowspan="1">E. coli,</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">M. morganii,</td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2">−</td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2">−</td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2">−</td><td colspan="1" rowspan="2"></td></tr><tr><td colspan="1" rowspan="1">P. aeruginosa</td></tr><tr><td colspan="1" rowspan="1">E. coli,</td><td colspan="1" rowspan="3">1</td><td colspan="1" rowspan="3">0</td><td colspan="1" rowspan="3">.</td><td colspan="1" rowspan="3"></td><td colspan="1" rowspan="3">.</td><td colspan="1" rowspan="3"></td><td colspan="1" rowspan="3"></td><td colspan="1" rowspan="3">.</td><td colspan="1" rowspan="3">.</td><td colspan="1" rowspan="3">.</td><td colspan="1" rowspan="3">.</td><td colspan="1" rowspan="3"></td><td colspan="1" rowspan="3"></td></tr><tr><td colspan="1" rowspan="1">P. jirovecii,</td></tr><tr><td colspan="1" rowspan="1">P. aeruginosa</td></tr><tr><td colspan="1" rowspan="1">E. coli,</td><td colspan="1" rowspan="3">1</td><td colspan="1" rowspan="3"></td><td colspan="1" rowspan="3">.</td><td colspan="1" rowspan="3"></td><td colspan="1" rowspan="3">.</td><td colspan="1" rowspan="3"></td><td colspan="1" rowspan="3"></td><td colspan="1" rowspan="3">-</td><td colspan="1" rowspan="3"></td><td colspan="1" rowspan="3"></td><td colspan="1" rowspan="3">.</td><td colspan="1" rowspan="3"></td><td colspan="1" rowspan="3"></td></tr><tr><td colspan="1" rowspan="1">P. aeruginosa,</td></tr><tr><td colspan="1" rowspan="1">S. marcescens</td></tr><tr><td colspan="1" rowspan="1">E. coli,</td><td colspan="1" rowspan="3">1</td><td colspan="1" rowspan="3"></td><td colspan="1" rowspan="3">.</td><td colspan="1" rowspan="3">.</td><td colspan="1" rowspan="3">.</td><td colspan="1" rowspan="3"></td><td colspan="1" rowspan="3"></td><td colspan="1" rowspan="3">.</td><td colspan="1" rowspan="3">.</td><td colspan="1" rowspan="3"></td><td colspan="1" rowspan="3">.</td><td colspan="1" rowspan="3"></td><td colspan="1" rowspan="3"></td></tr><tr><td colspan="1" rowspan="1">P. aeruginosa,</td></tr><tr><td colspan="1" rowspan="1">S. aureus</td></tr><tr><td colspan="1" rowspan="1">H. influenzae,</td><td colspan="1" rowspan="3">1</td><td colspan="1" rowspan="3">I</td><td colspan="1" rowspan="3">.</td><td colspan="1" rowspan="3">.</td><td colspan="1" rowspan="3">.</td><td colspan="1" rowspan="3"></td><td colspan="1" rowspan="3"></td><td colspan="1" rowspan="3">.</td><td colspan="1" rowspan="3">.</td><td colspan="1" rowspan="3">.</td><td colspan="1" rowspan="3">.</td><td colspan="1" rowspan="3"></td><td colspan="1" rowspan="3"></td></tr><tr><td colspan="1" rowspan="1">M. catarrhalis,</td></tr><tr><td colspan="1" rowspan="1">S. pneumoniae</td></tr><tr><td colspan="1" rowspan="1">K. oxytoca,</td><td colspan="1" rowspan="3">1</td><td colspan="1" rowspan="3">I</td><td colspan="1" rowspan="3">.</td><td colspan="1" rowspan="3">.</td><td colspan="1" rowspan="3">.</td><td colspan="1" rowspan="3"></td><td colspan="1" rowspan="3"></td><td colspan="1" rowspan="3">.</td><td colspan="1" rowspan="3">.</td><td colspan="1" rowspan="3">.</td><td colspan="1" rowspan="3">.</td><td colspan="1" rowspan="3"></td><td colspan="1" rowspan="3"></td></tr><tr><td colspan="1" rowspan="1">P. aeruginosa,</td></tr><tr><td colspan="1" rowspan="1">S. aureus</td></tr><tr><td colspan="1" rowspan="1">K. oxytoca,</td><td colspan="1" rowspan="3">1</td><td colspan="1" rowspan="3">I</td><td colspan="1" rowspan="3">.</td><td colspan="1" rowspan="3">.</td><td colspan="1" rowspan="3">.</td><td colspan="1" rowspan="3"></td><td colspan="1" rowspan="3"></td><td colspan="1" rowspan="3">.</td><td colspan="1" rowspan="3">.</td><td colspan="1" rowspan="3">.</td><td colspan="1" rowspan="3">.</td><td colspan="1" rowspan="3"></td><td colspan="1" rowspan="3"></td></tr><tr><td colspan="1" rowspan="1">P. aeruginosa,</td></tr><tr><td colspan="1" rowspan="1">S. maltophilia</td></tr><tr><td colspan="1" rowspan="1">K. pneumoniae,</td><td colspan="1" rowspan="3">1</td><td colspan="1" rowspan="3">0</td><td colspan="1" rowspan="3">.</td><td colspan="1" rowspan="3">.</td><td colspan="1" rowspan="3"></td><td colspan="1" rowspan="3"></td><td colspan="1" rowspan="3"></td><td colspan="1" rowspan="3">.</td><td colspan="1" rowspan="3">.</td><td colspan="1" rowspan="3">.</td><td colspan="1" rowspan="3">.</td><td colspan="1" rowspan="3"></td><td colspan="1" rowspan="3"></td></tr><tr><td colspan="1" rowspan="1">P. aeruginosa,</td></tr><tr><td colspan="1" rowspan="1">S. maltophilia</td></tr><tr><td colspan="1" rowspan="1">Proteus spp.,</td><td colspan="1" rowspan="3">1</td><td colspan="1" rowspan="3">0</td><td colspan="1" rowspan="3">.</td><td colspan="1" rowspan="3">.</td><td colspan="1" rowspan="3">.</td><td colspan="1" rowspan="3"></td><td colspan="1" rowspan="3"></td><td colspan="1" rowspan="3">1</td><td colspan="1" rowspan="3">.</td><td colspan="1" rowspan="3"></td><td colspan="1" rowspan="3">.</td><td colspan="1" rowspan="3"></td><td colspan="1" rowspan="3"></td></tr><tr><td colspan="1" rowspan="1">P. aeruginosa,</td></tr><tr><td colspan="1" rowspan="1">S. aureus</td></tr><tr><td colspan="1" rowspan="1">P. aeruginosa,</td><td colspan="1" rowspan="3">2</td><td colspan="1" rowspan="3">2</td><td colspan="1" rowspan="3">.</td><td colspan="1" rowspan="3"></td><td colspan="1" rowspan="3">.</td><td colspan="1" rowspan="3"></td><td colspan="1" rowspan="3"></td><td colspan="1" rowspan="3">.</td><td colspan="1" rowspan="3"></td><td colspan="1" rowspan="3"></td><td colspan="1" rowspan="3">.</td><td colspan="1" rowspan="3"></td><td colspan="1" rowspan="3"></td></tr><tr><td colspan="1" rowspan="1">S. marcescens,</td></tr><tr><td colspan="1" rowspan="1">S. maltophilia</td></tr><tr><td colspan="1" rowspan="1">Four OrganismsDetected</td><td colspan="1" rowspan="1">7(0.7%)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Acinetobacter spp.,</td><td colspan="1" rowspan="4">1</td><td colspan="1" rowspan="4">0</td><td colspan="1" rowspan="4">.</td><td colspan="1" rowspan="4">.</td><td colspan="1" rowspan="4">.</td><td colspan="1" rowspan="4"></td><td colspan="1" rowspan="4"></td><td colspan="1" rowspan="4">.</td><td colspan="1" rowspan="4">.</td><td colspan="1" rowspan="4">.</td><td colspan="1" rowspan="4"></td><td colspan="1" rowspan="4"></td><td colspan="1" rowspan="4">.</td></tr><tr><td colspan="1" rowspan="1">E. coli,</td></tr><tr><td colspan="1" rowspan="1">K. pneumoniae,</td></tr><tr><td colspan="1" rowspan="1">P. aeruginosa</td></tr><tr><td colspan="1" rowspan="1">Acinetobacter spp.,</td><td colspan="1" rowspan="4">1</td><td colspan="1" rowspan="4">0</td><td colspan="1" rowspan="4">.</td><td colspan="1" rowspan="4"></td><td colspan="1" rowspan="4"></td><td colspan="1" rowspan="4"></td><td colspan="1" rowspan="4"></td><td colspan="1" rowspan="4"></td><td colspan="1" rowspan="4"></td><td colspan="1" rowspan="4">.</td><td colspan="1" rowspan="4"></td><td colspan="1" rowspan="4"></td><td colspan="1" rowspan="4"></td></tr><tr><td colspan="1" rowspan="1">E. coli,</td></tr><tr><td colspan="1" rowspan="1">K. pneumoniae,</td></tr><tr><td colspan="1" rowspan="1">S. maltophilia</td></tr><tr><td colspan="1" rowspan="1">Acinetobacter spp.,</td><td colspan="1" rowspan="4">1</td><td colspan="1" rowspan="4"></td><td colspan="1" rowspan="4">.</td><td colspan="1" rowspan="4"></td><td colspan="1" rowspan="4">.</td><td colspan="1" rowspan="4"></td><td colspan="1" rowspan="4"></td><td colspan="1" rowspan="4">.</td><td colspan="1" rowspan="4">.</td><td colspan="1" rowspan="4">.</td><td colspan="1" rowspan="4"></td><td colspan="1" rowspan="4">.</td><td colspan="1" rowspan="4"></td></tr><tr><td colspan="1" rowspan="1">E. coli,</td></tr><tr><td colspan="1" rowspan="1">Proteus spp.,</td></tr><tr><td colspan="1" rowspan="1">P. aeruginosa</td></tr><tr><td colspan="1" rowspan="1">E. cloacae complex,</td><td colspan="1" rowspan="2">1</td><td colspan="1" rowspan="2">1</td><td colspan="1" rowspan="2">-</td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2">-</td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2">-</td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2">-</td><td colspan="1" rowspan="2"></td></tr><tr><td colspan="1" rowspan="1">E. coli,</td></tr><tr><td colspan="1" rowspan="1">K. oxytoca,</td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2">−</td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2">−</td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2">−</td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td></tr><tr><td colspan="1" rowspan="1">S. maltophilia</td></tr><tr><td colspan="1" rowspan="1">E. coli,</td><td colspan="1" rowspan="4"></td><td colspan="1" rowspan="4"></td><td colspan="1" rowspan="4">.</td><td colspan="1" rowspan="4"></td><td colspan="1" rowspan="4"></td><td colspan="1" rowspan="4"></td><td colspan="1" rowspan="4"></td><td colspan="1" rowspan="4"></td><td colspan="1" rowspan="4">.</td><td colspan="1" rowspan="4"></td><td colspan="1" rowspan="4"></td><td colspan="1" rowspan="4"></td><td colspan="1" rowspan="4"></td></tr><tr><td colspan="1" rowspan="1">H. influenzae,</td></tr><tr><td colspan="1" rowspan="1">P. jirovecii,</td></tr><tr><td colspan="1" rowspan="1">S. aureus</td></tr><tr><td colspan="1" rowspan="1">H. influenzae,</td><td colspan="1" rowspan="4">1</td><td colspan="1" rowspan="4">0</td><td colspan="1" rowspan="4">.</td><td colspan="1" rowspan="4"></td><td colspan="1" rowspan="4">.</td><td colspan="1" rowspan="4"></td><td colspan="1" rowspan="4"></td><td colspan="1" rowspan="4">.</td><td colspan="1" rowspan="4">.</td><td colspan="1" rowspan="4"></td><td colspan="1" rowspan="4"></td><td colspan="1" rowspan="4"></td><td colspan="1" rowspan="4"></td></tr><tr><td colspan="1" rowspan="1">Proteus spp.,</td></tr><tr><td colspan="1" rowspan="1">P. aeruginosa,</td></tr><tr><td colspan="1" rowspan="1">S. maltophilia</td></tr><tr><td colspan="1" rowspan="1">P. aeruginosa,</td><td colspan="1" rowspan="4">1</td><td colspan="1" rowspan="4"></td><td colspan="1" rowspan="4"></td><td colspan="1" rowspan="4"></td><td colspan="1" rowspan="4">.</td><td colspan="1" rowspan="4"></td><td colspan="1" rowspan="4"></td><td colspan="1" rowspan="4"></td><td colspan="1" rowspan="4">.</td><td colspan="1" rowspan="4"></td><td colspan="1" rowspan="4"></td><td colspan="1" rowspan="4"></td><td colspan="1" rowspan="4"></td></tr><tr><td colspan="1" rowspan="1">S. marcescens,</td></tr><tr><td colspan="1" rowspan="1">S. aureus,</td></tr><tr><td colspan="1" rowspan="1">S. maltophilia</td></tr><tr><td colspan="1" rowspan="1">Five OrganismsDetected</td><td colspan="1" rowspan="1">7(0.7%)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Acinetobacter spp.,</td><td colspan="1" rowspan="5">1</td><td colspan="1" rowspan="5"></td><td colspan="1" rowspan="5">.</td><td colspan="1" rowspan="5"></td><td colspan="1" rowspan="5"></td><td colspan="1" rowspan="5"></td><td colspan="1" rowspan="5"></td><td colspan="1" rowspan="5">.</td><td colspan="1" rowspan="5"></td><td colspan="1" rowspan="5"></td><td colspan="1" rowspan="5"></td><td colspan="1" rowspan="5"></td><td colspan="1" rowspan="5"></td></tr><tr><td colspan="1" rowspan="1">E. cloacae complex,</td></tr><tr><td colspan="1" rowspan="1">H. influenzae,</td></tr><tr><td colspan="1" rowspan="1">K. pneumoniae,</td></tr><tr><td colspan="1" rowspan="1">S. pneumoniae</td></tr><tr><td colspan="1" rowspan="1">C. freundii,</td><td colspan="1" rowspan="5"></td><td colspan="1" rowspan="5"></td><td colspan="1" rowspan="5">.</td><td colspan="1" rowspan="5"></td><td colspan="1" rowspan="5"></td><td colspan="1" rowspan="5"></td><td colspan="1" rowspan="5"></td><td colspan="1" rowspan="5"></td><td colspan="1" rowspan="5"></td><td colspan="1" rowspan="5"></td><td colspan="1" rowspan="5"></td><td colspan="1" rowspan="5"></td><td colspan="1" rowspan="5"></td></tr><tr><td colspan="1" rowspan="1">E. coli,</td></tr><tr><td colspan="1" rowspan="1">K. pneumoniae,</td></tr><tr><td colspan="1" rowspan="1">P. aeruginosa,</td></tr><tr><td colspan="1" rowspan="1">S. marcescens</td></tr><tr><td colspan="1" rowspan="1">E. cloacae complex,</td><td colspan="1" rowspan="5"></td><td colspan="1" rowspan="5"></td><td colspan="1" rowspan="5">.</td><td colspan="1" rowspan="5"></td><td colspan="1" rowspan="5">.</td><td colspan="1" rowspan="5"></td><td colspan="1" rowspan="5"></td><td colspan="1" rowspan="5">.</td><td colspan="1" rowspan="5"></td><td colspan="1" rowspan="5"></td><td colspan="1" rowspan="5"></td><td colspan="1" rowspan="5"></td><td colspan="1" rowspan="5"></td></tr><tr><td colspan="1" rowspan="1">E. coli,</td></tr><tr><td colspan="1" rowspan="1">K. oxytoca,</td></tr><tr><td colspan="1" rowspan="1">K. variicola,</td></tr><tr><td colspan="1" rowspan="1">Proteus spp.</td></tr><tr><td colspan="1" rowspan="1">E. coli,</td><td colspan="1" rowspan="5"></td><td colspan="1" rowspan="5">0</td><td colspan="1" rowspan="5">.</td><td colspan="1" rowspan="5"></td><td colspan="1" rowspan="5">.</td><td colspan="1" rowspan="5"></td><td colspan="1" rowspan="5"></td><td colspan="1" rowspan="5">.</td><td colspan="1" rowspan="5">.</td><td colspan="1" rowspan="5">.</td><td colspan="1" rowspan="5"></td><td colspan="1" rowspan="5"></td><td colspan="1" rowspan="5">.</td></tr><tr><td colspan="1" rowspan="1">K. pneumoniae,</td></tr><tr><td colspan="1" rowspan="1">P. aeruginosa,</td></tr><tr><td colspan="1" rowspan="1">S. marcescens,</td></tr><tr><td colspan="1" rowspan="1">S. maltophilia</td></tr><tr><td colspan="1" rowspan="1">E. coli,</td><td colspan="1" rowspan="5"></td><td colspan="1" rowspan="5">0</td><td colspan="1" rowspan="5">.</td><td colspan="1" rowspan="5"></td><td colspan="1" rowspan="5">.</td><td colspan="1" rowspan="5"></td><td colspan="1" rowspan="5"></td><td colspan="1" rowspan="5"></td><td colspan="1" rowspan="5"></td><td colspan="1" rowspan="5"></td><td colspan="1" rowspan="5"></td><td colspan="1" rowspan="5"></td><td colspan="1" rowspan="5"></td></tr><tr><td colspan="1" rowspan="1">M. morganii,</td></tr><tr><td colspan="1" rowspan="1">Proteus spp.,</td></tr><tr><td colspan="1" rowspan="1">P. aeruginosa,</td></tr><tr><td colspan="1" rowspan="1">S. maltophilia</td></tr><tr><td colspan="1" rowspan="1">H. influenzae,</td><td colspan="1" rowspan="4">1</td><td colspan="1" rowspan="4"></td><td colspan="1" rowspan="4">.</td><td colspan="1" rowspan="4">.</td><td colspan="1" rowspan="4">.</td><td colspan="1" rowspan="4">.</td><td colspan="1" rowspan="4">.</td><td colspan="1" rowspan="4">.</td><td colspan="1" rowspan="4">.</td><td colspan="1" rowspan="4">.</td><td colspan="1" rowspan="4"></td><td colspan="1" rowspan="4">.</td><td colspan="1" rowspan="4"></td></tr><tr><td colspan="1" rowspan="1">K. pneumoniae,</td></tr><tr><td colspan="1" rowspan="1">K. variicola,</td></tr><tr><td colspan="1" rowspan="1">Proteus spp.,</td></tr><tr><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">H. influenzae,</td><td colspan="1" rowspan="5">1</td><td colspan="1" rowspan="5">0</td><td colspan="1" rowspan="5">1</td><td colspan="1" rowspan="5"></td><td colspan="1" rowspan="5"></td><td colspan="1" rowspan="5"></td><td colspan="1" rowspan="5"></td><td colspan="1" rowspan="5"></td><td colspan="1" rowspan="5"></td><td colspan="1" rowspan="5">.</td><td colspan="1" rowspan="5"></td><td colspan="1" rowspan="5"></td><td colspan="1" rowspan="5">.</td></tr><tr><td colspan="1" rowspan="1">K. pneumoniae,</td></tr><tr><td colspan="1" rowspan="1">M. catarrhalis,</td></tr><tr><td colspan="1" rowspan="1">P. jirovecii,</td></tr><tr><td colspan="1" rowspan="1">S. marcescens</td></tr><tr><td colspan="1" rowspan="1">Six OrganismsDetected</td><td colspan="1" rowspan="1">2(0.2%)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Acinetobacter spp.,</td><td colspan="1" rowspan="6">1</td><td colspan="1" rowspan="6">0</td><td colspan="1" rowspan="6">-</td><td colspan="1" rowspan="6"></td><td colspan="1" rowspan="6"></td><td colspan="1" rowspan="6"></td><td colspan="1" rowspan="6"></td><td colspan="1" rowspan="6"></td><td colspan="1" rowspan="6"></td><td colspan="1" rowspan="6"></td><td colspan="1" rowspan="6">1</td><td colspan="1" rowspan="6"></td><td colspan="1" rowspan="6"></td></tr><tr><td colspan="1" rowspan="1">E. coli,</td></tr><tr><td colspan="1" rowspan="1">K. pneumoniae,</td></tr><tr><td colspan="1" rowspan="1">Proteus spp.,</td></tr><tr><td colspan="1" rowspan="1">P. aeruginosa,</td></tr><tr><td colspan="1" rowspan="1">S. aureus</td></tr><tr><td colspan="1" rowspan="1">E. cloacae complex,</td><td colspan="1" rowspan="6">1</td><td colspan="1" rowspan="6">0</td><td colspan="1" rowspan="6">1</td><td colspan="1" rowspan="6">-</td><td colspan="1" rowspan="6">-</td><td colspan="1" rowspan="6"></td><td colspan="1" rowspan="6">-</td><td colspan="1" rowspan="6">-</td><td colspan="1" rowspan="6"></td><td colspan="1" rowspan="6">-</td><td colspan="1" rowspan="6"></td><td colspan="1" rowspan="6">-</td><td colspan="1" rowspan="6"></td></tr><tr><td colspan="1" rowspan="1">H. influenzae,</td></tr><tr><td colspan="1" rowspan="1">M. catarrhalis,</td></tr><tr><td colspan="1" rowspan="1">M. morganii,</td></tr><tr><td colspan="1" rowspan="1">S. aureus,</td></tr><tr><td colspan="1" rowspan="1">S. maltophilia</td></tr><tr><td colspan="1" rowspan="1">Total</td><td colspan="1" rowspan="1">1,016</td><td colspan="1" rowspan="1">938</td><td colspan="1" rowspan="1">15</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">44</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td></tr></table>

\*Note that oxa-58 was not detected in the prospective study cohort.

# D Other Supportive Instrument Performance Characteristics Data:

None

# E Proposed Labeling:

The labeling supports the finding of substantial equivalence for this device.

# F Conclusion:

The submitted information in this premarket notification is complete and supports a substantial equivalence decision.