# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATIONDECISION SUMMARYASSAY AND INSTRUMENT

I Background Information:

A 510(k) Number

# B Applicant

Applied BioCode, Inc.

# C Proprietary and Established Names

BioCode Respiratory Pathogen Panel (RPP)

D Regulatory Information

<table><tr><td rowspan=1 colspan=1>ProductCode(s)</td><td rowspan=1 colspan=1>Classification</td><td rowspan=1 colspan=1>RegulationSection</td><td rowspan=1 colspan=1>Panel</td></tr><tr><td rowspan=1 colspan=1>OCC, OZE, OEP, OEM,OOU, OTG, OZX, OZY,OZZ, NSU</td><td rowspan=1 colspan=1>Class II</td><td rowspan=1 colspan=1>21 CFR 866.3980 -Respiratory Viral PanelMultiplex Nucleic AcidAssay</td><td rowspan=1 colspan=1>MI - Microbiology</td></tr></table>

# II Submission/Device Overview:

A Purpose for Submission: New device

# B Measurand:

Adenovirus, Human Metapneumovirus (hMPV) A/B, Influenza A (Flu A), Influenza A subtype H1 (Flu A/H1), Influenza A subtype H1 2009 Pandemic (Flu A/H1pdm09), Influenza A subtype H3 (Flu A/H3), Influenza B (Flu B), Coronavirus (229E, HKU1, OC43, and NL63), Parainfluenza virus 1 (PIV 1), Parainfluenza virus 2 (PIV 2), Parainfluenza virus 3 (PIV 3), Parainfluenza virus 4 (PIV 4), Human Rhinovirus/Enterovirus (HRV/HEV), Respiratory Syncytial Virus (RSV) A/B, Bordetella pertussis, Chlamydia pneumoniae, and Mycoplasma pneumoniae nucleic acid target sequences.

# C Type of Test:

A multiplexed nucleic acid test intended for use with the BioCode MDx-3000 Instrument for the simultaneous qualitative in vitro detection and identification of multiple respiratory viral and bacterial nucleic acids in nasopharyngeal swabs (NPS) collected in viral transport media (VTM) or universal transport media (UTM), and obtained from individuals suspected of respiratory tract infections.

# III Intended Use/Indications for Use:

A Intended Use(s): See Indications for Use below.

# B Indication(s) for Use:

The BioCode Respiratory Pathogen Panel (RPP) is a qualitative multiplexed nucleic acid-based in vitro diagnostic test intended for use with BioCode MDx-3000 Instrument. The BioCode RPP is capable of the simultaneous detection and identification of nucleic acids from multiple viruses and bacteria extracted from nasopharyngeal swab (NPS) samples obtained from individuals with signs and/or symptoms of respiratory tract infection. The following pathogens and subtypes are identified using the BioCode RPP:

Adenovirus Coronavirus (229E, OC43, HKU1, and NL63)   
• Human Metapneumovirus A/B   
• Influenza A, including subtypes H1, H1 2009 Pandemic, and H3 Influenza B Parainfluenza Virus 1 Parainfluenza Virus 2 Parainfluenza Virus 3 Parainfluenza Virus 4 Respiratory Syncytial Virus A/B Rhinovirus/Enterovirus Bordetella pertussis Chlamydia pneumoniae Mycoplasma pneumoniae

The detection and identification of specific viral and bacterial nucleic acids from individuals exhibiting signs and/or symptoms of a respiratory infection aids in the diagnosis of respiratory infection if used in conjunction with other clinical and epidemiological information. The results of this test should not be used as the sole basis for diagnosis, treatment, or other patient management decisions.

Negative results in the setting of a respiratory illness may be due to infection with pathogens that are not detected by this test, or lower respiratory tract infection that may not be detected by a nasopharyngeal swab specimen. Positive results do not rule out co-infection with other organisms: the agent(s) detected by the BioCode RPP may not be the definite cause of disease. Additional laboratory testing (e.g., bacterial and viral culture, immunofluorescence, and

radiography) may be necessary when evaluating a patient with possible respiratory tract infection.

Due to the genetic similarity between Human Rhinovirus and Enterovirus, the BioCode RPP cannot differentiate them. A positive BioCode RPP Rhinovirus/Enterovirus result should be followed up using an alternate method (e.g., cell culture or sequence analysis) if differentiation is required. The BioCode RPP detects Human Rhinovirus/Enterovirus with reduced sensitivity. If a more accurate Rhinovirus/Enterovirus result is required, it is recommended that specimens found to be negative for Human Rhinovirus/Enterovirus after examination using BioCode RPP be confirmed by an alternate method (e.g., FDA cleared molecular tests).

Performance characteristics for Influenza A were established when Influenza A H1 2009 Pandemic and A H3 were the predominant Influenza A viruses in circulation. Performance of detecting Influenza A may vary if other Influenza A strains are circulating or a novel Influenza A virus emerges. If infection with a novel Influenza A virus is suspected based on current clinical and epidemiological screening criteria recommended by public health authorities, specimens should be collected with appropriate infection control precautions for novel virulent Influenza viruses and sent to state or local health departments for testing. Viral culture should not be attempted in these cases unless a BSL $^ { 3 + }$ facility is available to receive and culture specimens.

C Special Conditions for Use Statement(s): Rx - For Prescription Use Only

D Special Instrument Requirements: BioCode MDx-3000 Instrument bioMérieux NucliSENS easyMAG system Roche MagNA Pure 96 system

# IV Device/System Characteristics:

# A Device Description:

The BioCode Respiratory Pathogen Panel (RPP) is a respiratory pathogen multiplex nucleic acid test designed for use with the BioCode MDx-3000 system. The BioCode MDx-3000 is an automated system that integrates PCR amplification, target capture, signal generation and optical detection for multiple viral and bacterial pathogens from a single nasopharyngeal swab specimen collected in VTM or UTM. Liquid specimens are processed, and nucleic acids extracted with the bioMérieux NucliSENS easyMAG or Roche MagNA Pure 96 automated systems. Once the PCR plate is manually set up and sealed, all other operations are automated on the MDx-3000. The BioCode RPP simultaneously tests for 17 pathogens and/or subtypes (see the Intended Use section above) from nasopharyngeal swab specimens collected in VTM or UTM. Results from the BioCode RPP test are available within about 5 hours.

# Materials Provided with Each BioCode RPP Kit:

BioCode Master Mix A • BioCode RPP Primer Mix BioCode RT Mix

• BioCode RNA IC2 • BioCode RPP BMB-Probe Mix • Individually-packaged Transfer Pipettes

# Materials Needed but Not Provided with the BioCode RPP Kit:

• BioCode SA-PE Mix   
• BioCode Buffer $\mathbf { A l 0 \% }$ bleach solution BioCode MDx-3000 Instrument BioCode MDx-3000 Consumables: Reagent Reservoirs (Integra 4332) Waste Bin and Lid (Applied BioCode 01-W0105) $2 0 \mu \mathrm { L }$ pipette tips (Beckman 717256) $2 5 0 \mu \mathrm { L }$ pipette tips (Beckman 717252) 96-well hard shell plate $0 . 1 \mathrm { m L }$ (Bio-Rad HSL9601) PCR Adhesive Foil (Thermo Fisher Scientific AB-0626) Microtiter plate (Greiner bio-one 655101) Microtiter plate lid (Nunc 5500)   
• NucliSENS easyMAG (bioMérieux) Extraction System   
• NucliSENS easyMAG (bioMérieux) Supplies for Extraction: Lysis Buffer Buffer 1 Buffer 2 Buffer 3 Magnetic silica Nuclease free water Consumables ELISA strip plate

or

MagNA Pure 96 (Roche) Extraction System   
• MagNA Pure 96 (Roche) Supplies for Extraction: MagNA Pure 96 DNA and Viral nucleic acid kit MagNA Pure 96 system fluid Consumables

• Vortex • Centrifuge

• Pipettes – single, multi-channel and/or repeater with accuracy range between $1 { - } 1 0 \mu \mathrm { L }$ , 10-200 $\mu \mathrm { L }$ , and $1 0 0 - 1 0 0 0 { \mu \mathrm { L } }$

• Sterile, RNase/DNase-free disposable aerosol-barrier micro pipettor tips • $1 . 5 \mathrm { m L }$ polypropylene micro centrifuge tubes and racks (RNase/DNase free recommended)

# Interpretation of Results

The BioCode MDx-3000 software will analyze data based on plate validity, sample validity and Median Fluorescent Intensity (MFI) compared to an MFI threshold.

# Run Controls and Internal Control Results

The BioCode MDx-3000 software will suppress results if Internal or Negative controls are invalid. The software will indicate if external positive controls are valid or invalid but will not suppress results if the positive control is not valid (see Internal Control) below.

External Negative Controls   
External Negative Controls can be RNase-free water, transport media, or well characterized negative specimens. The External Negative Controls should go through all processing steps (extractions, amplification, and detection). At least one External Negative Control is required for each plate/kit lot. The BioCode MDx-3000 software will suppress results for all samples if the External Negative Control is not valid (see Table 1 below).

Table 1: Criteria for Valid External Negative Control   

<table><tr><td rowspan=1 colspan=1>Control</td><td rowspan=1 colspan=1>Targets</td><td rowspan=1 colspan=1>RNA IC</td><td rowspan=1 colspan=1>Description</td></tr><tr><td rowspan=1 colspan=1>Negative Control</td><td rowspan=1 colspan=1>Not Detected</td><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>Plate Status: Valid.Samples can be interpreted.</td></tr><tr><td rowspan=1 colspan=1>Negative Control</td><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>Plate Status: Invalid.Samples results cannot be interpreted. Resultssuppressed by software.</td></tr><tr><td rowspan=1 colspan=1>Negative Control</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>Not Detected</td><td rowspan=1 colspan=1>Plate Status: Invalid.Samples results cannot be interpreted. Resultssuppressed by software.</td></tr></table>

# External Positive Controls

External Positive Controls can be well characterized clinical samples or positive strains. External Positive Controls can be single analytes or pooled multianalyte specimens. External Positive Controls should go through all processing steps (extractions, amplification, and detection). It is recommended that at least one External Positive Control be included for each plate/kit lot on a rotating schedule. Wells identified as External Positive Controls will be trended by the BioCode MDx-3000 software and the report will indicate a valid or invalid result on the report header (see Table 2 below). However, the software will not suppress results based on External Positive Control results. If an External Positive Control does not perform as expected, the user should review all samples in that plate to determine if results can be reported.

Table 2: Criteria for Valid External Positive Control   

<table><tr><td rowspan=1 colspan=1>Control</td><td rowspan=1 colspan=1>Targets</td><td rowspan=1 colspan=1>RNA IC</td><td rowspan=1 colspan=1>Recommendations</td></tr><tr><td rowspan=1 colspan=1>Positive Control</td><td rowspan=1 colspan=1>Expected Target Detected</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>Report will indicate Positive Control isValid. No user intervention required.</td></tr><tr><td rowspan=1 colspan=1>Positive Control</td><td rowspan=1 colspan=1>Expected Target Not Detected</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>Report will indicate Positive Control isInvalid. User should review results of allsamples in that plate prior to release.</td></tr><tr><td rowspan=1 colspan=1>Positive Control</td><td rowspan=1 colspan=1>Unexpected Target Detected</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>Report will indicate Positive Control isInvalid. User should review results of allsamples in that plate prior to release.</td></tr></table>

# Internal Control

An RNA Internal Control (RNA IC: bacteriophage MS2) is added to each sample during extraction. The Internal Control monitors the efficiency of the extraction, reverse transcription, amplification and detection stages of the assay. Positive target assay results may be reported in the absence of RNA IC detection. However, the BioCode MDx-3000 software will suppress target assay negative results for any wells with invalid RNA IC results (see Table 3 below). Lack of RNA IC signal may indicate sample-associated inhibition or reagent/instrumentation issues. Samples suspected of being inhibitory should be repeated from the extraction step. If reagent or instrument issues are suspected specimens may be repeated using residue nucleic acid extracts.

Table 3: Criteria for RNA Internal Control (RNA IC)   

<table><tr><td rowspan=1 colspan=1>Targets</td><td rowspan=1 colspan=1>RNA IC</td><td rowspan=1 colspan=1>Recommendations</td></tr><tr><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>Well status: Valid.Report all results.</td></tr><tr><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>Not Detected</td><td rowspan=1 colspan=1>Well status: Invalid.Detected results may be reported. Consider repeat/reflex testing.</td></tr><tr><td rowspan=1 colspan=1>Not Detected</td><td rowspan=1 colspan=1>Not Detected</td><td rowspan=1 colspan=1>Well status: Invalid.Not Detected results suppressed by software. Repeat/reflex testing.</td></tr></table>

# Interpretation of Pathogen Results

Fluorescent signals from barcoded magnetic beads (BMBs) with the same barcode are sorted and the median fluorescence index (MFI) is calculated for each analyte assay. The assays are considered “Detected” by comparing the MFI to a specific validated assay cutoff. If assay target specific MFI is at or above the threshold, the assay is positive.

For the following organisms detected by the BioCode RPP, the organism is reported as “Detected” if a single corresponding assay is positive.

Rhinovirus/Enterovirus • Parainfluenza Virus 1 • Parainfluenza Virus 2 • Parainfluenza Virus 3 • Parainfluenza Virus 4 • Respiratory Syncytial Virus A/B • Bordetella pertussis

• Chlamydia pneumoniae • Mycoplasma pneumoniae

For the following organisms detected by the BioCode RPP, the organism is reported as “Detected” if at least one of the two corresponding assays is positive.

Influenza B • Human Metapneumovirus A/B • Adenovirus

And for the following organism detected by the BioCode RPP, the organism is reported as “Detected” if at least one of the four corresponding assays is positive.

Coronavirus

Note: Each of the four Coronavirus (CoV) assays is specifically designed to detect CoV-229E, CoV-OC43, CoV-HKU1, and CoV-NL63, respectively.

In addition, the BioCode RPP contains one assay designed to detect Influenza A (FluA), and three hemagglutinin (HA) subtyping assays for A/H1, A/H1pdm09 and A/H3, respectively. Each of the individual assays is interpreted independently and the panel result reported for Influenza A is based on the combined results of the four assays as outlined in Table 4 below.

Table 4: Possible Target Assay Results for Influenza A and Corresponding Interpretation   

<table><tr><td colspan="1" rowspan="1">Target AssayBioCode RPPResult</td><td colspan="1" rowspan="1">FluA</td><td colspan="1" rowspan="1">A/H1</td><td colspan="1" rowspan="1">A/H1pdm09</td><td colspan="1" rowspan="1">A/H3</td><td colspan="1" rowspan="1">Action</td></tr><tr><td colspan="1" rowspan="1">Influenza A Not Detected</td><td colspan="1" rowspan="1">Not Detected</td><td colspan="1" rowspan="1">Not Detected</td><td colspan="1" rowspan="1">Not Detected</td><td colspan="1" rowspan="1">Not Detected</td><td colspan="1" rowspan="1">Report result</td></tr><tr><td colspan="1" rowspan="1">Influenza A and A/H1 Detected</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">Not Detected</td><td colspan="1" rowspan="1">Not Detected</td><td colspan="1" rowspan="1">Report result</td></tr><tr><td colspan="1" rowspan="1">Influenza A and A/H3 Detected</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">Not Detected</td><td colspan="1" rowspan="1">Not Detected</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">Report result</td></tr><tr><td colspan="1" rowspan="1">Influenza A and A/H1pdm09Detected</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">Not Detected</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">Not Detected</td><td colspan="1" rowspan="1">Report result</td></tr><tr><td colspan="1" rowspan="1">Influenza A/H1, and A/H3Detected</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">Not Detected</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">Multiple infections arepossible but rare, retest toconfirm result a</td></tr><tr><td colspan="1" rowspan="1">Influenza A/H1pdm09, andA/H3 Detected</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">Not Detected</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">Multiple infections arepossible but rare, retest toconfirm result a</td></tr><tr><td colspan="1" rowspan="1">Influenza A/H1, andA/H1pdm09 Detected</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">Not Detected</td><td colspan="1" rowspan="1">Multiple infections arepossible but rare, retest toconfirm result a</td></tr><tr><td colspan="1" rowspan="1">Influenza A/H1, A/H1pdm09,and A/H3 Detected</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">Multiple infections arepossible but rare, retest toconfirm result a</td></tr><tr><td colspan="1" rowspan="1">Influenza A (no subtypedetected)</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">Not Detected</td><td colspan="1" rowspan="1">Not Detected</td><td colspan="1" rowspan="1">Not Detected</td><td colspan="1" rowspan="1">Retest (see the section onInfluenza A, no subtypedetected below)</td></tr><tr><td colspan="1" rowspan="3">Influenza A Indeterminate</td><td colspan="1" rowspan="1">Not Detected</td><td colspan="1" rowspan="1">Detected orNot Detected</td><td colspan="1" rowspan="1">Detected orNot Detected</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="3">Retest b</td></tr><tr><td colspan="1" rowspan="1">Not Detected</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">Detected orNot Detected</td><td colspan="1" rowspan="1">Detected orNot Detected</td></tr><tr><td colspan="1" rowspan="1">Not Detected</td><td colspan="1" rowspan="1">Detected orNot Detected</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">Detected orNot Detected</td></tr></table>

a Repeated multiple positive should be further confirmed by other FDA-cleared influenza subtyping assays. b If the re-test result confirms the original result, it is recommended that the sample be further investigated using a different FDA-cleared influenza A subtyping assay and/or sending the residual sample to local public health laboratory for further testing.

# Influenza A (no subtype detected):

If the FluA assay is positive, but none of the hemagglutinin (HA) subtyping assays are positive, then the interpretation is Influenza A (no subtype detected). This result could occur when the titer of the virus in the specimen is low and not detected by the subtyping assays. This result could also indicate the presence of a novel Influenza A strain or a seasonal Influenza A/H3 or A/H1pdm09 strain with critical sequence mismatches to the primers and/or probes of the BioCode RPP influenza A HA subtyping assays. In both cases, the sample in question should be retested. If the retest provides a different result, test the sample a third time to ensure the accuracy of the result. If the retest provides the same result, then the function of the BioCode RPP should be verified by testing with appropriate external control materials (known positive samples for Influenza A/H1, Influenza A/H3 and Influenza A/H1pdm09), and a negative control should also be run to test for PCR-product contamination. If the BioCode RPP accurately identifies the external positive and negative controls, contact the appropriate public health authorities for confirmatory testing.

# BioCode RPP Reports

The analyzed BioCode MDx-3000 results are displayed in two report formats: Run Report for the entire run including multiple specimens, and Sample Report for individual specimens. Both reports can be exported as a PDF or CSV file. Each report includes fully analyzed and interpreted results for specimens and/or controls but is formatted differently.

The Run Report displays analyzed results in a tabular format for all wells (specimens/controls) in a run from a specific kit lot. If more than one lot is run together, separate Run Reports will be generated by the software for each lot. Possible results by target assay are: Detected, Not detected, Invalid, Indeterminant (for Influenza A only), or N/A (if not ordered).

The Sample Report displays results for a single well (specimen/control). In addition to results for each target assay, the Sample Reports include a results summary section which allows positive results to be reviewed at a glance. The Sample Report results summary will also indicate well validity based on BMB counts, background MFI, and external and internal controls. Sample reports also include any sample-specific comments entered during setup.

Both report headers provide traceability information for: run name, run start and finish time, user ID, software version, instrument ID, kit name, and reagent lots with expiration dates. The report headers also include sections for Run Status and Controls Status. The Run Status section will specify if the run is Incomplete, Valid or Invalid (based on the External Negative Control results for the specific run/kit lot). The Controls Status section indicates the results for the External Negative Controls (Valid or Invalid) and External Positive Controls (Valid, Invalid, or N/A if not assayed). The Run Status and Controls Status sections should be reviewed prior to review of target assay results. In addition, the software will also mask results in the detailed tabular sections based on plate and well validity requirements.

Completed reports can be reviewed electronically. Reviewer comments can be added to the report footer for traceability under the review section. In addition, MFI reports are available with information only for administrator level users.

# B Principle of Operation:

The BioCode MDx-3000 is an automated system that integrates PCR amplification, target capture, signal generation, and optical detection for multiple respiratory viruses and bacteria from a single nasopharyngeal swab (NPS) specimen, in either VTM or UTM. Nucleic acids from NPS are extracted with the bioMérieux NucliSENS easyMAG or Roche MagNA Pure 96 automated systems. Once the PCR plate is manually set up and sealed, all other operations are automated on the MDx-3000.

# Nucleic Acid Extraction

Nucleic acids (both RNA and DNA) are captured by coated magnetic beads and eluted on either the bioMérieux NucliSENS easyMAG or the Roche MagNA Pure 96 automated systems according to the respective manufacturer provided protocol.

Overview of a BioCode MDx-3000 Run:

# Reverse Transcription and Multiplex PCR

Since targeted pathogens of the BioCode RPP include RNA viruses, a reverse transcription (RT) step is performed to convert the viral RNA into cDNA prior to amplification. The purified nucleic acid solution is manually combined with a freshly prepared reaction mix for the RT step and subsequent thermal cycling for multiplex PCR to enrich the target nucleic acids present in the sample. One of the target-specific primers for each pathogen is biotinylated at the $5 '$ -end to generate labeled PCR product for subsequent detection.

# PCR and Product Transfer

For the PCR amplification, the robotic head dispenses BMB- Probe mix into the designated reaction wells of the capture plate using disposable pipette tips. After PCR amplification is completed, the robotic head pierces the foil seal with disposable pipette tips and transfers PCR products into corresponding wells of the capture plate.

# Target Capture

Amplified PCR products labeled with biotin are captured at a defined temperature by targetspecific probes that are covalently coupled to designated Barcoded Magnetic Beads (BMBs).

During this step, BMBs are kept in suspension by gentle agitation. Differentiation of captured targets is achieved by assigning a unique barcode pattern of BMBs for each pathogen and the Internal Control.

# Signal Generation

After washing off unbound PCR products and unused primers, a streptavidin-phycoerythrin (SAPE) conjugate is automatically added to the reaction by the robot. High affinity binding between biotin and streptavidin ensures that captured PCR products with the biotin moiety are labeled with phycoerythrin in close proximity to the BMBs.

# Optical Detection

Optical detection is performed for each reaction well of the capture plate, an optically clear, flatbottom microtiter plate. After washing off unbound SA-PE, excitation of the fluorophore at the designated wavelength emits fluorescence signal from BMBs tagged with SA-PE conjugates. Each reaction well is imaged at a specific emission wavelength for fluorescent signal and under bright field for identifying the barcode patterns (decoding).

# Interpretation of Results

The BioCode MDx-3000 Software controls the operation of the instrument, collects and analyzes data, and automatically generates interpretation for test reports at the end of the run. Fluorescent signals from BMBs with the same barcode are sorted and calculated to generate median fluorescence index (MFI) for each analyte. The presence or absence of a pathogen is determined relative to the validated assay cutoff by MFI. The software also analyzes the results of external and internal controls to validate the run and individual specimen results for reporting.

# C Instrument Description Information:

<table><tr><td rowspan=1 colspan=1>Modes of Operation</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>No</td></tr><tr><td rowspan=1 colspan=1>Does the applicant&#x27;s device contain the ability to transmit data to a computer,webserver, or mobile device?</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>□</td></tr><tr><td rowspan=1 colspan=1>Does the applicant&#x27;s device transmit data to a computer, webserver, or mobile deviceusing wireless transmission?</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>X</td></tr><tr><td rowspan=1 colspan=3>Software</td></tr><tr><td rowspan=1 colspan=1>FDA has reviewed applicant&#x27;s Hazard Analysis and software development processesfor this line of product types.</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1></td></tr></table>

1. Instrument Name: BioCode MDx-3000 Instrument

2. Specimen Identification: Specimen identity is provided by barcode magnetic beads.

3. Specimen Sampling and Handling:

After extraction with the NucliSENS easyMag system or the MagNA Pure 96 system and manually loading samples into a 96-well formatted plate, the BioCode MDx 3000 processes all RT-PCR, target capture, signal generation, and optical detection steps automatically.

4. Calibration:

Optical calibration of the BioCode MDx 3000 is performed twice a year by Applied BioCode. No calibration kit is available.

5. Quality Control:

Each laboratory should establish its own Quality Control ranges and frequency of QC testing based on applicable local laws, regulations, and good laboratory practices. The BioCode RPP uses an internal control (bacteriophage MS2) which is added to each sample prior to extraction. The internal control monitors the efficiency of the extraction, reverse transcription, amplification, and detection stages of the assay. Positive results may be reported in the absence of RNA IC detection. The BioCode RPP software will suppress negative results for any wells with invalid RNA IC results.

# V Substantial Equivalence Information:

A Predicate Device Name(s): FilmArray Respiratory Panel 2 (RP2)   
B Predicate 510(k) Number(s): K170604   
C Comparison with Predicate(s):

<table><tr><td colspan="3" rowspan="1">Similarities or Differences</td></tr><tr><td colspan="1" rowspan="1">Element</td><td colspan="1" rowspan="1">BioCode Respiratory Pathogen Panel (RPP)(K192485)</td><td colspan="1" rowspan="1">FilmArray Respiratory Panel 2 (RP2)(K170604)</td></tr><tr><td colspan="1" rowspan="1">Specimen Types</td><td colspan="1" rowspan="1">NPS (in VTM or UTM)</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">PathogensDetected</td><td colspan="1" rowspan="1">Adenovirus, Coronavirus (229E, HKU1, NL63,and OC43), Human Metapneumovirus A/B,Influenza A, Influenza A subtype H1, InfluenzaA subtype H3, Influenza A subtype 2009Pandemic, Influenza B, Parainfluenza Virus 1,Parainfluenza Virus 2, Parainfluenza Virus 3,Parainfluenza Virus 4, Rhinovirus/Enterovirus,Respiratory Syncytial Virus A/B, Bordetellapertussis, Chlamydia pneumoniae, andMycoplasma pneumoniae</td><td colspan="1" rowspan="1">Same, except that the four coronaviruses, 229E,HKU1, NL63, and OC43, are detected andreported separately, and the pathogens detectedalso include Bordetella parapertussis</td></tr><tr><td colspan="1" rowspan="1">Analyte</td><td colspan="1" rowspan="1">RNA/DNA</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">TechnologicalPrinciples</td><td colspan="1" rowspan="1">Multiplex nucleic acid</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Instrumentation</td><td colspan="1" rowspan="1">BioCode MDx-3000 Instrument,bioMérieux NucliSENS easyMAG system orRoche MagNA Pure 96 system</td><td colspan="1" rowspan="1">FilmArray 2.0 or FilmArray Torch</td></tr><tr><td colspan="1" rowspan="1">Time to Result</td><td colspan="1" rowspan="1">About 5.0 hours</td><td colspan="1" rowspan="1">About 45 minutes</td></tr><tr><td colspan="1" rowspan="1">Test Interpretation</td><td colspan="1" rowspan="1">Automated test interpretation and reportgeneration.</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Controls</td><td colspan="1" rowspan="1">An RNA Internal Control is added to eachsample during extraction.External Positive and Negative Controls areexternally sourced.</td><td colspan="1" rowspan="1">Two controls are included in each reagent pouchto control for sample processing and both stagesof PCR and melt analysis.External Positive and Negative Controls areexternally sourced.</td></tr><tr><td colspan="1" rowspan="1">Methodology</td><td colspan="1" rowspan="1">Multiplex RT-PCR in a single reaction andprobe hybridization followed by fluorescencedetection and decoding of barcoded magneticbeads (BMB) that capture biotinylated PCRproducts with streptavidin conjugate.</td><td colspan="1" rowspan="1">Nested multiplex PCR executed in two stages.First, a single, large volume, highly multiplexedreverse transcription PCR (RT-PCR) reaction.Second, nested PCR, is performed in singleplexfashion in each well of the array, followed bymelting curve analyses of the PCR reactions.</td></tr><tr><td colspan="1" rowspan="1">CLIA Complexity</td><td colspan="1" rowspan="1">High</td><td colspan="1" rowspan="1">Moderate</td></tr></table>

# VI Standards/Guidance Documents Referenced:

• FDA guidance document issued on August 27, 2014, titled “Highly Multiplexed Microbiological/Medical Countermeasure In Vitro Nucleic Acid Based Diagnostic Devices”   
FDA guidance document issued on October 9, 2009, titled “Class II Special Controls Guidance Document: Respiratory Viral Panel Multiplex Nucleic Acid Assay” FDA guidance document issued on October 9, 2009, titled “Class II Special Controls Guidance Document: Testing for Detection and Differentiation of Influenza A Virus Subtypes Using Multiplex Assays”   
FDA guidance document issued on October 9, 2009, titled “Testing for Human Metapneumovirus (hMPV) Using Nucleic Acid Assays”   
FDA guidance document issued on July 15, 2011, titled “Establishing the Performance Characteristics of In Vitro Diagnostic Devices for the Detection or Detection and Differentiation of Influenza Viruses”   
• FDA guidance document issued on April 25, 2005, titled “Guidance on Informed Consent for In Vitro Diagnostic Device Studies Using Leftover Human Specimens that are Not Individually Identifiable”

# VII Performance Characteristics (if/when applicable):

# A Analytical Performance:

1. Precision/Reproducibility:

A reproducibility study was conducted at three testing sites using a combination of NucliSENS easyMAG and MagNA Pure 96 automated systems for nucleic acids extraction (two sites used NucliSENS easyMAG and one site used MagNA Pure 96 for nucleic acids extraction). The study incorporated potential sources of variation introduced by site (three testing sites), day (five different days), and operator (two operators per site). One lot of reagents was assayed at three sites by two operators on one BioCode MDx-3000 instrument per site for five days. A total of 10 BioCode RPP runs were performed per site.

A total of six contrived NPS samples containing known quantities of various BioCode RPP analytes (Table 5 below) were prepared in a simulated NPS in UTM sample matrix. A NPS sample negative for all BioCode RPP analytes was also prepared using the simulated NPS in UTM sample matrix. The six contrived positive samples consisted of combinations of 12 representative pathogens, at $1 . 5 \mathrm { x }$ LoD (Low) and 3x LoD (Medium), spiked into a simulated NPS in UTM sample matrix.

Table 5: Reproducibility Study Panel – Sample Composition   

<table><tr><td rowspan=1 colspan=1>Medium (3x LoD)</td><td rowspan=1 colspan=1>Medium (3x LoD)</td><td rowspan=1 colspan=1>Low (1.5x LoD)</td><td rowspan=1 colspan=1>Low (1.5x LoD)</td><td rowspan=1 colspan=1>SampleName</td></tr><tr><td rowspan=1 colspan=1>Rhinovirus</td><td rowspan=1 colspan=1>Parainfluenza Virus 2</td><td rowspan=1 colspan=1>HumanMetapneumovirus</td><td rowspan=1 colspan=1>Bordetella pertussis</td><td rowspan=1 colspan=1>RP1</td></tr><tr><td rowspan=1 colspan=1>HumanMetapneumovirus</td><td rowspan=1 colspan=1>Bordetella pertussis</td><td rowspan=1 colspan=1>Rhinovirus</td><td rowspan=1 colspan=1>Parainfluenza Virus 2</td><td rowspan=1 colspan=1>RP2</td></tr><tr><td rowspan=1 colspan=1>Influenza B</td><td rowspan=1 colspan=1>Coronavirus NL63</td><td rowspan=1 colspan=1>Chlamydia pneumoniae</td><td rowspan=1 colspan=1>Parainfluenza Virus 3</td><td rowspan=1 colspan=1>RP3</td></tr><tr><td rowspan=1 colspan=1>Chlamydia pneumoniae</td><td rowspan=1 colspan=1>Parainfluenza Virus 3</td><td rowspan=1 colspan=1>Influenza B</td><td rowspan=1 colspan=1>Coronavirus NL63</td><td rowspan=1 colspan=1>RP4</td></tr><tr><td rowspan=1 colspan=1>Influenza A H3N2</td><td rowspan=1 colspan=1>Mycoplasma pneumoniae</td><td rowspan=1 colspan=1>Respiratory SyncytialVirus</td><td rowspan=1 colspan=1>Adenovirus C</td><td rowspan=1 colspan=1>RP5</td></tr><tr><td rowspan=1 colspan=1>Respiratory SyncytialVirus</td><td rowspan=1 colspan=1>Adenovirus C</td><td rowspan=1 colspan=1>Influenza A H3N2</td><td rowspan=1 colspan=1>Mycoplasma pneumoniae</td><td rowspan=1 colspan=1>RP6</td></tr><tr><td rowspan=1 colspan=4>Simulated Negative NPS in UTM matrix</td><td rowspan=1 colspan=1>RP7</td></tr></table>

Information regarding the specific strains and isolates of the 12 representative pathogens and concentrations that were used in building the reproducibility study panel samples are provided in Table 6 below.

Table 6: Reproducibility Study Panel - Strains/Isolates and Concentrations   

<table><tr><td rowspan=1 colspan=1>Pathogen</td><td rowspan=1 colspan=1>Strain/Serotype</td><td rowspan=1 colspan=1>Source/ ID</td><td rowspan=1 colspan=1>Limit ofDetection (LoD)</td><td rowspan=1 colspan=1>LowConcentration(1.5xLoD)</td><td rowspan=1 colspan=1>MediumConcentration(3xLoD)</td></tr><tr><td rowspan=1 colspan=1>Coronavirus NL63</td><td rowspan=1 colspan=1>NL63</td><td rowspan=1 colspan=1>Zeptometrix0810228CF</td><td rowspan=1 colspan=1>0.04 TCID50/mL</td><td rowspan=1 colspan=1>0.06 TCID50/mL</td><td rowspan=1 colspan=1>0.12 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza virus 2</td><td rowspan=1 colspan=1>Greer/Ohio/1955</td><td rowspan=1 colspan=1>ATCC VR-92</td><td rowspan=1 colspan=1>5.4 TCID50/mL</td><td rowspan=1 colspan=1>8.1 TCID50/mL</td><td rowspan=1 colspan=1>16.2 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Adenovirus C</td><td rowspan=1 colspan=1>Species C</td><td rowspan=1 colspan=1>ATCC VR-846</td><td rowspan=1 colspan=1>18.0 TCID50/mL</td><td rowspan=1 colspan=1>27.0 TCID50/mL</td><td rowspan=1 colspan=1>54.0 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Influenza A H3N2</td><td rowspan=1 colspan=1>A/Wisconsin/67/05</td><td rowspan=1 colspan=1>Zeptometrix0810252CF</td><td rowspan=1 colspan=1>4.0 TCID50/mL</td><td rowspan=1 colspan=1>6.0 TCID50/mL</td><td rowspan=1 colspan=1>12.0 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Human Rhinovirus</td><td rowspan=1 colspan=1>Type 1A</td><td rowspan=1 colspan=1>Zeptometrix0810012CF</td><td rowspan=1 colspan=1>1.2 TCID50/mL</td><td rowspan=1 colspan=1>1.8 TCID50/mL</td><td rowspan=1 colspan=1>3.6 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Mycoplasmapneumoniae</td><td rowspan=1 colspan=1>M129</td><td rowspan=1 colspan=1>Zeptometrix0801579</td><td rowspan=1 colspan=1>15.0 CCU/mL</td><td rowspan=1 colspan=1>22.5 CCU/mL</td><td rowspan=1 colspan=1>45.0 CCU/mL</td></tr><tr><td rowspan=1 colspan=1>Chlamydiapneumoniae</td><td rowspan=1 colspan=1>AR39</td><td rowspan=1 colspan=1>ATCC VR-53592</td><td rowspan=1 colspan=1>33.3 CFU/mL</td><td rowspan=1 colspan=1>50.0 CFU/mL</td><td rowspan=1 colspan=1>100.0 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Influenza B</td><td rowspan=1 colspan=1>Florida/4/2006(Yamagata)</td><td rowspan=1 colspan=1>Zeptometrix0810255CF</td><td rowspan=1 colspan=1>0.01 TCID50/mL</td><td rowspan=1 colspan=1>0.02 TCID50/mL</td><td rowspan=1 colspan=1>0.03 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza virus 3</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>Zeptometrix0810060CF</td><td rowspan=1 colspan=1>15.0 TCID50/mL</td><td rowspan=1 colspan=1>22.5 TCID50/mL</td><td rowspan=1 colspan=1>45.0 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>HumanMetapneumovirus</td><td rowspan=1 colspan=1>Type 16, Type A1IA10-2003</td><td rowspan=1 colspan=1>Zeptometrix0810161CF</td><td rowspan=1 colspan=1>15.0 TCID50/mL</td><td rowspan=1 colspan=1>22.5 TCID50/mL</td><td rowspan=1 colspan=1>45.0 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Respiratory SyncytialVirus</td><td rowspan=1 colspan=1>Type A</td><td rowspan=1 colspan=1>Zeptometrix0810040ACF</td><td rowspan=1 colspan=1>0.33 TCID50/mL</td><td rowspan=1 colspan=1>0.45 TCID50/mL</td><td rowspan=1 colspan=1>0.9 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Bordetella pertussis</td><td rowspan=1 colspan=1>A639</td><td rowspan=1 colspan=1>Zeptometrix0801459</td><td rowspan=1 colspan=1>45.0 CFU/mL</td><td rowspan=1 colspan=1>67.5 CFU/mL</td><td rowspan=1 colspan=1>135.0 CFU/mL</td></tr></table>

Once prepared, each sample of the reproducibility study panel was tested with the BioCode RPP to confirm that it contained the intended analytes at the intended concentration and then divided into single-use aliquots and stored frozen $( \leq - 7 0 ^ { \circ } \mathrm { C } )$ until the day of testing.

After being distributed to the sites, each sample was extracted in triplicates using either the NucliSENS easyMAG (Site 1 and Site 3) or the MagNA Pure 96 (Site 2) and assayed using the BioCode RPP. In total, 90 data points per sample were obtained, with 60 data points generated by the NucliSENS easyMAG system (Site 1 and Site 3) and 30 data points generated by the MagNA Pure 96 system.

A summary of the reproducibility study results, percent $( \% )$ agreement with the expected Detected or Not Detected result, for each analyte (by site and extraction system) is provided in Table 7 below.

Table 7: Reproducibility of BioCode RPP   

<table><tr><td colspan="1" rowspan="1">I able 7. meplod</td><td colspan="1" rowspan="1">uclbiey 0l Dl</td><td colspan="1" rowspan="1"></td><td></td><td></td><td></td><td></td><td></td><td></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="3">Analyte</td><td colspan="1" rowspan="3">ConcentrationTested</td><td colspan="1" rowspan="3">ExpectedResult</td><td colspan="6" rowspan="1">Agreement with Expected Result</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="3" rowspan="1">NucliSENS easyMAG</td><td colspan="2" rowspan="1">MagNA Pure 96</td><td colspan="1" rowspan="2">All Sites(95% CI)</td><td colspan="1" rowspan="2"></td></tr><tr><td colspan="1" rowspan="1">Site 1</td><td colspan="1" rowspan="1">Site 3</td><td colspan="1" rowspan="1">Sub-Total</td><td colspan="1" rowspan="1">Site 2</td><td colspan="1" rowspan="1">Sub-Total</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Viruses</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">b</td></tr><tr><td colspan="1" rowspan="3">Adenovirus</td><td colspan="1" rowspan="1">3× LoD</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">60/60100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">90/90100%(95.9%-100%)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">1.5× LoD</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">60/60100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">90/90100%(95.9%-100%)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">None(no analyte)</td><td colspan="1" rowspan="1">NotDetected</td><td colspan="1" rowspan="1">150/150100%</td><td colspan="1" rowspan="1">149/15099.3%</td><td colspan="1" rowspan="1">299/30099.7%</td><td colspan="1" rowspan="1">150/150100%</td><td colspan="1" rowspan="1">150/150100%</td><td colspan="1" rowspan="1">449/45099.8%(98.8%-100%)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="3">Coronavirus</td><td colspan="1" rowspan="1">3× LoD</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">60/60100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">90/90100%(95.9%-100%)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">1.5× LoD</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">60/60100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">90/90100%(95.9%-100%)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">None(no analyte)</td><td colspan="1" rowspan="1">NotDetected</td><td colspan="1" rowspan="1">150/150100%</td><td colspan="1" rowspan="1">150/150100%</td><td colspan="1" rowspan="1">300/300100%</td><td colspan="1" rowspan="1">150/150100%</td><td colspan="1" rowspan="1">150/150100%</td><td colspan="1" rowspan="1">450/450100%(99.2%-100%)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="3">HumanMetapneumovirus</td><td colspan="1" rowspan="1">3× LoD</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">60/60100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">90/90100%(95.9%-100%)</td><td></td></tr><tr><td colspan="1" rowspan="1">1.5× LoD</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">60/60100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">90/90100%(95.9%-100%)</td><td></td></tr><tr><td colspan="1" rowspan="1">None(no analyte)</td><td colspan="1" rowspan="1">NotDetected</td><td colspan="1" rowspan="1">150/150100%</td><td colspan="1" rowspan="1">150/150100%</td><td colspan="1" rowspan="1">300/300100%</td><td colspan="1" rowspan="1">150/150100%</td><td colspan="1" rowspan="1">150/150100%</td><td colspan="1" rowspan="1">450/450100%(99.2%-100%)</td><td></td></tr><tr><td colspan="1" rowspan="3">HumanRhinovirus/Enterovirus</td><td colspan="1" rowspan="1">3× LoD</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">60/60100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">90/90100%(95.9%-100%)</td><td></td></tr><tr><td colspan="1" rowspan="1">1.5× LoD</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">60/60100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">90/90100%(95.9%-100%)</td><td></td></tr><tr><td colspan="1" rowspan="1">None(no analyte)</td><td colspan="1" rowspan="1">NotDetected</td><td colspan="1" rowspan="1">150/150100%</td><td colspan="1" rowspan="1">150/150100%</td><td colspan="1" rowspan="1">300/300100%</td><td colspan="1" rowspan="1">150/150100%</td><td colspan="1" rowspan="1">150/150100%</td><td colspan="1" rowspan="1">450/450100%(99.2%-100%)</td><td></td></tr><tr><td colspan="1" rowspan="3">Analyte</td><td colspan="1" rowspan="3">ConcentrationTested</td><td colspan="1" rowspan="3">ExpectedResult</td><td colspan="6" rowspan="1">Agreement with Expected Result</td></tr><tr><td colspan="3" rowspan="1">NucliSENS easyMAG</td><td colspan="2" rowspan="1">MagNA Pure 96</td><td colspan="1" rowspan="2">All Sites(95% CI)</td></tr><tr><td colspan="1" rowspan="1">Site 1</td><td colspan="1" rowspan="1">Site 3</td><td colspan="1" rowspan="1">Sub-Total</td><td colspan="1" rowspan="1">Site 2</td><td colspan="1" rowspan="1">Sub-Total</td></tr><tr><td colspan="1" rowspan="3">Influenza A/H3</td><td colspan="1" rowspan="1">3× LoD</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">60/60100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">90/90100%(95.9%-100%)</td></tr><tr><td colspan="1" rowspan="1">1.5× LoD</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">29/30 a96.7%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">59/6098.3%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">89/9098.9%(94.0%-99.8%)</td></tr><tr><td colspan="1" rowspan="1">None(no analyte)</td><td colspan="1" rowspan="1">NotDetected</td><td colspan="1" rowspan="1">150/150100%</td><td colspan="1" rowspan="1">149/150 b99.%</td><td colspan="1" rowspan="1">299/30099.7%</td><td colspan="1" rowspan="1">150/150100%</td><td colspan="1" rowspan="1">150/150100%</td><td colspan="1" rowspan="1">449/45099.8%(98.8%-100%)</td></tr><tr><td colspan="1" rowspan="1">InfluenzaA/H1pdm09</td><td colspan="1" rowspan="1">None(no analyte)</td><td colspan="1" rowspan="1">NotDetected</td><td colspan="1" rowspan="1">210/210100%</td><td colspan="1" rowspan="1">210/210100%</td><td colspan="1" rowspan="1">420/420100%</td><td colspan="1" rowspan="1">210/210100%</td><td colspan="1" rowspan="1">210/210100%</td><td colspan="1" rowspan="1">630/630100%(99.4%-100%)</td></tr><tr><td colspan="1" rowspan="1">Influenza A/H1</td><td colspan="1" rowspan="1">None(no analyte)</td><td colspan="1" rowspan="1">NotDetected</td><td colspan="1" rowspan="1">210/210100%</td><td colspan="1" rowspan="1">210/210100%</td><td colspan="1" rowspan="1">420/420100%</td><td colspan="1" rowspan="1">210/210100%</td><td colspan="1" rowspan="1">210/210100%</td><td colspan="1" rowspan="1">630/630100%(99.4%-100%)</td></tr><tr><td colspan="1" rowspan="3">Influenza B</td><td colspan="1" rowspan="1">3× LoD</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">60/60100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">90/90100%(95.9%-100%)</td></tr><tr><td colspan="1" rowspan="1">1.5× LoD</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">29/3096.7%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">59/6098.3%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">89/9098.9%(94.0%-99.8%)</td></tr><tr><td colspan="1" rowspan="1">None(no analyte)</td><td colspan="1" rowspan="1">NotDetected</td><td colspan="1" rowspan="1">150/150100%</td><td colspan="1" rowspan="1">150/150100%</td><td colspan="1" rowspan="1">300/300100%</td><td colspan="1" rowspan="1">150/150100%</td><td colspan="1" rowspan="1">150/150100%</td><td colspan="1" rowspan="1">450/450100%(99.2%-100%)</td></tr><tr><td colspan="1" rowspan="1">ParainfluenzaVirus 1</td><td colspan="1" rowspan="1">None(no analyte)</td><td colspan="1" rowspan="1">NotDetected</td><td colspan="1" rowspan="1">210/210100%</td><td colspan="1" rowspan="1">210/210100%</td><td colspan="1" rowspan="1">420/420100%</td><td colspan="1" rowspan="1">210/210100%</td><td colspan="1" rowspan="1">210/210100%</td><td colspan="1" rowspan="1">630/630100%(99.4%-100%)</td></tr><tr><td colspan="1" rowspan="3">ParainfluenzaVirus 2</td><td colspan="1" rowspan="1">3× LoD</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">60/60100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">90/90100%(95.9%-100%)</td></tr><tr><td colspan="1" rowspan="1">1.5× LoD</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">60/60100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">90/9010%(95.9%-100%)</td></tr><tr><td colspan="1" rowspan="1">None(no analyte)</td><td colspan="1" rowspan="1">NotDetected</td><td colspan="1" rowspan="1">150/150100%</td><td colspan="1" rowspan="1">150/150100%</td><td colspan="1" rowspan="1">300/300100%</td><td colspan="1" rowspan="1">150/150100%</td><td colspan="1" rowspan="1">150/150100%</td><td colspan="1" rowspan="1">450/450100%(99.2%-100%)</td></tr><tr><td colspan="1" rowspan="3">ParainfluenzaVirus 3</td><td colspan="1" rowspan="1">3× LoD</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/3010%</td><td colspan="1" rowspan="1">60/60100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">90/90100%(95.9%-100%)</td></tr><tr><td colspan="1" rowspan="1">1.5× LoD</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">60/60100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/3010%</td><td colspan="1" rowspan="1">90/90100%(95.9%-100%)</td></tr><tr><td colspan="1" rowspan="1">None(no analyte)</td><td colspan="1" rowspan="1">NotDetected</td><td colspan="1" rowspan="1">150/150100%</td><td colspan="1" rowspan="1">150/150100%</td><td colspan="1" rowspan="1">300/300100%</td><td colspan="1" rowspan="1">150/150100%</td><td colspan="1" rowspan="1">150/150100%</td><td colspan="1" rowspan="1">450/450100%(99.2%-100%)</td></tr><tr><td colspan="1" rowspan="1">ParainfluenzaVirus 4</td><td colspan="1" rowspan="1">None(no analyte)</td><td colspan="1" rowspan="1">NotDetected</td><td colspan="1" rowspan="1">210/210100%</td><td colspan="1" rowspan="1">210/210100%</td><td colspan="1" rowspan="1">420/420100%</td><td colspan="1" rowspan="1">210/210100%</td><td colspan="1" rowspan="1">210/210100%</td><td colspan="1" rowspan="1">630/630100%(99.4%-100%)</td></tr><tr><td colspan="1" rowspan="3">RespiratorySyncytial Virus</td><td colspan="1" rowspan="1">3× LoD</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">29/30100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">59/6098.3%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">89/9098.9%(94.0%-99.8%)</td></tr><tr><td colspan="1" rowspan="1">1.5× LoD</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">29/30100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">59/6098.3%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">89/9098.9%(94.0%-99.8%)</td></tr><tr><td colspan="1" rowspan="1">None(no analyte)</td><td colspan="1" rowspan="1">NotDetected</td><td colspan="1" rowspan="1">150/150100%</td><td colspan="1" rowspan="1">150/150100%</td><td colspan="1" rowspan="1">300/300100%</td><td colspan="1" rowspan="1">150/150100%</td><td colspan="1" rowspan="1">150/150100%</td><td colspan="1" rowspan="1">450/450100%(99.2%-100%)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">B</td><td colspan="1" rowspan="1">Bacteria</td><td colspan="2" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="3">Mycoplasmapneumoniae</td><td colspan="1" rowspan="1">3× LoD</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">60/60100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">90/90100%(95.9%-100%)</td></tr><tr><td colspan="1" rowspan="1">1.5× LoD</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">60/60100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">90/90100%(95.9%-100%)</td></tr><tr><td colspan="1" rowspan="1">None(no analyte)</td><td colspan="1" rowspan="1">NotDetected</td><td colspan="1" rowspan="1">150/150100%</td><td colspan="1" rowspan="1">149/15099.3%</td><td colspan="1" rowspan="1">299/30099.7%</td><td colspan="1" rowspan="1">150/150100%</td><td colspan="1" rowspan="1">150/150100%</td><td colspan="1" rowspan="1">449/45099.8%(98.8%-100%)</td></tr><tr><td colspan="1" rowspan="3">Bordetellapertussis</td><td colspan="1" rowspan="1">3× LoD</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">60/60100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">90/90100%(95.9%-100%)</td></tr><tr><td colspan="1" rowspan="1">1.5× LoD</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/3010%</td><td colspan="1" rowspan="1">60/60100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">90/90100%(95.9%-100%)</td></tr><tr><td colspan="1" rowspan="1">None(no analyte)</td><td colspan="1" rowspan="1">NotDetected</td><td colspan="1" rowspan="1">150/150100%</td><td colspan="1" rowspan="1">150/150100%</td><td colspan="1" rowspan="1">300/300100%</td><td colspan="1" rowspan="1">150/150100%</td><td colspan="1" rowspan="1">150/150100%</td><td colspan="1" rowspan="1">450/450100%(99.2%-100%)</td></tr><tr><td colspan="1" rowspan="3">Chlamydiapneumoniae</td><td colspan="1" rowspan="1">3× LoD</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">60/60100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">90/90100%(95.9%-100%)</td></tr><tr><td colspan="1" rowspan="1">1.5× LoD</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">60/6010%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">90/90100%(95.9%-100%)</td></tr><tr><td colspan="1" rowspan="1">None(no analyte)</td><td colspan="1" rowspan="1">NotDetected</td><td colspan="1" rowspan="1">150/150100%</td><td colspan="1" rowspan="1">150/150100%</td><td colspan="1" rowspan="1">300/300100%</td><td colspan="1" rowspan="1">150/150100%</td><td colspan="1" rowspan="1">150/150100%</td><td colspan="1" rowspan="1">450/450100%(99.2%-100%)</td></tr><tr><td colspan="1" rowspan="1">a Thore wves an indoto</td><td colspan="1" rowspan="1">minote Flu A rogy</td><td></td><td></td><td></td><td></td><td></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr></table>

a There was an indeterminate Flu A result for Influenza A/H3 low positive sample.

b There was an indeterminate Flu A result for Influenza A/H3 negative sample.

2. Linearity: Not applicable, qualitative assay

3. Assay Reportable Range: Not applicable, qualitative assay

4. Traceability, Stability, Expected Values (Controls, Calibrators, or Methods):

# Assay Controls

# External Negative Controls

External Negative Controls are user provided. They can be RNase-free water, transport media, or well characterized negative specimens. The External Negative Controls should go through all processing steps (extractions, amplification, and detection). At least one External Negative Control is required for each plate/kit lot. The BioCode MDx-3000 software will suppress results for all samples if the External Negative Control is not valid.

# External Positive Controls

External Positive Controls are user provided. They can be well characterized clinical samples or cultured organisms. External Positive Controls can be single analytes or pooled multianalyte specimens. External Positive Controls should go through all processing steps (extractions, amplification, and detection). It is recommended that at least one External Positive Control be included for each plate/kit lot on a rotating schedule. Wells identified as External Positive Controls will be trended by the BioCode MDx-3000 software and the report will indicate a valid or invalid result on the report header. However, the software will not suppress results based on External Positive Control results. If an External Positive Control does not perform as expected, the user should review all samples in that plate to determine if results can be reported.

# Internal Control

An RNA Internal Control (RNA IC: bacteriophage MS2) is provided in the reagent kit and is added to each sample during extraction. The Internal Control monitors the efficiency of the extraction, reverse transcription, amplification, and detection stages of the assay. Positive target assay results may be reported in the absence of RNA IC detection. However, the BioCode MDx-3000 software will suppress target assay negative results for any wells with invalid RNA IC results.

External controls are not provided with the BioCode RPP. However, six frozen $( - 7 0 ^ { \circ } \mathrm { C } )$ external control mixes obtained from ZeptoMetrix as NATtrol controls for different pathogens (see Table 8 below) were provided to the clinical study sites for testing during the prospective clinical study and the clinical study testing retrospective and contrived specimens. Operators were required to follow the BioCode RPP instructions for use during the testing.

Table 8: External Control Mixes Utilized in the Clinical Evaluations   

<table><tr><td rowspan=1 colspan=1>External ControlMixes</td><td rowspan=1 colspan=1>Expected Calls</td></tr><tr><td rowspan=1 colspan=1>PC-A</td><td rowspan=1 colspan=1>Influenza B, Respiratory Syncytial Virus, Parainfluenza Virus 1, andRhinovirus/Enterovirus</td></tr><tr><td rowspan=1 colspan=1>PC-B</td><td rowspan=1 colspan=1>Coronavirus HKU1, Parainfluenza Virus 2, Human Metapneumovirus, andBordetella pertussis</td></tr><tr><td rowspan=1 colspan=1>PC-C</td><td rowspan=1 colspan=1>Influenza A/H1, Coronavirus NL63, and Adenovirus</td></tr><tr><td rowspan=1 colspan=1>PC-D</td><td rowspan=1 colspan=1>Influenza A/H1pdm09, Parainfluenza Virus 4, Coronavirus OC43, andChlamydia pneumoniae</td></tr><tr><td rowspan=1 colspan=1>PC-E</td><td rowspan=1 colspan=1>Influenza A/H3, Parainfluenza Virus 3, Coronavirus 229E, and Mycoplasmapneumoniae.</td></tr><tr><td rowspan=1 colspan=1>NC</td><td rowspan=1 colspan=1>Negative for all analytes</td></tr></table>

Performance of testing the external controls during the clinical studies are summarized in Table 9 below.

Table 9: External Controls Testing Summary   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=6>Agreement with Expected Results</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>PC-A</td><td rowspan=1 colspan=1>PC-B</td><td rowspan=1 colspan=1>PC-C</td><td rowspan=1 colspan=1>PC-D</td><td rowspan=1 colspan=1>PC-E</td><td rowspan=1 colspan=1>NC</td></tr><tr><td rowspan=1 colspan=1>NucliSENS easyMAG</td><td rowspan=1 colspan=1>9/10 a</td><td rowspan=1 colspan=1>8/9 a</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>11/11</td><td rowspan=1 colspan=1>8/8</td><td rowspan=1 colspan=1>50/53 b</td></tr><tr><td rowspan=1 colspan=1>MagNA Pure 96</td><td rowspan=1 colspan=1>4/4</td><td rowspan=1 colspan=1>5/5</td><td rowspan=1 colspan=1>4/4</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>20/20</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>13/14</td><td rowspan=1 colspan=1>13/14</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>14/14</td><td rowspan=1 colspan=1>11/11</td><td rowspan=1 colspan=1>70/73</td></tr></table>

a PC and NC switched during loading due to operator’s error. b Of the three failed NCs, one was due to RNA IC failure, one due to operator’s error (NC and PC switched during loading), and one due to detection of an unexpected target.

# Specimen Stability Study

Nasopharyngeal Swab (NPS) should be collected according to standard technique and immediately placed in $1 - 3 ~ \mathrm { m L }$ of VTM or UTM. Samples should be tested as soon as possible. An analytical study was performed to assess the specimen stability limitations for the optimal performance of the BioCode RPP on the BioCode MDx-3000. Nine representative pathogens from the BioCode RPP were spiked into prescreened negative natural NPS in UTM specimens at $3 \mathbf { x }$ LoD (see Table 10 below) and assayed with three extraction replicates on the easyMAG at each timepoint, except for baseline. A total of seven extraction replicates on the easyMAG were assayed at timepoint 0. Stability of extracted nucleic acids was also assessed in this study.

Table 10: Contrived Sample in Prescreened Negative Natural NPS in UTM Utilized in the Specimen Stability Study   

<table><tr><td rowspan=1 colspan=1>Sample Name</td><td rowspan=1 colspan=1>Pathogen Name and Concentration</td></tr><tr><td rowspan=1 colspan=1>RP A</td><td rowspan=1 colspan=1>Adenovirus (ADV at 3xLoD), Mycoplasma pneumoniae (MPN at 3xLoD), andInfluenza A/H3 (Flu A/H3 at 3xLoD)</td></tr><tr><td rowspan=1 colspan=1>RP B</td><td rowspan=1 colspan=1>Respiratory Syncytial Virus (RSV at 3xLoD), Influenza A/H1pdm09 (FluA/H1pdm09 at 3xLoD), and Human Metapneumovirus (hMPV at 3xLoD)</td></tr><tr><td rowspan=1 colspan=1>RPC</td><td rowspan=1 colspan=1>Parainfluenza Virus 3 (PIV 3 at 3xLoD), Coronavirus NL63 (CoV-NL63 at3xLoD), and Influenza B (Flu B at 3xLoD)</td></tr></table>

Specimen stability study summary results are presented in Table 11 below.

Table 11: Specimen Stability Study Summary Results   

<table><tr><td rowspan=3 colspan=1>SampleType</td><td rowspan=3 colspan=1>Temperature</td><td rowspan=3 colspan=1>Time(hoursor Days)</td><td rowspan=1 colspan=9>Agreement with Expected Results</td></tr><tr><td rowspan=1 colspan=3>Sample RP A</td><td rowspan=1 colspan=3>Sample RP B</td><td rowspan=1 colspan=3>Sample RP C</td></tr><tr><td rowspan=1 colspan=1>ADV</td><td rowspan=1 colspan=1>MPN</td><td rowspan=1 colspan=1>FluA/H3</td><td rowspan=1 colspan=1>RSV</td><td rowspan=1 colspan=1>Flu A/H1pdm09</td><td rowspan=1 colspan=1>hMPV</td><td rowspan=1 colspan=1>PIV 3</td><td rowspan=1 colspan=1>CoV-NL63</td><td rowspan=1 colspan=1>Flu B</td></tr><tr><td rowspan=1 colspan=1>Fresh(baseline)</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>7/7</td><td rowspan=1 colspan=1>7/7</td><td rowspan=1 colspan=1>7/7</td><td rowspan=1 colspan=1>7/7</td><td rowspan=1 colspan=1>7/7</td><td rowspan=1 colspan=1>7/7</td><td rowspan=1 colspan=1>7/7</td><td rowspan=1 colspan=1>7/7</td><td rowspan=1 colspan=1>7/7</td></tr><tr><td rowspan=8 colspan=1>ContrivedNPS</td><td rowspan=2 colspan=1>Room Temp</td><td rowspan=1 colspan=1>8 Hour</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>12 Hour</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=3 colspan=1>4°</td><td rowspan=1 colspan=1>5 Day</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>7 Day</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>10 Day</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=3 colspan=1>-80°</td><td rowspan=1 colspan=1>30 Day</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>60 Day</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>90 Day</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=2 colspan=1>Extracted</td><td rowspan=2 colspan=1>4°</td><td rowspan=1 colspan=1>8 Hour</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>12 Hour</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=3 colspan=1>NucleicAcids</td><td rowspan=3 colspan=1>-80(2x Freeze/Thaw)</td><td rowspan=1 colspan=1>30 Day</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>60 Day</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>90 Day</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td></tr></table>

The specimen stability testing data supports the following specimen stability claims in the product package insert:

NPS collected in VTM or UTM may be stored at room temperature for 8 hours, in a $2 { - } 8 ^ { \circ } \mathrm { C }$ refrigerator for 7 days, or at ${ \mathrm { < - 6 0 \mathrm { { } ^ { \circ } C } } }$ for up to 90 days.

Extracted nucleic acids may be stored in a $2 { - } 8 ^ { \circ } \mathrm { C }$ refrigerator for up to 12 hours, or at $\le - 6 0 ^ { \circ } \mathrm { C }$ for up to 90 days.

# Fresh vs. Frozen

The results of the Specimen Stability Study (see Table 11 above) also demonstrated that preservation of samples (by freezing at $- 8 0 ^ { \circ } \mathrm { C }$ ) does not affect the accuracy of test results compared to freshly prepared samples. Therefore, it is appropriate to utilize frozen archived prospective and retrospective clinical samples in the evaluation of BioCode RPP to supplement the prospective (fresh) clinical study data, and to use frozen simulated samples in analytical studies for this submission.

# Single-Spiked vs. Multi-Spiked Specimen Study

An analytical study was conducted to assess the performance of the BioCode RPP on the BioCode MDx-3000 with mixed analyte samples at or near the Limit of Detection (LoD) compared to single spiked samples to justify the use of multiple analyte spiked samples during analytical validation testing. LoD was determined for single spiked samples in simulated NPS for four representative pathogens on both the easyMAG and MagNA pure extraction systems. For initial range finding, four replicates of each concentration were extracted on the easyMAG and MagNA pure systems and tested in singlet with the BioCode RPP on the BioCode MDx-3000 system to estimate LoD. The LoD was further confirmed by extracting 20 replicates of each sample and testing each in singlet for a total of 20 replicates at or near the presumptive LoD. The single spiked LoDs were then challenged by pooling all four pathogens and testing at 1x LoD with 20 extraction replicates on both extraction systems. When 1x LoD for each pathogen in the mixed pool does not meet the $9 5 \%$ detection goal, the pooled pathogen samples were retested at 3x LoD with 20 extraction replicates. Results within $3 \mathbf { x }$ LoD were considered to be equivalent in this study.

Summary results of this study are presented in Table 12 below.

Table 12: Single-Spiked vs. Multi-Spiked Study Summary Results   

<table><tr><td rowspan=2 colspan=1>Pathogen</td><td rowspan=2 colspan=1>Targetssay</td><td rowspan=2 colspan=1>Single-Spikedor Multi-Spiked</td><td rowspan=1 colspan=2>NucliSENS easyMAG</td><td rowspan=1 colspan=2>MagNA Pure 96</td></tr><tr><td rowspan=1 colspan=1>Concentration</td><td rowspan=1 colspan=1>Detection(n/20)</td><td rowspan=1 colspan=1>Concentration</td><td rowspan=1 colspan=1>Detection(n/20)</td></tr><tr><td rowspan=4 colspan=1>Influenza A/H3N2</td><td rowspan=1 colspan=1>Flu A</td><td rowspan=2 colspan=1>Single-Spiked</td><td rowspan=2 colspan=1>1.3 TCID50/mL</td><td rowspan=1 colspan=1>20/20</td><td rowspan=4 colspan=1>1.3 TCID50/mL</td><td rowspan=1 colspan=1>20/20</td></tr><tr><td rowspan=1 colspan=1>Flu A/H3</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>20/20</td></tr><tr><td rowspan=1 colspan=1>Flu A</td><td rowspan=2 colspan=1>Multi-Spiked</td><td rowspan=2 colspan=1>4.0 TCID50/mL</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>20/20</td></tr><tr><td rowspan=1 colspan=1>Flu A/H3</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>20/20</td></tr><tr><td rowspan=2 colspan=1>Coronavirus NL63</td><td rowspan=2 colspan=1>NL 63</td><td rowspan=1 colspan=1>Single-Spiked</td><td rowspan=1 colspan=1>0.013 TCID50/mL</td><td rowspan=1 colspan=1>20/20</td><td rowspan=2 colspan=1>0.040 TCID50/mL</td><td rowspan=1 colspan=1>19/20</td></tr><tr><td rowspan=1 colspan=1>Multi-Spiked</td><td rowspan=1 colspan=1>0.040 TCID50/mL</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>20/20</td></tr><tr><td rowspan=2 colspan=1>Mycoplasma pneumoniae</td><td rowspan=2 colspan=1>MPN</td><td rowspan=1 colspan=1>Single-Spiked</td><td rowspan=1 colspan=1>5.0 CCU/mL</td><td rowspan=1 colspan=1>19/20</td><td rowspan=2 colspan=1>15.0 CCU/mL</td><td rowspan=1 colspan=1>20/20</td></tr><tr><td rowspan=1 colspan=1>Multi-Spiked</td><td rowspan=1 colspan=1>15.0 CCU/mL</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>20/20</td></tr><tr><td rowspan=2 colspan=1>Adenovirus C</td><td rowspan=2 colspan=1>ADV1</td><td rowspan=1 colspan=1>Single-Spiked</td><td rowspan=2 colspan=1>6.0 TCID50/mL</td><td rowspan=1 colspan=1>20/20</td><td rowspan=2 colspan=1>18.0 TCID50/mL</td><td rowspan=1 colspan=1>20/20</td></tr><tr><td rowspan=1 colspan=1>Multi-Spiked</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>20/20</td></tr></table>

The acceptance criteria of this study were met since all pathogens tested were within 3xLoD for single-spiked and multi-spiked samples. The study results seem to demonstrate that the performance of the BioCode RPP on the BioCode MDx-3000 with mixed analyte samples at or near the Limit of Detection (LoD) is similar to single spiked samples. Using multiple analyte spiked samples during analytical validation testing appears to be acceptable.

# Simulated vs. Natural NPS in UTM Specimen Study

Analytical validation studies are mostly performed with contrived specimens (i.e., spiking known concentrations of pathogens into negative NPS in UTM or VTM matrix). This would require a large volume of negative natural NPS to be collected and screened, while typically only between 1.0 to $2 . 0 \mathrm { m L }$ per donor could be obtained after NPS collection and screening to confirm negative status. This makes collection from donors burdensome. An analytical study was performed to assess the performance equivalency of testing a simulated NPS (sNPS) in UTM matrix compared to testing natural NPS in UTM using the BioCode RPP, with both the NucliSENS easyMAG and the MagNA Pure 96 extraction systems. The sNPS in UTM matrix consists of $2 \mathrm { x } 1 0 ^ { 3 }$ HeLa cells/mL diluted in UTM.

Samples were contrived in negative natural NPS in UTM matrix and in sNPS in UTM matrix with multi-spiked pathogens at close to LoD levels (approximately $1 . 5 \mathrm { x }$ to 15.0xLoD). Each sample was extracted in quadruplicate on both the easyMAG and the MagNA Pure 96 extraction system and tested singly with the BioCode RPP on the BioCode MDx-3000 system.

Summary results of this analytical study are presented in Table 13 below.

Table 13: Simulated vs. Natural NPS in UTM Specimen Study Summary Results   

<table><tr><td rowspan=3 colspan=1>Pools</td><td rowspan=3 colspan=1>Pathogen</td><td rowspan=3 colspan=1>Concentration</td><td rowspan=1 colspan=4>Detected (N of 4)</td></tr><tr><td rowspan=1 colspan=2>NucliSENS easyMAG</td><td rowspan=1 colspan=2>MagNA Pure 96</td></tr><tr><td rowspan=1 colspan=1>NaturalNPS inUTM</td><td rowspan=1 colspan=1>sNPS inUTM</td><td rowspan=1 colspan=1>NaturalNPS inUTM</td><td rowspan=1 colspan=1>sNPS inUTM</td></tr><tr><td rowspan=1 colspan=1>A</td><td rowspan=1 colspan=1>Adenovirus E Serotype 4</td><td rowspan=1 colspan=1>0.6 TCID50/mL (15xLoD)</td><td rowspan=1 colspan=1>4/4</td><td rowspan=1 colspan=1>4/4</td><td rowspan=1 colspan=1>4/4</td><td rowspan=1 colspan=1>4/4</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Chlamydia pneumoniae</td><td rowspan=1 colspan=1>75.0 CFU/mL (2.3 - 4.5xLoD)</td><td rowspan=1 colspan=1>4/4</td><td rowspan=1 colspan=1>4/4</td><td rowspan=1 colspan=1>4/4</td><td rowspan=1 colspan=1>4/4</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Influenza A/H3N2</td><td rowspan=1 colspan=1>6.0 TCID50/mL (1.5 -4.6xLoD)</td><td rowspan=1 colspan=1>4/4</td><td rowspan=1 colspan=1>4/4</td><td rowspan=1 colspan=1>4/4</td><td rowspan=1 colspan=1>4/4</td></tr><tr><td rowspan=1 colspan=1>B</td><td rowspan=1 colspan=1>Influenza A/H1N1</td><td rowspan=1 colspan=1>67.5 TCID50/mL (4.5 - 13.5xLoD)</td><td rowspan=1 colspan=1>4/4</td><td rowspan=1 colspan=1>4/4</td><td rowspan=1 colspan=1>4/4</td><td rowspan=1 colspan=1>4/4</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Respiratory SyncytialVirus A</td><td rowspan=1 colspan=1>1.5 TCID50/mL (4.5xLoD)</td><td rowspan=1 colspan=1>4/4</td><td rowspan=1 colspan=1>4/4</td><td rowspan=1 colspan=1>4/4</td><td rowspan=1 colspan=1>4/4</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>HumanMetapneumovirus</td><td rowspan=1 colspan=1>67.5 TCID50/mL (4.5xLoD)</td><td rowspan=1 colspan=1>4/4</td><td rowspan=1 colspan=1>4/4</td><td rowspan=1 colspan=1>4/4</td><td rowspan=1 colspan=1>4/4</td></tr><tr><td rowspan=1 colspan=1>C</td><td rowspan=1 colspan=1>Influenza B</td><td rowspan=1 colspan=1>0.06 TCID50/mL (6.0xLoD)</td><td rowspan=1 colspan=1>4/4</td><td rowspan=1 colspan=1>4/4</td><td rowspan=1 colspan=1>4/4</td><td rowspan=1 colspan=1>4/4</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Coronavirus OC43</td><td rowspan=1 colspan=1>0.06 TCID50/mL (1.5 - 6.0xLoD)</td><td rowspan=1 colspan=1>4/4</td><td rowspan=1 colspan=1>4/4</td><td rowspan=1 colspan=1>4/4</td><td rowspan=1 colspan=1>4/4</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Parainfluenza Virus 4</td><td rowspan=1 colspan=1>13.5 TCID50/mL (1.5xLoD)</td><td rowspan=1 colspan=1>4/4</td><td rowspan=1 colspan=1>4/4</td><td rowspan=1 colspan=1>4/4</td><td rowspan=1 colspan=1>4/4</td></tr><tr><td rowspan=1 colspan=2>Negative Control</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>4/4</td><td rowspan=1 colspan=1>4/4</td><td rowspan=1 colspan=1>4/4</td><td rowspan=1 colspan=1>4/4</td></tr></table>

The study results show that the performance testing contrived samples in this simulated NPS (sNPS) in UTM matrix compared to testing contrived samples in natural NPS in UTM matrix using the BioCode RPP, with both the NucliSENS easyMAG and the MagNA Pure 96 extraction systems, are similar. Therefore, it is acceptable to utilize the simulated NPS in UTM sample matrix in conducting the analytical studies in support of this submission.

# 5. Detection Limit:

Limit of detection (LoD) estimation and confirmation studies were carried out with contrived samples in simulated NPS in UTM (Remel M4 transport medium) matrix designed to resemble a natural clinical NPS in UTM specimen. An equivalence study was performed which demonstrated that the simulated matrix was similar to the natural clinical NPS matrix and did not impact BioCode RPP test performance. Refer to the “Simulated vs. Natural NPS in UTM Specimen Study” section.

Representative isolates of respiratory viruses and bacteria were selected to make contrived samples in order to obtain positive results for every target assay on the panel. In some cases, testing of more than one isolate/strain per analyte was performed to assess LoD for clinically important species or variants, specifically when more than one assay was needed to detect the expected diversity of an analyte (e.g. Adenovirus).

An estimated LoD concentration for each analyte was first determined by testing serial dilutions of contrived samples. Four extraction replicates per dilution on each extraction system, NucliSENS easyMAG and MagNA Pure 96, were tested using the BioCode RPP. The LoD was confirmed by extracting 20 replicates on each extraction system and testing each for a total of 20 replicates at or near the estimated LoD using the BioCode RPP. LoD for each isolate was defined as the lowest concentration with $2 9 5 \%$ detection of 20 replicates (at least 19 out of 20 replicates).

A multi-spiked approach was employed in both the LoD estimation and the LoD confirmation studies where samples spiked with two pathogens at various concentrations were tested in the LoD studies. An equivalence study was performed which demonstrated that the performance of the multi-spiked samples was similar to that of the single-spiked samples and employing a multi-spiking approach did not impact BioCode RPP test performance. Refer to the “Single-Spiked vs. Multi-Spiked Specimen Study” section.

Summary results of the LoD confirmation study are presented in Table 14 below.

Table 14: Limit of Detection Confirmation by Extraction System   

<table><tr><td rowspan=1 colspan=1>Pathogen</td><td rowspan=1 colspan=1>Strain/Isolate</td><td rowspan=1 colspan=1>Source</td><td rowspan=1 colspan=2>NucliSENS easyMAG</td><td rowspan=1 colspan=2>MagNA Pure 96</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Concentration</td><td rowspan=1 colspan=1>Detected(n of 20)</td><td rowspan=1 colspan=1>Concentration</td><td rowspan=1 colspan=1>Detected(n of 20)</td></tr><tr><td rowspan=2 colspan=1>Influenza A/H1</td><td rowspan=1 colspan=1>A/New/Caledonia/20/99</td><td rowspan=1 colspan=1>Zeptometrix080036CF</td><td rowspan=1 colspan=1>15.0 TCID50/mL</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>5.0 TCID50/mL</td><td rowspan=1 colspan=1>20/20</td></tr><tr><td rowspan=1 colspan=1>A/NWS/33</td><td rowspan=1 colspan=1>ATCC VR-219</td><td rowspan=1 colspan=1>27.0 TCID50/mL</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>9.0 TCID50/mL</td><td rowspan=1 colspan=1>20/20</td></tr><tr><td rowspan=1 colspan=1>Influenza A/H1pdm09</td><td rowspan=1 colspan=1>A(H1N1)/California/07/09</td><td rowspan=1 colspan=1>Zeptometrix0810165CF</td><td rowspan=1 colspan=1>0.4 TCID50/mL</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0.4 TCID50/mL</td><td rowspan=1 colspan=1>20/20</td></tr><tr><td rowspan=2 colspan=1>Influenza A/H3</td><td rowspan=1 colspan=1>A/Wisconsin/67/05</td><td rowspan=1 colspan=1>Zeptometrix0810252CF</td><td rowspan=1 colspan=1>4.0 TCID50/mL</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>1.3 TCID50/mL</td><td rowspan=1 colspan=1>20/20</td></tr><tr><td rowspan=1 colspan=1>A/Alice</td><td rowspan=1 colspan=1>ATCC VR-776</td><td rowspan=1 colspan=1>27.0 TCID50/mL</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>9.0 TCID50/mL</td><td rowspan=1 colspan=1>19/20</td></tr><tr><td rowspan=2 colspan=1>Influenza B</td><td rowspan=1 colspan=1>B/Florida/4/2006(Yamagata)</td><td rowspan=1 colspan=1>Zeptometrix0810255CF</td><td rowspan=1 colspan=1>0.01 TCID50/mL</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0.01 TCID50/mL</td><td rowspan=1 colspan=1>20/20</td></tr><tr><td rowspan=1 colspan=1>B/Hong Kong/S/1972(Victoria</td><td rowspan=1 colspan=1>ATCC VR-823</td><td rowspan=1 colspan=1>48.6 TCID50/mL</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>48.6 TCID50/mL</td><td rowspan=1 colspan=1>20/20</td></tr><tr><td rowspan=1 colspan=1>Respiratory SyncytialVirus</td><td rowspan=1 colspan=1>Type A</td><td rowspan=1 colspan=1>Zeptometrix0810040ACF</td><td rowspan=1 colspan=1>0.33 TCID50/mL</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0.33 TCID50/mL</td><td rowspan=1 colspan=1>20/20</td></tr><tr><td rowspan=1 colspan=1>HumanMetapneumovirus</td><td rowspan=1 colspan=1>16; Type A1 IA10-2003</td><td rowspan=1 colspan=1>Zeptometrix0810161CF</td><td rowspan=1 colspan=1>15.0 TCID50/mL</td><td rowspan=1 colspan=1>19/20</td><td rowspan=1 colspan=1>15.0 TCID50/mL</td><td rowspan=1 colspan=1>20/20</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza Virus 1</td><td rowspan=1 colspan=1>C-35/WashingtonDC/1957</td><td rowspan=1 colspan=1>ATCC VR-94</td><td rowspan=1 colspan=1>9.0 TCID50/mL</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>9.0 TCID50/mL</td><td rowspan=1 colspan=1>20/20</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza Virus 2</td><td rowspan=1 colspan=1>Geer/Ohio/1955</td><td rowspan=1 colspan=1>ATCC VR-92</td><td rowspan=1 colspan=1>1.8 TCID50/mL</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>5.4 TCID50/mL</td><td rowspan=1 colspan=1>20/20</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza Virus 3</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>Zeptometrix0810016F</td><td rowspan=1 colspan=1>15.0 TCID50/mL</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>15.0 TCID50/mL</td><td rowspan=1 colspan=1>20/20</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza Virus 4</td><td rowspan=1 colspan=1>Type 4a</td><td rowspan=1 colspan=1>Zeptometrix0810060CF</td><td rowspan=1 colspan=1>9.0 TCID50/mL</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>9.0 TCID50/mL</td><td rowspan=1 colspan=1>20/20</td></tr><tr><td rowspan=3 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>Species B Serotype 7A</td><td rowspan=1 colspan=1>Zeptometrix08021CF</td><td rowspan=1 colspan=1>1.2 TCID50/mL</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>1.2 TCID50/mL</td><td rowspan=1 colspan=1>20/20</td></tr><tr><td rowspan=1 colspan=1>Species C Serotype 2</td><td rowspan=1 colspan=1>ATCC VR-846</td><td rowspan=1 colspan=1>6.0 TCID50/mL</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>18.0 TCID50/mL</td><td rowspan=1 colspan=1>20/20</td></tr><tr><td rowspan=1 colspan=1>Species E Serotype 4</td><td rowspan=1 colspan=1>Zeptometrix00810070CF</td><td rowspan=1 colspan=1>0.04 TCID50/mL</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0.04 TCID50/mL</td><td rowspan=1 colspan=1>19/20</td></tr><tr><td rowspan=1 colspan=1>Coronavirus 229E</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>Zeptometrix0810229CF</td><td rowspan=1 colspan=1>0.6 TCID50/mL</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>1.8 TCID50/mL</td><td rowspan=1 colspan=1>20/20</td></tr><tr><td rowspan=1 colspan=1>Coronavirus HKU1</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>Clinical Sample4922 a</td><td rowspan=1 colspan=1>5.02x104Copies/mL</td><td rowspan=1 colspan=1>19/20</td><td rowspan=1 colspan=1>5.02x104Copies/mL</td><td rowspan=1 colspan=1>20/20</td></tr><tr><td rowspan=1 colspan=1>Coronavirus NL63</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>Zeptometrix0810228CF</td><td rowspan=1 colspan=1>0.04 TCID50/mL</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0.04 TCID50/mL</td><td rowspan=1 colspan=1>20/20</td></tr><tr><td rowspan=1 colspan=1>Coronavirus OC43</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>Zeptometrix0810024CF</td><td rowspan=1 colspan=1>0.04 TCID50/mL</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0.01 TCID50/mL</td><td rowspan=1 colspan=1>19/20</td></tr><tr><td rowspan=1 colspan=1>Rhinovirus</td><td rowspan=1 colspan=1>Type A1</td><td rowspan=1 colspan=1>Zeptometrix0810012CF</td><td rowspan=1 colspan=1>1.2 TCID50/mL</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0.4 TCID50/mL</td><td rowspan=1 colspan=1>19/20</td></tr><tr><td rowspan=1 colspan=1>Enterovirus</td><td rowspan=1 colspan=1>D68</td><td rowspan=1 colspan=1>Zeptometrix0810300CF</td><td rowspan=1 colspan=1>3.0 TCID50/mL</td><td rowspan=1 colspan=1>19/20</td><td rowspan=1 colspan=1>9.0 TCID50/mL</td><td rowspan=1 colspan=1>20/20</td></tr><tr><td rowspan=1 colspan=1>Bordetella pertussis</td><td rowspan=1 colspan=1>A639</td><td rowspan=1 colspan=1>Zeptometrix801459</td><td rowspan=1 colspan=1>15.0 CFU/mL</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>45.0 CFU/mL</td><td rowspan=1 colspan=1>19/20</td></tr><tr><td rowspan=1 colspan=1>Chlamydiapneumoniae</td><td rowspan=1 colspan=1>AR39</td><td rowspan=1 colspan=1>ATCC VR-53592</td><td rowspan=1 colspan=1>16.7 CFU/mL</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>33.3 CFU/mL</td><td rowspan=1 colspan=1>20/20</td></tr><tr><td rowspan=1 colspan=1>Mycoplasmapneumoniae</td><td rowspan=1 colspan=1>M129</td><td rowspan=1 colspan=1>Zeptometrix0801579</td><td rowspan=1 colspan=1>15.0 CCU/mL</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>15.0 CCU/mL</td><td rowspan=1 colspan=1>20/20</td></tr></table>

a Coronavirus HKU1 clinical sample quantified with Applied BioCode validated SYBR assay using an IVT RNA standard.

# 6. Analytical Reactivity:

An analytical study was performed to assess analytical reactivity/inclusivity of the BioCode RPP. Different strains, serotypes and genotypes were selected that represent temporal, geographic, and genetic diversity for each pathogen. A panel of titered stocks for relevant organisms/viruses diluted in simulated NPS in UTM matrix were tested starting at $3 \mathbf { x }$ LoD. Organisms/viruses not detected at $3 \mathbf { x }$ LoD were tested at higher concentrations. Since the confirmed LoD result for B/Hong Kong/S/1972 (Victoria lineage) was higher than expected at $4 8 . 6 \mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ , which could be related to difference in titration from vendor sources, testing of the influenza B Victoria strains in this study was performed starting at 0.03xLoD or 0.3xLoD. Strains of unknown lineages of influenza B were tested at $3 \mathbf { x }$ LoD for the B/Florida/4/2006 strain of the Yamagata lineage.

Each sample was extracted in triplicate on the NucliSENS easyMAG and tested with the BioCode RPP on the BioCode MDx-3000 according to the instructions for use.

Note: Although this analytical reactivity testing was performed in part to demonstrate that the BioCode RPP can detect various species and strains of a pathogen with similar analytical sensitivity, it is difficult to make comparative sensitivity evaluations between different organisms and different strains that have been quantified in $\mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ or $\mathrm { E I D } _ { 5 0 } / \mathrm { m L }$ . This quantification method measures the infectivity and cytotoxicity or lethality of a strain in tissue culture or in chick embryo, and is therefore subject to many influences (e.g., strain-tostrain differences in infectivity, viability of the original material, culturing conditions, etc.). Quantification by infectivity assay will not be equivalent between strains or organisms. Also, since the measurand for the BioCode RPP is nucleic acids, LoD concentrations established in $\mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ or $\mathrm { E I D } _ { 5 0 } / \mathrm { m L }$ can vary dramatically among different strains but may not actually reflect significant differences in analytical reactivity of detection as measured by target nucleic acids concentration.

Analytical reactivity testing results are summarized in Table 15 to Table 25 below.

Table 15: Adenovirus Isolates Tested and Detected by BioCode RPP   

<table><tr><td rowspan=1 colspan=1>Species a</td><td rowspan=1 colspan=1>Serotype</td><td rowspan=1 colspan=1>Isolate ID/Source</td><td rowspan=1 colspan=1>Strain/Location/Year</td><td rowspan=1 colspan=1>xLoDDetected</td><td rowspan=1 colspan=1>Result</td></tr><tr><td rowspan=3 colspan=1>A</td><td rowspan=1 colspan=1>31</td><td rowspan=1 colspan=1>Zeptometrix 0810073CF</td><td rowspan=1 colspan=1>Unknown</td><td rowspan=1 colspan=1>3x</td><td rowspan=17 colspan=1>AdenovirusDetected</td></tr><tr><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>ATCC VR-863</td><td rowspan=1 colspan=1>Huie/Massachusetts</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>18</td><td rowspan=1 colspan=1>ATCC VR-19</td><td rowspan=1 colspan=1>Washington DC/1954</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=4 colspan=1>B</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>ATCC VR-3</td><td rowspan=1 colspan=1>GB/Maryland/1953</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>Zeptometrix 0810108CF</td><td rowspan=1 colspan=1>Unknown</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>ATCC VR-17</td><td rowspan=1 colspan=1>CH.79/Saudi Arabia/1955</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>35</td><td rowspan=1 colspan=1>ATCC VR-718</td><td rowspan=1 colspan=1>Holden</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=3 colspan=1>C</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>Zeptometrix 0810050CF</td><td rowspan=1 colspan=1>Unknown</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>Zeptometrix 0810020CF</td><td rowspan=1 colspan=1>Unknown</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>ATCC VR-6</td><td rowspan=1 colspan=1>Tonsil 99/Washington DC</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=5 colspan=1>D</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>Zeptometrix 0810069CF</td><td rowspan=1 colspan=1>Unknown</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>ATCC VR-1836</td><td rowspan=1 colspan=1>CH.22/Saudi Arabia/1955</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>Zeptometrix 0810115CF</td><td rowspan=1 colspan=1>Unknown</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>26</td><td rowspan=1 colspan=1>Zeptometrix 0810117CF</td><td rowspan=1 colspan=1>Unknown</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>37</td><td rowspan=1 colspan=1>Zeptometrix 0810119CF</td><td rowspan=1 colspan=1>Unknown</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=2 colspan=1>F</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>Zeptometrix 0810084CF</td><td rowspan=1 colspan=1>Unknown</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>41</td><td rowspan=1 colspan=1>ATCC VR-930</td><td rowspan=1 colspan=1>Tak/73-3544/Netherlands/1973</td><td rowspan=1 colspan=1>3x</td></tr></table>

a In addition to the Adenovirus species E serotype 4 strain tested in the LoD study, in silico analysis of available sequences predicts that the BioCode RPP will also react with other Adenovirus species $\mathrm { E }$ serotypes.

Table 16: Coronavirus Isolates/Specimens Tested and Detected by BioCode RPP   

<table><tr><td rowspan=1 colspan=1>CoronavirusType</td><td rowspan=1 colspan=1>Isolate ID/Source</td><td rowspan=1 colspan=1>Location/Year</td><td rowspan=1 colspan=1>xLoDDetected</td><td rowspan=1 colspan=1>Result</td></tr><tr><td rowspan=1 colspan=1>229E</td><td rowspan=1 colspan=1>ATCC VR-740</td><td rowspan=1 colspan=1>Unknown</td><td rowspan=1 colspan=1>3x</td><td rowspan=1 colspan=1>CoronavirusDetected</td></tr><tr><td rowspan=1 colspan=1>NL63</td><td rowspan=1 colspan=1>BEI NR-470 a</td><td rowspan=1 colspan=1>Amsterdam/2003</td><td rowspan=1 colspan=1>3x</td><td rowspan=1 colspan=1>CoronavirusDetected</td></tr><tr><td rowspan=1 colspan=1>OC43</td><td rowspan=1 colspan=1>ATCC VR-1558</td><td rowspan=1 colspan=1>Unknown</td><td rowspan=1 colspan=1>3x</td><td rowspan=1 colspan=1>CoronavirusDetected</td></tr><tr><td rowspan=3 colspan=1>HKU1</td><td rowspan=1 colspan=1>Clinical Sample 5016</td><td rowspan=1 colspan=1>Unknown</td><td rowspan=1 colspan=1>3x</td><td rowspan=3 colspan=1>CoronavirusDetected</td></tr><tr><td rowspan=1 colspan=1>Clinical Sample 5036</td><td rowspan=1 colspan=1>Unknown</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>Clinical Sample 5037</td><td rowspan=1 colspan=1>Unknown</td><td rowspan=1 colspan=1>10x</td></tr></table>

a Organism obtained through BEI Resources, NIAID, NIH: Human Coronavirus NL63, NR-470.

able 17: Human Metapneumovirus Isolates Tested and Detected by BioCode RPP   

<table><tr><td rowspan=1 colspan=1>Genotype</td><td rowspan=1 colspan=1>Serotype</td><td rowspan=1 colspan=1>Isolate ID/Source</td><td rowspan=1 colspan=1>Location/Year</td><td rowspan=1 colspan=1>xLoDDetected</td><td rowspan=1 colspan=1>Result</td></tr><tr><td rowspan=1 colspan=1>A1</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>Zeptometrix 0810160CF</td><td rowspan=1 colspan=1>Iowa3/2002</td><td rowspan=1 colspan=1>3x</td><td rowspan=8 colspan=1>HumanMetapneumovirusDetected</td></tr><tr><td rowspan=1 colspan=1>A2</td><td rowspan=1 colspan=1>27</td><td rowspan=1 colspan=1>Zeptometrix 0810164CF</td><td rowspan=1 colspan=1>Iowa27/2004</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=2 colspan=1>B1</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>Zeptometrix 0810156CF</td><td rowspan=1 colspan=1>Peru2/2002</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>Zeptometrix 0810158CF</td><td rowspan=1 colspan=1>Peru3/2003</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=4 colspan=1>B2</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>Zeptometrix 0810157CF</td><td rowspan=1 colspan=1>Peru1/2002</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>Zeptometrix 0810159CF</td><td rowspan=1 colspan=1>Peru6/2003</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>18</td><td rowspan=1 colspan=1>Zeptometrix 0810162CF</td><td rowspan=1 colspan=1>Iowa18/2003</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>Unknown</td><td rowspan=1 colspan=1>BEI NR-22232 a</td><td rowspan=1 colspan=1>TN/91-316</td><td rowspan=1 colspan=1>3x</td></tr></table>

a Virus obtained through BEI Resources, NIAID, NIH: Human Metapneumovirus, TN/91-316, NR-22232.

Table 18: Human Rhinovirus and Enterovirus Isolates Tested and Detected by BioCode RPP   

<table><tr><td rowspan=1 colspan=1>Species</td><td rowspan=1 colspan=1>Serotype/Strain/Isolate</td><td rowspan=1 colspan=1>Source</td><td rowspan=1 colspan=1>xLoDDetected</td><td rowspan=1 colspan=1>Result</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=4>Human Rhinovirus</td></tr><tr><td rowspan=9 colspan=1>A</td><td rowspan=1 colspan=1>Serotype 7 [68-CV11]</td><td rowspan=1 colspan=1>ATCC VR-1601</td><td rowspan=1 colspan=1>3x</td><td rowspan=13 colspan=1>Human Rhinovirus/Enterovirus Detected</td></tr><tr><td rowspan=1 colspan=1>Serotype 16 [1A]</td><td rowspan=1 colspan=1>Zeptometrix 0810285CF</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>Serotype 16[11757/WashingtonDC/1960]</td><td rowspan=1 colspan=1>ATCC VR-283</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>Serotype 34 [173-3]</td><td rowspan=1 colspan=1>ATCC VR-1365</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>Serotype 57 [ch47]</td><td rowspan=1 colspan=1>ATCC VR-1600</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>Serotype 77 [130-63]</td><td rowspan=1 colspan=1>ATCC VR-1187</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>Serotype 80</td><td rowspan=1 colspan=1>Zeptometrix 0810288CF</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>Serotype 85 [50-525-CV54]</td><td rowspan=1 colspan=1>ATCC VR-1195</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>Serotype 95 [SF-998]</td><td rowspan=1 colspan=1>ATCC VR-1301</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=4 colspan=1>B</td><td rowspan=1 colspan=1>Serotype 3 [FEB]</td><td rowspan=1 colspan=1>ATCC VR-483</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>Serotype 14</td><td rowspan=1 colspan=1>Zeptometrix 0810284CF</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>Serotype 42</td><td rowspan=1 colspan=1>Zeptometrix 0810286CF</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>Serotype 70</td><td rowspan=1 colspan=1>Zeptometrix 0810287CF</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=2></td><td rowspan=1 colspan=1>Enterovirus</td><td rowspan=1 colspan=2></td></tr><tr><td rowspan=1 colspan=1>A</td><td rowspan=1 colspan=1>Enterovirus 71</td><td rowspan=1 colspan=1>Zeptometrix 0810236CF</td><td rowspan=1 colspan=1>3x</td><td rowspan=1 colspan=1>Human Rhinovirus/Enterovirus Detected</td></tr></table>

Table 19: Influenza A Isolates Tested and Detected by BioCode RPP   

<table><tr><td colspan="3" rowspan="1">Table 19: Thiluenza A Isolates Tested and Detected by BloCode RPP</td><td></td><td></td></tr><tr><td colspan="1" rowspan="1">Influenza A Type a</td><td colspan="1" rowspan="1">Strain</td><td colspan="1" rowspan="1">Source</td><td colspan="1" rowspan="1">xLoDDetected</td><td colspan="1" rowspan="1">Result</td></tr><tr><td colspan="1" rowspan="11">Influenza A H1N1</td><td colspan="1" rowspan="1">Solomon Island/03/06</td><td colspan="1" rowspan="1">Zeptometrix 0810036CFN</td><td colspan="1" rowspan="1">3x</td><td colspan="1" rowspan="12">Influenza A/H1Detected</td></tr><tr><td colspan="1" rowspan="1">Singapore/63/04</td><td colspan="1" rowspan="1">Zeptometrix 0810246CF</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="1">PR/8/34</td><td colspan="1" rowspan="1">Zeptometrix 0810245CF</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="1">A/Brisbane/59/2007</td><td colspan="1" rowspan="1">Zeptometrix 0810244CF</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="1">A/Taiwan/42/06</td><td colspan="1" rowspan="1">Zeptometrix 0810247CF</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="1">A/New jersey/8/76</td><td colspan="1" rowspan="1">ATCC VR-897</td><td colspan="1" rowspan="1">500x b</td></tr><tr><td colspan="1" rowspan="1">A/Denver/1/1957</td><td colspan="1" rowspan="1">VIRAPUR</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="1">A/FM/1/47</td><td colspan="1" rowspan="1">ATCC VR-97</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="1">A/Weiss/43</td><td colspan="1" rowspan="1">ATCC VR-96</td><td colspan="1" rowspan="1">1000x c</td></tr><tr><td colspan="1" rowspan="1">A/Beijing/262/95 d</td><td colspan="1" rowspan="1">BEI NR-12277</td><td colspan="1" rowspan="1">30x e</td></tr><tr><td colspan="1" rowspan="1">A/Mal/302/54</td><td colspan="1" rowspan="1">ATCC VR-98</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="1">Influenza A H1N2</td><td colspan="1" rowspan="1">Recombinant; Kilbourne F64,A/NWS/1934 (HA) xA/Rockefeller Institute/ 5/1957(NA) f</td><td colspan="1" rowspan="1">BEI NR-3682</td><td colspan="1" rowspan="1">5x g</td></tr><tr><td colspan="1" rowspan="9">Influenza A H1N1pdm09</td><td colspan="1" rowspan="1">NY/01/09</td><td colspan="1" rowspan="1">Zeptometrix 0810248CF</td><td colspan="1" rowspan="1">3x</td><td colspan="1" rowspan="9">Influenza A/H1pdm09 Detected</td></tr><tr><td colspan="1" rowspan="1">NY/02/09</td><td colspan="1" rowspan="1">Zeptometrix 0810109CFN</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="1">NY/03/09</td><td colspan="1" rowspan="1">Zeptometrix 0810249CF</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="1">A/Houston/3H/2009 (H1N1)pdm h</td><td colspan="1" rowspan="1">BEI NR-20340</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="1">Influenza A H1N1pdm09(Canada/6294/09)</td><td colspan="1" rowspan="1">Zeptometrix 0810109CFJ</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="1">Influenza A H1N1pdm09(Mexico/4108/09)</td><td colspan="1" rowspan="1">Zeptometrix 0810166CF</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="1">California/04/09, cell isolate i</td><td colspan="1" rowspan="1">BEI NR-13658</td><td colspan="1" rowspan="1">100xj</td></tr><tr><td colspan="1" rowspan="1">A/Christ Church/16/2010</td><td colspan="1" rowspan="1">CDC 2010839805</td><td colspan="1" rowspan="1">1000x k</td></tr><tr><td colspan="1" rowspan="1">A/Brisbane/02/2018</td><td colspan="1" rowspan="1">CDC 3000683658</td><td colspan="1" rowspan="1">100x1</td></tr><tr><td colspan="1" rowspan="2">Influenza A H3N2</td><td colspan="1" rowspan="1">A/Wisconsin/15/2009 m</td><td colspan="1" rowspan="1">BEI NR-42007</td><td colspan="1" rowspan="1">3x</td><td colspan="1" rowspan="2">Influenza A/H3Detected</td></tr><tr><td colspan="1" rowspan="1">A/Texas/50/12</td><td colspan="1" rowspan="1">Zeptometrix 0810238CF</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="1">A/Brisbane/10/2007</td><td colspan="1" rowspan="1">Zeptometrix 0810138CF</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="1">A/Port Chalmers/1/73</td><td colspan="1" rowspan="1">ATCC VR-810</td><td colspan="1" rowspan="1">50x n</td></tr><tr><td colspan="1" rowspan="1">A/Victoria/3/1975</td><td colspan="1" rowspan="1">VIRAPUR</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="1">A/Victoria/361/2011°</td><td colspan="1" rowspan="1">BEI NR-44022</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="1">A/Victoria/3/75</td><td colspan="1" rowspan="1">ATCC VR-822</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="1">A/Uruguay/716/07 p</td><td colspan="1" rowspan="1">BEI NR-42003</td><td colspan="1" rowspan="1">10x</td></tr><tr><td colspan="1" rowspan="1">A/HK/H090-756-V1(0)/2009 q</td><td colspan="1" rowspan="1">BEI NR-44344</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="1">A/Hong Kong/8/68</td><td colspan="1" rowspan="1">Zeptometrix 0810250CF</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="1">A/Switzerland/9715293/13</td><td colspan="1" rowspan="1">VIRAPUR</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="1">A/Aichi/2/68</td><td colspan="1" rowspan="1">ATCC VR-547</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="1">MRC-2</td><td colspan="1" rowspan="1">ATCC VR-777</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="1">A/Perth/16/2009</td><td colspan="1" rowspan="1">CDC 2009719818</td><td colspan="1" rowspan="1">10x</td></tr><tr><td colspan="1" rowspan="1">A/Kansas/14/2017</td><td colspan="1" rowspan="1">CDC 3026015402</td><td colspan="1" rowspan="1">2000x</td></tr></table>

a In silico analysis supports detection of Influenza A H2N3, H5N1, H5N2, H5N3, H5N8, H7N7, H7N9, H3N1, H3N2, H3N5, H3N7, H3N8 as Influenza A. However, predicted reactivities of the subtyping assays for these influenza A strains of animal origin are variable.

b Detected as Flu A (no subtype) at $3 \mathbf { x }$ LoD. Detected as dual positive $\mathrm { A } / \mathrm { H } 1$ and A/H1pdm09 at $5 0 0 \mathrm { x }$ LoD.

c Detected as Flu A (no subtype) at $1 0 0 \mathrm { x }$ LoD. In silico analysis showed several mismatches in the forward primer for the Flu A/H1 subtyping assay which may account for the observed lower sensitivity of the Flu A/H1 subtyping assay for this strain.

Virus obtained through BEI Resources, NIAID, NIH: Influenza A, A/Beijing/262/95 (H1N1), NR-12277.

e Detected as Flu A (no subtype) at 10xLoD. In silico analysis showed a G-A mismatch in the $_ { 3 } ,$ terminal position in the forward primer for the Flu A/H1 subtyping assay which may account for the observed lower sensitivity of the Flu A/H1 subtyping assay for this strain.

f Recombinant Virus obtained through BEI Resources, NIAID, NIH: Influenza A, Kilbourne F64, A/NWS/1934   
(HA) x A/Rockefeller Institute/5/1957 (NA)] (H1N2), NR-3682

g Detected as Flu A (no subtype) at 3x LoD.

h Virus obtained through BEI Resources, NIAID, NIH: Influenza A, A/Houston/3H/2009 (H1N1) pdm09, NR-20340 i Virus obtained through BEI Resources, NIAID, NIH: Influenza A, A/California/04/09, cell isolate (H1N1) pdm09, NR-13658

j In silico analysis of a partial sequence of this strain for the Flu A/H1pdm09 subtyping assay does not predict reduced analytical reactivity. Titering inconsistency from the vendor rather than reduced reactivity due to assay design is suggested.

k Virus obtained through the CDC Influenza Division. In silico analysis of partial sequences of this strain for the Flu A/H1pdm09 HA subtyping assay and the Flu A matrix gene assay does not predict reduced analytical reactivity.

Titering inconsistency from the source laboratory $\mathrm { \ E I D { : } 0 / m L }$ vs. $\mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ ) rather than reduced reactivity due to assay design is suggested.

l Virus obtained through the CDC Influenza Division. In silico analysis did not reveal any critical mismatch in the Flu A/H1pdm09 HA subtyping assay primers and probe binding regions or any mismatch in the FluA matrix gene assay primers and probe binding regions that would predict reduced analytical reactivity. Titering inconsistency from the source laboratory $\mathrm { \ E I D { \boldsymbol { \varsigma } } _ { 0 } / m L }$ vs. $\mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ ) rather than reduced reactivity due to assay design is suggested.

m Virus obtained through BEI Resources, NIAID, NIH: Influenza A, A/Wisconsin/15/2009 (H3N2), NR-42007

n In silico analysis revealed a few non-critical mismatches in the FluA/H3 subtyping HA assay probe binding region that could contribute to the observed lower reactivity. However, titering inconsistency from the vendor rather than reduced reactivity due to assay design is suspected.   
o Virus obtained through BEI Resources, NIAID, NIH: Influenza A, A/Victoria/361/2011 (H3N2), NR-44022 p Virus obtained through BEI Resources, NIAID, NIH: Influenza A, A/Uruguay/716/07 (H3N2), NR-42003 q Virus obtained through BEI Resources, NIAID, NIH: Influenza A, A/HK/H090-756-V1(0)/2009 (H3N2), NR44344   
r Virus obtained through the CDC Influenza Division. This strain was detected by BioCode RPP as Flu A (no subtype) at 50xLoD. In silico analysis did not reveal any mismatch in the Flu A matrix assay primers and probe binding regions but showed 3 mismatches in the Flu A/H3 subtyping HA assay primers binding regions that could predict reduced analytical reactivity, 1 mismatch at the $9 ^ { \mathrm { t h } }$ position from the $_ 3 \cdot$ end of the reverse primer (mismatch #1), 1 mismatch at the $1 8 ^ { \mathrm { t h } }$ position from the $_ 3 \cdot$ end of the reverse primer (mismatch #2), and 1 mismatch at the $2 0 ^ { \mathrm { t h } }$ position from the $_ { 3 } ,$ end of the reverse primer (mismatch #3). Wet testing data suggested that mismatch #1 is likely the root cause for the observed significant reduction in analytical reactivity for this strain. Based on an in silico analysis, of all the Flu A/H3 strains isolated in 2019 with published HA sequences, $7 3 . 8 \%$ of the strains harbor mismatch $\# 1$ . Therefore, for patient samples contain a Flu A/H3 strain that harbors mismatch #1 at lower concentrations, the BioCode RPP will likely report a Flu A (no subtype detected) result.

Table 20: Influenza B Isolates Tested and Detected by BioCode RPP   

<table><tr><td rowspan=1 colspan=1>Lineage</td><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>Source</td><td rowspan=1 colspan=1>xLoDDetected</td><td rowspan=1 colspan=1>Result</td></tr><tr><td rowspan=4 colspan=1>Unknown</td><td rowspan=1 colspan=1>B/Lee/1940</td><td rowspan=1 colspan=1>Zeptometrix0810257CF</td><td rowspan=1 colspan=1>3x</td><td rowspan=18 colspan=1>Influenza BDetected</td></tr><tr><td rowspan=1 colspan=1>B/Taiwan/2/1962</td><td rowspan=1 colspan=1>ATCC VR-295</td><td rowspan=1 colspan=1>500x a</td></tr><tr><td rowspan=1 colspan=1>B/Allen/45</td><td rowspan=1 colspan=1>ATCC VR-102</td><td rowspan=1 colspan=1>5x</td></tr><tr><td rowspan=1 colspan=1>B/Brigit/Russia/1969</td><td rowspan=1 colspan=1>ATCC VR-786</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=7 colspan=1>Victoria b</td><td rowspan=1 colspan=1>B/Malaysia/2506/2004c</td><td rowspan=1 colspan=1>BEI NR-9723</td><td rowspan=1 colspan=1>0.03x</td></tr><tr><td rowspan=1 colspan=1>B/Malaysia/2506/2004</td><td rowspan=1 colspan=1>Zeptometrix0810258CF</td><td rowspan=1 colspan=1>0.03x</td></tr><tr><td rowspan=1 colspan=1>B/Ohio/01/2005d</td><td rowspan=1 colspan=1>BEI NR-41801</td><td rowspan=1 colspan=1>0.3x</td></tr><tr><td rowspan=1 colspan=1>B/Brisbane/33/2008 e</td><td rowspan=1 colspan=1>BEI NR-42006</td><td rowspan=1 colspan=1>0.3x</td></tr><tr><td rowspan=1 colspan=1>B/Nevada/03/2011f</td><td rowspan=1 colspan=1>BEI NR-4023</td><td rowspan=1 colspan=1>0.03x</td></tr><tr><td rowspan=1 colspan=1>B/Michigan/09/2011</td><td rowspan=1 colspan=1>CDC 2012700901</td><td rowspan=1 colspan=1>0.3x</td></tr><tr><td rowspan=1 colspan=1>B/Colorado/06/2017</td><td rowspan=1 colspan=1>CDC 3025629447</td><td rowspan=1 colspan=1>0.3x</td></tr><tr><td rowspan=7 colspan=1>Yamagata</td><td rowspan=1 colspan=1>B/Texas/06/2011 g</td><td rowspan=1 colspan=1>BEI NR-44024</td><td rowspan=1 colspan=1>5000x h</td></tr><tr><td rowspan=1 colspan=1>B/Sydney/507/2006i</td><td rowspan=1 colspan=1>BEI NR-36526</td><td rowspan=1 colspan=1>800xj</td></tr><tr><td rowspan=1 colspan=1>B/Wisconsin/1/10</td><td rowspan=1 colspan=1>Zeptometrix0810241CF</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>B/Massachusetts/2/12</td><td rowspan=1 colspan=1>Zeptometrix0810239CF</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>B/Christchurch/33/2004 k</td><td rowspan=1 colspan=1>BEI NR-36536</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>B/New Hampshire/01/2016</td><td rowspan=1 colspan=1>CDC 3000415764</td><td rowspan=1 colspan=1>1000x1</td></tr><tr><td rowspan=1 colspan=1>B/Phuket/3073/2013m</td><td rowspan=1 colspan=1>CDC 2014768616</td><td rowspan=1 colspan=1>500x m</td></tr></table>

a In silico analysis could not be performed due to unavailability of sequence information for this strain. Titering inconsistency from the vendor rather than reduced reactivity due to assay design is suspected.

h In silico analysis does not predict reduced analytical reactivity. Titering inconsistency from the vendor rather than reduced reactivity due to assay design is suggested.

Virus obtained through BEI Resources, NIAID, NIH: Influenza B, B/Sydney/507/2006, NR-36526

In silico analysis does not predict reduced analytical reactivity. Titering inconsistency from the vendor rather than reduced reactivity due to assay design is suggested.

k Virus obtained through BEI Resources, NIAID, NIH: Influenza B, B/Christchurch/33/2004, NR-36536 l Virus obtained through the CDC Influenza Division. In silico analysis did not reveal any critical mismatch in the Flu B NS1 assay primers and probe binding regions that would predict reduced analytical reactivity. And In silico analysis did not reveal any mismatch in the Flu B matrix assay primers and probe binding regions. Titering inconsistency from the source laboratory $\mathrm { \ E I D _ { 5 0 } / m L }$ vs. $\mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ ) rather than reduced reactivity due to assay design is suggested.

m Virus obtained through the CDC Influenza Division. In silico analysis did not reveal any critical mismatch in the Flu B NS1 assay primers and probe binding regions that would predict reduced analytical reactivity. And In silico analysis did not reveal any mismatch in the Flu B matrix assay primers and probe binding regions. Titering inconsistency from the source laboratory $\mathrm { \ E I D { \boldsymbol { \varsigma } } _ { 0 } / m L }$ vs. $\mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ ) rather than reduced reactivity due to assay design is suggested.

Table 21: Parainfluenza Virus Isolates Tested and Detected by BioCode RPP   

<table><tr><td rowspan=1 colspan=1>Type</td><td rowspan=1 colspan=1>Subtype</td><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>Source</td><td rowspan=1 colspan=1>xLoDDetected</td><td rowspan=1 colspan=1>Result</td></tr><tr><td rowspan=3 colspan=2>1</td><td rowspan=1 colspan=1>FRA/27344044/2007 a</td><td rowspan=1 colspan=1>BEI NR-48681</td><td rowspan=1 colspan=1>3x</td><td rowspan=3 colspan=1>ParainfluenzaVirus 1 Detected</td></tr><tr><td rowspan=1 colspan=1>FRA/29221106/2009 b</td><td rowspan=1 colspan=1>BEI NR-48680</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>Unknown</td><td rowspan=1 colspan=1>Zeptometrix 0810014CF</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=2 colspan=2>2</td><td rowspan=1 colspan=1>Greer</td><td rowspan=1 colspan=1>BEI NR-3229</td><td rowspan=1 colspan=1>3x</td><td rowspan=2 colspan=1>ParainfluenzaVirus 2 Detected</td></tr><tr><td rowspan=1 colspan=1>Unknown</td><td rowspan=1 colspan=1>Zeptometrix 0810015CF</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=2 colspan=2>3</td><td rowspan=1 colspan=1>NIH 47885, Wash/47885/57 d</td><td rowspan=1 colspan=1>BEI NR-3233</td><td rowspan=1 colspan=1>3x</td><td rowspan=1 colspan=1>Parainfluenza</td></tr><tr><td rowspan=1 colspan=1>C-243/Washington DC/1957</td><td rowspan=1 colspan=1>ATCC VR-93</td><td rowspan=1 colspan=1>3x</td><td rowspan=1 colspan=1>Virus 3 Detected</td></tr><tr><td rowspan=3 colspan=1>4</td><td rowspan=1 colspan=1>a</td><td rowspan=1 colspan=1>M-25/1958 </td><td rowspan=1 colspan=1>BEI NR-3237</td><td rowspan=1 colspan=1>3x</td><td rowspan=2 colspan=1>ParainfluenzaVirus 4 Detected</td></tr><tr><td rowspan=2 colspan=1>b</td><td rowspan=1 colspan=1>CH-19503/WashingtonDC/1962</td><td rowspan=1 colspan=1>ATCC VR-1377</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>Unknown</td><td rowspan=1 colspan=1>Zeptometrix0810060BCF</td><td rowspan=1 colspan=1>3x</td><td rowspan=1 colspan=1></td></tr></table>

a Virus obtained through BEI Resources, NIAID, NIH: Parainfluenza Virus 1, HIPIV1/FRA/27344044/2007, NR48681 b Virus obtained through BEI Resources, NIAID, NIH: Parainfluenza Virus 1, HPIV1/FRA/29221106/2009, NR48680 c Virus obtained through BEI Resources, NIAID, NIH: Parainfluenza Virus 2, Greer, NR-3229 d Virus obtained through BEI Resources, NIAID, NIH: Parainfluenza Virus 3, NIH 47885, NR-3233 e Virus obtained through BEI Resources, NIAID, NIH: Parainfluenza Virus 4a, M-25, NR-3237

Table 22: Respiratory Syncytial Virus Isolates Tested and Detected by BioCode RPP   

<table><tr><td rowspan=1 colspan=1>Type</td><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>Source</td><td rowspan=1 colspan=1>xLoDDetected</td><td rowspan=1 colspan=1>Result</td></tr><tr><td rowspan=4 colspan=1>A</td><td rowspan=1 colspan=1>TN/1998/3-2 a</td><td rowspan=1 colspan=1>BEI NR-28529</td><td rowspan=1 colspan=1>3x</td><td rowspan=8 colspan=1>RespiratorySyncytial VirusDetected</td></tr><tr><td rowspan=1 colspan=1>TN/2000/3-4 b</td><td rowspan=1 colspan=1>BEI NR-28530</td><td rowspan=1 colspan=1>10x</td></tr><tr><td rowspan=1 colspan=1>TN/98/12-21</td><td rowspan=1 colspan=1>BEI NR-28528</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>Long/Maryland/1956</td><td rowspan=1 colspan=1>ATCC VR-26</td><td rowspan=1 colspan=1>10x</td></tr><tr><td rowspan=5 colspan=1>B</td><td rowspan=1 colspan=1>9320/Massachusetts/1977</td><td rowspan=1 colspan=1>ATCC VR-955</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>B1d</td><td rowspan=1 colspan=1>BEI NR-4052</td><td rowspan=1 colspan=1>10x</td></tr><tr><td rowspan=1 colspan=1>WV/14617/1985</td><td rowspan=1 colspan=1>ATCC VR-1400</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>18537/Washington DC/1962</td><td rowspan=1 colspan=1>ATCC VR-1580</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>CH-93 (18)-18</td><td rowspan=1 colspan=1>Zeptometrix 0810040CF</td><td rowspan=1 colspan=1>3x</td><td rowspan=1 colspan=1></td></tr></table>

a Virus obtained through BEI Resources, NIAID, NIH: RSV, TN/1998/3-2, NR-28529 b Virus obtained through BEI Resources, NIAID, NIH: RSV, TN/2000/3-4, NR-28530 c Virus obtained through BEI Resources, NIAID, NIH: RSV, TN/98/12-21, NR-28528 d Virus obtained through BEI Resources, NIAID, NIH: RSV, B1, NR-4052

Table 23: Bordetella pertussis Isolates Tested and Detected by BioCode RPP   

<table><tr><td rowspan=1 colspan=1>Isolate</td><td rowspan=1 colspan=1>Source</td><td rowspan=1 colspan=1>xLoD Detected</td><td rowspan=1 colspan=1>Result</td></tr><tr><td rowspan=1 colspan=1>F</td><td rowspan=1 colspan=1>ATCC 8467</td><td rowspan=1 colspan=1>3x</td><td rowspan=6 colspan=1>Bordetella pertussisDetected</td></tr><tr><td rowspan=1 colspan=1>5[17921]</td><td rowspan=1 colspan=1>ATCC 9340</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>10-536</td><td rowspan=1 colspan=1>ATCC 10380</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>CNCTC Hp 12/63,623</td><td rowspan=1 colspan=1>ATCC 51445</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>Tohama 1</td><td rowspan=1 colspan=1>ATCC BAA-589</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>MN2531</td><td rowspan=1 colspan=1>ATCC BAA-1335</td><td rowspan=1 colspan=1>3x</td></tr></table>

Table 24: Chlamydia pneumoniae Isolates Tested and Detected by BioCode RPP   

<table><tr><td rowspan=1 colspan=1>Isolate</td><td rowspan=1 colspan=1>Source</td><td rowspan=1 colspan=1>xLoD Detected</td><td rowspan=1 colspan=1>Result</td></tr><tr><td rowspan=1 colspan=1>CM-1/Georgia</td><td rowspan=1 colspan=1>ATCC VR-1360</td><td rowspan=1 colspan=1>3x</td><td rowspan=2 colspan=1>ChlamydiapneumoniaeDetected</td></tr><tr><td rowspan=1 colspan=1>CWL-029</td><td rowspan=1 colspan=1>ATCC VR-1310</td><td rowspan=1 colspan=1>3x</td></tr></table>

Table 25: Mycoplasma pneumoniae Isolates Tested and Detected by BioCode RPP   

<table><tr><td colspan="1" rowspan="1">Isolate</td><td colspan="1" rowspan="1">Source</td><td colspan="1" rowspan="1">xLoD Detected</td><td colspan="1" rowspan="1">Result</td></tr><tr><td colspan="1" rowspan="1">M129-B7</td><td colspan="1" rowspan="1">ATCC 29342</td><td colspan="1" rowspan="1">3x</td><td colspan="1" rowspan="2">MycoplasmapneumoniaeDetected</td></tr><tr><td colspan="1" rowspan="1">PI 1428</td><td colspan="1" rowspan="1">ATCC 29085</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="1">Mac</td><td colspan="1" rowspan="1">ATCC 15492</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="1">UTMB-10P</td><td colspan="1" rowspan="1">ATCC 49894</td><td colspan="1" rowspan="1">3x</td></tr></table>

Analytical reactivity was demonstrated with a wide variety of isolates/strains.

Due to public health concerns related to zoonotic transmission of influenza A viruses to humans (primarily swine and avian lineages), serial dilutions of the following isolates of swine and avian influenza A viruses (viral nucleic acids) were also tested assessing analytical reactivity:

• A/Japan/305/57 (H2N2)   
• A/duck/Pennsylvania/10218/1984 (H5N2)   
• A/turkey/Wisconsin/1/1966 (H9N2)   
• A/Anhui/1/2013 (H7N9)   
• A/Hubei/1/2010 (H5N1)   
• A/Minnesota/11/2010 (H3N2v)

Consistent with the in silico predictions of analytical reactivity to zoonotic influenza A viruses, each non-seasonal influenza A strain tested (H2N2, H5N2, H9N2, H7N9, and H5N1), except for the H3N2v strain, was detected as Influenza A (no subtype) at various concentrations tested (i.e., detected by the BioCode RPP Influenza A matrix assay only). For the H3N2v strain, it was detected as Influenza A/H3 at higher concentrations but was detected as Influenza A (no subtype) at the lowest concentration tested.

Analytical testing could not be performed to assess reactivity to H5N8, H1N2v and H1N1 swine viruses due to the lack of availability of such zoonotic influenza A strains for wet testing. Based on an in silico analysis, H5N8 (KP739416), H1N2v (MK239077), and H1N1 swine (KP822962.1) were predicted to be detected as Influenza A (no subtype), Influenza A/H1 (with reduced sensitivity for the A/H1 assay), and Influenza A (no subtype), respectively.

# 7. Analytical Specificity/Cross-Reactivity:

Analytical specificity (cross-reactivity) of the BioCode RPP was evaluated by challenging the system with contrived samples in simulated NPS in UTM matrix containing a high concentration of the potentially cross-reacting organism. On-panel organisms were tested to assess the potential for intra-panel cross-reactivity (e.g., does the Coronavirus OC43 assay cross-react with Coronavirus HKU1, etc.), while offpanel organisms (those not intended to be detected by the panel) were tested to assess the potential for non-specific amplification of respiratory flora or other respiratory pathogens that may be present in an NPS specimen.

Each organism was tested at $1 0 ^ { 6 } \mathrm { C F U / m L }$ for bacteria or fungi and $1 0 ^ { 5 } \mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ for viruses or higher when possible at Applied BioCode, Inc. Organism or virus stocks were spiked in simulated NPS in UTM matrix at desired concentrations at the time of extraction. Each sample was extracted in triplicate on the NucliSENS easyMAG and assayed in singlet with the BioCode RPP on the BioCode MDx-3000 system

according to the instructions for use. For each concentration tested, the number of replicates that gave valid results per the Interpretation Algorithm was determined. If any replicates were detected, testing was repeated from five additional extraction replicates. If detected after repeat with five additional replicates, serial dilutions were performed to determine the lower limit.

# On-Panel Organisms/Viruses Testing

A group of 26 on-panel analytes were tested at high concentration to assess the potential for intra-panel cross-reactivity. The organisms or viruses tested are shown in Table 26 below.

Table 26: On-panel Organisms/Viruses Tested by the BioCode RPP for Evaluation of Analytical Specificity   

<table><tr><td colspan="1" rowspan="1">Organism/Virus</td><td colspan="1" rowspan="1">Source</td><td colspan="1" rowspan="1">Titer Tested</td><td colspan="1" rowspan="1">Cross-ReactivityObserved(Y/N)</td></tr><tr><td colspan="1" rowspan="1">Influenza A H1N1/New Caledonia/20/99</td><td colspan="1" rowspan="1">Zeptometrix 0810036CF</td><td colspan="1" rowspan="1">1.15 x 105 TCID /mL50</td><td colspan="1" rowspan="1">N</td></tr><tr><td colspan="1" rowspan="1">Influenza A H1N1/NWS/33</td><td colspan="1" rowspan="1">ATCC VR-219</td><td colspan="1" rowspan="1">7.40 x 105 TCID /mL50</td><td colspan="1" rowspan="1">N</td></tr><tr><td colspan="1" rowspan="1">Influenza A H1N1 pdm09/California/07/09</td><td colspan="1" rowspan="1">Zeptometrix 0810165CF</td><td colspan="1" rowspan="1">1.31 x 105 TCID /mL50</td><td colspan="1" rowspan="1">N</td></tr><tr><td colspan="1" rowspan="1">Influenza A H3N2/Wisconsin/67/05a</td><td colspan="1" rowspan="1">Zeptometrix 0810252CF</td><td colspan="1" rowspan="1">1.08 x 105 TCID /mL50</td><td colspan="1" rowspan="1">N</td></tr><tr><td colspan="1" rowspan="1">Influenza A H3N2/Alice</td><td colspan="1" rowspan="1">ATCC VR-776</td><td colspan="1" rowspan="1">1.43 x 106 TCID /mL50</td><td colspan="1" rowspan="1">N</td></tr><tr><td colspan="1" rowspan="1">Influenza B/Florida/4/2006 (Yamagata)</td><td colspan="1" rowspan="1">Zeptometrix 0810255CF</td><td colspan="1" rowspan="1">1.08 x 105 TCID /mL50</td><td colspan="1" rowspan="1">N</td></tr><tr><td colspan="1" rowspan="1">Influenza B/Hong Kong/S/1972 (Victoria)</td><td colspan="1" rowspan="1">ATCC VR-823</td><td colspan="1" rowspan="1">8.57 x 105 TCID /mL50</td><td colspan="1" rowspan="1">N</td></tr><tr><td colspan="1" rowspan="1">Respiratory Syncytial Virus (Type A)</td><td colspan="1" rowspan="1">Zeptometrix 0810040ACF</td><td colspan="1" rowspan="1">4.57 x 105 TCID /mL50</td><td colspan="1" rowspan="1">N</td></tr><tr><td colspan="1" rowspan="1">Human Metapneumovirus 16 (Type A1)</td><td colspan="1" rowspan="1">Zeptometrix 0810161CF</td><td colspan="1" rowspan="1">8.51 x 105 TCID /mL50</td><td colspan="1" rowspan="1">N</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza Virus 1</td><td colspan="1" rowspan="1">ATCC VR-94</td><td colspan="1" rowspan="1">1.60 x 105 TCID /mL50</td><td colspan="1" rowspan="1">N</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza Virus 2</td><td colspan="1" rowspan="1">ATCC VR-92</td><td colspan="1" rowspan="1">1.35 x 105 TCID /mL50</td><td colspan="1" rowspan="1">N</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza Virus 3</td><td colspan="1" rowspan="1">Zeptometrix 0810016CF</td><td colspan="1" rowspan="1">3.39 x 105 TCID /mL50</td><td colspan="1" rowspan="1">N</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza Virus 4a</td><td colspan="1" rowspan="1">Zeptometrix 0810060CF</td><td colspan="1" rowspan="1">1.13 x 105 TCID /mL50</td><td colspan="1" rowspan="1">N</td></tr><tr><td colspan="1" rowspan="1">Adenovirus Species B Serotype 7A</td><td colspan="1" rowspan="1">Zeptometrix 0810021CF</td><td colspan="1" rowspan="1">5.83 x 105 TCID /mL50</td><td colspan="1" rowspan="1">N</td></tr><tr><td colspan="1" rowspan="1">Adenovirus Species C Serotype 2</td><td colspan="1" rowspan="1">ATCC AV-846</td><td colspan="1" rowspan="1">2.81 x 105 TCID /mL50</td><td colspan="1" rowspan="1">N</td></tr><tr><td colspan="1" rowspan="1">Adenovirus Species E Serotype 4</td><td colspan="1" rowspan="1">Zeptometrix 0810070CF</td><td colspan="1" rowspan="1">1.08 x 105 TCID /mL50</td><td colspan="1" rowspan="1">N</td></tr><tr><td colspan="1" rowspan="1">Coronavirus 229E</td><td colspan="1" rowspan="1">Zeptometrix 0810229CF</td><td colspan="1" rowspan="1">1.09 x 105 TCID /mL50</td><td colspan="1" rowspan="1">N</td></tr><tr><td colspan="1" rowspan="1">Coronavirus HKU1</td><td colspan="1" rowspan="1">Clinical Sample a</td><td colspan="1" rowspan="1">1.92 x 105 Copies/mL</td><td colspan="1" rowspan="1">N</td></tr><tr><td colspan="1" rowspan="1">Coronavirus NL63</td><td colspan="1" rowspan="1">Zeptometrix 0810228CF</td><td colspan="1" rowspan="1">1.08 x 105 TCID /mL50</td><td colspan="1" rowspan="1">N</td></tr><tr><td colspan="1" rowspan="1">Coronavirus OC43</td><td colspan="1" rowspan="1">Zeptometrix 0810024CF</td><td colspan="1" rowspan="1">1.08 x 105 TCID /mL50</td><td colspan="1" rowspan="1">N</td></tr><tr><td colspan="1" rowspan="1">Human Rhinovirus Type A1</td><td colspan="1" rowspan="1">Zeptometrix 0810012CF</td><td colspan="1" rowspan="1">1.05 x 105 TCID /mL50</td><td colspan="1" rowspan="1">N</td></tr><tr><td colspan="1" rowspan="1">Enterovirus D68</td><td colspan="1" rowspan="1">Zeptometrix 0810300CF</td><td colspan="1" rowspan="1">1.08 x 105 TCID /mL50</td><td colspan="1" rowspan="1">N</td></tr><tr><td colspan="1" rowspan="1">Bordetella pertussis</td><td colspan="1" rowspan="1">Zeptometrix 801459</td><td colspan="1" rowspan="1">3.86 x 107 CFU/mL</td><td colspan="1" rowspan="1">N</td></tr><tr><td colspan="1" rowspan="1">Mycoplasma pneumoniae</td><td colspan="1" rowspan="1">Zeptometrix 801579</td><td colspan="1" rowspan="1">1.06 x 106 CCU/mL</td><td colspan="1" rowspan="1">N</td></tr><tr><td colspan="1" rowspan="1">Chlamydia pneumoniae (AR-39)</td><td colspan="1" rowspan="1">ATCC 53592</td><td colspan="1" rowspan="1">1.24 x 106 CFU/mL</td><td colspan="1" rowspan="1">N</td></tr><tr><td colspan="1" rowspan="1">Chlamydia pneumoniae (CWL-029)</td><td colspan="1" rowspan="1">ATCC VR-1310</td><td colspan="1" rowspan="1">1.00 x 106 CFU/mL</td><td colspan="1" rowspan="1">N</td></tr></table>

a Coronavirus HKU1 clinical sample quantified with Applied BioCode validated SYBR assay using an IVT RNA standard.

# Off-Panel Organisms/Viruses Testing

In silico analysis of assay specificity was supplemented with wet testing of 52 offpanel bacteria, viruses, and fungi at high concentrations (typically $1 0 ^ { 6 } \mathrm { C F U / m L }$ for bacteria or fungi and $1 0 ^ { 5 } \mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ for viruses or higher when possible). The organisms and viruses tested are shown in Table 27 below.

Table 27: Off-panel Organisms and Viruses Tested by the BioCode RPP for Evaluation of Analytical Specificity   

<table><tr><td colspan="1" rowspan="1">Organism/Virus</td><td colspan="1" rowspan="1">Source</td><td colspan="1" rowspan="1">Titer Tested</td><td colspan="1" rowspan="1">Cross-ReactivityObserved(Y/N)</td></tr><tr><td colspan="1" rowspan="1">Acinetobacter baumannii</td><td colspan="1" rowspan="1">Zeptometrix 801597</td><td colspan="1" rowspan="1">9.67 x 106 CFU/mL</td><td colspan="1" rowspan="1">N</td></tr><tr><td colspan="1" rowspan="1">Aspergillus flavus</td><td colspan="1" rowspan="1">Zeptometrix 801598</td><td colspan="1" rowspan="1">1.72 x 106 CFU/mL</td><td colspan="1" rowspan="1">N</td></tr><tr><td colspan="1" rowspan="1">Bordetella bronchiseptica</td><td colspan="1" rowspan="1">Zeptometrix 801649</td><td colspan="1" rowspan="1">6.68 x 107 CFU/mL</td><td colspan="1" rowspan="1">N</td></tr><tr><td colspan="1" rowspan="1">Bordetella holmesii</td><td colspan="1" rowspan="1">Zeptometrix 801464</td><td colspan="1" rowspan="1">3.83 x 10° CFU/mL</td><td colspan="1" rowspan="1">$Y}$</td></tr><tr><td colspan="1" rowspan="1">Bordetella parapertussis</td><td colspan="1" rowspan="1">Zeptometrix 8011461</td><td colspan="1" rowspan="1">1.00 x 106 CFU/mL</td><td colspan="1" rowspan="1">N</td></tr><tr><td colspan="1" rowspan="1">Burkholderia cepacia</td><td colspan="1" rowspan="1">Zeptometrix 801584</td><td colspan="1" rowspan="1">4.13 x 107 CFU/mL</td><td colspan="1" rowspan="1">N</td></tr><tr><td colspan="1" rowspan="1">Candida albicans</td><td colspan="1" rowspan="1">Zeptometrix 801504</td><td colspan="1" rowspan="1">1.96 x 106 CFU/mL</td><td colspan="1" rowspan="1">N</td></tr><tr><td colspan="1" rowspan="1">Candida glabrata</td><td colspan="1" rowspan="1">Zeptometrix 801535</td><td colspan="1" rowspan="1">1.73 x 107 CFU/mL</td><td colspan="1" rowspan="1">N</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium diphtheriae</td><td colspan="1" rowspan="1">Zeptometrix 801882</td><td colspan="1" rowspan="1">4.57 x 106 CFU/mL</td><td colspan="1" rowspan="1">N</td></tr><tr><td colspan="1" rowspan="1">Haemophilus influenzae</td><td colspan="1" rowspan="1">Zeptometrix 801679</td><td colspan="1" rowspan="1">2.40 x 106 CFU/mL</td><td colspan="1" rowspan="1">N</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">Zeptometrix 801506</td><td colspan="1" rowspan="1">5.10 x 107 CFU/mL</td><td colspan="1" rowspan="1">N</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus plantarum</td><td colspan="1" rowspan="1">Zeptometrix 801507</td><td colspan="1" rowspan="1">1.80 x 107 CFU/mL</td><td colspan="1" rowspan="1">N</td></tr><tr><td colspan="1" rowspan="1">Legionella longbeachae</td><td colspan="1" rowspan="1">Zeptometrix 8101577</td><td colspan="1" rowspan="1">1.93 x 107 CFU/mL</td><td colspan="1" rowspan="1">N</td></tr><tr><td colspan="1" rowspan="1">Legionella micdadei</td><td colspan="1" rowspan="1">Zeptometrix 801576</td><td colspan="1" rowspan="1">2.70 x 107 CFU/mL</td><td colspan="1" rowspan="1">N</td></tr><tr><td colspan="1" rowspan="1">Legionella pneumophila</td><td colspan="1" rowspan="1">Zeptometrix 801645</td><td colspan="1" rowspan="1">3.17 x 107 CFU/mL</td><td colspan="1" rowspan="1">N</td></tr><tr><td colspan="1" rowspan="1">Moraxella catarrhalis</td><td colspan="1" rowspan="1">Zeptometrix 801509</td><td colspan="1" rowspan="1">1.13 x 106 CFU/mL</td><td colspan="1" rowspan="1">N</td></tr><tr><td colspan="1" rowspan="1">Mycobacterium tuberculosis</td><td colspan="1" rowspan="1">Zeptometrix 801660</td><td colspan="1" rowspan="1">7.23 x 106 CFU/mL</td><td colspan="1" rowspan="1">N</td></tr><tr><td colspan="1" rowspan="1">Mycoplasma genitalium</td><td colspan="1" rowspan="1">ATCC 33530</td><td colspan="1" rowspan="1">1.00 x 103 CFU/mL</td><td colspan="1" rowspan="1">N</td></tr><tr><td colspan="1" rowspan="1">Mycoplasma hominis</td><td colspan="1" rowspan="1">ATCC 23114</td><td colspan="1" rowspan="1">2.7 x 104 CFU/mL</td><td colspan="1" rowspan="1">N</td></tr><tr><td colspan="1" rowspan="1">Neisseria elongata</td><td colspan="1" rowspan="1">Zeptometrix 801510</td><td colspan="1" rowspan="1">1.74 x 107 CFU/mL</td><td colspan="1" rowspan="1">N</td></tr><tr><td colspan="1" rowspan="1">Neisseria gonorrhoeae</td><td colspan="1" rowspan="1">Zeptometrix 801482</td><td colspan="1" rowspan="1">1.26 x 107 CFU/mL</td><td colspan="1" rowspan="1">N</td></tr><tr><td colspan="1" rowspan="1">Neisseria meningitidis</td><td colspan="1" rowspan="1">Zeptometrix 801511</td><td colspan="1" rowspan="1">2.55 x 106 CFU/mL</td><td colspan="1" rowspan="1">N</td></tr><tr><td colspan="1" rowspan="1">Neisseria sicca</td><td colspan="1" rowspan="1">Zeptometrix 801754</td><td colspan="1" rowspan="1">1.02 x 106 CFU/mL</td><td colspan="1" rowspan="1">N</td></tr><tr><td colspan="1" rowspan="1">Proteus vulgaris</td><td colspan="1" rowspan="1">Zeptometrix 801898</td><td colspan="1" rowspan="1">4.13 x 107 CFU/mL</td><td colspan="1" rowspan="1">N</td></tr><tr><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">ATCC 39324</td><td colspan="1" rowspan="1">2.11 x 106 CFU/mL</td><td colspan="1" rowspan="1">N</td></tr><tr><td colspan="1" rowspan="1">Serratia marcescens</td><td colspan="1" rowspan="1">Zeptometrix 801723</td><td colspan="1" rowspan="1">2.06 x 107 CFU/mL</td><td colspan="1" rowspan="1">N</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus haemolyticus</td><td colspan="1" rowspan="1">Zeptometrix 801591</td><td colspan="1" rowspan="1">8.50 x 106 CFU/mL</td><td colspan="1" rowspan="1">N</td></tr><tr><td colspan="1" rowspan="1">Streptococcus agalactiae</td><td colspan="1" rowspan="1">Zeptometrix 801545</td><td colspan="1" rowspan="1">3.73 x 106 CFU/mL</td><td colspan="1" rowspan="1">N</td></tr><tr><td colspan="1" rowspan="1">Streptococcus dysgalactiae</td><td colspan="1" rowspan="1">Zeptometrix 801516</td><td colspan="1" rowspan="1">1.23 x 106 CFU/mL</td><td colspan="1" rowspan="1">N</td></tr><tr><td colspan="1" rowspan="1">Streptococcus intermedius</td><td colspan="1" rowspan="1">Zeptometrix 801895</td><td colspan="1" rowspan="1">5.07 x 106 CFU/mL</td><td colspan="1" rowspan="1">N</td></tr><tr><td colspan="1" rowspan="1">Streptococcus mitis</td><td colspan="1" rowspan="1">Zeptometrix 801695</td><td colspan="1" rowspan="1">5.73 x 106 CFU/mL</td><td colspan="1" rowspan="1">N</td></tr><tr><td colspan="1" rowspan="1">Streptococcus pneumoniae</td><td colspan="1" rowspan="1">Zeptometrix 801439</td><td colspan="1" rowspan="1">4.17 x 106 CFU/mL</td><td colspan="1" rowspan="1">N</td></tr><tr><td colspan="1" rowspan="1">Ureaplasma urealyticum</td><td colspan="1" rowspan="1">ATCC 27618</td><td colspan="1" rowspan="1">1.00 x 107 CFU/mL</td><td colspan="1" rowspan="1">N</td></tr><tr><td colspan="1" rowspan="1">Coxsackievirus A10</td><td colspan="1" rowspan="1">Zeptometrix 0810106CF</td><td colspan="1" rowspan="1">1.05 x 103 TCID50/mL</td><td colspan="1" rowspan="1">$Ybb}$</td></tr><tr><td colspan="1" rowspan="1">Coxsackievirus A21</td><td colspan="1" rowspan="1">Zeptometrix 0810235CF</td><td colspan="1" rowspan="1">≤ 1.03 x 102 TCID50/mL</td><td colspan="1" rowspan="1">$Y}$</td></tr><tr><td colspan="1" rowspan="1">Coxsackievirus A24</td><td colspan="1" rowspan="1">ATCC VR-583</td><td colspan="1" rowspan="1">1.14 x 101 TCID50/mL</td><td colspan="1" rowspan="1">Yb}$</td></tr><tr><td colspan="1" rowspan="1">Coxsackievirus B2</td><td colspan="1" rowspan="1">ATCC VR-29</td><td colspan="1" rowspan="1">5.62 x 103 TCID50/mL</td><td colspan="1" rowspan="1">$Y bb}$</td></tr><tr><td colspan="1" rowspan="1">Coxsackievirus B3</td><td colspan="1" rowspan="1">Zeptometrix 0810074CF</td><td colspan="1" rowspan="1">1.76 x 103 TCID50/mL</td><td colspan="1" rowspan="1">$Y}$</td></tr><tr><td colspan="1" rowspan="1">Coxsackievirus B4</td><td colspan="1" rowspan="1">Zeptometrix 0810075CF</td><td colspan="1" rowspan="1">1.36 x 104 TCID50/mL</td><td colspan="1" rowspan="1">$Y}$</td></tr><tr><td colspan="1" rowspan="1">Coxsackievirus B5</td><td colspan="1" rowspan="1">Zeptometrix 0810019CF</td><td colspan="1" rowspan="1">≤ 5.89 x 102 TCID50/mL</td><td colspan="1" rowspan="1">$Yrb}$</td></tr><tr><td colspan="1" rowspan="1">Coxsackievirus A9</td><td colspan="1" rowspan="1">Zeptometrix 0810017CF</td><td colspan="1" rowspan="1">1.38 x 103 TCID50/mL</td><td colspan="1" rowspan="1">$Y}$</td></tr><tr><td colspan="1" rowspan="1">Cytomegalovirus</td><td colspan="1" rowspan="1">Zeptometrix 0810003CF</td><td colspan="1" rowspan="1">4.17 x 104 TCID50/mL</td><td colspan="1" rowspan="1">N</td></tr><tr><td colspan="1" rowspan="1">Echovirus 11</td><td colspan="1" rowspan="1">Zeptometrix 0810023CF</td><td colspan="1" rowspan="1">1.68 x 103 TCID50/mL</td><td colspan="1" rowspan="1">$Y$</td></tr><tr><td colspan="1" rowspan="1">Echovirus 30</td><td colspan="1" rowspan="1">Zeptometrix 0810078CF</td><td colspan="1" rowspan="1">≤ 1.95 x 102 TCID50/mL</td><td colspan="1" rowspan="1">Y$</td></tr><tr><td colspan="1" rowspan="1">Echovirus 6</td><td colspan="1" rowspan="1">Zeptometrix 0810076CF</td><td colspan="1" rowspan="1">1.09 x 104 TCID50/mL</td><td colspan="1" rowspan="1">Y</td></tr><tr><td colspan="1" rowspan="1">Echovirus 9</td><td colspan="1" rowspan="1">Zeptometrix 0810077CF</td><td colspan="1" rowspan="1">1.07 x 101 TCID50/mL</td><td colspan="1" rowspan="1">$Y$</td></tr><tr><td colspan="1" rowspan="1">Epstein-Barr Virus</td><td colspan="1" rowspan="1">Zeptometrix 0810008CF</td><td colspan="1" rowspan="1">3.43 x 106 TCID50/mL</td><td colspan="1" rowspan="1">N</td></tr><tr><td colspan="1" rowspan="1">Herpes Simplex Virus Type 1</td><td colspan="1" rowspan="1">Zeptometrix 0810187CF</td><td colspan="1" rowspan="1">9.12 x 106 TCID50/mL</td><td colspan="1" rowspan="1">N</td></tr><tr><td colspan="1" rowspan="1">Measles Virus</td><td colspan="1" rowspan="1">Zeptometrix 0810025CF</td><td colspan="1" rowspan="1">1.31 x 105 TCID50/mL</td><td colspan="1" rowspan="1">N</td></tr><tr><td colspan="1" rowspan="1">Mumps Virus</td><td colspan="1" rowspan="1">Zeptometrix 0810176CF</td><td colspan="1" rowspan="1">1.89 x 105 TCID50/mL</td><td colspan="1" rowspan="1">N</td></tr><tr><td colspan="1" rowspan="1">SARS-CoV (formaldehyde and UV-inactivated, vaccine 1%)</td><td colspan="1" rowspan="1">BEI NR-3883</td><td colspan="1" rowspan="1">1:100 Dilution</td><td colspan="1" rowspan="1">N</td></tr><tr><td colspan="1" rowspan="1">MERS-CoV (EMC/2012, heat-inactivated)</td><td colspan="1" rowspan="1">BEI NR-50171</td><td colspan="1" rowspan="1">2.00 x 105 TCID50/mL</td><td colspan="1" rowspan="1">N</td></tr><tr><td colspan="1" rowspan="1">MERS-CoV gRNA</td><td colspan="1" rowspan="1">BEI NR-45843</td><td colspan="1" rowspan="1">1.01 x 107 Copies/mL</td><td colspan="1" rowspan="1">N</td></tr></table>

a Bordetella holmesii was detected by the Bordetella pertussis (BP) assay in 2 of 3 replicates down to $3 . 8 3 \mathrm { ~ x ~ } 1 0 ^ { 0 } \mathrm { C F U / m L }$ b The Coxsackieviruses assayed were detected by the Human Rhinovirus/Enterovirus (HRV) assay in at least 1 of the 3 replicates down to the concentrations indicated. c The Echoviruses assayed here were detected by the Human Rhinovirus/Enterovirus (HRV) assay in at least 1 of the 3 replicates down to the concentrations indicated.

# Analytical Specificity Evaluation Conclusion

Analytical specificity testing has demonstrated that the majority of the BioCode RPP assays are highly specific for the organisms/viruses they are designed to detect. Cross-reactivity that was observed during the testing was almost exclusively associated with near-neighbor species that carry the same genes or highly similar sequences as the targeted pathogen. Overall, the likelihood, risk, and impact of the BioCode RPP non-specific interactions are predicted to be minor.

The BioCode RPP includes assays to distinguish classical human Influenza A H1 and the A H1 2009 pandemic variant derived from swine. However, due to sequence similarity, some reactivity of the A/H1pdm09 assay may be observed with historical and/or novel A/H1N1 strains of swine origin. This was demonstrated by testing a swine origin influenza A A/New Jersey/8/1976 in the analytical reactivity study which was detected and reported as dual positive for Influenza $\mathrm { A } / \mathrm { H } 1$ and Influenza A/H1pdm09 when tested at a concentration of $7 . 5 \mathrm { ~ x ~ } 1 0 ^ { 3 } \mathrm { C E I D } 5 0 / \mathrm { m L }$ (approximately 500xLoD).

Coxsackievirus and Echovirus may be detected as Human Rhinovirus/Enterovirus even at low concentrations. Due to the genetic similarity between Human Rhinovirus and Coxsackievirus and Echovirus, this expected.

Other non-pertussis Bordetella species that harbor the repetitive element IS481 sequence or other related sequence variants, Bordetella holmesii and Bordetella bronchiseptica in particular, may be detected as Bordetella pertussis by the BioCode RPP.

All identified BioCode RPP cross-reactivity cases are indicated in the product Instructions for Use, as a precaution to minimize misinterpretation of results. A summary is provided in Table 28 below.

Table 28: Predicted and Observed Cross-Reactivity of the BioCode RPP   

<table><tr><td rowspan=1 colspan=1>Cross-reactive Organism</td><td rowspan=1 colspan=1>BioCode RPP Result</td><td rowspan=1 colspan=1>Description</td></tr><tr><td rowspan=1 colspan=1>Non-pertussis Bordetellaspecies that harborIS481 sequence, Bordetellaholmesii and Bordetellabronchiseptica in particular</td><td rowspan=1 colspan=1>Bordetella pertussis</td><td rowspan=1 colspan=1>The Bordetella pertussis assay may amplify IS481sequences in non-pertussis Bordetella species, suchas B. bronchiseptica and B. holmesii.</td></tr><tr><td rowspan=1 colspan=1>Influenza A H1N1(swine origin)</td><td rowspan=1 colspan=1>Influenza A/H1pdm09</td><td rowspan=1 colspan=1>The Influenza A/H1pdm09 assay may react withH1 hemagglutinin gene sequences from viruses ofswine origin. BioCode RPP results may be eitherInfluenza A/H1 or Influenza A/H1 pdm09,depending on the strain and concentration in thesample.</td></tr></table>

# 8. Assay Cut-Off:

Preliminary MFI cutoff values were determined by analyzing the results of 500 archived frozen NPS samples during verification based on two main criteria, signal to noise ratio and the relative levels of sensitivity and specificity based on the Receiver Operating Characteristic (ROC) analysis. The cutoff MFIs selected should produce a signal to noise ratio of ${ \ge } 5$ for $9 5 \%$ of test wells and a Sensitivity $2 9 5 \%$ and Specificity $2 9 5 \%$ based on the ROC analysis with an invalid rate (RNA IC) of $\leq 2 \%$ .

After completion of the analytical validation and clinical sample testing, the data was reviewed to determine if any of the cutoff values established initially could be improved. This review prompted the change of the ADV2 assay cutoff from 500 to 10000 MFI.

Table 29 below summarizes the MFI cutoff values for the BioCode RPP.

Table 29: Summary of Final MFI Cutoff Values for Each BMB-Probe Assay   

<table><tr><td rowspan=1 colspan=1>BMB-Probe Assay</td><td rowspan=1 colspan=1>Cutoff (≥)</td><td rowspan=1 colspan=1>BMB-Probe Assay</td><td rowspan=1 colspan=1>Cutoff (≥)</td><td rowspan=1 colspan=1>BMB-Probe Assay</td><td rowspan=1 colspan=1>Cutoff (≥)</td></tr><tr><td rowspan=1 colspan=1>Blank</td><td rowspan=1 colspan=1>&lt;3000</td><td rowspan=1 colspan=1>hMPV1</td><td rowspan=1 colspan=1>500</td><td rowspan=1 colspan=1>CoV-229E</td><td rowspan=1 colspan=1>300</td></tr><tr><td rowspan=1 colspan=1>Flu A</td><td rowspan=1 colspan=1>500</td><td rowspan=1 colspan=1>hMPV2</td><td rowspan=1 colspan=1>500</td><td rowspan=1 colspan=1>CoV-HKU1</td><td rowspan=1 colspan=1>400</td></tr><tr><td rowspan=1 colspan=1>Flu A/H1</td><td rowspan=1 colspan=1>1000</td><td rowspan=1 colspan=1>PIV1</td><td rowspan=1 colspan=1>500</td><td rowspan=1 colspan=1>CoV-NL63</td><td rowspan=1 colspan=1>500</td></tr><tr><td rowspan=1 colspan=1>Flu A/H1pdm09</td><td rowspan=1 colspan=1>500</td><td rowspan=1 colspan=1>PIV2</td><td rowspan=1 colspan=1>400</td><td rowspan=1 colspan=1>CoV-OC43</td><td rowspan=1 colspan=1>500</td></tr><tr><td rowspan=1 colspan=1>Flu A/H3</td><td rowspan=1 colspan=1>1000</td><td rowspan=1 colspan=1>PIV3</td><td rowspan=1 colspan=1>500</td><td rowspan=1 colspan=1>HRV</td><td rowspan=1 colspan=1>300</td></tr><tr><td rowspan=1 colspan=1>Flu B1</td><td rowspan=1 colspan=1>500</td><td rowspan=1 colspan=1>PIV4</td><td rowspan=1 colspan=1>500</td><td rowspan=1 colspan=1>BP</td><td rowspan=1 colspan=1>500</td></tr><tr><td rowspan=1 colspan=1>Flu B2</td><td rowspan=1 colspan=1>500</td><td rowspan=1 colspan=1>ADV1</td><td rowspan=1 colspan=1>500</td><td rowspan=1 colspan=1>MPN</td><td rowspan=1 colspan=1>500</td></tr><tr><td rowspan=1 colspan=1>RSV</td><td rowspan=1 colspan=1>500</td><td rowspan=1 colspan=1>ADV2</td><td rowspan=1 colspan=1>10000</td><td rowspan=1 colspan=1>CPN</td><td rowspan=1 colspan=1>1000</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>RNA IC</td><td rowspan=1 colspan=1>8000</td></tr></table>

9. Interfering Substances:

An analytical study was performed to assess potential inhibitory effects of substances and microorganisms that may be commonly found in nasopharyngeal specimens. Each member of the interfering substance and microorganism panel was added to simulated NPS (sNPS) matrix spiked with representative members of the BioCode RPP (see Table 30 below) at 3x LoD and a negative matrix comprised of only sNPS. Each sample was tested with and without potentially interfering substances or microbes. Each sample was prepared and extracted in triplicate on both NucliSENS easyMAG and MagNA Pure 96 extraction system and tested with the BioCode RPP on the BioCode MDx-3000 system. Substances that produce interference at the original test concentration were further tested at lower concentrations.

Table 30: Contrived Samples (3x LoD in sNPS)   

<table><tr><td rowspan=1 colspan=1>Sample Name</td><td rowspan=1 colspan=1>Organism/Virus</td><td rowspan=1 colspan=1>Source</td></tr><tr><td rowspan=3 colspan=1>RPP A</td><td rowspan=1 colspan=1>Adenovirus B Serotype 7A</td><td rowspan=1 colspan=1>Zeptometrix 0810060CF</td></tr><tr><td rowspan=1 colspan=1>Mycoplasma pneumoniae</td><td rowspan=1 colspan=1>Zeptometrix 801579</td></tr><tr><td rowspan=1 colspan=1>Influenza A H3N2 A/Wisconsin/67/2005</td><td rowspan=1 colspan=1>Zeptometrix 0810252CF</td></tr><tr><td rowspan=3 colspan=1>RPP B</td><td rowspan=1 colspan=1>Respiratory Syncytial Virus (Type A)</td><td rowspan=1 colspan=1>Zeptometrix 0810040ACF</td></tr><tr><td rowspan=1 colspan=1>Influenza A H1N1/California/07/09</td><td rowspan=1 colspan=1>Zeptometrix 0810165CF</td></tr><tr><td rowspan=1 colspan=1>Human Metapneumovirus (16; type A1)</td><td rowspan=1 colspan=1>Zeptometrix 0810161CF</td></tr><tr><td rowspan=3 colspan=1>RPP C</td><td rowspan=1 colspan=1>Parainfluenza Virus 3</td><td rowspan=1 colspan=1>Zeptometrix 0810016CF</td></tr><tr><td rowspan=1 colspan=1>Coronavirus NL63</td><td rowspan=1 colspan=1>Zeptometrix 0810228CF</td></tr><tr><td rowspan=1 colspan=1>Influenza B/Florida/4/2006</td><td rowspan=1 colspan=1>Zeptometrix 0810255CF</td></tr><tr><td rowspan=1 colspan=1>HRV</td><td rowspan=1 colspan=1>Human Rhinovirus</td><td rowspan=1 colspan=1>Zeptometrix 0810012CF</td></tr></table>

In total, 30 different endogenous and exogenous substances, potentially competing microorganisms, specimen collection materials (swabs and media), and disinfecting agents were evaluated in this study (see Table 31 below).

Table 31: Summary Results of the Interference Study   

<table><tr><td rowspan=1 colspan=1>Substance Interferent</td><td rowspan=1 colspan=1>Brand/Source</td><td rowspan=1 colspan=1>Concentration Tested</td><td rowspan=1 colspan=1>InterferenceYes (Y) or No (N)</td></tr><tr><td rowspan=1 colspan=1>Genomic DNA</td><td rowspan=1 colspan=1>Promega</td><td rowspan=1 colspan=1>10 ng/μl</td><td rowspan=1 colspan=1>N</td></tr><tr><td rowspan=1 colspan=1>Mucin (MagNA Pure 96)</td><td rowspan=1 colspan=1>Sigma</td><td rowspan=1 colspan=1>0.6% W/V</td><td rowspan=1 colspan=1>N</td></tr><tr><td rowspan=1 colspan=1>Mucin (easyMAG) a</td><td rowspan=1 colspan=1>Sigma</td><td rowspan=1 colspan=1>0.5% W/V</td><td rowspan=1 colspan=1>N</td></tr><tr><td rowspan=1 colspan=1>Human Blood</td><td rowspan=1 colspan=1>Poplar Health</td><td rowspan=1 colspan=1>1% V/V</td><td rowspan=1 colspan=1>N</td></tr><tr><td rowspan=1 colspan=1>Zanamivir</td><td rowspan=1 colspan=1>APExBIO</td><td rowspan=1 colspan=1>550 ng/mL</td><td rowspan=1 colspan=1>N</td></tr><tr><td rowspan=1 colspan=1>Oseltamivir</td><td rowspan=1 colspan=1>APExBIO</td><td rowspan=1 colspan=1>142 ng/mL</td><td rowspan=1 colspan=1>N</td></tr><tr><td rowspan=1 colspan=1>Nasal spray</td><td rowspan=1 colspan=1>Equate</td><td rowspan=1 colspan=1>1% V/V</td><td rowspan=1 colspan=1>N</td></tr><tr><td rowspan=1 colspan=1>Nasal decongestant spray</td><td rowspan=1 colspan=1>Bayer</td><td rowspan=1 colspan=1>1% V/V</td><td rowspan=1 colspan=1>N</td></tr><tr><td rowspan=1 colspan=1>Nasal Allergy spray (Fluticasone)</td><td rowspan=1 colspan=1>Equate</td><td rowspan=1 colspan=1>1.5% V/V</td><td rowspan=1 colspan=1>N</td></tr><tr><td rowspan=1 colspan=1>Petroleum Jelly</td><td rowspan=1 colspan=1>Equate</td><td rowspan=1 colspan=1>1% W/V</td><td rowspan=1 colspan=1>N</td></tr><tr><td rowspan=1 colspan=1>Analgesic Ointment</td><td rowspan=1 colspan=1>Vicks</td><td rowspan=1 colspan=1>1% W/V</td><td rowspan=1 colspan=1>N</td></tr><tr><td rowspan=1 colspan=1>Mupirocin</td><td rowspan=1 colspan=1>Alfa Aesar</td><td rowspan=1 colspan=1>2% W/V</td><td rowspan=1 colspan=1>N</td></tr><tr><td rowspan=1 colspan=1>Tobramycin</td><td rowspan=1 colspan=1>MP Biomedicals,LLC</td><td rowspan=1 colspan=1>0.6 mg/mL</td><td rowspan=1 colspan=1>N</td></tr><tr><td rowspan=1 colspan=1>Bleach (10%)</td><td rowspan=1 colspan=1>VWR</td><td rowspan=1 colspan=1>5% V/V</td><td rowspan=1 colspan=1>N</td></tr><tr><td rowspan=1 colspan=1>Disinfecting wipes</td><td rowspan=1 colspan=1>Clorox</td><td rowspan=1 colspan=1>50% V/V</td><td rowspan=1 colspan=1>N</td></tr><tr><td rowspan=1 colspan=1>Ethanol (70%)</td><td rowspan=1 colspan=1>LabChem</td><td rowspan=1 colspan=1>7% V/V</td><td rowspan=1 colspan=1>N</td></tr><tr><td rowspan=1 colspan=1>Remel M4 Media</td><td rowspan=1 colspan=1>Remel</td><td rowspan=1 colspan=1>90% V/V</td><td rowspan=1 colspan=1>N</td></tr><tr><td rowspan=1 colspan=1>Remel M4-RT Media</td><td rowspan=1 colspan=1>Remel</td><td rowspan=1 colspan=1>90% V/V</td><td rowspan=1 colspan=1>N</td></tr><tr><td rowspan=1 colspan=1>Remel M5 Media</td><td rowspan=1 colspan=1>Remel</td><td rowspan=1 colspan=1>90% V/V</td><td rowspan=1 colspan=1>N</td></tr><tr><td rowspan=1 colspan=1>Remel M6 Media</td><td rowspan=1 colspan=1>Remel</td><td rowspan=1 colspan=1>90% V/V</td><td rowspan=1 colspan=1>N</td></tr><tr><td rowspan=1 colspan=1>Copan FloQ (Flocked nylon/plasticshaft)</td><td rowspan=1 colspan=1>Copan</td><td rowspan=1 colspan=1>1 swab</td><td rowspan=1 colspan=1>N</td></tr><tr><td rowspan=1 colspan=1>Copan 168C (rayon/twisted aluminumshaft)</td><td rowspan=1 colspan=1>Copan</td><td rowspan=1 colspan=1>1 swab</td><td rowspan=1 colspan=1>N</td></tr><tr><td rowspan=1 colspan=1>Polyester / Aluminum shaft swab</td><td rowspan=1 colspan=1>Puritan/Copan</td><td rowspan=1 colspan=1>1 swab</td><td rowspan=1 colspan=1>N</td></tr><tr><td rowspan=1 colspan=1>DNAzap</td><td rowspan=1 colspan=1>Invitrogen</td><td rowspan=1 colspan=1>1% V/V</td><td rowspan=1 colspan=1>N</td></tr><tr><td rowspan=1 colspan=1>RNaseOut</td><td rowspan=1 colspan=1>Invitrogen</td><td rowspan=1 colspan=1>1% V/V</td><td rowspan=1 colspan=1>N</td></tr><tr><td rowspan=1 colspan=1>Microbial Interferent</td><td rowspan=1 colspan=1>Source</td><td rowspan=1 colspan=1>Concentration Tested</td><td rowspan=1 colspan=1>InterferenceYes (Y) or No (N)</td></tr><tr><td rowspan=1 colspan=1>Streptococcus pneumoniae</td><td rowspan=1 colspan=1>Zeptometrix</td><td rowspan=1 colspan=1>1 X 106 CFU/mL</td><td rowspan=1 colspan=1>N</td></tr><tr><td rowspan=1 colspan=1>Haemophilus influenzae</td><td rowspan=1 colspan=1>Zeptometrix</td><td rowspan=1 colspan=1>1 X 106 CFU/mL</td><td rowspan=1 colspan=1>N</td></tr><tr><td rowspan=1 colspan=1>Neisseria meningitidis</td><td rowspan=1 colspan=1>Zeptometrix</td><td rowspan=1 colspan=1>1 X 106 CFU/mL</td><td rowspan=1 colspan=1>N</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus aureus</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>1 X 106 CFU/mL</td><td rowspan=1 colspan=1>N</td></tr><tr><td rowspan=1 colspan=1>Cytomegalovirus</td><td rowspan=1 colspan=1>Zeptometrix</td><td rowspan=1 colspan=1>1 X 105 TCID50 /mL</td><td rowspan=1 colspan=1>N</td></tr></table>

a It was observed that mucin at higher concentration $( 0 . 6 \% )$ led to loss of signal for some targets (loss of analyte detection) when extracted with NucliSENS easyMAG.

In addition, nasal influenza vaccine Flu Mist was also evaluated in this study. It was tested without extraction in triplicates. Testing results are presented in Table 32 below.

Table 32: Nasal Influenza Vaccine (FluMist) Tested with the BioCode RPP   

<table><tr><td rowspan=2 colspan=1>FluMist 2010-2011 (V/V%)</td><td rowspan=1 colspan=3>Influenza A</td><td rowspan=2 colspan=1>Influenza B</td></tr><tr><td rowspan=1 colspan=1>A/H1</td><td rowspan=1 colspan=1>A/H1pdm09</td><td rowspan=1 colspan=1>A/H3</td></tr><tr><td rowspan=1 colspan=1>10%</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td></tr><tr><td rowspan=1 colspan=1>1%</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td></tr><tr><td rowspan=1 colspan=1>0.1%</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td></tr><tr><td rowspan=1 colspan=1>0.01%</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td></tr><tr><td rowspan=1 colspan=1>0.001%</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td></tr><tr><td rowspan=1 colspan=1>0.0001%</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+ a</td><td rowspan=1 colspan=1>+</td></tr><tr><td rowspan=1 colspan=1>0.00001%</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>+ a</td></tr><tr><td rowspan=1 colspan=1>0.000001%</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr></table>

a 2/3 replicates detected.

This study showed that none of the substances interfered with BioCode RPP at the concentrations tested. All substances and microorganisms tested appeared to be compatible with BioCode RPP assay at the concentrations tested. However, it was observed that mucin at higher concentration $( 0 . 6 \% )$ could lead to loss of signal for some targets (loss of analyte detection) when extracted with the NucliSENS easyMAG system. The effect of mucin was dependent on the concentration in the sample tested. Flu Mist was evaluated to be reactive as predicted with the BioCode RPP assay. Therefore, recent administration or contamination of specimens by flu vaccine prior to NPS collection could lead to false detection by BioCode RPP.

# 10. Carry-Over Contamination:

A formal carry-over study in support of this regulatory submission for the BioCode RPP was not performed, since carry-over studies with high positive samples followed by negative samples have been performed for the FDA-cleared BioCode GPP using both the NucliSENS easyMAG (K180041) and the MagNA Pure 96 systems (K190585). No carry-over has been observed.

# 11. Competitive Inhibition:

An analytical study was performed to evaluate the potential for inhibition in samples with mixed infections. Simulated NPS (sNPS) in UTM matrix was spiked with one target at high concentration $\mathrm { ( \ge 1 0 ^ { 6 } C F U / m L }$ or $\mathrm { C C U / m L }$ for bacteria and ${ \geq } 1 0 ^ { 5 }$ T $\mathrm { \Delta C I D { \scriptstyle 5 0 } / m L }$ for viruses) and two targets at low concentrations $( \leq 3 \mathbf { x }$ LoD).

Common respiratory tract co-infections were determined by reviewing results of previous Respiratory Pathogen Panel clinical trials from published $5 1 0 \mathrm { k }$ summaries, publications/posters and internal clinical sample testing. Each sample was extracted in triplicate using the NucliSENS easyMAG and each extraction tested in singlet with the BioCode RPP on the BioCode MDx-3000 system.

Summary results of this study are presented in Table 33 below. No inhibition was observed at the concentrations tested in this study.

Table 33: Competitive Inhibition Study Results   

<table><tr><td colspan="1" rowspan="1">Panel Designation</td><td colspan="1" rowspan="1">Viral/Bacteria Strain</td><td colspan="1" rowspan="1">Level</td><td colspan="1" rowspan="1">Titer Tested</td><td colspan="1" rowspan="1">Result(n of 3 Detected)</td></tr><tr><td colspan="1" rowspan="3">CompetitiveInhibition Sample 1</td><td colspan="1" rowspan="1">Adenovirus C Serotype 2</td><td colspan="1" rowspan="1">High</td><td colspan="1" rowspan="1">1x105 TCID50/mL</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Respiratory syncytial virusType A</td><td colspan="1" rowspan="1">Low</td><td colspan="1" rowspan="1">0.99 TCID50/mL</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Influenza A H3N2A/Wisconsin/67/05</td><td colspan="1" rowspan="1">Low</td><td colspan="1" rowspan="1">12 TCID50/mL</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="3">CompetitiveInhibition Sample 2</td><td colspan="1" rowspan="1">Respiratory syncytial virusType A</td><td colspan="1" rowspan="1">High</td><td colspan="1" rowspan="1">1x105 TCID50/mL</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Influenza A H3N2A/Wisconsin/67/05</td><td colspan="1" rowspan="1">Low</td><td colspan="1" rowspan="1">12 TCID50/mL</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Adenovirus C Serotype 2</td><td colspan="1" rowspan="1">Low</td><td colspan="1" rowspan="1">18 TCID50/mL</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="3">CompetitiveInhibition Sample 3</td><td colspan="1" rowspan="1">Influenza A H3N2A/Wisconsin/67/05</td><td colspan="1" rowspan="1">High</td><td colspan="1" rowspan="1">1x105 TCID50/mL</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Adenovirus C Serotype 2</td><td colspan="1" rowspan="1">Low</td><td colspan="1" rowspan="1">18 TCID50/mL</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Respiratory syncytial virusType A</td><td colspan="1" rowspan="1">Low</td><td colspan="1" rowspan="1">0.99 TCID50/mL</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="3">CompetitiveInhibition Sample 4</td><td colspan="1" rowspan="1">Coronavirus OC43</td><td colspan="1" rowspan="1">High</td><td colspan="1" rowspan="1">1x105 TCID50/mL</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Human Metapneumovirus</td><td colspan="1" rowspan="1">Low</td><td colspan="1" rowspan="1">45 TCID50/mL</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Bordetella pertussis</td><td colspan="1" rowspan="1">Low</td><td colspan="1" rowspan="1">45 CFU/mL</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="3">CompetitiveInhibition Sample 5</td><td colspan="1" rowspan="1">Human Metapneumovirus</td><td colspan="1" rowspan="1">High</td><td colspan="1" rowspan="1">1x105 TCID50/mL</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Bordetella pertussis</td><td colspan="1" rowspan="1">Low</td><td colspan="1" rowspan="1">45 CFU/mL</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Coronavirus OC43</td><td colspan="1" rowspan="1">Low</td><td colspan="1" rowspan="1">0.12 TCID50/mL</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="3">CompetitiveInhibition Sample 6</td><td colspan="1" rowspan="1">Bordetella pertussis</td><td colspan="1" rowspan="1">High</td><td colspan="1" rowspan="1">1x106 CFU/mL</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Coronavirus OC43</td><td colspan="1" rowspan="1">Low</td><td colspan="1" rowspan="1">0.12 TCID50/mL</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Human Metapneumovirus</td><td colspan="1" rowspan="1">Low</td><td colspan="1" rowspan="1">45 TCID50/mL</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="3">CompetitiveInhibition Sample 7</td><td colspan="1" rowspan="1">Influenza A H1N1 pdmCalifornia/07/09</td><td colspan="1" rowspan="1">High</td><td colspan="1" rowspan="1">1x105 TCID50/mL</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza Virus 3</td><td colspan="1" rowspan="1">Low</td><td colspan="1" rowspan="1">45 TCID50/mL</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Human Rhinovirus type A</td><td colspan="1" rowspan="1">Low</td><td colspan="1" rowspan="1">3.6 TCID50/mL</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="3">CompetitiveInhibition Sample 8</td><td colspan="1" rowspan="1">Parainfluenza Virus 3</td><td colspan="1" rowspan="1">High</td><td colspan="1" rowspan="1">1x105 TCID50/mL</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Human Rhinovirus type A</td><td colspan="1" rowspan="1">Low</td><td colspan="1" rowspan="1">3.6 TCID50/mL</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Influenza A H1N1 pdmCalifornia/07/09</td><td colspan="1" rowspan="1">Low</td><td colspan="1" rowspan="1">1.2 TCID50/mL</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="3">CompetitiveInhibition Sample 9</td><td colspan="1" rowspan="1">Human Rhinovirus type A</td><td colspan="1" rowspan="1">High</td><td colspan="1" rowspan="1">1x105 TCID50/mL</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Influenza A H1N1 pdmCalifornia/07/09</td><td colspan="1" rowspan="1">Low</td><td colspan="1" rowspan="1">1.2 TCID50/mL</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza Virus 3</td><td colspan="1" rowspan="1">Low</td><td colspan="1" rowspan="1">45 TCID50/mL</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="3">CompetitiveInhibition Sample 10</td><td colspan="1" rowspan="1">Mycoplasma pneumoniae</td><td colspan="1" rowspan="1">High</td><td colspan="1" rowspan="1">1x106 CCU/mL</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Coronavirus NL63</td><td colspan="1" rowspan="1">Low</td><td colspan="1" rowspan="1">1.2 TCID50/mL</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Influenza B/Florida/4/2006</td><td colspan="1" rowspan="1">Low</td><td colspan="1" rowspan="1">0.04 TCID50/mL</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="3">CompetitiveInhibition Sample 11</td><td colspan="1" rowspan="1">Coronavirus NL63</td><td colspan="1" rowspan="1">High</td><td colspan="1" rowspan="1">1x105 TCID50/mL</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Influenza B/Florida/4/2006</td><td colspan="1" rowspan="1">Low</td><td colspan="1" rowspan="1">0.04 TCID50/mL</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Mycoplasma pneumoniae</td><td colspan="1" rowspan="1">Low</td><td colspan="1" rowspan="1">45 CCU/mL</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="3">CompetitiveInhibition Sample 12</td><td colspan="1" rowspan="1">Influenza B/Florida/4/2006</td><td colspan="1" rowspan="1">High</td><td colspan="1" rowspan="1">1x105 TCID50/mL</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Mycoplasma pneumoniae</td><td colspan="1" rowspan="1">Low</td><td colspan="1" rowspan="1">45 CCU/mL</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Coronavirus NL63</td><td colspan="1" rowspan="1">Low</td><td colspan="1" rowspan="1">1.2 TCID50/mL</td><td colspan="1" rowspan="1">3/3</td></tr></table>

# B Comparison Studies:

1. Method Comparison with Predicate Device: Not applicable. Refer to the Clinical Studies Section of this document.

2. Matrix Comparison: Not applicable

# C Clinical Studies:

# 1. Prospective Clinical Study

The clinical performance of the BioCode RPP was established in a multi-center study conducted during periods of the 2017-2019 respiratory illness seasons. Residual (leftover) and de-identified nasopharyngeal swab (NPS) specimens in VTM or UTM that were prospectively collected from patients suspected of respiratory tract infections at five geographically diverse clinical sites in the U.S. were enrolled and tested with the BioCode RPP at five testing sites during the prospective clinical study. The enrolled prospective specimens were tested freshly with an FDA-cleared molecular multiplexed respiratory pathogen panel as part of the Stand of Care (SOC), and were either tested freshly with the BioCode RPP (Category I specimens, i.e., specimens that were stored in a $2 { - } 8 ^ { \circ } \mathrm { C }$ refrigerator for no more than 7 days), or stored frozen and then thawed and tested with the BioCode RPP at a testing site at a later date (Category II specimens, i.e., specimens that were initially stored in a $2 { - } 8 ^ { \circ } \mathrm { C }$ refrigerator but were not able to be tested by the BioCode RPP within 7 days from specimen collection).

A waiver of the informed consent requirement was obtained from the Institutional Review Boards (IRBs) at each specimen enrollment site for the use of residual NPS in VTM or UTM specimens.

The following information was recorded on the Case Report Form (CRF) for each subject from whom a specimen was enrolled:

Age and sex • Date and time of specimen collection • Standard of care (SOC) comparator test result Specimen storage status, i.e., fresh or frozen

A total of 2654 residual NPS specimens in VTM or UTM that were prospectively collected at the five clinical sites from August 2017 to May 2019 were enrolled initially for the clinical study. Five specimens were withdrawn from the clinical study due to incomplete data collection and testing, resulted in a total of 2649 prospective specimens (1401 Category I and 1248 Category II specimens) that were included in the prospective clinical study.

The prospective specimens enrolled for evaluation were tested at the five testing sites by trained laboratory personnel. DNA/RNA was extracted using either the BioMerieux NucliSENS easyMAG system or Roche MagNA Pure 96 system. After extraction, the samples were tested using the BioCode RPP on the BioCode MDx-3000 System according to the instructions for use.

Table 34: Specimen Enrollment Sites and Extraction Methods for the BioCode RPP Prospective Clinical Evaluation   

<table><tr><td rowspan=1 colspan=1>Site</td><td rowspan=1 colspan=1>Study SiteLocation</td><td rowspan=1 colspan=1>Category ISpecimenCollected</td><td rowspan=1 colspan=1>Category IISpecimenCollected</td><td rowspan=1 colspan=1>Total Number ofProspectiveSpecimens(Category I and II)</td><td rowspan=1 colspan=1>ExtractionNuclisENseasy MAG</td><td rowspan=1 colspan=1>ExtractionMagNA Pure96</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>New York City, NY</td><td rowspan=1 colspan=1>250</td><td rowspan=1 colspan=1>280</td><td rowspan=1 colspan=1>530</td><td rowspan=1 colspan=1>530</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>Memphis, TN</td><td rowspan=1 colspan=1>182</td><td rowspan=1 colspan=1>237</td><td rowspan=1 colspan=1>419</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>419</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>Tampa, FL</td><td rowspan=1 colspan=1>366</td><td rowspan=1 colspan=1>234</td><td rowspan=1 colspan=1>600</td><td rowspan=1 colspan=1>600</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>Los Angeles, CA</td><td rowspan=1 colspan=1>300</td><td rowspan=1 colspan=1>250</td><td rowspan=1 colspan=1>550</td><td rowspan=1 colspan=1>550</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>Aurora, CO</td><td rowspan=1 colspan=1>303</td><td rowspan=1 colspan=1>247</td><td rowspan=1 colspan=1>550</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>550</td></tr><tr><td rowspan=1 colspan=2>Total</td><td rowspan=1 colspan=1>1401</td><td rowspan=1 colspan=1>1248</td><td rowspan=1 colspan=1>2649</td><td rowspan=1 colspan=1>1680</td><td rowspan=1 colspan=1>969</td></tr></table>

Table 35 below provides a summary of demographic information for the 2649 specimens included in the prospective clinical study.

Table 35: Demographic Summary for the Prospective BioCode RPP Clinical Evaluation   

<table><tr><td rowspan=1 colspan=2></td><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>Site 1</td><td rowspan=1 colspan=1>Site 2</td><td rowspan=1 colspan=1>Site 3</td><td rowspan=1 colspan=1>Site 4</td><td rowspan=1 colspan=1>Site 5</td></tr><tr><td rowspan=2 colspan=1>S</td><td rowspan=1 colspan=1>Male</td><td rowspan=1 colspan=1>1346 (50.8%)</td><td rowspan=1 colspan=1>271</td><td rowspan=1 colspan=1>195</td><td rowspan=1 colspan=1>266</td><td rowspan=1 colspan=1>320</td><td rowspan=1 colspan=1>294</td></tr><tr><td rowspan=1 colspan=1>Female</td><td rowspan=1 colspan=1>1303 (49.2%)</td><td rowspan=1 colspan=1>259</td><td rowspan=1 colspan=1>224</td><td rowspan=1 colspan=1>334</td><td rowspan=1 colspan=1>230</td><td rowspan=1 colspan=1>256</td></tr><tr><td rowspan=4 colspan=1>RE</td><td rowspan=1 colspan=1>≤5 years</td><td rowspan=1 colspan=1>1004 (37.9%)</td><td rowspan=1 colspan=1>18</td><td rowspan=1 colspan=1>161</td><td rowspan=1 colspan=1>91</td><td rowspan=1 colspan=1>360</td><td rowspan=1 colspan=1>374</td></tr><tr><td rowspan=1 colspan=1>6 - 21 years</td><td rowspan=1 colspan=1>609 (23.0%)</td><td rowspan=1 colspan=1>32</td><td rowspan=1 colspan=1>172</td><td rowspan=1 colspan=1>45</td><td rowspan=1 colspan=1>187</td><td rowspan=1 colspan=1>173</td></tr><tr><td rowspan=1 colspan=1>22 - 59 years</td><td rowspan=1 colspan=1>531 (20.0%)</td><td rowspan=1 colspan=1>190</td><td rowspan=1 colspan=1>65</td><td rowspan=1 colspan=1>271</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>2</td></tr><tr><td rowspan=1 colspan=1>60+ years</td><td rowspan=1 colspan=1>505 (19.1%)</td><td rowspan=1 colspan=1>290</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>193</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=2>Total</td><td rowspan=1 colspan=1>2649</td><td rowspan=1 colspan=1>530</td><td rowspan=1 colspan=1>419</td><td rowspan=1 colspan=1>600</td><td rowspan=1 colspan=1>550</td><td rowspan=1 colspan=1>550</td></tr></table>

# Prospective Clinical Study System and Specimen Validity Rate

The overall success rate for initial specimen testing in the prospective study was $9 8 . 8 \%$ (2618/2649) $9 5 \%$ CI: $9 8 . 3 \% - 9 9 . 2 \% )$ ; 31 tests were unsuccessful (26 tests with an invalid result and 5 tests due to low BMB count/instrument error). Upon a single retest per the instructions for use, 29 of the 31 initially unsuccessful specimens generated a valid result. The final validity rate was $9 9 . 9 \%$ (2647/2649) $9 5 \%$ CI: $9 9 . 7 \% - 1 0 0 \%$ ).

# Prospective Clinical Study Performance

Performance of the BioCode RPP was evaluated by comparing the BioCode RPP test results with those of an FDA-cleared molecular multiplexed respiratory pathogen panel. Positive percent agreement (PPA) was calculated as T $\mathrm { P / ( T P + F N ) }$ ( $\mathrm { T P } =$ true positive or positive by both the comparator method and BioCode RPP; $\mathrm { F N } =$ false negative or negative by BioCode RPP only). Negative percent agreement was calculated as $\mathrm { T N } / ( \mathrm { T N } + \mathrm { F P } )$ ( $\mathrm { T N } =$ true negative or negative by both the comparator method and BioCode RPP; $\mathrm { F P = }$ false positive or positive by BioCode RPP only). The score binomial two-sided $9 5 \%$ confidence interval was also calculated. Samples for which false positive and/or false negative results (i.e., discrepant results) were obtained when comparing the BioCode RPP results to the comparator method results were further investigated. The discrepancy investigation was mainly conducted by performing independent molecular tests, including analytically validated PCR followed by bi-directional sequencing assays and alternate NAATs.

Of the 2649 specimens included in the prospective clinical study, two specimens, one Category I and one Category II specimens, obtained a final “invalid” result from the BioCode RPP, and were excluded from the performance analyses for all analytes. In addition, three specimens, one Category I and two Category II specimens, obtained a final influenza A “indeterminate” result by the BioCode RPP, and two specimens, one Category I and one Category II specimens obtained an influenza A “equivocal” result from the comparator method. They were included in the performance analyses for all analytes but excluded from the performance calculations for Flu A and Flu A subtypes. Furthermore, two Category II specimens obtained a valid influenza A result by the comparator method without the accompanying Flu A subtyping results. They were included in the performance analyses for all analytes but excluded from the performance calculations for Flu A subtypes.

The BioCode RPP prospective performance data as positive percent and negative percent agreements against the comparator method (all sites combined) are presented by pathogen in Table 36 below:

Table 36: BioCode RPP Prospective Clinical Performance Summary   

<table><tr><td colspan="2" rowspan="2">Analyte</td><td colspan="3" rowspan="1">Positive Percent Agreement</td><td colspan="3" rowspan="1">Negative Percent Agreement</td></tr><tr><td colspan="1" rowspan="1">TP/(TP + FN)</td><td colspan="1" rowspan="1">%</td><td colspan="1" rowspan="1">95%CI</td><td colspan="1" rowspan="1">TN/(TN + FP)</td><td colspan="1" rowspan="1">%</td><td colspan="1" rowspan="1">95%CI</td></tr><tr><td colspan="8" rowspan="1">Viruses</td></tr><tr><td colspan="1" rowspan="3">Adenovirus a</td><td colspan="1" rowspan="1">Fresh</td><td colspan="1" rowspan="1">31/40</td><td colspan="1" rowspan="1">77.5</td><td colspan="1" rowspan="1">62.5-87.7</td><td colspan="1" rowspan="1">1340/1360</td><td colspan="1" rowspan="1">98.5</td><td colspan="1" rowspan="1">97.7-99.0</td></tr><tr><td colspan="1" rowspan="1">Frozen</td><td colspan="1" rowspan="1">37/38</td><td colspan="1" rowspan="1">97.4</td><td colspan="1" rowspan="1">86.5-99.5</td><td colspan="1" rowspan="1">1188/1209</td><td colspan="1" rowspan="1">98.3</td><td colspan="1" rowspan="1">97.4-98.9</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">68/78</td><td colspan="1" rowspan="1">87.2</td><td colspan="1" rowspan="1">78.0-92.9</td><td colspan="1" rowspan="1">2528/2569</td><td colspan="1" rowspan="1">98.4</td><td colspan="1" rowspan="1">97.8-98.8</td></tr><tr><td colspan="1" rowspan="3">Coronavirus b</td><td colspan="1" rowspan="1">Fresh</td><td colspan="1" rowspan="1">35/50</td><td colspan="1" rowspan="1">70.0</td><td colspan="1" rowspan="1">56.2-80.9</td><td colspan="1" rowspan="1">1338/1350</td><td colspan="1" rowspan="1">99.1</td><td colspan="1" rowspan="1">98.5-99.5</td></tr><tr><td colspan="1" rowspan="1">Frozen</td><td colspan="1" rowspan="1">76/83</td><td colspan="1" rowspan="1">91.6</td><td colspan="1" rowspan="1">83.6-95.9</td><td colspan="1" rowspan="1">1154/1164</td><td colspan="1" rowspan="1">99.1</td><td colspan="1" rowspan="1">98.4-99.5</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">111/133</td><td colspan="1" rowspan="1">83.5</td><td colspan="1" rowspan="1">76.2-88.8</td><td colspan="1" rowspan="1">2492/2514</td><td colspan="1" rowspan="1">99.1</td><td colspan="1" rowspan="1">98.7-99.4</td></tr><tr><td colspan="1" rowspan="3">hMPV c</td><td colspan="1" rowspan="1">Fresh</td><td colspan="1" rowspan="1">89/93</td><td colspan="1" rowspan="1">95.7</td><td colspan="1" rowspan="1">89.5-98.3</td><td colspan="1" rowspan="1">1299/1307</td><td colspan="1" rowspan="1">99.4</td><td colspan="1" rowspan="1">98.8-99.7</td></tr><tr><td colspan="1" rowspan="1">Frozen</td><td colspan="1" rowspan="1">46/49</td><td colspan="1" rowspan="1">93.9</td><td colspan="1" rowspan="1">83.5-97.9</td><td colspan="1" rowspan="1">1189/1198</td><td colspan="1" rowspan="1">99.2</td><td colspan="1" rowspan="1">98.6-99.6</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">135/142</td><td colspan="1" rowspan="1">95.1</td><td colspan="1" rowspan="1">90.2-97.6</td><td colspan="1" rowspan="1">2488/2505</td><td colspan="1" rowspan="1">99.3</td><td colspan="1" rowspan="1">98.9-99.6</td></tr><tr><td colspan="1" rowspan="3">HRV/EVd</td><td colspan="1" rowspan="1">Fresh</td><td colspan="1" rowspan="1">221/261</td><td colspan="1" rowspan="1">84.7</td><td colspan="1" rowspan="1">79.8-88.5</td><td colspan="1" rowspan="1">1119/1139</td><td colspan="1" rowspan="1">98.2</td><td colspan="1" rowspan="1">97.3-98.9</td></tr><tr><td colspan="1" rowspan="1">Frozen</td><td colspan="1" rowspan="1">162/213</td><td colspan="1" rowspan="1">76.1</td><td colspan="1" rowspan="1">69.9-81.3</td><td colspan="1" rowspan="1">1020/1034</td><td colspan="1" rowspan="1">98.6</td><td colspan="1" rowspan="1">97.7-99.2</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">383/474</td><td colspan="1" rowspan="1">80.8</td><td colspan="1" rowspan="1">77.0-84.1</td><td colspan="1" rowspan="1">2139/2173</td><td colspan="1" rowspan="1">98.4</td><td colspan="1" rowspan="1">97.8-98.9</td></tr><tr><td colspan="1" rowspan="3">FluA e</td><td colspan="1" rowspan="1">Fresh</td><td colspan="1" rowspan="1">115/120</td><td colspan="1" rowspan="1">95.8</td><td colspan="1" rowspan="1">90.6-98.2</td><td colspan="1" rowspan="1">1265/1278</td><td colspan="1" rowspan="1">99.0</td><td colspan="1" rowspan="1">98.3-99.4</td></tr><tr><td colspan="1" rowspan="1">Frozen</td><td colspan="1" rowspan="1">98/101</td><td colspan="1" rowspan="1">97.0</td><td colspan="1" rowspan="1">91.6-99.0</td><td colspan="1" rowspan="1">1131/1143</td><td colspan="1" rowspan="1">99.0</td><td colspan="1" rowspan="1">98.2-99.4</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">213/221</td><td colspan="1" rowspan="1">96.4</td><td colspan="1" rowspan="1">93.0-98.2</td><td colspan="1" rowspan="1">2396/2421</td><td colspan="1" rowspan="1">99.0</td><td colspan="1" rowspan="1">98.5-99.3</td></tr><tr><td colspan="1" rowspan="3">FluA/H1</td><td colspan="1" rowspan="1">Fresh</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">1398/1398</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">99.7-100</td></tr><tr><td colspan="1" rowspan="1">Frozen</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">1242/1242</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">99.7-100</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">2640/2640</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">99.9-100</td></tr><tr><td colspan="1" rowspan="3">FluA/H1pdm09</td><td colspan="1" rowspan="1">Fresh</td><td colspan="1" rowspan="1">29/30</td><td colspan="1" rowspan="1">96.7</td><td colspan="1" rowspan="1">83.3-99.4</td><td colspan="1" rowspan="1">1365/1368</td><td colspan="1" rowspan="1">99.8</td><td colspan="1" rowspan="1">99.4-99.9</td></tr><tr><td colspan="1" rowspan="1">Frozen</td><td colspan="1" rowspan="1">23/23</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">85.7-100</td><td colspan="1" rowspan="1">1213/1219</td><td colspan="1" rowspan="1">99.5</td><td colspan="1" rowspan="1">98.9-99.8</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">52/53</td><td colspan="1" rowspan="1">98.1</td><td colspan="1" rowspan="1">90.1-99.7</td><td colspan="1" rowspan="1">2578/2587</td><td colspan="1" rowspan="1">99.7</td><td colspan="1" rowspan="1">99.3-99.8</td></tr><tr><td colspan="1" rowspan="3">FluA/H3 g</td><td colspan="1" rowspan="1">Fresh</td><td colspan="1" rowspan="1">82/88</td><td colspan="1" rowspan="1">93.2</td><td colspan="1" rowspan="1">85.9-96.8</td><td colspan="1" rowspan="1">1306/1310</td><td colspan="1" rowspan="1">99.7</td><td colspan="1" rowspan="1">99.2-99.9</td></tr><tr><td colspan="1" rowspan="1">Frozen</td><td colspan="1" rowspan="1">65/69</td><td colspan="1" rowspan="1">94.2</td><td colspan="1" rowspan="1">86.0-97.7</td><td colspan="1" rowspan="1">1168/1173</td><td colspan="1" rowspan="1">99.6</td><td colspan="1" rowspan="1">99.0-99.8</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">147/157</td><td colspan="1" rowspan="1">93.6</td><td colspan="1" rowspan="1">88.7-96.5</td><td colspan="1" rowspan="1">2474/2483</td><td colspan="1" rowspan="1">99.6</td><td colspan="1" rowspan="1">99.3-99.8</td></tr><tr><td colspan="1" rowspan="3">FluB h</td><td colspan="1" rowspan="1">Fresh</td><td colspan="1" rowspan="1">7/7</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">64.6-100</td><td colspan="1" rowspan="1">1388/1393</td><td colspan="1" rowspan="1">99.6</td><td colspan="1" rowspan="1">99.2-99.8</td></tr><tr><td colspan="1" rowspan="1">Frozen</td><td colspan="1" rowspan="1">44/47</td><td colspan="1" rowspan="1">93.6</td><td colspan="1" rowspan="1">82.8-97.8</td><td colspan="1" rowspan="1">1191/1200</td><td colspan="1" rowspan="1">99.2</td><td colspan="1" rowspan="1">98.6-99.6</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">51/54</td><td colspan="1" rowspan="1">94.4</td><td colspan="1" rowspan="1">84.9-98.1</td><td colspan="1" rowspan="1">2579/2593</td><td colspan="1" rowspan="1">99.5</td><td colspan="1" rowspan="1">99.1-99.7</td></tr><tr><td colspan="1" rowspan="3">PIV1</td><td colspan="1" rowspan="1">Fresh</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">51.0-100</td><td colspan="1" rowspan="1">1396/1396</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">99.7-100</td></tr><tr><td colspan="1" rowspan="1">Frozen</td><td colspan="1" rowspan="1">11/13</td><td colspan="1" rowspan="1">84.6</td><td colspan="1" rowspan="1">57.8-95.7</td><td colspan="1" rowspan="1">1234/1234</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">99.7-100</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">15/17</td><td colspan="1" rowspan="1">88.2</td><td colspan="1" rowspan="1">65.7-96.7</td><td colspan="1" rowspan="1">2630/2630</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">99.9-100</td></tr><tr><td colspan="1" rowspan="3">PIV2 j</td><td colspan="1" rowspan="1">Fresh</td><td colspan="1" rowspan="1">2/3</td><td colspan="1" rowspan="1">66.7</td><td colspan="1" rowspan="1">20.8-93.9</td><td colspan="1" rowspan="1">1396/1397</td><td colspan="1" rowspan="1">99.9</td><td colspan="1" rowspan="1">99.6-100</td></tr><tr><td colspan="1" rowspan="1">Frozen</td><td colspan="1" rowspan="1">8/9</td><td colspan="1" rowspan="1">88.9</td><td colspan="1" rowspan="1">56.5-98.0</td><td colspan="1" rowspan="1">1236/1238</td><td colspan="1" rowspan="1">99.8</td><td colspan="1" rowspan="1">99.4-100</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">10/12</td><td colspan="1" rowspan="1">83.3</td><td colspan="1" rowspan="1">55.2-95.3</td><td colspan="1" rowspan="1">2632/2635</td><td colspan="1" rowspan="1">99.9</td><td colspan="1" rowspan="1">99.7-100</td></tr><tr><td colspan="1" rowspan="3">PIV3 k</td><td colspan="1" rowspan="1">Fresh</td><td colspan="1" rowspan="1">77/79</td><td colspan="1" rowspan="1">97.5</td><td colspan="1" rowspan="1">91.2-99.3</td><td colspan="1" rowspan="1">1312/1321</td><td colspan="1" rowspan="1">99.3</td><td colspan="1" rowspan="1">98.7-99.6</td></tr><tr><td colspan="1" rowspan="1">Frozen</td><td colspan="1" rowspan="1">41/43</td><td colspan="1" rowspan="1">95.3</td><td colspan="1" rowspan="1">84.5-98.7</td><td colspan="1" rowspan="1">1196/1204</td><td colspan="1" rowspan="1">99.3</td><td colspan="1" rowspan="1">98.7-99.7</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">118/122</td><td colspan="1" rowspan="1">96.7</td><td colspan="1" rowspan="1">91.9-98.7</td><td colspan="1" rowspan="1">2508/2525</td><td colspan="1" rowspan="1">99.3</td><td colspan="1" rowspan="1">98.9-99.6</td></tr><tr><td colspan="1" rowspan="3">PIV4 1</td><td colspan="1" rowspan="1">Fresh</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">20.7-100</td><td colspan="1" rowspan="1">1399/1399</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">99.7-100</td></tr><tr><td colspan="1" rowspan="1">Frozen</td><td colspan="1" rowspan="1">15/17</td><td colspan="1" rowspan="1">88.2</td><td colspan="1" rowspan="1">65.7-96.7</td><td colspan="1" rowspan="1">1228/1230</td><td colspan="1" rowspan="1">99.8</td><td colspan="1" rowspan="1">99.4-100</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">16/18</td><td colspan="1" rowspan="1">88.9</td><td colspan="1" rowspan="1">67.2-96.9</td><td colspan="1" rowspan="1">2627/2629</td><td colspan="1" rowspan="1">99.9</td><td colspan="1" rowspan="1">99.7-100</td></tr><tr><td colspan="1" rowspan="3">RSV m</td><td colspan="1" rowspan="1">Fresh</td><td colspan="1" rowspan="1">91/93</td><td colspan="1" rowspan="1">97.8</td><td colspan="1" rowspan="1">92.5-99.4</td><td colspan="1" rowspan="1">1293/1307</td><td colspan="1" rowspan="1">98.9</td><td colspan="1" rowspan="1">98.2-99.4</td></tr><tr><td colspan="1" rowspan="1">Frozen</td><td colspan="1" rowspan="1">109/111</td><td colspan="1" rowspan="1">98.2</td><td colspan="1" rowspan="1">93.7-99.5</td><td colspan="1" rowspan="1">1129/1136</td><td colspan="1" rowspan="1">99.4</td><td colspan="1" rowspan="1">98.7-99.7</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">200/204</td><td colspan="1" rowspan="1">98.0</td><td colspan="1" rowspan="1">95.1-99.2</td><td colspan="1" rowspan="1">2422/2443</td><td colspan="1" rowspan="1">99.1</td><td colspan="1" rowspan="1">98.7-99.4</td></tr><tr><td colspan="2" rowspan="1"></td><td colspan="6" rowspan="1">Bacteria</td></tr><tr><td colspan="1" rowspan="3">. pertussis n</td><td colspan="1" rowspan="1">Fresh</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">20.7-100</td><td colspan="1" rowspan="1">1387/1399</td><td colspan="1" rowspan="1">99.1</td><td colspan="1" rowspan="1">98.5-99.5</td></tr><tr><td colspan="1" rowspan="1">Frozen</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">20.7-100</td><td colspan="1" rowspan="1">1239/1246</td><td colspan="1" rowspan="1">99.4</td><td colspan="1" rowspan="1">98.8-99.7</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">2/2</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">34.2-100</td><td colspan="1" rowspan="1">2626/2645</td><td colspan="1" rowspan="1">99.3</td><td colspan="1" rowspan="1">98.9-99.5</td></tr><tr><td colspan="1" rowspan="3">C. pneumoniae</td><td colspan="1" rowspan="1">Fresh</td><td colspan="1" rowspan="1">2/2</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">34.2-100</td><td colspan="1" rowspan="1">1397/1398</td><td colspan="1" rowspan="1">99.9</td><td colspan="1" rowspan="1">99.6-100</td></tr><tr><td colspan="1" rowspan="1">Frozen</td><td colspan="1" rowspan="1">2/2</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">34.2-100</td><td colspan="1" rowspan="1">1245/1245</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">99.7-100</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">51.0-100</td><td colspan="1" rowspan="1">2642/2643</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">99.8-100</td></tr><tr><td colspan="1" rowspan="3">M. pneumoniae P</td><td colspan="1" rowspan="1">Fresh</td><td colspan="1" rowspan="1">8/8</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">67.6-100</td><td colspan="1" rowspan="1">1381/1392</td><td colspan="1" rowspan="1">99.2</td><td colspan="1" rowspan="1">98.6-99.6</td></tr><tr><td colspan="1" rowspan="1">Frozen</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">72.2-100</td><td colspan="1" rowspan="1">1228/1237</td><td colspan="1" rowspan="1">99.3</td><td colspan="1" rowspan="1">98.6-99.6</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">18/18</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">82.4-100</td><td colspan="1" rowspan="1">2609/2629</td><td colspan="1" rowspan="1">99.2</td><td colspan="1" rowspan="1">98.8-99.5</td></tr></table>

a Adenovirus: PCR/bi-directional sequencing and alternative NAAT were performed on all FNs. PCR/bi-directional sequencing was also performed on all FPs. Of 10 FNs, none was detected by PCR/bi-directional sequencing or alternative NAAT. Of 41 FPs, 37 were not detected by PCR/bi-directional sequencing; 4 were indeterminate by PCR/bi-directional sequencing.

b Coronavirus: PCR/bi-directional sequencing was performed on all FNs and FPs. Alternative NAAT was performed on 14/19 FNs only, but not on FPs. Of 22 FNs, 5 were detected and 17 were not detected by PCR/bi-directional sequencing. 2 were detected and 12 were not detected by alternative NAAT. The 22 FPs were not detected by PCR/bi-directional sequencing.

c Human Metapneumovirus: PCR/bi-directional sequencing was performed on all FNs and FPs. Of 7 FNs, 3 were detected and 4 were not detected by PCR/bi-directional sequencing. Of 17 FPs, 7 were detected and 10 were not detected by PCR/bi-directional sequencing.

d Human Rhinovirus/Enterovirus: PCR/bi-directional sequencing was performed on all FNs and FPs. Alternative NAAT was also performed on 36/91 FNs, but not on FPs. Of 91 FNs, 63 were not detected by PCR/bi-directional sequencing; 18 were detected as HRV C, 1 as HRV A, 2 as Enterovirus D68, 5 as Coxsackieviruses, and 2 as indeterminate HRV C, by PCR/bi-directional sequencing. 25 of 36 FNs that were also tested by alternative NAAT generated a positive result for HRV/EV. Of 34 FPs, 26 were not detected by PCR/bi-directional sequencing; 3 were detected as HRV A, 1 as HRV B, 3 as HRV C, and 1 as Enterovirus D68, by PCR/ bi-directional sequencing.

e Influenza A: PCR/bi-directional sequencing was performed on 7/8 FNs and 24/25FPs. Of 7 FNs that were tested by PCR/bi-directional sequencing, all 7 were not detected by PCR/bi-directional sequencing. Of 24 FPs that were tested by PCR/bi-directional sequencing, 19 were not detected by, 4 were detected as Flu A H3N2, and 1 was detected as Flu A H1N1pdm09.

f Influenza A/H1pdm09: PCR/bi-directional sequencing was performed on all FPs. Of 9 FPs, 4 were detected by PCR/bi-directional sequencing; 5 were not detected by PCR/bi-directional sequencing.

g Influenza A/H3: PCR/bi-directional sequencing was performed on 8/10 FNs and 8/9 FPs. Of 8 FNs that were tested by PCR/bi-directional sequencing, 7 were not detected, and 1 was detected. Of 8 FPs that were tested by PCR/bidirectional sequencing, 3 were detected, 1 was indeterminate, and 4 were not detected.

h Influenza B: PCR/bi-directional sequencing was performed on all FNs and 13/14 FPs. Of 3 FNs, none was detected by PCR/bi-directional sequencing. Of 13 FPs that were tested by PCR/bi-directional sequencing, 1 was detected and 12 were not detected.

i Parainfluenza Virus 1: The 2 FNs were not detected by PCR/bi-directional sequencing.

j Parainfluenza Virus 2: PCR/bi-directional sequencing was performed on all FPs and FNs. Of 2 FNs, 1 was detected and 1 was not detected by PCR/bi-directional sequencing. All 3 FPs were detected by PCR/bi-directional sequencing.

k Parainfluenza Virus 3: PCR/bi-directional sequencing was performed on all FPs and FNs. Of 4 FNs, 2 were detected by PCR/bi-directional sequencing; 1 was indeterminate by PCR/bi-directional sequencing; and 1 was not

detected by PCR/bi-directional sequencing. Of 17 FPs, 8 were detected by PCR/bi-directional sequencing; 1 was indeterminate by PCR/bi-directional sequencing; and 8 were not detected by PCR/bi-directional sequencing. l Parainfluenza Virus 4: PCR/bi-directional sequencing was performed on all FPs and FNs. All 2 FNs and 2 FPs were detected by PCR/bi-directional sequencing.   
m Respiratory Syncytial Virus: PCR/bi-directional sequencing was performed on all FPs and FNs. The 4 FNs were not detected by PCR/bi-directional sequencing. Of 21 FPs, 18 were not detected by PCR/bi-directional sequencing; 2 were detected by PCR/bi-directional sequencing; 1 was indeterminate by PCR/bi-directional sequencing. n Bordetella pertussis: Of 19 FPs, 4 were detected by PCR/bi-directional sequencing; 3 were indeterminate by PCR/bi-directional sequencing; and 12 were not detected by PCR/bi-directional sequencing.   
o Chlamydia pneumoniae: The 1 FP was detected by PCR/bi-directional sequencing.   
p Mycoplasma pneumoniae: PCR/bi-directional sequencing was performed on 18/20 FPs. Of 18 FPs that were tested by PCR/bi-directional sequencing, 7 were detected, 1 was invalid, and 10 were not detected.

The performance of the BioCode RPP overall testing of Category I specimens was similar to that of testing Category II specimens. In addition, the performance of the BioCode RPP testing prospective specimens was also analyzed stratified by study sites and by extraction methods. No significant performance differences between the two extraction methods, NucliSENS easyMAG and MagNA Pure 96, or among the five study sites, were observed.

Prospective Clinical Study Influenza A and Subtyping Analysis

A total of 15 specimens generated an initial “Influenza A Indeterminate” result. 14 of the 15 samples had sufficient residual volume and were retested per the instructions for use. Upon retest, 12 of the 14 specimens obtained a final unequivocal Influenza A and subtyping result and two of the 14 specimens remained as “Influenza A Indeterminate”. The percentage of samples that reported a final “Influenza A Indeterminate” result after retest was $0 . 1 \%$ (3/2647).

A total of 29 specimens generated an initial “Influenza A (no subtype detected)” result. 15 of the 29 samples had sufficient residual volume and were retested per the instructions for use. Upon retest, six of the 15 samples remained to be “Influenza A (no subtype detected)”, one of the 15 samples generated an Influenza A/H3 result, and eight of the 15 samples generated an Influenza A negative result. Out of a total of 238 specimens that obtained a final Influenza A positive result after retest, 20 specimens were reported as “Influenza A (no subtype detected)” ( $8 . 4 \%$ , 20/238).

# Prospective Clinical Study Mixed Infection Analysis

BioCode RPP reported a total of 193 specimens with distinctive multiple pathogen detections $7 . 3 \%$ of all specimens, 193/2647; and $9 . 4 \%$ of all positive specimens, 193/2050). The majority of multiple detections, $8 7 . 0 \%$ (168/193) contained two pathogens, while $1 2 . 4 \%$ (24/193) contained three pathogens, $0 . 5 \%$ (1/193) contained more than three pathogens.

The three pathogens that were most prevalent in multiple detections were HRV/EV (101/193, $5 2 . 3 \%$ ), RSV (59/193, $3 0 . 1 \%$ , and adenovirus (51/193, $2 6 . 4 \%$ (see Table 37 below).

Table 37: Prevalence of Analytes in Multiple Detections as Determined by the BioCode RPP   

<table><tr><td rowspan=1 colspan=1>Analyte</td><td rowspan=1 colspan=2>Prevalence in Multiple Detections(N=193)</td></tr><tr><td rowspan=1 colspan=3>Viruses</td></tr><tr><td rowspan=1 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>51</td><td rowspan=1 colspan=1>26.4%</td></tr><tr><td rowspan=1 colspan=1>Coronavirus</td><td rowspan=1 colspan=1>47</td><td rowspan=1 colspan=1>24.4%</td></tr><tr><td rowspan=1 colspan=1>hMPV</td><td rowspan=1 colspan=1>35</td><td rowspan=1 colspan=1>18.1%</td></tr><tr><td rowspan=1 colspan=1>HRV/EV</td><td rowspan=1 colspan=1>101</td><td rowspan=1 colspan=1>52.3%</td></tr><tr><td rowspan=1 colspan=1>Flu A</td><td rowspan=1 colspan=1>37</td><td rowspan=1 colspan=1>19.2%</td></tr><tr><td rowspan=1 colspan=1>Flu B</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>4.1%</td></tr><tr><td rowspan=1 colspan=1>PIV1</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>1.0%</td></tr><tr><td rowspan=1 colspan=1>PIV2</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>2.1%</td></tr><tr><td rowspan=1 colspan=1>PIV3</td><td rowspan=1 colspan=1>39</td><td rowspan=1 colspan=1>20.2%</td></tr><tr><td rowspan=1 colspan=1>PIV4</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>4.1%</td></tr><tr><td rowspan=1 colspan=1>RSV</td><td rowspan=1 colspan=1>59</td><td rowspan=1 colspan=1>30.1%</td></tr><tr><td rowspan=1 colspan=3>Bacteria</td></tr><tr><td rowspan=1 colspan=1>B. pertussis</td><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>7.3%</td></tr><tr><td rowspan=1 colspan=1>C. pneumoniae</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.5%</td></tr><tr><td rowspan=1 colspan=1>M. pneumoniae</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>3.6%</td></tr></table>

A summary of specimens with co-infections as detected by the BioCode RPP during the prospective clinical study is presented in Table 38 below.

Table 38: Co-infections Detected by the BioCode RPP in the Prospective Clinical Trial   

<table><tr><td rowspan=1 colspan=1>Pathogens Detected Simultaneously</td><td rowspan=1 colspan=1>Detected Co-infections</td><td rowspan=1 colspan=1>Discrepant Co-infections a</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>168</td><td rowspan=1 colspan=1>83</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>9</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>Total Co-infections</td><td rowspan=1 colspan=1>193</td><td rowspan=1 colspan=1>93</td></tr></table>

a A discrepant sample of co-infection was defined as a sample that contains at least one pathogen that was detected by BioCode RPP but not detected by the comparator method.

A summary of the most prevalent co-infections as detected by the BioCode RPP $( \geq 5$ instances) during the prospective clinical study is presented in Table 39 below

Table 39: Most Prevalent Co-infections Detected by the BioCode RPP (≥ 5 Instances) in the Prospective Clinical Trial   

<table><tr><td rowspan=1 colspan=1>Co-Infection Combination</td><td rowspan=1 colspan=1>Number of Specimen</td></tr><tr><td rowspan=1 colspan=1>Human Rhinovirus/Enterovirus + Respiratory Syncytial Virus</td><td rowspan=1 colspan=1>17</td></tr><tr><td rowspan=1 colspan=1>Adenovirus + Human Rhinovirus/Enterovirus</td><td rowspan=1 colspan=1>13</td></tr><tr><td rowspan=1 colspan=1>Human Metapneumovirus + Human Rhinovirus/Enterovirus</td><td rowspan=1 colspan=1>13</td></tr><tr><td rowspan=1 colspan=1>Human Rhinovirus/Enterovirus + Influenza A</td><td rowspan=1 colspan=1>11</td></tr><tr><td rowspan=1 colspan=1>Adenovirus + Respiratory Syncytial Virus</td><td rowspan=1 colspan=1>10</td></tr><tr><td rowspan=1 colspan=1>Human Rhinovirus/Enterovirus + Parainfluenza Virus 3</td><td rowspan=1 colspan=1>10</td></tr><tr><td rowspan=1 colspan=1>Coronavirus + Human Rhinovirus/Enterovirus</td><td rowspan=1 colspan=1>8</td></tr><tr><td rowspan=1 colspan=1>Coronavirus + Respiratory Syncytial Virus</td><td rowspan=1 colspan=1>7</td></tr><tr><td rowspan=1 colspan=1>Bordetella pertussis + Human Rhinovirus/Enterovirus</td><td rowspan=1 colspan=1>6</td></tr><tr><td rowspan=1 colspan=1>Adenovirus + Parainfluenza Virus 3</td><td rowspan=1 colspan=1>5</td></tr><tr><td rowspan=1 colspan=1>Coronavirus + Human Metapneumovirus</td><td rowspan=1 colspan=1>5</td></tr><tr><td rowspan=1 colspan=1>Coronavirus + Influenza A</td><td rowspan=1 colspan=1>5</td></tr></table>

# 2. Retrospective Clinical Study

Several of the pathogens on the BioCode RPP were of low prevalence and were not encountered in sufficiently large numbers during the prospective study to adequately demonstrate system performance. To supplement the results of the prospective clinical study, an evaluation of preselected archived retrospective specimens was performed. These specimens were archived NPS in VTM or UTM specimens that were selected because they had previously tested positive for one of the following pathogens at the source laboratory: coronavirus 229E, coronavirus HKU1, parainfluenza virus 1, parainfluenza virus 2, parainfluenza virus 4, Bordetella pertussis, Chlamydia pneumoniae, and Mycoplasma pneumoniae, or had been negative in previous laboratory testing.

A total of 165 clinical specimens were enrolled for testing in this retrospective study. The specimens were randomized such that the users performing the BioCode RPP assay were blinded to the expected test result and shipped to one of the five testing sites participated in the prospective clinical study for testing.

A summary of the demographic information of the tested retrospective samples is provided in Table 40 below.

Table 40: Available Demographic Summary for All Retrospective Specimens Included in the Retrospective Study   

<table><tr><td rowspan=2 colspan=1>Sex</td><td rowspan=1 colspan=1>Female (%)</td><td rowspan=1 colspan=1>69 (41.8%)</td></tr><tr><td rowspan=1 colspan=1>Male (%)</td><td rowspan=1 colspan=1>96 (58.2%)</td></tr><tr><td rowspan=4 colspan=1>Age</td><td rowspan=1 colspan=1>≤ 5 years</td><td rowspan=1 colspan=1>77 (46.7%)</td></tr><tr><td rowspan=1 colspan=1>6 - 21 years</td><td rowspan=1 colspan=1>58 (35.2%)</td></tr><tr><td rowspan=1 colspan=1>22 - 59 years</td><td rowspan=1 colspan=1>15 (9.1%)</td></tr><tr><td rowspan=1 colspan=1>60+ years</td><td rowspan=1 colspan=1>15 (9.1%)</td></tr><tr><td rowspan=1 colspan=2>Total Specimens</td><td rowspan=1 colspan=1>165</td></tr></table>

# Retrospective Clinical Study Performance

The performance of the BioCode RPP was evaluated by comparing the BioCode RPP test results with those from an FDA-cleared molecular multiplexed respiratory pathogen panel, the same panel test as the one used as the comparator in the prospective clinical study. The BioCode RPP retrospective performance data expressed as positive percent and negative percent agreements against the comparator method are presented by pathogen in Table 41 below.

Table 41: BioCode RPP Retrospective Clinical Study Performance Summary   

<table><tr><td rowspan=2 colspan=1>Analyte</td><td rowspan=1 colspan=3>Positive Percent Agreement</td><td rowspan=1 colspan=3>Negative Percent Agreement</td></tr><tr><td rowspan=1 colspan=1>TP/(TP + FN)</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=1>TN/(TN +FP</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=5>Viruses</td></tr><tr><td rowspan=1 colspan=1>Adenovirus a</td><td rowspan=1 colspan=1>7/7</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>64.6-100</td><td rowspan=1 colspan=1>155/158</td><td rowspan=1 colspan=1>98.1</td><td rowspan=1 colspan=1>94.6-99.4</td></tr><tr><td rowspan=1 colspan=1>Coronavirus b</td><td rowspan=1 colspan=1>52/59</td><td rowspan=1 colspan=1>88.1</td><td rowspan=1 colspan=1>77.5-94.1</td><td rowspan=1 colspan=1>99/106</td><td rowspan=1 colspan=1>93.4</td><td rowspan=1 colspan=1>87.0-96.8</td></tr><tr><td rowspan=1 colspan=1>hMPV</td><td rowspan=1 colspan=1>4/4</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>51.0-100</td><td rowspan=1 colspan=1>161/161</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>97.7-100</td></tr><tr><td rowspan=1 colspan=1>HRV/EV</td><td rowspan=1 colspan=1>16/23</td><td rowspan=1 colspan=1>69.6</td><td rowspan=1 colspan=1>49.1-84.4</td><td rowspan=1 colspan=1>141/142</td><td rowspan=1 colspan=1>99.3</td><td rowspan=1 colspan=1>96.1-99.9</td></tr><tr><td rowspan=1 colspan=1>Influenza A</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>165/165</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>97.7-100</td></tr><tr><td rowspan=1 colspan=1>Influenza A/H1</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>165/165</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>97.7-100</td></tr><tr><td rowspan=1 colspan=1>Influenza A/H1pdm09</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>165/165</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>97.7-100</td></tr><tr><td rowspan=1 colspan=1>Influenza A/H3</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>165/165</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>97.7-100</td></tr><tr><td rowspan=1 colspan=1>Influenza B d</td><td rowspan=1 colspan=1>2/3</td><td rowspan=1 colspan=1>66.7</td><td rowspan=1 colspan=1>20.8-93.9</td><td rowspan=1 colspan=1>162/162</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>97.7-100</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza Virus 1e</td><td rowspan=1 colspan=1>12/13</td><td rowspan=1 colspan=1>92.3</td><td rowspan=1 colspan=1>66.7-98.6</td><td rowspan=1 colspan=1>152/152</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>97.5-100</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza Virus 2 f</td><td rowspan=1 colspan=1>19/20</td><td rowspan=1 colspan=1>95.0</td><td rowspan=1 colspan=1>76.4-99.1</td><td rowspan=1 colspan=1>144/145</td><td rowspan=1 colspan=1>99.3</td><td rowspan=1 colspan=1>96.2-99.9</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza Virus 3</td><td rowspan=1 colspan=1>1/1</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>20.7-100</td><td rowspan=1 colspan=1>164/164</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>97.7-100</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza Virus 4 g</td><td rowspan=1 colspan=1>14/15</td><td rowspan=1 colspan=1>93.3</td><td rowspan=1 colspan=1>70.2-98.8</td><td rowspan=1 colspan=1>150/150</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>97.5-100</td></tr><tr><td rowspan=1 colspan=1>RSV h</td><td rowspan=1 colspan=1>11/12</td><td rowspan=1 colspan=1>91.7</td><td rowspan=1 colspan=1>64.6-98.5</td><td rowspan=1 colspan=1>152/153</td><td rowspan=1 colspan=1>99.3</td><td rowspan=1 colspan=1>96.4-99.9</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2>Bacteria</td><td rowspan=1 colspan=2></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Bordetella pertussis</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>72.2-100</td><td rowspan=1 colspan=1>144/155</td><td rowspan=1 colspan=1>92.9</td><td rowspan=1 colspan=1>87.7-96.0</td></tr><tr><td rowspan=1 colspan=1>Chlamydia pneumoniae</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>72.2-100</td><td rowspan=1 colspan=1>155/155</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>97.6-100</td></tr><tr><td rowspan=1 colspan=1>Mycoplasma pneumoniae </td><td rowspan=1 colspan=1>7/7</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>64.6-100</td><td rowspan=1 colspan=1>153/158</td><td rowspan=1 colspan=1>96.8</td><td rowspan=1 colspan=1>92.8-98.6</td></tr></table>

a Adenovirus: PCR/bi-directional sequencing was performed on all FPs. Of 3 FPs, 1 was detected by PCR/bidirectional sequencing and 2 were not detected by PCR/bi-directional sequencing.

b Coronavirus: PCR/bi-directional sequencing was performed on all FNs and FPs. Of 7 FNs, 4 were detected and 3 were not detected by PCR/bi-directional sequencing. Of 7 FPs, none was detected by PCR/bi-directional sequencing.

c Human Rhinovirus/Enterovirus: PCR/bi-directional sequencing was performed on all FNs and FPs. Of 7 FNs and 1 FP, none was detected by PCR/bi-directional sequencing.   
d Influenza B: 1 FN was not detected by PCR/bi-directional sequencing.   
e Parainfluenza Virus 1: 1 FN was not detected by PCR/bi-directional sequencing   
f Parainfluenza Virus 2: PCR/bi-directional sequencing was performed on FN and FP samples. 1 FN was not detected by PCR/bi-directional sequencing. 1 FP was not detected by PCR/bi-directional sequencing.   
g Parainfluenza Virus 4: 1 FN was not detected by PCR/bi-directional sequencing.   
h Respiratory Syncytial Virus: PCR/bi-directional sequencing was performed on FN and FP samples. 1 FN and 1 FP were not detected by PCR/bi-directional sequencing.   
i Bordetella pertussis: PCR/bi-directional sequencing was performed on all FPs. Of 11 FPs, 7 were detected by PCR/bi-directional sequencing, and 4 were not detected by PCR/bi-directional sequencing.   
j Mycoplasma pneumoniae: PCR/bi-directional sequencing was performed on all FPs. Of 5 FPs, 3 were detected by PCR/bi-directional sequencing and 2 was not detected by PCR/bi-directional sequencing.

# 3. Testing of Contrived Clinical Specimens

Some respiratory pathogens are so rare that both the prospective and archived specimen collection efforts were insufficient to demonstrate the clinical performance. To supplement the prospective clinical study and retrospective archived study data, an evaluation of contrived specimens was performed for two pathogens: Chlamydia pneumoniae and Influenza A/H1. These contrived clinical specimens were prepared using 50 unique natural NPS in VTM or UTM specimens that were previously tested negative for all BioCode RPP analytes. Contrived specimens were spiked at concentrations of $2 \mathbf { x }$ LoD or greater using different strains for each pathogen. The 50 positive samples of each pathogen were prepared, interspersed with negative samples and randomized before testing at one of the five testing sites participated in the prospective clinical study.

A total of 110 samples, including 100 contrived positive samples, were tested. The results of the BioCode RPP testing are presented in Table 42 below.

Table 42: BioCode RPP Performance Testing Contrived Specimens   

<table><tr><td rowspan=1 colspan=1>Pathogen</td><td rowspan=1 colspan=1>Source</td><td rowspan=1 colspan=1>Strain/Isolate</td><td rowspan=1 colspan=1>Concentration(xLoD)</td><td rowspan=1 colspan=1>PPA (%)</td><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=1>NPA (%)</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=10 colspan=1>Chlamydiapneumoniae</td><td rowspan=3 colspan=1>ATCC 53592</td><td rowspan=3 colspan=1>AR-39</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>9/9 (100%)</td><td rowspan=1 colspan=1>70.1%, 100%</td><td rowspan=10 colspan=1>60/60(100%)</td><td rowspan=10 colspan=1>94.0%, 100%</td></tr><tr><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>5/5 (100%)</td><td rowspan=1 colspan=1>56.6%, 100%</td></tr><tr><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>4/4 (100%)</td><td rowspan=1 colspan=1>51.0%, 100%</td></tr><tr><td rowspan=3 colspan=1>ATCC VR- 1360</td><td rowspan=3 colspan=1>CM-1</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>8/8 (100%)</td><td rowspan=1 colspan=1>67.6%, 100%</td></tr><tr><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>5/5 (100%)</td><td rowspan=1 colspan=1>56.6%, 100%</td></tr><tr><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>3/3 (100%)</td><td rowspan=1 colspan=1>43.8%, 100%</td></tr><tr><td rowspan=3 colspan=1>ATCC VR- 1310</td><td rowspan=3 colspan=1>CWL-029</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>8/8 (100%)</td><td rowspan=1 colspan=1>67.6%, 100%</td></tr><tr><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>5/5 (100%)</td><td rowspan=1 colspan=1>56.6%, 100%</td></tr><tr><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>3/3 (100%)</td><td rowspan=1 colspan=1>43.8%, 100%</td></tr><tr><td rowspan=1 colspan=3>Combined</td><td rowspan=1 colspan=1>50/50 (100%)</td><td rowspan=1 colspan=1>92.9%, 100%</td></tr><tr><td rowspan=1 colspan=1>Pathogen</td><td rowspan=1 colspan=1>Source</td><td rowspan=1 colspan=1>Strain/Isolate</td><td rowspan=1 colspan=1>Concentration(xLoD)</td><td rowspan=1 colspan=1>PPA (%)</td><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=1>NPA (%)</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=2 colspan=1>InfluenzaA/H1N1</td><td rowspan=2 colspan=1>Zeptometrix0810036CF</td><td rowspan=2 colspan=1>A/NewCaledonia/20/99</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>5/5 (100%)</td><td rowspan=1 colspan=1>56.6%, 100%</td><td rowspan=2 colspan=1>60/60(100%)</td><td rowspan=2 colspan=1>94.0%, 100%</td></tr><tr><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>3/3 (100%)</td><td rowspan=1 colspan=1>43.8%, 100%</td></tr></table>

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>3/3 (100%)</td><td rowspan=1 colspan=1>43.8%, 100%</td><td rowspan=14 colspan=1>b</td></tr><tr><td rowspan=3 colspan=1>Zeptometrix0810247CF</td><td rowspan=3 colspan=1>A/Taiwan/42/06</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>5/5 (100%)</td><td rowspan=1 colspan=1>56.6%, 100%</td></tr><tr><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>3/3 (100%)</td><td rowspan=1 colspan=1>43.8%, 100%</td></tr><tr><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>2/2 (100%)</td><td rowspan=1 colspan=1>34.2%, 100%</td></tr><tr><td rowspan=3 colspan=1>Zeptometrix0810246CF</td><td rowspan=3 colspan=1>Singapore/63/04</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>5/5 (100%)</td><td rowspan=1 colspan=1>56.6%, 100%</td></tr><tr><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>2/2 (100%)</td><td rowspan=1 colspan=1>34.2%, 100%</td></tr><tr><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>2/2 (100%)</td><td rowspan=1 colspan=1>34.2%, 100%</td></tr><tr><td rowspan=3 colspan=1>Virapur</td><td rowspan=3 colspan=1>A/Denver/1/1957</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>5/5 (100%)</td><td rowspan=1 colspan=1>56.6%, 100%</td></tr><tr><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>2/2 (100%)</td><td rowspan=1 colspan=1>34.2%, 100%</td></tr><tr><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>2/2 (100%)</td><td rowspan=1 colspan=1>34.2%, 100%</td></tr><tr><td rowspan=3 colspan=1>ATCC VR-219</td><td rowspan=3 colspan=1>A/NWS/33</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>5/5 (100%)</td><td rowspan=1 colspan=1>56.6%,,100%</td></tr><tr><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>3/3 (100%)</td><td rowspan=1 colspan=1>43.8%, 100%</td></tr><tr><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>3/3 (100%)</td><td rowspan=1 colspan=1>43.8%, 100%</td></tr><tr><td rowspan=1 colspan=3>Combined</td><td rowspan=1 colspan=1>50/50 (100%)</td><td rowspan=1 colspan=1>92.9%, 100%</td></tr></table>

# D Clinical Cut-Off:

Not applicable

# E Expected Values/Reference Range:

In the BioCode RPP prospective clinical study, a total of 2647 NPS eluted in VTM or UTM specimens were evaluable by the BioCode RPP. The number and percentage of positive cases per site (Table 43) and per age group (Table 44), as determined by BioCode RPP, are presented below.

Table 43: BioCode RPP Expected Values by Specimen Collection Site   

<table><tr><td colspan="1" rowspan="2">Pathogen</td><td colspan="2" rowspan="1">Overall (n=2647)</td><td colspan="1" rowspan="1">Site</td><td colspan="1" rowspan="1">Site 1 (n=529)</td><td colspan="2" rowspan="1">Site 2 (n=419)</td><td colspan="2" rowspan="1">Site 3 (n=599)</td><td colspan="2" rowspan="1">Site 4 (n=550)</td><td colspan="2" rowspan="1">Site 5 (n=550)</td></tr><tr><td colspan="1" rowspan="1">N</td><td colspan="1" rowspan="1">Expectedvalue</td><td colspan="1" rowspan="1">N</td><td colspan="1" rowspan="1">Expectedvalue</td><td colspan="1" rowspan="1">N</td><td colspan="1" rowspan="1">Expectedvalue</td><td colspan="1" rowspan="1">N</td><td colspan="1" rowspan="1">Expectedvalue</td><td colspan="1" rowspan="1">N</td><td colspan="1" rowspan="1">Expectedvalue</td><td colspan="1" rowspan="1">N</td><td colspan="1" rowspan="1">Expectedvalue</td></tr><tr><td colspan="1" rowspan="1">Adenovirus</td><td colspan="1" rowspan="1">109</td><td colspan="1" rowspan="1">4.1%</td><td colspan="1" rowspan="1">8</td><td colspan="1" rowspan="1">1.5%</td><td colspan="1" rowspan="1">27</td><td colspan="1" rowspan="1">6.4%</td><td colspan="1" rowspan="1">21</td><td colspan="1" rowspan="1">3.5%</td><td colspan="1" rowspan="1">17</td><td colspan="1" rowspan="1">3.1%</td><td colspan="1" rowspan="1">36</td><td colspan="1" rowspan="1">6.5%</td></tr><tr><td colspan="1" rowspan="1">Coronavirus</td><td colspan="1" rowspan="1">133</td><td colspan="1" rowspan="1">5.0%</td><td colspan="1" rowspan="1">36</td><td colspan="1" rowspan="1">6.8%</td><td colspan="1" rowspan="1">21</td><td colspan="1" rowspan="1">5.0%</td><td colspan="1" rowspan="1">14</td><td colspan="1" rowspan="1">2.3%</td><td colspan="1" rowspan="1">26</td><td colspan="1" rowspan="1">4.7%</td><td colspan="1" rowspan="1">36</td><td colspan="1" rowspan="1">6.5%</td></tr><tr><td colspan="1" rowspan="1">Human Metapneumovirus</td><td colspan="1" rowspan="1">152</td><td colspan="1" rowspan="1">5.7%</td><td colspan="1" rowspan="1">18</td><td colspan="1" rowspan="1">3.4%</td><td colspan="1" rowspan="1">22</td><td colspan="1" rowspan="1">5.3%</td><td colspan="1" rowspan="1">26</td><td colspan="1" rowspan="1">4.3%</td><td colspan="1" rowspan="1">38</td><td colspan="1" rowspan="1">6.9%</td><td colspan="1" rowspan="1">48</td><td colspan="1" rowspan="1">8.7%</td></tr><tr><td colspan="1" rowspan="1">HumanRhinovirus/Enterovirus</td><td colspan="1" rowspan="1">417</td><td colspan="1" rowspan="1">15.8%</td><td colspan="1" rowspan="1">48</td><td colspan="1" rowspan="1">9.1%</td><td colspan="1" rowspan="1">83</td><td colspan="1" rowspan="1">19.8%</td><td colspan="1" rowspan="1">65</td><td colspan="1" rowspan="1">10.9%</td><td colspan="1" rowspan="1">125</td><td colspan="1" rowspan="1">22.7%</td><td colspan="1" rowspan="1">96</td><td colspan="1" rowspan="1">17.5%</td></tr><tr><td colspan="1" rowspan="1">Influenza A</td><td colspan="1" rowspan="1">238</td><td colspan="1" rowspan="1">9.0%</td><td colspan="1" rowspan="1">49</td><td colspan="1" rowspan="1">9.3%</td><td colspan="1" rowspan="1">50</td><td colspan="1" rowspan="1">11.9%</td><td colspan="1" rowspan="1">35</td><td colspan="1" rowspan="1">5.8%</td><td colspan="1" rowspan="1">41</td><td colspan="1" rowspan="1">7.5%</td><td colspan="1" rowspan="1">63</td><td colspan="1" rowspan="1">11.5%</td></tr><tr><td colspan="1" rowspan="1">Influenza A/H1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="1" rowspan="1">Influenza A/H1pdm09</td><td colspan="1" rowspan="1">62</td><td colspan="1" rowspan="1">2.3%</td><td colspan="1" rowspan="1">13</td><td colspan="1" rowspan="1">2.5%</td><td colspan="1" rowspan="1">7</td><td colspan="1" rowspan="1">1.7%</td><td colspan="1" rowspan="1">11</td><td colspan="1" rowspan="1">1.8%</td><td colspan="1" rowspan="1">22</td><td colspan="1" rowspan="1">4.0%</td><td colspan="1" rowspan="1">9</td><td colspan="1" rowspan="1">1.6%</td></tr><tr><td colspan="1" rowspan="1">Influenza A/H3</td><td colspan="1" rowspan="1">157</td><td colspan="1" rowspan="1">5.9%</td><td colspan="1" rowspan="1">32</td><td colspan="1" rowspan="1">6.0%</td><td colspan="1" rowspan="1">40</td><td colspan="1" rowspan="1">9.5%</td><td colspan="1" rowspan="1">19</td><td colspan="1" rowspan="1">3.2%</td><td colspan="1" rowspan="1">17</td><td colspan="1" rowspan="1">3.1%</td><td colspan="1" rowspan="1">49</td><td colspan="1" rowspan="1">8.9%</td></tr><tr><td colspan="1" rowspan="1">Influenza B</td><td colspan="1" rowspan="1">65</td><td colspan="1" rowspan="1">2.5%</td><td colspan="1" rowspan="1">12</td><td colspan="1" rowspan="1">2.3%</td><td colspan="1" rowspan="1">32</td><td colspan="1" rowspan="1">7.6%</td><td colspan="1" rowspan="1">15</td><td colspan="1" rowspan="1">2.5%</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0.4%</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">0.7%</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza virus 1</td><td colspan="1" rowspan="1">15</td><td colspan="1" rowspan="1">0.6%</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">0.8%</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">11</td><td colspan="1" rowspan="1">1.8%</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza virus 2</td><td colspan="1" rowspan="1">13</td><td colspan="1" rowspan="1">0.5%</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0.5%</td><td colspan="1" rowspan="1">8</td><td colspan="1" rowspan="1">1.5%</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0.4%</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza virus 3</td><td colspan="1" rowspan="1">135</td><td colspan="1" rowspan="1">5.1%</td><td colspan="1" rowspan="1">12</td><td colspan="1" rowspan="1">2.3%</td><td colspan="1" rowspan="1">28</td><td colspan="1" rowspan="1">6.7%</td><td colspan="1" rowspan="1">41</td><td colspan="1" rowspan="1">6.8%</td><td colspan="1" rowspan="1">9</td><td colspan="1" rowspan="1">1.6%</td><td colspan="1" rowspan="1">45</td><td colspan="1" rowspan="1">8.2%</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza virus 4</td><td colspan="1" rowspan="1">18</td><td colspan="1" rowspan="1">0.7%</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0.2%</td><td colspan="1" rowspan="1">9</td><td colspan="1" rowspan="1">1.5%</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">0.9%</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0.5%</td></tr><tr><td colspan="1" rowspan="1">Respiratory Syncytial Virus</td><td colspan="1" rowspan="1">221</td><td colspan="1" rowspan="1">8.3%</td><td colspan="1" rowspan="1">21</td><td colspan="1" rowspan="1">4.0%</td><td colspan="1" rowspan="1">35</td><td colspan="1" rowspan="1">8.4%</td><td colspan="1" rowspan="1">25</td><td colspan="1" rowspan="1">4.2%</td><td colspan="1" rowspan="1">49</td><td colspan="1" rowspan="1">8.9%</td><td colspan="1" rowspan="1">91</td><td colspan="1" rowspan="1">16.5%</td></tr><tr><td colspan="1" rowspan="2">Pathogen</td><td colspan="2" rowspan="1">Overall (n=2647)</td><td colspan="2" rowspan="1">Site 1 (n=529)</td><td colspan="2" rowspan="1">Site 2 (n=419)</td><td colspan="2" rowspan="1">Site 3 (n=599)</td><td colspan="2" rowspan="1">Site 4 (n=550)</td><td colspan="2" rowspan="1">Site 5 (n=550)</td></tr><tr><td colspan="1" rowspan="1">N</td><td colspan="1" rowspan="1">Expectedvalue</td><td colspan="1" rowspan="1">N</td><td colspan="1" rowspan="1">Expectedvalue</td><td colspan="1" rowspan="1">N</td><td colspan="1" rowspan="1">Expectedvalue</td><td colspan="1" rowspan="1">N</td><td colspan="1" rowspan="1">Expectedvalue</td><td colspan="1" rowspan="1">N</td><td colspan="1" rowspan="1">Expectedvalue</td><td colspan="1" rowspan="1">N</td><td colspan="1" rowspan="1">Expectedvalue</td></tr><tr><td colspan="1" rowspan="1">Bordetella Pertussis</td><td colspan="1" rowspan="1">21</td><td colspan="1" rowspan="1">0.8%</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">19</td><td colspan="1" rowspan="1">4.5%</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0.2%</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0.2%</td></tr><tr><td colspan="1" rowspan="1">Chlamydophila pneumoniae</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">0.2%</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0.7%</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0.2%</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0.2%</td></tr><tr><td colspan="1" rowspan="1">Mycoplasma pneumoniae</td><td colspan="1" rowspan="1">38</td><td colspan="1" rowspan="1">1.4%</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">0.8%</td><td colspan="1" rowspan="1">15</td><td colspan="1" rowspan="1">3.6%</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">0.8%</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0.5%</td><td colspan="1" rowspan="1">11</td><td colspan="1" rowspan="1">2.0%</td></tr></table>

Table 44: BioCode RPP Expected Values by Age Group   

<table><tr><td rowspan=2 colspan=1>Pathogen</td><td rowspan=1 colspan=2>Overall(n=2647)</td><td rowspan=1 colspan=2>≤ 5 Years(n=1004)</td><td rowspan=1 colspan=2>6-21 Years(n=609)</td><td rowspan=1 colspan=2>22-59 Years(n=531)</td><td rowspan=1 colspan=2>60+ Years(n=503)</td></tr><tr><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>Expectedvalue</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>Expectedvalue</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>Expectedvalue</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>Expectedvalue</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>Expectedvalue</td></tr><tr><td rowspan=1 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>109</td><td rowspan=1 colspan=1>4.1%</td><td rowspan=1 colspan=1>80</td><td rowspan=1 colspan=1>8.0%</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>3.3%</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>0.9%</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>0.8%</td></tr><tr><td rowspan=1 colspan=1>Coronavirus</td><td rowspan=1 colspan=1>133</td><td rowspan=1 colspan=1>5.0%</td><td rowspan=1 colspan=1>63</td><td rowspan=1 colspan=1>6.3%</td><td rowspan=1 colspan=1>27</td><td rowspan=1 colspan=1>4.4%</td><td rowspan=1 colspan=1>19</td><td rowspan=1 colspan=1>3.6%</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>4.8%</td></tr><tr><td rowspan=1 colspan=1>Human Metapneumovirus</td><td rowspan=1 colspan=1>152</td><td rowspan=1 colspan=1>5.7%</td><td rowspan=1 colspan=1>94</td><td rowspan=1 colspan=1>9.4%</td><td rowspan=1 colspan=1>26</td><td rowspan=1 colspan=1>4.3%</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>2.8%</td><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>3.4%</td></tr><tr><td rowspan=1 colspan=1>HumanRhinovirus/Enterovirus</td><td rowspan=1 colspan=1>417</td><td rowspan=1 colspan=1>15.8%</td><td rowspan=1 colspan=1>234</td><td rowspan=1 colspan=1>23.3%</td><td rowspan=1 colspan=1>101</td><td rowspan=1 colspan=1>16.6%</td><td rowspan=1 colspan=1>51</td><td rowspan=1 colspan=1>9.6%</td><td rowspan=1 colspan=1>31</td><td rowspan=1 colspan=1>6.2%</td></tr><tr><td rowspan=1 colspan=1>Influenza A</td><td rowspan=1 colspan=1>238</td><td rowspan=1 colspan=1>9.0%</td><td rowspan=1 colspan=1>71</td><td rowspan=1 colspan=1>7.1%</td><td rowspan=1 colspan=1>84</td><td rowspan=1 colspan=1>13.8%</td><td rowspan=1 colspan=1>47</td><td rowspan=1 colspan=1>8.9%</td><td rowspan=1 colspan=1>36</td><td rowspan=1 colspan=1>7.2%</td></tr><tr><td rowspan=1 colspan=1>Influenza A/H1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td></tr><tr><td rowspan=1 colspan=1>Influenza A/H1pdm09</td><td rowspan=1 colspan=1>62</td><td rowspan=1 colspan=1>2.3%</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>2.4%</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>2.5%</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>2.8%</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>1.6%</td></tr><tr><td rowspan=1 colspan=1>Influenza A/H3</td><td rowspan=1 colspan=1>157</td><td rowspan=1 colspan=1>5.9%</td><td rowspan=1 colspan=1>45</td><td rowspan=1 colspan=1>4.5%</td><td rowspan=1 colspan=1>61</td><td rowspan=1 colspan=1>10.0%</td><td rowspan=1 colspan=1>26</td><td rowspan=1 colspan=1>4.9%</td><td rowspan=1 colspan=1>25</td><td rowspan=1 colspan=1>5.0%</td></tr><tr><td rowspan=1 colspan=1>Influenza B</td><td rowspan=1 colspan=1>65</td><td rowspan=1 colspan=1>2.5%</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>1.3%</td><td rowspan=1 colspan=1>26</td><td rowspan=1 colspan=1>4.3%</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>2.8%</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>2.2%</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza virus 1</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>0.6%</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>0.4%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.2%</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>0.8%</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>1.2%</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza virus 2</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>0.5%</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>0.6%</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>0.7%</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0.6%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza virus 3</td><td rowspan=1 colspan=1>135</td><td rowspan=1 colspan=1>5.1%</td><td rowspan=1 colspan=1>74</td><td rowspan=1 colspan=1>7.4%</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>3.4%</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>4.0%</td><td rowspan=1 colspan=1>19</td><td rowspan=1 colspan=1>3.8%</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza virus 4</td><td rowspan=1 colspan=1>18</td><td rowspan=1 colspan=1>0.7%</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>1.2%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.2%</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0.4%</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0.6%</td></tr><tr><td rowspan=1 colspan=1>Respiratory Syncytial Virus</td><td rowspan=1 colspan=1>221</td><td rowspan=1 colspan=1>8.3%</td><td rowspan=1 colspan=1>156</td><td rowspan=1 colspan=1>15.5%</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>4.9%</td><td rowspan=1 colspan=1>19</td><td rowspan=1 colspan=1>3.6%</td><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>3.2%</td></tr><tr><td rowspan=1 colspan=1>Bordetella Pertussis</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>0.8%</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>0.8%</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>2.0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.2%</td></tr><tr><td rowspan=1 colspan=1>Chlamydophila pneumoniae</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>0.2%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.1%</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0.5%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.2%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td></tr><tr><td rowspan=1 colspan=1>Mycoplasma pneumoniae</td><td rowspan=1 colspan=1>38</td><td rowspan=1 colspan=1>1.4%</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>0.9%</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>3.4%</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>1.1%</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0.4%</td></tr></table>

# F Other Supportive Instrument Performance Characteristics Data:

Not applicable

# VIII Proposed Labeling:

The labeling supports the finding of substantial equivalence for this device.

# IX Conclusion:

The submitted information in this premarket notification is complete and supports a substantial equivalence decision.