Applied BioCode, Inc. Robert Tullio Regulatory Consultant 10020 Pioneer Blvd Suite 102 Santa Fe Springs, California 90670

Re: K192485 Trade/Device Name: BioCode Respiratory Pathogen Panel (RPP) Regulation Number: 21 CFR 866.3980 Regulation Name: Respiratory viral panel multiplex nucleic acid assay Regulatory Class: Class II Product Code: OCC, OZE, OEP, OEM, OOU, OTG, OZX, OZY, OZZ, NSU Dated: December 5, 2019 Received: December 6, 2019

Dear Robert Tullio:

We have reviewed your Section 510(k) premarket notification of intent to market the device referenced above and have determined the device is substantially equivalent (for the indications for use stated in the enclosure) to legally marketed predicate devices marketed in interstate commerce prior to May 28, 1976, the enactment date of the Medical Device Amendments, or to devices that have been reclassified in accordance with the provisions of the Federal Food, Drug, and Cosmetic Act (Act) that do not require approval of a premarket approval application (PMA). You may, therefore, market the device, subject to the general controls provisions of the Act. Although this letter refers to your product as a device, please be aware that some cleared products may instead be combination products. The 510(k) Premarket Notification Database located at https://www.accessdata.fda.gov/scripts/cdrh/cfdocs/cfpmn/pmn.cfm identifies combination product submissions. The general controls provisions of the Act include requirements for annual registration, listing of devices, good manufacturing practice, labeling, and prohibitions against misbranding and adulteration. Please note: CDRH does not evaluate information related to contract liability warranties. We remind you, however, that device labeling must be truthful and not misleading.

If your device is classified (see above) into either class II (Special Controls) or class III (PMA), it may be subject to additional controls. Existing major regulations affecting your device can be found in the Code of Federal Regulations, Title 21, Parts 800 to 898. In addition, FDA may publish further announcements concerning your device in the Federal Register.

Please be advised that FDA's issuance of a substantial equivalence determination does not mean that FDA has made a determination that your device complies with other requirements of the Act or any Federal statutes and regulations administered by other Federal agencies. You must comply with all the Act's requirements, including, but not limited to: registration and listing (21 CFR Part 807); labeling (21 CFR Part 801 and Part 809); medical device reporting (reporting of medical device-related adverse events) (21 CFR 803) for devices or postmarketing safety reporting (21 CFR 4, Subpart B) for combination products (see

https://www.fda.gov/combination-products/guidance-regulatory-information/postmarketing-safety-reportingcombination-products); good manufacturing practice requirements as set forth in the quality systems (QS) regulation (21 CFR Part 820) for devices or current good manufacturing practices (21 CFR 4, Subpart A) for combination products; and, if applicable, the electronic product radiation control provisions (Sections 531- 542 of the Act); 21 CFR 1000-1050.

Also, please note the regulation entitled, "Misbranding by reference to premarket notification" (21 CFR Part 807.97). For questions regarding the reporting of adverse events under the MDR regulation (21 CFR Part 803), please go to https://www.fda.gov/medical-devices/medical-device-safety/medical-device-reportingmdr-how-report-medical-device-problems.

For comprehensive regulatory information about medical devices and radiation-emitting products, including information about labeling regulations, please see Device Advice (https://www.fda.gov/medical  
devices/device-advice-comprehensive-regulatory-assistance) and CDRH Learn   
(https://www.fda.gov/training-and-continuing-education/cdrh-learn). Additionally, you may contact the Division of Industry and Consumer Education (DICE) to ask a question about a specific regulatory topic. See the DICE website (https://www.fda.gov/medical-devices/device-advice-comprehensive-regulatoryassistance/contact-us-division-industry-and-consumer-education-dice) for more information or contact DICE by email (DICE@fda.hhs.gov) or phone (1-800-638-2041 or 301-796-7100).

Sincerely,

Tamara Feldblyum, Ph.D.   
Chief   
Viral Respiratory and STI Branch   
Division of Microbiology Devices   
OHT7: Office of In Vitro Diagnostics   
and Radiological Health   
Office of Product Evaluation and Quality Center for Devices and Radiological Health

Enclosure

510(k) Number (if known) K192485

Device Name BioCode Respiratory Pathogen Panel (RPP)

Indications for Use (Describe)   
T Ri intndeor use wi BioCd MDx-3000 Instruent.The BiodRPPs capable the mltaneo deten nd tiiulvnbctrharnl l otaine fromindividuals wit signs and/or syptoms  respiratory tract infection.Thefollowng pathogens and subtypes are identified using the BioCode RPP:   
Adenovirus   
• Coronavirus (229E, OC43, HKU1, and NL63)   
•Human Metapneumovirus A/B   
• Influenza A, including subtypes H1, H1 2009 Pandemic, and H3   
Influenza B   
Parainfluenza Virus 1   
Parainfluenza Virus 2   
Parainfluenza Virus 3   
Parainfluenza Virus 4   
Respiratory Syncytial Virus A/B   
Rhinovirus/Enterovirus   
Bordetella pertussis   
Chlamydia pneumoniae   
Mycoplasma pneumoniae

Thettebclvl/ other patient management decisions.

N alaclra necessary when evaluating a patient with possible respiratory tract infection.

heus naliThidReHuuE viu ul neativ r HumanRhinvirs/Enterovirus erxamiatin usi BideR e nirmed by ltera (e.g. FDA cleared molecular tests).

Perorance haracteristics or InluenzaA were established when Influenza A H1 2009 Pandemic and A H3 whe strains are circulating or a novel Influenza A virus emerges.

reende  ubalriee houl colliriecntro e attempted in these cases unless a BSL $^ { 3 + }$ facility is available to receive and culture specimens.

# CONTINUE ON A SEPARATE PAGE IF NEEDED.

This section applies only to requirements of the Paperwork Reduction Act of 1995.

# DO NOT SEND YOUR COMPLETED FORM TO THE PRA STAFF EMAIL ADDRESS BELOW.

Teburenohoctioation tatverehours pse,nc time to reviewinstructions, search existing dat sources, gather and maintain thedata needed and coplete and review the colection f information. Send comments regarding this burden estimate or anyother aspect of this information collection, including suggestions for reducing this burden, to:

Department of Health and Human Services Food and Drug Administration Office of Chief Information Officer Paperwork Reduction Act (PRA) Staff PRAStaff@fda.hhs.gov

"An agency may not conduct or sponsor, and a person is not required to respond to, a collection of information unless it displays a currently valid OMB number."

# 1.0 510(k) SUMMARY

Introduction: According to the requirements of 21 CFR 807.92, the following provides information to understand the basis for a determination of substantial equivalence.

# Submitted by:

Applied BioCode®, Inc. 10020 Pioneer Blvd. Suite 102 Santa Fe Springs, CA 90670

# Contact:

Robert Di Tullio Regulatory Consultant rditullio@apbiocode.com Telephone: 310 801 1235 Fax: 323 372 3816

Date Submitted: September 9, 2019

Trade Name: BioCode® Respiratory Pathogen Panel (RPP)

Classification Name and Regulation Number: Respiratory Viral Panel Multiplex Nucleic Acid Assay (21 CFR 866.3980)

Predicate Device: K170604 – BioFire FilmArray Respiratory Pathogen Panel 2 (RP2)

# Intended Use:

# BioCode® Respiratory Pathogen Panel (RPP)

The BioCode® Respiratory Pathogen Panel (RPP) is a qualitative multiplexed nucleic acid-based in vitro diagnostic test intended for use with BioCode MDx-3000 Instrument. The BioCode RPP is capable of the simultaneous detection and identification of nucleic acids from multiple viruses and bacteria extracted from nasopharyngeal swab (NPS) samples obtained from individuals with signs and/or symptoms of respiratory tract infection. The following pathogens and subtypes are identified using the BioCode RPP:

• Adenovirus   
• Coronavirus (229E, OC43, HKU1, and NL63)   
• Human Metapneumovirus A/B   
• Influenza A, including subtypes H1, H1 2009 Pandemic, and H3   
• Influenza B   
• Parainfluenza 1   
• Parainfluenza 2   
• Parainfluenza 3   
• Parainfluenza 4

• Respiratory Syncytial Virus A/B • Rhinovirus/Enterovirus • Bordetella pertussis • Chlamydia pneumoniae Mycoplasma pneumoniae

The detection and identification of specific viral and bacterial nucleic acids from individuals exhibiting signs and/or symptoms of a respiratory infection aids in the diagnosis of respiratory infection if used in conjunction with other clinical and epidemiological information. The results of this test should not be used as the sole basis for diagnosis, treatment, or other patient management decisions.

Negative results in the setting of a respiratory illness may be due to infection with pathogens that are not detected by this test, or lower respiratory tract infection that may not be detected by a nasopharyngeal swab specimen. Positive results do not rule out co-infection with other organisms: the agent(s) detected by the BioCode RPP may not be the definite cause of disease. Additional laboratory testing (e.g. bacterial and viral culture, immunofluorescence, and radiography) may be necessary when evaluating a patient with possible respiratory tract infection.

Due to the genetic similarity between Human Rhinovirus and Enterovirus, the BioCode RPP cannot differentiate them. A positive BioCode RPP Rhinovirus/Enterovirus result should be followed up using an alternate method (e.g., cell culture or sequence analysis) if differentiation is required. The BioCode RPP detects Human Rhinovirus/Enterovirus with reduced sensitivity. If a more accurate HRV/EV result is required, it is recommended that specimens found to be negative for Human Rhinovirus/Enterovirus after examination using BioCode RPP be confirmed by an alternate method (e.g. FDA cleared molecular tests).

Performance characteristics for Influenza A were established when Influenza A H1 2009 Pandemic and A H3 were the predominant Influenza A viruses in circulation. Performance of detecting Influenza A may vary if other Influenza A strains are circulating or a novel Influenza A virus emerges. If infection with a novel Influenza A virus is suspected based on current clinical and epidemiological screening criteria recommended by public health authorities, specimens should be collected with appropriate infection control precautions for novel virulent Influenza viruses and sent to state or local health departments for testing. Viral culture should not be attempted in these cases unless a BSL $^ { 3 + }$ facility is available to receive and culture specimens.

# Device Description:

The BioCode® Respiratory Pathogen Panel is a respiratory pathogen multiplex nucleic acid test designed for use with the BioCode® MDx-3000 system. The BioCode® MDx-3000 is an automated system that integrates PCR amplification, target capture, signal generation and optical detection for multiple viral and bacterial pathogens from a single nasopharyngeal swab specimen collected in transport media. Specimens are processed and nucleic acids extracted with the NucliSens easyMAG or Roche MagNA Pure 96 automated systems. Once the PCR plate is set up and sealed, all other operations are automated on MDx-3000. The BioCode® RPP simultaneously tests for 17 pathogens and/or subtypes (see table below) from nasopharyngeal swab specimens collected in UTM or VTM. Results from the BioCode RPP test are available within about 5 hours, including off-board nucleic acids extraction.

<table><tr><td rowspan=1 colspan=1>Viruses</td><td rowspan=1 colspan=1>Viruses</td></tr><tr><td rowspan=1 colspan=1>Influenza A</td><td rowspan=1 colspan=1>Respiratory Syncytial Virus A and B</td></tr><tr><td rowspan=1 colspan=1>• Subtype H1</td><td rowspan=1 colspan=1>Human Metapneumovirus A and B</td></tr><tr><td rowspan=1 colspan=1>•  Subtype H1 2009pdm</td><td rowspan=1 colspan=1>Rhinovirus/Enterovirus</td></tr><tr><td rowspan=1 colspan=1>• Subtype H3</td><td rowspan=1 colspan=1>Coronavirus (229E, OC43, HKU1, and NL63)</td></tr><tr><td rowspan=1 colspan=1>Influenza B</td><td rowspan=1 colspan=1>Adenovirus</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza 1</td><td rowspan=1 colspan=1>Bacteria</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza 2</td><td rowspan=1 colspan=1>Mycoplasma pneumoniae</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza 3</td><td rowspan=1 colspan=1>Chlamydia pneumoniae</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza 4</td><td rowspan=1 colspan=1>Bordetella pertussis</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Internal Control (MS2)</td></tr></table>

# Device Comparison:

Comparison of the BioCode RPP with the Predicate Device   

<table><tr><td rowspan=1 colspan=1>Characteristic</td><td rowspan=1 colspan=1>Proposed Device</td><td rowspan=1 colspan=1>Predicate</td></tr><tr><td rowspan=1 colspan=1>Name</td><td rowspan=1 colspan=1>BioCode® Respiratory PathogenPanel (RPP)</td><td rowspan=1 colspan=1>BioFire FilmArray RespiratoryPathogen Panel 2 (RP2)</td></tr><tr><td rowspan=1 colspan=1>Common Name</td><td rowspan=1 colspan=1>Respiratory Pathogen PanelMultiplex Nucleic acid assay</td><td rowspan=1 colspan=1>Respiratory Viral PanelMultiplex Nucleic acid assay</td></tr><tr><td rowspan=1 colspan=1>510(k) No.</td><td rowspan=1 colspan=1>K192485</td><td rowspan=1 colspan=1>K170604</td></tr><tr><td rowspan=1 colspan=1>Regulation</td><td rowspan=1 colspan=1>21CFR 866.3980</td><td rowspan=1 colspan=1>21CFR 866.3980</td></tr><tr><td rowspan=1 colspan=1>Product Code</td><td rowspan=1 colspan=1>OCC, OZE, OEM, OOU, OEP,OTG, OZX, OZY, OZZ, NSU</td><td rowspan=1 colspan=1>OCC, OEM, OOU, OEP, OTG,OZX, OZY, OZZ, 001</td></tr><tr><td rowspan=1 colspan=1>Device Class</td><td rowspan=1 colspan=1>II</td><td rowspan=1 colspan=1>II</td></tr></table>

<table><tr><td>Similarities</td></tr><tr><td></td></tr></table>

<table><tr><td>the simultaneous detection and identification of nucleic acids from multiple viruses and bacteria extracted from nasopharyngeal swab (NPS) samples symptoms of respiratory tract infection. identified using the BioCode RPP: Adenovirus NL63) Influenza B Parainfluenza 1</td><td>obtained from individuals with signs and/or The following pathogens and subtypes are Coronavirus (229E, OC43, HKU1, and</td><td>detection and identification of multiple respiratory viral and bacterial nucleic acids in nasopharyngeal swabs (NPS) obtained from individuals suspected of respiratory tract infections. The following organism types and subtypes are identified using the FilmArray RP2: Adenovirus Coronavirus 229E Coronavirus HKU1 Coronavirus NL63</td></tr><tr><td></td><td>Human Metapneumovirus A/B Influenza A, including subtypes H1,</td><td>Coronavirus OC43 Human Metapneumovirus</td></tr><tr><td></td><td></td><td></td></tr><tr><td></td><td>H1 2009 Pandemic, and H3</td><td>Influenza A, including subtypes H1,</td></tr><tr><td></td><td></td><td>Human Rhinovirus/Enterovirus</td></tr><tr><td></td><td></td><td></td></tr><tr><td></td><td></td><td></td></tr><tr><td></td><td></td><td>H1-2009, and H3</td></tr><tr><td></td><td></td><td>Influenza B</td></tr><tr><td>Parainfluenza 2</td><td></td><td>Parainfluenza Virus 1</td></tr><tr><td>Parainfluenza 3</td><td></td><td>Parainfluenza Virus 2</td></tr><tr><td>Parainfluenza 4</td><td></td><td>Parainfluenza Virus 3</td></tr><tr><td></td><td>Respiratory Syncytial Virus A/B</td><td>Parainfluenza Virus 4</td></tr><tr><td>Bordetella pertussis</td><td>Rhinovirus/Enterovirus</td><td>Respiratory Syncytial Virus</td></tr><tr><td>Chlamydia pneumoniae</td><td></td><td>Bordetella parapertussis (IS1001) Bordetella pertussis (ptxP)</td></tr><tr><td></td><td>Mycoplasma pneumoniae</td><td>Chlamydia pneumoniae Mycoplasma pneumoniae</td></tr><tr><td></td><td>The detection and identification of specific viral and bacterial nucleic acids from</td><td>The detection and identification of specific viral and bacterial nucleic acids from</td></tr><tr><td></td><td>individuals exhibiting signs and/or</td><td>individuals exhibiting signs and/or symptoms</td></tr><tr><td></td><td>symptoms of a respiratory infection aids in</td><td>of a respiratory infection aids in the</td></tr><tr><td></td><td>the diagnosis of respiratory infection if</td><td></td></tr><tr><td></td><td>used in conjunction with other clinical and</td><td>diagnosis of respiratory infection if used in</td></tr><tr><td></td><td></td><td>conjunction with other clinical and</td></tr><tr><td></td><td>epidemiological information. The results of</td><td></td></tr><tr><td></td><td>this test should not be used as the sole</td><td>epidemiological information. The results of</td></tr><tr><td></td><td>basis for diagnosis, treatment, or other</td><td>this test should not be used as the sole basis</td></tr><tr><td></td><td></td><td>for diagnosis, treatment, or other patient</td></tr><tr><td></td><td>patient management decisions.</td><td>management decisions.</td></tr><tr><td></td><td>Negative results in the setting of a</td><td>Negative results in the setting of a</td></tr><tr><td></td><td>respiratory illness may be due to infection</td><td></td></tr><tr><td></td><td>with pathogens that are not detected by</td><td>respiratory illness may be due to infection</td></tr><tr><td></td><td>this test, or lower respiratory tract</td><td>with pathogens that are not detected by this</td></tr><tr><td></td><td>infection that may not be detected by a</td><td>test, or lower respiratory tract infection that</td></tr><tr><td></td><td>nasopharyngeal swab specimen. Positive</td><td>may not be detected by a nasopharyngeal</td></tr><tr><td></td><td></td><td></td></tr><tr><td></td><td>results do not rule out co-infection with</td><td>swab specimen. Positive results do not rule</td></tr><tr><td></td><td>other organisms: the agent(s) detected by</td><td>out co-infection with other organisms: the</td></tr><tr><td></td><td></td><td></td></tr><tr><td></td><td>the BioCode RPP may not be the definite</td><td>agent(s) detected by the FilmArray RP2 may</td></tr><tr><td></td><td></td><td></td></tr><tr><td></td><td></td><td></td></tr><tr><td></td><td></td><td></td></tr><tr><td></td><td></td><td>not be the definite cause of disease.</td></tr><tr><td></td><td></td><td></td></tr><tr><td></td><td></td><td></td></tr><tr><td></td><td></td><td></td></tr><tr><td></td><td></td><td></td></tr><tr><td></td><td></td><td></td></tr><tr><td></td><td></td><td></td></tr><tr><td></td><td>cause of disease. Additional laboratory</td><td></td></tr><tr><td></td><td></td><td></td></tr><tr><td></td><td></td><td>Additional laboratory testing (e.g. bacterial</td></tr><tr><td></td><td></td><td></td></tr><tr><td></td><td></td><td></td></tr><tr><td></td><td></td><td></td></tr><tr><td></td><td></td><td></td></tr><tr><td></td><td></td><td></td></tr><tr><td></td><td></td><td></td></tr><tr><td></td><td></td><td></td></tr><tr><td></td><td></td><td></td></tr><tr><td></td><td></td><td></td></tr><tr><td></td><td>testing (e.g. bacterial and viral culture,</td><td></td></tr><tr><td></td><td></td><td></td></tr><tr><td></td><td></td><td></td></tr><tr><td></td><td></td><td></td></tr><tr><td></td><td></td><td></td></tr><tr><td></td><td></td><td></td></tr><tr><td></td><td></td><td></td></tr><tr><td></td><td></td><td></td></tr><tr><td></td><td></td><td></td></tr><tr><td></td><td></td><td></td></tr><tr><td></td><td></td><td></td></tr><tr><td></td><td></td><td></td></tr><tr><td></td><td>immunofluorescence, and radiography)</td><td></td></tr><tr><td></td><td></td><td></td></tr><tr><td></td><td></td><td></td></tr><tr><td></td><td></td><td></td></tr><tr><td></td><td></td><td></td></tr><tr><td></td><td></td><td></td></tr><tr><td></td><td></td><td></td></tr><tr><td></td><td></td><td></td></tr><tr><td></td><td></td><td></td></tr><tr><td></td><td></td><td></td></tr><tr><td></td><td></td><td></td></tr><tr><td></td><td></td><td></td></tr><tr><td></td><td></td><td></td></tr><tr><td></td><td></td><td></td></tr><tr><td></td><td></td><td></td></tr><tr><td></td><td></td><td></td></tr><tr><td></td><td></td><td></td></tr><tr><td></td><td></td><td></td></tr><tr><td></td><td></td><td></td></tr><tr><td></td><td></td><td></td></tr><tr><td></td><td></td><td></td></tr><tr><td></td><td></td><td></td></tr><tr><td></td><td></td><td></td></tr><tr><td></td><td></td><td></td></tr><tr><td></td><td></td><td></td></tr><tr><td></td><td></td><td></td></tr><tr><td></td><td></td><td>and viral culture, immunofluorescence, and</td></tr><tr><td></td><td></td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">patient with possible respiratory tractinfection.Due to the genetic similarity betweenHuman Rhinovirus and Enterovirus, theBioCode RPP cannot differentiate them. Apositive BioCode RPPRhinovirus/Enterovirus result should befollowed up using an alternate method(e.g., cell culture or sequence analysis) ifdifferentiation is required. The BioCodeRPP detects Human Rhinovirus/Enteroviruswith reduced sensitivity. If a more accurateHRV/EV result is required, it isrecommended that specimens found to benegative for Human Rhinovirus/Enterovirusafter examination using BioCode RPP beconfirmed by an alternate method (e.g.FDA cleared molecular tests).Performance characteristics for Influenza Awere established when Influenza A H12009 Pandemic and A H3 were thepredominant Influenza A viruses incirculation. Performance of detectingInfluenza A may vary if other Influenza Astrains are circulating or a novel Influenza Avirus emerges. If infection with a novelInfluenza A virus is suspected based oncurrent clinical and epidemiologicalscreening criteria recommended by publichealth authorities, specimens should becollected with appropriate infectioncontrol precautions for novel virulentInfluenza viruses and sent to state or localhealth departments for testing. Viralculture should not be attempted in thesecases unless a BSL 3+ facility is available toreceive and culture specimens.</td><td colspan="1" rowspan="1">evaluating a patient with possiblerespiratory tract infection.Due to the genetic similarity betweenHuman Rhinovirus and Enterovirus, theFilmArray RP2 cannot reliably differentiatethem. A positive FilmArray RP2Rhinovirus/Enterovirus result should befollowed up using an alternate method (e.g.,cell culture or sequence analysis) ifdifferentiation is required.Performance characteristics for Influenza Awere established when Influenza A H1-2009,A H1, and A H3 were the predominantInfluenza A viruses in circulation.Performance of detecting Influenza A mayvary if other Influenza A strains arecirculating or a novel Influenza A virusemerges. If infection with a novel InfluenzaA virus is suspected based on current clinicaland epidemiological screening criteriarecommended by public health authorities,specimens should be collected withappropriate infection control precautions fornovel virulent Influenza viruses and sent tostate or local health departments for testing.Viral culture should not be attempted inthese cases unless a BSL 3+ facility isavailable to receive and culture specimens.</td></tr><tr><td colspan="1" rowspan="1">Sample Type</td><td colspan="1" rowspan="1">Nasopharyngeal swab in Viral or UniversalTransport Media</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Controls</td><td colspan="1" rowspan="1">Externally Sourced</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="3" rowspan="1">Differences</td></tr><tr><td colspan="1" rowspan="1">Methodology</td><td colspan="1" rowspan="1">Multiplex RT-PCR in a single reaction andprobe hybridization followed byfluorescence detection and decoding ofbarcoded magnetic beads (BMB) thatcapture biotinylated products withstreptavidin conjugate</td><td colspan="1" rowspan="1">Nested multiplex PCR executed in twostages. First, a single, large volume, highlymultiplexed reverse transcription PCR (RT-PCR) reaction. Second, nested PCR, isperformed in singleplex fashion in each wellof the array. Followed by fluorescentdetection of images of the PCR reactions.</td></tr></table>

# Summary of Performance Characteristics of the BioCode RPP®

# Clinical Performance

# Study Overview

The clinical performance of the BioCode RPP was established in a multi-center study conducted during periods of the 2017-2019 respiratory illness seasons. Residual (leftover) and de-identified nasopharyngeal swab (NPS) specimens in VTM or UTM that were prospectively collected from patients suspected of respiratory tract infections at five geographically diverse clinical sites in the U.S. were enrolled and tested with the BioCode RPP at five testing sites during the prospective clinical study. The enrolled prospective specimens were tested freshly with an FDA-cleared molecular multiplexed respiratory pathogen panel as part of the Stand of Care (SOC), and were either tested freshly with the BioCode RPP (i.e., specimens that were stored in a ${ 2 { - } 8 ^ { \circ } } \mathsf C$ refrigerator for no more than 7 days), or stored frozen and then thawed and tested with the BioCode RPP at a testing site at a later date (i.e., specimens that were initially stored in a ${ 2 { - } 8 ^ { \circ } } \mathsf C$ refrigerator but were not able to be tested by the BioCode RPP within 7 days from specimen collection).

A waiver of the informed consent requirement was obtained from the Institutional Review Boards (IRBs) at each specimen enrollment site for the use of residual NPS in VTM or UTM specimens.

The following information was recorded on the Case Report Form (CRF) for each subject from whom a specimen was enrolled:

Age and sex • Date and time of specimen collection Standard of care (SOC) comparator test result • Specimen storage status, i.e., fresh or frozen

A total of 2654 residual NPS specimens in VTM or UTM that were prospectively collected at the five clinical sites from August 2017 to May 2019 were enrolled initially for the clinical study. Five specimens were withdrawn from the clinical study due to incomplete data collection and testing, resulted in a total of 2649 prospective specimens (1401 fresh and 1248 frozen specimens) that were included in the prospective clinical study.

The prospective specimens enrolled for evaluation were tested at the five testing sites by trained laboratory personnel. DNA/RNA was extracted using either the BioMerieux NucliSENS easyMAG system or Roche MagNA Pure 96 system. After extraction, the samples were tested using the BioCode RPP on the BioCode MDx-3000 System according to the instructions for use.

Demographics – Prospective Samples   

<table><tr><td>Characteristic</td><td>Prospective Study Specimens</td></tr><tr><td>Total Specimens</td><td>2649</td></tr><tr><td>Gender (n/N (%))</td><td></td></tr><tr><td>Male</td><td>1346/2649 (50.8%)</td></tr><tr><td>Female</td><td>1303/2649 (49.2%)</td></tr><tr><td>Age Category (n/N (%))</td><td></td></tr><tr><td>0-5 yrs</td><td>1004/2649 (37.9%)</td></tr><tr><td>6-21 yrs</td><td>609/2649 (23.0%)</td></tr><tr><td>22-59 yrs</td><td>531/2649 (20.0%)</td></tr><tr><td>60+ yrs</td><td>505/2649 (19.1%)</td></tr><tr><td>Status (n/N (%))a</td><td></td></tr><tr><td>Inpatient</td><td>1555/2646 (58.8%)</td></tr><tr><td>Outpatient</td><td>1091/2646 (41.2%)</td></tr></table>

a – Inpatient/outpatient status was unavailable for 3 specimens.

Prospective Sample Type and Test Method Breakdown   

<table><tr><td colspan="2"></td><td colspan="2">Storage Method</td><td colspan="2">Extraction Method</td></tr><tr><td>Site</td><td>Samples Tested</td><td>Fresh</td><td>Frozen</td><td>easyMAG</td><td>MagNA Pure 96</td></tr><tr><td>Site 01</td><td>530</td><td>250</td><td>280</td><td>530</td><td>o</td></tr><tr><td>Site 02</td><td>419</td><td>182</td><td>237</td><td>0</td><td>419</td></tr><tr><td>Site 03</td><td>600</td><td>366</td><td>234</td><td>600</td><td>0</td></tr><tr><td>Site 04</td><td>550</td><td>300</td><td>250</td><td>550</td><td>0</td></tr><tr><td>Site 05</td><td>550</td><td>303</td><td>247</td><td>0</td><td>550</td></tr><tr><td>Total:</td><td>2649</td><td>1401</td><td>1248</td><td>1680</td><td>969</td></tr></table>

Performance of the BioCode RPP was evaluated by comparing the BioCode RPP test results with those from an FDA-cleared molecular multiplexed respiratory pathogen panel. Positive agreement was calculated as TP/ $\left( \mathsf { T } \mathsf { P } + \mathsf { F } \mathsf { N } \right)$ . TP $=$ true positive or positive by both the comparator test and BioCode RPP; $\mathsf { F N } =$ false negative or negative by BioCode RPP only. Negative agreement was calculated as TN/ $\left[ \mathsf { T N } + \right.$ FP). TN $=$ true negative or negative by both the comparator test and BioCode RPP; $\mathsf { F P } =$ false positive or positive by BioCode RPP only. The two-sided $9 5 \%$ confidence interval was calculated with Score method (per CLSI EP12-A2).

Samples for which false positive and/or false negative results (i.e., discrepant results) were obtained when comparing the BioCode RPP results to the comparator test results were further investigated. The discrepancy investigation was mainly conducted by performing independent molecular tests, including analytically validated PCR followed by bi-directional sequencing assays and alternate NAATs.

Of the 2649 specimens included in the prospective clinical study, two specimens, one fresh and one frozen specimen, obtained a final “invalid” result from the BioCode RPP, and were excluded from the performance analyses for all analytes. In addition, three specimens, one fresh and two frozen specimens, obtained a final influenza A “indeterminate” result by the BioCode RPP, and two specimens, one fresh and one frozen, obtained an influenza A “equivocal” result from the comparator test. They were included in the performance analyses for all analytes but excluded from the performance

calculations for Flu A and Flu A subtypes. Furthermore, two frozen specimens obtained a valid influenza A result from the comparator test without the accompanying Flu A subtyping results. They were included in the performance analyses for all analytes but excluded from the performance calculations for Flu A subtypes.

The Prospective study results stratified by storage condition are presented in the table below.

Table. Summary of Clinical Study results: Prospective specimens stratified by storage condition.   

<table><tr><td></td><td></td><td></td><td colspan="2">Positive Agreement</td><td colspan="2">Negative Agreement</td></tr><tr><td>Target</td><td>Storage</td><td>(n)</td><td>PA (%)</td><td>95% Cl</td><td>NA (%)</td><td>95% Cl</td></tr><tr><td>Adenovirusa</td><td>Fresh</td><td>1400</td><td>31/40 (77.5%)</td><td>(62.5%, 87.7%)</td><td>1340/1360 (98.5%)</td><td>(97.7%, 99.0%)</td></tr><tr><td></td><td>Frozen</td><td>1247</td><td>37/38 (97.4%)</td><td>(86.5%, 99.5%)</td><td>1188/1209 (98.3%)</td><td>(97.4%, 98.9%)</td></tr><tr><td></td><td>Total</td><td>2647</td><td>68/78 (87.2%)</td><td>(78.0%, 92.9%)</td><td>2528/2569 (98.4%)</td><td>(97.8%, 98.8%)</td></tr><tr><td>Bordetella pertussisb</td><td>Fresh</td><td>1400</td><td>1/1 (100%)</td><td>(20.7%, 100%)</td><td>1387/1399 (99.1%)</td><td>(98.5%, 99.5%)</td></tr><tr><td></td><td>Frozen</td><td>1247</td><td>1/1 (100%)</td><td>(20.7%, 100%)</td><td>1239/1246 (99.4%)</td><td>(98.8%, 99.7%)</td></tr><tr><td></td><td>Total</td><td>2647</td><td>2/2 (100%)</td><td>(34.2%, 100%)</td><td>2626/2645 (99.3%)</td><td>(98.9%, 99.5%)</td></tr><tr><td>Chlamydia pneumoniae</td><td>Fresh</td><td>1400</td><td>2/2 (100%)</td><td>(34.2%, 100%)</td><td>1397/1398 (99.9%)</td><td>(99.6%, 100%)</td></tr><tr><td></td><td>Frozen</td><td>1247</td><td>2/2 (100%)</td><td>(34.2%, 100%)</td><td>1245/1245 (100%)</td><td>(99.7%, 100%)</td></tr><tr><td></td><td>Total</td><td>2647</td><td>4/4 (100%)</td><td>(51.0%, 100%)</td><td>2642/2643 (100%)</td><td>(99.8%, 100%)</td></tr><tr><td>Coronavirusd</td><td>Fresh</td><td>1400</td><td>35/50 (70%)</td><td>(56.2%, 80.9%)</td><td>1338/1350 (99.1%)</td><td>(98.5%, 99.5%)</td></tr><tr><td></td><td>Frozen</td><td>1247</td><td>76/83 (91.6%)</td><td>(83.6%, 95.9%)</td><td>1154/1164 (99.1%)</td><td>(98.4%, 99.5%)</td></tr><tr><td></td><td>Total</td><td>2647</td><td>111/133 (83.5%)</td><td>(76.2%, 88.8%)</td><td>2492/2514 (99.1%)</td><td>(98.7%, 99.4%)</td></tr><tr><td>Human Metapneumoviruse</td><td>Fresh</td><td>1400</td><td>89/93 (95.7%)</td><td>(89.5%, 98.3%)</td><td>1299/1307 (99.4%)</td><td>(98.8%, 99.7%)</td></tr><tr><td></td><td>Frozen</td><td>1247</td><td>46/49 (93.9%)</td><td>(83.5%, 97.9%)</td><td>1189/1198 (99.2%)</td><td>(98.6%, 99.6%)</td></tr><tr><td></td><td>Total</td><td>2647</td><td>135/142 (95.1%)</td><td>(90.2%, 97.6%)</td><td>2488/2505 (99.3%)</td><td>(98.9%, 99.6%)</td></tr><tr><td>Human Rhinovirus/Enterovirusf</td><td>Fresh</td><td>1400</td><td>221/261 (84.7%)</td><td>(79.8%, 88.5%)</td><td>1119/1139 (98.2%)</td><td>(97.3%, 98.9%)</td></tr><tr><td></td><td>Frozen</td><td>1247</td><td>162/213 (76.1%)</td><td>(69.9%, 81.3%)</td><td>1020/1034 (98.6%)</td><td>(97.7%, 99.2%)</td></tr><tr><td></td><td>Total</td><td>2647</td><td>383/474 (80.8%)</td><td>(77.0%, 84.1%)</td><td>2139/2173 (98.4%)</td><td>(97.8%, 98.9%)</td></tr><tr><td>Influenza Ag</td><td>Fresh</td><td>1398</td><td>115/120 (95.8%)</td><td>(90.6%, 98.2%)</td><td>1265/1278 (99.0%)</td><td>(98.3%, 99.4%)</td></tr><tr><td></td><td>Frozen</td><td>1244</td><td>98/101 (97.0%)</td><td>(91.6%, 99.0%)</td><td>1131/1143 (99.0%)</td><td>(98.2%, 99.4%)</td></tr><tr><td></td><td>Total</td><td>2642</td><td>213/221 (96.4%)</td><td>(93.0%, 98.2%)</td><td>2396/2421 (99.0%)</td><td>(98.5%, 99.3%)</td></tr><tr><td>Influenza A H1</td><td>Fresh</td><td>1398</td><td>N/A+</td><td>N/A+</td><td>1398/1398 (100%)</td><td>(99.7%, 100%)</td></tr><tr><td></td><td>Frozen</td><td>1242</td><td>N/A†</td><td>N/A†</td><td>1242/1242 (100%)</td><td>(99.7%, 100%)</td></tr><tr><td></td><td>Total</td><td>2640</td><td>N/A†</td><td>N/A†</td><td>2640/2640 (100%)</td><td>(99.9%, 100%)</td></tr><tr><td>Influenza A H1 2009pdmh</td><td>Fresh</td><td>1398</td><td>29/30 (96.7%)</td><td>(83.3%, 99.4%)</td><td>1365/1368 (99.8%)</td><td>(99.4%, 99.9%)</td></tr><tr><td></td><td>Frozen</td><td>1242</td><td>23/23 (100%)</td><td>(85.7%, 100%)</td><td>1213/1219 (99.5%)</td><td>(98.9%, 99.8%)</td></tr><tr><td></td><td>Total</td><td>2640</td><td>52/53 (98.1%)</td><td>(90.1%, 99.7%)</td><td>2578/2587 (99.7%)</td><td>(99.3%, 99.8%)</td></tr></table>

Premarket Notification 510(k)   

<table><tr><td colspan="7">Positive Agreement Negative Agreement</td></tr><tr><td>Target</td><td>Storage</td><td>(n)</td><td>PA (%)</td><td>95% Cl</td><td>NA (%)</td><td>95% Cl</td></tr><tr><td>Influenza A H3i</td><td>Fresh</td><td>1398</td><td>82/88 (93.2%)</td><td>(85.9%, 96.8%)</td><td>1306/1310 (99.7%)</td><td>(99.2%, 99.9%)</td></tr><tr><td rowspan="5">Influenza Bi</td><td>Frozen</td><td>1242</td><td>65/69 (94.2%)</td><td>(86.0%, 97.7%)</td><td>1168/1173 (99.6%)</td><td>(99.0%, 99.8%)</td></tr><tr><td>Total</td><td>2640</td><td>147/157 (93.6%)</td><td>(88.7%, 96.5%)</td><td>2474/2483 (99.6%)</td><td>(99.3%, 99.8%)</td></tr><tr><td>Fresh</td><td>1400</td><td>7/7 (100%)</td><td>(64.6%, 100%)</td><td>1388/1393 (99.6%)</td><td>(99.2%, 99.8%)</td></tr><tr><td>Frozen</td><td>1247</td><td>44/47 (93.6%)</td><td>(82.8%, 97.8%)</td><td>1191/1200 (99.2%)</td><td>(98.6%, 99.6%)</td></tr><tr><td>Total</td><td>2647</td><td>51/54 (94.4%)</td><td>(84.9%, 98.1%)</td><td>2579/2593 (99.5%)</td><td>(99.1%, 99.7%)</td></tr><tr><td rowspan="3">Mycoplasma pneumoniaek</td><td>Fresh</td><td>1400</td><td>8/8 (100%)</td><td>(67.6%, 100%)</td><td>1381/1392 (99.2%)</td><td>(98.6%, 99.6%)</td></tr><tr><td>Frozen</td><td>1247</td><td>10/10 (100%)</td><td>(72.2%, 100%)</td><td>1228/1237 (99.3%)</td><td>(98.6%, 99.6%)</td></tr><tr><td>Total</td><td>2647</td><td>18/18 (100%)</td><td>(82.4%, 100%)</td><td>2609/2629 (99.2%)</td><td>(98.8%, 99.5%)</td></tr><tr><td>Parainfluenza Virus 1&#x27;</td><td>Fresh</td><td>1400</td><td>4/4 (100%)</td><td>(51.0%, 100%)</td><td>1396/1396 (100%)</td><td>(99.7%, 100%)</td></tr><tr><td rowspan="5">Parainfluenza Virus 2m</td><td>Frozen</td><td>1247</td><td>11/13 (84.6%)</td><td>(57.8%, 95.7%)</td><td>1234/1234 (100%)</td><td>(99.7%, 100%)</td></tr><tr><td>Total</td><td>2647</td><td>15/17 (88.2%)</td><td>(65.7%, 96.7%)</td><td>2630/2630 (100%)</td><td>(99.9%, 100%)</td></tr><tr><td>Fresh</td><td>1400</td><td>2/3 (66.7%)</td><td>(20.8%, 93.9%)</td><td>1396/1397 (99.9%)</td><td>(99.6%, 100%)</td></tr><tr><td>Frozen</td><td>1247</td><td>8/9 (88.9%)</td><td>(56.5%, 98.0%)</td><td>1236/1238 (99.8%)</td><td>(99.4%, 100%)</td></tr><tr><td>Total</td><td>2647</td><td>10/12 (83.3%)</td><td>(55.2%, 95.3%)</td><td>2632/2635 (99.9%)</td><td>(99.7%, 100%)</td></tr><tr><td>Parainfluenza Virus 3n</td><td>Fresh</td><td>1400</td><td>77/79 (97.5%)</td><td>(91.2%, 99.3%)</td><td>1312/1321 (99.3%)</td><td>(98.7%, 99.6%)</td></tr><tr><td></td><td>Frozen</td><td>1247</td><td>41/43 (95.3%)</td><td>(84.5%, 98.7%)</td><td>1196/1204 (99.3%)</td><td>(98.7%, 99.7%)</td></tr><tr><td></td><td>Total</td><td>2647</td><td>118/122 (96.7%)</td><td>(91.9%, 98.7%)</td><td>2508/2525 (99.3%)</td><td>(98.9%, 99.6%)</td></tr><tr><td rowspan="3">Parainfluenza Virus 4°</td><td>Fresh</td><td>1400</td><td>1/1 (100%)</td><td>(20.7%, 100%)</td><td>1399/1399 (100%)</td><td>(99.7%, 100%)</td></tr><tr><td>Frozen</td><td>1247</td><td>15/17 (88.2%)</td><td>(65.7%, 96.7%)</td><td>1228/1230 (99.8%)</td><td>(99.4%, 100%)</td></tr><tr><td>Total</td><td>2647</td><td>16/18 (88.9%)</td><td>(67.2%, 96.9%)</td><td>2627/2629 (99.9%)</td><td>(99.7%, 100%)</td></tr><tr><td rowspan="3">Respiratory Syncytial Virusp</td><td>Fresh</td><td>1400</td><td>91/93 (97.8%)</td><td>(92.5%, 99.4%)</td><td>1293/1307 (98.9%)</td><td>(98.2%, 99.4%)</td></tr><tr><td>Frozen</td><td>1247</td><td>109/111 (98.2%)</td><td>(93.7%, 99.5%)</td><td>1129/1136 (99.4%)</td><td>(98.7%, 99.7%)</td></tr><tr><td>Total</td><td>2647</td><td>200/204 (98.0%)</td><td>(95.1%, 99.2%)</td><td>2422/2443 (99.1%)</td><td>(98.7%, 99.4%)</td></tr></table>

† No positive reference results recorded

a – Adenovirus: The 10 FNs were not detected by PCR/bi-directional sequencing or alternative NAAT. Of 41 FPs, 37 were not detected and 4 were indeterminate by PCR/bi-directional sequencing.

b – Bordetella pertussis: Of ${ 1 9 } \mathsf { F P S } _ { \mathsf { \varepsilon } }$ , 4 were detected by PCR/bi-directional sequencing, 3 were indeterminate and 12 were not detected by PCR/bi-directional sequencing.

c – Chlamydia pneumoniae: The 1 FP was detected by PCR/bi-directional sequencing.

d – Coronavirus: Of 22 FNs, 5 were detected by PCR/bi-directional sequencing. 2 were not detected by PCR/bi-directional sequencing but detected by alternative NAAT. 12 were not detected by either alternative NAAT or PCR/bi-directional sequencing. 3 were not detected by PCR/bi-directional sequencing and were not tested by alternative NAAT. The ${ 2 2 } \mathsf { F P S }$ were not detected by PCR/bi-directional sequencing.

e – Human Metapneumovirus: Of 7 FNs, 3 were detected and 4 were not detected by PCR/bi-directional sequencing. Of 17 FPs, 7 were detected while 10 were not detected by PCR/bi-directional sequencing.

f – Human Rhinovirus/Enterovirus: Of 91 FNs, 26 were detected, and 2 were indeterminate by PCR/bi-directional sequencing. 25 were not detected by PCR/bi-directional sequencing but were detected by alternative NAAT. 11 were not detected by either PCR/bi-directional

sequencing or by alternative NAAT. 27 were not detected by PCR/bi-directional sequencing and had insufficient volume for alternative NAAT.   
Of 34 FPs, 8 were detected and 26 were not detected by PCR/bi-directional sequencing.

g – Influenza A: Of the 8 FNs, 7 were not detected by PCR/bi-directional sequencing. 1 had insufficient volume for follow-up testing. Of $2 5 \mathsf { F P S } ,$ 5 were detected by PCR/bi-directional sequencing, 19 were not detected by PCR/bi-directional sequencing, and 1 had insufficient volume for follow-up testing.

h – Influenza A H1 2009pdm: The 1 FN had insufficient volume for follow-up testing. Of 9 FPs, 4 were detected by PCR/bi-directional sequencing, and 5 were not detected by PCR/bi-directional sequencing.

i – Influenza A H3: Of 10 FNs, 7 were not detected and 1 was detected by PCR/bi-directional sequencing. 2 had insufficient volume for followup testing. Of 9 FPs, 3 were detected, 1 was indeterminate, and 4 were not detected by PCR/bi-directional sequencing. 1 had insufficient volume for follow-up testing.

j – Influenza B: The 3 FNs were not detected by PCR/bi-directional sequencing. Of 14 FPs, 1 was detected and 12 were not detected by PCR/bi directional sequencing, and 1 had insufficient volume for follow-up testing.

k – Mycoplasma pneumoniae: Of 20 FPs, 7 were detected, 10 were not detected, and 1 was invalid by PCR/bi-directional sequencing. 2 had insufficient volume for follow-up testing.

– Parainfluenza Virus 1: The 2 FNs were not detected by PCR/bi-directional sequencing.

m – Parainfluenza Virus 2: Of 2 FNs, 1 was detected and 1 was not detected by PCR/bi-directional sequencing. The 3 FPs were detected by PCR/bi-directional sequencing.

n – Parainfluenza Virus 3: Of 4 FNs, 2 were detected, 1 was not detected, and 1 was indeterminate, by PCR/bi-directional sequencing. Of 17 FPs, 8 were detected, 8 were not detected, and 1 was indeterminate, by PCR/bi-directional sequencing.

o – Parainfluenza Virus 4: The 2 FNs were detected by PCR/bi-directional sequencing. The 2 FPs were detected by PCR/bi-directional sequencing.

p – Respiratory Syncytial Virus: The 4 FNs were not detected by PCR/bi-directional sequencing. Of 21 FPs, 18 were not detected, 2 were detected, and 1 was indeterminate, by PCR/bi-directional sequencing.

Prospective prevalence as detected by BioCode® RPP, stratified by age or study site, are presented in tables below.

Table. Prospective prevalence detected by BioCode® RPP stratified by Age   

<table><tr><td>Analyte</td><td>Overall (N=2647)</td><td>≤5 yrs (N=1004)</td><td>6-21 yrs (N=609)</td><td>22-59 yrs (N=531)</td><td>60+ yrs (N=503)</td></tr><tr><td>Adenovirus</td><td>109 (4.1%)</td><td>80 (8.0%)</td><td>20 (3.3%)</td><td>5 (0.9%)</td><td>4 (0.8%)</td></tr><tr><td>Bordetella pertussis</td><td>21 (0.8%)</td><td>8 (0.8%)</td><td>12 (2.0%)</td><td>0 (0.0%)</td><td>1 (0.2%)</td></tr><tr><td>Chlamydia pneumoniae</td><td>5 (0.2%)</td><td>1 (0.1%)</td><td>3 (0.5%)</td><td>1 (0.2%)</td><td>0 (0.0%)</td></tr><tr><td>Coronavirus</td><td>133 (5.0%)</td><td>63 (6.3%)</td><td>27 (4.4%)</td><td>19 (3.6%)</td><td>24 (4.8%)</td></tr><tr><td>Human Metapneumovirus</td><td>152 (5.7%)</td><td>94 (9.4%)</td><td>26 (4.3%)</td><td>15 (2.8%)</td><td>17 (3.4%)</td></tr><tr><td>Human Rhinovirus/Enterovirus</td><td>417 (15.8%)</td><td>234 (23.3%)</td><td>101 (16.6%)</td><td>51 (9.6%)</td><td>31 (6.2%)</td></tr><tr><td>Influenza A</td><td>238 (9.0%)</td><td>71 (7.1%)</td><td>84 (13.8%)</td><td>47 (8.9%)</td><td>36 (7.2%)</td></tr><tr><td>Influenza A H1</td><td>0 (0.0%)</td><td>0 (0.0%)</td><td>0 (0.0%)</td><td>0 (0.0%)</td><td>0 (0.0%)</td></tr><tr><td>Influenza A H1 2009pdm</td><td>62 (2.3%)</td><td>24 (2.4%)</td><td>15 (2.5%)</td><td>15 (2.8%)</td><td>8 (1.6%)</td></tr><tr><td>Influenza A H3</td><td>157 (5.9%)</td><td>45 (4.5%)</td><td>61 (10.0%)</td><td>26 (4.9%)</td><td>25 (5.0%)</td></tr><tr><td>Influenza B</td><td>65 (2.5%)</td><td>13 (1.3%)</td><td>26 (4.3%)</td><td>15 (2.8%)</td><td>11 (2.2%)</td></tr><tr><td>Mycoplasma pneumoniae</td><td>38 (1.4%)</td><td>9 (0.9%)</td><td>21 (3.4%)</td><td>6 (1.1%)</td><td>2 (0.4%)</td></tr><tr><td>Parainfluenza Virus 1</td><td>15 (0.6%)</td><td>4 (0.4%)</td><td>1 (0.2%)</td><td>4 (0.8%)</td><td>6 (1.2%)</td></tr><tr><td>Parainfluenza Virus 2</td><td>13 (0.5%)</td><td>6 (0.6%)</td><td>4 (0.7%)</td><td>3 (0.6%)</td><td>0 (0.0%)</td></tr><tr><td>Parainfluenza Virus 3</td><td>135 (5.1%)</td><td>74 (7.4%)</td><td>21 (3.4%)</td><td>21 (4.0%)</td><td>19 (3.8%)</td></tr><tr><td>Parainfluenza Virus 4</td><td>18 (0.7%)</td><td>12 (1.2%)</td><td>1 (0.2%)</td><td>2 (0.4%)</td><td>3 (0.6%)</td></tr><tr><td>Respiratory Syncytial Virus</td><td>221 (8.3%)</td><td>156 (15.5%)</td><td>30 (4.9%)</td><td>19 (3.6%)</td><td>16 (3.2%)</td></tr></table>

Table. Prospective prevalence detected by BioCode® RPP stratified by site.   

<table><tr><td>Analyte</td><td>Overall (N=2647)</td><td>Site 1 (N=529)</td><td>Site 2 (N=419)</td><td>Site 3 (N=599)</td><td>Site 4 (N=550)</td><td>Site 5 (N=550)</td></tr><tr><td>Adenovirus</td><td>109 (4.1%)</td><td>8 (1.5%)</td><td>27 (6.4%)</td><td>21 (3.5%)</td><td>17 (3.1%)</td><td>36 (6.5%)</td></tr><tr><td>Bordetella pertussis</td><td>21 (0.8%)</td><td>0 (0.0%)</td><td>19 (4.5%)</td><td>1 (0.2%)</td><td>0 (0.0%)</td><td>1 (0.2%)</td></tr><tr><td>Chlamydia pneumoniae</td><td>5 (0.2%)</td><td>0 (0.0%)</td><td>3 (0.7%)</td><td>0 (0.0%)</td><td>1 (0.2%)</td><td>1 (0.2%)</td></tr><tr><td>Coronavirus</td><td>133 (5.0%)</td><td>36 (6.8%)</td><td>21 (5.0%)</td><td>14 (2.3%)</td><td>26 (4.7%)</td><td>36 (6.5%)</td></tr><tr><td>Human Metapneumovirus</td><td>152 (5.7%)</td><td>18 (3.4%)</td><td>22 (5.3%)</td><td>26 (4.3%)</td><td>38 (6.9%)</td><td>48 (8.7%)</td></tr><tr><td>Human Rhinovirus/Enterovirus</td><td>417 (15.8%)</td><td>48 (9.1%)</td><td>83 (19.8%)</td><td>65 (10.9%)</td><td>125 (22.7%)</td><td>96 (17.5%)</td></tr><tr><td>Influenza A</td><td>238 (9.0%)</td><td>49 (9.3%)</td><td>50 (11.9%)</td><td>35 (5.8%)</td><td>41 (7.5%)</td><td>63 (11.5%)</td></tr><tr><td>Influenza A H1</td><td>0 (0.0%)</td><td>0 (0.0%)</td><td>0 (0.0%)</td><td>0 (0.0%)</td><td>0 (0.0%)</td><td>0 (0.0%)</td></tr><tr><td>Influenza A H1 2009pdm</td><td>62 (2.3%)</td><td>13 (2.5%)</td><td>7 (1.7%)</td><td>11 (1.8%)</td><td>22 (4.0%)</td><td>9 (1.6%)</td></tr><tr><td>Influenza A H3</td><td>157 (5.9%)</td><td>32 (6.0%)</td><td>40 (9.5%)</td><td>19 (3.2%)</td><td>17 (3.1%)</td><td>49 (8.9%)</td></tr><tr><td>Influenza B</td><td>65 (2.5%)</td><td>12 (2.3%)</td><td>32 (7.6%)</td><td>15 (2.5%)</td><td>2 (0.4%)</td><td>4 (0.7%)</td></tr><tr><td>Mycoplasma pneumoniae</td><td>38 (1.4%)</td><td>4 (0.8%)</td><td>15 (3.6%)</td><td>5 (0.8%)</td><td>3 (0.5%)</td><td>11 (2.0%)</td></tr><tr><td>Parainfluenza Virus 1</td><td>15 (0.6%)</td><td>4 (0.8%)</td><td>0 (0.0%)</td><td>11 (1.8%)</td><td>0 (0.0%)</td><td>0 (0.0%)</td></tr><tr><td>Parainfluenza Virus 2</td><td>13 (0.5%)</td><td>0 (0.0%)</td><td>0 (0.0%)</td><td>3 (0.5%)</td><td>8 (1.5%)</td><td>2 (0.4%)</td></tr><tr><td>Parainfluenza Virus 3</td><td>135 (5.1%)</td><td>12 (2.3%)</td><td>28 (6.7%)</td><td>41 (6.8%)</td><td>9 (1.6%)</td><td>45 (8.2%)</td></tr><tr><td>Parainfluenza Virus 4</td><td>18 (0.7%)</td><td>0 (0.0%)</td><td>1 (0.2%)</td><td>9 (1.5%)</td><td>5 (0.9%)</td><td>3 (0.5%)</td></tr><tr><td>Respiratory Syncytial Virus</td><td>221 (8.3%)</td><td>21 (4.0%)</td><td>35 (8.4%)</td><td>25 (4.2%)</td><td>49 (8.9%)</td><td>91 (16.5%)</td></tr></table>

The overall success rate for initial specimen testing in the prospective study was $9 8 . 8 \%$ (2618/2649) $9 5 \%$ CI: $9 8 . 3 \% - 9 9 . 2 \%$ ; 31 tests were unsuccessful (26 tests with an invalid result and 5 tests due to low BMB count/instrument error). Upon a single retest per the instructions for use, 29 of the 31 initially unsuccessful specimens generated a valid result. The final validity rate was $9 9 . 9 \%$ (2647/2649) $9 5 \%$ CI: $9 9 . 7 \% - 1 0 0 \%$ ).

There were 193 samples with mixed infections by the BioCode® RPP in the prospective clinical study (193/2649 or $7 . 3 \%$ ). The distribution, prevalence and most common co-infections combinations detected by BioCode RPP in the prospective clinical study are summarized in the tables below.

Table. Distribution of co-infection combinations detected by BioCode® RPP from prospective clinical study   

<table><tr><td>Analytes Detected Simultaneously</td><td>Number of Specimen</td></tr><tr><td>2</td><td>168 (87.0%)</td></tr><tr><td>3</td><td>24 (12.4%)</td></tr><tr><td>5</td><td>1 (0.5%)</td></tr><tr><td>Total Co-Infections</td><td>193</td></tr></table>

Table. Prevalence of targets in co-infection combinations detected by BioCode® RPP from prospective clinical study

<table><tr><td>Analyte</td><td>Prevalence in Co-Infections (N=193)</td></tr><tr><td>Adenovirus</td><td>51 (26.4%)</td></tr><tr><td>Bordetella pertussis</td><td>14 (7.3%)</td></tr><tr><td>Chlamydia pneumoniae</td><td>1 (0.5%)</td></tr><tr><td>Coronavirus</td><td>47 (24.4%)</td></tr><tr><td>Human Metapneumovirus</td><td>35 (18.1%)</td></tr><tr><td>Human Rhinovirus/Enterovirus</td><td>101 (52.3%)</td></tr><tr><td>Influenza A</td><td>37 (19.2%)</td></tr><tr><td>Influenza B</td><td>8 (4.1%)</td></tr><tr><td>Mycoplasma pneumoniae</td><td>7 (3.6%)</td></tr><tr><td>Parainfluenza Virus 1</td><td>2 (1.0%)</td></tr><tr><td>Parainfluenza Virus 2</td><td>4 (2.1%)</td></tr><tr><td>Parainfluenza Virus 3</td><td>39 (20.2%)</td></tr><tr><td>Parainfluenza Virus 4</td><td>8 (4.1%)</td></tr><tr><td>Respiratory Syncytial Virus</td><td></td></tr><tr><td></td><td>59 (30.1%)</td></tr><tr><td></td><td></td></tr></table>

Table. Most prevalent multiple detection combinations (5 or more instances) detected by BioCode® RPP from prospective clinical study.

<table><tr><td>Co-Infection Combination</td><td>Number of Specimen</td></tr><tr><td>Human Rhinovirus/Enterovirus + Respiratory Syncytial Virus</td><td>17</td></tr><tr><td>Adenovirus + Human Rhinovirus/Enterovirus</td><td>13</td></tr><tr><td>Human Metapneumovirus + Human Rhinovirus/Enterovirus</td><td>13</td></tr><tr><td>Human Rhinovirus/Enterovirus + Influenza A</td><td>11</td></tr><tr><td>Adenovirus + Respiratory Syncytial Virus</td><td>10</td></tr><tr><td>Human Rhinovirus/Enterovirus + Parainfluenza Virus 3</td><td>10</td></tr><tr><td>Coronavirus + Human Rhinovirus/Enterovirus</td><td>8</td></tr><tr><td>Coronavirus + Respiratory Syncytial Virus</td><td>7</td></tr><tr><td>Bordetella pertussis + Human Rhinovirus/Enterovirus</td><td>6</td></tr></table>

Premarket Notification 510(k)   

<table><tr><td>Co-Infection Combination</td><td>Number of Specimen</td></tr><tr><td>Adenovirus + Parainfluenza Virus 3</td><td>5</td></tr><tr><td>Coronavirus + Human Metapneumovirus</td><td>5</td></tr><tr><td>Coronavirus + Influenza A</td><td>5</td></tr></table>

# Testing of Preselected Archived Specimens

Some of the pathogens on the BioCode RPP were of low prevalence and were not encountered in sufficiently large numbers during the prospective study to adequately demonstrate system performance. To supplement the results of the prospective clinical study, an evaluation of preselected archived retrospective specimens was performed. These specimens were archived NPS in VTM or UTM specimens that were selected because they had previously tested positive for one of the following pathogens at the source laboratory: coronavirus 229E, coronavirus HKU1, parainfluenza virus 1, parainfluenza virus 2, parainfluenza virus 4, Bordetella pertussis, Chlamydia pneumoniae, and Mycoplasma pneumoniae, or had been negative in previous laboratory testing.

A total of 165 clinical specimens were enrolled for testing in this retrospective study. The specimens were randomized such that the users performing the BioCode RPP assay were blinded to the expected test result and shipped to one of two of the testing sites participated in the prospective clinical study for testing.

A summary of the demographic information of the tested samples is provided in the following table:

Demographics of Archived Specimens   

<table><tr><td>Characteristic</td><td>Archived Study Specimens</td></tr><tr><td>Total Specimens</td><td>165</td></tr><tr><td>Gender (n/N (%))</td><td></td></tr><tr><td>Male</td><td>96/165 (58.2%)</td></tr><tr><td>Female</td><td>69/165 (41.8%)</td></tr><tr><td>Age Category (n/N (%))</td><td></td></tr><tr><td>0-5 yrs</td><td>77/165 (46.7%)</td></tr><tr><td>6-21 yrs</td><td>58/165 (35.2%)</td></tr><tr><td>22-59 yrs</td><td>15/165 (9.1%)</td></tr><tr><td>60+ yrs</td><td>15/165 (9.1%)</td></tr></table>

The performance of the BioCode RPP was evaluated by comparing the BioCode RPP test results with those from an FDA-cleared molecular multiplexed respiratory pathogen panel, the same panel test as the one used as the comparator in the prospective clinical study. The BioCode RPP retrospective performance data expressed as positive percent and negative percent agreements against the comparator method are presented by pathogen in the table below.

Table. Results from Archived Specimens tested by the BioCode® RPP with easyMAG extraction system.   

<table><tr><td>Target</td><td>(n)</td><td colspan="2">Positive Agreement</td><td colspan="2">Negative Agreement</td></tr><tr><td></td><td></td><td>PA (%)</td><td>95% Cl</td><td>NA (%)</td><td>95% Cl</td></tr><tr><td>Adenovirusa</td><td>165</td><td>7/7 (100%)</td><td>(64.6%, 100%)</td><td>155/158 (98.1%)</td><td>(94.6%, 99.4%)</td></tr><tr><td>Bordetella pertussisb</td><td>165</td><td>10/10 (100%)</td><td>(72.2%, 100%)</td><td>144/155 (92.9%)</td><td>(87.7%, 96.0%)</td></tr><tr><td>Chlamydia pneumoniae</td><td>165</td><td>10/10 (100%)</td><td>(72.2%, 100%)</td><td>155/155 (100%)</td><td>(97.6%, 100%)</td></tr><tr><td>Coronavirusc</td><td>165</td><td>52/59 (88.1%)</td><td>(77.5%, 94.1%)</td><td>99/106 (93.4%)</td><td>(87.0%, 96.8%)</td></tr><tr><td>Human Metapneumovirus</td><td>165</td><td>4/4 (100%)</td><td>(51.0%, 100%)</td><td>161/161 (100%)</td><td>(97.7%, 100%)</td></tr><tr><td>Human Rhinovirus/Enterovirusd</td><td>165</td><td>16/23 (69.6%)</td><td>(49.1%, 84.4%)</td><td>141/142 (99.3%)</td><td>(96.1%, 99.9%)</td></tr><tr><td>Influenza A</td><td>165</td><td>N/A+</td><td>N/A+</td><td>165/165 (100%)</td><td>(97.7%, 100%)</td></tr><tr><td>Influenza A H1</td><td>165</td><td>N/A†</td><td>N/A+</td><td>165/165 (100%)</td><td>(97.7%, 100%)</td></tr><tr><td>Influenza A H1 2009pdm</td><td>165</td><td>N/A†</td><td>N/A†</td><td>165/165 (100%)</td><td>(97.7%, 100%)</td></tr><tr><td>Influenza A H3</td><td>165</td><td>N/A+</td><td>N/A+</td><td>165/165 (100%)</td><td>(97.7%, 100%)</td></tr><tr><td>Influenza Be</td><td>165</td><td>2/3 (66.7%)</td><td>(20.8%, 93.9%)</td><td>162/162 (100%)</td><td>(97.7%, 100%)</td></tr><tr><td>Mycoplasma pneumoniaef</td><td>165</td><td>7/7 (100%)</td><td>(64.6%, 100%)</td><td>153/158 (96.8%)</td><td>(92.8%, 98.6%)</td></tr><tr><td>Parainfluenza Virus 18</td><td>165</td><td>12/13 (92.3%)</td><td>(66.7%, 98.6%)</td><td>152/152 (100%)</td><td>(97.5%, 100%)</td></tr><tr><td>Parainfluenza Virus 2h</td><td>165</td><td>19/20 (95%)</td><td>(76.4%, 99.1%)</td><td>144/145 (99.3%)</td><td>(96.2%, 99.9%)</td></tr><tr><td>Parainfluenza Virus 3</td><td>165</td><td>1/1 (100%)</td><td>(20.7%, 100%)</td><td>164/164 (100%)</td><td>(97.7%, 100%)</td></tr><tr><td>Parainfluenza Virus 4</td><td>165</td><td>14/15 (93.3%)</td><td>(70.2%, 98.8%)</td><td>150/150 (100%)</td><td>(97.5%, 100%)</td></tr><tr><td>Respiratory Syncytial Virusi</td><td>165</td><td>11/12 (91.7%)</td><td>(64.6%, 98.5%)</td><td>152/153 (99.3%)</td><td>(96.4%, 99.9%)</td></tr></table>

† No positive reference results recorded

a - Adenovirus: Of 3 FPs, 1 was detected by PCR/bi-directional sequencing and 2 were not detected by PCR/bi-directional sequencing.

b - Bordetella pertussis: Of 11 FPs, 7 were detected by PCR/bi-directional sequencing and 4 were not detected by PCR/bidirectional sequencing.

c - Coronavirus: Of 7 FNs, 4 were detected by PCR/bi-directional sequencing while 3 were not detected by PCR/bi-directional sequencing. The 7 FPs were not detected by PCR/bi-directional sequencing. All had low MFIs (<620) near the MFI cut-off on initial BioCode RPP results.

d - Human Rhinovirus/Enterovirus: The 7 FNs were not detected by PCR/bi-directional sequencing. The 1 FP was not detected by PCR/bi-directional sequencing.

e - Influenza B: 1 FN was not detected by PCR/bi-directional sequencing.

f - Mycoplasma pneumoniae: Of 5 FPs, 3 were detected by PCR/bi-directional sequencing and 2 were not detected by PCR/bi-directional sequencing.

g - Parainfluenza Virus 1: 1 FN was not detected by PCR/bi-directional sequencing.

h - Parainfluenza Virus 2: 1 FN was not detected by PCR/bi-directional sequencing. 1 FP was not detected by PCR/bidirectional sequencing.

i - Parainfluenza Virus 4: 1 FN was not detected by PCR/bi-directional sequencing.

j - Respiratory Syncytial Virus: 1 FN was not detected by PCR/bi-directional sequencing. 1 FP was not detected by PCR/bidirectional sequencing.

# Testing of Contrived Specimens

Some analytes are so rare that both prospective and archived specimen collection efforts were insufficient to demonstrate the clinical performance. To supplement the prospective and archived data, an evaluation of contrived specimens was performed for two pathogens: Chlamydia pneumoniae and Influenza A H1. These contrived clinical specimens were prepared using 50 unique natural NPS in VTM or UTM specimens that were previously tested negative for all BioCode RPP analytes. Contrived specimens were spiked at concentrations of 2X LOD or greater using different strains for each pathogen. The 50 positive samples of each pathogen were prepared, interspersed with negative samples and randomized before testing at one of the five testing sites participated in the prospective clinical study. A total of 110 samples, including 100 positives, were tested. The results of the BioCode RPP testing are presented in the following table:

Table. Results from contrived samples tested by the BioCode® RPP using the easyMAG extraction system.   

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Source</td><td rowspan=1 colspan=1>Strain/Isolate</td><td rowspan=1 colspan=1>FoldLoD</td><td rowspan=1 colspan=1>Concentration</td><td rowspan=1 colspan=1>PA (%)</td><td rowspan=1 colspan=1>95% Cl</td><td rowspan=1 colspan=1>NA (%)</td><td rowspan=1 colspan=1>95% Cl</td></tr><tr><td rowspan=10 colspan=1>Chlamydiapneumoniae</td><td rowspan=3 colspan=1>ATCC 53592</td><td rowspan=3 colspan=1>AR-39</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>33.4 CFU/mL</td><td rowspan=1 colspan=1>9/9 (100%)</td><td rowspan=1 colspan=1>70.1%, 100%</td><td rowspan=10 colspan=1>60/60(100%)</td><td rowspan=10 colspan=1>94.0%,100%</td></tr><tr><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>167 CFU/mL</td><td rowspan=1 colspan=1>5/5 (100%)</td><td rowspan=1 colspan=1>56.6%, 100%</td></tr><tr><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>1670 CFU/mL</td><td rowspan=1 colspan=1>4/4 (100%)</td><td rowspan=1 colspan=1>51.0%, 100%</td></tr><tr><td rowspan=3 colspan=1>ATCC VR-1360</td><td rowspan=3 colspan=1>CM-1</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>33.4 CFU/mL</td><td rowspan=1 colspan=1>8/8 (100%)</td><td rowspan=1 colspan=1>67.6%, 100%</td></tr><tr><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>167 CFU/mL</td><td rowspan=1 colspan=1>5/5 (100%)</td><td rowspan=1 colspan=1>56.6%, 100%</td></tr><tr><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>1670 CFU/mL</td><td rowspan=1 colspan=1>3/3 (100%)</td><td rowspan=1 colspan=1>43.9%, 100%</td></tr><tr><td rowspan=3 colspan=1>ATCC VR-1310</td><td rowspan=3 colspan=1>CWL-029</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>33.4 CFU/mL</td><td rowspan=1 colspan=1>8/8 (100%)</td><td rowspan=1 colspan=1>67.6%, 100%</td></tr><tr><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>167 CFU/mL</td><td rowspan=1 colspan=1>5/5 (100%)</td><td rowspan=1 colspan=1>56.6%, 100%</td></tr><tr><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>1670 CFU/mL</td><td rowspan=1 colspan=1>3/3 (100%)</td><td rowspan=1 colspan=1>43.9%, 100%</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=3>Combined</td><td rowspan=1 colspan=1>50/50 (100%)</td><td rowspan=1 colspan=1>92.9%, 100%</td></tr><tr><td rowspan=16 colspan=1>Influenza AH1N1</td><td rowspan=3 colspan=1>Zeptometrix0810036CF</td><td rowspan=3 colspan=1>A/NewCaledonia/20/99</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>30 TCID5o/ mL</td><td rowspan=1 colspan=1>5/5 (100%)</td><td rowspan=1 colspan=1>56.6%, 100%</td><td rowspan=16 colspan=1>60/60(10%)</td><td rowspan=16 colspan=1>94.0%,100%</td></tr><tr><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>150 TCID5o/ mL</td><td rowspan=1 colspan=1>3/3 (100%)</td><td rowspan=1 colspan=1>43.9%, 100%</td></tr><tr><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>1500 TCID5o/ mL</td><td rowspan=1 colspan=1>3/3 (100%)</td><td rowspan=1 colspan=1>43.9%, 100%</td></tr><tr><td rowspan=3 colspan=1>Zeptometrix0810247CF</td><td rowspan=3 colspan=1>A/Taiwan/42/06</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>30 TCID50/ mL</td><td rowspan=1 colspan=1>5/5 (100%)</td><td rowspan=1 colspan=1>56.6%, 100%</td></tr><tr><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>150 TCID5o/ mL</td><td rowspan=1 colspan=1>3/3 (100%)</td><td rowspan=1 colspan=1>43.9%, 100%</td></tr><tr><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>1500 TCID50/ mL</td><td rowspan=1 colspan=1>2/2 (100%)</td><td rowspan=1 colspan=1>34.2%, 100%</td></tr><tr><td rowspan=3 colspan=1>Zeptometrix0810246CF</td><td rowspan=3 colspan=1>Singapore/63/04</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>30 TCID50/ mL</td><td rowspan=1 colspan=1>5/5 (100%)</td><td rowspan=1 colspan=1>56.6%, 100%</td></tr><tr><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>150 TCID5o/ mL</td><td rowspan=1 colspan=1>2/2 (100%)</td><td rowspan=1 colspan=1>34.2%, 100%</td></tr><tr><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>1500 TCID5o/ mL</td><td rowspan=1 colspan=1>2/2 (100%)</td><td rowspan=1 colspan=1>34.2%, 100%</td></tr><tr><td rowspan=3 colspan=1>Virapur</td><td rowspan=3 colspan=1>A/Denver/1/1957</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>30 TCID50/ mL</td><td rowspan=1 colspan=1>5/5 (100%)</td><td rowspan=1 colspan=1>56.6%, 100%</td></tr><tr><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>150 TCID50/ mL</td><td rowspan=1 colspan=1>2/2 (100%)</td><td rowspan=1 colspan=1>34.2%, 100%</td></tr><tr><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>1500 TCID5o/ mL</td><td rowspan=1 colspan=1>2/2 (100%)</td><td rowspan=1 colspan=1>34.2%, 100%</td></tr><tr><td rowspan=3 colspan=1>ATCC VR-219</td><td rowspan=3 colspan=1>A/NWS/33</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>54 TCID50/ mL</td><td rowspan=1 colspan=1>5/5 (100%)</td><td rowspan=1 colspan=1>56.6%, 100%</td></tr><tr><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>270 TCID50/ mL</td><td rowspan=1 colspan=1>3/3 (100%)</td><td rowspan=1 colspan=1>43.9%, 100%</td></tr><tr><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>2700 TCID5o/ mL</td><td rowspan=1 colspan=1>3/3 (100%)</td><td rowspan=1 colspan=1>43.9%, 100%</td></tr><tr><td rowspan=1 colspan=4>Combined</td><td rowspan=1 colspan=1>50/50 (100%)</td><td rowspan=1 colspan=1>92.9%, 100%</td></tr></table>

# Performance of External Controls during Clinical Trials

During clinical evaluation of the BioCode® RPP, at least one negative control (NC) was included in each run. The negative control was UTM that was taken through all steps (extraction, amplification, and detection). It was recommended that external controls consisting of 5 pools of inactivated organism from ZeptoMetrix (catalog no. NATRPP-ABC) be assayed on a rotating basis.

Table. Recommended positive control pools formulation for inactivated organism from ZeptoMetrix.   

<table><tr><td rowspan=1 colspan=1>Pool</td><td rowspan=1 colspan=1>Panel Member</td><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>Dilution Factor</td></tr><tr><td rowspan=4 colspan=1>PC-A</td><td rowspan=1 colspan=1>Influenza B</td><td rowspan=1 colspan=1>B/Florida/02/06</td><td rowspan=1 colspan=1>1/4</td></tr><tr><td rowspan=1 colspan=1>Respiratory Syncytial Virus A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>1/4</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza virus Type 1</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>1/4</td></tr><tr><td rowspan=1 colspan=1>Rhinovirus 1A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>1/4</td></tr><tr><td rowspan=1 colspan=1></td><td></td><td></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=4 colspan=1>PC-B</td><td rowspan=1 colspan=1>HKU1 Construct</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>1/4</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza virus Type 2</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>1/4</td></tr><tr><td rowspan=1 colspan=1>Metapneumovirus 8</td><td rowspan=1 colspan=1>Peru6-2003</td><td rowspan=1 colspan=1>1/4</td></tr><tr><td rowspan=1 colspan=1>Bordetella pertussis</td><td rowspan=1 colspan=1>A639</td><td rowspan=1 colspan=1>1/4</td></tr><tr><td rowspan=1 colspan=1></td><td></td><td></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=3 colspan=1>PC-C</td><td rowspan=1 colspan=1>Influenza A H1N1</td><td rowspan=1 colspan=1>A/New Caledonia/20/99</td><td rowspan=1 colspan=1>1/4</td></tr><tr><td rowspan=1 colspan=1>Coronavirus NL63</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>1/4</td></tr><tr><td rowspan=1 colspan=1>Adenovirus Type 3</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>1/4</td></tr><tr><td rowspan=1 colspan=1></td><td></td><td></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=4 colspan=1>PC-D</td><td rowspan=1 colspan=1>Influenza A 2009 H1N1pdm</td><td rowspan=1 colspan=1>A/NY/02/09</td><td rowspan=1 colspan=1>1/4</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza virus Type 4</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>1/4</td></tr><tr><td rowspan=1 colspan=1>Coronavirus OC43</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>1/4</td></tr><tr><td rowspan=1 colspan=1>Chlamydia pneumoniae</td><td rowspan=1 colspan=1>CWL-029</td><td rowspan=1 colspan=1>1/4</td></tr><tr><td rowspan=1 colspan=1></td><td></td><td></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=4 colspan=1>PC-E</td><td rowspan=1 colspan=1>Influenza A H3</td><td rowspan=1 colspan=1>A/Brisbane/10/07</td><td rowspan=1 colspan=1>1/4</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza virus Type 3</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>1/4</td></tr><tr><td rowspan=1 colspan=1>Coronavirus 229E</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>1/4</td></tr><tr><td rowspan=1 colspan=1>Mycoplasma pneumoniae</td><td rowspan=1 colspan=1>M129</td><td rowspan=1 colspan=1>1/4</td></tr></table>

Table. Performance of controls during Clinical trials   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>PC-A</td><td rowspan=1 colspan=1>PC-B</td><td rowspan=1 colspan=1>PC-C</td><td rowspan=1 colspan=1>PC-D</td><td rowspan=1 colspan=1>PC-E</td><td rowspan=1 colspan=1>NC</td></tr><tr><td rowspan=1 colspan=1>easyMAG</td><td rowspan=1 colspan=1>9a3/10</td><td rowspan=1 colspan=1>8a/9</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>11/11</td><td rowspan=1 colspan=1>8/8</td><td rowspan=1 colspan=1>50b/53</td></tr><tr><td rowspan=1 colspan=1>MP96</td><td rowspan=1 colspan=1>4/4</td><td rowspan=1 colspan=1>5/5</td><td rowspan=1 colspan=1>4/4</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>20/20</td></tr></table>

a - Operator error NC/PC switched during loading b - Of 3 failed NCs, 1 was due to RNA IC failure, 1 was due to operator error (NC/PC switched on loading), 1 was due to detection of unexpected target (FP).

# General Performance of Assay During Clinical Trials

Table. Accounting of valid and invalid runs during clinical trials (prospective specimens)

<table><tr><td rowspan=1 colspan=1>Description</td><td rowspan=1 colspan=1>Number</td><td rowspan=1 colspan=1>% of Total</td></tr><tr><td rowspan=1 colspan=1>Valid runs with complete results</td><td rowspan=1 colspan=1>57</td><td rowspan=1 colspan=1>86.4%</td></tr><tr><td rowspan=1 colspan=1>Completed Runs with NC failures</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>4.5%</td></tr><tr><td rowspan=1 colspan=1>Partially or completely invalid runs due to other failures</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>9.1%</td></tr><tr><td rowspan=1 colspan=1>Incomplete runs due to instrument failures</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>66</td><td rowspan=1 colspan=1>100%</td></tr></table>

Table. Summary of issues causing Invalid runs during clinical trials (prospective specimens)

<table><tr><td rowspan=1 colspan=1>Reason for failure</td><td rowspan=1 colspan=1>Number</td><td rowspan=1 colspan=1>% of Total</td></tr><tr><td rowspan=1 colspan=1>User Error</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>44.4%</td></tr><tr><td rowspan=1 colspan=1>Instrument/ Alignmenta</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>22.2%</td></tr><tr><td rowspan=1 colspan=1>Negative controlb</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>33.3%</td></tr><tr><td rowspan=1 colspan=1>Total invalid runs</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>100%</td></tr></table>

a – 2 consecutive failures for alignment error that did not happen again after adjustment. b - Of 3 failed NCs, 1 was due to RNA IC failure, 1 was due to operator error (NC and PC switched on loading), 1 was due to detection of unexpected target (FP).

# Analytical Performance

The results of the analytical studies summarized in the following paragraphs met the acceptance criteria and demonstrated the analytical performance characteristics of the proposed BioCode RPP.

# Reproducibility Study

A study was performed to assess the Reproducibility of the BioCode® RPP on the BioCode® MDx-3000. Front end extraction was performed using the NucliSENS easyMAG and MagNA Pure 96 systems. This study was designed to assess intra-assay (within run), inter-assay (run-to-run), day-to-day and site-tosite reproducibility. One lot of reagents was assayed at 3 sites (2 sites with easyMAG and 1 site with MP96) by 2 operators on 1 instrument per site for 5 days (total of 10 runs per site). The reproducibility panel consisted of 6 contrived positive samples and 1 negative sample, each extracted in triplicate and each assayed in singlet. The samples consisted of combinations of 12 representative targets at $1 . 5 \times$ LoD (Low) and 3x LoD (Medium) spiked into simulated NPS in UTM matrix (see table below).

Table. Reproducibility panel   

<table><tr><td rowspan=1 colspan=1>Medium (3x LoD)</td><td rowspan=1 colspan=1>Medium (3x LoD)</td><td rowspan=1 colspan=1>Low (1.5x LoD)</td><td rowspan=1 colspan=1>Low (1.5x LoD)</td><td rowspan=1 colspan=1>Samplename</td></tr><tr><td rowspan=1 colspan=1>Human Rhinovirus</td><td rowspan=1 colspan=1>Parainfluenza Virus 2</td><td rowspan=1 colspan=1>HumanMetapneumovirus</td><td rowspan=1 colspan=1>Bordetella pertussis</td><td rowspan=1 colspan=1>RP1</td></tr><tr><td rowspan=1 colspan=1>HumanMetapneumovirus</td><td rowspan=1 colspan=1>Bordetella pertussis</td><td rowspan=1 colspan=1>Human Rhinovirus</td><td rowspan=1 colspan=1>Parainfluenza Virus 2</td><td rowspan=1 colspan=1>RP2</td></tr><tr><td rowspan=1 colspan=1>Influenza B</td><td rowspan=1 colspan=1>Coronavirus NL63</td><td rowspan=1 colspan=1>Chlamydia pneumoniae</td><td rowspan=1 colspan=1>Parainfluenza Virus 3</td><td rowspan=1 colspan=1>RP3</td></tr><tr><td rowspan=1 colspan=1>Chlamydia pneumoniae</td><td rowspan=1 colspan=1>Parainfluenza Virus 3</td><td rowspan=1 colspan=1>Influenza B</td><td rowspan=1 colspan=1>Coronavirus NL63</td><td rowspan=1 colspan=1>RP4</td></tr><tr><td rowspan=1 colspan=1>Influenza A H3N2</td><td rowspan=1 colspan=1>Mycoplasmapneumoniae</td><td rowspan=1 colspan=1>Respiratory SyncytialVirus</td><td rowspan=1 colspan=1>Adenovirus C</td><td rowspan=1 colspan=1>RP5</td></tr><tr><td rowspan=1 colspan=1>Respiratory SyncytialVirus</td><td rowspan=1 colspan=1>Adenovirus C</td><td rowspan=1 colspan=1>Influenza A H3N2</td><td rowspan=1 colspan=1>Mycoplasmapneumoniae</td><td rowspan=1 colspan=1>RP6</td></tr><tr><td rowspan=1 colspan=4>Simulated Negative matrix</td><td rowspan=1 colspan=1>RP7</td></tr></table>

For each target, the results were determined according to the Interpretation Algorithm. Percent positive agreement was calculated for medium and low positives separately. Samples not containing said target were used to calculate percent negative agreement for each RPP target (see data tables below).

Conclusions: Acceptance criteria were met.

# Results

Table. Results from the multi-site reproducibility study (viruses)

<table><tr><td rowspan=3 colspan=1>Analyte</td><td rowspan=3 colspan=1>ConcentrationTested</td><td rowspan=3 colspan=1>ExpectedResult</td><td rowspan=1 colspan=6>Agreement with Expected Result</td></tr><tr><td rowspan=1 colspan=3>NucliSENS easyMAG</td><td rowspan=1 colspan=2>MagNA Pure 96</td><td rowspan=2 colspan=1>All Sites(95% Cl)</td></tr><tr><td rowspan=1 colspan=1>Site 1</td><td rowspan=1 colspan=1>Site 3</td><td rowspan=1 colspan=1>Sub-Total</td><td rowspan=1 colspan=1>Site 2</td><td rowspan=1 colspan=1>Sub-Total</td></tr><tr><td rowspan=1 colspan=1>cYanmaGellowb</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Viruses</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=3 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>3× LoD</td><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>60/60100%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>90/90100%(95.9%-100%)</td></tr><tr><td rowspan=1 colspan=1>1.5× LoD</td><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>60/60100%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>90/90100%(95.9%-100%)</td></tr><tr><td rowspan=1 colspan=1>None(no analyte)</td><td rowspan=1 colspan=1>NotDetected</td><td rowspan=1 colspan=1>150/150100%</td><td rowspan=1 colspan=1>149/15099.30%</td><td rowspan=1 colspan=1>299/30099.70%</td><td rowspan=1 colspan=1>150/150100%</td><td rowspan=1 colspan=1>150/150100%</td><td rowspan=1 colspan=1>449/45099.80%(98.8%-100%)</td></tr><tr><td rowspan=3 colspan=1>Coronavirus</td><td rowspan=1 colspan=1>3× LoD</td><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>60/60100%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>90/90100%(95.9%-100%)</td></tr><tr><td rowspan=1 colspan=1>1.5× LoD</td><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>60/60100%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>90/90100%(95.9%-100%)</td></tr><tr><td rowspan=1 colspan=1>None(no analyte)</td><td rowspan=1 colspan=1>NotDetected</td><td rowspan=1 colspan=1>150/150100%</td><td rowspan=1 colspan=1>150/150100%</td><td rowspan=1 colspan=1>300/300100%</td><td rowspan=1 colspan=1>150/150100%</td><td rowspan=1 colspan=1>150/150100%</td><td rowspan=1 colspan=1>450/450100%(99.2%-100%)</td></tr><tr><td rowspan=3 colspan=1>HumanMetapneumovirus</td><td rowspan=1 colspan=1>3× LoD</td><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>60/60100%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>90/90100%(95.9%-100%)</td></tr><tr><td rowspan=1 colspan=1>1.5× LoD</td><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>60/60100%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>90/90100%(95.9%-100%)</td></tr><tr><td rowspan=1 colspan=1>None(no analyte)</td><td rowspan=1 colspan=1>NotDetected</td><td rowspan=1 colspan=1>150/150100%</td><td rowspan=1 colspan=1>150/150100%</td><td rowspan=1 colspan=1>300/300100%</td><td rowspan=1 colspan=1>150/150100%</td><td rowspan=1 colspan=1>150/150100%</td><td rowspan=1 colspan=1>450/450100%(99.2%-100%)</td></tr><tr><td rowspan=3 colspan=1>HumanRhinovirus/Enterovirus</td><td rowspan=1 colspan=1>3x LoD</td><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>60/60100%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>90/90100%(95.9%-100%)</td></tr><tr><td rowspan=1 colspan=1>1.5× LoD</td><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>60/60100%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>90/90100%(95.9%-100%)</td></tr><tr><td rowspan=1 colspan=1>None(no analyte)</td><td rowspan=1 colspan=1>NotDetected</td><td rowspan=1 colspan=1>150/150100%</td><td rowspan=1 colspan=1>150/150100%</td><td rowspan=1 colspan=1>300/300100%</td><td rowspan=1 colspan=1>150/150100%</td><td rowspan=1 colspan=1>150/150100%</td><td rowspan=1 colspan=1>450/450100%(99.2%-100%)</td></tr><tr><td rowspan=3 colspan=1>Influenza A/H3</td><td rowspan=1 colspan=1>3× LoD</td><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>60/60100%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>90/90100%(95.9%-100%)</td></tr><tr><td rowspan=1 colspan=1>1.5× LoD</td><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>29/30 a96.70%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>59/6098.30%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>89/9098.90%(94.0%-99.8%)</td></tr><tr><td rowspan=1 colspan=1>None(no analyte)</td><td rowspan=1 colspan=1>NotDetected</td><td rowspan=1 colspan=1>150/150100%</td><td rowspan=1 colspan=1>149/150 b99.30%</td><td rowspan=1 colspan=1>299/30099.70%</td><td rowspan=1 colspan=1>150/150100%</td><td rowspan=1 colspan=1>150/150100%</td><td rowspan=1 colspan=1>449/45099.80%(98.8%-100%)</td></tr><tr><td rowspan=1 colspan=1>Influenza A/H1pdm09</td><td rowspan=1 colspan=1>None(no analyte)</td><td rowspan=1 colspan=1>NotDetected</td><td rowspan=1 colspan=1>210/210100%</td><td rowspan=1 colspan=1>210/210100%</td><td rowspan=1 colspan=1>420/420100%</td><td rowspan=1 colspan=1>210/210100%</td><td rowspan=1 colspan=1>210/210100%</td><td rowspan=1 colspan=1>630/630100%(99.4%-100%)</td></tr><tr><td rowspan=1 colspan=1>Influenza A/H1</td><td rowspan=1 colspan=1>None(no analyte)</td><td rowspan=1 colspan=1>NotDetected</td><td rowspan=1 colspan=1>210/210100%</td><td rowspan=1 colspan=1>210/210100%</td><td rowspan=1 colspan=1>420/420100%</td><td rowspan=1 colspan=1>210/210100%</td><td rowspan=1 colspan=1>210/210100%</td><td rowspan=1 colspan=1>630/630100%(99.4%-100%)</td></tr><tr><td rowspan=1 colspan=1>Influenza B</td><td rowspan=1 colspan=1>3× LoD</td><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>60/60</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>90/90</td></tr></table>

# Premarket Notification 510(k)

<table><tr><td rowspan=3 colspan=1>Analyte</td><td rowspan=3 colspan=1>ConcentrationTested</td><td rowspan=3 colspan=1>ExpectedResult</td><td rowspan=1 colspan=6>Agreement with Expected Result</td></tr><tr><td rowspan=1 colspan=3>NucliSENS easyMAG</td><td rowspan=1 colspan=2>MagNA Pure 96</td><td rowspan=2 colspan=1>All Sites(95% Cl)</td></tr><tr><td rowspan=1 colspan=1>Site 1</td><td rowspan=1 colspan=1>Site 3</td><td rowspan=1 colspan=1>Sub-Total</td><td rowspan=1 colspan=1>Site 2</td><td rowspan=1 colspan=1>Sub-Total</td></tr><tr><td rowspan=3 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%(95.9%-100%)</td></tr><tr><td rowspan=1 colspan=1>1.5× LoD</td><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>29/3096.70%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>59/6098.30%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>89/9098.90%(94.0%-99.8%)</td></tr><tr><td rowspan=1 colspan=1>None(no analyte)</td><td rowspan=1 colspan=1>NotDetected</td><td rowspan=1 colspan=1>150/150100%</td><td rowspan=1 colspan=1>150/150100%</td><td rowspan=1 colspan=1>300/300100%</td><td rowspan=1 colspan=1>150/150100%</td><td rowspan=1 colspan=1>150/150100%</td><td rowspan=1 colspan=1>450/450100%(99.2%-100%)</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza Virus 1</td><td rowspan=1 colspan=1>None(no analyte)</td><td rowspan=1 colspan=1>NotDetected</td><td rowspan=1 colspan=1>210/210100%</td><td rowspan=1 colspan=1>210/210100%</td><td rowspan=1 colspan=1>420/420100%</td><td rowspan=1 colspan=1>210/210100%</td><td rowspan=1 colspan=1>210/210100%</td><td rowspan=1 colspan=1>630/630100%(99.4%-100%)</td></tr><tr><td rowspan=3 colspan=1>Parainfluenza Virus 2</td><td rowspan=1 colspan=1>3× LoD</td><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>60/60100%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>90/90100%(95.9%-100%)</td></tr><tr><td rowspan=1 colspan=1>1.5× LoD</td><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>60/60100%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>90/90100%(95.9%-100%)</td></tr><tr><td rowspan=1 colspan=1>None(no analyte)</td><td rowspan=1 colspan=1>NotDetected</td><td rowspan=1 colspan=1>150/150100%</td><td rowspan=1 colspan=1>150/150100%</td><td rowspan=1 colspan=1>300/300100%</td><td rowspan=1 colspan=1>150/150100%</td><td rowspan=1 colspan=1>150/150100%</td><td rowspan=1 colspan=1>450/450100%(99.2%-100%)</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza Virus 4</td><td rowspan=1 colspan=1>None(no analyte)</td><td rowspan=1 colspan=1>NotDetected</td><td rowspan=1 colspan=1>210/210100%</td><td rowspan=1 colspan=1>210/210100%</td><td rowspan=1 colspan=1>420/420100%</td><td rowspan=1 colspan=1>210/210100%</td><td rowspan=1 colspan=1>210/210100%</td><td rowspan=1 colspan=1>630/630100%(99.4%-100%)</td></tr><tr><td rowspan=3 colspan=1>Parainfluenza Virus 3</td><td rowspan=1 colspan=1>3× LoD</td><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>60/60100%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>90/90100%(95.9%-100%)</td></tr><tr><td rowspan=1 colspan=1>1.5× LoD</td><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>60/60100%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>90/90100%(95.9%-100%)</td></tr><tr><td rowspan=1 colspan=1>None(no analyte)</td><td rowspan=1 colspan=1>NotDetected</td><td rowspan=1 colspan=1>150/150100%</td><td rowspan=1 colspan=1>150/150100%</td><td rowspan=1 colspan=1>300/300100%</td><td rowspan=1 colspan=1>150/150100%</td><td rowspan=1 colspan=1>150/150100%</td><td rowspan=1 colspan=1>450/450100%(99.2%-100%)</td></tr><tr><td rowspan=3 colspan=1>Respiratory SyncytialVirus</td><td rowspan=1 colspan=1>3× LoD</td><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>29/30100%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>59/6098.30%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>89/9098.90%(94.0%-99.8%)</td></tr><tr><td rowspan=1 colspan=1>1.5× LoD</td><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>29/30100%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>59/6098.30%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>89/9098.90%(94.0%-99.8%)</td></tr><tr><td rowspan=1 colspan=1>None(no analyte)</td><td rowspan=1 colspan=1>NotDetected</td><td rowspan=1 colspan=1>150/150100%</td><td rowspan=1 colspan=1>150/150100%</td><td rowspan=1 colspan=1>300/300100%</td><td rowspan=1 colspan=1>150/150100%</td><td rowspan=1 colspan=1>150/150100%</td><td rowspan=1 colspan=1>450/450100%(99.2%-100%)</td></tr></table>

a – There was an indeterminate Flu A result for Influenza A H3 low positive sample RP6 b – There was an indeterminate Flu A result for Influenza A H3 negative sample

Table. Results from the multi-site reproducibility study (bacteria)   

<table><tr><td rowspan=3 colspan=1>Analyte</td><td rowspan=3 colspan=1>ConcentrationTested</td><td rowspan=3 colspan=1>ExpectedResult</td><td rowspan=1 colspan=6>Agreement with Expected Result</td></tr><tr><td rowspan=1 colspan=3>NucliSENS easyMAG</td><td rowspan=1 colspan=2>MagNA Pure 96</td><td rowspan=2 colspan=1>All Sites(95% Cl)</td></tr><tr><td rowspan=1 colspan=1>Site 1</td><td rowspan=1 colspan=1>Site 3</td><td rowspan=1 colspan=1>Sub-Total</td><td rowspan=1 colspan=1>Site 2</td><td rowspan=1 colspan=1>Sub-Total</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Bacteria</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=3 colspan=1>Mycoplasmapneumoniae</td><td rowspan=1 colspan=1>3× LoD</td><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>60/60100%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>90/90100%(95.9%-100%)</td></tr><tr><td rowspan=1 colspan=1>1.5× LoD</td><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>60/60100%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>90/90100%(95.9%-100%)</td></tr><tr><td rowspan=1 colspan=1>None(no analyte)</td><td rowspan=1 colspan=1>NotDetected</td><td rowspan=1 colspan=1>150/150100%</td><td rowspan=1 colspan=1>149/15099.30%</td><td rowspan=1 colspan=1>299/30099.70%</td><td rowspan=1 colspan=1>150/150100%</td><td rowspan=1 colspan=1>150/150100%</td><td rowspan=1 colspan=1>449/45099.80%(98.8%-100%)</td></tr><tr><td rowspan=3 colspan=1>Bordetella pertussis</td><td rowspan=1 colspan=1>3× LoD</td><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>60/60100%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>90/90100%(95.9%-100%)</td></tr><tr><td rowspan=1 colspan=1>1.5× LoD</td><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>60/60100%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>90/90100%(95.9%-100%)</td></tr><tr><td rowspan=1 colspan=1>None(no analyte)</td><td rowspan=1 colspan=1>NotDetected</td><td rowspan=1 colspan=1>150/150100%</td><td rowspan=1 colspan=1>150/150100%</td><td rowspan=1 colspan=1>300/300100%</td><td rowspan=1 colspan=1>150/150100%</td><td rowspan=1 colspan=1>150/150100%</td><td rowspan=1 colspan=1>450/450100%(99.2%-100%)</td></tr><tr><td rowspan=3 colspan=1>Chlamydia pneumoniae</td><td rowspan=1 colspan=1>3× LoD</td><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>60/60100%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>90/90100%(95.9%-100%)</td></tr><tr><td rowspan=1 colspan=1>1.5× LoD</td><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>60/60100%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>90/90100%(95.9%-100%)</td></tr><tr><td rowspan=1 colspan=1>None(no analyte)</td><td rowspan=1 colspan=1>NotDetected</td><td rowspan=1 colspan=1>150/150100%</td><td rowspan=1 colspan=1>150/150100%</td><td rowspan=1 colspan=1>300/300100%</td><td rowspan=1 colspan=1>150/150100%</td><td rowspan=1 colspan=1>150/150100%</td><td rowspan=1 colspan=1>450/450100%(99.2%-100%)</td></tr></table>

# Multiple analyte spiked samples Vs single spiked samples

This study is to assess the performance of the BioCode® RPP on the BioCode® MDx-3000 with mixed analyte samples at or near the Limit of Detection (LoD) compared to single spiked samples to justify the use of multiple analyte spiked samples during analytical validation testing. The LoD was confirmed by extracting 20 replicates of each sample type and testing each in singlet for a total of 20 replicates at or near presumptive LoD. LoD for each stock was defined as the lowest concentration with $2 9 5 \%$ detection of 20 replicates (19 out of 20). The single spiked LoD was then challenged by pooling all 4 organisms and testing at 1x LoD (20 replicates). If 1x LoD for each analyte in the mixed pool does not meet the $9 5 \%$ detection goal (19/20), the organism was retested at 3x LoD. Results within $3 \times$ LoD was considered equivalent.

Results: All organisms were within 3x LoD for single and multiple spiked samples.

Table. Results from Multiple Vs Single spiked sample study stratified by extraction system.   

<table><tr><td rowspan=2 colspan=1>Strain</td><td rowspan=2 colspan=1>Source</td><td rowspan=2 colspan=1>TargetProbe</td><td rowspan=2 colspan=1>Single /MultipleSpike</td><td rowspan=1 colspan=2>Concentration</td></tr><tr><td rowspan=1 colspan=1>EasyMAG</td><td rowspan=1 colspan=1>MP96</td></tr><tr><td rowspan=4 colspan=1>Influenza AH3N2/A/Wisconsin/67/05a</td><td rowspan=4 colspan=1>Zeptometrix0810252CF</td><td rowspan=1 colspan=1>FluA</td><td rowspan=2 colspan=1>Single Spike</td><td rowspan=2 colspan=1>1.3 TCID50/mL</td><td rowspan=4 colspan=1>1.3 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>FluAH3</td></tr><tr><td rowspan=1 colspan=1>FluA</td><td rowspan=2 colspan=1>Multi Spike</td><td rowspan=2 colspan=1>4.0 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>FluAH3</td></tr><tr><td rowspan=2 colspan=1>Coronavirus NL63</td><td rowspan=2 colspan=1>Zeptometrix0810228CF</td><td rowspan=2 colspan=1>NL63</td><td rowspan=1 colspan=1>Single Spike</td><td rowspan=1 colspan=1>0.013 TCID50/mL</td><td rowspan=2 colspan=1>0.040 TCID5o/mL</td></tr><tr><td rowspan=1 colspan=1>Multi Spike</td><td rowspan=1 colspan=1>0.040 TCID5o/mL</td></tr><tr><td rowspan=2 colspan=1>Mycoplasmapneumoniae (M129)</td><td rowspan=2 colspan=1>Zeptometrix0801579</td><td rowspan=2 colspan=1>MPN</td><td rowspan=1 colspan=1>Single Spike</td><td rowspan=1 colspan=1>5.0 CCU/mL</td><td rowspan=2 colspan=1>15.0 CCU/mL</td></tr><tr><td rowspan=1 colspan=1>Multi Spike</td><td rowspan=1 colspan=1>15.0 CCU/mL</td></tr><tr><td rowspan=1 colspan=1>Adenovirus C (type 2)</td><td rowspan=1 colspan=1>ATCC AV-846</td><td rowspan=1 colspan=1>ADV1</td><td rowspan=1 colspan=1>Single SpikeMulti Spike</td><td rowspan=1 colspan=1>6.0 TCID5o/mL</td><td rowspan=1 colspan=1>18.0 TCID5o/mL</td></tr></table>

Conclusion: Multiple spiked (mixed analyte) samples confirmed $9 5 \%$ LoD at the same or within 3-fold for each organism. Based on this testing multiple spiked samples may be used for other bench studies.

# Limit of Detection (LoD)

A study was performed to assess the performance of the BioCode® RPP on the BioCode® MDx-3000 at the Limit of Detection (LoD) for specimens. In this study the BioCode® RPP was tested with quantified bacteria or viral stocks spiked in simulated NPS in UTM matrix. For initial screening, four replicates of each concentration were extracted on the easyMAG and MagNA Pure 96 Systems and tested in singlet with the BioCode® RPP on the BioCode® MDx-3000 system to estimate LoD. The LoD was confirmed by extracting 20 replicates of each sample type and testing each in singlet for a total of 20 replicates at or near the presumptive LoD. LoD for each stock was defined as the lowest concentration with $2 9 5 \%$ detection of 20 replicates (19 out of 20).

Results: See table below.

Table. Limit of Detection by extraction system   

<table><tr><td rowspan=2 colspan=1>Target</td><td rowspan=2 colspan=1>Species/Strain/Isolate</td><td rowspan=2 colspan=1>Source</td><td rowspan=1 colspan=2>EasyMAG</td><td rowspan=1 colspan=2>MP96</td></tr><tr><td rowspan=1 colspan=1>Concentration</td><td rowspan=1 colspan=1>Detected(n of 20)</td><td rowspan=1 colspan=1>Concentration</td><td rowspan=1 colspan=1>Detected(n of )</td></tr><tr><td rowspan=2 colspan=1>Influenza A H1</td><td rowspan=1 colspan=1>A/NewCaledonia/20/99</td><td rowspan=1 colspan=1>Zeptometrix0810036CF</td><td rowspan=1 colspan=1>15.0 TCID5o/mL</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>5.0 TCID5o/mL</td><td rowspan=1 colspan=1>20/20</td></tr><tr><td rowspan=1 colspan=1>A/NWS/33</td><td rowspan=1 colspan=1>ATCC VR-219</td><td rowspan=1 colspan=1>27.0 TCID50/mL</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>9.0 TCID5o/mL</td><td rowspan=1 colspan=1>20/20</td></tr><tr><td rowspan=1 colspan=1>Influenza A H12009pdm</td><td rowspan=1 colspan=1>A(H1N1)/California/07/09</td><td rowspan=1 colspan=1>Zeptometrix0810165CF</td><td rowspan=1 colspan=1>0.4 TCID5o/mL</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0.4 TCID5o/mL</td><td rowspan=1 colspan=1>20/20</td></tr><tr><td rowspan=2 colspan=1>Influenza A H3</td><td rowspan=1 colspan=1>A/Wisconsin/67/05</td><td rowspan=1 colspan=1>Zeptometrix0810252CF</td><td rowspan=1 colspan=1>4.0 TCID50/mL</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>1.3 TCID50/mL</td><td rowspan=1 colspan=1>20/20</td></tr><tr><td rowspan=1 colspan=1>A/Alice</td><td rowspan=1 colspan=1>ATCC VR776</td><td rowspan=1 colspan=1>27.0 TCID50/mL</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>9.0 TCID5o/mL</td><td rowspan=1 colspan=1>19/20</td></tr><tr><td rowspan=2 colspan=1>Influenza B</td><td rowspan=1 colspan=1>Flu B/Florida/4/2006(Yamagata)</td><td rowspan=1 colspan=1>Zeptometrix0810255CF</td><td rowspan=1 colspan=1>0.01 TCID5o/mL</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0.01 TCID5o/mL</td><td rowspan=1 colspan=1>20/20</td></tr><tr><td rowspan=1 colspan=1>B/Hong Kong/S/1972(Viictoria)</td><td rowspan=1 colspan=1>ATCC VR-823</td><td rowspan=1 colspan=1>48.6 TCID5o/mL</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>48.6 TCID5o/mL</td><td rowspan=1 colspan=1>20/20</td></tr><tr><td rowspan=1 colspan=1>Respiratory SyncytialVirus</td><td rowspan=1 colspan=1>Type A</td><td rowspan=1 colspan=1>Zeptometrix0810040ACF</td><td rowspan=1 colspan=1>0.33 TCID5o/mL</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0.33 TCID5o/mL</td><td rowspan=1 colspan=1>20/20</td></tr><tr><td rowspan=1 colspan=1>HumanMetapneumovirus</td><td rowspan=1 colspan=1>16; Type A1 IA10-2003</td><td rowspan=1 colspan=1>Zeptometrix0810161CF</td><td rowspan=1 colspan=1>15.0 TCID50/mL</td><td rowspan=1 colspan=1>19/20</td><td rowspan=1 colspan=1>15.0 TCID5o/mL</td><td rowspan=1 colspan=1>20/20</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza Virus 1</td><td rowspan=1 colspan=1>C-35/WashingtonDC/1957</td><td rowspan=1 colspan=1>ATCC VR-94</td><td rowspan=1 colspan=1>9.0 TCID5o/mL</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>9.0 TCID50/mL</td><td rowspan=1 colspan=1>20/20</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza Virus 2</td><td rowspan=1 colspan=1>Greer/Ohio/1955</td><td rowspan=1 colspan=1>ATCC VR-92</td><td rowspan=1 colspan=1>1.8 TCID5o/mL</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>5.4 TCID5o/mL</td><td rowspan=1 colspan=1>20/20</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza Virus 3</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>Zeptometrix0810016CF</td><td rowspan=1 colspan=1>15.0 TCID5o/mL</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>15.0 TCID50/mL</td><td rowspan=1 colspan=1>20/20</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza Virus 4</td><td rowspan=1 colspan=1>Type 4a</td><td rowspan=1 colspan=1>Zeptometrix0810060OCF</td><td rowspan=1 colspan=1>9.0 TCID5o/mL</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>9.0 TCID5o/mL</td><td rowspan=1 colspan=1>20/20</td></tr><tr><td rowspan=1 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>Species B Serotype7A</td><td rowspan=1 colspan=1>Zeptometrix0810021CF</td><td rowspan=1 colspan=1>1.2 TCID5o/mL</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>1.2 TCID50/mL</td><td rowspan=1 colspan=1>20/20</td></tr><tr><td rowspan=1 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>Species C Serotype 2</td><td rowspan=1 colspan=1>ATCC VR-846</td><td rowspan=1 colspan=1>6.0 TCID5o/mL</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>18.0 TCID50/mL</td><td rowspan=1 colspan=1>20/20</td></tr><tr><td rowspan=1 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>Species E Serotype 4</td><td rowspan=1 colspan=1>Zeptometrix0810070CF</td><td rowspan=1 colspan=1>0.04 TCID5o/mL</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0.04 TCID50/mL</td><td rowspan=1 colspan=1>19/20</td></tr><tr><td rowspan=1 colspan=1>Coronavirus 229E</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>Zeptometrix0810229CF</td><td rowspan=1 colspan=1>0.6 TCID5o/mL</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>1.8 TCID5o/mL</td><td rowspan=1 colspan=1>20/20</td></tr><tr><td rowspan=1 colspan=1>Coronavirus HKU1</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>ClinicalSample4922a</td><td rowspan=1 colspan=1>5.02x104copies/mL</td><td rowspan=1 colspan=1>19/20</td><td rowspan=1 colspan=1>5.02x104copies/mL</td><td rowspan=1 colspan=1>20/20</td></tr><tr><td rowspan=1 colspan=1>Coronavirus NL63</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>Zeptometrix0810228CF</td><td rowspan=1 colspan=1>0.04 TCID5o/mL</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0.04 TCID5o/mL</td><td rowspan=1 colspan=1>20/20</td></tr><tr><td rowspan=1 colspan=1>Coronavirus OC43</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>Zeptometrix0810024CF</td><td rowspan=1 colspan=1>0.04 TCID5o/mL</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0.01 TCID50/mL</td><td rowspan=1 colspan=1>19/20</td></tr><tr><td rowspan=1 colspan=1>Rhinovirus</td><td rowspan=1 colspan=1>Type A1</td><td rowspan=1 colspan=1>Zeptometrix0810012CF</td><td rowspan=1 colspan=1>1.2 TCID5o/mL</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0.4 TCID5o/mL</td><td rowspan=1 colspan=1>19/20</td></tr><tr><td rowspan=1 colspan=1>Enterovirus</td><td rowspan=1 colspan=1>D68</td><td rowspan=1 colspan=1>Zeptometrix0810300CF</td><td rowspan=1 colspan=1>3.0 TCID50/mL</td><td rowspan=1 colspan=1>19/20</td><td rowspan=1 colspan=1>9.0 TCID50/mL</td><td rowspan=1 colspan=1>20/20</td></tr><tr><td rowspan=1 colspan=1>Bordetella pertussis</td><td rowspan=1 colspan=1>A639</td><td rowspan=1 colspan=1>Zeptometrix801459</td><td rowspan=1 colspan=1>15.0 CFU/mL</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>45.0 CFU/mL</td><td rowspan=1 colspan=1>19/20</td></tr></table>

Premarket Notification 510(k)   

<table><tr><td rowspan=2 colspan=1>Target</td><td rowspan=2 colspan=1>Species/Strain/Isolate</td><td rowspan=2 colspan=1>Source</td><td rowspan=1 colspan=2>EasyMAG</td><td rowspan=1 colspan=2>MP96</td></tr><tr><td rowspan=1 colspan=1>Concentration</td><td rowspan=1 colspan=1>Detected(n of 20)</td><td rowspan=1 colspan=1>Concentration</td><td rowspan=1 colspan=1>Detected(n of 20)</td></tr><tr><td rowspan=1 colspan=1>Chlamydia pneumoniae</td><td rowspan=1 colspan=1>AR39</td><td rowspan=1 colspan=1>ATCC VR-53592</td><td rowspan=1 colspan=1>16.7 CFU/mL</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>33.3 CFU/mL</td><td rowspan=1 colspan=1>20/20</td></tr><tr><td rowspan=1 colspan=1>Mycoplasmapneumoniae</td><td rowspan=1 colspan=1>M129</td><td rowspan=1 colspan=1>Zeptometrix0801579</td><td rowspan=1 colspan=1>15.0 CCU/mL</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>15.0 CCU/mL</td><td rowspan=1 colspan=1>20/20</td></tr></table>

a. Coronavirus HKU1 clinical sample quantified (copies/mL) with Applied BioCode validated SYBR assay using an IVT RNA standard.

Conclusion: LoDs were comparable (within 3-fold) for each extraction system.

# Analytical Reactivity/Inclusivity

A study was performed to verify Analytical Reactivity/Inclusivity of the BioCode® Respiratory Pathogen Panel (RPP). Different strains, serotypes and genotypes were selected that represent various temporal, geographic, and genetic diversity for each analyte. This study tested a panel of titered stocks of relevant organisms diluted in simulated NPS in UTM matrix starting at 3x LoD. Samples not detected at 3x LoD, were tested at higher concentrations. For Influenza B strains, all strains were grouped into Yamagata, Victoria or unknown lineages. Yamagata strains were tested according to the protocol at 3x and higher of the Yamagata strain LoD $( \mathsf { L O D } - 0 . 0 1 \mathsf { T C } | \mathsf { D } _ { 5 0 } / \mathsf { m L }$ Flu B/Florida/4/2006). However, for Victoria strains, the validation LoD result was higher than expected $\left( \mathsf { L O D - } 4 8 . 6 \mathsf { T C l D 5 0 / m L } \right.$ for Flu B /Hong Kong/S/1972) which could be related to difference in titration from vendor sources. Therefore, for Victoria strains, testing for inclusivity was performed starting at $0 . 0 3 \times$ and $0 . 3 \times$ of the Victoria strain LoD. Strains of unknown lineages of influenza B were tested at $3 \times$ LoD of Yamagata strain. Each sample was extracted in triplicate on the easyMAG and tested with the RPP on the BioCode® MDx-3000 system according to the instructions for use.

Assay reactivity for less common strains or serotypes that could not be tested due to unavailability was predicted using in silico analysis.

Results: The organisms listed below were detected at the concentrations indicated. See Table below.

Table. Influenza A isolates tested during the inclusivity study   

<table><tr><td rowspan=1 colspan=1>Organism/Typea</td><td rowspan=1 colspan=1>Strain/Location/Year</td><td rowspan=1 colspan=1>Vendor</td><td rowspan=1 colspan=1>Catalog #</td><td rowspan=1 colspan=1>ConcentrationDetected</td><td rowspan=1 colspan=1>Multiple ofLoDDetected</td></tr><tr><td rowspan=6 colspan=1>Influenza A H1N1</td><td rowspan=1 colspan=1>Solomon Island/03/06</td><td rowspan=1 colspan=1>Zeptometrix</td><td rowspan=1 colspan=1>0810036CFN</td><td rowspan=1 colspan=1>45 TCID5o/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>Singapore/63/04</td><td rowspan=1 colspan=1>Zeptometrix</td><td rowspan=1 colspan=1>0810246CF</td><td rowspan=1 colspan=1>45 TCID5o/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>PR/8/34</td><td rowspan=1 colspan=1>Zeptometrix</td><td rowspan=1 colspan=1>0810245CF</td><td rowspan=1 colspan=1>45 TCID5o/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>A/Brisbane/59/2007</td><td rowspan=1 colspan=1>Zeptometrix</td><td rowspan=1 colspan=1>0810244CF</td><td rowspan=1 colspan=1>45 TCID5o/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>A/Taiwan/42/06</td><td rowspan=1 colspan=1>Zeptometrix</td><td rowspan=1 colspan=1>0810247CF</td><td rowspan=1 colspan=1>45 TCID50/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>A/New jersey/8/76b</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>VR-897</td><td rowspan=1 colspan=1>7.5x10³3 CEID50/mL</td><td rowspan=1 colspan=1>500xb</td></tr></table>

# Premarket Notification 510(k)

<table><tr><td rowspan=1 colspan=1>Organism/Typea</td><td rowspan=1 colspan=1>Strain/Location/Year</td><td rowspan=1 colspan=1>Vendor</td><td rowspan=1 colspan=1>Catalog #</td><td rowspan=1 colspan=1>ConcentrationDetected</td><td rowspan=1 colspan=1>Multiple ofLoDDetected</td></tr><tr><td rowspan=6 colspan=1>b</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>A/Denver/1/1957</td><td rowspan=1 colspan=1>VIRAPUR</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>45 TCID5o/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>A/FM/1/47</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>VR-97</td><td rowspan=1 colspan=1>45 CEID50/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>A/Weiss/43c</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>VR-96</td><td rowspan=1 colspan=1>1.5x104 CEID50/mL</td><td rowspan=1 colspan=1>1000xc</td></tr><tr><td rowspan=1 colspan=1>A/Beijing/262/95d</td><td rowspan=1 colspan=1>BEI</td><td rowspan=1 colspan=1>NR-12277</td><td rowspan=1 colspan=1>450 CEID5o/mL</td><td rowspan=1 colspan=1>30xd</td></tr><tr><td rowspan=1 colspan=1>A/Mal/302/54</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>VR-98</td><td rowspan=1 colspan=1>45 CEID50/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>Influenza A H1N2</td><td rowspan=1 colspan=1>Recombinant; Kilbourne F64,A/NWS/1934 (HA) xA/Rockefeller Institute/ 5/1957(NA)e</td><td rowspan=1 colspan=1>BEI</td><td rowspan=1 colspan=1>NR-3682</td><td rowspan=1 colspan=1>135 CEID50/mL</td><td rowspan=1 colspan=1>5xe</td></tr><tr><td rowspan=9 colspan=1>Influenza A H1N1pdm09</td><td rowspan=1 colspan=1>NY/01/09</td><td rowspan=1 colspan=1>Zeptometrix</td><td rowspan=1 colspan=1>0810248CF</td><td rowspan=1 colspan=1>1.2 TCID50/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>NY/02/09</td><td rowspan=1 colspan=1>Zeptometrix</td><td rowspan=1 colspan=1>0810109CFN</td><td rowspan=1 colspan=1>1.2 TCID50/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>NY/03/09</td><td rowspan=1 colspan=1>Zeptometrix</td><td rowspan=1 colspan=1>0810249CF</td><td rowspan=1 colspan=1>1.2 TCID50/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>A/Houston/3H/2009(H1N1)pdm09f</td><td rowspan=1 colspan=1>BEI</td><td rowspan=1 colspan=1>NR-20340</td><td rowspan=1 colspan=1>1.2 TCID50/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>Influenza A H1N1pdm(Canada/6294/09)</td><td rowspan=1 colspan=1>Zeptometrix</td><td rowspan=1 colspan=1>0810109CFJ</td><td rowspan=1 colspan=1>1.2 TCID5o/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>Influenza A H1N1pdm(Mexico/4108/09)</td><td rowspan=1 colspan=1>Zeptometrix</td><td rowspan=1 colspan=1>0810166CF</td><td rowspan=1 colspan=1>1.2 TCID50/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>California/04/09, cell isolate</td><td rowspan=1 colspan=1>BEI</td><td rowspan=1 colspan=1>NR-13658</td><td rowspan=1 colspan=1>40 TCID50/mL</td><td rowspan=1 colspan=1>100x8</td></tr><tr><td rowspan=1 colspan=1>A/Christ Church/16/2010h</td><td rowspan=1 colspan=1>CDC</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>400 EID50/mL</td><td rowspan=1 colspan=1>1000xh</td></tr><tr><td rowspan=1 colspan=1>A/Brisbane/02/2018</td><td rowspan=1 colspan=1>CDC</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>40 EID50/mL</td><td rowspan=1 colspan=1>100xi</td></tr><tr><td rowspan=5 colspan=1>Influenza A H3N2</td><td rowspan=1 colspan=1>A/Wisconsin/15/2009i</td><td rowspan=1 colspan=1>BEI</td><td rowspan=1 colspan=1>NR-42007</td><td rowspan=1 colspan=1>12 CEID50/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>A/Texas/50/12</td><td rowspan=1 colspan=1>Zeptometrix</td><td rowspan=1 colspan=1>0810238CF</td><td rowspan=1 colspan=1>12 TCID5o/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>A/Brisbane/10/2007</td><td rowspan=1 colspan=1>Zeptometrix</td><td rowspan=1 colspan=1>0810138CF</td><td rowspan=1 colspan=1>12 TCID50/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>A/Port Chalmers/1/73</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>VR-810</td><td rowspan=1 colspan=1>200 CEID50/mL</td><td rowspan=1 colspan=1>50xk</td></tr><tr><td rowspan=1 colspan=1>A/Victoria/3/1975</td><td rowspan=1 colspan=1>VIRAPUR</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>12 TCID50/mL</td><td rowspan=1 colspan=1>3x</td></tr></table>

# Premarket Notification 510(k)

<table><tr><td rowspan=1 colspan=1>Organism/Typea</td><td rowspan=1 colspan=1>Strain/Location/Year</td><td rowspan=1 colspan=1>Vendor</td><td rowspan=1 colspan=1>Catalog #</td><td rowspan=1 colspan=1>ConcentrationDetected</td><td rowspan=1 colspan=1>Multiple ofLoDDetected</td></tr><tr><td rowspan=10 colspan=1></td><td rowspan=1 colspan=1>A/Victoria/361/2011&#x27;</td><td rowspan=1 colspan=1>BEI</td><td rowspan=1 colspan=1>NR-44022</td><td rowspan=1 colspan=1>12 CEID50/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>A/Victoria/3/75</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>VR-822</td><td rowspan=1 colspan=1>12 CEID50/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>A/Uruguay/716/07m</td><td rowspan=1 colspan=1>BEI</td><td rowspan=1 colspan=1>NR-42003</td><td rowspan=1 colspan=1>40 TCID50/mL</td><td rowspan=1 colspan=1>10x</td></tr><tr><td rowspan=1 colspan=1>A/HK/H090-756-V1(0)/2009n</td><td rowspan=1 colspan=1>BEI</td><td rowspan=1 colspan=1>NR-44344</td><td rowspan=1 colspan=1>12 TCID50/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>A/Hong Kong/8/68</td><td rowspan=1 colspan=1>Zeptometrix</td><td rowspan=1 colspan=1>0810250CF</td><td rowspan=1 colspan=1>12 TCID50/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>A/Switzerland/9715293/13</td><td rowspan=1 colspan=1>VIRAPUR</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>12 TCID5o/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>A/Aichi/2/68</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>VR-547</td><td rowspan=1 colspan=1>12 TCID50/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>MRC-2</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>VR-777</td><td rowspan=1 colspan=1>12 TCID50/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>A/Perth/16/2009</td><td rowspan=1 colspan=1>CDC</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>40 EID5o/mL</td><td rowspan=1 colspan=1>10x</td></tr><tr><td rowspan=1 colspan=1>A/Kansas/14/2017°</td><td rowspan=1 colspan=1>CDC</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>8000 EID50/mL</td><td rowspan=1 colspan=1>2000x°</td></tr></table>

a – In silico analysis predicts detection of Influenza A H2N3, H5N1, H5N2, H5N3, H5N8, H7N7, H7N9, H3N1, H3N2, H3N5, H3N7, H3N8 as Influenza A. However, predicted reactivities of the subtyping assays for these influenza A strains of animal origin are variable.

b – Influenza A H1N1 – [A/New jersey/8/76]. Detected as Flu A (no subtype) at $_ { 3 \times }$ LoD. Detected as dual positive $\mathsf { A } / \mathsf { H } \mathsf { 1 }$ and A/H1pdm09 at 500x LoD.

c – Influenza A H1N1 [A/Weiss/43]. Detected as Flu A (no subtype) at ${ 1 0 0 \times }$ LoD. In silico analysis showed several mismatches in the forward primer for the Flu A/H1 subtyping assay which may account for the observed lower sensitivity of the Flu A/H1 subtyping assay for this strain.

d – Influenza A H1N1 [A/Beijing/262/95]. Virus obtained through BEI Resources, NIAID, NIH: Influenza A, A/Beijing/262/95 (H1N1), NR-12277. Detected as Flu A (no subtype) at 10x LoD. In silico analysis showed a G-A mismatch in the ${ \mathfrak { z } } ^ { \prime }$ terminal position in the forward primer for the Flu A/H1 subtyping assay which may account for the observed lower sensitivity of the Flu A/H1 subtyping assay for this strain.

e - Influenza A H1N2 [Recombinant]. Recombinant Virus obtained through BEI Resources, NIAID, NIH: Influenza A, Kilbourne F64, A/NWS/1934 (HA) x A/Rockefeller Institute/5/1957 (NA)] (H1N2), NR-3682.Detected as Flu A (no subtype) at 3x LoD. f – Virus obtained through BEI Resources, NIAID, NIH: Influenza A, A/Houston/3H/2009 (H1N1)pdm09, NR-20340.

g – Virus obtained through BEI Resources, NIAID, NIH: Influenza A, A/California/04/09, cell isolate (H1N1) pdm09, NR-13658. In silico analysis of a partial sequence of this strain for the Flu A/H1pdm09 subtyping assay does not predict reduced analytical reactivity. Titering inconsistency from the vendor rather than reduced reactivity due to assay design is suggested.

h - Virus obtained through the CDC Influenza Division. In silico analysis of partial sequences of this strain for the Flu A/H1pdm09 HA subtyping assay and the Flu A matrix gene assay does not predict reduced analytical reactivity. Titering inconsistency from the source laboratory $\mathsf { E l D } _ { 5 0 } / \mathsf { m L }$ vs. $\mathsf { T C l D } _ { 5 0 } / \mathsf { m L } )$ ) rather than reduced reactivity due to assay design is suggested.

i - Virus obtained through the CDC Influenza Division. In silico analysis did not reveal any critical mismatch in the Flu A/H1pdm09 HA subtyping assay primers and probe binding regions or any mismatch in the FluA matrix gene assay primers and probe binding regions that would predict reduced analytical reactivity. Titering inconsistency from the source laboratory $( E | \ r _ { D } \ L _ { 5 0 } / \ r _ { \mathrm { m L } }$ vs. $\mathsf { T C l D } _ { 5 0 } / \mathsf { m L }$ ) rather than reduced reactivity due to assay design is suggested.

Virus obtained through BEI Resources, NIAID, NIH: Influenza A, A/Wisconsin/15/2009 (H3N2), NR-4200 k - In silico analysis revealed a few non-critical mismatches in the FluA/H3 subtyping HA assay probe binding region that could contribute to the observed lower reactivity. However, titering inconsistency from the vendor rather than reduced reactivity due to assay design is suspected.

l - Virus obtained through BEI Resources, NIAID, NIH: Influenza A, A/Victoria/361/2011 (H3N2), NR-44022 m - Virus obtained through BEI Resources, NIAID, NIH: Influenza A, A/Uruguay/716/07 (H3N2), NR-42003 n - Virus obtained through BEI Resources, NIAID, NIH: Influenza A, A/HK/H090-756-V1(0)/2009 (H3N2), NR-44344 o – Virus obtained through the CDC Influenza Division. This strain was detected by BioCode RPP as Flu A (no subtype) at 50xLoD. In silico analysis did not reveal any mismatch in the Flu A matrix assay primers and probe binding regions but showed 3 mismatches in the Flu A/H3 subtyping HA assay primers binding regions that could predict reduced analytical reactivity, 1 mismatch at the ${ \mathfrak { g } } ^ { { \mathrm { t h } } }$ position from the ${ \boldsymbol { 3 ^ { \prime } } }$ end of the reverse primer (mismatch #1), 1 mismatch at the $1 8 ^ { \mathrm { t h } }$ position from the ${ \boldsymbol { 3 ^ { \prime } } }$ end of the reverse primer (mismatch #2), and 1 mismatch at the ${ 2 0 ^ { \mathrm { { t h } } } }$ position from the ${ \boldsymbol { 3 ^ { \prime } } }$ end of the reverse primer (mismatch #3). Wet testing data suggested that mismatch #1 is likely the root cause for the observed significant reduction in analytical reactivity for this strain. For patient samples contain a Flu $\mathsf { A } / \mathsf { H } 3$ strain that harbors mismatch #1 at lower concentrations, the BioCode RPP will likely report a Flu A (no subtype detected) result. Although estimated prevalence of a sequence variant based solely on in silico analysis may not accurately reflect the actual prevalence of the sequence variant in circulation during an influenza season, based on an in silico analysis, of all the Flu $\mathsf { A } / \mathsf { H } 3$ strains isolated in 2019 with published HA sequences, $7 3 . 8 \%$ of the strains harbor mismatch #1.

Table. Influenza B isolates tested during the inclusivity study   

<table><tr><td rowspan=1 colspan=1>Organism/Lineage</td><td rowspan=1 colspan=1>Location/Strain/ Year</td><td rowspan=1 colspan=1>Vendor</td><td rowspan=1 colspan=1>Catalog #</td><td rowspan=1 colspan=1>ConcentrationDetected</td><td rowspan=1 colspan=1>Multiple ofLoDDetecteda</td></tr><tr><td rowspan=7 colspan=1>Influenza B (Victoriab)</td><td rowspan=1 colspan=1>B/Malaysia/2506/2004d</td><td rowspan=1 colspan=1>BEI</td><td rowspan=1 colspan=1>NR-9723</td><td rowspan=1 colspan=1>1.458TCID5o/mL</td><td rowspan=1 colspan=1>0.03x</td></tr><tr><td rowspan=1 colspan=1>B/Malaysia/2506/2004</td><td rowspan=1 colspan=1>Zeptometrix</td><td rowspan=1 colspan=1>0810258CF</td><td rowspan=1 colspan=1>1.458TCID5o/mL</td><td rowspan=1 colspan=1>0.03x</td></tr><tr><td rowspan=1 colspan=1>B/Ohio/01/2005e</td><td rowspan=1 colspan=1>BEI</td><td rowspan=1 colspan=1>NR-41801</td><td rowspan=1 colspan=1>14.58CEID5o/mL</td><td rowspan=1 colspan=1>0.3x</td></tr><tr><td rowspan=1 colspan=1>B/Brisbane/33/2008f</td><td rowspan=1 colspan=1>BEI</td><td rowspan=1 colspan=1>NR-42006</td><td rowspan=1 colspan=1>14.58CEID50/mL</td><td rowspan=1 colspan=1>0.3x</td></tr><tr><td rowspan=1 colspan=1>B/Nevada/03/2011g</td><td rowspan=1 colspan=1>BEI</td><td rowspan=1 colspan=1>NR-44023</td><td rowspan=1 colspan=1>1.458CEID5o/mL</td><td rowspan=1 colspan=1>0.03x</td></tr><tr><td rowspan=1 colspan=1>B/Michigan/09/2011</td><td rowspan=1 colspan=1>CDC</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>14.58 EID50/mL</td><td rowspan=1 colspan=1>0.3x</td></tr><tr><td rowspan=1 colspan=1>B/Colorado/06/2017</td><td rowspan=1 colspan=1>CDC</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>14.58 EID50/mL</td><td rowspan=1 colspan=1>0.3x</td></tr><tr><td rowspan=7 colspan=1>Influenza B (Yamagata)</td><td rowspan=1 colspan=1>B/Texas/06/2011h</td><td rowspan=1 colspan=1>BEI</td><td rowspan=1 colspan=1>NR-44024</td><td rowspan=1 colspan=1>50 TCID50/mL</td><td rowspan=1 colspan=1>5000x h</td></tr><tr><td rowspan=1 colspan=1>B/Sydney/507/2006i</td><td rowspan=1 colspan=1>BEI</td><td rowspan=1 colspan=1>NR-36526</td><td rowspan=1 colspan=1>8.0 TCID50/mL</td><td rowspan=1 colspan=1>800xi</td></tr><tr><td rowspan=1 colspan=1>B/Wisconsin/1/10</td><td rowspan=1 colspan=1>Zeptometrix</td><td rowspan=1 colspan=1>0810241CF</td><td rowspan=1 colspan=1>0.03 TCID50/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>B/Massachusetts/2/12</td><td rowspan=1 colspan=1>Zeptometrix</td><td rowspan=1 colspan=1>0810239CF</td><td rowspan=1 colspan=1>0.03 TCID5o/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>B/Christchurch/33/2004i</td><td rowspan=1 colspan=1>BEI</td><td rowspan=1 colspan=1>NR-36536</td><td rowspan=1 colspan=1>0.03 TCID50/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>B/New Hampshire/01/2016k</td><td rowspan=1 colspan=1>CDC</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>10 EID50/mL</td><td rowspan=1 colspan=1>1000xk</td></tr><tr><td rowspan=1 colspan=1>B/Phuket/3073/2013|</td><td rowspan=1 colspan=1>CDC</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>5 EID5o/mL</td><td rowspan=1 colspan=1>500x&#x27;</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>B/lee/1940</td><td rowspan=1 colspan=1>Zeptometrix</td><td rowspan=1 colspan=1>0810257CF</td><td rowspan=1 colspan=1>0.03 TCID50/mL</td><td rowspan=1 colspan=1>3x</td></tr></table>

# Premarket Notification 510(k)

<table><tr><td rowspan=1 colspan=1>Organism/Lineage</td><td rowspan=1 colspan=1>Location/Strain/ Year</td><td rowspan=1 colspan=1>Vendor</td><td rowspan=1 colspan=1>Catalog #</td><td rowspan=1 colspan=1>ConcentrationDetected</td><td rowspan=1 colspan=1>Multiple ofLoDDetecteda</td></tr><tr><td rowspan=4 colspan=1>Influenza B (unknownlineagec)</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>B/Taiwan/2/1962</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>VR-295</td><td rowspan=1 colspan=1>5.0 CEID50/mL</td><td rowspan=1 colspan=1>500xm</td></tr><tr><td rowspan=1 colspan=1>B/Allen/45</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>VR-102</td><td rowspan=1 colspan=1>0.05 CEID50/mL</td><td rowspan=1 colspan=1>5x</td></tr><tr><td rowspan=1 colspan=1>B/Brigit</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>VR-786</td><td rowspan=1 colspan=1>0.03 TCID5o/mL</td><td rowspan=1 colspan=1>3x</td></tr></table>

a - If either FluB assay has MFI above the cutoff, the software will report as Detected for Influenza B b - For Victoria lineage strains testing started at $0 . 0 3 \times$ LoD rather than $_ { 3 \times }$ LoD.   
c - Strains of unknown lineages were assayed starting at $_ { 3 \times }$ LoD of Yamagata strain.   
d - Virus obtained through BEI Resources, NIAID, NIH: Influenza B, B/Malaysia/2506/2004, NR-9723.   
e - Virus obtained through BEI Resources, NIAID, NIH: Influenza B, B/Ohio/01/2005, NR-41801.   
f - Virus obtained through BEI Resources, NIAID, NIH: Influenza B, B/Brisbane/33/2008, NR-42006.   
g - Virus obtained through BEI Resources, NIAID, NIH: Influenza B, B/Nevada/03/2011, NR-44023.

h - Virus obtained through BEI Resources, NIAID, NIH: Influenza B, B/Texas/06/2011, NR-44024. In silico analysis does not predict reduced analytical reactivity. Titering inconsistency from the vendor rather than reduced reactivity due to assay design is suggested.

i - Virus obtained through BEI Resources, NIAID, NIH: Influenza B, B/Sydney/507/2006, NR-36526. In silico analysis does not predict reduced analytical reactivity. Titering inconsistency from the vendor rather than reduced reactivity due to assay design is suggested.

j - Virus obtained through BEI Resources, NIAID, NIH: Influenza B, B/Christchurch/33/2004, NR-36536.

k - Virus obtained through the CDC Influenza Division. In silico analysis did not reveal any critical mismatch in the Flu B NS1 assay primers and probe binding regions that would predict reduced analytical reactivity. And In silico analysis did not reveal any mismatch in the Flu B matrix assay primers and probe binding regions. Titering inconsistency from the source laboratory $( E | D _ { 5 0 } / \mathsf { m L }$ vs. $\mathsf { T C l D } _ { 5 0 } / \mathsf { m L }$ ) rather than reduced reactivity due to assay design is suggested.

l - Virus obtained through the CDC Influenza Division. In silico analysis did not reveal any critical mismatch in the Flu B NS1 assay primers and probe binding regions that would predict reduced analytical reactivity. And In silico analysis did not reveal any mismatch in the Flu B matrix assay primers and probe binding regions. Titering inconsistency from the source laboratory $( E | D _ { 5 0 } / \mathsf { m L }$ vs. $\mathsf { T C l D } _ { 5 0 } / \mathsf { m L }$ ) rather than reduced reactivity due to assay design is suggested.

m - In silico analysis could not be performed due to unavailability of sequence information for this strain. Titering inconsistency from the vendor rather than reduced reactivity due to assay design is suspected.

Table. Respiratory Syncytial Virus isolates tested during the inclusivity study   

<table><tr><td rowspan=1 colspan=1>Organism/Type</td><td rowspan=1 colspan=1>Strain/Location/Year</td><td rowspan=1 colspan=1>Vendor</td><td rowspan=1 colspan=1>Catalog #</td><td rowspan=1 colspan=1>ConcentrationDetected</td><td rowspan=1 colspan=1>Multiple ofLoDDetected</td></tr><tr><td rowspan=4 colspan=1>Respiratory SyncytialVirus Type A</td><td rowspan=1 colspan=1>TN/1998/3-2a</td><td rowspan=1 colspan=1>BEI</td><td rowspan=1 colspan=1>NR-28529</td><td rowspan=1 colspan=1>0.99 TCID50/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>TN/2000/3-4b</td><td rowspan=1 colspan=1>BEI</td><td rowspan=1 colspan=1>NR-28530</td><td rowspan=1 colspan=1>3.3 TCID5o/mL</td><td rowspan=1 colspan=1>10x</td></tr><tr><td rowspan=1 colspan=1>TN/98/12-21c</td><td rowspan=1 colspan=1>BEI</td><td rowspan=1 colspan=1>NR-28528</td><td rowspan=1 colspan=1>0.99 TCID50/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>Long/Maryland/1956</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>VR-26</td><td rowspan=1 colspan=1>3.3 TCID50/mL</td><td rowspan=1 colspan=1>10x</td></tr><tr><td rowspan=5 colspan=1>Respiratory SyncytialVirus Type B</td><td rowspan=1 colspan=1>9320/Massachusetts/1977</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>VR-955</td><td rowspan=1 colspan=1>0.99 TCID5o0/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>B1d</td><td rowspan=1 colspan=1>BEI</td><td rowspan=1 colspan=1>NR-4052</td><td rowspan=1 colspan=1>3.3 TCID50/mL</td><td rowspan=1 colspan=1>10x</td></tr><tr><td rowspan=1 colspan=1>WV/14617/85</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>VR-1400</td><td rowspan=1 colspan=1>0.99 TCID50/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>18537/WashingtonDC/1962</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>VR-1580</td><td rowspan=1 colspan=1>0.99 TCID5o/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>CH93(18)18</td><td rowspan=1 colspan=1>ZeptometriX</td><td rowspan=1 colspan=1>0810040CF</td><td rowspan=1 colspan=1>0.99 TCID50/mL</td><td rowspan=1 colspan=1>3x</td></tr></table>

a - Virus obtained through BEI Resources, NIAID, NIH: RSV, TN/1998/3-2, NR-28529 b - Virus obtained through BEI Resources, NIAID, NIH: RSV, TN/2000/3-4, NR-28530 c - Virus obtained through BEI Resources, NIAID, NIH: RSV, TN/98/12-21, NR-28528 d - Virus obtained through BEI Resources, NIAID, NIH: RSV, B1, NR-4052

Table. Human Metapneumovirus isolates tested during the inclusivity study   

<table><tr><td rowspan=1 colspan=1>Genotype</td><td rowspan=1 colspan=1>Serotype</td><td rowspan=1 colspan=1>Location/Year</td><td rowspan=1 colspan=1>Vendor</td><td rowspan=1 colspan=1>Catalog #</td><td rowspan=1 colspan=1>ConcentrationDetected</td><td rowspan=1 colspan=1>Multipleof LoDDetecteda</td></tr><tr><td rowspan=1 colspan=1>HumanMetapneumovirusType A1</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>lowa3/2002</td><td rowspan=1 colspan=1>Zeptometrix</td><td rowspan=1 colspan=1>0810160CF</td><td rowspan=1 colspan=1>45 TCID50/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>HumanMetapneumovirusType A2</td><td rowspan=1 colspan=1>27</td><td rowspan=1 colspan=1>lowa27/2004</td><td rowspan=1 colspan=1>Zeptometrix</td><td rowspan=1 colspan=1>0810164CF</td><td rowspan=1 colspan=1>45 TCID50/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=2 colspan=1>HumanMetapneumovirusType B1</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>Peru2/2002</td><td rowspan=1 colspan=1>Zeptometrix</td><td rowspan=1 colspan=1>0810156CF</td><td rowspan=1 colspan=1>45 TCID5o/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>Peru3/2003</td><td rowspan=1 colspan=1>Zeptometrix</td><td rowspan=1 colspan=1>0810158CF</td><td rowspan=1 colspan=1>45 TCID5o/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=4 colspan=1>HumanMetapneumovirusType B2</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>Peru1/2002</td><td rowspan=1 colspan=1>Zeptometrix</td><td rowspan=1 colspan=1>0810157CF</td><td rowspan=1 colspan=1>45 TCID5o/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>Peru6/2003</td><td rowspan=1 colspan=1>Zeptometrix</td><td rowspan=1 colspan=1>0810159CF</td><td rowspan=1 colspan=1>45 TCID5o/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>18</td><td rowspan=1 colspan=1>lowa18/2003</td><td rowspan=1 colspan=1>Zeptometrix</td><td rowspan=1 colspan=1>0810162CF</td><td rowspan=1 colspan=1>45 TCID50/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>Unknown</td><td rowspan=1 colspan=1>TN/91-316b</td><td rowspan=1 colspan=1>BEI</td><td rowspan=1 colspan=1>NR-22232</td><td rowspan=1 colspan=1>45 TCID5o/mL</td><td rowspan=1 colspan=1>3x</td></tr></table>

a - If either assay has MFI above the cutoff, the software will report as Detected for Human Metapneumovirus b - Virus obtained through BEI Resources, NIAID, NIH: Human Metapneumovirus, TN/91-316, NR-22232

Table. Parainfluenza Virus (1-4) isolates tested during inclusivity.   

<table><tr><td rowspan=1 colspan=1>Organism/Subtype</td><td rowspan=1 colspan=1>Strain/Location/Year</td><td rowspan=1 colspan=1>Vendor</td><td rowspan=1 colspan=1>Catalog #</td><td rowspan=1 colspan=1>ConcentrationDetected</td><td rowspan=1 colspan=1>Multiple ofLoDDetected</td></tr><tr><td rowspan=3 colspan=1>Parainfluenza Virus 1</td><td rowspan=1 colspan=1>FRA/27344044/2007a</td><td rowspan=1 colspan=1>BEI</td><td rowspan=1 colspan=1>NR-48681</td><td rowspan=1 colspan=1>27 TCID50/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>FRA/29221106/2009b</td><td rowspan=1 colspan=1>BEI</td><td rowspan=1 colspan=1>NR-48680</td><td rowspan=1 colspan=1>27 TCID50/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>Unknown</td><td rowspan=1 colspan=1>Zeptometrix</td><td rowspan=1 colspan=1>0810014CF</td><td rowspan=1 colspan=1>27 TCID50/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=2 colspan=1>Parainfluenza Virus 2</td><td rowspan=1 colspan=1>Greerc</td><td rowspan=1 colspan=1>BEI</td><td rowspan=1 colspan=1>NR-3229</td><td rowspan=1 colspan=1>5.4 TCID50/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>Unknown</td><td rowspan=1 colspan=1>Zeptometrix</td><td rowspan=1 colspan=1>0810015CF</td><td rowspan=1 colspan=1>5.4 TCID50/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=2 colspan=1>Parainfluenza Virus 3</td><td rowspan=1 colspan=1>NIH 47885,Wash/47885/57d</td><td rowspan=1 colspan=1>BEI</td><td rowspan=1 colspan=1>NR-3233</td><td rowspan=1 colspan=1>45 TCID5o/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>C243/WashingtonDC/1957</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>VR-93</td><td rowspan=1 colspan=1>45 TCID50/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza Virus 4a</td><td rowspan=1 colspan=1>M-25/1958e</td><td rowspan=1 colspan=1>BEI</td><td rowspan=1 colspan=1>NR-3237</td><td rowspan=1 colspan=1>27 TCID50/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=2 colspan=1>Parainfluenza Virus 4b</td><td rowspan=1 colspan=1>CH-19503/WashingtonDC/1962</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>VR-1377</td><td rowspan=1 colspan=1>27 TCID50/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>Unknown</td><td rowspan=1 colspan=1>Zeptometrix</td><td rowspan=1 colspan=1>0810060BCF</td><td rowspan=1 colspan=1>27 TCID50/mL</td><td rowspan=1 colspan=1>3x</td></tr></table>

a - Virus obtained through BEI Resources, NIAID, NIH: Parainfluenza Virus 1, HIPIV1/FRA/27344044/2007, NR-48681 b - Virus obtained through BEI Resources, NIAID, NIH: Parainfluenza Virus 1, HPIV1/FRA/29221106/2009, NR-48680 c - Virus obtained through BEI Resources, NIAID, NIH: Parainfluenza Virus 2, Greer, NR-3229 d - Virus obtained through BEI Resources, NIAID, NIH: Parainfluenza Virus 3, NIH 47885, NR-3233 e - Virus obtained through BEI Resources, NIAID, NIH: Parainfluenza Virus 4a, M-25, NR-3237

Table. Adenovirus isolates tested during the inclusivity study   

<table><tr><td colspan="1" rowspan="1">Speciesb</td><td colspan="1" rowspan="1">Serotype</td><td colspan="1" rowspan="1">Strain/Location/Year</td><td colspan="1" rowspan="1">Vendor</td><td colspan="1" rowspan="1">Catalog #</td><td colspan="1" rowspan="1">ConcentrationDetected</td><td colspan="1" rowspan="1">Multiple ofLoDDetectedc</td></tr><tr><td colspan="1" rowspan="3">Adenovirus Aa</td><td colspan="1" rowspan="1">31</td><td colspan="1" rowspan="1">Unknown</td><td colspan="1" rowspan="1">Zeptometrix</td><td colspan="1" rowspan="1">0810073CF</td><td colspan="1" rowspan="1">18 TCID50/mL</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="1">12</td><td colspan="1" rowspan="1">Huie/Massachusetts</td><td colspan="1" rowspan="1">ATCC</td><td colspan="1" rowspan="1">VR-863</td><td colspan="1" rowspan="1">18 TCID50/mL</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="1">18</td><td colspan="1" rowspan="1">Washington DC/1954</td><td colspan="1" rowspan="1">ATCC</td><td colspan="1" rowspan="1">VR-19</td><td colspan="1" rowspan="1">18 TCID50/mL</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="4">Adenovirus B</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">GB/Maryland/1953</td><td colspan="1" rowspan="1">ATCC</td><td colspan="1" rowspan="1">VR-3</td><td colspan="1" rowspan="1">3.6 TCID50/mL</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="1">14</td><td colspan="1" rowspan="1">Unknown</td><td colspan="1" rowspan="1">Zeptometrix</td><td colspan="1" rowspan="1">0810108CF</td><td colspan="1" rowspan="1">3.6 TCID50/mL</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="1">16</td><td colspan="1" rowspan="1">CH.79/SaudiArabia/1955</td><td colspan="1" rowspan="1">ATCC</td><td colspan="1" rowspan="1">VR-17</td><td colspan="1" rowspan="1">3.6 TCID50/mL</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="1">35</td><td colspan="1" rowspan="1">Holden</td><td colspan="1" rowspan="1">ATCC</td><td colspan="1" rowspan="1">VR-718</td><td colspan="1" rowspan="1">3.6 TCID50/mL</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="2">Adenovirus C</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Unknown</td><td colspan="1" rowspan="1">Zeptometrix</td><td colspan="1" rowspan="1">0810050CF</td><td colspan="1" rowspan="1">18 TCID50/mL</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">Unknown</td><td colspan="1" rowspan="1">Zeptometrix</td><td colspan="1" rowspan="1">0810020CF</td><td colspan="1" rowspan="1">18 TCID50/mL</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">Tonsil 99/WashingtonDC</td><td colspan="1" rowspan="1">ATCC</td><td colspan="1" rowspan="1">VR-6</td><td colspan="1" rowspan="1">18 TCID50/mL</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="5">Adenovirus Da</td><td colspan="1" rowspan="1">8</td><td colspan="1" rowspan="1">Unknown</td><td colspan="1" rowspan="1">Zeptometrix</td><td colspan="1" rowspan="1">0810069CF</td><td colspan="1" rowspan="1">18 TCID5o/mL</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="1">17</td><td colspan="1" rowspan="1">CH. 22/SaudiArabia/1955</td><td colspan="1" rowspan="1">ATCC</td><td colspan="1" rowspan="1">VR-1836</td><td colspan="1" rowspan="1">18 TCID5o/mL</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">Unknown</td><td colspan="1" rowspan="1">Zeptometrix</td><td colspan="1" rowspan="1">0810115CF</td><td colspan="1" rowspan="1">18 TCID50/mL</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="1">26</td><td colspan="1" rowspan="1">Unknown</td><td colspan="1" rowspan="1">Zeptometrix</td><td colspan="1" rowspan="1">0810117CF</td><td colspan="1" rowspan="1">18 TCID5o/mL</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="1">37</td><td colspan="1" rowspan="1">Unknown</td><td colspan="1" rowspan="1">Zeptometrix</td><td colspan="1" rowspan="1">0810119CF</td><td colspan="1" rowspan="1">18 TCID50/mL</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="2">Adenovirus Fa</td><td colspan="1" rowspan="1">40</td><td colspan="1" rowspan="1">Unknown</td><td colspan="1" rowspan="1">Zeptometrix</td><td colspan="1" rowspan="1">0810084CF</td><td colspan="1" rowspan="1">18 TCID5o/mL</td><td colspan="1" rowspan="1">3x</td></tr><tr><td colspan="1" rowspan="1">41</td><td colspan="1" rowspan="1">Tak/73-3544/Netherlands/1973</td><td colspan="1" rowspan="1">ATCC</td><td colspan="1" rowspan="1">VR-930</td><td colspan="1" rowspan="1">18 TCID50/mL</td><td colspan="1" rowspan="1">3x</td></tr></table>

a – Adenovirus C LoD used as LoD was not determined for Species A, D or F. b – In silico analysis predicts detection of Adenovirus species E serotypes (Adenovirus 4 strain was tested as a part of LoD studies as well. c – If either AdV assay has MFI above the cutoff the software will report as Detected for Adenovirus.

Table. Coronavirus isolates tested during the inclusivity study   

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Location/Year</td><td rowspan=1 colspan=1>Vendor</td><td rowspan=1 colspan=1>Catalog #</td><td rowspan=1 colspan=1>ConcentrationDetected</td><td rowspan=1 colspan=1>Multipleof LoDDetected</td></tr><tr><td rowspan=1 colspan=1>Coronavirus 229E</td><td rowspan=1 colspan=1>Unknown</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>VR-740</td><td rowspan=1 colspan=1>1.8 TCID5o/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>Coronavirus NL63</td><td rowspan=1 colspan=1>Amsterdam/2003</td><td rowspan=1 colspan=1>BEIb</td><td rowspan=1 colspan=1>NR-470</td><td rowspan=1 colspan=1>0.12 TCID50/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>Coronavirus OC43</td><td rowspan=1 colspan=1>Unknown</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>VR-1558</td><td rowspan=1 colspan=1>0.12 TCID5o/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=3 colspan=1>HKU1a</td><td rowspan=1 colspan=1>Unknown</td><td rowspan=1 colspan=1>Clinical Sample 5016</td><td rowspan=1 colspan=1>Unknown</td><td rowspan=1 colspan=1>1.51x105 copies/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>Unknown</td><td rowspan=1 colspan=1>Clinical Sample 5036</td><td rowspan=1 colspan=1>Unknown</td><td rowspan=1 colspan=1>1.51x105 copies/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>Unknown</td><td rowspan=1 colspan=1>Clinical Sample 5037</td><td rowspan=1 colspan=1>Unknown</td><td rowspan=1 colspan=1>5.02x105 copies/mL</td><td rowspan=1 colspan=1>10x</td></tr></table>

a - Coronavirus HKU1 clinical samples titered with Applied BioCode validated SYBR assay using an IVT standard. b - Virus obtained through BEI Resources, NIAID, NIH: Human Coronavirus NL63, NR-470

Table. Human Rhinovirus and Enterovirus isolates tested during the inclusivity study   

<table><tr><td rowspan=1 colspan=1>Organism/Species</td><td rowspan=1 colspan=1>Serotype/Strain/Isolate</td><td rowspan=1 colspan=1>Vendor</td><td rowspan=1 colspan=1>Catalog #</td><td rowspan=1 colspan=1>ConcentrationDetected</td><td rowspan=1 colspan=1>Multiple ofLoDDetected</td></tr><tr><td rowspan=9 colspan=1>Rhinovirus A</td><td rowspan=1 colspan=1>Serotype 7/[68-CV11]</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>VR-1601</td><td rowspan=1 colspan=1>3.6 TCID5o/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>Serotype 16 [1A]</td><td rowspan=1 colspan=1>Zeptometrix</td><td rowspan=1 colspan=1>0810285CF</td><td rowspan=1 colspan=1>3.6 TCID50/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>Serotype 16 [11757/DC/1960]</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>VR-283</td><td rowspan=1 colspan=1>3.6 TCID5o/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>Serotype 34 [137-3]</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>VR-1365</td><td rowspan=1 colspan=1>3.6 TCID5o/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>Serotype 57 [Ch47}</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>VR-1600</td><td rowspan=1 colspan=1>3.6 TCID5o/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>Serotype 77 [130-63]</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>VR-1187</td><td rowspan=1 colspan=1>3.6 TCID50/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>Serotype 80</td><td rowspan=1 colspan=1>Zeptometrix</td><td rowspan=1 colspan=1>0810288CF</td><td rowspan=1 colspan=1>3.6 TCID50/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>Serotype 85 [50-525-CV54]</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>VR-1195</td><td rowspan=1 colspan=1>3.6 TCID5o/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>Serotype 95 [SF-998}</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>VR-1301</td><td rowspan=1 colspan=1>3.6 TCID50/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=4 colspan=1>Rhinovirus B</td><td rowspan=1 colspan=1>Serotype 3 [FEB]</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>VR-483</td><td rowspan=1 colspan=1>3.6 TCID5o/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>Serotype 14</td><td rowspan=1 colspan=1>Zeptometrix</td><td rowspan=1 colspan=1>0810284CF</td><td rowspan=1 colspan=1>3.6 TCID5o/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>Serotype 42</td><td rowspan=1 colspan=1>Zeptometrix</td><td rowspan=1 colspan=1>0810286CF</td><td rowspan=1 colspan=1>3.6 TCID5o/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>Serotype 70</td><td rowspan=1 colspan=1>Zeptometrix</td><td rowspan=1 colspan=1>0810287CF</td><td rowspan=1 colspan=1>3.6 TCID50/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>Enterovirus 71</td><td rowspan=1 colspan=1>EV 71</td><td rowspan=1 colspan=1>Zeptometrix</td><td rowspan=1 colspan=1>0810236CF</td><td rowspan=1 colspan=1>9.0 TCID50/mL</td><td rowspan=1 colspan=1>3x</td></tr></table>

Table. Bordetella pertussis isolates tested during the inclusivity study   

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Strain/Isolate</td><td rowspan=1 colspan=1>Vendor</td><td rowspan=1 colspan=1>Catalog #</td><td rowspan=1 colspan=1>ConcentrationDetected</td><td rowspan=1 colspan=1>Multipleof LoDDetected</td></tr><tr><td rowspan=6 colspan=1>Bordetella pertussis</td><td rowspan=1 colspan=1>F</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>8467</td><td rowspan=1 colspan=1>45 CFU/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>5[17921]</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>9340</td><td rowspan=1 colspan=1>45 CFU/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>10-536</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>10380</td><td rowspan=1 colspan=1>45 CFU/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>CNCTC Hp 12/63[623]</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>51445</td><td rowspan=1 colspan=1>45 CFU/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>Tohama 1</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>BAA-589</td><td rowspan=1 colspan=1>45 CFU/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>MN2531</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>BAA-1335</td><td rowspan=1 colspan=1>45 CFU/mL</td><td rowspan=1 colspan=1>3x</td></tr></table>

Table. Mycoplasma pneumoniae isolates tested during the inclusivity study   

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Strain/Isolate</td><td rowspan=1 colspan=1>Vendor</td><td rowspan=1 colspan=1>Catalog #</td><td rowspan=1 colspan=1>ConcentrationDetected</td><td rowspan=1 colspan=1>Multipleof LoDDetected</td></tr><tr><td rowspan=4 colspan=1>Mycoplasma pneumoniae</td><td rowspan=1 colspan=1>M129-B7</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>29342</td><td rowspan=1 colspan=1>45 CFU/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>PI 1428</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>29085</td><td rowspan=1 colspan=1>45 CCU/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>Mac</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>15492</td><td rowspan=1 colspan=1>45 CFU/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>UTMB-10P</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>49894</td><td rowspan=1 colspan=1>45 CCU/mL</td><td rowspan=1 colspan=1>3x</td></tr></table>

Table. Chlamydia pneumoniae isolates tested during the inclusivity study   

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Strain/Isolate</td><td rowspan=1 colspan=1>Vendor</td><td rowspan=1 colspan=1>Catalog #</td><td rowspan=1 colspan=1>ConcentrationDetected</td><td rowspan=1 colspan=1>Multipleof LoDDetected</td></tr><tr><td rowspan=2 colspan=1>Chlamydia pneumoniae</td><td rowspan=1 colspan=1>CM-1</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>VR-1360</td><td rowspan=1 colspan=1>50.1 CFU/mL</td><td rowspan=1 colspan=1>3x</td></tr><tr><td rowspan=1 colspan=1>CWL-029</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>VR-1310</td><td rowspan=1 colspan=1>50.1 CFU/mL</td><td rowspan=1 colspan=1>3x</td></tr></table>

Conclusion: This inclusivity testing suggests that the BioCode RPP is analytically reactive for the targeted organisms. BioCode RPP assay displayed good inclusivity for related organisms with similar genetic lineage/sequence.

# Zoonotic Influenza A Analytical Reactivity/Inclusivity Testing

Due to public health concerns related to zoonotic transmission of influenza A viruses to humans (primarily swine and avian lineages), serial dilutions of the following isolates of swine and avian influenza A viruses (viral nucleic acids) were also tested assessing analytical reactivity:

A/Japan/305/57 (H2N2)   
A/duck/Pennsylvania/10218/1984 (H5N2)   
A/turkey/Wisconsin/1/1966 (H9N2)   
A/Anhui/1/2013 (H7N9)   
A/Hubei/1/2010 (H5N1)   
A/Minnesota/11/2010 (H3N2v)

Consistent with the in silico predictions of analytical reactivity to zoonotic influenza A viruses, each nonseasonal influenza A strain tested (H2N2, H5N2, H9N2, H7N9, and H5N1), except for the H3N2v strain, was detected as Influenza A (no subtype) at various concentrations tested (i.e., detected by the BioCode RPP Influenza A matrix assay only). For the H3N2v strain, it was detected as Influenza $\mathsf { A } / \mathsf { H } 3$ at higher concentrations but was detected as Influenza A (no subtype) at the lowest concentration tested.

Analytical testing could not be performed to assess reactivity to H5N8, H1N2v and H1N1 swine viruses due to the lack of availability of such zoonotic influenza A strains for wet testing. Based on an in silico analysis, H5N8 (KP739416), H1N2v (MK239077), and H1N1 swine (KP822962.1) were predicted to be detected as Influenza A (no subtype), Influenza A/H1 (with reduced sensitivity for the A/H1 assay), and Influenza A (no subtype), respectively.

# Analytical Specificity/Cross Reactivity

A study was performed to verify that the BioCode® Respiratory Pathogen Panel (RPP) does not detect DNA or RNA from off-panel organisms commonly found in respiratory specimens or from organisms that can cause similar clinical symptoms. In addition, on-panel organisms were tested at high concentrations to ensure there is no cross-reactivity with other on-panel targets. This study tested a panel of titered stocks of relevant organisms. Organisms that were not available for wet testing were analyzed in silico comparing the whole organism sequence against all primers to assess potential for cross reactivity. Analysis was conducted using BLASTn and Primer-BLAST programs. All testing was performed at Applied BioCode®, Inc. Microorganisms were tested at $1 0 ^ { 6 } C F \mathsf { U / m L }$ for bacteria or fungi and $1 0 ^ { 5 } \mathsf { T C l D } _ { 5 0 } / \mathsf { m L }$ for virus or higher when possible. Stocks were combined with simulated NPS in UTM matrix at the time of extraction. Each organism was extracted in triplicate on the EasyMag and assayed in singlet with the RPP on the BioCode® MDx-3000 system according to the instructions for use. For each concentration tested, the number of replicates that gave valid results per the Interpretation Algorithm was determined. If any replicates were detected, testing was repeated from 5 additional extractions assayed in singlet. If detected after repeat with 5 additional replicates, serial dilutions were performed to determine the lower limit.

Table. Off-panel bacteria and fungi analyzed for analytical specificity (Cross reactivity)   

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Vendor</td><td rowspan=1 colspan=1>Catalog #</td><td rowspan=1 colspan=1>Titer tested</td><td rowspan=1 colspan=1>Cross-reactivity(Y/N)</td></tr><tr><td rowspan=1 colspan=1>Acinetobacter baumannii</td><td rowspan=1 colspan=1>Zeptometrix</td><td rowspan=1 colspan=1>801597</td><td rowspan=1 colspan=1>9.67 x 106 CFU/mL</td><td rowspan=1 colspan=1>N</td></tr><tr><td rowspan=1 colspan=1>Aspergillus flavus</td><td rowspan=1 colspan=1>Zeptometrix</td><td rowspan=1 colspan=1>801598</td><td rowspan=1 colspan=1>1.72 x 106 CFU/mL</td><td rowspan=1 colspan=1>N</td></tr><tr><td rowspan=1 colspan=1>Bordetella bronchiseptica</td><td rowspan=1 colspan=1>Zeptometrix</td><td rowspan=1 colspan=1>801649</td><td rowspan=1 colspan=1>6.68 x 107 CFU/mL</td><td rowspan=1 colspan=1>N</td></tr><tr><td rowspan=1 colspan=1>Bordetella holmesii</td><td rowspan=1 colspan=1>Zeptometrix</td><td rowspan=1 colspan=1>801464</td><td rowspan=1 colspan=1>3.83 x 10° CFU/mL</td><td rowspan=1 colspan=1>Ya</td></tr><tr><td rowspan=1 colspan=1>Bordetella parapertussis</td><td rowspan=1 colspan=1>Zeptometrix</td><td rowspan=1 colspan=1>8011461</td><td rowspan=1 colspan=1>1.00 x 106 CFU/mL</td><td rowspan=1 colspan=1>N</td></tr><tr><td rowspan=1 colspan=1>Burkholderia cepacia</td><td rowspan=1 colspan=1>Zeptometrix</td><td rowspan=1 colspan=1>801584</td><td rowspan=1 colspan=1>4.13 x 107 CFU/mL</td><td rowspan=1 colspan=1>N</td></tr><tr><td rowspan=1 colspan=1>Candida albicans</td><td rowspan=1 colspan=1>Zeptometrix</td><td rowspan=1 colspan=1>801504</td><td rowspan=1 colspan=1>1.96 x 106 CFU/mL</td><td rowspan=1 colspan=1>N</td></tr><tr><td rowspan=1 colspan=1>Candida glabrata</td><td rowspan=1 colspan=1>Zeptometrix</td><td rowspan=1 colspan=1>801535</td><td rowspan=1 colspan=1>1.73 x 107 CFU/mL</td><td rowspan=1 colspan=1>N</td></tr><tr><td rowspan=1 colspan=1>Corynebacterium diphtheriae</td><td rowspan=1 colspan=1>Zeptometrix</td><td rowspan=1 colspan=1>801882</td><td rowspan=1 colspan=1>4.57 x 106 CFU/mL</td><td rowspan=1 colspan=1>N</td></tr><tr><td rowspan=1 colspan=1>Haemophilus influenzae</td><td rowspan=1 colspan=1>Zeptometrix</td><td rowspan=1 colspan=1>801679</td><td rowspan=1 colspan=1>2.40 x 106 CFU/mL</td><td rowspan=1 colspan=1>N</td></tr><tr><td rowspan=1 colspan=1>Klebsiella pneumoniae</td><td rowspan=1 colspan=1>Zeptometrix</td><td rowspan=1 colspan=1>801506</td><td rowspan=1 colspan=1>5.10 x 107 CFU/mL</td><td rowspan=1 colspan=1>N</td></tr><tr><td rowspan=1 colspan=1>Lactobacillus plantarum</td><td rowspan=1 colspan=1>Zeptometrix</td><td rowspan=1 colspan=1>801507</td><td rowspan=1 colspan=1>1.80 x 107 CFU/mL</td><td rowspan=1 colspan=1>N</td></tr><tr><td rowspan=1 colspan=1>Legionella longbeachae</td><td rowspan=1 colspan=1>Zeptometrix</td><td rowspan=1 colspan=1>8101577</td><td rowspan=1 colspan=1>1.93 x 107 CFU/mL</td><td rowspan=1 colspan=1>N</td></tr><tr><td rowspan=1 colspan=1>Legionella micdadei</td><td rowspan=1 colspan=1>Zeptometrix</td><td rowspan=1 colspan=1>801576</td><td rowspan=1 colspan=1>2.70 x 107 CFU/mL</td><td rowspan=1 colspan=1>N</td></tr><tr><td rowspan=1 colspan=1>Legionella pneumophila</td><td rowspan=1 colspan=1>Zeptometrix</td><td rowspan=1 colspan=1>801645</td><td rowspan=1 colspan=1>3.17 x 107 CFU/mL</td><td rowspan=1 colspan=1>N</td></tr><tr><td rowspan=1 colspan=1>Moraxella catarrhalis</td><td rowspan=1 colspan=1>Zeptometrix</td><td rowspan=1 colspan=1>801509</td><td rowspan=1 colspan=1>1.13 x 106 CFU/mL</td><td rowspan=1 colspan=1>N</td></tr><tr><td rowspan=1 colspan=1>Mycobacterium tuberculosis</td><td rowspan=1 colspan=1>Zeptometrix</td><td rowspan=1 colspan=1>801660</td><td rowspan=1 colspan=1>7.23 x 106 CFU/mL</td><td rowspan=1 colspan=1>N</td></tr><tr><td rowspan=1 colspan=1>Mycoplasma genitalium</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>33530</td><td rowspan=1 colspan=1>1.00 x 103 CFU/mL</td><td rowspan=1 colspan=1>N</td></tr><tr><td rowspan=1 colspan=1>Mycoplasma hominis</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>23114</td><td rowspan=1 colspan=1>2.7 x 104 CFU/mL</td><td rowspan=1 colspan=1>N</td></tr><tr><td rowspan=1 colspan=1>Neisseria elongata</td><td rowspan=1 colspan=1>Zeptometrix</td><td rowspan=1 colspan=1>801510</td><td rowspan=1 colspan=1>1.74 x 107 CFU/mL</td><td rowspan=1 colspan=1>N</td></tr><tr><td rowspan=1 colspan=1>Neisseria gonorrhoeae</td><td rowspan=1 colspan=1>Zeptometrix</td><td rowspan=1 colspan=1>801482</td><td rowspan=1 colspan=1>1.26 x 107 CFU/mL</td><td rowspan=1 colspan=1>N</td></tr><tr><td rowspan=1 colspan=1>Neisseria meningitidis</td><td rowspan=1 colspan=1>Zeptometrix</td><td rowspan=1 colspan=1>801511</td><td rowspan=1 colspan=1>2.55 x 106 CFU/mL</td><td rowspan=1 colspan=1>N</td></tr><tr><td rowspan=1 colspan=1>Neisseria sicca</td><td rowspan=1 colspan=1>Zeptometrix</td><td rowspan=1 colspan=1>801754</td><td rowspan=1 colspan=1>1.02 x 106 CFU/mL</td><td rowspan=1 colspan=1>N</td></tr><tr><td rowspan=1 colspan=1>Proteus vulgaris</td><td rowspan=1 colspan=1>Zeptometrix</td><td rowspan=1 colspan=1>801898</td><td rowspan=1 colspan=1>4.13 x 107 CFU/mL</td><td rowspan=1 colspan=1>N</td></tr><tr><td rowspan=1 colspan=1>Pseudomonas aeruginosa</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>39324</td><td rowspan=1 colspan=1>2.11 x 106 CFU/mL</td><td rowspan=1 colspan=1>N</td></tr><tr><td rowspan=1 colspan=1>Serratia marcescens</td><td rowspan=1 colspan=1>Zeptometrix</td><td rowspan=1 colspan=1>801723</td><td rowspan=1 colspan=1>2.06 x 107 CFU/mL</td><td rowspan=1 colspan=1>N</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus haemolyticus</td><td rowspan=1 colspan=1>Zeptometrix</td><td rowspan=1 colspan=1>801591</td><td rowspan=1 colspan=1>8.50 x 106 CFU/mL</td><td rowspan=1 colspan=1>N</td></tr><tr><td rowspan=1 colspan=1>Streptococcus agalactiae</td><td rowspan=1 colspan=1>Zeptometrix</td><td rowspan=1 colspan=1>801545</td><td rowspan=1 colspan=1>3.73 x 106 CFU/mL</td><td rowspan=1 colspan=1>N</td></tr><tr><td rowspan=1 colspan=1>Streptococcus dysgalactiae</td><td rowspan=1 colspan=1>Zeptometrix</td><td rowspan=1 colspan=1>801516</td><td rowspan=1 colspan=1>1.23 x 106 CFU/mL</td><td rowspan=1 colspan=1>N</td></tr><tr><td rowspan=1 colspan=1>Streptococcus intermedius</td><td rowspan=1 colspan=1>Zeptometrix</td><td rowspan=1 colspan=1>801895</td><td rowspan=1 colspan=1>5.07 x 106 CFU/mL</td><td rowspan=1 colspan=1>N</td></tr><tr><td rowspan=1 colspan=1>Streptococcus mitis</td><td rowspan=1 colspan=1>Zeptometrix</td><td rowspan=1 colspan=1>801695</td><td rowspan=1 colspan=1>5.73 x 106 CFU/mL</td><td rowspan=1 colspan=1>N</td></tr><tr><td rowspan=1 colspan=1>Streptococcus pneumoniae</td><td rowspan=1 colspan=1>Zeptometrix</td><td rowspan=1 colspan=1>801439</td><td rowspan=1 colspan=1>4.17 x 106 CFU/mL</td><td rowspan=1 colspan=1>N</td></tr><tr><td rowspan=1 colspan=1>Ureaplasma urealyticum</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>27618</td><td rowspan=1 colspan=1>1.00 x 107 CFU/mL</td><td rowspan=1 colspan=1>N</td></tr></table>

- Bordetella holmesii was detected by the Bordetella pertussis (BP) assay with 2 of 3 replicates down to $3 . 8 3 \times 1 0 ^ { 0 } \mathsf { C F U / m L }$ .

Table. Off-panel Viruses analyzed for analytical specificity (Cross reactivity)   

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Vendor</td><td rowspan=1 colspan=1>Catalog #</td><td rowspan=1 colspan=1>Titer tested</td><td rowspan=1 colspan=1>Cross-reactivity(Y/N)</td></tr><tr><td rowspan=1 colspan=1>SARS-CoV, formaldehyde- and UV-inactivated,purified (vaccine)</td><td rowspan=1 colspan=1>BEI</td><td rowspan=1 colspan=1>NR-3883</td><td rowspan=1 colspan=1>1:100 Dilution</td><td rowspan=1 colspan=1>Na</td></tr><tr><td rowspan=1 colspan=1>MERS-CoV genomic RNA</td><td rowspan=1 colspan=1>BEI</td><td rowspan=1 colspan=1>NR-45843</td><td rowspan=1 colspan=1>1.01 x 107 Copies/mL</td><td rowspan=1 colspan=1>N</td></tr><tr><td rowspan=1 colspan=1>MERS-CoV EMC/2012, Heat-Inactivated</td><td rowspan=1 colspan=1>BEI</td><td rowspan=1 colspan=1>NR-50171</td><td rowspan=1 colspan=1>2 x 105 TCID50/ml</td><td rowspan=1 colspan=1>N</td></tr><tr><td rowspan=1 colspan=1>Coxsackievirus A10</td><td rowspan=1 colspan=1>Zeptometrix</td><td rowspan=1 colspan=1>0810106CF</td><td rowspan=1 colspan=1>1.05 x 10³ TCID50/mL</td><td rowspan=1 colspan=1>Yb</td></tr><tr><td rowspan=1 colspan=1>Coxsackievirus A21</td><td rowspan=1 colspan=1>Zeptometrix</td><td rowspan=1 colspan=1>0810235CF</td><td rowspan=1 colspan=1>≤ 1.03 x 10² TCID50/mL</td><td rowspan=1 colspan=1>Yb</td></tr><tr><td rowspan=1 colspan=1>Coxsackievirus A24</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>VR-583</td><td rowspan=1 colspan=1>1.14 x 101 TCID50/mL</td><td rowspan=1 colspan=1>Yb$</td></tr><tr><td rowspan=1 colspan=1>Coxsackievirus B2</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>VR-29</td><td rowspan=1 colspan=1>5.62 x 10³ TCID50/mL</td><td rowspan=1 colspan=1>Yb$</td></tr><tr><td rowspan=1 colspan=1>Coxsackievirus B3</td><td rowspan=1 colspan=1>Zeptometrix</td><td rowspan=1 colspan=1>0810074CF</td><td rowspan=1 colspan=1>1.76 x 10³ TCID50/mL</td><td rowspan=1 colspan=1>Yb$</td></tr><tr><td rowspan=1 colspan=1>Coxsackievirus B4</td><td rowspan=1 colspan=1>Zeptometrix</td><td rowspan=1 colspan=1>0810075CF</td><td rowspan=1 colspan=1>1.36 x 104 TCID50/mL</td><td rowspan=1 colspan=1>Yb</td></tr><tr><td rowspan=1 colspan=1>Coxsackievirus B5</td><td rowspan=1 colspan=1>Zeptometrix</td><td rowspan=1 colspan=1>0810019CF</td><td rowspan=1 colspan=1>≤ 5.89 x 10² TCID50/mL</td><td rowspan=1 colspan=1>Yb</td></tr><tr><td rowspan=1 colspan=1>Coxsackievirus A9</td><td rowspan=1 colspan=1>Zeptometrix</td><td rowspan=1 colspan=1>0810017CF</td><td rowspan=1 colspan=1>1.38 x 10³ TCID50/mL</td><td rowspan=1 colspan=1>Yb</td></tr><tr><td rowspan=1 colspan=1>Cytomegalovirus</td><td rowspan=1 colspan=1>Zeptometrix</td><td rowspan=1 colspan=1>0810003CF</td><td rowspan=1 colspan=1>4.17 x 104 TCID50/mL</td><td rowspan=1 colspan=1>N</td></tr><tr><td rowspan=1 colspan=1>Echovirus 11</td><td rowspan=1 colspan=1>Zeptometrix</td><td rowspan=1 colspan=1>0810023CF</td><td rowspan=1 colspan=1>1.68 x 103³ TCID50/mL</td><td rowspan=1 colspan=1>Yc</td></tr><tr><td rowspan=1 colspan=1>Echovirus 30</td><td rowspan=1 colspan=1>Zeptometrix</td><td rowspan=1 colspan=1>0810078CF</td><td rowspan=1 colspan=1>≤ 1.95 x 10² TCID50/mL</td><td rowspan=1 colspan=1>Yc</td></tr><tr><td rowspan=1 colspan=1>Echovirus 6</td><td rowspan=1 colspan=1>Zeptometrix</td><td rowspan=1 colspan=1>0810076CF</td><td rowspan=1 colspan=1>1.09 x 104 TCID50/mL</td><td rowspan=1 colspan=1>Yc</td></tr><tr><td rowspan=1 colspan=1>Echovirus 9</td><td rowspan=1 colspan=1>Zeptometrix</td><td rowspan=1 colspan=1>0810077CF</td><td rowspan=1 colspan=1>1.07 x 101 TCID50/mL</td><td rowspan=1 colspan=1>Yc</td></tr><tr><td rowspan=1 colspan=1>Epstein-Barr Virus</td><td rowspan=1 colspan=1>Zeptometrix</td><td rowspan=1 colspan=1>0810008CF</td><td rowspan=1 colspan=1>3.43 x 106 TCID50/mL</td><td rowspan=1 colspan=1>N</td></tr><tr><td rowspan=1 colspan=1>Herpes Simplex Virus Type 1</td><td rowspan=1 colspan=1>Zeptometrix</td><td rowspan=1 colspan=1>0810187CF</td><td rowspan=1 colspan=1>9.12 x 106 TCID50/mL</td><td rowspan=1 colspan=1>N</td></tr><tr><td rowspan=1 colspan=1>Measles Virus</td><td rowspan=1 colspan=1>Zeptometrix</td><td rowspan=1 colspan=1>0810025CF</td><td rowspan=1 colspan=1>1.31 x 105 TCID50/mL</td><td rowspan=1 colspan=1>N</td></tr><tr><td rowspan=1 colspan=1>Mumps Virus</td><td rowspan=1 colspan=1>Zeptometrix</td><td rowspan=1 colspan=1>0810176CF</td><td rowspan=1 colspan=1>1.89 x 105 TCID50/mL</td><td rowspan=1 colspan=1>N</td></tr></table>

a – Inhibitory (no RNA-IC detected) at 1:10 dilution b – The Coxsackieviruses assayed here were detected by the Human Rhinovirus/Enterovirus (HRV) assay in at least 1 or the 3 replicates down to the concentrations indicated. c – The Echoviruses assayed here were detected by the Human Rhinovirus/Enterovirus (HRV) assay in at least 1 or the 3 replicates down to the concentrations indicated.

Table. On-panel organisms analyzed for analytical specificity (Cross reactivity).   

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Vendor</td><td rowspan=1 colspan=1>Catalog #</td><td rowspan=1 colspan=1>Titer tested</td><td rowspan=1 colspan=1>Cross-reactivity(Y/N)</td></tr><tr><td rowspan=1 colspan=1>Influenza A H1N1/New Caledonia/20/99</td><td rowspan=1 colspan=1>Zeptometrix</td><td rowspan=1 colspan=1>0810036CF</td><td rowspan=1 colspan=1>1.15 x 105 TCID50/mL</td><td rowspan=1 colspan=1>N</td></tr><tr><td rowspan=1 colspan=1>Influenza A H1N1 /NWS/33</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>VR-219</td><td rowspan=1 colspan=1>7.40 x 105 TCID50/mL</td><td rowspan=1 colspan=1>N</td></tr><tr><td rowspan=1 colspan=1>Influenza A H1N1 pdm09/California/07/09</td><td rowspan=1 colspan=1>Zeptometrix</td><td rowspan=1 colspan=1>0810165CF</td><td rowspan=1 colspan=1>1.31 x 105 TCID50/mL</td><td rowspan=1 colspan=1>N</td></tr><tr><td rowspan=1 colspan=1>Influenza A H3N2 /Wisconsin/67/05a</td><td rowspan=1 colspan=1>Zeptometrix</td><td rowspan=1 colspan=1>0810252CF</td><td rowspan=1 colspan=1>1.08 x 105 TCID50/mL</td><td rowspan=1 colspan=1>N</td></tr><tr><td rowspan=1 colspan=1>Influenza A H3N2/Alice</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>VR-776</td><td rowspan=1 colspan=1>1.43 x 106 TCID50/mL</td><td rowspan=1 colspan=1>N</td></tr><tr><td rowspan=1 colspan=1>Influenza B/Florida/4/2006 (Yamagata)</td><td rowspan=1 colspan=1>Zeptometrix</td><td rowspan=1 colspan=1>0810255CF</td><td rowspan=1 colspan=1>1.08 x 105 TCID50/mL</td><td rowspan=1 colspan=1>N</td></tr><tr><td rowspan=1 colspan=1>Influenza B/Hong Kong/S/1972 (Victoria)</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>VR-823</td><td rowspan=1 colspan=1>8.57 x 105 TCID50/mL</td><td rowspan=1 colspan=1>N</td></tr><tr><td rowspan=1 colspan=1>Respiratory Syncytial Virus (Type A)</td><td rowspan=1 colspan=1>Zeptometrix</td><td rowspan=1 colspan=1>0810040ACF</td><td rowspan=1 colspan=1>4.57 x 105 TCID50/mL</td><td rowspan=1 colspan=1>N</td></tr><tr><td rowspan=1 colspan=1>Human Metapneumovirus 16 (Type A1)</td><td rowspan=1 colspan=1>Zeptometrix</td><td rowspan=1 colspan=1>0810161CF</td><td rowspan=1 colspan=1>8.51 x 105 TCID50/mL</td><td rowspan=1 colspan=1>N</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza Virus 1</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>VR-94</td><td rowspan=1 colspan=1>1.60 x 105 TCID50/mL</td><td rowspan=1 colspan=1>N</td></tr></table>

Premarket Notification 510(k)   

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Vendor</td><td rowspan=1 colspan=1>Catalog #</td><td rowspan=1 colspan=1>Titer tested</td><td rowspan=1 colspan=1>Cross-reactivity(Y/N)</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza Virus 2</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>VR-92</td><td rowspan=1 colspan=1>1.35 x 105 TCID50/mL</td><td rowspan=1 colspan=1>N</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza Virus 3</td><td rowspan=1 colspan=1>Zeptometrix</td><td rowspan=1 colspan=1>0810016CF</td><td rowspan=1 colspan=1>3.39 x 105 TCID50/mL</td><td rowspan=1 colspan=1>N</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza Virus 4a</td><td rowspan=1 colspan=1>Zeptometrix</td><td rowspan=1 colspan=1>0810060CF</td><td rowspan=1 colspan=1>1.13 x 105 TCID50/mL</td><td rowspan=1 colspan=1>N</td></tr><tr><td rowspan=1 colspan=1>Adenovirus Species B Serotype 7A</td><td rowspan=1 colspan=1>Zeptometrix</td><td rowspan=1 colspan=1>0810021CF</td><td rowspan=1 colspan=1>5.83 x 105 TCID50/mL</td><td rowspan=1 colspan=1>N</td></tr><tr><td rowspan=1 colspan=1>Adenovirus Species C Serotype 2</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>AV-846</td><td rowspan=1 colspan=1>2.81 x 105 TCID50/mL</td><td rowspan=1 colspan=1>N</td></tr><tr><td rowspan=1 colspan=1>Adenovirus Species E Serotype 4</td><td rowspan=1 colspan=1>Zeptometrix</td><td rowspan=1 colspan=1>0810070CF</td><td rowspan=1 colspan=1>1.08 x 105 TCID50/mL</td><td rowspan=1 colspan=1>N</td></tr><tr><td rowspan=1 colspan=1>Coronavirus 229E</td><td rowspan=1 colspan=1>Zeptometrix</td><td rowspan=1 colspan=1>0810229CF</td><td rowspan=1 colspan=1>1.09 x 105 TCID50/mL</td><td rowspan=1 colspan=1>N</td></tr><tr><td rowspan=1 colspan=1>Coronavirus HKU1</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>Clinicalsamplea</td><td rowspan=1 colspan=1>1.92 x 105 TCID50/mL</td><td rowspan=1 colspan=1>N</td></tr><tr><td rowspan=1 colspan=1>Coronavirus NL63</td><td rowspan=1 colspan=1>Zeptometrix</td><td rowspan=1 colspan=1>0810228CF</td><td rowspan=1 colspan=1>1.08 x 105 TCID50/mL</td><td rowspan=1 colspan=1>N</td></tr><tr><td rowspan=1 colspan=1>Coronavirus OC43</td><td rowspan=1 colspan=1>Zeptometrix</td><td rowspan=1 colspan=1>0810024CF</td><td rowspan=1 colspan=1>1.08 x 105 TCID50/mL</td><td rowspan=1 colspan=1>N</td></tr><tr><td rowspan=1 colspan=1>Human Rhinovirus Type A1</td><td rowspan=1 colspan=1>Zeptometrix</td><td rowspan=1 colspan=1>0810012CF</td><td rowspan=1 colspan=1>1.05 x 105 TCID5o/mL</td><td rowspan=1 colspan=1>N</td></tr><tr><td rowspan=1 colspan=1>Enterovirus D68</td><td rowspan=1 colspan=1>Zeptometrix</td><td rowspan=1 colspan=1>0810300CF</td><td rowspan=1 colspan=1>1.08 x 105 TCID50/mL</td><td rowspan=1 colspan=1>N</td></tr><tr><td rowspan=1 colspan=1>Bordetella pertussis</td><td rowspan=1 colspan=1>Zeptometrix</td><td rowspan=1 colspan=1>801459</td><td rowspan=1 colspan=1>3.86 x 107 CFU/mL</td><td rowspan=1 colspan=1>N</td></tr><tr><td rowspan=1 colspan=1>Mycoplasma pneumoniae</td><td rowspan=1 colspan=1>Zeptometrix</td><td rowspan=1 colspan=1>801579</td><td rowspan=1 colspan=1>1.06 x 106 CCU/mL</td><td rowspan=1 colspan=1>N</td></tr><tr><td rowspan=1 colspan=1>Chlamydia pneumoniae (AR-39)</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>53592</td><td rowspan=1 colspan=1>1.24 x 106 CFU/mL</td><td rowspan=1 colspan=1>N</td></tr><tr><td rowspan=1 colspan=1>Chlamydia pneumoniae (CWL-029)</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>VR-1310</td><td rowspan=1 colspan=1>1.00 x 106 CFU/mL</td><td rowspan=1 colspan=1>N</td></tr></table>

a - Coronavirus HKU1 clinical samples titered with Applied BioCode validated SYBR assay using an IVT RNA standard.

Conclusion: Cross reactivity was not observed with the on-panel or off-panel microorganisms at the concentrations tested in this study except the following:

Empirical testing and in silico sequence analysis indicate that the Rhinovirus/Enterovirus assay (HRV) may also react with other Enterovirus species (i.e., Coxsackievirus and Echoviruses; see table for testing results).   
In silico sequence analysis indicate that the Bordetella pertussis assay (BP) may react with Bordetella holmesii and Bordetella bronchiseptica.

# Interfering Substances/Microbes

A study was performed to demonstrate the accuracy of the BioCode® Respiratory Pathogen Panel on the BioCode® MDx-3000 in the presence of potentially interfering substances or microorganisms. Each member of the interfering substance panel was added to contrived samples in simulated NPS (sNPS) in UTM matrix containing representative members of the BioCode RPP at 3x LoD and a negative sample comprised of only sNPS in UTM matrix. Each sample was tested with and without potentially interfering substances. Each sample was extracted in triplicate on both the easyMAG and MP96 extraction systems and tested singly with the RPP on the BioCode® MDx-3000 system. Concentrations of interferents were determined by reviewing results of previous RP clinical trials. Substances that produce interference at the original test concentration were tested at lower concentrations.

Table . Contrived samples (3x LoD in sNPS)   

<table><tr><td rowspan=1 colspan=1>Sample Name</td><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Source</td></tr><tr><td rowspan=2 colspan=1>RPP A</td><td rowspan=1 colspan=1>Adenovirus B Serotype 7A</td><td rowspan=1 colspan=1>Zeptometrix 0810060CF</td></tr><tr><td rowspan=1 colspan=1>Mycoplasma pneumoniae</td><td rowspan=1 colspan=1>Zeptometrix 801579</td></tr></table>

# Premarket Notification 510(k)

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Influenza A H3N2 A/Wisconsin/67/2005</td><td rowspan=1 colspan=1>Zeptometrix 0810252CF</td></tr><tr><td rowspan=3 colspan=1>RPP B</td><td rowspan=1 colspan=1>Respiratory Syncytial Virus (Type A)</td><td rowspan=1 colspan=1>Zeptometrix 0810040ACF</td></tr><tr><td rowspan=1 colspan=1>Influenza A H1N1/California/07/09</td><td rowspan=1 colspan=1>Zeptometrix 0810165CF</td></tr><tr><td rowspan=1 colspan=1>Human Metapneumovirus (16; type A1)</td><td rowspan=1 colspan=1>Zeptometrix 0810161CF</td></tr><tr><td rowspan=3 colspan=1>RPP C</td><td rowspan=1 colspan=1>Parainfluenza Virus 3</td><td rowspan=1 colspan=1>Zeptometrix 0810016CF</td></tr><tr><td rowspan=1 colspan=1>Coronavirus NL63</td><td rowspan=1 colspan=1>Zeptometrix 0810228CF</td></tr><tr><td rowspan=1 colspan=1>Influenza B/Florida/4/2006</td><td rowspan=1 colspan=1>Zeptometrix 0810255CF</td></tr><tr><td rowspan=1 colspan=1>HRV</td><td rowspan=1 colspan=1>Human Rhinovirus</td><td rowspan=1 colspan=1>Zeptometrix 0810012CF</td></tr></table>

Results: All targets of Samples RPP A, RPP B, RPP C and HRV were detected $( 3 / 3 )$ at the concentrations below, suggesting no interference from these potential interferents at the concentrations tested.

Table: Evaluation for microbial interferents on BioCode® RPP   

<table><tr><td rowspan=1 colspan=1>Microbial Interferent</td><td rowspan=1 colspan=1>Brand/Source</td><td rowspan=1 colspan=1>Concentration</td><td rowspan=1 colspan=1>InterferenceYes (Y) or No (N)</td></tr><tr><td rowspan=1 colspan=1>Streptococcus pneumoniae</td><td rowspan=1 colspan=1>Zeptometrix</td><td rowspan=1 colspan=1>1 X 106 CFU/mL</td><td rowspan=1 colspan=1>N</td></tr><tr><td rowspan=1 colspan=1>Haemophilus influenzae</td><td rowspan=1 colspan=1>Zeptometrix</td><td rowspan=1 colspan=1>1 X 106 CFU/mL</td><td rowspan=1 colspan=1>N</td></tr><tr><td rowspan=1 colspan=1>Neisseria meningitidis</td><td rowspan=1 colspan=1>Zeptometrix</td><td rowspan=1 colspan=1>1 X 106 CFU/mL</td><td rowspan=1 colspan=1>N</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus aureus</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>1 X 106 CFU/mL</td><td rowspan=1 colspan=1>N</td></tr><tr><td rowspan=1 colspan=1>Cytomegalovirus</td><td rowspan=1 colspan=1>Zeptometrix</td><td rowspan=1 colspan=1>1 X 105 TCID50/mL</td><td rowspan=1 colspan=1>N</td></tr></table>

Table: Non-microbial Interfering substances tested for BioCode® RPP assay   

<table><tr><td rowspan=1 colspan=1>Substance Interferent</td><td rowspan=1 colspan=1>Brand/Source</td><td rowspan=1 colspan=1>Concentration</td><td rowspan=1 colspan=1>InterferenceYes (Y) or No (N)</td></tr><tr><td rowspan=1 colspan=1>Genomic DNA</td><td rowspan=1 colspan=1>Promega</td><td rowspan=1 colspan=1>10 ng/μl</td><td rowspan=1 colspan=1>N</td></tr><tr><td rowspan=1 colspan=1>Mucin (MagNA Pure 96)</td><td rowspan=1 colspan=1>Sigma</td><td rowspan=1 colspan=1>0.6% W/V</td><td rowspan=1 colspan=1>N</td></tr><tr><td rowspan=1 colspan=1>Mucin (EasyMAG)a</td><td rowspan=1 colspan=1>Sigma</td><td rowspan=1 colspan=1>0.5% W/V</td><td rowspan=1 colspan=1>N</td></tr><tr><td rowspan=1 colspan=1>Human Blood</td><td rowspan=1 colspan=1>Poplar Health</td><td rowspan=1 colspan=1>1% V/V</td><td rowspan=1 colspan=1>N</td></tr><tr><td rowspan=1 colspan=1>Zanamivir</td><td rowspan=1 colspan=1>APExBIO</td><td rowspan=1 colspan=1>550 ng/mL</td><td rowspan=1 colspan=1>N</td></tr><tr><td rowspan=1 colspan=1>Oseltamivir</td><td rowspan=1 colspan=1>APExBIO</td><td rowspan=1 colspan=1>142 ng/mL</td><td rowspan=1 colspan=1>N</td></tr><tr><td rowspan=1 colspan=1>Nasal spray</td><td rowspan=1 colspan=1>Equate</td><td rowspan=1 colspan=1>1% V/V</td><td rowspan=1 colspan=1>N</td></tr><tr><td rowspan=1 colspan=1>Nasal decongestant spray</td><td rowspan=1 colspan=1>Bayer</td><td rowspan=1 colspan=1>1% V/V</td><td rowspan=1 colspan=1>N</td></tr><tr><td rowspan=1 colspan=1>Nasal Allergy spray (Fluticasone)</td><td rowspan=1 colspan=1>Equate</td><td rowspan=1 colspan=1>1.5% V/V</td><td rowspan=1 colspan=1>N</td></tr><tr><td rowspan=1 colspan=1>Petroleum Jelly</td><td rowspan=1 colspan=1>Equate</td><td rowspan=1 colspan=1>1% W/V</td><td rowspan=1 colspan=1>N</td></tr><tr><td rowspan=1 colspan=1>Analgesic Ointment</td><td rowspan=1 colspan=1>Vicks</td><td rowspan=1 colspan=1>1% W/V</td><td rowspan=1 colspan=1>N</td></tr><tr><td rowspan=1 colspan=1>Mupirocin</td><td rowspan=1 colspan=1>Alfa Aesar™</td><td rowspan=1 colspan=1>2% W/V</td><td rowspan=1 colspan=1>N</td></tr><tr><td rowspan=1 colspan=1>Tobramycin</td><td rowspan=1 colspan=1>MP Biomedicals,LLC</td><td rowspan=1 colspan=1>0.6 mg/mL</td><td rowspan=1 colspan=1>N</td></tr><tr><td rowspan=1 colspan=1>Bleach (10%)</td><td rowspan=1 colspan=1>VWR</td><td rowspan=1 colspan=1>5% V/V</td><td rowspan=1 colspan=1>N</td></tr><tr><td rowspan=1 colspan=1>Disinfecting wipes</td><td rowspan=1 colspan=1>Clorox</td><td rowspan=1 colspan=1>50% V/V</td><td rowspan=1 colspan=1>N</td></tr><tr><td rowspan=1 colspan=1>Ethanol (70%)</td><td rowspan=1 colspan=1>LabChem</td><td rowspan=1 colspan=1>7% V/V</td><td rowspan=1 colspan=1>N</td></tr><tr><td rowspan=1 colspan=1>Remel M4 Media</td><td rowspan=1 colspan=1>Remel</td><td rowspan=1 colspan=1>90% V/V</td><td rowspan=1 colspan=1>N</td></tr><tr><td rowspan=1 colspan=1>Remel M4-RT Media</td><td rowspan=1 colspan=1>Remel</td><td rowspan=1 colspan=1>90% V/V</td><td rowspan=1 colspan=1>N</td></tr><tr><td rowspan=1 colspan=1>Remel M5 Media</td><td rowspan=1 colspan=1>Remel</td><td rowspan=1 colspan=1>90% V/V</td><td rowspan=1 colspan=1>N</td></tr><tr><td rowspan=1 colspan=1>Remel M6 Media</td><td rowspan=1 colspan=1>Remel</td><td rowspan=1 colspan=1>90% V/V</td><td rowspan=1 colspan=1>N</td></tr><tr><td rowspan=1 colspan=1>Copan FloQ (Flocked nylon/plastic shaft)</td><td rowspan=1 colspan=1>Copan</td><td rowspan=1 colspan=1>1 swab</td><td rowspan=1 colspan=1>N</td></tr></table>

Premarket Notification 510(k)   

<table><tr><td rowspan=1 colspan=1>Substance Interferent</td><td rowspan=1 colspan=1>Brand/Source</td><td rowspan=1 colspan=1>Concentration</td><td rowspan=1 colspan=1>InterferenceYes (Y) or No (N)</td></tr><tr><td rowspan=1 colspan=1>Copan 168C (rayon/twisted aluminumshaft)</td><td rowspan=1 colspan=1>Copan</td><td rowspan=1 colspan=1>1 swab</td><td rowspan=1 colspan=1>N</td></tr><tr><td rowspan=1 colspan=1>Polyester / Aluminum shaft swab</td><td rowspan=1 colspan=1>Puritan/Copan</td><td rowspan=1 colspan=1>1 swab</td><td rowspan=1 colspan=1>N</td></tr><tr><td rowspan=1 colspan=1>DNAzap</td><td rowspan=1 colspan=1>Invitrogen</td><td rowspan=1 colspan=1>1% V/V</td><td rowspan=1 colspan=1>N</td></tr><tr><td rowspan=1 colspan=1>RnaseOut</td><td rowspan=1 colspan=1>Invitrogen</td><td rowspan=1 colspan=1>1% V/V</td><td rowspan=1 colspan=1>N</td></tr></table>

a - It was observed that mucin at higher concentration $( 0 . 6 \% )$ led to loss of signal for some targets (loss of analyte detection) when extracted with NucliSENS easyMAG.

Table: Nasal influenza vaccine (FluMist) tested for BioCode® RPP assay   

<table><tr><td rowspan=2 colspan=1>FluMist® 2010-2011 (V/V%)</td><td rowspan=1 colspan=3>Influenza A</td><td rowspan=2 colspan=1>Influenza B</td></tr><tr><td rowspan=1 colspan=1>H1</td><td rowspan=1 colspan=1>H1N1-2009</td><td rowspan=1 colspan=1>H3</td></tr><tr><td rowspan=1 colspan=1>10%</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td></tr><tr><td rowspan=1 colspan=1>1%</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td></tr><tr><td rowspan=1 colspan=1>0.1%</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td></tr><tr><td rowspan=1 colspan=1>0.01%</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td></tr><tr><td rowspan=1 colspan=1>0.001%</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td></tr><tr><td rowspan=1 colspan=1>0.0001%</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+a$</td><td rowspan=1 colspan=1>+</td></tr><tr><td rowspan=1 colspan=1>0.00001%</td><td rowspan=1 colspan=1>:</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>+a$</td></tr><tr><td rowspan=1 colspan=1>0.000001%</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td></tr></table>

a - 2/3 replicates detected

Conclusion: None of the substances were shown to interfere with BioCode® RPP at the concentrations tested. However, it was observed that mucin at higher concentration $( 0 . 6 \% )$ could lead to loss of signal for some targets (loss of analyte detection) when extracted with easyMAG. The effect of mucin was dependent on the concentration in the sample tested.

Flu Mist was evaluated to be reactive as predicted with the BioCode RPP assay, therefore recent administration or contamination of specimens by Flu vaccine prior to NPS collection could lead to false detection by BioCode® RPP.

# Competitive Inhibition

A study was performed to evaluate the potential for inhibition in samples with mixed infections. Targets were spiked into simulated NPS in UTM matrix with one target at high concentration $( \ge 1 0 ^ { 6 } \mathsf { C F U / m L }$ for bacteria and ${ \ge } 1 0 ^ { 5 }$ units/mL for viruses) and two targets at low concentration $( \leq 3 \times \mathsf { L o D } )$ . Common coinfections were determined by reviewing results of previous Respiratory Panel clinical trials from 510k summaries, publications/posters and internal clinical sample testing. Each sample was extracted in triplicate on the easyMAG and each extraction tested in singlet with the RPP on the BioCode® MDx-3000 system.

Results: Results are shown in the table below.

Table. Competitive inhibition testing results   

<table><tr><td rowspan=1 colspan=1>PanelDesignation</td><td rowspan=1 colspan=1>Viral/Bacteria Strain</td><td rowspan=1 colspan=1>Source</td><td rowspan=1 colspan=1>Level</td><td rowspan=1 colspan=1>Titer Tested</td><td rowspan=1 colspan=1>Result (n of3 Detected)</td></tr><tr><td rowspan=4 colspan=1>CompetitivenhibitionSample 1</td><td rowspan=1 colspan=1>Adenovirus species C Serotype 2</td><td rowspan=1 colspan=1>ATCC VR-846</td><td rowspan=1 colspan=1>High</td><td rowspan=1 colspan=1>1x105TCID50/mL</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Respiratory syncytial virus TypeA</td><td rowspan=1 colspan=1>Zeptometrix0810040ACF</td><td rowspan=1 colspan=1>Low</td><td rowspan=1 colspan=1>0.99 TCID5o/mL</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=2 colspan=1>Influenza A H3N2A/Wisconsin/67/05a</td><td rowspan=2 colspan=1>Zeptometrix0810252CF</td><td rowspan=2 colspan=1>Low</td><td rowspan=2 colspan=1>12 TCID50/mL</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=4 colspan=1>CompetitiveInhibitionSample 2</td><td rowspan=1 colspan=1>Respiratory syncytial virus TypeA</td><td rowspan=1 colspan=1>Zeptometrix0810040ACF</td><td rowspan=1 colspan=1>High</td><td rowspan=1 colspan=1>1x105 TCID50/mL</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=2 colspan=1>Influenza A H3N2A/Wisconsin/67/05a</td><td rowspan=2 colspan=1>Zeptometrix0810252CF</td><td rowspan=2 colspan=1>Low</td><td rowspan=2 colspan=1>12 TCID5o/mL</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Adenovirus species C Serotype 2</td><td rowspan=1 colspan=1>ATCC VR-846</td><td rowspan=1 colspan=1>Low</td><td rowspan=1 colspan=1>18 TCID50/mL</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=4 colspan=1>CompetitiveInhibitionSample 3</td><td rowspan=2 colspan=1>Influenza A H3N2A/Wisconsin/67/05a</td><td rowspan=2 colspan=1>Zeptometrix0810252CF</td><td rowspan=2 colspan=1>High</td><td rowspan=2 colspan=1>1x105TCID5o/mL</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Adenovirus species C Serotype 2</td><td rowspan=1 colspan=1>ATCC VR-846</td><td rowspan=1 colspan=1>Low</td><td rowspan=1 colspan=1>18 TCID5o/mL</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Respiratory syncytial virus TypeA</td><td rowspan=1 colspan=1>Zeptometrix0810040ACF</td><td rowspan=1 colspan=1>Low</td><td rowspan=1 colspan=1>0.99 TCID5o/mL</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=3 colspan=1>CompetitiveInhibitionSample 4</td><td rowspan=1 colspan=1>Coronavirus OC43</td><td rowspan=1 colspan=1>Zeptometrix0810024CF</td><td rowspan=1 colspan=1>High</td><td rowspan=1 colspan=1>1x105TCID5o/mL</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Human Metapneumovirus</td><td rowspan=1 colspan=1>Zeptometrix0810161CF</td><td rowspan=1 colspan=1>Low</td><td rowspan=1 colspan=1>45 TCID5o/mL</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Bordetella pertussis</td><td rowspan=1 colspan=1>Zeptometrix801459</td><td rowspan=1 colspan=1>Low</td><td rowspan=1 colspan=1>45 CFU/mL</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=3 colspan=1>CompetitiveInhibitionSample 5</td><td rowspan=1 colspan=1>Human Metapneumovirus</td><td rowspan=1 colspan=1>Zeptometrix080161CF</td><td rowspan=1 colspan=1>High</td><td rowspan=1 colspan=1>1x105TCID5o/mL</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Bordetella pertussis</td><td rowspan=1 colspan=1>Zeptometrix801459</td><td rowspan=1 colspan=1>Low</td><td rowspan=1 colspan=1>45 CFU/mL</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Coronavirus OC43</td><td rowspan=1 colspan=1>Zeptometrix0810024CF</td><td rowspan=1 colspan=1>Low</td><td rowspan=1 colspan=1>0.12 TCID5o/mL</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=3 colspan=1>CompetitiveInhibitionSample 6</td><td rowspan=1 colspan=1>Bordetella pertussis</td><td rowspan=1 colspan=1>Zeptometrix801459</td><td rowspan=1 colspan=1>High</td><td rowspan=1 colspan=1>1x106 CFU/mL</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Coronavirus OC43</td><td rowspan=1 colspan=1>Zeptometrix0810024CF</td><td rowspan=1 colspan=1>Low</td><td rowspan=1 colspan=1>0.12 TCID5o/mL</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Human Metapneumovirus</td><td rowspan=1 colspan=1>Zeptometrix0810161CF</td><td rowspan=1 colspan=1>Low</td><td rowspan=1 colspan=1>45 TCID50/mL</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=4 colspan=1>CompetitiveInhibitionSample 7</td><td rowspan=2 colspan=1>Influenza A H1N1 pdmCalifornia/07/09</td><td rowspan=2 colspan=1>Zeptometrix0810165CF</td><td rowspan=2 colspan=1>High</td><td rowspan=2 colspan=1>1x105TCID50/mL</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza Virus 3</td><td rowspan=1 colspan=1>Zeptometrix0810016CF</td><td rowspan=1 colspan=1>Low</td><td rowspan=1 colspan=1>45 TCID50/mL</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Human Rhinovirus type A</td><td rowspan=1 colspan=1>Zeptometrix0810012CFN</td><td rowspan=1 colspan=1>Low</td><td rowspan=1 colspan=1>3.6 TCID5o/mL</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=2 colspan=1>CompetitiveIhibitionSample 8</td><td rowspan=1 colspan=1>Parainfluenza Virus 3</td><td rowspan=1 colspan=1>Zeptometrix0810016CF</td><td rowspan=1 colspan=1>High</td><td rowspan=1 colspan=1>1x105TCID5o/mL</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Human Rhinovirus type A</td><td rowspan=1 colspan=1>Zeptometrix0810012CFN</td><td rowspan=1 colspan=1>Low</td><td rowspan=1 colspan=1>3.6 TCID50/mL</td><td rowspan=1 colspan=1>3/3</td></tr></table>

Premarket Notification 510(k)   

<table><tr><td rowspan=1 colspan=1>PanelDesignation</td><td rowspan=1 colspan=1>Viral/Bacteria Strain</td><td rowspan=1 colspan=1>Source</td><td rowspan=1 colspan=1>Level</td><td rowspan=1 colspan=1>Titer Tested</td><td rowspan=1 colspan=1>Result (n of3 Detected)</td></tr><tr><td rowspan=2 colspan=1></td><td rowspan=2 colspan=1>Influenza A H1N1 pdmCalifornia/07/09</td><td rowspan=2 colspan=1>Zeptometrix0810165CF</td><td rowspan=2 colspan=1>Low</td><td rowspan=2 colspan=1>1.2 TCID5o/mL</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=4 colspan=1>CompetitiveInhibitionSample 9</td><td rowspan=1 colspan=1>Human Rhinovirus type A</td><td rowspan=1 colspan=1>Zeptometrix0810012CFN</td><td rowspan=1 colspan=1>High</td><td rowspan=1 colspan=1>1x105TCID5o/mL</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=2 colspan=1>Influenza A H1N1 pdmCalifornia/07/09</td><td rowspan=2 colspan=1>Zeptometrix0810165CF</td><td rowspan=2 colspan=1>Low</td><td rowspan=2 colspan=1>1.2 TCID5o/mL</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza Virus 3</td><td rowspan=1 colspan=1>Zeptometrix0810016CF</td><td rowspan=1 colspan=1>Low</td><td rowspan=1 colspan=1>45 TCID5o/mL</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=3 colspan=1>CompetitiveInhbitionSample 10</td><td rowspan=1 colspan=1>Mycoplasma pneumoniae</td><td rowspan=1 colspan=1>Zeptometrix801579</td><td rowspan=1 colspan=1>High</td><td rowspan=1 colspan=1>1x106 CCU/mL</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Coronavirus NL63</td><td rowspan=1 colspan=1>Zeptometrix0810228CF</td><td rowspan=1 colspan=1>Low</td><td rowspan=1 colspan=1>1.2 TCID5o/mL</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Influenza B/Florida/4/2006</td><td rowspan=1 colspan=1>Zeptometrix0810255CF</td><td rowspan=1 colspan=1>Low</td><td rowspan=1 colspan=1>0.04 TCID5o/mL</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=3 colspan=1>CompetitiveInhibitionSample 11</td><td rowspan=1 colspan=1>Coronavirus NL63</td><td rowspan=1 colspan=1>Zeptometrix0810228CF</td><td rowspan=1 colspan=1>High</td><td rowspan=1 colspan=1>1x105TCID5o/mL</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Influenza B/Florida/4/2006</td><td rowspan=1 colspan=1>Zeptometrix0810255CF</td><td rowspan=1 colspan=1>Low</td><td rowspan=1 colspan=1>0.04 TCID5o/mL</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Mycoplasma pneumoniae</td><td rowspan=1 colspan=1>Zeptometrix801579</td><td rowspan=1 colspan=1>Low</td><td rowspan=1 colspan=1>45 CCU/mL</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=3 colspan=1>CompetitiveInhibitionSample 12</td><td rowspan=1 colspan=1>Influenza B/Florida/4/2006</td><td rowspan=1 colspan=1>Zeptometrix0810255CF</td><td rowspan=1 colspan=1>High</td><td rowspan=1 colspan=1>1x105TCID50/mL</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Mycoplasma pneumoniae</td><td rowspan=1 colspan=1>Zeptometrix801579</td><td rowspan=1 colspan=1>Low</td><td rowspan=1 colspan=1>45 CCU/mL</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Coronavirus NL63</td><td rowspan=1 colspan=1>Zeptometrix0810228CF</td><td rowspan=1 colspan=1>Low</td><td rowspan=1 colspan=1>1.2 TCID50/mL</td><td rowspan=1 colspan=1>3/3</td></tr></table>

Conclusion: All replicates from each pooled sample were valid and detected. No competitive inhibition was observed at the concentrations tested.

# Cross Contamination/Sample Carryover

Carry-over contamination studies have been performed for the BioCode® MDx-3000 system in conjunction with the easyMAG (K180041) and MagNA Pure 96 systems (K190585). Since this study is not assay-specific, no additional testing was performed for BioCode® RPP.

# Specimen Stability

A study was performed to assess the Specimen Stability limitations for the optimal performance of the BioCode® Respiratory Pathogen Panel (RPP) on the BioCode® MDx-3000. Representative organisms from the BioCode® RPP were spiked into prescreened negative natural NPS in UTM matrix at 3x LoD and assayed with 3 extractions on the easyMAG at each timepoint (7 at timepoint 0). This study assessed the following storage conditions:

Table. Specimen stability conditions   

<table><tr><td rowspan=1 colspan=3>Nasopharyngeal Swabs in transport media</td></tr><tr><td rowspan=1 colspan=1>Storage temp</td><td rowspan=1 colspan=1>Target (storage time)</td><td rowspan=1 colspan=1>Test time points</td></tr><tr><td rowspan=1 colspan=1>Room Temperature 25°C</td><td rowspan=1 colspan=1>8 hours</td><td rowspan=1 colspan=1>8,12 hours</td></tr><tr><td rowspan=1 colspan=1>Refrigerated 4°</td><td rowspan=1 colspan=1>7 days</td><td rowspan=1 colspan=1>5, 7, 10 days</td></tr><tr><td rowspan=1 colspan=1>Fresh Vs Frozen (-70°C)</td><td rowspan=1 colspan=1>90 days (2x freeze/thaw)</td><td rowspan=1 colspan=1>30, 60, 90 days</td></tr><tr><td rowspan=1 colspan=3>Purified nucleic acids</td></tr><tr><td rowspan=1 colspan=1>Storage temp</td><td rowspan=1 colspan=1>Target (storage time)</td><td rowspan=1 colspan=1>Test time points</td></tr><tr><td rowspan=1 colspan=1>Refrigerated 4°</td><td rowspan=1 colspan=1>8 hours</td><td rowspan=1 colspan=1>8, 12 hours</td></tr><tr><td rowspan=1 colspan=1>Fresh Vs Frozen (-70°C)</td><td rowspan=1 colspan=1>90 days (2x freeze/thaw)</td><td rowspan=1 colspan=1>30, 60, 90 days</td></tr></table>

Table. Contrived samples (3x LoD in prescreened negative natural matrix)   

<table><tr><td rowspan=1 colspan=1>Sample Name</td><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Source</td></tr><tr><td rowspan=3 colspan=1>RP A</td><td rowspan=1 colspan=1>Adenovirus B Serotype 7a</td><td rowspan=1 colspan=1>Zeptometrix 0810021CF</td></tr><tr><td rowspan=1 colspan=1>Mycoplasma pneumoniae</td><td rowspan=1 colspan=1>Zeptometrix 0801579</td></tr><tr><td rowspan=1 colspan=1>Influenza A H3N2 A/Wisconsin/67/2005</td><td rowspan=1 colspan=1>Zeptometrix 0810252CF</td></tr><tr><td rowspan=3 colspan=1>RP B</td><td rowspan=1 colspan=1>Respiratory Syncytial Virus (Type A)</td><td rowspan=1 colspan=1>Zeptometrix 0810040ACF</td></tr><tr><td rowspan=1 colspan=1>H1N1pdm California/07/09</td><td rowspan=1 colspan=1>Zeptometrix 0810165CF</td></tr><tr><td rowspan=1 colspan=1>Human Metapneumovirus (16; type A1)</td><td rowspan=1 colspan=1>Zeptometrix 0810161CF</td></tr><tr><td rowspan=3 colspan=1>RP C</td><td rowspan=1 colspan=1>Parainfluenza Virus 3</td><td rowspan=1 colspan=1>Zeptometrix 0810016CF</td></tr><tr><td rowspan=1 colspan=1>Coronavirus NL63</td><td rowspan=1 colspan=1>Zeptometrix 0810228CF</td></tr><tr><td rowspan=1 colspan=1>Flu B/Florida/4/2006</td><td rowspan=1 colspan=1>Zeptometrix 0810255CF</td></tr></table>

Results: All replicates were Valid and detected for all conditions assayed. Summary results are shown in the table below.

Table. Qualitative results from Specimen Stability study   

<table><tr><td rowspan=1 colspan=3></td><td rowspan=1 colspan=3>Sample A</td><td rowspan=1 colspan=3>Sample B</td><td rowspan=1 colspan=3>Sample C</td></tr><tr><td rowspan=1 colspan=1>Sample Type</td><td rowspan=1 colspan=1>Temp</td><td rowspan=1 colspan=1>Time(HoursorDays)</td><td rowspan=1 colspan=1>ADV</td><td rowspan=1 colspan=1>MPN</td><td rowspan=1 colspan=1>Flu AH3N2</td><td rowspan=1 colspan=1>RSV</td><td rowspan=1 colspan=1>Flu AH1N12009pdm</td><td rowspan=1 colspan=1>hMPV</td><td rowspan=1 colspan=1>PIV3</td><td rowspan=1 colspan=1>NL63</td><td rowspan=1 colspan=1>Flu B</td></tr><tr><td rowspan=1 colspan=1>Fresh (baseline)</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>7/7</td><td rowspan=1 colspan=1>7/7</td><td rowspan=1 colspan=1>7/7</td><td rowspan=1 colspan=1>7/7</td><td rowspan=1 colspan=1>7/7</td><td rowspan=1 colspan=1>7/7</td><td rowspan=1 colspan=1>7/7</td><td rowspan=1 colspan=1>7/7</td><td rowspan=1 colspan=1>7/7</td></tr><tr><td rowspan=8 colspan=1>Spiked NPS</td><td rowspan=2 colspan=1>Room Temp</td><td rowspan=1 colspan=1>8 Hour</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>12 Hour</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=3 colspan=1>4°</td><td rowspan=1 colspan=1>5 Day</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>7 Day</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>10 Day</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=3 colspan=1>-80</td><td rowspan=1 colspan=1>30 Day</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>60 Day</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>90 Day</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=5 colspan=1>Extracted Nucleicacid</td><td rowspan=2 colspan=1>4°C</td><td rowspan=1 colspan=1>8 Hour</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>12 Hour</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=3 colspan=1>-80°(2x Freeze/thaw)</td><td rowspan=1 colspan=1>30 Day</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>60 Day</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>90 Day</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td></tr></table>

Conclusions: All replicates were Valid and detected for each specimen stability condition.

# Matrix equivalency: Simulated Nasopharyngeal Swabs (sNPS) vs Native Nasopharyngeal Swabs (NPS)

Analytical validation studies are mostly performed with contrived specimens (i.e., spiking known concentrations of pathogens into negative NPS in UTM or VTM matrix). This would require a large volume of negative natural NPS to be collected and screened, while typically only between 1.0 to $2 . 0 \mathsf { m L }$ per donor could be obtained after NPS collection and screening to confirm negative status. This makes collection from donors burdensome. An analytical study was performed to assess the performance equivalency of testing a simulated NPS (sNPS) in UTM matrix compared to testing natural NPS in UTM using the BioCode RPP, with both the NucliSENS easyMAG and the MagNA Pure 96 extraction systems. The sNPS in UTM matrix consists of $2 \times 1 0 ^ { 3 }$ HeLa cells/mL diluted in UTM.

Samples were contrived in negative natural NPS in UTM matrix and in sNPS in UTM matrix with multispiked pathogens at close to LoD levels (approximately $1 . 5 \times$ to 15.0xLoD). Each sample was extracted in quadruplicate on both the easyMAG and the MagNA Pure 96 extraction system and tested singly with the BioCode RPP on the BioCode MDx-3000 system.

Results: All four replicates of pathogens in three pools were Valid and detected. No unexpected targets were detected in the sNPS.

Table. Results of natural NPS Vs sNPS comparison   

<table><tr><td rowspan=3 colspan=1>Pools</td><td rowspan=3 colspan=1>Organism</td><td rowspan=3 colspan=1>Source</td><td rowspan=3 colspan=1>Concentration</td><td rowspan=1 colspan=2>Multiple of LoD tested</td><td rowspan=1 colspan=4>Detected (N of 4)</td></tr><tr><td rowspan=2 colspan=1>easyMAG</td><td rowspan=2 colspan=1>MagNAPure 96</td><td rowspan=1 colspan=2>easyMAG</td><td rowspan=1 colspan=2>MagNA Pure96</td></tr><tr><td rowspan=1 colspan=1>NaturalNPS</td><td rowspan=1 colspan=1>SNPS</td><td rowspan=1 colspan=1>NaturalNPS</td><td rowspan=1 colspan=1>SNPS</td></tr><tr><td rowspan=3 colspan=1>PoolA</td><td rowspan=1 colspan=1>Adenovirus ESerotype 4</td><td rowspan=1 colspan=1>Zeptometrix0810070CF</td><td rowspan=1 colspan=1>0.6 TCID5o/mL</td><td rowspan=1 colspan=1>15x LoD</td><td rowspan=1 colspan=1>15x LoD</td><td rowspan=1 colspan=1>4/4</td><td rowspan=1 colspan=1>4/4</td><td rowspan=1 colspan=1>4/4</td><td rowspan=1 colspan=1>4/4</td></tr><tr><td rowspan=1 colspan=1>Chlamydiapneumoniae</td><td rowspan=1 colspan=1>ATCC 53592</td><td rowspan=1 colspan=1>75.0 CFU/mL</td><td rowspan=1 colspan=1>4.5x LoD</td><td rowspan=1 colspan=1>2.3x LoD</td><td rowspan=1 colspan=1>4/4</td><td rowspan=1 colspan=1>4/4</td><td rowspan=1 colspan=1>4/4</td><td rowspan=1 colspan=1>4/4</td></tr><tr><td rowspan=1 colspan=1>Influenza A H3N2A/Wisconsin/67/2005</td><td rowspan=1 colspan=1>Zeptometrix0810252CF</td><td rowspan=1 colspan=1>6.0 TCID50/mL</td><td rowspan=1 colspan=1>1.5x LoD</td><td rowspan=1 colspan=1>4.6x LoD</td><td rowspan=1 colspan=1>4/4</td><td rowspan=1 colspan=1>4/4</td><td rowspan=1 colspan=1>4/4</td><td rowspan=1 colspan=1>4/4</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=3 colspan=1>PoolB</td><td rowspan=1 colspan=1>Influenza AH1N1/NewCaledonia/ 20/99</td><td rowspan=1 colspan=1>Zeptometrix0810036CF</td><td rowspan=1 colspan=1>67.5TCID50/mL</td><td rowspan=1 colspan=1>4.5x LoD</td><td rowspan=1 colspan=1>13.5x LoD</td><td rowspan=1 colspan=1>4/4</td><td rowspan=1 colspan=1>4/4</td><td rowspan=1 colspan=1>4/4</td><td rowspan=1 colspan=1>4/4</td></tr><tr><td rowspan=1 colspan=1>Respiratory SyncytialVirus (Type A)</td><td rowspan=1 colspan=1>Zeptometrix0810040ACF</td><td rowspan=1 colspan=1>1.5 TCID50/mL</td><td rowspan=1 colspan=1>4.5x LoD</td><td rowspan=1 colspan=1>4.5x LoD</td><td rowspan=1 colspan=1>4/4</td><td rowspan=1 colspan=1>4/4</td><td rowspan=1 colspan=1>4/4</td><td rowspan=1 colspan=1>4/4</td></tr><tr><td rowspan=1 colspan=1>HumanMetapneumovirus(16; type A1)</td><td rowspan=1 colspan=1>Zeptometrix0810161CF</td><td rowspan=1 colspan=1>67.5TCID50/mL</td><td rowspan=1 colspan=1>4.5x LoD</td><td rowspan=1 colspan=1>4.5x LoD</td><td rowspan=1 colspan=1>4/4</td><td rowspan=1 colspan=1>4/4</td><td rowspan=1 colspan=1>4/4</td><td rowspan=1 colspan=1>4/4</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=3 colspan=1>PoolC</td><td rowspan=1 colspan=1>InfluenzaB/Florida/4/2006</td><td rowspan=1 colspan=1>Zeptometrix0810255CF</td><td rowspan=1 colspan=1>0.06TCID50/mL</td><td rowspan=1 colspan=1>6x LoD</td><td rowspan=1 colspan=1>6x LoD</td><td rowspan=1 colspan=1>4/4</td><td rowspan=1 colspan=1>4/4</td><td rowspan=1 colspan=1>4/4</td><td rowspan=1 colspan=1>4/4</td></tr><tr><td rowspan=1 colspan=1>Coronavirus OC43</td><td rowspan=1 colspan=1>Zeptometrix0810024CF</td><td rowspan=1 colspan=1>0.06TCID50/mL</td><td rowspan=1 colspan=1>1.5x LoD</td><td rowspan=1 colspan=1>6x LoD</td><td rowspan=1 colspan=1>4/4</td><td rowspan=1 colspan=1>4/4</td><td rowspan=1 colspan=1>4/4</td><td rowspan=1 colspan=1>4/4</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza Virus 4</td><td rowspan=1 colspan=1>Zeptometrix0810060CF</td><td rowspan=1 colspan=1>13.5TCID50/mL</td><td rowspan=1 colspan=1>1.5x LoD</td><td rowspan=1 colspan=1>1.5x LoD</td><td rowspan=1 colspan=1>4/4</td><td rowspan=1 colspan=1>4/4</td><td rowspan=1 colspan=1>4/4</td><td rowspan=1 colspan=1>4/4</td></tr><tr><td rowspan=1 colspan=2>Negative Control</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td></tr></table>

Conclusions: No matrix effect was observed in this study. All replicates for each contrived sample assayed with the simulated matrix (sNPS- HeLa cells in UTM) and natural matrix were valid and detected for expected targets per the algorithm using both extraction systems easyMAG and MagNA Pure 96 extraction systems.