BioFire Diagnostics, LLC   
Kristen Kanack   
Senior Vice President, Regulatory and Clinical Affairs   
515 Colorow Drive   
Salt Lake City, Utah 84108

Re: K193519 Trade/Device Name: BioFire Blood Culture Identification 2 (BCID2) Panel Regulation Number: 21 CFR 866.3365 Regulation Name: Multiplex Nucleic Acid Assay for Identification of Microorganisms and Resistance Markers from Positive Blood Cultures Regulatory Class: Class II Product Code: PAM, PEO Dated: December 18, 2019 Received: December 19, 2019

Dear Kristen Kanack:

We have reviewed your Section 510(k) premarket notification of intent to market the device referenced above and have determined the device is substantially equivalent (for the indications for use stated in the enclosure) to legally marketed predicate devices marketed in interstate commerce prior to May 28, 1976, the enactment date of the Medical Device Amendments, or to devices that have been reclassified in accordance with the provisions of the Federal Food, Drug, and Cosmetic Act (Act) that do not require approval of a premarket approval application (PMA). You may, therefore, market the device, subject to the general controls provisions of the Act. Although this letter refers to your product as a device, please be aware that some cleared products may instead be combination products. The 510(k) Premarket Notification Database located at https://www.accessdata.fda.gov/scripts/cdrh/cfdocs/cfpmn/pmn.cfm identifies combination product submissions. The general controls provisions of the Act include requirements for annual registration, listing of devices, good manufacturing practice, labeling, and prohibitions against misbranding and adulteration. Please note: CDRH does not evaluate information related to contract liability warranties. We remind you, however, that device labeling must be truthful and not misleading.

If your device is classified (see above) into either class II (Special Controls) or class III (PMA), it may be subject to additional controls. Existing major regulations affecting your device can be found in the Code of Federal Regulations, Title 21, Parts 800 to 898. In addition, FDA may publish further announcements concerning your device in the Federal Register.

Please be advised that FDA's issuance of a substantial equivalence determination does not mean that FDA has made a determination that your device complies with other requirements of the Act or any Federal statutes and regulations administered by other Federal agencies. You must comply with all the Act's requirements, including, but not limited to: registration and listing (21 CFR Part 807); labeling (21 CFR Part

801 and Part 809); medical device reporting (reporting of medical device-related adverse events) (21 CFR 803) for devices or postmarketing safety reporting (21 CFR 4, Subpart B) for combination products (see https://www.fda.gov/combination-products/guidance-regulatory-information/postmarketing-safety-reportingcombination-products); good manufacturing practice requirements as set forth in the quality systems (QS) regulation (21 CFR Part 820) for devices or current good manufacturing practices (21 CFR 4, Subpart A) for combination products; and, if applicable, the electronic product radiation control provisions (Sections 531- 542 of the Act); 21 CFR 1000-1050.

Also, please note the regulation entitled, "Misbranding by reference to premarket notification" (21 CFR Part 807.97). For questions regarding the reporting of adverse events under the MDR regulation (21 CFR Part 803), please go to https://www.fda.gov/medical-devices/medical-device-safety/medical-device-reportingmdr-how-report-medical-device-problems.

For comprehensive regulatory information about medical devices and radiation-emitting products, including information about labeling regulations, please see Device Advice (https://www.fda.gov/medical  
devices/device-advice-comprehensive-regulatory-assistance) and CDRH Learn   
(https://www.fda.gov/training-and-continuing-education/cdrh-learn). Additionally, you may contact the Division of Industry and Consumer Education (DICE) to ask a question about a specific regulatory topic. See the DICE website (https://www.fda.gov/medical-devices/device-advice-comprehensive-regulatoryassistance/contact-us-division-industry-and-consumer-education-dice) for more information or contact DICE by email (DICE@fda.hhs.gov) or phone (1-800-638-2041 or 301-796-7100).

Sincerely,

Kristian Roth, Ph.D.   
Chief   
Bacterial Multiplex and Medical Counter Measures Branch Division of Microbiology Devices   
OHT7: Office of In Vitro Diagnostics   
and Radiological Health   
Office of Product Evaluation and Quality   
Center for Devices and Radiological Health

Enclosure

510(k) Number (if known) K193519

Device Name BioFire Blood Culture Identification 2 (BCID2) Panel

Indications for Use (Describe)   
The BioFire $^ \mathrm { \textregistered }$ Blood Culture Identification 2 (BCID2) Panel is a multiplexed nucleic acid test intended for use with   
FilmArray $^ \mathrm { \textregistered }$ 2.0 or FilmArray $\textsuperscript { \textregistered }$ Torch systems for the simultaneous qualitative detection and identification of multiple   
bacterial and yeast nucleic acids and select genetic determinants associated with antimicrobial resistance. The BioFire   
BCID2 Panel test is performed directly on blood culture samples identified as positive by a continuous monitoring blood   
culture system. Results are intended to be interpreted in conjunction with Gram stain results. The following organism   
types and subtypes are identified using the BioFire BCID2 Panel: Gram Positive Bacteria   
• Enterococcus faecalis   
• Staphylococcus spp.   
• Streptococcus spp.   
• Enterococcus faecium   
• Staphylococcus aureus   
• Streptococcus agalactiae (Group B) • Listeria monocytogenes   
• Staphylococcus epidermidis   
• Streptococcus pneumoniae   
• Staphylococcus lugdunensis   
• Streptococcus pyogenes (Group A)

Gram Negative Bacteria

• Acinetobacter calcoaceticus-baumannii complex • Enterobacterales   
• Bacteroides fragilis   
• Enterobacter cloacae complex   
• Haemophilus influenza   
• Escherichia coli   
• Neisseria meningitidis (encapsulated)   
• Klebsiella aerogenes   
• Pseudomonas aeruginosa   
• Klebsiella oxytoca   
• Stenotrophomonas maltophilia   
• Klebsiella pneumoniae group   
• Proteus spp.   
• Salmonella spp.   
• Serratia marcescens   
Yeast   
• Candida albicans   
• Candida krusei   
• Cryptococcus neoformans/gattii   
• Candida auris   
• Candida parapsilosis

• Candida glabrata • Candida tropicalis

The BioFire BCID2 Panel contains assays for the detection of genetic determinants associated with resistance to methicillin (mecA/C and mecA/C in conjunction with MREJ), vancomycin (vanA and vanB), ß-lactams including penicillins, cephalosporins, monobactams, and carbapenems (blaCTX-M, blaIMP, blaKPC, blaNDM, blaOXA48-like, blaVIM) to aid in the identification of potentially antimicrobial-resistant organisms in positive blood culture samples. In addition, the panel includes an assay for the detection of the mobilized genetic determinant mcr-1, an emerging marker of public health importance. The antimicrobial resistance gene or marker detected may or may not be associated with the agent responsible for disease. Negative results for these select antimicrobial resistance gene and marker assays do not indicate susceptibility, as multiple mechanisms of resistance to methicillin, vancomycin, ß-lactams, and colistin exist.

Antimicrobial Resistance Genes   
• CTX-M   
• KPC   
• mecA/C   
• NDM   
• vanA/B   
• IMP   
• mcr-1   
• mecA/C and MREJ (MRSA)   
• OXA-48-like   
• VIM

The BioFire BCID2 Panel is indicated as an aid in the diagnosis of specific agents of bloodstream infection and results should be used in conjunction with other clinical and laboratory findings. Positive results do not rule out co-infection with organisms not included in the BioFire BCID2 Panel. The BioFire BCID2 Panel is not intended to monitor treatment for bloodstream infection.

Subculturing of positive blood cultures is necessary to recover organisms for susceptibility testing and epidemiological typing, to identify organisms in the blood culture that are not detected by the BioFire BCID2 Panel, and for determination of species detected but not identified within complexes, groups, or genera by the BioFire BCID2 Panel assays.

# CONTINUE ON A SEPARATE PAGE IF NEEDED.

This section applies only to requirements of the Paperwork Reduction Act of 1995.

# O NOT SEND YOUR COMPLETED FORM TO THE PRA STAFF EMAIL ADDRESS BELOW.

The burden time for this collection of information is estimated to average 79 hours per response, including the time to review instructions, search existing data sources, gather and maintain the data needed and complete and review the collection of information. Send comments regarding this burden estimate or any other aspect of this information collection, including suggestions for reducing this burden, to:

Department of Health and Human Services Food and Drug Administration Office of Chief Information Officer Paperwork Reduction Act (PRA) Staff PRAStaff@fda.hhs.gov

“An agency may not conduct or sponsor, and a person is not required to respond to, a collection of information unless it displays a currently valid OMB number.”

# BioFire® Blood Culture Identification 2 (BCID2) Panel 510(k) Summary BioFire Diagnostics, LLC

# Introduction:

According to the requirements of 21 CFR 807.92, the following information provides sufficient detail to understand the basis for a determination of substantial equivalence.

# Submitted by:

BioFire Diagnostics, LLC 515 Colorow Drive Salt Lake City, UT 84108

Telephone: 801-736-6354

Facsimile: 801-588-0507

Contact: Kristen J. Kanack, ext. 1330

Date Submitted: December 18, 2019

# Device Name and Classification:

Trade Name: BioFire® Blood Culture Identification 2 (BCID2) Panel

Regulation Number: 21 CFR 866.3365

Classification Name: Multiplex nucleic acid assay for identification of microorganisms and resistance markers from positive blood cultures.

# Predicate Device:

K181493 – FilmArray® Blood Culture Identification (BCID) Panel

# Intended Use:

The BioFire® Blood Culture Identification 2 (BCID2) Panel is a multiplexed nucleic acid test intended for use with FilmArray ${ \mathfrak { e } } _ { 2 . 0 }$ or FilmArray® Torch systems for the simultaneous qualitative detection and identification of multiple bacterial and yeast nucleic acids and select genetic determinants associated with antimicrobial resistance. The BioFire BCID2 Panel test is performed directly on blood culture samples identified as positive by a continuous monitoring blood culture system. Results are intended to be interpreted in conjunction with Gram stain results. The following organism types and subtypes are identified using the BioFire BCID2 Panel:

<table><tr><td colspan="3">Gram Positive Bacteria</td></tr><tr><td>Enterococcus faecalis</td><td>Staphylococcus spp.</td><td>Streptococcus spp.</td></tr><tr><td>Enterococcus faecium</td><td>Staphylococcus aureus</td><td>Streptococcus agalactiae (Group B)</td></tr><tr><td>Listeria monocytogenes</td><td>Staphylococcus epidermidis</td><td>Streptococcus pneumoniae</td></tr><tr><td></td><td>Staphylococcus lugdunensis</td><td>Streptococcus pyogenes (Group A)</td></tr><tr><td colspan="3">Gram Negative Bacteria</td></tr><tr><td>Acinetobacter calcoaceticus-baumannii complex</td><td></td><td>Enterobacterales</td></tr><tr><td>Bacteroides fragilis</td><td></td><td>Enterobacter cloacae complex</td></tr><tr><td>Haemophilus influenzae Neisseria meningitidis (encapsulated)</td><td></td><td>Escherichia coli</td></tr><tr><td>Pseudomonas aeruginosa</td><td></td><td>Klebsiella aerogenes</td></tr><tr><td>Stenotrophomonas maltophilia</td><td></td><td>Klebsiella oxytoca</td></tr><tr><td></td><td></td><td>Klebsiella pneumoniae group</td></tr><tr><td></td><td></td><td>Proteus spp.</td></tr><tr><td></td><td></td><td>Salmonella spp.</td></tr><tr><td></td><td></td><td>Serratia marcescens</td></tr><tr><td colspan="3">Yeast</td></tr><tr><td>Candida albicans</td><td>Candida krusei</td><td>Cryptococcus neoformans/gattii</td></tr><tr><td>Candida auris</td><td>Candida parapsilosis</td><td></td></tr><tr><td>Candida glabrata</td><td>Candida tropicalis</td><td></td></tr></table>

The BioFire BCID2 Panel contains assays for the detection of genetic determinants associated with resistance to methicillin (mecA/C and mecA/C in conjunction with MREJ), vancomycin (vanA and vanB), ß-lactams including penicillins, cephalosporins, monobactams, and carbapenems (blaCTX-M, blaIMP, blaKPC, blaNDM, blaOXA48-like, blaVIM) to aid in the identification of potentially antimicrobial-resistant organisms in positive blood culture samples. In addition, the panel includes an assay for the detection of the mobilized genetic determinant mcr-1, an emerging marker of public health importance. The antimicrobial resistance gene or marker detected may or may not be associated with the agent responsible for disease. Negative results for these select antimicrobial resistance gene and marker assays do not indicate susceptibility, as multiple mechanisms of resistance to methicillin, vancomycin, ß-lactams, and colistin exist.

<table><tr><td colspan="5">Antimicrobial Resistance Genes</td></tr><tr><td>CTX-M</td><td>KPC</td><td>mecA/C</td><td>NDM</td><td>vanA/B</td></tr><tr><td>IMP</td><td>mcr-1</td><td>mecA/C and MREJ (MRSA)</td><td>OXA-48-like</td><td>VIM</td></tr></table>

The BioFire BCID2 Panel is indicated as an aid in the diagnosis of specific agents of bloodstream infection and results should be used in conjunction with other clinical and laboratory findings. Positive results do not rule out coinfection with organisms not included in the BioFire BCID2 Panel. The BioFire BCID2 Panel is not intended to monitor treatment for blood stream infection.

Subculturing of positive blood cultures is necessary to recover organisms for susceptibility testing and epidemiological typing, to identify organisms in the blood culture that are not detected by the BioFire BCID2 Panel, and for determination of species detected but not identified within complexes, groups, or genera by the BioFire BCID2 Panel assays.

# Device Description:

The BioFire Blood Culture Identification 2 (BCID2) Panel is designed to simultaneously identify 43 bacteria and yeast responsible for bloodstream infections, as well as select genetic determinants of antimicrobial resistance (see Table 1), in a timeframe(about an hour) that allows the test results to be used in determining appropriate patient treatment and management. The BioFire BCID2 Panel is performed directly on positive blood culture samples.

The BioFire BCID2 Panel is compatible with BioFire’s PCR-based in vitro diagnostic FilmArray 2.0 and FilmArray Torch systems for infectious disease testing. A specific software module (i.e., BioFire BCID2 Panel pouch module) is used to perform BioFire BCID2 Panel testing on these systems.

Table 1. Analytes detected by the BioFire BCID2 Panel   

<table><tr><td colspan="4">Gram Positive Bacteria</td></tr><tr><td>Enterococcus faecalis</td><td>Staphylococcus spp.</td><td colspan="2">Streptococcus spp.</td></tr><tr><td>Enterococcus faecium</td><td>Staphylococcus aureus</td><td colspan="2">Streptococcus agalactiae (Group B)</td></tr><tr><td>Listeria monocytogenes</td><td>Staphylococcus epidermidis</td><td colspan="2">Streptococcus pneumoniae</td></tr><tr><td colspan="3">Staphylococcus lugdunensis</td><td colspan="2">Streptococcus pyogenes (Group A)</td></tr><tr><td colspan="5">Gram Negative Bacteria Acinetobacter calcoaceticus-baumannii complex</td></tr><tr><td colspan="2">Bacteroides fragilis</td><td colspan="2">Enterobacterales</td></tr><tr><td colspan="2">Haemophilus influenzae</td><td colspan="2">Enterobacter cloacae complex</td></tr><tr><td colspan="2">Neisseria meningitidis</td><td colspan="2">Escherichia coli</td></tr><tr><td colspan="2">Pseudomonas aeruginosa</td><td colspan="2">Klebsiella aerogenes</td></tr><tr><td colspan="2">Stenotrophomonas maltophilia</td><td colspan="2">Klebsiella oxytoca</td></tr><tr><td colspan="2"></td><td colspan="2">Klebsiella pneumoniae group</td></tr><tr><td colspan="2"></td><td colspan="2">Proteus spp.</td></tr><tr><td colspan="2">Serratia marcescens</td><td colspan="2">Salmonella spp.</td></tr><tr><td colspan="4">Yeast</td></tr><tr><td colspan="4">Candida albicans Candida krusei Cryptococcus neoformans/gatti</td></tr><tr><td>Candida auris</td><td colspan="2">Candida parapsilosis</td><td colspan="2"></td></tr><tr><td colspan="2">Candida glabrata</td><td colspan="3">Candida tropicalis</td></tr><tr><td colspan="6">Antimicrobial Resistance Genes</td></tr><tr><td colspan="6"></td></tr><tr><td>CTX-M</td><td>KPC</td><td>mecA/C</td><td>NDM</td><td>vanA/B</td></tr><tr><td>IMP</td><td>mcr-1</td><td>mecA/C and MREJ (MRSA)</td><td>OXA-48-like</td><td>VIM</td></tr></table>

A test is initiated by loading Hydration Solution into one port of the FilmArray pouch and positive blood culture specimen mixed with the provided Sample Buffer into the other port of the BioFire BCID2 Panel pouch and placing it in a FilmArray instrument. The pouch contains all of the reagents required for specimen testing and analysis in a freeze-dried format; the addition of Hydration Solution and Sample/Buffer Mix rehydrates the reagents. After the pouch is prepared, the FilmArray Software guides the user through the steps of placing the pouch into the instrument, scanning the pouch barcode, entering the sample identification, and initiating the run.

The FilmArray instruments contain coordinated systems of inflatable bladders and seal points, which act on the pouch to control the movement of liquid between the pouch blisters. When a bladder is inflated over a reagent blister, it forces liquid from the blister into connecting channels. Alternatively, when a seal is placed over a connecting channel it acts as a valve to open or close a channel. In addition, electronically-controlled pneumatic pistons are positioned over multiple plungers in order to deliver the rehydrated reagents into the blisters at the appropriate times. Two Peltier devices control heating and cooling of the pouch to drive the PCR reactions and the melt curve analysis.

Nucleic acid extraction occurs within the FilmArray pouch using mechanical and chemical lysis followed by purification using standard magnetic bead technology. After extracting and purifying nucleic acids from the unprocessed sample, the FilmArray performs a nested multiplex PCR that is executed in two stages. During the first stage, the FilmArray performs a single, large volume, highly multiplexed reverse transcription PCR (rt-PCR) reaction. The products from first stage PCR are then diluted and combined with a fresh, primer-free master mix and a fluorescent double-stranded DNA binding dye (LC Green® Plus, BioFire Diagnostics). The solution is then distributed to each well of the array. Array wells contain sets of primers designed specifically to amplify sequences internal to the PCR products generated during the first stage PCR reaction. The 2nd stage PCR, or nested PCR, is performed in singleplex fashion in each well of the array. At the conclusion of the 2nd stage PCR, the array is interrogated by melt curve analysis for the detection of signature amplicons denoting the presence of specific targets. A digital camera placed in front of the 2nd stage PCR captures fluorescent images of the PCR reactions and software interprets the data.

The FilmArray Software automatically interprets the results of each DNA melt curve analysis and combines the data with the results of the internal pouch controls to provide a test result for each organism on the panel.

# Substantial Equivalence:

The BioFire Blood Culture Identification 2 (BCID2) Panel is substantially equivalent to the FilmArray Blood Culture Identification (BCID) Panel Application (K181493), which was cleared on Jul 07, 2018 and determined to be a Class II device under the classification code 21 CFR 866.3365.

Table 2 compares the BioFire BCID2 Panel to the FilmArray BCID Panel and outlines the similarities and difference between the two systems.

Table 2. Comparison of the BioFire BCID2 Panel and the FilmArray BCID Panel   

<table><tr><td>Element</td><td>Subject Device: BioFire Blood Culture Identification 2 (BCID2) Panel</td><td>Predicate: FilmArray Blood Culture Identification (BCID) Panel K181493</td></tr><tr><td>Specimen Type</td><td>Blood culture samples identified as positive by a continuous monitoring blood culture system.</td><td>Same</td></tr><tr><td rowspan="5">Organisms Detected</td><td>Gram-positive Bacteria Enterococcus faecalis, Enterococcus faecium, Listeria monocytogenes, Staphylococcus spp. (with specific differentiation of Staphylococcus aureus, Staphylococcus epidermidis, and Staphylococcus lugdunensis), Streptococcus spp. (with specific differentiation of Streptococcus agalactiae (Group B), Streptococcus pneumoniae, and Streptococcus pyogenes (Group A))</td><td>Gram-positive Bacteria Enterococcus spp., Listeria monocytogenes, Staphylococcus spp.(including specific differentiation of Staphylococcus aureus), Streptococcus spp. (with specific differentiation of Streptococcus agalactiae, Streptococcus pneumoniae, and Streptococcus pyogenes)</td></tr><tr><td>Gram-negative Bacteria Acinetobacter calcoaceticus-baumannii complex, Bacteroides fragilis, Haemophilus influenzae, Neisseria meningitidis (encapsulated), Pseudomonas aeruginosa, Stenotrophomonas maltophilia, Enterobacterales (with specific differentiation of Enterobacter cloacae complex, Escherichia coli, Klebsiella aerogenes, Klebsiella oxytoca, Klebsiella pneumoniae group, Proteus spp., Salmonella spp., and</td><td>Gram-negative Bacteria Acinetobacter baumannii, Enterobacteriaceae (including specific differentiation of the Enterobacter cloacae complex, Escherichia coli, Klebsiella oxytoca, Klebsiella pneumoniae, Proteus, and Serratia marcescens), Haemophilus influenzae, Neisseria meningitidis (encapsulated), Pseudomonas aeruginosa</td></tr><tr><td>Serratia marcescens) Yeast Candida albicans, Candida auris, Candida glabrata, Candida krusei, Candida parapsilosis, Candida</td><td>Yeast Candida albicans, Candida glabrata, Candida krusei, Candida parapsilosis, Candida tropicalis</td></tr><tr><td>tropicalis, and Cryptococcus neoformans/gatti Antimicrobial Resistance Genes CTX-M, IMP, KPC, mcr-1, mecA/C, mecA/C and MREJ</td><td>Antimicrobial Resistance Genes</td></tr><tr><td>(MRSA), NDM, OXA-48-like, vanA/B, VIM</td><td>mecA (detects mecA and mecC), vanA/B, and KPC</td></tr><tr><td>Analyte</td><td>DNA</td><td>Same</td></tr><tr><td>Technological Principles</td><td>Highly-multiplexed nested nucleic acid amplification test with melt analysis</td><td>Same</td></tr><tr><td>Instrumentation</td><td>FilmArray 2.0 or FilmArray Torch</td><td>FilmArray, FilmArray 2.0, or FilmArray Torch</td></tr><tr><td>Time to result</td><td>About 1 hour</td><td>Same</td></tr><tr><td>Reagent Storage</td><td>Room temperature</td><td>Same</td></tr><tr><td colspan="1" rowspan="1">Element</td><td colspan="1" rowspan="1">Subject Device:BioFire Blood Culture Identification 2 (BCID2) Panel</td><td colspan="1" rowspan="1">Predicate:FilmArray Blood Culture Identification (BCID) PanelK181493</td></tr><tr><td colspan="1" rowspan="1">Test Interpretation</td><td colspan="1" rowspan="1">Automated test interpretation and report generation.User cannot access raw data.</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Controls</td><td colspan="1" rowspan="1">Two controls are included in each reagent pouch tocontrol for sample processing and both stages of PCRand melt analysis.</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">User Complexity</td><td colspan="1" rowspan="1">Moderate/Low</td><td colspan="1" rowspan="1">Same</td></tr></table>

# Summary of Performance Data

# Clinical Performance

The clinical performance of the BioFire BCID2 Panel was established during a prospective multi-center study that was further supplemented with archived and seeded PBC specimens.

Blood culture bottle types evaluated in the prospective clinical study included 11 different media from two different manufacturers as shown in Table 3. Equivalent overall performance was observed when results from the different media were compared; therefore, the data collected from all media types are combined for all analyses. One exception was the detection of 53 false-positive Enterobacterales results from a limited number of lots of media identified to contain nucleic acid from non-viable E. coli; tables containing these data are footnoted.

Table 3. Blood Culture Media Types Evaluated in the BioFire BCID2 Panel Prospective Clinical Evaluation   

<table><tr><td rowspan="2">Blood Culture Media Type</td><td rowspan="2">Manufacturer and Product Name</td><td colspan="3">Prospective Study</td><td colspan="3">Seeded Study</td></tr><tr><td>N</td><td colspan="2">Overall Performancea Sensitivityl Specificity/N</td><td>N</td><td colspan="2">Overall Performancea Sensitivity/ Specificity/</td></tr><tr><td rowspan="4">Aerobic</td><td>BD Bactec Plus Aerobic/F</td><td>344</td><td>PPA 98.8%</td><td>PA 99.8%</td><td>354</td><td>PPA 99.2%</td><td>NPA</td></tr><tr><td></td><td></td><td>100%</td><td>100%</td><td>0</td><td></td><td>100%</td></tr><tr><td>BD Standard 10 Aerobic/F</td><td>3</td><td></td><td></td><td></td><td>-</td><td>-</td></tr><tr><td>bioMérieux BacT/ALERT FA plus bioMérieux BacT/ALERT SA</td><td>264 21</td><td>98.8% 100%</td><td>99.0% 99.5%</td><td>168 0</td><td>99.0%</td><td>100%</td></tr><tr><td rowspan="5">Anaerobic</td><td>BD Bactec Plus Anaerobic/F</td><td>86</td><td>99.1%</td><td>99.9%</td><td>15b</td><td>- 100%</td><td>- 100%</td></tr><tr><td>BD Bactec Standard Anaerobic/F</td><td>1</td><td>100%</td><td>-</td><td>0</td><td>-</td><td></td></tr><tr><td>BD Bactec Lytic/10 Anaerobic/F</td><td>187</td><td>99.6%</td><td>99.9%</td><td>0</td><td>-</td><td>-</td></tr><tr><td>bioMérieux BacT/ALERT FN plus</td><td>40</td><td>100%</td><td>99.8%</td><td>15</td><td>100%</td><td>- 100%</td></tr><tr><td>bioMérieux BacT/ALERT SN</td><td>83</td><td>97.9%</td><td>100%</td><td>0</td><td>-</td><td>-</td></tr><tr><td rowspan="2">Pediatric/Low Volume</td><td>BD Bactec Peds Plus/F</td><td>13</td><td>100%</td><td>99.6%</td><td>0</td><td>-</td><td>-</td></tr><tr><td>bioMérieux BacT/ALERT PF plus</td><td>32</td><td>95.8%</td><td>98.8%</td><td>0</td><td>-</td><td>-</td></tr></table>

a Note that these calculations do not include the performance for the individual Staphylococcus species, individual Streptococcus species, or individual Enterobacterales interpretations, as the grouped Staphylococcus spp., Streptococcus spp., and Enterobacterales interpretations are included instead. b Bacteroides fragilis only

Nine geographically distinct study sites (seven in the US and two in the EU) participated in the prospective clinical evaluation from October 2018 to May 2019. A total of 11 pouch lots were used for testing.

A total of 1093 residual PBC specimens were acquired for the prospective clinical study. At two of the US sites, 69 specimens enrolled between October 2018 and February 2019 were collected and immediately frozen for later testing at the source laboratory. The remaining 1024 specimens were collected and tested fresh. No difference in performance was observed when fresh and frozen specimen results were compared. Therefore, the data collected from 69 valid frozen specimens are combined with data from the valid 1005 fresh specimens for all analyses.

Nineteen (19) specimens were excluded from the final data analysis. The most common reason for specimen exclusion was that the specimen was found to not meet the inclusion criteria after the specimen was enrolled, most often due to the specimen being tested with the BioFire BCID2 Panel outside of the 24-hour window following positive indication by the continuous monitoring blood culture system.

For the prospective study, the performance of the BioFire BCID2 Panel was evaluated by comparing the test result for each analyte with the appropriate comparator/reference methods shown in Table 4.

Table 4. Comparator Methods for the Prospective BioFire BCID2 Panel Clinical Evaluation   

<table><tr><td>BioFire BCID2 Panel Result</td><td>Reference / Comparator Method(s)</td></tr><tr><td>Bacteria and Cryptococcus</td><td>Standard manual and automated microbiological/biochemical identification methods (performed for SOC and abstracted from the subject medical chart)</td></tr><tr><td>Candida species</td><td>SOC identification for genus level followed by PCR &amp; sequencing of isolates for species identification</td></tr><tr><td rowspan="4">AMR Genes</td><td>Method 1 - Assessment of BioFire BCID2 Panel performance (i) One PCR assay performed direct from PBC followed by sequencing of PCR amplicon</td></tr><tr><td>(CTX-M, IMP, KPC, NDM, OXA-48 like, VIM, and mcr-1) (i) Commercially available FDA-cleared and CE-marked molecular IVD assays performed on PBC</td></tr><tr><td>(mecA/C, mecA/C and MREJ (MRSA), KPC, and vanA/B)</td></tr><tr><td>Method 2  Assessment of genotype concordance</td></tr><tr><td rowspan="4"></td><td>PCR &amp; sequencing for specific resistance gene from applicable cultured isolates</td></tr><tr><td>Method 3  Assessment of phenotype concordance</td></tr><tr><td>Phenotypic AST of applicable cultured isolates</td></tr><tr><td></td></tr></table>

To supplement the prospective study for low prevalence analytes, a total of 427 frozen archived PBC specimens were collected from 12 external laboratories and retrospectively tested. Of these, 395 were evaluable. Prior to testing with the BioFire BCID2 Panel, the composition/integrity of the specimens was first confirmed with confirmatory molecular methods; 370 specimens contained confirmed analytes of interest.

Table 5 provides a summary of demographic information for the 1074 specimens included in the prospective study and the 370 specimens included in the archived study.

Table 5. Demographic Data for Prospective study   

<table><tr><td rowspan=1 colspan=2></td><td rowspan=1 colspan=1>Prospective</td><td rowspan=1 colspan=1>Archived</td></tr><tr><td rowspan=3 colspan=1>A</td><td rowspan=1 colspan=1>Male</td><td rowspan=1 colspan=1>573 (53%)</td><td rowspan=1 colspan=1>205 (55%)</td></tr><tr><td rowspan=1 colspan=1>Female</td><td rowspan=1 colspan=1>501 (47%)</td><td rowspan=1 colspan=1>156 (42%)</td></tr><tr><td rowspan=1 colspan=1>Unknown</td><td rowspan=1 colspan=1>0 (0%)</td><td rowspan=1 colspan=1>9 (2%)</td></tr><tr><td rowspan=7 colspan=1>A</td><td rowspan=1 colspan=1>&lt;1 year</td><td rowspan=1 colspan=1>118 (11%)</td><td rowspan=1 colspan=1>22 (6%)</td></tr><tr><td rowspan=1 colspan=1>1-17 years</td><td rowspan=1 colspan=1>143 (13%)</td><td rowspan=1 colspan=1>32 (9%)</td></tr><tr><td rowspan=1 colspan=1>18-44 years</td><td rowspan=1 colspan=1>125 (12%)</td><td rowspan=1 colspan=1>44 (12%)</td></tr><tr><td rowspan=1 colspan=1>45-64 years</td><td rowspan=1 colspan=1>257 (24%)</td><td rowspan=1 colspan=1>112 (30%)</td></tr><tr><td rowspan=1 colspan=1>65-84 years</td><td rowspan=1 colspan=1>333 (31%)</td><td rowspan=1 colspan=1>132 (36%)</td></tr><tr><td rowspan=1 colspan=1>85+ years</td><td rowspan=1 colspan=1>98 (9%)</td><td rowspan=1 colspan=1>27 (7%)</td></tr><tr><td rowspan=1 colspan=1>Unknown</td><td rowspan=1 colspan=1>0 (0%)</td><td rowspan=1 colspan=1>1 (&lt;1%)</td></tr><tr><td rowspan=1 colspan=2>Total</td><td rowspan=1 colspan=1>1074</td><td rowspan=1 colspan=1>370</td></tr></table>

A total of 552 seeded blood culture specimens were also evaluated to further supplement the prospective and archived studies for low prevalence analytes (including AMR genes) and to assess performance in seeded polymicrobial specimens. Seeded PBC specimens were prepared by inoculating human whole blood with a variety of different isolates/strains for each analyte at low concentrations and growing to positivity in a continuous monitoring blood culture system. The number of analytes tested, and the purpose of including each analyte in the seeded study is presented in Table 6 and Table 7.

Table 6. Seeded Specimen Analyte Composition   

<table><tr><td rowspan=1 colspan=1>Purpose</td><td rowspan=1 colspan=1>Analyte</td><td rowspan=1 colspan=1>Number Testeda</td></tr><tr><td rowspan=17 colspan=1>Low Prevalence Analyte</td><td rowspan=1 colspan=1>Listeria monocytogenes</td><td rowspan=1 colspan=1>36</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus lugdunensis</td><td rowspan=1 colspan=1>30</td></tr><tr><td rowspan=1 colspan=1>Bacteroides fragilis</td><td rowspan=1 colspan=1>30</td></tr><tr><td rowspan=1 colspan=1>Klebsiella aerogenes</td><td rowspan=1 colspan=1>42</td></tr><tr><td rowspan=1 colspan=1>Neisseria meningitidis</td><td rowspan=1 colspan=1>35</td></tr><tr><td rowspan=1 colspan=1>Salmonella spp.</td><td rowspan=1 colspan=1>37</td></tr><tr><td rowspan=1 colspan=1>Stenotrophomonas maltophilia</td><td rowspan=1 colspan=1>30</td></tr><tr><td rowspan=1 colspan=1>Candida auris</td><td rowspan=1 colspan=1>30</td></tr><tr><td rowspan=1 colspan=1>Candida krusei</td><td rowspan=1 colspan=1>33</td></tr><tr><td rowspan=1 colspan=1>Candida tropicalis</td><td rowspan=1 colspan=1>35</td></tr><tr><td rowspan=1 colspan=1>Cryptococcus neoformans/gatti</td><td rowspan=1 colspan=1>30</td></tr><tr><td rowspan=1 colspan=1>IMP</td><td rowspan=1 colspan=1>30</td></tr><tr><td rowspan=1 colspan=1>KPC</td><td rowspan=1 colspan=1>45</td></tr><tr><td rowspan=1 colspan=1>NDM</td><td rowspan=1 colspan=1>30</td></tr><tr><td rowspan=1 colspan=1>OXA-48-like</td><td rowspan=1 colspan=1>30</td></tr><tr><td rowspan=1 colspan=1>VIM</td><td rowspan=1 colspan=1>30</td></tr><tr><td rowspan=1 colspan=1>mcr-1</td><td rowspan=1 colspan=1>30</td></tr><tr><td rowspan=5 colspan=1>Evaluation of PolymicrobialSpecimens</td><td rowspan=1 colspan=1>E. faecalis</td><td rowspan=1 colspan=1>10</td></tr><tr><td rowspan=1 colspan=1>E. faecium</td><td rowspan=1 colspan=1>10</td></tr><tr><td rowspan=1 colspan=1>S. aureus</td><td rowspan=1 colspan=1>10</td></tr><tr><td rowspan=1 colspan=1>mecA/C and MREJ</td><td rowspan=1 colspan=1>5</td></tr><tr><td rowspan=1 colspan=1>C. albicans</td><td rowspan=1 colspan=1>10</td></tr><tr><td rowspan=4 colspan=1>AMR Gene Host and Evaluationof Polymicrobial Specimens</td><td rowspan=1 colspan=1>A. calcoaceticus-baumannii complex</td><td rowspan=1 colspan=1>19</td></tr><tr><td rowspan=1 colspan=1>K. pneumoniae group</td><td rowspan=1 colspan=1>92</td></tr><tr><td rowspan=1 colspan=1>E. coli</td><td rowspan=1 colspan=1>44</td></tr><tr><td rowspan=1 colspan=1>P. aeruginosa</td><td rowspan=1 colspan=1>26</td></tr><tr><td rowspan=3 colspan=1>AMR Gene Host</td><td rowspan=1 colspan=1>E. cloacae complex</td><td rowspan=1 colspan=1>8</td></tr><tr><td rowspan=1 colspan=1>K. oxytoca</td><td rowspan=1 colspan=1>6</td></tr><tr><td rowspan=1 colspan=1>Proteus</td><td rowspan=1 colspan=1>9</td></tr><tr><td rowspan=1 colspan=1>Present in Host with other RareAMR Genes</td><td rowspan=1 colspan=1>CTX-M</td><td rowspan=1 colspan=1>63</td></tr></table>

552 seeded specimens total including some with polymicrobial composition or Detected results reporting both an AMR gene and host organism

Table 7. Strains and Replicates Tested in Seeded Specimens   

<table><tr><td colspan="1" rowspan="1">Organism</td><td colspan="1" rowspan="1">AMR Gene(s)</td><td colspan="1" rowspan="1">Strain</td><td colspan="1" rowspan="1">Independent SpecimensTested</td></tr><tr><td colspan="1" rowspan="4">Acinetobacter baumannii</td><td colspan="1" rowspan="1">---</td><td colspan="1" rowspan="1">2 Individual clinical isolates</td><td colspan="1" rowspan="1">5 each (10)</td></tr><tr><td colspan="1" rowspan="1">NDM-1</td><td colspan="1" rowspan="1">AR-BANK #0083</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">NDM-1</td><td colspan="1" rowspan="1">AR-BANK #0088</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">IMP-4</td><td colspan="1" rowspan="1">Individual clinical isolate</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="3" rowspan="1">Acinetobacter baumannii total</td><td colspan="1" rowspan="1">19</td></tr><tr><td colspan="1" rowspan="2">Bacteroides fragilis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">ATCC 29771</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">---</td><td colspan="1" rowspan="1">ATCC 23745</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Amk Gene(s)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Tested</td></tr><tr><td colspan="1" rowspan="8"></td><td colspan="1" rowspan="1">---</td><td colspan="1" rowspan="1">ATCC 25285</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">ATCC 29768</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">ATCC 43858</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">--</td><td colspan="1" rowspan="1">ATCC 43860</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">ATCC 43936</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">ATCC 43937</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">ATCC BAA-2283</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Individual clinical isolate</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">Bacteroides fragilis total</td><td colspan="2" rowspan="1"></td><td colspan="1" rowspan="1">30</td></tr><tr><td colspan="1" rowspan="1">Candida albicans</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2 Individual clinical isolates</td><td colspan="1" rowspan="1">5 each (10)</td></tr><tr><td colspan="1" rowspan="1">Candida albicans total</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">10</td></tr><tr><td colspan="1" rowspan="10">Candida auris</td><td colspan="1" rowspan="1">---</td><td colspan="1" rowspan="1">AR-BANK #0381</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">AR-BANK #0382</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">AR-BANK #0383</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">AR-BANK #0384</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">AR-BANK #0385</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">AR-BANK #0386</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">AR-BANK #0387</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">AR-BANK #0388</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">AR-BANK #0389</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">AR-BANK #0390</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">Candida auris total</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">30</td></tr><tr><td colspan="1" rowspan="3">Candida krusei</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">ATCC 14243</td><td colspan="1" rowspan="1">15</td></tr><tr><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">ATCC 6258</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">15 Individual clinical isolates</td><td colspan="1" rowspan="1">1 each (15)</td></tr><tr><td colspan="1" rowspan="1">Candida krusei total</td><td colspan="2" rowspan="1"></td><td colspan="1" rowspan="1">33</td></tr><tr><td colspan="1" rowspan="1">Candida tropicalis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">35 Individual clinical isolates</td><td colspan="1" rowspan="1">1 each (35)</td></tr><tr><td colspan="1" rowspan="1">Candida tropicalis total</td><td colspan="2" rowspan="1"></td><td colspan="1" rowspan="1">35</td></tr><tr><td colspan="1" rowspan="5">Cryptococcus gatti</td><td colspan="1" rowspan="1">---</td><td colspan="1" rowspan="1">Individual clinical isolate</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">ATCC 56989</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">ATCC 56992</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">ATCC 64062</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">ATCC MYA-4560</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="3" rowspan="1">Cryptococcus gatti total</td><td colspan="1" rowspan="1">15</td></tr><tr><td colspan="1" rowspan="2">Cryptococcus neoformans</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">Individual clinical isolates</td><td colspan="1" rowspan="1">3 each (12)</td></tr><tr><td colspan="1" rowspan="1">--</td><td colspan="1" rowspan="1">ATCC 13690</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">Cryptococcus neoformans total</td><td colspan="2" rowspan="1"></td><td colspan="1" rowspan="1">15</td></tr><tr><td colspan="1" rowspan="3">Enterobacter cloacae</td><td colspan="1" rowspan="1">NDM-1, CTX-M-15</td><td colspan="1" rowspan="1">AR-BANK #0038</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">VIM-1</td><td colspan="1" rowspan="1">AR-BANK #0154</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">KPC</td><td colspan="1" rowspan="1">2 Individual clinical isolates</td><td colspan="1" rowspan="1">1 each (2)</td></tr><tr><td colspan="1" rowspan="1">Enterobacter cloacae total</td><td colspan="2" rowspan="1"></td><td colspan="1" rowspan="1">8</td></tr><tr><td colspan="1" rowspan="1">Enterococcus faecalis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2 Individual clinical isolates</td><td colspan="1" rowspan="1">5 each (10)</td></tr><tr><td colspan="3" rowspan="1">Enterococcus faecalis total</td><td colspan="1" rowspan="1">10</td></tr><tr><td colspan="1" rowspan="1">Organism</td><td colspan="1" rowspan="1">AMR Gene(s)</td><td colspan="1" rowspan="1">Strain</td><td colspan="1" rowspan="1">Independent SpecimensTested</td></tr><tr><td colspan="1" rowspan="1">Enterococcus faecium</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2 Individual clinical isolates</td><td colspan="1" rowspan="1">5 each (10)</td></tr><tr><td colspan="1" rowspan="1">Enterococcus faecium total</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">10</td></tr><tr><td colspan="1" rowspan="13">Escherichia coli</td><td colspan="1" rowspan="1">NDM-1</td><td colspan="1" rowspan="1">AR-BANK #0069</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">CTX-M-55, mcr-1</td><td colspan="1" rowspan="1">AR-BANK #0346</td><td colspan="1" rowspan="1">4</td></tr><tr><td colspan="1" rowspan="1">CTX-M-14, CTX-M-55, mcr-1</td><td colspan="1" rowspan="1">AR-BANK #0349</td><td colspan="1" rowspan="1">4</td></tr><tr><td colspan="1" rowspan="1">mcr-1</td><td colspan="1" rowspan="1">AR-BANK #0350</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">mcr-1</td><td colspan="1" rowspan="1">AR-BANK #0493</td><td colspan="1" rowspan="1">4</td></tr><tr><td colspan="1" rowspan="1">mcr-1</td><td colspan="1" rowspan="1">AR-BANK #0494</td><td colspan="1" rowspan="1">4</td></tr><tr><td colspan="1" rowspan="1">CTX-M, mcr-1</td><td colspan="1" rowspan="1">AR-BANK #0495</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">NDM-1</td><td colspan="1" rowspan="1">ATCC BAA-2452</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">IMP-4</td><td colspan="1" rowspan="1">Individual clinical isolate</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">IMP-3</td><td colspan="1" rowspan="1">Individual clinical isolate</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">IMP-9</td><td colspan="1" rowspan="1">Individual clinical isolate</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">VIM-7</td><td colspan="1" rowspan="1">Individual clinical isolate</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">KPC</td><td colspan="1" rowspan="1">Individual clinical isolate</td><td colspan="1" rowspan="1">4</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli total</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">44</td></tr><tr><td colspan="1" rowspan="5">Klebsiella aerogenes</td><td colspan="1" rowspan="1">---</td><td colspan="1" rowspan="1">10 Individual clinical isolates</td><td colspan="1" rowspan="1">3 each (30)</td></tr><tr><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">2 Individual clinical isolates</td><td colspan="1" rowspan="1">1 each (2)</td></tr><tr><td colspan="1" rowspan="1">OXA-48</td><td colspan="1" rowspan="1">AR-BANK #0074</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">IMP-4</td><td colspan="1" rowspan="1">AR-BANK #0161</td><td colspan="1" rowspan="1">4</td></tr><tr><td colspan="1" rowspan="1">CTX-M</td><td colspan="1" rowspan="1">Individual clinical isolate</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">Klebsiella aerogenes total</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">42</td></tr><tr><td colspan="1" rowspan="2">Klebsiella oxytoca</td><td colspan="1" rowspan="1">KPC-3</td><td colspan="1" rowspan="1">AR-BANK #0147</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">CTX-M</td><td colspan="1" rowspan="1">Individual clinical isolate</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">Klebsiella oxytoca total</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">6</td></tr><tr><td colspan="1" rowspan="18">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">2 Individual clinical isolates</td><td colspan="1" rowspan="1">5 each (10)</td></tr><tr><td colspan="1" rowspan="1">IMP-4</td><td colspan="1" rowspan="1">AR-BANK #0034</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">OXA-181; CTX-M-15</td><td colspan="1" rowspan="1">AR-BANK #0039</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">VIM-27; CTX-M-15</td><td colspan="1" rowspan="1">AR-BANK #0040</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">VIM-27; CTX-M-15</td><td colspan="1" rowspan="1">AR-BANK #0046</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">OXA-181, CTX-M-15</td><td colspan="1" rowspan="1">AR-BANK #0051</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">OXA232; CTX-M-15</td><td colspan="1" rowspan="1">AR-BANK #0066</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">OXA-232; CTX-M-15</td><td colspan="1" rowspan="1">AR-BANK #0075</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">IMP-4</td><td colspan="1" rowspan="1">AR-BANK #0080</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">VIM-1</td><td colspan="1" rowspan="1">AR-BANK #0135</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">NDM-7; CTX-M-15</td><td colspan="1" rowspan="1">AR-BANK #0138</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">OXA-181; CTX-M-15</td><td colspan="1" rowspan="1">AR-BANK #0140</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">OXA-181; CTX-M-15</td><td colspan="1" rowspan="1">AR-BANK #0141</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">OXA-181; CTX-M-15</td><td colspan="1" rowspan="1">AR-BANK #0142</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">NDM-1; OXA-232; CTX-M-15</td><td colspan="1" rowspan="1">AR-BANK #0153</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">OXA-48</td><td colspan="1" rowspan="1">AR-BANK #0160</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">mcr-1</td><td colspan="1" rowspan="1">AR-BANK #0497</td><td colspan="1" rowspan="1">4</td></tr><tr><td colspan="1" rowspan="1">NDM-1; CTX-M</td><td colspan="1" rowspan="1">ATCC BAA-2146</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">cYanmaGlob</td><td colspan="1" rowspan="1">Tested</td></tr><tr><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="1">KPC</td><td colspan="1" rowspan="1">Individual clinical isolate</td><td colspan="1" rowspan="1">8</td></tr><tr><td colspan="1" rowspan="1">KPC</td><td colspan="1" rowspan="1">22 Individual clinical isolates</td><td colspan="1" rowspan="1">1 each (22)</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae total</td><td colspan="2" rowspan="1"></td><td colspan="1" rowspan="1">92</td></tr><tr><td colspan="1" rowspan="7">Listeria monocytogenes</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">ATCC 19115</td><td colspan="1" rowspan="1">8</td></tr><tr><td colspan="1" rowspan="1">--</td><td colspan="1" rowspan="1">ATCC 35152</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">ATCC 43248</td><td colspan="1" rowspan="1">5</td></tr><tr><td colspan="1" rowspan="1">---</td><td colspan="1" rowspan="1">ATCC 51779</td><td colspan="1" rowspan="1">5</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Individual clinical isolate</td><td colspan="1" rowspan="1">2</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">7 Individual clinical isolates</td><td colspan="1" rowspan="1">1 each (7)</td></tr><tr><td colspan="1" rowspan="1">---</td><td colspan="1" rowspan="1">NCTC 10890</td><td colspan="1" rowspan="1">8</td></tr><tr><td colspan="1" rowspan="1">Listeria monocytogenes total</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">36</td></tr><tr><td colspan="1" rowspan="6">Neisseria meningitidis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">ATCC 13077</td><td colspan="1" rowspan="1">10</td></tr><tr><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">ATCC 13090</td><td colspan="1" rowspan="1">10</td></tr><tr><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">ATCC 13102</td><td colspan="1" rowspan="1">4</td></tr><tr><td colspan="1" rowspan="1">--</td><td colspan="1" rowspan="1">ATCC 13103</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">ATCC 35561</td><td colspan="1" rowspan="1">4</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">6 Individual clinical isolates</td><td colspan="1" rowspan="1">1 each (6)</td></tr><tr><td colspan="1" rowspan="1">Neisseria meningitidis total</td><td colspan="2" rowspan="1"></td><td colspan="1" rowspan="1">35</td></tr><tr><td colspan="1" rowspan="3">Proteus mirabilis</td><td colspan="1" rowspan="1">KPC-6</td><td colspan="1" rowspan="1">AR-BANK #0155</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">NDM-1</td><td colspan="1" rowspan="1">AR-BANK #0159</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">CTX-M</td><td colspan="1" rowspan="1">Individual clinical isolate</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">Proteus mirabilis total</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">9</td></tr><tr><td colspan="1" rowspan="8">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">VIM-4</td><td colspan="1" rowspan="1">AR-BANK #0054</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">KPC-5</td><td colspan="1" rowspan="1">AR-BANK #0090</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">IMP-14</td><td colspan="1" rowspan="1">AR-BANK #0092</td><td colspan="1" rowspan="1">4</td></tr><tr><td colspan="1" rowspan="1">VIM-2</td><td colspan="1" rowspan="1">AR-BANK #0100</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">IMP-1</td><td colspan="1" rowspan="1">AR-BANK #0103</td><td colspan="1" rowspan="1">4</td></tr><tr><td colspan="1" rowspan="1">VIM-2</td><td colspan="1" rowspan="1">AR-BANK #0108</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">VIM-2</td><td colspan="1" rowspan="1">AR-BANK #0110</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">VIM-2</td><td colspan="1" rowspan="1">AR-BANK #0111</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">Pseudomonas aeruginosa total</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">26</td></tr><tr><td colspan="1" rowspan="1">Salmonella sp.</td><td colspan="1" rowspan="1">---</td><td colspan="1" rowspan="1">3 Individual clinical isolates</td><td colspan="1" rowspan="1">3 each (9)</td></tr><tr><td colspan="1" rowspan="1">Salmonella enterica serogroup C</td><td colspan="1" rowspan="1">--</td><td colspan="1" rowspan="1">Individual clinical isolate</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">Salmonella enterica ser. Berta</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Individual clinical isolate</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="2">Salmonella enterica ser. Enteritidis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Individual clinical isolate</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">mcr-1</td><td colspan="1" rowspan="1">AR-BANK #0496</td><td colspan="1" rowspan="1">4</td></tr><tr><td colspan="1" rowspan="1">Salmonella enterica ser. Javiana</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Individual clinical isolate</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">Salmonella enterica ser. Newport</td><td colspan="1" rowspan="1">--</td><td colspan="1" rowspan="1">Individual clinical isolate</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">Salmonella enterica ser. Senftenberg</td><td colspan="1" rowspan="1">NDM-1</td><td colspan="1" rowspan="1">AR-BANK #0127</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">Salmonella enterica ser.Typhi</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Individual clinical isolate</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">Salmonella enterica ser. Typhimurium</td><td colspan="1" rowspan="1">--</td><td colspan="1" rowspan="1">Individual clinical isolate</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">Salmonella sp. total</td><td colspan="2" rowspan="1"></td><td colspan="1" rowspan="1">37</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">---</td><td colspan="1" rowspan="1">Individual clinical isolate</td><td colspan="1" rowspan="1">5</td></tr><tr><td colspan="1" rowspan="1">Organism</td><td colspan="1" rowspan="1">AMR Gene(s)</td><td colspan="1" rowspan="1">Strain</td><td colspan="1" rowspan="1">Independent SpecimensTested</td></tr><tr><td colspan="1" rowspan="1">Y</td><td colspan="1" rowspan="1">mecA &amp; MREJ</td><td colspan="1" rowspan="1">Individual clinical isolate</td><td colspan="1" rowspan="1">5</td></tr><tr><td colspan="3" rowspan="1">Staphylococcus aureus total</td><td colspan="1" rowspan="1">10</td></tr><tr><td colspan="1" rowspan="6">Staphylococcus lugdunensis</td><td colspan="1" rowspan="1">---</td><td colspan="1" rowspan="1">5 Individual clinical isolates</td><td colspan="1" rowspan="1">3 each (15)</td></tr><tr><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">ATCC 43809</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">ATCC 49576</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">--</td><td colspan="1" rowspan="1">ATCC 700328</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">---</td><td colspan="1" rowspan="1">ATCC 700582</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">--</td><td colspan="1" rowspan="1">NCTC 7990</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="3" rowspan="1">Staphylococcus lugdunensis total</td><td colspan="1" rowspan="1">30</td></tr><tr><td colspan="1" rowspan="1">Stenotrophomonas maltophilia</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">10 Individual clinical isolates</td><td colspan="1" rowspan="1">3 each (30)</td></tr><tr><td colspan="3" rowspan="1">Stenotrophomonas maltophilia total</td><td colspan="1" rowspan="1">30</td></tr></table>

Table 8. Carbapenem AST Results for Strains with Relevant AMR Gene(s) Used in Seeded Specimens   

<table><tr><td colspan="1" rowspan="2">Strain</td><td colspan="1" rowspan="2">AMR Gene(s)</td><td colspan="1" rowspan="2">Organism</td><td colspan="5" rowspan="1">Carbapenem AST</td></tr><tr><td colspan="1" rowspan="1">couedma</td><td colspan="1" rowspan="1">Breeedenm</td><td colspan="1" rowspan="1">cmudwm</td><td colspan="1" rowspan="1">weeedomew</td><td colspan="1" rowspan="1">peide o b</td></tr><tr><td colspan="8" rowspan="1">CLSI M100 ED30:2020 Used for Breakpoints</td></tr><tr><td colspan="1" rowspan="1">Individual clinical isolate</td><td colspan="1" rowspan="1">IMP-4</td><td colspan="1" rowspan="1">Acinetobacter baumanii</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">S</td><td colspan="1" rowspan="1">S</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Individual clinical isolate</td><td colspan="1" rowspan="1">IMP-4</td><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="7" rowspan="1">CLSI 2018 M100 S28 Used for Breakpoints</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">AR-BANK #0103</td><td colspan="1" rowspan="1">IMP-1</td><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">AR-BANK #0161</td><td colspan="1" rowspan="1">IMP-4</td><td colspan="1" rowspan="1">Klebsiella aerogenes</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">I</td><td colspan="1" rowspan="1">|</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">AR-BANK #0034</td><td colspan="1" rowspan="1">IMP-4</td><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">|</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">S</td><td colspan="1" rowspan="1">I</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">AR-BANK #0080</td><td colspan="1" rowspan="1">IMP-4</td><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">I</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">AR-BANK #0092</td><td colspan="1" rowspan="1">IMP-14</td><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">AR-BANK #0147</td><td colspan="1" rowspan="1">KPC-3</td><td colspan="1" rowspan="1">Klebsiella oxytoca</td><td colspan="1" rowspan="1">|</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">|</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">AR-BANK #0090</td><td colspan="1" rowspan="1">KPC-5</td><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">AR-BANK #0155</td><td colspan="1" rowspan="1">KPC-6</td><td colspan="1" rowspan="1">Proteus mirabilis</td><td colspan="1" rowspan="1">|</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">I</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">AR-BANK #0083</td><td colspan="1" rowspan="1">NDM-1</td><td colspan="1" rowspan="1">Acinetobacter baumannii</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">AR-BANK #0088</td><td colspan="1" rowspan="1">NDM-1</td><td colspan="1" rowspan="1">Acinetobacter baumannii</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">AR-BANK #0038a</td><td colspan="1" rowspan="1">NDM-1</td><td colspan="1" rowspan="1">Enterobacter cloacae</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">AR-BANK #0069</td><td colspan="1" rowspan="1">NDM-1</td><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">AR-BANK #0159</td><td colspan="1" rowspan="1">NDM-1</td><td colspan="1" rowspan="1">Proteus mirabilis</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">AR-BANK #0127</td><td colspan="1" rowspan="1">NDM-1</td><td colspan="1" rowspan="1">Salmonella enterica ser. Senftenberg</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">AR-BANK #0153a</td><td colspan="1" rowspan="1">NDM-1, OXA-232</td><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">AR-BANK #0138a</td><td colspan="1" rowspan="1">NDM-7</td><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">AR-BANK #0074</td><td colspan="1" rowspan="1">OXA-48</td><td colspan="1" rowspan="1">Klebsiella aerogenes</td><td colspan="1" rowspan="1">I</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">I</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">AR-BANK #0160</td><td colspan="1" rowspan="1">OXA-48</td><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">AR-BANK #0039a</td><td colspan="1" rowspan="1">OXA-181</td><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">I</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="2">Strain</td><td colspan="1" rowspan="2">AMR Gene(s)</td><td colspan="1" rowspan="2">Organism</td><td colspan="4" rowspan="1">Carbapenem AST</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">couedna</td><td colspan="1" rowspan="1">Breeedenm</td><td colspan="1" rowspan="1">ceuedam</td><td colspan="1" rowspan="1">Weedonem</td><td colspan="1" rowspan="1">peide to b</td></tr><tr><td colspan="1" rowspan="1">AR-BANK #0051a</td><td colspan="1" rowspan="1">OXA-181</td><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">AR-BANK #0140a</td><td colspan="1" rowspan="1">OXA-181</td><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">I</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">AR-BANK #0141a</td><td colspan="1" rowspan="1">OXA-181</td><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">I</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">AR-BANK #0142a</td><td colspan="1" rowspan="1">OXA-181</td><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">I</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">I</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">AR-BANK #0066a</td><td colspan="1" rowspan="1">OXA-232</td><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">AR-BANK #0075a</td><td colspan="1" rowspan="1">OXA-232</td><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">AR-BANK #0154</td><td colspan="1" rowspan="1">VIM-1</td><td colspan="1" rowspan="1">Enterobacter cloacae</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">I</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">AR-BANK #0135</td><td colspan="1" rowspan="1">VIM-1</td><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">AR-BANK #0100</td><td colspan="1" rowspan="1">VIM-2</td><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">AR-BANK #0108</td><td colspan="1" rowspan="1">VIM-2</td><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">AR-BANK #0110</td><td colspan="1" rowspan="1">VIM-2</td><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">AR-BANK #0111</td><td colspan="1" rowspan="1">VIM-2</td><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">AR-BANK #0054</td><td colspan="1" rowspan="1">VIM-4</td><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">I</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">AR-BANK #0040a</td><td colspan="1" rowspan="1">VIM-27</td><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">AR-BANK #0046a</td><td colspan="1" rowspan="1">VIM-27</td><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">CLSI 2012</td><td colspan="2" rowspan="1">CLSI 2012 M100 S22 Used for Breakpoints</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="2" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Individual clinical isolate</td><td colspan="1" rowspan="1">KPC</td><td colspan="1" rowspan="1">Enterobacter cloacae</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">R</td></tr><tr><td colspan="1" rowspan="1">Individual clinical isolate</td><td colspan="1" rowspan="1">KPC</td><td colspan="1" rowspan="1">Enterobacter cloacae</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">R</td></tr><tr><td colspan="1" rowspan="1">Individual clinical isolate</td><td colspan="1" rowspan="1">KPC</td><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">R</td></tr><tr><td colspan="1" rowspan="1">Individual clinical isolate</td><td colspan="1" rowspan="1">KPC</td><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">R</td></tr><tr><td colspan="1" rowspan="1">Individual clinical isolate</td><td colspan="1" rowspan="1">KPC</td><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">R</td></tr><tr><td colspan="1" rowspan="1">Individual clinical isolate</td><td colspan="1" rowspan="1">KPC</td><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">R</td></tr><tr><td colspan="1" rowspan="1">Individual clinical isolate</td><td colspan="1" rowspan="1">KPC</td><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">R</td></tr><tr><td colspan="1" rowspan="1">Individual clinical isolate</td><td colspan="1" rowspan="1">KPC</td><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">R</td></tr><tr><td colspan="1" rowspan="1">Individual clinical isolate</td><td colspan="1" rowspan="1">KPC</td><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">I</td></tr><tr><td colspan="1" rowspan="1">Individual clinical isolate</td><td colspan="1" rowspan="1">KPC</td><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">R</td></tr><tr><td colspan="1" rowspan="1">Individual clinical isolate</td><td colspan="1" rowspan="1">KPC</td><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">R</td></tr><tr><td colspan="1" rowspan="1">Individual clinical isolate</td><td colspan="1" rowspan="1">KPC</td><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">R</td></tr><tr><td colspan="1" rowspan="1">Individual clinical isolate</td><td colspan="1" rowspan="1">KPC</td><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">R</td></tr><tr><td colspan="1" rowspan="1">Individual clinical isolate</td><td colspan="1" rowspan="1">KPC</td><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">R</td></tr><tr><td colspan="1" rowspan="1">Individual clinical isolate</td><td colspan="1" rowspan="1">KPC</td><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">R</td></tr><tr><td colspan="1" rowspan="1">Individual clinical isolate</td><td colspan="1" rowspan="1">KPC</td><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">R</td></tr><tr><td colspan="1" rowspan="1">Individual clinical isolate</td><td colspan="1" rowspan="1">KPC</td><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">R</td></tr><tr><td colspan="1" rowspan="1">Individual clinical isolate</td><td colspan="1" rowspan="1">KPC</td><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">R</td></tr><tr><td colspan="1" rowspan="1">Individual clinical isolate</td><td colspan="1" rowspan="1">KPC</td><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">R</td></tr><tr><td colspan="1" rowspan="1">Individual clinical isolate</td><td colspan="1" rowspan="1">KPC</td><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">R</td></tr><tr><td colspan="1" rowspan="1">Individual clinical isolate</td><td colspan="1" rowspan="1">KPC</td><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">R</td></tr><tr><td colspan="1" rowspan="1">Individual clinical isolate</td><td colspan="1" rowspan="1">KPC</td><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">R</td></tr><tr><td colspan="1" rowspan="2">Strain</td><td colspan="1" rowspan="2">AMR Gene(s)</td><td colspan="1" rowspan="2">Organism</td><td colspan="5" rowspan="1">Carbapenem AST</td></tr><tr><td colspan="1" rowspan="1">couedmm</td><td colspan="1" rowspan="1">Breeedenm</td><td colspan="1" rowspan="1">caudmm</td><td colspan="1" rowspan="1">weuedomew</td><td colspan="1" rowspan="1">poide oc b</td></tr><tr><td colspan="1" rowspan="1">Individual clinical isolate</td><td colspan="1" rowspan="1">KPC</td><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">R</td></tr><tr><td colspan="1" rowspan="1">Individual clinical isolate</td><td colspan="1" rowspan="1">KPC</td><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">R</td></tr><tr><td colspan="1" rowspan="1">Individual clinical isolate</td><td colspan="1" rowspan="1">KPC</td><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">R</td></tr><tr><td colspan="1" rowspan="1">Individual clinical isolate</td><td colspan="1" rowspan="1">KPC</td><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">R</td></tr><tr><td colspan="6" rowspan="1">Unknown Source for Breakpoints or No AST Information Available</td><td colspan="2" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Individual clinical isolate</td><td colspan="1" rowspan="1">IMP-3</td><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Individual clinical isolate</td><td colspan="1" rowspan="1">IMP-9</td><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">ATCC BAA-2452</td><td colspan="1" rowspan="1">NDM-1</td><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">ATCC BAA-2146a,b</td><td colspan="1" rowspan="1">NDM-1</td><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Individual clinical isolate</td><td colspan="1" rowspan="1">VIM-7</td><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr></table>

aAlso carries CTX-M gene. bPresence of CTX-M gene verified by independent molecular method.

Table 9. ESBL AST Results for Strains with Relevant AMR Gene(s) Used in Seeded Specimens   

<table><tr><td colspan="1" rowspan="2">Strain</td><td colspan="1" rowspan="2">AMR Gene(s)</td><td colspan="1" rowspan="2">Organism</td><td colspan="8" rowspan="1">ESBL AST</td></tr><tr><td colspan="1" rowspan="1">A!!di</td><td colspan="1" rowspan="1">Aruoamm</td><td colspan="1" rowspan="1">ceazjn</td><td colspan="1" rowspan="1">ceedeme</td><td colspan="1" rowspan="1">cexnom</td><td colspan="1" rowspan="1">cexon</td><td colspan="1" rowspan="1">crpzye</td><td colspan="1" rowspan="1">cerx</td></tr><tr><td colspan="11" rowspan="1">CLSI M100 ED30:2020 Used for Breakpoints</td></tr><tr><td colspan="1" rowspan="1">Individual clinical isolate</td><td colspan="1" rowspan="1">CTX-M</td><td colspan="1" rowspan="1">Klebsiella aerogenes</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">S</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td></tr><tr><td colspan="1" rowspan="1">Individual clinical isolate</td><td colspan="1" rowspan="1">CTX-M</td><td colspan="1" rowspan="1">Klebsiella oxytoca</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">S</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">S</td><td colspan="1" rowspan="1">s</td><td colspan="1" rowspan="1">R</td></tr><tr><td colspan="1" rowspan="1">Individual clinical isolate</td><td colspan="1" rowspan="1">CTX-M</td><td colspan="1" rowspan="1">Proteus mirabilis</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">S</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">S</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">S</td><td colspan="1" rowspan="1">S</td><td colspan="1" rowspan="1">R</td></tr><tr><td colspan="11" rowspan="1">CLSI 2018 M100 S28 Used for Breakpoints</td></tr><tr><td colspan="1" rowspan="1">AR-BANK #0495a,e</td><td colspan="1" rowspan="1">CTX-M</td><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">S</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td></tr><tr><td colspan="1" rowspan="1">AR-BANK #0497a,e</td><td colspan="1" rowspan="1">CTX-M</td><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">I</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td></tr><tr><td colspan="1" rowspan="1">AR-BANK #0349a</td><td colspan="1" rowspan="1">CTX-M-14, CTX-M-55</td><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">I</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td></tr><tr><td colspan="1" rowspan="1">AR-BANK #0038b</td><td colspan="1" rowspan="1">CTX-M-15</td><td colspan="1" rowspan="1">Enterobacter cloacae</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td></tr><tr><td colspan="1" rowspan="1">AR-BANK #0039c</td><td colspan="1" rowspan="1">CTX-M-15</td><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td></tr><tr><td colspan="1" rowspan="1">AR-BANK #0040d</td><td colspan="1" rowspan="1">CTX-M-15</td><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td></tr><tr><td colspan="1" rowspan="1">AR-BANK #0046d</td><td colspan="1" rowspan="1">CTX-M-15</td><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td></tr><tr><td colspan="1" rowspan="1">AR-BANK #0051c</td><td colspan="1" rowspan="1">CTX-M-15</td><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td></tr><tr><td colspan="1" rowspan="1">AR-BANK #0066c</td><td colspan="1" rowspan="1">CTX-M-15</td><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td></tr><tr><td colspan="1" rowspan="1">AR-BANK #0075c</td><td colspan="1" rowspan="1">CTX-M-15</td><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td></tr><tr><td colspan="1" rowspan="1">AR-BANK #0138b</td><td colspan="1" rowspan="1">CTX-M-15</td><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td></tr><tr><td colspan="1" rowspan="1">AR-BANK #0140c</td><td colspan="1" rowspan="1">CTX-M-15</td><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td></tr><tr><td colspan="1" rowspan="1">AR-BANK #0141c</td><td colspan="1" rowspan="1">CTX-M-15</td><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">I</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td></tr><tr><td colspan="1" rowspan="1">AR-BANK #0142c</td><td colspan="1" rowspan="1">CTX-M-15</td><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td></tr><tr><td colspan="1" rowspan="1">AR-BANK #0153b,c</td><td colspan="1" rowspan="1">CTX-M-15</td><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td></tr><tr><td colspan="1" rowspan="1">AR-BANK #0346a</td><td colspan="1" rowspan="1">CTX-M-55</td><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td><td colspan="1" rowspan="1">R</td></tr><tr><td colspan="11" rowspan="1">No AST Information Available</td></tr><tr><td colspan="1" rowspan="1">ATCC BAA-2146b,e</td><td colspan="1" rowspan="1">CTX-M</td><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">→</td><td colspan="1" rowspan="1">→</td><td colspan="1" rowspan="1">→</td><td colspan="1" rowspan="1">→</td><td colspan="1" rowspan="1">→</td><td colspan="1" rowspan="1">→</td></tr></table>

a Also carries mcr-1 gene. bAlso carries NDM gene. c Also carries OXA-48-like gene. d Also carries VIM gene. ePresence of CTX-M gene verified by independent molecular method.

Table 10. Colistin MIC for Strains with Relevant AMR Gene(s) Used in Seeded Specimens   

<table><tr><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>AMR Gene(s)</td><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Colistin MIC (μg/ml)</td></tr><tr><td rowspan=1 colspan=1>AR-BANK #0346a</td><td rowspan=1 colspan=1>mcr-1</td><td rowspan=1 colspan=1>Escherichia coli</td><td rowspan=1 colspan=1>4</td></tr><tr><td rowspan=1 colspan=1>AR-BANK #0349a</td><td rowspan=1 colspan=1>mcr-1</td><td rowspan=1 colspan=1>Escherichia coli</td><td rowspan=1 colspan=1>2-4</td></tr><tr><td rowspan=1 colspan=1>AR-BANK #0350</td><td rowspan=1 colspan=1>mcr-1</td><td rowspan=1 colspan=1>Escherichia coli</td><td rowspan=1 colspan=1>4</td></tr><tr><td rowspan=1 colspan=1>AR-BANK #0493</td><td rowspan=1 colspan=1>mcr-1</td><td rowspan=1 colspan=1>Escherichia coli</td><td rowspan=1 colspan=1>8</td></tr><tr><td rowspan=1 colspan=1>AR-BANK #0494</td><td rowspan=1 colspan=1>mcr-1</td><td rowspan=1 colspan=1>Escherichia coli</td><td rowspan=1 colspan=1>8</td></tr><tr><td rowspan=1 colspan=1>AR-BANK #0495a,b</td><td rowspan=1 colspan=1>mcr-1</td><td rowspan=1 colspan=1>Escherichia coli</td><td rowspan=1 colspan=1>4</td></tr><tr><td rowspan=1 colspan=1>AR-BANK #0497a,b</td><td rowspan=1 colspan=1>mcr-1</td><td rowspan=1 colspan=1>Klebsiella pneumoniae</td><td rowspan=1 colspan=1>8</td></tr><tr><td rowspan=1 colspan=1>AR-BANK #0496</td><td rowspan=1 colspan=1>mcr-1</td><td rowspan=1 colspan=1>Salmonella sp.</td><td rowspan=1 colspan=1>8</td></tr></table>

a Also carries CTX-M gene. b Presence of CTX-M gene verified by independent molecular method. c As of February 2020, the United States Food and Drug Administration has not established or recognized minimum inhibitory concentration (MIC) breakpoints for colistin antimicrobial susceptibility testing (AST) related to mcr-1.

Table 11. Methicillin AST Result for Strain with Relevant AMR Gene(s) Used in Seeded Specimens   

<table><tr><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>AMR Gene(s)</td><td rowspan=1 colspan=1>Methicillin AST</td></tr><tr><td rowspan=1 colspan=1>Individual clinical isolate</td><td rowspan=1 colspan=1>Staphylococcus aureus</td><td rowspan=1 colspan=1>mecA &amp; MREJ</td><td rowspan=1 colspan=1>R</td></tr></table>

a CLSI 2012 M100 S22 used for breakpoints.

The results from all three clinical studies are summarized for each organism in Table 12 through Table 24. Performance is based on comparison of the BioFire BCID2 Panel results to the results from comparator methods for prospective specimens (Table 4), the confirmed analyte of interest for archived specimens, and to the known analyte composition for seeded specimens. Positive Percent Agreement (PPA) or Sensitivity for each analyte was calculated as $1 0 0 \% \times ( \mathsf { T P } / \left( \mathsf { T P } + \mathsf { F N } \right) )$ . True positive (TP) indicates that both the BioFire BCID2 Panel and the comparator method (or known analyte composition) had a positive result for the specific analyte, and false negative (FN) indicates that the BioFire BCID2 Panel was negative while the comparator result was positive. Negative Percent Agreement (NPA) or Specificity was calculated as $1 0 0 \% \times \left( \mathsf { T N } / \left( \mathsf { T N } + \mathsf { F P } \right) \right)$ ). True negative (TN) indicates that both the BioFire BCID2 Panel and the comparator method (or known analyte composition) had negative results, and false positive (FP) indicates that the BioFire BCID2 Panel was positive while the comparator result was negative. The exact binomial two-sided $9 5 \%$ confidence interval $( 9 5 \% 0 \% 5 1 )$ was calculated. Investigations of discrepant results are summarized in the footnotes.

Table 12. BioFire BCID2 Panel Clinical Performance Summary, Enterococcus spp.   

<table><tr><td rowspan=2 colspan=1>Analyte</td><td rowspan=2 colspan=1>Studya</td><td rowspan=1 colspan=3>Sensitivity</td><td rowspan=1 colspan=3>Specificity</td></tr><tr><td rowspan=1 colspan=1>TP/(TP + FN)</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95%CI</td><td rowspan=1 colspan=1>TN/(TN + FP)</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95%CI</td></tr><tr><td rowspan=3 colspan=1>Enterococcus faecalis</td><td rowspan=1 colspan=1>Prospective</td><td rowspan=1 colspan=1>31/33</td><td rowspan=1 colspan=1>93.9</td><td rowspan=1 colspan=1>80.4-98.3%</td><td rowspan=1 colspan=1>1040/1041</td><td rowspan=1 colspan=1>99.9</td><td rowspan=1 colspan=1>99.5-100%</td></tr><tr><td rowspan=1 colspan=1>Seeded</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>72.2-100%</td><td rowspan=1 colspan=1>542/542</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>99.3-100%</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>41/43b</td><td rowspan=1 colspan=1>95.3</td><td rowspan=1 colspan=1>84.5-98.7%</td><td rowspan=1 colspan=1>1582/1583c</td><td rowspan=1 colspan=1>99.9</td><td rowspan=1 colspan=1>99.6-100%</td></tr><tr><td rowspan=3 colspan=1>Enterococcus faecium</td><td rowspan=1 colspan=1>Prospective</td><td rowspan=1 colspan=1>27/27</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>87.5-100%</td><td rowspan=1 colspan=1>1044/1047</td><td rowspan=1 colspan=1>99.7</td><td rowspan=1 colspan=1>99.2-99.9%</td></tr><tr><td rowspan=1 colspan=1>Seeded</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>72.2-100%</td><td rowspan=1 colspan=1>542/542</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>99.3-100%</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>37/37</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>90.6-100%</td><td rowspan=1 colspan=1>1586/1589d</td><td rowspan=1 colspan=1>99.8</td><td rowspan=1 colspan=1>99.4-99.9%</td></tr></table>

a Archived testing not performed for E. faecalis or E. faecium b E. faecalis was detected in both FN specimens using an additional molecular method c The single FP specimen was negative for E. faecalis when tested with additional molecular methods

Table 13. BioFire BCID2 Panel Clinical Performance Summary, Listeria monocytogenes   

<table><tr><td rowspan=2 colspan=1>Analyte</td><td rowspan=2 colspan=1>Study</td><td rowspan=1 colspan=3>Sensitivity</td><td rowspan=1 colspan=3>Specificity</td></tr><tr><td rowspan=1 colspan=1>TP/(TP + FN)</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95%CI</td><td rowspan=1 colspan=1>TN/(TN + FP)</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95%CI</td></tr><tr><td rowspan=4 colspan=1>Listeria monocytogenes</td><td rowspan=1 colspan=1>Prospective</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>43.9-100%</td><td rowspan=1 colspan=1>1071/1071</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>99.6-100%</td></tr><tr><td rowspan=1 colspan=1>Archived</td><td rowspan=1 colspan=1>5/5</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>56.6-100%</td><td rowspan=1 colspan=1>231/231</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>98.4-100%</td></tr><tr><td rowspan=1 colspan=1>Seeded</td><td rowspan=1 colspan=1>36/36</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>90.4-100%</td><td rowspan=1 colspan=1>516/516</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>99.3-100%</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>44/44</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>92.0-100%</td><td rowspan=1 colspan=1>1818/1818</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>99.8-100%</td></tr></table>

Table 14. BioFire BCID2 Panel Clinical Performance Summary, Staphylococcus spp.   

<table><tr><td rowspan=2 colspan=1>Analyte</td><td rowspan=2 colspan=1>Studya</td><td rowspan=1 colspan=3>Sensitivity</td><td rowspan=1 colspan=3>Specificity</td></tr><tr><td rowspan=1 colspan=1>TP/(TP + FN)</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95%CI</td><td rowspan=1 colspan=1>TN/(TN + FP)</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95%CI</td></tr><tr><td rowspan=3 colspan=1>Staphylococcus spp.</td><td rowspan=1 colspan=1>Prospective</td><td rowspan=1 colspan=1>471/472</td><td rowspan=1 colspan=1>99.8</td><td rowspan=1 colspan=1>98.8-100%</td><td rowspan=1 colspan=1>589/602</td><td rowspan=1 colspan=1>97.8</td><td rowspan=1 colspan=1>96.3-98.7%</td></tr><tr><td rowspan=1 colspan=1>Seeded</td><td rowspan=1 colspan=1>40/40</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>91.2-100%</td><td rowspan=1 colspan=1>512/512</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>99.3-100%</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>511/512</td><td rowspan=1 colspan=1>99.8</td><td rowspan=1 colspan=1>98.9-100%</td><td rowspan=1 colspan=1>1101/1114b</td><td rowspan=1 colspan=1>98.8</td><td rowspan=1 colspan=1>98.0-99.3%</td></tr><tr><td rowspan=3 colspan=1>Staphylococcus aureus</td><td rowspan=1 colspan=1>Prospective</td><td rowspan=1 colspan=1>149/149</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>97.5-100%</td><td rowspan=1 colspan=1>923/925</td><td rowspan=1 colspan=1>99.8</td><td rowspan=1 colspan=1>99.2-99.9%</td></tr><tr><td rowspan=1 colspan=1>Seeded</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>72.2-100%</td><td rowspan=1 colspan=1>542/542</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>99.3-100%</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>159/159</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>97.6-100%</td><td rowspan=1 colspan=1>1465/1467c</td><td rowspan=1 colspan=1>99.9</td><td rowspan=1 colspan=1>99.5-100%</td></tr><tr><td rowspan=2 colspan=1>Staphylococcus epidermidis</td><td rowspan=1 colspan=1>Prospective</td><td rowspan=1 colspan=1>221/229</td><td rowspan=1 colspan=1>96.5</td><td rowspan=1 colspan=1>93.3-98.2%</td><td rowspan=1 colspan=1>816/845</td><td rowspan=1 colspan=1>96.6</td><td rowspan=1 colspan=1>95.1-97.6%</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>221/229d</td><td rowspan=1 colspan=1>96.5</td><td rowspan=1 colspan=1>93.3-98.2%</td><td rowspan=1 colspan=1>816/845°</td><td rowspan=1 colspan=1>96.6</td><td rowspan=1 colspan=1>95.1-97.6%</td></tr><tr><td rowspan=4 colspan=1>Staphylococcus lugdunensis</td><td rowspan=1 colspan=1>Prospective</td><td rowspan=1 colspan=1>4/4</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>51.0-100%</td><td rowspan=1 colspan=1>1067/1070</td><td rowspan=1 colspan=1>99.7</td><td rowspan=1 colspan=1>99.2-99.9%</td></tr><tr><td rowspan=1 colspan=1>Archived</td><td rowspan=1 colspan=1>16/16</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>80.6-100%</td><td rowspan=1 colspan=1>125/125</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>97.0-100%</td></tr><tr><td rowspan=1 colspan=1>Seeded</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>88.6-100%</td><td rowspan=1 colspan=1>522/522</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>99.3-100%</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>50/50</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>92.9-100%</td><td rowspan=1 colspan=1>1714/1717</td><td rowspan=1 colspan=1>99.8</td><td rowspan=1 colspan=1>99.5-99.9%</td></tr></table>

a Archived testing not performed for Staphylococcus spp., S. aureus, or S. epidermidis; seeded testing not performed for S. epidermidis b Staphylococcus spp. was detected in all 13 FP specimens using an additional molecular method ${ \mathsf { c } } _ { S } .$ aureus was detected in both FP specimens using an additional molecular method d S. epidermidis was detected in 3/8 FN specimens using an additional molecular method; sequencing of the remaining five FN specimens and their isolates identified them as other Staphylococcus spp. e S. epidermidis was detected in all 29 FP specimens using an additional molecular method f S. lugdunensis was detected in all three FP specimens using an additional molecular method

Table 15. BioFire BCID2 Panel Clinical Performance Summary, Streptococcus spp.   

<table><tr><td rowspan=2 colspan=1>Analyte</td><td rowspan=2 colspan=1>Studya</td><td rowspan=1 colspan=3>Sensitivity</td><td rowspan=1 colspan=3>Specificity</td></tr><tr><td rowspan=1 colspan=1>TP/(TP + FN)</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95%CI</td><td rowspan=1 colspan=1>TN(TN + FP)</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95%CI</td></tr><tr><td rowspan=2 colspan=1>Streptococcus spp.</td><td rowspan=1 colspan=1>Prospective</td><td rowspan=1 colspan=1>121/123</td><td rowspan=1 colspan=1>98.4</td><td rowspan=1 colspan=1>94.3-99.6%</td><td rowspan=1 colspan=1>949/951</td><td rowspan=1 colspan=1>99.8</td><td rowspan=1 colspan=1>99.2-99.9%</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>121/123b</td><td rowspan=1 colspan=1>98.4</td><td rowspan=1 colspan=1>94.3-99.6%</td><td rowspan=1 colspan=1>949/951c</td><td rowspan=1 colspan=1>99.8</td><td rowspan=1 colspan=1>99.2-99.9%</td></tr><tr><td rowspan=3 colspan=1>Streptococcus agalactiae (Group B)</td><td rowspan=1 colspan=1>Prospective</td><td rowspan=1 colspan=1>9/9</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>70.1-100%</td><td rowspan=1 colspan=1>1065/1065</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>99.6-100%</td></tr><tr><td rowspan=1 colspan=1>Archived</td><td rowspan=1 colspan=1>16/16</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>80.6-100%</td><td rowspan=1 colspan=1>220/220</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>98.3-100%</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>25/25</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>86.7-100%</td><td rowspan=1 colspan=1>1285/1285</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>99.7-100%</td></tr><tr><td rowspan=2 colspan=1>Streptococcus pneumoniae</td><td rowspan=1 colspan=1>Prospective</td><td rowspan=1 colspan=1>26/26</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>87.1-100%</td><td rowspan=1 colspan=1>1048/1048</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>99.6-100%</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>26/26</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>87.1-100%</td><td rowspan=1 colspan=1>1048/1048</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>99.6-100%</td></tr><tr><td rowspan=3 colspan=1>Streptococcus pyogenes (Group A)</td><td rowspan=1 colspan=1>Prospective</td><td rowspan=1 colspan=1>13/14</td><td rowspan=1 colspan=1>92.9</td><td rowspan=1 colspan=1>68.5-98.7%</td><td rowspan=1 colspan=1>1060/1060</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>99.6-100%</td></tr><tr><td rowspan=1 colspan=1>Archived</td><td rowspan=1 colspan=1>16/16</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>80.6-100%</td><td rowspan=1 colspan=1>220/220</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>98.3-100%</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>29/30</td><td rowspan=1 colspan=1>96.7</td><td rowspan=1 colspan=1>83.3-99.4%</td><td rowspan=1 colspan=1>1280/1280</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>99.7-100%</td></tr></table>

a Archived testing not performed for Streptococcus spp. or S. pneumoniae; seeded testing not performed for Streptococcus spp., S. agalactiae, S. pneumoniae, or S. pyogenes b Streptococcus spp. was detected in 1/2 FN specimens using an additional molecular method c Streptococcus spp. was detected in both FP specimens using an additional molecular method

Table 16. BioFire BCID2 Panel Clinical Performance Summary, Acinetobacter calcoaceticus-baumannii complex   

<table><tr><td rowspan=2 colspan=1>Analyte</td><td rowspan=2 colspan=1>Study</td><td rowspan=1 colspan=3>Sensitivity</td><td rowspan=1 colspan=3>Specificity</td></tr><tr><td rowspan=1 colspan=1>TP/(TP + FN)</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95%CI</td><td rowspan=1 colspan=1>TN/(TN + FP)</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95%CI</td></tr><tr><td rowspan=4 colspan=1>Acinetobacter calcoaceticus-baumanniicomplex</td><td rowspan=1 colspan=1>Prospective</td><td rowspan=1 colspan=1>12/13</td><td rowspan=1 colspan=1>92.3</td><td rowspan=1 colspan=1>66.7-98.6%</td><td rowspan=1 colspan=1>1060/1061</td><td rowspan=1 colspan=1>99.9</td><td rowspan=1 colspan=1>99.5-100%</td></tr><tr><td rowspan=1 colspan=1>Archived</td><td rowspan=1 colspan=1>34/35</td><td rowspan=1 colspan=1>97.1</td><td rowspan=1 colspan=1>85.5-99.5%</td><td rowspan=1 colspan=1>102/103</td><td rowspan=1 colspan=1>99.0</td><td rowspan=1 colspan=1>94.7-99.8%</td></tr><tr><td rowspan=1 colspan=1>Seeded</td><td rowspan=1 colspan=1>19/19</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>83.2-100%</td><td rowspan=1 colspan=1>533/533</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>99.3-100%</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>65/67</td><td rowspan=1 colspan=1>97.0</td><td rowspan=1 colspan=1>89.8-99.2%</td><td rowspan=1 colspan=1>1695/1697b</td><td rowspan=1 colspan=1>99.9</td><td rowspan=1 colspan=1>99.6-100%</td></tr></table>

a ACB complex was detected in both FN specimens; one was detected using an additional molecular method and one was detected upon BioFire BCID2 Panel retest b ACB complex was detected in both FP specimens using an additional molecular method

Table 17. BioFire BCID2 Panel Clinical Performance Summary, Bacteroides fragilis   

<table><tr><td rowspan=2 colspan=1>Analyte</td><td rowspan=2 colspan=1>Study</td><td rowspan=1 colspan=3>Sensitivity</td><td rowspan=1 colspan=3>Specificity</td></tr><tr><td rowspan=1 colspan=1>TP/(TP + FN)</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95%CI</td><td rowspan=1 colspan=1>TN/(TN + FP)</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95%CI</td></tr><tr><td rowspan=4 colspan=1>Bacteroides fragilis</td><td rowspan=1 colspan=1>Prospective</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>61.0-100%</td><td rowspan=1 colspan=1>1065/1068</td><td rowspan=1 colspan=1>99.7</td><td rowspan=1 colspan=1>99.2-99.9%</td></tr><tr><td rowspan=1 colspan=1>Archived</td><td rowspan=1 colspan=1>16/16</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>80.6-100%</td><td rowspan=1 colspan=1>125/125</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>97.0-100%</td></tr><tr><td rowspan=1 colspan=1>Seeded</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>88.6-100%</td><td rowspan=1 colspan=1>522/522</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>99.3-100%</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>52/52</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>93.1-100%</td><td rowspan=1 colspan=1>1712/1715a</td><td rowspan=1 colspan=1>99.8</td><td rowspan=1 colspan=1>99.5-99.9%</td></tr></table>

B. fragilis was detected in all three FP specimens using an additional molecular method

Table 18. BioFire BCID2 Panel Clinical Performance Summary, Enterobacterales   

<table><tr><td colspan="1" rowspan="2">Analyte</td><td colspan="1" rowspan="2">Studya</td><td colspan="3" rowspan="1">Sensitivity</td><td colspan="3" rowspan="1">Specificity</td></tr><tr><td colspan="1" rowspan="1">TP/(TP + FN)</td><td colspan="1" rowspan="1">%</td><td colspan="1" rowspan="1">95%CI</td><td colspan="1" rowspan="1">TN/(TN + FP)</td><td colspan="1" rowspan="1">%</td><td colspan="1" rowspan="1">95%CI</td></tr><tr><td colspan="1" rowspan="3">Enterobacterales</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">269/270</td><td colspan="1" rowspan="1">99.6</td><td colspan="1" rowspan="1">97.9-99.9%</td><td colspan="1" rowspan="1">750/804</td><td colspan="1" rowspan="1">93.3</td><td colspan="1" rowspan="1">91.3-94.8%</td></tr><tr><td colspan="1" rowspan="1">Seeded</td><td colspan="1" rowspan="1">228/228</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">98.3-100%</td><td colspan="1" rowspan="1">324/324</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">98.8-100%</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">497/498</td><td colspan="1" rowspan="1">99.8</td><td colspan="1" rowspan="1">98.9-100%</td><td colspan="1" rowspan="1">1074/1128b</td><td colspan="1" rowspan="1">95.2</td><td colspan="1" rowspan="1">93.8-96.3%</td></tr><tr><td colspan="1" rowspan="4">Enterobacter cloacae complex</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">16/16</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">80.6-100%</td><td colspan="1" rowspan="1">1058/1058</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">99.6-100%</td></tr><tr><td colspan="1" rowspan="1">Archived</td><td colspan="1" rowspan="1">16/16</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">80.6-100%</td><td colspan="1" rowspan="1">219/219</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">98.3-100%</td></tr><tr><td colspan="1" rowspan="1">Seeded</td><td colspan="1" rowspan="1">8/8</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">67.6-100%</td><td colspan="1" rowspan="1">544/544</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">99.3-100%</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">40/40</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">91.2-100%</td><td colspan="1" rowspan="1">1821/1821</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">99.8-100%</td></tr><tr><td colspan="1" rowspan="3">Escherichia coli</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">158/159</td><td colspan="1" rowspan="1">99.4</td><td colspan="1" rowspan="1">96.5-99.9%</td><td colspan="1" rowspan="1">913/915</td><td colspan="1" rowspan="1">99.8</td><td colspan="1" rowspan="1">99.2-99.9%</td></tr><tr><td colspan="1" rowspan="1">Seeded</td><td colspan="1" rowspan="1">44/44</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">92.0-100%</td><td colspan="1" rowspan="1">508/508</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">99.2-100%</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">202/203c</td><td colspan="1" rowspan="1">99.5</td><td colspan="1" rowspan="1">97.3-99.9%</td><td colspan="1" rowspan="1">1421/1423d</td><td colspan="1" rowspan="1">99.9</td><td colspan="1" rowspan="1">99.5-100%</td></tr><tr><td colspan="1" rowspan="4">Klebsiella aerogenes</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">2/2</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">34.2-100%</td><td colspan="1" rowspan="1">1072/1072</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">99.6-100%</td></tr><tr><td colspan="1" rowspan="1">Archived</td><td colspan="1" rowspan="1">16/16</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">80.6-100%</td><td colspan="1" rowspan="1">125/125</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">97.0-100%</td></tr><tr><td colspan="1" rowspan="1">Seeded</td><td colspan="1" rowspan="1">42/42</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">91.6-100%</td><td colspan="1" rowspan="1">510/510</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">99.3-100%</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">60/60</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">94.0-100%</td><td colspan="1" rowspan="1">1707/1707</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">99.8-100%</td></tr><tr><td colspan="1" rowspan="4">Klebsiella oxytoca</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">8/8</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">67.6-100%</td><td colspan="1" rowspan="1">1066/1066</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">99.6-100%</td></tr><tr><td colspan="1" rowspan="1">Archived</td><td colspan="1" rowspan="1">16/16</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">80.6-100%</td><td colspan="1" rowspan="1">219/219</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">98.3-100%</td></tr><tr><td colspan="1" rowspan="1">Seeded</td><td colspan="1" rowspan="1">6/6</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">61.0-100%</td><td colspan="1" rowspan="1">546/546</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">99.3-100%</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">88.6-100%</td><td colspan="1" rowspan="1">1831/1831</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">99.8-100%</td></tr><tr><td colspan="1" rowspan="3">Klebsiella pneumoniae group</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">55/56</td><td colspan="1" rowspan="1">98.2</td><td colspan="1" rowspan="1">90.6-99.7%</td><td colspan="1" rowspan="1">1018/1018</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">99.6-100%</td></tr><tr><td colspan="1" rowspan="1">Seeded</td><td colspan="1" rowspan="1">92/92</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">96.0-100%</td><td colspan="1" rowspan="1">460/460</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">99.2-100%</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">147/148</td><td colspan="1" rowspan="1">99.3</td><td colspan="1" rowspan="1">96.3-99.9%</td><td colspan="1" rowspan="1">1478/1478</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">99.7-100%</td></tr><tr><td colspan="1" rowspan="4">Proteus spp.</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">14/14</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">78.5-100%</td><td colspan="1" rowspan="1">1059/1060</td><td colspan="1" rowspan="1">99.9</td><td colspan="1" rowspan="1">99.5-100%</td></tr><tr><td colspan="1" rowspan="1">Archived</td><td colspan="1" rowspan="1">16/16</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">80.6-100%</td><td colspan="1" rowspan="1">208/208</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">98.2-100%</td></tr><tr><td colspan="1" rowspan="1">Seeded</td><td colspan="1" rowspan="1">9/9</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">70.1-100%</td><td colspan="1" rowspan="1">543/543</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">99.3-100%</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">39/39</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">91.0-100%</td><td colspan="1" rowspan="1">1810/1811†</td><td colspan="1" rowspan="1">99.9</td><td colspan="1" rowspan="1">99.7-100%</td></tr><tr><td colspan="1" rowspan="2">Salmonella spp.</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">5/5</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">56.6-100%</td><td colspan="1" rowspan="1">1069/1069</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">99.6-100%</td></tr><tr><td colspan="1" rowspan="1">Archived</td><td colspan="1" rowspan="1">16/16</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">80.6-100%</td><td colspan="1" rowspan="1">125/125</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">97.0-100%</td></tr><tr><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="1">Seeded</td><td colspan="1" rowspan="1">37/37</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">90.6-100%</td><td colspan="1" rowspan="1">515/515</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">99.3-100%</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">58/58</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">93.8-100%</td><td colspan="1" rowspan="1">1709/1709</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">99.8-100%</td></tr><tr><td colspan="1" rowspan="3">Serratia marcescens</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">11/11</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">74.1-100%</td><td colspan="1" rowspan="1">1063/1063</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">99.6-100%</td></tr><tr><td colspan="1" rowspan="1">Archived</td><td colspan="1" rowspan="1">16/16</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">80.6-100%</td><td colspan="1" rowspan="1">220/220</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">98.3-100%</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">27/27</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">87.5-100%</td><td colspan="1" rowspan="1">1283/1283</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">99.7-100%</td></tr></table>

a Archived testing not performed for Enterobacterales, E. coli, or K. pneumoniae group; seeded testing not performed for S. marcescens b Fifty-three (53) of 54 FP Enterobacterales results were attributed to the presence of nucleic acid from non-viable E. coli in specific lots of blood culture bottles. The remaining FP specimen was observed in a blood culture bottle from a different manufacturer; an Enterobacterales organism (E. coli) was detected in this specimen using an additional molecular method The single FN specimen was negative for $E .$ . coli when tested with Luminex Verigene BC-GN test The two FP specimens were attributed to the presence of nucleic acid from non-viable E. coli in the blood culture bottles $\kappa .$ pneumoniae group was detected in the single FN specimen using an additional molecular method Proteus spp. was detected in the single FP specimen using an additional molecular method

Table 19. BioFire BCID2 Panel Clinical Performance Summary, Haemophilus influenzae   

<table><tr><td rowspan=2 colspan=1>Analyte</td><td rowspan=2 colspan=1>Studya</td><td rowspan=1 colspan=3>Sensitivity</td><td rowspan=1 colspan=3>Specificity</td></tr><tr><td rowspan=1 colspan=1>TP/(TP + FN)</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95%CI</td><td rowspan=1 colspan=1>TN/(TN + FP)</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95%CI</td></tr><tr><td rowspan=3 colspan=1>Haemophilus influenzae</td><td rowspan=1 colspan=1>Prospective</td><td rowspan=1 colspan=1>8/8</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>67.6-100%</td><td rowspan=1 colspan=1>1066/1066</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>99.6-100%</td></tr><tr><td rowspan=1 colspan=1>Archived</td><td rowspan=1 colspan=1>24/25</td><td rowspan=1 colspan=1>96.0</td><td rowspan=1 colspan=1>80.5-99.3%</td><td rowspan=1 colspan=1>211/211</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>98.2-100%</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>32/33b</td><td rowspan=1 colspan=1>97.0</td><td rowspan=1 colspan=1>84.7-99.5%</td><td rowspan=1 colspan=1>1277/1277</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>99.7-100%</td></tr></table>

a Seeded testing not performed for H. influenzae b The single FN specimen was determined to contain a novel deletion in the BioFire BCID2 Panel assay target gene region

Table 20. BioFire BCID2 Panel Clinical Performance Summary, Neisseria meningitidis   

<table><tr><td rowspan=2 colspan=1>Analyte</td><td rowspan=2 colspan=1>Study</td><td rowspan=1 colspan=3>Sensitivity</td><td rowspan=1 colspan=3>Specificity</td></tr><tr><td rowspan=1 colspan=1>TP/(TP + FN)</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95%CI</td><td rowspan=1 colspan=1>TN/(TN + FP)</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95%CI</td></tr><tr><td rowspan=4 colspan=1>Neisseria meningitidis</td><td rowspan=1 colspan=1>Prospective</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>1074/1074</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>99.6-100%</td></tr><tr><td rowspan=1 colspan=1>Archived</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>43.9-100%</td><td rowspan=1 colspan=1>233/233</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>98.4-100%</td></tr><tr><td rowspan=1 colspan=1>Seeded</td><td rowspan=1 colspan=1>35/35</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>90.1-100%</td><td rowspan=1 colspan=1>517/517</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>99.3-100%</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>38/38</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>90.8-100%</td><td rowspan=1 colspan=1>1824/1824</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>99.8-100%</td></tr></table>

Table 21. BioFire BCID2 Panel Clinical Performance Summary, Pseudomonas aeruginosa   

<table><tr><td rowspan=2 colspan=1>Analyte</td><td rowspan=2 colspan=1>Studya</td><td rowspan=1 colspan=3>Sensitivity</td><td rowspan=1 colspan=3>Specificity</td></tr><tr><td rowspan=1 colspan=1>TP/(TP + FN)</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95%CI</td><td rowspan=1 colspan=1>TN/(TN + FP)</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95%CI</td></tr><tr><td rowspan=3 colspan=1>Pseudomonas aeruginosa</td><td rowspan=1 colspan=1>Prospective</td><td rowspan=1 colspan=1>29/29</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>88.3-100%</td><td rowspan=1 colspan=1>1043/1045</td><td rowspan=1 colspan=1>99.8</td><td rowspan=1 colspan=1>99.3-99.9%</td></tr><tr><td rowspan=1 colspan=1>Seeded</td><td rowspan=1 colspan=1>24/26</td><td rowspan=1 colspan=1>92.3</td><td rowspan=1 colspan=1>75.9-97.9%</td><td rowspan=1 colspan=1>526/526</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>99.3-100%</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>53/55b</td><td rowspan=1 colspan=1>96.4</td><td rowspan=1 colspan=1>87.7-99.0%</td><td rowspan=1 colspan=1>1569/1571c</td><td rowspan=1 colspan=1>99.9</td><td rowspan=1 colspan=1>99.5-100%</td></tr></table>

a Archived testing not performed for P. aeruginosa b 16/16 single seeded specimens were TP, and $8 / 1 0$ specimens that were co-seeded with E. faecalis were detected. E. faecalis was detected in $1 0 / 1 0$ of the coseeded specimens c P. aeruginosa was detected in both FP specimens using an additional molecular method

Table 22. BioFire BCID2 Panel Clinical Performance Summary, Stenotrophomonas maltophilia   

<table><tr><td rowspan=2 colspan=1>Analyte</td><td rowspan=2 colspan=1>Study</td><td rowspan=1 colspan=3>Sensitivity</td><td rowspan=1 colspan=3>Specificity</td></tr><tr><td rowspan=1 colspan=1>TP/(TP + FN)</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95%CI</td><td rowspan=1 colspan=1>TN/(TN + FP)</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95%CI</td></tr><tr><td rowspan=4 colspan=1>Stenotrophomonas maltophilia</td><td rowspan=1 colspan=1>Prospective</td><td rowspan=1 colspan=1>718</td><td rowspan=1 colspan=1>87.5</td><td rowspan=1 colspan=1>52.9-97.8%</td><td rowspan=1 colspan=1>1066/1066</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>99.6-100%</td></tr><tr><td rowspan=1 colspan=1>Archived</td><td rowspan=1 colspan=1>22/23</td><td rowspan=1 colspan=1>95.7</td><td rowspan=1 colspan=1>79.0-99.2%</td><td rowspan=1 colspan=1>116/116</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>96.8-100%</td></tr><tr><td rowspan=1 colspan=1>Seeded</td><td rowspan=1 colspan=1>25/30a</td><td rowspan=1 colspan=1>83.3</td><td rowspan=1 colspan=1>66.4-92.7%</td><td rowspan=1 colspan=1>522/522</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>99.3-100%</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>54/61b</td><td rowspan=1 colspan=1>88.5</td><td rowspan=1 colspan=1>78.2-94.3%</td><td rowspan=1 colspan=1>1704/1704</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>99.8-100%</td></tr></table>

a20/20 single seeded specimens were TP, and 5/10 specimens that were co-seeded with S. aureus were detected. S. aureus was detected in 10/10 of the co-seeded specimens.

S. maltophilia was detected in 2/7 FN specimens: one was detected using an additional molecular method and one was detected upon BioFire BCID2 Panel retest; the remaining five FN specimens were polymicrobial seeded specimens

Table 23. BioFire BCID2 Panel Clinical Performance Summary, Candida spp.   

<table><tr><td rowspan=2 colspan=1>Analyte</td><td rowspan=2 colspan=1>Studya</td><td rowspan=1 colspan=3>Sensitivity</td><td rowspan=1 colspan=3>Specificity</td></tr><tr><td rowspan=1 colspan=1>TP/(TP + FN)</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95%CI</td><td rowspan=1 colspan=1>TN/(TN + FP)</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95%CI</td></tr><tr><td rowspan=4 colspan=1>Candida albicans</td><td rowspan=1 colspan=1>Prospective</td><td rowspan=1 colspan=1>12/12</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>75.8-100%</td><td rowspan=1 colspan=1>1061/1062</td><td rowspan=1 colspan=1>99.9</td><td rowspan=1 colspan=1>99.5-100%</td></tr><tr><td rowspan=1 colspan=1>Archived</td><td rowspan=1 colspan=1>16/16</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>80.6-100%</td><td rowspan=1 colspan=1>218/218</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>98.3-100%</td></tr><tr><td rowspan=1 colspan=1>Seeded</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>72.2-100%</td><td rowspan=1 colspan=1>542/542</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>99.3-100%</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>38/38</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>90.8-100%</td><td rowspan=1 colspan=1>1821/1822b</td><td rowspan=1 colspan=1>99.9</td><td rowspan=1 colspan=1>99.7-100%</td></tr><tr><td rowspan=4 colspan=1>Candida auris</td><td rowspan=1 colspan=1>Prospective</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>1074/1074</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>99.6-100%</td></tr><tr><td rowspan=1 colspan=1>Archived</td><td rowspan=1 colspan=1>1/1</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>13/13</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>77.2-100%</td></tr><tr><td rowspan=1 colspan=1>Seeded</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>88.6-100%</td><td rowspan=1 colspan=1>522/522</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>99.3-100%</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>31/31</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>89.0-100%</td><td rowspan=1 colspan=1>1609/1609</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>99.8-100%</td></tr><tr><td rowspan=3 colspan=1>Candida glabrata</td><td rowspan=1 colspan=1>Prospective</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>72.2-100%</td><td rowspan=1 colspan=1>1063/1064</td><td rowspan=1 colspan=1>99.9</td><td rowspan=1 colspan=1>99.5-100%</td></tr><tr><td rowspan=1 colspan=1>Archived</td><td rowspan=1 colspan=1>16/16</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>80.6-100%</td><td rowspan=1 colspan=1>217/218</td><td rowspan=1 colspan=1>99.5</td><td rowspan=1 colspan=1>97.4-99.9%</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>26/26</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>87.1-100%</td><td rowspan=1 colspan=1>1280/1282c</td><td rowspan=1 colspan=1>99.8</td><td rowspan=1 colspan=1>99.4-100%</td></tr><tr><td rowspan=4 colspan=1>Candida krusei</td><td rowspan=1 colspan=1>Prospective</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>34.2-100%</td><td rowspan=1 colspan=1>1072/1072</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>99.6-100%</td></tr><tr><td rowspan=1 colspan=1>Archived</td><td rowspan=1 colspan=1>9/9</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>70.1-100%</td><td rowspan=1 colspan=1>227/227</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>98.3-100%</td></tr><tr><td rowspan=1 colspan=1>Seeded</td><td rowspan=1 colspan=1>33/33</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>89.6-100%</td><td rowspan=1 colspan=1>519/519</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>99.3-100%</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>44/44</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>92.0-100%</td><td rowspan=1 colspan=1>1818/1818</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>99.8-100%</td></tr><tr><td rowspan=3 colspan=1>Candida parapsilosis</td><td rowspan=1 colspan=1>Prospective</td><td rowspan=1 colspan=1>8/8</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>67.6-100%</td><td rowspan=1 colspan=1>1065/1066</td><td rowspan=1 colspan=1>99.9</td><td rowspan=1 colspan=1>99.5-100%</td></tr><tr><td rowspan=1 colspan=1>Archived</td><td rowspan=1 colspan=1>22/23</td><td rowspan=1 colspan=1>95.7</td><td rowspan=1 colspan=1>79.0-99.2%</td><td rowspan=1 colspan=1>211/211</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>98.2-100%</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>30/31d</td><td rowspan=1 colspan=1>96.8</td><td rowspan=1 colspan=1>83.8-99.4%</td><td rowspan=1 colspan=1>1276/1277e</td><td rowspan=1 colspan=1>99.9</td><td rowspan=1 colspan=1>99.6-100%</td></tr><tr><td rowspan=4 colspan=1>Candida tropicalis</td><td rowspan=1 colspan=1>Prospective</td><td rowspan=1 colspan=1>5/5</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>56.6-100%</td><td rowspan=1 colspan=1>1069/1069</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>99.6-100%</td></tr><tr><td rowspan=1 colspan=1>Archived</td><td rowspan=1 colspan=1>15/15</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>79.6-100%</td><td rowspan=1 colspan=1>219/220</td><td rowspan=1 colspan=1>99.5</td><td rowspan=1 colspan=1>97.5-99.9%</td></tr><tr><td rowspan=1 colspan=1>Seeded</td><td rowspan=1 colspan=1>35/35</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>90.1-100%</td><td rowspan=1 colspan=1>517/517</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>99.3-100%</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>55/55</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>93.5-100%</td><td rowspan=1 colspan=1>1805/1806f</td><td rowspan=1 colspan=1>99.9</td><td rowspan=1 colspan=1>99.7-100%</td></tr></table>

a Seeded testing was not performed for C. glabrata or C. parapsilosis b C. albicans was detected in the single FP specimen using an additional molecular method C. glabrata was detected in both FP specimens using an additional molecular method The single FN specimen was misidentified as C. parapsilosis by the source laboratory; molecular testing of the specimen identified it as C. orthopsilosis C. parapsilosis was detected in the single FP specimen using an additional molecular method The single FP specimen was identified as a cross-reactivity between the BioFire BCID2 Panel Ctropicalis assay and high titer C. parapsilosis (refer to Analytical Specificity section for additional information regarding this cross-reactivity)

Table 24. BioFire BCID2 Panel Clinical Performance Summary, Cryptococcus neoformans/gattii   

<table><tr><td rowspan=2 colspan=1>Analyte</td><td rowspan=2 colspan=1>Study</td><td rowspan=1 colspan=3>Sensitivity</td><td rowspan=1 colspan=3>Specificity</td></tr><tr><td rowspan=1 colspan=1>TP/(TP + FN)</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95%CI</td><td rowspan=1 colspan=1>TN/(TN + FP)</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95%CI</td></tr><tr><td rowspan=4 colspan=1>Cryptococcus neoformans/gatti</td><td rowspan=1 colspan=1>Prospective</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>1074/1074</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>99.6-100%</td></tr><tr><td rowspan=1 colspan=1>Archived</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>61.0-100%</td><td rowspan=1 colspan=1>135/135</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>97.2-100%</td></tr><tr><td rowspan=1 colspan=1>Seeded</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>88.6-100%</td><td rowspan=1 colspan=1>522/522</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>99.3-100%</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>36/36</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>90.4-100%</td><td rowspan=1 colspan=1>1731/1731</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>99.8-100%</td></tr></table>

BioFire BCID2 Panel assay performance stratified by species for Staphylococcus spp., Streptococcus spp., Enterobacterales, Enterobacter cloacae complex, Klebsiella pneumoniae group, Proteus spp., Salmonella spp., and Cryptococcus neoformans/gattii BioFire BCID2 Panel genus and group level organism results are presented in Table 25 through Table 28. Note: multiple organisms from a group may be detected in a single specimen, therefore the “Total” values in these tables may not match the performance values presented above, which are reported per specimen.

Table 25. Stratification of Staphylococcus spp. Assay Performance by Species   

<table><tr><td rowspan=2 colspan=1>Species</td><td rowspan=1 colspan=3>Sensitivity</td></tr><tr><td rowspan=1 colspan=1>Prospective</td><td rowspan=1 colspan=1>Archived</td><td rowspan=1 colspan=1>Seeded</td></tr><tr><td rowspan=1 colspan=1>S. aureus</td><td rowspan=1 colspan=1>149/149 (100%)</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>10/10 (100%)</td></tr><tr><td rowspan=1 colspan=1>S. auricularis</td><td rowspan=1 colspan=1>1/1 (100%)</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>S. capitis</td><td rowspan=1 colspan=1>19/19 (100%)</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>S. caprae</td><td rowspan=1 colspan=1>4/4 (100%)</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>S. epidermidis</td><td rowspan=1 colspan=1>229/229 (100%)</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>S. haemolyticus</td><td rowspan=1 colspan=1>24/24 (100%)</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>S. hominis</td><td rowspan=1 colspan=1>52/53 (98.1%)</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>S. lentus</td><td rowspan=1 colspan=1>1/1 (100%)</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>S. lugdunensis</td><td rowspan=1 colspan=1>4/4 (100%)</td><td rowspan=1 colspan=1>16/16 (100%)</td><td rowspan=1 colspan=1>30/30 (100%)</td></tr><tr><td rowspan=1 colspan=1>S. pettenkoferi</td><td rowspan=1 colspan=1>10/10 (100%)</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>S. simulans</td><td rowspan=1 colspan=1>3/3 (100%)</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>S. warneri</td><td rowspan=1 colspan=1>2/2 (100%)</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Total Staphylococcus spp.</td><td rowspan=1 colspan=1>498/499 (99.8%)95%CI: 98.9-100%</td><td rowspan=1 colspan=1>16/16 (100%)95%CI: 80.6-100%</td><td rowspan=1 colspan=1>40/40 (100%)95% CI: 91.2-100%</td></tr></table>

Table 26. Stratification of Streptococcus spp. Assay Performance by Species   

<table><tr><td colspan="1" rowspan="2">Species</td><td colspan="3" rowspan="1">Sensitivity</td></tr><tr><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">Archived</td><td colspan="1" rowspan="1">Seeded</td></tr><tr><td colspan="4" rowspan="1">Group A (Pyogenic)</td></tr><tr><td colspan="1" rowspan="1">S. pyogenes</td><td colspan="1" rowspan="1">13/14 (92.9%)</td><td colspan="1" rowspan="1">16/16 (100%)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="4" rowspan="1">Group B (Pyogenic)</td></tr><tr><td colspan="1" rowspan="1">S. agalactiae</td><td colspan="1" rowspan="1">9/9 (100%)</td><td colspan="1" rowspan="1">16/16 (100%)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Group C/G (Pyogenic)</td><td colspan="2" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">S. canis</td><td colspan="1" rowspan="1">1/1 (100%)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">S. dysgalactiae</td><td colspan="1" rowspan="1">5/5 (100%)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Group D (Bovis)</td><td colspan="2" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">S. bovis group</td><td colspan="1" rowspan="1">2/2 (100%)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">S. gallolyticus</td><td colspan="1" rowspan="1">2/2 (100%)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">S. lutetiensis</td><td colspan="1" rowspan="1">1/1 (100%)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Group F (Anginosus)</td><td colspan="2" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">S. anginosus</td><td colspan="1" rowspan="1">3/4 (75.0%)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">S. anginosus group</td><td colspan="1" rowspan="1">3/3 (100%)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">S. constellatus</td><td colspan="1" rowspan="1">2/2 (100%)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">S. intermedius</td><td colspan="1" rowspan="1">2/2 (100%)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">S. vestibularis</td><td colspan="1" rowspan="1">2/2 (100%)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="4" rowspan="1">Mitis Group</td></tr><tr><td colspan="1" rowspan="1">S. gordonii</td><td colspan="1" rowspan="1">1/1 (100%)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">S. mitis</td><td colspan="1" rowspan="1">10/10 (100%)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">S. mitis/oralis</td><td colspan="1" rowspan="1">7/7 (100%)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">S. oralis</td><td colspan="1" rowspan="1">1/1 (100%)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">S. parasanguinis</td><td colspan="1" rowspan="1">6/6 (100%)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">S. pneumoniae</td><td colspan="1" rowspan="1">26/26 (100%)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="4" rowspan="1">Mutans Group</td></tr><tr><td colspan="1" rowspan="1">S. mutans</td><td colspan="1" rowspan="1">2/2 (100%)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="4" rowspan="1">Salivarius Group</td></tr><tr><td colspan="1" rowspan="1">S. salivarius</td><td colspan="1" rowspan="1">8/8 (100%)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="4" rowspan="1">Other</td></tr><tr><td colspan="1" rowspan="1">Viridans streptococci</td><td colspan="1" rowspan="1">19/19 (100%)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Total Streptococcus spp.</td><td colspan="1" rowspan="1">125/127 (98.4%)95%CI: 94.4-99.6%</td><td colspan="1" rowspan="1">32/32 (100%)95%CI: 89.3-100%</td><td colspan="1" rowspan="1">.</td></tr></table>

Table 27. Stratification of Enterobacterales Assay Performance by Species   

<table><tr><td colspan="1" rowspan="2">Species</td><td colspan="1" rowspan="1"></td><td colspan="2" rowspan="1">Sensitivity</td></tr><tr><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">Archived</td><td colspan="1" rowspan="1">Seeded</td></tr><tr><td colspan="4" rowspan="1">Enterobacter cloacae complex</td></tr><tr><td colspan="1" rowspan="1">Enterobacter cloacae</td><td colspan="1" rowspan="1">10/10 (100%)</td><td colspan="1" rowspan="1">14/14 (100%)</td><td colspan="1" rowspan="1">8/8 (100%)</td></tr><tr><td colspan="1" rowspan="1">Enterobacter cloacae complex</td><td colspan="1" rowspan="1">3/3 (100%)</td><td colspan="1" rowspan="1">2/2 (100%)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Enterobacter hormaechei</td><td colspan="1" rowspan="1">2/2 (100%)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Enterobacter kobei</td><td colspan="1" rowspan="1">1/1 (100%)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Total</td><td colspan="1" rowspan="1">16/16 (100%)95%CI: 80.6-100%</td><td colspan="1" rowspan="1">16/16 (100%)95%CI: 80.6-100%</td><td colspan="1" rowspan="1">8/8 (100%)95%CI: 67.6-100%</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae group</td><td colspan="2" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">51/51 (100%)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">92/92 (100%)</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae group</td><td colspan="1" rowspan="1">3/3 (100%)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Klebsiella variicola</td><td colspan="1" rowspan="1">2/2 (100%)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Total</td><td colspan="1" rowspan="1">56/56 (100%)95%CI: 93.6-100%</td><td colspan="1" rowspan="1">.</td><td colspan="1" rowspan="1">92/92 (100%)95%CI: 96.0-100%</td></tr><tr><td colspan="1" rowspan="1">Proteus spp.</td><td colspan="2" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Proteus mirabilis</td><td colspan="1" rowspan="1">13/13 (100%)</td><td colspan="1" rowspan="1">14/14 (100%)</td><td colspan="1" rowspan="1">9/9 (100%)</td></tr><tr><td colspan="1" rowspan="1">Proteus penneri</td><td colspan="1" rowspan="1">1/1 (100%)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Proteus vulgaris</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1/1 (100%)</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Proteus sp.</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1/1 (100%)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Total</td><td colspan="1" rowspan="1">14/14 (100%)95%CI: 78.5-100%</td><td colspan="1" rowspan="1">16/16 (100%)95%CI: 80.6-100%</td><td colspan="1" rowspan="1">9/9 (100%)95%CI: 70.1-100%</td></tr><tr><td colspan="1" rowspan="1">Salmonella spp.</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Salmonella enterica sv. Berta</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">3/3 (100%)</td></tr><tr><td colspan="1" rowspan="1">Salmonella enterica sv. Enteritidis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">7/7 (100%)</td></tr><tr><td colspan="1" rowspan="1">Salmonella enterica sv. Javiana</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">3/3 (100%)</td></tr><tr><td colspan="1" rowspan="1">Salmonella enterica sv. Newport</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1/1 (100%)</td><td colspan="1" rowspan="1">3/3 (100%)</td></tr><tr><td colspan="1" rowspan="1">Salmonella enterica sv. Paratyphi</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1/1 (100%)</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Salmonella enterica sv. Senftenberg</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">3/3 (100%)</td></tr><tr><td colspan="1" rowspan="1">Salmonella enterica sv. Tel-el-kebir</td><td colspan="1" rowspan="1">1/1 (100%)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Salmonella enterica sv. Typhi</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1/1 (100%)</td><td colspan="1" rowspan="1">3/3 (100%)</td></tr><tr><td colspan="1" rowspan="1">Salmonella enterica sv. Typhimurium</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">3/3 (100%)</td></tr><tr><td colspan="1" rowspan="1">Salmonella enterica serogroup B</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1/1 (100%)</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="2">Species</td><td colspan="3" rowspan="1">Sensitivity</td></tr><tr><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">Archived</td><td colspan="1" rowspan="1">Seeded</td></tr><tr><td colspan="1" rowspan="1">Salmonella enterica serogroup C</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">3/3 (100%)</td></tr><tr><td colspan="1" rowspan="1">Salmonella enterica</td><td colspan="1" rowspan="1">2/2 (100%)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Salmonella spp.</td><td colspan="1" rowspan="1">2/2 (100%)</td><td colspan="1" rowspan="1">12/12 (100%)</td><td colspan="1" rowspan="1">9/9 (100%)</td></tr><tr><td colspan="1" rowspan="1">Total</td><td colspan="1" rowspan="1">5/5 (100%)95%CI: 56.6-100%</td><td colspan="1" rowspan="1">16/16 (100%)95%CI: 80.6-100%</td><td colspan="1" rowspan="1">37/37 (100%)95%CI: 90.6-100%</td></tr><tr><td colspan="1" rowspan="1">Other</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Citrobacter amalonaticus</td><td colspan="1" rowspan="1">1/1 (100%)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Citrobacter freundii</td><td colspan="1" rowspan="1">1/1 (100%)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Enterobacter amnigenus</td><td colspan="1" rowspan="1">1/1 (100%)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">159/159 (100%)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">44/44 (100%)</td></tr><tr><td colspan="1" rowspan="1">Hafnia alvei</td><td colspan="1" rowspan="1">1/1 (100%)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Klebsiella aerogenes</td><td colspan="1" rowspan="1">2/2 (100%)</td><td colspan="1" rowspan="1">16/16 (100%)</td><td colspan="1" rowspan="1">42/42 (100%)</td></tr><tr><td colspan="1" rowspan="1">Klebsiella oxytoca</td><td colspan="1" rowspan="1">8/8 (100%)</td><td colspan="1" rowspan="1">16/16 (100%)</td><td colspan="1" rowspan="1">6/6 (100%)</td></tr><tr><td colspan="1" rowspan="1">Pantoea septica/agglomerans</td><td colspan="1" rowspan="1">1/1 (100%)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Providencia stuartii</td><td colspan="1" rowspan="1">2/3 (66.7%)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Serratia liquefaciens complex</td><td colspan="1" rowspan="1">1/1 (100%)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Serratia marcescens</td><td colspan="1" rowspan="1">11/11 (100%)</td><td colspan="1" rowspan="1">16/16 (100%)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Total Enterobacterales</td><td colspan="1" rowspan="1">279/280 (99.6%)95%CI: 98.0-99.9%</td><td colspan="1" rowspan="1">96/96 (100%)95%CI: 96.2-100%</td><td colspan="1" rowspan="1">238/238 (100%)95%CI: 98.4-100%</td></tr></table>

Table 28. Stratification of Cryptococcus neoformans/gattii Assay Performance by Species   

<table><tr><td rowspan=2 colspan=1>Species</td><td rowspan=1 colspan=3>Sensitivity</td></tr><tr><td rowspan=1 colspan=1>Prospective</td><td rowspan=1 colspan=1>Archived</td><td rowspan=1 colspan=1>Seeded</td></tr><tr><td rowspan=1 colspan=1>C. gattii</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>15/15 (100%)</td></tr><tr><td rowspan=1 colspan=1>C. neoformans</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>6/6 (100%)</td><td rowspan=1 colspan=1>15/15 (100%)</td></tr><tr><td rowspan=1 colspan=1>Total Cryptococcusneoformans/gattii</td><td rowspan=1 colspan=1>.</td><td rowspan=1 colspan=1>6/6 (100%)95%CI: 61.0-100%</td><td rowspan=1 colspan=1>30/30 (100%)95%CI: 88.6-100%</td></tr></table>

Antimicrobial resistance (AMR) gene results are reported only when one or more applicable bacteria that may carry the gene are also detected in the sample. If no applicable bacteria are detected, the AMR gene results are reported as Not Applicable (N/A). The results are summarized for each AMR gene in Table 29 through Table 58. Note: the “Performance Summary” tables below do not include specimens for which a potential host organism was not reported (i.e. the AMR gene was reported as N/A); these specimens are instead accounted for in the “Distribution of Clinical Specimens” tables below.

Table 29. BioFire BCID2 Panel Clinical Performance Summary, CTX-M   

<table><tr><td rowspan=2 colspan=1>Analyte</td><td rowspan=2 colspan=1>Studya</td><td rowspan=1 colspan=3>Positive Percent Agreement</td><td rowspan=1 colspan=3>Negative Percent Agreement</td></tr><tr><td rowspan=1 colspan=1>TP/(TP + FN)</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95%CI</td><td rowspan=1 colspan=1>TN(TN + FP)</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95%CI</td></tr><tr><td rowspan=3 colspan=1>CTX-M</td><td rowspan=1 colspan=1>Prospective</td><td rowspan=1 colspan=1>46/47</td><td rowspan=1 colspan=1>97.9</td><td rowspan=1 colspan=1>88.9-99.6%</td><td rowspan=1 colspan=1>312/312</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>98.8-100%</td></tr><tr><td rowspan=1 colspan=1>Seeded</td><td rowspan=1 colspan=1>63/63</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>94.3-100%</td><td rowspan=1 colspan=1>198/198</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>98.1-100%</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>109/110</td><td rowspan=1 colspan=1>99.1</td><td rowspan=1 colspan=1>95.0-99.8%</td><td rowspan=1 colspan=1>510/510</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>99.3-100%</td></tr></table>

Archived testing was not performed for CTX-M

Table 30. Distribution of CTX-M in Prospective Clinical Specimens   

<table><tr><td rowspan=2 colspan=2>CTX-M</td><td rowspan=1 colspan=4>SOC: any associated organismPCR/seq: CTX-M</td></tr><tr><td rowspan=1 colspan=1>Org+ / Res+</td><td rowspan=1 colspan=1>Org+ / Res-</td><td rowspan=1 colspan=1>Org-</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=4 colspan=1>BCID2PanelResult</td><td rowspan=1 colspan=1>Org+ / Res+</td><td rowspan=1 colspan=1>46</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>46</td></tr><tr><td rowspan=1 colspan=1>Org+ / Res-</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>258</td><td rowspan=1 colspan=1>54a</td><td rowspan=1 colspan=1>313</td></tr><tr><td rowspan=1 colspan=1>Org -</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>713</td><td rowspan=1 colspan=1>715</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>47</td><td rowspan=1 colspan=1>260</td><td rowspan=1 colspan=1>767</td><td rowspan=1 colspan=1>1074</td></tr><tr><td rowspan=1 colspan=3>Performance</td><td rowspan=1 colspan=1>Agreement</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95%CI</td></tr><tr><td rowspan=1 colspan=3>Org+ / Res+</td><td rowspan=1 colspan=1>46/47</td><td rowspan=1 colspan=1>97.9</td><td rowspan=1 colspan=1>88.9-99.6%</td></tr><tr><td rowspan=1 colspan=3>Org+ / Res-</td><td rowspan=1 colspan=1>258/260</td><td rowspan=1 colspan=1>99.2</td><td rowspan=1 colspan=1>97.2-99.8%</td></tr><tr><td rowspan=1 colspan=3>Org -</td><td rowspan=1 colspan=1>713/767a</td><td rowspan=1 colspan=1>93.0</td><td rowspan=1 colspan=1>90.9-94.6%</td></tr></table>

a Fifty-three (53) FP results due to the presence of nucleic acid from non-viable E. coli in the blood culture bottles

Table 31. Stratification of CTX-M Clinical Performance by Associated Host Organism   

<table><tr><td colspan="1" rowspan="2">Analyte</td><td colspan="1" rowspan="2">Studya</td><td colspan="3" rowspan="1">Positive Percent Agreement</td><td colspan="3" rowspan="1">Negative Percent Agreement</td></tr><tr><td colspan="1" rowspan="1">TP/(TP + FN)</td><td colspan="1" rowspan="1">%</td><td colspan="1" rowspan="1">95%CI</td><td colspan="1" rowspan="1">TN(TN + FP)</td><td colspan="1" rowspan="1">%</td><td colspan="1" rowspan="1">95%CI</td></tr><tr><td colspan="1" rowspan="3">Overall(any associated organism identified)</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">46/47</td><td colspan="1" rowspan="1">97.9</td><td colspan="1" rowspan="1">88.9-99.6%</td><td colspan="1" rowspan="1">312/312</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">98.8-100%</td></tr><tr><td colspan="1" rowspan="1">Seeded</td><td colspan="1" rowspan="1">63/63</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">94.3-100%</td><td colspan="1" rowspan="1">198/198</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">98.1-100%</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">109/110</td><td colspan="1" rowspan="1">99.1</td><td colspan="1" rowspan="1">95.0-99.8%</td><td colspan="1" rowspan="1">510/510</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">99.3-100%</td></tr><tr><td colspan="1" rowspan="3">Acinetobacter calcoaceticus-baumanniicomplex</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">12/12</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">75.8-100%</td></tr><tr><td colspan="1" rowspan="1">Seeded</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">19/19</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">83.2-100%</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">•</td><td colspan="1" rowspan="1">31/31</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">89.0-100%</td></tr><tr><td colspan="1" rowspan="3">Enterobacterales</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">46/47</td><td colspan="1" rowspan="1">97.9</td><td colspan="1" rowspan="1">88.9-99.6%</td><td colspan="1" rowspan="1">276/276</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">98.6-100%</td></tr><tr><td colspan="1" rowspan="1">Seeded</td><td colspan="1" rowspan="1">63/63</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">94.3-100%</td><td colspan="1" rowspan="1">165/165</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">97.7-100%</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">109/110</td><td colspan="1" rowspan="1">99.1</td><td colspan="1" rowspan="1">95.0-99.8%</td><td colspan="1" rowspan="1">441/441</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">99.1-100%</td></tr><tr><td colspan="1" rowspan="3">Enterobacter cloacae complex</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">16/16</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">80.6-100%</td></tr><tr><td colspan="1" rowspan="1">Seeded</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">43.9-100%</td><td colspan="1" rowspan="1">5/5</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">56.6-100%</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">43.9-100%</td><td colspan="1" rowspan="1">21/21</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">84.5-100%</td></tr><tr><td colspan="1" rowspan="3">Escherichia coli</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">88.6-100%</td><td colspan="1" rowspan="1">130/130</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">97.1-100%</td></tr><tr><td colspan="1" rowspan="1">Seeded</td><td colspan="1" rowspan="1">11/11</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">74.1-100%</td><td colspan="1" rowspan="1">33/33</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">89.6-100%</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">41/41</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">91.4-100%</td><td colspan="1" rowspan="1">163/163</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">97.7-100%</td></tr><tr><td colspan="1" rowspan="3">Klebsiella aerogenes</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Seeded</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">43.9-100%</td><td colspan="1" rowspan="1">39/39</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">91.0-100%</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">51.0-100%</td><td colspan="1" rowspan="1">40/40</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">91.2-100%</td></tr><tr><td colspan="1" rowspan="3">Klebsiella oxytoca</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">8/8</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">67.6-100%</td></tr><tr><td colspan="1" rowspan="1">Seeded</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">43.9-100%</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">43.9-100%</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">43.9-100%</td><td colspan="1" rowspan="1">11/11</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">74.1-100%</td></tr><tr><td colspan="1" rowspan="3">Klebsiella pneumoniae group</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">12/12</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">75.8-100%</td><td colspan="1" rowspan="1">43/43</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">91.8-100%</td></tr><tr><td colspan="1" rowspan="1">Seeded</td><td colspan="1" rowspan="1">40/40</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">91.2-100%</td><td colspan="1" rowspan="1">52/52</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">93.1-100%</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">52/52</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">93.1-100%</td><td colspan="1" rowspan="1">95/95</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">96.1-100%</td></tr><tr><td colspan="1" rowspan="1">Proteus spp.</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">51.0-100%</td><td colspan="1" rowspan="1">11/11</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">74.1-100%</td></tr><tr><td colspan="1" rowspan="1">TP/(TP + FN)</td><td colspan="1" rowspan="1">%</td><td colspan="1" rowspan="1">95%CI</td><td colspan="1" rowspan="1">TN/(TN + FP)</td><td colspan="1" rowspan="1">%</td><td colspan="1" rowspan="1">95%CI</td></tr><tr><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="1">Seeded</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">43.9-100%</td><td colspan="1" rowspan="1">6/6</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">61.0-100%</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">7/7</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">64.6-100%</td><td colspan="1" rowspan="1">17/17</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">81.6-100%</td></tr><tr><td colspan="1" rowspan="3">Salmonella spp.</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">5/5</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">56.6-100%</td></tr><tr><td colspan="1" rowspan="1">Seeded</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">37/37</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">90.6-100%</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">•</td><td colspan="1" rowspan="1">42/42</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">91.6-100%</td></tr><tr><td colspan="1" rowspan="2">Serratia marcescens</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">0/1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">72.2-100%</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">0/1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">•</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">72.2-100%</td></tr><tr><td colspan="1" rowspan="3">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">2/2</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">34.2-100%</td><td colspan="1" rowspan="1">29/29</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">88.3-100%</td></tr><tr><td colspan="1" rowspan="1">Seeded</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">24/24</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">86.2-100%</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">2/2</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">34.2-100%</td><td colspan="1" rowspan="1">53/53</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">93.2-100%</td></tr></table>

Archived testing not performed for CTX-M; seeded testing not performed for CTX-M with S. marcescens

Table 32. BioFire BCID2 Panel Clinical Performance Summary, IMP   

<table><tr><td rowspan=2 colspan=1>Analyte</td><td rowspan=2 colspan=1>Studya</td><td rowspan=1 colspan=3>Positive Percent Agreement</td><td rowspan=1 colspan=3>Negative Percent Agreement</td></tr><tr><td rowspan=1 colspan=1>TP/(TP + FN)</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95%CI</td><td rowspan=1 colspan=1>TN(TN + FP)</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95%CI</td></tr><tr><td rowspan=3 colspan=1>IMP</td><td rowspan=1 colspan=1>Prospective</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>359/359</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>98.9-100%</td></tr><tr><td rowspan=1 colspan=1>Seeded</td><td rowspan=1 colspan=1>29/29</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>88.3-100%</td><td rowspan=1 colspan=1>232/232</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>98.4-100%</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>29/29</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>88.3-100%</td><td rowspan=1 colspan=1>591/591</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>99.4-100%</td></tr></table>

Archived testing not performed for IMP

Table 33. Distribution of IMP in Prospective Clinical Specimens   

<table><tr><td rowspan=2 colspan=2>IMP</td><td rowspan=1 colspan=4>SOC: any associated organismPCR/seq: IMP</td></tr><tr><td rowspan=1 colspan=1>Org+/Res+</td><td rowspan=1 colspan=1>Org+/ Res-</td><td rowspan=1 colspan=1>Org -</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=4 colspan=1>BCID2PanelResult</td><td rowspan=1 colspan=1>Org+ / Res+</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Org+ / Res-</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>305</td><td rowspan=1 colspan=1>54a</td><td rowspan=1 colspan=1>359</td></tr><tr><td rowspan=1 colspan=1>Org -</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>713</td><td rowspan=1 colspan=1>715</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>307</td><td rowspan=1 colspan=1>767</td><td rowspan=1 colspan=1>1074</td></tr><tr><td rowspan=1 colspan=3>Performance</td><td rowspan=1 colspan=1>Agreement</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95%CI</td></tr><tr><td rowspan=1 colspan=3>Org+ / Res+</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=3>Org+ / Res-</td><td rowspan=1 colspan=1>305/307</td><td rowspan=1 colspan=1>99.3</td><td rowspan=1 colspan=1>97.7-99.8%</td></tr><tr><td rowspan=1 colspan=3>Org -</td><td rowspan=1 colspan=1>713/767a</td><td rowspan=1 colspan=1>93.0</td><td rowspan=1 colspan=1>90.9-94.6%</td></tr></table>

Fifty-three (53) FP results due to the presence of nucleic acid from non-viable E. coli in the blood culture bottles

Table 34. Stratification of IMP Clinical Performance by Associated Host Organism, Seeded studya   

<table><tr><td rowspan=2 colspan=1>Analyte</td><td rowspan=1 colspan=3>Positive Percent Agreement</td><td rowspan=1 colspan=3>Negative Percent Agreement</td></tr><tr><td rowspan=1 colspan=1>TP/(TP + FN)</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95%CI</td><td rowspan=1 colspan=1>TN/(TN + FP)</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95%CI</td></tr><tr><td rowspan=1 colspan=1>Overall(any associated organism identified)</td><td rowspan=1 colspan=1>29/29</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>88.3-100%</td><td rowspan=1 colspan=1>232/232</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>98.4-100%</td></tr><tr><td rowspan=1 colspan=1>Acinetobacter calcoaceticus-baumannii complex</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>43.9-100%</td><td rowspan=1 colspan=1>16/16</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>80.6-100%</td></tr><tr><td rowspan=1 colspan=1>Enterobacterales</td><td rowspan=1 colspan=1>19/19</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>83.2-100%</td><td rowspan=1 colspan=1>209/209</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>98.2-100%</td></tr><tr><td rowspan=1 colspan=1>Enterobacter cloacae complex</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>8/8</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>67.6-100%</td></tr><tr><td rowspan=1 colspan=1>Escherichia coli</td><td rowspan=1 colspan=1>9/9</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>70.1-100%</td><td rowspan=1 colspan=1>35/35</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>90.1-100%</td></tr><tr><td rowspan=1 colspan=1>Klebsiella aerogenes</td><td rowspan=1 colspan=1>4/4</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>51.0-100%</td><td rowspan=1 colspan=1>38/38</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>90.8-100%</td></tr><tr><td rowspan=1 colspan=1>Klebsiella oxytoca</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>61.0-100%</td></tr><tr><td rowspan=1 colspan=1>Klebsiella pneumoniae group</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>61.0-100%</td><td rowspan=1 colspan=1>86/86</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>95.7-100%</td></tr><tr><td rowspan=1 colspan=1>Proteus spp.</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>9/9</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>70.1-100%</td></tr><tr><td rowspan=1 colspan=1>Salmonella spp.</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>37/37</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>90.6-100%</td></tr><tr><td rowspan=1 colspan=1>Pseudomonas aeruginosa</td><td rowspan=1 colspan=1>717</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>64.6-100%</td><td rowspan=1 colspan=1>17/17</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>81.6-100%</td></tr></table>

No observations for IMP in the prospective evaluation; archived testing not performed; seeded testing not performed for IMP with S. marcescens

Table 35. BioFire BCID2 Panel Clinical Performance Summary, KPC   

<table><tr><td rowspan=2 colspan=1>Analyte</td><td rowspan=2 colspan=1>Study</td><td rowspan=1 colspan=3>Positive Percent Agreement</td><td rowspan=1 colspan=3>Negative Percent Agreement</td></tr><tr><td rowspan=1 colspan=1>TP/(TP + FN)</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95%CI</td><td rowspan=1 colspan=1>TN/(TN + FP)</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95%CI</td></tr><tr><td rowspan=4 colspan=1>KPC</td><td rowspan=1 colspan=1>Prospective</td><td rowspan=1 colspan=1>4/4</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>51.0-100%</td><td rowspan=1 colspan=1>328/328</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>98.8-100%</td></tr><tr><td rowspan=1 colspan=1>Archived</td><td rowspan=1 colspan=1>14/14</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>78.5-100%</td><td rowspan=1 colspan=1>5/5</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>56.6-100%</td></tr><tr><td rowspan=1 colspan=1>Seeded</td><td rowspan=1 colspan=1>45/45</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>92.1-100%</td><td rowspan=1 colspan=1>216/216</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>98.3-100%</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>63/63</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>94.3-100%</td><td rowspan=1 colspan=1>549/549</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>99.3-100%</td></tr></table>

Twenty-seven (27) specimens were FN for an associated host organism by the molecular comparator method, thus providing an “N/A” result for KPC

Table 36. Distribution of KPC in Prospective Clinical Specimens   

<table><tr><td rowspan=2 colspan=2>KPC</td><td rowspan=1 colspan=4>SOC: any associated organismFDA-cleared test: KPC</td></tr><tr><td rowspan=1 colspan=1>Org+/Res+</td><td rowspan=1 colspan=1>Org+/ Res-</td><td rowspan=1 colspan=1>Org -</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=4 colspan=1>BCID2PanelResult</td><td rowspan=1 colspan=1>Org+/ Res+</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>4</td></tr><tr><td rowspan=1 colspan=1>Org+ / Res-</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>298</td><td rowspan=1 colspan=1>54a</td><td rowspan=1 colspan=1>352</td></tr><tr><td rowspan=1 colspan=1>Org -</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>713</td><td rowspan=1 colspan=1>713</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>298</td><td rowspan=1 colspan=1>767</td><td rowspan=1 colspan=1>1069b</td></tr><tr><td rowspan=1 colspan=3>Performance</td><td rowspan=1 colspan=1>Agreement</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95%CI</td></tr><tr><td rowspan=1 colspan=3>Org+ / Res+</td><td rowspan=1 colspan=1>4/4</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>51.0-100%</td></tr><tr><td rowspan=1 colspan=3>Org+ / Res-</td><td rowspan=1 colspan=1>298/298</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>98.7-100%</td></tr><tr><td rowspan=1 colspan=3>Org -</td><td rowspan=1 colspan=1>713/767a</td><td rowspan=1 colspan=1>93.0</td><td rowspan=1 colspan=1>90.9-94.6%</td></tr></table>

a Fifty-three (53) FP results due to the presence of nucleic acid from non-viable E. coli in the blood culture bottles b Five specimens were FN for the associated host organism by the molecular comparator method, thus providing an “N/A” result for KPC

Table 37. Stratification of KPC Clinical Performance by Associated Host Organism   

<table><tr><td colspan="1" rowspan="2">Analyte</td><td colspan="1" rowspan="2">Studya</td><td colspan="3" rowspan="1">Positive Percent Agreement</td><td colspan="3" rowspan="1">Negative Percent Agreement</td></tr><tr><td colspan="1" rowspan="1">TP/(TP + FN)</td><td colspan="1" rowspan="1">%</td><td colspan="1" rowspan="1">95%CI</td><td colspan="1" rowspan="1">TN/(TN + FP)</td><td colspan="1" rowspan="1">%</td><td colspan="1" rowspan="1">95%CI</td></tr><tr><td colspan="1" rowspan="4">Overall(any associated organism identified)</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">51.0-100%</td><td colspan="1" rowspan="1">328/328</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">98.8-100%</td></tr><tr><td colspan="1" rowspan="1">Archived</td><td colspan="1" rowspan="1">14/14</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">78.5-100%</td><td colspan="1" rowspan="1">5/5</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">56.6-100%</td></tr><tr><td colspan="1" rowspan="1">Seeded</td><td colspan="1" rowspan="1">45/45</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">92.1-100%</td><td colspan="1" rowspan="1">216/216</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">98.3-100%</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">63/63</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">94.3-100%</td><td colspan="1" rowspan="1">549/549</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">99.3-100%</td></tr><tr><td colspan="1" rowspan="2">Acinetobacter calcoaceticus-baumannii</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">13/13</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">77.2-100%</td></tr><tr><td colspan="1" rowspan="1">Archived</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">5/5</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">56.6-100%</td></tr><tr><td colspan="1" rowspan="2">complex</td><td colspan="1" rowspan="1">Seeded</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">19/19</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">83.2-100%</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">•</td><td colspan="1" rowspan="1">•</td><td colspan="1" rowspan="1">37/37</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">90.6-100%</td></tr><tr><td colspan="1" rowspan="4">Enterobacterales</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">51.0-100%</td><td colspan="1" rowspan="1">292/292</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">98.7-100%</td></tr><tr><td colspan="1" rowspan="1">Archived</td><td colspan="1" rowspan="1">14/14</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">78.5-100%</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Seeded</td><td colspan="1" rowspan="1">42/42</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">91.6-100%</td><td colspan="1" rowspan="1">186/186</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">98.0-100%</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">60/60</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">94.0-100%</td><td colspan="1" rowspan="1">478/478</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">99.2-100%</td></tr><tr><td colspan="1" rowspan="4">Enterobacter cloacae complex</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">16/16</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">80.6-100%</td></tr><tr><td colspan="1" rowspan="1">Archived</td><td colspan="1" rowspan="1">2/2</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">34.2-100%</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Seeded</td><td colspan="1" rowspan="1">2/2</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">34.2-100%</td><td colspan="1" rowspan="1">6/6</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">61.0-100%</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">51.0-100%</td><td colspan="1" rowspan="1">22/22</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">85.1-100%</td></tr><tr><td colspan="1" rowspan="3">Escherichia coli</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">159/159</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">97.6-100%</td></tr><tr><td colspan="1" rowspan="1">Seeded</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">51.0-100%</td><td colspan="1" rowspan="1">40/40</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">91.2-100%</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">51.0-100%</td><td colspan="1" rowspan="1">199/199</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">98.1-100%</td></tr><tr><td colspan="1" rowspan="3">Klebsiella aerogenes</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">2/2</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">34.2-100%</td></tr><tr><td colspan="1" rowspan="1">Seeded</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">42/42</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">91.6-100%</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">44/44</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">92.0-100%</td></tr><tr><td colspan="1" rowspan="4">Klebsiella oxytoca</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">8/8</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">67.6-100%</td></tr><tr><td colspan="1" rowspan="1">Archived</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Seeded</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">43.9-100%</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">43.9-100%</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">51.0-100%</td><td colspan="1" rowspan="1">11/11</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">74.1-100%</td></tr><tr><td colspan="1" rowspan="4">Klebsiella pneumoniae group</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">51.0-100%</td><td colspan="1" rowspan="1">51/51</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">93.0-100%</td></tr><tr><td colspan="1" rowspan="1">Archived</td><td colspan="1" rowspan="1">11/11</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">74.1-100%</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Seeded</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">88.6-100%</td><td colspan="1" rowspan="1">62/62</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">94.2-100%</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">45/45</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">92.1-100%</td><td colspan="1" rowspan="1">113/113</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">96.7-100%</td></tr><tr><td colspan="1" rowspan="3">Proteus spp.</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">15/15</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">79.6-100%</td></tr><tr><td colspan="1" rowspan="1">Seeded</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">43.9-100%</td><td colspan="1" rowspan="1">6/6</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">61.0-100%</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">43.9-100%</td><td colspan="1" rowspan="1">21/21</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">84.5-100%</td></tr><tr><td colspan="1" rowspan="3">Salmonella spp.</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">5/5</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">56.6-100%</td></tr><tr><td colspan="1" rowspan="1">Seeded</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">37/37</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">90.6-100%</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">•</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">42/42</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">91.6-100%</td></tr><tr><td colspan="1" rowspan="2">Serratia marcescens</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">11/11</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">74.1-100%</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">•</td><td colspan="1" rowspan="1">•</td><td colspan="1" rowspan="1">11/11</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">74.1-100%</td></tr><tr><td colspan="1" rowspan="3">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">31/31</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">89.0-100%</td></tr><tr><td colspan="1" rowspan="1">Seeded</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">43.9-100%</td><td colspan="1" rowspan="1">21/21</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">84.5-100%</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">43.9-100%</td><td colspan="1" rowspan="1">52/52</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">93.1-100%</td></tr></table>

a Archived testing not performed for KPC with E. coli, K. aerogenes, Proteus spp., Salmonella spp., S. marcescens, or P. aeruginosa; seeded testing not performed for KPC with S. marcescens

Table 38. BioFire BCID2 Panel Clinical Performance Summary, NDM   

<table><tr><td rowspan=2 colspan=1>Analyte</td><td rowspan=2 colspan=1>Study</td><td rowspan=1 colspan=3>Positive Percent Agreement</td><td rowspan=1 colspan=3>Negative Percent Agreement</td></tr><tr><td rowspan=1 colspan=1>TP/(TP + FN)</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95%CI</td><td rowspan=1 colspan=1>TN/(TN + FP)</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95%CI</td></tr><tr><td rowspan=4 colspan=1>NDM</td><td rowspan=1 colspan=1>Prospective</td><td rowspan=1 colspan=1>1/1</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>358/358</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>98.9-100%</td></tr><tr><td rowspan=1 colspan=1>Archived</td><td rowspan=1 colspan=1>5/5</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>56.6-100%</td><td rowspan=1 colspan=1>5/5</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>56.6-100%</td></tr><tr><td rowspan=1 colspan=1>Seeded</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>88.6-100%</td><td rowspan=1 colspan=1>231/231</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>98.4-100%</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>36/36</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>90.4-100%</td><td rowspan=1 colspan=1>594/594</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>99.4-100%</td></tr></table>

Table 39. Distribution of NDM in Prospective Clinical Specimens   

<table><tr><td rowspan=2 colspan=2>NDM</td><td rowspan=1 colspan=4>SOC: any associated organismPCR/seq: NDM</td></tr><tr><td rowspan=1 colspan=1>Org+ / Res+</td><td rowspan=1 colspan=1>Org+ / Res-</td><td rowspan=1 colspan=1>Org-</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=4 colspan=1>BCID2PanelResult</td><td rowspan=1 colspan=1>Org+ / Res+</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>Org+ / Res-</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>304</td><td rowspan=1 colspan=1>54a</td><td rowspan=1 colspan=1>358</td></tr><tr><td rowspan=1 colspan=1>Org -</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>713</td><td rowspan=1 colspan=1>715</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>306</td><td rowspan=1 colspan=1>767</td><td rowspan=1 colspan=1>1074</td></tr><tr><td rowspan=1 colspan=3>Performance</td><td rowspan=1 colspan=1>Agreement</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95%CI</td></tr><tr><td rowspan=1 colspan=3>Org+ / Res+</td><td rowspan=1 colspan=1>1/1</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=3>Org+ / Res-</td><td rowspan=1 colspan=1>304/306</td><td rowspan=1 colspan=1>99.3</td><td rowspan=1 colspan=1>97.6-99.8%</td></tr><tr><td rowspan=1 colspan=3>Org -</td><td rowspan=1 colspan=1>713/767a</td><td rowspan=1 colspan=1>93.0</td><td rowspan=1 colspan=1>90.9-94.6%</td></tr></table>

Fifty-three (53) FP results due to the presence of nucleic acid from non-viable E. coli in the blood culture bottles

Table 40. Stratification of NDM Clinical Performance by Associated Host Organism   

<table><tr><td colspan="1" rowspan="2">Analyte</td><td colspan="1" rowspan="2">Studya</td><td colspan="3" rowspan="1">Positive Percent Agreement</td><td colspan="3" rowspan="1">Negative Percent Agreement</td></tr><tr><td colspan="1" rowspan="1">TP/(TP + FN)</td><td colspan="1" rowspan="1">%</td><td colspan="1" rowspan="1">95%CI</td><td colspan="1" rowspan="1">TN/(TN + FP)</td><td colspan="1" rowspan="1">%</td><td colspan="1" rowspan="1">95%CI</td></tr><tr><td colspan="1" rowspan="4">Overall(any associated organism identified)</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">358/358</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">98.9-100%</td></tr><tr><td colspan="1" rowspan="1">Archived</td><td colspan="1" rowspan="1">5/5</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">56.6-100%</td><td colspan="1" rowspan="1">5/5</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">56.6-100%</td></tr><tr><td colspan="1" rowspan="1">Seeded</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">88.6-100%</td><td colspan="1" rowspan="1">231/231</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">98.4-100%</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">90.4-100%</td><td colspan="1" rowspan="1">594/594</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">99.4-100%</td></tr><tr><td colspan="1" rowspan="4">Acinetobacter calcoaceticus-baumanniicomplex</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">:</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">13/13</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">77.2-100%</td></tr><tr><td colspan="1" rowspan="1">Archived</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">5/5</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">56.6-100%</td></tr><tr><td colspan="1" rowspan="1">Seeded</td><td colspan="1" rowspan="1">6/6</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">61.0-100%</td><td colspan="1" rowspan="1">13/13</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">77.2-100%</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">6/6</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">61.0-100%</td><td colspan="1" rowspan="1">31/31</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">89.0-100%</td></tr><tr><td colspan="1" rowspan="2">Enterobacterales</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">322/322</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">98.8-100%</td></tr><tr><td colspan="1" rowspan="1">Archived</td><td colspan="1" rowspan="1">5/5</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">56.6-100%</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="1">Seeded</td><td colspan="1" rowspan="1">24/24</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">86.2-100%</td><td colspan="1" rowspan="1">204/204</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">98.2-100%</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">88.6-100%</td><td colspan="1" rowspan="1">526/526</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">99.3-100%</td></tr><tr><td colspan="1" rowspan="3">Enterobacter cloacae complex</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">16/16</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">80.6-100%</td></tr><tr><td colspan="1" rowspan="1">Seeded</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">43.9-100%</td><td colspan="1" rowspan="1">5/5</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">56.6-100%</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">43.9-100%</td><td colspan="1" rowspan="1">21/21</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">84.5-100%</td></tr><tr><td colspan="1" rowspan="4">Escherichia coli</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">160/160</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">97.7-100%</td></tr><tr><td colspan="1" rowspan="1">Archived</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Seeded</td><td colspan="1" rowspan="1">6/6</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">61.0-100%</td><td colspan="1" rowspan="1">38/38</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">90.8-100%</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">7/7</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">64.6-100%</td><td colspan="1" rowspan="1">198/198</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">98.1-100%</td></tr><tr><td colspan="1" rowspan="3">Klebsiella aerogenes</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2/2</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">34.2-100%</td></tr><tr><td colspan="1" rowspan="1">Seeded</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">42/42</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">91.6-100%</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">•</td><td colspan="1" rowspan="1">•</td><td colspan="1" rowspan="1">44/44</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">92.0-100%</td></tr><tr><td colspan="1" rowspan="3">Klebsiella oxytoca</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">8/8</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">67.6-100%</td></tr><tr><td colspan="1" rowspan="1">Seeded</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">6/6</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">61.0-100%</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">:</td><td colspan="1" rowspan="1">•</td><td colspan="1" rowspan="1">14/14</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">78.5-100%</td></tr><tr><td colspan="1" rowspan="4">Klebsiella pneumoniae group</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">54/54</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">93.4-100%</td></tr><tr><td colspan="1" rowspan="1">Archived</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">51.0-100%</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Seeded</td><td colspan="1" rowspan="1">9/9</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">70.1-100%</td><td colspan="1" rowspan="1">83/83</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">95.6-100%</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">14/14</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">75.8-100%</td><td colspan="1" rowspan="1">137/137</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">97.3-100%</td></tr><tr><td colspan="1" rowspan="3">Proteus spp.</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">15/15</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">79.6-100%</td></tr><tr><td colspan="1" rowspan="1">Seeded</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">43.9-100%</td><td colspan="1" rowspan="1">6/6</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">61.0-100%</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">43.9-100%</td><td colspan="1" rowspan="1">21/21</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">84.5-100%</td></tr><tr><td colspan="1" rowspan="3">Salmonella spp.</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">5/5</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">56.6-100%</td></tr><tr><td colspan="1" rowspan="1">Seeded</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">43.9-100%</td><td colspan="1" rowspan="1">34/34</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">89.8-100%</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">43.9-100%</td><td colspan="1" rowspan="1">39/39</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">91.0-100%</td></tr><tr><td colspan="1" rowspan="2">Serratia marcescens</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">11/11</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">74.1-100%</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">:</td><td colspan="1" rowspan="1">•</td><td colspan="1" rowspan="1">11/11</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">74.1-100%</td></tr><tr><td colspan="1" rowspan="3">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">88.6-100%</td></tr><tr><td colspan="1" rowspan="1">Seeded</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">24/24</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">86.2-100%</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">•</td><td colspan="1" rowspan="1">54/54</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">93.4-100%</td></tr></table>

a Archived testing not performed for NDM with E. cloacae complex, K. aerogenes, K. oxytoca, Proteus spp., Salmonella spp., S. marcescens, or P. aeruginosa; seeded testing not performed for NDM with S. marcescens

able 41. BioFire BCID2 Panel Clinical Performance Summary, OXA-48-like   

<table><tr><td rowspan=2 colspan=1>Analyte</td><td rowspan=2 colspan=1>Studya</td><td rowspan=1 colspan=3>Positive Percent Agreement</td><td rowspan=1 colspan=3>Negative Percent Agreement</td></tr><tr><td rowspan=1 colspan=1>TP/(TP + FN)</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95%CI</td><td rowspan=1 colspan=1>TN/(TN + FP)</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95%CI</td></tr><tr><td rowspan=3 colspan=1>OXA-48-like</td><td rowspan=1 colspan=1>Prospective</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>323/323</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>98.8-100%</td></tr><tr><td rowspan=1 colspan=1>Seeded</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>88.6-100%</td><td rowspan=1 colspan=1>198/198</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>98.1-100%</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>88.6-100%</td><td rowspan=1 colspan=1>521/521</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>99.3-100%</td></tr></table>

Archived testing not performed for OXA-48-like

Table 42. Distribution of OXA-48-like in Prospective Clinical Specimens   

<table><tr><td rowspan=2 colspan=2>OXA-48-like</td><td rowspan=1 colspan=4>SOC: any associated organismPCR/seq: OXA-48-like</td></tr><tr><td rowspan=1 colspan=1>Org+ / Res+</td><td rowspan=1 colspan=1>Org+ / Res-</td><td rowspan=1 colspan=1>Org-</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=4 colspan=1>BCID2PanelResult</td><td rowspan=1 colspan=1>Org+ / Res+</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Org+ / Res-</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>269</td><td rowspan=1 colspan=1>54a</td><td rowspan=1 colspan=1>323</td></tr><tr><td rowspan=1 colspan=1>Org -</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>750</td><td rowspan=1 colspan=1>751</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>270</td><td rowspan=1 colspan=1>804</td><td rowspan=1 colspan=1>1074</td></tr><tr><td rowspan=1 colspan=3>Performance</td><td rowspan=1 colspan=1>Agreement</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95%CI</td></tr><tr><td rowspan=1 colspan=3>Org+ / Res+</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=3>Org+ / Res-</td><td rowspan=1 colspan=1>269/270</td><td rowspan=1 colspan=1>99.6</td><td rowspan=1 colspan=1>97.9-99.9%</td></tr><tr><td rowspan=1 colspan=3>Org -</td><td rowspan=1 colspan=1>750/804a</td><td rowspan=1 colspan=1>93.3</td><td rowspan=1 colspan=1>91.3-94.8%</td></tr></table>

a Fifty-three (53) FP results due to the presence of nucleic acid from non-viable E. coli in the blood culture bottles

Table 43. Stratification of OXA-48-like Clinical Performance by Associated Host Organism, Seeded Studya   

<table><tr><td rowspan=2 colspan=1>Analyte</td><td rowspan=1 colspan=3>Positive Percent Agreement</td><td rowspan=1 colspan=3>Negative Percent Agreement</td></tr><tr><td rowspan=1 colspan=1>TP/(TP + FN)</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95%CI</td><td rowspan=1 colspan=1>TN(TN + FP)</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95%CI</td></tr><tr><td rowspan=1 colspan=1>Overall(any associated organism identified)</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>88.6-100%</td><td rowspan=1 colspan=1>198/198</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>98.1-100%</td></tr><tr><td rowspan=1 colspan=1>Enterobacterales</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>88.6-100%</td><td rowspan=1 colspan=1>198/198</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>98.1-100%</td></tr><tr><td rowspan=1 colspan=1>Enterobacter cloacae complex</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>8/8</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>67.6-100%</td></tr><tr><td rowspan=1 colspan=1>Escherichia coli</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>44/44</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>92.0-100%</td></tr><tr><td rowspan=1 colspan=1>Klebsiella aerogenes</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>43.9-100%</td><td rowspan=1 colspan=1>39/39</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>91.0-100%</td></tr><tr><td rowspan=1 colspan=1>Klebsiella oxytoca</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>61.0-100%</td></tr><tr><td rowspan=1 colspan=1>Klebsiella pneumoniae group</td><td rowspan=1 colspan=1>27/27</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>87.5-100%</td><td rowspan=1 colspan=1>65/65</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>94.4-100%</td></tr><tr><td rowspan=1 colspan=1>Proteus spp.</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>9/9</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>70.1-100%</td></tr><tr><td rowspan=1 colspan=1>Salmonella spp.</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>37/37</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>90.6-100%</td></tr></table>

a No observations for OXA-48-like in prospective evaluation; archived testing not performed; seeded testing not performed for OXA-48-like with S. marcescens

Table 44. BioFire BCID2 Panel Clinical Performance Summary, VIM   

<table><tr><td rowspan=2 colspan=1>Analyte</td><td rowspan=2 colspan=1>Study</td><td rowspan=1 colspan=3>Positive Percent Agreement</td><td rowspan=1 colspan=3>Negative Percent Agreement</td></tr><tr><td rowspan=1 colspan=1>TP/(TP + FN)</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95%CI</td><td rowspan=1 colspan=1>TN/(TN + FP)</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95%CI</td></tr><tr><td rowspan=4 colspan=1>VIM</td><td rowspan=1 colspan=1>Prospective</td><td rowspan=1 colspan=1>4/4</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>51.0-100%</td><td rowspan=1 colspan=1>355/355</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>98.9-100%</td></tr><tr><td rowspan=1 colspan=1>Archived</td><td rowspan=1 colspan=1>1/1</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>5/5</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>56.6-100%</td></tr><tr><td rowspan=1 colspan=1>Seeded</td><td rowspan=1 colspan=1>29/29</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>88.3-100%</td><td rowspan=1 colspan=1>232/232</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>98.4-100%</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>34/34</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>89.8-100%</td><td rowspan=1 colspan=1>592/592</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>99.4-100%</td></tr></table>

Table 45. Distribution of VIM in Prospective Clinical Specimens   

<table><tr><td colspan="2" rowspan="2">VIM</td><td colspan="4" rowspan="1">SOC: any associated organismPCR/seq: VIM</td></tr><tr><td colspan="1" rowspan="1">Org+ / Res+</td><td colspan="1" rowspan="1">Org+/ Res-</td><td colspan="1" rowspan="1">Org -</td><td colspan="1" rowspan="1">Total</td></tr><tr><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="1">Org+ / Res+</td><td colspan="1" rowspan="1">4a</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">4</td></tr><tr><td colspan="1" rowspan="1">Org+ / Res-</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">301</td><td colspan="1" rowspan="1">54b</td><td colspan="1" rowspan="1">355</td></tr><tr><td colspan="1" rowspan="1">Org+/Res+</td><td colspan="1" rowspan="1">Org+ / Res-</td><td colspan="1" rowspan="1">Org -</td><td colspan="1" rowspan="1">Total</td></tr><tr><td colspan="1" rowspan="2">BCID2PanelResult</td><td colspan="1" rowspan="1">Org -</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">713</td><td colspan="1" rowspan="1">715</td></tr><tr><td colspan="1" rowspan="1">Total</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">303</td><td colspan="1" rowspan="1">767</td><td colspan="1" rowspan="1">1074</td></tr><tr><td colspan="3" rowspan="1">Performance</td><td colspan="1" rowspan="1">Agreement</td><td colspan="1" rowspan="1">%</td><td colspan="1" rowspan="1">95%CI</td></tr><tr><td colspan="3" rowspan="1">Org+ / Res+</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">51.0-100%</td></tr><tr><td colspan="3" rowspan="1">Org+ / Res-</td><td colspan="1" rowspan="1">301/303</td><td colspan="1" rowspan="1">99.3</td><td colspan="1" rowspan="1">97.6-99.8%</td></tr><tr><td colspan="3" rowspan="1">Org -</td><td colspan="1" rowspan="1">713/767b</td><td colspan="1" rowspan="1">93.0</td><td colspan="1" rowspan="1">90.9-94.6%</td></tr></table>

a One specimen had co-detection of Klebsiella pneumoniae group with Pseudomonas aeruginosa b Fifty-three (53) FP results due to the presence of nucleic acid from non-viable E. coli in the blood culture bottles

Table 46. Stratification of VIM Clinical Performance by Associated Host Organism   

<table><tr><td colspan="1" rowspan="2">Analyte</td><td colspan="1" rowspan="2">Studya</td><td colspan="3" rowspan="1">Positive Percent Agreement</td><td colspan="3" rowspan="1">Negative Percent Agreement</td></tr><tr><td colspan="1" rowspan="1">TP/(TP + FN)</td><td colspan="1" rowspan="1">%</td><td colspan="1" rowspan="1">95%CI</td><td colspan="1" rowspan="1">TN/(TN + FP)</td><td colspan="1" rowspan="1">%</td><td colspan="1" rowspan="1">95%CI</td></tr><tr><td colspan="1" rowspan="4">Overall(any associated organism identified)</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">4/4b</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">51.0-100%</td><td colspan="1" rowspan="1">355/355</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">98.9-100%</td></tr><tr><td colspan="1" rowspan="1">Archived</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">5/5</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">56.6-100%</td></tr><tr><td colspan="1" rowspan="1">Seeded</td><td colspan="1" rowspan="1">29/29</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">88.3-100%</td><td colspan="1" rowspan="1">232/232</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">98.4-100%</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">34/34</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">89.8-100%</td><td colspan="1" rowspan="1">592/592</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">99.4-100%</td></tr><tr><td colspan="1" rowspan="4">Acinetobacter calcoaceticus-baumanniicomplex</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">13/13</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">77.2-100%</td></tr><tr><td colspan="1" rowspan="1">Archived</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">5/5</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">56.6-100%</td></tr><tr><td colspan="1" rowspan="1">Seeded</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">19/19</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">83.2-100%</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">•</td><td colspan="1" rowspan="1">•</td><td colspan="1" rowspan="1">37/37</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">90.6-100%</td></tr><tr><td colspan="1" rowspan="4">Enterobacterales</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">2/2</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">34.2-100%</td><td colspan="1" rowspan="1">321/321</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">98.8-100%</td></tr><tr><td colspan="1" rowspan="1">Archived</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Seeded</td><td colspan="1" rowspan="1">15/15</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">79.6-100%</td><td colspan="1" rowspan="1">213/213</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">98.2-100%</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">18/18</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">82.4-100%</td><td colspan="1" rowspan="1">534/534</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">99.3-100%</td></tr><tr><td colspan="1" rowspan="3">Enterobacter cloacae complex</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">16/16</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">80.6-100%</td></tr><tr><td colspan="1" rowspan="1">Seeded</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">43.9-100%</td><td colspan="1" rowspan="1">5/5</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">56.6-100%</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">43.9-100%</td><td colspan="1" rowspan="1">21/21</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">84.5-100%</td></tr><tr><td colspan="1" rowspan="3">Escherichia coli</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">160/160</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">97.7-100%</td></tr><tr><td colspan="1" rowspan="1">Seeded</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">43.9-100%</td><td colspan="1" rowspan="1">41/41</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">91.4-100%</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">43.9-100%</td><td colspan="1" rowspan="1">201/201</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">98.1-100%</td></tr><tr><td colspan="1" rowspan="3">Klebsiella aerogenes</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">2/2</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">34.2-100%</td></tr><tr><td colspan="1" rowspan="1">Seeded</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">42/42</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">91.6-100%</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">•</td><td colspan="1" rowspan="1">•</td><td colspan="1" rowspan="1">44/44</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">92.0-100%</td></tr><tr><td colspan="1" rowspan="4">Klebsiella oxytoca</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">8/8</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">67.6-100%</td></tr><tr><td colspan="1" rowspan="1">Archived</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Seeded</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">6/6</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">61.0-100%</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">•</td><td colspan="1" rowspan="1">14/14</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">78.5-100%</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae group</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">2/2</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">34.2-100%</td><td colspan="1" rowspan="1">53/53</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">93.2-100%</td></tr><tr><td colspan="1" rowspan="1">TP/(TP + FN)</td><td colspan="1" rowspan="1">%</td><td colspan="1" rowspan="1">95%CI</td><td colspan="1" rowspan="1">TN(TN + FP)</td><td colspan="1" rowspan="1">%</td><td colspan="1" rowspan="1">95%CI</td></tr><tr><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="1">Seeded</td><td colspan="1" rowspan="1">9/9</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">70.1-100%</td><td colspan="1" rowspan="1">83/83</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">95.6-100%</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">11/11</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">74.1-100%</td><td colspan="1" rowspan="1">136/136</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">97.3-100%</td></tr><tr><td colspan="1" rowspan="3">Proteus spp.</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">15/15</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">79.6-100%</td></tr><tr><td colspan="1" rowspan="1">Seeded</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">9/9</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">70.1-100%</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">•</td><td colspan="1" rowspan="1">•</td><td colspan="1" rowspan="1">24/24</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">86.2-100%</td></tr><tr><td colspan="1" rowspan="3">Salmonella spp.</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">5/5</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">56.6-100%</td></tr><tr><td colspan="1" rowspan="1">Seeded</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">37/37</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">90.6-100%</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">•</td><td colspan="1" rowspan="1">•</td><td colspan="1" rowspan="1">42/42</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">91.6-100%</td></tr><tr><td colspan="1" rowspan="2">Serratia marcescens</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">11/11</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">74.1-100%</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">•</td><td colspan="1" rowspan="1">•</td><td colspan="1" rowspan="1">11/11</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">74.1-100%</td></tr><tr><td colspan="1" rowspan="3">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">43.9-100%</td><td colspan="1" rowspan="1">28/28</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">87.9-100%</td></tr><tr><td colspan="1" rowspan="1">Seeded</td><td colspan="1" rowspan="1">14/14</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">78.5-100%</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">72.2-100%</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">17/17</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">81.6-100%</td><td colspan="1" rowspan="1">38/38</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">90.8-100%</td></tr></table>

Archived testing not performed for VIM with E. cloacae complex, E. coli, K. aerogenes, K. pneumoniae group, Proteus spp., Salmonella spp., S. marcescens, o P. aeruginosa; seeded testing not performed for VIM with S. marcescens One specimen had co-detection of Klebsiella pneumoniae group with Pseudomonas aeruginosa

Table 47. BioFire BCID2 Panel Clinical Performance Summary, mecA/C, Prospective Studya   

<table><tr><td rowspan=2 colspan=1>Analyte</td><td rowspan=1 colspan=3>Positive Percent Agreement</td><td rowspan=1 colspan=3>Negative Percent Agreement</td></tr><tr><td rowspan=1 colspan=1>TP/(TP + FN)</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95%CI</td><td rowspan=1 colspan=1>TN/(TN + FP)</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95%CI</td></tr><tr><td rowspan=1 colspan=1>mecA/C</td><td rowspan=1 colspan=1>195/195</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>98.1-100%</td><td rowspan=1 colspan=1>60/60</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>94.0-100%</td></tr></table>

a Archived and seeded testing not performed for mecA/C

Table 48. Distribution of mecA/C in Prospective Clinical Specimens   

<table><tr><td rowspan=2 colspan=2>mecA/C</td><td rowspan=1 colspan=4>SOC: any associated organismFDA-cleared test: mecA</td></tr><tr><td rowspan=1 colspan=1>Org+ / Res+</td><td rowspan=1 colspan=1>Org+ / Res-</td><td rowspan=1 colspan=1>Org -</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=4 colspan=1>BCID2PanelResult</td><td rowspan=1 colspan=1>Org+ /Res+</td><td rowspan=1 colspan=1>171a</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>195</td></tr><tr><td rowspan=1 colspan=1>Org+ / Res-</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>54</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>60</td></tr><tr><td rowspan=1 colspan=1>Org -</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>811</td><td rowspan=1 colspan=1>818</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>175</td><td rowspan=1 colspan=1>57</td><td rowspan=1 colspan=1>841</td><td rowspan=1 colspan=1>1073b</td></tr><tr><td rowspan=1 colspan=3>Performance</td><td rowspan=1 colspan=1>Agreement</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95%CI</td></tr><tr><td rowspan=1 colspan=3>Org+ / Res+</td><td rowspan=1 colspan=1>171/175</td><td rowspan=1 colspan=1>97.7</td><td rowspan=1 colspan=1>94.3-99.1%</td></tr><tr><td rowspan=1 colspan=3>Org+ / Res-</td><td rowspan=1 colspan=1>54/57</td><td rowspan=1 colspan=1>94.7</td><td rowspan=1 colspan=1>85.6-98.2%</td></tr><tr><td rowspan=1 colspan=3>Org -</td><td rowspan=1 colspan=1>811/841</td><td rowspan=1 colspan=1>96.4</td><td rowspan=1 colspan=1>95.0-97.5%</td></tr></table>

a Two specimens had co-detections of Staphylococcus epidermidis with Staphylococcus lugdunensis b One specimen was FN for the associated host organism by the molecular comparator method, thus providing an “N/A” result for mecA/C

Table 49. Stratification of mecA/C Clinical Performance by Associated Host Organism, Prospective Studya   

<table><tr><td rowspan=2 colspan=1>Analyte</td><td rowspan=1 colspan=3>Positive Percent Agreement</td><td rowspan=1 colspan=3>Negative Percent Agreement</td></tr><tr><td rowspan=1 colspan=1>TP/(TP + FN)</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95%CI</td><td rowspan=1 colspan=1>TN/(TN + FP)</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95%CI</td></tr><tr><td rowspan=1 colspan=1>Overall(any associated organism identified)</td><td rowspan=1 colspan=1>195/195b</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>98.1-100%</td><td rowspan=1 colspan=1>60/60</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>94.0-100%</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus epidermidis</td><td rowspan=1 colspan=1>194/194</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>98.1-100%</td><td rowspan=1 colspan=1>56/56</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>93.6-100%</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus lugdunensis</td><td rowspan=1 colspan=1>3/3c</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>43.9-100%</td><td rowspan=1 colspan=1>4/4</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>51.0-100%</td></tr></table>

a Archived and seeded testing not performed for mecA/C b Two specimens had co-detections of Staphylococcus epidermidis with Staphylococcus lugdunensis c All three specimens were identified as mixed cultures with mecA present in a different Staphylococcus species

Table 50. BioFire BCID2 Panel Clinical Performance Summary, mecA/C and MREJ (MRSA)   

<table><tr><td rowspan=2 colspan=1>Analyte</td><td rowspan=2 colspan=1>Studya</td><td rowspan=1 colspan=3>Positive Percent Agreement</td><td rowspan=1 colspan=3>Negative Percent Agreement</td></tr><tr><td rowspan=1 colspan=1>TP/(TP + FN)</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95%CI</td><td rowspan=1 colspan=1>TN/(TN + FP)</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95%CI</td></tr><tr><td rowspan=3 colspan=1>mecA/C and MREJ (MRSA)</td><td rowspan=1 colspan=1>Prospective</td><td rowspan=1 colspan=1>52/57</td><td rowspan=1 colspan=1>91.2</td><td rowspan=1 colspan=1>81.1-96.2%</td><td rowspan=1 colspan=1>92/94</td><td rowspan=1 colspan=1>97.9</td><td rowspan=1 colspan=1>92.6-99.4%</td></tr><tr><td rowspan=1 colspan=1>Seeded</td><td rowspan=1 colspan=1>5/5</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>56.6-100%</td><td rowspan=1 colspan=1>5/5</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>56.6-100%</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>57/62b</td><td rowspan=1 colspan=1>91.9</td><td rowspan=1 colspan=1>82.5-96.5%</td><td rowspan=1 colspan=1>97/99c</td><td rowspan=1 colspan=1>98.0</td><td rowspan=1 colspan=1>92.9-99.4%</td></tr></table>

a Archived testing not performed for mecA/C and MREJ (MRSA) b Isolates recovered from the five FN specimens were identified as MSSA by SOC phenotypic AST methods c Isolates recovered from the two FP specimens were identified as MRSA by SOC phenotypic AST methods

Table 51. Distribution of mecA/C and MREJ (MRSA) in Prospective Clinical Specimens   

<table><tr><td rowspan=2 colspan=2>mecA/C and MREJ (MRSA)</td><td rowspan=1 colspan=4>SOC: any associated organismFDA-cleared test: MRSA</td></tr><tr><td rowspan=1 colspan=1>Org+ / Res+</td><td rowspan=1 colspan=1>Org+ / Res-</td><td rowspan=1 colspan=1>Org -</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=4 colspan=1>BCID2PanelResult</td><td rowspan=1 colspan=1>Org+ / Res+</td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>54</td></tr><tr><td rowspan=1 colspan=1>Org+ / Res-</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>92</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>97</td></tr><tr><td rowspan=1 colspan=1>Org -</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>923</td><td rowspan=1 colspan=1>923</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>55</td><td rowspan=1 colspan=1>94</td><td rowspan=1 colspan=1>925</td><td rowspan=1 colspan=1>1074</td></tr><tr><td rowspan=1 colspan=3>Performance</td><td rowspan=1 colspan=1>Agreement</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95%CI</td></tr><tr><td rowspan=1 colspan=3>Org+ / Res+</td><td rowspan=1 colspan=1>50/55</td><td rowspan=1 colspan=1>90.9</td><td rowspan=1 colspan=1>80.4-96.1%</td></tr><tr><td rowspan=1 colspan=3>Org+ / Res-</td><td rowspan=1 colspan=1>92/94</td><td rowspan=1 colspan=1>97.9</td><td rowspan=1 colspan=1>92.6-99.4%</td></tr><tr><td rowspan=1 colspan=3>Org -</td><td rowspan=1 colspan=1>923/925</td><td rowspan=1 colspan=1>99.8</td><td rowspan=1 colspan=1>99.2-99.9%</td></tr></table>

The nature of the seven discrepant results for mecA/C and MREJ (MRSA) between the BioFire BCID2 Panel and the reference method (Cepheid Xpert® MRSA/SA BC) was investigated using various methods. As shown in Table 52, all S. aureus isolates recovered from the five false negative (FN) specimens were negative for the mecA/C genes by PCR/sequencing and had a methicillin sensitive antimicrobial susceptibility testing (AST) phenotype, indicating MSSA rather than MRSA. From three of these specimens, the laboratory also isolated a coagulase negative Staphylococcus (CoNS) that was methicillin resistant (i.e. carrying the mecA or mecC gene). Three of the five FN specimens were reported as SA (negative for MRSA) by the Cepheid Xpert® MRSA/SA BC test when residual specimen was retested.

Similarly, S. aureus isolates from both false positive (FP) specimens were positive for the mecA/C genes by PCR/sequencing and had a methicillin resistant AST phenotype. Additionally, one of the specimens had a result of MRSA when retested by the Cepheid Xpert® MRSA/SA BC test (Table 52).

In all cases, the BioFire BCID2 Panel mecA/C and MREJ (MRSA) results (Detected or Not Detected) were concordant with the AST phenotype of the S. aureus isolated from the blood culture, including instances where methicillin resistant CoNS were also present in the specimen.

Table 52. Investigation of Specimens with Discrepant mecA/C and MREJ (MRSA) Results   

<table><tr><td rowspan=2 colspan=1>Cisdes</td><td rowspan=1 colspan=1>LaboratoryInformation</td><td rowspan=1 colspan=1>BioFire BCID2Panel</td><td rowspan=1 colspan=1>Cepheid Xpert®MMRMRSABC</td><td rowspan=1 colspan=3>Investigation Summary</td></tr><tr><td rowspan=1 colspan=1>AdditionalStaphylococciIsolated</td><td rowspan=1 colspan=1>mecA/C and MREJ(MRSA) Resultsassociated withTP S. aureus)</td><td rowspan=1 colspan=1>MRSA/SA Results</td><td rowspan=1 colspan=1>Cepheid Xpert®MRSA/SA BCRetestc</td><td rowspan=1 colspan=1>mecA/C IsolatePCR/sequencingResut</td><td rowspan=1 colspan=1>SOC Isolate ASTResult (methicillinresistancephenotype)d</td></tr><tr><td rowspan=1 colspan=1>FN</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Not Detected</td><td rowspan=1 colspan=1>MRSA</td><td rowspan=1 colspan=1>SA</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>MSSA</td></tr><tr><td rowspan=1 colspan=1>FN</td><td rowspan=1 colspan=1>S. haemolyticus(methicillin resistant)</td><td rowspan=1 colspan=1>Not Detected</td><td rowspan=1 colspan=1>MRSA</td><td rowspan=1 colspan=1>SA</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>MSSA</td></tr><tr><td rowspan=1 colspan=1>FN</td><td rowspan=1 colspan=1>S. epidermidis(methicillin resistant)</td><td rowspan=1 colspan=1>Not Detected</td><td rowspan=1 colspan=1>MRSA</td><td rowspan=1 colspan=1>MRSA</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>MSSA</td></tr><tr><td rowspan=1 colspan=1>FN</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>Not Detected</td><td rowspan=1 colspan=1>MRSA</td><td rowspan=1 colspan=1>SA</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>MSSA</td></tr><tr><td rowspan=1 colspan=1>FN</td><td rowspan=1 colspan=1>S. epidermidis(methicillin resistant)</td><td rowspan=1 colspan=1>Not Detected</td><td rowspan=1 colspan=1>MRSA</td><td rowspan=1 colspan=1>MRSA</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>MSSA</td></tr><tr><td rowspan=1 colspan=1>FP</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>SA</td><td rowspan=1 colspan=1>MRSA</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>MRSA</td></tr><tr><td rowspan=1 colspan=1>FP</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>SA</td><td rowspan=1 colspan=1>SA</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>MRSA</td></tr></table>

a FN $=$ false negative; $\mathsf { F P } =$ false positive b TP $=$ true positive c MRSA $=$ methicillin resistant Staphylococcus aureus; $\mathtt { S A } =$ Staphylococcus aureus d SOC $=$ standard of care; MSSA $=$ methicillin sensitive Staphylococcus aureus; MRSA $=$ methicillin resistant Staphylococcus aureus

Table 53. BioFire BCID2 Panel Clinical Performance Summary, mcr-1   

<table><tr><td rowspan=2 colspan=1>Analyte</td><td rowspan=2 colspan=1>Studya</td><td rowspan=1 colspan=3>Positive Percent Agreement</td><td rowspan=1 colspan=3>Negative Percent Agreement</td></tr><tr><td rowspan=1 colspan=1>TP/(TP + FN)</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95%CI</td><td rowspan=1 colspan=1>TN/(TN + FP)</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95%CI</td></tr><tr><td rowspan=3 colspan=1>mcr-1</td><td rowspan=1 colspan=1>Prospective</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>240/240</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>98.4-100%</td></tr><tr><td rowspan=1 colspan=1>Seeded</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>88.6-100%</td><td rowspan=1 colspan=1>189/189</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>98.0-100%</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>88.6-100%</td><td rowspan=1 colspan=1>429/429</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>99.1-100%</td></tr></table>

Archived testing not performed for mcr-1

Table 54. Distribution of mcr-1 in Prospective Clinical Specimens   

<table><tr><td rowspan=2 colspan=2>Mcr-1</td><td rowspan=1 colspan=4>SOC: any associated organismPCR/seq: mcr-1</td></tr><tr><td rowspan=1 colspan=1>Org+ / Res+</td><td rowspan=1 colspan=1>Org+ / Res-</td><td rowspan=1 colspan=1>Org -</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=4 colspan=1>BCID2PanelResult</td><td rowspan=1 colspan=1>Org+ / Res+</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Org+ / Res-</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>238</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>240</td></tr><tr><td rowspan=1 colspan=1>Org -</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>834</td><td rowspan=1 colspan=1>834</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>238</td><td rowspan=1 colspan=1>836</td><td rowspan=1 colspan=1>1074</td></tr><tr><td rowspan=1 colspan=3>Performance</td><td rowspan=1 colspan=1>Agreement</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95%CI</td></tr><tr><td rowspan=1 colspan=3>Org+ / Res+</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=3>Org+ / Res-</td><td rowspan=1 colspan=1>238/238</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>98.4-100%</td></tr><tr><td rowspan=1 colspan=3>Org -</td><td rowspan=1 colspan=1>834/836</td><td rowspan=1 colspan=1>99.8</td><td rowspan=1 colspan=1>99.1-99.9%</td></tr></table>

Table 55. Stratification of mcr-1 Clinical Performance by Associated Host Organism, Seeded Studya   

<table><tr><td rowspan=2 colspan=1>Analyte</td><td rowspan=1 colspan=3>Positive Percent Agreement</td><td rowspan=1 colspan=3>Negative Percent Agreement</td></tr><tr><td rowspan=1 colspan=1>TP(TP + FN)</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95%CI</td><td rowspan=1 colspan=1>TN/(TN + FP)</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95%CI</td></tr><tr><td rowspan=1 colspan=1>Overall(any associated organism identified)</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>88.6-100%</td><td rowspan=1 colspan=1>189/189</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>98.0-100%</td></tr><tr><td rowspan=1 colspan=1>Enterobacter cloacae complex</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>8/8</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>67.6-100%</td></tr><tr><td rowspan=1 colspan=1>Escherichia coli</td><td rowspan=1 colspan=1>22/22</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>85.1-100%</td><td rowspan=1 colspan=1>22/22</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>85.1-100%</td></tr><tr><td rowspan=1 colspan=1>Klebsiella aerogenes</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>42/42</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>91.6-100%</td></tr><tr><td rowspan=1 colspan=1>Klebsiella oxytoca</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>61.0-100%</td></tr><tr><td rowspan=1 colspan=1>Klebsiella pneumoniae group</td><td rowspan=1 colspan=1>4/4</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>51.0-100%</td><td rowspan=1 colspan=1>88/88</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>95.8-100%</td></tr><tr><td rowspan=1 colspan=1>Salmonella spp.</td><td rowspan=1 colspan=1>4/4</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>51.0-100%</td><td rowspan=1 colspan=1>33/33</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>89.6-100%</td></tr></table>

No observations for mcr-1 in prospective evaluation; no archived testing performed

Table 56. BioFire BCID2 Panel Clinical Performance Summary, vanA/B   

<table><tr><td rowspan=2 colspan=1>Analyte</td><td rowspan=2 colspan=1>Studya</td><td rowspan=1 colspan=3>Positive Percent Agreement</td><td rowspan=1 colspan=3>Negative Percent Agreement</td></tr><tr><td rowspan=1 colspan=1>TP/(TP + FN)</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95%CI</td><td rowspan=1 colspan=1>TN(TN + FP)</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95%CI</td></tr><tr><td rowspan=3 colspan=1>vanA/B</td><td rowspan=1 colspan=1>Prospective</td><td rowspan=1 colspan=1>23/24</td><td rowspan=1 colspan=1>95.8</td><td rowspan=1 colspan=1>79.8-99.3%</td><td rowspan=1 colspan=1>38/38</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>90.8-100%</td></tr><tr><td rowspan=1 colspan=1>Archived</td><td rowspan=1 colspan=1>16/16</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>80.6-100%</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>39/40b</td><td rowspan=1 colspan=1>97.5</td><td rowspan=1 colspan=1>87.1-99.6%</td><td rowspan=1 colspan=1>38/38</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>90.8-100%</td></tr></table>

a Seeded testing not performed for vanA/B b vanA/B was detected in the single FN specimen upon BioFire BCID2 Panel retest; the isolate recovered from this specimen was vancomycin resistant by SOC phenotypic AST methods

Table 57. Distribution of vanA/B in Prospective Clinical Specimens   

<table><tr><td rowspan=2 colspan=2>vanA/B</td><td rowspan=1 colspan=4>SOC: any associated organismFDA-cleared test: vanA/B</td></tr><tr><td rowspan=1 colspan=1>Org+ / Res+</td><td rowspan=1 colspan=1>Org+ / Res-</td><td rowspan=1 colspan=1>Org -</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=4 colspan=1>BCID2PanelResult</td><td rowspan=1 colspan=1>Org+ / Res+</td><td rowspan=1 colspan=1>22</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>23</td></tr><tr><td rowspan=1 colspan=1>Org+ / Res-</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>35</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>39</td></tr><tr><td rowspan=1 colspan=1>Org -</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1010</td><td rowspan=1 colspan=1>1010</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>23</td><td rowspan=1 colspan=1>35</td><td rowspan=1 colspan=1>1014</td><td rowspan=1 colspan=1>1072a</td></tr><tr><td rowspan=1 colspan=3>Performance</td><td rowspan=1 colspan=1>Agreement</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95%CI</td></tr><tr><td rowspan=1 colspan=3>Org+ / Res+</td><td rowspan=1 colspan=1>22/23</td><td rowspan=1 colspan=1>95.7</td><td rowspan=1 colspan=1>79.0-99.2%</td></tr><tr><td rowspan=1 colspan=3>Org+ / Res-</td><td rowspan=1 colspan=1>35/35</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>90.1-100%</td></tr><tr><td rowspan=1 colspan=3>Org -</td><td rowspan=1 colspan=1>1010/1014</td><td rowspan=1 colspan=1>99.6</td><td rowspan=1 colspan=1>99.0-99.8%</td></tr></table>

Two specimens were FN for the associated host organism by the molecular comparator method, thus providing an “N/A” result for vanA/B

Table 58. Stratification of vanA/B Clinical Performance by Associated Host Organism   

<table><tr><td colspan="1" rowspan="2">Analyte</td><td colspan="1" rowspan="2">Studya</td><td colspan="3" rowspan="1">Positive Percent Agreement</td><td colspan="3" rowspan="1">Negative Percent Agreement</td></tr><tr><td colspan="1" rowspan="1">TP/(TP + FN)</td><td colspan="1" rowspan="1">%</td><td colspan="1" rowspan="1">95%CI</td><td colspan="1" rowspan="1">TN/(TN + FP)</td><td colspan="1" rowspan="1">%</td><td colspan="1" rowspan="1">95%CI</td></tr><tr><td colspan="1" rowspan="3">Overall(any associated organism identified)</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">23/24</td><td colspan="1" rowspan="1">95.8</td><td colspan="1" rowspan="1">79.8-99.3%</td><td colspan="1" rowspan="1">38/38</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">90.8-100%</td></tr><tr><td colspan="1" rowspan="1">Archived</td><td colspan="1" rowspan="1">16/16</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">80.6-100%</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">39/40</td><td colspan="1" rowspan="1">97.5</td><td colspan="1" rowspan="1">87.1-99.6%</td><td colspan="1" rowspan="1">38/38</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">90.8-100%</td></tr><tr><td colspan="1" rowspan="2">Enterococcus faecalis</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">31/31</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">89.0-100%</td></tr><tr><td colspan="1" rowspan="1">Archived</td><td colspan="1" rowspan="1">2/2</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">34.2-100%</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Y</td><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">43.9-100%</td><td colspan="1" rowspan="1">31/31</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">89.0-100%</td></tr><tr><td colspan="1" rowspan="3">Enterococcus faecium</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">22/23</td><td colspan="1" rowspan="1">95.7</td><td colspan="1" rowspan="1">79.0-99.2%</td><td colspan="1" rowspan="1">717</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">64.6-100%</td></tr><tr><td colspan="1" rowspan="1">Archived</td><td colspan="1" rowspan="1">14/14</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">78.5-100%</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">36/37</td><td colspan="1" rowspan="1">97.3</td><td colspan="1" rowspan="1">86.2-99.5%</td><td colspan="1" rowspan="1">7/7</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">64.6-100%</td></tr></table>

Seeded testing not performed for vanA/B

For prospective specimens only, correlation of the AMR gene results reported in the specimen by the BCID2 Panel to identification of the gene in the cultured isolates from that particular specimen was assessed using one polymerase chain reaction (PCR) assay followed by bidirectional sequencing, performed directly on the isolate. The results are shown only for isolates recovered from specimens with true positive results (i.e. concordant results between BCID2 Panel and culture), and further stratified by each applicable host organism recovered from that specimen. There were no observations by either the BioFire BCID2 Panel or the reference/comparator methods for IMP, OXA-48-like, and mcr- $\boldsymbol { \mathscr { I } }$ ; therefore, performance tables are not shown for these analytes. Performance for the remaining analytes is presented in Table 59 through Table 62.

<table><tr><td rowspan=2 colspan=1>Organism Identified by SOCand Detected by the BCID2 Panel</td><td rowspan=2 colspan=1>N</td><td rowspan=1 colspan=2>CTX-M</td><td rowspan=1 colspan=2>KPC</td><td rowspan=1 colspan=2>NDM</td><td rowspan=1 colspan=2>VIM</td><td rowspan=1 colspan=2>Overall(any resistance gene)</td></tr><tr><td rowspan=1 colspan=1>PPA</td><td rowspan=1 colspan=1>NPA</td><td rowspan=1 colspan=1>PPA</td><td rowspan=1 colspan=1>NPA</td><td rowspan=1 colspan=1>PPA</td><td rowspan=1 colspan=1>NPA</td><td rowspan=1 colspan=1>PPA</td><td rowspan=1 colspan=1>NPA</td><td rowspan=1 colspan=1>PPA</td><td rowspan=1 colspan=1>NPA</td></tr><tr><td rowspan=1 colspan=1>Overall(any associated organism identified)</td><td rowspan=1 colspan=1>317</td><td rowspan=1 colspan=1>46/46(100%)</td><td rowspan=1 colspan=1>267/271(98.5%)</td><td rowspan=1 colspan=1>4/4(100%)</td><td rowspan=1 colspan=1>313/313(100%)</td><td rowspan=1 colspan=1>1/1(100%)</td><td rowspan=1 colspan=1>315/316(99.7%)</td><td rowspan=1 colspan=1>4/4(100%)</td><td rowspan=1 colspan=1>312/313(997%)</td><td rowspan=1 colspan=1>53/53(100%)[93.2-100%]</td><td rowspan=1 colspan=1>261/264(98.9%)[96.7-99.6%]</td></tr><tr><td rowspan=1 colspan=1>Acinetobacter calcoaceticus-baumannii complex</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>0/0(-)</td><td rowspan=1 colspan=1>11/12(91.7%)</td><td rowspan=1 colspan=1>0/0(-)</td><td rowspan=1 colspan=1>12/12(100%)</td><td rowspan=1 colspan=1>0/0(-)</td><td rowspan=1 colspan=1>12/12(100%)</td><td rowspan=1 colspan=1>0/0(-)</td><td rowspan=1 colspan=1>12/12(100%)</td><td rowspan=1 colspan=1>0/0(-)</td><td rowspan=1 colspan=1>11/12(91.7%)</td></tr><tr><td rowspan=1 colspan=1>Enterobacterales</td><td rowspan=1 colspan=1>276</td><td rowspan=1 colspan=1>46/46(100%)</td><td rowspan=1 colspan=1>229/230(99.6%)</td><td rowspan=1 colspan=1>4/4(100%)</td><td rowspan=1 colspan=1>272/272(100%)</td><td rowspan=1 colspan=1>1/1(100%)</td><td rowspan=1 colspan=1>275/275(100%)</td><td rowspan=1 colspan=1>1/1(100%)</td><td rowspan=1 colspan=1>274/27599.6%)</td><td rowspan=1 colspan=1>50/50a(10%)</td><td rowspan=1 colspan=1>225/22699.6%)</td></tr><tr><td rowspan=1 colspan=1>Enterobacter cloacae complex</td><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>0/0(-)</td><td rowspan=1 colspan=1>16/1610%)</td><td rowspan=1 colspan=1>0/0(-)</td><td rowspan=1 colspan=1>16/16(100%)</td><td rowspan=1 colspan=1>0/0(-)</td><td rowspan=1 colspan=1>16/16(100%)</td><td rowspan=1 colspan=1>0/0(-)</td><td rowspan=1 colspan=1>16/16(100%)</td><td rowspan=1 colspan=1>0/0(-)</td><td rowspan=1 colspan=1>16/16(100%)</td></tr><tr><td rowspan=1 colspan=1>Escherichia coli</td><td rowspan=1 colspan=1>158</td><td rowspan=1 colspan=1>30/30(100%)</td><td rowspan=1 colspan=1>128/128(100%)</td><td rowspan=1 colspan=1>0/0(-)</td><td rowspan=1 colspan=1>158/158(1100%)</td><td rowspan=1 colspan=1>0/0(-)</td><td rowspan=1 colspan=1>158/158(100%)</td><td rowspan=1 colspan=1>0/0(-)</td><td rowspan=1 colspan=1>158/158(100%)</td><td rowspan=1 colspan=1>30/30(100%)</td><td rowspan=1 colspan=1>128/128100%)</td></tr><tr><td rowspan=1 colspan=1>Klebsiella aerogenes</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0/0(-)</td><td rowspan=1 colspan=1>1/2(50.0%)</td><td rowspan=1 colspan=1>0/0(-)</td><td rowspan=1 colspan=1>2/2(100%)</td><td rowspan=1 colspan=1>0/0(-)</td><td rowspan=1 colspan=1>2/2(100%)</td><td rowspan=1 colspan=1>0/0(-)</td><td rowspan=1 colspan=1>2/2(100%)</td><td rowspan=1 colspan=1>0/0(-)</td><td rowspan=1 colspan=1>1/2(50.0%)</td></tr><tr><td rowspan=1 colspan=1>Klebsiella oxytoca</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>0/0(-)</td><td rowspan=1 colspan=1>8/8(100%)</td><td rowspan=1 colspan=1>0/0(-)</td><td rowspan=1 colspan=1>8/8(100%)</td><td rowspan=1 colspan=1>0/0(-)</td><td rowspan=1 colspan=1>8/8(100%)</td><td rowspan=1 colspan=1>0/0(-)</td><td rowspan=1 colspan=1>8/8(100%)</td><td rowspan=1 colspan=1>0/0(-)</td><td rowspan=1 colspan=1>8/8(100%)</td></tr><tr><td rowspan=1 colspan=1>Klebsiella pneumoniae group</td><td rowspan=1 colspan=1>55</td><td rowspan=1 colspan=1>12/12(100%)</td><td rowspan=1 colspan=1>43/43(100%)</td><td rowspan=1 colspan=1>4/4(100%)</td><td rowspan=1 colspan=1>51/51(100%)</td><td rowspan=1 colspan=1>1/1(100%)</td><td rowspan=1 colspan=1>54/54100%)</td><td rowspan=1 colspan=1>1/1(100%)</td><td rowspan=1 colspan=1>53/54(98.1%)</td><td rowspan=1 colspan=1>16/16a(100%)</td><td rowspan=1 colspan=1>39/39100%)</td></tr><tr><td rowspan=1 colspan=1>Proteus spp.</td><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>4/4(100%)</td><td rowspan=1 colspan=1>10/10(00%)</td><td rowspan=1 colspan=1>0/0(-)</td><td rowspan=1 colspan=1>14/14(100%)</td><td rowspan=1 colspan=1>0/0(-)</td><td rowspan=1 colspan=1>14/14(100%)%)</td><td rowspan=1 colspan=1>0/0(-)</td><td rowspan=1 colspan=1>14/14(100%)</td><td rowspan=1 colspan=1>4/4(100%)</td><td rowspan=1 colspan=1>10/10(100%)</td></tr><tr><td rowspan=1 colspan=1>Salmonella spp.</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>0/0(-)</td><td rowspan=1 colspan=1>5/5(100%)</td><td rowspan=1 colspan=1>0/0(-)</td><td rowspan=1 colspan=1>5/5(100%)</td><td rowspan=1 colspan=1>$(</td><td rowspan=1 colspan=1>5/15(100%)</td><td rowspan=1 colspan=1>$(v$</td><td rowspan=1 colspan=1>5/5(100%)</td><td rowspan=1 colspan=1>0/0(-)</td><td rowspan=1 colspan=1>5/5(100%)</td></tr><tr><td rowspan=1 colspan=1>Serratia marcescens</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>0/0(-)</td><td rowspan=1 colspan=1>11/11(100%)</td><td rowspan=1 colspan=1>0/0(-))</td><td rowspan=1 colspan=1>11/11(100%)</td><td rowspan=1 colspan=1>0/0(-)</td><td rowspan=1 colspan=1>11/11(100%)</td><td rowspan=1 colspan=1>0/0(-)</td><td rowspan=1 colspan=1>11/11(100%)</td><td rowspan=1 colspan=1>0/0(-)</td><td rowspan=1 colspan=1>11/11(100%)</td></tr><tr><td rowspan=1 colspan=1>Pseudomonas aeruginosa</td><td rowspan=1 colspan=1>29</td><td rowspan=1 colspan=1>0/0(-)</td><td rowspan=1 colspan=1>27/29(93.1%)</td><td rowspan=1 colspan=1>0/0(-)</td><td rowspan=1 colspan=1>29/29(1100%)</td><td rowspan=1 colspan=1>0/0(-)</td><td rowspan=1 colspan=1>28/29 6.6%)</td><td rowspan=1 colspan=1>3/3(00%)</td><td rowspan=1 colspan=1>26/26100%)</td><td rowspan=1 colspan=1>3/3(100%)</td><td rowspan=1 colspan=1>25/26(96.2%)</td></tr></table>

-D

Table 60. mecA/C Performance Table (as compared to PCR/seq on cultured isolate(s) from prospective PBC specimens)   

<table><tr><td rowspan=2 colspan=1>Organism Identified by SOCand Detected by the BCID2 Panel</td><td rowspan=1 colspan=3>Positive Percent Agreement</td><td rowspan=1 colspan=3>Negative Percent Agreement</td></tr><tr><td rowspan=1 colspan=1>TP/(TP + FN)</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95%CI</td><td rowspan=1 colspan=1>TN/(TN + FP)</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95%CI</td></tr><tr><td rowspan=1 colspan=1>Overall(any associated organism identified)</td><td rowspan=1 colspan=1>162/165</td><td rowspan=1 colspan=1>98.2</td><td rowspan=1 colspan=1>94.8-99.4%</td><td rowspan=1 colspan=1>51/60</td><td rowspan=1 colspan=1>85.0</td><td rowspan=1 colspan=1>73.9-91.9%</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus epidermidis</td><td rowspan=1 colspan=1>162/165</td><td rowspan=1 colspan=1>98.2</td><td rowspan=1 colspan=1>94.8-99.4%</td><td rowspan=1 colspan=1>47/56</td><td rowspan=1 colspan=1>83.9</td><td rowspan=1 colspan=1>72.2-91.3%</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus lugdunensis</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>4/4</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>51.0-100%</td></tr></table>

Table 61. mecA/C and MREJ (MRSA) Performance Table (as compared to PCR/seq on cultured isolate(s) from prospective PBC specimens)   

<table><tr><td rowspan=2 colspan=1>Organism Identified by SOCand Detected by the BCID2 Panel</td><td rowspan=1 colspan=3>Positive Percent Agreement</td><td rowspan=1 colspan=3>Negative Percent Agreement</td></tr><tr><td rowspan=1 colspan=1>TP/(TP + FN)</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95%CI</td><td rowspan=1 colspan=1>TN/(TN + FP)</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95%CI</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus aureus</td><td rowspan=1 colspan=1>52/52</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>93.1-100%</td><td rowspan=1 colspan=1>97/97</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>96.2-100%</td></tr></table>

Table 62. vanA/B Performance Table (as compared to PCR/seq on cultured isolate(s) from prospective PBC specimens)   

<table><tr><td rowspan=2 colspan=1>Organism Identified by SOCand Detected by the BCID2 Panel</td><td rowspan=1 colspan=3>Positive Percent Agreement</td><td rowspan=1 colspan=3>Negative Percent Agreement</td></tr><tr><td rowspan=1 colspan=1>TP/(TP + FN)</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95%CI</td><td rowspan=1 colspan=1>TN(TN + FP)</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95%CI</td></tr><tr><td rowspan=1 colspan=1>Overall(any associated organism identified)</td><td rowspan=1 colspan=1>22/23</td><td rowspan=1 colspan=1>95.7</td><td rowspan=1 colspan=1>79.0-99.2%</td><td rowspan=1 colspan=1>35/35</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>90.1-100%</td></tr><tr><td rowspan=1 colspan=1>Enterococcus faecalis</td><td rowspan=1 colspan=1>1/1</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>88.6-100%</td></tr><tr><td rowspan=1 colspan=1>Enterococcus faecium</td><td rowspan=1 colspan=1>21/22</td><td rowspan=1 colspan=1>95.5</td><td rowspan=1 colspan=1>78.2-99.2%</td><td rowspan=1 colspan=1>5/5</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>56.6-100%</td></tr></table>

For prospective specimens, the BioFire BCID2 Panel AMR gene reporting in the specimen was also compared to phenotypic antimicrobial susceptibility testing (AST) methods performed on organism isolates recovered from those specimens. The results presented in Table 63 through Table 68 are only for specimens with concordant (true positive) results and are further stratified by each applicable host organism recovered from that specimen. Note that antimicrobial resistance, particularly extended-spectrum $\beta$ -lactamase (ESBL) activity and carbapenem resistance, may be due to mechanisms other than the presence of the AMR genes detected by the BioFire BCID2 Panel; conversely, detection of these genes may not always confer an antimicrobial resistance phenotype. Additionally, discordant results between mecA/C detection in a blood culture specimen by the BioFire BCID2 Panel and the observed methicillin (oxacillin/cefoxitin) resistance of cultured Staphylococcus isolates may be due to polymicrobial Staphylococcus cultures containing a mixture of resistant and sensitive organisms.

Table 63. CTX-M Performance Table (as compared to phenotypic AST methods for ESBL activity on cultured isolate(s) from prospective PBC specimens)   

<table><tr><td colspan="1" rowspan="2">Organism Identified by SOCand Detected by the BCID2 Panel</td><td colspan="2" rowspan="1">N</td><td colspan="3" rowspan="1">Positive Percent Agreement</td><td colspan="3" rowspan="1">Negative Percent Agreement</td></tr><tr><td colspan="1" rowspan="1">ESBL</td><td colspan="1" rowspan="1">Non-ESBL</td><td colspan="1" rowspan="1">TP/(TP + FN)</td><td colspan="1" rowspan="1">%</td><td colspan="1" rowspan="1">95%CI</td><td colspan="1" rowspan="1">TN(TN + FP)</td><td colspan="1" rowspan="1">%</td><td colspan="1" rowspan="1">95%CI</td></tr><tr><td colspan="1" rowspan="1">Overall(any associated organism identified)</td><td colspan="2" rowspan="1">31346 267</td><td colspan="1" rowspan="1">43/46</td><td colspan="1" rowspan="1">93.5</td><td colspan="1" rowspan="1">82.5-97.8%</td><td colspan="1" rowspan="1">260/267</td><td colspan="1" rowspan="1">97.4</td><td colspan="1" rowspan="1">94.7-98.7%</td></tr><tr><td colspan="1" rowspan="1">Acinetobacter calcoaceticus-baumanniicomplex</td><td colspan="2" rowspan="1">120   12</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">11/12</td><td colspan="1" rowspan="1">91.7</td><td colspan="1" rowspan="1">64.6-98.5%</td></tr><tr><td colspan="1" rowspan="1">Enterobacterales</td><td colspan="2" rowspan="1">27346227</td><td colspan="1" rowspan="1">43/46</td><td colspan="1" rowspan="1">93.5</td><td colspan="1" rowspan="1">82.5-97.8%</td><td colspan="1" rowspan="1">223/227</td><td colspan="1" rowspan="1">98.2</td><td colspan="1" rowspan="1">95.6-99.3%</td></tr><tr><td colspan="1" rowspan="1">Enterobacter cloacae complex</td><td colspan="2" rowspan="1">160 16</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">16/16</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">80.6-100%</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="2" rowspan="1">15730  127</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">88.6-100%</td><td colspan="1" rowspan="1">127/127</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">97.1-100%</td></tr><tr><td colspan="1" rowspan="1">Klebsiella aerogenes</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1/2</td><td colspan="1" rowspan="1">50.0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Klebsiella oxytoca</td><td colspan="1" rowspan="1">81</td><td colspan="1" rowspan="1">7</td><td colspan="1" rowspan="1">0/1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">7/7</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">64.6-100%</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae group</td><td colspan="1" rowspan="1">54</td><td colspan="1" rowspan="1">43</td><td colspan="1" rowspan="1">10/11</td><td colspan="1" rowspan="1">90.9</td><td colspan="1" rowspan="1">62.3-98.4%</td><td colspan="1" rowspan="1">41/43</td><td colspan="1" rowspan="1">95.3</td><td colspan="1" rowspan="1">84.5-98.7%</td></tr><tr><td colspan="1" rowspan="1">Proteus spp.</td><td colspan="1" rowspan="1">144</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">3/4</td><td colspan="1" rowspan="1">75.0</td><td colspan="1" rowspan="1">30.1-95.4%</td><td colspan="1" rowspan="1">9/10</td><td colspan="1" rowspan="1">90.0</td><td colspan="1" rowspan="1">59.6-98.2%</td></tr><tr><td colspan="1" rowspan="1">Salmonella spp.</td><td colspan="1" rowspan="1">50</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">5/5</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">56.6-100%</td></tr><tr><td colspan="1" rowspan="1">Serratia marcescens</td><td colspan="1" rowspan="1">110</td><td colspan="1" rowspan="1">11</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">11/11</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">74.1-100%</td></tr><tr><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">280</td><td colspan="1" rowspan="1">28</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">26/28</td><td colspan="1" rowspan="1">92.9</td><td colspan="1" rowspan="1">77.4-98.0%</td></tr></table>

<table><tr><td rowspan="2">Organism Identified by SOC and Detected by BioFire BCID2 Panel</td><td colspan="2">N</td><td colspan="2">IMP KPC</td><td colspan="2">NDM</td><td colspan="2">OXA-48-like</td><td colspan="2">VIM</td><td colspan="2">Overall (any resistance gene)</td></tr><tr><td>R S</td><td colspan="2">PPA NPA</td><td colspan="2">PPA NPA</td><td colspan="2">PPA NPA</td><td colspan="2">PPA NPA</td><td colspan="2">PPA NPA PPA</td><td>NPA</td></tr><tr><td>Overall</td><td>313</td><td>0/25 288/288</td><td>4/25</td><td>288/288</td><td>2/25</td><td>288/288</td><td>$( 0%)</td><td>267/267</td><td>4/25</td><td>287/288</td><td>8/25</td><td>287/288 (99.7%)</td></tr><tr><td rowspan="2">(any associated organism identified)</td><td rowspan="2">25 |288</td><td rowspan="2">(0%)</td><td rowspan="2">(0 (1%)</td><td rowspan="2">(16.0%) (100%)</td><td rowspan="2">(8.0%)</td><td rowspan="2">(10%)</td><td rowspan="2"></td><td rowspan="2"> (10%)</td><td rowspan="2">(16.0%)</td><td rowspan="2">((9.7%)</td><td rowspan="2">(32.0%) [17.2-51.6%]</td><td rowspan="2">90.8% </td></tr><tr><td>0/0</td></tr><tr><td>Acinetobacter calcoaceticus-baumannii complex</td><td>12 12 − 0</td><td>0/12 (0%%)</td><td>0/0 (-</td><td>$O079 (-)</td><td>0/12 (%)</td><td></td><td>$00</td><td>N/A</td><td>N/A</td><td>O019 0/0 (-</td><td>0/12 (%)</td><td>0/0 (-)</td></tr><tr><td>Enterobacterales</td><td>273 6 267</td><td>0/6</td><td>267/267</td><td>4/6 267/267</td><td>1/6</td><td>267/267</td><td>0/6</td><td>267/267</td><td>1/6</td><td>266/267</td><td>5/6a (83.3%)</td><td>266/267</td></tr><tr><td rowspan="2">Enterobacter cloacae complex</td><td rowspan="2">16</td><td>(0%)</td><td> (10%)</td><td>(66.7%) ( (%)</td><td>(16.7%)</td><td> (10%)</td><td>(0%) 00 </td><td>0 (10%) 16/16</td><td>(16.7%)</td><td>(9.6%)</td><td></td><td>( (99.6%)</td></tr><tr><td></td><td>16/16</td><td>16/16</td><td>0.0</td><td></td><td>16/16</td><td></td><td>(100%)</td><td>16/16 (100%)</td><td></td><td>16/16</td></tr><tr><td rowspan="2">0 Escherichia coli</td><td>16 157</td><td>00 </td><td>(100%)</td><td>00 (1000%)</td><td></td><td>(10(0%)</td><td></td><td></td><td>00 0/0</td><td>157/157</td><td>00 0/0</td><td>(100%)</td></tr><tr><td>0 157</td><td>0/0 (-)</td><td>1557/157 (100%)</td><td>0/0 157/157 (-) (100%)</td><td>0/0 (-)</td><td></td><td>157/157 (100%)</td><td>0/0 (-)</td><td>157/157 (100%)</td><td>(100%)</td><td>(-)</td><td>157/157</td></tr><tr><td rowspan="2">Klebsiella aerogenes</td><td>2</td><td>$00 </td><td>2/2</td><td>$00 2/2</td><td>$00</td><td>2/2</td><td>$00 </td><td>2/2</td><td>(-) $00 </td><td>2/2</td><td></td><td>(1000%) 2/2</td></tr><tr><td>0 2</td><td></td><td>(100%)</td><td>(100%)</td><td></td><td></td><td>(100%)</td><td></td><td>(100%)</td><td>(100%)</td><td>$0.0</td><td>(100%)</td></tr><tr><td rowspan="2">Klebsiella oxytoca</td><td>8</td><td></td><td>8/8</td><td>8/8</td><td></td><td>8/8</td><td></td><td>8/8</td><td>$00</td><td>8/8</td><td></td><td>8/8</td></tr><tr><td>0 8</td><td>$00 </td><td>(100%)</td><td>$00 (100%)</td><td>$00</td><td></td><td>(100%)</td><td>$00</td><td>(110%)</td><td>(10%)</td><td>$00</td><td>(100%)</td></tr><tr><td rowspan="2">Klebsiella pneumoniae group</td><td>54</td><td>0/6</td><td>48/48</td><td>4/6 48/48</td><td>1/6</td><td>48/48</td><td>0/6</td><td>48/48</td><td>1/6</td><td>47/48</td><td>5/6a</td><td>47/48</td></tr><tr><td>6</td><td>(0%)</td><td>(100%)</td><td>(66.7%) (10%)</td><td>(16.7%)</td><td></td><td>(100%)</td><td>(0%)</td><td>(100%) (16.7%)</td><td>(97.9%)</td><td>(83.3%)</td><td>(97.9%)</td></tr><tr><td rowspan="2">Proteus spp.</td><td>14 14</td><td></td><td>14/14</td><td>14/14</td><td>00</td><td>14/14</td><td></td><td>14/14</td><td>00</td><td>14/14</td><td>0/0</td><td>14/14</td></tr><tr><td>0</td><td>00</td><td>(100%)</td><td>00 (100%)</td><td></td><td></td><td>(10%)</td><td>00 </td><td>(100%)</td><td>(100%)</td><td>(-</td><td>(100%)</td></tr><tr><td rowspan="2">Salmonella spp.</td><td>5</td><td></td><td>5/5 (100%)</td><td>5/5 (100%)</td><td>$00</td><td></td><td>5/5 (100%)</td><td>(100%)</td><td>$00 5/5</td><td>5/5 (100%)</td><td></td><td></td></tr><tr><td>0 11</td><td>$00 0/0</td><td>11/11</td><td>$00 11/11</td><td></td><td></td><td>11/11</td><td>$00 0/0</td><td>11/11 $00</td><td>11/11</td><td>$0.0 0/0</td><td>(1% 1%) 11/11</td></tr><tr><td>Serratia marcescens</td><td>11 0</td><td>$(-)</td><td>(100%)</td><td>$00 </td><td>(100%)</td><td>$00</td><td>(100%)</td><td>(-)</td><td>(100%)</td><td>(100%)</td><td>$(-</td><td>(100%)</td></tr><tr><td>Pseudomonas aeruginosa</td><td>28</td><td>0/7</td><td>21/21 (0%) (100%)</td><td>0/7 (0%)</td><td>21/21 (100%)</td><td>1/7 (14.3%)</td><td>21/21 (100%)</td><td>N/A</td><td>N/A</td><td>3/7 21/21 (42.9%) (100%)</td><td>3/7b (42.9%)</td><td>21/21 (100%)</td></tr></table>

ei grouolaa presec al AMR ges  determi by e BCID nel NDM  VIM) One P.aeruginosa isolate had presence of dual AMR genes as determined by the BID2 Panel (NDM and VIM)

Table 65. mecA/C Performance Table (as compared to phenotypic AST methods for methicillin (oxacillin/cefoxitin) resistance on cultured isolate(s) from prospective PBC specimens)   

<table><tr><td rowspan=2 colspan=1>Organism Identified by SOCand Detected by the BCID2 Panel</td><td rowspan=1 colspan=2>N</td><td rowspan=1 colspan=3>Positive Percent Agreement</td><td rowspan=1 colspan=3>Negative Percent Agreement</td></tr><tr><td rowspan=1 colspan=1>R</td><td rowspan=1 colspan=1>S</td><td rowspan=1 colspan=1>TP/(TP + FN)</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95%CI</td><td rowspan=1 colspan=1>TN/(TN + FP)</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95%CI</td></tr><tr><td rowspan=1 colspan=1>Overall(any associated organism identified)</td><td rowspan=1 colspan=2>22516461</td><td rowspan=1 colspan=1>160/164</td><td rowspan=1 colspan=1>97.6</td><td rowspan=1 colspan=1>93.9-99.0%</td><td rowspan=1 colspan=1>50/61</td><td rowspan=1 colspan=1>82.0</td><td rowspan=1 colspan=1>70.5-89.6%</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus epidermidis</td><td rowspan=1 colspan=2>22116358</td><td rowspan=1 colspan=1>160/163</td><td rowspan=1 colspan=1>98.2</td><td rowspan=1 colspan=1>94.7-99.4%</td><td rowspan=1 colspan=1>47/58</td><td rowspan=1 colspan=1>81.0</td><td rowspan=1 colspan=1>69.1-89.1%</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus lugdunensis</td><td rowspan=1 colspan=2>41  3</td><td rowspan=1 colspan=1>0/1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>43.9-100%</td></tr></table>

Table 66. mecA/C and MREJ (MRSA) Performance Table (as compared to phenotypic AST methods for methicillin (oxacillin/cefoxitin) resistance on cultured isolate(s) from prospective PBC specimens)   

<table><tr><td rowspan=2 colspan=1>Organism Identified by SOCand Detected by the BCID2 Panel</td><td rowspan=1 colspan=2>N</td><td rowspan=1 colspan=3>Positive Percent Agreement</td><td rowspan=1 colspan=3>Negative Percent Agreement</td></tr><tr><td rowspan=1 colspan=1>R</td><td rowspan=1 colspan=1>S</td><td rowspan=1 colspan=1>TP/(TP + FN)</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95%CI</td><td rowspan=1 colspan=1>TN/(TN + FP)</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95%CI</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus aureus</td><td rowspan=1 colspan=2>1495297</td><td rowspan=1 colspan=1>52/52</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>93.1-100%</td><td rowspan=1 colspan=1>97/97</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>96.2-100%</td></tr></table>

Table 67. mcr-1 Performance Table (as compared to phenotypic AST methods for colistin on cultured isolate(s) from prospective PBC specimens)a   

<table><tr><td colspan="1" rowspan="1">Organism Identified by SOCand Detected by FilmArray BCID2 Panel</td><td colspan="1" rowspan="1">MICb(μg/mL)</td><td colspan="1" rowspan="1">N</td></tr><tr><td colspan="1" rowspan="6">Overall(any associated organism identified)</td><td colspan="1" rowspan="1">0.25</td><td colspan="1" rowspan="1">242</td></tr><tr><td colspan="1" rowspan="1">0.5</td><td colspan="1" rowspan="1">241</td></tr><tr><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">112</td></tr><tr><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">21</td></tr><tr><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">8</td></tr><tr><td colspan="1" rowspan="1">&gt;4</td><td colspan="1" rowspan="1">6</td></tr><tr><td colspan="1" rowspan="6">Enterobacter cloacae complex</td><td colspan="1" rowspan="1">0.25</td><td colspan="1" rowspan="1">16</td></tr><tr><td colspan="1" rowspan="1">0.5</td><td colspan="1" rowspan="1">15</td></tr><tr><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">12</td></tr><tr><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">5</td></tr><tr><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">2</td></tr><tr><td colspan="1" rowspan="1">&gt;4</td><td colspan="1" rowspan="1">2</td></tr><tr><td colspan="1" rowspan="6">Escherichia coli</td><td colspan="1" rowspan="1">0.25</td><td colspan="1" rowspan="1">157</td></tr><tr><td colspan="1" rowspan="1">0.5</td><td colspan="1" rowspan="1">157</td></tr><tr><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">60</td></tr><tr><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">7</td></tr><tr><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">&gt;4</td><td colspan="1" rowspan="1">2</td></tr><tr><td colspan="1" rowspan="6">Klebsiella aerogenes</td><td colspan="1" rowspan="1">0.25</td><td colspan="1" rowspan="1">2</td></tr><tr><td colspan="1" rowspan="1">0.5</td><td colspan="1" rowspan="1">2</td></tr><tr><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">&gt;4</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="6">Klebsiella oxytoca</td><td colspan="1" rowspan="1">0.25</td><td colspan="1" rowspan="1">8</td></tr><tr><td colspan="1" rowspan="1">0.5</td><td colspan="1" rowspan="1">8</td></tr><tr><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">5</td></tr><tr><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">&gt;4</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="6">Klebsiella pneumoniae group</td><td colspan="1" rowspan="1">0.25</td><td colspan="1" rowspan="1">54</td></tr><tr><td colspan="1" rowspan="1">0.5</td><td colspan="1" rowspan="1">54</td></tr><tr><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">29</td></tr><tr><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">6</td></tr><tr><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">&gt;4</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">Salmonella spp.</td><td colspan="1" rowspan="1">0.25</td><td colspan="1" rowspan="1">5</td></tr><tr><td colspan="1" rowspan="1">Organism Identified by SOCand Detected by FilmArrayBCID2 Panel</td><td colspan="1" rowspan="1">MICb(μg/mL)</td><td colspan="1" rowspan="1">N</td></tr><tr><td colspan="1" rowspan="5"></td><td colspan="1" rowspan="1">0.5</td><td colspan="1" rowspan="1">5</td></tr><tr><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">5</td></tr><tr><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">&gt;4</td><td colspan="1" rowspan="1">0</td></tr></table>

aNo mcr-1 specimens were identified by the BioFire BCID2 Panel in the prospective clinical evaluation. bMinimum inhibitory concentration (MIC) values were determined using microbroth dilution. As of February 2020, the United States Food and Drug Administration has not established or recognized MIC breakpoints for colistin antimicrobial susceptibility testing (AST) related to mcr-1.

Table 68. vanA/B Performance Table (as compared to phenotypic AST methods for vancomycin resistance on cultured isolate(s) from prospective PBC specimens)   

<table><tr><td rowspan=2 colspan=1>Organism Identified by SOCand Detected by the BCID2 Panel</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2>Positive Percent Agreement</td><td rowspan=1 colspan=1>greement</td><td rowspan=1 colspan=3>Negative Percent Agreement</td></tr><tr><td rowspan=1 colspan=1>R</td><td rowspan=1 colspan=1>S</td><td rowspan=1 colspan=1>TP/(TP + FN)</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95%CI</td><td rowspan=1 colspan=1>TN/(TN + FP)</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95%CI</td></tr><tr><td rowspan=1 colspan=1>Overall(any associated organism identified)</td><td rowspan=1 colspan=2>582236</td><td rowspan=1 colspan=1>21/22</td><td rowspan=1 colspan=1>95.5</td><td rowspan=1 colspan=1>78.2-99.2%</td><td rowspan=1 colspan=1>35/36</td><td rowspan=1 colspan=1>97.2</td><td rowspan=1 colspan=1>85.8-99.5%</td></tr><tr><td rowspan=1 colspan=1>Enterococcus faecalis</td><td rowspan=1 colspan=1>31</td><td rowspan=1 colspan=1>130</td><td rowspan=1 colspan=1>1/1</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>88.6-100%</td></tr><tr><td rowspan=1 colspan=1>Enterococcus faecium</td><td rowspan=1 colspan=2>2721 6</td><td rowspan=1 colspan=1>20/21</td><td rowspan=1 colspan=1>95.2</td><td rowspan=1 colspan=1>77.3-99.2%</td><td rowspan=1 colspan=1>5/6</td><td rowspan=1 colspan=1>83.3</td><td rowspan=1 colspan=1>43.6-97.0%</td></tr></table>

The overall success rate for initial specimen tests in all three clinical studies was $9 9 . 4 \%$ (2042/2055). Eight tests (8/2055; $0 . 4 \% )$ did not complete on the initial test attempt, resulting in an instrument success rate of $9 9 . 6 \%$ (2047/2055) for initial specimen tests. Seven of the eight specimens were able to be retested and valid results were produced after a single retest. Of the 2047 tests that successfully produced a completed run on the initial test, 2042 had valid pouch controls. This represents a $9 9 . 8 \%$ (2042/2047) success rate for pouch controls in completed runs in the initial specimen tests. The five specimens with invalid control(s) were able to be retested and valid results were produced on the first retest.

# Bench (Analytical) Performance

# Evaluation of Blood Culture Bottle Types

The BioFire BCID2 Panel was tested for compatibility with thirteen different blood culture bottle types. Over 35 different bacterial or yeast isolates were each mixed with human whole blood (acid citrate dextrose anticoagulant), seeded directly into a blood culture bottle (≤300 CFU/bottle), and incubated for growth in a continuous monitoring blood culture system (or standard incubator for the VersaTREK bottles) until indicated as positive for growth. For each positive bottle (note that some isolates did not grow to positivity in every bottle type), the sample was tested with the BioFire BCID2 Panel and was plate enumerated to determine the concentration of organism in the bottle. Samples were collected, tested, and enumerated within one hour of positive bottle indication and at 24 hours after positive bottle indication $( + 2 4 \mathsf { h } )$ .

The correct organism (and AMR gene) detection was reported by the BioFire BCID2 Panel for every positive bottle tested at the positive bottle indication and 24 hours after positivity $( 8 1 5 / 8 1 5 = 1 0 0 \%$ ; Table 69) for all bottle types evaluated.

Table 69. Compatibility of the BioFire BCID2 Panel with Different Blood Culture Bottle Types   

<table><tr><td colspan="1" rowspan="2">ManufacturerSystem</td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2">Bottle Type</td><td colspan="1" rowspan="2">Time Point</td><td colspan="3" rowspan="1">Seeded Organism</td></tr><tr><td colspan="1" rowspan="1">Gram-positiveBacteria</td><td colspan="1" rowspan="1">Gram-negativeBacteria</td><td colspan="1" rowspan="1">Yeast</td></tr><tr><td colspan="1" rowspan="8">bioMerieuxBaTALERTVIRTUO</td><td colspan="1" rowspan="4">Aerobic</td><td colspan="1" rowspan="2">BacT/ALERT® SA</td><td colspan="1" rowspan="1">Bottle Positive</td><td colspan="1" rowspan="1">11/11</td><td colspan="1" rowspan="1">18/18</td><td colspan="1" rowspan="1">8/8</td></tr><tr><td colspan="1" rowspan="1">Positive +24h</td><td colspan="1" rowspan="1">11/11</td><td colspan="1" rowspan="1">18/18</td><td colspan="1" rowspan="1">8/8</td></tr><tr><td colspan="1" rowspan="2">BacT/ALERT® FA Plus</td><td colspan="1" rowspan="1">Bottle Positive</td><td colspan="1" rowspan="1">11/11</td><td colspan="1" rowspan="1">18/18</td><td colspan="1" rowspan="1">8/8</td></tr><tr><td colspan="1" rowspan="1">Positive +24h</td><td colspan="1" rowspan="1">11/11</td><td colspan="1" rowspan="1">18/18</td><td colspan="1" rowspan="1">8/8</td></tr><tr><td colspan="1" rowspan="4">Anaerobic</td><td colspan="1" rowspan="2">BacT/ALERT® SN</td><td colspan="1" rowspan="1">Bottle Positive</td><td colspan="1" rowspan="1">11/11</td><td colspan="1" rowspan="1">19/19</td><td colspan="1" rowspan="1">717</td></tr><tr><td colspan="1" rowspan="1">Positive +24h</td><td colspan="1" rowspan="1">11/11</td><td colspan="1" rowspan="1">19/19</td><td colspan="1" rowspan="1">717</td></tr><tr><td colspan="1" rowspan="2">BacT/ALERT® FN Plus</td><td colspan="1" rowspan="1">Bottle Positive</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">15/15</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">Positive +24h</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">15/15</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="2">ManufacturerSystem</td><td colspan="2" rowspan="2">Bottle Type</td><td colspan="1" rowspan="2">Time Point</td><td colspan="3" rowspan="1">Seeded Organism</td></tr><tr><td colspan="1" rowspan="1">Gram-positiveBacteria</td><td colspan="1" rowspan="1">Gram-negativeBacteria</td><td colspan="1" rowspan="1">Yeast</td></tr><tr><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2">Pediatric</td><td colspan="1" rowspan="2">BacT/ALERT® PF Plus</td><td colspan="1" rowspan="1">Bottle Positive</td><td colspan="1" rowspan="1">11/11</td><td colspan="1" rowspan="1">18/18</td><td colspan="1" rowspan="1">8/8</td></tr><tr><td colspan="1" rowspan="1">Positive +24h</td><td colspan="1" rowspan="1">11/11</td><td colspan="1" rowspan="1">18/18</td><td colspan="1" rowspan="1">8/8</td></tr><tr><td colspan="1" rowspan="12">Becton DickinsonBACTEC™ FX40</td><td colspan="1" rowspan="4">Aerobic</td><td colspan="1" rowspan="2">BACTEC™ Standard</td><td colspan="1" rowspan="1">Bottle Positive</td><td colspan="1" rowspan="1">11/11</td><td colspan="1" rowspan="1">18/18</td><td colspan="1" rowspan="1">717</td></tr><tr><td colspan="1" rowspan="1">Positive +24h</td><td colspan="1" rowspan="1">11/11</td><td colspan="1" rowspan="1">18/18</td><td colspan="1" rowspan="1">7/17</td></tr><tr><td colspan="1" rowspan="2">BACTEC™ Plus+</td><td colspan="1" rowspan="1">Bottle Positive</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">18/18</td><td colspan="1" rowspan="1">8/8</td></tr><tr><td colspan="1" rowspan="1">Positive +24h</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">18/18</td><td colspan="1" rowspan="1">8/8</td></tr><tr><td colspan="1" rowspan="4">Anaerobic</td><td colspan="1" rowspan="2">BACTEC™ Standard</td><td colspan="1" rowspan="1">Bottle Positive</td><td colspan="1" rowspan="1">9/9</td><td colspan="1" rowspan="1">13/13</td><td colspan="1" rowspan="1">1/1</td></tr><tr><td colspan="1" rowspan="1">Positive +24h</td><td colspan="1" rowspan="1">9/9</td><td colspan="1" rowspan="1">13/13</td><td colspan="1" rowspan="1">1/1</td></tr><tr><td colspan="1" rowspan="2">BACTEC™ Plus+</td><td colspan="1" rowspan="1">Bottle Positive</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">13/13</td><td colspan="1" rowspan="1">1/1</td></tr><tr><td colspan="1" rowspan="1">Positive +24h</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">13/13</td><td colspan="1" rowspan="1">1/1</td></tr><tr><td colspan="1" rowspan="2">Pediatric</td><td colspan="1" rowspan="2">BACTEC™ Peds Plus+</td><td colspan="1" rowspan="1">Bottle Positive</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">18/18</td><td colspan="1" rowspan="1">8/8</td></tr><tr><td colspan="1" rowspan="1">Positive +24h</td><td colspan="1" rowspan="1">11/11</td><td colspan="1" rowspan="1">18/18</td><td colspan="1" rowspan="1">8/8</td></tr><tr><td colspan="1" rowspan="2">Lytic</td><td colspan="1" rowspan="2">BACTEC™ Lytic</td><td colspan="1" rowspan="1">Bottle Positive</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">12/12</td><td colspan="1" rowspan="1">1/1</td></tr><tr><td colspan="1" rowspan="1">Positive +24h</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">12/12</td><td colspan="1" rowspan="1">1/1</td></tr><tr><td colspan="1" rowspan="4">ThermoaScientificVersaTREK</td><td colspan="1" rowspan="2">Aerobic</td><td colspan="1" rowspan="2">VersaTREK™ REDOX 1 ™</td><td colspan="1" rowspan="1">Bottle Positive</td><td colspan="1" rowspan="1">9/9</td><td colspan="1" rowspan="1">18/18</td><td colspan="1" rowspan="1">4/4</td></tr><tr><td colspan="1" rowspan="1">Positive +24h</td><td colspan="1" rowspan="1">9/9</td><td colspan="1" rowspan="1">18/18</td><td colspan="1" rowspan="1">4/4</td></tr><tr><td colspan="1" rowspan="2">Anaerobic</td><td colspan="1" rowspan="2">VersaTREK™"REDOX 2</td><td colspan="1" rowspan="1">Bottle Positive</td><td colspan="1" rowspan="1">9/9</td><td colspan="1" rowspan="1">13/13</td><td colspan="1" rowspan="1">0/0</td></tr><tr><td colspan="1" rowspan="1">Positive +24h</td><td colspan="1" rowspan="1">9/9</td><td colspan="1" rowspan="1">13/13</td><td colspan="1" rowspan="1">1/1</td></tr><tr><td colspan="4" rowspan="2"># Correct Detected Results/Total Positive Bottles</td><td colspan="1" rowspan="1">265/265100%</td><td colspan="1" rowspan="1">422/422100%</td><td colspan="1" rowspan="1">128/128100%</td></tr><tr><td colspan="3" rowspan="1">815/815100%</td></tr></table>

a A VersaTREK system was not available for the evaluation. Seeded VersaTREK bottles were place in a standard incubator $3 7 ^ { \circ } \mathsf { C }$ , with or without agitation) for the average time to positivity required for the same isolate in the other bottles/systems.

The concentrations of each organism enumerated from bottles at the time of positivity and 24 hours after positivity are shown in Table 70; representing the approximate range of concentrations expected in a clinical setting for a mono-microbial blood culture.

Table 70. Concentration of Organism in a Blood Culture at Positivity and 24 Hours After Positivity (+24h)   

<table><tr><td colspan="1" rowspan="2">BioFire BCID2 PanelAnalyte</td><td colspan="1" rowspan="2">Organism[AMR Gene]</td><td colspan="1" rowspan="2">Isolate ID</td><td colspan="2" rowspan="1">Concentration a</td></tr><tr><td colspan="1" rowspan="1">PositiveBottle(CFU/mL)</td><td colspan="1" rowspan="1">+24h(CFU/mL)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Gram Positive Bacteria</td><td colspan="1" rowspan="1">ria</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Enterococcus faecalis</td><td colspan="1" rowspan="1">Enterococcus faecalis [vanA/B]</td><td colspan="1" rowspan="1">ATCC 51299</td><td colspan="1" rowspan="1">3.88E+08</td><td colspan="1" rowspan="1">1.16E+09</td></tr><tr><td colspan="1" rowspan="1">Enterococcus faecium</td><td colspan="1" rowspan="1">Enterococcus faecium [vanA/B]</td><td colspan="1" rowspan="1">ATCC 700221</td><td colspan="1" rowspan="1">1.37E+08</td><td colspan="1" rowspan="1">7.48E+08</td></tr><tr><td colspan="1" rowspan="1">Listeria monocytogenes</td><td colspan="1" rowspan="1">Listeria monocytogenes</td><td colspan="1" rowspan="1">ATCC 15313</td><td colspan="1" rowspan="1">9.28E+07</td><td colspan="1" rowspan="1">3.36E+08</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus spp.</td><td colspan="1" rowspan="1">Staphylococcus hominis</td><td colspan="1" rowspan="1">ATCC 25615</td><td colspan="1" rowspan="1">3.02E+06</td><td colspan="1" rowspan="1">6.52E+07</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">Staphylococcus aureus [mecA/C andMREJ]</td><td colspan="1" rowspan="1">ATCC BAA-38</td><td colspan="1" rowspan="1">2.88E+07</td><td colspan="1" rowspan="1">3.76E+08</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus epidermidis</td><td colspan="1" rowspan="1">Staphylococcus epidermidis</td><td colspan="1" rowspan="1">ATCC 12228</td><td colspan="1" rowspan="1">1.34E+07</td><td colspan="1" rowspan="1">6.19E+08</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus lugdunensis</td><td colspan="1" rowspan="1">Staphylococcus lugdunensis</td><td colspan="1" rowspan="1">ATCC 43809</td><td colspan="1" rowspan="1">7.05E+07</td><td colspan="1" rowspan="1">7.90E+08</td></tr><tr><td colspan="1" rowspan="1">Streptococcus spp.</td><td colspan="1" rowspan="1">Streptococcus mitis</td><td colspan="1" rowspan="1">ATCC 49456</td><td colspan="1" rowspan="1">2.29E+07</td><td colspan="1" rowspan="1">1.36E+08</td></tr><tr><td colspan="1" rowspan="1">Streptococcus agalactiae</td><td colspan="1" rowspan="1">Streptococcus agalactiae</td><td colspan="1" rowspan="1">ATCC 13813</td><td colspan="1" rowspan="1">3.82E+08</td><td colspan="1" rowspan="1">5.33E+08</td></tr><tr><td colspan="1" rowspan="1">Streptococcus pneumoniae</td><td colspan="1" rowspan="1">Streptococcus pneumoniae</td><td colspan="1" rowspan="1">ATCC 6303</td><td colspan="1" rowspan="1">6.86E+07</td><td colspan="1" rowspan="1">4.53E+07</td></tr><tr><td colspan="1" rowspan="1">Streptococcus pyogenes</td><td colspan="1" rowspan="1">Streptococcus pyogenes</td><td colspan="1" rowspan="1">ATCC 49399</td><td colspan="1" rowspan="1">2.04E+08</td><td colspan="1" rowspan="1">1.83E+08</td></tr><tr><td colspan="3" rowspan="1">Gram Negative Bacteria</td><td colspan="2" rowspan="1"></td></tr><tr><td colspan="1" rowspan="2">Acinetobacter calcoaceticus-baumannii complex</td><td colspan="1" rowspan="1">Acinetobacter baumannii [NDM]</td><td colspan="1" rowspan="1">CDC FDA AR Bank #0033</td><td colspan="1" rowspan="1">2.27E+08</td><td colspan="1" rowspan="1">4.22E+08</td></tr><tr><td colspan="1" rowspan="1">Acinetobacter calcoaceticus</td><td colspan="1" rowspan="1">ATCC 23055</td><td colspan="1" rowspan="1">1.23E+07</td><td colspan="1" rowspan="1">5.46E+07</td></tr><tr><td colspan="1" rowspan="1">Bacteroides fragilis</td><td colspan="1" rowspan="1">Bacteroides fragilis</td><td colspan="1" rowspan="1">ATCC 25285</td><td colspan="1" rowspan="1">1.98E+08</td><td colspan="1" rowspan="1">3.00E+09</td></tr><tr><td colspan="1" rowspan="3">Enterobacterales</td><td colspan="1" rowspan="1">Citrobacter freundii</td><td colspan="1" rowspan="1">ATCC 8090</td><td colspan="1" rowspan="1">1.57E+08</td><td colspan="1" rowspan="1">1.07E+09</td></tr><tr><td colspan="1" rowspan="1">Morganella morganii [CTX-M, NDM]</td><td colspan="1" rowspan="1">CDC FDA AR Bank #0057</td><td colspan="1" rowspan="1">6.73E+08</td><td colspan="1" rowspan="1">1.66E+09</td></tr><tr><td colspan="1" rowspan="1">Raoultella ornithinolytica</td><td colspan="1" rowspan="1">ATCC 31898</td><td colspan="1" rowspan="1">1.83E+08</td><td colspan="1" rowspan="1">1.18E+09</td></tr><tr><td colspan="1" rowspan="1">Enterobacter cloacae complex</td><td colspan="1" rowspan="1">Enterobacter cloacae [VIM]</td><td colspan="1" rowspan="1">CDC FDA AR Bank #0154</td><td colspan="1" rowspan="1">1.20E+08</td><td colspan="1" rowspan="1">9.89E+08</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">Escherichia coli [mcr-1]</td><td colspan="1" rowspan="1">CDC FDA AR Bank #0350</td><td colspan="1" rowspan="1">1.02E+08</td><td colspan="1" rowspan="1">1.25E+09</td></tr><tr><td colspan="1" rowspan="1">Haemophilus influenzae</td><td colspan="1" rowspan="1">Haemophilus influenzae</td><td colspan="1" rowspan="1">ATCC 10211</td><td colspan="1" rowspan="1">3.54E+08</td><td colspan="1" rowspan="1">2.60E+08</td></tr><tr><td colspan="1" rowspan="1">Klebsiella aerogenes</td><td colspan="1" rowspan="1">Klebsiella aerogenes [OXA-48-like]</td><td colspan="1" rowspan="1">CDC FDA AR Bank #0074</td><td colspan="1" rowspan="1">3.14E+08</td><td colspan="1" rowspan="1">1.63E+09</td></tr><tr><td colspan="1" rowspan="1">Klebsiella oxytoca</td><td colspan="1" rowspan="1">Klebsiella oxytoca</td><td colspan="1" rowspan="1">ATCC 13182</td><td colspan="1" rowspan="1">2.68E+08</td><td colspan="1" rowspan="1">1.33E+09</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae group</td><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">CDC FDA AR Bank #0097</td><td colspan="1" rowspan="1">1.45E+08</td><td colspan="1" rowspan="1">4.31E+08</td></tr><tr><td colspan="1" rowspan="2">BioFire BCID2 PanelAnalyte</td><td colspan="1" rowspan="2">OrganismAMR Gene]</td><td colspan="1" rowspan="2">Isolate ID</td><td colspan="2" rowspan="1">Concentration a</td></tr><tr><td colspan="1" rowspan="1">PositiveBotleCFU/mML)</td><td colspan="1" rowspan="1">+24h(CFU/mL)</td></tr><tr><td colspan="1" rowspan="1">Neisseria meningitidis</td><td colspan="1" rowspan="1">Neisseria meningitidis</td><td colspan="1" rowspan="1">ATCC 13090</td><td colspan="1" rowspan="1">2.07E+08</td><td colspan="1" rowspan="1">9.30E+07</td></tr><tr><td colspan="1" rowspan="1">Proteus spp.</td><td colspan="1" rowspan="1">Proteus mirabilis [CTX-M]</td><td colspan="1" rowspan="1">GRE 1254053</td><td colspan="1" rowspan="1">1.26E+08</td><td colspan="1" rowspan="1">9.17E+08</td></tr><tr><td colspan="1" rowspan="2">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">Pseudomonas aeruginosa [IMP]</td><td colspan="1" rowspan="1">CDC FDA AR Bank #0092</td><td colspan="1" rowspan="1">9.75E+07</td><td colspan="1" rowspan="1">5.87E+08</td></tr><tr><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">ATCC 10145</td><td colspan="1" rowspan="1">3.16E+08</td><td colspan="1" rowspan="1">9.00E+08</td></tr><tr><td colspan="1" rowspan="1">Salmonella spp.</td><td colspan="1" rowspan="1">Salmonella enterica [CTX-M]</td><td colspan="1" rowspan="1">CDC FDA AR Bank #0407</td><td colspan="1" rowspan="1">2.14E+08</td><td colspan="1" rowspan="1">1.33E+09</td></tr><tr><td colspan="1" rowspan="1">Serratia marcescens</td><td colspan="1" rowspan="1">Serratia marcescens [KPC]</td><td colspan="1" rowspan="1">JMI 697</td><td colspan="1" rowspan="1">9.09E+07</td><td colspan="1" rowspan="1">1.02E+09</td></tr><tr><td colspan="1" rowspan="1">Stenotrophomonas maltophilia</td><td colspan="1" rowspan="1">Stenotrophomonas maltophilia</td><td colspan="1" rowspan="1">ATCC 700475</td><td colspan="1" rowspan="1">2.80E+08</td><td colspan="1" rowspan="1">1.11E+09</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Yeast</td><td></td><td></td><td></td></tr><tr><td colspan="1" rowspan="1">Candida albicans</td><td colspan="1" rowspan="1">Candida albicans</td><td colspan="1" rowspan="1">ATCC 90028</td><td colspan="1" rowspan="1">4.28E+05</td><td colspan="1" rowspan="1">6.79E+06</td></tr><tr><td colspan="1" rowspan="1">Candida auris</td><td colspan="1" rowspan="1">Candida auris</td><td colspan="1" rowspan="1">CDC FDA AR Bank #0381</td><td colspan="1" rowspan="1">2.81E+06</td><td colspan="1" rowspan="1">2.49E+07</td></tr><tr><td colspan="1" rowspan="1">Candida glabrata</td><td colspan="1" rowspan="1">Candida glabrata</td><td colspan="1" rowspan="1">ATCC 15545</td><td colspan="1" rowspan="1">1.15E+06</td><td colspan="1" rowspan="1">3.68E+07</td></tr><tr><td colspan="1" rowspan="1">Candida krusei</td><td colspan="1" rowspan="1">Candida krusei</td><td colspan="1" rowspan="1">ATCC 6258</td><td colspan="1" rowspan="1">9.99E+05</td><td colspan="1" rowspan="1">1.57E+07</td></tr><tr><td colspan="1" rowspan="1">Candida parapsilosis</td><td colspan="1" rowspan="1">Candida parapsilosis</td><td colspan="1" rowspan="1">ATCC 34136</td><td colspan="1" rowspan="1">5.96E+05</td><td colspan="1" rowspan="1">1.39E+07</td></tr><tr><td colspan="1" rowspan="1">Candida tropicalis</td><td colspan="1" rowspan="1">Candida tropicalis</td><td colspan="1" rowspan="1">ATCC 201380</td><td colspan="1" rowspan="1">3.77E+05</td><td colspan="1" rowspan="1">1.19E+07</td></tr><tr><td colspan="1" rowspan="2">Cryptococcusneoformans/gattii</td><td colspan="1" rowspan="1">Cryptococcus gatti</td><td colspan="1" rowspan="1">ATCC MYA-4877</td><td colspan="1" rowspan="1">4.72E+05</td><td colspan="1" rowspan="1">2.83E+06</td></tr><tr><td colspan="1" rowspan="1">Cryptococcus neoformans</td><td colspan="1" rowspan="1">ATCC MYA-4564</td><td colspan="1" rowspan="1">1.54E+06</td><td colspan="1" rowspan="1">9.24E+06</td></tr></table>

a Mean concentration calculated from bottles cultured in the bioMerieux BacT/ALERT VIRTUO and Becton Dickinson BACTEC FX40 systems (VersaTREK bottle data excluded).

# Limit of Detection

A limit of detection (LoD) was established for the bacteria and yeast detected by the BioFire BCID2 Panel. Contrived samples of representative isolates were prepared at a known concentration in a simulated blood culture matrix consisting of human whole blood incubated with blood culture media. LoD was estimated by serial dilution and confirmed by testing at least twenty replicates on the FilmArray 2.0 and FilmArray Torch systems. Confirmation of LoD required detection in at least $9 5 \%$ of replicates tested and the confirmed LoD concentrations are listed in Table 71. Testing also confirmed that each AMR gene can be detected at the LoD concentration of the applicable bacteria with which it may be reported. LoD concentrations are approximately 30 - 200,000-fold lower than the concentrations measured in positive blood cultures.

Table 71. Limit of Detection (LoD) for Analytes Detected by the BioFire BCID2 Panel   

<table><tr><td colspan="1" rowspan="1">BioFire BCID2 Panel Analyte</td><td colspan="1" rowspan="1">Organism [AMR Gene] Tested</td><td colspan="1" rowspan="1">Isolate ID</td><td colspan="1" rowspan="1">LoDConcentration(CF/mL)</td></tr><tr><td></td><td colspan="1" rowspan="1">Gram Positive Bacteria</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Enterococcus faecalis</td><td colspan="1" rowspan="1">Enterococcus faecalis [vanB]</td><td colspan="1" rowspan="1">ATCC 51299</td><td colspan="1" rowspan="1">1.0E+05</td></tr><tr><td colspan="1" rowspan="1">Enterococcus faecium</td><td colspan="1" rowspan="1">Enterococcus faecium [vanA]</td><td colspan="1" rowspan="1">ATCC 7002211</td><td colspan="1" rowspan="1">1.0E+05</td></tr><tr><td colspan="1" rowspan="1">Listeria monocytogenes</td><td colspan="1" rowspan="1">Listeria monocytogenes</td><td colspan="1" rowspan="1">ATCC 15313</td><td colspan="1" rowspan="1">1.0E+04</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus spp.</td><td colspan="1" rowspan="1">Staphylococcus hominis</td><td colspan="1" rowspan="1">ATCC 25615</td><td colspan="1" rowspan="1">1.0E+05</td></tr><tr><td colspan="1" rowspan="2">Staphylococcus aureus</td><td colspan="1" rowspan="1">Staphylococcus aureus [mecA]</td><td colspan="1" rowspan="1">ATCC BAA-38</td><td colspan="1" rowspan="2">1.0E+04a</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus [mecC]</td><td colspan="1" rowspan="1">ATCC BAA-2313</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus epidermidis</td><td colspan="1" rowspan="1">Staphylococcus epidermidis [mecA]</td><td colspan="1" rowspan="1">ATCC 35984</td><td colspan="1" rowspan="1">1.0E+05</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus lugdunensis</td><td colspan="1" rowspan="1">Staphylococcus lugdunensis</td><td colspan="1" rowspan="1">ATCC 43809</td><td colspan="1" rowspan="1">1.0E+04</td></tr><tr><td colspan="1" rowspan="1">Streptococcus spp.</td><td colspan="1" rowspan="1">Streptococcus mitis</td><td colspan="1" rowspan="1">ATCC 49456</td><td colspan="1" rowspan="1">1.0E+04</td></tr><tr><td colspan="1" rowspan="1">Streptococcus agalactiae</td><td colspan="1" rowspan="1">Streptococcus agalactiae</td><td colspan="1" rowspan="1">ATCC 13813</td><td colspan="1" rowspan="1">5.0E+04</td></tr><tr><td colspan="1" rowspan="1">Streptococcus pneumoniae</td><td colspan="1" rowspan="1">Streptococcus pneumoniae</td><td colspan="1" rowspan="1">ATCC 6303</td><td colspan="1" rowspan="1">5.0E+03</td></tr><tr><td colspan="1" rowspan="1">Streptococcus pyogenes</td><td colspan="1" rowspan="1">Streptococcus pyogenes</td><td colspan="1" rowspan="1">ATCC 49399</td><td colspan="1" rowspan="1">5.0E+03</td></tr><tr><td></td><td colspan="1" rowspan="1">Gram Negative Bacteria</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Acinetobacter calcloaceticus-baumanniicomplex</td><td colspan="1" rowspan="1">Acinetobacter baumannii [NDM]</td><td colspan="1" rowspan="1">CDC-FDA AR Bank #0033</td><td colspan="1" rowspan="1">5.0E+03</td></tr><tr><td colspan="1" rowspan="1">Bacteroides fragilis</td><td colspan="1" rowspan="1">Bacteroides fragilis</td><td colspan="1" rowspan="1">ATCC 25285</td><td colspan="1" rowspan="1">1.0E+04</td></tr><tr><td colspan="1" rowspan="3">Enterobacterales</td><td colspan="1" rowspan="1">Citrobacter freundii</td><td colspan="1" rowspan="1">ATCC 8090</td><td colspan="1" rowspan="3">1.0E+05</td></tr><tr><td colspan="1" rowspan="1">Morganella morganii</td><td colspan="1" rowspan="1">ATCC 25830</td></tr><tr><td colspan="1" rowspan="1">Serratia plymuthica</td><td colspan="1" rowspan="1">ATCC 183</td></tr><tr><td colspan="1" rowspan="1">Enterobacter cloacae complex</td><td colspan="1" rowspan="1">Enterobacter cloacae [VIM]</td><td colspan="1" rowspan="1">CDC-FDA AR Bank #0154</td><td colspan="1" rowspan="1">1.0E+05</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">Escherichia coli [mcr-1]</td><td colspan="1" rowspan="1">CDC-FDA AR Bank #0350</td><td colspan="1" rowspan="1">5.0E+05</td></tr><tr><td colspan="1" rowspan="1">Haemophilus influenzae</td><td colspan="1" rowspan="1">Haemophilus influenzae</td><td colspan="1" rowspan="1">ATCC 10211</td><td colspan="1" rowspan="1">1.0E+04</td></tr><tr><td colspan="1" rowspan="1">Klebsiella aerogenes</td><td colspan="1" rowspan="1">Klebsiella aerogenes [OXA-48-like]</td><td colspan="1" rowspan="1">CDC-FDA AR Bank #0074</td><td colspan="1" rowspan="1">1.0E+05</td></tr><tr><td colspan="1" rowspan="1">Klebsiella oxytoca</td><td colspan="1" rowspan="1">Klebsiella oxytoca [CTX-M]</td><td colspan="1" rowspan="1">GRE 1254054</td><td colspan="1" rowspan="1">1.0E+05</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae group</td><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">ATCC 13883</td><td colspan="1" rowspan="1">5.0E+04</td></tr><tr><td colspan="1" rowspan="1">Neisseria meningitidis</td><td colspan="1" rowspan="1">Neisseria meningitidis</td><td colspan="1" rowspan="1">ATCC 13090</td><td colspan="1" rowspan="1">1.0E+03</td></tr><tr><td colspan="1" rowspan="1">BioFire BCID2 Panel Analyte</td><td colspan="1" rowspan="1">Organism [AMR Gene] Tested</td><td colspan="1" rowspan="1">Isolate ID</td><td colspan="1" rowspan="1">LoDConcentration(CFU/mL)</td></tr><tr><td colspan="1" rowspan="1">Proteus spp.</td><td colspan="1" rowspan="1">Proteus mirabilis</td><td colspan="1" rowspan="1">ATCC 29906</td><td colspan="1" rowspan="1">5.0E+05</td></tr><tr><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">Pseudomonas aeruginosa [IMP]</td><td colspan="1" rowspan="1">CDC-FDA AR Bank #0092</td><td colspan="1" rowspan="1">1.0E+04</td></tr><tr><td colspan="1" rowspan="1">Salmonella spp.</td><td colspan="1" rowspan="1">Salmonella enterica</td><td colspan="1" rowspan="1">ATCC 700720</td><td colspan="1" rowspan="1">5.0E+04</td></tr><tr><td colspan="1" rowspan="1">Serratia marcescens</td><td colspan="1" rowspan="1">Serratia marcescens [KPC]</td><td colspan="1" rowspan="1">JMI 697</td><td colspan="1" rowspan="1">1.0E+05</td></tr><tr><td colspan="1" rowspan="1">Stenotrophomonas maltophilia</td><td colspan="1" rowspan="1">Stenotrophomonas maltophilia</td><td colspan="1" rowspan="1">ATCC 700475</td><td colspan="1" rowspan="1">1.0E+06</td></tr><tr><td></td><td colspan="1" rowspan="1">Yeast</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Candida albicans</td><td colspan="1" rowspan="1">Candida albicans</td><td colspan="1" rowspan="1">ATCC 90028</td><td colspan="1" rowspan="1">1.0E+03</td></tr><tr><td colspan="1" rowspan="1">Candida auris</td><td colspan="1" rowspan="1">Candida auris</td><td colspan="1" rowspan="1">CDC-FDA AR Bank #0381</td><td colspan="1" rowspan="1">1.0E+03</td></tr><tr><td colspan="1" rowspan="1">Candida glabrata</td><td colspan="1" rowspan="1">Candida glabrata</td><td colspan="1" rowspan="1">ATCC 15545</td><td colspan="1" rowspan="1">1.0E+02</td></tr><tr><td colspan="1" rowspan="1">Candida krusei</td><td colspan="1" rowspan="1">Candida krusei</td><td colspan="1" rowspan="1">ATCC 28870</td><td colspan="1" rowspan="1">5.0E+03</td></tr><tr><td colspan="1" rowspan="1">Candida parapsilosis</td><td colspan="1" rowspan="1">Candida parapsilosis</td><td colspan="1" rowspan="1">ATCC 34136</td><td colspan="1" rowspan="1">1.0E+04</td></tr><tr><td colspan="1" rowspan="1">Candida tropicalis</td><td colspan="1" rowspan="1">Candida tropicalis</td><td colspan="1" rowspan="1">ATCC 201380</td><td colspan="1" rowspan="1">1.0E+04</td></tr><tr><td colspan="1" rowspan="2">Cryptococcus neoformans/gattii</td><td colspan="1" rowspan="1">Cryptococcus gattii</td><td colspan="1" rowspan="1">ATCC MYA-4877</td><td colspan="1" rowspan="2">5.0E+02</td></tr><tr><td colspan="1" rowspan="1">Cryptococcus neoformans</td><td colspan="1" rowspan="1">ATCC MYA-4564</td></tr></table>

a Confirmed LoD concentration for Staphylococcus aureus is the higher of the two LoD concentrations observed.

# Analytical Reactivity (Inclusivity)

The analytical reactivity of BioFire BCID2 Panel assays was assessed via a combination of in silico analysis of sequences available in public databases and testing of over 450 isolates representing the genetic, geographic, and temporal diversity of species, subspecies, and AMR gene types detected by the panel. Isolates were tested in triplicate at concentrations near LoD in simulated blood culture matrix.

Results for each isolate tested as well as in silico reactivity predictions for species or AMR gene types that were not tested are shown in Table 72 – Table 83. For isolates that were not detected at the initial near-LoD concentration, additional testing was performed at higher concentrations and the approximate concentration where detection was observed is indicated. In most cases, the detected concentration was equal to or less than the concentration expected in a positive blood culture. Alternately, a Not Detected result is indicated if the isolate was not detected at a concentration equivalent to a positive blood culture level. Additional limitations on reactivity predicted by in silico sequence analysis are noted.

Table 72. Results for Enterococcus faecalis Isolates Tested   

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Source ID</td><td rowspan=1 colspan=1>Strain/Location/Year</td><td rowspan=1 colspan=1>Result</td></tr><tr><td rowspan=7 colspan=1>Enterococcusfaecalis</td><td rowspan=1 colspan=1>ATCC 19433</td><td rowspan=1 colspan=1>Type Strain</td><td rowspan=7 colspan=1>EnterococcusfaecalisDetected</td></tr><tr><td rowspan=1 colspan=1>ATCC 29212</td><td rowspan=1 colspan=1>Portland</td></tr><tr><td rowspan=1 colspan=1>ATCC 49533</td><td rowspan=1 colspan=1>UWH/1936</td></tr><tr><td rowspan=1 colspan=1>ATCC 51299</td><td rowspan=1 colspan=1>NJ-3</td></tr><tr><td rowspan=1 colspan=1>ATCC 700802</td><td rowspan=1 colspan=1>V583</td></tr><tr><td rowspan=1 colspan=1>ATCC BAA-2573</td><td rowspan=1 colspan=1>bMx 0502240</td></tr><tr><td rowspan=1 colspan=1>JMI 12536</td><td rowspan=1 colspan=1>MA/2002</td></tr></table>

Table 73. Results for Enterococcus faecium Isolates Tested   

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Source ID</td><td rowspan=1 colspan=1>Strain/Location/Year</td><td rowspan=1 colspan=1>Result</td></tr><tr><td rowspan=6 colspan=1>Enterococcusfaecium</td><td rowspan=1 colspan=1>ATCC 19434</td><td rowspan=1 colspan=1>Type Strain</td><td rowspan=6 colspan=1>EnterococcusfaeciumDetected</td></tr><tr><td rowspan=1 colspan=1>ATCC 27270</td><td rowspan=1 colspan=1>X3 [F]</td></tr><tr><td rowspan=1 colspan=1>ATCC 51858</td><td rowspan=1 colspan=1>Vancomycin-dependent #4</td></tr><tr><td rowspan=1 colspan=1>ATCC 700221</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>ATCC BAA-2318</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>JMI 475</td><td rowspan=1 colspan=1>IN/2003</td></tr></table>

Table 74. Results for Listeria monocytogenes Isolates Tested   

<table><tr><td colspan="1" rowspan="1">Organism</td><td colspan="1" rowspan="1">Serotypea</td><td colspan="1" rowspan="1">Source ID</td><td colspan="1" rowspan="1">Strain/Location/Year</td><td colspan="1" rowspan="1">Result</td></tr><tr><td colspan="1" rowspan="4">Listeriamonocytogenes</td><td colspan="1" rowspan="2">1/2a</td><td colspan="1" rowspan="1">ATCC 15313</td><td colspan="1" rowspan="1">Type StrainUnited Kingdom/1924</td><td colspan="1" rowspan="4">ListeriamonocytogenesDetected</td></tr><tr><td colspan="1" rowspan="1">ATCC 19111</td><td colspan="1" rowspan="1">Li 20United Kingdom</td></tr><tr><td colspan="1" rowspan="1">1/2b</td><td colspan="1" rowspan="1">ATCC BAA-751</td><td colspan="1" rowspan="1">NSB 22072</td></tr><tr><td colspan="1" rowspan="1">4b</td><td colspan="1" rowspan="1">ATCC 13932</td><td colspan="1" rowspan="1">1071/53Germany</td></tr><tr><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">ATCC 43256</td><td colspan="1" rowspan="1">CDC F2380</td><td colspan="1" rowspan="2"></td></tr><tr><td colspan="1" rowspan="1">7</td><td colspan="1" rowspan="1">NCTC 10890</td><td colspan="1" rowspan="1">Li 2482Germany</td></tr></table>

a Assay reactivity is not serotype dependent, the assay will react with all serotypes (1/2a, 1/2b, 1/2c, 3a, 3b, 3c, 4a, 4b, 4c, 4d, 4e, and 7)

# Note: The BioFire BCID2 Panel may be able to detect strains of live, attenuated Listeria monocytogenes vaccines used in cancer immunotherapy.

<table><tr><td colspan="5" rowspan="1">Table75. Results or Staphylococcus sp. IsolatesTested and Predicted Reactivity for Species Not Tested</td></tr><tr><td colspan="2" rowspan="1">Organism</td><td colspan="1" rowspan="1">Source ID</td><td colspan="1" rowspan="1">Strain/Location/Year</td><td colspan="1" rowspan="1">Result</td></tr><tr><td colspan="2" rowspan="1">Staphylococcus argensis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">In silico prediction (not tested)</td><td colspan="1" rowspan="43">Staphylococcus spp.Detected</td></tr><tr><td colspan="2" rowspan="1">Staphylococcus arlettae</td><td colspan="1" rowspan="1">ATCC 43957</td><td colspan="1" rowspan="1">Type strain</td></tr><tr><td colspan="2" rowspan="1">Staphylococcus auricularis</td><td colspan="1" rowspan="1">ATCC 33753</td><td colspan="1" rowspan="1">Type strain</td></tr><tr><td colspan="1" rowspan="2">Staphylococcus capitis</td><td colspan="1" rowspan="1">ssp. capitis</td><td colspan="1" rowspan="1">ATCC 27842</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">ssp. urealyticus</td><td colspan="1" rowspan="1">ATCC 49326</td><td colspan="1" rowspan="1">Type strain</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus caprae</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">ATCC 55133</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus carnosus</td><td colspan="1" rowspan="1">ssp. carnosus</td><td colspan="1" rowspan="1">ATCC 51365</td><td colspan="1" rowspan="1">Type strain983</td></tr><tr><td colspan="1" rowspan="2">Staphylococcus cohnii</td><td colspan="1" rowspan="1">ssp. cohnii</td><td colspan="1" rowspan="1">ATCC 29972</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">ssp. urealyticus</td><td colspan="1" rowspan="1">ATCC 49330</td><td colspan="1" rowspan="1">Type strain</td></tr><tr><td colspan="2" rowspan="1">Staphylococcus condimenti</td><td colspan="1" rowspan="1">CCUG 39902T</td><td colspan="1" rowspan="1">Type strainJapan</td></tr><tr><td colspan="2" rowspan="1">Staphylococcus cornubiensis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="2" rowspan="1">Staphylococcus delphini</td><td colspan="1" rowspan="1">ATCC 49171</td><td colspan="1" rowspan="1">Type strainItaly</td></tr><tr><td colspan="2" rowspan="1">Staphylococcus devriesei</td><td colspan="1" rowspan="1">CCUG 58238T</td><td colspan="1" rowspan="1">Type StrainBelgium</td></tr><tr><td colspan="2" rowspan="1">Staphylococcus edaphicus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">In silico prediction (not tested)</td></tr><tr><td colspan="2" rowspan="1">Staphylococcus epidermidis</td><td colspan="1" rowspan="1">ATCC 35984</td><td colspan="1" rowspan="1">Tennessee</td></tr><tr><td colspan="2" rowspan="1">Staphylococcus felis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="2" rowspan="1">Staphylococcus gallinarum</td><td colspan="1" rowspan="1">ATCC 700401</td><td colspan="1" rowspan="1">France</td></tr><tr><td colspan="2" rowspan="1">Staphylococcus haemolyticus</td><td colspan="1" rowspan="1">ATCC 29968</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="2">Staphylococcus hominis</td><td colspan="1" rowspan="1">ssp. hominis</td><td colspan="1" rowspan="1">ATCC 25615</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">ssp. novobiosepticus</td><td colspan="1" rowspan="1">ATCC 700236</td><td colspan="1" rowspan="1">Type strainNew Jersey1992</td></tr><tr><td colspan="2" rowspan="1">Staphylococcus hyicus</td><td colspan="1" rowspan="1">ATCC 11249</td><td colspan="1" rowspan="1">Type strain</td></tr><tr><td colspan="2" rowspan="1">Staphylococcus intermedius</td><td colspan="1" rowspan="1">ATCC 29663</td><td colspan="1" rowspan="1">Type strain</td></tr><tr><td colspan="2" rowspan="1">Staphylococcus kloosii</td><td colspan="1" rowspan="1">ATCC 43959</td><td colspan="1" rowspan="1">Type strainUSA</td></tr><tr><td colspan="2" rowspan="1">Staphylococcus lugdunensis</td><td colspan="1" rowspan="1">ATCC 43809</td><td colspan="1" rowspan="1">Type strainFFrance</td></tr><tr><td colspan="2" rowspan="1">Staphylococcus lutrae</td><td colspan="1" rowspan="1">ATCC 700373</td><td colspan="1" rowspan="1">Type strain</td></tr><tr><td colspan="2" rowspan="1">Staphylococcus massiliensis</td><td colspan="1" rowspan="1">CCUG 55927T</td><td colspan="1" rowspan="1">Type strainFrace 2005</td></tr><tr><td colspan="2" rowspan="1">Staphylococcus microti</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">In silico prediction (not tested)</td></tr><tr><td colspan="2" rowspan="1">Staphylococcus nepalensis</td><td colspan="1" rowspan="1">CCUG 66326</td><td colspan="1" rowspan="1">Sweden 2014</td></tr><tr><td colspan="2" rowspan="1">Staphylococcus pasteuri</td><td colspan="1" rowspan="1">ATCC 51127</td><td colspan="1" rowspan="1">France</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus petrasii</td><td colspan="1" rowspan="1">ssp. jettensis</td><td colspan="1" rowspan="1">CCUG 62657T</td><td colspan="1" rowspan="1">Type strainBelgium</td></tr><tr><td colspan="2" rowspan="1">Staphylococcus pettenkoferi</td><td colspan="1" rowspan="1">CCUG 70234</td><td colspan="1" rowspan="1">Sweden 2017</td></tr><tr><td colspan="2" rowspan="1">Staphylococcus piscifermentans</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">In silico prediction (not tested)</td></tr><tr><td colspan="2" rowspan="1">Staphylococcus pseudintermedius</td><td colspan="1" rowspan="1">ATCC 49444</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="2" rowspan="1">Staphylococcus pseudolugdunensis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">In silico prediction (not tested)</td></tr><tr><td colspan="2" rowspan="1">Staphylococcus saccharolyticus</td><td colspan="1" rowspan="1">ATCC 14953</td><td colspan="1" rowspan="1">Type strain</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus saprophyticus</td><td colspan="1" rowspan="1">ssp. saprophyticus</td><td colspan="1" rowspan="1">ATCC 15305</td><td colspan="1" rowspan="1">Type strain1935</td></tr><tr><td colspan="1" rowspan="2">Staphylococcus schleiferi</td><td colspan="1" rowspan="1">ssp. coagulans</td><td colspan="1" rowspan="1">ATCC 49545</td><td colspan="1" rowspan="1">Type strain</td></tr><tr><td colspan="1" rowspan="1">ssp. schleiferi</td><td colspan="1" rowspan="1">ATCC 43808</td><td colspan="1" rowspan="1">Type strainFrance</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus sciuri</td><td colspan="1" rowspan="1">ssp. sciuri</td><td colspan="1" rowspan="1">ATCC 29061</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="2" rowspan="1">Staphylococcus simiae</td><td colspan="1" rowspan="1">GRE 1562010</td><td colspan="1" rowspan="1">Type strainCzech Republic</td></tr><tr><td colspan="2" rowspan="1">Staphylococcus simulans</td><td colspan="1" rowspan="1">ATCC 27848</td><td colspan="1" rowspan="1">Type strain</td></tr><tr><td colspan="2" rowspan="1">Staphylococcus stepanovicii</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="2" rowspan="1">Staphylococcus warneri</td><td colspan="1" rowspan="1">ATCC 25614</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="2" rowspan="1">Organism</td><td colspan="1" rowspan="1">Source ID</td><td colspan="1" rowspan="1">Strain/Location/Year</td><td colspan="1" rowspan="1">Result</td></tr><tr><td colspan="2" rowspan="1">Staphylococcus xylosus</td><td colspan="1" rowspan="1">ATCC 29966</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="2" rowspan="1">Staphylococcus agnetis</td><td colspan="2" rowspan="1">In silico prediction (not tested)</td><td colspan="1" rowspan="8">Staphylococcus spp.Detected(5.9E+06 CFU/mL)</td></tr><tr><td colspan="2" rowspan="1">Staphylococcus argenteusa</td><td colspan="1" rowspan="1">DSM 28299</td><td colspan="1" rowspan="1">Type strainAustralia/2006</td></tr><tr><td colspan="2" rowspan="1">Staphylococcus aureusa</td><td colspan="1" rowspan="1">Multiple isolates</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="2" rowspan="1">Staphylococcus chromogenes</td><td colspan="1" rowspan="1">ATCC 43764</td><td colspan="1" rowspan="1">Type strain</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus sciuri</td><td colspan="1" rowspan="1">ssp. rodentium</td><td colspan="2" rowspan="1">In silico prediction (not tested)</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus succinus</td><td colspan="1" rowspan="1">ssp. succinus</td><td colspan="1" rowspan="1">ATCC 700337</td><td colspan="1" rowspan="1">Type strainDominican Republic</td></tr><tr><td colspan="2" rowspan="1">Staphylococcus schweitzeri</td><td colspan="1" rowspan="1">DSM 28300</td><td colspan="1" rowspan="1">Type strainG abon/2010</td></tr><tr><td colspan="2" rowspan="1">Staphylococcus vitulinus</td><td colspan="1" rowspan="1">ATCC 51145</td><td colspan="1" rowspan="1">Type strain</td></tr><tr><td colspan="2" rowspan="1">Staphylococcus equorum</td><td colspan="1" rowspan="1">ATCC 43958</td><td colspan="1" rowspan="1">Type strainBelgium</td><td colspan="1" rowspan="5">Not Detected</td></tr><tr><td colspan="2" rowspan="1">Staphylococcus fleurettii</td><td colspan="1" rowspan="1">DSM 20047</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="2" rowspan="1">Staphylococcus lentus</td><td colspan="1" rowspan="1">ATCC 29070</td><td colspan="1" rowspan="1">Type strainFrance</td></tr><tr><td colspan="2" rowspan="1">Staphylococcus muscae</td><td colspan="2" rowspan="2">In silico prediction (not tested)</td></tr><tr><td colspan="2" rowspan="1">Staphylococcus rostri</td></tr></table>

a Also amplified by the Saureus assay at lower concentrations. Will be reported as Staphylococcus spp. Detected and Staphylococcus aureus Detected

Table 76. Results for Staphylococcus aureus Isolates Tested   

<table><tr><td rowspan=1 colspan=4>Organism</td><td rowspan=1 colspan=1>Source ID a</td><td rowspan=1 colspan=1>Strain/Location/Year</td><td rowspan=1 colspan=1>PFGE TypelPVL (if known)</td><td rowspan=1 colspan=1>Result</td></tr><tr><td rowspan=6 colspan=2>Staphylococcusaureus</td><td rowspan=1 colspan=2>ssp. anaerobius</td><td rowspan=1 colspan=1>ATCC 35844</td><td rowspan=1 colspan=1>MVF-7/Spain</td><td rowspan=1 colspan=1>unknown</td><td rowspan=30 colspan=1>StaphylococcusaureusDetected</td></tr><tr><td rowspan=5 colspan=2>ssp. aureus</td><td rowspan=1 colspan=1>ATCC 10832</td><td rowspan=1 colspan=1>Wood 46</td><td rowspan=1 colspan=1>unknown</td></tr><tr><td rowspan=1 colspan=1>ATCC 12600</td><td rowspan=1 colspan=1>Type strain1935</td><td rowspan=1 colspan=1>unknown</td></tr><tr><td rowspan=1 colspan=1>ATCC 14154</td><td rowspan=1 colspan=1>Rose</td><td rowspan=1 colspan=1>unknown</td></tr><tr><td rowspan=1 colspan=1>ATCC 25923</td><td rowspan=1 colspan=1>Seattle/1945</td><td rowspan=1 colspan=1>unknown</td></tr><tr><td rowspan=1 colspan=1>ATCC 43300</td><td rowspan=1 colspan=1>F182/Kansas</td><td rowspan=1 colspan=1>unknown</td></tr><tr><td rowspan=6 colspan=4></td><td rowspan=1 colspan=1>NARSA NRS705</td><td rowspan=1 colspan=1>NY-12</td><td rowspan=1 colspan=1>USA 100</td></tr><tr><td rowspan=1 colspan=1>ATCC BAA-41</td><td rowspan=1 colspan=1>New York/1994</td><td rowspan=1 colspan=1>USA 100/PVL-</td></tr><tr><td rowspan=1 colspan=1>NARSA NRS701</td><td rowspan=1 colspan=1>MN-082</td><td rowspan=1 colspan=1>USA 200</td></tr><tr><td rowspan=1 colspan=1>ATCC BAA-1720</td><td rowspan=1 colspan=1>MRSA252United Kingdom</td><td rowspan=1 colspan=1>USA 200</td></tr><tr><td rowspan=1 colspan=1>ATCC BAA-1717</td><td rowspan=1 colspan=1>TCH1516/Texas</td><td rowspan=1 colspan=1>USA 300</td></tr><tr><td rowspan=1 colspan=1>NARSA NRS683</td><td rowspan=1 colspan=1>GA-298Georgia/2005</td><td rowspan=1 colspan=1>USA 300/PVL+</td></tr><tr><td rowspan=18 colspan=4>Staphylococcus aureus</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>NARSA NRS662</td><td rowspan=1 colspan=1>CO-34</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>NARSA NRS707</td><td rowspan=1 colspan=1>NY-155New York/2005</td><td rowspan=1 colspan=1>USA 300/PVL+</td></tr><tr><td rowspan=1 colspan=1>ATCC BAA-1707</td><td rowspan=1 colspan=1>MW2North Dakota/1998</td><td rowspan=1 colspan=1>USA 400</td></tr><tr><td rowspan=1 colspan=1>NARSA NRS691</td><td rowspan=1 colspan=1>GA-62</td><td rowspan=1 colspan=1>USA 500</td></tr><tr><td rowspan=1 colspan=1>NARSA NRS385</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>USA 500</td></tr><tr><td rowspan=1 colspan=1>NARSA NRS648</td><td rowspan=1 colspan=1>CA-347</td><td rowspan=1 colspan=1>USA 600</td></tr><tr><td rowspan=1 colspan=1>NARSA NRS689</td><td rowspan=1 colspan=1>GA-442</td><td rowspan=1 colspan=1>USA 700</td></tr><tr><td rowspan=1 colspan=1>NARSA NRS668</td><td rowspan=1 colspan=1>CO-72Colorado/2005</td><td rowspan=1 colspan=1>USA 800</td></tr><tr><td rowspan=1 colspan=1>ATCC BAA-42</td><td rowspan=1 colspan=1>HDE288Portugal/1996</td><td rowspan=1 colspan=1>USA 800</td></tr><tr><td rowspan=1 colspan=1>ATCC BAA-1749</td><td rowspan=1 colspan=1>96:308</td><td rowspan=1 colspan=1>USA 900</td></tr><tr><td rowspan=1 colspan=1>ATCC BAA-1759</td><td rowspan=1 colspan=1>N7129</td><td rowspan=1 colspan=1>USA 900</td></tr><tr><td rowspan=1 colspan=1>NARSA NRS745</td><td rowspan=1 colspan=1>CA-629</td><td rowspan=1 colspan=1>USA 1000</td></tr><tr><td rowspan=1 colspan=1>BEI NR-46081</td><td rowspan=1 colspan=1>HIP 12899</td><td rowspan=1 colspan=1>USA 1100/PVL+</td></tr><tr><td rowspan=1 colspan=1>ATCC BAA-1765</td><td rowspan=1 colspan=1>102-04</td><td rowspan=1 colspan=1>USA 1200</td></tr><tr><td rowspan=1 colspan=1>ATCC BAA-1700</td><td rowspan=1 colspan=1>HFH-33798Illinois/2004</td><td rowspan=1 colspan=1>Not USA 100-1100</td></tr><tr><td rowspan=1 colspan=1>ATCC BAA-1691</td><td rowspan=1 colspan=1>HFH-30137Michigan/2003</td><td rowspan=1 colspan=1>Not USA 100-1100</td></tr><tr><td rowspan=1 colspan=1>ATCC 29213</td><td rowspan=1 colspan=1>Wichita</td><td rowspan=1 colspan=1>unknown</td></tr><tr><td rowspan=1 colspan=1>ATCC BAA-38</td><td rowspan=1 colspan=1>E2125/Denmark</td><td rowspan=1 colspan=1>unknown</td></tr></table>

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Source ID a</td><td rowspan=1 colspan=1>Strain/Location/Year</td><td rowspan=1 colspan=1>PFGE Type/PVL (if known)</td><td rowspan=1 colspan=1>Result</td></tr><tr><td rowspan=22 colspan=1>′</td><td rowspan=1 colspan=1>ATCC BAA-39</td><td rowspan=1 colspan=1>HUSA304Hungary/1993</td><td rowspan=1 colspan=1>unknown</td><td rowspan=21 colspan=1></td></tr><tr><td rowspan=1 colspan=1>ATCC BAA-40</td><td rowspan=1 colspan=1>CPS22Portugal/1994</td><td rowspan=1 colspan=1>unknown</td></tr><tr><td rowspan=1 colspan=1>ATCC BAA-44</td><td rowspan=1 colspan=1>HPV107Portugal/1996</td><td rowspan=1 colspan=1>PVL-</td></tr><tr><td rowspan=1 colspan=1>ATCC BAA-2312</td><td rowspan=1 colspan=1>M10/0061Ireland/2010</td><td rowspan=1 colspan=1>unknown</td></tr><tr><td rowspan=1 colspan=1>ATCC BAA-2313</td><td rowspan=1 colspan=1>M10/0148Ireland/2010</td><td rowspan=1 colspan=1>unknown</td></tr><tr><td rowspan=1 colspan=1>ATCC BAA-2421</td><td rowspan=1 colspan=1>Massachusetts/2010</td><td rowspan=1 colspan=1>unknown</td></tr><tr><td rowspan=1 colspan=1>ATCC BAA-2422</td><td rowspan=1 colspan=1>Massachusetts/2010</td><td rowspan=1 colspan=1>unknown</td></tr><tr><td rowspan=1 colspan=1>GRE 0759084</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>unknown</td></tr><tr><td rowspan=1 colspan=1>GRE 1055015</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>unknown</td></tr><tr><td rowspan=1 colspan=1>GRE 0860042</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>unknown</td></tr><tr><td rowspan=1 colspan=1>GRE 1052034</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>unknown</td></tr><tr><td rowspan=1 colspan=1>GRE 1151100</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>unknown</td></tr><tr><td rowspan=1 colspan=1>GRE 0960006</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>unknown</td></tr><tr><td rowspan=1 colspan=1>GRE 1055017</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>unknown</td></tr><tr><td rowspan=1 colspan=1>GRE 0759163</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>unknown</td></tr><tr><td rowspan=1 colspan=1>GRE 1062373</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>unknown</td></tr><tr><td rowspan=1 colspan=1>GRE 1057114</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>unknown</td></tr><tr><td rowspan=1 colspan=1>GRE 1062292</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>unknown</td></tr><tr><td rowspan=1 colspan=1>NARSA NRS686</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>unknown</td></tr><tr><td rowspan=1 colspan=1>Rennes 1060728</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>unknown</td></tr><tr><td rowspan=1 colspan=1>Sunnybrook SUN1</td><td rowspan=1 colspan=1>Toronto</td><td rowspan=1 colspan=1>unknown</td></tr><tr><td rowspan=1 colspan=1>GRE 1062264b</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>unknown</td><td rowspan=1 colspan=1>StaphylococcusaureusDetected(≥6.3E+05 CFU/mL)</td></tr></table>

a NARSA and BEI isolates sourced by the Network on Antimicrobial Resistance in Staphylococcus aureus (NARSA) for distribution by BEI Resources, NIAID, NIH b Isolate from private collection with variant sequence under assay primer(s). Similar variant sequences represent ${ \sim } 1 \%$ of over 10,000 S. aureus sequences evaluated

Table 77. Results for Staphylococcus epidermidis Isolates Tested   

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Source ID</td><td rowspan=1 colspan=1>Strain/Location/Year</td><td rowspan=1 colspan=1>Result</td></tr><tr><td rowspan=6 colspan=1>Staphylococcusepidermidis</td><td rowspan=1 colspan=1>ATCC 35984</td><td rowspan=1 colspan=1>RP62ATennesee</td><td rowspan=6 colspan=1>StaphylococcuseidermidisDetected</td></tr><tr><td rowspan=1 colspan=1>ATCC 12228</td><td rowspan=1 colspan=1>FDA strain/PCI 1200</td></tr><tr><td rowspan=1 colspan=1>ATCC 29887</td><td rowspan=1 colspan=1>255-01B</td></tr><tr><td rowspan=1 colspan=1>ATCC 35983</td><td rowspan=1 colspan=1>RP12Tennesee</td></tr><tr><td rowspan=1 colspan=1>ATCC 51625</td><td rowspan=1 colspan=1>CCF 15990Ohio</td></tr><tr><td rowspan=1 colspan=1>ATCC 700562</td><td rowspan=1 colspan=1>1191Virginia/1997</td></tr></table>

Table 78. Results for Staphylococcus lugdunensis Isolates Tested   

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Source ID</td><td rowspan=1 colspan=1>Strain/Location/Year</td><td rowspan=1 colspan=1>Result</td></tr><tr><td rowspan=5 colspan=1>Staphylococcuslugdunensis</td><td rowspan=1 colspan=1>ATCC 43809</td><td rowspan=1 colspan=1>Type strainFFrance</td><td rowspan=5 colspan=1>StaphylococcuslugdunensisDetected</td></tr><tr><td rowspan=1 colspan=1>NCTC 7990</td><td rowspan=1 colspan=1>KellyUnited Kingdom/1949</td></tr><tr><td rowspan=1 colspan=1>ATCC 49576</td><td rowspan=1 colspan=1>LRA/260.05.79</td></tr><tr><td rowspan=1 colspan=1>ATCC 700328</td><td rowspan=1 colspan=1>6733</td></tr><tr><td rowspan=1 colspan=1>ATCC 700582</td><td rowspan=1 colspan=1>7829Virginia/1997</td></tr></table>

Table 79. Results for Streptococcus spp. Isolates Tested and Predicted Reactivity for Species Not Tested   

<table><tr><td>Organism</td><td>Source ID</td><td>Strain/Location/Year</td><td>Result</td></tr><tr><td>sacidominimus</td><td colspan="2"></td><td>Streptococcus</td></tr></table>

<table><tr><td rowspan=1 colspan=3>Streptococcus agalactiae</td><td rowspan=1 colspan=1>ATCC 13813</td><td rowspan=1 colspan=1>Type Strain</td></tr><tr><td rowspan=1 colspan=3>Streptococcus anginosus</td><td rowspan=1 colspan=1>ATCC 33397</td><td rowspan=1 colspan=1>Havil</td></tr><tr><td rowspan=1 colspan=3>Streptococcus australis</td><td rowspan=1 colspan=1>ATCC 700641</td><td rowspan=1 colspan=1>Type strainAustralia/1987</td></tr><tr><td rowspan=1 colspan=3>Streptococcus azizii</td><td rowspan=1 colspan=2>In silico prediction (not tested)</td></tr><tr><td rowspan=1 colspan=3>Streptococcus bovimastitidis</td><td rowspan=1 colspan=2>In silico prediction (not tested)</td></tr><tr><td rowspan=1 colspan=3>Streptococcus bovis</td><td rowspan=1 colspan=1>ATCC 33317</td><td rowspan=1 colspan=1>Pearl 11</td></tr><tr><td rowspan=1 colspan=3>Streptococcus caballi</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>In lico icion t </td></tr><tr><td rowspan=1 colspan=3>Streptococcus canis</td><td rowspan=1 colspan=1>ATcC 43496</td><td rowspan=1 colspan=1>Type strainBelgi/1982</td></tr><tr><td rowspan=1 colspan=3>Streptococcus castoreus</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>In silico prediction (not tested)</td></tr><tr><td rowspan=1 colspan=3>Streptococcus constellatus</td><td rowspan=1 colspan=1>ATCC 27513</td><td rowspan=1 colspan=1>VPI7712</td></tr><tr><td rowspan=1 colspan=3>Streptococcus criceti</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>In silico prediction (not tested)</td></tr><tr><td rowspan=1 colspan=3>Streptococcus cristatus</td><td rowspan=1 colspan=1>ATCC 51100</td><td rowspan=1 colspan=1>Type strainUnited Kingdom</td></tr><tr><td rowspan=1 colspan=3>Streptococcus cuniculi</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>In silico prediction (not tested)</td></tr><tr><td rowspan=1 colspan=3>Streptococcus devriesei</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>In silico prediction (not tested)</td></tr><tr><td rowspan=1 colspan=3>Streptococcus didelphis</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>In silico prediction (not tested)</td></tr><tr><td rowspan=1 colspan=3>Streptococcus downei</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>In silico prediction (not tested)</td></tr><tr><td rowspan=4 colspan=2>Streptococcus dysgalactiae</td><td rowspan=2 colspan=1>ssp. dysgalactiae</td><td rowspan=2 colspan=1>ATCC 43078</td><td rowspan=2 colspan=1>Type strainUnited Kingdom/1970</td></tr><tr><td rowspan=1 colspan=1>ae</td></tr><tr><td rowspan=1 colspan=1>ssp. equisimilis</td><td rowspan=1 colspan=1>ATCC 12388</td><td rowspan=1 colspan=1>Type strainUnited Kingdom/1970</td></tr><tr><td rowspan=1 colspan=1>ssp. equisimilis</td><td rowspan=1 colspan=1>NCTC 8543</td><td rowspan=1 colspan=1>LRA 06 11 76</td></tr><tr><td rowspan=1 colspan=3>Streptococcus equinus</td><td rowspan=1 colspan=1>ATCC 9812</td><td rowspan=1 colspan=1>Type strain</td></tr><tr><td rowspan=1 colspan=3>Streptococcus ferus</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>In silico prediction (not tested)</td></tr><tr><td rowspan=2 colspan=2>Streptococcus gallolyticus</td><td rowspan=1 colspan=1>ssp. gallolyticus</td><td rowspan=1 colspan=1>ATCC BAA-2069</td><td rowspan=1 colspan=1>2001</td></tr><tr><td rowspan=1 colspan=1>ssp. pasteurianus</td><td rowspan=1 colspan=1>ATCC 700338</td><td rowspan=1 colspan=1>RG 1996</td></tr><tr><td rowspan=1 colspan=3>Streptococcus gordonii</td><td rowspan=1 colspan=1>ATCC 10558</td><td rowspan=1 colspan=1>Type strain</td></tr><tr><td rowspan=1 colspan=3>Streptococcus halotolerans</td><td rowspan=1 colspan=2>In silico prediction (not tested)</td></tr><tr><td rowspan=1 colspan=3>Streptococcus henryi</td><td rowspan=1 colspan=2>In silico prediction (not tested)</td></tr><tr><td rowspan=1 colspan=3>Streptococcus himalayensis</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>In silico prediction (not tested)</td></tr><tr><td rowspan=1 colspan=3>Streptococcus hongkongensis</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>In silico prediction (not tested)</td></tr><tr><td rowspan=1 colspan=3>Streptococcus hyointestinalis</td><td rowspan=1 colspan=2>In silico prediction (not tested)</td></tr><tr><td rowspan=1 colspan=3>Streptococcus ictaluri</td><td rowspan=1 colspan=2>In silico prediction (not tested)</td></tr><tr><td rowspan=1 colspan=3>Streptococcus infantarius   ssp. infantarius</td><td rowspan=1 colspan=1>ATCC BAA-102</td><td rowspan=1 colspan=1>HDP 90056</td></tr><tr><td rowspan=1 colspan=3>Streptococcus iniae</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>In silico prediction (not tested)</td></tr><tr><td rowspan=1 colspan=3>Streptococcus intermedius</td><td rowspan=1 colspan=1>ATCC 27335</td><td rowspan=1 colspan=1>1877</td></tr><tr><td rowspan=1 colspan=3>Streptococcus lactarius</td><td rowspan=1 colspan=2>In silico prediction (not tested)</td></tr><tr><td rowspan=1 colspan=3>Streptococcus lutetiensis</td><td rowspan=1 colspan=2>In silico prediction (not tested)</td></tr><tr><td rowspan=1 colspan=3>Streptococcus macacae</td><td rowspan=1 colspan=2>In silico prediction (not tested)</td></tr><tr><td rowspan=1 colspan=3>Streptococcus marimammalium</td><td rowspan=1 colspan=2>In silico prediction (not tested)</td></tr><tr><td rowspan=1 colspan=3>Streptococcus marmotae</td><td rowspan=1 colspan=2>In silico prediction (not tested)</td></tr><tr><td rowspan=1 colspan=3>Streptococcus massiliensis</td><td rowspan=1 colspan=2>In silico prediction (not tested)</td></tr><tr><td rowspan=1 colspan=3>Streptococcus merionis</td><td rowspan=1 colspan=2>In silico prediction (not tested)</td></tr><tr><td rowspan=1 colspan=3>Streptococcus milleri</td><td rowspan=1 colspan=2>In silico prediction (not tested)</td></tr><tr><td rowspan=1 colspan=3>Streptococcus minor</td><td rowspan=1 colspan=2>In silico prediction (not tested)</td></tr><tr><td rowspan=1 colspan=3>Streptococcus mitis</td><td rowspan=1 colspan=1>ATCC 49456</td><td rowspan=1 colspan=1>Type strain</td></tr><tr><td rowspan=1 colspan=3>Streptococcus mutans</td><td rowspan=1 colspan=1>ATCC 25175</td><td rowspan=1 colspan=1>Type Strain</td></tr><tr><td rowspan=1 colspan=3>Streptococcus oligofermentans</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>In silico prediction (not tested)</td></tr><tr><td rowspan=2 colspan=2>Streptococcus oralisa</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>ATCC 10557</td><td rowspan=1 colspan=1>SK2</td></tr><tr><td rowspan=1 colspan=1>ssp. tigurinus</td><td rowspan=1 colspan=1>DSM 24864</td><td rowspan=1 colspan=1>Type strain Switzerland</td></tr><tr><td rowspan=1 colspan=3>Streptococcus orisasini</td><td rowspan=1 colspan=2>In silico prediction (not tested)</td></tr><tr><td rowspan=1 colspan=3>Streptococcus orisratti</td><td rowspan=1 colspan=2>In silico prediction not tested)</td></tr><tr><td rowspan=1 colspan=3>Streptococcus ovis</td><td rowspan=1 colspan=2>In silico prediction (not tested)</td></tr><tr><td rowspan=1 colspan=3>Streptococcus parasanguinis</td><td rowspan=1 colspan=2>ATCC 31412                  Si-1</td></tr><tr><td rowspan=1 colspan=3>Streptococcus parasuis</td><td rowspan=1 colspan=2>In silico prediction (not tested)</td></tr><tr><td rowspan=1 colspan=3>Streptococcus parauberis</td><td rowspan=1 colspan=2>In silico prediction (not tested)</td></tr><tr><td rowspan=1 colspan=3>Streptococcus pasteurianus</td><td rowspan=1 colspan=2>In silico prediction (not tested)</td></tr><tr><td rowspan=1 colspan=3>Streptococcus penaeicida</td><td rowspan=1 colspan=2>In silico prediction (not tested)</td></tr><tr><td rowspan=1 colspan=3>Streptococcus peroris</td><td rowspan=1 colspan=1>ATCC 700780</td><td rowspan=1 colspan=1>Jap stai p</td></tr></table>

<table><tr><td rowspan=1 colspan=2>Organism</td><td rowspan=1 colspan=1>Source ID</td><td rowspan=1 colspan=1>Strain/Location/Year</td><td rowspan=1 colspan=1>Result</td></tr><tr><td rowspan=1 colspan=2>Streptococcus phocae</td><td rowspan=1 colspan=2>In silico prediction (not tested)</td><td rowspan=23 colspan=1>′</td></tr><tr><td rowspan=1 colspan=2>Streptococcus pluranimalium</td><td rowspan=1 colspan=2>In silico prediction (not tested)</td></tr><tr><td rowspan=1 colspan=2>Streptococcus plurextorum</td><td rowspan=1 colspan=2>In silico prediction (not tested)</td></tr><tr><td rowspan=1 colspan=2>Streptococcus pneumoniae</td><td rowspan=1 colspan=1>ATCC 33400</td><td rowspan=1 colspan=1>Type strain</td></tr><tr><td rowspan=1 colspan=2>Streptococcus porci</td><td rowspan=1 colspan=2>In silico prediction (not tested)</td></tr><tr><td rowspan=1 colspan=2>Streptococcus porcinus</td><td rowspan=1 colspan=2>In silico prediction (not tested)</td></tr><tr><td rowspan=1 colspan=2>Streptococcus pseudopneumoniae</td><td rowspan=1 colspan=1>ATCC BAA-960</td><td rowspan=1 colspan=1>Type strainCanada2002</td></tr><tr><td rowspan=1 colspan=2>Streptococcus pseudoporcinus</td><td rowspan=1 colspan=2>In silico prediction (not tested)</td></tr><tr><td rowspan=1 colspan=2>Streptococcus pyogenes</td><td rowspan=1 colspan=1>ATCC 49399</td><td rowspan=1 colspan=1>QC A62</td></tr><tr><td rowspan=1 colspan=2>Streptococcus ratti</td><td rowspan=1 colspan=2>In silico prediction (not tested)</td></tr><tr><td rowspan=1 colspan=2>Streptococcus respiraculi</td><td rowspan=1 colspan=2>In silico prediction (not tested)</td></tr><tr><td rowspan=1 colspan=2>Streptococcus ruminantium</td><td rowspan=1 colspan=2>In silico prediction (not tested)</td></tr><tr><td rowspan=2 colspan=1>Streptococcus salivarius</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>ATCC 13419</td><td rowspan=1 colspan=1>C699</td></tr><tr><td rowspan=1 colspan=1>ssp. thermophiles</td><td rowspan=1 colspan=1>ATCC 19258</td><td rowspan=1 colspan=1>Type Strain</td></tr><tr><td rowspan=1 colspan=2>Streptococcus sanguinis</td><td rowspan=1 colspan=1>ATCC 10556</td><td rowspan=1 colspan=1>Type strain</td></tr><tr><td rowspan=1 colspan=2>Streptococcus sinensis</td><td rowspan=1 colspan=1>DSM 14990</td><td rowspan=1 colspan=1>HKU4Hong Kong</td></tr><tr><td rowspan=1 colspan=2>Streptococcys sobrinusa</td><td rowspan=1 colspan=1>ATCC 33478</td><td rowspan=1 colspan=1>Type strain</td></tr><tr><td rowspan=1 colspan=2>Streptococcus suisa</td><td rowspan=1 colspan=1>ATCC 43765</td><td rowspan=1 colspan=1>Type strain</td></tr><tr><td rowspan=1 colspan=2>Streptococcus thoraltensis</td><td rowspan=1 colspan=2>In silico prediction (not tested)</td></tr><tr><td rowspan=1 colspan=2>Streptococcus troglodytae</td><td rowspan=1 colspan=2>In silico prediction (not tested)</td></tr><tr><td rowspan=1 colspan=2>Streptococcus uberisa</td><td rowspan=1 colspan=2>In silico prediction (not tested)</td></tr><tr><td rowspan=1 colspan=2>Streptococcus urinalis</td><td rowspan=1 colspan=2>In silico prediction (not tested)</td></tr><tr><td rowspan=1 colspan=2>Streptococcus vestibularis</td><td rowspan=1 colspan=1>ATCC 49124</td><td rowspan=1 colspan=1>Type strainUnited Kingdom</td></tr><tr><td rowspan=2 colspan=1>Streptococcus equi</td><td rowspan=1 colspan=1>ssp. equi</td><td rowspan=1 colspan=1>ATCC 33398</td><td rowspan=1 colspan=1>Type strain</td><td rowspan=6 colspan=1>Streptococcusspp.Detected(≥7.6E+06 FFU/mL)</td></tr><tr><td rowspan=1 colspan=1>ssp.zooepidemicus</td><td rowspan=1 colspan=1>ATCC 43079</td><td rowspan=1 colspan=1>Type strainUnited Kingdom</td></tr><tr><td rowspan=1 colspan=2>Streptococcus entericus</td><td rowspan=4 colspan=2>In silico prediction (not tested)</td></tr><tr><td rowspan=1 colspan=2>Strepotococcus halitosis</td></tr><tr><td rowspan=1 colspan=2>Streptococcus hyovaginalis</td></tr><tr><td rowspan=1 colspan=2>Streptococcus pantholopis</td></tr><tr><td rowspan=1 colspan=2>Other Streptococcus species</td><td rowspan=1 colspan=2>No Sequence (not tested)</td><td rowspan=1 colspan=1>Unknown</td></tr></table>

small percentage of publicly available sequences for this species have sequence variation under assay primer(s) that may have an impact on detection

Table 80. Results for Streptococcus agalactiae Isolates Tested   

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Serotypea</td><td rowspan=1 colspan=1>Source ID</td><td rowspan=1 colspan=1>Strain/Location/Year</td><td rowspan=1 colspan=1>Result</td></tr><tr><td rowspan=7 colspan=1>Streptococcusagalactiae</td><td rowspan=1 colspan=1>II</td><td rowspan=1 colspan=1>ATCC 13813</td><td rowspan=1 colspan=1>Type strain</td><td rowspan=7 colspan=1>Streptococcus agalactiaeDetected</td></tr><tr><td rowspan=1 colspan=1>II</td><td rowspan=1 colspan=1>ATCC 12403</td><td rowspan=1 colspan=1>Type strain</td></tr><tr><td rowspan=1 colspan=1>V</td><td rowspan=1 colspan=1>ATCC BAA-611</td><td rowspan=1 colspan=1>2603 V/R</td></tr><tr><td rowspan=1 colspan=1>VIII</td><td rowspan=1 colspan=1>ATCC BAA-2669</td><td rowspan=1 colspan=1>5030-08</td></tr><tr><td rowspan=3 colspan=1>Unknown</td><td rowspan=1 colspan=1>ATCC 12386</td><td rowspan=1 colspan=1>Grouping strain</td></tr><tr><td rowspan=1 colspan=1>NCTC 8017</td><td rowspan=1 colspan=1>MK 104 P</td></tr><tr><td rowspan=1 colspan=1>BF CI-2460</td><td rowspan=1 colspan=1>-</td></tr></table>

a Assay reactivity is not serotype-dependent, the assay will react with all serotypes

Table 81. Results for Streptococcus pneumoniae Isolates Tested   

<table><tr><td colspan="1" rowspan="1">Organism</td><td colspan="1" rowspan="1">Serotypea</td><td colspan="1" rowspan="1">Source ID</td><td colspan="1" rowspan="1">Strain/Location/Year</td><td colspan="1" rowspan="1">Result</td></tr><tr><td colspan="1" rowspan="8">Streptococcuspneumoniae</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">ATCC 6303</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="8">Streptococcus pneumoniaeDetected</td></tr><tr><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">ATCC 33400</td><td colspan="1" rowspan="1">Type strain</td></tr><tr><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">ATCC BAA-341</td><td colspan="1" rowspan="1">SPN1439-106Columbia/1995</td></tr><tr><td colspan="1" rowspan="1">14</td><td colspan="1" rowspan="1">ATCC 700672</td><td colspan="1" rowspan="1">VH14/Spain</td></tr><tr><td colspan="1" rowspan="1">11A</td><td colspan="1" rowspan="1">NCTC 11900</td><td colspan="1" rowspan="1">Gorman</td></tr><tr><td colspan="1" rowspan="1">19A</td><td colspan="1" rowspan="1">ATCC 700673</td><td colspan="1" rowspan="1">19A-6Hungary/1989</td></tr><tr><td colspan="1" rowspan="1">Non-capsulated</td><td colspan="1" rowspan="1">ATCC BAA-255</td><td colspan="1" rowspan="1">R6 (non-virulent)</td></tr><tr><td colspan="1" rowspan="1">unknown</td><td colspan="1" rowspan="1">ATCC BAA-1409</td><td colspan="1" rowspan="1">62076</td></tr><tr><td colspan="1" rowspan="1">Organism</td><td colspan="1" rowspan="1">Serotypea</td><td colspan="1" rowspan="1">Source ID</td><td colspan="1" rowspan="1">Strain/L_ocation/Year</td><td colspan="1" rowspan="1">Result</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Canada/2005</td><td colspan="1" rowspan="1"></td></tr></table>

a Assay reactivity is not serotype-dependent, the assay will react with all serotypes

Table 82. Results for Streptococcus pyogenes Isolates Tested   

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Serotypea</td><td rowspan=1 colspan=1>Source ID</td><td rowspan=1 colspan=1>Strain/Location/Year</td><td rowspan=1 colspan=1>Result</td></tr><tr><td rowspan=9 colspan=1>Streptococcuspyogenes</td><td rowspan=2 colspan=1>unknown</td><td rowspan=1 colspan=1>ATCC 49399</td><td rowspan=1 colspan=1>QC A62</td><td rowspan=8 colspan=1>Streptococcus pyogenesDetected</td></tr><tr><td rowspan=1 colspan=1>ATCC 19615</td><td rowspan=1 colspan=1>Bruno</td></tr><tr><td rowspan=3 colspan=1>1</td><td rowspan=1 colspan=1>ATCC 12344</td><td rowspan=1 colspan=1>Type strain</td></tr><tr><td rowspan=1 colspan=1>ATCC 700294</td><td rowspan=1 colspan=1>SF370/M1 GAS</td></tr><tr><td rowspan=1 colspan=1>ATCC BAA-947</td><td rowspan=1 colspan=1>MGAS 5005Canada/1996</td></tr><tr><td rowspan=2 colspan=1>3</td><td rowspan=1 colspan=1>ATCC 12384</td><td rowspan=1 colspan=1>C203</td></tr><tr><td rowspan=1 colspan=1>ATCC BAA-595</td><td rowspan=1 colspan=1>MGAS 315Texas 1980&#x27;s</td></tr><tr><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>ATCC 12348</td><td rowspan=1 colspan=1>S43</td></tr><tr><td rowspan=1 colspan=1>Unknown</td><td rowspan=1 colspan=1>Clinical Isolateb</td><td rowspan=1 colspan=1>Missouri/2019</td><td rowspan=1 colspan=1>Not Detected</td></tr></table>

a Assay reactivity is not serotype-dependent, the assay will react with all serotypes b Isolate of S. pyogenes with partial gene deletion

le 83. Results for Acinetobacter calcoaceticus-baumannii complex Isolates Tested   

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Source ID</td><td rowspan=1 colspan=1>Strain/Location/Year</td><td rowspan=1 colspan=1>Result</td></tr><tr><td rowspan=8 colspan=1>Acinetobacter baumannii</td><td rowspan=1 colspan=1>CDC FDA AR BANK #0033</td><td rowspan=1 colspan=1>-</td><td rowspan=15 colspan=1>Acinetobactercalcoaceticus-baumanniicomplex Detected</td></tr><tr><td rowspan=1 colspan=1>ATCC 9955</td><td rowspan=1 colspan=1>6-561/ltaly</td></tr><tr><td rowspan=1 colspan=1>ATCC 19606</td><td rowspan=1 colspan=1>Type strain</td></tr><tr><td rowspan=1 colspan=1>ATCC 17961</td><td rowspan=1 colspan=1>CDC 7788</td></tr><tr><td rowspan=1 colspan=1>GRE 1153064</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>GRE 1062081</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>ATCC 15308</td><td rowspan=1 colspan=1>Biol 1Maryland/1949</td></tr><tr><td rowspan=1 colspan=1>ATCC 51432</td><td rowspan=1 colspan=1>Pennsylvania</td></tr><tr><td rowspan=2 colspan=1>Acinetobacter calcoaceticus</td><td rowspan=1 colspan=1>ATCC 23055</td><td rowspan=1 colspan=1>Type strainNetherlands</td></tr><tr><td rowspan=1 colspan=1>ATCC 14987</td><td rowspan=1 colspan=1>HO-1/lowa</td></tr><tr><td rowspan=2 colspan=1>Acinetobacter nosocomialis(formerly genomospecies 13TU)</td><td rowspan=1 colspan=1>ATCC 17903</td><td rowspan=1 colspan=1>2210Rhode Island/1950</td></tr><tr><td rowspan=1 colspan=1>CCUG 57124</td><td rowspan=1 colspan=1>Sweden/2008</td></tr><tr><td rowspan=2 colspan=1>Acinetobacter pittii(formerly genomospecies 3)</td><td rowspan=1 colspan=1>ATCC 19004</td><td rowspan=1 colspan=1>Type strainUnited Kingdom/1966</td></tr><tr><td rowspan=1 colspan=1>ATCC 17922</td><td rowspan=1 colspan=1>Pennsylvania</td></tr><tr><td rowspan=1 colspan=1>Acinetobacter seifertii</td><td rowspan=1 colspan=1>CCUG 34785</td><td rowspan=1 colspan=1>Type strainDenmark</td></tr><tr><td rowspan=1 colspan=1>Acinetobacter nosocomialis</td><td rowspan=1 colspan=1>ATCC 700472a</td><td rowspan=1 colspan=1>France/1989</td><td rowspan=1 colspan=1>Not Detected</td></tr><tr><td rowspan=1 colspan=1>Acinetobacter dikshoorniae</td><td rowspan=1 colspan=2>No sequence (not tested)</td><td rowspan=1 colspan=1>Unknown</td></tr></table>

a Sequence data from this isolate suggests that it has been mischaracterized. Sequence data are not consistent with other sequences of A. nosocomialis nor with sequences from other species within the Acinetobacter calcoaceticus-baumannii complex

Table 84. Results for Bacteroides fragilis Isolates Tested   

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Source ID</td><td rowspan=1 colspan=1>Strain/Location/Year</td><td rowspan=1 colspan=1>Result</td></tr><tr><td rowspan=5 colspan=1>Bacteroidesfragilis</td><td rowspan=1 colspan=1>ATCC 25285</td><td rowspan=1 colspan=1>Type strainUnited Kingdom/1955</td><td rowspan=5 colspan=1>Bacteroides fragilisDetected</td></tr><tr><td rowspan=1 colspan=1>ATCC 29771</td><td rowspan=1 colspan=1>2044Florida/USA</td></tr><tr><td rowspan=1 colspan=1>ATCC 29768</td><td rowspan=1 colspan=1>12256</td></tr><tr><td rowspan=1 colspan=1>ATCC 43937</td><td rowspan=1 colspan=1>F1355</td></tr><tr><td rowspan=1 colspan=1>ATCC BAA-2283</td><td rowspan=1 colspan=1>2-1-56 FAA</td></tr></table>

Table 85. Results for Enterobacterales Isolates Tested and Predicted Reactivity for Species Not Tested   
Enterobacterales Detected   

<table><tr><td rowspan=1 colspan=1>Genus</td><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Source ID</td><td rowspan=1 colspan=1>Strain/Location/Year</td></tr><tr><td rowspan=2 colspan=1>Cedeceae</td><td rowspan=1 colspan=1>Cedecea davisae</td><td rowspan=1 colspan=1>ATCC 43023</td><td rowspan=1 colspan=1>CDC 2819-81South Carolina</td></tr><tr><td rowspan=1 colspan=1>Cedecea neteri</td><td rowspan=1 colspan=1>ATCC 33855</td><td rowspan=1 colspan=1>Type strainCalifornia</td></tr><tr><td rowspan=12 colspan=1>Citrobacter</td><td rowspan=1 colspan=1>Citrobacter amalonaticus</td><td rowspan=1 colspan=1>ATCC 25405</td><td rowspan=1 colspan=1>Type strain</td></tr><tr><td rowspan=1 colspan=1>Citrobacter braakii</td><td rowspan=1 colspan=1>ATCC 51113</td><td rowspan=1 colspan=1>Type strainFFrance</td></tr><tr><td rowspan=1 colspan=1>Citrobacter farmer</td><td rowspan=1 colspan=1>ATCC 51112</td><td rowspan=1 colspan=1>Type strainNew York</td></tr><tr><td rowspan=2 colspan=1>Citrobacter freundii</td><td rowspan=1 colspan=1>ATCC 8090</td><td rowspan=1 colspan=1>Type strain</td></tr><tr><td rowspan=1 colspan=1>CDC FDA AR Bank #0157</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=2 colspan=1>Citrobacter koseri</td><td rowspan=1 colspan=1>ATCC 27156</td><td rowspan=1 colspan=1>CDC 3613-63</td></tr><tr><td rowspan=1 colspan=1>ATCC 29223</td><td rowspan=1 colspan=1>CDC 1378/74</td></tr><tr><td rowspan=1 colspan=1>Citrobacter murliniae</td><td rowspan=1 colspan=1>ATCC 51118</td><td rowspan=1 colspan=1>Type strain</td></tr><tr><td rowspan=1 colspan=1>Citrobacter sedlakii</td><td rowspan=1 colspan=1>ATCC 51115</td><td rowspan=1 colspan=1>Type strainFFrance</td></tr><tr><td rowspan=1 colspan=1>Citrobacter rodentium</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>In silico prediction (not tested)</td></tr><tr><td rowspan=1 colspan=1>Citrobacter werkmanii</td><td rowspan=1 colspan=1>ATCC 51114</td><td rowspan=1 colspan=1>Type strainBelgium</td></tr><tr><td rowspan=1 colspan=1>Citrobacter youngae</td><td rowspan=1 colspan=1>ATCC 29935</td><td rowspan=1 colspan=1>Type strainSouth Carolina</td></tr><tr><td rowspan=1 colspan=1>Cosenzaea</td><td rowspan=1 colspan=1>Cosenzaea (Proteus)myxofaciens</td><td rowspan=1 colspan=1>ATCC 19692</td><td rowspan=1 colspan=1>Type strain</td></tr><tr><td rowspan=6 colspan=1>Cronobacter</td><td rowspan=1 colspan=1>Cronobacter condimenti</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>In silico prediction (not tested)</td></tr><tr><td rowspan=1 colspan=1>Cronobacter dublinensis</td><td rowspan=1 colspan=1>DSM 18706</td><td rowspan=1 colspan=1>Type strainSwitzerland/2004</td></tr><tr><td rowspan=1 colspan=1>Cronobacter malonaticus</td><td rowspan=1 colspan=1>DSM 18702</td><td rowspan=1 colspan=1>Type strainNew York</td></tr><tr><td rowspan=1 colspan=1>Crononbacter muytjensii</td><td rowspan=1 colspan=1>DSM 51329</td><td rowspan=1 colspan=1>Type strainFrance</td></tr><tr><td rowspan=1 colspan=1>Cronobacter sakazakii</td><td rowspan=1 colspan=1>ATCC 29544</td><td rowspan=1 colspan=1>Type strain</td></tr><tr><td rowspan=1 colspan=1>Cronobacter turicensis</td><td rowspan=1 colspan=1>CCUG 55852</td><td rowspan=1 colspan=1>Type strainSwitzerland 2005</td></tr><tr><td rowspan=5 colspan=1>Edwardsiella</td><td rowspan=1 colspan=1>Edwardsiellaanguillarum</td><td rowspan=1 colspan=2>In silico prediction (not tested)</td></tr><tr><td rowspan=1 colspan=1>Edwardsiella hoshinae</td><td rowspan=1 colspan=2>In silico prediction (not tested)</td></tr><tr><td rowspan=1 colspan=1>Edwardsiella ictaluri</td><td rowspan=1 colspan=2>In silico prediction (not tested)</td></tr><tr><td rowspan=1 colspan=1>Edwardseilla piscicida</td><td rowspan=1 colspan=2>In silico prediction (not tested)</td></tr><tr><td rowspan=1 colspan=1>Edwardsiella tarda</td><td rowspan=1 colspan=1>ATCC 15947</td><td rowspan=1 colspan=1>Type strainKentucky</td></tr><tr><td rowspan=4 colspan=1>Enterobacter</td><td rowspan=1 colspan=1>Enterobacterbugandensis</td><td rowspan=1 colspan=1>DSM 29888</td><td rowspan=1 colspan=1>Typestrain Tanzania</td></tr><tr><td rowspan=1 colspan=1>Enterobactercancerogenus</td><td rowspan=1 colspan=1>ATCC 35317</td><td rowspan=1 colspan=1>Type strainNew York</td></tr><tr><td rowspan=1 colspan=1>Enterobacterroggenkampii</td><td rowspan=1 colspan=2>In silico prediction (not tested)</td></tr><tr><td rowspan=1 colspan=1>Enterobacter soli</td><td rowspan=1 colspan=1>ATCC BAA-2102</td><td rowspan=1 colspan=1>Type strainPeru</td></tr><tr><td rowspan=3 colspan=1>Escherichia</td><td rowspan=1 colspan=1>Escherichia albertii</td><td rowspan=1 colspan=1>CCUG 46494</td><td rowspan=1 colspan=1>Type strainBangladesh</td></tr><tr><td rowspan=1 colspan=1>Escherichia fergusonii</td><td rowspan=1 colspan=1>ATCC 35469</td><td rowspan=1 colspan=1>Type strainMissouri</td></tr><tr><td rowspan=1 colspan=1>Escherichia hermanii</td><td rowspan=1 colspan=1>ATCC 33650</td><td rowspan=1 colspan=1>Type strain Lousiana</td></tr><tr><td rowspan=1 colspan=1>Erwinia</td><td rowspan=1 colspan=1>Erwinia billingiae</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=2 colspan=1>Hafnia</td><td rowspan=1 colspan=1>Hafnia alvei</td><td rowspan=1 colspan=1>ATCC 51815</td><td rowspan=1 colspan=1>C2Minnesota</td></tr><tr><td rowspan=1 colspan=1>Hafnia paralvei</td><td rowspan=1 colspan=1>ATCC 29927</td><td rowspan=1 colspan=1>Type strain</td></tr><tr><td rowspan=2 colspan=1>Klebsiella</td><td rowspan=1 colspan=1>Klebsiella grimontii</td><td rowspan=1 colspan=1>DSM 105630</td><td rowspan=1 colspan=1>06D021</td></tr><tr><td rowspan=1 colspan=1>Klebsiella michiganensis</td><td rowspan=1 colspan=1>ATCC BAA-2403</td><td rowspan=1 colspan=1>Type strainMichigan</td></tr><tr><td rowspan=4 colspan=1>Kluyvera</td><td rowspan=1 colspan=1>Kluyvera ascorbate</td><td rowspan=1 colspan=1>CDC FDA AR BANK #0144</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>Kluyvera cryocrescens</td><td rowspan=1 colspan=1>CCUG 18767T</td><td rowspan=1 colspan=1>Type strain</td></tr><tr><td rowspan=1 colspan=1>Kluyvera georgiana</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>In silico prediction (not tested)</td></tr><tr><td rowspan=1 colspan=1>Kluyvera intermedia</td><td rowspan=1 colspan=1>ATCC 33110</td><td rowspan=1 colspan=1>Type strain</td></tr><tr><td rowspan=2 colspan=1>Kosakonia</td><td rowspan=1 colspan=1>Kosakonia cowanii</td><td rowspan=1 colspan=1>CCUG 62758</td><td rowspan=1 colspan=1>Sweden/2012</td></tr><tr><td rowspan=1 colspan=1>Kosakonia oryzae</td><td rowspan=1 colspan=2>In silico prediction (not tested)</td></tr></table>

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2>Kosakonia radicincitans</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>In silico prediction (not tested)</td></tr><tr><td rowspan=1 colspan=1>Leclercia</td><td rowspan=1 colspan=2>Leclercia adecarboxylata</td><td rowspan=1 colspan=1>ATCC 23216</td><td rowspan=1 colspan=1>1783</td></tr><tr><td rowspan=2 colspan=1>Lelliottia</td><td rowspan=1 colspan=2>Lelliottia amnigena</td><td rowspan=1 colspan=1>ATCC 51816</td><td rowspan=1 colspan=1>C3</td></tr><tr><td rowspan=1 colspan=2>Leliottia nimipressuralis</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Mixta</td><td rowspan=1 colspan=2>Mixta (Pantoea) gaviniae</td><td rowspan=1 colspan=1>CCUG 66381</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=3 colspan=1>Morganella</td><td rowspan=3 colspan=1>Morganellamorganii</td><td rowspan=1 colspan=1>ssp.</td><td rowspan=1 colspan=1>ATCC 25830</td><td rowspan=1 colspan=1>M11</td></tr><tr><td rowspan=1 colspan=1>moorgani</td><td rowspan=1 colspan=1>CDC FDA AR BANK #0057</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>ssp.siboni</td><td rowspan=1 colspan=1>ATCC 49948</td><td rowspan=1 colspan=1>CDC 8103-85</td></tr><tr><td rowspan=3 colspan=1>Pantoea</td><td rowspan=1 colspan=2>Pantoea agglomerans</td><td rowspan=1 colspan=1>ATCC 27155</td><td rowspan=1 colspan=1>CDC 1461-67</td></tr><tr><td rowspan=1 colspan=2>Pantoea ananatis</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>In silico prediction (not tested)</td></tr><tr><td rowspan=1 colspan=2>Pantoea septica</td><td rowspan=1 colspan=1>CCUG 67124</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>Phytobacter</td><td rowspan=1 colspan=2>Phytobacter ursingii</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Plesiomonas</td><td rowspan=1 colspan=2>Plesiomonas shigelloides</td><td rowspan=1 colspan=1>ATCC 51572</td><td rowspan=1 colspan=1>CIP 69.35</td></tr><tr><td rowspan=1 colspan=1>Pluralibacter</td><td rowspan=1 colspan=2>Pluralibacter(Enterobacter) gergoviae</td><td rowspan=1 colspan=1>ATCC 33028</td><td rowspan=1 colspan=1>CDC 604-77</td></tr><tr><td rowspan=3 colspan=1>Providencia</td><td rowspan=1 colspan=2>Providencia alcalifaciens</td><td rowspan=1 colspan=1>ATCC 51902</td><td rowspan=1 colspan=1>GNI 3</td></tr><tr><td rowspan=1 colspan=2>Providencia rettgeri</td><td rowspan=1 colspan=1>ATCC 9250</td><td rowspan=1 colspan=1>NCTC 1501</td></tr><tr><td rowspan=1 colspan=2>Providencia stuartii</td><td rowspan=1 colspan=1>CDC FDA AR Bank #0026</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>Pseudoescherichia</td><td rowspan=1 colspan=2>PseudoescherichiaEscherichia) vulneris</td><td rowspan=1 colspan=1>ATCC 33821</td><td rowspan=1 colspan=1>CDC 875-72</td></tr><tr><td rowspan=1 colspan=1>Rahnella</td><td rowspan=1 colspan=2>Rahnella aquatilis</td><td rowspan=1 colspan=1>ATCC 33071</td><td rowspan=1 colspan=1>CUETM 77-115</td></tr><tr><td rowspan=3 colspan=1>Raoultella</td><td rowspan=1 colspan=2>Raoultella ornithinolytica</td><td rowspan=1 colspan=1>ATCC 31898</td><td rowspan=1 colspan=1>AST 111-4</td></tr><tr><td rowspan=1 colspan=2>Raoultella planticola</td><td rowspan=1 colspan=1>ATCC 31900</td><td rowspan=1 colspan=1>AST 151-7</td></tr><tr><td rowspan=1 colspan=2>Raoultella terrigena</td><td rowspan=1 colspan=1>ATCC 33257</td><td rowspan=1 colspan=1>Type strain</td></tr><tr><td rowspan=10 colspan=1>Serratia</td><td rowspan=1 colspan=2>Serratia aquatilis</td><td rowspan=1 colspan=1>DSM 100980</td><td rowspan=1 colspan=1>2015-2462-01</td></tr><tr><td rowspan=1 colspan=2>Serratia entomophila</td><td rowspan=1 colspan=1>ATCC 43705</td><td rowspan=1 colspan=1>Type strain</td></tr><tr><td rowspan=1 colspan=2>Serratia ficaria</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>In silico prediction (not tested)</td></tr><tr><td rowspan=1 colspan=2>Serratia fonticola</td><td rowspan=1 colspan=1>ATCC 29844</td><td rowspan=1 colspan=1>CUETM 77-165</td></tr><tr><td rowspan=1 colspan=2>Serratia grimesii</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>In silico prediction (not tested)</td></tr><tr><td rowspan=1 colspan=2>Serratia liquefaciens</td><td rowspan=1 colspan=1>ATCC 27592</td><td rowspan=1 colspan=1>CDC 1284-57</td></tr><tr><td rowspan=1 colspan=2>Serratia odorifera</td><td rowspan=1 colspan=1>ATCC 33077</td><td rowspan=1 colspan=1>1073</td></tr><tr><td rowspan=1 colspan=2>Serratia plymuthica</td><td rowspan=1 colspan=1>ATCC 183</td><td rowspan=1 colspan=1>K-7</td></tr><tr><td rowspan=1 colspan=2>Serratia proteamaculans</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=2>Serratia rubidaea</td><td rowspan=1 colspan=1>ATCC 27593</td><td rowspan=1 colspan=1>2199-72</td></tr><tr><td rowspan=1 colspan=1>Sodalis</td><td rowspan=1 colspan=2>Sodalis praecaptivus</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=4 colspan=1>Shigella</td><td rowspan=1 colspan=2>Shigella boydii</td><td rowspan=1 colspan=1>ATCC 9207</td><td rowspan=1 colspan=1>AMC 43-G-58</td></tr><tr><td rowspan=1 colspan=2>Shigella dysenteriae</td><td rowspan=1 colspan=1>ATCC 13313</td><td rowspan=1 colspan=1>Strain Newcastle</td></tr><tr><td rowspan=1 colspan=2>Shigella flexneri</td><td rowspan=1 colspan=1>CDC FDA AR Bank #0421</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=2>Shigella sonnei</td><td rowspan=1 colspan=1>ATCC 29930</td><td rowspan=1 colspan=1>WRAIR I virulent</td></tr><tr><td rowspan=1 colspan=1>Tatumella</td><td rowspan=1 colspan=2>Tatumella ptyseos</td><td rowspan=1 colspan=1>ATCC 33301</td><td rowspan=1 colspan=1>H36</td></tr><tr><td rowspan=1 colspan=1>Trabulsiella</td><td rowspan=1 colspan=2>Trabulsiella guamensis</td><td rowspan=1 colspan=1>ATCC 49490</td><td rowspan=1 colspan=1>Type strain</td></tr><tr><td rowspan=13 colspan=1>Yersinia</td><td rowspan=1 colspan=2>Yersinia aldovae</td><td rowspan=1 colspan=2>In silico prediction (not tested)</td></tr><tr><td rowspan=1 colspan=2>Yersinia aleksiciae</td><td rowspan=1 colspan=2>In silico prediction (not tested)</td></tr><tr><td rowspan=1 colspan=2>Yersinia enterocolitica</td><td rowspan=1 colspan=1>ATCC 9610</td><td rowspan=1 colspan=1>33114</td></tr><tr><td rowspan=1 colspan=2>Yersinia entomophage</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=2>Yersinia frederiksenii</td><td rowspan=1 colspan=1>ATCC 33641</td><td rowspan=1 colspan=1>CDC 1461-81</td></tr><tr><td rowspan=1 colspan=2>Yersinia intermedia</td><td rowspan=1 colspan=1>ATCC 33647</td><td rowspan=1 colspan=1>CDC 870-77</td></tr><tr><td rowspan=1 colspan=2>Yersinia kristensenii</td><td rowspan=1 colspan=1>ATCC 33639</td><td rowspan=1 colspan=1>CDC 1459-81</td></tr><tr><td rowspan=1 colspan=2>Yersinia massiliensis</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>In silico prediction (not tested)</td></tr><tr><td rowspan=1 colspan=2>Yersinia mollaretii</td><td rowspan=1 colspan=1>ATCC 43969</td><td rowspan=1 colspan=1>CDC 2465-87</td></tr><tr><td rowspan=1 colspan=2>Yersinia pestis</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>In silico prediction (not tested)</td></tr><tr><td rowspan=1 colspan=2>Yersinia rohdei</td><td rowspan=1 colspan=2>In silico prediction (not tested)</td></tr><tr><td rowspan=1 colspan=2>Yersinia ruckeri</td><td rowspan=1 colspan=2>In silico prediction (not tested)</td></tr><tr><td rowspan=1 colspan=2>Yersinia similis</td><td rowspan=1 colspan=2>In silico prediction (not tested)</td></tr><tr><td rowspan=1 colspan=1>Yokenella</td><td rowspan=1 colspan=2>Yokenella regensburgei</td><td rowspan=1 colspan=1>ATCC 35313</td><td rowspan=1 colspan=1>CDC 3349-72</td></tr><tr><td rowspan=1 colspan=1>Mixta</td><td rowspan=1 colspan=2>Mixta (Pantoea) calida</td><td rowspan=1 colspan=2>CCUG 68064                       -</td></tr><tr><td rowspan=1 colspan=1>Yersinia</td><td rowspan=1 colspan=2>Yersiniapseudotuberculosis</td><td rowspan=1 colspan=1>ATCC 29833</td><td rowspan=1 colspan=1>NCTC 10275</td></tr><tr><td rowspan=1 colspan=1>Photorabdus</td><td rowspan=1 colspan=2>Photorabdus asymbiotica</td><td rowspan=1 colspan=1>ATCC 43950</td><td rowspan=1 colspan=1>3265-86</td></tr><tr><td rowspan=1 colspan=1>Arsenophonus</td><td rowspan=1 colspan=2>Arsenophonus nasoniae</td><td rowspan=1 colspan=2>In silico prediction (not tested)</td></tr></table>

<table><tr><td rowspan=1 colspan=1>Genus</td><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Source ID</td><td rowspan=1 colspan=1>Strain/Location/Year</td><td rowspan=1 colspan=1>Result</td></tr><tr><td rowspan=1 colspan=1>Providencia</td><td rowspan=1 colspan=1>Providencia heimbachae</td><td rowspan=1 colspan=2>In silico prediction (not tested)</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=2>Other Enterobacterales species</td><td rowspan=1 colspan=2>In silico prediction (not tested)orNo sequence (not tested)</td><td rowspan=1 colspan=1>DetectedorUnknown</td></tr></table>

a Only tested at high concentration ${ \left( > 1 . 0 E + 0 9 \right. }$ CFU/mL), expected to be detected at positive blood culture levels and lower

Table 86. Results for Enterobacter cloacae complex Isolates Tested   

<table><tr><td rowspan=1 colspan=2>Organism</td><td rowspan=1 colspan=1>Source ID</td><td rowspan=1 colspan=1>Strain/Location/Year</td><td rowspan=1 colspan=1>Result</td></tr><tr><td rowspan=1 colspan=2>Enterobacter asburiae</td><td rowspan=1 colspan=1>GRE 1753006</td><td rowspan=1 colspan=1></td><td rowspan=16 colspan=1>EnterobactercloacaecomplexDetected</td></tr><tr><td rowspan=8 colspan=1>Enterobactercloacae</td><td rowspan=5 colspan=1></td><td rowspan=1 colspan=1>CDC FDA AR Bank #0154</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>CDC FDA AR Bank #0501</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>ATCC 49141</td><td rowspan=1 colspan=1>AmMS 204</td></tr><tr><td rowspan=1 colspan=1>ATCC BAA-2341</td><td rowspan=1 colspan=1>1101152</td></tr><tr><td rowspan=1 colspan=1>NCTC 13464</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=2 colspan=1>ssp. cloacae</td><td rowspan=1 colspan=1>ATCC 13047</td><td rowspan=1 colspan=1>Type strain</td></tr><tr><td rowspan=1 colspan=1>ATCC BAA-1143</td><td rowspan=1 colspan=1>Entb 55M</td></tr><tr><td rowspan=1 colspan=1>spp. dissolvens</td><td rowspan=1 colspan=1>ATCC 23373D</td><td rowspan=1 colspan=1>Type strain</td></tr><tr><td rowspan=4 colspan=1>Enterobacterhormaechei</td><td rowspan=1 colspan=1>•</td><td rowspan=1 colspan=1>ATCC BAA-2082</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>ssp. oharae</td><td rowspan=1 colspan=1>CCUG 53905T</td><td rowspan=1 colspan=1>Type strain Germany</td></tr><tr><td rowspan=1 colspan=1>ssp. steigerwalthii</td><td rowspan=1 colspan=1>CCUG 53904T</td><td rowspan=1 colspan=1>Type strainBelgium</td></tr><tr><td rowspan=1 colspan=1>ssp. xiangfangensis</td><td rowspan=1 colspan=1>DSM 46348</td><td rowspan=1 colspan=1>1080M</td></tr><tr><td rowspan=1 colspan=2>Enterobacter kobei</td><td rowspan=1 colspan=1>GRE 1753004</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=2>Enterobacter ludwigii</td><td rowspan=1 colspan=1>CCUG 23050</td><td rowspan=1 colspan=1>Sweden</td></tr><tr><td rowspan=1 colspan=2>Enterobacter mori</td><td rowspan=1 colspan=1>DSM 26271</td><td rowspan=1 colspan=1>Type strain/R18-2</td></tr><tr><td rowspan=5 colspan=2>Enterobacter asburiae</td><td rowspan=1 colspan=1>ATCC 35953</td><td rowspan=1 colspan=1>CDC 1497-78Rhode Island</td><td rowspan=6 colspan=1>EnterobactercloacaecomplexDetected(≥1.6E+07 CFU/mL)</td></tr><tr><td rowspan=1 colspan=1>ATCC 35954</td><td rowspan=1 colspan=1>Type strainMaryland</td></tr><tr><td rowspan=1 colspan=1>ATCC 35955</td><td rowspan=1 colspan=1>In silico prediction(not tested)</td></tr><tr><td rowspan=1 colspan=1>ATCC 35957</td><td rowspan=1 colspan=1>CDC 570-83Hawaii</td></tr><tr><td rowspan=1 colspan=1>GRE 0758100</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>Enterobacterhormaechei</td><td rowspan=1 colspan=1>ssp. hormaechei</td><td rowspan=1 colspan=1>ATCC 49162</td><td rowspan=1 colspan=1>Type strain California</td></tr></table>

Table 87. Results for Escherichia coli Isolates Tested   

<table><tr><td rowspan=1 colspan=2>Organism</td><td rowspan=1 colspan=1>Source ID</td><td rowspan=1 colspan=1>Strain/Location/Year</td><td rowspan=1 colspan=1>Result</td></tr><tr><td rowspan=13 colspan=2>Escherichia coli</td><td rowspan=1 colspan=1>CDC FDA AR BANK #0061</td><td rowspan=1 colspan=1></td><td rowspan=13 colspan=1>Escherichia coli Detected</td></tr><tr><td rowspan=1 colspan=1>CDC FDA AR BANK #0086</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>CDC FDA AR BANK #0137</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>CDC FDA AR BANK #0149</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>CDC FDA AR BANK #0150</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>CDC FDA AR BANK #0346</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>CDC FDA AR BANK #0349</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>CDC FDA AR BANK #0350</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>CDC FDA AR BANK #0495</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>ATCC 11775</td><td rowspan=1 colspan=1>Type strain</td></tr><tr><td rowspan=1 colspan=1>ATCC 25922</td><td rowspan=1 colspan=1>DA strainSeattle 1946</td></tr><tr><td rowspan=1 colspan=1>GRE 1062016</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>GRE 1256018</td><td rowspan=1 colspan=1>-</td></tr></table>

Table 88. Results for Klebsiella aerogenes Isolates Tested   

<table><tr><td colspan="1" rowspan="1">Organism</td><td colspan="1" rowspan="1">Source ID</td><td colspan="1" rowspan="1">Strain/Location/Year</td><td colspan="1" rowspan="1">Result</td></tr><tr><td colspan="1" rowspan="2">Klebsiella aerogenes</td><td colspan="1" rowspan="1">CDC FDA AR Bank #0062</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="2">Klebsiella aerogenesDetected</td></tr><tr><td colspan="1" rowspan="1">CDC FDA AR Bank #0074</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="4"></td><td colspan="1" rowspan="1">CDC FDA AR Bank #0161</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="4"></td></tr><tr><td colspan="1" rowspan="1">ATCC 13048</td><td colspan="1" rowspan="1">Type strain</td></tr><tr><td colspan="1" rowspan="1">ATCC 29751</td><td colspan="1" rowspan="1">MULB-250</td></tr><tr><td colspan="1" rowspan="1">GRE 1254066</td><td colspan="1" rowspan="1">-</td></tr></table>

Table 89. Results for Klebsiella oxytoca Isolates Tested   

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Source ID</td><td rowspan=1 colspan=1>Strain/Location/Year</td><td rowspan=1 colspan=1>Result</td></tr><tr><td rowspan=9 colspan=1>Klebsiella oxytoca</td><td rowspan=1 colspan=1>CDC FDA AR Bank #0147</td><td rowspan=1 colspan=1>-</td><td rowspan=9 colspan=1>Klebsiella oxytocaDetected</td></tr><tr><td rowspan=1 colspan=1>ATCC 8724</td><td rowspan=1 colspan=1>NRRL B-199</td></tr><tr><td rowspan=1 colspan=1>ATCC 13182</td><td rowspan=1 colspan=1>Type strain</td></tr><tr><td rowspan=1 colspan=1>ATCC 43086</td><td rowspan=1 colspan=1>Pasco 201California</td></tr><tr><td rowspan=1 colspan=1>ATCC 49131</td><td rowspan=1 colspan=1>AmMS 101</td></tr><tr><td rowspan=1 colspan=1>ATCC 700324</td><td rowspan=1 colspan=1>LBM 90.11.033</td></tr><tr><td rowspan=1 colspan=1>GRE 1254054</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>JMI 2523</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>JMI 7818</td><td rowspan=1 colspan=1>-</td></tr></table>

Table 90. Results for Klebsiella pneumoniae group Isolates Tested   

<table><tr><td rowspan=1 colspan=2>Organism</td><td rowspan=1 colspan=1>Source ID</td><td rowspan=1 colspan=1>Strain/Location/Year</td><td rowspan=1 colspan=1>Result</td></tr><tr><td rowspan=18 colspan=1>Klebsiellapneumoniae</td><td rowspan=13 colspan=1></td><td rowspan=1 colspan=1>GRE 1553001</td><td rowspan=1 colspan=1></td><td rowspan=22 colspan=1>Klebsiella pneumoniaegroupDetected</td></tr><tr><td rowspan=1 colspan=1>CDC FDA AR Bank #0040</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>CDC FDA AR Bank #0068</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>CDC FDA AR Bank #0076</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>CDC FDA AR Bank #0079</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>CDC FDA AR Bank #0080</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>CDC FDA AR Bank #0107</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>CDC FDA AR Bank #0497</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>CDC FDA AR Bank #0525</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>ATCC BAA-1705</td><td rowspan=1 colspan=1>ART 2008133</td></tr><tr><td rowspan=1 colspan=1>GRE 1062084</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>GRE 1355030</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>JMI 766</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=3 colspan=1>ssp. ozanae</td><td rowspan=1 colspan=1>CDC FDA AR Bank #0051</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>CDC FDA AR Bank #0096</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>ATCC 11296</td><td rowspan=1 colspan=1>Type strainSumatra/Indonesia</td></tr><tr><td rowspan=1 colspan=1>ssp. pneumoniae</td><td rowspan=1 colspan=1>ATCC 13883</td><td rowspan=1 colspan=1>Type strain</td></tr><tr><td rowspan=1 colspan=1>ssp. rhinoscleromatis</td><td rowspan=1 colspan=1>ATCC 13884</td><td rowspan=1 colspan=1>Type strainSumatra/Indonesia</td></tr><tr><td rowspan=3 colspan=1>Klebsiellaquasipneumoniae</td><td rowspan=2 colspan=1>ssp.similipneumoniae</td><td rowspan=1 colspan=1>ATCC 700603</td><td rowspan=1 colspan=1>K6</td></tr><tr><td rowspan=1 colspan=1>DSM 28212</td><td rowspan=1 colspan=1>Type strainGermany/1997</td></tr><tr><td rowspan=1 colspan=1>ssp.quasipneumoniae</td><td rowspan=1 colspan=1>DSM 28211</td><td rowspan=1 colspan=1>Type strainAustria/1997</td></tr><tr><td rowspan=1 colspan=2>Klebsiella variicola</td><td rowspan=1 colspan=1>ATCC BAA-830</td><td rowspan=1 colspan=1>Type strainMexico/2001</td></tr></table>

Table 91. Results for Proteus spp. Isolates Tested and Predicted Reactivity for Species Not Tested   

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Source ID</td><td rowspan=1 colspan=1>Strain/Location/Year</td><td rowspan=1 colspan=1>Result</td></tr><tr><td rowspan=1 colspan=1>Proteus alimentorum</td><td rowspan=1 colspan=2>In silico prediction (not tested)</td><td rowspan=11 colspan=1>Proteusspp.Detected</td></tr><tr><td rowspan=1 colspan=1>Proteus cibarius</td><td rowspan=1 colspan=2>In silico prediction (not tested)</td></tr><tr><td rowspan=1 colspan=1>Proteus columbae</td><td rowspan=1 colspan=2>In silico prediction (not tested)</td></tr><tr><td rowspan=2 colspan=1>Proteus hauseri</td><td rowspan=1 colspan=1>ATCC 13315</td><td rowspan=1 colspan=1>strain Lehmann</td></tr><tr><td rowspan=1 colspan=1>ATCC 700826</td><td rowspan=1 colspan=1>Type strainTennessee</td></tr><tr><td rowspan=6 colspan=1>Proteus mirabilis</td><td rowspan=1 colspan=1>CDC FDA AR Bank #0155</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>CDC FDA AR Bank #0156</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>CDC FDA AR Bank #0159</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>ATCC 29906</td><td rowspan=1 colspan=1>Type strain</td></tr><tr><td rowspan=1 colspan=1>ATCC 33583</td><td rowspan=1 colspan=1>571101</td></tr><tr><td rowspan=1 colspan=1>GRE 1254053</td><td rowspan=1 colspan=1>-</td></tr></table>

<table><tr><td rowspan=3 colspan=1>Proteus penneri</td><td rowspan=1 colspan=1>ATCC 33519</td><td rowspan=1 colspan=1>Type strainIllinois</td></tr><tr><td rowspan=1 colspan=1>ATCC 35197</td><td rowspan=1 colspan=1>CDC 1655-67Maryland</td></tr><tr><td rowspan=1 colspan=1>ATCC 27973</td><td rowspan=1 colspan=1>CDC 1787-64-SC1Connecticut</td></tr><tr><td rowspan=1 colspan=1>Proteus terrae</td><td rowspan=1 colspan=2>In silico prediction (not tested)</td></tr><tr><td rowspan=1 colspan=1>Proteus vulgaris</td><td rowspan=1 colspan=1>ATCC 29905</td><td rowspan=1 colspan=1>Type strain</td></tr></table>

Table 92. Results for Salmonella spp. Isolates Tested   

<table><tr><td rowspan=1 colspan=2>Organism(alternate ssp. designation)</td><td rowspan=1 colspan=1>Serotype</td><td rowspan=1 colspan=1>Source ID</td><td rowspan=1 colspan=1>Strain/Location/Year</td><td rowspan=1 colspan=1>Result</td></tr><tr><td rowspan=3 colspan=2>Salmonella bongori(V)</td><td rowspan=1 colspan=1>Brookfield</td><td rowspan=1 colspan=1>NCTC 10946</td><td rowspan=1 colspan=1></td><td rowspan=17 colspan=1>Salmonellaspp.Detected</td></tr><tr><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>SGSC 3100 SARC11</td><td rowspan=1 colspan=1>CDC 750-721972</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>ATCC 43975</td><td rowspan=1 colspan=1>Type strain</td></tr><tr><td rowspan=14 colspan=1>Salmonellaenterica</td><td rowspan=1 colspan=1>ssp. arizonae(lla)</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>ATCC 13314</td><td rowspan=1 colspan=1>Type strain</td></tr><tr><td rowspan=1 colspan=1>ssp.diarizonae(II1b)</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>SGSC 3069 SARC8</td><td rowspan=1 colspan=1>CDC 678-94California/1984</td></tr><tr><td rowspan=9 colspan=1>ssp. enterica()</td><td rowspan=1 colspan=1>Concord</td><td rowspan=1 colspan=1>CDC FDA AR Bank #0407</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Enteritidis</td><td rowspan=1 colspan=1>ATCC BAA-708</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>Heidelberg</td><td rowspan=1 colspan=1>SGSC 2210 SARA30</td><td rowspan=1 colspan=1>Pennsylvania/1987</td></tr><tr><td rowspan=1 colspan=1>Infantis</td><td rowspan=1 colspan=1>ATCC BAA-1675</td><td rowspan=1 colspan=1>MZ1479Senegal</td></tr><tr><td rowspan=1 colspan=1>Montevideo</td><td rowspan=1 colspan=1>ATCC BAA-710</td><td rowspan=1 colspan=1>G46391993</td></tr><tr><td rowspan=1 colspan=1>Newport</td><td rowspan=1 colspan=1>ATCC 27869</td><td rowspan=1 colspan=1>C487-69</td></tr><tr><td rowspan=1 colspan=1>Senftenberg</td><td rowspan=1 colspan=1>CDC FDA AR Bank #0127</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>Typhimurium</td><td rowspan=1 colspan=1>ATCC 700720</td><td rowspan=1 colspan=1>1948</td></tr><tr><td rowspan=1 colspan=1>Typhimurium</td><td rowspan=1 colspan=1>SGSC 3029 SARC1</td><td rowspan=1 colspan=1>England/1958</td></tr><tr><td rowspan=1 colspan=1>ssp. houtenae(IV)</td><td rowspan=1 colspan=1>45a,b:g, z51:-</td><td rowspan=1 colspan=1>SGSC 3074 SARC9</td><td rowspan=1 colspan=1>CDC 2584-68Panama/1968</td></tr><tr><td rowspan=1 colspan=1>ssp. indica(VI)</td><td rowspan=1 colspan=1>45:a:e,n,x</td><td rowspan=1 colspan=1>SGSC 3116 SARC13</td><td rowspan=1 colspan=1>CDC 1363-65India 1965</td></tr><tr><td rowspan=1 colspan=1>ssp. salamae()</td><td rowspan=1 colspan=1>42:f:g,t:-</td><td rowspan=1 colspan=1>SGSC 3047 SARC4</td><td rowspan=1 colspan=1>CDC3472-641964</td></tr></table>

Table 93. Results for Serratia marcescens Isolates Tested   

<table><tr><td rowspan=1 colspan=2>Organism</td><td rowspan=1 colspan=1>Source ID</td><td rowspan=1 colspan=1>Strain/Location/Year</td><td rowspan=1 colspan=1>Result</td></tr><tr><td rowspan=8 colspan=1>Serratiamarcescens</td><td rowspan=5 colspan=1>−</td><td rowspan=1 colspan=1>CDC FDA AR Bank #0517</td><td rowspan=1 colspan=1></td><td rowspan=8 colspan=1>Serratia marcescensDetected</td></tr><tr><td rowspan=1 colspan=1>ATCC 27137</td><td rowspan=1 colspan=1>CDC 3100-71Colorado</td></tr><tr><td rowspan=1 colspan=1>GRE 1659004</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>GRE 1659006</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>JMI 697</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=2 colspan=1>ssp. marcescens</td><td rowspan=1 colspan=1>ATCC 13880</td><td rowspan=1 colspan=1>Type strain</td></tr><tr><td rowspan=1 colspan=1>ATCC 43297</td><td rowspan=1 colspan=1>3GBelgium</td></tr><tr><td rowspan=1 colspan=1>ssp. sakuensis</td><td rowspan=1 colspan=1>ATCC BAA-885</td><td rowspan=1 colspan=1>Type strainJapan 1992</td></tr></table>

Table 94. Results for Haemophilus influenzae Isolates Tested   

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Serotypea</td><td rowspan=1 colspan=1>Source ID</td><td rowspan=1 colspan=1>Strain/Location/Year</td><td rowspan=1 colspan=1>Result</td></tr><tr><td rowspan=7 colspan=1>Haemophilusinfluenzae</td><td rowspan=1 colspan=1>Type a</td><td rowspan=1 colspan=1>ATCC 9006</td><td rowspan=1 colspan=1>AMC 36-A-3</td><td rowspan=7 colspan=1>HaemophilusinfluenzaeDetected</td></tr><tr><td rowspan=1 colspan=1>Type b</td><td rowspan=1 colspan=1>ATCC 10211</td><td rowspan=1 colspan=1>AMC 36-A-1</td></tr><tr><td rowspan=1 colspan=1>Type c</td><td rowspan=1 colspan=1>ATCC 49699</td><td rowspan=1 colspan=1>C 9007</td></tr><tr><td rowspan=1 colspan=1>Type d</td><td rowspan=1 colspan=1>ATCC 9008</td><td rowspan=1 colspan=1>AMC 36-A-6</td></tr><tr><td rowspan=1 colspan=1>Type e</td><td rowspan=1 colspan=1>ATCC 8142</td><td rowspan=1 colspan=1>AMC 36-A-7</td></tr><tr><td rowspan=1 colspan=1>Type f</td><td rowspan=1 colspan=1>ATCC 700223</td><td rowspan=1 colspan=1>GA1264/Georgia</td></tr><tr><td rowspan=1 colspan=1>Non-typeable</td><td rowspan=1 colspan=1>ATCC 33391</td><td rowspan=1 colspan=1>Type strain</td></tr></table>

<table><tr><td rowspan=3 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>United Kingdom/1950</td><td rowspan=3 colspan=1></td></tr><tr><td rowspan=1 colspan=1>ATCC 51907</td><td rowspan=1 colspan=1>Rd [KW20]</td></tr><tr><td rowspan=1 colspan=1>ATCC 51997</td><td rowspan=1 colspan=1>INT 1Missouri</td></tr><tr><td rowspan=1 colspan=1>Unknown</td><td rowspan=1 colspan=1>Clinical Isolateb</td><td rowspan=1 colspan=1>Utah/2012</td><td rowspan=1 colspan=1>Not Detected</td></tr></table>

a Assay reactivity is not serotype-dependent, the assay will react with all types and non-typeable isolates b Clinical isolate of $H .$ influenzae with partial gene deletion

Table 95. Results for Neisseria meningitidis Isolates Tested   

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Serotypea</td><td rowspan=1 colspan=1>Source ID</td><td rowspan=1 colspan=1>Strain/Location/Year</td><td rowspan=1 colspan=1>Result</td></tr><tr><td rowspan=6 colspan=1>Neisseriameningiidis(encapsulated)</td><td rowspan=1 colspan=1>A</td><td rowspan=1 colspan=1>ATCC 13077</td><td rowspan=1 colspan=1>M1027Chicago/1937</td><td rowspan=6 colspan=1>Neisseria meningitidisDetected</td></tr><tr><td rowspan=1 colspan=1>B</td><td rowspan=1 colspan=1>ATCC 13090</td><td rowspan=1 colspan=1>M2092Massachusetts</td></tr><tr><td rowspan=1 colspan=1>C</td><td rowspan=1 colspan=1>ATCC 13102</td><td rowspan=1 colspan=1>M1628Connecticut/1949</td></tr><tr><td rowspan=1 colspan=1>D</td><td rowspan=1 colspan=1>ATCC 13113</td><td rowspan=1 colspan=1>M158/[37A] 955</td></tr><tr><td rowspan=1 colspan=1>W-135</td><td rowspan=1 colspan=1>ATCC 43744</td><td rowspan=1 colspan=1>M-1574Ilinois</td></tr><tr><td rowspan=1 colspan=1>Y</td><td rowspan=1 colspan=1>ATCC 35561</td><td rowspan=1 colspan=1>M-112/[BO-6]</td></tr></table>

a Assay reactivity is not serotype-dependent, the assay will react with all serotypes

# NOTE: Unencapsulated strains of Neisseria meningitidis will not be detected.

Table 96. Results for Pseudomonas aeruginosa Isolates Tested   

<table><tr><td rowspan=1 colspan=2>Organism</td><td rowspan=1 colspan=1>Source ID</td><td rowspan=1 colspan=1>Strain/Location/Year</td><td rowspan=1 colspan=1>Result</td></tr><tr><td rowspan=12 colspan=2>Pseudomonasaeruginosa</td><td rowspan=1 colspan=1>CDC FDA AR BANK #0092</td><td rowspan=1 colspan=1></td><td rowspan=10 colspan=1>Pseudomonas aeruginosaDetected</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>CDC FDA AR BANK #0054</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>CDC FDA AR BANK #0100</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>CDC FDA AR BANK #0103</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>CDC FDA AR BANK #0239</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>ATCC 10145</td><td rowspan=1 colspan=1>Type strainCzechoslovakia/1963</td></tr><tr><td rowspan=1 colspan=1>ATCC 19429</td><td rowspan=1 colspan=1>Radlett FebUnited Kingdom</td></tr><tr><td rowspan=1 colspan=1>ATCC 27853</td><td rowspan=1 colspan=1>41501Boston</td></tr><tr><td rowspan=1 colspan=1>CUSM PS28</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>NCTC 13437</td><td rowspan=1 colspan=1>United Kingdom</td></tr><tr><td rowspan=1 colspan=1>ATCC 9027</td><td rowspan=1 colspan=1>IFO 13275</td><td rowspan=1 colspan=1>Pseudomonas aeruginosaDetected(3.1E+05 CFU/mL)</td></tr><tr><td rowspan=1 colspan=1>ATCC 25619a</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>Not Detecteda</td></tr></table>

This isolate and a subset of P. aeruginosa sequences ${ < } 1 \%$ of total sequences) have critical mismatches to assay primer(s) and will not be detected

Table 97. Results for Stenotrophomonas maltophilia Isolates Tested   

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Source ID</td><td rowspan=1 colspan=1>Strain/Location/Year</td><td rowspan=1 colspan=1>Result</td></tr><tr><td rowspan=5 colspan=1>Stenotrophomonasmaltophilia</td><td rowspan=1 colspan=1>ATCC 700475</td><td rowspan=1 colspan=1>CIP 104854</td><td rowspan=5 colspan=1>StenotrophomonasmaltophiliaDetected</td></tr><tr><td rowspan=1 colspan=1>ATCC 700269</td><td rowspan=1 colspan=1>wwWashington/1993</td></tr><tr><td rowspan=1 colspan=1>ATCC BAA-84</td><td rowspan=1 colspan=1>D457Spain</td></tr><tr><td rowspan=1 colspan=1>ATCC 13637</td><td rowspan=1 colspan=1>Type strainUnited States/1961</td></tr><tr><td rowspan=1 colspan=1>ATCC BAA-2423</td><td rowspan=1 colspan=1>K279a</td></tr></table>

Table 98. Results for CTX-M Isolates Tested and Predicted Reactivity for CTX-M Typesa   

<table><tr><td rowspan=1 colspan=1>CTX-M Type</td><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Source ID</td><td rowspan=1 colspan=1>Result</td></tr><tr><td rowspan=1 colspan=1>CTX-M-1</td><td rowspan=1 colspan=1>Klebsiella oxytoca</td><td rowspan=1 colspan=1>GRE 1254054</td><td rowspan=7 colspan=1>CTX-MDetected</td></tr><tr><td rowspan=1 colspan=1>CTX-M-3</td><td rowspan=1 colspan=1>Shigella flexneri</td><td rowspan=1 colspan=1>CDC FDA AR BANK #0421</td></tr><tr><td rowspan=1 colspan=1>CTX-M-8</td><td rowspan=1 colspan=1>Klebsiella aerogenes</td><td rowspan=1 colspan=1>GRE 1254066</td></tr><tr><td rowspan=1 colspan=1>CTX-M-9</td><td rowspan=1 colspan=1>Enterobacter cloacae</td><td rowspan=1 colspan=1>CDC FDA AR Bank #0501</td></tr><tr><td rowspan=2 colspan=1>CTX-M-14</td><td rowspan=1 colspan=1>Escherichia coli</td><td rowspan=1 colspan=1>CDC FDA AR Bank #0086</td></tr><tr><td rowspan=1 colspan=1>Klebsiella pneumoniae</td><td rowspan=1 colspan=1>CDC FDA AR Bank #0079</td></tr><tr><td rowspan=1 colspan=1>CTX-M-14, -55</td><td rowspan=1 colspan=1>Escherichia coli</td><td rowspan=1 colspan=1>CDC FDA AR Bank #0349</td></tr></table>

<table><tr><td>CTX-M Type</td><td>Organism</td><td colspan="2">Source ID</td><td>Result</td></tr><tr><td rowspan="7">CTX-M-15</td><td>Escherichia coli</td><td colspan="2">CDC FDA AR BANK #0137</td></tr><tr><td>Klebsiella pneumoniae ssp. ozaenae</td><td colspan="2">CDC FDA AR Bank #0051</td></tr><tr><td>Klebsiella pneumoniae</td><td colspan="2">CDC FDA AR Bank #0068</td></tr><tr><td>Klebsiella pneumoniae</td><td colspan="2">CDC FDA AR Bank #0040</td></tr><tr><td>Serratia marcescens</td><td colspan="2">CDC FDA AR Bank #0517</td></tr><tr><td>Salmonella enterica ssp. enterica</td><td colspan="2">CDC FDA AR Bank #0407</td></tr><tr><td>Morganella morganii</td><td colspan="2">CDC FDA AR Bank #0057</td></tr><tr><td rowspan="2">CTX-M-22</td><td>Proteus mirabilis</td><td colspan="2">GRE 1254053</td></tr><tr><td>Klebsiella pneumoniae</td><td colspan="2">CDC FDA AR Bank #0107</td></tr><tr><td>CTX-M-25</td><td>Enterobacter cloacae</td><td colspan="2">NCTC 13464</td></tr><tr><td>CTX-M-55</td><td>Escherichia coli</td><td colspan="2">CDC FDA AR Bank #0346</td></tr><tr><td>CTX-M-124</td><td>Kluyvera ascorbata</td><td colspan="2">CDC FDA AR Bank #0144</td></tr><tr><td colspan="4">In silico Reactivity Predictions</td></tr><tr><td colspan="2">Detected</td><td>Not Detected</td><td colspan="2">Unknown (no sequence)</td></tr><tr><td>CTX-M-1 - CTX-M-69 CTX-M-71 - CTX-M-117</td><td>CTX-M-141 - CTX-M-142</td><td>CTX-M-151</td><td>CTX-M-70</td><td>CTX-M-140</td></tr><tr><td>CTX-M-121 - CTX-M-127</td><td>CTX-M-144</td><td></td><td>CTX-M-119</td><td>CTX-M-143</td></tr><tr><td></td><td>CTX-M-146 - CTX-M-148</td><td></td><td>CTX-M-120</td><td>CTX-M-145</td></tr><tr><td>CTX-M-129 - CTX-M-132</td><td>CTX-M-150</td><td></td><td>CTX-M-128</td><td>CTX-M-149</td></tr><tr><td>CTX-M-134</td><td>CTX-M-152</td><td></td><td>CTX-M-133</td><td>CTX-M-153</td></tr><tr><td>CTX-M-136 - CTX-M-139</td><td>CTX-M-155 - CTX-M-229</td><td></td><td></td><td>CTX-M-154</td></tr><tr><td></td><td></td><td></td><td>CTX-M-135</td><td></td></tr></table>

a Isolates were tested at a concentration near the LoD for CTX-M.

Table 99. Results for IMP Isolates Tested and Predicted Reactivity for IMP Typesa   

<table><tr><td rowspan=1 colspan=1>IMP Type</td><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Source ID</td><td rowspan=1 colspan=1>Result</td></tr><tr><td rowspan=1 colspan=1>IMP-1</td><td rowspan=1 colspan=1>Pseudomonas aeruginosa</td><td rowspan=1 colspan=1>CDC FDA AR Bank #0103</td><td rowspan=7 colspan=1>IMPDetected</td></tr><tr><td rowspan=1 colspan=1>IMP-4</td><td rowspan=1 colspan=1>Acinetobacter baumannii</td><td rowspan=1 colspan=1>GRE 1062081</td></tr><tr><td rowspan=1 colspan=1>IMP-4</td><td rowspan=1 colspan=1>Escherichia coli</td><td rowspan=1 colspan=1>GRE 1062016</td></tr><tr><td rowspan=1 colspan=1>IMP-4</td><td rowspan=1 colspan=1>Klebsiella pneumoniae</td><td rowspan=1 colspan=1>CDC FDA AR Bank #0080</td></tr><tr><td rowspan=1 colspan=1>IMP-4</td><td rowspan=1 colspan=1>Klebsiella aerogenes</td><td rowspan=1 colspan=1>CDC FDA AR Bank #0161</td></tr><tr><td rowspan=1 colspan=1>IMP-8</td><td rowspan=1 colspan=1>Klebsiella pneumoniae</td><td rowspan=1 colspan=1>GRE 1062084</td></tr><tr><td rowspan=1 colspan=1>IMP-14</td><td rowspan=1 colspan=1>Pseudomonas aeruginosa</td><td rowspan=1 colspan=1>CDC FDA AR Bank #0092</td></tr><tr><td rowspan=1 colspan=3>In silico Reactivity Predictions</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=2>Detected</td><td rowspan=1 colspan=1>Not Detected</td><td rowspan=1 colspan=1>Unknown(no sequences)</td></tr><tr><td rowspan=1 colspan=2>IMP-1 - IMP-30b                IMP-58 - IMP-62IMP-32 - IMP-34                     IMP-64IMP-37 - IMP-45                IMP-66  IMP-84IMP-48 - IMP-49IMP-51 - IMP-56</td><td rowspan=1 colspan=1>IMP-10bIMP-31IMP-35IMP-46IMP-63</td><td rowspan=1 colspan=1>IMP-36IMP-47IMP-50IMP-57IMP-65</td></tr></table>

a Isolates were tested at a concentration near the LoD for IMP b A subset of IMP-10 sequences (1/6) have mismatches to the assay primer(s) that may have an impact on detection

Table 100. Results for KPC Isolates Tested and Predicted Reactivity for KPC Typesa   

<table><tr><td colspan="1" rowspan="1">KPC Type</td><td colspan="1" rowspan="1">Organism</td><td colspan="1" rowspan="1">Source ID</td><td colspan="1" rowspan="1">Strain/Location/Year</td><td colspan="1" rowspan="1">Result</td></tr><tr><td colspan="1" rowspan="4">KPC-2</td><td colspan="1" rowspan="1">Klebsiella oxytoca</td><td colspan="1" rowspan="1">JMI 7818</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="16">KPCDetected</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">ATCC BAA-1705</td><td colspan="1" rowspan="1">ART 2008133</td></tr><tr><td colspan="1" rowspan="1">Proteus mirabilis</td><td colspan="1" rowspan="1">CDC FDA AR Bank #0156</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Serratia marcescens</td><td colspan="1" rowspan="1">JMI 697</td><td colspan="1" rowspan="1">•</td></tr><tr><td colspan="1" rowspan="5">KPC-3</td><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">CDC FDA AR Bank #0061</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniaessp. ozaenae</td><td colspan="1" rowspan="1">CDC FDA AR Bank #0096</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Klebsiella oxytoca</td><td colspan="1" rowspan="1">CDC FDA AR Bank #0147</td><td colspan="1" rowspan="1">•</td></tr><tr><td colspan="1" rowspan="1">Serratia marcescens</td><td colspan="1" rowspan="1">CDC FDA AR Bank #0517</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Kluyvera ascorbata</td><td colspan="1" rowspan="1">CDC FDA AR Bank #0144</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">KPC-4</td><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">JMI 766</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">KPC-5</td><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">Creighton University PS28</td><td colspan="1" rowspan="1">•</td></tr><tr><td colspan="1" rowspan="1">KPC-6</td><td colspan="1" rowspan="1">Proteus mirabilis</td><td colspan="1" rowspan="1">CDC FDA AR Bank #0155</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">KPC-11</td><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">CDC FDA AR Bank #0525</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="3">Unknown</td><td colspan="1" rowspan="1">Enterobacter cloacae</td><td colspan="1" rowspan="1">ATCC BAA-2341</td><td colspan="1" rowspan="1">1101152</td></tr><tr><td colspan="1" rowspan="1">Enterobacter hormaechei</td><td colspan="1" rowspan="1">ATCC BAA-2082</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Klebsiella oxytoca</td><td colspan="1" rowspan="1">JMI 2523</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="5" rowspan="1">In silico Reactivity PredictionsDetected</td></tr><tr><td colspan="5">KPC-1 - KPC-46</td></tr></table>

a Isolates were tested at a concentration near the LoD for KPC

Table 101. Results for mcr-1 Isolates Testeda   

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Source ID</td><td rowspan=1 colspan=1>Result</td></tr><tr><td rowspan=4 colspan=1>Escherichia coli</td><td rowspan=1 colspan=1>CDC FDA AR Bank #0346</td><td rowspan=5 colspan=1>mcr-1Detected</td></tr><tr><td rowspan=1 colspan=1>CDC FDA AR Bank #0349</td></tr><tr><td rowspan=1 colspan=1>CDC FDA AR Bank #0350</td></tr><tr><td rowspan=1 colspan=1>CDC FDA AR Bank #0495</td></tr><tr><td rowspan=1 colspan=1>Klebsiella pneumoniae</td><td rowspan=1 colspan=1>CDC FDA AR Bank #0497</td></tr></table>

a Isolates were tested at a concentration near the LoD for mcr-1

Table 102. Results for mecA/C in Methicillin-resistant Staphylococcus epidermidis Isolates Testeda,b   

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Source ID</td><td rowspan=1 colspan=1>Strain/Location/Year</td><td rowspan=1 colspan=1>Result</td></tr><tr><td rowspan=5 colspan=1>Staphylococcus epidermidis(MRSE)</td><td rowspan=1 colspan=1>ATCC 29887</td><td rowspan=1 colspan=1>255-01B</td><td rowspan=5 colspan=1>mecA/CDetected</td></tr><tr><td rowspan=1 colspan=1>ATCC 35983</td><td rowspan=1 colspan=1>RP12Tennessee</td></tr><tr><td rowspan=1 colspan=1>ATCC 35984</td><td rowspan=1 colspan=1>RP62ATennessee</td></tr><tr><td rowspan=1 colspan=1>ATCC 51625</td><td rowspan=1 colspan=1>CCF 15990Ohio</td></tr><tr><td rowspan=1 colspan=1>ATCC 700562</td><td rowspan=1 colspan=1>1191Virginia/1997</td></tr></table>

a No methicillin-resistant isolates of Staphylococcus lugdunensis were available for testing b Isolates were tested at a concentration near the LoD for mecA/C

Table 103. Results for mecA/C and MREJ (MRSA) in Staphylococcus aureus Isolates Tested and Predicted Reactivity for MREJ Typesa   

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Source ID b</td><td rowspan=1 colspan=1>Strain/Location/Year</td><td rowspan=1 colspan=2>SCCmec Type/MREJ Type</td><td rowspan=1 colspan=1>Result</td></tr><tr><td rowspan=31 colspan=1>Staphylococcus aureus</td><td rowspan=1 colspan=1>ATCC BAA-2421c</td><td rowspan=1 colspan=1>Mass/2010</td><td rowspan=1 colspan=2>SCCmec Type II</td><td rowspan=31 colspan=1>mecA/C and MREJ(MRSA)Detected</td></tr><tr><td rowspan=1 colspan=1>NARSA NRS705</td><td rowspan=1 colspan=1>NY-12</td><td rowspan=1 colspan=2>SCCmec Type II</td></tr><tr><td rowspan=1 colspan=1>NARSA NRS701</td><td rowspan=1 colspan=1>MN-082</td><td rowspan=1 colspan=2>SCCmec Type II</td></tr><tr><td rowspan=1 colspan=1>ATCC BAA-1717</td><td rowspan=1 colspan=1>TCH1516</td><td rowspan=1 colspan=2>SCCmec Type IV</td></tr><tr><td rowspan=1 colspan=1>NARSA NRS683</td><td rowspan=1 colspan=1>GA-298</td><td rowspan=1 colspan=2>SCCmec Type IV</td></tr><tr><td rowspan=1 colspan=1>NARSA NRS662</td><td rowspan=1 colspan=1>CO-34</td><td rowspan=1 colspan=2>SCCmec Type IV</td></tr><tr><td rowspan=1 colspan=1>NARSA NRS707</td><td rowspan=1 colspan=1>NY-155</td><td rowspan=1 colspan=2>SCCmec Type IV</td></tr><tr><td rowspan=1 colspan=1>ATCC BAA-1707</td><td rowspan=1 colspan=1>MW2</td><td rowspan=1 colspan=2>SCCmec Type IV</td></tr><tr><td rowspan=1 colspan=1>NARSA NRS691</td><td rowspan=1 colspan=1>GA-62</td><td rowspan=1 colspan=2>SCCmec Type IV</td></tr><tr><td rowspan=1 colspan=1>NARSA NRS648</td><td rowspan=1 colspan=1>CA-347</td><td rowspan=1 colspan=2>SCCmec Type II or IV</td></tr><tr><td rowspan=1 colspan=1>NARSA NRS689</td><td rowspan=1 colspan=1>GA-442</td><td rowspan=1 colspan=2>SCCmec Type IV</td></tr><tr><td rowspan=1 colspan=1>NARSA NRS668</td><td rowspan=1 colspan=1>CO-72</td><td rowspan=1 colspan=2>SCCmec Type IV</td></tr><tr><td rowspan=1 colspan=1>ATCC BAA-1700</td><td rowspan=1 colspan=1>HFH-33798</td><td rowspan=1 colspan=2>SCCmec Type IV</td></tr><tr><td rowspan=1 colspan=1>BEI NR-46081(NARSA NRS484)</td><td rowspan=1 colspan=1>HIP12899</td><td rowspan=1 colspan=2>SCCmec Type IV</td></tr><tr><td rowspan=1 colspan=1>ATCC BAA-1691</td><td rowspan=1 colspan=1>HFH-30137</td><td rowspan=1 colspan=2>SCCmec Type IV</td></tr><tr><td rowspan=1 colspan=1>ATCC 43300</td><td rowspan=1 colspan=1>F182 Kansas</td><td rowspan=1 colspan=2>SCCmec Type II</td></tr><tr><td rowspan=1 colspan=1>ATCC BAA-2422</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=2>SCCmec Type II</td></tr><tr><td rowspan=1 colspan=1>ATCC BAA-1720</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=2>SCCmec Type II</td></tr><tr><td rowspan=1 colspan=1>NARSA NRS745</td><td rowspan=1 colspan=1>CA-629</td><td rowspan=1 colspan=2>SCCmec Type IV or V</td></tr><tr><td rowspan=1 colspan=1>ATCC BAA-38</td><td rowspan=1 colspan=1>-</td><td rowspan=2 colspan=2>MREJ Type i</td></tr><tr><td rowspan=1 colspan=1>NARSA NRS686</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>ATCC BAA-44</td><td rowspan=1 colspan=1>-</td><td rowspan=4 colspan=2>MREJ Type ii</td></tr><tr><td rowspan=1 colspan=1>ATCC BAA-41</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>NARSA NRS385</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>ATCC BAA-42</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>ATCC BAA-39</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=2>MREJ Type iii</td></tr><tr><td rowspan=1 colspan=1>ACC BAA-40</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=2>MREJ Type iv</td></tr><tr><td rowspan=1 colspan=1>GRE 1062264</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=2>MREJ Type iv</td></tr><tr><td rowspan=1 colspan=1>GRE 0759084</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=2>MREJ Type v</td></tr><tr><td rowspan=1 colspan=1>GRE 1055015</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=2>MREJ Type vi</td></tr><tr><td rowspan=1 colspan=1>GRE 0860042</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=2>MREJ Type vii</td></tr></table>

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Source ID b</td><td rowspan=1 colspan=1>Strain/Location/Year</td><td rowspan=1 colspan=1>SCCmec Type/MREJ Type</td><td rowspan=1 colspan=1>Result</td></tr><tr><td rowspan=8 colspan=1></td><td rowspan=1 colspan=1>GRE 1052034</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>MREJ Type ix</td><td rowspan=8 colspan=1></td></tr><tr><td rowspan=1 colspan=1>GRE 1151100</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>MREJ Type xi</td></tr><tr><td rowspan=1 colspan=1>GRE 0960006</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>MREJ Type xii</td></tr><tr><td rowspan=1 colspan=1>GRE 1055017</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>MREJ Type xiii</td></tr><tr><td rowspan=1 colspan=1>GRE 0759163</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>MREJ Type xiv</td></tr><tr><td rowspan=1 colspan=1>GRE 1057114</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>MREJ Type xvii</td></tr><tr><td rowspan=1 colspan=1>ATCC BAA-2313d</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>SCCmec Type XI</td></tr><tr><td rowspan=1 colspan=1>ATCC BAA-2312d</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>SCCmec Type XI</td></tr><tr><td rowspan=4 colspan=1>Staphylococcus aureus</td><td rowspan=1 colspan=1>Rennes 1060728</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Empty SCCmec cassette</td><td rowspan=4 colspan=1>Not Detected</td></tr><tr><td rowspan=1 colspan=1>GRE 1062519e</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>MREJ Type xix</td></tr><tr><td rowspan=1 colspan=1>GRE 1062373</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>MREJ Type XV</td></tr><tr><td rowspan=1 colspan=1>GRE 1062292f</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>MREJ Type xviii</td></tr><tr><td rowspan=1 colspan=4>In silico Reactivity Predictions for MREJ Types</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=3>Detected</td><td rowspan=1 colspan=1>Not Detected</td><td rowspan=1 colspan=1>Unknown(no sequence)</td></tr><tr><td rowspan=1 colspan=3>MREJ Type I,ia          MREJ Type vi          MREJ Type xiiiMREJ Type ii/xvi         MREJ Type vii         MREJ Type xivMREJ Type iii           MREJ Type ix9         MREJ Type xviMREJ Type iv           MREJ Type xi         MREJ Type xviiMREJ Type V           MREJ Type xii         MREJ Type xxi</td><td rowspan=1 colspan=1>MREJ Type ix9MREJ Type xvMREJ Type xviiiMREJ Type xix</td><td rowspan=1 colspan=1>MREJ Type viiiMREJ Type xMREJ Type xx</td></tr></table>

a Isolates were tested at a concentration near the LoD for Staphylococcus aureus b NARSA/BEI isolates were sourced by the Network on Antimicrobial Resistance in Staphylococcus aureus (NARSA) for distribution by BEI Resources, NIAID, NIH c Isolate is characterized as methicillin-sensitive Staphylococcus aureus (MSSA) with a non-functional mecA variant that is amplified by the mecA/C assay d Isolate is characterized as methicillin-resistant Staphylococcus aureus (MRSA) with the mecC gene e Isolate is characterized as MSSA, which matches the mecA/C and MREJ (MRSA) Not Detected result f Isolate is characterized as MRSA, but the MREJ type is not detected by the assay g A subset of MREJ Type ix sequences (2/8) have mismatches to the assay primer(s) that may have an impact on detection

Table 104. Results for NDM Isolates Tested and Predicted Reactivity for NDM Typesa   

<table><tr><td rowspan=1 colspan=1>NDM Type</td><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Source ID</td><td rowspan=1 colspan=1>Result</td></tr><tr><td rowspan=4 colspan=1>NDM-1</td><td rowspan=1 colspan=1>Acinetobacter baumannii</td><td rowspan=1 colspan=1>CDC FDA AR Bank #0033</td><td rowspan=8 colspan=1>NDMDetected</td></tr><tr><td rowspan=1 colspan=1>Salmonella enterica</td><td rowspan=1 colspan=1>CDC FDA AR Bank #0127</td></tr><tr><td rowspan=1 colspan=1>Klebsiella pneumoniae</td><td rowspan=1 colspan=1>CDC FDA AR Bank #0068</td></tr><tr><td rowspan=1 colspan=1>Proteus mirabilis</td><td rowspan=1 colspan=1>CDC FDA AR Bank #0159</td></tr><tr><td rowspan=1 colspan=1>NDM-2</td><td rowspan=1 colspan=1>Acinetobacter baumannii</td><td rowspan=1 colspan=1>GRE 1153064</td></tr><tr><td rowspan=1 colspan=1>NDM-5</td><td rowspan=3 colspan=1>Escherichia coli</td><td rowspan=1 colspan=1>CDC FDA AR Bank #0150</td></tr><tr><td rowspan=1 colspan=1>NDM-6</td><td rowspan=1 colspan=1>CDC FDA AR Bank #0137</td></tr><tr><td rowspan=1 colspan=1>NDM-7</td><td rowspan=1 colspan=1>CDC FDA AR Bank #0149</td></tr><tr><td rowspan=1 colspan=4>In silico Reactivity Predictions</td></tr><tr><td rowspan=1 colspan=2>Detected</td><td rowspan=1 colspan=2>Not Detected</td></tr><tr><td rowspan=1 colspan=2>NDM-1 - NDM-24b,c,d                    NDM-32NDM-27 - NDM-29                     NDM-40</td><td rowspan=1 colspan=2>NDM-1b                  NDM-18dNDM-4c</td></tr></table>

a Isolates were tested at a concentration near the LoD for NDM b A subset of NDM-1 sequences (5/753) have mismatches to assay primer(s) that may have an impact on detection c A subset of NDM-4 sequences (1/57) have mismatches to assay primer(s) that may have an impact on detection d A subset of NDM-18 sequences (1/3) have mismatches to assay primer(s) that may have an impact on detection

Table 105. Results for OXA-48-like Isolates Tested and Predicted Reactivity for OXA-48-like Typesa   

<table><tr><td rowspan=1 colspan=2>OXA-48 Like Type</td><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Source ID</td><td rowspan=1 colspan=3>Result</td></tr><tr><td rowspan=1 colspan=2>OXA-48</td><td rowspan=1 colspan=1>Klebsiella aerogenes</td><td rowspan=1 colspan=1>CDC FDA AR Bank #0074</td><td rowspan=5 colspan=3>OXA-48-likeDetected</td></tr><tr><td rowspan=1 colspan=2>OXA-48-like</td><td rowspan=1 colspan=1>Serratia marcescens</td><td rowspan=1 colspan=1>GRE 1659004</td></tr><tr><td rowspan=1 colspan=2>OXA-162</td><td rowspan=1 colspan=1>Klebsiella pneumoniae</td><td rowspan=1 colspan=1>GRE 1355030</td></tr><tr><td rowspan=1 colspan=2>OXA-181</td><td rowspan=1 colspan=1>Klebsiella pneumoniae</td><td rowspan=1 colspan=1>CDC FDA AR Bank #0051</td></tr><tr><td rowspan=1 colspan=2>OXA-232</td><td rowspan=1 colspan=1>Klebsiella pneumoniae</td><td rowspan=1 colspan=1>CDC FDA AR Bank #0068</td></tr><tr><td rowspan=1 colspan=2></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=4>In silico Reactivity Predictions</td></tr><tr><td rowspan=1 colspan=2></td><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=4>Not Detectedb</td></tr><tr><td rowspan=1 colspan=2>OXA-48</td><td rowspan=1 colspan=1>OXA-244              OXA-515</td><td rowspan=3 colspan=4>OXA-54c              OXA-439d                OXA-551dOXA-163d             OXA-517d               OXA-552dOXA-247d             OXA-535d                OXA-553dOXA-405d             OXA-538d               OXA-567d</td></tr><tr><td rowspan=1 colspan=2>OXA-48-like</td><td rowspan=1 colspan=1>OXA-245              OXA-519</td></tr><tr><td rowspan=1 colspan=2>OXA-162OXA-181</td><td rowspan=1 colspan=1>OXA-252             OXA-546OXA-370              OXA-547</td><td rowspan=3 colspan=4>OXA-247d             OXA-535d                OXA-553dOXA-405d             OXA-538d               OXA-567dOXA-416c             OXA-548d                OXA-731dOXA-436d             OXA-549dOXA-438d             OXA-550d</td></tr><tr><td rowspan=1 colspan=2>OXA-199OXA-204</td><td rowspan=1 colspan=1>OXA-484             OXA-566</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>OXA-204OXA-232</td><td rowspan=1 colspan=1>OXA-505OXA-514</td></tr></table>

a Isolates were tested at a concentration near the LoD for OXA-48-like

b Non-OXA-48-like types (e.g. OXA-23-like, OXA-40/24-like, OXA-51-like, and OXA-58-like, OXA-143a-like, and OXA-143-like) will not be detected c OXA-48-like progenitor found in Shewanella species. The OXAa assay was designed to exclude these types d OXA-48-like types with altered carbapenem hydrolysis activity. The OXAa assay was designed to exclude these variants

Table 106. Results for vanA/B in Vancomycin-resistant Enterococcus faecium and Enterococcus faecalis Isolates Testeda   

<table><tr><td rowspan=1 colspan=1>van Gene</td><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Source ID</td><td rowspan=1 colspan=1>Strain/Location/Year</td><td rowspan=1 colspan=1>Result</td></tr><tr><td rowspan=6 colspan=1>vanA</td><td rowspan=3 colspan=1>Enterococcus faecium</td><td rowspan=1 colspan=1>ATCC 700221</td><td rowspan=1 colspan=1>-</td><td rowspan=8 colspan=1>vanA/BDetected</td></tr><tr><td rowspan=1 colspan=1>JMI 475</td><td rowspan=1 colspan=1>IN/2003</td></tr><tr><td rowspan=1 colspan=1>ATCC BAA-2318</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=3 colspan=1>Enterococcus faecalis</td><td rowspan=1 colspan=1>ATCC BAA-2573</td><td rowspan=1 colspan=1>bMx 0502240</td></tr><tr><td rowspan=1 colspan=1>JMI 12536</td><td rowspan=1 colspan=1>MA/2002</td></tr><tr><td rowspan=1 colspan=1>ATCC 700802</td><td rowspan=1 colspan=1>V583</td></tr><tr><td rowspan=2 colspan=1>vanB</td><td rowspan=1 colspan=1>Enterococcus faecalis</td><td rowspan=1 colspan=1>ATCC 51299</td><td rowspan=1 colspan=1>NJ-3</td></tr><tr><td rowspan=1 colspan=1>Enterococcus faecium</td><td rowspan=1 colspan=1>ATCC 51858</td><td rowspan=1 colspan=1>Vancomycin-dependent #4</td></tr></table>

a Isolates were tested at a concentration near the LoD for Enterococcus faecium and Enterococcus faecalis

Table 107. Results for VIM Isolates Tested and Predicted Reactivity for VIM Typesa   

<table><tr><td rowspan=1 colspan=1>VIM Type</td><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Source ID</td><td rowspan=1 colspan=1>Result</td></tr><tr><td rowspan=3 colspan=1>VIM-1</td><td rowspan=1 colspan=1>Enterobacter cloacae</td><td rowspan=1 colspan=1>CDC FDA AR Bank #0154</td><td rowspan=9 colspan=1>VIMDetected</td></tr><tr><td rowspan=1 colspan=1>Klebsiella pneumoniae</td><td rowspan=1 colspan=1>CDC FDA AR Bank #0076</td></tr><tr><td rowspan=1 colspan=1>Enterobacter cloacae</td><td rowspan=1 colspan=1>CDC FDA AR Bank #0501</td></tr><tr><td rowspan=1 colspan=1>VIM-2</td><td rowspan=1 colspan=1>Pseudomonas aeruginosa</td><td rowspan=1 colspan=1>CDC FDA AR Bank #0100</td></tr><tr><td rowspan=1 colspan=1>VIM-4</td><td rowspan=1 colspan=1>Pseudomonas aeruginosa</td><td rowspan=1 colspan=1>CDC FDA AR-BANK #0054</td></tr><tr><td rowspan=1 colspan=1>VIM-7</td><td rowspan=1 colspan=1>Escherichia coli</td><td rowspan=1 colspan=1>GRE 1256018</td></tr><tr><td rowspan=1 colspan=1>VIM-10</td><td rowspan=1 colspan=1>Pseudomonas aeruginosa</td><td rowspan=1 colspan=1>NCTC 13437</td></tr><tr><td rowspan=1 colspan=1>VIM-11</td><td rowspan=1 colspan=1>Pseudomonas aeruginosa</td><td rowspan=1 colspan=1>CDC FDA AR Bank #0239</td></tr><tr><td rowspan=1 colspan=1>VIM-27</td><td rowspan=1 colspan=1>Klebsiella pneumoniae</td><td rowspan=1 colspan=1>CDC FDA AR Bank #0040</td></tr><tr><td rowspan=2 colspan=4>In silico Reactivity PredictionsDetected                                                                                           UnknownNot Detected(no sequences)</td></tr><tr><td rowspan=1 colspan=1>Not Detected</td><td rowspan=1 colspan=1>Unknown(no sequences)</td></tr><tr><td rowspan=1 colspan=2>VIM-1 - VIM-20b             VIM-47 - VIM-64VIM-23 - VIM-38                 VIM-66VIM-40 - VIM-44</td><td rowspan=1 colspan=1>VIM-2b                   VIM-46VIM-39                  VIM-65VIM-45                  VIM-67</td><td rowspan=1 colspan=1>VIM-21VIM-22</td></tr></table>

a Isolates were tested at a concentration near the LoD for VIM b A subset of VIM-2 sequences $( 3 / 1 8 2 )$ have mismatches to assay primer(s) that may have an impact on detection

Table 108. Results for Candida albicans Isolates Tested   

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Source ID</td><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>Result</td></tr><tr><td rowspan=6 colspan=1>Candida albicans</td><td rowspan=1 colspan=1>ATCC 11006</td><td rowspan=1 colspan=1>-</td><td rowspan=6 colspan=1>Candida albicansDetected</td></tr><tr><td rowspan=1 colspan=1>ATCC 14053</td><td rowspan=1 colspan=1>NIH 3172</td></tr><tr><td rowspan=1 colspan=1>ATCC 22972</td><td rowspan=1 colspan=1>M 97</td></tr><tr><td rowspan=1 colspan=1>ATCC 10231</td><td rowspan=1 colspan=1>DSM 1386</td></tr><tr><td rowspan=1 colspan=1>ATCC 90028</td><td rowspan=1 colspan=1>NCCLS 11</td></tr><tr><td rowspan=1 colspan=1>ATCC MYA-2876</td><td rowspan=1 colspan=1>SC5314</td></tr></table>

# Note: The assay for detection of C. albicans amplifies a gene within the mitochondrial genome and ‘petite’ strains that have lost mitochondrial DNA will not be detected.

Table 109. Results for Candida auris Isolates Testeda   

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Source ID</td><td rowspan=1 colspan=1>Strain/Location/Year</td><td rowspan=1 colspan=1>Result</td></tr><tr><td rowspan=7 colspan=1>Candida auris</td><td rowspan=1 colspan=1>CDC FDA AR Bank #0381</td><td rowspan=1 colspan=1>-</td><td rowspan=7 colspan=1>Candida aurisDetected</td></tr><tr><td rowspan=1 colspan=1>CDC FDA AR Bank #0383</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>CDC FDA AR Bank #0384</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>CDC FDA AR Bank #0385</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>CDC FDA AR Bank #0388</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>NCPF 8971</td><td rowspan=1 colspan=1>Strain 10 (non-aggregative)United Kingdom/2016</td></tr><tr><td rowspan=1 colspan=1>NCPF 8977</td><td rowspan=1 colspan=1>Strain 2 (aggregative)United Kingdom/2016</td></tr></table>

a A subset of Candida auris sequences (2/151) have mismatches to assay primer(s) that may have an impact on detection

Table 110. Results for Candida glabrata Isolates Tested   

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Source ID</td><td rowspan=1 colspan=1>Strain/Year</td><td rowspan=1 colspan=1>Result</td></tr><tr><td rowspan=6 colspan=1>Candida glabrata</td><td rowspan=1 colspan=1>ATCC 2001</td><td rowspan=1 colspan=1>Type strain</td><td rowspan=5 colspan=1>Candida glabrataDetected</td></tr><tr><td rowspan=1 colspan=1>ATCC 15545</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>ATCC 15126</td><td rowspan=1 colspan=1>Mutant TMAGR-23</td></tr><tr><td rowspan=1 colspan=1>ATCC 32554</td><td rowspan=1 colspan=1>26247-1</td></tr><tr><td rowspan=1 colspan=1>ATCC MYA-2950</td><td rowspan=1 colspan=1>303542</td></tr><tr><td rowspan=1 colspan=1>Clinical Isolatea</td><td rowspan=1 colspan=1>2018</td><td rowspan=1 colspan=1>Not Detected</td></tr></table>

a Isolate characterized as ‘petite mutant’ with loss of mitochondrial function/mitochondrial DNA

# Note: The assay for detection of C. glabrata amplifies a gene within the mitochondrial genome and ‘petite’ strains that have lost mitochondrial DNA will not be detected.

Table 111. Results for Candida krusei Isolates Tested   

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Source ID</td><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>Result</td></tr><tr><td rowspan=6 colspan=1>Candida krusei</td><td rowspan=1 colspan=1>ATCC 6258</td><td rowspan=1 colspan=1>Type strain</td><td rowspan=6 colspan=1>Candida kruseiDetected</td></tr><tr><td rowspan=1 colspan=1>ATCC 14243</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>ATCC 28870</td><td rowspan=1 colspan=1>CBS 2052</td></tr><tr><td rowspan=1 colspan=1>ATCC 34135</td><td rowspan=1 colspan=1>ST-112</td></tr><tr><td rowspan=1 colspan=1>ATCC 90878</td><td rowspan=1 colspan=1>B74</td></tr><tr><td rowspan=1 colspan=1>ATCC 201748</td><td rowspan=1 colspan=1>89-08-008</td></tr></table>

Table 112. Results for Candida parapsilosis Isolates Tested   

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Source ID</td><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>Result</td></tr><tr><td rowspan=5 colspan=1>Candida parapsilosis</td><td rowspan=1 colspan=1>ATCC 34136</td><td rowspan=1 colspan=1>ST-89</td><td rowspan=5 colspan=1>Candida parapsilosisDettected</td></tr><tr><td rowspan=1 colspan=1>ATCC 22019</td><td rowspan=1 colspan=1>Type strain</td></tr><tr><td rowspan=1 colspan=1>ATCC 28475</td><td rowspan=1 colspan=1>CBS 2915</td></tr><tr><td rowspan=1 colspan=1>ATCC 90875</td><td rowspan=1 colspan=1>B78</td></tr><tr><td rowspan=1 colspan=1>ATCC 96138</td><td rowspan=1 colspan=1>MC0433</td></tr></table>

# Note: The assay for detection of C. parapsilosis amplifies a gene within the mitochondrial genome and ‘petite’ strains that have lost mitochondrial DNA will not be detected.

Table 113. Results for Candida tropicalis Isolates Tested   

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Source ID</td><td rowspan=1 colspan=1>Strain/Location</td><td rowspan=1 colspan=1>Result</td></tr><tr><td rowspan=5 colspan=1>Candida tropicalis</td><td rowspan=1 colspan=1>ATCC 750</td><td rowspan=1 colspan=1>Type strain</td><td rowspan=5 colspan=1>Candida tropicalisDetected</td></tr><tr><td rowspan=1 colspan=1>ATCC 66029</td><td rowspan=1 colspan=1>AmMS 227</td></tr><tr><td rowspan=1 colspan=1>ATCC 90874</td><td rowspan=1 colspan=1>B79Germany</td></tr><tr><td rowspan=1 colspan=1>ATCC 201380</td><td rowspan=1 colspan=1>API 90 01 105</td></tr><tr><td rowspan=1 colspan=1>ATCC MYA-2734</td><td rowspan=1 colspan=1>509-12.1</td></tr></table>

# Note: The assay for detection of C. tropicalis amplifies a gene within the mitochondrial genome and ‘petite’ strains that have lost mitochondrial DNA will not be detected.

Table 114. Results for Cryptococcus neoformans/gattii Isolates Tested   

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Serotype/Genotypea</td><td rowspan=1 colspan=1>Source ID</td><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>Result</td></tr><tr><td rowspan=6 colspan=1>Cryptococcus gattii</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>ATCC MYA-4071</td><td rowspan=1 colspan=1>WM 276</td><td rowspan=13 colspan=1>Cryptococcusneoformans/gattiiDetected</td></tr><tr><td rowspan=1 colspan=1>B/VGIlb</td><td rowspan=1 colspan=1>ATCC MYA-4094</td><td rowspan=1 colspan=1>R272</td></tr><tr><td rowspan=1 colspan=1>B/VGI</td><td rowspan=1 colspan=1>ATCC MYA-4560</td><td rowspan=1 colspan=1>WM179</td></tr><tr><td rowspan=1 colspan=1>B/VGIII</td><td rowspan=1 colspan=1>ATCC MYA-4562</td><td rowspan=1 colspan=1>WM161</td></tr><tr><td rowspan=1 colspan=1>C/VGIV</td><td rowspan=1 colspan=1>ATCC MYA-4563</td><td rowspan=1 colspan=1>WM779</td></tr><tr><td rowspan=1 colspan=1>VGlIc</td><td rowspan=1 colspan=1>ATCC MYA-4877</td><td rowspan=1 colspan=1>A6MR38</td></tr><tr><td rowspan=7 colspan=1>Cryptococcus neoformans</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>ATCC 24067</td><td rowspan=1 colspan=1>52</td></tr><tr><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>ATCC 32045</td><td rowspan=1 colspan=1>Type strain</td></tr><tr><td rowspan=1 colspan=1>A/VNII</td><td rowspan=1 colspan=1>ATCC MYA-4564</td><td rowspan=1 colspan=1>WM148</td></tr><tr><td rowspan=1 colspan=1>AD/VNIII</td><td rowspan=1 colspan=1>ATCC MYA-4566</td><td rowspan=1 colspan=1>WM628</td></tr><tr><td rowspan=1 colspan=1>D/VNIV</td><td rowspan=1 colspan=1>ATCC MYA-4567</td><td rowspan=1 colspan=1>WM629</td></tr><tr><td rowspan=1 colspan=1>B</td><td rowspan=1 colspan=1>ATCC 24065</td><td rowspan=1 colspan=1>112</td></tr><tr><td rowspan=1 colspan=1>A</td><td rowspan=1 colspan=1>ATCC 208821b</td><td rowspan=1 colspan=1>Type strain</td></tr></table>

# Analytical Specificity (Cross-Reactivity and Exclusivity)

The potential for non-specific amplification and detection by the BioFire BCID2 Panel assays was evaluated by in silico analysis of available sequences and by testing of high concentrations of on-panel and off-panel organisms. Each organism was tested in triplicate with most bacteria tested at a concentration ${ > } 1 . 0 \mathsf { E } { + } 0 9 $ CFU/mL and most yeast tested at a concentration ${ > } 1 . 0 \mathsf { E } { + } 0 8 $ CFU/mL. Off-panel fungi, viruses, and parasites were tested at the highest cultured concentration possible.

The on-panel organisms tested to assess the potential for intra-panel cross-reactivity are listed in Table 115, with each observed or predicted cross-reactivity indicated. The off-panel organisms tested are listed in Table 116 and the list includes species genetically related to organisms or AMR genes detected by the panel (same genus or otherwise related) as well as unrelated organisms that may be found in blood and/or blood cultures as pathogens or contaminants (e.g. skin microorganisms, viruses, etc.). Off-panel AMR genes were also evaluated, and all observed or predicted cross-reactivities are indicated. Erroneous results due to cross-reactivity with organisms that were not evaluated or due to cross-reactivity with emerging or novel sequences are also possible.

Table 115. On-Panel Organisms and AMR Genes Tested for Evaluation of BioFire BCID2 Panel Analytical Specificity A risk of cross-reactivity was confirmed for the species in bold.   

<table><tr><td colspan="4" rowspan="1">ON-PANEL</td></tr><tr><td colspan="4" rowspan="1">Gram Positive Bacteria</td></tr><tr><td colspan="1" rowspan="1">Enterococcus faecalis</td><td colspan="1" rowspan="1">Staphylococcus intermedius</td><td colspan="1" rowspan="1">Staphylococcus xylosus</td><td colspan="1" rowspan="1">Streptococcus mitis</td></tr><tr><td colspan="1" rowspan="1">Enterococcus faecium</td><td colspan="1" rowspan="1">Staphylococcus lentus</td><td colspan="1" rowspan="1">Streptococcus agalactiae</td><td colspan="1" rowspan="1">Streptococcus mutans</td></tr><tr><td colspan="1" rowspan="1">Listeria monocytogenes</td><td colspan="1" rowspan="1">Staphylococcus lugdunensis</td><td colspan="1" rowspan="1">Streptococcus anginosus</td><td colspan="1" rowspan="1">Streptococcus oralis</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus argenteus²</td><td colspan="1" rowspan="1">Staphylococcus lutrae</td><td colspan="1" rowspan="1">Streptococcus australis</td><td colspan="1" rowspan="1">Streptococcus parasanguinis</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus (MRSA)</td><td colspan="1" rowspan="1">Staphylococcus nepalensis</td><td colspan="1" rowspan="1">Streptococcus bovis</td><td colspan="1" rowspan="1">Streptococcus pneumoniae</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus auricularis</td><td colspan="1" rowspan="1">Staphylococcus pasteuri</td><td colspan="1" rowspan="1">Streptococcus canis</td><td colspan="1" rowspan="1">Streptococcus pseudopneumoniae</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus capitis</td><td colspan="1" rowspan="1">Staphylococcus pettenkoferi</td><td colspan="1" rowspan="1">Streptococcus constellatus</td><td colspan="1" rowspan="1">Streptococcus pyogenes</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus caprae</td><td colspan="1" rowspan="1">Staphylococcus pseudintermedius</td><td colspan="1" rowspan="1">Streptococcus cristatus</td><td colspan="1" rowspan="1">Streptococcus salivarius</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus carnosus</td><td colspan="1" rowspan="1">Staphylococcus saprophyticus</td><td colspan="1" rowspan="1">Streptococcous dysgalactiae</td><td colspan="1" rowspan="1">Streptococcus sanguinis</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus cohnii</td><td colspan="1" rowspan="1">Staphylococcus schleiferi</td><td colspan="1" rowspan="1">Streptococcus equi</td><td colspan="1" rowspan="1">Streptococcus sobrinus</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus epidermidis (MRSE)</td><td colspan="1" rowspan="1">Staphylococcus schweitzer</td><td colspan="1" rowspan="1">Streptococcus equinus</td><td colspan="1" rowspan="1">Streptococcus suis</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus equorum</td><td colspan="1" rowspan="1">Staphylococcus sciuri</td><td colspan="1" rowspan="1">Streptococcus gallolyticus</td><td colspan="1" rowspan="1">Streptococcus vestibularis</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus haemolyticus</td><td colspan="1" rowspan="1">Staphylococcus simulans</td><td colspan="1" rowspan="1">Streptococcus gordonii</td><td colspan="1" rowspan="2"></td></tr><tr><td colspan="1" rowspan="1">Staphylococcus hominis</td><td colspan="1" rowspan="1">Staphylococcus warneri</td><td colspan="1" rowspan="1">Streptococcus intermedius</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Gram Negative Bacteria</td><td></td><td></td></tr><tr><td colspan="1" rowspan="1">Acinetobacter baumannii</td><td colspan="1" rowspan="1">Enterobacter mori</td><td colspan="1" rowspan="1">Metakosakonia massiliensis</td><td colspan="1" rowspan="1">Salmonella typhimurium</td></tr><tr><td colspan="1" rowspan="1">Acinetobacter calcoaceticus</td><td colspan="1" rowspan="1">Enterobacter soli</td><td colspan="1" rowspan="1">Mixta calida</td><td colspan="1" rowspan="1">Serratia entomophila</td></tr><tr><td colspan="1" rowspan="1">Acinetobacter nosocomialisb</td><td colspan="1" rowspan="1">Erwinia billingiae</td><td colspan="1" rowspan="1">Mixta gaviniae</td><td colspan="1" rowspan="1">Serratia ficaria</td></tr><tr><td colspan="1" rowspan="1">Acinetobacter pitti</td><td colspan="1" rowspan="1">Escherichia albertire</td><td colspan="1" rowspan="1">Morganella morganii</td><td colspan="1" rowspan="1">Serratia fonticola</td></tr><tr><td colspan="1" rowspan="1">Acinetobacter seifertii</td><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">Neisseria meningitidis</td><td colspan="1" rowspan="1">Serratia liquefaciens</td></tr><tr><td colspan="1" rowspan="1">Bacteroides fragilis</td><td colspan="1" rowspan="1">Escherichia fergusonif</td><td colspan="1" rowspan="1">Pantoea agglomerans</td><td colspan="1" rowspan="1">Serratia marcescens</td></tr><tr><td colspan="1" rowspan="1">Cedecea davisae</td><td colspan="1" rowspan="1">Escherichia hermannii</td><td colspan="1" rowspan="1">Pantoea septica</td><td colspan="1" rowspan="1">Serratia odorifera</td></tr><tr><td colspan="1" rowspan="1">Citrobacter braakii</td><td colspan="1" rowspan="1">Haemophilus influenzae</td><td colspan="1" rowspan="1">Photorhabdus asymbiotica</td><td colspan="1" rowspan="1">Serratia plymuthica</td></tr><tr><td colspan="1" rowspan="1">Citrobacter freundii</td><td colspan="1" rowspan="1">Hafnia alvei</td><td colspan="1" rowspan="1">Plesiomonas shigelloidesh</td><td colspan="1" rowspan="1">Serratia rubidaea</td></tr><tr><td colspan="1" rowspan="1">Citrobacter koseri</td><td colspan="1" rowspan="1">Hafnia paralvei</td><td colspan="1" rowspan="1">Pluralibacter gergoviae</td><td colspan="1" rowspan="1">Shigella boydiif</td></tr><tr><td colspan="1" rowspan="1">Cosenzaea myxofaciensc</td><td colspan="1" rowspan="1">Klebsiella aerogenes</td><td colspan="1" rowspan="1">Proteus hauseri</td><td colspan="1" rowspan="1">Shigella dysenteriae</td></tr><tr><td colspan="1" rowspan="1">Cronobacter malonaticus</td><td colspan="1" rowspan="1">Klebsiella grimontiig</td><td colspan="1" rowspan="1">Proteus mirabilis</td><td colspan="1" rowspan="1">Shigella flexneri</td></tr><tr><td colspan="1" rowspan="1">Cronobacter sakazakii</td><td colspan="1" rowspan="1">Klebsiella michiganensis</td><td colspan="1" rowspan="1">Proteus penneri</td><td colspan="1" rowspan="1">Shigella sonne</td></tr><tr><td colspan="1" rowspan="1">Cronobacter turicensis</td><td colspan="1" rowspan="1">Klebsiella oxytoca</td><td colspan="1" rowspan="1">Proteus vulgaris</td><td colspan="1" rowspan="1">Shimwellia blattae</td></tr><tr><td colspan="1" rowspan="1">Edwardsiella tarda</td><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">Providencia stuartii</td><td colspan="1" rowspan="1">Stenotrophomonas maltophilia</td></tr><tr><td colspan="1" rowspan="1">Enterobacter asburiae</td><td colspan="1" rowspan="1">Klebsiella quasipneumoniae</td><td colspan="1" rowspan="1">Pseudescherichia vulneris</td><td colspan="1" rowspan="1">Tatumella ptyseos</td></tr><tr><td colspan="1" rowspan="1">Enterobacter bugandensisd</td><td colspan="1" rowspan="1">Klebsiella variicola</td><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">Trabulsiella guamensis</td></tr><tr><td colspan="1" rowspan="1">Enterobacter cancerogenus</td><td colspan="1" rowspan="1">Kluyvera ascorbata</td><td colspan="1" rowspan="1">Rahnella aquatilis</td><td colspan="1" rowspan="1">Yersinia enterocolitica</td></tr><tr><td colspan="1" rowspan="1">Enterobacter cloacae</td><td colspan="1" rowspan="1">Kluyvera intermedia</td><td colspan="1" rowspan="1">Raoultella ornithinolytica</td><td colspan="1" rowspan="1">Yersinia frederiksenii</td></tr><tr><td colspan="1" rowspan="1">Enterobacter hormaechei</td><td colspan="1" rowspan="1">Kosakonia cowanii</td><td colspan="1" rowspan="1">Raoultella planticola</td><td colspan="1" rowspan="1">Yersinia pestis</td></tr><tr><td colspan="1" rowspan="1">Enterobacter hormaecheisubsp. xiangfangensis(aka Enterobacter xiangfangensis)</td><td colspan="1" rowspan="1">Leclercia adecarboxylata</td><td colspan="1" rowspan="1">Raoultella terrigena</td><td colspan="1" rowspan="1">Yersinia psuedotuberculosis</td></tr><tr><td colspan="1" rowspan="1">Enterobacter kobei</td><td colspan="1" rowspan="1">Lelliottia amnigena</td><td colspan="1" rowspan="1">Salmonella bongori</td><td colspan="1" rowspan="1">Yokenella regensburgei</td></tr><tr><td colspan="1" rowspan="1">Enterobacter ludwigii</td><td colspan="1" rowspan="1">Lelliottia nimipressuralis</td><td colspan="1" rowspan="1">Salmonella enterica</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="3" rowspan="1">Yeast</td></tr><tr><td colspan="1" rowspan="1">Candida albicans</td><td colspan="1" rowspan="1">Candida glabrata</td><td colspan="1" rowspan="1">Candida parapsilosis</td><td colspan="1" rowspan="1">Cryptococcus gattii</td></tr><tr><td colspan="1" rowspan="1">Candida auris</td><td colspan="1" rowspan="1">Candida krusei</td><td colspan="1" rowspan="1">Candida tropicalis *</td><td colspan="1" rowspan="1">Cryptococcus neoformans</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="2" rowspan="1">Antimicrobial Resistance Genes</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">CTX-M</td><td colspan="1" rowspan="1">mcr-1</td><td colspan="1" rowspan="1">NDM</td><td colspan="1" rowspan="1">VIM</td></tr><tr><td colspan="1" rowspan="1">IMP</td><td colspan="1" rowspan="1">mecA/C</td><td colspan="1" rowspan="1">OXA-48-like</td><td colspan="1" rowspan="2"></td></tr><tr><td colspan="1" rowspan="1">KPC</td><td colspan="1" rowspan="1">mecA/C and MREJ (MRSA)</td><td colspan="1" rowspan="1">vanA/B</td></tr></table>

a Detected as Staphylococcus aureus (mecA/C and MREJ (MRSA) also detected for S. argenteus); members of the Staphylococcus aureus compl b Risk of amplification by KPC assay predicted by sequence analysis; not observed when tested at $8 . 7 \mathsf { E } ^ { + } 0 9 $ CFU/mL c Detected as Proteus spp. at $2 8 . 8 \mathsf { E } \star 0 6$ CFU/mL. Nonpathogenic bacterium isolated from gypsy moths; formerly classified as Proteus d Detected as Enterobacter cloacae complex; newly described species e Detected as Escherichia coli at $2 8 . 0 \mathsf { E } \mathrm { + } 0 8 $ CFU/mL f Detected as Escherichia coli g Detected as Klebsiella oxytoca; Klebsiella grimontii was formerly classified as $\kappa$ oxytoca phylogroup Ko6 h Risk of amplification by Salmonella assay predicted by sequence analysis; not observed when tested at $7 . 4 \mathsf { E } { + } 0 9 $ CFU/mL i Detected as Enterobacter cloacae complex at ${ \tt 2 9 . 0 E { + 0 7 } }$ CFU/mL j Detected as Candida tropicalis at $2 2 . 8 \mathsf { E } + 0 7$ CFU/mL k Detected as Candida parapsilosis at ${ \ge } 6 . 3 5 { + } 0 7$ CFU/mL

Table 116. Off-Panel Organisms and AMR Genes Tested for Evaluation of BioFire BCID2 Panel Analytical Specificity A risk of cross-reactivity was confirmed for the species in bold.   

<table><tr><td colspan="4" rowspan="1">OFF-PANEL</td></tr><tr><td colspan="4" rowspan="1">Gram Positive Bacteria</td></tr><tr><td colspan="1" rowspan="1">Acintomyces naeslundii</td><td colspan="1" rowspan="1">Corynebacterium jeikeium</td><td colspan="1" rowspan="1">Enterococcus raffinosus</td><td colspan="1" rowspan="1">Micrococcus luteus</td></tr><tr><td colspan="1" rowspan="1">Actinomyces israelii</td><td colspan="1" rowspan="1">Corynebacterium striatum</td><td colspan="1" rowspan="1">Gemella morbillorum</td><td colspan="1" rowspan="1">Mycoplasma hominis</td></tr><tr><td colspan="1" rowspan="1">Actinomyces odontolyticus</td><td colspan="1" rowspan="1">Corynebacterium urealyticum</td><td colspan="1" rowspan="1">Granulicatella adiacens²</td><td colspan="1" rowspan="1">Mycoplasma pneumoniae</td></tr><tr><td colspan="1" rowspan="1">Aerococcus viridansa</td><td colspan="1" rowspan="1">Cutibacterium acnes</td><td colspan="1" rowspan="1">Kocuria kristinae</td><td colspan="1" rowspan="1">Nocardia farcinica</td></tr><tr><td colspan="1" rowspan="1">Arcanobacterium haemolyticum</td><td colspan="1" rowspan="1">Enterococcus avium</td><td colspan="1" rowspan="1">Lactobacillus acidophilus</td><td colspan="1" rowspan="1">Peptostreptococcus anaerobius</td></tr><tr><td colspan="1" rowspan="1">Bacillus cereus</td><td colspan="1" rowspan="1">Enterococcus casseliflavus</td><td colspan="1" rowspan="1">Lactococcus lactis</td><td colspan="1" rowspan="1">Rhodococcus equi</td></tr><tr><td colspan="1" rowspan="1">Bacillus licheniformis</td><td colspan="1" rowspan="1">Enterococcus cecorum²</td><td colspan="1" rowspan="1">Listeria grayi</td><td colspan="1" rowspan="1">Rothia mucilaginosa</td></tr><tr><td colspan="1" rowspan="1">Bacillus subtilis</td><td colspan="1" rowspan="1">Enterococcus dispar</td><td colspan="1" rowspan="1">Listeria innocua</td><td colspan="1" rowspan="1">Sarcina ventriculi</td></tr><tr><td colspan="1" rowspan="1">Clostridioides difficile</td><td colspan="1" rowspan="1">Enterococcus durans</td><td colspan="1" rowspan="1">Listeria ivanovii</td><td colspan="1" rowspan="1">Solibacillus silvestris</td></tr><tr><td colspan="1" rowspan="1">Clostridium perfringens</td><td colspan="1" rowspan="1">Enterococcus gallinarum</td><td colspan="1" rowspan="1">Listeria seeligeri</td><td colspan="1" rowspan="1">Ureaplasma parvum</td></tr><tr><td colspan="1" rowspan="1">Clostridium tetani</td><td colspan="1" rowspan="1">Enterococcus hirae</td><td colspan="1" rowspan="1">Listeria welsimeri</td><td colspan="1" rowspan="1">Ureaplasma urealyticum</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium diptheria</td><td colspan="1" rowspan="1">Enterococcus mundtii</td><td colspan="1" rowspan="1">Macrococcus caseolyticus</td><td colspan="1" rowspan="1">Vagococcus fluvialis</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Gram Neg</td><td colspan="1" rowspan="1">Gram Negative Bacteria</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Acinetobacter baylyi</td><td colspan="1" rowspan="1">Bacteroides uniformis</td><td colspan="1" rowspan="1">Kingella kingae</td><td colspan="1" rowspan="1">Pseudomonas fluorescens</td></tr><tr><td colspan="1" rowspan="1">Acinetobacter bereziniae</td><td colspan="1" rowspan="1">Bacteroides vulgatus</td><td colspan="1" rowspan="1">Kingella negevensis</td><td colspan="1" rowspan="1">Pseudomonas luteola</td></tr><tr><td colspan="1" rowspan="1">Acinetobacter guillouiae</td><td colspan="1" rowspan="1">Bordetella bronchiseptica</td><td colspan="1" rowspan="1">Kingella oralis</td><td colspan="1" rowspan="1">Pseudomonas nitroreducens</td></tr><tr><td colspan="1" rowspan="1">Acinetobacter haemolyticus</td><td colspan="1" rowspan="1">Bordetella parapertussis</td><td colspan="1" rowspan="1">Legionella pneumophila</td><td colspan="1" rowspan="1">Pseudomonas oleovorans</td></tr><tr><td colspan="1" rowspan="1">Acinetobacter johnsonii</td><td colspan="1" rowspan="1">Bordetella pertussis</td><td colspan="1" rowspan="1">Leptospira interrogans</td><td colspan="1" rowspan="1">Pseudomonas oryzihabitans</td></tr><tr><td colspan="1" rowspan="1">Acinetobacter junii</td><td colspan="1" rowspan="1">Burkholderia cepacia</td><td colspan="1" rowspan="1">Moraxella catarrhalis</td><td colspan="1" rowspan="1">Pseudomonas pertucinogena</td></tr><tr><td colspan="1" rowspan="1">Acinetobacter Iwoffii</td><td colspan="1" rowspan="1">Burkholderia mallei</td><td colspan="1" rowspan="1">Moraxella osloensis®</td><td colspan="1" rowspan="1">Pseudomonas putida</td></tr><tr><td colspan="1" rowspan="1">Acinetobacter parvus</td><td colspan="1" rowspan="1">Burkholderia multivorans</td><td colspan="1" rowspan="1">Mycobacterium tuberculosis</td><td colspan="1" rowspan="1">Pseudomonas stutzeri</td></tr><tr><td colspan="1" rowspan="1">Acinetobacter radioresistens</td><td colspan="1" rowspan="1">Burkholderia pseudomallei</td><td colspan="1" rowspan="1">Neisseria gonorrhoeae</td><td colspan="1" rowspan="1">Pseudomonas veronii</td></tr><tr><td colspan="1" rowspan="1">Acinetobacter schindlerib</td><td colspan="1" rowspan="1">Campylobacter hominis</td><td colspan="1" rowspan="1">Neisseria lactamica</td><td colspan="1" rowspan="1">Psychrobacter cryohalolentis</td></tr><tr><td colspan="1" rowspan="1">Acinetobacter soli</td><td colspan="1" rowspan="1">Chlamydia trachomatis</td><td colspan="1" rowspan="1">Neisseria meningitidis(unencapsulated)</td><td colspan="1" rowspan="1">Psychrobacter immobilis</td></tr><tr><td colspan="1" rowspan="1">Acinetobacter ursingii</td><td colspan="1" rowspan="1">Chlamydophila pneumoniae</td><td colspan="1" rowspan="1">Neisseria mucosa</td><td colspan="1" rowspan="1">Ralstonia mannitolilytica</td></tr><tr><td colspan="1" rowspan="1">Acintobacillus ureae</td><td colspan="1" rowspan="1">Chromobacterium violaceum</td><td colspan="1" rowspan="1">Neisseria sicca</td><td colspan="1" rowspan="1">Ralstonia pickettii</td></tr><tr><td colspan="1" rowspan="1">Aggregatibacteractinomycetemcomitans</td><td colspan="1" rowspan="1">Eikenella corrodens</td><td colspan="1" rowspan="1">Parabacteroides distasonis</td><td colspan="1" rowspan="1">Stenotrophomonas acidiminiphila</td></tr><tr><td colspan="1" rowspan="1">Actinobacillus hominis</td><td colspan="1" rowspan="1">Haemophilus aegyptiusd</td><td colspan="1" rowspan="1">Pasteurella aerogenes</td><td colspan="1" rowspan="1">Stenotrophomonasnitritireducens</td></tr><tr><td colspan="1" rowspan="1">Aeromonas caviae</td><td colspan="1" rowspan="1">Haemophilus ducreyi</td><td colspan="1" rowspan="1">Pasteurella canis</td><td colspan="1" rowspan="1">Stenotrophomonas rhizophila</td></tr><tr><td colspan="1" rowspan="1">Aeromonas hydrophila</td><td colspan="1" rowspan="1">Haemophilus haemolyticus</td><td colspan="1" rowspan="1">Pasteurella multocida</td><td colspan="1" rowspan="1">Treponema pallidum</td></tr><tr><td colspan="1" rowspan="1">Aeromonas sobria</td><td colspan="1" rowspan="1">Haemophilus parahaemolyticus</td><td colspan="1" rowspan="1">Pasteurella stomatis</td><td colspan="1" rowspan="1">Veillonella parvula</td></tr><tr><td colspan="1" rowspan="1">Aggregatibacter aphrophilus</td><td colspan="1" rowspan="1">Haemophilus parainfluenzae</td><td colspan="1" rowspan="1">Prevotella melaninogenica</td><td colspan="1" rowspan="1">Vibrio alginolyticus</td></tr><tr><td colspan="1" rowspan="1">Bacteriodes xylanisolvens</td><td colspan="1" rowspan="1">Haemophilus parasuis</td><td colspan="1" rowspan="1">Prevotella oralis</td><td colspan="1" rowspan="1">Vibrio parahaemolyticus</td></tr><tr><td colspan="1" rowspan="1">Bacteroides caccae</td><td colspan="1" rowspan="1">Haemophilus quentini</td><td colspan="1" rowspan="1">Pseudomonas alcaligenes</td><td colspan="1" rowspan="1">Vibrio vulnificus</td></tr><tr><td colspan="1" rowspan="1">Bacteroides ovatus</td><td colspan="1" rowspan="1">Haemophilus sputorum</td><td colspan="1" rowspan="1">Pseudomonas citronellolis</td><td colspan="1" rowspan="2"></td></tr><tr><td colspan="1" rowspan="1">Bacteroides thetaiotaomicron</td><td colspan="1" rowspan="1">Kingella denitrificans</td><td colspan="1" rowspan="1">Pseudomonas mendocina</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Y</td><td colspan="1" rowspan="1">Yeast</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Aspergillus flavus</td><td colspan="1" rowspan="1">Candida kefyr(Kluyveromyces marxianus)</td><td colspan="1" rowspan="1">Coccidioides posadasii</td><td colspan="1" rowspan="1">Millerozyma farinosa(Candida cacaoi)</td></tr><tr><td colspan="1" rowspan="1">Aspergillus fumigatus</td><td colspan="1" rowspan="1">Candida lusitaniaeClavispora lusitaniae)</td><td colspan="1" rowspan="1">Cryptococcus amylolentus</td><td colspan="1" rowspan="1">Naganishia albida(Cryptococcus albidus)</td></tr><tr><td colspan="1" rowspan="1">Aspergillus niger</td><td colspan="1" rowspan="1">Candida metapsilosis</td><td colspan="1" rowspan="1">Cryptococcus uniguttulatus</td><td colspan="1" rowspan="1">Papiliotrema laurentii(Cryptococcus laurenti)</td></tr><tr><td colspan="1" rowspan="1">Aspergillus terreus</td><td colspan="1" rowspan="1">Candida multis-gemmis</td><td colspan="1" rowspan="1">Cutaneotrichosporon curvatum(Cryptococcus curvatus)</td><td colspan="1" rowspan="1">Penicillum chrysogenum</td></tr><tr><td colspan="1" rowspan="1">Blastomyces dermatitidis</td><td colspan="1" rowspan="1">Candida nivariensis</td><td colspan="1" rowspan="1">Cyberlindnera fabianiiCandida fabianii)</td><td colspan="1" rowspan="1">Rhodotorula mucilaginosa</td></tr><tr><td colspan="1" rowspan="1">Candida dubliniensis</td><td colspan="1" rowspan="1">Candida norvegensis(Pichia norvegensis)</td><td colspan="1" rowspan="1">Histoplasma capsulatum</td><td colspan="1" rowspan="1">Saccharomyces cerevisiae</td></tr><tr><td colspan="1" rowspan="1">Candida famata(Debaryomyces hanseni)g</td><td colspan="1" rowspan="1">Candida orthopsilosis</td><td colspan="1" rowspan="1">Kluyveromyces lactis</td><td colspan="1" rowspan="1">Schizosaccharomyces pombe</td></tr><tr><td colspan="1" rowspan="1">Candida guilliermondii(Meyerozyma guilliermondii,Pichia guilliermondii)</td><td colspan="1" rowspan="1">Candida sojae</td><td colspan="1" rowspan="1">Kodameae ohmeri</td><td colspan="1" rowspan="1">Talaromyces marneffei</td></tr><tr><td colspan="1" rowspan="1">Candida haemulonii</td><td colspan="1" rowspan="1">Candida viswanthii</td><td colspan="1" rowspan="1">Lodderomyces elongisporus</td><td colspan="1" rowspan="1">Trichosporon asahii</td></tr><tr><td colspan="1" rowspan="1">Candida inconspicua(Pichia cactophila)</td><td colspan="1" rowspan="1">Coccidioides immitis</td><td colspan="1" rowspan="1">Magnusiomyces capitatus</td><td colspan="1" rowspan="1">Wickerhamomyces anomalus</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="2" rowspan="1">Viruses</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Chikungunya Virus</td><td colspan="1" rowspan="1">Hepatitis B virus</td><td colspan="1" rowspan="1">Human Herpes Virus-7</td><td colspan="1" rowspan="1">Varicella Zoster Virus</td></tr><tr><td colspan="1" rowspan="1">Cytomegalovirus</td><td colspan="1" rowspan="1">Hepatitis C virus</td><td colspan="1" rowspan="1">Measles Virus</td><td colspan="1" rowspan="1">West Nile Virus</td></tr><tr><td colspan="1" rowspan="1">Dengue Virus</td><td colspan="1" rowspan="1">Human Immunodeficiency Virus</td><td colspan="1" rowspan="1">Mumps Virus</td><td colspan="1" rowspan="1">Zika Virus</td></tr><tr><td colspan="1" rowspan="1">Enterovirus</td><td colspan="1" rowspan="1">Herpes Simplex Virus Type 1</td><td colspan="1" rowspan="1">Parvovirus B19</td><td colspan="1" rowspan="3"></td></tr><tr><td colspan="1" rowspan="1">Epstein Barr Virus</td><td colspan="1" rowspan="1">Herpes Simplex Virus Type 2</td><td colspan="1" rowspan="1">Polyoma Virus</td></tr><tr><td colspan="1" rowspan="1">Hepatitis A virus</td><td colspan="1" rowspan="1">Human Herpes Virus-6</td><td colspan="1" rowspan="1">Rubella Virus</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="2" rowspan="1">Parasites</td></tr><tr><td colspan="2" rowspan="1">Plasmodium falciparum</td><td colspan="2" rowspan="1">Trypanosoma cruzi</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Antimicrobial</td><td colspan="2" rowspan="1">Antimicrobial Resistance Genes</td></tr><tr><td colspan="1" rowspan="1">AmpC</td><td colspan="1" rowspan="1">mcr-3</td><td colspan="1" rowspan="1">SHV</td><td colspan="1" rowspan="1">vanC</td></tr><tr><td colspan="1" rowspan="1">blaRAHN*</td><td colspan="1" rowspan="1">mcr-4</td><td colspan="1" rowspan="1">SME</td><td colspan="1" rowspan="1">vanD</td></tr><tr><td colspan="1" rowspan="1">CMY</td><td colspan="1" rowspan="1">ompK36</td><td colspan="1" rowspan="1">SPM</td><td colspan="1" rowspan="1">vanM</td></tr><tr><td colspan="1" rowspan="1">mcr-2</td><td colspan="1" rowspan="1">OXA-24/65</td><td colspan="1" rowspan="1">TEM</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="4" rowspan="1"></td></tr></table>

a Risk of amplification by the Staphylococcus assay predicted by sequence analysis; not detected when tested at cells/mL (G. adiacens) or CFU/mL (A. viridans and E. cecorum). Similar risk predicted for Aerococcus christensenii and Aerococcus sanguinicola; not tested b Risk of amplification by the CTX-M assay at $2 8 . 1 \mathsf { E } \mathsf { + } 0 9$ CFU/mL; will not be detected unless an applicable bacterium is also detected c Detected as Bacteroides fragilis at $2 8 . 2 E + 0 6$ cells/mL. Similar risk predicted for Bacteroides ovatus; not tested d Detected as Haemophilus influenzae; also described as Haemophilus influenzae biogroup aegyptius e Risk of amplification by KPC assay predicted by sequence analysis; not observed when tested at $7 . 8 \mathsf { E } { + } 0 9 $ CFU/mL f Risk of amplification by the Bfragilis assay predicted by sequence analysis; not observed when tested at $5 . 0 \mathsf { E } ^ { + 0 8 }$ CFU/mL. Similar risk predicted for Aspergillus lentulus and Aspergillus viridinutans; not tested g Risk of amplification by the Ckrusei assay predicted by sequence analysis; not observed when tested at $8 . 9 \mathsf { E } ^ { + 0 8 }$ CFU/mL. Similar risk predicted for other yeast species; not tested h Detected as Candida krusei at $2 3 . 7 \mathsf { E } \mathsf { + } 0 5$ CFU/mL. Similar risk predicted for Pichia pseudocactophila and Pichia cactophila; not tested i Detected as Candida glabrata at ${ \geq } 8 . 0 5 { + } 0 6 $ CFU/mL. Candida glabrata complex species; similar risk predicted for Candida bracarensis; not tested j Detected as Cryptococcus neoformans/gattii; nonpathogenic fungus isolated from the frass of beetles k Risk of amplification by the CTX-M assay predicted by sequence analysis; not observed when Rahnella aquatilis was tested at $7 . 8 \mathsf { E } { + } 0 9 $ CFU/mL. Similar risk predicted for ampC in Leminorella species, blaOXY, and blaKLUC; not tested l Not tested; predicted to be detected as vanA/B if an applicable bacterium is also detected

# Reproducibility

tion gene.

E reproducibility of results (percent $( \% )$ ipoDulnalyys in Table 117.

Tabe 1.Reproduibility f the BioFi BCID Panel Results n FilmArray .and FilmArray Torc Ssems   

<table><tr><td colspan="1" rowspan="3">Analyte(Type/Species)Source ID</td><td colspan="1" rowspan="3">ConcentrationTested</td><td colspan="1" rowspan="3">ExpectedResult</td><td colspan="9" rowspan="1">Agreement with Expected Result</td></tr><tr><td colspan="4" rowspan="1">FilmArray 2.0</td><td colspan="4" rowspan="1">FilmArray Torch</td><td colspan="1" rowspan="2">All Sites/Systems [95%Confidence Interval</td></tr><tr><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">SystemTotal</td><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">SystemTotal</td></tr><tr><td colspan="1" rowspan="1"></td><td></td><td></td><td></td><td></td><td colspan="1" rowspan="1">Gram Positive Bacteria</td><td colspan="2" rowspan="1"></td><td colspan="2" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="2">Enterococcus faecalisATCC 51299</td><td colspan="1" rowspan="1">7.65E+08CF/mL</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">20/20 (100%)</td><td colspan="1" rowspan="1">20/20 (100%)</td><td colspan="1" rowspan="1">20/20 (100%)</td><td colspan="1" rowspan="1">60/60(1100%)</td><td colspan="1" rowspan="1">20/20 (100%)</td><td colspan="1" rowspan="1">20/20 (100%)</td><td colspan="1" rowspan="1">20/20(1100%)</td><td colspan="1" rowspan="1">60/60(1100%)</td><td colspan="1" rowspan="1">120/120100%[97.0%-100%]</td></tr><tr><td colspan="1" rowspan="1">Negative(no analyte)</td><td colspan="1" rowspan="1">NotDetected</td><td colspan="1" rowspan="1">100/100(100%)</td><td colspan="1" rowspan="1">100/100(100%)</td><td colspan="1" rowspan="1">100/100(100%)</td><td colspan="1" rowspan="1">300/300(1100%)</td><td colspan="1" rowspan="1">100/100(1100%)</td><td colspan="1" rowspan="1">100/100(100%)</td><td colspan="1" rowspan="1">100/100(100%)</td><td colspan="1" rowspan="1">300/300(1100%)</td><td colspan="1" rowspan="1">60/60000%[99.4%-100%]</td></tr><tr><td colspan="1" rowspan="2">Enterococcus faeciumATCC 27270</td><td colspan="1" rowspan="1">9.19E+08FU/mL</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">20/20(100%)</td><td colspan="1" rowspan="1">20/20 (100%)</td><td colspan="1" rowspan="1">20/20 (100%)</td><td colspan="1" rowspan="1">60/60(1100%)</td><td colspan="1" rowspan="1">20/20(100%)</td><td colspan="1" rowspan="1">20/20(100%)</td><td colspan="1" rowspan="1">20/20(100%)</td><td colspan="1" rowspan="1">60/60(1100%)</td><td colspan="1" rowspan="1">120/120100%[97.0%-100%]</td></tr><tr><td colspan="1" rowspan="1">Negative(no analyte)</td><td colspan="1" rowspan="1">NotDetected</td><td colspan="1" rowspan="1">100/100(10%)</td><td colspan="1" rowspan="1">100/100(100%)</td><td colspan="1" rowspan="1">100/100(10%)</td><td colspan="1" rowspan="1">300/300100%)</td><td colspan="1" rowspan="1">100/100(100%)</td><td colspan="1" rowspan="1">100/100(10%)</td><td colspan="1" rowspan="1">100/100(100%)</td><td colspan="1" rowspan="1">300/300(100%)</td><td colspan="1" rowspan="1">600/600100%[99.4%-100%]</td></tr><tr><td colspan="1" rowspan="1">Listeriamonocytogenes</td><td colspan="1" rowspan="1">Negative(no analyte)</td><td colspan="1" rowspan="1">NotDetected</td><td colspan="1" rowspan="1">120/120(100%)</td><td colspan="1" rowspan="1">120/120 (100%)</td><td colspan="1" rowspan="1">120/120(100%)</td><td colspan="1" rowspan="1">360/360(100%)</td><td colspan="1" rowspan="1">120/120(100%)</td><td colspan="1" rowspan="1">120/120(100%)</td><td colspan="1" rowspan="1">120/120(100%)</td><td colspan="1" rowspan="1">360/360(1100%)</td><td colspan="1" rowspan="1">720/72100%[99.5%-100%]</td></tr><tr><td colspan="1" rowspan="2">Staphylococcus spp.</td><td colspan="1" rowspan="1">Multiplea</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">59/608.3%)</td><td colspan="1" rowspan="1">60/60(1100%)</td><td colspan="1" rowspan="1">60/60 (100%)</td><td colspan="1" rowspan="1">179/180(99.4%)</td><td colspan="1" rowspan="1">60/60(1100%)</td><td colspan="1" rowspan="1">60/60(100%)</td><td colspan="1" rowspan="1">60/60(100%)</td><td colspan="1" rowspan="1">180/180(100%)</td><td colspan="1" rowspan="1">359/3609.7%[98.5%-99.9%]</td></tr><tr><td colspan="1" rowspan="1">Negative(no analyte)</td><td colspan="1" rowspan="1">NotDetected</td><td colspan="1" rowspan="1">60/60(100%)</td><td colspan="1" rowspan="1">60/60100%)</td><td colspan="1" rowspan="1">60/60(100%)</td><td colspan="1" rowspan="1">180/180(100%)</td><td colspan="1" rowspan="1">60/60(100%)</td><td colspan="1" rowspan="1">60/60(100%)</td><td colspan="1" rowspan="1">60/60100%)</td><td colspan="1" rowspan="1">180/180(100%)</td><td colspan="1" rowspan="1">360/360100%[99.0%-100%]</td></tr><tr><td colspan="1" rowspan="2">Staphylococcus aureusATCC BAA-38</td><td colspan="1" rowspan="1">2.44E+08CF/mL</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">19/20(95.%)</td><td colspan="1" rowspan="1">20/20100%)</td><td colspan="1" rowspan="1">20/20100%)</td><td colspan="1" rowspan="1">59/60(98.3%)</td><td colspan="1" rowspan="1">20/20100%)</td><td colspan="1" rowspan="1">20/20(100%)</td><td colspan="1" rowspan="1">20/20(100%)</td><td colspan="1" rowspan="1">60/60(100%)</td><td colspan="1" rowspan="1">119/12099..2%[95.4-99.9%]</td></tr><tr><td colspan="1" rowspan="1">Negative(no analyte)</td><td colspan="1" rowspan="1">NotDetected</td><td colspan="1" rowspan="1">100/10000%)</td><td colspan="1" rowspan="1">100/100(1100%)</td><td colspan="1" rowspan="1">100/10000%)</td><td colspan="1" rowspan="1">300/300100%)</td><td colspan="1" rowspan="1">100/100100%)</td><td colspan="1" rowspan="1">100/100(100%)</td><td colspan="1" rowspan="1">100/100((10%)</td><td colspan="1" rowspan="1">300/300(1100%)</td><td colspan="1" rowspan="1">600/600100%[99.4%-100%]</td></tr><tr><td colspan="1" rowspan="1">Staphylococcuse epidermidis</td><td colspan="1" rowspan="1">2.31E+06FU/mL</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">20/20(100%)</td><td colspan="1" rowspan="1">20/20(1100%)</td><td colspan="1" rowspan="1">19/20(95.0%)</td><td colspan="1" rowspan="1">59/60(98.3%)</td><td colspan="1" rowspan="1">20/20 (100%)</td><td colspan="1" rowspan="1">20/20(1100%)</td><td colspan="1" rowspan="1">20/20(1100%)</td><td colspan="1" rowspan="1">60/60(100%)</td><td colspan="1" rowspan="1">119/12099.2%[95.4-99.9%]</td></tr><tr><td colspan="1" rowspan="3">Analyte(Type/Species)Source ID</td><td colspan="1" rowspan="3">ConcentrationTested</td><td colspan="1" rowspan="3">ExpectedResult</td><td colspan="8" rowspan="1">Agreement with Expected Result</td><td colspan="1" rowspan="1">cYanmaGYellowb</td></tr><tr><td colspan="4" rowspan="1">FilmArray 2.0</td><td colspan="4" rowspan="1">FilmArray Torch</td><td colspan="1" rowspan="2">All Sites/Systems [95%Confidence Interval]</td></tr><tr><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">SystemTotal</td><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">SystemTotal</td></tr><tr><td colspan="1" rowspan="1">ATCC 12228</td><td colspan="1" rowspan="1">Negative(no analyte)</td><td colspan="1" rowspan="1">NotDDetected</td><td colspan="1" rowspan="1">100/100(1100%)</td><td colspan="1" rowspan="1">100/100(1100%)</td><td colspan="1" rowspan="1">100/100(1100%)</td><td colspan="1" rowspan="1">300/300(1100%)</td><td colspan="1" rowspan="1">100/100(1100%)</td><td colspan="1" rowspan="1">100/100(1100%)</td><td colspan="1" rowspan="1">100/100(100%)</td><td colspan="1" rowspan="1">300/300(1100%)</td><td colspan="1" rowspan="1">60/60000%[99.4%-100%]</td></tr><tr><td colspan="1" rowspan="2">StaphylococcuslugdunensisATCC 43809</td><td colspan="1" rowspan="1">1.67E+07CFU/mL</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">20/20(100%)</td><td colspan="1" rowspan="1">20/20(1100%)</td><td colspan="1" rowspan="1">20/20(100%)</td><td colspan="1" rowspan="1">60/60(100%)</td><td colspan="1" rowspan="1">20/20(1100%)</td><td colspan="1" rowspan="1">20/20 (100%)</td><td colspan="1" rowspan="1">20/20 (100%)</td><td colspan="1" rowspan="1">60/60(1100%)</td><td colspan="1" rowspan="1">112/1201100%[97.0%-100%]</td></tr><tr><td colspan="1" rowspan="1">Negative(no analyte)</td><td colspan="1" rowspan="1">NotDetected</td><td colspan="1" rowspan="1">100/100(1100%)</td><td colspan="1" rowspan="1">100/100(1100%)</td><td colspan="1" rowspan="1">100/100(100%)</td><td colspan="1" rowspan="1">300/300(100%)</td><td colspan="1" rowspan="1">100/100(1100%)</td><td colspan="1" rowspan="1">100/100100%)</td><td colspan="1" rowspan="1">100/100(100%)</td><td colspan="1" rowspan="1">300/300(100%)</td><td colspan="1" rowspan="1">60/60000%[99.4%-100%]</td></tr><tr><td colspan="1" rowspan="2">Streptococcus spp.</td><td colspan="1" rowspan="1">Multipleb</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">40/40100%)</td><td colspan="1" rowspan="1">40/40(100%)</td><td colspan="1" rowspan="1">40/40100%)</td><td colspan="1" rowspan="1">120/120(100%)</td><td colspan="1" rowspan="1">40/40(100%)</td><td colspan="1" rowspan="1">40/40100%)</td><td colspan="1" rowspan="1">40/40(100%)</td><td colspan="1" rowspan="1">120/120100%)</td><td colspan="1" rowspan="1">240/240100%[98.4%-100%]</td></tr><tr><td colspan="1" rowspan="1">Negative(no analyte)</td><td colspan="1" rowspan="1">NotDetected</td><td colspan="1" rowspan="1">80/80(100%)</td><td colspan="1" rowspan="1">80/80(100%)</td><td colspan="1" rowspan="1">80/80(100%)</td><td colspan="1" rowspan="1">240/240(100%)</td><td colspan="1" rowspan="1">80/8000%)</td><td colspan="1" rowspan="1">80/80(100%)</td><td colspan="1" rowspan="1">80/80(100%)</td><td colspan="1" rowspan="1">240/240(100%)</td><td colspan="1" rowspan="1">480/480100%[99.2%-100%]</td></tr><tr><td colspan="1" rowspan="1">Streptococcusagalactiae</td><td colspan="1" rowspan="1">Negative(no analyte)</td><td colspan="1" rowspan="1">NotDetected</td><td colspan="1" rowspan="1">120/120(1100%)</td><td colspan="1" rowspan="1">120/120(10%)</td><td colspan="1" rowspan="1">120/120(100%)</td><td colspan="1" rowspan="1">360/360100%)</td><td colspan="1" rowspan="1">120/120100%)</td><td colspan="1" rowspan="1">120/120(100%)</td><td colspan="1" rowspan="1">120/120(100%)</td><td colspan="1" rowspan="1">360/360100%)</td><td colspan="1" rowspan="1">70/720100%[99.5%-100%]</td></tr><tr><td colspan="1" rowspan="2">StreptococcuspneumoniaeATCC 6303</td><td colspan="1" rowspan="1">5.91E+07SF/mL</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">20/20(100%)</td><td colspan="1" rowspan="1">20/20 (100%)</td><td colspan="1" rowspan="1">20/20 (100%)</td><td colspan="1" rowspan="1">60/60(100%)</td><td colspan="1" rowspan="1">20/20 (1100%)</td><td colspan="1" rowspan="1">20/20(1100%)</td><td colspan="1" rowspan="1">20/20 (1100%)</td><td colspan="1" rowspan="1">60/60(100%)</td><td colspan="1" rowspan="1">120/120100%[97.0%-100%]</td></tr><tr><td colspan="1" rowspan="1">Negative(no analyte)</td><td colspan="1" rowspan="1">NotDetected</td><td colspan="1" rowspan="1">100/100(1100%)</td><td colspan="1" rowspan="1">100/100(1100%)</td><td colspan="1" rowspan="1">100/100100%)</td><td colspan="1" rowspan="1">300/300(1100%)</td><td colspan="1" rowspan="1">100/100(1100%)</td><td colspan="1" rowspan="1">100/100(100%)</td><td colspan="1" rowspan="1">100/100100%)</td><td colspan="1" rowspan="1">300/300(1100%)</td><td colspan="1" rowspan="1">60/60000%[99.4%-100%]</td></tr><tr><td colspan="1" rowspan="2">StreptococcuspyogenesATCC 49399</td><td colspan="1" rowspan="1">2.63E+07FU/mL</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">20/20100%)</td><td colspan="1" rowspan="1">20/20(1100%)</td><td colspan="1" rowspan="1">20/20 (100%)</td><td colspan="1" rowspan="1">60/60(100%)</td><td colspan="1" rowspan="1">20/20(100%)</td><td colspan="1" rowspan="1">20/20 (100%)</td><td colspan="1" rowspan="1">20/20(100%)</td><td colspan="1" rowspan="1">60/60(100%)</td><td colspan="1" rowspan="1">120/120100%[97.0%-100%]</td></tr><tr><td colspan="1" rowspan="1">None(no analyte)</td><td colspan="1" rowspan="1">NotDetected</td><td colspan="1" rowspan="1">100/100(100%)</td><td colspan="1" rowspan="1">100/100100%)</td><td colspan="1" rowspan="1">100/100(100%)</td><td colspan="1" rowspan="1">300/300(100%)</td><td colspan="1" rowspan="1">100/100(100%)</td><td colspan="1" rowspan="1">100/100((100%)</td><td colspan="1" rowspan="1">100/100(100%)</td><td colspan="1" rowspan="1">300/300(100%)</td><td colspan="1" rowspan="1">600/600100%[99.4%-100%]</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Gram Negative Bacteria</td><td colspan="1" rowspan="1">Bacteria</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="2">Acinetobactercalcoaceticus-baumannii complex(Acinetobacterbaumannii)AR Bank 0033</td><td colspan="1" rowspan="1">7.36E+07CF/mML</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">19/20(95.0%)</td><td colspan="1" rowspan="1">19/20(950%)</td><td colspan="1" rowspan="1">20/20(1100%)</td><td colspan="1" rowspan="1">58/60(96.7%)</td><td colspan="1" rowspan="1">20/20 (1100%)</td><td colspan="1" rowspan="1">20/20 (1100%)</td><td colspan="1" rowspan="1">20/20 (1100%)</td><td colspan="1" rowspan="1">60/60 (1100%)</td><td colspan="1" rowspan="1">118/12098.3%[94.1%-99.8%]</td></tr><tr><td colspan="1" rowspan="1">Negative(no analyte)</td><td colspan="1" rowspan="1">NotDetected</td><td colspan="1" rowspan="1">100/100(1100%)</td><td colspan="1" rowspan="1">100/100(1100%)</td><td colspan="1" rowspan="1">100/100(100%)</td><td colspan="1" rowspan="1">300/300(1100%)</td><td colspan="1" rowspan="1">100/100(1100%)</td><td colspan="1" rowspan="1">100/100(100%)</td><td colspan="1" rowspan="1">100/100(100%)</td><td colspan="1" rowspan="1">300/300(1100%)</td><td colspan="1" rowspan="1">60/600100%[99.4%-100%]</td></tr><tr><td colspan="1" rowspan="2">Bacteroides fragilisATCC 25285</td><td colspan="1" rowspan="1">8.62E+08FU/mL</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">20/20 (100%)</td><td colspan="1" rowspan="1">20/20(1100%)</td><td colspan="1" rowspan="1">20/20(100%)</td><td colspan="1" rowspan="1">60/60(100%)</td><td colspan="1" rowspan="1">20/20(100%)</td><td colspan="1" rowspan="1">20/20(100%)</td><td colspan="1" rowspan="1">20/20(100%)</td><td colspan="1" rowspan="1">60/60(100%)</td><td colspan="1" rowspan="1">120/1201000%[99.4%-100%]</td></tr><tr><td colspan="1" rowspan="1">Negative(no analyte)</td><td colspan="1" rowspan="1">NotDetected</td><td colspan="1" rowspan="1">100/100(1100%)</td><td colspan="1" rowspan="1">100/100(100%)</td><td colspan="1" rowspan="1">100/100(100%)</td><td colspan="1" rowspan="1">300/300(1100%)</td><td colspan="1" rowspan="1">100/100(1100%)</td><td colspan="1" rowspan="1">100/100(1100%)</td><td colspan="1" rowspan="1">100/100(100%)</td><td colspan="1" rowspan="1">300/300(100%)</td><td colspan="1" rowspan="1">60/60000%[99.4%-100%]</td></tr><tr><td colspan="1" rowspan="2">Enterobacterales</td><td colspan="1" rowspan="1">Multiplec</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">100/100(1100%)</td><td colspan="1" rowspan="1">100/100(1100%)</td><td colspan="1" rowspan="1">100/100(100%)</td><td colspan="1" rowspan="1">300/300(100%)</td><td colspan="1" rowspan="1">100/100(1100%)</td><td colspan="1" rowspan="1">100/100(1100%)</td><td colspan="1" rowspan="1">100/1001000%)</td><td colspan="1" rowspan="1">300/300(100%)</td><td colspan="1" rowspan="1">600/600100%[99.4%-100%]</td></tr><tr><td colspan="1" rowspan="1">Negativeno analyte)</td><td colspan="1" rowspan="1">NotDetected</td><td colspan="1" rowspan="1">20/20(100%)</td><td colspan="1" rowspan="1">20/20(100%)</td><td colspan="1" rowspan="1">20/20100%)</td><td colspan="1" rowspan="1">60/60(100%)</td><td colspan="1" rowspan="1">20/20(100%)</td><td colspan="1" rowspan="1">20/20100%)</td><td colspan="1" rowspan="1">20/20100%)</td><td colspan="1" rowspan="1">60/60(100%)</td><td colspan="1" rowspan="1">120/1201100%[97.0%-100%]</td></tr><tr><td colspan="1" rowspan="5">Analyte(Type/Species)Source ID</td><td></td><td></td><td colspan="9" rowspan="3">Agreement with Expected Result</td></tr><tr><td colspan="1" rowspan="4">ConcentrationTested</td><td colspan="1" rowspan="4">ExpectedResult</td></tr><tr></tr><tr><td colspan="4" rowspan="1">FilmArray 2.0</td><td colspan="2" rowspan="1">FilmArr</td><td colspan="2" rowspan="1">y Torch</td><td colspan="1" rowspan="2">All Sites/Systems [95%Confidence Interval]</td></tr><tr><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">SystemTotal</td><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">SystemTotal</td></tr><tr><td colspan="1" rowspan="1">Enterobacter cloacaecomplex</td><td colspan="1" rowspan="1">Negative(no analyte)</td><td colspan="1" rowspan="1">NotDetected</td><td colspan="1" rowspan="1">120/120(100%)</td><td colspan="1" rowspan="1">120/120(100%)</td><td colspan="1" rowspan="1">120/120(100%)</td><td colspan="1" rowspan="1">360/360100%)</td><td colspan="1" rowspan="1">120/120(100%)</td><td colspan="1" rowspan="1">120/120(100%)</td><td colspan="1" rowspan="1">120/12000%)</td><td colspan="1" rowspan="1">360/360(100%)</td><td colspan="1" rowspan="1">720/7201100%[99.5%-100%]</td></tr><tr><td colspan="1" rowspan="2">Escherichia coliCDC-FDA AR Bank#0350</td><td colspan="1" rowspan="1">8.79E+08FU/mL</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">20/20100%)</td><td colspan="1" rowspan="1">20/20100%)</td><td colspan="1" rowspan="1">19/20(95.0%)</td><td colspan="1" rowspan="1">59/60(98.3%)</td><td colspan="1" rowspan="1">20/20(100%)</td><td colspan="1" rowspan="1">20/20(100%)</td><td colspan="1" rowspan="1">20/20(100%)</td><td colspan="1" rowspan="1">60/60(100%)</td><td colspan="1" rowspan="1">119/120999.2%[95.4-%99.9%]</td></tr><tr><td colspan="1" rowspan="1">Negative(no analyte)</td><td colspan="1" rowspan="1">NotDetected</td><td colspan="1" rowspan="1">100/100100%)</td><td colspan="1" rowspan="1">100/100(1100%)</td><td colspan="1" rowspan="1">100/100100%)</td><td colspan="1" rowspan="1">300/300(100%)</td><td colspan="1" rowspan="1">100/100(100%)</td><td colspan="1" rowspan="1">100/100(100%)</td><td colspan="1" rowspan="1">100/100100%)</td><td colspan="1" rowspan="1">300/300(100%)</td><td colspan="1" rowspan="1">60/600100%[99.4%-100%]</td></tr><tr><td colspan="1" rowspan="2">Klebsiella aerogenesCDC-FDA AR Bank#0161</td><td colspan="1" rowspan="1">9.29E+08F/mL</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">20/20 (100%)</td><td colspan="1" rowspan="1">20/20(100%)</td><td colspan="1" rowspan="1">20/20(100%)</td><td colspan="1" rowspan="1">60/60(100%)</td><td colspan="1" rowspan="1">20/20(1100%)</td><td colspan="1" rowspan="1">20/20(100%)</td><td colspan="1" rowspan="1">20/20(100%)</td><td colspan="1" rowspan="1">60/60(1100%)</td><td colspan="1" rowspan="1">11220/120100%[97.0%-100%]</td></tr><tr><td colspan="1" rowspan="1">Negativeno analyte)</td><td colspan="1" rowspan="1">NotDetected</td><td colspan="1" rowspan="1">100/10000%)</td><td colspan="1" rowspan="1">100/100100%)</td><td colspan="1" rowspan="1">100/100((100%)</td><td colspan="1" rowspan="1">300/300(100%)</td><td colspan="1" rowspan="1">100/10000%)</td><td colspan="1" rowspan="1">100/100(100%)</td><td colspan="1" rowspan="1">100/100(100%)</td><td colspan="1" rowspan="1">300/300(100%)</td><td colspan="1" rowspan="1">600/600100%[99.4%-100%]</td></tr><tr><td colspan="1" rowspan="2">Klebsiella oxytocaCDC-FDA AR Bank#0147</td><td colspan="1" rowspan="1">2.44E+07CFU/mL</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">20/20 (100%)</td><td colspan="1" rowspan="1">20/20(1100%)</td><td colspan="1" rowspan="1">20/20(100%)</td><td colspan="1" rowspan="1">60/60(100%)</td><td colspan="1" rowspan="1">20/20(100%)</td><td colspan="1" rowspan="1">20/20100%)</td><td colspan="1" rowspan="1">20/20(100%)</td><td colspan="1" rowspan="1">60/60(100%)</td><td colspan="1" rowspan="1">120/120100%[97.0%-100%]</td></tr><tr><td colspan="1" rowspan="1">Negative(no analyte)</td><td colspan="1" rowspan="1">NotDetected</td><td colspan="1" rowspan="1">100/100100%)</td><td colspan="1" rowspan="1">100/100(1100%)</td><td colspan="1" rowspan="1">100/100(100%)</td><td colspan="1" rowspan="1">300/300(100%)</td><td colspan="1" rowspan="1">100/100(100%)</td><td colspan="1" rowspan="1">100/100(100%)</td><td colspan="1" rowspan="1">100/100(1100%)</td><td colspan="1" rowspan="1">300/300(1100%)</td><td colspan="1" rowspan="1">600/600100%[99.4%-100%]</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniaegroup</td><td colspan="1" rowspan="1">Negative(no analyte)</td><td colspan="1" rowspan="1">NotDetected</td><td colspan="1" rowspan="1">120/120(100%)</td><td colspan="1" rowspan="1">120/120(100%)</td><td colspan="1" rowspan="1">120/120(1100%)</td><td colspan="1" rowspan="1">360/360(100%)</td><td colspan="1" rowspan="1">120/120(100%)</td><td colspan="1" rowspan="1">120/120100%)</td><td colspan="1" rowspan="1">120/120(100%)</td><td colspan="1" rowspan="1">360/36000%)</td><td colspan="1" rowspan="1">720/720100%[99.5%-100%]</td></tr><tr><td colspan="1" rowspan="2">Proteus spp.(Proteus mirabilis)GRE 1254053</td><td colspan="1" rowspan="1">1.68E+09CF/mL</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">20/20 (1100%)</td><td colspan="1" rowspan="1">20/20 (1100%)</td><td colspan="1" rowspan="1">20/20(100%)</td><td colspan="1" rowspan="1">60/60(100%)</td><td colspan="1" rowspan="1">20/20 (1100%)</td><td colspan="1" rowspan="1">20/20 (1100%)</td><td colspan="1" rowspan="1">20/20(100%)</td><td colspan="1" rowspan="1">60/60(100%)</td><td colspan="1" rowspan="1">1220/120100%[97.0%-100%]</td></tr><tr><td colspan="1" rowspan="1">Negativeno analyte)</td><td colspan="1" rowspan="1">NotDetected</td><td colspan="1" rowspan="1">100/100(100%)</td><td colspan="1" rowspan="1">100/100(1100%)</td><td colspan="1" rowspan="1">100/100(100%)</td><td colspan="1" rowspan="1">300/300(1100%)</td><td colspan="1" rowspan="1">100/100(1100%)</td><td colspan="1" rowspan="1">100/100(100%)</td><td colspan="1" rowspan="1">100/100(1100%)</td><td colspan="1" rowspan="1">300/300(1100%)</td><td colspan="1" rowspan="1">600/600100%[99.4%-100%]</td></tr><tr><td colspan="1" rowspan="2">Salmonella spp.(Salmonella enterica)ATCC 700720</td><td colspan="1" rowspan="1">1.74E+09CF/mML</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">20/20(100%)</td><td colspan="1" rowspan="1">20/20(100%)</td><td colspan="1" rowspan="1">20/20(100%)</td><td colspan="1" rowspan="1">60/60(100%)</td><td colspan="1" rowspan="1">20/20(100%)</td><td colspan="1" rowspan="1">20/20(100%)</td><td colspan="1" rowspan="1">20/20(100%)</td><td colspan="1" rowspan="1">60/60(100%)</td><td colspan="1" rowspan="1">120/120100%[97.0%-100%]</td></tr><tr><td colspan="1" rowspan="1">Negativeno analyte)</td><td colspan="1" rowspan="1">NotDetected</td><td colspan="1" rowspan="1">100/100100%)</td><td colspan="1" rowspan="1">100/100(1100%)</td><td colspan="1" rowspan="1">100/100(100%)</td><td colspan="1" rowspan="1">300/300(100%)</td><td colspan="1" rowspan="1">100/100(100%)</td><td colspan="1" rowspan="1">100/100(100%)</td><td colspan="1" rowspan="1">100/100(100%)</td><td colspan="1" rowspan="1">300/300(1100%)</td><td colspan="1" rowspan="1">60/600100%[99.4%-100%]</td></tr><tr><td colspan="1" rowspan="2">Serratia marcescensGRE 1659004</td><td colspan="1" rowspan="1">3.05E+07FU/mL</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">20/20 (100%)</td><td colspan="1" rowspan="1">20/20(100%)</td><td colspan="1" rowspan="1">20/20(100%)</td><td colspan="1" rowspan="1">60/60(100%)</td><td colspan="1" rowspan="1">20/20(1100%)</td><td colspan="1" rowspan="1">20/20(1100%)</td><td colspan="1" rowspan="1">20/20(100%)</td><td colspan="1" rowspan="1">60/60(1100%)</td><td colspan="1" rowspan="1">120/120100%[97.0%-100%]</td></tr><tr><td colspan="1" rowspan="1">Negative(no analyte)</td><td colspan="1" rowspan="1">NotDetected</td><td colspan="1" rowspan="1">100/100(1100%)</td><td colspan="1" rowspan="1">100/100100%)</td><td colspan="1" rowspan="1">100/100100%)</td><td colspan="1" rowspan="1">300/300(100%)</td><td colspan="1" rowspan="1">100/100(100%)</td><td colspan="1" rowspan="1">100/100100%)</td><td colspan="1" rowspan="1">100/100100%)</td><td colspan="1" rowspan="1">300/300(100%)</td><td colspan="1" rowspan="1">60/600100%[99.4%-100%]</td></tr><tr><td colspan="1" rowspan="2">HaemophilusinffluenzaeATCC 10211</td><td colspan="1" rowspan="1">1.32E+08CFU/mL</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">19/20(95.0%)</td><td colspan="1" rowspan="1">20/20(100%)</td><td colspan="1" rowspan="1">20/20(100%)</td><td colspan="1" rowspan="1">59/60(98.3%)</td><td colspan="1" rowspan="1">20/20(100%)</td><td colspan="1" rowspan="1">20/20(100%)</td><td colspan="1" rowspan="1">20/20(1100%)</td><td colspan="1" rowspan="1">60/60(100%)</td><td colspan="1" rowspan="1">119/12099.2%[99.4%-99.9%]</td></tr><tr><td colspan="1" rowspan="1">Negative(no analyte)</td><td colspan="1" rowspan="1">NotDetected</td><td colspan="1" rowspan="1">100/1000(1100%)</td><td colspan="1" rowspan="1">100/100(1100%)</td><td colspan="1" rowspan="1">100/100(100%)</td><td colspan="1" rowspan="1">300/300(100%)</td><td colspan="1" rowspan="1">100/100(1100%)</td><td colspan="1" rowspan="1">100/100(100%)</td><td colspan="1" rowspan="1">100/100(1100%)</td><td colspan="1" rowspan="1">300/300(100%)</td><td colspan="1" rowspan="1">600/600100%[99.4%-100%]</td></tr><tr><td colspan="1" rowspan="1">Neisseria meningitidis</td><td colspan="1" rowspan="1">Negative(no analyte)</td><td colspan="1" rowspan="1">NotDetected</td><td colspan="1" rowspan="1">120/120100%)</td><td colspan="1" rowspan="1">120/120(100%)</td><td colspan="1" rowspan="1">120/120(100%)</td><td colspan="1" rowspan="1">360/360(100%)</td><td colspan="1" rowspan="1">120/120100%)</td><td colspan="1" rowspan="1">120/120(100%)</td><td colspan="1" rowspan="1">120/120(100%)</td><td colspan="1" rowspan="1">360/360(100%)</td><td colspan="1" rowspan="1">720/720100%[99.5%-100%]</td></tr><tr><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="2">PseudomonasaeruginosaCDC-FDA AR Bank#0054</td><td colspan="1" rowspan="1">7.22E+07CFU/mL</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">19/20(95.0%)</td><td colspan="1" rowspan="1">20/20 (100%)</td><td colspan="1" rowspan="1">20/20 (100%)</td><td colspan="1" rowspan="1">59/60(98.3%)</td><td colspan="1" rowspan="1">20/20 (100%)</td><td colspan="1" rowspan="1">20/20 (100%)</td><td colspan="1" rowspan="1">20/20(1100%)</td><td colspan="1" rowspan="1">60/60(100%)</td><td colspan="1" rowspan="1"> 119/12099.2%[95.4-99.9%]</td></tr><tr><td colspan="1" rowspan="1">Negative(no analyte)</td><td colspan="1" rowspan="1">NotDetected</td><td colspan="1" rowspan="1">100/100(1100%)</td><td colspan="1" rowspan="1">100/100100%)</td><td colspan="1" rowspan="1">100/100(100%)</td><td colspan="1" rowspan="1">300/300(100%)</td><td colspan="1" rowspan="1">100/100(100%)</td><td colspan="1" rowspan="1">100/100(100%)</td><td colspan="1" rowspan="1">100/100100%)</td><td colspan="1" rowspan="1">300/300(100%)</td><td colspan="1" rowspan="1">600/600100%[99.4%-100%]</td></tr><tr><td colspan="1" rowspan="2">StenotrophomonasmaltohiliaATCC 700475</td><td colspan="1" rowspan="1">1.13E+09CFU/mL</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">20/20100%)</td><td colspan="1" rowspan="1">20/20100%)</td><td colspan="1" rowspan="1">20/20(100%)</td><td colspan="1" rowspan="1">60/60100%)</td><td colspan="1" rowspan="1">20/20(100%)</td><td colspan="1" rowspan="1">20/20(100%)</td><td colspan="1" rowspan="1">20/20100%)</td><td colspan="1" rowspan="1">60/60(100%)</td><td colspan="1" rowspan="1">1220/120100%[97.0%-100%]</td></tr><tr><td colspan="1" rowspan="1">Negative(no analyte)</td><td colspan="1" rowspan="1">NotDetected</td><td colspan="1" rowspan="1">100/100(100%)</td><td colspan="1" rowspan="1">100/100100%)</td><td colspan="1" rowspan="1">100/10000%)</td><td colspan="1" rowspan="1">300/300(100%)</td><td colspan="1" rowspan="1">100/100(10%)</td><td colspan="1" rowspan="1">100/100((100%)</td><td colspan="1" rowspan="1">100/100100%)</td><td colspan="1" rowspan="1">300/300(100%)</td><td colspan="1" rowspan="1">60/600100%[99.4%-100%]</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">nicrobial Resis</td><td colspan="1" rowspan="1">stance Gene</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="2">CTX-M(CTX-M-22)(Proteus mirabilis)GRE 254053</td><td colspan="1" rowspan="1">1.68E+09CF/mL</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">19/20(95.0%)</td><td colspan="1" rowspan="1">20/20(1100%)</td><td colspan="1" rowspan="1">20/20100%)</td><td colspan="1" rowspan="1">59/60(98.3%)</td><td colspan="1" rowspan="1">20/20(100%)</td><td colspan="1" rowspan="1">20/20100%)</td><td colspan="1" rowspan="1">20/20(100%)</td><td colspan="1" rowspan="1">60/60(100%)</td><td colspan="1" rowspan="1">119/12099.2%[95.4-%99.9%]</td></tr><tr><td colspan="1" rowspan="1">Negative(no analyte)</td><td colspan="1" rowspan="1">NotDetected</td><td colspan="1" rowspan="1">100/100(100%)</td><td colspan="1" rowspan="1">100/100100%)</td><td colspan="1" rowspan="1">100/100(100%)</td><td colspan="1" rowspan="1">300/300(100%)</td><td colspan="1" rowspan="1">100/100(100%)</td><td colspan="1" rowspan="1">100/100(100%)</td><td colspan="1" rowspan="1">100/100100%)</td><td colspan="1" rowspan="1">300/300(1100%)</td><td colspan="1" rowspan="1">600/600100%[99.4%-100%]</td></tr><tr><td colspan="1" rowspan="2">IMP(IMP-4)(Klebsiella aerogenes)CDC-FDA AR Bank#0161</td><td colspan="1" rowspan="1">9.29E+08FU/mL</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">20/20(100%)</td><td colspan="1" rowspan="1">20/20 (100%)</td><td colspan="1" rowspan="1">20/20 (100%)</td><td colspan="1" rowspan="1">60/60(100%)</td><td colspan="1" rowspan="1">20/20 (100%)</td><td colspan="1" rowspan="1">20/20 (100%)</td><td colspan="1" rowspan="1">20/20 (1100%)</td><td colspan="1" rowspan="1">60/60(100%)</td><td colspan="1" rowspan="1">120/120100%[97.0%-100%]</td></tr><tr><td colspan="1" rowspan="1">Negative(no analyte)</td><td colspan="1" rowspan="1">NotDetected</td><td colspan="1" rowspan="1">100/100100%)</td><td colspan="1" rowspan="1">100/100100%)</td><td colspan="1" rowspan="1">100/100100%)</td><td colspan="1" rowspan="1">300/300(100%)</td><td colspan="1" rowspan="1">100/100(100%)</td><td colspan="1" rowspan="1">100/100100%)</td><td colspan="1" rowspan="1">100/100100%)</td><td colspan="1" rowspan="1">300/300(100%)</td><td colspan="1" rowspan="1">600/600100%[99.4%-100%]</td></tr><tr><td colspan="1" rowspan="2">KPC-3(Klebsiella oxytoca)ATCC 10211</td><td colspan="1" rowspan="1">2.44E+07CF/mL</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">20/20100%)</td><td colspan="1" rowspan="1">20/20100%)</td><td colspan="1" rowspan="1">20/20(100%)</td><td colspan="1" rowspan="1">60/60100%)</td><td colspan="1" rowspan="1">20/20(100%)</td><td colspan="1" rowspan="1">20/20(100%)</td><td colspan="1" rowspan="1">20/20(100%)</td><td colspan="1" rowspan="1">60/60(100%)</td><td colspan="1" rowspan="1">1220/120100%[97.0%-100%]</td></tr><tr><td colspan="1" rowspan="1">Negative(no analyte)</td><td colspan="1" rowspan="1">NotDetected</td><td colspan="1" rowspan="1">100/100(100%)</td><td colspan="1" rowspan="1">100/100100%)</td><td colspan="1" rowspan="1">100/100(100%)</td><td colspan="1" rowspan="1">300/300(100%)</td><td colspan="1" rowspan="1">100/100100%)</td><td colspan="1" rowspan="1">100/100(100%)</td><td colspan="1" rowspan="1">100/100(100%)</td><td colspan="1" rowspan="1">300/300(100%)</td><td colspan="1" rowspan="1">600/600100%[99.4%-100%]</td></tr><tr><td colspan="1" rowspan="2">mcr-1(Escherichia coli)CDC-FDA AR Bank#0350</td><td colspan="1" rowspan="1">8.79E+08CFU/mL</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">20/20 (100%)</td><td colspan="1" rowspan="1">20/20100%)</td><td colspan="1" rowspan="1">19/20d(95.0%)</td><td colspan="1" rowspan="1">59/60(98.3%)</td><td colspan="1" rowspan="1">20/20(100%)</td><td colspan="1" rowspan="1">20/20(100%)</td><td colspan="1" rowspan="1">20/20(100%)</td><td colspan="1" rowspan="1">60/60(100%)</td><td colspan="1" rowspan="1">119/12099.2%%[95.4%-99.9%]</td></tr><tr><td colspan="1" rowspan="1">Negative(no analyte)</td><td colspan="1" rowspan="1">NotDetectedor N/A</td><td colspan="1" rowspan="1">100/100100%)</td><td colspan="1" rowspan="1">100/100(1100%)</td><td colspan="1" rowspan="1">100/100(100%)</td><td colspan="1" rowspan="1">300/300100%)</td><td colspan="1" rowspan="1">100/100(100%)</td><td colspan="1" rowspan="1">100/100(100%)</td><td colspan="1" rowspan="1">100/1001000%)</td><td colspan="1" rowspan="1">300/300(100%)</td><td colspan="1" rowspan="1">600/60000%[99.4%-100%]</td></tr><tr><td colspan="1" rowspan="1">mecA/C</td><td colspan="1" rowspan="1">Negative(no analyte)</td><td colspan="1" rowspan="1">NotDetectedor N/A</td><td colspan="1" rowspan="1">120/120(100%)</td><td colspan="1" rowspan="1">120/120(100%)</td><td colspan="1" rowspan="1">120/120(100%)</td><td colspan="1" rowspan="1">360/360100%)</td><td colspan="1" rowspan="1">120/120(100%)</td><td colspan="1" rowspan="1">120/120(100%)</td><td colspan="1" rowspan="1">120/120100%)</td><td colspan="1" rowspan="1">360/360(100%)</td><td colspan="1" rowspan="1">720/720100%[99.5%-100%]</td></tr><tr><td colspan="1" rowspan="2">mecA/C and MREJ(MRSA)(Staphylococcusaureus)ATCC BAA-38</td><td colspan="1" rowspan="1">2.44E+08CF/mL</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">19/20(95.0%)</td><td colspan="1" rowspan="1">20/20100%)</td><td colspan="1" rowspan="1">20/20(100%)</td><td colspan="1" rowspan="1">59/60(98.3%)</td><td colspan="1" rowspan="1">20/20(100%)</td><td colspan="1" rowspan="1">20/20(100%)</td><td colspan="1" rowspan="1">20/20(100%)</td><td colspan="1" rowspan="1">60/60(100%)</td><td colspan="1" rowspan="1">119/12099.2%[95.4-99.9%]</td></tr><tr><td colspan="1" rowspan="1">Negative(no analyte)</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">100/100(1100%)</td><td colspan="1" rowspan="1">100/100100%)</td><td colspan="1" rowspan="1">100/100100%)</td><td colspan="1" rowspan="1">300/300(100%)</td><td colspan="1" rowspan="1">100/100(100%)</td><td colspan="1" rowspan="1">100/100100%)</td><td colspan="1" rowspan="1">100/100(100%)</td><td colspan="1" rowspan="1">300/300(100%)</td><td colspan="1" rowspan="1">600/600100%[99.4%-100%]</td></tr><tr><td colspan="1" rowspan="1">NDM(NDM-1)</td><td colspan="1" rowspan="1">7.36E+07CF/mML</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">19/205.0%)</td><td colspan="1" rowspan="1">20/20(100%)</td><td colspan="1" rowspan="1">20/20100%)</td><td colspan="1" rowspan="1">59/60(98.3%)</td><td colspan="1" rowspan="1">20/20(100%)</td><td colspan="1" rowspan="1">20/20(100%)</td><td colspan="1" rowspan="1">20/20(100%)</td><td colspan="1" rowspan="1">60/60(100%)</td><td colspan="1" rowspan="1">119/12099.2%[95.4-99.9%]</td></tr><tr><td colspan="1" rowspan="3">Analyte(Type/Species)Source ID</td><td colspan="1" rowspan="3">ConcentrationTested</td><td colspan="1" rowspan="3">ExpectedResult</td><td colspan="8" rowspan="1">Agreement with Expected Result</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="3" rowspan="1">FilmArray 2.0</td><td colspan="4" rowspan="1">FilmArray Torch</td><td colspan="1" rowspan="2">All Sites/Systems [95%Confidence Interval</td></tr><tr><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">SystemTotal</td><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">SystemTotal</td></tr><tr><td colspan="1" rowspan="1">(Acinetobacterbaumanni)CDC-FDA AR Bank#0033</td><td colspan="1" rowspan="1">Negative(no analyte)</td><td colspan="1" rowspan="1">NotDetected</td><td colspan="1" rowspan="1">100/100(1100%)</td><td colspan="1" rowspan="1">100/100(1100%)</td><td colspan="1" rowspan="1">100/100(100%)</td><td colspan="1" rowspan="1">300/300(1100%)</td><td colspan="1" rowspan="1">100/100(1100%)</td><td colspan="1" rowspan="1">100/100(100%)</td><td colspan="1" rowspan="1">100/100(1100%)</td><td colspan="1" rowspan="1">300/300(100%)</td><td colspan="1" rowspan="1">600/600610%[99.4%-100%]</td></tr><tr><td colspan="1" rowspan="2">OXA-48-like(OXA-48)(Serratia marcescens)GRE 1659004</td><td colspan="1" rowspan="1">3.05E+07F/mL</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">19/2095.0%)</td><td colspan="1" rowspan="1">20/20(100%)</td><td colspan="1" rowspan="1">20/20(100%)</td><td colspan="1" rowspan="1">59/60(98.3%)</td><td colspan="1" rowspan="1">20/20(1100%)</td><td colspan="1" rowspan="1">20/20(100%)</td><td colspan="1" rowspan="1">20/20(100%)</td><td colspan="1" rowspan="1">60/60(100%)</td><td colspan="1" rowspan="1">119/1209.2%[95.4-99.9%]</td></tr><tr><td colspan="1" rowspan="1">Negative(no analyte)</td><td colspan="1" rowspan="1">NotDetectedor N/A</td><td colspan="1" rowspan="1">100/100(1100%)</td><td colspan="1" rowspan="1">100/100(1100%)</td><td colspan="1" rowspan="1">100/100(100%)</td><td colspan="1" rowspan="1">300/300(1100%)</td><td colspan="1" rowspan="1">100/100100%)</td><td colspan="1" rowspan="1">100/100(100%)</td><td colspan="1" rowspan="1">100/100(100%)</td><td colspan="1" rowspan="1">300/300(1100%)</td><td colspan="1" rowspan="1">600/600100%[99.4%-100%]</td></tr><tr><td colspan="1" rowspan="2">vanA/B(vanB)(Enterococcus faecalis)ATCC 51299</td><td colspan="1" rowspan="1">7.65E+08FU/mL</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">20/20(1100%)</td><td colspan="1" rowspan="1">20/20(1100%)</td><td colspan="1" rowspan="1">20/20(100%)</td><td colspan="1" rowspan="1">60/60(100%)</td><td colspan="1" rowspan="1">20/20(1100%)</td><td colspan="1" rowspan="1">20/20(100%)</td><td colspan="1" rowspan="1">20/20(100%)</td><td colspan="1" rowspan="1">60/60(100%)</td><td colspan="1" rowspan="1">120/120100%[97.0%-100%]</td></tr><tr><td colspan="1" rowspan="1">Negative(no analyte)</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">100/100(100%)</td><td colspan="1" rowspan="1">100/100(100%)</td><td colspan="1" rowspan="1">100/100(100%)</td><td colspan="1" rowspan="1">300/300(100%)</td><td colspan="1" rowspan="1">100/100(1100%)</td><td colspan="1" rowspan="1">100/100(100%)</td><td colspan="1" rowspan="1">100/100(1100%)</td><td colspan="1" rowspan="1">300/300(100%)</td><td colspan="1" rowspan="1">600/600100%[99.4%-100%]</td></tr><tr><td colspan="1" rowspan="2">VIM(VIM-4)(Pseudomonasaeruginosa)CDC-FDA AR Bank#0054</td><td colspan="1" rowspan="1">7.22E+07CFU/mL</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">20/20(100%)</td><td colspan="1" rowspan="1">20/20 (100%)</td><td colspan="1" rowspan="1">20/20100%)</td><td colspan="1" rowspan="1">60/60(100%)</td><td colspan="1" rowspan="1">20/20(100%)</td><td colspan="1" rowspan="1">20/20(100%)</td><td colspan="1" rowspan="1">20/20100%)</td><td colspan="1" rowspan="1">60/60(100%)</td><td colspan="1" rowspan="1">120/120100%[97.0%-100%]</td></tr><tr><td colspan="1" rowspan="1">Negative(no analyte)</td><td colspan="1" rowspan="1">NotDetected</td><td colspan="1" rowspan="1">100/100(1100%)</td><td colspan="1" rowspan="1">100/100(100%)</td><td colspan="1" rowspan="1">100/100(100%)</td><td colspan="1" rowspan="1">300/300(100%)</td><td colspan="1" rowspan="1">100/100(100%)</td><td colspan="1" rowspan="1">100/100(100%)</td><td colspan="1" rowspan="1">100/100100%)</td><td colspan="1" rowspan="1">300/300(100%)</td><td colspan="1" rowspan="1">600/600100%[99.4%-100%]</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">—</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Yeast</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">—</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="2">Candida albicansATCC 90028</td><td colspan="1" rowspan="1">1.76E+05CFU/mL</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">20/20 (1100%)</td><td colspan="1" rowspan="1">20/20(1100%)</td><td colspan="1" rowspan="1">19/20(95.0%)</td><td colspan="1" rowspan="1">59/60(98.3%)</td><td colspan="1" rowspan="1">20/20 (1100%)</td><td colspan="1" rowspan="1">20/20 (100%)</td><td colspan="1" rowspan="1">20/20(100%)</td><td colspan="1" rowspan="1">60/60(1100%)</td><td colspan="1" rowspan="1">119/12099.2%[95.4%-99.9%]</td></tr><tr><td colspan="1" rowspan="1">Negative(no analyte)</td><td colspan="1" rowspan="1">NotDetected</td><td colspan="1" rowspan="1">100/100(1100%)</td><td colspan="1" rowspan="1">100/100(100%)</td><td colspan="1" rowspan="1">100/100(100%)</td><td colspan="1" rowspan="1">300/300(100%)</td><td colspan="1" rowspan="1">100/100(100%)</td><td colspan="1" rowspan="1">100/100(100%)</td><td colspan="1" rowspan="1">100/100(100%)</td><td colspan="1" rowspan="1">300/300(1100%)</td><td colspan="1" rowspan="1">600/600100%[99.4%-100%]</td></tr><tr><td colspan="1" rowspan="2">Candida aurisCDC-FDA AR Bank#0381</td><td colspan="1" rowspan="1">3.49E+07CFU/mL</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">19/20(95.0%)</td><td colspan="1" rowspan="1">20/20 (100%)</td><td colspan="1" rowspan="1">20/20(100%)</td><td colspan="1" rowspan="1">59/60(98.3%)</td><td colspan="1" rowspan="1">20/20 (100%)</td><td colspan="1" rowspan="1">20/20(100%)</td><td colspan="1" rowspan="1">20/20(100%)</td><td colspan="1" rowspan="1">60/60(100%)</td><td colspan="1" rowspan="1">19/12099.2%[95.4%-99.9%]</td></tr><tr><td colspan="1" rowspan="1">Negative(no analyte)</td><td colspan="1" rowspan="1">NotDetected</td><td colspan="1" rowspan="1">100/1000100%)</td><td colspan="1" rowspan="1">100/100100%)</td><td colspan="1" rowspan="1">100/100(100%)</td><td colspan="1" rowspan="1">300/300(1100%)</td><td colspan="1" rowspan="1">100/100(1100%)</td><td colspan="1" rowspan="1">100/100(1100%)</td><td colspan="1" rowspan="1">100/100(1100%)</td><td colspan="1" rowspan="1">300/300(1100%)</td><td colspan="1" rowspan="1">600/600100%[99.4%-100%]</td></tr><tr><td colspan="1" rowspan="2">Candida glabrataATCC 15545</td><td colspan="1" rowspan="1">3.82E+07F/mML</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">20/20(100%)</td><td colspan="1" rowspan="1">18/20(90.%)</td><td colspan="1" rowspan="1">20/20 (100%)</td><td colspan="1" rowspan="1">58/6096.7%)</td><td colspan="1" rowspan="1">20/20(1100%)</td><td colspan="1" rowspan="1">20/20(100%)</td><td colspan="1" rowspan="1">20/20 (100%)</td><td colspan="1" rowspan="1">60/60(100%)</td><td colspan="1" rowspan="1">1118/120988.3%[94.1%-99.8%]</td></tr><tr><td colspan="1" rowspan="1">Negative(no analyte)</td><td colspan="1" rowspan="1">NotDetected</td><td colspan="1" rowspan="1">100/100(100%)</td><td colspan="1" rowspan="1">100/100(100%)</td><td colspan="1" rowspan="1">100/100100%)</td><td colspan="1" rowspan="1">300/300(100%)</td><td colspan="1" rowspan="1">100/100(100%)</td><td colspan="1" rowspan="1">100/100100%)</td><td colspan="1" rowspan="1">100/100(100%)</td><td colspan="1" rowspan="1">300/300(100%)</td><td colspan="1" rowspan="1">600/600100%[99.4%-100%]</td></tr><tr><td colspan="1" rowspan="2">Candida kruseiATCC 6258</td><td colspan="1" rowspan="1">2.48E+05F/mL</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">18/20(90.0%)</td><td colspan="1" rowspan="1">20/20(100%)</td><td colspan="1" rowspan="1">20/20(100%)</td><td colspan="1" rowspan="1">58/60(96.7%)</td><td colspan="1" rowspan="1">20/20(100%)</td><td colspan="1" rowspan="1">20/20(100%)</td><td colspan="1" rowspan="1">20/2(100%)</td><td colspan="1" rowspan="1">60/60(100%)</td><td colspan="1" rowspan="1">118/12098.%[94.1%-99.8%]</td></tr><tr><td colspan="1" rowspan="1">Negative(no analyte)</td><td colspan="1" rowspan="1">NotDetected</td><td colspan="1" rowspan="1">100/1000(1100%)</td><td colspan="1" rowspan="1">100/100(1100%)</td><td colspan="1" rowspan="1">100/100(100%)</td><td colspan="1" rowspan="1">300/300(1100%)</td><td colspan="1" rowspan="1">100/1000(100%)</td><td colspan="1" rowspan="1">100/1000(100%)</td><td colspan="1" rowspan="1">100/100(100%)</td><td colspan="1" rowspan="1">300/300(1100%)</td><td colspan="1" rowspan="1">600/600100%[99.4%-100%]</td></tr><tr><td colspan="1" rowspan="1">Candida parapsilosisATCC 34136</td><td colspan="1" rowspan="1">2.91E+05CFU/mL</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">20/20(100%)</td><td colspan="1" rowspan="1">20/20(100%)</td><td colspan="1" rowspan="1">20/20(1100%)</td><td colspan="1" rowspan="1">60/60(1100%)</td><td colspan="1" rowspan="1">20/20 (1100%)</td><td colspan="1" rowspan="1">20/20(100%)</td><td colspan="1" rowspan="1">20/20(100%)</td><td colspan="1" rowspan="1">60/60(1100%)</td><td colspan="1" rowspan="1">120/120100%[99.4%-100%]</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td></td><td></td><td></td></tr><tr><td rowspan="2">Analyte</td><td rowspan="2"></td><td rowspan="2"></td><td colspan="8">Agreement with Expected Result</td><td rowspan="2"></td></tr><tr><td></td><td>FilmArray 2.0</td><td></td><td>System</td><td></td><td>FilmArray Torch</td><td></td><td>System</td></tr><tr><td>(Type/Species) Source ID</td><td>Concentration Tested</td><td>Expected Result</td><td>Site A</td><td>Site B</td><td>Site C</td><td>Total</td><td>Site A</td><td>Site B</td><td>Site C</td><td>Total</td><td>All Sites/Systems [95% Confidence Interval] 600/6600</td></tr><tr><td rowspan="3">Candida tropicalis</td><td>Negative (no analyte)</td><td>Not Detected</td><td>100/100 (100%)</td><td>100/100 (100%)</td><td>100/100 (1100%)</td><td>300/300 (100%)</td><td>100/100 (100%)</td><td>100/100 100%)</td><td>100/100 (100%)</td><td>300/300 (100%)</td><td>101% [99.4%-100%]</td></tr><tr><td>Negative</td><td>Not</td><td>120/120</td><td>120/120</td><td>120/120</td><td>360/360</td><td>120/120</td><td>120//120</td><td>120/120</td><td>360/360</td><td>720/720</td></tr><tr><td>(no analyte)</td><td>Detected</td><td>(100%)</td><td>100%)</td><td>(100%)</td><td>(100%)</td><td>(1100%)</td><td>(1100%)</td><td> (1100%)</td><td>(100%)</td><td>100% [99.5%-100%]</td></tr><tr><td>Cryptococcus neoformans/gattii</td><td>1.20E+07</td><td>Detected</td><td>20/20</td><td>20/20</td><td>20/20</td><td>60/60</td><td>20/20</td><td>20/20</td><td>20/20</td><td>60/60</td><td>120/120 100%</td></tr><tr><td>(Cryptococcus neoformmans)</td><td>CFU/mL</td><td></td><td>(100%)</td><td>(100%)</td><td>(100%)</td><td>(100%)</td><td>(1100%)</td><td>(100%)</td><td>(100%)</td><td>(100%)</td><td>[99.4%-100%] 600/600</td></tr><tr><td>ATCC MYA-4564</td><td>Negative (no analyte)</td><td>Not Detected</td><td>100/1000 (1100%)</td><td>100/1000 (1100%)</td><td>100/100 (1100%)</td><td>300/300 (100%)</td><td>100 /100 (1100%)</td><td>10010 /00 (1100%)</td><td>100 /100</td><td>300/300</td><td>100%</td></tr><tr><td rowspan="5">Overall Agreement with the Expected Results (All Analytes/All Test Concentrations)</td><td rowspan="5"></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td> (1100%)</td><td>(100%)</td><td>[99.4%-100%]</td></tr><tr><td></td><td>5148/5160</td><td>5157/5160</td><td>5157/5160</td><td>15461/ 15480</td><td>5160/5160</td><td>5160/5160</td><td>5160/5160</td><td>15480 15480</td><td>30941/30960 99.94%</td></tr><tr><td></td><td>99.8%</td><td>99.9%</td><td>99.9%</td><td>99.9%</td><td>100%</td><td>100%</td><td>100%</td><td>100%</td><td></td></tr><tr><td></td><td>99.6%</td><td>9.8%</td><td>9.8%</td><td>99.8%</td><td>999%</td><td>999%</td><td>9.9%</td><td>99.9%</td><td>[99.90%-99.96%]</td></tr><tr><td>[95% Confidence Interval]</td><td>a 9.9%</td><td>9.9%]</td><td>9.9%]</td><td>9.9%]</td><td> 100%</td><td> 100%</td><td> 100%]</td><td> 100%</td><td></td></tr></table>

Sccusaarsaplestagoiadcocsynatciorctis. sections.

# Interference

Potentially interfering substances that could be present in blood culture specimens were evaluated for their effect on BioFire BCID2 Panel performance. The substances tested included endogenous blood components (e.g. albumin, gamma-globulin, human genomic DNA), exogenous substances (e.g. prescribed or over-the-counter medications, blood anticoagulants, substances used to decontaminate or disinfect work areas, etc.), potentially competing commensal or infectious microorganisms, and various bottle/culture media.

Each substance was added to contrived samples containing representative panel organisms at concentrations near LoD (at least 10-fold lower than organism concentrations in a positive blood culture). The concentration of the substances or competing microorganisms added to the samples was equal to or greater than the highest level expected be in a blood culture.

Valid and accurate results were obtained for each sample containing endogenous and exogenous substances as well as various anticoagulants and disinfectants at the concentrations indicated in Table 118 (no interference).

Table 118. Endogenous Substances, Exogenous Substances, Anticoagulants, and Disinfectants Tested - No Interference Observed   

<table><tr><td rowspan=1 colspan=1>Substance Tested</td><td rowspan=1 colspan=1>Reference Rangein Whole Blooda</td><td rowspan=1 colspan=1>Concentration Expected inPositive Blood Culture</td><td rowspan=1 colspan=1>Test Concentration</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Endogenous</td><td rowspan=1 colspan=1>Substances</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Albumin</td><td rowspan=1 colspan=1>35 - 52 mg/mL</td><td rowspan=1 colspan=1>7.0 - 10.4 mg/mL</td><td rowspan=1 colspan=1>15 mg/mLc</td></tr><tr><td rowspan=1 colspan=1>Bilirubin (unconjugated)</td><td rowspan=1 colspan=1>0.0 - 0.2 mg/mL</td><td rowspan=1 colspan=1>0.00 - 0.04 mg/mL</td><td rowspan=1 colspan=1>0.40 mg/mL</td></tr><tr><td rowspan=1 colspan=1>Cholesterol</td><td rowspan=1 colspan=1>&lt;2.0 mg/mL</td><td rowspan=1 colspan=1>&lt;0.4 mg/mL</td><td rowspan=1 colspan=1>4.0 mg/mL</td></tr><tr><td rowspan=1 colspan=1>Fibrinogen</td><td rowspan=1 colspan=1>1.6 - 4.3 mg/mLd</td><td rowspan=1 colspan=1>0.32 - 0.86 mg/mL</td><td rowspan=1 colspan=1>4.3 mg/mL</td></tr><tr><td rowspan=1 colspan=1>Gamma-globulin</td><td rowspan=1 colspan=1>7 - 16 mg/mL</td><td rowspan=1 colspan=1>1.4 - 3.2 mg/mL</td><td rowspan=1 colspan=1>20 mg/mL</td></tr><tr><td rowspan=1 colspan=1>D-Glucose</td><td rowspan=1 colspan=1>0.74 - 1.00 mg/mL</td><td rowspan=1 colspan=1>0.15 - 0.20 mg/mL</td><td rowspan=1 colspan=1>10 mg/mL</td></tr><tr><td rowspan=1 colspan=1>Hemoglobin</td><td rowspan=1 colspan=1>&lt;0.03 mg/mL</td><td rowspan=1 colspan=1>&lt;0.006 mg/mL</td><td rowspan=1 colspan=1>10 mg/mL</td></tr><tr><td rowspan=1 colspan=1>Magnesium (MgSO4)</td><td rowspan=1 colspan=1>0.017 - 0.024 mg/mL</td><td rowspan=1 colspan=1>0.0034 - 0.0048 mg/mL</td><td rowspan=1 colspan=1>0.1 mg/mL</td></tr><tr><td rowspan=1 colspan=1>Triglycerides</td><td rowspan=1 colspan=1>&lt;1.50 mg/mL</td><td rowspan=1 colspan=1>&lt;0.30 mg/mL</td><td rowspan=1 colspan=1>15 mg/mL</td></tr><tr><td rowspan=1 colspan=1>Human Genomic DNA(e.g. nucleic acids from whiteblood cells)</td><td rowspan=1 colspan=1>4.0E+06 - 1.0E+07 cells/mL</td><td rowspan=1 colspan=1>8.0E+05-2.0E+06 cells/mL</td><td rowspan=1 colspan=1>0.07 mg/mL(1.0E+07 cells/mL)f</td></tr><tr><td rowspan=1 colspan=4>Exogenous Substances</td></tr><tr><td rowspan=1 colspan=4>Over-the-Counter Medications and Supplements</td></tr><tr><td rowspan=1 colspan=1>Acetaminophen</td><td rowspan=1 colspan=1>0.052 mg/mL</td><td rowspan=1 colspan=1>0.0104 mg/mL</td><td rowspan=1 colspan=1>0.156 mg/mL</td></tr><tr><td rowspan=1 colspan=1>Salicylic Acid</td><td rowspan=1 colspan=1>9.52 μg/mL</td><td rowspan=1 colspan=1>1.90 μg/mL</td><td rowspan=1 colspan=1>28.6 μg/mL</td></tr><tr><td rowspan=1 colspan=1>Ibuprofen</td><td rowspan=1 colspan=1>0.073 mg/mL</td><td rowspan=1 colspan=1>0.015 mg/mL</td><td rowspan=1 colspan=1>0.219 mg/mL</td></tr><tr><td rowspan=1 colspan=1>Biotin (Vitamin B7)</td><td rowspan=1 colspan=1>1.71 μg/mL</td><td rowspan=1 colspan=1>0.342 ug/mL</td><td rowspan=1 colspan=1>3.51 μg/mL</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Ara-C Triphosphate9(active metabolite ofCytarabine</td><td rowspan=1 colspan=1>Cmax = 13.2 μg/mLh</td><td rowspan=1 colspan=1>Cmax = 2.64 μg/mL</td><td rowspan=1 colspan=1>2.64 μg/mLi</td></tr><tr><td rowspan=1 colspan=2>Substance Tested</td><td rowspan=1 colspan=2>Test Concentration</td></tr><tr><td></td><td></td><td></td><td></td></tr><tr><td rowspan=1 colspan=2>Sodium Citrate</td><td rowspan=1 colspan=2>4x the vacutainer-specific ratio (anticoagulant:blood)</td></tr><tr><td rowspan=1 colspan=2>K2EDTA</td><td rowspan=1 colspan=2>4× the vacutainer-specific ratio (anticoagulant:blood)</td></tr><tr><td rowspan=1 colspan=2>K3EDTA</td><td rowspan=1 colspan=2>4× the vacutainer-specific ratio (anticoagulant:blood)</td></tr><tr><td rowspan=1 colspan=2>Lithium Heparin</td><td rowspan=1 colspan=2>4x the vacutainer-specific ratio (anticoagulant:blood)</td></tr><tr><td rowspan=1 colspan=2>Sodium Heparin</td><td rowspan=1 colspan=2>4× the vacutainer-specific ratio (anticoagulant:blood)</td></tr><tr><td rowspan=1 colspan=2>Acid Citrate Dextrose (ACD)</td><td rowspan=1 colspan=2>4× the vacutainer-specific ratio (anticoagulant:blood)</td></tr><tr><td rowspan=1 colspan=2>Sodium Polyethol Sulfate (SPS)</td><td rowspan=1 colspan=2>4× the vacutainer-specific ratio (anticoagulant:blood)</td></tr><tr><td></td><td></td><td></td><td></td></tr><tr><td rowspan=1 colspan=2>Bleach</td><td rowspan=1 colspan=2>5% v/v (3,000 ppm)</td></tr><tr><td rowspan=1 colspan=2>Ethanol</td><td rowspan=1 colspan=2>7% V/v</td></tr><tr><td rowspan=1 colspan=2>Povidone (iodinated)</td><td rowspan=1 colspan=2>1% v/v</td></tr></table>

a EP37: Supplemental Table for Interference Testing in Clinical Chemistry – First Edition (2018), unless otherwise noted b Calculated as a 5-fold dilution of the whole blood reference range (blood diluted in bottle media) c Albumin testing was performed over a range of concentrations; an effect on detection near LoD was observed at $> 1 5 \mathrm { \ m g / m L }$ d Wakeman, LJ et al, Fibrinogen reference range in adolescents. Blood 112, 4091 (2008) e EP07: Interference Testing in Clinical Chemistry; Approved Guideline—Third Edition (2018); values for IgG f Cells/mL calculated from mass and human genome size g Pyrimidine analog that inhibits DNA polymerase activity h Liston, D.R. & Davis, M. Clinically relevant concentrations of anticancer drugs: a guide for nonclinical studies. Clin Cancer Research 23, 3489-3498 (2017). $120 \%$ of $C _ { m a x }$ represents the maximum expected concentration of drug metabolite in blood (McEvoy, G.K. (ed.). American Hospital Formulary Service. AHFS Drug Information. American Society of Health-System Pharmacists, Bethesda, MD. 2007, p. 994)

Valid and accurate results were obtained for each sample spiked with the high concentrations of potentially competing microorganism indicated in Table 119 (no interference).

Table 119. Competing Microorganisms Tested - No Interference Observed   

<table><tr><td rowspan=1 colspan=1>Substance Tested</td><td rowspan=1 colspan=1>Test Concentration</td></tr><tr><td rowspan=1 colspan=2>On-Panel</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus epidermidis</td><td rowspan=1 colspan=1>8.78E+08 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Escherichia coli</td><td rowspan=1 colspan=1>1.38E+09 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Streptococcus mitis</td><td rowspan=1 colspan=1>6.95E+08 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Stenotrophomonas maltophilia</td><td rowspan=1 colspan=1>7.40E+08 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Bacteroides fragilis</td><td rowspan=1 colspan=1>5.40E+07 cells/mLa</td></tr><tr><td></td><td></td></tr><tr><td rowspan=1 colspan=1>Corynebacterium jeikeium</td><td rowspan=1 colspan=1>8.70E+08 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Bacillus cereus</td><td rowspan=1 colspan=1>8.40E+08 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Micrococcus luteus</td><td rowspan=1 colspan=1>8.40E+08 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Clostridium perfringens</td><td rowspan=1 colspan=1>1.76E+08 cells/mLa</td></tr><tr><td rowspan=1 colspan=1>Propionibacterium acnes</td><td rowspan=1 colspan=1>1.12E+07 cells/mLa</td></tr></table>

a Culture concentration in cells/mL determined via optical density $( \mathsf { O D } 6 0 0 )$

Valid and accurate results were obtained for each sample prepared in 1:1 (v/v) ratio with the various bottle/culture media (aerobic, anaerobic, pediatric, and myco media formulations) indicated in Table 120 (no interference).

Table 120. Blood Culture Media Tested - No Interference Observed   

<table><tr><td rowspan=1 colspan=2>Bottle Type/Description</td><td rowspan=1 colspan=1>Bottle Media Tested</td></tr><tr><td rowspan=6 colspan=1>Aaroe</td><td rowspan=1 colspan=1>Standard aerobic</td><td rowspan=1 colspan=1>bioMérieux BacT/ALERT® SA</td></tr><tr><td rowspan=1 colspan=1>Aerobic (with adsorbent polymeric beads)</td><td rowspan=1 colspan=1>bioMérieux BacT/ALERT® FA Plus</td></tr><tr><td rowspan=1 colspan=1>Aerobic (with activated charcoal)a</td><td rowspan=1 colspan=1>bioMérieux BacT/ALERT® FA a</td></tr><tr><td rowspan=1 colspan=1>Standard aerobic</td><td rowspan=1 colspan=1>BD BACTEC™ Standard/10 Aerobic/F</td></tr><tr><td rowspan=1 colspan=1>Aerobic (with resin for antibiotic neutralization)</td><td rowspan=1 colspan=1>BD BACTEC™ Plus Aerobic/F</td></tr><tr><td rowspan=1 colspan=1>Aerobic</td><td rowspan=1 colspan=1>Thermo Scientific™&quot; VersaTrek™&quot; REDOX™ 1</td></tr><tr><td rowspan=7 colspan=1>Aaaoe</td><td rowspan=1 colspan=1>Standard anaerobic</td><td rowspan=1 colspan=1>bioMérieux BacT/ALERT® SN</td></tr><tr><td rowspan=1 colspan=1>Anaerobic (with adsorbent polymeric beads)</td><td rowspan=1 colspan=1>bioMérieux BacT/ALERT® FN Plus</td></tr><tr><td rowspan=1 colspan=1>Anaerobic (with activated charcoal)a</td><td rowspan=1 colspan=1>bioMérieux BacT/ALERT® FNa</td></tr><tr><td rowspan=1 colspan=1>Standard anaerobic</td><td rowspan=1 colspan=1>BD BACTEC™ Standard Anaerobic/F</td></tr><tr><td rowspan=1 colspan=1>Anaerobic (with resin for antibiotic neutralization)</td><td rowspan=1 colspan=1>BD BACTEC™ Plus Anaerobic/F</td></tr><tr><td rowspan=1 colspan=1>Lytic anaerobic</td><td rowspan=1 colspan=1>BD BACTEC™&quot; Lytic/10 Anaerobic/F</td></tr><tr><td rowspan=1 colspan=1>Anaerobic</td><td rowspan=1 colspan=1>Thermo Scientific™&quot; VersaTrek™&quot; REDOX™ 2</td></tr><tr><td rowspan=3 colspan=1>peeiid</td><td rowspan=1 colspan=1>Pediatric (with adsorbent polymeric beads)</td><td rowspan=1 colspan=1>bioMérieux BacT/ALERT® PF Plus</td></tr><tr><td rowspan=1 colspan=1>Pediatric (with activated charcoal)a</td><td rowspan=1 colspan=1>bioMérieux BacT/ALERT® PFa</td></tr><tr><td rowspan=1 colspan=1>Pediatric (with resin for antibiotic neutralization)</td><td rowspan=1 colspan=1>BD BACTEC™ Peds Plus/F</td></tr><tr><td rowspan=2 colspan=1>oG</td><td rowspan=1 colspan=1>Mycobacteriab</td><td rowspan=1 colspan=1>bioMérieux BacT/ALERT® MPb</td></tr><tr><td rowspan=1 colspan=1>Mycobacteria and fungi</td><td rowspan=1 colspan=1>BD BACTEC™ Media Myco/F Lytic</td></tr></table>

a Bottles containing activated charcoal are not recommended for use with the BioFire BCID2 Panel b Bottle is not intended for use with blood specimens

# NOTE: The BioFire BCID2 Panel is not intended for testing of whole blood samples.

NOTE: The BioFire BCID2 Panel is not intended for use with media containing charcoal or culture bottle containing body fluids/specimens other than blood.

NOTE: Avoid contact between samples and bleach prior to testing (bleach can damage nucleic acids).

# External Control Material

External Controls should be used in accordance with laboratory protocols and the appropriate accrediting organization requirements, as applicable. Molecular grade water or saline can be used as an external negative control. Previously characterized positive samples or negative samples spiked with well-characterized organisms can be used as external positive controls.

It is ultimately the responsibility of each laboratory to determine the frequency of external control testing with the BioFire BCID2 Panel as part of the laboratory’s Quality Control program.