# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATION DECISION SUMMARY ASSAY ONLY

I Background Information:

# B Applicant

bioMérieux S.A.

# C Proprietary and Established Names

ETEST Plazomicin (PLZ) $( 0 . 0 1 6 { - } 2 5 6 ~ \mu \mathrm { g / m L } )$

# D Regulatory Information

<table><tr><td rowspan=1 colspan=1>ProductCode(s)</td><td rowspan=1 colspan=1>Classification</td><td rowspan=1 colspan=1>RegulationSection</td><td rowspan=1 colspan=1>Panel</td></tr><tr><td rowspan=1 colspan=1>JWY</td><td rowspan=1 colspan=1>Class Ⅱ</td><td rowspan=1 colspan=1>21 CFR 866.1640 -AntimicrobialSusceptibility Test Powder</td><td rowspan=1 colspan=1>MI- Microbiology</td></tr></table>

# II Submission/Device Overview:

# A Purpose for Submission:

To obtain a substantial equivalence determination for plazomicin at concentrations of 0.016 – $2 5 6 ~ \mu \mathrm { g / m L }$ for susceptibility testing with ETEST.

# B Measurand:

Plazomicin $0 . 0 1 6 - 2 5 6 ~ \mu \mathrm { g / m L }$

# C Type of Test:

Quantitative Antimicrobial Susceptibility Test growth-based detection

# III Intended Use/Indications for Use:

A Intended Use(s): See Indications for Use below.

# B Indication(s) for Use:

ETEST is a manual, quantitative technique for the determination of antimicrobial susceptibility of non-fastidious Gram-negative and Gram-positive aerobic bacteria and fastidious bacteria. The system comprises a predefined antibiotic gradient which is used to determine the Minimum Inhibitory Concentration (MIC, in $\mu \mathrm { g } / \mathrm { m L } )$ ) of different antimicrobia l agents against microorganisms tested on agar media after overnight incubation.

Plazomicin has been shown to be active against most isolates of the bacteria listed below according to the FDA label for this antimicrobial agent.

ETEST PLZ can be used to determine the MIC of Plazomicin against the following microorganisms:

Active both in vitro and in clinical infections:   
Escherichia coli   
Klebsiella pneumoniae   
Proteus mirabilis   
Enterobacter cloacae

In vitro data are available for the following microorganisms, but clinical significance is unknown:

Citrobacter freundii   
Citrobacter koseri   
Klebsiella (Enterobacter) aerogenes   
Klebsiella oxytoca   
Morganella morganii   
Proteus vulgaris   
Providentia stuartii   
Serratia marcescens

# C Special Conditions for Use Statement(s):

Rx - For Prescription Use Only

The ability of ETEST Plazomicin to detect the following resistant isolates is unknown because a sufficient number of resistant isolates were not available at the time of comparative testing: Citrobacter koseri, Serratia marcescens.

# D Special Instrument Requirements:

Manual reading only

# A Device Description:

The ETEST gradient technology is based on a combination of the concepts of dilution and diffusion principles for susceptibility testing.

The ETEST consists of a thin, inert, nonporous plastic strip that is used to determine the antimicrobial susceptibility of bacteria. One side of the strip has the minimum inhibitory concentration (MIC) reading scale expressed in $\mu \mathrm { g / m L }$ . The other side of the strip contains a predefined continuous exponential gradient of antibiotic concentrations.

When the strip is applied to an inoculated agar surface, the preformed antibiotic gradient immediately transfers into the agar matrix, then forming a stable, continuous and exponential gradient of antibiotic concentrations directly underneath the strip. Bacterial growth becomes visible during incubation, and a symmetrical inhibition ellipse centered along the strip appears. The MIC value is read from the scale in terms of $\mu \mathrm { g / m L }$ at complete inhibition of bacterial growth, where the pointed end of the ellipse intersects the strip.

ETEST Plazomicin contains a range of plazomicin from 0.006 to $2 5 6 ~ \mu \mathrm { g / m L }$ .

# B Principle of Operation:

When the ETEST strip is applied to an inoculated agar surface, the preformed antibiotic gradient immediately transfers into the agar matrix, then forming a stable, continuous and exponential gradient of antibiotic concentrations directly underneath the strip. Bacteria growth becomes visible during incubation, and a symmetrical inhibition ellipse centered along the strip appears. After incubation, the MIC value is read from the scale in terms of $\mu \mathrm { g / m L }$ at complete inhibition of bacterial growth, where the pointed end of the ellipse intersects the strip. Since ETEST generates MIC values which fall between two-fold dilutions for interpretation, the MIC value read must be recorded to the next two-fold dilution.

# V Substantial Equivalence Information:

A Predicate Device Name(s): ETEST Telavancin (TLA) (0.002-32 ug/mL) B Predicate 510(k) Number(s): K180936 C Comparison with Predicate(s):

<table><tr><td rowspan=1 colspan=1>Device &amp; PredicateDevice(s):</td><td rowspan=1 colspan=1>Device:K200512</td><td rowspan=1 colspan=1>Predicate:K180936</td></tr><tr><td rowspan=1 colspan=1>Device Trade Name</td><td rowspan=1 colspan=1>ETEST Plazomicin</td><td rowspan=1 colspan=1>ETEST Telavancin</td></tr><tr><td rowspan=1 colspan=3>Ge neral De vice Characte ristic Similarities</td></tr><tr><td rowspan=1 colspan=1>IntendedUse/Indications ForUse</td><td rowspan=1 colspan=1>ETEST is a manual, quantitativetechnique for determination ofantimicrobial susceptibility of non-fastidious Gram-negative and Gram-positive aerobic bacteria andfastidious bacteria. The systemcomprises a predefined antibioticgradient which is used to determinethe Minimum InhibitoryConcentration (MIC, in µg/mL) ofdifferent antimicrobial agentsagainst microorganisms tested onagar media after overnightincubation.</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=1>Test Design</td><td rowspan=1 colspan=1>Predefined exponential gradient ofthe dried and stabilized antibioticcovers a continuous concentrationrange across 15 two-fold dilutions ofa conventional MIC method</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=1>Inoculum</td><td rowspan=1 colspan=1>Isolated colonies from culture</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=1>Incubation</td><td rowspan=1 colspan=1>35°± 2° C for 16 - 20 hours</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=1>Result</td><td rowspan=1 colspan=1>MIC in µg/mL</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=3>Ge neral De vice Characte ristic Diffe re nces</td></tr><tr><td rowspan=1 colspan=1>Claimed Organisms</td><td rowspan=1 colspan=1>Escherichia coliKlebsiella pneumoniaeProteus mirabilisEnterobacter cloacaeCitrobacter freundiiCitrobacter koseriKlebsiella (Enterobacter) aerogenesKlebsiella oxytocaMorganella morganiiProteus vulgarisProvidentia stuartiiSerratia marcescens</td><td rowspan=1 colspan=1>Staphylococcus aureus (includingmethic illin-resistant isolates)Enterococcus faecalis (vancomycin-susceptible isolates only)</td></tr><tr><td rowspan=1 colspan=1>Antibiotic</td><td rowspan=1 colspan=1>Plazomic in</td><td rowspan=1 colspan=1>Telavanc in</td></tr><tr><td rowspan=1 colspan=1>AntimicrobialConcentrationRange</td><td rowspan=1 colspan=1>0.016 - 256 μg/mL</td><td rowspan=1 colspan=1>0.002 - 32 μg/mL</td></tr></table>

# VI Standards/Guidance Documents Referenced:

Guidance for Industry and FDA - Class II Special Controls Guidance Document: Antimicrobial Susceptibility Test (AST) Systems – August 28, 2009.

CLSI M07, 11th ed., “Methods for Dilution Antimicrobial Susceptibility Tests for Bacteria That Grow Aerobically; Approved Standard, 2018”. • CLSI M100, $2 9 ^ { \mathrm { t h } }$ ed., “Performance Standards for Antimicrobial Susceptibility Testing; Twenty-Ninth Informational Supplement, January 2019”.

VII Performance Characteristics (if/when applicable):

# A Analytical Performance:

# 1. Precision/Reproducibility:

Reproducibility testing was conducted at three sites over three days using a ten-organism panel that included the following isolates: $1 \ C .$ freundii, $1 E .$ cloacae, 1 E. coli, $1 K .$ aerogenes, $3 K .$ . pneumoniae, $1 P .$ . mirabilis, $1 ~ P .$ . stuartii and $1 S .$ . marcescens. The mode of MIC values was determined for each isolate and the reproducibility was calculated based on the number of MIC values that fell within $\pm 1$ doubling dilution of the mode.

The overall reproducibility results were acceptable at $100 \%$

2. Linearity: Not applicable

3. Analytical Specificity/Interference: Not applicable

4. Assay Reportable Range: Not applicable

5. Traceability, Stability, Expected Values (Controls, Calibrators, or Methods):

Inoculum Density Check. Inoculum density checks were performed for all quality control organism suspensions and for $10 \%$ of the suspensions prepared for susceptibility testing of the contemporary clinical isolates. Inoculum density checks were also performed on reproducibility organism suspensions.

The overall mean inoculum densities in colony forming units per milliliter $\mathrm { ( C F U / m L ) }$ ) for isolates tested with the reference method ranged from $2 . 9 6 \times 1 0 ^ { 5 }$ to $5 . 4 3 \mathrm { ~ x ~ } 1 0 ^ { 5 }$ CFU/mL. The overall mean inoculum densities for isolates tested with the ETEST ranged from $5 . 2 9 \times 1 0 ^ { 7 }$ to $2 . 3 1 \times 1 0 ^ { 8 }$ CFU/mL.

The inoculum densities were acceptable.

Purity Check. Verification of isolate purity was conducted on all clinical, challenge and quality control organism suspensions for each ETEST inoculum and from each growth control well of the broth microdilution (BMD) reference panel.

Growth or Device Failure. Growth was observed for all organisms and no device failures occurred in the ETEST Plazomicin clinical trial.

Quality Control Testing. CLSI recommended QC strain Escherichia coli ATCC 25922 was tested each day of comparative testing. The strain was tested at least 20 times per site at four sites. The results are summarized in Table 1.

Table 1. ETEST Plazomicin QC Results   

<table><tr><td rowspan=1 colspan=1>QC Organis m</td><td rowspan=1 colspan=1>Plazomicin Expe ctedRange</td><td rowspan=1 colspan=1>PlazomicinMIC(μg/mL)</td><td rowspan=1 colspan=1>Refe rence(BMD)Results</td><td rowspan=1 colspan=1>ETESTResults</td></tr><tr><td rowspan=6 colspan=1>E. coliATCC 25922</td><td rowspan=6 colspan=1>0.25 - 2 μg/mL</td><td rowspan=1 colspan=1>&lt;0.25</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>0.25</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>43</td><td rowspan=1 colspan=1>30</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>29</td><td rowspan=1 colspan=1>50</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>&gt;2</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr></table>

1 The expected MIC range of E. coli ATCC 25922 with plazomicin $( 0 . 2 5 - 2 \mu \mathrm { g / m L } )$ is within the MIC range of the ETEST Plazomicin $( 0 . 0 1 6 - 2 5 6 \mu \mathrm { g / m L } )$ .

The Quality Control results were within the recommended range $> 9 5 \%$ of the time which is acceptable.

6. Detection Limit: Not applicable

7. Assay Cut-Off: Not applicable

# B Comparison Studies:

1. Method Comparison with Predicate Device:

Results obtained with ETEST Plazomicin were compared to results obtained with the CLSI broth microdilution (BMD) reference panel. The reference panel, prepared and interpreted according to recommendations outlined in the CLSI M07 $1 1 ^ { \mathrm { t h } }$ ed., contained two-fold serial dilutions of plazomicin with a concentration range of $0 . 0 1 6 - 2 5 6 ~ \mu \mathrm { g / m L }$ . At the end of incubation, the MIC value obtained from the ETEST (where the complete inhibition of growth intersects the strip) was compared to MIC results obtained with the reference method.

The testing conditions for ETEST consisted of the following:

Inoculum: Direct colony suspension to achieve a suspension equivalent to a 0.5 McFarland standard suspension • Medium: Cation-adjusted Mueller Hinton broth or agar • Incubation: $3 5 ^ { \circ } \mathrm { ~ C ~ } \pm 2$ for 16-20 hours

Clinical testing for ETEST Plazomicin was evaluated at three external sites (two located within the United States and one located outside the United States). Each clinical isolate was tested one time by ETEST and BMD using the same initial standardized inoculum prepared in $0 . 8 5 \%$ saline. A total of 518 clinical isolates were tested which included $5 3 ~ C .$ freundii, 31 $C .$ . koseri, $3 2 K .$ . aerogenes, $5 0 E$ . cloacae, $6 5 E .$ . coli, $3 2 K .$ . oxytoca, $7 2 K .$ . pneumoniae, 28 $M .$ . morganii, $5 9 P .$ . mirabilis, $3 3 P .$ . stuartii, $2 9 ~ P _ { \mathrm { { \ell } } }$ . vulgaris, and $3 4 ~ S .$ . marcescens isolates. Of the tested clinical isolates, $5 5 . 2 \%$ (286/518) were considered contemporary (i.e., tested within six months of the organism’s original isolation from clinical specimen) and $4 4 . 8 \%$ (232/518) were considered stock (i.e., no time limit from isolation prior to testing).

Challenge testing was performed at one internal site using ETEST and BMD. A total of 80 challenge isolates were tested which included 6 C. freundii, 3 C. koseri, $6 K .$ aerogenes, 10 E. cloacae, $1 3 E .$ . coli, $7 K .$ . oxytoca, $1 7 K .$ . pneumoniae, 3 M. morganii, 4 P. mirabilis, $2 P .$ . stuartii, $5 P .$ . vulgaris, and $4 S .$ . marcescens isolates.

In total, the comparative study included 598 clinical and challenge isolates.

# Overall Performance

ETEST Plazomicin performance observed for clinical and challenge isolates is summarized in Table 2).

Table 2: Overall Performance of Clinical and Challenge Isolates (Combined)   

<table><tr><td rowspan=1 colspan=1>Plazomicin</td><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>EAN</td><td rowspan=1 colspan=1>EA%</td><td rowspan=1 colspan=1>Eval.Total</td><td rowspan=1 colspan=1>Eval.EAN</td><td rowspan=1 colspan=1>Eval.EA%</td><td rowspan=1 colspan=1>CAN</td><td rowspan=1 colspan=1>CA%</td><td rowspan=1 colspan=1>#R</td><td rowspan=1 colspan=1>#S</td><td rowspan=1 colspan=1>min</td><td rowspan=1 colspan=1>maj</td><td rowspan=1 colspan=1>vmj</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=13>Enterobacteriaceae</td></tr><tr><td rowspan=1 colspan=1>Clinical</td><td rowspan=1 colspan=1>518</td><td rowspan=1 colspan=1>512</td><td rowspan=1 colspan=1>98.8</td><td rowspan=1 colspan=1>498</td><td rowspan=1 colspan=1>492</td><td rowspan=1 colspan=1>98.8</td><td rowspan=1 colspan=1>486</td><td rowspan=1 colspan=1>93.2</td><td rowspan=1 colspan=1>35</td><td rowspan=1 colspan=1>423</td><td rowspan=1 colspan=1>34</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>Challenge</td><td rowspan=1 colspan=1>80</td><td rowspan=1 colspan=1>80</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>70</td><td rowspan=1 colspan=1>70</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>72</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>18</td><td rowspan=1 colspan=1>55</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Combined</td><td rowspan=1 colspan=1>598</td><td rowspan=1 colspan=1>592</td><td rowspan=1 colspan=1>99.0</td><td rowspan=1 colspan=1>568</td><td rowspan=1 colspan=1>562</td><td rowspan=1 colspan=1>98.9</td><td rowspan=1 colspan=1>555</td><td rowspan=1 colspan=1>92.8</td><td rowspan=1 colspan=1>53</td><td rowspan=1 colspan=1>478</td><td rowspan=1 colspan=1>42</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td></tr></table>

EA – Essential Agreement min – minor errors CA – Category Agreement maj – major errors EVAL – Evaluable isolates vmj – very major errors R – Resistant isolates

Essential Agreement (EA) is when the ETEST result agrees exactly or within one doubling dilution of the reference broth microdilution result. Category Agreement (CA) is when the ETEST result interpretation agrees exactly with the reference broth microdilution result interpretation.

ETEST Plazomicin performance for all Enterobacteriaceae isolates (clinical and challenge) is acceptable with $9 9 . 0 \%$ EA and $9 2 . 8 \%$ CA. There were zero major errors and one very major error $1 / 5 3 = 1 . 9 \%$ ) which is acceptable.

When the performance was evaluated individually by species, it was noted that the CA for $M .$ morganii, $P .$ . mirabilis, $P .$ . vulgaris, $P .$ . stuartii and S. marcescens was less than $90 \%$ for each organism (CA ranged from $6 7 . 5 \% - 8 9 . 5 \% )$ . However, performance is considered acceptable since the EA for each organism was above $90 \%$ and all categorical errors were minor with MIC values within one doubling dilution of the reference MIC value (i.e. in essential agreement), except for one isolate of S. marcescens. The following statement is included as a footnote to the performance table in the device labeling:

The Category Agreement was $< 9 0 \%$ for the following organisms: Morganella morganii $( 6 7 . 7 \% )$ , Proteus mirabilis $( 8 5 . 7 \% )$ , Providencia stuartii $( 7 4 . 3 \% )$ , Proteus vulgaris $( 8 5 . 3 \% )$ and Serratia marcescens $( 8 9 . 5 \% )$ . The performance is acceptable since the Essential Agreement was $590 \%$ and all categorical errors were minor and within essential agreement, except for one isolate of Serratia marcescens.

There were no resistant isolates of $C .$ . koseri tested in the comparative study. One resistant isolate of S. marcescens was tested; however, ETEST failed to provide a resistant result which generated a very major error $1 / 1 = 1 0 0 \%$ VMJ). This error was considered a random error.

To address the lack or insufficient testing of resistant isolates, the following statement is included as a limitation in the device labeling:

The ability of ETEST Plazomicin to detect the following resistant isolates is unknown because a sufficient number of resistant isolates were not available at the time of comparative testing: Citrobacter koseri, Serratia marcescens.

As required under 511A(b)(2)(C)(ii)(I) of the Federal Food, Drug and Cosmetic Act, the following statement is included in the Warnings and Precautions section of the device labeling to address the testing and reporting of non-indicated species:

Per the FDA-Recognized Susceptibility Test Interpretive Criteria website, the safety and efficacy of antimicrobial drugs, for which antimicrobial susceptibility is tested by this AST device, may or may not have been established in adequate and well-controlled clinical trials for treating clinical infections due to microorganisms outside of those found in the indications and usage in the drug label. The clinical significance of susceptibility information in those instances is unknown. The approved labeling for specific antimicrobial drugs provides the uses for which the antimicrobial drug is approved.

# Inoculator and ETEST Strip Applicator Options

Culture media plates for ETEST can be inoculated and streaked by swabs manually or with the RETRO C80 inoculator. ETEST strips can be applied onto inoculated media using forceps or the NEMA C88 vacuum pen.

The ETEST Plazomicin studies used both inoculation and strip application methods. The following statement is included as a footnote to the performance table in the device labeling:

The optional inoculator and ETEST strip applicator can be used for plate inoculation and applying ETEST strips onto agar media. In the ETEST Plazomicin clinical studies, swabs and the Inoculator RETRO C80 were used for plate inoculation/streaking and forceps and the Vacuum Pen NEMA C88 were used for ETEST strip application.

# Trending

A trending analysis was conducted using the combined data (clinical and challenge) for each organism species and Enterobacteriaceae as a group. This trending calculation analyzes

device MIC values that are determined to be one or more doubling dilutions lower or higher than the reference method. MIC values that are off-scale for both the reference and device are not considered in the trending analysis.

Trending results were stratified by species to determine if species-related trends were observed (Table 3). Species for which the difference between the percentage of isolates with higher versus lower MIC values was $23 0 \%$ and for which the confidence interval was determined to be statistically significant were considered to have evidence of trending and is addressed in labeling.

Table 3. Trending by Species (clinical and challenge isolates combined)   

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>TotalEvaluable forTrending</td><td rowspan=1 colspan=1>≥1 dil.Lower #(%)</td><td rowspan=1 colspan=1>Exact #(%)</td><td rowspan=1 colspan=1>≥1 dil.Higher #(%)</td><td rowspan=1 colspan=1>PercentDifference(95% CI)</td><td rowspan=1 colspan=1>TrendingNoted</td></tr><tr><td rowspan=1 colspan=1>Citrobacter freundii</td><td rowspan=1 colspan=1>58</td><td rowspan=1 colspan=1>2(3.45)</td><td rowspan=1 colspan=1>41(70.69)</td><td rowspan=1 colspan=1>15(25.86)</td><td rowspan=1 colspan=1>22.41%</td><td rowspan=1 colspan=1>No</td></tr><tr><td rowspan=1 colspan=1>Citrobacter koseri</td><td rowspan=1 colspan=1>34</td><td rowspan=1 colspan=1>5(14.71)</td><td rowspan=1 colspan=1>17(50.00)</td><td rowspan=1 colspan=1>12(35.29)</td><td rowspan=1 colspan=1>20.59%</td><td rowspan=1 colspan=1>No</td></tr><tr><td rowspan=1 colspan=1>Enterobacter cloacae</td><td rowspan=1 colspan=1>58</td><td rowspan=1 colspan=1>1(1.72)</td><td rowspan=1 colspan=1>40(68.97)</td><td rowspan=1 colspan=1>17(29.31)</td><td rowspan=1 colspan=1>27.59%</td><td rowspan=1 colspan=1>No</td></tr><tr><td rowspan=1 colspan=1>Escherichia coli</td><td rowspan=1 colspan=1>72</td><td rowspan=1 colspan=1>15(20.83)</td><td rowspan=1 colspan=1>45(62.50)</td><td rowspan=1 colspan=1>12(16.67)</td><td rowspan=1 colspan=1>-4.17%</td><td rowspan=1 colspan=1>No</td></tr><tr><td rowspan=1 colspan=1>Klebsiella (Enterobacter)aerogenes</td><td rowspan=1 colspan=1>37</td><td rowspan=1 colspan=1>3(8.11)</td><td rowspan=1 colspan=1>18(48.65)</td><td rowspan=1 colspan=1>16(43.24)</td><td rowspan=1 colspan=1>35.14%(15.48 to 51.84)</td><td rowspan=1 colspan=1>Yes</td></tr><tr><td rowspan=1 colspan=1>Klebsiella oxytoca</td><td rowspan=1 colspan=1>38</td><td rowspan=1 colspan=1>13(34.21)</td><td rowspan=1 colspan=1>19(50.00)</td><td rowspan=1 colspan=1>6(15.79)</td><td rowspan=1 colspan=1>-18.42%</td><td rowspan=1 colspan=1>No</td></tr><tr><td rowspan=1 colspan=1>Klebsiella pneumoniae</td><td rowspan=1 colspan=1>77</td><td rowspan=1 colspan=1>4(5.19)</td><td rowspan=1 colspan=1>35(45.45)</td><td rowspan=1 colspan=1>38(49.35)</td><td rowspan=1 colspan=1>44.16%(31.00 to 55.53)</td><td rowspan=1 colspan=1>Yes</td></tr><tr><td rowspan=1 colspan=1>Morganella morganii</td><td rowspan=1 colspan=1>31</td><td rowspan=1 colspan=1>13(41.94)</td><td rowspan=1 colspan=1>16(51.61)</td><td rowspan=1 colspan=1>2(6.45)</td><td rowspan=1 colspan=1>-35.48%(-53.40 to -14.40)</td><td rowspan=1 colspan=1>Yes</td></tr><tr><td rowspan=1 colspan=1>Proteus mirabilis</td><td rowspan=1 colspan=1>61</td><td rowspan=1 colspan=1>1(1.64)</td><td rowspan=1 colspan=1>49(80.33)</td><td rowspan=1 colspan=1>11(18.03)</td><td rowspan=1 colspan=1>16.39%</td><td rowspan=1 colspan=1>No</td></tr><tr><td rowspan=1 colspan=1>Proteus vulgaris</td><td rowspan=1 colspan=1>34</td><td rowspan=1 colspan=1>4(11.76)</td><td rowspan=1 colspan=1>24(70.59)</td><td rowspan=1 colspan=1>6(17.65)</td><td rowspan=1 colspan=1>5.88%</td><td rowspan=1 colspan=1>No</td></tr><tr><td rowspan=1 colspan=1>Providencia stuartii</td><td rowspan=1 colspan=1>31</td><td rowspan=1 colspan=1>7(22.58)</td><td rowspan=1 colspan=1>16(51.61)</td><td rowspan=1 colspan=1>8(25.81)</td><td rowspan=1 colspan=1>3.23%</td><td rowspan=1 colspan=1>No</td></tr><tr><td rowspan=1 colspan=1>Serratia marcescens</td><td rowspan=1 colspan=1>38</td><td rowspan=1 colspan=1>5(13.16)</td><td rowspan=1 colspan=1>22(57.89</td><td rowspan=1 colspan=1>11(28.95)</td><td rowspan=1 colspan=1>15.79%</td><td rowspan=1 colspan=1>No</td></tr><tr><td rowspan=1 colspan=1>All Enterobacteriaceae</td><td rowspan=1 colspan=1>569</td><td rowspan=1 colspan=1>73(12.83)</td><td rowspan=1 colspan=1>342(60.11)</td><td rowspan=1 colspan=1>154(27.07)</td><td rowspan=1 colspan=1>14.24%</td><td rowspan=1 colspan=1>No</td></tr></table>

A trend toward higher MIC values was observed with $K .$ . aerogenes and $K .$ . pneumoniae. The following statement is included as a footnote to the performance table in the device labeling:

ETEST Plazomicin MIC values tended to be in exact agreement or at least one doubling dilution higher when testing Klebsiella aerogenes and Klebsiella pneumoniae compared to the CLSI reference broth microdilution method. However, this trending did not impact the Essential or Category Agreement (Klebsiella aerogenes EA:100%, $C A : I O O \%$ ; Klebsiella pneumoniae EA:100%, $C A { : } 9 8 . 9 \% )$ .

A trend toward lower MIC values was observed with M. morganii. The following statement is included as a footnote to the performance table in the device labeling:

ETEST Plazomicin MIC values tended to be in exact agreement or at least one doubling dilution lower when testing Morganella morganii compared to the CLSI reference broth microdilution method.

# Resistance Markers

Molecular characterization for resistance to plazomicin, as indicated on the plazomicin drug label, was identified for challenge isolates by whole genome sequencing analysis. They consisted of genetic markers that confer resistance to aminoglycosides, including 16S rRNA methyltransferases and aminoglycoside modifying enzymes (AMEs). Other mechanisms of plazomicin resistance, such as altered efflux and loss of outer membrane porins, were not evaluated.

2. Matrix Comparison: Not applicable

# C Clinical Studie s:

1. Clinical Sensitivity: Not applicable

2. Clinical Specificity: Not applicable

3. Other Clinical Supportive Data (When 1. and 2. Are Not Applicable): Not applicable

# D Clinical Cut-Off:

Not applicable

# E Expected Values/Reference Range:

The FDA-identified susceptibility interpretive criteria for plazomic in are listed in Table 4.

Table 4: FDA-Identified Interpretive Criteria for Plazomicin (µg/mL)a   

<table><tr><td rowspan=1 colspan=1>Organis m</td><td rowspan=1 colspan=1>Sus ce ptible</td><td rowspan=1 colspan=1>Inte rme diate</td><td rowspan=1 colspan=1>Resistant</td></tr><tr><td rowspan=1 colspan=1>Enterobacteriaceae</td><td rowspan=1 colspan=1>≤2</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>≥8</td></tr></table>

a According to FDA STICWebsite

# VIII Proposed Labeling:

The labeling supports the finding of substantial equivalence for this device.

# IX Conclusion:

The submitted information in this premarket notification is complete and supports a substantial equivalence decision.

To support the implementation of changes to FDA-recognized susceptibility test interpretive criteria (i.e., breakpoints), this submission included a breakpoint change protocol that was reviewed and accepted by FDA. This protocol addresses future revisions to device labeling in response to breakpoint changes that are recognized on the FDA STIC webpage (https://www.fda.gov/Drugs/DevelopmentApprovalProcess/DevelopmentResources/ucm410971. htm). The protocol outlined the specific procedures and acceptance criteria that bioMérieux intends to use to evaluate the ETEST Plazomicin when revised breakpoints for plazomicin are published on the FDA STIC webpage. The breakpoint change protocol included with the submission indicated that if specific criteria are met, bioMérieux will update the plazomicin device label to include (1) the new breakpoints, (2) an updated performance section after reevaluation of data in this premarket notification with the new breakpoints, and (3) any new limitations as determined by their evaluation.