# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATION DECISION SUMMARY ASSAY ONLY

I Background Information:

A 510(k) Number K201269   
B Applicant Mesa Biotech, Inc.   
C Proprietary and Established Names Accula Strep A Test   
D Regulatory Information

<table><tr><td>Product Code(s)</td><td>Classification</td><td>Regulation Section</td><td>Panel</td></tr><tr><td>PGX</td><td>Class II</td><td>21 CFR 866.2680 - Streptococcus Spp. Nucleic Acid-Based Assay</td><td>MI - Microbiology</td></tr></table>

# II Submission/Device Overview:

# A Purpose for Submission:

This is a new 510(k) submission for the determination of Substantial Equivalence for the Mesa Biotech Accula Strep A Test. Mesa Biotech, Inc. has submitted a combined 510(k) and CLIA waiver package for dual review.

# B Measurand:

Group A Streptococcus nucleic acid

# C Type of Test:

Semi-automated PCR amplification followed by hybridization and colorimetric visualization of amplified products on a test strip for qualitative detection

# III Intended Use/Indications for Use:

A Intended Use(s): See Indications for Use below.

# B Indication(s) for Use:

The Accula Strep A Test performed on the Accula Dock is a molecular in vitro diagnostic test utilizing polymerase chain reaction (PCR) and lateral flow technologies for the qualitative, visual detection of Streptococcus pyogenes (Group A $\beta$ -hemolytic Streptococcus, Strep A) bacterial nucleic acid. It is intended to aid in the rapid diagnosis of Group A Streptococcus bacterial infections from throat swabs of patients with signs and symptoms of pharyngitis.

All negative test results should be confirmed by bacterial culture because negative results do not preclude infection with Group A Streptococcus and should not be used as the sole basis for treatment.

# C Special Conditions for Use Statement(s):

Rx - For Prescription Use Only

Additional follow-up testing by culture is required if the Accula Strep A result is negative and clinical symptoms persist, or in the event of an outbreak of acute rheumatic fever (ARF).

# D Special Instrument Requirements:

To be used only with the Accula Dock Instrument

# IV Device/System Characteristics: A Device Description:

The Accula Strep A Test is a semi-automated, colorimetric polymerase chain reaction (PCR) nucleic acid amplification test to qualitatively detect Streptococcus pyogenes (Group A $\beta \cdot$ - hemolytic Streptococcus, Strep A) bacterial nucleic acid from unprocessed throat swabs that have not undergone prior nucleic acid extraction. The system integrates nucleic acid extraction, a novel Mesa Biotech PCR nucleic acid amplification technology, and hybridization-based visual detection into a completely self-contained and automated system. The Accula Strep A system consists of a small reusable Dock to drive the automated testing process, and a single-use disposable test cassette that contains all the enzymes and reagents.

# B Principle of Operation:

Upon insertion of an Accula Strep A Test cassette, the dock will detect and identify the cassette type. After the user transfers a clinical sample into the cassette and closes the dock lid, the embedded firmware will control fluid flow of the sample into the various chambers of the cassette.

Amplicon detection requires the hybridization of two internal probes to generate a signal on the Accula Strep A detection strip. Dyed polystyrene microspheres are conjugated to oligonucleotide probes to form an amplicon-microsphere complex by hybridization to an internal region of the amplicon. The complex migrates through the pores of the detection strip membrane and across capture zones which contain oligonucleotides complementary to an amplicon region distinct from the detection probe binding site. Hybridization of the amplicon-microsphere complex to a capture zone probe retards the flow of the specific amplicon. This allows for the generation of a visible signal in the form of a colored line at the capture zone.

Interpretation of results:

Results are interpreted visually by the operator after the test has completed. The appearance of any shade of blue at the “Strep” position on the test strip indicates a valid result that is interpreted as positive for Strep A. A positive control $\left( ^ { 6 6 } \mathbf { C } ^ { 7 } \right)$ line at the beginning of the strip tests for amplification effectiveness and is necessary to interpret a test as “negative” for Strep A. A negative control (“NC”) line at the end of the test strip controls for non-specific binding or amplification and must be absent for a valid result.

# V Substantial Equivalence Information:

A Predicate Device Name(s): Liat Strep A Assay   
B Predicate 510(k) Number(s): K141338

C Comparison with Predicate(s):

<table><tr><td colspan="1" rowspan="1">Device &amp; PredicateDevice(s):</td><td colspan="1" rowspan="1">K201269</td><td colspan="1" rowspan="1">K141338</td></tr><tr><td colspan="1" rowspan="1">Device Trade Name</td><td colspan="1" rowspan="1">Accula Strep A Test</td><td colspan="1" rowspan="1">cobas Strep A Nucleic acid test(formerly, Liat Strep A Assay)</td></tr><tr><td colspan="3" rowspan="1">General Device Characteristic Similarities</td></tr><tr><td colspan="1" rowspan="1">Intended Use</td><td colspan="1" rowspan="1">The Accula Strep A Test performedon the Accula Dock is a molecularin vitro diagnostic test utilizingpolymerase chain reaction (PCR)and lateral flow technologies forthe qualitative, visual detection ofStreptococcus pyogenes (Group Aβ-hemolytic Streptococcus, StrepA) bacterial nucleic acid. It isintended to aid in the rapiddiagnosis of Group AStreptococcus bacterial infectionsfrom throat swabs of patients withsigns and symptoms of pharyngitis.All negative test results should beconfirmed by bacterial culturebecause negative results do notpreclude infection with Group A</td><td colspan="1" rowspan="1">The cobas Strep A nucleic acid testfor use on the cobas Liat System(cobas Strep A is a qualitative invitro diagnostic test for thedetection of Streptococcuspyogenes (Group A β-hemolyticStreptococcus, Strep A) in throatswab specimens from patients withsigns and symptoms of pharyngitis.The cobas Strep A assay utilizesnucleic acid purification andpolymerase chain reaction (PCR)technology to detect Streptococcuspyogenes by targeting a segment ofthe Streptococcus pyogenesgenome.</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Streptococcus and should not beused as the sole basis for treatment.</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Regulation</td><td colspan="1" rowspan="1">21 CFR 866.2690</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Product Code</td><td colspan="1" rowspan="1">PGX</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Analyte</td><td colspan="1" rowspan="1">Group A Streptococcus</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Sample Type</td><td colspan="1" rowspan="1">Throat swab</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Intended Users andUse Locations</td><td colspan="1" rowspan="1">Clinical lab and CLIA-waived sites</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Internal Control</td><td colspan="1" rowspan="1">Yes</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Positive andNegative ControlSwabs</td><td colspan="1" rowspan="1">Yes</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Strep A Target</td><td colspan="1" rowspan="1">Conserved region of Group AStreptococcus genome</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Reagent Format</td><td colspan="1" rowspan="1">Unitized ready for use</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Assay Automation</td><td colspan="1" rowspan="1">Yes</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Assay Result</td><td colspan="1" rowspan="1">Qualitative</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="2" rowspan="1">General Device Characteristic Differences</td></tr><tr><td colspan="1" rowspan="1">Bacterial Lysis</td><td colspan="1" rowspan="1">Detergent and heat</td><td colspan="1" rowspan="1">Chaotropic and enzymaticdigestion</td></tr><tr><td colspan="1" rowspan="1">Nucleic AcidPurification</td><td colspan="1" rowspan="1">None</td><td colspan="1" rowspan="1">Solid phase magnetic affinitycapture</td></tr><tr><td colspan="1" rowspan="1">Assay Technology</td><td colspan="1" rowspan="1">PCR amplification and visualidentification of amplificationproducts by hybridization to atest strip</td><td colspan="1" rowspan="1">PCR amplification and detection ofspecific amplification productsusing TaqMan (hydrolysis) probe-based PCR fluorescent probes</td></tr><tr><td colspan="1" rowspan="1">DetectionTechnique</td><td colspan="1" rowspan="1">Dyed microparticle conjugates tospecifically detect and identifyamplification reaction products.Visual interpretation of thepresence or the absence of coloredlines on a test strip.</td><td colspan="1" rowspan="1">Fluorescently-labeled Taqmanprobe to specifically identifyamplified cDNA products.Optical detection of fluorescence.</td></tr><tr><td colspan="1" rowspan="1">Equipment Required</td><td colspan="1" rowspan="1">Amplification performed on theAccula Dock.No detection by the instrument.</td><td colspan="1" rowspan="1">Assay amplification and detectionperformed on the Liat Analyzer.</td></tr><tr><td colspan="1" rowspan="1">Result Interpretation</td><td colspan="1" rowspan="1">Manual</td><td colspan="1" rowspan="1">Automated</td></tr><tr><td colspan="1" rowspan="1">Time-to-Result</td><td colspan="1" rowspan="1">~30 minutes</td><td colspan="1" rowspan="1">~15 minutes</td></tr></table>

# VI Standards/Guidance Documents Referenced:

ISO 14971:2007, Medical Devices – Application of Risk Management to Medical Devices, Second Edition IEC 62304:2015, Medical Device Software – Software Life Cycle Processes, Edition 1.1 CLSI Guideline EP25-A, Evaluation of Stability of In Vitro Diagnostic Reagents; Approved Guideline. Wayne, PA: Clinical and Laboratory Standards Institute; 2009.

# VII Performance Characteristics (if/when applicable):

# A Analytical Performance:

1. Precision/Reproducibility:

Reproducibility / Near the Cut-off Study To demonstrate the reproducibility of the Accula Strep A Test, a panel of contrived throat swabs were tested at three CLIA-waived sites by two non-laboratory operators at each site. The nine-member panel consisted of triplicate swab preparations spiked with varying Strep A concentrations, described in Table 1: low positive $\left( \sim 1 \mathrm { X } \mathrm { L o D } \right)$ , moderate positive ${ \sim } 2 \mathrm { X }$ LoD), and negative (no Strep A). Each positive sample was prepared by spiking Streptococcus pyogenes (BAA-946), diluted into a pooled negative throat swab matrix prepared with Strep A buffer (PNTS), onto a swab. The negative sample consisted of the PNTS alone. Each operator tested the coded, randomized panel on 5 non-consecutive days over a period of two weeks (3 sites x 2 operators x 3 replicate swabs x 5 days $= 9 0$ results/panel member).

Table 1. Reproducibility Panel Members   

<table><tr><td rowspan=1 colspan=1>Sample Type</td><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>Multipleof LoD</td><td rowspan=1 colspan=1>CFU/Test1</td><td rowspan=1 colspan=1>CFU/mL²</td><td rowspan=1 colspan=1>Expected Result forS. pyogenes </td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Not applicable</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>Negative</td></tr><tr><td rowspan=1 colspan=1>Low Positive</td><td rowspan=2 colspan=1>ATCCBAA-946</td><td rowspan=1 colspan=1>~1X</td><td rowspan=1 colspan=1>188</td><td rowspan=1 colspan=1>75</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>Moderate Positive</td><td rowspan=1 colspan=1>~2X</td><td rowspan=1 colspan=1>376</td><td rowspan=1 colspan=1>150</td><td rowspan=1 colspan=1>Positive</td></tr></table>

1 Number of organisms (colony forming units) applied to swab prior to elution. 2 Number of organisms in $2 . 5 ~ \mathrm { m L }$ Strep A buffer after elution from swab (assuming $100 \%$ bacteria recovery). 3 Used in determination of percent agreement.

Results are reported as percent agreement: observed result/expected result $\textbf { x } 1 0 0$ . Agreement by site is summarized in Table 2. Reproducibility was evaluated by site, operator and day.

Table 2. Reproducibility of Accula Strep A Test Stratified by Site1   

<table><tr><td rowspan=2 colspan=1>SampleType</td><td rowspan=1 colspan=2>Site 1</td><td rowspan=1 colspan=2>Site 2</td><td rowspan=1 colspan=2>Site 3</td><td rowspan=1 colspan=2>Combined</td></tr><tr><td rowspan=1 colspan=1>Count</td><td rowspan=1 colspan=1>PercentAgreement</td><td rowspan=1 colspan=1>Count</td><td rowspan=1 colspan=1>PercentAgreement</td><td rowspan=1 colspan=1>Count</td><td rowspan=1 colspan=1>PercentAgreement</td><td rowspan=1 colspan=1>Count</td><td rowspan=1 colspan=1>PercentAgreement(95% CI)</td></tr><tr><td rowspan=1 colspan=1>LowPositive</td><td rowspan=1 colspan=1>29/30</td><td rowspan=1 colspan=1>96.7%</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>89/90</td><td rowspan=1 colspan=1>98.9%(94.0-99.8)</td></tr><tr><td rowspan=1 colspan=1>ModeratePositive</td><td rowspan=1 colspan=1>29/30</td><td rowspan=1 colspan=1>96.7%</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>28/29</td><td rowspan=1 colspan=1>96.6%</td><td rowspan=1 colspan=1>87/89</td><td rowspan=1 colspan=1>97.8%(92.2-99.4)</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>28/30</td><td rowspan=1 colspan=1>93.3%</td><td rowspan=1 colspan=1>88/90</td><td rowspan=1 colspan=1>97.8%(92.3-99.4)</td></tr></table>

1Twenty $( 6 . 9 \% )$ of the 270 samples tested did not yield a valid result after initial testing (14 invalid results and 6

system errors). After retesting, one invalid result remained unresolved $1 / 2 9 0 = 0 . 3 \%$ ).

Agreement between observed and expected results was acceptable $( > 9 5 \% )$ . No significant differences were observed within run (replicates tested by one operator), between runs (across five days), between operators or between sites.

# Within-Laboratory Repeatability

To demonstrate the reproducibility of the Accula Strep A Test within the laboratory, a panel of contrived throat swabs (Table 1) were tested at an internal site by two operators using three lots of reagents over ten days. In addition, each operator used a set of dedicated Accula Docks (18 each). On each testing day, each operator performed three runs (one with each reagent lot) in two technical replicates (i.e., one swab eluant used for two tests) for each panel member (2 operators x 3 reagent lots $_ { \textrm { X 2 } }$ replicate tests $\mathrm { \Delta x \ 1 0 \ d a y s = 1 2 0 }$ results/panel member).

Results are reported as percent agreement: observed result/expected result $\textbf { x } 1 0 0$ . Agreement is summarized in Table 3. Reproducibility evaluated by operator, day, reagent lot and Accula Dock was acceptable $( > 9 5 \% )$ .

Table 3. Repeatability of Accula Strep A Test1   

<table><tr><td rowspan=1 colspan=1>Sample Type</td><td rowspan=1 colspan=1>No. Observed</td><td rowspan=1 colspan=1>No. Expected</td><td rowspan=1 colspan=1>% Agreement (95% CI)</td></tr><tr><td rowspan=1 colspan=1>Low Positive</td><td rowspan=1 colspan=1>120</td><td rowspan=1 colspan=1>120</td><td rowspan=1 colspan=1>100% (96.9 - 100%)</td></tr><tr><td rowspan=1 colspan=1>Moderate Positive</td><td rowspan=1 colspan=1>120</td><td rowspan=1 colspan=1>120</td><td rowspan=1 colspan=1>100% (96.9 - 100%)</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>120</td><td rowspan=1 colspan=1>120</td><td rowspan=1 colspan=1>100% (96.9 - 100%)</td></tr></table>

1 Fifteen $( 4 . 0 \% )$ of the 360 samples tested did not yield a valid result after initial testing (15 invalid results). After retesting, all invalid results were resolved.

2. Linearity/Assay Reportable Range: Not applicable

3. Traceability, Stability, Expected Values (Controls, Calibrators, or Methods):

Quality Control:

The Accula Strep A Test incorporates four controls:

Internal Positive Control: The positive control line, “C”, is for detection of positive control DNA (plant gene segment rbcL) that is present in the cassette. It monitors nucleic acid amplification and detection, as well as the automated testing process and performance of the Accula Dock. Detection of the internal positive control is not required for considering a positive test result as valid but is required for considering a negative test result as valid.

Internal Negative Control: The negative control line, “NC”, is for detection of nonspecific hybridization between labeled latex particles and capture probes immobilized onto the membrane. It monitors improper hybridization artifacts, which can result from faulty fluid movement, mixing, or rehydration of dyed latex particles. The absence of signal at the negative control line is required for a valid test.

External Positive Control: The external positive control swab is provided with the Accula Strep A Test. It contains inactivated S. pyogenes to monitor nucleic acid extraction, amplification and detection.

External Negative Control: The external negative control swab is provided with the Accula Strep A Test. It contains a dried buffer solution without S. pyogenes.

The external controls are recommended to verify reagent integrity when a new lot or new shipment of Accula Strep A kits is received, or once for each untrained operator.

# Evaluation of Control Performance:

During the Clinical Study to validate performance of the Accula Strep A Test, 2-3 contrived control swabs were tested by the intended user each day participants were enrolled in the study. Users were blinded as to the identity of the contrived control samples being tested. The performance when testing the positive and negative control swabs was acceptable $( > 9 5 \% )$ , as shown in Table 4.

Table 4. Evaluation of Contrived Control Swabs in Clinical Study1   

<table><tr><td rowspan=1 colspan=1>Contrived Control Swab</td><td rowspan=1 colspan=1>% Agreement</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>447/454 (98.5%)</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>453/458 (98.9%)</td></tr></table>

1 Sixty-nine $( 7 . 5 \% )$ of the 917 control swabs tested did not yield an invalid result after initial testing (34 invalid results and 35 system errors). After re-testing, seven results (4 invalid results and 3 system errors) remained unresolved $( 7 / 9 8 6 = 0 . 7 \%$ .

# Sample Stability:

The IFU for the Accula Strep A Test states that a direct throat swab can be stored in its original packaging at room temperature ( $1 5 \mathrm { { ^ \circ C } }$ to $3 0 ^ { \circ } \mathrm { C }$ ) for up to 2 hours or refrigerated $( 2 ^ { \circ } \mathrm { C }$ $\boldsymbol { \mathbf { \rho } } - 8 ^ { \circ } \boldsymbol { \mathbf { C } } )$ for 24 hours prior to testing. In addition, after elution with Strep A buffer, samples may be stored at $1 5 \mathrm { { ^ \circ C } }$ to $3 0 ^ { \circ } \mathrm { C }$ (room temperature) for up to one hour prior to testing. To evaluate sample stability, ${ \sim } 2 \mathrm { X }$ LoD Strep A contrived specimens (or PNTS alone) were spiked onto swabs and held at $1 5 { - } 3 0 ^ { \circ } \mathrm { C }$ and $2 { - } 8 ^ { \circ } \mathrm { C }$ . Additionally, to evaluate stability of the eluate, ${ \sim } 2 \mathrm { X }$ LoD Strep A contrived specimens were spiked onto swabs and eluted with PNTS and held at $1 5 { - } 3 0 ^ { \circ } \mathrm { C }$ , $2 { - } 8 ^ { \circ } \mathrm { C }$ , $- 2 0 ^ { \circ } \mathrm { C }$ , $- 9 0 ^ { \circ } \mathrm { C }$ to $- 6 0 ^ { \circ } \mathrm { C }$ , or for three freeze-thaw cycles at ${ } _ { - 2 0 } \mathrm { { } ^ { \circ } C }$ . Samples were tested at multiple time points depending on the storage temperature.

Sample storage conditions and test results are summarized in Table 5.

Table 5. Sample Stability of Swab and Eluate Samples   

<table><tr><td rowspan=2 colspan=1>TestSample</td><td rowspan=2 colspan=1>SampleType</td><td rowspan=2 colspan=1>StorageCondition</td><td rowspan=2 colspan=1>Duration</td><td rowspan=1 colspan=2>Positive Strep A</td><td rowspan=1 colspan=2>Negative Strep A</td></tr><tr><td rowspan=1 colspan=1>Observed/ Expected</td><td rowspan=1 colspan=1>Pass/Fail</td><td rowspan=1 colspan=1>Observed/ Expected</td><td rowspan=1 colspan=1>Pass/Fail</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=7 colspan=1>Eluate</td><td rowspan=1 colspan=1>None</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>Pass</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>Pass</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>15°</td><td rowspan=1 colspan=1>1 Hour</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>Pass</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>Pass</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>15°C</td><td rowspan=1 colspan=1>4 Hours</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>Pass</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>Pass</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>15°C</td><td rowspan=1 colspan=1>24 Hours</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>Pass</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>Pass</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>1 Hour</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>Pass</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>Pass</td></tr><tr><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>30°</td><td rowspan=1 colspan=1>4 Hours</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>Pass</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>Pass</td></tr><tr><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>30°</td><td rowspan=1 colspan=1>24 Hours</td><td rowspan=1 colspan=1>5/6</td><td rowspan=1 colspan=1>Fail</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>Pass</td></tr></table>

<table><tr><td rowspan=2 colspan=1>TestSample</td><td rowspan=2 colspan=3>SampleType</td><td rowspan=2 colspan=1>StorageCondition</td><td rowspan=2 colspan=1>Duration</td><td rowspan=1 colspan=2>Positive Strep A</td><td rowspan=1 colspan=2>Negative Strep A</td></tr><tr><td rowspan=1 colspan=1>Observed/ Expected</td><td rowspan=1 colspan=1>Pass/Fail</td><td rowspan=1 colspan=1>Observed/ Expected</td><td rowspan=1 colspan=1>Pass/Fail</td></tr><tr><td rowspan=1 colspan=1>8</td><td rowspan=7 colspan=3>b</td><td rowspan=1 colspan=1>2°</td><td rowspan=1 colspan=1>24 Hours</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>Pass</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>Pass</td></tr><tr><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>2°</td><td rowspan=1 colspan=1>72 Hours</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>Pass</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>Pass</td></tr><tr><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>8°C</td><td rowspan=1 colspan=1>24 Hours</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>Pass</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>Pass</td></tr><tr><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>8°C</td><td rowspan=1 colspan=1>72 Hours</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>Pass</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>Pass</td></tr><tr><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>-20°C</td><td rowspan=1 colspan=1>72 Hours</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>Pass</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>Pass</td></tr><tr><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>-20°</td><td rowspan=1 colspan=1>1 Week</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>Pass</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>Pass</td></tr><tr><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=2></td><td rowspan=1 colspan=1>-80°</td><td rowspan=1 colspan=1>72 Hours</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>Pass</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>Pass</td></tr><tr><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=2></td><td rowspan=2 colspan=1></td><td rowspan=1 colspan=1>-80°</td><td rowspan=1 colspan=1>2 Weeks</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>Pass</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>Pass</td></tr><tr><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=2></td><td rowspan=1 colspan=1>-20°</td><td rowspan=1 colspan=1>1 Freeze/thaw</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>Pass</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>Pass</td></tr><tr><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=2></td><td rowspan=2 colspan=1></td><td rowspan=1 colspan=1>-20°</td><td rowspan=1 colspan=1>2 Freeze/thaw</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>Pass</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>Pass</td></tr><tr><td rowspan=1 colspan=1>18</td><td></td><td></td><td rowspan=1 colspan=1>-20°C</td><td rowspan=1 colspan=1>3 Freeze/thaw</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>Pass</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>Pass</td></tr><tr><td rowspan=1 colspan=1>19</td><td rowspan=8 colspan=3>Swab</td><td rowspan=1 colspan=1>2°C</td><td rowspan=1 colspan=1>24 Hours</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>Pass</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>Pass</td></tr><tr><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>2°</td><td rowspan=1 colspan=1>72 Hours</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>Pass</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>Pass</td></tr><tr><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>8°C</td><td rowspan=1 colspan=1>24 Hours</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>Pass</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>Pass</td></tr><tr><td rowspan=1 colspan=1>22</td><td rowspan=1 colspan=1>8°C</td><td rowspan=1 colspan=1>72 Hours</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>Pass</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>Pass</td></tr><tr><td rowspan=1 colspan=1>23</td><td rowspan=1 colspan=1>15°C</td><td rowspan=1 colspan=1>4 Hours</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>Pass</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>Pass</td></tr><tr><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>15°C</td><td rowspan=1 colspan=1>24 Hours</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>Pass</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>Pass</td></tr><tr><td rowspan=1 colspan=1>25</td><td rowspan=1 colspan=1>30°</td><td rowspan=1 colspan=1>4 Hours</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>Pass</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>Pass</td></tr><tr><td rowspan=1 colspan=1>26</td><td rowspan=1 colspan=1>30°</td><td rowspan=1 colspan=1>24 Hours</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>Pass</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>Pass</td></tr></table>

The eluate samples yielded the expected test results after being held for: four hours at 15- $3 0 ^ { \circ } \mathrm { C }$ (room temperature), 72 hours at $2 { - } 8 ^ { \circ } \mathrm { C }$ , one week at $- 2 0 ^ { \circ } \mathrm { C }$ , two weeks at $- 8 0 ^ { \circ } \mathrm { C }$ , and three freeze/thaw cycles at $- 2 0 ^ { \circ } \mathrm { C }$ .

The swab samples yielded the expected test results after being held for: 24 hours at $1 5 { - } 3 0 ^ { \circ } \mathrm { C }$ (room temperature) and 24 hours at $2 { - } 8 ^ { \circ } \mathrm { C }$ .

# 4. Analytical Sensitivity (Detection of Limit):

The limit of detection (LoD) of the Accula Strep A Test is the concentration at which $\ge 9 5 \%$ of samples with the Strep A target yield positive results. The LoD was determined by testing two Strep A strains (BAA-946 and ATCC 19615) that were serially diluted into a pooled negative throat sample matrix and spiked onto a swab.

A range-finding study was first conducted to establish the lower limit of analytical sensitivity using 2-fold bacterial dilutions tested in five replicates. Bacterial concentrations which produced $\geq 4 / 5$ positive results were tested in twenty replicates across multiple days with multiple reagent lots. If the dilution tested did not result in at least 19/20 positive results, a two-fold higher concentration was tested again with twenty replicates. The final bacteria amount at which $\ge 9 5 \%$ of results were positive for the Strep A target is the LoD and ranged from 25 – 188 CFU $\mathrm { 1 0 - 7 5 \mathrm { C F U / m L } }$ ; Table 6):

Table 6. Accula Strep A Limit of Detection   

<table><tr><td rowspan=1 colspan=1>Strep A Strain</td><td rowspan=1 colspan=1>LoD (CFU) 1</td><td rowspan=1 colspan=1>LoD (CFU/mL) 2</td></tr><tr><td rowspan=1 colspan=1>BAA-946</td><td rowspan=1 colspan=1>188</td><td rowspan=1 colspan=1>75 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>ATCC 19615</td><td rowspan=1 colspan=1>25</td><td rowspan=1 colspan=1>10 CFU/mL</td></tr></table>

1 Number of organisms (colony forming units) applied to swab prior to elution. 2 Final concentration of organisms after $1 0 \mu \mathrm { L }$ of bacterial dilution is spiked onto swab and eluted in $2 . 5 ~ \mathrm { m L }$ Strep A buffer (assuming $100 \%$ recovery of bacteria).

# 5. Analytical Sensitivity (Inclusivity):

Wet testing - The analytical reactivity of the Accula Strep A Test was evaluated with four strains of S. pyogenes that were not evaluated in the LoD study. Each strain was tested in triplicate at 1.5X LoD and 3X LoD, based on the LoD determine for Strep A strain BAA-946 $( 7 5 \mathrm { C F U / m L } )$ . Contrived samples were prepared by diluting bacteria into a pooled negative throat sample matrix and spiking onto a swab. Each strain at 3X LoD was detected $100 \%$ (3/3) with the Accula Strep A Test (Table 7).

Table 7. Inclusivity of Accula Strep A Test   

<table><tr><td rowspan=1 colspan=1>Strep A Strain</td><td rowspan=1 colspan=1>Bacteria Titer</td><td rowspan=1 colspan=1>Percent Detection(# positive/3)</td></tr><tr><td rowspan=2 colspan=1>ATCC 10403</td><td rowspan=1 colspan=1>225 CFU/mL (3X LoD)</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>112.5 CFU/mL (1.5X LoD)</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=2 colspan=1>ATCC 21548</td><td rowspan=1 colspan=1>225 CFU/mL (3X LoD)</td><td rowspan=1 colspan=1>100%(3/3)</td></tr><tr><td rowspan=1 colspan=1>112.5 CFU/mL (1.5X LoD)</td><td rowspan=1 colspan=1>66.67% (2/3)</td></tr><tr><td rowspan=2 colspan=1>ATCC 700294</td><td rowspan=1 colspan=1>225 CFU/mL (3X LoD)</td><td rowspan=1 colspan=1>100%(3/3)</td></tr><tr><td rowspan=1 colspan=1>112.5 CFU/mL (1.5X LoD)</td><td rowspan=1 colspan=1>100%(3/3)</td></tr><tr><td rowspan=2 colspan=1>ATCC 700497</td><td rowspan=1 colspan=1>225 CFU/mL (3X LoD)</td><td rowspan=1 colspan=1>100%(3/3)</td></tr><tr><td rowspan=1 colspan=1>112.5 CFU/mL (1.5X LoD)</td><td rowspan=1 colspan=1>100% (3/3)</td></tr></table>

In silico analysis - An in silico evaluation was performed to assess the predicted reactivity of the Accula Strep A Test with a broad spectrum of S. pyogenes isolates. A screen of the NCBI nt database with Accula Strep A Test primer and probes sequences with stringent search criteria (allowing no more than 2 mismatches in the primers and enforcing perfect homology for 6 nucleotides in primer $_ 3 \cdot$ ends) resulted in 272 hits, all of which were $S .$ . pyogenes whole genome sequences. In addition, when querying the $2 7 6 S .$ pyogenes whole genome sequences (including duplicate sequences) retrieved from a NCBI search (“Streptococcus pyogenes” [Organism] AND “complete genome” [All Fields]), 272 sequences were again identified that met the stringent search criteria. The four entries that failed to meet the search criteria represented two S. pyogenes genomes (due to the presence of duplicate sequences in the database) that were isolated from skin or skin/soft tissue infection; neither isolate was associated with Streptococcal throat infection. Therefore, the primers and probes of the Accula Strep A Test are conserved across different strains of S. pyogenes.

6. Analytical Specificity/Interference:

# Analytical Exclusivity/Cross-reactivity

Wet testing - To evaluate potential cross-reactivity of the Accula Strep A Test, analytical specificity was evaluated by testing 47 potentially cross-reactive microorganisms (Table 8). Each microorganism was initially tested in triplicate at $\geq 1 0 ^ { 6 } \mathrm { C F U / m L }$ for bacteria and yeast and $\geq 1 0 ^ { 6 }$ T $\mathrm { \ ' C I D 5 0 / m L }$ for viruses in the absence of Strep A.

Interference with the Accula Strep A Test was evaluated by testing the same 47 microorganisms, in triplicate, in the presence of Strep A strain BAA-946 at 3X LoD. As a positive control, 3X LoD Strep A strain BAA-946 was tested in the absence of potentially cross-reacting organisms. A summary of the analytical specificity study is provided in Table 8.

Table 8. Accula Strep A Results with Potential Cross-Reactive Organisms in the Absence of Presence of 3X LoD Strep A   

<table><tr><td colspan="1" rowspan="2">OrganismKey #</td><td colspan="1" rowspan="2">Organism Name</td><td colspan="1" rowspan="2">Test Level</td><td colspan="2" rowspan="1">Test Results (# of Strep A Pos / 3)</td></tr><tr><td colspan="1" rowspan="1">Absence of Strep A</td><td colspan="1" rowspan="1">Presence of Strep A</td></tr><tr><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Adenovirus Type 1</td><td colspan="1" rowspan="1">1.00E+06TCID50/mL</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">Arcanobacterium haemolyticum</td><td colspan="1" rowspan="1">2.00E+07CFU/ml</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">Bacillus cereus</td><td colspan="1" rowspan="1">1.00E+06CFU/ml</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">Bordetella pertussis</td><td colspan="1" rowspan="1">2.00E+06CFU/ml</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">Burkholderia cepacia</td><td colspan="1" rowspan="1">2.00E+07CFU/ml</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">Campylobacter rectus</td><td colspan="1" rowspan="1">1.00E+06CFU/ml</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">7</td><td colspan="1" rowspan="1">Candida albicans</td><td colspan="1" rowspan="1">2.00E+06CFU/ml</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">8</td><td colspan="1" rowspan="1">Corynebacterium diphtheriae</td><td colspan="1" rowspan="1">5.00E+06CFU/ml</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">9</td><td colspan="1" rowspan="1">Enterococcus faecalis</td><td colspan="1" rowspan="1">2.00E+06CFU/ml</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">Escherichia coli 1</td><td colspan="1" rowspan="1">1.00 -2.00E+07CFU/ml</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">11</td><td colspan="1" rowspan="1">Fusobacterium necrophorum</td><td colspan="1" rowspan="1">1.00E+06CFU/ml</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">12</td><td colspan="1" rowspan="1">Haemophilus influenzae</td><td colspan="1" rowspan="1">1.00E+06CFU/ml</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">13</td><td colspan="1" rowspan="1">Human Influenza virus A</td><td colspan="1" rowspan="1">1.00E+06TCID50/mL</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">14</td><td colspan="1" rowspan="1">Human Influenza virus B</td><td colspan="1" rowspan="1">1.00E+06TCID50/mL</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">15</td><td colspan="1" rowspan="1">Human metapneumovirus</td><td colspan="1" rowspan="1">1.00E+06TCID50/mL</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">16</td><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">2.00E+07CFU/ml</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">17</td><td colspan="1" rowspan="1">Lactobacillus acidophilus</td><td colspan="1" rowspan="1">2.00E+06CFU/ml</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">18</td><td colspan="1" rowspan="1">Lactococcus lactis</td><td colspan="1" rowspan="1">2.00E+07CFU/ml</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">19</td><td colspan="1" rowspan="1">Legionella longbeachae</td><td colspan="1" rowspan="1">1.00E+07</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">CFU/ml</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">Moraxella catarrhalis</td><td colspan="1" rowspan="1">1.00E+06CFU/ml</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">21</td><td colspan="1" rowspan="1">Mycoplasma pneumoniae</td><td colspan="1" rowspan="1">1.00E+06CCU/ml</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">22</td><td colspan="1" rowspan="1">Neisseria gonorrhoeae</td><td colspan="1" rowspan="1">3.00E+06CFU/ml</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">23</td><td colspan="1" rowspan="1">Parainfluenza Type 3</td><td colspan="1" rowspan="1">1.00E+06TCID50/mL</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">24</td><td colspan="1" rowspan="1">Parvimonas micra(Peptostreptococcus micros)</td><td colspan="1" rowspan="1">1.50E+06CFU/ml</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">25</td><td colspan="1" rowspan="1">Prevotella oralis (Bacteroidesoralis</td><td colspan="1" rowspan="1">1.00E+06CFU/ml</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">26</td><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">1.00E+06CFU/ml</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">27</td><td colspan="1" rowspan="1">Respiratory syncytial virusType B</td><td colspan="1" rowspan="1">1.00E+06TCID50/mL</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">28</td><td colspan="1" rowspan="1">Rhinovirus 2</td><td colspan="1" rowspan="1">5.00E+05-1.00E+06TCID50/mL</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">29</td><td colspan="1" rowspan="1">Saccharomyces cerevisiae</td><td colspan="1" rowspan="1">2.00E+06CFU/ml</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">30</td><td colspan="1" rowspan="1">Staphylococcus epidermidis</td><td colspan="1" rowspan="1">5.00E+07CFU/ml</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">31</td><td colspan="1" rowspan="1">Stenotrophomonas maltophilia</td><td colspan="1" rowspan="1">5.00E+07CFU/ml</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">32</td><td colspan="1" rowspan="1">Streptococcus agalactiae</td><td colspan="1" rowspan="1">2.00E+06CFU/ml</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">33</td><td colspan="1" rowspan="1">Streptococcus anginosus</td><td colspan="1" rowspan="1">2.00E+06CFU/ml</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">34</td><td colspan="1" rowspan="1">Streptococcus bovis</td><td colspan="1" rowspan="1">5.00E+06CFU/ml</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">35</td><td colspan="1" rowspan="1">Streptococcus canis</td><td colspan="1" rowspan="1">2.00E+07CFU/ml</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">36</td><td colspan="1" rowspan="1">Streptococcus constellatussubsp. Pharyngis</td><td colspan="1" rowspan="1">3.00E+06CFU/ml</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">37</td><td colspan="1" rowspan="1">Streptococcus dysgalactiaesubsp. Equisimilis</td><td colspan="1" rowspan="1">1.00E+06CFU/ml</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">38</td><td colspan="1" rowspan="1">Streptococcus gallolyticus</td><td colspan="1" rowspan="1">2.00E+06CFU/ml</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">39</td><td colspan="1" rowspan="1">Streptococcus intermedius</td><td colspan="1" rowspan="1">2.00E+06CFU/ml</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">40</td><td colspan="1" rowspan="1">Streptococcus mitis</td><td colspan="1" rowspan="1">2.00E+06CFU/ml</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">41</td><td colspan="1" rowspan="1">Streptococcus mutans</td><td colspan="1" rowspan="1">2.00E+07CFU/ml</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">42</td><td colspan="1" rowspan="1">Streptococcus oralis</td><td colspan="1" rowspan="1">2.00E+06CFU/ml</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">43</td><td colspan="1" rowspan="1">Streptococcus pneumonia</td><td colspan="1" rowspan="1">2.00E+06CFU/ml</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">44</td><td colspan="1" rowspan="1">Streptococcus salivarius</td><td colspan="1" rowspan="1">2.00E+06CFU/ml</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">45</td><td colspan="1" rowspan="1">Streptococcus sanguinus</td><td colspan="1" rowspan="1">2.00E+06CFU/ml</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">46</td><td colspan="1" rowspan="1">Treponema denticola</td><td colspan="1" rowspan="1">2.00E+06CFU/ml</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">47</td><td colspan="1" rowspan="1">Veillonella parvula</td><td colspan="1" rowspan="1">2.00E+07CFU/ml</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td></tr></table>

1 Escherichia coli was originally tested at $2 \mathrm { \times 1 0 ^ { 7 } C F U / m L }$ and gave the expected $_ { 0 / 3 }$ positive results in the absence of Strep A but 2/3 positive results in the presence of Strep A. No cross-reactivity was observed at 1 $\mathrm { \Delta x \ 1 0 ^ { 7 } C F U / m L }$ . 2 Rhinovirus was tested at $1 \mathrm { \Delta x 1 0 ^ { 6 } T C I D 5 0 / m L }$ and gave the expected $_ { 0 / 3 }$ positive results in the absence of Strep A and $3 / 3$ in the presence of Strep A. However, the visually graded maximum line intensity of “ $3 ^ { , , }$ was not observed. No interference with line intensity was observed at a concentration of $5 \mathrm { ~ x ~ } 1 0 ^ { 5 }$ T $\mathrm { \ C I D 5 0 / m L }$ .

Cross-reactivity was not detected in any of the 47 organisms tested with in the Accula Strep A Test at the levels tested; in the absence of the 3X LoD Strep A strain, all samples were negative.

In general, 3X LoD Strep A samples were detected in the presence of each potentially crossreacting microorganisms tested. Escherichia coli was originally tested at $2 \mathrm { \Delta x 1 0 ^ { 7 } C F U / m L }$ and gave the expected 0/3 positive results in the absence of Strep A but $2 / 3$ positive results in the presence of Strep A suggesting interference of $E$ . coli at this concentration with Strep A detection. Further dilutions were tested; no cross-reactivity was observed at $1 \mathrm { ~ x ~ } 1 0 ^ { 7 } \mathrm { C F U / m L }$ . This is included as a footnote in the device labeling.

Rhinovirus was tested at $1 \mathrm { ~ x ~ } 1 0 ^ { 6 } \mathrm { T C I D } 5 0 / \mathrm { m L }$ and gave the expected 0/3 positive results in the absence of Strep A and $3 / 3$ in the presence of Strep A. However, the visually graded maximum line intensity of $" 3 "$ was not observed on the test strip with at least one replicate, suggesting interference of Rhinovirus at this concentration with Strep A detection. Further dilutions of this virus were tested; no interference with line intensity was observed at a concentration of $5 \mathrm { ~ x ~ } 1 0 ^ { 5 }$ TCID50/mL. This is included as a footnote in the device labeling.

In silico analysis - To evaluate potential cross-reactivity and interference with the Accula Strep A Test, in silico analyses were performed. Queries of the NCBI nt database allowing up to 11 mismatches in the Strep A primer sequences while requiring only a single perfect match at the $_ { 3 } ,$ end of each primer failed to identify hits other than S. pyogenes strains. Additional focused queries were performed using local and publicly available databases for organisms and viruses included in Table 9.

Table 9. In silico Analysis for Potential Cross-Reactive or Interfering Organisms   

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Sequences indatabase</td><td rowspan=1 colspan=1>NCBI Search String</td><td rowspan=1 colspan=1>Predictedamplicon hits to one primer</td><td rowspan=1 colspan=1>Predicted hitsamplicon hits to one primer</td></tr><tr><td rowspan=1 colspan=1>Enterococcus spp.</td><td rowspan=1 colspan=1>3040</td><td rowspan=1 colspan=1>&quot;Enterococcus&quot;[Organism] AND&quot;complete genome&quot;[All Fields]</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Klebsiella spp.</td><td rowspan=1 colspan=1>9738</td><td rowspan=1 colspan=1>&quot;Klebsiella&quot;[Organism] AND&quot;complete genome&quot;[All Fields]</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Lactococcus spp.</td><td rowspan=1 colspan=1>179</td><td rowspan=1 colspan=1>&quot;Lactococcus lactis&quot;[Organism] AND&quot;complete genome&quot;[All Fields]</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Legionella spp.</td><td rowspan=1 colspan=1>212</td><td rowspan=1 colspan=1>&quot;Legionella&quot;[Organism] AND&quot;complete genome&quot;[All Fields]</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Mycoplasmapneumoniae</td><td rowspan=1 colspan=1>155</td><td rowspan=1 colspan=1>&quot;Mycoplasma pneumoniae&quot;[Organism]AND &quot;complete genome&quot;[All Fields]</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Pseudomonas spp.</td><td rowspan=1 colspan=1>3828</td><td rowspan=1 colspan=1>&quot;Pseudomonas&quot;[Organism] AND&quot;complete genome&quot;[All Fields]</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Stenotrophomonasmaltophilia</td><td rowspan=1 colspan=1>225</td><td rowspan=1 colspan=1>&quot;Stenotrophomonasmaltophilia&quot;[Organism] AND&quot;complete genome&quot;[All Fields]</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Adenovirus Type 1</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>&quot;Adenovirus type 1&quot;[Organism] AND&quot;complete genome&quot;[All Fields]</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Adenovirus Type 7</td><td rowspan=1 colspan=1>87</td><td rowspan=1 colspan=1>&quot;Adenovirus type 7&quot;[Organism] AND&quot;complete genome&quot;[All Fields]</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Human parainfluenza</td><td rowspan=1 colspan=1>377</td><td rowspan=1 colspan=1>&quot;Human parainfluenza virus&quot;[AllFields] AND &quot;complete genome&quot;[AllFields]</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Humanmetapneumovirus</td><td rowspan=1 colspan=1>156</td><td rowspan=1 colspan=1>&quot;Human metapneumovirus&quot;[All Fields]AND &quot;complete genome&quot;[All Fields]</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Respiratory syncytialvirus</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>&quot;Respiratory syncytialvirus&quot;[Organism] AND &quot;completegenome&quot;[All Fields]</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Rhinovirus</td><td rowspan=1 colspan=1>3692</td><td rowspan=1 colspan=1>&quot;Rhinovirus&quot;[All Fields] AND&quot;complete genome&quot;[All Fields]</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Streptococcus spp.(No S. pyogenes)</td><td rowspan=1 colspan=1>2138</td><td rowspan=1 colspan=1>(&quot;Streptococcus&quot;[Organism] AND&quot;complete genome&quot;[All Fields]) NOT&quot;Streptococcus pyogenes&quot;[Organism]</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>15</td></tr></table>

BLASTn results were analyzed to identify any hits predicted to generate a PCR amplicon of 88 to 2000 bases. No database sequences were predicted to generate amplicons. However, of the databases evaluated, only the non- $S _ { \cdot }$ . pyogenes Streptococcus spp. database generated hits that met the search criteria for one primer (reverse). Further evaluation identified an $8 6 \%$ homology of the reverse primer and $67 \%$ homology of the forward primer with $S .$ . dysgalactiae and S. dysgalactiae subsp. equisimilis sequences. Despite significant homology with the reverse primer, cross-reactivity and interference studies using high levels of S. dysgalactiae subsp. equisimilis failed to identify any cross-reactivity or interference with Strep A detection (Table 8). In addition, the forward primer shares 12 bases of homology with S. canis at the $_ { 3 } ,$ end. After a single gap, the remaining 33 bases in the $5 '$ portion shares almost $8 8 \%$ homology with S. canis. The reverse primer shares less than $3 6 \%$ homology with

S. canis sequence. Experimental testing for $S .$ . canis cross-reactivity and interference with Strep A detection failed to detect any cross-reactivity or interference (Table 8). Therefore, despite some homology with the primers and probes of the Accula Strep A Test, the presence of other organisms or viruses are unlikely to cross-react or interfere with Strep A detection.

# Interfering Substances

To evaluate substances with the potential to interfere with the performance of the Accula Strep A Test, samples with and without S. pyogenes (strain BAA-946) were tested in triplicate with potentially interfering substances. To prepare contrived samples, swabs were dipped into a liquid sample containing the potential interferent at a concentration likely to be found in a respiratory sample. Swabs were then eluted into PNTS (prepared with Strep A buffer). Next, Strep A positive swabs were prepared at a final concentration of 3X LoD (225 CFU/mL) and eluted into the PNTS with or without the potential interferent. Substances shown to interfere with the detection of Strep A (for positive samples) or the internal control (IC, for negative samples) were further diluted and re-tested in the presence and absence of Strep A. The highest concentration that showed no interference (i.e., $3 / 3 = 1 0 \%$ agreement with expected results) is shown in Table 10.

Table 10. Evaluation of Potentially Interfering Substances on Accula Strep A Test   

<table><tr><td colspan="1" rowspan="1">Potential Interferent</td><td colspan="1" rowspan="1">Active Ingredient</td><td colspan="1" rowspan="1">AdsorbedConcentration</td><td colspan="1" rowspan="1">Approximate ElutedConcentration</td><td colspan="1" rowspan="1">Target</td></tr><tr><td colspan="1" rowspan="2">Blood (Human)</td><td colspan="1" rowspan="2">NA</td><td colspan="1" rowspan="1">50% (v/v)</td><td colspan="1" rowspan="1">4% (v/v)</td><td colspan="1" rowspan="1">Strep A</td></tr><tr><td colspan="1" rowspan="1">12.5% (v/v)</td><td colspan="1" rowspan="1">1% (v/v)</td><td colspan="1" rowspan="1">IC</td></tr><tr><td colspan="1" rowspan="2">Chloroseptic Max</td><td colspan="1" rowspan="2">Phenol 1.5%,Glycerin 33%</td><td colspan="1" rowspan="2">100% (v/v)</td><td colspan="1" rowspan="2">8% (v/v)</td><td colspan="1" rowspan="1">Strep A</td></tr><tr><td colspan="1" rowspan="1">IC</td></tr><tr><td colspan="1" rowspan="2">Cold &amp; Flu Relief Cough Syrup</td><td colspan="1" rowspan="2">Acetaminophen 21.7 mg/mL,Dextromethorphan 0.67mg/mL, Guaifenesin 13.3mg/mL, Phenylephrine 0.33mg/mL</td><td colspan="1" rowspan="2">100% (v/v)</td><td colspan="1" rowspan="2">8% (v/v)</td><td colspan="1" rowspan="1">Strep A</td></tr><tr><td colspan="1" rowspan="1">IC</td></tr><tr><td colspan="1" rowspan="2">Listerine Cool Mint AntisepticMouth Wash</td><td colspan="1" rowspan="2">Eucalyptol 0.092%,Menthol 0.042%,Methyl Salicylate 0.060%,Thymol 0.064%</td><td colspan="1" rowspan="2">100% (v/v)</td><td colspan="1" rowspan="2">8% (v/v)</td><td colspan="1" rowspan="1">Strep A</td></tr><tr><td colspan="1" rowspan="1">IC</td></tr><tr><td colspan="1" rowspan="2">Cepacol (throat lozenge)</td><td colspan="1" rowspan="2">Benzocaine,Menthol</td><td colspan="1" rowspan="2">0.3% Benzocaine (w/v),0.046% Menthol (w/v)</td><td colspan="1" rowspan="2">0.024% Benzocaine (w/v),0.0037% Menthol (w/v)</td><td colspan="1" rowspan="1">Strep A</td></tr><tr><td colspan="1" rowspan="1">IC</td></tr><tr><td colspan="1" rowspan="2">Sucrets</td><td colspan="1" rowspan="2">Dyclonine Hydrochloride,Menthol</td><td colspan="1" rowspan="2">0.06% DyclonineHydrochloride (w/v),0.12% Menthol (w/v)</td><td colspan="1" rowspan="2">0.0048% DyclonineHydrochloride (w/v),0.0096% Menthol (w/v)</td><td colspan="1" rowspan="1">Strep A</td></tr><tr><td colspan="1" rowspan="1">IC</td></tr><tr><td colspan="1" rowspan="2">Crest Pro Health FluorideToothpaste</td><td colspan="1" rowspan="2">Stannous Fluoride 0.454%(0.14% w/v Fluoride Ion)</td><td colspan="1" rowspan="2">100% (v/v)</td><td colspan="1" rowspan="2">8% (v/v)</td><td colspan="1" rowspan="1">Strep A</td></tr><tr><td colspan="1" rowspan="1">IC</td></tr><tr><td colspan="1" rowspan="2">Halls Triple Soothing CoughDrops1</td><td colspan="1" rowspan="2">Eucalyptus Oil</td><td colspan="1" rowspan="2">100% (v/v)</td><td colspan="1" rowspan="2">8% (v/v)</td><td colspan="1" rowspan="1">Strep A</td></tr><tr><td colspan="1" rowspan="1">IC</td></tr><tr><td colspan="1" rowspan="2">Advil Liqui-Gels</td><td colspan="1" rowspan="2">Ibuprofen</td><td colspan="1" rowspan="2">100% (v/v)</td><td colspan="1" rowspan="2">8% (v/v)</td><td colspan="1" rowspan="1">Strep A</td></tr><tr><td colspan="1" rowspan="1">IC</td></tr><tr><td colspan="1" rowspan="2">Miralax</td><td colspan="1" rowspan="2">Polyethylene Glycol</td><td colspan="1" rowspan="2">30.4% (w/v)</td><td colspan="1" rowspan="2">2.43% (w/v)</td><td colspan="1" rowspan="1">Strep A</td></tr><tr><td colspan="1" rowspan="1">IC</td></tr><tr><td colspan="1" rowspan="2">Tums Extra Strength</td><td colspan="1" rowspan="2">Calcium Carbonate</td><td colspan="1" rowspan="1">20 mg/mL</td><td colspan="1" rowspan="1">1.6 mg/mL</td><td colspan="1" rowspan="1">Strep A</td></tr><tr><td colspan="1" rowspan="1">30 mg/mL</td><td colspan="1" rowspan="1">2.4 mg/mL</td><td colspan="1" rowspan="1">IC</td></tr><tr><td colspan="1" rowspan="2">Food Dye</td><td colspan="1" rowspan="2">N/A</td><td colspan="1" rowspan="2">100% (v/v)</td><td colspan="1" rowspan="2">8% (v/v)</td><td colspan="1" rowspan="1">Strep A</td></tr><tr><td colspan="1" rowspan="1">IC</td></tr><tr><td colspan="1" rowspan="1">Whole Milk (Dairy)</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">12.50% (v/v)</td><td colspan="1" rowspan="1">1% (v/v)</td><td colspan="1" rowspan="1">Strep A</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Y</td><td colspan="1" rowspan="1">50% (v/v)</td><td colspan="1" rowspan="1">4% (v/v)</td><td colspan="1" rowspan="1">IC</td></tr><tr><td colspan="1" rowspan="2">Orange Juice</td><td colspan="1" rowspan="2">N/A</td><td colspan="1" rowspan="1">50% (v/v)</td><td colspan="1" rowspan="1">4%(v/v)</td><td colspan="1" rowspan="1">Strep A</td></tr><tr><td colspan="1" rowspan="1">100% (v/v)</td><td colspan="1" rowspan="1">8% (v/v)</td><td colspan="1" rowspan="1">IC</td></tr><tr><td colspan="1" rowspan="2">Penicillin G</td><td colspan="1" rowspan="2">Penicillin G SodiumSalt</td><td colspan="1" rowspan="2">10% (w/v)</td><td colspan="1" rowspan="2">0.8% (w/v)</td><td colspan="1" rowspan="1">Strep A</td></tr><tr><td colspan="1" rowspan="1">IC</td></tr><tr><td colspan="1" rowspan="2">Cephalexin</td><td colspan="1" rowspan="2">Cephalexin</td><td colspan="1" rowspan="2">2.5% (w/v)</td><td colspan="1" rowspan="2">0.2% (w/v)</td><td colspan="1" rowspan="1">Strep A</td></tr><tr><td colspan="1" rowspan="1">IC</td></tr><tr><td colspan="1" rowspan="2">Mucin, Type II (from porcinestomach)</td><td colspan="1" rowspan="2">Purified mucin protein</td><td colspan="1" rowspan="1">5% (w/v)</td><td colspan="1" rowspan="1">0.4% (w/v)</td><td colspan="1" rowspan="1">Strep A</td></tr><tr><td colspan="1" rowspan="1">10% (w/v)</td><td colspan="1" rowspan="1">0.8% (w/v)</td><td colspan="1" rowspan="1">IC</td></tr><tr><td colspan="1" rowspan="2">Tobramycin (antibacterial)</td><td colspan="1" rowspan="2">Tobramycin</td><td colspan="1" rowspan="2">7.5% (w/v)</td><td colspan="1" rowspan="2">0.6% (w/v)</td><td colspan="1" rowspan="1">Strep A</td></tr><tr><td colspan="1" rowspan="1">IC</td></tr><tr><td colspan="1" rowspan="2">Amoxicillin</td><td colspan="1" rowspan="2">Amoxicillin</td><td colspan="1" rowspan="2">10% (w/v)</td><td colspan="1" rowspan="2">0.8% (w/v)</td><td colspan="1" rowspan="1">Strep A</td></tr><tr><td colspan="1" rowspan="1">IC</td></tr></table>

IC: Internal Control, for negative samples 1 Eucalyptus oil was used in place of Halls Triple Soothing Cough Drops due to a lack of availability. Eucalyptus oil is an active ingredient in the Halls cough drops.

Expected results were obtained with all potential interferants at the final concentration noted in Table 10. However, prior to obtaining the final concentration, interference was observed for some substances at higher concentrations. This is addressed as a footnote in the device labeling and described below:

Human blood showed inhibition at $100 \%$ concentration for Strep A detection in the positive sample, but no inhibition at $50 \%$ concentration.   
Human blood showed inhibition at $100 \%$ , $50 \%$ and $2 5 \%$ concentrations for IC detection in the negative sample, but no inhibition at $1 2 . 5 \%$ concentration.   
Tums was initially tested with a solution of $1 . 5 ~ \mathrm { g / m L }$ (1 Tum dissolved into $2 . 5 ~ \mathrm { m L }$ of Strep A Buffer). This inhibited all reactions. The Strep A Positive sample was inhibited when tested with an additional 8x and $3 2 \mathrm { x }$ dilution. No inhibition was observed at a concentration of $2 0 \mathrm { m g / m L }$ for the Strep A Positive sample. The Negative Strep A sample showed inhibition when tested with an additional 8x dilution. No inhibition was observed at a concentration of $3 0 \mathrm { m g / m L }$ for the Strep A Negative sample.   
Milk showed inhibition at $100 \%$ , $50 \%$ and $2 5 \%$ concentrations for Strep A detection in the positive sample, but no inhibition at $1 2 . 5 \%$ concentration.   
Milk showed inhibition at the $100 \%$ concentration for IC detection in the negative sample, but no inhibition at $50 \%$ concentration.   
Orange juice showed inhibition at $100 \%$ concentration for detection of Strep A in the positive sample, but no inhibition at $50 \%$ .

7. Assay Cut-Off:

Not applicable

# B Comparison Studies:

1. Method Comparison with Predicate Device: Not applicable

Not applicable

# C Clinical Studies:

1. Clinical Sensitivity and Specificity:

The performance of the Accula Strep A Test was evaluated in a prospective, multi-center study conducted at nine point of care (POC; e.g., physician office laboratory, urgent care, outpatient clinics) investigational sites representative of a CLIA-waived environment in the United States between May 2019 and January 2020. Fresh throat swab specimens were collected from patients with signs and symptoms of pharyngitis: pharyngeal pain (sore throat), tonsillar swelling with exudates, pharyngeal erythema (reddened throat), tender cervical lymphadenopathy, or fever. Two throat swab specimens were collected from each participant: one Copan FLOQswab for use with the Accula Strep A Test and one Copan eSwab Liquid Amies Collection System for concurrent testing by Blood Agar culture and an FDA-cleared molecular assay.

Swab specimens were tested with the Accula Strep A Test on the same days as collection by non-laboratory health professionals who are representative of typical intended use operators (e.g., nurses, physician assistants, medical assistants) in POC locations. eSwab Liquid Amies samples used for comparator testing were stored refrigerated and shipped refrigerated to the reference laboratory within 24 hours of collection. Swabs were used to inoculate Blood Agar culture plates while swab eluates in Liquid Amies were used for testing with the FDAcleared molecular assay.

Table 11 summarizes the number of evaluable results included in the Accula Strep A performance analysis. Throat swabs were collected from 669 participants; however, only 663 Accula Strep A Test assays were performed due to exclusion of six samples with protocol deviations or incidents. An additional four samples with unresolved invalid/error Accula Strep A Test results were excluded from analysis. When using the Blood Agar culture comparator, an additional five results were excluded from the analysis due to protocol deviations or incidents, for a total of 654 Accula Strep A Test and Blood Agar culture results available for analysis. When using the molecular assay comparator, an additional 11 results were excluded from the analysis (six results excluded due to protocol deviations or incidents, and five results excluded due to invalid results), for a total of 648 Accula Strep A Test and molecular assay comparator results available for analysis. All samples with valid Accula Strep A Test and comparator results are included in the performance analysis. All samples with discrepant results when compared to either culture or the molecular comparator assay as well as $10 \%$ of samples with non-discrepant results were also tested with a second FDAcleared molecular assay.

Table 11. Summary of Evaluable Results for Accula Strep A Test Performance Analysis   

<table><tr><td colspan="1" rowspan="1">Exclusions</td><td colspan="1" rowspan="1">Accula Strep ATest</td><td colspan="1" rowspan="1">Blood Agarculture</td><td colspan="1" rowspan="1">MolecularComparator</td></tr><tr><td colspan="1" rowspan="1">Protocol Deviations/Incidents</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">6</td></tr><tr><td colspan="1" rowspan="1">Unresolved Invalid/Error Result</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">5</td></tr><tr><td colspan="1" rowspan="1">Total Excluded Results</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">11</td></tr><tr><td>Total Evaluable Results</td><td>659</td><td>6541</td><td>6481</td></tr></table>

1 Total number of Accula Strep A Test and comparator results included in performance analysis.

Performance of the Accula Strep A Test compared to Blood Agar culture (Table 12) and compared to an FDA-cleared molecular assay (Table 13) is shown below.

Table 12. Accula Strep A Test Clinical Performance vs. Blood Agar Culture   

<table><tr><td rowspan=2 colspan=2></td><td rowspan=1 colspan=3>Blood Agar Culture</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=4 colspan=1>Are V de S</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>126</td><td rowspan=1 colspan=1>132</td><td rowspan=1 colspan=1>139</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>51</td><td rowspan=1 colspan=1>510</td><td rowspan=1 colspan=1>515</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>131</td><td rowspan=1 colspan=1>523</td><td rowspan=1 colspan=1>654</td></tr><tr><td rowspan=1 colspan=4>Sensitivity: 126/131 = 96.2% (95% CI: 91.4-98.4%)Specificity: 510/523 = 97.5% (95% CI: 95.8-98.5%)Positive Predictive Value: 126/139 = 90.6% (95% CI: 84.7-94.5%)Negative Predictive Value: 510/515 = 99.0% (95% CI: 97.7-99.6%)</td></tr></table>

1 Strep A was not detected in 2/5 False Negative specimens using an FDA-cleared assay for discrepant analysis. 2 Strep A was detected in 7/13 False Positive specimens using an FDA-cleared assay for discrepant analysis.

Table 13. Accula Strep A Test Clinical Performance vs. FDA-cleared Molecular Assay   

<table><tr><td rowspan=2 colspan=2></td><td rowspan=1 colspan=3>FDA-cleared Assay</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=4 colspan=1>Are V dee s</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>137</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>138</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>91</td><td rowspan=1 colspan=1>501</td><td rowspan=1 colspan=1>510</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>146</td><td rowspan=1 colspan=1>502</td><td rowspan=1 colspan=1>648</td></tr><tr><td rowspan=1 colspan=4>Positive Percent Agreement: 137/146 = 93.8% (95% CI: 88.7-96.7%)Negative Percent Agreement: 501/502 = 99.8% (95% CI: 98.9-100%)Positive Predictive Value: 137/138 = 99.3% (95% CI: 96.0-99.9%)Negative Predictive Value: 501/513 = 98.2% (95% CI: 96.7-99.1%)</td></tr></table>

1 Strep A was not detected in 5/9 False Negative specimens using an FDA-cleared assay for discrepant analysis. 2 Strep A was detected in $_ { 0 / 1 }$ False Positive specimens using an FDA-cleared assay for discrepant analysis.

Performance of the Accula Strep A Test when compared to Blood Agar culture or the FDAcleared molecular assay, and stratified by clinical sites is shown in Table 14 and Table 15, respectively.

Table 14. Accula Strep A Test vs. Blood Agar Culture: Performance Stratified by Site   

<table><tr><td colspan="1" rowspan="1">Site</td><td colspan="1" rowspan="1">No. ofSamplesCollected</td><td colspan="1" rowspan="1">No. ofSamplesTested (%)</td><td colspan="1" rowspan="1">Culture Positive(% Prevalence)</td><td colspan="1" rowspan="1">Sensitivity %(Pos/True Positive)[95% CI]</td><td colspan="1" rowspan="1">Specificity %(Neg/True Negative)[95% CI]</td></tr><tr><td colspan="1" rowspan="1">ADP</td><td colspan="1" rowspan="1">150</td><td colspan="1" rowspan="1">148 (22.6)</td><td colspan="1" rowspan="1">40 (27.0)</td><td colspan="1" rowspan="1">97.5 (39/40)</td><td colspan="1" rowspan="1">97.2 (105/108)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">[87.1-99.6]</td><td colspan="1" rowspan="1">[92.1-99.1]</td></tr><tr><td colspan="1" rowspan="1">AIM</td><td colspan="1" rowspan="1">44</td><td colspan="1" rowspan="1">44 (6.7)</td><td colspan="1" rowspan="1">7 (15.9)</td><td colspan="1" rowspan="1">100 (7/7)[64.6-100]</td><td colspan="1" rowspan="1">100 (37/37)[90.6-100]</td></tr><tr><td colspan="1" rowspan="1">CAL</td><td colspan="1" rowspan="1">30</td><td colspan="1" rowspan="1">29 (4.4)</td><td colspan="1" rowspan="1">7 (24.1)</td><td colspan="1" rowspan="1">100 (7/7)[64.6-100]</td><td colspan="1" rowspan="1">95.5 (21/22)[78.2-99.2]</td></tr><tr><td colspan="1" rowspan="1">CFC</td><td colspan="1" rowspan="1">14</td><td colspan="1" rowspan="1">14 (2.1)</td><td colspan="1" rowspan="1">2 (14.3)</td><td colspan="1" rowspan="1">50.0 (1/2)[9.5-90.5]</td><td colspan="1" rowspan="1">100 (12/12)[75.8-100]</td></tr><tr><td colspan="1" rowspan="1">GVP</td><td colspan="1" rowspan="1">9</td><td colspan="1" rowspan="1">9 (1.4)</td><td colspan="1" rowspan="1">5 (55.6)</td><td colspan="1" rowspan="1">100 (5/5)[56.6-100]</td><td colspan="1" rowspan="1">75 (3/4)[30.1-95.4]</td></tr><tr><td colspan="1" rowspan="1">PAC</td><td colspan="1" rowspan="1">156</td><td colspan="1" rowspan="1">152 (23.2)</td><td colspan="1" rowspan="1">34 (22.4)</td><td colspan="1" rowspan="1">91.2 (31/34)[77.0-97.0]</td><td colspan="1" rowspan="1">95.8 (113/118)[90.5-98.2]</td></tr><tr><td colspan="1" rowspan="1">PEL</td><td colspan="1" rowspan="1">71</td><td colspan="1" rowspan="1">69 (10.6)</td><td colspan="1" rowspan="1">9 (13.0)</td><td colspan="1" rowspan="1">100 (9/9)[70.1-100]</td><td colspan="1" rowspan="1">98.3 (59/60)[91.1-99.7]</td></tr><tr><td colspan="1" rowspan="1">RPM</td><td colspan="1" rowspan="1">34</td><td colspan="1" rowspan="1">33 (5.0)</td><td colspan="1" rowspan="1">2 (6.1)</td><td colspan="1" rowspan="1">100 (2/2)[34.2-100]</td><td colspan="1" rowspan="1">100 (31/31)[89.0-100]</td></tr><tr><td colspan="1" rowspan="1">TRU</td><td colspan="1" rowspan="1">161</td><td colspan="1" rowspan="1">156 (23.9)</td><td colspan="1" rowspan="1">25 (16.0)</td><td colspan="1" rowspan="1">100 (25/25)[86.7-100]</td><td colspan="1" rowspan="1">98.5 (129/131)[94.6-99.6]</td></tr><tr><td colspan="1" rowspan="1">Total</td><td colspan="1" rowspan="1">668</td><td colspan="1" rowspan="1">654 (97.9)</td><td colspan="1" rowspan="1">131 (20.0)</td><td colspan="1" rowspan="1">96.2 (126/131)[91.4-98.4]</td><td colspan="1" rowspan="1">97.5 (510/523)[95.8-98.5]</td></tr></table>

Table 15. Accula Strep A Test vs FDA-Cleared Molecular Assay: Performance Stratified by Site   

<table><tr><td rowspan=1 colspan=1>Site</td><td rowspan=1 colspan=1>No. ofSamplesCollected</td><td rowspan=1 colspan=1>No. ofSamplesTested (%)</td><td rowspan=1 colspan=1>MolecularPositive(% Positivity)</td><td rowspan=1 colspan=1>Sensitivity %(Pos/True Positive)[95% CI]</td><td rowspan=1 colspan=1>Specificity %(Neg/True Negative)[95% CI]</td></tr><tr><td rowspan=1 colspan=1>ADP</td><td rowspan=1 colspan=1>150</td><td rowspan=1 colspan=1>147 (22.7)</td><td rowspan=1 colspan=1>45 (30.6)</td><td rowspan=1 colspan=1>93.3 (42/45)[82.1-97.7]</td><td rowspan=1 colspan=1>100 (102/102)[96.4-100]</td></tr><tr><td rowspan=1 colspan=1>AIM</td><td rowspan=1 colspan=1>44</td><td rowspan=1 colspan=1>44 (6.8)</td><td rowspan=1 colspan=1>8 (18.2)</td><td rowspan=1 colspan=1>87.5 (7/8)[52.9-97.8]</td><td rowspan=1 colspan=1>100 (36/36)[90.4-100]</td></tr><tr><td rowspan=1 colspan=1>CAL</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>29 (4.5)</td><td rowspan=1 colspan=1>9 (31.0)</td><td rowspan=1 colspan=1>88.9 (8/9)[56.5-98.0]</td><td rowspan=1 colspan=1>100 (20/20)[83.9-100]</td></tr><tr><td rowspan=1 colspan=1>CFC</td><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>14 (2.2)</td><td rowspan=1 colspan=1>1 (7.1)</td><td rowspan=1 colspan=1>100 (1/1)[20.7-100]</td><td rowspan=1 colspan=1>100 (13/13)[77.2-100]</td></tr><tr><td rowspan=1 colspan=1>GVP</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>9 (1.4)</td><td rowspan=1 colspan=1>6 (66.7)</td><td rowspan=1 colspan=1>100 (6/6)[61.0-100]</td><td rowspan=1 colspan=1>100 (3/3)[43.9-100]</td></tr><tr><td rowspan=1 colspan=1>PAC</td><td rowspan=1 colspan=1>156</td><td rowspan=1 colspan=1>151 (23.3)</td><td rowspan=1 colspan=1>39 (25.8)</td><td rowspan=1 colspan=1>89.7 (35/39)[76.4-95.9]</td><td rowspan=1 colspan=1>99.7 (111/112)[95.1-99.8]</td></tr><tr><td rowspan=1 colspan=1>PEL</td><td rowspan=1 colspan=1>71</td><td rowspan=1 colspan=1>68 (10.5)</td><td rowspan=1 colspan=1>9 (13.2)</td><td rowspan=1 colspan=1>100 (9/9)[70.1-100]</td><td rowspan=1 colspan=1>100 (59/59)[93.9-100]</td></tr><tr><td rowspan=1 colspan=1>RPM</td><td rowspan=1 colspan=1>34</td><td rowspan=1 colspan=1>31 (4.8)</td><td rowspan=1 colspan=1>2 (6.5)</td><td rowspan=1 colspan=1>100 (2/2)[34.2-100]</td><td rowspan=1 colspan=1>100 (29/29)[88.3-100]</td></tr><tr><td rowspan=1 colspan=1>TRU</td><td rowspan=1 colspan=1>161</td><td rowspan=1 colspan=1>155 (23.9)</td><td rowspan=1 colspan=1>27 (17.4)</td><td rowspan=1 colspan=1>100 (27/27)[87.5-100]</td><td rowspan=1 colspan=1>100 (128/128)[97.1-100]</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>668</td><td rowspan=1 colspan=1>648 (97.0)</td><td rowspan=1 colspan=1>146 (22.5)</td><td rowspan=1 colspan=1>93.8 (137/146)[88.7-96.7]</td><td rowspan=1 colspan=1>99.8 (501/502)[98.9-100]</td></tr></table>

During the prospective clinical study, 56 samples did not produce valid Accula Strep A Test results during the initial test $5 6 / 6 6 3 = 8 . 4 \%$ . Of the 56 samples, 39 samples $3 9 / 6 6 3 =$

$5 . 8 \% )$ gave invalid results due to failed internal controls and 17 samples ( $1 7 / 6 6 3 = 2 . 6 \%$ did not generate a result due to system errors caused by the Dock (e.g., cassette failure or misuse, inappropriate sample volume, or amplification error). After repeat testing of the 56 samples, seven results (5 invalids and 2 system errors) remained unresolved $( 7 / 7 1 9 = 1 . 0 \%$ ). The seven samples were tested for a third time; three provided valid results while four remained unresolved $( 4 / 7 2 6 = 0 . 6 \%$ ).

2. Other Clinical Supportive Data (When 1. and 2. Are Not Applicable):

# D Clinical Cut-Off:

Not applicable

# E Expected Values/Reference Range:

The clinical study analysis included results from 654 throat swab specimens collected from nine point of care sites in the U.S. between May 2019 and January 2020. The overall prevalence of $S _ { \cdot }$ . pyogenes was $2 0 . 0 \%$ (131/654) as determined by culture and the overall positivity was $2 1 . 3 \%$ (139/654) as determined by the Accula Strep A Test. The prevalence and positivity rate stratified by age and sex of the participants are shown in Table 16.

Table 16. Prevalence / Positivity Rate Stratified by Age and Sex   

<table><tr><td rowspan=2 colspan=1>Age / Sex</td><td rowspan=2 colspan=1>No.Tested</td><td rowspan=1 colspan=2>Blood Agar Culture</td><td rowspan=1 colspan=2>Accula Strep A Test</td></tr><tr><td rowspan=1 colspan=1>No. Positive</td><td rowspan=1 colspan=1>Prevalence</td><td rowspan=1 colspan=1>No. Positive</td><td rowspan=1 colspan=1>Positivity Rate</td></tr><tr><td rowspan=1 colspan=1>≤5 years</td><td rowspan=1 colspan=1>112</td><td rowspan=1 colspan=1>33</td><td rowspan=1 colspan=1>29.5%</td><td rowspan=1 colspan=1>34</td><td rowspan=1 colspan=1>30.4%</td></tr><tr><td rowspan=1 colspan=1>6-21 years</td><td rowspan=1 colspan=1>292</td><td rowspan=1 colspan=1>64</td><td rowspan=1 colspan=1>21.9%</td><td rowspan=1 colspan=1>70</td><td rowspan=1 colspan=1>24.0%</td></tr><tr><td rowspan=1 colspan=1>22-59 years</td><td rowspan=1 colspan=1>224</td><td rowspan=1 colspan=1>33</td><td rowspan=1 colspan=1>14.7%</td><td rowspan=1 colspan=1>34</td><td rowspan=1 colspan=1>15.2%</td></tr><tr><td rowspan=1 colspan=1>≥60 years</td><td rowspan=1 colspan=1>26</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>3.8%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>3.8%</td></tr><tr><td rowspan=1 colspan=1>Female</td><td rowspan=1 colspan=1>386</td><td rowspan=1 colspan=1>70</td><td rowspan=1 colspan=1>18.1%</td><td rowspan=1 colspan=1>74</td><td rowspan=1 colspan=1>19.2%</td></tr><tr><td rowspan=1 colspan=1>Male</td><td rowspan=1 colspan=1>268</td><td rowspan=1 colspan=1>61</td><td rowspan=1 colspan=1>22.8%</td><td rowspan=1 colspan=1>65</td><td rowspan=1 colspan=1>24.3%</td></tr><tr><td rowspan=1 colspan=1>TOTAL</td><td rowspan=1 colspan=1>654</td><td rowspan=1 colspan=1>131</td><td rowspan=1 colspan=1>20.0%</td><td rowspan=1 colspan=1>139</td><td rowspan=1 colspan=1>21.3%</td></tr></table>

# VIII Proposed Labeling:

The labeling supports the finding of substantial equivalence for this device.

# IX Conclusion:

The submitted information in this premarket notification is complete and supports a substantial equivalence decision.