# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATION DECISION SUMMARY

I Background Information:

# B Applicant

Roche Molecular Systems, Inc.

C Proprietary and Established Names

cobas BKV, cobas EBV/BKV Control Kit, cobas Buffer Negative Control Kit

# D Regulatory Information

<table><tr><td rowspan=1 colspan=1>ProductCode(s)</td><td rowspan=1 colspan=1>Classification</td><td rowspan=1 colspan=1>RegulationSection</td><td rowspan=1 colspan=1>Panel</td></tr><tr><td rowspan=1 colspan=1>QMI</td><td rowspan=1 colspan=1>Class II</td><td rowspan=1 colspan=1>21 CFR 21 CFR 866.3183- Quantitative ViralNucleic Acid Test forTransplant PatientManagement</td><td rowspan=1 colspan=1>MI</td></tr></table>

II Submission/Device Overview:

A Purpose for Submission: To obtain substantial equivalence determination of the cobas BKV assay.   
B Measurand: BKV DNA   
C Type of Test: The cobas BKV assay is a quantitative Polymerase Chain Reaction (PCR) performed on the cobas 6800/8800 automated systems, for the detection of BKV DNA in transplant patients.

# III Intended Use/Indications for Use:

# A Intended Use(s):

cobas® BKV is an in vitro nucleic acid amplification test for the quantitation of BK virus (BKV) DNA in human EDTA plasma on the cobas® 6800/8800 Systems.

cobas® BKV is intended for use as an aid in the management of BKV in transplant patients. In patients undergoing monitoring of BKV, serial DNA measurements can be used to indicate the need for potential treatment changes and to assess viral response to treatment.

The results from cobas® BKV are intended to be read and analyzed by a qualified licensed healthcare professional in conjunction with clinical signs and symptoms and relevant laboratory findings. Test results must not be the sole basis for patient management decisions.

cobas® BKV is not intended for use as a screening test for blood or blood products or human cells, tissues, and cellular and tissue-based products (HCT/Ps).

# B Indication(s) for Use:

Same as intended use

# C Special Conditions for Use Statement(s):

[Choose Rx or OTC]

Rx

# D Special Instrument Requirements:

The test is run on the cobas 6800/8800 instrument system

# IV Device/System Characteristics: A Device Description:

cobas BKV is a quantitative test performed on the cobas 6800 System and cobas 8800 System. cobas BKV enables the detection of BKV DNA in plasma specimens. The cobas BKV assay is a dual target assay, with both targets using the same dye. The DNA Internal Control, used to monitor the entire sample preparation and PCR amplification process, is introduced into each specimen during sample processing. cobas BKV enables the detection and quantitation of BKV DNA in EDTA plasma transplant patients.

cobas BKV is intended for use as an aid in the management of BKV in transplant patients. In patients undergoing monitoring of BKV, serial DNA measurements can be used to indicate the need for potential treatment changes and to assess viral response to treatment.

The cobas BKV Kit system consists of:

• Proteinase Solution

• DNA Quantitation Standard (DNA QS)   
• Elution Buffer   
• Master Mix Reagent 1   
• BKV Master Mix Reagent 2

The BKV viral load is quantified against a non-BKV DNA quantitation standard (DNA-QS), which is introduced into each specimen during sample preparation. The DNA-QS also functions as an internal control for sample preparation and the PCR amplification process.

In addition, the test utilizes the following separately packed and sold control materials:

1. cobas EBV/BKV Positive Control Kit :

EBV/BKV Low Positive Control (EBV/BKV $\mathrm { L } ( + ) \mathrm { C } )$ EBV/BKV High Positive Control (EBV/BKV $\mathrm { H + }$ )C)

The positive control contains phage packaged EBVBKV and BKV DNA in normal human plasma and serves as a control for the cobas BKV test.

2. cobas Negative Control Kit:

cobas Buffer Negative Control (BUF (-) C)

Testing with the cobas BKV test requires the following materials that are not provided:

cobas OMNI Reagents cobas BKV Assay Specific Analysis Package (ASAP) software

The cobas BKV test uses sample preparation (nucleic acid extraction and purification) followed by PCR amplification and detection, all steps are fully automated by the cobas 6800/8800 platform.

# B Principle of Operation:

The cobas BKV test is a quantitative PCR test performed on the fully automated cobas 6800/ 8800 Systems that detects and quantifies BKV DNA from (EDTA) plasma specimens of transplant patients as follows.

# Target Selection

Selective amplification of BKV target nucleic acid from the sample is achieved using specific forward and reverse primers which are selected to amplify highly-conserved regions of the BKV DNA VP-2 and T-antigen gene. Specific probes for each amplicon are used to detect and quantify the BKV targets from subtypes I, II, III and IV as well as subgroups Ia, Ib, Ic, IVa, IVb, and IVc. Selective amplification of DNA-QS is achieved using DNA-QS specific forward and reverse primers, selected to have no homology with the BKV genome, detected through a DNAQS specific probe.

# Sample Preparation (Nucleic Acid Extraction and Purification)

Nucleic acid from patient samples and external controls are extracted upon addition of a DNA Quantitation standard (DNA-QS). The DNA-QS molecules are extracted simultaneously with the samples/controls serving as an extraction control. Viral nucleic acid is released by addition of proteinase and lysis reagent to the sample. The released nucleic acid, along with the added DNAQS binds to magnetic glass particles. Unbound substances and impurities are removed with

subsequent wash reagent steps and purified nucleic acid is then eluted from the magnetic glass particles with elution buffer.

# Nucleic Acid Amplification and Target Detection

The cobas BKV master mix contains detection probes which are specific for the two BKV target sequences and the DNA-QS nucleic acid, respectively. The two BKV specific detection probes are labeled with the same HEX fluorescent dye while the DNA-QS detection probe is labeled with the CY5.5 fluorescent dye both acting as reporter dyes. Each probe also has a second dye, BHQ, which acts as a quencher that suppresses the fluorescent signals of the intact probes when they are not bound to their respective target sequence. Target bound probes, however, emit fluorescence of the two reporter dyes. This fluorescence is measured at defined wavelengths, thus permitting simultaneous detection and discrimination of the BKV targets and the DNA-QS amplification products generated by a thermostable DNA polymerase enzyme.

# BKV DNA Quantitation

During the extension phase of the PCR process, fluorescence readings are processed to generate Ct values for the BKV DNA target and the BKV QS DNA. The lot-specific calibration constants provided with the cobas BKV test are used to calculate the titer value for the specimens and controls based on both the BKV DNA target and the BKV QS DNA Ct values. BKV viral load results are reported in International Units/mL (IU/mL).

# C Instrument Description Information:

1. Instrument Name:

cobas 6800/8800

2. Specimen Identification:

The specimen identification information is captured and stored as a digital record. Whole blood collected in appropriate tubes as per instructions for use are prepared for the test procedure as described in detail in the cobas $\textsuperscript { \textregistered }$ 6800/8800 Systems – User Assistance and/or User Guide.

3. Specimen Sampling and Handling:

Managed by a trained technician.

4. Calibration:

The lot-specific calibration constants are provided with the cobas BKV test. The cobas 6800/8800 Systems automatically determine the BKV DNA concentration for the samples and controls. The BKV DNA concentration is expressed in International Units per milliliter (IU/mL).

5. Quality Control:

One negative control $[ ( - ) \mathrm { C } ]$ and two positive controls, a low positive control [EBV/BKV $\mathrm { L } ( + ) \mathrm { C } ]$ and a high positive control [EBV/BKV $\mathrm { H } ( + ) \mathrm { C } ]$ is processed with each batch. The batch is valid if no flags appear for all three controls. The negative control result is displayed as $( - ) \mathrm { { C } }$ and the low and high positive controls are displayed as EBV/BKV $\mathrm { L } ( + ) \mathrm { C }$ and EBV/BKV $\mathrm { H } ( + ) \mathrm { C }$ . Invalidation of results is performed automatically by the cobas $\textsuperscript { \textregistered }$ 6800/8800 software based on negative and positive control failures.

V Substantial Equivalence Information:

A Predicate Device Name(s): cobas EBV   
B Predicate 510(k) Number(s): DEN200015   
C Comparison with Predicate(s):

<table><tr><td colspan="1" rowspan="1">Device &amp; PredicateDevice(s):</td><td colspan="1" rowspan="1">New Device: cobas BKV(K202215)</td><td colspan="1" rowspan="1">Predicate Device: cobas EBV(DEN200015)</td></tr><tr><td colspan="1" rowspan="1">Device Trade Name</td><td colspan="1" rowspan="1">cobas BKV</td><td colspan="1" rowspan="1">cobas EBV</td></tr><tr><td colspan="3" rowspan="1">General Device Characteristic Similarities</td></tr><tr><td colspan="1" rowspan="1">Regulation</td><td colspan="1" rowspan="1">21 CFR 866.3183</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Regulation name</td><td colspan="1" rowspan="1">Quantitative viral nucleic acidtest for transplant patientmanagement</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Intended Use/Indicationsfor Use</td><td colspan="1" rowspan="1">cobas® BKV is an in vitronucleic acid amplification testfor the quantitation of BKvirus (BKV) DNA in humanEDTA plasma on the cobas®6800/8800 Systems.cobas® BKV is intended foruse as an aid in themanagement of BKV intransplant patients. In patientsundergoing monitoring ofBKV, serial DNAmeasurements can be used toindicate the need for potentialtreatment changes and toassess viral response totreatment.The results from cobas®BKV are intended to be readand analyzed by a qualifiedlicensed healthcareprofessional in conjunction</td><td colspan="1" rowspan="1">cobas® EBV is an in vitronucleic acid amplification testfor the quantitation ofEpstein-Barr virus (EBV)DNA in human EDTA plasmaon the cobas® 6800/8800Systems.cobas® EBV is intended foruse as an aid in themanagement of EBV intransplant patients. In patientsundergoing monitoring ofEBV, serial DNAmeasurements can be used toindicate the need for potentialtreatment changes and toassess response to treatment.The results from cobas® EBVare intended to be read andanalyzed by a qualifiedlicensed healthcareprofessional in conjunction</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">with clinical signs andsymptoms and relevantlaboratory findings.cobas® BKV is not intendedfor use as a screening test fordonors of blood or bloodproducts or human cells,tissues, and cellular andtissue-based products(HCT/Ps).</td><td colspan="1" rowspan="1">with clinical signs andsymptoms and relevantlaboratory findings.cobas® EBV is not intendedfor use as a screening test fordonors of blood or bloodproducts or human cells,tissues, and cellular andtissue-based products(HCT/Ps).</td></tr><tr><td colspan="1" rowspan="1">Conditions for use</td><td colspan="1" rowspan="1">For prescription use</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Patient population</td><td colspan="1" rowspan="1">Transplant recipients</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Sample Types</td><td colspan="1" rowspan="1">EDTA - plasma</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Sample PreparationProcedure and TargetDetection</td><td colspan="1" rowspan="1">cobas® 6800/8800 Systems</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Analyte</td><td colspan="1" rowspan="1">DNA</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Calibrators</td><td colspan="1" rowspan="1">Phagemid</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Amplification Technology</td><td colspan="1" rowspan="1">Real-time PCR</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Detection Chemistry</td><td colspan="1" rowspan="1">Paired reporter and quencherfluorescence labeled probes(TaqMan Technology) usingfluorescence resonanceenergy transfer (FRET)</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Controls used</td><td colspan="1" rowspan="1">Sample processing control(IC)Positive and negative control</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Result Analysis</td><td colspan="1" rowspan="1">Based on PCR cyclethreshold analysis</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="3" rowspan="1">General Device Characteristic Differences</td></tr><tr><td colspan="1" rowspan="1">Analyte Targets</td><td colspan="1" rowspan="1">BK Virus</td><td colspan="1" rowspan="1">EB Virus</td></tr></table>

# VI Standards/Guidance Documents Referenced:

EP05-A3-Evaluation of Precision of Quantitative Measurement Procedures; Approved Guideline—Third Edition

EP6-A-Evaluation of the Linearity of Quantitative Measurement Procedures: A Statistical Approach; Approved Guideline.

EP07-A2-Interference Testing in Clinical Chemistry; Approved Guideline—Second Edition

EP17-A2-Evaluation of Detection Capability for Clinical Laboratory Measurement Procedures; Approved Guideline – Second Edition

# A Analytical Performance:

# Precision:

Precision was assessed for the predominant BKV Subgroup 1b by serially diluting BKV 1b lambda stock in BKV negative EDTA-Plasma to generate a 7-member panel ranging from 60 $\mathrm { I U / m L }$ to $\mathrm { 1 x 1 0 ^ { 6 } \ : I U / m L }$ .

Precision was calculated on results generated with 21 replicates per run, 2 runs per day using 3 kit lots, 4 cobas 6800 systems and two operators. This testing design accounts for a total of 72 replicates per panel member (PM).

Results were analyzed according to CLSI guideline EP5-A. For data analysis only samples with tiers within the linear range of the assay were used (i.e., between $2 1 . 5 \ : \mathrm { I U / m L }$ and $\mathrm { 1 x ~ 1 0 ^ { 8 } ~ I U / m L }$ ). For the PM1, the assigned concentration was $9 . 8 3 \mathrm { ~ x ~ } 1 0 ^ { 7 } \ : \mathrm { I U / m L }$ with $6 3 / 7 1$ replicates above the Upper Limit of Quantification and the data were excluded from the analysis. The results for cobas BKV precision calculated for 6 dilution levels are shown in Table 1.

Table 1: Total Precision as SD of log10 Titer Results from EDTA-plasma   

<table><tr><td rowspan=2 colspan=1>NominalConcentration(IU/mL)</td><td rowspan=2 colspan=1>AssignedConcentration[IU/mL]</td><td rowspan=2 colspan=1>Assigned log10Titer</td><td rowspan=1 colspan=4>Standard Deviation</td></tr><tr><td rowspan=1 colspan=1>Lot 1</td><td rowspan=1 colspan=1>Lot 2</td><td rowspan=1 colspan=1>Lot 3</td><td rowspan=1 colspan=1>Alllots</td></tr><tr><td rowspan=1 colspan=1>1.00E+06</td><td rowspan=1 colspan=1>9.83E+05</td><td rowspan=1 colspan=1>5.99</td><td rowspan=1 colspan=1>0.02</td><td rowspan=1 colspan=1>0.02</td><td rowspan=1 colspan=1>0.04</td><td rowspan=1 colspan=1>0.03</td></tr><tr><td rowspan=1 colspan=1>1.00E+05</td><td rowspan=1 colspan=1>9.83E+04</td><td rowspan=1 colspan=1>4.99</td><td rowspan=1 colspan=1>0.03</td><td rowspan=1 colspan=1>0.04</td><td rowspan=1 colspan=1>0.04</td><td rowspan=1 colspan=1>0.04</td></tr><tr><td rowspan=1 colspan=1>1.00E+04</td><td rowspan=1 colspan=1>9.83E+03</td><td rowspan=1 colspan=1>3.99</td><td rowspan=1 colspan=1>0.04</td><td rowspan=1 colspan=1>0.05</td><td rowspan=1 colspan=1>0.03</td><td rowspan=1 colspan=1>0.04</td></tr><tr><td rowspan=1 colspan=1>6.00E+03</td><td rowspan=1 colspan=1>5.90E+03</td><td rowspan=1 colspan=1>3.77</td><td rowspan=1 colspan=1>0.03</td><td rowspan=1 colspan=1>0.05</td><td rowspan=1 colspan=1>0.03</td><td rowspan=1 colspan=1>0.04</td></tr><tr><td rowspan=1 colspan=1>1.00E+02</td><td rowspan=1 colspan=1>9.83E+01</td><td rowspan=1 colspan=1>1.99</td><td rowspan=1 colspan=1>0.09</td><td rowspan=1 colspan=1>0.11</td><td rowspan=1 colspan=1>0.11</td><td rowspan=1 colspan=1>0.11</td></tr><tr><td rowspan=1 colspan=1>6.00E+01</td><td rowspan=1 colspan=1>5.90E+01</td><td rowspan=1 colspan=1>1.77</td><td rowspan=1 colspan=1>0.14</td><td rowspan=1 colspan=1>0.11</td><td rowspan=1 colspan=1>0.13</td><td rowspan=1 colspan=1>0.13</td></tr></table>

\* Titer data are considered to be log-normally distributed and are analyzed following log10 transformation. Standard deviations (SD) columns present the total of the log-transformed titer for each of the three reagent lot

The variance component analysis demonstrated the contribution of the components of variance to the total precision variance (Table 2). Overall, the total precision as SD of the $\mathrm { L o g } _ { 1 0 }$ titer is comparable across all kits.

Table 2: Lognormal Percent Coefficient of Variation (% CV) \*   

<table><tr><td rowspan=1 colspan=2>Nominal concentration(IU/mL)</td><td rowspan=1 colspan=2>Assignedconcentration(IU/mL)</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Instr. /Operator</td><td rowspan=1 colspan=1>BetweenLot</td><td rowspan=1 colspan=1>BetweenDay</td><td rowspan=1 colspan=1>BetweenRun</td><td rowspan=1 colspan=1>WithinRun</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Titer</td><td rowspan=1 colspan=1>Log10titer</td><td rowspan=1 colspan=1>Titer</td><td rowspan=1 colspan=1>Log10titer</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=6>%CV</td></tr><tr><td rowspan=1 colspan=1>1.00 x106</td><td rowspan=1 colspan=1>6.00</td><td rowspan=1 colspan=1>9.83x105</td><td rowspan=1 colspan=1>5.99</td><td rowspan=1 colspan=1>72</td><td rowspan=1 colspan=1>2%</td><td rowspan=1 colspan=1>5%</td><td rowspan=1 colspan=1>3%</td><td rowspan=1 colspan=1>2%</td><td rowspan=1 colspan=1>5%</td><td rowspan=1 colspan=1>8%</td></tr><tr><td rowspan=1 colspan=1>1.00x105</td><td rowspan=1 colspan=1>5.00</td><td rowspan=1 colspan=1>9.83x104</td><td rowspan=1 colspan=1>4.99</td><td rowspan=1 colspan=1>71</td><td rowspan=1 colspan=1>3%</td><td rowspan=1 colspan=1>6%</td><td rowspan=1 colspan=1>3%</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>8%</td><td rowspan=1 colspan=1>11%</td></tr><tr><td rowspan=1 colspan=1>1.00x104</td><td rowspan=1 colspan=1>4.00</td><td rowspan=1 colspan=1>9.83x103</td><td rowspan=1 colspan=1>3.99</td><td rowspan=1 colspan=1>70</td><td rowspan=1 colspan=1>3%</td><td rowspan=1 colspan=1>7%</td><td rowspan=1 colspan=1>5%</td><td rowspan=1 colspan=1>3%</td><td rowspan=1 colspan=1>9%</td><td rowspan=1 colspan=1>13%</td></tr><tr><td rowspan=1 colspan=1>6.00x103</td><td rowspan=1 colspan=1>3.78</td><td rowspan=1 colspan=1>5.90x103</td><td rowspan=1 colspan=1>3.77</td><td rowspan=1 colspan=1>72</td><td rowspan=1 colspan=1>2%</td><td rowspan=1 colspan=1>8%</td><td rowspan=1 colspan=1>2%</td><td rowspan=1 colspan=1>1%</td><td rowspan=1 colspan=1>8%</td><td rowspan=1 colspan=1>12%</td></tr><tr><td rowspan=1 colspan=1>1.00x102</td><td rowspan=1 colspan=1>2.00</td><td rowspan=1 colspan=1>9.83x101</td><td rowspan=1 colspan=1>1.99</td><td rowspan=1 colspan=1>72</td><td rowspan=1 colspan=1>5%</td><td rowspan=1 colspan=1>8%</td><td rowspan=1 colspan=1>6%</td><td rowspan=1 colspan=1>4%</td><td rowspan=1 colspan=1>24%</td><td rowspan=1 colspan=1>26%</td></tr><tr><td rowspan=1 colspan=1>6.00x101</td><td rowspan=1 colspan=1>1.78</td><td rowspan=1 colspan=1>5.90x101</td><td rowspan=1 colspan=1>1.77</td><td rowspan=1 colspan=1>71</td><td rowspan=1 colspan=1>4%</td><td rowspan=1 colspan=1>14%</td><td rowspan=1 colspan=1>7%</td><td rowspan=1 colspan=1>15%</td><td rowspan=1 colspan=1>29%</td><td rowspan=1 colspan=1>36%</td></tr></table>

\* Titer data are considered to be log-normally distributed and the $\% \mathrm { C V }$ values are analyzed as Lognormal CV $( \% ) =$ sqrt $( 1 0 ^ { \wedge } \ : [ \mathrm { S D } ^ { \wedge } 2 ^ { \ast } \ln \left( 1 0 \right) ] \cdot 1 ) ^ { \ast } \ : 1 0 0 \%$

# Reproducibility Study:

Reproducibility of cobas BKV was evaluated in EDTA plasma across three Reagent Lots, three Test Sites, three Instruments (two cobas 6800 Systems and one cobas 8800 System). Two runs were performed per operator per day ( $1 \mathrm { r u n } = 1$ batch; 1 batch $= 1$ panel $^ + 3$ controls) over five days per reagent lot and each run had three replicates per panel member. The total number of tests (not including controls) was as follows: 3 lots $\times 3$ sites $\times 5$ days/lot $\times 2$ runs $\times 3$ replicates/concentration $= 2 7 0$ test results/concentration.

Test panel members were prepared from BKV VCA IgG sero-negative and DNA negative EDTA plasma spiked with either the BKV WHO Standard or BKV genotype 1b genomic material in the form of BKV cell culture supernatant. Test panel members had the following concentrations: Negative, $6 4 . 5 \mathrm { I U / m L }$ (3x LLoQ), $5 \mathrm { x } 1 0 ^ { 3 } \mathrm { I U / m L }$ , $5 \mathrm { x } 1 0 ^ { 4 } \mathrm { I U / m L }$ , $5 \mathrm { x } 1 0 ^ { 5 } \mathrm { I U / m L }$ , and $5 \mathrm { x } 1 0 ^ { 7 } \mathrm { I U / m L }$ ( $0 . 5 \mathrm { x }$ ULoQ). Across the three testing sites, positive panel members produced 1350 results of which 1 was invalid (operator error) and excluded from analysis. For negative panel members all 270 test results were valid. The results are summarized in Table 3 below.

Table 3: Reproducibility Study   

<table><tr><td rowspan=1 colspan=1>ExpectedBKV DNAConcentration(log10 IU/mL)</td><td rowspan=1 colspan=1>ObservedMean² BKVDNAConcentration(log10 IU/mL)</td><td rowspan=1 colspan=1>Numberof Testsb</td><td rowspan=1 colspan=1>Lot%TVc(CV%)d</td><td rowspan=1 colspan=1>Site%TVc(CV%)d</td><td rowspan=1 colspan=1>Day/Operator%%TV(CV%)d</td><td rowspan=1 colspan=1>Batch%TVc(CV%)d</td><td rowspan=1 colspan=1>Within-Batch%TVc(CV%)d</td><td rowspan=1 colspan=1>TotalPrecisionSDe</td><td rowspan=1 colspan=1>TotalPrecisionLog-normalCV(%)d</td></tr><tr><td rowspan=1 colspan=1>1.81</td><td rowspan=1 colspan=1>1.74</td><td rowspan=1 colspan=1>270</td><td rowspan=1 colspan=1>9%(20.63)</td><td rowspan=1 colspan=1>6%(17.69)</td><td rowspan=1 colspan=1>0%(0.00)</td><td rowspan=1 colspan=1>7%(19.15)</td><td rowspan=1 colspan=1>78%(68.05)</td><td rowspan=1 colspan=1>0.304</td><td rowspan=1 colspan=1>79.43</td></tr><tr><td rowspan=1 colspan=1>3.70</td><td rowspan=1 colspan=1>3.52</td><td rowspan=1 colspan=1>270</td><td rowspan=1 colspan=1>10%(9.79)</td><td rowspan=1 colspan=1>10%(9.57)</td><td rowspan=1 colspan=1>14%(11.44)</td><td rowspan=1 colspan=1>25%(15.16)</td><td rowspan=1 colspan=1>40%(19.38)</td><td rowspan=1 colspan=1>0.131</td><td rowspan=1 colspan=1>30.91</td></tr><tr><td rowspan=1 colspan=1>4.70</td><td rowspan=1 colspan=1>4.51</td><td rowspan=1 colspan=1>270</td><td rowspan=1 colspan=1>3%(4.42)</td><td rowspan=1 colspan=1>24%(13.46)</td><td rowspan=1 colspan=1>0%(0.00)</td><td rowspan=1 colspan=1>56%(20.58)</td><td rowspan=1 colspan=1>17%(11.27)</td><td rowspan=1 colspan=1>0.118</td><td rowspan=1 colspan=1>27.71</td></tr><tr><td rowspan=1 colspan=1>5.70</td><td rowspan=1 colspan=1>5.54</td><td rowspan=1 colspan=1>270</td><td rowspan=1 colspan=1>7%(5.66)</td><td rowspan=1 colspan=1>28%(11.50)</td><td rowspan=1 colspan=1>0%(0.00)</td><td rowspan=1 colspan=1>40%(13.85)</td><td rowspan=1 colspan=1>25%(10.84)</td><td rowspan=1 colspan=1>0.094</td><td rowspan=1 colspan=1>21.94</td></tr><tr><td rowspan=1 colspan=1>7.70</td><td rowspan=1 colspan=1>7.62</td><td rowspan=1 colspan=1>269</td><td rowspan=1 colspan=1>4%(3.27)</td><td rowspan=1 colspan=1>49%(11.00)</td><td rowspan=1 colspan=1>0%(0.00)</td><td rowspan=1 colspan=1>13%(5.60)</td><td rowspan=1 colspan=1>34%(9.10)</td><td rowspan=1 colspan=1>0.068</td><td rowspan=1 colspan=1>15.74</td></tr></table>

a Calculated using SAS MIXED procedure. Number of valid tests with detectable DNA level. $\mathrm { { } c \ { } \% T V = }$ Percent contribution to Total Variance. ${ } ^ { \mathrm { d } } \mathrm { C V } \% =$ Lognormal percent coefficient of variation $= \operatorname { s q r t } ( 1 0 ^ { \land } [ \operatorname { S D } ^ { \land } 2 ^ { \ast } \ln ( 1 0 ) ] - 1 ) \ ^ { \ast } \ 1 0 0$ e Calculated using the total variability from the SAS MIXED procedure Note: The table only includes results with detectable DNA level. $\mathrm { S D } =$ standard deviation. $\mathrm { C V } =$ coefficient of variation; and $\mathrm { B K V } = \mathrm { B K }$ Virus

Analysis of variance and a mixed model that included lot, site, day/operator, batch and withinbatch (random error) as random effects was performed. The variance contribution of each component to the total variance was estimated. The range of the total lognormal coefficient of variation, among positive panel members, was from $1 5 . 7 4 \%$ to $7 9 . 4 3 \%$ . The largest total lognormal coefficient of variation was observed in the lowest positive panel member (3x LLoQ). Most of that variability ( $78 \%$ of the total variance) was explained by the within-batch component attributed to one clinical testing site.

# Linearity:

A linearity panel composed of 11 panel members, prepared by diluting BKV subgroup 1b (GT 1b) lambda DNA (phagemid) and 7 panel members prepared by diluting a subgroup 1b clinical specimen, were tested at each concentration level (Table 4). The highest assigned titer for the phagemid panel member was $1 . 9 7 \mathrm { E } { + 0 8 } \ \mathrm { I U } / \mathrm { m L }$ and the highest assigned titer for the clinical panel member was $6 . 7 2 \mathrm { E } { + } 0 4 \mathrm { I U } / \mathrm { m L }$ . The phagemid and clinical panel members overlapped in the intermediate and low part of the measuring range (from a nominal titer of $1 . 0 0 \mathrm { E } { + 0 5 } \mathrm { I U } / \mathrm { m L }$ to $1 . 5 0 \mathrm { E + 0 1 \Pi U / m L }$ ).

Table 4: Assigned Titer of the BKV Linearity Panel   

<table><tr><td colspan="1" rowspan="1">Panel Member</td><td colspan="1" rowspan="1">Nominal Titer[IU/mL]</td><td colspan="1" rowspan="1">Assigned Titer[IU/mL]</td><td colspan="1" rowspan="1">Sample Type</td></tr><tr><td colspan="1" rowspan="1">PM01</td><td colspan="1" rowspan="1">2.00E+08</td><td colspan="1" rowspan="1">1.97E+08</td><td colspan="1" rowspan="1">Phagemid</td></tr><tr><td colspan="1" rowspan="1">PM02</td><td colspan="1" rowspan="1">1.00E+08</td><td colspan="1" rowspan="1">9.83E+07</td><td colspan="1" rowspan="1">Phagemid</td></tr><tr><td colspan="1" rowspan="1">PM03</td><td colspan="1" rowspan="1">1.00E+07</td><td colspan="1" rowspan="1">9.83E+06</td><td colspan="1" rowspan="1">Phagemid</td></tr><tr><td colspan="1" rowspan="1">PM04</td><td colspan="1" rowspan="1">1.00E+06</td><td colspan="1" rowspan="1">9.83E+05</td><td colspan="1" rowspan="1">Phagemid</td></tr><tr><td colspan="1" rowspan="2">PM05</td><td colspan="1" rowspan="2">1.00E+05</td><td colspan="1" rowspan="1">6.72E+04</td><td colspan="1" rowspan="1">Clinical</td></tr><tr><td colspan="1" rowspan="1">9.83E+04</td><td colspan="1" rowspan="1">Phagemid</td></tr><tr><td colspan="1" rowspan="2">PM06</td><td colspan="1" rowspan="2">1.00E+04</td><td colspan="1" rowspan="1">6.72E+03</td><td colspan="1" rowspan="1">Clinical</td></tr><tr><td colspan="1" rowspan="1">9.83E+03</td><td colspan="1" rowspan="1">Phagemid</td></tr><tr><td colspan="1" rowspan="2">PM07</td><td colspan="1" rowspan="2">6.00E+03</td><td colspan="1" rowspan="1">4.03E+03</td><td colspan="1" rowspan="1">Clinical</td></tr><tr><td colspan="1" rowspan="1">5.90E+03</td><td colspan="1" rowspan="1">Phagemid</td></tr><tr><td colspan="1" rowspan="2">PM08</td><td colspan="1" rowspan="2">1.00E+02</td><td colspan="1" rowspan="1">6.72E+01</td><td colspan="1" rowspan="1">Clinical</td></tr><tr><td colspan="1" rowspan="1">9.83E+01</td><td colspan="1" rowspan="1">Phagemid</td></tr><tr><td colspan="1" rowspan="2">PM09</td><td colspan="1" rowspan="2">6.00E+01</td><td colspan="1" rowspan="1">4.03E+01</td><td colspan="1" rowspan="1">Clinical</td></tr><tr><td colspan="1" rowspan="1">5.90E+01</td><td colspan="1" rowspan="1">Phagemid</td></tr><tr><td colspan="1" rowspan="2">PM10</td><td colspan="1" rowspan="2">3.00E+01</td><td colspan="1" rowspan="1">2.02E+01</td><td colspan="1" rowspan="1">Clinical</td></tr><tr><td colspan="1" rowspan="1">2.95E+01</td><td colspan="1" rowspan="1">Phagemid</td></tr><tr><td colspan="1" rowspan="2">PM11</td><td colspan="1" rowspan="2">1.50E+01</td><td colspan="1" rowspan="1">1.01E+01</td><td colspan="1" rowspan="1">Clinical</td></tr><tr><td colspan="1" rowspan="1">1.47E+01</td><td colspan="1" rowspan="1">Phagemid</td></tr></table>

Testing was performed using three cobas 8800 instruments by three operators on 12 replicates for each concentration using three test-specific kit lots (a total of 36 replicates per concentration). Twelve run were performed of which nine runs containing a total of 648 valid results were analyzed. Three runs were manually excluded due to operator error (error during sample preparation). Results were analyzed according to CLSI guideline EP6-A. The acceptance criteria for the maximal deviation between the linear regression and the better fitting higher order regression model was set as equal to or less than $\mathbf { \varepsilon } _ { \cdot } \pm 0 . 3 \log _ { 1 0 } \Pi / \mathrm { m L }$ .

The parameter estimates of b0, b1, b2, and b3 are shown for $1 ^ { \mathrm { s t } }$ order linear and $3 ^ { \mathrm { r d } }$ order polynomial fits (Table 5). For all the test kits and sample types combined, both the higher order coefficients (b2 and b3) were significant at $5 \%$ level. The $3 ^ { \mathrm { r d } }$ order fit was chosen as the best fitting regression model as it had the smaller Root Mean Square Error (Figure 1). The maximal difference between the linear and $3 ^ { \mathrm { r d } }$ order fits was observed for the lowest clinical panel member with a nominal titer of $1 . 5 0 \mathrm { E + 0 1 \ I U / m L }$ (below the LLoQ of the assay).

Table 5: cobas BKV Linearity with BKV Genotype 1b.   

<table><tr><td rowspan=1 colspan=1>Sampletype</td><td rowspan=1 colspan=1>Lot</td><td rowspan=1 colspan=1>Coefficientlinearrregression(b0, Std.error)</td><td rowspan=1 colspan=1>Coefficientlinearregression(b1, Std.error)</td><td rowspan=1 colspan=1>Coefficient forbetter fittinghigher ordermodelregression(b0, Std. error)</td><td rowspan=1 colspan=1>Coefficient forbette fittinghigher ordermodelregression(b1, Std. error)</td><td rowspan=1 colspan=1>Coefficient forbetter fittinghigher ordermodel regression(b2, Std. error)</td><td rowspan=1 colspan=1>Coefficient forbetter fittinghigher ordermodelregression(b3, Std. error)</td><td rowspan=1 colspan=1>Maximumdifference*(Log10 1U/mL)</td></tr><tr><td rowspan=1 colspan=1>Clinical</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.18, 0.05</td><td rowspan=1 colspan=1>0.95, 0.02</td><td rowspan=1 colspan=1>1.28, 0.33</td><td rowspan=1 colspan=1>-0.33,0.41</td><td rowspan=1 colspan=1>0.41,0.14</td><td rowspan=1 colspan=1>-0.04,0.02</td><td rowspan=1 colspan=1>0.18</td></tr><tr><td rowspan=1 colspan=1>Clinical</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0.17, 0.05</td><td rowspan=1 colspan=1>0.97, 0.02</td><td rowspan=1 colspan=1>1.03, 0.31</td><td rowspan=1 colspan=1>-0.05,0.38</td><td rowspan=1 colspan=1>0.33, 0.14</td><td rowspan=1 colspan=1>-0.03, 0.02</td><td rowspan=1 colspan=1>0.14</td></tr><tr><td rowspan=1 colspan=1>Clinical</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0.26, 0.08</td><td rowspan=1 colspan=1>0.93, 0.03</td><td rowspan=1 colspan=1>2.09, 0.47</td><td rowspan=1 colspan=1>-1.23,0.58</td><td rowspan=1 colspan=1>0.70,0.20</td><td rowspan=1 colspan=1>-0.07, 0.02</td><td rowspan=1 colspan=1>0.30</td></tr><tr><td rowspan=1 colspan=1>Phagemid</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.12,0.02</td><td rowspan=1 colspan=1>0.99,0.00</td><td rowspan=1 colspan=1>0.15,0.03</td><td rowspan=1 colspan=1>0.97,0.02</td><td rowspan=1 colspan=1>0.00,0.00</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>0.02</td></tr><tr><td rowspan=1 colspan=1>Phagemid</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0.23, 0.02</td><td rowspan=1 colspan=1>0.98, 0.00</td><td rowspan=1 colspan=1>0.32, 0.03</td><td rowspan=1 colspan=1>0.92, 0.02</td><td rowspan=1 colspan=1>0.01,0.00</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>0.04</td></tr><tr><td rowspan=1 colspan=1>Phagemid</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0.13, 0.02</td><td rowspan=1 colspan=1>0.99,0.00</td><td rowspan=1 colspan=1>0.21, 0.04</td><td rowspan=1 colspan=1>0.94, 0.02</td><td rowspan=1 colspan=1>0.01,0.00</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>0.03</td></tr><tr><td rowspan=1 colspan=1>Combined</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.11, 0.02</td><td rowspan=1 colspan=1>0.99,0.00</td><td rowspan=1 colspan=1>0.42,0.09</td><td rowspan=1 colspan=1>0.71,0.08</td><td rowspan=1 colspan=1>0.06, 0.02</td><td rowspan=1 colspan=1>0.00,0.00</td><td rowspan=1 colspan=1>0.09</td></tr><tr><td rowspan=1 colspan=1>Combined</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0.17, 0.02</td><td rowspan=1 colspan=1>0.99,0.00</td><td rowspan=1 colspan=1>0.45, 0.08</td><td rowspan=1 colspan=1>0.74,0.08</td><td rowspan=1 colspan=1>0.06, 0.02</td><td rowspan=1 colspan=1>0.00, 0.00</td><td rowspan=1 colspan=1>0.08</td></tr><tr><td rowspan=1 colspan=1>Combined</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0.15, 0.03</td><td rowspan=1 colspan=1>0.98, 0.01</td><td rowspan=1 colspan=1>0.62, 0.12</td><td rowspan=1 colspan=1>0.56, 0.11</td><td rowspan=1 colspan=1>0.10,0.03</td><td rowspan=1 colspan=1>-0.01,0.00</td><td rowspan=1 colspan=1>0.14</td></tr><tr><td rowspan=1 colspan=1>Combined</td><td rowspan=1 colspan=1>Alllots</td><td rowspan=1 colspan=1>0.14, 0.01</td><td rowspan=1 colspan=1>0.99, 0.00</td><td rowspan=1 colspan=1>0.50, 0.00</td><td rowspan=1 colspan=1>0.67, 0.05</td><td rowspan=1 colspan=1>0.07, 0.01</td><td rowspan=1 colspan=1>0.00, 0.00</td><td rowspan=1 colspan=1>0.10</td></tr></table>

\* Maximum difference between linear regression and the better fitting higher order model

# Figure 1: Linearity Across Both Panel Types and All Kit Lots Combined

![](images/df8667c800ee499352d04d625c4316dd6f70b107479a6941fbdd5b7dd6c951f5.jpg)

The linear range of cobas BKV, defined as the concentration range for which the deviation of predicted log10 titer of the better fitting regression (2nd or 3rd order) and the predicted $\log _ { 1 0 }$ titer of the linear regression (1st order) is within $\pm 0 . 3$ log10 was determined as:

Observed linear range: $1 . 0 1 \mathrm { E } { + 0 1 }$ to $1 . 9 7 \mathrm { E } { + 0 8 } \ \mathrm { I U } / \mathrm { m L }$

Claimed linear range: $2 1 . 5 \ : \mathrm { I U / m L }$ to $1 . 0 \mathrm { E + } 0 8 \mathrm { \ : I U / m L }$

Verification of Linearity of cobas BKV with other genotypes:

Linearity was assessed for five BKV subgroup/subtype panels, one for each genotype, spanning the linear range from the ULoQ of $1 . 0 0 \mathrm { E + } 0 8 \mathrm { \Pi { U } / m L }$ to LLoQ of $2 . 0 0 \mathrm { E + 0 1 ~ I U / m L }$ .

Intermediate Stock Solutions (ISS) for each genotype were verified using either the QIAGEN artus BK virus QS-RGQ PCR kit (subgroup Ia and phagemids of subtypes II, III and IV) or by digital droplet PCR (subgroup Ic).

Ten-fold serial dilutions from $1 . 0 0 \mathrm { E + } 0 8 \mathrm { I U / m L }$ to $2 . 0 0 \mathrm { E + } 0 1 \ \mathrm { I U / m L }$ (nominal titer), of cell culture supernatant of BKV subgroup Ia, phagemids of BKV subgroup Ic, and subtypes II, III, and IV were prepared in BKV-negative EDTA-plasma. Four replicates were tested per concentration level. Subgroup Ia was tested using $3 \mathrm { k i t }$ lots and 2 cobas 6800/8800 Systems. Subgroup Ic, and subtypes II, III and IV, were tested using three kit lots and one cobas 6800/8800 System. Twelve runs were performed of which all 12 were valid resulting in 480 data points.

Linear ( $1 ^ { \mathrm { s t } }$ order) and nonlinear, $2 ^ { \mathrm { n d } }$ order and $3 ^ { \mathrm { r d } }$ order, regression fits were performed for all genotypes tested for all lots combined. The Root Mean Square Error (MSE) of the $3 ^ { \mathrm { r d } }$ order was smallest and was chosen as the best fitting regression model to calculate the absolute difference between the predicted log titer obtained from the $1 ^ { \mathrm { s t } }$ and $3 ^ { \mathrm { r d } }$ order regression fits. Both higher order coefficients (b2 and b3) were significant at $5 \%$ level.

BKV subgroups Ia, Ic, and subtypes II, III, and IV were detectable within the linear range of $1 . 0 0 \mathrm { E + } 0 8 \mathrm { I U / m L }$ to $2 . 0 0 \mathrm { E } { = } 0 1$ IU/mL that was established for the predominant subgroup Ib (Table 6). For all genotypes tested, the absolute deviation was $\leq 0 . 2 1 \log _ { 1 0 } \mathrm { I U / m L }$ at all concentration levels.

Table 6: Equations for the Best Fitting Regression Models for BKV Genotypes   

<table><tr><td rowspan=1 colspan=1>Subgroup/Subtype</td><td rowspan=1 colspan=1>Linear regression</td><td rowspan=1 colspan=1>Better fitting higher order modelregression</td><td rowspan=1 colspan=1>Maximumdifference*(Log10 IU/mL)</td></tr><tr><td rowspan=1 colspan=1>Ia</td><td rowspan=1 colspan=1>y = 0.9795 x + 0.2793</td><td rowspan=1 colspan=1>y = -0.0120 x3 + 0.1792 x2 + 0.2015 x +1.1614</td><td rowspan=1 colspan=1>0.21</td></tr><tr><td rowspan=1 colspan=1>Ic</td><td rowspan=1 colspan=1>y = 0.9820 x + 0.1366</td><td rowspan=1 colspan=1>y = -0.0024 x3 + 0.0403 x2 + 0.7853 x +0.3881</td><td rowspan=1 colspan=1>0.06</td></tr><tr><td rowspan=1 colspan=1>Ⅱ</td><td rowspan=1 colspan=1>y = 0.9857 x + 0.1313</td><td rowspan=1 colspan=1>y = -0.0063 x3 + 0.0967 x2 + 0.5548 x +0.6352</td><td rowspan=1 colspan=1>0.12</td></tr><tr><td rowspan=1 colspan=1>III</td><td rowspan=1 colspan=1>y = 0.9742 x + 0.1748</td><td rowspan=1 colspan=1>y = -0.0039 x3 + 0.0693 x2 + 0.6211 x +0.6415</td><td rowspan=1 colspan=1>0.12</td></tr><tr><td rowspan=1 colspan=1>IV</td><td rowspan=1 colspan=1>y = 0.9803x + 0.1453</td><td rowspan=1 colspan=1>y = -0.0054 x3 + 0.0880 x2 + 0.5657 x +0.6484</td><td rowspan=1 colspan=1>0.14</td></tr></table>

\* Maximum difference between linear regression and the better fitting higher order model

# Lower Limit of Quantitation (LLoQ):

The analysis for Lower Limit of Quantitation was performed with data obtained from the LOD study using the WHO standard. The LLoQ is the lowest titer within the linear range with a hit rate of at least $9 5 \%$ which meets the acceptance criterion for the Total Analytical Error (TAE) and the “Difference between Measurements in SD” approach. The acceptance criteria for TAE is

1. The TAE, when calculated as $| \mathrm { B i a s } | + 2 \mathrm { S D }$ , is $\leq 1 . 0 \log _ { 1 0 } \mathrm { I U / m L }$ , and 2. The TAE has to be such that the standard deviation for the difference between two measurements calculated as SQRT(2) $\textbf { x } 2 \textbf { x }$ SD is $\leq 1 . 0 \log _ { 1 0 } \mathrm { I U / m L }$

Table 7 shows the LLoQ calculated for each kit lot and combined across all the kit lots to be 19 $\mathrm { I U / m L }$ . However, as the LLoQ of the assay cannot be below the claimed LoD, the LLoQ was set at $2 1 . 5 \ : \mathrm { I U / m L }$ $( \mathrm { L L o Q } = \mathrm { L o D } )$ .

Table 7: LLOQ - TAE and Difference between Measurements.   

<table><tr><td rowspan=1 colspan=1>Lot(#)</td><td rowspan=1 colspan=1>NominalTiter(IU/mL)</td><td rowspan=1 colspan=1>HitRate[%]</td><td rowspan=1 colspan=1>Assignedlog10 titer</td><td rowspan=1 colspan=1>MeanObservedtiter(log10IU/mL)</td><td rowspan=1 colspan=1>SD(log10IU/mL)</td><td rowspan=1 colspan=1>AbsoluteBias(log10IU/mL)</td><td rowspan=1 colspan=1>TAE(log10IU/mL)</td><td rowspan=1 colspan=1>DifferenceBetweenMeasurementin SD</td></tr><tr><td rowspan=3 colspan=1>1(ED3278)</td><td rowspan=1 colspan=1>80</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>1.90</td><td rowspan=1 colspan=1>1.89</td><td rowspan=1 colspan=1>0.26</td><td rowspan=1 colspan=1>0.01</td><td rowspan=1 colspan=1>0.52</td><td rowspan=1 colspan=1>0.73</td></tr><tr><td rowspan=1 colspan=1>38</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>1.58</td><td rowspan=1 colspan=1>1.62</td><td rowspan=1 colspan=1>0.25</td><td rowspan=1 colspan=1>0.04</td><td rowspan=1 colspan=1>0.53</td><td rowspan=1 colspan=1>0.69</td></tr><tr><td rowspan=1 colspan=1>19</td><td rowspan=1 colspan=1>95.2</td><td rowspan=1 colspan=1>1.28</td><td rowspan=1 colspan=1>1.39</td><td rowspan=1 colspan=1>0.25</td><td rowspan=1 colspan=1>0.11</td><td rowspan=1 colspan=1>0.61</td><td rowspan=1 colspan=1>0.71</td></tr><tr><td rowspan=3 colspan=1>2(ED3280)</td><td rowspan=1 colspan=1>80</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>1.90</td><td rowspan=1 colspan=1>2.02</td><td rowspan=1 colspan=1>0.27</td><td rowspan=1 colspan=1>0.11</td><td rowspan=1 colspan=1>0.65</td><td rowspan=1 colspan=1>0.76</td></tr><tr><td rowspan=1 colspan=1>38</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>1.58</td><td rowspan=1 colspan=1>1.76</td><td rowspan=1 colspan=1>0.21</td><td rowspan=1 colspan=1>0.18</td><td rowspan=1 colspan=1>0.60</td><td rowspan=1 colspan=1>0.59</td></tr><tr><td rowspan=1 colspan=1>19</td><td rowspan=1 colspan=1>96.8</td><td rowspan=1 colspan=1>1.28</td><td rowspan=1 colspan=1>1.50</td><td rowspan=1 colspan=1>0.26</td><td rowspan=1 colspan=1>0.22</td><td rowspan=1 colspan=1>0.74</td><td rowspan=1 colspan=1>0.74</td></tr><tr><td rowspan=3 colspan=1>3(ED3284)</td><td rowspan=1 colspan=1>80</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>1.90</td><td rowspan=1 colspan=1>1.91</td><td rowspan=1 colspan=1>0.19</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.38</td><td rowspan=1 colspan=1>0.53</td></tr><tr><td rowspan=1 colspan=1>38</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>1.58</td><td rowspan=1 colspan=1>1.66</td><td rowspan=1 colspan=1>0.26</td><td rowspan=1 colspan=1>0.08</td><td rowspan=1 colspan=1>0.59</td><td rowspan=1 colspan=1>0.72</td></tr><tr><td rowspan=1 colspan=1>19</td><td rowspan=1 colspan=1>95.2</td><td rowspan=1 colspan=1>1.28</td><td rowspan=1 colspan=1>1.47</td><td rowspan=1 colspan=1>0.27</td><td rowspan=1 colspan=1>0.19</td><td rowspan=1 colspan=1>0.72</td><td rowspan=1 colspan=1>0.75</td></tr><tr><td rowspan=3 colspan=1>Acrosslots</td><td rowspan=1 colspan=1>80</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>1.90</td><td rowspan=1 colspan=1>1.94</td><td rowspan=1 colspan=1>0.24</td><td rowspan=1 colspan=1>0.04</td><td rowspan=1 colspan=1>0.52</td><td rowspan=1 colspan=1>0.68</td></tr><tr><td rowspan=1 colspan=1>38</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>1.58</td><td rowspan=1 colspan=1>1.68</td><td rowspan=1 colspan=1>0.24</td><td rowspan=1 colspan=1>0.10</td><td rowspan=1 colspan=1>0.57</td><td rowspan=1 colspan=1>0.67</td></tr><tr><td rowspan=1 colspan=1>19</td><td rowspan=1 colspan=1>95.8</td><td rowspan=1 colspan=1>1.28</td><td rowspan=1 colspan=1>1.45</td><td rowspan=1 colspan=1>0.26</td><td rowspan=1 colspan=1>0.18</td><td rowspan=1 colspan=1>0.69</td><td rowspan=1 colspan=1>0.73</td></tr></table>

# 1. Analytical Specificity/Interference:

Cross reactivity:

For potential cross reactants 35 microorganisms, including 17 viral isolates, 15 bacterial strains and three fungal isolates were used and divided into 7 pools with 4 to 5 microorganisms per cross reactant pool and, HCV as single interferent. Interferent pools not meeting the acceptance criteria of the mean difference in $\log _ { 1 0 }$ Titer (Interferent – Control) of $\pm 0 . 5 \log _ { 1 0 } { }$ were resolved into individual components and retested. Potential cross reactants in BKV negative EDTA plasma were tested in the absence and presence of BKV DNA at a concentration of $1 0 0 \ : \mathrm { I U / m L }$ (approximately ${ 5 } \mathbf { x }$ LLoQ). Potential cross reactants were tested at $1 \mathrm { x } 1 0 ^ { 6 }$ copies $\mathrm { ' m L }$ , $\mathrm { I U / m L }$ , $\mathrm { T C I D 5 0 / m L }$ , cells/mL, $\mathrm { C F U / m L }$ , IFU/mL or $\mathrm { C C U / m L }$ for each organism except for HPV which was tested at $1 \mathrm { x } 1 0 ^ { 5 }$ . HCV was tested individually (i.e., not pooled with other organisms) at $3 \mathrm { x } 1 0 ^ { 5 } \mathrm { U } / \mathrm { m L }$ . Testing was performed using 6 replicates per cross reactive pool or individual interferents HCV, 12 replicates of the BKV positive and negative controls using three kit lots and one cobas 8800 system.

For BKV negative samples the negativity rate was determined. For BKV positive samples the positivity rate was determined together with the correct quantitation of BKV DNA by computing the Mean concentration detected across the replicates, the SD and the difference between the control condition (no cross reactant) and the test condition containing the potential cross reactant organism. Results are shown in Table 8 below for testing of interferents.

Table 8: Testing Results of Cross reactivity in Pools of Microorganisms Spiked with BKV DNA   

<table><tr><td colspan="7" rowspan="1">DKV DNA</td></tr><tr><td colspan="1" rowspan="2">Pool</td><td colspan="1" rowspan="2">Organisms</td><td colspan="1" rowspan="1">No BKV</td><td colspan="4" rowspan="1">With BKV</td></tr><tr><td colspan="1" rowspan="1">NegativityRate</td><td colspan="1" rowspan="1">PositivityRate</td><td colspan="1" rowspan="1">Mean[Log 10]</td><td colspan="1" rowspan="1">SD[Log 10]</td><td colspan="1" rowspan="1">MeanDifferencein log10Titer(Interferent Control)</td></tr><tr><td colspan="1" rowspan="5">1</td><td colspan="1" rowspan="1">HSV 1</td><td colspan="1" rowspan="5">6/6100%</td><td colspan="1" rowspan="5">6/6100%</td><td colspan="1" rowspan="5">2.23</td><td colspan="1" rowspan="5">0.40</td><td colspan="1" rowspan="5">0.19</td></tr><tr><td colspan="1" rowspan="1">HSV 2</td></tr><tr><td colspan="1" rowspan="1">HSV 6</td></tr><tr><td colspan="1" rowspan="1">HSV 7</td></tr><tr><td colspan="1" rowspan="1">HSV 8</td></tr><tr><td colspan="1" rowspan="5">2</td><td colspan="1" rowspan="1">Adenovirus Type 5</td><td colspan="1" rowspan="5">6/6100%</td><td colspan="1" rowspan="5">6/6100%</td><td colspan="1" rowspan="5">2.08</td><td colspan="1" rowspan="5">0.15</td><td colspan="1" rowspan="5">0.04</td></tr><tr><td colspan="1" rowspan="1">Candida albicans</td></tr><tr><td colspan="1" rowspan="1">Chlamydia trachomatis</td></tr><tr><td colspan="1" rowspan="1">Clostridium perfringens</td></tr><tr><td colspan="1" rowspan="1">CMV</td></tr><tr><td colspan="1" rowspan="5">3</td><td colspan="1" rowspan="1">Enterococcus faecalis</td><td colspan="1" rowspan="5">6/6100%</td><td colspan="1" rowspan="5">6/6100%</td><td colspan="1" rowspan="5">2.08</td><td colspan="1" rowspan="5">0.21</td><td colspan="1" rowspan="5">0.04</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td></tr><tr><td colspan="1" rowspan="1">HBV</td></tr><tr><td colspan="1" rowspan="1">HIV-1</td></tr><tr><td colspan="1" rowspan="1">HIV-2</td></tr><tr><td colspan="1" rowspan="5">4</td><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="5">6/6100%</td><td colspan="1" rowspan="5">6/6100%</td><td colspan="1" rowspan="5">1.95</td><td colspan="1" rowspan="5">0.30</td><td colspan="1" rowspan="5">-0.10</td></tr><tr><td colspan="1" rowspan="1">Listeria monocytogenes</td></tr><tr><td colspan="1" rowspan="1">Mycobacterium avium</td></tr><tr><td colspan="1" rowspan="1">Mycoplasmapneumoniae</td></tr><tr><td colspan="1" rowspan="1">Neisseria gonorrhoeae</td></tr><tr><td colspan="1" rowspan="5">5</td><td colspan="1" rowspan="1">Parvovirus B19</td><td colspan="1" rowspan="5">6/6100%</td><td colspan="1" rowspan="5">6/6100%</td><td colspan="1" rowspan="5">2.05</td><td colspan="1" rowspan="5">0.15</td><td colspan="1" rowspan="5">0.01</td></tr><tr><td colspan="1" rowspan="1">Propionibacteriumacnes</td></tr><tr><td colspan="1" rowspan="1">Salmonella enterica</td></tr><tr><td colspan="1" rowspan="1">Simian Virus 40</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td></tr><tr><td colspan="1" rowspan="5">6</td><td colspan="1" rowspan="1">Staphylococcusepidermis</td><td colspan="1" rowspan="5">6/6100%</td><td colspan="1" rowspan="5">6/6100%</td><td colspan="1" rowspan="5">1.95</td><td colspan="1" rowspan="5">0.17</td><td colspan="1" rowspan="5">-0.09</td></tr><tr><td colspan="1" rowspan="1">Streptococcus pyogenes</td></tr><tr><td colspan="1" rowspan="1">Streptococcuspneumoniae</td></tr><tr><td colspan="1" rowspan="1">VZV</td></tr><tr><td colspan="1" rowspan="1">Aspergillus niger</td></tr><tr><td colspan="1" rowspan="2">Pool</td><td colspan="1" rowspan="2">Organisms</td><td colspan="1" rowspan="1">No BKV</td><td colspan="4" rowspan="1">With BKV</td></tr><tr><td colspan="1" rowspan="1">NegativityRate</td><td colspan="1" rowspan="1">PositivityRate</td><td colspan="1" rowspan="1">Mean[Log 10]</td><td colspan="1" rowspan="1">SD[Log 10]</td><td colspan="1" rowspan="1">MeanDifferencein log10Titer(InterferentControl</td></tr><tr><td colspan="1" rowspan="4">7</td><td colspan="1" rowspan="1">Cryptococcusneoformans</td><td colspan="1" rowspan="1">6/6100%</td><td colspan="1" rowspan="1">6/6100%</td><td colspan="1" rowspan="1">2.23</td><td colspan="1" rowspan="1">0.55</td><td colspan="1" rowspan="1">0.18</td></tr><tr><td colspan="1" rowspan="1">Human PapillomaVirus (HPV)</td><td colspan="1" rowspan="1">6/6100%</td><td colspan="1" rowspan="1">6/6100%</td><td colspan="1" rowspan="1">2.28</td><td colspan="1" rowspan="1">0.22</td><td colspan="1" rowspan="1">0.37</td></tr><tr><td colspan="1" rowspan="1">JC Virus</td><td colspan="1" rowspan="1">6/6100%</td><td colspan="1" rowspan="1">6/6100%</td><td colspan="1" rowspan="1">1.87</td><td colspan="1" rowspan="1">0.09</td><td colspan="1" rowspan="1">-0.18</td></tr><tr><td colspan="1" rowspan="1">EB Virus</td><td colspan="1" rowspan="1">6/6100%</td><td colspan="1" rowspan="1">6/6100%</td><td colspan="1" rowspan="1">1.94</td><td colspan="1" rowspan="1">0.10</td><td colspan="1" rowspan="1">-0.11</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">HCV Hepatitis C Virus(HCV)</td><td colspan="1" rowspan="1">6/6100%</td><td colspan="1" rowspan="1">6/6100%</td><td colspan="1" rowspan="1">2.19</td><td colspan="1" rowspan="1">0.30</td><td colspan="1" rowspan="1">0.15</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Control (BKVnegative)</td><td colspan="1" rowspan="1">6/6100%</td><td colspan="1" rowspan="1">0/60%</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td></tr><tr><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1">Control (BKV positive) for Pools 1-6 andHCV</td><td colspan="1" rowspan="1">0/120%</td><td colspan="1" rowspan="1">12/12100%</td><td colspan="1" rowspan="1">2.04</td><td colspan="1" rowspan="1">0.27</td><td colspan="1" rowspan="1">N/A</td></tr></table>

The mean $\log _ { 1 0 }$ titer of the positive BKV samples containing potentially cross-reacting organisms were within $\pm 0 . 5 \log _ { 1 0 }$ of the mean $\log _ { 1 0 }$ titer of the respective positive spike control.

# Endogenous Interference:

The effect of potentially interfering endogenous substances on the sensitivity/quantitation of cobas BKV was determined by testing 20 individual clinical BKV-seronegative samples spiked with selected endogenous substances and tested in the presence of BKV DNA at a concentration of $1 0 0 \ : \mathrm { I U / m L }$ (approximately 5x LLoQ). The negative sample spiked solely with BKV target was used as a Positive Spike Control (PSC). To analyze specificity, the same 20 individual clinical negative samples were individually spiked with potentially interfering endogenous substances and tested in the absence of BKV target DNA. The unspiked samples were used as Negative Spiked Controls (NSC). Interferent concentrations were used as recommended by the CLSI guideline EP7-A2. Human DNA levels were tested at $2 \mathrm { m g / m L }$ . Control conditions were tested with one replicate per specimen, and test conditions were tested with 3 replicates per specimen. Results are summarized in Table 9.

Table 9: Endogenous Interference   

<table><tr><td rowspan=2 colspan=1>Interferent</td><td rowspan=2 colspan=1>C1</td><td rowspan=1 colspan=1>No BKV</td><td rowspan=1 colspan=5>With BKV [100 IU/mL]</td></tr><tr><td rowspan=1 colspan=1>NegativityRate</td><td rowspan=1 colspan=1>PositivityRate</td><td rowspan=1 colspan=1>MeanCt</td><td rowspan=1 colspan=1>Meanlog10titer</td><td rowspan=1 colspan=1>SD[Log1o]</td><td rowspan=1 colspan=1>Meandifferencein log1o titer</td></tr><tr><td rowspan=1 colspan=1>Control</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>20/20100%</td><td rowspan=1 colspan=1>60/60100%</td><td rowspan=1 colspan=1>35.72</td><td rowspan=1 colspan=1>2.10</td><td rowspan=1 colspan=1>0.33</td><td rowspan=1 colspan=1>0.00</td></tr><tr><td rowspan=1 colspan=1>NaOH2</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>20/20100%</td><td rowspan=1 colspan=1>60/60100%</td><td rowspan=1 colspan=1>35.60</td><td rowspan=1 colspan=1>2.08</td><td rowspan=1 colspan=1>0.24</td><td rowspan=1 colspan=1>-0.03</td></tr><tr><td rowspan=1 colspan=1>Albumin</td><td rowspan=1 colspan=1>60 g/L</td><td rowspan=1 colspan=1>20/20100%</td><td rowspan=1 colspan=1>59/59*100%</td><td rowspan=1 colspan=1>35.88</td><td rowspan=1 colspan=1>1.99</td><td rowspan=1 colspan=1>0.27</td><td rowspan=1 colspan=1>-0.11</td></tr><tr><td rowspan=1 colspan=1>Bilirubin(conj.)</td><td rowspan=1 colspan=1>0.2 g/L 3</td><td rowspan=1 colspan=1>20/20100%</td><td rowspan=1 colspan=1>60/60100%</td><td rowspan=1 colspan=1>35.76</td><td rowspan=1 colspan=1>2.07</td><td rowspan=1 colspan=1>0.33</td><td rowspan=1 colspan=1>-0.03</td></tr><tr><td rowspan=1 colspan=1>Bilirubin(unconj.)</td><td rowspan=1 colspan=1>0.2 g/L 3</td><td rowspan=1 colspan=1>20/20100%</td><td rowspan=1 colspan=1>60/60100%</td><td rowspan=1 colspan=1>35.53</td><td rowspan=1 colspan=1>2.11</td><td rowspan=1 colspan=1>0.24</td><td rowspan=1 colspan=1>0.01</td></tr><tr><td rowspan=1 colspan=1>Human DNA</td><td rowspan=1 colspan=1>2 mg/L</td><td rowspan=1 colspan=1>20/20100%</td><td rowspan=1 colspan=1>60/60100%</td><td rowspan=1 colspan=1>35.26</td><td rowspan=1 colspan=1>2.18</td><td rowspan=1 colspan=1>0.17</td><td rowspan=1 colspan=1>0.08</td></tr><tr><td rowspan=1 colspan=1>Hemoglobin</td><td rowspan=1 colspan=1>2 g/L</td><td rowspan=1 colspan=1>20/20100%</td><td rowspan=1 colspan=1>60/60100%</td><td rowspan=1 colspan=1>35.60</td><td rowspan=1 colspan=1>2.12</td><td rowspan=1 colspan=1>0.28</td><td rowspan=1 colspan=1>0.01</td></tr><tr><td rowspan=1 colspan=1>Triglycerides</td><td rowspan=1 colspan=1>37 mmol/L</td><td rowspan=1 colspan=1>20/20100%</td><td rowspan=1 colspan=1>60/60100%</td><td rowspan=1 colspan=1>35.71</td><td rowspan=1 colspan=1>2.08</td><td rowspan=1 colspan=1>0.23</td><td rowspan=1 colspan=1>-0.02</td></tr></table>

${ } ^ { 1 } C =$ Test Concentration; 2 solvent control; $^ { 3 } 0 . 2 ~ \mathrm { g / L } = 3 4 2 ~ \mathrm { \mu m o l / L }$ \* Titer obtained was lower than LLoQ and was excluded

All BKV-negative samples with endogenous interferents produced valid negative results (target not detected).

For BKV-positive samples with endogenous interferents the mean $\log _ { 1 0 }$ titer of each of the positive BKV samples containing endogenous interferents was within $\mathrm { \Omega } _ { \mathrm { l } } \pm 0 . 5 \log _ { 1 0 } \mathrm { g } _ { 1 0 }$ of the mean log10 titer of the spike control.

# Exogenous Interference

The effect of potentially interfering exogenous substances on the sensitivity/quantitation of cobas BKV was determined by testing 10 individual BKV-negative donor samples spiked with pools of 24 commercially available drugs at three times the plasma peak level per CLSI EP7-A2. The same samples were also tested in the presence of BKV target at $1 0 0 \ : \mathrm { I U / m L }$ (approximately 5x LLoQ). The negative sample spiked solely with BKV target was used as a Positive Spike Control (PSC). The un-spiked samples were used as Negative Spiked Controls (NSC). Conditions were tested with 3 replicates per specimen. The following drugs were tested, and the results are summarized in Table 10 below.

Pool 1: Azathioprine, Sulfamethoxzole, Trimethoprim, Cefotan, Cidofovir Pool 2: Foscarnet, Piperacillin, Tazobactam, Prednisone, Vancomycin • Pool 3: Cyclosporine, Everolimus, Fluconazole, Ganciclovir • Pool 4: Mycophenolate mofetil, Mycophenolic acid, Valganciclovir • Pool 5: Sirolimus, Tacrolimus C Pool 6: Letermovir, Micafungin, Acyclovir Clavulanate potassium • Pool 7: Ticarcillin disodium

Table 10: Exogenous Interference   

<table><tr><td rowspan=2 colspan=1>Pool</td><td rowspan=1 colspan=1>No BKV</td><td rowspan=1 colspan=5>With BKV [100 IU/mL]</td></tr><tr><td rowspan=1 colspan=1>NegativityRate</td><td rowspan=1 colspan=1>PositivityRate</td><td rowspan=1 colspan=1>MeanCt</td><td rowspan=1 colspan=1>Meanlog1o titer</td><td rowspan=1 colspan=1>SD[Log10]</td><td rowspan=1 colspan=1>MeanDifferencein log10Titer</td></tr><tr><td rowspan=1 colspan=1>Pool 11</td><td rowspan=1 colspan=1>10/10100%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>35.98</td><td rowspan=1 colspan=1>2.06</td><td rowspan=1 colspan=1>0.22</td><td rowspan=1 colspan=1>0.04</td></tr><tr><td rowspan=1 colspan=1>Pool 21</td><td rowspan=1 colspan=1>10/10100%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>35.82</td><td rowspan=1 colspan=1>2.05</td><td rowspan=1 colspan=1>0.25</td><td rowspan=1 colspan=1>0.04</td></tr><tr><td rowspan=1 colspan=1>Pool 32</td><td rowspan=1 colspan=1>10/10100%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>35.94</td><td rowspan=1 colspan=1>2.10</td><td rowspan=1 colspan=1>0.28</td><td rowspan=1 colspan=1>-0.01</td></tr><tr><td rowspan=1 colspan=1>Pool 42</td><td rowspan=1 colspan=1>10/10100%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>35.80</td><td rowspan=1 colspan=1>2.10</td><td rowspan=1 colspan=1>0.20</td><td rowspan=1 colspan=1>-0.01</td></tr><tr><td rowspan=1 colspan=1>Pool 53</td><td rowspan=1 colspan=1>10/10100%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>35.88</td><td rowspan=1 colspan=1>2.10</td><td rowspan=1 colspan=1>0.28</td><td rowspan=1 colspan=1>-0.01</td></tr><tr><td rowspan=1 colspan=1>Pool 61</td><td rowspan=1 colspan=1>10/10100%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>35.99</td><td rowspan=1 colspan=1>2.01</td><td rowspan=1 colspan=1>0.21</td><td rowspan=1 colspan=1>0.08</td></tr><tr><td rowspan=1 colspan=1>Pool 71</td><td rowspan=1 colspan=1>10/10100%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>36.18</td><td rowspan=1 colspan=1>1.93</td><td rowspan=1 colspan=1>0.25</td><td rowspan=1 colspan=1>0.17</td></tr><tr><td rowspan=1 colspan=1>PBS SC</td><td rowspan=1 colspan=1>10/10100%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>35.91</td><td rowspan=1 colspan=1>2.01</td><td rowspan=1 colspan=1>0.22</td><td rowspan=1 colspan=1>0.08</td></tr><tr><td rowspan=1 colspan=1>DMSO SC</td><td rowspan=1 colspan=1>10/10100%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>35.89</td><td rowspan=1 colspan=1>2.06</td><td rowspan=1 colspan=1>0.19</td><td rowspan=1 colspan=1>0.03</td></tr><tr><td rowspan=1 colspan=1>Ethanol SC</td><td rowspan=1 colspan=1>10/10100%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>36.13</td><td rowspan=1 colspan=1>2.02</td><td rowspan=1 colspan=1>0.23</td><td rowspan=1 colspan=1>0.07</td></tr><tr><td rowspan=1 colspan=1>NegativeControl</td><td rowspan=1 colspan=1>10/10100%</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td></tr><tr><td rowspan=1 colspan=1>PositiveControl</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>35.95</td><td rowspan=1 colspan=1>2.09</td><td rowspan=1 colspan=1>0.20</td><td rowspan=1 colspan=1>N/A</td></tr></table>

The superscripts in Pools 1-7 indicate the solvent that was used for constituting the interferents (i.e., $1 = \mathrm { P B S }$ ; 2 ${ \bf \Lambda } = { \bf D M S O }$ ; and $3 =$ Ethanol); $\mathbf { S C } =$ solvent control.

# 2. Assay Reportable Range:

Based on the linearity study and the LoD study, the reportable range claimed by the sponsor for the cobas BKV assay is $2 1 . 5 \ : \mathrm { I U / m L }$ to $1 . 0 0 \mathrm { E + } 0 8 \mathrm { I U / m L }$ .

3. Traceability, Stability, Expected Values (Controls, Calibrators, or Methods):

Traceability:

The cobas BKV assay is standardized against the 1st WHO International Standard for BK Virus DNA (NIBSC 14/212, predominantly genotype Ia) was used for developing the cobas BKV assay. The titer of the RMS BKV secondary standard, was established by diluting BKV cell culture supernatant (genotype 1a) in pooled BKV negative EDTA human plasma. The titer of the RMS BKV secondary standard was determined to be $4 . 2 7 \log 1 0 \mathrm { I U / m L }$ using the calibrator bracketing method.

Traceability of the calibration panel and RMS BKV secondary standard to the WHO standard was verified as shown in Figure 2. Bivariate fit of observed BKV DNA concentration vs

expected BKV DNA concentration was performed using testing data of 3 dilutions of the 1st WHO International Standard (panel members (PMs) ranging from $1 . 2 8 { - } 4 . 3 0 \log _ { 1 0 } \mathrm { T U / m L } )$ , the RMS BKV Secondary Standard (1PM; $4 . 2 7 \log _ { 1 0 } \mathrm { { I U / m L } }$ ) and the RMS BKV Calibration Panel (6 PMs; ranging from $3 . 0 0 { - } 8 . 7 0 \log _ { 1 0 } \mathrm { { I U / m L } }$ ).

![](images/6374e50e22741c756ad84aecb03f29144319aa9d4dfa29229250ffd5b769a025.jpg)  
Figure 2: Traceability of the calibration panel and RMS BKV secondary standard to the WHO standard

Quantitation values for the panel members are similar to the expected values with deviation o $\mathrm { f } \leq 0 . 1 9 \log _ { 1 0 } \mathrm { { I U } / m L }$ . Maximum deviation was observed at $1 9 . 0 \ \mathrm { I U / m L }$ (1.28 log10 IU/mL) close to the assay LLoQ value of $2 1 . 5 \ \mathrm { I U / m L }$ .

The following linear equations were obtained:

BKV Calibration Panel: $\mathbf { y } = 1 . 0 0 1 \mathrm { x } \cdot 0 . 0 1 1$ ; $\mathrm { R } ^ { 2 } = 0 . 9 9 9$   
BKV WHO Standard: $\mathbf { y } = 0 . 9 5 1 \ \mathbf { x } + 0 . 2 0 8$ ; $\mathrm { R } ^ { 2 } = 0 . 9 7 3$

Based on these results, the calibration and standardization process of cobas BKV provides quantitation values for the cobas BKV calibration panel and the RMS BKV Secondary Standard provide traceability to the 1st WHO international Standard for BKV.

Stability:

The clinical specimen stability of BKV viral target in whole blood specimens collected in EDTA-plasma preparation tubes and/or plasma samples after various storage conditions with or without freeze-thawing cycles was evaluated using cobas BKV for use on the cobas 6800/8800 Systems.

Freshly drawn whole blood from 12 unique individual BKV-negative donors collected in EDTA-plasma tubes (6 donor samples collected in PPT and 6 in lavender top tubes) were spiked with BKV to a concentration of $1 0 0 \ : \mathrm { I U / m L }$ (approximately 5x LLoQ). The reference time point (T0) was processed directly after spiking of the target by separating the blood into EDTA-plasma; all other samples were stored at indicated conditions until further processing.

All samples used for this study in each PPT/lavender top tube type were tested unspiked at time point 0 (T0).

All BKV-positive samples tested generated positive results for BKV and the mean $\log _ { 1 0 }$ titers including the two-sided 95 confidence interval of each of the tested time points/conditions and tube types was within $\pm 0 . 5 \log _ { 1 0 } \mathrm { { g } } _ { 1 0 }$ of the mean log10 titer of the respective reference condition ( $\mathrm { T 0 } =$ reference).

The results support the following storage conditions for whole blood collected in BD Vacutainer PPT or BD Vacutainer EDTA tubes (lavender top) and the respective separated plasma:

Whole blood collected in EDTA-plasma tubes (lavender top and PPT) may be stored or   
transported for up to 24 hours at $2 \mathrm { ^ { \circ } C }$ to $2 5 ^ { \circ } \mathrm { C }$ before further processing and plasma   
separation.   
Afterwards, whole blood samples should be centrifuged, and the resulting plasma   
samples are additionally stable for: o 24 hours at $2 \mathrm { ^ { \circ } C }$ to $3 0 \mathrm { { } ^ { \circ } C }$ in primary or secondary tubes, followed by o up to 6 days at $2 \mathrm { ^ { \circ } C }$ to $8 ^ { \circ } \mathrm { C }$ in primary or secondary tubes, or o up to 6 months at $- 1 5 ^ { \circ } \mathrm { C }$ to $- 8 0 ^ { \circ } \mathrm { C }$ in secondary tubes.

Plasma samples are stable in secondary tubes for up to four freeze/thaw cycles when frozen between $- 1 5 ^ { \circ } \mathrm { C }$ to $- 8 0 ^ { \circ } \mathrm { C }$ .

# Open kit and On-board Stability:

On-Board Stability and Open Kit Stability of the test-specific cassette was evaluated by testing five time points over the course of 91 days using one kit lot and four 6800/8800 Systems. In order to support the On-Board and Open Kit Stability, as well as the 40 times usage, the $1 9 2 \mathrm { T }$ cassette was incubated for a total of 41 hours at $3 7 ^ { \circ } \mathrm { C }$ .

The data from this study supports the claim that, once opened, the cobas $\textsuperscript { \textregistered }$ BKV 192T testspecific reagent cassettes are stable for up to 90 days at $2 - 8 ^ { \circ } \mathrm { C }$ (Open Kit Stability) and remain stable for up to 40 hours at $3 7 ^ { \circ } \mathrm { C }$ (On-Board Stability). Furthermore, $1 9 2 \mathrm { T }$ testspecific reagent cassettes can be used up to 40 times.

# Reagent Stability

Three lots of the cobas BKV were tested to demonstrate stability of the test-specific reagents of cobas BKV and cobas EBV/BKV Control Kit when stored at stressed temperature conditions (accelerated stability) and at the targeted storage temperature of $2 \mathrm { ^ { \circ } C }$ to $8 ^ { \circ } \mathrm { C }$ (realtime stability).

Accelerated Stability: The kit components BKV MMX-R2 192T, EBV/BKV $\mathrm { H } ( + ) \mathrm { C }$ , and the EBV/BKV $\mathrm { L } ( + ) \mathrm { C }$ were stored at $3 7 ^ { \circ } \mathrm { C } \pm 2 ^ { \circ } \mathrm { C }$ for up to 75 days and tested at various time points. All lots met the acceptance criteria at all testing time points (26d, 41d, 51d, 63d, and 75d). The 75 days accelerated stability results predict a 19-month real-time stability for the test-specific reagents of cobas BKV and cobas EBV/BKV Control Kit.

Real-time stability: The kit components BKV MMX-R2 192T, EBV/BKV $\mathrm { H } ( + ) \mathrm { C }$ , and the EBV/BKV $\mathrm { L } ( + ) \mathrm { C }$ were stored at $2 { - } 8 ^ { \circ } \mathrm { C }$ for up to 13 months and tested at various time points. All lots met the acceptance criteria at all testing time points (3m, 6m, $9 \mathrm { m }$ , $1 2 \mathrm { m }$ and $1 3 \mathrm { m }$ ).

Shelf-life: Based on the combined data of accelerated and real-time stability studies the results support a shelf-life of 12 months when stored at $2 { - } 8 ^ { \circ } \mathrm { C }$ .

Expected values for Quality Controls:

To monitor the assay performance, reagent performance, and procedural errors, positive and negative external controls must be run in accordance with the guidelines or requirements of local, state, and/or federal regulations or accrediting organizations.

External Controls are provided separately from the cobas BKV test kit in the cobas EBV/Positive Control Kit and the cobas Buffer Negative Control Kit. The cobas EBV/BKV positive control consists of BKV phagemid DNA diluted into BKV negative human plasma at two concentrations to yield the EBV/BKV Low Positive Control (EBV/BKV $\mathrm { L } ( + ) \mathrm { C } )$ and the EBV/BKV High Positive Control (EBV/BKV $\mathrm { H } ( + ) \mathrm { C } )$ . The cobas Buffer negative control consists of Tris buffer with $< 0 . 0 0 2 \%$ Poly rA RNA. The cobas BKV Negative Control, the BKV Low Positive Control, and the BKV High Positive Control must be included in each run. The validity of the results for the controls as well as for the DNA-QS (internal control) is determined by the assay specific analysis software package used by the cobas 6800/8800 instrument. The amount of BKV DNA for BKV high and low positive controls must fall within their acceptable titer ranges.

The failure rates of the kit controls and samples were assessed by evaluating outcomes of the nine analytical studies (LoD, LoD verification for Genotype, Linearity, Linearity verification for Genotype, Precision, Cross contamination, Lot interchangeability, Accuracy, and LoD in Plasma vs Buffer). Results demonstrated an overall invalid rate of $0 . 0 1 \%$ for QS, $0 \%$ for RMC and a $0 . 2 0 \%$ sample failure rate.

4. Detection Limit:

Limit of Detection (LoD) using the 1st WHO International Standard for BKV

The LoD of the cobas BKV test for the 1st WHO BKV Standard (BKV Subgroup 1b) was determined by analysis of serial dilutions of the Standard diluted into a pooled EDTA-plasma derived from BKV negative individuals. Panels of six concentration levels plus a blank were tested with three lots of cobas BKV test reagents and four instruments with multiple runs and operators over a period of 3 days. Each dilution was determined in 63 replicates per lot and day $\mathrm { ( n { = } 1 8 9 }$ total replicates per day).

The results from testing the WHO BKV Standard in EDTA plasma as well as the calculated LoD values are shown in the table below. The LoD and LLoQ values were determined by Probit analysis and by $9 5 \%$ hit rate (Table 11).

Table 11: LoD and LLoQ Values Estimated for Three Lots   

<table><tr><td rowspan=1 colspan=1>Kit Lot</td><td rowspan=1 colspan=1>NominalConcentration(IU/mL)</td><td rowspan=1 colspan=1>Number ofPositiveReplicates</td><td rowspan=1 colspan=1>Number ofValidReplicates</td><td rowspan=1 colspan=1>Hit Rate[%]</td><td rowspan=1 colspan=1>LOD byProbit [95%CI</td></tr><tr><td rowspan=7 colspan=1>Lot 1</td><td rowspan=1 colspan=1>80.0</td><td rowspan=1 colspan=1>63</td><td rowspan=1 colspan=1>63</td><td rowspan=1 colspan=1>100</td><td rowspan=7 colspan=1>21.5 IU/mL[16.3  32.4IU/mL]</td></tr><tr><td rowspan=1 colspan=1>38.0</td><td rowspan=1 colspan=1>63</td><td rowspan=1 colspan=1>63</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>19.0</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>63</td><td rowspan=1 colspan=1>95.2</td></tr><tr><td rowspan=1 colspan=1>9.5</td><td rowspan=1 colspan=1>46</td><td rowspan=1 colspan=1>63</td><td rowspan=1 colspan=1>73.0</td></tr><tr><td rowspan=1 colspan=1>4.75</td><td rowspan=1 colspan=1>36</td><td rowspan=1 colspan=1>63</td><td rowspan=1 colspan=1>57.1</td></tr><tr><td rowspan=1 colspan=1>2.38</td><td rowspan=1 colspan=1>23</td><td rowspan=1 colspan=1>63</td><td rowspan=1 colspan=1>36.5</td></tr><tr><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>62</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=7 colspan=1>Lot 2</td><td rowspan=1 colspan=1>80.0</td><td rowspan=1 colspan=1>62</td><td rowspan=1 colspan=1>62</td><td rowspan=1 colspan=1>100</td><td rowspan=7 colspan=1>19.7 IU/mL[15.0  29.2IU/mL]</td></tr><tr><td rowspan=1 colspan=1>38.0</td><td rowspan=1 colspan=1>63</td><td rowspan=1 colspan=1>63</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>19.0</td><td rowspan=1 colspan=1>61</td><td rowspan=1 colspan=1>63</td><td rowspan=1 colspan=1>96.8</td></tr><tr><td rowspan=1 colspan=1>9.5</td><td rowspan=1 colspan=1>48</td><td rowspan=1 colspan=1>63</td><td rowspan=1 colspan=1>76.2</td></tr><tr><td rowspan=1 colspan=1>4.75</td><td rowspan=1 colspan=1>34</td><td rowspan=1 colspan=1>63</td><td rowspan=1 colspan=1>54.0</td></tr><tr><td rowspan=1 colspan=1>2.38</td><td rowspan=1 colspan=1>23</td><td rowspan=1 colspan=1>62</td><td rowspan=1 colspan=1>37.1</td></tr><tr><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>62</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=7 colspan=1>Lot 3</td><td rowspan=1 colspan=1>80.0</td><td rowspan=1 colspan=1>63</td><td rowspan=1 colspan=1>63</td><td rowspan=1 colspan=1>100</td><td rowspan=7 colspan=1>19.3 IU/mL[14.8  28.5IU/mL]</td></tr><tr><td rowspan=1 colspan=1>38.0</td><td rowspan=1 colspan=1>63</td><td rowspan=1 colspan=1>63</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>19.0</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>63</td><td rowspan=1 colspan=1>95.2</td></tr><tr><td rowspan=1 colspan=1>9.5</td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>63</td><td rowspan=1 colspan=1>79.4</td></tr><tr><td rowspan=1 colspan=1>4.75</td><td rowspan=1 colspan=1>35</td><td rowspan=1 colspan=1>63</td><td rowspan=1 colspan=1>55.6</td></tr><tr><td rowspan=1 colspan=1>2.38</td><td rowspan=1 colspan=1>22</td><td rowspan=1 colspan=1>63</td><td rowspan=1 colspan=1>35.0</td></tr><tr><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>63</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=7 colspan=1>All lotscombined</td><td rowspan=1 colspan=1>80.0</td><td rowspan=1 colspan=1>188</td><td rowspan=1 colspan=1>188</td><td rowspan=1 colspan=1>100</td><td rowspan=7 colspan=1>20.2 IU/mL[17.0  24.8IU/mL]</td></tr><tr><td rowspan=1 colspan=1>38.0</td><td rowspan=1 colspan=1>189</td><td rowspan=1 colspan=1>189</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>19.0</td><td rowspan=1 colspan=1>181</td><td rowspan=1 colspan=1>189</td><td rowspan=1 colspan=1>95.8</td></tr><tr><td rowspan=1 colspan=1>9.5</td><td rowspan=1 colspan=1>144</td><td rowspan=1 colspan=1>189</td><td rowspan=1 colspan=1>76.2</td></tr><tr><td rowspan=1 colspan=1>4.75</td><td rowspan=1 colspan=1>105</td><td rowspan=1 colspan=1>189</td><td rowspan=1 colspan=1>55.6</td></tr><tr><td rowspan=1 colspan=1>2.38</td><td rowspan=1 colspan=1>68</td><td rowspan=1 colspan=1>188</td><td rowspan=1 colspan=1>36.2</td></tr><tr><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>187</td><td rowspan=1 colspan=1>0</td></tr></table>

When determined by Probit analysis, the different lots have similar LoD for all tested lots, the highest LoD of $2 1 . 5 \ : \mathrm { I U / m L }$ was obtained with lot 1, which is only slightly higher than the LoD determined by $9 5 \%$ hit rate. The LoD by $9 5 \%$ hit rate was $1 9 . 0 \ : \mathrm { I U / m L }$ (181/189) and was identical for all tested lots.

The claimed LoD value is $2 1 . 5 \ : \mathrm { I U / m L }$ and this concentration was used in studies for confirmation of the LoD.

Verification of the Limit of Detection (LoD) with genotypes of subgroup Ia, Ic, and subtypes II, II, and IV.

The Limit of Detection $( 2 1 . 5 \mathrm { I U } / \mathrm { m L } )$ ) was verified for the cobas BKV test with BKV subgroup Ia, Ic, and subtypes II, III, and IV following the CLSI Guideline EP17-A2. For subgroups Ia, subtypes II and IV clinical specimens were used. Due to lack of clinical specimens phagemids for subgroup Ic and subtype II were used.

BKV positive material was diluted in BKV- PCR negative pooled EDTA plasma to prepare the ISS. The ISS titer was confirmed by QIAGEN artus BK Virus QS-RGQ PCR

Kit with three replicates. For subgroup Ic, digital droplet PCR was used to confirm initial titer as primers in the QIAGEN kit were not able to detect this subgroup. Each panel consisted of 4 concentration levels around the LoD (1.5x ( $3 2 . 2 5 \mathrm { I U / m L }$ ), $1 . 0 \mathrm { x }$ (21.5 $\mathrm { I U / m L }$ ), $0 . 5 \mathrm { x }$ ( $1 0 . 8 \mathrm { I U / m L } )$ ) and $0 . 2 5 \mathrm { x }$ $( 5 . 4 ~ \mathrm { I U / m L }$ ) LoD). Initial testing was performed with a panel of 1.5x, 1.0x and $0 . 5 \mathrm { x }$ LoD panel members. For those sample with ${ < } 9 5 \%$ hit rate at $0 . 5 \mathrm { x }$ LoD an additional panel member of $0 . 0 2 5 \mathrm { x }$ LoD was tested. At $1 . 5 \mathrm { x }$ LoD $( 3 2 . 2 5 \ : \mathrm { I U / m L } )$ at genotypes showed a $100 \%$ positivity rate.

A total of 63 replicates per concentration level were tested across 3 dilution series, 3 instruments and $3 \mathrm { k i t }$ lots. A different operator tested each dilution series. The results are shown in Table 12 and verify that a hit rate of $9 5 \%$ or higher was observed at $2 1 . 5 \ : \mathrm { I U / m L }$ (1x LoD) for BKV genotypes. Thus, the observed hit rates verify the LoD for BKV at $2 1 . 5 \ : \mathrm { I U / m L }$ .

Table 12: Verification of the LoD for BKV Genotypes   

<table><tr><td rowspan=1 colspan=1>Genotype</td><td rowspan=1 colspan=1>Test concentration</td><td rowspan=1 colspan=1>Number of validreplicates (N)</td><td rowspan=1 colspan=1>Number ofpositives (n)</td><td rowspan=1 colspan=1>Hit rate(n/N)x100</td></tr><tr><td rowspan=1 colspan=1>Subgroup Ia</td><td rowspan=1 colspan=1>5.4 IU/mL</td><td rowspan=1 colspan=1>Not Tested</td><td rowspan=1 colspan=1>Not Tested</td><td rowspan=1 colspan=1>Not Tested</td></tr><tr><td rowspan=1 colspan=1>Subgroup Ia</td><td rowspan=1 colspan=1>10.8 IU/mL</td><td rowspan=1 colspan=1>63</td><td rowspan=1 colspan=1>54</td><td rowspan=1 colspan=1>85.7%</td></tr><tr><td rowspan=1 colspan=1>Subgroup Ia</td><td rowspan=1 colspan=1>21.5 IU/mL</td><td rowspan=1 colspan=1>63</td><td rowspan=1 colspan=1>63</td><td rowspan=1 colspan=1>100.0%</td></tr><tr><td rowspan=1 colspan=1>Subgroup Ic</td><td rowspan=1 colspan=1>5.4 IU/mL</td><td rowspan=1 colspan=1>62*</td><td rowspan=1 colspan=1>57</td><td rowspan=1 colspan=1>91.9%</td></tr><tr><td rowspan=1 colspan=1>Subgroup Ic</td><td rowspan=1 colspan=1>10.8 IU/mL</td><td rowspan=1 colspan=1>63</td><td rowspan=1 colspan=1>61</td><td rowspan=1 colspan=1>96.8%</td></tr><tr><td rowspan=1 colspan=1>Subgroup Ic</td><td rowspan=1 colspan=1>21.5 IU/mL</td><td rowspan=1 colspan=1>62*</td><td rowspan=1 colspan=1>62</td><td rowspan=1 colspan=1>100.0%</td></tr><tr><td rowspan=1 colspan=1>Subtype II</td><td rowspan=1 colspan=1>5.4 IU/mL</td><td rowspan=1 colspan=1>63</td><td rowspan=1 colspan=1>54</td><td rowspan=1 colspan=1>85.7%</td></tr><tr><td rowspan=1 colspan=1>Subtype II</td><td rowspan=1 colspan=1>10.8 IU/mL</td><td rowspan=1 colspan=1>63</td><td rowspan=1 colspan=1>63</td><td rowspan=1 colspan=1>100.0%</td></tr><tr><td rowspan=1 colspan=1>Subtype II</td><td rowspan=1 colspan=1>21.5 IU/mL</td><td rowspan=1 colspan=1>63</td><td rowspan=1 colspan=1>63</td><td rowspan=1 colspan=1>100.0%</td></tr><tr><td rowspan=1 colspan=1>Subtype III</td><td rowspan=1 colspan=1>5.4 IU/mL</td><td rowspan=1 colspan=1>63</td><td rowspan=1 colspan=1>49</td><td rowspan=1 colspan=1>77.8%</td></tr><tr><td rowspan=1 colspan=1>Subtype II</td><td rowspan=1 colspan=1>10.8 IU/mL</td><td rowspan=1 colspan=1>63</td><td rowspan=1 colspan=1>63</td><td rowspan=1 colspan=1>100.0%</td></tr><tr><td rowspan=1 colspan=1>Subtype II</td><td rowspan=1 colspan=1>21.5 IU/mL</td><td rowspan=1 colspan=1>63</td><td rowspan=1 colspan=1>63</td><td rowspan=1 colspan=1>100.0%</td></tr><tr><td rowspan=1 colspan=1>Subtype IV</td><td rowspan=1 colspan=1>5.4 IU/mL</td><td rowspan=1 colspan=1>63</td><td rowspan=1 colspan=1>57</td><td rowspan=1 colspan=1>90.5%</td></tr><tr><td rowspan=1 colspan=1>Subtype IV</td><td rowspan=1 colspan=1>10.8 IU/mL</td><td rowspan=1 colspan=1>63</td><td rowspan=1 colspan=1>63</td><td rowspan=1 colspan=1>100.0%</td></tr><tr><td rowspan=1 colspan=1>Subtype IV</td><td rowspan=1 colspan=1>21.5 IU/mL</td><td rowspan=1 colspan=1>63</td><td rowspan=1 colspan=1>63</td><td rowspan=1 colspan=1>100.0%</td></tr></table>

\* Sample excluded due to titer $<$ LLoQ

This study evaluated whether the Limit of Detection in Generic Specimen Diluent (GSD) is equivalent to the LoD in EDTA-plasma $( 2 1 . 5 \ : \mathrm { I U / m L } )$ in the cobas BKV assay, so that GSD can be used as a negative control for cobas BKV. Three independent dilution series (0.5x, $1 . 0 \mathrm { x }$ , and 1.5x LoD of cobas BKV) were prepared on three consecutive days using the $1 ^ { \mathrm { s t } }$ WHO BKV International Standard in GSD. Each dilution series was tested using one testspecific reagent kit lot. Three out of three runs performed were valid yielding 189 test results.

The results (Table 13) demonstrated a comparable LoD performance of cobas BKV in Plasma and GSD.

Table 13: Hit Rates for all Dilution Series combined   

<table><tr><td rowspan=1 colspan=1>ConcentrationLevel</td><td rowspan=1 colspan=1>Titer (IU/mL)</td><td rowspan=1 colspan=1>Hit Rate (%)</td><td rowspan=1 colspan=1>Two-sided 95% CI</td></tr><tr><td rowspan=1 colspan=1>1.5x LoD</td><td rowspan=1 colspan=1>32.25</td><td rowspan=1 colspan=1>98.41</td><td rowspan=1 colspan=1>91.47% - 99.96%</td></tr><tr><td rowspan=1 colspan=1>1.0x LoD</td><td rowspan=1 colspan=1>21.50</td><td rowspan=1 colspan=1>95.24</td><td rowspan=1 colspan=1>86.71% - 99.01%</td></tr><tr><td rowspan=1 colspan=1>0.5x LoD</td><td rowspan=1 colspan=1>10.75</td><td rowspan=1 colspan=1>82.54</td><td rowspan=1 colspan=1>70.90% - 90.95%</td></tr></table>

5. Assay Cut-Off:

Not applicable.

6. Accuracy (Instrument): Not applicable.

7. Carry-Over:

The carry-over rate for cobas BKV was determined by testing 240 replicates of an BKVnegative matrix sample and 225 replicates of a high titer BKV sample at $2 . 0 0 \mathrm { E + } 0 7 \ : \mathrm { I U / m L }$ (approximately $1 0 ^ { 6 } \mathrm { \overline { { X } } }$ LLoQ). The high titer sample was prepared by spiking a high-positive BKV cell culture supernatant (ATCC BKV VR-837) into negative pooled EDTA-plasma and titer verified by using QIAGEN artus BK Virus QS-RGQ PCR Kit.

In total, five runs were performed with positive and negative samples in a checkerboard configuration using one kit lot and one cobas 8800 system.

All 240 replicates of the negative sample were negative, resulting in a carry-over rate of $0 \%$ (upper one-sided $9 5 \%$ confidence interval: $1 . 2 4 \%$ ).

# B Comparison Studies:

# 1. Method Comparison with Predicate Device:

The comparator BKV assay is well validated, currently used in clinical practice at a major transplant center in the United States and is traceable to the BKV WHO standard. The use of this comparator is acceptable. Please see “Other Clinical Supportive Data” section below.

2. Matrix Comparison: Not applicable.

# C Clinical Studies:

1. Clinical Sensitivity: Not applicable

1. Clinical Specificity: Not applicable

2. Other Clinical Supportive Data (When 1. and 2. Are Not Applicable):

Concordance of cobas BKV with a Comparator BKV LDM

The clinical performance of the cobas BKV assay was compared with a validated, wellestablished laboratory developed nucleic acid method (LDM) (comparator BKV LDM) by measuring BKV DNA levels in longitudinal clinical samples (neat and diluted) of BKVinfected and non-infected patients. Contrived EDTA plasma samples spiked with cultured BKV virus were used to cover the linear range.

The comparator BKV LDM is well validated, currently used in clinical practice at a major transplant center in the United States and is traceable to the WHO standard. The use of this comparator is acceptable. The comparator BKV LDM amplifies VP2 gene sequences only whereas the cobas BKV assay amplifies sequences in both the VP2 and the small t-antigen. Due to different methods of measuring BKV viral load at each institution, BKV viral load quantitation may vary between laboratories. Therefore, viral load obtained from different laboratories on longitudinal samples from the same patient should not be compared. Clinical management decisions should be based on assessment of the patients BKV load with the same viral load test.

A total of 550 samples (217 neat and 303 diluted clinical samples from 129 transplant subjects and 30 contrived samples using cultured BKV genotype 1b) were valid on both assays and evaluable for the clinical concordance analysis. Results presented in Table 14 demonstrate a column percent agreement between the cobas BKV and BKV LDM ranging from between $9 4 . 1 \%$ to $100 \%$ depending on the analyte concentration in the samples. The LLoQ for BKV LDM was $2 0 0 \mathrm { \ : I U / m l }$ $( 2 . 3 \log _ { 1 0 } \mathrm { I U / m l } )$ . The analytical ranges BKV DNA were calculated based on standard deviation $( \sigma = 0 . 3 7 \log _ { 1 0 } \mathrm { { I U / m l } ) }$ of the comparator BKV LDM (Table 14). Analyte concentration of $3 . 0 \log _ { 1 0 } \mathrm { { I U / m L } }$ represented $\mathrm { L L o Q } + 2 \sigma$ , 3.7 $\mathrm { l o g _ { 1 0 } I U / m L }$ represented $\mathrm { L L o Q } + 4 \sigma$ and $4 . 4 \log _ { 1 0 } \mathrm { { I U / m L } }$ represented $\mathrm { L L o Q } + 6 \sigma$ with a range interval of $2 \sigma$ . Although, concordance between cobas BKV and BKV LDM of $9 4 . 1 \%$ , for the range of 2.3 to $< 3 . 0 \log _ { 1 0 } \mathrm { I U / m L }$ was below $9 5 \%$ , the $9 5 \%$ confidence interval ranged from $8 7 . 0 \%$ to $9 7 . 5 \%$ and was found to be acceptable.

Table 14: Concordance analysis between cobas BKV and the comparator LDM on BKV DNA level results for all samples   

<table><tr><td rowspan=1 colspan=1>cobas® BKV(log1o IU/mL)</td><td rowspan=1 colspan=1>ComparatorBBKV LDM(log10 IU/mL)Target NotDetected</td><td rowspan=1 colspan=1>ComparatorBKV LDM(log10 IU/mL)&lt;LLoQ(&lt; 2.3)</td><td rowspan=1 colspan=1>Comparator BKVLDM (log10I/mL2.3 to &lt; 3.0</td><td rowspan=1 colspan=1>Comparato BKVLDM (log10I/mL3.0 to &lt; 3.7</td><td rowspan=1 colspan=1>Comparator BKVLDM (log10I/mL3.7 to 4.4</td><td rowspan=1 colspan=1>ComparatoBKVLDM (log10IV/mL&gt; 4.4</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Target NotDetected</td><td rowspan=1 colspan=1>107</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>119</td></tr><tr><td rowspan=1 colspan=1>&lt; 2.3 (&lt; LLoQ)</td><td rowspan=1 colspan=1>23</td><td rowspan=1 colspan=1>51</td><td rowspan=1 colspan=1>39</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>113</td></tr><tr><td rowspan=1 colspan=1>2.3 to &lt; 3.0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>62</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>106</td></tr><tr><td rowspan=1 colspan=1>3.0 to &lt; 3.7</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>71</td><td rowspan=1 colspan=1>42</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>114</td></tr><tr><td rowspan=1 colspan=1>3.8 to 4.4</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>26</td><td rowspan=1 colspan=1>26</td><td rowspan=1 colspan=1>52</td></tr><tr><td rowspan=1 colspan=1>&gt; 4.4</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>45</td><td rowspan=1 colspan=1>46</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>130</td><td rowspan=1 colspan=1>61</td><td rowspan=1 colspan=1>85</td><td rowspan=1 colspan=1>133</td><td rowspan=1 colspan=1>70</td><td rowspan=1 colspan=1>71</td><td rowspan=1 colspan=1>550</td></tr><tr><td rowspan=1 colspan=1>ColumnAgreement (%)</td><td rowspan=1 colspan=1>130/130 (100.0%)</td><td rowspan=1 colspan=1>61/61(100.0%)</td><td rowspan=1 colspan=1>80/85(94.1%)</td><td rowspan=1 colspan=1>133/133(100.0%)</td><td rowspan=1 colspan=1>69/70(98.6%)</td><td rowspan=1 colspan=1>71/71(100.0%)</td><td rowspan=1 colspan=1>−</td></tr><tr><td rowspan=1 colspan=1>95% CI upperand lower boundsa</td><td rowspan=1 colspan=1>97.1%, 100%</td><td rowspan=1 colspan=1>94.1%, 100.0%</td><td rowspan=1 colspan=1>87.0%,97.5%</td><td rowspan=1 colspan=1>97.2%,100.0%</td><td rowspan=1 colspan=1>92.3%,99.7%</td><td rowspan=1 colspan=1>94.9%,100.0%</td><td rowspan=1 colspan=1>−</td></tr></table>

Note: $\mathrm { C I } =$ Confidence Interval; $\mathrm { L L o Q } =$ lower limit of quantitation of Comparator BKV LDM $( 2 0 0 \mathrm { I U / m L } )$ . Standard Deviation of Comparator BKV LDM estimated at $0 . 3 7 \log _ { 1 0 } \mathrm { { I U / m L } }$ (BKV LDM analytical precision study). Analyte concentration of $3 . 0 \log 1 0 \mathrm { I U / m L }$ represented $\mathrm { L L o Q } + 2 \sigma _ { \mathrm { \Omega } }$ , $3 . 7 \log 1 0 \mathrm { I U / m L }$ represented $\mathrm { L L o Q } + 4 \sigma$ and $4 . 4 \log 1 0 \ : \mathrm { I U / m L }$ represented $\mathrm { L L o Q } + 6 \sigma$ with a range interval of $2 \sigma$ . Paired samples evaluable for clinical concordance analysis were included in this table. a Assumed independence between all samples.

Discordant results were defined as those that are more than one box away from the diagonal (indicated by shaded cells in Table 14). For Target Not Detected (TND) by LDM Column Agreement the cobas BKV Target Not Detected and $< \mathrm { L L o Q } ( < 2 . 3 )$ cells were combined. The rationale for adding the adjacent <LLoQ and TND cells for the TND column is that the difference between a TND and <LLoQ is not clinically meaningful and that these are analytically at the lower end of the measuring range, which may be impacted by random error.

All 43 BKV DNA negative samples tested using the cobas BKV and the BKV LDM were negative on both assays. The Negative Percent Agreement was $1 0 0 \%$ ( $9 5 \%$ Exact CI: $9 1 . 8 \% - 1 0 0 \% )$ . Concordance between cobas BKV and the comparator BKV LDM was also evaluated using different clinical thresholds (Table 15).

Table 15: Concordance summary of cobas BKV and comparator BKV LDM using different thresholds   

<table><tr><td rowspan=1 colspan=1>Thresholds*</td><td rowspan=1 colspan=1>Percent Agreement&lt; Threshold(n/N)(95% CI)</td><td rowspan=1 colspan=1>Percent Agreement≥ Threshold(n/N)(95% CI)</td></tr><tr><td rowspan=1 colspan=1>Target Not Detected</td><td rowspan=1 colspan=1>100% (130/130)(97.2%, , 100%)</td><td rowspan=1 colspan=1>97.1% (408/420)(95.1%, 98.4%)</td></tr><tr><td rowspan=1 colspan=1>2.3 Log10 IU/mL(LLoQ)</td><td rowspan=1 colspan=1>98.4% (188/191)(95.5%, 99.5%)</td><td rowspan=1 colspan=1>87.7% (315/359)(83.9%, 90.7%)</td></tr><tr><td rowspan=1 colspan=1>3.0 Log10 IU/mL</td><td rowspan=1 colspan=1>99.6% (275/276)(98.0%, 99.9%)</td><td rowspan=1 colspan=1>77.0% (211/274)(71.7%, 81.6%)</td></tr><tr><td rowspan=1 colspan=1>4.0 Log10 IU/mL</td><td rowspan=1 colspan=1>100.0% (447/447)(99.1%, 100.0%)</td><td rowspan=1 colspan=1>67.0% (69/103)(57.4%, 75.3%)</td></tr></table>

Note: Samples with a Target Not Detected results were categorised as $<$ threshold value in $\mathrm { I U / m L }$ . $\mathrm { L L o Q } =$ lower limit of quantitation of Comparator BKV LDM $( 2 0 0 \mathrm { I U / m L } = 2 . 3 \mathrm { l o g _ { 1 0 } I U / m L } )$ . $9 5 \%$ confidence interval (CI) calculated by Score method assuming independence between all samples. \* Thresholds of $1 0 0 0 \mathrm { I U / m l } = 3 . 0 \mathrm { L o g _ { 1 0 } I U / m l }$ and $1 0 , 0 0 0 \mathrm { I U / m l } = 4 . 0 \mathrm { L o g _ { 1 0 } I U / m l }$ .

From all samples with valid paired results within the overlapping linear range of both the cobas BKV and the BKV LDM assay $\mathrm { 2 0 0 \ I U / m L }$ to $2 0 \mathrm { ~ x ~ } 1 0 ^ { 6 } \mathrm { \Pi U / m L }$ ), a total of 313 samples (133 neat and 159 diluted samples from 68 transplant subjects and 21 contrived samples) from the three testing sites were evaluated by correlation analysis.

The $9 5 \%$ CI of the y-intercept for the Deming linear regression analysis did not include 0, indicating that the results of the cobas BKV and the BKV LDM differ by a constant number (Figure 3). This difference could be caused by differences in calibration methods (e.g., type of quantitative calibrator, frequency of calibration) between the two tests. The $9 5 \%$ CIs of the slope included 1, indicating there is no statistical evidence of proportional difference between the two assays for all samples. The overall correlation coefficient was 0.96, indicating a strong correlation between cobas BKV and the comparator BKV LDM.

Bias plot analysis of DNA level differences indicated a systematic difference between both assays that is constant across the overlapping linear range. The $9 5 \%$ CI of the intercept of the fitted line in the bias plots was $4 0 . 4 \%$ to $1 6 . 8 \%$ , which is within $\pm 0 . 7 4 \log _ { 1 0 } \mathrm { { I U } / m L } \left( \pm 2 \right.$ times analytical precision standard deviation of comparator BKV LDM). Furthermore, the mean bias was estimated at $0 . 3 5 7 \log _ { 1 0 } \mathrm { I U / m L }$ and the systematic difference between both assays was $0 . 3 4 3 \log _ { 1 0 } \mathrm { I U / m L }$ and $0 . 3 6 2 \log _ { 1 0 } \mathrm { I U / m L }$ for samples with DNA levels at 3 and $4 \log _ { 1 0 } \mathrm { { I U / m L } }$ , respectively.

# Figure 3: Correlation between cobas BKV and comparator BKV LDM for all samples: Deming linear regression plot of DNA levels $\bf ( l o g _ { 1 0 } I U / m L )$

![](images/c3d45c457b4d71516ce99f385557de7cf33a62d3d40609d1c362d7fde7ade22b.jpg)

# B Clinical Cut-Off:

Not applicable. Recommendations regarding monitoring BKV viral load post-transplant and medically relevant BKV DNA thresholds vary among transplant type and transplant institutions. Results of the cobas BKV test should be interpreted in the context of other clinical data and should not be the sole basis for treatment decisions.

D Expected Values/Reference Range: Not applicable.   
E Other Supportive Instrument Performance Characteristics Data: Not applicable.

# VIII Proposed Labeling:

The labeling supports the finding of substantial equivalence for this device.

# IX Conclusion:

The submitted information in this premarket notification is complete and supports a substantial equivalence decision.