# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATIONDECISION SUMMARYASSAY AND INSTRUMENT

I Background Information:

A 510(k) Number K203220   
B Applicant Roche Molecular Systems, Inc.   
C Proprietary and Established Names cobas BKV   
D Regulatory Information

<table><tr><td rowspan=1 colspan=1>ProductCode(s)</td><td rowspan=1 colspan=1>Classification</td><td rowspan=1 colspan=1>RegulationSection</td><td rowspan=1 colspan=1>Panel</td></tr><tr><td rowspan=1 colspan=1>QLX</td><td rowspan=1 colspan=1>Class II</td><td rowspan=1 colspan=1>21 CFR 21 CFR 866.3183- Quantitative ViralNucleic Acid TestforTransplant PatientManagement</td><td rowspan=1 colspan=1></td></tr></table>

# II Submission/Device Overview:

Purpose for Submission: To obtain substantial equivalence determination of the cobas BKV with urine samples.

Measurand: BKV DNA

Type of Test: The cobas BKV assay is a quantitative Polymerase Chain Reaction (PCR) performed on the cobas 6800/8800 automated systems, for the detection of BKV DNA in transplant patients.

# III Intended Use/Indications for Use:

# A Intended Use:

cobas BKV is an in vitro nucleic acid amplification test for the quantitation of BK virus (BKV) DNA in human EDTA plasma and urine stabilized in cobas PCR Media on the cobas 6800/8800 Systems.

In EDTA plasma, cobas BKV is intended for use as an aid in the management of BKV in transplant patients. In patients undergoing monitoring of BKV in EDTA plasma, serial DNA measurements can be used to indicate the need for potential treatment changes and to assess viral response to treatment.

In urine stabilized in cobas PCR Media, cobas BKV is intended for use as an aid in the management of BKV in transplant patients.

The results from cobas BKV are intended to be read and analyzed by a qualified licensed healthcare professional in conjunction with clinical signs and symptoms and relevant laboratory findings. Test results must not be the sole basis for patient management decisions.

cobas BKV is not intended for use as a screening test for blood or blood products or human cells, tissues, and cellular and tissue-based products (HCT/Ps).

B Indication(s) for Use: Same as intended use   
C Special Conditions for Use Statement(s): Rx   
D Special Instrument Requirements: The test is run on the cobas 6800/8800 instrument system

# IV Device/System Characteristics:

# A Device Description:

The cobas BKV is a quantitative test performed on the cobas 6800 System and cobas 8800 System. cobas BKV enables the detection of BKV DNA in plasma and cobas PCR media stabilized urine specimens. The cobas BKV is a dual target assay, with both targets using the same dye. The DNA Internal Control, used to monitor the entire sample preparation and PCR amplification process, is introduced into each specimen during sample processing. The cobas BKV enables the detection and quantitation of BKV DNA in transplant patients.

The cobas BKV is intended for use as an aid in the management of BKV in transplant patients. In patients undergoing monitoring of BKV, serial DNA measurements can be used to indicate the need for potential treatment changes and to assess viral response to treatment.

The cobas BKV Kit system consists of:

• Proteinase Solution

• DNA Quantitation Standard (DNA QS)   
• Elution Buffer   
• Master Mix Reagent 1   
• BKV Master Mix Reagent 2

The BKV viral load is quantified against a non-BKV DNA Quantitation Standard (DNA-QS), which is introduced into each specimen during sample preparation. The DNA-QS also functions as an internal control for sample preparation and the PCR amplification process.

In addition, the test utilizes the following separately packed and sold control materials:

1. cobas EBV/BKV Positive Control Kit:

EBV/BKV Low Positive Control (EBV/BKV $\mathrm { L } ( + ) \mathrm { C } )$ • EBV/BKV High Positive Control (EBV/BKV $\mathrm { H } ( + ) \mathrm { C } )$ .

The positive control contains phage packaged EBVBKV and BKV DNA in normal human plasma and serves as a control for the cobas BKV test.

2. cobas Negative Control Kit:

cobas Buffer Negative Control (BUF (-) C)

Testing with the cobas BKV test requires the following materials that are not provided:

cobas OMNI Reagents cobas BKV Assay Specific Analysis Package (ASAP) software

The cobas BKV test uses sample preparation (nucleic acid extraction and purification) followed by PCR amplification and detection, all steps are fully automated by the cobas 6800/8800 platform.

# B Principle of Operation:

The cobas BKV test is a quantitative PCR test performed on the fully automated cobas 6800/ 8800 Systems that detects and quantifies BKV DNA from (EDTA) plasma and cobas PCR media stabilized urine specimens of transplant patients as follows.

# Target Selection

Selective amplification of BKV target nucleic acid from the sample is achieved using specific forward and reverse primers which are selected to amplify highly-conserved regions of the BKV DNA VP-2 and T-antigen gene. Specific fluorescence labeled probes for each amplicon are used to detect and quantify the BKV targets from subtypes I, II, III and IV as well as subgroups Ia, Ib, Ic, IVa, IVb, and IVc. Selective amplification and detection of a DNA Quantitation Standard (DNA-QS) is achieved using specific forward and reverse primers and a DNA-QS specific probe all specifically selected to have no homology with the BKV genome.

# Sample Preparation (Nucleic Acid Extraction and Purification)

Nucleic acid from patient samples and external controls are extracted upon addition of the DNA DNA-QS. The DNA-QS molecules are extracted simultaneously with the samples/controls

serving as an extraction control. Viral nucleic acid is released by addition of proteinase and lysis reagent to the sample. The released nucleic acid, along with the added DNA-QS binds to magnetic glass particles. Unbound substances and impurities are removed with subsequent wash reagent steps and purified nucleic acid is then eluted from the magnetic glass particles with elution buffer.

# Nucleic Acid Amplification and Target Detection

The cobas BKV master mix contains detection probes which are specific for the two BKV target sequences and the DNA-QS nucleic acid, respectively. The two BKV specific detection probes are labeled with the same HEX fluorescent dye while the DNA-QS detection probe is labeled with the CY5.5 fluorescent dye both acting as reporter dyes. Each probe also has a second dye, BHQ, which acts as a quencher that suppresses the fluorescent signals of the intact probes when they are not bound to their respective target sequence. Target bound probes, however, emit fluorescence of the two reporter dyes. This fluorescence is measured at defined wavelengths, thus permitting simultaneous detection and discrimination of the BKV targets and the DNA-QS amplification products generated by a thermostable DNA polymerase enzyme.

# BKV DNA Quantitation

During the extension phase of the PCR process, fluorescence readings are processed to generate Ct values for the BKV DNA target and the BKV QS DNA. The lot-specific calibration constants provided with the cobas BKV test are used to calculate the titer value for the specimens and controls based on both the BKV DNA target and the BKV QS DNA Ct values. BKV viral load results are reported in International Units/mL $\mathrm { ( I U / m L ) }$ ).

# C Interpretation of Results:

<table><tr><td rowspan=1 colspan=1>Results</td><td rowspan=1 colspan=1>Interpretation</td></tr><tr><td rowspan=1 colspan=1>Target Not Detected</td><td rowspan=1 colspan=1>BKV DNA not detected.Report results as &quot;BKV not detected&quot;.</td></tr><tr><td rowspan=1 colspan=1>&lt; Titer Mina</td><td rowspan=1 colspan=1>Calculated titer is below the Lower Limit of Quantitation(LLoQ) of the assay. Report results as &quot;BKV detected, less than(Titer Min)&quot;.EDTA plasma Titer Min = 21.5 IU/mLUrine Titer Min = 200 IU/mL</td></tr><tr><td rowspan=1 colspan=1>Titer</td><td rowspan=1 colspan=1>Calculated titer is within the Linear Range of the assay  greater than orequal to Titer Min and less than or equal to Titer Max.Report results as &quot;(Titer) of BKV detected&quot;.</td></tr><tr><td rowspan=1 colspan=1>&gt; Titer Maxb</td><td rowspan=1 colspan=1>Calculated titer is above the Upper Limit of Quantitation(ULoQ) of the assay. Report results as &quot;BKV detected, greaterthan (Titer Max)&quot;.EDTA plasma and urine Titer Max = 1.0E+08 IU/mL</td></tr></table>

a Sample results $<$ Titer min (Target Detected $<$ LLoQ) should be interpreted with the context of other clinical data and should not be the sole basis for treatment decisions.

b Sample result $>$ Titer Max refers to BKV positive samples detected with titers above the upper limit of quantitation (ULoQ). If a quantitative result is desired, the original sample should be diluted with BKV-negative human EDTA plasma and the test should be repeated. Multiply the reported result by the dilution factor.

# D Instrument Description Information:

1. Instrument Name:

cobas 6800/8800

2. Specimen Identification:

The specimen identification information is captured and stored as a digital record. Whole blood and Urine specimens are collected in appropriate sample collection tubes as per instructions for use (IFU) and prepared for the test procedure as described in detail in the cobas 6800/8800 Systems – User Assistance and/or User Guide.

3. Specimen Sampling and Handling:

Managed by a trained technician as per IFU. Urine specimens must be transferred into the cobas PCR media tube with stabilizer immediately after collection. The specimen can be stored at $2 \mathrm { ^ { \circ } C }$ to $3 0 \mathrm { { } ^ { \circ } C }$ up to 24 hours if it cannot be stabilized immediately.

4. Calibration:

The lot-specific calibration constants are provided with the cobas BKV test. The cobas 6800/8800 Systems automatically determine the BKV DNA concentration for the samples and controls. The BKV DNA concentration is expressed in International Units per milliliter $\mathrm { ( I U / m L ) }$ .

# 5. Quality Control:

One negative control $[ ( - ) \mathrm { C } ]$ and two positive controls, a low positive control [EBV/BKV $\mathrm { L } ( + ) \mathrm { C } ]$ and a high positive control [EBV/BKV $\mathrm { H } ( + ) \mathrm { C } ]$ is processed with each batch. The batch is valid if no flags appear for any of the three controls. The negative control result is displayed as $( - ) \mathrm { { C } }$ and the low and high positive controls are displayed as EBV/BKV $\mathrm { L } ( + ) \mathrm { C }$ and EBV/BKV $\mathrm { H } ( + ) \mathrm { C }$ . Invalidation of results is performed automatically by the cobas 6800/8800 software based on negative and positive control failures.

V Substantial Equivalence Information:

A Predicate Device Name(s): cobas BKV

B Predicate 510(k) Number(s): K202215

# C Comparison with Predicate(s):

<table><tr><td colspan="1" rowspan="1">Device &amp; PredicateDevice(s):</td><td colspan="1" rowspan="1">Predicate Device: cobasBKV (K202215)</td><td colspan="1" rowspan="1">New Device: cobas BKV(K203220)</td></tr><tr><td colspan="1" rowspan="1">Device Trade Name</td><td colspan="1" rowspan="1">cobas BKV</td><td colspan="1" rowspan="1">cobas BKV</td></tr><tr><td colspan="3" rowspan="1">General Device Characteristic Similarities</td></tr><tr><td colspan="1" rowspan="1">Regulation</td><td colspan="1" rowspan="1">21 CFR 866.3183</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Regulation name</td><td colspan="1" rowspan="1">Quantitative viral nucleic acidtest for transplant patientmanagement</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Intended Use/Indicationsfor Use</td><td colspan="1" rowspan="1">cobas BKV is an in vitronucleic acid amplification testfor the quantitation of BKvirus (BKV) DNA in humanEDTA plasma on the cobas6800/8800 Systems.cobas BKV is intended foruse as an aid in themanagement of BKV intransplant patients. In patientsundergoing monitoring ofBKV, serial DNAmeasurements can be used toindicate the need for potentialtreatment changes and toassess viral response totreatment.The results from cobas BKVare intended to be read andanalyzed by a qualifiedlicensed healthcareprofessional in conjunctionwith clinical signs andsymptoms and relevantlaboratory findings. Test resultsmust not be the sole basis forpatient management decisions.cobas BKV is not intended foruse as a screening test fordonors of blood or bloodproducts or human cells,tissues, and cellular andtissue-based products(HCT/Ps).</td><td colspan="1" rowspan="1">cobas BKV is an in vitronucleic acid amplification testfor the quantitation of BKvirus (BKV) DNA in humanEDTA plasma and urinestabilized in cobas PCRMedia on the cobas6800/8800 Systems.In EDTA plasma, cobas BKVis intended for use as an aid inthe management of BKV intransplant patients. In patientsundergoing monitoring ofBKV in EDTA plasma, serialDNA measurements can beused to indicate the need forpotential treatment changesand to assess viral response totreatment.In urine stabilized in cobasPCR Media, cobas BKV isintended for use as an aid inthe management of BKV intransplant patients.The results from cobas BKVare intended to be read andanalyzed by a qualifiedlicensed healthcareprofessional in conjunctionwith clinical signs andsymptoms and relevantlaboratory findings. Testresults must not be the solebasis for patient managementdecisions.</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">cobas BKV is not intended foruse as a screening test fordonors of blood or bloodproducts or human cells,tissues, and cellular andtissue-based products(HCT/Ps).</td></tr><tr><td colspan="1" rowspan="1">Conditions for use</td><td colspan="1" rowspan="1">For prescription use</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Patient population</td><td colspan="1" rowspan="1">Transplant recipients</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Sample PreparationProcedure and TargetDetection</td><td colspan="1" rowspan="1">cobas 6800/8800 Systems</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Analyte Target</td><td colspan="1" rowspan="1">BKV DNA</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Calibrators</td><td colspan="1" rowspan="1">Phagemid</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Amplification Technology</td><td colspan="1" rowspan="1">Real-time PCR</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Detection Chemistry</td><td colspan="1" rowspan="1">Paired reporter and quencherfluorescence labeled probes(TaqMan Technology) usingfluorescence resonanceenergy transfer (FRET)</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Controls Used</td><td colspan="1" rowspan="1">Sample processing control(IC)Positive and negative control</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Result Analysis</td><td colspan="1" rowspan="1">Based on PCR cyclethreshold analysis</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="3" rowspan="1">General Device Characteristic Differences</td></tr><tr><td colspan="1" rowspan="1">Sample Types</td><td colspan="1" rowspan="1">EDTA - Plasma</td><td colspan="1" rowspan="1">EDTA - Plasma, Urine</td></tr></table>

# VI Standards/Guidance Documents Referenced:

EP05-A3-Evaluation of Precision of Quantitative Measurement Procedures; Approved Guideline—Third Edition.

EP06-A-Evaluation of the Linearity of Quantitative Measurement Procedures: A Statistical Approach; Approved Guideline.

EP07-A2-Interference Testing in Clinical Chemistry; Approved Guideline—Second Edition.

EP17-A2-Evaluation of Detection Capability for Clinical Laboratory Measurement Procedures;   
Approved Guideline – Second Edition.

# VII Performance Characteristics (if/when applicable):

Performance characteristics with plasma specimens can be found in K202215

# A Analytical Performance:

# 1. Precision:

Precision was assessed for the predominant BKV Subgroup 1b by serially diluting BKV 1b lambda stock in BKV negative pooled urine stabilized in cobas PCR media (1:1 dilution) to generate a 7-member panel ranging from a nominal concentration of $3 0 0 \mathrm { \ : I U / m L }$ to $\mathrm { 1 x 1 0 ^ { 8 } \Pi U / m L }$ . Precision was calculated on results generated with 21 replicates per run, two runs per day for 12 days using three kit lots, two cobas 6800 systems and two operators for a total of 72 replicates per panel member (PM). Results were analyzed according to CLSI guideline EP05-A3. Only replicates with titers within the linear range of the assay $( 2 . 0 \mathrm { E } { + } 0 2$ to $1 . 0 \mathrm { E } { + } 8 \ \mathrm { I U / m L }$ ) were used for data analysis. For PM1, the assigned concentration was $7 . 4 1 \mathrm { ~ x ~ } 1 0 ^ { 7 } \ : \mathrm { I U / m L }$ and 15/21 replicates had results above the Upper Limit of Quantification; therefore, results from those 15 replicates were excluded from the analysis. For PM7 the assigned concentration was $2 2 2 \mathrm { I U / m L }$ and 2/72 replicates had results below the LLoQ; therefore, results from those two replicates were excluded from the analysis. The results for cobas BKV precision calculated for seven dilution levels are shown in Table 1.

Table 1: Total Precision as Standard Deviation of Log10 Titers   

<table><tr><td rowspan=2 colspan=1>Panel Member</td><td rowspan=2 colspan=1>NominalConcentration(IU/mL)</td><td rowspan=2 colspan=1>AssignedConcentration(IU/mL)</td><td rowspan=2 colspan=1>Assigned Log10Titer</td><td rowspan=1 colspan=4>Standard Deviation</td></tr><tr><td rowspan=1 colspan=1>Lot 1</td><td rowspan=1 colspan=1>Lot 2</td><td rowspan=1 colspan=1>Lot 3</td><td rowspan=1 colspan=1>All lots</td></tr><tr><td rowspan=1 colspan=1>PM 1</td><td rowspan=1 colspan=1>1.00E+08</td><td rowspan=1 colspan=1>7.41E+07</td><td rowspan=1 colspan=1>7.87</td><td rowspan=1 colspan=1>0.02</td><td rowspan=1 colspan=1>0.01</td><td rowspan=1 colspan=1>0.02</td><td rowspan=1 colspan=1>0.02</td></tr><tr><td rowspan=1 colspan=1>PM 2</td><td rowspan=1 colspan=1>1.00E+06</td><td rowspan=1 colspan=1>7.41E+05</td><td rowspan=1 colspan=1>5.87</td><td rowspan=1 colspan=1>0.02</td><td rowspan=1 colspan=1>0.02</td><td rowspan=1 colspan=1>0.02</td><td rowspan=1 colspan=1>0.02</td></tr><tr><td rowspan=1 colspan=1>PM 3</td><td rowspan=1 colspan=1>1.00E+05</td><td rowspan=1 colspan=1>7.41E+04</td><td rowspan=1 colspan=1>4.87</td><td rowspan=1 colspan=1>0.02</td><td rowspan=1 colspan=1>0.03</td><td rowspan=1 colspan=1>0.02</td><td rowspan=1 colspan=1>0.03</td></tr><tr><td rowspan=1 colspan=1>PM 4</td><td rowspan=1 colspan=1>1.00E+04</td><td rowspan=1 colspan=1>7.41E+03</td><td rowspan=1 colspan=1>3.87</td><td rowspan=1 colspan=1>0.03</td><td rowspan=1 colspan=1>0.03</td><td rowspan=1 colspan=1>0.03</td><td rowspan=1 colspan=1>0.03</td></tr><tr><td rowspan=1 colspan=1>PM 5</td><td rowspan=1 colspan=1>6.00E+03</td><td rowspan=1 colspan=1>4.44E+03</td><td rowspan=1 colspan=1>3.65</td><td rowspan=1 colspan=1>0.04</td><td rowspan=1 colspan=1>0.03</td><td rowspan=1 colspan=1>0.04</td><td rowspan=1 colspan=1>0.03</td></tr><tr><td rowspan=1 colspan=1>PM 6</td><td rowspan=1 colspan=1>1.00E+03</td><td rowspan=1 colspan=1>7.41E+02</td><td rowspan=1 colspan=1>2.87</td><td rowspan=1 colspan=1>0.05</td><td rowspan=1 colspan=1>0.05</td><td rowspan=1 colspan=1>0.04</td><td rowspan=1 colspan=1>0.05</td></tr><tr><td rowspan=1 colspan=1>PM 7</td><td rowspan=1 colspan=1>3.00E+02</td><td rowspan=1 colspan=1>2.22E+02</td><td rowspan=1 colspan=1>2.35</td><td rowspan=1 colspan=1>0.08</td><td rowspan=1 colspan=1>0.07</td><td rowspan=1 colspan=1>0.05</td><td rowspan=1 colspan=1>0.07</td></tr></table>

\* Titer data are considered to be log-normally distributed and are analyzed following log10 transformation. Standard deviations (SD) columns present the total of the log-transformed titer for each of the three reagent lots.

The variance component analysis demonstrated the contribution of the components of variance to the total precision variance (Table 2).   
Overall, the total precision as SD of the $\log _ { 1 0 }$ titer is comparable across the three kit lots tested and met the acceptance criteria.

Table 2: Lognormal Percent Coefficient of Variation (% CV) \*   

<table><tr><td rowspan=1 colspan=2>Nominal concentration(IU/mL)</td><td rowspan=1 colspan=2>Assignedconcentration(IU/mL)</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Instr. /Operator</td><td rowspan=1 colspan=1>BetweenLot</td><td rowspan=1 colspan=1>BetweenDay</td><td rowspan=1 colspan=1>BetweenRun</td><td rowspan=1 colspan=1>WithinRun</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Titer</td><td rowspan=1 colspan=1>Log10Titer</td><td rowspan=1 colspan=1>Titer</td><td rowspan=1 colspan=1>Log10Titer</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=6>%CV</td></tr><tr><td rowspan=1 colspan=1>1.00E+08</td><td rowspan=1 colspan=1>8.00</td><td rowspan=1 colspan=1>7.41E+07</td><td rowspan=1 colspan=1>7.87</td><td rowspan=1 colspan=1>57**</td><td rowspan=1 colspan=1>3%</td><td rowspan=1 colspan=1>5%</td><td rowspan=1 colspan=1>1%</td><td rowspan=1 colspan=1>1%</td><td rowspan=1 colspan=1>4%</td><td rowspan=1 colspan=1>7%</td></tr><tr><td rowspan=1 colspan=1>1.00E+06</td><td rowspan=1 colspan=1>6.00</td><td rowspan=1 colspan=1>7.41E+05</td><td rowspan=1 colspan=1>5.87</td><td rowspan=1 colspan=1>72</td><td rowspan=1 colspan=1>2%</td><td rowspan=1 colspan=1>6%</td><td rowspan=1 colspan=1>1%</td><td rowspan=1 colspan=1>2%</td><td rowspan=1 colspan=1>4%</td><td rowspan=1 colspan=1>8%</td></tr><tr><td rowspan=1 colspan=1>1.00E+05</td><td rowspan=1 colspan=1>5.00</td><td rowspan=1 colspan=1>7.41E+04</td><td rowspan=1 colspan=1>4.87</td><td rowspan=1 colspan=1>72</td><td rowspan=1 colspan=1>4%</td><td rowspan=1 colspan=1>7%</td><td rowspan=1 colspan=1>1%</td><td rowspan=1 colspan=1>2%</td><td rowspan=1 colspan=1>5%</td><td rowspan=1 colspan=1>10%</td></tr><tr><td rowspan=1 colspan=1>1.00E+04</td><td rowspan=1 colspan=1>4.00</td><td rowspan=1 colspan=1>7.41E+03</td><td rowspan=1 colspan=1>3.87</td><td rowspan=1 colspan=1>71</td><td rowspan=1 colspan=1>6%</td><td rowspan=1 colspan=1>9%</td><td rowspan=1 colspan=1>2%</td><td rowspan=1 colspan=1>1%</td><td rowspan=1 colspan=1>6%</td><td rowspan=1 colspan=1>12%</td></tr><tr><td rowspan=1 colspan=1>6.00E+03</td><td rowspan=1 colspan=1>3.78</td><td rowspan=1 colspan=1>4.44E+03</td><td rowspan=1 colspan=1>3.65</td><td rowspan=1 colspan=1>72</td><td rowspan=1 colspan=1>6%</td><td rowspan=1 colspan=1>7%</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>1%</td><td rowspan=1 colspan=1>7%</td><td rowspan=1 colspan=1>11%</td></tr><tr><td rowspan=1 colspan=1>1.00E+03</td><td rowspan=1 colspan=1>3.00</td><td rowspan=1 colspan=1>7.41E+02</td><td rowspan=1 colspan=1>2.87</td><td rowspan=1 colspan=1>72</td><td rowspan=1 colspan=1>3%</td><td rowspan=1 colspan=1>11%</td><td rowspan=1 colspan=1>2%</td><td rowspan=1 colspan=1>2%</td><td rowspan=1 colspan=1>11%</td><td rowspan=1 colspan=1>16%</td></tr><tr><td rowspan=1 colspan=1>3.00E+02</td><td rowspan=1 colspan=1>2.48</td><td rowspan=1 colspan=1>2.22E+02</td><td rowspan=1 colspan=1>2.35</td><td rowspan=1 colspan=1>70</td><td rowspan=1 colspan=1>5%</td><td rowspan=1 colspan=1>15%</td><td rowspan=1 colspan=1>5%</td><td rowspan=1 colspan=1>6%</td><td rowspan=1 colspan=1>15%</td><td rowspan=1 colspan=1>23%</td></tr></table>

\* Titer data are considered to be log-normally distributed and the $\% \mathrm { C V }$ values are analyzed as Lognormal CV $( \% ) =$ sqrt $( 1 0 \ ` \ : [ \mathrm { S D } \string ^ { \wedge } 2 \ : * \ : \ln \left( 1 0 \right) ] \cdot 1 \ : ) \ : ^ { * } \ : 1 0 0 \% .$ $^ { * * } 1 5 / 7 2$ replicates had results above the Upper Limit of Quantification and were excluded from the analysis.

# 2. Reproducibility Study:

Reproducibility of cobas BKV was evaluated in BKV PCR negative pooled urine stabilized in cobas PCR media (1:1 dilution) across three Reagent Lots, three Test Sites, three Instruments (two cobas 6800 Systems at two sites and one cobas 8800 System at the third site). Two runs were performed per operator per day ( $1 \mathrm { r u n } = 2$ batches; 1 batch $= 1$ panel $^ { + 3 }$ controls) over five days per reagent lot and each run had three replicates per panel member for a total of 270 replicates/concentration.

Test panel members were prepared from BKV PCR-negative pooled urine spiked with either the BKV WHO Standard or BKV genotype 1b genomic material in the form of BKV cell culture supernatant. Test panel members had the following concentrations: Negative, $6 0 0 \ : \mathrm { I U / m L }$ $( 3 \mathrm { x } \ L \ L \mathrm { o } \mathrm { Q } )$ , $5 \mathrm { x } 1 0 ^ { 3 } \mathrm { I U / m L }$ , $5 \mathrm { x } 1 0 ^ { 4 } \mathrm { I U / m L }$ , $5 \mathrm { x } 1 0 ^ { 5 } \mathrm { I U / m L }$ , and $5 \mathrm { x } 1 0 ^ { 7 } \mathrm { I U / m L }$ (0.5x ULoQ). One of the 92 batches tested failed due to operator error and one failed due to instrument error. Across the three testing sites, positive panel members produced 1,350 valid results. All 270 test results were valid for the negative panel members. The results are summarized in Table 3.

Table 3: Reproducibility Study   

<table><tr><td rowspan=1 colspan=1>Expected BKV DNAConcentration(log10 IU/mL)</td><td rowspan=1 colspan=1>ObservedMean² BKVDNAConcentration(log10 IU/mL)</td><td rowspan=1 colspan=1>Numberof Testsb</td><td rowspan=1 colspan=1>Lot%TVc(CV%)d</td><td rowspan=1 colspan=1>Site%TVc(CV%)d</td><td rowspan=1 colspan=1>Day/Operator%TVc(CV%)d</td><td rowspan=1 colspan=1>Batch%TVc(CV%)d</td><td rowspan=1 colspan=1>Within-Batch%TVc (CV%)d</td><td rowspan=1 colspan=1>TotalPrecisionSDe</td><td rowspan=1 colspan=1>TotalPrecisionLog-normalCV(%)</td></tr><tr><td rowspan=1 colspan=1>2.78</td><td rowspan=1 colspan=1>2.92</td><td rowspan=1 colspan=1>270</td><td rowspan=1 colspan=1>59%(12.64)</td><td rowspan=1 colspan=1>0%(1.15)</td><td rowspan=1 colspan=1>0%(0.00)</td><td rowspan=1 colspan=1>0%(0.00)</td><td rowspan=1 colspan=1>40%(10.41)</td><td rowspan=1 colspan=1>0.071</td><td rowspan=1 colspan=1>16.47</td></tr><tr><td rowspan=1 colspan=1>3.70</td><td rowspan=1 colspan=1>3.78</td><td rowspan=1 colspan=1>270</td><td rowspan=1 colspan=1>47%(8.14)</td><td rowspan=1 colspan=1>2%(1.62)</td><td rowspan=1 colspan=1>8%(3.31)</td><td rowspan=1 colspan=1>0%(0.00)</td><td rowspan=1 colspan=1>43%(7.72)</td><td rowspan=1 colspan=1>0.051</td><td rowspan=1 colspan=1>11.83</td></tr><tr><td rowspan=1 colspan=1>4.70</td><td rowspan=1 colspan=1>4.80</td><td rowspan=1 colspan=1>270</td><td rowspan=1 colspan=1>38%(5.02)</td><td rowspan=1 colspan=1>2%(1.28)</td><td rowspan=1 colspan=1>6%(2.07)</td><td rowspan=1 colspan=1>0%(0.00)</td><td rowspan=1 colspan=1>53%(5.96)</td><td rowspan=1 colspan=1>0.035</td><td rowspan=1 colspan=1>8.17</td></tr><tr><td rowspan=1 colspan=1>5.70</td><td rowspan=1 colspan=1>5.70</td><td rowspan=1 colspan=1>270</td><td rowspan=1 colspan=1>21%(3.12)</td><td rowspan=1 colspan=1>0%(0.00)</td><td rowspan=1 colspan=1>0%(0.00)</td><td rowspan=1 colspan=1>0%(0.00)</td><td rowspan=1 colspan=1>79%(6.12)</td><td rowspan=1 colspan=1>0.030</td><td rowspan=1 colspan=1>6.87</td></tr><tr><td rowspan=1 colspan=1>7.70</td><td rowspan=1 colspan=1>7.69</td><td rowspan=1 colspan=1>270</td><td rowspan=1 colspan=1>2%(1.51)</td><td rowspan=1 colspan=1>19%(4.84)</td><td rowspan=1 colspan=1>6%(2.79)</td><td rowspan=1 colspan=1>0%(0.00)</td><td rowspan=1 colspan=1>73%(9.53)</td><td rowspan=1 colspan=1>0.048</td><td rowspan=1 colspan=1>11.17</td></tr></table>

a Calculated using SAS MIXED procedure b Number of valid tests with detectable DNA level.

$\mathrm { { } c \ { } \% T V = }$ Percent contribution to Total Variance.   
${ } ^ { \mathrm { d } } \mathrm { C V } \% =$ Lognormal percent coefficient of variation $= \operatorname { s q r t } ( 1 0 ^ { \land } [ \operatorname { S D } ^ { \land } 2 ^ { * } \ln ( 1 0 ) ] - 1 ) \ ^ { * } \ 1 0 0$   
e Calculated using the total variability from the SAS MIXED procedure   
Note: The table only includes results with detectable DNA level. $\mathrm { S D } =$ standard deviation. $\mathrm { C V } =$ coefficient of variation; and $\mathrm { B K V } = \mathrm { B K }$ Virus

Analysis of variance and a mixed model that included lot, site, day/operator, batch and within-batch (random error) as random effects was performed. The variance contribution of each component to the total variance was estimated. The range of the total lognormal coefficient of variation, among positive panel members, ranged from $6 . 8 7 \%$ to $1 6 . 4 7 \%$ . The largest total lognormal coefficient of variation was observed in the lowest positive panel member (3x LLoQ). Most of that variability ( $5 9 \%$ of the total variance) was attributed to lot variability at one clinical testing site. High within-batch variability was also observed for panel members with BKV DNA tested at 5.7 and $7 . 7 \mathrm { \ : l o g _ { 1 0 } I U / m L }$ across the three clinical testing sites and may be due to the samples being tested close to the ULoQ $0 . 5 \mathrm { x }$ ULoQ).

# 3. Linearity:

Two linearity panels were tested: one panel composed of a dilution series of 12 panel members was prepared by diluting a BKV subgroup 1b (GT 1b) lambda DNA (phagemid) in a BKV-negative urine pool, and one panel composed of a dilution series of 10 panel members was prepared by diluting a subgroup 1b clinical specimen in a BKV-negative urine pool. Both panels were tested at each concentration level (Table 4). The highest assigned titer for the phagemid panel was $7 . 4 1 \mathrm { E } { + } 0 8 \ \mathrm { I U / m L }$ and the highest assigned titer for the clinical panel was $9 . 2 2 \mathrm { E } \mathrm { + } 0 6 \mathrm { I U / m L }$ . The phagemid and clinical panel members overlapped in the measuring range, from a nominal titer of $1 . 0 0 \mathrm { E } { + 0 7 } \ : \mathrm { I U / m L }$ to $1 . 0 0 \mathrm { E } { + 0 2 } \ : \mathrm { I U / m L }$ (below the LLoQ).

Table 4: Assigned Titer of the BKV Linearity Panel   

<table><tr><td colspan="1" rowspan="1">Panel Member</td><td colspan="1" rowspan="1">Nominal Titer(IU/mL)</td><td colspan="1" rowspan="1">Assigned Titer(IU/mL)</td><td colspan="1" rowspan="1">Sample Type</td></tr><tr><td colspan="1" rowspan="1">PM01</td><td colspan="1" rowspan="1">1.00E+09</td><td colspan="1" rowspan="1">7.41E+08</td><td colspan="1" rowspan="1">Phagemid</td></tr><tr><td colspan="1" rowspan="1">PM02</td><td colspan="1" rowspan="1">1.00E+08</td><td colspan="1" rowspan="1">7.41E+07</td><td colspan="1" rowspan="1">Phagemid</td></tr><tr><td colspan="1" rowspan="2">PM03</td><td colspan="1" rowspan="2">1.00E+07</td><td colspan="1" rowspan="1">9.22E+06</td><td colspan="1" rowspan="1">Clinical</td></tr><tr><td colspan="1" rowspan="1">7.41E+06</td><td colspan="1" rowspan="1">Phagemid</td></tr><tr><td colspan="1" rowspan="2">PM04</td><td colspan="1" rowspan="2">1.00E+06</td><td colspan="1" rowspan="1">9.22E+05</td><td colspan="1" rowspan="1">Clinical</td></tr><tr><td colspan="1" rowspan="1">7.41E+05</td><td colspan="1" rowspan="1">Phagemid</td></tr><tr><td colspan="1" rowspan="2">PM05</td><td colspan="1" rowspan="2">1.00E+05</td><td colspan="1" rowspan="1">9.22E+04</td><td colspan="1" rowspan="1">Clinical</td></tr><tr><td colspan="1" rowspan="1">7.41E+04</td><td colspan="1" rowspan="1">Phagemid</td></tr><tr><td colspan="1" rowspan="2">PM06</td><td colspan="1" rowspan="2">1.00E+04</td><td colspan="1" rowspan="1">9.22E+03</td><td colspan="1" rowspan="1">Clinical</td></tr><tr><td colspan="1" rowspan="1">7.41E+03</td><td colspan="1" rowspan="1">Phagemid</td></tr><tr><td colspan="1" rowspan="2">PM07</td><td colspan="1" rowspan="2">6.00E+03</td><td colspan="1" rowspan="1">5.53+E03</td><td colspan="1" rowspan="1">Clinical</td></tr><tr><td colspan="1" rowspan="1">4.44E+03</td><td colspan="1" rowspan="1">Phagemid</td></tr><tr><td colspan="1" rowspan="2">PM08</td><td colspan="1" rowspan="2">1.00E+03</td><td colspan="1" rowspan="1">9.22E+02</td><td colspan="1" rowspan="1">Clinical</td></tr><tr><td colspan="1" rowspan="1">7.41E+02</td><td colspan="1" rowspan="1">Phagemid</td></tr><tr><td colspan="1" rowspan="2">PM09</td><td colspan="1" rowspan="2">5.00E+02</td><td colspan="1" rowspan="1">4.61E+02</td><td colspan="1" rowspan="1">Clinical</td></tr><tr><td colspan="1" rowspan="1">3.70E+02</td><td colspan="1" rowspan="1">Phagemid</td></tr><tr><td colspan="1" rowspan="2">PM10</td><td colspan="1" rowspan="2">3.00E+02</td><td colspan="1" rowspan="1">2.77E+02</td><td colspan="1" rowspan="1">Clinical</td></tr><tr><td colspan="1" rowspan="1">2.22E+02</td><td colspan="1" rowspan="1">Phagemid</td></tr><tr><td colspan="1" rowspan="2">PM11</td><td colspan="1" rowspan="2">2.00E+02</td><td colspan="1" rowspan="1">1.84E+02</td><td colspan="1" rowspan="1">Clinical</td></tr><tr><td colspan="1" rowspan="1">1.48E+02</td><td colspan="1" rowspan="1">Phagemid</td></tr><tr><td colspan="1" rowspan="2">PM12</td><td colspan="1" rowspan="2">1.00E+02</td><td colspan="1" rowspan="1">9.22E+01</td><td colspan="1" rowspan="1">Clinical</td></tr><tr><td colspan="1" rowspan="1">7.41E+01</td><td colspan="1" rowspan="1">Phagemid</td></tr></table>

Testing was performed using three cobas 8800 instruments by three operators on 12 replicates for each concentration using three kit lots (a total of 36 replicates per concentration). Nine runs were performed, all of which were valid and yielded a total of 792 valid results. Results were analyzed according to CLSI standard EP06-A.

The parameter estimates of b0, b1, b2, and b3 are presented for $1 ^ { \mathrm { s t } }$ order linear and $3 ^ { \mathrm { r d } }$ order polynomial fits (Table 5). For all the test kits and sample types combined, both the higher order coefficients (b2 and b3) were significant at $5 \%$ level. The $3 ^ { \mathrm { r d } }$ order fit was chosen as the best fitting regression model as it had the smaller Root Mean Square Error (Figure 1). The maximal difference between the linear and $3 ^ { \mathrm { r d } }$ order fits was observed for the lowest clinical panel member with a nominal titer of $1 . 0 0 \mathrm { E } { + } 0 2 \ \mathrm { I U / m L }$ (below the LLoQ of the assay).

Table 5: cobas BKV Linearity with BKV Genotype 1b   

<table><tr><td rowspan=1 colspan=1>Sampletype</td><td rowspan=1 colspan=1>Lot</td><td rowspan=1 colspan=1>Coefficientlinearregression(b0, Std.error)</td><td rowspan=1 colspan=1>Coefficientlinearregression(b1, Std.error)</td><td rowspan=1 colspan=1>Coefficient forbetter  fittinghiger ordermodelregression(b0, Std. error)</td><td rowspan=1 colspan=1>Coefficient forbettr fittinghigher ordermodelregression(b1, Std. error)</td><td rowspan=1 colspan=1>Coefficient forbetter fittinghigher orrdermodel regression(b2, Std. error)</td><td rowspan=1 colspan=1>Coefficient forbettr fittinghigher ordermodelregression(b3, Std. error)</td><td rowspan=1 colspan=1>Maximumdifference*(log1o/mL)</td></tr><tr><td rowspan=1 colspan=1>Clinical</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.06, 0.02</td><td rowspan=1 colspan=1>1.00, 0.00</td><td rowspan=1 colspan=1>0.48, 0.14</td><td rowspan=1 colspan=1>0.69, 0.11</td><td rowspan=1 colspan=1>0.07, 0.03</td><td rowspan=1 colspan=1>0.00, 0.00</td><td rowspan=1 colspan=1>0.04</td></tr><tr><td rowspan=1 colspan=1>Clinical</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0.22, 0.01</td><td rowspan=1 colspan=1>0.98, 0.00</td><td rowspan=1 colspan=1>1.05, 0.10</td><td rowspan=1 colspan=1>0.38, 0.08</td><td rowspan=1 colspan=1>0.13, 0.02</td><td rowspan=1 colspan=1>-0.01, 0.00</td><td rowspan=1 colspan=1>0.08</td></tr><tr><td rowspan=1 colspan=1>Clinical</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0.15, 0.01</td><td rowspan=1 colspan=1>0.99, 0.00</td><td rowspan=1 colspan=1>0.62, 0.11</td><td rowspan=1 colspan=1>0.66, 0.09</td><td rowspan=1 colspan=1>0.07, 0.02</td><td rowspan=1 colspan=1>0.00, 0.00</td><td rowspan=1 colspan=1>0.050</td></tr><tr><td rowspan=1 colspan=1>Phagemid</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.11, 0.01</td><td rowspan=1 colspan=1>1.00, 0.00</td><td rowspan=1 colspan=1>0.35, 0.06</td><td rowspan=1 colspan=1>0.84, 0.04</td><td rowspan=1 colspan=1>0.03,0.01</td><td rowspan=1 colspan=1>0.00, 0.00</td><td rowspan=1 colspan=1>0.04</td></tr><tr><td rowspan=1 colspan=1>Phagemid</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0.21, 0.01</td><td rowspan=1 colspan=1>0.99,0.00</td><td rowspan=1 colspan=1>0.66, 0.06</td><td rowspan=1 colspan=1>0.69, 0.05</td><td rowspan=1 colspan=1>0.06, 0.01</td><td rowspan=1 colspan=1>0.00, 0.00</td><td rowspan=1 colspan=1>0.06</td></tr><tr><td rowspan=1 colspan=1>Phagemid</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0.15, 0.01</td><td rowspan=1 colspan=1>0.99, 0.00</td><td rowspan=1 colspan=1>0.67, 0.05</td><td rowspan=1 colspan=1>0.44, 0.04</td><td rowspan=1 colspan=1>0.07, 0.01</td><td rowspan=1 colspan=1>0.00, 0.00</td><td rowspan=1 colspan=1>0.08</td></tr><tr><td rowspan=1 colspan=1>Combined</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.08, 0.01</td><td rowspan=1 colspan=1>1.00, 0.00</td><td rowspan=1 colspan=1>0.35, 0.05</td><td rowspan=1 colspan=1>0.82, 0.04</td><td rowspan=1 colspan=1>0.03, 0.01</td><td rowspan=1 colspan=1>0.00,0.00</td><td rowspan=1 colspan=1>0.04</td></tr><tr><td rowspan=1 colspan=1>Combined</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0.21, 0.01</td><td rowspan=1 colspan=1>0.99, 0.00</td><td rowspan=1 colspan=1>0.72, 0.05</td><td rowspan=1 colspan=1>0.64, 0.03</td><td rowspan=1 colspan=1>0.07, 0.01</td><td rowspan=1 colspan=1>0.00, 0.00</td><td rowspan=1 colspan=1>0.08</td></tr><tr><td rowspan=1 colspan=1>Combined</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0.14, 0.01</td><td rowspan=1 colspan=1>0.99, 0.01</td><td rowspan=1 colspan=1>0.61, 0.04</td><td rowspan=1 colspan=1>0.68, 0.03</td><td rowspan=1 colspan=1>0.06, 0.01</td><td rowspan=1 colspan=1>0.00, 0.00</td><td rowspan=1 colspan=1>0.07</td></tr><tr><td rowspan=1 colspan=1>Combined</td><td rowspan=1 colspan=1>Alllots</td><td rowspan=1 colspan=1>0.15, 0.01</td><td rowspan=1 colspan=1>0.99, 0.00</td><td rowspan=1 colspan=1>0.56, 0.03</td><td rowspan=1 colspan=1>0.72, 0.02</td><td rowspan=1 colspan=1>0.05, 0.00</td><td rowspan=1 colspan=1>0.00, 0.00</td><td rowspan=1 colspan=1>0.06</td></tr></table>

\* Maximum difference between linear regression and the better fitting higher order model

![](images/01947db94028133d7ad1bb920b9df772ad8233817f9717835d46dace80d23059.jpg)  
Figure 1: Linearity Across Both Panel Types and All Kit Lots Combined

No bias was found between the $\log _ { 1 0 }$ titers for the Phagemid and Clinical panel members when analyzed separately. The linearity across both panel types (phagemid and clinical sample) and all kit lots combined is shown in Figure 1.

The linear range of cobas BKV, defined as the concentration range for which the deviation of predicted log10 titer of the better fitting regression (2nd or 3rd order) and the predicted log10 titer of the linear regression (1st order) is within $\pm 0 . 3 \log _ { 1 0 }$ was determined.

The observed linear range of the cobas BKV assay with urine is $7 . 4 0 \mathrm { E } { + } 0 1 $ to $7 . 4 1 \mathrm { E } \mathrm { + } 0 8 \mathrm { I U / m L }$ . The claimed linear range of the cobas BKV assay with urine is $2 . 0 0 \mathrm { E } { + } 0 2 \mathrm { I U / m L }$ to $1 . 0 \mathrm { E } { + 0 8 }$ $\mathrm { I U / m L }$ .

4. Verification of Linearity of cobas BKV with other genotypes:

Linearity was assessed for five BKV subgroup/subtype panels, one for each genotype, spanning the linear range from the LLoQ of $2 . 0 0 \mathrm { E } { + } 0 2 \ \mathrm { I U / m L }$ to the ULoQ of $1 . 0 0 \mathrm { E + } 0 8 \mathrm { I U / m L }$ .

Intermediate Stock Solutions (ISS) for each genotype were verified using either the QIAGEN artus BK virus QS-RGQ PCR kit (subgroup Ia and phagemids of subtypes II, III and IV) or by digital droplet PCR (subgroup Ic).

Serial dilutions ranging from a nominal titer of $1 . 0 0 \mathrm { E + } 0 8 \mathrm { I U / m L }$ (ULoQ) to $2 . 0 0 \mathrm { E + } 0 1 \ \mathrm { I U / m L }$ (LLoQ) were made from BKV subgroup Ia cell culture supernatant, phagemids of BKV subgroup Ic, and subtypes II, III, and IV. The dilutions were prepared in BKV-negative pooled urine stabilized in cobas PCR media, yielding an 8-member panel. Four replicates were tested per panel member. Subgroup Ia was tested using three kit lots and two cobas 6800/8800 Systems. Subgroup Ic, and subtypes II, III and IV, were tested using three kit lots and one cobas 6800/8800 System. Nine runs were performed of which all nine were valid yielding a total of 480 test results.

Linear ( $1 ^ { \mathrm { s t } }$ order) and nonlinear, $2 ^ { \mathrm { n d } }$ order and $3 ^ { \mathrm { r d } }$ order, regression fits were performed for all genotypes tested for all lots combined. The Root Mean Square Error (MSE) of the $3 ^ { \mathrm { r d } }$ order was smallest and was chosen as the best fitting regression model to calculate the absolute difference between the predicted log titer obtained from the $1 ^ { \mathrm { s t } }$ and $3 ^ { \mathrm { r d } }$ order regression fits. Both higher order coefficients (b2 and b3) were significant at $5 \%$ level.

The linear range of each BKV genotype (subgroups Ia, Ic, and subtypes II, III, and IV) was established from $2 . 0 0 \mathrm { E } { + } 0 2 \ \mathrm { I U / m L }$ to $1 . 0 0 \mathrm { E + } 0 8 \mathrm { I U / m L }$ , the same as that of the predominant subgroup Ib (Table 6). For all genotypes tested, the absolute deviation was $\leq 0 . 2 1 \ \mathrm { l o g } _ { 1 0 } \ \mathrm { I U } / \mathrm { m L }$ at all concentration levels.

Table 6: Equations for the Best Fitting Regression Models for BKV Genotypes   

<table><tr><td rowspan=1 colspan=1>Subgroup/Subtype</td><td rowspan=1 colspan=1>Linear regression</td><td rowspan=1 colspan=1>Better fitting higher order model regression</td><td rowspan=1 colspan=1>Maximumdifference*(log10 IU/mL)</td></tr><tr><td rowspan=1 colspan=1>Ia</td><td rowspan=1 colspan=1>y = 0.976 x + 0.237</td><td rowspan=1 colspan=1>y = -0.008 x3 + 0.137 x2 + 0.260 x + 1.351</td><td rowspan=1 colspan=1>0.12</td></tr><tr><td rowspan=1 colspan=1>Ic</td><td rowspan=1 colspan=1>y= 0.977 x + 0.205</td><td rowspan=1 colspan=1>y = -0.005 x3 + 0.090 x2 + 0.487 x + 0.993</td><td rowspan=1 colspan=1>0.10</td></tr><tr><td rowspan=1 colspan=1>I</td><td rowspan=1 colspan=1>y = 0.976 x + 0.227</td><td rowspan=1 colspan=1>y = -0.007 x3 + 0.126 x2 + 0.303 x + 1.297</td><td rowspan=1 colspan=1>0.12</td></tr><tr><td rowspan=1 colspan=1>III</td><td rowspan=1 colspan=1>y = 0.976 x + 0.227</td><td rowspan=1 colspan=1>y = -0.008 x3 + 0.137 x2 + 0. 265 x + 1.328</td><td rowspan=1 colspan=1>0.12</td></tr><tr><td rowspan=1 colspan=1>IV</td><td rowspan=1 colspan=1>y = 0.976 x + 0.203</td><td rowspan=1 colspan=1>y = -0.009 x3 + 0.143 x2 + 0.250 x + 1.293</td><td rowspan=1 colspan=1>0.13</td></tr></table>

\* Maximum difference between linear regression and the better fitting higher order model

# 5. Lower Limit of Quantitation (LLoQ):

The analysis of the Lower Limit of Quantitation was performed with data obtained from the Linearity study using the WHO standard. The LLoQ is the lowest titer within the linear range with a hit rate of at least $9 5 \%$ that meets the acceptance criterion for the Total Analytical Error (TAE) and the “Difference between Measurements in SD” approach. The acceptance criteria for TAE are:

1. The TAE, when calculated as $| \mathrm { B i a s } | + 2 \mathrm { S D }$ , is $\leq 1 . 0 \log _ { 1 0 } \mathrm { I U / m L }$ , and 2. The TAE has to be such that the standard deviation for the difference between two measurements calculated as SQRT(2) $\textbf { x } 2 \textbf { x }$ SD is $\leq 1 . 0 \log _ { 1 0 } \mathrm { I U / m L }$

The panel members at the lowest nominal concentration of $1 . 0 0 \mathrm { E } { + } 0 2 \ \mathrm { I U / m L }$ for both phagemid and clinical samples met the TAE and “Difference between the Measurements” criteria. For the clinical panel members in the linearity study, the maxim deviation of $0 . 0 8 \mathrm { L o g } _ { 1 0 } \mathrm { I U } / \mathrm { m L }$ between the linear regression and the better fitting higher order model was obtained at the nominal concentration of $1 . 0 0 \mathrm { E } { + } 0 2 \mathrm { I U } / \mathrm { m L }$ . However, the LLoQ was set at the concentration level of $2 . 0 0 \mathrm { E } { + } 0 2 \mathrm { I U / m L }$ to include the mean deviation between the observed vs. the assigned $\log _ { 1 0 }$ titer (Accuracy) being equal or less than $\pm 0 . 3 \log _ { 1 0 }$ , based on the upper $9 5 \%$ confidence interval of the worst performing lot (Lot#2, ED3280) using clinical specimen. Table 7 presents the LLoQ calculated for each kit lot and combined across all the kit lots.

Table 7: LLOQ - TAE and Difference between Measurements.   

<table><tr><td rowspan=1 colspan=1>Lot(#)</td><td rowspan=1 colspan=1>NominalTiter(IU/mL)</td><td rowspan=1 colspan=1>HitRate(%)</td><td rowspan=1 colspan=1>AssignedLog10Titer</td><td rowspan=1 colspan=1>MeanObservedTiter(log10IU/mL)</td><td rowspan=1 colspan=1>SD(log10IU/mL)</td><td rowspan=1 colspan=1>AbsoluteBias(log10IU/mL)</td><td rowspan=1 colspan=1>TAE(l0g10IU/mL)</td><td rowspan=1 colspan=1>DifferenceBetweenMeasurementin SD</td></tr><tr><td rowspan=3 colspan=1>1(ED3278)</td><td rowspan=1 colspan=1>3.00E+02</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>2.44</td><td rowspan=1 colspan=1>2.50</td><td rowspan=1 colspan=1>0.05</td><td rowspan=1 colspan=1>0.06</td><td rowspan=1 colspan=1>0.15</td><td rowspan=1 colspan=1>0.13</td></tr><tr><td rowspan=1 colspan=1>2.00E+02</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>2.27</td><td rowspan=1 colspan=1>2.37</td><td rowspan=1 colspan=1>0.07</td><td rowspan=1 colspan=1>0.11</td><td rowspan=1 colspan=1>0.25</td><td rowspan=1 colspan=1>0.21</td></tr><tr><td rowspan=1 colspan=1>1.00E+02</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>1.96</td><td rowspan=1 colspan=1>2.06</td><td rowspan=1 colspan=1>0.11</td><td rowspan=1 colspan=1>0.10</td><td rowspan=1 colspan=1>0.32</td><td rowspan=1 colspan=1>0.32</td></tr><tr><td rowspan=3 colspan=1>2(ED3280)</td><td rowspan=1 colspan=1>3.00E+02</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>2.44</td><td rowspan=1 colspan=1>2.63</td><td rowspan=1 colspan=1>0.05</td><td rowspan=1 colspan=1>0.19</td><td rowspan=1 colspan=1>0.29</td><td rowspan=1 colspan=1>0.14</td></tr><tr><td rowspan=1 colspan=1>2.00E+02</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>2.27</td><td rowspan=1 colspan=1>2.49</td><td rowspan=1 colspan=1>0.06</td><td rowspan=1 colspan=1>0.22</td><td rowspan=1 colspan=1>0.34</td><td rowspan=1 colspan=1>0.17</td></tr><tr><td rowspan=1 colspan=1>1.00E+02</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>1.96</td><td rowspan=1 colspan=1>2.22</td><td rowspan=1 colspan=1>0.09</td><td rowspan=1 colspan=1>0.26</td><td rowspan=1 colspan=1>0.44</td><td rowspan=1 colspan=1>0.25</td></tr><tr><td rowspan=3 colspan=1>3(ED3284)</td><td rowspan=1 colspan=1>3.00E+02</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>2.44</td><td rowspan=1 colspan=1>2.58</td><td rowspan=1 colspan=1>0.07</td><td rowspan=1 colspan=1>0.13</td><td rowspan=1 colspan=1>0.27</td><td rowspan=1 colspan=1>0.20</td></tr><tr><td rowspan=1 colspan=1>2.00E+02</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>2.27</td><td rowspan=1 colspan=1>2.41</td><td rowspan=1 colspan=1>0.07</td><td rowspan=1 colspan=1>0.15</td><td rowspan=1 colspan=1>0.28</td><td rowspan=1 colspan=1>0.19</td></tr><tr><td rowspan=1 colspan=1>1.00E+02</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>1.96</td><td rowspan=1 colspan=1>2.14</td><td rowspan=1 colspan=1>0.09</td><td rowspan=1 colspan=1>0.17</td><td rowspan=1 colspan=1>0.36</td><td rowspan=1 colspan=1>0.26</td></tr><tr><td rowspan=3 colspan=1>Acrosslots</td><td rowspan=1 colspan=1>3.00E+02</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>2.44</td><td rowspan=1 colspan=1>2.57</td><td rowspan=1 colspan=1>0.08</td><td rowspan=1 colspan=1>0.13</td><td rowspan=1 colspan=1>0.28</td><td rowspan=1 colspan=1>0.22</td></tr><tr><td rowspan=1 colspan=1>2.00E+02</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>2.27</td><td rowspan=1 colspan=1>2.42</td><td rowspan=1 colspan=1>0.08</td><td rowspan=1 colspan=1>0.16</td><td rowspan=1 colspan=1>0.32</td><td rowspan=1 colspan=1>0.23</td></tr><tr><td rowspan=1 colspan=1>1.00E+02</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>1.96</td><td rowspan=1 colspan=1>2.14</td><td rowspan=1 colspan=1>0.12</td><td rowspan=1 colspan=1>0.18</td><td rowspan=1 colspan=1>0.41</td><td rowspan=1 colspan=1>0.33</td></tr></table>

# 6. Analytical Specificity/Interference:

# Cross reactivity

For potential cross reactants 34 microorganisms, including two viral isolates, 27 bacterial strains, four fungal isolates and one cultured protozoa were used and divided into eight pools with two to five microorganisms per cross reactant pool. Mycoplasma genitalium (MGE) and Ureaplasma urealyticum (UUR) were used as single interferents.

Potential cross reactants in BKV negative pooled urine stabilized with cobas PCR media were tested in the absence and presence of BKV DNA at a concentration of $6 0 0 \ : \mathrm { I U / m L }$ (approximately 3x LLoQ). Potential cross reactants were tested at $1 \mathrm { x } 1 0 ^ { 6 }$ copies/mL, $\mathrm { I U / m L }$ , $\mathrm { T C I D 5 0 / m L }$ , cells/mL, $\mathrm { C F U / m L }$ , IFU/mL or $\mathrm { C C U / m L }$ for each organism as appropriate except for HPV and HSV-2, which were tested at $1 \mathrm { x } 1 0 ^ { 5 }$ . MGE and UUR was tested individually at $2 \mathrm { x } 1 0 ^ { 6 } \mathrm { C C U / m L }$ . Testing was performed using six replicates per cross reactive pool or individual interferents, 28 replicates of the BKV-positive and 24 replicates for the BKV - negative controls using three kit lots and one cobas 8800 system.

For BKV negative samples the negativity rate was determined. Four samples from Pools 3 and 5 spiked in BKV-negative pooled urine were positive for BKV DNA. Although the individual urine samples used for preparing the BKV-negative pooled urine were PCR tested to confirm non-reactivity, the individual donors were not screened for sero-negativity. Thus, to resolve the discrepancy, cross reactants in Pools 3 and 5 were tested individually in a second pool of confirmed BKV-negative urine (Table 8). All BKV-negative specimens spiked with cross reactants tested negative for BKV DNA.

For BKV-positive samples the positivity rate was determined together with the correct quantitation of BKV DNA by computing the Mean concentration detected across the replicates, the SD and the difference between the control condition (no cross reactant) and K203220 - Page 16 of 29 the test condition containing the potential cross reactant organism. Results are presented in Table 8 below for testing of interferents.

<table><tr><td colspan="7" rowspan="1">Table 8: Testing Results of Cross reactivity in Pools of Microorganisms</td></tr><tr><td colspan="1" rowspan="2">Pool</td><td colspan="1" rowspan="2">Organisms</td><td colspan="1" rowspan="1">No BKV</td><td colspan="4" rowspan="1">With BKV</td></tr><tr><td colspan="1" rowspan="1">NegativityRate</td><td colspan="1" rowspan="1">PositivityRate</td><td colspan="1" rowspan="1">Mean(log 10titer</td><td colspan="1" rowspan="1">SD(log 10titer)</td><td colspan="1" rowspan="1">MeanDifferencein Log10Titer(Interferent −Control)</td></tr><tr><td colspan="1" rowspan="5">1</td><td colspan="1" rowspan="1">Bacillus cereus</td><td colspan="1" rowspan="5">6/6100%</td><td colspan="1" rowspan="5">6/6100%</td><td colspan="1" rowspan="5">2.93</td><td colspan="1" rowspan="5">0.19</td><td colspan="1" rowspan="5">0.15</td></tr><tr><td colspan="1" rowspan="1">Bacillus subtilis</td></tr><tr><td colspan="1" rowspan="1">Candida albicans</td></tr><tr><td colspan="1" rowspan="1">Candida glabrata</td></tr><tr><td colspan="1" rowspan="1">Candida parapsilosis</td></tr><tr><td colspan="1" rowspan="5">2</td><td colspan="1" rowspan="1">Candida tropicalis</td><td colspan="1" rowspan="5">6/6100%</td><td colspan="1" rowspan="5">6/6100%</td><td colspan="1" rowspan="5">2.92</td><td colspan="1" rowspan="5">0.06</td><td colspan="1" rowspan="5">0.15</td></tr><tr><td colspan="1" rowspan="1">Neisseria gonorrhoeae</td></tr><tr><td colspan="1" rowspan="1">Corynebacteriumdiphteriae</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td></tr><tr><td colspan="1" rowspan="1">Enterobacter cloacae</td></tr><tr><td colspan="1" rowspan="5">3</td><td colspan="1" rowspan="1">Enterococcus faecalis</td><td colspan="1" rowspan="1">6/6100%</td><td colspan="1" rowspan="5">6/61000%</td><td colspan="1" rowspan="5">2.79</td><td colspan="1" rowspan="5">0.06</td><td colspan="1" rowspan="5">0.02</td></tr><tr><td colspan="1" rowspan="1">Enterococcus faecium</td><td colspan="1" rowspan="1">6/6100%</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">6/6100%</td></tr><tr><td colspan="1" rowspan="1">Lactobacillusacidophilus</td><td colspan="1" rowspan="1">6/6100%</td></tr><tr><td colspan="1" rowspan="1">Treponema pallidum</td><td colspan="1" rowspan="1">6/6100%</td></tr><tr><td colspan="1" rowspan="3">4</td><td colspan="1" rowspan="1">Lactobacilluscrispatus</td><td colspan="1" rowspan="3">6/6100%</td><td colspan="1" rowspan="3">6/6100%</td><td colspan="1" rowspan="3">2.80</td><td colspan="1" rowspan="3">0.03</td><td colspan="1" rowspan="3">0.02</td></tr><tr><td colspan="1" rowspan="1">Trichomonas vaginalis</td></tr><tr><td colspan="1" rowspan="1">Chlamydiatrachomatis</td></tr><tr><td colspan="1" rowspan="5">5</td><td colspan="1" rowspan="1">Lactobacillus jensenii</td><td colspan="1" rowspan="5">6/6100%</td><td colspan="1" rowspan="5">6/6100%</td><td colspan="1" rowspan="5">2.81</td><td colspan="1" rowspan="5">0.03</td><td colspan="1" rowspan="5">0.03</td></tr><tr><td colspan="1" rowspan="1">Lactobacillusvaginalis</td></tr><tr><td colspan="1" rowspan="1">Morganella morganii</td></tr><tr><td colspan="1" rowspan="1">Pseudomonasaeruginosa</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td></tr><tr><td colspan="1" rowspan="2">6</td><td colspan="1" rowspan="1">Staphylococcussaphrophyticus</td><td colspan="1" rowspan="2">6/6100%</td><td colspan="1" rowspan="2">6/6100%</td><td colspan="1" rowspan="2">2.81</td><td colspan="1" rowspan="2">0.06</td><td colspan="1" rowspan="2">0.04</td></tr><tr><td colspan="1" rowspan="1">Streptococcusagalactiae</td></tr><tr><td colspan="1" rowspan="2">Pool</td><td colspan="1" rowspan="2">Organisms</td><td colspan="1" rowspan="1">No BKV</td><td colspan="4" rowspan="1">With BKV</td></tr><tr><td colspan="1" rowspan="1">NegativityRate</td><td colspan="1" rowspan="1">PositivityRate</td><td colspan="1" rowspan="1">Mean(log 10titer</td><td colspan="1" rowspan="1">SD(log 10titer</td><td colspan="1" rowspan="1">MeanDifferencein Log10Titer(Interferent -Control)</td></tr><tr><td colspan="1" rowspan="3">b</td><td colspan="1" rowspan="1">Streptococcus bovis</td><td colspan="1" rowspan="3"></td><td colspan="1" rowspan="3"></td><td colspan="1" rowspan="3"></td><td colspan="1" rowspan="3">−</td><td colspan="1" rowspan="3"></td></tr><tr><td colspan="1" rowspan="1">Streptococcuspneumoniae</td></tr><tr><td colspan="1" rowspan="1">Streptococcusoralis/viridans</td></tr><tr><td colspan="1" rowspan="2">7</td><td colspan="1" rowspan="1">Proteus mirabilis</td><td colspan="1" rowspan="2">6/6100%</td><td colspan="1" rowspan="2">6/6100%</td><td colspan="1" rowspan="2">2.82</td><td colspan="1" rowspan="2">0.05</td><td colspan="1" rowspan="2">0.04</td></tr><tr><td colspan="1" rowspan="1">Staphylococcusepidermidis</td></tr><tr><td colspan="1" rowspan="2">8</td><td colspan="1" rowspan="1">Herpes Simplex Virus-2 (HS2)</td><td colspan="1" rowspan="1">6/6100%</td><td colspan="1" rowspan="2">6/6100%</td><td colspan="1" rowspan="2">2.83</td><td colspan="1" rowspan="2">0.08</td><td colspan="1" rowspan="2">0.06</td></tr><tr><td colspan="1" rowspan="1">HumanPapillomavirus 16 or18 (HPV)</td><td colspan="1" rowspan="1">6/6100%</td></tr><tr><td colspan="1" rowspan="1">UUR</td><td colspan="1" rowspan="1">Ureaplasmaurealyticum</td><td colspan="1" rowspan="1">6/6100%</td><td colspan="1" rowspan="1">6/6100%</td><td colspan="1" rowspan="1">2.80</td><td colspan="1" rowspan="1">0.07</td><td colspan="1" rowspan="1">0.02</td></tr><tr><td colspan="1" rowspan="1">MGE</td><td colspan="1" rowspan="1">Mycoplasmagenitalium</td><td colspan="1" rowspan="1">6/6100%</td><td colspan="1" rowspan="1">6/6100%</td><td colspan="1" rowspan="1">2.76</td><td colspan="1" rowspan="1">0.07</td><td colspan="1" rowspan="1">-0.02</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Control (BKVnegative)</td><td colspan="1" rowspan="1">12/12100%</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Control (BKVpositive</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">20/20100%</td><td colspan="1" rowspan="1">2.77</td><td colspan="1" rowspan="1">0.05</td><td colspan="1" rowspan="1">N/A</td></tr></table>

The mean $\log _ { 1 0 }$ titers of the positive BKV samples containing potentially cross-reacting organisms were within $: \pm 0 . 5 \log _ { 1 0 }$ of the mean $\log _ { 1 0 }$ titer of the respective positive spike control.

# Endogenous Interference

The effect of potentially interfering endogenous substances on the sensitivity/quantitation of cobas BKV was determined by testing 20 individual BKV-negative urine samples spiked with selected endogenous substances and tested in the presence of BKV DNA at a concentration of $6 0 0 \ : \mathrm { I U / m L }$ (approximately 3x LLoQ). The negative sample spiked solely with BKV target was used as a Positive Spike Control (PSC). To analyze specificity, the same 20 individual clinical negative samples were individually spiked with potentially interfering endogenous substances and tested in the absence of BKV target DNA. The unspiked samples were used as Negative Spiked Controls (NSC). Ten potentially interfering substances were tested. Control conditions were tested with one replicate per specimen, and test conditions were tested with three replicates per specimen. All specimens were stabilized in cobas PCR media (1:1 ratio) before testing. Results are summarized in Table 9.

Table 9: Endogenous Interference   

<table><tr><td rowspan=2 colspan=1>Interferent</td><td rowspan=2 colspan=1>C1</td><td rowspan=1 colspan=1>No BKV</td><td rowspan=1 colspan=5>With BKV [100 IU/mL]</td></tr><tr><td rowspan=1 colspan=1>NegativityRate</td><td rowspan=1 colspan=1>PositivityRate</td><td rowspan=1 colspan=1>MeanCt</td><td rowspan=1 colspan=1>Mean(log10titer</td><td rowspan=1 colspan=1>SD(log1o titer)</td><td rowspan=1 colspan=1>Meandifferencein Log10Titer</td></tr><tr><td rowspan=1 colspan=1>Albumin</td><td rowspan=1 colspan=1>0.05%</td><td rowspan=1 colspan=1>20/20100%</td><td rowspan=1 colspan=1>60/60100%</td><td rowspan=1 colspan=1>33.60</td><td rowspan=1 colspan=1>2.85</td><td rowspan=1 colspan=1>0.12</td><td rowspan=1 colspan=1>-0.01</td></tr><tr><td rowspan=1 colspan=1>Bilirubin conj.</td><td rowspan=1 colspan=1>1%</td><td rowspan=1 colspan=1>20/20100%</td><td rowspan=1 colspan=1>60/60100%</td><td rowspan=1 colspan=1>32.85</td><td rowspan=1 colspan=1>2.95</td><td rowspan=1 colspan=1>0.16</td><td rowspan=1 colspan=1>0.09</td></tr><tr><td rowspan=1 colspan=1>Glucose</td><td rowspan=1 colspan=1>1%</td><td rowspan=1 colspan=1>20/20100%</td><td rowspan=1 colspan=1>60/60100%</td><td rowspan=1 colspan=1>33.11</td><td rowspan=1 colspan=1>2.86</td><td rowspan=1 colspan=1>0.10</td><td rowspan=1 colspan=1>0.00</td></tr><tr><td rowspan=1 colspan=1>Human cells(PBMC)</td><td rowspan=1 colspan=1>1.00E+06cells/mL</td><td rowspan=1 colspan=1>20/20100%</td><td rowspan=1 colspan=1>60/60100%</td><td rowspan=1 colspan=1>33.24</td><td rowspan=1 colspan=1>2.90</td><td rowspan=1 colspan=1>0.12</td><td rowspan=1 colspan=1>0.04</td></tr><tr><td rowspan=1 colspan=1>Mucus</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>20/20100%</td><td rowspan=1 colspan=1>60/60100%</td><td rowspan=1 colspan=1>33.12</td><td rowspan=1 colspan=1>2.92</td><td rowspan=1 colspan=1>0.13</td><td rowspan=1 colspan=1>0.06</td></tr><tr><td rowspan=1 colspan=1>Acidic pH(4.0)</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>20/20100%</td><td rowspan=1 colspan=1>60/60100%</td><td rowspan=1 colspan=1>33.35</td><td rowspan=1 colspan=1>2.85</td><td rowspan=1 colspan=1>0.15</td><td rowspan=1 colspan=1>0.00</td></tr><tr><td rowspan=1 colspan=1>Alkaline pH(9.0)</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>20/20100%</td><td rowspan=1 colspan=1>58/58*100%</td><td rowspan=1 colspan=1>33.30</td><td rowspan=1 colspan=1>2.83</td><td rowspan=1 colspan=1>0.14</td><td rowspan=1 colspan=1>-0.03</td></tr><tr><td rowspan=1 colspan=1>Semen</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>20/20100%</td><td rowspan=1 colspan=1>60/60100%</td><td rowspan=1 colspan=1>33.04</td><td rowspan=1 colspan=1>2.96</td><td rowspan=1 colspan=1>0.10</td><td rowspan=1 colspan=1>0.10</td></tr><tr><td rowspan=1 colspan=1>Sodium</td><td rowspan=1 colspan=1>300mEq/L</td><td rowspan=1 colspan=1>20/20100%</td><td rowspan=1 colspan=1>60/60100%</td><td rowspan=1 colspan=1>33.11</td><td rowspan=1 colspan=1>2.87</td><td rowspan=1 colspan=1>0.11</td><td rowspan=1 colspan=1>0.01</td></tr><tr><td rowspan=1 colspan=1>Whole Blood</td><td rowspan=1 colspan=1>10%</td><td rowspan=1 colspan=1>20/20100%</td><td rowspan=1 colspan=1>60/60100%</td><td rowspan=1 colspan=1>33.20</td><td rowspan=1 colspan=1>2.79</td><td rowspan=1 colspan=1>0.12</td><td rowspan=1 colspan=1>-0.06</td></tr><tr><td rowspan=1 colspan=1>No interferent</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>20/20100%</td><td rowspan=1 colspan=1>60/60100%</td><td rowspan=1 colspan=1>33.15</td><td rowspan=1 colspan=1>2.86</td><td rowspan=1 colspan=1>0.12</td><td rowspan=1 colspan=1>0.00</td></tr></table>

$^ 1 C =$ Test Concentration \* Titer obtained was lower than LLoQ and was excluded

All BKV-negative samples with endogenous interferents produced valid negative results (target not detected).

For BKV-positive samples with endogenous interferents the mean $\log _ { 1 0 }$ titer of each of the positive BKV samples containing endogenous interferents was within $1 \pm 0 . 5 \log _ { 1 0 } { \mathrm { { g } } } _ { 1 0 }$ of the mean log10 titer of the spike control.

# Exogenous Interference

The effect of potentially interfering exogenous substances on the sensitivity/quantitation of cobas BKV Urine assay was determined by testing 10 individual BKV-negative urine samples spiked with commercially available drugs/substances. The same samples were also tested in the presence of BKV target at $6 0 0 \ : \mathrm { I U / m L }$ (approximately 3x LLoQ). The negative samples spiked solely with BKV target was used as a Positive Spike Control (PSC). The unspiked samples were used as Negative Spiked Controls (NSC). Conditions were tested with one replicate for the unspiked specimens and three replicates for the BKV spiked specimens. All specimens were stabilized in cobas PCR media (1:1 ratio) before testing. The results are summarized in Table 10 below.

Table 10: Exogenous Interference   

<table><tr><td rowspan=2 colspan=1>Interferent</td><td rowspan=2 colspan=1>Concentration</td><td rowspan=1 colspan=1>No BKV</td><td rowspan=1 colspan=5>With BKV [100 IU/mL]</td></tr><tr><td rowspan=1 colspan=1>NegativityRate</td><td rowspan=1 colspan=1>PositivityRate</td><td rowspan=1 colspan=1>MeanCt</td><td rowspan=1 colspan=1>Mean(log10titer</td><td rowspan=1 colspan=1>SD(log10titer)</td><td rowspan=1 colspan=1>MeanDifferencein Log10Titer</td></tr><tr><td rowspan=1 colspan=1>Acetaminophen 1</td><td rowspan=1 colspan=1>1324 μmol/L</td><td rowspan=1 colspan=1>10/10100%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>33.14</td><td rowspan=1 colspan=1>2.88</td><td rowspan=1 colspan=1>0.09</td><td rowspan=1 colspan=1>-0.01</td></tr><tr><td rowspan=1 colspan=1>AcetylsalicylicAcid1</td><td rowspan=1 colspan=1>3.62 mmol/L</td><td rowspan=1 colspan=1>10/10100%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>33.12</td><td rowspan=1 colspan=1>2.90</td><td rowspan=1 colspan=1>0.11</td><td rowspan=1 colspan=1>0.01</td></tr><tr><td rowspan=1 colspan=1>Clotrimazole1</td><td rowspan=1 colspan=1>100 μg/mL</td><td rowspan=1 colspan=1>10/10100%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>33.13</td><td rowspan=1 colspan=1>2.88</td><td rowspan=1 colspan=1>0.10</td><td rowspan=1 colspan=1>-0.02</td></tr><tr><td rowspan=1 colspan=1>Estradiol1</td><td rowspan=1 colspan=1>4.41 nmol/L</td><td rowspan=1 colspan=1>10/10100%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>33.22</td><td rowspan=1 colspan=1>2.87</td><td rowspan=1 colspan=1>0.08</td><td rowspan=1 colspan=1>-0.03</td></tr><tr><td rowspan=1 colspan=1>Ibuprofen1</td><td rowspan=1 colspan=1>2425 μmol/L</td><td rowspan=1 colspan=1>10/10100%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>33.09</td><td rowspan=1 colspan=1>2.89</td><td rowspan=1 colspan=1>0.08</td><td rowspan=1 colspan=1>0.00</td></tr><tr><td rowspan=1 colspan=1>Metronidazole 1</td><td rowspan=1 colspan=1>701 μmol/L</td><td rowspan=1 colspan=1>10/10100%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>33.21</td><td rowspan=1 colspan=1>2.86</td><td rowspan=1 colspan=1>0.09</td><td rowspan=1 colspan=1>-0.03</td></tr><tr><td rowspan=1 colspan=1>Naproxen 1</td><td rowspan=1 colspan=1>2170 μmol/L</td><td rowspan=1 colspan=1>10/10100%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>33.05</td><td rowspan=1 colspan=1>2.91</td><td rowspan=1 colspan=1>0.11</td><td rowspan=1 colspan=1>0.01</td></tr><tr><td rowspan=1 colspan=1>PhenazopyridineHydrochloride 2</td><td rowspan=1 colspan=1>200 μg/mL</td><td rowspan=1 colspan=1>10/10100%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>33.21</td><td rowspan=1 colspan=1>2.87</td><td rowspan=1 colspan=1>0.09</td><td rowspan=1 colspan=1>-0.02</td></tr><tr><td rowspan=1 colspan=1>PropyleneGlycol 2</td><td rowspan=1 colspan=1>1000 μg/mL</td><td rowspan=1 colspan=1>10/10100%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>33.18</td><td rowspan=1 colspan=1>2.86</td><td rowspan=1 colspan=1>0.08</td><td rowspan=1 colspan=1>-0.04</td></tr><tr><td rowspan=1 colspan=1>Talcum Powder 1</td><td rowspan=1 colspan=1>0.05% w/v</td><td rowspan=1 colspan=1>10/10100%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>33.44</td><td rowspan=1 colspan=1>2.77</td><td rowspan=1 colspan=1>0.19</td><td rowspan=1 colspan=1>-0.13</td></tr><tr><td rowspan=1 colspan=1>Ethanol SC</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>10/10100%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>33.21</td><td rowspan=1 colspan=1>2.87</td><td rowspan=1 colspan=1>0.10</td><td rowspan=1 colspan=1>-0.03</td></tr><tr><td rowspan=1 colspan=1>Water SC</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>10/10100%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>33.02</td><td rowspan=1 colspan=1>2.85</td><td rowspan=1 colspan=1>0.11</td><td rowspan=1 colspan=1>-0.04</td></tr><tr><td rowspan=1 colspan=1>NegativeControl</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>28/28100%</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>,</td></tr><tr><td rowspan=1 colspan=1>PositiveControl</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>84/84100%</td><td rowspan=1 colspan=1>33.13</td><td rowspan=1 colspan=1>2.89</td><td rowspan=1 colspan=1>0.12</td><td rowspan=1 colspan=1>0.00</td></tr></table>

1/2 The superscripts in interferents indicate the solvent that was used for constituting the interferents (i.e., $1 =$ Ethanol; $2 = \mathrm { W }$ ater); $\mathbf { S C } =$ solvent control.

All BKV-negative samples with exogenous interferents produced valid negative results (target not detected).

For BKV-positive samples with exogenous interferents the mean $\log _ { 1 0 }$ titer of each of the positive BKV samples containing endogenous interferents was within $\mathsf { \Omega } _ { 1 } \pm \mathsf { 0 } . 5 \log _ { 1 0 }$ of the mean $\log _ { 1 0 }$ titer of the spike control.

# 7. Assay Reportable Range:

Based on the linearity study, the reportable range of the cobas BKV assay in urine is $2 . 0 0 \mathrm { E } { + } 0 2 \mathrm { I U / m L }$ to $1 . 0 0 \mathrm { E + } 0 8 \mathrm { \Pi { U } / m L }$ .

8. Traceability, Stability, Expected Values (Controls, Calibrators, or Methods):

# Traceability

The traceability study for the WHO BKV standard was provided with cobas BKV for EDTAplasma (K202215). BKV DNA levels of WHO standard in urine stabilized in cobas PCR media were accurately determined across the linear range from $2 . 0 \mathrm { E } { + 0 2 }$ to $1 . 0 \mathrm { E } { + } 0 7 \mathrm { I U / m l }$ with a maximum inaccuracy less than $0 . 2 0 \log _ { 1 0 } \mathrm { I U / m l }$ close to the LLoQ. Additional data from the clinical reproducibility study, for BKV DNA WHO standard in spiked at 3xLLoQ to 250x LLoQ in stabilized urine, showed the observed mean $\mathrm { w a s } \pm 0 . 1 4 \log _ { 1 0 }$ of the expected titers.

# Stability

The clinical specimen stability of BKV viral target in neat urine specimens (NUS) prior to addition of the cobas PCR media as well as in urine specimens stabilized (USS) in the cobas PCR media after various storage, processing, and transportation conditions between $2 \mathrm { ^ { \circ } C }$ to $3 0 ^ { \circ } \mathrm { C }$ were evaluated using cobas BKV for use on the cobas 6800/8800 Systems. BKVpositive urine specimens collected from six individuals with titers above $2 \mathbf { x }$ LLoQ $( > 4 0 0$ $\mathrm { I U / m L }$ ) were used to prepare the Clinical Stability Study (CSS) panel. Time points for NUS included T0 (immediately after collection, used as reference), T1 ( $2 5 \mathrm { h }$ at $2 ^ { \circ } \mathrm { C }$ to $8 ^ { \circ } \mathrm { C }$ ), T2 (5h at $3 0 ^ { \circ } \mathrm { C } \pm 2 ^ { \circ } \mathrm { C } )$ , and T3 (5h at $3 0 ^ { \circ } C \pm 2 ^ { \circ } C + 2 1 \mathrm { h }$ at $2 \mathrm { ^ { \circ } C }$ to $8 ^ { \circ } \mathrm { C }$ ). For the USS specimens, panel members were stored at $3 0 ^ { \circ } \mathrm { C } \pm 2 ^ { \circ } \mathrm { C }$ and tested at 33, 47, 91 and 92 days.

Of the BKV-positive clinical urine specimens tested, the mean $\log _ { 1 0 }$ titers including the twosided 95 confidence interval of each of the tested time points/conditions and tube types was within $\pm 0 . 5 \log _ { 1 0 } \mathrm { { \Omega } }$ of the mean $\log _ { 1 0 }$ titer of the respective reference condition $\mathrm { T 0 } =$ reference). However, there was one sample which did not meet the acceptance criteria, and this may be due to non-homogeneity of the BKV-positive urine sample. A labeling precaution has been included in the package insert indicating that urine specimens must be transferred into cobas PCR media tubes and stabilized immediately after collection. Procedural limitations have been included in the IFU stating that the test is validated only for use with stabilized urine because quantitative variability of BKV target inherent to urine specimens has been observed in non-stabilized urine.

These results support the following storage conditions:

• Neat urine can be stored at $2 \mathrm { ^ { \circ } C }$ to $3 0 \mathrm { { } ^ { \circ } C }$ for up to 24 hours.   
Specimens stabilized in cobas PCR media can be stored at $2 \mathrm { ^ { \circ } C }$ to $3 0 ^ { \circ } \mathrm { C }$ for up to 90 days.

# Open kit and On-board Stability

On-Board Stability and Open Kit Stability has been established for the predicate device. The cobas BKV 192T test-specific reagent cassettes are stable for up to 90 days at $2 - 8 ^ { \circ } \mathrm { C }$ (Open Kit Stability) and remain stable for up to 40 hours at $3 7 ^ { \circ } \mathrm { C }$ (On-Board Stability). The 192T test-specific reagent cassettes can be used up to 40 times.

# Reagent Stability

Three lots of the cobas BKV were tested to demonstrate stability of the test-specific reagents of cobas BKV and cobas EBV/BKV Control Kit when stored at stressed temperature

conditions (accelerated stability) and at the targeted storage temperature of $2 \mathrm { ^ { \circ } C }$ to $8 ^ { \circ } \mathrm { C }$ (realtime stability).

Accelerated Stability: The kit components BKV MMX-R2 192T, EBV/BKV $\mathrm { H } ( + ) \mathrm { C }$ , and the EBV/BKV $\mathrm { L } ( + ) \mathrm { C }$ were stored at $3 7 ^ { \circ } \mathrm { C } \pm 2 ^ { \circ } \mathrm { C }$ for up to 75 days and tested at various time points. All lots met the acceptance criteria at all testing time points (26d, 41d, 51d, 63d, and 75d). The 75 days accelerated stability results predict a 19-month real-time stability for the test-specific reagents of cobas BKV and cobas EBV/BKV Control Kit.

Real-time stability: The kit components BKV MMX-R2 192T, EBV/BKV $\mathrm { H } ( + ) \mathrm { C }$ , and the EBV/BKV $\mathrm { L } ( + ) \mathrm { C }$ were stored at $2 { - } 8 ^ { \circ } \mathrm { C }$ for up to 13 months and tested at various time points. All lots met the acceptance criteria at all testing time points (3m, 6m, $9 \mathrm { m }$ , $1 2 \mathrm { m }$ , $1 3 \mathrm { m }$ , $1 6 \mathrm m$ , $1 8 \mathrm { m }$ , 19m).

Shelf-life: Based on the combined data of accelerated and real-time stability studies the results support a shelf-life of 18 months when stored at $2 { - } 8 ^ { \circ } \mathrm { C }$ .

Reagent stability has been established for 18 month upon the storage conditions indicated in the IFU.

# Expected values for Quality Controls

To monitor the assay performance, reagent performance, and procedural errors, positive and negative external controls must be run in accordance with the guidelines or requirements of local, state, and/or federal regulations or accrediting organizations.

External Controls are provided in the cobas EBV/Positive Control Kit and the cobas Buffer Negative Control Kit, separately from the cobas BKV test kit. The cobas EBV/BKV positive control consists of BKV phagemid DNA diluted into BKV-negative human plasma at two concentrations to yield the EBV/BKV Low Positive Control (EBV/BKV $\mathrm { L } ( + ) \mathrm { C } )$ and the EBV/BKV High Positive Control (EBV/BKV $\mathrm { H } ( + ) \mathrm { C } )$ . The cobas Buffer Negative Control consists of Tris buffer with $< 0 . 0 0 2 \%$ Poly rA RNA. The cobas BKV Negative Control, the BKV Low Positive Control, and the BKV High Positive Control must be included in each run. The validity of the results for the controls as well as for the DNA-QS (internal control) is determined by the assay-specific analysis software package used by the cobas 6800/8800 instrument. The amount of BKV DNA for BKV-high and -low positive controls must fall within their acceptable titer ranges.

The failure rates of the kit controls and samples were assessed by evaluating outcomes of the nine analytical studies (LoD, LoD verification for Genotype, Linearity, Linearity verification for Genotype, Precision, Carry-Over, Lot interchangeability, Accuracy, and LoD in in urine samples ). Results demonstrated an overall invalid rate of $0 . 0 0 \%$ for QS, $0 . 0 0 \%$ for RMC and a $0 . 0 2 \%$ sample failure rate.

9. Detection Limit:

The LoD of the cobas BKV test for the 1st WHO BKV Standard (BKV Subgroup 1b) was determined by analysis of serial dilutions of the Standard diluted into a pooled urine derived from BKV negative individuals and stabilized in cobas PCR media (1:1 dilution). Panels of six concentration levels plus a blank were tested with three lots of cobas BKV test reagents and four instruments with multiple runs and operators over a period of three days. Each dilution was tested in 63 replicates per lot and day $\mathrm { \Phi } _ { \mathrm { n } = 1 8 9 }$ total replicates per day).

The results from testing the WHO BKV Standard in pooled urine as well as the calculated LoD values are shown in the table below. The LoD and LLoQ values were determined by Probit analysis and by $9 5 \%$ hit rate (Table 11).

Table 11: LoD and LLoQ Values Estimated for Three Lots   

<table><tr><td rowspan=1 colspan=1>Kit Lot</td><td rowspan=1 colspan=1>NominalConcentration(IU/mL)</td><td rowspan=1 colspan=1>Number ofPositiveReplicates</td><td rowspan=1 colspan=1>Number ofValidReplicates</td><td rowspan=1 colspan=1>Hit Rate(%)</td><td rowspan=1 colspan=1>LoD byProbit[95% CI]</td></tr><tr><td rowspan=7 colspan=1>Lot 1</td><td rowspan=1 colspan=1>40.0</td><td rowspan=1 colspan=1>63</td><td rowspan=1 colspan=1>63</td><td rowspan=1 colspan=1>100</td><td rowspan=7 colspan=1>12.2IU/mL[9.2  18.3IU/mL]</td></tr><tr><td rowspan=1 colspan=1>20.0</td><td rowspan=1 colspan=1>63</td><td rowspan=1 colspan=1>63</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>10.0</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>63</td><td rowspan=1 colspan=1>95.2</td></tr><tr><td rowspan=1 colspan=1>5.0</td><td rowspan=1 colspan=1>47</td><td rowspan=1 colspan=1>63</td><td rowspan=1 colspan=1>74.6</td></tr><tr><td rowspan=1 colspan=1>2.5</td><td rowspan=1 colspan=1>25</td><td rowspan=1 colspan=1>63</td><td rowspan=1 colspan=1>39.7</td></tr><tr><td rowspan=1 colspan=1>1.25</td><td rowspan=1 colspan=1>26</td><td rowspan=1 colspan=1>63</td><td rowspan=1 colspan=1>41.3</td></tr><tr><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>63</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=7 colspan=1>Lot 2</td><td rowspan=1 colspan=1>40.0</td><td rowspan=1 colspan=1>63</td><td rowspan=1 colspan=1>63</td><td rowspan=1 colspan=1>100</td><td rowspan=7 colspan=1>11.9IU/mL[9.2  17.3IU/mL]</td></tr><tr><td rowspan=1 colspan=1>20.0</td><td rowspan=1 colspan=1>63</td><td rowspan=1 colspan=1>63</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>10.0</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>63</td><td rowspan=1 colspan=1>95.2</td></tr><tr><td rowspan=1 colspan=1>5.0</td><td rowspan=1 colspan=1>42</td><td rowspan=1 colspan=1>63</td><td rowspan=1 colspan=1>66.7</td></tr><tr><td rowspan=1 colspan=1>2.5</td><td rowspan=1 colspan=1>32</td><td rowspan=1 colspan=1>63</td><td rowspan=1 colspan=1>50.8</td></tr><tr><td rowspan=1 colspan=1>1.25</td><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>63</td><td rowspan=1 colspan=1>27.0</td></tr><tr><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>63</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=7 colspan=1>Lot 3</td><td rowspan=1 colspan=1>40.0</td><td rowspan=1 colspan=1>63</td><td rowspan=1 colspan=1>63</td><td rowspan=1 colspan=1>100</td><td rowspan=7 colspan=1>10.1IU/mL[7.8  14.7IU/mL]</td></tr><tr><td rowspan=1 colspan=1>20.0</td><td rowspan=1 colspan=1>63</td><td rowspan=1 colspan=1>63</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>10.0</td><td rowspan=1 colspan=1>61</td><td rowspan=1 colspan=1>63</td><td rowspan=1 colspan=1>96.8</td></tr><tr><td rowspan=1 colspan=1>5.0</td><td rowspan=1 colspan=1>46</td><td rowspan=1 colspan=1>63</td><td rowspan=1 colspan=1>73.0</td></tr><tr><td rowspan=1 colspan=1>2.5</td><td rowspan=1 colspan=1>39</td><td rowspan=1 colspan=1>63</td><td rowspan=1 colspan=1>61.9</td></tr><tr><td rowspan=1 colspan=1>1.25</td><td rowspan=1 colspan=1>19</td><td rowspan=1 colspan=1>63</td><td rowspan=1 colspan=1>30.2</td></tr><tr><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>63</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=7 colspan=1>All lotscombined</td><td rowspan=1 colspan=1>40.0</td><td rowspan=1 colspan=1>189</td><td rowspan=1 colspan=1>189</td><td rowspan=1 colspan=1>100</td><td rowspan=7 colspan=1>11.4IU/mL[9.7  14.0IU/mL]</td></tr><tr><td rowspan=1 colspan=1>20.0</td><td rowspan=1 colspan=1>189</td><td rowspan=1 colspan=1>189</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>10.0</td><td rowspan=1 colspan=1>181</td><td rowspan=1 colspan=1>189</td><td rowspan=1 colspan=1>95.8</td></tr><tr><td rowspan=1 colspan=1>5.0</td><td rowspan=1 colspan=1>135</td><td rowspan=1 colspan=1>189</td><td rowspan=1 colspan=1>76.2</td></tr><tr><td rowspan=1 colspan=1>2.5</td><td rowspan=1 colspan=1>96</td><td rowspan=1 colspan=1>189</td><td rowspan=1 colspan=1>55.6</td></tr><tr><td rowspan=1 colspan=1>1.25</td><td rowspan=1 colspan=1>62</td><td rowspan=1 colspan=1>189</td><td rowspan=1 colspan=1>36.2</td></tr><tr><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>189</td><td rowspan=1 colspan=1>0</td></tr></table>

When determined by Probit analysis, the LoD for all three kit lots combined is $1 1 . 4 \ : \mathrm { I U / m L }$ $9 5 \%$ CI: $9 . 7 - 1 4 . 0 \mathrm { I U / m L }$ ). The LoD by $9 5 \%$ hit rate was $1 0 . 0 \ : \mathrm { I U / m L }$ (181/189) and was identical for all tested lots. The highest LoD of $1 2 . 2 \ : \mathrm { I U / m L }$ was obtained with kit lot 1, which is only slightly higher than the LoD determined by $9 5 \%$ hit rate. The claimed LoD value is 12.2 $\mathrm { I U / m L }$ ( $9 5 \%$ CI: 9.2 – 18.3 IU/mL) and this concentration was used in studies for confirmation of the LoD.

Verification of the Limit of Detection (LoD) with genotypes of subgroup Ia, Ic, and subtypes II, II, and IV in urine

The LoD $( 1 2 . 2 \mathrm { I U / m L } )$ was verified for the cobas BKV test with BKV subgroup Ia, Ic, and subtypes II, III, and IV following the CLSI Guideline EP17-A2. For subgroups Ia, subtypes II and IV clinical specimens were used. Due to lack of clinical specimens phagemids for subgroup Ic was used. BKV-positive material was diluted in BKV-PCR negative pooled urine stabilized with cobas PCR media to prepare the ISS. The ISS titer was confirmed by an alternate assay with three replicates. For subgroup Ic, digital droplet PCR was used to confirm the initial titer as this subgroup was not detectable by the alternate assay. Each panel consisted of three concentration levels around the LoD $1 . 5 \mathrm { x }$ ( $1 8 . 3 \ \mathrm { I U / m L }$ ), $1 . 0 \mathrm { x }$ (12.2 $\mathrm { I U / m L }$ ), $0 . 5 \mathrm { x }$ (6.1 IU/mL).

A total of 63 replicates per concentration level were tested across three dilution series, three instruments and three kit lots. A different operator tested each dilution series.

The results are presented in Table 12 and verify that a hit rate of $9 5 \%$ or higher was observed at $1 2 . 2 \ : \mathrm { I U / m L }$ (1x LoD) for BKV genotypes. Thus, the observed hit rates verify the LoD for BKV in urine at $1 2 . 2 ~ \mathrm { I U / m L }$ .

Table 12: Verification of the LoD for BKV Genotypes   

<table><tr><td colspan="1" rowspan="1">Genotype</td><td colspan="1" rowspan="1">Test concentration</td><td colspan="1" rowspan="1">Number of validreplicates (N)</td><td colspan="1" rowspan="1">Number ofpositives (n)</td><td colspan="1" rowspan="1">Hit rate[(n/N)x100]</td></tr><tr><td colspan="1" rowspan="1">Subgroup Ia</td><td colspan="1" rowspan="1">18.3 IU/mL</td><td colspan="1" rowspan="1">63</td><td colspan="1" rowspan="1">62</td><td colspan="1" rowspan="1">98.41%</td></tr><tr><td colspan="1" rowspan="1">Subgroup Ia</td><td colspan="1" rowspan="1">12.2 IU/mL</td><td colspan="1" rowspan="1">63</td><td colspan="1" rowspan="1">61</td><td colspan="1" rowspan="1">96.83%</td></tr><tr><td colspan="1" rowspan="1">Subgroup Ia</td><td colspan="1" rowspan="1">6.1 IU/mL</td><td colspan="1" rowspan="1">63</td><td colspan="1" rowspan="1">53</td><td colspan="1" rowspan="1">84.13%</td></tr><tr><td colspan="1" rowspan="1">Subgroup Ic</td><td colspan="1" rowspan="1">18.3 IU/mL</td><td colspan="1" rowspan="1">63</td><td colspan="1" rowspan="1">63</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Subgroup Ic</td><td colspan="1" rowspan="1">12.2 IU/mL</td><td colspan="1" rowspan="1">63</td><td colspan="1" rowspan="1">62</td><td colspan="1" rowspan="1">98.41%</td></tr><tr><td colspan="1" rowspan="1">Subgroup Ic</td><td colspan="1" rowspan="1">6.1 IU/mL</td><td colspan="1" rowspan="1">63</td><td colspan="1" rowspan="1">50</td><td colspan="1" rowspan="1">79.37%</td></tr><tr><td colspan="1" rowspan="1">Subtype II</td><td colspan="1" rowspan="1">18.3 IU/mL</td><td colspan="1" rowspan="1">63</td><td colspan="1" rowspan="1">63</td><td colspan="1" rowspan="1">98.41%</td></tr><tr><td colspan="1" rowspan="1">Subtype II</td><td colspan="1" rowspan="1">12.2 IU/mL</td><td colspan="1" rowspan="1">63</td><td colspan="1" rowspan="1">61</td><td colspan="1" rowspan="1">96.83%</td></tr><tr><td colspan="1" rowspan="1">Subtype II</td><td colspan="1" rowspan="1">6.1 IU/mL</td><td colspan="1" rowspan="1">63</td><td colspan="1" rowspan="1">56</td><td colspan="1" rowspan="1">88.89%</td></tr><tr><td colspan="1" rowspan="1">Subtype II</td><td colspan="1" rowspan="1">18.3 IU/mL</td><td colspan="1" rowspan="1">63</td><td colspan="1" rowspan="1">63</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Subtype III</td><td colspan="1" rowspan="1">12.2 IU/mL</td><td colspan="1" rowspan="1">63</td><td colspan="1" rowspan="1">62</td><td colspan="1" rowspan="1">98.41%</td></tr><tr><td colspan="1" rowspan="1">Subtype III</td><td colspan="1" rowspan="1">6.1 IU/mL</td><td colspan="1" rowspan="1">63</td><td colspan="1" rowspan="1">60</td><td colspan="1" rowspan="1">95.24%</td></tr><tr><td colspan="1" rowspan="1">Subtype IV</td><td colspan="1" rowspan="1">18.3 IU/mL</td><td colspan="1" rowspan="1">63</td><td colspan="1" rowspan="1">63</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Subtype IV</td><td colspan="1" rowspan="1">12.2 IU/mL</td><td colspan="1" rowspan="1">63</td><td colspan="1" rowspan="1">63</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Subtype IV</td><td colspan="1" rowspan="1">6.1 IU/mL</td><td colspan="1" rowspan="1">63</td><td colspan="1" rowspan="1">54</td><td colspan="1" rowspan="1">85.71%</td></tr></table>

10. Assay Cut-Off: Not applicable.

11. Accuracy (Instrument): Not applicable

12. Carry-Over:

The carry-over rate for cobas BKV was determined by testing 240 replicates of an BKVnegative pooled urine sample and 225 replicates of a high titer BKV sample at $1 . 0 0 \mathrm { E } { + } 0 9 $ $\mathrm { I U / m L }$ $\bar { 5 \mathrm { x } } 1 0 ^ { 6 } \mathrm { x }$ LLoQ). The high titer sample was prepared by spiking a high-positive BKV plasmid (pRMMYBKV12, subgroup Ib) into BKV-negative pooled urine and titer verified by using digital droplet PCR.

In total, five runs were performed with positive and negative samples in a checkerboard configuration using one kit lot and one cobas 8800 system.

All 240 replicates of the negative sample were negative, resulting in a carry-over rate of $0 \%$ (upper one-sided $9 5 \%$ confidence interval: $1 . 2 4 \%$ ).

# B Comparison Studies:

1. Method Comparison with Predicate Device:

The comparator BKV assay is a well validated quantitative BKV viral load test, currently used in clinical practice at a major transplant center in the United States and is traceable to the BKV WHO standard. Please see “Other Clinical Supportive Data” section below for assay results.

# 2. Matrix Comparison:

Not applicable

# C Clinical Studies with Urine Specimens:

1. Clinical Sensitivity: Not applicable

2. Clinical Specificity: Not applicable

3. Other Clinical Supportive Data (When 1. and 2. Are Not Applicable):

Concordance of cobas BKV with a Comparator BKV LDT

The clinical performance of the cobas BKV Urine assay was compared to the performance of a validated, well-established laboratory developed nucleic acid test (LDT) (comparator BKV LDT) by measuring BKV DNA levels in longitudinal clinical samples of urine specimens from BKV-infected and non-infected patients. Contrived BKV-negative urine samples spiked with cultured BKV virus were used to cover the linear range.

Urine specimens were collected and shipped frozen to RMS. The urine specimens were then thawed, stabilized with cobas PCR media, aliquoted into coded tubes, randomized and tested with cobas BKV and the comparator LDT. cobas BKV testing was performed at three sites (two external and one internal test sites) using three kit lots per site; comparator testing was performed at one site.

All 308 urine samples (from 84 transplant subjects) were valid on both assays and evaluable for the clinical concordance analysis. Results presented in Table 14 demonstrate concordance between the cobas BKV and BKV LDT ranging from between $7 5 \%$ to $100 \%$ depending on the analyte concentration in the samples. The LLoQ for BKV LDT in urine specimens was $1 0 0 0 \mathrm { \ : I U / m L }$ $( 3 \mathrm { l o g } _ { 1 0 } \mathrm { I U } / \mathrm { m L } )$ ) and the ULoQ for BKV LDT in urine specimens was $2 . 0 0 \mathrm { E + } 0 7 \ : \mathrm { I U / m L }$ . The analytical ranges of the BKV DNA were calculated based on standard deviation $( \sigma = 0 . 1 5 \mathrm { \ : l o g 1 0 \ : I U / m L } )$ of the comparator BKV LDT (Table 14). An analyte concentration of $3 . 0 \log _ { 1 0 } \mathrm { { I U / m L } }$ represented $\mathrm { L L o Q } + 2 \sigma$ , $3 . 3 \log _ { 1 0 } \mathrm { { I U / m L } }$ represented $\mathrm { L L o Q } + 4 \sigma$ and $3 . 9 \log _ { 1 0 } \mathrm { { I U / m L } }$ represented $\mathrm { L L o Q } + 6 \sigma$ with a range interval of 2σ.

Table 14: Concordance analysis between cobas BKV and the comparator LDT on BKV DNA level results for all samples (stabilized urine)   

<table><tr><td rowspan=1 colspan=1>cobas BKV(log1o IU/mL)</td><td rowspan=1 colspan=1>Comparator BKVLDT (log1o IU/mL)Target NotDetected</td><td rowspan=1 colspan=1>Comparator BKVLD (log10IU/mL&lt;LLoQ(&lt; 3.0)</td><td rowspan=1 colspan=1>ComparatorBKV LDT(log10 IU/mL)3.0 to &lt; 3.3</td><td rowspan=1 colspan=1>ComparatorBKV LDT(log1o IU/mL)3.3 to &lt; 3.6</td><td rowspan=1 colspan=1>ComparatorBKV LDT(log10 IU/mL)3.6 to 3.9</td><td rowspan=1 colspan=1>ComparatorBKV LDT(log1o IU/mL)&gt; 3.9</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Target Not Detected</td><td rowspan=1 colspan=1>62</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>68</td></tr><tr><td rowspan=1 colspan=1>&lt; 3.0 (&lt; LLoQ)</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>22</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>27</td></tr><tr><td rowspan=1 colspan=1>3.0 to &lt; 3.3</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2</td></tr><tr><td rowspan=1 colspan=1>3.3 to &lt; 3.6</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>9</td></tr><tr><td rowspan=1 colspan=1>3.6 to 3.9</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>23</td></tr><tr><td rowspan=1 colspan=1>&gt;3.9</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>169</td><td rowspan=1 colspan=1>179</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>66</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>18</td><td rowspan=1 colspan=1>170</td><td rowspan=1 colspan=1>308</td></tr><tr><td rowspan=1 colspan=1>Column Agreement(%)</td><td rowspan=1 colspan=1>66/66 (100.0%)</td><td rowspan=1 colspan=1>30/30 (100.0%)</td><td rowspan=1 colspan=1>6/8(75.0%)</td><td rowspan=1 colspan=1>14/16 (87.5%)</td><td rowspan=1 colspan=1>18/18 (100%)</td><td rowspan=1 colspan=1>169/170(99.4%)</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>95% CI upper andlower bounds a</td><td rowspan=1 colspan=1>94.5%, 100%</td><td rowspan=1 colspan=1>88.6%, 100.0%</td><td rowspan=1 colspan=1>40.9%, 92.9%</td><td rowspan=1 colspan=1>64.0%, 96.5%</td><td rowspan=1 colspan=1>82.4%, 100.0%</td><td rowspan=1 colspan=1>96.7%, 99.9%</td><td rowspan=1 colspan=1></td></tr></table>

Note: $\mathrm { C I } =$ Confidence Interval; $\mathrm { L L o Q } =$ lower limit of quantitation of Comparator BKV LDT $( 1 0 0 0 \ : \mathrm { I U / m L } )$ . Standard Deviation of Comparator BKV LDT estimated at $0 . 1 5 \log _ { 1 0 } \mathrm { I U / m L }$ (BKV LDT validation study). Analyte concentration of $3 . 3 \log 1 0 \mathrm { I U / m L }$ represented LLoQ $+ 2 \sigma$ , $3 . 6 \log 1 0 \mathrm { I U / m L }$ represented $\mathrm { L L o Q } + 4 \sigma$ and $3 . 9 \log 1 0 \mathrm { I U / m L }$ represented $\mathrm { L L o Q } + 6 \sigma$ with a range interval of $2 \sigma$ . Paired samples evaluable for clinical concordance analysis were included in this table. a Assumed independence between all samples.

Discordant results were defined as those that are more than one box away from the diagonal (indicated by shaded cells in Table 14). For Target Not Detected (TND) by LDT Column Agreement the cobas BKV Target Not Detected and $< \mathrm { L L o Q } ( < 3 . 0 )$ cells were combined. The rationale for adding the adjacent <LLoQ and TND cells for the TND column is that the difference between a TND and <LLoQ is not clinically meaningful and that the lower end of the measuring range is usually more impacted by random error than the linear range of the test.

In a separate study with 66 pre-characterized BKV DNA-negative samples tested using the cobas BKV and the BKV LDT, 61 samples were negative on both assays. The Negative Percent Agreement (NPA) was $100 \%$ ( $9 5 \%$ Exact CI: $9 4 . 1 \% - 1 0 0 \% )$ . The other five samples tested positive on the comparator LDT and were excluded from the NPA calculation.

Concordance between cobas BKV and the comparator BKV LDT was also evaluated using different clinical thresholds (Table 15).

Table 15: Concordance summary of cobas BKV and comparator BKV LDT using different thresholds   

<table><tr><td rowspan=1 colspan=1>Thresholds*</td><td rowspan=1 colspan=1>Percent Agreement&lt; Threshold(n/N)(95% CI)</td><td rowspan=1 colspan=1>Percent Agreement≥ Threshold(n/N)(95% CI)</td></tr><tr><td rowspan=1 colspan=1>Target Not Detected</td><td rowspan=1 colspan=1>93.9% (62/66)(85.4%, 97.6%)</td><td rowspan=1 colspan=1>97.5% (236/242)(94.7%, 98.9%)</td></tr><tr><td rowspan=1 colspan=1>3.0 Log10 IU/mL(LLoQ)</td><td rowspan=1 colspan=1>97.9% (94/96)(92.7%,99.4%)</td><td rowspan=1 colspan=1>99.5% (211/212)(97.4%, 99.9%)</td></tr><tr><td rowspan=1 colspan=1>4.0 Log10 IU/mL</td><td rowspan=1 colspan=1>90.9% (130/143)(85.1%,94.6%)</td><td rowspan=1 colspan=1>99.4% (164/165)(96.6%,99.9%)</td></tr><tr><td rowspan=1 colspan=1>7.0 Log10 IU/mL</td><td rowspan=1 colspan=1>97.2% (242/249)(94.3%, 98.6%)</td><td rowspan=1 colspan=1>94.9% (56/59)(86.1%, 98.3%)</td></tr></table>

Note: Samples with a Target Not Detected results were categorised as <threshold value in $\mathrm { I U / m L }$ . $\mathrm { L L o Q } =$ lower limit of quantitation of Comparator BKV LDT $( 1 0 0 0 \mathrm { I U / m L } = 3 . 0 \log _ { 1 0 } \mathrm { I U / m L } )$ ). $9 5 \%$ confidence interval (CI) calculated by Score method assuming independence between all samples. \* Thresholds of $1 . 0 \mathrm { E } + 0 4 \mathrm { I U } / \mathrm { m L } = 4 . 0 \mathrm { L o g } _ { 1 0 } \mathrm { I U } / \mathrm { m L }$ and $1 . 0 \mathrm { E } + 0 7 \mathrm { I U } / \mathrm { m L } = 7 . 0 \mathrm { L o g } _ { 1 0 } \mathrm { I U } / \mathrm { m L }$ .

From all samples with valid paired results within the overlapping linear range of both the cobas BKV and the BKV LDT assay $\mathrm { 1 . 0 E + 0 3 ~ I U / m L }$ to $2 . 0 \mathrm { E + } 0 7 \mathrm { I U / m L }$ ), a total of 153 urine samples stabilized in cobas PCR media (from 55 out of the 84 transplant subjects) from the three testing sites were evaluated by correlation analysis.

The $9 5 \%$ CI of the y-intercept for the Deming linear regression analysis did not include 0, indicating that the results of the cobas BKV and the BKV LDT differ by a constant number (Figure 3). This difference could be caused by differences in calibration methods (e.g., type of quantitative calibrator, frequency of calibration) between the two tests. The $9 5 \%$ CIs of the slope included 1, indicating there is no statistical evidence of proportional difference between the two assays for all samples. The overall correlation coefficient was 0.98, indicating a strong correlation between cobas BKV and the comparator BKV LDT.

Bias plot analysis of DNA level differences indicated a systematic difference between the assays that is constant across the overlapping linear range. The $9 5 \%$ CI of the intercept of the fitted line in the bias plots was $( 4 8 . 8 \%$ to $1 6 . 8 \%$ , which is withi $1 \pm 0 . 5 0 \log _ { 1 0 } \mathrm { { I U / m L } }$ . The mean bias was estimated at $0 . 2 3 1 \log _ { 1 0 }$ $\mathrm { I U / m L }$ and the systematic difference between both assays was $0 . 2 4 8 \log _ { 1 0 } \mathrm { I U / m L }$ and $0 . 1 8 8 \log _ { 1 0 } \mathrm { { I U / m L } }$ for samples with DNA levels at 4 and $7 \log _ { 1 0 } \mathrm { { I U / m L } }$ , respectively. As noted in the limitations, results obtained across different assays are not directly comparable due to the potential for variability in BKV DNA measurements caused by differences in technologies, primer binding sites, specimen handling and analytical performance. Thus, the same device should be used to manage individual patients over the course of the disease.

![](images/103e1feba58b0cc657f9a10671e4467cac84464bf124fbba2e7daf215b89ab1d.jpg)  
Figure 3: Correlation between cobas BKV and comparator BKV LDT for all samples: Deming linear regression plot of DNA levels $\bf ( l o g _ { 1 0 } I U / m L )$

# D Clinical Cut-Off:

Not applicable. Recommendations regarding monitoring BKV viral load post-transplant and medically relevant BKV DNA thresholds vary among transplant type, transplant institutions and viral load test used and is based on recommendations of transplant societies. Results of the cobas BKV test should be interpreted in the context of other clinical data and should not be the sole basis for treatment decisions.

# E Expected Values/Reference Range:

Not applicable

F Other Supportive Instrument Performance Characteristics Data:

Not applicable

# VIII Proposed Labeling:

The labeling supports the finding of substantial equivalence for this device.

# IX Conclusion:

The submitted information in this premarket notification is complete and supports a substantial equivalence decision.