# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATION DECISION SUMMARY ASSAY ONLY

I Background Information:

A 510(k) Number K203748   
B Applicant Immunexpress, Inc   
C Proprietary and Established Names SeptiCyte RAPID   
D Regulatory Information

<table><tr><td rowspan=1 colspan=1>ProductCode(s)</td><td rowspan=1 colspan=1>Classification</td><td rowspan=1 colspan=1>RegulationSection</td><td rowspan=1 colspan=1>Panel</td></tr><tr><td rowspan=1 colspan=1>PRE</td><td rowspan=1 colspan=1>Class II</td><td rowspan=1 colspan=1>21 CFR 866.3215 - Rt-Qpcr Assay For MrnaTranscript ImmuneBiomarkers</td><td rowspan=1 colspan=1>MI - Microbiology</td></tr></table>

# II Submission/Device Overview:

# A Purpose for Submission:

To obtain a substantial equivalence determination for SeptiCyte RAPID. The device is a new device based on a subset of two of the four mRNA transcript immune biomarkers measured by the predicate.

# B Measurand:

mRNA transcripts for host immune biomarkers PLA2G7 and PLAC8.

# C Type of Test:

Reverse transcription $^ +$ quantitative PCR (RT-qPCR)

# III Intended Use/Indications for Use:

# A Intended Use(s):

The SeptiCyte RAPID test is a gene expression assay using reverse transcription polymerase chain reaction to quantify the relative expression levels of host response genes isolated from whole blood collected in the PAXgene Blood RNA Tube. The SeptiCyte RAPID test is used in conjunction with clinical assessments and other laboratory findings as an aid to differentiate infection-positive (sepsis) from infection-negative systemic inflammation in patients suspected of sepsis on their first day of ICU admission. The SeptiCyte RAPID test generates a score (SeptiScore) that falls within one of four discrete Interpretation Bands based on the increasing likelihood of infection-positive systemic inflammation. SeptiCyte RAPID is intended for in-vitro diagnostic use on the Biocartis Idylla System.

B Indication(s) for Use: Same as Intended Use   
C Special Conditions for Use Statement(s): Rx - For Prescription Use Only   
D Special Instrument Requirements: SeptiCyte RAPID is designed to run on the Biocartis Idylla system, K163628

# IV Device/System Characteristics:

# A Device Description:

The SeptiCyte RAPID is an in vitro diagnostic test for simultaneous amplification and detection of two RNA transcripts (PLA2G7 and PLAC8) using total RNA extracted from human blood. The test has been designed, manufactured, and validated for use on the Biocartis Idylla real-time PCR system. The SeptiCyte RAPID test is performed with an Idylla Cartridge, a single-use, disposable, multi-chambered fluidic cartridge that runs on the Biocartis Idylla System (K163628). All processing steps are automated and occur within an Idylla Cartridge, including sample extraction/purification, RT-qPCR for the detection and relative quantification of the two human mRNA targets PLAC8, PLA2G7. All cartridge steps in this process, following the addition of the sample, are fully automated and completely integrated. Test results (measured Cq values and calculated SeptiScore) are available in about 65 minutes.

The specimen used for the SeptiCyte RAPID is a sample of whole blood collected by venipuncture using the PAXgene blood collection tubes within the PAXgene Blood RNA System (K042613). The cartridge contains all of the necessary reagents to perform RNA isolation from the sample.

SeptiCyte RAPID uses quantitative, real-time determination of the amount of each transcript in the sample based on the detection of fluorescence by the Biocartis Idylla qPCR instrument function. The cartridge includes the reagents for reverse transcription and PCR. Transcripts PLAC8 and PLA2G7 are amplified and quantified. These values are combined to produce the SeptiScore, which is interpreted by means of four discrete bands. These four bands reflect a monotonically increasing likelihood of sepsis as shown in the following figure:

<table><tr><td rowspan="2">SIRS</td><td colspan="4"></td></tr><tr><td></td><td>BAND 2</td><td></td><td>SEPSIS</td></tr><tr><td>BAND 1</td><td>5.0</td><td>BAND 3 6.2</td><td>7.4</td><td>BAND 4</td></tr></table>

# B Principle of Operation:

The SeptiCyte RAPID test employs a coupled reverse transcription - quantitative polymerase chain reaction (RT-qPCR) system to measure the expression levels of the host response genes PLA2G7 and PLAC8 isolated from whole blood. Amplicons generated by the RT-qPCR process are detected and quantitated by fluorescent dyes, upon exonucleolytic release of the dyes from oligonucleotide probes that specifically bind to the amplicons. The test is configured to run on the Biocartis Idylla System. The assayspecific software uses the measured PLA2G7 and PLAC8 Cq values to generate a SeptiScore, which falls into one of four discrete sepsis probability bands, each representing an increased risk of sepsis.

# V Substantial Equivalence Information:

A Predicate Device Name(s): SeptiCyte LAB   
B Predicate 510(k) Number(s): K163260

C Comparison with Predicate(s):

<table><tr><td colspan="1" rowspan="1">Device &amp; Predicate Device(s):</td><td colspan="1" rowspan="1">K203748</td><td colspan="1" rowspan="1">K163260</td></tr><tr><td colspan="1" rowspan="1">Device Trade Name</td><td colspan="1" rowspan="1">SeptiCyte Rapid</td><td colspan="1" rowspan="1">SeptiCyte LAB</td></tr><tr><td colspan="1" rowspan="1">General DeviceCharacteristic Similarities</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Intended Use/Indications for Use</td><td colspan="1" rowspan="1">The SeptiCyte RAPID test is agene expression assay usingreverse transcription polymerasechain reaction to quantify therelative expression levels of hostresponse genes isolated fromwhole blood collected in thePAXgene Blood RNA Tube. TheSeptiCyte RAPID test is used inconjunction with clinicalassessments and other laboratoryfindings as an aid to differentiateinfection-positive (sepsis) frominfection-negative systemicinflammation in patientssuspected of sepsis on their firstday of ICU admission. TheSeptiCyte RAPID test generates ascore (SeptiScore) that fallswithin one of four discreteInterpretation Bands based on theincreasing likelihood of infection-positive systemic inflammation.SeptiCyte RAPID is intended forin-vitro diagnostic use on the</td><td colspan="1" rowspan="1">The SeptiCyte RAPID test is agene expression assay usingreverse transcription polymerasechain reaction to quantify therelative expression levels of hostresponse genes isolated fromwhole blood collected in thePAXgene Blood RNA Tube. TheSeptiCyte RAPID test is used inconjunction with clinicalassessments and other laboratoryfindings as an aid to differentiateinfection-positive (sepsis) frominfection-negative systemicinflammation in patients suspectedof sepsis on their first day of ICUadmission. The SeptiCyte RAPIDtest generates a score (SeptiScore)that falls within one of fourdiscrete Interpretation Bands basedon the increasing likelihood ofinfection-positive systemicinflammation. SeptiCyte RAPID isintended for in-vitro diagnostic useon the Biocartis Idylla System.</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Biocartis Idylla System.</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Intended Use Population</td><td colspan="1" rowspan="1">Same</td><td colspan="1" rowspan="1">Patients suspected of sepsis ontheir first day of Intensive CareUnit (ICU) admission</td></tr><tr><td colspan="1" rowspan="1">Specimen Type</td><td colspan="1" rowspan="1">Same</td><td colspan="1" rowspan="1">Whole blood collected inPAXgene Blood RNA tube</td></tr><tr><td colspan="1" rowspan="1">Assay Principle</td><td colspan="1" rowspan="1">Same</td><td colspan="1" rowspan="1">Quantitative gene expressionassay, based on real-timegeneration of fluorescence fromhydrolysis of dye-quencherhydrolysis probes during cycles ofPCR amplification of nucleic acidtemplates</td></tr><tr><td colspan="1" rowspan="1">General DeviceCharacteristic Differences</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Analytes</td><td colspan="1" rowspan="1">Two mRNA transcript immunebiomarkers: PLA2G7, PLAC8</td><td colspan="1" rowspan="1">Four mRNA transcript immunebiomarkers: PLA2G7, PLAC8,LAMP1, CEACAM4</td></tr><tr><td colspan="1" rowspan="1">Specimen Processing</td><td colspan="1" rowspan="1">Automated extraction of materialusing the Idylla System</td><td colspan="1" rowspan="1">1. Manual extraction of materialin PAXgene Blood RNATube, using the IVD versionof the QIAGEN PAXgeneBlood RNA Kit .2.Semi-automated extractionPAXgene Blood RNA Tube,using the QIAGEN QIAcubeSystem</td></tr><tr><td colspan="1" rowspan="1">Instrument Platform</td><td colspan="1" rowspan="1">Biocartis Idylla System</td><td colspan="1" rowspan="1">Applied Biosystems 7500 Fast DxReal-Time PCR Instrument</td></tr><tr><td colspan="1" rowspan="1">PCR Chemistry</td><td colspan="1" rowspan="1">One-step RT-qPCR; ThreeSingleplex reactions; Driedformat</td><td colspan="1" rowspan="1">Two-Step RT-qPCR; oneSingleplex, one Triplex reaction;Wet format</td></tr><tr><td colspan="1" rowspan="1">Controls</td><td colspan="1" rowspan="1">MS2 bacteriophage particles,serving as sample processingcontrol (SPC), i.e., as within-cartridge positive control for boththe sample extraction step and thecoupled RT-qPCR step.External controls not providedwith the assay but are described inlabeling with protocols availablefrom sponsor.</td><td colspan="1" rowspan="1">High Positive Control, LowPositive Control and NegativeControl for each in vitro transcript(LAMP1, CEACAM4, PLA2G7,PLAC8 (IVTs) formulated inneutral buffer, with IVTconcentrations designed toproduce high, medium, or lowSeptiSCORE values</td></tr><tr><td colspan="1" rowspan="1">Result Output</td><td colspan="1" rowspan="1">SeptiScore, calculated from theexpression levels of the twomRNA analytesPLA2G7, PLAC8. TheSeptiScore is placed into fourdiscrete bands that describe amonotonically increasinglikelihood of sepsis vs. SystemicInflammatory ResponseSyndrome (SIRS).</td><td colspan="1" rowspan="1">SeptiScore, calculated from theexpression levels of the fourmRNA analytes LAMP1,CEACAM4, PLA2G7, PLAC8,and describing the relativelikelihood of sepsis vs. SystemicInflammatory ResponseSyndrome (SIRS).</td></tr><tr><td colspan="1" rowspan="1">Limitations</td><td colspan="1" rowspan="1">SeptiCyte RAPID is for use onthe Idylla™M System only.</td><td colspan="1" rowspan="1">Slightly higher SeptiSCOREvalues were observed inAfrican Americans as compared</td></tr></table>

<table><tr><td>SeptiCyte RAPID is not findings. valid.</td><td>to Whites, Hisp intended for use in patients not Asians in the cli suspected of sepsis. A study of Predictive value healthy individuals showed &gt; likelihood ratios 50% of subjects with results in prevalence of di Band 2 (Asian American 70%, Laboratories an African American 53%, should establish Hispanic 48%, White 37%), reference interv and 16% in Band 3 or above. patient populati SeptiCyte® RAPID results SeptiCyte LAB should only be interpreted in potential source conjunction with clinical such as patient g assessment and other laboratory age, and prepara The SeptiCyte L Predictive values (estimated in conjunction w probabilities of sepsis or SIRS) assessments and are dependent on prevalence of findings as an ai disease and likelihood ratios infection-positiv measured for the clinical trial infection-negativ population. Users should inflammation in establish or verify that these suspected of sep day of ICU admi parameter values are appropriate for the patient assessment of wl population being tested. signs and sympto The advised sample volume for an infection or n the test is 900 µl whole blood always be based collected in PAXgene Blood consideration of RNA tubes. When using information, and samples that do not meet these solely on the Sep criteria, it is possible that the Test results. results will not be reliable or The results of tes with other condit Improper specimen collection, individuals (i.e., processing and handling can outside the Inten Population) may result in degraded RNA and may affect the results obtained SeptiSCORE tha with the test. of the interpretiv Using whole blood collected in However, those PAXgene Blood RNA tubes but indicate the prese not maintained at ambient of sepsis, infecti medical conditio temperature could generate invalid and/or incorrect results. This procedure is The diagnostic value of the measure in syste SeptiCyte RAPID test in inflamed patients suspected sepsis patients has of blood RNA tr not been validated in US biomarkers of th patients younger than 18. response to infec SeptiCyte RAPID test is not to from the instruct this package inse be used if the patient has a WBC count &lt; 25 cells/L. erroneous results quality of the RN SeptiCyte RAPID test has not the test results. been studied in patients with The SeptiCyte L neutropenia or is only designed immunocompromised patients. on the Applied Fast Dx Instrume A signal is gener</td></tr></table>

<table><tr><td></td><td>and absence of reverse transcriptase enzyme. The PLAC8 signal from genomic DNA should not be detected when using the SeptiCyte LAB Test following the recommended assay procedure. The diagnostic value of the SeptiCyte LAB Test in suspected sepsis patients has not been validated in US patients younger than 18. •The SeptiCyte LAB Test is not to be used if the patient has a WBC count &lt; 2 x 105 cells/mL. Predictive values depend on the likelihood ratios and the prevalence of disease. Laboratories and other users should establish their own reference intervals for their patient populations using the SeptiCyteTM Lab Test to reflect potential sources of variability, such as patient gender, race, age, and preparation techniques. If the RNA input per RT reaction for the SeptiCyte LAB Test is not within the linear range (i.e., 20 ng (2 ng/uL) to 500ng (50 ng/uL), the controls will indicate that the run passed even if the input RNA for the sample is not within the linear range.</td></tr></table>

# VI Standards/Guidance Documents Referenced:

Standard Standard Name   
EP05-A3 Evaluation of Precision of Quantitative Measurement Procedures   
EP07-A2 Interference Testing in Clinical Chemistry   
EP17-A2 Evaluation of Detection Capability for Clinical Laboratory Measurement Procedures   
EP25-A Evaluation of Stability of In Vitro Diagnostic Reagents   
Guidance:   
FDA Guidance for the Content of Premarket Submissions for Software Contained in Medical Devices   
(issued 11 May 2005)

VII Performance Characteristics (if/when applicable):

A Analytical Performance:

1. Precision/Reproducibility:

Repeatability and intermediate precision of the SeptiCyte Rapid was evaluated at one laboratory. Blood sample pools were constructed to yield High $( \sim 1 0 )$ , Medium $( \sim 6 )$ , or Low $( \sim 5 )$ SeptiSCORE values. Repeatability (within run precision) was tested with the low sample pool only, using within run values from testing performed over three days with two operators, three instruments and one cartridge lot. Results are summarized in the left-hand columns of Table 1 below. For intermediate precision, identical replicates (prepared as aliquots from the blood sample pools) were tested. A total of 144 runs were conducted, with 48 runs for each sample type (low medium, high), over a total of 12 days, using two different operators and three different instruments. Each operator made 24 runs on each of the three sample types (L, M, H), and each operator’s 24 runs were parsed as follows: (2 runs/ day) $\mathbf { X }$ (4 days) x (3 instruments). On each instrument, 16 runs were conducted with each of the three sample types $( \mathrm { L } , \mathrm { M } , \mathrm { H } )$ . The 16 runs for a given instrument and given sample type were parsed as follows: (2 runs/operator/day) x (2 operators) $\mathbf { X }$ (4 days). Results were also analyzed for each analyte separately.

The sponsor’s acceptance criterion $\mathrm { ( S D < 0 . 5 }$ units) was met for each component of precision, as well as for the precision calculated across the combination of instruments, operators, and days.

Table 1: Summary of Results for Repeatability and Intermediate Precision (SeptiScore)   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2>Repeatability</td><td rowspan=1 colspan=5>Intermediate Precision</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>AcrossDays</td><td rowspan=1 colspan=1>AcrossOperators</td><td rowspan=1 colspan=1>AcrossInstruments</td><td rowspan=1 colspan=1>TotalVariation</td></tr><tr><td rowspan=1 colspan=1>Pool</td><td rowspan=1 colspan=1>MEAN</td><td rowspan=1 colspan=1>CV (%)</td><td rowspan=1 colspan=1>MEAN</td><td rowspan=1 colspan=1>CV (%)</td><td rowspan=1 colspan=1>CV (%)</td><td rowspan=1 colspan=1>CV (%)</td><td rowspan=1 colspan=1>CV (%)</td></tr><tr><td rowspan=1 colspan=1>Low</td><td rowspan=1 colspan=1>4.56</td><td rowspan=1 colspan=1>4.0</td><td rowspan=1 colspan=1>4.48</td><td rowspan=1 colspan=1>4.5</td><td rowspan=1 colspan=1>4.8</td><td rowspan=1 colspan=1>4.9</td><td rowspan=1 colspan=1>5.0</td></tr><tr><td rowspan=1 colspan=1>Medium</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>6.33</td><td rowspan=1 colspan=1>4.0</td><td rowspan=1 colspan=1>3.9</td><td rowspan=1 colspan=1>4.3</td><td rowspan=1 colspan=1>4.3</td></tr><tr><td rowspan=1 colspan=1>High</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>10.33</td><td rowspan=1 colspan=1>1.8</td><td rowspan=1 colspan=1>1.9</td><td rowspan=1 colspan=1>1.9</td><td rowspan=1 colspan=1>2.0</td></tr></table>

# 2. Lot-to-lot Reproducibility

Samples prepared and used as above, i.e., a low, medium, and high pool, and were studied with three cartridge lots over three non-consecutive days, using two instruments and one operator. Each combination of lot, day, and instrument was tested with two replicates

Table 2: Summary of Results for Lot-to-Lot Reproducibility   

<table><tr><td rowspan=1 colspan=2></td><td rowspan=1 colspan=4>Lot-to-Lot reproducibility</td></tr><tr><td rowspan=1 colspan=1>Variable</td><td rowspan=1 colspan=1>Pool</td><td rowspan=1 colspan=1>MEAN</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>CV (%)</td></tr><tr><td rowspan=1 colspan=1>PLA2G7 Cq</td><td rowspan=1 colspan=1>Low</td><td rowspan=1 colspan=1>32.85</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>0.30</td><td rowspan=1 colspan=1>0.91</td></tr><tr><td rowspan=1 colspan=1>PLAC8 Cq</td><td rowspan=1 colspan=1>Low</td><td rowspan=1 colspan=1>28.49</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>0.33</td><td rowspan=1 colspan=1>1.17</td></tr><tr><td rowspan=1 colspan=1>SeptiScore</td><td rowspan=1 colspan=1>Low</td><td rowspan=1 colspan=1>4.36</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>0.14</td><td rowspan=1 colspan=1>3.31</td></tr><tr><td rowspan=1 colspan=1>PLA2G7 Cq</td><td rowspan=1 colspan=1>Medium</td><td rowspan=1 colspan=1>31.80</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>0.21</td><td rowspan=1 colspan=1>0.66</td></tr><tr><td rowspan=1 colspan=1>PLAC8 Cq</td><td rowspan=1 colspan=1>Medium</td><td rowspan=1 colspan=1>25.56</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>0.21</td><td rowspan=1 colspan=1>0.80</td></tr><tr><td rowspan=1 colspan=1>SeptiScore</td><td rowspan=1 colspan=1>Medium</td><td rowspan=1 colspan=1>6.25</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>0.19</td><td rowspan=1 colspan=1>3.01</td></tr><tr><td rowspan=1 colspan=1>PLA2G7 Cq</td><td rowspan=1 colspan=1>High</td><td rowspan=1 colspan=1>33.87</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>0.40</td><td rowspan=1 colspan=1>1.17</td></tr><tr><td rowspan=1 colspan=1>PLAC8 Cq</td><td rowspan=1 colspan=1>High</td><td rowspan=1 colspan=1>23.67</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>0.33</td><td rowspan=1 colspan=1>1.38</td></tr><tr><td rowspan=1 colspan=1>SeptiScore</td><td rowspan=1 colspan=1>High</td><td rowspan=1 colspan=1>10.20</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>0.19</td><td rowspan=1 colspan=1>1.82</td></tr></table>

The $\% C V s$ for each parameter and for the overall SeptiScore were acceptable and below the acceptance criteria of ${ < } 4 \%$ .

# 3. Linearity:

Linearity was defined within the sponsor’s study for the assay reportable range and assessed by the relationship between WBC increase and PLAC8 and PLA2G7 increase. as measured by linear $( \mathrm { r } ^ { 2 } > 0 . 8 5 )$ reaction dynamics (i.e., $\mathrm { C q }$ inversely proportional to log of input WBC amount) for each of the two transcripts, over the claimed input range. Ranges tested and results for linearity testing are summarized in Table 3 and Figure 1 below:

Table 3: Results for linearity testing.   

<table><tr><td rowspan=1 colspan=1>Sample ID</td><td rowspan=1 colspan=1>WBC Concentration(cells/ml)</td><td rowspan=1 colspan=1>Total Tests (N)</td><td rowspan=1 colspan=1>Mean SeptiScore</td><td rowspan=1 colspan=1>SD SeptiScore</td><td rowspan=1 colspan=1>CV (%)</td></tr><tr><td rowspan=1 colspan=1>WBC1</td><td rowspan=1 colspan=1>50 x 106</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>3.81</td><td rowspan=1 colspan=1>0.40</td><td rowspan=1 colspan=1>10.5</td></tr><tr><td rowspan=1 colspan=1>WBC2</td><td rowspan=1 colspan=1>25 x 106</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>3.22</td><td rowspan=1 colspan=1>0.17</td><td rowspan=1 colspan=1>5.3</td></tr><tr><td rowspan=1 colspan=1>WBC3</td><td rowspan=1 colspan=1>12.5 x 106</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>3.59</td><td rowspan=1 colspan=1>0.22</td><td rowspan=1 colspan=1>6.1</td></tr><tr><td rowspan=1 colspan=1>WBC4</td><td rowspan=1 colspan=1>5 x 106</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>3.92</td><td rowspan=1 colspan=1>0.24</td><td rowspan=1 colspan=1>6.0</td></tr><tr><td rowspan=1 colspan=1>WBC5</td><td rowspan=1 colspan=1>2.5 x 106</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>3.70</td><td rowspan=1 colspan=1>0.25</td><td rowspan=1 colspan=1>6.7</td></tr><tr><td rowspan=1 colspan=1>WBC6</td><td rowspan=1 colspan=1>1 x 106</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>4.14</td><td rowspan=1 colspan=1>0.41</td><td rowspan=1 colspan=1>9.8</td></tr><tr><td rowspan=1 colspan=1>WBC7</td><td rowspan=1 colspan=1>0.25 x 106</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>4.19</td><td rowspan=1 colspan=1>0.30</td><td rowspan=1 colspan=1>7.3</td></tr></table>

![](images/b3966e8c8eba74bf95ef0c0198cddb02272a005f935b451bd8f50c8589ce6c51.jpg)  
Figure 1: Summary of analyte Cq values from the reportable range study. Cq values versus WBC input concentration for (A) PLA2G7, (B) PLAC8, and (C) SeptiScore

The sponsor notes that a small Hook effect was noted at high WBC levels $( > 2 5 \mathrm { x } 1 0 ^ { 6 }$ WBC cells/ml; however, this appears to be similar for both analytes and is compensated by score calculation which is the difference of these two analytes.

These results are considered acceptable.

# 4. Detection limit

a. Limit of Blank:

Platelet reduced plasma was used to determine the Limit of Blank as the sponsor states that leukocyte depleted blood used for the predicate was unavailable. Ten replicates of platelet reduced plasma did not generate a PCR signal $\mathrm { ( C q > 4 0 ) }$ for PLA2G7 and, accordingly, did not generate a SeptiScore,

b. Limit of Detection

Different concentrations of white blood cells (WBC) from a single donor into a matrix consisting of platelet-reduced plasma to determine the level at which a SeptiScore was generated, i.e., the LoD was defined as the lowest titer for which at least $9 5 \%$ (19/20) WBC replicates generate a SeptiScore.

Table 4: Results for Limit of Detection   

<table><tr><td rowspan=1 colspan=1>Sample ID</td><td rowspan=1 colspan=1>WBC Concentration(cells/ml)</td><td rowspan=1 colspan=1>InvalidTests (N)</td><td rowspan=1 colspan=1>Total Tests(N)</td><td rowspan=1 colspan=1>Failure Rate</td><td rowspan=1 colspan=1>Result</td></tr><tr><td rowspan=1 colspan=1>WBC4</td><td rowspan=1 colspan=1>5x 106</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>Above LoD</td></tr><tr><td rowspan=1 colspan=1>WBC5</td><td rowspan=1 colspan=1>2.5 x 106</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>Above LoD</td></tr><tr><td rowspan=1 colspan=1>WBC6</td><td rowspan=1 colspan=1>1 x 106</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>Above LoD</td></tr><tr><td rowspan=1 colspan=1>WBC7</td><td rowspan=1 colspan=1>0.25 x 106</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>Above LoD</td></tr><tr><td rowspan=1 colspan=1>WBC8</td><td rowspan=1 colspan=1>0.1 x 106</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>Above LoD</td></tr><tr><td rowspan=1 colspan=1>WBC9</td><td rowspan=1 colspan=1>0.025 x 106</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>LoD</td></tr><tr><td rowspan=1 colspan=1>WBC10</td><td rowspan=1 colspan=1>0.001 x 106</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>50%</td><td rowspan=1 colspan=1>Below LoD</td></tr><tr><td rowspan=1 colspan=1>WBC11</td><td rowspan=1 colspan=1>0.025 x 105</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>Below LoD</td></tr><tr><td rowspan=1 colspan=1>WBC12</td><td rowspan=1 colspan=1>0.001 x 105</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>Below LoD</td></tr></table>

LoD was defined as $0 . 0 2 5 \mathrm { ~ x ~ } 1 0 ^ { 6 }$ WBC cells/ml, below the level expected with clinical use.

c. Limit of Quantitation

LoQ was defined as the lowest WBC concentration for which at least $9 5 \%$ of WBC replicates generate a SeptiScore with a standard deviation of 1.0 or less; results indicated that the LoQ was similar to the LoD at $0 . 0 2 5 \mathrm { ~ x ~ } 1 0 ^ { 6 }$ WBC cells/ml.

Table 5: Results for Limit of Quantitation   

<table><tr><td rowspan=1 colspan=1>Sample ID</td><td rowspan=1 colspan=1>WBC Concentration(cells/ml)</td><td rowspan=1 colspan=1>ValidTests (N)</td><td rowspan=1 colspan=1>TotalTests (N)</td><td rowspan=1 colspan=1>FailureRate</td><td rowspan=1 colspan=1>MeanSeptiScore</td><td rowspan=1 colspan=1>Std DevSeptiScore</td><td rowspan=1 colspan=1>CV (%)</td><td rowspan=1 colspan=1>Result</td></tr><tr><td rowspan=1 colspan=1>WBC4</td><td rowspan=1 colspan=1>5 x 106</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>3.92</td><td rowspan=1 colspan=1>0.24</td><td rowspan=1 colspan=1>6.0</td><td rowspan=1 colspan=1>Above LoQ</td></tr><tr><td rowspan=1 colspan=1>WBC5</td><td rowspan=1 colspan=1>2.5 x 106</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>3.70</td><td rowspan=1 colspan=1>0.25</td><td rowspan=1 colspan=1>6.7</td><td rowspan=1 colspan=1>Above LoQ</td></tr><tr><td rowspan=1 colspan=1>WBC6</td><td rowspan=1 colspan=1>1 x 106</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>4.14</td><td rowspan=1 colspan=1>0.41</td><td rowspan=1 colspan=1>9.9</td><td rowspan=1 colspan=1>Above LoQ</td></tr><tr><td rowspan=1 colspan=1>WBC7</td><td rowspan=1 colspan=1>0.25 x 106</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>4.19</td><td rowspan=1 colspan=1>0.30</td><td rowspan=1 colspan=1>7.2</td><td rowspan=1 colspan=1>Above LoQ</td></tr><tr><td rowspan=1 colspan=1>WBC8</td><td rowspan=1 colspan=1>0.1 x 106</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>4.06</td><td rowspan=1 colspan=1>0.29</td><td rowspan=1 colspan=1>7.1</td><td rowspan=1 colspan=1>Above LoQ</td></tr><tr><td rowspan=1 colspan=1>WBC9</td><td rowspan=1 colspan=1>0.025 x 106</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>4.00</td><td rowspan=1 colspan=1>0.63</td><td rowspan=1 colspan=1>15.9</td><td rowspan=1 colspan=1>LoQ</td></tr><tr><td rowspan=1 colspan=1>WBC10</td><td rowspan=1 colspan=1>0.001 x 106</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>50%</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>Below LoQ</td></tr><tr><td rowspan=1 colspan=1>WBC11</td><td rowspan=1 colspan=1>0.025 x 105</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>Below LoQ</td></tr><tr><td rowspan=1 colspan=1>WBC12</td><td rowspan=1 colspan=1>0.001 x 105</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>Below LoQ</td></tr></table>

The LoD observed is acceptable for clinical practice and would include all patient groups with the exception of patients with severe neutropenia.

# 5. Analytical Specificity/Interference:

a. Analytical Specificity:

Analytical specificity was tested by reactivity against blank samples and genomic DNA and both transcripts are endogenous markers. For blank cartridges, 20/20 blank cartridges (PBS mixed with PAXgene RNA stabilizer) generated undetermined or invalid SeptiScore values; 10 cartridges with no sample added similarly generated undetermined or invalid SeptiScore values. External positive controls run the same day generated expected results.

For reactivity against genomic DNA, testing was conducted on 10 replicates of each of two commercially procured total human gDNA samples. Each lot was tested at $5 0 0 \mathrm { n g }$ and $1 0 0 0 \mathrm { n g }$ input per reaction.

Table 6: Specimen Panel Characteristics and Expected Results   

<table><tr><td rowspan=1 colspan=1>Sample Description</td><td rowspan=1 colspan=1>Total Reps</td><td rowspan=1 colspan=1>SeptiScore Expected?</td></tr><tr><td rowspan=1 colspan=1>gDNA #1 500ng</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>No</td></tr><tr><td rowspan=1 colspan=1>gDNA #2 500ng</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>No</td></tr><tr><td rowspan=1 colspan=1>gDNA #1 1000ng</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>No</td></tr><tr><td rowspan=1 colspan=1>gDNA #2 1000ng</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>No</td></tr><tr><td rowspan=1 colspan=1>Total Assays</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1></td></tr></table>

The SeptiCyte RAPID test produced invalid SeptiScore values for each of the 40 reactions run in this part of the study, with no valid $\operatorname { C q }$ values generated for any of the cartridges used to test the $\mathrm { g D N A }$ samples.

# b. Interferants

A specimen panel was prepared by spiking potential interferents into pooled, PAXgene-stabilized blood collected from healthy donors. The potential interferents were dissolved in appropriate solvents, and then spiked into the PAXgene blood samples at concentrations higher than the maxima of their normal or expected reference ranges. Three replicates were tested at each concentration of potential interferent and compared to matched PAXgene blood controls (containing the appropriate solvent but no interferent). The substances tested, concentration of each, and results are reproduced below in Table 7:

Table 7: Effect of Potential Interferants on SeptiScore Results   

<table><tr><td rowspan=1 colspan=1>Sample ID</td><td rowspan=1 colspan=1>Potential Interferent</td><td rowspan=1 colspan=1>Concentration</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>MeanSeptiScore</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>DeltaSeptiScore(relative tocontrol)</td><td rowspan=1 colspan=1>Result</td></tr><tr><td rowspan=1 colspan=1>INF1</td><td rowspan=1 colspan=1>Rheumatoid Factor</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>4.24</td><td rowspan=1 colspan=1>0.11</td><td rowspan=1 colspan=1>0.22</td><td rowspan=1 colspan=1>PASS</td></tr><tr><td rowspan=1 colspan=1>INF2</td><td rowspan=1 colspan=1>Heparin</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>4.24</td><td rowspan=1 colspan=1>0.09</td><td rowspan=1 colspan=1>0.22</td><td rowspan=1 colspan=1>PASS</td></tr><tr><td rowspan=1 colspan=1>INF3</td><td rowspan=1 colspan=1>Imipenem</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>4.36</td><td rowspan=1 colspan=1>0.30</td><td rowspan=1 colspan=1>0.10</td><td rowspan=1 colspan=1>PASS</td></tr><tr><td rowspan=1 colspan=1>INF4</td><td rowspan=1 colspan=1>Bilirubin</td><td rowspan=1 colspan=1>45 IU/mL</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>4.30</td><td rowspan=1 colspan=1>0.13</td><td rowspan=1 colspan=1>0.16</td><td rowspan=1 colspan=1>PASS</td></tr><tr><td rowspan=1 colspan=1>INF5</td><td rowspan=1 colspan=1>Triglycerides</td><td rowspan=1 colspan=1>3000 U/L</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>4.18</td><td rowspan=1 colspan=1>0.05</td><td rowspan=1 colspan=1>0.28</td><td rowspan=1 colspan=1>PASS</td></tr><tr><td rowspan=1 colspan=1>INF6</td><td rowspan=1 colspan=1>Vancomycin</td><td rowspan=1 colspan=1>1.2 mg/mL</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>4.68</td><td rowspan=1 colspan=1>0.20</td><td rowspan=1 colspan=1>0.22</td><td rowspan=1 colspan=1>PASS</td></tr><tr><td rowspan=1 colspan=1>INF7</td><td rowspan=1 colspan=1>Cefotaxime</td><td rowspan=1 colspan=1>20 mg/dL</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>4.27</td><td rowspan=1 colspan=1>0.15</td><td rowspan=1 colspan=1>0.19</td><td rowspan=1 colspan=1>PASS</td></tr><tr><td rowspan=1 colspan=1>INF8</td><td rowspan=1 colspan=1>Dopamine</td><td rowspan=1 colspan=1>500 mg/dL</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>4.31</td><td rowspan=1 colspan=1>0.20</td><td rowspan=1 colspan=1>0.15</td><td rowspan=1 colspan=1>PASS</td></tr><tr><td rowspan=1 colspan=1>INF9</td><td rowspan=1 colspan=1>C-reactive protein(CRP)</td><td rowspan=1 colspan=1>70 μmol/L</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>4.24</td><td rowspan=1 colspan=1>0.04</td><td rowspan=1 colspan=1>0.22</td><td rowspan=1 colspan=1>PASS</td></tr><tr><td rowspan=1 colspan=1>INF10</td><td rowspan=1 colspan=1>Noradrenaline</td><td rowspan=1 colspan=1>670 μmol/L</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>4.39</td><td rowspan=1 colspan=1>0.09</td><td rowspan=1 colspan=1>0.07</td><td rowspan=1 colspan=1>PASS</td></tr><tr><td rowspan=1 colspan=1>INF11</td><td rowspan=1 colspan=1>Dobutamine</td><td rowspan=1 colspan=1>6.0 μmol/L</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>4.49</td><td rowspan=1 colspan=1>0.24</td><td rowspan=1 colspan=1>0.03</td><td rowspan=1 colspan=1>PASS</td></tr><tr><td rowspan=1 colspan=1>INF12</td><td rowspan=1 colspan=1>Hemoglobin</td><td rowspan=1 colspan=1>4 mg/dL</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>4.31</td><td rowspan=1 colspan=1>0.11</td><td rowspan=1 colspan=1>0.15</td><td rowspan=1 colspan=1>PASS</td></tr><tr><td rowspan=1 colspan=1>INF13</td><td rowspan=1 colspan=1>Albumin</td><td rowspan=1 colspan=1>700 pg/mL</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>4.27</td><td rowspan=1 colspan=1>0.14</td><td rowspan=1 colspan=1>0.19</td><td rowspan=1 colspan=1>PASS</td></tr><tr><td rowspan=1 colspan=1>INF14</td><td rowspan=1 colspan=1>Furosemide</td><td rowspan=1 colspan=1>12 μg/mL</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>4.44</td><td rowspan=1 colspan=1>0.10</td><td rowspan=1 colspan=1>0.18</td><td rowspan=1 colspan=1>PASS</td></tr><tr><td rowspan=1 colspan=1>INF15</td><td rowspan=1 colspan=1>Soluble CD14(sCD14)</td><td rowspan=1 colspan=1>20 g/dL</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>4.48</td><td rowspan=1 colspan=1>0.28</td><td rowspan=1 colspan=1>0.02</td><td rowspan=1 colspan=1>PASS</td></tr><tr><td rowspan=1 colspan=1>INF16</td><td rowspan=1 colspan=1>IL-6</td><td rowspan=1 colspan=1>5 g/dL</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>4.40</td><td rowspan=1 colspan=1>0.20</td><td rowspan=1 colspan=1>0.06</td><td rowspan=1 colspan=1>PASS</td></tr><tr><td rowspan=1 colspan=1>INF17</td><td rowspan=1 colspan=1>Lipopolysaccharidebinding protein(LBP)</td><td rowspan=1 colspan=1>180 μmol/L</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>4.33</td><td rowspan=1 colspan=1>0.10</td><td rowspan=1 colspan=1>0.12</td><td rowspan=1 colspan=1>PASS</td></tr><tr><td rowspan=1 colspan=1>CON1</td><td rowspan=1 colspan=1>Solvent Control -Methanol/Blood</td><td rowspan=1 colspan=1>5 μg/mL</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>4.25</td><td rowspan=1 colspan=1>0.12</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td></tr><tr><td rowspan=1 colspan=1>CON2</td><td rowspan=1 colspan=1>Solvent Control -Water/Blood</td><td rowspan=1 colspan=1>15 pg/mL</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>4.46</td><td rowspan=1 colspan=1>0.21</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td></tr><tr><td rowspan=1 colspan=1>CON3</td><td rowspan=1 colspan=1>Solvent Control -PBS/Blood</td><td rowspan=1 colspan=1>45 μg/mL</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>4.45</td><td rowspan=1 colspan=1>0.07</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td></tr></table>

Results were consistent with no effects observed from the interferants tested.

6. Assay Reportable Range: See linearity above

7. Traceability, Stability, Expected Values (Controls, Calibrators, or Methods):

a. Real-time Stability

Real-time stability studies are ongoing to support product claims. Real-time stability study protocols and acceptance criteria were reviewed and found to be acceptable.

In-use stability

The performance of the SeptiCyte RAPID test cartridge was evaluated when removed from the pouch and left on the lab bench for a period of time, both with and without the addition of sample to the cartridge. Specifically, test cartridges were removed from their pouches and placed at $3 0 ^ { \circ } \mathrm { C }$ and $7 5 \%$ relative humidity for up to 3 days. No significant degradation in performance was observed and these data support the sponsor’s recommendation in labeling that testing should be performed within one hour of adding samples to the pouch.

# b. Shipping Stability

Shipping simulations were performed to evaluate robustness of the SeptiCyte RAPID test to thermal cycles as well as shocks and vibrations of a transport cycle when placed in the packaging used for international shipping. To simulate thermal stress, cartridges were subjected to three winter cycles and a summer cycle (2-3 days at ${ } _ { - 2 0 } \mathrm { { } ^ { \circ } C }$ followed by a day at $2 5 \mathrm { { ^ \circ C } }$ for a winter cycle, and 3 days at $4 0 \mathrm { { } ^ { \circ } C }$ for a summer cycle). Immediately after thermal stress, 6 cartridges were tested with PAXgene citrate whole blood samples $\mathrm { 9 0 0 u l }$ tested per cartridge). As with other stability studies, a maximum drift of $\pm 1 0 \%$ was considered acceptable. All samples evaluated are part of this study successfully met the acceptance criteria. These results confirm that secondary packaging is sufficient to maintain stable performance of the test for shipping conditions anticipated during distribution of the product.

# c. PAXgene Sample Collection/Handling

The sample collection study evaluated stability of the test result both when samples were processed either immediately after a blood draw or when frozen prior to testing. Blood samples were collected from each of 15 healthy donors with the first sample processed in real-time and tested both immediately and after a 120-minute incubation at room temperature to mimic the sample processing conditions for the PAXgene sample tubes. Results from the stability test of freshly collected blood samples demonstrated no significant differences in SeptiCyte score when evaluated immediately or after a 120-minute incubation. These data support SeptiScore testing immediately after blood draw or after sample processing according to the manufacture instructions for the PAXgene sample tube.

To support the use of banked frozen samples in their validation studies, the sponsor performed an equivalency study between fresh and frozen specimens. Specimens processed immediately were compared to paired samples that underwent one freeze thaw cycle and were stored for one month at ${ < } _ { - } 7 0 { ^ \circ } \mathrm { C }$ . Across all evaluated samples, the mean Ct shift in the PLA2G7 transcript Cq was 0.26, the mean shift in PLAC8 was 0.2, and the mean shift in SeptiScore was 0.06. Cumulatively, these results demonstrate that a single freeze-thaw cycle did not significantly affect SeptiScore reported results.

8. Detection Limit: See Section A.4 above.

9. Assay Cut-Off: See clinical cutoff (Section D) below.

# B Comparison Studies:

1. Method Comparison with Predicate Device: N/A

2. Matrix Comparison: N/A

# C Clinical Studies:

1. Clinical Sensitivity:

Clinical performance of the SeptiCyte Rapid was evaluated in a manner similar to that for the Septicyte LAB predicate (K163260). Residual samples with sufficient volume remaining from the two observational, non-interventional, prospective clinical trials (the Molecular Diagnosis and Risk Stratification of Sepsis [MARS- NCT01905033] study and the Septic Gene Expression Using SeptiCyte [VENUS - NCT02127502] study) used for validation of the Septicyte LAB predicate were tested. The MARS and VENUS studies were conducted across eight clinical sites in the United States and Europe. Of the original 447 subjects, PAXgene Blood RNA tubes were available for 356. This represented ${ \sim } 8 0 \%$ of the original cohort: 139/198 of the sample from the MARS study $7 0 . 2 \%$ representation), 101/129 from the original Venus sample ( $7 8 . 3 \%$ representation), and 116/120 of the subjects from the Venus Supplement $( 9 6 . 7 \%$ representation).

An additional 30 prospective specimens were collected as part of the NEPTUNE (NEar PatienT MolecUlar TestiNg in Sepsis, NEPTUNE) study. The NEPTUNE protocol was a prospective study similar in design to the MARS and VENUS studies; however, this study was terminated early due to difficulty enrollment with the onset of the COVID epidemic. Study protocols were similar enough that combining subject results was considered acceptable.

For the retrospective studies, the primary objective was to determine the diagnostic performance of SeptiCyte LAB in distinguishing sepsis from SIRS in adult critical care patients, either as a stand-alone test or in combination with other clinical variables and laboratory assessments used to confirm or exclude a diagnosis of sepsis. A “sepsis event” was defined operationally to have occurred when a patient displayed two or more signs of systemic inflammation and was given therapeutic systemic antibiotics by the attending clinician within 24 hours of ICU admission.

Adult critical care patients aged 18–89 years were considered qualified for enrollment if, upon admission to an ICU, they displayed two or more of the following signs:

temperature above $3 8 \mathrm { { ^ \circ C } }$ or less than $3 6 \mathrm { { ^ \circ C } }$ heart rate greater than 90 beats per minute tachypnea greater than 20 breaths per minute or PaCO2 less than $3 2 \mathrm { m m H g }$ White blood cell (WBC) count greater than $1 2 , 0 0 0 / \mathrm { m m } 3$ or less than $4 { , } 0 0 0 / \mathrm { m m } 3$ or greater than $10 \%$ immature neutrophils (bands).

Exclusion criteria included:

Clinical cultures or serologic tests were not obtained when sepsis was suspected ICU admission more than 24 hours before trial enrollment Delay of more than 24 hours between trial enrollment and sample draw • Treatment with antibiotics more than 24 hours before ICU admission for any reason other than surgical prophylaxis (VENUS and Venus Supplement only), or Documented sepsis event that occurred more than 3 days before ICU admission or more than 2 days after ICU admission (MARS only).

The SeptiScore for each subject was compared to a Retrospective Physician Diagnosis (RPD) comparator based on clinical case review by medical experts who independently reviewed clinical information about each subject while remaining blinded to the SeptiCyte LAB results. Case reviews occurred after the patient was discharged from the hospital. The RPD Case Review Panel based their diagnosis on clinical data. RPD assessment and analyses were approached three ways (four in indeterminate as sepsis is considered):

1. Forced RPD: Subjects were stratified by diagnosis (Systemic Inflammatory Response Syndrome (SIRS) or sepsis) according to the majority opinion of the RPD Case Review Panelists. The category of Indeterminate was not allowed; all subjects were forced into either the SIRS or sepsis categories, with no subjects excluded. The performance evaluation of the SeptiCyte LAB was based on the Forced RPD.

2. Consensus RPD: Subjects were stratified by diagnosis (SIRS, sepsis or indeterminate) according to the majority opinion of three expert reviewers. Indeterminate cases $\left( \mathrm { n } { = } 3 7 \right)$ ) were those for which a definitive diagnosis could not be reached by expert review and were not included in the analysis. Indeterminate as sepsis: An analysis where subjects with indeterminate results were considered as sepsis was also included.

3. Unanimous RPD: the subset of subjects where all three assessments were in agreement

The inclusion and exclusion criteria for the additional prospective subjects was similar to that for retrospective subjects, as was the method for RPD diagnosis, allowing data to be effectively pooled.

The following is the summary of demographic data for the retrospective subjects from the sponsor’s submission:

Table 8: Demographic Characteristics of Retrospectively Archived Subjects $( \mathbf { n } = 3 5 6 )$ .   

<table><tr><td rowspan=1 colspan=2>Demographic &amp; Clinical Variables</td><td rowspan=1 colspan=1>Indeterminate(Consensus)N (%)</td><td rowspan=1 colspan=1>SIRS(Consensus)N (%)</td><td rowspan=1 colspan=1>Sepsis(Consensus)N (%)</td></tr><tr><td rowspan=2 colspan=1>Sex</td><td rowspan=1 colspan=1>Female</td><td rowspan=1 colspan=1>13 (3.7%)</td><td rowspan=1 colspan=1>86 (24.2%)</td><td rowspan=1 colspan=1>58 (16.3%)</td></tr><tr><td rowspan=1 colspan=1>Male</td><td rowspan=1 colspan=1>19 (5.3%)</td><td rowspan=1 colspan=1>106 (29.8%)</td><td rowspan=1 colspan=1>74 (20.8%)</td></tr><tr><td rowspan=5 colspan=1>Race/Ethnicity</td><td rowspan=1 colspan=1>Asian</td><td rowspan=1 colspan=1>1 (0.28%)</td><td rowspan=1 colspan=1>10 (2.81%)</td><td rowspan=1 colspan=1>9 (2.53%)</td></tr><tr><td rowspan=1 colspan=1>Black (American or European of African descent)</td><td rowspan=1 colspan=1>9 (2.53%)</td><td rowspan=1 colspan=1>47 (13.20%)</td><td rowspan=1 colspan=1>26 (7.30%)</td></tr><tr><td rowspan=1 colspan=1>Hispanic</td><td rowspan=1 colspan=1>0 (0%)</td><td rowspan=1 colspan=1>6 (1.69%)</td><td rowspan=1 colspan=1>4 (1.12%)</td></tr><tr><td rowspan=1 colspan=1>Unknown</td><td rowspan=1 colspan=1>0 (0%)</td><td rowspan=1 colspan=1>2 (0.56%)</td><td rowspan=1 colspan=1>2 (0.56%)</td></tr><tr><td rowspan=1 colspan=1>White (Caucasian)</td><td rowspan=1 colspan=1>22 (6.18%)</td><td rowspan=1 colspan=1>127 (35.67%)</td><td rowspan=1 colspan=1>91 (25.56%)</td></tr><tr><td rowspan=7 colspan=1>Source ofadmission</td><td rowspan=1 colspan=1>Emergency department</td><td rowspan=1 colspan=1>22 (6.18%)</td><td rowspan=1 colspan=1>120 (33.71%)</td><td rowspan=1 colspan=1>81 (22.75%)</td></tr><tr><td rowspan=1 colspan=1>Nursing ward</td><td rowspan=1 colspan=1>3 (0.84%)</td><td rowspan=1 colspan=1>13 (3.65%)</td><td rowspan=1 colspan=1>25 (7.02%)</td></tr><tr><td rowspan=1 colspan=1>Other</td><td rowspan=1 colspan=1>1 (0.28%)</td><td rowspan=1 colspan=1>13 (3.65%)</td><td rowspan=1 colspan=1>6 (1.69%)</td></tr><tr><td rowspan=1 colspan=1>Post-anesthesia care facility</td><td rowspan=1 colspan=1>0 (0%)</td><td rowspan=1 colspan=1>14 (3.93%)</td><td rowspan=1 colspan=1>2 (0.56%)</td></tr><tr><td rowspan=1 colspan=1>Coronary care facility</td><td rowspan=1 colspan=1>0 (0%)</td><td rowspan=1 colspan=1>4 (1.12%)</td><td rowspan=1 colspan=1>1 (0.28%)</td></tr><tr><td rowspan=1 colspan=1>Intensive Care Unit, other hospital</td><td rowspan=1 colspan=1>3 (0.84%)</td><td rowspan=1 colspan=1>14 (3.93%)</td><td rowspan=1 colspan=1>12 (3.37%)</td></tr><tr><td rowspan=1 colspan=1>Operating theatre</td><td rowspan=1 colspan=1>3 (0.84%)</td><td rowspan=1 colspan=1>14 (3.93%)</td><td rowspan=1 colspan=1>5 (1.40%)</td></tr><tr><td rowspan=4 colspan=1>Impression onDischarge(Consensus of 3x PrincipalInvestigators)</td><td rowspan=1 colspan=1>Definite Infection</td><td rowspan=1 colspan=1>3 (0.84%)</td><td rowspan=1 colspan=1>0 (0%)</td><td rowspan=1 colspan=1>78 (21.91%)</td></tr><tr><td rowspan=1 colspan=1>Possible Infection</td><td rowspan=1 colspan=1>14 (3.93%)</td><td rowspan=1 colspan=1>5 (1.40%)</td><td rowspan=1 colspan=1>26 (7.30%)</td></tr><tr><td rowspan=1 colspan=1>Probable Infection</td><td rowspan=1 colspan=1>3 (0.84%)</td><td rowspan=1 colspan=1>0 (0%)</td><td rowspan=1 colspan=1>23 (6.46%)</td></tr><tr><td rowspan=1 colspan=1>SIRS</td><td rowspan=1 colspan=1>12 (3.37%)</td><td rowspan=1 colspan=1>187 (52.53%)</td><td rowspan=1 colspan=1>5 (1.40%)</td></tr><tr><td rowspan=3 colspan=1>Death</td><td rowspan=1 colspan=1>Unknown</td><td rowspan=1 colspan=1>0 (0%)</td><td rowspan=1 colspan=1>0 (0%)</td><td rowspan=1 colspan=1>1 (0.28%)</td></tr><tr><td rowspan=1 colspan=1>No</td><td rowspan=1 colspan=1>29 (8.15%)</td><td rowspan=1 colspan=1>173 (48.60%)</td><td rowspan=1 colspan=1>111 (31.18%)</td></tr><tr><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>3 (0.84%)</td><td rowspan=1 colspan=1>19 (5.34%)</td><td rowspan=1 colspan=1>20 (5.62%)</td></tr><tr><td rowspan=2 colspan=1>InvasiveMechanicalVentilation</td><td rowspan=1 colspan=1>No</td><td rowspan=1 colspan=1>19 (5.3%)</td><td rowspan=1 colspan=1>116 (32.6%)</td><td rowspan=1 colspan=1>87 (24.4%)</td></tr><tr><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>13 (3.7%)</td><td rowspan=1 colspan=1>76 (21.3%)</td><td rowspan=1 colspan=1>45 (12.6%)</td></tr></table>

Table 9: Demographic Characteristics of Prospective Subjects $( \mathbf { n } = 3 \mathbf { 0 } )$   

<table><tr><td rowspan=1 colspan=5>Demographic &amp; Clinical Variables</td><td rowspan=1 colspan=1>Indeterminate(Consensus)N (%)</td><td rowspan=1 colspan=1>SIRS(Consensus)N (%)</td><td rowspan=1 colspan=1>Sepsis(Consensus)N (%)</td></tr><tr><td rowspan=2 colspan=4>Sex</td><td rowspan=1 colspan=1>Female</td><td rowspan=1 colspan=1>2 (6.7%)</td><td rowspan=1 colspan=1>4 (13.3%)</td><td rowspan=1 colspan=1>7 (23.3%)</td></tr><tr><td rowspan=1 colspan=1>Male</td><td rowspan=1 colspan=1>4 (13.3%)</td><td rowspan=1 colspan=1>8 (26.7%)</td><td rowspan=1 colspan=1>5 (16.7%)</td></tr><tr><td rowspan=6 colspan=4>Ethnicity/Race</td><td rowspan=1 colspan=1>Asian</td><td rowspan=1 colspan=1>1 (3.3%)</td><td rowspan=1 colspan=1>0 (0%)</td><td rowspan=1 colspan=1>0 (0%)</td></tr><tr><td rowspan=1 colspan=1>Black or African American</td><td rowspan=1 colspan=1>2 (6.7%)</td><td rowspan=1 colspan=1>3 (10.0%)</td><td rowspan=1 colspan=1>2 (6.7%)</td></tr><tr><td rowspan=1 colspan=1>Hispanic</td><td rowspan=1 colspan=1>1 (3.3%)</td><td rowspan=1 colspan=1>5 (16.7%)</td><td rowspan=1 colspan=1>5 (16.7%)</td></tr><tr><td rowspan=1 colspan=1>Other</td><td rowspan=1 colspan=1>0 (0%)</td><td rowspan=1 colspan=1>1 (3.3%)</td><td rowspan=1 colspan=1>1 (3.3%)</td></tr><tr><td rowspan=1 colspan=1>Unknown</td><td rowspan=1 colspan=1>1 (3.3%)</td><td rowspan=1 colspan=1>0 (0%)</td><td rowspan=1 colspan=1>0 (0%)</td></tr><tr><td rowspan=1 colspan=1>White</td><td rowspan=1 colspan=1>1 (3.3%)</td><td rowspan=1 colspan=1>3 (10.0%)</td><td rowspan=1 colspan=1>4 (13.3%)</td></tr><tr><td rowspan=5 colspan=4>Source ofAdmission</td><td rowspan=1 colspan=1>ED</td><td rowspan=1 colspan=1>5 (16.7 %)</td><td rowspan=1 colspan=1>6 (20.0%)</td><td rowspan=1 colspan=1>7 (23.3%)</td></tr><tr><td rowspan=1 colspan=1>ICU</td><td rowspan=1 colspan=1>1 (3.3%)</td><td rowspan=1 colspan=1>1 (3.3%)</td><td rowspan=1 colspan=1>2 (6.7%)</td></tr><tr><td rowspan=1 colspan=1>ICU (other hospital)</td><td rowspan=1 colspan=1>0 (0%)</td><td rowspan=1 colspan=1>1 (3.3%)</td><td rowspan=1 colspan=1>2 (6.7%)</td></tr><tr><td rowspan=1 colspan=1>PACU or Operating rooms</td><td rowspan=1 colspan=1>0 (0%)</td><td rowspan=1 colspan=1>4 (13.3%)</td><td rowspan=1 colspan=1>0 (0%)</td></tr><tr><td rowspan=1 colspan=1>Wards</td><td rowspan=1 colspan=1>0 (0%)</td><td rowspan=1 colspan=1>0 (0%)</td><td rowspan=1 colspan=1>1 (3.3%)</td></tr><tr><td rowspan=3 colspan=4>Impression onDischargeConsensus of 3 x</td><td rowspan=1 colspan=1>None (i.e., SIRS, No Infection)</td><td rowspan=1 colspan=1>4 (13.3%)</td><td rowspan=1 colspan=1>10 (33.3%)</td><td rowspan=1 colspan=1>1 (3.3%)</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Possible Infection</td><td rowspan=1 colspan=1>1 (3.3%)</td><td rowspan=1 colspan=1>2 (6.7%)</td><td rowspan=1 colspan=1>2 (6.7%)</td></tr><tr><td rowspan=1 colspan=1> x</td><td rowspan=1 colspan=1>Probable</td><td rowspan=1 colspan=1>0 (0%)</td><td rowspan=1 colspan=1>0 (0%)</td><td rowspan=1 colspan=1>1 (3.3%)</td></tr><tr><td rowspan=2 colspan=4>PrincipalInvestigators)</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=3>Definite</td></tr><tr><td rowspan=1 colspan=1>Not reported</td><td rowspan=1 colspan=1>1 (3.3%)</td><td rowspan=1 colspan=1>0 (0%)</td><td rowspan=1 colspan=1>1 (3.3%)</td></tr><tr><td rowspan=2 colspan=4>Death</td><td rowspan=1 colspan=1>No</td><td rowspan=1 colspan=1>6 (20.0%)</td><td rowspan=1 colspan=1>12 (40.0%)</td><td rowspan=1 colspan=1>11 (36.7%)</td></tr><tr><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>0 (0%)</td><td rowspan=1 colspan=1>0 (0%)</td><td rowspan=1 colspan=1>1 (3.3%)</td></tr><tr><td rowspan=2 colspan=4>InvasiveMechanicalVentilation</td><td rowspan=1 colspan=1>No</td><td rowspan=1 colspan=1>5 (16.7%)</td><td rowspan=1 colspan=1>9 (30.0%)</td><td rowspan=1 colspan=1>10 (33.3%)</td></tr><tr><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>1 (3.3%)</td><td rowspan=1 colspan=1>3 (10.0%)</td><td rowspan=1 colspan=1>2 (6.7%)</td></tr></table>

Overall results for the retrospective and prospective subjects combined across all four RPD categories is illustrated in Figure 2 below:

![](images/0824d16bd0519e20a30da0b68bf687813ffd766a273d812ae5c4db57b2ba41a9.jpg)  
Figure 2: Probability of Sepsis by Band for Combined Prospective and Prospectively Archived Samples.

Results are presented for (A) Forced, (B) Consensus, and (C) Unanimous RPD methods. The percentage (prevalence) of sepsis is indicated for each Band with error bars representing $80 \%$ confidence intervals. The number (N) of sepsis and SIRS in each Band is indicated to the right of the point estimate (NSepsis:NSIRS) as determined by each RPD method. Results across all approaches to analyzing RPD results are consistent, i.e., there is a clear relationship between SeptiScore and the increasing likelihood of sepsis across each SeptiScore Interpretation Band.

Table 10 below presents the same results in tabular form:

Table 10: Probability of Sepsis by RPD method across bands   

<table><tr><td rowspan=2 colspan=1>RPD Method</td><td rowspan=2 colspan=1>N</td><td rowspan=1 colspan=4>Probability of Sepsis (80% Confidence Interval)</td><td rowspan=1 colspan=2>Delta Differential</td></tr><tr><td rowspan=1 colspan=1>Band 1</td><td rowspan=1 colspan=1>Band 2</td><td rowspan=1 colspan=1>Band 3</td><td rowspan=1 colspan=1>Band 4</td><td rowspan=1 colspan=1>Band 3/Band 1</td><td rowspan=1 colspan=1>Band 4/Band 2</td></tr><tr><td rowspan=1 colspan=1>Forced</td><td rowspan=1 colspan=1>386</td><td rowspan=1 colspan=1>0.12(0.08 - 0.18)</td><td rowspan=1 colspan=1>0.24(0.18 - 0.3)</td><td rowspan=1 colspan=1>0.48(0.4 - 0.56)</td><td rowspan=1 colspan=1>0.8(0.74 - 0.84)</td><td rowspan=1 colspan=1>0.221</td><td rowspan=1 colspan=1>0.440</td></tr><tr><td rowspan=1 colspan=1>Consensus</td><td rowspan=1 colspan=1>348</td><td rowspan=1 colspan=1>0.1(0.06 - 0.15)</td><td rowspan=1 colspan=1>0.22(0.16 - 0.29)</td><td rowspan=1 colspan=1>0.45(0.37 - 0.54)</td><td rowspan=1 colspan=1>0.81(0.75 - 0.85)</td><td rowspan=1 colspan=1>0.214</td><td rowspan=1 colspan=1>0.459</td></tr><tr><td rowspan=1 colspan=1>Unanimous</td><td rowspan=1 colspan=1>246</td><td rowspan=1 colspan=1>0.11(0.06 - 0.18)</td><td rowspan=1 colspan=1>0.21(0.14 - 0.29)</td><td rowspan=1 colspan=1>0.43(0.33 - 0.54)</td><td rowspan=1 colspan=1>0.8(0.73 - 0.86)</td><td rowspan=1 colspan=1>0.141</td><td rowspan=1 colspan=1>0.448</td></tr><tr><td rowspan=1 colspan=1>Indeterminateas-sepsis</td><td rowspan=1 colspan=1>386</td><td rowspan=1 colspan=1>0.16(0.11 - 0.21)</td><td rowspan=1 colspan=1>0.31(0.25 - 0.38)</td><td rowspan=1 colspan=1>0.53(0.45 - 0.61)</td><td rowspan=1 colspan=1>0.82(0.77 - 0.87)</td><td rowspan=1 colspan=1>0.239</td><td rowspan=1 colspan=1>0.391</td></tr></table>

Using a method identical to the analysis performed for the predicate K163260, results from the SeptiCyte RAPID test were analyzed with a backwards-elimination (BW-Elimination) variable selection procedure, combined with a logistic regression decision rule, for a subset of the clinical and laboratory variables from the complete dataset (see Table 11 below). Area Under Curve (AUC) for SIRS versus sepsis was used for assessing model performance for each analysis. Models were constructed using the Forced or Consensus RPD and the top five clinical variables contributing to the AUC were identified. For all logistic regression models, BW-Elimination and k-fold cross-validation were used to determine if the test provided diagnostic clinical utility beyond that provided by other clinical variables and laboratory assessments available within the first\~24 hours of the suspicion of sepsis. The backward elimination technique started with the full model including all independent variables. Models were constructed including or excluding Procalcitonin (PCT). At each step, the effect showing the smallest contribution to the model was deleted. The logistic regression model for discriminating between sepsis and SIRS found SeptiScore to be the most significant variable for each model. This observation was found for both Consensus RPD and Forced RPD.

Table 11: List of Variables Evaluated in Logistic Regression Modeling   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Variable</td><td rowspan=1 colspan=1>Brief Description of Variable*</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>Age</td><td rowspan=1 colspan=1>Age of patient</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>Race</td><td rowspan=1 colspan=1>Race of patient: African descent or Non-African descent</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>Gender</td><td rowspan=1 colspan=1>Gender of patient</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>Glucose (Maximum)</td><td rowspan=1 colspan=1>Peak blood glucose concentration</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>HeartRate (Minimum)</td><td rowspan=1 colspan=1>Lowest heart rate</td></tr><tr><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>HeartRate (Maximum)</td><td rowspan=1 colspan=1>Peak heart rate</td></tr><tr><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>Infection status: Present</td><td rowspan=1 colspan=1>Has an infection been confirmed?</td></tr><tr><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>Infection status: Fungal?</td><td rowspan=1 colspan=1>Is a fungal infection present?</td></tr><tr><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>Infection status: Gramnegative?</td><td rowspan=1 colspan=1>Is a Gram negative infection present?</td></tr><tr><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>Infection status: Grampositive?</td><td rowspan=1 colspan=1>Is a Gram positive infection present?</td></tr><tr><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>Infection status: Mixed?</td><td rowspan=1 colspan=1>Is a mixed infection (i.e., more than one Category of infectiousagent)</td></tr><tr><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>Infection status: Viral?</td><td rowspan=1 colspan=1>Is a viral infection present?</td></tr><tr><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>Mean Arterial Pressure(MAP Maximum)</td><td rowspan=1 colspan=1>Peak mean arterial blood pressure</td></tr><tr><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>SIRS Criteria</td><td rowspan=1 colspan=1>Number of SIRS criteria observed</td></tr><tr><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>Temperature (Maximum)</td><td rowspan=1 colspan=1>Peak body temperature</td></tr><tr><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>Temperature (Minimum)</td><td rowspan=1 colspan=1>Lowest body temperature</td></tr><tr><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>WBC (Maximum)</td><td rowspan=1 colspan=1>Maximum WBC count</td></tr><tr><td rowspan=1 colspan=1>18</td><td rowspan=1 colspan=1>WBC (Minimum)</td><td rowspan=1 colspan=1>Minimum WBC count</td></tr><tr><td rowspan=1 colspan=1>19</td><td rowspan=1 colspan=1>PCT/Procalcitonin**</td><td rowspan=1 colspan=1>Plasma PCT concentration (log2)</td></tr><tr><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>SeptiSCORE**</td><td rowspan=1 colspan=1>The SeptiCyte LAB result</td></tr></table>

\* All variables were measured and recorded within the first 24 hours of subject enrollment except Infection status. \*\*Variables that were not used in every model built.

2. Clinical specificity:

See discussion of Clinical sensitivity (section C.1) above.

3. Other clinical supportive data (when a. and b. are not applicable):

N/A

# Lead Reviewer or Consulting Reviewer Comments for Internal Discussion Only

# D Clinical Cut-Off:

SeptiSCORE RAPID cut-off values were established prior to the clinical trial. The sponsor notes that the original design goal was for three bands, with the following performance targets as based on ‘clinical feedback and risk analysis’:

Band 1 Sensitivity $\geq 9 0 \%$ : Minimize the number of FN (patients with sepsis who would go undetected, hence untreated)

Band 3 Specificity $\geq 8 0 \%$ : Minimize the number of FP (minimize sepsis overdiagnosis, leading to unnecessary treatment)

Sensitivity and specificity were originally defined as follows:

Band 1 sensitivity $=$ TPband1/2/(TPband1/2+FNband1/2)

Band 3 specificity $=$ TNband2/3/(TNband2/3+FPband2/3)

Ranges for the band threshold point estimates were identified by testing PAXgene blood samples from a Training cohort $\mathrm { ( N = 1 9 8 ) }$ ) of independent patients using pilot GMP-grade SeptiCyte RAPID cartridges. The Training cohort consisted of the following:

1. A panel of 100 US samples banked from the VENUS clinical trial meeting the following criteria:

a. The sample set spanned the measurement range of the assay with representation of low, medium, and high SeptiScores.   
b. Comparison data were previously generated with the SeptiCyte LAB kit (predicate device) for all of the selected specimens.   
c. None of these samples were used in the clinical validation of SeptiCyte LAB or SeptiCyte RAPID for the purposes of FDA 510(k) submission.   
d. Three (3) samples from the VENUS portion of the Training cohort were excluded from the final threshold analysis/selection as they obtained an “indeterminate” diagnosis from the retrospective physician panel. These indeterminate samples were only included in the platform comparison to establish the direct correlation between test thresholds.

2. A panel of 98 European clinical samples collected from the MARS clinical trial. The UMCU clinical sample cohort was selected on the basis of clinical data related to the patients’ ICU stay. The diagnostic classification of each patient was made based on the infection likelihood of the patient, retrospectively determined by a panel of UMCU physicians, and linked to a sepsis event within 24 hours prior to or 24 hours after ICU admission. Subjects included 46 sepsis cases (23 culture proven, and 23 probable infections based on infection likelihood) and 52 SIRS cases. Indeterminate cases (infection likelihood “possible”) were not included.

Thresholds for SeptiCyte RAPID were determined and locked in a two-part study using pilot GMP-grade cartridges. Part 1 was a correlation/method comparison analysis to determine a direct relationship between the scores, bands, and thresholds for SeptiCyte LAB (predicate device) and SeptiCyte RAPID (new device). Part 2 was a resampling-based threshold analysis using training set to determine final locked thresholds.

The three SeptiScore interpretation bands that were resolved had the following cut points:

Band 1: [0, 4.95) Band 2: [4.95, 7.35) Band 3: [7.35, 15]

However, subsequent analysis of the prospectively archived/retrospective study showed a large percentage of subjects in Band 2, which also substantially overlapped the reference range obtained from studies of healthy volunteers. Accordingly, to improve clinical interpretation, banding was subsequently revised to a 4 Band interpretation consistent with the predicate device by evenly splitting the “Intermediate” band of into two bands, as shown below in Table 12.

Table 12: Concordance between 3-Band and 4-Band schemes   

<table><tr><td rowspan=1 colspan=2>3-Band Reporting Scheme</td><td rowspan=1 colspan=2>4-Band Reporting Scheme</td></tr><tr><td rowspan=1 colspan=1>Interpretation Band</td><td rowspan=1 colspan=1>SeptiScore Range</td><td rowspan=1 colspan=1>Interpretation Band</td><td rowspan=1 colspan=1>SeptiScore Range</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0 to 4.9</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0 to 4.9</td></tr><tr><td rowspan=2 colspan=1>2</td><td rowspan=2 colspan=1>5.0 to 7.3</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>5.0 to 6.1</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>6.2 to 7.3</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>7.4 to 15</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>7.4 to 15</td></tr></table>

# E Expected Values/Reference Range:

The sponsor studied references ranges in a cohort of 119 ‘presumably healthy individuals’ $> 2 0$ years of age balanced by sex (male/female) and race/ethnicity (White, African American, Asian, and Hispanic). Samples were obtained at a single site, Discovery Life Sciences; Huntsville, AL. Results are summarized as follows:

Table 13: SeptiCyte RAPID Reference Range by Race/Ethnicity   

<table><tr><td rowspan=1 colspan=6>SeptiCyte RAPID Reference Ranges</td></tr><tr><td rowspan=2 colspan=2>Race/Ethnicity</td><td rowspan=1 colspan=4>SeptiScore Interpretation Band</td></tr><tr><td rowspan=1 colspan=1>1 (0-4.9)</td><td rowspan=1 colspan=1>2 (5.0-6.1)</td><td rowspan=1 colspan=1>3 (6.2-7.3)</td><td rowspan=1 colspan=1>4 (7.4-15)</td></tr><tr><td rowspan=2 colspan=1>Asian</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>13.3</td><td rowspan=1 colspan=1>70.0</td><td rowspan=1 colspan=1>16.7</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=2 colspan=1>Black</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>3</td></tr><tr><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>10.0</td><td rowspan=1 colspan=1>53.3</td><td rowspan=1 colspan=1>26.7</td><td rowspan=1 colspan=1>10.0</td></tr><tr><td rowspan=2 colspan=1>Hispanic</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>44.8</td><td rowspan=1 colspan=1>48.3</td><td rowspan=1 colspan=1>6.9</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=2 colspan=1>White</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>18</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>60.0</td><td rowspan=1 colspan=1>36.7</td><td rowspan=1 colspan=1>3.3</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=2 colspan=1>Total</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>38</td><td rowspan=1 colspan=1>62</td><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>3</td></tr><tr><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>31.9</td><td rowspan=1 colspan=1>52.1</td><td rowspan=1 colspan=1>13.4</td><td rowspan=1 colspan=1>2.5</td></tr></table>

This information is essential for proper use of this device and has been included as a section in the device labeling, as well as incorporated into limitations in the labeling for the device. Test results for the device include the following interpretation note:

Increased scores were observed in African Americans relative to other racial/ethnic groups, in the clinical study of patients suspected of sepsis. For African-American subjects, only Band 4 showed a higher likelihood of Sepsis versus SIRS.

The labeling supports a finding of substantial equivalence for this device.

# IX Conclusion:

The submitted information in this premarket notification is complete and supports a substantial equivalence decision.