# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATIONDECISION SUMMARYASSAY AND INSTRUMENT

I Background Information:

A 510(k) Number K211079   
B Applicant BioFire Defense, LLC   
C Proprietary and Established Names BioFire COVID-19 Test 2   
D Regulatory Information

<table><tr><td rowspan=1 colspan=1>ProductCode(s)</td><td rowspan=1 colspan=1>Classification</td><td rowspan=1 colspan=1>RegulationSection</td><td rowspan=1 colspan=1>Panel</td></tr><tr><td rowspan=1 colspan=1>QQX</td><td rowspan=1 colspan=1>Class II</td><td rowspan=1 colspan=1>21 CFR 866.3981 -Device to detect andidentify nucleic acidtargets in respiratoryspecimens frommicrobial agents thatcause the SARS-CoV-2 respiratory infectionand other microbialagents when in amulti-target test</td><td rowspan=1 colspan=1>MI - Microbiology</td></tr></table>

# II Submission/Device Overview:

A Purpose for Submission: New device   
B Measurand: SARS-CoV-2

# C Type of Test:

A nucleic acid test intended for use with the FilmArray 2.0 or FilmArray Torch systems for the qualitative in vitro detection and identification of nucleic acids from severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) in nasopharyngeal swabs (NPS) from individuals suspected of COVID-19.

# III Intended Use/Indications for Use:

A Intended Use(s): See Indications for Use below.

# B Indication(s) for Use:

The BioFire COVID-19 Test 2 is a qualitative nested multiplexed RT-PCR in vitro diagnostic test intended for use with the BioFire FilmArray 2.0 and BioFire FilmArray Torch Systems. The BioFire COVID-19 Test 2 detects nucleic acids from severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) in nasopharyngeal swabs (NPS) from symptomatic individuals suspected of COVID-19 by their healthcare provider.

Results are for the identification of SARS-CoV-2 RNA. The SARS-CoV-2 RNA is generally detectable in NPS specimens during the acute phase of infection. Positive results are indicative of the presence of SARS-CoV-2 RNA; clinical correlation with patient history and other diagnostic information is necessary to determine patient infection status. Positive results do not rule out co-infection with other pathogens.

Results are meant to be used in conjunction with other clinical, epidemiologic, and laboratory data, in accordance with the guidelines provided by the relevant public health authorities. The BioFire COVID-19 Test 2 is intended for use by trained medical and laboratory professionals in a laboratory setting or under the supervision of a trained laboratory professional.

# C Special Instrument Requirements:

FilmArray 2.0 or FilmArray Torch systems

# IV Device/System Characteristics:

# A Device Description:

The BioFire COVID-19 Test 2 is a multiplexed nucleic acid-based test designed to be used with BioFire FilmArray systems (BioFire FilmArray 2.0 or BioFire FilmArray Torch). The BioFire COVID-19 Test 2 consists of the BioFire COVID-19 Test 2 pouch which contains freeze-dried reagents to perform nucleic acid purification and nested, multiplexed polymerase chain reaction (PCR) with DNA melt analysis. The BioFire COVID-19 Test 2 conducts three independent tests for the detection of severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) RNA in nasopharyngeal swabs (NPS) eluted in transport medium or saline. Results from the BioFire COVID19 Test 2 are available in about 45 minutes.

A test is initiated by loading Hydration Solution into one port of the pouch and a NPS specimen mixed with the provided Sample Buffer into the other port of the pouch. The pouch contains all the reagents required for specimen testing and analysis in a freeze-dried format; the addition of Hydration Solution and the Sample Buffer rehydrates the reagents. After the pouch is prepared, the FilmArray Software on the FilmArray system guides the user through the steps of placing the pouch into the instrument, scanning the pouch barcode, entering the sample identification, selecting the appropriate protocol, and initiating the run on the FilmArray system.

The FilmArray instruments contain a coordinated system of inflatable bladders and seal points, which act on the pouch to control the movement of liquid between the pouch blisters. When a bladder is inflated over a reagent blister, it forces liquid from the blister into connecting channels. Alternatively, when a seal is placed over a connecting channel it acts as a valve to open or close a channel. In addition, electronically controlled pneumatic pistons are positioned over multiple plungers to deliver the rehydrated reagents into the blisters at the appropriate times. Two Peltier devices control heating and cooling of the pouch to drive the PCR reactions and subsequent melt. Nucleic acid extraction occurs within the FilmArray pouch using mechanical and chemical lysis followed by purification using standard magnetic bead technology. After extracting and purifying nucleic acids from the unprocessed sample, a nested multiplexed PCR is executed in two stages. During the first stage, a single, large volume, multiplexed reverse transcription PCR (rt-PCR) reaction is performed. The products from first stage PCR are then diluted and combined with a fresh, primer-free master mix and a fluorescent double stranded DNA binding dye (LC Green Plus, BioFire Defense, LLC). The solution is then distributed to each well of the array. Array wells contain sets of primers designed specifically to amplify sequences internal to the PCR products generated during the first stage PCR reaction. The 2nd stage PCR and melt, or nested PCR, is performed in each well of the array. At the conclusion of the 2nd stage PCR, the array is interrogated by melt curve analysis for the detection of signature amplicons denoting the presence of specific targets. A digital camera placed in front of the array captures fluorescent images of the PCR2 reactions The FilmArray software automatically analyzes the results of each DNA melt curve and the results of the internal pouch controls to provide a final test interpretation.

# Materials provided in each BioFire COVID-19 Test 2 Kit (6-test kit; DFA2-ASY-0015) or 30 samples (30-test kit; DFA2-ASY-0016):

• Individually packaged BioFire COVID-19 Test 2 Pouches   
• Single-use $( 1 . 0 \mathrm { m L } )$ ) Sample Buffer Tubes   
$\bullet$ Single-use pre-filled $( 1 . 5 \mathrm { m L } )$ ) Hydration Injection Vials (blue)   
• Individually packaged Sample Injection Vials (red) Individually packaged Transfer Pipettes

# Materials required but not provided:

• $10 \%$ bleach solution   
• FilmArray system including: $\bigcirc$ FilmArray 2.0 instruments or FilmArray Torch modules and accompanying software $\bigcirc$ FilmArray Pouch Loading Station

Once a test run is completed, the software automatically interprets the results and displays a test report. The report can be printed and/or saves as a file. The test report is a single page

containing—Run Summary, Result Summary, and Run Details. The Run Summary section provides an overall summary of the run including the sample identification, time/date of the run, internal controls result, and overall test result. A summary of the test result is displayed for the “Detected” field within the Run Summary section. If the target analyte has been detected and internal controls have passed then the field will display “SARS-CoV-2”. If the internal controls have passed and the overall test result is “Not Detected” then the field will display “None”. For any incomplete run or failed internal controls, the field will display “Invalid”.

The following is an example of the report:

<table><tr><td colspan="4">BioFire® BIO FIRE COVID-19 Test 2 v1.0 F www.BioFireDefense.com</td></tr><tr><td colspan="4">Run Summary Sample ID: Example Report</td></tr><tr><td>Detected:</td><td>SARS-CoV-2</td><td>Run Date: Internal Controls:</td><td>31 Dec 2019 8:00 AM Passed</td></tr><tr><td colspan="4"></td></tr><tr><td colspan="4">Result Summary Viruses</td></tr><tr><td colspan="4">Detected SARS-CoV-2</td></tr><tr><td>Detected SARS-CoV-2a Not Detected SARS-CoV-2d Not Detected SARS-CoV-2e</td><td colspan="3"></td></tr><tr><td>Run Details</td><td></td><td></td><td></td></tr><tr><td></td><td></td><td></td><td></td></tr><tr><td>Pouch:</td><td>COVID-19 Test 2 v1.0</td><td>Protocol:</td><td>Sample v3.2</td></tr><tr><td>Run Status: Serial No.:</td><td>Completed 1234567</td><td>Operator: Instrument: A000</td><td>Anonymous</td></tr></table>

The “Internal Controls” field in the sample report displays “Passed”, “Failed”, or “Invalid” based on the following interpretation rules—

Table 1. Interpretation of Internal Controls Field on the BioFire COVID-19 Test 2 Report   

<table><tr><td colspan="1" rowspan="1">InternalControlResult</td><td colspan="1" rowspan="1">Explanation</td><td colspan="1" rowspan="1">Action</td></tr><tr><td colspan="1" rowspan="1">Passed</td><td colspan="1" rowspan="1">The run was successfully completedANDBoth internal pouch controls (RNAProcess Control and PCR2 Control)passed.</td><td colspan="1" rowspan="1">Report the results provided on the test report.</td></tr><tr><td colspan="1" rowspan="1">Failed</td><td colspan="1" rowspan="1">The run was successfully completedBUT</td><td colspan="1" rowspan="1">Repeat the test using a new pouch.If the error persists, contact BioFire DefenseTechnical Support for further instruction.</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">At least one of the internal pouchcontrols (RNA Process Control and/orPCR2 Control) failed.</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Invalid</td><td colspan="1" rowspan="1">The internal controls are invalid becausethe run was not successfully completed.(Typically, this indicates a software orhardware error.)</td><td colspan="1" rowspan="1">Note any error codes displayed during the runand the Run Status field in the Run Detailssection of the report. Refer to the appropriateFilmArray operator's manual or contact BioFireDefense Technical Support for furtherinstruction.Once the error is resolved, repeat the test orrepeat the test using another instrument.</td></tr></table>

The Result Summary section provides a complete list of test results for each SARS-CoV-2 assay contained in the BioFire COVID-19 Test 2 device (Figure 1). The overall test result for the device and analyte is listed first followed by the individual assay result(s). The possible results for each assay are “Detected”, “Not Detected”, or “Invalid”. The following table provides an explanation for the interpretation of results and any follow-up actions necessary for the BioFire COVID-19 Test 2 device.

Table 2. Explanation of Result Summary assay results and required actions   

<table><tr><td rowspan=1 colspan=1>OverallSARS-CoV-2Result</td><td rowspan=1 colspan=1>Explanation</td><td rowspan=1 colspan=1>Action</td></tr><tr><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>The run was successfully completedANDThe pouch controls passedANDOne or more assays for the virus were &#x27;Detected&#x27;</td><td rowspan=1 colspan=1>Report results.</td></tr><tr><td rowspan=1 colspan=1>NotDetected</td><td rowspan=1 colspan=1>The run was successfully completedANDThe pouch controls passedANDThe three assays for the virus were &#x27;Not Detected&#x27;</td><td rowspan=1 colspan=1>Report results.</td></tr><tr><td rowspan=1 colspan=1>Invalid</td><td rowspan=1 colspan=1>The pouch controls were not successful (Failed orInvalidORThe run was not successful(Run Status displayed as: Aborted, Incomplete,Instrument Error, or Software Error)</td><td rowspan=1 colspan=1>See Table 1. Interpretation ofInternal Controls Field on theBioFire COVID-19 Test 2Report.</td></tr></table>

# B Principle of Operation:

The FilmArray instrument, software, and pouch work together to perform sample lysis and purification, amplification, and detection of nucleic acid. The basic sequence of actions and their associated instrument functions are outlined in Figure IV2. These steps are similar to what was described in DEN200043 and K170883.

![](images/6116f43a3be87f2c6b92fd2ac266708e6264aa56fd23d0f7341e2c81261f4e8d.jpg)  
Figure IV. The basic steps performed during a BioFire COVID-19 Test 2 run

The FilmArray instrument drives the various steps in the testing process by moving liquids to the appropriate locations within the pouch. Liquid reagents are moved to the blisters of the pouch by means of pneumatic actuators in the instrument piston block, which press on the syringe-like plunging devices in the pouch fitment. There are twelve pistons that operate in a specified sequence to deliver reagents to the appropriate blisters in the pouch at the appropriate times.

Within the pouch, liquids are moved by using bladders and hard seals to exert pressure on the exterior of the pouch. The instrument is never in contact with the liquids contained in the pouch. The bladders are inflatable elastomeric membranes used to compress the pouch blisters, thus forcing the liquid out of the compressed blister and along any open connecting channels. The hard-seals are piston-driven acutators that pinch shut specific channels, thereby directing the flow of liquids between the blisters.

Thermal interactions between the pouch and instrument play a crucial role in amplifying target nucleic acids. Temperature control is driven by a pair of numerically controlled Peltier devices. The Peltier devices heat and cool to thermocycle the first- and second-stage PCR reactions (PCR1 and PCR2, respectively) and carefully control the temperature across the array during high-resolution DNA melting.

The instrument uses blue LED to illuminate the second-stage PCR array and a digital camera to record fluorescence generated in PCR2. The optical system is designed to detect the fluorescence signal generated during DNA melting.

The instrument communicates with the computer and the FilmArray Software using Ethernet cables. The software provides instructions to the instrument to control each of the steps described above.

A detailed explanation of specific steps in the testing process are as follows:

1. Sample Lysis and Purification

Sample lysis and nucleic acid purification uses bead beating followed by adsorption to magnetic beads.

a) Sample Lysis

Prior to loading the sample into the pouch, nucleases are inactivated by mixing the sample with a denaturing buffer (BioFire FilmArray Sample Buffer). The sample/buffer mixture is then loaded into the pouch via the sample injection port. The pouch vacuum pulls the liquid into the sample well of the pouch fitment where it rehydrates a freezedried pellet of Schizosaccharomyces pombe. The S. pombe is processed in parallel with the patient sample through each step of the test. The RNA Process Control assay targets an mRNA of S. pombe, so that a positive result for the RNA Process Control assay indicates that all steps in the test functioned properly.

A piston located above the fitment plunges the sample/buffer mixture from the sample well into the sample lysis blister which contains silica beads. The instrument seals the fitment to prevent sample loss, then activates the bead beater assembly. During the bead beating process, cells and organisms/viruses are lysed and the cell/capsid contents (including nucleic acids) are released into the reaction mixture.

# b) Nucleic Acid Purification

Following sample lysis, bladder inflation mixes magnetic beads with the lysed sample to adsorb free nucleic acids. The bound nucleic acid on the magnetic beads are then separated from the silica beads, the former retained in the collection blister while the latter are moved back into the lysis blister with unbound waste. Alternating bladder inflation facilitates movement of wash buffer over the magnetic beads to remove unbound proteins, cell debris, and other potential PCR inhibitors, and this waste is sent to the lysis blister. After the washes are complete, a piston plunges elution buffer into the pouch to mix with the nucleic acid bound beads. This releases the nucleic acids from the magnetic beads and the retractable magnet retains the beads in the collection blister while bladder inflation moves the eluted nucleic acid solution to the first-stage PCR (PCR1) blister.

# 2. Reverse Transcription and PCR1

The introduction of the eluted, purified nucleic acid solution into the first-stage PCR1 blister rehydrates the reagents for this step in the assay. Subsequently, the master mix solution and the eluted nucleic acid solution with the PCR primers are individual preheated prior to mixing the two components together to setup a “hot start” RT reaction. The actuation of pistons enables the mixing of the heated master mix with the nucleic acids and primers. The RT reaction converts viral RNA into cDNA that is then amplified in the PCR1 reaction. The Peltier device cycles the appropriate temperature for the RT and PCR1 reactions.

3. Dilution, PCR2, and DNA Melt Analysis

The system then moves most of the leftover PCR1 solution and products to the lysis blister so that a smaller volume is left behind to be diluted with dilution buffer. Mechanical actuation then enables PCR2 master mix with double-stranded NDA-binding dye (LCGreen Plus, BioFire) to be added to the pouch and it mixes with the diluted PCR1 products. Bladder inflation then forces this mixture into the array where PCR2 is carried out. As the mixture distributes over the array, it rehydrates and distributes the inner primers in each well. These “nested” (i.e., internal to the PCR1 products) primers target specific nucleic acid sequences from SARS-CoV-2 or the S. pombe RNA Process Control.

The array also includes primers for the PCR2 Control. The PCR2 Control consists of a specific set of PCR2 assay primers along with their corresponding template. Failure of the PCR2 Control invalidates the run and indicates a test failure specific to the PCR2 process.

A Peltier device controls the temperature during PCR2 and DNA melting. An optical system records the fluorescence of the LCGreen Plus dye, which fluoresces in the presence of double-stranded DNA in each array well at each amplification cycle. At the conclusion of PCR2, the temperature of the array is gradually increased to denature the double-stranded DNA, and the optical system records the consequent decrease in fluorescence in each well. The instrument then transfers images and temperature measurements to the FilmArray Software for analysis to generate a melt curve for each well.

# 4. Data Analysis and Result Reporting

The temperature at which a specific PCR product melts (melting temperature of Tm) is consistent and predictable. The FilmArray software automatically evaluates the results from replicate wells of each assay for the detection of amplicons with a specific Tm, which denotes the presence of specific viral targets. The FilmArray software uses the following steps to interpret the melt curve data generated from each BioFire COVID-19 Test 2 assay—

a) Analysis of Melt Curves

First, the BioFire FilmArray Melt Detector performs a set of basic calculations on the melt data to determine if a PCR reaction occurred in each well. If the melt profile indicates that a PCR product is present, then the analysis software calculates one or two Tm values, depending on the number of melt curves present in the data, and the Tm values are compared against an expected melt range for the associated assay. If the software determines that the melt is positive and the melt curve falls inside the assay’s specific melt range, then the curve is called positive. If the software determines that the melt is negative or that it is not in the appropriate range, then the curve is called negative.

# b) Analysis of Replicates

The analysis software then evaluates the replicates for each assay (targets and controls) to determine if the assay is positive or negative. For an assay to be called positive, two of the three associated melt curves must be called positive and both Tm’s must be similar (i.e., within $1 ~ ^ { \circ } \mathrm { C }$ ). Assays with replicates that do not meet these criteria are called negative.

c) Analysis of Controls

Results for control assays are compared to their expected values and assigned a single pass or fail result for each control. There are pouch-specific rules that define how control failures affect result interpretations. The default rule specifies that any control failure invalidates the entire run. For the BioFire COVID-19 Test 2, failure of the RNA Process Control and/or the PCR2 Control is interpreted as a control failure and all target assays (regardless of the assay result) are assigned a test result of invalid.

d) Interpretation of Assay Results

Once the results for the individual assays are determined, the software applies interpretation rules to determine the final test result. The final test result is either “DETECTED” (positive result), “NOT DETECTED” (negative result) or “INVALID” (when either there is a control failure as detailed above or a run failure).

# C Instrument Description Information:

1. Instrument Name: FilmArray 2.0 or FilmArray Torch System

2. Specimen Identification: Specimen identification can be entered manually or via a barcode

3. Specimen Sampling and Handling:

The BioFire COVID-19 Test 2 is intended for use with nasopharyngeal swab (NPS) specimens collected in transport medium or saline. The operator places a Hydration Injection Vial and a Sample Injection Vial into the FilmArray Pouch Loading Station. The operator first hydrates the test pouch with the Hydration Injection Vial, and then using a transfer pipette provided in the kit, the operator adds $0 . 3 \mathrm { m L }$ $( 3 0 0 ~ \mu \mathrm { l } )$ of specimen into the Sample Injection Vial. Sample Buffer is then added into the Sample Injection Vial using the provided Sample Buffer tube. The operator closes the Sample Injection Vial, removes the Sample Injection Vial containing the sample mixture from the Pouch Loading Station, inverts the vial at least three times to mix, and then places the Sample Injection Vial back onto the red well of the Pouch Loading Station. A 5 second wait time after uncapping the Sample Injection Vial is instructed to reduce contamination. The Sample Injection Vial is then inserted into the pouch sample port where the proper amount of specimen is pulled into the BioFire COVID-19 Test 2 pouch by vacuum. The BioFire COVID-19 Test 2 pouch is then placed in the FilmArray 2.0 instrument or the available module of a FilmArray Torch system for testing.

4. Calibration:

Not applicable

# 5. Quality Control:

Two process controls are included in each BioFire COVID-19 Test 2 pouch:

# 1. RNA Process Control

The RNA Process Control assay targets an RNA transcript from the yeast Schizosaccharomyces pombe. The yeast is present in the pouch in a freeze-dried form and becomes rehydrated when sample is loaded. The control material is carried through all stages of the test process, including lysis, nucleic acid purification, reverse transcription, PCR1, dilution, PCR2, and DNA melting. A positive result indicates that all steps carried out in the BioFire COVID-19 Test 2 pouch were successful.

# 2. PCR2 Control

The PCR2 Control assay detects a DNA target that is dried into wells of the array along with the corresponding primers. A positive result indicates that PCR2 was successful.

Both control assays must be positive for the test run to pass. If controls fail, the sample should be retested using a new pouch.

This medical device product has functions subject to FDA premarket review as well as functions that are not subject to FDA premarket review. For this application, if the product has functions that are not subject to FDA premarket review, FDA assessed those functions only to the extent that they either could adversely impact the safety and effectiveness of the functions subject to FDA premarket review or they are included as a labeled positive impact that was considered in the assessment of the functions subject to FDA premarket review.

# V Substantial Equivalence Information:

A Predicate Device Name(s): BioFire Respiratory Panel 2.1 (RP2.1)

B Predicate 510(k) Number(s): DEN200031

C Comparison with Predicate(s):

<table><tr><td rowspan=1 colspan=1>Device &amp; PredicateDevice(s):</td><td rowspan=1 colspan=1>K211079</td><td rowspan=1 colspan=1>DEN200031</td></tr><tr><td rowspan=1 colspan=1>Device Trade Name</td><td rowspan=1 colspan=1>BioFire COVID-19 Test2</td><td rowspan=1 colspan=1>BioFire RespiratoryPanel 2.1</td></tr><tr><td rowspan=1 colspan=1>General DeviceCharacteristic Similarities</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Intended Use/ IndicationsFor Use</td><td rowspan=1 colspan=1>The BioFire COVID-19Test 2 is a qualitativenested multiplexed RT-PCR in vitro diagnostic</td><td rowspan=1 colspan=1>Similar except theBioFire RespiratoryPanel 2.1 (RP2.1) isintended for</td></tr></table>

<table><tr><td>provider.</td><td>test intended for use with simultaneous the BioFire FilmArray qualitative detection 2.0 and BioFire and identification of FilmArray Torch multiple respiratory Systems. The BioFire viral and bacterial COVID-19 Test 2 nucleic acids including detects nucleic acids SARS-CoV-2. from severe acute respiratory syndrome coronavirus 2 (SARS- Similarly indicated as CoV-2) in an aid in diagnosing nasopharyngeal swabs COVID-19 in (NPS) from individuals conjunction with patient suspected of COVID-19 history and other by their healthcare clinical and epidemiological information. Results are for the identification of SARS- Similarly should not be CoV-2 RNA. The SARS-CoV-2 RNA is used as the sole basis for diagnosis, treatment, generally detectable in NPS specimens during or other patient the acute phase of management decisions. infection. Positive results are indicative of the presence of SARS-CoV- 2 RNA; clinical correlation with patient history and other diagnostic information is necessary to determine patient infection status. Positive results do not rule out co-infection with other pathogens. Results are meant to be</td></tr></table>

<table><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">under the supervision ofa trained laboratoryprofessional.</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Specimen Types</td><td colspan="1" rowspan="1">Nasopharyngeal swabseluted in transportmedium or saline</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Technological Principles</td><td colspan="1" rowspan="1">Nested multiplex RT-PCR followed by highresolution meltinganalysis to confirmidentity of amplifiednucleic acids</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Instrumentation</td><td colspan="1" rowspan="1">FilmArray 2.0 orFilmArray Torch</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Time-to-Result</td><td colspan="1" rowspan="1">About 45 minutes</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Reagent Storage</td><td colspan="1" rowspan="1">Room Temperature</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Test Interpretation</td><td colspan="1" rowspan="1">Automated testinterpretation and reportgeneration. User cannotaccess raw data.</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Controls</td><td colspan="1" rowspan="1">Two internal controlsare included in eachreagent pouch forquality control ofsample processing andboth PCR stages andmelt analysis</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">General DeviceCharacteristic Differences</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Organisms Detected</td><td colspan="1" rowspan="1">SARS-CoV-2 (multipletargets different fromBioFire RP2.1)</td><td colspan="1" rowspan="1">SARS-CoV-2ANDadenovirus, coronavirus229E, coronavirusHKU1, coronavirusNL63, coronavirusOC43, humanmetapneumovirus,humanrhinovirus/enterovirus,influenza A (includingsubtypes H1, H3, andH1-2009), influenza B,parainfluenza virus 1,parainfluenza virus 2,parainfluenza virus 3,parainfluenza virus 4,respiratory syncytial</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">virus, Bordetellaparapertussis, Bordetellapertussis, Chlamydiapneumoniae,Mycoplasmapneumoniae</td></tr><tr><td colspan="1" rowspan="1">Analyte</td><td colspan="1" rowspan="1">RNA</td><td colspan="1" rowspan="1">RNA/DNA</td></tr></table>

# VI Standards/Guidance Documents Referenced:

The study designs and 510(k) premarket notification request were prepared with reference to FDA-recognized standards and guidance documents as listed below. The referenced documents were applied with general use; modifications to recommended study designs were made as appropriate to address the particular characteristics of the subject device.

1. Standards

ISO 14971:2007 ‘Medical devices – Application of risk management to medical devices’ • ISO 62304:2006, ‘Medical device software – Software life-cycle processes’ – IEC 62304:2006, November 27, 2008 ISO 15223-1:2012, ‘Medical Devices – Symbols to be used with medical device labels, labeling and information to be supplied – Part 1: General requirements’

2. Guidance Documents

Format for Traditional and Abbreviated 510(k)s, Guidance for Industry and FDA Staff (September 13, 2019)   
• Policy for Coronavirus Disease-2019 Tests During the Public Health Emergency, Guidance for Clinical Laboratories, Commercial Manufacturers, and FDA Staff (Revised, May 2020) Policy for Evaluating Impact of Viral Mutations on COVID-19 Tests, Guidance for Test Developers and FDA Staff (February 2021)   
• Guidance for the Content of Premarket Submissions for Software Contained in Medical Devices, Guidance for Industry and FDA Staff (May 11, 2005) Off-The-Shelf Software Use in Medical Devices, Guidance for Industry and FDA Staff (September 27, 2019)   
• General Principle of Software Validation, Final Guidance for Industry and FDA Staff (January 11, 2002)   
• Content of Premarket Submissions for Management of Cybersecurity in Medical Devices, Guidance for Industry and FDA Staff (October 2, 2014)   
• Use of Symbols on Labels and in Labeling of In Vitro Diagnostic Devices Intended for Professional Use, FDA Guidance Document (November 30, 2004) Statistical Guidance on Reporting Results from Studies Evaluating Diagnostic Tests, FDA Guidance Document (March 13, 2007)   
Acceptance of Clinical Data to Support Medical Device Applications and Submissions: Frequently Asked Questions, Guidance for Industry and FDA Staff (February 2018)   
WMA Declaration of Helsinki, Ethical Principles for Medical Research Involving Human Subjects

3. Performance Standards

User Protocol for Evaluation of Qualitative Test Performance, Clinical and Laboratory Standards Institute (CLSI) Approved Guideline – Second Edition, EP12-A2 (January 2008) • Molecular Diagnostic Methods for Infectious Diseases, Clinical and Laboratory Standards Institute (CLSI) Guideline, MM03-P2 (February 2015) Interference Testing in Clinical Chemistry, 3rd Edition, Clinical and Laboratory Standards Institute (CLSI) Approved Guideline, EP07-A2 (April 2018)

4. Special Control

As per 21 CFR 866.3981 Special Control (4)(ii), and following the guidance “Policy for Evaluating Impact of Viral Mutations on COVID-19 Tests, Guidance for Test Developers and FDA Staff (February 2021)”, the sponsor provided a proposed risk analysis and documentation to continuously monitor for genetic mutation and/or novel respiratory pathogen isolates or strains that may adversely impact device performance.

VII Performance Characteristics (if/when applicable):

# A Analytical Performance:

# 1. Precision/Reproducibility:

A reproducibility study was performed to evaluate the potential for run-to-run and/or day-today variation in the BioFire COVID-19 Test 2 performance, and any potential variation due to test site, reagent lot, operator, and/or FilmArray instrument. The study was performed using three samples, prepared using pooled negative clinical NPS specimens in transport medium as the sample matrix. One sample was spiked with SARS-CoV-2 (heat inactivated; USA-WA1/2020) at 3x LoD (moderate positive), another sample was spiked at 1x LoD (low positive), and the third sample was not spiked (negative sample). Six replicates of each sample were tested on five different days at three test locations for a total of 90 replicate test results per sample (30 per site).

Reproducibility was evaluated using both FilmArray 2.0 instruments and Torch modules (45 tests per sample type on each FilmArray system). BioFire COVID-19 Test 2 pouch reagent lots were rotated daily according to site-specific rotation schedules. In total, 270 valid test result were obtained.

The collected data was evaluated for the following parameters:

The percent agreement between the observed and expected test results at each test level (acceptance criteria $= 2 9 5 \%$ for each sample at each test site, and overall).

The reproducibility of assay Tm values (i.e., standard deviation; acceptance criteria $=$ Assay Tm standard deviation of all replicates for the same assay tested at the same concentration is $0 . 5 ^ { \circ } \mathrm { C }$ or less).   
Any observed detection trends.

During the study, though the protocol specified that testing should be performed at each site with the same pre-assigned FilmArray instrument(s)/module(s) for all five test days, an Torch module was replaced at one of the test sites due to a reset/hardware stability issue. This was replaced with a functioning unit and no other study deviations were reported.

Overall, there were no reported test or run failures reported at all three sites, instruments, operators, or test days. All 270 total runs for this study completed with valid results (270/270; $100 \%$ ).

A summary of reproducibility test results is shown below (Table 1). The SARS-CoV-2 Detected rate for the spiked samples met the acceptance critera. However, for the negative sample, Sites 2 and 3 had unexpected Detected results. The three unexpected results occurred with three different pouch reagent lots, three different users, three different FilmArray instruments, and on three different test days after swab test checks prior to conducting the study for the day did not show contamination. Further investigation to trace the cause of the unexpected results did not yield any conclusive explanation on why the Detected results were observed.

Table 3. Summary of BioFire COVID-19 Test 2 Reproducibility Test Results   

<table><tr><td rowspan=2 colspan=1>Analyte(Source / ID)</td><td rowspan=2 colspan=1>ConcentrationTested×LoD(GE/mL)</td><td rowspan=2 colspan=1>ExpectedResult</td><td rowspan=1 colspan=4>Detection Rate (n/N)(% Agreement with Expected Result)</td></tr><tr><td rowspan=1 colspan=1>Site 1</td><td rowspan=1 colspan=1>Site 2</td><td rowspan=1 colspan=1>Site 3</td><td rowspan=1 colspan=1>All Sites[95% ConfidenceInterval</td></tr><tr><td rowspan=3 colspan=1>SARS-CoV-2USA-WA1/2020(BEI / NR-52286)</td><td rowspan=1 colspan=1>Moderate Positive3×LoD(9.9E+02)</td><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>30/30(100%)</td><td rowspan=1 colspan=1>30/30(100%)</td><td rowspan=1 colspan=1>30/30(100%)</td><td rowspan=1 colspan=1>90/90(100%)[95.9-100%]</td></tr><tr><td rowspan=1 colspan=1>Low Positive1×LoD(3.3E+02)</td><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>30/30(100%)</td><td rowspan=1 colspan=1>30/30(100%)</td><td rowspan=1 colspan=1>30/30(100%)</td><td rowspan=1 colspan=1>90/90(100%)[95.9-100%]</td></tr><tr><td rowspan=1 colspan=1>Negative(No Analyte)</td><td rowspan=1 colspan=1>NotDetected</td><td rowspan=1 colspan=1>30/30(100%)</td><td rowspan=1 colspan=1>28/30 a(93.3%)</td><td rowspan=1 colspan=1>29/30 b(96.7%)</td><td rowspan=1 colspan=1>87/90(96.7%)[90.7-98.9%]</td></tr></table>

a One unexpected SARS-CoV-2 Detected result for the SARS-COV-2a assay, and one unexpected SARS-CoV-2 Detected result for the SARS-CoV-2e assay. b One unexpected SARS-CoV-2 Detected result for the SARS-CoV-2e assay.

A secondary assessment of reproducibility is based on variability in the Tm of the amplification products measured as standard deviation. Table 2 shows the results per test site, and table 3 shows the results per FilmArray system. The melt temperature mean, standard deviation (StDev) and coefficient of variation (CV) are shown for internal pouch control assays and the SARS-CoV-2 assays. The variability in the melt temperatures for the assays were all within the expected range $( { \leq } 0 . 5 ^ { \circ } \mathrm { C } )$ for each assay per site and per FilmArray system. Mean Tm values were within $0 . 2 ^ { \circ } \mathrm { C }$ between the two different FilmArray systems.

Table 4. Summary of Tm $( ^ { \circ } \mathbf { C } )$ Analyses for BioFire COVID-19 Test 2 Internal Control and Analyte Assays per Test Site   

<table><tr><td rowspan=3 colspan=1>AssayType</td><td rowspan=3 colspan=1>AssayName</td><td rowspan=1 colspan=12>Observed Tm (</td></tr><tr><td rowspan=1 colspan=3>Site 1</td><td rowspan=1 colspan=3>Site 2</td><td rowspan=1 colspan=3>Site 3</td><td rowspan=1 colspan=3>All Sites</td></tr><tr><td rowspan=1 colspan=1>Mean</td><td rowspan=1 colspan=1>±StDev</td><td rowspan=1 colspan=1>CV</td><td rowspan=1 colspan=1>Mean</td><td rowspan=1 colspan=1>±StDev</td><td rowspan=1 colspan=1>CV</td><td rowspan=1 colspan=1>Mean</td><td rowspan=1 colspan=1>±StDev</td><td rowspan=1 colspan=1>CV</td><td rowspan=1 colspan=1>Mean</td><td rowspan=1 colspan=1>±StDev</td><td rowspan=1 colspan=1>CV</td></tr><tr><td rowspan=2 colspan=1>Controls</td><td rowspan=1 colspan=1>Yeast RNA</td><td rowspan=1 colspan=1>82.0</td><td rowspan=1 colspan=1>±0.3</td><td rowspan=1 colspan=1>0.4%</td><td rowspan=1 colspan=1>81.8</td><td rowspan=1 colspan=1>±0.3</td><td rowspan=1 colspan=1>0.4%</td><td rowspan=1 colspan=1>82.1</td><td rowspan=1 colspan=1>±0.3</td><td rowspan=1 colspan=1>0.4%</td><td rowspan=1 colspan=1>82.0</td><td rowspan=1 colspan=1>±0.3</td><td rowspan=1 colspan=1>0.4%</td></tr><tr><td rowspan=1 colspan=1>PCR2</td><td rowspan=1 colspan=1>75.8</td><td rowspan=1 colspan=1>±0.3</td><td rowspan=1 colspan=1>0.4%</td><td rowspan=1 colspan=1>75.6</td><td rowspan=1 colspan=1>±0.3</td><td rowspan=1 colspan=1>0.4%</td><td rowspan=1 colspan=1>75.9</td><td rowspan=1 colspan=1>±0.3</td><td rowspan=1 colspan=1>0.4%</td><td rowspan=1 colspan=1>75.7</td><td rowspan=1 colspan=1>±0.3</td><td rowspan=1 colspan=1>0.4%</td></tr><tr><td rowspan=3 colspan=1>Analyte</td><td rowspan=1 colspan=1>SARS-CoV-2a</td><td rowspan=1 colspan=1>79.0</td><td rowspan=1 colspan=1>±0.2</td><td rowspan=1 colspan=1>0.3%</td><td rowspan=1 colspan=1>78.7</td><td rowspan=1 colspan=1>±0.3</td><td rowspan=1 colspan=1>0.4%</td><td rowspan=1 colspan=1>79.0</td><td rowspan=1 colspan=1>±0.3</td><td rowspan=1 colspan=1>0.4%</td><td rowspan=1 colspan=1>78.9</td><td rowspan=1 colspan=1>±0.3</td><td rowspan=1 colspan=1>0.4%</td></tr><tr><td rowspan=1 colspan=1>SARS-CoV-2d</td><td rowspan=1 colspan=1>81.3</td><td rowspan=1 colspan=1>±0.3</td><td rowspan=1 colspan=1>0.4%</td><td rowspan=1 colspan=1>81.1</td><td rowspan=1 colspan=1>±0.3</td><td rowspan=1 colspan=1>0.4%</td><td rowspan=1 colspan=1>81.4</td><td rowspan=1 colspan=1>±0.3</td><td rowspan=1 colspan=1>0.4%</td><td rowspan=1 colspan=1>81.3</td><td rowspan=1 colspan=1>±0.3</td><td rowspan=1 colspan=1>0.4%</td></tr><tr><td rowspan=1 colspan=1>SARS-oV-2e</td><td rowspan=1 colspan=1>78.8</td><td rowspan=1 colspan=1>±0.3</td><td rowspan=1 colspan=1>0.4%</td><td rowspan=1 colspan=1>78.6</td><td rowspan=1 colspan=1>±0.3</td><td rowspan=1 colspan=1>0.4%</td><td rowspan=1 colspan=1>78.9</td><td rowspan=1 colspan=1>±0.3</td><td rowspan=1 colspan=1>0.4%</td><td rowspan=1 colspan=1>78.8</td><td rowspan=1 colspan=1>±0.3</td><td rowspan=1 colspan=1>0.4%</td></tr></table>

Table 5. Summary of Tm $( ^ { \circ } \mathbf { C } )$ Analyses for BioFire COVID-19 Test 2 Internal Control and Analyte Assays per FilmArray System   

<table><tr><td rowspan=3 colspan=1>Assay Type</td><td rowspan=3 colspan=1>AssayName</td><td rowspan=1 colspan=6>Observed Tm (C)</td></tr><tr><td rowspan=1 colspan=3>FilmArray 2.0</td><td rowspan=1 colspan=3>Torch</td></tr><tr><td rowspan=1 colspan=1>Mean</td><td rowspan=1 colspan=1>±StDev</td><td rowspan=1 colspan=1>CV</td><td rowspan=1 colspan=1>Mean</td><td rowspan=1 colspan=1>±StDev</td><td rowspan=1 colspan=1>CV</td></tr><tr><td rowspan=2 colspan=1>Controls</td><td rowspan=1 colspan=1>Yeast RNA</td><td rowspan=1 colspan=1>82.1</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>0.4%</td><td rowspan=1 colspan=1>81.9</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>0.4%</td></tr><tr><td rowspan=1 colspan=1>PCR2</td><td rowspan=1 colspan=1>75.8</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>0.4%</td><td rowspan=1 colspan=1>75.7</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>0.4%</td></tr><tr><td rowspan=3 colspan=1>Analyte</td><td rowspan=1 colspan=1>SARS-CoV-2a</td><td rowspan=1 colspan=1>79.0</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>0.4%</td><td rowspan=1 colspan=1>78.8</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>0.4%</td></tr><tr><td rowspan=1 colspan=1>SARS-CoV-2d</td><td rowspan=1 colspan=1>81.3</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>0.4%</td><td rowspan=1 colspan=1>81.2</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>0.4%</td></tr><tr><td rowspan=1 colspan=1>SARS-CoV-2e</td><td rowspan=1 colspan=1>78.8</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>0.4%</td><td rowspan=1 colspan=1>78.7</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>0.4%</td></tr></table>

Overall, there were no detection trends among the three different sites and two FilmArray Systems evaluated that would indicate a reproducibility issue (i.e., no significant differences in observed values across the different test sites). Although unexpected Detected results for unspiked samples were observed when evaluating the percent agreement between observed and expected test results, they were not related to the variables evaluated in the multi-site reproducibility evaluation (i.e., pouch reagent lot, instrument/system, operator, etc.). A root cause for the unexpected Detected results in the study could not be determined. However, in consideration of the percent agreement between observed and expected test results for most of samples tested, Tm variability within the expected range, and the lack of detection trends from testing at all sites, operators, instruments, and days, the BioFire COVID-19 Test 2 device performance appears to be reproducible.

The device can provide accurate and highly reproducible test results in the context of multiple variables that may be expected in a clinical testing environment, including analyte concentration, location, test day, operator, FilmArray system, and reagent lot.

2. Linearity:

Not applicable; this is a qualitative assay.

3. Analytical Specificity/Interference:

a) Specificity

The potential for assay cross-reactivity was evaluated by challenging the test with a panel of viruses and organisms at high concentrations. These viruses and organisms were selected based on their possible presence in respiratory clinical matrix. Exclusivity of the BioFire COVID-19 Test 2 was evaluated by spiking viruses and organisms at the highest concentration possible. Each virus/organism was tested in triplicate. All three SARS-CoV-2 assays in each replicate were expected to be negative and any invalid runs due to failure of controls or instrument/software errors were to be repeated.

It was noted that there was one study deviation in that a commercial source of bocavirus was not available for testing. In lieu of a commercial stock, an archived clinical sample (NPS) positive for bocavirus was tested that was previously evaluated using the FilmArray Respiratory Panel. This sample appeared to be a high positive and thus was diluted 10-fold in sterile saline for further evaluation in this study by the subject device.

A total of 124 FilmArray system runs were initiated and completed successfully (124/124; $1 0 0 \%$ , where the run were initiated on the FilmArray 2.0 and FilmArray Torch instruments in equal proportion (i.e., 62 FilmArray 2.0, 62 FilmArray Torch). Further 29 viruses, 43 bacteria, and 5 Fungi were evaluated with the BioFire COVID-19 Test 2 device.

A summary of the cross-reactivity evaluation is as follows. No cross-reactivity was detected for the various analytes (i.e., 0/3 detection for each assay/replicate/analyte).

Table 6. Evaluated Viruses and Organisms   

<table><tr><td colspan="1" rowspan="1">Organism/Virus</td><td colspan="1" rowspan="1">Source/ID</td><td colspan="1" rowspan="1">Test Concentration1</td><td colspan="1" rowspan="1">Cross-ReactivityDetected</td></tr><tr><td colspan="1" rowspan="1">Human coronavirus 229E</td><td colspan="1" rowspan="1">Zeptometrix0810229CF</td><td colspan="1" rowspan="1">1.26E+06TCID50/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Human coronavirus HKU1</td><td colspan="1" rowspan="1">Clinical Specimen(NPS)</td><td colspan="1" rowspan="1">~1.0E+08copies/mL 2</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Human coronavirus NL63</td><td colspan="1" rowspan="1">Zeptometrix0810228CF</td><td colspan="1" rowspan="1">2.51E+05TCID50/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Human coronavirus OC43</td><td colspan="1" rowspan="1">Zeptometrix0810024CF</td><td colspan="1" rowspan="1">9.55E+06TCID50/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Middle East RespiratorySyndrome coronavirus</td><td colspan="1" rowspan="1">BEINR-44260</td><td colspan="1" rowspan="1">2.7E+08GE/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td rowspan="2">Organism/Virus</td><td rowspan="2">Source/ID</td><td rowspan="2">Test Concentration1</td><td rowspan="2">Cross-Reactivity Detected</td></tr><tr><td></td></tr><tr><td>(MERS-CoV; EMC/2012)</td><td></td><td></td><td></td></tr><tr><td>Severe Acute Respiratory Syndrome coronavirus</td><td>BEI</td><td>5.3E+08</td><td>None</td></tr><tr><td>(SARS-CoV; Urbani)</td><td>NR-18925</td><td>GGC/mL</td><td></td></tr><tr><td></td><td>Zeptometrix</td><td>3.39E+07</td><td></td></tr><tr><td>Adenovirus 1</td><td>0810050CF</td><td>TCID50/mL</td><td>None</td></tr><tr><td>(species C) Adenovirus 4</td><td>Zeptometrix</td><td>7.05E+04</td><td></td></tr><tr><td>(species E)</td><td>080070CF</td><td>TCID50/mL</td><td>None</td></tr><tr><td>Adenovirus 7</td><td>Zeptometrix</td><td>5.10E+07</td><td></td></tr><tr><td>(species B)</td><td>081021CF</td><td>TCID50/mL</td><td>None</td></tr><tr><td>Bocavirus</td><td>Clinical Specimen</td><td>10-fold dilution of</td><td>None</td></tr><tr><td></td><td>(NPS) Zeptometrix</td><td>Specimen{3 4.2E+04</td><td></td></tr><tr><td>Cytomegalovirus</td><td>0810003CF</td><td>TCID50/mL</td><td>None</td></tr><tr><td>Enterovirus species A</td><td>NCPV</td><td>5.0E+08</td><td>None</td></tr><tr><td>(EV71)</td><td>0812215v</td><td>TCID50/mL</td><td></td></tr><tr><td>Enterovirus species B (Echovirus 6)</td><td>Zeptometrix 0810076CF</td><td>5.10E+07</td><td>None</td></tr><tr><td></td><td></td><td>TCID50/mL</td><td></td></tr><tr><td>Enterovirus species C (Coxsackievirus A17)</td><td>ATCC VR-1023</td><td>7.90E+05 TCID50/mL</td><td>None</td></tr><tr><td>Enterovirus species D</td><td>Zeptometrix</td><td>1.58E+06</td><td></td></tr><tr><td>(68)</td><td>0810237CF</td><td>TCID50/mL</td><td>None</td></tr><tr><td>Epstein-Barr virus</td><td>Zeptometrix</td><td>1.3E+07</td><td>None</td></tr><tr><td>(B95-8)</td><td>0810008CF Zeptometrix</td><td>Copies/mL</td><td></td></tr><tr><td>Herpes simplex virus</td><td>0810005F</td><td>3.4E+06 TCID50/mL</td><td>None</td></tr><tr><td>Human Metapneumovirus</td><td>Zeptometrix 081061CF</td><td>1.78E+05 TCID50/mL</td><td>None</td></tr><tr><td>Influenza A subtype H1</td><td>Zeptometrix</td><td>7.05E+04</td><td>None</td></tr><tr><td></td><td>01036FN Zeptometrix</td><td>TCID50/mL 7.05E+04</td><td></td></tr><tr><td>Influenza A subtype H3</td><td>0810252CF Zeptometrix</td><td>TCID50/mL 4.78E+06</td><td>None</td></tr><tr><td>Influenza B</td><td>0810239CF Zeptometrix</td><td>TCID50/mL 1.3E+05</td><td>None</td></tr><tr><td>Measles virus</td><td>0810025F Zeptometrix</td><td>TCID50/mL 2.4E+04</td><td>None</td></tr><tr><td>Mumps virus</td><td>081079F BEI</td><td>TCID50/mL 8.0E+05</td><td>None</td></tr><tr><td>Parainfluenza virus 1</td><td>NR-48681 Zeptometrix</td><td>TCID50/mL 1.10E+06</td><td>None</td></tr><tr><td>Parainfluenza virus 2</td><td>0810504CF BEI</td><td>TCID50/mL</td><td>None</td></tr><tr><td>Parainfluenza virus 3</td><td>NR-3233</td><td>5.10E+07 TCID50/mL</td><td>None</td></tr><tr><td>Parainfluenza virus 4</td><td>Zeptometrix 08010060BCF</td><td>1.70E+07 TCID50/mL</td><td>None</td></tr><tr><td>Respiratory syncytial virus</td><td>Zeptometrix 0810040ACF</td><td>1.05E+06 TCID50/mL</td><td>None</td></tr><tr><td>Rhinovirus</td><td>Zeptometrix 0810012CFN</td><td>1.26E+06 TCID/mL</td><td>None</td></tr><tr><td>Acinetobacter calcoaceticus</td><td>Zeptometrix 0804096</td><td>5.0E+07 CFU/mL</td><td>None</td></tr><tr><td>Bordetella avium</td><td>ATCC BAA-1003</td><td>10-fold dilution of organism stock4</td><td>None</td></tr><tr><td colspan="1" rowspan="1">Organism/Virus</td><td colspan="1" rowspan="1">Source/ID</td><td colspan="1" rowspan="1">Test Concentration1</td><td colspan="1" rowspan="1">Cross-ReactivityDetected</td></tr><tr><td colspan="1" rowspan="1">Bordetella bronchiseptica</td><td colspan="1" rowspan="1">Zeptometrix0801649</td><td colspan="1" rowspan="1">2.3E+09CF/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Bordetella hinzii</td><td colspan="1" rowspan="1">ATCC51784</td><td colspan="1" rowspan="1">10-fold dilution oforganism stock4</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Bordetella holmesii</td><td colspan="1" rowspan="1">Zeptometrix0801464</td><td colspan="1" rowspan="1">5.2E+07F/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Bordetella parapertussis</td><td colspan="1" rowspan="1">Zeptometrix0801461</td><td colspan="1" rowspan="1">1.3E+07CFU/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Bordetella pertussis</td><td colspan="1" rowspan="1">Zeptometrix0801459</td><td colspan="1" rowspan="1">6.70E+09F/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Chlamydia pneumoniae</td><td colspan="1" rowspan="1">ATCC53592</td><td colspan="1" rowspan="1">2.90E+07I U/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Chlamydia trachomatis</td><td colspan="1" rowspan="1">Zeptometrix0801775</td><td colspan="1" rowspan="1">1.2E+08IFU/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium diptheriae</td><td colspan="1" rowspan="1">Zeptometrix0801882</td><td colspan="1" rowspan="1">1.1E+08CFU/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">Zeptometrix001517</td><td colspan="1" rowspan="1">7.9E+08CF/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Fluoribacter bozemanae(Legionella bozemanii)</td><td colspan="1" rowspan="1">ATCC33217</td><td colspan="1" rowspan="1">1.1E+06CFU/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Fluoribacter dumoffii(Legionella dumoffi)</td><td colspan="1" rowspan="1">ATCC33279</td><td colspan="1" rowspan="1">10-fold dilution oforganism stock4</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Haemophilus influenzae</td><td colspan="1" rowspan="1">ATCC700223</td><td colspan="1" rowspan="1">4.20E+08CFU/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Klebsiella aerogenes(Enterobacter aerogenes)</td><td colspan="1" rowspan="1">Zeptometrix001518</td><td colspan="1" rowspan="1">5.9E+08CF/mML</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Klebsiella oxytoca</td><td colspan="1" rowspan="1">Zeptometrix801881</td><td colspan="1" rowspan="1">8.8E+08F/mML</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">Zeptometrix0801506</td><td colspan="1" rowspan="1">8.6E+08F/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus acidophilus</td><td colspan="1" rowspan="1">Zeptometrix08015400</td><td colspan="1" rowspan="1">8.6E+07F/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus plantarum</td><td colspan="1" rowspan="1">Zeptometrix08801507</td><td colspan="1" rowspan="1">1.4E+08CF/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Legionella feeleii</td><td colspan="1" rowspan="1">ATCC3502</td><td colspan="1" rowspan="1">10-fold dilution oforganism stock4</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Legionella longbeacheae</td><td colspan="1" rowspan="1">Zeptometrix00157</td><td colspan="1" rowspan="1">1.4E+08CFU/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Legionella pneumophila</td><td colspan="1" rowspan="1">Zeptometrix0801530</td><td colspan="1" rowspan="1">2.63E+09CF/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Moraxella catarrhalis</td><td colspan="1" rowspan="1">Zeptometrix0801509</td><td colspan="1" rowspan="1">2.5E+07F/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Mycobacterium tuberculosis(attenuated strain)</td><td colspan="1" rowspan="1">Zeptometrix0801660</td><td colspan="1" rowspan="1">3.04E+07CFU/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Mycoplasma genitalium</td><td colspan="1" rowspan="1">ATCC33530</td><td colspan="1" rowspan="1">1.0E+06Bacteria/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Mycoplasma hominis</td><td colspan="1" rowspan="1">Zeptometrix0804011</td><td colspan="1" rowspan="1">3.2E+08CC/mML</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Mycoplasma orale</td><td colspan="1" rowspan="1">BEINR-3852</td><td colspan="1" rowspan="1">10-fold dilution oforganism stock4</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Mycoplasma pneumoniae</td><td colspan="1" rowspan="1">Zeptometrix0801579</td><td colspan="1" rowspan="1">3.98E+07CCU/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Neisseria elongate</td><td colspan="1" rowspan="1">Zeptometrix801510</td><td colspan="1" rowspan="1">7.0E+08FU/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Neisseria gonorrhoeae</td><td colspan="1" rowspan="1">Zeptometrix0801482</td><td colspan="1" rowspan="1">4.1E+06CFU/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Neisseria meningitidis</td><td colspan="1" rowspan="1">Zeptometrix0801511</td><td colspan="1" rowspan="1">2.1E+07CFU/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Proteus mirabilis</td><td colspan="1" rowspan="1">Zeptometrix0801544</td><td colspan="1" rowspan="1">2.4E+08CFU/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">ATCC10145</td><td colspan="1" rowspan="1">5.68E+08CFU/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Serratia marcescens</td><td colspan="1" rowspan="1">Zeptometrix0801723</td><td colspan="1" rowspan="1">9.2E+08CFU/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">Zeptometrix0801638</td><td colspan="1" rowspan="1">1.4E+09CFU/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus epidermidis</td><td colspan="1" rowspan="1">ATCC29887</td><td colspan="1" rowspan="1">7.43E+09CFU/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Stenotrophomonas maltophilia</td><td colspan="1" rowspan="1">Zeptometrix0801569</td><td colspan="1" rowspan="1">1.2E+09CFU/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Streptococcus agalactiae</td><td colspan="1" rowspan="1">Zeptometrix0801545</td><td colspan="1" rowspan="1">1.8E+08CFU/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Streptococcus pneumoniae</td><td colspan="1" rowspan="1">ATCC6303</td><td colspan="1" rowspan="1">8.90E+07CFU/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Streptococcus pyogenes</td><td colspan="1" rowspan="1">ATCC49399</td><td colspan="1" rowspan="1">4.65E+08CFU/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Streptococcus salivarius</td><td colspan="1" rowspan="1">ATCC13419</td><td colspan="1" rowspan="1">7.38E+09CFU/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Tatlockia micdadei(Legionella micdadei)</td><td colspan="1" rowspan="1">ATCC33218</td><td colspan="1" rowspan="1">10-fold dilution oforganism stock4</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Ureaplasma urealyticum</td><td colspan="1" rowspan="1">BEINR-3861</td><td colspan="1" rowspan="1">10-fold dilution oforganism stock4</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Aspergillus flavus</td><td colspan="1" rowspan="1">Zeptometrix0801598</td><td colspan="1" rowspan="1">3.3E+06CFU/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Aspergillus fumigatus</td><td colspan="1" rowspan="1">Zeptometrix0801716</td><td colspan="1" rowspan="1">1.7E+07CFU/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Candida albicans</td><td colspan="1" rowspan="1">ATCCMYA-2876</td><td colspan="1" rowspan="1">7.88E+08CFU/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Cryptococcus neoformans</td><td colspan="1" rowspan="1">Zeptometrix0801803</td><td colspan="1" rowspan="1">6.8E+07CFU/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Pneumocystis jirovecii</td><td colspan="1" rowspan="1">ATCCPRA-159</td><td colspan="1" rowspan="1">1E+07CFU/mL</td><td colspan="1" rowspan="1">None</td></tr></table>

# in silico Exclusivity Analysis

Exclusivity alignment

in silico analyses were performed to evaluate any non-specific or off-target amplification that may occur during the out (PCR1) and/or inner (PCR2) reactions. Exclusivity of nearneighbor Coronaviridae species was evaluated by MAFFT alignment of available NCBI genomes. Alignment of 1,124 genomes were assessed representing relevant human Coronaviridae and other near neighbors, including bat coronaviruses. All primers were mapped to the reference sequence and the binding region was extracted and sorted by percent homology to the reference.

Exclusivity alignment indicated that individual COVID-19 primers show greater than or equal to $80 \%$ homology to one or more exclusivity genome of the Coronaviridae family. However, a detection result is not expected by the BioFire COVID-19 Test 2 device for the majority of these interactions because it would take both PCR2 primers to have $80 \%$ homology or greater for an assay detection. Nevertheless, it is possible that cross-reactivity of individual primers could interfere with PCR reactions by sequestering reagents needed for efficient amplification of targets. To address this risk of reduced sensitivity, the BioFire COVID-19 Test 2 includes an excess of primer in the reaction.

Only two exclusivity genomes showed homology to assay specific primer sets and are predicted to be detected by the BioFire COVID-19 Test 2 device. They included—Bat coronavirus RaTG13 (accession #MN996532) and Pangolin coronavirus isolate MP789 (accession #MT084071). The Bat coronavirus RaTG13 is the closes relative to the SARSCoV-2 with an overall genome sequence identity near $9 6 \%$ . The Pangolin coronavirus MP789 shows $80 \%$ or greater homology to multiple COVID-19 Test 2 primers but is only predicted to be detected by the SARS-CoV-2e assay. No other significant amplification of non-target sequences is predicted.

# Primer BLAST analysis

Specificity against relevant non-target organisms was evaluated using the BLAST algorithm with primer pairs. Since the FilmArray assay detection is based on efficient amplification from PCR1 and PCR2 reactions, organisms outside the Coronaviridae are not expected to have $80 \%$ homology to the nested primer pairs that comprise the device assays. However, because detection results are generated by the fluorescence of the inner PCR2 reaction, inner primer pairs were further assessed in silico using the NCBI primer BLAST tool. All combinations of PCR2 primers for a given assay were analyzed (i.e., forward/reverse, forward/forward, and reverse/reverse). Additional parameters were evaluated to allow more thorough BLAST assessment to include results with $80 \%$ or greater homology to one or both primers. Predicted amplicons greater than 1000bp were excluded on the basis that the device does not allow efficient production of such product length. The NCBI BLAST nr database was used in this analysis.

As a result of the BLAST alignment, exclusivity genomes identified are not considered to be concerning as the potential target amplicons are between 700-850bp and exceed the typical size that the FilmArray device would amplify (i.e., 100-200bp). Further, BLAST alignment had identified human genomic DNA as a potential amplified target with the primers. But the potential for cross-reactivity was actually evaluated experimentally in negativity testing with nasopharyngeal swabs specimens.

Overall, the in silico analysis indicated that only two exclusivity genomes showed significant homology to assay-specific PCR2 primer sets and thus predicted to be detected by the BioFire COVID-19 Test 2 device. However, the two identified isolates are not anticipated to be found in clinical NPS specimens. At present, little is known about the potential of these exclusivity species to infect a human host or their evolutionary relationship to SARS-CoV-2.

No other significant amplification of non-target sequences is predicted.

b) Interference

Potential interference was evaluated by comparing BioFire COVID-19 Test 2 results from contrived samples of SARS-CoV-2 in clinical NPS matrix at LoD concentration to results from samples with the same analyte composition but including a potential interfering substance. Three replicates of each potential interferrent were evaluated. The concentrations of substances tested were selected to represent normal to high levels and exceed what may be present in clinical specimens. Reproducible control failures or loss of analyte detection in the presence of test substances may be indicative of possible interference.

The primary data for evaluation from this study were the Pass/Fail results for the pouch control assays and the Detected/Not Detected results for the analyte. An unexpected result was defined as a control failure or unexpected replicate negative or positive analyte result. An unexpected result in a single replicate (1/3) indicated the need for testing two additional replicates. If ${ \ge } 2 / 3$ or ${ \ge } 2 / 5$ unexpected results were observed the substance may be interfering; retesting at the same interfering substance concentration and additional testing at lower test substance concentrations were performed to determine at what concentration interference is no longer observed. Note that SARS-COV-2 was spiked at 1x LoD.

The following table provides a list of potentially interfering substances and the observed results—

Table 7. Results for Potentially Interfering Substances Tested on the BioFire COVID-19 Test 2 Panel.   

<table><tr><td colspan="1" rowspan="1">Substance</td><td colspan="1" rowspan="1">Specific Active Ingredient</td><td colspan="1" rowspan="1">Concentrationtested</td><td colspan="1" rowspan="1">Results</td></tr><tr><td colspan="1" rowspan="1">Toothpaste (Colgate Total)</td><td colspan="1" rowspan="1">Stannous fluoride 0.454%</td><td colspan="1" rowspan="1">2% v/v</td><td colspan="1" rowspan="1">No interference</td></tr><tr><td colspan="1" rowspan="1">Tobacco (Camel Snus)</td><td colspan="1" rowspan="1">Tobacco</td><td colspan="1" rowspan="1">10 mg/mL</td><td colspan="1" rowspan="1">No interference</td></tr><tr><td colspan="1" rowspan="1">Oral Rinse (Listerine)</td><td colspan="1" rowspan="1">Eucalyptol 0.092%Menthol 0.042%Methyl Salicylate 0.060%Thymol 0.064%</td><td colspan="1" rowspan="1">1% v/v</td><td colspan="1" rowspan="1">No interference</td></tr><tr><td colspan="1" rowspan="1">Throat lozenges (Cepacol)</td><td colspan="1" rowspan="1">Benzocaine 7.5 mgDextromethorphan HBr 5 mg</td><td colspan="1" rowspan="1">2.2 mg/mL</td><td colspan="1" rowspan="1">No interference</td></tr><tr><td colspan="1" rowspan="1">Oral anesthetic andanalgesic (Mouth SoreRelief)</td><td colspan="1" rowspan="1">Benzocaine 20%</td><td colspan="1" rowspan="1">1% v/v</td><td colspan="1" rowspan="1">No interference</td></tr><tr><td colspan="1" rowspan="1">Cough Drops</td><td colspan="1" rowspan="1">Menthol 5.4 mg</td><td colspan="1" rowspan="1">2.2 mg/mL</td><td colspan="1" rowspan="1">No interference</td></tr><tr><td colspan="1" rowspan="1">Cleanser (Dye-FreeAntiseptic Cleanser)</td><td colspan="1" rowspan="1">Chlorhexidine Gluconate 4%</td><td colspan="1" rowspan="1">1% v/v</td><td colspan="1" rowspan="1">No interference</td></tr><tr><td colspan="1" rowspan="1">Nicotine</td><td colspan="1" rowspan="1">Nicotine</td><td colspan="1" rowspan="1">10 mg/mL</td><td colspan="1" rowspan="1">No interference</td></tr><tr><td colspan="1" rowspan="1">Mucin</td><td colspan="1" rowspan="1">Bovine submaxillary gland, type I-S Sigma M3895</td><td colspan="1" rowspan="1">5 mg/mL</td><td colspan="1" rowspan="1">No interference</td></tr><tr><td colspan="1" rowspan="1">Blood (human)</td><td colspan="1" rowspan="1">Human DNA</td><td colspan="1" rowspan="1">5% v/v</td><td colspan="1" rowspan="1">No interference</td></tr><tr><td colspan="1" rowspan="1">Leukocytes</td><td colspan="1" rowspan="1">Human DNA</td><td colspan="1" rowspan="1">1% v/v</td><td colspan="1" rowspan="1">No interference</td></tr><tr><td colspan="1" rowspan="1">Nasal spray (Wal-FourNasal Spray)</td><td colspan="1" rowspan="1">Phenylephrine hydrochloride 1%</td><td colspan="1" rowspan="1">10% v/v</td><td colspan="1" rowspan="1">No interference</td></tr><tr><td colspan="1" rowspan="1">Afrin Nasal Spray</td><td colspan="1" rowspan="1">Oxymetazoline hydrochloride0.05%</td><td colspan="1" rowspan="1">10% v/v</td><td colspan="1" rowspan="1">No interference</td></tr><tr><td colspan="1" rowspan="1">Saline Nasal Spray</td><td colspan="1" rowspan="1">NaC1 0.65% with preservatives(Phenylcarbinol, BenzalkoniumChloride)</td><td colspan="1" rowspan="1">10% v/v</td><td colspan="1" rowspan="1">No interference</td></tr><tr><td colspan="1" rowspan="3">Nasal corticosteroids</td><td colspan="1" rowspan="1">Beclomethasone</td><td colspan="1" rowspan="1">2 mg/mL</td><td colspan="1" rowspan="1">No interference</td></tr><tr><td colspan="1" rowspan="1">Dexamethasone</td><td colspan="1" rowspan="1">1.5 mg/mL</td><td colspan="1" rowspan="1">No interference</td></tr><tr><td colspan="1" rowspan="1">Flunisolide</td><td colspan="1" rowspan="1">2 mg/mL</td><td colspan="1" rowspan="1">No interference</td></tr><tr><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="1">Triamcinolone</td><td colspan="1" rowspan="1">5 mg/mL</td><td colspan="1" rowspan="1">No interference</td></tr><tr><td colspan="1" rowspan="1">Mometasone</td><td colspan="1" rowspan="1">1 mg/mL</td><td colspan="1" rowspan="1">No interference</td></tr><tr><td colspan="1" rowspan="1">Budesonide Nasal Spray</td><td colspan="1" rowspan="1">Budesonide 32 mcg/ spray</td><td colspan="1" rowspan="1">1% v/v</td><td colspan="1" rowspan="1">No interference</td></tr><tr><td colspan="1" rowspan="1">Allergy Nasal Spray</td><td colspan="1" rowspan="1">Fluticasone 50 mcg/ spray</td><td colspan="1" rowspan="1">1% v/v</td><td colspan="1" rowspan="1">No interference</td></tr><tr><td colspan="1" rowspan="1">Nasal gel(Zicam)</td><td colspan="1" rowspan="1">Luffa opperculata 4xGalphimia glauca 4xSabadilla 4x</td><td colspan="1" rowspan="1">1% v/v</td><td colspan="1" rowspan="1">No interference</td></tr><tr><td colspan="1" rowspan="1">Sulfur</td><td colspan="1" rowspan="1">Sulfur</td><td colspan="1" rowspan="1">0.17 mg/mL</td><td colspan="1" rowspan="1">No interference</td></tr><tr><td colspan="1" rowspan="1">Allergy Relief(RhinAllergy)</td><td colspan="1" rowspan="1">Histaminum hydrochloricum 9CHPUS</td><td colspan="1" rowspan="1">10 mg/mL</td><td colspan="1" rowspan="1">No interference</td></tr><tr><td colspan="1" rowspan="1">Anti-viral drugs</td><td colspan="1" rowspan="1">Zanamivir</td><td colspan="1" rowspan="1">5.5 mg/mL</td><td colspan="1" rowspan="1">No interference</td></tr><tr><td colspan="1" rowspan="1">Antibiotic, Nasal ointment</td><td colspan="1" rowspan="1">Mupirocin</td><td colspan="1" rowspan="1">3.3 mg/mL</td><td colspan="1" rowspan="1">No interference</td></tr><tr><td colspan="1" rowspan="1">Antibacterial, systemic</td><td colspan="1" rowspan="1">Tobramycin</td><td colspan="1" rowspan="1">4 μg/mL</td><td colspan="1" rowspan="1">No interference</td></tr><tr><td colspan="1" rowspan="9">Transport Media</td><td colspan="1" rowspan="1">Remel M4 (R12503)</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">No interference</td></tr><tr><td colspan="1" rowspan="1">Remel M4RT (R12591)</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">No interference</td></tr><tr><td colspan="1" rowspan="1">Copan UTM-RT (UTM 330C)</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">No interference</td></tr><tr><td colspan="1" rowspan="1">PrimeStore MTM (MTM-LH102)</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">No interference</td></tr><tr><td colspan="1" rowspan="1">Merit Medical Cultura Media</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">No interference</td></tr><tr><td colspan="1" rowspan="1">Neuronics VTM</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">No interference</td></tr><tr><td colspan="1" rowspan="1">Azer UTM</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">No interference</td></tr><tr><td colspan="1" rowspan="1">Bartels FlexTrans</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">No interference</td></tr><tr><td colspan="1" rowspan="1">S2 VTM</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">No interference</td></tr><tr><td colspan="1" rowspan="5">Diluents</td><td colspan="1" rowspan="1">Ethanol</td><td colspan="1" rowspan="1">10% v/v</td><td colspan="1" rowspan="1">No interference</td></tr><tr><td colspan="1" rowspan="1">DMSO</td><td colspan="1" rowspan="1">10% v/v</td><td colspan="1" rowspan="1">No interference</td></tr><tr><td colspan="1" rowspan="1">Methanol</td><td colspan="1" rowspan="1">10% v/v</td><td colspan="1" rowspan="1">No interference</td></tr><tr><td colspan="1" rowspan="1">Chloroform</td><td colspan="1" rowspan="1">10% v/v</td><td colspan="1" rowspan="1">No interference</td></tr><tr><td colspan="1" rowspan="1">DMF</td><td colspan="1" rowspan="1">10% v/v</td><td colspan="1" rowspan="1">No interference</td></tr></table>

None of the substances tested were determined to be inhibitory to the BioFire COVID-19 Test 2 according to the acceptance criteria.

4. Assay Reportable Range: Not applicable; this is a qualitative assay.

5. Traceability, Stability, Expected Values (Controls, Calibrators, or Methods):

# Assay Controls

Two process controls are included in each BioFire COVID-19 Test 2 pouch:

# RNA Process Control

The RNA Process Control assay targets an RNA transcript from the yeast Schizosaccharomyces pombe. The yeast is present in the pouch in a freeze-dried form and becomes rehydrated when sample is loaded. The control material is carried through all stages of the test process, including lysis, nucleic acid purification, reverse transcription, PCR1, dilution, PCR2, and DNA melting. A positive result indicates that all steps carried out in the BioFire COVID-19 Test 2 pouch were successful.

# PCR2 Control

The PCR2 Control assay detects a DNA target that is dried into wells of the array along with the corresponding primers. A positive result indicates that PCR2 was successful.

Both control assays must be positive for the test run to pass. If any of the controls fail, the user is instructed to re-run the sample using a new pouch.

# Specimen Stability

Specimens should be stored at the following conditions: up to 4 hours ambient $( 1 5 . 3 0 ^ { \circ } \mathrm { C } )$ , up to 3 days refrigerated $\left( 2 \mathrm { - } 8 ^ { \circ } \mathrm { C } \right)$ , and up to 30 days frozen $( \leq - 1 5 ^ { \circ } \mathrm { C } )$ .

A study was performed to evaluate the claimed storage conditions for specimens and likely encountered during standard of care testing. Ten specimens contrived by spiking pooled residual NPS specimens with inactivated SARS-CoV-2 at 3x, 5x, and $1 0 \mathrm { x }$ LoD. Each contrived specimen was tested immediately as the control, then aliquoted and stored under the different storage conditions corresponding to specimen storage claims. Storage conditions were evaluated at each time point based on the detection rate and identification of any concerning trends in detection.

Detailed documentation concerning NPS in saline sample storage and transport regarding the stability of SARS-CoV-2 in that media was provided. It was demonstrated that SARS-CoV-2 was stable in NPS specimens collected in saline for up to 4 hours at ambient temperature $( 3 0 ^ { \circ } \mathrm { C } )$ , 3 days refrigeration, and 30 days frozen. The stability of SARS-CoV-2 in NPS specimens collected in saline had been previously demonstrated using the FilmArray system.

Overall, the recommended specimen storage conditions for the device appear to follow from the various evaluations of the FilmArray systems, including the detection of all replicates according to the acceptance criteria in the analytical study at the different contrived sample concentrations, temperature, and corresponding storage duration.

# Fresh vs. Frozen Study

A fresh vs. frozen study was not performed separately for this device but detailed documentation for NPS in VTM specimens had been previously included in other submissions involving the FilmArray systems (e.g., K103175, K110764, K120267). Further, frozen archived clinical samples were not used to supplement clinical and/or analytical study data for the BioFire COVID-19 Test 2 device.

# 6. Detection Limit:

The Limit of Detection (LoD) in NPS was determined using infectious and heat inactivated SARS-CoV-2 (USA_WA1/2020) spiked into unique clinical NPS specimens in transport medium. LoD is defined as the lowest concentration of SARS-CoV-2 RNA that can be detected at a rate of at least $9 5 \%$ (19/20 runs). Tentative LoD was determined using three-fold dilutions of quantified infectious virus. Subsequently, the tentative LoD was confirmed by testing additional replicates to demonstrate a $2 9 5 \%$ detection rate at LoD, and a ${ < } 9 5 \%$ detection rate at $0 . 1 { \times } \mathrm { L o D }$ . The LoD evaluation was performed using the FilmArray 2.0 system.

To evaluate the potential effects of variable clinical sample matrix on analyte detection, all testing in this study was performed using individual NPS specimens in transport media previously collected from patients suspected of having respiratory disease. Virus stock

dilutions (infectious or heat activated) were prepared in saline and subsequently spiked into unique NPS specimens in transport media for testing with the BioFire COVID-19 Test 2 device.

# Infectious SARS-CoV-2 LoD

For the tentative LoD, four replicates were tested for seven 3-fold serial dilution starting at $3 . 3 \mathrm { E } { + 0 4 }$ genomic copies $\mathrm { ( G C ) / m L }$ . The following table shows the test results in the tentative LoD study for infectious SARS-CoV-2, including the Detection result for the three individual SARS-CoV-2 assays—

Table 8. SARS-CoV-2 USA-WA1/2020 Estimation of LoD Test Results $\mathbf { ( D = }$ Detected; $\mathbf { N D } = \mathbf { N o t }$ Detected)   

<table><tr><td rowspan=2 colspan=1>ConcentrationGC/mL(TCID50/mL)</td><td rowspan=1 colspan=2>Overall TestResult Rate</td><td rowspan=2 colspan=1>Replicate</td><td rowspan=2 colspan=1>SARS-CoV-2a</td><td rowspan=2 colspan=1>SARS-CoV-2d</td><td rowspan=2 colspan=1>SARS-CoV-2e</td></tr><tr><td rowspan=1 colspan=1>D</td><td rowspan=1 colspan=1>ND</td></tr><tr><td rowspan=4 colspan=1>3.3E+04(2.2E+00)</td><td rowspan=4 colspan=1>4/4</td><td rowspan=4 colspan=1></td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>D</td><td rowspan=1 colspan=1>D</td><td rowspan=1 colspan=1>D</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>D</td><td rowspan=1 colspan=1>D</td><td rowspan=1 colspan=1>D</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>D</td><td rowspan=1 colspan=1>D</td><td rowspan=1 colspan=1>D</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>D</td><td rowspan=1 colspan=1>D</td><td rowspan=1 colspan=1>D</td></tr><tr><td rowspan=4 colspan=1>1.0E+04(6.6E-01)</td><td rowspan=4 colspan=1>4/4</td><td rowspan=4 colspan=1></td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>D</td><td rowspan=1 colspan=1>D</td><td rowspan=1 colspan=1>D</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>D</td><td rowspan=1 colspan=1>D</td><td rowspan=1 colspan=1>D</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>D</td><td rowspan=1 colspan=1>D</td><td rowspan=1 colspan=1>D</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>D</td><td rowspan=1 colspan=1>D</td><td rowspan=1 colspan=1>D</td></tr><tr><td rowspan=4 colspan=1>3.3E+03(2.2E-01)</td><td rowspan=4 colspan=1>4/4</td><td rowspan=4 colspan=1></td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>D</td><td rowspan=1 colspan=1>D</td><td rowspan=1 colspan=1>D</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>D</td><td rowspan=1 colspan=1>D</td><td rowspan=1 colspan=1>D</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>D</td><td rowspan=1 colspan=1>D</td><td rowspan=1 colspan=1>D</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>D</td><td rowspan=1 colspan=1>D</td><td rowspan=1 colspan=1>D</td></tr><tr><td rowspan=4 colspan=1>1.0E+03(6.6E-02)</td><td rowspan=4 colspan=1>4/4</td><td rowspan=4 colspan=1>−</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>D</td><td rowspan=1 colspan=1>D</td><td rowspan=1 colspan=1>D</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>D</td><td rowspan=1 colspan=1>D</td><td rowspan=1 colspan=1>D</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>D</td><td rowspan=1 colspan=1>D</td><td rowspan=1 colspan=1>D</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>D</td><td rowspan=1 colspan=1>D</td><td rowspan=1 colspan=1>D</td></tr><tr><td rowspan=4 colspan=1>3.3E+02(2.2E-02)</td><td rowspan=4 colspan=1>4/4</td><td rowspan=4 colspan=1></td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>D</td><td rowspan=1 colspan=1>D</td><td rowspan=1 colspan=1>D</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>D</td><td rowspan=1 colspan=1>D</td><td rowspan=1 colspan=1>D</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>D</td><td rowspan=1 colspan=1>D</td><td rowspan=1 colspan=1>D</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>D</td><td rowspan=1 colspan=1>D</td><td rowspan=1 colspan=1>D</td></tr><tr><td rowspan=4 colspan=1>1.0E+02(6.6E-03)</td><td rowspan=4 colspan=1>4/4</td><td rowspan=4 colspan=1></td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>D</td><td rowspan=1 colspan=1>D</td><td rowspan=1 colspan=1>D</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>D</td><td rowspan=1 colspan=1>D</td><td rowspan=1 colspan=1>D</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>D</td><td rowspan=1 colspan=1>D</td><td rowspan=1 colspan=1>D</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>D</td><td rowspan=1 colspan=1>ND</td><td rowspan=1 colspan=1>D</td></tr><tr><td rowspan=4 colspan=1>3.3E+01(2.2E-03)</td><td rowspan=4 colspan=1>2/4</td><td rowspan=4 colspan=1>2/4</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>ND</td><td rowspan=1 colspan=1>ND</td><td rowspan=1 colspan=1>ND</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>D</td><td rowspan=1 colspan=1>ND</td><td rowspan=1 colspan=1>ND</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>D</td><td rowspan=1 colspan=1>ND</td><td rowspan=1 colspan=1>ND</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>ND</td><td rowspan=1 colspan=1>ND</td><td rowspan=1 colspan=1>ND</td></tr></table>

At $1 . 0 \mathrm { E } { + } 0 2 ~ \mathrm { G C / m L }$ there was a negative result for one of the SARS-CoV-2 assays. The lowest concentration evaluated in the tentative LoD study yielded only $50 \%$ detection.

Therefore, the concentration of $3 . 3 \mathrm { E } { + 0 2 }$ GC/mL was further evaluated for LoD confirmation. The confirmed LoD replicate evaluation with infectious SARS-CoV-2 was summarized as follows—

Table 9. LoD Confirmation Test Results at $\mathbf { 1 } \times \mathbf { L o D }$ (3.3E+02 GC/mL; 2.2E-02 TCID50/mL)  

<table><tr><td rowspan=2 colspan=1>Replicate</td><td rowspan=1 colspan=3>SARS-CoV-2 Assay</td></tr><tr><td rowspan=1 colspan=1>a</td><td rowspan=1 colspan=1>d</td><td rowspan=1 colspan=1>e</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>D</td><td rowspan=1 colspan=1>D</td><td rowspan=1 colspan=1>D</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>D</td><td rowspan=1 colspan=1>D</td><td rowspan=1 colspan=1>D</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>D</td><td rowspan=1 colspan=1>D</td><td rowspan=1 colspan=1>D</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>D</td><td rowspan=1 colspan=1>D</td><td rowspan=1 colspan=1>D</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>D</td><td rowspan=1 colspan=1>D</td><td rowspan=1 colspan=1>D</td></tr><tr><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>D</td><td rowspan=1 colspan=1>D</td><td rowspan=1 colspan=1>D</td></tr><tr><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>D</td><td rowspan=1 colspan=1>D</td><td rowspan=1 colspan=1>D</td></tr><tr><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>D</td><td rowspan=1 colspan=1>D</td><td rowspan=1 colspan=1>D</td></tr><tr><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>D</td><td rowspan=1 colspan=1>D</td><td rowspan=1 colspan=1>D</td></tr><tr><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>D</td><td rowspan=1 colspan=1>D</td><td rowspan=1 colspan=1>D</td></tr><tr><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>D</td><td rowspan=1 colspan=1>D</td><td rowspan=1 colspan=1>D</td></tr><tr><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>D</td><td rowspan=1 colspan=1>D</td><td rowspan=1 colspan=1>D</td></tr><tr><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>D</td><td rowspan=1 colspan=1>D</td><td rowspan=1 colspan=1>D</td></tr><tr><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>D</td><td rowspan=1 colspan=1>D</td><td rowspan=1 colspan=1>D</td></tr><tr><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>D</td><td rowspan=1 colspan=1>D</td><td rowspan=1 colspan=1>D</td></tr><tr><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>D</td><td rowspan=1 colspan=1>D</td><td rowspan=1 colspan=1>D</td></tr><tr><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>D</td><td rowspan=1 colspan=1>D</td><td rowspan=1 colspan=1>D</td></tr><tr><td rowspan=1 colspan=1>18</td><td rowspan=1 colspan=1>D</td><td rowspan=1 colspan=1>D</td><td rowspan=1 colspan=1>D</td></tr><tr><td rowspan=1 colspan=1>19</td><td rowspan=1 colspan=1>D</td><td rowspan=1 colspan=1>D</td><td rowspan=1 colspan=1>D</td></tr><tr><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>D</td><td rowspan=1 colspan=1>D</td><td rowspan=1 colspan=1>D</td></tr><tr><td rowspan=1 colspan=1>DetectionRate</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>20/20</td></tr></table>

All 20 replicates tested at $3 . 3 \mathrm { E } { + } 0 2 ~ \mathrm { G C / m L }$ had a Detected result for SARS-CoV-2 and therefore, LoD confirmation testing continued at 10-fold lower. The detection rate at $3 . 3 \mathrm { E } { + 0 1 }$ $\mathrm { G C / m L }$ was only $8 7 . 5 \%$ or less than the confirmed LoD acceptance criteria.

Therefore, the LoD was confirmed at $\mathbf { 3 . 3 + 0 2 ~ G C / m L }$ for the infectious SARS-CoV-2.

# Inactivated SARS-CoV-2 LoD

For the tentative LoD, five replicates were tested for five 3-fold serial dilution starting at $3 . 3 \mathrm { E } { + 0 3 }$ genomic equivalents $\mathrm { ( G E ) / m L }$ . The following is the result of the tentative LoD study with heat inactivated SARS-CoV-2—

Table 10. Tentative LoD Results for Heat Inactivated SARS-CoV-2 $\mathbf { G E } =$ genomic equivalents)

<table><tr><td rowspan=1 colspan=1>Test Level(GE/mL)</td><td rowspan=1 colspan=1>Detection Raten/N (%)</td><td rowspan=1 colspan=1>Replicate</td><td rowspan=1 colspan=1>SARS-CoV-2a</td><td rowspan=1 colspan=1>SARS-CoV-2d</td><td rowspan=1 colspan=1>SARS-CoV-2e</td></tr><tr><td rowspan=5 colspan=1>3.3E+03</td><td rowspan=5 colspan=1>5/5(100%)</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>D</td><td rowspan=1 colspan=1>D</td><td rowspan=1 colspan=1>D</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>D</td><td rowspan=1 colspan=1>D</td><td rowspan=1 colspan=1>D</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>D</td><td rowspan=1 colspan=1>D</td><td rowspan=1 colspan=1>D</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>D</td><td rowspan=1 colspan=1>D</td><td rowspan=1 colspan=1>D</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>D</td><td rowspan=1 colspan=1>D</td><td rowspan=1 colspan=1>D</td></tr><tr><td rowspan=5 colspan=1>1.0E+03</td><td rowspan=5 colspan=1>5/5(100%)</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>D</td><td rowspan=1 colspan=1>D</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>D</td><td rowspan=1 colspan=1>D</td><td rowspan=1 colspan=1>D</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>D</td><td rowspan=1 colspan=1>D</td><td rowspan=1 colspan=1>D</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>D</td><td rowspan=1 colspan=1>D</td><td rowspan=1 colspan=1>D</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>D</td><td rowspan=1 colspan=1>D</td><td rowspan=1 colspan=1>D</td></tr><tr><td rowspan=5 colspan=1>3.3E+02</td><td rowspan=5 colspan=1>5/5(100%)</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>D</td><td rowspan=1 colspan=1>D</td><td rowspan=1 colspan=1>D</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>D</td><td rowspan=1 colspan=1>D</td><td rowspan=1 colspan=1>D</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>D</td><td rowspan=1 colspan=1>D</td><td rowspan=1 colspan=1>D</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>D</td><td rowspan=1 colspan=1>D</td><td rowspan=1 colspan=1>D</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>D</td><td rowspan=1 colspan=1>D</td><td rowspan=1 colspan=1>D</td></tr><tr><td rowspan=5 colspan=1>1.0E+02</td><td rowspan=5 colspan=1>5/5(100%)</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>D</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>D</td><td rowspan=1 colspan=1>D</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>D</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>D</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>D</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>D</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=5 colspan=1>3.3E+01</td><td rowspan=5 colspan=1>4/5(80%)</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>D</td><td rowspan=1 colspan=1>D</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>D</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>D</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>D</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>D</td></tr></table>

Subsequently the confirmed LoD for heat inactivated SARS-CoV-2 was evaluated at $3 . 3 \mathrm { E } { + } 0 2 ~ \mathrm { G E / m L }$ . Twenty individually contrived replicates in pooled NPS matrix was evaluated. The results are as follows—

Table 11. LoD Confirmation Test Results at $3 . 3 \mathrm { E } { + } \mathbf { 0 } 2 ~ \mathrm { G E } / \mathbf { m L }$ (1× LoD)   

<table><tr><td colspan="1" rowspan="2">Replicate</td><td colspan="3" rowspan="1">SARS-CoV-2 Assay</td></tr><tr><td colspan="1" rowspan="1">a</td><td colspan="1" rowspan="1">d</td><td colspan="1" rowspan="1">e</td></tr><tr><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">D</td><td colspan="1" rowspan="1">D</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">D</td><td colspan="1" rowspan="1">D</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">D</td><td colspan="1" rowspan="1">D</td><td colspan="1" rowspan="1">D</td></tr><tr><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">D</td><td colspan="1" rowspan="1">D</td><td colspan="1" rowspan="1">D</td></tr><tr><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">D</td><td colspan="1" rowspan="1">D</td><td colspan="1" rowspan="1">D</td></tr><tr><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">D</td><td colspan="1" rowspan="1">D</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">7</td><td colspan="1" rowspan="1">D</td><td colspan="1" rowspan="1">D</td><td colspan="1" rowspan="1">D</td></tr><tr><td colspan="1" rowspan="1">8</td><td colspan="1" rowspan="1">D</td><td colspan="1" rowspan="1">D</td><td colspan="1" rowspan="1">D</td></tr><tr><td colspan="1" rowspan="1">9</td><td colspan="1" rowspan="1">D</td><td colspan="1" rowspan="1">D</td><td colspan="1" rowspan="1">D</td></tr><tr><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">D</td><td colspan="1" rowspan="1">D</td><td colspan="1" rowspan="1">D</td></tr><tr><td colspan="1" rowspan="1">11</td><td colspan="1" rowspan="1">D</td><td colspan="1" rowspan="1">D</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">12</td><td colspan="1" rowspan="1">D</td><td colspan="1" rowspan="1">D</td><td colspan="1" rowspan="1">D</td></tr><tr><td colspan="1" rowspan="1">13</td><td colspan="1" rowspan="1">D</td><td colspan="1" rowspan="1">D</td><td colspan="1" rowspan="1">D</td></tr><tr><td colspan="1" rowspan="1">14</td><td colspan="1" rowspan="1">D</td><td colspan="1" rowspan="1">D</td><td colspan="1" rowspan="1">D</td></tr><tr><td colspan="1" rowspan="1">15</td><td colspan="1" rowspan="1">D</td><td colspan="1" rowspan="1">b</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">16</td><td colspan="1" rowspan="1">D</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">D</td></tr><tr><td colspan="1" rowspan="1">17</td><td colspan="1" rowspan="1">D</td><td colspan="1" rowspan="1">D</td><td colspan="1" rowspan="1">D</td></tr><tr><td colspan="1" rowspan="1">18</td><td colspan="1" rowspan="1">D</td><td colspan="1" rowspan="1">D</td><td colspan="1" rowspan="1">D</td></tr><tr><td colspan="1" rowspan="1">19</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">D</td><td colspan="1" rowspan="1">D</td></tr><tr><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">D</td><td colspan="1" rowspan="1">D</td><td colspan="1" rowspan="1">D</td></tr><tr><td colspan="1" rowspan="1">DetectionRate</td><td colspan="1" rowspan="1">19/20</td><td colspan="1" rowspan="1">18/20</td><td colspan="1" rowspan="1">15/20</td></tr></table>

All 20 replicates tested at $3 . 3 \mathrm { E } { + 0 2 }$ GE/mL returned “Detected” results for SARS-CoV-2, even though there was variable performance at that concentration for the individual assays. LoD confirmation proceeded with testing twenty replicates at $3 . 3 \mathrm { E } { + 0 1 }$ GE/mL (0.1x LoD). Only eight replicates returned a “Detected” result which was $< 1 9 / 2 0$ (0.1x LoD).

Therefore, the LoD was confirmed at $\mathbf { 3 . 3 + 0 2 ~ G E / m L }$ for the heat inactivated SARS-CoV-2.

A summary of the confirmed LoD for the BioFire COVID-19 Test 2 device is as follows—

Table 12. Summary of LoD Results for the BioFire COVID-19 Test 2   

<table><tr><td rowspan=2 colspan=1>Variant(Source)</td><td rowspan=1 colspan=3>LoD Concentration</td><td rowspan=2 colspan=1># Detected/Total(% Detection)</td></tr><tr><td rowspan=1 colspan=1>×LoD</td><td rowspan=1 colspan=1>Nucleic Acid</td><td rowspan=1 colspan=1>TCID50/mL</td></tr><tr><td rowspan=2 colspan=1>USA-WA1/2020(Infectious culture; WRCEVA) a</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>3.3E+02 GC/mL</td><td rowspan=1 colspan=1>2.2E-02</td><td rowspan=1 colspan=1>20/20(100%)</td></tr><tr><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>3.3E+01 GC/mL</td><td rowspan=1 colspan=1>2.2E-03</td><td rowspan=1 colspan=1>19/20(95%)</td></tr><tr><td rowspan=2 colspan=1>USA-WA1/2020(heat inactivated;BEI NR-52286) b</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>3.3E+02 GE/mL</td><td rowspan=1 colspan=1>4.3E-02</td><td rowspan=1 colspan=1>20/20(100%)</td></tr><tr><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>3.3E+01 GE/mL</td><td rowspan=1 colspan=1>4.3E-03</td><td rowspan=1 colspan=1>8/20(40%)</td></tr></table>

a Obtained for culture in a biosafety level 3 laboratory from the World Reference Center for Emerging Viruses and Arboviruses (WRCEVA). Concentration determined by quantitative real-time PCR as described on the World Health Organization website: https://www.who.int/docs/default-source/coronaviruse/protocol-v2-1.pdf b Concentration determined by digital droplet PCR as indicated on the Certificate of Analysis provided by BEI Resources. $\mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ was determined prior to inactivation.

# FDA Reference Panel Evaluation

The analytical sensitivity of the BioFire COVID-19 Test 2 device was evaluated using the U.S. FDA SARS-CoV-2 Verification Panel. The verification panel consists of five samples (T1-T5) containing heat-inactivated strains of SARS-CoV-2 and Middle East Respiratory Syndrome (MERS)-CoV. The study was divided into three types of evaluations—range finding to obtain an estimate of LoD, confirmation of the LoD, and testing of blinded control samples. Samples containing MERS-CoV are expected to return “Not Detected” results and were used to indicate specificity. All testing was performed with pooled negative nasopharyngeal swabs (NPS) as a sample matrix.

During the range-finding part of testing to estimate LoD, serial 10-fold dilutions of one of the samples on the verification panel was used to span concentrations from $1 . 8 \mathrm { E } { + 0 6 }$ to $1 . 8 \mathrm { E } { + } 0 0 $ NDU/mL (where ${ \mathrm { N D U } } = \mathrm { N }$ ucleic Acid Amplification Test detectable units). Three replicates at each concentration was evaluated and the lowest concentration at which all three replicates returned a “Detected” result was selected for further confirmation. The initial estimated, lowest concentration that met the criteria was $1 . 8 \mathrm { E } { + } 0 3 ~ \mathrm { N D U / m L }$ . This concentration was further evaluated by testing an additional 17 replicates (total of 20). In addition, 20 replicates at concentrations 3-fold above and 3-fold below the target concentration was also evaluated.

In addition, four blinded samples were evaluated according to the supplied protocol. The protocol instructed the dilutions of panel samples that should be evaluated in five replicates each. Further, the firm was instructed to calculate the concentration of blinded samples by comparing their Detection results to a standard curve determined in the initial range-finding testing. This was performed solely for the purposes of verifying the panel samples as the BioFire COVID-19 Test 2 device is qualitative and does not report quantitative values in test results. These calculations were then reported for the four blinded samples. Based on the blinded testing results and further review of the reference panel testing, an LoD was adjusted to the next confirmed concentration from the preliminary assessment of $1 . 8 \mathrm { E } { + } 0 3 ~ \mathrm { N D U / m L }$ .

The LoD according to the FDA reference panel testing was confirmed to be $5 . 4 \mathrm { E } { + 0 3 }$ NDU/mL. No cross-reactivity with MERS-CoV was reported.

# 7. Analytical Reactivity:

In silico Inclusivity Analysis

An in silico evaluation of primer sequences in the BioFire COVID-19 Test 2 device was performed periodically to track any variants appearing in the primer binding regions as more SARS-CoV-2 sequences became available.

BioFire Defense employs a custom bioinformatics tool to monitor inclusivity of assay primer regions in emerging global variants of the target pathogen, and to therefore mitigate any potential risks that these variants may have on device performance. In essence, the tool compares primer binding regions to corresponding regions on the reference sequence and classifies any changes in a binning function. The binning function helps identify perfect matches, mismatches, and deletions (accounting for mutation type, nucleotide change, location, and affected primer), and also sequences with ambiguities in the primer region. This classification scheme with the binning function facilitates identifying variant frequencies as results are captured into a database for continual updating and querying.

Overall, two key criterion are employed in the variant monitoring. First, only variants occurring at or above $0 . 1 \%$ frequency of the total sequences collected quarterly are considered to filter out possible sequencing errors, rare genotypes, or variants without competitive advantage. Second, sequences that meet the first criteria are further filtered out for variants that do not fall within 10bp of the $_ { 3 } ,$ end of the primer.

Global in silico analysis (as of August, 2021) by BioFire Defense predicted that the BioFire COVID-19 Test 2 SARS-CoV-2 assays will detect all 497,598 sequences evaluated. The analysis included all Variants of Interest (VOIs) and Variants of Concern (VOCs), as defined by the WHO on June 1, 2021 (https://www.who.int/en/activities/tracking-sars-cov-2-variants) and included as follows—

Table 13. WHO List of Variants of Interest and Variants of Concern (June 2021)   

<table><tr><td rowspan=1 colspan=1>WHODesignation</td><td rowspan=1 colspan=1>WHOLabel</td><td rowspan=1 colspan=1>PangoLineage</td><td rowspan=1 colspan=1>GISAIDClade/Lineage</td><td rowspan=1 colspan=1>NextstrainClade</td><td rowspan=1 colspan=1>EarliestDocumented</td><td rowspan=1 colspan=1>Date of Designation</td></tr><tr><td rowspan=4 colspan=1>VOC</td><td rowspan=1 colspan=1>Alpha</td><td rowspan=1 colspan=1>B.1.1.7</td><td rowspan=1 colspan=1>GRY (formerlyGR/501Y.V1)</td><td rowspan=1 colspan=1>201 (V1)</td><td rowspan=1 colspan=1>UK, Sept-2020</td><td rowspan=1 colspan=1>18-Dec-20</td></tr><tr><td rowspan=1 colspan=1>Beta</td><td rowspan=1 colspan=1>B.1.351</td><td rowspan=1 colspan=1>GH/501Y.V2</td><td rowspan=1 colspan=1>20H (V2)</td><td rowspan=1 colspan=1>South Africa,May-2020</td><td rowspan=1 colspan=1>18-Dec-20</td></tr><tr><td rowspan=1 colspan=1>Gamma</td><td rowspan=1 colspan=1>P.1</td><td rowspan=1 colspan=1>GR/501Y.V3</td><td rowspan=1 colspan=1>20J (V3)</td><td rowspan=1 colspan=1>Brazil, Nov-2020</td><td rowspan=1 colspan=1>11-Jan-21</td></tr><tr><td rowspan=1 colspan=1>Delta</td><td rowspan=1 colspan=1>B.1.617.2</td><td rowspan=1 colspan=1>G/478K.V1</td><td rowspan=1 colspan=1>21A</td><td rowspan=1 colspan=1>India, Oct-2020</td><td rowspan=1 colspan=1>VOI: 4-Apr-2021</td></tr><tr><td rowspan=7 colspan=1>VOI</td><td rowspan=1 colspan=1>Epsilon</td><td rowspan=1 colspan=1>B.1.427/B.1.429</td><td rowspan=1 colspan=1>GH/452R.V1</td><td rowspan=1 colspan=1>21C</td><td rowspan=1 colspan=1>USA, Mar-2020</td><td rowspan=1 colspan=1>5-Mar-2021</td></tr><tr><td rowspan=1 colspan=1>Zeta</td><td rowspan=1 colspan=1>P.2</td><td rowspan=1 colspan=1>GR/484K.V2</td><td rowspan=1 colspan=1>20B/S.484K</td><td rowspan=1 colspan=1>Brazil, Apr-2020</td><td rowspan=1 colspan=1>17-Mar-2021</td></tr><tr><td rowspan=1 colspan=1>Eta</td><td rowspan=1 colspan=1>B.1.525</td><td rowspan=1 colspan=1>G/484K.V3</td><td rowspan=1 colspan=1>21D</td><td rowspan=1 colspan=1>Multiplecountries,Dec-2020</td><td rowspan=1 colspan=1>17-Mar-2021</td></tr><tr><td rowspan=1 colspan=1>Theta</td><td rowspan=1 colspan=1>P.3</td><td rowspan=1 colspan=1>GR/1092K.V1</td><td rowspan=1 colspan=1>21E</td><td rowspan=1 colspan=1>Philippines,Jan-2021</td><td rowspan=1 colspan=1>24-Mar-2021</td></tr><tr><td rowspan=1 colspan=1>Iota</td><td rowspan=1 colspan=1>B.1.526</td><td rowspan=1 colspan=1>GH/253G.V1</td><td rowspan=1 colspan=1>21F</td><td rowspan=1 colspan=1>USA, Nov-2020</td><td rowspan=1 colspan=1>24-Mar-2021</td></tr><tr><td rowspan=1 colspan=1>Kappa</td><td rowspan=1 colspan=1>B.1.617.1</td><td rowspan=1 colspan=1>G/452R.V3</td><td rowspan=1 colspan=1>21B</td><td rowspan=1 colspan=1>India, Oct-2020</td><td rowspan=1 colspan=1>4-Apr-2021</td></tr><tr><td rowspan=1 colspan=1>Lambda</td><td rowspan=1 colspan=1>C.37</td><td rowspan=1 colspan=1>GR/452Q.V1</td><td rowspan=1 colspan=1>20D</td><td rowspan=1 colspan=1>Peru, Aug-2020</td><td rowspan=1 colspan=1>14-Jun-21</td></tr></table>

In addition, BioFire Defense indicated that one mutation passes the $5 \%$ frequency threshold of representation in the sequence database that may signify increased public health risk with the variant However, due to the position on the primer , the mutation is not expected to affect assay performance nor overall device performance. It appears that the mutation is mostly associated with the “Alpha” Variant of Concern (i.e., B.1.1.7).

Overall, the firm indicated that through the in silico analysis, among the 497,598 sequences analyzed (as of August 2021), there appears to be no examples of variant lineages that the BioFire COVID-19 Test 2 device will not be detected either by a single assay or overall test result.

# Inclusivity Empirical Analysis

The inclusivity in detection of SARS-CoV-2 variants by the BioFire COVID-19 Test 2 device was further evaluated through empirical evaluation of particular strains. Five variants of infectious SARS-CoV-2 were spiked in pooled NPS at concentrations near LoD. One of the variants evaluated was the SARS-CoV-2 USA-WA1/2020 reference strain. The other

four variants were based on geographic location. For each SARS-CoV-2 variant, the detection rate and the melt temperature Tm of the nucleic acid amplification products were analyzed. The results of this testing are summarized as follows—

Table 14. BioFire COVID-19 Test 2 Analytical Reactivity (Inclusivity) - Infectious SARSCoV-2 Material   

<table><tr><td rowspan=2 colspan=1>SARS-CoV-2 Variant(Source / ID)</td><td rowspan=2 colspan=1>SARS-CoV-2DetectionRate</td><td rowspan=1 colspan=3>Positivity RateSARS-CoV-2Assay</td><td rowspan=1 colspan=2>ConcentrationDetected</td></tr><tr><td rowspan=1 colspan=1>a</td><td rowspan=1 colspan=1>d</td><td rowspan=1 colspan=1>e</td><td rowspan=1 colspan=1>(GC/mL) a</td><td rowspan=1 colspan=1>PFU/mL</td></tr><tr><td rowspan=1 colspan=1>USA-WA1/2020 b(BEI /NR-52281)</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>1/3</td><td rowspan=1 colspan=1>9.9E+02</td><td rowspan=1 colspan=1>1.3E-02</td></tr><tr><td rowspan=1 colspan=1>Chile/Santiago__op4d1/2020(BEI / NR-52439)</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>2/3</td><td rowspan=1 colspan=1>1/3</td><td rowspan=1 colspan=1>9.9E+02</td><td rowspan=1 colspan=1>6.3E-02</td></tr><tr><td rowspan=1 colspan=1>HongKong/VM20001061/2020(BEI / NR-52282)</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>9.9E+02</td><td rowspan=1 colspan=1>1.5E-02</td></tr><tr><td rowspan=1 colspan=1>Italy-INMI1(BEI /NR-52284)</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>1/3</td><td rowspan=1 colspan=1>9.9E+02</td><td rowspan=1 colspan=1>7.6E-02</td></tr><tr><td rowspan=1 colspan=1>New York-PV08410/2020(BEI /NR-53514)</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>2/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>2/3</td><td rowspan=1 colspan=1>3.3E+03</td><td rowspan=1 colspan=1>2.4E-02</td></tr></table>

a RNA of infectious SARS-CoV-2 stocks was extracted using the Zymo Quick-RNA Viral Kit and quantified using the CDC 2019-Novel Coronavirus (2019-nCoV) RealTime RT-PCR Diagnostic Panel (N1 target).

$^ { \mathrm { c } } 9 . 9 \mathrm { E } { + } 0 2 ~ \mathrm { G C / m L }$ is 3-fold greater than the LoD determined for infectious material (Table 11). Unexpected dropouts of the SARS-CoV-2e assay and a review of the amplification data suggested that spike levels (for all variants evaluated in this study) were lower than $3 { \times } \mathrm { L o D }$ and closer to $1 \times \mathrm { L o D }$ .

The inclusivity evaluation started at $9 . 9 \mathrm { E } { + 0 2 }$ GC/mL (3x LoD) of the SARS-CoV-2 USAWA1/2020 reference variant to verify reliable detection at the concentration. The New YorkPV08410/2020 variant had three SARS-CoV-2 Detected results for all three replicates tested at a concentration that was 3-fold above the concentration assessed for the other variants including USA-WA1/2020 reference strain. Additional assessments of the observed detection results as compared to those observed with the reference strain during LoD confirmation studies suggested that the actual concentration evaluated of the various variants may have been much less than 1x LoD. The amplicon Tm values for each variant fell within the Tm range of each SARS-CoV-2 assay.

Overall, all four SARS-CoV-2 variants that were empirically tested were detected at levels within 3-fold of the USA-WA1/2020 reference variant and no sensitivity limitations appeared to be observed.

8. Assay Cut-Off:

The BioFire COVID-19 Test 2 is part of the FilmArray system. The FilmArray system is designed to interpret the test data and automatically report the test results to the operator. The FilmArray system uses the results of the melt detector to determine each test result. The melt detector analyzes the fluorescence signal for each DNA melt curve using a series of mathematical tests or filters. These tests evaluate the melt curves and their derivatives and determine the presence or absence of specific PCR products from background noise or other non-specific products. Each test uses specific cutoff thresholds (established during tuning) to determine if a curve is positive or negatives. Once a curve is assigned a results (positive or negative) no further tests are performed, otherwise analysis continues until a determination can be made by the software. To maximize the sensitivity and specific of the melt detector, the algorithm was tuned against a large data set comprising typical and atypical melt curves (i.e., training data set) with expert annotation (positive or negative) to each melt curve during the development of the device. Using an automated process, the melt detector algorithm was then tuned so that software calls most closely matched those from expert users.

The BioFire FilmArray melt detector was tuned using data from runs performed both at the firm and at external sites. The melt detector can be tuned to use different thresholds (or cutoffs) for each test panel in order to achieve optimal sensitivity and specificity for each panel. The amplicons generated by the assays contained in BioFire COVID-19 Test 2 are expected to melt at specific temperatures (Tm). The range of expected Tm values for any amplicon is affected by system variation and by sequence diversity (e.g., different isolates or strains of specific target organisms). While effort was made to minimize variation in Tm values with the assay design of the BioFire COVID-19 Test 2 device, some sequence diversity is expected and thus there are assay specific “melt ranges” to account for potential variations. For each assay in the device, the Tm value for the PCR2 amplicon is empirically determined and set as the mean Tm. The appropriate range appropriate range is then determined by evaluating known sequence diversity using a combination of mathematical modeling and empiric testing. Data collected when evaluating well-characterized samples during development including known isolates and clinical samples further established melt ranges. At the conclusion of analytical and clinical studies, a final validation of melt ranges is performed to ensure optimal sensitivity and specificity for each assay on the device.

Overall, by using an adjusted model fit to account for empirical observations and primary reference data, $9 9 . 7 6 \%$ of the observed Tm values for assays were within 1 degree of the predicted Tm values. Thus, the melt ranges were optimized and incorporated into the BioFire COVID-19 Test 2 pouch module.

# 9. Carry-Over:

A formal carry-over study in support of this regulatory submission for the BioFire COVID19 Test 2 was not performed, since carry-over studies with high positive samples followed by negative samples have been performed for other FDA-cleared FilmArray Panels (i.e., FilmArray RP, BCID, and GI) for both the FilmArray 2.0 and the FilmArray Torch systems. No detectable carry-over has been observed.

# B Comparison Studies:

1. Method Comparison with Predicate Device:

Not applicable. Refer to the Clinical Studies Section of this document.

2. Matrix Comparison:

Not applicable.

# C Clinical Studies:

# Prospective Study

The prospectively collected samples were nasopharyngeal swabs (NPS) in transport medium, collected from subjects, that were leftover following standard of care (SoC) testing for SARS-CoV-2. Three study sites enrolled specimens for the clinical study. The inclusion and exclusion criteria were:

# Inclusion criteria:

Specimen is residual NPS in transport medium leftover from standard of care testing   
for SARS-CoV-2 by an assay that includes an extraction step and that has received an   
EUA designation   
Specimen has been held at room temperature for less than or equal to 4 hours or $4 \ ^ { \circ } \mathrm { C }$   
for less than or equal to three days before enrollment (testing must be completed   
within this window)   
At least $1 . 3 ~ \mathrm { m L }$ of specimen is remaining after standard of care testing and is   
available for use in the study

# Exclusion criteria:

Specimen other than NPS in transport medium (e.g., nasopharyngeal aspirate, anterior or mid-turbinate swab, oropharyngeal swab, NPS eluted in saline) Specimen cannot be tested within the defined storage parameters Insufficient specimen volume

Once specimens met the eligibility criteria, a study code number was assigned to de-identify aliquots, and sample identities were blinded from test operators and personnel analyzing results. Laboratory personnel performed testing on the FDA cleared comparator method (CM) and BioFire COVID-19 Test 2 according to the respective Instructions for Use. Runs that produced invalid results were retested if there was sufficient specimen volume.

For this study, the BioFire COVID-19 Test 2 was evaluated against the comparator. The two devices have different targeted regions and number of targets and result calling algorithm for their respective assays to detect the SARS-CoV-2 analyte. Additional EUA tests were used for discrepancy analysis with the BioFire COVID-19 Test 2 device. In this study, a specimen having agreement between the BioFire COVID-19 Test 2 and the comparator are either both positive (True Positive) or both negative (True Negative, TN). The exact binomial two-sided $9 5 \%$ confidence interval was calculated for both PPA and NPA. Both the FilmArray 2.0 and the FilmArray Torch instruments were used in the study.

Overall, from the three study sites, a total of 534 specimens were enrolled. Eleven (11/534) were excluded as indicated below. Seven (7/11) specimens failed the inclusion criteria

including one specimen where the BioFire COVID-19 Test 2 testing was never attempted due to insufficient specimen volume for retesting after a CM invalid result. Four specimens had device failures with the BioFire COVID-19 Test 2.

Table 15. Summary of Specimens Excluded from Study   

<table><tr><td rowspan=1 colspan=1>Reason for Exclusion</td><td rowspan=1 colspan=1>Number ofAffected SCNs</td></tr><tr><td rowspan=1 colspan=1>Specimen did not meet inclusion criteria (storage conditions orvolume</td><td rowspan=1 colspan=1>6</td></tr><tr><td rowspan=1 colspan=1>Invalid comparator result and insufficient specimen volume forretest</td><td rowspan=1 colspan=1>11</td></tr><tr><td rowspan=1 colspan=1>BioFire COVID-19 Test 2 System Failure (pouch hydration,run aborted, or pouch internal control failure) and insufficientspecimen volume for retest</td><td rowspan=1 colspan=1>42</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>11</td></tr></table>

1 Specimen was negative by SoC test; testing was not attempted on BioFire COVID19 Test 2 nor at central reference laboratory. 2 Specimens were negative by SoC, CM, and central reference laboratory test.

Thus for the 527 specimens that met the inclusion criteria and for which pouch loading was attempted, 524 pouches hydrated and drew sample properly on the first attempt. For the three failures, hydration and sample loading was reattempted where two were successful and one reattempt was not successful. In total this translated into a $9 9 . 2 \%$ success rate in pouch function. As for instrument performance, for the 526 specimens that were successfully loaded into pouches, there were three system failures—two incomplete runs and one control failure for a success rate of $9 9 . 4 \%$ (523/526).

The study population were comprised of the following demographics—

Table 16. Demographics of study participants (Overall and Per Site)   

<table><tr><td rowspan=1 colspan=3>Overall</td><td rowspan=1 colspan=1>Site 1</td><td rowspan=1 colspan=1>Site 2</td><td rowspan=1 colspan=1>Site 3</td></tr><tr><td rowspan=3 colspan=1>S</td><td rowspan=1 colspan=1>Female</td><td rowspan=1 colspan=1>253(48.4%)</td><td rowspan=1 colspan=1>141(45.5%)</td><td rowspan=1 colspan=1>54(50.0%)</td><td rowspan=1 colspan=1>58(55.2%)</td></tr><tr><td rowspan=1 colspan=1>Male</td><td rowspan=1 colspan=1>267(51.1%)</td><td rowspan=1 colspan=1>169(54.5%)</td><td rowspan=1 colspan=1>51(47.2%)</td><td rowspan=1 colspan=1>47(44.8%)</td></tr><tr><td rowspan=1 colspan=1>Unknown</td><td rowspan=1 colspan=1>3 (0.6%)</td><td rowspan=1 colspan=1>0 (0%)</td><td rowspan=1 colspan=1>3 (2.8%)</td><td rowspan=1 colspan=1>0 (0%)</td></tr><tr><td rowspan=4 colspan=1>Aan </td><td rowspan=1 colspan=1>0-18 years</td><td rowspan=1 colspan=1>55(10.5%)</td><td rowspan=1 colspan=1>24 (7.7%)</td><td rowspan=1 colspan=1>18(16.7%)</td><td rowspan=1 colspan=1>13(12.4%)</td></tr><tr><td rowspan=1 colspan=1>19-40years</td><td rowspan=1 colspan=1>170(32.5%)</td><td rowspan=1 colspan=1>102(32.9%)</td><td rowspan=1 colspan=1>45(41.7%)</td><td rowspan=1 colspan=1>23(21.9%)</td></tr><tr><td rowspan=1 colspan=1>41-60years</td><td rowspan=1 colspan=1>146(27.9%)</td><td rowspan=1 colspan=1>94 (30.3%)</td><td rowspan=1 colspan=1>32(29.6%)</td><td rowspan=1 colspan=1>20(19.0%)</td></tr><tr><td rowspan=1 colspan=1>61+ years</td><td rowspan=1 colspan=1>152(29.1%)</td><td rowspan=1 colspan=1>90 (29.0%)</td><td rowspan=1 colspan=1>13(12.0%)</td><td rowspan=1 colspan=1>49(46.7%)</td></tr><tr><td rowspan=1 colspan=2>Total</td><td rowspan=1 colspan=1>523</td><td rowspan=1 colspan=1>310</td><td rowspan=1 colspan=1>108</td><td rowspan=1 colspan=1>105</td></tr></table>

Overall, 523 specimens produced valid results and were used in assessing the BioFire COVID-19 Test 2 clinical performance. The performance of the BioFire COVID-19 Test 2 in comparison to the CM device is as follows—

Table 17. Comparator Performance for the BioFire COVID-19 Test 2 Device   

<table><tr><td rowspan="7">BioFire Pos COVID-19 Test 2</td><td rowspan="3"></td><td colspan="2">Comparator</td><td colspan="2">PPA/NPA Panel</td></tr><tr><td>Pos</td><td>Neg</td><td>Performance</td><td>95%CI</td></tr><tr><td>68</td><td>2</td><td>68/69 (98.6%)</td><td>92.2-99.7%</td></tr><tr><td>Neg</td><td>1</td><td>452</td><td>452/454 (99.6%)</td><td>98.4-99.9%</td></tr><tr><td>Total</td><td>69</td><td>454</td><td></td><td></td></tr></table>

# D Clinical Cut-Off:

Not applicable

# E Expected Values/Reference Range:

The expected value of BioFire COVID-19 Test 2 results stratified by site are presented in Table . The expected value of BioFire COVID-19 Test 2 results stratified by age group are presented in Table . SARS-CoV-2 was Detected in $1 3 . 4 \%$ (70/523) specimens. Prevalence across sites ranged between $7 . 6 \%$ (Site 3) and $1 6 . 1 \%$ (Site 1). The age group with the highest prevalence was 19-40 $( 1 6 . 5 \% )$ , followed by 41-60 $( 1 5 . 8 \% )$ .

Table 20. The Expected Value of SARS-CoV-2 as Determined by BioFire COVID-19 Test 2, Stratified by Site; $\# =$ Number; $\mathbf { E V } =$ Expected Value   

<table><tr><td rowspan=2 colspan=1>SARS-CoV-2</td><td rowspan=1 colspan=2>Overall(n=523)</td><td rowspan=1 colspan=2>1 - TGH (n=310)</td><td rowspan=1 colspan=2>2 - NHL (n=108)</td><td rowspan=1 colspan=2>3 - LUMC(n=105)</td></tr><tr><td rowspan=1 colspan=1>#</td><td rowspan=1 colspan=1>EV</td><td rowspan=1 colspan=1>#</td><td rowspan=1 colspan=1>EV</td><td rowspan=1 colspan=1>#</td><td rowspan=1 colspan=1>EV</td><td rowspan=1 colspan=1>#</td><td rowspan=1 colspan=1>EV</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>70</td><td rowspan=1 colspan=1>13.4%</td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>16.1%</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>11.1%</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>7.6%</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>453</td><td rowspan=1 colspan=1>86.6%</td><td rowspan=1 colspan=1>260</td><td rowspan=1 colspan=1>83.9%</td><td rowspan=1 colspan=1>96</td><td rowspan=1 colspan=1>88.9%</td><td rowspan=1 colspan=1>97</td><td rowspan=1 colspan=1>92.4%</td></tr></table>

Table 21. The Expected Value of SARS-CoV-2 as Determined by BioFire COVID-19 Test 2, Stratified by Age Group; $\# =$ Number; $\mathbf { E V } =$ Expected Value   

<table><tr><td colspan="1" rowspan="2">Sex</td><td colspan="2" rowspan="1">Overall(n=523)</td><td colspan="2" rowspan="1">age 0-18(n=55)</td><td colspan="2" rowspan="1">age 19-40(n=170)</td><td colspan="2" rowspan="1">age 41-60(n=146)</td><td colspan="2" rowspan="1">age 61+(n=152)</td></tr><tr><td colspan="1" rowspan="1">#</td><td colspan="1" rowspan="1">EV</td><td colspan="1" rowspan="1">#</td><td colspan="1" rowspan="1">EV</td><td colspan="1" rowspan="1">#</td><td colspan="1" rowspan="1">EV</td><td colspan="1" rowspan="1">#</td><td colspan="1" rowspan="1">EV</td><td colspan="1" rowspan="1">#</td><td colspan="1" rowspan="1">EV</td></tr><tr><td colspan="1" rowspan="1">Male(n=267)</td><td colspan="1" rowspan="1">32</td><td colspan="1" rowspan="1">12.0%</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">6.1%</td><td colspan="1" rowspan="1">8</td><td colspan="1" rowspan="1">8.3%</td><td colspan="1" rowspan="1">13</td><td colspan="1" rowspan="1">19.7%</td><td colspan="1" rowspan="1">9</td><td colspan="1" rowspan="1">12.5%</td></tr><tr><td colspan="1" rowspan="1">Female(n=253)</td><td colspan="1" rowspan="1">38</td><td colspan="1" rowspan="1">15.0%</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">13.6%</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">27.4%</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">12.7%</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">6.3%</td></tr><tr><td colspan="1" rowspan="1">Unknown(n=3)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0%</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0%</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0%</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0%</td></tr><tr><td colspan="1" rowspan="1">Overall(n=523)</td><td colspan="1" rowspan="1">70</td><td colspan="1" rowspan="1">13.4%</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">9.1%</td><td colspan="1" rowspan="1">28</td><td colspan="1" rowspan="1">16.5%</td><td colspan="1" rowspan="1">23</td><td colspan="1" rowspan="1">15.8%</td><td colspan="1" rowspan="1">14</td><td colspan="1" rowspan="1">9.2%</td></tr></table>

# F Other Supportive Instrument Performance Characteristics Data:

Not applicable

# VIII Proposed Labeling:

The labeling supports the finding of substantial equivalence for this device.

# IX Conclusion:

The submitted information in this premarket notification is complete and supports a substantial equivalence decision.