# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATIONDECISION SUMMARYASSAY AND INSTRUMENT

I Background Information:

A 510(k) Number K212147   
B Applicant DiaSorin Molecular LLC   
C Proprietary and Established Names Simplexa COVID-19 Direct   
D Regulatory Information

<table><tr><td>Product Code(s)</td><td>Classification</td><td>Regulation Section</td><td>Panel</td></tr><tr><td>QQX</td><td>Class II</td><td>21 CFR 866.3981 - Device to detect and identify nucleic acid targets in respiratory specimens from microbial agents that cause the SARS-CoV-2 respiratory infection and other microbial agents when in a multi-target test</td><td>MI - Microbiology</td></tr></table>

II Submission/Device Overview:

A Purpose for Submission: New device   
B Measurand: SARS-CoV-2 nucleic acids   
C Type of Test:

Simplexa COVID-19 Direct is a real-time RT-PCR test for use with LIAISON MDX instrument for the qualitative in vitro detection and identification of nucleic acids from severe acute

respiratory syndrome coronavirus 2 (SARS-CoV-2) in nasopharyngeal swabs (NPS) and nasal (NS) swabs from individuals with symptoms of upper respiratory tract infection suspected of COVID-19.

# III Intended Use/Indications for Use:

A Intended Use(s):

See Indications for Use below.

# B Indication(s) for Use:

The DiaSorin Molecular Simplexa COVID-19 Direct is a real-time RT-PCR assay intended for use on the LIAISON MDX instrument for the in vitro qualitative detection of nucleic acid from severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) in nasopharyngeal swabs (NPS) and nasal swabs (NS)] from symptomatic individuals suspected of COVID 19 by their healthcare provider. The Simplexa COVID-19 Direct assay is an aid in the diagnosis of SARS-CoV-2 infection.

Positive results are indicative of the presence of SARS-CoV-2 RNA. Clinical correlation with patient history and other diagnostic information is necessary to determine patient infection status. Positive results do not rule out co-infection with other pathogens.

Negative results do not preclude SARS-CoV-2 infection and should not be used as the sole basi for patient management decisions.

Results are meant to be used in conjunction with other clinical, epidemiologic, and laboratory data, in accordance with the guidelines provided by the relevant public health authorities.

C Special Conditions for Use Statement(s):

Rx - For Prescription Use Only

# D Special Instrument Requirements:

LIAISON MDX Instrument with LIAISON MDX Studio Software.

# IV Device/System Characteristics:

# A Device Description:

Simplexa COVID-19 Direct is a real-time RT-PCR (rRT-PCR) system that enables the direct amplification and detection of SARS-CoV-2 (COVID-19) RNA from nasopharyngeal swab or nasal swab specimens that have not undergone nucleic acid extraction.

The test system consists of the following:

Simplexa COVID-19 Direct reaction mix, (DNA polymerase, reverse transcriptase, RNase inhibitor, buffer, dNTPs, encapsulated RNA template (Internal Control),

fluorescent probes and corresponding forward and reverse primers specific for detection of SARS-CoV-2 RNA and for the RNA Internal Control.)

Direct Amplification Disc (DAD), which has eight (8) separate wedges where up to eight specimens (or controls) may be processed on each disc. Each wedge contains sample and reagent input wells, microfluidic channels and laser activated valves to control the fluid flow, and a reaction chamber.

• Control Pack - Simplexa COVID-19 Positive Control Gen II Pack LIAISON MDX instrument, a real-time Polymerase Chain Reaction (PCR) thermocycler used for the identification of nucleic acids from biological specimens. The LIAISON MDX instrument is controlled by an external computer running the LIAISON MDX Studio Software and uses real-time fluorescence detection to identify targets within the sample wells of a Direct Amplification Disc (DAD).

o The LIAISON MDX, previously referred to as 3M Integrated Cycler, was cleared under K102314.

# B Principle of Operation:

The assay uses forward and reverse primers and associated fluorescent probe(s) included in the reaction mix to amplify SARS-CoV-2 cDNA reverse transcribed from RNA. The primers and probe sets are designed to detect SARS-CoV-2 ORF1ab and S gene from the viral RNA in nasopharyngeal swab or nasal swab. An RNA internal control, with associated primers and a fluorescent probe, is included in the reaction mix to detect RT-PCR failure and/or inhibition.

To start processing a patient sample, the user adds $5 0 ~ \mu \mathrm { L }$ of Reaction Mix to the reagent input well (R) on the DAD using a pipette, followed by $5 0 ~ \mu \mathrm { L }$ of unextracted specimen to the sample input well (SAMPLE). A DAD with reagents and sample(s) is subsequently loaded on the LIAISON MDX. Centrifugal force aboard the LIAISON MDX moves the fluid into the metering chamber of the DAD. The reagent chamber is specifically designed to measure $4 0 ~ \mu \mathrm { L }$ of reagent and the sample chamber is specifically designed to measure $1 0 ~ \mu \mathrm { L }$ of sample. Excess reagent and sample are forced into the waste chambers by centrifugal force.

After the centrifugal force has mixed the sample and reagent within the reaction chamber, coat protein denaturation, reverse transcription and PCR amplification cycles begin. In the process, the probe anneals to a specific target sequence located between the forward and reverse primers. During the extension phase of the PCR cycle, the $5 '$ nuclease activity of DNA polymerase degrades the probe, causing the reporter dye to separate from the quencher dye, generating a fluorescent signal. The S gene probe has a FAM fluorophore and the ORF1ab gene probe has JOE fluorophore. Fluorescence intensity is monitored at each PCR cycle by detection modules in the LIAISON MDX. A sample is considered positive for a particular target if intensity of the optical reading crosses a particular threshold before a predetermined cut-off cycle. Results are obtained by using the LIAISON MDX Studio software version 1.1 or above. According to the result interpretation if at least one target (S gene or ORF1ab) is Detected for the sample then the sample is positive for SARS-CoV-2.

An RNA internal control, with associated primers and a fluorescent probe, is included in the reaction mix to detect RT-PCR failure and/or inhibition.

After the run is complete the results are displayed by the software as shown in the following table:

<table><tr><td rowspan=1 colspan=2>Results</td><td rowspan=3 colspan=1>Interpretation</td></tr><tr><td rowspan=1 colspan=2>SARS-Cov-2 Target</td></tr><tr><td rowspan=1 colspan=1>ORF1ab gene</td><td rowspan=1 colspan=1>S gene</td></tr><tr><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>Result indicates the presence of SARS-CoV-2 RNA in the patient sample.</td></tr><tr><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>----</td><td rowspan=1 colspan=1>Result indicates the presence of SARS-CoV-2 RNA in the patient sample.</td></tr><tr><td rowspan=1 colspan=1>----</td><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>Result indicates the presence of SARS-CoV-2 RNA in the patient sample.</td></tr><tr><td rowspan=1 colspan=1>Not Detected</td><td rowspan=1 colspan=1>Not Detected</td><td rowspan=1 colspan=1>Result indicates the absence of SARS-CoV-2 RNA in the patient sample.</td></tr><tr><td rowspan=1 colspan=2>Invalid</td><td rowspan=1 colspan=1>Result indicates inability to conclusively determine presence or absence ofSARS-CoV- 2 RNA in the patient sample. This result may be due to 1)Internal Control (IC) failure, or 2) failure to detect sufficient specimenvolume. The sample needs to be retested. If the problem persists, contactTechnical Service.</td></tr><tr><td rowspan=1 colspan=2>Results</td><td rowspan=1 colspan=1>Interpretation</td></tr><tr><td rowspan=1 colspan=2>EC500</td><td rowspan=1 colspan=1>Data processing error due to noise, weak or late amplification in the signalRepeat the sample. If the problem persists, contact Technical Service.</td></tr><tr><td rowspan=1 colspan=2>EC505</td><td rowspan=1 colspan=1>Insufficient information to determine whether amplification was present.If the problem persists, contact Technical Service.</td></tr><tr><td rowspan=1 colspan=2>EC515</td><td rowspan=1 colspan=1>Internal Control Amplification is not within specification. Result isinvalid, repeat the sample. If the problem persists, contact TechnicalService.</td></tr></table>

# C Instrument Description Information:

1. Instrument Name: LIAISON MDX instrument with LIAISON MDX Studio Software (version 1.1 or above)

2. Specimen Identification: Specimen identification can be entered via barcode insert containing the assay definition.

3. Specimen Sampling and Handling:

Simplexa COVID-19 Direct is a no extraction RT-PCR assay. In this assay the operator adds $5 0 \mathrm { u l }$ of the reaction mix to the direct amplification disc (DAD) and adds another $5 0 \mathrm { u l }$ of unextracted sample in designated wells. The disc is then loaded with the sample and necessary controls are run on the LIAISON MDX instrument.

4. Calibration:

No calibration is required.

5. Quality Control: External controls are provided separately.

# V Substantial Equivalence Information:

A Predicate Device Name(s): BioFire COVID-19 Test 2 B Predicate 510(k) Number(s): K211079 C Comparison with Predicate(s):

<table><tr><td>Device &amp; Predicate Device(s):</td><td>K212147</td><td>K211079</td></tr><tr><td>Device Trade Name</td><td>Simplexa COVID-19 Direct</td><td>BioFire COVID-19 Test 2</td></tr><tr><td>General Device Characteristic Similarities</td><td>The DiaSorin Molecular</td><td></td></tr><tr><td>Intended Use/Indications for Use</td><td>Simplexa COVID-19 Direct is a real-time RT-PCR assay intended for use on the LIAISON MDX instrument for the in vitro qualitative detection of nucleic acid from severe acute respiratory syndrome coronavirus 2 (SARS- CoV-2) in nasopharyngeal swabs (NPS) and nasal swabs (NS) from symptomatic individuals suspected of COVID 19 by their healthcare provider. The Simplexa COVID- 19 Direct assay is an aid in the diagnosis of</td><td>The BioFire COVID-19 Test 2 is a qualitative nested multiplexed RT- PCR in vitro diagnostic test intended for use with the BioFire FilmArray 2.0 and BioFire FilmArray Torch Systems. The BioFire COVID-19 Test 2 detects nucleic acids from severe acute respiratory syndrome coronavirus 2 (SARS-CoV- 2) in nasopharyngeal swabs (NPS) from symptomatic individuals suspected of COVID-19 by their healthcare provider. Results are for the identification of SARS- CoV-2 RNA. The SARS- CoV-2 RNA is generally detectable in NPS</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">presence of SARS-CoV-2 RNA. Clinicalcorrelation with patienthistory and otherdiagnostic informationis necessary todetermine patientinfection status.Positive results do notrule out co-infectionwith other pathogens.Negative results do notpreclude SARS-CoV-2infection and should notbe used as the sole basisfor patient managementdecisions.Results are meant to beused in conjunctionwith other clinical,epidemiologic, andlaboratory data, inaccordance with theguidelines provided bythe relevant publichealth authorities.</td><td colspan="1" rowspan="1">Positive results areindicative of the presenceof SARS-CoV-2 RNA;clinical correlation withpatient history and otherdiagnostic information isnecessary to determinepatient infection status.Positive results do not ruleout co-infection with otherpathogens.Results are meant to beused in conjunction withother clinical,epidemiologic, andlaboratory data, inaccordance with theguidelines provided by therelevant public healthauthorities.The BioFire COVID-19Test 2 is intended for useby trained medical andlaboratory professionals ina laboratory setting orunder the supervision of atrained laboratoryprofessional</td></tr><tr><td colspan="1" rowspan="1">Assay principle</td><td colspan="1" rowspan="1">PCR-based system fordetecting the presenceor absence of viral RNAin clinical specimens.</td><td colspan="1" rowspan="1">same</td></tr><tr><td colspan="1" rowspan="1">Analyte</td><td colspan="1" rowspan="1">RNA</td><td colspan="1" rowspan="1">same</td></tr><tr><td colspan="1" rowspan="1">Test interpretation</td><td colspan="1" rowspan="1">Automated testinterpretation</td><td colspan="1" rowspan="1">same</td></tr><tr><td colspan="1" rowspan="1">Controls</td><td colspan="1" rowspan="1">Assay contains aninternal control for PCRfunction.</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Organism Detected</td><td colspan="1" rowspan="1">SARS-CoV-2</td><td colspan="1" rowspan="1">same</td></tr><tr><td colspan="1" rowspan="1">General DeviceCharacteristic Differences</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Instrumentation</td><td colspan="1" rowspan="1">LIAISON MDX</td><td colspan="1" rowspan="1">FilmArray2.0 or FilmArray Torch</td></tr><tr><td colspan="1" rowspan="1">Specimen type</td><td colspan="1" rowspan="1">Nasopharyngeal andNasal swabs collected</td><td colspan="1" rowspan="1">Nasopharyngeal swabscollected in transport media</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">in transport media(Copan UTM, BDVTM, or 0.9% saline)</td><td colspan="1" rowspan="1">(UTM, VTM or saline)</td></tr><tr><td colspan="1" rowspan="1">Amplification technology</td><td colspan="1" rowspan="1">Real-Time PCRfollowed by analysis ofthe result by software.</td><td colspan="1" rowspan="1">Nested-multiplex RT-PCRfollowed by high resolutionmelting analysis to confirmidentity of amplifiednucleic acids.</td></tr></table>

# VI Standards/Guidance Documents Referenced:

Class II Special Controls as per 21 CFR 866.3981.

# VII Performance Characteristics:

# A Analytical Performance:

1. Precision/Reproducibility:

Reproducibility of the Simplexa COVID-19 Direct

The reproducibility study was conducted at three intended use sites (two external and one internal /in-house), over a period of five testing days, using one lot of reagents, with testing performed by two operators at each site. The test panel consisted of five panel members: one low positive (1-2x LoD), one medium positive $\left( 3 { - } 5 \mathbf { x } \ L \mathbf { o } \mathbf { D } \right)$ , contrived with COVID-19 strain USA-WA1/2020, a negative sample (UTM as NTC), and one positive Control Sample prepared in clinical (NPS) matrix. Each sample was tested in three replicates once per day by each operator, for a total of 90 measurements per sample (3 sites $_ { \textrm { X 2 } }$ users x 5 days x 3 reps $_ { \mathrm { \scriptsize = } 9 0 }$ measurements). A total of five LIAISON MDX instruments (at least one per site) were used in the study. The table below summarizes the results from the reproducibility study, showing the mean CT values and calculated SD and $\% \mathrm { C V }$ for each panel member.

Reproducibility Study, Summary of Qualitative Results   

<table><tr><td rowspan=1 colspan=1>Sample PanelMember</td><td rowspan=1 colspan=1>Expected Qualitative Result</td><td rowspan=1 colspan=1>% Agreement withExpected Result(# detected/#tested)</td></tr><tr><td rowspan=1 colspan=1>Low positive sample(1-2x)</td><td rowspan=1 colspan=1>COVID-19 Detected</td><td rowspan=1 colspan=1>98.9% (89/90)95% CI: 94-100.0%</td></tr><tr><td rowspan=1 colspan=1>Moderate positivesample (3-5x)</td><td rowspan=1 colspan=1>COVID-19 Detected</td><td rowspan=1 colspan=1>100.0% (90/90)95% CI: 95.9-100.0%</td></tr><tr><td rowspan=1 colspan=1>Negative Control(UTM)</td><td rowspan=1 colspan=1>COVID-19 Not Detected</td><td rowspan=1 colspan=1>100.0% (90/90)*95% CI: 95.9-100.0%</td></tr><tr><td rowspan=1 colspan=1>Positive Control</td><td rowspan=1 colspan=1>COVID-19 Detected</td><td rowspan=1 colspan=1>100.0% (90/90)95% CI: 95.9-100.0%</td></tr></table>

The expected result for the Negative Panel Member is “Not Detected”

Reproducibility Study, Summary of Results   

<table><tr><td rowspan=2 colspan=1>Analyte</td><td rowspan=2 colspan=1>Sample PanelMMember</td><td rowspan=2 colspan=1>N</td><td rowspan=2 colspan=1>MeanCt</td><td rowspan=1 colspan=2>Repeatability</td><td rowspan=1 colspan=2>Between-Day</td><td rowspan=1 colspan=2>Between-Opperator</td><td rowspan=1 colspan=2>Between-Site</td><td rowspan=1 colspan=2>Total</td></tr><tr><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td></tr><tr><td rowspan=4 colspan=1>Sgngene AM)</td><td rowspan=1 colspan=1>NPS_Lowpositive</td><td rowspan=1 colspan=1>89a</td><td rowspan=1 colspan=1>32.2</td><td rowspan=1 colspan=1>0.53</td><td rowspan=1 colspan=1>1.7</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.40</td><td rowspan=1 colspan=1>1.2</td><td rowspan=1 colspan=1>1.87</td><td rowspan=1 colspan=1>5.8</td><td rowspan=1 colspan=1>1.99</td><td rowspan=1 colspan=1>6.2</td></tr><tr><td rowspan=1 colspan=1>NPS_Moderatepositive</td><td rowspan=1 colspan=1>90b</td><td rowspan=1 colspan=1>31.2</td><td rowspan=1 colspan=1>1.63</td><td rowspan=1 colspan=1>5.2</td><td rowspan=1 colspan=1>1.41</td><td rowspan=1 colspan=1>4.5</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.31</td><td rowspan=1 colspan=1>1.0</td><td rowspan=1 colspan=1>2.18</td><td rowspan=1 colspan=1>7.0</td></tr><tr><td rowspan=1 colspan=1>UTM</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>N/A</td></tr><tr><td rowspan=1 colspan=1>Positive Controlas is</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>25.7</td><td rowspan=1 colspan=1>0.27</td><td rowspan=1 colspan=1>1.1</td><td rowspan=1 colspan=1>0.14</td><td rowspan=1 colspan=1>0.6</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.31</td><td rowspan=1 colspan=1>1.2</td></tr><tr><td rowspan=4 colspan=1>ORFlab(JOE)</td><td rowspan=1 colspan=1>NPS_Lowpsitive</td><td rowspan=1 colspan=1>89c</td><td rowspan=1 colspan=1>32.5</td><td rowspan=1 colspan=1>0.50</td><td rowspan=1 colspan=1>1.5</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.34</td><td rowspan=1 colspan=1>1.0</td><td rowspan=1 colspan=1>1.71</td><td rowspan=1 colspan=1>5.3</td><td rowspan=1 colspan=1>1.81</td><td rowspan=1 colspan=1>5.6</td></tr><tr><td rowspan=1 colspan=1>NPS_Moderatepositive</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>31.2</td><td rowspan=1 colspan=1>1.01</td><td rowspan=1 colspan=1>3.3</td><td rowspan=1 colspan=1>0.45</td><td rowspan=1 colspan=1>1.4</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.37</td><td rowspan=1 colspan=1>1.2</td><td rowspan=1 colspan=1>1.17</td><td rowspan=1 colspan=1>3.7</td></tr><tr><td rowspan=1 colspan=1>UTM</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>N/A</td></tr><tr><td rowspan=1 colspan=1>Positive Controlas is</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>26.0</td><td rowspan=1 colspan=1>0.27</td><td rowspan=1 colspan=1>1.0</td><td rowspan=1 colspan=1>0.08</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>0.02</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>0.24</td><td rowspan=1 colspan=1>0.9</td><td rowspan=1 colspan=1>0.37</td><td rowspan=1 colspan=1>1.4</td></tr><tr><td rowspan=4 colspan=1>InternalTCControl(Q670)</td><td rowspan=1 colspan=1>NPS_Lowpositive</td><td rowspan=1 colspan=1>90d</td><td rowspan=1 colspan=1>32.0</td><td rowspan=1 colspan=1>1.16</td><td rowspan=1 colspan=1>3.6</td><td rowspan=1 colspan=1>0.88</td><td rowspan=1 colspan=1>2.7</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.82</td><td rowspan=1 colspan=1>2.6</td><td rowspan=1 colspan=1>1.67</td><td rowspan=1 colspan=1>5.2</td></tr><tr><td rowspan=1 colspan=1>NPS_Moderatepositive</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>31.8</td><td rowspan=1 colspan=1>0.92</td><td rowspan=1 colspan=1>2.9</td><td rowspan=1 colspan=1>0.60</td><td rowspan=1 colspan=1>1.9</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.61</td><td rowspan=1 colspan=1>1.9</td><td rowspan=1 colspan=1>1.26</td><td rowspan=1 colspan=1>3.9</td></tr><tr><td rowspan=1 colspan=1>UTM</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>32.0</td><td rowspan=1 colspan=1>0.79</td><td rowspan=1 colspan=1>2.5</td><td rowspan=1 colspan=1>0.41</td><td rowspan=1 colspan=1>1.3</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.75</td><td rowspan=1 colspan=1>2.4</td><td rowspan=1 colspan=1>1.17</td><td rowspan=1 colspan=1>3.6</td></tr><tr><td rowspan=1 colspan=1>Positive Controlas is</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>31.6</td><td rowspan=1 colspan=1>0.45</td><td rowspan=1 colspan=1>1.4</td><td rowspan=1 colspan=1>0.31</td><td rowspan=1 colspan=1>1.0</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>1.09</td><td rowspan=1 colspan=1>3.4</td><td rowspan=1 colspan=1>1.21</td><td rowspan=1 colspan=1>3.8</td></tr></table>

$\sp { \mathrm { ~ a ~ } } - 4 / 8 9$ were not detected for S gene but had an interpretation of positive, therefore assigned a Ct of 40 for calculation. One of the replicates was not detected and therefore not used to calculate variances.

$^ { \mathrm { b } } - 4 / 9 0$ were not detected for S gene but had an interpretation of positive, therefore assigned a Ct of 40 for calculation.

$^ \mathrm { c } - 3 / 8 9$ were not detected for ORF1ab gene but was positive, therefore assigned a Ct of 40 for calculation. One of the replicates was not detected and therefore not used to calculate variances.

$^ \mathrm { d } - 2 / 9 0$ were not detected for RNA IC but were positive for COVID-19 detection. A Ct for RNA IC is not applicable for positive samples and therefore was acceptable and assigned a Ct of 40 and used in the calculation of the variance components.

# Lot-to-lot Reproducibility of the Simplexa COVID-19 Direct Kit

The lot-to-lot reagent reproducibility of the Simplexa COVID-19 Direct kit was evaluated in a study conducted at one internal site, over a period of 12 non-consecutive days (18-23 and 25-30 January 2021), using three lots of Simplexa COVID-19 Direct kit, two lots of Simplexa COVID-19 Gen II Positive Control, with testing performed by two operators at the site. The test panel consisted of four panel members: one low positive (1-2x LoD) and one medium positive (3-5x LoD), each contrived with 2019-nCoV/USA-WA1/2020 strain, a negative sample (UTM as No Template Control), and one Positive Control sample (from Control Pack) prepared in negative clinical (NPS) matrix. Each sample was tested in duplicate per kit lot per run, in two runs per day, for a total of 12 non-consecutive days (2 replicates/kit lot x 3 kit lots/run x 2 runs/day x 12 days $= 1 4 4$ /sample panel member). A total of two LIAISON MDX instruments were used in the study. The tables below summarize the results from the inter-lot precision study for the Simplexa COVID-19 Direct kit, showing the mean Ct values and calculated SD and $\% C V$ for each panel member.

Qualitative Summary of -Lot-to-lot Reproducibility for Simplexa COVID-19 Direct   

<table><tr><td rowspan=1 colspan=1>Sample Panel Member</td><td rowspan=1 colspan=1>Observed Qualitative Result</td></tr><tr><td rowspan=1 colspan=1>Low positive sample (1-2x)</td><td rowspan=1 colspan=1>99.3% (143/144) COVID-19 Detected95% CI: (96.2% to 99.9%)</td></tr><tr><td rowspan=1 colspan=1>Moderate positive sample (3-5x)</td><td rowspan=1 colspan=1>100.0% (144/144) COVID-19 Detected95% CI: (97.5% to 100%)</td></tr><tr><td rowspan=1 colspan=1>Positive control</td><td rowspan=1 colspan=1>100.0% (144/144) COVID-19 Detected95% CI: (97.5% to 100%)</td></tr></table>

Lot-to-lot Reproducibility, Summary of Results   

<table><tr><td rowspan=2 colspan=1>Analyte</td><td rowspan=2 colspan=1>PanelMember</td><td rowspan=1 colspan=2>Kit Lot-X9470N</td><td rowspan=1 colspan=2>Kit Lot - 9586N</td><td rowspan=1 colspan=2>Kit Lot - 9589N</td><td rowspan=1 colspan=2>All Lots Combined</td></tr><tr><td rowspan=1 colspan=1>Qualitative Results</td><td rowspan=1 colspan=1>MeanCt ±SD(%CV)</td><td rowspan=1 colspan=1>Qualitative Results</td><td rowspan=1 colspan=1>MeanCt ±SD(%CV)</td><td rowspan=1 colspan=1>Qualitative Results</td><td rowspan=1 colspan=1>MeanCt ±SD(%CV)</td><td rowspan=1 colspan=1>Qualitative Results</td><td rowspan=1 colspan=1>MeanCt ±SD(%CV)</td></tr><tr><td rowspan=4 colspan=1>S gene(FAM)</td><td rowspan=1 colspan=1>LP</td><td rowspan=1 colspan=1>95.8% (46/48)</td><td rowspan=1 colspan=1>32.2 ±1.21(3.7%)</td><td rowspan=1 colspan=1>95.8%(46/48)</td><td rowspan=1 colspan=1>31.9 ±0.99(3.1%)</td><td rowspan=1 colspan=1>97.9%(47/48)</td><td rowspan=1 colspan=1>32.0 ±0.85(2.6%)</td><td rowspan=1 colspan=1>96.5%(139/144)</td><td rowspan=1 colspan=1>32.0 ±1.03(3.2%)</td></tr><tr><td rowspan=1 colspan=1>MP</td><td rowspan=1 colspan=1>97.9% (47/48)</td><td rowspan=1 colspan=1>30.9 ±0.78(2.5%)</td><td rowspan=1 colspan=1>100.0%(48/48)</td><td rowspan=1 colspan=1>30.8 ±0.47(1.5%)</td><td rowspan=1 colspan=1>100.0%(48/48)</td><td rowspan=1 colspan=1>30.8 ±0.78(2.5%)</td><td rowspan=1 colspan=1>99.3%(143/144)</td><td rowspan=1 colspan=1>30.8 ±0.69(2.2%)</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>0.0% (0/48)</td><td rowspan=1 colspan=1>N/A ±N/A(N/A%)</td><td rowspan=1 colspan=1>0.0%(0/48)</td><td rowspan=1 colspan=1>N/A ±N/A(N/A%)</td><td rowspan=1 colspan=1>0.0%(0/48)</td><td rowspan=1 colspan=1>N/A ±N/A(N/A%)</td><td rowspan=1 colspan=1>0.0%(0/144)</td><td rowspan=1 colspan=1>N/A ±N/A(N/A%)</td></tr><tr><td rowspan=1 colspan=1>PC</td><td rowspan=1 colspan=1>100.0%(48/48)</td><td rowspan=1 colspan=1>25.8 ±0.36(1.4%)</td><td rowspan=1 colspan=1>100.0%(48/48)</td><td rowspan=1 colspan=1>25.7 ±0.29(1.1%)</td><td rowspan=1 colspan=1>100.0%(48/48)</td><td rowspan=1 colspan=1>25.6 ±0.28(1.1%)</td><td rowspan=1 colspan=1>100.0%(144/144)</td><td rowspan=1 colspan=1>25.7 ±0.33(1.3%)</td></tr><tr><td rowspan=4 colspan=1>ORF1ab gene(JOE)</td><td rowspan=1 colspan=1>LP</td><td rowspan=1 colspan=1>97.9% (47/48)</td><td rowspan=1 colspan=1>31.9 ±1.35(4.2%)</td><td rowspan=1 colspan=1>97.9%(47/48)</td><td rowspan=1 colspan=1>31.5 ±0.80(2.5%)</td><td rowspan=1 colspan=1>100.0%(48/48)</td><td rowspan=1 colspan=1>31.8 ±0.76(2.4%)</td><td rowspan=1 colspan=1>98.6%(142/144)</td><td rowspan=1 colspan=1>31.7 ±1.01(3.2%)</td></tr><tr><td rowspan=1 colspan=1>MP</td><td rowspan=1 colspan=1>97.9% (47/48)</td><td rowspan=1 colspan=1>30.6 ±0.90(2.9%)</td><td rowspan=1 colspan=1>100.0%(48/48)</td><td rowspan=1 colspan=1>30.5 ±0.50(1.6%)</td><td rowspan=1 colspan=1>100.0%(48/48)</td><td rowspan=1 colspan=1>30.5 ±0.65(2.1%)</td><td rowspan=1 colspan=1>99.3%(143/144)</td><td rowspan=1 colspan=1>30.5 ±0.70(2.3%)</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>0.0% (0/48)</td><td rowspan=1 colspan=1>N/A ±N/A(N/A%)</td><td rowspan=1 colspan=1>0.0%(0/48)</td><td rowspan=1 colspan=1>N/A ±N/A(N/A%)</td><td rowspan=1 colspan=1>0.0%(0/48)</td><td rowspan=1 colspan=1>N/A ±N/A(N/A%)</td><td rowspan=1 colspan=1>0.0%(0/144)</td><td rowspan=1 colspan=1>N/A ±N/A(N/A%)</td></tr><tr><td rowspan=1 colspan=1>PC</td><td rowspan=1 colspan=1>100.0%(48/48)</td><td rowspan=1 colspan=1>26.0 ±0.52(2.0%)</td><td rowspan=1 colspan=1>100.0%(48/48)</td><td rowspan=1 colspan=1>25.9 ±0.38(1.5%)</td><td rowspan=1 colspan=1>100.0%(48/48)</td><td rowspan=1 colspan=1>25.9 ±0.37(1.4%)</td><td rowspan=1 colspan=1>100.0%(144/144)</td><td rowspan=1 colspan=1>26.0 ±0.43(1.7%)</td></tr><tr><td rowspan=4 colspan=1>InternalControl(Q670)</td><td rowspan=1 colspan=1>LP</td><td rowspan=1 colspan=1>97.9% (47/48)</td><td rowspan=1 colspan=1>31.5 ±0.63(2.0%)</td><td rowspan=1 colspan=1>100.0%(48/48)</td><td rowspan=1 colspan=1>30.9 ±0.58(1.9%)</td><td rowspan=1 colspan=1>100.0%(48/48)</td><td rowspan=1 colspan=1>31.0 ±0.39(1.3%)</td><td rowspan=1 colspan=1>99.3%(143/144)</td><td rowspan=1 colspan=1>31.1 ±0.60(1.9%)</td></tr><tr><td rowspan=1 colspan=1>MP</td><td rowspan=1 colspan=1>97.9% (47/48)</td><td rowspan=1 colspan=1>31.7 ±1.17(3.7%)</td><td rowspan=1 colspan=1>100.0%(48/48)</td><td rowspan=1 colspan=1>30.9 ±0.34(1.1%)</td><td rowspan=1 colspan=1>100.0%(48/48)</td><td rowspan=1 colspan=1>31.2 ±0.64(2.1%)</td><td rowspan=1 colspan=1>99.3%(143/144)</td><td rowspan=1 colspan=1>31.2 ±0.86(2.8%)</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>100.0%(48/48)</td><td rowspan=1 colspan=1>31.1 ±0.37(1.2%)</td><td rowspan=1 colspan=1>100.0%(48/48)</td><td rowspan=1 colspan=1>30.5 ±0.38(1.2%)</td><td rowspan=1 colspan=1>100.0%(48/48)</td><td rowspan=1 colspan=1>30.4 ±0.48(1.6%)</td><td rowspan=1 colspan=1>100.0%(144/144)</td><td rowspan=1 colspan=1>30.7 ±0.52(1.7%)</td></tr><tr><td rowspan=1 colspan=1>PC</td><td rowspan=1 colspan=1>100.0%(48/48)</td><td rowspan=1 colspan=1>31.2 ±0.51(1.6%)</td><td rowspan=1 colspan=1>100.0%(48/48)</td><td rowspan=1 colspan=1>30.6 ±0.34(1.1%)</td><td rowspan=1 colspan=1>100.0%(48/48)</td><td rowspan=1 colspan=1>30.5 ±0.37(1.2%)</td><td rowspan=1 colspan=1>100.0%(144/144)</td><td rowspan=1 colspan=1>30.7 ±0.50(1.6%)</td></tr></table>

$\mathrm { L P = }$ low positive; $\mathrm { M P = }$ moderate positive; PC=positive control

The precision of the Simplexa COVID-19 Direct Gen II Positive Control kit for the lot-to-lot reagent reproducibility were evaluated in a study conducted at one internal site, over a period of 12 non-consecutive days (18-23 and 25-30 January 2021) using one lot of Simplexa COVID-19 Direct kit, three lots of Simplexa COVID-19 Gen II Positive Control with testing performed by two operators at the site, using one LIAISON MDX instrument. Forty eight (48) replicates of each of three lots of the Simplexa COVID-19 Direct Gen II Positive Control were tested for both channels (S gene [FAM] and ORF1ab gene [JOE]) and were detected in each of the 144 total replicates.

The table below summarizes the results from the inter lot precision study (Simplexa COVID19 Direct Gen II Positive Control kit), showing the mean Ct values and calculated SD and $\% \mathrm { C V }$ for each panel member.

Qualitative Summary of Inter-lot Reproducibility of Simplexa COVID-19 Gen II Positive Control   

<table><tr><td rowspan=1 colspan=1>Sample PanelMember</td><td rowspan=1 colspan=1>Expected Qualitative Result</td><td rowspan=1 colspan=1>Observed Qualitative Result</td></tr><tr><td rowspan=1 colspan=1>Positive Control</td><td rowspan=1 colspan=1>100% of the tested replicateresults are COVID-19Positive</td><td rowspan=1 colspan=1>100.0% (144/144) COVID-19Positive95% CI: 97.5 to 100.0%</td></tr></table>

Lot-to-lot Reproducibility Test Results for Simplexa COVID-19 Gen II Positive Control   

<table><tr><td rowspan=2 colspan=1>Analyte</td><td rowspan=2 colspan=1>PanelMember</td><td rowspan=1 colspan=2>PC Lot - V9878N</td><td rowspan=1 colspan=2>PC Lot - V9881N</td><td rowspan=1 colspan=2>PC Lot - V9882N</td><td rowspan=1 colspan=2>All Lots Combined</td></tr><tr><td rowspan=1 colspan=1>QualitativeResults</td><td rowspan=1 colspan=1>MeanCt ±SD(%CV)</td><td rowspan=1 colspan=1>QualitativeResults</td><td rowspan=1 colspan=1>MeanCt ±SD(%CV)</td><td rowspan=1 colspan=1>QualitativeResults</td><td rowspan=1 colspan=1>MeanCt ±SD(%CV)</td><td rowspan=1 colspan=1>QualitativeResults</td><td rowspan=1 colspan=1>MeanCt ±SD(%CV)</td></tr><tr><td rowspan=1 colspan=1>S gene(FAM)</td><td rowspan=3 colspan=1>PositiveControl</td><td rowspan=1 colspan=1>100% (48/48)</td><td rowspan=1 colspan=1>30.3 ±0.61(2.0%)</td><td rowspan=1 colspan=1>100%(48/48)</td><td rowspan=1 colspan=1>29.2 ±0.67(2.3%)</td><td rowspan=1 colspan=1>100%(48/48)</td><td rowspan=1 colspan=1>27.8 ±0.52(1.9%)</td><td rowspan=1 colspan=1>100%(144/144)</td><td rowspan=1 colspan=1>29.1 ±1.19(4.1%)</td></tr><tr><td rowspan=1 colspan=1>ORF1abgene(JOE)</td><td rowspan=1 colspan=1>100% (48/48)</td><td rowspan=1 colspan=1>30.2 ±0.69(2.3%)</td><td rowspan=1 colspan=1>100%(48/48)</td><td rowspan=1 colspan=1>29.3 ±0.69(2.4%)</td><td rowspan=1 colspan=1>100%(48/48)</td><td rowspan=1 colspan=1>28.1 ±0.62(2.2%)</td><td rowspan=1 colspan=1>100%(144/144)</td><td rowspan=1 colspan=1>29.2 ±1.09(3.7%)</td></tr><tr><td rowspan=1 colspan=1>InternalControl(Q670)</td><td rowspan=1 colspan=1>97.9% (47/48)</td><td rowspan=1 colspan=1>31.7 ±0.6(1.9%)</td><td rowspan=1 colspan=1>100.0%(48/48)</td><td rowspan=1 colspan=1>31.9 ±0.42(1.3%)</td><td rowspan=1 colspan=1>100.0%(48/48)</td><td rowspan=1 colspan=1>31.8 ±0.68(2.1%)</td><td rowspan=1 colspan=1>99.3%(143/144)</td><td rowspan=1 colspan=1>31.8 ±0.57(1.8%)</td></tr></table>

2. Linearity:

Not applicable; this is a qualitative assay.

3. Analytical Specificity/Interference:

# Cross Reactivity:

The potential for assay cross-reactivity was evaluated by addition of a panel of microorganisms selected based on their possible presence in respiratory specimens. Exclusivity of the Simplexa COVID-19 Direct was evaluated by spiking viruses at $\mathrm { 1 x 1 0 ^ { 5 } T C I D 5 0 / m L }$ , and bacteria or fungi at $1 \mathrm { x } 1 0 ^ { 6 } \mathrm { C F U / m L }$ (where possible). Each organism was tested in triplicate. All three SARS-CoV-2 assay targets were expected to be negative in each replicate and any invalid runs due to failure of controls or instrument/software errors were to be repeated. A total of 47 different organisms were tested in addition to the human nasal fluid which closely represents diverse microbial flora and human genomic DNA.

In silico (BLAST) analysis was performed for Bacillus anthracis, Influenza C and Pneumocystis jirovecii as these organisms were not available for wet testing. Human coronavirus 229E, human coronavirus NL63, human metapneumovirus (hMPV) and Leptospira interrogans could not be sourced at the specified concentration so it was supplemented by in silico analysis.

A summary of the cross-reactivity evaluation is shown below. No cross-reactivity was detected for any of the organisms tested (i.e., 0/3 detection for each assay/replicate/analyte).

Simplexa COVID-19 Direct Cross-Reactivity (Analytical Specificity)   

<table><tr><td colspan="1" rowspan="1">Organisms/Virus</td><td colspan="1" rowspan="1">Test Concentration</td><td colspan="1" rowspan="1">Cross-Reactivity Detected</td></tr><tr><td colspan="1" rowspan="1">Adenovirus C (Type 1)</td><td colspan="1" rowspan="1">1 x 105 U/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Adenovirus 7A</td><td colspan="1" rowspan="1">1 x 105 TCID50 /mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Bordetella pertussis</td><td colspan="1" rowspan="1">1 x 106 CFU/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Candida albicans</td><td colspan="1" rowspan="1">1 x 106 CFU/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Chlamydophila pneumoniae</td><td colspan="1" rowspan="1">1 x 106 IFU/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Chlamydophila psittaci (genomicDNA)</td><td colspan="1" rowspan="1">1 x 106 copies/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium diphtheriae</td><td colspan="1" rowspan="1">1 x 106 CFU/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Coxiella burnetii (genomic DNA)</td><td colspan="1" rowspan="1">1 x 106 copies/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Cytomegalovirus</td><td colspan="1" rowspan="1">1 x 105 U/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Enterovirus 68</td><td colspan="1" rowspan="1">1 x 105 U/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Epstein-Barr virus</td><td colspan="1" rowspan="1">1 x 105 copies/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">1 x 106 CFU/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Haemophilus influenzae</td><td colspan="1" rowspan="1">1 x 106 CFU/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Human coronavirus 229E*</td><td colspan="1" rowspan="1">3 x 104 TCID50 /mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Human coronavirus HKU1 (RNA)</td><td colspan="1" rowspan="1">1 x 105 genomecopies/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Human coronavirus NL63*</td><td colspan="1" rowspan="1">3 x 104 U/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Human coronavirus OC43</td><td colspan="1" rowspan="1">1 x 105 TCID50 /mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Human genomic DNA (Leukocytes)</td><td colspan="1" rowspan="1">1 x 106 cells/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Human metapneumovirus (hMPV)*</td><td colspan="1" rowspan="1">3 x 104 TCID50 /mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Influenza A/Perth/16/2009</td><td colspan="1" rowspan="1">1 x 105 EID50 /mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Influenza B/Florida/02/06</td><td colspan="1" rowspan="1">1 x 105 U/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus plantarum 17-5</td><td colspan="1" rowspan="1">1 x 106 CFU/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Legionella longbeachae</td><td colspan="1" rowspan="1">1 x 106 CFU/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Legionella pneumophila</td><td colspan="1" rowspan="1">1 x 106 CFU/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Leptospira interrogans</td><td colspan="1" rowspan="1">1:10 Dilution</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Measles</td><td colspan="1" rowspan="1">1 x 105 TCID50 /mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">MERS-coronavirus</td><td colspan="1" rowspan="1">1 x 105 TCID50 /mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Moraxella catarrhalis</td><td colspan="1" rowspan="1">1 x 106 CFU/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Mumps</td><td colspan="1" rowspan="1">1 x 105 U/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Mycobacterium tuberculosis(genomic DNA)</td><td colspan="1" rowspan="1">1 x 106 copies/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Mycoplasma pneumoniae</td><td colspan="1" rowspan="1">1 x 106 CCU/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Neisseria elongata</td><td colspan="1" rowspan="1">1 x 106 CFU/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Neisseria meningitidis</td><td colspan="1" rowspan="1">1 x 106 CFU/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza virus 1</td><td colspan="1" rowspan="1">1 x 105 U/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza virus 2</td><td colspan="1" rowspan="1">1 x 105 U/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza virus 3</td><td colspan="1" rowspan="1">1 x 105 TCID50 /mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza virus 4</td><td colspan="1" rowspan="1">1 x 105 U/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Parechovirus 3</td><td colspan="1" rowspan="1">1 x 105 U/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">1 x 106 CFU/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Respiratory syncytial Virus A</td><td colspan="1" rowspan="1">1 x 105 TCID50 /mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Respiratory syncytial Virus B</td><td colspan="1" rowspan="1">1 x 105 TCID50 /mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Rhinovirus</td><td colspan="1" rowspan="1">1 x 105 U/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">SARS-coronavirus (RNA)</td><td colspan="1" rowspan="1">1 x 105 copies/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">1 x 106 CFU/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus epidermidis</td><td colspan="1" rowspan="1">1 x 106 CFU/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Streptococcus pneumoniae</td><td colspan="1" rowspan="1">1 x 106 CFU/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Streptococcus pyogenes</td><td colspan="1" rowspan="1">1 x 106 CFU/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Streptococcus salivarius</td><td colspan="1" rowspan="1">1 x 106 CFU/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Pooled Human Nasal Fluid</td><td colspan="1" rowspan="1">1:1 Dilution</td><td colspan="1" rowspan="1">None</td></tr></table>

$^ * \mathrm { A }$ lower concentration was tested due to inability to obtain stock material with high titer $1 \mathrm { C C U / m L } =$ Color Changing Units/milliliter, $\mathrm { C F U / m L } =$ Colony Forming Units/milliliter, $\mathrm { I F U / m L = }$ Infectious units/milliliter, $\mathrm { U / m L = }$ Units/milliliter, $\Gamma \mathrm { C I D } 5 0 / \mathrm { m L } =$ Tissue Culture Infectious Dose/milliliter

# Interference

Simplexa COVID-19 Direct was tested for potential interference by substances that may be present in respiratory specimens. Test samples were contrived in clinical NPS matrix, spiked with SARS-CoV-2(2019-nCoV/USA-WA1/2020 strain at 3x LoD $( 1 5 0 0 \mathrm { c p / m l } )$ and adding the potentially interfering substances at concentrations shown below. Each test sample was run in triplicate at one site using four LIAISON MDX instruments by three operators, over the course of six days. No interference was observed at the tested concentrations. The following table provides the substances and concentrations tested. The FluMist nasal vaccine was not tested as it was unavailable at the time of the study. A limitation regarding FluMist evaluation is added to the assay labeling.

Interfering Substances Tested on the Simplexa COVID-19 Direct   

<table><tr><td colspan="1" rowspan="1">Potentially InterferingSubstance</td><td colspan="1" rowspan="1">Active Ingredient</td><td colspan="1" rowspan="1">TestedConcentration</td><td colspan="1" rowspan="1">SARS-CoV-2QualitativeRests: %Detection (#Detected/#Tested)</td><td colspan="1" rowspan="1">IC QualitativeResults: %Detection (#Detected/#Tested)</td></tr><tr><td colspan="1" rowspan="1">Antibiotic nasal ointment(Mupirocin)</td><td colspan="1" rowspan="1">Mupirocin</td><td colspan="1" rowspan="1">6.6 mg/mL</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">100.0% (3/3)</td></tr><tr><td colspan="1" rowspan="1">Anti-viral drug (Oseltamivir)</td><td colspan="1" rowspan="1">Oseltamivir</td><td colspan="1" rowspan="1">3.3 mg/mL</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">100.0% (3/3)</td></tr><tr><td colspan="1" rowspan="1">Cold Eeze (Throat lozenges, Oralanesthetic and analgesic)</td><td colspan="1" rowspan="1">Zincum gluconicum2X</td><td colspan="1" rowspan="1">2.5% (w/v)</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">100.0% (3/3)</td></tr><tr><td colspan="1" rowspan="1">Homeopathic allergy relief medicine</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">10% (v/v)</td><td colspan="1" rowspan="1">100.0% (3/3)</td><td colspan="1" rowspan="1">100.0% (3/3)</td></tr><tr><td colspan="1" rowspan="1">Mucin (Bovine submaxillary gland,type I-S)</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">5 mg/mL</td><td colspan="1" rowspan="1">100.0% (3/3)</td><td colspan="1" rowspan="1">100.0% (3/3)</td></tr><tr><td colspan="1" rowspan="1">Nasal corticosteroids (Fluticasone)</td><td colspan="1" rowspan="1">Fluticasone</td><td colspan="1" rowspan="1">5% (v/v)</td><td colspan="1" rowspan="1">100.0% (3/3)</td><td colspan="1" rowspan="1">100.0% (3/3)</td></tr><tr><td colspan="1" rowspan="1">Allergy Relief Swabs (Nasal Gel,Zicam)</td><td colspan="1" rowspan="1">Luffa opperculata,Galphimia glauca,histaminumhydrochloricum,Sulphur</td><td colspan="1" rowspan="1">5% (w/v)</td><td colspan="1" rowspan="1">100.0% (3/3)</td><td colspan="1" rowspan="1">100.0% (3/3)</td></tr><tr><td colspan="1" rowspan="1">Nasal spray or drops(Oxymetazoline)</td><td colspan="1" rowspan="1">Oxymetazoline</td><td colspan="1" rowspan="1">15% (v/v)</td><td colspan="1" rowspan="1">100.0% (3/3)</td><td colspan="1" rowspan="1">100.0% (3/3)</td></tr><tr><td colspan="1" rowspan="2">Saliva</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">10% (v/v)</td><td colspan="1" rowspan="1">83.3% (5/6)</td><td colspan="1" rowspan="1">83.3% (5/6)</td></tr><tr><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">5% (v/v)*</td><td colspan="1" rowspan="1">100.0% (6/6)</td><td colspan="1" rowspan="1">100.0% (6/6)</td></tr><tr><td colspan="1" rowspan="1">Systemic antibacterial (Tobramycin)</td><td colspan="1" rowspan="1">Tobramycin</td><td colspan="1" rowspan="1">4 μg/mL</td><td colspan="1" rowspan="1">100.0% (3/3)</td><td colspan="1" rowspan="1">100.0% (3/3)</td></tr><tr><td colspan="1" rowspan="1">Whole Blood</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">2% (v/v)</td><td colspan="1" rowspan="1">100.0% (3/3)</td><td colspan="1" rowspan="1">100.0% (3/3)</td></tr><tr><td colspan="1" rowspan="1">Zanamivir</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">3 mg/mL</td><td colspan="1" rowspan="1">100.0% (6/6)</td><td colspan="1" rowspan="1">83.3% (5/6)</td></tr></table>

\*Interference from saliva was observed at a concentration a above $5 \%$ $\mathbf { m } \mathbf { g } =$ milligram, $\mathrm { m L } =$ milliliter, $\mathbf { v } / \mathbf { v } =$ volume to volume, $\mathbf { w } / \mathbf { v } =$ weight to volume, ${ \mu } { \mathrm { g } } =$ microgram, $\mathrm { N A } =$ Not Applicable

# Microbial Interference:

Simplexa COVID-19 Direct was evaluated for ability to identify SARS-CoV-2 when other potentially cross-reacting organisms were present. Potentially cross-reacting organisms were individually spiked into a pool of negative nasopharyngeal swab matrix with a low concentration of inactivated SARS-CoV-2 at approximately two times the limit of detection (2x LoD). Except for Lactobacillus plantarum $I 7  – 5$ , no inhibition by other organisms was observed at the tested concentration. Testing Lactobacillus plantarum $I 7  – 5$ at a lower concentration $( 5 \mathrm { ~ x ~ } 1 0 ^ { 5 } \mathrm { C F U / m L } )$ ) resulted in no inhibition.

Microbial Inhibition Tested on the Simplexa COVID-19 Direct   

<table><tr><td colspan="1" rowspan="2">Organism</td><td colspan="1" rowspan="2">TestedConcentration</td><td colspan="1" rowspan="2">COVID-19QualitativeResults: %Detection (#Detected/#Tested)</td><td colspan="2" rowspan="1">IC (Q670)</td></tr><tr><td colspan="1" rowspan="1">% Detection (#Detected / #Tested)</td><td colspan="1" rowspan="1">Mean Ct ±SD (%CV)</td></tr><tr><td colspan="1" rowspan="1">Adenovirus C</td><td colspan="1" rowspan="1">1 x 105 TCID50/mL</td><td colspan="1" rowspan="1">100.0% (3/3)</td><td colspan="1" rowspan="1">100.0% (3/3)</td><td colspan="1" rowspan="1">31.0 ± 0.6(1.9%)</td></tr><tr><td colspan="1" rowspan="1">Adenovirus 7A</td><td colspan="1" rowspan="1">1 x 105 TCID50/mL</td><td colspan="1" rowspan="1">100.0% (3/3)</td><td colspan="1" rowspan="1">100.0% (3/3)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Bordetella pertussis</td><td colspan="1" rowspan="1">1 x 106 CFU/mL</td><td colspan="1" rowspan="1">100.0% (3/3)</td><td colspan="1" rowspan="1">100.0% (3/3)</td><td colspan="1" rowspan="1">30.8 ± 0.2(0.6%)</td></tr><tr><td colspan="1" rowspan="1">Candida albicans</td><td colspan="1" rowspan="1">1 x 106 CFU/mL</td><td colspan="1" rowspan="1">100.0% (3/3)</td><td colspan="1" rowspan="1">100.0% (3/3)</td><td colspan="1" rowspan="1">30.9 ± 0.5(1.6%)</td></tr><tr><td colspan="1" rowspan="1">Chlamydiapneumoniae</td><td colspan="1" rowspan="1">1 x 106 IFU/mL</td><td colspan="1" rowspan="1">100.0% (3/3)</td><td colspan="1" rowspan="1">100.0% (3/3)</td><td colspan="1" rowspan="1">31.2 ± 0.2(0.6%)</td></tr><tr><td colspan="1" rowspan="1">Chlamydophilapsittaci(genomic DNA)</td><td colspan="1" rowspan="1">2.5 x105copies/mL *</td><td colspan="1" rowspan="1">100.0% (3/3)</td><td colspan="1" rowspan="1">100.0% (3/3)</td><td colspan="1" rowspan="1">31.1 ± 0.0(0.0%)</td></tr><tr><td colspan="1" rowspan="1">Corynebacteriumdiphtheriae</td><td colspan="1" rowspan="1">1 x 106 CFU/mL</td><td colspan="1" rowspan="1">100.0% (3/3)</td><td colspan="1" rowspan="1">100.0% (3/3)</td><td colspan="1" rowspan="1">31.3 ± 0.5(1.6%)</td></tr><tr><td colspan="1" rowspan="1">Coxiella burnetii(genomic DNA)</td><td colspan="1" rowspan="1">1 x 106 copies/mL</td><td colspan="1" rowspan="1">100.0% (3/3)</td><td colspan="1" rowspan="1">100.0% (3/3)</td><td colspan="1" rowspan="1">31.3 ± 0.1(0.3%)</td></tr><tr><td colspan="1" rowspan="1">Cytomegalovirus</td><td colspan="1" rowspan="1">1 x 105 U/mL</td><td colspan="1" rowspan="1">100.0% (3/3)</td><td colspan="1" rowspan="1">100.0% (3/3)</td><td colspan="1" rowspan="1">31.3 ± 0.4(1.3%)</td></tr><tr><td colspan="1" rowspan="1">Enterovirus 68</td><td colspan="1" rowspan="1">1 x 105 TCID50/mL</td><td colspan="1" rowspan="1">100.0% (3/3)</td><td colspan="1" rowspan="1">100.0% (3/3)</td><td colspan="1" rowspan="1">31.6 ± 0.3(0.9%)</td></tr><tr><td colspan="1" rowspan="1">Epstein-Barr virus</td><td colspan="1" rowspan="1">1 x 105 copies/mL</td><td colspan="1" rowspan="1">100.0% (3/3)</td><td colspan="1" rowspan="1">100.0% (3/3)</td><td colspan="1" rowspan="1">32.0 ± 0.2(0.6%)</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">1 x 106 CFU/mL</td><td colspan="1" rowspan="1">100.0% (3/3)</td><td colspan="1" rowspan="1">100.0% (3/3)</td><td colspan="1" rowspan="1">31.6 ± 1.0(3.2%)</td></tr><tr><td colspan="1" rowspan="1">Haemophilusinfluenzae</td><td colspan="1" rowspan="1">1 x 106 CFU/mL</td><td colspan="1" rowspan="1">100.0% (3/3)</td><td colspan="1" rowspan="1">100.0% (3/3)</td><td colspan="1" rowspan="1">31.2 ± 0.4(1.3%)</td></tr><tr><td colspan="1" rowspan="1">Human coronavirus229E</td><td colspan="1" rowspan="1">1.5 x 104TCID50/mL*</td><td colspan="1" rowspan="1">100.0% (3/3)</td><td colspan="1" rowspan="1">100.0% (3/3)</td><td colspan="1" rowspan="1">31.5 ± 0.0(0.0%)</td></tr><tr><td colspan="1" rowspan="1">Human coronavirusHKU1 (RNA)</td><td colspan="1" rowspan="1">1 x 105 genomecopies/mL</td><td colspan="1" rowspan="1">100.0% (3/3)</td><td colspan="1" rowspan="1">100.0% (3/3)</td><td colspan="1" rowspan="1">32.6 ± 0.2(0.6%)</td></tr><tr><td colspan="1" rowspan="1">Human coronavirusNL63</td><td colspan="1" rowspan="1">1.5 x104TCID50/mL</td><td colspan="1" rowspan="1">100.0% (3/3)</td><td colspan="1" rowspan="1">100.0% (3/3)</td><td colspan="1" rowspan="1">31.6 ± 0.2(0.6%)</td></tr><tr><td colspan="1" rowspan="1">Human coronavirusOC43</td><td colspan="1" rowspan="1">1 x 105 TCID50/mL</td><td colspan="1" rowspan="1">100.0% (3/3)</td><td colspan="1" rowspan="1">100.0% (3/3)</td><td colspan="1" rowspan="1">31.0 ± 0.7(2.3%)</td></tr><tr><td colspan="1" rowspan="1">Human genomicDNA (Leukocytes)</td><td colspan="1" rowspan="1">1 x 106 copies/mL</td><td colspan="1" rowspan="1">100.0% (3/3)</td><td colspan="1" rowspan="1">100.0% (3/3)</td><td colspan="1" rowspan="1">31.1 ± 0.3(1.0%)</td></tr><tr><td colspan="1" rowspan="1">HumanMetapneumovirus(hMPV)</td><td colspan="1" rowspan="1">1.5 x 104TCID50/mL*</td><td colspan="1" rowspan="1">100.0% (3/3)</td><td colspan="1" rowspan="1">100.0% (3/3)</td><td colspan="1" rowspan="1">31.9 ± 0.4(1.3%)</td></tr><tr><td colspan="1" rowspan="1">InfluenzaA/Perth/16/2009</td><td colspan="1" rowspan="1">1 x 105 EID50/mL</td><td colspan="1" rowspan="1">100.0% (3/3)</td><td colspan="1" rowspan="1">100.0% (3/3)</td><td colspan="1" rowspan="1">30.8 ± 0.3(1.0%)</td></tr><tr><td colspan="1" rowspan="1">InfluenzaB/Phuket/3073/2013</td><td colspan="1" rowspan="1">1 x 105 CEID50/mL</td><td colspan="1" rowspan="1">100.0% (3/3)</td><td colspan="1" rowspan="1">100.0% (3/3)</td><td colspan="1" rowspan="1">30.9 ± 0.3(1.0%)</td></tr><tr><td colspan="1" rowspan="2">Lactobacillusplantarum 17-5</td><td colspan="1" rowspan="1">1 x 106 CFU/mL</td><td colspan="1" rowspan="1">90.0% (18/20)</td><td colspan="1" rowspan="1">100.0% (20/20)</td><td colspan="1" rowspan="1">31.5 ± 0.4(1.3%)</td></tr><tr><td colspan="1" rowspan="1">5 x 105 CFU/mL **</td><td colspan="1" rowspan="1">100.0% (6/6)</td><td colspan="1" rowspan="1">100.0% (6/6)</td><td colspan="1" rowspan="1">32.1 ± 0.9(2.8%)</td></tr><tr><td colspan="1" rowspan="1">Legionellalongbeachae</td><td colspan="1" rowspan="1">1 x 106 CFU/mL</td><td colspan="1" rowspan="1">100.0% (3/3)</td><td colspan="1" rowspan="1">100.0% (3/3)</td><td colspan="1" rowspan="1">31.7 ± 0.3(0.9%)</td></tr><tr><td colspan="1" rowspan="1">Legionellapneumophila</td><td colspan="1" rowspan="1">1 x 106 CFU/mL</td><td colspan="1" rowspan="1">100.0% (3/3)</td><td colspan="1" rowspan="1">100.0% (3/3)</td><td colspan="1" rowspan="1">30.7 ± 0.3(1.0%)</td></tr><tr><td colspan="1" rowspan="1">Leptospirainterrogans</td><td colspan="1" rowspan="1">1:10 Dilution</td><td colspan="1" rowspan="1">100.0% (6/6)</td><td colspan="1" rowspan="1">100.0% (6/6)</td><td colspan="1" rowspan="1">33.2 ± 1.1(3.3%)</td></tr><tr><td colspan="1" rowspan="1">Measles</td><td colspan="1" rowspan="1">1 x 105 TCID50/mL</td><td colspan="1" rowspan="1">100.0% (3/3)</td><td colspan="1" rowspan="1">100.0% (3/3)</td><td colspan="1" rowspan="1">31.3 ± 0.2(0.6%)</td></tr><tr><td colspan="1" rowspan="1">MERS-coronavirus</td><td colspan="1" rowspan="1">1 x 105 TCID50/mL</td><td colspan="1" rowspan="1">100.0% (3/3)</td><td colspan="1" rowspan="1">100.0% (3/3)</td><td colspan="1" rowspan="1">30.8 ± 0.3(1.0%)</td></tr><tr><td colspan="1" rowspan="1">Moraxellacatarrhalis</td><td colspan="1" rowspan="1">1 x 106 CFU/mL</td><td colspan="1" rowspan="1">100.0% (3/3)</td><td colspan="1" rowspan="1">100.0% (3/3)</td><td colspan="1" rowspan="1">31.4 ± 0.3(1.0%)</td></tr><tr><td colspan="1" rowspan="1">Mumps</td><td colspan="1" rowspan="1">1 x 105 U/mL</td><td colspan="1" rowspan="1">95.0% (19/20)</td><td colspan="1" rowspan="1">100.0% (20/20)</td><td colspan="1" rowspan="1">31.9 ± 0.4(1.3%)</td></tr><tr><td colspan="1" rowspan="1">Mycobacteriumtuberculosis((genomicDNA</td><td colspan="1" rowspan="1">1 x 106 copies/mL</td><td colspan="1" rowspan="1">100.0% (3/3)</td><td colspan="1" rowspan="1">100.0% (3/3)</td><td colspan="1" rowspan="1">30.7 ± 0.1(0.3%)</td></tr><tr><td colspan="1" rowspan="1">Mycoplasmapneumoniae</td><td colspan="1" rowspan="1">1 x 106 CCU/mL</td><td colspan="1" rowspan="1">100.0% (3/3)</td><td colspan="1" rowspan="1">100.0% (3/3)</td><td colspan="1" rowspan="1">30.9 ± 0.4(1.3%)</td></tr><tr><td colspan="1" rowspan="1">Neisseria elongata</td><td colspan="1" rowspan="1">1 x 106 CFU/mL</td><td colspan="1" rowspan="1">100.0% (3/3)</td><td colspan="1" rowspan="1">100.0% (3/3)</td><td colspan="1" rowspan="1">32.5 ± 0.4(1.2%)</td></tr><tr><td colspan="1" rowspan="1">Neisseriameningitidis</td><td colspan="1" rowspan="1">1 x 106 CFU/mL</td><td colspan="1" rowspan="1">100.0% (3/3)</td><td colspan="1" rowspan="1">100.0% (3/3)</td><td colspan="1" rowspan="1">32.4 ± 0.3(0.9%)</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza virus</td><td colspan="1" rowspan="1">1 x 105 TCID50/mL</td><td colspan="1" rowspan="1">100.0% (3/3)</td><td colspan="1" rowspan="1">100.0% (3/3)</td><td colspan="1" rowspan="1">30.7 ± 0.7(2.3%)</td></tr><tr><td colspan="1" rowspan="1">Streptococcuspyogenes</td><td colspan="1" rowspan="1">1 x 106 CFU/mL</td><td colspan="1" rowspan="1">100.0% (3/3)</td><td colspan="1" rowspan="1">100.0% (3/3)</td><td colspan="1" rowspan="1">31.1 ± 0.1(0.3%)</td></tr><tr><td colspan="1" rowspan="1">Streptococcussalivarius</td><td colspan="1" rowspan="1">1 x 106 CFU/mL</td><td colspan="1" rowspan="1">100.0% (3/3)</td><td colspan="1" rowspan="1">100.0% (3/3)</td><td colspan="1" rowspan="1">31.3 ± 0.5(1.6%)</td></tr><tr><td colspan="1" rowspan="1">Pooled human nasalfluid</td><td colspan="1" rowspan="1">1:1 Dilution</td><td colspan="1" rowspan="1">100.0% (3/3)</td><td colspan="1" rowspan="1">100.0% (3/3)</td><td colspan="1" rowspan="1">31.6 ± 0.4(1.3%)</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza virus2</td><td colspan="1" rowspan="1">1 x 105 TCID50/mL</td><td colspan="1" rowspan="1">100.0% (3/3)</td><td colspan="1" rowspan="1">100.0% (3/3)</td><td colspan="1" rowspan="1">31.8 ± 0.9(2.8%)</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza virus3</td><td colspan="1" rowspan="1">1 x 105 TCID50/mL</td><td colspan="1" rowspan="1">100.0% (3/3)</td><td colspan="1" rowspan="1">100.0% (3/3)</td><td colspan="1" rowspan="1">31.1 ± 0.4(1.3%)</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza virus4</td><td colspan="1" rowspan="1">1 x 105 TCID50/mL</td><td colspan="1" rowspan="1">100.0% (3/3)</td><td colspan="1" rowspan="1">100.0% (3/3)</td><td colspan="1" rowspan="1">30.9 ± 0.2(0.6%)</td></tr><tr><td colspan="1" rowspan="1">Parechovirus 3</td><td colspan="1" rowspan="1">1 x 105 TCID50/mL</td><td colspan="1" rowspan="1">100.0% (3/3)</td><td colspan="1" rowspan="1">100.0% (3/3)</td><td colspan="1" rowspan="1">31.8 ± 0.7(2.2%)</td></tr><tr><td colspan="1" rowspan="1">Pseudomonasaeruginosa</td><td colspan="1" rowspan="1">1 x 106 CFU/mL</td><td colspan="1" rowspan="1">100.0% (3/3)</td><td colspan="1" rowspan="1">100.0% (3/3)</td><td colspan="1" rowspan="1">30.9 ± 0.6(1.9%)</td></tr><tr><td colspan="1" rowspan="1">Respiratorysyncytial virus A</td><td colspan="1" rowspan="1">5 x 104 TCID50/mL</td><td colspan="1" rowspan="1">100.0% (3/3)</td><td colspan="1" rowspan="1">100.0% (3/3)</td><td colspan="1" rowspan="1">30.9 ± 0.2(0.6%)</td></tr><tr><td colspan="1" rowspan="1">Respiratorysyncytial virus B</td><td colspan="1" rowspan="1">1 x 105 TCID50/mL</td><td colspan="1" rowspan="1">100.0% (3/3)</td><td colspan="1" rowspan="1">100.0% (3/3)</td><td colspan="1" rowspan="1">30.8 ± 0.5(1.6%)</td></tr><tr><td colspan="1" rowspan="1">Rhinovirus</td><td colspan="1" rowspan="1">1 x 105 TCID50/mL</td><td colspan="1" rowspan="1">100.0% (3/3)</td><td colspan="1" rowspan="1">100.0% (3/3)</td><td colspan="1" rowspan="1">31.0 ± 0.3(1.0%)</td></tr><tr><td colspan="1" rowspan="1">SARS-Coronavirus(RNA)</td><td colspan="1" rowspan="1">1 x 105 copies/mL</td><td colspan="1" rowspan="1">100.0% (3/3)</td><td colspan="1" rowspan="1">100.0% (3/3)</td><td colspan="1" rowspan="1">32.3 ± 0.3(0.9%)</td></tr><tr><td colspan="1" rowspan="1">Staphylococcusaureus</td><td colspan="1" rowspan="1">1 x 106 CFU/mL</td><td colspan="1" rowspan="1">100.0% (3/3)</td><td colspan="1" rowspan="1">100.0% (3/3)</td><td colspan="1" rowspan="1">31.0 ± 0.2(0.6%)</td></tr><tr><td colspan="1" rowspan="1">Staphylococcusepidermidis</td><td colspan="1" rowspan="1">1 x 106 CFU/mL</td><td colspan="1" rowspan="1">100.0% (3/3)</td><td colspan="1" rowspan="1">100.0% (3/3)</td><td colspan="1" rowspan="1">31.5 ± 0.2(0.6%)</td></tr><tr><td colspan="1" rowspan="1">Streptococcuspneumoniae</td><td colspan="1" rowspan="1">1 x 106 CFU/mL</td><td colspan="1" rowspan="1">100.0% (3/3)</td><td colspan="1" rowspan="1">100.0% (3/3)</td><td colspan="1" rowspan="1">31.1 ± 0.1(0.3%)</td></tr></table>

$^ { * } \mathbf { A }$ lower concentration was tested due to inability to obtain stock material with high titer $^ { \ast \ast }$ Interfernce with Lactobacillus plantarum 17-5 was observed at a concentration above 5 x 105 CFU/mL. ${ } ^ { 1 } \mathrm { C C U } =$ Color changing units/milliliter ${ \mathrm { C F U / m L } } =$ Colony forming units/milliliter $\mathrm { { I F U / m l } = }$ Infectious units/milliliter $\mathrm { U / m L = }$ Units/milliliter $\mathrm { T C I D } _ { 5 0 } / \mathrm { m L } =$ Tissue Culture Infectious Dose per milliliter

4. Assay Reportable Range:

Not applicable; this is a qualitative assay.

5. Traceability, Stability, Expected Values (Controls, Calibrators, or Methods):

# Assay Controls:

The assay contains an internal control for PCR function. DiaSorin Molecular offers optional external QC materials that are intended for use with the assay. The positive control is marketed as Simplexa COVID-19 Positive Control Gen II Pack. Controls are packaged in single use aliquots and stored frozen; once thawed the controls are stable for thirty (30) minutes at ambient laboratory temperature.

The following table shows daily controls, including count, mean, standard deviation (SD), $\% \mathrm { C V }$ , minimum and maximum for all valid PC and NTC results during the clinical study.   

<table><tr><td rowspan=2 colspan=1>Ctrl</td><td rowspan=1 colspan=6>S gene(FAM)</td><td rowspan=1 colspan=6>ORF1ab(JOE)</td><td rowspan=1 colspan=6>Internal Control (Q670)</td></tr><tr><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>Mean</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>CV</td><td rowspan=1 colspan=1>Min</td><td rowspan=1 colspan=1>MaX</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>Mean</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>CV</td><td rowspan=1 colspan=1>Min</td><td rowspan=1 colspan=1>MaX</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>Mean</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>CV</td><td rowspan=1 colspan=1>Min</td><td rowspan=1 colspan=1>Max</td></tr><tr><td rowspan=1 colspan=1>NTC</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>31.1</td><td rowspan=1 colspan=1>0.69</td><td rowspan=1 colspan=1>2.2%</td><td rowspan=1 colspan=1>30.0</td><td rowspan=1 colspan=1>33.9</td></tr><tr><td rowspan=1 colspan=1>PC</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>28.1</td><td rowspan=1 colspan=1>2.04</td><td rowspan=1 colspan=1>7.3</td><td rowspan=1 colspan=1>24...</td><td rowspan=1 colspan=1>$3_2a2.</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>28.3</td><td rowspan=1 colspan=1>2.04</td><td rowspan=1 colspan=1>7.2%</td><td rowspan=1 colspan=1>25.1</td><td rowspan=1 colspan=1>$3ra2$</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>31.2</td><td rowspan=1 colspan=1>0.82</td><td rowspan=1 colspan=1>2.6%</td><td rowspan=1 colspan=1>28</td><td rowspan=1 colspan=1>34.6</td></tr></table>

# Fresh versus frozen Study:

A fresh vs. frozen study was designed for the candidate device to determine the assay performance testing NPS swabs in UTM and saline after freezing. SARS-CoV2 negative NPS swab samples in UTM/saline were used to prepare negative or positive samples spiked with heat-inactivated SARS-COV-2 viral particles from the 2019-nCoV/USA-WA1/2020 strain at two concentrations (10 samples at 5x LoD and 30 samples at 2x LoD). All samples were tested fresh prior to freezing for 24 hrs at $\leq 7 0 ^ { \circ }$ ; up to three freeze/thaw cycles following the storage condition, as described below.

![](images/e6f043f833cfad7a8a7970622ddc08641fa5f0f1f7cdacf97b15de3601eb94b7.jpg)

The Fresh vs. Frozen study was conducted for each time point and showed a $100 \%$ detection (positivity) for samples prepared at 5x LoD (except for time point C, one replicate at 5X LoD at time point C gave negative result initially; the same replicate was repeated on the same day on two different runs with successful detection of all targets.), $2 9 5 \%$ detection (positivity) for samples prepared at $2 \mathbf { x }$ LoD and $0 \%$ detection of all negative (un-spiked) samples.

# Specimen Stability:

Nasopharyngeal swabs (NPS) collected in UTM and in saline were tested as negative or as contrived positive samples by spiking the heat-inactivated SARS-CoV-2 viral particles from the 2019-nCoV/USA-WA1/2020 strain (ATCC, Manassas, VA) at two different concentrations: $2 \mathbf { x }$ and 5x LoD. Samples were tested fresh, then stored refrigerated $\left( 2 \mathrm { - } 8 ^ { \circ } \mathrm { C } \right)$ and tested in 10 replicates at Day 3, Day 5, Day 7, and Day 10.

Each sample were frozen and tested for up to three days for three freeze-thaw cycles. The study data supported sample storage at $2 { - } 8 ~ ^ { \circ } \mathrm { C }$ for up to 7 days post collection and if there is a delay the specimens can be stored at $- 7 0 ~ ^ { \circ } \mathrm { C }$ for up to three days. for three freeze thaw cycles

# 6. Detection Limit:

The Limit of Detection (LoD) in NPS and NS samples was determined using heat inactivated SARS-CoV-2 (USA_WA1/2020) spiked into clinical NPS and NS specimens in UTM. LoD is defined as the lowest concentration of SARS-CoV-2 RNA that can be detected at a rate of at least $9 5 \%$ . Tentative LoD was determined using serial dilutions of inactivated virus in NPS matrix in UTM and NS matrix in UTM. The tentative LoD in NPS and NS is then confirmed by testing 40 and 20 replicates respectively.

Simplexa COVID-19 Direct Limit of Detection – Nasopharyngeal Swab   

<table><tr><td rowspan=1 colspan=1>Copies/ml</td><td rowspan=1 colspan=1>Interpretation*</td><td rowspan=1 colspan=1>S gene</td><td rowspan=1 colspan=1>ORF1ab gene</td></tr><tr><td rowspan=1 colspan=1>500</td><td rowspan=1 colspan=1>100% (40/40)</td><td rowspan=1 colspan=1>90% (36/40)</td><td rowspan=1 colspan=1>100% (40/40)</td></tr></table>

\*Interpretation as per the Results interpretation algorithm.

Simplexa COVID-19 Direct Limit of Detection – Nasal Swab   

<table><tr><td rowspan=1 colspan=1>Copies/ml</td><td rowspan=1 colspan=1>Interpretation*</td><td rowspan=1 colspan=1>S gene</td><td rowspan=1 colspan=1>ORF1ab gene</td></tr><tr><td rowspan=1 colspan=1>242</td><td rowspan=1 colspan=1>100% (20/20)</td><td rowspan=1 colspan=1>80% (16/20)</td><td rowspan=1 colspan=1>80% (16/20)</td></tr></table>

\*Interpretation as per the Results interpretation algorithm.

The LoD was confirmed at $5 0 0 \mathrm { c p / m l }$ for the NPS matrix and $2 4 2 \mathrm { c p } / \mathrm { m l }$ for NS matrix with heat inactivated SARS-CoV-2.

# Study Results with WHO International Standard Material:

An additional LoD study was done using the SARS-CoV-2 WHO International Standard virus particles to further evaluate the sensitivity of the Simplexa COVID-19 Direct. The tentative LoD was identified with serial dilutions of the viral particles in NPS matrix. The final LoD was confirmed to be the lowest concentration quantified in International Units per milliliter $\mathrm { ( I U / m L ) }$ ) resulting in positive detection with a minimum $9 5 \%$ positivity.

The following table shows the confirmatory LoD study results with additional replicates.

LoD Confirmation with SARS-CoV-2 WHO International Standard   

<table><tr><td rowspan=1 colspan=1>IU/mL</td><td rowspan=1 colspan=1>S gene(% Detection)</td><td rowspan=1 colspan=1>ORF1ab gene(% Detection)</td><td rowspan=1 colspan=1>RNA IC(% Detection)</td><td rowspan=1 colspan=1>ResultInterpretation(%Detection)</td></tr><tr><td rowspan=1 colspan=1>1500</td><td rowspan=1 colspan=1>100% (20/20)</td><td rowspan=1 colspan=1>100% (20/20)</td><td rowspan=1 colspan=1>100% (20/20)</td><td rowspan=1 colspan=1>100% (20/20)</td></tr><tr><td rowspan=1 colspan=1>500</td><td rowspan=1 colspan=1>95% (38/40)</td><td rowspan=1 colspan=1>85% (34/40)</td><td rowspan=1 colspan=1>100% (40/40)</td><td rowspan=1 colspan=1>97.5% (39/40)</td></tr><tr><td rowspan=1 colspan=1>167</td><td rowspan=1 colspan=1>40% (8/20)</td><td rowspan=1 colspan=1>30% (6/20)</td><td rowspan=1 colspan=1>100% (20/20)</td><td rowspan=1 colspan=1>55% (11/20)</td></tr></table>

The results confirmed that the LoD with SARS-CoV-2 WHO International Standard virus particles in NPS matrix is at $5 0 0 ~ \mathrm { I U / m l }$ with $9 7 . 5 \%$ detection.

7. Analytical Reactivity:

# In silico

An in silico inclusivity analysis of the oligonucleotide (oligo) sequences for the SARS-CoV2 ORF1ab and S gene sets were performed against all SARS-CoV-2 sequences available in the GISAID database submitted from November 01, 2021 to January 31, 2022 and February 01, 2022 to April 30, 2022. The analysis included 2,170,584 and 377,668 sequences in the amplicon regions of the ORF1ab and S gene oligo sets, respectively. Only target sequences with full coverage of all three oligo-binding regions (forward primer, reverse primer, and

probe) are included in the analyses for both oligo sets. Partial target sequences and sequences with ambiguous or degenerate bases in an oligo binding region are excluded from this inclusivity analysis.

Based on in silico analysis of the percent homology between assay oligos and target sequences, potential impact of location of the mismatches on extension and/or binding, and the mismatch Tm values of each oligo sequence to its binding region on each SARS-CoV-2 sequence, it is predicted that the Simplexa COVID-19 Direct will detect all analyzed SARSCoV-2 sequences from the GISAID database, including sequences of the Omicron BA.1, Omicron BA, Omicron BA.2.12.1, BA.4 and BA.5 subvariants 2 and IHU variants.

An additional in silico inclusivity analysis was performed for complete SARS-CoV-2 genome sequences available in the GISAID database submitted from May 01, 2022 to July 31, 2022 including sequences of the Omicron BA.2.12.1, BA.2.75, BA.4 and BA.5 subvariants. The analysis included 211,224 sequences in the amplicon regions of the ORF1ab and S gene primer/probe regions. Only target sequences with full coverage of all three ORF1ab and S gene forward and reverse primer as well as probe region were included in the analyses. The analysis showed that the Simplexa COVID-19 Direct target regions had no mismatch to 208,582 sequences $( \sim 9 8 . 7 \% )$ and were predicted to be detected by the assay based on sequence homology. There were 2602 $( \sim 1 . 2 \% )$ sequences with no mismatches for one gene oligo set (either ORF1ab or S gene), and there were 40 sequences $( \sim 0 . 0 2 \% )$ with mismatches in at least one primer or probe binding region, region in either ORF1ab or S gene target region.

A Tm analysis was conducted, and results are summarized below:

Summary of Tm Analysis Results   

<table><tr><td rowspan=1 colspan=1>Timeframe of SequencesAnalyzed</td><td rowspan=1 colspan=1>Number of accessionsin GISAID Databasefor the timeframe</td><td rowspan=1 colspan=1>Number of sequenceswhere at least onetarget oligo set meetsTm criteria</td><td rowspan=1 colspan=1>Identity to SARS-CoV-2 gene design</td></tr><tr><td rowspan=1 colspan=1>Nov. 1, 2021, to Jan. 31, 2022</td><td rowspan=1 colspan=1>2,170,584</td><td rowspan=1 colspan=1>2,170,584</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Feb. 1, 2022, to Apr. 30, 2022</td><td rowspan=1 colspan=1>377,668</td><td rowspan=1 colspan=1>377,668</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>May 1, 2022, to July 31, 2022</td><td rowspan=1 colspan=1>211,224</td><td rowspan=1 colspan=1>211,224</td><td rowspan=1 colspan=1>100%</td></tr></table>

Wet Testing   
The analytical reactivity was evaluated using the following five strains (of SARS-CoV-2) for   
wet testing  
•Hong Kong/VM200001061/2020   
•England/204820464/2020   
•South Africa/KRISP-EC-K005325/2020   
•Japan/TY7-503/2021   
•hCoV19/USA/PHC658/2021

The SARS-CoV-2 viral particles listed above were diluted at 1,000 copies/ml (2X LoD) in NPS clinical matrix, in triplicate. All replicates tested “Positive” for COVID-19.

Summary of Analytical Reactivity   

<table><tr><td rowspan=1 colspan=1>COVID-19 Strain</td><td rowspan=1 colspan=1>TestedConcentration</td><td rowspan=1 colspan=1>SARS-CoV-2Qualitative Results: %Detection(# Detected /#Tested)</td><td rowspan=1 colspan=1>IC QualitativeResults: % Detection(# Detected /#Tested)</td></tr><tr><td rowspan=1 colspan=1>HongKong/VM200001061/2020</td><td rowspan=1 colspan=1>1000 copies/mL</td><td rowspan=1 colspan=1>100% (3/3) Positive</td><td rowspan=1 colspan=1>100.0% (3/3)</td></tr><tr><td rowspan=1 colspan=1>England/204820464/2020</td><td rowspan=1 colspan=1>1000 copies/mL</td><td rowspan=1 colspan=1>100% (3/3) Positive</td><td rowspan=1 colspan=1>100.0% (3/3)</td></tr><tr><td rowspan=1 colspan=1>South Africa/KRISP-EC-K005325/2020</td><td rowspan=1 colspan=1>1000 copies/mL</td><td rowspan=1 colspan=1>100% (3/3) Positive</td><td rowspan=1 colspan=1>100.0% (3/3)</td></tr><tr><td rowspan=1 colspan=1>Japan/TY7-503/2021</td><td rowspan=1 colspan=1>1000 copies/mL</td><td rowspan=1 colspan=1>100% (3/3) Positive</td><td rowspan=1 colspan=1>100.0% (3/3)</td></tr><tr><td rowspan=1 colspan=1>hCoV19/USA/PHC658/2021</td><td rowspan=1 colspan=1>1000 copies/mL</td><td rowspan=1 colspan=1>100% (3/3) Positive</td><td rowspan=1 colspan=1>100.0% (3/3)</td></tr></table>

8. Assay Cut-Off:

Simplexa COVID-19 Direct test kit can be only used with LIAISON MDX System. It is a thermocycler which is capable of heating, cooling, mixing of sample and reagents. The system can detect up to four fluorophores at distinct wavelengths and the sensors on the instrument can monitor the primary functions.

The assay cut-off for this device was established by adjusting the target fluorescent channel threshold and the RT-PCR cycle threshold (Ct) cut-offs to values that allow for sensitivity while maintaining analyte specificity. The fluorescent channel threshold was optimized with the RT-PCR Ct cut-offs for the S gene (FAM), ORF1ab gene (JOE), and Internal Control (Q670), during the Development, Verification and Transfer studies during the assay development, according to the sponsor’s internal protocols. The MDX system is designed to interpret the test data and automatically report the test results to the operator.

# 9. Carry-Over:

Since carry-over studies with high positive NPS samples followed by negative samples have been previously performed for other FDA-cleared Simplexa assays, e.g., Simplexa Flu A/B & RSV Direct Gen II (K201505), a carry-over study was not performed for this submission.in support of the Simplexa COVID-19 Direct assay. The FDA-cleared Simplexa Flu A/B & RSV Direct Gen II assay utilizes identical DAD discs to those used for the Simplexa COVID-19 Direct assay. For details on the carry-over study performed for the cleared Simplexa Flu A/B & RSV Direct Gen II, please refer to file K201505

# B Comparison Studies:

1. Method Comparison with Predicate Device: Not applicable. Refer to the Clinical Studies Section of this document.

2. Matrix Comparison:

Not applicable.

# C Clinical Studies:

The clinical study for Simplexa COVID-19 Direct was conducted with prospective fresh and/or prospective frozen NPS and/or NS specimens, collected from four geographically diverse collection sites and four testing sites (three external clinical sites and one internal site). The study utilized leftover, de-identified specimens prospectively collected in transport media (Copan UTM, Cepheid UTM, BD VTM and saline $0 . 9 \%$ ), between October 14, 2020, and April 30, 2021, from individuals with signs and symptoms of respiratory tract infection.

The performance of the assay was compared to the test results obtained with three commercially available high-performing FDA EUA authorized SARS-CoV-2 molecular RT-PCR assays.

Specimens were collected from 1150 patients of which, 409 were NPS specimens and 741 were NS specimens. All the patient samples were enrolled as all comers that met the study protocol inclusion and exclusion criteria

Each specimen, after collection in the media, was tested with Simplexa COVID-19 Direct. For comparison, the leftover samples were also tested with three comparator methods according to the IFU for each of the assays, where the third assay was only used for samples which gave discrepant results between the other two methods, for a two-out-of-three result interpretation. There were 114 samples that were collected in media other than UTM or saline that were excluded from analysis. Additionally, 139 specimens were excluded from the performance analysis due to instrument failure (LIASON MDX) or user error (24), or an indeterminate comparator result (1).

Among the 1011 specimens available for the final calculations of performance estimates, 443 $( 4 3 . 8 \% )$ were males ranging in age from 76 days old to 96 years old, and 568 $( 5 6 . 2 \% )$ were females ranging in age from 51 days old to 91 years old.

The clinical performance of the Simplexa COVID-19 Direct assay when compared with the comparator reference method (CRM) across the four sites combined is shown below.

Clinical Performance of Simplexa COVID-19 Direct vs. CRM (NPS and NS combined)   

<table><tr><td colspan="1" rowspan="1">ProspectiveSample</td><td colspan="1" rowspan="1">CRMDetected</td><td colspan="1" rowspan="1">CRMNot detected</td><td colspan="1" rowspan="1">Total</td></tr><tr><td colspan="1" rowspan="1">Simplexa COVID-19DirectDetected</td><td colspan="1" rowspan="1">108</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">112</td></tr><tr><td colspan="1" rowspan="1">Simplexa COVID-19DirectNot Detected</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">897</td><td colspan="1" rowspan="1">899</td></tr><tr><td colspan="1" rowspan="1">Total</td><td colspan="1" rowspan="1">110</td><td colspan="1" rowspan="1">901</td><td colspan="1" rowspan="1">1011</td></tr><tr><td colspan="1" rowspan="1">PPA</td><td colspan="3" rowspan="1">98.2% (108/110)95% CI: (93.6% to 99.5%)</td></tr><tr><td colspan="1" rowspan="1">NPA</td><td colspan="3" rowspan="1">99.6% (897/901)</td></tr><tr><td></td><td>95% CI:</td></tr></table>

# Clinical Performance of Simplexa COVID-19 Direct vs. CRM (NPS only)

<table><tr><td rowspan=1 colspan=1>ProspectiveNPS Sample</td><td rowspan=1 colspan=1>CRMDetected</td><td rowspan=1 colspan=1>CRMNot detected</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Simplexa COVID-19DirectDetected</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>61</td></tr><tr><td rowspan=1 colspan=1>Simplexa COVID-19DirectNot Detected</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>237</td><td rowspan=1 colspan=1>238</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>61</td><td rowspan=1 colspan=1>238</td><td rowspan=1 colspan=1>299</td></tr><tr><td rowspan=1 colspan=1>PPA</td><td rowspan=1 colspan=3>98.4% (60/61)95% CI: (91.3% to 99.7%)</td></tr><tr><td rowspan=1 colspan=1>NPA</td><td rowspan=1 colspan=3>99.6% (237/238)95% CI: (97.7% to 99.9%)</td></tr></table>

Clinical Performance of Simplexa COVID-19 Direct vs. CRM (NS specimens)   

<table><tr><td rowspan=1 colspan=1>ProspectiveNS Sample</td><td rowspan=1 colspan=1>CRMDetected</td><td rowspan=1 colspan=1>CRMNot Detected</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Simplexa COVID-19DirectDetected</td><td rowspan=1 colspan=1>48</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>51</td></tr><tr><td rowspan=1 colspan=1>Simplexa COVID-19DirectNot Detected</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>660</td><td rowspan=1 colspan=1>661</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>49</td><td rowspan=1 colspan=1>663</td><td rowspan=1 colspan=1>712</td></tr><tr><td rowspan=1 colspan=1>PPA</td><td rowspan=1 colspan=3>98.0% (48/49)95% CI: (89.3% to 99.6%)</td></tr><tr><td rowspan=1 colspan=1>NPA</td><td rowspan=1 colspan=3>99.5% (660/663)95% CI: 98.7% to 99.8%</td></tr></table>

# D Clinical Cut-Off:

Not applicable

# E Expected Values/Reference Range:

The expected values for the Simplexa COVID-19 Direct, based on the positivity rate observed during the clinical study, are presented stratified by site in the table below. Overall, SARS-CoV2 was detected in $1 0 . 7 \%$ (108/1011) of specimens. The positivity rates across sites ranged between $6 . 2 8 \%$ (Site 4) to $3 0 . 5 \%$ (Site 2).

Positivity by Simplexa COVID-19 Direct Observed During the Study, Stratified by Site   

<table><tr><td rowspan=2 colspan=1>SARS-CoV-2</td><td rowspan=1 colspan=2>Overall(n=1011)</td><td rowspan=1 colspan=2>1 - LabCorpSouth Bend, IN(n=261)</td><td rowspan=1 colspan=2>2 - TriCoreAlbuquerque,NM (n=59)</td><td rowspan=1 colspan=2>3 - UCLALos Angeles,CA (n=182)</td><td rowspan=1 colspan=2>4 CerbaXpertFrance(n=509)</td></tr><tr><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>%</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>110</td><td rowspan=1 colspan=1>10.7%</td><td rowspan=1 colspan=1>38</td><td rowspan=1 colspan=1>14.55%</td><td rowspan=1 colspan=1>18</td><td rowspan=1 colspan=1>30.5%</td><td rowspan=1 colspan=1>22</td><td rowspan=1 colspan=1>12.1%</td><td rowspan=1 colspan=1>32</td><td rowspan=1 colspan=1>6.28%</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>901</td><td rowspan=1 colspan=1>88.7%</td><td rowspan=1 colspan=1>223</td><td rowspan=1 colspan=1>84.44%</td><td rowspan=1 colspan=1>41</td><td rowspan=1 colspan=1>69.5%</td><td rowspan=1 colspan=1>160</td><td rowspan=1 colspan=1>87.91%</td><td rowspan=1 colspan=1>477</td><td rowspan=1 colspan=1>93.71%</td></tr></table>

# F Other Supportive Instrument Performance Characteristics Data:

Not applicable

# VIII Proposed Labeling:

The labeling supports the finding of substantial equivalence for this device.

# IX Conclusion:

The submitted information in this premarket notification is complete and supports a substantial equivalence decision.