GenMark Diagnostics, Incorporated   
Alan Maderazo   
VP, Quality, Regulatory & Clinical Affairs   
5964 La Place Court   
Carlsbad, California 92008

Re: K213236 Trade/Device Name: ePlex Blood Culture Identification Gram Negative (BCID-GN) Panel Regulation Number: 21 CFR 866.3365 Regulation Name: Multiplex Nucleic Acid Assay For Identification Of Microorganisms And Resistance Markers From Positive Blood Cultures Regulatory Class: Class II Product Code: PEN, PAM, PEO Dated: September 29, 2021 Received: September 30, 2021

Dear Alan Maderazo:

We have reviewed your Section 510(k) premarket notification of intent to market the device referenced above and have determined the device is substantially equivalent (for the indications for use stated in the enclosure) to legally marketed predicate devices marketed in interstate commerce prior to May 28, 1976, the enactment date of the Medical Device Amendments, or to devices that have been reclassified in accordance with the provisions of the Federal Food, Drug, and Cosmetic Act (Act) that do not require approval of a premarket approval application (PMA). You may, therefore, market the device, subject to the general controls provisions of the Act. Although this letter refers to your product as a device, please be aware that some cleared products may instead be combination products. The 510(k) Premarket Notification Database located at https://www.accessdata.fda.gov/scripts/cdrh/cfdocs/cfpmn/pmn.cfm identifies combination product submissions. The general controls provisions of the Act include requirements for annual registration, listing of devices, good manufacturing practice, labeling, and prohibitions against misbranding and adulteration. Please note: CDRH does not evaluate information related to contract liability warranties. We remind you, however, that device labeling must be truthful and not misleading.

If your device is classified (see above) into either class II (Special Controls) or class III (PMA), it may be subject to additional controls. Existing major regulations affecting your device can be found in the Code of Federal Regulations, Title 21, Parts 800 to 898. In addition, FDA may publish further announcements concerning your device in the Federal Register.

Please be advised that FDA's issuance of a substantial equivalence determination does not mean that FDA has made a determination that your device complies with other requirements of the Act or any Federal statutes and regulations administered by other Federal agencies. You must comply with all the Act's

requirements, including, but not limited to: registration and listing (21 CFR Part 807); labeling (21 CFR Part 801 and Part 809); medical device reporting (reporting of medical device-related adverse events) (21 CFR 803) for devices or postmarketing safety reporting (21 CFR 4, Subpart B) for combination products (see https://www.fda.gov/combination-products/guidance-regulatory-information/postmarketing-safety-reportingcombination-products); good manufacturing practice requirements as set forth in the quality systems (QS) regulation (21 CFR Part 820) for devices or current good manufacturing practices (21 CFR 4, Subpart A) for combination products; and, if applicable, the electronic product radiation control provisions (Sections 531- 542 of the Act); 21 CFR 1000-1050.

Also, please note the regulation entitled, "Misbranding by reference to premarket notification" (21 CFR Part 807.97). For questions regarding the reporting of adverse events under the MDR regulation (21 CFR Part 803), please go to https://www.fda.gov/medical-devices/medical-device-safety/medical-device-reportingmdr-how-report-medical-device-problems.

For comprehensive regulatory information about medical devices and radiation-emitting products, including information about labeling regulations, please see Device Advice (https://www.fda.gov/medical  
devices/device-advice-comprehensive-regulatory-assistance) and CDRH Learn   
(https://www.fda.gov/training-and-continuing-education/cdrh-learn). Additionally, you may contact the Division of Industry and Consumer Education (DICE) to ask a question about a specific regulatory topic. See the DICE website (https://www.fda.gov/medical-devices/device-advice-comprehensive-regulatoryassistance/contact-us-division-industry-and-consumer-education-dice) for more information or contact DICE by email (DICE@fda.hhs.gov) or phone (1-800-638-2041 or 301-796-7100).

Sincerely,

Noel Gerald, Ph.D.   
Chief   
Division of Microbiology Devices   
OHT7: Office of In Vitro Diagnostics   
and Radiological Health   
Office of Product Evaluation and Quality Center for Devices and Radiological Health

Enclosure

Device Name ePlex Blood Culture Identification Gram-Negative (BCID-GN) Panel

Indications for Use (Describe)   
The GenMark ePlex $^ \mathrm { \textregistered }$ Blood Culture Identification Gram-Negative (BCID-GN) Panel is a qualitative nucleic acid mueMar   uti lposicrasvalpenT BCID- an  lultp  si y culture system and which contain gram-negative organism.

ThN Bakaaka Enterobacter (non-cloacae complex), Escherichia coli, Fusobacterium necrophorum, Fusobacterium nucleatum, Haemohilusiuenzae, Klebsiellaoxytoca, Klebsiella eonagroup, MoranellaorganiNesseaeis, Proteus, Proteus mirabilis, Pseudomonas aeruginosa, Salmonella, Serratia, Serratiamarccens, Stenotrophomonas maltophilia, CTX-M (blaCTX-M), IMP (blaIMP) , KPC (blaKPC) , NDM (blaNDM), OXA (blaOXA) (OXA-23 and OXA-48 groups only), and VIM (blaVIM).

The ePlex BCID-GN Panel contains assays or the detection of genetic determinant associated wit resistanc t antimicrobial agents including CTX-M(blaCTX-M), which is associated with resistance to extended spectrum betalactamase (ESBL)-mediated resistance to penicllins, cephalosporins, and monobactams, as wel as OXA (blaOXA) (OXA-23 and OXA-48 groups only), KPC (blaKPC), and metallo-beta-lactamases IMP (blaIMP), VIM (blaVIM), and NDM (blaNDM), which is associated with carbapenemase-mediated resistance. The antimicrobial resistance gene detected may or may not be associated with the agent responsible for disease. Negative results for these selec negative bacteria.

The ePlex BCID-G Panel also contains targets designed t detect a broad rangeof organisms with a potentially ha c San  esyhi  o detect four Candida species: Candida albicans, Candida glabrata, Candida krusei, and Candida parapsilosis.

Tvl syptobloremcootremetheusti cu u

N BC anel ma otbe ee us il boastu-culu posi blooclture encatonanis o dtected bye B an anoy a and clinical presentation must be taken into consideration in the final diagnosis of bloodstream infection.

# CONTINUE ON A SEPARATE PAGE IF NEEDED.

This section applies only to requirements of the Paperwork Reduction Act of 1995.

# DO NOT SEND YOUR COMPLETED FORM TO THE PRA STAFF EMAIL ADDRESS BELOW.

Turenohoctioationatvehurs sec time toreview instructions, search existing data sources, gatherand maintain the data needed and coplet a review the collection fnformation. Send comments regarding this burden estimateor anyother aspect of this information collection, including suggestions for reducing this burden, to:

Department of Health and Human Services Food and Drug Administration Office of Chief Information Officer Paperwork Reduction Act (PRA) Staff PRAStaff@fda.hhs.gov

"An agency may not conduct or sponsor, and a person is not required to respond to, a collection of information unless it displays a currently valid OMB number."

# 5.0 510(k) Summary

The 510(k) Summary was updated to align with the most current revision of the package insert, which incorporates changes in the Limit of Detection (LoD) section based on the results from studies conducted to support implementation of a design change (i.e., addition of oligonucleotides to improve robustness and inclusivity of the E.coli, Citrobacter, Enterococcus, and $P .$ . aeruginosa assays). The addition of oligonucleotides to specific PCR reactions introduces performance risks to the resident assays contained in those affected PCR reactions. As a result, the following studies were conducted to mitigate the identified risks to product performance.

1. Evaluation of overall BCID-GN Panel Performance: this study utilized a multianalyte test mix (containing a representative analyte from each of the eight multiplex PCR pools) to systematically assess overall performance of the BCID-GN Panel. This design was utilized to ensure that the proposed change does not introduce any unexpected issues that result in a systematic assay failure.

2. LoD Verification Study: this study verified that the LoDs of the targets affected by the change are not adversely impacted. These results demonstrate that the analytical sensitivity of the test remains equivalent.

3. Clinical Sample Evaluation: this study utilized characterized clinical samples to verify that the proposed change does not adversely impact clinical performance.

All studies met the predetermined acceptance criteria demonstrating no adverse impact to the BCIDGN Panel performance, which supports implementation of the proposed change.

The changes to the 510(k) Summary include identification of the following strains (in Table 56 of the 510(k) Summary) that were tested as part of the LoD verification study:

• H. influenzae (ATCC33930) • N. meningitidis (NCTC10026) • E. coli (JHU01-D80401147) • P. aeruginosa (SDx071)

All other information in the 510(k) Summary are unchanged.

# 510(k) Summary

Summary of Safety and Effectiveness

# Submitter Information

Submitter: GenMark Diagnostics, Incorporated 5964 La Place Court Carlsbad, CA 92008

Manufacturer: GenMark Diagnostics, Incorporated 5964 La Place Court Carlsbad, CA 92008

Establishment Registration Number: 3008632402

Contact:

Phone:

Fax:

Alternate Contact: Beth Stofka Sr. Regulatory Affairs Specialist

Phone:

# Name of Device and Classification

roduct Name: ePlex® Blood Culture Identification Gram-Negative (BCID-GN) Panel

Device Classification: 866.3980, Multiplex nucleic acid assay for identification of microorganisms and resistance markers from positive blood cultures, Class II

Product Code(s): PEN, PAM, PEO

# Predicate Device

Predicate: The ePlex Blood Culture Identification Gram-Negative (BCID-GN) Panel; GenMark Diagnostics, Inc.; K182619

# Device Description

The ePlex Blood Culture Identification Gram-Negative (BCID-GN) Panel is based on the principles of competitive nucleic acid hybridization using a sandwich assay format, wherein a single-stranded target binds concurrently to a sequence-specific solution-phase signal probe and a solid-phase electrode-bound capture probe. The test employs nucleic acid extraction, target amplification via polymerase chain reaction (PCR) or reverse transcription PCR (RT-PCR) and hybridization of target DNA. In the process, the double-stranded PCR amplicons are digested with exonuclease to generate single-stranded DNA suitable for hybridization.

Nucleic acid extraction from biological samples occurs within the cartridge via cell lysis, nucleic acid capture onto magnetic beads, and release for amplification. The nucleic acid extraction is processed through microfluidic liquid handling. Once the nucleic acid targets are captured and inhibitors are washed away, the magnetic particles are delivered to the electrowetting environment on the printed circuit board (PCB) and the targets are eluted from the particles and amplified.

During hybridization, the single-stranded target DNA binds to a complementary, single-stranded capture probe immobilized on the working gold electrode surface. Single-stranded signal probes (labeled with electrochemically active ferrocenes) bind to specific target sequence / region adjacent to the capture probe. Simultaneous hybridization of target to signal probes and capture probe is detected by alternating current voltammetry (ACV). Each working electrode on the array contains specific capture probes, and sequential analysis of each electrode allows detection of multiple analyte targets.

# Intended Use/Indications for Use

The GenMark ePlex® Blood Culture Identification Gram-Negative (BCID-GN) Panel is a qualitative nucleic acid multiplex in vitro diagnostic test intended for use on GenMark’s ePlex Instrument for simultaneous qualitative detection and identification of multiple potentially pathogenic gram-negative bacterial organisms and select determinants associated with antimicrobial resistance in positive blood culture. In addition, the ePlex BCID-GN Panel is capable of detecting several gram-positive bacteria (Pan Gram-Positive assay) and several Candida species (Pan Candida assay). The ePlex BCID-GN Panel is performed directly on blood culture samples identified as positive by a continuous monitoring blood culture system and which contain gram-negative organism.

The following bacterial organisms and genes associated with antibiotic resistance are identified using the ePlex BCID-GN Panel: Acinetobacter baumannii, Bacteroides fragilis, Citrobacter, Cronobacter sakazakii, Enterobacter cloacae complex, Enterobacter (non-cloacae complex), Escherichia coli, Fusobacterium necrophorum, Fusobacterium nucleatum, Haemophilus influenzae, Klebsiella oxytoca, Klebsiella pneumoniae group, Morganella morganii, Neisseria meningitidis, Proteus, Proteus mirabilis, Pseudomonas aeruginosa, Salmonella, Serratia, Serratia marcescens, Stenotrophomonas maltophilia, CTX-M (blaCTX-M), IMP (blaIMP) , KPC (blaKPC) , NDM (blaNDM), OXA (blaOXA) (OXA-23 and OXA-48 groups only), and VIM (blaVIM).

The ePlex BCID-GN Panel contains assays for the detection of genetic determinants associated with resistance to antimicrobial agents including CTX-M(blaCTX-M), which is associated with resistance to extended spectrum beta-lactamase (ESBL)-mediated resistance to penicillins, cephalosporins and monobactams, as well as OXA (blaOXA) (OXA-23 and OXA-48 groups only), KPC (blaKPC), and metallo-beta-lactamases IMP (blaIMP), VIM (blaVIM), and NDM (blaNDM), which is associated with carbapenemase-mediated resistance. The antimicrobial resistance gene detected may or may not be associated with the agent responsible for disease. Negative results for these select antimicrobial resistance assays do not indicate susceptibility, as there are multiple mechanisms of resistance in gram-negative bacteria.

The ePlex BCID-GN Panel also contains targets designed to detect a broad range of organisms with a potentially misleading Gram stain result or organisms that may be missed by Gram staining altogether, for example in the case of co-infections. These include a broad Pan GramPositive assay (which is designed to detect Bacillus cereus group, Bacillus subtilis group, Enterococcus, Staphylococcus, and Streptococcus), as well as a Pan Candida assay, which is designed to detect four Candida species: Candida albicans, Candida glabrata, Candida krusei, and Candida parapsilosis.

The detection and identification of specific bacterial and fungal nucleic acids from individuals exhibiting signs and/or symptoms of bloodstream infection aids in the diagnosis of bloodstream infection when used in conjunction with other clinical information. The results from the ePlex BCID-GN Panel are intended to be interpreted in conjunction with Gram stain results and should not be used as the sole basis for diagnosis, treatment, or other patient management decisions.

Negative results in the setting of a suspected bloodstream infection may be due to infection with pathogens that are not detected by this test. Positive results do not rule out co-infection with other organisms; the organism(s) detected by the ePlex BCID-GN Panel may not be the definite cause of disease. Additional laboratory testing (e.g. sub-culturing of positive blood cultures for identification of organisms not detected by ePlex BCID-GN Panel and for susceptibility testing, differentiation of mixed growth, and association of antimicrobial resistance marker genes to a specific organism) and clinical presentation must be taken into consideration in the final diagnosis of bloodstream infection.

# Summary of Technological Characteristics of the Device Compared to the Predicate Device

The updated GenMark ePlex Blood Culture Identification Gram-Negative (BCID-GN) Panel (identified as the “SubjectDevice”) and the legally marketed device, the original GenMark ePlex BCID-GN Panel (K182619) (identified as the “Predicate Device”) are described below:

<table><tr><td colspan="1" rowspan="1">Characteristic</td><td colspan="1" rowspan="1">Predicate Device (K182619)</td><td colspan="1" rowspan="1">Subject Device (Updated)</td></tr><tr><td colspan="1" rowspan="1">Product Name</td><td colspan="1" rowspan="1">ePlex BCID-GN Panel</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Manufacturer</td><td colspan="1" rowspan="1">GenMark Diagnostics, Inc.</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">OrganismsDetected</td><td colspan="1" rowspan="1">•  Acinetobacter baumanniiBacteroides fragilisCitrobacterCronobacter sakazakiiEnterobacter cloacae complexEnterobacter (non-cloacae complex)Escherichia coliFusobacterium necrophorumFusobacterium nucleatumHaemophilus influenzaeKlebsiella oxytocaKlebsiella pneumoniaeMorganella morganiiNeisseria meningitidisProteusProteus mirabilisPseudomonas aeruginosaSalmonellaSerratiaSerratia marcescensStenotrophomonas Saltophiliapyogenes(GAS)</td><td colspan="1" rowspan="1">SameAdditional strains were tested aspart of the Limit of Detection(LoD) study which include thefollowing:H. influenzae (ATCC33930)• N. meningitidis (NCTC10026)E. coli (JHU01-D80401147)P. aeruginosa (SDx071)</td></tr><tr><td colspan="1" rowspan="1">Resistance GenesDetected</td><td colspan="1" rowspan="1">CTX-M, IMP, KPC, NDM, OXA, andVIM</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Indication forUse</td><td colspan="1" rowspan="1">The ePlex BCID-GN Panel is indicated as anaid in the diagnosis of specific agents ofbacteremia. The use of additional laboratorytesting (e.g. sub- culturing of positive bloodcultures for identification of organisms notdetected by the ePlex BCID-GN Panel and forsusceptibility testing, differentiation of mixedgrowth, and association of antimicrobialresistance marker genes toa specificorganism) and clinical presentation must betaken into consideration in the final diagnosisof blood stream infection.</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Specimen Type</td><td colspan="1" rowspan="1">Blood culture samples identified aspositive by a continuous monitoringbloodculture system that demonstrates thepresence of organisms as confirmed byGram stain.</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Chemistry</td><td colspan="1" rowspan="1">Reagents on cartridge include: samplelysis and nucleic acid extraction, PCRamplification and hybridization-basedelectrochemical detection reagents.</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Hardware</td><td colspan="1" rowspan="1">GenMark ePlex Instrument &amp; SingleUseCartridge</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">SoftwareInterfaceResult Reporting</td><td colspan="1" rowspan="1">•  GenMark ePlex System Software•   GenMark ePlex BCID-GN PanelSoftware</td><td colspan="1" rowspan="1">Same</td></tr></table>

Analysis of the similarities and differences indicate that the devices are substantially equivalent in their intended uses/indications for use, and are generally the same regarding user process, ease of use and general operator protocol. Comparison of technological similarities and differences between the proposed device and the predicate do not raise new or different questions of safety and effectiveness, and therefore render the proposed device as substantially equivalent to the predicate device.

# Summary of Performance Data

# Expected Values

A prospective, multicenter clinical study was conducted to evaluate the clinical performance of the ePlex BCID-GN Panel in positive blood culture samples. A total of 349 samples were prospectively collected at 7 clinical sites in 2 phases from patients of all ages and genders. In the first phase from June 2014 through July 2016, 182 samples were prospectively collected and frozen; from June through July 2018, 167 samples were prospectively collected and tested fresh (never frozen). The expected values of individual analytes based on the ePlex BCID-GN Panel results in prospective samples are summarized by age group and by site in Tables 1 and 2 below.

Table 1: Expected Value by Age Group (Prospective Samples)   

<table><tr><td rowspan=1 colspan=1>Target</td><td rowspan=1 colspan=1>All AgesN3 9n (%)</td><td rowspan=1 colspan=1>Age &lt;1(N7n((%</td><td rowspan=1 colspan=1>Age 1-17(N=10n (%)</td><td rowspan=1 colspan=1>Age 18-44(N50n (%)</td><td rowspan=1 colspan=1>Age 45-64(N124)n %)</td><td rowspan=1 colspan=1>Age 65-84(N=12 n (%)</td><td rowspan=1 colspan=1>Age 85+AN-33 </td></tr><tr><td rowspan=1 colspan=1>Acinetobacter baumannii</td><td rowspan=1 colspan=1>4 (1.1)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>1 (2.0)</td><td rowspan=1 colspan=1>2 ((1.6)</td><td rowspan=1 colspan=1>1 (0.8)</td><td rowspan=1 colspan=1>0 (0.0)</td></tr><tr><td rowspan=1 colspan=1>Bacteroides fragilis</td><td rowspan=1 colspan=1>11 (3.2)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>2 (4.0)</td><td rowspan=1 colspan=1>4 (3.2)</td><td rowspan=1 colspan=1>2 (1.6)</td><td rowspan=1 colspan=1>3 (9.1)</td></tr><tr><td rowspan=1 colspan=1>Citrobacter</td><td rowspan=1 colspan=1>8 (2.3)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>2 (4.0)</td><td rowspan=1 colspan=1>1 (0.8)</td><td rowspan=1 colspan=1>2 (1.6)</td><td rowspan=1 colspan=1>3 (9.1)</td></tr><tr><td rowspan=1 colspan=1>Cronobacter sakazakii</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td></tr><tr><td rowspan=1 colspan=1>Enterobacter (non-cloacaecomplex)</td><td rowspan=1 colspan=1>9 (2.6)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>1 (10.0)</td><td rowspan=1 colspan=1>2 (4.0)</td><td rowspan=1 colspan=1>5 (4.0)</td><td rowspan=1 colspan=1>1 (0.8)</td><td rowspan=1 colspan=1>0 (0.0)</td></tr><tr><td rowspan=1 colspan=1>Enterobacter cloacae complex</td><td rowspan=1 colspan=1>23 (6.6)</td><td rowspan=1 colspan=1>3 (42.9)</td><td rowspan=1 colspan=1>1 (10.0)</td><td rowspan=1 colspan=1>6 (12.0)</td><td rowspan=1 colspan=1>5 (4.0)</td><td rowspan=1 colspan=1>8 (6.4)</td><td rowspan=1 colspan=1>0 (0.0)</td></tr><tr><td rowspan=1 colspan=1>Escherichia coli</td><td rowspan=1 colspan=1>132 (37.8)</td><td rowspan=1 colspan=1>2 (28.6)</td><td rowspan=1 colspan=1>2 (20.0)</td><td rowspan=1 colspan=1>16 (32.0)</td><td rowspan=1 colspan=1>41 (33.1)</td><td rowspan=1 colspan=1>55 (44.0)</td><td rowspan=1 colspan=1>16 (48.5)</td></tr><tr><td rowspan=1 colspan=1>Fusobacterium necrophorum</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td></tr><tr><td rowspan=1 colspan=1>Fusobacterium nucleatum</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td></tr><tr><td rowspan=1 colspan=1>Haemophilus influenzae</td><td rowspan=1 colspan=1>7 (2.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>3 (6.0)</td><td rowspan=1 colspan=1>1 (0.8)</td><td rowspan=1 colspan=1>1 (0.8)</td><td rowspan=1 colspan=1>2 (6.1)</td></tr><tr><td rowspan=1 colspan=1>Klebsiella oxytoca</td><td rowspan=1 colspan=1>12 (3.4)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>3 (6.0)</td><td rowspan=1 colspan=1>8 (6.5)</td><td rowspan=1 colspan=1>1 (0.8)</td><td rowspan=1 colspan=1>0 (0.0)</td></tr><tr><td rowspan=1 colspan=1>Klebsiella pneumoniae group</td><td rowspan=1 colspan=1>59 (16.9)</td><td rowspan=1 colspan=1>1 (14.3)</td><td rowspan=1 colspan=1>1 (10.0)</td><td rowspan=1 colspan=1>10 (20.0)</td><td rowspan=1 colspan=1>26 (21.0)</td><td rowspan=1 colspan=1>17 (13.6)</td><td rowspan=1 colspan=1>4 (12.1)</td></tr><tr><td rowspan=1 colspan=1>Morganella morganii</td><td rowspan=1 colspan=1>3 (0.9)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>2 (1.6)</td><td rowspan=1 colspan=1>1 (0.8)</td><td rowspan=1 colspan=1>0 (0.0)</td></tr><tr><td rowspan=1 colspan=1>Neisseria meningitidis</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td></tr><tr><td rowspan=1 colspan=1>Proteus</td><td rowspan=1 colspan=1>22 (6.3)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>6 (4.8)</td><td rowspan=1 colspan=1>13 (10.4)</td><td rowspan=1 colspan=1>3 (9.1)</td></tr><tr><td rowspan=1 colspan=1>Proteus mirabilis</td><td rowspan=1 colspan=1>22 (6.3)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>6 (4.8)</td><td rowspan=1 colspan=1>13 (10.4)</td><td rowspan=1 colspan=1>3 (9.1)</td></tr><tr><td rowspan=1 colspan=1>Pseudomonas aeruginosa</td><td rowspan=1 colspan=1>28 (8.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>2 (20.0)</td><td rowspan=1 colspan=1>3 (6.0)</td><td rowspan=1 colspan=1>12 (9.7)</td><td rowspan=1 colspan=1>10 (8.0)</td><td rowspan=1 colspan=1>1 (3.0)</td></tr><tr><td rowspan=1 colspan=1>Salmonella</td><td rowspan=1 colspan=1>2 (0.6)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>1 (0.8)</td><td rowspan=1 colspan=1>1 (0.8)</td><td rowspan=1 colspan=1>0 (0.0)</td></tr><tr><td rowspan=1 colspan=1>Serratia</td><td rowspan=1 colspan=1>10 (2.9)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>1 (10.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>5 (4.0)</td><td rowspan=1 colspan=1>4 (3.2)</td><td rowspan=1 colspan=1>0 (0.0)</td></tr><tr><td rowspan=1 colspan=1>Serratia marcescens</td><td rowspan=1 colspan=1>9 (2.6)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>1 (10.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>4 (3.2)</td><td rowspan=1 colspan=1>4 (3.2)</td><td rowspan=1 colspan=1>0 (0.0)</td></tr><tr><td rowspan=1 colspan=1>Stenotrophomonas maltophilia</td><td rowspan=1 colspan=1>3 (0.9)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>3 (6.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td></tr><tr><td rowspan=1 colspan=1>Pan Candida</td><td rowspan=1 colspan=1>2 (0.6)</td><td rowspan=1 colspan=1>1 (14.3)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>1 (0.8)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td></tr><tr><td rowspan=1 colspan=1>Pan Gram-Positive</td><td rowspan=1 colspan=1>24 (6.9)</td><td rowspan=1 colspan=1>1 (14.3)</td><td rowspan=1 colspan=1>2 (20.0)</td><td rowspan=1 colspan=1>5 (10.0)</td><td rowspan=1 colspan=1>7(5.6)</td><td rowspan=1 colspan=1>7(5.6)</td><td rowspan=1 colspan=1>2 (6.1)</td></tr><tr><td rowspan=1 colspan=1>CTX-M</td><td rowspan=1 colspan=1>24 (6.9)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>2 (4.0)</td><td rowspan=1 colspan=1>7(5.6)</td><td rowspan=1 colspan=1>12 (9.6)</td><td rowspan=1 colspan=1>3 (9.1)</td></tr><tr><td rowspan=1 colspan=1>IMP</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td></tr><tr><td rowspan=1 colspan=1>KPC</td><td rowspan=1 colspan=1>3 (0.9)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>1 (2.0)</td><td rowspan=1 colspan=1>1 (0.8)</td><td rowspan=1 colspan=1>1 (0.8)</td><td rowspan=1 colspan=1>0 (0.0)</td></tr><tr><td rowspan=1 colspan=1>NDM</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td></tr><tr><td rowspan=1 colspan=1>OXA</td><td rowspan=1 colspan=1>1 (0.3)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>1 (0.8)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td></tr><tr><td rowspan=1 colspan=1>VIM</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td></tr></table>

Table 2: Expected Value by Collection Site (Prospective Samples)   

<table><tr><td rowspan=1 colspan=1>Target</td><td rowspan=1 colspan=1>All Sites(N39n (%</td><td rowspan=1 colspan=1>Site 1(N 88 n(%)</td><td rowspan=1 colspan=1>Site 2(N 2 </td><td rowspan=1 colspan=1>Site 3N 98 n(%)</td><td rowspan=1 colspan=1>Site 4(N ) )</td><td rowspan=1 colspan=1>Site 5(N 4 )</td><td rowspan=1 colspan=1>Site 6(N=28 n (%)</td><td rowspan=1 colspan=1>Site 7ON ) </td></tr><tr><td rowspan=1 colspan=1>Acinetobacter baumannii</td><td rowspan=1 colspan=1>4 (1.1)</td><td rowspan=1 colspan=1>3 (3.4)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>1 (1.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td></tr><tr><td rowspan=1 colspan=1>Bacteroides fragilis</td><td rowspan=1 colspan=1>11 (3.2)</td><td rowspan=1 colspan=1>2 (2.3)</td><td rowspan=1 colspan=1>3 (13.0)</td><td rowspan=1 colspan=1>3 (3.1)</td><td rowspan=1 colspan=1>2 (3.4)</td><td rowspan=1 colspan=1>1 (2.2)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td></tr><tr><td rowspan=1 colspan=1>Citrobacter</td><td rowspan=1 colspan=1>8 (2.3)</td><td rowspan=1 colspan=1>2 (2.3)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>3 (3.1)</td><td rowspan=1 colspan=1>1 (1.7)</td><td rowspan=1 colspan=1>1 (2.2)</td><td rowspan=1 colspan=1>1 (3.6)</td><td rowspan=1 colspan=1>0 (0.0)</td></tr><tr><td rowspan=1 colspan=1>Cronobacter sakazakii</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td></tr><tr><td rowspan=1 colspan=1>Enterobacter(non-cloacae complex)</td><td rowspan=1 colspan=1>9 (2.6)</td><td rowspan=1 colspan=1>2 (2.3)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>4 (4.1)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>1 (2.2)</td><td rowspan=1 colspan=1>2 (7.1)</td><td rowspan=1 colspan=1>0 (0.0)</td></tr><tr><td rowspan=1 colspan=1>Enterobacter cloacae complex</td><td rowspan=1 colspan=1>23 (6.6)</td><td rowspan=1 colspan=1>3 (3.4)</td><td rowspan=1 colspan=1>1 (4.3)</td><td rowspan=1 colspan=1>10 (10.2)</td><td rowspan=1 colspan=1>1 (1.7)</td><td rowspan=1 colspan=1>6 (13.0)</td><td rowspan=1 colspan=1>2 (7.1)</td><td rowspan=1 colspan=1>0 (0.0)</td></tr><tr><td rowspan=1 colspan=1>Escherichia coli</td><td rowspan=1 colspan=1>132 (37.8)</td><td rowspan=1 colspan=1>30 (34.1)</td><td rowspan=1 colspan=1>8 (34.8)</td><td rowspan=1 colspan=1>37 (37.8)</td><td rowspan=1 colspan=1>25 (43.1)</td><td rowspan=1 colspan=1>17 (37.0)</td><td rowspan=1 colspan=1>12 (42.9)</td><td rowspan=1 colspan=1>3 (37.5)</td></tr><tr><td rowspan=1 colspan=1>Fusobacterium necrophorum</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td></tr><tr><td rowspan=1 colspan=1>Fusobacterium nucleatum</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td></tr><tr><td rowspan=1 colspan=1>Haemophilus influenzae</td><td rowspan=1 colspan=1>7 (2.0)</td><td rowspan=1 colspan=1>1 (1.1)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>2 (2.0)</td><td rowspan=1 colspan=1>2 (3.4)</td><td rowspan=1 colspan=1>1 (2.2)</td><td rowspan=1 colspan=1>1 (3.6)</td><td rowspan=1 colspan=1>0 (0.0)</td></tr><tr><td rowspan=1 colspan=1>Klebsiella oxytoca</td><td rowspan=1 colspan=1>12 (3.4)</td><td rowspan=1 colspan=1>5(5.7)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>3 (3.1)</td><td rowspan=1 colspan=1>1 (1.7)</td><td rowspan=1 colspan=1>2 (4.3)</td><td rowspan=1 colspan=1>1 (3.6)</td><td rowspan=1 colspan=1>0 (0.0)</td></tr><tr><td rowspan=1 colspan=1>Klebsiella pneumoniae group</td><td rowspan=1 colspan=1>59 (16.9)</td><td rowspan=1 colspan=1>17 (19.3)</td><td rowspan=1 colspan=1>3 (13.0)</td><td rowspan=1 colspan=1>20 (20.4)</td><td rowspan=1 colspan=1>5(8.6)</td><td rowspan=1 colspan=1>7(15.2)</td><td rowspan=1 colspan=1>4(14.3)</td><td rowspan=1 colspan=1>3 (37.5)</td></tr><tr><td rowspan=1 colspan=1>Morganella morganii</td><td rowspan=1 colspan=1>3 (0.9)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>1 (4.3)</td><td rowspan=1 colspan=1>2 (2.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td></tr><tr><td rowspan=1 colspan=1>Neisseria meningitidis</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td></tr><tr><td rowspan=1 colspan=1>Proteus</td><td rowspan=1 colspan=1>22 (6.3)</td><td rowspan=1 colspan=1>9 (10.2)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>5 (5.1)</td><td rowspan=1 colspan=1>5 (8.6)</td><td rowspan=1 colspan=1>2 (4.3)</td><td rowspan=1 colspan=1>1 (3.6)</td><td rowspan=1 colspan=1>0 (0.0)</td></tr><tr><td rowspan=1 colspan=1>Proteus mirabilis</td><td rowspan=1 colspan=1>22 (6.3)</td><td rowspan=1 colspan=1>9 (10.2)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>5(5.1)</td><td rowspan=1 colspan=1>5(8.6)</td><td rowspan=1 colspan=1>2 (4.3)</td><td rowspan=1 colspan=1>1 (3.6)</td><td rowspan=1 colspan=1>0 (0.0)</td></tr><tr><td rowspan=1 colspan=1>Pseudomonas aeruginosa</td><td rowspan=1 colspan=1>28 (8.0)</td><td rowspan=1 colspan=1>5 (5.7)</td><td rowspan=1 colspan=1>2 (8.7)</td><td rowspan=1 colspan=1>10 (10.2)</td><td rowspan=1 colspan=1>8 (13.8)</td><td rowspan=1 colspan=1>2 (4.3)</td><td rowspan=1 colspan=1>1 (3.6)</td><td rowspan=1 colspan=1>0 (0.0)</td></tr><tr><td rowspan=1 colspan=1>Salmonella</td><td rowspan=1 colspan=1>2 (0.6)</td><td rowspan=1 colspan=1>1 (1.1)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>1 (2.2)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td></tr><tr><td rowspan=1 colspan=1>Serratia</td><td rowspan=1 colspan=1>10 (2.9)</td><td rowspan=1 colspan=1>1 (1.1)</td><td rowspan=1 colspan=1>2 (8.7)</td><td rowspan=1 colspan=1>1 (1.0)</td><td rowspan=1 colspan=1>3 (5.2)</td><td rowspan=1 colspan=1>3 (6.5)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td></tr><tr><td rowspan=1 colspan=1>Serratia marcescens</td><td rowspan=1 colspan=1>9 (2.6)</td><td rowspan=1 colspan=1>1 (1.1)</td><td rowspan=1 colspan=1>2 (8.7)</td><td rowspan=1 colspan=1>1 (1.0)</td><td rowspan=1 colspan=1>3 (5.2)</td><td rowspan=1 colspan=1>2 (4.3)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td></tr><tr><td rowspan=1 colspan=1>Stenotrophomonas maltophilia</td><td rowspan=1 colspan=1>3 (0.9)</td><td rowspan=1 colspan=1>1 (1.1)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>1 (1.7)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>1 (3.6)</td><td rowspan=1 colspan=1>0 (0.0)</td></tr><tr><td rowspan=1 colspan=1>Pan Candida</td><td rowspan=1 colspan=1>2 (0.6)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>1 (1.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>1 (3.6)</td><td rowspan=1 colspan=1>0 (0.0)</td></tr><tr><td rowspan=1 colspan=1>Pan Gram-Positive</td><td rowspan=1 colspan=1>24 (6.9)</td><td rowspan=1 colspan=1>15 (17.0)</td><td rowspan=1 colspan=1>1 (4.3)</td><td rowspan=1 colspan=1>5(5.1)</td><td rowspan=1 colspan=1>1 (1.7)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>2 (7.1)</td><td rowspan=1 colspan=1>0 (0.0)</td></tr><tr><td rowspan=1 colspan=1>CTX-M</td><td rowspan=1 colspan=1>24 (6.9)</td><td rowspan=1 colspan=1>9 (10.2)</td><td rowspan=1 colspan=1>1 (4.3)</td><td rowspan=1 colspan=1>3 (3.1)</td><td rowspan=1 colspan=1>4 (6.9)</td><td rowspan=1 colspan=1>5 (10.9)</td><td rowspan=1 colspan=1>2 (7.1)</td><td rowspan=1 colspan=1>0 (0.0)</td></tr><tr><td rowspan=1 colspan=1>IMP</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td></tr><tr><td rowspan=1 colspan=1>KPC</td><td rowspan=1 colspan=1>3 (0.9)</td><td rowspan=1 colspan=1>1 (1.1)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>1 (1.0)</td><td rowspan=1 colspan=1>1 (1.7)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td></tr><tr><td rowspan=1 colspan=1>NDM</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td></tr><tr><td rowspan=1 colspan=1>OXA</td><td rowspan=1 colspan=1>1 (0.3)</td><td rowspan=1 colspan=1>1 (1.1)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td></tr><tr><td rowspan=1 colspan=1>VIM</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td></tr></table>

# PERFORMANCE CHARACTERISTICS

# Clinical Performance

Samples with final, valid ePlex BCID-GN Panel test results and a valid comparator result were evaluable and included in summaries and analyses of demographics, expected values (positivity rate), and performance characteristics. Evaluable samples included 167 prospective fresh and 182 prospective frozen samples as well as 577 retrospective samples and 777 contrived samples.

# Comparator Method

The performance of the ePlex BCID-GN Panel was compared to standard laboratory procedures for identification of blood culture isolates, including traditional and automated identification methods, MALDI-TOF IVD, and microbiological and biochemical techniques. Identification for samples with Acinetobacter baumannii or Candida parapsilosis identified by standard laboratory procedures was confirmed using analytically validated PCR assays followed by bi-directional sequencing. For antibiotic resistance genes, the ePlex BCID-GN Panel was compared to analytically validated qPCR amplification assays followed by bi-directional sequencing in samples with an associated organism identified by culture (See Table 3 for organism associations).

Table 3: Resistance Marker Organism Associations   

<table><tr><td rowspan=2 colspan=1>Organism</td><td rowspan=1 colspan=6>Resistance Gene</td></tr><tr><td rowspan=1 colspan=1>CTX-M</td><td rowspan=1 colspan=1>IMP</td><td rowspan=1 colspan=1>KPC</td><td rowspan=1 colspan=1>NDM</td><td rowspan=1 colspan=1>OXA</td><td rowspan=1 colspan=1>VIM</td></tr><tr><td rowspan=1 colspan=1>Acinetobacter baumannii</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td></tr><tr><td rowspan=1 colspan=1>Bacteroides fragilis</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Citrobacter</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td></tr><tr><td rowspan=1 colspan=1>Cronobacter sakazakii</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Enterobacter cloacae complex</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td></tr><tr><td rowspan=1 colspan=1>Enterobacter (non-cloacae complex)</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td></tr><tr><td rowspan=1 colspan=1>Escherichia coli</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td></tr><tr><td rowspan=1 colspan=1>Fusobacterium necrophorum</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Fusobacterium nucleatum</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Haemophilus influenzae</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Klebsiella oxytoca</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td></tr><tr><td rowspan=1 colspan=1>Klebsiella pneumoniae</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td></tr><tr><td rowspan=1 colspan=1>Morganella morganii</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td></tr><tr><td rowspan=1 colspan=1>Neisseria meningitidis</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>−</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Proteus</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td></tr><tr><td rowspan=1 colspan=1>Proteus mirabilis</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td></tr><tr><td rowspan=1 colspan=1>Pseudomonas aeruginosa</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td></tr><tr><td rowspan=1 colspan=1>Salmonella</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td></tr><tr><td rowspan=1 colspan=1>Serratia</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td></tr><tr><td rowspan=1 colspan=1>Serratia marcescens</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td></tr><tr><td rowspan=1 colspan=1>Stenotrophomonas maltophilia</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr></table>

The comparator method(s) results were used to determine the Detected / Not Detected status for each target organism on the ePlex BCID-GN Panel. The comparator methods for each target are summarized in Table 4.

Table 4: Comparator Method(s) by ePlex BCID-GN Panel Target   

<table><tr><td rowspan=1 colspan=2>Target</td><td rowspan=1 colspan=1>Comparator Method</td></tr><tr><td rowspan=1 colspan=2>Acinetobacter baumannii</td><td rowspan=1 colspan=1>Standard laboratory procedures for organism ID.PCR/sequencing to confirm Acinetobacter baumannii or differentAcinetobacter species not included in this panel.</td></tr><tr><td rowspan=1 colspan=2>Bacteroides fragilis</td><td rowspan=23 colspan=1>Standard laboratory procedures for organism identification.</td></tr><tr><td rowspan=1 colspan=2>Citrobacter</td></tr><tr><td rowspan=1 colspan=2>Cronobacter sakazakii</td></tr><tr><td rowspan=1 colspan=2>Enterobacter cloacae complex</td></tr><tr><td rowspan=1 colspan=2>Enterobacter non-cloacae complex</td></tr><tr><td rowspan=1 colspan=2>Escherichia coli</td></tr><tr><td rowspan=2 colspan=2>Fusobacterium necrophorum</td><td rowspan=2 colspan=1></td></tr><tr><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=2>Fusobacterium nucleatum</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=2>Haemophilus influenzae</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=2>Klebsiella oxytoca</td></tr><tr><td rowspan=1 colspan=2>Klebsiella pneumoniae</td></tr><tr><td rowspan=1 colspan=2>Morganella morganii</td></tr><tr><td rowspan=1 colspan=2>Neisseria meningitidis</td></tr><tr><td rowspan=1 colspan=2>Proteus</td></tr><tr><td rowspan=1 colspan=2>Proteus mirabilis</td></tr><tr><td rowspan=1 colspan=2>Pseudomonas aeruginosa</td></tr><tr><td rowspan=1 colspan=2>Salmonella</td></tr><tr><td rowspan=1 colspan=2>Serratia</td></tr><tr><td rowspan=2 colspan=2>Serratia marcescens</td><td rowspan=2 colspan=1></td></tr><tr><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=2>Stenotrophomonas maltophilia</td></tr><tr><td rowspan=1 colspan=2>Pan Gram-Negative</td></tr><tr><td rowspan=1 colspan=2>Pan Candida</td><td rowspan=1 colspan=1>Standard laboratory procedures for organism ID.PCR/sequencing to confirm C. parapsilosis or identifyC. metapsilosis, C. orthopsilosis.</td></tr><tr><td rowspan=1 colspan=2>CTX-M, IMP, KPC, NDM, OXA, VIM</td><td rowspan=1 colspan=1>qPCR/sequencing in samples with associated organism detectedby comparator method. See Table 3 for organism associations.</td></tr></table>

# Demographics of Clinical Samples

Clinical performance was evaluated in positive blood culture samples prospectively and retrospectively collected. Prospective samples were collected at 7 clinical sites in 2 phases. From June 2014 through July 2016, 183 samples were prospectively collected and frozen; from June through July 2018, 171 samples were prospectively collected and tested fresh (never frozen) for a total of 354 samples across the 2 phases. One of these samples was withdrawn due to organism identification from unacceptable methods. Of the 353 prospectively-collected samples eligible for testing, 349 were evaluable. Samples with final, valid ePlex BCID-GN Panel results and a valid comparator result were evaluable. Four samples were not evaluable because they did not have final, valid ePlex BCID-GN Panel results and were excluded from performance evaluations. Demographic information for prospectively-collected samples is described in Table 5. Subjects enrolled in this study were from a diverse demographic distribution and represent the intended patient population.

To supplement the number of positives for low prevalence targets in the prospective collection, 578 samples were collected retrospectively, 577 were evaluable. One sample was not evaluable because it did not have a final, valid ePlex BCID-GN Panel result and was excluded from performance evaluations. Demographic information for retrospectively-collected samples is described in Table 6.

Table 5: Demographic Data for Clinical Samples by Collection Site (Prospective Collection)   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>All SitesN = 349n (%)</td><td rowspan=1 colspan=1>Site 1N = 88n (%)</td><td rowspan=1 colspan=1>Site 2N = 23n (%)</td><td rowspan=1 colspan=1>Site 3N =98n (%)</td><td rowspan=1 colspan=1>Site 4N =58n (%)</td><td rowspan=1 colspan=1>Site 5N=46n (%)</td><td rowspan=1 colspan=1>Site 6N =28n (%)</td><td rowspan=1 colspan=1>Site 7N =8n(%)</td></tr><tr><td rowspan=1 colspan=8>Sex</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Male</td><td rowspan=1 colspan=1>168 (48.1)</td><td rowspan=1 colspan=1>37 (42.0)</td><td rowspan=1 colspan=1>12 (52.2)</td><td rowspan=1 colspan=1>52 (53.1)</td><td rowspan=1 colspan=1>28 (48.3)</td><td rowspan=1 colspan=1>21 (45.7)</td><td rowspan=1 colspan=1>13 (46.4)</td><td rowspan=1 colspan=1>5 (62.5)</td></tr><tr><td rowspan=1 colspan=1>Female</td><td rowspan=1 colspan=1>181 (51.9)</td><td rowspan=1 colspan=1>51 (58.0)</td><td rowspan=1 colspan=1>11 (47.8)</td><td rowspan=1 colspan=1>46 (46.9)</td><td rowspan=1 colspan=1>30 (51.7)</td><td rowspan=1 colspan=1>25 (54.3)</td><td rowspan=1 colspan=1>15 (53.6)</td><td rowspan=1 colspan=1>3 (37.5)</td></tr><tr><td rowspan=1 colspan=8>Age</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>&lt;1 yr</td><td rowspan=1 colspan=1>7 (2.0)</td><td rowspan=1 colspan=1>2 (2.3)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>4 (4.1)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>1 (2.2)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td></tr><tr><td rowspan=1 colspan=1>1-17 yrs</td><td rowspan=1 colspan=1>10 (2.9)</td><td rowspan=1 colspan=1>4 (4.5)</td><td rowspan=1 colspan=1>1 (4.3)</td><td rowspan=1 colspan=1>3 (3.1)</td><td rowspan=1 colspan=1>1 (1.7)</td><td rowspan=1 colspan=1>1 (2.2)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td></tr><tr><td rowspan=1 colspan=1>18-44 yrs</td><td rowspan=1 colspan=1>50 (14.3)</td><td rowspan=1 colspan=1>10 (11.4)</td><td rowspan=1 colspan=1>3 (13.0)</td><td rowspan=1 colspan=1>20 (20.4)</td><td rowspan=1 colspan=1>3 (5.2)</td><td rowspan=1 colspan=1>8 (17.4)</td><td rowspan=1 colspan=1>6 (21.4)</td><td rowspan=1 colspan=1>0 (0.0)</td></tr><tr><td rowspan=1 colspan=1>45-64 yrs</td><td rowspan=1 colspan=1>124 (35.5)</td><td rowspan=1 colspan=1>35 (39.8)</td><td rowspan=1 colspan=1>9 (39.1)</td><td rowspan=1 colspan=1>28 (28.6)</td><td rowspan=1 colspan=1>21 (36.2)</td><td rowspan=1 colspan=1>14 (30.4)</td><td rowspan=1 colspan=1>13 (46.4)</td><td rowspan=1 colspan=1>4 (50.0)</td></tr><tr><td rowspan=1 colspan=1>65-84 yrs</td><td rowspan=1 colspan=1>125 (35.8)</td><td rowspan=1 colspan=1>29 (33.0)</td><td rowspan=1 colspan=1>8 (34.8)</td><td rowspan=1 colspan=1>35 (35.7)</td><td rowspan=1 colspan=1>25 (43.1)</td><td rowspan=1 colspan=1>17 (37.0)</td><td rowspan=1 colspan=1>7 (25.0)</td><td rowspan=1 colspan=1>4 (50.0)</td></tr><tr><td rowspan=1 colspan=1>85+ yrs</td><td rowspan=1 colspan=1>33 (9.5)</td><td rowspan=1 colspan=1>8 (9.1)</td><td rowspan=1 colspan=1>2 (8.7)</td><td rowspan=1 colspan=1>8 (8.2)</td><td rowspan=1 colspan=1>8 (13.8)</td><td rowspan=1 colspan=1>5 (10.9)</td><td rowspan=1 colspan=1>2 (7.1)</td><td rowspan=1 colspan=1>0 (0.0)</td></tr></table>

Table 6: Demographic Data for Clinical Samples by Collection Site (Retrospective Collection)   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>All SitesN =577n (%)</td><td rowspan=1 colspan=1>Site 1N = 78n (%)</td><td rowspan=1 colspan=1>Site 2N = 73n (%)</td><td rowspan=1 colspan=1>Site 3N = 31n (%)</td><td rowspan=1 colspan=1>Site 4N = 93n (%)</td><td rowspan=1 colspan=1>Site 5N = 1n (%)</td><td rowspan=1 colspan=1>Site 6N = 80n (%)</td><td rowspan=1 colspan=1>Site 7N =  7n (%)</td><td rowspan=1 colspan=1>Site 8N = 48n (%)</td><td rowspan=1 colspan=1>Site 9N = 29n (%)</td><td rowspan=1 colspan=1>Site 10N = 77n (%)</td></tr><tr><td rowspan=1 colspan=11>Sex</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Male</td><td rowspan=1 colspan=1>307 (53.2)</td><td rowspan=1 colspan=1>36 (46.2)</td><td rowspan=1 colspan=1>41 (56.2)</td><td rowspan=1 colspan=1>15 (48.4)</td><td rowspan=1 colspan=1>49 (52.7)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>47 (58.8)</td><td rowspan=1 colspan=1>38 (56.7)</td><td rowspan=1 colspan=1>29 (60.4)</td><td rowspan=1 colspan=1>19 (65.5)</td><td rowspan=1 colspan=1>33 (42.9)</td></tr><tr><td rowspan=1 colspan=1>Female</td><td rowspan=1 colspan=1>270 (46.8)</td><td rowspan=1 colspan=1>42 (53.8)</td><td rowspan=1 colspan=1>32 (43.8)</td><td rowspan=1 colspan=1>16 (51.6)</td><td rowspan=1 colspan=1>44 (47.3)</td><td rowspan=1 colspan=1>1 (100)</td><td rowspan=1 colspan=1>33 (41.3)</td><td rowspan=1 colspan=1>29 (43.3)</td><td rowspan=1 colspan=1>19 (39.6)</td><td rowspan=1 colspan=1>10 (34.5)</td><td rowspan=1 colspan=1>44 (57.1)</td></tr><tr><td rowspan=1 colspan=1>Age</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>&lt;1 yr</td><td rowspan=1 colspan=1>9 (1.6)</td><td rowspan=1 colspan=1>1 (1.3)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>3 (3.2)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>2 (2.5)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>1 (2.1)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>2 (2.6)</td></tr><tr><td rowspan=1 colspan=1>1-17 yrs</td><td rowspan=1 colspan=1>20 (3.5)</td><td rowspan=1 colspan=1>1 (1.3)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>1 (3.2)</td><td rowspan=1 colspan=1>8 (8.6)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>6 (7.5)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>1 (3.4)</td><td rowspan=1 colspan=1>3 (3.9)</td></tr><tr><td rowspan=1 colspan=1>18-44 yrs</td><td rowspan=1 colspan=1>78 (13.5)</td><td rowspan=1 colspan=1>13 (16.7)</td><td rowspan=1 colspan=1>7 (9.6)</td><td rowspan=1 colspan=1>2 (6.5)</td><td rowspan=1 colspan=1>10 (10.8)</td><td rowspan=1 colspan=1>1 (100)</td><td rowspan=1 colspan=1>15 (18.8)</td><td rowspan=1 colspan=1>8 (11.9)</td><td rowspan=1 colspan=1>8 (16.7)</td><td rowspan=1 colspan=1>6(20.7)</td><td rowspan=1 colspan=1>8 (10.4)</td></tr><tr><td rowspan=1 colspan=1>45-64 yrs</td><td rowspan=1 colspan=1>193 (33.4)</td><td rowspan=1 colspan=1>27 (34.6)</td><td rowspan=1 colspan=1>18 (24.7)</td><td rowspan=1 colspan=1>13 (41.9)</td><td rowspan=1 colspan=1>27 (29.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>32 (40.0)</td><td rowspan=1 colspan=1>27 (40.3)</td><td rowspan=1 colspan=1>16 (33.3)</td><td rowspan=1 colspan=1>9 (31.0)</td><td rowspan=1 colspan=1>24 (31.2)</td></tr><tr><td rowspan=1 colspan=1>65-84 yrs</td><td rowspan=1 colspan=1>226 (39.2)</td><td rowspan=1 colspan=1>29 (37.2)</td><td rowspan=1 colspan=1>40 (54.8)</td><td rowspan=1 colspan=1>11 (35.5)</td><td rowspan=1 colspan=1>40 (43.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>20 (25.0)</td><td rowspan=1 colspan=1>24 (35.8)</td><td rowspan=1 colspan=1>21 (43.8)</td><td rowspan=1 colspan=1>11 (37.9)</td><td rowspan=1 colspan=1>30 (39.0)</td></tr><tr><td rowspan=1 colspan=1>85+ yrs</td><td rowspan=1 colspan=1>49 (8.5)</td><td rowspan=1 colspan=1>7 (9.0)</td><td rowspan=1 colspan=1>8 (11.0)</td><td rowspan=1 colspan=1>4 (12.9)</td><td rowspan=1 colspan=1>5 (5.4)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>5 (6.3)</td><td rowspan=1 colspan=1>6 (9.0)</td><td rowspan=1 colspan=1>2 (4.2)</td><td rowspan=1 colspan=1>2 (6.9)</td><td rowspan=1 colspan=1>10 (13.0)</td></tr><tr><td rowspan=1 colspan=1>Unknown</td><td rowspan=1 colspan=1>2 (0.3)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>2 (3.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td></tr></table>

# Clinical Performance

Sensitivity or positive percent agreement (PPA) was calculated by dividing the number of true positive (TP) results by the sum of TP and false negative (FN) results, while specificity or negative percent agreement (NPA) was calculated by dividing the number of true negative (TN) results by the sum of TN and false positive (FP) results. A TP result being defined as a sample where the detected ePlex BCID-GN Panel result matched the detected comparator method result, while a TN result was one where a negative ePlex BCID-GN Panel result matched a negative comparator method result. The two-sided $9 5 \%$ confidence interval was also calculated.

A total of 349 prospectively-collected samples (167 tested fresh and 182 tested after previously frozen) and 577 retrospectively collected samples from blood culture bottles flagged positive in a continuously monitoring blood culture system and removed from the system within 8 hours of positivity were evaluated for the ePlex BCID-GN Panel targets. Specimens evaluated were determined to contain gram-negative or gram-variable organisms based on Gram stain. A total of 777 contrived samples were prepared by spiking an isolate into a blood culture bottle with human whole blood and growing until flagged positive by a continuously monitoring blood culture system. Contrived samples were removed from the system within 8 hours of positivity and stored frozen until the time of testing. PPA and NPA results are summarized by target in Tables 7-34 below, and the strains used to contrive samples are summarized in Table 35.

Table7: Clinical Performance for Acinetobacter baumannii   

<table><tr><td rowspan=2 colspan=1>Target</td><td rowspan=2 colspan=1>Sample Type</td><td rowspan=1 colspan=2>Sensitivity/PPA</td><td rowspan=1 colspan=2>Specificity/NPA</td></tr><tr><td rowspan=1 colspan=1>TP/TP+FN</td><td rowspan=1 colspan=1>% (95% CI)</td><td rowspan=1 colspan=1>TN/TN+FP</td><td rowspan=1 colspan=1>% (95% CI)</td></tr><tr><td rowspan=7 colspan=1>Acinetobacterbaumannii</td><td rowspan=1 colspan=1>Prospective (Fresh)</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>---</td><td rowspan=1 colspan=1>167/167</td><td rowspan=1 colspan=1>100 (97.8-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective (Frozen)</td><td rowspan=1 colspan=1>4/4</td><td rowspan=1 colspan=1>100 (51.0-100)</td><td rowspan=1 colspan=1>178/178</td><td rowspan=1 colspan=1>100 (97.9-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective (All)</td><td rowspan=1 colspan=1>4/4</td><td rowspan=1 colspan=1>100 (51.0-100)</td><td rowspan=1 colspan=1>345/345</td><td rowspan=1 colspan=1>100 (98.9-100)</td></tr><tr><td rowspan=1 colspan=1>Retrospective</td><td rowspan=1 colspan=1>15/15</td><td rowspan=1 colspan=1>100 (79.6-100)</td><td rowspan=1 colspan=1>560/561</td><td rowspan=1 colspan=1>99.8 (99.0-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective / Retrospective</td><td rowspan=1 colspan=1>19/19</td><td rowspan=1 colspan=1>100 (83.2-100)</td><td rowspan=1 colspan=1>905/9064</td><td rowspan=1 colspan=1>99.9 (99.4-100)</td></tr><tr><td rowspan=1 colspan=1>Contrived</td><td rowspan=1 colspan=1>55/55</td><td rowspan=1 colspan=1>100 (93.5-100)</td><td rowspan=1 colspan=1>722/722</td><td rowspan=1 colspan=1>100 (99.5-100)</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>74/74</td><td rowspan=1 colspan=1>100 (95.1-100)</td><td rowspan=1 colspan=1>1627/1628</td><td rowspan=1 colspan=1>99.9 (99.7-100)</td></tr></table>

$\mathrm { C I } =$ Confidence Interval A. Acinetobacter baumanii was detected in the 1/1 false positive sample using PCR/sequencing.

Table 8: Clinical Performance for Bacteroides fragilis   

<table><tr><td rowspan=2 colspan=1>Target</td><td rowspan=2 colspan=1>Sample Type</td><td rowspan=1 colspan=2>Sensitivity/PPA</td><td rowspan=1 colspan=2>Specificity/NPA</td></tr><tr><td rowspan=1 colspan=1>TP/TP+FN</td><td rowspan=1 colspan=1>% (95% CI)</td><td rowspan=1 colspan=1>TN/TN+FP</td><td rowspan=1 colspan=1>% (95% CI)</td></tr><tr><td rowspan=7 colspan=1>Bacteroides fragilis</td><td rowspan=1 colspan=1>Prospective (Fresh)</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>100 (61.0-100)</td><td rowspan=1 colspan=1>161/161</td><td rowspan=1 colspan=1>100 (97.7-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective (Frozen)</td><td rowspan=1 colspan=1>5/5</td><td rowspan=1 colspan=1>100 (56.6-100)</td><td rowspan=1 colspan=1>177/177</td><td rowspan=1 colspan=1>100 (97.9-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective (All)</td><td rowspan=1 colspan=1>11/11</td><td rowspan=1 colspan=1>100 (74.1-100)</td><td rowspan=1 colspan=1>338/338</td><td rowspan=1 colspan=1>100 (98.9-100)</td></tr><tr><td rowspan=1 colspan=1>Retrospective</td><td rowspan=1 colspan=1>14/17</td><td rowspan=1 colspan=1>82.4 (59.0-93.8)</td><td rowspan=1 colspan=1>558/560</td><td rowspan=1 colspan=1>99.6 (98.7-99.9)</td></tr><tr><td rowspan=1 colspan=1>Prospective / Retrospective</td><td rowspan=1 colspan=1>25/28A</td><td rowspan=1 colspan=1>89.3 (72.8-96.3)</td><td rowspan=1 colspan=1>896/898B</td><td rowspan=1 colspan=1>99.8 (99.2-99.9)</td></tr><tr><td rowspan=1 colspan=1>Contrived</td><td rowspan=1 colspan=1>40/40</td><td rowspan=1 colspan=1>100 (91.2-100)</td><td rowspan=1 colspan=1>737/737</td><td rowspan=1 colspan=1>100 (99.5-100)</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>65/68</td><td rowspan=1 colspan=1>95.6 (87.8-98.5)</td><td rowspan=1 colspan=1>1633/1635</td><td rowspan=1 colspan=1>99.9 (99.6-100)</td></tr></table>

A. B. fragilis was not detected in 2 false negative samples, but PCR/sequencing instead detected $B$ . caccae and B. thetaiotaomicron, which were not identified by standard laboratory procedures. B. B. fragilis was detected in 2/2 false positive samples using PCR/sequencing.

Table 9: Clinical Performance for Citrobacter   

<table><tr><td rowspan=2 colspan=2>Target</td><td rowspan=2 colspan=2>Sample Type</td><td rowspan=1 colspan=2>Sensitivity/PPA</td><td rowspan=1 colspan=2>Specificity/NPA</td></tr><tr><td rowspan=1 colspan=1>TP/TP+FN</td><td rowspan=1 colspan=1>% (95% CI)</td><td rowspan=1 colspan=1>TN/TN+FP</td><td rowspan=1 colspan=1>% (95% CI)</td></tr><tr><td rowspan=7 colspan=2>Citrobacter</td><td rowspan=1 colspan=2>Prospective (Fresh)</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>100 (43.9-100)</td><td rowspan=1 colspan=1>163/164</td><td rowspan=1 colspan=1>99.4 (96.6-99.9)</td></tr><tr><td rowspan=1 colspan=2>Prospective (Frozen)</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>100 (34.2-100)</td><td rowspan=1 colspan=1>178/180</td><td rowspan=1 colspan=1>98.9 (96.0-99.7)</td></tr><tr><td rowspan=1 colspan=2>Prospective (All)</td><td rowspan=1 colspan=1>5/5</td><td rowspan=1 colspan=1>100 (56.6-100)</td><td rowspan=1 colspan=1>341/344</td><td rowspan=1 colspan=1>99.1 (97.5-99.7)</td></tr><tr><td rowspan=1 colspan=2>Retrospective</td><td rowspan=1 colspan=1>20/21</td><td rowspan=1 colspan=1>95.2 (77.3-99.2)</td><td rowspan=1 colspan=1>555/556</td><td rowspan=1 colspan=1>99.8 (99.0-100)</td></tr><tr><td rowspan=1 colspan=2>Prospective / Retrospective</td><td rowspan=1 colspan=1>25/26</td><td rowspan=1 colspan=1>96.2 (81.1-99.3)</td><td rowspan=1 colspan=1>896/9004</td><td rowspan=1 colspan=1>99.6 (98.9-99.8)</td></tr><tr><td rowspan=1 colspan=2>Contrived</td><td rowspan=1 colspan=1>43/43</td><td rowspan=1 colspan=1>100 (91.8-100)</td><td rowspan=1 colspan=1>734/734</td><td rowspan=1 colspan=1>100 (99.5-100)</td></tr><tr><td rowspan=1 colspan=2>Overall</td><td rowspan=1 colspan=1>68/69</td><td rowspan=1 colspan=1>98.6 (92.2-99.7)</td><td rowspan=1 colspan=1>1630/1634</td><td rowspan=1 colspan=1>99.8 (99.4-99.9)</td></tr></table>

A. Citrobacter braakii (2) and Citrobacter freundii (2) were detected in $4 / 4$ false positive samples using PCR/sequencing.

Table 10: Clinical Performance for Cronobacter sakazakii   

<table><tr><td rowspan=2 colspan=1>Target</td><td rowspan=2 colspan=1>Sample Type</td><td rowspan=1 colspan=2>Sensitivity/PPA</td><td rowspan=1 colspan=2>Specificity/NPA</td></tr><tr><td rowspan=1 colspan=1>TP/TP+FN</td><td rowspan=1 colspan=1>% (95% CI)</td><td rowspan=1 colspan=1>TN/TN+FP</td><td rowspan=1 colspan=1>% (95% CI)</td></tr><tr><td rowspan=7 colspan=1>Cronobacter sakazakii</td><td rowspan=1 colspan=1>Prospective (Fresh)</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>167/167</td><td rowspan=1 colspan=1>100 (97.8-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective (Frozen)</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>182/182</td><td rowspan=1 colspan=1>100 (97.9-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective (All)</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>---</td><td rowspan=1 colspan=1>349/349</td><td rowspan=1 colspan=1>100 (98.9-100)</td></tr><tr><td rowspan=1 colspan=1>Retrospective</td><td rowspan=1 colspan=1>1/1</td><td rowspan=1 colspan=1>100 (20.7-100)</td><td rowspan=1 colspan=1>576/576</td><td rowspan=1 colspan=1>100 (99.3-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective / Retrospective</td><td rowspan=1 colspan=1>1/1</td><td rowspan=1 colspan=1>100 (20.7-100)</td><td rowspan=1 colspan=1>925/925</td><td rowspan=1 colspan=1>100 (99.6-100)</td></tr><tr><td rowspan=1 colspan=1>Contrived</td><td rowspan=1 colspan=1>45/45</td><td rowspan=1 colspan=1>100 (92.1-100)</td><td rowspan=1 colspan=1>732/732</td><td rowspan=1 colspan=1>100 (99.5-100)</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>46/46</td><td rowspan=1 colspan=1>100 (92.3-100)</td><td rowspan=1 colspan=1>1657/1657</td><td rowspan=1 colspan=1>100 (99.8-100)</td></tr></table>

Table 11: Clinical Performance for Enterobacter cloacae complex   

<table><tr><td rowspan=2 colspan=1>Target</td><td rowspan=2 colspan=1>Sample Type</td><td rowspan=1 colspan=2>Sensitivity/PPA</td><td rowspan=1 colspan=2>Specificity/NPA</td></tr><tr><td rowspan=1 colspan=1>TP/TP+FN</td><td rowspan=1 colspan=1>% (95% CI)</td><td rowspan=1 colspan=1>TN/TN+FP</td><td rowspan=1 colspan=1>% (95% CI)</td></tr><tr><td rowspan=7 colspan=1>Enterobacter cloacaecomplex</td><td rowspan=1 colspan=1>Prospective (Fresh)</td><td rowspan=1 colspan=1>12/12</td><td rowspan=1 colspan=1>100 (75.8-100)</td><td rowspan=1 colspan=1>153/155</td><td rowspan=1 colspan=1>98.7 (95.4-99.6)</td></tr><tr><td rowspan=1 colspan=1>Prospective (Frozen)</td><td rowspan=1 colspan=1>7/7</td><td rowspan=1 colspan=1>100 (64.6-100)</td><td rowspan=1 colspan=1>173/175</td><td rowspan=1 colspan=1>98.9 (95.9-99.7)</td></tr><tr><td rowspan=1 colspan=1>Prospective (All)</td><td rowspan=1 colspan=1>19/19</td><td rowspan=1 colspan=1>100 (83.2-100)</td><td rowspan=1 colspan=1>326/330</td><td rowspan=1 colspan=1>98.8 (96.9-99.5)</td></tr><tr><td rowspan=1 colspan=1>Retrospective</td><td rowspan=1 colspan=1>47/50</td><td rowspan=1 colspan=1>94.0 (83.8-97.9)</td><td rowspan=1 colspan=1>526/527</td><td rowspan=1 colspan=1>99.8 (98.9-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective / Retrospective</td><td rowspan=1 colspan=1>66/694</td><td rowspan=1 colspan=1>95.7 (88.0-98.5)</td><td rowspan=1 colspan=1>852/857B</td><td rowspan=1 colspan=1>99.4 (98.6-99.8)</td></tr><tr><td rowspan=1 colspan=1>Contrived</td><td rowspan=1 colspan=1>35/37C</td><td rowspan=1 colspan=1>94.6 (82.3-98.5)</td><td rowspan=1 colspan=1>739/740</td><td rowspan=1 colspan=1>99.9 (99.2-100)</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>101/106</td><td rowspan=1 colspan=1>95.3 (89.4-98.0)</td><td rowspan=1 colspan=1>1591/1597</td><td rowspan=1 colspan=1>99.6 (99.2-99.8)</td></tr></table>

A. A species of the Enterobacter cloacae complex was not detected in 1 false negative sample, but PCR/sequencing and MALDI-TOF instead detected E. coli. Standard laboratory procedures identified $E$ . cloacae only. B. E. cloacae was detected in 2/5 false positive samples using PCR/sequencing. C. E. cloacae complex was not detected in 2 samples containing Enterobacter asburiae.

Table 12: Clinical Performance for Enterobacter (non-cloacae complex)   

<table><tr><td rowspan=2 colspan=1>Target</td><td rowspan=2 colspan=1>Sample Type</td><td rowspan=1 colspan=2>Sensitivity/PPA</td><td rowspan=1 colspan=2>Specificity/NPA</td></tr><tr><td rowspan=1 colspan=1>TP/TP+FN</td><td rowspan=1 colspan=1>% (95% CI)</td><td rowspan=1 colspan=1>TN/TN+FP</td><td rowspan=1 colspan=1>% (95% CI)</td></tr><tr><td rowspan=7 colspan=1>Enterobacter -non-cloacae complex</td><td rowspan=1 colspan=1>Prospective (Fresh)</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>100 (43.9-100)</td><td rowspan=1 colspan=1>163/164</td><td rowspan=1 colspan=1>99.4 (96.6-99.9)</td></tr><tr><td rowspan=1 colspan=1>Prospective (Frozen)</td><td rowspan=1 colspan=1>5/7</td><td rowspan=1 colspan=1>71.4 (35.9-91.8)</td><td rowspan=1 colspan=1>175/175</td><td rowspan=1 colspan=1>100 (97.9-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective (All)</td><td rowspan=1 colspan=1>8/10</td><td rowspan=1 colspan=1>80.0 (49.0-94.3)</td><td rowspan=1 colspan=1>338/339</td><td rowspan=1 colspan=1>99.7 (98.3-99.9)</td></tr><tr><td rowspan=1 colspan=1>Retrospective</td><td rowspan=1 colspan=1>12/12</td><td rowspan=1 colspan=1>100 (75.8-100)</td><td rowspan=1 colspan=1>565/565</td><td rowspan=1 colspan=1>100 (99.3-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective / Retrospective</td><td rowspan=1 colspan=1>20/22A</td><td rowspan=1 colspan=1>90.9 (72.2-97.5)</td><td rowspan=1 colspan=1>903/904B</td><td rowspan=1 colspan=1>99.9 (99.4-100)</td></tr><tr><td rowspan=1 colspan=1>Contrived</td><td rowspan=1 colspan=1>36/36</td><td rowspan=1 colspan=1>100 (90.4-100)</td><td rowspan=1 colspan=1>741/741</td><td rowspan=1 colspan=1>100 (99.5-100)</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>56/58</td><td rowspan=1 colspan=1>96.6 (88.3-99.0)</td><td rowspan=1 colspan=1>1644/1645</td><td rowspan=1 colspan=1>99.9 (99.7-100)</td></tr></table>

A. A species of the Enterobacter non-cloacae complex was not detected in 2 false negative samples. Standard laboratory procedures identified E. aerogenes and PCR/sequencing detected E. cloacae.,. B. A species of the Enterobacter non-cloacae complex was not detected in the false positive sample using PCR/sequencing.

Table 13: Clinical Performance for Escherichia coli   

<table><tr><td rowspan=2 colspan=1>Target</td><td rowspan=2 colspan=1>Sample Type</td><td rowspan=1 colspan=2>Sensitivity/PPA</td><td rowspan=1 colspan=2>Specificity/NPA</td></tr><tr><td rowspan=1 colspan=1>TP/TP+FN</td><td rowspan=1 colspan=1>% (95% CI)</td><td rowspan=1 colspan=1>TN/TN+FP</td><td rowspan=1 colspan=1>% (95% CI)</td></tr><tr><td rowspan=7 colspan=1>Escherichia coli</td><td rowspan=1 colspan=1>Prospective (Fresh)</td><td rowspan=1 colspan=1>59/60</td><td rowspan=1 colspan=1>98.3 (91.1-99.7)</td><td rowspan=1 colspan=1>106/107</td><td rowspan=1 colspan=1>99.1 (94.9-99.8)</td></tr><tr><td rowspan=1 colspan=1>Prospective (Frozen)</td><td rowspan=1 colspan=1>72/73</td><td rowspan=1 colspan=1>98.6 (92.6-99.8)</td><td rowspan=1 colspan=1>109/109</td><td rowspan=1 colspan=1>100 (96.6-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective (All)</td><td rowspan=1 colspan=1>131/133</td><td rowspan=1 colspan=1>98.5 (94.7-99.6)</td><td rowspan=1 colspan=1>215/216</td><td rowspan=1 colspan=1>99.5 (97.4-99.9)</td></tr><tr><td rowspan=1 colspan=1>Retrospective</td><td rowspan=1 colspan=1>132/140</td><td rowspan=1 colspan=1>94.3 (89.1-97.1)</td><td rowspan=1 colspan=1>435/437</td><td rowspan=1 colspan=1>99.5 (98.3-99.9)</td></tr><tr><td rowspan=1 colspan=1>Prospective / Retrospective</td><td rowspan=1 colspan=1>263/273</td><td rowspan=1 colspan=1>96.3 (93.4-98.0)</td><td rowspan=1 colspan=1>650/653A</td><td rowspan=1 colspan=1>99.5 (98.7-99.8)</td></tr><tr><td rowspan=1 colspan=1>Contrived</td><td rowspan=1 colspan=1>52/52</td><td rowspan=1 colspan=1>100 (93.1-100)</td><td rowspan=1 colspan=1>725/725</td><td rowspan=1 colspan=1>100 (99.5-100)</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>315/325</td><td rowspan=1 colspan=1>96.9 (94.4-98.3)</td><td rowspan=1 colspan=1>1375/1378</td><td rowspan=1 colspan=1>99.8 (99.4-99.9)</td></tr></table>

A. E. coli was detected in 3/3 false positive samples using PCR/sequencing.

Table 14: Clinical Performance for Fusobacterium necrophorum   

<table><tr><td rowspan=2 colspan=1>Target</td><td rowspan=2 colspan=1>Sample Type</td><td rowspan=1 colspan=2>Sensitivity/PPA</td><td rowspan=1 colspan=2>Specificity/NPA</td></tr><tr><td rowspan=1 colspan=1>TP/TP+FN</td><td rowspan=1 colspan=1>% (95% CI)</td><td rowspan=1 colspan=1>TN/TN+FP</td><td rowspan=1 colspan=1>% (95% CI)</td></tr><tr><td rowspan=7 colspan=1>Fusobacteriumnecrophorum</td><td rowspan=1 colspan=1>Prospective (Fresh)</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>167/167</td><td rowspan=1 colspan=1>100 (97.8-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective (Frozen)</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>---</td><td rowspan=1 colspan=1>182/182</td><td rowspan=1 colspan=1>100 (97.9-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective (All)</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>---</td><td rowspan=1 colspan=1>349/349</td><td rowspan=1 colspan=1>100 (98.9-100)</td></tr><tr><td rowspan=1 colspan=1>Retrospective</td><td rowspan=1 colspan=1>1/1</td><td rowspan=1 colspan=1>100 (20.7-100)</td><td rowspan=1 colspan=1>576/576</td><td rowspan=1 colspan=1>100 (99.3-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective / Retrospective</td><td rowspan=1 colspan=1>1/1</td><td rowspan=1 colspan=1>100 (20.7-100)</td><td rowspan=1 colspan=1>925/925</td><td rowspan=1 colspan=1>100 (99.6-100)</td></tr><tr><td rowspan=1 colspan=1>Contrived</td><td rowspan=1 colspan=1>47/48</td><td rowspan=1 colspan=1>97.9 (89.1-99.6)</td><td rowspan=1 colspan=1>729/729</td><td rowspan=1 colspan=1>100 (99.5-100)</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>48/49</td><td rowspan=1 colspan=1>98.0 (89.3-99.6)</td><td rowspan=1 colspan=1>1654/1654</td><td rowspan=1 colspan=1>100 (99.8-100)</td></tr></table>

Table 15: Clinical Performance for Fusobacterium nucleatum   

<table><tr><td rowspan=2 colspan=1>Target</td><td rowspan=2 colspan=1>Sample Type</td><td rowspan=1 colspan=2>Sensitivity/PPA</td><td rowspan=1 colspan=2>Specificity/NPA</td></tr><tr><td rowspan=1 colspan=1>TP/TP+FN</td><td rowspan=1 colspan=1>% (95% CI)</td><td rowspan=1 colspan=1>TN/TN+FP</td><td rowspan=1 colspan=1>% (95% CI)</td></tr><tr><td rowspan=7 colspan=1>Fusobacteriumnucleatum</td><td rowspan=1 colspan=1>Prospective (Fresh)</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>167/167</td><td rowspan=1 colspan=1>100 (97.8-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective (Frozen)</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>182/182</td><td rowspan=1 colspan=1>100 (97.9-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective (All)</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>---</td><td rowspan=1 colspan=1>349/349</td><td rowspan=1 colspan=1>100 (98.9-100)</td></tr><tr><td rowspan=1 colspan=1>Retrospective</td><td rowspan=1 colspan=1>5/5</td><td rowspan=1 colspan=1>100 (56.6-100)</td><td rowspan=1 colspan=1>571/572</td><td rowspan=1 colspan=1>99.8 (99.0-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective / Retrospective</td><td rowspan=1 colspan=1>5/5</td><td rowspan=1 colspan=1>100 (56.6-100)</td><td rowspan=1 colspan=1>920/9214</td><td rowspan=1 colspan=1>99.9 (99.4-100)</td></tr><tr><td rowspan=1 colspan=1>Contrived</td><td rowspan=1 colspan=1>47/47</td><td rowspan=1 colspan=1>100 (92.4-100)</td><td rowspan=1 colspan=1>730/730</td><td rowspan=1 colspan=1>100 (99.5-100)</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>52/52</td><td rowspan=1 colspan=1>100 (93.1-100)</td><td rowspan=1 colspan=1>1650/1651</td><td rowspan=1 colspan=1>99.9 (99.7-100)</td></tr></table>

A. F. nucleatum was detected in the 1/1 false positive sample using PCR/sequencing.

Table 16: Clinical Performance for Haemophilus influenzae   

<table><tr><td rowspan=2 colspan=1>Target</td><td rowspan=2 colspan=1>Sample Type</td><td rowspan=1 colspan=2>Sensitivity/PPA</td><td rowspan=1 colspan=2>Specificity/NPA</td></tr><tr><td rowspan=1 colspan=1>TP/TP+FN</td><td rowspan=1 colspan=1>% (95% CI)</td><td rowspan=1 colspan=1>TN/TN+FP</td><td rowspan=1 colspan=1>% (95% CI)</td></tr><tr><td rowspan=7 colspan=1>Haemophilusinfluenzae</td><td rowspan=1 colspan=1>Prospective (Fresh)</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>100 (43.9-100)</td><td rowspan=1 colspan=1>164/164</td><td rowspan=1 colspan=1>100 (97.7-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective (Frozen)</td><td rowspan=1 colspan=1>4/4</td><td rowspan=1 colspan=1>100 (51.0-100)</td><td rowspan=1 colspan=1>178/178</td><td rowspan=1 colspan=1>100 (97.9-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective (All)</td><td rowspan=1 colspan=1>7/7</td><td rowspan=1 colspan=1>100 (64.6-100)</td><td rowspan=1 colspan=1>342/342</td><td rowspan=1 colspan=1>100 (98.9-100)</td></tr><tr><td rowspan=1 colspan=1>Retrospective</td><td rowspan=1 colspan=1>7/7</td><td rowspan=1 colspan=1>100 (64.6-100)</td><td rowspan=1 colspan=1>570/570</td><td rowspan=1 colspan=1>100 (99.3-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective / Retrospective</td><td rowspan=1 colspan=1>14/14</td><td rowspan=1 colspan=1>100 (78.5-100)</td><td rowspan=1 colspan=1>912/912</td><td rowspan=1 colspan=1>100 (99.6-100)</td></tr><tr><td rowspan=1 colspan=1>Contrived</td><td rowspan=1 colspan=1>41/41</td><td rowspan=1 colspan=1>100 (91.4-100)</td><td rowspan=1 colspan=1>736/736</td><td rowspan=1 colspan=1>100 (99.5-100)</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>55/55</td><td rowspan=1 colspan=1>100 (93.5-100)</td><td rowspan=1 colspan=1>1648/1648</td><td rowspan=1 colspan=1>100 (99.8-100)</td></tr></table>

Table 17: Clinical Performance for Klebsiella oxytoca   

<table><tr><td rowspan=2 colspan=1>Target</td><td rowspan=2 colspan=1>Sample Type</td><td rowspan=1 colspan=2>Sensitivity/PPA</td><td rowspan=1 colspan=2>Specificity/NPA</td></tr><tr><td rowspan=1 colspan=1>TP/TP+FN</td><td rowspan=1 colspan=1>% (95% CI)</td><td rowspan=1 colspan=1>TN/TN+FP</td><td rowspan=1 colspan=1>% (95% CI)</td></tr><tr><td rowspan=7 colspan=1>Klebsiella oxytoca</td><td rowspan=1 colspan=1>Prospective (Fresh)</td><td rowspan=1 colspan=1>4/6</td><td rowspan=1 colspan=1>66.7 (30.0-90.3)</td><td rowspan=1 colspan=1>160/161</td><td rowspan=1 colspan=1>99.4 (96.6-99.9)</td></tr><tr><td rowspan=1 colspan=1>Prospective (Frozen)</td><td rowspan=1 colspan=1>7/7</td><td rowspan=1 colspan=1>100 (64.6-100)</td><td rowspan=1 colspan=1>175/175</td><td rowspan=1 colspan=1>100 (97.9-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective (All)</td><td rowspan=1 colspan=1>11/13</td><td rowspan=1 colspan=1>84.6 (57.8-95.7)</td><td rowspan=1 colspan=1>335/336</td><td rowspan=1 colspan=1>99.7 (98.3-99.9)</td></tr><tr><td rowspan=1 colspan=1>Retrospective</td><td rowspan=1 colspan=1>29/34</td><td rowspan=1 colspan=1>85.3 (69.9-93.6)</td><td rowspan=1 colspan=1>541/543</td><td rowspan=1 colspan=1>99.6 (98.7-99.9)</td></tr><tr><td rowspan=1 colspan=1>Prospective / Retrospective</td><td rowspan=1 colspan=1>40/47A</td><td rowspan=1 colspan=1>85.1 (72.3-92.6)</td><td rowspan=1 colspan=1>876/879B</td><td rowspan=1 colspan=1>99.7 (99.0-99.9)</td></tr><tr><td rowspan=1 colspan=1>Contrived</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>100 (83.9-100)</td><td rowspan=1 colspan=1>757/757</td><td rowspan=1 colspan=1>100 (99.5-100)</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>60/67</td><td rowspan=1 colspan=1>89.6 (80.0-94.8)</td><td rowspan=1 colspan=1>1633/1636</td><td rowspan=1 colspan=1>99.8 (99.5-99.9)</td></tr></table>

A. $K .$ oxytoca was not detected in 2 false negative samples using PCR/sequencing, but 16S sequencing instead detected Raoultella ornithinolytica and Raoultella planticola, which were not identified by standard laboratory procedures. B. Klebsiella oxytoca was detected in 3/3 false positive samples using PCR/sequencing.

Table 18: Clinical Performance for Klebsiella pneumoniae group   

<table><tr><td rowspan=2 colspan=1>Target</td><td rowspan=2 colspan=1>Sample Type</td><td rowspan=1 colspan=2>Sensitivity/PPA</td><td rowspan=1 colspan=2>Specificity/NPA</td></tr><tr><td rowspan=1 colspan=1>TP/TP+FN</td><td rowspan=1 colspan=1>% (95% CI)</td><td rowspan=1 colspan=1>TN/TN+FP</td><td rowspan=1 colspan=1>% (95% CI)</td></tr><tr><td rowspan=7 colspan=1>Klebsiella pneumoniaegroup</td><td rowspan=1 colspan=1>Prospective (Fresh)</td><td rowspan=1 colspan=1>29/30</td><td rowspan=1 colspan=1>96.7 (83.3-99.4)</td><td rowspan=1 colspan=1>136/137</td><td rowspan=1 colspan=1>99.3 (96.0-99.9)</td></tr><tr><td rowspan=1 colspan=1>Prospective (Frozen)</td><td rowspan=1 colspan=1>29/31</td><td rowspan=1 colspan=1>93.5 (79.3-98.2)</td><td rowspan=1 colspan=1>151/151</td><td rowspan=1 colspan=1>100 (97.5-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective (All)</td><td rowspan=1 colspan=1>58/61</td><td rowspan=1 colspan=1>95.1 (86.5-98.3)</td><td rowspan=1 colspan=1>287/288</td><td rowspan=1 colspan=1>99.7 (98.1-99.9)</td></tr><tr><td rowspan=1 colspan=1>Retrospective</td><td rowspan=1 colspan=1>106/108</td><td rowspan=1 colspan=1>98.1 (93.5-99.5)</td><td rowspan=1 colspan=1>466/469</td><td rowspan=1 colspan=1>99.4 (98.1-99.8)</td></tr><tr><td rowspan=1 colspan=1>Prospective / Retrospective</td><td rowspan=1 colspan=1>164/169A</td><td rowspan=1 colspan=1>97.0 (93.3-98.7)</td><td rowspan=1 colspan=1>753/757B</td><td rowspan=1 colspan=1>99.5 (98.6-99.8)</td></tr><tr><td rowspan=1 colspan=1>Contrived</td><td rowspan=1 colspan=1>72/72</td><td rowspan=1 colspan=1>100 (94.9-100)</td><td rowspan=1 colspan=1>705/705</td><td rowspan=1 colspan=1>100 (99.5-100)</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>236/241</td><td rowspan=1 colspan=1>97.9 (95.2-99.1)</td><td rowspan=1 colspan=1>1458/1462</td><td rowspan=1 colspan=1>99.7 (99.3-99.9)</td></tr></table>

A. K. pneumoniae was not detected in 1 false negative sample, but PCR/sequencing and MALDI-TOF instead detected $K$ . oxytoca, which was not identified by standard laboratory procedures. B. K. pneumoniae was detected in 4/4 false positive samples using PCR/sequencing.

Table 19: Clinical Performance for Morganella morganii   

<table><tr><td rowspan=2 colspan=1>Target</td><td rowspan=2 colspan=1>Sample Type</td><td rowspan=1 colspan=2>Sensitivity/PPA</td><td rowspan=1 colspan=2>Specificity/NPA</td></tr><tr><td rowspan=1 colspan=1>TP/TP+FN</td><td rowspan=1 colspan=1>% (95% CI)</td><td rowspan=1 colspan=1>TN/TN+FP</td><td rowspan=1 colspan=1>% (95% CI)</td></tr><tr><td rowspan=7 colspan=1>Morganella morganii</td><td rowspan=1 colspan=1>Prospective (Fresh)</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>100 (43.9-100)</td><td rowspan=1 colspan=1>164/164</td><td rowspan=1 colspan=1>100 (97.7-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective (Frozen)</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>182/182</td><td rowspan=1 colspan=1>100 (97.9-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective (All)</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>100 (43.9-100)</td><td rowspan=1 colspan=1>346/346</td><td rowspan=1 colspan=1>100 (98.9-100)</td></tr><tr><td rowspan=1 colspan=1>Retrospective</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>100 (72.2-100)</td><td rowspan=1 colspan=1>566/567</td><td rowspan=1 colspan=1>99.8 (99.0-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective / Retrospective</td><td rowspan=1 colspan=1>13/13</td><td rowspan=1 colspan=1>100 (77.2-100)</td><td rowspan=1 colspan=1>912/9134</td><td rowspan=1 colspan=1>99.9 (99.4-100)</td></tr><tr><td rowspan=1 colspan=1>Contrived</td><td rowspan=1 colspan=1>49/49</td><td rowspan=1 colspan=1>100 (92.7-100)</td><td rowspan=1 colspan=1>728/728</td><td rowspan=1 colspan=1>100 (99.5-100)</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>62/62</td><td rowspan=1 colspan=1>100 (94.2-100)</td><td rowspan=1 colspan=1>1640/1641</td><td rowspan=1 colspan=1>99.9 (99.7-100)</td></tr></table>

A. M. morganii was detected in $1 / 1$ false positive clinical samples using PCR/sequencing.

Table 20: Clinical Performance for Neisseria meningitidis   

<table><tr><td rowspan=2 colspan=1>Target</td><td rowspan=2 colspan=1>Sample Type</td><td rowspan=1 colspan=2>Sensitivity/PPA</td><td rowspan=1 colspan=2>Specificity/NPA</td></tr><tr><td rowspan=1 colspan=1>TP/TP+FN</td><td rowspan=1 colspan=1>% (95% CI)</td><td rowspan=1 colspan=1>TN/TN+FP</td><td rowspan=1 colspan=1>% (95% CI)</td></tr><tr><td rowspan=7 colspan=1>Neisseria meningitidis</td><td rowspan=1 colspan=1>Prospective (Fresh)</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>---</td><td rowspan=1 colspan=1>167/167</td><td rowspan=1 colspan=1>100 (97.8-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective (Frozen)</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>182/182</td><td rowspan=1 colspan=1>100 (97.9-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective (All)</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>349/349</td><td rowspan=1 colspan=1>100 (98.9-100)</td></tr><tr><td rowspan=1 colspan=1>Retrospective</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>576/577</td><td rowspan=1 colspan=1>99.8 (99.0-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective / Retrospective</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>925/926A</td><td rowspan=1 colspan=1>99.9 (99.4-100)</td></tr><tr><td rowspan=1 colspan=1>Contrived</td><td rowspan=1 colspan=1>44/44</td><td rowspan=1 colspan=1>100 (92.0-100)</td><td rowspan=1 colspan=1>733/733</td><td rowspan=1 colspan=1>100 (99.5-100)</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>44/44</td><td rowspan=1 colspan=1>100 (92.0-100)</td><td rowspan=1 colspan=1>1658/1659</td><td rowspan=1 colspan=1>99.9 (99.7-100)</td></tr></table>

A. N. meningitidis was not detected in the false positive sample using PCR/sequencing.

Table 21: Clinical Performance for Proteus   

<table><tr><td rowspan=2 colspan=1>Target</td><td rowspan=2 colspan=1>Sample Type</td><td rowspan=1 colspan=2>Sensitivity/PPA</td><td rowspan=1 colspan=2>Specificity/NPA</td></tr><tr><td rowspan=1 colspan=1>TP/TP+FN</td><td rowspan=1 colspan=1>% (95% CI)</td><td rowspan=1 colspan=1>TN/TN+FP</td><td rowspan=1 colspan=1>% (95% CI)</td></tr><tr><td rowspan=7 colspan=1>Proteus</td><td rowspan=1 colspan=1>Prospective (Fresh)</td><td rowspan=1 colspan=1>7/8</td><td rowspan=1 colspan=1>87.5 (52.9-97.8)</td><td rowspan=1 colspan=1>159/159</td><td rowspan=1 colspan=1>100 (97.6-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective (Frozen)</td><td rowspan=1 colspan=1>15/15</td><td rowspan=1 colspan=1>100 (79.6-100)</td><td rowspan=1 colspan=1>167/167</td><td rowspan=1 colspan=1>100 (97.8-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective (All)</td><td rowspan=1 colspan=1>22/23</td><td rowspan=1 colspan=1>95.7 (79.0-99.2)</td><td rowspan=1 colspan=1>326/326</td><td rowspan=1 colspan=1>100 (98.8-100)</td></tr><tr><td rowspan=1 colspan=1>Retrospective</td><td rowspan=1 colspan=1>54/55</td><td rowspan=1 colspan=1>98.2 (90.4-99.7)</td><td rowspan=1 colspan=1>522/522</td><td rowspan=1 colspan=1>100 (99.3-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective / Retrospective</td><td rowspan=1 colspan=1>76/78</td><td rowspan=1 colspan=1>97.4 (91.1-99.3)</td><td rowspan=1 colspan=1>848/848</td><td rowspan=1 colspan=1>100 (99.5-100)</td></tr><tr><td rowspan=1 colspan=1>Contrived</td><td rowspan=1 colspan=1>9/9</td><td rowspan=1 colspan=1>100 (70.1-100)</td><td rowspan=1 colspan=1>768/768</td><td rowspan=1 colspan=1>100 (99.5-100)</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>85/87</td><td rowspan=1 colspan=1>97.7 (92.0-99.4)</td><td rowspan=1 colspan=1>1616/1616</td><td rowspan=1 colspan=1>100 (99.8-100)</td></tr></table>

Table 22: Clinical Performance for Proteus mirabilis   

<table><tr><td rowspan=2 colspan=1>Target</td><td rowspan=2 colspan=1>Sample Type</td><td rowspan=1 colspan=2>Sensitivity/PPA</td><td rowspan=1 colspan=2>Specificity/NPA</td></tr><tr><td rowspan=1 colspan=1>TP/TP+FN</td><td rowspan=1 colspan=1>% (95% CI)</td><td rowspan=1 colspan=1>TN/TN+FP</td><td rowspan=1 colspan=1>% (95% CI)</td></tr><tr><td rowspan=7 colspan=1>Proteus mirabilis</td><td rowspan=1 colspan=1>Prospective (Fresh)</td><td rowspan=1 colspan=1>7/8</td><td rowspan=1 colspan=1>87.5 (52.9-97.8)</td><td rowspan=1 colspan=1>159/159</td><td rowspan=1 colspan=1>100 (97.6-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective (Frozen)</td><td rowspan=1 colspan=1>15/15</td><td rowspan=1 colspan=1>100 (79.6-100)</td><td rowspan=1 colspan=1>167/167</td><td rowspan=1 colspan=1>100 (97.8-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective (All)</td><td rowspan=1 colspan=1>22/23</td><td rowspan=1 colspan=1>95.7 (79.0-99.2)</td><td rowspan=1 colspan=1>326/326</td><td rowspan=1 colspan=1>100 (98.8-100)</td></tr><tr><td rowspan=1 colspan=1>Retrospective</td><td rowspan=1 colspan=1>50/51</td><td rowspan=1 colspan=1>98.0 (89.7-99.7)</td><td rowspan=1 colspan=1>526/526</td><td rowspan=1 colspan=1>100 (99.3-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective / Retrospective</td><td rowspan=1 colspan=1>72/74</td><td rowspan=1 colspan=1>97.3 (90.7-99.3)</td><td rowspan=1 colspan=1>852/852</td><td rowspan=1 colspan=1>100 (99.6-100)</td></tr><tr><td rowspan=1 colspan=1>Contrived</td><td rowspan=1 colspan=1>9/9</td><td rowspan=1 colspan=1>100 (70.1-100)</td><td rowspan=1 colspan=1>768/768</td><td rowspan=1 colspan=1>100 (99.5-100)</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>81/83</td><td rowspan=1 colspan=1>97.6 (91.6-99.3)</td><td rowspan=1 colspan=1>1620/1620</td><td rowspan=1 colspan=1>100 (99.8-100)</td></tr></table>

Table 23: Clinical Performance for Pseudomonas aeruginosa   

<table><tr><td rowspan=2 colspan=1>Target</td><td rowspan=2 colspan=1>Sample Type</td><td rowspan=1 colspan=2>Sensitivity/PPA</td><td rowspan=1 colspan=2>Specificity/NPA</td></tr><tr><td rowspan=1 colspan=1>TP/TP+FN</td><td rowspan=1 colspan=1>% (95% CI)</td><td rowspan=1 colspan=1>TN/TN+FP</td><td rowspan=1 colspan=1>% (95% CI)</td></tr><tr><td rowspan=7 colspan=1>Pseudomonasaeruginosa</td><td rowspan=1 colspan=1>Prospective (Fresh)</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>100 (72.2-100)</td><td rowspan=1 colspan=1>157/157</td><td rowspan=1 colspan=1>100 (97.6-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective (Frozen)</td><td rowspan=1 colspan=1>17/18</td><td rowspan=1 colspan=1>94.4 (74.2-99.0)</td><td rowspan=1 colspan=1>163/164</td><td rowspan=1 colspan=1>99.4 (96.6-99.9)</td></tr><tr><td rowspan=1 colspan=1>Prospective (All)</td><td rowspan=1 colspan=1>27/28</td><td rowspan=1 colspan=1>96.4 (82.3-99.4)</td><td rowspan=1 colspan=1>320/321</td><td rowspan=1 colspan=1>99.7 (98.3-99.9)</td></tr><tr><td rowspan=1 colspan=1>Retrospective</td><td rowspan=1 colspan=1>56/60</td><td rowspan=1 colspan=1>93.3 (84.1-97.4)</td><td rowspan=1 colspan=1>514/517</td><td rowspan=1 colspan=1>99.4 (98.3-99.8)</td></tr><tr><td rowspan=1 colspan=1>Prospective / Retrospective</td><td rowspan=1 colspan=1>83/88</td><td rowspan=1 colspan=1>94.3 (87.4-97.5)</td><td rowspan=1 colspan=1>834/8384</td><td rowspan=1 colspan=1>99.5 (98.8-99.8)</td></tr><tr><td rowspan=1 colspan=1>Contrived</td><td rowspan=1 colspan=1>32/32</td><td rowspan=1 colspan=1>100 (89.3-100)</td><td rowspan=1 colspan=1>745/745</td><td rowspan=1 colspan=1>100 (99.5-100)</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>115/120</td><td rowspan=1 colspan=1>95.8 (90.6-98.2)</td><td rowspan=1 colspan=1>1579/1583</td><td rowspan=1 colspan=1>99.7 (99.4-99.9)</td></tr></table>

A. P. aeruginosa was detected in 2/4 false positive samples using PCR/sequencing.

Table 24: Clinical Performance for Salmonella   

<table><tr><td rowspan=2 colspan=1>Target</td><td rowspan=2 colspan=1>Sample Type</td><td rowspan=1 colspan=2>Sensitivity/PPA</td><td rowspan=1 colspan=2>Specificity/NPA</td></tr><tr><td rowspan=1 colspan=1>TP/TP+FN</td><td rowspan=1 colspan=1>% (95% CI)</td><td rowspan=1 colspan=1>TN/TN+FP</td><td rowspan=1 colspan=1>% (95% CI)</td></tr><tr><td rowspan=7 colspan=1>Salmonella</td><td rowspan=1 colspan=1>Prospective (Fresh)</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>100 (34.2-100)</td><td rowspan=1 colspan=1>165/165</td><td rowspan=1 colspan=1>100 (97.7-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective (Frozen)</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>---</td><td rowspan=1 colspan=1>182/182</td><td rowspan=1 colspan=1>100 (97.9-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective (All)</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>100 (34.2-100)</td><td rowspan=1 colspan=1>347/347</td><td rowspan=1 colspan=1>100 (98.9-100)</td></tr><tr><td rowspan=1 colspan=1>Retrospective</td><td rowspan=1 colspan=1>18/19</td><td rowspan=1 colspan=1>94.7 (75.4-99.1)</td><td rowspan=1 colspan=1>558/558</td><td rowspan=1 colspan=1>100 (99.3-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective / Retrospective</td><td rowspan=1 colspan=1>20/21A</td><td rowspan=1 colspan=1>95.2 (77.3-99.2)</td><td rowspan=1 colspan=1>905/905</td><td rowspan=1 colspan=1>100 (99.6-100)</td></tr><tr><td rowspan=1 colspan=1>Contrived</td><td rowspan=1 colspan=1>34/35</td><td rowspan=1 colspan=1>97.1 (85.5-99.5)</td><td rowspan=1 colspan=1>742/742</td><td rowspan=1 colspan=1>100 (99.5-100)</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>54/56</td><td rowspan=1 colspan=1>96.4 (87.9-99.0)</td><td rowspan=1 colspan=1>1647/1647</td><td rowspan=1 colspan=1>100 (99.8-100)</td></tr></table>

A. Salmonella was not detected in 1 false negative sample, but PCR/sequencing instead detected $E .$ . coli, which was not identified by standard laboratory procedures.

Table 25: Clinical Performance for Serratia   

<table><tr><td rowspan=2 colspan=1>Target</td><td rowspan=2 colspan=1>Sample Type</td><td rowspan=1 colspan=2>Sensitivity/PPA</td><td rowspan=1 colspan=2>Specificity/NPA</td></tr><tr><td rowspan=1 colspan=1>TP/TP+FN</td><td rowspan=1 colspan=1>% (95% CI)</td><td rowspan=1 colspan=1>TN/TN+FP</td><td rowspan=1 colspan=1>% (95% CI)</td></tr><tr><td rowspan=7 colspan=1>Serratia</td><td rowspan=1 colspan=1>Prospective (Fresh)</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>100 (61.0-100)</td><td rowspan=1 colspan=1>161/161</td><td rowspan=1 colspan=1>100 (97.7-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective (Frozen)</td><td rowspan=1 colspan=1>4/4</td><td rowspan=1 colspan=1>100 (51.0-100)</td><td rowspan=1 colspan=1>178/178</td><td rowspan=1 colspan=1>100 (97.9-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective (All)</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>100 (72.2-100)</td><td rowspan=1 colspan=1>339/339</td><td rowspan=1 colspan=1>100 (98.9-100)</td></tr><tr><td rowspan=1 colspan=1>Retrospective</td><td rowspan=1 colspan=1>34/34</td><td rowspan=1 colspan=1>100 (89.8-100)</td><td rowspan=1 colspan=1>542/543</td><td rowspan=1 colspan=1>99.8 (99.0-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective / Retrospective</td><td rowspan=1 colspan=1>44/44</td><td rowspan=1 colspan=1>100 (92.0-100)</td><td rowspan=1 colspan=1>881/882A</td><td rowspan=1 colspan=1>99.9 (99.4-100)</td></tr><tr><td rowspan=1 colspan=1>Contrived</td><td rowspan=1 colspan=1>36/36</td><td rowspan=1 colspan=1>100 (90.4-100)</td><td rowspan=1 colspan=1>741/741</td><td rowspan=1 colspan=1>100 (99.5-100)</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>80/80</td><td rowspan=1 colspan=1>100 (95.4-100)</td><td rowspan=1 colspan=1>1622/1623</td><td rowspan=1 colspan=1>99.9 (99.7-100)</td></tr></table>

A. S. marcescens was detected in the 1/1 false positive sample using PCR/sequencing.

Table 26: Clinical Performance for Serratia marcescens   

<table><tr><td rowspan=2 colspan=1>Target</td><td rowspan=2 colspan=1>Sample Type</td><td rowspan=1 colspan=2>Sensitivity/PPA</td><td rowspan=1 colspan=2>Specificity/NPA</td></tr><tr><td rowspan=1 colspan=1>TP/TP+FN</td><td rowspan=1 colspan=1>% (95% CI)</td><td rowspan=1 colspan=1>TN/TN+FP</td><td rowspan=1 colspan=1>% (95% CI)</td></tr><tr><td rowspan=7 colspan=1>Serratia marcescens</td><td rowspan=1 colspan=1>Prospective (Fresh)</td><td rowspan=1 colspan=1>5/5</td><td rowspan=1 colspan=1>100 (56.6-100)</td><td rowspan=1 colspan=1>162/162</td><td rowspan=1 colspan=1>100 (97.7-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective (Frozen)</td><td rowspan=1 colspan=1>4/4</td><td rowspan=1 colspan=1>100 (51.0-100)</td><td rowspan=1 colspan=1>178/178</td><td rowspan=1 colspan=1>100 (97.9-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective (All)</td><td rowspan=1 colspan=1>9/9</td><td rowspan=1 colspan=1>100 (70.1-100)</td><td rowspan=1 colspan=1>340/340</td><td rowspan=1 colspan=1>100 (98.9-100)</td></tr><tr><td rowspan=1 colspan=1>Retrospective</td><td rowspan=1 colspan=1>34/34</td><td rowspan=1 colspan=1>100 (89.8-100)</td><td rowspan=1 colspan=1>542/543</td><td rowspan=1 colspan=1>99.8 (99.0-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective / Retrospective</td><td rowspan=1 colspan=1>43/43</td><td rowspan=1 colspan=1>100 (91.8-100)</td><td rowspan=1 colspan=1>882/8834</td><td rowspan=1 colspan=1>99.9 (99.4-100)</td></tr><tr><td rowspan=1 colspan=1>Contrived</td><td rowspan=1 colspan=1>19/19</td><td rowspan=1 colspan=1>100 (83.2-100)</td><td rowspan=1 colspan=1>758/758</td><td rowspan=1 colspan=1>100 (99.5-100)</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>62/62</td><td rowspan=1 colspan=1>100 (94.2-100)</td><td rowspan=1 colspan=1>1640/1641</td><td rowspan=1 colspan=1>99.9 (99.7-100)</td></tr></table>

A. S. marcescens was detected in the 1/1 false positive sample using PCR/sequencing.

Table 27: Clinical Performance for Stenotrophomonas maltophilia   

<table><tr><td rowspan=2 colspan=1>Target</td><td rowspan=2 colspan=1>Sample Type</td><td rowspan=1 colspan=2>Sensitivity/PPA</td><td rowspan=1 colspan=2>Specificity/NPA</td></tr><tr><td rowspan=1 colspan=1>TP/TP+FN</td><td rowspan=1 colspan=1>% (95% CI)</td><td rowspan=1 colspan=1>TN/TN+FP</td><td rowspan=1 colspan=1>% (95% CI)</td></tr><tr><td rowspan=7 colspan=1>Stenotrophomonasmaltophilia</td><td rowspan=1 colspan=1>Prospective (Fresh)</td><td rowspan=1 colspan=1>2/3</td><td rowspan=1 colspan=1>66.7 (20.8-93.9)</td><td rowspan=1 colspan=1>164/164</td><td rowspan=1 colspan=1>100 (97.7-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective (Frozen)</td><td rowspan=1 colspan=1>1/1</td><td rowspan=1 colspan=1>100 (20.7-100)</td><td rowspan=1 colspan=1>181/181</td><td rowspan=1 colspan=1>100 (97.9-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective (All)</td><td rowspan=1 colspan=1>3/4</td><td rowspan=1 colspan=1>75.0 (30.1-95.4)</td><td rowspan=1 colspan=1>345/345</td><td rowspan=1 colspan=1>100 (98.9-100)</td></tr><tr><td rowspan=1 colspan=1>Retrospective</td><td rowspan=1 colspan=1>8/10</td><td rowspan=1 colspan=1>80.0 (49.0-94.3)</td><td rowspan=1 colspan=1>566/567</td><td rowspan=1 colspan=1>99.8 (99.0-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective / Retrospective</td><td rowspan=1 colspan=1>11/14</td><td rowspan=1 colspan=1>78.6 (52.4-92.4)</td><td rowspan=1 colspan=1>911/912A</td><td rowspan=1 colspan=1>99.9 (99.4-100)</td></tr><tr><td rowspan=1 colspan=1>Contrived</td><td rowspan=1 colspan=1>36/36</td><td rowspan=1 colspan=1>100 (90.4-100)</td><td rowspan=1 colspan=1>741/741</td><td rowspan=1 colspan=1>100 (99.5-100)</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>47/50</td><td rowspan=1 colspan=1>94.0 (83.8-97.9)</td><td rowspan=1 colspan=1>1652/1653</td><td rowspan=1 colspan=1>99.9 (99.7-100)</td></tr></table>

A. S. maltophilia was detected in the 1/1 false positive sample using PCR/sequencing.

Table 28: Clinical Performance for CTX-M   

<table><tr><td rowspan=2 colspan=1>Target</td><td rowspan=2 colspan=1>Sample Type</td><td rowspan=1 colspan=2>Sensitivity/PPA</td><td rowspan=1 colspan=2>Specificity/NPA</td></tr><tr><td rowspan=1 colspan=1>TP/TP+FN</td><td rowspan=1 colspan=1>% (95% CI)</td><td rowspan=1 colspan=1>TN/TN+FP</td><td rowspan=1 colspan=1>% (95% CI)</td></tr><tr><td rowspan=7 colspan=1>CTX-M</td><td rowspan=1 colspan=1>Prospective (Fresh)</td><td rowspan=1 colspan=1>10/13</td><td rowspan=1 colspan=1>76.9 (49.7-91.8)</td><td rowspan=1 colspan=1>127/127</td><td rowspan=1 colspan=1>100 (97.1-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective (Frozen)</td><td rowspan=1 colspan=1>12/16</td><td rowspan=1 colspan=1>75.0 (50.5-89.8)</td><td rowspan=1 colspan=1>144/144</td><td rowspan=1 colspan=1>100 (97.4-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective (All)</td><td rowspan=1 colspan=1>22/29</td><td rowspan=1 colspan=1>75.9 (57.9-87.8)</td><td rowspan=1 colspan=1>271/271</td><td rowspan=1 colspan=1>100 (98.6-100)</td></tr><tr><td rowspan=1 colspan=1>Retrospective</td><td rowspan=1 colspan=1>52/56</td><td rowspan=1 colspan=1>92.9 (83.0-97.2)</td><td rowspan=1 colspan=1>483/483</td><td rowspan=1 colspan=1>100 (99.2-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective / Retrospective</td><td rowspan=1 colspan=1>74/85A</td><td rowspan=1 colspan=1>87.1 (78.3-92.6)</td><td rowspan=1 colspan=1>754/754</td><td rowspan=1 colspan=1>100 (99.5-100)</td></tr><tr><td rowspan=1 colspan=1>Contrived</td><td rowspan=1 colspan=1>75/75</td><td rowspan=1 colspan=1>100 (95.1-100)</td><td rowspan=1 colspan=1>437/437</td><td rowspan=1 colspan=1>100 (99.1-100)</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>149/160</td><td rowspan=1 colspan=1>93.1 (88.1-96.1)</td><td rowspan=1 colspan=1>1191/1191</td><td rowspan=1 colspan=1>100 (99.7-100)</td></tr></table>

A. $\mathrm { I n } 3 / 1 1$ false negative samples, CTX-M signal was above the threshold for detection; however, an associated organism was not detected by the BCID-GN Panel and the CTX-M target was reported as $\mathbf { \hat { N } } / \mathbf { A } ^ { \prime }$ . Further testing of the 8/11 remaining false negative samples indicated that 7 of the 8 samples may have been contaminated during the original comparator extraction process and misidentified as having CTX-M present. Specifically, results for 7 of the 8 samples were negative for CTX-M from the following additional testing: 1) $\mathrm { \ q P C R }$ testing of 2 repeat extractions from the original sample, 2) $\mathrm { { q P C R } }$ testing of an extraction from the isolate, and 3) testing of the original sample with an FDA-cleared multiplex assay. For these 7 samples, the originally extracted sample was re-tested with qPCR and CTX-M was again detected, suggesting contamination during the original extraction process. The remaining 8th sample was positive for CTX-M from the repeat extractions, negative for CTX-M from the isolate, and negative for CTX-M when tested with an FDA-cleared multiplex assay. These inconsistent detection results suggest the 8th sample may be a true low-copy CTX-M positive sample.

Table 29: Clinical Performance for IMP   

<table><tr><td rowspan=2 colspan=1>Target</td><td rowspan=2 colspan=1>Sample Type</td><td rowspan=1 colspan=2>Sensitivity/PPA</td><td rowspan=1 colspan=2>Specificity/NPA</td></tr><tr><td rowspan=1 colspan=1>TP/TP+FN</td><td rowspan=1 colspan=1>% (95% CI)</td><td rowspan=1 colspan=1>TN/TN+FP</td><td rowspan=1 colspan=1>% (95% CI)</td></tr><tr><td rowspan=7 colspan=1>IMP</td><td rowspan=1 colspan=1>Prospective (Fresh)</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>138/138</td><td rowspan=1 colspan=1>100 (97.3-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective (Frozen)</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>159/159</td><td rowspan=1 colspan=1>100 (97.6-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective (All)</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>297/297</td><td rowspan=1 colspan=1>100 (98.7-100)</td></tr><tr><td rowspan=1 colspan=1>Retrospective</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>---</td><td rowspan=1 colspan=1>532/532</td><td rowspan=1 colspan=1>100 (99.3-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective / Retrospective</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>829/829</td><td rowspan=1 colspan=1>100 (99.5-100)</td></tr><tr><td rowspan=1 colspan=1>Contrived</td><td rowspan=1 colspan=1>40/40</td><td rowspan=1 colspan=1>100 (91.2-100)</td><td rowspan=1 colspan=1>436/436</td><td rowspan=1 colspan=1>100 (99.1-100)</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>40/40</td><td rowspan=1 colspan=1>100 (91.2-100)</td><td rowspan=1 colspan=1>1265/1265</td><td rowspan=1 colspan=1>100 (99.7-100)</td></tr></table>

Table 30: Clinical Performance for KPC   

<table><tr><td rowspan=2 colspan=1>Target</td><td rowspan=2 colspan=1>Sample Type</td><td rowspan=1 colspan=2>Sensitivity/PPA</td><td rowspan=1 colspan=2>Specificity/NPA</td></tr><tr><td rowspan=1 colspan=1>TP/TP+FN</td><td rowspan=1 colspan=1>% (95% CI)</td><td rowspan=1 colspan=1>TN/TN+FP</td><td rowspan=1 colspan=1>% (95% CI)</td></tr><tr><td rowspan=7 colspan=1>KPC</td><td rowspan=1 colspan=1>Prospective (Fresh)</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>100 (34.2-100)</td><td rowspan=1 colspan=1>136/136</td><td rowspan=1 colspan=1>100 (97.3-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective (Frozen)</td><td rowspan=1 colspan=1>1/1</td><td rowspan=1 colspan=1>100 (20.7-100)</td><td rowspan=1 colspan=1>158/158</td><td rowspan=1 colspan=1>100 (97.6-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective (All)</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>100 (43.9-100)</td><td rowspan=1 colspan=1>294/294</td><td rowspan=1 colspan=1>100 (98.7-100)</td></tr><tr><td rowspan=1 colspan=1>Retrospective</td><td rowspan=1 colspan=1>4/5</td><td rowspan=1 colspan=1>80.0 (37.6-96.4)</td><td rowspan=1 colspan=1>527/528</td><td rowspan=1 colspan=1>99.8 (98.9-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective / Retrospective</td><td rowspan=1 colspan=1>7/8</td><td rowspan=1 colspan=1>87.5 (52.9-97.8)</td><td rowspan=1 colspan=1>821/822</td><td rowspan=1 colspan=1>99.9 (99.3-100)</td></tr><tr><td rowspan=1 colspan=1>Contrived</td><td rowspan=1 colspan=1>44/44</td><td rowspan=1 colspan=1>100 (92.0-100)</td><td rowspan=1 colspan=1>477/477</td><td rowspan=1 colspan=1>100 (99.2-100)</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>51/52</td><td rowspan=1 colspan=1>98.1 (89.9-99.7)</td><td rowspan=1 colspan=1>1298/1299</td><td rowspan=1 colspan=1>99.9 (99.6-100)</td></tr></table>

Table 31: Clinical Performance for NDM   

<table><tr><td rowspan=2 colspan=1>Target</td><td rowspan=2 colspan=1>Sample Type</td><td rowspan=1 colspan=2>Sensitivity/PPA</td><td rowspan=1 colspan=2>Specificity/NPA</td></tr><tr><td rowspan=1 colspan=1>TP/TP+FN</td><td rowspan=1 colspan=1>% (95% CI)</td><td rowspan=1 colspan=1>TN/TN+FP</td><td rowspan=1 colspan=1>% (95% CI)</td></tr><tr><td rowspan=7 colspan=1>NDM</td><td rowspan=1 colspan=1>Prospective (Fresh)</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>138/138</td><td rowspan=1 colspan=1>100 (97.3-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective (Frozen)</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>159/159</td><td rowspan=1 colspan=1>100 (97.6-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective (All)</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>297/297</td><td rowspan=1 colspan=1>100 (98.7-100)</td></tr><tr><td rowspan=1 colspan=1>Retrospective</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>---</td><td rowspan=1 colspan=1>532/532</td><td rowspan=1 colspan=1>100 (99.3-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective / Retrospective</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>829/829</td><td rowspan=1 colspan=1>100 (99.5-100)</td></tr><tr><td rowspan=1 colspan=1>Contrived</td><td rowspan=1 colspan=1>54/54</td><td rowspan=1 colspan=1>100 (93.4-100)</td><td rowspan=1 colspan=1>422/422</td><td rowspan=1 colspan=1>100 (99.1-100)</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>54/54</td><td rowspan=1 colspan=1>100 (93.4-100)</td><td rowspan=1 colspan=1>1251/1251</td><td rowspan=1 colspan=1>100 (99.7-100)</td></tr></table>

Table 32: Clinical Performance for OXA   

<table><tr><td rowspan=2 colspan=1>Target</td><td rowspan=2 colspan=1>Sample Type</td><td rowspan=1 colspan=2>Sensitivity/PPA</td><td rowspan=1 colspan=2>Specificity/NPA</td></tr><tr><td rowspan=1 colspan=1>TP/TP+FN</td><td rowspan=1 colspan=1>% (95% CI)</td><td rowspan=1 colspan=1>TN/TN+FP</td><td rowspan=1 colspan=1>% (95% CI)</td></tr><tr><td rowspan=7 colspan=1>OXA</td><td rowspan=1 colspan=1>Prospective (Fresh)</td><td rowspan=1 colspan=1>0/1</td><td rowspan=1 colspan=1>0.0 (0.0-79.3)</td><td rowspan=1 colspan=1>137/137</td><td rowspan=1 colspan=1>100 (97.3-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective (Frozen)</td><td rowspan=1 colspan=1>1/1</td><td rowspan=1 colspan=1>100 (20.7-100)</td><td rowspan=1 colspan=1>158/158</td><td rowspan=1 colspan=1>100 (97.6-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective (All)</td><td rowspan=1 colspan=1>1/2</td><td rowspan=1 colspan=1>50.0 (9.5-90.5)</td><td rowspan=1 colspan=1>295/295</td><td rowspan=1 colspan=1>100 (98.7-100)</td></tr><tr><td rowspan=1 colspan=1>Retrospective</td><td rowspan=1 colspan=1>9/11</td><td rowspan=1 colspan=1>81.8 (52.3-94.9)</td><td rowspan=1 colspan=1>519/521</td><td rowspan=1 colspan=1>99.6 (98.6-99.9)</td></tr><tr><td rowspan=1 colspan=1>Prospective / Retrospective</td><td rowspan=1 colspan=1>10/13A</td><td rowspan=1 colspan=1>76.9 (49.7-91.8)</td><td rowspan=1 colspan=1>814/816</td><td rowspan=1 colspan=1>99.8 (99.1-99.9)</td></tr><tr><td rowspan=1 colspan=1>Contrived</td><td rowspan=1 colspan=1>37/37</td><td rowspan=1 colspan=1>100 (90.6-100)</td><td rowspan=1 colspan=1>439/439</td><td rowspan=1 colspan=1>100 (99.1-100)</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>47/50</td><td rowspan=1 colspan=1>94.0 (83.8-97.9)</td><td rowspan=1 colspan=1>1253/1255</td><td rowspan=1 colspan=1>99.8 (99.4-100)</td></tr></table>

A. In 1/3 false negative samples, OXA signal was above the threshold for detection; however, an associated organism was not detected by the BCID-GN Panel and the OXA target was reported as $\mathbf { \hat { N } } / \mathbf { A } ^ { \prime }$ . One additional false negative sample was tested with an FDA-cleared multiplex assay and OXA was not detected. The isolate from the remaining false negative sample tested negative for OXA-23 and OXA-48 by qPCR.

Table 33: Clinical Performance for VIM   

<table><tr><td rowspan=2 colspan=1>Target</td><td rowspan=2 colspan=1>Sample Type</td><td rowspan=1 colspan=2>Sensitivity/PPA</td><td rowspan=1 colspan=2>Specificity/NPA</td></tr><tr><td rowspan=1 colspan=1>TP/TP+FN</td><td rowspan=1 colspan=1>% (95% CI)</td><td rowspan=1 colspan=1>TN/TN+FP</td><td rowspan=1 colspan=1>% (95% CI)</td></tr><tr><td rowspan=7 colspan=1>VIM</td><td rowspan=1 colspan=1>Prospective (Fresh)</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>---</td><td rowspan=1 colspan=1>138/138</td><td rowspan=1 colspan=1>100 (97.3-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective (Frozen)</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>159/159</td><td rowspan=1 colspan=1>100 (97.6-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective (All)</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>297/297</td><td rowspan=1 colspan=1>100 (98.7-100)</td></tr><tr><td rowspan=1 colspan=1>Retrospective</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>532/532</td><td rowspan=1 colspan=1>100 (99.3-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective / Retrospective</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>829/829</td><td rowspan=1 colspan=1>100 (99.5-100)</td></tr><tr><td rowspan=1 colspan=1>Contrived</td><td rowspan=1 colspan=1>42/42</td><td rowspan=1 colspan=1>100 (91.6-100)</td><td rowspan=1 colspan=1>434/434</td><td rowspan=1 colspan=1>100 (99.1-100)</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>42/42</td><td rowspan=1 colspan=1>100 (91.6-100)</td><td rowspan=1 colspan=1>1263/1263</td><td rowspan=1 colspan=1>100 (99.7-100)</td></tr></table>

# Pan Targets

In addition to the evaluable prospective and retrospective samples that contain gram-negative organisms, the clinical performance of the Pan Candida and Pan Gram-Positive targets was evaluated by testing an additional 741 non-intended use retrospective samples with grampositive or fungal organisms; these are denoted as Retrospective (Non-Intended Use) samples. Results for the Pan targets are summarized in Table 34 and results stratified by species are summarized in Table 36. A summary of the strains used to prepare contrived samples and the number of strains for each target is shown in Table 35.

Table 34: Clinical Performance for Pan Targets   

<table><tr><td rowspan=2 colspan=1>Target</td><td rowspan=2 colspan=1>Sample Type</td><td rowspan=1 colspan=2>Sensitivity/PPA</td><td rowspan=1 colspan=2>Specificity/NPA</td></tr><tr><td rowspan=1 colspan=1>TP/TP+FN</td><td rowspan=1 colspan=1>% (95% CI)</td><td rowspan=1 colspan=1>TN/TN+FP</td><td rowspan=1 colspan=1>% (95% CI)</td></tr><tr><td rowspan=6 colspan=1>Pan Candida</td><td rowspan=1 colspan=1>Prospective (Fresh)</td><td rowspan=1 colspan=1>1/1</td><td rowspan=1 colspan=1>100 (20.7-100)</td><td rowspan=1 colspan=1>165/166</td><td rowspan=1 colspan=1>99.4 (96.7-99.9)</td></tr><tr><td rowspan=1 colspan=1>Prospective (Frozen)</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>---</td><td rowspan=1 colspan=1>182/182</td><td rowspan=1 colspan=1>100 (97.9-100)</td></tr><tr><td rowspan=1 colspan=1>Prospective (All)</td><td rowspan=1 colspan=1>1/1</td><td rowspan=1 colspan=1>100 (20.7-100)</td><td rowspan=1 colspan=1>347/3484</td><td rowspan=1 colspan=1>99.7 (98.4-99.9)</td></tr><tr><td rowspan=1 colspan=1>Retrospective</td><td rowspan=1 colspan=1>4/7B</td><td rowspan=1 colspan=1>57.1 (25.0-84.2)</td><td rowspan=1 colspan=1>569/570C</td><td rowspan=1 colspan=1>99.8 (99.0-100)</td></tr><tr><td rowspan=1 colspan=1>Retrospective (Non-Intended Use)</td><td rowspan=1 colspan=1>99/102D</td><td rowspan=1 colspan=1>97.1 (91.7-99.0)</td><td rowspan=1 colspan=1>638/639E</td><td rowspan=1 colspan=1>99.8 (99.1-100)</td></tr><tr><td rowspan=1 colspan=1>Contrived</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>---</td><td rowspan=1 colspan=1>777/777</td><td rowspan=1 colspan=1>100 (99.5-100)</td></tr><tr><td rowspan=6 colspan=1>Pan Gram-Positive</td><td rowspan=1 colspan=1>Prospective (Fresh)</td><td rowspan=1 colspan=1>7/8</td><td rowspan=1 colspan=1>87.5 (52.9-97.8)</td><td rowspan=1 colspan=1>155/159</td><td rowspan=1 colspan=1>97.5 (93.7-99.0)</td></tr><tr><td rowspan=1 colspan=1>Prospective (Frozen)</td><td rowspan=1 colspan=1>10/15</td><td rowspan=1 colspan=1>66.7 (41.7-84.8)</td><td rowspan=1 colspan=1>164/167</td><td rowspan=1 colspan=1>98.2 (94.9-99.4)</td></tr><tr><td rowspan=1 colspan=1>Prospective (All)</td><td rowspan=1 colspan=1>17/23F</td><td rowspan=1 colspan=1>73.9 (53.5-87.5)</td><td rowspan=1 colspan=1>319/326G</td><td rowspan=1 colspan=1>97.9 (95.6-99.0)</td></tr><tr><td rowspan=1 colspan=1>Retrospective</td><td rowspan=1 colspan=1>44/55H</td><td rowspan=1 colspan=1>80.0 (67.6-88.4)</td><td rowspan=1 colspan=1>512/5221</td><td rowspan=1 colspan=1>98.1 (96.5-99.0)</td></tr><tr><td rowspan=1 colspan=1>Retrospective (Non-Intended Use)</td><td rowspan=1 colspan=1>567/571</td><td rowspan=1 colspan=1>99.3 (98.2-99.7)</td><td rowspan=1 colspan=1>165/170&#x27;</td><td rowspan=1 colspan=1>97.1 (93.3-98.7)</td></tr><tr><td rowspan=1 colspan=1>Contrived</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>--</td><td rowspan=1 colspan=1>776/777</td><td rowspan=1 colspan=1>99.9 (99.3-100)</td></tr></table>

A. Candida glabrata was detected in the $1 / 1$ false positive sample using PCR/sequencing.   
B. 3 of 3 $( 1 0 0 \% )$ false negative results occurred in samples in mixed infections with bacterial organisms where the BCID-GN Panel results were correct for the other infections in these samples.   
C. Candida albicans was detected in the $1 / 1$ false positive sample using PCR/sequencing.   
D. 2 of 3 $( 6 7 \% )$ false negative results occurred in samples in mixed infections with bacterial organisms where the BCID-GN Panel results were correct for the other infections in these samples.   
E. Candida glabrata was detected in the $1 / 1$ false positive sample using PCR/sequencing.   
F. Bacillus (the gram-positive organism identified by standard laboratory procedures) was not detected in 2 false negative samples using PCR/sequencing, but 16S sequencing instead detected Paenibacillus lautus and Paenibacillus urinalis, which were not identified by standard laboratory procedures. 3 of the remaining 4 $( 7 5 \% )$ false negative results occurred in samples in mixed infections with bacterial organisms where the BCID-GN Panel correctly detected the gram-negative organisms present.   
G. Enterococcus (1), Staphylococcus (3), or Streptococcus (2) were detected in $6 / 7$ false positive samples using PCR/sequencing (testing was not performed for the remaining 1/7 false positive samples).   
H. 11 of 11 $( 1 0 0 \% )$ false negative results occurred in samples in mixed infections with bacterial organisms where the BCID-GN Panel correctly detected the gram-negative organisms present.   
I. Enterococcus (2), Staphylococcus (1), or Streptococcus (5) were detected in 8/10 false positive samples using PCR/sequencing (testing was not performed for the remaining 2/10 false positive samples).   
J. Bacillus (1) or Streptococcus (1) were detected in 2/5 false positive samples using PCR/sequencing (testing was not performed for the remaining 3/5 false positive samples).

Table 35: Contrived Sample Summary   

<table><tr><td colspan="1" rowspan="1">Target</td><td colspan="1" rowspan="1">Organism</td><td colspan="1" rowspan="1">Strain</td><td colspan="1" rowspan="1">Independent ContrivedSamples Tested</td></tr><tr><td colspan="1" rowspan="12">Acinetobacter baumannii</td><td colspan="1" rowspan="7">Acinetobacter baumannii</td><td colspan="1" rowspan="1">ATCC19606</td><td colspan="1" rowspan="1">2</td></tr><tr><td colspan="1" rowspan="1">ATCCBAA-2093</td><td colspan="1" rowspan="1">4</td></tr><tr><td colspan="1" rowspan="1">ATCCBAA-747</td><td colspan="1" rowspan="1">5</td></tr><tr><td colspan="1" rowspan="1">NCIMB12457</td><td colspan="1" rowspan="1">2</td></tr><tr><td colspan="1" rowspan="1">NCTC13302</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">NCTC13303</td><td colspan="1" rowspan="1">4</td></tr><tr><td colspan="1" rowspan="1">NCTC13423</td><td colspan="1" rowspan="1">17</td></tr><tr><td colspan="1" rowspan="1">Acinetobacter baumannii, NDM</td><td colspan="1" rowspan="1">CDC#0033</td><td colspan="1" rowspan="1">5$</td></tr><tr><td colspan="1" rowspan="3">Acinetobacter baumannii, OXA-23</td><td colspan="1" rowspan="1">ATCCBAA-1605</td><td colspan="1" rowspan="1">5</td></tr><tr><td colspan="1" rowspan="1">NCTC13304</td><td colspan="1" rowspan="1">5</td></tr><tr><td colspan="1" rowspan="1">NCTC13305</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="2" rowspan="1">Acinetobacter baumannii Total</td><td colspan="1" rowspan="1">55</td></tr><tr><td colspan="1" rowspan="6">Bacteroides fragilis</td><td colspan="1" rowspan="5">Bacteroides fragilis</td><td colspan="1" rowspan="1">ATCC23745</td><td colspan="1" rowspan="1">8</td></tr><tr><td colspan="1" rowspan="1">ATCC25285</td><td colspan="1" rowspan="1">8</td></tr><tr><td colspan="1" rowspan="1">ATCC43860</td><td colspan="1" rowspan="1">8</td></tr><tr><td colspan="1" rowspan="1">ATCC700786</td><td colspan="1" rowspan="1">8</td></tr><tr><td colspan="1" rowspan="1">NCTC9343</td><td colspan="1" rowspan="1">8</td></tr><tr><td colspan="2" rowspan="1">Bacteroides fragilis Total</td><td colspan="1" rowspan="1">40</td></tr><tr><td colspan="1" rowspan="11">Citrobacter</td><td colspan="1" rowspan="2">Citrobacter braakii</td><td colspan="1" rowspan="1">ATCC43162</td><td colspan="1" rowspan="1">4</td></tr><tr><td colspan="1" rowspan="1">ATCC51113</td><td colspan="1" rowspan="1">4</td></tr><tr><td colspan="1" rowspan="4">Citrobacter freundii</td><td colspan="1" rowspan="1">ATCC43864</td><td colspan="1" rowspan="1">4</td></tr><tr><td colspan="1" rowspan="1">ATCC8090</td><td colspan="1" rowspan="1">4</td></tr><tr><td colspan="1" rowspan="1">NCTC8581</td><td colspan="1" rowspan="1">4</td></tr><tr><td colspan="1" rowspan="1">NCTC9750</td><td colspan="1" rowspan="1">5</td></tr><tr><td colspan="1" rowspan="1">Citrobacter freundii, CTX-M</td><td colspan="1" rowspan="1">JMI2047</td><td colspan="1" rowspan="1">6</td></tr><tr><td colspan="1" rowspan="1">Citrobacter freundii, KPC</td><td colspan="1" rowspan="1">CDC#0116</td><td colspan="1" rowspan="1">4</td></tr><tr><td colspan="1" rowspan="1">Citrobacter koseri</td><td colspan="1" rowspan="1">ATCC27156</td><td colspan="1" rowspan="1">4</td></tr><tr><td colspan="1" rowspan="1">Citrobacter youngae</td><td colspan="1" rowspan="1">ATCC29935</td><td colspan="1" rowspan="1">4</td></tr><tr><td colspan="2" rowspan="1">Citrobacter Total</td><td colspan="1" rowspan="1">43</td></tr><tr><td colspan="1" rowspan="1">Cronobacter sakazakii</td><td colspan="1" rowspan="1">Cronobacter sakazakii</td><td colspan="1" rowspan="1">ATCC12868</td><td colspan="1" rowspan="1">2</td></tr><tr><td colspan="1" rowspan="13"></td><td colspan="1" rowspan="12">−</td><td colspan="1" rowspan="1">ATCC29004</td><td colspan="1" rowspan="1">2</td></tr><tr><td colspan="1" rowspan="1">ATCC29544</td><td colspan="1" rowspan="1">4</td></tr><tr><td colspan="1" rowspan="1">ATCCBAA-894</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">FSLF6-0023</td><td colspan="1" rowspan="1">4</td></tr><tr><td colspan="1" rowspan="1">FSLF6-0028</td><td colspan="1" rowspan="1">4</td></tr><tr><td colspan="1" rowspan="1">FSLF6-0029</td><td colspan="1" rowspan="1">4</td></tr><tr><td colspan="1" rowspan="1">FSLF6-0034</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">FSLF6-0035</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">FSLF6-0043</td><td colspan="1" rowspan="1">4</td></tr><tr><td colspan="1" rowspan="1">FSLF6-0049</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">FSLF6-0050</td><td colspan="1" rowspan="1">4</td></tr><tr><td colspan="1" rowspan="1">FSLF6-0051</td><td colspan="1" rowspan="1">5</td></tr><tr><td colspan="2" rowspan="1">Cronobacter sakazakii Total</td><td colspan="1" rowspan="1">45</td></tr><tr><td colspan="1" rowspan="12">Enterobacter cloacaecomplex</td><td colspan="1" rowspan="4">Enterobacter asburiae</td><td colspan="1" rowspan="1">ATCC35953</td><td colspan="1" rowspan="1">2</td></tr><tr><td colspan="1" rowspan="1">ATCC35955</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">ATCC35956</td><td colspan="1" rowspan="1">4</td></tr><tr><td colspan="1" rowspan="1">ATCC35957</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="2">Enterobacter cloacae, CTX-M</td><td colspan="1" rowspan="1">CDC#0038</td><td colspan="1" rowspan="1">4</td></tr><tr><td colspan="1" rowspan="1">NCTC13464</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">Enterobacter cloacae, CTX-M, KPC</td><td colspan="1" rowspan="1">CDC#0163</td><td colspan="1" rowspan="1">2</td></tr><tr><td colspan="1" rowspan="2">Enterobacter cloacae, CTX-M, NDM</td><td colspan="1" rowspan="1">CDC#0038</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">JMI53571</td><td colspan="1" rowspan="1">12</td></tr><tr><td colspan="1" rowspan="1">Enterobacter cloacae, VIM</td><td colspan="1" rowspan="1">CDC#0154</td><td colspan="1" rowspan="1">6</td></tr><tr><td colspan="1" rowspan="1">Enterobacter hormaechei, KPC</td><td colspan="1" rowspan="1">ATCCBAA-2082</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="2" rowspan="1">Enterobacter cloacae complex Total</td><td colspan="1" rowspan="1">37</td></tr><tr><td colspan="1" rowspan="6">Enterobacter (non-cloacae complex)</td><td colspan="1" rowspan="3">Enterobacter aerogenes</td><td colspan="1" rowspan="1">ATCC13048</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">ATCC29010</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">ATCC51697</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">Enterobacter aerogenes, IMP</td><td colspan="1" rowspan="1">CDC#0161</td><td colspan="1" rowspan="1">5</td></tr><tr><td colspan="1" rowspan="1">Enterobacter aerogenes, OXA-48</td><td colspan="1" rowspan="1">CDC#0074</td><td colspan="1" rowspan="1">12</td></tr><tr><td colspan="1" rowspan="1">Enterobacter amnigenus</td><td colspan="1" rowspan="1">ATCC33072</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="3"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="1">ATCC33731</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">ATCC51816</td><td colspan="1" rowspan="1">4</td></tr><tr><td colspan="2" rowspan="1">Enterobacter (non-cloacae complex) Total</td><td colspan="1" rowspan="1">36</td></tr><tr><td colspan="1" rowspan="14">Escherichia coli</td><td colspan="1" rowspan="4">Escherichia coli, CTX-M</td><td colspan="1" rowspan="1">CDC#0086</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">NCTC13452</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">NCTC13461</td><td colspan="1" rowspan="1">6</td></tr><tr><td colspan="1" rowspan="1">NCTC13463</td><td colspan="1" rowspan="1">2</td></tr><tr><td colspan="1" rowspan="2">Escherichia coli, CTX-M, NDM</td><td colspan="1" rowspan="1">CDC#0137</td><td colspan="1" rowspan="1">6</td></tr><tr><td colspan="1" rowspan="1">CDC#0151</td><td colspan="1" rowspan="1">2</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli, IMP</td><td colspan="1" rowspan="1">NCTC13476</td><td colspan="1" rowspan="1">7</td></tr><tr><td colspan="1" rowspan="2">Escherichia coli, KPC</td><td colspan="1" rowspan="1">ATCCBAA-2340</td><td colspan="1" rowspan="1">2</td></tr><tr><td colspan="1" rowspan="1">CDC#0114</td><td colspan="1" rowspan="1">4</td></tr><tr><td colspan="1" rowspan="3">Escherichia coli, NDM</td><td colspan="1" rowspan="1">CDC#0118</td><td colspan="1" rowspan="1">6</td></tr><tr><td colspan="1" rowspan="1">CDC#0150</td><td colspan="1" rowspan="1">5</td></tr><tr><td colspan="1" rowspan="1">CDC#0151</td><td colspan="1" rowspan="1">4</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli, VIM</td><td colspan="1" rowspan="1">JMI32465</td><td colspan="1" rowspan="1">2</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli Total</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">52</td></tr><tr><td colspan="1" rowspan="3">Fusobacteriumnecrophorum</td><td colspan="1" rowspan="1">Fusobacterium necrophorum subsp.funduliforme</td><td colspan="1" rowspan="1">ATCC51357</td><td colspan="1" rowspan="1">24</td></tr><tr><td colspan="1" rowspan="1">Fusobacterium necrophorum subsp.necrophorum</td><td colspan="1" rowspan="1">ATCC27852</td><td colspan="1" rowspan="1">24</td></tr><tr><td colspan="2" rowspan="1">Fusobacterium necrophorum Total</td><td colspan="1" rowspan="1">48</td></tr><tr><td colspan="1" rowspan="4">Fusobacterium nucleatum</td><td colspan="1" rowspan="3">Fusobacterium nucleatum</td><td colspan="1" rowspan="1">ATCC23726</td><td colspan="1" rowspan="1">8</td></tr><tr><td colspan="1" rowspan="1">ATCC25586</td><td colspan="1" rowspan="1">20</td></tr><tr><td colspan="1" rowspan="1">ATCC31647</td><td colspan="1" rowspan="1">19</td></tr><tr><td colspan="2" rowspan="1">Fusobacterium nucleatum Total</td><td colspan="1" rowspan="1">47</td></tr><tr><td colspan="1" rowspan="6">Haemophilus influenzae</td><td colspan="1" rowspan="5">Haemophilus influenzae</td><td colspan="1" rowspan="1">ATCC10211</td><td colspan="1" rowspan="1">9</td></tr><tr><td colspan="1" rowspan="1">ATCC43065</td><td colspan="1" rowspan="1">6</td></tr><tr><td colspan="1" rowspan="1">ATCC49144</td><td colspan="1" rowspan="1">9</td></tr><tr><td colspan="1" rowspan="1">NCTC12699</td><td colspan="1" rowspan="1">9</td></tr><tr><td colspan="1" rowspan="1">NCTC8468</td><td colspan="1" rowspan="1">8</td></tr><tr><td colspan="2" rowspan="1">Haemophilus influenzae Total</td><td colspan="1" rowspan="1">41</td></tr><tr><td colspan="1" rowspan="7">Klebsiella oxytoca</td><td colspan="1" rowspan="6">Klebsiella oxytoca</td><td colspan="1" rowspan="1">ATCC43086</td><td colspan="1" rowspan="1">4</td></tr><tr><td colspan="1" rowspan="1">ATCC43165</td><td colspan="1" rowspan="1">4</td></tr><tr><td colspan="1" rowspan="1">ATCC43863</td><td colspan="1" rowspan="1">2</td></tr><tr><td colspan="1" rowspan="1">ATCC49131</td><td colspan="1" rowspan="1">4</td></tr><tr><td colspan="1" rowspan="1">ATCC51817</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">ATCC700324</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="2" rowspan="1">Klebsiella oxytoca Total</td><td colspan="1" rowspan="1">20</td></tr><tr><td colspan="1" rowspan="16">Klebsiella pneumoniaegroup</td><td colspan="1" rowspan="1">Klebsiella pneumoniae, CTX-M</td><td colspan="1" rowspan="1">NCTC13465</td><td colspan="1" rowspan="1">5</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae, CTX-M, NDM</td><td colspan="1" rowspan="1">ATCCBAA-2146</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae, CTX-M, OXA</td><td colspan="1" rowspan="1">CDC#0140</td><td colspan="1" rowspan="1">12</td></tr><tr><td colspan="1" rowspan="2">Klebsiella pneumoniae, IMP</td><td colspan="1" rowspan="1">CDC#0034</td><td colspan="1" rowspan="1">8</td></tr><tr><td colspan="1" rowspan="1">CDC#0080</td><td colspan="1" rowspan="1">6</td></tr><tr><td colspan="1" rowspan="7">Klebsiella pneumoniae, KPC</td><td colspan="1" rowspan="1">CDC#0112</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">CDC#0113</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">CDC#0115</td><td colspan="1" rowspan="1">4</td></tr><tr><td colspan="1" rowspan="1">CDC#0117</td><td colspan="1" rowspan="1">4</td></tr><tr><td colspan="1" rowspan="1">CDC#0120</td><td colspan="1" rowspan="1">4</td></tr><tr><td colspan="1" rowspan="1">CDC#0125</td><td colspan="1" rowspan="1">4</td></tr><tr><td colspan="1" rowspan="1">CDC#0129</td><td colspan="1" rowspan="1">4</td></tr><tr><td colspan="1" rowspan="3">Klebsiella pneumoniae, VIM</td><td colspan="1" rowspan="1">LMC_DR00015</td><td colspan="1" rowspan="1">6</td></tr><tr><td colspan="1" rowspan="1">NCTC13439</td><td colspan="1" rowspan="1">5</td></tr><tr><td colspan="1" rowspan="1">NCTC13440</td><td colspan="1" rowspan="1">5</td></tr><tr><td colspan="2" rowspan="1">Klebsiella pneumoniae group Total</td><td colspan="1" rowspan="1">72</td></tr><tr><td colspan="1" rowspan="8">Morganella morganii</td><td colspan="1" rowspan="5">Morganella morganii</td><td colspan="1" rowspan="1">148-200</td><td colspan="1" rowspan="1">8</td></tr><tr><td colspan="1" rowspan="1">148-204</td><td colspan="1" rowspan="1">8</td></tr><tr><td colspan="1" rowspan="1">148-205</td><td colspan="1" rowspan="1">8</td></tr><tr><td colspan="1" rowspan="1">148-206</td><td colspan="1" rowspan="1">9</td></tr><tr><td colspan="1" rowspan="1">148-209</td><td colspan="1" rowspan="1">7</td></tr><tr><td colspan="1" rowspan="1">Morganella morganii, CTX-M1, NDM</td><td colspan="1" rowspan="1">CDC#0057</td><td colspan="1" rowspan="1">5</td></tr><tr><td colspan="1" rowspan="1">Morganella morganii, KPC</td><td colspan="1" rowspan="1">CDC#0133</td><td colspan="1" rowspan="1">4</td></tr><tr><td colspan="2" rowspan="1">Morganella morganii Total</td><td colspan="1" rowspan="1">49</td></tr><tr><td colspan="1" rowspan="7">Neisseria meningitidis</td><td colspan="1" rowspan="6">Neisseria meningitidis</td><td colspan="1" rowspan="1">ATCC13077</td><td colspan="1" rowspan="1">9</td></tr><tr><td colspan="1" rowspan="1">ATCC13090</td><td colspan="1" rowspan="1">8</td></tr><tr><td colspan="1" rowspan="1">ATCC13102</td><td colspan="1" rowspan="1">8</td></tr><tr><td colspan="1" rowspan="1">ATCC13113</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">ATCC35561</td><td colspan="1" rowspan="1">10</td></tr><tr><td colspan="1" rowspan="1">NCTC10026</td><td colspan="1" rowspan="1">6</td></tr><tr><td colspan="2" rowspan="1">Neisseria meningitidis Total</td><td colspan="1" rowspan="1">44</td></tr><tr><td colspan="1" rowspan="3">Proteus mirabilis</td><td colspan="1" rowspan="1">Proteus mirabilis, KPC</td><td colspan="1" rowspan="1">CDC#0155</td><td colspan="1" rowspan="1">4</td></tr><tr><td colspan="1" rowspan="1">Proteus mirabilis, NDM</td><td colspan="1" rowspan="1">CDC#0159</td><td colspan="1" rowspan="1">5</td></tr><tr><td colspan="2" rowspan="1">Proteus mirabilis Total</td><td colspan="1" rowspan="1">9</td></tr><tr><td colspan="1" rowspan="8">Pseudomonas aeruginosa</td><td colspan="1" rowspan="2">Pseudomonas aeruginosa, IMP</td><td colspan="1" rowspan="1">CDC#0092</td><td colspan="1" rowspan="1">5</td></tr><tr><td colspan="1" rowspan="1">CDC#0103</td><td colspan="1" rowspan="1">8</td></tr><tr><td colspan="1" rowspan="1">Pseudomonas aeruginosa, KPC</td><td colspan="1" rowspan="1">CDC#0090</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="4">Pseudomonas aeruginosa, VIM</td><td colspan="1" rowspan="1">CDC#0054</td><td colspan="1" rowspan="1">5</td></tr><tr><td colspan="1" rowspan="1">CDC#0100</td><td colspan="1" rowspan="1">4</td></tr><tr><td colspan="1" rowspan="1">CDC#0108</td><td colspan="1" rowspan="1">4</td></tr><tr><td colspan="1" rowspan="1">NCTC13437</td><td colspan="1" rowspan="1">5</td></tr><tr><td colspan="2" rowspan="1">Pseudomonas aeruginosa Total</td><td colspan="1" rowspan="1">32</td></tr><tr><td colspan="1" rowspan="10">Salmonella</td><td colspan="1" rowspan="1">Salmonella 4,5,12:I:-</td><td colspan="1" rowspan="1">FSLS5-0580</td><td colspan="1" rowspan="1">2</td></tr><tr><td colspan="1" rowspan="1">Salmonella Heidelberg</td><td colspan="1" rowspan="1">ATCC8326</td><td colspan="1" rowspan="1">2</td></tr><tr><td colspan="1" rowspan="1">Salmonella Infantis</td><td colspan="1" rowspan="1">ATCCBAA-1675</td><td colspan="1" rowspan="1">2</td></tr><tr><td colspan="1" rowspan="1">Salmonella Javiana</td><td colspan="1" rowspan="1">ATCC10721</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">Salmonella Montevideo</td><td colspan="1" rowspan="1">ATCC8387</td><td colspan="1" rowspan="1">8</td></tr><tr><td colspan="1" rowspan="1">Salmonella Muenchen</td><td colspan="1" rowspan="1">ATCC8388</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">Salmonella Newport</td><td colspan="1" rowspan="1">ATCC6962</td><td colspan="1" rowspan="1">6</td></tr><tr><td colspan="1" rowspan="1">Salmonella Typhimurium</td><td colspan="1" rowspan="1">ATCC13311</td><td colspan="1" rowspan="1">7</td></tr><tr><td colspan="1" rowspan="1">Salmonella enterica subspecies entericaEnteritidis (Group D1)</td><td colspan="1" rowspan="1">ATCCBAA-708</td><td colspan="1" rowspan="1">6</td></tr><tr><td colspan="2" rowspan="1">Salmonella Total</td><td colspan="1" rowspan="1">35</td></tr><tr><td colspan="1" rowspan="2">Serratia</td><td colspan="1" rowspan="1">Serratia ficaria</td><td colspan="1" rowspan="1">ATCC33105</td><td colspan="1" rowspan="1">4</td></tr><tr><td colspan="1" rowspan="1">Serratia grimesii</td><td colspan="1" rowspan="1">ATCC14460</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="4"></td><td colspan="1" rowspan="1">Serratia plymuthica</td><td colspan="1" rowspan="1">ATCC53858</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="2">Serratia rubidaea</td><td colspan="1" rowspan="1">ATCC27593</td><td colspan="1" rowspan="1">4</td></tr><tr><td colspan="1" rowspan="1">ATCC29025</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="2" rowspan="1">Serratia Total</td><td colspan="1" rowspan="1">17</td></tr><tr><td colspan="1" rowspan="8">Serratia marcescens</td><td colspan="1" rowspan="6">Serratia marcescens</td><td colspan="1" rowspan="1">ATCC13880</td><td colspan="1" rowspan="1">2</td></tr><tr><td colspan="1" rowspan="1">ATCC14041</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">ATCC14756</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">ATCC43861</td><td colspan="1" rowspan="1">4</td></tr><tr><td colspan="1" rowspan="1">ATCC43862</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">ATCC8100</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">Serratia marcescens, IMP</td><td colspan="1" rowspan="1">LMC-DR23105</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="2" rowspan="1">Serratia marcescens Total</td><td colspan="1" rowspan="1">19</td></tr><tr><td colspan="1" rowspan="6">Stenotrophomonasmaltophilia</td><td colspan="1" rowspan="5">Stenotrophomonas maltophilia</td><td colspan="1" rowspan="1">148-201</td><td colspan="1" rowspan="1">7</td></tr><tr><td colspan="1" rowspan="1">148-22</td><td colspan="1" rowspan="1">6</td></tr><tr><td colspan="1" rowspan="1">148-223</td><td colspan="1" rowspan="1">7</td></tr><tr><td colspan="1" rowspan="1">148-224</td><td colspan="1" rowspan="1">8</td></tr><tr><td colspan="1" rowspan="1">148-225</td><td colspan="1" rowspan="1">8</td></tr><tr><td colspan="2" rowspan="1">Stenotrophomonas maltophilia Total</td><td colspan="1" rowspan="1">36</td></tr></table>

# Genus and Group Assay Species Stratification

The ePlex BCID-GN Panel reports genus or group level results for Citrobacter, Enterobacter cloacae complex, Enterobacter non-cloacae complex, Proteus, Salmonella, Serratia, Pan Candida and Pan Gram-Positive targets. Sensitivity/PPA of these genus and group level targets for species as determined by comparator methods for all evaluable samples tested are summarized in Table 36 and for the Pan targets for non-intended use samples in Table 37.

Table 36: Species Detected in Genus and Group Assays by Comparator Methods   

<table><tr><td>Target Species Detected by Comparator Method</td><td colspan="2">Prospective Samples Sensitivity/PPA</td><td colspan="2">Retrospective Samples Sensitivity/PPA</td><td colspan="2">Contrived Samples Sensitivity/PPA</td><td colspan="2">Combined Samples Sensitivity/PPA</td></tr><tr><td></td><td>TP/TP+FN</td><td>% (95% CI)</td><td>TP/TP+FN</td><td>% (95% CI)</td><td>TP/TP+FN</td><td>% (95% CI)</td><td>TP/TP+FN</td><td>% (95% CI)</td></tr><tr><td>Citrobacter</td><td>5/5</td><td>100 (56.6-100)</td><td>20/21</td><td>95.2 (77.3-99.2)</td><td>43/43</td><td>100 (91.8-100)</td><td>68/69</td><td>98.6 (92.2-99.7)</td></tr><tr><td>Citrobacter braakii</td><td></td><td></td><td>2/3</td><td>66.7 (20.8-93.9)</td><td>8/8</td><td>100 (67.6-100)</td><td>10/11</td><td>90.9 (62.3-98.4)</td></tr><tr><td>Citrobacter freundii</td><td>4/4</td><td>100 (51.0-100)</td><td>13/13</td><td>100 (77.2-100)</td><td>27/27</td><td>100 (87.5-100)</td><td>44/44</td><td>100 (92.0-100)</td></tr><tr><td>Citrobacter koseri</td><td>1/1</td><td>100 (20.7-100)</td><td>4/4</td><td>100 (51.0-100)</td><td>4/4</td><td>100 (51.0-100)</td><td>9/9</td><td>100 (70.1-100)</td></tr><tr><td>Citrobacter youngae</td><td></td><td></td><td>1/1</td><td>100 (20.7-100)</td><td>4/4</td><td>100 (51.0-100)</td><td>5/5</td><td>100 (56.6-100)</td></tr><tr><td>Enterobacter (non-cloacae complex)</td><td>8/10</td><td>80.0 (49.0-94.3)</td><td>12/12</td><td>100 (75.8-100)</td><td>36/36</td><td>100 (90.4-100)</td><td>56/58</td><td>96.6 (88.3-99.0)</td></tr><tr><td>Enterobacter aerogenes</td><td>7/9</td><td>77.8 (45.3-93.7)</td><td>12/12</td><td>100 (75.8-100)</td><td>26/26</td><td>100 (87.1-100)</td><td>45/47</td><td>95.7 (85.8-98.8)</td></tr><tr><td>Enterobacter amnigenus</td><td></td><td></td><td></td><td></td><td>10/10</td><td>100 (72.2-100)</td><td>10/10</td><td>100 (72.2-100)</td></tr><tr><td>Enterobacter gergoviae</td><td>1/1</td><td>100 (20.7-100)</td><td></td><td></td><td></td><td></td><td>1/1</td><td>100 (20.7-100)</td></tr><tr><td>Enterobacter cloacae complex</td><td>19/19</td><td>100 (83.2-100)</td><td>47/50</td><td>94.0 (83.8-97.9)</td><td>35/37</td><td>94.6 (82.3-98.5)</td><td>101/106</td><td>95.3 (89.4-98.0)</td></tr><tr><td>Enterobacter asburiae</td><td></td><td></td><td></td><td></td><td>6/8</td><td>75.0 (40.9-92.9)</td><td>6/8</td><td>75.0 (40.9-92.9)</td></tr><tr><td>Enterobacter cloacae</td><td>19/19</td><td>100 (83.2-100)</td><td>46/49</td><td>93.9 (83.5-97.9)</td><td>28/28</td><td>100 (87.9-100)</td><td>93/96</td><td>96.9 (91.2-98.9)</td></tr><tr><td>Enterobacter hormaechei</td><td></td><td></td><td>1/1</td><td>100 (20.7-100)</td><td>1/1</td><td>100 (20.7-100)</td><td>2/2</td><td>100 (34.2-100)</td></tr><tr><td>Proteus</td><td>22/23</td><td>95.7 (79.0-99.2)</td><td>54/55</td><td>98.2 (90.4-99.7)</td><td>9/9</td><td>100 (70.1-100)</td><td>85/87</td><td>97.7 (92.0-99.4)</td></tr><tr><td>Proteus mirabilis</td><td>22/23</td><td>95.7 (79.0-99.2)</td><td>50/51</td><td>98.0 (89.7-99.7)</td><td>9/9</td><td>100 (70.1-100)</td><td>81/83</td><td>97.6 (91.6-99.3)</td></tr><tr><td>Proteus vulgaris</td><td></td><td></td><td>5/5</td><td>100 (56.6-100)</td><td></td><td></td><td>5/5</td><td>100 (56.6-100)</td></tr><tr><td>Salmonella</td><td>2/2</td><td>100 (34.2-100)</td><td>18/19</td><td>94.7 (75.4-99.1)</td><td>34/35</td><td>97.1 (85.5-99.5)</td><td>54/56</td><td>96.4 (87.9-99.0)</td></tr><tr><td>Salmonella</td><td>2/2</td><td>100 (34.2-100)</td><td>15/15</td><td>100 (79.6-100)</td><td></td><td></td><td>17/17</td><td>100 (81.6-100)</td></tr><tr><td>Salmonella 4,5,12:i:-</td><td></td><td></td><td></td><td></td><td>2/2</td><td>100 (34.2-100)</td><td>2/2</td><td>100 (34.2-100)</td></tr><tr><td>Salmonella Heidelberg</td><td></td><td></td><td></td><td></td><td>2/2</td><td>100 (34.2-100)</td><td>2/2</td><td>100 (34.2-100)</td></tr><tr><td>Salmonella Infantis</td><td></td><td></td><td></td><td></td><td>2/2</td><td>100 (34.2-100)</td><td>2/2</td><td>100 (34.2-100)</td></tr><tr><td>Salmonella Javiana</td><td></td><td></td><td></td><td></td><td>1/1</td><td>100 (20.7-100)</td><td>1/1</td><td>100 (20.7-100)</td></tr><tr><td>Salmonella Montevideo</td><td></td><td></td><td></td><td></td><td>7/8</td><td>87.5 (52.9-97.8)</td><td>7/8</td><td>87.5 (52.9-97.8)</td></tr><tr><td>Salmonella Muenchen</td><td></td><td></td><td></td><td></td><td>1/1</td><td>100 (20.7-100)</td><td>1/1</td><td>100 (20.7-100)</td></tr><tr><td>Salmonella Newport</td><td></td><td></td><td></td><td></td><td>6/6</td><td>100 (61.0-100)</td><td>6/6</td><td>100 (61.0-100)</td></tr><tr><td>Salmonella Typhimurium</td><td></td><td></td><td></td><td></td><td>7/7</td><td>100 (64.6-100)</td><td>7/7</td><td>100 (64.6-100)</td></tr><tr><td>Salmonella choleraesuis subsp. Arizonae</td><td></td><td></td><td>0/1</td><td>0.0 (0.0-79.3)</td><td></td><td></td><td>0/1</td><td>0.0 (0.0-79.3)</td></tr><tr><td>Salmonella enterica subsp. enterica Enteritidis (Group D1</td><td></td><td></td><td></td><td></td><td>6/6</td><td>100 (61.0-100)</td><td>6/6</td><td>100 (61.0-100)</td></tr><tr><td rowspan="3">Target Species DDetected by</td><td colspan="2">Prospective Samples</td><td colspan="2">Retrospective Samples</td><td colspan="2">Contrived Samples</td><td colspan="2">Combined Samples</td></tr><tr><td colspan="2">Sensitivity/PPA</td><td colspan="2">Sensitivity/PPA</td><td colspan="2">Sensitivity/PPA</td><td colspan="2">Sensitivity/PPA</td></tr><tr><td>TP/TP+FN</td><td>% (95% CI)</td><td>TP/TP+FN</td><td>% (95% CI)</td><td>TP/TP+FN</td><td>% (95% CI)</td><td>TP/TP+FN</td><td>% (95% CI)</td></tr><tr><td>Salmonella enterica subsp. enterica serovar Typhimurium</td><td></td><td></td><td>1/1</td><td>100 (20.7-100)</td><td></td><td></td><td>1/1</td><td>100 (20.7-100)</td></tr><tr><td>Salmonella Typhi</td><td></td><td></td><td>2/2</td><td>100 (34.2-100)</td><td></td><td></td><td>2/2</td><td>100 (34.2-100)</td></tr><tr><td>Serratia</td><td>10/10</td><td>100 (72.2-100)</td><td>34/34</td><td>100 (89.8-100)</td><td>36/36</td><td>100 (90.4-100)</td><td>80/80</td><td>100 (95.4-100)</td></tr><tr><td>Serratia ficaria</td><td></td><td></td><td></td><td></td><td>4/4</td><td>100 (51.0-100)</td><td>4/4</td><td>100 (51.0-100)</td></tr><tr><td>Serratia grimesii</td><td></td><td>100 (20.7-100)</td><td></td><td></td><td>3/3</td><td>100 (43.9-100)</td><td>3/3</td><td>100 (43.9-100)</td></tr><tr><td>Serratia liquefaciens</td><td>1/1</td><td>100 (70.1-100)</td><td>34/34</td><td>100 (89.8-100)</td><td></td><td></td><td>1/1</td><td>100 (20.7-100)</td></tr><tr><td>Serratia marcescens</td><td>9/9</td><td></td><td></td><td></td><td>19/19</td><td>100 (83.2-100)</td><td>62/62</td><td>100 (94.2-100)</td></tr><tr><td>Serratia plymuthica</td><td></td><td></td><td></td><td></td><td>3/3</td><td>100 (43.9-100)</td><td>3/3</td><td>100 (43.9-100)</td></tr><tr><td>Serratia rubidaea</td><td></td><td></td><td></td><td></td><td>7/7</td><td>100 (64.6-100)</td><td>7/7</td><td>100 (64.6-100)</td></tr><tr><td>Pan Candida</td><td>1/1</td><td>100 (20.7-100)</td><td>4/7</td><td>57.1 (25.0-84.2)</td><td>N/A</td><td>N/A</td><td>5/8</td><td>62.5 (30.6-86.3)</td></tr><tr><td>Candida albicans</td><td>1/1</td><td>100 (20.7-100)</td><td>2/4</td><td>50.0 (15.0-85.0)</td><td></td><td></td><td>3/5</td><td>60.0 (23.1-88.2)</td></tr><tr><td>Candida glabrata</td><td></td><td></td><td>1/2</td><td>50.0 (9.5-90.5)</td><td></td><td></td><td>1/2</td><td>50.0 (9.5-90.5)</td></tr><tr><td>Candida krusei</td><td></td><td></td><td>1/1</td><td>100 (20.7-100)</td><td></td><td></td><td>1/1</td><td>100 (20.7-100)</td></tr><tr><td>Candida parapsilosis</td><td></td><td>73.9 (53.5-87.5)</td><td>44/55</td><td>80.0 (67.6-88.4)</td><td>N/A</td><td></td><td></td><td></td></tr><tr><td>Pan Gram-Positive</td><td>17/23</td><td></td><td></td><td></td><td></td><td>N/A</td><td>61/78</td><td>78.2 (67.8-85.9)</td></tr><tr><td>Bacillus (unspeciated)</td><td>1/4</td><td>25.0 (4.6-69.9)</td><td></td><td></td><td></td><td></td><td>1/4</td><td>25.0 (4.6-69.9)</td></tr><tr><td>Enterococcus (unspeciated)</td><td></td><td></td><td>0/1</td><td>0.0 (0.0-79.3)</td><td></td><td></td><td>0/1</td><td>0.0 (0.0-79.3)</td></tr><tr><td>Enterococcus casseliflavus</td><td></td><td></td><td>0/1</td><td>0.0 (0.0-79.3)</td><td></td><td></td><td>0/1</td><td>0.0 (0.0-79.3)</td></tr><tr><td>Enterococcus faecalis</td><td>5/7</td><td>71.4 (35.9-91.8)</td><td>18/20</td><td>90.0 (69.9-97.2)</td><td></td><td></td><td>23/27</td><td>85.2 (67.5-94.1)</td></tr><tr><td>Enterococcus faecium</td><td>1/1</td><td>100 (20.7-100)</td><td>8/9</td><td>88.9 (56.5-98.0)</td><td></td><td></td><td>9/10</td><td>90.0 (59.6-98.2)</td></tr><tr><td>Staphylococcus (unspeciated)</td><td>3/3</td><td>100 (43.9-100)</td><td>4/6</td><td>66.7 (30.0-90.3)</td><td></td><td></td><td>7/9</td><td>77.8 (45.3-93.7)</td></tr><tr><td>Staphylococcus aureus</td><td>2/2</td><td>100 (34.2-100)</td><td>5/6</td><td>83.3 (43.6-97.0)</td><td></td><td></td><td>7/8</td><td>87.5 (52.9-97.8)</td></tr><tr><td>Staphylococcus cohnii</td><td>1/1</td><td>100 (20.7-100)</td><td></td><td></td><td></td><td></td><td>1/1</td><td>100 (20.7-100)</td></tr><tr><td>Staphylococcus epidermidis</td><td>1/1</td><td>100 (20.7-100)</td><td>2/3</td><td>66.7 (20.8-93.9)</td><td></td><td></td><td>3/4</td><td>75.0 (30.1-95.4)</td></tr><tr><td>Staphylococcus haemolyticus</td><td>2/2</td><td>100 (34.2-100)</td><td></td><td></td><td></td><td></td><td>2/2</td><td>100 (34.2-100)</td></tr><tr><td>Staphylococcus hominis</td><td>1/1</td><td>100 (20.7-100)</td><td>3/3</td><td>100 (43.9-100)</td><td></td><td></td><td>4/4</td><td>100 (51.0-100)</td></tr><tr><td>Streptococcus</td><td>0/1</td><td>0.0 (0.0-79.3)</td><td></td><td></td><td></td><td></td><td>0/1</td><td>0.0 (0.0-79.3)</td></tr><tr><td>Streptococcus - viridans group</td><td>1/1</td><td>100 (20.7-100)</td><td>0/1</td><td>0.0 (0.0-79.3)</td><td></td><td></td><td>1/2</td><td>50.0 (9.5-90.5)</td></tr><tr><td rowspan="3">Target Species Deected by Comparator Method</td><td colspan="2">Prospective Samples</td><td colspan="2">Retrospective Samples</td><td colspan="2">Contrived Samples</td><td colspan="2">Combined Samples</td></tr><tr><td colspan="2">Sensitivity/PPA</td><td colspan="2">Sensitivity/PPA</td><td colspan="2">Sensitivity/PPA</td><td colspan="2">Sensitivity/PPA</td></tr><tr><td>TP/TP+FN</td><td>% (95% CI)</td><td>TP/TP+FN</td><td>% (95% CI)</td><td>TP/TP+FN</td><td>% (95% CI)</td><td>TP/TP+FN</td><td>% (95% CI)</td></tr><tr><td>Streptococcus anginosus group</td><td>1/1</td><td>100 (20.7-100)</td><td>4/5</td><td>80.0 (37.6-96.4)</td><td></td><td></td><td>5/6</td><td>83.3 (43.6-97.0)</td></tr><tr><td>Streptococcus infantarius</td><td></td><td></td><td>1/1</td><td>100 (20.7-100)</td><td></td><td></td><td>1/1</td><td>100 (20.7-100)</td></tr><tr><td>Streptococcus mitis group</td><td></td><td></td><td>1/1</td><td>100 (20.7-100)</td><td></td><td></td><td>1/1</td><td>100 (20.7-100)</td></tr><tr><td>Streptococcus oralis</td><td></td><td></td><td>1/1</td><td>100 (20.7-100)</td><td></td><td></td><td>1/1</td><td>100 (20.7-100)</td></tr><tr><td>Streptococcus pneumoniae</td><td></td><td></td><td>1/1</td><td>100 (20.7-100)</td><td></td><td></td><td>1/1</td><td>100 (20.7-100)</td></tr><tr><td>Streptococcus salivarius</td><td></td><td></td><td>1/1</td><td>100 (20.7-100)</td><td></td><td></td><td>1/1</td><td>100 (20.7-100)</td></tr></table>

Table 37. Species Detected in Pan Assays by Comparator Methods for Samples in Retrospective (Non-Intended Use) Samples with Gram-Positive or Fungal Organisms   

<table><tr><td rowspan="3">Target Species Detected by Comparator Method</td><td colspan="2">Retrospective (Non-Intended Use) Samples</td></tr><tr><td>Sensitivity/PPA</td><td></td></tr><tr><td>TP/TP+FN</td><td>% (95% CI)</td></tr><tr><td>Pan Candida</td><td>99/102</td><td>97.1 (91.7-99.0)</td></tr><tr><td>Candida albicans</td><td>47/48</td><td>97.9 (89.1-99.6)</td></tr><tr><td>Candida glabrata</td><td>37/38</td><td>97.4 (86.5-99.5)</td></tr><tr><td>Candida krusei</td><td>3/3</td><td>100 (43.9-100)</td></tr><tr><td>Candida parapsilosis</td><td>15/16</td><td>93.8 (71.7-98.9)</td></tr><tr><td>Pan Gram-Positive</td><td>567/571</td><td>99.3 (98.2-99.7)</td></tr><tr><td>Bacillus (unspeciated)</td><td>4/4</td><td>100 (51.0-100)</td></tr><tr><td>Bacillus cereus</td><td>4/5</td><td>80.0 (37.6-96.4)</td></tr><tr><td>Bacillus cereus group - not anthracis</td><td>1/1</td><td>100 (20.7-100)</td></tr><tr><td>Enterococcus</td><td>1/1</td><td>100 (20.7-100)</td></tr><tr><td>Enterococcus faecalis</td><td>34/34</td><td>100 (89.8-100)</td></tr><tr><td>Enterococcus faecium</td><td>14/14</td><td>100 (78.5-100)</td></tr><tr><td>Coagulase-negative Staphylococci (CoNS)</td><td>14/14</td><td>100 (78.5-100)</td></tr><tr><td>Staphylococcus (unspeciated)</td><td>86/86</td><td>100 (95.7-100)</td></tr><tr><td>Staphylococcus aureus</td><td>173/174</td><td>99.4 (96.8-99.9)</td></tr><tr><td>Staphylococcus auricularis</td><td>3/3</td><td>100 (43.9-100)</td></tr><tr><td>Staphylococcus capitis</td><td>7/7</td><td>100 (64.6-100)</td></tr><tr><td>Staphylococcus cohnii</td><td>1/1</td><td>100 (20.7-100)</td></tr><tr><td rowspan="2">Target Species Detected by Comparator Method</td><td colspan="2">Retrospective (Non-Intended Use) Samples</td></tr><tr><td colspan="2">Sensitivity/PPA</td></tr><tr><td></td><td>TP/TP+FN</td><td>% (95% CI)</td></tr><tr><td>Staphylococcus epidermidis</td><td>86/87</td><td>98.9 (93.8-99.8)</td></tr><tr><td>Staphylococcus haemolyticus</td><td>6/6</td><td>100 (61.0-100)</td></tr><tr><td>Staphylococcus hominis</td><td>19/19</td><td>100 (83.2-100)</td></tr><tr><td>Staphylococcus hominis ssp hominis</td><td>21/21</td><td>100 (84.5-100)</td></tr><tr><td>Staphylococcus intermedius</td><td>1/1</td><td>100 (20.7-100)</td></tr><tr><td>Staphylococcus lugdunensis</td><td>1/1</td><td>100 (20.7-100)</td></tr><tr><td>Staphylococcus saccharolyticus</td><td>1/1</td><td>100 (20.7-100)</td></tr><tr><td>Staphylococcus saprophyticus</td><td>1/1</td><td>100 (20.7-100)</td></tr><tr><td>Staphylococcus sciuri</td><td>1/1</td><td>100 (20.7-100)</td></tr><tr><td>Staphylococcus simulans</td><td>2/2</td><td>100 (34.2-100)</td></tr><tr><td>Staphylococcus warneri</td><td>4/4</td><td>100 (51.0-100)</td></tr><tr><td>Alpha Hemolytic Streptococcus</td><td>1/1</td><td>100 (20.7-100)</td></tr><tr><td>Beta Hemolytic Streptococci, Group G</td><td>1/1</td><td>100 (20.7-100)</td></tr><tr><td>Gamma Hemolytic Streptococcus</td><td>1/1</td><td>100 (20.7-100)</td></tr><tr><td>Streptococcus (unspeciated)</td><td>9/9</td><td>100 (70.1-100)</td></tr><tr><td>Streptococcus - viridans group</td><td>17/17</td><td>100 (81.6-100)</td></tr><tr><td>Streptococcus agalactiae</td><td>21/21</td><td>100 (84.5-100)</td></tr><tr><td>Streptococcus anginosus</td><td>2/2</td><td>100 (34.2-100)</td></tr><tr><td>Streptococcus bovis</td><td>2/2</td><td>100 (34.2-100)</td></tr><tr><td>Streptococcus constellatus ssp constellatus</td><td>1/1</td><td>100 (20.7-100)</td></tr><tr><td>Streptococcus dysgalactiae (Group G)</td><td>4/4</td><td>100 (51.0-100)</td></tr><tr><td>Streptococcus gordonii</td><td>1/1</td><td>100 (20.7-100)</td></tr><tr><td>Streptococcus intermedius</td><td>1/1</td><td>100 (20.7-100)</td></tr><tr><td>Streptococcus mitis</td><td>11/12</td><td>91.7 (64.6-98.5)</td></tr><tr><td>Streptococcus mitis group</td><td>2/2</td><td>100 (34.2-100)</td></tr><tr><td>Streptococcus pneumoniae</td><td>22/22</td><td>100 (85.1-100)</td></tr><tr><td>Streptococcus pyogenes</td><td>9/9</td><td>100 (70.1-100)</td></tr><tr><td>Streptococcus salivarius</td><td>2/2</td><td>100 (34.2-100)</td></tr></table>

# Resistance Gene Assay Species Stratification

Test results for resistance genes are only reported when an associated organism assay is positive in the same sample. (See Table 3 for organisms specifically associated with the six resistance markers on the ePlex BCID-GN Panel).

# CTX-M

The PPA and NPA of the BCID-GN Panel CTX-M target stratified by the organism identified by comparator methods for prospective, retrospective and contrived samples are shown in Table 38.

Table 38: Clinical Performance of CTX-M Target in Samples with Associated Organisms Detected by Comparator Methods   

<table><tr><td colspan="2" rowspan="2">Species Detected by Comparator Method</td><td colspan="2" rowspan="1">Sensitivity/PPA</td><td colspan="2" rowspan="1">Specificity/NPA</td></tr><tr><td colspan="1" rowspan="1">TP/TP+FN</td><td colspan="1" rowspan="1">% (95% CI)</td><td colspan="1" rowspan="1">TN/TN+FP</td><td colspan="1" rowspan="1">% (95% CI)</td></tr><tr><td colspan="1" rowspan="4">Acinetobacter baumannii</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">---</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">100 (51.0-100)</td></tr><tr><td colspan="1" rowspan="1">Retrospective</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">---</td><td colspan="1" rowspan="1">15/15</td><td colspan="1" rowspan="1">100 (79.6-100)</td></tr><tr><td colspan="1" rowspan="1">Contrived</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">55/55</td><td colspan="1" rowspan="1">100 (93.5-100)</td></tr><tr><td colspan="1" rowspan="1">Combined</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">74/74</td><td colspan="1" rowspan="1">100 (95.1-100)</td></tr><tr><td colspan="1" rowspan="4">Citrobacter</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">---</td><td colspan="1" rowspan="1">5/5</td><td colspan="1" rowspan="1">100 (56.6-100)</td></tr><tr><td colspan="1" rowspan="1">Retrospective</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100 (20.7-100)</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">100 (83.9-100)</td></tr><tr><td colspan="1" rowspan="1">Contrived</td><td colspan="1" rowspan="1">6/6</td><td colspan="1" rowspan="1">100 (61.0-100)</td><td colspan="1" rowspan="1">37/37</td><td colspan="1" rowspan="1">100 (90.6-100)</td></tr><tr><td colspan="1" rowspan="1">Combined</td><td colspan="1" rowspan="1">7/7</td><td colspan="1" rowspan="1">100 (64.6-100)</td><td colspan="1" rowspan="1">62/62</td><td colspan="1" rowspan="1">100 (94.2-100)</td></tr><tr><td colspan="1" rowspan="4">Enterobacter (non-cloacae complex)</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">100 (72.2-100)</td></tr><tr><td colspan="1" rowspan="1">Retrospective</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">--</td><td colspan="1" rowspan="1">12/12</td><td colspan="1" rowspan="1">100 (75.8-100)</td></tr><tr><td colspan="1" rowspan="1">Contrived</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">---</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">100 (90.4-100)</td></tr><tr><td colspan="1" rowspan="1">Combined</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">58/58</td><td colspan="1" rowspan="1">100 (93.8-100)</td></tr><tr><td colspan="1" rowspan="4">Enterobacter cloacae complex</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">19/19</td><td colspan="1" rowspan="1">100 (83.2-100)</td></tr><tr><td colspan="1" rowspan="1">Retrospective</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">---</td><td colspan="1" rowspan="1">50/50</td><td colspan="1" rowspan="1">100 (92.9-100)</td></tr><tr><td colspan="1" rowspan="1">Contrived</td><td colspan="1" rowspan="1">22/22</td><td colspan="1" rowspan="1">100 (85.1-100)</td><td colspan="1" rowspan="1">15/15</td><td colspan="1" rowspan="1">100 (79.6-100)</td></tr><tr><td colspan="1" rowspan="1">Combined</td><td colspan="1" rowspan="1">22/22</td><td colspan="1" rowspan="1">100 (85.1-100)</td><td colspan="1" rowspan="1">84/84</td><td colspan="1" rowspan="1">100 (95.6-100)</td></tr><tr><td colspan="1" rowspan="4">Escherichia coli</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">16/18</td><td colspan="1" rowspan="1">88.9 (67.2-96.9)</td><td colspan="1" rowspan="1">115/115</td><td colspan="1" rowspan="1">100 (96.8-100)</td></tr><tr><td colspan="1" rowspan="1">Retrospective</td><td colspan="1" rowspan="1">35/37</td><td colspan="1" rowspan="1">94.6 (82.3-98.5)</td><td colspan="1" rowspan="1">103/103</td><td colspan="1" rowspan="1">100 (96.4-100)</td></tr><tr><td colspan="1" rowspan="1">Contrived</td><td colspan="1" rowspan="1">22/22</td><td colspan="1" rowspan="1">100 (85.1-100)</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">100 (88.6-100)</td></tr><tr><td colspan="1" rowspan="1">Combined</td><td colspan="1" rowspan="1">73/77</td><td colspan="1" rowspan="1">94.8 (87.4-98.0)</td><td colspan="1" rowspan="1">248/248</td><td colspan="1" rowspan="1">100 (98.5-100)</td></tr><tr><td colspan="1" rowspan="4">Klebsiella oxytoca</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">0/1</td><td colspan="1" rowspan="1">0.0 (0.0-79.3)</td><td colspan="1" rowspan="1">12/12</td><td colspan="1" rowspan="1">100 (75.8-100)</td></tr><tr><td colspan="1" rowspan="1">Retrospective</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">34/34</td><td colspan="1" rowspan="1">100 (89.8-100)</td></tr><tr><td colspan="1" rowspan="1">Contrived</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">---</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">100 (83.9-100)</td></tr><tr><td colspan="1" rowspan="1">Combined</td><td colspan="1" rowspan="1">0/1</td><td colspan="1" rowspan="1">0.0 (0.0-79.3)</td><td colspan="1" rowspan="1">66/66</td><td colspan="1" rowspan="1">100 (94.5-100)</td></tr><tr><td colspan="1" rowspan="4">Klebsiella pneumoniae group</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">5/5</td><td colspan="1" rowspan="1">100 (56.6-100)</td><td colspan="1" rowspan="1">56/56</td><td colspan="1" rowspan="1">100 (93.6-100)</td></tr><tr><td colspan="1" rowspan="1">Retrospective</td><td colspan="1" rowspan="1">14/15</td><td colspan="1" rowspan="1">93.3 (70.2-98.8)</td><td colspan="1" rowspan="1">93/93</td><td colspan="1" rowspan="1">100 (96.0-100)</td></tr><tr><td colspan="1" rowspan="1">Contrived</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">100 (83.9-100)</td><td colspan="1" rowspan="1">52/52</td><td colspan="1" rowspan="1">100 (93.1-100)</td></tr><tr><td colspan="1" rowspan="1">Combined</td><td colspan="1" rowspan="1">39/40</td><td colspan="1" rowspan="1">97.5 (87.1-99.6)</td><td colspan="1" rowspan="1">201/201</td><td colspan="1" rowspan="1">100 (98.1-100)</td></tr><tr><td colspan="1" rowspan="4">Morganella morganii</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">---</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">100 (43.9-100)</td></tr><tr><td colspan="1" rowspan="1">Retrospective</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">---</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">100 (72.2-100)</td></tr><tr><td colspan="1" rowspan="1">Contrived</td><td colspan="1" rowspan="1">5/5</td><td colspan="1" rowspan="1">100 (56.6-100)</td><td colspan="1" rowspan="1">44/44</td><td colspan="1" rowspan="1">100 (92.0-100)</td></tr><tr><td colspan="1" rowspan="1">Combined</td><td colspan="1" rowspan="1">5/5</td><td colspan="1" rowspan="1">100 (56.6-100)</td><td colspan="1" rowspan="1">57/57</td><td colspan="1" rowspan="1">100 (93.7-100)</td></tr><tr><td colspan="1" rowspan="4">Proteus</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">2/5</td><td colspan="1" rowspan="1">40.0 (11.8-76.9)</td><td colspan="1" rowspan="1">18/18</td><td colspan="1" rowspan="1">100 (82.4-100)</td></tr><tr><td colspan="1" rowspan="1">Retrospective</td><td colspan="1" rowspan="1">2/3</td><td colspan="1" rowspan="1">66.7 (20.8-93.9)</td><td colspan="1" rowspan="1">52/52</td><td colspan="1" rowspan="1">100 (93.1-100)</td></tr><tr><td colspan="1" rowspan="1">Contrived</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">9/9</td><td colspan="1" rowspan="1">100 (70.1-100)</td></tr><tr><td colspan="1" rowspan="1">Combined</td><td colspan="1" rowspan="1">4/8</td><td colspan="1" rowspan="1">50.0 (21.5-78.5)</td><td colspan="1" rowspan="1">79/79</td><td colspan="1" rowspan="1">100 (95.4-100)</td></tr><tr><td colspan="1" rowspan="4">Proteus mirabilis</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">2/5</td><td colspan="1" rowspan="1">40.0 (11.8-76.9)</td><td colspan="1" rowspan="1">18/18</td><td colspan="1" rowspan="1">100 (82.4-100)</td></tr><tr><td colspan="1" rowspan="1">Retrospective</td><td colspan="1" rowspan="1">2/3</td><td colspan="1" rowspan="1">66.7 (20.8-93.9)</td><td colspan="1" rowspan="1">48/48</td><td colspan="1" rowspan="1">100 (92.6-100)</td></tr><tr><td colspan="1" rowspan="1">Contrived</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">9/9</td><td colspan="1" rowspan="1">100 (70.1-100)</td></tr><tr><td colspan="1" rowspan="1">Combined</td><td colspan="1" rowspan="1">4/8</td><td colspan="1" rowspan="1">50.0 (21.5-78.5)</td><td colspan="1" rowspan="1">75/75</td><td colspan="1" rowspan="1">100 (95.1-100)</td></tr><tr><td colspan="1" rowspan="4">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">0/1</td><td colspan="1" rowspan="1">0.0 (0.0-79.3)</td><td colspan="1" rowspan="1">27/27</td><td colspan="1" rowspan="1">100 (87.5-100)</td></tr><tr><td colspan="1" rowspan="1">Retrospective</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">---</td><td colspan="1" rowspan="1">60/60</td><td colspan="1" rowspan="1">100 (94.0-100)</td></tr><tr><td colspan="1" rowspan="1">Contrived</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">32/32</td><td colspan="1" rowspan="1">100 (89.3-100)</td></tr><tr><td colspan="1" rowspan="1">Combined</td><td colspan="1" rowspan="1">0/1</td><td colspan="1" rowspan="1">0.0 (0.0-79.3)</td><td colspan="1" rowspan="1">119/119</td><td colspan="1" rowspan="1">100 (96.9-100)</td></tr><tr><td colspan="1" rowspan="4">Salmonella</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">---</td><td colspan="1" rowspan="1">2/2</td><td colspan="1" rowspan="1">100 (34.2-100)</td></tr><tr><td colspan="1" rowspan="1">Retrospective</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100 (20.7-100)</td><td colspan="1" rowspan="1">18/18</td><td colspan="1" rowspan="1">100 (82.4-100)</td></tr><tr><td colspan="1" rowspan="1">Contrived</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">35/35</td><td colspan="1" rowspan="1">100 (90.1-100)</td></tr><tr><td colspan="1" rowspan="1">Combined</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100 (20.7-100)</td><td colspan="1" rowspan="1">55/55</td><td colspan="1" rowspan="1">100 (93.5-100)</td></tr><tr><td colspan="1" rowspan="4">Serratia</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">100 (72.2-100)</td></tr><tr><td colspan="1" rowspan="1">Retrospective</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">--</td><td colspan="1" rowspan="1">34/34</td><td colspan="1" rowspan="1">100 (89.8-100)</td></tr><tr><td colspan="1" rowspan="1">Contrived</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">---</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">100 (90.4-100)</td></tr><tr><td colspan="1" rowspan="1">Combined</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">---</td><td colspan="1" rowspan="1">80/80</td><td colspan="1" rowspan="1">100 (95.4-100)</td></tr><tr><td colspan="1" rowspan="4">Serratia marcescens</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">9/9</td><td colspan="1" rowspan="1">100 (70.1-100)</td></tr><tr><td colspan="1" rowspan="1">Retrospective</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">34/34</td><td colspan="1" rowspan="1">100 (89.8-100)</td></tr><tr><td colspan="1" rowspan="1">Contrived</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">---</td><td colspan="1" rowspan="1">19/19</td><td colspan="1" rowspan="1">100 (83.2-100)</td></tr><tr><td colspan="1" rowspan="1">Combined</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">62/62</td><td colspan="1" rowspan="1">100 (94.2-100)</td></tr><tr><td colspan="1" rowspan="4">Stenotrophomonas maltophilia</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">100 (51.0-100)</td></tr><tr><td colspan="1" rowspan="1">Retrospective</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">100 (72.2-100)</td></tr><tr><td colspan="1" rowspan="1">Contrived</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">100 (90.4-100)</td></tr><tr><td colspan="1" rowspan="1">Combined</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">50/50</td><td colspan="1" rowspan="1">100 (92.9-100)</td></tr></table>

A comparison of CTX-M identified by comparator methods versus the ePlex BCID-GN Panel results are shown in Table 39 for prospective and retrospective samples.

Table 39: Distribution of CTX-M in Clinical Samples   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=4>Comparator Method</td></tr><tr><td rowspan=1 colspan=1>BCID-GN</td><td rowspan=1 colspan=1>Org+/ARG+</td><td rowspan=1 colspan=1>Org+/ARG-</td><td rowspan=1 colspan=1>Org-</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Org+/ARG+</td><td rowspan=1 colspan=1>74</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>76</td></tr><tr><td rowspan=1 colspan=1>Org+/ARG-</td><td rowspan=1 colspan=1>8A</td><td rowspan=1 colspan=1>741</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>751</td></tr><tr><td rowspan=1 colspan=1>Org-</td><td rowspan=1 colspan=1>3B</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>83</td><td rowspan=1 colspan=1>99</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>85</td><td rowspan=1 colspan=1>754</td><td rowspan=1 colspan=1>87</td><td rowspan=1 colspan=1>926</td></tr></table>

$\%$ Agreement $( 9 5 \% \mathrm { C I } )$ for $\mathrm { O r g ^ { + / A R G ^ { + } } }$ : $7 4 / 8 5 { = } 8 7 . 1 \%$ (78.3-92.6) $\%$ Agreement $( 9 5 \% \mathrm { C I } )$ for Org+/ARG-: $7 4 1 / 7 5 4 { = } 9 8 . 3 \%$ (97.1-99.0) $\%$ Agreement $( 9 5 \% \mathrm { C I } )$ for Org-: $8 3 / 8 7 { = } 9 5 . 4 \%$ (88.8-98.2)

A. Further testing of these 8 false negative samples indicated that 7 of the 8 samples may have been contaminated during the original extraction process and misidentified as having CTX-M present. Specifically, results for 7 of the 8 samples were negative for CTX-M from the following additional testing: 1) $\mathrm { \ q P C R }$ testing of 2 repeat extractions from the original sample, 2) qPCR testing of an extraction from the isolate, and 3) testing of the original sample with an FDA-cleared multiplex assay. For these 7 samples, the originally extracted sample was re-tested with $\mathsf { q P C R }$ and CTX-M was again detected, suggesting contamination during the original extraction process. The remaining 8th sample was positive for CTX-M from the repeat extractions, negative for CTX-M from the isolate, and negative for CTX-M when tested with an FDA-cleared multiplex assay. These inconsistent detection results suggest the 8th sample may be a true low-copy CTX-M positive sample.

B. For these 3 samples, CTX-M signal was above the threshold for detection; however, an associated organism was not detected by the ePlex BCID-GN Panel and the CTX-M target was reported as ‘N/A’.

# IMP

The PPA and NPA of the BCID-GN Panel IMP target stratified by the contrived organism are shown below in Table 40. No prospective or retrospective samples were found to contain IMP.

Table 40: Clinical Performance of IMP Target in Contrived Samples   

<table><tr><td rowspan=2 colspan=1>Species Detected by Comparator Method</td><td rowspan=1 colspan=2>Sensitivity/PPA</td><td rowspan=1 colspan=2>Specificity/NPA</td></tr><tr><td rowspan=1 colspan=1>TP/TP+FN</td><td rowspan=1 colspan=1>% (95% CI)</td><td rowspan=1 colspan=1>TN/TN+FP</td><td rowspan=1 colspan=1>% (95% CI)</td></tr><tr><td rowspan=1 colspan=1>Acinetobacter baumannii</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>74/74</td><td rowspan=1 colspan=1>100 (95.1-100)</td></tr><tr><td rowspan=1 colspan=1>Citrobacter</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>69/69</td><td rowspan=1 colspan=1>100 (94.7-100)</td></tr><tr><td rowspan=1 colspan=1>Enterobacter (non-cloacae complex)</td><td rowspan=1 colspan=1>5/5</td><td rowspan=1 colspan=1>100 (56.6-100)</td><td rowspan=1 colspan=1>53/53</td><td rowspan=1 colspan=1>100 (93.2-100)</td></tr><tr><td rowspan=1 colspan=1>Enterobacter cloacae complex</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>106/106</td><td rowspan=1 colspan=1>100 (96.5-100)</td></tr><tr><td rowspan=1 colspan=1>Escherichia coli</td><td rowspan=1 colspan=1>7/7</td><td rowspan=1 colspan=1>100 (64.6-100)</td><td rowspan=1 colspan=1>318/318</td><td rowspan=1 colspan=1>100 (98.8-100)</td></tr><tr><td rowspan=1 colspan=1>Klebsiella oxytoca</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>67/67</td><td rowspan=1 colspan=1>100 (94.6-100)</td></tr><tr><td rowspan=1 colspan=1>Klebsiella pneumoniae group</td><td rowspan=1 colspan=1>14/14</td><td rowspan=1 colspan=1>100 (78.5-100)</td><td rowspan=1 colspan=1>227/227</td><td rowspan=1 colspan=1>100 (98.3-100)</td></tr><tr><td rowspan=1 colspan=1>Morganella morganii</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>62/62</td><td rowspan=1 colspan=1>100 (94.2-100)</td></tr><tr><td rowspan=1 colspan=1>Proteus</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>87/87</td><td rowspan=1 colspan=1>100 (95.8-100)</td></tr><tr><td rowspan=1 colspan=1>Proteus mirabilis</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>83/83</td><td rowspan=1 colspan=1>100 (95.6-100)</td></tr><tr><td rowspan=1 colspan=1>Pseudomonas aeruginosa</td><td rowspan=1 colspan=1>13/13</td><td rowspan=1 colspan=1>100 (77.2-100)</td><td rowspan=1 colspan=1>107/107</td><td rowspan=1 colspan=1>100 (96.5-100)</td></tr><tr><td rowspan=1 colspan=1>Salmonella</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>56/56</td><td rowspan=1 colspan=1>100 (93.6-100)</td></tr><tr><td rowspan=1 colspan=1>Serratia</td><td rowspan=1 colspan=1>1/1</td><td rowspan=1 colspan=1>100 (20.7-100)</td><td rowspan=1 colspan=1>79/79</td><td rowspan=1 colspan=1>100 (95.4-100)</td></tr><tr><td rowspan=1 colspan=1>Serratia marcescens</td><td rowspan=1 colspan=1>1/1</td><td rowspan=1 colspan=1>100 (20.7-100)</td><td rowspan=1 colspan=1>61/61</td><td rowspan=1 colspan=1>100 (94.1-100)</td></tr></table>

A comparison of IMP identified by comparator methods versus the ePlex BCID-GN Panel results are shown in Table 41 for prospective and retrospective samples.

Table 41: Distribution of IMP in Clinical Samples   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=4>Comparator Method</td></tr><tr><td rowspan=1 colspan=1>BCID-GN</td><td rowspan=1 colspan=1>Org+/ARG+</td><td rowspan=1 colspan=1>Org+/ARG-</td><td rowspan=1 colspan=1>Org-</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Org+/ARG+</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Org+/ARG-</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>812</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>816</td></tr><tr><td rowspan=1 colspan=1>Org-</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>93</td><td rowspan=1 colspan=1>110</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>829</td><td rowspan=1 colspan=1>97</td><td rowspan=1 colspan=1>926</td></tr></table>

$\%$ Agreement $( 9 5 \% \mathrm { C I } )$ for Org+/ARG+: $0 / 0 { = } \mathbf { N } / \mathbf { A }$ $\%$ Agreement $( 9 5 \%$ CI) for Org+/ARG-: $8 1 2 / 8 2 9 { = } 9 7 . 9 \%$ (96.7-98.7) $\%$ Agreement $( 9 5 \% \mathrm { C I } )$ for Org-: $9 3 / 9 7 { = } 9 5 . 9 \%$ (89.9-98.4)

# KPC

The PPA and NPA of the BCID-GN Panel KPC target stratified by the organism identified by comparator methods for prospective, retrospective and contrived samples are shown in Table 42.

Table 42: Clinical Performance of KPC Target in Samples with Associated Organisms Detected by Comparator Methods   

<table><tr><td colspan="2" rowspan="2">Species Detected by Comparator Method</td><td colspan="2" rowspan="1">Sensitivity/PPA</td><td colspan="2" rowspan="1">Specificity/NPA</td></tr><tr><td colspan="1" rowspan="1">TP/TP+FN</td><td colspan="1" rowspan="1">% (95% CI)</td><td colspan="1" rowspan="1">TN/TN+FP</td><td colspan="1" rowspan="1">% (95% CI)</td></tr><tr><td colspan="1" rowspan="4">Acinetobacter baumannii</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">100 (51.0-100)</td></tr><tr><td colspan="1" rowspan="1">Retrospective</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">---</td><td colspan="1" rowspan="1">15/15</td><td colspan="1" rowspan="1">100 (79.6-100)</td></tr><tr><td colspan="1" rowspan="1">Contrived</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">---</td><td colspan="1" rowspan="1">55/55</td><td colspan="1" rowspan="1">100 (93.5-100)</td></tr><tr><td colspan="1" rowspan="1">Combined</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">---</td><td colspan="1" rowspan="1">74/74</td><td colspan="1" rowspan="1">100 (95.1-100)</td></tr><tr><td colspan="1" rowspan="4">Citrobacter</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">5/5</td><td colspan="1" rowspan="1">100 (56.6-100)</td></tr><tr><td colspan="1" rowspan="1">Retrospective</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">---</td><td colspan="1" rowspan="1">21/21</td><td colspan="1" rowspan="1">100 (84.5-100)</td></tr><tr><td colspan="1" rowspan="1">Contrived</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">100 (51.0-100)</td><td colspan="1" rowspan="1">39/39</td><td colspan="1" rowspan="1">100 (91.0-100)</td></tr><tr><td colspan="1" rowspan="1">Combined</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">100 (51.0-100)</td><td colspan="1" rowspan="1">65/65</td><td colspan="1" rowspan="1">100 (94.4-100)</td></tr><tr><td colspan="1" rowspan="4">Cronobacter sakazakii</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">---</td><td colspan="1" rowspan="1">--</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Retrospective</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">---</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100 (20.7-100)</td></tr><tr><td colspan="1" rowspan="1">Contrived</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">---</td><td colspan="1" rowspan="1">45/45</td><td colspan="1" rowspan="1">100 (92.1-100)</td></tr><tr><td colspan="1" rowspan="1">Combined</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">---</td><td colspan="1" rowspan="1">46/46</td><td colspan="1" rowspan="1">100 (92.3-100)</td></tr><tr><td colspan="1" rowspan="4">Enterobacter (non-cloacae complex)</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">---</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">100 (72.2-100)</td></tr><tr><td colspan="1" rowspan="1">Retrospective</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">---</td><td colspan="1" rowspan="1">12/12</td><td colspan="1" rowspan="1">100 (75.8-100)</td></tr><tr><td colspan="1" rowspan="1">Contrived</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">---</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">100 (90.4-100)</td></tr><tr><td colspan="1" rowspan="1">Combined</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">---</td><td colspan="1" rowspan="1">58/58</td><td colspan="1" rowspan="1">100 (93.8-100)</td></tr><tr><td colspan="1" rowspan="4">Enterobacter cloacae complex</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">19/19</td><td colspan="1" rowspan="1">100 (83.2-100)</td></tr><tr><td colspan="1" rowspan="1">Retrospective</td><td colspan="1" rowspan="1">0/1</td><td colspan="1" rowspan="1">0.0 (0.0-79.3)</td><td colspan="1" rowspan="1">49/49</td><td colspan="1" rowspan="1">100 (92.7-100)</td></tr><tr><td colspan="1" rowspan="1">Contrived</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">100 (43.9-100)</td><td colspan="1" rowspan="1">34/34</td><td colspan="1" rowspan="1">100 (89.8-100)</td></tr><tr><td colspan="1" rowspan="1">Combined</td><td colspan="1" rowspan="1">3/4</td><td colspan="1" rowspan="1">75.0 (30.1-95.4)</td><td colspan="1" rowspan="1">102/102</td><td colspan="1" rowspan="1">100 (96.4-100)</td></tr><tr><td colspan="1" rowspan="4">Escherichia coli</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100 (20.7-100)</td><td colspan="1" rowspan="1">132/132</td><td colspan="1" rowspan="1">100 (97.2-100)</td></tr><tr><td colspan="1" rowspan="1">Retrospective</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">---</td><td colspan="1" rowspan="1">140/140</td><td colspan="1" rowspan="1">100 (97.3-100)</td></tr><tr><td colspan="1" rowspan="1">Contrived</td><td colspan="1" rowspan="1">6/6</td><td colspan="1" rowspan="1">100 (61.0-100)</td><td colspan="1" rowspan="1">46/46</td><td colspan="1" rowspan="1">100 (92.3-100)</td></tr><tr><td colspan="1" rowspan="1">Combined</td><td colspan="1" rowspan="1">7/7</td><td colspan="1" rowspan="1">100 (64.6-100)</td><td colspan="1" rowspan="1">318/318</td><td colspan="1" rowspan="1">100 (98.8-100)</td></tr><tr><td colspan="1" rowspan="4">Klebsiella oxytoca</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">13/13</td><td colspan="1" rowspan="1">100 (77.2-100)</td></tr><tr><td colspan="1" rowspan="1">Retrospective</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">---</td><td colspan="1" rowspan="1">34/34</td><td colspan="1" rowspan="1">100 (89.8-100)</td></tr><tr><td colspan="1" rowspan="1">Contrived</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">---</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">100 (83.9-100)</td></tr><tr><td colspan="1" rowspan="1">Combined</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">---</td><td colspan="1" rowspan="1">67/67</td><td colspan="1" rowspan="1">100 (94.6-100)</td></tr><tr><td colspan="1" rowspan="4">Klebsiella pneumoniae group</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">2/2</td><td colspan="1" rowspan="1">100 (34.2-100)</td><td colspan="1" rowspan="1">59/59</td><td colspan="1" rowspan="1">100 (93.9-100)</td></tr><tr><td colspan="1" rowspan="1">Retrospective</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">100 (51.0-100)</td><td colspan="1" rowspan="1">103/104</td><td colspan="1" rowspan="1">99.0 (94.8-99.8)</td></tr><tr><td colspan="1" rowspan="1">Contrived</td><td colspan="1" rowspan="1">22/22</td><td colspan="1" rowspan="1">100 (85.1-100)</td><td colspan="1" rowspan="1">50/50</td><td colspan="1" rowspan="1">100 (92.9-100)</td></tr><tr><td colspan="1" rowspan="1">Combined</td><td colspan="1" rowspan="1">28/28</td><td colspan="1" rowspan="1">100 (87.9-100)</td><td colspan="1" rowspan="1">212/213</td><td colspan="1" rowspan="1">99.5 (97.4-99.9)</td></tr><tr><td colspan="1" rowspan="4">Morganella morganii</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">100 (43.9-100)</td></tr><tr><td colspan="1" rowspan="1">Retrospective</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">100 (72.2-100)</td></tr><tr><td colspan="1" rowspan="1">Contrived</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">100 (51.0-100)</td><td colspan="1" rowspan="1">45/45</td><td colspan="1" rowspan="1">100 (92.1-100)</td></tr><tr><td colspan="1" rowspan="1">Combined</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">100 (51.0-100)</td><td colspan="1" rowspan="1">58/58</td><td colspan="1" rowspan="1">100 (93.8-100)</td></tr><tr><td colspan="1" rowspan="4">Proteus</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">23/23</td><td colspan="1" rowspan="1">100 (85.7-100)</td></tr><tr><td colspan="1" rowspan="1">Retrospective</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">55/55</td><td colspan="1" rowspan="1">100 (93.5-100)</td></tr><tr><td colspan="1" rowspan="1">Contrived</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">100 (51.0-100)</td><td colspan="1" rowspan="1">5/5</td><td colspan="1" rowspan="1">100 (56.6-100)</td></tr><tr><td colspan="1" rowspan="1">Combined</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">100 (51.0-100)</td><td colspan="1" rowspan="1">83/83</td><td colspan="1" rowspan="1">100 (95.6-100)</td></tr><tr><td colspan="1" rowspan="4">Proteus mirabilis</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">23/23</td><td colspan="1" rowspan="1">100 (85.7-100)</td></tr><tr><td colspan="1" rowspan="1">Retrospective</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">51/51</td><td colspan="1" rowspan="1">100 (93.0-100)</td></tr><tr><td colspan="1" rowspan="1">Contrived</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">100 (51.0-100)</td><td colspan="1" rowspan="1">5/5</td><td colspan="1" rowspan="1">100 (56.6-100)</td></tr><tr><td colspan="1" rowspan="1">Combined</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">100 (51.0-100)</td><td colspan="1" rowspan="1">79/79</td><td colspan="1" rowspan="1">100 (95.4-100)</td></tr><tr><td colspan="1" rowspan="4">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">28/28</td><td colspan="1" rowspan="1">100 (87.9-100)</td></tr><tr><td colspan="1" rowspan="1">Retrospective</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">60/60</td><td colspan="1" rowspan="1">100 (94.0-100)</td></tr><tr><td colspan="1" rowspan="1">Contrived</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100 (20.7-100)</td><td colspan="1" rowspan="1">31/31</td><td colspan="1" rowspan="1">100 (89.0-100)</td></tr><tr><td colspan="1" rowspan="1">Combined</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100 (20.7-100)</td><td colspan="1" rowspan="1">119/119</td><td colspan="1" rowspan="1">100 (96.9-100)</td></tr><tr><td colspan="1" rowspan="4">Salmonella</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">---</td><td colspan="1" rowspan="1">2/2</td><td colspan="1" rowspan="1">100 (34.2-100)</td></tr><tr><td colspan="1" rowspan="1">Retrospective</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">--</td><td colspan="1" rowspan="1">19/19</td><td colspan="1" rowspan="1">100 (83.2-100)</td></tr><tr><td colspan="1" rowspan="1">Contrived</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">35/35</td><td colspan="1" rowspan="1">100 (90.1-100)</td></tr><tr><td colspan="1" rowspan="1">Combined</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">--</td><td colspan="1" rowspan="1">56/56</td><td colspan="1" rowspan="1">100 (93.6-100)</td></tr><tr><td colspan="1" rowspan="4">Serratia</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">---</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">100 (72.2-100)</td></tr><tr><td colspan="1" rowspan="1">Retrospective</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">---</td><td colspan="1" rowspan="1">34/34</td><td colspan="1" rowspan="1">100 (89.8-100)</td></tr><tr><td colspan="1" rowspan="1">Contrived</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">---</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">100 (90.4-100)</td></tr><tr><td colspan="1" rowspan="1">Combined</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">80/80</td><td colspan="1" rowspan="1">100 (95.4-100)</td></tr><tr><td colspan="1" rowspan="4">Serratia marcescens</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">---</td><td colspan="1" rowspan="1">9/9</td><td colspan="1" rowspan="1">100 (70.1-100)</td></tr><tr><td colspan="1" rowspan="1">Retrospective</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">---</td><td colspan="1" rowspan="1">34/34</td><td colspan="1" rowspan="1">100 (89.8-100)</td></tr><tr><td colspan="1" rowspan="1">Contrived</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">---</td><td colspan="1" rowspan="1">19/19</td><td colspan="1" rowspan="1">100 (83.2-100)</td></tr><tr><td colspan="1" rowspan="1">Combined</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">62/62</td><td colspan="1" rowspan="1">100 (94.2-100)</td></tr></table>

A comparison of KPC identified by comparator methods versus the ePlex BCID-GN Panel results are shown in Table 43 for prospective and retrospective samples.

Table 43: Distribution of KPC in Clinical Samples   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=4>Comparator Method</td></tr><tr><td rowspan=1 colspan=1>BCID-GN</td><td rowspan=1 colspan=1>Org+/ARG+</td><td rowspan=1 colspan=1>Org+/ARG-</td><td rowspan=1 colspan=1>Org-</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Org+/ARG+</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>8</td></tr><tr><td rowspan=1 colspan=1>Org+/ARG-</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>804</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>809</td></tr><tr><td rowspan=1 colspan=1>Org-</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>92</td><td rowspan=1 colspan=1>109</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>822</td><td rowspan=1 colspan=1>96</td><td rowspan=1 colspan=1>926</td></tr></table>

$\%$ Agreement $( 9 5 \% \mathrm { C I } )$ for $\mathrm { O r g ^ { + / A R G ^ { + } } }$ : $7 / 8 { = } 8 7 . 5 \%$ (52.9-97.8) $\%$ Agreement $( 9 5 \% \mathrm { C I } )$ for Org+/ARG-: $8 0 4 / 8 2 2 { = } 9 7 . 8 \%$ (96.6-98.6) $\%$ Agreement $( 9 5 \%$ CI) for Org-: $9 2 / 9 6 { = } 9 5 . 8 \%$ (89.8-98.4)

# NDM

The PPA and NPA of the BCID-GN Panel NDM target stratified by the contrived organism are shown below in Table 44. No prospective or retrospective samples were found to contain NDM.

Table 44: Clinical Performance of NDM Target in Contrived Samples by Organism   

<table><tr><td rowspan=2 colspan=1>Species Detected by Comparator Method</td><td rowspan=1 colspan=2>Sensitivity/PPA</td><td rowspan=1 colspan=2>Specificity/NPA</td></tr><tr><td rowspan=1 colspan=1>TP/TP+FN</td><td rowspan=1 colspan=1>% (95% CI)</td><td rowspan=1 colspan=1>TN/TN+FP</td><td rowspan=1 colspan=1>% (95% CI)</td></tr><tr><td rowspan=1 colspan=1>Acinetobacter baumannii</td><td rowspan=1 colspan=1>5/5</td><td rowspan=1 colspan=1>100 (56.6-100)</td><td rowspan=1 colspan=1>69/69</td><td rowspan=1 colspan=1>100 (94.7-100)</td></tr><tr><td rowspan=1 colspan=1>Citrobacter</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>---</td><td rowspan=1 colspan=1>69/69</td><td rowspan=1 colspan=1>100 (94.7-100)</td></tr><tr><td rowspan=1 colspan=1>Enterobacter (non-cloacae complex)</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>--</td><td rowspan=1 colspan=1>58/58</td><td rowspan=1 colspan=1>100 (93.8-100)</td></tr><tr><td rowspan=1 colspan=1>Enterobacter cloacae complex</td><td rowspan=1 colspan=1>13/13</td><td rowspan=1 colspan=1>100 (77.2-100)</td><td rowspan=1 colspan=1>93/93</td><td rowspan=1 colspan=1>100 (96.0-100)</td></tr><tr><td rowspan=1 colspan=1>Escherichia coli</td><td rowspan=1 colspan=1>23/23</td><td rowspan=1 colspan=1>100 (85.7-100)</td><td rowspan=1 colspan=1>302/302</td><td rowspan=1 colspan=1>100 (98.7-100)</td></tr><tr><td rowspan=1 colspan=1>Klebsiella oxytoca</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>---</td><td rowspan=1 colspan=1>67/67</td><td rowspan=1 colspan=1>100 (94.6-100)</td></tr><tr><td rowspan=1 colspan=1>Klebsiella pneumoniae group</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>100 (43.9-100)</td><td rowspan=1 colspan=1>238/238</td><td rowspan=1 colspan=1>100 (98.4-100)</td></tr><tr><td rowspan=1 colspan=1>Morganella morganii</td><td rowspan=1 colspan=1>5/5</td><td rowspan=1 colspan=1>100 (56.6-100)</td><td rowspan=1 colspan=1>57/57</td><td rowspan=1 colspan=1>100 (93.7-100)</td></tr><tr><td rowspan=1 colspan=1>Proteus</td><td rowspan=1 colspan=1>5/5</td><td rowspan=1 colspan=1>100 (56.6-100)</td><td rowspan=1 colspan=1>82/82</td><td rowspan=1 colspan=1>100 (95.5-100)</td></tr><tr><td rowspan=1 colspan=1>Proteus mirabilis</td><td rowspan=1 colspan=1>5/5</td><td rowspan=1 colspan=1>100 (56.6-100)</td><td rowspan=1 colspan=1>78/78</td><td rowspan=1 colspan=1>100 (95.3-100)</td></tr><tr><td rowspan=1 colspan=1>Pseudomonas aeruginosa</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>120/120</td><td rowspan=1 colspan=1>100 (96.9-100)</td></tr><tr><td rowspan=1 colspan=1>Salmonella</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>--</td><td rowspan=1 colspan=1>56/56</td><td rowspan=1 colspan=1>100 (93.6-100)</td></tr><tr><td rowspan=1 colspan=1>Serratia</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>---</td><td rowspan=1 colspan=1>80/80</td><td rowspan=1 colspan=1>100 (95.4-100)</td></tr><tr><td rowspan=1 colspan=1>Serratia marcescens</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>--</td><td rowspan=1 colspan=1>62/62</td><td rowspan=1 colspan=1>100 (94.2-100)</td></tr></table>

A comparison of NDM identified by comparator methods versus the ePlex BCID-GN Panel results are shown in Table 45 for prospective and retrospective samples.

Table 45: Distribution of NDM in Clinical Samples   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=4>Comparator Method</td></tr><tr><td rowspan=1 colspan=1>BCID-GN</td><td rowspan=1 colspan=1>Org+/ARG+</td><td rowspan=1 colspan=1>Org+/ARG-</td><td rowspan=1 colspan=1>Org-</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Org+/ARG+</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Org+/ARG-</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>812</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>816</td></tr><tr><td rowspan=1 colspan=1>Org-</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>93</td><td rowspan=1 colspan=1>110</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>829</td><td rowspan=1 colspan=1>97</td><td rowspan=1 colspan=1>926</td></tr></table>

% Agreement $( 9 5 \% \mathrm { C l } )$ ) for $\mathsf { O r g } { + } / \mathsf { A R G } { + } : 0 / 0 { = } \mathsf { N } / \mathsf { A }$ $\%$ Agreement $( 9 5 \% \mathrm { C l } )$ ) for Org+/ARG‐: 812/ $8 2 9 { = } 9 7 . 9 \%$ (96.7‐98.7) $\%$ Agreement $( 9 5 \% \mathrm { C l } )$ ) for Org‐: $9 3 / 9 7 { = } 9 5 . 9 \%$ (89.9‐98.4)

# OXA

The PPA and NPA of the BCID-GN Panel OXA target stratified by target identified by comparator methods for prospective, retrospective, and contrived samples are shown in Table 46.

Table 46: Clinical Performance of OXA Target in Samples with Associated Organisms Detected by Comparator Methods   

<table><tr><td colspan="2" rowspan="2">Species Detected by Comparator Method</td><td colspan="2" rowspan="1">Sensitivity/PPA</td><td colspan="2" rowspan="1">Specificity/NPA</td></tr><tr><td colspan="1" rowspan="1">TP/TP+FN</td><td colspan="1" rowspan="1">% (95% CI)</td><td colspan="1" rowspan="1">TN/TN+FP</td><td colspan="1" rowspan="1">% (95% CI)</td></tr><tr><td colspan="1" rowspan="4">Acinetobacter baumannii</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100 (20.7-100)</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">100 (43.9-100)</td></tr><tr><td colspan="1" rowspan="1">Retrospective</td><td colspan="1" rowspan="1">7/7</td><td colspan="1" rowspan="1">100 (64.6-100)</td><td colspan="1" rowspan="1">8/8</td><td colspan="1" rowspan="1">100 (67.6-100)</td></tr><tr><td colspan="1" rowspan="1">Contrived</td><td colspan="1" rowspan="1">13/13</td><td colspan="1" rowspan="1">100 (77.2-100)</td><td colspan="1" rowspan="1">42/42</td><td colspan="1" rowspan="1">100 (91.6-100)</td></tr><tr><td colspan="1" rowspan="1">Combined</td><td colspan="1" rowspan="1">21/21</td><td colspan="1" rowspan="1">100 (84.5-100)</td><td colspan="1" rowspan="1">53/53</td><td colspan="1" rowspan="1">100 (93.2-100)</td></tr><tr><td colspan="1" rowspan="4">Citrobacter</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">---</td><td colspan="1" rowspan="1">5/5</td><td colspan="1" rowspan="1">100 (56.6-100)</td></tr><tr><td colspan="1" rowspan="1">Retrospective</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">---</td><td colspan="1" rowspan="1">21/21</td><td colspan="1" rowspan="1">100 (84.5-100)</td></tr><tr><td colspan="1" rowspan="1">Contrived</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">---</td><td colspan="1" rowspan="1">43/43</td><td colspan="1" rowspan="1">100 (91.8-100)</td></tr><tr><td colspan="1" rowspan="1">Combined</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">---</td><td colspan="1" rowspan="1">69/69</td><td colspan="1" rowspan="1">100 (94.7-100)</td></tr><tr><td colspan="1" rowspan="4">Enterobacter (non-cloacae complex)</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">100 (72.2-100)</td></tr><tr><td colspan="1" rowspan="1">Retrospective</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">---</td><td colspan="1" rowspan="1">12/12</td><td colspan="1" rowspan="1">100 (75.8-100)</td></tr><tr><td colspan="1" rowspan="1">Contrived</td><td colspan="1" rowspan="1">12/12</td><td colspan="1" rowspan="1">100 (75.8-100)</td><td colspan="1" rowspan="1">24/24</td><td colspan="1" rowspan="1">100 (86.2-100)</td></tr><tr><td colspan="1" rowspan="1">Combined</td><td colspan="1" rowspan="1">12/12</td><td colspan="1" rowspan="1">100 (75.8-100)</td><td colspan="1" rowspan="1">46/46</td><td colspan="1" rowspan="1">100 (92.3-100)</td></tr><tr><td colspan="1" rowspan="4">Enterobacter cloacae complex</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">---</td><td colspan="1" rowspan="1">19/19</td><td colspan="1" rowspan="1">100 (83.2-100)</td></tr><tr><td colspan="1" rowspan="1">Retrospective</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">---</td><td colspan="1" rowspan="1">50/50</td><td colspan="1" rowspan="1">100 (92.9-100)</td></tr><tr><td colspan="1" rowspan="1">Contrived</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">--</td><td colspan="1" rowspan="1">37/37</td><td colspan="1" rowspan="1">100 (90.6-100)</td></tr><tr><td colspan="1" rowspan="1">Combined</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">106/106</td><td colspan="1" rowspan="1">100 (96.5-100)</td></tr><tr><td colspan="1" rowspan="4">Escherichia coli</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">0/1</td><td colspan="1" rowspan="1">0.0 (0.0-79.3)</td><td colspan="1" rowspan="1">132/132</td><td colspan="1" rowspan="1">100 (97.2-100)</td></tr><tr><td colspan="1" rowspan="1">Retrospective</td><td colspan="1" rowspan="1">1/2</td><td colspan="1" rowspan="1">50.0 (9.5-90.5)</td><td colspan="1" rowspan="1">138/138</td><td colspan="1" rowspan="1">100 (97.3-100)</td></tr><tr><td colspan="1" rowspan="1">Contrived</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">52/52</td><td colspan="1" rowspan="1">100 (93.1-100)</td></tr><tr><td colspan="1" rowspan="1">Combined</td><td colspan="1" rowspan="1">1/3</td><td colspan="1" rowspan="1">33.3 (6.1-79.2)</td><td colspan="1" rowspan="1">322/322</td><td colspan="1" rowspan="1">100 (98.8-100)</td></tr><tr><td colspan="1" rowspan="4">Klebsiella oxytoca</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">13/13</td><td colspan="1" rowspan="1">100 (77.2-100)</td></tr><tr><td colspan="1" rowspan="1">Retrospective</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">---</td><td colspan="1" rowspan="1">34/34</td><td colspan="1" rowspan="1">100 (89.8-100)</td></tr><tr><td colspan="1" rowspan="1">Contrived</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">---</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">100 (83.9-100)</td></tr><tr><td colspan="1" rowspan="1">Combined</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">67/67</td><td colspan="1" rowspan="1">100 (94.6-100)</td></tr><tr><td colspan="1" rowspan="4">Klebsiella pneumoniae group</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">---</td><td colspan="1" rowspan="1">61/61</td><td colspan="1" rowspan="1">100 (94.1-100)</td></tr><tr><td colspan="1" rowspan="1">Retrospective</td><td colspan="1" rowspan="1">0/1</td><td colspan="1" rowspan="1">0.0 (0.0-79.3)</td><td colspan="1" rowspan="1">107/107</td><td colspan="1" rowspan="1">100 (96.5-100)</td></tr><tr><td colspan="1" rowspan="1">Contrived</td><td colspan="1" rowspan="1">12/12</td><td colspan="1" rowspan="1">100 (75.8-100)</td><td colspan="1" rowspan="1">60/60</td><td colspan="1" rowspan="1">100 (94.0-100)</td></tr><tr><td colspan="1" rowspan="1">Combined</td><td colspan="1" rowspan="1">12/13</td><td colspan="1" rowspan="1">92.3 (66.7-98.6)</td><td colspan="1" rowspan="1">228/228</td><td colspan="1" rowspan="1">100 (98.3-100)</td></tr><tr><td colspan="1" rowspan="4">Morganella morganii</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">100 (43.9-100)</td></tr><tr><td colspan="1" rowspan="1">Retrospective</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">100 (72.2-100)</td></tr><tr><td colspan="1" rowspan="1">Contrived</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">---</td><td colspan="1" rowspan="1">49/49</td><td colspan="1" rowspan="1">100 (92.7-100)</td></tr><tr><td colspan="1" rowspan="1">Combined</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">62/62</td><td colspan="1" rowspan="1">100 (94.2-100)</td></tr><tr><td colspan="1" rowspan="4">Proteus</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">---</td><td colspan="1" rowspan="1">23/23</td><td colspan="1" rowspan="1">100 (85.7-100)</td></tr><tr><td colspan="1" rowspan="1">Retrospective</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100 (20.7-100)</td><td colspan="1" rowspan="1">53/54</td><td colspan="1" rowspan="1">98.1 (90.2-99.7)</td></tr><tr><td colspan="1" rowspan="1">Contrived</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">---</td><td colspan="1" rowspan="1">9/9</td><td colspan="1" rowspan="1">100 (70.1-100)</td></tr><tr><td colspan="1" rowspan="1">Combined</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100 (20.7-100)</td><td colspan="1" rowspan="1">85/86</td><td colspan="1" rowspan="1">98.8 (93.7-99.8)</td></tr><tr><td colspan="1" rowspan="4">Proteus mirabilis</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">23/23</td><td colspan="1" rowspan="1">100 (85.7-100)</td></tr><tr><td colspan="1" rowspan="1">Retrospective</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">50/51</td><td colspan="1" rowspan="1">98.0 (89.7-99.7)</td></tr><tr><td colspan="1" rowspan="1">Contrived</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">--</td><td colspan="1" rowspan="1">9/9</td><td colspan="1" rowspan="1">100 (70.1-100)</td></tr><tr><td colspan="1" rowspan="1">Combined</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">---</td><td colspan="1" rowspan="1">82/83</td><td colspan="1" rowspan="1">98.8 (93.5-99.8)</td></tr><tr><td colspan="1" rowspan="4">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">---</td><td colspan="1" rowspan="1">28/28</td><td colspan="1" rowspan="1">100 (87.9-100)</td></tr><tr><td colspan="1" rowspan="1">Retrospective</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">---</td><td colspan="1" rowspan="1">59/60</td><td colspan="1" rowspan="1">98.3 (91.1-99.7)</td></tr><tr><td colspan="1" rowspan="1">Contrived</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">---</td><td colspan="1" rowspan="1">32/32</td><td colspan="1" rowspan="1">100 (89.3-100)</td></tr><tr><td colspan="1" rowspan="1">Combined</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">--</td><td colspan="1" rowspan="1">119/120</td><td colspan="1" rowspan="1">99.2 (95.4-99.9)</td></tr><tr><td colspan="1" rowspan="3">Salmonella</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2/2</td><td colspan="1" rowspan="1">100 (34.2-100)</td></tr><tr><td colspan="1" rowspan="1">Retrospective</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">19/19</td><td colspan="1" rowspan="1">100 (83.2-100)</td></tr><tr><td colspan="1" rowspan="1">Contrived</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">---</td><td colspan="1" rowspan="1">35/35</td><td colspan="1" rowspan="1">100 (90.1-100)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Combined</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">56/56</td><td colspan="1" rowspan="1">100 (93.6-100)</td></tr><tr><td colspan="1" rowspan="4">Serratia</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">---</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">100 (72.2-100)</td></tr><tr><td colspan="1" rowspan="1">Retrospective</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">34/34</td><td colspan="1" rowspan="1">100 (89.8-100)</td></tr><tr><td colspan="1" rowspan="1">Contrived</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">---</td><td colspan="1" rowspan="1">36/36</td><td colspan="1" rowspan="1">100 (90.4-100)</td></tr><tr><td colspan="1" rowspan="1">Combined</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">80/80</td><td colspan="1" rowspan="1">100 (95.4-100)</td></tr><tr><td colspan="1" rowspan="4">Serratia marcescens</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">---</td><td colspan="1" rowspan="1">9/9</td><td colspan="1" rowspan="1">100 (70.1-100)</td></tr><tr><td colspan="1" rowspan="1">Retrospective</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">34/34</td><td colspan="1" rowspan="1">100 (89.8-100)</td></tr><tr><td colspan="1" rowspan="1">Contrived</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">19/19</td><td colspan="1" rowspan="1">100 (83.2-100)</td></tr><tr><td colspan="1" rowspan="1">Combined</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">---</td><td colspan="1" rowspan="1">62/62</td><td colspan="1" rowspan="1">100 (94.2-100)</td></tr></table>

A comparison of OXA identified by comparator methods versus the ePlex BCID-GN Panel results are shown in Table 47 for prospective and retrospective samples.

Table 47: Distribution of OXA in Clinical Samples   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=4>Comparator Method</td></tr><tr><td rowspan=1 colspan=1>BCID-GN</td><td rowspan=1 colspan=1>Org+/ARG+</td><td rowspan=1 colspan=1>Org+/ARG-</td><td rowspan=1 colspan=1>Org-</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Org+/ARG+</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>12</td></tr><tr><td rowspan=1 colspan=1>Org+/ARG-</td><td rowspan=1 colspan=1>2A</td><td rowspan=1 colspan=1>798</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>804</td></tr><tr><td rowspan=1 colspan=1>Org-</td><td rowspan=1 colspan=1>1B</td><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>93</td><td rowspan=1 colspan=1>110</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>816</td><td rowspan=1 colspan=1>97</td><td rowspan=1 colspan=1>926</td></tr></table>

$\%$ Agreement $( 9 5 \% \mathrm { C I } )$ for Org+/ARG+: $1 0 / 1 3 { = } 7 6 . 9 \%$ (49.7-91.8) $\%$ Agreement $( 9 5 \% \mathrm { C I } )$ for $\mathrm { O r g ^ { + / A R G - } }$ : $7 9 8 / 8 1 6 { = } 9 7 . 8 \%$ (96.5-98.6) $\%$ Agreement $( 9 5 \% \mathrm { C I } )$ for Org-: $9 3 / 9 7 { = } 9 5 . 9 \%$ (89.9-98.4)

A. One false negative sample was tested with an FDA-cleared multiplex assay and OXA was not detected. The isolate from the remaining false negative sample tested negative for OXA-23 and OXA-48 by $\mathrm { { q P C R } }$ .   
B. For this sample, OXA signal was above the threshold for detection; however, an associated organism was not detected by the ePlex BCID-GN Panel and the OXA target was reported as $\mathbf { \hat { N } } / \mathbf { A } ^ { \prime }$ .

# VIM

The PPA and NPA of the BCID-GN Panel VIM target stratified by the contrived organism are shown below in Table 48. No prospective or retrospective samples were found to contain VIM.

Table 48: Clinical Performance of VIM Target in Contrived Samples by Organism   

<table><tr><td rowspan=2 colspan=1>Species Detected by Comparator Method</td><td rowspan=1 colspan=2>Sensitivity/PPA</td><td rowspan=1 colspan=2>Specificity/NPA</td></tr><tr><td rowspan=1 colspan=1>TP/TP+FN</td><td rowspan=1 colspan=1>% (95% CI)</td><td rowspan=1 colspan=1>TN/TN+FP</td><td rowspan=1 colspan=1>% (95% CI)</td></tr><tr><td rowspan=1 colspan=1>Acinetobacter baumannii</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>74/74</td><td rowspan=1 colspan=1>100 (95.1-100)</td></tr><tr><td rowspan=1 colspan=1>Citrobacter</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>69/69</td><td rowspan=1 colspan=1>100 (94.7-100)</td></tr><tr><td rowspan=1 colspan=1>Enterobacter (non-cloacae complex)</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>---</td><td rowspan=1 colspan=1>58/58</td><td rowspan=1 colspan=1>100 (93.8-100)</td></tr><tr><td rowspan=1 colspan=1>Enterobacter cloacae complex</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>100 (61.0-100)</td><td rowspan=1 colspan=1>100/100</td><td rowspan=1 colspan=1>100 (96.3-100)</td></tr><tr><td rowspan=1 colspan=1>Escherichia coli</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>100 (34.2-100)</td><td rowspan=1 colspan=1>323/323</td><td rowspan=1 colspan=1>100 (98.8-100)</td></tr><tr><td rowspan=1 colspan=1>Klebsiella oxytoca</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>---</td><td rowspan=1 colspan=1>67/67</td><td rowspan=1 colspan=1>100 (94.6-100)</td></tr><tr><td rowspan=1 colspan=1>Klebsiella pneumoniae group</td><td rowspan=1 colspan=1>16/16</td><td rowspan=1 colspan=1>100 (80.6-100)</td><td rowspan=1 colspan=1>225/225</td><td rowspan=1 colspan=1>100 (98.3-100)</td></tr><tr><td rowspan=1 colspan=1>Morganella morganii</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>62/62</td><td rowspan=1 colspan=1>100 (94.2-100)</td></tr><tr><td rowspan=1 colspan=1>Proteus</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>---</td><td rowspan=1 colspan=1>87/87</td><td rowspan=1 colspan=1>100 (95.8-100)</td></tr><tr><td rowspan=1 colspan=1>Proteus mirabilis</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>---</td><td rowspan=1 colspan=1>83/83</td><td rowspan=1 colspan=1>100 (95.6-100)</td></tr><tr><td rowspan=1 colspan=1>Pseudomonas aeruginosa</td><td rowspan=1 colspan=1>18/18</td><td rowspan=1 colspan=1>100 (82.4-100)</td><td rowspan=1 colspan=1>102/102</td><td rowspan=1 colspan=1>100 (96.4-100)</td></tr><tr><td rowspan=1 colspan=1>Salmonella</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>--</td><td rowspan=1 colspan=1>56/56</td><td rowspan=1 colspan=1>100 (93.6-100)</td></tr><tr><td rowspan=1 colspan=1>Serratia</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>--</td><td rowspan=1 colspan=1>80/80</td><td rowspan=1 colspan=1>100 (95.4-100)</td></tr><tr><td rowspan=1 colspan=1>Serratia marcescens</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>---</td><td rowspan=1 colspan=1>62/62</td><td rowspan=1 colspan=1>100 (94.2-100)</td></tr></table>

A comparison of VIM identified by comparator methods versus the ePlex BCID-GN Panel results are shown in Table 49 for prospective and retrospective samples.

Table 49: Distribution of VIM in Clinical Samples   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=4>Comparator Method</td></tr><tr><td rowspan=1 colspan=1>BCID-GN</td><td rowspan=1 colspan=1>Org+/ARG+</td><td rowspan=1 colspan=1>Org+/ARG-</td><td rowspan=1 colspan=1>Org-</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Org+/ARG+</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Org+/ARG-</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>812</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>816</td></tr><tr><td rowspan=1 colspan=1>Org-</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>93</td><td rowspan=1 colspan=1>110</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>829</td><td rowspan=1 colspan=1>97</td><td rowspan=1 colspan=1>926</td></tr></table>

$\%$ Agreement $( 9 5 \% \mathrm { C I } )$ for $\mathrm { O r g ^ { + / A R G ^ { + } } }$ : $0 / 0 { = } \mathbf { N } / \mathbf { A }$ $\%$ Agreement $( 9 5 \%$ CI) for Org+/ARG-: $8 1 2 / 8 2 9 { = } 9 7 . 9 \%$ (96.7-98.7) $\%$ Agreement $( 9 5 \%$ CI) for Org-: $9 3 / 9 7 { = } 9 5 . 9 \%$ (89.9-98.4)

# Resistance Markers and Antimicrobial Resistance Susceptibility

A supplemental comparison of the CTX-M BCID-GN Panel target versus phenotypic antimicrobial susceptibility testing (AST) for extended spectrum beta-lactamase (ESBL) activity, Ceftazidime, Ceftriaxone, and Aztreonam, and a combination of the 4 results (ESBL/Combo) is provided in Table 57 for clinical isolates with available AST results. In total, 162 isolates had ESBL confirmatory testing and 770 isolates had ESBL confirmatory testing and/or AST results for 1 or more of Ceftazidime, Ceftriaxone, or Aztreonam. A true positive (TP) result was defined where CTX-M was detected by the BCID-GN Panel and the isolate was positive for the ESBL confirmatory test or resistant (R) or intermediate (I) to the specific antimicrobial. A false negative (FN) result was defined similarly when CTX-M was not detected by the BCID-GN Panel. PPA was calculated as $1 0 0 \mathrm { { x } }$ (TP/(TP+FN)). A true negative (TN) result was defined where CTX-M was not detected by the BCID-GN Panel and the isolate was negative for the ESBL confirmatory test or susceptible (S) to the specific antimicrobial. A false positive (FP) result was defined similarly when CTX-M was detected by the BCID-GN Panel. NPA was calculated as $1 0 0 \mathbf { x }$ (TN/(TN+FP)). For the ESBL/Combo analysis, the phenotypic AST result was positive or negative based on the ESBL confirmatory test, if available. If the ESBL confirmatory test was not available, then the phenotypic AST result was positive if any of the 3 antimicrobials were resistant or intermediate, otherwise, the result was negative if any of the 3 antimicrobials were susceptible. Note: ESBL resistance can be due to mechanisms other than acquisition of the CTX-M resistance gene.

Table 50: Clinical Performance of the BCID-GN Panel target CTX-M to Phenotypic Antimicrobial Susceptibility Testing for ESBL, Ceftazidime, Ceftriaxone, and Aztreonam   

<table><tr><td colspan="1" rowspan="2">AssociatedOrganism</td><td colspan="2" rowspan="1">ESBL confirmatory</td><td colspan="2" rowspan="1">Ceftazidime</td><td colspan="2" rowspan="1">Ceftriaxone</td><td colspan="2" rowspan="1">Aztreonam</td><td colspan="2" rowspan="1">ESBL/Combo</td></tr><tr><td colspan="1" rowspan="1">PPATP/TP+FN(%)</td><td colspan="1" rowspan="1">NPATN/TN+FP(%)</td><td colspan="1" rowspan="1">PPATP/TP+FN(%)</td><td colspan="1" rowspan="1">NPATN/TN+FP(%)</td><td colspan="1" rowspan="1">PPATP/TP+FN(%)</td><td colspan="1" rowspan="1">NPATN/TN+FP(%)</td><td colspan="1" rowspan="1">PPATP/TP+FN(%)</td><td colspan="1" rowspan="1">NPATN/TN+FP(%)</td><td colspan="1" rowspan="1">PPATP/TP+FN(%)</td><td colspan="1" rowspan="1">NPATN/TN+FP(%)</td></tr><tr><td colspan="1" rowspan="1">A. baumannii</td><td colspan="1" rowspan="1">---</td><td colspan="1" rowspan="1">---</td><td colspan="1" rowspan="1">0/10(0.0%)</td><td colspan="1" rowspan="1">3/3(100%)</td><td colspan="1" rowspan="1">0/11(0.0%)</td><td colspan="1" rowspan="1">1/1(100%)</td><td colspan="1" rowspan="1">0/3(0.0%)</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">0/15(0.0%)</td><td colspan="1" rowspan="1">2/2(100%)</td></tr><tr><td colspan="1" rowspan="1">Citrobacter</td><td colspan="1" rowspan="1">---</td><td colspan="1" rowspan="1">---</td><td colspan="1" rowspan="1">0/2(0.0%)</td><td colspan="1" rowspan="1">14/15(93.3%)</td><td colspan="1" rowspan="1">0/3(0.0%)</td><td colspan="1" rowspan="1">21/22(95.5%)</td><td colspan="1" rowspan="1">0/1(0.0%)</td><td colspan="1" rowspan="1">16/17(94.1%)</td><td colspan="1" rowspan="1">0/3(0.0%)</td><td colspan="1" rowspan="1">21/22(95.5%)</td></tr><tr><td colspan="1" rowspan="1">Enterobacter</td><td colspan="1" rowspan="1">---</td><td colspan="1" rowspan="1">---</td><td colspan="1" rowspan="1">0/2(0.0%)</td><td colspan="1" rowspan="1">12/12(100%)</td><td colspan="1" rowspan="1">0/2(0.0%)</td><td colspan="1" rowspan="1">18/18(100%)</td><td colspan="1" rowspan="1">0/1(0.0%)</td><td colspan="1" rowspan="1">9/9(100%)</td><td colspan="1" rowspan="1">0/3(0.0%)</td><td colspan="1" rowspan="1">17/17 (100%)</td></tr><tr><td colspan="1" rowspan="1">E. cloacaecomplex</td><td colspan="1" rowspan="1">---</td><td colspan="1" rowspan="1">---</td><td colspan="1" rowspan="1">0/10(0.0%)</td><td colspan="1" rowspan="1">30/30(100%)</td><td colspan="1" rowspan="1">0/17(0.0%)</td><td colspan="1" rowspan="1">39/39(100%)</td><td colspan="1" rowspan="1">0/5(0.0%)</td><td colspan="1" rowspan="1">25/25(100%)</td><td colspan="1" rowspan="1">0/18(0.0%)</td><td colspan="1" rowspan="1">39/39 (100%)</td></tr><tr><td colspan="1" rowspan="1">E. col</td><td colspan="1" rowspan="1">28/30(93.3%)</td><td colspan="1" rowspan="1">82/82(100%)</td><td colspan="1" rowspan="1">29/38(76.3%)</td><td colspan="1" rowspan="1">159/167(95.2%)</td><td colspan="1" rowspan="1">47/59(79.7%)</td><td colspan="1" rowspan="1">180/181(99.4%)</td><td colspan="1" rowspan="1">23/27(85.2%)</td><td colspan="1" rowspan="1">118/120(98.3%)</td><td colspan="1" rowspan="1">47/53(88.7%)</td><td colspan="1" rowspan="1">196/197(99.5%)</td></tr><tr><td colspan="1" rowspan="1">K. oxytoca</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">10/10(100%)</td><td colspan="1" rowspan="1">0/1(0.0%)</td><td colspan="1" rowspan="1">29/29(100%)</td><td colspan="1" rowspan="1">0/2(0.0%)</td><td colspan="1" rowspan="1">40/40(100%)</td><td colspan="1" rowspan="1">0/2(0.0%)</td><td colspan="1" rowspan="1">20/20(100%)</td><td colspan="1" rowspan="1">0/2(0.0%)</td><td colspan="1" rowspan="1">41/41 (100%)</td></tr><tr><td colspan="1" rowspan="2">K. pneumoniaegroup</td><td colspan="1" rowspan="2">7/11 (63.6%)</td><td colspan="1" rowspan="2">25/25(100%)</td><td colspan="1" rowspan="2">13/23(56.5%)</td><td colspan="1" rowspan="2">83/83(100%)</td><td colspan="1" rowspan="2">17/27(63.0%)</td><td colspan="1" rowspan="1">115/115</td><td colspan="1" rowspan="2">10/12(83.3%)</td><td colspan="1" rowspan="2">72/72(100%)</td><td colspan="1" rowspan="2">17/26(65.4%)</td><td colspan="1" rowspan="2">119/119(100%)</td></tr><tr><td colspan="1" rowspan="1">(100%)</td></tr><tr><td colspan="1" rowspan="1">M. morganii</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">0/1(0.0%)</td><td colspan="1" rowspan="1">6/6(100%)</td><td colspan="1" rowspan="1">0/2(0.0%)</td><td colspan="1" rowspan="1">9/9(100%)</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">4/4(100%)</td><td colspan="1" rowspan="1">0/2(0.0%)</td><td colspan="1" rowspan="1">9/9(100%)</td></tr><tr><td colspan="1" rowspan="1">Proteus</td><td colspan="1" rowspan="1">3/3(100%)</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">1/2(50.0%)</td><td colspan="1" rowspan="1">46/48(95.8%)</td><td colspan="1" rowspan="1">4/7(57.1%)</td><td colspan="1" rowspan="1">59/59(100%)</td><td colspan="1" rowspan="1">2/3(66.7%)</td><td colspan="1" rowspan="1">35/36(97.2%)</td><td colspan="1" rowspan="1">4/8(50.0%)</td><td colspan="1" rowspan="1">59/59 (100%)</td></tr><tr><td colspan="1" rowspan="1">P. mirabilis</td><td colspan="1" rowspan="1">3/3(100%)</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">1/2(50.0%)</td><td colspan="1" rowspan="1">43/45(95.6%)</td><td colspan="1" rowspan="1">4/5(80.0%)</td><td colspan="1" rowspan="1">56/56(100%)</td><td colspan="1" rowspan="1">2/3(66.7%)</td><td colspan="1" rowspan="1">32/33(97.0%)</td><td colspan="1" rowspan="1">4/6(66.7%)</td><td colspan="1" rowspan="1">56/56 (100%)</td></tr><tr><td colspan="1" rowspan="1">P. aeruginosa</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">0/7(0.0%)</td><td colspan="1" rowspan="1">46/46(100%)</td><td colspan="1" rowspan="1">0/20(0.0%)</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">0/9(0.0%)</td><td colspan="1" rowspan="1">25/25(100%)</td><td colspan="1" rowspan="1">0/32(0.0%)</td><td colspan="1" rowspan="1">41/41 (100%)</td></tr><tr><td colspan="1" rowspan="1">Salmonella</td><td colspan="1" rowspan="1">1/1(100%)</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">6/6(100%)</td><td colspan="1" rowspan="1">1/2(50.0%)</td><td colspan="1" rowspan="1">12/12(100%)</td><td colspan="1" rowspan="1">1/1(100%)</td><td colspan="1" rowspan="1">8/8(100%)</td><td colspan="1" rowspan="1">1/2(50.0%)</td><td colspan="1" rowspan="1">15/15 (100%)</td></tr><tr><td colspan="1" rowspan="1">Serratia</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">0/1(0.0%)</td><td colspan="1" rowspan="1">25/25(100%)</td><td colspan="1" rowspan="1">0/4(0.0%)</td><td colspan="1" rowspan="1">32/32(100%)</td><td colspan="1" rowspan="1">0/1(0.0%)</td><td colspan="1" rowspan="1">20/20(100%)</td><td colspan="1" rowspan="1">0/4(0.0%)</td><td colspan="1" rowspan="1">34/34 (100%)</td></tr><tr><td colspan="1" rowspan="1">S. marcescens</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">0/1(0.0%)</td><td colspan="1" rowspan="1">24/24(100%)</td><td colspan="1" rowspan="1">0/4(0.0%)</td><td colspan="1" rowspan="1">31/31(100%)</td><td colspan="1" rowspan="1">0/1(0.0%)</td><td colspan="1" rowspan="1">19/19(100%)</td><td colspan="1" rowspan="1">0/4(0.0%)</td><td colspan="1" rowspan="1">33/33 (100%)</td></tr><tr><td colspan="1" rowspan="1">S. maltophilia</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">0/2(0.0%)</td><td colspan="1" rowspan="1">2/2(100%)</td><td colspan="1" rowspan="1">0/3(0.0%)</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">0/5(0.0%)</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">0/5(0.0%)</td><td colspan="1" rowspan="1">2/2(100%)</td></tr><tr><td colspan="1" rowspan="3">Any OrganismCI</td><td colspan="1" rowspan="3">42/48(87.5%)(75.3-94.1)</td><td colspan="1" rowspan="3">117/117(100%)(96.8-10)</td><td colspan="1" rowspan="3">44/102(43.1%)(33.9-52.8)</td><td colspan="1" rowspan="3">528/541(97.6%)(95.98.6)</td><td colspan="1" rowspan="3">73/168(43.5%)(36.2-51.0)</td><td colspan="1" rowspan="1">613/615</td><td colspan="1" rowspan="1">38/74</td><td colspan="1" rowspan="1">403/408</td><td colspan="1" rowspan="1">73/183</td><td colspan="1" rowspan="3">684/686(99.7%%(988.9-99.9)</td></tr><tr><td colspan="1" rowspan="1">(99.7%%</td><td colspan="1" rowspan="1">(51.4%)</td><td colspan="1" rowspan="1">(98.8%)</td><td colspan="1" rowspan="1">(39.9%%</td></tr><tr><td colspan="1" rowspan="1">(98.8-99.9)</td><td colspan="1" rowspan="1">(40.2-62.4)</td><td colspan="1" rowspan="1">(97.2-99.5)</td><td colspan="1" rowspan="1">(33.1-47.1)</td></tr></table>

$\mathrm { C I } =$ confidence interval

A supplemental comparison of the 5 BCID-GN Panel carbapenemase resistance gene (OXA, KPC, IMP, NDM, VIM) targets versus phenotypic antimicrobial susceptibility testing (AST) for Ertapenem, Imipenem, and Meropenem is provided in Table 51 for clinical isolates with available AST results. In total, 731 isolates had AST results for 1 or more of Ertapenem, Imipenem, or Meropenem. A true positive (TP) result was defined where OXA, KPC, IMP, NDM and/or VIM was detected by the BCID-GN Panel and the isolate was resistant (R) or intermediate (I) to Ertapenem, Imipenem, or Meropenem. A false negative (FN) result was defined similarly when OXA, KPC, IMP, NDM and/or VIM was not detected by the BCID-GN Panel. PPA was calculated as $1 0 0 \mathrm { { x } }$ (TP/(TP+FN)). A true negative (TN) result was defined where OXA, KPC, IMP, NDM and/or VIM was not detected by the BCID-GN Panel and the isolate was susceptible (S) to Ertapenem, Imipenem, or Meropenem. A false positive (FP) result was defined similarly when OXA, KPC, IMP, NDM and/or VIM was detected by the BCID-GN Panel. NPA was calculated as $1 0 0 \mathrm { x ( T N / ( T N / + F P ) } )$ ). Note: Carbapenemase resistance can be due to mechanisms other than acquisition of the OXA, KPC, IMP, NDM and/or VIM resistance genes.

Table 51: Clinical Performance of the BCID-GN Panel Resistance Gene Targets Compared to Phenotypic Antimicrobial Susceptibility Testing for Ertapenem, Imipenem, and Meropenem   

<table><tr><td rowspan=1 colspan=1>Associated</td><td rowspan=1 colspan=2>OXA</td><td rowspan=1 colspan=2>KPC</td><td rowspan=1 colspan=2>IMP</td><td rowspan=1 colspan=2>NDM</td><td rowspan=1 colspan=2>VIM</td><td rowspan=1 colspan=2>Any ResistanceMarker</td></tr><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>PPATP/TP+FN(%)</td><td rowspan=1 colspan=1>NPATN/TN+FP(%)</td><td rowspan=1 colspan=1>PPATP/TP+FN(%)</td><td rowspan=1 colspan=1>NPATN/TN+FP(%)</td><td rowspan=1 colspan=1>PPATP/TP+FN(%)</td><td rowspan=1 colspan=1>NPATN/TN+FP(%)</td><td rowspan=1 colspan=1>PPATP/TP+FN(%)</td><td rowspan=1 colspan=1>NPATN/TN+FP(%)</td><td rowspan=1 colspan=1>PPATP/TP+FN(%)</td><td rowspan=1 colspan=1>NPATN/TN+FP(%)</td><td rowspan=1 colspan=1>PPATP/TP+FN(%)</td><td rowspan=1 colspan=1>NPATN/TN+FP(%)</td></tr><tr><td rowspan=1 colspan=1>A. baumannii</td><td rowspan=1 colspan=1>8/9 (88.9%)</td><td rowspan=1 colspan=1>9/9(100%)</td><td rowspan=1 colspan=1>0/9(0.0%)</td><td rowspan=1 colspan=1>9/9 (100%)</td><td rowspan=1 colspan=1>0/9(0.0%)</td><td rowspan=1 colspan=1>9/9(100%)</td><td rowspan=1 colspan=1>0/9(0.0%)</td><td rowspan=1 colspan=1>9/9(100%)</td><td rowspan=1 colspan=1>0/9(0.0%)</td><td rowspan=1 colspan=1>9/9(100%)</td><td rowspan=1 colspan=1>8/9(88.9%)</td><td rowspan=1 colspan=1>9/9(100%)</td></tr><tr><td rowspan=1 colspan=1>Citrobacter</td><td rowspan=1 colspan=1>0/1(0.0%)</td><td rowspan=1 colspan=1>22/22(100%)</td><td rowspan=1 colspan=1>0/1(0.0%)</td><td rowspan=1 colspan=1>22/22(100%)</td><td rowspan=1 colspan=1>0/1(0.0%)</td><td rowspan=1 colspan=1>22/22(100%)</td><td rowspan=1 colspan=1>0/1(0.0%)</td><td rowspan=1 colspan=1>22/22(100%)</td><td rowspan=1 colspan=1>0/1(0.0%)</td><td rowspan=1 colspan=1>22/22(100%)</td><td rowspan=1 colspan=1>0/1(0.0%)</td><td rowspan=1 colspan=1>22/22 (100%)</td></tr><tr><td rowspan=1 colspan=1>C. sakazakii</td><td rowspan=1 colspan=1>---</td><td rowspan=1 colspan=1>---</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>1/1 (100%)</td><td rowspan=1 colspan=1>---</td><td rowspan=1 colspan=1>---</td><td rowspan=1 colspan=1>---</td><td rowspan=1 colspan=1>---</td><td rowspan=1 colspan=1>---</td><td rowspan=1 colspan=1>---</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>---</td></tr><tr><td rowspan=1 colspan=1>Enterobacter</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>15/15 (100%)</td></tr><tr><td rowspan=1 colspan=1>E. cloacaecomplex</td><td rowspan=1 colspan=1>0/1(0.0%)</td><td rowspan=1 colspan=1>51/51(100%)</td><td rowspan=1 colspan=1>0/1(0.0%)</td><td rowspan=1 colspan=1>51/51(1000%)</td><td rowspan=1 colspan=1>0/1(0.0%)</td><td rowspan=1 colspan=1>51/51(100%)</td><td rowspan=1 colspan=1>0/1(0.0%)</td><td rowspan=1 colspan=1>51/51(10%)</td><td rowspan=1 colspan=1>0/1(0.0%)</td><td rowspan=1 colspan=1>51/51(100%)</td><td rowspan=1 colspan=1>0/1(0.0%)</td><td rowspan=1 colspan=1>51/51 (100%)</td></tr><tr><td rowspan=1 colspan=1>E. coli</td><td rowspan=1 colspan=1>0/2(0.0%)</td><td rowspan=1 colspan=1>247/248(99.6%)</td><td rowspan=1 colspan=1>1/2(50.0%)</td><td rowspan=1 colspan=1>248/248(100%)</td><td rowspan=1 colspan=1>0/2(0.0%)</td><td rowspan=1 colspan=1>248/248(100%)</td><td rowspan=1 colspan=1>0/2(0.0%)</td><td rowspan=1 colspan=1>248/248(100%)</td><td rowspan=1 colspan=1>0/2(0.0%)</td><td rowspan=1 colspan=1>248/248(100%)</td><td rowspan=1 colspan=1>1/2 (50.0%)</td><td rowspan=1 colspan=1>247/248(99.6%)</td></tr><tr><td rowspan=1 colspan=1>K. oxytoca</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>40/40(100%)</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>40/40(100%)</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>40/40(100%)</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>40/40(100%)</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>40/40(100%)</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>40/40 (100%)</td></tr><tr><td rowspan=1 colspan=1>K. pneumoniaegroup</td><td rowspan=1 colspan=1>0/8(0.0%)</td><td rowspan=1 colspan=1>136/136(100%)</td><td rowspan=1 colspan=1>5/8(62.5%)</td><td rowspan=1 colspan=1>135/136(99.3%)</td><td rowspan=1 colspan=1>0/8(0.0%)</td><td rowspan=1 colspan=1>136/136(100%)</td><td rowspan=1 colspan=1>0/8(0.0%)</td><td rowspan=1 colspan=1>136/136(100%)</td><td rowspan=1 colspan=1>0/8(0.0%)</td><td rowspan=1 colspan=1>136/136(100%)</td><td rowspan=1 colspan=1>5/8 (62.5%)</td><td rowspan=1 colspan=1>135/136(99.3%)</td></tr><tr><td rowspan=1 colspan=1>. morganii</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>10/10(100%)</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>10/10(100%)</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>10/10(100%)</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>10/10(100%)</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>10/10(100%)</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>10/10 (100%)</td></tr><tr><td rowspan=1 colspan=1>Proteus</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>61/62(98.4%)</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>62/62(100%)</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>62/62(100%)</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>62/62(100%)</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>62/62(100%)</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>61/62(98.4%)</td></tr><tr><td rowspan=1 colspan=1>P. mirabilis</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>57/57(100%)</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>57/57(100%)</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>57/57(100%)</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>57/57(100%)</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>57/57(100%)</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>57/57 (100%)</td></tr><tr><td rowspan=1 colspan=1>P. aeruginosa</td><td rowspan=1 colspan=1>0/5(0.0%)</td><td rowspan=1 colspan=1>666(98.5%)</td><td rowspan=1 colspan=1>0/5(0.0%)</td><td rowspan=1 colspan=1>66/66(100%)</td><td rowspan=1 colspan=1>0/5(0.0%)</td><td rowspan=1 colspan=1>66/66(100%)</td><td rowspan=1 colspan=1>0/5(0.0%)</td><td rowspan=1 colspan=1>66/66(100%)</td><td rowspan=1 colspan=1>0/5(0.0%)</td><td rowspan=1 colspan=1>66/66(100%)</td><td rowspan=1 colspan=1>0/5(0.0%)</td><td rowspan=1 colspan=1>65/66(98.5%)</td></tr><tr><td rowspan=1 colspan=1>Salmonella</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>8/8 (100%)</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>8/8 (100%)</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>8/8 (100%)</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>8/8 (100%)</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>8/8 (100%)</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>8/8 (100%)</td></tr><tr><td rowspan=1 colspan=1>Serratia</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>38/38(100%)</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>38/38(100%)</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>38/38(100%)</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>38/38(100%)</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>38/38(100%)</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>38/38 (100%)</td></tr><tr><td rowspan=1 colspan=1>S. marcescens</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>37/37(100%)</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>37/37(100%%)</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>37/37(100%)</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>37/37(100%))</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>37/37(100%)</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>37/37 (100%)</td></tr><tr><td rowspan=1 colspan=1>Any OrganismCI</td><td rowspan=1 colspan=1>8/26(30.8%)(16.5-50.0)</td><td rowspan=1 colspan=1>796/799(99.6%)(98.9-99.9)</td><td rowspan=1 colspan=1>6/26(23.1%)(11.0-42.1)</td><td rowspan=1 colspan=1>799/800(99.9%)(99.3-100)</td><td rowspan=1 colspan=1>0/26(0.%(00-12.9)</td><td rowspan=1 colspan=1>799/799(100%)(99.5-100)</td><td rowspan=1 colspan=1>0/26(0.0%)%(0.0-12.9)</td><td rowspan=1 colspan=1>799/799(100%)(99.5-100)</td><td rowspan=1 colspan=1>0/26(0.0%)(0.0-12.9)</td><td rowspan=1 colspan=1>799/799(100%)(99.5-100)</td><td rowspan=1 colspan=1>14/26(53.8%)(35.5-71.2)</td><td rowspan=1 colspan=1>796/800(99.5%(98.7-99.8)</td></tr></table>

$\mathrm { C I } =$ confidence interval

# Co-detections in Clinical Samples

The ePlex BCID-GN Panel identified a total of 103 bacterial co-detections in prospective and retrospective samples. Of the 349 prospective samples, 320 $( 9 1 . 7 \% )$ had single detections, 22 $( 6 . 2 \% )$ had double detections, and 7 $( 2 . 0 \% )$ had triple detections. Of the 577 retrospective samples, 503 $( 8 7 . 2 \% )$ had single detections, 62 $( 1 0 . 7 \% )$ had double detections, and 12 $( 2 . 1 \% )$ had triple detections. Tables 52-55 below summarize co-detections detected in prospective and retrospective samples.

Table 52: Co-Detections Identified by the ePlex BCID-GN Panel (Prospective Samples)   

<table><tr><td colspan="4" rowspan="1">Distinct Co-Detection Combinations Detected by the ePlexBCID-GN Panel in Prospective Clinical Samples</td><td colspan="1" rowspan="2">Number ofSamples(NumberDiscrepant)</td><td colspan="1" rowspan="2">DiscrepantOrganism(s) / ResistanceMarker(s)A,B</td></tr><tr><td colspan="1" rowspan="1">Target 1</td><td colspan="1" rowspan="1">Target 2</td><td colspan="1" rowspan="1">Target 3</td><td colspan="1" rowspan="1">ResistanceMarker</td></tr><tr><td colspan="1" rowspan="1">A. baumannii</td><td colspan="1" rowspan="1">Pan Gram-Positive</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Citrobacter</td><td colspan="1" rowspan="1">E. cloacae complex</td><td colspan="1" rowspan="1">K. oxytoca</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2 (2)</td><td colspan="1" rowspan="1">Citrobacter (2), E. cloacaecomplex (2)</td></tr><tr><td colspan="1" rowspan="1">Citrobacter</td><td colspan="1" rowspan="1">K. oxytoca</td><td colspan="1" rowspan="1">K. pneumoniae group</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1(1)</td><td colspan="1" rowspan="1">Citrobacter (1)</td></tr><tr><td colspan="1" rowspan="1">Citrobacter</td><td colspan="1" rowspan="1">P. mirabilis</td><td colspan="1" rowspan="1">Pan Gram-Positive</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (1)</td><td colspan="1" rowspan="1">Pan Gram-Positive (1)</td></tr><tr><td colspan="1" rowspan="1">E. cloacae complex</td><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">K. pneumoniae group</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">E. cloacae complex</td><td colspan="1" rowspan="1">Pan Candida</td><td colspan="1" rowspan="1">Pan Gram-Positive</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">E. cloacae complex</td><td colspan="1" rowspan="1">Pan Gram-Positive</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">K. oxytoca</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2 (1)</td><td colspan="1" rowspan="1">K. oxytoca (1)</td></tr><tr><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">K. pneumoniae group</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">CTX-M</td><td colspan="1" rowspan="1">1 (1)</td><td colspan="1" rowspan="1">E. coli (1)</td></tr><tr><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">Pan Gram-Positive</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2 (1)</td><td colspan="1" rowspan="1">Pan Gram-Positive (1)</td></tr><tr><td colspan="1" rowspan="1">Enterobacter</td><td colspan="1" rowspan="1">K. pneumoniae group</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1(1)</td><td colspan="1" rowspan="1">Enterobacter (1)</td></tr><tr><td colspan="1" rowspan="1">K. oxytoca</td><td colspan="1" rowspan="1">Pan Gram-Positive</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">K. oxytoca</td><td colspan="1" rowspan="1">S. marcescens</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">K. pneumoniae group</td><td colspan="1" rowspan="1">P. mirabilis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">K. pneumoniae group</td><td colspan="1" rowspan="1">Pan Gram-Positive</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">K. pneumoniae group</td><td colspan="1" rowspan="1">Pan Gram-Positive</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">CTX-M, KPC</td><td colspan="1" rowspan="1">1 (1)</td><td colspan="1" rowspan="1">Pan Gram-Positive (1)</td></tr><tr><td colspan="1" rowspan="1">M. morganii</td><td colspan="1" rowspan="1">P. mirabilis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">P. aeruginosa</td><td colspan="1" rowspan="1">P. mirabilis</td><td colspan="1" rowspan="1">Pan Gram-Positive</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">P. aeruginosa</td><td colspan="1" rowspan="1">Pan Gram-Positive</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">P. mirabilis</td><td colspan="1" rowspan="1">Pan Gram-Positive</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">3 (2)</td><td colspan="1" rowspan="1">Pan Gram-Positive (2)</td></tr><tr><td colspan="1" rowspan="1">P. mirabilis</td><td colspan="1" rowspan="1">Pan Gram-Positive</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">CTX-M</td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr></table>

A. A discrepant organism or resistance marker is defined as one that was detected by the BCID-GN Panel but not by the comparator method(s). B. 12/13 false positive organisms were investigated using PCR/sequencing; the discrepant organism was detected in 11/13 and not detected in 1. One false positive Pan Gram-Positive sample was not tested. i. In 3/3 false positive Citrobacter samples, Citrobacter was detected. ii. In 2/2 false positive $E$ . cloacae complex samples, $E .$ . cloacae complex was detected. iii. An Enterobacter species was not detected in the 1 false positive Enterobacter (non-cloacae complex) sample. iv. In 1/1 false positive $E$ . coli sample, $E .$ . coli was detected. v. In 1/1 false positive K. oxytoca sample, K. oxytoca was detected. vi. $\mathrm { I n } 4 / 4$ false positive Pan Gram-Positive samples, a Pan Gram-Positive organism was detected.

# Table 53: Co-Detections Identified by the ePlex BCID-GN Panel (Retrospective Samples)

<table><tr><td colspan="4" rowspan="1">Distinct Co-Detection Combinations Detected by the ePlexBCID-GN Panel in Retrospective Clinical Samples</td><td colspan="2" rowspan="2">Number             DiscrepantSamples(Number           Organism(s) /Resistance Marker(s)A,BDiscrepant)</td></tr><tr><td colspan="1" rowspan="1">Target 1</td><td colspan="1" rowspan="1">Target 2</td><td colspan="1" rowspan="1">Target 3</td><td colspan="1" rowspan="1">ResistanceMarker</td></tr><tr><td colspan="1" rowspan="1">A. baumannii</td><td colspan="1" rowspan="1">K. pneumoniae group</td><td colspan="1" rowspan="1">Pan Gram-Positive</td><td colspan="1" rowspan="1">CTX-M, OXA</td><td colspan="1" rowspan="1">1 (1)</td><td colspan="1" rowspan="1">A. baumannii (1), K.pneumoniae group (1), PanGram-Positive (1)</td></tr><tr><td colspan="1" rowspan="1">A. baumannii</td><td colspan="1" rowspan="1">Pan Gram-Positive</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">A. baumannii</td><td colspan="1" rowspan="1">Pan Gram-Positive</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">OXA</td><td colspan="1" rowspan="1">4 (1)</td><td colspan="1" rowspan="1">Pan Gram-Positive (1)</td></tr><tr><td colspan="1" rowspan="1">B. fragilis</td><td colspan="1" rowspan="1">E. cloacae complex</td><td colspan="1" rowspan="1">Pan Gram-Positive</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (1)</td><td colspan="1" rowspan="1">B. fragilis (1)</td></tr><tr><td colspan="1" rowspan="1">B. fragilis</td><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2 (1)</td><td colspan="1" rowspan="1">B. fragilis (1)</td></tr><tr><td colspan="1" rowspan="1">B. fragilis</td><td colspan="1" rowspan="1">Pan Gram-Positive</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Citrobacter</td><td colspan="1" rowspan="1">E. cloacae complex</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (1)</td><td colspan="1" rowspan="1">E. cloacae complex (1)</td></tr><tr><td colspan="1" rowspan="1">Citrobacter</td><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Citrobacter</td><td colspan="1" rowspan="1">K. oxytoca</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (1)</td><td colspan="1" rowspan="1">Citrobacter (1)</td></tr><tr><td colspan="1" rowspan="1">Citrobacter</td><td colspan="1" rowspan="1">K. oxytoca</td><td colspan="1" rowspan="1">K. pneumoniae group</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1(1)</td><td colspan="1" rowspan="1">K. oxytoca (1)</td></tr><tr><td colspan="1" rowspan="1">Citrobacter</td><td colspan="1" rowspan="1">K. pneumoniae group</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="4" rowspan="1">Distinct Co-Detection Combinations Detected by the ePlexBCID-GN Panel in Retrospective Clinical Samples</td><td colspan="2" rowspan="2">Number            DiscrepantSamples(Number          Organism(s) /Resistance Marker(s)A,BDiscrepant)</td></tr><tr><td colspan="1" rowspan="1">Target 1</td><td colspan="1" rowspan="1">Target 2</td><td colspan="1" rowspan="1">Target 3</td><td colspan="1" rowspan="1">ResistanceMarker</td></tr><tr><td colspan="1" rowspan="1">Citrobacter</td><td colspan="1" rowspan="1">K. pneumoniae group</td><td colspan="1" rowspan="1">Pan Gram-Positive</td><td colspan="1" rowspan="1">CTX-M</td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Citrobacter</td><td colspan="1" rowspan="1">M. morganii</td><td colspan="1" rowspan="1">Pan Gram-Positive</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (1)</td><td colspan="1" rowspan="1">M. morganii (1)</td></tr><tr><td colspan="1" rowspan="1">Citrobacter</td><td colspan="1" rowspan="1">Pan Gram-Positive</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">3 (2)</td><td colspan="1" rowspan="1">Pan Gram-Positive (2)</td></tr><tr><td colspan="1" rowspan="1">E. cloacaecomplex</td><td colspan="1" rowspan="1">K. pneumoniae group</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">E. cloacaecomplex</td><td colspan="1" rowspan="1">P. aeruginosa</td><td colspan="1" rowspan="1">Pan Gram-Positive</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (1)</td><td colspan="1" rowspan="1">P. aeruginosa (1)</td></tr><tr><td colspan="1" rowspan="1">E. cloacaecomplex</td><td colspan="1" rowspan="1">Pan Candida</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (1)</td><td colspan="1" rowspan="1">Pan Candida (1)</td></tr><tr><td colspan="1" rowspan="1">E. cloacaecomplex</td><td colspan="1" rowspan="1">Pan Gram-Positive</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2 (1)</td><td colspan="1" rowspan="1">Pan Gram-Positive (1)</td></tr><tr><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">K. oxytoca</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">K. oxytoca</td><td colspan="1" rowspan="1">Pan Gram-Positive</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">K. pneumoniae group</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">M. morganii</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">P. mirabilis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">3 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">P. mirabilis</td><td colspan="1" rowspan="1">Pan Gram-Positive</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">Pan Gram-Positive</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">8 (2)</td><td colspan="1" rowspan="1">Pan Gram-Positive (2)</td></tr><tr><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">Pan Gram-Positive</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">CTX-M</td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Enterobacter</td><td colspan="1" rowspan="1">Pan Candida</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Enterobacter</td><td colspan="1" rowspan="1">Pan Gram-Positive</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">H. influenzae</td><td colspan="1" rowspan="1">N. meningitidis</td><td colspan="1" rowspan="1">P. aeruginosa</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (1)</td><td colspan="1" rowspan="1">N. meningitidis (1), P.aeruginosa (1)</td></tr><tr><td colspan="1" rowspan="1">K. oxytoca</td><td colspan="1" rowspan="1">K. pneumoniae group</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2 (1)</td><td colspan="1" rowspan="1">K. pneumoniae group (1)</td></tr><tr><td colspan="1" rowspan="1">K. oxytoca</td><td colspan="1" rowspan="1">Pan Gram-Positive</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">3 (2)</td><td colspan="1" rowspan="1">Pan Gram-Positive (2)</td></tr><tr><td colspan="1" rowspan="1">K. oxytoca</td><td colspan="1" rowspan="1">S. marcescens</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (1)</td><td colspan="1" rowspan="1">S. marcescens (1)</td></tr><tr><td colspan="1" rowspan="1">K. pneumoniaegroup</td><td colspan="1" rowspan="1">Pan Gram-Positive</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">4(1)</td><td colspan="1" rowspan="1">Pan Gram-Positive (1)</td></tr><tr><td colspan="1" rowspan="1">K. pneumoniaegroup</td><td colspan="1" rowspan="1">Pan Gram-Positive</td><td colspan="1" rowspan="1">S. marcescens</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (1)</td><td colspan="1" rowspan="1">K. pneumoniae group (1)</td></tr><tr><td colspan="4" rowspan="1">Distinct Co-Detection Combinations Detected by the ePlexBCID-GN Panel in Retrospective Clinical Samples</td><td colspan="2" rowspan="2">Number             DiscrepantSamplesOrganism(s) /(NumberDiscrepant)    Resistance Marker(s)A,B</td></tr><tr><td colspan="1" rowspan="1">Target 1</td><td colspan="1" rowspan="1">Target 2</td><td colspan="1" rowspan="1">Target 3</td><td colspan="1" rowspan="1">ResistanceMarker</td></tr><tr><td colspan="1" rowspan="1">K. pneumoniaegroup</td><td colspan="1" rowspan="1">S. maltophilia</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">M. morganii</td><td colspan="1" rowspan="1">P. aeruginosa</td><td colspan="1" rowspan="1">Pan Gram-Positive</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (1)</td><td colspan="1" rowspan="1">P. aeruginosa (1)</td></tr><tr><td colspan="1" rowspan="1">M. morganii</td><td colspan="1" rowspan="1">P. mirabilis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">M. morganii</td><td colspan="1" rowspan="1">Pan Gram-Positive</td><td colspan="1" rowspan="1">Proteus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">P. aeruginosa</td><td colspan="1" rowspan="1">Pan Gram-Positive</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">P. mirabilis</td><td colspan="1" rowspan="1">Pan Gram-Positive</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">5 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Pan Candida</td><td colspan="1" rowspan="1">Pan Gram-Positive</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Pan Gram-Positive</td><td colspan="1" rowspan="1">S. maltophilia</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Pan Gram-Positive</td><td colspan="1" rowspan="1">S. marcescens</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">3 (0)</td><td colspan="1" rowspan="1"></td></tr></table>

A. A discrepant organism or resistance marker is defined as one that was detected by the BCID-GN Panel but not by the comparator method(s). B. 24/26 false positive organisms were investigated using PCR/sequencing; the discrepant organism was detected in 21/24, not detected in 2, and was indeterminate for one organism.

i. In 1/1 false positive A.baumannii sample, A. baumannii was detected. ii. In 2/2 false positive B. fragilis samples, $B .$ . fragilis was detected. iii. In 1/1 false positive Citrobacter sample, Citrobacter was detected. iv. In the one false positive $E$ . cloacae complex sample, PCR/sequencing was indeterminate. v. In 1/1 false positive K. oxytoca sample, K. oxytoca was detected. vi. $\operatorname { I n } 3 / 3$ false positive K. pneumoniae group samples, K. pneumoniae group was detected. vii. In 1/1 false positive sample, M. morganii was detected. viii. N. meningitidis was not detected in the 1 N. meningitidis false positive sample. ix. In 2/3 false positive P. aeruginosa samples, P. aeruginosa was detected. P. aeruginosa was not detected in the remaining sample. x. In 1/1 false positive pan Candida sample, Pan Candida was detected. xi. In 8/8 false positive Pan Gram-Positive samples, a Pan Gram-Positive organism was detected. xii. In 1/1 false positive S. marcescens sample, S. marcescens was detected.

Table 54: Additional Co-Detections Identified by the Comparator Method(s) (Prospective Samples)   

<table><tr><td colspan="5" rowspan="1">Distinct Co-Detection Combinations Detected by the ComparatorMethod(s) in Prospective Clinical Samples</td><td colspan="1" rowspan="2">NumberSamples(NumberDiscrepant)</td><td colspan="1" rowspan="2">DiscrepantOrganism(s) /ResistanceMarker(s)A</td></tr><tr><td colspan="1" rowspan="1">Target 1</td><td colspan="1" rowspan="1">Target 2</td><td colspan="1" rowspan="1">Target 3</td><td colspan="1" rowspan="1">Target 4</td><td colspan="1" rowspan="1">ResistanceMarker</td></tr><tr><td colspan="1" rowspan="1">A. baumannii</td><td colspan="1" rowspan="1">E. faecium</td><td colspan="1" rowspan="1">Staphylococcus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">A. baumannii</td><td colspan="1" rowspan="1">Staphylococcus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Achromobacterxylosoxidans*</td><td colspan="1" rowspan="1">E. cloacae</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="5" rowspan="1">Distinct Co-Detection Combinations Detected by the ComparatorMethod(s) in Prospective Clinical Samples</td><td colspan="1" rowspan="2">NumberSamples(NumberDisscrepant)</td><td colspan="1" rowspan="2">DiscrepantOrganism(s) /esistanceMarker(s)A</td></tr><tr><td colspan="1" rowspan="1">Target 1</td><td colspan="1" rowspan="1">Target 2</td><td colspan="1" rowspan="1">Target 3</td><td colspan="1" rowspan="1">Target 4</td><td colspan="1" rowspan="1">ResistanceMarker</td></tr><tr><td colspan="1" rowspan="1">Acinetobacterlwofffi*</td><td colspan="1" rowspan="1">Staphylococcushominis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Acinetobacterpitti*</td><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Aerococcusviridans*</td><td colspan="1" rowspan="1">K. oxytoca</td><td colspan="1" rowspan="1">S. epidermidis</td><td colspan="1" rowspan="1">Staphylococcuscohni</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Aerococcusviridans*</td><td colspan="1" rowspan="1">Staphylococcushominis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">B. fragilis</td><td colspan="1" rowspan="1">Clostridiumspecies*</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Bacillus</td><td colspan="1" rowspan="1">E. cloacae</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">C. acnes*</td><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">C. albicans</td><td colspan="1" rowspan="1">E. cloacae</td><td colspan="1" rowspan="1">E. faecalis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">C. freundii</td><td colspan="1" rowspan="1">P. mirabilis</td><td colspan="1" rowspan="1">Providenciastuartii*</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Candida lusitaniae*</td><td colspan="1" rowspan="1">S. liquefaciens</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Citrobacteramalonaticus*</td><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">E. aerogenes</td><td colspan="1" rowspan="1">K. oxytoca</td><td colspan="1" rowspan="1">Leclerciaadecarboxylata*</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2 (2)</td><td colspan="1" rowspan="1">E. aerogenes (2)</td></tr><tr><td colspan="1" rowspan="1">E. aerogenes</td><td colspan="1" rowspan="1">P. aeruginosa</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (1)</td><td colspan="1" rowspan="1">P. aeruginosa (1)</td></tr><tr><td colspan="1" rowspan="1">E. cloacae</td><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">K. pneumoniae</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">E. cloacae</td><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">E. faecalis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">K. pneumoniae</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (1)</td><td colspan="1" rowspan="1">E. coli (1)</td></tr><tr><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">P. mirabilis</td><td colspan="1" rowspan="1">Providenciastuartii*</td><td colspan="1" rowspan="1">S. anginosusgroup</td><td colspan="1" rowspan="1">CTX-M</td><td colspan="1" rowspan="1">1 (1)</td><td colspan="1" rowspan="1">E coli (1)</td></tr><tr><td colspan="1" rowspan="1">E. faecalis</td><td colspan="1" rowspan="1">K. pneumoniae</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">3 (1)</td><td colspan="1" rowspan="1">E. faecalis (1)</td></tr><tr><td colspan="1" rowspan="1">E. faecalis</td><td colspan="1" rowspan="1">M. morganii</td><td colspan="1" rowspan="1">P. mirabilis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (1)</td><td colspan="1" rowspan="1">E. faecalis (1)</td></tr><tr><td colspan="1" rowspan="1">E. faecalis</td><td colspan="1" rowspan="1">Providenciastuartii*</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Enterobacteriaceae*</td><td colspan="1" rowspan="1">K. pneumoniae</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="5" rowspan="1">Distinct Co-Detection Combinations Detected by the ComparatorMethod(s) in Prospective Clinical Samples</td><td colspan="1" rowspan="2">NumberSamples(NumberDiscrepant)</td><td colspan="1" rowspan="2">DiscrepantOrganism(s) /ResistanceMarker(s)4</td></tr><tr><td colspan="1" rowspan="1">Target 1</td><td colspan="1" rowspan="1">Target 2</td><td colspan="1" rowspan="1">Target 3</td><td colspan="1" rowspan="1">Target 4</td><td colspan="1" rowspan="1">ResistanceMarker</td></tr><tr><td colspan="1" rowspan="1">K. pneumoniae</td><td colspan="1" rowspan="1">Staphylococcushaemolyticus</td><td colspan="1" rowspan="1">non-fermentingGram-Negativebacilli*</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (1)</td><td colspan="1" rowspan="1">K. pneumoniae (1)</td></tr><tr><td colspan="1" rowspan="1">Lactococcus lactis*</td><td colspan="1" rowspan="1">P. mirabilis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Micrococcusluteus*</td><td colspan="1" rowspan="1">Sphingomonaspaucimobilis*</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">P. aeruginosa</td><td colspan="1" rowspan="1">P. mirabilis</td><td colspan="1" rowspan="1">Streptococcus -viridans group</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">P. aeruginosa</td><td colspan="1" rowspan="1">S. maltophilia</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (1)</td><td colspan="1" rowspan="1">S. maltophilia (1)</td></tr><tr><td colspan="1" rowspan="1">P. aeruginosa</td><td colspan="1" rowspan="1">Staphylococcushaemolyticus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">P. mirabilis</td><td colspan="1" rowspan="1">Staphylococcus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">S. maltophilia</td><td colspan="1" rowspan="1">Streptococcus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (1)</td><td colspan="1" rowspan="1">Streptococcus (1)</td></tr></table>

\* Indicates an off‐panel organism not detected by the BCID‐GN Panel.

A. A discrepant organism or resistance marker is defined as one that was detected by the comparator method(s) but not by the BCID‐GN Panel (excludes organisms not detected by the BCID‐GN panel).

Table 55: Additional Co-Detections Identified by the Comparator Method(s) (Retrospective Samples)   

<table><tr><td colspan="5" rowspan="1">Distinct Co-Detection Combinations Detected by the Comparator Method(s) inRetrospective Clinical Samples</td><td colspan="1" rowspan="2">NumberSamples(NumberDiscrepant)</td><td colspan="1" rowspan="2">DiscrepantOrganism(s) /ResistanceMarker(s)4</td></tr><tr><td colspan="1" rowspan="1">Target 1</td><td colspan="1" rowspan="1">Target 2</td><td colspan="1" rowspan="1">Target 3</td><td colspan="1" rowspan="1">Target 4</td><td colspan="1" rowspan="1">ResistanceMarker</td></tr><tr><td colspan="1" rowspan="1">A. baumannii</td><td colspan="1" rowspan="1">E. faecalis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">OXA</td><td colspan="1" rowspan="1">2 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">A. baumannii</td><td colspan="1" rowspan="1">E. faecalis</td><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">A. baumannii</td><td colspan="1" rowspan="1">E. faecium</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">OXA</td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">A. baumannii</td><td colspan="1" rowspan="1">Staphylococcus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Acinetobacterradioresistens*</td><td colspan="1" rowspan="1">P. vulgaris</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">OXA</td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Aeromonas caviae*</td><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">Enterococcuscasseliflavus</td><td colspan="1" rowspan="1">K. oxytoca</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (1)</td><td colspan="1" rowspan="1">E. casseliflavus (1)</td></tr><tr><td colspan="1" rowspan="1">Aeromonas veronii*</td><td colspan="1" rowspan="1">E. cloacae</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (1)</td><td colspan="1" rowspan="1">E. cloacae (1)</td></tr><tr><td colspan="1" rowspan="1">B. fragilis</td><td colspan="1" rowspan="1">S. anginosus group</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="5" rowspan="1">Distinct Co-Detection Combinations Detected by the Comparator Method(s) inRetrospective Clinical Samples</td><td colspan="1" rowspan="2">NumberSamples(NumberDisccrepant)</td><td colspan="1" rowspan="2">DiscrepantOrganism(s) / ResistanceMarker(s)A</td></tr><tr><td colspan="1" rowspan="1">Target 1</td><td colspan="1" rowspan="1">Target 2</td><td colspan="1" rowspan="1">Target 3</td><td colspan="1" rowspan="1">Target 4</td><td colspan="1" rowspan="1">ResistanceMarker</td></tr><tr><td colspan="1" rowspan="1">C. albicans</td><td colspan="1" rowspan="1">E. faecalis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">C. albicans</td><td colspan="1" rowspan="1">E. faecium</td><td colspan="1" rowspan="1">Staphylococcushominis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (1)</td><td colspan="1" rowspan="1">C. albicans (1)</td></tr><tr><td colspan="1" rowspan="1">C. albicans</td><td colspan="1" rowspan="1">P. aeruginosa</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (1)</td><td colspan="1" rowspan="1">C. albicans (1)</td></tr><tr><td colspan="1" rowspan="1">C. albicans</td><td colspan="1" rowspan="1">S. epidermidis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">C. braakii</td><td colspan="1" rowspan="1">E. cloacae</td><td colspan="1" rowspan="1">K. oxytoca</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (1)</td><td colspan="1" rowspan="1">C. braakii (1), K.oxytoca (1)</td></tr><tr><td colspan="1" rowspan="1">C. braakii</td><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">C. braakii</td><td colspan="1" rowspan="1">Streptococcus oralis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">C. freundii</td><td colspan="1" rowspan="1">Enterococcus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (1)</td><td colspan="1" rowspan="1">Enterococcus (1)</td></tr><tr><td colspan="1" rowspan="1">C. freundii</td><td colspan="1" rowspan="1">K. pneumoniae</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">C. freundii</td><td colspan="1" rowspan="1">K. pneumoniae</td><td colspan="1" rowspan="1">Staphylococcushominis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">CTX-M</td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">C. glabrata</td><td colspan="1" rowspan="1">E. aerogenes</td><td colspan="1" rowspan="1">Staphylococcus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (1)</td><td colspan="1" rowspan="1">Staphylococcus (1)</td></tr><tr><td colspan="1" rowspan="1">C. glabrata</td><td colspan="1" rowspan="1">P. mirabilis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (1)</td><td colspan="1" rowspan="1">C. glabrata (1)</td></tr><tr><td colspan="1" rowspan="1">C. koseri</td><td colspan="1" rowspan="1">E. faecalis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">C. krusei</td><td colspan="1" rowspan="1">S. epidermidis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (1)</td><td colspan="1" rowspan="1">S. epidermidis (1)</td></tr><tr><td colspan="1" rowspan="1">C. youngae</td><td colspan="1" rowspan="1">K. oxytoca</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (1)</td><td colspan="1" rowspan="1">K. oxytoca (1)</td></tr><tr><td colspan="1" rowspan="1">Clostridiumperfringens*</td><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">E. aerogenes</td><td colspan="1" rowspan="1">S. anginosus group</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">E. cloacae</td><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (1)</td><td colspan="1" rowspan="1">E. coli (1)</td></tr><tr><td colspan="1" rowspan="1">E. cloacae</td><td colspan="1" rowspan="1">E. faecalis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (1)</td><td colspan="1" rowspan="1">E. faecalis (1)</td></tr><tr><td colspan="1" rowspan="1">E. cloacae</td><td colspan="1" rowspan="1">E. faecium</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">E. cloacae</td><td colspan="1" rowspan="1">E. faecium</td><td colspan="1" rowspan="1">Staphylococcushominis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">E. cloacae</td><td colspan="1" rowspan="1">K. pneumoniae</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">E. cloacae</td><td colspan="1" rowspan="1">M. morganii</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (1)</td><td colspan="1" rowspan="1">E. cloacae (1)</td></tr><tr><td colspan="1" rowspan="1">E. cloacae</td><td colspan="1" rowspan="1">S. anginosus group</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">E. cloacae</td><td colspan="1" rowspan="1">S. maltophilia</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (1)</td><td colspan="1" rowspan="1">S. maltophilia (1)</td></tr><tr><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">E. faecalis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="5" rowspan="1">Distinct Co-Detection Combinations Detected by the Comparator Method(s) inRetrospective Clinical Samples</td><td colspan="1" rowspan="2">NumberSamples(NumberDiscrepant)</td><td colspan="1" rowspan="2">DiscrepantOrganism(s) /esistanceMMarker(s)A</td></tr><tr><td colspan="1" rowspan="1">Target 1</td><td colspan="1" rowspan="1">Target 2</td><td colspan="1" rowspan="1">Target 3</td><td colspan="1" rowspan="1">Target 4</td><td colspan="1" rowspan="1">ResistanceMarker</td></tr><tr><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">E. faecalis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">CTX-M</td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">E. faecalis</td><td colspan="1" rowspan="1">K. pneumoniae</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (1)</td><td colspan="1" rowspan="1">E. coli (1)</td></tr><tr><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">E. faecalis</td><td colspan="1" rowspan="1">P. mirabilis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">E. faecium</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">E. faecium</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">CTX-M</td><td colspan="1" rowspan="1">1(1)</td><td colspan="1" rowspan="1">E. faecium (1)</td></tr><tr><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">K. oxytoca</td><td colspan="1" rowspan="1">Streptococcusinfantarius</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">P. aeruginosa</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (1)</td><td colspan="1" rowspan="1">P. aeruginosa (1)</td></tr><tr><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">P. mirabilis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (1)</td><td colspan="1" rowspan="1">E. coli (1)</td></tr><tr><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">P. mirabilis</td><td colspan="1" rowspan="1">P. vulgaris</td><td colspan="1" rowspan="1">Streptococcus -viridans group</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (1)</td><td colspan="1" rowspan="1">S. viridans group(1)</td></tr><tr><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">Propionibacteria*</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">S. anginosus gp</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (1)</td><td colspan="1" rowspan="1">S. anginosus group(1)</td></tr><tr><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">S. pneumoniae</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">Staphylococcus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">E. faecalis</td><td colspan="1" rowspan="1">K. pneumoniae</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (1)</td><td colspan="1" rowspan="1">K. pneumoniae (1)</td></tr><tr><td colspan="1" rowspan="1">E. faecalis</td><td colspan="1" rowspan="1">M. morganii</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">E. faecalis</td><td colspan="1" rowspan="1">M. morganii</td><td colspan="1" rowspan="1">P. vulgaris</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">E. faecalis</td><td colspan="1" rowspan="1">P. aeruginosa</td><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (1)</td><td colspan="1" rowspan="1">P. aeruginosa (1)</td></tr><tr><td colspan="1" rowspan="1">E. faecalis</td><td colspan="1" rowspan="1">P. mirabilis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">3 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">E. faecalis</td><td colspan="1" rowspan="1">S. maltophilia</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1(1)</td><td colspan="1" rowspan="1">E. faecalis (1)</td></tr><tr><td colspan="1" rowspan="1">E. faecalis</td><td colspan="1" rowspan="1">S. marcescens</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">E. faecium</td><td colspan="1" rowspan="1">K. pneumoniae</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">E. faecium</td><td colspan="1" rowspan="1">P. aeruginosa</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">E. faecium</td><td colspan="1" rowspan="1">P. mirabilis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">K. oxytoca</td><td colspan="1" rowspan="1">S. anginosus group</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">K. pneumoniae</td><td colspan="1" rowspan="1">P. aeruginosa</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (1)</td><td colspan="1" rowspan="1">P. aeruginosa (1)</td></tr><tr><td colspan="1" rowspan="1">K. pneumoniae</td><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2 (1)</td><td colspan="1" rowspan="1">S. aureus (1)</td></tr><tr><td colspan="5" rowspan="1">Distinct Co-Detection Combinations Detected by the Comparator Method(s) inRetrospective Clinical Samples</td><td colspan="1" rowspan="2">NumberSamples(NumberDiscrepant)</td><td colspan="1" rowspan="2">DiscrepantOrganism(s) /ResistanceMarker(s)A</td></tr><tr><td colspan="1" rowspan="1">Target 1</td><td colspan="1" rowspan="1">Target 2</td><td colspan="1" rowspan="1">Target 3</td><td colspan="1" rowspan="1">Target 4</td><td colspan="1" rowspan="1">ResistanceMarker</td></tr><tr><td colspan="1" rowspan="1">K. pneumoniae</td><td colspan="1" rowspan="1">Staphylococcus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (1)</td><td colspan="1" rowspan="1">Staphylococcus (1)</td></tr><tr><td colspan="1" rowspan="1">P. aeruginosa</td><td colspan="1" rowspan="1">S. maltophilia</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1(1)</td><td colspan="1" rowspan="1">S. maltophilia (1)</td></tr><tr><td colspan="1" rowspan="1">P. mirabilis</td><td colspan="1" rowspan="1">Peptostreptococcusanaerobius*</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">P. mirabilis</td><td colspan="1" rowspan="1">Providenciastuartii*</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2 (1)</td><td colspan="1" rowspan="1">P. mirabilis (1)</td></tr><tr><td colspan="1" rowspan="1">P. mirabilis</td><td colspan="1" rowspan="1">Staphylococcus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Pseudomonasputida*</td><td colspan="1" rowspan="1">S. epidermidis</td><td colspan="1" rowspan="1">S. maltophilia</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1">S. marcescens</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">S. marcescens</td><td colspan="1" rowspan="1">Staphylococcus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">S. marcescens</td><td colspan="1" rowspan="1">Streptococcus mitisgroup</td><td colspan="1" rowspan="1">Streptococcussalivarius</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1 (0)</td><td colspan="1" rowspan="1"></td></tr></table>

\* Indicates an off-panel organism not detected by the BCID-GN Panel.

A. A discrepant organism or resistance marker is defined as one that was detected by the comparator method(s) but not by the BCID-GN Panel (excludes organisms not detected by the BCID-GN panel).

# Clinical Study ePlex Instrument Performance

A total of 2460 samples (including prospective, retrospective, and contrived samples) were initially tested in the clinical evaluations. Of these, 23/2460 $( 0 . 9 \% )$ did not complete the run and the sample was retested. After repeat testing, all 2460 samples completed testing and 2334/2460 $( 9 4 . 9 \%$ , $9 5 \%$ CI: $9 3 . 9 \% . 9 5 . 7 \%$ ) generated valid results and 126/2460 ( $5 . 1 \%$ , $9 5 \%$ CI: $4 . 3 \%$ - $6 . 1 \%$ ) generated invalid results on the first completed attempt.

Upon repeat testing of the 126 samples with initially invalid results, 1/126 $( 0 . 8 \% )$ did not complete the run and the sample was retested. After repeat testing, all 126 samples completed testing and 114/126 $( 9 0 . 5 \% )$ generated valid results. Overall, after final testing, 12/2460 $( 0 . 5 \%$ , $9 5 \%$ CI: $0 . 3 \% { - } 0 . 9 \%$ ) had final, invalid results, resulting in a final validity rate of 2448/2460 $( 9 9 . 5 \%$ , $9 5 \%$ CI: $9 9 . 1 \% 9 9 . 7 \%$ ).

# ANALYTICAL PERFORMANCE CHARACTERISTICS

# Limit of Detection (LoD)

The limit of detection (LoD), or analytical sensitivity, was identified and verified for each target on the BCID-GN Panel using quantified reference strains in simulated blood culture sample matrix, which is defined as a whole blood with EDTA added to a blood culture bottle in the same ratio as the manufacturer recommends and incubated for 8 hours. At least 20 replicates per target were tested for each condition. The limit of detection was defined as the lowest concentration of each target that is detected in $2 9 5 \%$ of tested replicates. The confirmed LoD for each ePlex BCID-GN Panel organism is shown in Table 56.

Table 56: LoD Results Summary   

<table><tr><td colspan="1" rowspan="1">Target</td><td colspan="1" rowspan="1">Organism</td><td colspan="1" rowspan="1">Strain</td><td colspan="1" rowspan="1">LoDConcentration(CFU/mL)</td></tr><tr><td colspan="1" rowspan="2">Acinetobacter baumannii</td><td colspan="1" rowspan="1">Acinetobacter baumannii</td><td colspan="1" rowspan="1">NCTC 13421</td><td colspan="1" rowspan="1">1 x 106</td></tr><tr><td colspan="1" rowspan="1">Acinetobacter baumannii</td><td colspan="1" rowspan="1">NCTC 13304</td><td colspan="1" rowspan="1">1 x 106</td></tr><tr><td colspan="1" rowspan="2">Bacteroides fragilis</td><td colspan="1" rowspan="1">Bacteroides fragilis</td><td colspan="1" rowspan="1">ATCC 25285</td><td colspan="1" rowspan="1">1 x 105</td></tr><tr><td colspan="1" rowspan="1">Bacteroides fragilis</td><td colspan="1" rowspan="1">ATCC 43860</td><td colspan="1" rowspan="1">1 x 104</td></tr><tr><td colspan="1" rowspan="2">Citrobacter</td><td colspan="1" rowspan="1">Citrobacter freundii</td><td colspan="1" rowspan="1">NCTC 9750</td><td colspan="1" rowspan="1">1 x 106</td></tr><tr><td colspan="1" rowspan="1">Citrobacter koseri</td><td colspan="1" rowspan="1">ATCC 27156</td><td colspan="1" rowspan="1">1 x 106</td></tr><tr><td colspan="1" rowspan="2">Cronobacter sakazakii</td><td colspan="1" rowspan="1">Cronobacter sakazakii</td><td colspan="1" rowspan="1">ATCC 29544</td><td colspan="1" rowspan="1">1 x 105</td></tr><tr><td colspan="1" rowspan="1">Cronobacter sakazakii</td><td colspan="1" rowspan="1">ATCC 29004</td><td colspan="1" rowspan="1">1 x 106</td></tr><tr><td colspan="1" rowspan="3">Enterobacter (non-cloacae complex)</td><td colspan="1" rowspan="1">Enterobacter aerogenes</td><td colspan="1" rowspan="1">CDC#0074</td><td colspan="1" rowspan="1">1 x 106</td></tr><tr><td colspan="1" rowspan="1">Enterobacter aerogenes</td><td colspan="1" rowspan="1">CDC#0161</td><td colspan="1" rowspan="1">1 x 105</td></tr><tr><td colspan="1" rowspan="1">Enterobacter amnigenus</td><td colspan="1" rowspan="1">ATCC 33072</td><td colspan="1" rowspan="1">1 x 106</td></tr><tr><td colspan="1" rowspan="3">Enterobacter cloacaeComplex</td><td colspan="1" rowspan="1">Enterobacter cloacae</td><td colspan="1" rowspan="1">CDC#0154</td><td colspan="1" rowspan="1">1 x 106</td></tr><tr><td colspan="1" rowspan="1">Enterobacter asburiae</td><td colspan="1" rowspan="1">ATCC 35957</td><td colspan="1" rowspan="1">1 x 106</td></tr><tr><td colspan="1" rowspan="1">Enterobacter hormaechei</td><td colspan="1" rowspan="1">ATCC BAA-2082</td><td colspan="1" rowspan="1">1 x 106</td></tr><tr><td colspan="1" rowspan="3">Escherichia coli</td><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">CDC#0118</td><td colspan="1" rowspan="1">1 x 107</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">NCTC 13441</td><td colspan="1" rowspan="1">1 x 106</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">JHU01-D80401147</td><td colspan="1" rowspan="1">1 x 107</td></tr><tr><td colspan="1" rowspan="2">Fusobacterium necrophorum</td><td colspan="1" rowspan="1">Fusobacterium necrophorum</td><td colspan="1" rowspan="1">ATCC 51357</td><td colspan="1" rowspan="1">1 x 108</td></tr><tr><td colspan="1" rowspan="1">Fusobacterium necrophorum</td><td colspan="1" rowspan="1">ATCC 27852</td><td colspan="1" rowspan="1">1 x 107</td></tr><tr><td colspan="1" rowspan="2">Fusobacterium nucleatum</td><td colspan="1" rowspan="1">Fusobacterium nucleatum</td><td colspan="1" rowspan="1">ATCC 25586</td><td colspan="1" rowspan="1">1 x 107</td></tr><tr><td colspan="1" rowspan="1">Fusobacterium nucleatum</td><td colspan="1" rowspan="1">ATCC 23726</td><td colspan="1" rowspan="1">1 x 105</td></tr><tr><td colspan="1" rowspan="3">Haemophilus influenzae</td><td colspan="1" rowspan="1">Haemophilus influenzae</td><td colspan="1" rowspan="1">ATCC 19418</td><td colspan="1" rowspan="1">1 x 105</td></tr><tr><td colspan="1" rowspan="1">Haemophilus influenzae</td><td colspan="1" rowspan="1">ATCC 9006</td><td colspan="1" rowspan="1">1x 107</td></tr><tr><td colspan="1" rowspan="1">Haemophilus influenzae</td><td colspan="1" rowspan="1">ATCC 33930</td><td colspan="1" rowspan="1">1 x 104</td></tr><tr><td colspan="1" rowspan="2">Klebsiella oxytoca</td><td colspan="1" rowspan="1">Klebsiella oxytoca</td><td colspan="1" rowspan="1">ATCC 43165</td><td colspan="1" rowspan="1">1 x 107</td></tr><tr><td colspan="1" rowspan="1">Klebsiella oxytoca</td><td colspan="1" rowspan="1">ATCC 8724</td><td colspan="1" rowspan="1">1 x 107</td></tr><tr><td colspan="1" rowspan="2">Klebsiella pneumoniae group</td><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">CDC#0160</td><td colspan="1" rowspan="1">1 x 106</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">CDC#0107</td><td colspan="1" rowspan="1">1 x 106</td></tr><tr><td colspan="1" rowspan="2">Morganella morganii</td><td colspan="1" rowspan="1">Morganella morganii</td><td colspan="1" rowspan="1">ATCC 25829</td><td colspan="1" rowspan="1">1 x 107</td></tr><tr><td colspan="1" rowspan="1">Morganella morganii</td><td colspan="1" rowspan="1">CDC#0133</td><td colspan="1" rowspan="1">1 x 107</td></tr><tr><td colspan="1" rowspan="3">Neisseria meningitidis</td><td colspan="1" rowspan="1">Neisseria meningitidis</td><td colspan="1" rowspan="1">ATCC 13090</td><td colspan="1" rowspan="1">1 x 105</td></tr><tr><td colspan="1" rowspan="1">Neisseria meningitidis</td><td colspan="1" rowspan="1">ATCC 13102</td><td colspan="1" rowspan="1">1x 104</td></tr><tr><td colspan="1" rowspan="1">Neisseria meningitidis</td><td colspan="1" rowspan="1">NCTC 10026</td><td colspan="1" rowspan="1">1 x 104</td></tr><tr><td colspan="1" rowspan="2">Proteus</td><td colspan="1" rowspan="1">Proteus vulgaris</td><td colspan="1" rowspan="1">ATCC 6896</td><td colspan="1" rowspan="1">1 x 107</td></tr><tr><td colspan="1" rowspan="1">Proteus vulgaris</td><td colspan="1" rowspan="1">ATCC 6380</td><td colspan="1" rowspan="1">1 x 107</td></tr><tr><td colspan="1" rowspan="2">Proteus mirabilis</td><td colspan="1" rowspan="1">Proteus mirabilis</td><td colspan="1" rowspan="1">CDC#0159</td><td colspan="1" rowspan="1">1 x 106</td></tr><tr><td colspan="1" rowspan="1">Proteus mirabilis</td><td colspan="1" rowspan="1">ATCC 43071</td><td colspan="1" rowspan="1">1 x 106</td></tr><tr><td colspan="1" rowspan="3">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">CDC#0103</td><td colspan="1" rowspan="1">1 x 106</td></tr><tr><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">NCTC 13437</td><td colspan="1" rowspan="1">1 x 106</td></tr><tr><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">SDx071</td><td colspan="1" rowspan="1">1 x 105</td></tr><tr><td colspan="1" rowspan="2">Salmonella</td><td colspan="1" rowspan="1">Salmonella bongori</td><td colspan="1" rowspan="1">ATCC 43975</td><td colspan="1" rowspan="1">1 x 105</td></tr><tr><td colspan="1" rowspan="1">Salmonella enterica</td><td colspan="1" rowspan="1">ATCC 6962</td><td colspan="1" rowspan="1">1x 105</td></tr><tr><td colspan="1" rowspan="2">Serratia</td><td colspan="1" rowspan="1">Serratia liquefaciens</td><td colspan="1" rowspan="1">ATCC 27592</td><td colspan="1" rowspan="1">1 x 106</td></tr><tr><td colspan="1" rowspan="1">Serratia plymuthica</td><td colspan="1" rowspan="1">ATCC 53858</td><td colspan="1" rowspan="1">1x 107</td></tr><tr><td colspan="1" rowspan="2">Serratia marcescens</td><td colspan="1" rowspan="1">Serratia marcescens</td><td colspan="1" rowspan="1">ATCC 14041</td><td colspan="1" rowspan="1">1 x 107</td></tr><tr><td colspan="1" rowspan="1">Serratia marcescens</td><td colspan="1" rowspan="1">ATCC 14756</td><td colspan="1" rowspan="1">1 x 105</td></tr><tr><td colspan="1" rowspan="2">Stenotrophomonas maltophilia</td><td colspan="1" rowspan="1">Stenotrophomonas maltophilia</td><td colspan="1" rowspan="1">ATCC 13637</td><td colspan="1" rowspan="1">1 x 106</td></tr><tr><td colspan="1" rowspan="1">Stenotrophomonas maltophilia</td><td colspan="1" rowspan="1">ATCC 17666</td><td colspan="1" rowspan="1">1 x 107</td></tr><tr><td colspan="1" rowspan="2">Pan Candida</td><td colspan="1" rowspan="1">Candia albicans</td><td colspan="1" rowspan="1">ATCC 10231</td><td colspan="1" rowspan="1">1 x 106</td></tr><tr><td colspan="1" rowspan="1">Candia glabrata</td><td colspan="1" rowspan="1">ATCC 15126</td><td colspan="1" rowspan="1">1 x 105</td></tr><tr><td colspan="1" rowspan="6">Pan Gram-Positive</td><td colspan="1" rowspan="1">Enterococcus faecalis</td><td colspan="1" rowspan="1">ATCC 51575</td><td colspan="1" rowspan="1">1 x 105</td></tr><tr><td colspan="1" rowspan="1">Enterococcus faecium</td><td colspan="1" rowspan="1">ATCC 31282</td><td colspan="1" rowspan="1">1 x 107</td></tr><tr><td colspan="1" rowspan="1">Bacillus subtilis</td><td colspan="1" rowspan="1">ATCC 21008</td><td colspan="1" rowspan="1">1 x 106</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">ATCC BAA-2313</td><td colspan="1" rowspan="1">1 x 105</td></tr><tr><td colspan="1" rowspan="1">Streptococcus agalactiae</td><td colspan="1" rowspan="1">ATCC 13813</td><td colspan="1" rowspan="1">1 x 106</td></tr><tr><td colspan="1" rowspan="1">Streptococcus anginosus</td><td colspan="1" rowspan="1">ATCC 33397</td><td colspan="1" rowspan="1">1 x 106</td></tr><tr><td colspan="1" rowspan="2">CTX-M</td><td colspan="1" rowspan="1">Escherichia coli (CTX-M-15)</td><td colspan="1" rowspan="1">NCTC 13441</td><td colspan="1" rowspan="1">1x 104</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae (CTX-M-2)</td><td colspan="1" rowspan="1">CDC#0107</td><td colspan="1" rowspan="1">1 x 105</td></tr><tr><td colspan="1" rowspan="2">IMP</td><td colspan="1" rowspan="1">Enterobacter aerogenes (IMP-4)</td><td colspan="1" rowspan="1">CDC#0161</td><td colspan="1" rowspan="1">1 x 106</td></tr><tr><td colspan="1" rowspan="1">Pseudomonas aeruginosa (IMP-1)</td><td colspan="1" rowspan="1">CDC#0103</td><td colspan="1" rowspan="1">1 x 105</td></tr><tr><td colspan="1" rowspan="2">KPC</td><td colspan="1" rowspan="1">Enterobacter hormaechei (KPCvariant not known)</td><td colspan="1" rowspan="1">ATCC BAA-2082</td><td colspan="1" rowspan="1">1 x 106</td></tr><tr><td colspan="1" rowspan="1">Morganella morganii (KPC-2)</td><td colspan="1" rowspan="1">CDC#0133</td><td colspan="1" rowspan="1">1 x 106</td></tr><tr><td colspan="1" rowspan="2">NDM</td><td colspan="1" rowspan="1">Escherichia coli (NDM-1)</td><td colspan="1" rowspan="1">CDC#0118</td><td colspan="1" rowspan="1">1 x 105</td></tr><tr><td colspan="1" rowspan="1">Proteus mirabilis (NDM-1)</td><td colspan="1" rowspan="1">CDC#0159</td><td colspan="1" rowspan="1">1 x 105</td></tr><tr><td colspan="1" rowspan="4">OXA</td><td colspan="1" rowspan="1">Acinetobacter baumannii (OXA-23)</td><td colspan="1" rowspan="1">NCTC 13421</td><td colspan="1" rowspan="1">1 x 105</td></tr><tr><td colspan="1" rowspan="1">Acinetobacter baumannii (OXA-27)</td><td colspan="1" rowspan="1">NCTC 13304</td><td colspan="1" rowspan="1">1 x 105</td></tr><tr><td colspan="1" rowspan="1">Enterobacter aerogenes (OXA-48)</td><td colspan="1" rowspan="1">CDC#0074</td><td colspan="1" rowspan="1">1 x 106</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae (OXA-48)</td><td colspan="1" rowspan="1">CDC#0160</td><td colspan="1" rowspan="1">1 x 106</td></tr><tr><td colspan="1" rowspan="2">VIM</td><td colspan="1" rowspan="1">Enterobacter cloacae (VIM-1)</td><td colspan="1" rowspan="1">CDC#0154</td><td colspan="1" rowspan="1">1 x 106</td></tr><tr><td colspan="1" rowspan="1">Pseudomonas aeruginosa (VIM-10)</td><td colspan="1" rowspan="1">NCTC 13437</td><td colspan="1" rowspan="1">1 x 105</td></tr></table>

# Analytical Reactivity (Inclusivity)

A panel of 336 strains/isolates representing the genetic, temporal and geographic diversity of each target on the ePlex BCID-GN Panel was evaluated to demonstrate analytical reactivity. Bacteria were tested at $1 \mathrm { ~ x ~ } 1 0 ^ { 9 } \mathrm { C F U / m L }$ or less and fungal strains were tested at $1 \mathrm { ~ x ~ } 1 0 ^ { 6 }$ $\mathrm { C F U / m L }$ . In the cases where the initial testing concentration did not result in a “Detected” result, the concentration was increased to the point where detection was observed (see footnotes for concentration of these strains). Organisms detected are shown in Table 57. Additional strains were detected as part of the Limit of Detection (Analytical Sensitivity) Study and can be found in Table 56. Citrobacter strains that were tested but not detected include the following: $C .$ . amalonaticus, C. farmer, C. gillenii, $C .$ . murliniae, and $C .$ . sedlakii. Serratia odorifera and Staphylococcus simulans were not detected at concentrations of $1 \mathrm { ~ x ~ } 1 0 ^ { 8 } \mathrm { C F U / m L }$ and only one of three replicates were detected at concentrations of $1 \mathrm { ~ x ~ } 1 0 ^ { 9 } \mathrm { C F U / m L }$ .

Table 57: Analytical Reactivity (Inclusivity)   

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Strain</td></tr><tr><td rowspan=1 colspan=1>Acinetobacter baumannii</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=7 colspan=1>Acinetobacter baumannii</td><td rowspan=1 colspan=1>CDC#0052</td></tr><tr><td rowspan=1 colspan=1>NCTC 13302</td></tr><tr><td rowspan=1 colspan=1>NCTC 13303</td></tr><tr><td rowspan=1 colspan=1>NCTC 13305</td></tr><tr><td rowspan=1 colspan=1>NCTC 13420</td></tr><tr><td rowspan=1 colspan=1>NCTC 13422</td></tr><tr><td rowspan=1 colspan=1>NCTC 13423</td></tr><tr><td rowspan=1 colspan=1>Acinetobacter baumannii (NDM-1)</td><td rowspan=1 colspan=1>CDC#0033</td></tr><tr><td rowspan=5 colspan=1>Acinetobacter baumannii (OXA-23)</td><td rowspan=1 colspan=1>ATCC BAA-1605</td></tr><tr><td rowspan=1 colspan=1>CDC#0045</td></tr><tr><td rowspan=1 colspan=1>CDC#0056</td></tr><tr><td rowspan=1 colspan=1>NCTC 13301</td></tr><tr><td rowspan=1 colspan=1>NCTC 13424</td></tr><tr><td rowspan=1 colspan=1>Acinetobacter spp. (IMP only)</td><td rowspan=1 colspan=1>JMI4084A</td></tr><tr><td rowspan=1 colspan=1>Bacteroides fragilis</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=3 colspan=1>Bacteroides fragilis</td><td rowspan=1 colspan=1>ATCC 23745</td></tr><tr><td rowspan=1 colspan=1>ATCC 700786</td></tr><tr><td rowspan=1 colspan=1>NCTC 9343</td></tr><tr><td rowspan=1 colspan=1>Citrobacter</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=2 colspan=1>Citrobacter braakii</td><td rowspan=1 colspan=1>ATCC 43162</td></tr><tr><td rowspan=1 colspan=1>ATCC 51113</td></tr><tr><td rowspan=2 colspan=1>Citrobacter freundii</td><td rowspan=1 colspan=1>ATCC 6879</td></tr><tr><td rowspan=1 colspan=1>ATCC 8090</td></tr><tr><td rowspan=1 colspan=1>Citrobacter freundii (CTX)</td><td rowspan=1 colspan=1>JMI2047</td></tr><tr><td rowspan=1 colspan=1>Citrobacter freundii (KPC-2)</td><td rowspan=1 colspan=1>CDC#0116</td></tr><tr><td rowspan=4 colspan=1>Citrobacter koseri</td><td rowspan=1 colspan=1>ATCC 25409</td></tr><tr><td rowspan=1 colspan=1>ATCC 27028</td></tr><tr><td rowspan=1 colspan=1>ATCC 29225</td></tr><tr><td rowspan=1 colspan=1>ATCC 29936</td></tr><tr><td rowspan=1 colspan=1>Citrobacter species (CTX-15, NDM-1)</td><td rowspan=1 colspan=1>CDC #0157</td></tr><tr><td rowspan=1 colspan=1>Citrobacter werkmanii</td><td rowspan=1 colspan=1>ATCC 51114</td></tr><tr><td rowspan=1 colspan=1>Citrobacter youngae</td><td rowspan=1 colspan=1>ATCC 29935</td></tr></table>

<table><tr><td colspan="1" rowspan="1">Organism</td><td colspan="1" rowspan="1">Strain</td></tr><tr><td colspan="1" rowspan="1">Cronobacter sakazakii</td><td colspan="1" rowspan="1">cYanmaGellowb</td></tr><tr><td colspan="1" rowspan="3">Cronobacter sakazakii</td><td colspan="1" rowspan="1">ATCC 12868</td></tr><tr><td colspan="1" rowspan="1">ATCC BAA-894</td></tr><tr><td colspan="1" rowspan="1">FSL F6-0023</td></tr><tr><td colspan="1" rowspan="1">Enterobacter (non-cloacae complex)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="3">Enterobacter aerogenes</td><td colspan="1" rowspan="1">ATCC 13048</td></tr><tr><td colspan="1" rowspan="1">ATCC 29010</td></tr><tr><td colspan="1" rowspan="1">ATCC 51697</td></tr><tr><td colspan="1" rowspan="2">Enterobacter amnigenus</td><td colspan="1" rowspan="1">ATCC 33731</td></tr><tr><td colspan="1" rowspan="1">ATCC 51816B</td></tr><tr><td colspan="1" rowspan="2">Enterobacter gergoviae</td><td colspan="1" rowspan="1">ATCC 33028</td></tr><tr><td colspan="1" rowspan="1">ATCC 33426</td></tr><tr><td colspan="1" rowspan="1">Enterobacter cloacae complex</td><td colspan="1" rowspan="1">cYanmaGellowb</td></tr><tr><td colspan="1" rowspan="3">Enterobacter asburiae</td><td colspan="1" rowspan="1">ATCC 35954</td></tr><tr><td colspan="1" rowspan="1">ATCC 35955</td></tr><tr><td colspan="1" rowspan="1">ATCC 35956</td></tr><tr><td colspan="1" rowspan="1">Enterobacter cloacae (CTX-15)</td><td colspan="1" rowspan="1">CDC#0038</td></tr><tr><td colspan="1" rowspan="1">Enterobacter cloacae (CTX-9)</td><td colspan="1" rowspan="1">NCTC 13464</td></tr><tr><td colspan="1" rowspan="1">Enterobacter cloacae (CTX-15, KPC-2)</td><td colspan="1" rowspan="1">CDC#0163</td></tr><tr><td colspan="1" rowspan="1">Enterobacter cloacae (CTX, NDM)</td><td colspan="1" rowspan="1">JMI53571</td></tr><tr><td colspan="1" rowspan="2">Enterobacter cloacae subsp. cloacae</td><td colspan="1" rowspan="1">ATCC 23355</td></tr><tr><td colspan="1" rowspan="1">ATCC 35030</td></tr><tr><td colspan="1" rowspan="1">Enterobacter cloacae subsp. dissolvens</td><td colspan="1" rowspan="1">ATCC 23373</td></tr><tr><td colspan="1" rowspan="1">Enterobacter hormaechei</td><td colspan="1" rowspan="1">ATCC 700323</td></tr><tr><td colspan="1" rowspan="1">Enterobacter hormaechei subsp. hormaechei</td><td colspan="1" rowspan="1">ATCC 49162</td></tr><tr><td colspan="1" rowspan="1">Enterobacter hormaechei subsp. oharae</td><td colspan="1" rowspan="1">ATCC 49163</td></tr><tr><td colspan="1" rowspan="1">Enterobacter hormaechei subsp. steigerwaltii</td><td colspan="1" rowspan="1">CIP108489T</td></tr><tr><td colspan="1" rowspan="1">Enterobacter ludwigii</td><td colspan="1" rowspan="1">DSM-16688</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="6">Escherichia coli</td><td colspan="1" rowspan="1">ATCC 14948</td></tr><tr><td colspan="1" rowspan="1">ATCC 25922</td></tr><tr><td colspan="1" rowspan="1">ATCC 33605</td></tr><tr><td colspan="1" rowspan="1">ATCC 33876</td></tr><tr><td colspan="1" rowspan="1">ATCC 35150</td></tr><tr><td colspan="1" rowspan="1">ATCC 4157</td></tr><tr><td colspan="1" rowspan="35"></td><td colspan="1" rowspan="1">ATCC 43888</td></tr><tr><td colspan="1" rowspan="1">ATCC 51446</td></tr><tr><td colspan="1" rowspan="1">ATCC 51755</td></tr><tr><td colspan="1" rowspan="1">ATCC 53498</td></tr><tr><td colspan="1" rowspan="1">ATCC 700728</td></tr><tr><td colspan="1" rowspan="1">NCIMB 8545</td></tr><tr><td colspan="1" rowspan="1">NCTC 8620</td></tr><tr><td colspan="1" rowspan="1">ATCC 9637</td></tr><tr><td colspan="1" rowspan="1">ATCC BAA-196</td></tr><tr><td colspan="1" rowspan="1">ATCC BAA-197</td></tr><tr><td colspan="1" rowspan="1">ATCC BAA-198</td></tr><tr><td colspan="1" rowspan="1">ATCC BAA-199</td></tr><tr><td colspan="1" rowspan="1">ATCC BAA-200</td></tr><tr><td colspan="1" rowspan="1">ATCC BAA-201</td></tr><tr><td colspan="1" rowspan="1">ATCC BAA-202</td></tr><tr><td colspan="1" rowspan="1">ATCC BAA-203</td></tr><tr><td colspan="1" rowspan="1">ATCC BAA-204</td></tr><tr><td colspan="1" rowspan="1">LMC_243094647</td></tr><tr><td colspan="1" rowspan="1">LMC_243098776</td></tr><tr><td colspan="1" rowspan="1">LMC_243098947</td></tr><tr><td colspan="1" rowspan="1">LMC_243108047</td></tr><tr><td colspan="1" rowspan="1">LMC_243109799</td></tr><tr><td colspan="1" rowspan="1">LMC_243112411</td></tr><tr><td colspan="1" rowspan="1">LMC_244006281</td></tr><tr><td colspan="1" rowspan="1">LMC_244006433</td></tr><tr><td colspan="1" rowspan="1">LMC_244008038</td></tr><tr><td colspan="1" rowspan="1">LMC_244012579</td></tr><tr><td colspan="1" rowspan="1">NCTC 13351</td></tr><tr><td colspan="1" rowspan="1">NCTC 10279</td></tr><tr><td colspan="1" rowspan="1">ATCC 10536</td></tr><tr><td colspan="1" rowspan="1">ATCC 10538</td></tr><tr><td colspan="1" rowspan="1">ATCC 10799</td></tr><tr><td colspan="1" rowspan="1">ATCC 11229</td></tr><tr><td colspan="1" rowspan="1">ATCC 13762</td></tr><tr><td colspan="1" rowspan="1">ATCC 14169</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli (CTX-14)</td><td colspan="1" rowspan="1">CDC#0086</td></tr><tr><td colspan="1" rowspan="5">Escherichia coli (CTX-15)</td><td colspan="1" rowspan="1">ATCC BAA-2326</td></tr><tr><td colspan="1" rowspan="1">NCTC 13353</td></tr><tr><td colspan="1" rowspan="1">NCTC 13400</td></tr><tr><td colspan="1" rowspan="1">NCTC 13450</td></tr><tr><td colspan="1" rowspan="1">NCTC 13451</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli (CTX-3)</td><td colspan="1" rowspan="1">NCTC 13452</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli (CTX-1)</td><td colspan="1" rowspan="1">NCTC 13461</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli (CTX-2)</td><td colspan="1" rowspan="1">NCTC 13462</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli (CTX-8)</td><td colspan="1" rowspan="1">NCTC 13463</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli (CTX-15, NDM-6)</td><td colspan="1" rowspan="1">CDC#0137</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli (CTX-15, NDM-7)</td><td colspan="1" rowspan="1">CDC#0162</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli (IMP)</td><td colspan="1" rowspan="1">NCTC 13476</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli (KPC)</td><td colspan="1" rowspan="1">ATCC BAA-2340</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli (NDM-5)</td><td colspan="1" rowspan="1">CDC#0150</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli (OXA)</td><td colspan="1" rowspan="1">LMC_DR00012</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli (VIM)</td><td colspan="1" rowspan="1">JMI32465</td></tr><tr><td colspan="1" rowspan="1">Fusobacterium necrophorum</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="3">Fusobacterium necrophorum subsp.necrophorum</td><td colspan="1" rowspan="1">ATCC 25286</td></tr><tr><td colspan="1" rowspan="1">NCTC 10575</td></tr><tr><td colspan="1" rowspan="1">NCTC 10577</td></tr><tr><td colspan="1" rowspan="1">Fusobacterium nucleatum</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Fusobacterium nucleatum subsp. nucleatum</td><td colspan="1" rowspan="1">ATCC 31647</td></tr><tr><td colspan="1" rowspan="1">Fusobacterium nucleatum subsp. fusiforme</td><td colspan="1" rowspan="1">ATCC 51190</td></tr></table>

<table><tr><td colspan="1" rowspan="1">Organism</td><td colspan="1" rowspan="1">Strain</td></tr><tr><td colspan="1" rowspan="1">Fusobacterium nucleatum subsp. vincentii</td><td colspan="1" rowspan="1">ATCC 49256</td></tr><tr><td colspan="1" rowspan="1">Haemophilus influenzae</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="6">Haemophilus influenzae</td><td colspan="1" rowspan="1">ATCC 33930</td></tr><tr><td colspan="1" rowspan="1">ATCC 43065</td></tr><tr><td colspan="1" rowspan="1">ATCC 43163</td></tr><tr><td colspan="1" rowspan="1">NCTC 11931</td></tr><tr><td colspan="1" rowspan="1">NCTC 12699</td></tr><tr><td colspan="1" rowspan="1">NCTC 8143</td></tr><tr><td colspan="1" rowspan="1">Haemophilus influenzae Type b</td><td colspan="1" rowspan="1">ATCC 10211</td></tr><tr><td colspan="1" rowspan="1">Haemophilus influenzae Type c</td><td colspan="1" rowspan="1">ATCC 9007</td></tr><tr><td colspan="1" rowspan="1">Haemophilus influenzae Type d</td><td colspan="1" rowspan="1">ATCC 9332</td></tr><tr><td colspan="1" rowspan="1">Haemophilus influenzae Type e</td><td colspan="1" rowspan="1">NCTC 8472</td></tr><tr><td colspan="1" rowspan="1">Haemophilus influenzae ype </td><td colspan="1" rowspan="1">ATCC 9833</td></tr><tr><td colspan="1" rowspan="1">Klebsiella oxytoca</td><td colspan="1" rowspan="1">cYanmaGellow</td></tr><tr><td colspan="1" rowspan="5">Klebsiella oxytoca</td><td colspan="1" rowspan="1">ATCC 43086</td></tr><tr><td colspan="1" rowspan="1">ATCC 43863</td></tr><tr><td colspan="1" rowspan="1">ATCC 49131</td></tr><tr><td colspan="1" rowspan="1">ATCC 700324</td></tr><tr><td colspan="1" rowspan="1">ATCC 51817</td></tr><tr><td colspan="1" rowspan="1">Klebsiella oxytoca (KPC-3)</td><td colspan="1" rowspan="1">CDC#0147</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae group</td><td colspan="1" rowspan="1">cYanmaGllow</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae (CTX-15)</td><td colspan="1" rowspan="1">CDC#0109</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae (CTX-25)</td><td colspan="1" rowspan="1">NCTC 13465</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae (CTX, KPC)</td><td colspan="1" rowspan="1">IMH-C2261309</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae (CTX, NDM-1)</td><td colspan="1" rowspan="1">NCTC 13443</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae (CTX-15; NDM-1;OXA-232)</td><td colspan="1" rowspan="1">CDC#0153</td></tr><tr><td colspan="1" rowspan="2">Klebsiella pneumoniae (CTX-15, OXA-232)</td><td colspan="1" rowspan="1">CDC#0075</td></tr><tr><td colspan="1" rowspan="1">CDC#0066</td></tr><tr><td colspan="1" rowspan="4">Klebsiella pneumoniae (CTX-15, OXA-181)</td><td colspan="1" rowspan="1">CDC#0039</td></tr><tr><td colspan="1" rowspan="1">CDC#0140</td></tr><tr><td colspan="1" rowspan="1">CDC#0141</td></tr><tr><td colspan="1" rowspan="1">CDC#0142</td></tr><tr><td colspan="1" rowspan="2">Klebsiella pneumoniae (IMP-4)</td><td colspan="1" rowspan="1">CDC#0034</td></tr><tr><td colspan="1" rowspan="1">CDC#0080</td></tr><tr><td colspan="1" rowspan="3">Klebsiella pneumoniae (KPC-3)</td><td colspan="1" rowspan="1">CDC#0125</td></tr><tr><td colspan="1" rowspan="1">CDC#0112</td></tr><tr><td colspan="1" rowspan="1">CDC#0113</td></tr><tr><td colspan="1" rowspan="5">Klebsiella pneumoniae (KPC)</td><td colspan="1" rowspan="1">ATCC BAA-1705</td></tr><tr><td colspan="1" rowspan="1">IMH-C2260742</td></tr><tr><td colspan="1" rowspan="1">IMH-C3151729</td></tr><tr><td colspan="1" rowspan="1">IMH-C4151728</td></tr><tr><td colspan="1" rowspan="1">IMH-C4171868</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae (OXA-48)</td><td colspan="1" rowspan="1">NCTC 13442</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae (CTX-15; VIM-27)</td><td colspan="1" rowspan="1">CDC#0040</td></tr><tr><td colspan="1" rowspan="3">Klebsiella pneumoniae (VIM-1)</td><td colspan="1" rowspan="1">CDC#0135</td></tr><tr><td colspan="1" rowspan="1">NCTC 13439</td></tr><tr><td colspan="1" rowspan="1">NCTC 13440</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae subsp. Ozaenae</td><td colspan="1" rowspan="1">ATCC 11296</td></tr><tr><td colspan="1" rowspan="4">Klebsiella peie ubsp.e</td><td colspan="1" rowspan="1">ATCC 13883</td></tr><tr><td colspan="1" rowspan="1">ATCC 27736</td></tr><tr><td colspan="1" rowspan="1">ATCC 51503</td></tr><tr><td colspan="1" rowspan="1">ATCC 51504</td></tr><tr><td colspan="1" rowspan="1">Klebsiella quasipneumoniae</td><td colspan="1" rowspan="1">ATCC 700603</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae ubsp.rhinoscleromatis</td><td colspan="1" rowspan="1">ATCC 9436</td></tr><tr><td colspan="1" rowspan="1">Klebsiella variicola</td><td colspan="1" rowspan="1">ATCC BAA-830</td></tr><tr><td colspan="1" rowspan="1">Morganella morganii</td><td colspan="1" rowspan="1">cYanmaGellow</td></tr><tr><td colspan="1" rowspan="2">Morganella morganii</td><td colspan="1" rowspan="1">ATCC 25830</td></tr><tr><td colspan="1" rowspan="1">GM148-209</td></tr><tr><td colspan="1" rowspan="1">Morganella morganii (CTX-15; NDM-1)</td><td colspan="1" rowspan="1">CDC#0057c</td></tr><tr><td colspan="1" rowspan="1">Neisseria meningitidis</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Neisseria meningitidis Serotype A</td><td colspan="1" rowspan="1">ATCC 13077</td></tr><tr><td colspan="1" rowspan="1">Neisseria meningitidis Serotype B</td><td colspan="1" rowspan="1">NCTC 10026</td></tr><tr><td colspan="1" rowspan="1">Neisseria meningitidis Serotype W135</td><td colspan="1" rowspan="1">NCTC 11203</td></tr><tr><td colspan="1" rowspan="1">Neisseria meningitidis Serotype Y</td><td colspan="1" rowspan="1">ATCC 35561</td></tr><tr><td colspan="1" rowspan="1">Proteus</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Proteus hauseri</td><td colspan="1" rowspan="1">ATCC 13315</td></tr><tr><td colspan="1" rowspan="2">Proteus mirabilis</td><td colspan="1" rowspan="1">ATCC 33583</td></tr><tr><td colspan="1" rowspan="1">ATCC BAA-663</td></tr><tr><td colspan="1" rowspan="1">Proteus mirabilis (IMP)</td><td colspan="1" rowspan="1">JMI955389</td></tr><tr><td colspan="1" rowspan="1">Proteus mirabilis (KPC-6)</td><td colspan="1" rowspan="1">CDC#0155</td></tr><tr><td colspan="1" rowspan="1">Proteus penneri</td><td colspan="1" rowspan="1">ATCC 35197</td></tr><tr><td colspan="1" rowspan="4">Proteus vulgaris</td><td colspan="1" rowspan="1">ATCC 33420</td></tr><tr><td colspan="1" rowspan="1">ATCC 49132</td></tr><tr><td colspan="1" rowspan="1">ATCC 8427</td></tr><tr><td colspan="1" rowspan="1">NCTC 4636</td></tr><tr><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Pseudomonas aeruginosa (IMP-14)</td><td colspan="1" rowspan="1">CDC#0092</td></tr><tr><td colspan="1" rowspan="1">Pseudomonas aeruginosa (IMP-1)</td><td colspan="1" rowspan="1">CDC#0241</td></tr><tr><td colspan="1" rowspan="1">Pseudomonas aeruginosa (IMP)</td><td colspan="1" rowspan="1">CDC#0439</td></tr><tr><td colspan="1" rowspan="1">Pseudomonas aeruginosa (KPC-5)</td><td colspan="1" rowspan="1">CDC#0090</td></tr><tr><td colspan="1" rowspan="1">Pseudomonas aeruginosa (VIM-2)</td><td colspan="1" rowspan="1">CDC#0100</td></tr><tr><td colspan="1" rowspan="1">Pseudomonas aeruginosa (VIM-4)</td><td colspan="1" rowspan="1">CDC#0054</td></tr><tr><td colspan="1" rowspan="1">Salmonella</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Salmonella enterica serovar 4,[5],12:i</td><td colspan="1" rowspan="1">FSL S5-0580</td></tr><tr><td colspan="1" rowspan="1">Salmonella enterica serovar Agona</td><td colspan="1" rowspan="1">ATCC 51957</td></tr><tr><td colspan="1" rowspan="1">Salmonella enterica serovar Bareilly</td><td colspan="1" rowspan="1">ATCC 9115</td></tr><tr><td colspan="1" rowspan="1">Salmonella enterica serovar Braenderup</td><td colspan="1" rowspan="1">ATCC 700136</td></tr><tr><td colspan="1" rowspan="1">Salmonella enterica serovar Enteritidis</td><td colspan="1" rowspan="1">ATCC BAA-708</td></tr><tr><td colspan="1" rowspan="1">Salmonella enterica serovar Hadar</td><td colspan="1" rowspan="1">ATCC 51956</td></tr><tr><td colspan="1" rowspan="1">Salmonella enterica serovar Heidelberg</td><td colspan="1" rowspan="1">ATCC 8326</td></tr><tr><td colspan="1" rowspan="1">Salmonella enterica serovar Infantis</td><td colspan="1" rowspan="1">ATCC BAA-1675</td></tr><tr><td colspan="1" rowspan="1">Salmonella enterica serovar Javiana</td><td colspan="1" rowspan="1">ATCC 10721</td></tr><tr><td colspan="1" rowspan="1">Salmonella enterica serovar Montevideo</td><td colspan="1" rowspan="1">ATCC 8387</td></tr><tr><td colspan="1" rowspan="1">Salmonella enterica serovar Muenchen</td><td colspan="1" rowspan="1">ATCC 8388</td></tr><tr><td colspan="1" rowspan="1">Salmonella enterica serovar Oranienburg</td><td colspan="1" rowspan="1">ATCC 9239</td></tr><tr><td colspan="1" rowspan="1">Salmonella enterica serovar Paratyphi B</td><td colspan="1" rowspan="1">FSL S5-0447</td></tr><tr><td colspan="1" rowspan="1">Salmonella enterica serovar Saintpaul</td><td colspan="1" rowspan="1">ATCC 9712</td></tr><tr><td colspan="1" rowspan="1">Salmonella enterica serovar Thompson</td><td colspan="1" rowspan="1">ATCC 8391</td></tr><tr><td colspan="1" rowspan="1">Salmonella enterica serovar Tyhi</td><td colspan="1" rowspan="1">ATCC 19430</td></tr><tr><td colspan="1" rowspan="1">Salmonella enterica subsp. arizonae</td><td colspan="1" rowspan="1">ATCC 13314</td></tr><tr><td colspan="1" rowspan="1">Salmonella enterica subsp. diarizonae</td><td colspan="1" rowspan="1">ATCC 12325</td></tr><tr><td colspan="1" rowspan="1">Salmonella enterica sbsp. enterica seovarTyphimurium</td><td colspan="1" rowspan="1">ATCC 14028</td></tr><tr><td colspan="1" rowspan="1">Salmonella enterica subsp. houtenae</td><td colspan="1" rowspan="1">ATCC 29834</td></tr><tr><td colspan="1" rowspan="1">Salmonella enterica subsp. indica</td><td colspan="1" rowspan="1">ATCC BAA-1578</td></tr><tr><td colspan="1" rowspan="1">Salmonella enterica subsp. salamae</td><td colspan="1" rowspan="1">ATCC 6959</td></tr><tr><td colspan="1" rowspan="1">Salmonella enterica subsp. enterica serovarMississippi</td><td colspan="1" rowspan="1">FSL A4-0633</td></tr><tr><td colspan="1" rowspan="1">Sal tSchwarzengrund</td><td colspan="1" rowspan="1">FSL S5-0458</td></tr><tr><td colspan="1" rowspan="1">Serratia</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Serratia ficaria</td><td colspan="1" rowspan="1">ATCC 33105</td></tr><tr><td colspan="1" rowspan="1">Serratia fonticola</td><td colspan="1" rowspan="1">ATCC 29844</td></tr><tr><td colspan="1" rowspan="1">Serratia grimesii</td><td colspan="1" rowspan="1">ATCC 14460</td></tr><tr><td colspan="1" rowspan="3">Serratia marcescens</td><td colspan="1" rowspan="1">ATCC 13880</td></tr><tr><td colspan="1" rowspan="1">ATCC 43861</td></tr><tr><td colspan="1" rowspan="1">ATCC 43862</td></tr><tr><td colspan="1" rowspan="1">Serratia marcescens (CTX)</td><td colspan="1" rowspan="1">JMI10244</td></tr></table>

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Strain</td></tr><tr><td rowspan=2 colspan=1>Serratia rubidaea</td><td rowspan=1 colspan=1>ATCC 27593</td></tr><tr><td rowspan=1 colspan=1>ATCC 29025</td></tr><tr><td rowspan=1 colspan=1>Stenotrophomonas maltophilia</td><td rowspan=1 colspan=1>cYnmallow</td></tr><tr><td rowspan=3 colspan=1>Stenotrophomonas maltophilia</td><td rowspan=1 colspan=1>ATCC 13636</td></tr><tr><td rowspan=1 colspan=1>GM148-207</td></tr><tr><td rowspan=1 colspan=1>GM148-208</td></tr><tr><td rowspan=1 colspan=1>Pan-Gram Positive</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Bacillus amyloliquefaciens</td><td rowspan=1 colspan=1>ATCC 23845</td></tr><tr><td rowspan=1 colspan=1>Bacillus atrophaeus</td><td rowspan=1 colspan=1>ATCC 49337</td></tr><tr><td rowspan=1 colspan=1>Bacillus cereus</td><td rowspan=1 colspan=1>ATCC 10876</td></tr><tr><td rowspan=1 colspan=1>Bacillus licheniformis</td><td rowspan=1 colspan=1>ATCC 21039</td></tr><tr><td rowspan=1 colspan=1>Bacillus thuringiensis</td><td rowspan=1 colspan=1>ATCC 35646</td></tr><tr><td rowspan=1 colspan=1>Enterococcus avium</td><td rowspan=1 colspan=1>ATCC 14025</td></tr><tr><td rowspan=1 colspan=1>Enterococcus casseliflavus</td><td rowspan=1 colspan=1>ATCC 700327</td></tr><tr><td rowspan=1 colspan=1>Enterococcus faecalis</td><td rowspan=1 colspan=1>JMI876745</td></tr><tr><td rowspan=1 colspan=1>Enterococcus gallinarum</td><td rowspan=1 colspan=1>ATCC 49573</td></tr><tr><td rowspan=1 colspan=1>Enterococcus hirae</td><td rowspan=1 colspan=1>ATCC 49479</td></tr><tr><td rowspan=1 colspan=1>Enterococcus raffinosus</td><td rowspan=1 colspan=1>ATCC 49464</td></tr><tr><td rowspan=1 colspan=1>Enterococcus saccharolyticus</td><td rowspan=1 colspan=1>ATCC 43076</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus capitis</td><td rowspan=1 colspan=1>NRS866</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus chromogenes</td><td rowspan=1 colspan=1>ATCC 43764</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus cohnii</td><td rowspan=1 colspan=1>ATCC 29974</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus epidermidis</td><td rowspan=1 colspan=1>ATCC 35984</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus gallinarum</td><td rowspan=1 colspan=1>ATCC 700401</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus haemolyticus</td><td rowspan=1 colspan=1>ATCC 29970</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus hominis</td><td rowspan=1 colspan=1>ATCC 27844</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus hyicus</td><td rowspan=1 colspan=1>ATCC 11249</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus lentus</td><td rowspan=1 colspan=1>ATCC 700403</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus lugdunensis</td><td rowspan=1 colspan=1>ATCC 49576</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus pasteuri</td><td rowspan=1 colspan=1>ATCC 51128</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus vitulinus</td><td rowspan=1 colspan=1>ATCC 51699</td></tr><tr><td rowspan=1 colspan=1>Streptococcus constellatus</td><td rowspan=1 colspan=1>ATCC 27513</td></tr><tr><td rowspan=1 colspan=1>Streptococcus dysgalactiae</td><td rowspan=1 colspan=1>ATCC 35666</td></tr><tr><td rowspan=1 colspan=1>Streptococcus equi</td><td rowspan=1 colspan=1>ATCC 9528</td></tr><tr><td rowspan=1 colspan=1>Streptococcus gallolyticus</td><td rowspan=1 colspan=1>ATCC 9809</td></tr><tr><td rowspan=1 colspan=1>Streptococcus gordonii</td><td rowspan=1 colspan=1>ATCC 35557</td></tr><tr><td rowspan=1 colspan=1>Streptococcus infantis</td><td rowspan=1 colspan=1>ATCC 700779</td></tr><tr><td rowspan=1 colspan=1>Streptococcus intermedius</td><td rowspan=1 colspan=1>ATCC 27335</td></tr><tr><td rowspan=1 colspan=1>Streptococcus mitis</td><td rowspan=1 colspan=1>ATCC 49456</td></tr><tr><td rowspan=1 colspan=1>Streptococcus oralis</td><td rowspan=1 colspan=1>ATCC 35037</td></tr><tr><td rowspan=1 colspan=1>Streptococcus parasanguinis</td><td rowspan=1 colspan=1>ATCC 15909</td></tr><tr><td rowspan=1 colspan=1>Streptococcus pneumoniae</td><td rowspan=1 colspan=1>ATCC 8335</td></tr><tr><td rowspan=1 colspan=1>Streptococcus pyogenes</td><td rowspan=1 colspan=1>ATCC 12344</td></tr><tr><td rowspan=1 colspan=1>Streptococcus salivarius</td><td rowspan=1 colspan=1>ATCC 7073</td></tr><tr><td rowspan=1 colspan=1>Streptococcus thoraltensis</td><td rowspan=1 colspan=1>ATCC 700865E</td></tr><tr><td rowspan=1 colspan=1>Pan Candida</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=2 colspan=1>Candida albicans</td><td rowspan=1 colspan=1>ATCC 24433</td></tr><tr><td rowspan=1 colspan=1>ATCC 90028</td></tr><tr><td rowspan=2 colspan=1>Candida glabrata</td><td rowspan=1 colspan=1>ATCC 2001</td></tr><tr><td rowspan=1 colspan=1>ATCC 66032</td></tr><tr><td rowspan=3 colspan=1>Candida krusei</td><td rowspan=1 colspan=1>ATCC 14243</td></tr><tr><td rowspan=1 colspan=1>ATCC 32196</td></tr><tr><td rowspan=1 colspan=1>ATCC 34135F</td></tr><tr><td rowspan=3 colspan=1>Candida parapsilosis</td><td rowspan=1 colspan=1>ATCC 22019</td></tr><tr><td rowspan=1 colspan=1>ATCC 58895</td></tr><tr><td rowspan=1 colspan=1>ATCC 90018G</td></tr></table>

A. Non-target species used to evaluate resistance marker.   
B. $5 / 6$ replicates detected at $2 . 0 \mathrm { ~ x ~ } 1 0 ^ { 8 } \mathrm { C F U / m L }$ .   
C. $5 / 6$ replicates detected at $4 . 5 \mathrm { ~ x ~ } 1 0 ^ { 8 } \mathrm { C F U / m L }$ .   
D. Strain may have reduced sensitivity and was not $1 0 0 \%$ detected at concentrations $< 1 \mathrm { ~ x ~ } 1 0 ^ { 8 } \mathrm { C F U / m L }$ .   
E. Strain may have reduced sensitivity and was not $1 0 0 \%$ detected at concentrations $< 4 \mathrm { ~ x ~ } 1 0 ^ { 8 } \mathrm { C F U / m L }$ .   
F. In initial testing, 1/6 replicates was detected at $1 \mathrm { x } \ 1 0 ^ { 6 } \mathrm { C F U / m L }$ ; during additional testing, $3 / 3$ replicates detected at bottle positivity.   
G. In initial testing, 2/6 replicates were detected at $1 \mathrm { ~ x ~ } 1 0 ^ { 6 } \mathrm { C F U / m L }$ ; during additional testing, 6/6 replicates detected at bottle positivity.

# Predicted (in silico) Reactivity for Genus and Group Assays

In addition to species-specific assays, the ePlex BCID-GN Panel contains a number of broader genus or group-level assays including: Citrobacter, Enterobacter cloacae complex, Enterobacter (non-cloacae complex), Klebsiella pneumoniae group, Proteus, Serratia, Pan Candida, and Pan Gram-Positive assays. Tables 58-65 highlight the-predicted (in silico) reactivity (inclusivity) for these assay targets.

Note: the performance of the ePlex BCID-GN Panel has not been established for all of the organisms listed in the tables below. See the Analytical Reactivity section for data on organisms for which performance characteristics have been established (indicated with an asterisk in Tables 58-65). Some species were not assessed in silico due to lack of sequence data, though they may appear in the analytical sensitivity or specificity studies.

# Table 58: Predicted (in silico) Reactivity (Inclusivity) Results for Citrobacter

<table><tr><td rowspan=1 colspan=3>Detection Predicted for ≥95% of target sequences</td></tr><tr><td rowspan=1 colspan=1>Citrobacter koseri*</td><td rowspan=1 colspan=1>Citrobacter intermedius</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=3>Detection Predicted for 85%-94% of target sequences</td></tr><tr><td rowspan=1 colspan=1>Citrobacter freundii*</td><td rowspan=1 colspan=1>Citrobacter braakii*</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=3>Detection Predicted for &lt;85.0% of target sequences</td></tr><tr><td rowspan=1 colspan=1>Citrobacter werkmanii* (66.7%)</td><td rowspan=1 colspan=1>Citrobacter youngae* (50.0%)</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=3>Detection Not Predicted</td></tr><tr><td rowspan=1 colspan=1>Citrobacter europaeus</td><td rowspan=1 colspan=1>Citrobacter gillenii</td><td rowspan=1 colspan=1>Citrobacter amalonaticusA</td></tr><tr><td rowspan=1 colspan=1>Citrobacter farmeri</td><td rowspan=1 colspan=1>Citrobacter sedlakii</td><td rowspan=1 colspan=1>Citrobacter murliniae</td></tr></table>

A. Detection predicted in silico, however ATCCBAA-2563 was not detected in wet testing.

Table 59: Predicted (in silico) Reactivity (Inclusivity) Results for Enterobacter cloacae complex   

<table><tr><td rowspan=1 colspan=3>Detection Predicted for ≥95% of target sequences</td></tr><tr><td rowspan=1 colspan=1>Enterobacter cloacae*</td><td rowspan=1 colspan=1>Enterobacter asburiae*</td><td rowspan=1 colspan=1>Enterobacter hormaechei*</td></tr><tr><td rowspan=1 colspan=1>Enterobacter xiangfangensis</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=3>Detection Predicted for 85%-94% of target sequences</td></tr><tr><td rowspan=1 colspan=3>None Identified</td></tr><tr><td rowspan=1 colspan=3>Detection Predicted for &lt;85.0% of target sequences</td></tr><tr><td rowspan=1 colspan=1>Enterobacter ludwigii* (68.4%)</td><td rowspan=1 colspan=1>Enterobacter nimipressuralis(25.0%)</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=3>Detection Not Predicted</td></tr><tr><td rowspan=1 colspan=1>Enterobacter kobei</td><td rowspan=1 colspan=1>Enterobacter cancerogenus</td><td rowspan=1 colspan=1></td></tr></table>

Table 60: Predicted (in silico) Reactivity (Inclusivity) Results for Enterobacter (non-cloacae complex)   

<table><tr><td rowspan=1 colspan=3>Detection Predicted for ≥95% of target sequences</td></tr><tr><td rowspan=1 colspan=1>Enterobacter aerogenes*</td><td rowspan=1 colspan=1>Enterobacter gergoviae*</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=3>Detection Predicted for 85%-94% of target sequences</td></tr><tr><td rowspan=1 colspan=3>None Identified</td></tr><tr><td rowspan=1 colspan=3>Detection Predicted for &lt;85.0% of target sequences</td></tr><tr><td rowspan=1 colspan=1>Enterobacter amnigenus* (62.5%)</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=3>Detection Not Predicted</td></tr><tr><td rowspan=1 colspan=3>None Identified</td></tr></table>

Table 61: Predicted (in silico) Reactivity (Inclusivity) Results for Klebsiella pneumoniae Group   

<table><tr><td rowspan=1 colspan=3>Detection Predicted for ≥95% of target sequences</td></tr><tr><td rowspan=1 colspan=1>Klebsiella pneumoniae*</td><td rowspan=1 colspan=1>Klebsiella quasipneumoniae*</td><td rowspan=1 colspan=1>Klebsiella variicola*</td></tr><tr><td rowspan=1 colspan=3>Detection Predicted for 85%-94% of target sequences</td></tr><tr><td rowspan=1 colspan=3>None Identified</td></tr><tr><td rowspan=1 colspan=3>Detection Predicted for &lt;85.0% of target sequences</td></tr><tr><td rowspan=1 colspan=3>None Identified</td></tr><tr><td rowspan=1 colspan=3>Detection Not Predicted</td></tr><tr><td rowspan=1 colspan=3>None Identified</td></tr></table>

Table 62: Predicted (in silico) Reactivity (Inclusivity) Results for Proteus   

<table><tr><td rowspan=1 colspan=3>Detection Predicted for ≥95% of target sequences</td></tr><tr><td rowspan=1 colspan=1>Proteus mirabilis*</td><td rowspan=1 colspan=1>Proteus penneri*</td><td rowspan=1 colspan=1>Proteus vulgaris*</td></tr><tr><td rowspan=1 colspan=1>Proteus hauseri*</td><td rowspan=1 colspan=1>Proteus cibarius</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=3>Detection Predicted for 85%-94% of target sequences</td></tr><tr><td rowspan=1 colspan=3>None Identified</td></tr><tr><td rowspan=1 colspan=3>Detection Predicted for &lt;85.0% of target sequences</td></tr><tr><td rowspan=1 colspan=3>None Identified</td></tr><tr><td rowspan=1 colspan=3>Detection Not Predicted</td></tr><tr><td rowspan=1 colspan=3>Proteus myxofaciens</td></tr></table>

Table 63: Predicted (in silico) Reactivity (Inclusivity) Results for Serratia   

<table><tr><td rowspan=1 colspan=3>Detection Predicted for ≥95% of target sequences</td></tr><tr><td rowspan=1 colspan=1>Serratia marcescens*</td><td rowspan=1 colspan=1>Serratia grimesii*</td><td rowspan=1 colspan=1>Serratia rubidaea*</td></tr><tr><td rowspan=1 colspan=1>Serratia ficaria*</td><td rowspan=1 colspan=1>Serratia liquefaciens*</td><td rowspan=1 colspan=1>Serratia proteamaculans</td></tr><tr><td rowspan=1 colspan=1>Serratia fonticola*</td><td rowspan=1 colspan=1>Serratia plymuthica*</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=3>Detection Predicted for 85%-94% of target sequences</td></tr><tr><td rowspan=1 colspan=3>None Identified</td></tr><tr><td rowspan=1 colspan=3>Detection Predicted for &lt;85.0% of target sequences</td></tr><tr><td rowspan=1 colspan=1>Serratia quinivorans (33.3%)</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=3>Detection Not Predicted</td></tr><tr><td rowspan=1 colspan=1>Serratia nematodiphila</td><td rowspan=1 colspan=1>Serratia odoriferaA*</td><td rowspan=1 colspan=1>Serratia ureilytica</td></tr></table>

B. Not predicted in silico, however ATCC 33077 was intermittently detected in wet testing. See Analytical Reactivity (Inclusivity) Study.

# Table 64: Predicted (in silico) Reactivity (Inclusivity) Results for Pan Candida

<table><tr><td rowspan=1 colspan=3>Detection Predicted for ≥95% of target sequences</td></tr><tr><td rowspan=1 colspan=1>Candida albicans*</td><td rowspan=1 colspan=1>Candida glabrata*</td><td rowspan=1 colspan=1>Candida krusei*</td></tr><tr><td rowspan=1 colspan=1>Candida parapsilosis*</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=3>Detection Predicted for 85%-94% of target sequences</td></tr><tr><td rowspan=1 colspan=3>None Identified</td></tr><tr><td rowspan=1 colspan=3>Detection Predicted for &lt;85% of target sequences</td></tr><tr><td rowspan=1 colspan=3>None Identified</td></tr><tr><td rowspan=1 colspan=3>Detection Not Predicted</td></tr><tr><td rowspan=1 colspan=3>Den waso rc nalyeiataasva.</td></tr></table>

Table 65: Predicted (in silico) Reactivity (Inclusivity) Results for Pan Gram-Positive   

<table><tr><td colspan="3" rowspan="1">Detection Predicted for ≥95% of target sequences</td></tr><tr><td colspan="3" rowspan="1">Bacillus</td></tr><tr><td colspan="1" rowspan="1">Bacillus amyloliquefaciens*</td><td colspan="1" rowspan="1">Bacillus paralicheniformis</td><td colspan="1" rowspan="1">Bacillus toyonensis</td></tr><tr><td colspan="1" rowspan="1">Bacillus atrophaeus*</td><td colspan="1" rowspan="1">Bacillus siamensis</td><td colspan="1" rowspan="1">Bacillus vallismortis</td></tr><tr><td colspan="1" rowspan="1">Bacillus bombysepticus</td><td colspan="1" rowspan="1">Bacillus subtilis*</td><td colspan="1" rowspan="1">Bacillus velezensis</td></tr><tr><td colspan="1" rowspan="1">Bacillus licheniformis*</td><td colspan="1" rowspan="1">Bacillus tequilensis</td><td colspan="1" rowspan="1">Bacillus weihenstephanensis</td></tr><tr><td colspan="1" rowspan="1">Bacillus methylotrophicus</td><td colspan="1" rowspan="1">Bacillus thuringiensis*</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="3" rowspan="1">Enterococcus</td></tr><tr><td colspan="1" rowspan="1">Enterococcus avium*</td><td colspan="1" rowspan="1">Enterococcus faecium*</td><td colspan="1" rowspan="1">Enterococcus raffinosus*</td></tr><tr><td colspan="1" rowspan="1">Enterococcus dispar</td><td colspan="1" rowspan="1">Enterococcus flavescens</td><td colspan="1" rowspan="1">Enterococcus saccharolyticus*</td></tr><tr><td colspan="1" rowspan="1">Enterococcus durans</td><td colspan="1" rowspan="1">Enterococcus malodoratus</td><td colspan="1" rowspan="1">Enterococcus thailandicus</td></tr><tr><td colspan="1" rowspan="1">Enterococcus faecalis*</td><td colspan="1" rowspan="1">Enterococcus pseudoavium</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="3" rowspan="1">Staphylococcus</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus*</td><td colspan="1" rowspan="1">Staphylococcus haemolyticus*</td><td colspan="1" rowspan="1">Staphylococcus pseudintermedius</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus agnetis</td><td colspan="1" rowspan="1">Staphylococcus hominis*</td><td colspan="1" rowspan="1">Staphylococcus pseudolugunensis</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus argensis</td><td colspan="1" rowspan="1">Staphylococcus hominis subsp.novobiosepticus</td><td colspan="1" rowspan="1">Staphylococcus pulvereri</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus argenteus</td><td colspan="1" rowspan="1">Staphylococcus hyicus*</td><td colspan="1" rowspan="1">Staphylococcus rostri</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus auricularis</td><td colspan="1" rowspan="1">Staphylococcus jettensis</td><td colspan="1" rowspan="1">Staphylococcus saprophyticus</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus capitis*</td><td colspan="1" rowspan="1">Staphylococcus kloosii</td><td colspan="1" rowspan="1">Staphylococcus schleiferi</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus caprae</td><td colspan="1" rowspan="1">Staphylococcus lentus*</td><td colspan="1" rowspan="1">Staphylococcus schweitzeri</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus carnosus</td><td colspan="1" rowspan="1">Staphylococcus lugdunensis*</td><td colspan="1" rowspan="1">Staphylococcus sciuri</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus chromogenes*</td><td colspan="1" rowspan="1">Staphylococcus lutrae</td><td colspan="1" rowspan="1">Staphylococcus simiae</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus cohnii*</td><td colspan="1" rowspan="1">Staphylococcus massiliensis</td><td colspan="1" rowspan="1">Staphylococcus simulans</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus delphini</td><td colspan="1" rowspan="1">Staphylococcus microti</td><td colspan="1" rowspan="1">Staphylococcus stepanovicii</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus devriesei</td><td colspan="1" rowspan="1">Staphylococcus muscae</td><td colspan="1" rowspan="1">Staphylococcus succinus</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus epidermidis*</td><td colspan="1" rowspan="1">Staphylococcus nepalensis</td><td colspan="1" rowspan="1">Staphylococcus vitulinus*</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus equorum</td><td colspan="1" rowspan="1">Staphylococcus pasteuri*</td><td colspan="1" rowspan="1">Staphylococcus warneri</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus felis</td><td colspan="1" rowspan="1">Staphylococcus petrasii</td><td colspan="1" rowspan="1">Staphylococcus xylosus</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus fleurettii</td><td colspan="1" rowspan="1">Staphylococus pettenkoferi</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Staphylococcus gallinarum*</td><td colspan="1" rowspan="1">Staphylococcus piscifermentans</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="3" rowspan="1">Streptococcus</td></tr><tr><td colspan="1" rowspan="1">Streptococcus agalactiae</td><td colspan="1" rowspan="1">Streptococcus infantarius</td><td colspan="1" rowspan="1">Streptococcus phocae</td></tr><tr><td colspan="1" rowspan="1">Streptococcus alactolyticus</td><td colspan="1" rowspan="1">Streptococcus infantis*</td><td colspan="1" rowspan="1">Streptococcus pneumoniae*</td></tr><tr><td colspan="1" rowspan="1">Streptococcus anginosus</td><td colspan="1" rowspan="1">Streptococcus intermedius*</td><td colspan="1" rowspan="1">Streptococcus porcinus</td></tr><tr><td colspan="1" rowspan="1">Streptococcus australis</td><td colspan="1" rowspan="1">Streptococcus intestinalis</td><td colspan="1" rowspan="1">Streptococcus porcorum</td></tr><tr><td colspan="1" rowspan="1">Streptococcus caballi</td><td colspan="1" rowspan="1">Streptococcus lactarius</td><td colspan="1" rowspan="1">Streptococcus pseudopneumoniae</td></tr><tr><td colspan="1" rowspan="1">Streptococcus constellatus</td><td colspan="1" rowspan="1">Streptococcus loxodontisalivarius</td><td colspan="1" rowspan="1">Streptococcus pseudoporcinus</td></tr><tr><td colspan="1" rowspan="1">Streptococcus criceti</td><td colspan="1" rowspan="1">Streptococcus luteciae</td><td colspan="1" rowspan="1">Strepotococcus pyogenes</td></tr><tr><td colspan="1" rowspan="1">Streptococcus cristatus</td><td colspan="1" rowspan="1">Streptococcus lutetiensis</td><td colspan="1" rowspan="1">Streptococcus rifensis</td></tr><tr><td colspan="1" rowspan="1">Streptococcus danieliae</td><td colspan="1" rowspan="1">Streptococcus macedonicus</td><td colspan="1" rowspan="1">Streptococcus rubneri</td></tr><tr><td colspan="1" rowspan="1">Streptococcus dentasini</td><td colspan="1" rowspan="1">Streptococcus marimammalium</td><td colspan="1" rowspan="1">Streptococcus salivarius*</td></tr><tr><td colspan="1" rowspan="1">Streptococcus dentisani</td><td colspan="1" rowspan="1">Streptococcus massiliensis</td><td colspan="1" rowspan="1">Streptococcus saliviloxodontae</td></tr><tr><td colspan="1" rowspan="1">Streptococcus didelphis</td><td colspan="1" rowspan="1">Streptococcus mitis*</td><td colspan="1" rowspan="1">Streptococcus sanguinis</td></tr><tr><td colspan="1" rowspan="1">Streptococcus difficilis</td><td colspan="1" rowspan="1">Streptococcus moroccensis</td><td colspan="1" rowspan="1">Streptococcus seminale</td></tr><tr><td colspan="1" rowspan="1">Streptococcus dysgalactiae subsp.dysgalactiae</td><td colspan="1" rowspan="1">Streptococcus oligofermentans</td><td colspan="1" rowspan="1">Streptococcus sinensis</td></tr><tr><td colspan="1" rowspan="1">Streptococcus dysgalactiae subsp.equisimilis</td><td colspan="1" rowspan="1">Streptococcus oralis*</td><td colspan="1" rowspan="1">Streptococcus suis</td></tr><tr><td colspan="1" rowspan="1">Streptococcus dysgalactiae*</td><td colspan="1" rowspan="1">Streptococcus oricebi</td><td colspan="1" rowspan="1">Streptococcus thermophilus</td></tr><tr><td colspan="1" rowspan="1">Streptococcus equi*</td><td colspan="1" rowspan="1">Streptococcus orisratti</td><td colspan="1" rowspan="1">Streptococcus thoraltensis*</td></tr><tr><td colspan="1" rowspan="1">Streptococcus equinus</td><td colspan="1" rowspan="1">Streptococcus panodentis</td><td colspan="1" rowspan="1">Streptococcus tigurinus</td></tr><tr><td colspan="1" rowspan="1">Streptococcus fryi</td><td colspan="1" rowspan="1">Streptococcus parasanguinis*</td><td colspan="1" rowspan="1">Streptococcus troglodytae</td></tr><tr><td colspan="1" rowspan="1">Streptococcus gallolyticus*</td><td colspan="1" rowspan="1">Streptococcus parasuis</td><td colspan="1" rowspan="1">Streptococcus troglodytidis</td></tr><tr><td colspan="1" rowspan="1">Streptococcus gordonii*</td><td colspan="1" rowspan="1">Streptococcus parauberis</td><td colspan="1" rowspan="1">Streptococcus urinalis</td></tr><tr><td colspan="1" rowspan="1">Streptococcus himalayensis</td><td colspan="1" rowspan="1">Streptococcus pasteuri</td><td colspan="1" rowspan="1">Streptococcus ursoris</td></tr><tr><td colspan="1" rowspan="1">Streptococcus hongkongensis</td><td colspan="1" rowspan="1">Streptococcus pasteurianus</td><td colspan="1" rowspan="1">Streptococcus vestibularis</td></tr><tr><td colspan="1" rowspan="1">Streptococcus hyointestinalis</td><td colspan="1" rowspan="1">Streptococcus peroris</td><td colspan="1" rowspan="1">Streptococcus waiu</td></tr><tr><td colspan="3" rowspan="1">Detection Predicted for 85%-94% of target sequences</td></tr><tr><td colspan="1" rowspan="1">Bacillus cereus*</td><td colspan="1" rowspan="1">Enterococcus hirae*</td><td colspan="1" rowspan="1">Staphylococcus saccharolyticus</td></tr><tr><td colspan="1" rowspan="1">Enterococcus casseliflavus*</td><td colspan="1" rowspan="1">Staphylococcus arlettae</td><td colspan="1" rowspan="1">Streptococcus bovis</td></tr><tr><td colspan="1" rowspan="1">Enterococcus cecorum</td><td colspan="1" rowspan="1">Staphylococcus condimenti</td><td colspan="1" rowspan="1">Streptococcus uberis</td></tr><tr><td colspan="1" rowspan="1">Enterococcus gallinarum</td><td colspan="1" rowspan="1">Staphylococcus intermedius</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="3" rowspan="1">Detection Predicted for &lt;85.0% of target sequences</td></tr><tr><td colspan="1" rowspan="1">Bacillus mojavensis (77.8%)</td><td colspan="1" rowspan="1">Bacillus sonorensis (83.3%)</td><td colspan="1" rowspan="1">Streptococcus halichoeri (66.7%)</td></tr><tr><td colspan="1" rowspan="1">Streptococcus ratti (75.0%)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="3" rowspan="1">Detection Not Predicted</td></tr><tr><td colspan="1" rowspan="1">Bacillus pseudomycoides</td><td colspan="1" rowspan="1">Enterococcus sulfureus</td><td colspan="1" rowspan="1">Streptococcus hyovaginalis</td></tr><tr><td colspan="1" rowspan="1">Enterococcus aquimarinus</td><td colspan="1" rowspan="1">Enterococcus termitis</td><td colspan="1" rowspan="1">Streptococcus ictaluri</td></tr><tr><td colspan="1" rowspan="1">Enterococcus asini</td><td colspan="1" rowspan="1">Enterococcus ureasiticus</td><td colspan="1" rowspan="1">Streptococcus iniae</td></tr><tr><td colspan="1" rowspan="1">Enterococcus caccae</td><td colspan="1" rowspan="1">Enterococcus ureilyticus</td><td colspan="1" rowspan="1">Streptococcus lactis</td></tr><tr><td colspan="1" rowspan="1">Enterococcus camelliae</td><td colspan="1" rowspan="1">Enterococcus villorum</td><td colspan="1" rowspan="1">Streptococcus macacae</td></tr><tr><td colspan="1" rowspan="1">Enterococcus canintestini</td><td colspan="1" rowspan="1">Staphylococcus caseolyticus</td><td colspan="1" rowspan="1">Streptococcus marmotae</td></tr><tr><td colspan="1" rowspan="1">Enterococcus canis</td><td colspan="1" rowspan="1">Streptococcus acidominimus</td><td colspan="1" rowspan="1">Streptococcus merionis</td></tr><tr><td colspan="1" rowspan="1">Enterococcus columbae</td><td colspan="1" rowspan="1">Streptococcus azizii</td><td colspan="1" rowspan="1">Streptococcus milleri</td></tr><tr><td colspan="1" rowspan="1">Enterococcus devriesei</td><td colspan="1" rowspan="1">Streptococcus cameli</td><td colspan="1" rowspan="1">Streptococcus minor</td></tr><tr><td colspan="1" rowspan="1">Enterococcus haemoperoxidus</td><td colspan="1" rowspan="1">Streptococcus canis</td><td colspan="1" rowspan="1">Streptococcus oriloxodontae</td></tr><tr><td colspan="1" rowspan="1">Enterococcus hawaiiensis</td><td colspan="1" rowspan="1">Streptococcus castoreus</td><td colspan="1" rowspan="1">Streptococcus orisasini</td></tr><tr><td colspan="1" rowspan="1">Enterococcus hermanniensis</td><td colspan="1" rowspan="1">Streptococcus cremoris</td><td colspan="1" rowspan="1">Streptococcus orisuis</td></tr><tr><td colspan="1" rowspan="1">Enterococcus italicus</td><td colspan="1" rowspan="1">Streptococcus criae</td><td colspan="1" rowspan="1">Streptococcus ovis</td></tr><tr><td colspan="1" rowspan="1">Enterococcus mundtii</td><td colspan="1" rowspan="1">Streptococcus cuniculi</td><td colspan="1" rowspan="1">Streptococcus pharyngis</td></tr><tr><td colspan="1" rowspan="1">Enterococcus pallens</td><td colspan="1" rowspan="1">Streptococcus dentapri</td><td colspan="1" rowspan="1">Streptococcus pluranimalium</td></tr><tr><td colspan="1" rowspan="1">Enterococcus pernyi</td><td colspan="1" rowspan="1">Streptococcus dentiloxodontae</td><td colspan="1" rowspan="1">Streptococcus plurextorum</td></tr><tr><td colspan="1" rowspan="1">Enterococcus phoeniculicola</td><td colspan="1" rowspan="1">Streptococcus dentirousetti</td><td colspan="1" rowspan="1">Streptococcus plutanimalium</td></tr><tr><td colspan="1" rowspan="1">Enterococcus plantarum</td><td colspan="1" rowspan="1">Streptococcus devriesei</td><td colspan="1" rowspan="1">Streptococcus porci</td></tr><tr><td colspan="1" rowspan="1">Enterococcus quebecensis</td><td colspan="1" rowspan="1">Streptococcus downei</td><td colspan="1" rowspan="1">Streptococcus rupicaprae</td></tr><tr><td colspan="1" rowspan="1">Enterococcus ratti</td><td colspan="1" rowspan="1">Streptococcus entericus</td><td colspan="1" rowspan="1">Streptococcus sobrinus</td></tr><tr><td colspan="1" rowspan="1">Enterococcus rivorum</td><td colspan="1" rowspan="1">Streptococcus ferus</td><td colspan="1" rowspan="1">Streptococcus tangierensis</td></tr><tr><td colspan="1" rowspan="1">Enterococcus rotai</td><td colspan="1" rowspan="1">Streptococcus gallinaceus</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Enterococcus silesiacus</td><td colspan="1" rowspan="1">Streptococcus henryi</td><td colspan="1" rowspan="1"></td></tr></table>

# Predicted (in silico) Reactivity for Resistance Markers

The ePlex BCID-GN Panel contains six resistance markers that were each assessed for predicted in silico reactivity. Tables 66-76 highlight the-predicted (in silico) reactivity for CTX-M, IMP, KPC, NDM, OXA, and VIM. Strains that were tested as part of the Analytical Reactivity (Inclusivity) Study are marked with an asterisk in Tables 66-76. It is noted that the performance of the ePlex BCID-GN Panel has not been established for all organisms listed in Tables 66-76. Table 77 includes all variants that are not predicted to be detected by the BCID-GN Panel by in silico analysis.

The CTX-M assay on the ePlex BCID-GN Panel is designed to detect the following CTX-M groups: CTX-M-1, CTX-M-2, CTX-M-8, CTX-M-9, and CTX-M-25.

Table 66: Predicted (in silico) Reactivity (Inclusivity) Results for CTX-M-1   

<table><tr><td rowspan=1 colspan=1>Target</td><td rowspan=1 colspan=1>Associated Organism</td><td rowspan=1 colspan=1>VariantDetected</td></tr><tr><td rowspan=1 colspan=1>Acinetobacterbaumannii</td><td rowspan=1 colspan=1>Acinetobacter baumannii</td><td rowspan=1 colspan=1>CTX-M-15</td></tr><tr><td rowspan=6 colspan=1>Ctirobacter</td><td rowspan=4 colspan=1>Citrobacter freundii</td><td rowspan=1 colspan=1>CTX-M-3</td></tr><tr><td rowspan=1 colspan=1>CTX-M-15*</td></tr><tr><td rowspan=1 colspan=1>CTX-M-30</td></tr><tr><td rowspan=1 colspan=1>CTX-M-55</td></tr><tr><td rowspan=2 colspan=1>Citrobacter koseri</td><td rowspan=1 colspan=1>CTX-M-3</td></tr><tr><td rowspan=1 colspan=1>CTX-M-15</td></tr><tr><td rowspan=3 colspan=1>Enterobacter(non-cloacaeComplex)</td><td rowspan=2 colspan=1>Enterobacter aerogenes</td><td rowspan=1 colspan=1>CTX-M-3</td></tr><tr><td rowspan=1 colspan=1>CTX-M-15</td></tr><tr><td rowspan=1 colspan=1>Enterobacter gergoviae</td><td rowspan=1 colspan=1>CTX-M-15</td></tr><tr><td rowspan=8 colspan=1>Enterobactercloacae Complex</td><td rowspan=1 colspan=1>Enterobacter asburiae</td><td rowspan=1 colspan=1>CTX-M-15</td></tr><tr><td rowspan=6 colspan=1>Enterobacter cloacae</td><td rowspan=1 colspan=1>CTX-M-3</td></tr><tr><td rowspan=1 colspan=1>CTX-M-15*</td></tr><tr><td rowspan=1 colspan=1>CTX-M-22</td></tr><tr><td rowspan=1 colspan=1>CTX-M-37</td></tr><tr><td rowspan=1 colspan=1>CTX-M-89</td></tr><tr><td rowspan=1 colspan=1>CTX-M-177</td></tr><tr><td rowspan=1 colspan=1>Enterobacter hormaechei</td><td rowspan=1 colspan=1>CTX-M-15</td></tr><tr><td rowspan=11 colspan=1>Escherichia coli</td><td rowspan=11 colspan=1>Escherichia coli</td><td rowspan=1 colspan=1>CTX-M-1*</td></tr><tr><td rowspan=1 colspan=1>CTX-M-3*</td></tr><tr><td rowspan=1 colspan=1>CTX-M-10</td></tr><tr><td rowspan=1 colspan=1>CTX-M-12</td></tr><tr><td rowspan=1 colspan=1>CTX-M-14*</td></tr><tr><td rowspan=1 colspan=1>CTX-M-15*</td></tr><tr><td rowspan=1 colspan=1>CTX-M-22</td></tr><tr><td rowspan=1 colspan=1>CTX-M-23</td></tr><tr><td rowspan=1 colspan=1>CTX-M-28</td></tr><tr><td rowspan=1 colspan=1>CTX-M-29</td></tr><tr><td rowspan=1 colspan=1>CTX-M-32</td></tr></table>

<table><tr><td colspan="1" rowspan="1">Target</td><td colspan="1" rowspan="1">Associated Organism</td><td colspan="1" rowspan="1">VariantDetected</td></tr><tr><td colspan="1" rowspan="30"></td><td colspan="1" rowspan="30"></td><td colspan="1" rowspan="1">CTX-M-33</td></tr><tr><td colspan="1" rowspan="1">CTX-M-34</td></tr><tr><td colspan="1" rowspan="1">CTX-M-36</td></tr><tr><td colspan="1" rowspan="1">CTX-M-38</td></tr><tr><td colspan="1" rowspan="1">CTX-M-42</td></tr><tr><td colspan="1" rowspan="1">CTX-M-55</td></tr><tr><td colspan="1" rowspan="1">CTX-M-58</td></tr><tr><td colspan="1" rowspan="1">CTX-M-61</td></tr><tr><td colspan="1" rowspan="1">CTX-M-65</td></tr><tr><td colspan="1" rowspan="1">CTX-M-69</td></tr><tr><td colspan="1" rowspan="1">CTX-M-79</td></tr><tr><td colspan="1" rowspan="1">CTX-M-82</td></tr><tr><td colspan="1" rowspan="1">CTX-M-101</td></tr><tr><td colspan="1" rowspan="1">CTX-M-103</td></tr><tr><td colspan="1" rowspan="1">CTX-M-117</td></tr><tr><td colspan="1" rowspan="1">CTX-M-123</td></tr><tr><td colspan="1" rowspan="1">CTX-M-127</td></tr><tr><td colspan="1" rowspan="1">CTX-M-132</td></tr><tr><td colspan="1" rowspan="1">CTX-M-138</td></tr><tr><td colspan="1" rowspan="1">CTX-M-139</td></tr><tr><td colspan="1" rowspan="1">CTX-M-142</td></tr><tr><td colspan="1" rowspan="1">CTX-M-144</td></tr><tr><td colspan="1" rowspan="1">CTX-M-150</td></tr><tr><td colspan="1" rowspan="1">CTX-M-158</td></tr><tr><td colspan="1" rowspan="1">CTX-M-163</td></tr><tr><td colspan="1" rowspan="1">CTX-M-166</td></tr><tr><td colspan="1" rowspan="1">CTX-M-169</td></tr><tr><td colspan="1" rowspan="1">CTX-M-170</td></tr><tr><td colspan="1" rowspan="1">CTX-M-171</td></tr><tr><td colspan="1" rowspan="1">CTX-M-172</td></tr><tr><td colspan="1" rowspan="8"></td><td colspan="1" rowspan="8"></td><td colspan="1" rowspan="1">CTX-M-174</td></tr><tr><td colspan="1" rowspan="1">CTX-M-175</td></tr><tr><td colspan="1" rowspan="1">CTX-M-179</td></tr><tr><td colspan="1" rowspan="1">CTX-M-180</td></tr><tr><td colspan="1" rowspan="1">CTX-M-181</td></tr><tr><td colspan="1" rowspan="1">CTX-M-182</td></tr><tr><td colspan="1" rowspan="1">CTX-M-184</td></tr><tr><td colspan="1" rowspan="1">CTX-M-186</td></tr><tr><td colspan="1" rowspan="4">Klebsiella oxytoca</td><td colspan="1" rowspan="4">Klebsiella oxytoca</td><td colspan="1" rowspan="1">CTX-M-3</td></tr><tr><td colspan="1" rowspan="1">CTX-M-15</td></tr><tr><td colspan="1" rowspan="1">CTX-M-28</td></tr><tr><td colspan="1" rowspan="1">CTX-M-162</td></tr><tr><td colspan="1" rowspan="22">Klebsiellapneumoniae group</td><td colspan="1" rowspan="22">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">CTX-M-1</td></tr><tr><td colspan="1" rowspan="1">CTX-M-3</td></tr><tr><td colspan="1" rowspan="1">CTX-M-10</td></tr><tr><td colspan="1" rowspan="1">CTX-M-11</td></tr><tr><td colspan="1" rowspan="1">CTX-M-12</td></tr><tr><td colspan="1" rowspan="1">CTX-M-15*</td></tr><tr><td colspan="1" rowspan="1">CTX-M-22</td></tr><tr><td colspan="1" rowspan="1">CTX-M-28</td></tr><tr><td colspan="1" rowspan="1">CTX-M-32</td></tr><tr><td colspan="1" rowspan="1">CTX-M-52</td></tr><tr><td colspan="1" rowspan="1">CTX-M-54</td></tr><tr><td colspan="1" rowspan="1">CTX-M-55</td></tr><tr><td colspan="1" rowspan="1">CTX-M-57</td></tr><tr><td colspan="1" rowspan="1">CTX-M-60</td></tr><tr><td colspan="1" rowspan="1">CTX-M-62</td></tr><tr><td colspan="1" rowspan="1">CTX-M-71</td></tr><tr><td colspan="1" rowspan="1">CTX-M-72</td></tr><tr><td colspan="1" rowspan="1">CTX-M-96</td></tr><tr><td colspan="1" rowspan="1">CTX-M-155</td></tr><tr><td colspan="1" rowspan="1">CTX-M-156</td></tr><tr><td colspan="1" rowspan="1">CTX-M-157</td></tr><tr><td colspan="1" rowspan="1">CTX-M-173</td></tr></table>

<table><tr><td rowspan=1 colspan=1>Target</td><td rowspan=1 colspan=1>Associated Organism</td><td rowspan=1 colspan=1>VariantDetected</td></tr><tr><td rowspan=4 colspan=1></td><td rowspan=4 colspan=1></td><td rowspan=1 colspan=1>CTX-M-176</td></tr><tr><td rowspan=1 colspan=1>CTX-M-183</td></tr><tr><td rowspan=1 colspan=1>CTX-M-197</td></tr><tr><td rowspan=1 colspan=1>CTX-M-204</td></tr><tr><td rowspan=3 colspan=1>Morganellamorganii</td><td rowspan=3 colspan=1>Morganella morganii</td><td rowspan=1 colspan=1>CTX-M-3</td></tr><tr><td rowspan=1 colspan=1>CTX-M-15*</td></tr><tr><td rowspan=1 colspan=1>CTX-M-55</td></tr><tr><td rowspan=9 colspan=1>Proteus mirabilis</td><td rowspan=9 colspan=1>Proteus mirabilis</td><td rowspan=1 colspan=1>CTX-M-1</td></tr><tr><td rowspan=1 colspan=1>CTX-M-3</td></tr><tr><td rowspan=1 colspan=1>CTX-M-15</td></tr><tr><td rowspan=1 colspan=1>CTX-M-32</td></tr><tr><td rowspan=1 colspan=1>CTX-M-66</td></tr><tr><td rowspan=1 colspan=1>CTX-M-116</td></tr><tr><td rowspan=1 colspan=1>CTX-M-136</td></tr><tr><td rowspan=1 colspan=1>CTX-M-164</td></tr><tr><td rowspan=1 colspan=1>CTX-M-167</td></tr><tr><td rowspan=3 colspan=1>Pseudomonasaeruginosa</td><td rowspan=3 colspan=1>Pseudomonas aeruginosa</td><td rowspan=1 colspan=1>CTX-M-15</td></tr><tr><td rowspan=1 colspan=1>CTX-M-28</td></tr><tr><td rowspan=1 colspan=1>CTX-M-32</td></tr><tr><td rowspan=9 colspan=1>Salmonella</td><td rowspan=6 colspan=1>Salmonella enterica</td><td rowspan=1 colspan=1>CTX-M-15</td></tr><tr><td rowspan=1 colspan=1>CTX-M-53</td></tr><tr><td rowspan=1 colspan=1>CTX-M-55</td></tr><tr><td rowspan=1 colspan=1>CTX-M-57</td></tr><tr><td rowspan=1 colspan=1>CTX-M-61</td></tr><tr><td rowspan=1 colspan=1>CTX-M-88</td></tr><tr><td rowspan=1 colspan=1>Salmonella sp.</td><td rowspan=1 colspan=1>CTX-M-3</td></tr><tr><td rowspan=2 colspan=1>Salmonella typhimurium</td><td rowspan=1 colspan=1>CTX-M-37</td></tr><tr><td rowspan=1 colspan=1>CTX-M-61</td></tr><tr><td rowspan=1 colspan=1>Serratia</td><td rowspan=1 colspan=1>Serratia liquefaciens</td><td rowspan=1 colspan=1>CTX-M-22</td></tr><tr><td rowspan=3 colspan=1>Serratiamarcescens/Serratia</td><td rowspan=3 colspan=1>Serratia marcescens</td><td rowspan=1 colspan=1>CTX-M-3</td></tr><tr><td rowspan=1 colspan=1>CTX-M-15</td></tr><tr><td rowspan=1 colspan=1>CTX-M-55</td></tr><tr><td rowspan=1 colspan=1>Stenotrophomonasmaltophilia</td><td rowspan=1 colspan=1>Stenotrophomonasmaltophilia</td><td rowspan=1 colspan=1>CTX-M-15</td></tr></table>

# Table 67: Predicted (in silico) Reactivity (Inclusivity) Results for CTX-M-2

<table><tr><td rowspan=1 colspan=1>Target</td><td rowspan=1 colspan=1>AssociatedOrganism</td><td rowspan=1 colspan=1>VariantDetected</td></tr><tr><td rowspan=4 colspan=1>Acinetobacterbaumannii</td><td rowspan=4 colspan=1>Acinetobacterbaumannii</td><td rowspan=1 colspan=1>CTX-M-2</td></tr><tr><td rowspan=1 colspan=1>CTX-M-5</td></tr><tr><td rowspan=1 colspan=1>CTX-M-43</td></tr><tr><td rowspan=1 colspan=1>CTX-M-115</td></tr><tr><td rowspan=1 colspan=1>Enterobacter cloacaecomplex</td><td rowspan=1 colspan=1>Enterobacter cloacae</td><td rowspan=1 colspan=1>CTX-M-5</td></tr><tr><td rowspan=6 colspan=1>Escherichia coli</td><td rowspan=6 colspan=1>Escherichia coli</td><td rowspan=1 colspan=1>CTX-M-2*</td></tr><tr><td rowspan=1 colspan=1>CTX-M-44</td></tr><tr><td rowspan=1 colspan=1>CTX-M-56</td></tr><tr><td rowspan=1 colspan=1>CTX-M-92</td></tr><tr><td rowspan=1 colspan=1>CTX-M-97</td></tr><tr><td rowspan=1 colspan=1>CTX-M-124</td></tr><tr><td rowspan=5 colspan=1>Klebsiellapneumoniae Group</td><td rowspan=5 colspan=1>Klebsiellapneumoniae</td><td rowspan=1 colspan=1>CTX-M-2</td></tr><tr><td rowspan=1 colspan=1>CTX-M-35</td></tr><tr><td rowspan=1 colspan=1>CTX-M-59</td></tr><tr><td rowspan=1 colspan=1>CTX-M-141</td></tr><tr><td rowspan=1 colspan=1>CTX-M-165</td></tr></table>

<table><tr><td rowspan=1 colspan=1>Target</td><td rowspan=1 colspan=1>AssociatedOrganism</td><td rowspan=1 colspan=1>VariantDetected</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>EY</td><td rowspan=1 colspan=1>CTX-M-200</td></tr><tr><td rowspan=1 colspan=1>Morganella morganii</td><td rowspan=1 colspan=1>Morganella morganii</td><td rowspan=1 colspan=1>CTX-M-2</td></tr><tr><td rowspan=3 colspan=1>Proteusmirabilis/Proteus</td><td rowspan=3 colspan=1>Proteus mirabilis</td><td rowspan=1 colspan=1>CTX-M-2</td></tr><tr><td rowspan=1 colspan=1>CTX-M-20</td></tr><tr><td rowspan=1 colspan=1>CTX-M-171</td></tr><tr><td rowspan=2 colspan=1>Pseudomonasaeruginosa</td><td rowspan=2 colspan=1>Pseudomonasaeruginosa</td><td rowspan=1 colspan=1>CTX-1-2</td></tr><tr><td rowspan=1 colspan=1>CTX-M-2</td></tr><tr><td rowspan=9 colspan=1>Salmonella</td><td rowspan=5 colspan=1>Salmonella enterica</td><td rowspan=1 colspan=1>CTX-M-2</td></tr><tr><td rowspan=1 colspan=1>CTX-M-4</td></tr><tr><td rowspan=1 colspan=1>CTX-M-5</td></tr><tr><td rowspan=1 colspan=1>CTX-M-6</td></tr><tr><td rowspan=1 colspan=1>CTX-M-7</td></tr><tr><td rowspan=4 colspan=1>Salmonellatyphimurium</td><td rowspan=1 colspan=1>CTX-M-2</td></tr><tr><td rowspan=1 colspan=1>CTX-M-4</td></tr><tr><td rowspan=1 colspan=1>CTX-M-5</td></tr><tr><td rowspan=1 colspan=1>CTX-M-7</td></tr></table>

Table 68: Predicted (in silico) Reactivity (Inclusivity) Results for CTX-M-8   

<table><tr><td rowspan=1 colspan=1>Target</td><td rowspan=1 colspan=1>AssociatedOrganism</td><td rowspan=1 colspan=1>VariantDetected</td></tr><tr><td rowspan=1 colspan=1>Citrobacter</td><td rowspan=1 colspan=1>Citrobacteramalonaticus</td><td rowspan=1 colspan=1>CTX-M-8</td></tr><tr><td rowspan=1 colspan=1>Enterobacter cloacaeComplex</td><td rowspan=1 colspan=1>Enterobacter cloacae</td><td rowspan=1 colspan=1>CTX-M-8</td></tr><tr><td rowspan=1 colspan=1>Escherichia coli</td><td rowspan=1 colspan=1>Escherichia coli</td><td rowspan=1 colspan=1>CTX-M-8*</td></tr></table>

<table><tr><td rowspan=1 colspan=1>Target</td><td rowspan=1 colspan=1>AssociatedOrganism</td><td rowspan=1 colspan=1>VariantDetected</td></tr><tr><td rowspan=2 colspan=1>Klebsiellapneumoniae Group</td><td rowspan=2 colspan=1>Klebsiellapneumoniae</td><td rowspan=1 colspan=1>CTX-M-8</td></tr><tr><td rowspan=1 colspan=1>CTX-M-63</td></tr><tr><td rowspan=1 colspan=1>Morganella morganii</td><td rowspan=1 colspan=1>Morganella morganii</td><td rowspan=1 colspan=1>CTX-M-63</td></tr><tr><td rowspan=1 colspan=1>Salmonella</td><td rowspan=1 colspan=1>Salmonella enterica</td><td rowspan=1 colspan=1>CTX-M-8</td></tr></table>

# Table 69: Predicted (in silico) Reactivity (Inclusivity) Results for CTX-M-9

<table><tr><td rowspan=1 colspan=1>Target</td><td rowspan=1 colspan=1>AssociatedOrganism</td><td rowspan=1 colspan=1>VariantDetected</td></tr><tr><td rowspan=2 colspan=1>Citrobacter</td><td rowspan=2 colspan=1>Citrobacter freundii</td><td rowspan=1 colspan=1>CTX-M-14</td></tr><tr><td rowspan=1 colspan=1>CTX-M-65</td></tr><tr><td rowspan=1 colspan=1>Enterobacter non-cloacae Complex</td><td rowspan=1 colspan=1>Enterobacteraerogenes</td><td rowspan=1 colspan=1>CTX-M-9</td></tr><tr><td rowspan=5 colspan=1>Enterobacter cloacaeComplex</td><td rowspan=5 colspan=1>Enterobactercloacae</td><td rowspan=1 colspan=1>CTX-M-9*</td></tr><tr><td rowspan=1 colspan=1>CTX-M-13</td></tr><tr><td rowspan=1 colspan=1>CTX-M-14</td></tr><tr><td rowspan=1 colspan=1>CTX-M-64</td></tr><tr><td rowspan=1 colspan=1>CTX-M-125</td></tr><tr><td rowspan=37 colspan=1>Escherichia coli</td><td rowspan=37 colspan=1>Escherichia coli</td><td rowspan=1 colspan=1>CTX-M-1/CTX-M-65</td></tr><tr><td rowspan=1 colspan=1>CTX-M-9</td></tr><tr><td rowspan=1 colspan=1>CTX-M-13</td></tr><tr><td rowspan=1 colspan=1>CTX-M-14</td></tr><tr><td rowspan=1 colspan=1>CTX-M-14/CTX-M-15</td></tr><tr><td rowspan=1 colspan=1>CTX-M-15</td></tr><tr><td rowspan=1 colspan=1>CTX-M-16</td></tr><tr><td rowspan=1 colspan=1>CTX-M-19</td></tr><tr><td rowspan=1 colspan=1>CTX-M-21</td></tr><tr><td rowspan=1 colspan=1>CTX-M-24</td></tr><tr><td rowspan=1 colspan=1>CTX-M-27</td></tr><tr><td rowspan=1 colspan=1>CTX-M-38</td></tr><tr><td rowspan=1 colspan=1>CTX-M-47</td></tr><tr><td rowspan=1 colspan=1>CTX-M-51</td></tr><tr><td rowspan=1 colspan=1>CTX-M-64</td></tr><tr><td rowspan=1 colspan=1>CTX-M-65</td></tr><tr><td rowspan=1 colspan=1>CTX-M-67</td></tr><tr><td rowspan=1 colspan=1>CTX-M-73</td></tr><tr><td rowspan=1 colspan=1>CTX-M-82</td></tr><tr><td rowspan=1 colspan=1>CTX-M-87</td></tr><tr><td rowspan=1 colspan=1>CTX-M-93</td></tr><tr><td rowspan=1 colspan=1>CTX-M-97</td></tr><tr><td rowspan=1 colspan=1>CTX-M-98</td></tr><tr><td rowspan=1 colspan=1>CTX-M-102</td></tr><tr><td rowspan=1 colspan=1>CTX-M-104</td></tr><tr><td rowspan=1 colspan=1>CTX-M-105</td></tr><tr><td rowspan=1 colspan=1>CTX-M-106</td></tr><tr><td rowspan=1 colspan=1>CTX-M-121</td></tr><tr><td rowspan=1 colspan=1>CTX-M-122</td></tr><tr><td rowspan=1 colspan=1>CTX-M-126</td></tr><tr><td rowspan=1 colspan=1>CTX-M-129</td></tr><tr><td rowspan=1 colspan=1>CTX-M-130</td></tr><tr><td rowspan=1 colspan=1>CTX-M-132</td></tr><tr><td rowspan=1 colspan=1>CTX-M-134</td></tr><tr><td rowspan=1 colspan=1>CTX-M-137</td></tr><tr><td rowspan=1 colspan=1>CTX-M-148</td></tr><tr><td rowspan=1 colspan=1>CTX-M-161</td></tr></table>

<table><tr><td rowspan=1 colspan=1>Target</td><td rowspan=1 colspan=1>AssociatedOrganism</td><td rowspan=1 colspan=1>VariantDetected</td></tr><tr><td rowspan=10 colspan=1>−</td><td rowspan=10 colspan=1>−</td><td rowspan=1 colspan=1>CTX-M-168</td></tr><tr><td rowspan=1 colspan=1>CTX-M-173</td></tr><tr><td rowspan=1 colspan=1>CTX-M-174</td></tr><tr><td rowspan=1 colspan=1>CTX-M-176</td></tr><tr><td rowspan=1 colspan=1>CTX-M-177</td></tr><tr><td rowspan=1 colspan=1>CTX-M-191</td></tr><tr><td rowspan=1 colspan=1>CTX-M-195</td></tr><tr><td rowspan=1 colspan=1>CTX-M-196</td></tr><tr><td rowspan=1 colspan=1>CTX-M-198</td></tr><tr><td rowspan=1 colspan=1>CTX-M-199</td></tr><tr><td rowspan=19 colspan=1>Klebsiella pneumoniaeGroup</td><td rowspan=19 colspan=1>Klebsiellapneumoniae</td><td rowspan=1 colspan=1>CTX-M-9</td></tr><tr><td rowspan=1 colspan=1>CTX-M-13</td></tr><tr><td rowspan=1 colspan=1>CTX-M-14</td></tr><tr><td rowspan=1 colspan=1>CTX-M-17</td></tr><tr><td rowspan=1 colspan=1>CTX-M-18</td></tr><tr><td rowspan=1 colspan=1>CTX-M-19</td></tr><tr><td rowspan=1 colspan=1>CTX-M-24</td></tr><tr><td rowspan=1 colspan=1>CTX-M-38</td></tr><tr><td rowspan=1 colspan=1>CTX-M-46</td></tr><tr><td rowspan=1 colspan=1>CTX-M-48</td></tr><tr><td rowspan=1 colspan=1>CTX-M-49</td></tr><tr><td rowspan=1 colspan=1>CTX-M-50</td></tr><tr><td rowspan=1 colspan=1>CTX-M-65</td></tr><tr><td rowspan=1 colspan=1>CTX-M-81</td></tr><tr><td rowspan=1 colspan=1>CTX-M-99</td></tr><tr><td rowspan=1 colspan=1>CTX-M-104</td></tr><tr><td rowspan=1 colspan=1>CTX-M-147</td></tr><tr><td rowspan=1 colspan=1>CTX-M-159</td></tr><tr><td rowspan=1 colspan=1>CTX-M-201</td></tr><tr><td rowspan=5 colspan=1>Proteusmirabilis/Proteus</td><td rowspan=5 colspan=1>Proteus mirabilis</td><td rowspan=1 colspan=1>CTX-M-13</td></tr><tr><td rowspan=1 colspan=1>CTX-M-14</td></tr><tr><td rowspan=1 colspan=1>CTX-M-24</td></tr><tr><td rowspan=1 colspan=1>CTX-M-65</td></tr><tr><td rowspan=1 colspan=1>CTX-M-90</td></tr><tr><td rowspan=10 colspan=1>Salmonella</td><td rowspan=10 colspan=1>Salmonella enterica</td><td rowspan=1 colspan=1>CTX-M-9</td></tr><tr><td rowspan=1 colspan=1>CTX-M-14</td></tr><tr><td rowspan=1 colspan=1>CTX-M-25</td></tr><tr><td rowspan=1 colspan=1>CTX-M-27</td></tr><tr><td rowspan=1 colspan=1>CTX-M-65</td></tr><tr><td rowspan=1 colspan=1>CTX-M-83</td></tr><tr><td rowspan=1 colspan=1>CTX-M-84</td></tr><tr><td rowspan=1 colspan=1>CTX-M-85</td></tr><tr><td rowspan=1 colspan=1>CTX-M-86</td></tr><tr><td rowspan=1 colspan=1>CTX-M-143</td></tr><tr><td rowspan=1 colspan=1>Serratia</td><td rowspan=1 colspan=1>Serratialiquefaciens</td><td rowspan=1 colspan=1>CTX-M-14</td></tr></table>

Table 70: Predicted (in silico) Reactivity (Inclusivity) Results for CTX-M-25   

<table><tr><td rowspan=1 colspan=1>Target</td><td rowspan=1 colspan=1>AssociatedOrganism</td><td rowspan=1 colspan=1>VariantDetected</td></tr><tr><td rowspan=4 colspan=1>Escherichia coli</td><td rowspan=4 colspan=1>Escherichia coli</td><td rowspan=1 colspan=1>CTX-M-25</td></tr><tr><td rowspan=1 colspan=1>CTX-M-39</td></tr><tr><td rowspan=1 colspan=1>CTX-M-94</td></tr><tr><td rowspan=1 colspan=1>CTX-M-100</td></tr><tr><td rowspan=1 colspan=1>Klebsiellapneumoniae Group</td><td rowspan=1 colspan=1>Klebsiellapneumoniae4*</td><td rowspan=1 colspan=1>CTX-M-26</td></tr></table>

A. CTX-M-25 detected in Analytical Reactivity (Inclusivity) study.

<table><tr><td rowspan=1 colspan=1>Target</td><td rowspan=1 colspan=1>AssociatedOrganism</td><td rowspan=1 colspan=1>VariantDetected</td></tr><tr><td rowspan=4 colspan=1>Proteusmirabilis/Proteus</td><td rowspan=4 colspan=1>Proteus mirabilis</td><td rowspan=1 colspan=1>CTX-M-41</td></tr><tr><td rowspan=1 colspan=1>CTX-M-89</td></tr><tr><td rowspan=1 colspan=1>CTX-M-91</td></tr><tr><td rowspan=1 colspan=1>CTX-M-160</td></tr><tr><td rowspan=1 colspan=1>Salmonella</td><td rowspan=1 colspan=1>Salmonella enterica</td><td rowspan=1 colspan=1>CTX-M-25</td></tr></table>

Table 71: Predicted (in silico) Reactivity (Inclusivity) Results for IMP   

<table><tr><td rowspan=1 colspan=1>Target</td><td rowspan=1 colspan=1>AssociatedOrganism</td><td rowspan=1 colspan=1>VariantDetected</td></tr><tr><td rowspan=11 colspan=1>Acinetobacterbaumannii</td><td rowspan=11 colspan=1>Acinetobacterbaumannii</td><td rowspan=1 colspan=1>IMP-1</td></tr><tr><td rowspan=1 colspan=1>IMP-2</td></tr><tr><td rowspan=1 colspan=1>IMP-4</td></tr><tr><td rowspan=1 colspan=1>IMP-5</td></tr><tr><td rowspan=1 colspan=1>IMP-8</td></tr><tr><td rowspan=1 colspan=1>IMP-10</td></tr><tr><td rowspan=1 colspan=1>IMP-11</td></tr><tr><td rowspan=1 colspan=1>IMP-14</td></tr><tr><td rowspan=1 colspan=1>IMP-19</td></tr><tr><td rowspan=1 colspan=1>IMP-55</td></tr><tr><td rowspan=1 colspan=1>IMP-61</td></tr><tr><td rowspan=5 colspan=1>Citrobacter</td><td rowspan=5 colspan=1>Citrobacter freundii</td><td rowspan=1 colspan=1>IMP-1</td></tr><tr><td rowspan=1 colspan=1>IMP-4</td></tr><tr><td rowspan=1 colspan=1>IMP-8</td></tr><tr><td rowspan=1 colspan=1>IMP-23</td></tr><tr><td rowspan=1 colspan=1>IMP-38</td></tr><tr><td rowspan=1 colspan=1>Enterobacter non-cloacae Complex</td><td rowspan=1 colspan=1>Enterobacter aerogenes</td><td rowspan=1 colspan=1>IMP-4</td></tr><tr><td rowspan=7 colspan=1>Enterobacter cloacaeComplex</td><td rowspan=7 colspan=1>Enterobacter cloacae</td><td rowspan=1 colspan=1>IMP-1</td></tr><tr><td rowspan=1 colspan=1>IMP-4</td></tr><tr><td rowspan=1 colspan=1>IMP-8</td></tr><tr><td rowspan=1 colspan=1>IMP-11</td></tr><tr><td rowspan=1 colspan=1>IMP-26</td></tr><tr><td rowspan=1 colspan=1>IMP-34</td></tr><tr><td rowspan=1 colspan=1>IMP-60</td></tr><tr><td rowspan=2 colspan=1>Enterobacter cloacaeComplex</td><td rowspan=2 colspan=1>Enterobacterhormaechei</td><td rowspan=1 colspan=1>IMP-13</td></tr><tr><td rowspan=1 colspan=1>IMP-14</td></tr><tr><td rowspan=9 colspan=1>Escherichia coli</td><td rowspan=9 colspan=1>Escherichia coli</td><td rowspan=1 colspan=1>IMP-1</td></tr><tr><td rowspan=1 colspan=1>IMP-4</td></tr><tr><td rowspan=1 colspan=1>IMP-6</td></tr><tr><td rowspan=1 colspan=1>IMP-8</td></tr><tr><td rowspan=1 colspan=1>IMP-14</td></tr><tr><td rowspan=1 colspan=1>IMP-30</td></tr><tr><td rowspan=1 colspan=1>IMP-52</td></tr><tr><td rowspan=1 colspan=1>IMP-59</td></tr><tr><td rowspan=1 colspan=1>IMP-66</td></tr><tr><td rowspan=5 colspan=1>Klebsiella oxytoca</td><td rowspan=5 colspan=1>Klebsiella oxytoca</td><td rowspan=1 colspan=1>IMP-1</td></tr><tr><td rowspan=1 colspan=1>IMP-4</td></tr><tr><td rowspan=1 colspan=1>IMP-8</td></tr><tr><td rowspan=1 colspan=1>IMP-28</td></tr><tr><td rowspan=1 colspan=1>IMP-34</td></tr><tr><td rowspan=6 colspan=1>Klebsiella pneumoniaeGroup</td><td rowspan=6 colspan=1>Klebsiella pneumoniae</td><td rowspan=1 colspan=1>IMP-1</td></tr><tr><td rowspan=1 colspan=1>IMP-4*</td></tr><tr><td rowspan=1 colspan=1>IMP-6</td></tr><tr><td rowspan=1 colspan=1>IMP-8</td></tr><tr><td rowspan=1 colspan=1>IMP-10</td></tr><tr><td rowspan=1 colspan=1>IMP-13</td></tr></table>

<table><tr><td colspan="1" rowspan="1">Target</td><td colspan="1" rowspan="1">AssociatedOrganism</td><td colspan="1" rowspan="1">VariantDetected</td></tr><tr><td colspan="1" rowspan="4"></td><td colspan="1" rowspan="4"></td><td colspan="1" rowspan="1">IMP-19</td></tr><tr><td colspan="1" rowspan="1">IMP-26</td></tr><tr><td colspan="1" rowspan="1">IMP-32</td></tr><tr><td colspan="1" rowspan="1">IMP-38</td></tr><tr><td colspan="1" rowspan="3">Proteusmirabilis/Proteus</td><td colspan="1" rowspan="3">Proteus mirabilis</td><td colspan="1" rowspan="1">IMP-1</td></tr><tr><td colspan="1" rowspan="1">IMP-27</td></tr><tr><td colspan="1" rowspan="1">IMP-64</td></tr><tr><td colspan="1" rowspan="38">Pseudomonasaeruginosa</td><td colspan="1" rowspan="38">Pseudomonasaeruginosa</td><td colspan="1" rowspan="1">IMP-1*</td></tr><tr><td colspan="1" rowspan="1">IMP-2</td></tr><tr><td colspan="1" rowspan="1">IMP-4</td></tr><tr><td colspan="1" rowspan="1">IMP-6</td></tr><tr><td colspan="1" rowspan="1">IMP-7</td></tr><tr><td colspan="1" rowspan="1">IMP-9</td></tr><tr><td colspan="1" rowspan="1">IMP-10</td></tr><tr><td colspan="1" rowspan="1">IMP-11</td></tr><tr><td colspan="1" rowspan="1">IMP-13</td></tr><tr><td colspan="1" rowspan="1">IMP-14*</td></tr><tr><td colspan="1" rowspan="1">IMP-15</td></tr><tr><td colspan="1" rowspan="1">IMP-16</td></tr><tr><td colspan="1" rowspan="1">IMP-17</td></tr><tr><td colspan="1" rowspan="1">IMP-18</td></tr><tr><td colspan="1" rowspan="1">IMP-19</td></tr><tr><td colspan="1" rowspan="1">IMP-20</td></tr><tr><td colspan="1" rowspan="1">IMP-21</td></tr><tr><td colspan="1" rowspan="1">IMP-22</td></tr><tr><td colspan="1" rowspan="1">IMP-25</td></tr><tr><td colspan="1" rowspan="1">IMP-26</td></tr><tr><td colspan="1" rowspan="1">IMP-29</td></tr><tr><td colspan="1" rowspan="1">IMP-30</td></tr><tr><td colspan="1" rowspan="1">IMP-33</td></tr><tr><td colspan="1" rowspan="1">IMP-34</td></tr><tr><td colspan="1" rowspan="1">IMP-37</td></tr><tr><td colspan="1" rowspan="1">IMP-40</td></tr><tr><td colspan="1" rowspan="1">IMP-41</td></tr><tr><td colspan="1" rowspan="1">IMP-43</td></tr><tr><td colspan="1" rowspan="1">IMP-44</td></tr><tr><td colspan="1" rowspan="1">IMP-45</td></tr><tr><td colspan="1" rowspan="1">IMP-48</td></tr><tr><td colspan="1" rowspan="1">IMP-49</td></tr><tr><td colspan="1" rowspan="1">IMP-51</td></tr><tr><td colspan="1" rowspan="1">IMP-53</td></tr><tr><td colspan="1" rowspan="1">IMP-54</td></tr><tr><td colspan="1" rowspan="1">IMP-56</td></tr><tr><td colspan="1" rowspan="1">IMP-62</td></tr><tr><td colspan="1" rowspan="1">IMP-63</td></tr><tr><td colspan="1" rowspan="1">Salmonella</td><td colspan="1" rowspan="1">Salmonella enterica</td><td colspan="1" rowspan="1">IMP-4</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Serratia marcescens</td><td colspan="1" rowspan="1">IMP-1</td></tr><tr><td colspan="1" rowspan="3">Serratiamarcescens/Serratia</td><td colspan="1" rowspan="3"></td><td colspan="1" rowspan="1">IMP-2</td></tr><tr><td colspan="1" rowspan="1">IMP-6</td></tr><tr><td colspan="1" rowspan="1">IMP-8</td></tr></table>

<table><tr><td rowspan=1 colspan=1>Target</td><td rowspan=1 colspan=1>AssociatedOrganism</td><td rowspan=1 colspan=1>VariantDetected</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>IMP-24</td></tr><tr><td rowspan=1 colspan=1>Stenotrophomonasmaltophilia</td><td rowspan=1 colspan=1>Stenotrophomonasmaltophilia</td><td rowspan=1 colspan=1>IMP-25</td></tr></table>

<table><tr><td rowspan=1 colspan=1>Target</td><td rowspan=1 colspan=1>Associated Organism</td><td rowspan=1 colspan=1>VariantDetected</td></tr><tr><td rowspan=3 colspan=1>Acinetobacter baumannii</td><td rowspan=3 colspan=1>Acinetobacterbaumannii</td><td rowspan=1 colspan=1>KPC-2</td></tr><tr><td rowspan=1 colspan=1>KPC-3</td></tr><tr><td rowspan=1 colspan=1>KPC-10</td></tr><tr><td rowspan=3 colspan=1>Citrobacter</td><td rowspan=3 colspan=1>Citrobacter freundii</td><td rowspan=1 colspan=1>KPC-2*</td></tr><tr><td rowspan=1 colspan=1>KPC-3</td></tr><tr><td rowspan=1 colspan=1>KPC-18</td></tr><tr><td rowspan=2 colspan=1>Enterobacter non-cloacaeComplex</td><td rowspan=2 colspan=1>Enterobacter aerogenes</td><td rowspan=1 colspan=1>KPC-2</td></tr><tr><td rowspan=1 colspan=1>KPC-3</td></tr><tr><td rowspan=5 colspan=1>Enterobacter cloacaeComplex</td><td rowspan=5 colspan=1>Enterobacter cloacae</td><td rowspan=1 colspan=1>KPC-2*</td></tr><tr><td rowspan=1 colspan=1>KPC-3</td></tr><tr><td rowspan=1 colspan=1>KPC-4</td></tr><tr><td rowspan=1 colspan=1>KPC-13</td></tr><tr><td rowspan=1 colspan=1>KPC-18</td></tr><tr><td rowspan=8 colspan=1>Escherichia coli</td><td rowspan=8 colspan=1>Escherichia coli</td><td rowspan=1 colspan=1>KPC-2</td></tr><tr><td rowspan=1 colspan=1>KPC-3</td></tr><tr><td rowspan=1 colspan=1>KPC-4</td></tr><tr><td rowspan=1 colspan=1>KPC-5</td></tr><tr><td rowspan=1 colspan=1>KPC-9</td></tr><tr><td rowspan=1 colspan=1>KPC-18</td></tr><tr><td rowspan=1 colspan=1>KPC-21</td></tr><tr><td rowspan=1 colspan=1>KPC-28</td></tr><tr><td rowspan=2 colspan=1>Klebsiella oxytoca</td><td rowspan=2 colspan=1>Klebsiella oxytoca</td><td rowspan=1 colspan=1>KPC-2</td></tr><tr><td rowspan=1 colspan=1>KPC-3*</td></tr><tr><td rowspan=4 colspan=1>Klebsiella pneumoniaeGroup</td><td rowspan=4 colspan=1>Klebsiella pneumoniae</td><td rowspan=1 colspan=1>KPC-1</td></tr><tr><td rowspan=1 colspan=1>KPC-2</td></tr><tr><td rowspan=1 colspan=1>KPC-3*</td></tr><tr><td rowspan=1 colspan=1>KPC-4</td></tr></table>

Table 72: Predicted (in silico) Reactivity (Inclusivity) Results for KPC   

<table><tr><td rowspan=1 colspan=1>Target</td><td rowspan=1 colspan=1>Associated Organism</td><td rowspan=1 colspan=1>VariantDetected</td></tr><tr><td rowspan=17 colspan=1>b</td><td rowspan=17 colspan=1>b</td><td rowspan=1 colspan=1>KPC-5</td></tr><tr><td rowspan=1 colspan=1>KPC-6</td></tr><tr><td rowspan=1 colspan=1>KPC-7</td></tr><tr><td rowspan=1 colspan=1>KPC-8</td></tr><tr><td rowspan=1 colspan=1>KPC-11</td></tr><tr><td rowspan=1 colspan=1>KPC-12</td></tr><tr><td rowspan=1 colspan=1>KPC-14</td></tr><tr><td rowspan=1 colspan=1>KPC-15</td></tr><tr><td rowspan=1 colspan=1>KPC-16</td></tr><tr><td rowspan=1 colspan=1>KPC-17</td></tr><tr><td rowspan=1 colspan=1>KPC-19</td></tr><tr><td rowspan=1 colspan=1>KPC-22</td></tr><tr><td rowspan=1 colspan=1>KPC-24</td></tr><tr><td rowspan=1 colspan=1>KPC-25</td></tr><tr><td rowspan=1 colspan=1>KPC-26</td></tr><tr><td rowspan=1 colspan=1>KPC-27</td></tr><tr><td rowspan=1 colspan=1>KPC-30</td></tr><tr><td rowspan=2 colspan=1>Proteus mirabilis/Proteus</td><td rowspan=2 colspan=1>Proteus mirabilis</td><td rowspan=1 colspan=1>KPC-2</td></tr><tr><td rowspan=1 colspan=1>KPC-6*</td></tr><tr><td rowspan=2 colspan=1>Pseudomonas aeruginosa</td><td rowspan=2 colspan=1>Pseudomonasaeruginosa</td><td rowspan=1 colspan=1>KPC-2</td></tr><tr><td rowspan=1 colspan=1>KPC-5*</td></tr><tr><td rowspan=1 colspan=1>Salmonella</td><td rowspan=1 colspan=1>Salmonella enterica</td><td rowspan=1 colspan=1>KPC-2</td></tr><tr><td rowspan=2 colspan=1>Serratiamarcescens/Serratia</td><td rowspan=2 colspan=1>Serratia marcescens</td><td rowspan=1 colspan=1>KPC-2</td></tr><tr><td rowspan=1 colspan=1>KPC-3</td></tr></table>

Table 73: Predicted (in silico) Reactivity (Inclusivity) Results for VIM   

<table><tr><td rowspan=1 colspan=1>Target</td><td rowspan=1 colspan=1>AssociatedOrganism</td><td rowspan=1 colspan=1>VariantDetected</td></tr><tr><td rowspan=4 colspan=1>Acinetobacter baumannii</td><td rowspan=4 colspan=1>Acinetobacterbaumannii</td><td rowspan=1 colspan=1>VIM-1</td></tr><tr><td rowspan=1 colspan=1>VIM-2</td></tr><tr><td rowspan=1 colspan=1>VIM-6</td></tr><tr><td rowspan=1 colspan=1>VIM-11</td></tr><tr><td rowspan=4 colspan=1>Citrobacter</td><td rowspan=4 colspan=1>Citrobacter freundii</td><td rowspan=1 colspan=1>VIM-1</td></tr><tr><td rowspan=1 colspan=1>VIM-2</td></tr><tr><td rowspan=1 colspan=1>VIM-4</td></tr><tr><td rowspan=1 colspan=1>VIM-23</td></tr><tr><td rowspan=1 colspan=1>Enterobacter non-cloacae Complex</td><td rowspan=1 colspan=1>Enterobacteraerogenes</td><td rowspan=1 colspan=1>VIM-1</td></tr><tr><td rowspan=10 colspan=1>Enterobacter cloacaeComplex</td><td rowspan=6 colspan=1>Enterobacter cloacae</td><td rowspan=1 colspan=1>VIM-1</td></tr><tr><td rowspan=1 colspan=1>VIM-2</td></tr><tr><td rowspan=1 colspan=1>VIM-4</td></tr><tr><td rowspan=1 colspan=1>VIM-23</td></tr><tr><td rowspan=1 colspan=1>VIM-31</td></tr><tr><td rowspan=1 colspan=1>VIM-40</td></tr><tr><td rowspan=3 colspan=1>Enterobacterhormaechei</td><td rowspan=1 colspan=1>VIM-1</td></tr><tr><td rowspan=1 colspan=1>VIM-4</td></tr><tr><td rowspan=1 colspan=1>VIM-23</td></tr><tr><td rowspan=1 colspan=1>Enterobacterxiangfangensis</td><td rowspan=1 colspan=1>VIM-1</td></tr><tr><td rowspan=4 colspan=1>Escherichia coli</td><td rowspan=4 colspan=1>Escherichia coliA</td><td rowspan=1 colspan=1>VIM-1</td></tr><tr><td rowspan=1 colspan=1>VIM-2</td></tr><tr><td rowspan=1 colspan=1>VIM-19</td></tr><tr><td rowspan=1 colspan=1>VIM-29</td></tr><tr><td rowspan=5 colspan=1>Klebsiella oxytoca</td><td rowspan=5 colspan=1>Klebsiella oxytoca</td><td rowspan=1 colspan=1>VIM-1</td></tr><tr><td rowspan=1 colspan=1>VIM-2</td></tr><tr><td rowspan=1 colspan=1>VIM-4</td></tr><tr><td rowspan=1 colspan=1>VIM-32</td></tr><tr><td rowspan=1 colspan=1>VIM-35</td></tr><tr><td rowspan=10 colspan=1>Klebsiella pneumoniaeGroup</td><td rowspan=10 colspan=1>Klebsiella pneumoniae</td><td rowspan=1 colspan=1>VIM-1*</td></tr><tr><td rowspan=1 colspan=1>VIM-2</td></tr><tr><td rowspan=1 colspan=1>VIM-4</td></tr><tr><td rowspan=1 colspan=1>VIM-12</td></tr><tr><td rowspan=1 colspan=1>VIM-19</td></tr><tr><td rowspan=1 colspan=1>VIM-24</td></tr><tr><td rowspan=1 colspan=1>VIM-26</td></tr><tr><td rowspan=1 colspan=1>VIM-27*</td></tr><tr><td rowspan=1 colspan=1>VIM-33</td></tr><tr><td rowspan=1 colspan=1>VIM-34</td></tr></table>

A. Unspecified VIM variant detected in Analytical Reactivity (Inclusivity) study.   

<table><tr><td rowspan=1 colspan=1>Target</td><td rowspan=1 colspan=1>AssociatedOrganism</td><td rowspan=1 colspan=1>VariantDetected</td></tr><tr><td rowspan=4 colspan=1></td><td rowspan=4 colspan=1></td><td rowspan=1 colspan=1>VIM-39</td></tr><tr><td rowspan=1 colspan=1>VIM-42</td></tr><tr><td rowspan=1 colspan=1>VIM-51</td></tr><tr><td rowspan=1 colspan=1>VIM-52</td></tr><tr><td rowspan=1 colspan=1>Morganella morganii</td><td rowspan=1 colspan=1>Morganella morganii</td><td rowspan=1 colspan=1>VIM-4</td></tr><tr><td rowspan=1 colspan=1>Proteus mirabilis/Proteus</td><td rowspan=1 colspan=1>Proteus mirabilis</td><td rowspan=1 colspan=1>VIM-1</td></tr><tr><td rowspan=27 colspan=1>Pseudomonas aeruginosa</td><td rowspan=27 colspan=1>Pseudomonasaeruginosa</td><td rowspan=1 colspan=1>VIM-1</td></tr><tr><td rowspan=1 colspan=1>VIM-2*</td></tr><tr><td rowspan=1 colspan=1>VIM-3</td></tr><tr><td rowspan=1 colspan=1>VIM-4*</td></tr><tr><td rowspan=1 colspan=1>VIM-5</td></tr><tr><td rowspan=1 colspan=1>VIM-6</td></tr><tr><td rowspan=1 colspan=1>VIM-8</td></tr><tr><td rowspan=1 colspan=1>VIM-9</td></tr><tr><td rowspan=1 colspan=1>VIM-10</td></tr><tr><td rowspan=1 colspan=1>VIM-11</td></tr><tr><td rowspan=1 colspan=1>VIM-14</td></tr><tr><td rowspan=1 colspan=1>VIM-15</td></tr><tr><td rowspan=1 colspan=1>VIM-16</td></tr><tr><td rowspan=1 colspan=1>VIM-17</td></tr><tr><td rowspan=1 colspan=1>VIM-18</td></tr><tr><td rowspan=1 colspan=1>VIM-20</td></tr><tr><td rowspan=1 colspan=1>VIM-28</td></tr><tr><td rowspan=1 colspan=1>VIM-30</td></tr><tr><td rowspan=1 colspan=1>VIM-36</td></tr><tr><td rowspan=1 colspan=1>VIM-37</td></tr><tr><td rowspan=1 colspan=1>VIM-41</td></tr><tr><td rowspan=1 colspan=1>VIM-43</td></tr><tr><td rowspan=1 colspan=1>VIM-44</td></tr><tr><td rowspan=1 colspan=1>VIM-45</td></tr><tr><td rowspan=1 colspan=1>VIM-46</td></tr><tr><td rowspan=1 colspan=1>VIM-48</td></tr><tr><td rowspan=1 colspan=1>VIM-50</td></tr><tr><td rowspan=1 colspan=1>Salmonella</td><td rowspan=1 colspan=1>Salmonella enterica</td><td rowspan=1 colspan=1>VIM-1</td></tr><tr><td rowspan=3 colspan=1>Serratiamarcescens/Serratia</td><td rowspan=3 colspan=1>Serratia marcescens</td><td rowspan=1 colspan=1>VIM-2</td></tr><tr><td rowspan=1 colspan=1>VIM-4</td></tr><tr><td rowspan=1 colspan=1>VIM-54</td></tr><tr><td rowspan=1 colspan=1>Stenotrophomonasmaltophilia</td><td rowspan=1 colspan=1>Stenotrophomonasmaltophilia</td><td rowspan=1 colspan=1>VIM-2</td></tr></table>

Table 74: Predicted (in silico) Reactivity (Inclusivity) Results for OXA-23   

<table><tr><td rowspan=1 colspan=1>Target</td><td rowspan=1 colspan=1>AssociatedOrganism</td><td rowspan=1 colspan=1>VariantDetected</td></tr><tr><td rowspan=18 colspan=1>Acinetobacterbaumannii</td><td rowspan=9 colspan=1>Acinetobacter</td><td rowspan=1 colspan=1>OXA-23*</td></tr><tr><td rowspan=1 colspan=1>OXA-49</td></tr><tr><td rowspan=1 colspan=1>OXA-23/OXA-104</td></tr><tr><td rowspan=1 colspan=1>OXA-23/OXA-64</td></tr><tr><td rowspan=1 colspan=1>OXA-23/OXA-66</td></tr><tr><td rowspan=1 colspan=1>OXA-23/OXA-69</td></tr><tr><td rowspan=1 colspan=1>OXA-27</td></tr><tr><td rowspan=1 colspan=1>OXA-65/OXA-239</td></tr><tr><td rowspan=1 colspan=1>OXA-68</td></tr><tr><td rowspan=9 colspan=1>baumannii</td><td rowspan=1 colspan=1>OXA-146</td></tr><tr><td rowspan=1 colspan=1>OXA-165</td></tr><tr><td rowspan=1 colspan=1>OXA-166</td></tr><tr><td rowspan=1 colspan=1>OXA-167</td></tr><tr><td rowspan=1 colspan=1>OXA-168</td></tr><tr><td rowspan=1 colspan=1>OXA-169</td></tr><tr><td rowspan=1 colspan=1>OXA-170</td></tr><tr><td rowspan=1 colspan=1>OXA-171</td></tr><tr><td rowspan=1 colspan=1>OXA-183</td></tr></table>

<table><tr><td rowspan=1 colspan=1>Target</td><td rowspan=1 colspan=1>AssociatedOrganism</td><td rowspan=1 colspan=1>VariantDetected</td></tr><tr><td rowspan=12 colspan=1></td><td rowspan=12 colspan=1>b</td><td rowspan=1 colspan=1>OXA-225</td></tr><tr><td rowspan=1 colspan=1>OXA-366</td></tr><tr><td rowspan=1 colspan=1>OXA-398</td></tr><tr><td rowspan=1 colspan=1>OXA-422</td></tr><tr><td rowspan=1 colspan=1>OXA-423</td></tr><tr><td rowspan=1 colspan=1>OXA-435</td></tr><tr><td rowspan=1 colspan=1>OXA-440</td></tr><tr><td rowspan=1 colspan=1>OXA-469</td></tr><tr><td rowspan=1 colspan=1>OXA-481</td></tr><tr><td rowspan=1 colspan=1>OXA-482</td></tr><tr><td rowspan=1 colspan=1>OXA-483</td></tr><tr><td rowspan=1 colspan=1>OXA-565</td></tr><tr><td rowspan=1 colspan=1>Escherichia coli</td><td rowspan=1 colspan=1>Escherichia coli</td><td rowspan=1 colspan=1>OXA-23</td></tr><tr><td rowspan=1 colspan=1>Klebsiellapneumoniae Group</td><td rowspan=1 colspan=1>Klebsiellapneumoniae</td><td rowspan=1 colspan=1>OXA-73</td></tr><tr><td rowspan=1 colspan=1>Proteus mirabilis</td><td rowspan=1 colspan=1>Proteusmirabilis</td><td rowspan=1 colspan=1>OXA-23</td></tr></table>

Table 75: Predicted (in silico) Reactivity (Inclusivity) Results for OXA-48   

<table><tr><td rowspan=1 colspan=1>Target</td><td rowspan=1 colspan=1>AssociatedOrganism</td><td rowspan=1 colspan=1>VariantDetected</td></tr><tr><td rowspan=1 colspan=1>Acinetobacter baumannii</td><td rowspan=1 colspan=1>Acinetobacterbaumannii</td><td rowspan=1 colspan=1>OXA-48</td></tr><tr><td rowspan=2 colspan=1>Citrobacter</td><td rowspan=2 colspan=1>Citrobacterfreundii</td><td rowspan=1 colspan=1>OXA-48</td></tr><tr><td rowspan=1 colspan=1>OXA-181</td></tr><tr><td rowspan=1 colspan=1>Enterobacter non-cloacae Complex</td><td rowspan=1 colspan=1>Enterobacteraerogenes</td><td rowspan=1 colspan=1>OXA-244</td></tr><tr><td rowspan=5 colspan=1>Enterobacter cloacaeComplex</td><td rowspan=3 colspan=1>Enterobactercloacae</td><td rowspan=1 colspan=1>OXA-48</td></tr><tr><td rowspan=1 colspan=1>OXA-163</td></tr><tr><td rowspan=1 colspan=1>OXA-181</td></tr><tr><td rowspan=1 colspan=1>Enterobacterhormaechei</td><td rowspan=1 colspan=1>OXA-370</td></tr><tr><td rowspan=1 colspan=1>Enterobacterludwigii</td><td rowspan=1 colspan=1>OXA-48</td></tr><tr><td rowspan=9 colspan=1>Escherichia coli</td><td rowspan=9 colspan=1>Escherichia coliA</td><td rowspan=1 colspan=1>OXA-48</td></tr><tr><td rowspan=1 colspan=1>OXA-163</td></tr><tr><td rowspan=1 colspan=1>OXA-181</td></tr><tr><td rowspan=1 colspan=1>OXA-204</td></tr><tr><td rowspan=1 colspan=1>OXA-232</td></tr><tr><td rowspan=1 colspan=1>OXA-244</td></tr><tr><td rowspan=1 colspan=1>OXA-438</td></tr><tr><td rowspan=1 colspan=1>OXA-439</td></tr><tr><td rowspan=1 colspan=1>OXA-566</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>OXA-1/OXA-48</td></tr></table>

A. Unspecified OXA variant detected in the Analytical Reactivity (Inclusivity) study.

<table><tr><td rowspan=1 colspan=1>Target</td><td rowspan=1 colspan=1>AssociatedOrganism</td><td rowspan=1 colspan=1>VariantDetected</td></tr><tr><td rowspan=14 colspan=1>Klebsiella pneumoniaeGroup</td><td rowspan=13 colspan=1>Klebsiellapneumoniae</td><td rowspan=1 colspan=1>OXA-48*</td></tr><tr><td rowspan=1 colspan=1>OXA-10</td></tr><tr><td rowspan=1 colspan=1>OXA-162</td></tr><tr><td rowspan=1 colspan=1>OXA-181*</td></tr><tr><td rowspan=1 colspan=1>OXA-204</td></tr><tr><td rowspan=1 colspan=1>OXA-232*</td></tr><tr><td rowspan=1 colspan=1>OXA-244</td></tr><tr><td rowspan=1 colspan=1>OXA-245</td></tr><tr><td rowspan=1 colspan=1>OXA-247</td></tr><tr><td rowspan=1 colspan=1>OXA-484</td></tr><tr><td rowspan=1 colspan=1>OXA-505</td></tr><tr><td rowspan=1 colspan=1>OXA-517</td></tr><tr><td rowspan=1 colspan=1>OXA-519</td></tr><tr><td rowspan=1 colspan=1>Klebsiellavariicola</td><td rowspan=1 colspan=1>OXA-181</td></tr><tr><td rowspan=1 colspan=1>Morganella morganii</td><td rowspan=1 colspan=1>Morganellamorganii</td><td rowspan=1 colspan=1>OXA-181</td></tr><tr><td rowspan=2 colspan=1>Proteus mirabilis/Proteus</td><td rowspan=2 colspan=1>Proteus mirabilis</td><td rowspan=1 colspan=1>OXA-48</td></tr><tr><td rowspan=1 colspan=1>OXA-244</td></tr><tr><td rowspan=2 colspan=1>Serratiamarcescens/Serratia</td><td rowspan=2 colspan=1>Serratiamarcescens</td><td rowspan=1 colspan=1>OXA-48</td></tr><tr><td rowspan=1 colspan=1>OXA-405</td></tr></table>

Table 76: Predicted (in silico) Reactivity (Inclusivity) Results for NDM   

<table><tr><td rowspan=1 colspan=1>Target</td><td rowspan=1 colspan=1>AssociatedOrganism</td><td rowspan=1 colspan=1>VariantDetected</td></tr><tr><td rowspan=2 colspan=1>Acinetobacterbaumannii</td><td rowspan=2 colspan=1>Acinetobacterbaumannii</td><td rowspan=1 colspan=1>NDM-1*</td></tr><tr><td rowspan=1 colspan=1>NDM-2</td></tr><tr><td rowspan=1 colspan=1>Citrobacter</td><td rowspan=1 colspan=1>Citrobacter braakii</td><td rowspan=1 colspan=1>NDM-4</td></tr><tr><td rowspan=4 colspan=1>Citrobacter</td><td rowspan=4 colspan=1>Citrobacter freundii</td><td rowspan=1 colspan=1>NDM-1A</td></tr><tr><td rowspan=1 colspan=1>NDM-4</td></tr><tr><td rowspan=1 colspan=1>NDM-6</td></tr><tr><td rowspan=1 colspan=1>NDM-7</td></tr><tr><td rowspan=4 colspan=1>Enterobacter non-cloacae Complex</td><td rowspan=4 colspan=1>Enterobacteraerogenes</td><td rowspan=1 colspan=1>NDM-1</td></tr><tr><td rowspan=1 colspan=1>NDM-4</td></tr><tr><td rowspan=1 colspan=1>NDM-5</td></tr><tr><td rowspan=1 colspan=1>NDM-7</td></tr><tr><td rowspan=5 colspan=1>Enterobacter cloacaeComplex</td><td rowspan=3 colspan=1>EnterobactercloacaeB</td><td rowspan=1 colspan=1>NDM-1</td></tr><tr><td rowspan=1 colspan=1>NDM-4</td></tr><tr><td rowspan=1 colspan=1>NDM-7</td></tr><tr><td rowspan=1 colspan=1>Enterobacterhormaechei</td><td rowspan=1 colspan=1>NDM-1</td></tr><tr><td rowspan=1 colspan=1>Enterobacterludwigii</td><td rowspan=1 colspan=1>NDM-1</td></tr><tr><td rowspan=10 colspan=1>Escherichia coli</td><td rowspan=10 colspan=1>Escherichia coli</td><td rowspan=1 colspan=1>NDM-1</td></tr><tr><td rowspan=1 colspan=1>NDM-3</td></tr><tr><td rowspan=1 colspan=1>NDM-4</td></tr><tr><td rowspan=1 colspan=1>NDM-5*</td></tr><tr><td rowspan=1 colspan=1>NDM-6*</td></tr><tr><td rowspan=1 colspan=1>NDM-7*</td></tr><tr><td rowspan=1 colspan=1>NDM-8</td></tr><tr><td rowspan=1 colspan=1>NDM-11</td></tr><tr><td rowspan=1 colspan=1>NDM-12</td></tr><tr><td rowspan=1 colspan=1>NDM-13</td></tr></table>

A. Detected in a Citrobacter species in the Analytical Reactivity (Inclusivity) study. B. Unspecified NDM variant detected in the Analytical Reactivity (Inclusivity) study. C. NDM-1 was detected in Morganella morganii in the Analytical Reactivity (Inclusivity) study but no sequences were available for in silico analysis.

<table><tr><td rowspan=1 colspan=1>Target</td><td rowspan=1 colspan=1>AssociatedOrganism</td><td rowspan=1 colspan=1>VariantDetected</td></tr><tr><td rowspan=5 colspan=1></td><td rowspan=5 colspan=1></td><td rowspan=1 colspan=1>NDM-15</td></tr><tr><td rowspan=1 colspan=1>NDM-16</td></tr><tr><td rowspan=1 colspan=1>NDM-17</td></tr><tr><td rowspan=1 colspan=1>NDM-18</td></tr><tr><td rowspan=1 colspan=1>NDM-19</td></tr><tr><td rowspan=2 colspan=1>Klebsiella oxytoca</td><td rowspan=2 colspan=1>Klebsiella oxytoca</td><td rowspan=1 colspan=1>NDM-1</td></tr><tr><td rowspan=1 colspan=1>NDM-4</td></tr><tr><td rowspan=7 colspan=1>Klebsiellapneumoniae Group</td><td rowspan=7 colspan=1>Klebsiellapneumoniae</td><td rowspan=1 colspan=1>NDM-1*</td></tr><tr><td rowspan=1 colspan=1>NDM-4</td></tr><tr><td rowspan=1 colspan=1>NDM-5</td></tr><tr><td rowspan=1 colspan=1>NDM-6</td></tr><tr><td rowspan=1 colspan=1>NDM-7</td></tr><tr><td rowspan=1 colspan=1>NDM-10</td></tr><tr><td rowspan=1 colspan=1>NDM-16</td></tr><tr><td rowspan=1 colspan=1>Morganella morganii</td><td rowspan=1 colspan=1>Morganellamorganii</td><td rowspan=1 colspan=1>NDM-1c</td></tr><tr><td rowspan=1 colspan=1>Proteusmirabilis/Proteus</td><td rowspan=1 colspan=1>Proteus mirabilis</td><td rowspan=1 colspan=1>NDM-1</td></tr><tr><td rowspan=2 colspan=1>Pseudomonasaeruginosa</td><td rowspan=2 colspan=1>Pseudomonasaeruginosa</td><td rowspan=1 colspan=1>NDM-1</td></tr><tr><td rowspan=1 colspan=1>NDM-5</td></tr><tr><td rowspan=2 colspan=1>Salmonella</td><td rowspan=2 colspan=1>Salmonella enterica</td><td rowspan=1 colspan=1>NDM-1</td></tr><tr><td rowspan=1 colspan=1>NDM-5</td></tr><tr><td rowspan=1 colspan=1>Serratiamarcescens/Serratia</td><td rowspan=1 colspan=1>Serratia marcescens</td><td rowspan=1 colspan=1>NDM-1</td></tr><tr><td rowspan=1 colspan=1>Stenotrophomonasmaltophilia</td><td rowspan=1 colspan=1>Stenotrophomonasmaltophilia</td><td rowspan=1 colspan=1>NDM-1</td></tr></table>

Table 77: Predicted (in silico) Reactivity (Inclusivity) Results for Variants Not Detected   

<table><tr><td rowspan=1 colspan=1>ResistanceMarker</td><td rowspan=1 colspan=1>Variant NotDetected</td><td rowspan=1 colspan=1>AssociatedOrganism</td><td rowspan=1 colspan=1>No. ofSequences</td></tr><tr><td rowspan=3 colspan=1>CTX-M-1</td><td rowspan=1 colspan=1>CTX-M-80</td><td rowspan=3 colspan=1>Klebsiellapneumoniae</td><td rowspan=3 colspan=1>3</td></tr><tr><td rowspan=1 colspan=1>CTX-M-15</td></tr><tr><td rowspan=1 colspan=1>Not Specified</td></tr><tr><td rowspan=3 colspan=1>IMP</td><td rowspan=1 colspan=1>IMP-31</td><td rowspan=3 colspan=1>Pseudomonasaeruginosa</td><td rowspan=1 colspan=1>2</td></tr><tr><td rowspan=1 colspan=1>IMP-35</td><td rowspan=1 colspan=1>2</td></tr><tr><td rowspan=1 colspan=1>IMP-7</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=12 colspan=1>NDM</td><td rowspan=3 colspan=1>NDM-1</td><td rowspan=1 colspan=1>Escherichia coli</td><td rowspan=1 colspan=1>6</td></tr><tr><td rowspan=1 colspan=1>Klebsiellavariicola</td><td rowspan=1 colspan=1>3</td></tr><tr><td rowspan=1 colspan=1>Salmonellaenterica</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>NDM-3</td><td rowspan=1 colspan=1>Acinetobacterbaumannii</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>NDM-4</td><td rowspan=1 colspan=1>Escherichia coli</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=3 colspan=1>NDM-9</td><td rowspan=1 colspan=1>Cronobactersakazakii</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>Escherichia coli</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>Klebsiellapneumoniae</td><td rowspan=1 colspan=1>2</td></tr><tr><td rowspan=4 colspan=1>Not specified</td><td rowspan=1 colspan=1>Escherichia coli</td><td rowspan=1 colspan=1>2</td></tr><tr><td rowspan=1 colspan=1>Klebsiellapneumoniae</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>Klebsiella sp</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>Pseudomonasaeruginosa</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=9 colspan=1>VIM</td><td rowspan=2 colspan=1>VIM-1</td><td rowspan=1 colspan=1>Pseudomonasaeruginosa</td><td rowspan=1 colspan=1>3</td></tr><tr><td rowspan=1 colspan=1>Providenciavermicola</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>VIM-2</td><td rowspan=1 colspan=1>Klebsiellapneumoniae</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=2 colspan=1>VIM-5</td><td rowspan=1 colspan=1>Enterobactercloacae</td><td rowspan=1 colspan=1>2</td></tr><tr><td rowspan=1 colspan=1>Klebsiellapneumoniae</td><td rowspan=1 colspan=1>3</td></tr><tr><td rowspan=1 colspan=1>VIM-7</td><td rowspan=2 colspan=1>Pseudomonasaeruginosa</td><td rowspan=1 colspan=1>4</td></tr><tr><td rowspan=1 colspan=1>VIM-13</td><td rowspan=1 colspan=1>3</td></tr><tr><td rowspan=2 colspan=1>VIM-25</td><td rowspan=1 colspan=1>Acinetobacterbaumannii</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>Proteus mirabilis</td><td rowspan=1 colspan=1>2</td></tr></table>

<table><tr><td rowspan=1 colspan=1>ResistanceMarker</td><td rowspan=1 colspan=1>Variant NotDetected</td><td rowspan=1 colspan=1>AssociatedOrganism</td><td rowspan=1 colspan=1>No. ofSequences</td></tr><tr><td rowspan=4 colspan=1>−</td><td rowspan=1 colspan=1>VIM-38</td><td rowspan=4 colspan=1>Pseudomonasaeruginosa</td><td rowspan=1 colspan=1>2</td></tr><tr><td rowspan=1 colspan=1>VIM-47</td><td rowspan=1 colspan=1>2</td></tr><tr><td rowspan=1 colspan=1>VIM-49</td><td rowspan=1 colspan=1>2</td></tr><tr><td rowspan=1 colspan=1>Not specified</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>OXA-48</td><td rowspan=1 colspan=1>OXA-232</td><td rowspan=1 colspan=1>Escherichia coli</td><td rowspan=1 colspan=1>1</td></tr></table>

# Analytical Specificity (Cross-Reactivity and Exclusivity)

Cross-reactivity of on-panel and off-panel analytes was evaluated with the BCID-GN Panel. Bacterial targets were tested in triplicate at a concentration of ${ \sim } 1 \mathrm { x } 1 0 ^ { 9 } \mathrm { C F U / m L }$ while fungi were tested in triplicate at a concentration of ${ \sim } 1 \mathrm { x } 1 0 ^ { 7 } \mathrm { C F U / m L }$ . If the target concentration could not be reached, the organism was diluted 2-fold from stock for use (indicated with an asterisk in Tables 78-81).

No cross reactivity was observed for any of the on-panel organisms. The following off-panel organisms showed cross reactivity: Acinetobacter anitratus (at a concentration of ${ \mathrm { > } } 1 \mathrm { x } 1 0 ^ { 4 }$ $\mathrm { C F U / m L }$ ) cross-reacts with the Acinetobacter baumannii assay, Enterobacter cowanii (at a concentration of $\mathrm { \ddot { \ s l } x { l } 0 ^ { 8 } C F U / m L }$ ) cross-reacts with the Enterobacter cloacae complex assay, Escherichia hermanii cross-reacts with the Enterobacter (non-cloacae complex) assay (at a concentration of ${ > } 1 \mathrm { x } 1 0 ^ { 6 } \mathrm { C F U / m L }$ ) and with the Serratia assay (at a concentration of ${ \bf \tau } > 1 \mathbf { x } 1 0 ^ { 7 }$ $\mathrm { C F U / m L }$ ), Fusobacterium periodonticum (at a concentration of $5 \mathrm { x } 1 0 ^ { 8 } \mathrm { C F U / m L }$ ) and Fusobacterium simiae (at a concentration of $2 . 9 { \mathrm { x } } 1 0 ^ { 8 } \mathrm { C F U / m L }$ ) cross-react with the Fusobacterium nucleatum assay, and Shigella (at a concentration of $1 \mathrm { x } 1 0 ^ { 9 } \mathrm { C F U / m L }$ ) cross-reacts with the Escherichia coli assay (off-panel organisms showing cross-reactivity are bolded in the tables below). See Table 56 for a summary of the on-panel strains tested and Tables 78-81 for a summary of off-panel strains tested.

Additional in silico analysis was performed to identify any off-panel gram-negative and grampositive organisms that may cross-react with the BCID-GN Panel (Tables 82-83).

Note: the performance of the ePlex BCID-GN Panel has not been established for organisms evaluated by in silico analysis alone.

# Off-Panel Exclusivity

Table 78: Off-Panel Gram-Negative Organisms Assessed for Cross-reactivity with the ePlex BCID-GN Panel (Exclusivity)   

<table><tr><td rowspan=1 colspan=1>Gram Negative Organism</td><td rowspan=1 colspan=1>Strain ID</td></tr><tr><td rowspan=1 colspan=1>Acinetobacter haemolyticus</td><td rowspan=1 colspan=1>ATCC 19002</td></tr><tr><td rowspan=1 colspan=1>Acinetobacter lwoffi</td><td rowspan=1 colspan=1>ATCC 15309</td></tr><tr><td rowspan=1 colspan=1>Acinetobacter junii</td><td rowspan=1 colspan=1>ATCC 17908</td></tr><tr><td rowspan=1 colspan=1>Acinetobacter anitratus</td><td rowspan=1 colspan=1>ATCC 49139</td></tr><tr><td rowspan=1 colspan=1>Aeromonas hydrophila</td><td rowspan=1 colspan=1>JMI 938982</td></tr><tr><td rowspan=1 colspan=1>Aeromonas salmonicida</td><td rowspan=1 colspan=1>ATCC 33658</td></tr><tr><td rowspan=1 colspan=1>Aeromonas sobria</td><td rowspan=1 colspan=1>ATCC 35993</td></tr><tr><td rowspan=1 colspan=1>Bacteroides distasonis (Parabacteroides)</td><td rowspan=1 colspan=1>ATCC 8503</td></tr><tr><td rowspan=1 colspan=1>Bacteroides merdae</td><td rowspan=1 colspan=1>ATCC 43184</td></tr><tr><td rowspan=1 colspan=1>Bacteroides thetaiotaomicron</td><td rowspan=1 colspan=1>ATCC 29741</td></tr><tr><td rowspan=1 colspan=1>Bacteroides vulgatus*</td><td rowspan=1 colspan=1>ATCC 8482</td></tr><tr><td rowspan=1 colspan=1>Bacteroides caccae</td><td rowspan=1 colspan=1>ATCC 700189</td></tr><tr><td rowspan=1 colspan=1>Bacteroides eggerthii</td><td rowspan=1 colspan=1>ATCC 27754</td></tr><tr><td rowspan=1 colspan=1>Bacteroides ovatus*</td><td rowspan=1 colspan=1>ATCC BAA-1296</td></tr><tr><td rowspan=1 colspan=1>Bacteroides ureolyticus*</td><td rowspan=1 colspan=1>ATCC 33387</td></tr><tr><td rowspan=1 colspan=1>Bordetella pertussis</td><td rowspan=1 colspan=1>ATCC 9797</td></tr><tr><td rowspan=1 colspan=1>Burkholderia cepacia</td><td rowspan=1 colspan=1>ATCC 25416</td></tr><tr><td rowspan=1 colspan=1>Citrobacter amalonaticus</td><td rowspan=1 colspan=1>ATCC BAA-2563</td></tr><tr><td rowspan=1 colspan=1>Citrobacter gillenii</td><td rowspan=1 colspan=1>ATCC 51640</td></tr><tr><td rowspan=1 colspan=1>Citrobacter sedlakii</td><td rowspan=1 colspan=1>ATCC 51493</td></tr><tr><td rowspan=1 colspan=1>Citrobacter farmer</td><td rowspan=1 colspan=1>ATCC 51112</td></tr><tr><td rowspan=1 colspan=1>Citrobacter murliniae</td><td rowspan=1 colspan=1>ATCC 51642</td></tr><tr><td rowspan=1 colspan=1>Edwardsiella tarda</td><td rowspan=1 colspan=1>ATCC 15947</td></tr><tr><td rowspan=1 colspan=1>Enterobacter kobei</td><td rowspan=1 colspan=1>ATCC BAA-260</td></tr><tr><td rowspan=1 colspan=1>Enterobacter cancerogenus</td><td rowspan=1 colspan=1>ATCC 35315</td></tr><tr><td rowspan=1 colspan=1>Enterobacter cowaniiB</td><td rowspan=1 colspan=1>DSM-18146</td></tr><tr><td rowspan=1 colspan=1>Escherichia albertii</td><td rowspan=1 colspan=1>DSM-17582</td></tr><tr><td rowspan=1 colspan=1>Escherichia fergusonii</td><td rowspan=1 colspan=1>ATCC 35469</td></tr><tr><td rowspan=1 colspan=1>Escherichia hermaniic</td><td rowspan=1 colspan=1>ATCC 33650</td></tr><tr><td rowspan=1 colspan=1>Ewingella americana*</td><td rowspan=1 colspan=1>ATCC 33853</td></tr><tr><td rowspan=1 colspan=1>Eikenella corrodens</td><td rowspan=1 colspan=1>ATCC BAA-1152</td></tr><tr><td rowspan=1 colspan=1>Fusobacterium naviforme*</td><td rowspan=1 colspan=1>ATCC 25832</td></tr><tr><td rowspan=1 colspan=1>Fusobacterium gonidiaformans</td><td rowspan=1 colspan=1>ATCC 25563</td></tr><tr><td rowspan=1 colspan=1>Fusobacterium necrogenes*</td><td rowspan=1 colspan=1>ATCC 25556</td></tr><tr><td rowspan=1 colspan=1>Fusobacterium periodonticum*D</td><td rowspan=1 colspan=1>ATCC 33693</td></tr><tr><td rowspan=1 colspan=1>Fusobacterium simiae*D</td><td rowspan=1 colspan=1>ATCC 33568</td></tr><tr><td rowspan=1 colspan=1>Fusobacterium varium</td><td rowspan=1 colspan=1>ATCC 27725</td></tr><tr><td rowspan=1 colspan=1>Fusobacterium russii*</td><td rowspan=1 colspan=1>ATCC 25533</td></tr><tr><td rowspan=1 colspan=1>Fusobacterium ulcerans</td><td rowspan=1 colspan=1>ATCC 49186</td></tr><tr><td rowspan=1 colspan=1>Haemophilus haemolyticus</td><td rowspan=1 colspan=1>ATCC 33390</td></tr><tr><td rowspan=1 colspan=1>Haemophilus parahaemolyticus</td><td rowspan=1 colspan=1>ATCC 10014</td></tr><tr><td rowspan=1 colspan=1>Hafnia alvei</td><td rowspan=1 colspan=1>ATCC 51815</td></tr><tr><td rowspan=1 colspan=1>Kingella kingae*</td><td rowspan=1 colspan=1>ATCC 23331</td></tr></table>

A. Cross-reactivity seen with Acinetobacter baumanii at a concentration $> 1 \mathrm { x } 1 0 ^ { 4 } \mathrm { C F U / m L }$ . B. Cross-reactivity seen with Enterobacter cloacae complex at a concentration $> 1 \mathrm { x } 1 0 ^ { 8 } \mathrm { C F U / m L }$ . C. Cross-reactivity seen with Enterobacter (non-cloacae complex) at a concentration $> 1 \mathrm { x } 1 0 ^ { 6 } \mathrm { C F U / m L }$ and Serratia at a concentration of $> 1 \mathrm { x } 1 0 ^ { 7 }$ CFU/mL. D. Cross-reactivity seen with the Fusobacterium nucleatum assay. E. Cross-reactivity seen with the Escherichia coli assay.

<table><tr><td rowspan=1 colspan=1>Gram Negative Organism</td><td rowspan=1 colspan=1>Strain ID</td></tr><tr><td rowspan=1 colspan=1>Kluyvera cochleae</td><td rowspan=1 colspan=1>ATCC 51609</td></tr><tr><td rowspan=1 colspan=1>Legionella pneumoniae</td><td rowspan=1 colspan=1>ATCC 33823</td></tr><tr><td rowspan=1 colspan=1>Leclercia adecarboxylata</td><td rowspan=1 colspan=1>ATCC 700325</td></tr><tr><td rowspan=1 colspan=1>Methylobacterium mesophilicum*</td><td rowspan=1 colspan=1>ATCC 29983</td></tr><tr><td rowspan=1 colspan=1>Neisseria gonorrhoeae</td><td rowspan=1 colspan=1>ATCC 19424</td></tr><tr><td rowspan=1 colspan=1>Neisseria mucosa</td><td rowspan=1 colspan=1>ATCC 19695</td></tr><tr><td rowspan=1 colspan=1>Neisseria sicca</td><td rowspan=1 colspan=1>ATCC 29193</td></tr><tr><td rowspan=1 colspan=1>Neisseria flavecens</td><td rowspan=1 colspan=1>ATCC 13115</td></tr><tr><td rowspan=1 colspan=1>Neisseria lactamica</td><td rowspan=1 colspan=1>ATCC 23970</td></tr><tr><td rowspan=1 colspan=1>Neisseria perflava</td><td rowspan=1 colspan=1>ATCC 14799</td></tr><tr><td rowspan=1 colspan=1>Ochrobactrum anthropi</td><td rowspan=1 colspan=1>ATCC BAA-749</td></tr><tr><td rowspan=1 colspan=1>Pantoea agglomerans</td><td rowspan=1 colspan=1>ATCC 14537</td></tr><tr><td rowspan=1 colspan=1>Pantoea ananatis</td><td rowspan=1 colspan=1>NRRL B-41502</td></tr><tr><td rowspan=1 colspan=1>Pasteurella aerogenes</td><td rowspan=1 colspan=1>ATCC 27883</td></tr><tr><td rowspan=1 colspan=1>Pasteurella multicida subsp multicide</td><td rowspan=1 colspan=1>ATCC 12945</td></tr><tr><td rowspan=1 colspan=1>Prevotella intermedia</td><td rowspan=1 colspan=1>ATCC 15032</td></tr><tr><td rowspan=1 colspan=1>Prevotella corporis*</td><td rowspan=1 colspan=1>ATCC 33547</td></tr><tr><td rowspan=1 colspan=1>Prevotella oralis*</td><td rowspan=1 colspan=1>ATCC 33269</td></tr><tr><td rowspan=1 colspan=1>Prevotella nigrescens*</td><td rowspan=1 colspan=1>ATCC 33563</td></tr><tr><td rowspan=1 colspan=1>Providencia rettgeri</td><td rowspan=1 colspan=1>ATCC 9250</td></tr><tr><td rowspan=1 colspan=1>Providencia stuartii</td><td rowspan=1 colspan=1>ATCC 33672</td></tr><tr><td rowspan=1 colspan=1>Providencia alcalifaciens</td><td rowspan=1 colspan=1>ATCC 9886</td></tr><tr><td rowspan=1 colspan=1>Pseudomonas fluorescens</td><td rowspan=1 colspan=1>ATCC 13525</td></tr><tr><td rowspan=1 colspan=1>Pseudomonas putida</td><td rowspan=1 colspan=1>ATCC 49128</td></tr><tr><td rowspan=1 colspan=1>Pseudomonas alcaligenes</td><td rowspan=1 colspan=1>ATCC 14909</td></tr><tr><td rowspan=1 colspan=1>Ralstonia insidiosa</td><td rowspan=1 colspan=1>ATCC 49129</td></tr><tr><td rowspan=1 colspan=1>Ralstonia pickettii</td><td rowspan=1 colspan=1>ATCC 27511</td></tr><tr><td rowspan=1 colspan=1>Raoultella planticola (Klebsiella planticola)</td><td rowspan=1 colspan=1>ATCC 31900</td></tr><tr><td rowspan=1 colspan=1>Raoultella ornithinolytica</td><td rowspan=1 colspan=1>CDC# 0134</td></tr><tr><td rowspan=1 colspan=1>Raoultella terrigena (Klebsiella terrigena)</td><td rowspan=1 colspan=1>ATCC 55553</td></tr><tr><td rowspan=1 colspan=1>Shigella boydiiE</td><td rowspan=1 colspan=1>ATCC 9207</td></tr><tr><td rowspan=1 colspan=1>Shigella sonnei</td><td rowspan=1 colspan=1>ATCC 25931</td></tr><tr><td rowspan=1 colspan=1>Shigella flexneriE</td><td rowspan=1 colspan=1>ATCC 9199</td></tr><tr><td rowspan=1 colspan=1>Vibrio furnissii</td><td rowspan=1 colspan=1>NCTC11218</td></tr><tr><td rowspan=1 colspan=1>Vibrio alginolyticus</td><td rowspan=1 colspan=1>ATCC 17749</td></tr><tr><td rowspan=1 colspan=1>Vibrio parahaemolyticus</td><td rowspan=1 colspan=1>ATCC 17802</td></tr><tr><td rowspan=1 colspan=1>Yersinia enterocolitica subsp enterocolitica</td><td rowspan=1 colspan=1>ATCC 9610</td></tr><tr><td rowspan=1 colspan=1>Yersinia ruckeri</td><td rowspan=1 colspan=1>ATCC 29473</td></tr><tr><td rowspan=1 colspan=1>Yersinia kristensenii</td><td rowspan=1 colspan=1>ATCC 33639</td></tr></table>

# Table 79: Off-Panel Gram-Positive Organisms Assessed for Cross-reactivity with the ePlex BCID-GN Panel (Exclusivity)

<table><tr><td rowspan=1 colspan=1>Gram Positive Organisms</td><td rowspan=1 colspan=1>Strain ID</td></tr><tr><td rowspan=1 colspan=1>Actinomyces odontolyticus</td><td rowspan=1 colspan=1>ATCC 17929</td></tr><tr><td rowspan=1 colspan=1>Clostridium perfringens</td><td rowspan=1 colspan=1>ATCC 13124</td></tr><tr><td rowspan=1 colspan=1>Corynebacterium jeikeium</td><td rowspan=1 colspan=1>ATCC BAA-949</td></tr><tr><td rowspan=1 colspan=1>Corynebacterium renale</td><td rowspan=1 colspan=1>ATCC 19412</td></tr><tr><td rowspan=1 colspan=1>Corynebacterium ulcerans</td><td rowspan=1 colspan=1>ATCC 51799</td></tr><tr><td rowspan=1 colspan=1>Corynebacterium xerosis*</td><td rowspan=1 colspan=1>ATCC 373</td></tr><tr><td rowspan=1 colspan=1>Corynebacterium durum</td><td rowspan=1 colspan=1>ATCC 33449</td></tr><tr><td rowspan=1 colspan=1>Corynebacteriumdiphtheriae*</td><td rowspan=1 colspan=1>ATCC 13812</td></tr><tr><td rowspan=1 colspan=1>Corynebacteriumpseudodiphtheriticum</td><td rowspan=1 colspan=1>ATCC 10700</td></tr><tr><td rowspan=1 colspan=1>Corynebacterium striatum*</td><td rowspan=1 colspan=1>ATCC 43735</td></tr><tr><td rowspan=1 colspan=1>Corynebacterium urealyticum</td><td rowspan=1 colspan=1>ATCC 43044</td></tr><tr><td rowspan=1 colspan=1>Lactobacillus casei</td><td rowspan=1 colspan=1>ATCC 39392</td></tr><tr><td rowspan=1 colspan=1>Lactobacillus paracasei*</td><td rowspan=1 colspan=1>ATCC 25598</td></tr><tr><td rowspan=1 colspan=1>Lactobacillus acidophilus*</td><td rowspan=1 colspan=1>ATCC 314</td></tr><tr><td rowspan=1 colspan=1>Lactobacillus crispatus</td><td rowspan=1 colspan=1>ATCC 33197</td></tr><tr><td rowspan=1 colspan=1>Lactobacillus rhamnosus</td><td rowspan=1 colspan=1>ATCC 39595</td></tr><tr><td rowspan=1 colspan=1>Lactococcus lactis</td><td rowspan=1 colspan=1>ATCC 49032</td></tr><tr><td rowspan=1 colspan=1>Listeria innocua</td><td rowspan=1 colspan=1>ATCC 33090</td></tr><tr><td rowspan=1 colspan=1>Listeria monocytogenes</td><td rowspan=1 colspan=1>ATCC 7644</td></tr><tr><td rowspan=1 colspan=1>Micrococcus luteus</td><td rowspan=1 colspan=1>ATCC 10240</td></tr><tr><td rowspan=1 colspan=1>Peptostreptococcusanaerobius</td><td rowspan=1 colspan=1>ATCC 27337</td></tr><tr><td rowspan=1 colspan=1>Propionibacterium acnes</td><td rowspan=1 colspan=1>ATCC 11827</td></tr><tr><td rowspan=1 colspan=1>Rothia mucilaginosa</td><td rowspan=1 colspan=1>ATCC 25296</td></tr></table>

Table 80: Off-Panel Fungal Organisms Assessed for Cross-reactivity with the ePlex BCIDGN Panel (Exclusivity)   

<table><tr><td rowspan=1 colspan=1>Fungal Pathogens</td><td rowspan=1 colspan=1>Strain ID</td><td rowspan=1 colspan=1>Concentration Tested</td></tr><tr><td rowspan=1 colspan=1>Aspergillus fumigatus*</td><td rowspan=1 colspan=1>ATCC 204305</td><td rowspan=1 colspan=1>2.50 x 106 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Candida orthopsilosis</td><td rowspan=1 colspan=1>ATCC 96139</td><td rowspan=1 colspan=1>1 x 107 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Candida metapsilosis</td><td rowspan=1 colspan=1>ATCC 96144</td><td rowspan=1 colspan=1>1 x 107 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Candida tropicalis</td><td rowspan=1 colspan=1>ATCC 1369</td><td rowspan=1 colspan=1>1 x 107 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Cryptococcus grubii</td><td rowspan=1 colspan=1>ATCC 208821</td><td rowspan=1 colspan=1>1 x 107 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Cryptococcus gattii</td><td rowspan=1 colspan=1>ATCC 76108</td><td rowspan=1 colspan=1>1 x 107 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Cryptococcus neoformans</td><td rowspan=1 colspan=1>ATCC 14116</td><td rowspan=1 colspan=1>1 x 107 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Geotrichum capitatum</td><td rowspan=1 colspan=1>ATCC 10663</td><td rowspan=1 colspan=1>1 x 107 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Histoplasma capsulatum</td><td rowspan=1 colspan=1>In silico</td><td rowspan=1 colspan=1>N/A</td></tr><tr><td rowspan=1 colspan=1>Penicillium marneffei</td><td rowspan=1 colspan=1>ATCC 200050</td><td rowspan=1 colspan=1>1 x 107 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Rhodotorula glutinis</td><td rowspan=1 colspan=1>ATCC 32765</td><td rowspan=1 colspan=1>1 x 107 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Rhodotorula mucilaginosa</td><td rowspan=1 colspan=1>ATCC 9449</td><td rowspan=1 colspan=1>1 x 107 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Rhodotorula minuta</td><td rowspan=1 colspan=1>ATCC 36236</td><td rowspan=1 colspan=1>1 x 107 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Saccharomyces cervisiae*</td><td rowspan=1 colspan=1>ATCC 18824</td><td rowspan=1 colspan=1>5.55 x 106 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Trichosporon dermatis</td><td rowspan=1 colspan=1>ATCC MYA-4294</td><td rowspan=1 colspan=1>1 x 107 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Trichosporon mucoides</td><td rowspan=1 colspan=1>ATCC 90046</td><td rowspan=1 colspan=1>1 x 107 CFU/mL</td></tr></table>

Table 81: Off-Panel Resistance Genes Assessed for Cross-reactivity with the ePlex BCIDGN Panel (Exclusivity)   

<table><tr><td rowspan=1 colspan=1>Antimicrobial Resistance Genes</td><td rowspan=1 colspan=1>Strain ID</td><td rowspan=1 colspan=1>Concentration Tested</td></tr><tr><td rowspan=1 colspan=1>FOX (Carried by Klebsiella oxytoca)*A</td><td rowspan=1 colspan=1>JMI 954306</td><td rowspan=1 colspan=1>8 x 108 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>MOX (Carried by Aeromonas hydrophila)</td><td rowspan=1 colspan=1>JMI 938982</td><td rowspan=1 colspan=1>1 x 10 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>SME (Carried by Serratia marcescens)A</td><td rowspan=1 colspan=1>CDC #0091</td><td rowspan=1 colspan=1>1 x 10 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>SHV (Carried by Klebsiella pneumoniae)A</td><td rowspan=1 colspan=1>CDC #0087</td><td rowspan=1 colspan=1>1 x 10 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>TEM (Carried by Escherichia coli)A</td><td rowspan=1 colspan=1>NCTC 13351</td><td rowspan=1 colspan=1>1 x 10 CFU/mL</td></tr></table>

A. The on-panel organism associated with the resistance gene was detected by the BCID-GN panel as expected

Table 82: Off-Panel Gram-Negative Organisms Assessed for Cross-Reactivity with the ePlex BCID-GN Panel based on In Silico Analysis   

<table><tr><td rowspan=1 colspan=1>Cross-reactive Organism</td><td rowspan=1 colspan=1>ePlex BCID-GN Target</td><td rowspan=1 colspan=1>No. ofSequences</td><td rowspan=1 colspan=1>Predicted Cross-Reactive Sequences*n (%)</td></tr><tr><td rowspan=1 colspan=1>Fusobacterium hwasookii</td><td rowspan=1 colspan=1>F. nucleatum</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>5 (50%)</td></tr><tr><td rowspan=1 colspan=1>Haemophilus aegyptius</td><td rowspan=1 colspan=1>H. influenzae</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>3 (100%)</td></tr><tr><td rowspan=1 colspan=1>Klebsiella michiganensis</td><td rowspan=1 colspan=1>Klebsiella oxytoca</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>40 (100%)</td></tr><tr><td rowspan=1 colspan=1>Pseudomonas denitrificans</td><td rowspan=1 colspan=1>Pseudomonas aeruginosa</td><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>16 (94.1%)</td></tr></table>

Table 83: Off-Panel Gram-Positive Organisms Assessed for Cross-Reactivity with the Pan Gram-Positive Assay based on In Silico Analysis   

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Number ofSequences</td><td rowspan=1 colspan=1>Predicted Cross-ReactiveSequencesn (%)</td></tr><tr><td rowspan=1 colspan=1>Brevibacterium halotolerans</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>3 (100%)</td></tr><tr><td rowspan=1 colspan=1>Domibacillus indicus</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1 (100%)</td></tr><tr><td rowspan=1 colspan=1>Domibacillus robiginosus</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1 (100%)</td></tr><tr><td rowspan=1 colspan=1>Salinibacillus aidingensis</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>1 (50%)</td></tr><tr><td rowspan=1 colspan=1>Terribacillus aidingensis</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1 (100%</td></tr><tr><td rowspan=1 colspan=1>Terribacillus halophilus</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>1 (50%)</td></tr><tr><td rowspan=1 colspan=1>Terribacillus saccharophilus</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1 (100%)</td></tr><tr><td rowspan=1 colspan=1>Planomicrobium okeanokoites</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1 (100%)</td></tr><tr><td rowspan=1 colspan=1>Lactococcus chungangensis</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>4 (100%)</td></tr><tr><td rowspan=1 colspan=1>Lactococcus laudensis</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1 (100%)</td></tr><tr><td rowspan=1 colspan=1>Lactococcus piscium</td><td rowspan=1 colspan=1>18</td><td rowspan=1 colspan=1>18 (100%)</td></tr><tr><td rowspan=1 colspan=1>Lactococcus plantarum</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>5 (83.8%)</td></tr><tr><td rowspan=1 colspan=1>Lactococcus rffinolactis</td><td rowspan=1 colspan=1>49</td><td rowspan=1 colspan=1>46 (93.9%)</td></tr><tr><td rowspan=1 colspan=1>Okadaella gastrococcus</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>4 (100%)</td></tr></table>

# Bottle Positivity

Several representative bacterial and fungal organisms were spiked into blood culture bottles along with the manufacturer’s recommended volume of human whole blood and grown to positivity in a commercially-available continuously monitoring blood culture system. Bottles were removed from the incubator within two hours of being identified as positive as well as eight hours after bottle positivity. At least two independent positive blood culture replicates and three blood replicates were quantified for each organism on culture plates. Organisms tested and approximate bottle positivity concentrations are summarized in Table 84. Concentrations shown below represent approximate levels that may be observed in a clinical setting. All estimated bottle positivity concentrations are equivalent or greater than the established Limit of Detection (LOD) for each of the assays of the ePlex BCID-GP Panel. The following bottle types were used for the Bottle Positivity Study: BD BACTEC Plus Aerobic/F blood culture bottle (E. faecium, S. aureus, S.anginosus, A. baumannii, E. cloacae, E. coli, H. influenzae, K. oxytoca, N. meningitidis, $P$ . auerginosa, and S. marcescens), BD BACTEC Lytic/10 Anaerobic/F (B. fragilis and $F _ { \cdot }$ . nucleatum), and BD BACTEC Myco F/Lytic blood culture bottle (C. albicans).

Table 84: Bottle Positivity Concentrations   

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Strain ID</td><td rowspan=1 colspan=1>Mean Bottle PositivityConcentration</td><td rowspan=1 colspan=1>Mean Bottle Positivity+8 hours Concentration</td></tr><tr><td rowspan=1 colspan=4>Gram-positive Organisms</td></tr><tr><td rowspan=1 colspan=1>Enterococcus faecium</td><td rowspan=1 colspan=1>ATCC BAA-2317</td><td rowspan=1 colspan=1>4.9 x 107 CFU/mL</td><td rowspan=1 colspan=1>3.6 x 107 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus aureus</td><td rowspan=1 colspan=1>NRS 483</td><td rowspan=1 colspan=1>2.8 x 107 CFU/mL</td><td rowspan=1 colspan=1>2.1 x 107 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Streptococcus anginosus</td><td rowspan=1 colspan=1>ATCC 33397</td><td rowspan=1 colspan=1>4.1 x 107 CFU/mL</td><td rowspan=1 colspan=1>4.0 x 108 CFU/mL</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2>Gram-negative Organisms</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Acinetobacter baumannii</td><td rowspan=1 colspan=1>NCTC 13301</td><td rowspan=1 colspan=1>4.4 x 108 CFU/mL</td><td rowspan=1 colspan=1>3.8 x 108 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Bacteroides fragilis</td><td rowspan=1 colspan=1>ATCC 700786</td><td rowspan=1 colspan=1>4.7 x 108 CFU/mL</td><td rowspan=1 colspan=1>6.7 x 109 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Enterobacter cloacae</td><td rowspan=1 colspan=1>NCTC 13464</td><td rowspan=1 colspan=1>2.8 x 108 CFU/mL</td><td rowspan=1 colspan=1>7.7 x 108 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Escherichia coli</td><td rowspan=1 colspan=1>NCTC 13476</td><td rowspan=1 colspan=1>2.3 x 108 CFU/mL</td><td rowspan=1 colspan=1>1.5 x 109 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Fusobacterium nucleatum</td><td rowspan=1 colspan=1>ATCC 31647</td><td rowspan=1 colspan=1>6.5 x 107 CFU/mL</td><td rowspan=1 colspan=1>4.9 x 108 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Haemophilus influenzae</td><td rowspan=1 colspan=1>ATCC 19418</td><td rowspan=1 colspan=1>6.9 x 108 CFU/mL</td><td rowspan=1 colspan=1>1.2 x 109 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Klebsiella oxytoca</td><td rowspan=1 colspan=1>CDC #0147</td><td rowspan=1 colspan=1>9.3 x 108 CFU/mL</td><td rowspan=1 colspan=1>1.5 x 109 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Neisseria meningitidis</td><td rowspan=1 colspan=1>ATCC 13102</td><td rowspan=1 colspan=1>3.2 x 107 CFU/mL</td><td rowspan=1 colspan=1>2.1 x 108 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Pseudomonas aeruginosa</td><td rowspan=1 colspan=1>NCTC 13476</td><td rowspan=1 colspan=1>1.6 x 108 CFU/mL</td><td rowspan=1 colspan=1>8.4 x 108 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Serratia marcescens</td><td rowspan=1 colspan=1>ATCC 14041</td><td rowspan=1 colspan=1>1.2 x 109 CFU/mL</td><td rowspan=1 colspan=1>2.2 x 109 CFU/mL</td></tr><tr><td rowspan=1 colspan=3>Fungal Organisms</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Candida albicans</td><td rowspan=1 colspan=1>ATCC 90082</td><td rowspan=1 colspan=1>1.6 x 106 CFU/mL</td><td rowspan=1 colspan=1>1.4 x 106 CFU/mL</td></tr></table>

# Reproducibility

Three positive mixes including 11 on-panel organisms and 5 antibiotic resistance genes representing 17 targets at two concentrations and one negative mix including an off-panel organism were tested. Concentrations in the positive mixes reflected those observed at time of bottle positivity plus 8 hours and time of bottle positivity, and the negative mix contained Cutibacterium granulosum grown in BD BACTEC Lytic/10 Anaerobic/F blood culture bottles to bottle positivity and bottle positivity plus eight hours, which is expected to yield a negative result. Bottle concentrations used in this study are summarized in Table 85. Each of the three positive mixes at two concentrations and the one negative mix were tested a minimum of 108 times. Testing occurred at three sites, with two operators testing the mixes over six days using three cartridge lots. For the negative mix, agreement with the expected negative result was $100 \%$ for all targets in the ePlex BCID-GN Panel.

Table 85: Bottle Positivity Concentrations   

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Bottle PositivityConcentration</td><td rowspan=1 colspan=1>Bottle Positivity +8Hours Concentration</td></tr><tr><td rowspan=1 colspan=1>Acinetobacter baumannii (OXA)</td><td rowspan=1 colspan=1>1 x 108 CFU/mL</td><td rowspan=1 colspan=1>1 x 109 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Enterobacter cloacae (CTX-M, KPC)</td><td rowspan=1 colspan=1>1 x 108 CFU/mL</td><td rowspan=1 colspan=1>1 x 109 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Escherichia coli (IMP)</td><td rowspan=1 colspan=1>1 x 108 CFU/mL</td><td rowspan=1 colspan=1>1 x 109 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Fusobacterium nucleatum</td><td rowspan=1 colspan=1>1 x 107 CFU/mL</td><td rowspan=1 colspan=1>1 x 108 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Haemophilus influenzae</td><td rowspan=1 colspan=1>1 x 108 CFU/mL</td><td rowspan=1 colspan=1>1 x 109 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Klebsiella oxytoca</td><td rowspan=1 colspan=1>1 x 108 CFU/mL</td><td rowspan=1 colspan=1>1 x 109 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Neisseria meningitidis</td><td rowspan=1 colspan=1>3 x 107 CFU/mL</td><td rowspan=1 colspan=1>3 x 108 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Pseudomonas aeruginosa (VIM)</td><td rowspan=1 colspan=1>1 x 108 CFU/mL</td><td rowspan=1 colspan=1>1 x 109 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Serratia marcescens</td><td rowspan=1 colspan=1>1 x 108 CFU/mL</td><td rowspan=1 colspan=1>1 x 109 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Candida albicans (Pan Candida target)</td><td rowspan=1 colspan=1>1 x 106 CFU/mL</td><td rowspan=1 colspan=1>1 x 107 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus aureus (Pan Gram-Positive target)</td><td rowspan=1 colspan=1>1 x 107 CFU/mL</td><td rowspan=1 colspan=1>1 x 108 CFU/mL</td></tr></table>

The percent agreement of each target with the expected result is summarized in Tables 86-102. The ePlex BCID-GN assay demonstrates a high level of agreement $( { \geq } 9 8 \% )$ with the expected results.

Table 86: Percent Agreement for Acinetobacter baumannii   

<table><tr><td rowspan=2 colspan=1>Concentration of Acinetobacterbaumannii</td><td rowspan=2 colspan=1>Site</td><td rowspan=1 colspan=3>Agreement with Expected Results</td></tr><tr><td rowspan=1 colspan=1>Agreed / N</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=4 colspan=1>Bottle Positive+8 Hours(1x109 CFU/mL)</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>36/36</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>(90.4-100)</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>36/36</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>(90.4-100)</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>36/36</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>(90.4-100)</td></tr><tr><td rowspan=1 colspan=1>All</td><td rowspan=1 colspan=1>108/108</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>(96.6-100)</td></tr><tr><td rowspan=4 colspan=1>Bottle Positive(1x108 CFU/mL)</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>36/36</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>(90.4-100)</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>36/36</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>(90.4-100)</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>36/36</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>(90.4-100)</td></tr><tr><td rowspan=1 colspan=1>All</td><td rowspan=1 colspan=1>108/108</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>(96.6-100)</td></tr><tr><td rowspan=4 colspan=1>Negative</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>179/179</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>(97.9-100)</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>178/179</td><td rowspan=1 colspan=1>99.4</td><td rowspan=1 colspan=1>(96.9-99.9)</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>180/180</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>(97.9-100)</td></tr><tr><td rowspan=1 colspan=1>All</td><td rowspan=1 colspan=1>537/538</td><td rowspan=1 colspan=1>99.8</td><td rowspan=1 colspan=1>(99.0-100)</td></tr></table>

CI=Confidence Interval

Table 87: Percent Agreement for Enterobacter cloacae complex   

<table><tr><td rowspan=2 colspan=1>Concentration ofEnterobacter cloacae</td><td rowspan=2 colspan=1>Site</td><td rowspan=1 colspan=3>Agreement with Expected Results</td></tr><tr><td rowspan=1 colspan=1>Agreed / N</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=4 colspan=1>Bottle Positive+8 Hours(1x109 CFU/mL)</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>36/36</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>(90.4-100)</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>36/36</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>(90.4-100)</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>36/36</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>(90.4-100)</td></tr><tr><td rowspan=1 colspan=1>All</td><td rowspan=1 colspan=1>108/108</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>(96.6-100)</td></tr><tr><td rowspan=4 colspan=1>Bottle Positive(1x108 CFU/mL)</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>35/35</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>(90.1-100)</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>36/36</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>(90.4-100)</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>36/36</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>(90.4-100)</td></tr><tr><td rowspan=1 colspan=1>All</td><td rowspan=1 colspan=1>107/107</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>(96.5-100)</td></tr><tr><td rowspan=4 colspan=1>Negative</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>180/180</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>(97.9-100)</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>179/179</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>(97.9-100)</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>180/180</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>(97.9-100)</td></tr><tr><td rowspan=1 colspan=1>All</td><td rowspan=1 colspan=1>539/539</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>(99.3-100)</td></tr></table>

Table 88: Percent Agreement for Escherichia coli   

<table><tr><td rowspan=2 colspan=1>Concentration ofEscherichia coli</td><td rowspan=2 colspan=1>Site</td><td rowspan=1 colspan=3>Agreement with Expected Results</td></tr><tr><td rowspan=1 colspan=1>Agreed / N</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=4 colspan=1>Bottle Positive+8 Hours(1x109 CFU/mL)</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>36/36</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>(90.4-100)</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>36/36</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>(90.4-100)</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>36/36</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>(90.4-100)</td></tr><tr><td rowspan=1 colspan=1>All</td><td rowspan=1 colspan=1>108/108</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>(96.6-100)</td></tr><tr><td rowspan=4 colspan=1>Bottle Positive(1x108 CFU/mL)</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>36/36</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>(90.4-100)</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>35/35</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>(90.1-100)</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>36/36</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>(90.4-100)</td></tr><tr><td rowspan=1 colspan=1>All</td><td rowspan=1 colspan=1>107/107*</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>(96.5-100)</td></tr><tr><td rowspan=4 colspan=1>Negative</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>179/179</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>(97.9-100)</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>180/180</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>(97.9-100)</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>180/180</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>(97.9-100)</td></tr><tr><td rowspan=1 colspan=1>All</td><td rowspan=1 colspan=1>539/539</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>(99.3-100)</td></tr></table>

\* Two samples had a false positive Bacteroides fragilis result.

Table 89: Percent Agreement for Fusobacterium nucleatum   

<table><tr><td rowspan=2 colspan=1>Concentration of Fusobacteriumnucleatum</td><td rowspan=2 colspan=1>Site</td><td rowspan=1 colspan=3>Agreement with Expected Results</td></tr><tr><td rowspan=1 colspan=1>Agreed / N</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=4 colspan=1>Bottle Positive+8 Hours(1x108 CFU/mL)</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>36/36</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>(90.4-100)</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>35/35</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>(90.1-100)</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>36/36</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>(90.4-100)</td></tr><tr><td rowspan=1 colspan=1>All</td><td rowspan=1 colspan=1>107/107</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>(96.5-100)</td></tr><tr><td rowspan=4 colspan=1>Bottle Positive(1x107 CFU/mL)</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>36/36</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>(90.4-100)</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>36/36</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>(90.4-100)</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>36/36</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>(90.4-100)</td></tr><tr><td rowspan=1 colspan=1>All</td><td rowspan=1 colspan=1>108/108*</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>(96.6-100)</td></tr><tr><td rowspan=4 colspan=1>Negative</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>179/179</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>(97.9-100)</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>180/180</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>(97.9-100)</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>180/180</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>(97.9-100)</td></tr><tr><td rowspan=1 colspan=1>All</td><td rowspan=1 colspan=1>539/539</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>(99.3-100)</td></tr></table>

\* One sample had a false positive Fusobacterium necrophorum result.

Table 90: Percent Agreement for Haemophilus influenzae   

<table><tr><td rowspan=2 colspan=1>Concentration ofHaemophilus influenzae</td><td rowspan=2 colspan=1>Site</td><td rowspan=1 colspan=3>Agreement with Expected Results</td></tr><tr><td rowspan=1 colspan=1>Agreed / N</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=4 colspan=1>Bottle Positive+8 Hours(1x109 CFU/mL)</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>36/36</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>(90.4-100)</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>36/36</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>(90.4-100)</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>36/36</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>(90.4-100)</td></tr><tr><td rowspan=1 colspan=1>All</td><td rowspan=1 colspan=1>108/108</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>(96.6-100)</td></tr><tr><td rowspan=4 colspan=1>Bottle Positive(1x108 CFU/mL)</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>36/36</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>(90.4-100)</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>36/36</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>(90.4-100)</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>36/36</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>(90.4-100)</td></tr><tr><td rowspan=1 colspan=1>All</td><td rowspan=1 colspan=1>108/108</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>(96.6-100)</td></tr><tr><td rowspan=4 colspan=1>Negative</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>179/179</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>(97.9-100)</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>179/179</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>(97.9-100)</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>180/180</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>(97.9-100)</td></tr><tr><td rowspan=1 colspan=1>All</td><td rowspan=1 colspan=1>538/538</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>(99.3-100)</td></tr></table>

Table 91: Percent Agreement for Klebsiella oxytoca   

<table><tr><td rowspan=2 colspan=1>Concentration ofKlebsiella oxytoca</td><td rowspan=2 colspan=1>Site</td><td rowspan=1 colspan=3>Agreement with Expected Results</td></tr><tr><td rowspan=1 colspan=1>Agreed / N</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=4 colspan=1>Bottle Positive+8 Hours(1x109 CFU/mL)</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>36/36</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>(90.4-100)</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>36/36</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>(90.4-100)</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>36/36</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>(90.4-100)</td></tr><tr><td rowspan=1 colspan=1>All</td><td rowspan=1 colspan=1>108/108</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>(96.6-100)</td></tr><tr><td rowspan=4 colspan=1>Bottle Positive(1x108 CFU/mL)</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>36/36</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>(90.4-100)</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>36/36</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>(90.4-100)</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>36/36</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>(90.4-100)</td></tr><tr><td rowspan=1 colspan=1>All</td><td rowspan=1 colspan=1>108/108</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>(96.6-100)</td></tr><tr><td rowspan=4 colspan=1>Negative</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>179/179</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>(97.9-100)</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>179/179</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>(97.9-100)</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>180/180</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>(97.9-100)</td></tr><tr><td rowspan=1 colspan=1>All</td><td rowspan=1 colspan=1>538/538</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>(99.3-100)</td></tr></table>

Table 92: Percent Agreement for Neisseria meningitidis   

<table><tr><td rowspan=2 colspan=1>Concentration ofNeisseria meningitidis</td><td rowspan=2 colspan=1>Site</td><td rowspan=1 colspan=3>Agreement with Expected Results</td></tr><tr><td rowspan=1 colspan=1>Agreed / N</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=4 colspan=1>Bottle Positive+8 Hours(3x108 CFU/mL)</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>35/35</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>(90.1-100)</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>36/36</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>(90.4-100)</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>36/36</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>(90.4-100)</td></tr><tr><td rowspan=1 colspan=1>All</td><td rowspan=1 colspan=1>107/107</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>(96.5-100)</td></tr><tr><td rowspan=4 colspan=1>Bottle Positive(3x107 CFU/mL)</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>36/36</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>(90.4-100)</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>36/36</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>(90.4-100)</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>36/36</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>(90.4-100)</td></tr><tr><td rowspan=1 colspan=1>All</td><td rowspan=1 colspan=1>108/108</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>(96.6-100)</td></tr><tr><td rowspan=4 colspan=1>Negative</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>180/180</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>(97.9-100)</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>179/179</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>(97.9-100)</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>180/180</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>(97.9-100)</td></tr><tr><td rowspan=1 colspan=1>All</td><td rowspan=1 colspan=1>539/539</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>(99.3-100)</td></tr></table>

Table 93: Percent Agreement for Pseudomonas aeruginosa   

<table><tr><td rowspan=2 colspan=1>Concentration of Pseudomonasaeruginosa</td><td rowspan=2 colspan=1>Site</td><td rowspan=1 colspan=3>Agreement with Expected Results</td></tr><tr><td rowspan=1 colspan=1>Agreed / N</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=4 colspan=1>Bottle Positive+8 Hours(1x109 CFU/mL)</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>36/36</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>(90.4-100)</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>35/35</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>(90.1-100)</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>36/36</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>(90.4-100)</td></tr><tr><td rowspan=1 colspan=1>All</td><td rowspan=1 colspan=1>107/107</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>(96.5-100)</td></tr><tr><td rowspan=4 colspan=1>Bottle Positive(1x108 CFU/mL)</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>36/36</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>(90.4-100)</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>36/36</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>(90.4-100)</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>36/36</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>(90.4-100)</td></tr><tr><td rowspan=1 colspan=1>All</td><td rowspan=1 colspan=1>108/108</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>(96.6-100)</td></tr><tr><td rowspan=4 colspan=1>Negative</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>179/179</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>(97.9-100)</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>180/180</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>(97.9-100)</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>180/180</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>(97.9-100)</td></tr><tr><td rowspan=1 colspan=1>All</td><td rowspan=1 colspan=1>539/539</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>(99.3-100)</td></tr></table>

Table 94: Percent Agreement for Serratia   

<table><tr><td rowspan=2 colspan=1>Concentration ofSerratia marcescens</td><td rowspan=2 colspan=1>Site</td><td rowspan=1 colspan=3>Agreement with Expected Results</td></tr><tr><td rowspan=1 colspan=1>Agreed / N</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=4 colspan=1>Bottle Positive+8 Hours(1x109 CFU/mL)</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>35/35</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>(90.1-100)</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>36/36</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>(90.4-100)</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>36/36</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>(90.4-100)</td></tr><tr><td rowspan=1 colspan=1>All</td><td rowspan=1 colspan=1>107/107</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>(96.5-100)</td></tr><tr><td rowspan=4 colspan=1>Bottle Positive(1x108 CFU/mL)</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>36/36</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>(90.4-100)</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>33/33</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>(89.6-100)</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>33/33</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>(89.6-100)</td></tr><tr><td rowspan=1 colspan=1>All</td><td rowspan=1 colspan=1>102/102</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>(96.4-100)</td></tr><tr><td rowspan=4 colspan=1>Negative</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>180/180</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>(97.9-100)</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>179/179</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>(97.9-100)</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>180/180</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>(97.9-100)</td></tr><tr><td rowspan=1 colspan=1>All</td><td rowspan=1 colspan=1>539/539</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>(99.3-100)</td></tr></table>

Table 95: Percent Agreement for Serratia marcescens   

<table><tr><td rowspan=2 colspan=1>Concentration ofSerratia marcescens</td><td rowspan=2 colspan=1>Site</td><td rowspan=1 colspan=3>Agreement with Expected Results</td></tr><tr><td rowspan=1 colspan=1>Agreed / N</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=4 colspan=1>Bottle Positive+8 Hours(1x109 CFU/mL)</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>35/35</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>(90.1-100)</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>36/36</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>(90.4-100)</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>36/36</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>(90.4-100)</td></tr><tr><td rowspan=1 colspan=1>All</td><td rowspan=1 colspan=1>107/107</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>(96.5-100)</td></tr><tr><td rowspan=4 colspan=1>Bottle Positive(1x108 CFU/mL)</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>36/36</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>(90.4-100)</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>36/36</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>(90.4-100)</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>36/36</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>(90.4-100)</td></tr><tr><td rowspan=1 colspan=1>All</td><td rowspan=1 colspan=1>108/108</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>(96.6-100)</td></tr><tr><td rowspan=4 colspan=1>Negative</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>180/180</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>(97.9-100)</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>179/179</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>(97.9-100)</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>180/180</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>(97.9-100)</td></tr><tr><td rowspan=1 colspan=1>All</td><td rowspan=1 colspan=1>539/539</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>(99.3-100)</td></tr></table>

Table 96: Percent Agreement for Pan Candida   

<table><tr><td rowspan=2 colspan=1>Concentration ofCandida albicans</td><td rowspan=2 colspan=1>Site</td><td rowspan=1 colspan=3>Agreement with Expected Results</td></tr><tr><td rowspan=1 colspan=1>Agreed / N</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=4 colspan=1>Bottle Positive+8 Hours(1x107 CFU/mL)</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>35/35</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>(90.1-100)</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>36/36</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>(90.4-100)</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>36/36</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>(90.4-100)</td></tr><tr><td rowspan=1 colspan=1>All</td><td rowspan=1 colspan=1>107/107</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>(96.5-100)</td></tr><tr><td rowspan=4 colspan=1>Bottle Positive(1x106 CFU/mL)</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>35/36</td><td rowspan=1 colspan=1>97.2</td><td rowspan=1 colspan=1>(85.8-99.5)</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>36/36</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>(90.4-100)</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>36/36</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>(90.4-100)</td></tr><tr><td rowspan=1 colspan=1>All</td><td rowspan=1 colspan=1>107/108</td><td rowspan=1 colspan=1>99.1</td><td rowspan=1 colspan=1>(94.9-99.8)</td></tr><tr><td rowspan=4 colspan=1>Negative</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>180/180</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>(97.9-100)</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>179/179</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>(97.9-100)</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>180/180</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>(97.9-100)</td></tr><tr><td rowspan=1 colspan=1>All</td><td rowspan=1 colspan=1>539/539</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>(99.3-100)</td></tr></table>

Table 97: Percent Agreement for Pan Gram-Positive   

<table><tr><td rowspan=2 colspan=1>Concentration ofStaphylococcus aureus</td><td rowspan=2 colspan=1>Site</td><td rowspan=1 colspan=3>Agreement with Expected Results</td></tr><tr><td rowspan=1 colspan=1>Agreed / N</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=4 colspan=1>Bottle Positive+8 Hours(1x108 CFU/mL)</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>36/36</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>(90.4-100)</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>36/36</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>(90.4-100)</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>36/36</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>(90.4-100)</td></tr><tr><td rowspan=1 colspan=1>All</td><td rowspan=1 colspan=1>108/108</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>(96.6-100)</td></tr><tr><td rowspan=4 colspan=1>Bottle Positive(1x107 CFU/mL)</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>34/36</td><td rowspan=1 colspan=1>94.4</td><td rowspan=1 colspan=1>(81.9-98.5)</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>35/35</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>(90.1-100)</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>36/36</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>(90.4-100)</td></tr><tr><td rowspan=1 colspan=1>All</td><td rowspan=1 colspan=1>105/107</td><td rowspan=1 colspan=1>98.1</td><td rowspan=1 colspan=1>(93.4-99.5)</td></tr><tr><td rowspan=4 colspan=1>Negative</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>179/179</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>(97.9-100)</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>179/180</td><td rowspan=1 colspan=1>99.4</td><td rowspan=1 colspan=1>(96.9-99.9)</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>180/180</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>(97.9-100)</td></tr><tr><td rowspan=1 colspan=1>All</td><td rowspan=1 colspan=1>538/539</td><td rowspan=1 colspan=1>99.8</td><td rowspan=1 colspan=1>(99.0-100)</td></tr></table>

Table 98: Percent Agreement for CTX-M   

<table><tr><td rowspan=2 colspan=1>Concentration ofEnterobacter cloacae(CTX-M+, KPC+)</td><td rowspan=2 colspan=1>Site</td><td rowspan=1 colspan=3>Agreement with Expected Results</td></tr><tr><td rowspan=1 colspan=1>Agreed / N</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=4 colspan=1>Bottle Positive+8 Hours(1x109 CFU/mL)</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>36/36</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>(90.4-100)</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>36/36</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>(90.4-100)</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>36/36</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>(90.4-100)</td></tr><tr><td rowspan=1 colspan=1>All</td><td rowspan=1 colspan=1>108/108</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>(96.6-100)</td></tr><tr><td rowspan=4 colspan=1>Bottle Positive(1x108 CFU/mL)</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>35/35</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>(90.1-100)</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>36/36</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>(90.4-100)</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>36/36</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>(90.4-100)</td></tr><tr><td rowspan=1 colspan=1>All</td><td rowspan=1 colspan=1>107/107</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>(96.5-100)</td></tr><tr><td rowspan=4 colspan=1>Negative</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>144/144</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>(97.4-100)</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>143/143</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>(97.4-100)</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>144/144</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>(97.4-100)</td></tr><tr><td rowspan=1 colspan=1>All</td><td rowspan=1 colspan=1>431/431</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>(99.1-100)</td></tr></table>

Table 99: Percent Agreement for IMP   

<table><tr><td rowspan=2 colspan=1>Concentration ofEscherichia coli (IMP+)</td><td rowspan=2 colspan=1>Site</td><td rowspan=1 colspan=3>Agreement with Expected Results</td></tr><tr><td rowspan=1 colspan=1>Agreed / N</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=4 colspan=1>Bottle Positive+8 Hours(1x109 CFU/mL)</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>36/36</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>(90.4-100)</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>35/36</td><td rowspan=1 colspan=1>97.2</td><td rowspan=1 colspan=1>(85.8-99.5)</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>36/36</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>(90.4-100)</td></tr><tr><td rowspan=1 colspan=1>All</td><td rowspan=1 colspan=1>107/108</td><td rowspan=1 colspan=1>99.1</td><td rowspan=1 colspan=1>(94.9-99.8)</td></tr><tr><td rowspan=4 colspan=1>Bottle Positive(1x108 CFU/mL)</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>36/36</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>(90.4-100)</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>35/35</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>(90.1-100)</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>35/36</td><td rowspan=1 colspan=1>97.2</td><td rowspan=1 colspan=1>(85.8-99.5)</td></tr><tr><td rowspan=1 colspan=1>All</td><td rowspan=1 colspan=1>106/107</td><td rowspan=1 colspan=1>99.1</td><td rowspan=1 colspan=1>(94.9-99.8)</td></tr><tr><td rowspan=4 colspan=1>Negative</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>143/143</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>(97.4-100)</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>144/144</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>(97.4-100)</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>144/144</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>(97.4-100)</td></tr><tr><td rowspan=1 colspan=1>All</td><td rowspan=1 colspan=1>431/431</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>(99.1-100)</td></tr></table>

Table 100: Percent Agreement for KPC   

<table><tr><td rowspan=2 colspan=1>Concentration ofEnterobacter cloacae(CTX-M+, KPC+)</td><td rowspan=2 colspan=1>Site</td><td rowspan=1 colspan=3>Agreement with Expected Results</td></tr><tr><td rowspan=1 colspan=1>Agreed / N</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=4 colspan=1>Bottle Positive+8 Hours(1x109 CFU/mL)</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>36/36</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>(90.4-100)</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>36/36</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>(90.4-100)</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>36/36</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>(90.4-100)</td></tr><tr><td rowspan=1 colspan=1>All</td><td rowspan=1 colspan=1>107/107</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>(96.5-100)</td></tr><tr><td rowspan=4 colspan=1>Bottle Positive(1x108 CFU/mL)</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>35/35</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>(90.16-100)</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>36/36</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>(90.4-100)</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>36/36</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>(90.4-100)</td></tr><tr><td rowspan=1 colspan=1>All</td><td rowspan=1 colspan=1>107/107</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>(96.4-100)</td></tr><tr><td rowspan=4 colspan=1>Negative</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>144/144</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>(97.4-100)</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>143/143</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>(97.4-100)</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>144/144</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>(97.4-100)</td></tr><tr><td rowspan=1 colspan=1>All</td><td rowspan=1 colspan=1>431/431</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>(99.1-100)</td></tr></table>

Table 101: Percent Agreement for OXA   

<table><tr><td rowspan=2 colspan=1>Concentration ofAcinetobacter baumannii(OXA-23+)</td><td rowspan=2 colspan=1>Site</td><td rowspan=1 colspan=3>Agreement with Expected Results</td></tr><tr><td rowspan=1 colspan=1>Agreed / N</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=4 colspan=1>Bottle Positive+8 Hours(1x109 CFU/mL)</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>36/36</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>(90.4-100)</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>36/36</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>(90.4-100)</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>36/36</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>(90.4-100)</td></tr><tr><td rowspan=1 colspan=1>All</td><td rowspan=1 colspan=1>108/108</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>(96.6-100)</td></tr><tr><td rowspan=4 colspan=1>Bottle Positive(1x108 CFU/mL)</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>36/36</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>(90.4-100)</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>36/36</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>(90.4-100)</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>36/36</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>(90.4-100)</td></tr><tr><td rowspan=1 colspan=1>All</td><td rowspan=1 colspan=1>108/108</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>(96.6-100)</td></tr><tr><td rowspan=4 colspan=1>Negative</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>143/143</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>(97.4-100)</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>143/143</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>(97.4-100)</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>144/144</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>(97.4-100)</td></tr><tr><td rowspan=1 colspan=1>All</td><td rowspan=1 colspan=1>430/430</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>(99.1-100)</td></tr></table>

Table 102: Percent Agreement for VIM   

<table><tr><td rowspan=2 colspan=1>Concentration ofPseudomonas aeruginosa(VIM+)</td><td rowspan=2 colspan=1>Site</td><td rowspan=1 colspan=3>Agreement with Expected Results</td></tr><tr><td rowspan=1 colspan=1>Agreed / N</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=4 colspan=1>Bottle Positive+8 Hours(1x109 CFU/mL)</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>36/36</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>(90.4-100)</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>35/35</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>(90.1-100)</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>36/36</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>(90.4-100)</td></tr><tr><td rowspan=1 colspan=1>All</td><td rowspan=1 colspan=1>107/107</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>(96.5-100)</td></tr><tr><td rowspan=4 colspan=1>Bottle Positive(1x108 CFU/mL)</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>36/36</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>(90.4-100)</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>36/36</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>(90.4-100)</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>36/36</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>(90.4-100)</td></tr><tr><td rowspan=1 colspan=1>All</td><td rowspan=1 colspan=1>108/108</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>(96.6-100)</td></tr><tr><td rowspan=4 colspan=1>Negative</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>143/1437</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>(97.4-100)</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>144/144</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>(97.4-100)</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>144/144</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>(97.4-100)</td></tr><tr><td rowspan=1 colspan=1>All</td><td rowspan=1 colspan=1>431/431</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>(99.1-100)</td></tr></table>

# Interfering Substances and Sample Matrix Equivalency (Bottle Evaluation)

Three organism mixes consisting of 12 on-panel organisms representing 16 targets and negative blood matrix were used to assess potentially interfering substances and bottle types for interference. The concentration of each organism tested is summarized in Table 103.

Table 103: Organism Concentrations for Interfering Substance and Bottle Equivalency Evaluations   

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Concentration</td></tr><tr><td rowspan=1 colspan=1>Acinetobacter baumanii</td><td rowspan=1 colspan=1>4 x 108 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Bacteroides fragilis</td><td rowspan=1 colspan=1>4 x 108 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Enterobacter aerogenes</td><td rowspan=1 colspan=1>2 x 108 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Enterobacter cloacae (CTX-M)</td><td rowspan=1 colspan=1>2 x 108 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Escherichia coli (OXA)</td><td rowspan=1 colspan=1>2 x 108 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Haemophilus influenzae</td><td rowspan=1 colspan=1>6 x 108 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Klebsiella oxytoca</td><td rowspan=1 colspan=1>9 x 108 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Neisseria meningitidis</td><td rowspan=1 colspan=1>3 x 107 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Pseudomonas aeruginosa (IMP)</td><td rowspan=1 colspan=1>1 x 108 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Serratia marcescens</td><td rowspan=1 colspan=1>1 x 109 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus aureus (Pan Gram-Positive target)</td><td rowspan=1 colspan=1>2 x 107 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Candida albicans (Pan Candida target)</td><td rowspan=1 colspan=1>1 x 106 CFU/mL</td></tr></table>

# Interfering Substances

Eighteen substances were used to assess the ePlex BCID-GN Panel for potential interference. The organisms in Table 103 were spiked into negative blood matrix and tested in triplicate with and without each potentially interfering substance. Negative blood matrix was tested to control for potential positive interference. Potentially interfering substances are summarized in Table 104. None of the eighteen substances commonly found in blood culture specimens or as medications commonly used to treat skin or bloodstream infections were found to inhibit the ePlex BCID-GN Panel at clinically relevant concentrations. The effect of interfering substances has only been evaluated for the substances listed in Table 104. Interference due to substances other than those described in this section can lead to erroneous results.

Table 104: Potentially Interfering Substances: Substance List   

<table><tr><td rowspan=1 colspan=1>Endogenous Substances</td><td rowspan=1 colspan=1>Testing Concentration</td></tr><tr><td rowspan=1 colspan=1>Bilirubin</td><td rowspan=1 colspan=1>60 μg/mL</td></tr><tr><td rowspan=1 colspan=1>Hemoglobin</td><td rowspan=1 colspan=1>0.6 g/L</td></tr><tr><td rowspan=1 colspan=1>Human Genomic DNA</td><td rowspan=1 colspan=1>6 x 105 copies/mL</td></tr><tr><td rowspan=1 colspan=1>Triglycerides</td><td rowspan=1 colspan=1>1000 mg/dL</td></tr><tr><td rowspan=1 colspan=1>γ-globulin</td><td rowspan=1 colspan=1>1.7 g/dL</td></tr><tr><td rowspan=1 colspan=1>Exogenous Substances</td><td rowspan=1 colspan=1>Testing Concentration</td></tr><tr><td rowspan=1 colspan=1>Amoxicillin/Clavulanate</td><td rowspan=1 colspan=1>3.5 μg/mL</td></tr><tr><td rowspan=1 colspan=1>Amphotericin B</td><td rowspan=1 colspan=1>2 μg/mL</td></tr><tr><td rowspan=1 colspan=1>Caspofungin</td><td rowspan=1 colspan=1>5 μg/mL</td></tr><tr><td rowspan=1 colspan=1>Ceftriaxone</td><td rowspan=1 colspan=1>0.23 mg/mL</td></tr><tr><td rowspan=1 colspan=1>Ciprofloxacin</td><td rowspan=1 colspan=1>3 mg/L</td></tr><tr><td rowspan=1 colspan=1>Fluconazole</td><td rowspan=1 colspan=1>25 mg/L</td></tr><tr><td rowspan=1 colspan=1>Flucytosine</td><td rowspan=1 colspan=1>90 μg/mL</td></tr><tr><td rowspan=1 colspan=1>Gentamicin sulfate</td><td rowspan=1 colspan=1>3 μg/mL</td></tr><tr><td rowspan=1 colspan=1>Heparin</td><td rowspan=1 colspan=1>0.9 U/mL</td></tr><tr><td rowspan=1 colspan=1>Imipenem</td><td rowspan=1 colspan=1>83 μg/mL</td></tr><tr><td rowspan=1 colspan=1>Sodium Polyanethol Sulfonate</td><td rowspan=1 colspan=1>0.25% w/v</td></tr><tr><td rowspan=1 colspan=1>Tetracycline</td><td rowspan=1 colspan=1>5 mg/L</td></tr><tr><td rowspan=1 colspan=1>Vancomycin</td><td rowspan=1 colspan=1>30 mg/L</td></tr></table>

# Sample Matrix Equivalency (Bottle Evaluation)

Thirteen bottle types were tested for interference with each of the organisms listed in Table103. Five replicates of each organism were tested in each of two bottle lots. Negative blood matrix was run as a negative control. Twelve bottle types tested showed no interference for any of the targets tested. One of three lots of the BACTECTM Lytic Anaerobic bottles tested showed reduced sensitivity for some targets. A summary of the bottle types assessed and the study outcomes is found in Table 105.

Table 105: Sample Matrix Equivalency (Bottle Evaluation) Bottle Types   

<table><tr><td rowspan=1 colspan=1>Manufacturer</td><td rowspan=1 colspan=1>Bottle Brand</td><td rowspan=1 colspan=1>Bottle Type</td><td rowspan=1 colspan=1>Study Outcome</td></tr><tr><td rowspan=1 colspan=1>BD</td><td rowspan=1 colspan=1>BACTECTM</td><td rowspan=1 colspan=1>Plus Aerobic</td><td rowspan=1 colspan=1>No interference observed</td></tr><tr><td rowspan=1 colspan=1>BD</td><td rowspan=1 colspan=1>BACTEC</td><td rowspan=1 colspan=1>Plus Anaerobic</td><td rowspan=1 colspan=1>No interference observed</td></tr><tr><td rowspan=1 colspan=1>BD</td><td rowspan=1 colspan=1>BACTEC</td><td rowspan=1 colspan=1>Standard Aerobic</td><td rowspan=1 colspan=1>No interference observed</td></tr><tr><td rowspan=1 colspan=1>BD</td><td rowspan=1 colspan=1>BACTEC</td><td rowspan=1 colspan=1>Standard Anaerobic</td><td rowspan=1 colspan=1>No interference observed</td></tr><tr><td rowspan=1 colspan=1>BD</td><td rowspan=1 colspan=1>BACTEC</td><td rowspan=1 colspan=1>Peds PlusTM</td><td rowspan=1 colspan=1>No interference observed</td></tr><tr><td rowspan=1 colspan=1>BD</td><td rowspan=1 colspan=1>BACTEC</td><td rowspan=1 colspan=1>Lytic Anaerobic*</td><td rowspan=1 colspan=1>False negative results were observed for PanCandida, Enterobacter cloacae, Escherichiacoli, CTX-M and OXA in one of three lots.*</td></tr><tr><td rowspan=1 colspan=1>bioMérieux</td><td rowspan=1 colspan=1>BACT/ALERT®</td><td rowspan=1 colspan=1>SA Standard Aerobic</td><td rowspan=1 colspan=1>No interference observed</td></tr><tr><td rowspan=1 colspan=1>bioMérieux</td><td rowspan=1 colspan=1>BACT/ALERT</td><td rowspan=1 colspan=1>SN Standard Anaerobic</td><td rowspan=1 colspan=1>No interference observed</td></tr><tr><td rowspan=1 colspan=1>bioMérieux</td><td rowspan=1 colspan=1>BACT/ALERT</td><td rowspan=1 colspan=1>FA Plus</td><td rowspan=1 colspan=1>No interference observed</td></tr><tr><td rowspan=1 colspan=1>bioMérieux</td><td rowspan=1 colspan=1>BACT/ALERT</td><td rowspan=1 colspan=1>FN Plus</td><td rowspan=1 colspan=1>No interference observed</td></tr><tr><td rowspan=1 colspan=1>bioMérieux</td><td rowspan=1 colspan=1>BACT/ALERT</td><td rowspan=1 colspan=1>PF Plus</td><td rowspan=1 colspan=1>No interference observed</td></tr><tr><td rowspan=1 colspan=1>Thermo Scientific™</td><td rowspan=1 colspan=1>VersaTREKTM</td><td rowspan=1 colspan=1>REDOXTM 1 EZ Draw Aerobic</td><td rowspan=1 colspan=1>No interference observed</td></tr><tr><td rowspan=1 colspan=1>Thermo Scientific</td><td rowspan=1 colspan=1>VersaTREK</td><td rowspan=1 colspan=1>REDOX 2 EZ Draw Anaerobic</td><td rowspan=1 colspan=1>No interference observed</td></tr></table>

\* 2/15 replicates were false negative for Pan Candida; 1/15 replicates was false negative for Enterobacter cloacae; 1/15 replicates was false negative for Escherichia coli (OXA-48); 2/15 replicates were false negative for CTX-M

# Carryover and Cross-Contamination

Carryover and cross-contamination were evaluated for the ePlex BCID-GN Panel within and between runs by alternating high positive and negative samples across multiple runs over 5 rounds of testing. A high-titer mix of OXA positive Escherichia coli, CTX-M and KPC positive Enterobacter cloacae, Salmonella enterica, and Enterococcus faecalis (a Pan Gram-Positive target organism) was prepared at $1 \mathrm { ~ x ~ } 1 0 ^ { 9 } \mathrm { C F U / m L }$ each as well as Candida krusei (a Pan Candida target organism) at $1 \mathrm { { x } 1 0 ^ { 7 } C F U / m L }$ to simulate clinically relevant high positive samples for positive testing. Negative blood culture matrix was used to represent negative samples. Over 120 runs, all valid positive runs resulted in detection of Escherichia coli, Enterobacter cloacae complex, Salmonella, OXA, CTX-M, KPC, Pan Gram-Positive, and Pan Candida and no false positives were detected in the negative runs.

# Competitive Inhibition Study

Competitive inhibition was evaluated for the ePlex BCID-GN Panel by pairing eight clinically relevant organisms (including a Pan Gram-Positive assay target and an off-panel gram-positive organism) in four simulated dual infection sample mixes. Each dual infection mix was tested in combination with each of the three other mixes, such that all organisms were tested at low titer (concentrations expected at bottle positivity) while in the presence of other organisms at higher titer (concentrations expected at 8 hours beyond bottle positivity, or $\sim$ one log higher than that expected at bottle positivity). No competitive inhibition was observed in any replicates of the twelve testing conditions. A summary of the organisms assessed and testing concentrations is found in Table 106.

Table 106: Competitive Inhibition Organisms and Concentrations Tested   

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>HighConcentration</td><td rowspan=1 colspan=1>LowConcentration</td></tr><tr><td rowspan=1 colspan=1>Klebsiella pneumoniae</td><td rowspan=1 colspan=1>1 x 10 CFU/mL</td><td rowspan=1 colspan=1>9 x 108 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Escherichia coli (CTX-M+)</td><td rowspan=1 colspan=1>1 x 109 CFU/mL</td><td rowspan=1 colspan=1>2 x 108 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Enterobacter cloacae (VIM+)</td><td rowspan=1 colspan=1>7 x 108CFU/mL</td><td rowspan=1 colspan=1>2 x 108 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Klebsiella oxytoca (KPC+)</td><td rowspan=1 colspan=1>1 x 1099CFU/mL</td><td rowspan=1 colspan=1>9 x 108 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Pseudomonas aeruginosa (IMP+)</td><td rowspan=1 colspan=1>8 x 108 CFU/mL</td><td rowspan=1 colspan=1>1 x 108 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Serratia marcescens</td><td rowspan=1 colspan=1>2 x 109 CFU/mL</td><td rowspan=1 colspan=1>1 x 109 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus aureus</td><td rowspan=1 colspan=1>1 x 108 CFU/mL</td><td rowspan=1 colspan=1>2 x 107 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Corynebacterium striatum4</td><td rowspan=1 colspan=1>2 x 109 CFU/mL</td><td rowspan=1 colspan=1>4 x 106 CFU/mL</td></tr></table>

AOff-panel organism