# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATIONDECISION SUMMARYASSAY AND INSTRUMENT

I Background Information:

# B Applicant

Roche Molecular Systems, Inc.

# C Proprietary and Established Names

cobas SARS-CoV-2 Qualitative for use on the cobas 6800/8800 Systems

# D Regulatory Information

<table><tr><td rowspan=1 colspan=1>ProductCode(s)</td><td rowspan=1 colspan=1>Classification</td><td rowspan=1 colspan=1>RegulationSection</td><td rowspan=1 colspan=1>Panel</td></tr><tr><td rowspan=1 colspan=1>QQX</td><td rowspan=1 colspan=1>Class II</td><td rowspan=1 colspan=1>21 CFR 866.3981 - Deviceto detect and identifynucleic acid targets inrespiratory specimensfrom microbial agents thatcause the SARS-CoV-2respiratory infection andother microbial agentswhen in a multi-target test</td><td rowspan=1 colspan=1>MI - Microbiology</td></tr></table>

II Submission/Device Overview:

A Purpose for Submission: New Device   
B Measurand: SARS-CoV-2 RNA   
C Type of Test:

Nucleic acid amplification test with paired reporter and quencher fluorescence labeled probes (TaqMan Technology) using fluorescence resonance energy transfer (FRET)

# III Intended Use/Indications for Use:

A Intended Use(s):

See Indications for Use below.

# B Indication(s) for Use:

cobas SARS-CoV-2 Qualitative for use on the cobas 6800/8800 Systems is a real-time RT-PCR test intended for the qualitative detection of nucleic acids from SARS-CoV-2 in nasal and nasopharyngeal specimens collected from symptomatic individuals suspected of COVID-19 by their healthcare provider.

Results are for the detection of SARS-CoV-2 RNA. Positive results are indicative of the presence of SARS-CoV-2 RNA; clinical correlation with patient history and other diagnostic information is necessary to determine patient infection status. Positive results do not rule out bacterial infection or co-infection with other pathogens.

Negative results do not preclude SARS-CoV-2 infection and should not be used as the sole basis for patient management decisions. Results are meant to be used in conjuction with clinical observations, patient history, recent exposures and epidemiological information, and laboratory data, in accordance with the guidelines provided by the relevant public health authorities. cobas SARS-CoV-2 is intended for use by qualified clinical laboratory personnel specifically instructed and trained in the techniques of real-time PCR and on the use of the cobas 6800/8800 Systems.

C Special Conditions for Use Statement(s):

Rx - For Prescription Use Only

# D Special Instrument Requirements:

This test is to be used with the Roche cobas 6800/8800 system only.

# IV Device/System Characteristics:

# A Device Description:

cobas SARS-CoV-2 Qualitative is based on fully automated sample preparation (nucleic acid extraction and purification) followed by PCR amplification and detection. The cobas 6800/8800 Systems consists of two separate instruments, the cobas 6800 System and the cobas 8800 System, which both are fully integrated and allow users to perform multiple PCR-based in vitro nucleic acid amplification tests. The cobas 6800/8800 Systems consist of the sample supply module, the transfer module, the processing module, and the analytic module. Automated data management is performed by the cobas 6800/8800 Systems software(s), which assigns test results for all tests. Results can be reviewed directly on the system screen and printed as a report.

# cobas 6600 instrument

![](images/7c69a50465612606695b9ef962ea87708dd646bcf430eb87e7c34e5705f4694d.jpg)  
cobas 8800 instrument

![](images/13108b770e5baca02ab0d3a3f8e23396c64b9ec84675f35a5a4bef3ec000b730.jpg)

The main system functionality is provided by two Software components: the cobas 6800/8800 System Software and Assay Specific Analysis Packages (ASAPs). The cobas 6800/8800 System Software provides basic functionality such as a Graphical User Interface (GUI), instrument management, database functionality, report engines, and LIS interfaces. ASAPs are built using a common Software framework and provides the assay test run conditions (sample preparation and PCR parameters), result analysis functionality (result calculation and algorithms), and result report formatting.

Nucleic acid from patient samples and added internal control RNA (RNA IC) molecules are simultaneously extracted. Nucleic acid is released by addition of proteinase and lysis reagent to the sample. The released nucleic acid binds to the silica surface of the added magnetic glass particles. Unbound substances and impurities, such as denatured protein, cellular debris, and potential PCR inhibitors, are removed with subsequent wash steps and purified nucleic acid is eluted from the magnetic glass particles with elution buffer at elevated temperature. External controls (positive and negative) are processed in the same way.

Selective amplification of target nucleic acid from the sample is achieved using target specific forward and reverse primers for ORF1 a/b non-structural region that is unique to SARS- CoV-2. Additionally, a conserved region in the structural protein envelope E-gene were chosen for panSarbecovirus detection. The pan-Sarbecovirus detection sets will also detect SARS-CoV- 2 virus. Selective amplification of RNA IC is achieved using non-competitive sequence specific forward and reverse primers which have no homology with the coronavirus genome. A thermostable DNA polymerase enzyme is used for amplification.

# B Principle of Operation:

The cobas SARS-CoV-2 Qualitative master mix contains detection probes which are specific for the coronavirus type SARS-CoV-2, members of the Sarbecovirus subgenus, and the RNA IC nucleic acid. The coronavirus and RNA IC detection probes are each labeled with unique fluorescent dyes that act as a reporter. Each probe also has a second dye which acts as a quencher. When not bound to the target sequence, the fluorescent signals of the intact probes are suppressed by the quencher dye. During the PCR amplification step, hybridization of the probes to the specific single-stranded DNA template results in cleavage of the probe by the $5 "$ to $3 "$ exonuclease activity of the DNA polymerase resulting in separation of the reporter and quencher dyes and the generation of a fluorescent signal. With each PCR cycle, increasing amounts of cleaved probes are generated and the cumulative signal of the reporter dye increases concomitantly. Each reporter dye is measured at defined wavelengths, which enables simultaneous detection and discrimination of the amplified coronavirus target and the RNA IC. The master mix includes deoxy uridine triphosphate (dUTP), instead of deoxythymidine triphosphate (dTTP), which is incorporated into the newly synthesized DNA (amplicon). Any contaminating amplicons from previous PCR runs are destroyed by the AmpErase enzyme [uracil-N-glycosylase], which is included in the PCR mix, when heated in the first thermal cycling step. However, newly formed amplicons are not destroyed since the AmpErase enzyme is inactivated once exposed to temperatures above $5 5 \mathrm { { ^ \circ C } }$ .

Reagents loaded onto the cobas 6800/8800 Systems are stored at appropriate temperatures and their expiration is monitored by the system. The cobas 6800/8800 Systems allow reagents to be used only if all the conditions are met. The system automatically prevents use of expired reagents via encoded radio frequency identification (RFID) tags that are placed on the reagent cassettes, RMC cassettes and reagent bottles. RFID Labels are used to apply information on test specific and generic reagent cassettes, control tube containers and generic reagent bottles utilized on the cobas 6800/8800 Systems.

# Components of the cobas SARS-CoV-2 Qualitative Kit

<table><tr><td>Components Name</td><td>Formulation</td></tr><tr><td>Proteinase Solution (PASE)</td><td>Tris buffer, &lt; 0.05% EDTA, calcium chloride, calcium acetate, 8% proteinase, glycerol EUH210: Safety data sheet available on request. EUH208: Contains Subtilisin. May produce an allergic reaction.</td></tr><tr><td>RNA Internal Control (RNA IC)</td><td>Tris buffer, &lt; 0.05% EDTA, &lt; 0.001% non-Sarbecovirus related armored RNA construct containing primer and probe specific primer sequence regions (non- infectious RNA in MS2 bacteriophage), &lt; 0.1% sodium azide</td></tr><tr><td colspan="1" rowspan="1">Elution Buffer(EB)</td><td colspan="1" rowspan="1">Tris buffer, 0.2% methyl-4 hydroxybenzoate</td></tr><tr><td colspan="1" rowspan="1">Master Mix Reagent 1(MMX-R1)</td><td colspan="1" rowspan="1">Manganese acetate, potassium hydroxide, &lt;0.1% sodium azide</td></tr><tr><td colspan="1" rowspan="1">SARS-CoV-2 Master MixReagent 2(SARS-CoV-2 MMX-R2)</td><td colspan="1" rowspan="1">Tricine buffer, potassium acetate, &lt; 18% dimethyl sulfoxide,glycerol, &lt; 0.1% Tween 20, EDTA, &lt; 0.12% dATP, dCTP, dGTP,dUTPs, &lt; 0.01% upstream and downstream SARS-CoV-2 and Sarbecovirusprimers, &lt; 0.01% Internal Control forward and reverse primers,&lt; 0.01% fluorescent-labeled oligonucleotide probes specific for SARS-CoV-2,Sarbecovirus, and the RNA Internal Control, &lt; 0.01% oligonucleotide aptamer,&lt; 0.1% Z05D DNA polymerase, &lt; 0.10% AmpErase (uracil-N-glycosylase) enzyme(microbial), &lt; 0.1% sodium azide</td></tr></table>

# Components of the cobas SARS-CoV-2 Qualitative Control Kit

<table><tr><td>Components Name</td><td>Formulatio n</td></tr><tr><td>SARS-CoV-2 Positive Control (SARS-CoV-2 (+) C)</td><td>Tris buffer, &lt; 0.05% Sodium azide, &lt; 0.005% EDTA, &lt; 0.003% Poly rA, &lt; 0.01% Non-infectious plasmid DNA (microbial) containing SARS-CoV-2 sequence, &lt;0.01% Non-infectious plasmid DNA (microbial) containing pan-Sarbecovirus sequence</td></tr></table>

# Components of the cobas Buffer Negative Control Kit

<table><tr><td>Components Name</td><td>Formulation</td></tr><tr><td>cobas Buffer Negative Control (BUF (-) C)</td><td>Tris buffer, &lt; 0.1% sodium azide, EDTA, &lt; 0.002% Poly rA RNA (synthetic)</td></tr></table>

Automated data management is performed by the Assay Specific Analysis Package software which assigns test results for all tests. Results can be review directly on the system screen, exported, or printed as a report.

The results interpretation for detecting SARS-CoV-2 is summarized below.

# cobas SARS-CoV-2 Qualitative Results Interpretation

<table><tr><td colspan="1" rowspan="1">Target 1(SCoV2)</td><td colspan="1" rowspan="1">Target 2(PanSarb)</td><td colspan="1" rowspan="1">Interpretation</td></tr><tr><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">PanSarb Positive</td><td colspan="1" rowspan="1">All Target Results were valid. Result for SARS-CoV-2 RNAis Detected.</td></tr><tr><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">All Target Results were valid.Result for SARS-CoV-2 RNA is Detected. A positiveTarget 1 result and a negative Target 2 result is suggestiveof 1) a sample at concentrations near or below the limit ofdetection of the test,2) a mutation in the Target 2, target region, or 3) otherfactors.</td></tr><tr><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">All Target Results were valid.Result for SARS-CoV-2 RNA is Presumptive Positive. Anegative Target 1 result and a positive Target 2 result issuggestive of 1) a sample at concentrations near or belowthe limit of detection of the test,2) a mutation in the Target 1 target region in the oligobinding sites, or 3) infection with some other Sarbecovirus(e.g., SARS-CoV or some other Sarbecovirus previouslyunknown to infect humans), or 4) other factors. Forsamples with a Presumptive Positive result, additionalconfirmatory testing may be conducted, if it is necessary todifferentiate between SARS-CoV-2 and SARS-CoV-1 orother Sarbecovirus currently unknown to infect humans, forepidemiological purposes or clinical management.</td></tr><tr><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">All Target Results were valid.Result for SARS-CoV-2 RNA is Not Detected.</td></tr><tr><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Invalid</td><td colspan="1" rowspan="1">Not all Target Results were valid.Result for SARS-CoV-2 RNA is Detected.</td></tr><tr><td colspan="1" rowspan="1">Invalid</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Not all Target Results were valid.Result for SARS-CoV-2 RNA is Presumptive Positive.For samples with a Presumptive Positive result, additionalconfirmatory testing may be conducted, if it is necessary todifferentiate between SARS-CoV-2 and SARS-CoV-1 orother Sarbecovirus currently unknown to infect humans, forepidemiological purposes or clinical management.</td></tr><tr><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Invalid</td><td colspan="1" rowspan="1">Not all Target Results were valid.Sample should be retested. If the result is still invalid, anew specimen should be obtained.</td></tr><tr><td colspan="1" rowspan="1">Invalid</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Not all Target Results were valid.Sample should be retested. If the result is still invalid, anew specimen should be obtained.</td></tr><tr><td colspan="1" rowspan="1">Invalid</td><td colspan="1" rowspan="1">Invalid</td><td colspan="1" rowspan="1">All Target Results were invalid.Sample should be retested. If the result is still invalid, anew specimen should be obtained.</td></tr></table>

# C Instrument Description Information:

1. Instrument Name: cobas 6800/8800 Systems and software version 1.4.7; cobas SCoV2-QL ASAP Software version 12.1.0

2. Specimen Identification: Specimen identification can be configured in an automated fashion or entered manually. All corresponding user interactions are recorded in the audit trail on the Instrument Gateway

3. Specimen Sampling and Handling: Nasopharyngeal swab (NPS) or Anterior Nasal swab (ANS) specimens collected in transport media.

4. Calibration: Not Applicable

5. Quality Control: The RNA Internal Control (RNA-IC), used to monitor the entire sample preparation and PCR amplification process, is introduced into each specimen during sample processing. In addition, the test utilizes external controls- cobas SARS-CoV-2 Qualitative Control Kit (low titer positive control) and cobas Buffer Negative Control Kit (negative control), provided separately.

V Substantial Equivalence Information:

A Predicate Device Name(s): BioFire COVID-19 Test 2 B Predicate 510(k) Number(s): K211079 C Comparison with Predicate(s):

<table><tr><td colspan="1" rowspan="1">Device &amp; PredicateDevice(s):</td><td colspan="1" rowspan="1">K213804</td><td colspan="1" rowspan="1">K211079</td></tr><tr><td colspan="1" rowspan="1">Device Trade Name</td><td colspan="1" rowspan="1">cobas SARS-CoV-2 Qualitative</td><td colspan="1" rowspan="1">BioFire COVID-19 Test 2</td></tr><tr><td colspan="1" rowspan="1">Regulation Number</td><td colspan="1" rowspan="1">21 CFR 866.3981</td><td colspan="1" rowspan="1">same</td></tr><tr><td colspan="1" rowspan="1">Regulation Name</td><td colspan="1" rowspan="1">Device to detect and identify nucleicacid targets in respiratory specimensfrom microbial agents that cause theSARS-CoV-2 respiratory infectionand other microbial agents when in amulti-target test</td><td colspan="1" rowspan="1">same</td></tr><tr><td colspan="1" rowspan="1">Product Code</td><td colspan="1" rowspan="1">QQX</td><td colspan="1" rowspan="1">same</td></tr><tr><td colspan="1" rowspan="1">Intended Use</td><td colspan="1" rowspan="1">cobas SARS-CoV-2 Qualitative foruse on the cobas 6800/8800 Systemsis a real-time RT-PCR test intendedfor the qualitative detection ofnucleic acids from SARS-CoV-2 innasal and nasopharyngeal specimenscollected from symptomaticindividuals suspected of COVID-19by their healthcare provider.Results are for the detection ofSARS-CoV-2 RNA. Positive resultsare indicative of the presence ofSARS-CoV-2 RNA; clinicalcorrelation with patient history andother diagnostic information isnecessary to determine patientinfection status. Positive results donot rule out bacterial infection or co-infection with other pathogens.Negative results do not precludeSARS-CoV-2 infection and should</td><td colspan="1" rowspan="1">The BioFire COVID-19 Test 2 is aqualitative nested multiplexed RT-PCR in vitro diagnostic testintended for use with the BioFireFilmArray 2.0 and BioFireFilmArray Torch Systems. TheBioFire COVID-19 Test 2 detectsnucleic acids from severe acuterespiratory syndrome coronavirus2 (SARS-CoV-2) innasopharyngeal swabs (NPS) fromsymptomatic individuals suspectedof COVID-19 by their healthcareprovider.Results are for the identification ofSARS-CoV-2 RNA. The SARS-CoV-2 RNA is generallydetectable in NPS specimensduring the acute phase of infection.Positive results are indicative ofthe presence of SARS-CoV-2</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">not be used as the sole basis forpatient management decisions.Results are meant to be used inconjunction with clinicalobservations, patient history, recentexposures and epidemiologicalinformation, and laboratory data, inaccordance with the guidelinesprovided by the relevant publichealth authorities. cobas SARS-CoV-2 is intended for use by qualifiedclinical laboratory personnelspecifically instructed and trained inthe techniques of real-time PCR andon the use of the cobas 6800/8800Systems.</td><td colspan="1" rowspan="1">RNA; clinical correlation withpatient history and other diagnosticinformation is necessary todetermine patient infection status.Positive results do not rule out co-infection with other pathogens.Results are meant to be used inconjunction with other clinical,epidemiologic, and laboratorydata, in accordance with theguidelines provided by the relevantpublic health authorities. TheBioFire COVID-19 Test 2 isintended for use by trained medicaland laboratory professionals in alaboratory setting or under thesupervision of a trained laboratoryprofessional.</td></tr><tr><td colspan="1" rowspan="1">Conditions for use</td><td colspan="1" rowspan="1">For prescription use</td><td colspan="1" rowspan="1">same</td></tr><tr><td colspan="1" rowspan="1">Sample Types</td><td colspan="1" rowspan="1">Nasopharyngeal swab specimenNasal swab specimen</td><td colspan="1" rowspan="1">Nasopharyngeal swab specimen</td></tr><tr><td colspan="1" rowspan="1">Analyte Targets</td><td colspan="1" rowspan="1">SARS-CoV-2</td><td colspan="1" rowspan="1">SARS-CoV-2</td></tr><tr><td colspan="1" rowspan="1">Sample PreparationProcedure</td><td colspan="1" rowspan="1">Automated by cobas 6800/8800Systems</td><td colspan="1" rowspan="1">Automated by BioFire FilmArray2.0 or BioFire FilmArray Torchsystems</td></tr><tr><td colspan="1" rowspan="1">AmplificationTechnology</td><td colspan="1" rowspan="1">Real-time PCR</td><td colspan="1" rowspan="1">Nested multiplex RT-PCR</td></tr><tr><td colspan="1" rowspan="1">Analyte</td><td colspan="1" rowspan="1">RNA</td><td colspan="1" rowspan="1">RNA</td></tr><tr><td colspan="1" rowspan="1">Detection Chemistry</td><td colspan="1" rowspan="1">Paired reporter and quencherfluorescence labeled probes(TaqMan Technology) usingfluorescence resonance energytransfer (FRET)</td><td colspan="1" rowspan="1">Two Step Nested multiplex PCR:Reverse transcription, followed bya multiplexed first stage PCRreaction (PCR1).Multiple simultaneous second-stage PCR reactions (PCR2) toamplify sequences within thePCR1 products usingfluorescence double strandedbinding dye. Endpoint meltingcurve data to detect target-specific amplicons</td></tr><tr><td colspan="1" rowspan="1">Controls used</td><td colspan="1" rowspan="1">1.The RNA Internal Control(RNA-IC)External Positive and negativecontrols</td><td colspan="1" rowspan="1">Two process controls:1. RNA Process Control (IC)2. PCR2 Control (A positive resultindicates that PCR2 wassuccessful)</td></tr><tr><td colspan="1" rowspan="1">Result Analysis</td><td colspan="1" rowspan="1">Based on PCR cycle thresholdanalysis</td><td colspan="1" rowspan="1">Endpoint melting curve data todetect target-specific amplicons</td></tr><tr><td colspan="1" rowspan="1">Test Interpretation</td><td colspan="1" rowspan="1">Automated test interpretation andreport generation. User cannot accessraw data.</td><td colspan="1" rowspan="1">same</td></tr></table>

# VI Standards/Guidance Documents Referenced:

Class II Special Controls as per 21 CFR 866.3981.

# VII Performance Characteristics:

# A Analytical Performance:

1. Precision/Reproducibility:

a) Within-laboratory Precision:

Within-laboratory precision was examined using a panel of heat-inactivated SARS-CoV-2 (USA-WA1/2020) cultures diluted in simulated clinical matrix stabilized in UTM. Sources of variability were examined with a panel consisting of three concentration levels (0.3x, 1x and 3x LoD), using three lots of cobas SARS-CoV-2 Qualitative reagents and three instruments over a course of 15 testing days (2 runs/day x 3 instruments x 5 days/instrument) for a total of 30 runs. A total of 90 replicates per concentration level were tested over the course of the study. A negative panel was tested along with the positive panel members. The tables below summarize the results from the inter-lot precision study, showing the mean Ct values and calculated SD and $\%$ CV for each panel member.

Qualitative Summary of Within Laboratory Precision   

<table><tr><td rowspan=1 colspan=1>Target</td><td rowspan=1 colspan=1>Panel Member</td><td rowspan=1 colspan=1>Level(x LoD)</td><td rowspan=1 colspan=1>TotalResults (N)</td><td rowspan=1 colspan=1>PositiveResults</td><td rowspan=1 colspan=1>Positivity %</td><td rowspan=1 colspan=1>Two-sided95% CILower Bound</td><td rowspan=1 colspan=1>Two-sided95% CI UpperBound</td></tr><tr><td rowspan=3 colspan=1>Target 1(SARS-CoV-2)</td><td rowspan=1 colspan=1>High negative</td><td rowspan=1 colspan=1>~0.3x</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>10%</td><td rowspan=1 colspan=1>5%</td><td rowspan=1 colspan=1>18%</td></tr><tr><td rowspan=1 colspan=1>Low positive</td><td rowspan=1 colspan=1>~1.0x</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>82</td><td rowspan=1 colspan=1>91%</td><td rowspan=1 colspan=1>83%</td><td rowspan=1 colspan=1>96%</td></tr><tr><td rowspan=1 colspan=1>Moderate positive</td><td rowspan=1 colspan=1>~3.0x</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>96%</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=3 colspan=1>Target 2(pan-Sarbecovirus)</td><td rowspan=1 colspan=1>High negative</td><td rowspan=1 colspan=1>~0.3x</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>31</td><td rowspan=1 colspan=1>34%</td><td rowspan=1 colspan=1>25%</td><td rowspan=1 colspan=1>45%</td></tr><tr><td rowspan=1 colspan=1>Low positive</td><td rowspan=1 colspan=1>~1.0x</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>84</td><td rowspan=1 colspan=1>93%</td><td rowspan=1 colspan=1>86%</td><td rowspan=1 colspan=1>97%</td></tr><tr><td rowspan=1 colspan=1>Moderate positive</td><td rowspan=1 colspan=1>~3.0x</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>96%</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Blank</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>4%</td></tr></table>

# Summary of the Precision Studies based on Ct values

<table><tr><td colspan="1" rowspan="2">Target</td><td colspan="1" rowspan="2">Level(x LoD)</td><td colspan="1" rowspan="2">Hit rate</td><td colspan="1" rowspan="2">MeanCt</td><td colspan="2" rowspan="1">Within Run</td><td colspan="2" rowspan="1">Run-to-Run</td><td colspan="2" rowspan="1">Day-to-Day</td><td colspan="2" rowspan="1">Instrument-to-Instrument</td><td colspan="2" rowspan="1">Lot-to-Lot</td><td colspan="2" rowspan="1">Total</td></tr><tr><td colspan="1" rowspan="1">SD</td><td colspan="1" rowspan="1">SD</td><td colspan="1" rowspan="1">SD</td><td colspan="1" rowspan="1">CV%</td><td colspan="1" rowspan="1">CV%</td><td colspan="1" rowspan="1">CV%</td><td colspan="1" rowspan="1">SD</td><td colspan="1" rowspan="1">CV%</td><td colspan="1" rowspan="1">SD</td><td colspan="1" rowspan="1">CV%</td><td colspan="1" rowspan="1">SD</td><td colspan="1" rowspan="1">CV%</td></tr><tr><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="1">~0.3x</td><td colspan="1" rowspan="1">10.0%</td><td colspan="1" rowspan="1">32.51</td><td colspan="1" rowspan="1">0.5</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">1.4</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">0.5</td><td colspan="1" rowspan="1">1.4</td></tr><tr><td colspan="1" rowspan="1">~1.0x</td><td colspan="1" rowspan="1">91.1%</td><td colspan="1" rowspan="1">32.1</td><td colspan="1" rowspan="1">0.6</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">0.1</td><td colspan="1" rowspan="1">0.3</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">1.8</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">0.2</td><td colspan="1" rowspan="1">0.6</td><td colspan="1" rowspan="1">0.6</td><td colspan="1" rowspan="1">1.9</td></tr><tr><td colspan="1" rowspan="3">TargetTarget 1(SARS-CoV-2)</td><td colspan="1" rowspan="2">Level(x LoD)</td><td colspan="1" rowspan="2">Hit rate</td><td colspan="1" rowspan="2">MeanCt</td><td colspan="2" rowspan="1">Within Run</td><td colspan="2" rowspan="1">Run-to-Run</td><td colspan="2" rowspan="1">Day-to-Day</td><td colspan="2" rowspan="1">Instrument-to-Instrument</td><td colspan="2" rowspan="1">Lot-to-Lot</td><td colspan="2" rowspan="1">Total</td></tr><tr><td colspan="1" rowspan="1">SD</td><td colspan="1" rowspan="1">SD</td><td colspan="1" rowspan="1">SD</td><td colspan="1" rowspan="1">CV%</td><td colspan="1" rowspan="1">CV%</td><td colspan="1" rowspan="1">CV%</td><td colspan="1" rowspan="1">SD</td><td colspan="1" rowspan="1">CV%</td><td colspan="1" rowspan="1">SD</td><td colspan="1" rowspan="1">CV%</td><td colspan="1" rowspan="1">SD</td><td colspan="1" rowspan="1">CV%</td></tr><tr><td colspan="1" rowspan="1">~3.0x</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">31.18</td><td colspan="1" rowspan="1">0.3</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">0.9</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">0.2</td><td colspan="1" rowspan="1">0.7</td><td colspan="1" rowspan="1">0.4</td><td colspan="1" rowspan="1">1.1</td></tr><tr><td colspan="1" rowspan="3">Target 2(pan-Sarbeco-virus</td><td colspan="1" rowspan="1">~0.3x</td><td colspan="1" rowspan="1">34.4%</td><td colspan="1" rowspan="1">35.36</td><td colspan="1" rowspan="1">0.5</td><td colspan="1" rowspan="1">0.1</td><td colspan="1" rowspan="1">0.3</td><td colspan="1" rowspan="1">0.8</td><td colspan="1" rowspan="1">0.2</td><td colspan="1" rowspan="1">1.5</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">0.5</td><td colspan="1" rowspan="1">1.3</td><td colspan="1" rowspan="1">0.8</td><td colspan="1" rowspan="1">2.2</td></tr><tr><td colspan="1" rowspan="1">~1.0x</td><td colspan="1" rowspan="1">93.3%</td><td colspan="1" rowspan="1">34.21</td><td colspan="1" rowspan="1">0.7</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">0.2</td><td colspan="1" rowspan="1">0.6</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">0.1</td><td colspan="1" rowspan="1">0.3</td><td colspan="1" rowspan="1">0.7</td><td colspan="1" rowspan="1">2.2</td></tr><tr><td colspan="1" rowspan="1">~3.0x</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">32.9</td><td colspan="1" rowspan="1">0.4</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">1.1</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">0.1</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">0.4</td><td colspan="1" rowspan="1">1.1</td></tr></table>

# b) Reproducibility:

The reproducibility of cobas SARS-CoV-2 Qualitative was evaluated at three testing sites, using three reagent lots, with a four-member panel of positive and negative samples resulting in a total number 216 tests per concentration (not including controls). The positive panel members contained the SARS-CoV-2 target in negative simulated clinical matrix stabilized in UTM at three different concentrations (0.3x, 1x, 3x LoD). Each site tested two reagent lots for six days. Two runs were performed each day and three replicates of each panel member were performed for each run. An overall SARS-CoV-2 positive result was determined by a positive detection in either or both SARS-CoV-2 or/and pan-Sarbecovirus channels. The study results are summarized in Table below.

Summary of Reproducibility Variance Components based on Ct values   

<table><tr><td rowspan=2 colspan=1>ViralTarget</td><td rowspan=2 colspan=1>PanelMemberConcen-tration</td><td rowspan=2 colspan=1>$n^ a</td><td rowspan=2 colspan=1>PercentAgree-ment*((%))</td><td rowspan=2 colspan=1>MeanCt</td><td rowspan=1 colspan=2>BetweenSite</td><td rowspan=1 colspan=2>BetweenLot</td><td rowspan=1 colspan=2>BetweenDay</td><td rowspan=1 colspan=2>BetweenRuns</td><td rowspan=1 colspan=2>WithinRun</td><td rowspan=2 colspan=1>TotalSD</td><td rowspan=2 colspan=1>TotalCV(%)</td></tr><tr><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>CV(%)</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>CV(%)</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>CV(%)</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>CV(%)</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>CV(%)</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>214/216</td><td rowspan=1 colspan=1>99.1</td><td rowspan=1 colspan=1>nc</td><td rowspan=1 colspan=1>nc</td><td rowspan=1 colspan=1>nc</td><td rowspan=1 colspan=1>nc</td><td rowspan=1 colspan=1>nc</td><td rowspan=1 colspan=1>nc</td><td rowspan=1 colspan=1>nc</td><td rowspan=1 colspan=1>nc</td><td rowspan=1 colspan=1>nc</td><td rowspan=1 colspan=1>nc</td><td rowspan=1 colspan=1>nc</td><td rowspan=1 colspan=1>nc</td><td rowspan=1 colspan=1>nc</td></tr><tr><td rowspan=1 colspan=1>SARS-CoV-2</td><td rowspan=1 colspan=1>~0.3xLoD</td><td rowspan=1 colspan=1>45/216</td><td rowspan=1 colspan=1>20.8</td><td rowspan=1 colspan=1>33.6</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.11</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.35</td><td rowspan=1 colspan=1>1.1</td><td rowspan=1 colspan=1>0.37</td><td rowspan=1 colspan=1>1.1</td></tr><tr><td rowspan=1 colspan=1>SARS-CoV-2</td><td rowspan=1 colspan=1>~1xLoD</td><td rowspan=1 colspan=1>196/216</td><td rowspan=1 colspan=1>90.7</td><td rowspan=1 colspan=1>33.2</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.09</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.17</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>0.37</td><td rowspan=1 colspan=1>1.1</td><td rowspan=1 colspan=1>0.42</td><td rowspan=1 colspan=1>1.3</td></tr><tr><td rowspan=1 colspan=1>SARS-CoV-2</td><td rowspan=1 colspan=1>~3xLoD</td><td rowspan=1 colspan=1>216/216</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>32.2</td><td rowspan=1 colspan=1>0.05</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>0.02</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.03</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>0.24</td><td rowspan=1 colspan=1>0.8</td><td rowspan=1 colspan=1>0.25</td><td rowspan=1 colspan=1>0.8</td></tr><tr><td rowspan=1 colspan=1>pan-Sarbecovirus</td><td rowspan=1 colspan=1>~0.3xLoD</td><td rowspan=1 colspan=1>158/216</td><td rowspan=1 colspan=1>73.1</td><td rowspan=1 colspan=1>36.5</td><td rowspan=1 colspan=1>0.18</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.71</td><td rowspan=1 colspan=1>2.0</td><td rowspan=1 colspan=1>0.74</td><td rowspan=1 colspan=1>2.0</td></tr><tr><td rowspan=1 colspan=1>pan-Sarbecovirus</td><td rowspan=1 colspan=1>~1xLoD</td><td rowspan=1 colspan=1>214/216</td><td rowspan=1 colspan=1>99.1</td><td rowspan=1 colspan=1>35.4</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.67</td><td rowspan=1 colspan=1>1.9</td><td rowspan=1 colspan=1>0.67</td><td rowspan=1 colspan=1>1.9</td></tr><tr><td rowspan=1 colspan=1>pan-Sarbecovirus</td><td rowspan=1 colspan=1>~3xLoD</td><td rowspan=1 colspan=1>216/216</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>34.1</td><td rowspan=1 colspan=1>0.11</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>0.05</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.32</td><td rowspan=1 colspan=1>0.9</td><td rowspan=1 colspan=1>0.34</td><td rowspan=1 colspan=1>1.0</td></tr></table>

$\mathrm { C t } =$ cycle threshold, LoD $=$ limit of detection, $\mathrm { S D = }$ standard deviation, $C \mathrm { V } ( \% ) =$ percent coefficient of variation, SARS CoV $2 =$ severe acute respiratory syndrome coronavirus 2, nc $=$ not calculable Note: SARS-CoV-2 is a dual target assay. Inactivated viral culture material was diluted to ${ \sim } 0 . 3 / 1 / 3 \mathrm { x }$ LoD based on the target 2 (SARS-CoV-2) LoD. $^ { \mathrm { ~ a ~ } } _ { \mathrm { ~ n ~ } }$ is the number of positive tests which contribute Ct values to the analysis. $_ \mathrm { N }$ is the total number of valid tests for the panel member. b Percent agreement with expected results. c Two negative panel members were tested positive (one for each target). Sequencing showed that one of these samples was positive and the other was negative. The Ct values and the curve analysis of these samples suggest a low level of contamination during specimen handling.

# 2. Linearity:

Not applicable; this is a qualitative assay.

3. Analytical Specificity/Interference:

a) Cross-Reactivity:

The potential for cross-reactivity (analytical specificity) for cobas SARS-CoV-2 Qualitative assay was evaluated in a study testing samples containing organisms that are commonly found in the respiratory tract and might cause similar clinical symptoms as SARS-CoV-2. A panel of 47 viruses, bacteria, and fungi were included in the study, with each organism tested in three replicates. The organisms were spiked at concentrations of $1 \mathrm { x } 1 0 ^ { 5 }$ units/mL for viruses and $1 \mathrm { x } 1 0 ^ { 6 }$ units/mL for bacteria and fungi. Testing was performed with each potential interfering organism in the absence and presence of SARS-CoV-2 target (spiked at ${ \sim } 3 \mathrm { x L o D } )$ . No cross reactivity or microbial interference was observed with the organisms at the concentrations tested, as shown below. Additional in-silico analysis confirmed no homology was found between any primers or probes for Bordetella parapertussis, Aspergillus $s p$ , and Fusobacterium necrophorum, however no wet testing with these organisms was performed.

Summary of Analytical Cross-Reactivity for cobas SARS-CoV-2 Qualitative   

<table><tr><td colspan="1" rowspan="1">Microorganism</td><td colspan="1" rowspan="1">Concentration</td><td colspan="1" rowspan="1">Positivity Rate</td><td colspan="1" rowspan="1">Negativity rate</td></tr><tr><td colspan="1" rowspan="1">Human coronavirus 229E</td><td colspan="1" rowspan="1">1.0E+05 TCID50/mL</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Human coronavirus OC43</td><td colspan="1" rowspan="1">1.0E+05 TCID5o/mL</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Human coronavirus HKU1</td><td colspan="1" rowspan="1">1.0E+05 TCID5o/mL</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Human coronavirus NL63</td><td colspan="1" rowspan="1">1.0E+05 TCID50/mL</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">MERS coronavirus</td><td colspan="1" rowspan="1">1.0E+05 genomicequivalent/mL</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">SARS coronavirus</td><td colspan="1" rowspan="1">1.0E+05 PFU/mL</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%*</td></tr><tr><td colspan="1" rowspan="1">Adenovirus B (Type 34)</td><td colspan="1" rowspan="1">1.0E+05 TCID50/mL</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Bocavirus</td><td colspan="1" rowspan="1">1.0E+05 cp/mL</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Cytomegalovirus</td><td colspan="1" rowspan="1">1.0E+05 TCID5o/mL</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Epstein Barr virus</td><td colspan="1" rowspan="1">1.0E+05 cp/mL</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Human Metapneumovirus(hMPV)</td><td colspan="1" rowspan="1">1.0E+05 TCID50/mL</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Measles virus</td><td colspan="1" rowspan="1">1.0E+05 TCID50/mL</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Mumps virus</td><td colspan="1" rowspan="1">1.0E+05 TCID50/mL</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza virus Type 1</td><td colspan="1" rowspan="1">1.0E+05 TCID50/mL</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza virus Type 2</td><td colspan="1" rowspan="1">1.0E+05 TCID50/mL</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza virus Type 3</td><td colspan="1" rowspan="1">1.0E+05 TCID50/mL</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza virus Type 4</td><td colspan="1" rowspan="1">1.0E+05 TCID50/mL</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Influenza A (H1N1)</td><td colspan="1" rowspan="1">1.0E+05 TCID5o/mL</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Influenza A virus (H1N1-2009)</td><td colspan="1" rowspan="1">1.0E+05 TCID50/mL</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Influenza A virus (H1N3)</td><td colspan="1" rowspan="1">1.0E+05 TCID5o/mL</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Influenza A virus (H3N2)</td><td colspan="1" rowspan="1">1.0E+05 TCID5o/mL</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Influenza B</td><td colspan="1" rowspan="1">1.0E+05 TCID5o/mL</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Enterovirus E (Type 1)</td><td colspan="1" rowspan="1">1.0E+05 TCID5o/mL</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Parechovirus</td><td colspan="1" rowspan="1">1.0E+05 TCID5o/mL</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Respiratory syncytial virus</td><td colspan="1" rowspan="1">1.0E+05 PFU/mL</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Rhinovirus</td><td colspan="1" rowspan="1">1.0E+05 TCID5o/mL</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Candida albicans</td><td colspan="1" rowspan="1">1.0E+06 CFU/mL</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Chlamydia pneumoniae</td><td colspan="1" rowspan="1">1.0E+06 TCID5o/mL</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium diphtheriae</td><td colspan="1" rowspan="1">1.0E+06 CFU/mL</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">1.0E+06 CFU/mL</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Haemophilus influenzae</td><td colspan="1" rowspan="1">1.0E+06 CFU/mL</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus gasseri</td><td colspan="1" rowspan="1">1.0E+06 CFU/mL</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Legionella pneumophila</td><td colspan="1" rowspan="1">1.0E+06 CFU/mL</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Legionella jordanis (non-pneumophila)</td><td colspan="1" rowspan="1">1.0E+06 CFU/mL</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Moraxella catarrhalis</td><td colspan="1" rowspan="1">1.0E+06 CFU/mL</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Mycobacterium tuberculosis</td><td colspan="1" rowspan="1">1.0E+06 cells/mL</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Neisseria elongata</td><td colspan="1" rowspan="1">1.0E+06 CFU/mL</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Neisseria meningitidis</td><td colspan="1" rowspan="1">1.0E+06 CFU/mL</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">1.0E+06 CFU/mL</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Pneumocystis jirovecii</td><td colspan="1" rowspan="1">1:20 of Patient Sample</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">1.0E+06 CFU/mL</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus epidermidis</td><td colspan="1" rowspan="1">1.0E+06 CFU/mL</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Streptococcus pneumoniae</td><td colspan="1" rowspan="1">1.0E+06 CFU/mL</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Streptococcus pyrogenes</td><td colspan="1" rowspan="1">1.0E+06 CFU/mL</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Streptococcus salivarius</td><td colspan="1" rowspan="1">1.0E+06 CFU/mL</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Bordetella pertussis</td><td colspan="1" rowspan="1">1.0E+06 CFU/mL</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Mycoplasma pneumoniae</td><td colspan="1" rowspan="1">1.0E+06 CFU/mL</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td></tr></table>

\* Testing of SARS-CoV-1 generated an expected Pan-Sarbecovirus positive result.

# b) Interfering substances study:

The potential for interference for the cobas SARS-CoV-2 Qualitative assay was evaluated with substances that may be present in respiratory specimens. A total of 20 potentially interfering substances were tested at or above clinically relevant levels in negative simulated clinical matrix stabilized in UTM in absence and presence of SARS-CoV-2 target (spiked at ${ \sim } 3 \mathbf { x }$ LoD). Each sample was tested in 10 replicates. The following table provides the substances and concentrations tested. The FluMist nasal vaccine was not tested.

# Summary of Interfering Substances Study for Cobas SARS-CoV-2 Qualitative

<table><tr><td rowspan=2 colspan=1>Name</td><td rowspan=2 colspan=1>Active Ingredients</td><td rowspan=2 colspan=1>Testconcentration</td><td rowspan=2 colspan=1>TotalN</td><td rowspan=1 colspan=2>Positivity %</td></tr><tr><td rowspan=1 colspan=1>Target 1(SARS-CoV-2)</td><td rowspan=1 colspan=1>Target 2(Pan-Sarbeco)</td></tr><tr><td rowspan=1 colspan=1>Control</td><td rowspan=1 colspan=1>No Substance</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>Human Genomic DNA</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>20 ng/μl</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>Mucus</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>One sputumswab/mL</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>Human PBMCs</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>1.0E+03 cells/μL</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>Human Whole Blood</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>1%, 2% and 5%(V/v)</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>Afrin Nasal Spray</td><td rowspan=1 colspan=1>Oxymetazoline</td><td rowspan=1 colspan=1>0.011 mg/mL</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>Budesonide Nasal spray</td><td rowspan=1 colspan=1>Budesonide (glucocorticoid)</td><td rowspan=1 colspan=1>0.039 mg/mL</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>Flonase Nasal Spray</td><td rowspan=1 colspan=1>Fluticasone Propionate</td><td rowspan=1 colspan=1>0.167 mg/mL</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>Zicam Nasal Spray</td><td rowspan=1 colspan=1>Galphimia glauca, Luffaoperculata, Sabadilla</td><td rowspan=1 colspan=1>0.023 mg/mL</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>Saline Nasal Spray withPreservatives</td><td rowspan=1 colspan=1>0.65% NaCl, phenylcarbino,Benzalkonium chloride</td><td rowspan=1 colspan=1>1.0% (v/v)</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>Cepacol(Sore Throat Lozenges)</td><td rowspan=1 colspan=1>Benzocaine and menthol</td><td rowspan=1 colspan=1>5.0 mg/mL</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>Chloraseptic</td><td rowspan=1 colspan=1>Phenol</td><td rowspan=1 colspan=1>0.47 mg/mL</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>Liposomal NUMB520 Spray</td><td rowspan=1 colspan=1>Lidocaine and Phenylephrine</td><td rowspan=1 colspan=1>2.68 mg/mL</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>Mupirocin ointment UPS</td><td rowspan=1 colspan=1>Mupirocin</td><td rowspan=1 colspan=1>0.20 mg/mL</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>Relenza (Inhalation powder)</td><td rowspan=1 colspan=1>Zanamivir</td><td rowspan=1 colspan=1>0.0015 mg/mL</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>Antiviral drug - Tamiflu</td><td rowspan=1 colspan=1>Oseltamivir</td><td rowspan=1 colspan=1>0.0073 mg/mL</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>DMSO</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>Tobramycin ophthalmicSolution</td><td rowspan=1 colspan=1>Tobramycin</td><td rowspan=1 colspan=1>0.018 mg/mL</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>Vaseline</td><td rowspan=1 colspan=1>Petroleum Jelly</td><td rowspan=1 colspan=1>1% (w/v)</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>Snuff Tobacco</td><td rowspan=1 colspan=1>Nicotine</td><td rowspan=1 colspan=1>1% (w/v)</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>Analgesic ointment(Vicks@VapoRub)</td><td rowspan=1 colspan=1>Camphor-syntheticeucalyptus oil and mentholointment</td><td rowspan=1 colspan=1>1% (w/v)</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td></tr></table>

\* FluMist was not evaluated to assess potential interference.

# 4. Assay Reportable Range:

Not applicable; this is a qualitative assay.

5. Traceability, Stability, Expected Values (Controls, Calibrators, or Methods):

a) Quality Controls

The RNA Internal Control (RNA-IC), used to monitor the entire sample preparation and PCR amplification process, is introduced into each specimen during sample processing. In

addition, the test utilizes external controls- cobas SARS-CoV-2 Qualitative Control Ki t (low titer positive control) and cobas Buffer Negative Control Kit (negative control), provid ed separately. Three lots of Positive Control were included in the clinical evaluation of th e cobas SARS-CoV-2 Qualitative conducted at three testing sites. The summary of the C t values for valid positive and negative control results obtained for each target, combine d for sites and lots, is presented below.

Summary of the Ct values for Controls used in Clinical Studies   

<table><tr><td rowspan=1 colspan=1>Target Analyte</td><td rowspan=1 colspan=1>Total N</td><td rowspan=1 colspan=1>Mean Target Ct</td><td rowspan=1 colspan=1>Ct SD</td><td rowspan=1 colspan=1>Ct %CV</td></tr><tr><td rowspan=1 colspan=1>SARS-CoV-2</td><td rowspan=1 colspan=1>131</td><td rowspan=1 colspan=1>33.66</td><td rowspan=1 colspan=1>0.21</td><td rowspan=1 colspan=1>0.66</td></tr><tr><td rowspan=1 colspan=1>pan-Sarbeco</td><td rowspan=1 colspan=1>131</td><td rowspan=1 colspan=1>35.56</td><td rowspan=1 colspan=1>0.34</td><td rowspan=1 colspan=1>1.00</td></tr><tr><td rowspan=1 colspan=1>IC (positive samples)</td><td rowspan=1 colspan=1>131</td><td rowspan=1 colspan=1>33.23</td><td rowspan=1 colspan=1>0.22</td><td rowspan=1 colspan=1>0.66</td></tr><tr><td rowspan=1 colspan=1>IC (negative samples)</td><td rowspan=1 colspan=1>131</td><td rowspan=1 colspan=1>33.1</td><td rowspan=1 colspan=1>0.24</td><td rowspan=1 colspan=1>0.7</td></tr></table>

# b) Clinical Specimen stability:

The stability of SARS-CoV-2 viral targets during handling, transportation, and storage, was evaluated using NS and NPS specimens collected in different collection media. Five pools of NS specimens (collected in cobas PCR media, $0 . 9 \%$ saline and copan UTM) and five pools of NPS specimens (collected in UTM) were spiked with the inactivated quantified SARS$\mathrm { C o V - } 2$ virus at 3xLoD. Three replicates per specimen type/storage condition were used. The study data supported storage for specimens collected in UTM and $0 . 9 \%$ physiological saline at $2 { - } 2 5 ^ { \circ } \mathrm { C }$ for up to 48 hours followed by up to 3 days at $2 { - } 8 ^ { \circ } \mathrm { C }$ and at $\le - 7 0 ^ { \circ } \mathrm { C }$ for up to 30 days. Specimens collected in cobas PCR Media can be stored at 2- $2 5 \mathrm { { ^ \circ C } }$ for up to 24 hours followed by up to 3 days at $2 { - } 8 ^ { \circ } \mathrm { C }$ and at $\le - 7 0 ^ { \circ } \mathrm { C }$ for up to 30 days.

# 6. Detection Limit:

The Limit of Detection (LoD) for cobas SARS-CoV-2 Qualitative was determined using an inactivated quantified SARS-CoV-2 virus (WHO International Standard for SARS-CoV-2, NIBSC code: 20/146). LoD is defined as the lowest concentration of SARS-CoV-2 RNA that can be detected at a rate of at least $9 5 \%$ . A total of 5 concentration levels (500, 250, 125, 62.5 and $3 1 . 2 5 \ : \mathrm { I U / m l } $ ) were prepared by diluting the SARS-CoV-2 target in negative simulated clinical matrix stabilized in UTM. Three independent dilution series with three lots of reagents were tested with a total of 24 replicates per concentration.

The concentration level with observed hit rates greater than or equal to $9 5 \%$ was determined to be the LoD for each of the two targets (SARS-CoV-2 and pan-Sarbecovirus) as described below.

# Summary of LoD for SARS-CoV-2 (Target-1)

<table><tr><td rowspan=1 colspan=1>Viral Strain</td><td rowspan=1 colspan=1>Kit lot</td><td rowspan=1 colspan=1>Hit rate ≥ 95%[IU/mL](no. of replicatesdetected)</td><td rowspan=1 colspan=1>Mean Ct at ≥ 95% Hitrate</td></tr><tr><td rowspan=3 colspan=1>WHO InternationalStandard for SARS-CoV-2</td><td rowspan=1 colspan=1>Lot 1</td><td rowspan=1 colspan=1>250 (24/24)</td><td rowspan=1 colspan=1>33.2</td></tr><tr><td rowspan=1 colspan=1>Lot 2</td><td rowspan=1 colspan=1>125 (23/24)</td><td rowspan=1 colspan=1>34.1</td></tr><tr><td rowspan=1 colspan=1>Lot 3</td><td rowspan=1 colspan=1>250 (23/24)</td><td rowspan=1 colspan=1>33.2</td></tr></table>

The LoD was confirmed at $2 5 0 \mathrm { \ : I U / m L }$ for SARS-CoV-2 (Target 1). For all three reagent lots, at least 23/24 replicates detected the target at $2 5 0 ~ \mathrm { I U / m l }$ .

Summary of LoD for pan-Sarbecovirus (Target-2)   

<table><tr><td rowspan=1 colspan=1>Viral Strain</td><td rowspan=1 colspan=1>Kit lot</td><td rowspan=1 colspan=1>Hit rate ≥ 95%[IU/mL](no. of replicatesdetected)</td><td rowspan=1 colspan=1>Mean Ct at ≥ 95% Hitrate</td></tr><tr><td rowspan=3 colspan=1>WHO InternationalStandard for SARS-CoV-2</td><td rowspan=1 colspan=1>Lot 1</td><td rowspan=1 colspan=1>125 (24/24)</td><td rowspan=1 colspan=1>35.2</td></tr><tr><td rowspan=1 colspan=1>Lot 2</td><td rowspan=1 colspan=1>125 (24/24)</td><td rowspan=1 colspan=1>36.0</td></tr><tr><td rowspan=1 colspan=1>Lot 3</td><td rowspan=1 colspan=1>125 (23/24)</td><td rowspan=1 colspan=1>34.8</td></tr></table>

The LoD was confirmed at $1 2 5 \mathrm { \ : I U / m L }$ for pan-Sarbecovirus (Target 2). For all three reagent lots, at least 23/24 replicates detected the target at $1 2 5 \mathrm { I U / m l }$ .

# 7. Matrix equivalency study:

Because some analytical studies were conducted using simulated matrix, equivalency between simulated and real clinical matrix was evaluated. SARS-CoV-2 target was diluted to make the test panel of four concentration levels (0x, 0.3x, 1x and 3x LoD) in each of three matrices (simulated clinical matrix, nasopharyngeal swab specimen and nasal swab specimen). 25 replicates were tested per each combination.

# Summary of matrix equivalency study for cobas SARS-CoV-2 Qualitative

<table><tr><td colspan="1" rowspan="1">Sample TypeMatrix(Stabilized inUTM)</td><td colspan="1" rowspan="1">PanelMember</td><td colspan="1" rowspan="1">Level(x LoD)</td><td colspan="1" rowspan="1">Conc.IU/mL</td><td colspan="1" rowspan="1">TotalResults (N)</td><td colspan="1" rowspan="1">PositiveResults</td><td colspan="1" rowspan="1">Positivity%</td></tr><tr><td colspan="1" rowspan="4">Nasal Swab</td><td colspan="1" rowspan="1">Above LoD</td><td colspan="1" rowspan="1">~3x</td><td colspan="1" rowspan="1">750</td><td colspan="1" rowspan="1">25</td><td colspan="1" rowspan="1">25</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">At LoD</td><td colspan="1" rowspan="1">~1.0x</td><td colspan="1" rowspan="1">250</td><td colspan="1" rowspan="1">25</td><td colspan="1" rowspan="1">25</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Below LoD</td><td colspan="1" rowspan="1">~0.3x</td><td colspan="1" rowspan="1">83</td><td colspan="1" rowspan="1">25</td><td colspan="1" rowspan="1">25</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Ox</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">25</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="1">Above LoD</td><td colspan="1" rowspan="1">~3x</td><td colspan="1" rowspan="1">750</td><td colspan="1" rowspan="1">25</td><td colspan="1" rowspan="1">25</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">At LoD</td><td colspan="1" rowspan="1">~1.0x</td><td colspan="1" rowspan="1">250</td><td colspan="1" rowspan="1">25</td><td colspan="1" rowspan="1">25</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="2">NasopharyngealSwab</td><td colspan="1" rowspan="1">Below LoD</td><td colspan="1" rowspan="1">~0.3x</td><td colspan="1" rowspan="1">83</td><td colspan="1" rowspan="1">25</td><td colspan="1" rowspan="1">25</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">0x</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">25</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="1" rowspan="4">SimulatedClinical Matrix</td><td colspan="1" rowspan="1">Above LoD</td><td colspan="1" rowspan="1">~3x</td><td colspan="1" rowspan="1">750</td><td colspan="1" rowspan="1">25</td><td colspan="1" rowspan="1">25</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">At LoD</td><td colspan="1" rowspan="1">~1.0x</td><td colspan="1" rowspan="1">250</td><td colspan="1" rowspan="1">25</td><td colspan="1" rowspan="1">25</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Below LoD</td><td colspan="1" rowspan="1">~0.3x</td><td colspan="1" rowspan="1">83</td><td colspan="1" rowspan="1">25</td><td colspan="1" rowspan="1">24</td><td colspan="1" rowspan="1">96%</td></tr><tr><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">0x</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">25</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0%</td></tr></table>

The results showed the performance of the assay with simulated clinical matrix, nasopharyngeal swab, and nasal swab matrices is equivalent.

8. Collection media equivalency (UTM, cobas PCR Media and $0 . 9 \%$ physiological saline)

Equivalency between copan UTM, cobas PCR Media (CPM) and $0 . 9 \%$ physiological saline was evaluated during the development of the subject IVD device. Paired individual negative nasal swab samples were collected and tested before and after spiking heat inactivated SARS-CoV-2 virus (WHO International Standard). At least 20 low positive (1.5-2x LoD) and 10 moderate positive (4x LoD) samples was prepared for each collection media. All negative and positive samples generated expected results for SARS-CoV-2 in the three collection media tested. The mean Ct differences between the three collection media was less than 2.

# 9. Analytical Reactivity (Inclusivity):

The inclusivity of cobas SARS-CoV-2 Qualitative assay for the detection of SARS-CoV-2 was confirmed by testing nine SARS-CoV-2 strains, including six variant strains. Qualified stocks of different SARS-CoV-2 were used to prepare concentration levels at and around the LoD. Each level was tested with four replicates. The lowest concentration at which all four replicates tested positive are reported in below.

# Analytical Reactivity with SARS-CoV-2 Strains

<table><tr><td rowspan=1 colspan=1>SARS-CoV-2 Strain</td><td rowspan=1 colspan=1>Target Material</td><td rowspan=1 colspan=1>Concentration tested with100% Positivity</td></tr><tr><td rowspan=1 colspan=1>Hong Kong/VM20001061/2020</td><td rowspan=1 colspan=1>Inactivated Virus</td><td rowspan=1 colspan=1>1.06E+02 cp/mL</td></tr><tr><td rowspan=1 colspan=1>Italy-INMI1</td><td rowspan=1 colspan=1>Inactivated Virus</td><td rowspan=1 colspan=1>1.00E+02 cp/mL</td></tr><tr><td rowspan=1 colspan=1>USA-WA1/2020</td><td rowspan=1 colspan=1>Inactivated Virus</td><td rowspan=1 colspan=1>5.03E+01 cp/mL</td></tr><tr><td rowspan=1 colspan=1>UK (B.1.1.7)</td><td rowspan=1 colspan=1>Inactivated Virus</td><td rowspan=1 colspan=1>2.4E+01 cp/mL</td></tr><tr><td rowspan=1 colspan=1>Japan / Brazil (P.1)</td><td rowspan=1 colspan=1>Extracted RNA</td><td rowspan=1 colspan=1>1.9E+02 cp/mL</td></tr><tr><td rowspan=1 colspan=1>South Africa (B.1.351)</td><td rowspan=1 colspan=1>Inactivated Virus</td><td rowspan=1 colspan=1>2.4E+01 cp/mL</td></tr><tr><td rowspan=1 colspan=1>US NY (B.1.526)</td><td rowspan=1 colspan=1>Inactivated Virus</td><td rowspan=1 colspan=1>1.9E+02 cp/mL</td></tr><tr><td rowspan=1 colspan=1>India (B.1.617.1)</td><td rowspan=1 colspan=1>Inactivated Virus</td><td rowspan=1 colspan=1>2.5E+02 cp/mL</td></tr><tr><td rowspan=1 colspan=1>India (B.1.617.2)</td><td rowspan=1 colspan=1>Inactivated Virus</td><td rowspan=1 colspan=1>7.0E+01 cp/mL</td></tr></table>

In silico analysis of additional SARS-CoV-2 sequences indicates that ${ > } 9 9 . 9 \%$ of known sequences for SARS-CoV-2 have no changes in primer/probe binding sites at both target regions simultaneously. All sequences are predicted to be detected by at least one of the two target regions.

In silico Analysis of cobas SARS-CoV-2 Test Oligo Design as of October 15, 2022   

<table><tr><td rowspan=1 colspan=1>Assay</td><td rowspan=1 colspan=4>Orf1ab (Target 1)</td><td rowspan=1 colspan=4>E-gene (Target 2)</td><td rowspan=1 colspan=4>Orf1ab &amp; E-gene (Dual Targets)</td></tr><tr><td rowspan=1 colspan=1>Database</td><td rowspan=1 colspan=2>NCBI</td><td rowspan=1 colspan=2>GISAID</td><td rowspan=1 colspan=2>NCBI</td><td rowspan=1 colspan=2>GISAID</td><td rowspan=1 colspan=2>NCBI</td><td rowspan=1 colspan=2>GISAID</td></tr><tr><td rowspan=1 colspan=1>total</td><td rowspan=1 colspan=1>6113594</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>12695489</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>6113594</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>12695489</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>6113594</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>12695489</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Withmismatch</td><td rowspan=1 colspan=1>251910</td><td rowspan=1 colspan=1>4.12%</td><td rowspan=1 colspan=1>464658</td><td rowspan=1 colspan=1>3.66%</td><td rowspan=1 colspan=1>32124</td><td rowspan=1 colspan=1>0.53%</td><td rowspan=1 colspan=1>83852</td><td rowspan=1 colspan=1>0.66%</td><td rowspan=1 colspan=1>1125</td><td rowspan=1 colspan=1>0.02%</td><td rowspan=1 colspan=1>2971</td><td rowspan=1 colspan=1>0.02%</td></tr><tr><td rowspan=1 colspan=1>dCp&gt;5 orTm&lt;65</td><td rowspan=1 colspan=1>211</td><td rowspan=1 colspan=1>0.00%</td><td rowspan=1 colspan=1>493</td><td rowspan=1 colspan=1>0.00%</td><td rowspan=1 colspan=1>34</td><td rowspan=1 colspan=1>0.00%</td><td rowspan=1 colspan=1>1025</td><td rowspan=1 colspan=1>0.01%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.00%</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0.00%</td></tr></table>

In Silico Analysis of SARS-CoV-2 Variants of Concern (VOC) as of October 10, 2022   

<table><tr><td rowspan=1 colspan=1>WHO label</td><td rowspan=1 colspan=1>Status</td><td rowspan=1 colspan=1>Pango Lineage</td><td rowspan=1 colspan=1>Nextstrain Clade</td><td rowspan=1 colspan=1>cobas SARS CoV-2 foruse on the cobas6800/8800 Systems</td></tr><tr><td rowspan=1 colspan=1>Alpha</td><td rowspan=1 colspan=1>de-escalated</td><td rowspan=1 colspan=1>B.1.1.7</td><td rowspan=1 colspan=1>20/501Y.V1</td><td rowspan=1 colspan=1>detected</td></tr><tr><td rowspan=1 colspan=1>Beta</td><td rowspan=1 colspan=1>de-escalated</td><td rowspan=1 colspan=1>B.1.351</td><td rowspan=1 colspan=1>20H/501Y.V2</td><td rowspan=1 colspan=1>detected</td></tr><tr><td rowspan=1 colspan=1>Gamma</td><td rowspan=1 colspan=1>de-escalated</td><td rowspan=1 colspan=1>P.1</td><td rowspan=1 colspan=1>20J/501Y.V3</td><td rowspan=1 colspan=1>detected</td></tr><tr><td rowspan=1 colspan=1>Delta</td><td rowspan=1 colspan=1>Active VOC</td><td rowspan=1 colspan=1>B.1.617.2</td><td rowspan=1 colspan=1>21A,I,J</td><td rowspan=1 colspan=1>detected</td></tr><tr><td rowspan=1 colspan=1>Epsilon</td><td rowspan=1 colspan=1>de-escalated</td><td rowspan=1 colspan=1>B.1.427/B.1.429</td><td rowspan=1 colspan=1>20C/S:452R</td><td rowspan=1 colspan=1>detected</td></tr><tr><td rowspan=1 colspan=1>Zeta</td><td rowspan=1 colspan=1>de-escalated</td><td rowspan=1 colspan=1>P.2</td><td rowspan=1 colspan=1>20J</td><td rowspan=1 colspan=1>detected</td></tr><tr><td rowspan=1 colspan=1>Eta</td><td rowspan=1 colspan=1>de-escalated</td><td rowspan=1 colspan=1>B.1.525</td><td rowspan=1 colspan=1>20A/S:484K</td><td rowspan=1 colspan=1>detected</td></tr><tr><td rowspan=1 colspan=1>Theta</td><td rowspan=1 colspan=1>de-escalated</td><td rowspan=1 colspan=1>P.3</td><td rowspan=1 colspan=1>21E</td><td rowspan=1 colspan=1>detected</td></tr><tr><td rowspan=1 colspan=1>lota</td><td rowspan=1 colspan=1>de-escalated</td><td rowspan=1 colspan=1>B.1.526</td><td rowspan=1 colspan=1>20C/S:484K</td><td rowspan=1 colspan=1>detected</td></tr><tr><td rowspan=1 colspan=1>Kappa</td><td rowspan=1 colspan=1>de-escalated</td><td rowspan=1 colspan=1>B.1.617.1</td><td rowspan=1 colspan=1>20A/S:154K</td><td rowspan=1 colspan=1>detected</td></tr><tr><td rowspan=1 colspan=1>Lambda</td><td rowspan=1 colspan=1>de-escalated</td><td rowspan=1 colspan=1>C.37</td><td rowspan=1 colspan=1>20D</td><td rowspan=1 colspan=1>detected</td></tr><tr><td rowspan=1 colspan=1>Mu</td><td rowspan=1 colspan=1>de-escalated</td><td rowspan=1 colspan=1>B.1.621</td><td rowspan=1 colspan=1>21H</td><td rowspan=1 colspan=1>detected</td></tr><tr><td rowspan=1 colspan=1>Omicron (BA.1 - BA.5, BQ.1,BQ.1.1, BF.7)</td><td rowspan=1 colspan=1>Active VOC</td><td rowspan=1 colspan=1>B.1.1.529</td><td rowspan=1 colspan=1>21K,L,M22A,B,C,D,E</td><td rowspan=1 colspan=1>detected</td></tr></table>

# 10. Assay Cut-Off:

The assay cut-off for this device was established based on variety of parameter thresholds taken together by the algorithm to make the qualitative decision whether a curve is to be considered positive or negative. These parameters (e.g., Ct, max. RFI, max. slope, sigmoidality of curve shape) are initially set by the algorithm. As the product development progresses, the data is annotated, and the algorithm is adjusted (“trained”) accordingly. This iterative process in the end leads to a final set of parameter cut-offs. Selected data from testing of clinical specimens or cultures spiked into clinical background were used to confirm adequate separation between the latest Ct value observed in positive specimens in each target detection channel and the assay cutoff.

# 11. Carry-Over:

Carry-over/cross contamination rate for cobas SARS-CoV-2 Qualitative assay was evaluated by testing SARS-CoV-2 high-positive samples and negative samples in a checkerboard configuration. Five runs, with each run containing 48 replicates of SARS-CoV-2 negative samples and 46 replicates of high positive samples were performed. A total of 230 SARS-CoV-2 high positive samples and 239 negative samples were tested. No cross-contamination was observed, as described in the table below.

Cross Contamination Run Results   

<table><tr><td rowspan=2 colspan=1>Run Number</td><td rowspan=2 colspan=1>Total Positive Results fromNegative Samples</td><td rowspan=1 colspan=3>Total Samples</td><td rowspan=2 colspan=1>Cross Contamination Rate</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Controls</td></tr><tr><td rowspan=1 colspan=1>Run 1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>46</td><td rowspan=1 colspan=1>47</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0.0 %</td></tr><tr><td rowspan=1 colspan=1>Run 2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>46</td><td rowspan=1 colspan=1>48</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0.0 %</td></tr><tr><td rowspan=1 colspan=1>Run 3</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>46</td><td rowspan=1 colspan=1>48</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0.0 %</td></tr><tr><td rowspan=1 colspan=1>Run 4</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>46</td><td rowspan=1 colspan=1>48</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0.0 %</td></tr><tr><td rowspan=1 colspan=1>Run 5</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>46</td><td rowspan=1 colspan=1>48</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0.0 %</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>230</td><td rowspan=1 colspan=1>239</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>0.0 %(upper one-sided 95% Cl of 1.25%)</td></tr></table>

# B Comparison Studies:

1. Method Comparison with Predicate Device:

Not applicable. Refer to the Clinical Studies Section of this document.

# C Clinical Studies:

Clinical Study Design

The performance of cobas SARS-CoV-2 Qualitative was evaluated in a multi-center study with three external testing sites evaluating prospectively collected clinical specimens in UTM-RT or UVT from individuals with signs and symptoms of respiratory infection. Participants from 12 geographically distributed enrollment centers each provided nasopharyngeal swab (NPS) and nasal swab (NS, anterior nares) specimens as part of a dual collection where (a) the collection order (first specimen collected) was alternated between the NPS and NS specimen, and (b) the collection method for NS specimens was also alternated with $50 \%$ of the NS specimens were self-collected on-site with healthcare provider (HCP) instructions while the other $50 \%$ were collected by the healthcare provider. The study used a composite comparator method wherein laboratory sites used up to three highly sensitive EUA SARS-CoV-2 molecular assays, testing NPS specimen from each subject. The composite comparator result was defined as the concordant results from two comparator assays (test A and test B). In case of discordance between the initial two comparator assays, the sample was tested by a third assay (test C) and the result of the third test determined the composite comparator result. The composite comparator result was indetermined when valid results could not be obtained from two assays (i.e., insufficient volume for repeat testing of invalid/failed results).

From March to June 2021, a total of 1,154 participants were enrolled, of which samples from 968 participants were included in the evaluation. Samples from 186 participants were not included: 184 specimens were excluded due to issues associated with specimen shipments and/or being unable to complete testing within the times identified by manufacturer’s instructions, and two subjects were excluded for being previously enrolled in the study (exclusion criteria). When self-reporting COVID-19 vaccination status, 207 $( 2 1 . 4 \% )$ of the 968 participants were fully vaccinated.

Of the 968 participants, 961 contributed a NPS specimen which resulted in 942 participants with a confirmed infected status. For NPS, 4 specimens had failed/invalid cobas SARS-CoV-2 Qualitative results, resulting in 938 evaluable NPS results. For NS, 8 specimens were invalid/missing cobas SARS-CoV-2 Qualitative NS results, resulting in 934 evaluable results.

When compared with the NPS composite comparator result, cobas SARS-CoV-2 Qualitative yielded a positive percent agreement (PPA) of $9 8 . 7 \%$ for NPS and $9 7 . 4 \%$ for NS specimens. The negative percent agreement (NPA) was $9 9 . 7 \%$ and $9 9 . 9 \%$ for NPS and NS specimens, respectively.

Summary of clinical performance of cobas SARS-CoV-2 Qualitative for nasopharyngeal (NPS) and nasal swabs (NS) versus the NPS composite comparator   

<table><tr><td rowspan=1 colspan=1>Specimen Type</td><td rowspan=1 colspan=1>Total(N)</td><td rowspan=1 colspan=1>PPA</td><td rowspan=1 colspan=1>PPA 2-sided 95%Score Cl</td><td rowspan=1 colspan=1>NPA</td><td rowspan=1 colspan=1>NPA 2-sided 95%Score Cl</td></tr><tr><td rowspan=1 colspan=1>Nasopharyngeal(NPS)</td><td rowspan=1 colspan=1>938</td><td rowspan=1 colspan=1>98.7%(77/78)</td><td rowspan=1 colspan=1>(93.1 %, 99.8 %)</td><td rowspan=1 colspan=1>99.7%(857/860)</td><td rowspan=1 colspan=1>(99.0 %, 99.9 %)</td></tr><tr><td rowspan=1 colspan=1>Nasal Swab(NS)*</td><td rowspan=1 colspan=1>934</td><td rowspan=1 colspan=1>97.4%(76/78)</td><td rowspan=1 colspan=1>(91.1 %, 99.3 %)</td><td rowspan=1 colspan=1>99.9%(855/856)</td><td rowspan=1 colspan=1>(99.3 %, 100 %)</td></tr></table>

\*Healthcare provider-collected nasal swab specimens and nasal swab specimens self-collected on-site with healthcare provider instructions

# D Clinical Cut-Off:

Not applicable

# E Expected Values/Reference Range:

The SARS-CoV-2 positivity determined by the cobas SARS-CoV-2 Qualitative is presented by enrollment site in table below for both NPS and NS specimens. Across the 12 enrollment sites from 8 states, the SARS-CoV-2 positivity ranged from $0 \%$ to $1 6 . 2 \%$ for NPS and from $0 \%$ to $1 5 . 8 \%$ for NS.

SARS-CoV-2 Prevalence by Collection Site based on cobas SARS-CoV-2 Qualitative   

<table><tr><td rowspan=1 colspan=1>Site location</td><td rowspan=1 colspan=1>NPSresults</td><td rowspan=1 colspan=1>NPSPrevalence</td><td rowspan=1 colspan=1>NSresults</td><td rowspan=1 colspan=1>NSPrevalence</td></tr><tr><td rowspan=1 colspan=1>Oakland, CA</td><td rowspan=1 colspan=1>114</td><td rowspan=1 colspan=1>0.0% (0/114)</td><td rowspan=1 colspan=1>117</td><td rowspan=1 colspan=1>0.0% (0/117)</td></tr></table>

Food and Drug Administration 10903 New Hampshire Avenue Silver Spring, MD 20993-0002 www.fda.govK213804 - Page 19 of 20

<table><tr><td rowspan=1 colspan=1>Site location</td><td rowspan=1 colspan=1>NPSresults</td><td rowspan=1 colspan=1>NPSPrevalence</td><td rowspan=1 colspan=1>NSresults</td><td rowspan=1 colspan=1>NSPrevalence</td></tr><tr><td rowspan=1 colspan=1>St.Louis, MO</td><td rowspan=1 colspan=1>117</td><td rowspan=1 colspan=1>11.1% (13/117)</td><td rowspan=1 colspan=1>117</td><td rowspan=1 colspan=1>10.3% (12/117)</td></tr><tr><td rowspan=1 colspan=1>Syracuse, NY</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>9.1% (1/11)</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>9.1% (1/11)</td></tr><tr><td rowspan=1 colspan=1>Bronx, NY</td><td rowspan=1 colspan=1>64</td><td rowspan=1 colspan=1>7.8% (5/64)</td><td rowspan=1 colspan=1>64</td><td rowspan=1 colspan=1>9.4% (6/64)</td></tr><tr><td rowspan=1 colspan=1>Nashville, TN</td><td rowspan=1 colspan=1>92</td><td rowspan=1 colspan=1>12.0% (11/92)</td><td rowspan=1 colspan=1>93</td><td rowspan=1 colspan=1>11.8% (11/93)</td></tr><tr><td rowspan=1 colspan=1>Easley, SC</td><td rowspan=1 colspan=1>127</td><td rowspan=1 colspan=1>13.4% (17/127)</td><td rowspan=1 colspan=1>128</td><td rowspan=1 colspan=1>12.5% (16/128)</td></tr><tr><td rowspan=1 colspan=1>Powdersville, SC</td><td rowspan=1 colspan=1>117</td><td rowspan=1 colspan=1>16.2% (19/117)</td><td rowspan=1 colspan=1>114</td><td rowspan=1 colspan=1>15.8% (18/114)</td></tr><tr><td rowspan=1 colspan=1>Dallas, TX</td><td rowspan=1 colspan=1>147</td><td rowspan=1 colspan=1>6.8% (10/147)</td><td rowspan=1 colspan=1>147</td><td rowspan=1 colspan=1>6.8% (10/147)</td></tr><tr><td rowspan=1 colspan=1>Dallas, TX</td><td rowspan=1 colspan=1>31</td><td rowspan=1 colspan=1>0.0% (0/31)</td><td rowspan=1 colspan=1>31</td><td rowspan=1 colspan=1>0.0% (0/31)</td></tr><tr><td rowspan=1 colspan=1>New Orleans, LA</td><td rowspan=1 colspan=1>46</td><td rowspan=1 colspan=1>4.3% (2/46)</td><td rowspan=1 colspan=1>46</td><td rowspan=1 colspan=1>2.2% (1/46)</td></tr><tr><td rowspan=1 colspan=1>Statesville, NC</td><td rowspan=1 colspan=1>33</td><td rowspan=1 colspan=1>6.1% (2/33)</td><td rowspan=1 colspan=1>34</td><td rowspan=1 colspan=1>5.9% (2/34)</td></tr><tr><td rowspan=1 colspan=1>Moorseville, NC</td><td rowspan=1 colspan=1>42</td><td rowspan=1 colspan=1>0.0% (0/42)</td><td rowspan=1 colspan=1>42</td><td rowspan=1 colspan=1>0.0% (0/42)</td></tr><tr><td rowspan=1 colspan=1>ALL</td><td rowspan=1 colspan=1>941</td><td rowspan=1 colspan=1>8.5% (80/941)</td><td rowspan=1 colspan=1>944</td><td rowspan=1 colspan=1>8.2% (77/944)</td></tr></table>

Note: The table included all specimens with valid SARS-CoV-2 results on cobas SARS-CoV-2. A positive SARS-CoV-2 is defined based on the cobas SARS-CoV-2. ${ \mathsf { N P S } } =$ nasopharyngeal swab specimen; NS $=$ nasal swab specimen.

# F Other Supportive Instrument Performance Characteristics Data:

Not applicable

# VIII Proposed Labeling:

The labeling supports the finding of substantial equivalence for this device.

# IX Conclusion:

The submitted information in this premarket notification is complete and supports a substantial equivalence decision.