# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATION DECISION SUMMARY ASSAY ONLY

I Background Information:

A 510(k) Number K213236   
B Applicant GenMark Diagnostics, Incorporated   
C Proprietary and Established Names ePlex Blood Culture Identification Gram Negative (BCID-GN) Panel   
D Regulatory Information

<table><tr><td rowspan=1 colspan=1>ProductCode(s)</td><td rowspan=1 colspan=1>Classification</td><td rowspan=1 colspan=1>RegulationSection</td><td rowspan=1 colspan=1>Panel</td></tr><tr><td rowspan=1 colspan=1>PEN</td><td rowspan=1 colspan=1>Class II</td><td rowspan=1 colspan=1>21 CFR 866.3365 -Multiplex Nucleic AcidAssay For IdentificationOf Microorganisms AndResistance Markers FromPositive Blood Cultures</td><td rowspan=1 colspan=1>MI - Microbiology</td></tr></table>

# II Submission/Device Overview:

# A Purpose for Submission:

The purpose of this submission is to modify the ePlex Blood Culture Identification GramNegative (BCID-GN) Panel for use on the GenMark ePlex Instrument to include the detection of nucleic acids from additional strains of E. coli, Citrobacter, Enterococcus, and $P .$ . aeruginosa.

# B Measurand:

Nucleic acid sequences from the following gram negative bacteria, gram positive bacteria, fungi and antibiotic resistance markers: Acinetobacter baumannii, Bacteroides fragilis, Citrobacter, Cronobacter sakazakii, Enterobacter cloacae complex, Enterobacter (non-cloacae complex), Escherichia coli, Fusobacterium necrophorum, Fusobacterium nucleatum, Haemophilus influenzae, Klebsiella oxytoca, Klebsiella pneumoniae group, Morganella morganii, Neisseria meningitidis, Proteus, Proteus mirabilis, Pseudomonas aeruginosa, Salmonella, Serratia,

Serratia marcescens, Stenotrophomonas maltophilia, CTX-M, IMP, KPC, NDM, OXA, and VIM, Pan Gram-Positive, Pan Candida

# C Type of Test:

A multiplexed nucleic acid test intended for use with the GenMark ePlex instrument for the simultaneous qualitative in vitro detection and identification of multiple bacterial nucleic acids from positive blood culture specimens.

# III Intended Use/Indications for Use:

A Intended Use(s): See Indications for Use below.

# B Indication(s) for Use:

The GenMark ePlex Blood Culture Identification Gram-Negative (BCID-GN) Panel is a qualitative nucleic acid multiplex in vitro diagnostic test intended for use on GenMark’s ePlex Instrument for simultaneous qualitative detection and identification of multiple potentially pathogenic gram-negative bacterial organisms and, select determinants associated with antimicrobial resistance in positive blood culture. In addition, the ePlex BCID-GN Panel is capable of detecting, several gram-positive bacteria (Pan Gram-Positive assay) and several Candida species (Pan Candida assay) in positive blood culture. The ePlex BCID-GN test is performed directly on blood culture samples identified as positive by a continuous monitoring blood culture system and which contain gram-negative organism.

The following bacterial organisms and genes associated with antibiotic resistance are identified using the ePlex BCID-GN Panel: Acinetobacter baumannii, Bacteroides fragilis, Citrobacter, Cronobacter sakazakii, Enterobacter cloacae complex, Enterobacter (non-cloacae complex), Escherichia coli, Fusobacterium necrophorum, Fusobacterium nucleatum, Haemophilus influenzae, Klebsiella oxytoca, Klebsiella pneumoniae group, Morganella morganii, Neisseria meningitidis, Proteus, Proteus mirabilis, Pseudomonas aeruginosa, Salmonella, Serratia, Serratia marcescens, Stenotrophomonas maltophilia, CTX-M (blaCTX-M), IMP (blaIMP) , KPC (blaKPC) , NDM (blaNDM), OXA (blaOXA) (OXA-23 and OXA-48 groups only), and VIM (blaVIM).

The ePlex BCID-GN panel contains assays for the detection of genetic determinants associated with resistance to antimicrobial agents including CTX-M(blaCTX-M), which is associated with resistance to extended spectrum beta-lactamase (ESBL)-mediated resistance to penicillins, cephalosporins and monobactams, as well as OXA (blaOXA) (OXA-23 and OXA-48 groups only), KPC (blaKPC), and metallo-beta-lactamases IMP (blaIMP), VIM (blaVIM), and NDM (blaNDM), which is associated with carbapenemase-mediated resistance. The antimicrobial resistance gene detected may or may not be associated with the agent responsible for disease. Negative results for these select antimicrobial resistance assays do not indicate susceptibility, as there are multiple mechanisms of resistance in gram-negative bacteria.

The ePlex BCID-GN Panel also contains targets designed to detect a broad range of organisms with a potentially misleading Gram stain result or organisms that may be missed by Gram staining altogether, for example in the case of co-infections. These include a broad Pan GramPositive assay (which is designed to detect Bacillus cereus group, Bacillus subtilis group,

Enterococcus, Staphylococcus, and Streptococcus), as well as a Pan Candida assay, which is designed to detect four Candida species: Candida albicans, Candida glabrata, Candida krusei, and Candida parapsilosis.

The detection and identification of specific bacterial and fungal nucleic acids from individuals exhibiting signs and/or symptoms of bloodstream infection aids in the diagnosis of bloodstream infection when used in conjunction with other clinical information. The results from the ePlex BCID-GN Panel are intended to be interpreted in conjunction with Gram stain results and should not be used as the sole basis for diagnosis, treatment, or other patient management decisions.

Negative results in the setting of a suspected bloodstream infection may be due to infection with pathogens that are not detected by this test. Positive results do not rule out co-infection with other organisms; the organism(s) detected by the ePlex BCID-GN Panel may not be the definite cause of disease. Additional laboratory testing (e.g. sub-culturing of positive blood cultures for identification of organisms not detected by ePlex BCID-GN Panel and for susceptibility testing, differentiation of mixed growth, and association of antimicrobial resistance marker genes to a specific organism) and clinical presentation must be taken into consideration in the final diagnosis of bloodstream infection.

C Special Conditions for Use Statement(s): Rx - For Prescription Use Only

See K182619.

D Special Instrument Requirements:

GenMark ePlex Instrument and Software

# IV Device/System Characteristics:

# A Device Description:

See K182619 for additional details. The ePlex Blood Culture Identification Gram-Negative (BCID-GN) Panel is designed for use with the GenMark ePlex Instrument. The original BCIDGN Panel was cleared for identification of microorganisms and resistance markers from positive blood cultures in K182619. The modified BCID-GN panel adds primers and/or probes to improve inclusivity of four assays in the original panel: E.coli, Pseudomonas aeruginosa, Citrobacter and Enterococcus. The Panel is contained on a consumable cartridge which consists of a printed circuit board (PCB) surface electrodes. This single use cartridge incorporates digital microfluidics and eSensor technology to automate all assay steps. The cartridge chamber contains immiscible fluid to facilitate sample transport and contains reagent droplets for sample processing.

Nucleic acid extraction from whole blood samples occurs within the cartridge via cell lysis, nucleic acid capture onto magnetic beads, and release for amplification. The nucleic acid extraction is processed through microfluidic liquid handling. Once the nucleic acid targets are captured and inhibitors are washed away, the magnetic particles are delivered to the electrowetting environment on the PCB and the targets are eluted from the particles and amplified by RT-PCR.

After amplification, the double-stranded PCR amplicons are digested with exonuclease to generate single-stranded DNA suitable for hybridization. During hybridization, the singlestranded target DNA binds to a complementary, single-stranded capture probe immobilized on the working gold electrode surface. Single-stranded signal probes (labeled with electrochemically active ferrocenes) bind to specific target sequence / region adjacent to the capture probe. Simultaneous hybridization of target to signal probes and capture probe is detected by alternating current voltammetry (ACV). Each working electrode on the array contains specific capture probes, and sequential analysis of each electrode allows detection of multiple analyte targets.

A summary of microorganisms and resistance markers detected by the ePlex BCID-GN Panel are presented in Table 1:

Table 1: ePlex BCID-GN Panel, organism and resistance marker targets   

<table><tr><td rowspan=1 colspan=2>Bacterial Targets</td></tr><tr><td rowspan=1 colspan=1>Acinetobacter baumanii</td><td rowspan=1 colspan=1>Klebsiella pneumonia group</td></tr><tr><td rowspan=1 colspan=1>Bacteroides fragilis</td><td rowspan=1 colspan=1>Morganella morganii</td></tr><tr><td rowspan=1 colspan=1>Citrobacter</td><td rowspan=1 colspan=1>Neisseria meningitidis</td></tr><tr><td rowspan=1 colspan=1>Cronobacter sakazakii</td><td rowspan=1 colspan=1>Proteus</td></tr><tr><td rowspan=1 colspan=1>Enterobacter cloacae complex</td><td rowspan=1 colspan=1>Proteus mirabilis</td></tr><tr><td rowspan=1 colspan=1>Enterobacter (non-cloacae complex)</td><td rowspan=1 colspan=1>Pseudomonas aeruginosa</td></tr><tr><td rowspan=1 colspan=1>Escherichia coli</td><td rowspan=1 colspan=1>Salmonella</td></tr><tr><td rowspan=1 colspan=1>Fusobacterium necrophorum</td><td rowspan=1 colspan=1>Serratia</td></tr><tr><td rowspan=1 colspan=1>Fusobacterium nucleatum</td><td rowspan=1 colspan=1>Serratia marcescens</td></tr><tr><td rowspan=1 colspan=1>Haemophilus influenzae</td><td rowspan=1 colspan=1>Stenotrophomonas maltophilia</td></tr><tr><td rowspan=1 colspan=1>Klebsiella oxytoca</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=2>Antimicrobial Resistance Markers</td></tr><tr><td rowspan=1 colspan=1>CTX-M (blaCTX-M)</td><td rowspan=1 colspan=1>NDM (blaNDM)</td></tr><tr><td rowspan=1 colspan=1>IMP (blaIMP)</td><td rowspan=1 colspan=1>OXA (blaOXA)</td></tr><tr><td rowspan=1 colspan=1>KPC (blaKPC)</td><td rowspan=1 colspan=1>VIM (blaVIM)</td></tr><tr><td rowspan=1 colspan=2>Pan Targets</td></tr><tr><td rowspan=1 colspan=1>Pan Gram-Positive</td><td rowspan=1 colspan=1>Pan Candida</td></tr></table>

# Materials provided

ePlex Blood Culture Identification Gram-Negative Panel kit containing 12 individual test cartridges. Storage is at $2 { - } 8 ^ { \circ } \mathrm { C }$ .

# Materials needed but not provided

Equipment

• GenMark ePlex instrument and software • Pipettes calibrated to deliver $5 0 \mu \mathrm { L }$ Printer (optional)

# B Principle of Operation:

See K182619 for additional details. The test is performed directly on blood culture samples identified as positive by continuous monitoring blood culture system that demonstrates the presence of organisms as confirmed by Gram stain. After the sample is loaded onto the single use ePlex cartridge, no further operator intervention is necessary, and the following automated procedures occur inside the cartridge. Nucleic acids are extracted and purified via magnetic solid phase extraction. For each of the BCID-GN Panel analytes, there are one or more forward or reverse primer pairs. PCR amplification of DNA targets occurs in eight reaction mixes or drops. After amplification double stranded DNA is digested with lambda exonuclease to generate single stranded DNA, to which ferrocene-labeled signal probes and gold electrode-bound capture probes hybridize. The presence of each target is determined by voltammetry which generates specific electrical signals from ferrocene-labeled signal probe which is measured by gold electrodes. Sequential analysis of each electrode allows detection of multiple analyte targets. The ePlex Instrument measures and interprets this electrical output to determine the results for each target on the BCID-GN Panel (Detected or Not Detected).

# V Substantial Equivalence Information:

A Predicate Device Name(s): ePlex Blood Culture Identification Gram Negative (BCID-GN) Panel   
B Predicate 510(k) Number(s): K182619

C Comparison with Predicate(s):

<table><tr><td colspan="1" rowspan="1">Item:</td><td colspan="1" rowspan="1">Predicate DeviceK182619</td><td colspan="1" rowspan="1">DeviceK213236</td></tr><tr><td colspan="1" rowspan="1">Device Trade Name</td><td colspan="1" rowspan="1">ePlex BCID-GN Panel</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Manufacturer</td><td colspan="1" rowspan="1">GenMark Diagnostics, Inc.</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="3" rowspan="1">General Device Characteristic Similarities</td></tr><tr><td colspan="1" rowspan="1">Intended Use</td><td colspan="1" rowspan="1">The ePlex Blood Culture IdentificationGram-Negative (BCID-GN) Panel is aqualitative nucleic acid multiplex invitro diagnostic test intended for useon GenMark's ePlex instrument forsimultaneous detection andidentification of multiple potentiallypathogenic gram negative bacterialorganisms and select determinants ofantimicrobial resistance in positiveblood culture. In addition, the ePlexPanel is capable of detecting severalgram-positive bacteria (Pan Gram-Positive assay), and several Candidaspecies (Pan Candida assay). TheePlex BCID-GN Panel is performeddirectly on blood culture samplesidentified as positive by a continuousmonitoring blood culture system thatdemonstrates the presence oforganisms as confirmed by Gramstain.</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Resistance GenesDetected</td><td colspan="1" rowspan="1">CTX-M, IMP, KPC, NDM, OXA andVIM</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Specimen Type</td><td colspan="1" rowspan="1">Blood culture samples identified aspositive by a continuous monitoringblood culture system that demonstratesthe presence of organisms (Gram-positive, Gram-negative, yeasts) asconfirmed by Gram stain</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Chemistry</td><td colspan="1" rowspan="1">Reagents on cartridge include: samplelysis and nucleic acid extraction, PCRamplification and hybridization-basedelectrochemical detection</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Hardware</td><td colspan="1" rowspan="1">GenMark ePlex Instrument &amp; SingleUse Cartridge</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Software</td><td colspan="1" rowspan="1">GenMark ePlex System SoftwareGenMark ePlex BCID-GN PanelSoftware</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="3" rowspan="1">General Device Characteristic Differences</td></tr><tr><td colspan="1" rowspan="1">Organisms Detected</td><td colspan="1" rowspan="1">Acinetobacter baumannii, Bacteroidesfragilis, Citrobacter, Cronobactersakazakii, Enterobacter cloacaecomplex, Enterobacter (non-cloacaecomplex), Escherichia coli,Fusobacterium necrophorum,Fusobacterium nucleatum,Haemophilus influenzae, Klebsiellapneumoniae, Klebsiella oxytoca,Morganella morganii, Neisseriameningitidis, Proteus, Proteusmirabilis, Pseudomonas aeruginosa,Salmonella, Serratia, Serratiamarcescens, Stenotrophomonasmaltophilia, Pan Gram-Positive(selected organisms), Pan Candida(selected species)</td><td colspan="1" rowspan="1">Additional strainsdetected:H. influenzae(ATCC33930)N. meningitidis(NCTC10026)E. coli (JHU01-D80401147)P. aeruginosa(SDx071)</td></tr></table>

# VI Standards/Guidance Documents Referenced:

Class II Special Controls Guideline: Multiplex Nucleic Acid Assay for Identification of Microorganisms and Resistance Markers from Positive Blood Cultures   
CLSI MM17-A, Vol. 9, Verification and Validation of Multiplex Nucleic Acid Assays CLSI EP17-A2 Evaluation of Detection Capability for Clinical Laboratory Measurement Procedures; Approved Guideline-Second Edition   
CLSI EP07-A2: Interference Testing in Clinical Chemistry; Approved Guideline-Second Edition

CLSP EP-25A Evaluation of Stability of In Vitro Diagnostic Reagents; Approved Guideline

# VII Performance Characteristics (if/when applicable):

# A Analytical Performance:

1. Linearity: Not applicable.

2. Analytical Specificity/Interference:

# Analytical Specificity/Cross-reactivity:

Analytical specificity of all oligonucleotides in the original ePlex BCID-GN Panel was evaluated previously (refer to K182619). An in silico analysis was conducted to evaluate offpanel and on-panel targets for potential cross-reactivity with the newly added primers/probes.

A BLASTN search for the amplicon sequence matching newly added primers and/or probes while filtering out the correctly identified species ( $P .$ . aeruginosa, E. coli, Citrobacter or Enterococcus) was performed. Results for organisms that are predicted to generate false positive results with the ePlex BCID-GN Panel are presented in Table 2.

Table 2: Cross-reactivity with new oligonucleotides, in silico analysis   

<table><tr><td rowspan=1 colspan=1>ePlex BCID-GNtarget</td><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>No. ofSequences</td><td rowspan=1 colspan=1>PredictedCross-reactiveSequences (%)</td></tr><tr><td rowspan=1 colspan=1>P. aeruginosa</td><td rowspan=1 colspan=1>None</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td></tr><tr><td rowspan=9 colspan=1>E.coli</td><td rowspan=1 colspan=1>Shigella flexneri</td><td rowspan=1 colspan=1>63</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Shigella sonnei</td><td rowspan=1 colspan=1>44</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Shigella boydii</td><td rowspan=1 colspan=1>26</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Shigella dysenteriae</td><td rowspan=1 colspan=1>29</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>unclassified Shigella</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>unclassified Escherichia</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Salmonella sp. HNK130</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Salmonella sp. S13</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Enterobacter hormaechei strainRHBSTW-00218</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Citrobacter</td><td rowspan=1 colspan=1>E. coli strain Colony214</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=3 colspan=1>Enterococcus</td><td rowspan=1 colspan=1>Tetragenococcus halophilus</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>0%</td></tr><tr><td rowspan=1 colspan=1>Lactococcus lactis</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Lactobacilus sp. Koumiss</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>100%</td></tr></table>

3. Assay Reportable Range:

Not Applicable

4. Traceability, Stability, Expected Values (Controls, Calibrators, or Methods): See K182619.

5. Detection Limit:

The limit of detection (LoD) for each target on the ePlex BCID-GN Panel was determined previously (refer to K182619).

For the new target strains that were not tested with the original panel (E.coli JHU01- D80401147, $P .$ . aeruginosa SDx071, H. influenzae ATCC33930, and N. meningitidis NCTC10026), LoD was determined using samples prepared in negative blood culture matrix and tested with the ePlex BCID-GN Panel. Organisms were pooled together and serially diluted to create 11 organism mixes containing two organisms in negative blood culture matrix. At least ten replicates were tested at four dilutions and the lowest concentration that resulted in $2 9 5 \%$ detection was selected as the estimated LoD. A minimum of 20 replicates per organism mix were tested to confirm the final LoD for each assay target. The limit of detection was defined as the lowest concentration at which each target is detected at least $9 5 \%$ of the time. Results from the study demonstrated that all targets were detected i $n \geq 9 5 \%$ of tested replicates at $\leq 1 \mathrm { ~ x ~ } 1 0 ^ { 8 } \mathrm { C F U / m L }$ (see Table 3).

Table 3: LoD results summary for additional strains   

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>LoD Concentration(CFU/ml)</td></tr><tr><td rowspan=1 colspan=1>H. influenzae ATCC33930</td><td rowspan=1 colspan=1>1 x 104</td></tr><tr><td rowspan=1 colspan=1>N. meningitidis NCTC10026</td><td rowspan=1 colspan=1>1 x 104</td></tr><tr><td rowspan=1 colspan=1>E.coli JHU01-D80401147</td><td rowspan=1 colspan=1>1 x 107</td></tr><tr><td rowspan=1 colspan=1>P.aeruginosa SDx071</td><td rowspan=1 colspan=1>1 x 105</td></tr></table>

Additional wet testing was conducted to assess the impact of the newly incorporated primers and probes on the LoD for the organism targets on the BCID-GN Panel. LoD verification testing was conducted to evaluate the original organism targets of the modified PCR mixes: Citrobacter sp., $E$ . coli, $P _ { - }$ . mirabilis, $A$ . baumannii, Serratia sp., Enterococcus sp., $P$ . aeruginosa, $H .$ . influenzae, $N .$ meningitidis, $M .$ morganii, $K .$ . pneumoniae, E. faecalis. Panel targets were tested in twenty replicates each of at least two different strains per target organisms at the LoD concentration $\mathrm { ( C F U / m L ) }$ ) previously determined for the BCID-GN panel (refer to K182619). Two strains from the original study were not available for verification of LoD and were replaced with new strains: $H .$ influenzae ATCC33930 (replacing ATCC19418 evaluated with the original BCID-GN panel), N. meningitidis NCTC10026 (replacing ATCC13102 evaluated with the original BCID-GN panel). All organisms were detected $\mathrm { i n } \geq 9 5 \%$ of tested replicates at $\leq 1 \mathrm { ~ x ~ } 1 0 ^ { 8 } \mathrm { C F U / m L }$ with the modified panel. The results are shown in Table 4.

Table 4: LoD verification results summary   

<table><tr><td colspan="1" rowspan="1">Target</td><td colspan="1" rowspan="1">Strain</td><td colspan="1" rowspan="1">LoD Concentration(CFU/ml)</td></tr><tr><td colspan="1" rowspan="2">Acinetobacter baumannii</td><td colspan="1" rowspan="1">NCTC13421</td><td colspan="1" rowspan="1">1x106</td></tr><tr><td colspan="1" rowspan="1">NCTC13304</td><td colspan="1" rowspan="1">1x106</td></tr><tr><td colspan="1" rowspan="2">Citrobacter</td><td colspan="1" rowspan="1">NCTC9750</td><td colspan="1" rowspan="1">1x106</td></tr><tr><td colspan="1" rowspan="1">ATCC27156</td><td colspan="1" rowspan="1">1x106</td></tr><tr><td colspan="1" rowspan="3">E. coli</td><td colspan="1" rowspan="1">NCTC13441</td><td colspan="1" rowspan="1">1x106</td></tr><tr><td colspan="1" rowspan="1">CDC#0118</td><td colspan="1" rowspan="1">1x107</td></tr><tr><td colspan="1" rowspan="1">JHU01-D80401147*</td><td colspan="1" rowspan="1">1x107</td></tr><tr><td colspan="1" rowspan="2">H. influenzae</td><td colspan="1" rowspan="1">ATCC9006</td><td colspan="1" rowspan="1">1x107</td></tr><tr><td colspan="1" rowspan="1">ATCC33930*</td><td colspan="1" rowspan="1">1x104</td></tr><tr><td colspan="1" rowspan="2">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">CDC#0106</td><td colspan="1" rowspan="1">1x106</td></tr><tr><td colspan="1" rowspan="1">CDC#0107</td><td colspan="1" rowspan="1">1x106</td></tr><tr><td colspan="1" rowspan="2">Morganella morganii</td><td colspan="1" rowspan="1">ATCC25829</td><td colspan="1" rowspan="1">1x107</td></tr><tr><td colspan="1" rowspan="1">CDC#0133</td><td colspan="1" rowspan="1">1x107</td></tr><tr><td colspan="1" rowspan="2">Neisseria meningitidis</td><td colspan="1" rowspan="1">ATCC13090</td><td colspan="1" rowspan="1">1x105</td></tr><tr><td colspan="1" rowspan="1">NCTC10026*</td><td colspan="1" rowspan="1">1x104</td></tr><tr><td colspan="1" rowspan="2">Proteus mirabilis</td><td colspan="1" rowspan="1">CDC#0159</td><td colspan="1" rowspan="1">1x106</td></tr><tr><td colspan="1" rowspan="1">ATCC43071</td><td colspan="1" rowspan="1">1x106</td></tr><tr><td colspan="1" rowspan="3">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">CDC#0103</td><td colspan="1" rowspan="1">1x106</td></tr><tr><td colspan="1" rowspan="1">NCTC13437</td><td colspan="1" rowspan="1">1x106</td></tr><tr><td colspan="1" rowspan="1">SDx071*</td><td colspan="1" rowspan="1">1x105</td></tr><tr><td colspan="1" rowspan="2">Serratia</td><td colspan="1" rowspan="1">ATCC27592</td><td colspan="1" rowspan="1">1x106</td></tr><tr><td colspan="1" rowspan="1">ATCC53858</td><td colspan="1" rowspan="1">1x107</td></tr><tr><td colspan="1" rowspan="2">Pan Gram positive</td><td colspan="1" rowspan="1">ATCC51575 (E. faecalis)</td><td colspan="1" rowspan="1">1x105</td></tr><tr><td colspan="1" rowspan="1">ATCC31282 (E. faecium)</td><td colspan="1" rowspan="1">1x107</td></tr><tr><td colspan="1" rowspan="2">CTX</td><td colspan="1" rowspan="1">CDC#0107 (Klebsiella pneumoniae)</td><td colspan="1" rowspan="1">1x106</td></tr><tr><td colspan="1" rowspan="1">NCTC13441 (E. coli)</td><td colspan="1" rowspan="1">1x106</td></tr><tr><td colspan="1" rowspan="1">IMP</td><td colspan="1" rowspan="1">CDC#0103 (P. aeruginosa)</td><td colspan="1" rowspan="1">1x106</td></tr><tr><td colspan="1" rowspan="1">KPC</td><td colspan="1" rowspan="1">CDC#0133 (M. morganii)</td><td colspan="1" rowspan="1">1x107</td></tr><tr><td colspan="1" rowspan="1">NDM</td><td colspan="1" rowspan="1">CDC#0159 (P. mirabilis)</td><td colspan="1" rowspan="1">1x106</td></tr><tr><td colspan="1" rowspan="2">OXA-23</td><td colspan="1" rowspan="1">NCTC13421 (A. baumannii. OXA- 23)</td><td colspan="1" rowspan="1">1x106</td></tr><tr><td colspan="1" rowspan="1">NCTC13304 (A. baumannii, OXA-27)</td><td colspan="1" rowspan="1">1x107</td></tr><tr><td colspan="1" rowspan="1">OXA-48</td><td colspan="1" rowspan="1">CDC#0160 (K. pneumoniae, OXA-48)</td><td colspan="1" rowspan="1">1x106</td></tr></table>

\*LoD established for additional target strains using a range finding study and confirmation of detection $2 9 5 \%$ of the time.

Additional evaluations were performed to demonstrate that the incorporation of the oligonucleotides in two PCR reaction mixes do not adversely impact the performance of the assay targets in other PCR mixes on the BCID-GN. Specifically, performance was assessed for several targets using a multianalyte test mix that contained one representative target from each of the eight multiplex PCR mixes in the previously cleared BCID-GN panel. At least 20 replicates per target were tested at the following three concentrations: HIGH (target levels 10X above LoD), MED (target levels near the LoD) and LOW (target levels 10X below the LoD).

The study acceptance criteria were as follows: detection of $2 9 5 \%$ replicates at HIGH and MED concentrations and detection of ${ \leq } 9 5 \%$ replicates at the LOW concentration. Results from the study are shown in Table 5.

Table 5: Detection rates of all targets at indicated concentrations   

<table><tr><td rowspan=2 colspan=1>Organism</td><td rowspan=1 colspan=3>Test concentrationCFU/mL (% detected)</td></tr><tr><td rowspan=1 colspan=1>HIGH</td><td rowspan=1 colspan=1>MED</td><td rowspan=1 colspan=1>LOW</td></tr><tr><td rowspan=1 colspan=1>E. coli</td><td rowspan=1 colspan=1>1x107(100%)</td><td rowspan=1 colspan=1>1x106(100%)</td><td rowspan=1 colspan=1>1x105(70%)</td></tr><tr><td rowspan=1 colspan=1>CTX-M</td><td rowspan=1 colspan=1>1x107(100%)</td><td rowspan=1 colspan=1>1x106(100%)</td><td rowspan=1 colspan=1>1x105}$(80%)</td></tr><tr><td rowspan=1 colspan=1>S. marcescens</td><td rowspan=1 colspan=1>1x106(100%)</td><td rowspan=1 colspan=1>1x105}$(100%)</td><td rowspan=1 colspan=1>1x104(60%)</td></tr><tr><td rowspan=1 colspan=1>S.aureus</td><td rowspan=1 colspan=1>2x106(100%)</td><td rowspan=1 colspan=1>2x10^r}$(100%)</td><td rowspan=1 colspan=1>2x104(20%)</td></tr><tr><td rowspan=1 colspan=1>H. influenzae</td><td rowspan=1 colspan=1>6x106(100%)</td><td rowspan=1 colspan=1>6x10^}$(100%)</td><td rowspan=1 colspan=1>6x104(80%)</td></tr><tr><td rowspan=1 colspan=1>K.a oxytoca</td><td rowspan=1 colspan=1>1x107(100%)</td><td rowspan=1 colspan=1>1x106(95%)</td><td rowspan=1 colspan=1>1x105(25%)</td></tr><tr><td rowspan=1 colspan=1>KPC</td><td rowspan=1 colspan=1>1x107(100%)</td><td rowspan=1 colspan=1>1x106(100%)</td><td rowspan=1 colspan=1>1x105(80%)</td></tr><tr><td rowspan=1 colspan=1>C. albicans*</td><td rowspan=1 colspan=1>1x106*(100%)</td><td rowspan=1 colspan=1>1x106(95%)</td><td rowspan=1 colspan=1>1x105(80%)</td></tr></table>

\* The maximum concentration possible for $C .$ . albicans is $1 \mathrm { x } 1 0 ^ { 6 } \mathrm { C F U / m L }$ , and this concentration represents both the bottle positive concentration and LoD concentration. Therefore, the same concentration was used for the HIGH and MED levels for this organism.

# Analytical Reactivity/Inclusivity:

Inclusivity of all oligonucleotides in the original BCID-GN Panel was evaluated previously (refer to K182619). The performance of the modified ePlex BCID-GN Panel for the detection of strains targeted by the modified PCR mixes was evaluated in a clinical agreement study to support inclusivity. The study included testing with multiple unique clinical specimens and contrived specimens prepared from unique isolates targeted by the PCR mixes that were modified by the addition of new oligonucleotides. A total of 122 clinical samples collected from multiple sites and 138 individually contrived blood culture bottles were used. A minimum of ten unique clinical patient samples or strains were tested per target, with the exception of Neisseria and $H .$ influenzae for which six and eight samples were evaluated, respectively. The results showed acceptable positive percent agreement (PPA) and negative percent agreement (NPA) of $9 5 \%$ or more for all targets. These results support the inclusivity of the modified panel. See Table 7 below.

6. Assay Cut-Off:

See K182619.

# B Comparison Studies:

1. Method Comparison with Predicate Device: Not applicable.

2. Matrix Comparison: Not applicable.

# C Clinical Studies:

1. Clinical Sensitivity:

Clinical validation of the BCID-GN Panel was established in a previous clinical study testing prospective, retrospective, and contrived specimens (refer to K182619). To evaluate the impact of new assay primers and probes, additional clinical and contrived specimens were tested as described below.

New oligonucleotides were added to PCR reaction mixes that include the following organism targets: Acinetobacter baumannii, Citrobacter, Escherichia coli, Haemophilus influenzae, Klebsiella pneumoniae, Morganella morganii, Neisseria meningitidis, Proteus mirabilis, Pseudomonas aeruginosa, Serratia, Pan-Gram positive Enterococcus faecalis, and Pan-Gram positive Enterococcus. The new oligonucleotides were added to detect additional strains of $E$ . coli (JHU01-D80401147) and Pseudomonas aeruginosa (SDx071) and to increase detection of Citrobacter and Enterococcus strains.

Additional testing was conducted to demonstrate that the modified ePlex BCID-GN Panel detects the targeted strains not detected by the original ePlex BCID-GN Panel and to verify that the clinical performance of the modified ePlex BCID-GN Panel for other organism targets is not adversely impacted by the introduction of the additional oligonucleotides. The performance of the modified device was evaluated by testing a total of 20 samples for each organism target that included a combination of unique clinical specimens and contrived blood culture specimens as indicated in Table 6 and Table 7. The identity of strain $E .$ . coli JHU01-D80401147 was confirmed by PCR and sequencing. The other organism targets evaluated in the study were obtained from previously collected frozen samples. Equivalence between fresh and frozen specimens on the ePlex BCID-GN Panel was previously established in K182619.

Contrived specimens were prepared by independently inoculating low concentrations of organism into blood culture bottles and incubating on a continuously monitoring blood culture system until the bottle was identified as positive. The inoculated concentrations $\mathrm { ( C F U / m L ) }$ ) were at least four logs below the load observed at bottle positivity. The strains used to prepare contrived specimens are shown in Table 6.

Table 6: Contrived specimen study, organisms evaluated   

<table><tr><td rowspan=1 colspan=1>Target</td><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>Independent ContrivedSpecimens Tested</td></tr><tr><td rowspan=10 colspan=1>Acinetobacter baumannii</td><td rowspan=1 colspan=1>ATCCBAA-747</td><td rowspan=1 colspan=1>3</td></tr><tr><td rowspan=1 colspan=1>ATCCBAA-1605</td><td rowspan=1 colspan=1>3</td></tr><tr><td rowspan=1 colspan=1>ATCCBAA-2093</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>ATCC19606</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>AR-0033</td><td rowspan=1 colspan=1>2</td></tr><tr><td rowspan=1 colspan=1>NCTC13302</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>NCTC13303</td><td rowspan=1 colspan=1>2</td></tr><tr><td rowspan=1 colspan=1>NCTC13304</td><td rowspan=1 colspan=1>2</td></tr><tr><td rowspan=1 colspan=1>NCTC13305</td><td rowspan=1 colspan=1>2</td></tr><tr><td rowspan=1 colspan=1>NCTC13423</td><td rowspan=1 colspan=1>3</td></tr><tr><td rowspan=10 colspan=1>Citrobacter</td><td rowspan=1 colspan=1>ATCC8090</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>ATCC27156</td><td rowspan=1 colspan=1>2</td></tr><tr><td rowspan=1 colspan=1>ATCC29935</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>ATCC43864</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>ATCC51113</td><td rowspan=1 colspan=1>2</td></tr><tr><td rowspan=1 colspan=1>ATCC51493</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>CDC AR-0116</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>JMI2047</td><td rowspan=1 colspan=1>2</td></tr><tr><td rowspan=1 colspan=1>NCTC8581</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>NCTC9750</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=2 colspan=1>Haemophilus influenzae</td><td rowspan=1 colspan=1>ATCC10211</td><td rowspan=1 colspan=1>3</td></tr><tr><td rowspan=1 colspan=1>ATCC43065</td><td rowspan=1 colspan=1>3</td></tr></table>

<table><tr><td rowspan=1 colspan=1>Target</td><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=2>Independent ContrivedSpecimens Tested</td></tr><tr><td rowspan=3 colspan=1></td><td rowspan=1 colspan=1>ATCC49144</td><td rowspan=1 colspan=2>4</td></tr><tr><td rowspan=1 colspan=1>NCTC8468</td><td rowspan=1 colspan=2>3</td></tr><tr><td rowspan=1 colspan=1>NCTC12699</td><td rowspan=1 colspan=2>4</td></tr><tr><td rowspan=1 colspan=1>Klebsiella pneumoniae</td><td rowspan=1 colspan=1>CDC AR-140</td><td rowspan=1 colspan=2>1</td></tr><tr><td rowspan=7 colspan=1>Morganella morganii</td><td rowspan=1 colspan=1>CDC AR-0057</td><td rowspan=1 colspan=2>2</td></tr><tr><td rowspan=1 colspan=1>CDCAR-0133</td><td rowspan=1 colspan=2>2</td></tr><tr><td rowspan=1 colspan=1>GM148-200</td><td rowspan=1 colspan=2>3</td></tr><tr><td rowspan=1 colspan=1>GM148-204</td><td rowspan=1 colspan=2>3</td></tr><tr><td rowspan=1 colspan=1>GM148-205</td><td rowspan=1 colspan=2>2</td></tr><tr><td rowspan=1 colspan=1>GM148-206</td><td rowspan=1 colspan=2>2</td></tr><tr><td rowspan=1 colspan=1>GM148-209</td><td rowspan=1 colspan=2>2</td></tr><tr><td rowspan=6 colspan=1>Neisseria meningitidis</td><td rowspan=1 colspan=1>ATCC13077</td><td rowspan=1 colspan=2>4</td></tr><tr><td rowspan=1 colspan=1>ATCC13090</td><td rowspan=1 colspan=2>4</td></tr><tr><td rowspan=1 colspan=1>ATCC13102</td><td rowspan=1 colspan=2>4</td></tr><tr><td rowspan=1 colspan=1>ATCC13113</td><td rowspan=1 colspan=2>3</td></tr><tr><td rowspan=1 colspan=1>ATCC35561</td><td rowspan=1 colspan=2>4</td></tr><tr><td rowspan=1 colspan=1>NCTC10026</td><td rowspan=1 colspan=2>1</td></tr><tr><td rowspan=2 colspan=1>Proteus mirabilis</td><td rowspan=1 colspan=1>CDC AR-0155</td><td rowspan=1 colspan=2>4</td></tr><tr><td rowspan=1 colspan=1>CDC AR-0159</td><td rowspan=1 colspan=2>5</td></tr><tr><td rowspan=6 colspan=1>Pseudomonas aeruginosa</td><td rowspan=1 colspan=1>CDC AR-0054</td><td rowspan=1 colspan=2>3</td></tr><tr><td rowspan=1 colspan=1>CDC AR-0092</td><td rowspan=1 colspan=2>2</td></tr><tr><td rowspan=1 colspan=1>CDC AR-100</td><td rowspan=1 colspan=2>1</td></tr><tr><td rowspan=1 colspan=1>CDC AR-103</td><td rowspan=1 colspan=2>3</td></tr><tr><td rowspan=1 colspan=1>CDC AR-108</td><td rowspan=1 colspan=2>1</td></tr><tr><td rowspan=1 colspan=1>NCTC13437</td><td rowspan=1 colspan=2>2</td></tr><tr><td rowspan=11 colspan=1>Serratia</td><td rowspan=1 colspan=1>ATCC8100</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>ATCC13880</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>ATCC14460</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>ATCC14756</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>ATCC29025</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>ATCC33105</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>ATCC43861</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>ATCC43862</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>ATCC53858</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>JMI10244</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>LMC-DR23105</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=2 colspan=1>Pan-GP (Enterococcusfaecalis)</td><td rowspan=1 colspan=1>ATCCBAA-236</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>ATCC700802</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=16 colspan=1>Pan-GP (Enterococcus)</td><td rowspan=1 colspan=1>ATCCBAA-2316</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>ATCCBAA-2318</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>ATCCBAA-2319</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>ATCCBAA-2320</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>ATCC10541</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>ATCC49996</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>ATCC51559</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>ATCC51858</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>ATCC700221</td><td rowspan=1 colspan=2>2</td></tr><tr><td rowspan=1 colspan=1>ATCC700425</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>LMC002867</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>LMC003921</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>LMC032261</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>LMC055971</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>LMC103676</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>LMC104266</td><td rowspan=1 colspan=2></td></tr></table>

Overall, two hundred forty negative clinical samples were tested for NPA determination. These samples were collected previously as part of the clinical agreement study for the original device (refer to K182619 for details).

A summary of the modified ePlex BCID-GN Panel performance for each organism target for clinical and contrived specimens is shown in Table 7. Performance is based on comparison of ePlex BCID-GN Panel results to results from comparator methods for clinical specimens and based on comparison of ePlex BCID-GN Panel results to the expected result for contrived specimens.

Table 7: Clinical performance summary   

<table><tr><td rowspan=2 colspan=3>Target</td><td rowspan=2 colspan=1>Specimen Type</td><td rowspan=1 colspan=1>Sensitivity/PPA</td><td rowspan=1 colspan=1>Specificity/NPA</td></tr><tr><td rowspan=1 colspan=1>TP/TP+FN(%)</td><td rowspan=1 colspan=1>TN/TN+FP(%)</td></tr><tr><td rowspan=2 colspan=3>A. baumanii</td><td rowspan=1 colspan=1>Contrived</td><td rowspan=1 colspan=1>20/20 (100)</td><td rowspan=1 colspan=1>---</td></tr><tr><td rowspan=1 colspan=1>Clinical</td><td rowspan=1 colspan=1>---</td><td rowspan=1 colspan=1>239/240 (99.6)*</td></tr><tr><td rowspan=2 colspan=3>Citrobacter</td><td rowspan=1 colspan=1>Contrived</td><td rowspan=1 colspan=1>13/13(100)</td><td rowspan=1 colspan=1>---</td></tr><tr><td rowspan=1 colspan=1>Clinical</td><td rowspan=1 colspan=1>7/7 (100)</td><td rowspan=1 colspan=1>240/240 (100)</td></tr><tr><td rowspan=2 colspan=3>E. coli</td><td rowspan=1 colspan=1>Contrived</td><td rowspan=1 colspan=1>---</td><td rowspan=1 colspan=1>---</td></tr><tr><td rowspan=1 colspan=1>Clinical</td><td rowspan=1 colspan=1>40/40 (100)</td><td rowspan=1 colspan=1>240/240 (100)</td></tr><tr><td rowspan=2 colspan=3>H. influenzae</td><td rowspan=1 colspan=1>Contrived</td><td rowspan=1 colspan=1>17/17 (100)</td><td rowspan=1 colspan=1>---</td></tr><tr><td rowspan=1 colspan=1>Clinical</td><td rowspan=1 colspan=1>3/3 (100)</td><td rowspan=1 colspan=1>240/240 (100)</td></tr><tr><td rowspan=2 colspan=3>K. pneumoniae</td><td rowspan=1 colspan=1>Contrived</td><td rowspan=1 colspan=1>1/1 (100)</td><td rowspan=1 colspan=1>---</td></tr><tr><td rowspan=1 colspan=1>Clinical</td><td rowspan=1 colspan=1>19/19 (100)</td><td rowspan=1 colspan=1>240/240 (100)</td></tr><tr><td rowspan=2 colspan=3>M. morganii</td><td rowspan=1 colspan=1>Contrived</td><td rowspan=1 colspan=1>16/16(100)</td><td rowspan=1 colspan=1>---</td></tr><tr><td rowspan=1 colspan=1>Clinical</td><td rowspan=1 colspan=1>4/4 (100)</td><td rowspan=1 colspan=1>240/240 (100)</td></tr><tr><td rowspan=2 colspan=3>N. meningitidis</td><td rowspan=1 colspan=1>Contrived</td><td rowspan=1 colspan=1>20/20 (100)</td><td rowspan=1 colspan=1>---</td></tr><tr><td rowspan=1 colspan=1>Clinical</td><td rowspan=1 colspan=1>---</td><td rowspan=1 colspan=1>240/240 (100)</td></tr><tr><td rowspan=2 colspan=3>P. mirabilis</td><td rowspan=1 colspan=1>Contrived</td><td rowspan=1 colspan=1>9/9 (100)</td><td rowspan=1 colspan=1>---</td></tr><tr><td rowspan=1 colspan=1>Clinical</td><td rowspan=1 colspan=1>11/11 (100)</td><td rowspan=1 colspan=1>239/240 (99.6)*</td></tr><tr><td rowspan=2 colspan=3>P. aeruginosa</td><td rowspan=1 colspan=1>Contrived</td><td rowspan=1 colspan=1>12/12(100)</td><td rowspan=1 colspan=1>---</td></tr><tr><td rowspan=1 colspan=1>Clinical</td><td rowspan=1 colspan=1>8/8 (100)</td><td rowspan=1 colspan=1>240/240 (100)</td></tr><tr><td rowspan=2 colspan=3>Serratia</td><td rowspan=1 colspan=1>Contrived</td><td rowspan=1 colspan=1>11/11(100)</td><td rowspan=1 colspan=1>---</td></tr><tr><td rowspan=1 colspan=1>−</td><td rowspan=1 colspan=1>Clinical</td><td rowspan=1 colspan=1>9/9 (100)</td><td rowspan=1 colspan=1>240/240 (100)</td></tr><tr><td rowspan=2 colspan=2>Pan-GP (Enterococcusfaecalis)</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Contrived</td><td rowspan=1 colspan=1>2/2(100)</td><td rowspan=1 colspan=1>---</td></tr><tr><td rowspan=1 colspan=2></td><td rowspan=1 colspan=1>Clinical</td><td rowspan=1 colspan=1>18/18 (100)</td><td rowspan=1 colspan=1>240/240 (100)</td></tr><tr><td rowspan=2 colspan=3>Pan-GP (Enterococcus)</td><td rowspan=1 colspan=1>−</td><td rowspan=1 colspan=1>Contrived</td><td rowspan=1 colspan=1>17/17(100)</td></tr><tr><td rowspan=1 colspan=1>Clinical</td><td rowspan=1 colspan=1>3/3 (100)</td><td rowspan=1 colspan=1>240/240 (100)</td></tr></table>

\* Root cause analysis of the false positive results indicate workflow contamination during preparation/loading of the samples.

2. Clinical Specificity: See Clinical Sensitivity section above.

# Clinical Cut-Off:D

Not Applicable.

# E Expected Values/Reference Range:

For the expected values from a prospectively enrolled clinical study population, refer to K182619.

# VIII Proposed Labeling:

The labeling supports the finding of substantial equivalence for this device.

# IX Conclusion:

The submitted information in this premarket notification is complete and supports a substantial equivalence decision.