# February 3, 2023

BioFire Diagnostics   
Kevin Bourzac   
Vice President, Regulatory & Clinical Affairs   
515 Colorow Drive   
Salt Lake City, Utah 84108

Re: K213954

Regulation Number: 21 CFR 866.3981

Regulation Name: Device to detect and identify nucleic acid targets in respiratory specimens from microbial agents that cause the SARS-CoV-2 respiratory infection and other microbial agents when in a multi-target test

Regulatory Class: Class II   
Product Code: QOF, OEM, OOU, OTG, OZE, OZX, OZY, OZZ, OCC, NSU   
Dated: December 16, 2021   
Received: December 17, 2021

Dear Kevin Bourzac:

We have reviewed your Section 510(k) premarket notification of intent to market the device referenced above and have determined the device is substantially equivalent (for the indications for use stated in the enclosure) to legally marketed predicate devices marketed in interstate commerce prior to May 28, 1976, the enactment date of the Medical Device Amendments, or to devices that have been reclassified in accordance with the provisions of the Federal Food, Drug, and Cosmetic Act (Act) that do not require approval of a premarket approval application (PMA). You may, therefore, market the device, subject to the general controls provisions of the Act. Although this letter refers to your product as a device, please be aware that some cleared products may instead be combination products. The 510(k) Premarket Notification Database located at https://www.accessdata.fda.gov/scripts/cdrh/cfdocs/cfpmn/pmn.cfm identifies combination product submissions. The general controls provisions of the Act include requirements for annual registration, listing of devices, good manufacturing practice, labeling, and prohibitions against misbranding and adulteration. Please note: CDRH does not evaluate information related to contract liability warranties. We remind you, however, that device labeling must be truthful and not misleading.

If your device is classified (see above) into either class II (Special Controls) or class III (PMA), it may be subject to additional controls. Existing major regulations affecting your device can be found in the Code of Federal Regulations, Title 21, Parts 800 to 898. In addition, FDA may publish further announcements concerning your device in the Federal Register.

Please be advised that FDA's issuance of a substantial equivalence determination does not mean that FDA has made a determination that your device complies with other requirements of the Act or any Federal statutes and regulations administered by other Federal agencies. You must comply with all the Act's requirements, including, but not limited to: registration and listing (21 CFR Part 807); labeling (21 CFR Part 801 and Part 809); medical device reporting (reporting of medical device-related adverse events) (21 CFR 803) for devices or postmarketing safety reporting (21 CFR 4, Subpart B) for combination products (see https://www.fda.gov/combination-products/guidance-regulatory-information/postmarketing-safety-reportingcombination-products); good manufacturing practice requirements as set forth in the quality systems (QS) regulation (21 CFR Part 820) for devices or current good manufacturing practices (21 CFR 4, Subpart A) for combination products; and, if applicable, the electronic product radiation control provisions (Sections 531- 542 of the Act); 21 CFR 1000-1050.

Also, please note the regulation entitled, "Misbranding by reference to premarket notification" (21 CFR Part 807.97). For questions regarding the reporting of adverse events under the MDR regulation (21 CFR Part 803), please go to https://www.fda.gov/medical-devices/medical-device-safety/medical-device-reportingmdr-how-report-medical-device-problems.

For comprehensive regulatory information about medical devices and radiation-emitting products, including information about labeling regulations, please see Device Advice (https://www.fda.gov/medical  
devices/device-advice-comprehensive-regulatory-assistance) and CDRH Learn   
(https://www.fda.gov/training-and-continuing-education/cdrh-learn). Additionally, you may contact the Division of Industry and Consumer Education (DICE) to ask a question about a specific regulatory topic. See the DICE website (https://www.fda.gov/medical-devices/device-advice-comprehensive-regulatoryassistance/contact-us-division-industry-and-consumer-education-dice) for more information or contact DICE by email (DICE@fda.hhs.gov) or phone (1-800-638-2041 or 301-796-7100).

Sincerely,

![](images/bf87e050024d4aa7d105e34d5f82c4ac41ad34b5b81d9163abdb813b7d09c6b6.jpg)

Joseph Briggs   
Deputy Branch Chief   
Division of Microbiology Devices   
OHT7: Office of In Vitro Diagnostics   
Office of Product Evaluation and Quality   
Center for Devices and Radiological Health

Enclosure

510(k) Number (if known) K213954

Device Name BIOFIRE $\textsuperscript { \textregistered }$ SPOTFIRE $\textsuperscript { \textregistered }$ Respiratory (R) Panel

Indications for Use (Describe) The BIOFIRE $^ { \textregistered }$ SPOTFIRE $^ \mathrm { \textregistered }$ Respiratory (R) Panel (SPOTFIRE R Panel) is a multiplexed polymerase chain reaction (PCR) test intended for use with the BIOFIRE $^ { \textregistered }$ SPOTFIRE $^ { \textregistered }$ System for the simultaneous, qualitative detection and identification of multiple respiratory viral and bacterial nucleic acids in nasopharyngeal swab (NPS) specimens obtained from individuals with signs and symptoms of respiratory tract infection, including COVID-19.

The following organism types and subtypes are identified and differentiated using the SPOTFIRE R Panel:

Viruses:   
Adenovirus   
Coronavirus (seasonal)   
Coronavirus SARS-CoV-2   
Human metapneumovirus   
Human rhinovirus/enterovirus   
Influenza A virus   
Influenza A virus A/H1-2009   
Influenza A virus A/H3   
Influenza B virus   
Parainfluenza virus   
Respiratory syncytial virus

Bacteria: Bordetella parapertussis Bordetella pertussis Chlamydia pneumoniae Mycoplasma pneumoniae

Nucleic acids from the viral and bacterial organisms identified by this test are generally detectable in NPS specimens during the acute phase of infection. The detection and identification of specific viral and bacterial nucleic acids from individuals exhibiting signs and/or symptoms of respiratory infection are indicative of the presence of the identified microorganism and aids in diagnosis if used in conjunction with other clinical and epidemiological information, and laboratory findings. The results of this test should not be used as the sole basis for diagnosis, treatment, or other patient management decisions.

Negative results in the setting of a respiratory illness may be due to infection with pathogens that are not detected by this test, or lower respiratory tract infection that may not be detected by an NPS specimen. Positive results do not rule out coinfection with other organisms. The agent(s) detected by the SPOTFIRE R Panel may not be the definite cause of disease.

Additional laboratory testing (e.g., bacterial and viral culture, immunofluorescence, and radiography) may be necessary when evaluating a patient with possible respiratory tract infection.

This section applies only to requirements of the Paperwork Reduction Act of 1995.

# \*DO NOT SEND YOUR COMPLETED FORM TO THE PRA STAFF EMAIL ADDRESS BELOW.\*

The burden time for this collection of information is estimated to average 79 hours per response, including the time to review instructions, search existing data sources, gather and maintain the data needed and complete and review the collection of information. Send comments regarding this burden estimate or any other aspect of this information collection, including suggestions for reducing this burden, to:

Department of Health and Human Services Food and Drug Administration Office of Chief Information Officer Paperwork Reduction Act (PRA) Staff PRAStaff@fda.hhs.gov

“An agency may not conduct or sponsor, and a person is not required to respond to, a collection of information unless it displays a currently valid OMB number.”

# BIOFIRE® SPOTFIRE® Respiratory (R) Panel 510(k) Summary BioFire Diagnostics, LLC

# Introduction:

According to the requirements of 21 CFR 807.92, the following information provides sufficient detail to understand the basis for a determination of substantial equivalence.

# Submitted by:

BioFire Diagnostics, LLC 515 Colorow Drive Salt Lake City, UT 84108

Telephone: 801-736-6354

Facsimile: 801-588-0507

Contact: Kevin Bourzac, Ph.D ext. 1358

Date Submitted: December 16, 2021

# Device Name and Classification:

Trade Name: BIOFIRE® SPOTFIRE® Respiratory (R) Panel

Regulation Number: 21 CFR 866.3981

Classification Name: Multi-target Respiratory Specimen Nucleic Acid Test Including SARS-CoV-2 and Othe Microbial Agents

# Predicate Device:

DEN200031 – BIOFIRE® Respiratory Panel 2.1 (RP2.1)

# Intended Use:

The BIOFIRE® SPOTFIRE® Respiratory (R) Panel (SPOTFIRE R Panel) is a multiplexed polymerase chain reaction (PCR) test intended for use with the BIOFIRE® SPOTFIRE® System for the simultaneous, qualitative detection and identification of multiple respiratory viral and bacterial nucleic acids in nasopharyngeal swab (NPS) specimens obtained from individuals with signs and symptoms of respiratory tract infection, including COVID-19.

The following organism types and subtypes are identified and differentiated using the SPOTFIRE R Panel:

<table><tr><td>Viruses</td><td>Bacteria</td></tr><tr><td>Adenovirus</td><td>Bordetella parapertussis</td></tr><tr><td>Coronavirus (seasonal)</td><td>Bordetella pertussis</td></tr><tr><td>Coronavirus SARS-CoV-2</td><td>Chlamydia pneumoniae</td></tr><tr><td>Human metapneumovirus</td><td>Mycoplasma pneumoniae</td></tr><tr><td>Human rhinovirus/enterovirus</td><td></td></tr><tr><td>Influenza A virus</td><td></td></tr><tr><td>Influenza A virus A/H1-2009</td><td></td></tr><tr><td>Influenza A virus A/H3</td><td></td></tr><tr><td>Influenza B virus</td><td></td></tr><tr><td>Parainfluenza virus</td><td></td></tr><tr><td></td><td></td></tr><tr><td>Respiratory syncytial virus</td><td></td></tr></table>

Nucleic acids from the viral and bacterial organisms identified by this test are generally detectable in NPS specimens during the acute phase of infection. The detection and identification of specific viral and bacterial nucleic acids from individuals exhibiting signs and/or symptoms of respiratory infection are indicative of the presence of the identified microorganism and aids in diagnosis if used in conjunction with other clinical and epidemiological information, and laboratory findings. The results of this test should not be used as the sole basis for diagnosis, treatment, or other patient management decisions.

Negative results in the setting of a respiratory illness may be due to infection with pathogens that are not detected by this test, or lower respiratory tract infection that may not be detected by an NPS specimen. Positive results do not rule out coinfection with other organisms. The agent(s) detected by the SPOTFIRE R Panel may not be the definite cause of disease.

Additional laboratory testing (e.g., bacterial and viral culture, immunofluorescence, and radiography) may be necessary when evaluating a patient with possible respiratory tract infection.

# Device Description:

The BIOFIRE $\textsuperscript { \textregistered }$ SPOTFIRE $\textcircled{8}$ Respiratory (R) Panel (SPOTFIRE R Panel) simultaneously identifies 15 different respiratory viral and bacterial pathogens in nasopharyngeal swabs (NPS) from individuals with signs and symptoms of respiratory tract infection (see Table 1) . The SPOTFIRE R Panel is compatible with the BIOFIRE® SPOTFIRE® System, a polymerase chain reaction (PCR)-based in vitro diagnostic system for infectious disease testing. The SPOTFIRE System Software executes the SPOTFIRE R Panel test and interprets and reports the test results. The SPOTFIRE R Panel was designed to be used in CLIA-waived environments.

Table 1. Analytes Detected by the SPOTFIRE (R) Panel   

<table><tr><td>Viruses</td><td>Bacteria</td></tr><tr><td>Adenovirus</td><td>Bordetella parapertussis Bordetella pertussis</td></tr><tr><td>Coronavirus (seasonal)</td><td rowspan="3">Chlamydia pneumoniae Mycoplasma pneumoniae</td></tr><tr><td>Coronavirus SARS-CoV-2</td></tr><tr><td>Human metapneumovirus Human rhinovirus/enterovirus</td></tr></table>

A test is initiated by loading Hydration Solution into one port of the SPOTFIRE R Panel pouch and a NPS specimen mixed with the provided Sample Buffer into the other port of the SPOTFIRE R Panel pouch and placing it in the SPOTFIRE System. The pouch contains all of the reagents required for specimen testing and analysis in a freezedried format; the addition of Hydration Solution and Sample/Buffer Mix rehydrates the reagents. After the pouch is prepared, the SPOTFIRE System Software guides the user through the steps of placing the pouch into the instrument, scanning the pouch barcode, entering the sample identification, and initiating the run.

The SPOTFIRE System contains coordinated systems of inflatable bladders and seal points, which act on the pouch to control the movement of liquid between the pouch blisters. When a bladder is inflated over a reagent blister, it forces liquid from the blister into connecting channels. Alternatively, when a seal is placed over a connecting channel it acts as a valve to open or close a channel. In addition, electronically-controlled pneumatic pistons are positioned over multiple plungers in order to deliver the rehydrated reagents into the blisters at the appropriate times. Two Peltier devices control heating and cooling of the pouch to drive the PCR reactions and the melt curve analysis.

Nucleic acid extraction occurs within the SPOTFIRE R Panel pouch using mechanical and chemical lysis followed by purification using standard magnetic bead technology. After extracting and purifying nucleic acids from the unprocessed sample, the SPOTFIRE system performs a nested multiplex PCR that is executed in two stages. During the first stage, the SpotFire system performs a single, large volume, highly multiplexed reverse transcription PCR (rt-PCR) reaction. The products from first stage PCR are then diluted and combined with a fresh, primer-free master mix and a fluorescent double-stranded DNA binding dye (LC Green® Plus, BioFire Diagnostics). The solution is then distributed to each well of the array. Array wells contain sets of primers designed specifically to amplify sequences internal to the PCR products generated during the first stage PCR reaction. The 2nd stage PCR, or nested PCR, is performed in singleplex fashion in each well of the array. At the conclusion of the 2nd stage PCR, the array is interrogated by melt curve analysis for the detection of signature amplicons denoting the presence of specific targets. A digital camera placed in front of the 2nd stage PCR captures fluorescent images of the PCR reactions and software interprets the data.

The SPOTFIRE System Software automatically interprets the results of each DNA melt curve analysis and combines the data with the results of the internal pouch controls to provide a test result for each organism on the panel.

# Substantial Equivalence:

The SPOTFIRE R Panel is substantially equivalent to the BioFire Respiratory Panel 2.1 (BioFire RP2.1) (DEN200031), which was cleared on March 17, 2021 and determined to be a Class II device under the classification code 21 CFR 866.3981.

A table comparing the SPOTFIRE R Panel to the BioFire RP2.1 is provided in Table 2.   
Table 2. Similarities and differences between the BioFire RP2.1 and the SPOTFIRE R Panel   

<table><tr><td colspan="1" rowspan="1">Element</td><td colspan="1" rowspan="1">Predicate: BioFire RP2.1 (DEN200031)</td><td colspan="1" rowspan="1">New Device: SPOTFIRE R Panel</td></tr><tr><td colspan="1" rowspan="1">Intended Use</td><td colspan="1" rowspan="1">The BioFire Respiratory Panel 2.1 (RP2.1)is a PCR-based multiplexed nucleic acidtest intended for use with the BioFire®FilmArray® 2.0 or BioFire® FilmArray®Torch Systems for the simultaneousqualitative detection and identification ofmultiple respiratory viral and bacterialnucleic acids in nasopharyngeal swabs(NPS) obtained from individuals suspectedof respiratory tract infections, includingCOVID-19</td><td colspan="1" rowspan="1">The SPOTFIRE R Panel is a PCR-basedmultiplexed nucleic acid test intended foruse with the SPOTFIRE System for the forthe simultaneous, qualitative detection andidentification of multiple respiratory viral andbacterial nucleic acids in nasopharyngealswab (NPS) specimens obtained fromindividuals with signs and symptoms ofrespiratory tract infection, including COVID-19.</td></tr><tr><td colspan="1" rowspan="1">Specimen Types</td><td colspan="1" rowspan="1">Nasopharyngeal swab in transport media orsaline</td><td colspan="1" rowspan="1">Nasopharyngeal swab in transport media</td></tr><tr><td colspan="1" rowspan="1">Organisms detected</td><td colspan="1" rowspan="1">VirusesAdenovirusCoronavirus 229ECoronavirus HKU1Coronavirus NL63Coronavirus OC43</td><td colspan="1" rowspan="1">Same, except:No assay for Influenza A subtype H1Coronavirus 229E/HKU1/NL63/OC43combined call (coronavirus seasonal)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Severe Acute Respiratory SyndromeCoronavirus 2 (SARS-CoV-2)Human MetapneumovirusHuman Rhinovirus/EnterovirusInfluenza ASubtypes: H1, H3 and H1-2009Influenza BParainfluenza Virus 1Parainfluenza Virus 2Parainfluenza Virus 3Parainfluenza Virus 4Respiratory Syncytial VirusBacteriaBordetella parapertussisBordetella pertussisChlamydia pneumoniaeMycoplasma pneumoniae</td><td colspan="1" rowspan="1">Combined Parainfluenza call for serotypes1-4</td></tr><tr><td colspan="1" rowspan="1">Analytes</td><td colspan="1" rowspan="1">DNA/RNA</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Technological Principles</td><td colspan="1" rowspan="1">Highly multiplexed nested nucleic acidamplification test with melt analysis</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Instrumentation</td><td colspan="1" rowspan="1">BioFire FilmArray 2.0 or BioFire FilmArrayTorch Systems</td><td colspan="1" rowspan="1">SPOTFIRE System (this submission)</td></tr><tr><td colspan="1" rowspan="1">Time to result</td><td colspan="1" rowspan="1">About 1 hour</td><td colspan="1" rowspan="1">About 15 minutes</td></tr><tr><td colspan="1" rowspan="1">Reagent Storage</td><td colspan="1" rowspan="1">Room Temperature</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Test Interpretation</td><td colspan="1" rowspan="1">Automated test interpretation and reporting.User cannot access raw data.</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Controls</td><td colspan="1" rowspan="1">Two controls are included in each reagentpouch to control for sample processing andboth stages of PCR and melt analysis.</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">User complexity</td><td colspan="1" rowspan="1">Moderate</td><td colspan="1" rowspan="1">Low (CLIA-waived)</td></tr><tr><td colspan="1" rowspan="2">Panel Software Functions</td><td colspan="1" rowspan="1">Defines panel-specific parameters,instrument protocols and reportrequirements.</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Analyzes processed image data(fluorescence and temperature data) andprovides test results.</td><td colspan="1" rowspan="1">Same</td></tr></table>

# Summary of Performance Data

# Clinical Performance

# Prospective Clincal Evaluation

The clinical performance (encompassing both accuracy and ease of use) of the SPOTFIRE R Panel was established during a prospective multi-center study that was further supplemented with archived specimens. Five geographically distinct urgent care or emergency department study sites representative of the intended use setting (four in the US and one in the UK) participated in these studies from December 2020 to June 2021. All SPOTFIRE R Panel testing was performed according to the manufacturer’s instructions by minimally trained operators. No hands-on training was provided to the SPOTFIRE R Panel test operators; rather, training was limited to written materials (i.e. quick reference guides) that were intended to be included with the BioFire SPOTFIRE System.

A total of 1215 NPS specimens were enrolled from consented volunteers or obtained as residual leftover specimens from subjects of all ages for the prospective clinical study; 95 of these NPS specimens were excluded. The most common reason for specimen exclusion was that a valid SPOTFIRE R Panel test was not obtained due to an invalid SPOTFIRE R Panel test. The final data set consisted of 1120 NPS specimens. Across the five study sites, 259 NPS specimens were initially collected and immediately frozen for later testing at the source study site. The remaining 861 NPS specimens were collected and tested fresh (without freezing). No difference in performance was observed when fresh and frozen specimen results were compared. Therefore, the data collected from 259 valid frozen specimens are combined with data from the valid 861 fresh specimens for all analyses.

Table 3 provides a summary of demographic information for the specimens included in the study.

Table 3. Demographic Summary for Prospective SPOTFIRE R Panel Clinical Evaluation   

<table><tr><td rowspan=1 colspan=2>Category</td><td rowspan=1 colspan=1>ProspectiveNPS Specimens</td></tr><tr><td rowspan=2 colspan=1>3</td><td rowspan=1 colspan=1>Male</td><td rowspan=1 colspan=1>587 (52.4%)</td></tr><tr><td rowspan=1 colspan=1>Female</td><td rowspan=1 colspan=1>533 (47.6%)</td></tr><tr><td rowspan=5 colspan=1>%</td><td rowspan=1 colspan=1>≤5 years</td><td rowspan=1 colspan=1>457 (40.8%)</td></tr><tr><td rowspan=1 colspan=1>6-18 years</td><td rowspan=1 colspan=1>258 (23.0%)</td></tr><tr><td rowspan=1 colspan=1>19-40 years</td><td rowspan=1 colspan=1>160 (14.3%)</td></tr><tr><td rowspan=1 colspan=1>41-60 years</td><td rowspan=1 colspan=1>147 (13.1%)</td></tr><tr><td rowspan=1 colspan=1>61+ years</td><td rowspan=1 colspan=1>98 (8.8%)</td></tr><tr><td rowspan=1 colspan=2>Total</td><td rowspan=1 colspan=1>1120</td></tr></table>

The performance of the SPOTFIRE R Panel was evaluated by comparing the test results with those from FDAcleared multiplexed respiratory pathogen panels. The performance for the prospective study is summarized in Table 4. Positive percent agreement (PPA) for each analyte was calculated as $1 0 0 \% \times ( \mathsf { T P } / \left( \mathsf { T P } + \mathsf { F N } \right) )$ . True positive (TP) indicates that both the SPOTFIRE R Panel and the comparator method had a positive result for the specific analyte, and false negative (FN) indicates that the SPOTFIRE R Panel was negative while the comparator result was positive. Negative percent agreement (NPA) was calculated as $1 0 0 \% \times ( \mathsf { T N } / \mathsf { T N } + \mathsf { F P } ) )$ . True negative (TN) indicates that both the SPOTFIRE R Panel and the comparator method had negative results, and false positive (FP) indicates that the SPOTFIRE R Panel was positive while the comparator method was negative. The exact binomial two-sided $9 5 \%$ confidence interval $( 9 5 \% \mathsf { C l } )$ was calculated. Investigations of discrepant results are summarized in the footnotes.

Table 4. SPOTFIRE R Panel Clinical Performance Summary for NPS Specimens   

<table><tr><td rowspan=2 colspan=1>SPOTFIRE R Panel Analyte</td><td rowspan=1 colspan=3>Positive Percent Agreement</td><td rowspan=1 colspan=3>Negative Percent Agreement</td></tr><tr><td rowspan=1 colspan=1>TP/(TP + FN)</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95%CI</td><td rowspan=1 colspan=1>TN/(TN + FP)</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95%CI</td></tr><tr><td rowspan=1 colspan=7>Viruses</td></tr><tr><td rowspan=1 colspan=1>Adenovirusa</td><td rowspan=1 colspan=1>32/33</td><td rowspan=1 colspan=1>97.0</td><td rowspan=1 colspan=1>84.7-99.5%</td><td rowspan=1 colspan=1>1058/1082</td><td rowspan=1 colspan=1>97.8</td><td rowspan=1 colspan=1>96.7-98.5%</td></tr><tr><td rowspan=1 colspan=1>Coronavirus SARS-CoV-2b</td><td rowspan=1 colspan=1>71/73</td><td rowspan=1 colspan=1>97.3</td><td rowspan=1 colspan=1>90.5-99.2%</td><td rowspan=1 colspan=1>1031/1037</td><td rowspan=1 colspan=1>99.4</td><td rowspan=1 colspan=1>98.7-99.7%</td></tr><tr><td rowspan=1 colspan=1>Coronavirus (seasonal)c</td><td rowspan=1 colspan=1>101/102</td><td rowspan=1 colspan=1>99.0</td><td rowspan=1 colspan=1>94.7-99.8%</td><td rowspan=1 colspan=1>1000/1013</td><td rowspan=1 colspan=1>98.7</td><td rowspan=1 colspan=1>97.8-99.2%</td></tr><tr><td rowspan=1 colspan=1>Human metapneumovirus</td><td rowspan=1 colspan=1>1/1</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>1114/1114</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>99.7-100%</td></tr><tr><td rowspan=1 colspan=1>Human rhinovirus/enterovirusd</td><td rowspan=1 colspan=1>345/348</td><td rowspan=1 colspan=1>99.1</td><td rowspan=1 colspan=1>97.5-99.7%</td><td rowspan=1 colspan=1>695/767</td><td rowspan=1 colspan=1>90.6</td><td rowspan=1 colspan=1>88.3-92.5%</td></tr><tr><td rowspan=1 colspan=1>Influenza A virus</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>1115/1115</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>99.7-100%</td></tr><tr><td rowspan=1 colspan=1>Influenza A virus A/H1-2009</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>1115/1115</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>99.7-100%</td></tr><tr><td rowspan=1 colspan=1>Influenza A virus A/H3</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>1115/1115</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>99.7-100%</td></tr><tr><td rowspan=1 colspan=1>Influenza B virus</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>1110/1110</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>99.7-100%</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza virus</td><td rowspan=1 colspan=1>96/98</td><td rowspan=1 colspan=1>98.0</td><td rowspan=1 colspan=1>92.9-99.4%</td><td rowspan=1 colspan=1>1006/1017</td><td rowspan=1 colspan=1>98.9</td><td rowspan=1 colspan=1>98.1-99.4%</td></tr><tr><td rowspan=1 colspan=1>Respiratory syncytial virusf</td><td rowspan=1 colspan=1>26/27</td><td rowspan=1 colspan=1>96.3</td><td rowspan=1 colspan=1>81.7-99.3%</td><td rowspan=1 colspan=1>1086/1088</td><td rowspan=1 colspan=1>99.8</td><td rowspan=1 colspan=1>99.3-99.9%</td></tr><tr><td rowspan=1 colspan=1>Bacteria</td><td></td><td></td><td></td><td></td><td></td><td></td></tr><tr><td rowspan=1 colspan=1>Bordetella parapertussis</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>1110/1110</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>99.7-100%</td></tr><tr><td rowspan=1 colspan=1>Bordetella pertussis</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>1115/1115</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>99.7-100%</td></tr><tr><td rowspan=1 colspan=1>Chlamydia pneumoniae</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>1115/1115</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>99.7-100%</td></tr><tr><td rowspan=1 colspan=1>Mycoplasma pneumoniae</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>1115/1115</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>99.7-100%</td></tr></table>

a Adenovirus was not detected in the single FN specimen upon SPOTFIRE R Panel retest. Adenovirus was detected in 21/24 FP specimens using an additional molecular method. b SARS-CoV-2 was detected in $_ { 1 / 2 }$ FN specimens upon SPOTFIRE R Panel retest. SARS- $\mathtt { C o V - 2 }$ was detected in 2/6 FP specimens using an additional molecular method. Seasonal coronavirus (229E/HKU1/NL63/OC43) was detected in the single FN specimen upon SPOTFIRE R Panel retest. Seasonal coronavirus (229E/HKU1/NL63/OC43) was detected in $_ { 8 / 1 3 }$ FP specimens using an additional molecular method. d Human rhinovirus/enterovirus was detected in 1/3 FN specimens upon SPOTFIRE R Panel retest. Human rhinovirus/enterovirus was detected in 48/72 FP specimens using an additional molecular method. e Parainfluenza virus was detected in both FN specimens during discrepancy investigation upon SPOTFIRE R Panel retest. Parainfluenza virus was detected in 9/11 FP specimens using an additional molecular method. Respiratory syncytial virus was detected in the single FN specimen upon SPOTFIRE R Panel retest. Respiratory syncytial virus was detected in 1/2 FP specimens using an additional molecular method.

The overall success rate for initial specimen tests was $9 6 . 7 \%$ (1158/1198). Eight tests $( 8 / 1 1 9 8 ; 0 . 7 \% )$ did not complete on the initial test attempt, resulting in an instrument success rate of $9 9 . 3 \%$ (1190/1198) for initial specimen tests. Retests were not possible due to insufficient specimen volume. Of the 1190 tests that successfully produced a completed run on the initial test, 1158 had valid internal process controls. This represents a $9 7 . 3 \%$ (1158/1190) success rate for internal process controls in completed runs in the initial specimen tests.

# Testing of Preselected Archived Specimens

A number of analytes on the SPOTFIRE R Panel were of low prevalence during the prospective study and were not encountered in large enough numbers to adequately demonstrate system performance. To supplement the results of the prospective clinical study, an evaluation of preselected archived retrospective NPS specimens was performed.

A total of 562 frozen archived NPS specimens were obtained from 15 external laboratories world-wide and retrospectively tested at the four US clinical sites. Of these, 542 NPS specimens had valid results that were included in performance analysis. The analyte composition of the archived specimens was confirmed using the same comparator methods as the prospective study (described above) for the analyte result to be included in the performance analysis.

The specimens were randomized such that the users performing both the confirmation and the SPOTFIRE R Panel testing were blinded to the expected test result. A summary of the available demographic information of the tested specimens is provided in Table 5, and the results of the SPOTFIRE R Panel performance for these archived specimens is shown in Table 6.

Table 5. Demographic Summary for Valid Archived NPS Specimens   

<table><tr><td rowspan=1 colspan=2>Category</td><td rowspan=1 colspan=1>Archived NPSSpecimens</td></tr><tr><td rowspan=3 colspan=1></td><td rowspan=1 colspan=1>Male</td><td rowspan=1 colspan=1>254 (46.9%)</td></tr><tr><td rowspan=1 colspan=1>Female</td><td rowspan=1 colspan=1>185 (34.1%)</td></tr><tr><td rowspan=1 colspan=1>Unknown</td><td rowspan=1 colspan=1>103 (19.0%)</td></tr><tr><td rowspan=6 colspan=1>R</td><td rowspan=1 colspan=1>≤5 years</td><td rowspan=1 colspan=1>234 (43.2%)</td></tr><tr><td rowspan=1 colspan=1>6-18 years</td><td rowspan=1 colspan=1>98 (18.1%)</td></tr><tr><td rowspan=1 colspan=1>19-40 years</td><td rowspan=1 colspan=1>36 (6.6%)</td></tr><tr><td rowspan=1 colspan=1>41-60 years</td><td rowspan=1 colspan=1>35 (6.5%)</td></tr><tr><td rowspan=1 colspan=1>61+ years</td><td rowspan=1 colspan=1>39 (7.2%)</td></tr><tr><td rowspan=1 colspan=1>Unknown</td><td rowspan=1 colspan=1>100 (18.5%)</td></tr><tr><td rowspan=1 colspan=2>Total</td><td rowspan=1 colspan=1>542</td></tr></table>

Table 6. SPOTFIRE R Panel Archived Performance Summary for NPS Specimens   

<table><tr><td rowspan=2 colspan=1>SPOTFIRE R Panel Analyte</td><td rowspan=1 colspan=3>Positive Percent Agreement</td><td rowspan=1 colspan=3>Negative Percent Agreement</td></tr><tr><td rowspan=1 colspan=1>TP/(TP + FN)</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=1>TN(TN + FP)</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=7>Viruses</td></tr><tr><td rowspan=1 colspan=1>Adenovirusa</td><td rowspan=1 colspan=1>31/31</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>89.0-100%</td><td rowspan=1 colspan=1>439/453</td><td rowspan=1 colspan=1>96.9</td><td rowspan=1 colspan=1>94.9-98.2%</td></tr><tr><td rowspan=1 colspan=1>Coronavirus SARS-CoV-2</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>Coronavirus (seasonal)</td><td rowspan=1 colspan=1>95/96</td><td rowspan=1 colspan=1>99.0</td><td rowspan=1 colspan=1>94.3-99.8%</td><td rowspan=1 colspan=1>381/388</td><td rowspan=1 colspan=1>98.2</td><td rowspan=1 colspan=1>96.3-99.1%</td></tr><tr><td rowspan=1 colspan=1>Human metapneumovirus</td><td rowspan=1 colspan=1>32/33</td><td rowspan=1 colspan=1>97.0</td><td rowspan=1 colspan=1>84.7-99.5%</td><td rowspan=1 colspan=1>451/451</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>99.2-100%</td></tr><tr><td rowspan=1 colspan=1>Human rhinovirus/enterovirusd</td><td rowspan=1 colspan=1>29/30</td><td rowspan=1 colspan=1>96.7</td><td rowspan=1 colspan=1>83.3-99.4%</td><td rowspan=1 colspan=1>439/454</td><td rowspan=1 colspan=1>96.7</td><td rowspan=1 colspan=1>94.6-98.0%</td></tr><tr><td rowspan=1 colspan=1>Influenza A virus</td><td rowspan=1 colspan=1>58/59</td><td rowspan=1 colspan=1>98.3</td><td rowspan=1 colspan=1>91.0-99.7%</td><td rowspan=1 colspan=1>423/423</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>99.1-100%</td></tr><tr><td rowspan=1 colspan=1>Influenza A virus A/H1-2009e</td><td rowspan=1 colspan=1>31/32</td><td rowspan=1 colspan=1>96.9</td><td rowspan=1 colspan=1>84.3-99.4%</td><td rowspan=1 colspan=1>450/450</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>99.2-100%</td></tr><tr><td rowspan=1 colspan=1>Influenza A virus A/H3</td><td rowspan=1 colspan=1>27/27</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>87.5-100%</td><td rowspan=1 colspan=1>455/455</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>99.2-100%</td></tr><tr><td rowspan=1 colspan=1>Influenza B virus</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>88.6-100%</td><td rowspan=1 colspan=1>28/28</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>87.9-100%</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza virus</td><td rowspan=1 colspan=1>116/118</td><td rowspan=1 colspan=1>98.3</td><td rowspan=1 colspan=1>94.0-99.5%</td><td rowspan=1 colspan=1>359/366</td><td rowspan=1 colspan=1>98.1</td><td rowspan=1 colspan=1>96.1-99.1%</td></tr><tr><td rowspan=1 colspan=1>Respiratory syncytial virus</td><td rowspan=1 colspan=1>37/37</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>90.6-100%</td><td rowspan=1 colspan=1>440/447</td><td rowspan=1 colspan=1>98.4</td><td rowspan=1 colspan=1>96.8-99.2%</td></tr><tr><td rowspan=1 colspan=7>Bacteria</td></tr><tr><td rowspan=1 colspan=1>Bordetella parapertussish</td><td rowspan=1 colspan=1>24/25</td><td rowspan=1 colspan=1>96.0</td><td rowspan=1 colspan=1>80.5-99.3%</td><td rowspan=1 colspan=1>33/33</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>89.6-100%</td></tr><tr><td rowspan=1 colspan=1>Bordetella pertussis</td><td rowspan=1 colspan=1>27/28</td><td rowspan=1 colspan=1>96.4</td><td rowspan=1 colspan=1>82.3-99.4%</td><td rowspan=1 colspan=1>452/456</td><td rowspan=1 colspan=1>99.1</td><td rowspan=1 colspan=1>97.8-99.7%</td></tr><tr><td rowspan=1 colspan=1>Chlamydia pneumoniaei</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>88.6-100%</td><td rowspan=1 colspan=1>452/454</td><td rowspan=1 colspan=1>99.6</td><td rowspan=1 colspan=1>98.4-99.9%</td></tr><tr><td rowspan=1 colspan=1>Mycoplasma pneumoniaek</td><td rowspan=1 colspan=1>33/33</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>89.6-100%</td><td rowspan=1 colspan=1>446/451</td><td rowspan=1 colspan=1>98.9</td><td rowspan=1 colspan=1>97.4-99.5%</td></tr></table>

a Adenovirus was detected in $_ { 6 / 1 2 }$ FP specimens during discrepancy investigation: one was detected by standard of care and five were detected using an additional molecular method; two additional FP specimens were unable to be investigated.   
b The single FN specimen was unable to be investigated. Seasonal coronavirus (229E/HKU1/NL63/OC43) was detected in 3/6 FP specimens during discrepancy investigation using an additional molecular method; one additional FP   
specimen was unable to be investigated.   
c Human metapneumovirus was detected in the single FN specimen by standard of care.   
d The single FN specimen was unable to be investigated. Human rhinovirus/enterovirus was detected in 4/14 FP specimens during discrepancy investigation using an additional molecular method; one additional FP specimen was   
unable to be investigated.   
e Influenza A virus A/H1-2009 was detected in the single FN specimen by standard of care.   
f Parainfluenza virus was detected in the single FN specimen by standard of care; one additional FN specimen was   
unable to be investigated. Parainfluenza virus was detected in all seven FP specimens during discrepancy investigation: six were detected by standard of care and one was detected using an additional molecular method.   
g Respiratory syncytial virus was detected in 4/6 FP specimens during discrepancy investigation using an additional molecular method; one additional FP specimen was unable to be investigated.   
h Bordetella parapertussis was detected in the single FN specimen by standard of care.   
i Bordetella pertussis was detected in the single FN specimen by standard of care. Bordetella pertussis was detected in   
3/4 FP specimens by standard of care.   
j Chlamydia pneumoniae was detected in both FP specimens by standard of care.   
k Mycoplasma pneumoniae was detected in all five FP specimens during discrepancy investigation: four were detected by standard of care and one was detected using an additional molecular method.

# Analytical Performance Characteristics

# Limit of Detection

The limit of detection (LoD) for SPOTFIRE R Panel analytes was estimated by testing dilutions of contrived samples containing known concentrations of organism in artificial nasopharyngeal swab matrix in VTM. The LoD concentrations were confirmed by testing at least 20 replicates at the estimated LoD. Confirmation of LoD required detection in at least $9 5 \%$ of replicates tested. The confirmed LoD concentrations for the SPOTFIRE R Panel are listed in Table 7.

Table 7. Limit of Detection (LoD) for the SPOTFIRE R Panel Analytes   

<table><tr><td colspan="1" rowspan="1">Analyte</td><td colspan="1" rowspan="1">IsolateSource ID</td><td colspan="1" rowspan="1">LoD Concentrationa</td></tr><tr><td></td><td colspan="1" rowspan="1">Viruses</td><td></td></tr><tr><td colspan="1" rowspan="6">Adenovirus</td><td colspan="1" rowspan="1">Species A Serotype 31ZeptoMetrix 0810073CF</td><td colspan="1" rowspan="1">4.1E-03 TCID5o/mL(1.0E+02 copies/mL)</td></tr><tr><td colspan="1" rowspan="1">Species B Serotype 3ZeptoMetrix 0810062CF</td><td colspan="1" rowspan="1">8.0E-01 TCID50/mL(8.4E+02 copies/mL)</td></tr><tr><td colspan="1" rowspan="1">Species C Serotype 2WHO I.S. NIBSC 16-324</td><td colspan="1" rowspan="1">8.2E+02 IU/mLb(8.2E+02 copies/mLb)</td></tr><tr><td colspan="1" rowspan="1">Species D Serotype 37ZeptoMetrix 0810119CF</td><td colspan="1" rowspan="1">1.1E-02 TCID50/mL(4.5E+02 copies/mL)</td></tr><tr><td colspan="1" rowspan="1">Species E Serotype 4ZeptoMetrix 0810070CF</td><td colspan="1" rowspan="1">1.8E-02 TCID50/mL(1.0E+04 copies/mL)</td></tr><tr><td colspan="1" rowspan="1">Species F Serotype 41ZeptoMetrix 0810085CF</td><td colspan="1" rowspan="1">1.4E-02 TCID50/mL(1.0E+02 copies/mL)</td></tr><tr><td colspan="1" rowspan="4">Coronavirus (seasonal)</td><td colspan="1" rowspan="1">229EATCC VR-740</td><td colspan="1" rowspan="1">6.5E-01 TCID50/mL(1.1E+01 copies/mL)</td></tr><tr><td colspan="1" rowspan="1">HKU1Clinical Specimen</td><td colspan="1" rowspan="1">1.8E+04 copies/mL</td></tr><tr><td colspan="1" rowspan="1">OC43ZeptoMetrix 0810024CF</td><td colspan="1" rowspan="1">1.6E-02 TCID5o/mL(6.3E+01 copies/mL)</td></tr><tr><td colspan="1" rowspan="1">NL63ZeptoMetrix 0810228CF</td><td colspan="1" rowspan="1">2.5E-03 TCID5o/mL(4.7E+00 copies/mL)</td></tr><tr><td colspan="1" rowspan="1">Coronavirus SARS-CoV-2</td><td colspan="1" rowspan="1">USA-WA1/2020(heat inactivated)ATCC VR-1986HK</td><td colspan="1" rowspan="1">1.1E-01 TCID50/mL(2.5E+02 copies/mL)</td></tr><tr><td colspan="1" rowspan="4">Human metapneumovirus</td><td colspan="1" rowspan="1">A1-16 lowa 10/2003ZeptoMetrix 0810161CF</td><td colspan="1" rowspan="1">3.2E+00 TCID5o/mL(2.4E+02 copies/mL)</td></tr><tr><td colspan="1" rowspan="1">B1-3 Peru2-2002ZeptoMetrix 0810156CF</td><td colspan="1" rowspan="1">2.5E-01 TCID50/mL(5.4E+02 copies/mL)</td></tr><tr><td colspan="1" rowspan="1">A2-27 lowa A/2004ZeptoMetrix 0810164CF</td><td colspan="1" rowspan="1">5.8E-01 TCID5o/mL(1.8E+03 copies/mL)</td></tr><tr><td colspan="1" rowspan="1">B2-18 IA18-2003ZeptoMetrix 0810162CF</td><td colspan="1" rowspan="1">2.0E+00 TCID5o/mL(7.7E+02 copies/mL)</td></tr><tr><td colspan="1" rowspan="2">Human rhinovirus/enterovirus</td><td colspan="1" rowspan="1">Human Rhinovirus 1AZeptoMetrix 0810012CFN</td><td colspan="1" rowspan="1">2.1E-01 TCID50/mL(1.1E+00 copies/mL)</td></tr><tr><td colspan="1" rowspan="1">Enterovirus D68 US/MO/14-18947ATCC VR-1823</td><td colspan="1" rowspan="1">1.1E+01 TCID5o/mL(5.4E+01 copies/mL)</td></tr><tr><td colspan="1" rowspan="1">Influenza A virusSubtype H1-2009</td><td colspan="1" rowspan="1">Influenza A H1N1 pdmA/Michigan/45/15ZeptoMetrix 0810538CF</td><td colspan="1" rowspan="1">8.2E-01 TCID50/mL(3.4E+02 copies/mL</td></tr><tr><td colspan="1" rowspan="1">Influenza A virusSubtype H3</td><td colspan="1" rowspan="1">Influenza A H3N2A/Hong Kong/4801/14ZeptoMetrix 0810526CF</td><td colspan="1" rowspan="1">8.6E-01 TCID5o/mL(3.4E+02 copies/mL)</td></tr><tr><td colspan="1" rowspan="3">Influenza B virus</td><td colspan="1" rowspan="1">B/Florida/02/06 (Victoria Lineage)ZeptoMetrix 0810037CF</td><td colspan="1" rowspan="1">3.3E-02 TCID50/mL(1.6E+02 copies/mL)</td></tr><tr><td colspan="1" rowspan="1">B/Nevada/03/2011 (Victoria Lineage)BEI NR-44023</td><td colspan="1" rowspan="1">1.6E+00 CEID50/mL(4.3E+00 copies/mL)</td></tr><tr><td colspan="1" rowspan="1">B/Florida/04/06 (Yamagata Lineage)ZeptoMetrix 0810255CF</td><td colspan="1" rowspan="1">4.0E-01 TCID50/mL(3.2E+01 copies/mL)</td></tr><tr><td colspan="1" rowspan="3">Parainfluenza virus</td><td colspan="1" rowspan="1">Serotype 1ZeptoMetrix 0810014CF</td><td colspan="1" rowspan="1">4.6E+00 TCID5o/mL(1.4E+03 copies/mL)</td></tr><tr><td colspan="1" rowspan="1">Serotype 2ZeptoMetrix 0810015CF</td><td colspan="1" rowspan="1">1.4E+01 TCID50/mL(1.6E+02 copies/mL)</td></tr><tr><td colspan="1" rowspan="1">Serotype 3ZeptoMetrix 0810016CF</td><td colspan="1" rowspan="1">2.6E+01 TCID50/mL(6.1E+01 copies/mL)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Serotype 4ZeptoMetrix 0810060CF</td><td colspan="1" rowspan="1">2.0E+02 TCID5o/mL(1.1E+03 copies/mL)</td></tr><tr><td colspan="1" rowspan="2">Respiratory syncytial virus</td><td colspan="1" rowspan="1">Type A 2006ZeptoMetrix 0810040ACF</td><td colspan="1" rowspan="1">6.2E-02 TCID5o/mL(2.2E+01 copies/mL)</td></tr><tr><td colspan="1" rowspan="1">Type B 3/2015 Isolate #1ZeptoMetrix 0810479CF</td><td colspan="1" rowspan="1">2.8E-02 TCID50/mL(2.4E+01 copies/mL)</td></tr><tr><td></td><td colspan="1" rowspan="1">Bacteria</td><td></td></tr><tr><td colspan="1" rowspan="1">Chlamydia pneumoniae</td><td colspan="1" rowspan="1">AR-39ATCC 53592</td><td colspan="1" rowspan="1">2.0E+01 IFU/mL(1.4E+02 copies/mL)</td></tr><tr><td colspan="1" rowspan="1">Mycoplasma pneumoniae</td><td colspan="1" rowspan="1">M129ZeptoMetrix 0801579</td><td colspan="1" rowspan="1">1.0E+01 CCU/mL(2.1E+03 copies/mL)</td></tr><tr><td colspan="1" rowspan="1">Bordetella parapertussis</td><td colspan="1" rowspan="1">E595ZeptoMetrix 0801462</td><td colspan="1" rowspan="1">4.0E+01 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Bordetella pertussis</td><td colspan="1" rowspan="1">A639ZeptoMetrix 0801459</td><td colspan="1" rowspan="1">3.3E+02 CFU/mL(3.8E+02 copies/mL)</td></tr></table>

a LoD concentration may vary from what is listed based on the accuracy and precision of the quantification method. b IU $=$ International Units; BioFire Diagnostics quantified the WHO International Standard by quantitative real-time PCR to demonstrate that $8 . 2 \mathsf { E } ^ { + } 0 2$ IU/mL $= 8 . 2 \mathsf { E } + 0 2$ copies/mL c Testing for Coronavirus HKU1 utilized a clinical specimen due to the lack of availability of a cultured isolate. Viral concentration was determined in RNA copies/mL by quantitative real-time RT-PCR.

NOTE: LoD concentrations in copies/mL in Error! Reference source not found. above are based on extraction of nucleic acids from isolate cultures followed by quantitative real-time PCR (qPCR) or digital PCR (dPCR). The accuracy of concentrations may be affected by extraction efficiency, standard curve accuracy (qPCR only), assay conditions, inhibitors, and/or sequence variance. The quantification has not been compared to a reference material or other quantification methods.

NOTE: LoD concentrations of cultured viruses provided in units of $\tau c l D _ { 5 0 }$ $50 \%$ Tissue Culture Infectious Dose) or $\mathtt { c E I D 5 0 }$ $50 \%$ Chicken Embryo Infectious Dose) are not a direct count of viral particles or nucleic acid, but an indirect measure of viral concentration based on infectivity and cytotoxicity. $\pmb { T C I D 5 o / m L }$ and CEID50/mL will therefore vary depending on technique and methodology (including cell type, culture media and conditions, cytotoxicity of the virus, etc.). It is not appropriate to make determinations on relative sensitivity of detection for different cultures and/or different molecular assays based on LoD values measured in TCID50/mL or CEID50/mL.

# Analytical Reactivity (Inclusivity)

The analytical reactivity (inclusivity) of the SPOTFIRE R Panel assays was assessed by testing viral and bacterial isolates that represented clinical and genetic diversity and included the available phylogenetic, geographic, and temporal diversity of each SPOTFIRE R Panel analyte. Isolates were tested in triplicate at concentrations near the LoD.

A summary of analytical reactivity is included in Table 8 to Table 19.

NOTE: Influenza A assays will react variably with non-human influenza A viruses and rarely encountered human influenza A viruses that are not H1-2009 or H3; generally producing Uncertain: Influenza A virus or Influenza A virus (No Subtype Identified) results.

NOTE: The SPOTFIRE R Panel assays may react with vaccines that contain specific segments of the pathogen genome or full genome or vaccines containing attenuated/inactivated pathogen, including vaccines for SARS-CoV-2, influenza A (various subtypes), influenza B, poliovirus (human rhinovirus/enterovirus), and Bordetella pertussis. Care should be taken to minimize contamination of samples with vaccines, and clinical history of vaccine administration should be considered in the interpretation of results, particularly for vaccines administered by nasal spray.

Table 8. Summary of Reactivity to Adenovirus Isolates   

<table><tr><td rowspan=1 colspan=2>Species</td><td rowspan=1 colspan=1>Type</td><td rowspan=1 colspan=1>Source/Isolate ID</td><td rowspan=1 colspan=2>[Strain/Location/Year]</td><td rowspan=1 colspan=1>Result</td></tr><tr><td rowspan=3 colspan=2>A</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>ATCC VR-863</td><td rowspan=1 colspan=2>[Huie/Massachusetts]</td><td rowspan=28 colspan=1>Adenovirus Positive</td></tr><tr><td rowspan=1 colspan=1>18</td><td rowspan=1 colspan=1>ATCC VR-19</td><td rowspan=1 colspan=2>[Washington DC/1954]</td></tr><tr><td rowspan=1 colspan=1>31</td><td rowspan=1 colspan=1>ZeptoMetrix 0810073CF</td><td></td><td></td></tr><tr><td rowspan=11 colspan=2>B</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>ZeptoMetrix 0810062CF</td><td rowspan=1 colspan=2></td></tr><tr><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>ATCCVR-7</td><td rowspan=1 colspan=2>[Gomen/California/1954]</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>7a</td><td rowspan=1 colspan=1>ZeptoMetrix 0810021CF</td><td rowspan=1 colspan=2></td></tr><tr><td rowspan=1 colspan=1>7d/d2</td><td rowspan=1 colspan=1>UIRF</td><td rowspan=1 colspan=2>[lowa/2001]</td></tr><tr><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>ATCC VR-12</td><td rowspan=1 colspan=2>[Slobitski/Massachusetts]</td></tr><tr><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>ATCC VR-15</td><td rowspan=1 colspan=2>[De Wit/Netherlands/1955]</td></tr><tr><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>ATCC VR-17</td><td rowspan=1 colspan=2>[CH.79/Saudi Arabia/1955]</td></tr><tr><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>ATCC VR-1833</td><td rowspan=1 colspan=2>[128/Saudi Arabia/1956]</td></tr><tr><td rowspan=1 colspan=1>34</td><td rowspan=1 colspan=1>ATCC VR-716</td><td rowspan=1 colspan=2>[Compton/1972]</td></tr><tr><td rowspan=1 colspan=1>35</td><td rowspan=1 colspan=1>ATCC VR-718</td><td rowspan=1 colspan=2>[Holden]</td></tr><tr><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>ATCC VR-1602</td><td rowspan=1 colspan=2>[Wan, RIVM no. 88-1773]</td></tr><tr><td rowspan=4 colspan=2>C</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>ZeptoMetrix 0810050CF</td><td rowspan=1 colspan=2>-</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>W.H.O NIBSC 16/324</td><td rowspan=1 colspan=2>-</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>ZeptoMetrix 0810020CF</td><td rowspan=1 colspan=2>-</td></tr><tr><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>ATCC VR-6</td><td rowspan=1 colspan=2>[Tonsil 99]</td></tr><tr><td rowspan=3 colspan=2>D</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>ZeptoMetrix 0810069CF</td><td rowspan=1 colspan=2></td></tr><tr><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>ZeptoMetrix 0810115CF</td><td rowspan=1 colspan=2>[KB]</td></tr><tr><td rowspan=1 colspan=1>37</td><td rowspan=1 colspan=1>ZeptoMetrix 0810119CF</td><td rowspan=1 colspan=2></td></tr><tr><td rowspan=3 colspan=2>E</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>ZeptoMetrix 0810070CF</td><td rowspan=1 colspan=2>-</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>ATCC VR-1572</td><td rowspan=1 colspan=2>[RI-67/Missouri/1952-1953]</td></tr><tr><td rowspan=1 colspan=1>4a</td><td rowspan=1 colspan=1>UIRF</td><td rowspan=1 colspan=2>[S.Carolina/2004]</td></tr><tr><td rowspan=4 colspan=2>F</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>NCPV 0101141v</td><td rowspan=1 colspan=2></td></tr><tr><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>ZeptoMetrix 0810084CF</td><td rowspan=1 colspan=2>-</td></tr><tr><td rowspan=1 colspan=1>41</td><td rowspan=1 colspan=1>ATCC VR-930</td><td rowspan=1 colspan=2>[Tak (73-3544)]</td></tr><tr><td rowspan=1 colspan=1>41</td><td rowspan=1 colspan=1>ZeptoMetrix 0810085CF</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>[Tak]</td></tr></table>

Table 9. Summary of Reactivity to Bordetella parapertussis Isolates   

<table><tr><td rowspan=1 colspan=1>Source/Isolate ID</td><td rowspan=1 colspan=2>Strain/Location/Year</td><td rowspan=1 colspan=1>Result</td></tr><tr><td rowspan=1 colspan=1>ZeptoMetrix 0801462</td><td rowspan=1 colspan=2>[E595]</td><td rowspan=10 colspan=1>Bordetella parapertussisPositive</td></tr><tr><td rowspan=1 colspan=1>ATCC 9305</td><td rowspan=1 colspan=2>[517]</td></tr><tr><td rowspan=1 colspan=1>ATCC 53892</td><td rowspan=1 colspan=2>[PT28G]</td></tr><tr><td rowspan=1 colspan=1>ATCC 53893</td><td rowspan=1 colspan=2>[PT 26/28G]</td></tr><tr><td rowspan=1 colspan=1>ATCC 15237</td><td rowspan=1 colspan=2>[NCTC 10853]</td></tr><tr><td rowspan=1 colspan=1>ATCC 15311</td><td rowspan=1 colspan=2>[NCTC 5952]</td></tr><tr><td rowspan=1 colspan=1>ATCC BAA-587</td><td rowspan=1 colspan=2>[12822/Germany/1993]</td></tr><tr><td rowspan=1 colspan=1>ZeptoMetrix 0801461</td><td rowspan=1 colspan=2>[A747]</td></tr><tr><td rowspan=1 colspan=1>ZeptoMetrix 0801643</td><td rowspan=1 colspan=1>[C510]</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>ZeptoMetrix 0801644</td><td rowspan=1 colspan=1>[E838]</td><td rowspan=1 colspan=1></td></tr></table>

Table 10. Summary of Reactivity to Bordetella pertussis Isolates   

<table><tr><td rowspan=1 colspan=1>Source/Isolate ID</td><td rowspan=1 colspan=2>Strain/Location/Year</td><td rowspan=1 colspan=1>Result</td></tr><tr><td rowspan=1 colspan=1>ZeptoMetrix 0801459</td><td rowspan=1 colspan=2>[A639]</td><td rowspan=9 colspan=1>Bordetella pertussis Positive</td></tr><tr><td rowspan=1 colspan=1>ATCC 10380</td><td rowspan=1 colspan=2>[10-536]</td></tr><tr><td rowspan=1 colspan=1>ATCC 51445</td><td rowspan=1 colspan=2>[CNCTC Hp 12/63 [623]]</td></tr><tr><td rowspan=1 colspan=1>ATCC 8467</td><td rowspan=1 colspan=2>[F]</td></tr><tr><td rowspan=1 colspan=1>ATCC 9340</td><td rowspan=1 colspan=2>[5 [17921]</td></tr><tr><td rowspan=1 colspan=1>ATCC 9797</td><td rowspan=1 colspan=2>[18323 [NCTC 10739]</td></tr><tr><td rowspan=1 colspan=1>ATCC BAA-1335</td><td rowspan=1 colspan=1>[MN2531]</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>ATCC BAA-589</td><td rowspan=1 colspan=1>[Tohama]</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>ZeptoMetrix 0801460</td><td rowspan=1 colspan=2>[E431]</td></tr></table>

Table 11. Summary of Reactivity to Chlamydia pneumoniae Isolates   

<table><tr><td rowspan=1 colspan=1>Source/Isolate ID</td><td rowspan=1 colspan=3>Strain/Location/Year</td><td rowspan=1 colspan=1>Result</td></tr><tr><td rowspan=1 colspan=1>ATCC 53592</td><td rowspan=1 colspan=3>[AR-39]</td><td rowspan=7 colspan=1>Chlamydia pneumoniaePositive</td></tr><tr><td rowspan=1 colspan=1>ATCC VR-1310</td><td rowspan=1 colspan=3>[CWL-029]</td></tr><tr><td rowspan=1 colspan=1>ATCC VR-1356</td><td rowspan=1 colspan=3>[TWAR strain 2023]</td></tr><tr><td rowspan=1 colspan=1>ATCC VR-1360</td><td rowspan=1 colspan=2>[CM-1]</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>ATCC VR-1435</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>[J-21]</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>ATCC VR-1452</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>[A03]</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>ATCC VR-2282</td><td rowspan=1 colspan=3>[TWAR strain, TW-183/Taiwan/1965]</td></tr></table>

Table 12. Summary of Reactivity to Coronavirus Isolates   

<table><tr><td colspan="1" rowspan="1">Type</td><td colspan="1" rowspan="1">Source/Isolate ID</td><td colspan="1" rowspan="1">Strain/L</td><td colspan="1" rowspan="1">Result</td></tr><tr><td colspan="1" rowspan="1">NL63</td><td colspan="1" rowspan="1">ZeptoMetrix 0810228CF</td><td></td><td colspan="1" rowspan="1">Coronavirus (seasonal)</td></tr><tr><td colspan="1" rowspan="1">Type</td><td colspan="1" rowspan="1">Source/Isolate ID</td><td colspan="1" rowspan="1">Strain/Location/Year</td><td colspan="1" rowspan="1">Result</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">BEI NR-470</td><td colspan="1" rowspan="1">[Amsterdam/2003]</td><td colspan="1" rowspan="9">Positive</td></tr><tr><td colspan="1" rowspan="2">229E</td><td colspan="1" rowspan="1">ATCC VR-740</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">ZeptoMetrix 0810229CF</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="4">HKU1</td><td colspan="1" rowspan="1">Clinical Specimen</td><td colspan="1" rowspan="1">[Columbus OH, 2016]</td></tr><tr><td colspan="1" rowspan="1">Clinical Specimen</td><td colspan="1" rowspan="1">[South Carolina/2010]</td></tr><tr><td colspan="1" rowspan="1">Clinical Specimen</td><td colspan="1" rowspan="1">[France/2016]</td></tr><tr><td colspan="1" rowspan="1">Clinical Specimen</td><td colspan="1" rowspan="1">[France/2016]</td></tr><tr><td colspan="1" rowspan="2">OC43</td><td colspan="1" rowspan="1">ZeptoMetrix 0810024CF</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">ATCC VR-759</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="13">SARS-CoV-2 a</td><td colspan="1" rowspan="1">ATCC VR-1986HK</td><td colspan="1" rowspan="1">[USA-WA1/2020]</td><td colspan="1" rowspan="13">Coronavirus SARS-CoV-2Positive</td></tr><tr><td colspan="1" rowspan="1">ATCC VR-1991D</td><td colspan="1" rowspan="1">[Hong Kong/VM20001061/2020]</td></tr><tr><td colspan="1" rowspan="1">ATCC VR-1992D</td><td colspan="1" rowspan="1">[2019-nCoV/Italy-INMI1]</td></tr><tr><td colspan="1" rowspan="1">ATCC VR-1994D</td><td colspan="1" rowspan="1">[Germany/BavPat1/2020]</td></tr><tr><td colspan="1" rowspan="1">ATCC VR-3326D</td><td colspan="1" rowspan="1">[USA/CA_CDC_5574/2020]</td></tr><tr><td colspan="1" rowspan="1">BEI NR-52499 b</td><td colspan="1" rowspan="1">[England/02/2020]</td></tr><tr><td colspan="1" rowspan="1">BEI NR-52501</td><td colspan="1" rowspan="1">[Singapore/2/2020]</td></tr><tr><td colspan="1" rowspan="1">BEI NR-52503 d</td><td colspan="1" rowspan="1">[USA-IL1/2020]</td></tr><tr><td colspan="1" rowspan="1">BEI NR-52505 </td><td colspan="1" rowspan="1">[USA-AZ1/2020]</td></tr><tr><td colspan="1" rowspan="1">BEI NR-52507 f</td><td colspan="1" rowspan="1">[USA-CA3/2020]</td></tr><tr><td colspan="1" rowspan="1">BEI NR-52510 9</td><td colspan="1" rowspan="1">[Chile/Santiago_op4d1/2020]</td></tr><tr><td colspan="1" rowspan="1">BEI NR-53518 h</td><td colspan="1" rowspan="1">[New York-PV08410/2020]</td></tr><tr><td colspan="1" rowspan="1">LGC SeraCare AccuPlex™0505-0298</td><td colspan="1" rowspan="1">[Omicron B.1.1.529 Variant]</td></tr></table>

a See Error! Reference source not found. for additional SARS-CoV-2 reactivity predictions based on in silico analysis. b The following reagent was deposited by Professor Maria Zambon and obtained through BEI Resources, NIAID, NIH: Genomic RNA from SARS-Related Coronavirus 2, Isolate England/02/2020, NR-52499. c The following reagent was contributed by Duke-National University of Singapore, Programme in Emerging Infectious Diseases for distribution through BEI Resources, NIAID, NIH: Genomic RNA from SARS-Related Coronavirus 2, Isolate Singapore/2/2020, NR-52501. d The following reagent was deposited by the Centers for Disease Control and Prevention and obtained through BEI Resources, NIAID, NIH: Genomic RNA from SARS-Related Coronavirus 2, Isolate USA-IL1/2020, NR-52503. e The following reagent was deposited by the Centers for Disease Control and Prevention and obtained through BEI Resources, NIAID, NIH: Genomic RNA from SARS-Related Coronavirus 2, Isolate USA-AZ1/2020, NR-52505. f The following reagent was deposited by the Centers for Disease Control and Prevention and obtained through BEI Resources, NIAID, NIH: Genomic RNA from SARS-Related Coronavirus 2, Isolate USA-CA3/2020, NR-52507. g The following reagent was obtained through BEI Resources, NIAID, NIH: Genomic RNA from SARS-Related Coronavirus 2, Isolate Chile/Santiago_op4d1/2020, NR-52510. h The following reagent was obtained through BEI Resources, NIAID, NIH: Genomic RNA from SARS-Related Coronavirus 2, Isolate New York-PV08410/2020, NR53518. i Recombinant alphavirus that contains the full SARS-CoV-2 genome with mutations identified in the S and N gene of the SARS-CoV-2 Omicron variant B.1.1.529.

Table 13. Summary of Reactivity to Human Rhinovirus and Enterovirus Isolates   

<table><tr><td rowspan=1 colspan=1>Species</td><td rowspan=1 colspan=2>Serotype           Source/Isolate ID</td><td rowspan=1 colspan=1>[Strain/Location/Year]</td><td rowspan=1 colspan=1>Result</td></tr><tr><td rowspan=1 colspan=1></td><td></td><td></td><td rowspan=1 colspan=1>Human Rhinovirus</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=8 colspan=1>A</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>ZeptoMetrix0810012CFN</td><td rowspan=1 colspan=1>[1A]</td><td rowspan=14 colspan=1>Human rhinovirus/enterovirusPositive</td></tr><tr><td rowspan=1 colspan=1>77</td><td rowspan=1 colspan=1>ATCC VR-1187</td><td rowspan=1 colspan=1>[130-63]</td></tr><tr><td rowspan=1 colspan=1>85</td><td rowspan=1 colspan=1>ATCC VR-1195</td><td rowspan=1 colspan=1>[50-525-CV54]</td></tr><tr><td rowspan=1 colspan=1>34</td><td rowspan=1 colspan=1>ATCC VR-1365</td><td rowspan=1 colspan=1>[137-3]</td></tr><tr><td rowspan=1 colspan=1>57</td><td rowspan=1 colspan=1>ATCC VR-1600</td><td rowspan=1 colspan=1>[Ch47]</td></tr><tr><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>ATCC VR-1601</td><td rowspan=1 colspan=1>[68-CV11]</td></tr><tr><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>ATCC VR-283</td><td rowspan=1 colspan=1>[11757]</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>ATCC VR-482</td><td rowspan=1 colspan=1>[HGP]</td></tr><tr><td rowspan=6 colspan=1>B</td><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>ATCC VR-1663</td><td rowspan=1 colspan=1>[33342]</td></tr><tr><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>ATCC VR-284</td><td rowspan=1 colspan=1>[1059]</td></tr><tr><td rowspan=1 colspan=1>42</td><td rowspan=1 colspan=1>ATCC VR-1950</td><td rowspan=1 colspan=1>[56822]</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>ATCC VR-483</td><td rowspan=1 colspan=1>[FEB]</td></tr><tr><td rowspan=1 colspan=1>27</td><td rowspan=1 colspan=1>ATCC VR-1137</td><td rowspan=1 colspan=1>[5870]</td></tr><tr><td rowspan=1 colspan=1>83</td><td rowspan=1 colspan=1>ATCC VR-1193</td><td rowspan=1 colspan=1>[Baylor 7]</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Enterovirus</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=2 colspan=1>A</td><td rowspan=1 colspan=1>Enterovirus 71</td><td rowspan=1 colspan=1>ATCC VR-1432</td><td rowspan=1 colspan=1>[71 H]</td><td rowspan=11 colspan=1>Human rhinovirus/enterovirusPositive</td></tr><tr><td rowspan=1 colspan=1>Coxsackievirus 10</td><td rowspan=1 colspan=1>ATCC VR-168</td><td rowspan=1 colspan=1>[NY/1950]</td></tr><tr><td rowspan=6 colspan=1>B</td><td rowspan=1 colspan=1>Coxsackievirus 9</td><td rowspan=1 colspan=1>ZeptoMetrix 0810017CF</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>Echovirus 11</td><td rowspan=1 colspan=1>ZeptoMetrix 0810023CF</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>Coxsackievirus B3</td><td rowspan=1 colspan=1>ZeptoMetrix 0810074CF</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Coxsackievirus B4</td><td rowspan=1 colspan=1>ZeptoMetrix 0810075CF</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>Echovirus 6</td><td rowspan=1 colspan=1>ZeptoMetrix 0810076CF</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>Echovirus 9</td><td rowspan=1 colspan=1>ZeptoMetrix 0810077CF</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=2 colspan=1>C</td><td rowspan=1 colspan=1>Coxsackievirus A24</td><td rowspan=1 colspan=1>ATCC VR-583</td><td rowspan=1 colspan=1>[DN-19/TX/1963]</td></tr><tr><td rowspan=1 colspan=1>Coxsackievirus A21</td><td rowspan=1 colspan=1>ATCC VR-850</td><td rowspan=1 colspan=1>[Kuykendall/CA/1952]</td></tr><tr><td rowspan=1 colspan=1>D</td><td rowspan=1 colspan=1>Enterovirus D68</td><td rowspan=1 colspan=1>ATCC VR-1823</td><td rowspan=1 colspan=1>[US/MO/14-18947]</td></tr></table>

Table 14. Summary of Reactivity to Human Metapneumovirus Isolates   

<table><tr><td rowspan=1 colspan=1>Genotype</td><td rowspan=1 colspan=1>Serotype</td><td rowspan=1 colspan=1>Source/Isolate ID</td><td rowspan=1 colspan=1>[Location/Year]</td><td rowspan=1 colspan=1>Result</td></tr><tr><td rowspan=2 colspan=1>A1</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>ZeptoMetrix 0810160CF</td><td rowspan=1 colspan=1>[lowa 3/2002]</td><td rowspan=11 colspan=1>Human metapneumovirusPositive</td></tr><tr><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>ZeptoMetrix 0810161CF</td><td rowspan=1 colspan=1>[lowa 10/2003]</td></tr><tr><td rowspan=2 colspan=1>A2</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>ZeptoMetrix 0810163CF</td><td rowspan=1 colspan=1>[lowa 14/2003]</td></tr><tr><td rowspan=1 colspan=1>27</td><td rowspan=1 colspan=1>ZeptoMetrix 0810164CF</td><td rowspan=1 colspan=1>[lowa 27/2004]</td></tr><tr><td rowspan=2 colspan=1>B1</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>ZeptoMetrix 0810156CF</td><td rowspan=1 colspan=1>[Peru2-2002]</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>ZeptoMetrix 0810158CF</td><td rowspan=1 colspan=1>[Peru 3/2003]</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=5 colspan=1>B2</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>ZeptoMetrix 0810157CF</td><td rowspan=1 colspan=1>[Peru 1/2002]</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>ZeptoMetrix 0810159CF</td><td rowspan=1 colspan=1>[Peru 6/2003]</td></tr><tr><td rowspan=2 colspan=1>18</td><td rowspan=2 colspan=1>ZeptoMetrix 0810162CF</td><td rowspan=2 colspan=1>[IA18-2003]</td></tr><tr></tr><tr><td rowspan=1 colspan=1>Unknown</td><td rowspan=1 colspan=1>BEI NR-22232</td><td rowspan=1 colspan=1>[TN/91-316]</td></tr></table>

Table 15. Summary of Reactivity to Influenza A Isolates   

<table><tr><td rowspan=1 colspan=1>Type</td><td rowspan=1 colspan=1>Host</td><td rowspan=1 colspan=1>Source/Isolate ID</td><td rowspan=1 colspan=2>Strain/Location/Year</td><td rowspan=1 colspan=1>Result</td></tr><tr><td rowspan=8 colspan=1>H1N1pdm09</td><td rowspan=8 colspan=1>Human</td><td rowspan=1 colspan=1>ZeptoMetrix 0810538CF</td><td rowspan=1 colspan=2>[Michigan/45/15]</td><td rowspan=8 colspan=1>Influenza A virus(Subtype H1-2009) Positive</td></tr><tr><td rowspan=1 colspan=1>BEI NR-19823</td><td rowspan=1 colspan=2>[Netherlands/2629/2009]</td></tr><tr><td rowspan=1 colspan=1>BEI NR-42938</td><td rowspan=1 colspan=2>[Georgia/F32551/2012]</td></tr><tr><td rowspan=1 colspan=1>BEI NR-44345</td><td rowspan=1 colspan=2>[Hong Kong/H090-761-V1(0)/2009]</td></tr><tr><td rowspan=1 colspan=1>ZeptoMetrix 0810109CFJ</td><td rowspan=1 colspan=2>[Canada/6294/2009]</td></tr><tr><td rowspan=1 colspan=1>ZeptoMetrix 0810165CF</td><td rowspan=1 colspan=2>[California/07/2009]</td></tr><tr><td rowspan=1 colspan=1>ZeptoMetrix 0810166CF</td><td rowspan=1 colspan=1>[Mexico/4108/2009]</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>ZeptoMetrix 0810249CF</td><td rowspan=1 colspan=1>[SwineNY/03/2009]</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=9 colspan=1>H3N2</td><td rowspan=9 colspan=1>Human</td><td rowspan=1 colspan=1>ATCC VR-544</td><td rowspan=1 colspan=2>[Hong Kong/8/1968]</td><td rowspan=9 colspan=1>Influenza A virus(Subtype H3) Positive</td></tr><tr><td rowspan=1 colspan=1>ATCC VR-547</td><td rowspan=1 colspan=2>[Aichi/2/1968]</td></tr><tr><td rowspan=1 colspan=1>ATCC VR-776</td><td rowspan=1 colspan=2>[Alice]</td></tr><tr><td rowspan=1 colspan=1>ATCC VR-810</td><td rowspan=1 colspan=2>[Port Chalmers/1/1973]</td></tr><tr><td rowspan=1 colspan=1>ATCC VR-822</td><td rowspan=1 colspan=2>[Victoria/3/1975]</td></tr><tr><td rowspan=1 colspan=1>ZeptoMetrix 0810138CF</td><td rowspan=1 colspan=2>[Brisbane/10/2007]</td></tr><tr><td rowspan=1 colspan=1>ZeptoMetrix 0810238CF</td><td rowspan=1 colspan=2>[Texas/50/2012]</td></tr><tr><td rowspan=1 colspan=1>ZeptoMetrix 0810252CF</td><td rowspan=1 colspan=2>[Wisconsin/67/2005]</td></tr><tr><td rowspan=1 colspan=1>ZeptoMetrix 0810526CF</td><td rowspan=1 colspan=2>[Hong Kong/4801/14]</td></tr><tr><td rowspan=6 colspan=1>H1N1</td><td rowspan=3 colspan=1>Human</td><td rowspan=1 colspan=1>ZeptoMetrix 0810036CF</td><td rowspan=1 colspan=2>[New Caledonia/20/1999]</td><td rowspan=8 colspan=1>Influenza A virusPositive (No SubtypeIdentified)</td></tr><tr><td rowspan=1 colspan=1>ZeptoMetrix 0810036CFN</td><td rowspan=1 colspan=2>[Solomon Islands/3/2006]</td></tr><tr><td rowspan=1 colspan=1>ZeptoMetrix 0810244CF</td><td rowspan=1 colspan=2>[Brisbane/59/2007]</td></tr><tr><td rowspan=3 colspan=1>Swine</td><td rowspan=1 colspan=1>ATCC VR-333</td><td rowspan=1 colspan=2>[Swine/lowa/15/1930]</td></tr><tr><td rowspan=1 colspan=1>ATCC VR-897</td><td rowspan=1 colspan=2>[Swine/A/New Jersey/8/76]</td></tr><tr><td rowspan=1 colspan=1>ATCC VR-99</td><td rowspan=1 colspan=2>[Swine/1976/1931]</td></tr><tr><td rowspan=1 colspan=1>H2N2</td><td rowspan=1 colspan=1>Human</td><td rowspan=1 colspan=1>BEI NR-2775 a</td><td rowspan=1 colspan=2>[A/Japan/305/1957]</td></tr><tr><td rowspan=1 colspan=1>H5N3</td><td rowspan=1 colspan=1>Avian</td><td rowspan=1 colspan=1>BEI NR-9682 b</td><td rowspan=1 colspan=2>[A/Duck/Singapore/645/97]</td></tr><tr><td rowspan=1 colspan=1>H1N2</td><td rowspan=1 colspan=1>Recombinant</td><td rowspan=1 colspan=1>BEI NR-3478c</td><td rowspan=1 colspan=2>[Kilbourne F63 A/NWS/1934(HA) x A/RockefellerInstitute/5/1957 (NA)]</td><td rowspan=2 colspan=1>Uncertain: Influenza A virus</td></tr><tr><td rowspan=1 colspan=1>H10N7</td><td rowspan=1 colspan=1>Avian</td><td rowspan=1 colspan=1>BEI NR-2765d</td><td rowspan=1 colspan=2>[A/Chicken/Germany/N/49]</td></tr></table>

a The following reagent was obtained through BEI Resources, NIAID, NIH: Genomic RNA from Influenza A Virus, A/Japan/305/1957 (H2N2), NR-2775. b The following reagent was obtained through BEI Resources, NIAID, NIH: Genomic RNA from Kilbourne F181: A/duck/Singapore/645/1997 (H5N3), Wild Type, NR-9682. c The following reagent was obtained through BEI Resources, NIAID, NIH: Kilbourne F63: A/NWS/1934 (HA) $\mathsf { x }$ A/Rockefeller Institute/5/1957 (NA) (H1N2), Reassortant NWS-F, NR-3478. d The following reagent was obtained through BEI Resources, NIAID, NIH: Genomic RNA from Influenza A Virus, A/chicken/Germany/N/1949 (H10N7), NR-2765.

Table 16. Summary of Reactivity to Influenza B Isolates   

<table><tr><td rowspan=1 colspan=1>Lineage</td><td rowspan=1 colspan=1>Source/Isolate ID</td><td rowspan=1 colspan=1>[Strain/Location/Year]</td><td rowspan=1 colspan=1>Result</td></tr><tr><td rowspan=4 colspan=1>Yamagata</td><td rowspan=1 colspan=1>ZeptoMetrix 0810255CF</td><td rowspan=1 colspan=1>[Florida/04/06]</td><td rowspan=15 colspan=1>Influenza B virusPositive</td></tr><tr><td rowspan=1 colspan=1>ZeptoMetrix 0810239CF</td><td rowspan=1 colspan=1>[2/Massachusetts/2012]</td></tr><tr><td rowspan=1 colspan=1>ZeptoMetrix 0810241CF</td><td rowspan=1 colspan=1>[1/Wisconsin/2010]</td></tr><tr><td rowspan=1 colspan=1>ZeptoMetrix 0810256CF</td><td rowspan=1 colspan=1>[07/Florida/2004]</td></tr><tr><td rowspan=5 colspan=1>Victoria</td><td rowspan=1 colspan=1>ZeptoMetrix 0810037CF</td><td rowspan=1 colspan=1>[B/Florida/02/06]</td></tr><tr><td rowspan=1 colspan=1>BEI NR-44023</td><td rowspan=1 colspan=1>[B/Nevada/03/2011]</td></tr><tr><td rowspan=1 colspan=1>ATCC VR-823</td><td rowspan=1 colspan=1>[5/Hong Kong/1972]</td></tr><tr><td rowspan=1 colspan=1>CDC 2005743348</td><td rowspan=1 colspan=1>[1/Ohio/2005]</td></tr><tr><td rowspan=1 colspan=1>ZeptoMetrix 0810258CF</td><td rowspan=1 colspan=1>[2506/Malaysia/2004]</td></tr><tr><td rowspan=6 colspan=1>Unknown</td><td rowspan=1 colspan=1>ATCC VR-101</td><td rowspan=1 colspan=1>[Lee/1940]</td></tr><tr><td rowspan=1 colspan=1>ATCC VR-102</td><td rowspan=1 colspan=1>[Allen/1945]</td></tr><tr><td rowspan=1 colspan=1>ATCC VR-103</td><td rowspan=1 colspan=1>[GL/1739/1954]</td></tr><tr><td rowspan=1 colspan=1>ATCC VR-295</td><td rowspan=1 colspan=1>[2/Taiwan/1962]</td></tr><tr><td rowspan=1 colspan=1>ATCC VR-296</td><td rowspan=1 colspan=1>[1/Maryland/1959]</td></tr><tr><td rowspan=1 colspan=1>ATCC VR-786</td><td rowspan=1 colspan=1>[Brigit/Russia/1969]</td></tr></table>

Table 17. Summary of Reactivity to Mycoplasma pneumoniae Isolates   

<table><tr><td colspan="1" rowspan="1">Source/Isolate ID</td><td colspan="1" rowspan="1">Strain/Location/Year</td><td colspan="1" rowspan="1">Result</td></tr><tr><td colspan="1" rowspan="1">ZeptoMetrix 0801579</td><td colspan="1" rowspan="1">[M129]</td><td colspan="1" rowspan="1">EY</td></tr><tr><td colspan="1" rowspan="1">ATCC 29085</td><td colspan="1" rowspan="1">[PI 1428]</td><td colspan="1" rowspan="8">Mycoplasma pneumoniaePositive</td></tr><tr><td colspan="1" rowspan="1">ATCC 29342</td><td colspan="1" rowspan="1">[M129-B7]</td></tr><tr><td colspan="1" rowspan="1">ATCC 15492</td><td colspan="1" rowspan="1">[Mac]</td></tr><tr><td colspan="1" rowspan="1">ATCC 15531-TTR</td><td colspan="1" rowspan="1">[FH strain of Eaton Agent [NCTC 10119]</td></tr><tr><td colspan="1" rowspan="1">ATCC 15293</td><td colspan="1" rowspan="1">[M52]</td></tr><tr><td colspan="1" rowspan="1">ATCC 15377</td><td colspan="1" rowspan="1">[Bru]</td></tr><tr><td colspan="1" rowspan="1">ATCC 39505</td><td colspan="1" rowspan="1">[Mutant 22]</td></tr><tr><td colspan="1" rowspan="1">ATCC 49894</td><td colspan="1" rowspan="1">[UTMP-10P]</td></tr></table>

Table 18. Summary of Reactivity to Parainfluenza Virus Isolates   

<table><tr><td rowspan=1 colspan=1>Serotype</td><td rowspan=1 colspan=1>Subtype</td><td rowspan=1 colspan=1>Source/lsolate ID</td><td rowspan=1 colspan=1>Strain/Location/Year</td><td rowspan=1 colspan=1>Result</td></tr><tr><td rowspan=3 colspan=2>1</td><td rowspan=1 colspan=1>ZeptoMetrix 0810014CF</td><td></td><td rowspan=13 colspan=1>Parainfluenza virusPositive</td></tr><tr><td rowspan=1 colspan=1>ATCC VR-94</td><td rowspan=1 colspan=1>[C-35/1957]</td></tr><tr><td rowspan=1 colspan=1>BEI NR-48680</td><td rowspan=1 colspan=1>[FRA/29221106/2009]</td></tr><tr><td rowspan=2 colspan=1>2</td><td rowspan=2 colspan=1></td><td rowspan=1 colspan=1>ZeptoMetrix 0810015CF</td><td></td></tr><tr><td rowspan=1 colspan=1>ATCC VR-92</td><td rowspan=1 colspan=1>[Greer/1955]</td></tr><tr><td rowspan=3 colspan=1>3</td><td rowspan=3 colspan=1></td><td rowspan=1 colspan=1>ZeptoMetrix 0810016CF</td><td></td></tr><tr><td rowspan=1 colspan=1>ATCC VR-93</td><td rowspan=1 colspan=1>[C-243/1957]</td></tr><tr><td rowspan=1 colspan=1>BEI NR-3233</td><td rowspan=1 colspan=1>[NIH 47885 Wash/47885/57]</td></tr><tr><td rowspan=5 colspan=1>4</td><td rowspan=3 colspan=1>A</td><td rowspan=1 colspan=1>ZeptoMetrix 0810060CF</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>ATCC VR-1378</td><td rowspan=1 colspan=1>[M-25/1958]</td></tr><tr><td rowspan=1 colspan=1>ZeptoMetrix 0810060CF</td><td></td></tr><tr><td rowspan=2 colspan=1>B</td><td rowspan=1 colspan=1>ATCC VR-1377</td><td rowspan=1 colspan=1>[CH-19503/1962]</td></tr><tr><td rowspan=1 colspan=1>ZeptoMetrix 0810060BCF</td><td rowspan=1 colspan=1></td></tr></table>

Table 19. Summary of Reactivity to Respiratory Syncytial Virus Isolates   

<table><tr><td rowspan=1 colspan=1>Type</td><td rowspan=1 colspan=1>Source/Isolate ID</td><td rowspan=1 colspan=1>Strain/Location/Year</td><td rowspan=1 colspan=1>Result</td></tr><tr><td rowspan=5 colspan=1>A</td><td rowspan=1 colspan=1>ZeptoMetrix 0810040ACF</td><td rowspan=1 colspan=1>[2006]</td><td rowspan=11 colspan=1>Respiratory syncytial virusPositive</td></tr><tr><td rowspan=1 colspan=1>ATCC VR-26</td><td rowspan=1 colspan=1>[Long/Maryland/1956]</td></tr><tr><td rowspan=1 colspan=1>ATCC VR-1540</td><td rowspan=1 colspan=1>[A2/Melbourne/1961]</td></tr><tr><td rowspan=1 colspan=1>ZeptoMetrix 0810474CF</td><td rowspan=1 colspan=1>[2/2015 Isolate #2]</td></tr><tr><td rowspan=1 colspan=1>ZeptoMetrix 0810452CF</td><td rowspan=1 colspan=1>[12/2014 Isolate #2]</td></tr><tr><td rowspan=6 colspan=1>B</td><td rowspan=1 colspan=1>ZeptoMetrix 0810479CF</td><td rowspan=1 colspan=1>[3/2015 Isolate #1]</td></tr><tr><td rowspan=1 colspan=1>ZeptoMetrix 0810040CF</td><td rowspan=1 colspan=1>[Ch-93 (18)-18]</td></tr><tr><td rowspan=1 colspan=1>ATCC VR-1400</td><td rowspan=1 colspan=1>[WV/14617/1985]</td></tr><tr><td rowspan=1 colspan=1>ATCC VR-955</td><td rowspan=1 colspan=1>[9320/Massachusetts/1977]</td></tr><tr><td rowspan=1 colspan=1>ATCC VR-1580</td><td rowspan=1 colspan=1>[18537/WashingtonDC/1962]</td></tr><tr><td rowspan=1 colspan=1>ZeptoMetrix 0810451CF</td><td rowspan=1 colspan=1>[11/2014 Isolate #2]</td></tr></table>

# In Silico Reactivity Predictions for Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2) Assays

Evaluation of analytical reactivity for the SPOTFIRE R Panel SARS-CoV-2 assays (SARS-CoV2-1 and SARSCoV2-2) was based on in silico analysis of all available sequences in the GISAID database as of December 21, 2022. In total, 11,989,970 sequences from around the globe were aligned to the assay primers.

This analysis determined that the ${ > } 9 9 . 9 9 \%$ of 11,079,180 sequences will be detected by one or both SPOTFIRE R Panel SARS-CoV-2 assays based on homology and mismatch location with one or both sets of primers. A limitation on detection (both assays impaired) is predicted for less than $0 . 0 0 4 \%$ of the sequences evaluated (427 / 11,989,970) (Table 20).

The sequences evaluated include lineages and variants of concern (VOC) or variants under investigation (VUI) that may have important epidemiological, immunological, or pathogenic properties from a public health perspective, such as Delta and Omicron variants. Variants evaluated are listed in the BioFire® Respiratory Panels SARS-CoV-2 Reactivity Tech Note technical note at www.biofiredx.com/support/documents.

All lineages and variants of public health interest identified as of December 2022 are predicted to be detected; new sequences and variants will continue to be monitored for impacts on detection by the SPOTFIRE R Panel assays.

# Table 20. In silico Prediction of SARS-CoV-2 Detection by SPOTFIRE R Panel Assays

$+ / +$ indicates detected by both assays with no impairment, $+ / -$ - indicates detection by one assay with no impairment and potential for impaired detection by the other assay, $\mathrm { _ - } / \mathrm { _ - }$ indicates potential for impaired detection by both assays   

<table><tr><td rowspan=1 colspan=5>In silico prediction</td></tr><tr><td rowspan=1 colspan=2>Predicted Assay Result</td><td rowspan=1 colspan=2>SARS-CoV2-1</td><td rowspan=2 colspan=1>Total Sequences</td></tr><tr><td rowspan=1 colspan=2>Number of Sequences</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>•</td></tr><tr><td rowspan=2 colspan=1>SARS-CoV2-2</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>11,759,674</td><td rowspan=1 colspan=1>185,745</td><td rowspan=1 colspan=1>11,989,543 / 11,989,970</td></tr><tr><td rowspan=1 colspan=1>:</td><td rowspan=1 colspan=1>44,124</td><td rowspan=1 colspan=1>427</td><td rowspan=1 colspan=1>99.9964%</td></tr></table>

# Analytical Specificity (Cross-Reactivity & Exclusivity)

The potential for non-specific amplification and detection by the SPOTFIRE R Panel assays was evaluated by in silico analysis of available sequences and also by testing of high concentrations of on-panel and off-panel organisms. The organisms evaluated included relevant bacteria, fungi, and viruses that are either phylogenetically related to organisms detected by the SPOTFIRE R Panel or pathogenic/commensal organisms that may be present in NPS specimens. Each organism was tested in triplicate at the highest possible concentration (generally ${ \ge } 1 . 0 \mathsf { E } { + } 0 7 $ units/mL for bacteria and ${ \ge } 1 . 0 \mathsf { E } { + } 0 5 $ units/mL for viruses).

In silico analysis and testing identified a risk of SARS-CoV-2 assay cross-reactivity with a few sequences of SARSlike viruses isolated from bats and pangolin as well as intra-panel cross-reactivity with Bordetella species and Influenza A subtypes of swine origin. A summary of potential cross-reactivity is provided in Table 21. The on-panel and off-panel isolates and concentrations tested are listed in Table 22 and Table 23 respectively.

Table 21. Predicted and Observed Cross-Reactivity of the SPOTFIRE R Panel   

<table><tr><td rowspan=1 colspan=1>Cross-Reactive Organism/Sequence</td><td rowspan=1 colspan=1>SPOTFIRE R Panel Analyte Result</td><td rowspan=1 colspan=1>Description</td></tr><tr><td rowspan=1 colspan=1>Bat coronavirusPangolin coronavirusBat SARS-like coronavirus</td><td rowspan=1 colspan=1>Coronavirus SARS-CoV-2a</td><td rowspan=1 colspan=1>The SARS-CoV-2 assays can amplify selectsequences from closely related sarbecovirusesisolated from bats and pangolin.</td></tr><tr><td rowspan=1 colspan=1>Bordetella bronchiseptica(isolates containing IS1001 gene)</td><td rowspan=1 colspan=1>Bordetella parapertussis</td><td rowspan=1 colspan=1>Some strains of B. bronchiseptica containIS1001 insertion sequences identical to thosepresent in B. parapertussis. These sequenceswill be amplified by the IS1001 assay andreported as Bordetella parapertussis.</td></tr><tr><td rowspan=1 colspan=1>Bordetella bronchisepticabBordetella parapertussisbBordetella pertussis</td><td rowspan=1 colspan=1>Human rhinovirus/enterovirusc</td><td rowspan=1 colspan=1>The HRV/EV assay designed to target HumanRhinovirus/Enterovirus may amplify theoxidoreductase gene from Bordetella species (B.pertussis, B. parapertussis, and B.bronchiseptica) when organisms are present at ahigh concentration. Cross-reactivity with B.pertussis was observed at a concentration of1.3E+10 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Bovine picornavirusCanine picornavirus</td><td rowspan=1 colspan=1>Human rhinovirus/enterovirusa</td><td rowspan=1 colspan=1>Bovine and canine picornaviruses may bedetected and reported as Humanrhinovirus/enterovirus when present at highconcentration.</td></tr><tr><td rowspan=1 colspan=1>Influenza A H1N1(swine origin)</td><td rowspan=1 colspan=1>Influenza A Subtype H1-2009a</td><td rowspan=1 colspan=1>The FluA-H1-2009 assay may react with the H1hemagglutinin gene sequences from viruses ofswine origin.The SPOTFIRE R Panel will report eitherInfluenza A Virus (Subtype H1-2009) or InfluenzaA Virus (No Subtype Identified), depending onthe strain and concentration in the sample.</td></tr><tr><td rowspan=1 colspan=1>Chlamydia gallinacea</td><td rowspan=1 colspan=1>Chlamydia pneumoniaea</td><td rowspan=1 colspan=1>Chlamydia gallinacea may be detected andreported as Chlamydia pneumoniae whenpresent at high concentration.</td></tr></table>

a Indicated cross-reactivity is predicted based on in silico analysis. b Cross-reactivity between the HRV/EV assay and B. bronchiseptica and $B .$ . parapertussis is predicted based on in silico analysis but was not observed when testing organisms at the highest possible concentrations $8 . 3 \mathsf { E } ^ { + 0 9 }$ cells/mL for B. bronchiseptica and $4 . 6 \mathsf E { + } 0 9 $ CFU/mL for B. parapertussis). c Cross-reactivity between the HRV/EV assay and B. parapertussis or B. pertussis will have positive results reported for Human Rhinovirus/Enterovirus and B. parapertussis or B. pertussis, respectively. Cross-reactivity between the HRV/EV assay and B. bronchiseptica isolates that do not carry the IS1001 sequence will only have positive results reported for Human rhinovirus/enterovirus. Cross-reactivity between the HRV/EV assay and B. bronchiseptica isolates that contain the IS1001 sequence will have positive results reported for Human Rhinovirus/Enterovirus and B. parapertussis.

Table 22. Summary of Results for On-Panel Organisms Tested During Evaluation of Analytical Specificity of the SPOTFIRE R Panel   

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Isolate ID</td><td rowspan=1 colspan=1>Concentration Tested</td><td rowspan=1 colspan=1>Observed Cross-Reactivity</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Bacteria</td><td rowspan=1 colspan=1>eria</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Bordetella parapertussis</td><td rowspan=1 colspan=1>ZeptoMetrix 0801462</td><td rowspan=1 colspan=1>4.6E+09 CFU/mL</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=2 colspan=1>Bordetella pertussis</td><td rowspan=2 colspan=1>ZeptoMetrix 0801459</td><td rowspan=1 colspan=1>1.3E+10 CFU/mL</td><td rowspan=1 colspan=1>Human rhinovirus/enterovirusa</td></tr><tr><td rowspan=1 colspan=1>1.3E+09 CFU/mL</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Chlamydia pneumoniae</td><td rowspan=1 colspan=1>ATCC 53592</td><td rowspan=1 colspan=1>2.9E+07 IFU/mL</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Mycoplasma pneumoniae</td><td rowspan=1 colspan=1>ZeptoMetrix 0801579</td><td rowspan=1 colspan=1>2.5E+07 CCU/mL</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Viruses</td><td rowspan=1 colspan=1>ses</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Adenovirus A</td><td rowspan=1 colspan=1>ZeptoMetrix 0810073CF</td><td rowspan=1 colspan=1>1.4E+05 TCID5o/mL</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Adenovirus B</td><td rowspan=1 colspan=1>ZeptoMetrix 0810062CF</td><td rowspan=1 colspan=1>1.2E+07 TCID5o/mL</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Adenovirus C</td><td rowspan=1 colspan=1>ZeptoMetrix 0810110CF</td><td rowspan=1 colspan=1>2.2E+06 TCID5o/mL</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Adenovirus D</td><td rowspan=1 colspan=1>ZeptoMetrix 0810119CF</td><td rowspan=1 colspan=1>1.7E+05 TCID50/mL</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Adenovirus E</td><td rowspan=1 colspan=1>ZeptoMetrix 0810070CF</td><td rowspan=1 colspan=1>1.4E+05 TCID5o/mL</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Adenovirus F</td><td rowspan=1 colspan=1>ZeptoMetrix 0810085CF</td><td rowspan=1 colspan=1>1.1E+06 TCID50/mL</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Coronavirus 229E</td><td rowspan=1 colspan=1>ATCC VR-740</td><td rowspan=1 colspan=1>8.9E+06 TCID5o/mL</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Coronavirus HKU1</td><td rowspan=1 colspan=1>Clinical Specimens</td><td rowspan=1 colspan=1>4.5E+07 copies/mL</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Coronavirus NL63</td><td rowspan=1 colspan=1>ZeptoMetrix 0810228CF</td><td rowspan=1 colspan=1>5.0E+05 TCID5o/mL</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Coronavirus OC43</td><td rowspan=1 colspan=1>ZeptoMetrix 0810024CF</td><td rowspan=1 colspan=1>3.6E+05 TCID5o/mL</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Coronavirus SARS-CoV-2 (heat-inactivated)</td><td rowspan=1 colspan=1>ATCC VR-1986HK</td><td rowspan=1 colspan=1>7.6E+07 copies/mL</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Enterovirus D68</td><td rowspan=1 colspan=1>ATCC VR-1823</td><td rowspan=1 colspan=1>1.6E+07 TCID5o/mL</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Human Metapneumovirus A1</td><td rowspan=1 colspan=1>ZeptoMetrix 0810161CF</td><td rowspan=1 colspan=1>2.5E+05 TCID5o/mL</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Human Metapneumovirus A2</td><td rowspan=1 colspan=1>ZeptoMetrix 0810164CF</td><td rowspan=1 colspan=1>3.6E+05 TCID50/mL</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Human Metapneumovirus B1</td><td rowspan=1 colspan=1>ZeptoMetrix 0810156CF</td><td rowspan=1 colspan=1>1.6E+04 TCID50/mL</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Human Metapneumovirus B2</td><td rowspan=1 colspan=1>ZeptoMetrix 0810162CF</td><td rowspan=1 colspan=1>1.3E+06 TCID5o/mL</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Human rhinovirus A1</td><td rowspan=1 colspan=1>ZeptoMetrix 0810012CFN</td><td rowspan=1 colspan=1>1.3E+06 TCID5o/mL</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Influenza A H1N1pdm09</td><td rowspan=1 colspan=1>ZeptoMetrix 0810538CF</td><td rowspan=1 colspan=1>1.4E+05 TCID50/mL</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Influenza A H3N2</td><td rowspan=1 colspan=1>ZeptoMetrix 0810526CF</td><td rowspan=1 colspan=1>7.2E+05 TCID5o/mL</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=2 colspan=1>Influenza B (Victoria Lineage</td><td rowspan=1 colspan=1>BEI NR-44023</td><td rowspan=1 colspan=1>2.8E+08 CEID5o/mL</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>ZeptoMetrix 0810037CF</td><td rowspan=1 colspan=1>2.5E+05 TCID50/mL</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Influenza B (Yamagata Lineage)</td><td rowspan=1 colspan=1>ZeptoMetrix 0810256CF</td><td rowspan=1 colspan=1>2.1E+04 TCID5o/mL</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza virus 1</td><td rowspan=1 colspan=1>ZeptoMetrix 0810014CF</td><td rowspan=1 colspan=1>4.2E+05 TCID5o/mL</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza virus 2</td><td rowspan=1 colspan=1>ZeptoMetrix 0810015CF</td><td rowspan=1 colspan=1>1.2E+07 TCID5o/mL</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza virus 3</td><td rowspan=1 colspan=1>ZeptoMetrix 0810016CF</td><td rowspan=1 colspan=1>3.4E+07 TCID5o/mL</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza virus 4</td><td rowspan=1 colspan=1>ZeptoMetrix 0810060CF</td><td rowspan=1 colspan=1>3.4E+07 TCID5o/mL</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Respiratory syncytial virus A</td><td rowspan=1 colspan=1>ZeptoMetrix 0810040ACF</td><td rowspan=1 colspan=1>4.2E+05 TCID5o/mL</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Respiratory syncytial virus B</td><td rowspan=1 colspan=1>ZeptoMetrix 0810479CF</td><td rowspan=1 colspan=1>4.2E+05 TCID5o/mL</td><td rowspan=1 colspan=1>None</td></tr></table>

a The HRV/EV assay may amplify off-target sequences found in strains of Bordetella species (B. pertussis, B. parapertussis, and B. bronchiseptica) when present at a concentration ${ \ge } 1 . 3 5 { + } 1 0 $ CFU/mL.

Table 23. Summary of Results for Off-Panel Organisms Tested During Evaluation of Analytical Specificity of the SPOTFIRE R Panel   

<table><tr><td colspan="1" rowspan="1">Organism</td><td colspan="1" rowspan="1">Isolate ID</td><td colspan="1" rowspan="1">Concentration Tested</td><td colspan="1" rowspan="1">Concentration Tested Observed Cross-Reactivity</td></tr><tr><td></td><td colspan="1" rowspan="1">Bacteria</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Arcanobacterium bernardiae</td><td colspan="1" rowspan="1">ATCC BAA-441</td><td colspan="1" rowspan="1">1.6E+09 cells/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Arcanobacterium haemolyticum</td><td colspan="1" rowspan="1">ATCC 9345</td><td colspan="1" rowspan="1">1.5E+08 cells/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Arcanobacterium pyogenes</td><td colspan="1" rowspan="1">ATCC 49698</td><td colspan="1" rowspan="1">6.7E+09 cells/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Bacillus cereus</td><td colspan="1" rowspan="1">ATCC 7064</td><td colspan="1" rowspan="1">8.3E+09 cells/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="6">Bordetella bronchiseptica</td><td colspan="1" rowspan="1">ATCC 10580</td><td colspan="1" rowspan="1">8.3E+09 cells/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">ATCC 4617</td><td colspan="1" rowspan="1">7.9E+09 cells/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">ATCC 19395</td><td colspan="1" rowspan="1">7.9E+09 cells/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">NRRL B-59914</td><td colspan="1" rowspan="1">7.1E+09 cells/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="2">NRRL B-59909a</td><td colspan="1" rowspan="1">2.8E+01 cells/mLb</td><td colspan="1" rowspan="1">Bordetella parapertussisa</td></tr><tr><td colspan="1" rowspan="1">2.8E+00 cells/mLb</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Bordetella holmesii</td><td colspan="1" rowspan="1">ATCC 700052</td><td colspan="1" rowspan="1">8.3E+09 cells/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Burkholderia cepacia</td><td colspan="1" rowspan="1">ATCC 51671</td><td colspan="1" rowspan="1">7.9E+09 cells/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Campylobacter rectus</td><td colspan="1" rowspan="1">ATCC 33238</td><td colspan="1" rowspan="1">7.6E+07 cells/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Chlamydia trachomatis</td><td colspan="1" rowspan="1">ZeptoMetrix 0801775</td><td colspan="1" rowspan="1">1.3E+08 IFU/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium diptheriae</td><td colspan="1" rowspan="1">ATCC 27010</td><td colspan="1" rowspan="1">8.0E+09 cells/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium pseudodiphtheriticum</td><td colspan="1" rowspan="1">ATCC 10700</td><td colspan="1" rowspan="1">8.7E+09 cells/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Enterococcus casseliflavus</td><td colspan="1" rowspan="1">ATCC 49605</td><td colspan="1" rowspan="1">8.0E+09 cells/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Enterococcus faecalis</td><td colspan="1" rowspan="1">ZeptoMetrix 0801637</td><td colspan="1" rowspan="1">8.0E+09 CFU/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">ATCC BAA-2196</td><td colspan="1" rowspan="1">7.2E+09 cells/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Fusobacterium necrophorum ssp. funduliforme</td><td colspan="1" rowspan="1">ATCC 51357</td><td colspan="1" rowspan="1">4.4E+08 cells/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Fusobacterium nucleatum</td><td colspan="1" rowspan="1">ATCC 25586</td><td colspan="1" rowspan="1">4.9E+08 cells/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Fusobacterium varium</td><td colspan="1" rowspan="1">ATCC 27725</td><td colspan="1" rowspan="1">1.6E+08 cells/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Gemella haemolysans</td><td colspan="1" rowspan="1">ATCC 10379</td><td colspan="1" rowspan="1">4.0E+09 cells/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Gemella morbillorum</td><td colspan="1" rowspan="1">ATCC 27824</td><td colspan="1" rowspan="1">1.0E+08 cells/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Granulicatella adiacens</td><td colspan="1" rowspan="1">ATCC 49175</td><td colspan="1" rowspan="1">1.3E+09 cells/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Haemophilus influenzae</td><td colspan="1" rowspan="1">ATCC 10211</td><td colspan="1" rowspan="1">8.3E+09 cells/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Haemophilus parahaemolyticus</td><td colspan="1" rowspan="1">ATCC 49700</td><td colspan="1" rowspan="1">8.7E+09 cells/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">CDC AR#0115</td><td colspan="1" rowspan="1">7.3E+09 CFU/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Organism</td><td colspan="1" rowspan="1">Isolate ID</td><td colspan="1" rowspan="1">Concentration Tested</td><td colspan="1" rowspan="1">Observed Cross-Reactivity</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus rhamnosus</td><td colspan="1" rowspan="1">ATCC 7469</td><td colspan="1" rowspan="1">7.9E+09 cells/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Lactococcus lactis</td><td colspan="1" rowspan="1">ATCC 29146</td><td colspan="1" rowspan="1">6.2E+09 cells/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Legionella pneumophila</td><td colspan="1" rowspan="1">ATCC 33215</td><td colspan="1" rowspan="1">7.0E+09 cells/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Leptotrichia buccalis</td><td colspan="1" rowspan="1">ATCC 14201</td><td colspan="1" rowspan="1">4.4E+08 cells/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Moraxella catarrhalis</td><td colspan="1" rowspan="1">ATCC 43627</td><td colspan="1" rowspan="1">7.2E+09 cells/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Mycobacterium tuberculosis</td><td colspan="1" rowspan="1">ZeptoMetrix 0801660</td><td colspan="1" rowspan="1">6.1E+06 CFU/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Mycoplasma buccale</td><td colspan="1" rowspan="1">Mycoplasma ExperienceNC10136</td><td colspan="1" rowspan="1">1.4E+07 CFU/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Mycoplasma faucium</td><td colspan="1" rowspan="1">Mycoplasma ExperienceNC10174</td><td colspan="1" rowspan="1">1.4E+06 CFU/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Mycoplasma fermentans</td><td colspan="1" rowspan="1">Mycoplasma ExperienceNC1017</td><td colspan="1" rowspan="1">2.8E+07 CFU/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Mycoplasma genitalium</td><td colspan="1" rowspan="1">Mycoplasma ExperienceNC10195</td><td colspan="1" rowspan="1">1.8E+06 CFU/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Mycoplasma hominis</td><td colspan="1" rowspan="1">Mycoplasma ExperienceNC10111</td><td colspan="1" rowspan="1">1.2E+07 CFU/ml</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Mycoplasma lipophilum</td><td colspan="1" rowspan="1">Mycoplasma ExperienceNC10173</td><td colspan="1" rowspan="1">1.5E+06 CFU/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Mycoplasma orale</td><td colspan="1" rowspan="1">Mycoplasma ExperienceNC1012</td><td colspan="1" rowspan="1">2.2E+07 CFU/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Mycoplasma salivarium</td><td colspan="1" rowspan="1">Mycoplasma ExperienceNC10113</td><td colspan="1" rowspan="1">4.4E+06 CFU/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Neisseria elongata</td><td colspan="1" rowspan="1">ATCC 25295</td><td colspan="1" rowspan="1">8.5E+09 cells/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Neisseria gonorrhoeae</td><td colspan="1" rowspan="1">ZeptoMetrix 0801482</td><td colspan="1" rowspan="1">4.9E+07 CFU/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Neisseria lactamica</td><td colspan="1" rowspan="1">ATCC 23971</td><td colspan="1" rowspan="1">2.7E+09 cells/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Neisseria meningitidis</td><td colspan="1" rowspan="1">ATCC 13113</td><td colspan="1" rowspan="1">7.4E+09 cells/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Neisseria sicca</td><td colspan="1" rowspan="1">ATCC 9913</td><td colspan="1" rowspan="1">7.2E+09 cells/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Neisseria subflava</td><td colspan="1" rowspan="1">ATCC 49275</td><td colspan="1" rowspan="1">8.0E+09 cells/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Parvimonas micra</td><td colspan="1" rowspan="1">ATCC 33270</td><td colspan="1" rowspan="1">6.0E+07 cells/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Pneumocystis carinii</td><td colspan="1" rowspan="1">ATCC PRA-159</td><td colspan="1" rowspan="1">1.0E+07 nuclei/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Porphyromonas endodontalis</td><td colspan="1" rowspan="1">ATCC 35406</td><td colspan="1" rowspan="1">1.6E+07 cells/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Porphyromonas gingivalis</td><td colspan="1" rowspan="1">ATCC BAA-308</td><td colspan="1" rowspan="1">5.0E+08 cells/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Prevotella histicola</td><td colspan="1" rowspan="1">BEI HM-471</td><td colspan="1" rowspan="1">9.0E+08 cell/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Prevotella melaninogenica</td><td colspan="1" rowspan="1">ATCC 25845</td><td colspan="1" rowspan="1">6.9E+08 cells/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Prevotella oralis</td><td colspan="1" rowspan="1">ATCC 33322</td><td colspan="1" rowspan="1">6.2E+08 cells/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">CDC AR#0092</td><td colspan="1" rowspan="1">8.3E+09 cells/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Rhodococcus equi</td><td colspan="1" rowspan="1">ATCC 33706</td><td colspan="1" rowspan="1">7.3E+09 cells/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Serratia marcescens</td><td colspan="1" rowspan="1">ATCC 27137</td><td colspan="1" rowspan="1">8.9E+09 cells/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">ATCC BAA-1700</td><td colspan="1" rowspan="1">7.4E+09 cells/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus epidermidis</td><td colspan="1" rowspan="1">ATCC 12228</td><td colspan="1" rowspan="1">8.0E+09 cells/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus haemolyticus</td><td colspan="1" rowspan="1">ATCC 29968</td><td colspan="1" rowspan="1">8.0E+09 cells/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus intermedius</td><td colspan="1" rowspan="1">ATCC 29663</td><td colspan="1" rowspan="1">8.2E+09 cells/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus saprophyticus</td><td colspan="1" rowspan="1">ATCC 15305</td><td colspan="1" rowspan="1">8.1E+09 cells/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Streptococcus agalactiae</td><td colspan="1" rowspan="1">ATCC 13813</td><td colspan="1" rowspan="1">6.0E+09 cells/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Streptococcus anginosus</td><td colspan="1" rowspan="1">ATCC 700231</td><td colspan="1" rowspan="1">7.1E+09 cells/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Streptococcus constellatus ssp. pharyngis</td><td colspan="1" rowspan="1">NCTC 13122</td><td colspan="1" rowspan="1">5.6E+08 cells/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="7">Streptococcus dysgalactiae ssp. dysgalactiae</td><td colspan="1" rowspan="1">ATCC 43078</td><td colspan="1" rowspan="1">6.7E+09 cells/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">NCTC 4669</td><td colspan="1" rowspan="1">7.4E+09 cells/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">NCTC 4335</td><td colspan="1" rowspan="1">8.4E+09 cells/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">NCTC 4670</td><td colspan="1" rowspan="1">6.6E+09 cells/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">CCUG 27665</td><td colspan="1" rowspan="1">7.4E+09 cells/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">CCUG 28112</td><td colspan="1" rowspan="1">6.7E+09 cells/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">CCUG 28114</td><td colspan="1" rowspan="1">7.5E+09 cells/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Streptococcus dysgalactiae spequismilis(isolated from human)</td><td colspan="1" rowspan="1">ZeptoMetrix 0801516</td><td colspan="1" rowspan="1">7.8E+08 CFU/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">occus dysgalacti seuismilis(isolated from pig)</td><td colspan="1" rowspan="1">CCUG 28117</td><td colspan="1" rowspan="1">7.1E+09 cells/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="2">Streptococcus dysgalactiae ssp. equisimilis(isolated from horse)</td><td colspan="1" rowspan="1">CCUG 27664</td><td colspan="1" rowspan="1">7.5E+09 cells/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">ATCC 10009</td><td colspan="1" rowspan="1">6.9E+09 cells/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Streptococcus gallolyticus</td><td colspan="1" rowspan="1">ATCC 43143</td><td colspan="1" rowspan="1">2.8E+09 cells/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Streptococcus gordonii</td><td colspan="1" rowspan="1">ATCC 10558</td><td colspan="1" rowspan="1">4.5E+09 cells/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Streptococcus intermedius</td><td colspan="1" rowspan="1">ATCC 27335</td><td colspan="1" rowspan="1">2.9E+09 cells/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Streptococcus mitis</td><td colspan="1" rowspan="1">ATCC 15914</td><td colspan="1" rowspan="1">3.2E+09 cells/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Streptococcus mutans</td><td colspan="1" rowspan="1">ATCC 25175</td><td colspan="1" rowspan="1">2.3E+09 cells/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Streptococcus oralis</td><td colspan="1" rowspan="1">ATCC 10557</td><td colspan="1" rowspan="1">1.1E+09 cells/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Streptococcus parasanguinis</td><td colspan="1" rowspan="1">ATCC 15912</td><td colspan="1" rowspan="1">7.8E+09 cells/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Streptococcus pneumoniae</td><td colspan="1" rowspan="1">ATCC 49619</td><td colspan="1" rowspan="1">2.5E+08 cells/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Streptococcus pyogenes</td><td colspan="1" rowspan="1">ATCC 12344</td><td colspan="1" rowspan="1">3.4E+08 cells/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Streptococcus salivarius</td><td colspan="1" rowspan="1">ATCC 13419</td><td colspan="1" rowspan="1">6.6E+09 cells/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Streptococcus sanguinis</td><td colspan="1" rowspan="1">ATCC 10556</td><td colspan="1" rowspan="1">1.1E+09 cells/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Tannerella forsythia</td><td colspan="1" rowspan="1">ATCC BAA-2717</td><td colspan="1" rowspan="1">2.6E+08 cells/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Treponema denticola</td><td colspan="1" rowspan="1">ATCC 33520</td><td colspan="1" rowspan="1">2.2E+08 cells/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Ureaplasma urealyticum</td><td colspan="1" rowspan="1">ATCC 27618</td><td colspan="1" rowspan="1">5.7E+07 cells/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Veillonella parvula</td><td colspan="1" rowspan="1">ATCC 10790</td><td colspan="1" rowspan="1">4.7E+08 cells/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td></td><td colspan="1" rowspan="1">Fungi</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Candida albicans</td><td colspan="1" rowspan="1">ATCC MYA-2876</td><td colspan="1" rowspan="1">2.8E+08 cells/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Saccharomyces cerevisiae</td><td colspan="1" rowspan="1">ATCC 18824</td><td colspan="1" rowspan="1">1.9E+08 cells/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td></td><td colspan="1" rowspan="1">Viruses</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Cytomegalovirus</td><td colspan="1" rowspan="1">ZeptoMetrix 0810003CF</td><td colspan="1" rowspan="1">1.9E+05 TCID5o/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Epstein-Barr virus</td><td colspan="1" rowspan="1">ZeptoMetrix 0810008CF</td><td colspan="1" rowspan="1">5.9E+06 copies/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Human herpes simplex virus 1</td><td colspan="1" rowspan="1">ATCC VR-260</td><td colspan="1" rowspan="1">8.9E+06 TCID5o/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Measles virus</td><td colspan="1" rowspan="1">ZeptoMetrix 0810025CF</td><td colspan="1" rowspan="1">2.5E+05 TCID5o/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Middle East respiratory syndrome coronavirus(heat-inactivated)</td><td colspan="1" rowspan="1">ZeptoMetrix 0810575CFHI</td><td colspan="1" rowspan="1">1.2E+05 TCID5o/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Mumps virus</td><td colspan="1" rowspan="1">ZeptoMetrix 0810079CF</td><td colspan="1" rowspan="1">2.0E+06 TCID5o/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Severe acute respiratory syndrome coronavirus(purified genomic RNA)</td><td colspan="1" rowspan="1">BEI NR-52346</td><td colspan="1" rowspan="1">5.3E+05 genomes/mL</td><td colspan="1" rowspan="1">None</td></tr></table>

a Bordetella bronchiseptica strain NRRL B-59909 contains genomic insertions of the IS1001 gene and is reactive with the SPOTFIRE R Panel Bordetella parapertussis IS1001 assay . b Positive results were observed for Bordetella parapertussis in all replicates tested at concentrations $2 2 . 8 \mathsf { E } { + } 0 1$ cells/mL and one out of three replicates tested at $2 . 8 \mathsf { E } ^ { + 0 0 }$ cells/mL. c Parvimonas micra was formerly classified as Micromonas micros and Peptostreptococcus micros.

# Near-LoD/Reproducibility Evaluation

A near-LoD/reproducibility evaluation was performed to demonstrate that the SPOTFIRE R Panel could reproducibly provide accurate results for weak-positive and negative samples when used by minimally trained operators. Contrived samples were tested at three of the prospective clinical study sites and additionally on three unique SPOTFIRE Systems at BioFire Diagnostics (BioFire) by trained BioFire personnel. The contrived samples contained combinations of SPOTFIRE R Panel analytes prepared in artificial matrix at or near ( $1 \times$ to $_ { 3 \times }$ ) the LoD. For testing performed at clinical sites, samples were tested over five testing events (non-consecutive days) by two operators during the course of their normal workday routine. Each site was equipped with a single SPOTFIRE System. Testing at all three sites was performed with a single reagent lot. For each testing event, each operator ran two replicate pouches for a total of 20 replicates per site and 60 total replicates across all three sites. For testing performed at BioFire, samples were tested over five consecutive days, by two operators per system, using three different reagent lots. Each day of testing, the two operators each tested three replicates on each system for a total of 30 replicates per system and 90 total replicates across all systems. When combined, each analyte was tested in a total of 150 replicates by at least 12 different operators across six different SPOTFIRE Systems.

A summary of results (percent $( \% )$ agreement with the expected positive or negative result) for each analyte (by site and system) is provided in Table 24. The SPOTFIRE R Panel reported the expected positive results for panel analytes in $9 4 \% - 1 0 0 \%$ of samples and the expected negative results for all analytes in $100 \%$ of samples. Comparison of the positive percent agreement between user groups $( 9 8 . 8 \%$ for trained operators at BioFire versus $9 8 . 9 \%$ for minimally trained operators) demonstrates that the accuracy of the SPOTFIRE R Panel is not dependent upon the specific expertise of the user.

Table 24. Reproducibility of Results on the SPOTFIRE R Panel and BIOFIRE SPOTFIRE System   

<table><tr><td colspan="2" rowspan="3">Analytesolate(Source ID)</td><td colspan="1" rowspan="3">ConcentrationTestedtest level)</td><td colspan="1" rowspan="3">ExpectedResult</td><td colspan="8" rowspan="1">SpotFire System testing</td><td colspan="1" rowspan="3">All Sites/Systems95%ConfidenceInterval]</td></tr><tr><td colspan="4" rowspan="1">BioFire Dx</td><td colspan="4" rowspan="1">Clinical</td></tr><tr><td colspan="1" rowspan="1">System A</td><td colspan="1" rowspan="1">System B</td><td colspan="1" rowspan="1">System C</td><td colspan="1" rowspan="1">Total</td><td colspan="1" rowspan="1">Site 1</td><td colspan="1" rowspan="1">Site 2</td><td colspan="1" rowspan="1">Site 3</td><td colspan="1" rowspan="1">Total</td></tr><tr><td colspan="2" rowspan="2">AdenovirusSppecies B(ZeptoMetrix 0810062CF)</td><td colspan="1" rowspan="1">No Analyte</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">120/120(1100%)</td><td colspan="1" rowspan="1">120/120(100%)</td><td colspan="1" rowspan="1">120/120(100%)</td><td colspan="1" rowspan="1">360/360(100%)</td><td colspan="1" rowspan="1">80/80(100%)</td><td colspan="1" rowspan="1">80/80(100%)</td><td colspan="1" rowspan="1">80/80(100%)</td><td colspan="1" rowspan="1">240/240100%)</td><td colspan="1" rowspan="1">600/600100%[99.4-100%]</td></tr><tr><td colspan="1" rowspan="1">2.4E+00 TCID50/mL(3× LoD)</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">30/30(1100%)</td><td colspan="1" rowspan="1">30/30 (100%)</td><td colspan="1" rowspan="1">30/30(1100%)</td><td colspan="1" rowspan="1">90/90(1100%)</td><td colspan="1" rowspan="1">20/20(1100%)</td><td colspan="1" rowspan="1">20/20(100%)</td><td colspan="1" rowspan="1">20/20(1100%)</td><td colspan="1" rowspan="1">60/60 (1100%)</td><td colspan="1" rowspan="1">150/150100%[97.6-100%]</td></tr><tr><td colspan="2" rowspan="2">Bordetella parapertussis(ZeptoMetrix 0801462)</td><td colspan="1" rowspan="1">No Analyte</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">120/120(100%)</td><td colspan="1" rowspan="1">120/120(100%)</td><td colspan="1" rowspan="1">120/120(100%)</td><td colspan="1" rowspan="1">360/360(100%)</td><td colspan="1" rowspan="1">80/80(100%)</td><td colspan="1" rowspan="1">80/80(100%)</td><td colspan="1" rowspan="1">80/80(100%)</td><td colspan="1" rowspan="1">240/240(100%)</td><td colspan="1" rowspan="1">600/600100%[99.4-100%]</td></tr><tr><td colspan="1" rowspan="1">1.2E+02 CFU/mL× LoD)</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">30/30(100%)</td><td colspan="1" rowspan="1">30/30 (1100%)</td><td colspan="1" rowspan="1">30/30(1100%)</td><td colspan="1" rowspan="1">90/90(100%)</td><td colspan="1" rowspan="1">20/20(1100%)</td><td colspan="1" rowspan="1">20/20 (100%)</td><td colspan="1" rowspan="1">20/20 (100%)</td><td colspan="1" rowspan="1">60/60(1100%)</td><td colspan="1" rowspan="1">150/150100%[97.6-100%]</td></tr><tr><td colspan="2" rowspan="2">Bordetella pertussis(ZeptoMetrix 0801459)</td><td colspan="1" rowspan="1">No Analyte</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">120/120100%)</td><td colspan="1" rowspan="1">120/120(100%)</td><td colspan="1" rowspan="1">120/120(100%)</td><td colspan="1" rowspan="1">360/360(100%)</td><td colspan="1" rowspan="1">80/80100%)</td><td colspan="1" rowspan="1">80/80(100%)</td><td colspan="1" rowspan="1">80/80(1100%)</td><td colspan="1" rowspan="1">240/240(100%)</td><td colspan="1" rowspan="1">600/600100%[99.4-100%]</td></tr><tr><td colspan="1" rowspan="1">9.9E+02 CFU/mL(3× LoD)</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">29/30(6.7%)</td><td colspan="1" rowspan="1">29/306.7%)</td><td colspan="1" rowspan="1">30/30(1100%)</td><td colspan="1" rowspan="1">88/90(7.8%)</td><td colspan="1" rowspan="1">20/20(1100%)</td><td colspan="1" rowspan="1">20/20 (100%)</td><td colspan="1" rowspan="1">20/20(1100%)</td><td colspan="1" rowspan="1">60/60(100%)</td><td colspan="1" rowspan="1">148/15098.7%[95.3-99.8%]</td></tr><tr><td colspan="2" rowspan="2">Chlamydia pneumoniae(ATCC 53592)</td><td colspan="1" rowspan="1">No Analyte</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">120/120100%)</td><td colspan="1" rowspan="1">120/120100%)</td><td colspan="1" rowspan="1">120/120(100%)</td><td colspan="1" rowspan="1">360/360(100%)</td><td colspan="1" rowspan="1">80/80100%)</td><td colspan="1" rowspan="1">80/80100%)</td><td colspan="1" rowspan="1">80/80(100%)</td><td colspan="1" rowspan="1">240/240(100%)</td><td colspan="1" rowspan="1">600/600100%[99.4-100%]</td></tr><tr><td colspan="1" rowspan="1">2.0E+01 IFU/mL(1× LoD)</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">30/30(1100%)</td><td colspan="1" rowspan="1">30/30100%)</td><td colspan="1" rowspan="1">30/30(1100%)</td><td colspan="1" rowspan="1">90/90(1100%)</td><td colspan="1" rowspan="1">20/20(100%)</td><td colspan="1" rowspan="1">20/20(100%)</td><td colspan="1" rowspan="1">20/20100%)</td><td colspan="1" rowspan="1">60/60(100%)</td><td colspan="1" rowspan="1">150/150100%[97.6-100%]</td></tr><tr><td colspan="1" rowspan="4">(oooss) ssean</td><td colspan="2" rowspan="1">No Analyte</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">60/60(100%)</td><td colspan="1" rowspan="1">60/60(100%)</td><td colspan="1" rowspan="1">60/60(100%)</td><td colspan="1" rowspan="1">180/180(100%)</td><td colspan="1" rowspan="1">40/40100%)</td><td colspan="1" rowspan="1">40/40100%)</td><td colspan="1" rowspan="1">40/40(100%)</td><td colspan="1" rowspan="1">120/120(100%)</td><td colspan="1" rowspan="1">300/30000%[98.8-100%]</td></tr><tr><td colspan="1" rowspan="1">229E(ATCC VR-740)</td><td colspan="1" rowspan="1">2.0E+00 TCID5o/mL(3× LoD)</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">30/30(1100%)</td><td colspan="1" rowspan="1">30/30(1100%)</td><td colspan="1" rowspan="1">29/30(96.7%)</td><td colspan="1" rowspan="1">89/90(98.9%)</td><td colspan="1" rowspan="1">20/20(1100%)</td><td colspan="1" rowspan="1">20/20(100%)</td><td colspan="1" rowspan="1">20/20(100%)</td><td colspan="1" rowspan="1">60/60(100%)</td><td colspan="1" rowspan="1">149/15099.3%[96.3-100%]</td></tr><tr><td colspan="1" rowspan="1">OC43(ZeptoMetrix080024CF)</td><td colspan="1" rowspan="1">1.6E-02 TCID5o/mL(1× LoD)</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">30/30(1100%)</td><td colspan="1" rowspan="1">30/30100%)</td><td colspan="1" rowspan="1">30/30(1100%)</td><td colspan="1" rowspan="1">90/90(1100%)</td><td colspan="1" rowspan="1">20/20 (100%)</td><td colspan="1" rowspan="1">20/20(100%)</td><td colspan="1" rowspan="1">20/20(1100%)</td><td colspan="1" rowspan="1">60/60 (1100%)</td><td colspan="1" rowspan="1">150/150100%[97.6-100%]</td></tr><tr><td colspan="1" rowspan="1">NL63(Zep oMetrix0810228SC)</td><td colspan="1" rowspan="1">2.5E-03 TCID50/mL( LOD)</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">27/30(90.0%)</td><td colspan="1" rowspan="1">30/30(1100%)</td><td colspan="1" rowspan="1">30/30100%)</td><td colspan="1" rowspan="1">87/90(96.7%)</td><td colspan="1" rowspan="1">18/20(90.0%)</td><td colspan="1" rowspan="1">20/20 (1100%)</td><td colspan="1" rowspan="1">18/20(90.0%)</td><td colspan="1" rowspan="1">56/60(93.3%)</td><td colspan="1" rowspan="1">143/15095.3%[90.6-98.1%]</td></tr><tr><td colspan="2" rowspan="2">Coronavirus SARS-CoV-2Severe Acute RespiratorySyndrome Coronavirus 2(ATCC VR-1986HK)</td><td colspan="1" rowspan="1">No Analyte</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">120/120(100%)</td><td colspan="1" rowspan="1">120/120(100%)</td><td colspan="1" rowspan="1">120/120(100%)</td><td colspan="1" rowspan="1">360/360((100%)</td><td colspan="1" rowspan="1">80/80(100%)</td><td colspan="1" rowspan="1">80/80(100%)</td><td colspan="1" rowspan="1">80/80(100%)</td><td colspan="1" rowspan="1">240/240(100%)</td><td colspan="1" rowspan="1">600/600100%[99.4-100%]</td></tr><tr><td colspan="1" rowspan="1">2.5E+02 copies/mL(1× LoD)</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">30/30(00%)</td><td colspan="1" rowspan="1">30/30(1100%)</td><td colspan="1" rowspan="1">30/30100%)</td><td colspan="1" rowspan="1">90/90(1100%)</td><td colspan="1" rowspan="1">20/20(100%)</td><td colspan="1" rowspan="1">20/20(100%)</td><td colspan="1" rowspan="1">20/20(100%)</td><td colspan="1" rowspan="1">60/60(1100%)</td><td colspan="1" rowspan="1">150/150100%[97.6-100%]</td></tr><tr><td colspan="2" rowspan="3">Analytesolate(Source ID)</td><td colspan="1" rowspan="3">ConcentrationTestedtest level)</td><td colspan="1" rowspan="3">ExpectedResult</td><td colspan="8" rowspan="1">SpotFire System testing</td><td colspan="1" rowspan="3">All Sites/Systems95%ConfidenceIterval]</td></tr><tr><td colspan="4" rowspan="1">BioFire Dx</td><td colspan="4" rowspan="1">Clinical</td></tr><tr><td colspan="1" rowspan="1">System A</td><td colspan="1" rowspan="1">System B</td><td colspan="1" rowspan="1">System C</td><td colspan="1" rowspan="1">Total</td><td colspan="1" rowspan="1">Site 1</td><td colspan="1" rowspan="1">Site 2</td><td colspan="1" rowspan="1">Site 3</td><td colspan="1" rowspan="1">Total</td></tr><tr><td colspan="2" rowspan="2">Human metapneumovirus3 Type B1(ZeptoMetrix 0810156CF)</td><td colspan="1" rowspan="1">No Analyte</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">120/120(100%)</td><td colspan="1" rowspan="1">120/120(100%)</td><td colspan="1" rowspan="1">120/120(100%)</td><td colspan="1" rowspan="1">360/360(1100%)</td><td colspan="1" rowspan="1">80/80(100%)</td><td colspan="1" rowspan="1">80/80100%)</td><td colspan="1" rowspan="1">80/80(100%)</td><td colspan="1" rowspan="1">240/240(1100%)</td><td colspan="1" rowspan="1">600/600100%[99.4-100%]</td></tr><tr><td colspan="1" rowspan="1">7.5E-01 TCID50/mL LOD)</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">30/30(1100%)</td><td colspan="1" rowspan="1">30/30(1100%)</td><td colspan="1" rowspan="1">30/30(1100%)</td><td colspan="1" rowspan="1">90/90(1100%)</td><td colspan="1" rowspan="1">19/20(95.0%)</td><td colspan="1" rowspan="1">20/20(100%)</td><td colspan="1" rowspan="1">19/20955.0%)</td><td colspan="1" rowspan="1">58/60(6.7%)</td><td colspan="1" rowspan="1">148/15098.7%[95.3-99.8%]</td></tr><tr><td colspan="2" rowspan="2">Human rhinovirus/enterovirusEnterovirus D68US/MO/14-18947(ATCC VR-1823)</td><td colspan="1" rowspan="1">No Analyte</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">120/120(100%)</td><td colspan="1" rowspan="1">120/120(100%)</td><td colspan="1" rowspan="1">120/120(100%)</td><td colspan="1" rowspan="1">360/360(100%)</td><td colspan="1" rowspan="1">80/80(100%)</td><td colspan="1" rowspan="1">80/80100%)</td><td colspan="1" rowspan="1">80/80(1100%)</td><td colspan="1" rowspan="1">240/240(100%)</td><td colspan="1" rowspan="1">600/600100%[99.4-100%]</td></tr><tr><td colspan="1" rowspan="1">1.1E+01 TCID5o/mL1 LOD)</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">30/30(100%)</td><td colspan="1" rowspan="1">30/30(1100%)</td><td colspan="1" rowspan="1">30/30(1100%)</td><td colspan="1" rowspan="1">90/90(1100%)</td><td colspan="1" rowspan="1">20/20(100%)</td><td colspan="1" rowspan="1">20/20(1100%)</td><td colspan="1" rowspan="1">20/20(100%)</td><td colspan="1" rowspan="1">60/60(100%)</td><td colspan="1" rowspan="1">150/150100%[97.6-100%]</td></tr><tr><td colspan="1" rowspan="3">iin v ezunus</td><td colspan="2" rowspan="1">No Analyte</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">90/90(100%)</td><td colspan="1" rowspan="1">90/90(1100%)</td><td colspan="1" rowspan="1">90/90(1100%)</td><td colspan="1" rowspan="1">270/270(100%)</td><td colspan="1" rowspan="1">60/60(100%)</td><td colspan="1" rowspan="1">60/60(100%)</td><td colspan="1" rowspan="1">60/60(100%)</td><td colspan="1" rowspan="1">180/180(100%)</td><td colspan="1" rowspan="1">450/450100%[99.2-100%]</td></tr><tr><td colspan="1" rowspan="1">Influenza AH11pdm( ptoMetrix0810538CF)</td><td colspan="1" rowspan="1">2.5E+00 TCID5o/mL(3× LoD)</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">30/30(100%)</td><td colspan="1" rowspan="1">30/30(100%)</td><td colspan="1" rowspan="1">30/30(100%)</td><td colspan="1" rowspan="1">90/9000%)</td><td colspan="1" rowspan="1">20/20 (100%)</td><td colspan="1" rowspan="1">20/20(1100%)</td><td colspan="1" rowspan="1">20/20(1100%)</td><td colspan="1" rowspan="1">60/60(100%)</td><td colspan="1" rowspan="1">150/150100%[97.6-100%]</td></tr><tr><td colspan="1" rowspan="1">Influenza A H3N2(ZeptoMetrix080526CF)</td><td colspan="1" rowspan="1">2.6E+00 TCID5o/mL LOD)</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">30/30(1100%)</td><td colspan="1" rowspan="1">30/30(100%)</td><td colspan="1" rowspan="1">30/30(1100%)</td><td colspan="1" rowspan="1">90/90(1100%)</td><td colspan="1" rowspan="1">19/2095.0%)</td><td colspan="1" rowspan="1">19/20(95.0%)</td><td colspan="1" rowspan="1">20/20(100%)</td><td colspan="1" rowspan="1">58/60(96.7%)</td><td colspan="1" rowspan="1">148/15098.7%[95.3-99.8%]</td></tr><tr><td colspan="2" rowspan="2">Influenza A virus A/H1-2009Influenza A H1N1pdm(ZeptoMetrix 0810538CF)</td><td colspan="1" rowspan="1">No Analyte</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">120/120100%)</td><td colspan="1" rowspan="1">120/120100%)</td><td colspan="1" rowspan="1">120/120100%)</td><td colspan="1" rowspan="1">360/360(100%)</td><td colspan="1" rowspan="1">80/80100%)</td><td colspan="1" rowspan="1">80/80100%)</td><td colspan="1" rowspan="1">80/80100%)</td><td colspan="1" rowspan="1">240/240(100%)</td><td colspan="1" rowspan="1">600/600100%[99.4-100%]</td></tr><tr><td colspan="1" rowspan="1">2.5E+00 TCID50/mL(3× LoD)</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">30/30(100%)</td><td colspan="1" rowspan="1">30/30(100%)</td><td colspan="1" rowspan="1">30/30(1100%)</td><td colspan="1" rowspan="1">90/90(1100%)</td><td colspan="1" rowspan="1">20/20(100%)</td><td colspan="1" rowspan="1">20/20(100%)</td><td colspan="1" rowspan="1">20/20100%)</td><td colspan="1" rowspan="1">60/60(100%)</td><td colspan="1" rowspan="1">150/150100%[97.6-100%]</td></tr><tr><td colspan="2" rowspan="2">Influenza A virus A/H3Influenza A H3N2(ZeptoMetrix 0810526CF)</td><td colspan="1" rowspan="1">No Analyte</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">120/120(100%)</td><td colspan="1" rowspan="1">120/120(100%)</td><td colspan="1" rowspan="1">120/120(100%)</td><td colspan="1" rowspan="1">360/360(1100%)</td><td colspan="1" rowspan="1">80/80(100%)</td><td colspan="1" rowspan="1">80/80(100%)</td><td colspan="1" rowspan="1">80/80(1100%)</td><td colspan="1" rowspan="1">240/240(100%)</td><td colspan="1" rowspan="1">600/600100%[99.4-100%]</td></tr><tr><td colspan="1" rowspan="1">2.6E+00 TCID5o/mL LOD)</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">30/30(100%)</td><td colspan="1" rowspan="1">30/30(100%)</td><td colspan="1" rowspan="1">30/30 (100%)</td><td colspan="1" rowspan="1">90/90(1100%)</td><td colspan="1" rowspan="1">19/205.0%)</td><td colspan="1" rowspan="1">19/20(95.0%)</td><td colspan="1" rowspan="1">20/20 (100%)</td><td colspan="1" rowspan="1">58/60(96.7%)</td><td colspan="1" rowspan="1">148/1509.7%[95.3-99.8%]</td></tr><tr><td colspan="2" rowspan="2">Influenza B virus(ZeptoMetrix 0810037CF)</td><td colspan="1" rowspan="1">No Analyte</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">120/120(100%)</td><td colspan="1" rowspan="1">120/120(100%)</td><td colspan="1" rowspan="1">120/120(100%)</td><td colspan="1" rowspan="1">360/360(100%)</td><td colspan="1" rowspan="1">80/80(100%)</td><td colspan="1" rowspan="1">80/80100%)</td><td colspan="1" rowspan="1">80/80(100%)</td><td colspan="1" rowspan="1">240/240(100%)</td><td colspan="1" rowspan="1">600/600100%[99.4-100%]</td></tr><tr><td colspan="1" rowspan="1">9.9E-02 TCID5o/mL(3× LoD)</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">29/30(6.7%)</td><td colspan="1" rowspan="1">30/30(1100%)</td><td colspan="1" rowspan="1">30/30(1100%)</td><td colspan="1" rowspan="1">89/90(8.9%)</td><td colspan="1" rowspan="1">20/20(100%)</td><td colspan="1" rowspan="1">20/20(100%)</td><td colspan="1" rowspan="1">20/20(100%)</td><td colspan="1" rowspan="1">60/60(100%)</td><td colspan="1" rowspan="1">149/15099.3%[96.3-100%]</td></tr><tr><td colspan="2" rowspan="1">Mycoplasma pneumoniae(ZeptoMetrix 0801579)</td><td colspan="1" rowspan="1">No Analyte</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">120/120(100%)</td><td colspan="1" rowspan="1">120/120(100%)</td><td colspan="1" rowspan="1">120/120(100%)</td><td colspan="1" rowspan="1">360/360100%)</td><td colspan="1" rowspan="1">80/80(100%)</td><td colspan="1" rowspan="1">80/80(100%)</td><td colspan="1" rowspan="1">80/80(100%)</td><td colspan="1" rowspan="1">240/240100%)</td><td colspan="1" rowspan="1">600/600100%[99.4-100%]</td></tr><tr><td colspan="2" rowspan="3">AnalyteIsolate(Source ID)</td><td colspan="1" rowspan="3">ConcentrationTestedttest level)</td><td colspan="1" rowspan="3">ExpectedResult</td><td colspan="8" rowspan="1">SpotFire System testing</td><td colspan="1" rowspan="3">All Sites/ystems95%Confidencenterval]</td></tr><tr><td colspan="4" rowspan="1">BioFire Dx</td><td colspan="4" rowspan="1">Clinical</td></tr><tr><td colspan="1" rowspan="1">System A</td><td colspan="1" rowspan="1">System B</td><td colspan="1" rowspan="1">System C</td><td colspan="1" rowspan="1">Total</td><td colspan="1" rowspan="1">Site 1</td><td colspan="1" rowspan="1">Site 2</td><td colspan="1" rowspan="1">Site 3</td><td colspan="1" rowspan="1">Total</td></tr><tr><td colspan="2" rowspan="1"></td><td colspan="1" rowspan="1">1.0E+01 CCU/mL LoD)</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">29/30(96.7%)</td><td colspan="1" rowspan="1">30/30(1100%)</td><td colspan="1" rowspan="1">30/30 (1100%)</td><td colspan="1" rowspan="1">89/90g9.3%)</td><td colspan="1" rowspan="1">20/20(1100%)</td><td colspan="1" rowspan="1">20/20(100%)</td><td colspan="1" rowspan="1">20/20(1100%)</td><td colspan="1" rowspan="1">60/60(100%)</td><td colspan="1" rowspan="1">149/15099.3%[96.3-100%]</td></tr><tr><td colspan="1" rowspan="5">Prr eanus</td><td colspan="2" rowspan="1">No Analyte</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">30/30(1100%)</td><td colspan="1" rowspan="1">30/30(100%)</td><td colspan="1" rowspan="1">30/30(1100%)</td><td colspan="1" rowspan="1">90/90(1100%)</td><td colspan="1" rowspan="1">20/20(1100%)</td><td colspan="1" rowspan="1">20/20(1100%)</td><td colspan="1" rowspan="1">20/20(100%)</td><td colspan="1" rowspan="1">60/60(100%)</td><td colspan="1" rowspan="1">150/150100%[97.6 100%]</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza virus 1(ptoMetrix80014C)</td><td colspan="1" rowspan="1">4.6E+00 TCID50/mL(1 LoD)</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">30/30(100%)</td><td colspan="1" rowspan="1">30/30(100%)</td><td colspan="1" rowspan="1">30/30(1100%)</td><td colspan="1" rowspan="1">90/90(1100%)</td><td colspan="1" rowspan="1">20/20 (1100%)</td><td colspan="1" rowspan="1">20/20(100%)</td><td colspan="1" rowspan="1">20/20(100%)</td><td colspan="1" rowspan="1">60/60(100%)</td><td colspan="1" rowspan="1">150/150100%[97.6-100%]</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza virus 2(ZeptoMetrix080015CF)</td><td colspan="1" rowspan="1">4.2E+01 TCID50/mL LOD)</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">30/30(1100%)</td><td colspan="1" rowspan="1">29/306.7%)</td><td colspan="1" rowspan="1">28/30(93.3%)</td><td colspan="1" rowspan="1">87/90(96.7%)</td><td colspan="1" rowspan="1">20/20(1100%)</td><td colspan="1" rowspan="1">20/20 (1100%)</td><td colspan="1" rowspan="1">20/20 (1100%)</td><td colspan="1" rowspan="1">60/60(100%)</td><td colspan="1" rowspan="1">147/15098.0%[94.3-99.6%]</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza virus 3(ZeptoMetrix080016CF)</td><td colspan="1" rowspan="1">8.8E+00 TCID5/mL(1× LoD)</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">28/30(933%)</td><td colspan="1" rowspan="1">30/30(100%)</td><td colspan="1" rowspan="1">30/30(100%)</td><td colspan="1" rowspan="1">88/90(97.8%)</td><td colspan="1" rowspan="1">19/20(95.0%)</td><td colspan="1" rowspan="1">18/20(90.0%)</td><td colspan="1" rowspan="1">20/20(100%)</td><td colspan="1" rowspan="1">57/60(95.0%)</td><td colspan="1" rowspan="1">145/15096.%[92.4-98.9%]</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza virus 4(ZeptoMetrix080060CF)</td><td colspan="1" rowspan="1">2.0E+02 TCID5o/mL(1× LoD)</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">29/30(96.7%)</td><td colspan="1" rowspan="1">30/30(100%)</td><td colspan="1" rowspan="1">30/30(100%)</td><td colspan="1" rowspan="1">89/90(98.9%)</td><td colspan="1" rowspan="1">20/20(100%)</td><td colspan="1" rowspan="1">20/20(100%)</td><td colspan="1" rowspan="1">20/20 (1100%)</td><td colspan="1" rowspan="1">60/60(100%)</td><td colspan="1" rowspan="1">149/15099.3%[96.3-100%]</td></tr><tr><td colspan="2" rowspan="2">Respiratory syncytial virus(ZeptoMetrix 0810040ACF)</td><td colspan="1" rowspan="1">No Analyte</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">120/120(100%)</td><td colspan="1" rowspan="1">120/120(100%)</td><td colspan="1" rowspan="1">120/120(100%)</td><td colspan="1" rowspan="1">360/360(100%)</td><td colspan="1" rowspan="1">80/80(100%)</td><td colspan="1" rowspan="1">80/80(100%)</td><td colspan="1" rowspan="1">80/80(100%)</td><td colspan="1" rowspan="1">240/240(100%)</td><td colspan="1" rowspan="1">600/600100%[99.4-100%]</td></tr><tr><td colspan="1" rowspan="1">6.2E-02 TCID50/mL(1× LoD)</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">30/30(100%)</td><td colspan="1" rowspan="1">30/30(1100%)</td><td colspan="1" rowspan="1">30/30 (1100%)</td><td colspan="1" rowspan="1">90/90(1100%)</td><td colspan="1" rowspan="1">20/20(1100%)</td><td colspan="1" rowspan="1">19/20(95.0%)</td><td colspan="1" rowspan="1">20/20(100%)</td><td colspan="1" rowspan="1">59/60(98.3%)</td><td colspan="1" rowspan="1">149/15099.%[96.3-100%]</td></tr><tr><td colspan="4" rowspan="1">Total positive agreement (%)by system/user group</td><td colspan="1" rowspan="1">621/63098.6%</td><td colspan="1" rowspan="1">628/63099.7%</td><td colspan="1" rowspan="1">627/63099.5%</td><td colspan="1" rowspan="1">1876/189099.3%</td><td colspan="1" rowspan="1">414/42098.6%</td><td colspan="1" rowspan="1">415/42098.8%</td><td colspan="1" rowspan="1">417/42099.3%</td><td colspan="1" rowspan="1">1246/12608.9%</td><td colspan="1" rowspan="2">3122/315099.1%[98.7-99.4%]</td></tr><tr><td colspan="12" rowspan="1">Overall positive agreement (%)[95% Confidence Interval]</td></tr></table>

# Interference

Potentially interfering substances that could be present in NPS specimens or may be introduced into specimens during collection or subsequent handling and testing were evaluated for their effect on SPOTFIRE R Panel performance. The substances tested included endogenous substances that may be found at normal or elevated levels in clinical specimens (e.g. blood, mucus/mucin, human genomic DNA), various commensal or infectious microorganisms, medications, washes or topical applications for the nasal passage, various swabs and transport media for specimen collection, and substances used to clean, decontaminate, or disinfect work areas.

Each substance was added to contrived samples containing representative organisms at concentrations near $( 3 \times )$ the LoD. The concentration of substance added to the samples was equal to or greater than the highest level expected to be in NPS specimens.

Valid and accurate results were obtained for each sample containing substances and microorganisms at the concentrations listed in Table 25.

Table 25. Substances Tested on the SPOTFIRE R Panel - No Interference Observed   

<table><tr><td colspan="1" rowspan="1">Substance Tested</td><td colspan="1" rowspan="1">Concentration Tested</td></tr><tr><td colspan="1" rowspan="1">Endogenous Substances</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Human Whole Blood (with Na Citrate)</td><td colspan="1" rowspan="1">10% (v/V)</td></tr><tr><td colspan="1" rowspan="1">Human Sputum/Mucus</td><td colspan="1" rowspan="1">1% (V/V)</td></tr><tr><td colspan="1" rowspan="1">Human Genomic DNA</td><td colspan="1" rowspan="1">20 ng/μL</td></tr><tr><td colspan="1" rowspan="1">Exogenous Substancesa</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Promethazine hydrochloride</td><td colspan="1" rowspan="1">1.04 µmol/L (3.34E-04 mg/mL)</td></tr><tr><td colspan="1" rowspan="1">Acetaminophen (paracetamol)</td><td colspan="1" rowspan="1">1.0E+03 μmol/L (1.5E-01 mg/mL)</td></tr><tr><td colspan="1" rowspan="1">Acetylsalicylic acid (Aspirin)</td><td colspan="1" rowspan="1">167 μmol/L (3.0E-02 mg/mL)</td></tr><tr><td colspan="1" rowspan="1">Ibuprofen</td><td colspan="1" rowspan="1">1060 µmol/L (2.2E-01 mg/mL)</td></tr><tr><td colspan="1" rowspan="1">Albuterol sulfate (common ingredient in rescue inhalers)</td><td colspan="1" rowspan="1">0.188 μmol/L (5.4E-05 mg/mL)</td></tr><tr><td colspan="1" rowspan="1">Triple antibiotic ointment (neomycin/polymyxin B/bacitracin)</td><td colspan="1" rowspan="1">2% w/v</td></tr><tr><td colspan="1" rowspan="1">Mucinex® Severe Nasal Congestion Relief Clear &amp; Cool Nasal Spray (Oxymetazoline hydrochloride 0.05%)</td><td colspan="1" rowspan="1">1% V/V</td></tr><tr><td colspan="1" rowspan="1">Saline nasal spray (sodium chloride 0.65%, disodium phosphate, phenylcarbinol, monosodium phosphate andbenzalkonium chloride solution)</td><td colspan="1" rowspan="1">1% v/v</td></tr><tr><td colspan="1" rowspan="1">Vicks® VapoRub® Cough Suppressant Topical Analgesic (Camphor 4.8%, eucalyptus oil 1.2%, and menthol 2.6%)</td><td colspan="1" rowspan="1">1% w/v</td></tr><tr><td colspan="1" rowspan="1">Vaseline® Petroleum Jelly (100% white petrolatum)</td><td colspan="1" rowspan="1">1% w/v</td></tr><tr><td colspan="1" rowspan="1">Orajel™ (benzalkonium chloride 0.13%, benzocaine 20%, menthol 0.5%, zinc chloride 0.15%)</td><td colspan="1" rowspan="1">2% w/v</td></tr><tr><td colspan="1" rowspan="1">Chloraseptic® Sore Throat Spray (Phenol 1.4%)</td><td colspan="1" rowspan="1">1% V/v</td></tr><tr><td colspan="1" rowspan="1">Vicks® Formula 44™ DM (dextromethorphan hydrobromide 0.67 mg/mL, guaifenesin 13 mg/mL) (cough syrup)</td><td colspan="1" rowspan="1">1% V/V</td></tr><tr><td colspan="1" rowspan="1">Phenylephrine hydrochloride (common ingredient in nasal decongestants)</td><td colspan="1" rowspan="1">1% w/v</td></tr><tr><td colspan="1" rowspan="1">Nasal spray (fluticasone propionate 50 mcg)</td><td colspan="1" rowspan="1">1% V/V</td></tr><tr><td colspan="1" rowspan="1">Sucrets® Sore Throat (dyclonine hydrochloride 2.0 mg/lozenge)</td><td colspan="1" rowspan="1">1% w/v</td></tr><tr><td colspan="1" rowspan="1">Benadry® Allergy Liqui-gels® (diphenhydramine hydrochloride 25 mg/capsule)</td><td colspan="1" rowspan="1">1% V/V</td></tr><tr><td colspan="1" rowspan="1">Zicam® Cold Remedy (Galphimia Glauca 4x, Luffa Operculata 4x, Sabadilla 4x)</td><td colspan="1" rowspan="1">1% V/v</td></tr><tr><td colspan="1" rowspan="1">Cold-eeze® (zinc gluconate 2.3%)</td><td colspan="1" rowspan="1">1% w/v</td></tr><tr><td colspan="1" rowspan="1">HALLS lozenge (menthol 5 mg/lozenge)</td><td colspan="1" rowspan="1">1% w/v</td></tr><tr><td colspan="1" rowspan="1">Listerine® Cool Mint® (menthol 0.042%, thymol 0.064%, methyl salicylate 0.06%, eucalyptol 0.092%)</td><td colspan="1" rowspan="1">6.5% v/v</td></tr><tr><td colspan="1" rowspan="1">Copenhagen® Snuff (Tobacco)</td><td colspan="1" rowspan="1">1% W/V</td></tr><tr><td colspan="1" rowspan="1">JUICE HEAD (30% propylene glycol, 70% vegetable glycerin) (e-juice)</td><td colspan="1" rowspan="1">1% V/V</td></tr><tr><td colspan="1" rowspan="1">Technique-Specific Substances</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Rayon swab (COPAN Diagnostics Inc.)</td><td colspan="1" rowspan="1">1 swab</td></tr><tr><td colspan="1" rowspan="1">Nylon flocked swab (COPAN Diagnostics Inc.)</td><td colspan="1" rowspan="1">1 swab</td></tr><tr><td colspan="1" rowspan="1">Polyester swab (COPAN Diagnostics Inc.)</td><td colspan="1" rowspan="1">1 swab</td></tr><tr><td colspan="1" rowspan="1">Calcium Alginate swab (Puritan®)</td><td colspan="1" rowspan="1">1 swab</td></tr><tr><td colspan="1" rowspan="1">Cary-Blair</td><td colspan="1" rowspan="1">90% v/v</td></tr><tr><td colspan="1" rowspan="1">Dulbecco's Modified Eagles-Medium (DMEM)</td><td colspan="1" rowspan="1">90% v/v</td></tr><tr><td colspan="1" rowspan="1">Hanks Balanced Salt Solution</td><td colspan="1" rowspan="1">100% v/v</td></tr><tr><td colspan="1" rowspan="1">0.9% Normal Saline</td><td colspan="1" rowspan="1">100% v/v</td></tr><tr><td colspan="1" rowspan="1">BD™ Universal Viral Transport</td><td colspan="1" rowspan="1">100% v/v</td></tr><tr><td colspan="1" rowspan="1">Remel MicroTestTM M4RT Tube w/o beads</td><td colspan="1" rowspan="1">100% v/v</td></tr><tr><td colspan="1" rowspan="1">Remel MicroTest™M M4 Tube w/o beads</td><td colspan="1" rowspan="1">90% v/v</td></tr><tr><td colspan="1" rowspan="1">Viral Preservative Media (VPM)</td><td colspan="1" rowspan="1">90% v/v</td></tr><tr><td colspan="1" rowspan="1">Phosphate Buffered Saline (PBS)</td><td colspan="1" rowspan="1">90% v/v</td></tr><tr><td colspan="1" rowspan="1">PrimeStore® MTM Molecular Transport Media</td><td colspan="1" rowspan="1">90% v/v</td></tr><tr><td colspan="1" rowspan="1">Stuart Transport Medium</td><td colspan="1" rowspan="1">90% v/v</td></tr><tr><td colspan="1" rowspan="1">eNAT™ Molecular Transport Medium</td><td colspan="1" rowspan="1">90% v/v</td></tr><tr><td colspan="1" rowspan="1">Bleach</td><td colspan="1" rowspan="1">1% V/v, 2% V/vb</td></tr><tr><td colspan="1" rowspan="1">Ethanol</td><td colspan="1" rowspan="1">7% v/v</td></tr><tr><td colspan="1" rowspan="1">Disinfecting wipes (ammonium chloride)</td><td colspan="1" rowspan="1">0.25 - 0.5 inch square/sample</td></tr><tr><td colspan="1" rowspan="1">DNAZap™M</td><td colspan="1" rowspan="1">1% v/v</td></tr><tr><td colspan="1" rowspan="1">RNaseZap™M</td><td colspan="1" rowspan="1">1% V/V</td></tr><tr><td colspan="2" rowspan="1">Competing Microorganisms</td></tr><tr><td colspan="2" rowspan="1">On-Panel</td></tr><tr><td colspan="1" rowspan="1">Coronavirus 229E</td><td colspan="1" rowspan="1">1.5E+07 copies/mL</td></tr><tr><td colspan="1" rowspan="1">Enterovirus D68</td><td colspan="1" rowspan="1">7.8 E+07 copies/mL</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza virus 3</td><td colspan="1" rowspan="1">8.0E+06 copies/mL</td></tr><tr><td colspan="1" rowspan="1">Respiratory syncytial virus A</td><td colspan="1" rowspan="1">1.5E+07 copies/mL</td></tr><tr><td colspan="1" rowspan="1">Adenovirus A31</td><td colspan="1" rowspan="1">1.6E+07 copies/mL</td></tr><tr><td colspan="1" rowspan="1">Bordetella pertussis</td><td colspan="1" rowspan="1">1.6E+09 copies/mL</td></tr><tr><td colspan="1" rowspan="1">Off-Panel</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Cytomegalovirus (CMV)</td><td colspan="1" rowspan="1">4.2E+04 TCIDs/mL</td></tr><tr><td colspan="1" rowspan="1">Herpes simplex virus 1</td><td colspan="1" rowspan="1">9.0E+06 TCID5/mL</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">7.4E+08 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Streptococcus pneumoniae</td><td colspan="1" rowspan="1">2.5E+07 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Streptococcus pyogenes</td><td colspan="1" rowspan="1">2.2E+08 copies/mL</td></tr><tr><td colspan="1" rowspan="1">Haemophilus influenzae</td><td colspan="1" rowspan="1">8.3E+08 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Candida albicans</td><td colspan="1" rowspan="1">2.8E+07 CFU/mL</td></tr></table>

a Nasal influenza vaccines (e.g. FluMist®) were not evaluated but are predicted to be reactive with the Influenza A (subtype) and Influenza B assays. b Incubation of sample with $1 \%$ (v/v) bleach for 15 minutes, 4 hours, or \~18.5-hour (overnight) or $2 \%$ (v/v) bleach for 15 minutes did not result in interference.

# NOTE: Avoid contact between samples and bleach prior to testing (bleach can damage nucleic acids and prevent amplification and detection by the panel).

NOTE: Compatibility of the SPOTFIRE R Panel with NPS in PrimeStore® MTM has not been evaluated in the intended use setting. PrimeStore® MTM and BIOFIRE Sample Buffer contain guanidine salts that will react with bleach to form a toxic gas. Use caution if using bleach for disinfection purposes when collecting or testing NPS specimens.

# External Control Material

# Quality Control Testing

External control material is available directly from bioMérieux and Maine Molecular Quality Controls, Inc. (MMQCI) (online at www.mmqci.com). The MMQCI controls listed in Table 26 are positive and negative control solutions for all viruses and bacteria detected by the SPOTFIRE R Panel, along with an insert explaining the expected results.

Table 26. Control Materials Compatible with all Analytes on the SPOTFIRE R Panel   

<table><tr><td rowspan=1 colspan=1>Vendor</td><td rowspan=1 colspan=1>Product Name</td><td rowspan=1 colspan=1>Part Number</td></tr><tr><td rowspan=2 colspan=1>Maine Molecular QualityControls, Inc. (MMQCI)</td><td rowspan=1 colspan=1>SPOTFIRERSPNegative Control</td><td rowspan=1 colspan=1>M42738</td></tr><tr><td rowspan=1 colspan=1>SPOTFIRE® RSPPositive Control</td><td rowspan=1 colspan=1>M42638</td></tr></table>

# NOTE: MMQCI control test solutions can be stored at room temperature $( 1 8 ^ { \circ } C \cdot 2 5 ^ { \circ } C )$ .

It is recommended that when using the SPOTFIRE R Panel that Positive and Negative QC testing should be performed when:

Training a new operator • Receiving a new shipment or lot of SPOTFIRE R Panel test kits Receiving a new SpotFire Control Station

A Negative QC test should be run at least monthly for each testing location to monitor for environmental contamination. Additional Negative QC tests may be run if contamination is suspected.