# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATIONDECISION SUMMARYASSAY AND INSTRUMENT

I Background Information:

# B Applicant

DiaSorin Molecular LLC

# C Proprietary and Established Names

Simplexa COVID-19 & Flu A/B Direct

D Regulatory Information

<table><tr><td rowspan=1 colspan=1>ProductCode(s)</td><td rowspan=1 colspan=1>Classification</td><td rowspan=1 colspan=1>RegulationSection</td><td rowspan=1 colspan=1>Panel</td></tr><tr><td rowspan=1 colspan=1>QOF</td><td rowspan=1 colspan=1>Class II</td><td rowspan=1 colspan=1>21 CFR 866.3981 - DeviceTo Detect And IdentifyNucleic Acid Targets InRespiratory SpecimensFrom Microbial AgentsThat Cause The SARS-Cov-2 RespiratoryInfection And OtherMicrobial Agents When InA Multi-Target Test</td><td rowspan=1 colspan=1>MI - Microbiology</td></tr><tr><td rowspan=1 colspan=1>00I</td><td rowspan=1 colspan=1>Class II</td><td rowspan=1 colspan=1>21 CFR 862.2570 -Instrumentation forclinical multiplex testsystems</td><td rowspan=1 colspan=1>CH - Clinical Chemistry</td></tr></table>

# II Submission/Device Overview:

# A Purpose for Submission:

This submission is a Traditional 510(k) to obtain clearance for a new device, the Simplexa COVID-19 & Flu A/B Direct.

# B Measurand:

Viral RNA from Influenza A (Flu A), Influenza B (Flu B), and SARS-CoV-2

# C Type of Test:

Real-time Reverse Transcription Polymerase Chain Reaction (RT-PCR) system, qualitative

# III Intended Use/Indications for Use:

A Intended Use(s): See Indications for Use below.

# B Indication(s) for Use:

The DiaSorin Molecular Simplexa COVID-19 & Flu A/B Direct is a real-time RT-PCR assay intended for use on the LIAISON MDX instrument for the in vitro qualitative detection and differentiation of nucleic acid from severe acute respiratory syndrome coronavirus 2 (SARSCoV-2), influenza A virus and influenza B virus in nasopharyngeal swabs (NPS) from individuals with signs and symptoms of respiratory tract infection.

The Simplexa COVID-19 & Flu A/B Direct assay is intended for use as an aid in the differential diagnosis of SARS-CoV-2, influenza A and influenza B infection. Negative results do not preclude SARS-CoV-2, influenza A or influenza B infection and should not be used as the sole basis for patient management decisions. Positive results do not rule out coinfections with other organisms. Results should be combined with clinical observations, patient history, and epidemiological information.

The Simplexa COVID-19 & Flu A/B Direct assay is intended for use by qualified and trained clinical laboratory personnel specifically instructed and trained in the techniques of real-time PCR and in vitro diagnostic procedures.

C Special Conditions for Use Statement(s): Rx - For Prescription Use Only

D Special Instrument Requirements:

LIAISON MDX instrument with LIAISON MDX Studio Software

# IV Device/System Characteristics:

# A Device Description:

The system consists of the Simplexa COVID-19 & Flu A/B assay reagents (supplied in quantity for 24 reactions), the LIAISON MDX (with LIAISON MDX Studio Software), the Direct Amplification Disc and associated accessories. The LIAISON MDX instrument is a real-time Polymerase Chain Reaction (PCR) thermocycler used for the identification of nucleic acid from biological specimens. The Direct Amplification Disc (DAD) is compartmentalized into eight separate wedges and up to eight separate specimens or controls may be processed on each disc.

Each wedge contains sample and reagent input wells, microfluidic channels and laser activated valves to control the fluid flow, and a reaction chamber. The user adds $5 0 ~ \mu \mathrm { L }$ of Reaction Mix to the reagent input well and $5 0 ~ \mu \mathrm { L }$ of unextracted specimen to the sample input well. The reverse transcription, amplification and detection are performed automatically by the instrument.

# B Principle of Operation:

The Simplexa COVID-19 & Flu A/B Direct assay system is a real-time RT-PCR system that enables the direct amplification, detection and differentiation of SARS-CoV-2 RNA, human influenza A (Flu A) virus RNA and human influenza B (Flu B) virus RNA from unprocessed nasopharyngeal swabs (NPS) that have not undergone nucleic acid extraction. The system consists of the Simplexa COVID-19 & Flu A/B Direct assay, the LIAISON MDX (with LIAISON MDX Studio Software), the Direct Amplification Disc and associated accessories.

In the Simplexa COVID-19 & Flu A/B Direct assay, fluorescent probes are used together with corresponding forward and reverse primers to amplify SARS-CoV-2, Flu A, Flu B and internal control RNA targets. For COVID-19 detection, the assay targets two different regions specific to the SARS-CoV-2 genome; the S gene which encodes the spike glycoprotein and the ORF1ab region which encodes wellconserved non-structural proteins and therefore is less susceptible to recombination. For Flu detection the assay targets conserved regions of influenza A viruses (matrix gene) and influenza B viruses (matrix gene). The assay provides three results; COVID19 (ORF1ab and/or S gene detection), influenza A viruses (matrix gene detection) and influenza B viruses (matrix gene detection). An RNA internal control is used to detect RT-PCR failure and/or inhibition.

# C Instrument Description Information:

1. Instrument Name: Liaison MDX System

2. Specimen Identification: Barcode scanner or manual entry

3. Specimen Sampling and Handling:

Simplexa COVID-19 & Flu A/B Direct is a no extraction RT-PCR assay. The operator adds $5 0 \mu \mathrm { L }$ of unextracted sample to the sample input well and adds $5 0 ~ \mu \mathrm { L }$ of the reaction mix to the reagent input well in the direct amplification disc (DAD).

4. Calibration:

Each reagent kit comes with a barcode card, which contains assay specific parameters and lot information. The barcode card is scanned prior to each run.

5. Quality Control:

The Simplexa COVID-19 & Flu A/B Positive Control Pack may be used as an external positive control. Universal transport media (UTM) may be used as a negative external control (No Template Control).

# V Substantial Equivalence Information:

A Predicate Device Name(s): BioFire Respiratory Panel 2.1 (RP2.1)   
B Predicate 510(k) Number(s): DEN200031   
C Comparison with Predicate(s):

Table 1. Comparison of Simplexa COVID-19 & Flu A/B Direct with Predicate   

<table><tr><td rowspan=1 colspan=1>Device &amp; Predicate Device(s):</td><td rowspan=1 colspan=1>K220963</td><td rowspan=1 colspan=1>DEN200031</td></tr><tr><td rowspan=1 colspan=1>Device Trade Name</td><td rowspan=1 colspan=1>Simplexa COVID-19 &amp; FluA/B Direct</td><td rowspan=1 colspan=1>BioFire Respiratory Panel2.1</td></tr><tr><td rowspan=1 colspan=1>General Device CharacteristicSimilarities</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Product Code</td><td rowspan=1 colspan=1>QOF</td><td rowspan=1 colspan=1>QOF</td></tr><tr><td rowspan=1 colspan=1>Regulation Number</td><td rowspan=1 colspan=1>21 CFR 866.3981</td><td rowspan=1 colspan=1>21 CFR 866.3981</td></tr><tr><td rowspan=1 colspan=1>Intended Use/Indications For Use</td><td rowspan=1 colspan=1>The DiaSorin MolecularSimplexa COVID-19 &amp; FluA/B Direct is a real-time RT-PCR assay intended for useon the LIAISON MDXinstrument for the in vitroqualitative detection anddifferentiation of nucleicacid from severe acuterespiratory syndromecoronavirus 2 (SARS-CoV-2), influenza A virus andinfluenza B virus innasopharyngeal swabs (NPS)from individuals with signsand symptoms of respiratorytract infection.The Simplexa COVID-19 &amp;Flu A/B Direct assay isintended for use as an aid inthe differential diagnosis ofSARS-CoV-2, influenza Aand influenza B infection.Negative results do notpreclude SARS-CoV-2,influenza A or influenza Binfection and should not beused as the sole basis for</td><td rowspan=1 colspan=1>The BioFire RespiratoryPanel 2.1 (RP2.1) is a PCR-based multiplexed nucleicacid test intended for usewith the BioFire FilmArray2.0 or BioFire FilmArrayTorch systems for thesimultaneous qualitativedetection and identificationof multiple respiratory viraland bacterial nucleic acids innasopharyngeal swabs (NPS)obtained from individualssuspected of respiratory tractinfections, includingCOVID-19. The followingorganism types and subtypesare identified using theBioFire RP2.1: Adenovirus,Coronavirus 229E,Coronavirus HKU1,Coronavirus NL63,Coronavirus OC43, SevereAcute Respiratory SyndromeCoronavirus (SARS CoV-2),Human Metapneumovirus,HumanRhinovirus/Enterovirus,Influenza A, including</td></tr></table>

<table><tr><td colspan="3">patient management subtypes H</td></tr><tr><td></td><td>decisions. Positive results do not rule out coinfection with other organisms. Results should be combined with clinical observations, patient history, and epidemiological information. The Simplexa COVID-19 &amp; Flu A/B Positive control Pack intended to be used as a control with the Simplexa COVID-19 &amp; Flu A/B Direct kit for use on the LIAISON MDX instrument. This control is not intended for use with other assays or systems.</td><td>H3, Influen Parainfluen Parainfluen Parainfluen Parainfluen Respiratory Bordetella (IS1001), B pertussis (p pneumonia Mycoplasm Nucleic aci respiratory bacterial or identified b generally d specimens phase of in detection a of specific nucleic acio individuals and/or sym respiratory indicative the identifie and aids in respiratory in conjunct clinical and information this test sho as the sole diagnosis, t patient man decisions. Negative re setting of a illness may infection w are not dete or lower res infection th detected by specimen. I not rule out other organ agent(s) de BioFire RP the definite Additional (e.g. bacter culture,</td></tr></table>

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>necessary when evaluating apatient with possiblerespiratory tract infection.</td></tr><tr><td rowspan=1 colspan=1>Sample to answer system</td><td rowspan=1 colspan=1>Automated</td><td rowspan=1 colspan=1>Automated</td></tr><tr><td rowspan=1 colspan=1>Specimen Types</td><td rowspan=1 colspan=1>Nasopharyngeal Swab</td><td rowspan=1 colspan=1>Nasopharyngeal Swab</td></tr><tr><td rowspan=1 colspan=1>General Device CharacteristicDifferences</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Device Technology</td><td rowspan=1 colspan=1>RT-PCR using DirectAmplification Disc (DAD)moves reagents throughreaction chambers usinglaser-operated valve openingand centrifugal forces</td><td rowspan=1 colspan=1>RT-PCR using disposablepouch module with blisterpacks to separate chemicalsteps and move reagents viamechanical force</td></tr><tr><td rowspan=1 colspan=1>Organisms Detected</td><td rowspan=1 colspan=1>SARS-CoV-2Influenza AInfluenza B</td><td rowspan=1 colspan=1>SARS-CoV-2Influenza AInfluenza BAdenovirusCoronavirus 229ECoronavirus HKU1Coronavirus NL63Coronavirus OC43Human MetapneumovirusHumanRhinovirus/EnterovirusParainfluenza Virus 1Parainfluenza Virus 2Parainfluenza Virus 3Parainfluenza Virus 4Respiratory Syncytial VirusBordetella parapertussisBordetella pertussisChlamydia pneumoniaeMycoplasma pneumoniae</td></tr><tr><td rowspan=1 colspan=1>Measurand</td><td rowspan=1 colspan=1>RNA from SARS-CoV-2,Influenza A, Influenza B</td><td rowspan=1 colspan=1>Nucleic acid from SARS-CoV-2, Influenza A,Influenza B, Adenovirus,Coronavirus 229E,Coronavirus HKU1,Coronavirus NL63,Coronavirus OC43, HumanMetapneumovirus, HumanRhinovirus/Enterovirus,Parainfluenza Virus 1,Parainfluenza Virus 2,Parainfluenza Virus 3,Parainfluenza Virus 4,Respiratory Syncytial Virus,Bordetella parapertussis,Bordetella pertussis,Chlamydia pneumoniae,Mycoplasma pneumoniae</td></tr><tr><td rowspan=1 colspan=1>Instrument</td><td rowspan=1 colspan=1>LIAISON MDX</td><td rowspan=1 colspan=1>BioFire FilmArray 2.0 orBioFire FilmArray Torch</td></tr></table>

# VI Standards/Guidance Documents Referenced:

Class II Special Controls as per 21 CFR 866.3981

# VII Performance Characteristics (if/when applicable):

# A Analytical Performance:

1. Analytical Reactivity:

The analytical reactivity of the Simplexa COVID-19 & A/B Direct with various strains of influenza A, influenza B and SARS-CoV-2, was evaluated by testing dilutions of quantified virus stocks prepared by spiking into pooled negative nasopharyngeal swab matrix (swabs collected in UTM) at concentrations near the LoD. A total of 63 Flu A strains, 21 Flu B strains and five SARS-CoV-2 strains were tested in three replicates. Samples that were not detected in all three replicates at the initially contrived concentrations were retested at a higher concentration. Tables 2, 3, and 4 below show the strains that were positive in all three replicates at the concentrations shown.

Table 2. Inclusivity for Simplexa COVID-19 & Flu A/B Direct – Influenza A   

<table><tr><td colspan="1" rowspan="1">Subtype</td><td colspan="1" rowspan="1">Influenza A Strain</td><td colspan="1" rowspan="1">TestedConcentration</td><td colspan="1" rowspan="1"># Detected / #Tested</td></tr><tr><td colspan="1" rowspan="1">H7N9</td><td colspan="1" rowspan="1">A/Anhui/1/2013</td><td colspan="1" rowspan="1">1:100,000dilution</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">H1N9</td><td colspan="1" rowspan="1">A/American green-winged teal/Mississippi/300/2010</td><td colspan="1" rowspan="1">100 CEID50/mL</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">H13N6</td><td colspan="1" rowspan="1">A/black-legged kittiwake/Quebec/02838-1/2009</td><td colspan="1" rowspan="1">100 CEID50/mL</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">H1N1pdm09</td><td colspan="1" rowspan="1">A/Brisbane/02/2018</td><td colspan="1" rowspan="1">100 EID50/mL</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">H3N2</td><td colspan="1" rowspan="1">A/Brisbane/10/07</td><td colspan="1" rowspan="1">100 TCID50/mL</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">H1N1</td><td colspan="1" rowspan="1">A/Brisbane/59/07</td><td colspan="1" rowspan="1">100 TCID50/mL</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">H3N2</td><td colspan="1" rowspan="1">A/California/02/2014</td><td colspan="1" rowspan="1">100 TCID50/mL</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">H1N1pdm09</td><td colspan="1" rowspan="1">A/California/4/2009</td><td colspan="1" rowspan="1">100 TCID50/mL</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">H10N7</td><td colspan="1" rowspan="1">A/chicken/Germany/N/49</td><td colspan="1" rowspan="1">100 CEID50/mL</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="2">H1N1pdm09</td><td colspan="1" rowspan="2">A/Christ Church/16/2010</td><td colspan="1" rowspan="1">100 EID50/mL</td><td colspan="1" rowspan="1">2/3</td></tr><tr><td colspan="1" rowspan="1">1000 EID50/mL</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">H3N8</td><td colspan="1" rowspan="1">A/duck/Chabarovsk/1610/1972</td><td colspan="1" rowspan="1">100 CEID50/mL</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="2">H4N6</td><td colspan="1" rowspan="2">A/duck/Czechoslovakia/1956</td><td colspan="1" rowspan="1">100 CEID50/mL</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">5000 CEID50/mL</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">H12N6</td><td colspan="1" rowspan="1">A/duck/Wisconsin/480/1979</td><td colspan="1" rowspan="1">100 CEID50/mL</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">H1N1pdm09</td><td colspan="1" rowspan="1">A/Guangdong-Maonan/1536/2019</td><td colspan="1" rowspan="1">100 EID50/mL</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">H1N1</td><td colspan="1" rowspan="1">A/Hawaii/15/2001</td><td colspan="1" rowspan="1">100 CEID50/mL</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">H3N2</td><td colspan="1" rowspan="1">A/Hong Kong/267/2019</td><td colspan="1" rowspan="1">100 EID50/mL</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">H9N2</td><td colspan="1" rowspan="1">A/Hong Kong/33982/2009(H9N2)-PR8-IDCDC RG26</td><td colspan="1" rowspan="1">100 CEID50/mL</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">H3N2</td><td colspan="1" rowspan="1">A/Kansas/14/2017</td><td colspan="1" rowspan="1">100 EID50/mL</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">H10N7</td><td colspan="1" rowspan="1">A/mallard/Illinois/100S4334/2010</td><td colspan="1" rowspan="1">100 CEID50/mL</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">H12N5</td><td colspan="1" rowspan="1">A/mallard/Wisconsin/4218/2009</td><td colspan="1" rowspan="1">100 CEID50/mL</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">H10N1</td><td colspan="1" rowspan="1">A/mallard/Wisconsin/4230/2009</td><td colspan="1" rowspan="1">100 CEID50/mL</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">H1N1pdm09</td><td colspan="1" rowspan="1">A/Massachusetts/15/2013</td><td colspan="1" rowspan="1">100 CEID50/mL</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">H1N1pdm09</td><td colspan="1" rowspan="1">A/Mexico/4108/2009</td><td colspan="1" rowspan="1">100 CEID50/mL</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">H1N2</td><td colspan="1" rowspan="1">A/Minnesota/19/2011</td><td colspan="1" rowspan="1">100 CEID50/mL</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">H1N1</td><td colspan="1" rowspan="1">A/New Caledonia/20/99</td><td colspan="1" rowspan="1">100 TCID50/mL</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">H1N1pdm09</td><td colspan="1" rowspan="1">A/New York/18/2009</td><td colspan="1" rowspan="1">100 CEID50/mL</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">H3N2</td><td colspan="1" rowspan="1">A/New York/55/2004</td><td colspan="1" rowspan="1">100 CEID50/mL</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">H3N2</td><td colspan="1" rowspan="1">A/Perth/16/2009</td><td colspan="1" rowspan="1">100 EID50/mL</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">H10N8</td><td colspan="1" rowspan="1">A/quail/Italy/1117/1965</td><td colspan="1" rowspan="1">100 CEID50/mL</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">H1N8</td><td colspan="1" rowspan="1">A/red knot/Delaware Bay/240/1994</td><td colspan="1" rowspan="1">100 CEID50/mL</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="2">H4N6</td><td colspan="1" rowspan="2">A/red knot/Delaware/541/1988</td><td colspan="1" rowspan="1">100 CEID50/mL</td><td colspan="1" rowspan="1">1/3</td></tr><tr><td colspan="1" rowspan="1">1000 CEID50/mL</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">H3N6</td><td colspan="1" rowspan="1">A/redhead/Alberta/192/2002</td><td colspan="1" rowspan="1">100 CEID50/mL</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">H3N2</td><td colspan="1" rowspan="1">A/Rhode Island/01/2010</td><td colspan="1" rowspan="1">100 CEID50/mL</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">H3N2</td><td colspan="1" rowspan="1">A/Santiago/7981/2006</td><td colspan="1" rowspan="1">100 CEID50/mL</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="2">H1N3</td><td colspan="1" rowspan="2">A/shorebird/Delaware Bay/211/1994</td><td colspan="1" rowspan="1">100 CEID50/mL</td><td colspan="1" rowspan="1">2/3</td></tr><tr><td colspan="1" rowspan="1">1000 CEID50/mL</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="2">H16N3</td><td colspan="1" rowspan="2">A/shorebird/Delaware/172/2006</td><td colspan="1" rowspan="1">100 CEID50/mL</td><td colspan="1" rowspan="1">2/3</td></tr><tr><td colspan="1" rowspan="1">1000 CEID50/mL</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">H1N1</td><td colspan="1" rowspan="1">A/Solomon Island/3/2006</td><td colspan="1" rowspan="1">100 TCID50/mL</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">H1N1</td><td colspan="1" rowspan="1">A/Swine/1976/31</td><td colspan="1" rowspan="1">100 TCID50/mL</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">H1N1</td><td colspan="1" rowspan="1">A/Swine/Iowa/15/30</td><td colspan="1" rowspan="1">100 TCID50/mL</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">H1N2</td><td colspan="1" rowspan="1">A/swine/Ohio/09SW1477/2009</td><td colspan="1" rowspan="1">100 TCID50/mL</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">H3N2</td><td colspan="1" rowspan="1">A/swine/Ohio/09SW83E/2009</td><td colspan="1" rowspan="1">100 CEID50/mL</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">H1N1</td><td colspan="1" rowspan="1">A/Taiwan/42/06</td><td colspan="1" rowspan="1">100 TCID50/mL</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="2">H6N2</td><td colspan="1" rowspan="2">A/turkey/Massachusetts/3740/1965</td><td colspan="1" rowspan="1">100 CEID50/mL</td><td colspan="1" rowspan="1">1/3</td></tr><tr><td colspan="1" rowspan="1">2000 CEID50/mL</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">H1N1</td><td colspan="1" rowspan="1">A/WS/33</td><td colspan="1" rowspan="1">100 TCID50/mL</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">H1N1pdm09</td><td colspan="1" rowspan="1">A/California/7/2009</td><td colspan="1" rowspan="1">100 TCID50/mL</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">H3N2</td><td colspan="1" rowspan="1">A/Hong Kong/4801/2014</td><td colspan="1" rowspan="1">100 TCID50/mL</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">H3N2v</td><td colspan="1" rowspan="1">A/Indiana/08/2011</td><td colspan="1" rowspan="1">100 TCID50/mL</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">H3N2v</td><td colspan="1" rowspan="1">A/Minnesota/11/2010</td><td colspan="1" rowspan="1">100 CEID50/mL</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">H1N1pdm09</td><td colspan="1" rowspan="1">A/NY/02/09</td><td colspan="1" rowspan="1">100 TCID50/mL</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">H3N2</td><td colspan="1" rowspan="1">A/Ohio/02/2012</td><td colspan="1" rowspan="1">100 CEID50/mL</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">H3N2</td><td colspan="1" rowspan="1">A/Port Chalmers/1/1973</td><td colspan="1" rowspan="1">100 TCID50/mL</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">H3N2</td><td colspan="1" rowspan="1">A/Singapore/INFIMH-16-0019/2016</td><td colspan="1" rowspan="1">100 TCID50/mL</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">H3N2</td><td colspan="1" rowspan="1">A/Texas/50/2012</td><td colspan="1" rowspan="1">100 TCID50/mL</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">H3N2</td><td colspan="1" rowspan="1">A/Wisconsin/67/05</td><td colspan="1" rowspan="1">100 TCID50/mL</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">H1N1</td><td colspan="1" rowspan="1">A/PR/8/34</td><td colspan="1" rowspan="1">100 TCID50/mL</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">H7N9</td><td colspan="1" rowspan="1">A/Anhui/1/2013</td><td colspan="1" rowspan="1">1:100,000dilution</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">H5N1</td><td colspan="1" rowspan="1">A/chicken/Vietnam/NCVD-016/2008(H5N1)-PR8-IDCDC-RG12</td><td colspan="1" rowspan="1">1:100,000dilution</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">H5N1</td><td colspan="1" rowspan="1">A/Egypt/N03072/2010(H5N1)-PR8-IDCDC-RG29</td><td colspan="1" rowspan="1">1:100,000dilution</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">H5N1</td><td colspan="1" rowspan="1">A/Hubei/1/2010(H5N1)-PR8-IDCDC-RG30</td><td colspan="1" rowspan="1">1:100,000dilution</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">H5N1</td><td colspan="1" rowspan="1">A/India/NIV/2006(H5N1)-PR8-IBCDC-RG7</td><td colspan="1" rowspan="1">1:100,000dilution</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">H7N7</td><td colspan="1" rowspan="1">A/mallard/Netherlands/12/2000(H7N7)/PR8-IBCDC-1</td><td colspan="1" rowspan="1">1:100,000dilution</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">H5N2</td><td colspan="1" rowspan="1">A/pheasant/New Jersey/1355/1998(H5N2)-PR8-IBCDC-4</td><td colspan="1" rowspan="1">1:100,000dilution</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td>H7N2</td><td>A/turkey/Virginia/4529/2002(H7N2)xPr8-IBCDC-5</td><td>1:100,000 dilution</td><td>3/3</td></tr></table>

Table 3. Inclusivity for Simplexa COVID-19 & Flu A/B Direct – Influenza B   

<table><tr><td rowspan=1 colspan=1>Lineage</td><td rowspan=1 colspan=1>Influenza B Strain</td><td rowspan=1 colspan=1>TestedConcentration</td><td rowspan=1 colspan=1># Detected / #Tested</td></tr><tr><td rowspan=1 colspan=1>Victoria</td><td rowspan=1 colspan=1>B/Brisbane/33/2008</td><td rowspan=1 colspan=1>100 CEID50/mL</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Victoria</td><td rowspan=1 colspan=1>B/Brisbane/60/2008</td><td rowspan=1 colspan=1>100 TCID50/mL</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Victoria</td><td rowspan=1 colspan=1>B/Colorado/06/2017</td><td rowspan=1 colspan=1>100 TCID50/mL</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Victoria</td><td rowspan=1 colspan=1>B/Florida/02/2006</td><td rowspan=1 colspan=1>100 TCID50/mL</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Victoria</td><td rowspan=1 colspan=1>B/Michigan/09/2011</td><td rowspan=1 colspan=1>100 EID50/mL</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Victoria</td><td rowspan=1 colspan=1>B/Nevada/03/2011</td><td rowspan=1 colspan=1>100 CEID50/mL</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Victoria</td><td rowspan=1 colspan=1>B/Texas/02/2013</td><td rowspan=1 colspan=1>100 TCID50/mL</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Victoria</td><td rowspan=1 colspan=1>B/Victoria/304/2006</td><td rowspan=1 colspan=1>100 CEID50/mL</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Victoria</td><td rowspan=1 colspan=1>B/Washington/02/2019</td><td rowspan=1 colspan=1>100 EID50/mL</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Yamagata</td><td rowspan=1 colspan=1>B/ChristChurch/33/2004</td><td rowspan=1 colspan=1>100 TCID50/mL</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Yamagata</td><td rowspan=1 colspan=1>B/Florida/04/2006</td><td rowspan=1 colspan=1>100 TCID50/mL</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Yamagata</td><td rowspan=1 colspan=1>B/Florida/07/04</td><td rowspan=1 colspan=1>100 TCID50/mL</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=2 colspan=1>Yamagata</td><td rowspan=2 colspan=1>B/Guangdong-Liwan/1133/2014</td><td rowspan=1 colspan=1>100 CEID50/mL</td><td rowspan=1 colspan=1>1/3</td></tr><tr><td rowspan=1 colspan=1>1000 CEID50/mL</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Yamagata</td><td rowspan=1 colspan=1>B/Maryland/1/59</td><td rowspan=1 colspan=1>100 TCID50/mL</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Yamagata</td><td rowspan=1 colspan=1>B/Massachusetts/02/2012</td><td rowspan=1 colspan=1>100 TCID50/mL</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Yamagata</td><td rowspan=1 colspan=1>B/New Hampshire/01/2016</td><td rowspan=1 colspan=1>100 EID50/mL</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Yamagata</td><td rowspan=1 colspan=1>B/Panama/45/90</td><td rowspan=1 colspan=1>100 TCID50/mL</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Yamagata</td><td rowspan=1 colspan=1>B/Texas/81/2016</td><td rowspan=1 colspan=1>100 EID50/mL</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Yamagata</td><td rowspan=1 colspan=1>B/Utah/09/2014</td><td rowspan=1 colspan=1>100 CEID50/mL</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Yamagata</td><td rowspan=1 colspan=1>B/Wisconsin/01/2010</td><td rowspan=1 colspan=1>100 CEID50/mL</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Unknown</td><td rowspan=1 colspan=1>B/Great Lakes/1739/54</td><td rowspan=1 colspan=1>100 TCID50/mL</td><td rowspan=1 colspan=1>3/3</td></tr></table>

Table 4. Inclusivity for Simplexa COVID-19 & Flu A/B Direct – SARS-CoV-2   

<table><tr><td rowspan=1 colspan=1>SARS-CoV-2 Strain</td><td rowspan=1 colspan=1>TestedConcentration</td><td rowspan=1 colspan=1>Flu ADetection #Detected / #Tested</td><td rowspan=1 colspan=1>Flu BDetection #Detected / #Tested</td><td rowspan=1 colspan=1>SARS-CoV-2Detection #Detected / #Tested</td></tr><tr><td rowspan=1 colspan=1>Hong Kong/VM200001061/2020</td><td rowspan=1 colspan=1>1000 copies/mL</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>England/20480464/2020</td><td rowspan=1 colspan=1>1000 copies/mL</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>South Africa/KRISP-EC-K005325/2020</td><td rowspan=1 colspan=1>1000 copies/mL</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Japan/TY7-503/2021</td><td rowspan=1 colspan=1>1000 copies/mL</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>hCoV19/USA/PHC658/2021</td><td rowspan=1 colspan=1>1500 copies/mL</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>3/3</td></tr></table>

Additional testing was performed using the CDC panels and results are shown below in Tables 5-8.

Table 5. 2018-2019 CDC Influenza Panel Strains Tested with Simplexa COVID-19 & Flu A/B Direct   

<table><tr><td rowspan=1 colspan=1>Virus</td><td rowspan=1 colspan=1>Subtype</td><td rowspan=1 colspan=1>Organism</td></tr><tr><td rowspan=4 colspan=1>Flu A</td><td rowspan=2 colspan=1>A(H3N2)</td><td rowspan=1 colspan=1>A/Perth/16/2009</td></tr><tr><td rowspan=1 colspan=1>A/Singapore/INFIMH-16-0019/2016*</td></tr><tr><td rowspan=2 colspan=1>A(H1N1) pdm09</td><td rowspan=1 colspan=1>A/California/07/2009</td></tr><tr><td rowspan=1 colspan=1>A/Michigan/45/2015*</td></tr><tr><td rowspan=4 colspan=1>Flu B</td><td rowspan=2 colspan=1>B(Victoria lineage)</td><td rowspan=1 colspan=1>B/Brisbane/60/2008</td></tr><tr><td rowspan=1 colspan=1>B/Colorado/06/2017*</td></tr><tr><td rowspan=2 colspan=1>B(Yamagata lineage)</td><td rowspan=1 colspan=1>B/Wisconsin/01/2010</td></tr><tr><td rowspan=1 colspan=1>B/Phuket/3073/2013*</td></tr></table>

\*WHO recommended vaccine strains

Table 6. 2019-2020 CDC Influenza Panel Strains Tested with Simplexa COVID-19 & Flu A/B Direct   

<table><tr><td rowspan=1 colspan=1>Virus</td><td rowspan=1 colspan=1>Subtype</td><td rowspan=1 colspan=1>Organism</td></tr><tr><td rowspan=4 colspan=1>Flu A</td><td rowspan=2 colspan=1>A(H3N2)</td><td rowspan=1 colspan=1>A/Perth/16/2009</td></tr><tr><td rowspan=1 colspan=1>A/Kansas/14/2017*</td></tr><tr><td rowspan=2 colspan=1>A(H1N1) pdm09</td><td rowspan=1 colspan=1>A/Christ Church/16/2010</td></tr><tr><td rowspan=1 colspan=1>A/Brisbane/02/2018*</td></tr><tr><td rowspan=4 colspan=1>Flu B</td><td rowspan=2 colspan=1>B(Victoria lineage)</td><td rowspan=1 colspan=1>B/Michigan/09/2011</td></tr><tr><td rowspan=1 colspan=1>B/Colorado/06/2017*</td></tr><tr><td rowspan=2 colspan=1>B(Yamagata lineage)</td><td rowspan=1 colspan=1>B/New Hampshire/01/2016</td></tr><tr><td rowspan=1 colspan=1>B/Phuket/3073/2013*</td></tr></table>

\*WHO recommended vaccine strains

Table 7. 2020-2021 CDC Influenza Panel Strains Tested with Simplexa COVID-19 & Flu A/B Direct   

<table><tr><td rowspan=1 colspan=1>Virus</td><td rowspan=1 colspan=1>Subtype</td><td rowspan=1 colspan=1>Organism</td></tr><tr><td rowspan=4 colspan=1>Flu A</td><td rowspan=2 colspan=1>A(H3N2)</td><td rowspan=1 colspan=1>A/Perth/16/2009</td></tr><tr><td rowspan=1 colspan=1>A/Hong Kong/2671/2019*</td></tr><tr><td rowspan=2 colspan=1>A(H1N1) pdm09</td><td rowspan=1 colspan=1>A/Christ Church/16/2010</td></tr><tr><td rowspan=1 colspan=1>A/Guangdong-Maonan/1536/2019*</td></tr><tr><td rowspan=4 colspan=1>Flu B</td><td rowspan=2 colspan=1>B(Victoria lineage)</td><td rowspan=1 colspan=1>B/Michigan/09/2011</td></tr><tr><td rowspan=1 colspan=1>B/Washington/02/2019*</td></tr><tr><td rowspan=2 colspan=1>B(Yamagata lineage)</td><td rowspan=1 colspan=1>B/Texas/81/2016</td></tr><tr><td rowspan=1 colspan=1>B/Phuket/3073/2013*</td></tr></table>

\*WHO recommended vaccine strains

An in silico inclusivity analysis of the assay oligo sequences for SARS-CoV-2 was performed against SARS-CoV-2 sequences available in the GISAID EpiCoV database submitted from May 01, 2022 to July 31, 2022, including sequences of the Omicron BA.2.12.1, BA.2.75, BA.4 and BA.5 subvariants. The analysis included 211, 224 sequences in the amplicon regions of the ORF1ab and S gene oligo sets. Only target sequences with full coverage of all three oligo-binding regions (forward primer, reverse primer, and probe) are included in the analyses for both oligo sets. In this sequence set, there are 208, 582 $( \sim 9 8 . 7 \% )$ sequences with no mismatches in the oligo binding regions for both genes, 2602 $( \sim 1 . 2 \% )$ sequences with no mismatches for one gene oligo set (either ORF1ab or S gene), and 40 $( \sim 0 . 0 2 \% )$ sequences with mismatches in at least one oligo binding region for both gene oligo sets. Based on in silico analysis of the percent homology between assay oligos and target sequences, potential impact of location of the mismatches on extension and/or binding, and the mismatch melt temperature (Tm) values of the oligo sequence to its binding region on each analyzed SARS-CoV-2 sequence, it is predicted that the assay will detect $100 \%$ of the 211,224 SARS-CoV-2 sequences, including sequences of all defined variants of concern or variants of interest, available in the GISAID EpiCoV database from May 01, 2022 to July 31, 2022.

An additional in silico inclusivity analysis of the oligonucleotide (oligo) sequences for the SARS-CoV-2 ORF1ab and S gene sets were performed against all SARS-CoV-2 sequences (Human host, Complete, High Coverage) submitted to the GISAID EpiCoV database from November 1, 2022 to February 8, 2023. Based on in silico inclusivity analysis of the SARSCoV-2 oligo sequences of both gene oligo sets, it is predicted that $100 \%$ of analyzed sequences $\mathbf { \Phi } ( \mathbf { n } = 1 2 , 3 7 8$ ) can be detected by the assay.

For Influenza A, in silico inclusivity analysis was performed using human host sequences available from the GISAID EpiFlu database and collected between November 1, 2022 and February 8, 2023. Based on in silico inclusivity analysis of the Influenza A oligos, it is predicted that the assay can detect ${ \sim } 9 9 \%$ $( \mathbf { n } = 9 1 5 0 )$ ) of all human host Influenza A sequences collected between November 1, 2022 and February 8, 2023.

# 2. Precision/Reproducibility:

The device was evaluated in a reproducibility study testing a panel of eight samples. The panel members were prepared by diluting viral stocks of influenza A, influenza B and SARSCoV-2 strains into pooled negative clinical matrix (NP swabs collected in UTM). The panel included a Low Positive (2x LoD) and a Moderate Positive (4x LoD) level for each virus; a Positive Control and a negative sample consisting of UTM (no-template control) were also included.

Table 8. Reproducibility Sample Panel for Simplexa COVID-19 & Flu A/B Direct   

<table><tr><td rowspan=1 colspan=1>Panel Member or Strain</td><td rowspan=1 colspan=1>Panel RelativeTesting Level</td><td rowspan=1 colspan=1>Target LoDLevel</td><td rowspan=1 colspan=1>Concentration(copies/mL)</td></tr><tr><td rowspan=1 colspan=1>2019-nCoV/USA-WA1/2020</td><td rowspan=1 colspan=1>Low Positive</td><td rowspan=1 colspan=1>2x</td><td rowspan=1 colspan=1>1000</td></tr><tr><td rowspan=1 colspan=1>2019-nCoV/USA-WA1/2020</td><td rowspan=1 colspan=1>Moderate Positive</td><td rowspan=1 colspan=1>4x</td><td rowspan=1 colspan=1>2000</td></tr><tr><td rowspan=1 colspan=1>Influenza A/Hong Kong/8/68</td><td rowspan=1 colspan=1>Low Positive</td><td rowspan=1 colspan=1>2x</td><td rowspan=1 colspan=1>1000</td></tr><tr><td rowspan=1 colspan=1>Influenza A/Hong Kong/8/68</td><td rowspan=1 colspan=1>Moderate Positive</td><td rowspan=1 colspan=1>4x</td><td rowspan=1 colspan=1>2000</td></tr><tr><td rowspan=1 colspan=1>Influenza B/Malaysia/2506/04</td><td rowspan=1 colspan=1>Low Positive</td><td rowspan=1 colspan=1>2x</td><td rowspan=1 colspan=1>500</td></tr><tr><td rowspan=1 colspan=1>Influenza B/Malaysia/2506/04</td><td rowspan=1 colspan=1>Moderate Positive</td><td rowspan=1 colspan=1>4x</td><td rowspan=1 colspan=1>1000</td></tr><tr><td rowspan=1 colspan=1>UTM as NTC</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td></tr><tr><td rowspan=1 colspan=1>PC</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td></tr></table>

UTM $=$ Universal Transport Media, ${ \mathrm { N T C } } = { \mathrm { N o } }$ Template Control, $\mathrm { P C } =$ Positive Control/ $\mathrm { { N / A = } }$ Not applicable

The study was conducted at two external and one internal testing sites with each site having two designated LIAISON MDX instruments and two dedicated operators. Each of the two operators used a unique lot of the reaction mix (for a total of two lots). Each panel member was tested in three replicates, in two runs per day, over five days, at three locations. A total of 90 measurements (3 replicates x 2 runs/day x 5 days x 3 sites) were generated for each panel member. The study design allowed for evaluation of multiple components of variance. The data was assessed for (a) qualitative results, i.e., percent agreement with expected results, and (b) quantitative analysis of variance components, i.e., average Ct values and calculated SD and $\% C V$ .

Table 9. Reproducibility Qualitative Results   

<table><tr><td rowspan=2 colspan=1>Panel Member</td><td rowspan=1 colspan=1>Site 1</td><td rowspan=1 colspan=1>Site 2</td><td rowspan=1 colspan=1>Site 3</td><td rowspan=1 colspan=1>All Sites</td><td rowspan=2 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=1>Agreementwith ExpectedResults</td><td rowspan=1 colspan=1>Agreementwith ExpectedResults</td><td rowspan=1 colspan=1>Agreementwith ExpectedResults</td><td rowspan=1 colspan=1>Agreementwith ExpectedResults</td></tr><tr><td rowspan=1 colspan=1>Flu A HongKong - LP</td><td rowspan=1 colspan=1>30/30 (100.0%)</td><td rowspan=1 colspan=1>30/30 (100.0%)</td><td rowspan=1 colspan=1>30/30 (100.0%)</td><td rowspan=1 colspan=1>30/30 (100.0%)</td><td rowspan=1 colspan=1>95.9%-100.0%</td></tr><tr><td rowspan=1 colspan=1>Flu A HongKong - MP</td><td rowspan=1 colspan=1>30/30 (100.0%)</td><td rowspan=1 colspan=1>30/30 (100.0%)</td><td rowspan=1 colspan=1>30/30 (100.0%)</td><td rowspan=1 colspan=1>30/30 (100.0%)</td><td rowspan=1 colspan=1>95.9%-100.0%</td></tr><tr><td rowspan=1 colspan=1>Flu B Malaysia-LP</td><td rowspan=1 colspan=1>30/30 (100.0%)</td><td rowspan=1 colspan=1>30/30 (100.0%)</td><td rowspan=1 colspan=1>30/30 (100.0%)</td><td rowspan=1 colspan=1>30/30 (100.0%)</td><td rowspan=1 colspan=1>95.9%-100.0%</td></tr><tr><td rowspan=1 colspan=1>Flu B Malaysia- MP</td><td rowspan=1 colspan=1>30/30 (100.0%)</td><td rowspan=1 colspan=1>30/30 (100.0%)</td><td rowspan=1 colspan=1>30/30 (100.0%)</td><td rowspan=1 colspan=1>30/30 (100.0%)</td><td rowspan=1 colspan=1>95.9%-100.0%</td></tr><tr><td rowspan=1 colspan=1>2019-nCoV LP</td><td rowspan=1 colspan=1>30/30 (100.0%)</td><td rowspan=1 colspan=1>30/30 (100.0%)</td><td rowspan=1 colspan=1>30/30 (100.0%)</td><td rowspan=1 colspan=1>30/30 (100.0%)</td><td rowspan=1 colspan=1>95.9%-100.0%</td></tr><tr><td rowspan=1 colspan=1>2019-nCoV MP</td><td rowspan=1 colspan=1>30/30 (100.0%)</td><td rowspan=1 colspan=1>30/30 (100.0%)</td><td rowspan=1 colspan=1>30/30 (100.0%)</td><td rowspan=1 colspan=1>30/30 (100.0%)</td><td rowspan=1 colspan=1>95.9%-100.0%</td></tr><tr><td rowspan=1 colspan=1>PC Flu A</td><td rowspan=1 colspan=1>30/30 (100.0%)</td><td rowspan=1 colspan=1>30/30 (100.0%)</td><td rowspan=1 colspan=1>30/30 (100.0%)</td><td rowspan=1 colspan=1>30/30 (100.0%)</td><td rowspan=1 colspan=1>95.9%-100.0%</td></tr><tr><td rowspan=1 colspan=1>PC Flu B</td><td rowspan=1 colspan=1>30/30 (100.0%)</td><td rowspan=1 colspan=1>30/30 (100.0%)</td><td rowspan=1 colspan=1>30/30 (100.0%)</td><td rowspan=1 colspan=1>30/30 (100.0%)</td><td rowspan=1 colspan=1>95.9%-100.0%</td></tr><tr><td rowspan=1 colspan=1>PC nCoV</td><td rowspan=1 colspan=1>30/30 (100.0%)</td><td rowspan=1 colspan=1>30/30 (100.0%)</td><td rowspan=1 colspan=1>30/30 (100.0%)</td><td rowspan=1 colspan=1>30/30 (100.0%)</td><td rowspan=1 colspan=1>95.9%-100.0%</td></tr><tr><td rowspan=1 colspan=1>NTC Flu A</td><td rowspan=1 colspan=1>30/30 (100.0%)</td><td rowspan=1 colspan=1>30/30 (100.0%)</td><td rowspan=1 colspan=1>30/30 (100.0%)</td><td rowspan=1 colspan=1>30/30 (100.0%)</td><td rowspan=1 colspan=1>95.9%-100.0%</td></tr><tr><td rowspan=1 colspan=1>NTC Flu B</td><td rowspan=1 colspan=1>30/30 (100.0%)</td><td rowspan=1 colspan=1>30/30 (100.0%)</td><td rowspan=1 colspan=1>30/30 (100.0%)</td><td rowspan=1 colspan=1>30/30 (100.0%)</td><td rowspan=1 colspan=1>95.9%-100.0%</td></tr><tr><td rowspan=1 colspan=1>NTC nCoV</td><td rowspan=1 colspan=1>30/30 (100.0%)</td><td rowspan=1 colspan=1>30/30 (100.0%)</td><td rowspan=1 colspan=1>30/30 (100.0%)</td><td rowspan=1 colspan=1>30/30 (100.0%)</td><td rowspan=1 colspan=1>95.9%-100.0%</td></tr></table>

$\mathrm { L P = L o w }$ Positve, $\mathbf { M P } =$ Moderate Positive, nCoV = SARS-CoV-2, UTM $=$ Universal Transport Media, NTC $\mathbf { \tau } = \mathbf { N } \mathbf { o }$ Template Control, $\mathrm { P C } = ]$ Positive Control/ $\mathrm { N } / \mathrm { A } = \mathrm { N o t }$ applicable

Table 10. Reproducibility Variance Results by Site   

<table><tr><td rowspan=2 colspan=1>PanelMember</td><td rowspan=1 colspan=2>Site 1</td><td rowspan=1 colspan=2>Site 2</td><td rowspan=1 colspan=2>Site 3</td><td rowspan=1 colspan=3>All Sites</td></tr><tr><td rowspan=1 colspan=1>Avg.Ct</td><td rowspan=1 colspan=1>% CV</td><td rowspan=1 colspan=1>Avg.Ct</td><td rowspan=1 colspan=1>% CV</td><td rowspan=1 colspan=1>Avg.Ct</td><td rowspan=1 colspan=1>% CV</td><td rowspan=1 colspan=1>Avg.Ct</td><td rowspan=1 colspan=1>% CV</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=1>Flu A HongKong - LP</td><td rowspan=1 colspan=1>32.3</td><td rowspan=1 colspan=1>1.7%</td><td rowspan=1 colspan=1>32.9</td><td rowspan=1 colspan=1>3.7%</td><td rowspan=1 colspan=1>32.8</td><td rowspan=1 colspan=1>1.7%</td><td rowspan=1 colspan=1>32.7</td><td rowspan=1 colspan=1>2.6%</td><td rowspan=1 colspan=1>95.9-100.0%</td></tr><tr><td rowspan=1 colspan=1>Flu A HongKong - MP</td><td rowspan=1 colspan=1>31.7</td><td rowspan=1 colspan=1>1.5%</td><td rowspan=1 colspan=1>31.5</td><td rowspan=1 colspan=1>1.4%</td><td rowspan=1 colspan=1>31.9</td><td rowspan=1 colspan=1>1.4%</td><td rowspan=1 colspan=1>31.7</td><td rowspan=1 colspan=1>1.5%</td><td rowspan=1 colspan=1>95.9-100.0%</td></tr><tr><td rowspan=1 colspan=1>Flu BMalaysia -LP</td><td rowspan=1 colspan=1>30.8</td><td rowspan=1 colspan=1>2.2%</td><td rowspan=1 colspan=1>30.9</td><td rowspan=1 colspan=1>1.9%</td><td rowspan=1 colspan=1>31.5</td><td rowspan=1 colspan=1>2.0%</td><td rowspan=1 colspan=1>31.1</td><td rowspan=1 colspan=1>2.2%</td><td rowspan=1 colspan=1>95.9-100.0%</td></tr><tr><td rowspan=1 colspan=1>Flu BMalaysia -MP</td><td rowspan=1 colspan=1>30.0</td><td rowspan=1 colspan=1>1.2%</td><td rowspan=1 colspan=1>29.9</td><td rowspan=1 colspan=1>1.5%</td><td rowspan=1 colspan=1>30.4</td><td rowspan=1 colspan=1>1.6%</td><td rowspan=1 colspan=1>30.1</td><td rowspan=1 colspan=1>1.6%</td><td rowspan=1 colspan=1>95.9-100.0%</td></tr><tr><td rowspan=1 colspan=1>2019-nCoVLP</td><td rowspan=1 colspan=1>30.2</td><td rowspan=1 colspan=1>1.4%</td><td rowspan=1 colspan=1>29.5</td><td rowspan=1 colspan=1>1.4%</td><td rowspan=1 colspan=1>30.2</td><td rowspan=1 colspan=1>2.2%</td><td rowspan=1 colspan=1>30.0</td><td rowspan=1 colspan=1>2.0%</td><td rowspan=1 colspan=1>95.9-100.0%</td></tr><tr><td rowspan=1 colspan=1>2019-nCoVMP</td><td rowspan=1 colspan=1>29.0</td><td rowspan=1 colspan=1>1.6%</td><td rowspan=1 colspan=1>28.8</td><td rowspan=1 colspan=1>1.6%</td><td rowspan=1 colspan=1>29.4</td><td rowspan=1 colspan=1>2.1%</td><td rowspan=1 colspan=1>29.1</td><td rowspan=1 colspan=1>1.9%</td><td rowspan=1 colspan=1>95.9-100.0%</td></tr><tr><td rowspan=1 colspan=1>PC Flu A</td><td rowspan=1 colspan=1>26.2</td><td rowspan=1 colspan=1>1.2%</td><td rowspan=1 colspan=1>26.1</td><td rowspan=1 colspan=1>0.8%</td><td rowspan=1 colspan=1>26.0</td><td rowspan=1 colspan=1>0.8%</td><td rowspan=1 colspan=1>26.1</td><td rowspan=1 colspan=1>1.0%</td><td rowspan=1 colspan=1>95.9-100.0%</td></tr><tr><td rowspan=1 colspan=1>PC Flu B</td><td rowspan=1 colspan=1>27.6</td><td rowspan=1 colspan=1>3.2%</td><td rowspan=1 colspan=1>28.2</td><td rowspan=1 colspan=1>1.4%</td><td rowspan=1 colspan=1>27.1</td><td rowspan=1 colspan=1>0.8%</td><td rowspan=1 colspan=1>27.6</td><td rowspan=1 colspan=1>2.6%</td><td rowspan=1 colspan=1>95.9-100.0%</td></tr><tr><td rowspan=1 colspan=1>PC nCoV</td><td rowspan=1 colspan=1>27.8</td><td rowspan=1 colspan=1>4.2%</td><td rowspan=1 colspan=1>27.2</td><td rowspan=1 colspan=1>0.9%</td><td rowspan=1 colspan=1>26.8</td><td rowspan=1 colspan=1>0.8%</td><td rowspan=1 colspan=1>27.3</td><td rowspan=1 colspan=1>2.9%</td><td rowspan=1 colspan=1>95.9-100.0%</td></tr><tr><td rowspan=1 colspan=1>NTC Flu A</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>95.9-100.0%</td></tr><tr><td rowspan=1 colspan=1>NTC Flu B</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>95.9-100.0%</td></tr><tr><td rowspan=1 colspan=1>NTC nCoV</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>95.9-100.0%</td></tr></table>

$\mathrm { L P = L o w }$ Positve, MP $=$ Moderate Positive, nCoV = SARS-CoV-2, ${ \mathrm { N T C } } = { \mathrm { N o } }$ Template Control, $\mathrm { P C } =$ Positive Control/ $\mathrm { { N / A } = 1 }$ Not applicable

Table 11. Reproducibility Variance Components   

<table><tr><td colspan="1" rowspan="2">PanelMember</td><td colspan="1" rowspan="2">N</td><td colspan="1" rowspan="2">MeanCt</td><td colspan="2" rowspan="1">BetweenDay</td><td colspan="2" rowspan="1">BetweenOperator*</td><td colspan="2" rowspan="1">BetweenSite</td><td colspan="2" rowspan="1">Within Day</td><td colspan="2" rowspan="1">TotalReproducibility</td></tr><tr><td colspan="1" rowspan="1">SD</td><td colspan="1" rowspan="1">%CV</td><td colspan="1" rowspan="1">SD</td><td colspan="1" rowspan="1">%CV</td><td colspan="1" rowspan="1">SD</td><td colspan="1" rowspan="1">%CV</td><td colspan="1" rowspan="1">SD</td><td colspan="1" rowspan="1">%CV</td><td colspan="1" rowspan="1">SD</td><td colspan="1" rowspan="1">% CV</td></tr><tr><td colspan="1" rowspan="1">Flu A HongKong - LP</td><td colspan="1" rowspan="1">90</td><td colspan="1" rowspan="1">32.7</td><td colspan="1" rowspan="1">0.24</td><td colspan="1" rowspan="1">0.7</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">0.83</td><td colspan="1" rowspan="1">2.5</td><td colspan="1" rowspan="1">0.87</td><td colspan="1" rowspan="1">2.7</td></tr><tr><td colspan="1" rowspan="1"> FluA HongKong  MP</td><td colspan="1" rowspan="1">90</td><td colspan="1" rowspan="1">31.7</td><td colspan="1" rowspan="1">0.13</td><td colspan="1" rowspan="1">0.4</td><td colspan="1" rowspan="1">0.13</td><td colspan="1" rowspan="1">0.4</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">0.44</td><td colspan="1" rowspan="1">1.4</td><td colspan="1" rowspan="1">0.48</td><td colspan="1" rowspan="1">1.5</td></tr><tr><td colspan="1" rowspan="1">Flu BMalaysia-LP</td><td colspan="1" rowspan="1">90</td><td colspan="1" rowspan="1">31.1</td><td colspan="1" rowspan="1">0.31</td><td colspan="1" rowspan="1">1.0</td><td colspan="1" rowspan="1">0.21</td><td colspan="1" rowspan="1">0.7</td><td colspan="1" rowspan="1">0.29</td><td colspan="1" rowspan="1">0.9</td><td colspan="1" rowspan="1">0.55</td><td colspan="1" rowspan="1">1.8</td><td colspan="1" rowspan="1">0.73</td><td colspan="1" rowspan="1">2.3</td></tr><tr><td colspan="1" rowspan="1">Flu BMalaysia -MP</td><td colspan="1" rowspan="1">90</td><td colspan="1" rowspan="1">30.1</td><td colspan="1" rowspan="1">0.26</td><td colspan="1" rowspan="1">0.9</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">0.17</td><td colspan="1" rowspan="1">0.6</td><td colspan="1" rowspan="1">0.40</td><td colspan="1" rowspan="1">1.3</td><td colspan="1" rowspan="1">0.50</td><td colspan="1" rowspan="1">1.7</td></tr><tr><td colspan="1" rowspan="1">2019-nCoVLP</td><td colspan="1" rowspan="1">90</td><td colspan="1" rowspan="1">30.0</td><td colspan="1" rowspan="1">0.37</td><td colspan="1" rowspan="1">1.2</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">0.13</td><td colspan="1" rowspan="1">0.4</td><td colspan="1" rowspan="1">0.49</td><td colspan="1" rowspan="1">1.6</td><td colspan="1" rowspan="1">0.63</td><td colspan="1" rowspan="1">2.1</td></tr><tr><td colspan="1" rowspan="1">2019-nCoVMP</td><td colspan="1" rowspan="1">90</td><td colspan="1" rowspan="1">29.1</td><td colspan="1" rowspan="1">0.28</td><td colspan="1" rowspan="1">1.0</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">0.05</td><td colspan="1" rowspan="1">0.2</td><td colspan="1" rowspan="1">0.51</td><td colspan="1" rowspan="1">1.7</td><td colspan="1" rowspan="1">0.58</td><td colspan="1" rowspan="1">2.0</td></tr><tr><td colspan="1" rowspan="1">PC Flu A</td><td colspan="1" rowspan="1">90</td><td colspan="1" rowspan="1">26.1</td><td colspan="1" rowspan="1">0.06</td><td colspan="1" rowspan="1">0.2</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">0.15</td><td colspan="1" rowspan="1">0.6</td><td colspan="1" rowspan="1">0.21</td><td colspan="1" rowspan="1">0.8</td><td colspan="1" rowspan="1">0.26</td><td colspan="1" rowspan="1">1.0</td></tr><tr><td colspan="1" rowspan="1">PC Flu B</td><td colspan="1" rowspan="1">90</td><td colspan="1" rowspan="1">27.6</td><td colspan="1" rowspan="1">0.50</td><td colspan="1" rowspan="1">1.8</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">0.48</td><td colspan="1" rowspan="1">1.7</td><td colspan="1" rowspan="1">0.32</td><td colspan="1" rowspan="1">1.2</td><td colspan="1" rowspan="1">0.76</td><td colspan="1" rowspan="1">2.8</td></tr><tr><td colspan="1" rowspan="1">PC nCoV</td><td colspan="1" rowspan="1">90</td><td colspan="1" rowspan="1">27.3</td><td colspan="1" rowspan="1">0.43</td><td colspan="1" rowspan="1">1.6</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">0.68</td><td colspan="1" rowspan="1">2.5</td><td colspan="1" rowspan="1">0.26</td><td colspan="1" rowspan="1">1.0</td><td colspan="1" rowspan="1">0.85</td><td colspan="1" rowspan="1">3.1</td></tr><tr><td colspan="1" rowspan="1">NTC Flu A</td><td colspan="1" rowspan="1">90</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">N/A</td></tr><tr><td colspan="1" rowspan="1">NTC Flu B</td><td colspan="1" rowspan="1">90</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">N/A</td></tr><tr><td colspan="1" rowspan="1">NTC nCoV</td><td colspan="1" rowspan="1">90</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">N/A</td></tr></table>

$\mathrm { L P = L o w }$ Positve, $\mathbf { M P } =$ Moderate Positive, nCoV $=$ SARS-CoV-2, ${ \mathrm { N T C } } = { \mathrm { N o } }$ Template Control, PC $=$ Positive Control/ $\mathrm { N } / \mathrm { A } = \mathrm { N o t }$ applicable   
\* Since each operator used a different lot of reagents, Between Operator and Between Lot statistics are confounded variables.

3. Linearity:

Not applicable. This is a qualitative test.

4. Analytical Specificity/Interference:

Cross-reactivity

The Simplexa COVID-19 & Flu A/B Direct assay’s analytical specificity was evaluated in a study testing samples containing organisms that are present as normal flora in nasopharyngeal passages and those that cause similar clinical symptoms as influenza A, influenza B and/or SARS-CoV-2. Forty-four different pathogens including bacteria, viruses and one fungus were included in the study, with each organism tested in three replicates. No cross reactivity was observed with the organisms at the concentrations tested, as shown below.

Table 12. Cross-reactivity for Simplexa COVID-19 & Flu A/B Direct   

<table><tr><td colspan="1" rowspan="1">Organism</td><td colspan="1" rowspan="1">TestConcentration</td><td colspan="1" rowspan="1">Flu A %Detection (#positive/ #tested)</td><td colspan="1" rowspan="1">Flu B %Detection (#positive/ #tested)</td><td colspan="1" rowspan="1">SARS-CoV-2 %Detection (#positive/ #tested)</td><td colspan="1" rowspan="1">IC %Detection (#positive/ #tested)</td></tr><tr><td colspan="6" rowspan="1">Viruses</td></tr><tr><td colspan="1" rowspan="1">Adenovirus Type 1</td><td colspan="1" rowspan="1">1 x 105TCID50/mL</td><td colspan="1" rowspan="1">0.0% (0/3)</td><td colspan="1" rowspan="1">0.0% (0/3)</td><td colspan="1" rowspan="1">0.0% (0/3)</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">Adenovirus Type 7a</td><td colspan="1" rowspan="1">1 x 105TCID50/mL</td><td colspan="1" rowspan="1">0.0% (0/3)</td><td colspan="1" rowspan="1">0.0% (0/3)</td><td colspan="1" rowspan="1">0.0% (0/3)</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">Cytomegalovirus</td><td colspan="1" rowspan="1">1 x 105TCID50/mL</td><td colspan="1" rowspan="1">0.0% (0/3)</td><td colspan="1" rowspan="1">0.0% (0/3)</td><td colspan="1" rowspan="1">0.0% (0/3)</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">Enterovirus Type 68</td><td colspan="1" rowspan="1">1 x 105TCID50/mL</td><td colspan="1" rowspan="1">0.0% (0/3)</td><td colspan="1" rowspan="1">0.0% (0/3)</td><td colspan="1" rowspan="1">0.0% (0/3)</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">Enterovirus Type 71</td><td colspan="1" rowspan="1">1 x 105TCID50/mL</td><td colspan="1" rowspan="1">0.0% (0/3)</td><td colspan="1" rowspan="1">0.0% (0/3)</td><td colspan="1" rowspan="1">0.0% (0/3)</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">Epstein-Barr Virus</td><td colspan="1" rowspan="1">1 x 105copies/mL</td><td colspan="1" rowspan="1">0.0% (0/3)</td><td colspan="1" rowspan="1">0.0% (0/3)</td><td colspan="1" rowspan="1">0.0% (0/3)</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">Human Coronavirus229E</td><td colspan="1" rowspan="1">1 x 104TCID50/mL</td><td colspan="1" rowspan="1">0.0% (0/3)</td><td colspan="1" rowspan="1">0.0% (0/3)</td><td colspan="1" rowspan="1">0.0% (0/3)</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">Human CoronavirusNL63</td><td colspan="1" rowspan="1">1 x 104TCID50/mL</td><td colspan="1" rowspan="1">0.0% (0/3)</td><td colspan="1" rowspan="1">0.0% (0/3)</td><td colspan="1" rowspan="1">0.0% (0/3)</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">Human CoronavirusOC43</td><td colspan="1" rowspan="1">1 x 105TCID50/mL</td><td colspan="1" rowspan="1">0.0% (0/3)</td><td colspan="1" rowspan="1">0.0% (0/3)</td><td colspan="1" rowspan="1">0.0% (0/3)</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">Human CoronavirusRNA HKU1</td><td colspan="1" rowspan="1">1 x 105 genomecopies/mL</td><td colspan="1" rowspan="1">0.0% (0/3)</td><td colspan="1" rowspan="1">0.0% (0/3)</td><td colspan="1" rowspan="1">0.0% (0/3)</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">HumanMetapneumovirus 9</td><td colspan="1" rowspan="1">1 x 104TCID50/mL</td><td colspan="1" rowspan="1">0.0% (0/3)</td><td colspan="1" rowspan="1">0.0% (0/3)</td><td colspan="1" rowspan="1">0.0% (0/3)</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">Measles virus</td><td colspan="1" rowspan="1">1 x 105TCID50/mL</td><td colspan="1" rowspan="1">0.0% (0/3)</td><td colspan="1" rowspan="1">0.0% (0/3)</td><td colspan="1" rowspan="1">0.0% (0/3)</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">MERS-Coronavirus</td><td colspan="1" rowspan="1">1 x 105TCID50/mL</td><td colspan="1" rowspan="1">0.0% (0/3)</td><td colspan="1" rowspan="1">0.0% (0/3)</td><td colspan="1" rowspan="1">0.0% (0/3)</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">Mumps virus</td><td colspan="1" rowspan="1">1 x 105TCID50/mL</td><td colspan="1" rowspan="1">0.0% (0/3)</td><td colspan="1" rowspan="1">0.0% (0/3)</td><td colspan="1" rowspan="1">0.0% (0/3)</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza Type 1</td><td colspan="1" rowspan="1">1 x 105TCID50/mL</td><td colspan="1" rowspan="1">0.0% (0/3)</td><td colspan="1" rowspan="1">0.0% (0/3)</td><td colspan="1" rowspan="1">0.0% (0/3)</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza Type 2</td><td colspan="1" rowspan="1">1 x 105TCID50/mL</td><td colspan="1" rowspan="1">0.0% (0/3)</td><td colspan="1" rowspan="1">0.0% (0/3)</td><td colspan="1" rowspan="1">0.0% (0/3)</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza Type 3</td><td colspan="1" rowspan="1">1 x 105TCID50/mL</td><td colspan="1" rowspan="1">0.0% (0/3)</td><td colspan="1" rowspan="1">0.0% (0/3)</td><td colspan="1" rowspan="1">0.0% (0/3)</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza Type 4</td><td colspan="1" rowspan="1">1 x 105TCID50/mL</td><td colspan="1" rowspan="1">0.0% (0/3)</td><td colspan="1" rowspan="1">0.0% (0/3)</td><td colspan="1" rowspan="1">0.0% (0/3)</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">Parechovirus Type 3</td><td colspan="1" rowspan="1">1 x 105TCID50/mL</td><td colspan="1" rowspan="1">0.0% (0/3)</td><td colspan="1" rowspan="1">0.0% (0/3)</td><td colspan="1" rowspan="1">0.0% (0/3)</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">Rhinovirus 1A</td><td colspan="1" rowspan="1">1 x 104TCID50/mL</td><td colspan="1" rowspan="1">0.0% (0/3)</td><td colspan="1" rowspan="1">0.0% (0/3)</td><td colspan="1" rowspan="1">0.0% (0/3)</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">RSV-A</td><td colspan="1" rowspan="1">1 x 105TCID50/mL</td><td colspan="1" rowspan="1">0.0% (0/3)</td><td colspan="1" rowspan="1">0.0% (0/3)</td><td colspan="1" rowspan="1">0.0% (0/3)</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">RSV-B</td><td colspan="1" rowspan="1">1 x 105TCID50/mL</td><td colspan="1" rowspan="1">0.0% (0/3)</td><td colspan="1" rowspan="1">0.0% (0/3)</td><td colspan="1" rowspan="1">0.0% (0/3)</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Bacteria</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Bordetella pertussis</td><td colspan="1" rowspan="1">1 x 106 CFU/mL</td><td colspan="1" rowspan="1">0.0% (0/3)</td><td colspan="1" rowspan="1">0.0% (0/3)</td><td colspan="1" rowspan="1">0.0% (0/3)</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">Chlamydiapneumoniae</td><td colspan="1" rowspan="1">1 x 106 IFU/mL</td><td colspan="1" rowspan="1">0.0% (0/3)</td><td colspan="1" rowspan="1">0.0% (0/3)</td><td colspan="1" rowspan="1">0.0% (0/3)</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">Corynebacteriumdiphtheriae</td><td colspan="1" rowspan="1">1 x 106 CFU/mL</td><td colspan="1" rowspan="1">0.0% (0/3)</td><td colspan="1" rowspan="1">0.0% (0/3)</td><td colspan="1" rowspan="1">0.0% (0/3)</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">Coxiella burnetii</td><td colspan="1" rowspan="1">1 x 106copies/mL</td><td colspan="1" rowspan="1">0.0% (0/3)</td><td colspan="1" rowspan="1">0.0% (0/3)</td><td colspan="1" rowspan="1">0.0% (0/3)</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli0157:H7</td><td colspan="1" rowspan="1">1 x 106 CFU/mL</td><td colspan="1" rowspan="1">0.0% (0/3)</td><td colspan="1" rowspan="1">0.0% (0/3)</td><td colspan="1" rowspan="1">0.0% (0/3)</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">Haemophilusinfluenzae</td><td colspan="1" rowspan="1">1 x 106 CFU/mL</td><td colspan="1" rowspan="1">0.0% (0/3)</td><td colspan="1" rowspan="1">0.0% (0/3)</td><td colspan="1" rowspan="1">0.0% (0/3)</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">Lactobacillusplantarum</td><td colspan="1" rowspan="1">1 x 106 CFU/mL</td><td colspan="1" rowspan="1">0.0% (0/3)</td><td colspan="1" rowspan="1">0.0% (0/3)</td><td colspan="1" rowspan="1">0.0% (0/3)</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">Legionellalongbeachae</td><td colspan="1" rowspan="1">1 x 106 CFU/mL</td><td colspan="1" rowspan="1">0.0% (0/3)</td><td colspan="1" rowspan="1">0.0% (0/3)</td><td colspan="1" rowspan="1">0.0% (0/3)</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">Legionellapneumophila</td><td colspan="1" rowspan="1">1 x 106 CFU/mL</td><td colspan="1" rowspan="1">0.0% (0/3)</td><td colspan="1" rowspan="1">0.0% (0/3)</td><td colspan="1" rowspan="1">0.0% (0/3)</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">Leptospirainterrogans</td><td colspan="1" rowspan="1">1 x 106copies/mL</td><td colspan="1" rowspan="1">0.0% (0/3)</td><td colspan="1" rowspan="1">0.0% (0/3)</td><td colspan="1" rowspan="1">0.0% (0/3)</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">Moraxellacatarrhalis</td><td colspan="1" rowspan="1">1 x 106 CFU/mL</td><td colspan="1" rowspan="1">0.0% (0/3)</td><td colspan="1" rowspan="1">0.0% (0/3)</td><td colspan="1" rowspan="1">0.0% (0/3)</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">Mycobacteriumtuberculosis DNA</td><td colspan="1" rowspan="1">1 x 106copies/mL</td><td colspan="1" rowspan="1">0.0% (0/3)</td><td colspan="1" rowspan="1">0.0% (0/3)</td><td colspan="1" rowspan="1">0.0% (0/3)</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">Mycoplasmapneumoniae</td><td colspan="1" rowspan="1">1 x 106 CCU/mL</td><td colspan="1" rowspan="1">0.0% (0/3)</td><td colspan="1" rowspan="1">0.0% (0/3)</td><td colspan="1" rowspan="1">0.0% (0/3)</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">Neisseria elongata</td><td colspan="1" rowspan="1">1 x 106 CFU/mL</td><td colspan="1" rowspan="1">0.0% (0/3)</td><td colspan="1" rowspan="1">0.0% (0/3)</td><td colspan="1" rowspan="1">0.0% (0/3)</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">Neisseriameningitidis</td><td colspan="1" rowspan="1">1 x 106 CFU/mL</td><td colspan="1" rowspan="1">0.0% (0/3)</td><td colspan="1" rowspan="1">0.0% (0/3)</td><td colspan="1" rowspan="1">0.0% (0/3)</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">Pseudomonasaeruginosa</td><td colspan="1" rowspan="1">1 x 106 CFU/mL</td><td colspan="1" rowspan="1">0.0% (0/3)</td><td colspan="1" rowspan="1">0.0% (0/3)</td><td colspan="1" rowspan="1">0.0% (0/3)</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">Staphylococcusaureus</td><td colspan="1" rowspan="1">1 x 106 CFU/mL</td><td colspan="1" rowspan="1">0.0% (0/3)</td><td colspan="1" rowspan="1">0.0% (0/3)</td><td colspan="1" rowspan="1">0.0% (0/3)</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">Staphylococcusepidermidis</td><td colspan="1" rowspan="1">1 x 106 CFU/mL</td><td colspan="1" rowspan="1">0.0% (0/3)</td><td colspan="1" rowspan="1">0.0% (0/3)</td><td colspan="1" rowspan="1">0.0% (0/3)</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">Streptococcuspneumoniae</td><td colspan="1" rowspan="1">1 x 106 CFU/mL</td><td colspan="1" rowspan="1">0.0% (0/3)</td><td colspan="1" rowspan="1">0.0% (0/3)</td><td colspan="1" rowspan="1">0.0% (0/3)</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">Streptococcuspyogenes</td><td colspan="1" rowspan="1">1 x 106 CFU/mL</td><td colspan="1" rowspan="1">0.0% (0/3)</td><td colspan="1" rowspan="1">0.0% (0/3)</td><td colspan="1" rowspan="1">0.0% (0/3)</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">Streptococcussalivarius</td><td colspan="1" rowspan="1">1 x 106 CFU/mL</td><td colspan="1" rowspan="1">0.0% (0/3)</td><td colspan="1" rowspan="1">0.0% (0/3)</td><td colspan="1" rowspan="1">0.0% (0/3)</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="6" rowspan="1">Fungus</td></tr><tr><td colspan="1" rowspan="1">Candida albicans</td><td colspan="1" rowspan="1">1 x 106 CFU/mL</td><td colspan="1" rowspan="1">0.0% (0/3)</td><td colspan="1" rowspan="1">0.0% (0/3)</td><td colspan="1" rowspan="1">0.0% (0/3)</td><td colspan="1" rowspan="1">100% (3/3)</td></tr></table>

$\mathrm { T C I D } _ { 5 0 } =$ Tissue Culture $50 \%$ Infectious Dose, CFU $=$ Colony-forming units, $\operatorname { I F U } =$ Infectious units, $\mathrm { C C U } =$ Color changing units

# Competitive Interference

The performance of the Simplexa COVID-19 & Flu A/B Direct assay was evaluated for its ability to accurately detect influenza A, influenza B, and SARS-CoV-2 in cases of co-infection with the assay’s target organisms. Baseline samples were prepared by spiking COVID19/USA-WA1/2020, Influenza A/Hong Kong/8/1968 or Influenza B/Phuket/3073/2013 at 4x LoD into nasopharyngeal swab (NPS) matrix in UTM. A high level (1,000x LoD) of each of the viruses was added to each of the baseline samples and tested. No interference was observed for the target analytes in the presence of potentially competing co-infection analytes at the concentrations and combinations tested.

Table 13. Competitive Interference of the Simplexa COVID-19 & Flu A/B Direct   

<table><tr><td rowspan=1 colspan=1>TargetAnalyte</td><td rowspan=1 colspan=1>Interferent</td><td rowspan=1 colspan=1>InterferentConcentration</td><td rowspan=1 colspan=1>Flu ADetectiona</td><td rowspan=1 colspan=1>Flu AMeanCt</td><td rowspan=1 colspan=1>Flu BDetection</td><td rowspan=1 colspan=1>Flu BMeanCt</td><td rowspan=1 colspan=1>nCoVDetection</td><td rowspan=1 colspan=1>nCoVMeanCt</td></tr><tr><td rowspan=1 colspan=1>Flu A</td><td rowspan=1 colspan=1>None</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>5/5</td><td rowspan=1 colspan=1>32.7</td><td rowspan=1 colspan=1>0/5</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>0/5</td><td rowspan=1 colspan=1>N/A</td></tr><tr><td rowspan=1 colspan=1>Flu B</td><td rowspan=1 colspan=1>None</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>0/5</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>5/5</td><td rowspan=1 colspan=1>31.0</td><td rowspan=1 colspan=1>0/5</td><td rowspan=1 colspan=1>N/A</td></tr><tr><td rowspan=1 colspan=1>nCoV</td><td rowspan=1 colspan=1>None</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>0/5</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>0/5</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>5/5</td><td rowspan=1 colspan=1>29.2</td></tr><tr><td rowspan=1 colspan=1>Flu A</td><td rowspan=1 colspan=1>Flu B</td><td rowspan=1 colspan=1>7.5 x 105</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>32.5</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>22.5</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>N/A</td></tr><tr><td rowspan=1 colspan=1>Flu A</td><td rowspan=1 colspan=1>nCoV</td><td rowspan=1 colspan=1>5.0 x 105</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>32.7</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>21.6</td></tr><tr><td rowspan=1 colspan=1>Flu B</td><td rowspan=1 colspan=1>Flu A</td><td rowspan=1 colspan=1>5.0 x 105</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>24.4</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>32.4</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>N/A</td></tr><tr><td rowspan=1 colspan=1>Flu B</td><td rowspan=1 colspan=1>nCoV</td><td rowspan=1 colspan=1>5.0 x 105</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>30.7</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>21.2</td></tr><tr><td rowspan=1 colspan=1>nCoV</td><td rowspan=1 colspan=1>Flu A</td><td rowspan=1 colspan=1>5.0 x 105</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>24.2</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>29.4</td></tr><tr><td rowspan=1 colspan=1>nCoV</td><td rowspan=1 colspan=1>Flu B</td><td rowspan=1 colspan=1>7.5 x 105</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>22.8</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>28.6</td></tr></table>

a “Detection” defined as # of positive tests / # tested nCoV = SARS-CoV-2, $\mathrm { { N / A = } }$ not applicable

Microbial Interference

The performance of the Simplexa COVID-19 & Flu A/B Direct assay was evaluated for its ability to accurately detect influenza A, influenza B, and SARS-CoV-2 in the presence of other clinically relevant pathogens and related biological material. The same panel as above in the cross-reactivity study, consisting of forty-seven potentially inhibitory organisms, was individually spiked into a pool containing low concentrations (approximately 2x LoD) each of the influenza A, influenza B and SARS-CoV-2 viruses. Samples were assayed in three replicates. No inhibition was observed for influenza A, influenza B, or SARS-CoV-2 by the organisms at the concentrations shown below.

Table 14. Microbial Interference of the Simplexa COVID-19 & Flu A/B Direct   

<table><tr><td colspan="1" rowspan="1">Organism</td><td colspan="1" rowspan="1">TestConcentration</td><td colspan="1" rowspan="1">Flu A %Detection (#positive/#tested)</td><td colspan="1" rowspan="1">Flu B %Detection (#positive/ #tested)</td><td colspan="1" rowspan="1">SARS-CoV-2% Detection(# positive/ #teested</td><td colspan="1" rowspan="1">IC %Detection (#positive/##tested)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Viruses</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Adenovirus Type 1</td><td colspan="1" rowspan="1">1 x 105TCID50/mL</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">Adenovirus Type 7a</td><td colspan="1" rowspan="1">1 x 105TCID50/mL</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">Cytomegalovirus</td><td colspan="1" rowspan="1">1 x 105TCID50/mL</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">Enterovirus Type 68</td><td colspan="1" rowspan="1">1 x 105TCID50/mL</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">Enterovirus Type 71</td><td colspan="1" rowspan="1">1 x 105TCID50/mL</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">Epstein-Barr Virus</td><td colspan="1" rowspan="1">1 x 105copies/mL</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">Human Coronavirus229E</td><td colspan="1" rowspan="1">1 x 104TCID50/mL</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">Human CoronavirusNL63</td><td colspan="1" rowspan="1">1 x 104TCID50/mL</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">Human CoronavirusOC43</td><td colspan="1" rowspan="1">1 x 105TCID50/mL</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">Human CoronavirusRNA HKU1</td><td colspan="1" rowspan="1">1 x 105$TCID50/mL</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">HumanMetapneumovirus 9</td><td colspan="1" rowspan="1">1 x 104TCID50/mL</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">Measles virus</td><td colspan="1" rowspan="1">1 x 105TCID50/mL</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">MERS-Coronavirus</td><td colspan="1" rowspan="1">1 x 105TCID50/mL</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">Mumps virus</td><td colspan="1" rowspan="1">1 x 105TCID50/mL</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza Type 1</td><td colspan="1" rowspan="1">1 x 105TCID50/mL</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza Type 2</td><td colspan="1" rowspan="1">1 x 105TCID50/mL</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza Type 3</td><td colspan="1" rowspan="1">1 x 105TCID50/mL</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza Type 4</td><td colspan="1" rowspan="1">1 x 105TCID50/mL</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">Parechovirus Type 3</td><td colspan="1" rowspan="1">1 x 105TCID50/mL</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">Rhinovirus 1A</td><td colspan="1" rowspan="1">1 x 104TCID50/mL</td><td colspan="1" rowspan="1">95% (19/20)</td><td colspan="1" rowspan="1">100% (20/20)</td><td colspan="1" rowspan="1">100% (20/20)</td><td colspan="1" rowspan="1">100% (20/20)</td></tr><tr><td colspan="1" rowspan="1">RSV-A</td><td colspan="1" rowspan="1">1 x 105TCID50/mL</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">RSV-B</td><td colspan="1" rowspan="1">1 x 105TCID50/mL</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Bacteria</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Bordetella pertussis</td><td colspan="1" rowspan="1">1 x 106 CFU/mL</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">Chlamydiapneumoniae</td><td colspan="1" rowspan="1">1 x 106 IFU/mL</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">Corynebacteriumdiphtheriae</td><td colspan="1" rowspan="1">1 x 106 CFU/mL</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">Coxiella burnetii</td><td colspan="1" rowspan="1">1 x 106copies/mL</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli0157:H7</td><td colspan="1" rowspan="1">1 x 106 CFU/mL</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">Haemophilusinfluenzae</td><td colspan="1" rowspan="1">1 x 106 CFU/mL</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">Lactobacillusplantarum</td><td colspan="1" rowspan="1">1 x 106 CFU/mL</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">Legionellalongbeachae</td><td colspan="1" rowspan="1">1 x 106 CFU/mL</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">Legionellapneumophila</td><td colspan="1" rowspan="1">1 x 106 CFU/mL</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">Leptospirainterrogans</td><td colspan="1" rowspan="1">1 x 106copies/mL</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">Moraxellacatarrhalis</td><td colspan="1" rowspan="1">1 x 106 CFU/mL</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">Mycobacteriumtuberculosis DNA</td><td colspan="1" rowspan="1">1 x 106copies/mL</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">Mycoplasmapneumoniae</td><td colspan="1" rowspan="1">1 x 106 CCU/mL</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">Neisseria elongata</td><td colspan="1" rowspan="1">1 x 106 CFU/mL</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">Neisseriameningitidis</td><td colspan="1" rowspan="1">1 x 106 CFU/mL</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">Pseudomonasaeruginosa</td><td colspan="1" rowspan="1">1 x 106 CFU/mL</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">Staphylococcusaureus</td><td colspan="1" rowspan="1">1 x 106 CFU/mL</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">Staphylococcusepidermidis</td><td colspan="1" rowspan="1">1 x 106 CFU/mL</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">Streptococcuspneumoniae</td><td colspan="1" rowspan="1">1 x 106 CFU/mL</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">Streptococcuspyogenes</td><td colspan="1" rowspan="1">1 x 106 CFU/mL</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">Streptococcussalivarius</td><td colspan="1" rowspan="1">1 x 106 CFU/mL</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Fungus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Candida albicans</td><td colspan="1" rowspan="1">1 x 106 CFU/mL</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Other Biological Material</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Human genomicDNA</td><td colspan="1" rowspan="1">1 x 106 cells/mL</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">Pooled Human NasalWash</td><td colspan="1" rowspan="1">Neat (un-diluted)</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">100% (3/3)</td></tr><tr><td colspan="1" rowspan="1">SARS-CoV-1Synthetic RNA</td><td colspan="1" rowspan="1">1 x 105 genomecopies/mL</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">100% (3/3)</td><td colspan="1" rowspan="1">100% (3/3)</td></tr></table>

Interfering Substances

The performance of the Simplexa COVID-19 & Flu $\mathrm { A } / \mathrm { B }$ Direct assay was evaluated with potentially interfering substances that may be present in nasopharyngeal passages. A total of 12 potentially interfering substances were individually spiked into a pooled nasopharyngeal swab matrix containing SARS-CoV-2 (2019-nCoV/USA-WA1/2020), Flu A (Hong Kong/8/68) and Flu B (Malaysia/2506/04) inactivated viral particles, each at a targeted concentration approximately $3 \mathbf { x }$ LoD; each sample was tested in three replicates. There was one potential occurrence of interference with Cold Eeze oral analgesic at the higher concentration. All other substances showed no interference at the concentrations listed, including Cold Eeze oral

analgesic at the lower concentration. The FluMist nasal vaccine was not tested as an interfering substance due to its unavailability at the time of this study.

Table 15. Interference of Endogenous and Exogenous Substances on the Simplexa COVID-19 & Flu A/B Direct   

<table><tr><td rowspan=1 colspan=1>PotentialInterferent</td><td rowspan=1 colspan=1>Active Inredient</td><td rowspan=1 colspan=1>TestConcentration</td><td rowspan=1 colspan=1>Flu Adetection</td><td rowspan=1 colspan=1>Flu Bdetection</td><td rowspan=1 colspan=1>nCoVdetection</td><td rowspan=1 colspan=1>ICdetection</td></tr><tr><td rowspan=1 colspan=1>Afrin NasalSpray</td><td rowspan=1 colspan=1>Oxymetazoline</td><td rowspan=1 colspan=1>15% (v/v)</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Antibacterial</td><td rowspan=1 colspan=1>Tobramycin</td><td rowspan=1 colspan=1>4 μg/mL</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Nasalantibiotic</td><td rowspan=1 colspan=1>Mupirocin</td><td rowspan=1 colspan=1>6.6 mg/mL</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Blood</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>2% (v/v)</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=2 colspan=1>Bovinesubmaxillarygland, I-S</td><td rowspan=2 colspan=1>Purified MucinProtein</td><td rowspan=1 colspan=1>60 μg/mL</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>5 mg/mL</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=2 colspan=1>Cold Eeze oralanalgesic</td><td rowspan=2 colspan=1>N/A</td><td rowspan=1 colspan=1>2.5% (w/v)</td><td rowspan=1 colspan=1>4/4</td><td rowspan=1 colspan=1>4/4</td><td rowspan=1 colspan=1>2/2*</td><td rowspan=1 colspan=1>4/4</td></tr><tr><td rowspan=1 colspan=1>1.25% (w/v)</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Nasalcorticosteroid</td><td rowspan=1 colspan=1>Beclomethasone</td><td rowspan=1 colspan=1>5% (w/v)</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Nasalcorticosteroid</td><td rowspan=1 colspan=1>Fluticasone</td><td rowspan=1 colspan=1>5% (w/v)</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Relenza</td><td rowspan=1 colspan=1>Zanamivir</td><td rowspan=1 colspan=1>3.3 mg/mL</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Tamiflu</td><td rowspan=1 colspan=1>Oseltamivir</td><td rowspan=1 colspan=1>1 μM</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Zicam NasalGel</td><td rowspan=1 colspan=1>Luffaopperculata,Galphimiaglauca,histaminumhydrochloricum</td><td rowspan=1 colspan=1>5% (w/v)</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Zicam NasalSpray</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>10% (v/v)</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td></tr></table>

$\mathrm { n C o V } = \mathrm { S A R S } – \mathrm { C o V } { - 2 }$ , $\mathrm { { N / A = } }$ not applicable \* There was one error result for SARS- $\mathrm { C o V } { - } 2$ detection using Cold Eeze oral analgesic at the $2 . 5 \%$ concentration. The test was repeated and an error occurred again. Cold Eeze oral analgesic was retested at the $1 . 2 5 \%$ concentration and all analytes were detected as expected.

# 5. Assay Reportable Range:

Not applicable. This is a qualitative assay.

6. Traceability and Stability (Controls, Calibrators, or Methods):

Specimen Stability

For the specimen stability study, samples were contrived in individual negative samples as described for the Fresh vs. Frozen study (See below). The samples were tested, then stored at $2 { - } 8 ^ { \circ } \mathrm { C }$ and tested after three days, after five days, after seven days, and after ten days. For this analysis, the results for the low positive samples ( $1 . 5 \mathrm { x }$ LoD and 2x LoD) were combined. Results for positive specimens for each target analyte are summarized in the table below:

Table 16. Specimen Stability for the Simplexa COVID-19 & Flu A/B Direct   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=5>Timepoint</td></tr><tr><td rowspan=1 colspan=1>Target</td><td rowspan=1 colspan=1>SampleType</td><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>3 Days 8°C</td><td rowspan=1 colspan=1>5 Days 8°C</td><td rowspan=1 colspan=1>7 Days 8°C</td><td rowspan=1 colspan=1>10 Days 8°C</td></tr><tr><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>Media</td><td rowspan=1 colspan=1>0/10</td><td rowspan=1 colspan=1>0/10</td><td rowspan=1 colspan=1>0/10</td><td rowspan=1 colspan=1>0/10</td><td rowspan=1 colspan=1>0/10</td></tr><tr><td rowspan=2 colspan=1>Flu A</td><td rowspan=1 colspan=1>5x LoD</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>10/10</td></tr><tr><td rowspan=1 colspan=1>1.5x-2x LoD</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>20/20</td></tr><tr><td rowspan=2 colspan=1>Flu B</td><td rowspan=1 colspan=1>5x LoD</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>10/10</td></tr><tr><td rowspan=1 colspan=1>1.5x-2x LoD</td><td rowspan=1 colspan=1>18/20</td><td rowspan=1 colspan=1>18/20</td><td rowspan=1 colspan=1>19/20</td><td rowspan=1 colspan=1>19/20</td><td rowspan=1 colspan=1>20/20</td></tr><tr><td rowspan=2 colspan=1>SARS-CoV-2</td><td rowspan=1 colspan=1>5x LoD</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>10/10</td></tr><tr><td rowspan=1 colspan=1>1.5x-2x LoD</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>20/20</td></tr></table>

Although $90 \%$ detection was achieved for Influenza B after 3 days of storage, it was consistent with the results obtained on fresh samples and, therefore, was likely due to the concentration being close to the analytical LoD. Subsequent timepoints showed $9 5 \%$ to $100 \%$ detection for Influenza B. The package insert indicates that specimens can be stored for up to 7 days at $2 { - } 8 ~ ^ { \circ } \mathrm { C }$ .

Fresh versus Frozen

The Simplexa COVID-19 & Flu A/B Direct assay was evaluated for the product’s ability to detect low levels of SARS-CoV-2, influenza A and influenza B in nasopharyngeal swabs in UTM, which were stored below $- 7 0 \mathrm { { } ^ { \circ } C }$ . Fresh vs frozen stability study showed that samples can be frozen for three days at or below $- 7 0 \mathrm { { } ^ { \circ } C }$ for three freeze-thaw cycles.

Samples were stored at $\le - 7 0 ^ { \circ } \mathrm { C }$ for at least twenty-four 24 hours before thawing and testing. Additional freeze/thaw timepoints were tested (i.e., thaws $\# 2$ and #3) with each test point being performed following at least a full 24-hour time frozen at $- 7 0 \mathrm { { } ^ { \circ } C }$ between.

Table 17. Simplexa COVID-19 & Flu A/B Direct Fresh versus Frozen Equivalency   

<table><tr><td rowspan=1 colspan=1>Target</td><td rowspan=1 colspan=1>Sample Type</td><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>1st Thaw</td><td rowspan=1 colspan=1>2nd Thaw</td><td rowspan=1 colspan=1>3rd Thaw</td></tr><tr><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>Media</td><td rowspan=1 colspan=1>0/10</td><td rowspan=1 colspan=1>0/10</td><td rowspan=1 colspan=1>0/10</td><td rowspan=1 colspan=1>0/10</td></tr><tr><td rowspan=2 colspan=1>Flu A</td><td rowspan=1 colspan=1>5x LoD</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>10/10</td></tr><tr><td rowspan=1 colspan=1>1.5x-2x LoD</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>19/20</td></tr><tr><td rowspan=2 colspan=1>Flu B</td><td rowspan=1 colspan=1>5x LoD</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>9/10a</td><td rowspan=1 colspan=1>10/10</td></tr><tr><td rowspan=1 colspan=1>1.5x-2x LoD</td><td rowspan=1 colspan=1>18/20b</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>19/20</td><td rowspan=1 colspan=1>20/20</td></tr><tr><td rowspan=2 colspan=1>SARS-CoV-2</td><td rowspan=1 colspan=1>5x LoD</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>10/10</td></tr><tr><td rowspan=1 colspan=1>1.5x-2x LoD</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>18/20c</td></tr></table>

a Negative result was later retested and confirmed positive. 10/10 detections at the subsequent timepoint confirmed the frozen stability of that specimen type and concentration. b Two samples tested negative, possibly due to virus loads being at or below LoD. Subsequent tests showed at least $9 5 \%$ detection for the three freeze/thaw timepoints. c Two samples tested negative, possibly due to virus lots being at or below LoD. The experiment was repeated the next day after the $4 ^ { \mathrm { t h } }$ freeze thaw and obtained the expected results.

# 7. Detection Limit:

The Limit of Detection (LoD) study was performed using five LIAISON MDX Instruments, two lots of Reaction Mix and one lot of Positive Control. A total of 77 runs were performed by three operators over four days. The test samples were prepared by diluting viral stocks of influenza A, influenza B, and heat-inactivated SARS-CoV-2 into pooled nasopharyngeal swab (NPS) specimens collected in UTM. Two strains of each influenza virus and one strain of SARS-CoV-2 were tested. Each of the virus strains was tested at five or more concentrations near the expected LoD during the initial screening, followed by confirmation testing at one or more concentrations with two lots of the Reaction Mix, in 40 replicates. The LoD was defined as the concentration of the virus that resulted in at least $9 5 \%$ detection during confirmation testing for each of the virus strains. The confirmed LoD values are shown below.

Table 18. Simplexa COVID-19 & Flu A/B Direct Limit of Detection Summary   

<table><tr><td rowspan=1 colspan=1>Simplexa COVID-19&amp; Flu A/B DirectTarget</td><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>Limit ofDetection</td><td rowspan=1 colspan=1># Detected/# Tested</td><td rowspan=1 colspan=1>Mean Ct±SD</td></tr><tr><td rowspan=2 colspan=1>Flu A</td><td rowspan=1 colspan=1>Influenza A/Hong Kong/8/68(H3N2)</td><td rowspan=1 colspan=1>500 copies/mL</td><td rowspan=1 colspan=1>40/40</td><td rowspan=1 colspan=1>34.0 ± 1.09</td></tr><tr><td rowspan=1 colspan=1>Influenza A/Michigan/45/2015(H1N1)</td><td rowspan=1 colspan=1>500 copies/mL</td><td rowspan=1 colspan=1>39/40</td><td rowspan=1 colspan=1>33.1 ± 1.01</td></tr><tr><td rowspan=2 colspan=1>Flu B</td><td rowspan=1 colspan=1>Influenza B/Malaysia/2506/04(Victoria)</td><td rowspan=1 colspan=1>250 copies/mL</td><td rowspan=1 colspan=1>39/40</td><td rowspan=1 colspan=1>32.0 ± 0.76</td></tr><tr><td rowspan=1 colspan=1>Influenza B/Phuket/3073/2013(Yamagata)</td><td rowspan=1 colspan=1>750 copies/mL</td><td rowspan=1 colspan=1>39/40</td><td rowspan=1 colspan=1>31.9 ± 0.60</td></tr><tr><td rowspan=1 colspan=1>SARS-CoV-2</td><td rowspan=1 colspan=1>COVID-19 (USA-WA1/2020)</td><td rowspan=1 colspan=1>500 copies/mL</td><td rowspan=1 colspan=1>40/40</td><td rowspan=1 colspan=1>30.8 ± 0.70</td></tr><tr><td rowspan=1 colspan=1>SARS-CoV-2 RNA</td><td rowspan=1 colspan=1>WHO International Standard</td><td rowspan=1 colspan=1>651 IU/mL</td><td rowspan=1 colspan=1>39/40</td><td rowspan=1 colspan=1>31.7 ± 1.40</td></tr></table>

# 8. Assay Cut-Off:

The fluorescent thresholds and assay Ct cutoffs remain unchanged from the settings determined in K201505 and K212147 for the Simplexa Flu A/B & RSV Direct Gen II and the Simplexa COVID-19 Direct, respectively.

9. Carry-Over:

Carry-over studies were performed as part of the analytical studies for K120413 and K201505 for the Simplexa Flu A/B & RSV Direct and Simplexa Flu A/B & RSV Direct Gen II.

# B Comparison Studies:

1. Method Comparison with Predicate Device: Not applicable.

2. Matrix Comparison: Not applicable.

# C Clinical Studies:

# 1. Clinical Study Performance

The performance of Simplexa COVID-19 & Flu A/B Direct was evaluated in a clinical comparison study using prospective nasopharyngeal swab specimens, and retrospective archived nasopharyngeal swab specimens from patients with signs and symptoms of respiratory tract infection.

Prospective specimens were collected from six (6) geographically diverse clinical sites within the United States between August 2021 and March 2022. There were a total of 1427 specimens enrolled for testing. Amplicon contamination at one testing site resulted in removal of 226 enrolled specimens. Total excluded samples were 257 which left 1170 prospective specimens suitable for testing. There were 1152/1170 valid results with the Simplexa COVID-19 & Flu A/B Direct after the first test resulting in an initial invalid rate of $1 . 5 \%$ . The final invalid rate after re-test (according to test protocols) was $0 . 2 \%$ (2/1170 invalid results). Evaluable specimens for comparator testing with Influenza A and Influenza B were 1168 and 1165, respectively.

Retrospective archived specimens consisted of 82 positive influenza A, 114 positive influenza B, and 62 negative specimens blinded and randomized for the study. There were no excluded samples in the retrospective study.

The Simplexa COVID-19 & Flu A/B Direct clinical agreement testing was performed at two (2) external clinical sites and one (1) internal site. The comparator for influenza A and B targets was an FDA cleared molecular test. For SARS-CoV-2 comparator testing, a composite test method (CRM) consisting of three COVID-19 Emergency Use Authorized RT-PCR assays was performed. Two out of three positive results determined “Detected” CRM and two out of three negative results determined “Not Detected” CRM. One of the three EUA-authorized tests became unavailable partway through the prospective clinical study and therefore only 496 of the prospective specimens were available for SARS-CoV-2 comparison testing.

Performance was evaluated using a multi-assay comparator algorithm (for SARS-CoV-2 performance) and an FDA-cleared comparator assay (for Influenza A and Influenza B). Tables 19-21 shows the results of the Simplexa COVID-19 & Flu A/B Direct assay and comparator assay results in the prospective study analysis and Tables 22-24 shows the results of the retrospective study analysis. The positive percent agreement (PPA) and negative percent agreement (NPA) were calculated for each target.

Table 19. Clinical Performance for the Simplexa COVID-19 & Flu A/B Di rect – Prospective Specimens (Influenza A)   

<table><tr><td rowspan=2 colspan=1>COVID-19 &amp; Flu A/BDirect</td><td rowspan=1 colspan=2>Comparator</td><td rowspan=2 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>Not Detected</td></tr><tr><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>57</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>59</td></tr><tr><td rowspan=1 colspan=1>Not Detected</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>1104</td><td rowspan=1 colspan=1>1109</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>62</td><td rowspan=1 colspan=1>1106</td><td rowspan=1 colspan=1>1168</td></tr><tr><td rowspan=1 colspan=1>PPA = 91.9% (57/62)a</td><td rowspan=1 colspan=3>NPA = 99.8% (1104/1106)b</td></tr><tr><td rowspan=1 colspan=1>95% C.I.: 82.5%-96.5%</td><td rowspan=1 colspan=3>95% C.I.: 99.3%-100%</td></tr></table>

a Five (5) specimens were negative by an additional FDA cleared NAAT. b Two (2) specimens were positive by an additional FDA cleared NAAT. One of the two specimens (1/2) was tested with PCR followed by BDS and was positive.

Table 20. Clinical Performance for the Simplexa COVID-19 & Flu A/B Direct – Prospective Specimens (Influenza B)   

<table><tr><td rowspan=2 colspan=1>COVID-19 &amp; Flu A/BDirect</td><td rowspan=1 colspan=2>Comparator</td><td rowspan=2 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>Not Detected</td></tr><tr><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Not Detected</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1165</td><td rowspan=1 colspan=1>1165</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1165</td><td rowspan=1 colspan=1>1165</td></tr><tr><td rowspan=1 colspan=1>PPA = N/A</td><td rowspan=1 colspan=3>NPA = 100% (1165/1165)</td></tr><tr><td rowspan=1 colspan=1>95% C.I.: N/A</td><td rowspan=1 colspan=3>95% C.I.: 99.7%-100%</td></tr></table>

Table 21. Clinical Performance for the Simplexa COVID-19 & Flu A/B Direct – Prospective Specimens (SARS-CoV-2)   

<table><tr><td rowspan=2 colspan=1>COVID-19 &amp; Flu A/BDirect</td><td rowspan=1 colspan=2>Comparator</td><td rowspan=2 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>Not Detected</td></tr><tr><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>67</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>78</td></tr><tr><td rowspan=1 colspan=1>Not Detected</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>417</td><td rowspan=1 colspan=1>418</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>68</td><td rowspan=1 colspan=1>428</td><td rowspan=1 colspan=1>496</td></tr><tr><td rowspan=1 colspan=1>PPA = 98.5% (67/68)a</td><td rowspan=1 colspan=3>NPA = 97.4% (417/428)b</td></tr><tr><td rowspan=1 colspan=1>95% C.I.: 92.1%-99.7%</td><td rowspan=1 colspan=3>95% C.I.: 95.5%-98.6%</td></tr></table>

a One specimen was positive by an additional FDA cleared NAAT. b Nine of the eleven specimens (9/11) were positive by an additional FDA cleared NAAT and four (4) were positive by PCR followed by BDS.

Table 22. Clinical Performance for the Simplexa COVID-19 & Flu A/B Direct – Retrospective Specimens (Influenza A)   

<table><tr><td rowspan=2 colspan=1>COVID-19 &amp; Flu A/BDirect</td><td rowspan=1 colspan=2>Comparator</td><td rowspan=2 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>Not Detected</td></tr><tr><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>80</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>80</td></tr><tr><td rowspan=1 colspan=1>Not Detected</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>176</td><td rowspan=1 colspan=1>178</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>82</td><td rowspan=1 colspan=1>176</td><td rowspan=1 colspan=1>258</td></tr><tr><td rowspan=1 colspan=1>PPA = 97.6% (80/82)</td><td rowspan=1 colspan=3>NPA = 100% (176/176)</td></tr><tr><td rowspan=1 colspan=1>95% C.I.: 91.5%-99.3%</td><td rowspan=1 colspan=3>95% C.I.: 97.9%-100%</td></tr></table>

Table 23. Clinical Performance for the Simplexa COVID-19 & Flu A/B Direct – Retrospective Specimens (Influenza B)   

<table><tr><td rowspan=2 colspan=1>COVID-19 &amp; Flu A/BDirect</td><td rowspan=1 colspan=2>Comparator</td><td rowspan=2 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>Not Detected</td></tr><tr><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>112</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>112</td></tr><tr><td rowspan=1 colspan=1>Not Detected</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>144</td><td rowspan=1 colspan=1>146</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>114</td><td rowspan=1 colspan=1>144</td><td rowspan=1 colspan=1>258</td></tr><tr><td rowspan=1 colspan=1>PPA = 98.2% (112/114)</td><td rowspan=1 colspan=3>NPA = 100% (144/144)</td></tr><tr><td rowspan=1 colspan=1>95% C.I.: 93.8%-99.5%</td><td rowspan=1 colspan=3>95% C.I.: 97.4%-100%</td></tr></table>

Table 24. Clinical Performance for the Simplexa COVID-19 & Flu A/B Direct – Retrospective Specimens (SARS-CoV-2)   

<table><tr><td rowspan=2 colspan=1>COVID-19 &amp; Flu A/BDirect</td><td rowspan=1 colspan=2>Comparator</td><td rowspan=2 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>Not Detected</td></tr><tr><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Not Detected</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>252</td><td rowspan=1 colspan=1>252</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>252</td><td rowspan=1 colspan=1>252</td></tr><tr><td rowspan=1 colspan=1>PPA = N/A</td><td rowspan=1 colspan=3>NPA = 100%(252/252)</td></tr><tr><td rowspan=1 colspan=1>95% C.I.: N/A</td><td rowspan=1 colspan=3>95% C.I.: 98.5%-100%</td></tr></table>

$\mathrm { { N / A = } }$ Not applicable

# D Clinical Cut-Off:

Not applicable.

# E Expected Values/Reference Range:

Table 25. Simplexa COVID-19 & Flu A/B Direct Observed Positivity Rate   

<table><tr><td rowspan=1 colspan=1>Analyte</td><td rowspan=1 colspan=1># Positive &lt;5Years</td><td rowspan=1 colspan=1># Positive6-21 Years</td><td rowspan=1 colspan=1># Positive 22-65 Years</td><td rowspan=1 colspan=1># Positive &gt;65Years</td><td rowspan=1 colspan=1>Total Positivityrate</td></tr><tr><td rowspan=1 colspan=1>Influenza A</td><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>5.0% (59/1168)</td></tr><tr><td rowspan=1 colspan=1>Influenza B</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0% (0/1165)</td></tr><tr><td rowspan=1 colspan=1>SARS-CoV-2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>63</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>15.7% (78/496)</td></tr></table>

# VIII Proposed Labeling:

The labeling supports the finding of substantial equivalence for this device.

# IX Conclusion:

The submitted information in this premarket notification is complete and supports a substantial equivalence decision.