QIAGEN GmbH   
$\%$ Selina Salthouse   
Regulatory Affairs Manager   
QIAGEN Manchester Ltd   
CityLabs 2.0, 200 Hathersage Road   
Manchester, Manchester M13 0BH, United Kingdom

Re: K220062 Trade/Device Name: QIAstat-Dx Gastrointestinal Panel 2 Regulation Number: 21 CFR 866.3990 Regulation Name: Gastrointestinal microorganism multiplex nucleic acid-based assay Regulatory Class: Class II Product Code: PCH Dated: April 4, 2023 Received: April 4, 2023

Dear Selina Salthouse:

We have reviewed your section $5 1 0 ( \mathrm { k } )$ premarket notification of intent to market the device referenced above and have determined the device is substantially equivalent (for the indications for use stated in the enclosure) to legally marketed predicate devices marketed in interstate commerce prior to May 28, 1976, the enactment date of the Medical Device Amendments, or to devices that have been reclassified in accordance with the provisions of the Federal Food, Drug, and Cosmetic Act (the Act) that do not require approval of a premarket approval application (PMA). You may, therefore, market the device, subject to the general controls provisions of the Act. Although this letter refers to your product as a device, please be aware that some cleared products may instead be combination products. The 510(k) Premarket Notification Database available at https://www.accessdata.fda.gov/scripts/cdrh/cfdocs/cfpmn/pmn.cfm identifies combination product submissions. The general controls provisions of the Act include requirements for annual registration, listing of devices, good manufacturing practice, labeling, and prohibitions against misbranding and adulteration. Please note: CDRH does not evaluate information related to contract liability warranties. We remind you, however, that device labeling must be truthful and not misleading.

If your device is classified (see above) into either class II (Special Controls) or class III (PMA), it may be subject to additional controls. Existing major regulations affecting your device can be found in the Code of Federal Regulations, Title 21, Parts 800 to 898. In addition, FDA may publish further announcements concerning your device in the Federal Register.

Your device is also subject to, among other requirements, the Quality System (QS) regulation (21 CFR Part 820), which includes, but is not limited to, 21 CFR 820.30, Design controls; 21 CFR 820.90, Nonconforming product; and 21 CFR 820.100, Corrective and preventive action. Please note that regardless of whether a change requires premarket review, the QS regulation requires device manufacturers to review and approve changes to device design and production (21 CFR 820.30 and 21 CFR 820.70) and document changes and approvals in the device master record (21 CFR 820.181).

Please be advised that FDA's issuance of a substantial equivalence determination does not mean that FDA has made a determination that your device complies with other requirements of the Act or any Federal statutes and regulations administered by other Federal agencies. You must comply with all the Act's requirements, including, but not limited to: registration and listing (21 CFR Part 807); labeling (21 CFR Part 801 and Part 809); medical device reporting (reporting of medical device-related adverse events) (21 CFR Part 803) for devices or postmarketing safety reporting (21 CFR Part 4, Subpart B) for combination products (see https://www.fda.gov/combination-products/guidance-regulatory-information/postmarketing-safetyreporting-combination-products); good manufacturing practice requirements as set forth in the quality systems (QS) regulation (21 CFR Part 820) for devices or current good manufacturing practices (21 CFR Part 4, Subpart A) for combination products; and, if applicable, the electronic product radiation control provisions (Sections 531-542 of the Act); 21 CFR Parts 1000-1050.

Also, please note the regulation entitled, "Misbranding by reference to premarket notification" (21 CFR 807.97). For questions regarding the reporting of adverse events under the MDR regulation (21 CFR Part 803), please go to https://www.fda.gov/medical-devices/medical-device-safety/medical-device-reportingmdr-how-report-medical-device-problems.

For comprehensive regulatory information about medical devices and radiation-emitting products, including information about labeling regulations, please see Device Advice (https://www.fda.gov/medical  
devices/device-advice-comprehensive-regulatory-assistance) and CDRH Learn   
(https://www.fda.gov/training-and-continuing-education/cdrh-learn). Additionally, you may contact the Division of Industry and Consumer Education (DICE) to ask a question about a specific regulatory topic. See the DICE website (https://www.fda.gov/medical-devices/device-advice-comprehensive-regulatoryassistance/contact-us-division-industry-and-consumer-education-dice) for more information or contact DICE by email (DICE@fda.hhs.gov) or phone (1-800-638-2041 or 301-796-7100).

Sincerely,

# Noel J. Gerald -S

Noel J. Gerald, Ph.D.   
Branch Chief   
Bacterial Respiratory and Medical Countermeasures Branch Division of Microbiology Devices   
OHT7: Office of In Vitro Diagnostics   
Office of Product Evaluation and Quality   
Center for Devices and Radiological Health

Enclosure

Device Name QIAstat-Dx Gastrointestinal Panel 2

Indications for Use (Describe) Th Aa al uexciii IAsaDz 10u and parasites direcly from preserve stool sample (Para-Pak C&S or FecalSwab)btained romindividuals wi ss o Shigella pathotypes), and parasites are identified with the QIAstat-Dx Gastrointestinal Panel 2 :

Pathogen Para-Pak C&S FecalSwab Adenovirus F40/F41 Reported Reported Astrovirus Reported Reported Norovirus GI/GII Reported Reported Rotavirus A Reported Reported Campylobacter (C. jejuni, C. coli and C. upsaliensis) Reported Reported Shigella/Enteroinvasive Escherichia coli (EIEC) Reported Reported Enteropathogenic Escherichia coli (EPEC) Reported Not Reported •Enterotoxigenic Escherichia coli (ETEC) It/st Reported Reported Shiga-like toxin-producing Escherichia coli (STEC) stx1/stx2 Reported Not Reported (including specific identification of E. coli O157 serogroup within STEC)   
Salmonella Reported Reported Plesiomonas shigelloides Reported Reported Yersinia enterocolitica Reported Reported Cryptosporidium Reported Reported Cyclospora cayetanensis Reported Reported Entamoeba histolytica Reported Reported Giardia lamblia\* Reported Reported \*(Also known as Giardia intestinalis and Giardia duodenalis)   
Concomitant culture is necessary for organism recovery and further typing of bacterial agents.

Th IAsa Gastial ait snheeentl h iitssou on  oet   Asta- Gttial Thei ete may e sole or definitive cause of the disease.

Nl irritable bowel syndrome, or Crohn's disease.

This section applies only to requirements of the Paperwork Reduction Act of 1995.

# \*DO NOT SEND YOUR COMPLETED FORM TO THE PRA STAFF EMAIL ADDRESS BELOW.\*

Teurenohoctioationatverhours sec time o reviw instructions, searh existing data sources, gatherand maintain the data needed and coplet a review the collection finformation. Send comments regarding this burden estimateor anyother aspect of this information collection, including suggestions for reducing this burden, to:

Department of Health and Human Services Food and Drug Administration Office of Chief Information Officer Paperwork Reduction Act (PRA) Staff PRAStaff@fda.hhs.gov

"An agency may not conduct or sponsor, and a person is not required to respond to, a collection of information unless it displays a currently valid OMB number."

# 5.0 510(k) SUMMARY

# General Information

Submitted by:

QIAGEN GmbH Qiagen Strasse 1 Hilden, 40724,Germany

# Contact Person:

Selina Salthouse Manager, Regulatory Affairs QIAGEN Manchester Ltd Citylabs 2.0, 200 Hathersage Road Manchester, M13 0BH, UK

Phone: +44 7823 862 321 Fax: 44 161 204 1200 Email: selina.salthouse@qiagen.com

Date Prepared: May 30, 2024

Device Name:

QIAstat- $\mathbf { \nabla } \cdot \mathrm { D } \mathbf { x } ^ { \circledast }$ Gastrointestinal Panel 2

Trade Name: QIAstat- $\mathbf { D } \mathbf { x } ^ { \circled { \ 8 } }$ Gastrointestinal Panel 2

Common Name: QIAstat- $\mathbf { \nabla } \cdot \mathrm { D } \mathbf { x } ^ { \circledast }$ Gastrointestinal Panel 2

Classification: 21 CFR 866.3990 - Gastrointestinal Pathogen Panel Multiplex Nucleic Acid-Based Assay System

Product Code: PCH

# Predicate Device

<table><tr><td rowspan=1 colspan=1>Manufacturer</td><td rowspan=1 colspan=1>Product Name</td><td rowspan=1 colspan=1>510(k) No.</td></tr><tr><td rowspan=1 colspan=1>BioFire Diagnostics LLC</td><td rowspan=1 colspan=1>FilmArrayGastrointestinal (GI) Panel</td><td rowspan=1 colspan=1>K140407</td></tr></table>

# Device Description

QIAstat-Dx is based on single-test cartridges with pre-packaged reagents including both wet and dry chemistry to handle the sample preparation and detection steps for the presence of a range of selected analytes by PCR technology. After insertion of the sample, the QIAstat-Dx assay cartridge is processed by the QIAstat-Dx Analyzer 1.0.

# Principle of Operation

The QIAstat-Dx Gastrointestinal Panel 2 is part of the QIAstat-Dx system and works with the QIAstat-Dx Analyzer 1.0.

The QIAstat-Dx Gastrointestinal Panel 2 is intended to be used with stool samples in ParaPak C&S or FecalSwab transport media.

Once the cartridge has been inserted into the instrument, the test starts automatically and runs for approximately 78 minutes. When the test is finished, the cartridge is removed by the user and discarded. The QIAstat-Dx Analyzer 1.0 automatically interprets test results and displays a summary on the analyzer display screen. The results can be printed using a connected printer if needed. The detected analytes are displayed in red. For other analytes tested, they are displayed in green if not detected or in gray if not applicable or invalid. The analyzer will report if an error occurs during processing, in which case the test will fail and no results will be provided (screen will show “FAIL”).

# Sample Pre-treatment for PCR Inhibitors removal

Following insertion of the cartridge, the buffer located in Reservoir 1 is added inside the lysis chamber and homogenized with the sample using a rotor in the presence of silica beads. This enables separation of commonly found inhibitory substances in stool from the DNA/RNA by chemical means.

# Resuspension of Internal Control (IC) and Proteinase K

Following sample pre-treatment, the IC and Proteinase K is resuspended with the buffer located in Reservoir 2 (resuspension buffer). The buffer from Reservoir 2 is added to the interconnected IC cavity and Proteinase K cavity and transferred repeatedly between the Transfer Chamber and the cavities to ensure resuspension. The resuspended IC and Proteinase K are transferred to the sample cavity.

# Cell Lysis

Primary lysis of the cells and analytes present in a stool sample and IC occurs by a combination of chemical and mechanical processes using a rotor inside the lysis chamber in the presence of silica beads and a buffer that acts as a chemical agent in aiding the mechanical process. The fast movement of the rotor in the presence of the silica beads results in sample agitation, which creates turbulence and shear forces that favor the lysis of the cell wall.

After mechanical lysis is completed, the primary lysate is transferred to the purification chamber through a frit with $2 \AA { \cdot } 5 ~ { \mu \mathrm { m } }$ pore size. The second lysis buffer (from Reservoir 3) is added to the primary lysate to complete chemical lysis.

# Purification

Binding reagent (from Reservoir 4) is added to the lysate in the purification chamber, and the mix is passed through the silica membrane. In this process, the DNA/RNA molecules stick to the membrane, and the remaining components of the lysate are delivered to the waste chamber. Then the membrane is washed with a first washing buffer (from Reservoir 5) to wash away proteins. This is followed by a second washing step with a second washing buffer (from Reservoir 6) to remove any remaining substances other than the nucleic acids. A subsequent drying step eliminates volatile substances from the silica membrane. Prior to the elution step, the Transfer Chamber is rinsed with the rinsing buffer (from Reservoir 7) in order to remove any potential inhibitors from previous processing steps. At the end of the process, the nucleic acids are released from the membrane using an elution buffer (from Reservoir 8). The eluate is collected in the Transfer Chamber.

# Rehydration of Master Mix

A defined volume $( 1 3 5 \mu \mathrm { L } )$ of the eluate is delivered to the reservoir of the Dry Chemistry Container (DCC) to rehydrate the Master Mix. Any remaining eluate is transferred to the Reservoir 7. The eluate/Master Mix solution is mixed by repeated transfer between the Transfer Chamber and the DCC.

# Aliquoting and PCR

Defined aliquots $( 1 5 \mu \mathrm { L } )$ of mixed eluate/Master Mix are sequentially transferred from the Transfer Chamber to each of eight Reaction Chambers containing the specified, air dried primers and probes.

Within each Reaction Chamber, a reverse-transcription step followed by real time, multiplex PCR (“rtPCR”) is performed. Increase in fluorescence (indicative of detection of each target analyte) is detected directly within each Reaction Chamber.

The rtPCR process is conducted by two submodules of the QIAstat-Dx Analyzer 1.0: the Thermal Cycler and the qPCR Sensor.

# Components Description

QIAstat-Dx Gastrointestinal Panel 2 Cartridge:

The QIAstat-Dx Gastrointestinal Panel 2 cartridge is a disposable plastic device that allows performing fully automated molecular assays. The main features of the QIAstat-Dx Gastrointestinal Panel 2 cartridge for the Gastrointestinal assay include the ability to test liquid samples and the capacity to store all necessary reagents within the cartridge needed for such testing. All sample preparation and assay steps will be performed within the cartridge.

All the reagents required for the complete execution of the test are pre-loaded and selfcontained in the QIAstat-Dx Gastrointestinal Panel 2. The user does not need to manipulate any reagents. During the test, reagents are handled by pneumatically-operated microfluidics without any direct contact with the user or the analyzer actuators. This eliminates any possibility of exposure of the user or the analyzer to chemicals contained in the cartridge during the test and up to the disposal of used cartridges.

Reagents may be found in three different physical forms: liquid, air-dried on surfaces or lyophilized powder cake.

Within the cartridge, multiple steps are automatically performed in sequence by using pneumatic pressure and a multiport valve to transfer sample and fluids via the Transfer Chamber to their intended destinations.

# QIAstat-Dx Analyzer 1.0

The QIAstat-Dx Analyzer 1.0 is the unit that hosts a cartridge and, on command from the user, is able to run predefined assay protocols. The software specific to this test is preloaded on the QIAstat-Dx Analyzer 1.0.

# Other Materials

Each QIAstat-Dx Gastrointestinal Panel 2 cartridge will be used with a transfer pipette (provided with device). The stool sample from the patient will be collected in Para-Pak C&S or FecalSwab transport medium following the manufacturer’s instructions for use (not provided with device).

QIAstat-Dx Analyzer 1.0 – the QIAstat-Dx Gastrointestinal Panel 2 cartridge can only be run on the QIAstat-Dx Analyzer 1.0.

# Calibrators and/or Controls

Blank controls are not applicable to the device because it is a single test disposable cartridge. Negative and positive external controls are recommended by the company but not provided with the QIAstat-Dx Gastrointestinal Panel 2.

QIAGEN provides an IC within the QIAstat-Dx Gastrointestinal Panel 2 cartridge which provides a full process control covering lysis, nucleic acid purification, reverse transcription and DNA amplification. The IC is Schizosaccharomyces pombe. The IC is located in the IC cavity and is mixed with the sample during sample preparation and the eluate is mixed with the Master Mix, then aliquoted in all Reaction Chambers. The results screen displays a message indicating that the Internal Control “Passed” when the test is run successfully. An IC message of “Failed” indicates that the internal control was not amplified; ‘Positive’ test results are then reported as POSITIVE\* (positive with warning), all ‘Negative’ results are invalid.

The QIAstat-Dx Analyzer 1.0 is provided factory calibrated and does not require user calibration. The QIAstat-Dx Analyzer 1.0 includes self-check controls to verify the performance of all sensors and actuators and will alert the user in case of failure.

The RCA will provide the results to the Application Software (SW). The Application SW will store the information related to a given result in the database and will display a summary of detected analytes and the result for the IC. All POSITIVE analytes will be listed as “DETECTED PATHOGENS”. The screen will also display the complete list of all “TESTED PATHOGENS”, including positive, negative, not applicable or invalid analytes. All results for tests executed on each QIAstat-Dx Analyzer 1.0 are stored and can be reviewed in a specific archive on each QIAstat-Dx Analyzer 1.0.

# Specimen collection and transport materials

Stool samples are collected and resuspended in Para-Pak C&S or FecalSwab transport medium following the manufacturer’s instructions (Para-Pak C&S (Meridian Bioscience) or FecalSwab (COPAN)).

# Intended Use

The QIAstat- $\mathbf { D } \mathbf { x } ^ { \mathbb { \left( B \right) } }$ Gastrointestinal Panel 2 is a multiplexed nucleic acid test intended for use with the QIAstat-Dx Analyzer 1.0 for the simultaneous in vitro qualitative detection and identification of nucleic acids from multiple viruses, bacteria, and parasites directly from preserved stool samples (Para-Pak $\textsuperscript { \textregistered }$ C&S or FecalSwabTM) obtained from individuals with signs and/or symptoms of gastrointestinal infection. The following viruses, bacteria (including several diarrheagenic E.coli/Shigella pathotypes), and parasites are identified with the QIAstat- $\mathbf { \nabla \cdot D x } ^ { \mathbb { \left( B \right) } }$ Gastrointestinal Panel 2:

<table><tr><td rowspan=1 colspan=1>Pathogen</td><td rowspan=1 colspan=1>Para-Pak C&amp;S</td><td rowspan=1 colspan=1>FecalSwab</td></tr><tr><td rowspan=1 colspan=1>Adenovirus F40/F41</td><td rowspan=1 colspan=1>✓</td><td rowspan=1 colspan=1>√</td></tr><tr><td rowspan=1 colspan=1>Astrovirus</td><td rowspan=1 colspan=1>✓</td><td rowspan=1 colspan=1>√</td></tr><tr><td rowspan=1 colspan=1>Norovirus GI/GII</td><td rowspan=1 colspan=1>✓</td><td rowspan=1 colspan=1>✓</td></tr><tr><td rowspan=1 colspan=1>Rotavirus A</td><td rowspan=1 colspan=1>✓</td><td rowspan=1 colspan=1>✓</td></tr><tr><td rowspan=1 colspan=1>Campylobacter (C. jejuni, C. coli and C. upsaliensis)</td><td rowspan=1 colspan=1>✓</td><td rowspan=1 colspan=1>✓</td></tr><tr><td rowspan=1 colspan=1>Shigella/Enteroinvasive Escherichia coli (EIEC)</td><td rowspan=1 colspan=1>✓</td><td rowspan=1 colspan=1>✓</td></tr><tr><td rowspan=1 colspan=1>Enteropathogenic Escherichia coli (EPEC)</td><td rowspan=1 colspan=1>✓</td><td rowspan=1 colspan=1>Not reported</td></tr><tr><td rowspan=1 colspan=1>Enterotoxigenic Escherichia coli (ETEC) lt/st</td><td rowspan=1 colspan=1>✓</td><td rowspan=1 colspan=1>✓</td></tr><tr><td rowspan=1 colspan=1>Shiga-like toxin-producing Escherichia coli (STEC)stx1/stx2 (including specific identification of E. coli0157 serogroup within STEC)</td><td rowspan=1 colspan=1>✓</td><td rowspan=1 colspan=1>Not reported</td></tr><tr><td rowspan=1 colspan=1>Salmonella</td><td rowspan=1 colspan=1>✓</td><td rowspan=1 colspan=1>✓</td></tr><tr><td rowspan=1 colspan=1>Plesiomonas shigelloides</td><td rowspan=1 colspan=1>✓</td><td rowspan=1 colspan=1>✓</td></tr><tr><td rowspan=1 colspan=1>Yersinia enterocolitica</td><td rowspan=1 colspan=1>✓</td><td rowspan=1 colspan=1>✓</td></tr><tr><td rowspan=1 colspan=1>Cryptosporidium</td><td rowspan=1 colspan=1>✓</td><td rowspan=1 colspan=1>✓</td></tr><tr><td rowspan=1 colspan=1>Cyclospora cayetanensis</td><td rowspan=1 colspan=1>✓</td><td rowspan=1 colspan=1>✓</td></tr><tr><td rowspan=1 colspan=1>Entamoeba histolytica</td><td rowspan=1 colspan=1>✓</td><td rowspan=1 colspan=1>✓</td></tr><tr><td rowspan=1 colspan=1>Giardia lamblia*</td><td rowspan=1 colspan=1>✓</td><td rowspan=1 colspan=1>✓</td></tr></table>

\* Also known as Giardia intestinalis and Giardia duodenalis

Concomitant culture is necessary for organism recovery and further typing of bacterial agents.

The QIAstat- $\mathbf { \nabla } \cdot \mathrm { D } \mathbf { x } ^ { \circledast }$ Gastrointestinal Panel 2 is indicated as an aid in the diagnosis of specific agents of gastrointestinal illness, in conjunction with other clinical, laboratory, and epidemiological data. Positive results do not rule-out co-infection with organisms not detected by the QIAstat- $\mathbf { D } \mathbf { x } ^ { \circled { \ 8 } }$ Gastrointestinal Panel 2. The organisms detected may not be the sole or definitive cause of the disease.

Negative QIAstat- $\mathbf { \nabla } \cdot \mathrm { D } \mathbf { x } ^ { \circledast }$ Gastrointestinal Panel 2 results in the setting of clinical illness compatible with gastroenteritis may be due to infection by pathogens that are not detected by this assay test or non-infectious causes such as ulcerative colitis, irritable bowel syndrome, or Crohn’s disease.

# Comparison of the QIAstat- $\mathbf { D } \mathbf { X } ^ { \mathbf { ( B ) } }$ Gastrointestinal Panel 2 and the Predicate Device

The QIAstat-Dx Gastrointestinal Panel 2 is substantially equivalent to the predicate device:

• K140407: BioFire Diagnostics LLC FilmArray® Gastrointestinal (GI) Panel

Similarities and differences between the QIAstat-Dx Gastrointestinal Panel 2 and the predicate device are shown in Table 5.1.

Table 5.1: Comparison of the QIAstat-Dx Gastrointestinal Panel 2 with the predicate device   

<table><tr><td colspan="3" rowspan="1">device</td></tr><tr><td colspan="1" rowspan="1">Characteristic</td><td colspan="1" rowspan="1">Device</td><td colspan="1" rowspan="1">Predicate</td></tr><tr><td colspan="1" rowspan="1">Name</td><td colspan="1" rowspan="1">QIAstat-Dx Gastrointestinal Panel2</td><td colspan="1" rowspan="1">BioFire Diagnostics, LLC.FilmArray Gastrointestinal(GI) Panel</td></tr><tr><td colspan="1" rowspan="1">510(k) No.</td><td colspan="1" rowspan="1">K220062</td><td colspan="1" rowspan="1">K140407</td></tr><tr><td colspan="1" rowspan="1">Regulation</td><td colspan="1" rowspan="1">21 CFR 866.3990</td><td colspan="1" rowspan="1">21 CFR 866.3990</td></tr><tr><td colspan="1" rowspan="1">Product Code</td><td colspan="1" rowspan="1">PCH</td><td colspan="1" rowspan="1">PCH</td></tr><tr><td colspan="1" rowspan="1">Device Class</td><td colspan="1" rowspan="1">Class II</td><td colspan="1" rowspan="1">Class II</td></tr><tr><td colspan="1" rowspan="1">Similarities</td><td></td><td></td></tr><tr><td colspan="1" rowspan="1">Intended Use</td><td colspan="1" rowspan="1">The QIAstat-Dx® GastrointestinalPanel 2 is a multiplexed nucleicacid test intended for use with theQIAstat-Dx Analyzer 1.0 for thesimultaneous qualitative detectionand identification of nucleic acidsfrom multiple viruses, bacteria,and parasites directly frompreserved stool samples (Para-Pak® C&amp;S or FecalSwabTM)obtained from individuals withsigns and/or symptoms ofgastrointestinal infection. Thefollowing viruses, bacteria(including several diarrheagenicE.coli/Shigella pathotypes), andparasites are identified with theQIAstat-Dx® GastrointestinalPanel 2:Adenovirus F40/F41AstrovirusNorovirus GI/GIIRotavirus ACampylobacter (C. jejuni, C.coli and C. upsaliensis)Shigella/EnteroinvasiveEscherichia coli (EIEC)Enteropathogenic Escherichiacoli (EPEC) (only with Para-Pak C&amp;S, not reported forFecalSwab)Enterotoxigenic Escherichiacoli (ETEC) lt/st</td><td colspan="1" rowspan="1">The FilmArray®Gastrointestinal (GI) Panel is aqualitative multiplexed nucleicacid-based test intended for usewith the FilmArray®Instrument for thesimultaneous detection andidentification of nucleic acidsfrom multiple bacteria, viruses,and parasites directly fromstool samples in Cary Blairmedium, obtained fromindividuals with signs and/orsymptoms of gastrointestinalinfection. The followingbacteria (including severaldiarrheagenic E. coli/Shigellapathotypes), parasites, andviruses are identified using theFilmArray GI Panel:Campylobacter (C.jejuni/C. coli/C. upsaliensisClostridium difficile (C.difficile) toxin A/BPlesiomonas shigelloidesSalmonellaVibrio (V&gt;parahaemolyticus/V.vulnificus/ V. cholerae)including specificidentification of VibriocholeraeYersinia enterocolitica−</td></tr><tr><td colspan="3"></td></tr><tr><td>acteristic Device</td><td></td><td>Predicate Enteroaggregative</td></tr><tr><td></td><td>Shiga-like toxin-producing • Escherichia coli (STEC) stx1/stx2 (including specific identification of E. coli O157 serogroup within STEC) (only with Para-Pak C&amp;S, not reported for FecalSwab) Salmonella Plesiomonas shigelloides Yersinia enterocolitica Cryptosporidium Cyclospora cayetanensis Entamoeba histolytica Giardia lamblia (Also known as Giardia intestinalis and Giardia duodenalis) Concomitant culture is necessary for organism recovery and further typing of bacterial agents. The QIAstat-Dx® Gastrointestinal Panel 2 is indicated as an aid in the diagnosis of specific agents of gastrointestinal illness in conjunction with other clinical, laboratory, and epidemiological data. Positive results do not rule- out co-infection with organisms not detected by the QIAstat-Dx® Gastrointestinal Panel 2. The organisms detected may not be the sole or definitive cause of the disease. Negative QIAstat-Dx® Gastrointestinal Panel 2 results in the setting of clinical illness compatible with gastroenteritis may be due to infection by pathogens that are not detected by</td><td>• Escherichia coli (EAEC) Enteropathogenic Escherichia coli (EPEC) Enterotoxigenic Escherichia coli (ETEC) lt/st Shiga-like toxin-producing Escherichia coli (STEC) stx1/stx2 (including specific identification of the E. coli 0157 serogroup within STEC) Shigella/Enteroinvasive Escherichia coli (EIEC) Cryptosporidium Cyclospora cayetanensis Entamoeba histolytica Giardia lamblia (also known as G. intestinalis and G. duodenalis) Adenovirus F 40/41 Astrovirus Norovirus GI/GII Rotavirus A Sapovirus (Genogroups I, II, IV and V) The FilmArray Gastrointestinal Panel is indicated as an aid in the diagnosis of specific agents of gastrointestinal illness and results are meant to be used in conjunction with other clinical, laboratory, and epidemiological data. Positive results do not rule out co-infection with organisms not included in the FilmArray® GI Panel. The</td></tr><tr><td>Characteristic</td><td>Device irritable bowel syndrome, or</td><td>Predicate Concomitant culture is</td></tr><tr><td rowspan="3"></td><td>Crohn's disease.</td><td>necessary for organism recovery and further typing of bacterial agents. This device is not intended to monitor or guide treatment for C. difficile infection.</td></tr><tr><td></td><td>Due to the small number of positive specimens collected for certain organisms during the prospective clinical study, performance characteristics for E. coli O157, Plesiomonas shigelloides, Yersinia</td></tr><tr><td></td><td>enterocolitica, Astrovirus, and Rotavirus A were established primarily with retrospective clinical specimens. Performance characteristics for Entamoeba histolytica, and Vibrio (V parahaemolyticus, V. vulnificus, and Vibrio cholerae) were established primarily using contrived clinical specimens. Negative FilmArray GI Panel</td></tr><tr><td colspan="1" rowspan="1">Characteristic</td><td colspan="1" rowspan="1">Device</td><td colspan="1" rowspan="1">Predicate</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">identification of acutegastroenteritis in the context ofoutbreaks.</td></tr><tr><td colspan="1" rowspan="1">Specimen Type</td><td colspan="1" rowspan="1">Preserved stool in Para-Pak C&amp;Sor FecalSwab transport media</td><td colspan="1" rowspan="1">Stool in Cary-Blair Medium</td></tr><tr><td colspan="1" rowspan="1">Analyte Detected</td><td colspan="1" rowspan="1">RNA/DNA</td><td colspan="1" rowspan="1">RNA/DNA</td></tr><tr><td colspan="1" rowspan="1">Organism Detected</td><td colspan="1" rowspan="1">See above</td><td colspan="1" rowspan="1">See above</td></tr><tr><td colspan="1" rowspan="1">Amplification andDetectionTechnology</td><td colspan="1" rowspan="1">PCR</td><td colspan="1" rowspan="1">PCR</td></tr><tr><td colspan="1" rowspan="1">Assay Controls</td><td colspan="1" rowspan="1">One internal control in eachcartridge to control for sampleprocessing that is subjected to allnucleic acid extraction andamplification steps similar topatient samples. Labelingrecommends use of negative andpositive external controlsregularly. Use transport media asthe external Negative Control,and previously characterizedpositive samples or negativesample spiked with wellcharacterized target organisms asexternal positive controls.</td><td colspan="1" rowspan="1">Two controls are included ineach reagent pouch to controlfor sample processing and bothstages of PCR and meltanalysis. Labelingrecommends the use ofexternal positive and negativecontrols regularly. Enterictransport media can be used asan external negative control,and previously characterizedpositive samples or negativesamples spiked with wellcharacterized organisms asexternal positive controls.</td></tr><tr><td colspan="3" rowspan="1">Differences</td></tr><tr><td colspan="1" rowspan="1">Assay Targets</td><td colspan="1" rowspan="1">No additional targets to thoselisted above which are incommon with the predicatedevice.</td><td colspan="1" rowspan="1">The FilmArray®Gastrointestinal Panel has fouradditional targets: a genericVibrio target whichdifferentiates Vibrio cholerae ,Clostridium difficile (C.difficile) toxin A/B,Enteroaggregative Escherichiacoli (EAEC) and Sapovirus.</td></tr><tr><td colspan="1" rowspan="1">Nucleic AcidExtraction</td><td colspan="1" rowspan="1">Extraction of nucleic acids usingsilica membrane</td><td colspan="1" rowspan="1">Extraction of nucleic acidsusing magnetic beads</td></tr><tr><td colspan="1" rowspan="1">Technology</td><td colspan="1" rowspan="1">QIAstat-Dx GastrointestinalPanel 2 detection of amplifiedtargets uses an increase influorescence due to specific probebinding to generate the assayresults.</td><td colspan="1" rowspan="1">The FilmArray®Gastrointestinal Panel usesmelting curve analysis toconfirm the identity ofamplified targets to generateassay results</td></tr><tr><td colspan="1" rowspan="1">Operational</td><td colspan="1" rowspan="1">The sample is loaded straight intothe cartridge.</td><td colspan="1" rowspan="1">The FilmArray®Gastrointestinal Panel has torehydrate the cartridge beforeuse</td></tr><tr><td colspan="1" rowspan="1">Amplification andDetection InstrumentSystem</td><td colspan="1" rowspan="1">QIAstat-Dx Analyzer 1.0</td><td colspan="1" rowspan="1">FilmArray Instrument</td></tr></table>

# Performance Characteristics – Non-Clinical Studies

# Sensitivity (Limit of Detection)

The Analytical Sensitivity, or Limit of Detection (LoD), is defined as the lowest concentration at which $2 9 5 \%$ of the tested samples generate a positive call.

The LoD for each of the QIAstat-Dx Gastrointestinal Panel 2 target pathogenic organisms was assessed, using in total 36 pathogens strains, by analyzing serial dilutions of analytical samples prepared from culture isolates from commercial suppliers (e.g., ZeptoMetrix® and $\mathbf { A T C C ^ { \mathbb { ( B ) } } } )$ or clinical samples positive for target analytes commercially unavailable. Each sample tested was prepared in human stool matrix, which consists of a pool of previously tested negative clinical stool specimens resuspended in Para-Pak C&S (Meridian Biosciences) transport medium following the manufacturer’s instructions collection device. A matrix equivalency study between Para-Pak C&S and FecalSwab transport media was conducted to support the conclusions in the section, except for EPEC, STEC stx1/stx2 and STEC O157 that are only applicable for Para-Pak C&S resuspended samples.

Individual LoD values for each QIAstat-Dx Gastrointestinal Panel 2 target is shown in Table 5.2.

Table 5.2: LoD concentration details for the 36 pathogen strains tested with QIAstat-Dx Gastrointestinal Panel 2   

<table><tr><td colspan="1" rowspan="1">QIAstat-DxGastrointestinalPanel 2 Target</td><td colspan="1" rowspan="1">PathogenStrain</td><td colspan="1" rowspan="1">Supplier &amp;Catalog ID</td><td colspan="1" rowspan="1">Dilutionfromthestock</td><td colspan="1" rowspan="1">DetectionRate</td><td colspan="1" rowspan="1">Concentration(molecularunits) #</td><td colspan="1" rowspan="1">Concentration(microbiologicalunits)</td></tr><tr><td colspan="1" rowspan="1">Campylobacter</td><td colspan="1" rowspan="1">Campylobacter coli 76-GA2[LMG 21266]</td><td colspan="1" rowspan="1">ATCC43478</td><td colspan="1" rowspan="1">1.00E-06</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">5802copies/mL</td><td colspan="1" rowspan="1">1.2 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">QIAstat-DxGastrointestinalPanel 2 Target</td><td colspan="1" rowspan="1">PathogenStrain</td><td colspan="1" rowspan="1">Supplier &amp;Catalog ID</td><td colspan="1" rowspan="1">Dilutionfromthestock</td><td colspan="1" rowspan="1">DetectionRate</td><td colspan="1" rowspan="1">Concentration(molecularunits) #</td><td colspan="1" rowspan="1">Concentration(microbiologicalunits</td></tr><tr><td colspan="1" rowspan="5">′</td><td colspan="1" rowspan="1">Campylobacter coli CIP7080</td><td colspan="1" rowspan="1">ATCC33559</td><td colspan="1" rowspan="1">1.00E-05</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">8941copies/mL</td><td colspan="1" rowspan="1">0.6 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Campylobacter jejuni Z086</td><td colspan="1" rowspan="1">ZeptoMetrix0801650</td><td colspan="1" rowspan="1">1.00E-06</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">14491copies/mL</td><td colspan="1" rowspan="1">1660 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Campylobacter jejuni subsp.JejuniRM3193</td><td colspan="1" rowspan="1">ATCCBAA-1234</td><td colspan="1" rowspan="1">1.00E-06</td><td colspan="1" rowspan="1">19/20</td><td colspan="1" rowspan="1">7210copies/mL</td><td colspan="1" rowspan="1">110 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Campylobacter upsaliensisNCTC 11541</td><td colspan="1" rowspan="1">ZeptoMetrix0801999</td><td colspan="1" rowspan="1">3.16E-06</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">56165copies/mL</td><td colspan="1" rowspan="1">2259.4 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Campylobacter upsaliensisRM3195</td><td colspan="1" rowspan="1">ATCCBAA-1059</td><td colspan="1" rowspan="1">1.00E-06</td><td colspan="1" rowspan="1">19/20</td><td colspan="1" rowspan="1">7631copies/mL</td><td colspan="1" rowspan="1">35 CFU/mL</td></tr><tr><td colspan="1" rowspan="2">Plesiomonasshigelloides</td><td colspan="1" rowspan="1">Bader</td><td colspan="1" rowspan="1">ATCC14029</td><td colspan="1" rowspan="1">3.16E-07</td><td colspan="1" rowspan="1">19/20</td><td colspan="1" rowspan="1">116copies/mL</td><td colspan="1" rowspan="1">2.7 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Z130</td><td colspan="1" rowspan="1">ZeptoMetrix0801899</td><td colspan="1" rowspan="1">3.16E-07</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">481copies/mL</td><td colspan="1" rowspan="1">2291 CFU/mL</td></tr><tr><td colspan="1" rowspan="2">Salmonella</td><td colspan="1" rowspan="1">SalmonellaentericaSerovarTyphimuriumZ005</td><td colspan="1" rowspan="1">ZeptoMetrix0801437</td><td colspan="1" rowspan="1">3.16E-07</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">1441copies/mL</td><td colspan="1" rowspan="1">4518.8 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">SalmonellaentericaSerovarcholeraseus</td><td colspan="1" rowspan="1">ATCC13312</td><td colspan="1" rowspan="1">3.16E-07</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">647copies/mL</td><td colspan="1" rowspan="1">91.6 CFU/mL</td></tr><tr><td colspan="1" rowspan="2">Yersiniaenterocolitica</td><td colspan="1" rowspan="1">subsp.enterocoliticaNTCC 11175,Biotype 4,serotype 3</td><td colspan="1" rowspan="1">ATCC700822</td><td colspan="1" rowspan="1">3.16E-07</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">2496copies/mL</td><td colspan="1" rowspan="1">120.1 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Z036</td><td colspan="1" rowspan="1">ZeptoMetrix0801734</td><td colspan="1" rowspan="1">1.00E-07</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">719copies/mL</td><td colspan="1" rowspan="1">2070 CFU/mL</td></tr><tr><td colspan="1" rowspan="2">EnteropathogenicE. coli (EPEC)*</td><td colspan="1" rowspan="1">Escherichiacoli O0111:NM(EPEC)</td><td colspan="1" rowspan="1">ZeptoMetrix8801747</td><td colspan="1" rowspan="1">3.16E-07</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">1817copies/mL</td><td colspan="1" rowspan="1">2581.7 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Escherichiacoli 7.1493;EPEC;O8428</td><td colspan="1" rowspan="1">ZeptoMetrix0801938</td><td colspan="1" rowspan="1">1.00E-06</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">29021copies/mL</td><td colspan="1" rowspan="1">1190 CFU/mL</td></tr><tr><td colspan="1" rowspan="2">EnterotoxigenicE. coli (ETEC)lt/st</td><td colspan="1" rowspan="1">Escherichiacoli ETEC;ST+, LT+</td><td colspan="1" rowspan="1">ZeptoMetrix0801624</td><td colspan="1" rowspan="1">1.00E-07</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">855copies/mL</td><td colspan="1" rowspan="1">567 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Escherichiacoli H10407,O78:H11</td><td colspan="1" rowspan="1">ATCC35401</td><td colspan="1" rowspan="1">3.16E-08</td><td colspan="1" rowspan="1">19/20</td><td colspan="1" rowspan="1">367copies/mL</td><td colspan="1" rowspan="1">10.1 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">EY</td><td colspan="1" rowspan="1">EY</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td></td><td colspan="1" rowspan="1"></td><td></td></tr><tr><td colspan="1" rowspan="1">QIAstat-DxGastrointestinalPanel 2 Target</td><td colspan="1" rowspan="1">PathogenStrain</td><td colspan="1" rowspan="1">Supplier &amp;Catalog ID</td><td colspan="1" rowspan="1">Dilutionfromthestock</td><td colspan="1" rowspan="1">DetectionRate</td><td colspan="1" rowspan="1">Concentration(molecularunits) #</td><td colspan="1" rowspan="1">Concentration(microbiologicalunts)</td></tr><tr><td colspan="1" rowspan="2">ShigellalEnteroinvasiveE. coli (EIEC)</td><td colspan="1" rowspan="1">Escherichiacoli CDC EDL1282,O29:NM</td><td colspan="1" rowspan="1">ATCC43892</td><td colspan="1" rowspan="1">1.00E-08</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">1431copies/mL</td><td colspan="1" rowspan="1">41.3 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Shigellasonnei NCDC1120-66</td><td colspan="1" rowspan="1">ATCC25931</td><td colspan="1" rowspan="1">3.16E-08</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">488copies/mL</td><td colspan="1" rowspan="1">0.2 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Shiga-like toxin-producingE.coli (STEC)stx1/stx2*</td><td colspan="1" rowspan="1">Escherichiacoli O26:H4</td><td colspan="1" rowspan="1">ZeptoMetrix8801748</td><td colspan="1" rowspan="1">3.16E-07</td><td colspan="1" rowspan="1">STECstx1/stx2:20/20</td><td colspan="1" rowspan="1">2012copies/mL</td><td colspan="1" rowspan="1">726.8 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Shiga-like toxin-producing E.coli (STEC)0157*</td><td colspan="1" rowspan="1">Escherichiacoli 0157:H7;EDL933</td><td colspan="1" rowspan="1">ZeptoMetrix0801622</td><td colspan="1" rowspan="1">3.16E-07</td><td colspan="1" rowspan="1">STECstx1/stx2:19/200157:19/20</td><td colspan="1" rowspan="1">1217copies/mL</td><td colspan="1" rowspan="1">2281.5 CFU/mL</td></tr><tr><td colspan="1" rowspan="2">Cryptosporidium</td><td colspan="1" rowspan="1">Cryptosporidium parvum Iowa isolate</td><td colspan="1" rowspan="1">Waterborne P102C</td><td colspan="1" rowspan="1">1.00E-05</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">661copies/mL</td><td colspan="1" rowspan="1">n/a</td></tr><tr><td colspan="1" rowspan="1">Cryptosporidium hominis</td><td colspan="1" rowspan="1">PublicHealthWales UKM84</td><td colspan="1" rowspan="1">3.16E-03</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">357copies/mL</td><td colspan="1" rowspan="1">n/a</td></tr><tr><td colspan="1" rowspan="2">Cyclosporacayetanensis</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">LACNY-ClinicalsampleLAC2825</td><td colspan="1" rowspan="1">1.00E-04</td><td colspan="1" rowspan="1">19/20</td><td colspan="1" rowspan="1">53 copies/mL</td><td colspan="1" rowspan="1">n/a</td></tr><tr><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">LACNYClinicalsampleLAC2827</td><td colspan="1" rowspan="1">3.16E-04</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">137copies/mL</td><td colspan="1" rowspan="1">n/a</td></tr><tr><td colspan="1" rowspan="2">Entamoebahistolytica</td><td colspan="1" rowspan="1">HM-1:IMSS(Mexico City1967)</td><td colspan="1" rowspan="1">ATCC30459</td><td colspan="1" rowspan="1">1.00E-06</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">7 copies/mL</td><td colspan="1" rowspan="1">0.2 cells/mL</td></tr><tr><td colspan="1" rowspan="1">HK-9 (Korea)</td><td colspan="1" rowspan="1">ATCC30015</td><td colspan="1" rowspan="1">1.00E-07</td><td colspan="1" rowspan="1">19/20</td><td colspan="1" rowspan="1">1 copy/mL</td><td colspan="1" rowspan="1">0.13 cells/mL</td></tr><tr><td colspan="1" rowspan="2">Giardia lamblia</td><td colspan="1" rowspan="1">WB(Bethesda)</td><td colspan="1" rowspan="1">ATCC30957</td><td colspan="1" rowspan="1">3.16E-05</td><td colspan="1" rowspan="1">19/20</td><td colspan="1" rowspan="1">11850copies/mL</td><td colspan="1" rowspan="1">790 cells/mL</td></tr><tr><td colspan="1" rowspan="1">Portland-1</td><td colspan="1" rowspan="1">ATCC30888</td><td colspan="1" rowspan="1">1.00E-04</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">14500copies/mL</td><td colspan="1" rowspan="1">635 cells/mL</td></tr><tr><td colspan="1" rowspan="2">AdenovirusF40/F41</td><td colspan="1" rowspan="1">Type 40(Dugan)</td><td colspan="1" rowspan="1">ZeptoMetrix0810084CF</td><td colspan="1" rowspan="1">1.00E-06</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">11726copies/mL</td><td colspan="1" rowspan="1">0.1 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Type 41 (Tak)</td><td colspan="1" rowspan="1">ZeptoMetrix0810085CF</td><td colspan="1" rowspan="1">1.00E-07</td><td colspan="1" rowspan="1">19/20</td><td colspan="1" rowspan="1">979copies/mL</td><td colspan="1" rowspan="1">0.05 TCID50/mL</td></tr><tr><td colspan="1" rowspan="2">Astrovirus</td><td colspan="1" rowspan="1">ERE IID 2371(type 8)</td><td colspan="1" rowspan="1">Zeptometrix0810277CF</td><td colspan="1" rowspan="1">1.00E-04</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">11586371copies/mL</td><td colspan="1" rowspan="1">11.7 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">ERE IID 2868(type 4)</td><td colspan="1" rowspan="1">Zeptometrix0810276CF</td><td colspan="1" rowspan="1">3.16E-06</td><td colspan="1" rowspan="1">19/20</td><td colspan="1" rowspan="1">52184copies/mL</td><td colspan="1" rowspan="1">1.3 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">QIAstat-DxGastrointestinalPanel 2 Target</td><td colspan="1" rowspan="1">PathogenStrain</td><td colspan="1" rowspan="1">Supplier &amp;Catalog ID</td><td colspan="1" rowspan="1">Dilutionfromthestock</td><td colspan="1" rowspan="1">DetectionRate</td><td colspan="1" rowspan="1">Concentration(molecularunits) #</td><td colspan="1" rowspan="1">Concentration(microbiologicalunits)</td></tr><tr><td colspan="1" rowspan="2">NorovirusGI/GII</td><td colspan="1" rowspan="1">GI.1(recombinant)</td><td colspan="1" rowspan="1">ZeptoMetrix0810086CF</td><td colspan="1" rowspan="1">3.16E-05</td><td colspan="1" rowspan="1">19/20</td><td colspan="1" rowspan="1">24629copies/mL</td><td colspan="1" rowspan="1">891.1 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">GII.4(recombinant)</td><td colspan="1" rowspan="1">ZeptoMetrix0810087CF</td><td colspan="1" rowspan="1">1.00E-05</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">8998copies/mL</td><td colspan="1" rowspan="1">10.5 TCID50/mL</td></tr><tr><td colspan="1" rowspan="2">Rotavirus A</td><td colspan="1" rowspan="1">Wa</td><td colspan="1" rowspan="1">ZeptoMetrix0810041CF</td><td colspan="1" rowspan="1">1.00E-04</td><td colspan="1" rowspan="1">19/20</td><td colspan="1" rowspan="1">5201copies/mL</td><td colspan="1" rowspan="1">14.1 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">69M</td><td colspan="1" rowspan="1">ZeptoMetrix0810280CF</td><td colspan="1" rowspan="1">3.16E-05</td><td colspan="1" rowspan="1">19/20</td><td colspan="1" rowspan="1">5787copies/mL</td><td colspan="1" rowspan="1">436.1 TCID50/mL</td></tr></table>

# Molecular unit titers were determined using in-house developed and validated qPCR assays. \* Target applicable for Para-Pak C&S samples only.

# Analytical Reactivity (Inclusivity)

Analytical Reactivity (Inclusivity) was evaluated with gastrointestinal pathogen isolates/strains that were selected based on clinical relevance and genetic, temporal and geographical diversity. Based on in vitro (wet) testing and in silico analysis, the QIAstatDx Gastrointestinal Panel 2 primers and probes are specific and inclusive for clinically prevalent and relevant strains for each pathogen tested.

# In vitro testing

QIAstat-Dx Gastrointestinal Panel 2 is inclusive for $100 \%$ (114 out of 114) of the pathogen strains tested in vitro. Most pathogen strains evaluated in laboratory testing were detected at $\leq 3$ -fold (104/114) of the corresponding LoD reference strain which is written in bold (refer to Table 5.3 to Error! Reference source not found.). Less than $100 \%$ detection was observed for one strain each of ETEC, EIEC/Shigella and Rotavirus and two strains each of STEC (one STEC O157), Adenovirus and Norovirus at 3x LoD. Testing of these strains at 10x LoD generated the expected positive resulted for all replicates.

Table 5.3: Inclusivity test results for Campylobacter strains \* Strain tested during LoD verification study

<table><tr><td rowspan=1 colspan=1>QIAstat-DxGastrointestinalPanel 2 Target</td><td rowspan=1 colspan=1>Pathogen</td><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>Supplier</td><td rowspan=1 colspan=1>CatalogID</td><td rowspan=1 colspan=1>x-foldLoDdetected</td></tr><tr><td rowspan=11 colspan=1>Campylobacter</td><td rowspan=1 colspan=1>Campylobacter coli</td><td rowspan=1 colspan=1>76-GA2[LMG21266]</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>43478*</td><td rowspan=1 colspan=1>1xLoD</td></tr><tr><td rowspan=1 colspan=1>Campylobacter coli</td><td rowspan=1 colspan=1>Z293</td><td rowspan=1 colspan=1>ZeptoMetrix</td><td rowspan=1 colspan=1>804272</td><td rowspan=1 colspan=1>1xLoD</td></tr><tr><td rowspan=1 colspan=1>Campylobacter coli</td><td rowspan=1 colspan=1>CIP 7080[1407, CIP70.80]</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>33559*</td><td rowspan=1 colspan=1>3xLoD</td></tr><tr><td rowspan=1 colspan=1>Campylobacter jejuni</td><td rowspan=1 colspan=1>Z086</td><td rowspan=1 colspan=1>ZeptoMetrix</td><td rowspan=1 colspan=1>0801650*</td><td rowspan=1 colspan=1>1xLoD</td></tr><tr><td rowspan=1 colspan=1>Campylobacter jejuni</td><td rowspan=1 colspan=1>subsp. jejuniRM3193</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>BAA-1234*</td><td rowspan=1 colspan=1>0.1xLoD</td></tr><tr><td rowspan=1 colspan=1>Campylobacter jejunisubsp. jejuni</td><td rowspan=1 colspan=1>0:19 HL7;D3180</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>BAA-218</td><td rowspan=1 colspan=1>0.1xLoD</td></tr><tr><td rowspan=1 colspan=1>Campylobacter jejunisubsp. jejuni</td><td rowspan=1 colspan=1>AS-83-79</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>33291</td><td rowspan=1 colspan=1>0.1xLoD</td></tr><tr><td rowspan=1 colspan=1>Campylobacter jejunisubsp. doylei</td><td rowspan=1 colspan=1>NCTC11951</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>49349</td><td rowspan=1 colspan=1>0.1xLoD</td></tr><tr><td rowspan=1 colspan=1>Campylobacterupsaliensis</td><td rowspan=1 colspan=1>NCTC11541</td><td rowspan=1 colspan=1>ZeptoMetrix</td><td rowspan=1 colspan=1>0801999*</td><td rowspan=1 colspan=1>1xLoD</td></tr><tr><td rowspan=1 colspan=1>Campylobacterupsaliensis</td><td rowspan=1 colspan=1>RM 3195(1994)</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>BAA-1059*</td><td rowspan=1 colspan=1>0.3xLoD</td></tr><tr><td rowspan=1 colspan=1>Campylobacterupsaliensis</td><td rowspan=1 colspan=1>NCTC11541[C231]</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>43954</td><td rowspan=1 colspan=1>1xLoD</td></tr></table>

\* Strain tested during LoD verification study

Table 5.4: Inclusivity test results for Plesiomonas shigelloides strains   

<table><tr><td rowspan=1 colspan=1>QIAstat-DxGastrointestinalPanel 2 Target</td><td rowspan=1 colspan=1>Pathogen</td><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>Supplier</td><td rowspan=1 colspan=1>CatalogID</td><td rowspan=1 colspan=1>x-foldLoDdetected</td></tr><tr><td rowspan=3 colspan=1>Plesiomonasshigelloides</td><td rowspan=1 colspan=1>Plesiomonasshigelloides</td><td rowspan=1 colspan=1>Z130</td><td rowspan=1 colspan=1>ZeptoMetrix</td><td rowspan=1 colspan=1>0801899*</td><td rowspan=1 colspan=1>1xLoD</td></tr><tr><td rowspan=1 colspan=1>Plesiomonasshigelloides</td><td rowspan=1 colspan=1>GNI 14</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>51903</td><td rowspan=1 colspan=1>1xLoD</td></tr><tr><td rowspan=1 colspan=1>Plesiomonasshigelloides</td><td rowspan=1 colspan=1>CDC 3085-55 [BaderM51, NCIB9242, NCTC10360, RH798]</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>14029*</td><td rowspan=1 colspan=1>0.3xLoD</td></tr></table>

Table 5.5: Inclusivity test results for Salmonella strains   

<table><tr><td colspan="1" rowspan="1">QIAstat-DxGastrointestinalPanel 2 Target</td><td colspan="1" rowspan="1">Pathogen</td><td colspan="1" rowspan="1">Strain</td><td colspan="1" rowspan="1">Supplier</td><td colspan="1" rowspan="1">CatalogID</td><td colspan="1" rowspan="1">x-foldLoDdetected</td></tr><tr><td colspan="1" rowspan="9">Salmonella</td><td colspan="1" rowspan="1">Salmonella enterica</td><td colspan="1" rowspan="1">SerovarTyphimurium Z005</td><td colspan="1" rowspan="1">ZeptoMetrix</td><td colspan="1" rowspan="1">0801437*</td><td colspan="1" rowspan="1">1xLoD</td></tr><tr><td colspan="1" rowspan="1">Salmonella enterica</td><td colspan="1" rowspan="1">Subsp.Enterica,serovarBareilly</td><td colspan="1" rowspan="1">NCTC</td><td colspan="1" rowspan="1">NC05745</td><td colspan="1" rowspan="1">1xLoD</td></tr><tr><td colspan="1" rowspan="1">Salmonella enterica</td><td colspan="1" rowspan="1">Subsp.Enterica,serovartyphi, Z152</td><td colspan="1" rowspan="1">ZeptoMetrix</td><td colspan="1" rowspan="1">0801933</td><td colspan="1" rowspan="1">0.1xLoD</td></tr><tr><td colspan="1" rowspan="1">Salmonella enterica</td><td colspan="1" rowspan="1">Subsp.Enterica,serovarEnteridis,CDC K-1891[ATCC25928]</td><td colspan="1" rowspan="1">ATCC</td><td colspan="1" rowspan="1">13076</td><td colspan="1" rowspan="1">0.1xLoD</td></tr><tr><td colspan="1" rowspan="1">Salmonella enterica</td><td colspan="1" rowspan="1">Subsp.Enterica,serovarInfantis,MZ1479[SARB27]</td><td colspan="1" rowspan="1">ATCC</td><td colspan="1" rowspan="1">BAA-1675</td><td colspan="1" rowspan="1">0.1xLoD</td></tr><tr><td colspan="1" rowspan="1">Salmonella enterica</td><td colspan="1" rowspan="1">Subsp.Enterica,serovarMontevideo,G4639</td><td colspan="1" rowspan="1">ATCC</td><td colspan="1" rowspan="1">BAA-710</td><td colspan="1" rowspan="1">0.1xLoD</td></tr><tr><td colspan="1" rowspan="1">Salmonella enterica</td><td colspan="1" rowspan="1">Subsp.Enterica,serovarJaviana</td><td colspan="1" rowspan="1">NCTC</td><td colspan="1" rowspan="1">NC06495</td><td colspan="1" rowspan="1">0.1xLoD</td></tr><tr><td colspan="1" rowspan="1">Salmonella enterica</td><td colspan="1" rowspan="1">Subsp.Enterica,serovarThompson</td><td colspan="1" rowspan="1">NCTC</td><td colspan="1" rowspan="1">NC08496</td><td colspan="1" rowspan="1">0.1xLoD</td></tr><tr><td colspan="1" rowspan="1">Salmonella enterica</td><td colspan="1" rowspan="1">Subsp.Enterica,</td><td colspan="1" rowspan="1">ATCC</td><td colspan="1" rowspan="1">9712</td><td colspan="1" rowspan="1">0.1xLoD</td></tr><tr><td colspan="1" rowspan="10"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">serovarSaintpaul</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Salmonella enterica</td><td colspan="1" rowspan="1">Subsp.Enterica,serovar Berta</td><td colspan="1" rowspan="1">NCTC</td><td colspan="1" rowspan="1">NC05770</td><td colspan="1" rowspan="1">0.1xLoD</td></tr><tr><td colspan="1" rowspan="1">Salmonella enterica</td><td colspan="1" rowspan="1">Subps.Salame, IINCTC10310[JT945,SS140/61]</td><td colspan="1" rowspan="1">ATCC</td><td colspan="1" rowspan="1">700151</td><td colspan="1" rowspan="1">0.1xLoD</td></tr><tr><td colspan="1" rowspan="1">Salmonella enterica</td><td colspan="1" rowspan="1">Subps.diarizonaeIIIb, 62</td><td colspan="1" rowspan="1">ATCC</td><td colspan="1" rowspan="1">29934</td><td colspan="1" rowspan="1">0.1xLoD</td></tr><tr><td colspan="1" rowspan="1">Salmonella enterica</td><td colspan="1" rowspan="1">Subps.houtenae IV,CIP 82.32[264.66]</td><td colspan="1" rowspan="1">ATCC</td><td colspan="1" rowspan="1">43974</td><td colspan="1" rowspan="1">0.1xLoD</td></tr><tr><td colspan="1" rowspan="1">Salmonella enterica</td><td colspan="1" rowspan="1">Subps.Indica VI,CIP 102501F.Kauffmann1240]</td><td colspan="1" rowspan="1">ATCC</td><td colspan="1" rowspan="1">43976</td><td colspan="1" rowspan="1">0.1xLoD</td></tr><tr><td colspan="1" rowspan="1">Salmonella enterica</td><td colspan="1" rowspan="1">Subsp.Enterica,serovarAgona, CDC873 [CDC1111-61]</td><td colspan="1" rowspan="1">ATCC</td><td colspan="1" rowspan="1">51957</td><td colspan="1" rowspan="1">0.1xLoD</td></tr><tr><td colspan="1" rowspan="1">Salmonella enterica</td><td colspan="1" rowspan="1">Subsp.Enterica,serovarMuenchen,54</td><td colspan="1" rowspan="1">ATCC</td><td colspan="1" rowspan="1">8388</td><td colspan="1" rowspan="1">0.1xLoD</td></tr><tr><td colspan="1" rowspan="1">Salmonella enterica</td><td colspan="1" rowspan="1">Subsp.Enterica,serovarOranienburg,E1093</td><td colspan="1" rowspan="1">ATCC</td><td colspan="1" rowspan="1">9239</td><td colspan="1" rowspan="1">0.1xLoD</td></tr><tr><td colspan="1" rowspan="1">Salmonella enterica</td><td colspan="1" rowspan="1">Subsp.Enterica,</td><td colspan="1" rowspan="1">ATCC</td><td colspan="1" rowspan="1">51962</td><td colspan="1" rowspan="1">0.1xLoD</td></tr><tr><td colspan="1" rowspan="10"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">serovarParatyphi Bvar. Java,CDC 5</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1">−</td></tr><tr><td colspan="1" rowspan="1">Salmonella bongori</td><td colspan="1" rowspan="1">CIP 82.33[1224.72]</td><td colspan="1" rowspan="1">ATCC</td><td colspan="1" rowspan="1">43975</td><td colspan="1" rowspan="1">0.3xLoD</td></tr><tr><td colspan="1" rowspan="1">Salmonella enterica</td><td colspan="1" rowspan="1">Subsp.Enterica,serovarCholeraesius,NCTC 5735[1348, K.34]</td><td colspan="1" rowspan="1">ATCC</td><td colspan="1" rowspan="1">13312*</td><td colspan="1" rowspan="1">0.3xLoD</td></tr><tr><td colspan="1" rowspan="1">Salmonella enterica</td><td colspan="1" rowspan="1">Subsp.Enterica,serovarNewport,C487-69</td><td colspan="1" rowspan="1">ATCC</td><td colspan="1" rowspan="1">27869</td><td colspan="1" rowspan="1">0.3xLoD</td></tr><tr><td colspan="1" rowspan="1">Salmonella enterica</td><td colspan="1" rowspan="1">Subsp.Enterica, 4,5, 12:7:-,serovarTyphimurium</td><td colspan="1" rowspan="1">NCTC</td><td colspan="1" rowspan="1">NC13952</td><td colspan="1" rowspan="1">0.3xLoD</td></tr><tr><td colspan="1" rowspan="1">Salmonella enterica</td><td colspan="1" rowspan="1">Subsp.Enterica,serovarBraenderup</td><td colspan="1" rowspan="1">ATCC</td><td colspan="1" rowspan="1">700136</td><td colspan="1" rowspan="1">0.3xLoD</td></tr><tr><td colspan="1" rowspan="1">Salmonella enterica</td><td colspan="1" rowspan="1">Subsp.Enterica,serovarAnatum</td><td colspan="1" rowspan="1">NCTC</td><td colspan="1" rowspan="1">NC05779</td><td colspan="1" rowspan="1">0.3xLoD</td></tr><tr><td colspan="1" rowspan="1">Salmonella enterica</td><td colspan="1" rowspan="1">Subps.arizonae IIIa,NCTC 7311[CDAI 426]</td><td colspan="1" rowspan="1">ATCC</td><td colspan="1" rowspan="1">700156</td><td colspan="1" rowspan="1">0.3xLoD</td></tr><tr><td colspan="1" rowspan="1">Salmonella enterica</td><td colspan="1" rowspan="1">Subsp.Enterica,serovarHeidelberg,[16]</td><td colspan="1" rowspan="1">ATCC</td><td colspan="1" rowspan="1">8326</td><td colspan="1" rowspan="1">0.3xLoD</td></tr><tr><td colspan="1" rowspan="1">Salmonella enterica</td><td colspan="1" rowspan="1">Subsp.Enterica,</td><td colspan="1" rowspan="1">ATCC</td><td colspan="1" rowspan="1">BAA-2739</td><td colspan="1" rowspan="1">0.3xLoD</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">serovarMississippi,CDC 2012K-0487</td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1">−</td></tr></table>

\* Strain tested during LoD verification study

Table 5.6: Inclusivity test results for Yersinia enterocolitica strains   

<table><tr><td rowspan=1 colspan=1>QIAstat-DxGastrointestinalPanel 2 Target</td><td rowspan=1 colspan=1>Pathogen</td><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>Supplier</td><td rowspan=1 colspan=1>Catalog ID</td><td rowspan=1 colspan=1>x-foldLoDdetected</td></tr><tr><td rowspan=4 colspan=1>Yersiniaenterocolitica</td><td rowspan=1 colspan=1>Yersiniaenterocolitica</td><td rowspan=1 colspan=1>Z036</td><td rowspan=1 colspan=1>ZeptoMetrix</td><td rowspan=1 colspan=1>0801734*</td><td rowspan=1 colspan=1>1x LoD</td></tr><tr><td rowspan=1 colspan=1>Yersiniaenterocolitica</td><td rowspan=1 colspan=1>NTCC 11175,Biotype 4,serotype 3(0:3)</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>700822*</td><td rowspan=1 colspan=1>1x LoD</td></tr><tr><td rowspan=1 colspan=1>Yersiniaenterocolitica</td><td rowspan=1 colspan=1>33114 [CCUG11291,CCUG12369, CIP80.27, DSM4780, LMG7899, NCTC12982], Biovar1, O:8</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>9610</td><td rowspan=1 colspan=1>1x LoD</td></tr><tr><td rowspan=1 colspan=1>Yersiniaenterocolitica</td><td rowspan=1 colspan=1>0:9</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>55075</td><td rowspan=1 colspan=1>3x LoD</td></tr></table>

\* Strain tested during LoD verification study

Table 5.7: Inclusivity test results for Enteropathogenic $E _ { \ast }$ . coli (EPEC) strains. Only applicable for Para-Pak C&S samples.   

<table><tr><td rowspan=1 colspan=1>QIAstat-DxGastrointestinalPanel 2 Target</td><td rowspan=1 colspan=1>Pathogen</td><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>Supplier</td><td rowspan=1 colspan=1>CatalogID</td><td rowspan=1 colspan=1>x-foldLoDdetected</td></tr><tr><td rowspan=3 colspan=1>Enteropathogenic E.coli (EPEC)</td><td rowspan=1 colspan=1>Enteropathogenic E. coli(EPEC)</td><td rowspan=1 colspan=1>0111:NM</td><td rowspan=1 colspan=1>ZeptoMetriX</td><td rowspan=1 colspan=1>0801747*</td><td rowspan=1 colspan=1>1xLoD</td></tr><tr><td rowspan=1 colspan=1>Enteropathogenic E. coli(EPEC)</td><td rowspan=1 colspan=1>7.1493,084:H28</td><td rowspan=1 colspan=1>ZeptoMetrix</td><td rowspan=1 colspan=1>0801938*</td><td rowspan=1 colspan=1>1xLoD</td></tr><tr><td rowspan=1 colspan=1>Enteropathogenic E. coli(EPEC)</td><td rowspan=1 colspan=1>StokeW,0111:K58(B4):H-</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>33780</td><td rowspan=1 colspan=1>1xLoD</td></tr></table>

\* Strain tested during LoD verification study

Table 5.8: Inclusivity test results for Enterotoxigenic E. coli (ETEC) strains   

<table><tr><td rowspan=1 colspan=1>QIAstat-DxGastrointestinalPanel 2 Target</td><td rowspan=1 colspan=1>Pathogen</td><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>Supplier</td><td rowspan=1 colspan=1>CatalogID</td><td rowspan=1 colspan=1>x-foldLoDdetected</td></tr><tr><td rowspan=5 colspan=1>Enterotoxigenic E.coli (ETEC) lt/st</td><td rowspan=1 colspan=1>Enterotoxigenic E. coli(ETEC) lt/st</td><td rowspan=1 colspan=1>ST+, LT+</td><td rowspan=1 colspan=1>ZeptoMetrix</td><td rowspan=1 colspan=1>0801624*</td><td rowspan=1 colspan=1>1x LoD</td></tr><tr><td rowspan=1 colspan=1>Enterotoxigenic E. coli(ETEC) lt/st</td><td rowspan=1 colspan=1>H10407,078:H11,LT(+) / ctxA11(+</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>35401*</td><td rowspan=1 colspan=1>0.3xLoD</td></tr><tr><td rowspan=1 colspan=1>Enterotoxigenic E. coli(ETEC) lt/st</td><td rowspan=1 colspan=1>027:H7,ST (+)/LT (-)</td><td rowspan=1 colspan=1>SSIDiagnostica</td><td rowspan=1 colspan=1>82173</td><td rowspan=1 colspan=1>0.1xLoD</td></tr><tr><td rowspan=1 colspan=1>Enterotoxigenic E. coli(ETEC) lt/st</td><td rowspan=1 colspan=1>0115:H15,ST(+)/LT (-)</td><td rowspan=1 colspan=1>SSIDiagnostica</td><td rowspan=1 colspan=1>82174</td><td rowspan=1 colspan=1>3x LoD</td></tr><tr><td rowspan=1 colspan=1>Enterotoxigenic E. coli(ETEC) lt/st</td><td rowspan=1 colspan=1>0169:H-,ST (-) /LT (+)</td><td rowspan=1 colspan=1>SSIDiagnostica</td><td rowspan=1 colspan=1>82172</td><td rowspan=1 colspan=1>10xLoD#</td></tr></table>

\* Strain tested during LoD verification study # Testing at a lower concentration resulted in a detection rate of ${ < } 1 0 0 \%$

Table 5.9: Inclusivity test results for Enteroinvasive E. coli (EIEC)/Shigella strains   

<table><tr><td rowspan=1 colspan=1>QIAstat-DxGastrointestinalPanel 2 Target</td><td rowspan=1 colspan=1>Pathogen</td><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>Supplier</td><td rowspan=1 colspan=1>CatalogID</td><td rowspan=1 colspan=1>x-foldLoDdetected</td></tr><tr><td rowspan=9 colspan=1>Enteroinvasive E.coli (EIEC)/Shigella</td><td rowspan=1 colspan=1>Enteroinvasive E. coli(EIEC)</td><td rowspan=1 colspan=1>CDC EDL1282,029:NM</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>43892*</td><td rowspan=1 colspan=1>1x LoD</td></tr><tr><td rowspan=1 colspan=1>Enteroinvasive E. coli(EIEC)</td><td rowspan=1 colspan=1>0172:H-</td><td rowspan=1 colspan=1>SSIDiagnostica</td><td rowspan=1 colspan=1>82171</td><td rowspan=1 colspan=1>3x LoD</td></tr><tr><td rowspan=1 colspan=1>Shigella sonnei</td><td rowspan=1 colspan=1>NCDC 1120-66</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>25931*</td><td rowspan=1 colspan=1>1x LoD</td></tr><tr><td rowspan=1 colspan=1>Shigella boydii(Serogroup C)</td><td rowspan=1 colspan=1>Z131</td><td rowspan=1 colspan=1>ZeptoMetrix</td><td rowspan=1 colspan=1>0801900</td><td rowspan=1 colspan=1>1x LoD</td></tr><tr><td rowspan=1 colspan=1>Shigella flexneri(Serogroup B)</td><td rowspan=1 colspan=1>AMC 43-G-68[EVL 82,M134]</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>9199</td><td rowspan=1 colspan=1>1x LoD</td></tr><tr><td rowspan=1 colspan=1>Shigella flexneri(Serogroup B)</td><td rowspan=1 colspan=1>Z046</td><td rowspan=1 colspan=1>ZeptoMetrix</td><td rowspan=1 colspan=1>0801757</td><td rowspan=1 colspan=1>1x LoD</td></tr><tr><td rowspan=1 colspan=1>Shigella sonnei(Serogroup D)</td><td rowspan=1 colspan=1>WRAIR Ivirulent</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>29930</td><td rowspan=1 colspan=1>1x LoD</td></tr><tr><td rowspan=1 colspan=1>Shigella sonnei(Serogroup D)</td><td rowspan=1 colspan=1>Z004</td><td rowspan=1 colspan=1>ZeptoMetrix</td><td rowspan=1 colspan=1>0801627</td><td rowspan=1 colspan=1>3x LoD</td></tr><tr><td rowspan=1 colspan=1>Shigella boydii(Serogroup C)</td><td rowspan=1 colspan=1>AMC 43-G-58[M44 (Type170)]</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>9207</td><td rowspan=1 colspan=1>10xLoD#</td></tr></table>

\* Strain tested during LoD verification study # Testing at a lower concentration resulted in a detection rate of ${ < } 1 0 0 \%$

Table 5.10: Inclusivity test results for Shiga-like toxin-producing E. coli (STEC) (stx1/stx2-carrier strains). Only applicable for Para-Pak C&S samples.   

<table><tr><td rowspan=1 colspan=1>QIAstat-DxGastrointestinalPanel 2 Target</td><td rowspan=1 colspan=1>Pathogen</td><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>Supplier</td><td rowspan=1 colspan=1>CatalogID</td><td rowspan=1 colspan=1>x-foldLoDdetected</td></tr><tr><td rowspan=11 colspan=1>Shiga-like toxinproducing E. coli(STEC) stx1/stx2</td><td rowspan=1 colspan=1>Shiga-like toxin producingE. coli (STEC) - stx1/stx2</td><td rowspan=1 colspan=1>0157:H7;EDL933</td><td rowspan=1 colspan=1>ZeptoMetrix</td><td rowspan=1 colspan=1>0801622*</td><td rowspan=1 colspan=1>1x LoD</td></tr><tr><td rowspan=1 colspan=1>Shiga-like toxin producingE. coli (STEC) - stx1</td><td rowspan=1 colspan=1>026:H4,stx1 (+)</td><td rowspan=1 colspan=1>ZeptoMetrix</td><td rowspan=1 colspan=1>0801748*</td><td rowspan=1 colspan=1>1x LoD</td></tr><tr><td rowspan=1 colspan=1>Shiga-like toxin producingE. coli (STEC) - stx1/stx2</td><td rowspan=1 colspan=1>ReferenceATCC® 35150(EDL931),0157:H7,stx1 (+), stx2 (+)</td><td rowspan=1 colspan=1>Microbiologics</td><td rowspan=1 colspan=1>617</td><td rowspan=1 colspan=1>3x LoD</td></tr><tr><td rowspan=1 colspan=1>Shiga-like toxin producingE. coli (STEC) - stx1</td><td rowspan=1 colspan=1>Reference CDC00-3039,045:H2,unknown</td><td rowspan=1 colspan=1>Microbiologics</td><td rowspan=1 colspan=1>1098</td><td rowspan=1 colspan=1>1x LoD</td></tr><tr><td rowspan=1 colspan=1>Shiga-like toxin producingE. coli (STEC) - stx1</td><td rowspan=1 colspan=1>0103:H2,stx1 (+)</td><td rowspan=1 colspan=1>SSI Diagnostica</td><td rowspan=1 colspan=1>82170</td><td rowspan=1 colspan=1>3x LoD</td></tr><tr><td rowspan=1 colspan=1>Shiga-like toxin producingE. coli (STEC) - stx1/stx2</td><td rowspan=1 colspan=1>O22:H8,stx1c (+),stx2b (+)</td><td rowspan=1 colspan=1>SSI Diagnostica</td><td rowspan=1 colspan=1>91350</td><td rowspan=1 colspan=1>1x LoD</td></tr><tr><td rowspan=1 colspan=1>Shiga-like toxin producingE. coli (STEC) - stx2</td><td rowspan=1 colspan=1>092,0107:K+:H48,stx2d (+)</td><td rowspan=1 colspan=1>SSI Diagnostica</td><td rowspan=1 colspan=1>91352</td><td rowspan=1 colspan=1>10x LoD#</td></tr><tr><td rowspan=1 colspan=1>Shiga-like toxin producingE. coli (STEC) - stx2</td><td rowspan=1 colspan=1>0101:K32:H-,stx2e(+)</td><td rowspan=1 colspan=1>SSI Diagnostica</td><td rowspan=1 colspan=1>91354</td><td rowspan=1 colspan=1>0.3x LoD</td></tr><tr><td rowspan=1 colspan=1>Shiga-like toxin producingE. coli (STEC) - stx1/stx2</td><td rowspan=1 colspan=1>0128ac:H-,stx2f(+)</td><td rowspan=1 colspan=1>SSI Diagnostica</td><td rowspan=1 colspan=1>91355</td><td rowspan=1 colspan=1>10x LoD#</td></tr><tr><td rowspan=1 colspan=1>Shiga-like toxin producingE. coli (STEC) - stx2</td><td rowspan=1 colspan=1>026:H11,stx2a(+)</td><td rowspan=1 colspan=1>SSI Diagnostica</td><td rowspan=1 colspan=1>95211</td><td rowspan=1 colspan=1>1x LoD</td></tr><tr><td rowspan=1 colspan=1>Shiga-like toxin producingE. coli (STEC) - stx1</td><td rowspan=1 colspan=1>08 ,stx1d (+)</td><td rowspan=1 colspan=1>SSI Diagnostica</td><td rowspan=1 colspan=1>91349</td><td rowspan=1 colspan=1>1x LoD</td></tr></table>

\* Strain tested during LoD verification study # Testing at a lower concentration resulted in a detection rate of ${ < } 1 0 0 \%$ .

Table 5.11: Inclusivity test results for Shiga-like toxin producing E. coli (STEC) O157 strains. Only applicable for Para-Pak C&S samples.   

<table><tr><td rowspan=1 colspan=1>QIAstat-DxGastrointestinal Panel2 Target</td><td rowspan=1 colspan=1>Pathogen</td><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>Supplier</td><td rowspan=1 colspan=1>Catalog ID</td><td rowspan=1 colspan=1>x-foldLoDdetected</td></tr><tr><td rowspan=3 colspan=1>Shiga-like toxinproducing E. coli(STEC) O157</td><td rowspan=1 colspan=1>Shiga-like toxin producingE. coli (STEC) 0157</td><td rowspan=1 colspan=1>0157:H7;EDL933</td><td rowspan=1 colspan=1>ZeptoMetrix</td><td rowspan=1 colspan=1>0801622*</td><td rowspan=1 colspan=1>1x LoD</td></tr><tr><td rowspan=1 colspan=1>Shiga-like toxin producingE. coli (STEC) 0157</td><td rowspan=1 colspan=1>0128ac:H-,stx2f(+)</td><td rowspan=1 colspan=1>SSIDiagnostica</td><td rowspan=1 colspan=1>91355#</td><td rowspan=1 colspan=1>10xLoDs</td></tr><tr><td rowspan=1 colspan=1>Shiga-like toxin producingE. coli (STEC) 0157</td><td rowspan=1 colspan=1>ReferenceATCC® 35150(EDL 931),0157:H7, stx1(+), stx2 (+)</td><td rowspan=1 colspan=1>MicrobiologicS</td><td rowspan=1 colspan=1>617</td><td rowspan=1 colspan=1>1x LoD</td></tr></table>

\* Strain tested during LoD verification study # The $E$ . coli strain 91355 from SSI Diagnostica is reported as following in its catalog: $\nu t x 2 f \mathrm { + }$ , eae+. However, it was found to amplify for $E .$ . coli O157 in both QIAstat-Dx and FilmArray device \$Testing at a lower concentration resulted in a detection rate of ${ < } 1 0 0 \%$ .

Table 5.12: Inclusivity test results for Cryptosporidium strains   

<table><tr><td rowspan=1 colspan=1>QIAstat-DxGastrointestinal Panel2 Target</td><td rowspan=1 colspan=1>Pathogen</td><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>Supplier</td><td rowspan=1 colspan=1>Catalog ID</td><td rowspan=1 colspan=1>x-foldLoDdetected</td></tr><tr><td rowspan=4 colspan=1>Cryptosporidium</td><td rowspan=1 colspan=1>Cryptosporidium parvum</td><td rowspan=1 colspan=1>Iowa isolate</td><td rowspan=1 colspan=1>Waterborne</td><td rowspan=1 colspan=1>P102C*</td><td rowspan=1 colspan=1>1xLoD</td></tr><tr><td rowspan=1 colspan=1>Cryptosporidium hominis</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>PublicHealthWales</td><td rowspan=1 colspan=1>Clinicalsample; UKM84*</td><td rowspan=1 colspan=1>0.01xLoD</td></tr><tr><td rowspan=1 colspan=1>Cryptosporidium parvum</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>PRA-67DQ(isolatedgenomicDNA)</td><td rowspan=1 colspan=1>&lt;0.01LoD</td></tr><tr><td rowspan=1 colspan=1>Cryptosporidium meleagridis</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>PublicHealthWales</td><td rowspan=1 colspan=1>Clinicalsample;UKMEL 14</td><td rowspan=1 colspan=1>&lt;0.01LoD</td></tr></table>

\* Strain tested during LoD verification study

Table 5.13: Inclusivity test results for Cyclospora cayetanensis strains   

<table><tr><td rowspan=1 colspan=1>QIAstat-DxGastrointestinal Panel2 Target</td><td rowspan=1 colspan=1>Pathogen</td><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>Supplier</td><td rowspan=1 colspan=1>Catalog ID</td><td rowspan=1 colspan=1>x-foldLoDdetected</td></tr><tr><td rowspan=3 colspan=1>Cyclospora cayetanensis</td><td rowspan=1 colspan=1>Cyclosporacayetanensis</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>LACNY</td><td rowspan=1 colspan=1>Clinicalsample,LAC2825*</td><td rowspan=1 colspan=1>1xLoD</td></tr><tr><td rowspan=1 colspan=1>Cyclosporacayetanensis</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>LACNY</td><td rowspan=1 colspan=1>Clinicalsample,LAC2827*</td><td rowspan=1 colspan=1>1xLoD</td></tr><tr><td rowspan=1 colspan=1>Cyclosporacayetanensis</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>PRA-3000SD</td><td rowspan=1 colspan=1>1xLoD</td></tr></table>

\* Strain tested during LoD verification study

Table 5.14: Inclusivity test results for Entamoeba histolytica strains   

<table><tr><td rowspan=1 colspan=1>QIAstat-DxGastrointestinal Panel 2Target</td><td rowspan=1 colspan=1>Pathogen</td><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>Supplier</td><td rowspan=1 colspan=1>Catalog ID</td><td rowspan=1 colspan=1>x-foldLoDdetected</td></tr><tr><td rowspan=3 colspan=1>Entamoeba histolytica</td><td rowspan=1 colspan=1>Entamoebahistolytica</td><td rowspan=1 colspan=1>HM-1:IMSS(Mexico City1967)</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>30459*</td><td rowspan=1 colspan=1>1xLoD</td></tr><tr><td rowspan=1 colspan=1>Entamoebahistolytica</td><td rowspan=1 colspan=1>HK-9 (Korea)</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>30015*</td><td rowspan=1 colspan=1>1xLoD</td></tr><tr><td rowspan=1 colspan=1>Entamoebahistolytica</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>HospitalValld&#x27;Hebrón</td><td rowspan=1 colspan=1>Clinicalsample; 1</td><td rowspan=1 colspan=1>1xLoD</td></tr></table>

\* Strain tested during LoD verification study

Table 5.15: Inclusivity test results for Giardia lamblia strains   

<table><tr><td rowspan=1 colspan=1>QIAstat-DxGastrointestinalPanel 2 Target</td><td rowspan=1 colspan=1>Pathogen</td><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>Supplier</td><td rowspan=1 colspan=1>Catalog ID</td><td rowspan=1 colspan=1>x-foldLoDdetected</td></tr><tr><td rowspan=3 colspan=1>Giardia lamblia</td><td rowspan=1 colspan=1>Giardia lamblia</td><td rowspan=1 colspan=1>Portland -1(Portland, OR,1971)</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>30888*</td><td rowspan=1 colspan=1>1x LoD</td></tr><tr><td rowspan=1 colspan=1>Giardia lamblia</td><td rowspan=1 colspan=1>WB (Bethesda, MD,1979</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>30957*</td><td rowspan=1 colspan=1>1x LoD</td></tr><tr><td rowspan=1 colspan=1>Giardia intestinalis</td><td rowspan=1 colspan=1>H3 isolate</td><td rowspan=1 colspan=1>Waterborne</td><td rowspan=1 colspan=1>P101</td><td rowspan=1 colspan=1>1x LoD</td></tr></table>

\* Strain tested during LoD verification study

Table 5.16: Inclusivity test results for Adenovirus F40/F41 strains   

<table><tr><td rowspan=1 colspan=1>QIAstat-DxGastrointestinalPanel 2 Target</td><td rowspan=1 colspan=1>Pathogen</td><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>Supplier</td><td rowspan=1 colspan=1>Catalog ID</td><td rowspan=1 colspan=1>x-fold LoDdetected</td></tr><tr><td rowspan=4 colspan=1>Adenovirus F40/F41</td><td rowspan=1 colspan=1>Human Adenovirus F41</td><td rowspan=1 colspan=1>Tak</td><td rowspan=1 colspan=1>ZeptoMetrix</td><td rowspan=1 colspan=1>0810085CF*</td><td rowspan=1 colspan=1>1x LoD</td></tr><tr><td rowspan=1 colspan=1>Human Adenovirus F41</td><td rowspan=1 colspan=1>Tak (73-3544)</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>VR-930</td><td rowspan=1 colspan=1>10x LoD#</td></tr><tr><td rowspan=1 colspan=1>Human Adenovirus F40</td><td rowspan=1 colspan=1>Dugan [79-18025]</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>VR-931</td><td rowspan=1 colspan=1>10x LoD#</td></tr><tr><td rowspan=1 colspan=1>Human Adenovirus F 40</td><td rowspan=1 colspan=1>Dugan</td><td rowspan=1 colspan=1>ZeptoMetrix</td><td rowspan=1 colspan=1>0810084CF*</td><td rowspan=1 colspan=1>3x LoD</td></tr></table>

\* Strain tested during LoD verification study # Testing at a lower concentration resulted in a detection rate of ${ < } 1 0 0 \%$ .

Table 5.17: Inclusivity test results for Astrovirus strains   

<table><tr><td rowspan=1 colspan=1>QIAstat-DxGastrointestinal Panel 2Target</td><td rowspan=1 colspan=1>Pathogen</td><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>Supplier</td><td rowspan=1 colspan=1>Catalog ID</td><td rowspan=1 colspan=1>x-foldLoDdetected</td></tr><tr><td rowspan=4 colspan=1>Astrovirus</td><td rowspan=1 colspan=1>Human Astrovirus</td><td rowspan=1 colspan=1>ERE IID 2371 (type8)</td><td rowspan=1 colspan=1>ZeptoMetrix</td><td rowspan=1 colspan=1>0810277CF*</td><td rowspan=1 colspan=1>1x LoD</td></tr><tr><td rowspan=1 colspan=1>Human Astrovirus</td><td rowspan=1 colspan=1>HAstV-1</td><td rowspan=1 colspan=1>Universitatde Barcelona</td><td rowspan=1 colspan=1>Clinical sample;160521599</td><td rowspan=1 colspan=1>1x LoD</td></tr><tr><td rowspan=1 colspan=1>Human Astrovirus</td><td rowspan=1 colspan=1>ERE IID 2868 (type4)</td><td rowspan=1 colspan=1>ZeptoMetrix</td><td rowspan=1 colspan=1>0810276CF*</td><td rowspan=1 colspan=1>1x LoD</td></tr><tr><td rowspan=1 colspan=1>Human Astrovirus</td><td rowspan=1 colspan=1>HAstV-3</td><td rowspan=1 colspan=1>Universitatde Barcelona</td><td rowspan=1 colspan=1>Clinical sample;151601306</td><td rowspan=1 colspan=1>1x LoD</td></tr></table>

\* Strain tested during LoD verification study

Table 5.18: Inclusivity test results for Norovirus GI/GII strains   

<table><tr><td rowspan=1 colspan=1>QIAstat-DxGastrointestina1Panel 2 Target</td><td rowspan=1 colspan=1>Pathogen</td><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>Supplier</td><td rowspan=1 colspan=1>Catalog ID</td><td rowspan=1 colspan=1>x-foldLoDdetected</td></tr><tr><td rowspan=12 colspan=1>NorovirusGI/GII</td><td rowspan=1 colspan=1>Human NorovirusGenogroup 1</td><td rowspan=1 colspan=1>Recombinant GI.1</td><td rowspan=1 colspan=1>ZeptoMetrix</td><td rowspan=1 colspan=1>0810086CF*</td><td rowspan=1 colspan=1>1x LoD</td></tr><tr><td rowspan=1 colspan=1>Human NorovirusGenogroup 1</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>IndianaUniversityHealth</td><td rowspan=1 colspan=1>Clinical sample;IU3156</td><td rowspan=1 colspan=1>1x LoD</td></tr><tr><td rowspan=1 colspan=1>Human NorovirusGenogroup 1</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>IndianaUniversityHealth</td><td rowspan=1 colspan=1>Clinical sample;IU3220</td><td rowspan=1 colspan=1>1x LoD</td></tr><tr><td rowspan=1 colspan=1>Human NorovirusGenogroup 1</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>TriCoreReferenceLaboratories</td><td rowspan=1 colspan=1>Clinical sample;TC4274</td><td rowspan=1 colspan=1>3x LoD</td></tr><tr><td rowspan=1 colspan=1>Human NorovirusGenogroup 2</td><td rowspan=1 colspan=1>Recombinant GII.4</td><td rowspan=1 colspan=1>ZeptoMetrix</td><td rowspan=1 colspan=1>0810087CF*</td><td rowspan=1 colspan=1>1x LoD</td></tr><tr><td rowspan=1 colspan=1>Human NorovirusGenogroup 2</td><td rowspan=1 colspan=1>GII.2</td><td rowspan=1 colspan=1>Hospital Valld&#x27;Hebrón</td><td rowspan=1 colspan=1>Clinical sample;198058327</td><td rowspan=1 colspan=1>1x LoD</td></tr><tr><td rowspan=1 colspan=1>Human NorovirusGenogroup 2</td><td rowspan=1 colspan=1>GII.4</td><td rowspan=1 colspan=1>Universitat deBarcelona</td><td rowspan=1 colspan=1>Clinicialsample;N26.2TA</td><td rowspan=1 colspan=1>1x LoD</td></tr><tr><td rowspan=1 colspan=1>Human NorovirusGenogroup 2</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>LACNY</td><td rowspan=1 colspan=1>Clinical sample;LAC2019</td><td rowspan=1 colspan=1>1x LoD</td></tr><tr><td rowspan=1 colspan=1>Human NorovirusGenogroup 2</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>NationwideChildren&#x27;sHospital</td><td rowspan=1 colspan=1>Clinical sample;NWC6063</td><td rowspan=1 colspan=1>1x LoD</td></tr><tr><td rowspan=1 colspan=1>Human NorovirusGenogroup 2</td><td rowspan=1 colspan=1>GII.6</td><td rowspan=1 colspan=1>QIAGENBarcelona(STAT-Dx)</td><td rowspan=1 colspan=1>Clinical sample;GI 12</td><td rowspan=1 colspan=1>3x LoD</td></tr><tr><td rowspan=1 colspan=1>Human NorovirusGenogroup 2</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>LACNY</td><td rowspan=1 colspan=1>Clinical sample;LAC2133</td><td rowspan=1 colspan=1>10xLoD#</td></tr><tr><td rowspan=1 colspan=1>Human NorovirusGenogroup 2</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>LACNY</td><td rowspan=1 colspan=1>Clinical sample;LAC2074</td><td rowspan=1 colspan=1>10xLoD#</td></tr></table>

\* Strain tested during LoD verification study # Testing at a lower concentration resulted in a detection rate of ${ < } 1 0 0 \%$ .

Table 5.19: Inclusivity test results for Rotavirus A strains   

<table><tr><td rowspan=1 colspan=1>QIAs QIAstat-DxGastrointestinalPanel 2 Target</td><td rowspan=1 colspan=1>Pathogen</td><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>Supplier</td><td rowspan=1 colspan=1>CatalogID</td><td rowspan=1 colspan=1>x-foldLoDdetected</td></tr><tr><td rowspan=5 colspan=1>Rotavirus A</td><td rowspan=1 colspan=1>HumanRotavirus A</td><td rowspan=1 colspan=1>69M</td><td rowspan=1 colspan=1>ZeptoMetrix</td><td rowspan=1 colspan=1>0810280CF*</td><td rowspan=1 colspan=1>1xLoD</td></tr><tr><td rowspan=1 colspan=1>HumanRotavirus A</td><td rowspan=1 colspan=1>Wa,G1P1A[8]</td><td rowspan=1 colspan=1>ZeptoMetrix</td><td rowspan=1 colspan=1>0810041CF*</td><td rowspan=1 colspan=1>1xLoD</td></tr><tr><td rowspan=1 colspan=1>HumanRotavirus A</td><td rowspan=1 colspan=1>DS-1,G2P1B[4]</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>VR-2550</td><td rowspan=1 colspan=1>1xLoD</td></tr><tr><td rowspan=1 colspan=1>HumanRotavirus A</td><td rowspan=1 colspan=1>Va70</td><td rowspan=1 colspan=1>ZeptoMetrix</td><td rowspan=1 colspan=1>0810281CF</td><td rowspan=1 colspan=1>1xLoD</td></tr><tr><td rowspan=1 colspan=1>HumanRotavirus A</td><td rowspan=1 colspan=1>RRV</td><td rowspan=1 colspan=1>ZeptoMetrix</td><td rowspan=1 colspan=1>0810530CF</td><td rowspan=1 colspan=1>10xLoD#</td></tr></table>

\* Strain tested during LoD verification study # Testing at a lower concentration resulted in a detection rate of ${ < } 1 0 0 \%$ .

# In silico analysis

In silico analysis of potential reactivity showed that the following organisms (including species, subspecies, subtypes, serotypes or serovars) are predicted to be detected with the QIAstat-Dx Gastrointestinal Panel 2 (Table 5.20).

Table 5.20: Organisms with predicted reactivity based on in silico analysis   

<table><tr><td colspan="1" rowspan="1">QIAstat-Dx GI Panel2 Target</td><td colspan="1" rowspan="1">Organisms with predicted reactivity</td></tr><tr><td colspan="2" rowspan="1">Bacteria</td></tr><tr><td colspan="1" rowspan="1">Campylobacter</td><td colspan="1" rowspan="1">Campylobacter coli*, Campylobacter jejuni, Campylobacter jejuni subsp.jejuni, Campylobacter jejuni subsp.doylei, Campylobacter upsaliensis</td></tr><tr><td colspan="1" rowspan="2">Salmonella</td><td colspan="1" rowspan="1">Salmonella bongori*, Salmonella enterica subsp. salamae ⅡI (e.g. serovar 55:k:z39), Salmonella entericau.arizoae Ia  erovar 63:g:5 Saloelaenteric ubsdiaone Ib erovar7:v:,Salmonella enterica subsp. houtenae IV (e.g. serovar 43:z4), Salmonella enterica subsp. indica VI.</td></tr><tr><td colspan="1" rowspan="1">Salmonella enterica subsp. enterica (up to 92 different serovars including Agona, Anatum, Bareilly,Choleraesuis, Enteritidis, Heidelberg, Infantis, Kentucky, Montevideo, Newport, aratyphiA, Senftenberg,Tennessee, Thompson, Typhi, Typhimurium, Weltevreden*)</td></tr><tr><td colspan="1" rowspan="1">Plesiomonasshigelloides</td><td colspan="1" rowspan="1">Plesiomonas shigelloides (e.g. strains NCTC10360, ATCC 14029T, R4605035)</td></tr><tr><td colspan="1" rowspan="1">Yersiniaenterocolitica</td><td colspan="1" rowspan="1">Yersinia enterocolitica, Yersinia enterocolitica subsp. palearctica, Yersinia enterocolitica subsp.enterocolitica</td></tr><tr><td colspan="1" rowspan="1">Enteroinvasive E. coli(EIEC)/Shigella</td><td colspan="1" rowspan="1">EnteronvasiveE coli (EEC), Escherichia coli sp. Shigella fexner, Shigell dysenteria, Shigellaboydii,Shigella sonnei.</td></tr><tr><td colspan="1" rowspan="1">Enteropathogenic E.coli (EPEC) β</td><td colspan="1" rowspan="1">Enteropathogenic E. coli (EPEC) (e.g. including serotypes OUT: HND, OUT:H6, OUT:H34, OUT:H21,055:H7, 0119:HNM, O117)</td></tr><tr><td colspan="1" rowspan="1">Enterotoxigenic E. coli(ETEC) &amp;</td><td colspan="1" rowspan="1">Enterotoxigenic E. coli (ETEC) (including H10407 and E24377A strains and serotypes O169:H41, O25:H42,8:H28, O6:H16) carrier of Heat-labile enterotoxin gene subtype LT-I and Heat-stable enterotoxin genevariant Sta, subtypes STp and STh</td></tr><tr><td colspan="1" rowspan="1">Shiga-like toxin-producing E. coli(STEC) - x1/stx2 β</td><td colspan="1" rowspan="1">Shiga-like toxin-producing E. coli (STEC) including O157:H7 and O157:NM serotype and non-O157serotypes (O111:NM, 0111:H-, 026:H11, 0145:NM, 0145:H28, 045:H2, 026:H11, ONT:NM, O0104:H4,O121:H19, O145:H34, O113:H21, ONT:H-, O128:H2, OUT:HNM, O124:HNM E. coli strains carier of:</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">s 1, tx1 stx2 stx2b, stx2c, stx2 x2 x2e, stx2 x2, x2h, sx2 tx2 2an2.Other stx-carrying bacteria: Shigella sonnei, Shigella dysenteriae</td></tr><tr><td colspan="1" rowspan="1">Shiga-like toxin-producing E. coli(STEC) O157B</td><td colspan="1" rowspan="1">Echerichia coli O157 including: STEC O157:H7 strains (e.g. EDL933) and E. coli O157: non-H7 groupsincluding non-Shiga-toxigenic E. coli O157 bacteria (e.g. serotype O157:H45)</td></tr><tr><td colspan="2" rowspan="1">Parasites</td></tr><tr><td colspan="1" rowspan="1">Cryptosporidium#</td><td colspan="1" rowspan="1">Common Cryptosporidium species involved in human disease: C. parvum, C. hominis.Less common Cryptosporidium species involved in human infections: C. meleagridis, C. felis , C. bovis, C.viatorum, C. ubiquitum,C. tyzzeri, C. cuniculus, Cryptosporidium sp. Chipmunk genotype I, C. canis*.Rare or non-human species: Cryptosporidium wrairi</td></tr><tr><td colspan="1" rowspan="1">Cyclosporacayetanensis</td><td colspan="1" rowspan="1">Cyclospora cayetanensis (including strains LG, CY9, NP20 and NP21) *</td></tr><tr><td colspan="1" rowspan="1">Entamoebahistolytica</td><td colspan="1" rowspan="1">Entamoeba histolytica (e.g. strains HM-1: IMSS, EHMfas1, HK-9)*</td></tr><tr><td colspan="1" rowspan="1">Giardia lamblia</td><td colspan="1" rowspan="1">Giardia lamblia (aka Giardia duodenalis, Giardia intestinalis)*</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Viruses</td></tr><tr><td colspan="1" rowspan="1">Adenovirus</td><td colspan="1" rowspan="1">Human Adenovirus F40/41</td></tr><tr><td colspan="1" rowspan="1">Astrovirus$</td><td colspan="1" rowspan="1">Human Astrovirus (including types 1, 2, 3, 4, 5, 6, 7, 8)</td></tr><tr><td colspan="1" rowspan="2">Norovirus GI/GII</td><td colspan="1" rowspan="1">Norovirus genogroup II genotypes: GII.1, GII.2, GII.3*, GII.4*, GII.5, GII.6, GII.7, GII.8, GII.9, GII.10,GII.12, GII.13, GI.14, GII.16, GII.17, GII.20, GII.21, GII.22, GII.23, GII.24*, GII.25, GII.26, GII.27,GII.NA1 and GII.NA2*</td></tr><tr><td colspan="1" rowspan="1">Norovirus genogroup I genotypes: GI.1, GI2, GI.3*, GI.4*, GI.5, GI.6*, GI.7*, GI.8, GI.9.</td></tr><tr><td colspan="1" rowspan="1">Rotavirus</td><td colspan="1" rowspan="1">Rotavirus A including genotypes: G1P[8]*, G2P[4]*, G3P[8]*, G4P[8]*, G9P[6], G9P[8]*, G12P[6]* andG12P[8]*</td></tr></table>

\* Certain sequences are predicted to be detected with reduced sensitivity due to the presence of a small number of mismatches at critical positions of the primer-probe design. &The assay is not predicted to detect bacteria carrier of Heat-labile enterotoxin gene subtype LT-II and/or of Heat-stable enterotoxin gene variant Stb e. #The assay is not predicted to detect other Cryptosporidium spp. less involved in human disease: C. andersoni and C. muris. \$ The assay is not predicted to detect Human Astrovirus types MLB1-3 and VA1-5. β Target only applicable for Para-Pak C&S samples.

# Analytical Specificity (Cross-Reactivity and Exclusivity)

The analytical specificity study was carried out by in vitro testing and in silico analysis to assess the potential cross-reactivity of the QIAstat-Dx Gastrointestinal Panel 2. On-panel organisms were tested to assess the potential for intra-panel cross-reactivity and Off-panel organisms were tested to evaluate cross-reactivity with organisms not covered by the panel content. The On-panel and Off-panel organisms tested are shown in Table 5.21 and Table 5.22, respectively.

Samples were prepared by single spiking organisms into negative stool resuspended in ParaPak C&S media at the highest concentration possible based on the organism stock, preferably at $1 0 ^ { 5 } \mathrm { T C I D } 5 0 / \mathrm { m l }$ for viral, $1 0 ^ { 5 } \mathrm { c e l l s / m l }$ for parasite targets and $1 0 ^ { 6 } \mathrm { C F U / m l }$ for bacterial targets. The pathogens were tested in 3 replicates. There was no intra-panel or Off-panel cross-reactivity for all pathogens tested in vitro, except for two non-targeted

Campylobacter species (C. helveticus and C. lari) that cross-reacted with the Campylobacter assay oligonucleotides included in the QIAstat-Dx Gastrointestinal Panel 2.

Table 5.21: List of Analytical Specificity On-Panel Pathogens Tested   

<table><tr><td rowspan=1 colspan=1>Type</td><td rowspan=1 colspan=1>Pathogen</td></tr><tr><td rowspan=10 colspan=1>Bacteria</td><td rowspan=1 colspan=1>Campylobacter coli</td></tr><tr><td rowspan=1 colspan=1>Campylobacter jejuni</td></tr><tr><td rowspan=1 colspan=1>Campylobacter upsaliensis</td></tr><tr><td rowspan=1 colspan=1>Escherichia coli (EPEC)*</td></tr><tr><td rowspan=1 colspan=1>Escherichia coli (ETEC lt/st)</td></tr><tr><td rowspan=1 colspan=1>Escherichia coli (STEC)*</td></tr><tr><td rowspan=1 colspan=1>Plesiomonas shigelloides</td></tr><tr><td rowspan=1 colspan=1>Salmonella enterica</td></tr><tr><td rowspan=1 colspan=1>Shigella sonnei</td></tr><tr><td rowspan=1 colspan=1>Yersinia enterocolitica</td></tr><tr><td rowspan=4 colspan=1>Parasites</td><td rowspan=1 colspan=1>Cryptosporidium parvum</td></tr><tr><td rowspan=1 colspan=1>Cyclospora cayetanensis</td></tr><tr><td rowspan=1 colspan=1>Entamoeba histolytica</td></tr><tr><td rowspan=1 colspan=1>Giardia lamblia</td></tr><tr><td rowspan=5 colspan=1>Viruses</td><td rowspan=1 colspan=1>Adenovirus F41</td></tr><tr><td rowspan=1 colspan=1>Astrovirus</td></tr><tr><td rowspan=1 colspan=1>Norovirus GI</td></tr><tr><td rowspan=1 colspan=1>Norovirus GII</td></tr><tr><td rowspan=1 colspan=1>Rotavirus</td></tr></table>

\* Target only applicable for Para-Pak C&S samples.

Table 5.22: List of Analytical Specificity Off-Panel Pathogens Tested   

<table><tr><td colspan="1" rowspan="1">Type</td><td colspan="1" rowspan="1">Pathogen (Potential Cross-Reactant)</td></tr><tr><td colspan="1" rowspan="42">Bacteria</td><td colspan="1" rowspan="1">Abiotrophia defectiva</td></tr><tr><td colspan="1" rowspan="1">Acinetobacter baumannii</td></tr><tr><td colspan="1" rowspan="1">Aeromonas hydrophila</td></tr><tr><td colspan="1" rowspan="1">Arcobacter cryaerophilus</td></tr><tr><td colspan="1" rowspan="1">Bacillus subtilis</td></tr><tr><td colspan="1" rowspan="1">Bifidobacterium bifidum</td></tr><tr><td colspan="1" rowspan="1">Campylobacter fetus</td></tr><tr><td colspan="1" rowspan="1">Campylobacter gracilis</td></tr><tr><td colspan="1" rowspan="1">Campylobacter helveticus (Cross-reactive for Campylobacter target)</td></tr><tr><td colspan="1" rowspan="1">Campylobacter hominis</td></tr><tr><td colspan="1" rowspan="1">Campylobacter lari (Cross-reactive for Campylobacter target)</td></tr><tr><td colspan="1" rowspan="1">Campylobacter mucosalis</td></tr><tr><td colspan="1" rowspan="1">Campylobacter rectus</td></tr><tr><td colspan="1" rowspan="1">Citrobacter freundii</td></tr><tr><td colspan="1" rowspan="1">Clostridium difficile non-toxigenic</td></tr><tr><td colspan="1" rowspan="1">Clostridium perfringens</td></tr><tr><td colspan="1" rowspan="1">Clostridium septicum</td></tr><tr><td colspan="1" rowspan="1">Clostridium tetani</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium genitalium</td></tr><tr><td colspan="1" rowspan="1">Enterobacter aerogenes</td></tr><tr><td colspan="1" rowspan="1">Enterobacter cloacae</td></tr><tr><td colspan="1" rowspan="1">Enterococcus faecalis</td></tr><tr><td colspan="1" rowspan="1">Enterococcus faecium</td></tr><tr><td colspan="1" rowspan="1">Escherichia fergusonii</td></tr><tr><td colspan="1" rowspan="1">Escherichia hermannii</td></tr><tr><td colspan="1" rowspan="1">Escherichia vulneris</td></tr><tr><td colspan="1" rowspan="1">Faecalibacterium prausnitzii</td></tr><tr><td colspan="1" rowspan="1">Gardnerella vaginalis</td></tr><tr><td colspan="1" rowspan="1">Haemophilus influenzae</td></tr><tr><td colspan="1" rowspan="1">Hathewaya histolytica (Clostridium)</td></tr><tr><td colspan="1" rowspan="1">Helicobacter pylori</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus casei</td></tr><tr><td colspan="1" rowspan="1">Listeria monocytogenes</td></tr><tr><td colspan="1" rowspan="1">Proteus mirabilis</td></tr><tr><td colspan="1" rowspan="1">Proteus vulgaris</td></tr><tr><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus subsp. aureus</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus epidermidis</td></tr><tr><td colspan="1" rowspan="1">Streptococcus agalactiae</td></tr><tr><td colspan="1" rowspan="1">Streptococcus pyogenes</td></tr><tr><td colspan="1" rowspan="4">Fungi</td><td colspan="1" rowspan="1">Aspergillus fumigatus</td></tr><tr><td colspan="1" rowspan="1">Candida albicans</td></tr><tr><td colspan="1" rowspan="1">Saccharomyces boulardii</td></tr><tr><td colspan="1" rowspan="1">Saccharomyces cerevisiae</td></tr><tr><td colspan="1" rowspan="6">Parasite</td><td colspan="1" rowspan="1">Babesia microti</td></tr><tr><td colspan="1" rowspan="1">Blastocystis hominis</td></tr><tr><td colspan="1" rowspan="1">Chlamydia trachomatis</td></tr><tr><td colspan="1" rowspan="1">Giardia muris</td></tr><tr><td colspan="1" rowspan="1">Toxoplasma gondii</td></tr><tr><td colspan="1" rowspan="1">Trichomonas tenax</td></tr><tr><td colspan="1" rowspan="15">Viruses</td><td colspan="1" rowspan="1">Adenovirus C:2</td></tr><tr><td colspan="1" rowspan="1">Adenovirus B:34</td></tr><tr><td colspan="1" rowspan="1">Adenovirus B3</td></tr><tr><td colspan="1" rowspan="1">Adenovirus E:4a</td></tr><tr><td colspan="1" rowspan="1">Adenovirus serotype 1</td></tr><tr><td colspan="1" rowspan="1">Adenovirus serotype 5</td></tr><tr><td colspan="1" rowspan="1">Adenovirus serotype 8</td></tr><tr><td colspan="1" rowspan="1">Bocavirus Type 1</td></tr><tr><td colspan="1" rowspan="1">Coronavirus 229E</td></tr><tr><td colspan="1" rowspan="1">Coxsackievirus B3</td></tr><tr><td colspan="1" rowspan="1">Cytomegalovirus</td></tr><tr><td colspan="1" rowspan="1">Enterovirus 6 (Echovirus)</td></tr><tr><td colspan="1" rowspan="1">Enterovirus 68</td></tr><tr><td colspan="1" rowspan="1">Herpes Simplex Virus Type 2</td></tr><tr><td colspan="1" rowspan="1">Rhinovirus 1A</td></tr></table>

In silico predictions of potential cross-reactions show that the following cross-reactions may occur when testing stool samples with the QIAstat-Dx Gastrointestinal Panel 2 (Table 5.23)

Table 5.23: Potential Cross-Reactions Based On in silico Analysis   

<table><tr><td rowspan=1 colspan=1>QIAstat-Dx GastrointestinalPanel 2 Target</td><td rowspan=1 colspan=1>DesignAbbreviation</td><td rowspan=1 colspan=1>Potential cross-reactiveorganisms</td></tr><tr><td rowspan=1 colspan=1>Enteropathogenic E. coli (EPEC)</td><td rowspan=1 colspan=1>EPEC_eae</td><td rowspan=1 colspan=1>Shigella boydii1,2,3,Escherichia albertii,2</td></tr><tr><td rowspan=1 colspan=1>Campylobacter.</td><td rowspan=1 colspan=1>Cupsal_cpn60</td><td rowspan=1 colspan=1>Campylobacter lari4,Campylobacter helveticus4</td></tr><tr><td rowspan=3 colspan=1>Shiga-like toxin-producing E. coli(STEC) stx1/stx2 7</td><td rowspan=1 colspan=1>STEC_stx1</td><td rowspan=1 colspan=1>Shigella sonnei1,3, Shigelladysenteriae1,3</td></tr><tr><td rowspan=1 colspan=1>STEC_stx2</td><td rowspan=1 colspan=1>Acinetobacter haemolyticus1,5,Citrobacter freundi.5Enterobacter cloacae1,5Aeromonas caviae1,5</td></tr><tr><td rowspan=1 colspan=1>STEC stx2f</td><td rowspan=1 colspan=1>Escherichia albertiil,5</td></tr><tr><td rowspan=1 colspan=1>E. coli 0157 7</td><td rowspan=1 colspan=1>0157</td><td rowspan=1 colspan=1>Non-STEC E. coli O157strains6</td></tr></table>

1Note that these potential cross-reactions identified by in-silico analysis reflects sequences which can be acquired between species by horizontal gene transfer. 2Rare or less common eae intimin carrier organisms $^ { 3 } \mathrm { O n }$ -panel target. $^ 4 \mathrm { I n }$ vitro testing of Campylobacter lari and Campylobacter helveticus strains at high concentration confirmed potential cross-reactivity of these Campylobacter species with the QIAstat-Dx Gastrointestinal Panel 2 assay. 5Rare or less common Stx toxins producers $^ 6 E$ . coli O157 reported by the QIAstat-Dx Gastrointestinal Panel will only be called when there is a positive amplification for the $E$ . coli (STEC) design according to the calling algorithm. An infrequent case of an E. coli (STEC) and an $E _ { \ast }$ . coli O157 co-infection they will not be differentiated from a single infection caused by an STEC O157:H7 strain. 7 Target only applicable for Para-Pak C&S samples.

# Interfering Substances

The effect of potentially interfering substances on the detectability of the QIAstat-Dx Gastrointestinal Panel 2 organisms was evaluated. Thirty-four (34) potentially interfering substances were spiked into the sample mixes at a level predicted to be above the concentration of the substance likely to be found in stool specimens. Each organism was tested at 3x LoD and testing was performed in triplicates. Endogenous substances such as human whole blood, human genomic DNA and several pathogens were tested alongside exogenous substances like antibiotics, other gastrointestinal-related medications and different technique-specific substances.

For the vast majority of substances tested, no inhibition was observed, with the exceptions of mucin, calcium carbonate, nonoxynol-9 and Rotavirus reassortants, that may cause inhibition at high concentration.

Mucin at $5 \%$ w/v was found to generate false positives results for the Yersinia target. These signals were investigated by testing the interfering substance with and FDA-cleared method and they were confirmed to be present in the endogenous substance.

Calcium carbonate was found to interfere with the detection of all the QIAstat-Dx Gastrointestinal Panel 2 targets at concentrations above $0 . 5 \% \mathrm { w / v }$ .

Nonoxynol-9 was found to interfere with the detection of Entamoeba histolytica at concentrations above $0 . 0 2 \% \mathrm { \Omega } \mathrm { v }$ .

As predicted, Rotavirus reassortants WC3:2-5, R574(9) and WI79-4,9 used in Rotavirus A vaccines generated positive results for Rotavirus A in the QIAstat-Dx Gastrointestinal Panel 2. Final concentrations without interference (i.e. no false positive results for Rotavirus) for WC3:2-5, R574(9) and WI79-4,9 were $8 . 8 9 \mathrm { x } 1 0 ^ { - 5 }$ $\mathrm { T C I D } _ { 5 0 } / \mathrm { m l }$ and 1.10 PFU/ml, respectively (see Table 5.24 for other concentrations tested).

Results from the 34 interfering substances that could be present or introduced in a stool specimen are provided in Table 5.24.

Table 5.24: Final highest concentrations per potentially interfering substance without observable interference on the QIAstat-Dx Gastrointestinal Panel 2   

<table><tr><td colspan="1" rowspan="1">Substance tested</td><td colspan="1" rowspan="1">Concentration tested</td><td colspan="1" rowspan="1">Result</td></tr><tr><td colspan="3" rowspan="1">Endogenous substances</td></tr><tr><td colspan="1" rowspan="1">Bovine and ovine bile</td><td colspan="1" rowspan="1">12% w/v</td><td colspan="1" rowspan="1">No Interference</td></tr><tr><td colspan="1" rowspan="1">Cholesterol</td><td colspan="1" rowspan="1">1.5% w/v</td><td colspan="1" rowspan="1">No Interference</td></tr><tr><td colspan="1" rowspan="1">Fatty acids (palmitic acid)</td><td colspan="1" rowspan="1">0.2% w/v</td><td colspan="1" rowspan="1">No Interference</td></tr><tr><td colspan="1" rowspan="1">Fatty acids (stearic acid)</td><td colspan="1" rowspan="1">0.4% w/v</td><td colspan="1" rowspan="1">No Interference</td></tr><tr><td colspan="1" rowspan="3">Human genomic DNA</td><td colspan="1" rowspan="1">20 μg/mL</td><td colspan="1" rowspan="1">No Interference</td></tr><tr><td colspan="1" rowspan="1">10 μg/mL</td><td colspan="1" rowspan="1">No Interference</td></tr><tr><td colspan="1" rowspan="1">5 μg/mL</td><td colspan="1" rowspan="1">No Interference</td></tr><tr><td colspan="1" rowspan="1">Human stool (overfill of Cary Blair vial)</td><td colspan="1" rowspan="1">300 mg/mL</td><td colspan="1" rowspan="1">No Interference</td></tr><tr><td colspan="1" rowspan="1">Human urine</td><td colspan="1" rowspan="1">50% v/v</td><td colspan="1" rowspan="1">No Interference</td></tr><tr><td colspan="1" rowspan="1">Human whole blood with Na Citrate</td><td colspan="1" rowspan="1">40% v/v</td><td colspan="1" rowspan="1">No Interference</td></tr><tr><td colspan="1" rowspan="2">Mucin from bovine submaxillary</td><td colspan="1" rowspan="1">5% w/v</td><td colspan="1" rowspan="1">Interference#</td></tr><tr><td colspan="1" rowspan="1">2.5% w/v</td><td colspan="1" rowspan="1">No Interference</td></tr><tr><td colspan="1" rowspan="1">Triglycerides</td><td colspan="1" rowspan="1">5% w/v</td><td colspan="1" rowspan="1">No Interference</td></tr><tr><td colspan="1" rowspan="1">Exogenous substances</td><td colspan="1" rowspan="1">Exogenous substances</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Bacitracin</td><td colspan="1" rowspan="1">250U/mL</td><td colspan="1" rowspan="1">No Interference</td></tr><tr><td colspan="1" rowspan="2">Bisacodyl</td><td colspan="1" rowspan="1">0.3% w/v</td><td colspan="1" rowspan="1">No Interference</td></tr><tr><td colspan="1" rowspan="1">0.15% w/v</td><td colspan="1" rowspan="1">No Interference</td></tr><tr><td colspan="1" rowspan="1">Bismuth subsalicylate</td><td colspan="1" rowspan="1">0.35% w/v</td><td colspan="1" rowspan="1">No Interference</td></tr><tr><td colspan="1" rowspan="2">Calcium carbonate (TUMS® ExtraStrength 750)</td><td colspan="1" rowspan="1">5% w/v</td><td colspan="1" rowspan="1">Interference</td></tr><tr><td colspan="1" rowspan="1">0.5% w/v</td><td colspan="1" rowspan="1">No Interference</td></tr><tr><td colspan="1" rowspan="1">Docusate sodium</td><td colspan="1" rowspan="1">2.5% w/v</td><td colspan="1" rowspan="1">No Interference</td></tr><tr><td colspan="1" rowspan="1">Doxycycline hydrochloride</td><td colspan="1" rowspan="1">0.05% w/v</td><td colspan="1" rowspan="1">No Interference</td></tr><tr><td colspan="1" rowspan="1">Glycerin</td><td colspan="1" rowspan="1">50% v/v</td><td colspan="1" rowspan="1">No Interference</td></tr><tr><td colspan="1" rowspan="1">Hydrocortisone</td><td colspan="1" rowspan="1">0.5% w/v</td><td colspan="1" rowspan="1">No Interference</td></tr><tr><td colspan="1" rowspan="1">Loperamide hydrochloride</td><td colspan="1" rowspan="1">0.078% w/v</td><td colspan="1" rowspan="1">No Interference</td></tr><tr><td colspan="1" rowspan="1">Magnesium hydroxide</td><td colspan="1" rowspan="1">0.1% w/v</td><td colspan="1" rowspan="1">No Interference</td></tr><tr><td colspan="1" rowspan="1">Metronidazole</td><td colspan="1" rowspan="1">1.5% w/v</td><td colspan="1" rowspan="1">No Interference</td></tr><tr><td colspan="1" rowspan="1">Mineral oil</td><td colspan="1" rowspan="1">50% v/v</td><td colspan="1" rowspan="1">No Interference</td></tr><tr><td colspan="1" rowspan="1">Naproxen sodium</td><td colspan="1" rowspan="1">0.7% w/v</td><td colspan="1" rowspan="1">No Interference</td></tr><tr><td colspan="1" rowspan="6">Nonoxynol-9</td><td colspan="1" rowspan="1">1.2% v/v</td><td colspan="1" rowspan="1">Interference</td></tr><tr><td colspan="1" rowspan="1">0.6% v/v</td><td colspan="1" rowspan="1">Interference</td></tr><tr><td colspan="1" rowspan="1">0.3% v/v</td><td colspan="1" rowspan="1">Interference</td></tr><tr><td colspan="1" rowspan="1">0.15% v/v</td><td colspan="1" rowspan="1">Interference</td></tr><tr><td colspan="1" rowspan="1">0.075% v/v</td><td colspan="1" rowspan="1">Interference</td></tr><tr><td colspan="1" rowspan="1">0.02% v/v</td><td colspan="1" rowspan="1">No Interference</td></tr><tr><td colspan="1" rowspan="1">Nystatin</td><td colspan="1" rowspan="1">10000 USP units/mL</td><td colspan="1" rowspan="1">No Interference</td></tr><tr><td colspan="1" rowspan="1">Phenylephrine hydrochloride</td><td colspan="1" rowspan="1">0.075% w/v</td><td colspan="1" rowspan="1">No Interference</td></tr><tr><td colspan="1" rowspan="1">Sodium phosphate</td><td colspan="1" rowspan="1">5% w/v</td><td colspan="1" rowspan="1">No Interference</td></tr><tr><td colspan="1" rowspan="1">Vaccine components</td><td colspan="1" rowspan="1">mponents</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="3">Rotavirus reassortant WC3:2-5, R574(9) -VR 2195</td><td colspan="1" rowspan="1">8.89 x 10-3 TCID50/mL</td><td colspan="1" rowspan="1">Interference</td></tr><tr><td colspan="1" rowspan="1">8.89 x 10-4 TCID50/mL</td><td colspan="1" rowspan="1">Interference</td></tr><tr><td colspan="1" rowspan="1">8.89 x 10-5 TCID50/mL</td><td colspan="1" rowspan="1">No Interference</td></tr><tr><td colspan="1" rowspan="3">Rotavirus reassortant WI79-4,9 - VR 2415</td><td colspan="1" rowspan="1">1.10 x 102 pfu/mL</td><td colspan="1" rowspan="1">Interference</td></tr><tr><td colspan="1" rowspan="1">1.10 x 101 pfu/mL</td><td colspan="1" rowspan="1">Interference</td></tr><tr><td colspan="1" rowspan="1">1.10 pfu/mL</td><td colspan="1" rowspan="1">No Interference</td></tr><tr><td colspan="1" rowspan="1">Technique-specific Substances, Transport Media</td><td colspan="1" rowspan="1">Technique-specific Substances, Transport Media</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Bleach</td><td colspan="1" rowspan="1">0.5% v/v</td><td colspan="1" rowspan="1">No Interference</td></tr><tr><td colspan="1" rowspan="1">Ethanol</td><td colspan="1" rowspan="1">0.2% v/v</td><td colspan="1" rowspan="1">No Interference</td></tr><tr><td colspan="1" rowspan="1">Puritan Fecal Opti-Swab Collection &amp;Transport System with Cary-BlairMedium*</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">No Interference</td></tr><tr><td colspan="1" rowspan="1">PurSafe DNA/RNA Preservative*</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">No Interference</td></tr><tr><td colspan="1" rowspan="1">Sigma Fecal Transwab*</td><td colspan="1" rowspan="1">1 swab/2mL Cary Blair</td><td colspan="1" rowspan="1">No Interference</td></tr></table>

\*Performance not stablished for this transport media # This substance was tested by another FDA-cleared that also detected Yersinia positive signals.

# Microbial Interference

A microbial interference study was conducted to assess the inhibitory effects of select nontarget organisms on the ability to detect the QIAstat-Dx Gastrointestinal Panel 2 targets. Clinically relevant and challenging concentrations of non-target organisms were individually mixed with negative clinical stool matrix with spiked targeted pathogens at $3 \mathbf { x }$ LoD. Testing was performed in triplicate. All combinations and replicates successfully detected all the QIAstat-Dx Gastrointestinal Panel 2 targets. See Table 5.25 for a list of the non-target organisms tested and the result summary.

Table 5.25: Final highest concentration without observable inhibitory effect   

<table><tr><td rowspan=1 colspan=1>Substance tested</td><td rowspan=1 colspan=1>Concentration tested</td><td rowspan=1 colspan=1>Result</td></tr><tr><td rowspan=1 colspan=1>Non-target microorganisms</td><td rowspan=1 colspan=2>Non-target microorganisms</td></tr><tr><td rowspan=1 colspan=1>Aeromonas hydrophila</td><td rowspan=1 colspan=1>1 x 106 units/mL</td><td rowspan=1 colspan=1>No Interference</td></tr><tr><td rowspan=1 colspan=1>Bacteroides vulgatus</td><td rowspan=1 colspan=1>1 x 106 units/mL</td><td rowspan=1 colspan=1>No Interference</td></tr><tr><td rowspan=1 colspan=1>Bifidobacterium bifidum</td><td rowspan=1 colspan=1>1 x 106 units/mL</td><td rowspan=1 colspan=1>No Interference</td></tr><tr><td rowspan=1 colspan=1>Enterovirus Specie D, Serotype EV-D68</td><td rowspan=1 colspan=1>1 x 10 units/mL</td><td rowspan=1 colspan=1>No Interference</td></tr><tr><td rowspan=1 colspan=1>Non-pathogenic E. coli</td><td rowspan=1 colspan=1>1 x 106 units/mL</td><td rowspan=1 colspan=1>No Interference</td></tr><tr><td rowspan=1 colspan=1>Helicobacter pylori</td><td rowspan=1 colspan=1>1 x 106 units/mL</td><td rowspan=1 colspan=1>No Interference</td></tr><tr><td rowspan=1 colspan=1>Saccharomyces cerevisiae (deposited as S.boulardii)</td><td rowspan=1 colspan=1>1 x 10³ units/mL</td><td rowspan=1 colspan=1>No Interference</td></tr></table>

# Competitive Interference

Competitive interference was tested in a subset of pathogens. No interference was observed when evaluating competitive interference by target pathogens when two QIAstat-Dx Gastrointestinal Panel target pathogens were tested by spiking samples with one pathogen target at 3x LoD and one at 50x LoD. Results from the pathogen targets tested are provided in Table 5.26.

Table 5.26: QIAstat-Dx Gastrointestinal Panel 2 Detection Rate Results for Competitive Interference   

<table><tr><td rowspan=1 colspan=1>Mix</td><td rowspan=1 colspan=1>Pathogen</td><td rowspan=1 colspan=1>Finalconcentration(molecularunits)*</td><td rowspan=1 colspan=1>Finalconcentrationtested xLoD</td><td rowspan=1 colspan=1>Detectionrate</td><td rowspan=1 colspan=1>Target detected</td></tr><tr><td rowspan=2 colspan=1>Norovirus50x -Rotavirus3x</td><td rowspan=1 colspan=1>NorovirusGI/GII</td><td rowspan=1 colspan=1>4.5E+05copies/mL</td><td rowspan=1 colspan=1>50x</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>Yes</td></tr><tr><td rowspan=1 colspan=1>Rotavirus A</td><td rowspan=1 colspan=1>1.7E+04copies/mL</td><td rowspan=1 colspan=1>3x</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>Yes</td></tr><tr><td rowspan=2 colspan=1>Norovirus3x -Rotavirus50x</td><td rowspan=1 colspan=1>NorovirusGI/GII</td><td rowspan=1 colspan=1>2.7E+04copies/mL</td><td rowspan=1 colspan=1>3x</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>Yes</td></tr><tr><td rowspan=1 colspan=1>Rotavirus A</td><td rowspan=1 colspan=1>2.9E+05copies/mL</td><td rowspan=1 colspan=1>50x</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>Yes</td></tr><tr><td rowspan=2 colspan=1>Giardia 50x-Adenovirus3x</td><td rowspan=1 colspan=1>Giardialamblia</td><td rowspan=1 colspan=1>7.2 E+05copies/mL</td><td rowspan=1 colspan=1>50x</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>Yes</td></tr><tr><td rowspan=1 colspan=1>AdenovirusF40/F41</td><td rowspan=1 colspan=1>2.9E+03copies/mL</td><td rowspan=1 colspan=1>3x</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>Yes</td></tr></table>

\* Molecular unit titers were determined using in house developed and validated qPCR assays.

# Carryover

A carryover study was performed to evaluate the potential occurrence of crosscontamination between consecutive runs when using the QIAstat-Dx Gastrointestinal Panel 2 on the QIAstat-Dx Analyzer 1.0.

Pathogen samples of stool sample matrix, with alternating high-positive $( 1 0 ^ { 5 } – 1 0 ^ { 6 }$ organism $/ \mathrm { m L }$ ) and negative samples, were conducted on two QIAstat-Dx Analyzer 1.0.

No carryover between samples was observed in the QIAstat-Dx Gastrointestinal Panel 2, demonstrating that the system design and recommended sample handling and testing practices are effective in preventing false positive results due to carryover or crosscontamination between samples.

# Sample Stability

Sample stability testing demonstrated that the QIAstat-Dx Gastrointestinal Panel 2 assay is capable of processing samples which are stored prior to analysis under conditions typically utilized for stool specimens according to the intended use.

The results of this study support the following recommendations for storage of stool resuspended in Para-Pak C&S and FecalSwab transport media before testing:

Room temperature up to 4 days at $1 5 { - } 2 5 ^ { \circ } \mathrm { C }$ Refrigerated up to 4 days at $2 { - } 8 ^ { \circ } \mathrm { C }$

# Reproducibility

Reproducibility testing of contrived samples was performed at three test sites including one internal site (Site A) and two external sites (Site B and Site C). The study incorporated a range of potential variation introduced by sites, days, replicates, cartridge lots, operators, and QIAstat-Dx analyzers. For each site, testing was performed across 5 non-consecutive days with 6 replicates per day (leading to a total of 30 replicates per target, concentration and site), 4 QIAstat-Dx Analyzers (2 analyzers per operator and per site), and at least 2 operators on each testing day. A total of 5 sample mixes (two combined samples at 1x LoD and $3 \mathbf { x }$ LoD plus one negative sample) were prepared. For each mix, 6 replicates were tested and evaluated. shows the detection rate per target and concentration for each site of the Reproducibility study. In addition, data obtained at all three sites have been compiled to calculate the exact 2-sided $9 5 \%$ Confidence Interval by target and concentration.

Table 5.27 shows the detection rate per target and concentration for each site of the Reproducibility study. In addition, data obtained at all three sites have been compiled to calculate the exact 2-sided $9 5 \%$ Confidence Interval by target and concentration.

Table 5.27: Detection rate per target and concentration for each site of the Reproducibility study and exact 2-sided $9 5 \%$ Confidence Interval by target and concentration   

<table><tr><td colspan="1" rowspan="2">Pathogen Tested</td><td colspan="1" rowspan="2">ConcentrationTested</td><td colspan="1" rowspan="2">ExpectedResult</td><td colspan="4" rowspan="1">% Agreement with Expected Result</td></tr><tr><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">All Sites(95%ConfidenceInterval</td></tr><tr><td colspan="1" rowspan="3">Adenovirus F41ZeptoMetrix0810085CF</td><td colspan="1" rowspan="1">3x LoD</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">90/90100%(95.98 -100.00%)</td></tr><tr><td colspan="1" rowspan="1">1x LoD</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">90/90100%(95.98 -100.00%)</td></tr><tr><td colspan="1" rowspan="1">None</td><td colspan="1" rowspan="1">NotDetected</td><td colspan="1" rowspan="1">29/3096.67%</td><td colspan="1" rowspan="1">29/3096.67%</td><td colspan="1" rowspan="1">29/3096.67%</td><td colspan="1" rowspan="1">87/90a96.67%(90.98-98.9%)</td></tr><tr><td colspan="1" rowspan="3">CampylobacterZeptoMetrix0801650</td><td colspan="1" rowspan="1">3x LoD</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">90/90100%(95.98 -100.00%)</td></tr><tr><td colspan="1" rowspan="1">1x LoD</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/30b100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">90/90100%(95.98 -100.00%)</td></tr><tr><td colspan="1" rowspan="1">None</td><td colspan="1" rowspan="1">NotDetected</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">90/90100%(95.98 -100.00%)</td></tr><tr><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">All Sites(95%5%ConfidenceInterval</td></tr><tr><td colspan="1" rowspan="3">Escherichia coliPECdZeptoMetrix0801747</td><td colspan="1" rowspan="1">3x LoD</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">90/90100%(95.98 -100.00%)</td></tr><tr><td colspan="1" rowspan="1">1x LoD</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">29/3096.67%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">89/9098.89%(93.96 99.97%)</td></tr><tr><td colspan="1" rowspan="1">None</td><td colspan="1" rowspan="1">NotDetected</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">90/90100%(95.98 -100.00%)</td></tr><tr><td colspan="1" rowspan="3">EntamoebahistolyticaATCC30459</td><td colspan="1" rowspan="1">3x LoD</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">90/90100%(95.98 -100.00%)</td></tr><tr><td colspan="1" rowspan="1">1x LoD</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">29/3096.67%</td><td colspan="1" rowspan="1">89/9098.89%(93.96 99.97%)</td></tr><tr><td colspan="1" rowspan="1">None</td><td colspan="1" rowspan="1">NotDetected</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">90/90100%(95.98 -100.00%)</td></tr><tr><td colspan="1" rowspan="3">Giardia lambliabATCC30888</td><td colspan="1" rowspan="1">3x LoD</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">90/90100%(95.98 -100.00%)</td></tr><tr><td colspan="1" rowspan="1">1x LoD</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">90/90100%(95.98 -100.00%)</td></tr><tr><td colspan="1" rowspan="1">None</td><td colspan="1" rowspan="1">NotDetected</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">90/90100%(95.98 -100.00%)</td></tr><tr><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">All Sites(95%ConfidenceInterval</td></tr><tr><td colspan="1" rowspan="3">Norovirus GIIZeptoMetrix0810087CF</td><td colspan="1" rowspan="1">3x LoD</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">90/90100%(95.98 -100.00%)</td></tr><tr><td colspan="1" rowspan="1">1x LoD</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">29/3096.67%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">89/9098.89%(93.96 99.97%)</td></tr><tr><td colspan="1" rowspan="1">None</td><td colspan="1" rowspan="1">NotDetected</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">90/90100%(95.98 -100.00%)</td></tr><tr><td colspan="1" rowspan="3">Rotavirus AcZeptoMetrix0810280CF</td><td colspan="1" rowspan="1">3x LoD</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">29/3096.67%</td><td colspan="1" rowspan="1">29/3096.67%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">88/9097.8%(92.20 -99.73%)</td></tr><tr><td colspan="1" rowspan="1">1x LoD</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">23/3076.67%</td><td colspan="1" rowspan="1">26/3086.67%</td><td colspan="1" rowspan="1">12/12100%</td><td colspan="1" rowspan="1">61/7284.7%(74.31 92.12%)</td></tr><tr><td colspan="1" rowspan="1">None</td><td colspan="1" rowspan="1">NotDetected</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">90/90100%(95.98 -100.00%)</td></tr><tr><td colspan="1" rowspan="3">Escherichia coli(STEC) 0157:H7dZeptoMetrix0801622</td><td colspan="1" rowspan="1">3x LoD</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">90/90100%(95.98 -100.00%)</td></tr><tr><td colspan="1" rowspan="1">1x LoD</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">29/3096.67%</td><td colspan="1" rowspan="1">89/9098.89%(93.96 99.97%)</td></tr><tr><td colspan="1" rowspan="1">None</td><td colspan="1" rowspan="1">NotDetected</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">90/90100%(95.98 -100.00%)</td></tr><tr><td colspan="1" rowspan="3">Escherichia coli(STEC) stx1/stx2dZeptoMetrix0801622</td><td colspan="1" rowspan="1">3x LoD</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">90/90100%(95.98 -100.00%)</td></tr><tr><td colspan="1" rowspan="1">1x LoD</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">90/90100%(95.98 -100.00%)</td></tr><tr><td colspan="1" rowspan="1">None</td><td colspan="1" rowspan="1">NotDetected</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">90/90100%(95.98 -100.00%)</td></tr><tr><td colspan="1" rowspan="3">SalmonellaentericaZeptoMetrix0801437</td><td colspan="1" rowspan="1">3x LoD</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">90/90100%(95.98 -100.00%)</td></tr><tr><td colspan="1" rowspan="1">1x LoD</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">29/3096.67%</td><td colspan="1" rowspan="1">29/3096.67%</td><td colspan="1" rowspan="1">88/9097.78%(92.20 99.73%)</td></tr><tr><td colspan="1" rowspan="1">None</td><td colspan="1" rowspan="1">NotDetected</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">90/90100%(95.98 100.00%)</td></tr><tr><td colspan="1" rowspan="3">YersiniaenterocoliticaZeptoMetrix0801734</td><td colspan="1" rowspan="1">3x LoD</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">90/90100%(95.98 -100.00%)</td></tr><tr><td colspan="1" rowspan="1">1x LoD</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">90/90100%(95.98 -100.00%)</td></tr><tr><td colspan="1" rowspan="1">None</td><td colspan="1" rowspan="1">NotDetected</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">90/90100%(95.98 100.00%)</td></tr></table>

a Three (3) Adenovirus F40/41 false positives were observed when testing negative sample. Retesting of the three samples resulted in the expected negative results. b One (1) Giardia lamblia false positive was observed when testing a positive sample not containing the pathogen. Repeat testing of this sample resulted in the expected negative result. c The Reproducibility study was fully re-tested for Rotavirus A with a new sample set due to an unexpected number of false negatives for Rotavirus A at the 1x LoD concentration. This was observed during an interim data evaluation (61/72, $8 4 . 7 \% )$ ) and was attributed to the sample manufacture while unrelated to the study workflow variables (operator, lot, day, instrument and site). Test runs derived from Rotavirus A new sample set resulted in 90/90 ( $100 \%$ ; $9 5 . 9 8 \substack { - 1 0 0 \% }$ CI) for the 3x LoD and 89/90 $9 8 . 8 9 \%$ ; $9 3 . 9 6 \substack { - 9 9 . 9 7 \% }$ CI) for the 1x LoD. During this testing, one (1) Campylobacter false positive was observed. Retesting of this sample resulted in the expected negative result.

dTarget only applicable for Para-Pak C&S samples.

During the reproducibility study, potential variation introduced by sites, days, replicates, cartridge lots, operators, and QIAstat-Dx analyzers was analyzed showing no significant contribution to variability (Standard Deviation and Coefficient of Variation values below 1 and $5 \%$ , respectively) caused by any of the assessed variables (Table 5.28).

Table 5.28: Variability attributable to each variable according to ANOVA model for QIAstat-Dx Gastrointestinal Panel 2.   

<table><tr><td rowspan=1 colspan=5>EY</td><td rowspan=1 colspan=7>Variance component (SD, %CV*)</td></tr><tr><td rowspan=1 colspan=1>Pathogen</td><td rowspan=1 colspan=1>Concentrationtested</td><td rowspan=1 colspan=1>Positive(n)</td><td rowspan=1 colspan=1>Negative(n)</td><td rowspan=1 colspan=1>Mean</td><td rowspan=1 colspan=1>BetweenSite</td><td rowspan=1 colspan=1>BetweenLot</td><td rowspan=1 colspan=1>BetweenDay</td><td rowspan=1 colspan=1>BetweenOperator</td><td rowspan=1 colspan=1>BetweenInstrument</td><td rowspan=1 colspan=1>BetweenReplicate</td><td rowspan=1 colspan=1>Total**</td></tr><tr><td rowspan=2 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>1x LoD</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>34.15</td><td rowspan=1 colspan=1>(0.2979,0.87%)</td><td rowspan=1 colspan=1>(0.1630,0.48%)</td><td rowspan=1 colspan=1>(0.1135,0.33%)</td><td rowspan=1 colspan=1>(0.0000,0.00%</td><td rowspan=1 colspan=1>(0.1883,0.55%)</td><td rowspan=1 colspan=1>(0.6153,1.80%)</td><td rowspan=1 colspan=1>(0.7067,2.07%)</td></tr><tr><td rowspan=1 colspan=1>3x LoD</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>32.14</td><td rowspan=1 colspan=1>(0.0000,0.00%</td><td rowspan=1 colspan=1>(0.0741,0.23%</td><td rowspan=1 colspan=1>(0.0000,000%)</td><td rowspan=1 colspan=1>(0.0000,0.00%)</td><td rowspan=1 colspan=1>(0.0000,0.00%)</td><td rowspan=1 colspan=1>(0.3716,1.16%</td><td rowspan=1 colspan=1>(0.3766,1.17%</td></tr><tr><td rowspan=2 colspan=1>Campylobacter</td><td rowspan=1 colspan=1>1x LoD</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>32.64</td><td rowspan=1 colspan=1>(0.1289,0.39%</td><td rowspan=1 colspan=1>(0.0000,000%</td><td rowspan=1 colspan=1>(0.0200,006%</td><td rowspan=1 colspan=1>(0.0000,000%</td><td rowspan=1 colspan=1>(0.0661,0.20%</td><td rowspan=1 colspan=1>(0.4232,1.30%</td><td rowspan=1 colspan=1>(0.4411,1.35%</td></tr><tr><td rowspan=1 colspan=1>3x LoD</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>31.00</td><td rowspan=1 colspan=1>(0.0397,0.13%)</td><td rowspan=1 colspan=1>(0.1446,0.47%)</td><td rowspan=1 colspan=1>(0.0806,0.26%)</td><td rowspan=1 colspan=1>(0.0000,0.00%)</td><td rowspan=1 colspan=1>(0.1488,0.48%)</td><td rowspan=1 colspan=1>(0.3420,1.10%)</td><td rowspan=1 colspan=1>(0.3855,1.24%)</td></tr><tr><td rowspan=2 colspan=1>EPEC</td><td rowspan=1 colspan=1>1x LoD</td><td rowspan=1 colspan=1>89</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>34.24</td><td rowspan=1 colspan=1>(0.4422,1.29%)</td><td rowspan=1 colspan=1>(0.2000,0.58%)</td><td rowspan=1 colspan=1>(0.0000,0.00%)</td><td rowspan=1 colspan=1>(0.1599,0.47%)</td><td rowspan=1 colspan=1>(0.1044,0.30%</td><td rowspan=1 colspan=1>(0.6699,1.96%)</td><td rowspan=1 colspan=1>(0.7927,2.31%)</td></tr><tr><td rowspan=1 colspan=1>3x LoD</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>32.49</td><td rowspan=1 colspan=1>(0.1053,0.32%</td><td rowspan=1 colspan=1>(0.1429,0.44%)</td><td rowspan=1 colspan=1>(0.0000,0.00%</td><td rowspan=1 colspan=1>(0.2081,0.64%)</td><td rowspan=1 colspan=1>(0.0000,0.00%</td><td rowspan=1 colspan=1>(0.5238,1.61%)</td><td rowspan=1 colspan=1>(0.5710,1.76%</td></tr><tr><td rowspan=2 colspan=1>Entamoeba</td><td rowspan=1 colspan=1>1x LoD</td><td rowspan=1 colspan=1>89</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>33.01</td><td rowspan=1 colspan=1>(0.0000,0.00%)</td><td rowspan=1 colspan=1>(0.1349,0.41%)</td><td rowspan=1 colspan=1>(0.0000,0.00%)</td><td rowspan=1 colspan=1>(0.1175,0.6%</td><td rowspan=1 colspan=1>(0.0000,0.00%)</td><td rowspan=1 colspan=1>(0.3182,0.96%)</td><td rowspan=1 colspan=1>(0.3456,1.05%)</td></tr><tr><td rowspan=1 colspan=1>3x LoD</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>31.34</td><td rowspan=1 colspan=1>(0.1826,0.58%)</td><td rowspan=1 colspan=1>(0.1265,0.40%)</td><td rowspan=1 colspan=1>(0.0000,0.00%)</td><td rowspan=1 colspan=1>(0.0000,0.00%)</td><td rowspan=1 colspan=1>(0.0906,0.29%)</td><td rowspan=1 colspan=1>(0.4000,1.28%)</td><td rowspan=1 colspan=1>(0.4449,1.42%)</td></tr><tr><td rowspan=2 colspan=1>Giardia</td><td rowspan=1 colspan=1>1x LoD</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>28.65</td><td rowspan=1 colspan=1>(0.2341,0.82%)</td><td rowspan=1 colspan=1>(0.0000,0.00%)</td><td rowspan=1 colspan=1>(0.0000,0.00%)</td><td rowspan=1 colspan=1>(0.0000,0.00%)</td><td rowspan=1 colspan=1>(0.1737),0.61%)</td><td rowspan=1 colspan=1>(0.5386,1.88%)</td><td rowspan=1 colspan=1>(0.5953,2.08%)</td></tr><tr><td rowspan=1 colspan=1>3x LoD</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>26.68</td><td rowspan=1 colspan=1>(0.0840,0.31%</td><td rowspan=1 colspan=1>(0.0000,000%</td><td rowspan=1 colspan=1>(0.0000,000%</td><td rowspan=1 colspan=1>(0.1025,0.38%</td><td rowspan=1 colspan=1>(0.0857,0.32%</td><td rowspan=1 colspan=1>(0.3531,1.32%</td><td rowspan=1 colspan=1>(0.3798,1.42%)</td></tr><tr><td rowspan=2 colspan=1>Norovirus GII</td><td rowspan=1 colspan=1>1x LoD</td><td rowspan=1 colspan=1>89</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>33.53</td><td rowspan=1 colspan=1>(0.1346,0.40%</td><td rowspan=1 colspan=1>(0.3090,0.92%)</td><td rowspan=1 colspan=1>(0.1111,0.33%</td><td rowspan=1 colspan=1>(0.0000,0.00%)</td><td rowspan=1 colspan=1>(0.0431,0.13%</td><td rowspan=1 colspan=1>(0.4102,1.22%)</td><td rowspan=1 colspan=1>(0.5083,1.52%)</td></tr><tr><td rowspan=1 colspan=1>3x LoD</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>32.42</td><td rowspan=1 colspan=1>(0.1742,0.54%)</td><td rowspan=1 colspan=1>(0.3405,1.05%)</td><td rowspan=1 colspan=1>(0.2895,0.89%)</td><td rowspan=1 colspan=1>(0.0632,0.19%</td><td rowspan=1 colspan=1>(0.1972,0.61%</td><td rowspan=1 colspan=1>(0.4966,1.53%)</td><td rowspan=1 colspan=1>(0.6721,2.07%)</td></tr><tr><td rowspan=2 colspan=1>Rotavirus ***</td><td rowspan=1 colspan=1>1x LoD</td><td rowspan=1 colspan=1>61</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>36.71</td><td rowspan=1 colspan=1>(0.0000,0.00%)</td><td rowspan=1 colspan=1>(0.1487,0.41%)</td><td rowspan=1 colspan=1>(0.0000,0.00%)</td><td rowspan=1 colspan=1>(0.1824,0.50%)</td><td rowspan=1 colspan=1>(0.0000,0.00%)</td><td rowspan=1 colspan=1>(0.9172,2.50%)</td><td rowspan=1 colspan=1>(0.9356,2.55%)</td></tr><tr><td rowspan=1 colspan=1>3x LoD</td><td rowspan=1 colspan=1>88</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>35.95</td><td rowspan=1 colspan=1>(0.3468,096%</td><td rowspan=1 colspan=1>(0.4463,1.24%)</td><td rowspan=1 colspan=1>(0.1231,0.34%</td><td rowspan=1 colspan=1>(0.1295,0.36%)</td><td rowspan=1 colspan=1>(0.1132,0.31%</td><td rowspan=1 colspan=1>(1.2073,3.36%</td><td rowspan=1 colspan=1>(1.3014,3.62%)</td></tr><tr><td rowspan=2 colspan=1>STEC_0157</td><td rowspan=1 colspan=1>1x LoD</td><td rowspan=1 colspan=1>89</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>33.83</td><td rowspan=1 colspan=1>(0.2218,0.66%</td><td rowspan=1 colspan=1>(0.3083,0.91%</td><td rowspan=1 colspan=1>(0.0000,0.00%)</td><td rowspan=1 colspan=1>(0.0204,0.06%)</td><td rowspan=1 colspan=1>(0.0000,0.00%</td><td rowspan=1 colspan=1>(0.5455,1.61%)</td><td rowspan=1 colspan=1>(0.6295,1.86%)</td></tr><tr><td rowspan=1 colspan=1>3x LoD</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>32.22</td><td rowspan=1 colspan=1>(0.2431,0.75%</td><td rowspan=1 colspan=1>(0.2146,0.67%</td><td rowspan=1 colspan=1>(0.0000,0.00%</td><td rowspan=1 colspan=1>(0.0000,0.00%)</td><td rowspan=1 colspan=1>(0.2051,0.64%)</td><td rowspan=1 colspan=1>(0.4625,1.44%)</td><td rowspan=1 colspan=1>(0.5633,1.75%)</td></tr><tr><td rowspan=2 colspan=1>STEC_stx1</td><td rowspan=1 colspan=1>1x LoD</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>34.07</td><td rowspan=1 colspan=1>(0.2297,0.67%)</td><td rowspan=1 colspan=1>(0.1214,0.36%</td><td rowspan=1 colspan=1>(0.1281,0.38%)</td><td rowspan=1 colspan=1>(0.0000,0.00%)</td><td rowspan=1 colspan=1>(0.1170,0.34%)</td><td rowspan=1 colspan=1>(0.4206,1.23%)</td><td rowspan=1 colspan=1>(0.4991,1.46%)</td></tr><tr><td rowspan=1 colspan=1>3x LoD</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>32.54</td><td rowspan=1 colspan=1>(0.2380,0.73%)</td><td rowspan=1 colspan=1>(0.2222,0.68%)</td><td rowspan=1 colspan=1>(0.0802,0.25%)</td><td rowspan=1 colspan=1>(0.0000,0.00%)</td><td rowspan=1 colspan=1>(0.2000,0.61%)</td><td rowspan=1 colspan=1>(0.4919,1.51%)</td><td rowspan=1 colspan=1>(0.5887,1.81%)</td></tr><tr><td rowspan=2 colspan=1>STEC_stx2</td><td rowspan=1 colspan=1>1x LoD</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>34.11</td><td rowspan=1 colspan=1>(0.0620,0.18%</td><td rowspan=1 colspan=1>(0.0979,0.29%</td><td rowspan=1 colspan=1>(0.0310,0.09%</td><td rowspan=1 colspan=1>(0.0000,0.00%</td><td rowspan=1 colspan=1>(0.0000,000%</td><td rowspan=1 colspan=1>(0.3987,1.17%)</td><td rowspan=1 colspan=1>(0.4109,1.20%)</td></tr><tr><td rowspan=1 colspan=1>3x LoD</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>32.69</td><td rowspan=1 colspan=1>(0.1173,0.36%</td><td rowspan=1 colspan=1>(0.2599,0.80%)</td><td rowspan=1 colspan=1>(0.0000,0.00%)</td><td rowspan=1 colspan=1>(0.0000,0.00%)</td><td rowspan=1 colspan=1>(0.1316,0.40%)</td><td rowspan=1 colspan=1>(0.4131,1.26%)</td><td rowspan=1 colspan=1>(0.4821,1.47%)</td></tr><tr><td rowspan=2 colspan=1>Salmonella</td><td rowspan=1 colspan=1>1x LoD</td><td rowspan=1 colspan=1>88</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>34.53</td><td rowspan=1 colspan=1>(0.2888,0.4%</td><td rowspan=1 colspan=1>(0.2631,0.76%</td><td rowspan=1 colspan=1>(0.0000,0.00%)</td><td rowspan=1 colspan=1>(0.0000,000%</td><td rowspan=1 colspan=1>(0.0000,000%</td><td rowspan=1 colspan=1>(0.4880,1.41%)</td><td rowspan=1 colspan=1>(0.5857,1.70%</td></tr><tr><td rowspan=1 colspan=1>3x LoD</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>33.44</td><td rowspan=1 colspan=1>(0.0279,0.08%)</td><td rowspan=1 colspan=1>(0.2100,0.63%)</td><td rowspan=1 colspan=1>(0.0000,0.00%)</td><td rowspan=1 colspan=1>(0.1814,0.54%)</td><td rowspan=1 colspan=1>(0.17),0.35%)</td><td rowspan=1 colspan=1>(0.4088,1.22%)</td><td rowspan=1 colspan=1>(0.4715,1.41%)</td></tr><tr><td rowspan=2 colspan=1>Yersinia</td><td rowspan=1 colspan=1>1x LoD</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>34.53</td><td rowspan=1 colspan=1>(0.1616,0.47%)</td><td rowspan=1 colspan=1>(0.1688,0.49%)</td><td rowspan=1 colspan=1>(0.0564,0.16%)</td><td rowspan=1 colspan=1>(0.0000,0.00%)</td><td rowspan=1 colspan=1>(0.1036,0.30%)</td><td rowspan=1 colspan=1>(0.5172,1.50%)</td><td rowspan=1 colspan=1>(0.5637,1.63%</td></tr><tr><td rowspan=1 colspan=1>3x LoD</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>33.02</td><td rowspan=1 colspan=1>(0.0503,015%</td><td rowspan=1 colspan=1>(0.2129,0.64%</td><td rowspan=1 colspan=1>(0.0370,0.11%</td><td rowspan=1 colspan=1>(0.0000,0.00%)</td><td rowspan=1 colspan=1>(0.08001,0.2%</td><td rowspan=1 colspan=1>(0.3434,1.04%%</td><td rowspan=1 colspan=1>(0.3951,1.20%)</td></tr></table>

\* SD is first figure presented, subsequent figure is CV (Coefficient of variation presented as a percentage $( \% )$ $^ { * * }$ TOTAL means total variance observed across the study \*\*\* Results in table correspond to the original sample set analyzed. Results from the new dataset were:

<table><tr><td colspan="6"></td><td colspan="7">Variance component (SD, %CV*)</td></tr><tr><td>Pathogen</td><td>Concentration</td><td>Positive</td><td>Negative</td><td>Mean</td><td>Between</td><td>Between</td><td>Between</td><td>Between Operator</td><td>Between Instrument</td><td>Between Replicate</td><td>Total**</td></tr><tr><td rowspan="3">Rotavirus</td><td>tested</td><td>(n)</td><td>(n)</td><td>34.86</td><td>Site (0.1706,</td><td>Lot (0.0000,</td><td>Day (0.3458,</td><td>(0.1207,</td><td>(0.3224,</td><td>(0.9814,</td><td>(1.0891,</td></tr><tr><td>1x LoD</td><td>89</td><td>1</td><td></td><td>0.49%)</td><td>0.00%</td><td>0.99%</td><td>0.35%</td><td>0.92%</td><td>2.82%)</td><td>3.12%)</td></tr><tr><td>3x LoD</td><td>90</td><td>0</td><td>32.42</td><td>(0.17442, 0.54%</td><td>(0.3405, 1.05%</td><td>(0.2895, 0.89%</td><td>(0.0632, 0.19%</td><td>(0.1972, 0.6%</td><td>(0.4966, 1.53%</td><td>(0.6721, 2.07%</td></tr></table>

# Repeatability

A repeatability study was conducted on the QIAstat-Dx Analyzer 1.0 instruments using a set of samples composed of low-concentrated analytes spiked into stool matrix (3x LoD and 1x LoD) and negative stool samples. QIAstat-Dx Gastrointestinal Panel 2 detected pathogens included in the positive samples were Adenovirus, Campylobacter, Enteropathogenic $E$ . coli (EPEC), Entamoeba histolytica, Giardia lamblia, Norovirus GII, Rotavirus, $E$ . coli O157, STEC stx1/stx2, Salmonella enterica, and Yersinia enterocolitica. Each sample was tested with the same instrument over 12 days. In total, 60 replicates of 1x LoD and 60 replicates of 3x LoD per each of the tested targets and 60 replicates of negative samples were run. Overall results showed a $9 3 . 3 3 \substack { - 1 0 0 . 0 0 - \% }$ and $9 5 . 0 0 \substack { - 1 0 0 . 0 0 \% }$ detection rate for 1x LoD and 3x LoD samples, respectively. Negative samples showed $100 \%$ of negative calls for all panel analytes.

# Performance Characteristics - Clinical Studies

# Expected values

In the prospective clinical performance evaluation of the QIAstat-Dx Gastrointestinal Panel 2, 1939 eligible specimens (stool preserved in Para-Pak C&S (Meridian Bioscience) or FecalSwab (COPAN)) were collected and tested at 13 clinical sites across 5 countries (4 sites in Europe and 9 sites in USA) from May 2021 to July 2021. The number and percentage of positive results as determined by the QIAstat-Dx Gastrointestinal Panel 2, stratified by age group, are presented in Table 5.29. Overall, the QIAstat-Dx Gastrointestinal Panel 2 detected at least 1 organism in in $1 7 . 4 \%$ (213/1222) and $2 3 . 8 \%$ (171/717) of the prospectively collected stool specimens in FecalSwab and Para-Pak C&S, respectively (Table 5.29).

Table 5.29: Expected Values Summary by Age Group for the Prospective Clinical study as determined by the QIAstat-Dx Gastrointestinal Panel 2   

<table><tr><td rowspan=1 colspan=1>Pathogen</td><td rowspan=1 colspan=1>Medium Brand</td><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>0-5 years</td><td rowspan=1 colspan=1>6-21 years</td><td rowspan=1 colspan=1>22-49 years</td><td rowspan=1 colspan=1>50+ years</td><td rowspan=1 colspan=1>NotReported</td></tr><tr><td rowspan=1 colspan=8>Viruses</td></tr><tr><td rowspan=2 colspan=1>Adenovirus F40/F41</td><td rowspan=1 colspan=1>FecalSwab</td><td rowspan=1 colspan=1>5 (0.4%)</td><td rowspan=1 colspan=1>3 (1.7%)</td><td rowspan=1 colspan=1>2 (1.7%)</td><td rowspan=1 colspan=1>0 (0.0%)</td><td rowspan=1 colspan=1>0 (0.0%)</td><td rowspan=1 colspan=1>0 (0.0%)</td></tr><tr><td rowspan=1 colspan=1>Para-Pak C&amp;S</td><td rowspan=1 colspan=1>2 (0.3%)</td><td rowspan=1 colspan=1>1 (3.2%)</td><td rowspan=1 colspan=1>0 (0.0%)</td><td rowspan=1 colspan=1>0 (0.0%)</td><td rowspan=1 colspan=1>1 (0.2%)</td><td rowspan=1 colspan=1>0 (0.0%)</td></tr><tr><td rowspan=2 colspan=1>Astrovirus</td><td rowspan=1 colspan=1>FecalSwab</td><td rowspan=1 colspan=1>3 (0.2%)</td><td rowspan=1 colspan=1>3 (1.6%)</td><td rowspan=1 colspan=1>0 (0.0%)</td><td rowspan=1 colspan=1>0 (0.0%)</td><td rowspan=1 colspan=1>0 (0.0%)</td><td rowspan=1 colspan=1>0 (0.0%)</td></tr><tr><td rowspan=1 colspan=1>Para-Pak C&amp;S</td><td rowspan=1 colspan=1>6 (0.8%)</td><td rowspan=1 colspan=1>2 (6.5%)</td><td rowspan=1 colspan=1>0 (0.0%)</td><td rowspan=1 colspan=1>3 (1.4%)</td><td rowspan=1 colspan=1>1 (0.2%)</td><td rowspan=1 colspan=1>0 (0.0%)</td></tr><tr><td rowspan=2 colspan=1>Norovirus GI/GII</td><td rowspan=1 colspan=1>FecalSwab</td><td rowspan=1 colspan=1>43 (3.5%)</td><td rowspan=1 colspan=1>22 (12.1%)</td><td rowspan=1 colspan=1>1 (0.8%)</td><td rowspan=1 colspan=1>14 (4.8%)</td><td rowspan=1 colspan=1>6 (1.0%)</td><td rowspan=1 colspan=1>0 (0.0%)</td></tr><tr><td rowspan=1 colspan=1>Para-Pak C&amp;S</td><td rowspan=1 colspan=1>16 (2.3%)</td><td rowspan=1 colspan=1>3 (9.7%)</td><td rowspan=1 colspan=1>1 (2.8%)</td><td rowspan=1 colspan=1>3 (1.4%)</td><td rowspan=1 colspan=1>9 (2.2%)</td><td rowspan=1 colspan=1>0 (0.0%)</td></tr><tr><td rowspan=2 colspan=1>Rotavirus A</td><td rowspan=1 colspan=1>FecalSwab</td><td rowspan=1 colspan=1>23 (1.9%)</td><td rowspan=1 colspan=1>13 (7.1%)</td><td rowspan=1 colspan=1>2 (1.7%)</td><td rowspan=1 colspan=1>7 (2.4%)</td><td rowspan=1 colspan=1>1 (0.2%)</td><td rowspan=1 colspan=1>0 (0.0%)</td></tr><tr><td rowspan=1 colspan=1>Para-Pak C&amp;S</td><td rowspan=1 colspan=1>4 (0.6%)</td><td rowspan=1 colspan=1>2 (6.5%)</td><td rowspan=1 colspan=1>0 (0.0%)</td><td rowspan=1 colspan=1>0 (0.0%)</td><td rowspan=1 colspan=1>2 (0.5%)</td><td rowspan=1 colspan=1>0 (0.0%)</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=7>Bacteria</td></tr><tr><td rowspan=2 colspan=1>Campylobacter</td><td rowspan=1 colspan=1>FecalSwab</td><td rowspan=1 colspan=1>69 (5.6%)</td><td rowspan=1 colspan=1>25 (13.7%)</td><td rowspan=1 colspan=1>7(5.8%)</td><td rowspan=1 colspan=1>17 (5.9%)</td><td rowspan=1 colspan=1>20 (3.2%)</td><td rowspan=1 colspan=1>0 (0.0%)</td></tr><tr><td rowspan=1 colspan=1>Para-Pak C&amp;S</td><td rowspan=1 colspan=1>30 (4.2%)</td><td rowspan=1 colspan=1>2 (6.5%)</td><td rowspan=1 colspan=1>0 (0.0%)</td><td rowspan=1 colspan=1>10 (4.7%)</td><td rowspan=1 colspan=1>18 (4.3%)</td><td rowspan=1 colspan=1>0 (0.0%)</td></tr><tr><td rowspan=2 colspan=1>Plesiomonas shigelloides</td><td rowspan=1 colspan=1>FecalSwab</td><td rowspan=1 colspan=1>2 (0.2%)</td><td rowspan=1 colspan=1>0 (0.0%)</td><td rowspan=1 colspan=1>0 (0.0%)</td><td rowspan=1 colspan=1>2 (0.7%)</td><td rowspan=1 colspan=1>0 (0.0%)</td><td rowspan=1 colspan=1>0 (0.0%)</td></tr><tr><td rowspan=1 colspan=1>Para-Pak C&amp;S</td><td rowspan=1 colspan=1>7 (1.0%)</td><td rowspan=1 colspan=1>1 (3.2%)</td><td rowspan=1 colspan=1>0 (0.0%)</td><td rowspan=1 colspan=1>4 (1.9%)</td><td rowspan=1 colspan=1>2 (0.5%)</td><td rowspan=1 colspan=1>0 (0.0%)</td></tr><tr><td rowspan=2 colspan=1>Salmonella</td><td rowspan=1 colspan=1>FecalSwab</td><td rowspan=1 colspan=1>14 (1.1%)</td><td rowspan=1 colspan=1>5 (2.7%)</td><td rowspan=1 colspan=1>4 (3.3%)</td><td rowspan=1 colspan=1>3 (1.0%)</td><td rowspan=1 colspan=1>2 (0.3%)</td><td rowspan=1 colspan=1>0 (0.0%)</td></tr><tr><td rowspan=1 colspan=1>Para-Pak C&amp;S</td><td rowspan=1 colspan=1>17 (2.4%)</td><td rowspan=1 colspan=1>4 (12.9%)</td><td rowspan=1 colspan=1>0 (0.0%)</td><td rowspan=1 colspan=1>3 (1.4%)</td><td rowspan=1 colspan=1>10 (2.4%)</td><td rowspan=1 colspan=1>0 (0.0%)</td></tr><tr><td rowspan=2 colspan=1>Yersinia enterocolitica</td><td rowspan=1 colspan=1>FecalSwab</td><td rowspan=1 colspan=1>22 (1.8%)</td><td rowspan=1 colspan=1>3 (1.6%)</td><td rowspan=1 colspan=1>2 (1.7%)</td><td rowspan=1 colspan=1>9 (3.1%)</td><td rowspan=1 colspan=1>8 (1.3%)</td><td rowspan=1 colspan=1>0 (0.0%)</td></tr><tr><td rowspan=1 colspan=1>Para-Pak C&amp;S</td><td rowspan=1 colspan=1>8 (1.1%)</td><td rowspan=1 colspan=1>0 (0.0%)</td><td rowspan=1 colspan=1>0 (0.0%)</td><td rowspan=1 colspan=1>4 (1.9%)</td><td rowspan=1 colspan=1>4 (1.0%)</td><td rowspan=1 colspan=1>0 (0.0%)</td></tr><tr><td rowspan=1 colspan=2></td><td rowspan=1 colspan=6>Diarrheagenic E. coli/Shigella</td></tr><tr><td rowspan=1 colspan=1>Enteropathogenic E. coli EPEC)</td><td rowspan=1 colspan=1>Para-Pak C&amp;S</td><td rowspan=1 colspan=1>56 (7.9%)</td><td rowspan=1 colspan=1>9 (29.0%)</td><td rowspan=1 colspan=1>2 (5.6%)</td><td rowspan=1 colspan=1>18 (8.4%)</td><td rowspan=1 colspan=1>27 (6.5%)</td><td rowspan=1 colspan=1>0 (0.0%)</td></tr><tr><td rowspan=2 colspan=1>Enterotoxigenic E. coli(ETEC) It/st</td><td rowspan=1 colspan=1>FecalSwab</td><td rowspan=1 colspan=1>18 (1.5%)</td><td rowspan=1 colspan=1>2 (1.1%)</td><td rowspan=1 colspan=1>2 (1.7%)</td><td rowspan=1 colspan=1>11 (3.8%)</td><td rowspan=1 colspan=1>3 (0.5%)</td><td rowspan=1 colspan=1>0 (0.0%)</td></tr><tr><td rowspan=1 colspan=1>Para-Pak C&amp;S</td><td rowspan=1 colspan=1>17 (2.4%)</td><td rowspan=1 colspan=1>1 (3.2%)</td><td rowspan=1 colspan=1>0 (0.0%)</td><td rowspan=1 colspan=1>7 (3.3%)</td><td rowspan=1 colspan=1>9 (2.2%)</td><td rowspan=1 colspan=1>0 (0.0%)</td></tr><tr><td rowspan=1 colspan=1>Shiga-like toxin E. coli(STEC) stx1/stx2</td><td rowspan=1 colspan=1>Para-Pak C&amp;S</td><td rowspan=1 colspan=1>9 (1.3%)</td><td rowspan=1 colspan=1>0 (0.0%)</td><td rowspan=1 colspan=1>0 (0.0%)</td><td rowspan=1 colspan=1>6 (2.8%)</td><td rowspan=1 colspan=1>3 (0.7%)</td><td rowspan=1 colspan=1>0 (0.0%)</td></tr><tr><td rowspan=1 colspan=1>E. coli 0157</td><td rowspan=1 colspan=1>Para-Pak C&amp;S</td><td rowspan=1 colspan=1>0 (0.0%)</td><td rowspan=1 colspan=1>0 (0.0%)</td><td rowspan=1 colspan=1>0 (0.0%)</td><td rowspan=1 colspan=1>0 (0.0%)</td><td rowspan=1 colspan=1>0 (0.0%)</td><td rowspan=1 colspan=1>0 (0.0%)</td></tr><tr><td rowspan=2 colspan=1>Shigella/EnteroinvasiveE. colili (EIEC)</td><td rowspan=1 colspan=1>FecalSwab</td><td rowspan=1 colspan=1>10 (0.8%)</td><td rowspan=1 colspan=1>1 (0.5%)</td><td rowspan=1 colspan=1>0 (0.0%)</td><td rowspan=1 colspan=1>6 (2.1%)</td><td rowspan=1 colspan=1>3 (0.5%)</td><td rowspan=1 colspan=1>0 (0.0%)</td></tr><tr><td rowspan=1 colspan=1>Para-Pak C&amp;S</td><td rowspan=1 colspan=1>3 (0.4%)</td><td rowspan=1 colspan=1>0 (0.0%)</td><td rowspan=1 colspan=1>0 (0.0%)</td><td rowspan=1 colspan=1>1 (0.5%)</td><td rowspan=1 colspan=1>2 (0.5%)</td><td rowspan=1 colspan=1>0 (0.0%)</td></tr><tr><td rowspan=1 colspan=8>Parasites</td></tr><tr><td rowspan=2 colspan=1>Cryptosporidium</td><td rowspan=1 colspan=1>FecalSwab</td><td rowspan=1 colspan=1>2 (0.2%)</td><td rowspan=1 colspan=1>0 (0.0%)</td><td rowspan=1 colspan=1>1 (0.8%)</td><td rowspan=1 colspan=1>1 (0.3%)</td><td rowspan=1 colspan=1>0 (0.0%)</td><td rowspan=1 colspan=1>0 (0.0%)</td></tr><tr><td rowspan=1 colspan=1>Para-Pak C&amp;S</td><td rowspan=1 colspan=1>7 (1.0%)</td><td rowspan=1 colspan=1>0 (0.0%)</td><td rowspan=1 colspan=1>1 (2.8%)</td><td rowspan=1 colspan=1>4 (1.9%)</td><td rowspan=1 colspan=1>2 (0.5%)</td><td rowspan=1 colspan=1>0 (0.0%)</td></tr><tr><td rowspan=2 colspan=1>Cyclospora cayetanensis</td><td rowspan=1 colspan=1>FecalSwab</td><td rowspan=1 colspan=1>3 (0.2%)</td><td rowspan=1 colspan=1>0 (0.0%)</td><td rowspan=1 colspan=1>1 (0.8%)</td><td rowspan=1 colspan=1>2 (0.7%)</td><td rowspan=1 colspan=1>0 (0.0%)</td><td rowspan=1 colspan=1>0 (0.0%)</td></tr><tr><td rowspan=1 colspan=1>Para-Pak C&amp;S</td><td rowspan=1 colspan=1>18 (2.5%)</td><td rowspan=1 colspan=1>0 (0.0%)</td><td rowspan=1 colspan=1>0 (0.0%)</td><td rowspan=1 colspan=1>6 (2.8%)</td><td rowspan=1 colspan=1>12 (2.9%)</td><td rowspan=1 colspan=1>0 (0.0%)</td></tr><tr><td rowspan=2 colspan=1>Giardia lamblia</td><td rowspan=1 colspan=1>FecalSwab</td><td rowspan=1 colspan=1>15 (1.2%)</td><td rowspan=1 colspan=1>3 (1.6%)</td><td rowspan=1 colspan=1>1 (0.8%)</td><td rowspan=1 colspan=1>7 (2.4%)</td><td rowspan=1 colspan=1>4 (0.6%)</td><td rowspan=1 colspan=1>0 (0.0%)</td></tr><tr><td rowspan=1 colspan=1>Para-Pak C&amp;S</td><td rowspan=1 colspan=1>1 (0.1%)</td><td rowspan=1 colspan=1>1 (3.2%)</td><td rowspan=1 colspan=1>0 (0.0%)</td><td rowspan=1 colspan=1>0 (0.0%)</td><td rowspan=1 colspan=1>0 (0.0%)</td><td rowspan=1 colspan=1>0 (0.0%)</td></tr><tr><td rowspan=2 colspan=1>Entamoeba histolytica</td><td rowspan=1 colspan=1>FecalSwab</td><td rowspan=1 colspan=1>0 (0.0%)</td><td rowspan=1 colspan=1>0 (0.0%)</td><td rowspan=1 colspan=1>0 (0.0%)</td><td rowspan=1 colspan=1>0 (0.0%)</td><td rowspan=1 colspan=1>0 (0.0%)</td><td rowspan=1 colspan=1>0 (0.0%)</td></tr><tr><td rowspan=1 colspan=1>Para-Pak C&amp;S</td><td rowspan=1 colspan=1>0 (0.0%)</td><td rowspan=1 colspan=1>0 (0.0%)</td><td rowspan=1 colspan=1>0 (0.0%)</td><td rowspan=1 colspan=1>0 (0.0%)</td><td rowspan=1 colspan=1>0 (0.0%)</td><td rowspan=1 colspan=1>0 (0.0%)</td></tr></table>

# Clinical Performance

The clinical performance of QIAstat-Dx Gastrointestinal Panel 2 was established during a multi-center international prospective study conducted at thirteen clinical settings representative of different geographical areas within USA and Europe (9 sites in USA and 4 sites in Europe) between May and July 2021. All study sites were hospital-associated or independent clinical diagnostics laboratories that perform routine diagnostics of GI infections. Table 5.30 provides a summary of prospective specimen’s distribution across all study sites.

Table 5.30: Prospective Specimens Distribution Across the study sites   
Table 5.31: provides all demographic information for the 1,939 specimens evaluated in the prospective study.   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=3>No. of prospective specimens</td></tr><tr><td rowspan=1 colspan=1>Site/Country</td><td rowspan=1 colspan=1>FecalSwab</td><td rowspan=1 colspan=1>Para-Pak C&amp;S</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Germany</td><td rowspan=1 colspan=1>293</td><td rowspan=1 colspan=1>46</td><td rowspan=1 colspan=1>339</td></tr><tr><td rowspan=1 colspan=1>Denmark</td><td rowspan=1 colspan=1>293</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>293</td></tr><tr><td rowspan=1 colspan=1>Spain</td><td rowspan=1 colspan=1>247</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>247</td></tr><tr><td rowspan=1 colspan=1>France</td><td rowspan=1 colspan=1>63</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>63</td></tr><tr><td rowspan=1 colspan=1>USA site 1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>186</td><td rowspan=1 colspan=1>186</td></tr><tr><td rowspan=1 colspan=1>USA site 2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>43</td><td rowspan=1 colspan=1>43</td></tr><tr><td rowspan=1 colspan=1>USA site 3</td><td rowspan=1 colspan=1>282</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>282</td></tr><tr><td rowspan=1 colspan=1>USA site 4</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>177</td><td rowspan=1 colspan=1>177</td></tr><tr><td rowspan=1 colspan=1>USA site 5</td><td rowspan=1 colspan=1>44</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>44</td></tr><tr><td rowspan=1 colspan=1>USA site 6</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>39</td><td rowspan=1 colspan=1>39</td></tr><tr><td rowspan=1 colspan=1>USA site 7</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0 a</td></tr><tr><td rowspan=1 colspan=1>USA site 8</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>131</td><td rowspan=1 colspan=1>131</td></tr><tr><td rowspan=1 colspan=1>USA site 9</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>95</td><td rowspan=1 colspan=1>95</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>1222</td><td rowspan=1 colspan=1>717</td><td rowspan=1 colspan=1>1939</td></tr></table>

a the specimens from this site (148) were excluded from the analysis because they were collected with another device different to Para-Pak C&S or FecalSwab

Table 5.31: Demographic data for prospective evaluated specimens   

<table><tr><td rowspan=1 colspan=1>Demographic data</td><td rowspan=1 colspan=2>FecalSwab</td><td rowspan=1 colspan=2>Para-Pak C&amp;S</td></tr><tr><td rowspan=1 colspan=1>Gender</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Female</td><td rowspan=1 colspan=1>628</td><td rowspan=1 colspan=1>32.4</td><td rowspan=1 colspan=1>442</td><td rowspan=1 colspan=1>22.8</td></tr><tr><td rowspan=1 colspan=1>Male</td><td rowspan=1 colspan=1>594</td><td rowspan=1 colspan=1>30.6</td><td rowspan=1 colspan=1>275</td><td rowspan=1 colspan=1>14.2</td></tr><tr><td rowspan=1 colspan=1>Age Group</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>0-5 years</td><td rowspan=1 colspan=1>182</td><td rowspan=1 colspan=1>9.4</td><td rowspan=1 colspan=1>31</td><td rowspan=1 colspan=1>1.6</td></tr><tr><td rowspan=1 colspan=1>6-21 years</td><td rowspan=1 colspan=1>121</td><td rowspan=1 colspan=1>6.2</td><td rowspan=1 colspan=1>38</td><td rowspan=1 colspan=1>2.0</td></tr><tr><td rowspan=1 colspan=1>22-49 years</td><td rowspan=1 colspan=1>290</td><td rowspan=1 colspan=1>15.0</td><td rowspan=1 colspan=1>215</td><td rowspan=1 colspan=1>11.1</td></tr><tr><td rowspan=1 colspan=1>50+ years</td><td rowspan=1 colspan=1>629</td><td rowspan=1 colspan=1>32.4</td><td rowspan=1 colspan=1>426</td><td rowspan=1 colspan=1>22.0</td></tr><tr><td rowspan=1 colspan=1>Not Reported</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>0.4</td></tr><tr><td rowspan=1 colspan=1>Patient population</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Emergency room</td><td rowspan=1 colspan=1>46</td><td rowspan=1 colspan=1>2.4</td><td rowspan=1 colspan=1>29</td><td rowspan=1 colspan=1>1.5</td></tr><tr><td rowspan=1 colspan=1>Hospitalized</td><td rowspan=1 colspan=1>342</td><td rowspan=1 colspan=1>17.6</td><td rowspan=1 colspan=1>143</td><td rowspan=1 colspan=1>7.4</td></tr><tr><td rowspan=1 colspan=1>Immunocompromised</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0</td></tr><tr><td rowspan=1 colspan=1>Outpatient</td><td rowspan=1 colspan=1>491</td><td rowspan=1 colspan=1>25.3</td><td rowspan=1 colspan=1>325</td><td rowspan=1 colspan=1>16.8</td></tr><tr><td rowspan=1 colspan=1>No information available</td><td rowspan=1 colspan=1>340</td><td rowspan=1 colspan=1>17.5</td><td rowspan=1 colspan=1>220</td><td rowspan=1 colspan=1>11.3</td></tr><tr><td rowspan=1 colspan=5>No. Days between Symptom Onset and QIAstat-Dx Testing</td></tr><tr><td rowspan=1 colspan=1>&gt; 7 days</td><td rowspan=1 colspan=1>89</td><td rowspan=1 colspan=1>4.6</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0</td></tr><tr><td rowspan=1 colspan=1>≤ 7 days</td><td rowspan=1 colspan=1>146</td><td rowspan=1 colspan=1>7.5</td><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>0.8</td></tr><tr><td rowspan=1 colspan=1>Not Reported</td><td rowspan=1 colspan=1>987</td><td rowspan=1 colspan=1>50.9</td><td rowspan=1 colspan=1>701</td><td rowspan=1 colspan=1>36.2</td></tr></table>

The performance of the QIAstat-Dx Gastrointestinal Panel 2 was evaluated for each panel test result using one FDA-cleared test as comparator for the most analytes. A composite comparator consisting of either three independent FDA-cleared test methods or two independent FDA-cleared tests methods and two validated PCR assays followed by bidirectional sequencing was used for Norovirus GI/GII, ETEC, STEC and Giardia lamblia. (Table 5.32: ). The composite endpoint was determined as the majority of the three results.

Table 5.32: QIAstat-Dx Gastrointestinal Panel 2 Clinical studies comparator method   

<table><tr><td rowspan=1 colspan=1>QIAstat-Dx GI Panel 2 Test Result</td><td rowspan=1 colspan=1>Comparator Method</td></tr><tr><td rowspan=1 colspan=1>Adenovirus F40/41</td><td rowspan=15 colspan=1>One FDA-cleared test method</td></tr><tr><td rowspan=1 colspan=1>Astrovirus</td></tr><tr><td rowspan=1 colspan=1>Rotavirus A</td></tr><tr><td rowspan=2 colspan=1>Sapovirus</td><td rowspan=2 colspan=1></td></tr><tr><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Campylobacter</td></tr><tr><td rowspan=1 colspan=1>Plesiomonas shigelloides</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Salmonella</td></tr><tr><td rowspan=1 colspan=1>Yersinia enterocolitica</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>E. coli 0157 a</td></tr><tr><td rowspan=1 colspan=1>Enteropathogenic E. coli (EPEC) a</td></tr><tr><td rowspan=1 colspan=1>Shigella/Enteroinvasive E. coli (EIEC)</td></tr><tr><td rowspan=1 colspan=1>Cryptosporidium</td></tr><tr><td rowspan=1 colspan=1>Cyclospora cayetanensis</td></tr><tr><td rowspan=1 colspan=1>Entamoeba histolytica</td></tr><tr><td rowspan=1 colspan=1>Norovirus GI/GII</td><td rowspan=1 colspan=1>Composite of three FDA-cleared testmethods</td></tr><tr><td rowspan=1 colspan=1>Enterotoxigenic E. coli (ETEC) lt/st</td><td rowspan=1 colspan=1>Composite of three FDA-cleared testmethods</td></tr><tr><td rowspan=1 colspan=1>Shiga-like toxin- E. coli (STEC) stx1/stx2 a</td><td rowspan=1 colspan=1>Composite of three FDA-cleared testmethods</td></tr><tr><td rowspan=1 colspan=1>Giardia lamblia</td><td rowspan=1 colspan=1>Composite of two FDA-cleared test methodsand two validated PCR tests followed by bi-directional sequencing b</td></tr></table>

a Targets evaluated in Para-Pak C&S specimens only. b Each of the two PCR assays used were well-characterized and validated nucleic acid amplification tests (NAAT) followed by bi-directional sequencing analysis. Both assays were designed to amplify different sequences than those targeted by the QIAstat-Dx Gastrointestinal Panel 2. Positive results required to generate sequences from bi-directional sequencing with at least 200 bases of adequate quality that by BLAST analyses matched a sequence of the expected organism or gene from NCBI GenBank database with at least $9 5 \%$ query coverage and at least $9 5 \%$ identity compared to the reference.

Additional prospective archived samples were collected for Norovirus GI/GII (81 samples) and STEC (18 samples). These were prospectively collected samples from four different collection sites (3 US and 1 EU), where only those positive for the pathogen by standard of care method were archived for analysis alongside 20 negative specimens.

In addition, to supplement the results of the prospective clinical studies, a total of 750 preselected archived frozen (retrospective) specimens were also evaluated. These specimens served to increase the sample size for analytes that showed low prevalence in the clinical prospective study or that were less represented in a particular sample type (ParaPak C&S or FecalSwab). The same Comparator Methods detailed in Table 5.31 were used as confirmatory testing for the presence of the nucleic acids from the expected.

In total 2808 specimens (1939 prospective, 119 prospective archived and 750 retrospective) were evaluated in the clinical study. These specimens were collected using Para-Pak C&S (1217) or FecalSwab (1591).

The positive percentage agreement (PPA) and the negative percentage agreement (NPA) were calculated for the prospective and retrospective studies and for each sample type (Para-Pak C&S and FecalSwab) separately.

The PPA was calculated as $100 \%$ x $\left( \mathrm { T P } / \left( \mathrm { T P } + \mathrm { F N } \right) \right)$ . True positive (TP) indicates that both the QIAstat-Dx Gastrointestinal Panel 2 and comparator method showed a positive result for this specific target, and false negative (FN) indicates that the QIAstat-Dx Gastrointestinal Panel 2 result was negative while the comparator method result was positive. The NPA was calculated as $100 \%$ x (TN / $( \mathrm { T N } + \mathrm { F P } )$ ). True negative (TN) indicates that both the QIAstat-Dx Gastrointestinal Panel 2 and the comparator method showed negative results, and a false positive (FP) indicates that the QIAstat-Dx Gastrointestinal Panel 2 result was positive, but the comparator method result was negative. The PPA and NPA exact binomial two-sided $9 5 \%$ confidence interval was calculated.

Where a composite comparator was used (Table 5.32), the result was determined as the majority of the three individual test results (i.e., a positive composite comparator result is based on positive results for at least two comparator tests and a negative composite comparator result is based on negative results for at least two comparator tests). If insufficient pathogen positive sample was available to obtain a majority test result a worstcase model was applied in the PPA calculation. In this model the PPA was calculated including all observed true positive and false negative samples between QIAstat-Dx and the composite comparator while for the samples where it was not possible to conduct testing with the complete comparator due to insufficient sample volume, the following was done:

samples that were negative in QIAstat-Dx and positive for one comparator assay, negative (or insufficient volume) for a second comparator and insufficient volume for a third comparator were included in the calculations as worst-case false negatives, ⚫ samples that were positive in QIAstat-Dx and positive in one comparator test, negative (or insufficient volume) for a second comparator and insufficient volume for the third comparator, were considered as worst-case false positives and therefore, excluded in the PPA calculations.

The results of the clinical performance of the prospective, prospective archived and retrospective studies are summarized in Tables: Table 5.33: , Table 5.34 and Table 5.35, respectively.

Discrepancies between the QIAstat-Dx Gastrointestinal Panel 2 and the comparator methods were investigated for the analytes that the QIAstat-Dx Gastrointestinal Panel 2 test result was compared to one FDA-cleared method. Discrepancies analyses are footnoted on each clinical performance summary Table below (Table 5.33, and Table 5.35).

Table 5.33: Clinical Performance in the Prospective study   

<table><tr><td colspan="2" rowspan="1"></td><td colspan="1" rowspan="1">Positiv</td><td colspan="2" rowspan="1">Positive Percent Agreement</td><td colspan="3" rowspan="1">Negative Percent Agreement</td></tr><tr><td colspan="1" rowspan="1">Analyte</td><td colspan="1" rowspan="1">MediumBrand</td><td colspan="1" rowspan="1">TP/TP+FN</td><td colspan="1" rowspan="1">%</td><td colspan="1" rowspan="1">95% CI</td><td colspan="1" rowspan="1">TN/TN+FP</td><td colspan="1" rowspan="1">%</td><td colspan="1" rowspan="1">95% CI</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Viruses</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="2">Adenovirus F40/F41</td><td colspan="1" rowspan="1">FecalSwab</td><td colspan="1" rowspan="1">5/6 a</td><td colspan="1" rowspan="1">83.3</td><td colspan="1" rowspan="1">43.7-97.0</td><td colspan="1" rowspan="1">1214 / 1214</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">99.7-100.0</td></tr><tr><td colspan="1" rowspan="1">Para-PakC&amp;S</td><td colspan="1" rowspan="1">1/2b</td><td colspan="1" rowspan="1">50.0</td><td colspan="1" rowspan="1">9.5-90.6</td><td colspan="1" rowspan="1">703 / 704 b</td><td colspan="1" rowspan="1">99.9</td><td colspan="1" rowspan="1">99.2-100.0</td></tr><tr><td colspan="1" rowspan="2">Astrovirus</td><td colspan="1" rowspan="1">FecalSwab</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">43.9-100.0</td><td colspan="1" rowspan="1">1219 / 1219</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">99.7-100.0</td></tr><tr><td colspan="1" rowspan="1">Para-PakC&amp;S</td><td colspan="1" rowspan="1">6/6</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">61.0-100.0</td><td colspan="1" rowspan="1">700 / 700</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">99.5-100.0</td></tr><tr><td colspan="1" rowspan="2">Norovirus GI/GII</td><td colspan="1" rowspan="1">FecalSwab</td><td colspan="1" rowspan="1">31/33c</td><td colspan="1" rowspan="1">93.9</td><td colspan="1" rowspan="1">80.4-98.3</td><td colspan="1" rowspan="1">493 / 495c</td><td colspan="1" rowspan="1">99.6</td><td colspan="1" rowspan="1">98.6-100.0</td></tr><tr><td colspan="1" rowspan="1">Para-PakC&amp;S</td><td colspan="1" rowspan="1">14 /18 d</td><td colspan="1" rowspan="1">77.8</td><td colspan="1" rowspan="1">54.8-91.0</td><td colspan="1" rowspan="1">399 / 399 d</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">99.1-00.0</td></tr><tr><td colspan="1" rowspan="2">Rotavirus A</td><td colspan="1" rowspan="1">FecalSwab</td><td colspan="1" rowspan="1">21 /23 </td><td colspan="1" rowspan="1">91.3</td><td colspan="1" rowspan="1">73.2-97.6</td><td colspan="1" rowspan="1">1197 / 1199e</td><td colspan="1" rowspan="1">99.8</td><td colspan="1" rowspan="1">99.4-100.0</td></tr><tr><td colspan="1" rowspan="1">Para-PakC&amp;S</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">43.9-100.0</td><td colspan="1" rowspan="1">702 / 703</td><td colspan="1" rowspan="1">99.9</td><td colspan="1" rowspan="1">99.2-100.0</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Bacteria</td><td colspan="1" rowspan="1">a</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="2">Campylobacter</td><td colspan="1" rowspan="1">FecalSwab</td><td colspan="1" rowspan="1">65/67 8</td><td colspan="1" rowspan="1">97.0</td><td colspan="1" rowspan="1">89.8-99.2</td><td colspan="1" rowspan="1">1151/1155g</td><td colspan="1" rowspan="1">99.7</td><td colspan="1" rowspan="1">99.1-99.9</td></tr><tr><td colspan="1" rowspan="1">Para-PakC&amp;S</td><td colspan="1" rowspan="1">30 /31 h</td><td colspan="1" rowspan="1">96.8</td><td colspan="1" rowspan="1">83.8-99.4</td><td colspan="1" rowspan="1">675 / 677 h</td><td colspan="1" rowspan="1">99.7</td><td colspan="1" rowspan="1">98.9-99.9</td></tr><tr><td colspan="1" rowspan="2">Plesiomonasshigelloides</td><td colspan="1" rowspan="1">FecalSwab</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">1220 / 1222i</td><td colspan="1" rowspan="1">99.8</td><td colspan="1" rowspan="1">99.4-100.0</td></tr><tr><td colspan="1" rowspan="1">Para-PakC&amp;S</td><td colspan="1" rowspan="1">5/61</td><td colspan="1" rowspan="1">83.3</td><td colspan="1" rowspan="1">43.7-97.0</td><td colspan="1" rowspan="1">698 / 700 j</td><td colspan="1" rowspan="1">99.7</td><td colspan="1" rowspan="1">99.0-99.9</td></tr><tr><td colspan="1" rowspan="2">Salmonella</td><td colspan="1" rowspan="1">FecalSwab</td><td colspan="1" rowspan="1">14/16k</td><td colspan="1" rowspan="1">87.5</td><td colspan="1" rowspan="1">64.0-96.5</td><td colspan="1" rowspan="1">1206 / 1206</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">99.7-100.0</td></tr><tr><td colspan="1" rowspan="1">Para-PakC&amp;S</td><td colspan="1" rowspan="1">19 / 201</td><td colspan="1" rowspan="1">95.0</td><td colspan="1" rowspan="1">76.4-99.1</td><td colspan="1" rowspan="1">688 / 688</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">99.4-100.0</td></tr><tr><td colspan="1" rowspan="2">Yersinia enterocolitica</td><td colspan="1" rowspan="1">FecalSwab</td><td colspan="1" rowspan="1">15 / 16 m</td><td colspan="1" rowspan="1">93.8</td><td colspan="1" rowspan="1">71.7-99.0</td><td colspan="1" rowspan="1">1199 / 1206m</td><td colspan="1" rowspan="1">99.4</td><td colspan="1" rowspan="1">98.8-99.7</td></tr><tr><td colspan="1" rowspan="1">Para-PakC&amp;S</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">43.9-100.0</td><td colspan="1" rowspan="1">698 / 703 n</td><td colspan="1" rowspan="1">99.3</td><td colspan="1" rowspan="1">98.4-99.7</td></tr><tr><td colspan="8" rowspan="1">Diarrheagenic E. coli/Shigella</td></tr><tr><td colspan="1" rowspan="1">Enteropathogenic E.coli (EPEC)</td><td colspan="1" rowspan="1">Para-PakTC&amp;</td><td colspan="1" rowspan="1">57/65</td><td colspan="1" rowspan="1">87.7</td><td colspan="1" rowspan="1">77.6-93.6</td><td colspan="1" rowspan="1">632 / 632</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">99.4-100.0</td></tr><tr><td colspan="1" rowspan="2">Enterotoxigenic E. coli(ETEC) lt/st</td><td colspan="1" rowspan="1">FecalSwab</td><td colspan="1" rowspan="1">9/10°</td><td colspan="1" rowspan="1">90.0</td><td colspan="1" rowspan="1">59.6-99.2</td><td colspan="1" rowspan="1">427 / 430°</td><td colspan="1" rowspan="1">99.3</td><td colspan="1" rowspan="1">98.0-99.8</td></tr><tr><td colspan="1" rowspan="1">Para-PakTC </td><td colspan="1" rowspan="1">9/10 P</td><td colspan="1" rowspan="1">90.0</td><td colspan="1" rowspan="1">59.6-99.2</td><td colspan="1" rowspan="1">390 / 395 P</td><td colspan="1" rowspan="1">98.7</td><td colspan="1" rowspan="1">97.1-99.5</td></tr><tr><td colspan="1" rowspan="1">Shiga-like toxin E. coli(STEC) stx1/stx2</td><td colspan="1" rowspan="1">Para-PakC&amp;S</td><td colspan="1" rowspan="1">5/69</td><td colspan="1" rowspan="1">83.3</td><td colspan="1" rowspan="1">43.6-97.0</td><td colspan="1" rowspan="1">397 / 4009</td><td colspan="1" rowspan="1">99.3</td><td colspan="1" rowspan="1">97.8-99.7</td></tr><tr><td colspan="1" rowspan="1">E. coli O157</td><td colspan="1" rowspan="1">Para-PakT C&amp;S</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">5/5</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">56.6-100.0</td></tr><tr><td colspan="1" rowspan="2">Shigella/EnteroinvasiveE. coli (EIEC)</td><td colspan="1" rowspan="1">FecalSwab</td><td colspan="1" rowspan="1">10 / 10</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">72.3-100.0</td><td colspan="1" rowspan="1">1212 / 1212</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">99.7-100.0</td></tr><tr><td colspan="1" rowspan="1">Para-PakC&amp;S</td><td colspan="1" rowspan="1">2/2</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">34.2-100.0</td><td colspan="1" rowspan="1">703 / 704</td><td colspan="1" rowspan="1">99.9</td><td colspan="1" rowspan="1">99.2-100.0</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Parasites</td><td colspan="1" rowspan="1">cYanmaGlow</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="2">Cryptosporidium</td><td colspan="1" rowspan="1">FecalSwab</td><td colspan="1" rowspan="1">2/4</td><td colspan="1" rowspan="1">50.0</td><td colspan="1" rowspan="1">15.0-85.0</td><td colspan="1" rowspan="1">1218 / 1218</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">99.7-100.0</td></tr><tr><td colspan="1" rowspan="1">Para-Pak&amp;</td><td colspan="1" rowspan="1">6/6</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">61.0-100.0</td><td colspan="1" rowspan="1">699 / 700 s</td><td colspan="1" rowspan="1">99.9</td><td colspan="1" rowspan="1">99.2-100.0</td></tr><tr><td colspan="1" rowspan="2">Cyclosporacayetanensis</td><td colspan="1" rowspan="1">FecalSwab</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">100.00</td><td colspan="1" rowspan="1">43.9-100.0</td><td colspan="1" rowspan="1">1219 / 1219</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">99.7-100.0</td></tr><tr><td colspan="1" rowspan="1">Para-PakC&amp;S</td><td colspan="1" rowspan="1">18 / 19t</td><td colspan="1" rowspan="1">94.7</td><td colspan="1" rowspan="1">75.4-99.1</td><td colspan="1" rowspan="1">687 / 687</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">99.4-100.0</td></tr><tr><td colspan="2" rowspan="1"></td><td colspan="3" rowspan="1">Positive Percent Agreement</td><td colspan="3" rowspan="1">Negative Percent Agreement</td></tr><tr><td colspan="1" rowspan="1">Analyte</td><td colspan="1" rowspan="1">MediumBrand</td><td colspan="1" rowspan="1">TP/TP+FN</td><td colspan="1" rowspan="1">%</td><td colspan="1" rowspan="1">95% CI</td><td colspan="1" rowspan="1">TN/TN+FP</td><td colspan="1" rowspan="1">%</td><td colspan="1" rowspan="1">95% CI</td></tr><tr><td colspan="1" rowspan="2">Entamoeba histolytica</td><td colspan="1" rowspan="1">FecalSwab</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">1222 / 1222</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">99.7-100.0</td></tr><tr><td colspan="1" rowspan="1">Para-PakC&amp;S</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">706 / 706</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">99.5-100.0</td></tr><tr><td colspan="1" rowspan="2">Giardia lamblia</td><td colspan="1" rowspan="1">FecalSwab</td><td colspan="1" rowspan="1">6/8 </td><td colspan="1" rowspan="1">75.0</td><td colspan="1" rowspan="1">40.9-92.9</td><td colspan="1" rowspan="1">434 / 441 </td><td colspan="1" rowspan="1">98.4</td><td colspan="1" rowspan="1">96.8-99.2</td></tr><tr><td colspan="1" rowspan="1">Para-PakC&amp;S</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">20.7-100.0</td><td colspan="1" rowspan="1">406 / 406v</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">99.1-100.0</td></tr></table>

a Adenovirus F40/41 was not detected in the single false negative specimen $( 0 / 1 )$ in FecalSwab using a different FDA-cleared test method

b Adenovirus F40/41 was not detected in the single false negative specimen (0/1) and in the single false positive specimen $( 0 / 1 )$ in Para-Pak C&S using a different FDA-cleared test method

c ten (10) FecalSwab samples positive for Norovirus GI/GII in both QIAstat-Dx and one FDA-cleared comparator were excluded from the PPA calculations because the samples did not have sufficient volume for complete composite comparator testing. The sample size for NPA is smaller for Norovirus GI/GII as only a portion of the samples with a negative result in QIAstat-Dx and in one FDA-cleared comparator was tested with the complete composite comparator in the prospective study.

d two (2) Para-Pak C&S samples positive for Norovirus GI/GII in both QIAstat-Dx and one FDA-cleared comparator were excluded from the PPA calculations because the samples did not have sufficient volume for complete composite comparator testing. One (1) Para-Pak C&S sample negative in QIAstat-Dx and positive with one FDA-cleared comparator with insufficient volume for complete composite comparator testing were classed as false negative in the PPA calculations. The sample size for NPA is smaller for Norovirus GI/GII as only a portion of the samples with a negative result in QIAstat-Dx and in one FDA-cleared comparator was tested with the complete composite comparator in the prospective study.

$\mathbf { e }$ Rotavirus A was detected in one of the two false negative specimens (1/2) and was not detected in the two false positive specimens (0/2) in FecalSwab using a different FDA-cleared test method.

f Rotavirus A was not detected in the single false positive specimen $( 0 / 1 )$ in Para-Pak C&S using a different FDA-cleared test metho g Campylobacter was not detected in the two false negative specimens (0/2) and was detected in three of the four false positive specimens (3/4) in FecalSwab using a different FDA-cleared test method.

h Campylobacter was not detected in the single false negative specimens (0/1) and was detected in one of the two false positive specimens (1/2) in Para-Pak C&S using a different FDA-cleared test method.

i Plesiomonas shigelloides was not detected in the two false positive specimens (0/2) in FecalSwab using a different FDA-cleared test method

j Plesiomonas shigelloides was not detected in the single false negative specimen $( 0 / 1 )$ and was not detected in the two false positive specimens (0/2) in Para-Pak C&S using a different FDA-cleared test method

k Salmonella was not detected in the two false negative specimens (0/2) in FecalSwab using a different FDA-cleared test method. l Salmonella was not detected in the single false negative specimen (0/1) in Para-Pak C&S using a different FDA-cleared test method. m Yersinia enterocolitica was not detected in the single false negative specimen (0/1) and was not detected in the seven false positive specimens (0/7) in FecalSwab using a different FDA-cleared test method.

ia enterocolitica was not detected in the five false positive specimens (0/5) using a different FDA-cleared test metho o six (6) FecalSwab samples positive for ETEC in both QIAstat-Dx and the primary FDA-cleared comparator were excluded from the PPA calculations because the samples did not have sufficient volume for complete composite comparator testing. The sample size for NPA is smaller for ETEC as only a portion of the samples with a negative result in QIAstat-Dx and in one FDA-cleared comparator was tested with the complete composite comparator in the prospective study.

p three (3) Para-Pak C&S samples positive for ETEC in both QIAstat-Dx and one FDA-cleared comparator were excluded from the PPA calculations because the samples did not have sufficient volume for complete composite comparator testing. The sample size for NPA is smaller for ETEC as only a portion of the samples with a negative result in QIAstat-Dx and in one FDA-cleared comparator was tested with the complete composite comparator in the prospective study.

q one (1) FecalSwab sample positive for STEC in both QIAstat-Dx and one FDA-cleared comparator was excluded from the PPA calculations because the samples did not have sufficient volume for complete composite comparator testing. The sample size for NPA is smaller for STEC as only a portion of the samples with a negative result in QIAstat-Dx and in one FDA-cleared comparator was tested with the complete composite comparator in the prospective study.

r Shigella/ EIEC was detected in the single false positive specimen (1/1) in Para-Pak C&S using a different FDA-cleared test method. s Cryptosporidium was not detected in the single false positive specimen (0/1) in Para-Pak C&S using PCR followed by bi-directional sequence analysis.

t For Cyclospora cayetanensis there was one (1) false negative specimen in Para-Pak C&S that was not further investigated by discrepant analyses.

u two (2) FecalSwab samples positive for Giardia lamblia in both QIAstat-Dx and one FDA-cleared comparator were excluded from the PPA calculations because the samples did not have sufficient volume for complete composite comparator testing. Two (2)

FecalSwab samples negative in QIAstat-Dx and positive with one FDA-cleared comparator with insufficient volume for complete composite comparator testing were classed as false negative in the PPA calculations. The sample size for NPA is smaller for Giardia lamblia as only a portion of the samples with a negative result in QIAstat-Dx and in one FDA-cleared comparator was tested with the complete composite comparator in the prospective study.

v the sample size for NPA is smaller for Giardia lamblia as only a portion of the samples with a negative result in QIAstat-Dx and in one FDA-cleared comparator was tested with the complete composite comparator in the prospective study.

Table 5.34: Clinical Performance in the Prospective Archived study   

<table><tr><td rowspan=1 colspan=2></td><td rowspan=1 colspan=3>Positive Percent Agreement</td><td rowspan=1 colspan=3>Negative Percent Agreement</td></tr><tr><td rowspan=1 colspan=1>Analyte</td><td rowspan=1 colspan=1>Medium Brand</td><td rowspan=1 colspan=1>TP/TP+FN</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=1>TN/TN+FP</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=2 colspan=1>NorovirusGI/GII</td><td rowspan=1 colspan=1>FecalSwab</td><td rowspan=1 colspan=1>48 / 49 </td><td rowspan=1 colspan=1>98.0</td><td rowspan=1 colspan=1>89.3 - 99.6</td><td rowspan=1 colspan=1>2/4a</td><td rowspan=1 colspan=1>50.0</td><td rowspan=1 colspan=1>15.0-85.0</td></tr><tr><td rowspan=1 colspan=1>Para-Pak C&amp;S</td><td rowspan=1 colspan=1>26 / 28 a,b</td><td rowspan=1 colspan=1>92.9</td><td rowspan=1 colspan=1>77.4 - 98.0</td><td rowspan=1 colspan=1>37 / 38 a</td><td rowspan=1 colspan=1>97.4</td><td rowspan=1 colspan=1>86.5-99.5</td></tr><tr><td rowspan=1 colspan=1>Shiga-liketoxin E. coli(STEC)stx1/stx2</td><td rowspan=1 colspan=1>Para-Pak C&amp;S</td><td rowspan=1 colspan=1>12 / 13 c,d</td><td rowspan=1 colspan=1>92.3</td><td rowspan=1 colspan=1>66.7 - 98.6</td><td rowspan=1 colspan=1>51/52c</td><td rowspan=1 colspan=1>98.1</td><td rowspan=1 colspan=1>89.9-99.7</td></tr></table>

a For Norovirus GI/GII/GII four out of the eighty-one (4/81) positive prospectively archived samples (positive by standard of care) were negative by the composite comparator and therefore included as negative samples in the NPA calculations. b One (1) Para-Pak C&S sample negative in QIAstat-Dx and positive for Norovirus GI/GII with one FDA-cleared comparator with insufficient volume for complete composite comparator testing was classed as false negative in the PPA calculations. c For STEC five out of the eighteen (5/18) positive prospectively archived samples (positive by standard of care) were negative by the composite comparator and therefore included as negative samples in the NPA calculations. d One (1) Para-Pak C&S sample positive for STEC in both QIAstat-Dx and one FDA-cleared comparator was excluded from the PPA calculations because the samples did not have sufficient volume for complete composite comparator testing.

Table 5.35: Clinical Performance in the Retrospective study   

<table><tr><td rowspan=1 colspan=2></td><td rowspan=1 colspan=3>Positive Percent Agreement</td><td rowspan=1 colspan=3>Negative Percent Agreement</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>MediumBrand</td><td rowspan=1 colspan=1>TP/TP+FN</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=1>TN/TN+FP</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>V</td><td rowspan=1 colspan=1>Viruses</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=2 colspan=1>Adenovirus F40/F41</td><td rowspan=1 colspan=1>FecalSwab</td><td rowspan=1 colspan=1>23 / 26 a</td><td rowspan=1 colspan=1>88.5</td><td rowspan=1 colspan=1>71.0-96.0</td><td rowspan=1 colspan=1>203 /203</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>98.2-100.0</td></tr><tr><td rowspan=1 colspan=1>Para-PakC&amp;S</td><td rowspan=1 colspan=1>29 / 29</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>88.3-100.0</td><td rowspan=1 colspan=1>39 / 39</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>91.0-100.0</td></tr><tr><td rowspan=2 colspan=1>Astrovirus</td><td rowspan=1 colspan=1>FecalSwab</td><td rowspan=1 colspan=1>2/3b</td><td rowspan=1 colspan=1>66.7</td><td rowspan=1 colspan=1>20.8-93.9</td><td rowspan=1 colspan=1>191 /191</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>98.0-100.0</td></tr><tr><td rowspan=1 colspan=1>Para-PakC&amp;S</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>14/14</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>78.5-100.0</td></tr><tr><td rowspan=2 colspan=1>Norovirus GI/GII</td><td rowspan=1 colspan=1>FecalSwab</td><td rowspan=1 colspan=1>28 /32 c</td><td rowspan=1 colspan=1>87.5</td><td rowspan=1 colspan=1>71.9-95.0</td><td rowspan=1 colspan=1>74/75</td><td rowspan=1 colspan=1>98.7</td><td rowspan=1 colspan=1>92.8-100.0</td></tr><tr><td rowspan=1 colspan=1>Para-PakC&amp;S</td><td rowspan=1 colspan=1>27 /29</td><td rowspan=1 colspan=1>93.1</td><td rowspan=1 colspan=1>78.0-98.1</td><td rowspan=1 colspan=1>86 / 86 d</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>95.8-100.0</td></tr><tr><td rowspan=2 colspan=1>Rotavirus A</td><td rowspan=1 colspan=1>FecalSwab</td><td rowspan=1 colspan=1>8/9c</td><td rowspan=1 colspan=1>88.9</td><td rowspan=1 colspan=1>56.5-98.0</td><td rowspan=1 colspan=1>185 /185</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>98.0-100.0</td></tr><tr><td rowspan=1 colspan=1>Para-PakC&amp;s</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>34.2-100.0</td><td rowspan=1 colspan=1>12 / 12</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>75.8-100.0</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=2 colspan=1>Campylobacter</td><td rowspan=1 colspan=1>FecalSwab</td><td rowspan=1 colspan=1>31/31</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>89.0-100.0</td><td rowspan=1 colspan=1>161 /163</td><td rowspan=1 colspan=1>98.8</td><td rowspan=1 colspan=1>95.6-99.7</td></tr><tr><td rowspan=1 colspan=1>Para-PakC&amp;S</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>43.9-100.0</td><td rowspan=1 colspan=1>11/11</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>74.1-100.0</td></tr><tr><td rowspan=2 colspan=1>Plesiomonasshigelloides</td><td rowspan=1 colspan=1>FecalSwab</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>34.2-100.0</td><td rowspan=1 colspan=1>192 / 192</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>98.0-100.0</td></tr><tr><td rowspan=1 colspan=1>Para-PakC&amp;S</td><td rowspan=1 colspan=1>33 / 36 g</td><td rowspan=1 colspan=1>91.7</td><td rowspan=1 colspan=1>78.2-97.1</td><td rowspan=1 colspan=1>117/117</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>96.8-100.0</td></tr><tr><td rowspan=2 colspan=1>Salmonella</td><td rowspan=1 colspan=1>FecalSwab</td><td rowspan=1 colspan=1>30 /31h</td><td rowspan=1 colspan=1>96.8</td><td rowspan=1 colspan=1>83.8-99.4</td><td rowspan=1 colspan=1>161 /163 h</td><td rowspan=1 colspan=1>98.8</td><td rowspan=1 colspan=1>95.6-99.7</td></tr><tr><td rowspan=1 colspan=1>Para-PakC&amp;S</td><td rowspan=1 colspan=1>1/1</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>20.7-100.0</td><td rowspan=1 colspan=1>13 /13</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>77.2-100.0</td></tr><tr><td rowspan=2 colspan=1>Yersinia enterocolitica</td><td rowspan=1 colspan=1>FecalSwab</td><td rowspan=1 colspan=1>32 /34</td><td rowspan=1 colspan=1>94.1</td><td rowspan=1 colspan=1>80.9-98.4</td><td rowspan=1 colspan=1>160 /160</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>97.7-100.0</td></tr><tr><td rowspan=1 colspan=1>Para-Pak&amp;</td><td rowspan=1 colspan=1>1/1</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>20.7-100.0</td><td rowspan=1 colspan=1>14/14</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>78.5-100.0</td></tr><tr><td rowspan=1 colspan=8>Diarrheagenic E. coli/Shigella</td></tr><tr><td rowspan=1 colspan=1>Enteropathogenic E.coli (EPEC)</td><td rowspan=1 colspan=1>Para-PakC&amp;S</td><td rowspan=1 colspan=1>60 /65j</td><td rowspan=1 colspan=1>92.3</td><td rowspan=1 colspan=1>83.2-96.7</td><td rowspan=1 colspan=1>42 / 42</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>91.6-100.0</td></tr><tr><td rowspan=2 colspan=1>Enterotoxigenic E. coli(ETEC) lt/st</td><td rowspan=1 colspan=1>FecalSwab</td><td rowspan=1 colspan=1>22 /24k</td><td rowspan=1 colspan=1>91.7</td><td rowspan=1 colspan=1>74.2-97.7</td><td rowspan=1 colspan=1>85 /86 k</td><td rowspan=1 colspan=1>98.8</td><td rowspan=1 colspan=1>93.7-99.8</td></tr><tr><td rowspan=1 colspan=1>Para-PakC&amp;S</td><td rowspan=1 colspan=1>23 /24</td><td rowspan=1 colspan=1>95.8</td><td rowspan=1 colspan=1>79.8-99.3</td><td rowspan=1 colspan=1>61/611</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>94.1-100.0</td></tr><tr><td rowspan=1 colspan=1>Shiga-like toxin E. coli(STEC) stx1/stx2</td><td rowspan=1 colspan=1>Para-PakC&amp;S</td><td rowspan=1 colspan=1>60 / 64</td><td rowspan=1 colspan=1>93.8</td><td rowspan=1 colspan=1>85.0-97.5</td><td rowspan=1 colspan=1>44 / 44 m</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>92.0-100.0</td></tr><tr><td rowspan=1 colspan=1>E. coli O157</td><td rowspan=1 colspan=1>Para-PakC&amp;S</td><td rowspan=1 colspan=1>39 / 42 n</td><td rowspan=1 colspan=1>92.9%</td><td rowspan=1 colspan=1>80.1-99.4</td><td rowspan=1 colspan=1>16 /16</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>80.6-100.0</td></tr><tr><td rowspan=2 colspan=1>Shigella/EnteroinvasiveE. coli (EIEC)</td><td rowspan=1 colspan=1>FecalSwab</td><td rowspan=1 colspan=1>22 /24°</td><td rowspan=1 colspan=1>91.7</td><td rowspan=1 colspan=1>73.0-99.0</td><td rowspan=1 colspan=1>170 /170</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>97.8-100.0</td></tr><tr><td rowspan=1 colspan=1>Para-PakC&amp;</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>14/14</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>78.5-100.0</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Pa</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=2 colspan=1>Cryptosporidium</td><td rowspan=1 colspan=1>FecalSwab</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>54.1-100.0</td><td rowspan=1 colspan=1>186 /188 P</td><td rowspan=1 colspan=1>98.9</td><td rowspan=1 colspan=1>96.2-99.9</td></tr><tr><td rowspan=1 colspan=1>Para-PakC&amp;S</td><td rowspan=1 colspan=1>26 / 26</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>86.8-100.0</td><td rowspan=1 colspan=1>117/117</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>96.9-100.0</td></tr><tr><td rowspan=2 colspan=1>Cyclosporacayetanensis</td><td rowspan=1 colspan=1>FecalSwab</td><td rowspan=1 colspan=1>1/1</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>20.7-100.0</td><td rowspan=1 colspan=1>193 /193</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>98.1-100.0</td></tr><tr><td rowspan=1 colspan=1>Para-PakC&amp;S</td><td rowspan=1 colspan=1>1/1</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>20.7-100.0</td><td rowspan=1 colspan=1>13 /13</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>77.2-100.0</td></tr><tr><td rowspan=2 colspan=1>Entamoeba histolytica</td><td rowspan=1 colspan=1>FecalSwab</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>194 /194</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>98.1-100.0</td></tr><tr><td rowspan=1 colspan=1>Para-PakC&amp;S</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>14/14</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>76.5-100.0</td></tr><tr><td rowspan=2 colspan=1>Giardia lamblia</td><td rowspan=1 colspan=1>FecalSwab</td><td rowspan=1 colspan=1>29 /31q</td><td rowspan=1 colspan=1>93.6</td><td rowspan=1 colspan=1>79.3-98.2</td><td rowspan=1 colspan=1>46 / 48 q</td><td rowspan=1 colspan=1>95.8</td><td rowspan=1 colspan=1>86.0-98.9</td></tr><tr><td rowspan=1 colspan=1>Para-Pak&amp;</td><td rowspan=1 colspan=1>27 /28</td><td rowspan=1 colspan=1>96.4</td><td rowspan=1 colspan=1>82.3-99.4</td><td rowspan=1 colspan=1>92 /92r</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>96.0-100.0</td></tr></table>

a Adenovirus F40/41 was detected in one of the three false negatives $( 1 / 3 )$ in FecalSwab using a different FDA-cleared test method b Astrovirus was detected in the single false negative specimen (1/1) in FecalSwab using a different FDA-cleared test method. c two (2) FecalSwab samples positive for Norovirus GI/GII in both QIAstat-Dx and one FDA-cleared comparator were excluded from the PPA calculations because the samples did not have sufficient volume for complete composite comparator testing. The sample size for NPA is smaller for Norovirus GI/GII as only a portion of the samples with a negative result in QIAstat-Dx and in one FDA-cleared comparator was tested with the complete composite comparator in the retrospective study. d the sample size for NPA is smaller for Norovirus GI/GII in Para-Pak C&S as only a portion of the samples with a negative result in QIAstat-Dx and in one FDA-cleared comparator was tested with the complete composite comparator in the retrospective study e Rotavirus A was detected in the single false negative specimen (1/1) in FecalSwab using a different FDA-cleared test method.

f Campylobacter was detected in one of the two false positive specimens (1/2) in FecalSwab using a different FDA-cleared test method.

g Plesiomonas shigelloides was detected in one of the three false negative specimens (1/3) in Para-Pak C&S using a different FDAcleared test method.

h Salmonella was not detected in the single false negative specimen (0/1) and was not detected in the two false positive specimens (0/2) in FecalSwab using a different FDA-cleared test method.

i Yersinia enterocolitica was not detected in the two false negative specimens (0/2) in FecalSwab using a different FDA-cleared test method.

j Enteropathogenic $E .$ . coli (EPEC) was detected in all three false negative specimens (3/3) in Para-Pak C&S using PCR followed by bi-directional sequence analysis. There were two (2) other false negative specimens that were not further investigated by discrepant analyses.

k ten (10) FecalSwab samples positive for ETEC in both QIAstat-Dx and one FDA-cleared comparator were excluded from the PPA calculations because the samples did not have sufficient volume for complete composite comparator testing. One (1) FecalSwab sample negative in QIAstat-Dx and positive with one FDA-cleared comparator with insufficient volume for complete composite comparator testing was classed as false negative in the PPA calculations. The sample size for NPA is smaller for ETEC as only a portion of the samples with a negative result in QIAstat-Dx and one FDA-cleared comparator was tested with the complete composite comparator in the retrospective study.

l The sample size for NPA is smaller for ETEC in Para-Pak CS&S as only a portion of the samples with a negative result in QIAstatDx and one FDA-cleared comparator was tested with the complete composite comparator in the retrospective study).

m The sample size for NPA is smaller for STEC in Para-Pak C&S as only a portion of the samples with a negative result in QIAstatDx and in one FDA-cleared comparator was tested with the complete composite comparator in the retrospective study n E. coli O157 was not detected in two false negative specimens (0/2) in Para-Pak C&S using a different FDA-cleared test method.

was one (1) false negative specimen in Para-Pak C&S that was not further investigated by discrepant analyses.

o Shigella/ EIEC was detected in one of the two false negative specimens (1/2) in FecalSwab using a different FDA-cleared test method.

p Cryptosporidium was not detected in the two false positive specimens (0/2) in FecalSwab using PCR followed by bi-directional sequence analysis

q four (4) samples positive for Giardia lamblia in both QIAstat-Dx and one FDA-cleared comparator were excluded from the PPA calculations because the samples did not have sufficient volume for complete composite comparator testing. Two (2) FecalSwab samples negative in QIAstat-Dx and positive with one FDA-cleared comparator with insufficient volume for complete composite comparator testing was classed as false negative in the PPA calculations. The sample size for NPA is smaller for Giardia lamblia as only a portion of the samples with a negative result in QIAstat-Dx and in one FDA-cleared comparator was tested with the complete composite comparator in the retrospective study.

r One (1) Para-Pak C&S samples positive for Giardia lamblia in both QIAstat-Dx and primary FDA-cleared comparator (were excluded from the PPA calculations because the samples did not have sufficient volume for complete composite comparator testing. One (1) Para-Pak C&S sample negative in QIAstat-Dx and positive with one FDA-cleared comparator with insufficient volume for complete composite comparator testing was classed as false negative in the PPA calculations. The sample size for NPA is smaller for Giardia lamblia as only a portion of the samples with a negative result in QIAstat-Dx and in one FDA-cleared comparator was tested with the complete composite comparator in the retrospective study.

The proportion of failed runs on initial attempt, and following repeats are summarized in Table 5.36. The error breakdown due to instrument, invalid results, ‘sample too concentrated’ failures and other run failures are summarized in Table 5.37.

Table 5.36 Failure Rates Summary   

<table><tr><td rowspan=2 colspan=1>TransportMedia</td><td rowspan=2 colspan=1>Study</td><td rowspan=1 colspan=3>Initial Runs</td><td rowspan=1 colspan=3>Final Runs</td></tr><tr><td rowspan=1 colspan=1>N / Total</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=1>N / Total</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=4 colspan=1>FecalSwab</td><td rowspan=1 colspan=1>Prospective</td><td rowspan=1 colspan=1>16 / 1227</td><td rowspan=1 colspan=1>1.3</td><td rowspan=1 colspan=1>0.7 - 2.1</td><td rowspan=1 colspan=1>3/1227</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>0.1 - 0.7</td></tr><tr><td rowspan=1 colspan=1>ProspectiveArchived</td><td rowspan=1 colspan=1>0 / 53</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0 - 6.7</td><td rowspan=1 colspan=1>0 /53</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0 - 6.7</td></tr><tr><td rowspan=1 colspan=1>Retrospective</td><td rowspan=1 colspan=1>11 / 366</td><td rowspan=1 colspan=1>3.0</td><td rowspan=1 colspan=1>1.5- 5.3</td><td rowspan=1 colspan=1>5/366</td><td rowspan=1 colspan=1>1.4</td><td rowspan=1 colspan=1>0.4 - 3.2</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>27 / 1646</td><td rowspan=1 colspan=1>1.6</td><td rowspan=1 colspan=1>1.1 - 2.4</td><td rowspan=1 colspan=1>8 /1646</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>0.2 - 1.0</td></tr><tr><td rowspan=4 colspan=1>Para-Pak</td><td rowspan=1 colspan=1>Prospective</td><td rowspan=1 colspan=1>66 / 740</td><td rowspan=1 colspan=1>8.9</td><td rowspan=1 colspan=1>7.0 - 11.2</td><td rowspan=1 colspan=1>21/ 740</td><td rowspan=1 colspan=1>2.8</td><td rowspan=1 colspan=1>1.8 - 4.3</td></tr><tr><td rowspan=1 colspan=1>ProspectiveArchived</td><td rowspan=1 colspan=1>3 / 66</td><td rowspan=1 colspan=1>4.5</td><td rowspan=1 colspan=1>0.9 - 12.7</td><td rowspan=1 colspan=1>0 /66</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0 - 5.4</td></tr><tr><td rowspan=1 colspan=1>Retrospective</td><td rowspan=1 colspan=1>46 / 454</td><td rowspan=1 colspan=1>10.1</td><td rowspan=1 colspan=1>7.5 - 13.3</td><td rowspan=1 colspan=1>25 / 454</td><td rowspan=1 colspan=1>5.5</td><td rowspan=1 colspan=1>3.6 - 8.0</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>115 / 1260</td><td rowspan=1 colspan=1>9.1</td><td rowspan=1 colspan=1>7.6 - 10.9</td><td rowspan=1 colspan=1>46 / 1260</td><td rowspan=1 colspan=1>3.7</td><td rowspan=1 colspan=1>2.7 - 4.8</td></tr></table>

Table 5.37 Failure Types Breakdown   

<table><tr><td rowspan=2 colspan=1>TransportMedia</td><td rowspan=2 colspan=1>Study</td><td rowspan=2 colspan=1>Failure Reason</td><td rowspan=1 colspan=2>Initial Runs</td><td rowspan=1 colspan=2>Final Runs</td></tr><tr><td rowspan=1 colspan=1>N / Total</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>N / Total</td><td rowspan=1 colspan=1>%</td></tr><tr><td rowspan=12 colspan=1>FecalSwab</td><td rowspan=4 colspan=1>Prospective</td><td rowspan=1 colspan=1>Instrument</td><td rowspan=1 colspan=1>0 /1227</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0 / 1227</td><td rowspan=1 colspan=1>0.0</td></tr><tr><td rowspan=1 colspan=1>Invalid a</td><td rowspan=1 colspan=1>0 / 1227</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0 / 1227</td><td rowspan=1 colspan=1>0.0</td></tr><tr><td rowspan=1 colspan=1>Sample too Concentrated b</td><td rowspan=1 colspan=1>5/1227</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>0 /1227</td><td rowspan=1 colspan=1>0.0</td></tr><tr><td rowspan=1 colspan=1>Other</td><td rowspan=1 colspan=1>11 / 1227</td><td rowspan=1 colspan=1>0.9</td><td rowspan=1 colspan=1>3 /1227</td><td rowspan=1 colspan=1>0.2</td></tr><tr><td rowspan=4 colspan=1>ProspectiveArchived</td><td rowspan=1 colspan=1>Instrument</td><td rowspan=1 colspan=1>0/53</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0/53</td><td rowspan=1 colspan=1>0.0</td></tr><tr><td rowspan=1 colspan=1>Invalid</td><td rowspan=1 colspan=1>0 /53</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0 /53</td><td rowspan=1 colspan=1>0.0</td></tr><tr><td rowspan=1 colspan=1>Sample too Concentrated</td><td rowspan=1 colspan=1>0 /53</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0 /53</td><td rowspan=1 colspan=1>0.0</td></tr><tr><td rowspan=1 colspan=1>Other</td><td rowspan=1 colspan=1>0/53</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0 /53</td><td rowspan=1 colspan=1>0.0</td></tr><tr><td rowspan=4 colspan=1>Retrospective</td><td rowspan=1 colspan=1>Instrument</td><td rowspan=1 colspan=1>1 /366</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>0 / 366</td><td rowspan=1 colspan=1>0.0</td></tr><tr><td rowspan=1 colspan=1>Invalid</td><td rowspan=1 colspan=1>1/366</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>0 /366</td><td rowspan=1 colspan=1>0.0</td></tr><tr><td rowspan=1 colspan=1>Sample too Concentrated</td><td rowspan=1 colspan=1>0 /366</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0 /366</td><td rowspan=1 colspan=1>0.0</td></tr><tr><td rowspan=1 colspan=1>Other</td><td rowspan=1 colspan=1>9 /366</td><td rowspan=1 colspan=1>2.5</td><td rowspan=1 colspan=1>5 /366</td><td rowspan=1 colspan=1>1.4</td></tr><tr><td rowspan=12 colspan=1>Para-Pak C&amp;S</td><td rowspan=4 colspan=1>Prospective</td><td rowspan=1 colspan=1>Instrument</td><td rowspan=1 colspan=1>9/740</td><td rowspan=1 colspan=1>1.2</td><td rowspan=1 colspan=1>2/740</td><td rowspan=1 colspan=1>0.3</td></tr><tr><td rowspan=1 colspan=1>Invalid</td><td rowspan=1 colspan=1>5 / 740</td><td rowspan=1 colspan=1>0.7</td><td rowspan=1 colspan=1>5/740</td><td rowspan=1 colspan=1>0.7</td></tr><tr><td rowspan=1 colspan=1>Sample too Concentrated</td><td rowspan=1 colspan=1>35 /740</td><td rowspan=1 colspan=1>4.7</td><td rowspan=1 colspan=1>7/740</td><td rowspan=1 colspan=1>0.9</td></tr><tr><td rowspan=1 colspan=1>Other</td><td rowspan=1 colspan=1>17/740</td><td rowspan=1 colspan=1>2.3</td><td rowspan=1 colspan=1>7/740</td><td rowspan=1 colspan=1>0.9</td></tr><tr><td rowspan=4 colspan=1>ProspectiveArchived</td><td rowspan=1 colspan=1>Instrument</td><td rowspan=1 colspan=1>0 /66</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0 /66</td><td rowspan=1 colspan=1>0.0</td></tr><tr><td rowspan=1 colspan=1>Invalid</td><td rowspan=1 colspan=1>1/66</td><td rowspan=1 colspan=1>1.5</td><td rowspan=1 colspan=1>0 /66</td><td rowspan=1 colspan=1>0.0</td></tr><tr><td rowspan=1 colspan=1>Sample too Concentrated</td><td rowspan=1 colspan=1>1 / 66</td><td rowspan=1 colspan=1>1.5</td><td rowspan=1 colspan=1>0 /66</td><td rowspan=1 colspan=1>0.0</td></tr><tr><td rowspan=1 colspan=1>Other</td><td rowspan=1 colspan=1>1 /66</td><td rowspan=1 colspan=1>1.5</td><td rowspan=1 colspan=1>0 /66</td><td rowspan=1 colspan=1>0.0</td></tr><tr><td rowspan=4 colspan=1>RetrospectiveArchivedFrozen</td><td rowspan=1 colspan=1>Instrument</td><td rowspan=1 colspan=1>1/454</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>0 /454</td><td rowspan=1 colspan=1>0.0</td></tr><tr><td rowspan=1 colspan=1>Invalid</td><td rowspan=1 colspan=1>10 / 454</td><td rowspan=1 colspan=1>2.2</td><td rowspan=1 colspan=1>6/454</td><td rowspan=1 colspan=1>1.3</td></tr><tr><td rowspan=1 colspan=1>Sample too Concentrated</td><td rowspan=1 colspan=1>10 / 454</td><td rowspan=1 colspan=1>2.2</td><td rowspan=1 colspan=1>2/ 454</td><td rowspan=1 colspan=1>0.4</td></tr><tr><td rowspan=1 colspan=1>Other</td><td rowspan=1 colspan=1>25 / 454</td><td rowspan=1 colspan=1>5.5</td><td rowspan=1 colspan=1>17 / 454</td><td rowspan=1 colspan=1>3.7</td></tr></table>

a Internal Control failures with at least one analyte detected and the other analytes reported as ‘invalid’ b Run failures related to ‘sample concentration too high’. These specimens were repeated with 100 microliters as detailed in Appendix C. c Run failures related to workflow checkpoints.

# Co-infections

The QIAstat-Dx Gastrointestinal Panel 2 reported multiple organism detections (i.e., coinfections) for a total of 15 and 29 prospective specimens in FecalSwab and Para-Pak C&S, respectively. This represents $7 . 0 \%$ of positive specimens (15/213) in FecalSwab and $1 7 . 0 \%$ of positive specimens (29/171) in Para-Pak C&S. Most multiple detections in FecalSwab specimens (14/15; $9 3 \%$ ) contained two organisms, while $6 . 7 \%$ (1/15) contained three organisms. In Para-Pak C&S specimens, most multiple detections (22/29; $7 5 . 9 \%$ ) contained two organisms, while $2 4 . 1 \%$ (7/29) contained three organisms. The most common multiple infections are shown in Table 5.38 and Table 5.39 below.

Table 5.38: Most Prevalent Multiple Detection Combinations ${ \geq } 2$ instances) as Determined by the QIAstat-Dx Gastrointestinal Panel 2 in FecalSwab specimen   

<table><tr><td rowspan=1 colspan=1>Multiple Detection Combination</td><td rowspan=1 colspan=1>Number of FecalSwabSpecimens</td></tr><tr><td rowspan=1 colspan=1>Campylobacter + Norovirus GI/GII</td><td rowspan=1 colspan=1>2</td></tr><tr><td rowspan=1 colspan=1>Enterotoxigenic E. coli (ETEC) lt/st + Norovirus GI/GII</td><td rowspan=1 colspan=1>3</td></tr><tr><td rowspan=1 colspan=1>Campylobacter + Rotavirus A</td><td rowspan=1 colspan=1>4</td></tr></table>

Table 5.39: Most Prevalent Multiple Detection Combinations $_ { \perp 2 }$ instances) as Determined by the QIAstat-Dx Gastrointestinal Panel 2 in Para-Pak C&S specimens   

<table><tr><td rowspan=1 colspan=1>Multiple Detection Combination</td><td rowspan=1 colspan=1>Number of Para-Pak C&amp;SSpecimens</td></tr><tr><td rowspan=1 colspan=1>Campylobacter + Enteropathogenic E. coli (EPEC)</td><td rowspan=1 colspan=1>3</td></tr><tr><td rowspan=1 colspan=1>Enteropathogenic E. coli (EPEC) + Salmonella</td><td rowspan=1 colspan=1>3</td></tr><tr><td rowspan=1 colspan=1>Enteropathogenic E. coli (EPEC) + Enterotoxigenic E. coli (ETEC)lt/st</td><td rowspan=1 colspan=1>4</td></tr></table>

The analytes most commonly found in mixed infections in the FecalSwab specimens were Campylobacter (9), Norovirus GI/GII (7), Rotavirus (4) and ETEC (3) as shown in Table 5.4040 while the analytes most commonly found in mixed infections in the Para-Pak C&S specimens were EPEC (17), ETEC (8), Campylobacter (7), Norovirus GI/GII (5), Rotavirus (4) and STEC (5) as shown in Table 5.4141 and Table 5.4242.

Table 5.4040: Prevalence of Analytes in Mixed Infections in FecalSwab specimens as determined by the QIAstat-Dx Gastrointestinal Panel 2   

<table><tr><td rowspan=1 colspan=1>Analyte</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>%</td></tr><tr><td rowspan=1 colspan=1>Adenovirus F40/F41</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>3.2</td></tr><tr><td rowspan=1 colspan=1>Astrovirus</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>3.2</td></tr><tr><td rowspan=1 colspan=1>Campylobacter</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>29.0</td></tr><tr><td rowspan=1 colspan=1>Enterotoxigenic E. coli (ETEC) lt/st</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>9.7</td></tr><tr><td rowspan=1 colspan=1>Giardia lamblia</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>6.5</td></tr><tr><td rowspan=1 colspan=1>Norovirus GI/GII</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>22.6</td></tr><tr><td rowspan=1 colspan=1>Plesiomonas shigelloides</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>3.2</td></tr><tr><td rowspan=1 colspan=1>Rotavirus A</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>12.9</td></tr><tr><td rowspan=1 colspan=1>Shigella/Enteroinvasive E. coli (EIEC)</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>6.5</td></tr><tr><td rowspan=1 colspan=1>Yersinia enterocolitica</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>3.2</td></tr></table>

Table 5.4141: Prevalence of Analytes in Mixed Infections in Para-Pak C&S specimens as determined by the QIAstat-Dx Gastrointestinal Panel 2   

<table><tr><td rowspan=1 colspan=1>Analyte</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>%</td></tr><tr><td rowspan=1 colspan=1>Adenovirus F40/F41</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1.5</td></tr><tr><td rowspan=1 colspan=1>Astrovirus</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1.5</td></tr><tr><td rowspan=1 colspan=1>Campylobacter</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>10.8</td></tr><tr><td rowspan=1 colspan=1>Cryptosporidium</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>3.1</td></tr><tr><td rowspan=1 colspan=1>Cyclospora cayetanensis</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>3.1</td></tr><tr><td rowspan=1 colspan=1>Enteropathogenic E. coli (EPEC)</td><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>26.2</td></tr><tr><td rowspan=1 colspan=1>Enterotoxigenic E. coli (ETEC) lt/st</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>12.3</td></tr><tr><td rowspan=1 colspan=1>Giardia lamblia</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1.5</td></tr><tr><td rowspan=1 colspan=1>Norovirus GI/GII</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>7.7</td></tr><tr><td rowspan=1 colspan=1>Plesiomonas shigelloides</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>10.8</td></tr><tr><td rowspan=1 colspan=1>Rotavirus A</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1.5</td></tr><tr><td rowspan=1 colspan=1>Salmonella</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>6.2</td></tr><tr><td rowspan=1 colspan=1>Shiga-like toxin E. coli (STEC) stx1/stx2</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>7.7</td></tr><tr><td rowspan=1 colspan=1>Shigella/Enteroinvasive E. coli (EIEC)</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>4.6</td></tr><tr><td rowspan=1 colspan=1>Yersinia enterocolitica</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1.5</td></tr></table>

# Contrived Specimens Testing

Several analytes, such as Entamoeba histolytica are so rare that both prospective and archived testing efforts were insufficient to demonstrate system performance. To

supplement the prospective and archived specimens’ test results, an evaluation of contrived specimens was performed. Contrived specimens were prepared using negative residual specimens that had previously tested negative by QIAstat-Dx Gastrointestinal Panel 2 and comparator methods. At least, $50 \%$ of these specimens were spiked at concentrations at concentrations slightly above the Limit of Detection (2x LoD) and the rest at 5x and 10x LoD, using quantified strains for each pathogen. A minimum of 50 contrived specimens were tested for each evaluated analyte. The analyte status of each contrived specimen was blinded to the users analyzing the specimens. Results are summarized in Table 5.4242 Table 5.42.

Table 5.4242: Test Results Summary for Contrived Specimens   

<table><tr><td rowspan=1 colspan=1>QIAstat-Dx GI2 Target</td><td rowspan=1 colspan=1>Medium Brand</td><td rowspan=1 colspan=1>Fraction</td><td rowspan=1 colspan=1>Percentage</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=2 colspan=1>Astrovirus</td><td rowspan=1 colspan=1>FecalSwab</td><td rowspan=1 colspan=1>33 / 34</td><td rowspan=1 colspan=1>97.1</td><td rowspan=1 colspan=1>85.1-99.5</td></tr><tr><td rowspan=1 colspan=1>Para-Pak C&amp;S</td><td rowspan=1 colspan=1>34 / 34</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>89.8-100.0</td></tr><tr><td rowspan=2 colspan=1>Rotavirus A</td><td rowspan=1 colspan=1>FecalSwab</td><td rowspan=1 colspan=1>35 / 35</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>90.1-100.0</td></tr><tr><td rowspan=1 colspan=1>Para-Pak C&amp;S</td><td rowspan=1 colspan=1>34 /35</td><td rowspan=1 colspan=1>97.1</td><td rowspan=1 colspan=1>85.5-99.5</td></tr><tr><td rowspan=2 colspan=1>Plesiomonas shigelloides</td><td rowspan=1 colspan=1>FecalSwab</td><td rowspan=1 colspan=1>33 / 33</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>89.6-100.0</td></tr><tr><td rowspan=1 colspan=1>Para-Pak C&amp;S</td><td rowspan=1 colspan=1>34/35</td><td rowspan=1 colspan=1>97.1</td><td rowspan=1 colspan=1>85.5-99.5</td></tr><tr><td rowspan=2 colspan=1>Yersinia enterocolitica</td><td rowspan=1 colspan=1>FecalSwab</td><td rowspan=1 colspan=1>34 /34</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>89.8-100.0</td></tr><tr><td rowspan=1 colspan=1>Para-Pak C&amp;S</td><td rowspan=1 colspan=1>34 / 35</td><td rowspan=1 colspan=1>97.1</td><td rowspan=1 colspan=1>85.5-99.5</td></tr><tr><td rowspan=2 colspan=1>Shigella/EIEC</td><td rowspan=1 colspan=1>FecalSwab</td><td rowspan=1 colspan=1>35 / 35</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>90.1-100.0</td></tr><tr><td rowspan=1 colspan=1>Para-Pak C&amp;S</td><td rowspan=1 colspan=1>34 / 34</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>89.8-100.0</td></tr><tr><td rowspan=2 colspan=1>Cryptosporidium</td><td rowspan=1 colspan=1>FecalSwab</td><td rowspan=1 colspan=1>27/27</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>87.5-100.0</td></tr><tr><td rowspan=1 colspan=1>Para-Pak C&amp;S</td><td rowspan=1 colspan=1>31/31</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>89.0-100.0</td></tr><tr><td rowspan=2 colspan=1>Cyclospora cayetanensis</td><td rowspan=1 colspan=1>FecalSwab</td><td rowspan=1 colspan=1>26 / 26</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>87.1-100.0</td></tr><tr><td rowspan=1 colspan=1>Para-Pak C&amp;S</td><td rowspan=1 colspan=1>30 / 30</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>88.6-100.0</td></tr><tr><td rowspan=2 colspan=1>Entamoeba histolytica</td><td rowspan=1 colspan=1>FecalSwab</td><td rowspan=1 colspan=1>35 /35</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>90.1-100.0</td></tr><tr><td rowspan=1 colspan=1>Para-Pak C&amp;S</td><td rowspan=1 colspan=1>34 / 35</td><td rowspan=1 colspan=1>97.1</td><td rowspan=1 colspan=1>85.5-99.5</td></tr></table>

# Conclusions

The QIAstat-Dx Gastrointestinal Panel 2 is substantially equivalent to the legally marketed: BioFire Diagnostics LLC FilmArray Gastrointestinal (GI) Panel 510(k) # K140407.