# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATION DECISION SUMMARY

I Background Information:

A 510(k) Number K220062   
B Applicant QIAGEN GmbH   
C Proprietary and Established Names QIAstat-Dx Gastrointestinal Panel 2   
D Regulatory Information

<table><tr><td rowspan=1 colspan=1>ProductCode(s)</td><td rowspan=1 colspan=1>Classification</td><td rowspan=1 colspan=1>RegulationSection</td><td rowspan=1 colspan=1>Panel</td></tr><tr><td rowspan=1 colspan=1>PCH</td><td rowspan=1 colspan=1>Class II</td><td rowspan=1 colspan=1>21 CFR 866.3990 -GastrointestinalMicroorganism MultiplexNucleic Acid-Based Assay</td><td rowspan=1 colspan=1>MI - Microbiology</td></tr></table>

# II Submission/Device Overview:

A multiplexed nucleic acid-based test intended for use with QIAstat-Dx system for the qualitative in vitro detection and identification of multiple bacteria, viruses, and parasites. The QIAstat-Dx Gastrointestinal Panel 2 assay is performed directly from stool samples collected in Para-Pak C&S (Meridian Bioscience) and FecalSwab (COPAN) transport media.

# A Purpose for Submission:

To obtain a substantial equivalence determination for the QIAstat-Dx Gastrointestinal Panel 2 microorganism multiplex nucleic acid-based assay.

# B Measurand:

Targeted nucleic acid sequences of the following gastrointestinal microorganisms:

Adenovirus F40/F41 • Astrovirus

• Norovirus GI/GII   
• Rotavirus A   
• Campylobacter (C. jejuni, C. coli and C. upsaliensis) Shigella/Enteroinvasive Escherichia coli (EIEC) Enteropathogenic Escherichia coli (EPEC) Enterotoxigenic Escherichia coli (ETEC) lt/st Shiga-like toxin-producing Escherichia coli (STEC) stx1/stx2\* (including specific identification of E. coli O157 serogroup within STEC)   
• Salmonella Plesiomonas shigelloides   
• Yersinia enterocolitica   
• Cryptosporidium   
• Cyclospora cayetanensis Entamoeba histolytica   
• Giardia lamblia

# C Type of Test:

A multiplex nucleic acid-based test intended for use with the QIAstat-Dx system for the qualitative in vitro detection and identification of multiple bacteria, viruses, and parasites in preserved stool samples collected from individuals suspected of gastrointestinal infection.

# III Intended Use/Indications for Use:

A Intended Use(s):

The QIAstat-Dx Gastrointestinal Panel 2 is a multiplexed nucleic acid test intended for use with the QIAstat-Dx Analyzer 1.0. for the simultaneous in vitro qualitative detection and identification of nucleic acids from multiple viruses, bacteria, and parasites directly from preserved stool samples (Para-Pak C&S or FecalSwab) obtained from individuals with signs and/or symptoms of gastrointestinal infection. The following viruses, bacteria (including several diarrheagenic $E$ . coli/ Shigella pathotypes), and parasites are identified with the QIAstat-Dx Gastrointestinal Panel 2:

<table><tr><td rowspan=1 colspan=1>Pathogen</td><td rowspan=1 colspan=1>Para-Pak C&amp;S</td><td rowspan=1 colspan=1>FecalSwab</td></tr><tr><td rowspan=1 colspan=1>Adenovirus F40/F41</td><td rowspan=1 colspan=1>✓</td><td rowspan=1 colspan=1>✓</td></tr><tr><td rowspan=1 colspan=1>Astrovirus</td><td rowspan=1 colspan=1>✓</td><td rowspan=1 colspan=1>✓</td></tr><tr><td rowspan=1 colspan=1>Norovirus GI/GII</td><td rowspan=1 colspan=1>✓</td><td rowspan=1 colspan=1>✓</td></tr><tr><td rowspan=1 colspan=1>Rotavirus A</td><td rowspan=1 colspan=1>✓</td><td rowspan=1 colspan=1>✓</td></tr><tr><td rowspan=1 colspan=1>Campylobacter (C. jejuni, C.coli and C. upsaliensis)</td><td rowspan=1 colspan=1>√</td><td rowspan=1 colspan=1>✓</td></tr><tr><td rowspan=1 colspan=1>Shigella/EnteroinvasiveEscherichia coli (EIEC)</td><td rowspan=1 colspan=1>✓</td><td rowspan=1 colspan=1>✓</td></tr><tr><td rowspan=1 colspan=1>Enteropathogenic Escherichiacoli (EPEC)</td><td rowspan=1 colspan=1>✓</td><td rowspan=1 colspan=1>Not Reported</td></tr></table>

\*Also known as Giardia intestinalis and Giardia duodenalis   

<table><tr><td rowspan=1 colspan=1>Enterotoxigenic Escherichia coli(ETEC) lt/st</td><td rowspan=1 colspan=1>√</td><td rowspan=1 colspan=1>L</td></tr><tr><td rowspan=1 colspan=1>Shiga-like toxin-producingEscherichia coli (STEC)stx1/stx2 (including specificidentification of E. coli O157serogroup within STEC)</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Not Reported</td></tr><tr><td rowspan=1 colspan=1>Salmonella</td><td rowspan=1 colspan=1>✓</td><td rowspan=1 colspan=1>✓</td></tr><tr><td rowspan=1 colspan=1>Plesiomonas shigelloides</td><td rowspan=1 colspan=1>✓</td><td rowspan=1 colspan=1>✓</td></tr><tr><td rowspan=1 colspan=1>Yersinia enterocolitica</td><td rowspan=1 colspan=1>✓</td><td rowspan=1 colspan=1>✓</td></tr><tr><td rowspan=1 colspan=1>Cryptosporidium</td><td rowspan=1 colspan=1>✓</td><td rowspan=1 colspan=1>√</td></tr><tr><td rowspan=1 colspan=1>Cyclospora cayetanensis</td><td rowspan=1 colspan=1>✓</td><td rowspan=1 colspan=1>✓</td></tr><tr><td rowspan=1 colspan=1>Entamoeba histolytica</td><td rowspan=1 colspan=1>✓</td><td rowspan=1 colspan=1>✓</td></tr><tr><td rowspan=1 colspan=1>Giardia lamblia*</td><td rowspan=1 colspan=1>✓</td><td rowspan=1 colspan=1>✓</td></tr></table>

Concomitant culture is necessary for organism recovery and further typing of bacterial agents. The QIAstat-Dx Gastrointestinal Panel 2 is indicated as an aid in the diagnosis of specific agents of gastrointestinal illness, in conjunction with other clinical, laboratory, and epidemiological data. Positive results do not rule-out co-infection with organisms not detected by the QIAstat-Dx Gastrointestinal Panel 2. The organisms detected may not be the sole or definitive cause of the disease.

Negative QIAstat-Dx Gastrointestinal Panel 2 results in the setting of clinical illness compatible with gastroenteritis may be due to infection by pathogens that are not detected by this test or non-infectious causes such as ulcerative colitis, irritable bowel syndrome, or Crohn’s disease.

# B Special Conditions for Use Statement(s):

Rx - For Prescription Use Only

For in vitro Diagnostic Use Only

# C Special Instrument Requirements:

The QIAstat-Dx Gastrointestinal Panel 2 is intended for use only with the QIAstat-Dx Analyzer 1.0.

# IV Device/System Characteristics:

# A Device Description:

The QIAstat-Dx Gastrointestinal Panel 2 assay is run on the QIAstat-Dx Analyzer 1.0. The QIAstat-Dx Gastrointestinal Panel 2 cartridge is a single-use cartridge that includes all reagents needed for nucleic acid extraction, nucleic acid amplification, and detection of bacteria, viruses or parasites associated with gastrointestinal infection. Testing requires a 200 $\mu \mathrm { L }$ specimen volume and minimal hands-on time, and the results are available in approximately 78 minutes.

# B Principle of Operation:

The QIAstat-Dx Gastrointestinal Panel 2 is part of the QIAstat-Dx system and works with the QIAstat-Dx Analyzer 1.0.

The QIAstat-Dx Gastrointestinal Panel 2 is intended to be used with stool samples collected in Para-Pak C&S or FecalSwab transport media.

Once the cartridge has been inserted into the instrument, the test starts automatically and runs for approximately 78 minutes. he QIAstat-Dx Analyzer 1.0 automatically interprets test results and displays a summary on the analyzer display screen.

# Sample Pre-treatment for PCR Inhibitors removal

Following insertion of the cartridge, the buffer located in Reservoir 1 is added inside the lysis chamber and homogenized with the sample using a rotor in the presence of silica beads. This enables separation of commonly found inhibitory substances in stool from the DNA/RNA by chemical means.

# Resuspension of Internal Control (IC) and Proteinase K

Following sample pre-treatment, the IC and Proteinase K is resuspended with the buffer located in Reservoir 2 (resuspension buffer). The buffer from Reservoir 2 is added to the interconnected IC cavity and Proteinase K cavity and transferred repeatedly between the Transfer Chamber and the cavities to ensure resuspension. The resuspended IC and Proteinase K are transferred to the sample cavity.

# Cell Lysis

Primary lysis of the cells and analytes present in a stool sample and IC occurs by a combination of chemical and mechanical processes using a rotor inside the lysis chamber in the presence of silica beads and a buffer that acts as a chemical agent in aiding the mechanical process. The fast movement of the rotor in the presence of the silica beads results in sample agitation, which creates turbulence and shear forces that favor the lysis of the cell wall.

After mechanical lysis is completed, the primary lysate is transferred to the purification chamber through a frit with $2 \AA { \cdot } 5 \mu \mathrm { m }$ pore size. The second lysis buffer (from Reservoir 3) is added to the primary lysate to complete chemical lysis.

# Purification

Binding reagent (from Reservoir 4) is added to the lysate in the purification chamber, and the mix is passed through the silica membrane. In this process, the DNA/RNA molecules stick to the membrane, and the remaining components of the lysate are delivered to the waste chamber. Then the membrane is washed with a first washing buffer (from Reservoir 5) to wash away proteins. This is followed by a second washing step with a second washing buffer (from Reservoir 6) to remove any remaining substances other than the nucleic acids. A subsequent drying step

eliminates volatile substances from the silica membrane. Prior to the elution step, the Transfer Chamber is rinsed with the rinsing buffer (from Reservoir 7) in order to remove any potential inhibitors from previous processing steps. At the end of the process, the nucleic acids are released from the membrane using an elution buffer (from Reservoir 8). The eluate is collected in the Transfer Chamber.

# Rehydration of Master Mix

A defined volume $( 1 3 5 \mu \mathrm { L } )$ of the eluate is delivered to the reservoir of the Dry Chemistry Container (DCC) to rehydrate the Master Mix. Any remaining eluate is transferred to the Reservoir 7. The eluate/Master Mix solution is mixed by repeated transfer between the Transfer Chamber and the DCC.

# Aliquoting and PCR

Defined aliquots $( 1 5 \mu \mathrm { L } )$ of mixed eluate/Master Mix are sequentially transferred from the Transfer Chamber to each of eight Reaction Chambers containing the specified, air dried primers and probes.

Within each Reaction Chamber, a reverse-transcription step followed by real time, multiplex PCR (“rtPCR”) is performed. Increase in fluorescence (indicative of detection of each target analyte) is detected directly within each Reaction Chamber.

The rtPCR process is conducted by two submodules of the QIAstat-Dx Analyzer 1.0: the Thermal Cycler and the qPCR Sensor.

# Components Description

QIAstat-Dx Gastrointestinal Panel 2 Cartridge:

The QIAstat-Dx Gastrointestinal Panel 2 cartridge is a disposable plastic device that allows performing fully automated molecular assays. The main features of the QIAstat-Dx Gastrointestinal Panel 2 cartridge for the Gastrointestinal assay include the ability to test liquid samples and the capacity to store all necessary reagents within the cartridge needed for such testing. All sample preparation and assay steps will be performed within the cartridge.

All the reagents required for the complete execution of the test are pre-loaded and self-contained in the QIAstat-Dx Gastrointestinal Panel 2. The user does not need to manipulate any reagents. During the test, reagents are handled by pneumatically operated microfluidics without any direct contact with the user or the analyzer actuators. This eliminates any possibility of exposure of the user or the analyzer to chemicals contained in the cartridge during the test and up to the disposal of used cartridges.

Reagents may be found in three different physical forms: liquid, air-dried on surfaces or lyophilized powder cake.

Within the cartridge, multiple steps are automatically performed in sequence by using pneumatic pressure and a multiport valve to transfer sample and fluids via the Transfer Chamber to their intended destinations.

# QIAstat-Dx Analyzer 1.0

The QIAstat-Dx Analyzer 1.0 is the unit that hosts a cartridge and, on command from the user, is able to run predefined assay protocols. The software specific to this test is pre-loaded on the QIAstat-Dx Analyzer 1.0.

# Other Materials

Each QIAstat-Dx Gastrointestinal Panel 2 cartridge will be used with a transfer pipette (provided with device). The stool sample from the patient will be collected in Cary-Blair medium following the manufacturer’s instructions for use (not provided with device).

QIAstat-Dx Analyzer 1.0 – the QIAstat-Dx Gastrointestinal Panel 2 cartridge can only be run on the QIAstat-Dx Analyzer 1.0.

# Calibrators and/or Controls

Blank controls are not applicable to the device because it is a single test disposable cartridge. Negative and positive external controls are recommended but not provided with the QIAstat-Dx Gastrointestinal Panel 2.

QIAGEN provides an internal control (“IC”) within the QIAstat-Dx Gastrointestinal Panel 2 cartridge which provides a full process control covering lysis, nucleic acid purification, reverse transcription and DNA amplification. The IC is Schizosaccharomyces pombe. The IC is located in the IC cavity and is mixed with the sample during sample preparation and the eluate is mixed with the Master Mix, then aliquoted in all Reaction Chambers. The results screen displays a message indicating that the Internal Control “Passed” when the test is run successfully. An IC message of “Failed” indicates that the internal control was not amplified; ‘Positive’ test results are then reported as POSITIVE\* (positive with warning), all ‘Negative’ results are invalid.

The QIAstat-Dx Analyzer 1.0 is provided factory calibrated and does not require user calibration. The QIAstat-Dx Analyzer 1.0 includes self-check controls to verify the performance of all sensors and actuators and will alert the user in case of failure.

# V Substantial Equivalence Information:

A Predicate Device Name(s): FilmArray Gastrointestinal (GI) Panel

B Predicate 510(k) Number(s): K140407

C Comparison with Predicate(s):

<table><tr><td colspan="1" rowspan="1">Device &amp; PredicateDevice(s):</td><td colspan="1" rowspan="1">K220062</td><td colspan="1" rowspan="1">K140407</td></tr><tr><td colspan="1" rowspan="1">Device Trade Name</td><td colspan="1" rowspan="1">QIAstat-Dx GastrointestinalPanel 2</td><td colspan="1" rowspan="1">FilmArray Gastrointestinal (GI)Panel</td></tr><tr><td colspan="3" rowspan="1">General Device Characteristics Similarities</td></tr><tr><td colspan="1" rowspan="1">Pathogens Detected</td><td colspan="1" rowspan="1">Adenovirus F40/F41,AstrovirusNorovirus GI/GIIRotavirus ACampylobacter (C. jejuni, C.coli, C. upsaliensis),Shigella/Enteroinvasive E. coli(EIEC)Enteropathogenic E. coli(EPEC)Enterotoxigenic E. coli(ETEC)Shiga-like toxin-producingE.coli (STEC) stx1/stx2(including specificidentification of E. coli O157serogroup within STEC)SalmonellaPlesiomonas shigelloidesYersinia enterocoliticaCryptosporidiumCyclospora cayetanensisEntamoeba histolyticaGiardia lamblia</td><td colspan="1" rowspan="1">Same plus additional pathogensdetected (See below fordifferences )</td></tr><tr><td colspan="1" rowspan="1">Analyte</td><td colspan="1" rowspan="1">DNA/RNA</td><td colspan="1" rowspan="1">DNA/RNA</td></tr><tr><td colspan="1" rowspan="1">Amplification andDetectionTechnology</td><td colspan="1" rowspan="1">PCR</td><td colspan="1" rowspan="1">PCR</td></tr><tr><td colspan="1" rowspan="1">TechnologicalPrinciples</td><td colspan="1" rowspan="1">Multiplex nucleic acid test</td><td colspan="1" rowspan="1">Same (See below fordifferences)</td></tr><tr><td colspan="1" rowspan="1">IntendedUse/Indications ForUse</td><td colspan="1" rowspan="1">The QIAstat-DxGastrointestinal Panel 2 is amultiplexed nucleic acid testintended for use with theQIAstat-Dx Analyzer 1.0 forthe simultaneous qualitativedetection and identification ofnucleic acids from multipleviruses, bacteria, and parasitesdirectly from preserved stoolsamples (Para-Pak C&amp;S andFecalSwab) obtained fromindividuals with signs and/orsymptoms of gastrointestinalinfection. The followingviruses, bacteria (including</td><td colspan="1" rowspan="1">The FilmArray Gastrointestinal(GI) Panel is a qualitativemultiplexed nucleic acid-basedtest intended for use with theFilmArray Instrument for thesimultaneous detection andidentification of nucleic acidsfrom multiple bacteria, viruses,and parasites directly from stoolsamples in Cary Blair medium,obtained from individuals withsigns and/or symptoms ofgastrointestinal infection. Thefollowing bacteria (includingseveral diarrheagenic E.coli/Shigella pathotypes),</td></tr><tr><td colspan="3"></td></tr><tr><td>ice &amp; Predicate ice(s):</td><td>K220062 several diarrheagenic</td><td>K140407 parasites, and viruses are</td></tr><tr><td></td><td>E.coli/Shigella pathotypes), and parasites are identified with the QIAstat-Dx Gastrointestinal Panel 2: Adenovirus F40/F41 Astrovirus Norovirus GI/GII Rotavirus A • Campylobacter (C. jejuni, C. coli and C. upsaliensis) Shigella/Enteroinvasive Escherichia coli (EIEC) Enteropathogenic Escherichia coli (EPEC) (only with Para-Pak C&amp;S, not reported for FecalSwab) Enterotoxigenic Escherichia coli (ETEC) lt/st Shiga-like toxin-producing Escherichia coli (STEC) stx1/stx2* (including specific identification of E. coli 0157 serogroup within STEC) (only with Para-Pak C&amp;S, not reported for FecalSwab) Salmonella Plesiomonas shigelloides Yersinia enterocolitica Cryptosporidium • Cyclospora cayetanensis •Entamoeba histolytica Giardia lamblia (Also known as G. intestinalis and G. duodenalis) Concomitant culture is necessary for organism recovery and further typing of bacterial agents. The QIAstat-Dx Gastrointestinal Panel 2 is indicated as an aid in the diagnosis of specific agents of gastrointestinal illness in</td><td>identified using the FilmArray GI Panel: Campylobacter (C. jejuni/C. coli/C. upsaliensis Clostridium difficile (C. difficile) toxin A/B Plesiomonas shigelloides Salmonella Vibrio (V&gt; parahaemolyticus/V. vulnificus/ V. cholerae) including specific identification of Vibrio cholerae Yersinia enterocolitica Enteroaggregative Escherichia coli (EAEC) Enteropathogenic Escherichia coli (EPEC) Enterotoxigenic Escherichia coli (ETEC) lt/st Shiga-like toxin-producing Escherichia coli (STEC) stx1/stx2 (including specific identification of the E. coli 0157 serogroup within STEC) Shigella/Enteroinvasive Escherichia coli (EIEC) Cryptosporidium Cyclospora cayetanensis Entamoeba histolytica Giardia lamblia (also known as G. intestinalis and G. duodenalis) Adenovirus F 40/41 Astrovirus Norovirus GI/GII Rotavirus A Sapovirus (Genogroups I, II, IV and V) The FilmArray Gastrointestinal Panel is indicated as an aid in the diagnosis of specific agents of gastrointestinal illness and results are meant to be used in conjunction with other clinical,</td></tr><tr><td colspan="1" rowspan="1">Device &amp; PredicateDevice(s):</td><td colspan="1" rowspan="1">K220062</td><td colspan="1" rowspan="1">K140407</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">conjunction with other clinical,laboratory, andepidemiological data. Positiveresults do not rule-out co-infection with organisms notdetected by the QIAstat-DxGastrointestinal Panel 2. Theorganisms detected may not bethe sole or definitive cause ofthe disease.Negative QIAstat-DxGastrointestinal Panel 2 resultsin the setting of clinical illnesscompatible with gastroenteritismay be due to infection bypathogens that are not detectedby this assay test or non-infectious causes such asulcerative colitis, irritable bowelsyndrome, or Crohn's disease.</td><td colspan="1" rowspan="1">laboratory, and epidemiologicaldata. Positive results do not ruleout co-infection with organismsnot included in the FilmArrayGI Panel. The agent detectedmay not be the definite cause ofthe disease.Concomitant culture is necessaryfor organism recovery and furthertyping of bacterial agents.This device is not intended tomonitor or guide treatment for C.difficile infection.Due to the small number ofpositive specimens collected forcertain organisms during theprospective clinical study,performance characteristics for E.coli O157, Plesiomonasshigelloides, Yersiniaenterocolitica, Astrovirus, andRotavirus A were establishedprimarily with retrospectiveclinical specimens.Performance characteristics forEntamoeba histolytica, andVibrio (V parahaemolyticus, V.vulnificus, and Vibrio cholerae)were established primarily usingcontrived clinical specimens.Negative FilmArray GI Panelresults in the setting of clinicalillness compatible withgastroenteritis may be due toinfection by pathogens that arenot detected by this test or non-infectious causes such asulcerative colitis, irritable bowelsyndrome, or Crohn's disease.A gastrointestinal microorganismmultiplex nucleic acid-basedassay also aids in the detectionand identification of acutegastroenteritis in the context ofoutbreaks.</td></tr><tr><td colspan="1" rowspan="1">Assay Controls</td><td colspan="1" rowspan="1">One internal control in eachcartridge to control for sample</td><td colspan="1" rowspan="1">Two controls are included in eachreagent pouch to control for</td></tr><tr><td colspan="1" rowspan="1">b</td><td colspan="1" rowspan="1">processing that is subjected toall nucleic acid extraction andamplification steps similar topatient samples. Externalcontrols are not provided withthe QIAstat-Dx GastrointestinalPanel 2. Labeling recommendsuse of negative and positiveexternal controls regularly. Usetransport media as the externalNegative Control, andpreviously characterizedpositive samples or negativesample spiked with wellcharacterized target organismsas external positive controls.</td><td colspan="1" rowspan="1">sample processing and bothstages of PCR and melt analysis.Labeling recommends the use ofexternal positive and negativecontrols regularly. Enterictransport media can be used as anexternal negative control, andpreviously characterized positivesamples or negative samplesspiked with well characterizedorganisms as external positivecontrols.</td></tr><tr><td colspan="3" rowspan="1">General Device Characteristic Differences</td></tr><tr><td colspan="1" rowspan="1">Assay Targets</td><td colspan="1" rowspan="1">No additional targets to thoselisted above which are incommon with the predicatedevice.</td><td colspan="1" rowspan="1">The FilmArray GastrointestinalPanel has four additionaltargets: Clostridium difficile (C.difficile) toxin A/B,Enteroaggregative Escherichiacoli (EAEC), Sapovirus and ageneric Vibrio target whichdifferentiates Vibrio cholerae.</td></tr><tr><td colspan="1" rowspan="1">Specimen Types</td><td colspan="1" rowspan="1">Human stool sample collectedin Para-Pak C&amp;S andFecalSwab transport media.</td><td colspan="1" rowspan="1">Human stool sample collected inCary Blair transport media</td></tr><tr><td colspan="1" rowspan="1">Nucleic AcidExtraction</td><td colspan="1" rowspan="1">Extraction of nucleic acids usingsilica membrane</td><td colspan="1" rowspan="1">Extraction of nucleic acids usingmagnetic beads</td></tr><tr><td colspan="1" rowspan="1">Technology</td><td colspan="1" rowspan="1">QIAstat-Dx GastrointestinalPanel 2 detection of amplifiedtargets uses an increase influorescence due to specificprobe binding to generate theassay results.</td><td colspan="1" rowspan="1">The FilmArray GastrointestinalPanel uses melting curve analysisto confirm the identity ofamplified targets to generateassay results.</td></tr><tr><td colspan="1" rowspan="1">Amplification andDetection InstrumentSystem</td><td colspan="1" rowspan="1">QIAstat-Dx Analyzer 1.0</td><td colspan="1" rowspan="1">FilmArray Instrument</td></tr></table>

# VI Standards/Guidance Documents Referenced:

# VII Performance Characteristics (if/when applicable):

# A Analytical Performance:

1. Reproducibility/Repeatability:

Reproducibility Study

Reproducibility testing evaluating contrived samples was performed at three test sites, including one internal site (Site A) and two external sites (Site B and Site C). The study incorporated a range of factors that may introduce variability in assay results, including sites, days, replicates, cartridge lots, operators, and QIAstat-Dx analyzers. For each site, testing was performed across five nonconsecutive days with two replicates per day (total of 30 replicates per target, concentration, and site), four QIAstat-Dx Analyzers (two analyzers per operator and per site), and at least two operators on each testing day. A total of five sample mixes (including evaluation of analyte levels corresponding to 1x LoD and 3x LoD as well as negative samples containing no analyte) were prepared in a matrix consisting of pooled stool in Para-Pak C&S transport medium. For each mix, six replicates were tested and evaluated, and data obtained at all three sites were compiled to calculate the two-sided $9 5 \%$ Confidence Interval by target and concentration.

The percent agreement with expected results for all analytes was $2 9 5 \%$ for samples tested at 1x and 3x LoD, meeting the acceptance criteria for the study. All analytes evaluated were detected within performance expectations for qualitative agreement across all concentrations tested. The Reproducibility Study site-to-site qualitative results (agreements with expected results) are presented in Table 1 below.

Table 1: Reproducibility Study, Qualitative Results   

<table><tr><td colspan="1" rowspan="2">Pathogen Tested</td><td colspan="1" rowspan="2">ConcentrationTested</td><td colspan="1" rowspan="2">ExpectedResult</td><td colspan="4" rowspan="1">% Agreement with Expected Result</td></tr><tr><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">All Sites(95%ConfidenceInterval)</td></tr><tr><td colspan="1" rowspan="3">Adenovirus F41ZeptoMetrix0810085CF</td><td colspan="1" rowspan="1">3x LoD</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">90/90100%(95.98 -100%)</td></tr><tr><td colspan="1" rowspan="1">1x LoD</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">90/90100%(95.98 -100%)</td></tr><tr><td colspan="1" rowspan="1">None</td><td colspan="1" rowspan="1">NotDetected</td><td colspan="1" rowspan="1">29/30100%</td><td colspan="1" rowspan="1">29/30100%</td><td colspan="1" rowspan="1">29/30100%</td><td colspan="1" rowspan="1">87/90196.7%(90.98 98.9%)</td></tr><tr><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">All Sites(95%ConfidenceInterval</td></tr><tr><td colspan="1" rowspan="3">CampylobacterZeptoMetrix801650</td><td colspan="1" rowspan="1">3x LoD</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">90/90100%(95.98 -100%)</td></tr><tr><td colspan="1" rowspan="1">1x LoD</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">90/90100%(95.98 -100%)</td></tr><tr><td colspan="1" rowspan="1">None</td><td colspan="1" rowspan="1">NotDetected</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">90/90100%(95.98 -100%</td></tr><tr><td colspan="1" rowspan="3">Escherichia coliEPECZeptoMetrix801747</td><td colspan="1" rowspan="1">3x LoD</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">90/90100%(95.98 -100%)</td></tr><tr><td colspan="1" rowspan="1">1x LoD</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">30/3000%</td><td colspan="1" rowspan="1">29/3096.67%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">89/9098.89%(93.96 99.97%)</td></tr><tr><td colspan="1" rowspan="1">None</td><td colspan="1" rowspan="1">NotDetected</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">90/90100%(95.98 -100%)</td></tr><tr><td colspan="1" rowspan="3">EntamoebahistolyticaATCC30459</td><td colspan="1" rowspan="1">3x LoD</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">90/90100%(95.98 -100%</td></tr><tr><td colspan="1" rowspan="1">1x LoD</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">29/3096.67%</td><td colspan="1" rowspan="1">89/9098.89%(93.96 99.97%)</td></tr><tr><td colspan="1" rowspan="1">None</td><td colspan="1" rowspan="1">NotDetected</td><td colspan="1" rowspan="1">30/3000%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">90/90100%(95.98 -100%</td></tr><tr><td colspan="1" rowspan="3">Giardia lamblia²ATCC30888</td><td colspan="1" rowspan="1">3x LoD</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/3000%</td><td colspan="1" rowspan="1">90/90100%(95.98 -100%)</td></tr><tr><td colspan="1" rowspan="1">1x LoD</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">90/90100%(95.98 -100%)</td></tr><tr><td colspan="1" rowspan="1">None</td><td colspan="1" rowspan="1">NotDetected</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">90/90100%(95.98 -100%)</td></tr><tr><td colspan="1" rowspan="3">Norovirus GIIZeptoMetrix080087CF</td><td colspan="1" rowspan="1">3x LoD</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">90/90100%(95.98 -100%)</td></tr><tr><td colspan="1" rowspan="1">1x LoD</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">29/3096.67%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">89/9098.89%(93.96 99.97%)</td></tr><tr><td colspan="1" rowspan="1">None</td><td colspan="1" rowspan="1">NotDetected</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">90/90100%(95.98 -100%)</td></tr><tr><td colspan="1" rowspan="3">Rotavirus A3ZeptoMetrix0810280CF</td><td colspan="1" rowspan="1">3x LoD</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">29/3096.67%</td><td colspan="1" rowspan="1">29/3096.67%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">88/9097.8%(92.20 99.73%)</td></tr><tr><td colspan="1" rowspan="1">1x LoD</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">23/3076.67%</td><td colspan="1" rowspan="1">26/3086.67%</td><td colspan="1" rowspan="1">12/12100%</td><td colspan="1" rowspan="1">61/7284.7%(74.3192.12%)</td></tr><tr><td colspan="1" rowspan="1">None</td><td colspan="1" rowspan="1">NotDetected</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">90/90100%(95.98 -100%)</td></tr><tr><td colspan="1" rowspan="3">Escherichia coli(STEC) 0157:H74ZeptoMetrix0801622</td><td colspan="1" rowspan="1">3x LoD</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">90/90100%(95.98 -100%)</td></tr><tr><td colspan="1" rowspan="1">1x LoD</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">29/3096.67%</td><td colspan="1" rowspan="1">89/9098.89%(93.96 99.97%)</td></tr><tr><td colspan="1" rowspan="1">None</td><td colspan="1" rowspan="1">NotDetected</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">90/90100%(95.98 -100%)</td></tr><tr><td colspan="1" rowspan="3">Escherichia coli (STEC)*stx1/stx2ZeptoMetrix0801622</td><td colspan="1" rowspan="1">3x LoD</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">90/90100%(95.98 -100%)</td></tr><tr><td colspan="1" rowspan="1">1x LoD</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">90/90100%(95.98 -100%</td></tr><tr><td colspan="1" rowspan="1">None</td><td colspan="1" rowspan="1">NotDetected</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">90/90100%(95.98 -100%)</td></tr><tr><td colspan="1" rowspan="3">SalmonellaentericaZeptoMetrix0801437</td><td colspan="1" rowspan="1">3x LoD</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">90/90100%(95.98 -100%)</td></tr><tr><td colspan="1" rowspan="1">1x LoD</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">29/3096.67%</td><td colspan="1" rowspan="1">29/3096.67%</td><td colspan="1" rowspan="1">88/9097.78%(92.20 -99.73%)</td></tr><tr><td colspan="1" rowspan="1">None</td><td colspan="1" rowspan="1">NotDetected</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">90/90100%(95.98 100%)</td></tr><tr><td colspan="1" rowspan="3">YersiniaenterocoliticaZeptometrix0801734</td><td colspan="1" rowspan="1">3x LoD</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">90/90100%(95.98 -100%</td></tr><tr><td colspan="1" rowspan="1">1x LoD</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">90/90100%(95.98 -100%)</td></tr><tr><td colspan="1" rowspan="1">None</td><td colspan="1" rowspan="1">NotDetected</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">30/30100%</td><td colspan="1" rowspan="1">90/90100%(95.98 100%)</td></tr></table>

1 Three (3) Adenovirus F40/41 false positives were observed when testing negative sample. Retesting of the three samples resulted in the expected negative results. 2 One (1) Giardia lamblia false positive was observed when testing a positive sample not containing the pathogen. Repeat testing of this sample resulted in the expected negative result. 3 The Reproducibility study was fully re-tested for Rotavirus A with a new sample set due to an unexpected number of false negatives for Rotavirus A at the 1x LoD concentration. This was observed with during an interim data evaluation (61/72, $8 4 . 7 \%$ ) and that was attributed to the sample manufacture and while unrelated to the study workflow variables (operator, lot, day, instrument and site). Test runs derived from Rotavirus A new sample set resulted in 90/90 ( $100 \%$ ; $9 5 . 9 8 \substack { - 1 0 0 \% }$ CI) for the $3 \mathrm { x }$ LoD and 89/90 ( $9 8 . 8 9 \%$ ; $9 3 . 9 6 \substack { - 9 9 . 9 7 \% }$ CI) for the 1x LoD. During this testing, one (1) Campylobacter false positive was observed Retesting of this sample resulted in the expected negative result. 4 Only applicable for Para-Pak C&S samples

During the study, there were three false positive results for Adenovirus F40/41 in negative (nonspiked) samples as well as one false positive result for Campylobacter and one false positive result for Giardia lamblia in samples containing other analytes. Repeat testing of the same five samples yielded the expected negative result for each applicable analyte. The investigation into the unexpected false positive results showed that these events were observed in five different analyzers. For the sample that was false positive for Campylobacter, the sample tested immediately prior was also positive for Campylobacter. For the other four false positive samples, the samples run immediately prior were not positive for the applicable pathogen suggesting that these unexpected results were likely not due to inter-run carryover.

A total of six cartridge failures $( 6 / 6 4 5 , 0 . 9 3 \% )$ were observed during the study. Failures included cartridge errors (5) and an instrument error (1). No invalid results due to internal control failure were observed.

An additional analysis of variation was conducted across specific potential sources of variability such as sites, days, replicates, cartridge lots, operators, and QIAstat-Dx analyzers. Results from the analysis showed no significant contribution to variability (Standard Deviation and Coefficient of Variation values below 1 and $5 \%$ , respectively) caused by any of the assessed variables. Overall, Ct variability was low, and the study demonstrates assay variability within an acceptable range. Results are presented in Table 2 below.

Table 2: Reproducibility Study, Numerical Analysis of Variability   

<table><tr><td rowspan=1 colspan=5></td><td rowspan=1 colspan=7>Variance component (SD, %CV*)</td></tr><tr><td rowspan=1 colspan=1>Pathogen</td><td rowspan=1 colspan=1>tested</td><td rowspan=1 colspan=1>(n)</td><td rowspan=1 colspan=1>(n)</td><td rowspan=1 colspan=1>Mean</td><td rowspan=1 colspan=1>Site</td><td rowspan=1 colspan=1>Lot</td><td rowspan=1 colspan=1>Day</td><td rowspan=1 colspan=1>BetweenOperator</td><td rowspan=1 colspan=1>BetweenInstrument</td><td rowspan=1 colspan=1>BetweenReplicate</td><td rowspan=1 colspan=1>Total**</td></tr><tr><td rowspan=2 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>1x LoD</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>34.15</td><td rowspan=1 colspan=1>(0.2979,0.87%)</td><td rowspan=1 colspan=1>(0.1630,0.48%)</td><td rowspan=1 colspan=1>0.1135,0.33%</td><td rowspan=1 colspan=1>(0.0000,0.00%%</td><td rowspan=1 colspan=1>(0.1883,0.55%)</td><td rowspan=1 colspan=1>(0.6153,1.80%</td><td rowspan=1 colspan=1>(0.7067,2.07%</td></tr><tr><td rowspan=1 colspan=1>3x LoD</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>32.14</td><td rowspan=1 colspan=1>0.0000,0.00%)</td><td rowspan=1 colspan=1>(0.0741,0.23%</td><td rowspan=1 colspan=1>(0.0000,0.00%)</td><td rowspan=1 colspan=1>(0.0000,0.00%%</td><td rowspan=1 colspan=1>(0.0000,0.0%</td><td rowspan=1 colspan=1>(0.3716,1.16%)</td><td rowspan=1 colspan=1>(0.3766,1.17%)</td></tr><tr><td rowspan=2 colspan=1>Campylobacter</td><td rowspan=1 colspan=1>1x LoD</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>32.64</td><td rowspan=1 colspan=1>(0.1289,0.39%</td><td rowspan=1 colspan=1>(0.0000,0.00%)</td><td rowspan=1 colspan=1>0.0200,0.06%)</td><td rowspan=1 colspan=1>(0.0000,0.0%</td><td rowspan=1 colspan=1>(0.0661,0.20%)</td><td rowspan=1 colspan=1>(0.4232,1.30%</td><td rowspan=1 colspan=1>(0.4411,1.35%</td></tr><tr><td rowspan=1 colspan=1>3x LoD</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>31.00</td><td rowspan=1 colspan=1>(0.0397,0.13%</td><td rowspan=1 colspan=1>(0.1446,0.47%)</td><td rowspan=1 colspan=1>(0.0806,0.26%)</td><td rowspan=1 colspan=1>(0.0000,0.0%</td><td rowspan=1 colspan=1>(0.1488,0.48%)</td><td rowspan=1 colspan=1>(0.3420,1.10%)</td><td rowspan=1 colspan=1>(0.3855,1.24%)</td></tr><tr><td rowspan=2 colspan=1>EPEC</td><td rowspan=1 colspan=1>1x LoD</td><td rowspan=1 colspan=1>89</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>34.24</td><td rowspan=1 colspan=1>(0.4422,1.29%)</td><td rowspan=1 colspan=1>(0.2000,0.58%)</td><td rowspan=1 colspan=1>(0.0000,0.00%)</td><td rowspan=1 colspan=1>(0.1599,0.47%)</td><td rowspan=1 colspan=1>(0.1044,0.30%)</td><td rowspan=1 colspan=1>(0.6699,1.96%</td><td rowspan=1 colspan=1>(0.7927,2.31%)</td></tr><tr><td rowspan=1 colspan=1>3x LoD</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>32.49</td><td rowspan=1 colspan=1>(0.1053,0.32%</td><td rowspan=1 colspan=1>(0.1429,0.44%)</td><td rowspan=1 colspan=1>(0.0000,0.00%</td><td rowspan=1 colspan=1>(0.2081,0.64%</td><td rowspan=1 colspan=1>(0.0000,000%)</td><td rowspan=1 colspan=1>(0.5238,1.61%</td><td rowspan=1 colspan=1>(0.5710,1.76%)</td></tr><tr><td rowspan=2 colspan=1>Entamoeba</td><td rowspan=1 colspan=1>1x LoD</td><td rowspan=1 colspan=1>89</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>33.01</td><td rowspan=1 colspan=1>0.0000,0.00%)</td><td rowspan=1 colspan=1>(0.1349,0.41%)</td><td rowspan=1 colspan=1>(0.0000,0.00%</td><td rowspan=1 colspan=1>(0.1175,0.36%</td><td rowspan=1 colspan=1>(0.0000,0.00%</td><td rowspan=1 colspan=1>(0.3182,0.96%%</td><td rowspan=1 colspan=1>(0.3456,1.05%)</td></tr><tr><td rowspan=1 colspan=1>3x LoD</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>31.34</td><td rowspan=1 colspan=1>(0.1826,0.58%</td><td rowspan=1 colspan=1>(0.1265,0.40%)</td><td rowspan=1 colspan=1>0.0000,0.00%</td><td rowspan=1 colspan=1>(0.0000,0.00%</td><td rowspan=1 colspan=1>(0.0906,0.29%</td><td rowspan=1 colspan=1>(0.4000,1.28%)</td><td rowspan=1 colspan=1>(0.4449,1.42%)</td></tr><tr><td rowspan=2 colspan=1>Giardia</td><td rowspan=1 colspan=1>1x LoD</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>28.65</td><td rowspan=1 colspan=1>(0.2341,0.82%</td><td rowspan=1 colspan=1>(0.0000,0.00%</td><td rowspan=1 colspan=1>(0.0000,0.00%</td><td rowspan=1 colspan=1>(0.000,0.00%</td><td rowspan=1 colspan=1>(0.1737,0.61%</td><td rowspan=1 colspan=1>(0.5386,1.88%)</td><td rowspan=1 colspan=1>(0.5953,2.08%)</td></tr><tr><td rowspan=1 colspan=1>3x LoD</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>26.68</td><td rowspan=1 colspan=1>0.0840,0.31%</td><td rowspan=1 colspan=1>(0.0000,0.00%</td><td rowspan=1 colspan=1>(0.0000,0.00%</td><td rowspan=1 colspan=1>(0.1025,0.38%</td><td rowspan=1 colspan=1>(0.0857),0.32%</td><td rowspan=1 colspan=1>(0.3531,1.32%)</td><td rowspan=1 colspan=1>(0.3798,1.42%)</td></tr><tr><td rowspan=2 colspan=1>Norovirus GII</td><td rowspan=1 colspan=1>1x LoD</td><td rowspan=1 colspan=1>89</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>33.53</td><td rowspan=1 colspan=1>(0.1346,0.40%</td><td rowspan=1 colspan=1>(0.3090,0.92%)</td><td rowspan=1 colspan=1>(0.1111,0.33%</td><td rowspan=1 colspan=1>(0.0000,0.00%</td><td rowspan=1 colspan=1>(0.0431,0.13%</td><td rowspan=1 colspan=1>(0.4102,1.22%)</td><td rowspan=1 colspan=1>(0.5083,1.52%</td></tr><tr><td rowspan=1 colspan=1>3x LoD</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>32.42</td><td rowspan=1 colspan=1>(0.1742,0.54%)</td><td rowspan=1 colspan=1>(0.3405,1.05%</td><td rowspan=1 colspan=1>(0.2895,0.89%</td><td rowspan=1 colspan=1>(0.0632,0.19%)</td><td rowspan=1 colspan=1>(0.1972,0.61%</td><td rowspan=1 colspan=1>(0.4966,1.53%</td><td rowspan=1 colspan=1>(0.6721,2.07%)</td></tr><tr><td rowspan=2 colspan=1>Rotavirus***</td><td rowspan=1 colspan=1>1x LoD</td><td rowspan=1 colspan=1>89</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>34.86</td><td rowspan=1 colspan=1>0.1706,0.49%</td><td rowspan=1 colspan=1>(0.0000,0.00%</td><td rowspan=1 colspan=1>(0.3458,0.99%</td><td rowspan=1 colspan=1>(0.1207,0.35%</td><td rowspan=1 colspan=1>(0.3224,0.92%)</td><td rowspan=1 colspan=1>(0.9814,2.82%)</td><td rowspan=1 colspan=1>(1.0891,3.12%)</td></tr><tr><td rowspan=1 colspan=1>3x LoD</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>33.85</td><td rowspan=1 colspan=1>(0.8986,2.65%</td><td rowspan=1 colspan=1>(0.2050,0.61%</td><td rowspan=1 colspan=1>(0.5187,1.53%)</td><td rowspan=1 colspan=1>(0.0000,0.00%)</td><td rowspan=1 colspan=1>(0.2847,0.84%)</td><td rowspan=1 colspan=1>(1.3126,3.88%)</td><td rowspan=1 colspan=1>(1.6070,4.75%</td></tr><tr><td rowspan=2 colspan=1>STEC_0157</td><td rowspan=1 colspan=1>1x LoD</td><td rowspan=1 colspan=1>89</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>33.83</td><td rowspan=1 colspan=1>(0.2218,0.66%</td><td rowspan=1 colspan=1>(0.3083,0.91%</td><td rowspan=1 colspan=1>(0.0000,0.00%</td><td rowspan=1 colspan=1>(0.0204,0.06%</td><td rowspan=1 colspan=1>(0.0000,0.00%)</td><td rowspan=1 colspan=1>(0.5455,1.61%)</td><td rowspan=1 colspan=1>(0.6295,1.86%</td></tr><tr><td rowspan=1 colspan=1>3x LoD</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>32.22</td><td rowspan=1 colspan=1>(0.2431,0.75%</td><td rowspan=1 colspan=1>(0.2146,0.67%</td><td rowspan=1 colspan=1>0.0000,0.00%)</td><td rowspan=1 colspan=1>(0.0000,0.00%</td><td rowspan=1 colspan=1>(0.2051,0.64%)</td><td rowspan=1 colspan=1>(0.4625,1.44%</td><td rowspan=1 colspan=1>(0.5633,1.75%%</td></tr><tr><td rowspan=2 colspan=1>STEC_stx1</td><td rowspan=1 colspan=1>1x LoD</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>34.07</td><td rowspan=1 colspan=1>0.2297,0.67%)</td><td rowspan=1 colspan=1>(0.1214,0.36%)</td><td rowspan=1 colspan=1>(0.1281,0.38%</td><td rowspan=1 colspan=1>(0.00,0.00%)</td><td rowspan=1 colspan=1>(0.1170,0.34%)</td><td rowspan=1 colspan=1>(0.4206,1.23%)</td><td rowspan=1 colspan=1>(0.4991,1.46%</td></tr><tr><td rowspan=1 colspan=1>3x LoD</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>32.54</td><td rowspan=1 colspan=1>(0.2380,0.73%</td><td rowspan=1 colspan=1>(0.2222,0.68%)</td><td rowspan=1 colspan=1>(0.0802,0.25%)</td><td rowspan=1 colspan=1>(0.0000,0.00%)</td><td rowspan=1 colspan=1>(0.2000,0.61%)</td><td rowspan=1 colspan=1>(0.4919,1.51%)</td><td rowspan=1 colspan=1>(0.588 87,1.81%)</td></tr><tr><td rowspan=2 colspan=1>STEC_stx2</td><td rowspan=1 colspan=1>1x LoD</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>34.11</td><td rowspan=1 colspan=1>(0.0620,0.18%</td><td rowspan=1 colspan=1>(0.0979,0.29%</td><td rowspan=1 colspan=1>(0.0310,0.09%</td><td rowspan=1 colspan=1>(0.0000,0.00%)</td><td rowspan=1 colspan=1>(0.0000,0.00%)</td><td rowspan=1 colspan=1>(0.3987,1.17%)</td><td rowspan=1 colspan=1>(0.4109,1.20%)</td></tr><tr><td rowspan=1 colspan=1>3x LoD</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>32.69</td><td rowspan=1 colspan=1>0.1173,0.36%)</td><td rowspan=1 colspan=1>(0.2599,0.80%</td><td rowspan=1 colspan=1>(0.0000,0.00%)</td><td rowspan=1 colspan=1>(0.0000,0.00%%</td><td rowspan=1 colspan=1>(0.1316,0.40%</td><td rowspan=1 colspan=1>(0.4131,1.26%)</td><td rowspan=1 colspan=1>(0.4821,1.47%)</td></tr><tr><td rowspan=2 colspan=1>Salmonella</td><td rowspan=1 colspan=1>1x LoD</td><td rowspan=1 colspan=1>88</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>34.53</td><td rowspan=1 colspan=1>(0.2888,0.84%)</td><td rowspan=1 colspan=1>(0.2631,0.76%</td><td rowspan=1 colspan=1>(0.0000,0.00%)</td><td rowspan=1 colspan=1>(0.0000,0.0%</td><td rowspan=1 colspan=1>(0.0000,0.0%</td><td rowspan=1 colspan=1>(0.4880,1.41%)</td><td rowspan=1 colspan=1>(0.5857,1.70%)</td></tr><tr><td rowspan=1 colspan=1>3x LoD</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>33.44</td><td rowspan=1 colspan=1>(0.0279,0.08%)</td><td rowspan=1 colspan=1>(0.2100,0.63%</td><td rowspan=1 colspan=1>(0.0000,0.00%)</td><td rowspan=1 colspan=1>(0.1814,0.54%)</td><td rowspan=1 colspan=1>(0.1177,0.35%)</td><td rowspan=1 colspan=1>(0.4088,1.22%)</td><td rowspan=1 colspan=1>(0.4715,1.41%)</td></tr><tr><td rowspan=2 colspan=1>Yersinia</td><td rowspan=1 colspan=1>1x LoD</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>34.53</td><td rowspan=1 colspan=1>(0.1616,0.47%)</td><td rowspan=1 colspan=1>(0.1688,0.49%)</td><td rowspan=1 colspan=1>(0.0564,0.16%)</td><td rowspan=1 colspan=1>(0.0000,0.00%%</td><td rowspan=1 colspan=1>(0.1036,0.30%)</td><td rowspan=1 colspan=1>(0.5172,1.50%)</td><td rowspan=1 colspan=1>(0.5637,1.63%</td></tr><tr><td rowspan=1 colspan=1>3x LoD</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>33.02</td><td rowspan=1 colspan=1>(0.0503,0.15%)</td><td rowspan=1 colspan=1>(0.2129,0.64%)</td><td rowspan=1 colspan=1>(0.0370,0.11%)</td><td rowspan=1 colspan=1>(0.0000,0.00%</td><td rowspan=1 colspan=1>(0.0801,0.24%</td><td rowspan=1 colspan=1>(0.3434,1.04%)</td><td rowspan=1 colspan=1>(0.3951,1.20%)</td></tr></table>

\* SD is first figure presented, subsequent figure is CV (Coefficient of variation presented as a percentage $( \% )$ $^ { * * }$ TOTAL means total variance observed across the study $^ { \ast \ast \ast }$ Results in table correspond to the original sample set analyzed for Rotavirus. Results from the new dataset were

<table><tr><td rowspan=1 colspan=5>EY</td><td rowspan=1 colspan=7>Variance component (SD, %CV*)</td></tr><tr><td rowspan=1 colspan=1>Pathogen</td><td rowspan=1 colspan=1>tested</td><td rowspan=1 colspan=1>(n)</td><td rowspan=1 colspan=1>(n)</td><td rowspan=1 colspan=1>Mean</td><td rowspan=1 colspan=1>BetweenSite</td><td rowspan=1 colspan=1>BetweenLot</td><td rowspan=1 colspan=1>BetweenDay</td><td rowspan=1 colspan=1>BetweenOperator</td><td rowspan=1 colspan=1>BetweenInstrument</td><td rowspan=1 colspan=1>BetweenReplicate</td><td rowspan=1 colspan=1>Total**</td></tr><tr><td rowspan=2 colspan=1>Rotavirus</td><td rowspan=1 colspan=1>1x LoD</td><td rowspan=1 colspan=1>89</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>34.86</td><td rowspan=1 colspan=1>(0.1706,0.49%</td><td rowspan=1 colspan=1>(0.0000,0.00%</td><td rowspan=1 colspan=1>(0.3458,0.99%)</td><td rowspan=1 colspan=1>(0.1207,0.35%)</td><td rowspan=1 colspan=1>(0.3224,0.92%)</td><td rowspan=1 colspan=1>(0.9814,2.82%)</td><td rowspan=1 colspan=1>(1.0891,3.12%)</td></tr><tr><td rowspan=1 colspan=1>3x LoD</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>32.42</td><td rowspan=1 colspan=1>(0.1742,0.54%</td><td rowspan=1 colspan=1>(0.3405,1.05%</td><td rowspan=1 colspan=1>(0.2895,0.89%</td><td rowspan=1 colspan=1>(0.0632,0.19%</td><td rowspan=1 colspan=1>(0.1972,0.61%)</td><td rowspan=1 colspan=1>(0.4966,1.53%</td><td rowspan=1 colspan=1>(0.6721,2.07%</td></tr></table>

# Repeatability Study:

To further assess the precision of the QIAstat-Dx Gastrointestinal Panel 2, an in-house repeatability study was conducted using a set of samples containing analyte concentrations of 3X LoD and 1X LoD as well as negative samples prepared in stool matrix in Para-Pak C&S transport medium. Specifically, pathogens included in the positive samples were Adenovirus, Campylobacter, Entamoeba histolytica, Giardia lamblia, Norovirus GII, Rotavirus, Salmonella enterica, and Yersinia enterocolitica, in addition to Enteropathogenic $E .$ . coli (EPEC), STEC stx1/stx2, and E. coli O157 which are only applicable for Para-Pak C&S samples. Each panel member was tested with the same instrument over 12 days. In total, 60 replicates at 1x LoD and 60 replicates at 3x LoD per each of the tested targets and 60 negative samples were run in the study. Overall results showed a $9 3 . 3 3 \substack { - 1 0 0 \% }$ and $9 5 . 0 0 \mathrm { - } 1 0 0 \%$ detection rate for 1x LoD and $3 \mathbf { x }$ LoD samples, respectively. Negative samples showed $100 \%$ of negative calls for all panel analytes. One failed run due to a cartridge error occurred during the study $( 1 / 4 2 6 , 0 . 2 3 \% )$ . Testing results met the study acceptance criteria demonstrating acceptable repeatability of the QIAstat-Dx Gastrointestinal Panel 2.

2. Linearity:

Not applicable.

3. Analytical Specificity/Interference:

# Analytical Specificity/Cross-Reactivity

The Analytical Specificity study was performed to demonstrate the absence of crossreactivity between the QIAstat-Dx Gastrointestinal Panel 2 assay on-panel microorganisms and viruses (intra-panel cross-reactivity) and between on-panel and off-panel microorganisms/viruses (i.e., organisms not covered by the panel and therefore not intended to be detected by the QIAstat-Dx Gastrointestinal Panel 2).

Samples were prepared by single spiking microorganisms/viruses at high concentrations into negative stool resuspended in Para-Pak C&S transport medium. Sample concentrations tested were $1 . 0 \mathrm { E } { + 0 6 }$ CFU/mL for bacteria, $1 . 0 \mathrm { E } { + 0 5 }$ TCID50/mL for viruses and $1 . 0 \mathrm { E } { + 0 5 }$ cells/mL for parasites, or the highest available concentration based on the pathogen stock. Testing was conducted in triplicate.

Study results indicated no false positive results for on-panel or off-panel analytes with the exception of two non-targeted Campylobacter species (C. helveticus and C. lari) that crossreacted with the Campylobacter assay oligonucleotides included in the QIAstat-Dx Gastrointestinal Panel 2. The on-panel and off-panel microorganisms/viruses evaluated in the study are presented in Tables 3 and 4 respectively.

Table 3: List of Analytical Specificity On-Panel Pathogens Tested   

<table><tr><td rowspan=1 colspan=1>Type</td><td rowspan=1 colspan=1>Pathogen</td><td rowspan=1 colspan=1>Concentration Tested</td></tr><tr><td rowspan=10 colspan=1>Bacteria</td><td rowspan=1 colspan=1>Campylobacter coli</td><td rowspan=1 colspan=1>1.0E+06 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Campylobacter jejuni</td><td rowspan=1 colspan=1>1.0E+06 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Campylobacter upsaliensis</td><td rowspan=1 colspan=1>1.0E+06 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Escherichia coli (EPEC)</td><td rowspan=1 colspan=1>1.0E+06 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Escherichia coli (ETEC lt/st)</td><td rowspan=1 colspan=1>1.0E+06 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Escherichia coli (STEC)</td><td rowspan=1 colspan=1>1.0E+06 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Plesiomonas shigelloides</td><td rowspan=1 colspan=1>1.0E+06 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Salmonella enterica</td><td rowspan=1 colspan=1>1.0E+06 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Shigella sonnei</td><td rowspan=1 colspan=1>1.0E+06 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Yersinia enterocolitica</td><td rowspan=1 colspan=1>1.0E+06 CFU/mL</td></tr><tr><td rowspan=4 colspan=1>Parasites</td><td rowspan=1 colspan=1>Cryptosporidium parvum</td><td rowspan=1 colspan=1>1.0E+05 oocysts/mL</td></tr><tr><td rowspan=1 colspan=1>Cyclospora cayetanensis</td><td rowspan=1 colspan=1>1.0E+05 genome copies/mL</td></tr><tr><td rowspan=1 colspan=1>Entamoeba histolytica</td><td rowspan=1 colspan=1>1.0E+05 cells/mL</td></tr><tr><td rowspan=1 colspan=1>Giardia lamblia</td><td rowspan=1 colspan=1>1.0E+05 cells/mL</td></tr><tr><td rowspan=5 colspan=1>Viruses</td><td rowspan=1 colspan=1>Adenovirus F41</td><td rowspan=1 colspan=1>1.0E+05 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Astrovirus</td><td rowspan=1 colspan=1>1.0E+05 copies/mL</td></tr><tr><td rowspan=1 colspan=1>Norovirus GI</td><td rowspan=1 colspan=1>1.0E+05 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Norovirus GII</td><td rowspan=1 colspan=1>1.0E+05 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Rotavirus</td><td rowspan=1 colspan=1>1.0E+05 U/mL</td></tr></table>

Table 4: List of Analytical Specificity Off-Panel Pathogens Tested   

<table><tr><td colspan="1" rowspan="1">Type</td><td colspan="1" rowspan="1">Pathogen (Potential Cross-Reactant)</td><td colspan="1" rowspan="1">Concentration Tested</td></tr><tr><td colspan="1" rowspan="15">Bacteria</td><td colspan="1" rowspan="1">Abiotrophia defectiva</td><td colspan="1" rowspan="1">1.0E+06 cells/mL</td></tr><tr><td colspan="1" rowspan="1">Acinetobacter baumannii</td><td colspan="1" rowspan="1">1.0E+06 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Aeromonas hydrophila</td><td colspan="1" rowspan="1">1.0E+06 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Arcobacter cryaerophilus</td><td colspan="1" rowspan="1">1.0E+06 cells/mL</td></tr><tr><td colspan="1" rowspan="1">Bacillus subtilis</td><td colspan="1" rowspan="1">1.0E+06 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Bifidobacterium bifidum</td><td colspan="1" rowspan="1">1.0E+06 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Campylobacter fetus</td><td colspan="1" rowspan="1">1.0E+06 cells/mL</td></tr><tr><td colspan="1" rowspan="1">Campylobacter gracilis</td><td colspan="1" rowspan="1">1.0E+06 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Campylobacter helveticus*</td><td colspan="1" rowspan="1">1.0E+06 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Campylobacter hominis</td><td colspan="1" rowspan="1">1.0E+06 cells/mL</td></tr><tr><td colspan="1" rowspan="1">Campylobacter lari*</td><td colspan="1" rowspan="1">1.0E+06 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Campylobacter mucosalis</td><td colspan="1" rowspan="1">1.0E+06 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Campylobacter rectus</td><td colspan="1" rowspan="1">1.0E+06 cells/mL</td></tr><tr><td colspan="1" rowspan="1">Chlamydia trachomatis</td><td colspan="1" rowspan="1">1.0E+06 cells/mL</td></tr><tr><td colspan="1" rowspan="1">Citrobacter freundii</td><td colspan="1" rowspan="1">1.0E+06 CFU/mL</td></tr><tr><td colspan="1" rowspan="28"></td><td colspan="1" rowspan="1">Clostridium difficile non-toxigenic</td><td colspan="1" rowspan="1">1.0E+06 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Clostridium perfringens</td><td colspan="1" rowspan="1">1.0E+06 cells/mL</td></tr><tr><td colspan="1" rowspan="1">Clostridium septicum</td><td colspan="1" rowspan="1">1.0E+06 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Clostridium tetani</td><td colspan="1" rowspan="1">1.0E+06 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium genitalium</td><td colspan="1" rowspan="1">1.0E+06 cells/mL</td></tr><tr><td colspan="1" rowspan="1">Enterobacter aerogenes</td><td colspan="1" rowspan="1">1.0E+06 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Enterobacter cloacae</td><td colspan="1" rowspan="1">1.0E+06 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Enterococcus faecalis</td><td colspan="1" rowspan="1">1.0E+06 cells/mL</td></tr><tr><td colspan="1" rowspan="1">Enterococcus faecium</td><td colspan="1" rowspan="1">1.0E+06 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Escherichia fergusonii</td><td colspan="1" rowspan="1">1.0E+06 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Escherichia hermannii</td><td colspan="1" rowspan="1">1.0E+06 cells/mL</td></tr><tr><td colspan="1" rowspan="1">Escherichia vulneris</td><td colspan="1" rowspan="1">1.0E+06 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Faecalibacterium prausnitzii</td><td colspan="1" rowspan="1">1.0E+06 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Gardnerella vaginalis</td><td colspan="1" rowspan="1">1.0E+06 cells/mL</td></tr><tr><td colspan="1" rowspan="1">Haemophilus influenzae</td><td colspan="1" rowspan="1">1.0E+06 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Hathewaya histolytica(Clostridium)</td><td colspan="1" rowspan="1">1.0E+06 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Helicobacter pylori</td><td colspan="1" rowspan="1">1.0E+06 cells/mL</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">1.0E+06 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus casei</td><td colspan="1" rowspan="1">1.0E+06 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Listeria monocytogenes</td><td colspan="1" rowspan="1">1.0E+06 cells/mL</td></tr><tr><td colspan="1" rowspan="1">Proteus mirabilis</td><td colspan="1" rowspan="1">1.0E+06 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Proteus vulgaris</td><td colspan="1" rowspan="1">1.0E+06 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">1.0E+06 cells/mL</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">1.0E+06 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus subsp.aureus</td><td colspan="1" rowspan="1">1.0E+06 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus epidermidis</td><td colspan="1" rowspan="1">1.0E+06 cells/mL</td></tr><tr><td colspan="1" rowspan="1">Streptococcus agalactiae</td><td colspan="1" rowspan="1">1.0E+06 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Streptococcus pyogenes</td><td colspan="1" rowspan="1">1.0E+06 CFU/mL</td></tr><tr><td colspan="1" rowspan="4">Fungi</td><td colspan="1" rowspan="1">Aspergillus fumigatus</td><td colspan="1" rowspan="1">1.0E+05 cells/mL</td></tr><tr><td colspan="1" rowspan="1">Candida albicans</td><td colspan="1" rowspan="1">1.0E+05 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Saccharomyces boulardii</td><td colspan="1" rowspan="1">1.0E+05 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Saccharomyces cerevisiae</td><td colspan="1" rowspan="1">1.0E+05 copies/mL</td></tr><tr><td colspan="1" rowspan="5">Parasite</td><td colspan="1" rowspan="1">Babesia microti</td><td colspan="1" rowspan="1">N/A**</td></tr><tr><td colspan="1" rowspan="1">Blastocystis hominis</td><td colspan="1" rowspan="1">1.0E+05 cells/mL</td></tr><tr><td colspan="1" rowspan="1">Giardia muris</td><td colspan="1" rowspan="1">1.0E+05 cysts/mL</td></tr><tr><td colspan="1" rowspan="1">Toxoplasma gondii</td><td colspan="1" rowspan="1">1.0E+05 cells/mL</td></tr><tr><td colspan="1" rowspan="1">Trichomonas tenax</td><td colspan="1" rowspan="1">1.0E+05 cells/mL</td></tr><tr><td colspan="1" rowspan="7">Viruses</td><td colspan="1" rowspan="1">Adenovirus C:2</td><td colspan="1" rowspan="1">1.0E+05 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Adenovirus B:34</td><td colspan="1" rowspan="1">1.0E+05 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Adenovirus B3</td><td colspan="1" rowspan="1">1.0E+05 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Adenovirus E:4a</td><td colspan="1" rowspan="1">1.0E+05 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Adenovirus serotype 1</td><td colspan="1" rowspan="1">1.0E+05 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Adenovirus serotype 5</td><td colspan="1" rowspan="1">1.0E+05 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Adenovirus serotype 8</td><td colspan="1" rowspan="1">1.0E+05 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Bocavirus Type 1</td><td colspan="1" rowspan="1">1.0E+05 copies/mL</td></tr><tr><td colspan="1" rowspan="1">Coronavirus 229E</td><td colspan="1" rowspan="1">1.0E+05 U/mL</td></tr><tr><td colspan="1" rowspan="1">Coxsackievirus B3</td><td colspan="1" rowspan="1">1.0E+05 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Cytomegalovirus</td><td colspan="1" rowspan="1">1.0E+05 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Enterovirus 6 (Echovirus)</td><td colspan="1" rowspan="1">1.0E+05 U/mL</td></tr><tr><td colspan="1" rowspan="1">Enterovirus 68</td><td colspan="1" rowspan="1">1.0E+05 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Herpes Simplex Virus Type 2</td><td colspan="1" rowspan="1">1.0E+05 PFU/mL</td></tr><tr><td colspan="1" rowspan="1">Rhinovirus 1A</td><td colspan="1" rowspan="1">1.0E+05 TCID50/mL</td></tr></table>

\*Positive results generated for Campylobacter $^ { \ast \ast }$ The stock concentration for Babesia microti is expressed as $32 \%$ parasitemia (Number of infected cells/Total cells $^ { * } 1 0 0$ )

In silico analysis of potential cross-reactivity predicted that the following microorganisms may cross-react with QIAstat-Dx Gastrointestinal Panel 2 targets (Table 5).

Table 5: Potential Cross-Reactivity Based On in silico Analysis   

<table><tr><td rowspan=1 colspan=1>QIAstat-Dx GastrointestinalPanel 2 Target</td><td rowspan=1 colspan=1>Potential cross-reactive microorganisms</td></tr><tr><td rowspan=1 colspan=1>Enteropathogenic E. coli (EPEC)</td><td rowspan=1 colspan=1>Shigella boydiil,2,3, Escherichia albertii,2</td></tr><tr><td rowspan=1 colspan=1>Campylobacter.</td><td rowspan=1 colspan=1>Campylobacter lari4, Campylobacter helveticus4</td></tr><tr><td rowspan=1 colspan=1>Shiga-like toxin-producing E. coli(STEC) stx1/stx2</td><td rowspan=1 colspan=1>Shigella sonneil,3, ShigellaDysenteriael3, Acinetobacter haemolyticus1,5,Citrobacter rend,Enteobacter clAeromonas caviael&#x27;5, Escherichia albertii 1,5</td></tr><tr><td rowspan=1 colspan=1>E. coli O157</td><td rowspan=1 colspan=1>Non-STEC E.coli O157 strains6</td></tr></table>

1Predicted cross-reactivity identified by in silico analysis reflects sequences that can be acquired between species by horizontal gene transfer.

2Rare or less common eae intimin carrier organisms

$^ { 3 } \mathrm { O n }$ -panel target.

4In vitro testing of Campylobacter lari and Campylobacter helveticus strains at high concentration confirmed potential cross-reactivity of these Campylobacter species with the QIAstat-Dx Gastrointestinal Panel 2 assay.

5Rare or less common $\mathbf { s } t x$ toxins producers

$^ 6 E .$ . coli O157 will only be reported by the QIAstat-Dx Gastrointestinal Panel 2 assay when there is a positive amplification for the $\dot { E }$ . coli (STEC) design according to the calling algorithm. An infrequent case of an $E$ . coli (STEC) and an $E$ . coli O157 co-infection will not be differentiated from a single infection caused by a STEC O157:H7 strain.

# Interfering Substances Study:

The effect of potentially interfering substances on the performance of the QIAstat-Dx Gastrointestinal Panel 2 was evaluated. Thirty-five (35) potentially interfering substances were spiked into the sample mixes at a level predicted to be above the concentration of the substance likely to be found in stool specimens. Substances included endogenous, exogenous as well as technique-specific substances.

Testing included samples containing negative clinical stool matrix in Para-Pak C&S medium with and without the addition of each potentially interfering substance. Samples containing organism mixes with one strain for each targeted pathogen were tested at a concentration of 3x LoD. Testing was performed in triplicates. Additionally, for endogenous substances, negative specimens (stool matrix in Para-Pak C&S medium matrix with no organism mix) were spiked with only the test substance to evaluate the potential for false positive results due to the test substance itself.

For the majority of substances tested, no interference was observed, with the exceptions of mucin, calcium carbonate, nonoxynol-9, and Rotavirus reassortants which demonstrated interference at high concentrations.

The mucin at $5 \%$ w/v sample generated unexpected false positives results for the Yersinia target. These results were investigated by testing the interfering substance with an FDAcleared method that also detected Yersinia in the sample, demonstrating that the unexpected positive results were not due to mucin interference.

Calcium carbonate at concentrations above $0 . 5 \%$ w/v was found to generate false negative results for all the QIAstat-Dx Gastrointestinal Panel 2 targets and the internal control.

Nonoxynol-9 at concentrations above $0 . 0 2 \% \mathrm { \Omega } \mathrm { v }$ was found to generate false negative results for detection of Entamoeba histolytica.

As expected, Rotavirus reassortants WC3:2-5, R574(9) and WI79-4,9 used in Rotavirus A vaccines generated positive results for Rotavirus A in the QIAstat-Dx Gastrointestinal Panel 2. Final concentrations without interference (i.e., no false positive results for Rotavirus) for WC3:2-5, R574(9) and WI79-4,9 were $8 . 8 9 \mathrm { x } 1 0 ^ { - 5 }$ $\Gamma \mathrm { C I D } _ { 5 0 } / \mathrm { m L }$ and 1.10 PFU/mL, respectively.

The strains used for sample mixes tested in the study are presented in Table 6. Results from testing the 35 endogenous and exogenous substances evaluated are provided in Table 7.

Table 6: Interfering Substances Study, Microorganisms/Viruses Evaluated   

<table><tr><td colspan="1" rowspan="1">Mix</td><td colspan="1" rowspan="1">Target</td><td colspan="1" rowspan="1">Strain</td><td colspan="1" rowspan="1">Supplier</td><td colspan="1" rowspan="1">Catalog ID</td><td colspan="1" rowspan="1">FinalConcentrationtested (3x LoD)</td></tr><tr><td colspan="1" rowspan="4">1</td><td colspan="1" rowspan="1">Giardia lamblia</td><td colspan="1" rowspan="1">Portland-1</td><td colspan="1" rowspan="1">ATCC</td><td colspan="1" rowspan="1">30888</td><td colspan="1" rowspan="1">1.91E+03cells/mL</td></tr><tr><td colspan="1" rowspan="1">Enteropathogenic E.c (EPEC)</td><td colspan="1" rowspan="1">B171-8 (0111:NM)</td><td colspan="1" rowspan="1">ZeptoMetrix</td><td colspan="1" rowspan="1">0801747</td><td colspan="1" rowspan="1">7.75E+03CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Yersinia enterocolitica</td><td colspan="1" rowspan="1">Z036</td><td colspan="1" rowspan="1">ZeptoMetrix</td><td colspan="1" rowspan="1">0801734</td><td colspan="1" rowspan="1">6.21E+03CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Salmonella</td><td colspan="1" rowspan="1">Salmonella entericaSerovar Typhimurium;Z005</td><td colspan="1" rowspan="1">ZeptoMetrix</td><td colspan="1" rowspan="1">0801437</td><td colspan="1" rowspan="1">1.36E+04CFU/mL</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Adenovirus F40/F41</td><td colspan="1" rowspan="1">Adenovirus Type 41 (Tak)</td><td colspan="1" rowspan="1">ZeptoMetrix</td><td colspan="1" rowspan="1">0810085CF</td><td colspan="1" rowspan="1">1.50E-01TCID50/mL</td></tr><tr><td colspan="1" rowspan="4">2</td><td colspan="1" rowspan="1">Norovirus GI/GII</td><td colspan="1" rowspan="1">Norovirus GII-recombinant</td><td colspan="1" rowspan="1">ZeptoMetrix</td><td colspan="1" rowspan="1">0810087CF</td><td colspan="1" rowspan="1">3.15E+01TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Entamoeba histolytica</td><td colspan="1" rowspan="1">HM-1:IMSS</td><td colspan="1" rowspan="1">ATCC</td><td colspan="1" rowspan="1">30459</td><td colspan="1" rowspan="1">6.60E-01cells/mL</td></tr><tr><td colspan="1" rowspan="1">Campylobacter .</td><td colspan="1" rowspan="1">Campylobacter jejuni Z086 ZeptoMetrix</td><td colspan="1" rowspan="1">ZeptoMetrix</td><td colspan="1" rowspan="1">0801650</td><td colspan="1" rowspan="1">4.98E+03CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Shiga-like toxin-producing E. coli(STEC) 0157</td><td colspan="1" rowspan="1">0157:H7; EDL933</td><td colspan="1" rowspan="1">ZeptoMetrix</td><td colspan="1" rowspan="1">0801622</td><td colspan="1" rowspan="1">6.84E+03CFU/mL</td></tr><tr><td colspan="1" rowspan="3">3</td><td colspan="1" rowspan="1">Cryptosporidium</td><td colspan="1" rowspan="1">Cryptosporidium parvum,Iowa isolate</td><td colspan="1" rowspan="1">Waterborne</td><td colspan="1" rowspan="1">P102C</td><td colspan="1" rowspan="1">1.88E+03oocysts/mL</td></tr><tr><td colspan="1" rowspan="1">Plesiomonasshiglloides</td><td colspan="1" rowspan="1">Z130</td><td colspan="1" rowspan="1">ZeptoMetrix</td><td colspan="1" rowspan="1">0801899</td><td colspan="1" rowspan="1">6.87E+03CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Rotavirus A</td><td colspan="1" rowspan="1">69M</td><td colspan="1" rowspan="1">ZeptoMetrix</td><td colspan="1" rowspan="1">0810280CF</td><td colspan="1" rowspan="1">1.31E+03U/mL</td></tr><tr><td colspan="1" rowspan="4">4</td><td colspan="1" rowspan="1">Astrovirus</td><td colspan="1" rowspan="1">(Type 8) ERE IID 2371</td><td colspan="1" rowspan="1">ZeptoMetrix</td><td colspan="1" rowspan="1">0810277CF</td><td colspan="1" rowspan="1">3.51E+01U/mL</td></tr><tr><td colspan="1" rowspan="1">Cyclosporacayetanensis</td><td colspan="1" rowspan="1">n/a</td><td colspan="1" rowspan="1">LACNY</td><td colspan="1" rowspan="1">LAC2825</td><td colspan="1" rowspan="1">1.37E+04copies/mL</td></tr><tr><td colspan="1" rowspan="1">Enterotoxigenic E.coli ETEC) l/st</td><td colspan="1" rowspan="1">ETEC; ST+, LT+</td><td colspan="1" rowspan="1">ZeptoMetrix</td><td colspan="1" rowspan="1">0801624</td><td colspan="1" rowspan="1">1.70E+03CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Enteroinvasive E. coli(EIEC)/Shigella</td><td colspan="1" rowspan="1">CDC EDL 1282;Escherichia coli serotype29:NM</td><td colspan="1" rowspan="1">ATCC</td><td colspan="1" rowspan="1">43892</td><td colspan="1" rowspan="1">1.24E+02CFU/mL</td></tr><tr><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">Negative (IC)</td><td colspan="1" rowspan="1">n/a</td><td colspan="1" rowspan="1">n/a</td><td colspan="1" rowspan="1">n/a</td><td colspan="1" rowspan="1">n/a</td></tr></table>

Table 7: Interfering Substances Study Results   

<table><tr><td colspan="1" rowspan="1">Substance tested</td><td colspan="1" rowspan="1">Concentration tested</td><td colspan="1" rowspan="1">Result</td></tr><tr><td colspan="3" rowspan="1">Endogenous substances</td></tr><tr><td colspan="1" rowspan="1">Bovine and ovine bile</td><td colspan="1" rowspan="1">12% w/v</td><td colspan="1" rowspan="1">No Interference</td></tr><tr><td colspan="1" rowspan="1">Cholesterol</td><td colspan="1" rowspan="1">1.5% w/v</td><td colspan="1" rowspan="1">No Interference</td></tr><tr><td colspan="1" rowspan="1">Fatty acids (palmitic acid)</td><td colspan="1" rowspan="1">0.2% w/v</td><td colspan="1" rowspan="1">No Interference</td></tr><tr><td colspan="1" rowspan="1">Fatty acids (stearic acid)</td><td colspan="1" rowspan="1">0.4% w/v</td><td colspan="1" rowspan="1">No Interference</td></tr><tr><td colspan="1" rowspan="1">Human genomic DNA</td><td colspan="1" rowspan="1">20 μg/mL5 ug/mL</td><td colspan="1" rowspan="1">No InterferenceNo Interference</td></tr><tr><td colspan="1" rowspan="1">Human stool (overfill of transportmedium vial)</td><td colspan="1" rowspan="1">300 mg/mL</td><td colspan="1" rowspan="1">No Interference</td></tr><tr><td colspan="1" rowspan="1">Human urine</td><td colspan="1" rowspan="1">50% v/v</td><td colspan="1" rowspan="1">No Interference</td></tr><tr><td colspan="1" rowspan="1">Human whole blood with NaCitrate</td><td colspan="1" rowspan="1">40% v/v</td><td colspan="1" rowspan="1">No Interference</td></tr><tr><td colspan="1" rowspan="1">Mucin from bovine submaxillary</td><td colspan="1" rowspan="1">5% w/v2.5% w/v</td><td colspan="1" rowspan="1">Interference1No Interference</td></tr><tr><td colspan="1" rowspan="1">Triglycerides</td><td colspan="1" rowspan="1">5% w/v</td><td colspan="1" rowspan="1">No Interference</td></tr><tr><td colspan="3" rowspan="1">Exogenous substances</td></tr><tr><td colspan="1" rowspan="1">Bacitracin</td><td colspan="1" rowspan="1">250U/mL</td><td colspan="1" rowspan="1">No Interference</td></tr><tr><td colspan="1" rowspan="1">Bisacodyl</td><td colspan="1" rowspan="1">0.3% w/v</td><td colspan="1" rowspan="1">No Interference</td></tr><tr><td colspan="1" rowspan="1">Bismuth subsalicylate</td><td colspan="1" rowspan="1">0.35% w/v</td><td colspan="1" rowspan="1">No Interference</td></tr><tr><td colspan="1" rowspan="1">Calcium carbonate (TUMS ExtraStrength 750)</td><td colspan="1" rowspan="1">5%w/v0.5% w/v</td><td colspan="1" rowspan="1">InterferenceNo Interference</td></tr><tr><td colspan="1" rowspan="1">Docusate sodium</td><td colspan="1" rowspan="1">2.5% w/v</td><td colspan="1" rowspan="1">No Interference</td></tr><tr><td colspan="1" rowspan="1">Doxycycline hydrochloride</td><td colspan="1" rowspan="1">0.05% w/v</td><td colspan="1" rowspan="1">No Interference</td></tr><tr><td colspan="1" rowspan="1">Glycerin</td><td colspan="1" rowspan="1">50% v/v</td><td colspan="1" rowspan="1">No Interference</td></tr><tr><td colspan="1" rowspan="1">Hydrocortisone</td><td colspan="1" rowspan="1">0.5% w/v</td><td colspan="1" rowspan="1">No Interference</td></tr><tr><td colspan="1" rowspan="1">Loperamide hydrochloride</td><td colspan="1" rowspan="1">0.078% w/v</td><td colspan="1" rowspan="1">No Interference</td></tr><tr><td colspan="1" rowspan="1">Magnesium hydroxide</td><td colspan="1" rowspan="1">0.1% w/v</td><td colspan="1" rowspan="1">No Interference</td></tr><tr><td colspan="1" rowspan="1">Metronidazole</td><td colspan="1" rowspan="1">1.5% w/v</td><td colspan="1" rowspan="1">No Interference</td></tr><tr><td colspan="1" rowspan="1">Mineral oil</td><td colspan="1" rowspan="1">50% v/v</td><td colspan="1" rowspan="1">No Interference</td></tr><tr><td colspan="1" rowspan="1">Naproxen sodium</td><td colspan="1" rowspan="1">0.7% w/v</td><td colspan="1" rowspan="1">No Interference</td></tr><tr><td colspan="1" rowspan="1">Nonoxynol-9</td><td colspan="1" rowspan="1">1.2% v/v0.6% v/v0.3% v/v0.15%v/v0.075% v/v0.02% v/v</td><td colspan="1" rowspan="1">InterferenceInterferenceInterferenceInterferenceInterferenceNo Interference</td></tr><tr><td colspan="1" rowspan="1">Nystatin</td><td colspan="1" rowspan="1">10000 USP units/mL</td><td colspan="1" rowspan="1">No Interference</td></tr><tr><td colspan="1" rowspan="1">Phenylephrine hydrochloride</td><td colspan="1" rowspan="1">0.075% w/v</td><td colspan="1" rowspan="1">No Interference</td></tr><tr><td colspan="1" rowspan="1">Sodium phosphate</td><td colspan="1" rowspan="1">5% w/v</td><td colspan="1" rowspan="1">No Interference</td></tr><tr><td colspan="3" rowspan="1">Vaccine components</td></tr><tr><td colspan="1" rowspan="1">Rotavirus reassortant WC3:2-5,R574(9) - VR 2195</td><td colspan="1" rowspan="1">8.89 x 10-3 TCID50/mL8.89 x 10-4 TCID50/mL8.89 x 10-5 TCID50/mL</td><td colspan="1" rowspan="1">InterferenceInterferenceNo Interference</td></tr><tr><td colspan="1" rowspan="1">Rotavirus reassortant WI79-4,9 -VR 2415</td><td colspan="1" rowspan="1">1.10 x 10² pfu/mL1.10 x 101 pfu/mL1.10 pfu/mL</td><td colspan="1" rowspan="1">InterferenceInterferenceNo Interference</td></tr><tr><td colspan="3" rowspan="1">Technique-specific Substances, Transport Media</td></tr><tr><td colspan="1" rowspan="1">Bleach</td><td colspan="1" rowspan="1">0.5% v/v</td><td colspan="1" rowspan="1">No Interference</td></tr><tr><td colspan="1" rowspan="1">Ethanol</td><td colspan="1" rowspan="1">0.2% v/v</td><td colspan="1" rowspan="1">No Interference</td></tr><tr><td colspan="1" rowspan="1">Puritan Fecal Opti-SwabCollection &amp;Transport Systemwith Cary-Blair Medium2</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">No Interference</td></tr><tr><td colspan="1" rowspan="1">Puritan PurSafe DNA/RNAPreservative2</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">No Interference</td></tr><tr><td colspan="1" rowspan="1">Sigma Fecal Transwab2</td><td colspan="1" rowspan="1">1 swab/2mL Cary Blair</td><td colspan="1" rowspan="1">No Interference</td></tr></table>

1 This substance was tested by another FDA-cleared test that also detected Yersinia positive signals. 2 Note: Performance has not established for these transport media

# Microbial Interference:

A microbial interference study was conducted to assess whether non-target microorganisms/viruses can interfere with the detection of panel targets. Clinically relevant and challenging concentrations of non-target organisms ( $1 . 0 \mathrm { { E + } 0 6 \times 1 0 ^ { 6 } \mathrm { { C F U / m L } } }$ for bacteria, 1.0E $+ 0 5$ cells/mL for yeast and $1 . 0 \mathrm { E } { + 0 5 }$ $\mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ for viruses) were individually co-spiked with each of the targeted pathogens (Table 6 above) at $3 \mathbf { x }$ LoD. Samples were prepared in negative stool matrix in Para-Pak C&S transport medium. Testing was performed in triplicate. Microbial interference was not observed as all combinations and replicates tested successfully detected all the QIAstat-Dx Gastrointestinal Panel 2 targets. Results from the study are presented in Table 8.

Table 8: Microbial Interference Testing Results   

<table><tr><td rowspan=1 colspan=1>Non-Target Microrganism/Virus Tested</td><td rowspan=1 colspan=1>Concentration tested</td><td rowspan=1 colspan=1>Result</td></tr><tr><td rowspan=1 colspan=1>Aeromonas hydrophila</td><td rowspan=1 colspan=1>1.0E+06 units/mL</td><td rowspan=1 colspan=1>No Interference</td></tr><tr><td rowspan=1 colspan=1>Bacteroides vulgatus</td><td rowspan=1 colspan=1>1.0E+06 units/mL</td><td rowspan=1 colspan=1>No Interference</td></tr><tr><td rowspan=1 colspan=1>Bifidobacterium bifidum</td><td rowspan=1 colspan=1>1.0E+06 units/mL</td><td rowspan=1 colspan=1>No Interference</td></tr><tr><td rowspan=1 colspan=1>Enterovirus Species D, Serotype EV-D68</td><td rowspan=1 colspan=1>1.0E+05 units/mL</td><td rowspan=1 colspan=1>No Interference</td></tr><tr><td rowspan=1 colspan=1>Non-pathogenic E. coli</td><td rowspan=1 colspan=1>1.0E+06 units/mL</td><td rowspan=1 colspan=1>No Interference</td></tr><tr><td rowspan=1 colspan=1>Helicobacter pylori</td><td rowspan=1 colspan=1>1.0E+06 units/mL</td><td rowspan=1 colspan=1>No Interference</td></tr><tr><td rowspan=1 colspan=1>Saccharomyces cerevisiae (deposited as S.boulardii)</td><td rowspan=1 colspan=1>1.0E+05 units/mL</td><td rowspan=1 colspan=1>No Interference</td></tr></table>

# Competitive Interference

The potential for competitive interference was evaluated for a subset of analytes detected by the QIAstat-Dx Gastrointestinal Panel 2. The samples were prepared as a combination of two targeted pathogens in stool matrix (negative pooled stools resuspended in Para-Pak C&S transport medium) with one pathogen spiked at a high concentration (50x LoD) and the other spiked at a low concentration (3x LoD). Testing was conducted in triplicate. No false positives or false negative results were observed during the study. Results are presented in Table 9.

Table 9: QIAstat-Dx Gastrointestinal Panel 2 Detection Rate Results for Competitive Interference   

<table><tr><td rowspan=1 colspan=1>Mix</td><td rowspan=1 colspan=1>Pathogen</td><td rowspan=1 colspan=1>Concentrationtested x LoD</td><td rowspan=1 colspan=1>Concentrationtested</td><td rowspan=1 colspan=1>Detectionrate</td><td rowspan=1 colspan=1>CoinfectionDetected</td></tr><tr><td rowspan=2 colspan=1>Norovirus 50x -Rotavirus 3x</td><td rowspan=1 colspan=1>NorovirusGI/GII</td><td rowspan=1 colspan=1>50x</td><td rowspan=1 colspan=1>4.5E+05copies/mL</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>Yes</td></tr><tr><td rowspan=1 colspan=1>Rotavirus A</td><td rowspan=1 colspan=1>3x</td><td rowspan=1 colspan=1>1.7E+04copies/mL</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>Yes</td></tr><tr><td rowspan=2 colspan=1>Norovirus 3x -Rotavirus 50x</td><td rowspan=1 colspan=1>NorovirusGI/GII</td><td rowspan=1 colspan=1>3x</td><td rowspan=1 colspan=1>2.7E+04copies/mL</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>Yes</td></tr><tr><td rowspan=1 colspan=1>Rotavirus A</td><td rowspan=1 colspan=1>50x</td><td rowspan=1 colspan=1>2.9E+05copies/mL</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>Yes</td></tr><tr><td rowspan=2 colspan=1>Giardia 50x -Adenovirus 3x</td><td rowspan=1 colspan=1>Giardialamblia</td><td rowspan=1 colspan=1>50x</td><td rowspan=1 colspan=1>7.2 E+05copies/mL</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>Yes</td></tr><tr><td rowspan=1 colspan=1>AdenovirusF40/F41</td><td rowspan=1 colspan=1>3x</td><td rowspan=1 colspan=1>2.9E+03copies/mL</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>Yes</td></tr></table>

\*Target only applicable for Para-Pak C&S samples

# Analytical Reactivity (Inclusivity)

Analytical Reactivity (Inclusivity) testing was conducted to assess the ability of the QIAstatDx Gastrointestinal panel 2 to detect gastrointestinal pathogen isolates/strains that were selected based on clinical relevance as well as temporal, geographical and phylogenetic diversity. Samples were prepared with pathogen mixes in negative stool matrix collected in Para-Pak C&S transport medium. Based on the previously established LoD for each assay target, three serial log dilutions were tested with four replicates per concentration and per strain/isolate. The lowest concentration for which all four replicates were detected was identified and two more replicates were tested for a total of six sample replicates. Most pathogen strains evaluated (104/114) were detected at ${ \le } 3$ -fold of the corresponding LoD reference strain. Less than $100 \%$ detection was observed for one strain each of ETEC, EIEC/Shigella and Rotavirus and two strains each of STEC (one STEC O157), Adenovirus and Norovirus when tested at a concentration of $< 3 \mathbf { x }$ LoD. Testing of these strains at $1 0 \mathrm { x }$ LoD generated the expected positive resulted for all replicates. For each strain evaluated in the study, the lowest concentration (based on a factor of LoD) that generated positive results for $6 / 6$ replicates are presented in tables 10-26 below. LoD reference strains are shown in bold for each targeted analyte.

An additonal evaluation of inclusivity for the QIAstat-Dx GI Panel 2 targets was conducted using in silico analysis of bacterial, viral, and parasite strains with available sequences in the database. All single primers and probes used for detection of each assay target included in the QIAstat-Dx Gastrointestinal Panel 2 were analyzed to predict the reactivity of the panel using BLAST. Further analysis of the predicted reactivity was performed through alignment

analysis including both entire genome sequences and specific target gene sequences available from the identified BLAST accession hits. Results from in silico analysis are presented in Table 27 below.

Based on the results observed for inclusivity in vitro (wet) testing and in silico analysis, the QIAstat-Dx GI Panel 2 primers and probes are inclusive for each targeted analyte for clinically relevant strains, including species, subspecies, subtypes, serotypes or serovars, as applicable.

Table 10: Inclusivity, Campylobacter strains   

<table><tr><td rowspan=1 colspan=1>QIAstat-Dx target</td><td rowspan=1 colspan=1>Pathogen</td><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>Supplier</td><td rowspan=1 colspan=1>Catalog ID</td><td rowspan=1 colspan=1>Factorof LoD</td></tr><tr><td rowspan=11 colspan=1>Campylobacter</td><td rowspan=1 colspan=1>Campylobacter coli</td><td rowspan=1 colspan=1>76-GA2[LMG21266]</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>434781</td><td rowspan=1 colspan=1>1x LoD</td></tr><tr><td rowspan=1 colspan=1>Campylobacter coli</td><td rowspan=1 colspan=1>Z293</td><td rowspan=1 colspan=1>ZeptoMetrix</td><td rowspan=1 colspan=1>0804272</td><td rowspan=1 colspan=1>1x LoD</td></tr><tr><td rowspan=1 colspan=1>Campylobacter coli</td><td rowspan=1 colspan=1>CIP 7080[1407, CIP70.801</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>335591</td><td rowspan=1 colspan=1>3x LoD</td></tr><tr><td rowspan=1 colspan=1>Campylobacter jejuni</td><td rowspan=1 colspan=1>Z086</td><td rowspan=1 colspan=1>ZeptoMetrix</td><td rowspan=1 colspan=1>08016501</td><td rowspan=1 colspan=1>1x LoD</td></tr><tr><td rowspan=1 colspan=1>Campylobacter jejuni</td><td rowspan=1 colspan=1>subsp. jejuniRM3193</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>BAA-1234*</td><td rowspan=1 colspan=1>0.1x LoD</td></tr><tr><td rowspan=1 colspan=1>Campylobacter jejunisubsp. jejuni</td><td rowspan=1 colspan=1>O:19 HL7;D3180</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>BAA-218</td><td rowspan=1 colspan=1>0.1x LoD</td></tr><tr><td rowspan=1 colspan=1>Campylobacter jejunisubsp. jejuni</td><td rowspan=1 colspan=1>AS-83-79</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>33291</td><td rowspan=1 colspan=1>0.1x LoD</td></tr><tr><td rowspan=1 colspan=1>Campylobacter jejunisubsp. doylei</td><td rowspan=1 colspan=1>NCTC11951</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>49349</td><td rowspan=1 colspan=1>0.1x LoD</td></tr><tr><td rowspan=1 colspan=1>Campylobacterupsaliensis</td><td rowspan=1 colspan=1>NCTC11541</td><td rowspan=1 colspan=1>ZeptoMetrix</td><td rowspan=1 colspan=1>08019991</td><td rowspan=1 colspan=1>1x LoD</td></tr><tr><td rowspan=1 colspan=1>Campylobacterupsaliensis</td><td rowspan=1 colspan=1>RM 3195(1994)</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>BAA-1059</td><td rowspan=1 colspan=1>0.3x LoD</td></tr><tr><td rowspan=1 colspan=1>Campylobacterupsaliensis</td><td rowspan=1 colspan=1>NCTC11541[C231]</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>43954</td><td rowspan=1 colspan=1>1x LoD</td></tr></table>

1Strain tested during LoD verification study.

Table 11: Inclusivity, Plesiomonas shigelloides strains   

<table><tr><td colspan="1" rowspan="1">QIAstat-Dx target</td><td colspan="1" rowspan="1">Pathogen</td><td colspan="1" rowspan="1">Strain</td><td colspan="1" rowspan="1">Supplier</td><td colspan="1" rowspan="1">Catalog ID</td><td colspan="1" rowspan="1">Factorof LoD</td></tr><tr><td colspan="1" rowspan="2">Plesiomonasshigelloides</td><td colspan="1" rowspan="1">Plesiomonasshigelloides</td><td colspan="1" rowspan="1">Z130</td><td colspan="1" rowspan="1">ZeptoMetrix</td><td colspan="1" rowspan="1">08018991</td><td colspan="1" rowspan="1">1x LoD</td></tr><tr><td colspan="1" rowspan="1">Plesiomonasshigelloides</td><td colspan="1" rowspan="1">GNI 14</td><td colspan="1" rowspan="1">ATCC</td><td colspan="1" rowspan="1">51903</td><td colspan="1" rowspan="1">1x LoD</td></tr><tr><td>QIAstat-Dx target</td><td>Pathogen</td><td>Strain</td><td>Supplier</td><td>Catalog ID</td><td>Factor of LoD</td></tr><tr><td></td><td>Plesiomonas shigelloides</td><td>CDC 3085- 55 [Bader M51, NCIB 9242, NCTC 10360, RH 798]</td><td>ATCC</td><td>140291</td><td>0.3x LoD</td></tr></table>

1 Strain tested during LoD verification study.

Table 12: Inclusivity, Salmonella strains   

<table><tr><td colspan="1" rowspan="1">QIAstat-Dxtarget</td><td colspan="1" rowspan="1">Pathogen</td><td colspan="1" rowspan="1">Strain</td><td colspan="1" rowspan="1">Supplier</td><td colspan="1" rowspan="1">Catalog ID</td><td colspan="1" rowspan="1">Factor ofLoD</td></tr><tr><td colspan="1" rowspan="15">Salmonella</td><td colspan="1" rowspan="1">Salmonellaenterica</td><td colspan="1" rowspan="1">Serovar TyphimuriumZ005</td><td colspan="1" rowspan="1">ZeptoMetrix</td><td colspan="1" rowspan="1">08014371</td><td colspan="1" rowspan="1">1x LoD</td></tr><tr><td colspan="1" rowspan="1">Salmonellaenterica</td><td colspan="1" rowspan="1">Subsp. Enterica,serovar Bareilly</td><td colspan="1" rowspan="1">NCTC</td><td colspan="1" rowspan="1">NC05745</td><td colspan="1" rowspan="1">1x LoD</td></tr><tr><td colspan="1" rowspan="1">Salmonellaenterica</td><td colspan="1" rowspan="1">Subsp. Enterica, serovartyphi, Z152</td><td colspan="1" rowspan="1">ZeptoMetrix</td><td colspan="1" rowspan="1">0801933</td><td colspan="1" rowspan="1">0.1x LoD</td></tr><tr><td colspan="1" rowspan="1">Salmonellaenterica</td><td colspan="1" rowspan="1">Subsp. Enterica, serovarEnteridis, CDC K-1891[ATCC 25928]</td><td colspan="1" rowspan="1">ATCC</td><td colspan="1" rowspan="1">13076</td><td colspan="1" rowspan="1">0.1x LoD</td></tr><tr><td colspan="1" rowspan="1">Salmonellaenterica</td><td colspan="1" rowspan="1">Subsp. Enterica, serovarInfantis, MZ1479[SARB27]</td><td colspan="1" rowspan="1">ATCC</td><td colspan="1" rowspan="1">BAA-1675</td><td colspan="1" rowspan="1">0.1x LoD</td></tr><tr><td colspan="1" rowspan="1">Salmonellaenterica</td><td colspan="1" rowspan="1">Subsp. Enterica, serovarMontevideo, G4639</td><td colspan="1" rowspan="1">ATCC</td><td colspan="1" rowspan="1">BAA-710</td><td colspan="1" rowspan="1">0.1x LoD</td></tr><tr><td colspan="1" rowspan="1">Salmonellaenterica</td><td colspan="1" rowspan="1">Subsp. Enterica, serovarJaviana</td><td colspan="1" rowspan="1">NCTC</td><td colspan="1" rowspan="1">NC06495</td><td colspan="1" rowspan="1">0.1x LoD</td></tr><tr><td colspan="1" rowspan="1">Salmonellaenterica</td><td colspan="1" rowspan="1">Subsp. Enterica, serovarThompson</td><td colspan="1" rowspan="1">NCTC</td><td colspan="1" rowspan="1">NC08496</td><td colspan="1" rowspan="1">0.1x LoD</td></tr><tr><td colspan="1" rowspan="1">Salmonellaenterica</td><td colspan="1" rowspan="1">Subsp. Enterica, serovarSaintpaul</td><td colspan="1" rowspan="1">ATCC</td><td colspan="1" rowspan="1">9712</td><td colspan="1" rowspan="1">0.1x LoD</td></tr><tr><td colspan="1" rowspan="1">Salmonellaenterica</td><td colspan="1" rowspan="1">Subsp. Enterica, serovarBerta</td><td colspan="1" rowspan="1">NCTC</td><td colspan="1" rowspan="1">NC05770</td><td colspan="1" rowspan="1">0.1x LoD</td></tr><tr><td colspan="1" rowspan="1">Salmonellaenterica</td><td colspan="1" rowspan="1">Subsp. Salame, II NCTC10310 [JT945, SS140/61]</td><td colspan="1" rowspan="1">ATCC</td><td colspan="1" rowspan="1">700151</td><td colspan="1" rowspan="1">0.1x LoD</td></tr><tr><td colspan="1" rowspan="1">Salmonellaenterica</td><td colspan="1" rowspan="1">Subsp. diarizonae IIIb, 62</td><td colspan="1" rowspan="1">ATCC</td><td colspan="1" rowspan="1">29934</td><td colspan="1" rowspan="1">0.1x LoD</td></tr><tr><td colspan="1" rowspan="1">Salmonellaenterica</td><td colspan="1" rowspan="1">Subsp. houtenae IV, CIP82.32 [264.66]</td><td colspan="1" rowspan="1">ATCC</td><td colspan="1" rowspan="1">43974</td><td colspan="1" rowspan="1">0.1x LoD</td></tr><tr><td colspan="1" rowspan="1">Salmonellaenterica</td><td colspan="1" rowspan="1">Subsp. Indica VI, CIP102501 [F. Kauffmann1240]</td><td colspan="1" rowspan="1">ATCC</td><td colspan="1" rowspan="1">43976</td><td colspan="1" rowspan="1">0.1x LoD</td></tr><tr><td colspan="1" rowspan="1">Salmonellaenterica</td><td colspan="1" rowspan="1">Subsp. Enterica, serovarAgona, CDC 873 [CDC1-61]</td><td colspan="1" rowspan="1">ATCC</td><td colspan="1" rowspan="1">51957</td><td colspan="1" rowspan="1">0.1x LoD</td></tr><tr><td colspan="1" rowspan="12"></td><td colspan="1" rowspan="1">Salmonellaenterica</td><td colspan="1" rowspan="1">Subsp. Enterica, serovarMuenchen, 54</td><td colspan="1" rowspan="1">ATCC</td><td colspan="1" rowspan="1">8388</td><td colspan="1" rowspan="1">0.1x LoD</td></tr><tr><td colspan="1" rowspan="1">Salmonellaenterica</td><td colspan="1" rowspan="1">Subsp. Enterica, serovarOranienburg, E1093</td><td colspan="1" rowspan="1">ATCC</td><td colspan="1" rowspan="1">9239</td><td colspan="1" rowspan="1">0.1x LoD</td></tr><tr><td colspan="1" rowspan="1">Salmonellaenterica</td><td colspan="1" rowspan="1">Subsp. Enterica, serovarParatyphi B var. Java, CDC5</td><td colspan="1" rowspan="1">ATCC</td><td colspan="1" rowspan="1">51962</td><td colspan="1" rowspan="1">0.1x LoD</td></tr><tr><td colspan="1" rowspan="1">Salmonellabongori</td><td colspan="1" rowspan="1">CIP 82.33 [1224.72]</td><td colspan="1" rowspan="1">ATCC</td><td colspan="1" rowspan="1">43975</td><td colspan="1" rowspan="1">0.3x LoD</td></tr><tr><td colspan="1" rowspan="1">Salmonellaenterica</td><td colspan="1" rowspan="1">Subsp. Enterica, serovarCholeraesius,NCTC 5735[1348, K.34]</td><td colspan="1" rowspan="1">ATCC</td><td colspan="1" rowspan="1">133121</td><td colspan="1" rowspan="1">0.3x LoD</td></tr><tr><td colspan="1" rowspan="1">Salmonellaenterica</td><td colspan="1" rowspan="1">Subsp. Enterica, serovarNewport, C487-69</td><td colspan="1" rowspan="1">ATCC</td><td colspan="1" rowspan="1">27869</td><td colspan="1" rowspan="1">0.3x LoD</td></tr><tr><td colspan="1" rowspan="1">Salmonellaenterica</td><td colspan="1" rowspan="1">Subsp. Enterica, 4, 5, 12:7serovar Typhimurium</td><td colspan="1" rowspan="1">NCTC</td><td colspan="1" rowspan="1">NC13952</td><td colspan="1" rowspan="1">0.3x LoD</td></tr><tr><td colspan="1" rowspan="1">Salmonellaenterica</td><td colspan="1" rowspan="1">Subsp. Enterica, serovarBraenderup</td><td colspan="1" rowspan="1">ATCC</td><td colspan="1" rowspan="1">700136</td><td colspan="1" rowspan="1">0.3x LoD</td></tr><tr><td colspan="1" rowspan="1">Salmonellaenterica</td><td colspan="1" rowspan="1">Subsp. Enterica, serovarAnatum</td><td colspan="1" rowspan="1">NCTC</td><td colspan="1" rowspan="1">NC05779</td><td colspan="1" rowspan="1">0.3x LoD</td></tr><tr><td colspan="1" rowspan="1">Salmonellaenterica</td><td colspan="1" rowspan="1">Subps. arizonae IIIa, NCTC7311 [CDAI 426]</td><td colspan="1" rowspan="1">ATCC</td><td colspan="1" rowspan="1">700156</td><td colspan="1" rowspan="1">0.3x LoD</td></tr><tr><td colspan="1" rowspan="1">Salmonellaenterica</td><td colspan="1" rowspan="1">Subsp. Enterica, serovarHeidelberg, [16]</td><td colspan="1" rowspan="1">ATCC</td><td colspan="1" rowspan="1">8326</td><td colspan="1" rowspan="1">0.3x LoD</td></tr><tr><td colspan="1" rowspan="1">Salmonellaenterica</td><td colspan="1" rowspan="1">Subsp. Enterica, serovarMississippi, CDC 2012K-0487</td><td colspan="1" rowspan="1">ATCC</td><td colspan="1" rowspan="1">BAA-2739</td><td colspan="1" rowspan="1">0.3x LoD</td></tr></table>

1 Strain tested during LoD verification study

Table 13: Inclusivity, Yersinia enterocolitica strains   

<table><tr><td rowspan=1 colspan=1>QIAstat-Dx target</td><td rowspan=1 colspan=1>Pathogen</td><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>Supplier</td><td rowspan=1 colspan=1>Catalog ID</td><td rowspan=1 colspan=1>TimesLoD</td></tr><tr><td rowspan=3 colspan=1>Yersiniaenterocolitica</td><td rowspan=1 colspan=1>Yersiniaenterocolitica</td><td rowspan=1 colspan=1>Z036</td><td rowspan=1 colspan=1>ZeptoMetrix</td><td rowspan=1 colspan=1>08017341</td><td rowspan=1 colspan=1>1x LoD</td></tr><tr><td rowspan=1 colspan=1>Yersiniaenterocolitica</td><td rowspan=1 colspan=1>NTCC 11175,Biotype 4,serotype 3 (0:3)</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>700822*</td><td rowspan=1 colspan=1>1x LoD</td></tr><tr><td rowspan=1 colspan=1>Yersiniaenterocolitica</td><td rowspan=1 colspan=1>33114 [CCUG11291, CCUG12369, CIP80.27, DSM4780, LMG 7899,NCTC 12982],Biovar 1, O:8</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>9610</td><td rowspan=1 colspan=1>1x LoD</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Yersiniaenterocolitica</td><td rowspan=1 colspan=1>0:9</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>55075</td><td rowspan=1 colspan=1>3x LoD</td></tr></table>

1 Strain tested during LoD verification study

Table 14: Inclusivity, Enteropathogenic E. coli (EPEC) strains   

<table><tr><td rowspan=1 colspan=1>QIAstat-Dx target</td><td rowspan=1 colspan=1>Pathogen</td><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>Supplier</td><td rowspan=1 colspan=1>CatalogID</td><td rowspan=1 colspan=1>TimesLoD</td></tr><tr><td rowspan=3 colspan=1>Enteropathogenic E.coli (EPEC)</td><td rowspan=1 colspan=1>EnteropathogenicE. coli (EPEC)</td><td rowspan=1 colspan=1>0111:NM</td><td rowspan=1 colspan=1>ZeptoMetrix</td><td rowspan=1 colspan=1>08017471</td><td rowspan=1 colspan=1>1xLoD</td></tr><tr><td rowspan=1 colspan=1>EnteropathogenicE. coli (EPEC)</td><td rowspan=1 colspan=1>7.1493,084:H28</td><td rowspan=1 colspan=1>ZeptoMetrix</td><td rowspan=1 colspan=1>0801938</td><td rowspan=1 colspan=1>1x LoD</td></tr><tr><td rowspan=1 colspan=1>EnteropathogenicE. coli (EPEC)</td><td rowspan=1 colspan=1>StokeW,O111:K58(B4):H-</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>33780</td><td rowspan=1 colspan=1>1x LoD</td></tr></table>

1 Strain tested during LoD verification study

Table 15: Inclusivity, Enterotoxigenic E. coli (ETEC) strains   

<table><tr><td rowspan=1 colspan=1>QIAstat-Dxtarget</td><td rowspan=1 colspan=1>Pathogen</td><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>Supplier</td><td rowspan=1 colspan=1>CatalogID</td><td rowspan=1 colspan=1>TimesLoD</td></tr><tr><td rowspan=5 colspan=1>EnterotoxigenicE. coli (ETEC)lt/st</td><td rowspan=1 colspan=1>EnterotoxigenicE. coli (ETEC)lt/st</td><td rowspan=1 colspan=1>ST+, LT+</td><td rowspan=1 colspan=1>ZeptoMetrix</td><td rowspan=1 colspan=1>08016241</td><td rowspan=1 colspan=1>1xLoD</td></tr><tr><td rowspan=1 colspan=1>EnterotoxigenicE. coli (ETEC) lt/st</td><td rowspan=1 colspan=1>H10407,O78:H11,LT(+)/ctxA11(+)</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>354011</td><td rowspan=1 colspan=1>0.3xLoD</td></tr><tr><td rowspan=1 colspan=1>EnterotoxigenicE. coli (ETEC) lt/st</td><td rowspan=1 colspan=1>027:H7,ST (+)/LT (-)</td><td rowspan=1 colspan=1>SSI Diagnostica</td><td rowspan=1 colspan=1>82173</td><td rowspan=1 colspan=1>0.1xLoD</td></tr><tr><td rowspan=1 colspan=1>EnterotoxigenicE. coli (ETEC) lt/st</td><td rowspan=1 colspan=1>0115:H15,ST (+)/LT (-)</td><td rowspan=1 colspan=1>SSI Diagnostica</td><td rowspan=1 colspan=1>82174</td><td rowspan=1 colspan=1>3xLoD</td></tr><tr><td rowspan=1 colspan=1>EnterotoxigenicE. coli (ETEC) lt/st</td><td rowspan=1 colspan=1>0169:H-,ST (-)/LT (+)</td><td rowspan=1 colspan=1>SSI Diagnostica</td><td rowspan=1 colspan=1>82172</td><td rowspan=1 colspan=1>10xLoD2</td></tr></table>

1 Strain tested during LoD verification study 2 Testing at a lower concentration resulted in a detection rate of ${ < } 1 0 0 \%$

Table 16: Inclusivity, Enteroinvasive E. coli (EIEC)/Shigella strains   

<table><tr><td colspan="1" rowspan="1">QIAstat-Dxtarget</td><td colspan="1" rowspan="1">Pathogen</td><td colspan="1" rowspan="1">Strain</td><td colspan="1" rowspan="1">Supplier</td><td colspan="1" rowspan="1">Catalog ID</td><td colspan="1" rowspan="1">Times LoD</td></tr><tr><td colspan="1" rowspan="7">Enteroinvasive E. coli(EIEC)/Shigella</td><td colspan="1" rowspan="1">EnteroinvasiveE. coli (EIEC)</td><td colspan="1" rowspan="1">CDC EDL1282,029:NM</td><td colspan="1" rowspan="1">ATCC</td><td colspan="1" rowspan="1">438921</td><td colspan="1" rowspan="1">1x LoD</td></tr><tr><td colspan="1" rowspan="1">EnteroinvasiveE. coli (EIEC)</td><td colspan="1" rowspan="1">0172:H-</td><td colspan="1" rowspan="1">SSI Diagnostica</td><td colspan="1" rowspan="1">82171</td><td colspan="1" rowspan="1">3x LoD</td></tr><tr><td colspan="1" rowspan="1">Shigella sonnei</td><td colspan="1" rowspan="1">NCDC1120-66</td><td colspan="1" rowspan="1">ATCC</td><td colspan="1" rowspan="1">259311</td><td colspan="1" rowspan="1">1x LoD</td></tr><tr><td colspan="1" rowspan="1">Shigella boydii(Serogroup C)</td><td colspan="1" rowspan="1">Z131</td><td colspan="1" rowspan="1">ZeptoMetrix</td><td colspan="1" rowspan="1">0801900</td><td colspan="1" rowspan="1">1x LoD</td></tr><tr><td colspan="1" rowspan="1">Shigella flexneri(Serogroup B)</td><td colspan="1" rowspan="1">AMC 43-G-68 [EVL 82,M134]</td><td colspan="1" rowspan="1">ATCC</td><td colspan="1" rowspan="1">9199</td><td colspan="1" rowspan="1">1x LoD</td></tr><tr><td colspan="1" rowspan="1">Shigella flexneri(Serogroup B)</td><td colspan="1" rowspan="1">Z046</td><td colspan="1" rowspan="1">ZeptoMetrix</td><td colspan="1" rowspan="1">0801757</td><td colspan="1" rowspan="1">1x LoD</td></tr><tr><td colspan="1" rowspan="1">Shigella sonnei(Serogroup D)</td><td colspan="1" rowspan="1">WRAIR Ivirulent</td><td colspan="1" rowspan="1">ATCC</td><td colspan="1" rowspan="1">29930</td><td colspan="1" rowspan="1">1x LoD</td></tr><tr><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="1">Shigella sonnei(Serogroup D)</td><td colspan="1" rowspan="1">Z004</td><td colspan="1" rowspan="1">ZeptoMetrix</td><td colspan="1" rowspan="1">0801627</td><td colspan="1" rowspan="1">3x LoD</td></tr><tr><td colspan="1" rowspan="1">Shigella boydii(Serogroup C)</td><td colspan="1" rowspan="1">AMC 43-G-58 [M44(Type 170)]</td><td colspan="1" rowspan="1">ATCC</td><td colspan="1" rowspan="1">9207</td><td colspan="1" rowspan="1">10x LoD2</td></tr></table>

1 Strain tested during LoD verification study 2 Testing at a lower concentration resulted in a detection rate of ${ < } 1 0 0 \%$

Table 17: Inclusivity, Shiga-like toxin-producing E. coli (STEC) (stx2-carrier strains)   

<table><tr><td rowspan=1 colspan=1>QIAstat-Dxtarget</td><td rowspan=1 colspan=1>Pathogen</td><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>Supplier</td><td rowspan=1 colspan=1>Catalog</td><td rowspan=1 colspan=1>TimesLoD</td></tr><tr><td rowspan=11 colspan=1>Shiga-liketoxinproducing E. coli(sSTEC)-stx1/2</td><td rowspan=1 colspan=1>Shiga-like toxin producing E.coli (STEC) - stx1/stx2</td><td rowspan=1 colspan=1>0157:H7;EDL933</td><td rowspan=1 colspan=1>ZeptoMetrix</td><td rowspan=1 colspan=1>08016221</td><td rowspan=1 colspan=1>1x LoD</td></tr><tr><td rowspan=1 colspan=1>Shiga-like toxin producing E.coli (STEC) - stx1</td><td rowspan=1 colspan=1>026:H4,stx1 (+)</td><td rowspan=1 colspan=1>ZeptoMetrix</td><td rowspan=1 colspan=1>08017481</td><td rowspan=1 colspan=1>1x LoD</td></tr><tr><td rowspan=1 colspan=1>Shiga-like toxin producing E.coli (STEC) - stx1/stx2</td><td rowspan=1 colspan=1>Reference ATCC®35150 (EDL931),0157:H7,stx1(+),stx2(+)</td><td rowspan=1 colspan=1>Microbiologics</td><td rowspan=1 colspan=1>617</td><td rowspan=1 colspan=1>3x LoD</td></tr><tr><td rowspan=1 colspan=1>Shiga-like toxin producing E.coli (STEC) - stx1</td><td rowspan=1 colspan=1>Reference CDC 00-3039,O45:H2,unknown</td><td rowspan=1 colspan=1>Microbiologics</td><td rowspan=1 colspan=1>1098</td><td rowspan=1 colspan=1>1x LoD</td></tr><tr><td rowspan=1 colspan=1>Shiga-like toxin producing E.coli (STEC) - stx1</td><td rowspan=1 colspan=1>0103:H2,stx1 (+)</td><td rowspan=1 colspan=1>SSIDiagnostica</td><td rowspan=1 colspan=1>82170</td><td rowspan=1 colspan=1>3x LoD</td></tr><tr><td rowspan=1 colspan=1>Shiga-like toxin producing E.coli (STEC) - stx1/stx2</td><td rowspan=1 colspan=1>O22:H8,stx1c (+),stx2b(+)</td><td rowspan=1 colspan=1>SSIDiagnostica</td><td rowspan=1 colspan=1>91350</td><td rowspan=1 colspan=1>1x LoD</td></tr><tr><td rowspan=1 colspan=1>Shiga-like toxin producing E.coli (STEC) - stx2</td><td rowspan=1 colspan=1>092,0107:K+:H48,stx2d(+)</td><td rowspan=1 colspan=1>SSIDiagnostica</td><td rowspan=1 colspan=1>91352</td><td rowspan=1 colspan=1>10x LoD2</td></tr><tr><td rowspan=1 colspan=1>Shiga-like toxin producing E.coli (STEC) - stx2</td><td rowspan=1 colspan=1>0101:K32:H-,stx2e(+)</td><td rowspan=1 colspan=1>SSIDiagnostica</td><td rowspan=1 colspan=1>91354</td><td rowspan=1 colspan=1>0.3x LoD</td></tr><tr><td rowspan=1 colspan=1>Shiga-like toxin producing E.coli (STEC) - stx1/stx2</td><td rowspan=1 colspan=1>0128ac:H-,stx2f(+)</td><td rowspan=1 colspan=1>SSIDiagnostica</td><td rowspan=1 colspan=1>91355</td><td rowspan=1 colspan=1>10x LoD2</td></tr><tr><td rowspan=1 colspan=1>Shiga-like toxin producing E.coli (STEC) - stx2</td><td rowspan=1 colspan=1>026:H11,stx2a (+)</td><td rowspan=1 colspan=1>SSIDiagnostica</td><td rowspan=1 colspan=1>95211</td><td rowspan=1 colspan=1>1x LoD</td></tr><tr><td rowspan=1 colspan=1>Shiga-like toxin producing E.coli (STEC) - stx1</td><td rowspan=1 colspan=1>08 ,sx1d (+)</td><td rowspan=1 colspan=1>SSIDiagnostica</td><td rowspan=1 colspan=1>91349</td><td rowspan=1 colspan=1>1x LoD</td></tr></table>

1 Strain tested during LoD verification study 2 Testing at a lower concentration resulted in a detection rate of ${ < } 1 0 0 \%$

Table 18: Inclusivity, Shiga-like toxin producing E. coli (STEC) O157 strains   

<table><tr><td rowspan=1 colspan=1>QIAstat-Dxtarget</td><td rowspan=1 colspan=1>Pathogen</td><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>Supplier</td><td rowspan=1 colspan=1>Catalogue ID</td><td rowspan=1 colspan=1>TimesLoD</td></tr><tr><td rowspan=3 colspan=1>Shiga-like toxinproducing E.coli (STEC)0157</td><td rowspan=1 colspan=1>Shiga-like toxinproducing E.coli (STEC) -0157</td><td rowspan=1 colspan=1>0157:H7;EDL933</td><td rowspan=1 colspan=1>ZeptoMetrix</td><td rowspan=1 colspan=1>08016221</td><td rowspan=1 colspan=1>1x LoD</td></tr><tr><td rowspan=1 colspan=1>Shiga-like toxinproducing E. coli(STEC) 0157</td><td rowspan=1 colspan=1>0128ac:H-,stx2f(+)</td><td rowspan=1 colspan=1>SSI Diagnostica</td><td rowspan=1 colspan=1>913552</td><td rowspan=1 colspan=1>10x LoD3</td></tr><tr><td rowspan=1 colspan=1>Shiga-like toxinproducing E. coli(STEC) O157</td><td rowspan=1 colspan=1>Reference ATCC35150 (EDL931), O157:H7,stx1 (+), stx2 (+)</td><td rowspan=1 colspan=1>Microbiologics</td><td rowspan=1 colspan=1>617</td><td rowspan=1 colspan=1>1x LoD</td></tr></table>

1 Strain tested during LoD verification study 2 The $E .$ . coli strain 91355 from SSI Diagnostica is reported as following in its catalog: $\nu t x 2 f \mathrm { + }$ , eae+. However, it was found to amplify for $E$ . coli O157 in both QIAstat-Dx and FilmArray device

3 Testing at a lower concentration resulted in a detection rate of ${ < } 1 0 0 \%$

Table 19: Inclusivity, Cryptosporidium strains   

<table><tr><td rowspan=1 colspan=1>QIAstat-Dx target</td><td rowspan=1 colspan=1>Pathogen</td><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>Supplier</td><td rowspan=1 colspan=1>Catalog ID</td><td rowspan=1 colspan=1>TimesLoD</td></tr><tr><td rowspan=5 colspan=1>Cryptosporidium</td><td rowspan=1 colspan=1>Cryptosporidium parvum</td><td rowspan=1 colspan=1>Iowaisolate</td><td rowspan=1 colspan=1>Waterborne</td><td rowspan=1 colspan=1>P102C1</td><td rowspan=1 colspan=1>1x LoD</td></tr><tr><td rowspan=1 colspan=1>Cryptosporidium hominis</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>Public HealthWales</td><td rowspan=1 colspan=1>Clinicalsample;UKM 841</td><td rowspan=1 colspan=1>0.01xLoD</td></tr><tr><td rowspan=1 colspan=1>Cryptosporidium parvum</td><td rowspan=1 colspan=1>−</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>PRA-67DQ(isolatedgenomicDNA</td><td rowspan=1 colspan=1>&lt;0.01LoD</td></tr><tr><td rowspan=1 colspan=1>Cryptosporidiummeleagridis</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>Public HealthWales</td><td rowspan=1 colspan=1>Clinicalsample;UKMEL 14</td><td rowspan=1 colspan=1>&lt;0.01LoD</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr></table>

Strain tested during LoD verification study

Table 20: Inclusivity, Cyclospora cayetanensis strains   

<table><tr><td rowspan=1 colspan=1>QIAstat-Dx target</td><td rowspan=1 colspan=1>Pathogen</td><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>Supplier</td><td rowspan=1 colspan=1>Catalogue ID</td><td rowspan=1 colspan=1>Times LoD</td></tr><tr><td rowspan=3 colspan=1>Cyclosporacayetanensis</td><td rowspan=1 colspan=1>Cyclosporacayetanensis</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>Clinicalsample</td><td rowspan=1 colspan=1>LAC28251</td><td rowspan=1 colspan=1>1x LoD</td></tr><tr><td rowspan=1 colspan=1>Cyclosporacayetanensis</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>Clinicalsample</td><td rowspan=1 colspan=1>LAC28271</td><td rowspan=1 colspan=1>1x LoD</td></tr><tr><td rowspan=1 colspan=1>Cyclosporacayetanensis</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>PRA-3000SD</td><td rowspan=1 colspan=1>1x LoD</td></tr></table>

1 Strain tested during LoD verification study

Table 21: Inclusivity, Entamoeba histolytica strains   

<table><tr><td rowspan=1 colspan=1>QIAstat-Dxtarget</td><td rowspan=1 colspan=1>Pathogen</td><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>Supplier</td><td rowspan=1 colspan=1>Catalogue ID</td><td rowspan=1 colspan=1>TimesLoD</td></tr><tr><td rowspan=3 colspan=1>Entamoebahistolytica</td><td rowspan=1 colspan=1>Entamoeba histolytica</td><td rowspan=1 colspan=1>HM-1:IMSS(Mexico City1967)</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>304591</td><td rowspan=1 colspan=1>1x LoD</td></tr><tr><td rowspan=1 colspan=1>Entamoeba histolytica</td><td rowspan=1 colspan=1>HK-9 (Korea)</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>300151</td><td rowspan=1 colspan=1>1x LoD</td></tr><tr><td rowspan=1 colspan=1>Entamoeba histolytica</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Vall d&#x27;Hebrón</td><td rowspan=1 colspan=1>Clinical sample</td><td rowspan=1 colspan=1>1x LoD</td></tr></table>

1 Strain tested during LoD verification study

Table 22: Inclusivity, Giardia lamblia strains   

<table><tr><td rowspan=1 colspan=1>QIAstat-Dxtarget</td><td rowspan=1 colspan=1>Pathogen</td><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>Supplier</td><td rowspan=1 colspan=1>Catalogue ID</td><td rowspan=1 colspan=1>Times LoD</td></tr><tr><td rowspan=3 colspan=1>Giardialamblia</td><td rowspan=1 colspan=1>Giardia lamblia</td><td rowspan=1 colspan=1>Portland -1(Portland, OR,1971)</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>308881</td><td rowspan=1 colspan=1>1x LoD</td></tr><tr><td rowspan=1 colspan=1>Giardia lamblia</td><td rowspan=1 colspan=1>WB (Bethesda,MD, 1979</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>309571</td><td rowspan=1 colspan=1>1x LoD</td></tr><tr><td rowspan=1 colspan=1>Giardia intestinalis</td><td rowspan=1 colspan=1>H3 isolate</td><td rowspan=1 colspan=1>Waterborne</td><td rowspan=1 colspan=1>P101</td><td rowspan=1 colspan=1>1x LoD</td></tr></table>

Strain tested during LoD verification study

Table 23: Inclusivity, Adenovirus F40/F41 strains   

<table><tr><td rowspan=1 colspan=1>QIAstat-Dxtarget</td><td rowspan=1 colspan=1>Pathogen</td><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>Supplier</td><td rowspan=1 colspan=1>Catalog ID</td><td rowspan=1 colspan=1>TimesLoD</td></tr><tr><td rowspan=4 colspan=1>AdenovirusF40/F41</td><td rowspan=1 colspan=1>HumanAdenovirus F41</td><td rowspan=1 colspan=1>Tak</td><td rowspan=1 colspan=1>ZeptoMetrix</td><td rowspan=1 colspan=1>0810085CF1</td><td rowspan=1 colspan=1>1x LoD</td></tr><tr><td rowspan=1 colspan=1>HumanAdenovirus F41</td><td rowspan=1 colspan=1>Tak (73-3544)</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>VR-930</td><td rowspan=1 colspan=1>10xLoD2</td></tr><tr><td rowspan=1 colspan=1>HumanAdenovirus F40</td><td rowspan=1 colspan=1>Dugan[79-18025]</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>VR-931</td><td rowspan=1 colspan=1>10xLoD2</td></tr><tr><td rowspan=1 colspan=1>HumanAdenovirus Type40</td><td rowspan=1 colspan=1>Dugan</td><td rowspan=1 colspan=1>ZeptoMetrix</td><td rowspan=1 colspan=1>0810084CF1</td><td rowspan=1 colspan=1>3x LoD</td></tr></table>

1 Strain tested during LoD verification study 2 Testing at a lower concentration resulted in a detection rate of ${ < } 1 0 0 \%$

Table 24: Inclusivity, Astrovirus strains   

<table><tr><td colspan="1" rowspan="1">QIAstat-Dxtarget</td><td colspan="1" rowspan="1">Pathogen</td><td colspan="1" rowspan="1">Strain</td><td colspan="1" rowspan="1">Supplier</td><td colspan="1" rowspan="1">Catalog ID</td><td colspan="1" rowspan="1">Times LoD</td></tr><tr><td colspan="1" rowspan="2">Astrovirus</td><td colspan="1" rowspan="1">HumanAstrovirus</td><td colspan="1" rowspan="1">ERE IID2371 (type8)</td><td colspan="1" rowspan="1">ZeptoMetrix</td><td colspan="1" rowspan="1">0810277CF1</td><td colspan="1" rowspan="1">1x LoD</td></tr><tr><td colspan="1" rowspan="1">HumanAstrovirus</td><td colspan="1" rowspan="1">HAstV-1</td><td colspan="1" rowspan="1">Universitat deBarcelona</td><td colspan="1" rowspan="1">Clinical sample;160521599</td><td colspan="1" rowspan="1">1x LoD</td></tr><tr><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="1">HumanAstrovirus</td><td colspan="1" rowspan="1">ERE IID2868 (type4)</td><td colspan="1" rowspan="1">ZeptoMetrix</td><td colspan="1" rowspan="1">0810276CF1</td><td colspan="1" rowspan="1">1x LoD</td></tr><tr><td colspan="1" rowspan="1">HumanAstrovirus</td><td colspan="1" rowspan="1">HAstV-3</td><td colspan="1" rowspan="1">Universitat deBarcelona</td><td colspan="1" rowspan="1">Clinical sample;151601306</td><td colspan="1" rowspan="1">1x LoD</td></tr></table>

1 Strain tested during LoD verification study

Table 25: Inclusivity, Norovirus GI/GII strains   

<table><tr><td rowspan=1 colspan=1>QIAstat-Dxtarget</td><td rowspan=1 colspan=1>Pathogen</td><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>Supplier</td><td rowspan=1 colspan=1>Catalog ID</td><td rowspan=1 colspan=1>TimesLoD</td></tr><tr><td rowspan=12 colspan=1>NorovirusGI/GII</td><td rowspan=1 colspan=1>HumanNorovirusGenogroup 1</td><td rowspan=1 colspan=1>RecombinantGI.1</td><td rowspan=1 colspan=1>ZeptoMetrix</td><td rowspan=1 colspan=1>0810086CF1</td><td rowspan=1 colspan=1>1x LoD</td></tr><tr><td rowspan=1 colspan=1>HumanNorovirusGenogroup 1</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Indiana UniversityHealth</td><td rowspan=1 colspan=1>Clinical sample;IU3156</td><td rowspan=1 colspan=1>1x LoD</td></tr><tr><td rowspan=1 colspan=1>HumanNorovirusGenogroup 1</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Indiana UniversityHealth</td><td rowspan=1 colspan=1>Clinical sample;IU3220</td><td rowspan=1 colspan=1>1x LoD</td></tr><tr><td rowspan=1 colspan=1>HumanNorovirusGenogroup 1</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>TriCore ReferenceLaboratories</td><td rowspan=1 colspan=1>Clinical sample;TC4274</td><td rowspan=1 colspan=1>3x LoD</td></tr><tr><td rowspan=1 colspan=1>HumanNorovirusGenogroup 2</td><td rowspan=1 colspan=1>RecombinantGII.4</td><td rowspan=1 colspan=1>ZeptoMetrix</td><td rowspan=1 colspan=1>0810087CF1</td><td rowspan=1 colspan=1>1x LoD</td></tr><tr><td rowspan=1 colspan=1>HumanNorovirusGenogroup 2</td><td rowspan=1 colspan=1>GII.2</td><td rowspan=1 colspan=1>Vall d&#x27;Hebrón</td><td rowspan=1 colspan=1>Clinical sample;198058327</td><td rowspan=1 colspan=1>1x LoD</td></tr><tr><td rowspan=1 colspan=1>HumanNorovirusGenogroup 2</td><td rowspan=1 colspan=1>GII.4</td><td rowspan=1 colspan=1>Universitat deBarcelona</td><td rowspan=1 colspan=1>Clinical sample;N26.2TA</td><td rowspan=1 colspan=1>1x LoD</td></tr><tr><td rowspan=1 colspan=1>HumanNorovirusGenogroup 2</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>Lacny Hospital</td><td rowspan=1 colspan=1>Clinical sample;LAC2019</td><td rowspan=1 colspan=1>1x LoD</td></tr><tr><td rowspan=1 colspan=1>HumanNorovirusGenogroup 2</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>NationwideChildren&#x27;sHospital</td><td rowspan=1 colspan=1>Clinical sample;NWC6063</td><td rowspan=1 colspan=1>1x LoD</td></tr><tr><td rowspan=1 colspan=1>HumanNorovirusGenogroup 2</td><td rowspan=1 colspan=1>GII.6</td><td rowspan=1 colspan=1>QIAGENBarcelona(STAT-Dx)</td><td rowspan=1 colspan=1>Clinical sample;GI 12</td><td rowspan=1 colspan=1>3x LoD</td></tr><tr><td rowspan=1 colspan=1>HumanNorovirusGenogroup 2</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Lacny Hospital</td><td rowspan=1 colspan=1>Clinical sample;LAC2133</td><td rowspan=1 colspan=1>10xLoD2</td></tr><tr><td rowspan=1 colspan=1>HumanNorovirusGenogroup 2</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Lacny Hospital</td><td rowspan=1 colspan=1>Clinical sample;LAC2074</td><td rowspan=1 colspan=1>10xLoD2</td></tr></table>

1 Strain tested during LoD verification study 2 Testing at a lower concentration resulted in a detection rate of ${ < } 1 0 0 \%$

Table 26: Inclusivity, Rotavirus A strains   

<table><tr><td rowspan=1 colspan=1>QIAstat-Dxtarget</td><td rowspan=1 colspan=1>Pathogen</td><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>Supplier</td><td rowspan=1 colspan=1>Catalog ID</td><td rowspan=1 colspan=1>TimesLoD</td></tr><tr><td rowspan=5 colspan=1>Rotavirus A</td><td rowspan=1 colspan=1>HumanRotavirus A</td><td rowspan=1 colspan=1>69M</td><td rowspan=1 colspan=1>ZeptoMetrix</td><td rowspan=1 colspan=1>0810280CF1</td><td rowspan=1 colspan=1>1x LoD</td></tr><tr><td rowspan=1 colspan=1>HumanRotavirus A</td><td rowspan=1 colspan=1>Wa,G1P1A[8]</td><td rowspan=1 colspan=1>ZeptoMetrix</td><td rowspan=1 colspan=1>0810041CF1</td><td rowspan=1 colspan=1>1x LoD</td></tr><tr><td rowspan=1 colspan=1>HumanRotavirus A</td><td rowspan=1 colspan=1>DS-1,G2P1B[4]</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>VR-2550</td><td rowspan=1 colspan=1>1x LoD</td></tr><tr><td rowspan=1 colspan=1>HumanRotavirus A</td><td rowspan=1 colspan=1>Va70</td><td rowspan=1 colspan=1>ZeptoMetrix</td><td rowspan=1 colspan=1>0810281CF</td><td rowspan=1 colspan=1>1x LoD</td></tr><tr><td rowspan=1 colspan=1>HumanRotavirus A</td><td rowspan=1 colspan=1>RRV</td><td rowspan=1 colspan=1>ZeptoMetrix</td><td rowspan=1 colspan=1>0810530CF</td><td rowspan=1 colspan=1>10x LoD2</td></tr></table>

1 Strain tested during LoD verification study 2 Testing at a lower concentration resulted in a detection rate of ${ < } 1 0 0 \%$

Table 27: Microorganisms/Viruses with Predicted Reactivity Based on In silico Analysis   

<table><tr><td colspan="1" rowspan="1">QIAstat-Dx GIPanel 2 Target</td><td colspan="1" rowspan="1">Microorganisms/viruses with predicted reactivity</td></tr><tr><td colspan="1" rowspan="1">Bacteria</td><td></td></tr><tr><td colspan="1" rowspan="1">Campylobacter</td><td colspan="1" rowspan="1">Campylobacter coli*, Campylobacter jejuni, Campylobacter jejuni subsp. jejuni,Campylobacter jejuni subsp. doylei, Campylobacter upsaliensis</td></tr><tr><td colspan="1" rowspan="2">Salmonella</td><td colspan="1" rowspan="1">Salmonella bongori*, Salmonella enterica subsp. salamae II (e.g. serovar 55:k:z39),Salmonella enterica subsp. arizonae IIa (e.g. serovar 63:g:z51), Salmonella enterica subsp.diarizonae IIIb (e.g. serovar 47:l,v:Z), Salmonella enterica subsp. houtenae IV (e.g. serovar43:z4), Salmonella enterica subsp. indica VI.</td></tr><tr><td colspan="1" rowspan="1">Salmonella enterica subsp. enterica (up to 92 different serovars including Agona, Anatum,Bareilly, Choleraesuis, Enteritidis, Heidelberg, Infantis, Kentucky, Montevideo, Newport,Paratyphi A*, Senftenberg, Tennessee, Thompson, Typhi, Typhimurium, Weltevreden*)</td></tr><tr><td colspan="1" rowspan="1">Plesiomonasshigelloides</td><td colspan="1" rowspan="1">Plesiomonas shigelloides (e.g. strains NCTC10360, ATCC 14029T, R4605035)</td></tr><tr><td colspan="1" rowspan="1">Yersiniaenterocolitica</td><td colspan="1" rowspan="1">Yersinia enterocolitica, Yersinia enterocolitica subsp. palearctica, Yersinia enterocoliticasubsp. enterocolitica</td></tr><tr><td colspan="1" rowspan="1">Enteroinvasive E.coli(EIEC)/Shigella</td><td colspan="1" rowspan="1">Enteroinvasive E. coli (EIEC), Escherichia coli sp., Shigella flexneri, Shigella dysenteriae,Shigella boydii, Shigella sonnei.</td></tr><tr><td colspan="1" rowspan="1">Enteropathogenic E.coli (EPEC) β</td><td colspan="1" rowspan="1">Enteropathogenic E. coli (EPEC) (e.g. including serotypes OUT: HND, OUT:H6,OUT:H34, 0UT:H21, 055:H7, 0119:HNM, 0117)</td></tr><tr><td colspan="1" rowspan="1">Enterotoxigenic E.coli (ETEC) &amp;</td><td colspan="1" rowspan="1">Enterotoxigenic E. coli (ETEC) (including H10407 and E24377A strains and serotypes0169:H41, O25:H42, O148:H28, O6:H16) carrier of: Heat-labile enterotoxin gene subtypeLT-I and Heat-stable enterotoxin gene variant Sta, subtypes STp and STh</td></tr><tr><td colspan="1" rowspan="1">Shiga-like toxin-producing E. coli(STEC) - stx1/stx2 β</td><td colspan="1" rowspan="1">Shiga-like toxin-producing E. coli (STEC) including O157:H7 and O157:NM serotype andnon-O157 serotypes (O111:NM, O111:H-, 026:H11, 0145:NM, 0145:H28, 045:H2,</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">O26:H11, ONT:NM, O104:H4, O121:H19, O145:H34, O113:H21, ONT:H-, O128:H2,OUT:HNM, 0124:HNME. coli strains carrier of:stxla, stx1c, stx1d, stx2a, stx2b, stx2c, stx2d, stx2d, stx2e, stx2f, stx2g, stx2h, stx2i, stx2j,stx2k and stx21.Other stx-carrying bacteria: Shigella sonnei, Shigella dysenteriae</td></tr><tr><td colspan="1" rowspan="1">Shiga-like toxin-producing E. coli(STEC) 0157 β</td><td colspan="1" rowspan="1">Escherichia coli O157 including: STEC O157:H7 strains (e.g. EDL933) and E. coli O157:non-H7 groups including non-Shiga-toxigenic E. coli O157 bacteria (e.g. serotype0157:H45)</td></tr><tr><td colspan="1" rowspan="1">Parasites</td><td></td></tr><tr><td colspan="1" rowspan="1">Cryptosporidium#</td><td colspan="1" rowspan="1">Common Cryptosporidium species involved in human disease: C. parvum, C. hominis.Less common Cryptosporidium species involved in human infections: C. meleagridis, C.felis , C. bovis, C. viatorum, C. ubiquitum,C. tyzzeri, C. cuniculus, Cryptosporidium sp.Chipmunk genotype I, C. canis*.Rare or non-human species: Cryptosporidium wrairi</td></tr><tr><td colspan="1" rowspan="1">Cyclosporacayetanensis</td><td colspan="1" rowspan="1">Cyclospora cayetanensis (including strains LG, CY9, NP20 and NP21) *</td></tr><tr><td colspan="1" rowspan="1">Entamoebahistolytica</td><td colspan="1" rowspan="1">Entamoeba histolytica (e.g. strains HM-1: IMSS, EHMfas1, HK-9)*</td></tr><tr><td colspan="1" rowspan="1">Giardia lamblia</td><td colspan="1" rowspan="1">Giardia lamblia (aka Giardia duodenalis, Giardia intestinalis)*</td></tr><tr><td colspan="1" rowspan="1">Viruses</td><td></td></tr><tr><td colspan="1" rowspan="1">Adenovirus</td><td colspan="1" rowspan="1">Human Adenovirus F40/41</td></tr><tr><td colspan="1" rowspan="1">Astrovirus$</td><td colspan="1" rowspan="1">Human Astrovirus (including types 1, 2, 3, 4, 5, 6, 7, 8)</td></tr><tr><td colspan="1" rowspan="2">Norovirus GI/GII</td><td colspan="1" rowspan="1">Norovirus genogroup II genotypes: GII.1, GII.2, GII.3*, GII.4*, GII.5, GII.6, GII.7, GII.8,GII.9, GII.10, GII.12, GII.13, GII.14, GII.16, GII.17, GII.20, GII.21, GII.22, GII.23,GII.24*, GII.25, GII.26, GII.27, GII.NA1 and GII.NA2*</td></tr><tr><td colspan="1" rowspan="1">Norovirus genogroup I genotypes: GI.1, GI2, GI.3*, GI.4*, GI.5, GI.6*, GI.7*, GI.8, GI.9.</td></tr><tr><td colspan="1" rowspan="1">Rotavirus</td><td colspan="1" rowspan="1">Rotavirus A including genotypes: G1P[8]*, G2P[4]*, G3P[8]*, G4P[8]*, G9P[6], G9P[8]*,G12P[6]* and G12P[8]*</td></tr></table>

\* Certain sequences are predicted to be detected with reduced sensitivity due to the presence of a reduced number of mismatches at critical positions of the primer-probe design. &The assay is not predicted to detect bacteria carrier of Heat-labile enterotoxin gene subtype LT-II and/or of Heatstable enterotoxin gene variant Stb e. # The assay is not predicted to detect other Cryptosporidium spp. less involved in human disease: C. andersoni and $C$ . muris. \$ The assay is not predicted to detect Human Astrovirus types MLB1-3 and VA1-5. β Only applicable for samples with Para-Pak C&S collection device

4. Assay Reportable Range: Not applicable   
5. Traceability, Stability, Expected Values (Controls, Calibrators, or Methods):

# Sample Stability

A study was conducted to evaluate the stability of stool specimens collected in Para-Pak C&S or FecalSwab transport media that are stored under claimed storage conditions (i.e., up to four days at room temperature ( $1 5 ^ { \circ } \mathrm { C }$ to $2 5 ^ { \circ } \mathrm { C }$ ) or four days at refrigerated conditions $2 ^ { \circ } \mathrm { C }$ to $8 ^ { \circ } \mathrm { C }$ )). After storage under each storage condition, samples were tested with QIAstat-Dx Gastrointestinal Panel 2. For Para-Pak C&S, at least one strain was evaluated for each reportable analyte (26 strains). For FecalSwab, 11 representative bacterial, viral, and parasite strains were evaluated. Individual samples were prepared by spiking mixtures of up to seven different analytes at $2 . 5 \mathrm { x }$ LoD into human stool matrix in Para-Pak C&S and FecalSwab transport media. The study acceptance criteria to support claims for each storage condition required at least $90 \%$ (e.g., ${ \ge } 9 / 1 0 $ ) of test replicates to generate a positive signal for each spiked pathogen. Sample replicates were tested fresh as well as after each day of storage at room temperature or refrigerated conditions.

Overall, all tested conditions resulted in a detection rate of $2 9 0 \%$ for all strains and storage conditions except for the following:

For Para-Pak C&S, one strain of Cyclospora cayetanensis showed ${ < 9 0 \% }$ detection at several time points for both refrigerated and room temperature storage. These results were found to be due to samples that were prepared at levels below the LoD. Testing of newly prepared samples generated the expected positive results for each test condition evaluated. • For Para-Pak C&S, one strain of Enteroinvasive $E$ . coli (EIEC)/Shigella (EIEC, ATCC 43892) showed ${ < } 9 0 \%$ detection at multiple time points for samples stored at room temperature. Retesting of additional replicates for this strain generated the expected positive results. An additional Shigella strain (Shigella sonnei (ATCC ref. 25931) was also evaluated and generated acceptable results for all test conditions.

Overall results of the sample stability study support the following storage conditions for stool specimens resuspended in Para-Pak C&S and FecalSwab transport media before testing with the QIAstat-Dx Gastrointestinal Panel 2:

Room temperature up to 4 days at $1 5 { - } 2 5 ^ { \circ } \mathrm { C }$ Refrigerated up to 4 days at $2 { - } 8 ^ { \circ } \mathrm { C }$

Assay Controls

# External Controls

External Controls are not provided with the QIAstat-Dx Gastrointestinal Panel 2. The following information regarding external control testing is included in the Instructions for Use:

All external quality control requirements and testing should be performed in accordance with local, state, and federal regulations or accreditation organizations and should follow the user’s laboratory standard quality control procedures.

Blank controls are not applicable to the device because it is a single test disposable cartridge. Regular testing of negative and positive external controls is recommended by the company but controls are not provided with the QIAstat-Dx Gastrointestinal Panel 2. Use transport media as the external Negative Control, and previously characterized positive samples or negative sample spiked with well characterized target organisms as external positive controls.

# Internal Control

The QIAstat-Dx Gastrointestinal Panel Cartridge includes a full process Internal Control, which is titered Schizosaccharomyces pombe. Schizosaccharomyces pombe is a yeast (fungus) that is included in the cartridge in dried form and is rehydrated upon sample loading. This Internal Control material verifies all steps of the analysis process, including sample homogenization, lysis of viral and cellular structures (by means of chemical and mechanical disruption), nucleic acid purification, reverse transcription, and real-time PCR. A passed result for the Internal Control indicates that all processing steps performed by the QIAstat-Dx Gastrointestinal Panel Cartridge were successful. A failed result of the Internal Control does not negate any positive results for detected and identified targets, but it does invalidate all negative results in the analysis. Therefore, the test should be repeated if the Internal Control signal is negative.

# Expected Values:

See Section VII(E) below

# 6. Detection Limit:

To establish the limit of detection (LoD) for each of the QIAstat-Dx Gastrointestinal Panel 2 targets, a total of 36 pathogen strains were evaluated in samples prepared from serial dilutions of culture isolates or positive clinical samples. Samples were prepared in human stool matrix, consisting of a pool of previously tested negative clinical stool specimens resuspended in Para-Pak C&S transport medium.

The LoD was established as the lowest concentration at which $2 9 5 \%$ of replicates are detected. A minimum of 20 replicates were tested to confirm the LoD for each strain. Additional testing of samples with concentrations below the LoD $0 . 3 { \bf x }$ LoD) demonstrated ${ < } 9 5 \%$ detection, confirming the LoD results. LoD results for each microorganism/virus tested are presented in Table 28.

Table 28: LoD Study Results   

<table><tr><td colspan="1" rowspan="1">Target Organism/Analyte</td><td colspan="1" rowspan="1">Strain</td><td colspan="1" rowspan="1">Source/Catalog ID</td><td colspan="1" rowspan="1">LoDconcentration(microbiological (molecular units)1units)</td><td colspan="1" rowspan="1">LoDconcentration(microbiological (molecular units)1</td><td colspan="1" rowspan="1">LoDDetectionrate</td><td colspan="1" rowspan="1">MeanCtValue</td></tr><tr><td colspan="1" rowspan="1">Campylobacter coli</td><td colspan="1" rowspan="1">Campylobactercoli 76-GA2[LMG 21266]</td><td colspan="1" rowspan="1">ATCC 43478</td><td colspan="1" rowspan="1">1.2 CFU/mL</td><td colspan="1" rowspan="1">5802 copies/mL</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">35.6</td></tr><tr><td colspan="1" rowspan="1">Campylobacter coli</td><td colspan="1" rowspan="1">Campylobactercoli CIP 7080</td><td colspan="1" rowspan="1">ATCC 33559</td><td colspan="1" rowspan="1">0.6 CFU/mL</td><td colspan="1" rowspan="1">8941 copies/mL</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">34.5</td></tr><tr><td colspan="1" rowspan="1">Campylobacter jejuni</td><td colspan="1" rowspan="1">Campylobacterjejuni Z086</td><td colspan="1" rowspan="1">ZeptoMetrix801650</td><td colspan="1" rowspan="1">1660 CFU/mL</td><td colspan="1" rowspan="1">14491 copies/mL</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">33.3</td></tr><tr><td colspan="1" rowspan="1">Campylobacter jejuni</td><td colspan="1" rowspan="1">Campylobacterjejuni subsp.jejuni RM3193</td><td colspan="1" rowspan="1">ATCC BAA-1234</td><td colspan="1" rowspan="1">110 CFU/mL</td><td colspan="1" rowspan="1">7210 copies/mL</td><td colspan="1" rowspan="1">19/20</td><td colspan="1" rowspan="1">36.3</td></tr><tr><td colspan="1" rowspan="1">Campylobacterupsaliensis</td><td colspan="1" rowspan="1">CampylobacterupsaliensisNCTC 11541</td><td colspan="1" rowspan="1">ZeptoMetrix801999</td><td colspan="1" rowspan="1">2259 CFU/mL</td><td colspan="1" rowspan="1">56165 copies/mL</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">34.7</td></tr><tr><td colspan="1" rowspan="1">Campylobacterupsaliensis</td><td colspan="1" rowspan="1">CampylobacterupsaliensisRM3195</td><td colspan="1" rowspan="1">ATCCBAA-10599689</td><td colspan="1" rowspan="1">35.0 CFU/mL</td><td colspan="1" rowspan="1">7631 copies/mL</td><td colspan="1" rowspan="1">19/20</td><td colspan="1" rowspan="1">36.1</td></tr><tr><td colspan="1" rowspan="1">Plesiomonasshigelloides</td><td colspan="1" rowspan="1">Bader</td><td colspan="1" rowspan="1">ATCC 14029</td><td colspan="1" rowspan="1">2.7 CFU/mL</td><td colspan="1" rowspan="1">116 copies/mL</td><td colspan="1" rowspan="1">19/20</td><td colspan="1" rowspan="1">34.6</td></tr><tr><td colspan="1" rowspan="1">Plesiomonasshigelloides</td><td colspan="1" rowspan="1">Z130</td><td colspan="1" rowspan="1">ZeptoMetrix801899</td><td colspan="1" rowspan="1">2291 CFU/mL</td><td colspan="1" rowspan="1">481 copies/mL</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">33.7</td></tr><tr><td colspan="1" rowspan="1">Salmonella enterica</td><td colspan="1" rowspan="1">SalmonellaentericaSerovarTyphimuriumZ005</td><td colspan="1" rowspan="1">ZeptoMetrix801437</td><td colspan="1" rowspan="1">4518 CFU/mL</td><td colspan="1" rowspan="1">1441 copies/mL</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">32.5</td></tr><tr><td colspan="1" rowspan="1">Salmonella enterica</td><td colspan="1" rowspan="1">SalmonellaentericaSerovarcholerasuis</td><td colspan="1" rowspan="1">ATCC 13312</td><td colspan="1" rowspan="1">91.6 CFU/mL</td><td colspan="1" rowspan="1">647 copies/mL</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">33.1</td></tr><tr><td colspan="1" rowspan="1">Yersinia enterocolitica</td><td colspan="1" rowspan="1">Z036</td><td colspan="1" rowspan="1">ATCC700822</td><td colspan="1" rowspan="1">120 CFU/mL</td><td colspan="1" rowspan="1">2496 copies/mL</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">34.5</td></tr><tr><td colspan="1" rowspan="1">Yersinia enterocolitica</td><td colspan="1" rowspan="1">subsp.enterocoliticaNTCC 11175,Biotype 4,serotype 3</td><td colspan="1" rowspan="1">ZeptoMetrix801734</td><td colspan="1" rowspan="1">2070 CFU/mL</td><td colspan="1" rowspan="1">719 copies/mL</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">35.0</td></tr><tr><td colspan="1" rowspan="1">Shigella/EnteroinvasiveE. coli (EIEC)</td><td colspan="1" rowspan="1">Escherichiacoli CDC EDL1282, O29:NM</td><td colspan="1" rowspan="1">ATCC43892</td><td colspan="1" rowspan="1">41 CFU/mL</td><td colspan="1" rowspan="1">1431 copies/mL</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">35.4</td></tr><tr><td colspan="1" rowspan="1">Shigella/EnteroinvasiveE. coli (EIEC)</td><td colspan="1" rowspan="1">Shigella sonneiNCDC 1120-66</td><td colspan="1" rowspan="1">ATCC25931</td><td colspan="1" rowspan="1">0.2 CFU/mL</td><td colspan="1" rowspan="1">488 copies/mL</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">34.8</td></tr><tr><td colspan="1" rowspan="1">Target Organism/Analyte</td><td colspan="1" rowspan="1">Strain</td><td colspan="1" rowspan="1">Source/Catalog ID</td><td colspan="1" rowspan="1">LoDconcentration(microbiological(molecular units)1units)</td><td colspan="1" rowspan="1">LoDconcentration(microbiological(molecular units)1</td><td colspan="1" rowspan="1">LoDDetectionrate</td><td colspan="1" rowspan="1">MeanCtValue</td></tr><tr><td colspan="1" rowspan="1">Enteropathogenic E.coli(EPEC)</td><td colspan="1" rowspan="1">Escherichiacoli 0111:NM(EPEC)</td><td colspan="1" rowspan="1">ZeptoMetrix801747</td><td colspan="1" rowspan="1">2581 CFU/mL</td><td colspan="1" rowspan="1">1817 copies/mL</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">35.5</td></tr><tr><td colspan="1" rowspan="1">Enteropathogenic E.coli(EPEC)</td><td colspan="1" rowspan="1">Escherichiacoli 7.1493;EPEC;084:H28</td><td colspan="1" rowspan="1">ZeptoMetrix801938</td><td colspan="1" rowspan="1">1190.0 CFU/mL</td><td colspan="1" rowspan="1">29021 copies/mL</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">35.1</td></tr><tr><td colspan="1" rowspan="1">Enterotoxigenic E. coli(ETEC) lt/st</td><td colspan="1" rowspan="1">Escherichiacoli ETEC;ST+, LT+</td><td colspan="1" rowspan="1">ZeptoMetrix801624</td><td colspan="1" rowspan="1">567 CFU/mL</td><td colspan="1" rowspan="1">855 copies/mL</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">33.7</td></tr><tr><td colspan="1" rowspan="1">Enterotoxigenic E. coli(ETEC) lt/st</td><td colspan="1" rowspan="1">Escherichiacoli H10407,078:H11</td><td colspan="1" rowspan="1">ATCC35401</td><td colspan="1" rowspan="1">10.1 CFU/mL</td><td colspan="1" rowspan="1">367 copies/mL</td><td colspan="1" rowspan="1">19/20</td><td colspan="1" rowspan="1">34.2</td></tr><tr><td colspan="1" rowspan="1">Shiga-like toxin-producingE.coli (STEC)</td><td colspan="1" rowspan="1">Escherichiacoli O26:H4</td><td colspan="1" rowspan="1">ZeptoMetrix801748</td><td colspan="1" rowspan="1">726.8 CFU/mL</td><td colspan="1" rowspan="1">2012 copies/mL</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">36.1</td></tr><tr><td colspan="1" rowspan="1">Shiga-like toxin-producingE.coli (STEC)</td><td colspan="1" rowspan="1">Escherichiacoli O157:H7;EDL933</td><td colspan="1" rowspan="1">ZeptoMetrix801622</td><td colspan="1" rowspan="1">2281 CFU/mL</td><td colspan="1" rowspan="1">1217 copies/mL</td><td colspan="1" rowspan="1">19/20</td><td colspan="1" rowspan="1">35.3</td></tr><tr><td colspan="1" rowspan="1">Cryptosporidiumparvum</td><td colspan="1" rowspan="1">Cryptosporidium parvum Iowa isolate</td><td colspan="1" rowspan="1">WaterborneP102C</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">661 copies/mL</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">28.3</td></tr><tr><td colspan="1" rowspan="1">Cryptosporidiumhominis</td><td colspan="1" rowspan="1">Cryptosporidium hominis</td><td colspan="1" rowspan="1">Clinical SampleUKM 84</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">357 copies/mL</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">32.4</td></tr><tr><td colspan="1" rowspan="1">Cyclosporacayetanensis</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Clinical SampleLAC2825</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">53 copies/mL</td><td colspan="1" rowspan="1">19/20</td><td colspan="1" rowspan="1">35.3</td></tr><tr><td colspan="1" rowspan="1">Cyclosporacayetanensis</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Clinical SampleLAC2827</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">137 copies/mL</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">35.0</td></tr><tr><td colspan="1" rowspan="1">Entamoeba histolytica</td><td colspan="1" rowspan="1">HM-1:IMSS(Mexico City1967)</td><td colspan="1" rowspan="1">ATCC30459</td><td colspan="1" rowspan="1">0.2 cells/mL</td><td colspan="1" rowspan="1">7 copies/mL</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">36.6</td></tr><tr><td colspan="1" rowspan="1">Entamoeba histolytica</td><td colspan="1" rowspan="1">HK-9 (Korea)</td><td colspan="1" rowspan="1">ATCC30015</td><td colspan="1" rowspan="1">0.13 cells/mL</td><td colspan="1" rowspan="1">1 copy/mL</td><td colspan="1" rowspan="1">19/20</td><td colspan="1" rowspan="1">37.0</td></tr><tr><td colspan="1" rowspan="1">Giardia lamblia</td><td colspan="1" rowspan="1">WB (Bethesda)</td><td colspan="1" rowspan="1">ATCC30957</td><td colspan="1" rowspan="1">790 cells/mL</td><td colspan="1" rowspan="1">11850 copies/mL</td><td colspan="1" rowspan="1">19/20</td><td colspan="1" rowspan="1">32.1</td></tr><tr><td colspan="1" rowspan="1">Giardia lamblia</td><td colspan="1" rowspan="1">Portland-1</td><td colspan="1" rowspan="1">ATCC30888</td><td colspan="1" rowspan="1">635.0 cells/mL</td><td colspan="1" rowspan="1">14500 copies/mL</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">31.2</td></tr><tr><td colspan="1" rowspan="1">Adenovirus F40</td><td colspan="1" rowspan="1">Dugan</td><td colspan="1" rowspan="1">ZeptoMetrix0810084CF</td><td colspan="1" rowspan="1">0.1 TCID50/mL</td><td colspan="1" rowspan="1">11726 copies/mL</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">35.7</td></tr><tr><td colspan="1" rowspan="1">Adenovirus F41</td><td colspan="1" rowspan="1">Type 41 (Tak)</td><td colspan="1" rowspan="1">ZeptoMetrix0810085CF</td><td colspan="1" rowspan="1">0.05 TCID50/mL</td><td colspan="1" rowspan="1">979 copies/mL</td><td colspan="1" rowspan="1">19/20</td><td colspan="1" rowspan="1">36.6</td></tr><tr><td colspan="1" rowspan="1">Target Organism/Analyte</td><td colspan="1" rowspan="1">Strain</td><td colspan="1" rowspan="1">Source/Catalog ID</td><td colspan="1" rowspan="1">LoDconcentration(microbiologicalunits)</td><td colspan="1" rowspan="1">LoDconcentration(molecular units)1</td><td colspan="1" rowspan="1">LoDDetectionrate</td><td colspan="1" rowspan="1">MeanCtValue</td></tr><tr><td colspan="1" rowspan="1">Astrovirus</td><td colspan="1" rowspan="1">ERE IID 2371(type 8)</td><td colspan="1" rowspan="1">ZeptoMetrix0810277CF</td><td colspan="1" rowspan="1">11.7 TCID50/mL</td><td colspan="1" rowspan="1">11586371copies/mL</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">32.4</td></tr><tr><td colspan="1" rowspan="1">Astrovirus</td><td colspan="1" rowspan="1">ERE IID 2868(type 4)</td><td colspan="1" rowspan="1">ZeptoMetrix0810276CF</td><td colspan="1" rowspan="1">1.3 TCID50/mL</td><td colspan="1" rowspan="1">52184 copies/mL</td><td colspan="1" rowspan="1">19/20</td><td colspan="1" rowspan="1">32.4</td></tr><tr><td colspan="1" rowspan="1">Norovirus GI</td><td colspan="1" rowspan="1">GI.1(recombinant)</td><td colspan="1" rowspan="1">ZeptoMetrix0810086CF</td><td colspan="1" rowspan="1">891.1TCID50/mL</td><td colspan="1" rowspan="1">24629 copies/mL</td><td colspan="1" rowspan="1">19/20</td><td colspan="1" rowspan="1">33.6</td></tr><tr><td colspan="1" rowspan="1">Norovirus GII</td><td colspan="1" rowspan="1">GII.4(recombinant)</td><td colspan="1" rowspan="1">ZeptoMetrix0810087CF</td><td colspan="1" rowspan="1">10.5 TCID50/mL</td><td colspan="1" rowspan="1">8998 copies/mL</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">33.6</td></tr><tr><td colspan="1" rowspan="1">Rotavirus A</td><td colspan="1" rowspan="1">WA</td><td colspan="1" rowspan="1">ZeptoMetrix0810041CF</td><td colspan="1" rowspan="1">14.1 TCID50/mL</td><td colspan="1" rowspan="1">5201 copies/mL</td><td colspan="1" rowspan="1">19/20</td><td colspan="1" rowspan="1">36.4</td></tr><tr><td colspan="1" rowspan="1">Rotavirus A</td><td colspan="1" rowspan="1">69M</td><td colspan="1" rowspan="1">ZeptoMetrix0810280CF</td><td colspan="1" rowspan="1">436TCID50/mL</td><td colspan="1" rowspan="1">5787 copies/mL</td><td colspan="1" rowspan="1">19/20</td><td colspan="1" rowspan="1">36.9</td></tr></table>

1 Pathogen titer from clinical samples was determined by molecular units (copies $/ \mathrm { m L }$ ) using in-house developed and validated qPCR assays. Pathogen titer of clinical samples is not provided in microbiological units.

# 7. Carryover/Cross-Contamination Study:

A study was performed to evaluate the potential for carryover or cross-contamination between consecutive runs when using the QIAstat-Dx Gastrointestinal Panel 2 on the QIAstat-Dx Analyzer 1.0. Samples were prepared in stool matrix in Para-Pak C&S transport medium. Testing included alternating high-positive samples $\mathrm { 1 . 0 E ^ { + } 0 6 C F U / m L }$ for bacteria, $1 . 0 \mathrm { E } { + 0 5 }$ $\mathrm { T C I D } _ { 5 0 }$ for viruses, or $1 . 0 \mathrm { E } { + 0 5 }$ organisms $\mathrm { \Delta } ^ { \prime } \mathrm { m L }$ for parasites) and negative samples. A total of 31 positive samples and 31 negative samples were tested over five days using two QIAstat-Dx Analyzer 1.0 instruments. Results from testing generated the expected results for all positive and negative samples, demonstrating no carryover or cross-contamination for the QIAstat-Dx Gastrointestinal Panel 2 when run on the QIAstat-Dx analyzer. Analyte mixes used to prepare high positive samples are presented in Table 29.

Table 29: Carryover Study, Analytes Tested   

<table><tr><td colspan="1" rowspan="1">Sample</td><td colspan="1" rowspan="1">Pathogen/Target</td><td colspan="1" rowspan="1">Concentration tested</td></tr><tr><td colspan="1" rowspan="3">Positive Mix 1</td><td colspan="1" rowspan="1">Norovirus GI</td><td colspan="1" rowspan="1">1.0E+05 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Plesiomonas shigelloides</td><td colspan="1" rowspan="1">1.0E+06 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Shiga-like toxin-producing E. coli (STEC)(stx1+)</td><td colspan="1" rowspan="1">1.0E+06 CFU/mL</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Cryptosporidium parvum</td><td colspan="1" rowspan="1">1.0E+05 oocysts/mL</td></tr><tr><td colspan="1" rowspan="1">Positive Mix 2</td><td colspan="1" rowspan="1">Campylobacter jejuni</td><td colspan="1" rowspan="1">1.0E+06 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Negative mix</td><td colspan="1" rowspan="1">None</td><td colspan="1" rowspan="1">N/A</td></tr></table>

8. Transport Media Equivalency Study:

A transport media equivalency study was conducted to assess potential performance differences between Para-Pak C&S and FecalSwab transport devices. Specimens tested in the study were prepared using raw clinical stool specimens that were known positive for representative analytes detected by the QIAstat-Dx Gastrointestinal Panel 2. Each positive raw stool was inoculated into both Para-Pak C&S and FecalSwab transport media in accordance with the instuctions for use for each transport device. Positive percent agreement (PPA) and negative percent agreement (NPA) were calculated based on comparison to the expected result. Results from testing demonstrated equivalent performance between the two transport media, when compared to the expected positive and negative results for each analyte evaluated. Results for positive and negative agreement are presented in Tables 30 and 31 respectively.

Table 30: Transport Media Equivalency, Clinical Specimens - PPA   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=3>Para-Pak C&amp;S</td><td rowspan=1 colspan=3>FecalSwab</td></tr><tr><td rowspan=1 colspan=1>Target</td><td rowspan=1 colspan=1>Detected/Total</td><td rowspan=1 colspan=1>PPA</td><td rowspan=1 colspan=1>Confidence Interval95%</td><td rowspan=1 colspan=1>DetectedTotal</td><td rowspan=1 colspan=1>PPA</td><td rowspan=1 colspan=1>Confidence Interval95%</td></tr><tr><td rowspan=1 colspan=1>Adenovirus F40/F41*</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>n/a</td></tr><tr><td rowspan=1 colspan=1>Cryptosporidium*</td><td rowspan=1 colspan=1>0/1</td><td rowspan=1 colspan=1>0.00%</td><td rowspan=1 colspan=1>0.00%- 97.50%</td><td rowspan=1 colspan=1>1/1</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>2.50%-100%</td></tr><tr><td rowspan=1 colspan=1>Enteropathogenic E. coli(EPEC)</td><td rowspan=1 colspan=1>58 / 61</td><td rowspan=1 colspan=1>95.08%</td><td rowspan=1 colspan=1>86.29%-98.97%</td><td rowspan=1 colspan=1>57 / 61</td><td rowspan=1 colspan=1>93.44%</td><td rowspan=1 colspan=1>84.05%-98.18%</td></tr><tr><td rowspan=1 colspan=1>Giardia lamblia</td><td rowspan=1 colspan=1>47 / 48</td><td rowspan=1 colspan=1>97.92%</td><td rowspan=1 colspan=1>88.93%-99.95%</td><td rowspan=1 colspan=1>47 / 48</td><td rowspan=1 colspan=1>97.92%</td><td rowspan=1 colspan=1>88.93%-99.95%</td></tr><tr><td rowspan=1 colspan=1>Norovirus GI/GII*</td><td rowspan=1 colspan=1>8/8</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>63.06%-100%</td><td rowspan=1 colspan=1>7/8</td><td rowspan=1 colspan=1>87.50%</td><td rowspan=1 colspan=1>47.35%-99.68%</td></tr></table>

\*Low sample size; therefore, equivalence for these analytes was based on the testing of contrived specimens

Table 31: Transport Media Equivalency, Clinical Specimens - NPA   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=3>Para-Pak C&amp;S</td><td rowspan=1 colspan=3>FecalSwab</td></tr><tr><td rowspan=1 colspan=1>Target</td><td rowspan=1 colspan=1>#NotDetected</td><td rowspan=1 colspan=1>NPA</td><td rowspan=1 colspan=1>ConfidenceInterval</td><td rowspan=1 colspan=1>#NotDetected</td><td rowspan=1 colspan=1>NPA</td><td rowspan=1 colspan=1>ConfidenceInterval</td></tr><tr><td rowspan=1 colspan=1>Adenovirus F40/F41</td><td rowspan=1 colspan=1>174/174</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>97.90% 100%</td><td rowspan=1 colspan=1>174/174</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>97.90%-100%</td></tr><tr><td rowspan=1 colspan=1>Cryptosporidium</td><td rowspan=1 colspan=1>171/173</td><td rowspan=1 colspan=1>98.84%</td><td rowspan=1 colspan=1>95.89%-99.86%</td><td rowspan=1 colspan=1>170 / 173</td><td rowspan=1 colspan=1>98.27%</td><td rowspan=1 colspan=1>95.02%-99.64%</td></tr><tr><td rowspan=1 colspan=1>Enteropathogenic E. coli(EPEC)</td><td rowspan=1 colspan=1>52 / 58</td><td rowspan=1 colspan=1>89.66%</td><td rowspan=1 colspan=1>78.83%-96.11%</td><td rowspan=1 colspan=1>49 / 58</td><td rowspan=1 colspan=1>84.48%</td><td rowspan=1 colspan=1>72.58%-92.65%</td></tr><tr><td rowspan=1 colspan=1>Giardia lamblia</td><td rowspan=1 colspan=1>123 / 127</td><td rowspan=1 colspan=1>96.85%</td><td rowspan=1 colspan=1>92.13%-99.14%</td><td rowspan=1 colspan=1>124 / 127</td><td rowspan=1 colspan=1>97.64%</td><td rowspan=1 colspan=1>93.25%-99.51%</td></tr><tr><td rowspan=1 colspan=1>Norovirus GI/GII</td><td rowspan=1 colspan=1>164 / 164</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>97.78%-100%</td><td rowspan=1 colspan=1>164 / 164</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>97.78%-100%</td></tr></table>

Additonal testing was conducted using individual negative stool matrices spiked with selected target analytes at concentrations of $2 \mathbf { x }$ , 5x and $1 0 \mathrm { x }$ LoD. Positive stool samples were inoculated into Para-Pak C&S and FecalSwab transport devices. Testing demonstrated equivalent performance for all analytes for contrived clinical specimens in Para-Pak C&S and FecalSwab transport devices. Results from testing demonstrated equivalent analytical performance between Para-Pak C&S and FecalSwab transport media. Results for positive and negative agreement for testing of contrived specimens are presented in Tables 32 and 33 respectively.

Table 32: Transport Media Equivalency, Contrived Specimens - PPA   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=4>Para-Pak C&amp;S</td><td rowspan=1 colspan=4>FecalSwab</td></tr><tr><td rowspan=1 colspan=1>Target</td><td rowspan=1 colspan=1>Detected/Total</td><td rowspan=1 colspan=1>PPA</td><td rowspan=1 colspan=1>Lower</td><td rowspan=1 colspan=1>Upper</td><td rowspan=1 colspan=1>Detected/Total</td><td rowspan=1 colspan=1>PPA</td><td rowspan=1 colspan=1>Lower</td><td rowspan=1 colspan=1>Upper</td></tr><tr><td rowspan=1 colspan=1>Adenovirus F40/F41</td><td rowspan=1 colspan=1>35 / 36</td><td rowspan=1 colspan=1>97.22%</td><td rowspan=1 colspan=1>85.47%</td><td rowspan=1 colspan=1>99.93%</td><td rowspan=1 colspan=1>35 / 36</td><td rowspan=1 colspan=1>97.22%</td><td rowspan=1 colspan=1>85.47%</td><td rowspan=1 colspan=1>99.93%</td></tr><tr><td rowspan=1 colspan=1>Norovirus GI/GII</td><td rowspan=1 colspan=1>36 / 36</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>90.26%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>36 / 36</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>90.26%</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Cryptosporidium</td><td rowspan=1 colspan=1>36 / 36</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>90.26%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>36 / 36</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>90.26%</td><td rowspan=1 colspan=1>100%</td></tr></table>

Table 33: Transport Media Equivalency, Contrived Specimens - NPA   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=4>Para-Pak C&amp;S</td><td rowspan=1 colspan=4>FecalSwab</td></tr><tr><td rowspan=1 colspan=1>Target</td><td rowspan=1 colspan=1>NotDetected</td><td rowspan=1 colspan=1>NPA</td><td rowspan=1 colspan=1>Lower</td><td rowspan=1 colspan=1>Upper</td><td rowspan=1 colspan=1>NotDetected</td><td rowspan=1 colspan=1>NPA</td><td rowspan=1 colspan=1>Lower</td><td rowspan=1 colspan=1>Upper</td></tr><tr><td rowspan=1 colspan=1>Adenovirus F40/F41</td><td rowspan=1 colspan=1>167 / 168</td><td rowspan=1 colspan=1>99.40%</td><td rowspan=1 colspan=1>96.73%</td><td rowspan=1 colspan=1>99.98%</td><td rowspan=1 colspan=1>168 / 168</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>97.83%</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Norovirus GI/GII</td><td rowspan=1 colspan=1>168 / 168</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>97.83%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>168 / 168</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>97.83%</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Cryptosporidium</td><td rowspan=1 colspan=1>168 / 168</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>97.83%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>167 / 168</td><td rowspan=1 colspan=1>99.40%</td><td rowspan=1 colspan=1>96.73%</td><td rowspan=1 colspan=1>99.98%</td></tr></table>

9. Fresh Versus Frozen Study: Para-Pak C&S Medium

To support inclusion of frozen archived Para-Pak C&S clinical specimens in the clinical study, an equivalency study was conducted to demonstrate equivalent performance between fresh and frozen specimens. Samples for the study were prepared by spiking different pathogen strains at low concentrations (i.e., $2 \mathrm { x } \mathrm { L o D } )$ into stool sample matrix in Para-Pak C&S transport medium. Fresh (non-stored) samples were tested immediately after preparation. Frozen sample aliquots were stored at -70 to $- 8 0 \mathrm { { ^ \circ C } }$ or $- 1 5$ to $- 2 5 ^ { \circ } \mathrm { C }$ ) and for at least 46 days and subjected to two freeze and thaw cycles prior testing. Results from the study demonstrated equivalent performance between fresh and frozen stool samples in ParaPak C&S medium. Results are presented in Table 34.

Table 34: Fresh/Frozen Study, Para-Pak C&S, 2x LoD   

<table><tr><td colspan="1" rowspan="1">Organismype</td><td colspan="2" rowspan="1">Pathogen</td><td colspan="1" rowspan="1">Timepoint</td><td colspan="1" rowspan="1">Storage time point</td><td colspan="1" rowspan="1">Detectionrate</td></tr><tr><td colspan="1" rowspan="27">Bacteria</td><td colspan="2" rowspan="3">Campylobacter jejuni</td><td colspan="1" rowspan="1">TPO (0 days)</td><td colspan="1" rowspan="1">Fresh</td><td colspan="1" rowspan="1">40/40</td></tr><tr><td colspan="1" rowspan="1">TP1 (57 days)</td><td colspan="1" rowspan="1">Frozen -15 to -25°C</td><td colspan="1" rowspan="1">40/40</td></tr><tr><td colspan="1" rowspan="1">TP1 (48 days)</td><td colspan="1" rowspan="1">Frozen -70 to -80°</td><td colspan="1" rowspan="1">42/42</td></tr><tr><td colspan="2" rowspan="3">Clostridium difficile</td><td colspan="1" rowspan="1">TPO (0 days)</td><td colspan="1" rowspan="1">Fresh</td><td colspan="1" rowspan="1">40/40</td></tr><tr><td colspan="1" rowspan="1">TP1 (57 days)</td><td colspan="1" rowspan="1">Frozen -15 to -25°C</td><td colspan="1" rowspan="1">40/40</td></tr><tr><td colspan="1" rowspan="1">TP1 (48 days)</td><td colspan="1" rowspan="1">Frozen -70 to -80°</td><td colspan="1" rowspan="1">42/42</td></tr><tr><td colspan="2" rowspan="3">Enteropathogenic E. coli(EPEC)</td><td colspan="1" rowspan="1">TP0 (0 days)</td><td colspan="1" rowspan="1">Fresh</td><td colspan="1" rowspan="1">40/40</td></tr><tr><td colspan="1" rowspan="1">TP1 (57 days)</td><td colspan="1" rowspan="1">Frozen -15 to -25°C</td><td colspan="1" rowspan="1">40/40</td></tr><tr><td colspan="1" rowspan="1">TP1 (48 days)</td><td colspan="1" rowspan="1">Frozen -70 to -80°</td><td colspan="1" rowspan="1">40/40</td></tr><tr><td colspan="1" rowspan="9">Shiga-liketoxin-producingE. coli (STEC)0157</td><td colspan="1" rowspan="3">0157</td><td colspan="1" rowspan="1">TPO (0 days)</td><td colspan="1" rowspan="1">Fresh</td><td colspan="1" rowspan="1">40/40</td></tr><tr><td colspan="1" rowspan="1">TP1 (57 days)</td><td colspan="1" rowspan="1">Frozen -15 to -25°C</td><td colspan="1" rowspan="1">40/40</td></tr><tr><td colspan="1" rowspan="1">TP1 (48 days)</td><td colspan="1" rowspan="1">Frozen -70 to -80°</td><td colspan="1" rowspan="1">42/42</td></tr><tr><td colspan="1" rowspan="3">Stx1</td><td colspan="1" rowspan="1">TPO (0 days)</td><td colspan="1" rowspan="1">Fresh</td><td colspan="1" rowspan="1">40/40</td></tr><tr><td colspan="1" rowspan="1">TP1 (57 days)</td><td colspan="1" rowspan="1">Frozen -15 to -25°</td><td colspan="1" rowspan="1">40/40</td></tr><tr><td colspan="1" rowspan="1">TP1 (48 days)</td><td colspan="1" rowspan="1">Frozen -70 to -80</td><td colspan="1" rowspan="1">42/42</td></tr><tr><td colspan="1" rowspan="3">Stx2</td><td colspan="1" rowspan="1">TPO (0 days)</td><td colspan="1" rowspan="1">Fresh</td><td colspan="1" rowspan="1">40/40</td></tr><tr><td colspan="1" rowspan="1">TP1 (57 days)</td><td colspan="1" rowspan="1">Frozen -15 to -25°C</td><td colspan="1" rowspan="1">40/40</td></tr><tr><td colspan="1" rowspan="1">TP1 (48 days)</td><td colspan="1" rowspan="1">Frozen -70 to -80°</td><td colspan="1" rowspan="1">42/42</td></tr><tr><td colspan="2" rowspan="3">Salmonella enterica</td><td colspan="1" rowspan="1">TPO (0 days)</td><td colspan="1" rowspan="1">Fresh</td><td colspan="1" rowspan="1">40/40</td></tr><tr><td colspan="1" rowspan="1">TP1 (57 days)</td><td colspan="1" rowspan="1">Frozen -15 to -25°C</td><td colspan="1" rowspan="1">40/40</td></tr><tr><td colspan="1" rowspan="1">TP1 (48 days)</td><td colspan="1" rowspan="1">Frozen -70 to -80°</td><td colspan="1" rowspan="1">41/41</td></tr><tr><td colspan="2" rowspan="3">Vibrio parahaemolyticus</td><td colspan="1" rowspan="1">TPO (0 days)</td><td colspan="1" rowspan="1">Fresh</td><td colspan="1" rowspan="1">40/40</td></tr><tr><td colspan="1" rowspan="1">TP1 (57 days)</td><td colspan="1" rowspan="1">Frozen -15 to -25°C</td><td colspan="1" rowspan="1">40/40</td></tr><tr><td colspan="1" rowspan="1">TP1 (48 days)</td><td colspan="1" rowspan="1">Frozen -70 to -80°</td><td colspan="1" rowspan="1">41/41</td></tr><tr><td colspan="2" rowspan="3">Yersinia enterocolitica</td><td colspan="1" rowspan="1">TPO (0 days)</td><td colspan="1" rowspan="1">Fresh</td><td colspan="1" rowspan="1">40/40</td></tr><tr><td colspan="1" rowspan="1">TP1 (57 days)</td><td colspan="1" rowspan="1">Frozen -15 to -25°</td><td colspan="1" rowspan="1">40/40</td></tr><tr><td colspan="1" rowspan="1">TP1 (48 days)</td><td colspan="1" rowspan="1">Frozen -70 to -80°</td><td colspan="1" rowspan="1">41/41</td></tr><tr><td colspan="1" rowspan="4">Parasites</td><td colspan="2" rowspan="3">Entamoeba histolytica</td><td colspan="1" rowspan="1">TP0 (0 days)</td><td colspan="1" rowspan="1">Fresh</td><td colspan="1" rowspan="1">41/41</td></tr><tr><td colspan="1" rowspan="1">TP1 (57 days)</td><td colspan="1" rowspan="1">Frozen -15 to -25°C</td><td colspan="1" rowspan="1">41/41</td></tr><tr><td colspan="1" rowspan="1">TP1 (48 days)</td><td colspan="1" rowspan="1">Frozen -70 to -80</td><td colspan="1" rowspan="1">38/40</td></tr><tr><td colspan="2" rowspan="1">Giardia lamblia</td><td colspan="1" rowspan="1">TPO (0 days)</td><td colspan="1" rowspan="1">Fresh</td><td colspan="1" rowspan="1">40/40</td></tr><tr><td colspan="1" rowspan="1">OrganismType</td><td colspan="1" rowspan="1">Pathogen</td><td colspan="1" rowspan="1">Timepoint</td><td colspan="1" rowspan="1">Storage time point</td><td colspan="1" rowspan="1">Detectionrate</td></tr><tr><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="1">TP1 (57 days)</td><td colspan="1" rowspan="1">Frozen -15 to -25°C</td><td colspan="1" rowspan="1">40/40</td></tr><tr><td colspan="1" rowspan="1">TP1 (48 days)</td><td colspan="1" rowspan="1">Frozen -70 to -80°</td><td colspan="1" rowspan="1">42/42</td></tr><tr><td colspan="1" rowspan="9">Viruses</td><td colspan="1" rowspan="3">Adenovirus F40</td><td colspan="1" rowspan="1">TPO (0 days)</td><td colspan="1" rowspan="1">Fresh</td><td colspan="1" rowspan="1">40/40</td></tr><tr><td colspan="1" rowspan="1">TP1 (57 days)</td><td colspan="1" rowspan="1">Frozen -15 to -25°C</td><td colspan="1" rowspan="1">40/40</td></tr><tr><td colspan="1" rowspan="1">TP1 (48 days)</td><td colspan="1" rowspan="1">Frozen -70 to -80°</td><td colspan="1" rowspan="1">41/41</td></tr><tr><td colspan="1" rowspan="3">Norovirus GI</td><td colspan="1" rowspan="1">TPO (0 days)</td><td colspan="1" rowspan="1">Fresh</td><td colspan="1" rowspan="1">40/40</td></tr><tr><td colspan="1" rowspan="1">TP1 (57 days)</td><td colspan="1" rowspan="1">Frozen -15 to -25°C</td><td colspan="1" rowspan="1">40/40</td></tr><tr><td colspan="1" rowspan="1">TP1 (48 days)</td><td colspan="1" rowspan="1">Frozen -70 to -80°</td><td colspan="1" rowspan="1">41/41</td></tr><tr><td colspan="1" rowspan="3">Rotavirus A</td><td colspan="1" rowspan="1">TPO (0 days)</td><td colspan="1" rowspan="1">Fresh</td><td colspan="1" rowspan="1">41/41</td></tr><tr><td colspan="1" rowspan="1">TP1 (57 days)</td><td colspan="1" rowspan="1">Frozen -15 to -25°C</td><td colspan="1" rowspan="1">40/41</td></tr><tr><td colspan="1" rowspan="1">TP1 (48 days)</td><td colspan="1" rowspan="1">Frozen -70 to -80°</td><td colspan="1" rowspan="1">40/40</td></tr></table>

# FecalSwab Medium

To support inclusion of frozen archived FecalSwab clinical specimens in the clinical study, an equivalency study was conducted to demonstrate equivalent performance between fresh and frozen specimens. Samples for the study were prepared by spiking different pathogen strains into stool sample matrix in FecalSwab transport medium. Fresh (non-stored) samples were tested immediately after preparation. Frozen sample aliquots were stored at $- 7 0$ to - $8 0 \mathrm { { ^ \circ C } }$ for at least 46 days and subjected to two freeze/thaw cycles prior testing. Results from the study demonstrated equivalent performance between fresh and frozen samples prepared in FecalSwab transport medium. Results are presented in Table 35.

Table 35: Fresh/Frozen Study, FecalSwab, 2x LoD   

<table><tr><td colspan="1" rowspan="1">Pathogen</td><td colspan="1" rowspan="1">TimePoint</td><td colspan="1" rowspan="1">Sampletype</td><td colspan="1" rowspan="1">Conc</td><td colspan="1" rowspan="1">DetectionRate</td><td colspan="1" rowspan="1">Detection RateOverall</td><td colspan="1" rowspan="1">%Agreement</td></tr><tr><td colspan="1" rowspan="4">Campylobacter jejuni(ZeptoMetrix,0801650)</td><td colspan="1" rowspan="2">TPO (0days)</td><td colspan="1" rowspan="2">Fresh</td><td colspan="1" rowspan="1">2x LoD</td><td colspan="1" rowspan="1">41/41</td><td colspan="1" rowspan="2">71/71</td><td colspan="1" rowspan="2">100</td></tr><tr><td colspan="1" rowspan="1">5x LoD</td><td colspan="1" rowspan="1">30/30</td></tr><tr><td colspan="1" rowspan="1">TP1 (47days)</td><td colspan="1" rowspan="1">Frozen</td><td colspan="1" rowspan="1">2x LoD</td><td colspan="1" rowspan="1">40/40</td><td colspan="1" rowspan="2">70/70</td><td colspan="1" rowspan="2">100</td></tr><tr><td colspan="1" rowspan="1">TP1 (46days</td><td colspan="1" rowspan="1">-80</td><td colspan="1" rowspan="1">5x LoD</td><td colspan="1" rowspan="1">30/30</td></tr><tr><td colspan="1" rowspan="4">Salmonella enterica (ZeptoMetrix,801437)</td><td colspan="1" rowspan="2">TPO (0days)</td><td colspan="1" rowspan="2">Fresh</td><td colspan="1" rowspan="1">2x LoD</td><td colspan="1" rowspan="1">40/40</td><td colspan="1" rowspan="2">70/70</td><td colspan="1" rowspan="2">100</td></tr><tr><td colspan="1" rowspan="1">5x LoD</td><td colspan="1" rowspan="1">30/30</td></tr><tr><td colspan="1" rowspan="2">TP1 (47days)</td><td colspan="1" rowspan="2">Frozen-80</td><td colspan="1" rowspan="1">2x LoD</td><td colspan="1" rowspan="1">41/41</td><td colspan="1" rowspan="2">72/72</td><td colspan="1" rowspan="2">100</td></tr><tr><td colspan="1" rowspan="1">5x LoD</td><td colspan="1" rowspan="1">31/31</td></tr><tr><td colspan="1" rowspan="4">Yersinia enterocolitica(ZeptoMetrix,0801734)</td><td colspan="1" rowspan="2">TPO (0days)</td><td colspan="1" rowspan="2">Fresh</td><td colspan="1" rowspan="1">2x LoD</td><td colspan="1" rowspan="1">40/40</td><td colspan="1" rowspan="2">70/70</td><td colspan="1" rowspan="2">100</td></tr><tr><td colspan="1" rowspan="1">5x LoD</td><td colspan="1" rowspan="1">30/30</td></tr><tr><td colspan="1" rowspan="2">TP1 (47days)</td><td colspan="1" rowspan="2">Frozen-80°</td><td colspan="1" rowspan="1">2x LoD</td><td colspan="1" rowspan="1">41/41</td><td colspan="1" rowspan="2">72/72</td><td colspan="1" rowspan="2">100</td></tr><tr><td colspan="1" rowspan="1">5x LoD</td><td colspan="1" rowspan="1">31/31</td></tr><tr><td colspan="1" rowspan="4">Enteropathogenic E. coli (EPEC)(ZeptoMetrix, 0801747)</td><td colspan="1" rowspan="2">TPO (0days)</td><td colspan="1" rowspan="2">Fresh</td><td colspan="1" rowspan="1">2x LoD</td><td colspan="1" rowspan="1">40/40</td><td colspan="1" rowspan="2">70/70</td><td colspan="1" rowspan="2">100</td></tr><tr><td colspan="1" rowspan="1">5x LoD</td><td colspan="1" rowspan="1">30/30</td></tr><tr><td colspan="1" rowspan="2">TP1 (47days)</td><td colspan="1" rowspan="2">Frozen-80C</td><td colspan="1" rowspan="1">2x LoD</td><td colspan="1" rowspan="1">41/41</td><td colspan="1" rowspan="2">72/72</td><td colspan="1" rowspan="2">100</td></tr><tr><td colspan="1" rowspan="1">5x LoD</td><td colspan="1" rowspan="1">31/31</td></tr><tr><td colspan="1" rowspan="4">Shiga-like toxin-producing E.coli (STEC) O157:H7 -0157 (ZeptoMetrix,0801622)</td><td colspan="1" rowspan="2">TPO (0days</td><td colspan="1" rowspan="2">Fresh</td><td colspan="1" rowspan="1">2x LoD</td><td colspan="1" rowspan="1">41/41</td><td colspan="1" rowspan="2">71/71</td><td colspan="1" rowspan="2">100</td></tr><tr><td colspan="1" rowspan="1">5x LoD</td><td colspan="1" rowspan="1">30/30</td></tr><tr><td colspan="1" rowspan="1">TP1 (47days)</td><td colspan="1" rowspan="2">Frozen-80C</td><td colspan="1" rowspan="1">2x LoD</td><td colspan="1" rowspan="1">40/40</td><td colspan="1" rowspan="2">70/70</td><td colspan="1" rowspan="2">100</td></tr><tr><td colspan="1" rowspan="1">TP1 (46days)</td><td colspan="1" rowspan="1">5x LoD</td><td colspan="1" rowspan="1">30/30</td></tr><tr><td colspan="1" rowspan="4">Shiga-like toxin-producingE. coli (STEC) 0157:H7-stx1/stx2 (ZeptoMetrix,080162)</td><td colspan="1" rowspan="2">TPO (0days</td><td colspan="1" rowspan="2">Fresh</td><td colspan="1" rowspan="1">2x LoD</td><td colspan="1" rowspan="1">41/41</td><td colspan="1" rowspan="2">71/71</td><td colspan="1" rowspan="2">100</td></tr><tr><td colspan="1" rowspan="1">5x LoD</td><td colspan="1" rowspan="1">30/30</td></tr><tr><td colspan="1" rowspan="1">TP1 (47days</td><td colspan="1" rowspan="2">Frozen-80°</td><td colspan="1" rowspan="1">2x LoD</td><td colspan="1" rowspan="1">40/40</td><td colspan="1" rowspan="2">70/70</td><td colspan="1" rowspan="2">100</td></tr><tr><td colspan="1" rowspan="1">TP1 (46days)</td><td colspan="1" rowspan="1">5x LoD</td><td colspan="1" rowspan="1">30/30</td></tr><tr><td colspan="1" rowspan="4">Entamoeba histolytica(ATCC, 30459)</td><td colspan="1" rowspan="2">TPO (0dys</td><td colspan="1" rowspan="2">Fresh</td><td colspan="1" rowspan="1">2x LoD</td><td colspan="1" rowspan="1">41/41</td><td colspan="1" rowspan="2">71/71</td><td colspan="1" rowspan="2">100</td></tr><tr><td colspan="1" rowspan="1">5x LoD</td><td colspan="1" rowspan="1">30/30</td></tr><tr><td colspan="1" rowspan="1">TP1 (47days)</td><td colspan="1" rowspan="2">Frozen-80C</td><td colspan="1" rowspan="1">2x LoD</td><td colspan="1" rowspan="1">40/40</td><td colspan="1" rowspan="2">70/70</td><td colspan="1" rowspan="2">100</td></tr><tr><td colspan="1" rowspan="1">TP1 (46days)</td><td colspan="1" rowspan="1">5x LoD</td><td colspan="1" rowspan="1">30/30</td></tr><tr><td colspan="1" rowspan="4">Giardia lamblia (ATCC,30957</td><td colspan="1" rowspan="2">TPO (0days</td><td colspan="1" rowspan="2">Fresh</td><td colspan="1" rowspan="1">2x LoD</td><td colspan="1" rowspan="1">40/40</td><td colspan="1" rowspan="2">70/70</td><td colspan="1" rowspan="2">100</td></tr><tr><td colspan="1" rowspan="1">5x LoD</td><td colspan="1" rowspan="1">30/30</td></tr><tr><td colspan="1" rowspan="2">TP1 (46days)</td><td colspan="1" rowspan="2">Frozen-80°</td><td colspan="1" rowspan="1">2x LoD</td><td colspan="1" rowspan="1">40/40</td><td colspan="1" rowspan="2">70/70</td><td colspan="1" rowspan="2">100</td></tr><tr><td colspan="1" rowspan="1">5x LoD</td><td colspan="1" rowspan="1">30/30</td></tr><tr><td colspan="1" rowspan="4">Adenovirus F40(ZeptoMetrix,0810085CF)</td><td colspan="1" rowspan="2">TPO (0days</td><td colspan="1" rowspan="2">Fresh</td><td colspan="1" rowspan="1">2x LoD</td><td colspan="1" rowspan="1">40/40</td><td colspan="1" rowspan="2">70/70</td><td colspan="1" rowspan="2">100</td></tr><tr><td colspan="1" rowspan="1">5x LoD</td><td colspan="1" rowspan="1">30/30</td></tr><tr><td colspan="1" rowspan="2">TP1 (47das</td><td colspan="1" rowspan="2">Frozen-80C</td><td colspan="1" rowspan="1">2x LoD</td><td colspan="1" rowspan="1">41/41</td><td colspan="1" rowspan="2">72/72</td><td colspan="1" rowspan="2">100</td></tr><tr><td colspan="1" rowspan="1">5x LoD</td><td colspan="1" rowspan="1">31/31</td></tr><tr><td colspan="1" rowspan="4">Norovirus GII(ZeptoMetrix, 0810087CF)</td><td colspan="1" rowspan="2">TPO (0days)</td><td colspan="1" rowspan="2">Fresh</td><td colspan="1" rowspan="1">2x LoD</td><td colspan="1" rowspan="1">40/40</td><td colspan="1" rowspan="2">70/70</td><td colspan="1" rowspan="2">100</td></tr><tr><td colspan="1" rowspan="1">5x LoD</td><td colspan="1" rowspan="1">30/30</td></tr><tr><td colspan="1" rowspan="2">TP1 (47days</td><td colspan="1" rowspan="2">Frozen-80</td><td colspan="1" rowspan="1">2x LoD</td><td colspan="1" rowspan="1">41/41</td><td colspan="1" rowspan="2">72/72</td><td colspan="1" rowspan="2">100</td></tr><tr><td colspan="1" rowspan="1">5x LoD</td><td colspan="1" rowspan="1">31/31</td></tr><tr><td colspan="1" rowspan="4">Rotavirus A (ZeptoMetrix,0810280CF)</td><td colspan="1" rowspan="2">TPO (0days</td><td colspan="1" rowspan="2">Fresh</td><td colspan="1" rowspan="1">2x LoD</td><td colspan="1" rowspan="1">40/40</td><td colspan="1" rowspan="2">70/70</td><td colspan="1" rowspan="2">100</td></tr><tr><td colspan="1" rowspan="1">5x LoD</td><td colspan="1" rowspan="1">30/30</td></tr><tr><td colspan="1" rowspan="2">TP1 (47days)</td><td colspan="1" rowspan="2">Frozen-80°</td><td colspan="1" rowspan="1">2x LoD</td><td colspan="1" rowspan="1">41/41</td><td colspan="1" rowspan="2">72/72</td><td colspan="1" rowspan="2">100</td></tr><tr><td colspan="1" rowspan="1">5x LoD</td><td colspan="1" rowspan="1">31/31</td></tr></table>

10. Sample Volume Equivalency Study: $2 0 0 \mu \mathrm { L }$ versus $\underline { { 1 0 0 \mu \mathrm { L } } }$ sample volumes:

When an error message is generated by the QIAstat-Dx analyzer indicating that the sample concentration is too high, the user is instructed to repeat testing with half the sample volume $( 1 0 0 ~ \mu \mathrm { L } )$ . This error is most likely to occur when transport media containers are overfilled. To assess the potential for an impact to assay performance when testing the lower sample volume, an equivalency study was conducted comparing testing with a $1 0 0 \mu \mathrm { L }$ sample input volume to the standard sample input volume $( 2 0 0 ~ \mu \mathrm { L } )$ . Samples were prepared by spiking analyte mixes containing one strain for each targeted analyte at $1 \mathrm { x } \mathrm { L o D }$ in stool matrix resuspended in Para-Pak C&S transport medium. Results from the study demonstrated equivalent results between the two sample input volumes and for all assay targets, with the exception of Adenovirus F40/F41, Cyclospora cayetanensis, Entamoeba histolytica, STEC, and $E$ . coli O157. For these targets, additional testing was conducted to determine the assay LoD for samples tested using the $1 0 0 \mu \mathrm { L }$ sample volume. For these four analytes, the LoD when testing a $1 0 0 ~ \mu \mathrm { L }$ sample volume was 3.16x higher than the LoD for samples tested with a $2 0 0 \mu \mathrm { L }$ sample volume. Results from the study are presented in Table 36 below.

The following limitation statements are included in the package insert to warn the user of the risk of overfilling Para-Pak C&S or FecalSwab collection devices and the potential for lower assay sensitivity for the four analytes when testing with the $1 0 0 \mu \mathrm { L }$ sample volume:

The overfilling of Para-Pak C&S or FecalSwab collection devices can result in a failed test with an error indicating ' Sample concentration too high’. • The assay sensitivity to detect Cyclospora cayetanensis, Adenovirus F40/F41, Entamoeba histolytica and the Shiga-like toxin-producing Escherichia coli (STEC) might be reduced up to 3.16-fold when using half-input sample volume $( 1 0 0 ~ \mu \mathrm { L } )$ workflow.

Table 36: LoD Equivalency: 200 and 100 µL Sample Volumes   

<table><tr><td colspan="1" rowspan="1"></td><td colspan="3" rowspan="1">LoD (200 μL)</td><td colspan="4" rowspan="1">LoD (100 μL)</td></tr><tr><td colspan="1" rowspan="1">Target</td><td colspan="1" rowspan="1">Catalog ID.</td><td colspan="1" rowspan="1">Ctmean</td><td colspan="1" rowspan="1">Detectionrate</td><td colspan="1" rowspan="1">Conclusion</td><td colspan="1" rowspan="1">Ctmean</td><td colspan="1" rowspan="1">Detectionrate</td><td colspan="1" rowspan="1">Conclusion</td></tr><tr><td colspan="1" rowspan="1">Campylobacterjejuni</td><td colspan="1" rowspan="1">ZeptoMetrix0801650</td><td colspan="1" rowspan="1">32.9</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">Equivalent</td><td colspan="1" rowspan="1">34.7</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">Equivalent</td></tr><tr><td colspan="1" rowspan="1">Clostridium difficiletoxin A/B</td><td colspan="1" rowspan="1">ATCC 9689</td><td colspan="1" rowspan="1">32.5</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">Equivalent</td><td colspan="1" rowspan="1">33.5</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">Equivalent</td></tr><tr><td colspan="1" rowspan="1">Plesiomonasshigelloides</td><td colspan="1" rowspan="1">ZeptoMetrix0801899</td><td colspan="1" rowspan="1">34.8</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">Equivalent</td><td colspan="1" rowspan="1">35.2</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">Equivalent</td></tr><tr><td colspan="1" rowspan="1">Salmonella</td><td colspan="1" rowspan="1">ZeptoMetrix0801437</td><td colspan="1" rowspan="1">33.5</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">Equivalent</td><td colspan="1" rowspan="1">34.5</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">Equivalent</td></tr><tr><td colspan="1" rowspan="1">Vibrio cholerae</td><td colspan="1" rowspan="1">ZeptoMetrix0801902</td><td colspan="1" rowspan="1">33.9</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">Equivalent</td><td colspan="1" rowspan="1">34.9</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">Equivalent</td></tr><tr><td colspan="1" rowspan="1">Vibrioparahaemolyticus</td><td colspan="1" rowspan="1">ATCC17802</td><td colspan="1" rowspan="1">32.2</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">Equivalent</td><td colspan="1" rowspan="1">33.3</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">Equivalent</td></tr><tr><td colspan="1" rowspan="1">Vibrio vulnificus</td><td colspan="1" rowspan="1">ATCC27562</td><td colspan="1" rowspan="1">32.2</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">Equivalent</td><td colspan="1" rowspan="1">33.6</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">Equivalent</td></tr><tr><td colspan="1" rowspan="1">Yersiniaenterocolitica</td><td colspan="1" rowspan="1">ZeptoMetrix0801734</td><td colspan="1" rowspan="1">34.9</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">Equivalent</td><td colspan="1" rowspan="1">35.7</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">Equivalent</td></tr><tr><td colspan="1" rowspan="1">EnteroaggregativeE. coli (EAEC)</td><td colspan="1" rowspan="1">ZeptoMetrix801919</td><td colspan="1" rowspan="1">33.1</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">Equivalent</td><td colspan="1" rowspan="1">34</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">Equivalent</td></tr><tr><td colspan="1" rowspan="1">EnteropathogenicE. coli (EPEC)</td><td colspan="1" rowspan="1">ZeptoMetrix0801747</td><td colspan="1" rowspan="1">34.8</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">Equivalent</td><td colspan="1" rowspan="1">35.9</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">Equivalent</td></tr><tr><td colspan="1" rowspan="1">Enterotoxigenic E.coli(ETEC) lt/st</td><td colspan="1" rowspan="1">ZeptoMetrix0801624</td><td colspan="1" rowspan="1">33.3</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">Equivalent</td><td colspan="1" rowspan="1">34.3</td><td colspan="1" rowspan="1">19/20</td><td colspan="1" rowspan="1">Equivalent</td></tr><tr><td colspan="1" rowspan="1">Enteroinvasive E.coli(EIEC)/Shigella</td><td colspan="1" rowspan="1">ATCC43892</td><td colspan="1" rowspan="1">34.6</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">Equivalent</td><td colspan="1" rowspan="1">36.4</td><td colspan="1" rowspan="1">19/20</td><td colspan="1" rowspan="1">Equivalent</td></tr><tr><td colspan="1" rowspan="1">Shiga-like toxin-producingE. coli (STEC) stx1</td><td colspan="1" rowspan="1">ZeptoMetrix0801622</td><td colspan="1" rowspan="1">35.3</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">Equivalent</td><td colspan="1" rowspan="1">36.6</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">Equivalent</td></tr><tr><td colspan="1" rowspan="2">Shiga-like toxin-producingE. coli (STEC) stx2</td><td colspan="1" rowspan="2">ZeptoMetrix0801622</td><td colspan="1" rowspan="1">35.1</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">Equivalent</td><td colspan="1" rowspan="1">36.4</td><td colspan="1" rowspan="1">18/20</td><td colspan="1" rowspan="1">Notequivalent</td></tr><tr><td colspan="1" rowspan="1">n/a</td><td colspan="1" rowspan="1">n/a</td><td colspan="1" rowspan="1">n/a</td><td colspan="1" rowspan="1">34.7</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">New LoD*with half-volume input</td></tr><tr><td colspan="4" rowspan="1">LoD (200 μL)</td><td colspan="4" rowspan="1">LoD (100 μL)</td></tr><tr><td colspan="1" rowspan="1">Target</td><td colspan="1" rowspan="1">Catalog ID.</td><td colspan="1" rowspan="1">Ctmean</td><td colspan="1" rowspan="1">Detectionrate</td><td colspan="1" rowspan="1">Conclusion</td><td colspan="1" rowspan="1">Ctmean</td><td colspan="1" rowspan="1">Detectionrate</td><td colspan="1" rowspan="1">Conclusion</td></tr><tr><td colspan="1" rowspan="2">Shiga-like toxin-producingE. coli (STEC)0157</td><td colspan="1" rowspan="2">ZeptoMetrix0801622</td><td colspan="1" rowspan="1">35.3</td><td colspan="1" rowspan="1">19/20</td><td colspan="1" rowspan="1">Equivalent</td><td colspan="1" rowspan="1">36.2</td><td colspan="1" rowspan="1">16/20</td><td colspan="1" rowspan="1">Notequivalent</td></tr><tr><td colspan="1" rowspan="1">n/a</td><td colspan="1" rowspan="1">n/a</td><td colspan="1" rowspan="1">n/a</td><td colspan="1" rowspan="1">34.7</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">New LoD*with half-volume input</td></tr><tr><td colspan="1" rowspan="1">Cryptosporidium</td><td colspan="1" rowspan="1">WaterborneP102C</td><td colspan="1" rowspan="1">29.4</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">Equivalent</td><td colspan="1" rowspan="1">31.7</td><td colspan="1" rowspan="1">19/20</td><td colspan="1" rowspan="1">Equivalent</td></tr><tr><td colspan="1" rowspan="2">Cyclosporacayetanensis</td><td colspan="1" rowspan="2">ClinicalsampleLAC2825</td><td colspan="1" rowspan="1">35.5</td><td colspan="1" rowspan="1">19/20</td><td colspan="1" rowspan="1">Equivalent</td><td colspan="1" rowspan="1">36.6</td><td colspan="1" rowspan="1">17/20</td><td colspan="1" rowspan="1">Notequivalent</td></tr><tr><td colspan="1" rowspan="1">n/a</td><td colspan="1" rowspan="1">n/a</td><td colspan="1" rowspan="1">n/a</td><td colspan="1" rowspan="1">35.6</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">New LoD*with half-volume input</td></tr><tr><td colspan="1" rowspan="2">Entamoebahistolytica</td><td colspan="1" rowspan="2">ATCC30459</td><td colspan="1" rowspan="1">35.7</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">Equivalent</td><td colspan="1" rowspan="1">36.2</td><td colspan="1" rowspan="1">17/20</td><td colspan="1" rowspan="1">Notequivalent</td></tr><tr><td colspan="1" rowspan="1">n/a</td><td colspan="1" rowspan="1">n/a</td><td colspan="1" rowspan="1">n/a</td><td colspan="1" rowspan="1">35</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">New LoD*with half-volume input</td></tr><tr><td colspan="1" rowspan="1">Giardia lamblia</td><td colspan="1" rowspan="1">ATCC30888</td><td colspan="1" rowspan="1">28.7</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">Equivalent</td><td colspan="1" rowspan="1">31.5</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">Equivalent</td></tr><tr><td colspan="1" rowspan="2">AdenovirusF40/F41</td><td colspan="1" rowspan="2">ZeptoMetrix0810085CF</td><td colspan="1" rowspan="1">36.4</td><td colspan="1" rowspan="1">19/20</td><td colspan="1" rowspan="1">Equivalent</td><td colspan="1" rowspan="1">36.9</td><td colspan="1" rowspan="1">13/20</td><td colspan="1" rowspan="1">Notequivalent</td></tr><tr><td colspan="1" rowspan="1">n/a</td><td colspan="1" rowspan="1">n/a</td><td colspan="1" rowspan="1">n/a</td><td colspan="1" rowspan="1">36</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">New LoD*with half-volume input</td></tr><tr><td colspan="1" rowspan="1">Astrovirus</td><td colspan="1" rowspan="1">ZeptoMetrix0810277CF</td><td colspan="1" rowspan="1">31.1</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">Equivalent</td><td colspan="1" rowspan="1">32.1</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">Equivalent</td></tr><tr><td colspan="1" rowspan="1">Norovirus GI/GII</td><td colspan="1" rowspan="1">ZeptoMetrix0810087CF</td><td colspan="1" rowspan="1">32.7</td><td colspan="1" rowspan="1">19/20</td><td colspan="1" rowspan="1">Equivalent</td><td colspan="1" rowspan="1">33.5</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">Equivalent</td></tr><tr><td colspan="1" rowspan="1">Rotavirus A</td><td colspan="1" rowspan="1">ZeptoMetrix0810280CF</td><td colspan="1" rowspan="1">36.2</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">Equivalent</td><td colspan="1" rowspan="1">36.6</td><td colspan="1" rowspan="1">19/20</td><td colspan="1" rowspan="1">Equivalent</td></tr></table>

\*New LoD for testing $1 0 0 \mu \mathrm { L }$ sample volume was determined to be 3.16x higher than the LoD for testing $2 0 0 ~ \mu \mathrm { L }$ sample volume.

The study results support that the risk to clinical performance when testing the reduced sample volume is likely to be minimal as the new LoD was only 3.16-fold higher for the five analytes listed above.

# 11. Assay Cut-Off:

The QIAstat Dx Gastrointestinal Panel 2 includes defined Ct value cutoffs for each assay target. The Ct cutoffs are included as automatic calculations in the assay definition file (ADF).

# B Comparison Studies:

1. Method Comparison with Predicate Device:

Not applicable.

2. Matrix Comparison:

Not applicable.

# C Clinical Studies:

Prospective Clinical Study:

The clinical performance of QIAstat-Dx Gastrointestinal Panel 2 was established during a multicenter international prospective study conducted at thirteen clinical settings representative of different geographical areas within the US and Europe (nine sites in US and four sites in Europe) between May and July 2021. All study sites were hospital-associated or independent clinical diagnostics laboratories that perform routine diagnostics of gastrointestinal infections. A total of 1939 prospectively collected evaluable stool specimens (stool in Para-Pak C&S (Meridian Bioscience) or FecalSwab (COPAN) transport media) were obtained from patients with clinical indications of diarrhea caused by gastrointestinal infection. For the 1939 evaluable specimens tested in the prospective study, Table 37 provides a summary of specimen distribution across all study sites and Table 38 includes a summary of patient demographic information.

Table 37: Prospective Clinical Study, Specimen Distribution   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=3>No. of prospective specimens</td></tr><tr><td rowspan=1 colspan=1>Site/Country</td><td rowspan=1 colspan=1>FecalSwab</td><td rowspan=1 colspan=1>Para-Pak C&amp;S</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Germany</td><td rowspan=1 colspan=1>293</td><td rowspan=1 colspan=1>46</td><td rowspan=1 colspan=1>339</td></tr><tr><td rowspan=1 colspan=1>Denmark</td><td rowspan=1 colspan=1>293</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>293</td></tr><tr><td rowspan=1 colspan=1>Spain</td><td rowspan=1 colspan=1>247</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>247</td></tr><tr><td rowspan=1 colspan=1>France</td><td rowspan=1 colspan=1>63</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>63</td></tr><tr><td rowspan=1 colspan=1>USA site 1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>186</td><td rowspan=1 colspan=1>186</td></tr><tr><td rowspan=1 colspan=1>USA site 2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>43</td><td rowspan=1 colspan=1>43</td></tr><tr><td rowspan=1 colspan=1>USA site 3</td><td rowspan=1 colspan=1>282</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>282</td></tr><tr><td rowspan=1 colspan=1>USA site 4</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>177</td><td rowspan=1 colspan=1>177</td></tr><tr><td rowspan=1 colspan=1>USA site 5</td><td rowspan=1 colspan=1>44</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>44</td></tr><tr><td rowspan=1 colspan=1>USA site 6</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>39</td><td rowspan=1 colspan=1>39</td></tr><tr><td rowspan=1 colspan=1>USA site 7a</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>USA site 8</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>131</td><td rowspan=1 colspan=1>131</td></tr><tr><td rowspan=1 colspan=1>USA site 9</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>95</td><td rowspan=1 colspan=1>95</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>1222</td><td rowspan=1 colspan=1>717</td><td rowspan=1 colspan=1>1939</td></tr></table>

a Specimens from Site 7 were excluded from the analysis because they were collected with a device other than ParaPak C&S or FecalSwab.

Table 38: Demographic data for enrolled prospective specimens   

<table><tr><td rowspan=1 colspan=1>Demographic data</td><td rowspan=1 colspan=2>FecalSwab</td><td rowspan=1 colspan=2>Para-Pak C&amp;S</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>%</td></tr><tr><td rowspan=1 colspan=5>Gender</td></tr><tr><td rowspan=1 colspan=1>Female</td><td rowspan=1 colspan=1>628</td><td rowspan=1 colspan=1>32.4</td><td rowspan=1 colspan=1>442</td><td rowspan=1 colspan=1>22.8</td></tr><tr><td rowspan=1 colspan=1>Male</td><td rowspan=1 colspan=1>594</td><td rowspan=1 colspan=1>30.6</td><td rowspan=1 colspan=1>275</td><td rowspan=1 colspan=1>14.2</td></tr><tr><td rowspan=1 colspan=5>Age Group</td></tr><tr><td rowspan=1 colspan=1>0-5 years</td><td rowspan=1 colspan=1>182</td><td rowspan=1 colspan=1>9.4</td><td rowspan=1 colspan=1>31</td><td rowspan=1 colspan=1>1.6</td></tr><tr><td rowspan=1 colspan=1>6-21 years</td><td rowspan=1 colspan=1>121</td><td rowspan=1 colspan=1>6.2</td><td rowspan=1 colspan=1>38</td><td rowspan=1 colspan=1>2.0</td></tr><tr><td rowspan=1 colspan=1>22-49 years</td><td rowspan=1 colspan=1>290</td><td rowspan=1 colspan=1>15.0</td><td rowspan=1 colspan=1>215</td><td rowspan=1 colspan=1>11.1</td></tr><tr><td rowspan=1 colspan=1>50+ years</td><td rowspan=1 colspan=1>629</td><td rowspan=1 colspan=1>32.4</td><td rowspan=1 colspan=1>426</td><td rowspan=1 colspan=1>22.0</td></tr><tr><td rowspan=1 colspan=1>Not Reported</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>0.4</td></tr><tr><td rowspan=1 colspan=5>Patient population</td></tr><tr><td rowspan=1 colspan=1>Emergency room</td><td rowspan=1 colspan=1>46</td><td rowspan=1 colspan=1>2.4</td><td rowspan=1 colspan=1>29</td><td rowspan=1 colspan=1>1.5</td></tr><tr><td rowspan=1 colspan=1>Hospitalized</td><td rowspan=1 colspan=1>342</td><td rowspan=1 colspan=1>17.6</td><td rowspan=1 colspan=1>143</td><td rowspan=1 colspan=1>7.4</td></tr><tr><td rowspan=1 colspan=1>Immunocompromised</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0</td></tr><tr><td rowspan=1 colspan=1>Outpatient</td><td rowspan=1 colspan=1>491</td><td rowspan=1 colspan=1>25.3</td><td rowspan=1 colspan=1>325</td><td rowspan=1 colspan=1>16.8</td></tr><tr><td rowspan=1 colspan=1>No information available</td><td rowspan=1 colspan=1>340</td><td rowspan=1 colspan=1>17.5</td><td rowspan=1 colspan=1>220</td><td rowspan=1 colspan=1>11.3</td></tr><tr><td rowspan=1 colspan=5>No. Days between Symptom Onset and QIAstat-Dx Testing</td></tr><tr><td rowspan=1 colspan=1>&gt; 7 days</td><td rowspan=1 colspan=1>89</td><td rowspan=1 colspan=1>4.6</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0</td></tr><tr><td rowspan=1 colspan=1>≤ 7 days</td><td rowspan=1 colspan=1>146</td><td rowspan=1 colspan=1>7.5</td><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>0.8</td></tr><tr><td rowspan=1 colspan=1>Not Reported</td><td rowspan=1 colspan=1>987</td><td rowspan=1 colspan=1>50.9</td><td rowspan=1 colspan=1>701</td><td rowspan=1 colspan=1>36.2</td></tr></table>

The performance of the QIAstat-Dx Gastrointestinal Panel 2 was evaluated for each panel test result using one FDA-cleared test as comparator for the most analytes. A composite comparator consisting of either three independent FDA-cleared test methods or two independent FDAcleared tests methods and two validated PCR assays followed by bi-directional sequencing was used for Norovirus GI/GII, ETEC, STEC and Giardia lamblia (Table 39).

Table 39: QIAstat-Dx Gastrointestinal Panel 2 Clinical studies comparator method   

<table><tr><td rowspan=1 colspan=1>QIAstat-DxGastrointestinal Panel 2target</td><td rowspan=1 colspan=1>Comparator method</td></tr><tr><td rowspan=1 colspan=1>Adenovirus F40/F41</td><td rowspan=1 colspan=1>FDA-cleared molecular assay</td></tr><tr><td rowspan=1 colspan=1>Astrovirus</td><td rowspan=1 colspan=1>FDA-cleared molecular assay</td></tr><tr><td rowspan=1 colspan=1>Norovirus GI/GII</td><td rowspan=1 colspan=1>Composite of three FDA-cleared molecularassays</td></tr><tr><td rowspan=1 colspan=1>Rotavirus A</td><td rowspan=1 colspan=1>FDA-cleared molecular assay</td></tr><tr><td rowspan=1 colspan=1>Campylobacter (C. jejuni,C. coli and C. upsaliensis)</td><td rowspan=1 colspan=1>FDA-cleared molecular assay</td></tr><tr><td rowspan=1 colspan=1>Shigella/EnteroinvasiveEscherichia coli (EIEC)</td><td rowspan=1 colspan=1>FDA-cleared molecular assay</td></tr><tr><td rowspan=1 colspan=1>EnteropathogenicEscherichia coli (EPEC)*</td><td rowspan=1 colspan=1>FDA-cleared molecular assay</td></tr><tr><td rowspan=1 colspan=1>Enterotoxigenic Escherichiacoli (ETEC) lt/st</td><td rowspan=1 colspan=1>Composite of three FDA-cleared molecularassays</td></tr><tr><td rowspan=1 colspan=1>Shiga-like toxin- producingEscherichia coli (STEC)stx1/stx2*</td><td rowspan=1 colspan=1>Composite of three FDA-cleared molecularassays</td></tr><tr><td rowspan=1 colspan=1>E. coli 0157 serogroup*</td><td rowspan=1 colspan=1>FDA-cleared molecular assay</td></tr><tr><td rowspan=1 colspan=1>Salmonella</td><td rowspan=1 colspan=1>FDA-cleared molecular assay</td></tr><tr><td rowspan=1 colspan=1>Plesiomonas shigelloides</td><td rowspan=1 colspan=1>FDA-cleared molecular assay</td></tr><tr><td rowspan=1 colspan=1>Yersinia enterocolitica</td><td rowspan=1 colspan=1>FDA-cleared molecular assay</td></tr><tr><td rowspan=1 colspan=1>Cryptosporidium</td><td rowspan=1 colspan=1>FDA-cleared molecular assay</td></tr><tr><td rowspan=1 colspan=1>Cyclospora cayetanensis</td><td rowspan=1 colspan=1>FDA-cleared molecular assay</td></tr><tr><td rowspan=1 colspan=1>Entamoeba histolytica</td><td rowspan=1 colspan=1>FDA-cleared molecular assay</td></tr><tr><td rowspan=1 colspan=1>Giardia lamblia</td><td rowspan=1 colspan=1>Composite of two FDA-cleared molecularassays and two validated PCR/Sequencingassays**</td></tr></table>

\*Targets evaluated in Para-Pak C&S only \*\* For Giardia lamblia, each of the two PCR comparator assays used were well-characterized and validated nucleic acid amplification tests (NAAT) followed by bi-directional sequencing analysis. Both assays were designed to amplify different sequences than those targeted by the QIAstat-Dx Gastrointestinal Panel 2. Positive results required to generate sequences from bi-directional sequencing with at least 200 bases of

adequate quality that by BLAST analyses matched a sequence of the expected organism or gene from NCBI GenBank database with at least $9 5 \%$ query coverage and at least $9 5 \%$ identity compared to the reference.

Where a composite comparator was used, the result was determined as the majority of the three individual test results (i.e., a positive composite comparator result is based on positive results for at least two comparator tests and a negative composite comparator result is based on negative results for at least two comparator tests). If insufficient pathogen positive sample was available to obtain a majority test result, a worst-case model was applied in the PPA calculation. In this model, the PPA was calculated including all observed true positive and false negative samples between QIAstat-Dx Gastrointestinal Panel 2 and the composite comparator, while the following procedure was used for the samples where it was not possible to conduct testing with the complete comparator due to insufficient sample volume:

Samples that were negative for the QIAstat-Dx Gastrointestinal Panel 2 and positive for one comparator test, negative (or insufficient volume) for a second comparator, and insufficient volume for a third comparator were included in the calculations as worst-case false negatives.   
Samples that were positive for the QIAstat-Dx Gastrointestinal Panel 2 and positive for one comparator test, negative (or insufficient volume) for a second comparator, and insufficient volume for the third comparator, were considered as worst-case false positives and therefore excluded from the PPA calculations.

The PPA was calculated as $1 0 0 \% \times \left( \mathrm { T P / \left( T P + F N \right) } \right)$ . True positive (TP) indicates that both the QIAstat-Dx Gastrointestinal Panel 2 and comparator method showed a positive result for the specific target, and false negative (FN) indicates that the QIAstat-Dx Gastrointestinal Panel 2 result was negative while the comparator method result was positive. The NPA was calculated as $100 \% \mathbf { x }$ $\left( \mathrm { T N } / \left( \mathrm { T N } + \mathrm { F P } \right) \right)$ . True negative (TN) indicates that both the QIAstat-Dx Gastrointestinal Panel 2 and the comparator method showed negative results, and a false positive (FP) indicates that the QIAstat-Dx Gastrointestinal Panel 2 result was positive when the comparator method result was negative. The exact binomial two-sided $9 5 \%$ confidence intervals for PPA and NPA were calculated.

Discrepancies between the QIAstat-Dx Gastrointestinal Panel 2 and the comparator methods were investigated for the analytes where the QIAstat-Dx Gastrointestinal Panel 2 test result was compared to one FDA-cleared method. Discrepant result analyses are footnoted in the clinical performance summary table.

Results for the prospective clinical study are presented in Table 40.

Table 40: QIAstat-Dx Gastrointestinal Panel 2 Clinical Performance, Prospective study   

<table><tr><td colspan="2" rowspan="1"></td><td colspan="3" rowspan="1">Positive PercentAgreement</td><td colspan="3" rowspan="1">Negative PercentAgreement</td></tr><tr><td colspan="1" rowspan="1">Analyte</td><td colspan="1" rowspan="1">MediumBrand</td><td colspan="1" rowspan="1">TP/TP+FN</td><td colspan="1" rowspan="1">%</td><td colspan="1" rowspan="1">95% CI</td><td colspan="1" rowspan="1">TN/TN+FP</td><td colspan="1" rowspan="1">%</td><td colspan="1" rowspan="1">95% CI</td></tr><tr><td colspan="8" rowspan="1">Viruses</td></tr><tr><td colspan="1" rowspan="2">Adenovirus F40/F41</td><td colspan="1" rowspan="1">FecalSwab</td><td colspan="1" rowspan="1">5/61</td><td colspan="1" rowspan="1">83.3</td><td colspan="1" rowspan="1">497.0</td><td colspan="1" rowspan="1">1216 / 1216</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">90- </td></tr><tr><td colspan="1" rowspan="1">Para-Pak C&amp;S</td><td colspan="1" rowspan="1">1 /22</td><td colspan="1" rowspan="1">50.0</td><td colspan="1" rowspan="1">9.5-90.6</td><td colspan="1" rowspan="1">703 / 7042</td><td colspan="1" rowspan="1">99.9</td><td colspan="1" rowspan="1">90</td></tr><tr><td colspan="1" rowspan="2">Astrovirus</td><td colspan="1" rowspan="1">FecalSwab</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">43.9-4100</td><td colspan="1" rowspan="1">1219 / 1219</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">90- -</td></tr><tr><td colspan="1" rowspan="1">Para-Pak C&amp;S</td><td colspan="1" rowspan="1">6/6</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">6100-</td><td colspan="1" rowspan="1">700 / 700</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">90- </td></tr><tr><td colspan="1" rowspan="2">Norovirus GI/GII</td><td colspan="1" rowspan="1">FecalSwab</td><td colspan="1" rowspan="1">31 /33 3</td><td colspan="1" rowspan="1">93.9</td><td colspan="1" rowspan="1">98.4 </td><td colspan="1" rowspan="1">493 / 495 3</td><td colspan="1" rowspan="1">99.6</td><td colspan="1" rowspan="1">906</td></tr><tr><td colspan="1" rowspan="1">Para-Pak C&amp;S</td><td colspan="1" rowspan="1">14 /184</td><td colspan="1" rowspan="1">77.8</td><td colspan="1" rowspan="1">91.0 </td><td colspan="1" rowspan="1">399 / 399 4</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">990- -</td></tr><tr><td colspan="1" rowspan="2">Rotavirus A</td><td colspan="1" rowspan="1">FecalSwab</td><td colspan="1" rowspan="1">21 /235</td><td colspan="1" rowspan="1">91.3</td><td colspan="1" rowspan="1">97.0 </td><td colspan="1" rowspan="1">1197 / 11995</td><td colspan="1" rowspan="1">99.8</td><td colspan="1" rowspan="1">90 </td></tr><tr><td colspan="1" rowspan="1">Para-Pak C&amp;S</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">4300</td><td colspan="1" rowspan="1">702 / 703 6</td><td colspan="1" rowspan="1">99.9</td><td colspan="1" rowspan="1">900 -</td></tr><tr><td colspan="8" rowspan="1">Bacteria</td></tr><tr><td colspan="1" rowspan="2">Campylobacter</td><td colspan="1" rowspan="1">FecalSwab</td><td colspan="1" rowspan="1">65 / 677</td><td colspan="1" rowspan="1">97.0</td><td colspan="1" rowspan="1">9.82</td><td colspan="1" rowspan="1">1151 /11557</td><td colspan="1" rowspan="1">99.7</td><td colspan="1" rowspan="1">99 9</td></tr><tr><td colspan="1" rowspan="1">Para-Pak C&amp;S</td><td colspan="1" rowspan="1">30 /318</td><td colspan="1" rowspan="1">96.8</td><td colspan="1" rowspan="1">99.4</td><td colspan="1" rowspan="1">675 / 677 8</td><td colspan="1" rowspan="1">99.7</td><td colspan="1" rowspan="1">999 9</td></tr><tr><td colspan="1" rowspan="2">Plesiomonas shigelloides</td><td colspan="1" rowspan="1">FecalSwab</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">1220 / 12229</td><td colspan="1" rowspan="1">99.8</td><td colspan="1" rowspan="1">99.4-100</td></tr><tr><td colspan="1" rowspan="1">Para-Pak C&amp;S</td><td colspan="1" rowspan="1">5/6 10</td><td colspan="1" rowspan="1">83.3</td><td colspan="1" rowspan="1">497.0</td><td colspan="1" rowspan="1">698 / 700 10</td><td colspan="1" rowspan="1">99.7</td><td colspan="1" rowspan="1">9999</td></tr><tr><td colspan="1" rowspan="2">Salmonella</td><td colspan="1" rowspan="1">FecalSwab</td><td colspan="1" rowspan="1">14 / 16 11</td><td colspan="1" rowspan="1">87.5</td><td colspan="1" rowspan="1">640.0 </td><td colspan="1" rowspan="1">1206 / 1206</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">90- -</td></tr><tr><td colspan="1" rowspan="1">Para-Pak C&amp;S</td><td colspan="1" rowspan="1">19 / 20 12</td><td colspan="1" rowspan="1">95.0</td><td colspan="1" rowspan="1">76.4</td><td colspan="1" rowspan="1">688 / 688</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">90 </td></tr><tr><td colspan="1" rowspan="2">Yersinia enterocolitica</td><td colspan="1" rowspan="1">FecalSwab</td><td colspan="1" rowspan="1">15 / 16 13</td><td colspan="1" rowspan="1">93.8</td><td colspan="1" rowspan="1">71.7-99.0</td><td colspan="1" rowspan="1">1199/1206 13</td><td colspan="1" rowspan="1">99.4</td><td colspan="1" rowspan="1">99. </td></tr><tr><td colspan="1" rowspan="1">Para-Pak C&amp;S</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">43.9-0</td><td colspan="1" rowspan="1">698 / 703 14</td><td colspan="1" rowspan="1">99.3</td><td colspan="1" rowspan="1">99.4 </td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Diarrheage</td><td colspan="4" rowspan="1">Diarrheagenic E. coli/Shigella</td><td colspan="2" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Enteropathogenic E. coli (EPEC)</td><td colspan="1" rowspan="1">Para-Pak C&amp;S</td><td colspan="1" rowspan="1">57 / 65</td><td colspan="1" rowspan="1">87.7</td><td colspan="1" rowspan="1">99.6 </td><td colspan="1" rowspan="1">632 / 632</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">9 </td></tr><tr><td colspan="1" rowspan="2">Enterotoxigenic E. coli (ETEC) It/st</td><td colspan="1" rowspan="1">FecalSwab</td><td colspan="1" rowspan="1">9 / 10 15</td><td colspan="1" rowspan="1">90.0</td><td colspan="1" rowspan="1">9.20 </td><td colspan="1" rowspan="1">427 / 430 15</td><td colspan="1" rowspan="1">99.3</td><td colspan="1" rowspan="1">99.0 </td></tr><tr><td colspan="1" rowspan="1">Para-Pak C&amp;S</td><td colspan="1" rowspan="1">9 / 1016</td><td colspan="1" rowspan="1">90.0</td><td colspan="1" rowspan="1">59.2 5</td><td colspan="1" rowspan="1">390 / 395 16</td><td colspan="1" rowspan="1">98.7</td><td colspan="1" rowspan="1">915 5</td></tr><tr><td colspan="1" rowspan="1">Shiga-like toxin E. coli (STEC)stx1/stx2</td><td colspan="1" rowspan="1">Para-Pak C&amp;S</td><td colspan="1" rowspan="1">5/617</td><td colspan="1" rowspan="1">83.3</td><td colspan="1" rowspan="1">497.0</td><td colspan="1" rowspan="1">397 / 400 17</td><td colspan="1" rowspan="1">99.3</td><td colspan="1" rowspan="1">9.8 </td></tr><tr><td colspan="1" rowspan="1">E. coli 0157</td><td colspan="1" rowspan="1">Para-Pak C&amp;S</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">5/5</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">560 </td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">FecalSwab</td><td colspan="1" rowspan="1">10 / 10</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">70</td><td colspan="1" rowspan="1">1212 / 1212</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">90 -</td></tr><tr><td colspan="1" rowspan="1">Shigella/Enteroinvasive E. Coli(EIEC)</td><td colspan="1" rowspan="1">Para-Pak C&amp;S</td><td colspan="1" rowspan="1">2/2</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">34.2-S 100</td><td colspan="1" rowspan="1">703 / 704 18</td><td colspan="1" rowspan="1">99.9</td><td colspan="1" rowspan="1">99.2-0</td></tr><tr><td colspan="8" rowspan="1">Parasites</td></tr><tr><td colspan="1" rowspan="2">Cryptosporidium</td><td colspan="1" rowspan="1">FecalSwab</td><td colspan="1" rowspan="1">2/4</td><td colspan="1" rowspan="1">50.0</td><td colspan="1" rowspan="1">15.0-85.0</td><td colspan="1" rowspan="1">1218 / 1218</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">99.7-00</td></tr><tr><td colspan="1" rowspan="1">Para-Pak C&amp;S</td><td colspan="1" rowspan="1">6/6</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">61.0-100</td><td colspan="1" rowspan="1">699 / 700 19</td><td colspan="1" rowspan="1">99.9</td><td colspan="1" rowspan="1">99.2-10</td></tr><tr><td colspan="1" rowspan="2">Cyclospora cayetanensis</td><td colspan="1" rowspan="1">FecalSwab</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">43.9-100</td><td colspan="1" rowspan="1">1219 / 1219</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">99.7-00</td></tr><tr><td colspan="1" rowspan="1">Para-Pak C&amp;S</td><td colspan="1" rowspan="1">18 / 19 20</td><td colspan="1" rowspan="1">94.7</td><td colspan="1" rowspan="1">75.4-99.1</td><td colspan="1" rowspan="1">687 / 687</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">99.4-10</td></tr><tr><td colspan="1" rowspan="2">Entamoeba histolytica</td><td colspan="1" rowspan="1">FecalSwab</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">1222 / 1222</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">99.7-1</td></tr><tr><td colspan="1" rowspan="1">Para-Pak C&amp;S</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">706 / 706</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">99.5-10</td></tr><tr><td colspan="1" rowspan="2">Giardia lamblia</td><td colspan="1" rowspan="1">FecalSwab</td><td colspan="1" rowspan="1">6 /8 21</td><td colspan="1" rowspan="1">75.0</td><td colspan="1" rowspan="1">40.9-92.9</td><td colspan="1" rowspan="1">434 / 441 21</td><td colspan="1" rowspan="1">98.4</td><td colspan="1" rowspan="1">96.8-99.2</td></tr><tr><td colspan="1" rowspan="1">Para-Pak C&amp;S</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">20.7-100</td><td colspan="1" rowspan="1">406 / 40622</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">99.1-10</td></tr></table>

1 Adenovirus F40/41 was not detected in the single false negative specimen $( 0 / 1 )$ in FecalSwab using a different FDA-cleared test method.

2 Adenovirus F40/41 was not detected in the single false negative specimen $( 0 / 1 )$ and in the single false positive specimen (0/1) in Para-Pak C&S using a different FDA-cleared test method.

3 Ten (10) FecalSwab samples positive for Norovirus GI/GII by both QIAstat-Dx and one FDA-cleared comparator were excluded from the PPA calculations because the samples did not have sufficient volume for complete composite comparator testing. The sample size for NPA is smaller for Norovirus GI/GII as only a portion of the samples with a negative result by QIAstat-Dx and by one FDA-cleared comparator was tested with the complete composite comparator in the prospective study.

4 Two (2) Para-Pak C&S samples positive for Norovirus GI/GII by both QIAstat-Dx and one FDA-cleared comparator were excluded from the PPA calculations because the samples did not have sufficient volume for complete composite comparator testing. One (1) Para-Pak C&S sample negative by QIAstat-Dx and positive with one FDA-cleared comparator with insufficient volume for complete composite comparator testing were classed as false negative in the PPA calculations. The sample size for NPA is smaller for Norovirus GI/GII as only a portion of the samples with a negative result by QIAstat-Dx and by one FDAcleared comparator was tested with the complete composite comparator in the prospective study.

5 Rotavirus A was detected in one of the two false negative specimens (1/2) and was not detected in the two false positive specimens (0/2) in FecalSwab using a different FDA-cleared test method.

6 Rotavirus A was not detected in the single false positive specimen $( 0 / 1 )$ ) in Para-Pak C&S using a different FDA-cleared test method.

7 Campylobacter was not detected in the two false negative specimens $( 0 / 2 )$ and was detected in three of the four false positive specimens (3/4) in FecalSwab using a different FDA-cleared test method.

8 Campylobacter was not detected in the single false negative specimens $( 0 / 1 )$ and was detected in one of the two false positive specimens (1/2) in Para-Pak C&S using a different FDA-cleared test method.

9 Plesiomonas shigelloides was not detected in the two false positive specimens (0/2) in FecalSwab using a different FDA-cleare test method.

10 Plesiomonas shigelloides was not detected in the single false negative specimen (0/1) and was not detected in the two false positive specimens (0/2) in Para-Pak C&S using a different FDA-cleared test method.

11 Salmonella was not detected in the two false negative specimens (0/2) in FecalSwab using a different FDA-cleared test method.

12 Salmonella was not detected in the single false negative specimen (0/1) in Para-Pak C&S using a different FDA-cleared test method.

13 Yersinia enterocolitica was not detected in the single false negative specimen (0/1) and was not detected in the seven false positive specimens (0/7) in FecalSwab using a different FDA-cleared test method.

14 Yersinia enterocolitica was not detected in the five false positive specimens (0/5) using a different FDA-cleared test method.

$^ { 1 5 } \mathrm { S i x }$ (6) FecalSwab samples positive for ETEC by both QIAstat-Dx and the primary FDA-cleared comparator were excluded from the PPA calculations because the samples did not have sufficient volume for complete composite comparator testing. The sample size for NPA is smaller for ETEC as only a portion of the samples with a negative result by QIAstat-Dx and by one FDAcleared comparator was tested with the complete composite comparator in the prospective study.

16 Three (3) Para-Pak C&S samples positive for ETEC by both QIAstat-Dx and one FDA-cleared comparator were excluded from the PPA calculations because the samples did not have sufficient volume for complete composite comparator testing. The sample size for NPA is smaller for ETEC as only a portion of the samples with a negative result by QIAstat-Dx and by one FDA-cleared comparator was tested with the complete composite comparator in the prospective study.

17 One (1) FecalSwab sample positive for STEC by both QIAstat-Dx and one FDA-cleared comparator was excluded from the PPA calculations because the samples did not have sufficient volume for complete composite comparator testing. The sample size for NPA is smaller for STEC as only a portion of the samples with a negative result by QIAstat-Dx and by one FDA-cleared comparator was tested with the complete composite comparator in the prospective study.

18 Shigella/Enteroinvasive E. coli (EIEC) was detected in the single false positive specimen (1/1) in Para-Pak C&S using a different FDA-cleared test method.

19 Cryptosporidium was not detected in the single false positive specimen $( 0 / 1 )$ in Para-Pak C&S using PCR followed by bidirectional sequence analysis.

20 For Cyclospora cayetanensis there was one (1) false negative specimen in Para-Pak C&S that was not further investigated by discrepant analyses.

21 Two (2) FecalSwab samples positive for Giardia lamblia by both QIAstat-Dx and one FDA-cleared comparator were excluded from the PPA calculations because the samples did not have sufficient volume for complete composite comparator testing. Two (2) FecalSwab samples negative by QIAstat-Dx and positive with one FDA-cleared comparator with insufficient volume for complete composite comparator testing were classed as false negative in the PPA calculations. The sample size for NPA is smaller for Giardia lamblia as only a portion of the samples with a negative result by QIAstat-Dx and by one FDA-cleared comparator was tested with the complete composite comparator in the prospective study.

22the sample size for NPA is smaller for Giardia lamblia as only a portion of the samples with a negative result in QIAstat-Dx and in one FDA-cleared comparator was tested with the complete composite comparator in the prospective study.

The QIAstat-Dx Gastrointestinal Panel 2 reported multiple organism detections (i.e., mixed infections) for a total of 15 and 29 prospective specimens in FecalSwab and Para-Pak C&S, respectively. This represents $7 . 0 \%$ of positive specimens (15/213) in FecalSwab and $1 7 . 0 \%$ of positive specimens (29/171) in Para-Pak C&S. Most multiple detections in FecalSwab specimens (14/15; $9 3 \%$ ) contained two organisms, while $6 . 7 \%$ (1/15) contained three organisms. In ParaPak C&S specimens, most multiple detections (22/29; $7 5 . 9 \%$ ) contained two organisms, while $2 4 . 1 \%$ (7/29) contained three organisms. The most common multiple organism detections are presented in Table 41 and Table 42 below.

Table 41: Most Prevalent Multiple Detection Combinations $_ { \mathrm { 2 2 } }$ instances) as Determined by the QIAstat-Dx Gastrointestinal Panel 2 in the Prospective Clinical study in FecalSwab specimens   

<table><tr><td rowspan=1 colspan=1>Multiple Detection Combination</td><td rowspan=1 colspan=1>Number of FecalSwabSpecimens</td></tr><tr><td rowspan=1 colspan=1>Campylobacter + Norovirus GI/GII</td><td rowspan=1 colspan=1>2</td></tr><tr><td rowspan=1 colspan=1>Enterotoxigenic E. coli (ETEC) lt/st + Norovirus GI/GII</td><td rowspan=1 colspan=1>3</td></tr><tr><td rowspan=1 colspan=1>Campylobacter + Rotavirus A</td><td rowspan=1 colspan=1>4</td></tr></table>

Table 42. Most Prevalent Multiple Detection Combinations ${ \geq } 2$ instances) as Determined by the QIAstat-Dx Gastrointestinal Panel 2 in the Prospective Clinical study in Para-Pak C&S specimens   

<table><tr><td rowspan=1 colspan=1>Multiple Detection Combination</td><td rowspan=1 colspan=1>Number of Para-PakC&amp;S Specimens</td></tr><tr><td rowspan=1 colspan=1>Campylobacter + Enteropathogenic E. coli (EPEC)</td><td rowspan=1 colspan=1>3</td></tr><tr><td rowspan=1 colspan=1>Enteropathogenic E. coli (EPEC) + Salmonella</td><td rowspan=1 colspan=1>3</td></tr><tr><td rowspan=1 colspan=1>Enteropathogenic E. coli (EPEC) + Enterotoxigenic E. coli (ETEC) lt/st</td><td rowspan=1 colspan=1>4</td></tr></table>

The analytes most commonly found in mixed infections in FecalSwab specimens were Campylobacter (9), Norovirus GI/GII (7), Rotavirus (4) and ETEC (3)) as shown in Table 43, while the analytes most commonly found in mixed infections in Para-Pak C&S specimens were EPEC (17), ETEC (8), Campylobacter (7), Norovirus GI/GII (5), Rotavirus (4) and STEC (5) as shown in Table 44.

Table 43: Prevalence of Analytes in Mixed Infections in FecalSwab specimens as determined by the QIAstat-Dx Gastrointestinal Panel 2   

<table><tr><td rowspan=1 colspan=1>Analyte</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>%</td></tr><tr><td rowspan=1 colspan=1>Adenovirus F40/F41</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>3.2</td></tr><tr><td rowspan=1 colspan=1>Astrovirus</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>3.2</td></tr><tr><td rowspan=1 colspan=1>Campylobacter</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>29.0</td></tr><tr><td rowspan=1 colspan=1>Enterotoxigenic E. coli (ETEC) lt/st</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>9.7</td></tr><tr><td rowspan=1 colspan=1>Giardia lamblia</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>6.5</td></tr><tr><td rowspan=1 colspan=1>Norovirus GI/GII</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>22.6</td></tr><tr><td rowspan=1 colspan=1>Plesiomonas shigelloides</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>3.2</td></tr><tr><td rowspan=1 colspan=1>Rotavirus A</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>12.9</td></tr><tr><td rowspan=1 colspan=1>Shigella/Enteroinvasive E. coli (EIEC)</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>6.5</td></tr><tr><td rowspan=1 colspan=1>Yersinia enterocolitica</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>3.2</td></tr></table>

Table 44: Prevalence of Analytes in Mixed Infections in Para-Pak C&S specimens as determined by the QIAstat-Dx Gastrointestinal Panel 2   

<table><tr><td colspan="1" rowspan="1">Analyte</td><td colspan="1" rowspan="1">N</td><td colspan="1" rowspan="1">%</td></tr><tr><td colspan="1" rowspan="1">Adenovirus F40/F41</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1.5</td></tr><tr><td colspan="1" rowspan="1">Astrovirus</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1.5</td></tr><tr><td colspan="1" rowspan="1">Campylobacter</td><td colspan="1" rowspan="1">7</td><td colspan="1" rowspan="1">10.8</td></tr><tr><td colspan="1" rowspan="1">Cryptosporidium</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">3.1</td></tr><tr><td colspan="1" rowspan="1">Cyclospora cayetanensis</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">3.1</td></tr><tr><td colspan="1" rowspan="1">Enteropathogenic E. coli (EPEC)</td><td colspan="1" rowspan="1">17</td><td colspan="1" rowspan="1">26.2</td></tr><tr><td colspan="1" rowspan="1">Enterotoxigenic E. coli (ETEC) lt/st</td><td colspan="1" rowspan="1">8</td><td colspan="1" rowspan="1">12.3</td></tr><tr><td colspan="1" rowspan="1">Giardia lamblia</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1.5</td></tr><tr><td colspan="1" rowspan="1">Norovirus GI/GII</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">7.7</td></tr><tr><td colspan="1" rowspan="1">Plesiomonas shigelloides</td><td colspan="1" rowspan="1">7</td><td colspan="1" rowspan="1">10.8</td></tr><tr><td colspan="1" rowspan="1">Rotavirus A</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1.5</td></tr><tr><td colspan="1" rowspan="1">Salmonella</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">6.2</td></tr><tr><td colspan="1" rowspan="1">Shiga-like toxin E. coli (STEC) stx1/stx2</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">7.7</td></tr><tr><td colspan="1" rowspan="1">Shigella/Enteroinvasive E. coli (EIEC)</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">4.6</td></tr><tr><td colspan="1" rowspan="1">Yersinia enterocolitica</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1.5</td></tr></table>

Tables 45-48 include details on prospective clinical specimens (FecalSwab and Para-Pak C&S) with multiple organisms detected by the QIAstat-Dx Gastrointestinal Panel 2 and comparator methods.

Table 45: Multi-Detections for QIAstat-Dx Gastrointestinal Panel 2, FecalSwab   

<table><tr><td rowspan=1 colspan=1>Analyte 1</td><td rowspan=1 colspan=1>Analyte 2</td><td rowspan=1 colspan=1>Analyte 3</td><td rowspan=1 colspan=1># Multi-Detections</td><td rowspan=1 colspan=1># FPanalytes byQIAStat</td><td rowspan=1 colspan=1>Discrepant Analyte(s)</td></tr><tr><td rowspan=1 colspan=1>Adenovirus F40/F41</td><td rowspan=1 colspan=1>Campylobacter</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>N/A</td></tr><tr><td rowspan=1 colspan=1>Astrovirus</td><td rowspan=1 colspan=1>Giardia lamblia</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>Giardia lamblia</td></tr><tr><td rowspan=1 colspan=1>Campylobacter</td><td rowspan=1 colspan=1>Giardia lamblia</td><td rowspan=1 colspan=1>Shigella/EnteroinvasiveE. coli (EIEC)</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>Shigella/EnteroinvasiveE. coli (EIEC)</td></tr><tr><td rowspan=1 colspan=1>Campylobacter</td><td rowspan=1 colspan=1>Norovirus GI/GII</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>N/A</td></tr><tr><td rowspan=1 colspan=1>Campylobacter</td><td rowspan=1 colspan=1>Plesiomonas shigelloides</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>Plesiomonas shigelloides</td></tr><tr><td rowspan=1 colspan=1>Campylobacter</td><td rowspan=1 colspan=1>Rotavirus A</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>Rotavirus A</td></tr><tr><td rowspan=1 colspan=1>Enterotoxigenic E.coli (ETEC) lt/st</td><td rowspan=1 colspan=1>Norovirus GI/GII</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>Norovirus GI/GII</td></tr><tr><td rowspan=1 colspan=1>Norovirus GI/GII</td><td rowspan=1 colspan=1>Shigella/EnteroinvasiveE. coli (EIEC)</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>Norovirus GI/GII</td></tr><tr><td rowspan=1 colspan=1>Norovirus GI/GII</td><td rowspan=1 colspan=1>Yersinia enterocolitica</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>Yersinia enterocolitica</td></tr><tr><td rowspan=2 colspan=3>Total Multi-DetectionsDouble Detections</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=3>Triple Detections</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1></td></tr></table>

Table 46: Multi-Detections for Comparator Methods, FecalSwab   

<table><tr><td rowspan=1 colspan=1>Analyte 1</td><td rowspan=1 colspan=1>Analyte 2</td><td rowspan=1 colspan=1>Analyte 3</td><td rowspan=1 colspan=1># Multi-Detections</td><td rowspan=1 colspan=1>#FNanalytes byQIAStat</td><td rowspan=1 colspan=1>DiscrepantAnalyte(s)</td></tr><tr><td rowspan=1 colspan=1>Adenovirus F40/F41</td><td rowspan=1 colspan=1>Campylobacter</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>N/A</td></tr><tr><td rowspan=1 colspan=1>Adenovirus F40/F41</td><td rowspan=1 colspan=1>Norovirus GI/GII</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>Adenovirus F40/F41</td></tr><tr><td rowspan=1 colspan=1>Campylobacter</td><td rowspan=1 colspan=1>Shigella/EnteroinvasiveE. coli (EIEC)</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>N/A</td></tr><tr><td rowspan=1 colspan=1>Campylobacter</td><td rowspan=1 colspan=1>Norovirus GI/GII</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>N/A</td></tr><tr><td rowspan=1 colspan=1>Campylobacter</td><td rowspan=1 colspan=1>Norovirus GI/GII</td><td rowspan=1 colspan=1>Rotavirus A</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>Norovirus GI/GII</td></tr><tr><td rowspan=1 colspan=1>Campylobacter</td><td rowspan=1 colspan=1>Rotavirus A</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>N/A</td></tr><tr><td rowspan=1 colspan=1>Cryptosporidium</td><td rowspan=1 colspan=1>Norovirus GI/GII</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>Cryptosporidium</td></tr><tr><td rowspan=1 colspan=1>Enterotoxigenic E.coli (ETEC) lt/st</td><td rowspan=1 colspan=1>Norovirus GI/GII</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>N/A</td></tr><tr><td rowspan=2 colspan=3>Total Multi-DetectionsDouble Detections</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=3>Triple Detections</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1></td></tr></table>

Table 47: Multi-Detections for QIAstat-Dx Gastrointestinal Panel 2, Para-Pak C&S   

<table><tr><td colspan="3" rowspan="1">Distinct Multi-Detection Combinations Detected by QIAstatDx GI2 Panel</td><td colspan="3" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Analyte 1</td><td colspan="1" rowspan="1">Analyte 2</td><td colspan="1" rowspan="1">Analyte 3</td><td colspan="1" rowspan="1"># Multi-Detections</td><td colspan="1" rowspan="1"># FPanalytesbyQIAStat</td><td colspan="1" rowspan="1">Discrepant Analyte(s)</td></tr><tr><td colspan="1" rowspan="1">AdenovirusF40/F41</td><td colspan="1" rowspan="1">EnteropathogenicE. coli (EPEC)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">N/A</td></tr><tr><td colspan="1" rowspan="1">Astrovirus</td><td colspan="1" rowspan="1">Campylobacter</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">N/A</td></tr><tr><td colspan="1" rowspan="1">Campylobacter</td><td colspan="1" rowspan="1">EnteropathogenicE. coli (EPEC)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">N/A</td></tr><tr><td colspan="1" rowspan="1">Campylobacter</td><td colspan="1" rowspan="1">EnteropathogenicE. coli (EPEC)</td><td colspan="1" rowspan="1">Shigella/EnteroinvasiveE. coli (EIEC)</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Shigella/EnteroinvasiveE. coli (EIEC)</td></tr><tr><td colspan="1" rowspan="1">Campylobacter</td><td colspan="1" rowspan="1">Plesiomonasshigelloides</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Plesiomonasshigelloides</td></tr><tr><td colspan="1" rowspan="1">Campylobacter</td><td colspan="1" rowspan="1">Plesiomonasshigelloides</td><td colspan="1" rowspan="1">Shiga-like toxin E. coli(STEC) stx1/stx2</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Shiga-like toxin E. coli(STEC) stx1/stx2</td></tr><tr><td colspan="1" rowspan="1">Cryptosporidium</td><td colspan="1" rowspan="1">Cyclosporacayetanensis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Cryptosporidium</td></tr><tr><td colspan="1" rowspan="1">Cryptosporidium</td><td colspan="1" rowspan="1">Plesiomonasshigelloides</td><td colspan="1" rowspan="1">Shiga-like toxin E. coli(STEC) stx1/stx2</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">N/A</td></tr><tr><td colspan="1" rowspan="1">Cyclosporacayetanensis</td><td colspan="1" rowspan="1">EnteropathogenicE. coli (EPEC)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">N/A</td></tr><tr><td colspan="1" rowspan="1">EnteropathogenicE. coli (EPEC)</td><td colspan="1" rowspan="1">Enterotoxigenic E.coli (ETEC) lt/st</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">Enterotoxigenic E. coli(ETEC) lt/st</td></tr><tr><td colspan="1" rowspan="1">EnteropathogenicE. coli (EPEC)</td><td colspan="1" rowspan="1">Enterotoxigenic E.coli (ETEC) lt/st</td><td colspan="1" rowspan="1">Norovirus GI/GII</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Enterotoxigenic E. coli(ETEC) lt/st +Norovirus GI/GII</td></tr><tr><td colspan="1" rowspan="1">EnteropathogenicE. coli (EPEC)</td><td colspan="1" rowspan="1">Enterotoxigenic E.coli (ETEC) lt/st</td><td colspan="1" rowspan="1">Shigella/EnteroinvasiveE. coli (EIEC)</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">N/A</td></tr><tr><td colspan="1" rowspan="1">EnteropathogenicE. coli (EPEC)</td><td colspan="1" rowspan="1">Norovirus GI/GII</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">N/A</td></tr><tr><td colspan="1" rowspan="1">EnteropathogenicE. coli (EPEC)</td><td colspan="1" rowspan="1">Salmonella</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">N/A</td></tr><tr><td colspan="1" rowspan="1">EnteropathogenicE. coli (EPEC)</td><td colspan="1" rowspan="1">Yersiniaenterocolitica</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Yersinia enterocolitica</td></tr><tr><td colspan="1" rowspan="1">Enterotoxigenic E.coli (ETEC) lt/st</td><td colspan="1" rowspan="1">Plesiomonasshigelloides</td><td colspan="1" rowspan="1">Shiga-like toxin E. coli(STEC) stx1/stx2</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Plesiomonasshigelloides + Shiga-like toxin E. coli(STEC) stx1/stx2</td></tr><tr><td colspan="1" rowspan="1">Enterotoxigenic E.coli (ETEC) lt/st</td><td colspan="1" rowspan="1">Shiga-like toxin E.coli (STEC)stx 1/stx2</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Enterotoxigenic E. coli(ETEC) lt/st</td></tr><tr><td colspan="1" rowspan="1">Giardia lamblia</td><td colspan="1" rowspan="1">Norovirus GI/GII</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">N/A</td></tr><tr><td colspan="1" rowspan="1">Norovirus GI/GII</td><td colspan="1" rowspan="1">Plesiomonasshigelloides</td><td colspan="1" rowspan="1">Shigella/EnteroinvasiveE. coli (EIEC)</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">N/A</td></tr><tr><td colspan="1" rowspan="1">Norovirus GI/GII</td><td colspan="1" rowspan="1">Shiga-like toxin E.coli (STEC)stx1/stx2</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Norovirus GI/GII +Shiga-like toxin E. coli(STEC) stx1/stx2</td></tr><tr><td colspan="1" rowspan="1">Plesiomonasshigelloides</td><td colspan="1" rowspan="1">Rotavirus A</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">N/A</td></tr><tr><td colspan="1" rowspan="1">Plesiomonasshigelloides</td><td colspan="1" rowspan="1">Salmonella</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">N/A</td></tr><tr><td colspan="3" rowspan="2">Total Multi-DetectionsDouble Detections</td><td colspan="1" rowspan="1">29</td><td colspan="1" rowspan="1">12</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">22</td><td colspan="1" rowspan="1">8</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="3" rowspan="1">Triple Detections</td><td colspan="1" rowspan="1">7</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1"></td></tr></table>

Table 48: Multi-Detections for Comparator Methods, Para-Pak C&S   

<table><tr><td colspan="1" rowspan="1">Analyte 1</td><td colspan="1" rowspan="1">Analyte 2</td><td colspan="1" rowspan="1">Analyte 3</td><td colspan="1" rowspan="1"># Multi-Detections</td><td colspan="1" rowspan="1">#FNanalytes byQIAStat</td><td colspan="1" rowspan="1">DiscrepantAnalyte(s)</td></tr><tr><td colspan="1" rowspan="1">AdenovirusF40/F41</td><td colspan="1" rowspan="1">EnteropathogenicE. coli (EPEC)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">AdenovirusF40/F41</td></tr><tr><td colspan="1" rowspan="1">Astrovirus</td><td colspan="1" rowspan="1">Campylobacter</td><td colspan="1" rowspan="1">Enteropathogenic E.coli (EPEC)</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">EnteropathogenicE. coli (EPEC)</td></tr><tr><td colspan="1" rowspan="1">Campylobacter</td><td colspan="1" rowspan="1">EnteropathogenicE. coli (EPEC)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">N/A</td></tr><tr><td colspan="1" rowspan="1">Campylobacter</td><td colspan="1" rowspan="1">EnteropathogenicE. coli (EPEC)</td><td colspan="1" rowspan="1">Plesiomonasshigelloides</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">EnteropathogenicE. coli (EPEC)</td></tr><tr><td colspan="1" rowspan="1">Campylobacter</td><td colspan="1" rowspan="1">Enterotoxigenic E.coli (ETEC) lt/st</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Enterotoxigenic E.coli (ETEC) lt/st</td></tr><tr><td colspan="1" rowspan="1">Cryptosporidium</td><td colspan="1" rowspan="1">Plesiomonasshigelloides</td><td colspan="1" rowspan="1">Shiga-like toxin E.coli (STEC)stx1/stx2</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">N/A</td></tr><tr><td colspan="1" rowspan="1">Cyclosporacayetanensis</td><td colspan="1" rowspan="1">EnteropathogenicE. coli (EPEC)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">EnteropathogenicE. coli (EPEC)</td></tr><tr><td colspan="1" rowspan="1">Cyclosporacayetanensis</td><td colspan="1" rowspan="1">Enterotoxigenic E.coli (ETEC) lt/st</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Cyclosporacayetanensis</td></tr><tr><td colspan="1" rowspan="1">EnteropathogenicE. coli (EPEC)</td><td colspan="1" rowspan="1">Enterotoxigenic E.coli (ETEC) lt/st</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">N/A</td></tr><tr><td colspan="1" rowspan="1">EnteropathogenicE. coli (EPEC)</td><td colspan="1" rowspan="1">Enterotoxigenic E.coli (ETEC) lt/st</td><td colspan="1" rowspan="1">Shigella/Enteroinvasive E. coli (EIEC)</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">N/A</td></tr><tr><td colspan="1" rowspan="1">EnteropathogenicE. coli (EPEC)</td><td colspan="1" rowspan="1">Norovirus GI/GII</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">EnteropathogenicE. coli (EPEC)</td></tr><tr><td colspan="1" rowspan="1">EnteropathogenicE. coli (EPEC)</td><td colspan="1" rowspan="1">Plesiomonasshigelloides</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">EnteropathogenicE. coli (EPEC)</td></tr><tr><td colspan="1" rowspan="1">EnteropathogenicE. coli (EPEC)</td><td colspan="1" rowspan="1">Salmonella</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">N/A</td></tr><tr><td colspan="1" rowspan="1">Enterotoxigenic E.coli (ETEC) lt/st</td><td colspan="1" rowspan="1">Salmonella</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Salmonella</td></tr><tr><td colspan="1" rowspan="1">Giardia lamblia</td><td colspan="1" rowspan="1">Norovirus GI/GII</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">N/A</td></tr><tr><td colspan="1" rowspan="1">Norovirus GI/GII</td><td colspan="1" rowspan="1">Plesiomonasshigelloides</td><td colspan="1" rowspan="1">Shigella/Enteroinvasive E. coli (EIEC)</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">N/A</td></tr><tr><td colspan="1" rowspan="1">Plesiomonasshigelloides</td><td colspan="1" rowspan="1">Rotavirus A</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">N/A</td></tr><tr><td colspan="1" rowspan="1">Plesiomonasshigelloides</td><td colspan="1" rowspan="1">Salmonella</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">N/A</td></tr><tr><td colspan="3" rowspan="2">Total Multi-DetectionsDouble Detections</td><td colspan="1" rowspan="1">27</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">22</td><td colspan="1" rowspan="1">8</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="3" rowspan="1">Triple Detections</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1"></td></tr></table>

Prospective Archived Specimen Testing:

Additional prospectively archived stool specimens were collected for Norovirus GI/GII (81 specimens) and STEC (18 specimens). Specimens were prospectively collected from four different sites (3 US and 1 EU). All positive specimens (i.e., all-comer positives) by standard of care testing for Norovirus (GI/GII) or STEC were archived for inclusion in the study. An additional 20 negative specimens were included in the study for blinding purposes. Testing included a total of 119 total specimens. Of the initial 81 specimens positive for Norovirus by standard of care at the collection site, 77 specimens were determined to be positive for Norovirus by the composite comparator method. Of the initial 18 specimens positive for STEC by standard of care, 13 specimens were determined to be positive by the composite comparator method. The remaining specimens that were not confirmed to be positive by the composite comparator were included in the analysis as negative specimens. Results from the prospective archived specimen testing are presented in Table 49.

Table 49: QIAstat-Dx Gastrointestinal Panel 2, Clinical Performance, Prospective Archived Specimen Testing   

<table><tr><td rowspan=1 colspan=2></td><td rowspan=1 colspan=3>Positive Percent Agreement</td><td rowspan=1 colspan=3>Negative Percent Agreement</td></tr><tr><td rowspan=1 colspan=1>Analyte</td><td rowspan=1 colspan=1>MediumBrand</td><td rowspan=1 colspan=1>TP/TP+FN</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=1>TN/TN+FP</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=2 colspan=1>Norovirus GI/GII</td><td rowspan=1 colspan=1>FecalSwab</td><td rowspan=1 colspan=1>48 / 49 1</td><td rowspan=1 colspan=1>98.0</td><td rowspan=1 colspan=1>89.3 - 99.6</td><td rowspan=1 colspan=1>2/4</td><td rowspan=1 colspan=1>50.0</td><td rowspan=1 colspan=1>15.0-85.0</td></tr><tr><td rowspan=1 colspan=1>Para-Pak C&amp;S</td><td rowspan=1 colspan=1>26 / 28 2</td><td rowspan=1 colspan=1>92.9</td><td rowspan=1 colspan=1>77.4 - 98.0</td><td rowspan=1 colspan=1>37 /38</td><td rowspan=1 colspan=1>97.4</td><td rowspan=1 colspan=1>86.5-99.5</td></tr><tr><td rowspan=1 colspan=1>Shiga-like toxin E. coli(STEC) stx1/stx2</td><td rowspan=1 colspan=1>Para-Pak C&amp;S</td><td rowspan=1 colspan=1>12 / 13 3,4</td><td rowspan=1 colspan=1>92.3</td><td rowspan=1 colspan=1>66.7 - 98.6</td><td rowspan=1 colspan=1>51 /52</td><td rowspan=1 colspan=1>98.1</td><td rowspan=1 colspan=1>89.9-99.7</td></tr></table>

1 For Norovirus GI/GII four out of the eighty-one (4/81) prospectively archived samples (positive by standard of care) were negative by the composite comparator and therefore included as negative samples in the NPA calculations

2 One (1) Para-Pak C&S sample negative by QIAstat-Dx and positive with one FDA-cleared comparator with insufficient volume for complete composite comparator testing was classed as false negative in the PPA calculations.

3 For STEC five out of the eighteen (5/18) prospectively archived samples (positive by standard of care) were negative by the composite comparator and therefore included as negative samples in the NPA calculations.

4 One (1) Para-Pak C&S sample positive for STEC in both QIAstat-Dx and one FDA-cleared comparator was excluded from the PPA calculations because the samples did not have sufficient volume for complete composite comparator testing.

# Retrospective Archived Study (Pre-selected specimens)

To supplement the results of the prospective clinical study, a total of 750 preselected frozen archived retrospective specimens were evaluated. These specimens served to increase the sample size for analytes that showed low prevalence in the prospective study or that were less represented in a particular sample type (Para-Pak C&S or FecalSwab). Specimens were selected based on standard of care result and were excluded from the data analysis if the comparator methods did not confirm the preselected target as positive. Performance for the QIAstat Gastrointestinal Panel 2 for retrospectively tested archived specimens is presented in Table 50.

Table 50. Clinical Performance in the Retrospective study   

<table><tr><td colspan="2" rowspan="1"></td><td colspan="3" rowspan="1">Positive Percent Agreement</td><td colspan="3" rowspan="1">Negative Percent Agreement</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">MediumBrand</td><td colspan="1" rowspan="1">TP/TP+FN</td><td colspan="1" rowspan="1">%</td><td colspan="1" rowspan="1">95% CI</td><td colspan="1" rowspan="1">TN/TN+FP</td><td colspan="1" rowspan="1">%</td><td colspan="1" rowspan="1">95% CI</td></tr><tr><td colspan="8" rowspan="1">Viruses</td></tr><tr><td colspan="1" rowspan="2">Adenovirus F40/F41</td><td colspan="1" rowspan="1">FecalSwab</td><td colspan="1" rowspan="1">23 / 261</td><td colspan="1" rowspan="1">88.5</td><td colspan="1" rowspan="1">71.0-96.0</td><td colspan="1" rowspan="1">203 / 203</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">98.1-100</td></tr><tr><td colspan="1" rowspan="1">Para-Pak C&amp;S</td><td colspan="1" rowspan="1">29 / 29</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">88.3-100</td><td colspan="1" rowspan="1">39 / 39</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">91.0-100</td></tr><tr><td colspan="1" rowspan="2">Astrovirus</td><td colspan="1" rowspan="1">FecalSwab</td><td colspan="1" rowspan="1">2/32</td><td colspan="1" rowspan="1">66.7</td><td colspan="1" rowspan="1">20.8-93.9</td><td colspan="1" rowspan="1">191 / 191</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">98.0-100</td></tr><tr><td colspan="1" rowspan="1">Para-Pak C&amp;S</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">14/14</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">78.5-100</td></tr><tr><td colspan="1" rowspan="2">NorovirusGI/GII</td><td colspan="1" rowspan="1">FecalSwab</td><td colspan="1" rowspan="1">28 / 32 3</td><td colspan="1" rowspan="1">87.5</td><td colspan="1" rowspan="1">71.9-95.0</td><td colspan="1" rowspan="1">74 /75 3</td><td colspan="1" rowspan="1">98.7</td><td colspan="1" rowspan="1">92.8-99.8</td></tr><tr><td colspan="1" rowspan="1">Para-Pak C&amp;S</td><td colspan="1" rowspan="1">27 / 29</td><td colspan="1" rowspan="1">93.1</td><td colspan="1" rowspan="1">78.0-98.1</td><td colspan="1" rowspan="1">86 / 86 4</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">95.7-100</td></tr><tr><td colspan="1" rowspan="2">Rotavirus A</td><td colspan="1" rowspan="1">FecalSwab</td><td colspan="1" rowspan="1">8/95</td><td colspan="1" rowspan="1">88.9</td><td colspan="1" rowspan="1">56.5-98.0</td><td colspan="1" rowspan="1">185 / 185</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">98.0-100</td></tr><tr><td colspan="1" rowspan="1">Para-Pak C&amp;S</td><td colspan="1" rowspan="1">2/26</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">34.2-100</td><td colspan="1" rowspan="1">12 / 12</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">75.8-100</td></tr><tr><td colspan="3" rowspan="1"></td><td colspan="5" rowspan="1">Bacteria</td></tr><tr><td colspan="1" rowspan="2">Campylobacter</td><td colspan="1" rowspan="1">FecalSwab</td><td colspan="1" rowspan="1">31/31</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">89.0-100</td><td colspan="1" rowspan="1">161 / 163 6</td><td colspan="1" rowspan="1">98.8</td><td colspan="1" rowspan="1">95.6-99.7</td></tr><tr><td colspan="1" rowspan="1">Para-Pak C&amp;S</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">43.9-100</td><td colspan="1" rowspan="1">11/11</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">74.1-100</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">FecalSwab</td><td colspan="1" rowspan="1">2/2</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">34.2-100</td><td colspan="1" rowspan="1">192 / 192</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">98.0-100</td></tr><tr><td colspan="1" rowspan="1">Plesiomonasshigelloides</td><td colspan="1" rowspan="1">Para-Pak C&amp;S</td><td colspan="1" rowspan="1">33 / 367</td><td colspan="1" rowspan="1">91.7</td><td colspan="1" rowspan="1">78.2-97.1</td><td colspan="1" rowspan="1">117/117</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">96.8-100</td></tr><tr><td colspan="1" rowspan="2">Salmonella</td><td colspan="1" rowspan="1">FecalSwab</td><td colspan="1" rowspan="1">30 /318</td><td colspan="1" rowspan="1">96.8</td><td colspan="1" rowspan="1">83.8-99.4</td><td colspan="1" rowspan="1">161 / 163 8</td><td colspan="1" rowspan="1">98.8</td><td colspan="1" rowspan="1">95.6-99.7</td></tr><tr><td colspan="1" rowspan="1">Para-Pak C&amp;S</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">20.7-100</td><td colspan="1" rowspan="1">13 / 13</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">77.2-100</td></tr><tr><td colspan="1" rowspan="3">Yersiniaenterocolitica</td><td colspan="1" rowspan="1">FecalSwab</td><td colspan="1" rowspan="1">32 / 349</td><td colspan="1" rowspan="1">94.1</td><td colspan="1" rowspan="1">80.9-98.4</td><td colspan="1" rowspan="1">160 / 160</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">97.7-100</td></tr><tr><td colspan="1" rowspan="1">Para-Pak C&amp;S</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">20.7-100</td><td colspan="1" rowspan="1">14/14</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">78.5-100</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="8" rowspan="1">Diarrheagenic E. coli/Shigella</td></tr><tr><td colspan="1" rowspan="1">Enteropathogenic E. coli(EPEC)</td><td colspan="1" rowspan="1">Para-Pak C&amp;S</td><td colspan="1" rowspan="1">60 / 6510</td><td colspan="1" rowspan="1">92.3</td><td colspan="1" rowspan="1">83.2-96.7</td><td colspan="1" rowspan="1">42 / 42</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">91.6-100</td></tr><tr><td colspan="1" rowspan="2">EnterotoxigenicE. coli (ETEC)It/st</td><td colspan="1" rowspan="1">FecalSwab</td><td colspan="1" rowspan="1">22 / 24 11</td><td colspan="1" rowspan="1">91.7</td><td colspan="1" rowspan="1">74.2-97.7</td><td colspan="1" rowspan="1">85 / 86 11</td><td colspan="1" rowspan="1">98.8</td><td colspan="1" rowspan="1">93.7-99.8</td></tr><tr><td colspan="1" rowspan="1">Para-Pak C&amp;S</td><td colspan="1" rowspan="1">23 / 24</td><td colspan="1" rowspan="1">95.8</td><td colspan="1" rowspan="1">79.8-99.3</td><td colspan="1" rowspan="1">61 / 61 12</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">94.1-100</td></tr><tr><td colspan="1" rowspan="1">Shiga-like toxinE. coli (STEC)stx1/stx2</td><td colspan="1" rowspan="1">Para-Pak C&amp;S</td><td colspan="1" rowspan="1">60 / 64</td><td colspan="1" rowspan="1">93.8</td><td colspan="1" rowspan="1">85.0-97.5</td><td colspan="1" rowspan="1">44 / 44 13</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">92.0-100</td></tr><tr><td colspan="1" rowspan="1">E. coli O157</td><td colspan="1" rowspan="1">Para-Pak C&amp;S</td><td colspan="1" rowspan="1">39 / 42 14</td><td colspan="1" rowspan="1">$2.9</td><td colspan="1" rowspan="1">80.1-99.4</td><td colspan="1" rowspan="1">16 / 16</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">80.6-100</td></tr><tr><td colspan="1" rowspan="2">Shigella/Enteroinvasive E. coli(EIEC)</td><td colspan="1" rowspan="1">FecalSwab</td><td colspan="1" rowspan="1">22 / 24 15</td><td colspan="1" rowspan="1">91.7</td><td colspan="1" rowspan="1">74.2-97.7</td><td colspan="1" rowspan="1">170 / 170</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">97.8-100</td></tr><tr><td colspan="1" rowspan="1">Para-Pak C&amp;S</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">14 /14</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">78.5-100</td></tr><tr><td colspan="8" rowspan="1">Parasites</td></tr><tr><td colspan="1" rowspan="2">Cryptosporidium</td><td colspan="1" rowspan="1">FecalSwab</td><td colspan="1" rowspan="1">6/6</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">61-100</td><td colspan="1" rowspan="1">186 / 188 16</td><td colspan="1" rowspan="1">98.9</td><td colspan="1" rowspan="1">96.2-99.7</td></tr><tr><td colspan="1" rowspan="1">Para-Pak C&amp;S</td><td colspan="1" rowspan="1">26 / 26</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">87.1-100</td><td colspan="1" rowspan="1">117/117</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">96.8-100</td></tr><tr><td colspan="1" rowspan="2">Cyclosporaetanensis</td><td colspan="1" rowspan="1">FecalSwab</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">20.7-100</td><td colspan="1" rowspan="1">193 / 193</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">98.1-100</td></tr><tr><td colspan="1" rowspan="1">Para-Pak C&amp;S</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">20.7-100</td><td colspan="1" rowspan="1">13 / 13</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">77.2-100</td></tr><tr><td colspan="1" rowspan="2">Entamoeba histlytica</td><td colspan="1" rowspan="1">FecalSwab</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">194 / 194</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">98.1-100</td></tr><tr><td colspan="1" rowspan="1">Para-Pak C&amp;S</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">14/14</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">76.5-100</td></tr><tr><td colspan="1" rowspan="2">Giardia lamblia</td><td colspan="1" rowspan="1">FecalSwab</td><td colspan="1" rowspan="1">29 / 3117</td><td colspan="1" rowspan="1">93.6</td><td colspan="1" rowspan="1">79.3-98.2</td><td colspan="1" rowspan="1">46 / 48 18</td><td colspan="1" rowspan="1">95.8</td><td colspan="1" rowspan="1">86.0-98.9</td></tr><tr><td colspan="1" rowspan="1">Para-Pak C&amp;S</td><td colspan="1" rowspan="1">27 / 28 17</td><td colspan="1" rowspan="1">96.4</td><td colspan="1" rowspan="1">82.3-99.4</td><td colspan="1" rowspan="1">92 / 92 17</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">96.0-100</td></tr></table>

Adenovirus F40/41 was detected in one of the three false negatives (1/3) in FecalSwab using a different FDA-cleared test method

Astrovirus was detected in the single false negative specimen (1/1) in FecalSwab using a different FDA-cleared test method.

3 Two (2) FecalSwab samples positive for Norovirus GI/GII by both QIAstat-Dx and one FDA-cleared comparator were excluded from the PPA calculations because the samples did not have sufficient volume for complete composite comparator testing. The sample size for NPA is smaller for Norovirus GI/GII as only a portion of the samples with a negative result by QIAstat-Dx and by one FDA-cleared comparator was tested with the complete composite comparator in the retrospective study.

4 The sample size for NPA is smaller for Norovirus GI/GII in Para-Pak C&S as only a portion of the samples with a negative result by QIAstat-Dx and by one FDA-cleared comparator was tested with the complete composite comparator in the retrospective study

5 Rotavirus A was detected in the single false negative specimen (1/1) in FecalSwab using a different FDA-cleared test method.

6 Campylobacter was detected in one of the two false positive specimens (1/2) in FecalSwab using a different FDA-cleared test method.

7 Plesiomonas shigelloides was detected in one of the three false negative specimens (1/3) in Para-Pak C&S using a different FDAcleared test method.

8 Salmonella was not detected in the single false negative specimen $( 0 / 1 )$ and was not detected in the two false positive specimens (0/2) in FecalSwab using a different FDA-cleared test method.

9 Yersinia enterocolitica was not detected in the two false negative specimens (0/2) in FecalSwab using a different FDA-cleared test method.

10 Enteropathogenic E. coli (EPEC) was detected in all three false negative specimens (3/3) in Para-Pak C&S using PCR followed by bi-directional sequence analysis. There were two (2) other false negative specimens that were not further investigated by discrepant analyses.

11 Ten (10) FecalSwab samples positive for ETEC in both QIAstat-Dx and one FDA-cleared comparator were excluded from the PPA calculations because the samples did not have sufficient volume for complete composite comparator testing. One (1) FecalSwab sample negative by QIAstat-Dx and positive with one FDA-cleared comparator with insufficient volume for complete composite comparator testing was classed as false negative in the PPA calculations. The sample size for NPA is smaller for ETEC as only a portion of the samples with a negative result by QIAstat-Dx and one FDA-cleared comparator was tested with the complete composite comparator in the retrospective study.

12 The sample size for NPA is smaller for ETEC in Para-Pak CS&S as only a portion of the samples with a negative result by QIAstat-Dx and one FDA-cleared comparator was tested with the complete composite comparator in the retrospective study).

13 The sample size for NPA is smaller for STEC in Para-Pak C&S as only a portion of the samples with a negative result by QIAstatDx and in one FDA-cleared comparator was tested with the complete composite comparator in the retrospective study1414 Shigella/Enteroinvasive $E$ . coli (EIEC) was detected in one of the two false negative specimens (1/2) in FecalSwab using a different FDA-cleared test method.

$^ { 1 4 } E$ . coli O157 was not detected in two false negative specimens (0/2) in Para-Pak C&S using a different FDA-cleared test method.   
There was one (1) false negative specimen in Para-Pak C&S that was not further investigated by discrepant analyses.

15 Shigella/Enteroinvasive $E$ . coli (EIEC) was detected in one of the two false negative specimens (1/2) in FecalSwab using a different FDA-cleared test method.

16Cryptosporidium was not detected in the two false positive specimens (0/2) in FecalSwab using PCR followed by bi-directional sequence analysis

17 Four (4) samples positive for Giardia lamblia by both QIAstat-Dx and one FDA-cleared comparator were excluded from the PPA calculations because the samples did not have sufficient volume for complete composite comparator testing. Two (2) FecalSwab samples negative by QIAstat-Dx and positive with one FDA-cleared comparator with insufficient volume for complete composite comparator testing was classed as false negative in the PPA calculations. The sample size for NPA is smaller for Giardia lamblia as only a portion of the samples with a negative result by QIAstat-Dx and by one FDA-cleared comparator was tested with the complete composite comparator in the retrospective study.

18 One (1) Para-Pak C&S samples positive for Giardia lamblia in both QIAstat-Dx and primary FDA-cleared comparator (were excluded from the PPA calculations because the samples did not have sufficient volume for complete composite comparator testing. One (1) Para-Pak C&S sample negative by QIAstat-Dx and positive with one FDA-cleared comparator with insufficient volume for complete composite comparator testing was classed as false negative in the PPA calculations. The sample size for NPA is smaller for Giardia lamblia as only a portion of the samples with a negative result by QIAstat-Dx and by one FDA-cleared comparator was tested with the complete composite comparator in the retrospective study.

For the clinical study, the proportion of failed or invalid runs on initial attempt and following repeat testing are summarized in Table 51. The failure rates due to instrument, invalid result (IC failure), ‘sample too concentrated’ and other run failures are summarized in Table 52.

Table 51: Failure Rates Summary   

<table><tr><td colspan="1" rowspan="2">Transportmedium</td><td colspan="1" rowspan="2">Study</td><td colspan="3" rowspan="1">Initial Runs</td><td colspan="3" rowspan="1">Final Runs</td></tr><tr><td colspan="1" rowspan="1">N / Total</td><td colspan="1" rowspan="1">%</td><td colspan="1" rowspan="1">95% CI</td><td colspan="1" rowspan="1">N / Total</td><td colspan="1" rowspan="1">%</td><td colspan="1" rowspan="1">95% CI</td></tr><tr><td colspan="1" rowspan="4">FecalSwab</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">16 / 1227</td><td colspan="1" rowspan="1">1.3</td><td colspan="1" rowspan="1">0.7 - 2.1</td><td colspan="1" rowspan="1">3 / 1227</td><td colspan="1" rowspan="1">0.2</td><td colspan="1" rowspan="1">0.1 - 0.7</td></tr><tr><td colspan="1" rowspan="1">ProspectiveArchived</td><td colspan="1" rowspan="1">0 /53</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">0.0 - 6.7</td><td colspan="1" rowspan="1">0 /53</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">0.0 - 6.7</td></tr><tr><td colspan="1" rowspan="1">Retrospective</td><td colspan="1" rowspan="1">11 / 366</td><td colspan="1" rowspan="1">3.0</td><td colspan="1" rowspan="1">1.5 - 5.3</td><td colspan="1" rowspan="1">5 / 366</td><td colspan="1" rowspan="1">1.4</td><td colspan="1" rowspan="1">0.4 - 3.2</td></tr><tr><td colspan="1" rowspan="1">Total</td><td colspan="1" rowspan="1">27 / 1646</td><td colspan="1" rowspan="1">1.6</td><td colspan="1" rowspan="1">1.1 - 2.4</td><td colspan="1" rowspan="1">8 / 1646</td><td colspan="1" rowspan="1">0.5</td><td colspan="1" rowspan="1">0.2 - 1.0</td></tr><tr><td colspan="1" rowspan="1">Para-Pak</td><td colspan="1" rowspan="1">Prospective</td><td colspan="1" rowspan="1">66 / 740</td><td colspan="1" rowspan="1">8.9</td><td colspan="1" rowspan="1">7.0 - 11.2</td><td colspan="1" rowspan="1">21 / 740</td><td colspan="1" rowspan="1">2.8</td><td colspan="1" rowspan="1">1.8 - 4.3</td></tr><tr><td colspan="1" rowspan="3"></td><td colspan="1" rowspan="1">ProspectiveArchived</td><td colspan="1" rowspan="1">3 / 66</td><td colspan="1" rowspan="1">4.5</td><td colspan="1" rowspan="1">0.9 - 12.7</td><td colspan="1" rowspan="1">0 / 66</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">0.0 - 5.4</td></tr><tr><td colspan="1" rowspan="1">Retrospective</td><td colspan="1" rowspan="1">46 / 454</td><td colspan="1" rowspan="1">10.1</td><td colspan="1" rowspan="1">7.5 - 13.3</td><td colspan="1" rowspan="1">25 / 454</td><td colspan="1" rowspan="1">5.5</td><td colspan="1" rowspan="1">3.6 - 8.0</td></tr><tr><td colspan="1" rowspan="1">Total</td><td colspan="1" rowspan="1">115 / 1260</td><td colspan="1" rowspan="1">9.1</td><td colspan="1" rowspan="1">7.6 - 10.9</td><td colspan="1" rowspan="1">46 / 1260</td><td colspan="1" rowspan="1">3.7</td><td colspan="1" rowspan="1">2.7 - 4.8</td></tr></table>

Table 52: Summary of Run Failure Types   

<table><tr><td rowspan=2 colspan=1>TransportMedium</td><td rowspan=2 colspan=1>Study</td><td rowspan=2 colspan=1>Failure Reason</td><td rowspan=1 colspan=2>Initial Runs</td><td rowspan=1 colspan=2>Final Runs</td></tr><tr><td rowspan=1 colspan=1>N / Total</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>N / Total</td><td rowspan=1 colspan=1>%</td></tr><tr><td rowspan=12 colspan=1>FecalSwab</td><td rowspan=4 colspan=1>Prospective</td><td rowspan=1 colspan=1>Instrument</td><td rowspan=1 colspan=1>0 / 1227</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0 /1227</td><td rowspan=1 colspan=1>0.0</td></tr><tr><td rowspan=1 colspan=1>Invalid1</td><td rowspan=1 colspan=1>0 / 1227</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0 / 1227</td><td rowspan=1 colspan=1>0.0</td></tr><tr><td rowspan=1 colspan=1>Sample tooConcentrated 2</td><td rowspan=1 colspan=1>5 / 1227</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>0 / 1227</td><td rowspan=1 colspan=1>0.0</td></tr><tr><td rowspan=1 colspan=1>Other 3</td><td rowspan=1 colspan=1>11 / 1227</td><td rowspan=1 colspan=1>0.9</td><td rowspan=1 colspan=1>3 / 1227</td><td rowspan=1 colspan=1>0.2</td></tr><tr><td rowspan=4 colspan=1>ProspectiveArchived</td><td rowspan=1 colspan=1>Instrument</td><td rowspan=1 colspan=1>0 /53</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0 /53</td><td rowspan=1 colspan=1>0.0</td></tr><tr><td rowspan=1 colspan=1>Invalid</td><td rowspan=1 colspan=1>0 /53</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0 /53</td><td rowspan=1 colspan=1>0.0</td></tr><tr><td rowspan=1 colspan=1>Sample tooConcentrated</td><td rowspan=1 colspan=1>0 /53</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0 / 53</td><td rowspan=1 colspan=1>0.0</td></tr><tr><td rowspan=1 colspan=1>Other</td><td rowspan=1 colspan=1>0 /53</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0 / 53</td><td rowspan=1 colspan=1>0.0</td></tr><tr><td rowspan=4 colspan=1>Retrospective</td><td rowspan=1 colspan=1>Instrument</td><td rowspan=1 colspan=1>1 / 366</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>0 / 366</td><td rowspan=1 colspan=1>0.0</td></tr><tr><td rowspan=1 colspan=1>Invalid</td><td rowspan=1 colspan=1>1 / 366</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>0 / 366</td><td rowspan=1 colspan=1>0.0</td></tr><tr><td rowspan=1 colspan=1>Sample tooConcentrated</td><td rowspan=1 colspan=1>0 / 366</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0 / 366</td><td rowspan=1 colspan=1>0.0</td></tr><tr><td rowspan=1 colspan=1>Other</td><td rowspan=1 colspan=1>9/366</td><td rowspan=1 colspan=1>2.5</td><td rowspan=1 colspan=1>5 / 366</td><td rowspan=1 colspan=1>1.4</td></tr><tr><td rowspan=12 colspan=1>Para-PakC&amp;S</td><td rowspan=4 colspan=1>Prospective</td><td rowspan=1 colspan=1>Instrument</td><td rowspan=1 colspan=1>9/740</td><td rowspan=1 colspan=1>1.2</td><td rowspan=1 colspan=1>2/740</td><td rowspan=1 colspan=1>0.3</td></tr><tr><td rowspan=1 colspan=1>Invalid</td><td rowspan=1 colspan=1>5/740</td><td rowspan=1 colspan=1>0.7</td><td rowspan=1 colspan=1>5 / 740</td><td rowspan=1 colspan=1>0.7</td></tr><tr><td rowspan=1 colspan=1>Sample tooConcentrated</td><td rowspan=1 colspan=1>35 / 740</td><td rowspan=1 colspan=1>4.7</td><td rowspan=1 colspan=1>7 / 740</td><td rowspan=1 colspan=1>0.9</td></tr><tr><td rowspan=1 colspan=1>Other</td><td rowspan=1 colspan=1>17 / 740</td><td rowspan=1 colspan=1>2.3</td><td rowspan=1 colspan=1>7/740</td><td rowspan=1 colspan=1>0.9</td></tr><tr><td rowspan=4 colspan=1>ProspectiveArchived</td><td rowspan=1 colspan=1>Instrument</td><td rowspan=1 colspan=1>0 /66</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0 / 66</td><td rowspan=1 colspan=1>0.0</td></tr><tr><td rowspan=1 colspan=1>Invalid</td><td rowspan=1 colspan=1>1/66</td><td rowspan=1 colspan=1>1.5</td><td rowspan=1 colspan=1>0 / 66</td><td rowspan=1 colspan=1>0.0</td></tr><tr><td rowspan=1 colspan=1>Sample tooConcentrated</td><td rowspan=1 colspan=1>1 / 66</td><td rowspan=1 colspan=1>1.5</td><td rowspan=1 colspan=1>0 / 66</td><td rowspan=1 colspan=1>0.0</td></tr><tr><td rowspan=1 colspan=1>Other</td><td rowspan=1 colspan=1>1 /66</td><td rowspan=1 colspan=1>1.5</td><td rowspan=1 colspan=1>0 /66</td><td rowspan=1 colspan=1>0.0</td></tr><tr><td rowspan=4 colspan=1>Retrospective</td><td rowspan=1 colspan=1>Instrument</td><td rowspan=1 colspan=1>1/454</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>0 / 454</td><td rowspan=1 colspan=1>0.0</td></tr><tr><td rowspan=1 colspan=1>Invalid</td><td rowspan=1 colspan=1>10 / 454</td><td rowspan=1 colspan=1>2.2</td><td rowspan=1 colspan=1>6 / 454</td><td rowspan=1 colspan=1>1.3</td></tr><tr><td rowspan=1 colspan=1>Sample tooConcentrated</td><td rowspan=1 colspan=1>10 / 454</td><td rowspan=1 colspan=1>2.2</td><td rowspan=1 colspan=1>2 / 454</td><td rowspan=1 colspan=1>0.4</td></tr><tr><td rowspan=1 colspan=1>Other</td><td rowspan=1 colspan=1>25 /454</td><td rowspan=1 colspan=1>5.5</td><td rowspan=1 colspan=1>17 / 454</td><td rowspan=1 colspan=1>3.7</td></tr></table>

a Run failures re a Internal Control failures with at least one analyte detected and the other analytes reported as ‘invalid’ b Run failures related to ‘sample concentration too high’. These specimens were repeated with 100 microliters as instructed in the package insert

# Contrived Specimen Testing

Prospective and retrospective specimen testing generated low numbers of positive specimens for at least one specimen type for Astrovirus, Rotavirus A, Plesiomonas shigelloides, Yersinia enterocolitica, Shigella/EIEC, Cryptosporidium, Cyclospora cayetanensis and Entamoeba histolytica. To supplement the prospective and retrospective specimens test results for these targets, an evaluation of contrived specimens was performed. Each contrived specimen was prepared using an individual residual stool specimen matrix in Para-Pak C&S transport medium that had previously tested negative by QIAstat-Dx Gastrointestinal Panel 2 and comparator methods. At least, $50 \%$ of the contrived specimens were spiked at concentrations slightly above the Limit of Detection (2x LoD) and the remainder were spiked at concentrations of ${ 5 } \mathrm { x }$ and $1 0 \mathrm { x }$ LoD, using quantified strains for each pathogen. A minimum of 50 contrived specimens were tested for each analyte evaluated. The analyte status of each contrived specimen was blinded to the operators conducting the testing. Results are summarized in Table 53.

Table 53. Test Results Summary for Contrived Specimens   

<table><tr><td rowspan=2 colspan=1>Assay Target</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=3>Positive Percent Agreement (PPA)</td></tr><tr><td rowspan=1 colspan=1>Medium Brand</td><td rowspan=1 colspan=1>Fraction</td><td rowspan=1 colspan=1>Percentage</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=2 colspan=1>Astrovirus</td><td rowspan=1 colspan=1>FecalSwab</td><td rowspan=1 colspan=1>33 / 34</td><td rowspan=1 colspan=1>97.1</td><td rowspan=1 colspan=1>85.1-99.5</td></tr><tr><td rowspan=1 colspan=1>Para-Pak C&amp;S</td><td rowspan=1 colspan=1>34 / 34</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>89.8-100</td></tr><tr><td rowspan=2 colspan=1>Rotavirus A</td><td rowspan=1 colspan=1>FecalSwab</td><td rowspan=1 colspan=1>35 / 35</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>90.1-100</td></tr><tr><td rowspan=1 colspan=1>Para-Pak C&amp;S</td><td rowspan=1 colspan=1>34 / 35</td><td rowspan=1 colspan=1>97.1</td><td rowspan=1 colspan=1>85.5-99.5</td></tr><tr><td rowspan=2 colspan=1>Plesiomonasshigelloides</td><td rowspan=1 colspan=1>FecalSwab</td><td rowspan=1 colspan=1>33 / 33</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>89.6-100</td></tr><tr><td rowspan=1 colspan=1>Para-Pak C&amp;S</td><td rowspan=1 colspan=1>34 / 35</td><td rowspan=1 colspan=1>97.1</td><td rowspan=1 colspan=1>85.5-99.5</td></tr><tr><td rowspan=2 colspan=1>Yersinia enterocolitica</td><td rowspan=1 colspan=1>FecalSwab</td><td rowspan=1 colspan=1>34 / 34</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>89.8-100</td></tr><tr><td rowspan=1 colspan=1>Para-Pak C&amp;S</td><td rowspan=1 colspan=1>34 / 35</td><td rowspan=1 colspan=1>97.1</td><td rowspan=1 colspan=1>85.5-99.5</td></tr><tr><td rowspan=2 colspan=1>Shigella/EIEC</td><td rowspan=1 colspan=1>FecalSwab</td><td rowspan=1 colspan=1>35 / 35</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>90.1-100</td></tr><tr><td rowspan=1 colspan=1>Para-Pak C&amp;S</td><td rowspan=1 colspan=1>34 / 34</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>89.8-100</td></tr><tr><td rowspan=2 colspan=1>Cryptosporidium</td><td rowspan=1 colspan=1>FecalSwab</td><td rowspan=1 colspan=1>27 /27</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>87.5-100</td></tr><tr><td rowspan=1 colspan=1>Para-Pak C&amp;S</td><td rowspan=1 colspan=1>31/31</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>89.0-100</td></tr><tr><td rowspan=2 colspan=1>Cyclospora cayetanensis</td><td rowspan=1 colspan=1>FecalSwab</td><td rowspan=1 colspan=1>26 / 26</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>87.1-100</td></tr><tr><td rowspan=1 colspan=1>Para-Pak C&amp;S</td><td rowspan=1 colspan=1>30 / 30</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>88.6-100</td></tr><tr><td rowspan=2 colspan=1>Entamoeba histolytica</td><td rowspan=1 colspan=1>FecalSwab</td><td rowspan=1 colspan=1>35 / 35</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>90.1-100</td></tr><tr><td rowspan=1 colspan=1>Para-Pak C&amp;S</td><td rowspan=1 colspan=1>34 / 35</td><td rowspan=1 colspan=1>97.1</td><td rowspan=1 colspan=1>85.5-99.5</td></tr></table>

# D Clinical Cut-Off:

Not applicable.

# E Expected Values/Reference Range:

The number and percentage of positive results as determined by the QIAstat-Dx Gastrointestinal Panel 2 in the prospective clinical evaluation, stratified by age group, are presented in Table 54. Overall, the QIAstat-Dx Gastrointestinal Panel 2 detected at least one organism in $1 7 . 4 \%$ (213/1222) and $2 3 . 8 \%$ (171/717) of the prospectively collected stool specimens in FecalSwab and Para-Pak C&S, respectively.

Table 54: Expected values Summary by Age Group for the Prospective Clinical study as determined by the QIAstat-Dx Gastrointestinal Panel 2   

<table><tr><td colspan="1" rowspan="1">Pathogen</td><td colspan="1" rowspan="1">Medium Brand</td><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">0-5 years</td><td colspan="1" rowspan="1">6-21 years</td><td colspan="1" rowspan="1">22-49 years</td><td colspan="1" rowspan="1">50+ years</td><td colspan="1" rowspan="1">Not Reported</td></tr><tr><td colspan="8" rowspan="1">Viruses</td></tr><tr><td colspan="1" rowspan="2">Adenovirus F40/F41</td><td colspan="1" rowspan="1">FecalSwab</td><td colspan="1" rowspan="1">5 (0.4%)</td><td colspan="1" rowspan="1">3 (1.7%)</td><td colspan="1" rowspan="1">2 (1.7%)</td><td colspan="1" rowspan="1">0 (0.0%)</td><td colspan="1" rowspan="1">0 (0.0%)</td><td colspan="1" rowspan="1">0 (0.0%)</td></tr><tr><td colspan="1" rowspan="1">Para-Pak C&amp;S</td><td colspan="1" rowspan="1">2 (0.3%)</td><td colspan="1" rowspan="1">1 (3.2%)</td><td colspan="1" rowspan="1">0 (0.0%)</td><td colspan="1" rowspan="1">0 (0.0%)</td><td colspan="1" rowspan="1">1 (0.2%)</td><td colspan="1" rowspan="1">0 (0.0%)</td></tr><tr><td colspan="1" rowspan="2">Astrovirus</td><td colspan="1" rowspan="1">FecalSwab</td><td colspan="1" rowspan="1">3 (0.2%)</td><td colspan="1" rowspan="1">3 (1.6%)</td><td colspan="1" rowspan="1">0 (0.0%)</td><td colspan="1" rowspan="1">0 (0.0%)</td><td colspan="1" rowspan="1">0 (0.0%)</td><td colspan="1" rowspan="1">0 (0.0%)</td></tr><tr><td colspan="1" rowspan="1">Para-Pak C&amp;S</td><td colspan="1" rowspan="1">6 (0.8%)</td><td colspan="1" rowspan="1">2 (6.5%)</td><td colspan="1" rowspan="1">0 (0.0%)</td><td colspan="1" rowspan="1">3 (1.4%)</td><td colspan="1" rowspan="1">1 (0.2%)</td><td colspan="1" rowspan="1">0 (0.0%)</td></tr><tr><td colspan="1" rowspan="2">Norovirus GI/GII</td><td colspan="1" rowspan="1">FecalSwab</td><td colspan="1" rowspan="1">43 (3.5%)</td><td colspan="1" rowspan="1">22 (12.1%)</td><td colspan="1" rowspan="1">1 (0.8%)</td><td colspan="1" rowspan="1">14 (4.8%)</td><td colspan="1" rowspan="1">6 (1.0%)</td><td colspan="1" rowspan="1">0 (0.0%)</td></tr><tr><td colspan="1" rowspan="1">Para-Pak C&amp;S</td><td colspan="1" rowspan="1">16 (2.3%)</td><td colspan="1" rowspan="1">3 (9.7%)</td><td colspan="1" rowspan="1">1 (2.8%)</td><td colspan="1" rowspan="1">3 (1.4%)</td><td colspan="1" rowspan="1">9 (2.2%)</td><td colspan="1" rowspan="1">0 (0.0%)</td></tr><tr><td colspan="1" rowspan="2">Rotavirus A</td><td colspan="1" rowspan="1">FecalSwab</td><td colspan="1" rowspan="1">23 (1.9%)</td><td colspan="1" rowspan="1">13 (7.1%)</td><td colspan="1" rowspan="1">2 (1.7%)</td><td colspan="1" rowspan="1">7 (2.4%)</td><td colspan="1" rowspan="1">1 (0.2%)</td><td colspan="1" rowspan="1">0 (0.0%)</td></tr><tr><td colspan="1" rowspan="1">Para-Pak C&amp;S</td><td colspan="1" rowspan="1">4 (0.6%)</td><td colspan="1" rowspan="1">2 (6.5%)</td><td colspan="1" rowspan="1">0 (0.0%)</td><td colspan="1" rowspan="1">0 (0.0%)</td><td colspan="1" rowspan="1">2 (0.5%)</td><td colspan="1" rowspan="1">0 (0.0%)</td></tr><tr><td colspan="8" rowspan="1">Bacteria</td></tr><tr><td colspan="1" rowspan="2">Campylobacter</td><td colspan="1" rowspan="1">FecalSwab</td><td colspan="1" rowspan="1">69 (5.6%)</td><td colspan="1" rowspan="1">25 (13.7%)</td><td colspan="1" rowspan="1">7 (5.8%)</td><td colspan="1" rowspan="1">17 (5.9%)</td><td colspan="1" rowspan="1">20 (3.2%)</td><td colspan="1" rowspan="1">0 (0.0%)</td></tr><tr><td colspan="1" rowspan="1">Para-Pak C&amp;S</td><td colspan="1" rowspan="1">30 (4.2%)</td><td colspan="1" rowspan="1">2 (6.5%)</td><td colspan="1" rowspan="1">0 (0.0%)</td><td colspan="1" rowspan="1">10 (4.7%)</td><td colspan="1" rowspan="1">18 (4.3%)</td><td colspan="1" rowspan="1">0 (0.0%)</td></tr><tr><td colspan="1" rowspan="2">Plesiomonasshigelloides</td><td colspan="1" rowspan="1">FecalSwab</td><td colspan="1" rowspan="1">2 (0.2%)</td><td colspan="1" rowspan="1">0 (0.0%)</td><td colspan="1" rowspan="1">0 (0.0%)</td><td colspan="1" rowspan="1">2 (0.7%)</td><td colspan="1" rowspan="1">0 (0.0%)</td><td colspan="1" rowspan="1">0 (0.0%)</td></tr><tr><td colspan="1" rowspan="1">Para-Pak C&amp;S</td><td colspan="1" rowspan="1">7(1.0%)</td><td colspan="1" rowspan="1">1 (3.2%)</td><td colspan="1" rowspan="1">0 (0.0%)</td><td colspan="1" rowspan="1">4 (1.9%)</td><td colspan="1" rowspan="1">2 (0.5%)</td><td colspan="1" rowspan="1">0 (0.0%)</td></tr><tr><td colspan="1" rowspan="2">Salmonella</td><td colspan="1" rowspan="1">FecalSwab</td><td colspan="1" rowspan="1">14 (1.1%)</td><td colspan="1" rowspan="1">5(2.7%)</td><td colspan="1" rowspan="1">4 (3.3%)</td><td colspan="1" rowspan="1">3 (1.0%)</td><td colspan="1" rowspan="1">2 (0.3%)</td><td colspan="1" rowspan="1">0 (0.0%)</td></tr><tr><td colspan="1" rowspan="1">Para-Pak C&amp;S</td><td colspan="1" rowspan="1">17 (2.4%)</td><td colspan="1" rowspan="1">4 (12.9%)</td><td colspan="1" rowspan="1">0 (0.0%)</td><td colspan="1" rowspan="1">3 (1.4%)</td><td colspan="1" rowspan="1">10 (2.4%)</td><td colspan="1" rowspan="1">0 (0.0%)</td></tr><tr><td colspan="1" rowspan="2">Yersiniaenterocolitica</td><td colspan="1" rowspan="1">FecalSwab</td><td colspan="1" rowspan="1">22 (1.8%)</td><td colspan="1" rowspan="1">3 (1.6%)</td><td colspan="1" rowspan="1">2 (1.7%)</td><td colspan="1" rowspan="1">9 (3.1%)</td><td colspan="1" rowspan="1">8 (1.3%)</td><td colspan="1" rowspan="1">0 (0.0%)</td></tr><tr><td colspan="1" rowspan="1">Para-Pak C&amp;S</td><td colspan="1" rowspan="1">8 (1.1%)</td><td colspan="1" rowspan="1">0 (0.0%)</td><td colspan="1" rowspan="1">0 (0.0%)</td><td colspan="1" rowspan="1">4 (1.9%)</td><td colspan="1" rowspan="1">4 (1.0%)</td><td colspan="1" rowspan="1">0 (0.0%)</td></tr><tr><td colspan="8" rowspan="1">Diarrheagenic E. coli/Shigella</td></tr><tr><td colspan="1" rowspan="1">EnteropathogenicE. coli (EPEC)</td><td colspan="1" rowspan="1">Para-Pak C&amp;S</td><td colspan="1" rowspan="1">56 (7.9%)</td><td colspan="1" rowspan="1">9 (29.0%)</td><td colspan="1" rowspan="1">2 (5.6%)</td><td colspan="1" rowspan="1">18 (8.4%)</td><td colspan="1" rowspan="1">27 (6.5%)</td><td colspan="1" rowspan="1">0 (0.0%)</td></tr><tr><td colspan="1" rowspan="2">Enterotoxigenic E.coli(ETEC) It/st</td><td colspan="1" rowspan="1">FecalSwab</td><td colspan="1" rowspan="1">18 (1.5%)</td><td colspan="1" rowspan="1">2 (1.1%)</td><td colspan="1" rowspan="1">2 (1.7%)</td><td colspan="1" rowspan="1">11 (3.8%)</td><td colspan="1" rowspan="1">3 (0.5%)</td><td colspan="1" rowspan="1">0 (0.0%)</td></tr><tr><td colspan="1" rowspan="1">Para-Pak C&amp;S</td><td colspan="1" rowspan="1">17 (2.4%)</td><td colspan="1" rowspan="1">1 (3.2%)</td><td colspan="1" rowspan="1">0 (0.0%)</td><td colspan="1" rowspan="1">7 (3.3%)</td><td colspan="1" rowspan="1">9 (2.2%)</td><td colspan="1" rowspan="1">0 (0.0%)</td></tr><tr><td colspan="1" rowspan="1">Shiga-like toxin E.coli(STEC) stx1/stx2</td><td colspan="1" rowspan="1">Para-Pak C&amp;S</td><td colspan="1" rowspan="1">9 (1.3%)</td><td colspan="1" rowspan="1">0 (0.0%)</td><td colspan="1" rowspan="1">0 (0.0%)</td><td colspan="1" rowspan="1">6 (2.8%)</td><td colspan="1" rowspan="1">3 (0.7%)</td><td colspan="1" rowspan="1">0 (0.0%)</td></tr><tr><td colspan="1" rowspan="1">E. coli 0157</td><td colspan="1" rowspan="1">Para-Pak C&amp;S</td><td colspan="1" rowspan="1">0 (0.0%)</td><td colspan="1" rowspan="1">0 (0.0%)</td><td colspan="1" rowspan="1">0 (0.0%)</td><td colspan="1" rowspan="1">0 (0.0%)</td><td colspan="1" rowspan="1">0 (0.0%)</td><td colspan="1" rowspan="1">0 (0.0%)</td></tr><tr><td colspan="1" rowspan="2">Shigella/Enteroinvasive E. coli (EIEC)</td><td colspan="1" rowspan="1">FecalSwab</td><td colspan="1" rowspan="1">10 (0.8%)</td><td colspan="1" rowspan="1">1 (0.5%)</td><td colspan="1" rowspan="1">0 (0.0%)</td><td colspan="1" rowspan="1">6 (2.1%)</td><td colspan="1" rowspan="1">3 (0.5%)</td><td colspan="1" rowspan="1">0 (0.0%)</td></tr><tr><td colspan="1" rowspan="1">Para-Pak C&amp;S</td><td colspan="1" rowspan="1">3 (0.4%)</td><td colspan="1" rowspan="1">0 (0.0%)</td><td colspan="1" rowspan="1">0 (0.0%)</td><td colspan="1" rowspan="1">1 (0.5%)</td><td colspan="1" rowspan="1">2 (0.5%)</td><td colspan="1" rowspan="1">0 (0.0%)</td></tr><tr><td colspan="8" rowspan="1">Parasites</td></tr><tr><td colspan="1" rowspan="2">Cryptosporidium</td><td colspan="1" rowspan="1">FecalSwab</td><td colspan="1" rowspan="1">2 (0.2%)</td><td colspan="1" rowspan="1">0 (0.0%)</td><td colspan="1" rowspan="1">1 (0.8%)</td><td colspan="1" rowspan="1">1 (0.3%)</td><td colspan="1" rowspan="1">0 (0.0%)</td><td colspan="1" rowspan="1">0 (0.0%)</td></tr><tr><td colspan="1" rowspan="1">Para-Pak C&amp;S</td><td colspan="1" rowspan="1">7(1.0%)</td><td colspan="1" rowspan="1">0 (0.0%)</td><td colspan="1" rowspan="1">1 (2.8%)</td><td colspan="1" rowspan="1">4 (1.9%)</td><td colspan="1" rowspan="1">2 (0.5%)</td><td colspan="1" rowspan="1">0 (0.0%)</td></tr><tr><td colspan="1" rowspan="2">Cyclosporacayetanensis</td><td colspan="1" rowspan="1">FecalSwab</td><td colspan="1" rowspan="1">3 (0.2%)</td><td colspan="1" rowspan="1">0 (0.0%)</td><td colspan="1" rowspan="1">1 (0.8%)</td><td colspan="1" rowspan="1">2 (0.7%)</td><td colspan="1" rowspan="1">0 (0.0%)</td><td colspan="1" rowspan="1">0 (0.0%)</td></tr><tr><td colspan="1" rowspan="1">Para-Pak C&amp;S</td><td colspan="1" rowspan="1">18 (2.5%)</td><td colspan="1" rowspan="1">0 (0.0%)</td><td colspan="1" rowspan="1">0 (0.0%)</td><td colspan="1" rowspan="1">6 (2.8%)</td><td colspan="1" rowspan="1">12 (2.9%)</td><td colspan="1" rowspan="1">0 (0.0%)</td></tr><tr><td colspan="1" rowspan="2">Giardia lamblia</td><td colspan="1" rowspan="1">FecalSwab</td><td colspan="1" rowspan="1">15 (1.2%)</td><td colspan="1" rowspan="1">3 (1.6%)</td><td colspan="1" rowspan="1">1 (0.8%)</td><td colspan="1" rowspan="1">7 (2.4%)</td><td colspan="1" rowspan="1">4 (0.6%)</td><td colspan="1" rowspan="1">0 (0.0%)</td></tr><tr><td colspan="1" rowspan="1">Para-Pak C&amp;S</td><td colspan="1" rowspan="1">1 (0.1%)</td><td colspan="1" rowspan="1">1 (3.2%)</td><td colspan="1" rowspan="1">0 (0.0%)</td><td colspan="1" rowspan="1">0 (0.0%)</td><td colspan="1" rowspan="1">0 (0.0%)</td><td colspan="1" rowspan="1">0 (0.0%)</td></tr><tr><td colspan="1" rowspan="2">Entamoebahistlytica</td><td colspan="1" rowspan="1">FecalSwab</td><td colspan="1" rowspan="1">0 (0.0%)</td><td colspan="1" rowspan="1">0 (0.0%)</td><td colspan="1" rowspan="1">0 (0.0%)</td><td colspan="1" rowspan="1">0 (0.0%)</td><td colspan="1" rowspan="1">0 (0.0%)</td><td colspan="1" rowspan="1">0 (0.0%)</td></tr><tr><td colspan="1" rowspan="1">Para-Pak C&amp;S</td><td colspan="1" rowspan="1">0 (0.0%)</td><td colspan="1" rowspan="1">0 (0.0%)</td><td colspan="1" rowspan="1">0 (0.0%)</td><td colspan="1" rowspan="1">0 (0.0%)</td><td colspan="1" rowspan="1">0 (0.0%)</td><td colspan="1" rowspan="1">0 (0.0%)</td></tr></table>

# VIII Proposed Labeling:

The labeling supports the finding of substantial equivalence for this device.

# IX Conclusion:

The submitted information in this premarket notification is complete and supports a substantial equivalence decision.