# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATIONDECISION SUMMARYASSAY AND INSTRUMENT

I Background Information:

A 510(k) Number K221007   
B Applicant Roche Molecular Systems, Inc.   
C Proprietary and Established Names cobas HCV   
D Regulatory Information

<table><tr><td rowspan=1 colspan=1>ProductCode(s)</td><td rowspan=1 colspan=1>Classification</td><td rowspan=1 colspan=1>RegulationSection</td><td rowspan=1 colspan=1>Panel</td></tr><tr><td rowspan=1 colspan=1>MZP</td><td rowspan=1 colspan=1>Class II</td><td rowspan=1 colspan=1>21 CFR 866.3170 -Nucleic Acid-BasedHepatitis C VirusRibonucleic Acid Tests</td><td rowspan=1 colspan=1>MI - Microbiology</td></tr></table>

# II Submission/Device Overview:

# A Purpose for Submission:

To allow use of the cobas 5800 system with the cobas HCV assay. The cobas HCV assay was originally approved under P150015 for use with the cobas 6800 and cobas 8800.

B Measurand: Hepatitis C virus (HCV) RNA   
C Type of Test: Nucleic acid amplification test

# III Intended Use/Indications for Use:

A Intended Use(s): See Indications for Use below.

# B Indication(s) for Use:

cobas $\textsuperscript { \textregistered }$ HCV is an in vitro nucleic acid amplification test for both the detection and quantitation of hepatitis C virus (HCV) RNA, in human EDTA plasma or serum, of HCV antibody positive or HCV-infected individuals. Specimens containing HCV genotypes 1 to 6 are validated for detection and quantitation in the assay.

cobas $\textsuperscript { \textregistered }$ HCV is intended for use as an aid in the diagnosis of HCV infection in the following populations: individuals with antibody evidence of HCV with evidence of liver disease, individuals suspected to be actively infected with HCV antibody evidence, and individuals at risk for HCV infection with antibodies to HCV. Detection of HCV RNA indicates that the virus is replicating and therefore is evidence of active infection.

cobas $\textsuperscript { \textregistered }$ HCV is intended for use as an aid in the management of HCV-infected patients undergoing anti-viral therapy. The assay can be used to measure HCV RNA levels at baseline, during treatment, at the end of treatment, and at the end of follow up of treatment to determine sustained or non-sustained viral response. The results must be interpreted within the context of all relevant clinical and laboratory findings.

cobas $\textsuperscript { \textregistered }$ HCV has not been approved for use as a screening test for the presence of HCV in blood or blood products.

Assay performance characteristics have been established for individuals treated with certain direct-acting antiviral agents (DAA) regimens. No information is available on the assay’s predictive value when other DAA combination therapies are used.

C Special Conditions for Use Statement(s): Rx - For Prescription Use Only

# D Special Instrument Requirements:

For use on the cobas 5800/6800/8800 Systems

# IV Device/System Characteristics:

# A Device Description:

The cobas HCV assay is a quantitative test performed on the cobas 5800 System, cobas 6800 System or cobas 8800 System. The test detects and quantitates HCV RNA in EDTA plasma or serum of infected patients. Dual probes are used to detect and quantify, but not discriminate genotypes 1–6. The viral load is quantified against a non-HCV armored RNA quantitation standard (RNA-QS), which is introduced into each specimen during sample preparation. The RNA-QS also functions as an internal control to assess substantial failures during the sample preparation and PCR amplification processes.

# B Principle of Operation:

Nucleic acid from patient samples, external controls and added armored RNA-QS molecules are simultaneously extracted by addition of proteinase and lysis reagent to the sample. The released nucleic acid binds to the silica surface of the added magnetic glass particles. Unbound substances and impurities, such as denatured protein, cellular debris and potential PCR inhibitors are removed with subsequent wash buffer steps and purified nucleic acid is eluted from the magnetic glass particles with elution buffer at elevated temperature.

Selective amplification of target nucleic acid from the sample is achieved by the use of target virus-specific forward and reverse primers which are selected from highly conserved regions of HCV. Selective amplification of RNA-QS is achieved by the use of sequence-specific forward and reverse primers which are selected to have no homology with the HCV genome. A thermostable DNA polymerase enzyme is used for both reverse-transcription and PCR amplification. The target and RNA-QS sequences are amplified simultaneously utilizing a universal PCR amplification profile with predefined temperature steps and number of cycles. The master mix includes deoxyuridine triphosphate (dUTP), instead of deoxythimidine triphosphate (dTTP), which is incorporated into the newly synthesized DNA (amplicon).1-3 Any contaminating amplicon from previous PCR runs are eliminated by the AmpErase enzyme, which is included in the PCR mix, during the first thermal cycling step. However, newly formed amplicon are not eliminated since the AmpErase enzyme is inactivated once exposed to temperatures above $5 5 \mathrm { { ^ \circ C } }$ .

The cobas HCV master mix contains dual detection probes specific for the HCV target sequences and one for the RNA-QS. The probes are labeled with target-specific fluorescent reporter dyes allowing simultaneous detection of HCV target and RNA-QS in two different target channels.

When not bound to the target sequence, the fluorescent signal of the intact probe is suppressed by a quencher dye. During the PCR amplification step, hybridization of the probes to the specific single-stranded DNA template results in cleavage of the probe by the $5 "$ -to-3' nuclease activity of the DNA polymerase resulting in separation of the reporter and quencher dyes and the generation of a fluorescent signal. With each PCR cycle, increasing amounts of cleaved probes are generated and the cumulative signal of the reporter dye increases concomitantly. Real-time detection and discrimination of PCR products is accomplished by measuring the fluorescence of the released reporter dyes for the viral targets and RNA-QS.

# C Instrument Description Information:

1. Instrument Name: cobas 5800 System

2. Specimen Identification: The cobas 5800 supports multiple types of barcodes. Loaded samples are automatically moved for barcode scanning and processing.

3. Specimen Sampling and Handling: Specimen processing is fully automated on the cobas 5800 system.

4. Calibration: No instrument calibration is required by the user.

5. Quality Control: Refer to cobas HCV assay labeling.

V Substantial Equivalence Information:

A Predicate Device Name(s): cobas HCV   
B Predicate 510(k) Number(s): P150015

C Comparison with Predicate:

<table><tr><td colspan="1" rowspan="1">Device &amp; PredicateDevice(s):</td><td colspan="1" rowspan="1">K221007</td><td colspan="1" rowspan="1">P150015</td></tr><tr><td colspan="1" rowspan="1">Device Trade Name</td><td colspan="1" rowspan="1">cobas HCV</td><td colspan="1" rowspan="1">cobas HCV</td></tr><tr><td colspan="1" rowspan="1">General DeviceCharacteristicSimilarities</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">IntendedUse/Indications For Use</td><td colspan="1" rowspan="1">The cobas HCV is an in vitronucleic acid amplification testfor both the detection andquantitation of hepatitis C virus(HCV) RNA, in human EDTAplasma or serum, of HCVantibody positive or HCV-infected individuals. Specimenscontaining HCV genotypes 1 to6 are validated for detection andquantitation in the assay.cobas HCV is intended for useas an aid in the diagnosis ofHCV infection in the followingpopulations: individuals withantibody evidence of HCV withevidence of liver disease,individuals suspected to beactively infected with HCVantibody evidence, andindividuals at risk for HCVinfection with antibodies toHCV. Detection of HCV RNAindicates that the virus isreplicating and therefore isevidence of active infection.cobas HCV is intended for useas an aid in the management of</td><td colspan="1" rowspan="1">The cobas HCV is an in vitronucleic acid amplification testfor both the detection andquantitation of hepatitis C virus(HCV) RNA, in human EDTAplasma or serum, of HCVantibody positive or HCV-infected individuals. Specimenscontaining HCV genotypes 1 to6 are validated for detection andquantitation in the assay.cobas HCV is intended for useas an aid in the diagnosis ofHCV infection in the followingpopulations: individuals withantibody evidence of HCV withevidence of liver disease,individuals suspected to beactively infected with HCVantibody evidence, andindividuals at risk for HCVinfection with antibodies toHCV. Detection of HCV RNAindicates that the virus isreplicating and therefore isevidence of active infection.cobas HCV is intended for useas an aid in the management of</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">HCV-infected patientsundergoing anti-viral therapy.The assay can be used tomeasure HCV RNA levels atbaseline, during treatment, at theend of treatment, and at the endof follow up of treatment todetermine sustained or non-sustained viral response. Theresults must be interpretedwithin the context of all relevantclinical and laboratory findings.cobas HCV has not beenapproved for use as a screeningtest for the presence of HCV inblood or blood products.Assay performancecharacteristics have beenestablished for individualstreated with certain direct-actingantiviral agents (DAA)regimens. No information isavailable on the assay'spredictive value when otherDAA combination therapies areused.</td><td colspan="1" rowspan="1">HCV-infected patientsundergoing anti-viral therapy.The assay can be used tomeasure HCV RNA levels atbaseline, during treatment, at theend of treatment, and at the endof follow up of treatment todetermine sustained or non-sustained viral response. Theresults must be interpretedwithin the context of all relevantclinical and laboratory findings.cobas HCV has not beenapproved for use as a screeningtest for the presence of HCV inblood or blood products.Assay performancecharacteristics have beenestablished for individualstreated with certain direct-actingantiviral agents (DAA)regimens. No information isavailable on the assay'spredictive value when otherDAA combination therapies areused.</td></tr><tr><td colspan="1" rowspan="1">Conditions for Use</td><td colspan="1" rowspan="1">For Prescription Use</td><td colspan="1" rowspan="1">For Prescription Use</td></tr><tr><td colspan="1" rowspan="1">Sample Types</td><td colspan="1" rowspan="1">EDTA Plasma and Serum</td><td colspan="1" rowspan="1">EDTA Plasma and Serum</td></tr><tr><td colspan="1" rowspan="1">Analyte Targets</td><td colspan="1" rowspan="1">Hepatitis C RNA genotypes 1-6</td><td colspan="1" rowspan="1">Hepatitis C RNA genotypes 1-6</td></tr><tr><td colspan="1" rowspan="1">Sample Preparation</td><td colspan="1" rowspan="1">Automated RNA extraction &amp;purification</td><td colspan="1" rowspan="1">Automated RNA extraction &amp;purification</td></tr><tr><td colspan="1" rowspan="1">AmplificationTechnology</td><td colspan="1" rowspan="1">Real Time PCR</td><td colspan="1" rowspan="1">Real Time PCR</td></tr><tr><td colspan="1" rowspan="1">Detection Chemistry</td><td colspan="1" rowspan="1">Dual detection probes labeledwith target-specific fluorescentreporter dyes allowingsimultaneous detection of HCVtarget and RNA-QS in twodifferent target channels. Realtime detection anddiscrimination of PCR productsis accomplished by measuringthe fluorescence of the releasedreporter dyes.</td><td colspan="1" rowspan="1">Dual detection probes labeledwith target-specific fluorescentreporter dyes allowingsimultaneous detection of HCVtarget and RNA-QS in twodifferent target channels. Realtime detection anddiscrimination of PCR productsis accomplished by measuringthe fluorescence of the releasedreporter dyes.</td></tr><tr><td colspan="1" rowspan="1">Assay Controls</td><td colspan="1" rowspan="1">RNA-QS functions as aninternal control.Three external controls: HighTiter Positive, Low TiterPositive, Negative Control</td><td colspan="1" rowspan="1">RNA-QS functions as aninternal control.Three external controls: HighTiter Positive, Low TiterPositive, Negative Control</td></tr><tr><td colspan="1" rowspan="1">Results Analysis</td><td colspan="1" rowspan="1">PCR cycle threshold analysis.</td><td colspan="1" rowspan="1">PCR cycle threshold analysis.</td></tr><tr><td colspan="1" rowspan="1">General DeviceCharacteristicDifferences</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Insrumentation</td><td colspan="1" rowspan="1">cobas 5800 System</td><td colspan="1" rowspan="1">Cobas 6800/8800 Systems</td></tr><tr><td colspan="1" rowspan="1">Design Concept</td><td colspan="1" rowspan="1">Same</td><td colspan="1" rowspan="1">Sample loading, sample prep,and PCR amplification/detectionin one instrument</td></tr><tr><td colspan="1" rowspan="1">Throughput</td><td colspan="1" rowspan="1">~144 tests in 8 hours</td><td colspan="1" rowspan="1">~384 / 960 tests in 8 hours</td></tr><tr><td colspan="1" rowspan="1">Configuration</td><td colspan="1" rowspan="1">Single deck rather than separatemodules. Same technology butfewer components.</td><td colspan="1" rowspan="1">Separate modules (samplesupply, transfer, processing,analytic), duplicate componentsfor high throughput.</td></tr><tr><td colspan="1" rowspan="1">Consumables</td><td colspan="1" rowspan="1">24-well sample prep andamplification/detection plateswith same material, geometry,and volume per well. Samedisposable pipette tips.</td><td colspan="1" rowspan="1">48 and 96-well samplepreparation andamplification/detection plates.Disposable pipette tips.</td></tr><tr><td colspan="1" rowspan="1">Sample Pipetting</td><td colspan="1" rowspan="1">Single two-channel pipettor,smaller but using sametechnology and volumes</td><td colspan="1" rowspan="1">Single four-channel pipettor</td></tr><tr><td colspan="1" rowspan="1">Reagent Pipetting</td><td colspan="1" rowspan="1">Smaller reagent pipetting headwith 2 reagent needles, smallerbut same technology</td><td colspan="1" rowspan="1">Reagent pipetting head with asingle 8 channel pipettor</td></tr><tr><td colspan="1" rowspan="1">Process Pipetting</td><td colspan="1" rowspan="1">24-well processing plate andsingle 24-channel processpipettor, smaller but sametechnology</td><td colspan="1" rowspan="1">48-well processing plate andsingle 48-channel processpipettor</td></tr><tr><td colspan="1" rowspan="1">Controls</td><td colspan="1" rowspan="1">Default setting the same(positive and negative controlson every plate), but due tosmaller plates, options foralternate control frequency canbe implemented based on labrequirements and localregulations, e.g., time-based(every 24 hours). Controls willbe required at least for eachreagent lot change.</td><td colspan="1" rowspan="1">Positive control and negativecontrol included on everyamplification/detection plate</td></tr><tr><td colspan="1" rowspan="1">Reagent Storage</td><td colspan="1" rowspan="1">Temperature-controlled (notrefrigerated)</td><td colspan="1" rowspan="1">On-board refrigerated reagentstorage</td></tr><tr><td colspan="1" rowspan="1">Sample Prep</td><td colspan="1" rowspan="1">Minor changes in timing ofreagent pipetting steps due tosmaller batches; sameincubation times and temps</td><td colspan="1" rowspan="1">Sample Setup, Lysis Binding,Washing and Elution</td></tr><tr><td colspan="1" rowspan="1">Amplification/Detection</td><td colspan="1" rowspan="1">Same technology but smallerthermal cyclerSame temperatures and times for</td><td colspan="1" rowspan="1">Real-time PCR usingfluorescence spectroscopy</td></tr><tr><td></td><td>denaturation, annealing, and elongation steps Same result calculation and interpretation methods</td><td></td></tr></table>

# VI Standards/Guidance Documents Referenced:

CLSI: Evaluation of Detection Capability for Clinical Laboratory Measurement Procedures; Approved Guideline- 2nd Edition, EP17-A2, Wayne, PA: Clinical and Laboratory Standards Institute

CLSI: Evaluation of Precision of Quantitative Measurement Procedures; Approved Guideline—Third Edition. CLSI document EP05-A3. Wayne, PA: Clinical and Laboratory Standards Institute; 2014.

# VII Performance Characteristics (if/when applicable):

# A Analytical Performance:

1. Precision/Reproducibility:

Precision: See P150015

Reproducibility was assessed with a multi-site study testing a prepared panel ranging from $1 . 5 0 \mathrm { E + 0 1 \ I U / m L }$ to $1 . 0 0 \mathrm { E + } 0 8 \mathrm { \Pi { U } / m L }$ . A dilution series consisting of a clinical specimen and plasmid RNA sample was used to generate the reproducibility panel. HCV positive material was serially diluted in plasma to create a panel of 7 members that includes concentration levels at, below, and above medical decision points. Panel members were tested in 3 replicates/run, 2 runs/day, over 5 days, using 3 cobas 5800 instruments at 3 different sites (1 internal and 2 external) and 3 cobas 6800/8800 instruments at 1 internal site, 2 runs per day per instrument and by using overall three different lots of the cobas HCV kits

# Results:

Table 1: Standard Deviation and Coefficient of Variance $( \% )$ for cobas HCV on cobas 5800 and cobas 6800/8800. Three Systems at three Sites combined (Absolute and Percentage)   

<table><tr><td colspan="1" rowspan="3">Platform</td><td colspan="1" rowspan="3">PanelMember</td><td colspan="12" rowspan="1">Standard Deviation (SD) and Percent Coefficient of Variation (CV)</td></tr><tr><td colspan="1" rowspan="2">Meanobserved og100M Titer(IU/mL)</td><td colspan="1" rowspan="2">N</td><td colspan="2" rowspan="1">Site</td><td colspan="2" rowspan="1">Day</td><td colspan="2" rowspan="1">Run</td><td colspan="2" rowspan="1">Within-Run</td><td colspan="2" rowspan="1">Total</td></tr><tr><td colspan="1" rowspan="1">SD</td><td colspan="1" rowspan="1">Cv</td><td colspan="1" rowspan="1">SD</td><td colspan="1" rowspan="1">CV[%]</td><td colspan="1" rowspan="1">SD</td><td colspan="1" rowspan="1">CV[%]</td><td colspan="1" rowspan="1">SD</td><td colspan="1" rowspan="1">CV[%]</td><td colspan="1" rowspan="1">SD</td><td colspan="1" rowspan="1">CV[%]</td></tr><tr><td colspan="1" rowspan="6">cobas5800</td><td colspan="1" rowspan="1">PM1</td><td colspan="1" rowspan="1">7.80</td><td colspan="1" rowspan="1">90</td><td colspan="1" rowspan="1">0.18</td><td colspan="1" rowspan="1">2.31</td><td colspan="1" rowspan="1">0.08</td><td colspan="1" rowspan="1">1.00</td><td colspan="1" rowspan="1">0.02</td><td colspan="1" rowspan="1">0.26</td><td colspan="1" rowspan="1">0.05</td><td colspan="1" rowspan="1">0.66</td><td colspan="1" rowspan="1">0.20</td><td colspan="1" rowspan="1">2.61</td></tr><tr><td colspan="1" rowspan="1">PM2</td><td colspan="1" rowspan="1">6.65</td><td colspan="1" rowspan="1">90</td><td colspan="1" rowspan="1">0.14</td><td colspan="1" rowspan="1">2.17</td><td colspan="1" rowspan="1">0.07</td><td colspan="1" rowspan="1">1.03</td><td colspan="1" rowspan="1">0.01</td><td colspan="1" rowspan="1">0.12</td><td colspan="1" rowspan="1">0.04</td><td colspan="1" rowspan="1">0.67</td><td colspan="1" rowspan="1">0.17</td><td colspan="1" rowspan="1">2.50</td></tr><tr><td colspan="1" rowspan="1">PM3</td><td colspan="1" rowspan="1">5.79</td><td colspan="1" rowspan="1">90</td><td colspan="1" rowspan="1">0.15</td><td colspan="1" rowspan="1">2.65</td><td colspan="1" rowspan="1">0.06</td><td colspan="1" rowspan="1">1.11</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.06</td><td colspan="1" rowspan="1">1.04</td><td colspan="1" rowspan="1">0.18</td><td colspan="1" rowspan="1">3.05</td></tr><tr><td colspan="1" rowspan="1">PM4</td><td colspan="1" rowspan="1">4.07</td><td colspan="1" rowspan="1">90</td><td colspan="1" rowspan="1">0.11</td><td colspan="1" rowspan="1">2.65</td><td colspan="1" rowspan="1">0.07</td><td colspan="1" rowspan="1">1.77</td><td colspan="1" rowspan="1">0.04</td><td colspan="1" rowspan="1">1.07</td><td colspan="1" rowspan="1">0.06</td><td colspan="1" rowspan="1">1.54</td><td colspan="1" rowspan="1">0.15</td><td colspan="1" rowspan="1">3.70</td></tr><tr><td colspan="1" rowspan="1">PM5</td><td colspan="1" rowspan="1">2.01</td><td colspan="1" rowspan="1">90</td><td colspan="1" rowspan="1">0.15</td><td colspan="1" rowspan="1">7.68</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.04</td><td colspan="1" rowspan="1">1.96</td><td colspan="1" rowspan="1">0.16</td><td colspan="1" rowspan="1">8.17</td><td colspan="1" rowspan="1">0.23</td><td colspan="1" rowspan="1">11.38</td></tr><tr><td colspan="1" rowspan="1">PM6</td><td colspan="1" rowspan="1">1.50</td><td colspan="1" rowspan="1">89</td><td colspan="1" rowspan="1">0.09</td><td colspan="1" rowspan="1">5.70</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.05</td><td colspan="1" rowspan="1">3.65</td><td colspan="1" rowspan="1">0.23</td><td colspan="1" rowspan="1">15.14</td><td colspan="1" rowspan="1">0.25</td><td colspan="1" rowspan="1">16.59</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">PM7</td><td colspan="1" rowspan="1">1.28</td><td colspan="1" rowspan="1">90</td><td colspan="1" rowspan="1">0.16</td><td colspan="1" rowspan="1">12.20</td><td colspan="1" rowspan="1">0.09</td><td colspan="1" rowspan="1">6.94</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.36</td><td colspan="1" rowspan="1">0.27</td><td colspan="1" rowspan="1">21.37</td><td colspan="1" rowspan="1">0.33</td><td colspan="1" rowspan="1">25.57</td></tr><tr><td colspan="1" rowspan="7">cobas6800/8800</td><td colspan="1" rowspan="1">PM1</td><td colspan="1" rowspan="1">7.99</td><td colspan="1" rowspan="1">90</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.04</td><td colspan="1" rowspan="1">0.50</td><td colspan="1" rowspan="1">0.01</td><td colspan="1" rowspan="1">0.07</td><td colspan="1" rowspan="1">0.04</td><td colspan="1" rowspan="1">0.49</td><td colspan="1" rowspan="1">0.06</td><td colspan="1" rowspan="1">0.71</td></tr><tr><td colspan="1" rowspan="1">PM2</td><td colspan="1" rowspan="1">6.78</td><td colspan="1" rowspan="1">90</td><td colspan="1" rowspan="1">0.01</td><td colspan="1" rowspan="1">0.09</td><td colspan="1" rowspan="1">0.03</td><td colspan="1" rowspan="1">0.39</td><td colspan="1" rowspan="1">0.02</td><td colspan="1" rowspan="1">0.27</td><td colspan="1" rowspan="1">0.04</td><td colspan="1" rowspan="1">0.65</td><td colspan="1" rowspan="1">0.05</td><td colspan="1" rowspan="1">0.81</td></tr><tr><td colspan="1" rowspan="1">PM3</td><td colspan="1" rowspan="1">5.93</td><td colspan="1" rowspan="1">89</td><td colspan="1" rowspan="1">0.01</td><td colspan="1" rowspan="1">0.24</td><td colspan="1" rowspan="1">0.04</td><td colspan="1" rowspan="1">0.59</td><td colspan="1" rowspan="1">0.02</td><td colspan="1" rowspan="1">0.35</td><td colspan="1" rowspan="1">0.03</td><td colspan="1" rowspan="1">0.58</td><td colspan="1" rowspan="1">0.06</td><td colspan="1" rowspan="1">0.93</td></tr><tr><td colspan="1" rowspan="1">PM4</td><td colspan="1" rowspan="1">4.12</td><td colspan="1" rowspan="1">90</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.02</td><td colspan="1" rowspan="1">0.41</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.06</td><td colspan="1" rowspan="1">1.51</td><td colspan="1" rowspan="1">0.06</td><td colspan="1" rowspan="1">1.57</td></tr><tr><td colspan="1" rowspan="1">PM5</td><td colspan="1" rowspan="1">2.14</td><td colspan="1" rowspan="1">90</td><td colspan="1" rowspan="1">0.01</td><td colspan="1" rowspan="1">0.64</td><td colspan="1" rowspan="1">0.07</td><td colspan="1" rowspan="1">3.26</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.12</td><td colspan="1" rowspan="1">5.66</td><td colspan="1" rowspan="1">0.14</td><td colspan="1" rowspan="1">6.56</td></tr><tr><td colspan="1" rowspan="1">PM6</td><td colspan="1" rowspan="1">1.58</td><td colspan="1" rowspan="1">90</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.05</td><td colspan="1" rowspan="1">3.20</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.24</td><td colspan="1" rowspan="1">15.21</td><td colspan="1" rowspan="1">0.25</td><td colspan="1" rowspan="1">15.55</td></tr><tr><td colspan="1" rowspan="1">PM7</td><td colspan="1" rowspan="1">1.26</td><td colspan="1" rowspan="1">88</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.25</td><td colspan="1" rowspan="1">19.80</td><td colspan="1" rowspan="1">0.25</td><td colspan="1" rowspan="1">19.80</td></tr></table>

2. Linearity:

Assay linearity was assessed using the predominant genotype (GT1) in plasma. The linearity panels were prepared as serial dilutions designed to cover the entire linear range of the assay. The dilution series consisted of 16 concentration levels spanning the intended linear range. The dilution series were prepared by diluting an arHCV-MS08-pCP1 specimen and an HCV positive clinical specimen in negative pooled EDTA-plasma. The Titer assignment of the study panels has been performed via Calibrator Bracketing Method (CBM). The panels were tested with three different lots of the cobas HCV kits. The dilution series covered a total of 36 replicates per concentration level in negative pooled EDTA plasma which were tested with $3 \mathrm { k i t }$ lots and tested on 2 cobas 5800 Systems over the course of 6 days.

Table 2: Linearity Panel Layout   

<table><tr><td colspan="1" rowspan="1">Panel Member</td><td colspan="1" rowspan="1">Concentration level description</td><td colspan="1" rowspan="1">Concentration aRNA(IU/mL)</td><td colspan="1" rowspan="1">Concentration clinicalspecimen (IU/mL)</td></tr><tr><td colspan="1" rowspan="1">PM01</td><td colspan="1" rowspan="1">above upper limit of quantification (ULoQ)</td><td colspan="1" rowspan="1">2.00E+08</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">PM02</td><td colspan="1" rowspan="1">ULoQ</td><td colspan="1" rowspan="1">1.00E+08</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">PM03</td><td colspan="1" rowspan="1">intermediate level</td><td colspan="1" rowspan="1">1.00E+07</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">PM04</td><td colspan="1" rowspan="1">intermediate level</td><td colspan="1" rowspan="1">1.00E+06</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">PM05</td><td colspan="1" rowspan="1">intermediate level</td><td colspan="1" rowspan="1">4.00E+05</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">PM06</td><td colspan="1" rowspan="1">intermediate level</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">5.00E+04</td></tr><tr><td colspan="1" rowspan="1">PM07</td><td colspan="1" rowspan="1">intermediate level</td><td colspan="1" rowspan="1">1.00E+04</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">PM08</td><td colspan="1" rowspan="1">intermediate level</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1.00E+04</td></tr><tr><td colspan="1" rowspan="1">CBMaRNA*</td><td colspan="1" rowspan="1">for calibrator bracketing method</td><td colspan="1" rowspan="1">5.50E+03</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">CBMclin*</td><td colspan="1" rowspan="1">for calibrator bracketing method</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">5.50E+03</td></tr><tr><td colspan="1" rowspan="1">PM09</td><td colspan="1" rowspan="1">intermediate level</td><td colspan="1" rowspan="1">4.00E+03</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">PM10</td><td colspan="1" rowspan="1">intermediate level</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1.00E+03</td></tr><tr><td colspan="1" rowspan="1">PM11</td><td colspan="1" rowspan="1">intermediate level</td><td colspan="1" rowspan="1">1.00E+03</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">PM12</td><td colspan="1" rowspan="1">intermediate level</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1.00E+02</td></tr><tr><td colspan="1" rowspan="1">PM13</td><td colspan="1" rowspan="1">intermediate level</td><td colspan="1" rowspan="1">1.00E+02</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">PM14</td><td colspan="1" rowspan="1">intermediate level</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">5.00E+01</td></tr><tr><td colspan="1" rowspan="1">PM15</td><td colspan="1" rowspan="1">lower limit of quantification (LLoQ)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1.50E+01</td></tr><tr><td>PM16</td><td>below LLoQ</td><td></td><td>5.00E+ 00</td></tr></table>

\* CBMaRNA $( 5 . 5 0 \mathrm { E } + 0 3 \mathrm { I U } / \mathrm { m L } )$ ) was used for the titer assignment of the aRNA panel members; CBMclin $( 5 . 5 0 \mathrm { E } + 0 3 ~ \mathrm { I U } / \mathrm { m L } )$ ) was used for the titer assignment of the clinical specimen panel members.

Table 3: Linearity Results   

<table><tr><td rowspan=2 colspan=1>PanelMember</td><td rowspan=2 colspan=1>NominalTiter[IU/mL]</td><td rowspan=2 colspan=1>AssignedTiter[IU/mL]</td><td rowspan=2 colspan=1>Assignedlog10 Titer</td><td rowspan=2 colspan=1>MeanObservedlog10 Titer</td><td rowspan=1 colspan=2>Predicted log Titer</td><td rowspan=2 colspan=1>AbsoluteDifference</td><td rowspan=2 colspan=1>MeetsAcceptanceCriterion?</td></tr><tr><td rowspan=1 colspan=1>1st Order</td><td rowspan=1 colspan=1>3rd Order</td></tr><tr><td rowspan=1 colspan=1>PM01</td><td rowspan=1 colspan=1>2.00E+08</td><td rowspan=1 colspan=1>1.66E+08</td><td rowspan=1 colspan=1>8.22</td><td rowspan=1 colspan=1>8.29</td><td rowspan=1 colspan=1>8.31</td><td rowspan=1 colspan=1>8.26</td><td rowspan=1 colspan=1>-0.05</td><td rowspan=1 colspan=1>Yes</td></tr><tr><td rowspan=1 colspan=1>PM02</td><td rowspan=1 colspan=1>1.00E+08</td><td rowspan=1 colspan=1>8.31E+07</td><td rowspan=1 colspan=1>7.92</td><td rowspan=1 colspan=1>7.99</td><td rowspan=1 colspan=1>8.01</td><td rowspan=1 colspan=1>8.00</td><td rowspan=1 colspan=1>-0.01</td><td rowspan=1 colspan=1>Yes</td></tr><tr><td rowspan=1 colspan=1>PM03</td><td rowspan=1 colspan=1>1.00E+07</td><td rowspan=1 colspan=1>8.31E+06</td><td rowspan=1 colspan=1>6.92</td><td rowspan=1 colspan=1>7.02</td><td rowspan=1 colspan=1>7.01</td><td rowspan=1 colspan=1>7.07</td><td rowspan=1 colspan=1>0.06</td><td rowspan=1 colspan=1>Yes</td></tr><tr><td rowspan=1 colspan=1>PM04</td><td rowspan=1 colspan=1>1.00E+06</td><td rowspan=1 colspan=1>8.31E+05</td><td rowspan=1 colspan=1>5.92</td><td rowspan=1 colspan=1>6.07</td><td rowspan=1 colspan=1>6.01</td><td rowspan=1 colspan=1>6.07</td><td rowspan=1 colspan=1>0.06</td><td rowspan=1 colspan=1>Yes</td></tr><tr><td rowspan=1 colspan=1>PM05</td><td rowspan=1 colspan=1>4.00E+05</td><td rowspan=1 colspan=1>3.32E+05</td><td rowspan=1 colspan=1>5.52</td><td rowspan=1 colspan=1>5.68</td><td rowspan=1 colspan=1>5.61</td><td rowspan=1 colspan=1>5.65</td><td rowspan=1 colspan=1>0.05</td><td rowspan=1 colspan=1>Yes</td></tr><tr><td rowspan=1 colspan=1>PM06</td><td rowspan=1 colspan=1>5.00E+04</td><td rowspan=1 colspan=1>6.54E+04</td><td rowspan=1 colspan=1>4.82</td><td rowspan=1 colspan=1>4.90</td><td rowspan=1 colspan=1>4.90</td><td rowspan=1 colspan=1>4.92</td><td rowspan=1 colspan=1>0.02</td><td rowspan=1 colspan=1>Yes</td></tr><tr><td rowspan=1 colspan=1>PM07</td><td rowspan=1 colspan=1>1.00E+04</td><td rowspan=1 colspan=1>8.31E+03</td><td rowspan=1 colspan=1>4.12</td><td rowspan=1 colspan=1>4.19</td><td rowspan=1 colspan=1>4.20</td><td rowspan=1 colspan=1>4.18</td><td rowspan=1 colspan=1>-0.02</td><td rowspan=1 colspan=1>Yes</td></tr><tr><td rowspan=1 colspan=1>PM08</td><td rowspan=1 colspan=1>1.00E+04</td><td rowspan=1 colspan=1>1.31E+04</td><td rowspan=1 colspan=1>3.92</td><td rowspan=1 colspan=1>3.99</td><td rowspan=1 colspan=1>4.01</td><td rowspan=1 colspan=1>3.98</td><td rowspan=1 colspan=1>-0.03</td><td rowspan=1 colspan=1>Yes</td></tr><tr><td rowspan=1 colspan=1>PM09</td><td rowspan=1 colspan=1>4.00E+03</td><td rowspan=1 colspan=1>3.32E+03</td><td rowspan=1 colspan=1>3.52</td><td rowspan=1 colspan=1>3.58</td><td rowspan=1 colspan=1>3.61</td><td rowspan=1 colspan=1>3.57</td><td rowspan=1 colspan=1>-0.04</td><td rowspan=1 colspan=1>Yes</td></tr><tr><td rowspan=1 colspan=1>PM10</td><td rowspan=1 colspan=1>1.00E+03</td><td rowspan=1 colspan=1>1.31E+03</td><td rowspan=1 colspan=1>3.12</td><td rowspan=1 colspan=1>3.17</td><td rowspan=1 colspan=1>3.20</td><td rowspan=1 colspan=1>3.15</td><td rowspan=1 colspan=1>-0.05</td><td rowspan=1 colspan=1>Yes</td></tr><tr><td rowspan=1 colspan=1>PM11</td><td rowspan=1 colspan=1>1.00E+03</td><td rowspan=1 colspan=1>8.31E+02</td><td rowspan=1 colspan=1>2.92</td><td rowspan=1 colspan=1>2.96</td><td rowspan=1 colspan=1>3.01</td><td rowspan=1 colspan=1>2.95</td><td rowspan=1 colspan=1>-0.05</td><td rowspan=1 colspan=1>Yes</td></tr><tr><td rowspan=1 colspan=1>PM12</td><td rowspan=1 colspan=1>1.00E+02</td><td rowspan=1 colspan=1>1.31E+02</td><td rowspan=1 colspan=1>2.12</td><td rowspan=1 colspan=1>2.15</td><td rowspan=1 colspan=1>2.20</td><td rowspan=1 colspan=1>2.17</td><td rowspan=1 colspan=1>-0.03</td><td rowspan=1 colspan=1>Yes</td></tr><tr><td rowspan=1 colspan=1>PM13</td><td rowspan=1 colspan=1>1.00E+02</td><td rowspan=1 colspan=1>8.31E+01</td><td rowspan=1 colspan=1>1.92</td><td rowspan=1 colspan=1>2.00</td><td rowspan=1 colspan=1>2.00</td><td rowspan=1 colspan=1>1.99</td><td rowspan=1 colspan=1>-0.02</td><td rowspan=1 colspan=1>Yes</td></tr><tr><td rowspan=1 colspan=1>PM14</td><td rowspan=1 colspan=1>5.00E+01</td><td rowspan=1 colspan=1>6.54E+01</td><td rowspan=1 colspan=1>1.82</td><td rowspan=1 colspan=1>1.83</td><td rowspan=1 colspan=1>1.90</td><td rowspan=1 colspan=1>1.89</td><td rowspan=1 colspan=1>-0.01</td><td rowspan=1 colspan=1>Yes</td></tr><tr><td rowspan=1 colspan=1>PM15</td><td rowspan=1 colspan=1>1.50E+01</td><td rowspan=1 colspan=1>1.96E+01</td><td rowspan=1 colspan=1>1.29</td><td rowspan=1 colspan=1>1.38</td><td rowspan=1 colspan=1>1.38</td><td rowspan=1 colspan=1>1.42</td><td rowspan=1 colspan=1>0.04</td><td rowspan=1 colspan=1>Yes</td></tr><tr><td rowspan=1 colspan=1>PM16</td><td rowspan=1 colspan=1>5.00E+00</td><td rowspan=1 colspan=1>6.54E+00</td><td rowspan=1 colspan=1>0.82</td><td rowspan=1 colspan=1>1.09</td><td rowspan=1 colspan=1>0.90</td><td rowspan=1 colspan=1>1.01</td><td rowspan=1 colspan=1>0.11</td><td rowspan=1 colspan=1>Yes</td></tr></table>

# Regression Plot:

![](images/984f86e088ecc54cbb284dfa5cffe1a0835e5aece498df60b5a4bb68d5df1dfb.jpg)

3. Analytical Specificity/Interference: See P150015

4. Assay Reportable Range:

The results of the linearity study and calculation of the Lower and Upper Limits of Quantitation confirm the assay reportable range is consistent across analyzer platforms: 15 $\mathrm { I U / m L - 1 . 0 0 E + 0 8 \ : I U / m L }$ .

5. Traceability, Stability, Expected Values (Controls, Calibrators, or Methods): See P150015

6. Detection Limit:

The Limit of Detection (LoD) was assessed with a sample panel prepared with an HCV secondary standard and HCV negative pooled EDTA plasma. The dilution series was prepared with 6 concentration levels - 3 levels below the expected LoD, 1 level near the expected LoD and 2 levels above the expected LoD by diluting the secondary standard in pooled negative EDTA plasma. The panel was tested with $3 \mathrm { k i t }$ lots, on 7 days, on 3 cobas 5800 systems and 1 cobas 6800/8800 system over multiple runs, with multiple operators and with multiple replicates per run.

Table 4: Results Summary HCV on the cobas 6800/8800   

<table><tr><td rowspan=2 colspan=1>ConcentrationHCV (IU/mL)</td><td rowspan=1 colspan=4>HCV</td></tr><tr><td rowspan=1 colspan=2>Number of ValidReplicates</td><td rowspan=1 colspan=1>Number of Positives</td><td rowspan=1 colspan=1>% Positives</td></tr><tr><td rowspan=1 colspan=1>20.0</td><td rowspan=1 colspan=2>66</td><td rowspan=1 colspan=1>66</td><td rowspan=1 colspan=1>100.00%</td></tr><tr><td rowspan=1 colspan=1>15.0</td><td rowspan=1 colspan=2>66</td><td rowspan=1 colspan=1>63</td><td rowspan=1 colspan=1>95.45%</td></tr><tr><td rowspan=1 colspan=1>10.0</td><td rowspan=1 colspan=2>66</td><td rowspan=1 colspan=1>57</td><td rowspan=1 colspan=1>86.36%</td></tr><tr><td rowspan=1 colspan=1>8.0</td><td rowspan=1 colspan=2>66</td><td rowspan=1 colspan=1>57</td><td rowspan=1 colspan=1>86.36%</td></tr><tr><td rowspan=1 colspan=1>5.0</td><td rowspan=1 colspan=2>66</td><td rowspan=1 colspan=1>46</td><td rowspan=1 colspan=1>69.70%</td></tr><tr><td rowspan=1 colspan=1>2.5</td><td rowspan=1 colspan=2>66</td><td rowspan=1 colspan=1>27</td><td rowspan=1 colspan=1>40.91%</td></tr><tr><td rowspan=1 colspan=2>LoD by PROBIT analysis (95% HitRate)</td><td rowspan=1 colspan=3>13.70 IU/mL(95% CI: 11.19  18.31 )</td></tr><tr><td rowspan=1 colspan=2>LoD by Hit Rate</td><td rowspan=1 colspan=3>15 IU/mL</td></tr><tr><td rowspan=1 colspan=2>log10 Probit 95%</td><td rowspan=1 colspan=3>1.14 log10 IU/mL</td></tr></table>

Table 5: Results Summary on the cobas 5800   

<table><tr><td colspan="1" rowspan="2">ConcentrationHCV (IU/mL)</td><td colspan="3" rowspan="1">HCV</td></tr><tr><td colspan="1" rowspan="1">Number of ValidReplicates</td><td colspan="1" rowspan="1">Number of Positives</td><td colspan="1" rowspan="1">% Positives</td></tr><tr><td colspan="1" rowspan="1">20.0</td><td colspan="1" rowspan="1">66</td><td colspan="1" rowspan="1">65</td><td colspan="1" rowspan="1">98.48%</td></tr><tr><td colspan="1" rowspan="1">15.0</td><td colspan="1" rowspan="1">66</td><td colspan="1" rowspan="1">63</td><td colspan="1" rowspan="1">95.45%</td></tr><tr><td colspan="1" rowspan="1">10.0</td><td colspan="1" rowspan="1">66</td><td colspan="1" rowspan="1">59</td><td colspan="1" rowspan="1">89.39%</td></tr><tr><td colspan="1" rowspan="1">8.0</td><td colspan="1" rowspan="1">66</td><td colspan="1" rowspan="1">55</td><td colspan="1" rowspan="1">83.33%</td></tr><tr><td>5.0</td><td>65*</td><td>47</td><td>72.31%</td></tr><tr><td>2.5</td><td>66</td><td>30</td><td>45.45%</td></tr><tr><td>LoD by PROBIT analysis (95% Hit Rate)</td><td></td><td>14.41 IU/mL (95% CIl: 11.55 - 19.99)</td><td></td></tr><tr><td>LoD by Hit Rate</td><td></td><td>15 IU/mL</td><td></td></tr><tr><td>log10 Probit 95%</td><td colspan="3">1.16 log 10 IU/mL</td></tr></table>

\*one (1) sample was invalid

# 7. Assay Cut-Off:

See P150015

# 8. Carry-Over:

Sample carryover on the cobas 5800 was evaluated with the cobas HBV assay. Alternating negative and HBV high positive samples were processed and detected with several checkerboard configurations on the cobas 5800 Systems. A high titer HBV plasmid sample used in this study was prepared by diluting in pooled negative EDTA plasma to yield a mean titer of $9 . 1 1 \mathrm { { x } 1 0 ^ { 8 } \mathrm { { I U } / m L } }$ .

Eighty-three runs were performed yielding 1926 valid sample results.

Table 6: Summary of Cross Contamination Study   

<table><tr><td rowspan=2 colspan=1></td><td rowspan=2 colspan=1>Total Positive Resultsfrom Negative Samples</td><td rowspan=1 colspan=2>Total Samples</td><td rowspan=2 colspan=1>CrossContamination Rate</td><td rowspan=2 colspan=1>Acceptance Criteria</td></tr><tr><td rowspan=1 colspan=1>positivesamples</td><td rowspan=1 colspan=1>negativesamples</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>960</td><td rowspan=1 colspan=1>966</td><td rowspan=1 colspan=1>0.104 %(upper one-sided95% Cl of 0.490%)</td><td rowspan=1 colspan=1>met</td></tr></table>

# B Comparison Studies:

# 1. Method Comparison with Predicate Device:

A method comparison study was conducted using 150 archived, well-characterized HCV positive plasma specimens and $3 0 \mathrm { H C V }$ negative individual plasma specimens. Some of the individual positive specimens were diluted to achieve the desired number of samples at specified concentrations: ${ \sim } 5 0 \mathrm { \ : H C V }$ specimens between $1 . 5 \mathrm { E } { + } 0 1$ to $3 \mathrm { E + } 0 3 \mathrm { \Pi \mathrm { U / m L } }$ , ${ \sim } 5 0$ specimens between $3 \mathrm { E } { + } 0 3$ to $5 \mathrm { E + } 0 5 \mathrm { \Pi U / m L }$ and ${ \sim } 5 0$ specimens between $5 \mathrm { E } { + } 0 5$ to $1 \mathrm { E } { + } 0 8$ $\mathrm { I U / m L }$ . Each specimen was tested on the cobas 5800 System at three different sites (1 internal and 2 external) and on a cobas 6800/8800 System at one site (internal) using three reagent lots.

Results:

For the 30 samples with target not detected (TND) results by cobas 6800/8800, one sample (1/30) by cobas 5800 test at one site was positive (with concentration ${ < } 1 . 5 \mathrm { E } { + } 0 1$ , out of the linear range). At the other two sites the $\mathrm { N P A } { = } 1 0 0 \%$ (30/30).

All of the positive samples were within the linear range of $1 . 5 \mathrm { E } { + } 1 \ \mathrm { I U / m L }$ to $1 \mathrm { E } { + } 8 \ \mathrm { I U / m L }$ by cobas 6800/8800 ( $1 5 0 / 1 5 0 = 1 0 0 \% )$ and by cobas 5800 ( $4 5 0 / 4 5 0 { = } 1 0 0 \% )$ . Among the three categories (low, medium, and high viral loads) within the linear range, the test results between the two systems were highly concordant.

Table 7 summarizes the parameter estimates (slope and intercept) of Deming regression between cobas 5800 and cobas 6800/8800 by site.

Table 7: Parameter estimates from Deming regression analysis by site   

<table><tr><td rowspan=1 colspan=1>Site</td><td rowspan=1 colspan=1>Parameter</td><td rowspan=1 colspan=1>Number of Pairswithin LinearRange</td><td rowspan=1 colspan=1>ParameterEstimate</td><td rowspan=1 colspan=1>StandardError</td><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=1>r2</td></tr><tr><td rowspan=2 colspan=1>1</td><td rowspan=1 colspan=1>Intercept</td><td rowspan=2 colspan=1>150</td><td rowspan=1 colspan=1>-0.044</td><td rowspan=1 colspan=1>0.029</td><td rowspan=1 colspan=1>(-0.102, 0.014)</td><td rowspan=2 colspan=1>0.996</td></tr><tr><td rowspan=1 colspan=1>Slope</td><td rowspan=1 colspan=1>1.011</td><td rowspan=1 colspan=1>0.006</td><td rowspan=1 colspan=1>( 1.000, 1.022)</td></tr><tr><td rowspan=2 colspan=1>2</td><td rowspan=1 colspan=1>Intercept</td><td rowspan=2 colspan=1>150</td><td rowspan=1 colspan=1>-0.055</td><td rowspan=1 colspan=1>0.037</td><td rowspan=1 colspan=1>(-0.127, 0.018)</td><td rowspan=2 colspan=1>0.996</td></tr><tr><td rowspan=1 colspan=1>Slope</td><td rowspan=1 colspan=1>1.014</td><td rowspan=1 colspan=1>0.007</td><td rowspan=1 colspan=1>(1.000, 1.028)</td></tr><tr><td rowspan=2 colspan=1>3</td><td rowspan=1 colspan=1>Intercept</td><td rowspan=2 colspan=1>150</td><td rowspan=1 colspan=1>-0.031</td><td rowspan=1 colspan=1>0.032</td><td rowspan=1 colspan=1>(-0.094, 0.032)</td><td rowspan=2 colspan=1>0.996</td></tr><tr><td rowspan=1 colspan=1>Slope</td><td rowspan=1 colspan=1>1.010</td><td rowspan=1 colspan=1>0.006</td><td rowspan=1 colspan=1>( 0.998, 1.023)</td></tr></table>

The figure below shows the scatter plot of the average of $\log _ { 1 0 } .$ -tranformed viral load concentration across 3 sites from the cobas 5800 versus $\log _ { 1 0 }$ -tranformed viral load concentration from the cobas 6800/8800 with fitted lines created by Deming regression for all sites combined using bootstrap method.

![](images/12dc5e6ac10eda53a2ad5287757360ed23993ac0271510760df54e65e33c5d0f.jpg)

Using Deming regression analysis, an estimate of the systematic bias between the log10- transformed viral load concentration of the two systems (cobas 5800 and cobas 6800/8800) was calculated at medical decision levels for each site. Jackknife method was used to estimate the $9 5 \%$ CI of systematic bias.

Table 8 presents the estimated systematic bias between the two systems at the medical decision points by site. The biases are very close to zero in all cases, ranging from -0.035 to 0.040 and are all below the SDs observed in the reproducibility study for cobas 6800/8800 (range from 0.098 to 0.244).

Table 8: Systematic bias at medical decision point by site   

<table><tr><td rowspan=1 colspan=1>Site</td><td rowspan=1 colspan=1>MedicalDecisionPoint(IU/mL)</td><td rowspan=1 colspan=1>Number ofPairswithin LinearRange</td><td rowspan=1 colspan=1>MedicalDecisionPoint in log10(IU/mL)</td><td rowspan=1 colspan=1>PredictedValue atMedicalDecision Pointin log10(IU/mL)1</td><td rowspan=1 colspan=1>Bias in log10(IU/mL)</td><td rowspan=1 colspan=1>95% CIof Bias</td></tr><tr><td rowspan=3 colspan=1>1</td><td rowspan=1 colspan=1>25</td><td rowspan=1 colspan=1>150</td><td rowspan=1 colspan=1>1.398</td><td rowspan=1 colspan=1>1.369</td><td rowspan=1 colspan=1>-0.029</td><td rowspan=1 colspan=1>(-0.072, 0.015)</td></tr><tr><td rowspan=1 colspan=1>800,000</td><td rowspan=1 colspan=1>150</td><td rowspan=1 colspan=1>5.903</td><td rowspan=1 colspan=1>5.924</td><td rowspan=1 colspan=1>0.021</td><td rowspan=1 colspan=1>( 0.004, 0.038)</td></tr><tr><td rowspan=1 colspan=1>6,000,000</td><td rowspan=1 colspan=1>150</td><td rowspan=1 colspan=1>6.778</td><td rowspan=1 colspan=1>6.808</td><td rowspan=1 colspan=1>0.030</td><td rowspan=1 colspan=1>( 0.007, 0.053)</td></tr><tr><td rowspan=3 colspan=1>2</td><td rowspan=1 colspan=1>25</td><td rowspan=1 colspan=1>150</td><td rowspan=1 colspan=1>1.398</td><td rowspan=1 colspan=1>1.363</td><td rowspan=1 colspan=1>-0.035</td><td rowspan=1 colspan=1>(-0.089, 0.019)</td></tr><tr><td rowspan=1 colspan=1>800,000</td><td rowspan=1 colspan=1>150</td><td rowspan=1 colspan=1>5.903</td><td rowspan=1 colspan=1>5.931</td><td rowspan=1 colspan=1>0.028</td><td rowspan=1 colspan=1>( 0.009, 0.046)</td></tr><tr><td rowspan=1 colspan=1>6,000,000</td><td rowspan=1 colspan=1>150</td><td rowspan=1 colspan=1>6.778</td><td rowspan=1 colspan=1>6.818</td><td rowspan=1 colspan=1>0.040</td><td rowspan=1 colspan=1>( 0.013, 0.067)</td></tr><tr><td rowspan=3 colspan=1>3</td><td rowspan=1 colspan=1>25</td><td rowspan=1 colspan=1>150</td><td rowspan=1 colspan=1>1.398</td><td rowspan=1 colspan=1>1.382</td><td rowspan=1 colspan=1>-0.016</td><td rowspan=1 colspan=1>(-0.064, 0.031)</td></tr><tr><td rowspan=1 colspan=1>800,000</td><td rowspan=1 colspan=1>150</td><td rowspan=1 colspan=1>5.903</td><td rowspan=1 colspan=1>5.934</td><td rowspan=1 colspan=1>0.031</td><td rowspan=1 colspan=1>( 0.013, 0.048)</td></tr><tr><td rowspan=1 colspan=1>6,000,000</td><td rowspan=1 colspan=1>150</td><td rowspan=1 colspan=1>6.778</td><td rowspan=1 colspan=1>6.818</td><td rowspan=1 colspan=1>0.040</td><td rowspan=1 colspan=1>( 0.015, 0.065)</td></tr><tr><td rowspan=1 colspan=7>Estimated from the linear equation established by Deming regression method.</td></tr></table>

A Bland-Altman (BA) plot between the results of the two systems (cobas 5800 and cobas 6800/8800 instrument) with allowable total difference (ATD) zone were created for each site and for all sites combined.

Table 9 shows the percentage of samples falling within the ATD zone at low, medium and high viral load ranges and over the entire range along with the one-sided $9 5 \%$ CIs for all sites combined. Over the entire range, $100 \%$ of the samples fell within the ATD zone and the lower bound of the one-sided $9 5 \%$ CI was $100 \%$ .

Table 9: Percentage of samples within the ATD zone and $\pm 0 . 5 \log 1 0$ -range - all sites combined   

<table><tr><td rowspan=1 colspan=1>Viral LoadRange Category</td><td rowspan=1 colspan=1>Viral LoadRange (IU/mL)</td><td rowspan=1 colspan=1>Percentage of Sampleswithin ATD Zone(95% One-Sided CI)</td><td rowspan=1 colspan=1>Percentage of Sampleswith differencewithin ±0.5 log10(95% One-Sided Cl)</td></tr><tr><td rowspan=1 colspan=1>Low</td><td rowspan=1 colspan=1>1.5E+1 - 3.0E+3</td><td rowspan=1 colspan=1>100.00% (47/47) (94.56%)</td><td rowspan=1 colspan=1>100.00% (47/47) (94.56%)</td></tr><tr><td rowspan=1 colspan=1>Medium</td><td rowspan=1 colspan=1>3.0E+3 - 5.0E+5</td><td rowspan=1 colspan=1>100.00% (46/46) (94.45%)</td><td rowspan=1 colspan=1>100.00% (46/46) (94.45%)</td></tr><tr><td rowspan=1 colspan=1>High</td><td rowspan=1 colspan=1>5.0E+5 - 1.0E+8</td><td rowspan=1 colspan=1>100.00% (57/57) (95.47%)</td><td rowspan=1 colspan=1>100.00% (57/57) (95.47%)</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>100.00% (150/150) (98.23%)</td><td rowspan=1 colspan=1>100.00% (150/150) (98.23%)</td></tr></table>

# 2. Matrix Comparison:

See P150015

# C Clinical Studies:

1. Clinical Sensitivity: See P150015

2. Clinical Specificity: See P150015

# D Expected Values/Reference Range:

See P150015

# E Reagent Stability:

Studies were conducted to determine the On-Board and Open Kit stability for the cobas HCV assay when used on the cobas 5800 System. The results of the studies demonstrate the cobas HCV reagent kit cassette for use on the cobas 5800 System are stable for up to 90 days at $2 - 8 ^ { \circ } \mathrm { C }$ once opened and remain stable for up to 36 days at $2 5 \mathrm { { ^ \circ C } }$ (On Board Stability), and can be used up to 40 times.

# VIII Proposed Labeling:

The labeling supports the finding of substantial equivalence for this device.

# IX Conclusion:

The submitted information in this premarket notification is complete and supports a substantial equivalence decision.