# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATIONDECISION SUMMARYASSAY AND INSTRUMENT

I Background Information:

# B Applicant

Hologic, Inc.

# C Proprietary and Established Names

Panther Fusion SARS-CoV-2/Flu A/B/RSV Assay

# D Regulatory Information

<table><tr><td rowspan=1 colspan=1>ProductCode(s)</td><td rowspan=1 colspan=1>Classification</td><td rowspan=1 colspan=1>RegulationSection</td><td rowspan=1 colspan=1>Panel</td></tr><tr><td rowspan=1 colspan=1>QOF</td><td rowspan=1 colspan=1>Class II</td><td rowspan=1 colspan=1>21 CFR 866.3981 - DeviceTo Detect And IdentifyNucleic Acid Targets InRespiratory SpecimensFrom Microbial AgentsThat Cause The SARS-Cov-2 RespiratoryInfection And OtherMicrobial Agents When InA Multi-Target Test</td><td rowspan=1 colspan=1>MI - Microbiology</td></tr><tr><td rowspan=1 colspan=1>00I</td><td rowspan=1 colspan=1>Class II</td><td rowspan=1 colspan=1>21 CFR 862.2570 -Instrumentation forclinical multiplex testsystems</td><td rowspan=1 colspan=1>CH - Clinical Chemistry</td></tr></table>

# II Submission/Device Overview:

# A Purpose for Submission:

The purpose of this submission is to show that the Panther Fusion SARS-CoV-2/Flu A/B/RSV Assay (for use on the Panther Fusion system) is substantially equivalent to the BioFire Respiratory Panel 2.1 (RP2.1) (DEN200031) and to obtain clearance for the Panther Fusion SARS-CoV-2/Flu A/B/RSV Assay.

# B Measurand:

The Panther Fusion SARS-CoV-2/Flu A/B/RSV Assay detects SARS-CoV-2, influenza A, influenza B, and respiratory syncytial virus RNA isolated from nasopharyngeal swab specimens from patients with signs and symptoms of respiratory infection.

# C Type of Test:

This assay is a multiplex nucleic acid assay that detects and differentiates SARS-CoV-2, influenza A, influenza B, and RSV through nucleic acid extraction, amplification, and detection using real-time RT-PCR. All steps of the assay are automated, after the manual addition of sample into the sample lysis tube (SLT) and performed within the Panther and Panther Fusion system.

# III Intended Use/Indications for Use:

A Intended Use(s): See Indications for Use below.

B Indication(s) for Use: Panther Fusion SARS-CoV-2/Flu A/B/RSV Assay:

The Panther Fusion SARS-CoV-2/Flu A/B/RSV assay is a fully automated multiplexed real-time polymerase chain reaction (RT-PCR) in vitro diagnostic test intended for the qualitative detection and differentiation of severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), influenza A virus (Flu A), influenza B virus (Flu B), and respiratory syncytial virus (RSV). Nucleic acids are isolated and purified from nasopharyngeal (NP) specimens obtained from individuals exhibiting signs and symptoms of a respiratory tract infection. Clinical signs and symptoms of respiratory viral infection due to SARS-CoV-2, influenza, and RSV can be similar. This assay is intended to aid in the differential diagnosis of SARS-CoV-2, Flu A, Flu B, and RSV infections in humans and is not intended to detect influenza C virus infections.

Nucleic acids from the viral organisms identified by this test are generally detectable in NP specimens during the actute phase of infection. The detection and identification of specific viral nucleic acids from individuals exhibiting signs and symptoms of respiratory tract infection are indicative of the presence of the identified virus and aids in diagnosis if used in conjunction with other clinical and epidemiological information, and laboratory findings. The results of this test should not be used as the sole basis for diagnosis, treatment, or other patient management decisions.

Positive results do not rule out coinfection with other organisms. The organism(s) detected by the Panther Fusion SARS-CoV-2/Flu A/B/RSV assay may not be the definite cause of disease.

Negative results do not preclude SARS-CoV-2, influenza A virus, influenza B virus, or RSV infections. This assay is designed for use on the Panther Fusion system.

The Hologic RespDirect Collection Kit can be used to collect NP specimens for testing with the Panther Fusion SARS-CoV-2/Flu A/B/RSV Assay. Additionally, other NP swabs (not provided with the Hologic RespDirect Collection Kit) may be used to collect NP specimens in $3 \mathrm { m L }$ of VTM or UTM.

# Ancillary Collection Kit:

RespDirect Collection Kit

The Hologic RespDirect Collection Kit is intended to be used for the collection of nasopharyngeal (NP) swab specimens (collected by a healthcare provider) for testing with the Panther Fusion SARS-CoV-2/Flu A/B/RSV Assay.

C Special Conditions for Use Statement(s): Rx - For Prescription Use Only

For in vitro diagnostic use only

D Special Instrument Requirements: For use with the Panther Fusion System, only.

IV Device/System Characteristics:

A Device Description:

# Panther Fusion SARS-CoV-2/Flu A/B/RSV Assay

The Panther Fusion SARS-CoV-2/Flu A/B/RSV assay is a multiplex real-time reverse transcriptase PCR (RT-PCR) in vitro diagnostic test developed for use on the fully automated Panther Fusion system to detect and differentiate SARS-CoV-2, influenza A, influenza B, and respiratory syncytial virus (RSV) directly from nasopharyngeal swab specimens.

The Panther Fusion SARS-CoV-2/Flu A/B/RSV assay cartridge contains the same sample preparation and PCR reaction chemistry as the previously cleared Panther Fusion Flu A/B/RSV assay (K171963). To accommodate addition of the SARS-CoV-2 reagents (primers/probes) to the multiplexed reagents, minor changes were made to the previously cleared analyte primer/probe concentrations and RFU cutoffs. Additionally, the fluorophore for Flu B was changed from ROX to RED647 to accommodate the addition of SARS-CoV-2 to the assay.

The Panther Fusion SARS-CoV-2/Flu A/B/RSV assay involves the following steps:

a. Sample lysis - Prior to processing and testing on the Panther Fusion system, specimens are transferred to a Specimen Lysis Tube (SLT) containing specimen transport media (STM). Alternatively, samples can be collected with the RespDirect Collection kit which contains enhanced specimen transport media (eSTM). STM and eSTM lyse the cells, release target nucleic acid and protect them from degradation during storage.

b. Nucleic acid capture and elution - These steps take place in a single tube on the Panther Fusion system. The eluate is transferred to the Panther Fusion system reaction tube containing the assay reagents. The Internal Control-S (IC-S) is added to each test specimen and controls via the working Panther Fusion Capture Reagent-S (wFCR-S). The IC-S in the reagent is used to monitor specimen processing, amplification, and detection. Magnetic particles with covalently bound oligonucleotides mediate the nucleic acid capture. Capture oligonucleotides hybridize to total nucleic acid in the test specimen. Hybridized nucleic acid is then separated from the lysed specimen in a magnetic field. Wash and aspiration steps remove extraneous components debris from the reaction tube. The elution step elutes purified nucleic acid.

c. Elution transfer and multiplex RT-PCR - Eluted nucleic acid is transferred to a Panther Fusion reaction tube already containing oil and reconstituted master mix. A reverse transcriptase generates a DNA copy of the target sequence. Target specific forward and reverse primers and probes then amplify targets while simultaneously detecting and discriminating multiple target types via multiplex RT-PCR. The Panther Fusion system compares the fluorescence signal to a predetermined cut-off to produce a qualitative result for the presence or absence of the analyte. The positive result for each analyte will be accompanied by the cycle threshold (Ct value).

# Hologic RespDirect Collection Kit

An ancillary collection kit that consists of the RespDirect Swab, intended for collection of NP specimens, and the enhanced Direct Load Tube (eDLT), containing enhanced specimen transport media (eSTM). This transport media lyses cells, releasing target nucleic acids and protecting them from degradation during storage.

# B Principle of Operation:

The assay detects viral nucleic acids that have been extracted from a patient respiratory sample. A multiplex Real-time RT-PCR reaction is carried out under optimized conditions generating amplicons for SARS-CoV-2, influenza A, influenza B, and RSV. The Internal Control-S (IC-S) is added to each test specimen before processing to act as a control for specimen processing, amplification, and detection. Identification of SARS-CoV-2, influenza A, influenza B, RSV, and the IC-S occurs by the use of target-specific primers and fluorescent-labeled probes that hybridize to conserved regions in the viral genomes (Table 1).

Table 1. Assay Primer and Probe Targets   

<table><tr><td rowspan=1 colspan=1>Analyte</td><td rowspan=1 colspan=1>Gene Targeted</td><td rowspan=1 colspan=1>Instrument Channel</td></tr><tr><td rowspan=1 colspan=1>SARS-CoV-2</td><td rowspan=1 colspan=1>ORF1ab</td><td rowspan=1 colspan=1>ROX</td></tr><tr><td rowspan=1 colspan=1>Influenza A Virus</td><td rowspan=1 colspan=1>Matrix</td><td rowspan=1 colspan=1>FAM</td></tr><tr><td rowspan=1 colspan=1>Respiratory Syncytial Virus A/B</td><td rowspan=1 colspan=1>Matrix</td><td rowspan=1 colspan=1>HEX</td></tr><tr><td rowspan=1 colspan=1>Influenza B Virus</td><td rowspan=1 colspan=1>Matrix</td><td rowspan=1 colspan=1>RED647</td></tr><tr><td rowspan=1 colspan=1>Internal Control-S</td><td rowspan=1 colspan=1>Not applicable*</td><td rowspan=1 colspan=1>RED677</td></tr></table>

\*Internal Control-S is a non-infectious synthetic nucleic acid sequence that is extracted and detected through targeted primers and probes.

# C Instrument Description Information:

1. Instrument Name: Panther System and Panther Fusion System, software version 7.2.5

2. Specimen Identification: Specimen identification is entered via barcode.

3. Specimen Sampling and Handling: Nasopharyngeal swab (NPS) specimens collected in transport media.

4. Calibration:

Real Time Fluorometers (RTF) undergo a single calibration during manufacturing. No additional calibration is performed by the end user.

5. Quality Control:

The assay contains an internal control (IC-S) which is added to each test specimen via the working Panther Fusion Capture Reagent-S (wFCR-S). The IC-S is used to monitor specimen processing, amplification, and detection.

Two external controls are also included with this assay in a single use vial, the Panther Fusion SARS-CoV-2/Flu A/B/RSV Positive Control and the Panther Fusion Negative Control. The controls were validated in the analytical and clinical studies.

# V Substantial Equivalence Information:

A Predicate Device Name(s): BioFire Respiratory Panel 2.1 (RP2.1)

B Predicate 510(k) Number(s): DEN200031

C Comparison with Predicate(s):

<table><tr><td colspan="1" rowspan="1">Device &amp; PredicateDevice(s):</td><td colspan="1" rowspan="1">K222736</td><td colspan="1" rowspan="1">DEN200031</td></tr><tr><td colspan="1" rowspan="1">Device Trade Name</td><td colspan="1" rowspan="1">Panther Fusion SARS-CoV-2/Flu A/B/RSV assay</td><td colspan="1" rowspan="1">BioFire Respiratory Panel 2.1(RP2.1)</td></tr><tr><td colspan="1" rowspan="1">Regulation Number/Name</td><td colspan="1" rowspan="1">Same</td><td colspan="1" rowspan="1">21 CFR 866.3981; Multi-TargetRespiratory Specimen NucleicAcid Test Including SARS-CoV-2 And Other MicrobialAgents</td></tr><tr><td colspan="1" rowspan="1">Product Code(s)</td><td colspan="1" rowspan="1">QOF, OOI</td><td colspan="1" rowspan="1">QOF</td></tr><tr><td colspan="1" rowspan="1">Prescription Use Only</td><td colspan="1" rowspan="1">Same</td><td colspan="1" rowspan="1">Yes</td></tr><tr><td colspan="1" rowspan="1">Intended Use</td><td colspan="1" rowspan="1">The Panther Fusion SARS-CoV-2/Flu A/B/RSV assay is afully automated multiplexedreal-time polymerase chainreaction (RT-PCR) in vitrodiagnostic test intended for thequalitative detection anddifferentiation of severe acuterespiratory syndromecoronavirus 2 (SARS-CoV-2),influenza A virus (Flu A),influenza B virus (Flu B), andrespiratory syncytial virus(RSV). Nucleic acids areisolated and purified from</td><td colspan="1" rowspan="1">The BioFire Respiratory Panel2.1 (RP2.1) is a PCR-basedmultiplexed nucleic acid testintended for use with theBioFire FilmArray 2.0 orBioFire FilmArray Torchsystems for the simultaneousqualitative detection andidentification of multiplerespiratory viral and bacterialnucleic acids in nasopharyngealswabs (NPS) obtained fromindividuals suspected ofrespiratory tract infections,including COVID-19.</td></tr><tr><td></td><td>nasopharyngeal (NP) specimens obtained from individuals exhibiting signs and symptoms of a respiratory tract infection. Clinical signs and symptoms of respiratory viral infection due to SARS-CoV-2, influenza, and RSV can be similar. This assay is intended to aid in the differential diagnosis of SARS-CoV-2, Flu A, Flu B, and RSV infections in humans and is not intended to detect influenza C virus infections. Nucleic acids from the viral organisms identified by this test are generally detectable in NP specimens during the acute phase of infection. The detection and identification of specific viral nucleic acids from individuals exhibiting signs and symptoms of respiratory tract infection are indicative of the presence of the identified virus and aids in diagnosis if used in conjunction with other clinical and epidemiological information, and laboratory findings. The results of this test should not be used as the sole basis for diagnosis, treatment, or other patient management decisions. Positive results do not rule out</td><td>The following organism types and subtypes are identified using the BioFire RP2.1: Adenovirus, •Coronavirus 229E, • Coronavirus HKU1, Coronavirus NL63, • Coronavirus OC43, Severe Acute Respiratory Syndrome Coronavirus (SARS- CoV-2), Human Metapneumovirus, Human Rhinovirus/Enterovirus, Influenza A, including subtypes H1, H1-2009, and H3, Influenza B, Parainfluenza Virus 1, •Parainfluenza Virus 2, •Parainfluenza Virus 3, Parainfluenza Virus 4, Respiratory Syncytial Virus, Bordetella parapertussis (IS1001), Bordetella pertussis (ptxP), • Chlamydia pneumoniae, and •Mycoplasma pneumoniae Nucleic acids from the respiratory viral and bacterial organisms identified by this test are generally detectable in NPS specimens during the acute phase of infection. The detection and identification of specific viral and bacterial nucleic acids from individuals exhibiting signs and/or symptoms of respiratory infection is</td></tr></table>

<table><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">The Hologic RespDirectCollection Kit can be used tocollect NP specimens fortesting with the Panther FusionSARS-CoV-2/Flu A/B/RSVAssay. Additionally, other NPswabs (not provided with theHologic RespDirect CollectionKit) may be used to collect NPspecimens in 3mL of VTM orUTM.Ancillary Collection Kit:RespDirect Collection KitThe Hologic RespDirectCollection Kit is intended to beused for the collection ofnasopharyngeal (NP) swabspecimens (collected by ahealthcare provider) for testingwith the Panther Fusion SARS-CoV-2/Flu A/B/RSV Assay.</td><td colspan="1" rowspan="1">infection with pathogens that arenot detected by this test, orlower respiratory tract infectionthat may not be detected by anNPS specimen. Positive resultsdo not rule out coinfection withother organisms. The agent(s)detected by the BioFire RP2.1may not be the definite cause ofdisease. Additional laboratorytesting (e.g. bacterial and viralculture, immunofluorescence,and radiography) may benecessary when evaluating apatient with possible respiratorytract infection.</td></tr><tr><td colspan="1" rowspan="1">Intended User</td><td colspan="1" rowspan="1">Same</td><td colspan="1" rowspan="1">Professional use</td></tr><tr><td colspan="1" rowspan="1">Principle of Operation</td><td colspan="1" rowspan="1">Same</td><td colspan="1" rowspan="1">Reverse transcriptasemultiplexed polymerase chainreaction test</td></tr><tr><td colspan="1" rowspan="1">Specimen Types</td><td colspan="1" rowspan="1">Same</td><td colspan="1" rowspan="1">Nasopharyngeal swab (NPS)specimens</td></tr><tr><td colspan="1" rowspan="1">Assay Controls</td><td colspan="1" rowspan="1">Same</td><td colspan="1" rowspan="1">Internal and external controls</td></tr><tr><td colspan="1" rowspan="1">Analyte Targets</td><td colspan="1" rowspan="1">SARS-CoV-2, Flu A, Flu B,RSV (RSV A and RSV B)</td><td colspan="1" rowspan="1">Adenovirus, Coronaviruses(including 229E, HKU1, NL63,OC43), SARS-CoV-2, HumanMetapneumovirus, HumanRhinovirus/Enterovirus,Influenza A (including subtypesH1, H1-2009, and H3),Influenza B, ParainfluenzaViruses (including 1, 2, 3, and4), Respiratory Syncytial Virus(RSV), Bordetellaparapertussis, Bordetellapertussis, Chlamydiapneumoniae, Mycoplasmapneumoniae</td></tr><tr><td colspan="1" rowspan="1">Flu A Subtyping</td><td colspan="1" rowspan="1">No</td><td colspan="1" rowspan="1">Yes</td></tr><tr><td colspan="1" rowspan="1">Platform</td><td colspan="1" rowspan="1">Automated nucleic acidamplification platform.Uses Panther Fusion system forall steps including nucleic acid</td><td colspan="1" rowspan="1">Automated nucleic acidamplification platform.Uses BioFire FilmArray 2.0 orBioFire FilmArray Torch</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">extraction, amplification,detection, and resultprocessing.</td><td colspan="1" rowspan="1">systems including integratedsample preparation,amplification, detection, andanalysis.</td></tr><tr><td colspan="1" rowspan="1">Time to Obtain TestResults</td><td colspan="1" rowspan="1">~ 2.5 hours</td><td colspan="1" rowspan="1">~ 45 minutes</td></tr></table>

# VI Standards/Guidance Documents Referenced:

Class II Special Controls as per 21 CFR 866.3981 Guidance for Industry and Food and Drug Administration Staff, Recommended Content and Format of Non-Clinical Bench Performance Testing Information in Premarket Submissions (December 20, 2019)   
• Guidance for Industry and Food and Drug Administration Staff, eCopy Program for Medical Device Submissions (April 27, 2020)   
• Guidance for Industry and FDA Staff, Format for Traditional and Abbreviated 510(k)s (September 13, 2019) Guidance for Industry and Food and Drug Administration Staff, Refuse to Accept Policy for 510(k)s (April 21, 2022) Guidance for Industry and Food and Drug Administration Staff, The 510(k) Program: Evaluating Substantial Equivalence in Premarket Notifications [510(k)] (July 28, 2014)   
• Guidance for Industry and FDA Staff, Establishing the Performance Characteristics of In Vitro Diagnostic Devices for the Detection or Detection and Differentiation of Influenza Viruses (July 7, 2011)

# VII Performance Characteristics (if/when applicable):

# A Analytical Performance:

1. Precision/Reproducibility:

a. Within-Laboratory Precision

Within-laboratory precision was examined at one site using the Panther Fusion SARSCoV-2/Flu A/B/RSV assay. A total of 5 contrived panels containing known quantities of the target analytes were prepared in negative clinical NPS matrix. The viral materials used to generate the positive panel members are denoted in Table 2. The contrived positive panels consisted of combinations of the target analytes at low concentrations (2x LoD) and moderate concentrations (5x LoD) (Table 3). The study was conducted with two operators, three lots of reagents, and three Panther Fusion instruments over the course of 12 days. Each panel member was tested in duplicate by each operator on two different reagent lots on all 12 days generating a total of 96 replicates per panel member (2 operators x 2 replicates per test event x 2 lots x 12 days).

Table 2. Viral Strains for Within-Laboratory Precision Study   

<table><tr><td rowspan=1 colspan=1>Description</td><td rowspan=1 colspan=1>Vendor</td><td rowspan=1 colspan=1>Catalog Number</td></tr><tr><td rowspan=1 colspan=1>SARS-CoV-2 (USA-WA1/2020)</td><td rowspan=1 colspan=1>BEI</td><td rowspan=1 colspan=1>NR-52281</td></tr><tr><td rowspan=1 colspan=1>Influenza A/H3N2/Kansas/14/17</td><td rowspan=1 colspan=1>Zeptometrix</td><td rowspan=1 colspan=1>0810586CF</td></tr><tr><td rowspan=1 colspan=1>Influenza B/Victoria/Washington/02/19</td><td rowspan=1 colspan=1>Zeptometrix</td><td rowspan=1 colspan=1>0810611CF</td></tr><tr><td rowspan=1 colspan=1>RSV B/ CH93(18)-18</td><td rowspan=1 colspan=1>Zeptometrix</td><td rowspan=1 colspan=1>0810040CF</td></tr></table>

Table 3. Precision Study Sample Panel   

<table><tr><td rowspan=1 colspan=1>Panel ID</td><td rowspan=1 colspan=1>Description</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>Negative (unspiked)</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>SARS-CoV-2 (2x) / Flu A (2x)</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>Flu B (2x) / RSV (2x)</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>SARS-CoV-2 (5x) / Flu A (5x)</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>Flu B (5x)/ RSV (5x)</td></tr></table>

The qualitative (i.e., $\%$ agreement with expected results) and quantitative results from the study are illustrated in Table 4 and Table 5, respectively.

Table 4. Within-Laboratory Precision Study - Qualitive Results   

<table><tr><td rowspan=1 colspan=1>Target</td><td rowspan=1 colspan=1>Panel ID</td><td rowspan=1 colspan=1>Panel Conc.</td><td rowspan=1 colspan=1>% Positive(pos n/ valid n)</td><td rowspan=1 colspan=1>% Agreement withExpected Results/(95% CI)</td></tr><tr><td rowspan=3 colspan=1>SARS-CoV-2</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>Low positive</td><td rowspan=1 colspan=1>100% (96/96)</td><td rowspan=1 colspan=1>100% (96.2-100%)</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>Mod. positive</td><td rowspan=1 colspan=1>100% (96/96)</td><td rowspan=1 colspan=1>100%(96.2-100%)</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>1.0% (1/96)</td><td rowspan=1 colspan=1>99.0%(94.3-99.8%)</td></tr><tr><td rowspan=3 colspan=1>Flu A</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>Low positive</td><td rowspan=1 colspan=1>100% (96/96)</td><td rowspan=1 colspan=1>100%(96.2-100%)</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>Mod. positive</td><td rowspan=1 colspan=1>100% (96/96)</td><td rowspan=1 colspan=1>100%(96.2-100%)</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>0% (0/96)</td><td rowspan=1 colspan=1>100%(96.2-100%)</td></tr><tr><td rowspan=3 colspan=1>Flu B</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>Low positive</td><td rowspan=1 colspan=1>100% (96/96)</td><td rowspan=1 colspan=1>100%(96.2-100%)</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>Mod. positive</td><td rowspan=1 colspan=1>100% (96/96)</td><td rowspan=1 colspan=1>100%(96.2-100%)</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>0% (0/96)</td><td rowspan=1 colspan=1>100%(96.2-100%)</td></tr><tr><td rowspan=3 colspan=1>RSV</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>Low positive</td><td rowspan=1 colspan=1>100% (96/96)</td><td rowspan=1 colspan=1>100%(96.2-100%)</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>Mod. positive</td><td rowspan=1 colspan=1>100% (96/96)</td><td rowspan=1 colspan=1>100%(96.2-100%)</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>0% (0/96)</td><td rowspan=1 colspan=1>100% (96.2-100%)</td></tr></table>

Note: Results are shown only for the intended targets. Panel members co-spiked with two different targets are presented twice.

All low (2x) and moderate (5x) panel members were $100 \%$ positive for the spiked target analytes. The negative panel member was $0 . 0 \%$ positive for Flu A, Flu B, and RSV, while $1 . 0 \%$ positive for SARS-CoV-2 (1/96) (Table 4, above).

Table 5. Within-Laboratory Precision Study – Ct Signal Variability Analysis Results   

<table><tr><td colspan="1" rowspan="2">Panel ID</td><td colspan="1" rowspan="2">Target</td><td colspan="1" rowspan="2">Mean(Ct)</td><td colspan="2" rowspan="1">BetweenReagentLots</td><td colspan="2" rowspan="1">BetweenInstrument</td><td colspan="2" rowspan="1">BetweenOperators</td><td colspan="2" rowspan="1">BetweenDays</td><td colspan="2" rowspan="1">BetweenRuns</td><td colspan="2" rowspan="1">Within Run</td><td colspan="2" rowspan="1">Total</td></tr><tr><td colspan="1" rowspan="1">SD</td><td colspan="1" rowspan="1">%CV</td><td colspan="1" rowspan="1">SD</td><td colspan="1" rowspan="1">%CV</td><td colspan="1" rowspan="1">SD</td><td colspan="1" rowspan="1">%CV</td><td colspan="1" rowspan="1">SD</td><td colspan="1" rowspan="1">%CV</td><td colspan="1" rowspan="1">SD</td><td colspan="1" rowspan="1">%CV</td><td colspan="1" rowspan="1">SD</td><td colspan="1" rowspan="1">%CV</td><td colspan="1" rowspan="1">SD</td><td colspan="1" rowspan="1">%CV</td></tr><tr><td colspan="1" rowspan="1">1(Negative)</td><td colspan="1" rowspan="1">IC</td><td colspan="1" rowspan="1">33.7</td><td colspan="1" rowspan="1">0.19</td><td colspan="1" rowspan="1">0.57</td><td colspan="1" rowspan="1">0.08</td><td colspan="1" rowspan="1">0.23</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.21</td><td colspan="1" rowspan="1">0.62</td><td colspan="1" rowspan="1">0.29</td><td colspan="1" rowspan="1">0.86</td><td colspan="1" rowspan="1">0.42</td><td colspan="1" rowspan="1">1.23</td></tr><tr><td colspan="1" rowspan="2">2 (LowPos)</td><td colspan="1" rowspan="1">Flu A</td><td colspan="1" rowspan="1">35.1</td><td colspan="1" rowspan="1">0.33</td><td colspan="1" rowspan="1">0.93</td><td colspan="1" rowspan="1">0.06</td><td colspan="1" rowspan="1">0.17</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.30</td><td colspan="1" rowspan="1">0.85</td><td colspan="1" rowspan="1">0.56</td><td colspan="1" rowspan="1">1.59</td><td colspan="1" rowspan="1">0.72</td><td colspan="1" rowspan="1">2.04</td></tr><tr><td colspan="1" rowspan="1">SARS-CoV-2</td><td colspan="1" rowspan="1">35.9</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.13</td><td colspan="1" rowspan="1">0.36</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.60</td><td colspan="1" rowspan="1">1.67</td><td colspan="1" rowspan="1">0.61</td><td colspan="1" rowspan="1">1.71</td></tr><tr><td colspan="1" rowspan="2">3 (LowPos)</td><td colspan="1" rowspan="1">Flu B</td><td colspan="1" rowspan="1">36.0</td><td colspan="1" rowspan="1">0.14</td><td colspan="1" rowspan="1">0.40</td><td colspan="1" rowspan="1">0.09</td><td colspan="1" rowspan="1">0.25</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.36</td><td colspan="1" rowspan="1">0.99</td><td colspan="1" rowspan="1">0.39</td><td colspan="1" rowspan="1">1.09</td></tr><tr><td colspan="1" rowspan="1">RSV</td><td colspan="1" rowspan="1">36.1</td><td colspan="1" rowspan="1">0.12</td><td colspan="1" rowspan="1">0.33</td><td colspan="1" rowspan="1">0.28</td><td colspan="1" rowspan="1">0.77</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.37</td><td colspan="1" rowspan="1">1.04</td><td colspan="1" rowspan="1">0.53</td><td colspan="1" rowspan="1">1.46</td><td colspan="1" rowspan="1">0.71</td><td colspan="1" rowspan="1">1.97</td></tr><tr><td colspan="1" rowspan="2">4 (ModPos</td><td colspan="1" rowspan="1">Flu A</td><td colspan="1" rowspan="1">33.9</td><td colspan="1" rowspan="1">0.23</td><td colspan="1" rowspan="1">0.66</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.19</td><td colspan="1" rowspan="1">0.56</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.47</td><td colspan="1" rowspan="1">1.37</td><td colspan="1" rowspan="1">0.55</td><td colspan="1" rowspan="1">1.63</td></tr><tr><td colspan="1" rowspan="1">SARS-CoV-2</td><td colspan="1" rowspan="1">34.7</td><td colspan="1" rowspan="1">0.21</td><td colspan="1" rowspan="1">0.62</td><td colspan="1" rowspan="1">0.16</td><td colspan="1" rowspan="1">0.45</td><td colspan="1" rowspan="1">0.06</td><td colspan="1" rowspan="1">0.17</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.45</td><td colspan="1" rowspan="1">1.30</td><td colspan="1" rowspan="1">0.52</td><td colspan="1" rowspan="1">1.51</td></tr><tr><td colspan="1" rowspan="2">5 (ModPos</td><td colspan="1" rowspan="1">Flu B</td><td colspan="1" rowspan="1">34.7</td><td colspan="1" rowspan="1">0.15</td><td colspan="1" rowspan="1">0.44</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.06</td><td colspan="1" rowspan="1">0.18</td><td colspan="1" rowspan="1">0.28</td><td colspan="1" rowspan="1">0.80</td><td colspan="1" rowspan="1">0.32</td><td colspan="1" rowspan="1">0.93</td></tr><tr><td colspan="1" rowspan="1">RSV</td><td colspan="1" rowspan="1">34.5</td><td colspan="1" rowspan="1">0.10</td><td colspan="1" rowspan="1">0.30</td><td colspan="1" rowspan="1">0.18</td><td colspan="1" rowspan="1">0.51</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.40</td><td colspan="1" rowspan="1">1.15</td><td colspan="1" rowspan="1">0.44</td><td colspan="1" rowspan="1">1.29</td></tr></table>

$_ \mathrm { C t } = \mathrm { c y c l e }$ threshold; $\mathrm { C V } =$ coefficient of variation; Mod $=$ moderate; Pos $=$ positive; $\mathrm { S D } =$ standard deviation . Note: Variability from some factors may be numerically negative, which can occur if the variability due to those factors is very small. When this occurs, $\mathrm { S D } { = } 0$ and $C V { = } 0 \%$

The mean and variability analysis between instruments, between reagent lots, between operators, between days, between runs, within runs, and overall (total) for Ct values is shown in Table 5. Overall $\% \mathrm { C V }$ was $\leq 2 . 0 4 \%$ . The greatest source of variability was from SARS-CoV-2 in the Flu A and SARS-CoV-2 low positive panel (within-run $\%$ CV of $1 . 6 7 \%$ ). Overall variability was low, and the study demonstrates assay variability within an acceptable range.

# b. Reproducibility

A reproducibility study was conducted at three testing sites using the Panther Fusion SARS-CoV-2/Flu A/B/RSV assay. The study incorporated potential sources of variation introduced by site (three testing sites), day (5 different days), and operator (two operators per site). One lot of Panther Fusion SARS-CoV-2/Flu A/B/RSV assay reagents was tested at three sites by two operators per site on one Panther Fusion instrument per site over five days.

A total of 6 contrived panels containing known quantities of various Panther Fusion SARS-CoV-2/Flu A/B/RSV assay analytes were prepared in negative clinical NPS matrix. The same viral materials used in the precision study (see Table 2) were also included in the reproducibility study. The four contrived positive panels consisted of combinations of the target analytes at low concentrations (i.e., $1 - 2 \mathbf { x }$ LoD) and moderate concentrations (3-5x LoD) (Table 6). Three replicates of each panel member were tested by each operator at each site on all 5 days of testing generating a total of 90 replicates per panel member. The qualitative and quantitative results of the study are illustrated in Table 7 and Table 8, respectively.

Table 6. Reproducibility Study Sample Panel   

<table><tr><td rowspan=1 colspan=1>Panel ID</td><td rowspan=1 colspan=1>Description</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>Negative (unspiked)</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>SARS-CoV-2 (1-2x) / Flu A (1-2x)</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>Flu B (1-2x) / RSV (1-2x)</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>SARS-CoV-2 (3-5x) / Flu A (3-5x)</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>Flu B (3-5x) / RSV (3-5x)</td></tr></table>

Table 7. Reproducibility Study – Qualitative Results   

<table><tr><td colspan="1" rowspan="2">Target</td><td colspan="1" rowspan="2">PanelID</td><td colspan="1" rowspan="2">PanelConc.</td><td colspan="4" rowspan="1">% Agreement with Expected Results/(95% CI)</td></tr><tr><td colspan="1" rowspan="1">Site 1</td><td colspan="1" rowspan="1">Site 2</td><td colspan="1" rowspan="1">Site 3</td><td colspan="1" rowspan="1">Overall</td></tr><tr><td colspan="1" rowspan="1">SARS-CoV-2</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">Lowpositive</td><td colspan="1" rowspan="1">100%(30/30)(88.7-100)</td><td colspan="1" rowspan="1">100%(30/30)(88.7-100)</td><td colspan="1" rowspan="1">100%(30/30)(88.7-100)</td><td colspan="1" rowspan="1">100%(90/90)(95.9-100)</td></tr><tr><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">Mod.positive</td><td colspan="1" rowspan="1">100%(30/30)(88.7-100)</td><td colspan="1" rowspan="1">100%(30/30)(88.7-100)</td><td colspan="1" rowspan="1">100%(30/30)(88.7-100)</td><td colspan="1" rowspan="1">100%(90/90)(95.9-100)</td></tr><tr><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">100%(30/30)(88.7-100)</td><td colspan="1" rowspan="1">100%(30/30)(88.7-100)</td><td colspan="1" rowspan="1">100%(30/30)(88.7-100)</td><td colspan="1" rowspan="1">100%(90/90)(95.9-100)</td></tr><tr><td colspan="1" rowspan="3">Flu A</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">Lowpositive</td><td colspan="1" rowspan="1">100%(30/30)(88.7-100)</td><td colspan="1" rowspan="1">96.7%(29/30)(83.3-99.4)</td><td colspan="1" rowspan="1">100%(30/30)(88.7-100)</td><td colspan="1" rowspan="1">98.9%(89/90)(94.0-99.8)</td></tr><tr><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">Mod.positive</td><td colspan="1" rowspan="1">100%(30/30)(88.7-100)</td><td colspan="1" rowspan="1">100%(30/30)(88.7-100)</td><td colspan="1" rowspan="1">100%(30/30)(88.7-100)</td><td colspan="1" rowspan="1">100%(90/90)(95.9-100)</td></tr><tr><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">100%(30/30)(88.7-100)</td><td colspan="1" rowspan="1">100%(30/30)(88.7-100)</td><td colspan="1" rowspan="1">100%(30/30)(88.7-100)</td><td colspan="1" rowspan="1">100%(90/90)(95.9-100)</td></tr><tr><td colspan="1" rowspan="3">Flu B</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">Lowpositive</td><td colspan="1" rowspan="1">100%(30/30)(88.7-100)</td><td colspan="1" rowspan="1">100%(30/30)(88.7-100)</td><td colspan="1" rowspan="1">100%(30/30)(88.7-100)</td><td colspan="1" rowspan="1">100%(90/90)(95.9-100)</td></tr><tr><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">Mod.positive</td><td colspan="1" rowspan="1">100%(30/30)(88.7-100)</td><td colspan="1" rowspan="1">100%(30/30)(88.7-100)</td><td colspan="1" rowspan="1">100%(30/30)(88.7-100)</td><td colspan="1" rowspan="1">100%(90/90)(95.9-100)</td></tr><tr><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">100%(30/30)(88.7-100)</td><td colspan="1" rowspan="1">96.7%(29/30)(83.3-99.4)</td><td colspan="1" rowspan="1">100%(30/30)(88.7-100)</td><td colspan="1" rowspan="1">98.9%(89/90)(94.0-99.8)</td></tr><tr><td colspan="1" rowspan="3">RSV</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">Lowpositive</td><td colspan="1" rowspan="1">100%(30/30)(88.7-100)</td><td colspan="1" rowspan="1">100%(30/30)(88.7-100)</td><td colspan="1" rowspan="1">100%(30/30)(88.7-100)</td><td colspan="1" rowspan="1">100%(90/90)(95.9-100)</td></tr><tr><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">Mod.positive</td><td colspan="1" rowspan="1">100%(30/30)(88.7-100)</td><td colspan="1" rowspan="1">100%(30/30)(88.7-100)</td><td colspan="1" rowspan="1">100%(30/30)(88.7-100)</td><td colspan="1" rowspan="1">100%(90/90)(95.9-100)</td></tr><tr><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">100%(30/30)(88.7-100)</td><td colspan="1" rowspan="1">100%(30/30)(88.7-100)</td><td colspan="1" rowspan="1">100%(30/30)(88.7-100)</td><td colspan="1" rowspan="1">100%(90/90)(95.9-100)</td></tr></table>

Mod $=$ moderate; Note: Results are shown only for the intended targets. Panel members co-spiked with two different targets are presented twice

The Panther Fusion SARS-CoV-2/Flu A/B/RSV assay demonstrated $100 \%$ agreement for SARS-CoV-2, Flu A, Flu B, and RSV moderate positive panel members. For low positive panel members, the assay yielded $100 \%$ agreement for SARS-CoV-2, Flu B, and RSV detection and $9 8 . 9 \%$ agreement for Flu A detection (Table 7 above). A lower agreement for low positive panel members was expected, since the analyte concentration of their panel member analytes ranged between 1x and $2 \mathbf { x }$ the limit of detection, which is expected to yield $2 9 5 \%$ detection rate. This performance is acceptable and demonstrates acceptable assay reproducibility.

Table 8. Reproducibility Study – Ct Signal Variability Analysis Results   

<table><tr><td colspan="1" rowspan="2">Target</td><td colspan="1" rowspan="2">PanelID/conc.</td><td colspan="1" rowspan="2">Mean(Ct)</td><td colspan="2" rowspan="1">BetweenSites</td><td colspan="2" rowspan="1">BetweenOperators/Run1</td><td colspan="2" rowspan="1">BetweenDays</td><td colspan="2" rowspan="1">Within Run</td><td colspan="2" rowspan="1">Total</td></tr><tr><td colspan="1" rowspan="1">SD</td><td colspan="1" rowspan="1">%CV</td><td colspan="1" rowspan="1">SD</td><td colspan="1" rowspan="1">%CV</td><td colspan="1" rowspan="1">SD</td><td colspan="1" rowspan="1">%CV</td><td colspan="1" rowspan="1">SD</td><td colspan="1" rowspan="1">%CV</td><td colspan="1" rowspan="1">SD</td><td colspan="1" rowspan="1">%CV</td></tr><tr><td colspan="1" rowspan="2">SARS-CoV-2</td><td colspan="1" rowspan="1">2-Low Pos</td><td colspan="1" rowspan="1">35.53</td><td colspan="1" rowspan="1">0.24</td><td colspan="1" rowspan="1">0.68</td><td colspan="1" rowspan="1">0.18</td><td colspan="1" rowspan="1">0.50</td><td colspan="1" rowspan="1">0.19</td><td colspan="1" rowspan="1">0.52</td><td colspan="1" rowspan="1">0.49</td><td colspan="1" rowspan="1">1.38</td><td colspan="1" rowspan="1">0.60</td><td colspan="1" rowspan="1">1.70</td></tr><tr><td colspan="1" rowspan="1">4-Mod Pos</td><td colspan="1" rowspan="1">34.15</td><td colspan="1" rowspan="1">0.11</td><td colspan="1" rowspan="1">0.32</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.40</td><td colspan="1" rowspan="1">1.16</td><td colspan="1" rowspan="1">0.41</td><td colspan="1" rowspan="1">1.20</td></tr><tr><td colspan="1" rowspan="2">Flu A</td><td colspan="1" rowspan="1">2-Low Pos</td><td colspan="1" rowspan="1">34.55</td><td colspan="1" rowspan="1">0.57</td><td colspan="1" rowspan="1">1.66</td><td colspan="1" rowspan="1">0.62</td><td colspan="1" rowspan="1">1.81</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">3.68</td><td colspan="1" rowspan="1">10.64</td><td colspan="1" rowspan="1">3.77</td><td colspan="1" rowspan="1">10.92</td></tr><tr><td colspan="1" rowspan="1">4-Mod Pos</td><td colspan="1" rowspan="1">33.55</td><td colspan="1" rowspan="1">0.09</td><td colspan="1" rowspan="1">0.27</td><td colspan="1" rowspan="1">0.03</td><td colspan="1" rowspan="1">0.10</td><td colspan="1" rowspan="1">0.17</td><td colspan="1" rowspan="1">0.49</td><td colspan="1" rowspan="1">0.48</td><td colspan="1" rowspan="1">1.42</td><td colspan="1" rowspan="1">0.51</td><td colspan="1" rowspan="1">1.53</td></tr><tr><td colspan="1" rowspan="1">Flu B</td><td colspan="1" rowspan="1">3-Low pos</td><td colspan="1" rowspan="1">35.80</td><td colspan="1" rowspan="1">0.12</td><td colspan="1" rowspan="1">0.35</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.22</td><td colspan="1" rowspan="1">0.60</td><td colspan="1" rowspan="1">0.39</td><td colspan="1" rowspan="1">1.10</td><td colspan="1" rowspan="1">0.47</td><td colspan="1" rowspan="1">1.30</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">5-Mod Pos</td><td colspan="1" rowspan="1">34.56</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.10</td><td colspan="1" rowspan="1">0.29</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.29</td><td colspan="1" rowspan="1">0.83</td><td colspan="1" rowspan="1">0.30</td><td colspan="1" rowspan="1">0.88</td></tr><tr><td colspan="1" rowspan="2">RSV</td><td colspan="1" rowspan="1">3-Low pos</td><td colspan="1" rowspan="1">35.78</td><td colspan="1" rowspan="1">0.07</td><td colspan="1" rowspan="1">0.20</td><td colspan="1" rowspan="1">0.23</td><td colspan="1" rowspan="1">0.65</td><td colspan="1" rowspan="1">0.14</td><td colspan="1" rowspan="1">0.39</td><td colspan="1" rowspan="1">0.59</td><td colspan="1" rowspan="1">1.64</td><td colspan="1" rowspan="1">0.65</td><td colspan="1" rowspan="1">1.82</td></tr><tr><td colspan="1" rowspan="1">5-Mod Pos</td><td colspan="1" rowspan="1">34.41</td><td colspan="1" rowspan="1">0.05</td><td colspan="1" rowspan="1">0.14</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.43</td><td colspan="1" rowspan="1">1.25</td><td colspan="1" rowspan="1">0.43</td><td colspan="1" rowspan="1">1.26</td></tr></table>

$\mathrm { C t } =$ cycle threshold; $\mathrm { C V } =$ coefficient of variation; Mod $=$ moderate; Pos $=$ positive; $\mathrm { S D } =$ standard deviation. Note: Variability from some factors may be numerically negative, which can occur if the variability due to those factors is very small. When this occurs, $\mathrm { S D } { = } 0$ and $C V { = } 0 \%$ 1 Between-Operator may be confounded with Between-Run; therefore, Between-Operator and Between-Run estimates are combined in Between-Operator/Run

The total signal variability, as measured by the standard deviation, was less than or equal to 0.65 across all target viruses and concentrations, with the exception of the Flu A low positive panel member $\mathrm { ( S D } = 3 . 7 7$ ), which may be attributed to 1 false negative result (Table 8 above). For all positive panel members, the within-run factor (i.e., random error) was the largest contributor to total variability. These results indicate that the repeatability and reproducibility of the Panther Fusion SARS-CoV-2/Flu A/B/RSV assay on the Panther Fusion system are robust in NPS samples.

# 2. Linearity:

Not applicable; this is a qualitative assay.

3. Analytical Specificity/Interference: Analytical Reactivity (Inclusivity)

# a. Wet-Testing

This study was performed to determine the analytical reactivity of the Panther Fusion SARS-CoV-2/Flu A/B/RSV assay with clinically relevant strains, serotypes, or subtypes of the target species (i.e., SARS-CoV-2, Flu A, Flu B, and RSV). The inclusivity panel was prepared by spiking various SARS-CoV-2/Flu A/Flu B/RSV virus serotypes/subtypes/strains encompassing temporal and geographical diversity into negative clinical NP swab VTM/UTM matrix at a concentration of $3 \mathbf { x }$ LoD and testing in triplicate. Strains that did not yield $100 \%$ reactivity at $3 \mathrm { x \ L o D }$ were prepared at higher concentrations and tested until the minimum concentration that achieved $100 \%$ reactivity was reached. The strains evaluated and the lowest concentration that achieved $100 \%$ reactivity are shown in Table 9, below.

Table 9. Inclusivity Wet-Testing Study Results   

<table><tr><td colspan="1" rowspan="2">Strain</td><td colspan="1" rowspan="2">Concentration(units/xLoD)</td><td colspan="4" rowspan="1">Result**</td></tr><tr><td colspan="1" rowspan="1">SARS-CoV-2</td><td colspan="1" rowspan="1">FluA</td><td colspan="1" rowspan="1">FluB</td><td colspan="1" rowspan="1">RSV</td></tr><tr><td colspan="6" rowspan="1">SARS-CoV-2</td></tr><tr><td colspan="1" rowspan="1">USA-WA1/2020*</td><td colspan="1" rowspan="1">0.09 TCID50/mL (3x)</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">USA-CA1/2020</td><td colspan="1" rowspan="1">0.09 TCID50/mL (3x)</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">USA-AZ1/2020</td><td colspan="1" rowspan="1">0.09 TCID50/mL (3x)</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">USA-WI1/2020</td><td colspan="1" rowspan="1">0.09 TCID50/mL (3x)</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">USA/OR-OHSU-PHL00037/2021 B.1.1.7</td><td colspan="1" rowspan="1">0.09 TCID50/mL (3x)</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Uganda/MUWRP-20200195568/2020|A.23.1</td><td colspan="1" rowspan="1">0.09 TCID50/mL (3x)</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">SA/PHC658/2021|B.1.617.2</td><td colspan="1" rowspan="1">0.09 TCID50/mL (3x)</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">USA/MD-HP05285/2021|B.1.617.2</td><td colspan="1" rowspan="1">0.09 TCID50/mL (3x)</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">USA/CA/VRLC009/2021|B.1.427</td><td colspan="1" rowspan="1">0.09 TCID50/mL (3x)</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">USA/CA/VRLC012/2021|P.2</td><td colspan="1" rowspan="1">0.3 TCID50/mL (10x)</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">USA/MD-HP03056/2021|B.1.525</td><td colspan="1" rowspan="1">0.3 TCID50/mL (10x)</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">USA/CA-Stanford-16_S02/2021B.1.617.1</td><td colspan="1" rowspan="1">0.09 TCID50/mL (3x)</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Peru/un-CDC-2-4069945/2021|C.37</td><td colspan="1" rowspan="1">0.09 TCID50/mL (3x)</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">SA/MD-HP20874/2021B.1.1.529</td><td colspan="1" rowspan="1">0.09 TCID50/mL (3x)</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">USA/GA-EHC-2811C/2021|B.1.1.529</td><td colspan="1" rowspan="1">0.09 TCID50/mL (3x)</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Influenza A H1N1</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">A/Brisbane/02/18*</td><td colspan="1" rowspan="1">0.18 TCID50/mL (3x)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">A/Michigan/45/2015</td><td colspan="1" rowspan="1">0.18 TCID50/mL (3x)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">A/Christ Church/16/2010</td><td colspan="1" rowspan="1">180 TCID50/mL (3000x)1</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">A/Kentucky/2/06</td><td colspan="1" rowspan="1">0.6 TCID50/mL(10x)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">A/Solomon Islands/03/06</td><td colspan="1" rowspan="1">0.6 TCID50/mL(10x)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">A/Guangdong-maonan/1536/2019</td><td colspan="1" rowspan="1">180 TCID50/mL (3000x)1</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">A/Taiwan/42/2006</td><td colspan="1" rowspan="1">0.6 TCID50/mL(10x)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">A/Henan/8/05</td><td colspan="1" rowspan="1">0.6 TCID50/mL(10x)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">A/Hawaii/15/01</td><td colspan="1" rowspan="1">18 TCID50/mL (300x)3</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">A/California/07/2009</td><td colspan="1" rowspan="1">0.18 TCID50/mL (3x)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">A/Hawaii/66/2019</td><td colspan="1" rowspan="1">180 CEID50/mL (NC)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">A/Indiana/02/2020</td><td colspan="1" rowspan="1">60 CEID50/mL (NC)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">A/Michigan/45/2015 pdm09-like virus</td><td colspan="1" rowspan="1">0.6 TCID50/mL(10x)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Influenza A H3N2</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">A/Kansas/14/17*</td><td colspan="1" rowspan="1">0.33 TCID50/mL (3.3x)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">A/Arizona/45/2018</td><td colspan="1" rowspan="1">3.3 FFU/mL (NC)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">A/New York/21/2020</td><td colspan="1" rowspan="1">3.3 FFU/mL (NC)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">A/Hong Kong/45/2019</td><td colspan="1" rowspan="1">3.3 FFU/mL (NC)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">A/Singapore/INFIMH-16-0019/2016</td><td colspan="1" rowspan="1">110 CEID50/mL (NC)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">A/Hong Kong/2671/2019</td><td colspan="1" rowspan="1">11 TCID50/mL (110x)2</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">A/Hiroshima/52/05</td><td colspan="1" rowspan="1">1.1 TCID50/mL (11x)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">A/Costa Rica/07/99</td><td colspan="1" rowspan="1">11 TCID50/mL (110x)3</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">A/Port Chalmers/1/73</td><td colspan="1" rowspan="1">1.1 TCID50/mL (11x)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">A/Brazil/113/99</td><td colspan="1" rowspan="1">1.1 TCID50/mL (11x)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">A/Perth/16/2009</td><td colspan="1" rowspan="1">0.33 TCID50/mL (3.3x)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">A/Texas/50/2012</td><td colspan="1" rowspan="1">0.33 TCID50/mL (3.3x)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">A/Hong Kong/4801/2014</td><td colspan="1" rowspan="1">1.1 TCID50/mL (11x)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">A/Indiana/08/2011</td><td colspan="1" rowspan="1">1.1 TCID50/mL (11x)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Influenza A H5N1</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">A/Hong Kong/486/97</td><td colspan="1" rowspan="1">0.01 ng/mL (NC)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Influenza B Victoria</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">B/Washington/02/2019*</td><td colspan="1" rowspan="1">0.09 TCID50/mL (3x)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">B/Colorado/06/2017</td><td colspan="1" rowspan="1">0.09 TCID50/mL(3x)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">B/Florida/78/2015</td><td colspan="1" rowspan="1">0.3 TCID50/mL (10x)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">B/Alabama/2/17</td><td colspan="1" rowspan="1">0.09 TCID50/mL (3x)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">B/Ohio/1/2005</td><td colspan="1" rowspan="1">0.3 TCID50/mL (10x)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">B/Michigan/09/2011</td><td colspan="1" rowspan="1">3 TCID50/mL (100x)3</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">B/Hawaii/01/2018 (NA D197N)</td><td colspan="1" rowspan="1">0.9 TCID50/mL (30x)1</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">B/Brisbane/33/08</td><td colspan="1" rowspan="1">0.09 TCID50/mL (3x)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="6" rowspan="1">Flu B Yamagata</td></tr><tr><td colspan="1" rowspan="1">B/Phuket/3073/2013*</td><td colspan="1" rowspan="1">0.006 TCID50/mL (2x)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">B/Wisconsin/1/2010</td><td colspan="1" rowspan="1">2 TCID50/mL (666x)1</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">B/Utah/9/14</td><td colspan="1" rowspan="1">0.006 TCID50/mL (2x)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">B/St. Petersburg/04/06</td><td colspan="1" rowspan="1">0.06 TCID50/mL (2x)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">B/Texas/81/2016</td><td colspan="1" rowspan="1">2 TCID50/mL (666x)1</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">B/Indiana/17/2017</td><td colspan="1" rowspan="1">0.6 TCID50/mL (200x)1</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">B/Oklahoma/10/2018</td><td colspan="1" rowspan="1">2 TCID50/mL (666x)1</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">B/Massachusetts/02/2012</td><td colspan="1" rowspan="1">0.2 TCID50/mL (66x)4</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Flu B</td><td colspan="1" rowspan="1"></td><td colspan="2" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">B/Lee/40</td><td colspan="1" rowspan="1">0.09 TCID50/mL (NC)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">RSV A</td><td colspan="4" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">RSV-A/2006 Isolate*</td><td colspan="1" rowspan="1">0.06 TCID50/mL (2x)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1">RSV A/4/2015 isolate #1</td><td colspan="1" rowspan="1">0.06 TCID50/mL (2x)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1">RSV A/A2</td><td colspan="1" rowspan="1">0.06 TCID50/mL (2x)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1">RSV A/12/2014 isolate #2</td><td colspan="1" rowspan="1">0.06 TCID50/mL (2x)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">RSV B</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">RSV-B/CH93(18)-18*</td><td colspan="1" rowspan="1">0.3 TCID50/mL (10x)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1">RSV B/3/2015 isolate #1</td><td colspan="1" rowspan="1">0.09 TCID50/mL (3x)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1">RSV B/9320</td><td colspan="1" rowspan="1">0.09 TCID50/mL (3x)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td></tr></table>

\*Strain used to establish LoD and for which the concentration in multiples of the LoD (xLoD) was calculated.

${ } ^ { * * } \mathrm { A }$ positive symbol $( + )$ indicates that reactivity was observed for $100 \%$ of replicates (3/3) while a negative symbol $( - )$ indicates that reactivity was observed for $0 \%$ of replicates (0/3)

$^ { 1 } I n$ silico analysis showed $100 \%$ homology to amplification region. Virus stock degradation or error in $\mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ quantification may have impacted the nucleic acid target concentration at $100 \%$ detection.

$^ 2 I n$ silico analysis identified a single mismatch in the forward and reverse primers. Due to the location of these mismatches, amplification and detection are not expected to be impacted. Virus stock degradation or error in $\mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ quantification may have impacted the nucleic acid target concentration at $100 \%$ detection.

3 Sequence of strain in targeted amplification regions are not available in NCBI or GISAID to further evaluate sensitivity.

$^ 4 I n$ silico analysis identified a single mismatch in the reverse primer. Due to the location of this mismatch, amplification and detection are not expected to be impacted. Virus stock degradation or error in $\mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ quantification may have impacted the nucleic acid target concentration at $100 \%$ detection.

NC – Not calculable

The results from this study demonstrate that the Panther Fusion SARS-CoV-2/Flu A/B/RSV assay is capable of detecting multiple clinically relevant strains of SARS-CoV2, Flu A, Flu B, and RSV.

# b. In silico

The inclusivity of the Panther Fusion SARS-CoV-2/Flu A/B/RSV assay was evaluated using in silico analysis of the forward primers, reverse primers, and probes for the SARS-CoV-2, Flu A, Flu B and RSV target systems in relation to sequences available in the NCBI and GISAID gene databases. For SARS-CoV-2, all available sequences up to June 25, 2022 from the GISAID and NCBI gene databases were evaluated. Due to the large size of the database $( > 9 . 3$ million sequences) every 1 out of 10 sequences was randomly sampled from each database. For Flu A, Flu B, RSV A, and RSV B, all available sequences from GISAID and NCBI gene databases entered from January 1, 2015 to February 15, 2022 were evaluated. In total, 934,493 SARS-CoV-2 and 88,128 Flu A sequences were evaluated that contained both assay target regions, while an additional 31,801 Flu B sequences, 1,599 RSV A sequences, and 1,240 RSV B sequences containing the assay single target region were included in the analysis. The SARS-CoV-2 sequences included the following lineages and variants of concern (VOC) or variants of interest (VOI) that may have important epidemiological, immunological, or pathogenic properties from a public health perspective: Delta, Alpha, Omicron BA.1, Omicron BA.2, Omicron BA.4, Omicron BA.5, Gamma, Epsilon, Iota, Beta, Mu, Zeta, Kappa, Eta, Lamba, Theta and others.

Sequence alignments were generated using the multiple sequence alignment program MAFFT (Multiple Alignment using Fast Fourier Transform). All non-human isolates were removed from the analysis. Additionally, any sequence with missing or ambiguous sequence information for the target region was removed. Due to the dual amplification systems for SARS-CoV-2 and Flu A, only sequences with mismatches in both regions were assessed for potential impact to the inclusivity of the assay.

Based on the in silico analysis of GISAID and NCBI sequences available up to June 25, 2022 for SARS-CoV-2, the Panther Fusion SARS-CoV-2/Flu A/B/RSV assay is predicted to detect all 934,493 SARS-CoV-2 sequences evaluated.

Based on in silico analysis of all sequences available from January 01, 2015 to February 15, 2022 in GISAID and NCBI databases, the Panther Fusion SARS-CoV2/Flu A/B/RSV assay is predicted to detect ${ \geq } 9 9 . 9 9 8 \%$ of 88,128 Flu A, ${ \geq } 9 9 . 9 4 \%$ of 31,801 Flu B, $2 9 8 . 1 2 \%$ of 1,599 RSV A, and $2 9 8 . 2 3 \%$ of 1,240 RSV B sequences evaluated.

# Exclusivity Testing

This study was performed to demonstrate that non-indicated influenza subtypes, specifically influenza C and D, are not detected by the Panther Fusion SARS-CoV-2/Flu A/B/RSV assay. For this study, one Flu C strain (C/Taylor/1233/1947) and one Flu D strain (Bovine/Mississippi/C00046N/14) were diluted into negative clinical NP swab matrix at $\mathrm { 1 x 1 0 ^ { 5 } C E I D 5 0 / m L }$ and $\mathrm { 1 x 1 0 ^ { 4 } T C I D 5 0 / m L }$ concentrations, respectively. Seven replicates were tested for both strains and no positive results were obtained for any of the assay’s target channels, demonstrating that Flu C and Flu D are not detected by the assay.

# Cross-Reactivity/Microbial Interference

# a. Wet-Testing

This study evaluated the analytical specificity (cross-reactivity) and sensitivity of the Panther Fusion SARS-CoV-2/Flu A/B/RSV assay in the presence of non-targeted microorganisms that may be found in a respiratory tract clinical specimen. Forty-one (41) non-target microorganisms (Table 10) were evaluated in the study. Panel members were composed of 3 different non-target microorganisms spiked into negative clinical NP swab VTM/UTM matrix at $1 0 ^ { 3 } – 1 0 ^ { 6 } \mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ or copies $/ \mathrm { m L }$ (for viruses), $1 0 ^ { 5 } – 1 0 ^ { 8 } \mathrm { C F U / m L }$ or rRNA copies $/ \mathrm { m L }$ (for bacteria) or $1 0 ^ { 6 } \mathrm { C F U / m L }$ (for fungi). To evaluate cross-reactivity, each panel was evaluated in triplicate in the absence of the target organisms. To evaluate microbial interference (negative effect on sensitivity), each panel was tested in triplicate in the presence of SARS-CoV-2, Flu A, Flu B, and RSV at 3x LoD. The organisms evaluated in the study are shown in the table below. No cross-reactivity or microbial interference was observed at the concentrations tested.

Table 10. Cross-Reactivity Study Microorganisms Evaluated By Wet-Testing   

<table><tr><td rowspan=1 colspan=1>Viruses</td><td rowspan=1 colspan=1>Conc.1</td><td rowspan=1 colspan=1>Bacteria/Fungi</td><td rowspan=1 colspan=1>Conc.1</td></tr><tr><td rowspan=1 colspan=1>Adenovirus 1</td><td rowspan=1 colspan=1>1x105 TCID50/mL</td><td rowspan=1 colspan=1>Bordetella pertussis</td><td rowspan=1 colspan=1>1x106 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Adenovirus 7a</td><td rowspan=1 colspan=1>1x105 TCID50/mL</td><td rowspan=1 colspan=1>Candida albicans</td><td rowspan=1 colspan=1>1x106 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>CMV Strain AD 169</td><td rowspan=1 colspan=1>1x104 TCID50/mL</td><td rowspan=1 colspan=1>Chlamydophila pneumoniae</td><td rowspan=1 colspan=1>1x106IFU/mL</td></tr><tr><td rowspan=1 colspan=1>Human coronavirus 229E</td><td rowspan=1 colspan=1>1x104 TCID50/mL</td><td rowspan=1 colspan=1>Corynebacterium diphtheriae</td><td rowspan=1 colspan=1>1x106CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Human coronavirus NL63</td><td rowspan=1 colspan=1>1x104 TCID50/mL</td><td rowspan=1 colspan=1>Escherichia coli</td><td rowspan=1 colspan=1>1x106CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Human coronavirus OC43</td><td rowspan=1 colspan=1>1x105 TCID50/mL</td><td rowspan=1 colspan=1>Haemophilus influenzae</td><td rowspan=1 colspan=1>1x106 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Epstein-Barr virus (EBV)</td><td rowspan=1 colspan=1>1x106 copies/mL</td><td rowspan=1 colspan=1>Lactobacillus plantarum</td><td rowspan=1 colspan=1>1x106 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Enterovirus (e.g., EV68)</td><td rowspan=1 colspan=1>1x105 TCID50/mL</td><td rowspan=1 colspan=1>Legionella pneumophila</td><td rowspan=1 colspan=1>1x106 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Human coronavirus HKU12</td><td rowspan=1 colspan=1>1x106 copies/mL</td><td rowspan=1 colspan=1>Moraxella catarrhalis</td><td rowspan=1 colspan=1>1x105 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Human Metapneumovirus(hMPV)</td><td rowspan=1 colspan=1>1x105 TCID50/mL</td><td rowspan=1 colspan=1>Mycobacterium tuberculosis</td><td rowspan=1 colspan=1>1x109rRNAcopies/mL3</td></tr><tr><td rowspan=1 colspan=1>HPIV-1</td><td rowspan=1 colspan=1>1x105 TCID50/mL</td><td rowspan=1 colspan=1>Mycoplasma pneumoniae</td><td rowspan=1 colspan=1>1x109rRNAcopies/mL 3</td></tr><tr><td rowspan=1 colspan=1>HPIV-2</td><td rowspan=1 colspan=1>1x105 TCID50/mL</td><td rowspan=1 colspan=1>Neisseria spp.</td><td rowspan=1 colspan=1>1x106 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>HPIV-3</td><td rowspan=1 colspan=1>1x105 TCID50/mL</td><td rowspan=1 colspan=1>Neisseria meningitides</td><td rowspan=1 colspan=1>1x106 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>HPIV-4</td><td rowspan=1 colspan=1>1x104 TCID50/mL</td><td rowspan=1 colspan=1>Neisseria mucosa</td><td rowspan=1 colspan=1>1x106 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Measles</td><td rowspan=1 colspan=1>1x104 TCID50/mL</td><td rowspan=1 colspan=1>Pneumocystis jirovecii</td><td rowspan=1 colspan=1>1x106 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>MERS-Coronavirus</td><td rowspan=1 colspan=1>5x104 TCID50/mL</td><td rowspan=1 colspan=1>Pseudomonas aeruginosa</td><td rowspan=1 colspan=1>1x106 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Mumps virus</td><td rowspan=1 colspan=1>1x105 TCID50/mL</td><td rowspan=1 colspan=1>Staphylococcus aureus</td><td rowspan=1 colspan=1>1x106CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Rhinovirus 1A</td><td rowspan=1 colspan=1>1x104 TCID50/mL</td><td rowspan=1 colspan=1>Staphylococcus epidermidis</td><td rowspan=1 colspan=1>1x106CFU/mL</td></tr><tr><td rowspan=1 colspan=1>SARS coronavirus 12</td><td rowspan=1 colspan=1>1x106 copies/mL</td><td rowspan=1 colspan=1>Streptococcus pneumoniae</td><td rowspan=1 colspan=1>1x106 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Varicella Zoster Virus</td><td rowspan=1 colspan=1>1x103 TCID50/mL</td><td rowspan=1 colspan=1>Streptococcus pyogenes</td><td rowspan=1 colspan=1>1x106 CFU/mL</td></tr><tr><td rowspan=1 colspan=2></td><td rowspan=1 colspan=1>Streptococcus salivarius</td><td rowspan=1 colspan=1>1x106 CFU/mL</td></tr></table>

1 CFU $=$ Colony Forming Units; $\operatorname { I F U } =$ Inclusion Forming Units; $\mathrm { T C I D } _ { 5 0 } =$ Median Tissue Culture Infectious Dose.

2 Cultured virus and whole genome purified nucleic acid for Human HKU1 and SARS-coronavirus were not readily available at the time of testing. HKU1 and SARS-coronavirus in vitro transcript (IVT) corresponding to the ORF1ab gene regions targeted by the assay were used to evaluate crossreactivity and microbial interference.

3 The $1 \mathrm { x } 1 0 ^ { 9 }$ rRNA copies/mL concentration is equivalent to $2 \mathrm { x } 1 0 ^ { 5 } \mathrm { C F U / m L }$ .

# b. In silico

An in silico analysis was performed to demonstrate that the Panther Fusion SARS-CoV2/Flu A/B/RSV assay does not cross-react with closely related or commonly encountered microorganisms in a respiratory clinical specimen. A total of 545 individual sequences from GenBank, associated with 143 microorganisms, were analyzed with BLAST for homology to the primer and probe sequences for the SARS-CoV-2, Flu A, Flu B, RSV A, and RSV B targets included in the assay. A microorganism was determined to have no cross-reactivity when its sequence had $- 8 0 \%$ homology with all of the assay’s primers and probes. When $2 8 0 \%$ homology was identified for a single primer or probe, proximity and mismatch analysis was used to demonstrate minimal likelihood of amplicon generation and/or detection. Specifically, microorganism sequences showing $2 8 0 \%$

homology to a single primer or probe and $< 5 0 \%$ homology to the other primer/probe, were not expected to be amplified. For microorganism sequences showing $2 8 0 \%$ homology to a single primer or probe and $2 5 0 \%$ homology to the other primer/probe, proximity analysis was performed to determine the likelihood of amplification.

Seven organisms showed $2 8 0 \%$ homology to one of the primers and $2 5 0 \%$ homology to a second primer and probe. Six of these organisms were determined to be low risk and unlikely to cross-react due to proximity analysis. The remaining organism, SARS-1 coronavirus, was identified as having a small possibility of amplification due to a high level of homology to the primers and probe in the SARS-CoV-2 channel. However, mismatches at the $_ { 3 } ,$ end of the primers suggest that hybridization and amplification is unlikely. This observation was confirmed in a cross-reactivity wet-testing study performed at high concentrations ( $1 . 0 1 \mathrm { x } 1 0 ^ { 6 }$ copies $/ \mathrm { m L }$ ) of SARS-1 coronavirus, which demonstrated SARS-1 coronavirus does not cross react with any of the target assays.

A total of 29 organisms showed $2 8 0 \%$ homology to one of the primers, $2 5 0 \%$ homology to a second primer, but $< 5 0 \%$ homology to a probe. Twenty-seven of these organisms were determined to be low risk and unlikely to interfere with assay analytes due to proximity, mismatches at the 3’end of the primers, low homology to at least 1 primer, and/or ${ < } 1 0$ continuous bases between the target and primers. Serratia marcescens was identified as having a possibility of low amplification without detection. To determine if S. marcescens cross-reacted with the Panther Fusion SARS-CoV-2/Flu A/B/RSV assay, wet-testing was performed at high concentrations $( 1 \mathrm { x } 1 0 ^ { 6 } \mathrm { C F U / m L } )$ ) of S. marcescens. No cross-reactivity was observed at the concentrations tested.

# Interfering Substances

This study evaluated the performance of the Panther Fusion SARS-CoV-2/Flu A/B/RSV assay in the presence of medications, over the counter products, and other potentially interfering substances found in a clinical respiratory specimen. Assay results were evaluated to determine if the presence of potentially interfering substances in target analyte negative or target analyte positive samples had an effect on assay performance. The analyte negative pools contained negative NP swab VTM/UTM clinical matrix and the potentially interfering substances, only. The analyte positive pools contained negative NP swab VTM/UTM clinical matrix, the potentially interfering substance(s), and one representative strain of each targeted analyte (i.e., SARS-CoV-2, Flu A, Flu B, and RSV B) spiked to a final testing concentration of $3 \mathbf { x }$ LoD. Three replicates were tested per pool both in the presence and absence of the target analytes. The substances evaluated are denoted in Table 11 below. All target-spiked panels were $100 \%$ positive, and all target-negative panels were $100 \%$ negative, indicating that none of the evaluated substances interfered with the Panther Fusion SARSCoV-2/Flu A/B/RSV assay.

Table 11. Interfering Substances Testing Panel   

<table><tr><td colspan="1" rowspan="1">Panel</td><td colspan="1" rowspan="1">Type</td><td colspan="1" rowspan="1">PotentiallyInterferingSubstance</td><td colspan="1" rowspan="1">ActiveIngredient(s)</td><td colspan="1" rowspan="1">Concentration1</td></tr><tr><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="2">Endogenous</td><td colspan="1" rowspan="1">Bovinesubmaxillarygland, type I-S</td><td colspan="1" rowspan="1">Purified mucinprotein</td><td colspan="1" rowspan="1">60 ug/mL</td></tr><tr><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">Blood (human)</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">2% v/v</td></tr><tr><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Neo-Synephrine</td><td colspan="1" rowspan="1">Phenylephrine</td><td colspan="1" rowspan="1">15 % v/v</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="3">Nasal sprays ordrops</td><td colspan="1" rowspan="1">Anefrin</td><td colspan="1" rowspan="1">Oxymetazoline</td><td colspan="1" rowspan="1">15 % v/v</td></tr><tr><td colspan="1" rowspan="2">4</td><td colspan="1" rowspan="1">Saline</td><td colspan="1" rowspan="1">Sodium chloride</td><td colspan="1" rowspan="1">15 % v/v</td></tr><tr><td colspan="1" rowspan="1">Ventolin HFA2</td><td colspan="1" rowspan="1">Albuterol</td><td colspan="1" rowspan="1">45 ng/mL</td></tr><tr><td colspan="1" rowspan="3">5</td><td colspan="1" rowspan="7">Nasalcorticosteroids</td><td colspan="1" rowspan="1">QVAR BeconaseAQ{2$</td><td colspan="1" rowspan="1">Beclomethasone</td><td colspan="1" rowspan="1">15 ng/mL</td></tr><tr><td colspan="1" rowspan="1">Dexacort2</td><td colspan="1" rowspan="1">Dexamethasone</td><td colspan="1" rowspan="1">12 ug/mL</td></tr><tr><td colspan="1" rowspan="1">Nasacort</td><td colspan="1" rowspan="1">Triamcinolone</td><td colspan="1" rowspan="1">5 % v/v</td></tr><tr><td colspan="1" rowspan="2">6</td><td colspan="1" rowspan="1">Flonase</td><td colspan="1" rowspan="1">Fluticasone</td><td colspan="1" rowspan="1">5 % v/v</td></tr><tr><td colspan="1" rowspan="1">Rhinocort</td><td colspan="1" rowspan="1">Budesonide</td><td colspan="1" rowspan="1">5 % v/v</td></tr><tr><td colspan="1" rowspan="2">7</td><td colspan="1" rowspan="1">Nasonex2</td><td colspan="1" rowspan="1">Mometasone</td><td colspan="1" rowspan="1">0.5 ng/mL</td></tr><tr><td colspan="1" rowspan="1">AEROSPAN2</td><td colspan="1" rowspan="1">Flunisolide</td><td colspan="1" rowspan="1">10 ug/mL</td></tr><tr><td colspan="1" rowspan="1">8</td><td colspan="1" rowspan="1">Nasalgel/homeopathicallergy reliefmedicine</td><td colspan="1" rowspan="1">Zicam (AllergyRelief)</td><td colspan="1" rowspan="1">Luffa opperculata,Galphimia, Glauca,Histaminumhydrochloricum,sulfur</td><td colspan="1" rowspan="1">5 % v/v</td></tr><tr><td colspan="1" rowspan="1">9</td><td colspan="1" rowspan="1">Throat lozenges,oral anesthetic,and analgesic</td><td colspan="1" rowspan="1">Cepacol ExtraStrength</td><td colspan="1" rowspan="1">Benzocaine,Menthol</td><td colspan="1" rowspan="1">0.7 mg/mL</td></tr><tr><td colspan="1" rowspan="3">10</td><td colspan="1" rowspan="3">Anti-viral drugs</td><td colspan="1" rowspan="1">Relenza{</td><td colspan="1" rowspan="1">Zanamivir</td><td colspan="1" rowspan="1">3.3 mg/mL</td></tr><tr><td colspan="1" rowspan="1">TamiFlu2</td><td colspan="1" rowspan="1">Oseltamivir</td><td colspan="1" rowspan="1">400 ug/mL</td></tr><tr><td colspan="1" rowspan="1">Virazole2</td><td colspan="1" rowspan="1">Ribavirin</td><td colspan="1" rowspan="1">10.5 ug/mL</td></tr><tr><td colspan="1" rowspan="1">11</td><td colspan="1" rowspan="1">Antibiotic, nasalointment</td><td colspan="1" rowspan="1">Bactroban cream2</td><td colspan="1" rowspan="1">Mupirocin</td><td colspan="1" rowspan="1">1.6 ug/mL</td></tr><tr><td colspan="1" rowspan="1">12</td><td colspan="1" rowspan="1">Antibacterial,systemic</td><td colspan="1" rowspan="1">Tobramycin2</td><td colspan="1" rowspan="1">Tobramycin</td><td colspan="1" rowspan="1">33.1 ug/mL</td></tr><tr><td colspan="1" rowspan="1">13</td><td colspan="1" rowspan="2">Solvent Control</td><td colspan="1" rowspan="1">Water</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">5% v/v</td></tr><tr><td colspan="1" rowspan="1">14</td><td colspan="1" rowspan="1">DimethylSulfoxide (DMSO)</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">5%v/v</td></tr><tr><td colspan="1" rowspan="1">15</td><td colspan="1" rowspan="1">Control</td><td colspan="1" rowspan="1">None</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td></tr></table>

$^ 1 \mathrm { v / v } \mathrm { ; }$ : volume by volume 2 Active ingredient tested, not substance

# Competitive Interference

The purpose of this study was to demonstrate that samples tested with the Panther Fusion SARS-CoV-2/Flu A/B/RSV assay that are co-infected with multiple types of targeted microorganisms do not inhibit the detection of either one (competitive interference). One representative strain of each targeted organism (SARS-CoV-2, Flu A, Flu B and RSV) was tested. Co-infection panels were made by spiking one target organism at high concentration (up to $1 0 , 0 0 0 \mathrm { T C I D } _ { 5 0 } / \mathrm { m L } )$ ) and another target organism at low concentration ${ \bf \Pi } \sim 3 { \bf x }$ LoD) in negative clinical NP swab clinical matrix. If less than $100 \%$ positivity was observed for the low target, the high target was diluted down from $1 0 , 0 0 0 \mathrm { T C I D } { \scriptstyle 5 0 / \mathrm { m L } }$ to a concentration where $100 \%$ positivity of the low target was achieved. The on-target analyte combinations evaluated and the results of this evaluation, are shown in Table 12.

Table 12. Competitive Interference Study Sample Panel Composition & Study Results   

<table><tr><td colspan="2" rowspan="1">Target 1 (Low Conc.)</td><td colspan="2" rowspan="1">Target 2 (High Conc.)</td><td colspan="4" rowspan="1">% Detected</td></tr><tr><td colspan="1" rowspan="1">Virus</td><td colspan="1" rowspan="1">Conc.</td><td colspan="1" rowspan="1">Virus</td><td colspan="1" rowspan="1">Conc.</td><td colspan="1" rowspan="1">SARS-CoV-2</td><td colspan="1" rowspan="1">Flu A</td><td colspan="1" rowspan="1">Flu B</td><td colspan="1" rowspan="1">RSV</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Flu A</td><td colspan="1" rowspan="1">1x104 TCID50/mL</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="2">Target 1 (Low Conc.)</td><td colspan="2">Target 2 (High Conc.)</td><td colspan="3">% Detected</td></tr><tr><td>Virus</td><td>Conc.</td><td>Virus</td><td>Conc.</td><td>SARS- Flu A CoV-2</td><td>Flu B</td><td>RSV</td></tr><tr><td rowspan="9">SARS- CoV-2</td><td rowspan="9">0.09 TCID50/mL (3x LoD)</td><td>(H3N2)</td><td>(&gt;90,000x LoD)</td><td>(3/3)</td><td>(3/3) 0%</td><td>(0/3) (0/3)</td></tr><tr><td>Flu B (Victoria)</td><td>1x104 TCID50/mL (&gt;300,000x LoD)</td><td>100% (3/3) (0/3)</td><td>100% (3/3)</td><td>0% (0/3)</td></tr><tr><td>RSV-A</td><td>1x104 TCID50/mL (500,000x LoD)</td><td>100% 0% (3/3) (0/3)</td><td>0% (0/3)</td><td>100% (3/3)</td></tr><tr><td></td><td>1x104 TCID50/mL (&gt;300,000x LoD)</td><td>0% (0/3)</td><td>0% 0% (0/3)</td><td>100% (3/3)</td></tr><tr><td rowspan="4">RSV-B</td><td>1x103TCID50/mL</td><td>0%</td><td>(0/3) 0%</td><td>0% 100%</td></tr><tr><td>(&gt;30,000x LoD)</td><td>(0/3)</td><td>(0/3) (0/3)</td><td>(3/3)</td></tr><tr><td>100 TCID50/mL (&gt;3000x LoD)</td><td>66.7% (2/3)</td><td>0% 0%</td><td>100% (3/3)</td></tr><tr><td>30 TCID50/mL</td><td>10%</td><td>(0/3) 0%</td><td>(0/3) 0% 100% (3/3)</td></tr><tr><td>(1000x LoD)</td><td>(3/3)</td><td>(0/3)</td><td>(0/3)</td></tr><tr><td rowspan="17">0.33 Flu A</td><td rowspan="5">SARS-</td><td>1x104 TCID50/mL (&gt;300,000x LoD)</td><td>100% (3/3)</td><td>0% (0/3)</td><td>0% (0/3)</td><td>0% (0/3)</td></tr><tr><td>CoV-2 (&gt;30,000x LoD)</td><td>1x103 TCID50/mL 100% (3/3)</td><td>0% (0/3)</td><td>0% (0/3)</td><td>0% (0/3)</td></tr><tr><td></td><td>1x102 TCID50/mL (&gt;3000x LoD)</td><td>100% 100% (3/3) (3/3)</td><td>0% (0/3)</td><td>0% (0/3)</td></tr><tr><td>Flu B</td><td>1x104 TCID50/mL</td><td>0% 100%</td><td>100%</td><td>0%</td></tr><tr><td>RSV-A</td><td>(&gt;300,000x LoD) (0/3) 1x104 TCID50/mL</td><td>(3/3) 100%</td><td>(3/3) 0%</td><td>(0/3) 100%</td></tr><tr><td>TCID50/mL (3.3x LoD)</td><td>(500,000x LoD)</td><td>0% (0/3)</td><td>(3/3)</td><td>(0/3)</td><td>(3/3)</td></tr><tr><td rowspan="5"></td><td rowspan="5">RSV-B</td><td>1x104 TCID50/mL</td><td>0%</td><td>0%</td><td>0% 100%</td></tr><tr><td>(&gt;300,000x LoD) 1x103 TCID50/mL</td><td>(0/3)</td><td>(0/3) 0%</td><td>(0/3) (3/3) 0%</td></tr><tr><td>(&gt;30,000x LoD)</td><td>0% (0/3)</td><td>(0/3)</td><td>100% (0/3) (3/3)</td></tr><tr><td>100 TCID50/mL</td><td>0%</td><td>0%</td><td>0% 100% (3/3)</td></tr><tr><td>(&gt;3000x LoD)</td><td>(0/3)</td><td>(0/3)</td><td>(0/3)</td></tr><tr><td></td><td>30 TCID50/mL (1000x LoD)</td><td>0% (0/3) (3/3)</td><td>100%</td><td>0% 100%</td></tr><tr><td rowspan="11">Flu B</td><td rowspan="2">SARS- CoV-2</td><td></td><td></td><td></td><td>(0/3)</td><td>(3/3)</td></tr><tr><td>1x104 TCID50/mL</td><td>100%</td><td>0%</td><td>100%</td><td>0%</td></tr><tr><td rowspan="2">Flu A</td><td>(&gt;300,000x LoD)</td><td>(3/3)</td><td>(0/3)</td><td>(3/3)</td><td>(0/3)</td></tr><tr><td>1x104 TCID50/mL</td><td>0%</td><td>100%</td><td>100%</td><td>0%</td></tr><tr><td rowspan="2">0.09 TCID50/mL</td><td></td><td>(&gt;90,000x LoD) (0/3)</td><td>(3/3)</td><td>(3/3)</td><td>(0/3)</td></tr><tr><td>RSV-A</td><td>1x104 TCID50/mL</td><td>0% 0%</td><td>100%</td><td>10%</td></tr><tr><td rowspan="2">(3x LoD)</td><td rowspan="2"></td><td>(500,000x LoD)</td><td>(0/3)</td><td>(0/3)</td><td>(3/3) (3/3)</td></tr><tr><td>1x104 TCID50/mL (&gt;300,000xLoD)</td><td>0% (0/3)</td><td>0% (0/3)</td><td>66.7% 100% (2/3) (3/3)</td></tr><tr><td rowspan="2"></td><td rowspan="2">RSV-B</td><td>1x103 TCID50/mL</td><td>0%</td><td>0%</td><td>10% 100%</td></tr><tr><td>(&gt;30,000x LoD)</td><td>(0/3)</td><td>(0/3)</td><td>(3/3) (3/3)</td></tr><tr><td></td><td>0.06</td><td>SARS- 1x104 TCID50/mL</td><td>100%</td><td>0%</td><td>0%</td><td>100%</td></tr><tr><td rowspan="2">RSV-A</td><td>TCID50/mL</td><td>CoV-2</td><td>(&gt;300,000x LoD) (3/3)</td><td>(0/3)</td><td>(0/3</td><td>(3/3)</td></tr><tr><td>(x LoD) Flu A</td><td>1x104 TCID50/mL (&gt;90,000x LoD)</td><td>0% (0/3)</td><td>100% (3/3)</td><td>0% (0/3)</td><td>100% (3/3)</td></tr><tr><td colspan="2" rowspan="1">Target 1 (Low Conc.)</td><td colspan="2" rowspan="1">Target 2 (High Conc.)</td><td colspan="4" rowspan="1">% Detected</td></tr><tr><td colspan="1" rowspan="1">Virus</td><td colspan="1" rowspan="1">Conc.</td><td colspan="1" rowspan="1">Virus</td><td colspan="1" rowspan="1">Conc.</td><td colspan="1" rowspan="1">SARS-CoV-2</td><td colspan="1" rowspan="1">Flu A</td><td colspan="1" rowspan="1">Flu B</td><td colspan="1" rowspan="1">RSV</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Flu B</td><td colspan="1" rowspan="1">1x104 TCID50/mL(&gt;300,000x LoD)</td><td colspan="1" rowspan="1">0%(0/3)</td><td colspan="1" rowspan="1">0%(0/3)</td><td colspan="1" rowspan="1">100%(3/3)</td><td colspan="1" rowspan="1">100%(3/3)</td></tr></table>

<table><tr><td rowspan="3">RSV-B</td><td rowspan="3">0.09 TCID50/mL (3x LoD)</td><td>SARS- CoV-2</td><td>1x104 TCID50/mL (&gt;300,000x LoD)</td><td>100% (3/3)</td><td>0% (0/3)</td><td>0% (0/3)</td><td>100% (3/3)</td></tr><tr><td>Flu A</td><td>1x104 TCID50/mL (&gt;90,000x LoD)</td><td>0% (0/3)</td><td>100% (3/3)</td><td>0% (0/3)</td><td>100% (3/3)</td></tr><tr><td>Flu B</td><td>1x104 TCID50/mL (&gt;300,000x LoD)</td><td>0% (0/3)</td><td>0% (0/3)</td><td>100% (3/3)</td><td>100% (3/3)</td></tr></table>

Interference was observed for the following co-infections:

SARS-CoV-2 (low concentration) in the presence of RSV B (high concentrations of $1 \mathrm { x } 1 0 ^ { 4 }$ , $1 \mathrm { x } 1 0 ^ { 3 }$ , and $\mathrm { 1 x 1 0 ^ { 2 } T C I D 5 0 / m L }$ ). Interference was no longer observed at an RSV B concentration of $3 0 \mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ .   
Flu A (low concentration) in the presence of SARS-CoV-2 (high concentrations of $1 \mathrm { x } 1 0 ^ { 4 }$ and $\mathrm { 1 x 1 0 ^ { 3 } T C I D _ { 5 0 } / m L }$ ). Interference was no longer observed at a SARS-CoV-2 concentration of $\mathrm { 1 x 1 0 ^ { 2 } T C I D 5 0 / m L }$ .   
Flu A (low concentration) in the presence of RSV B (high concentrations of $1 \mathrm { x } 1 0 ^ { 4 }$ , $1 \mathrm { x } 1 0 ^ { 3 }$ , and $\mathrm { 1 x 1 0 ^ { 2 } T C I D 5 0 / m L } \mathrm { \Omega }$ ). Interference was no longer observed at an RSV B concentration of $3 0 \mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ .   
Flu B (low concentration) in the presence of RSV B (high concentration of $1 \mathrm { x } 1 0 ^ { 4 }$ $\mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ ). Interference was no longer observed at an RSV B concentration of $1 \mathrm { x } 1 0 ^ { 3 }$ $\mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ .

4. Assay Reportable Range:

Not applicable; this is a qualitative assay.

5. Traceability, Stability, Expected Values (Controls, Calibrators, or Methods):

# a. Controls

The assay contains an internal control (IC-S) added to each test specimen and external positive and negative controls. For more information, see Section IV.C.5. Quality Control, above.

# b. Sample Stability

Stability studies have been performed to support the following claims:

# Primary Specimen (NPS storage in VTM/UTM)

Specimens can be stored in VTM/UTM (prior to transfer to the Panther Fusion Specimen Lysis Tube (SLT)) under the following conditions:

Refrigerated $\left( 2 \mathrm { - } 8 ^ { \circ } \mathrm { C } \right)$ for up to 96 hours before transfer to the Panther Fusion Specimen Lysis Tube or, Frozen at $- 7 0 ^ { \circ } \mathrm { C }$ . Samples may be freeze/thawed up to 3 times prior to testing.

# Processed Specimen (NPS storage in a Panther Fusion SLT)

Once a patient specimen in VTM/UTM is transferred to the Panther Fusion SLT, samples can be stored under the following conditions:

Room temperature $( 1 5 . 3 0 ^ { \circ } \mathrm { C } )$ for up to 6 days or, Refrigerated $\left( 2 \mathrm { - } 8 ^ { \circ } \mathrm { C } \right)$ for up to 3 months or, • Frozen at $- 2 0 ^ { \circ } \mathrm { C }$ for up to 3 months. Freeze/thaw cycles should be minimized due to potential for sample degradation. Frozen at $- 7 0 ^ { \circ } \mathrm { C }$ for up to 3 months. Freeze/thaw cycles should be minimized due to potential for sample degradation.

# RespDirect Collection Kit (eSTM)

A NPS specimen collected with the RespDirect Collection Kit can be stored under the following conditions:

Room temperature $( 1 5 . 3 0 ^ { \circ } \mathrm { C } )$ for up to 6 days or, Refrigerated $\left( 2 \mathrm { - } 8 ^ { \circ } \mathrm { C } \right)$ for up to 3 months or,   
• Frozen at $- 2 0 ^ { \circ } \mathrm { C }$ for up to 3 months. Samples may be freeze/thawed up to 3 times prior to testing or, Frozen at $- 7 0 ^ { \circ } \mathrm { C }$ for up to 3 months. Samples may be freeze/thawed up to 3 times prior to testing.

c. Kit Stability

# Panther Fusion SARS-CoV-2/Flu A/B/RSV Reagents

Stability of the Panther Fusion SARS-CoV-2/Flu A/B/RSV assay cartridge and positive control were evaluated. Stability of universal Panther Fusion respiratory assay fluids (i.e., Panther Fusion Capture Reagents-S, Panther Fusion Enhancer Reagent-S, Panther Fusion Internal Control-S, Panther Fusion Elution Buffer, Panther Fusion Oil, Panther Fusion Reconstitution Buffer 1, and the negative control) were not assessed, as stability for these reagents has been previously established for Panther Fusion respiratory assays, including the Panther Fusion Flu A/B/RSV assay (K171963). Two types of stability studies were performed: Shelf-Life Stability and In-Use (or On-Board) Stability. The study data supports the storage conditions for the Panther Fusion SARS-CoV-2/Flu A/B/RSV Assay Cartridge and Positive Control illustrated in Table 13.

Table 13. Panther Fusion SARS-CoV-2/Flu A/B/RSV Assay Specific Reagent Storage Conditions   

<table><tr><td rowspan=1 colspan=1>Reagents</td><td rowspan=1 colspan=1>UnopenedStorage</td><td rowspan=1 colspan=1>On-Board/OpenStability</td><td rowspan=1 colspan=1>OpenedStorage</td></tr><tr><td rowspan=1 colspan=1>Panther Fusion SARS-CoV-2/Flu A/B/RSV Assay Cartridge</td><td rowspan=1 colspan=1>2 to 8</td><td rowspan=1 colspan=1>60 days</td><td rowspan=1 colspan=1>2 to 8</td></tr><tr><td rowspan=1 colspan=1>Panther Fusion SARS-CoV-2/Flu A/B/RSV Positive Control</td><td rowspan=1 colspan=1>2 to 8</td><td rowspan=1 colspan=1>Singleuse vial</td><td rowspan=1 colspan=1>NA*</td></tr></table>

\*NA-Not Applicable, as controls are single use.

# RespDirect Collection Kit

Stability of the RespDirect Collection Kit was evaluated by performing a Shelf-Life Study. The study data supports the storage conditions indicated in Table 14.

Table 14. RespDirect Collection Kit Storage Conditions   

<table><tr><td rowspan=1 colspan=1>Reagents</td><td rowspan=1 colspan=1>Unopened Storage</td><td rowspan=1 colspan=1>On-Board/Open Stability</td><td rowspan=1 colspan=1>OpenedStorage</td></tr><tr><td rowspan=1 colspan=1>RespDirect Collection Kit</td><td rowspan=1 colspan=1>15°C to 30</td><td rowspan=1 colspan=1>NA*</td><td rowspan=1 colspan=1>NA*</td></tr></table>

\*NA-Not Applicable

d. Shipping Stability

# Panther Fusion SARS-CoV-2/Flu A/B/RSV Reagents

The purpose of this study was to demonstrate that exposure to extreme hot or cold temperatures potentially encountered by the Panther Fusion SARS-CoV-/Flu A/B/RSV assay components during shipment would not impact the performance of the assay. One lot of cartridges and positive controls for the Panther Fusion SARS-CoV-/Flu A/B/RSV assay were tested in this study. Similar to the Kit Stability Study (see Section c. Kit Stability, Panther Fusion SARS-CoV-2/Flu A/B/RSV Reagents, above), stability of the universal Panther Fusion respiratory assay fluids were not assessed, as stability for these reagents has previously been shown. Components were exposed to the extreme temperatures in their final container closure systems.

The Panther Fusion SARS-CoV-/Flu A/B/RSV assay cartridges and positive controls were cycled between the extreme low temperature $( - 4 0 \pm 3 ^ { \circ } \mathrm { C } )$ , room temperature (28 $\pm 2 ^ { \circ } \mathrm { C } )$ , and extreme high temperature $( 5 5 \pm 5 ^ { \circ } \mathrm { C } )$ . One stress cycle consisted of incubation for at least 9 hours at extreme high temperatures and then at least 15 hours at roomtemperature, followed by at least 9 hours at extreme low temperature, followed by at least 15 hour incubation at room temperature (1 cycle $=$ approx. 48 hours). A total of 5 cycles were performed to evaluate the worst case scenario of shipment to customers.

All runs tested gave the expected result for spiked and un-spiked samples. This study demonstrates that Panther Fusion SARS-CoV-/Flu A/B/RSV assay cartridge and positive control are not altered by exposure to extreme hot or cold temperatures that may be encountered during shipping.

# RespDirect Collection Kit

The purpose of this study was to demonstrate that exposure to extreme hot or cold temperatures potentially encountered by the RespDirect Collection Kit during shipment would not impact performance of the RespDirect when tested with the Panther Fusion SARS-CoV-2/Flu A/B/RSV assay. One RespDirect Collection Kit lot was tested in this study. For this study, RespDirect Collection Kits were exposed to extreme temperatures in their final container closure systems.

RespDirect Collection Kits were cycled between the extreme low temperature $( - 7 0 ^ { \circ } \mathrm { C }$ , ranging from $- 6 5 ^ { \circ } \mathrm { C }$ to $- 8 5 ^ { \circ } \mathrm { C }$ ), room temperature $( 2 8 \pm 2 ^ { \circ } \mathrm { C } )$ , and extreme high temperature $( 5 5 \pm 5 ^ { \circ } \mathrm { C } )$ . One stress cycle consisted of incubation for at least 24 hours at extreme low temperatures and then at least 4 hours at room-temperature, followed by 24 hours at extreme high temperatures and then 4 hours at room-temperature (1 cycle $=$ approx. 56 hours). A total of 5 cycles were performed to evaluate the worst case scenario of shipment to customers.

All runs tested gave the expected result for spiked and un-spiked samples, except for one spiked sample at 3x LoD that was negative for SARS-CoV-2. This study demonstrates that the RespDirect Collection Kit is not altered by exposure to extreme hot or cold temperatures that may be encountered during shipping.

6. Detection Limit:

# Processed Specimens (i.e., NP Swab Specimen in VTM/UTM Diluted into Panther Fusion Specimen Lysis Tube, containing STM)

The analytical sensitivity (limit of detection or LoD) of the Panther Fusion SARS-CoV-2/Flu A/B/RSV assay was determined by testing processed negative clinical NP swab matrix spiked with multiple concentrations of the WHO International Standard for SARS-CoV-2 (NIBSC, 20/146) or viral cultures of SARS-CoV-2 (1 strain), Influenza A (one H1N1 and one H3N2 strain), Influenza B (one Victoria lineage and one Yamagata lineage), and RSV (one RSV A and one RSV B strain). To estimate the LoD, a minimum of 24 replicates were tested per concentration with each of three reagent lots. Using the estimated LoD study results, the LoD for each target was further estimated by Probit analysis for each reagent lot and was confirmed with an additional 24 replicates using a single reagent lot. The concentration with $2 9 5 \%$ detection was determined to be the final, confirmed LoD for each assay target. Confirmed LoD concentrations for each analyte are summarized in Table 15.

Table 15. Confirmatory LoD Study Results   

<table><tr><td rowspan=2 colspan=1>Target</td><td rowspan=2 colspan=1>Strain</td><td rowspan=2 colspan=1>LoD Conc.</td><td rowspan=1 colspan=4>Percent Positivity for each Target Analyte</td></tr><tr><td rowspan=1 colspan=1>SARS-CoV-2</td><td rowspan=1 colspan=1>Flu A</td><td rowspan=1 colspan=1>Flu B</td><td rowspan=1 colspan=1>RSV</td></tr><tr><td rowspan=2 colspan=1>SARS-CoV-2</td><td rowspan=1 colspan=1>USA-WA1/2020</td><td rowspan=1 colspan=1>0.03 TCID50/mL</td><td rowspan=1 colspan=1>95.8%(23/24)</td><td rowspan=1 colspan=1>0%(0/24)</td><td rowspan=1 colspan=1>0%(0/24)</td><td rowspan=1 colspan=1>0%(0/24)</td></tr><tr><td rowspan=1 colspan=1>WHOInternationalStandard (NIBSC,20/146)</td><td rowspan=1 colspan=1>47.20 IU/mL</td><td rowspan=1 colspan=1>95.8%(23/24)</td><td rowspan=1 colspan=1>0%(0/24)</td><td rowspan=1 colspan=1>0%(0/24)</td><td rowspan=1 colspan=1>0%(0/24)</td></tr><tr><td rowspan=2 colspan=1>Flu A</td><td rowspan=1 colspan=1>Brisbane/02/18(H1N1)</td><td rowspan=1 colspan=1>0.06 TCID50/mL</td><td rowspan=1 colspan=1>0%(0/24)</td><td rowspan=1 colspan=1>95.8%(23/24)</td><td rowspan=1 colspan=1>0%(0/24)</td><td rowspan=1 colspan=1>0%(0/24)</td></tr><tr><td rowspan=1 colspan=1>Kansas/14/17(H3N2)</td><td rowspan=1 colspan=1>0.1 TCID50/mL</td><td rowspan=1 colspan=1>0%(0/24)</td><td rowspan=1 colspan=1>100%(24/24)</td><td rowspan=1 colspan=1>0%(0/24)</td><td rowspan=1 colspan=1>0%(0/24)</td></tr><tr><td rowspan=2 colspan=1>Flu B</td><td rowspan=1 colspan=1>Phuket/3073/13(Yamagata)</td><td rowspan=1 colspan=1>0.003 TCID50/mL</td><td rowspan=1 colspan=1>0%(0/24)</td><td rowspan=1 colspan=1>0%(0/24)</td><td rowspan=1 colspan=1>95.8%(23/24)</td><td rowspan=1 colspan=1>0%(0/24)</td></tr><tr><td rowspan=1 colspan=1>Washington/02/19(Victoria)</td><td rowspan=1 colspan=1>0.03 TCID50/mL</td><td rowspan=1 colspan=1>0%(0/24)</td><td rowspan=1 colspan=1>0%(0/24)</td><td rowspan=1 colspan=1>100%(24/24)</td><td rowspan=1 colspan=1>0%(0/24)</td></tr><tr><td rowspan=2 colspan=1>RSV</td><td rowspan=1 colspan=1>RSV A</td><td rowspan=1 colspan=1>0.03 TCID50/mL</td><td rowspan=1 colspan=1>0%(0/24)</td><td rowspan=1 colspan=1>0%(0/24)</td><td rowspan=1 colspan=1>100%(24/24)</td><td rowspan=1 colspan=1>0%(0/24)</td></tr><tr><td rowspan=1 colspan=1>RSVB</td><td rowspan=1 colspan=1>0.03 TCID50/mL</td><td rowspan=1 colspan=1>0%(0/24)</td><td rowspan=1 colspan=1>0%(0/24)</td><td rowspan=1 colspan=1>100%(24/24)</td><td rowspan=1 colspan=1>0%(0/24)</td></tr></table>

The LoD for co-analyte spiked samples was also evaluated and shown to be equivalent to single analyte spiked samples.

RespDirect Collection Kit (eSTM) The LoD of the Panther Fusion SARS-CoV-2/Flu A/B/RSV assay was assessed with the RespDirect Collection Kit (containing eSTM). For this study, negative clinical eSTM matrix spiked with WHO International Standard for SARS-CoV-2 (NIBSC, 20/146) or viral cultures of Influenza A (one H3N2 strain), Influenza B (one Victoria lineage), and RSV (one RSV A and one RSV B strain) were evaluated. Thirty (30) replicates were tested for each concentration/RespDirect Collection Kit workflow. The RespDirect Collection Kit can be handled according to the following workflows described in the Panther Fusion SARS-CoV2/Flu A/B/RSV assay IFU: (1) directly load tube on the Panther Fusion Instrument or, (2) vortex the tube for 10 minutes at $1 { , } 8 0 0 \mathrm { r p m }$ on a multi-tube vortexer prior to loading on the Panther Fusion Instrument or, (3) vortex tube for 15 seconds using a standard bench top vortexer prior to loading on the Panther Fusion Instrument. The concentration with $2 9 5 \%$ detection was determined to be the final, confirmed LoD for each of the assay targets. The RespDirect Collection Kit workflows yielded equivalent LoDs for each assay target. The LoD concentrations for each analyte are outlined below.

• SARS-CoV-2, WHO International Standard (NIBSC, 20/146): 98.6 IU/mL  
• Influenza A (H3N2): 0.11 TCID50/mL  
• Influenza B (Victoria lineage): 0.03 TCID50/mL  
• RSV A: 0.03 TCID50/mL  
• RSV B: 0.05 TCID50/mL

The LoD for co-analyte spiked samples was also evaluated and shown to be equivalent to single analyte spiked samples.

# 7. Assay Cut-Off:

The relative fluorescence units (RFU) range is the difference between maximum and minimum fluorescent signal seen in a sample during amplification. For Panther Fusion SARS-CoV-2/Flu A/B/RSV positive samples, the amplification curve rises above the background florescence RFU resulting in a high RFU range for the intended targets. The IC resembles a positive target and also has a high RFU range. For the Panther Fusion SARSCoV-2/Flu A/B/RSV negative samples, the amplification curve remains flat resulting in a low RFU range for the intended targets. This difference in RFU range is the primary criteria used to distinguish positive and negative specimens. RFU range thresholds were set at 500 for SARS-CoV-2, 1,000 for Flu A, 1000 for Flu B, and 775 for RSV for determination of positivity in their specific channels (ROX, FAM, RED647, HEX, respectively) and 1000 RFU range for IC for determination of validity in its specific channel (RED677). The RFU range thresholds were set based on data from multiple RFU readings from known positive samples. The cycle number at which the amplification curve crosses target specific assigned RFU range threshold value is called Ct. Target specific Ct will be generated only for positive valid reaction for that target.

8. Accuracy (Instrument): Not applicable

9. Carry-Over:

RespDirect Collection Kit   
Carry-over/cross-contamination with processed specimens (i.e., NP specimens in VTM/UTM diluted into STM) was previously evaluated in K171963 and found acceptable. Therefore, carry-over/cross-contamination with processed specimens was not evaluated with the Panther Fusion SARS-CoV-2/Flu A/B/RSV assay.

The purpose of this study was to demonstrate that no carry-over/cross-contamination is observed for NP specimens collected in eSTM using the RespDirect Collection Kit when tested with the Panther Fusion SARS-CoV-2/Flu A/B/RSV assay.

Carry-over or cross-contamination in the RespDirect Collection Kit was assessed with pooled negative clinical matrix collected in RespDirect Collection Kits. Pooled negative clinical matrix were used to prepare negative and high-titer positive panels. A high-titer positive panel was prepared by spiking one strain of Flu A (H3N2, Kansas/14/17) in negative clinical matrix at a concentration of $1 \mathrm { \Delta x 1 0 ^ { 4 } T C I D 5 0 / m L }$ . Non-spiked negative clinical matrix was used as the negative panel. Panels were tested in a checkboard pattern by alternating hightiter positive and negative samples over 5 total runs on each of two Panther Fusion instruments. Each run included 30 positive and 30 negative reactions (60 total reactions). A baseline run of 100 negative reactions was run prior to the checkerboard pattern runs. One lot of Panther Fusion SARS-CoV-2 /Flu A/B/RSV assay cartridges was used. No carry-over was observed.

# B Comparison Studies:

1. Method Comparison with Predicate Device: Not applicable

2. Matrix Comparison:

# a. Transport Media Equivalency Study

The purpose of this study was to demonstrate equivalency between viral transport media types indicated for use with the Panther Fusion SARS-CoV-2/Flu A/B/RSV assay. The following transport media types were included in the study: Remel Micro Test M4RT, Remel Micro Test M5, Remel Micro Test M6, BD Universal Viral Transport Media, Copan Universal Transport Medium, and Hardy Diagnostics Viral Transport Media. To demonstrate equivalency between the viral transport medias, contrived positive samples were created by spiking one strain of SARS-CoV-2, Flu A H3N2, Flu B Victoria, and RSV B into either true negative clinical NP swab matrix or simulated NP swab matrix (i.e., transport media spiked with HeLa cells at a concentration of $2 \mathrm { x } 1 0 ^ { 4 }$ cells/mL) at 3 concentrations (0.5x, 1x, and 5x LoD). Flu A, Flu B, and RSV B were co-spiked while SARS-CoV-2 was evaluated independently. Contrived positives generated with true clinical NP matrix were included to demonstrate equivalency between clinical NP matrix and simulated NP matrix, supporting the use of simulated viral transport media in the study. Twenty (20) replicates per target concentration were tested for each transport media type and analyte concentration. The results are shown in Table 16 and support that the Panther Fusion SARS-CoV-2/Flu A/B RSV assay can be used with the evaluated viral transport media types.

Table 16. Transport Media Equivalency Study Results   

<table><tr><td colspan="1" rowspan="1">Target</td><td colspan="1" rowspan="1">Media</td><td colspan="2" rowspan="1">Concentration</td><td colspan="1" rowspan="1">% Positivity(n pos/n Tested)</td></tr><tr><td colspan="1" rowspan="4">None, NegativeSamples</td><td colspan="1" rowspan="1">Remel M4RT</td><td colspan="2" rowspan="1"></td><td colspan="1" rowspan="1">0% (0/20)</td></tr><tr><td colspan="1" rowspan="1">BD UVT</td><td colspan="2" rowspan="3">Negative</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">0%(0/20)</td></tr><tr><td colspan="1" rowspan="1">CopanUTM</td><td colspan="1" rowspan="1">0%(0/20)</td></tr><tr><td colspan="1" rowspan="1">Hardy VTM</td><td colspan="1" rowspan="1">0% (0/20)</td></tr><tr><td colspan="1" rowspan="5"></td><td colspan="1" rowspan="1">NP Samples1</td><td colspan="2" rowspan="1">Y</td><td colspan="1" rowspan="1">0% (0/20)</td></tr><tr><td colspan="1" rowspan="3">Remel M5</td><td colspan="1" rowspan="3"></td><td></td><td></td></tr><tr><td colspan="1" rowspan="2"></td><td></td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">0% (0/20)</td></tr><tr><td colspan="1" rowspan="1">Remel M6</td><td colspan="2" rowspan="1">Y</td><td colspan="1" rowspan="1">0% (0/20)</td></tr><tr><td colspan="1" rowspan="21">SARS-CoV-2</td><td colspan="1" rowspan="3">Remel M4RT</td><td colspan="2" rowspan="1">0.5x LoD</td><td colspan="1" rowspan="1">90% (18/20)</td></tr><tr><td colspan="2" rowspan="1">1x LoD</td><td colspan="1" rowspan="1">100% (20/20)</td></tr><tr><td colspan="2" rowspan="1">5x LoD</td><td colspan="1" rowspan="1">100% (20/20)</td></tr><tr><td colspan="1" rowspan="3">BD UVT</td><td colspan="2" rowspan="1">0.5x LoD</td><td colspan="1" rowspan="1">95% (19/20)</td></tr><tr><td colspan="2" rowspan="1">1x LoD</td><td colspan="1" rowspan="1">100% (20/20)</td></tr><tr><td colspan="2" rowspan="1">5x LoD</td><td colspan="1" rowspan="1">100% (20/20)</td></tr><tr><td colspan="1" rowspan="3">Copan UTM</td><td colspan="2" rowspan="1">0.5x LoD</td><td colspan="1" rowspan="1">80% (16/20)</td></tr><tr><td colspan="2" rowspan="1">1x LoD</td><td colspan="1" rowspan="1">95% (19/20)</td></tr><tr><td colspan="2" rowspan="1">5x LoD</td><td colspan="1" rowspan="1">100% (20/20)</td></tr><tr><td colspan="1" rowspan="3">Hardy VTM</td><td colspan="2" rowspan="1">0.5x LoD</td><td colspan="1" rowspan="1">80% (16/20)</td></tr><tr><td colspan="2" rowspan="1">1x LoD</td><td colspan="1" rowspan="1">100% (20/20)</td></tr><tr><td colspan="2" rowspan="1">5x LoD</td><td colspan="1" rowspan="1">100% (20/20)</td></tr><tr><td colspan="1" rowspan="3">NP Samples</td><td colspan="2" rowspan="1">0.5x LoD</td><td colspan="1" rowspan="1">95% (19/20)</td></tr><tr><td colspan="2" rowspan="1">1x LoD</td><td colspan="1" rowspan="1">100% (20/20)</td></tr><tr><td colspan="2" rowspan="1">5x LoD</td><td colspan="1" rowspan="1">100% (20/20)</td></tr><tr><td colspan="1" rowspan="3">Remel M5</td><td colspan="2" rowspan="1">0.5x LoD</td><td colspan="1" rowspan="1">90% (18/20)</td></tr><tr><td colspan="2" rowspan="1">1x LoD</td><td colspan="1" rowspan="1">100% (20/20)</td></tr><tr><td colspan="2" rowspan="1">5x LoD</td><td colspan="1" rowspan="1">100% (20/20)</td></tr><tr><td colspan="1" rowspan="3">Remel M6</td><td colspan="2" rowspan="1">0.5x LoD</td><td colspan="1" rowspan="1">95% (19/20)</td></tr><tr><td colspan="2" rowspan="1">1x LoD</td><td colspan="1" rowspan="1">100% (20/20)</td></tr><tr><td colspan="2" rowspan="1">5x LoD</td><td colspan="1" rowspan="1">100% (20/20)</td></tr><tr><td colspan="1" rowspan="21">Flu A H3N2</td><td colspan="1" rowspan="3">Remel M4RT</td><td colspan="2" rowspan="1">0.5x LoD</td><td colspan="1" rowspan="1">95% (19/20)</td></tr><tr><td colspan="2" rowspan="1">1x LoD</td><td colspan="1" rowspan="1">95% (19/20)</td></tr><tr><td colspan="2" rowspan="1">5x LoD</td><td colspan="1" rowspan="1">100% (20/20)</td></tr><tr><td colspan="1" rowspan="3">BD UVT</td><td colspan="2" rowspan="1">0.5x LoD</td><td colspan="1" rowspan="1">85% (17/20)</td></tr><tr><td colspan="2" rowspan="1">1x LoD</td><td colspan="1" rowspan="1">100% (20/20)</td></tr><tr><td colspan="2" rowspan="1">5x LoD</td><td colspan="1" rowspan="1">100% (20/20)</td></tr><tr><td colspan="1" rowspan="3">Copan UTM</td><td colspan="2" rowspan="1">0.5x LoD</td><td colspan="1" rowspan="1">75% (15/20)</td></tr><tr><td colspan="2" rowspan="1">1x LoD</td><td colspan="1" rowspan="1">100% (20/20)</td></tr><tr><td colspan="2" rowspan="1">5x LoD</td><td colspan="1" rowspan="1">100% (20/20)</td></tr><tr><td colspan="1" rowspan="3">Hardy VTM</td><td colspan="2" rowspan="1">0.5x LoD</td><td colspan="1" rowspan="1">90% (18/20)</td></tr><tr><td colspan="2" rowspan="1">1x LoD</td><td colspan="1" rowspan="1">100% (20/20)</td></tr><tr><td colspan="2" rowspan="1">5x LoD</td><td colspan="1" rowspan="1">100% (20/20)</td></tr><tr><td colspan="1" rowspan="3">NP Samples</td><td colspan="2" rowspan="1">0.5x LoD</td><td colspan="1" rowspan="1">85% (17/20)</td></tr><tr><td colspan="2" rowspan="1">1x LoD</td><td colspan="1" rowspan="1">100% (20/20)</td></tr><tr><td colspan="2" rowspan="1">5x LoD</td><td colspan="1" rowspan="1">100% (20/20)</td></tr><tr><td colspan="1" rowspan="3">Remel M5</td><td colspan="2" rowspan="1">0.5x LoD</td><td colspan="1" rowspan="1">95% (19/20)</td></tr><tr><td colspan="2" rowspan="1">1x LoD</td><td colspan="1" rowspan="1">100% (20/20)</td></tr><tr><td colspan="2" rowspan="1">5x LoD</td><td colspan="1" rowspan="1">100% (20/20)</td></tr><tr><td colspan="1" rowspan="3">Remel M6</td><td colspan="2" rowspan="1">0.5x LoD</td><td colspan="1" rowspan="1">70% (14/20)</td></tr><tr><td colspan="2" rowspan="1">1x LoD</td><td colspan="1" rowspan="1">100% (20/20)</td></tr><tr><td colspan="2" rowspan="1">5x LoD</td><td colspan="1" rowspan="1">100% (20/20)</td></tr><tr><td colspan="1" rowspan="6">Flu B Victoria</td><td colspan="1" rowspan="3">Remel M4RT</td><td colspan="2" rowspan="1">0.5x LoD</td><td colspan="1" rowspan="1">95% (19/20)</td></tr><tr><td colspan="2" rowspan="1">1x LoD</td><td colspan="1" rowspan="1">100% (20/20)</td></tr><tr><td colspan="2" rowspan="1">5x LoD</td><td colspan="1" rowspan="1">100% (20/20)</td></tr><tr><td colspan="1" rowspan="3">BD UVT</td><td colspan="2" rowspan="1">0.5x LoD</td><td colspan="1" rowspan="1">95% (19/20)</td></tr><tr><td colspan="2" rowspan="1">1x LoD</td><td colspan="1" rowspan="1">100% (20/20)</td></tr><tr><td colspan="2" rowspan="1">5x LoD</td><td colspan="1" rowspan="1">100% (20/20)</td></tr></table>

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>EY</td></tr><tr><td rowspan=15 colspan=1></td><td rowspan=3 colspan=1>Copan UTM</td><td rowspan=1 colspan=1>0.5x LoD</td><td rowspan=1 colspan=1>100% (20/20)</td></tr><tr><td rowspan=1 colspan=1>1x LoD</td><td rowspan=1 colspan=1>100% (20/20)</td></tr><tr><td rowspan=1 colspan=1>5x LoD</td><td rowspan=1 colspan=1>100%(20/20)</td></tr><tr><td rowspan=3 colspan=1>Hardy VTM</td><td rowspan=1 colspan=1>0.5x LoD</td><td rowspan=1 colspan=1>100%(20/20)</td></tr><tr><td rowspan=1 colspan=1>1x LoD</td><td rowspan=1 colspan=1>100%(20/20)</td></tr><tr><td rowspan=1 colspan=1>5x LoD</td><td rowspan=1 colspan=1>100%(20/20)</td></tr><tr><td rowspan=3 colspan=1>NP Samples</td><td rowspan=1 colspan=1>0.5x LoD</td><td rowspan=1 colspan=1>100% (20/20)</td></tr><tr><td rowspan=1 colspan=1>1x LoD</td><td rowspan=1 colspan=1>100%(20/20)</td></tr><tr><td rowspan=1 colspan=1>5x LoD</td><td rowspan=1 colspan=1>100% (20/20)</td></tr><tr><td rowspan=3 colspan=1>Remel M5</td><td rowspan=1 colspan=1>0.5x LoD</td><td rowspan=1 colspan=1>100% (20/20)</td></tr><tr><td rowspan=1 colspan=1>1x LoD</td><td rowspan=1 colspan=1>100%(20/20)</td></tr><tr><td rowspan=1 colspan=1>5x LoD</td><td rowspan=1 colspan=1>100% (20/20)</td></tr><tr><td rowspan=3 colspan=1>Remel M6</td><td rowspan=1 colspan=1>0.5x LoD</td><td rowspan=1 colspan=1>95% (19/20)</td></tr><tr><td rowspan=1 colspan=1>1x LoD</td><td rowspan=1 colspan=1>95% (19/20)</td></tr><tr><td rowspan=1 colspan=1>5x LoD</td><td rowspan=1 colspan=1>100% (20/20)</td></tr><tr><td rowspan=21 colspan=1>RSV B</td><td rowspan=3 colspan=1>Remel M4RT</td><td rowspan=1 colspan=1>0.5x LoD</td><td rowspan=1 colspan=1>90% (18/20)</td></tr><tr><td rowspan=1 colspan=1>1x LoD</td><td rowspan=1 colspan=1>100% (20/20)</td></tr><tr><td rowspan=1 colspan=1>5x LoD</td><td rowspan=1 colspan=1>100% (20/20)</td></tr><tr><td rowspan=3 colspan=1>BD UVT</td><td rowspan=1 colspan=1>0.5x LoD</td><td rowspan=1 colspan=1>95% (19/20)</td></tr><tr><td rowspan=1 colspan=1>1x LoD</td><td rowspan=1 colspan=1>100% (20/20)</td></tr><tr><td rowspan=1 colspan=1>5x LoD</td><td rowspan=1 colspan=1>100% (20/20)</td></tr><tr><td rowspan=3 colspan=1>Copan UTM</td><td rowspan=1 colspan=1>0.5x LoD</td><td rowspan=1 colspan=1>95% (19/20)</td></tr><tr><td rowspan=1 colspan=1>1x LoD</td><td rowspan=1 colspan=1>100% (20/20)</td></tr><tr><td rowspan=1 colspan=1>5x LoD</td><td rowspan=1 colspan=1>100% (20/20)</td></tr><tr><td rowspan=3 colspan=1>Hardy VTM</td><td rowspan=1 colspan=1>0.5x LoD</td><td rowspan=1 colspan=1>95% (19/20)</td></tr><tr><td rowspan=1 colspan=1>1x LoD</td><td rowspan=1 colspan=1>100% (20/20)</td></tr><tr><td rowspan=1 colspan=1>5x LoD</td><td rowspan=1 colspan=1>100% (20/20)</td></tr><tr><td rowspan=3 colspan=1>NP Samples</td><td rowspan=1 colspan=1>0.5x LoD</td><td rowspan=1 colspan=1>95% (19/20)</td></tr><tr><td rowspan=1 colspan=1>1x LoD</td><td rowspan=1 colspan=1>100% (20/20)</td></tr><tr><td rowspan=1 colspan=1>5x LoD</td><td rowspan=1 colspan=1>100% (20/20)</td></tr><tr><td rowspan=3 colspan=1>Remel M5</td><td rowspan=1 colspan=1>0.5x LoD</td><td rowspan=1 colspan=1>90% (18/20)</td></tr><tr><td rowspan=1 colspan=1>1x LoD</td><td rowspan=1 colspan=1>100% (20/20)</td></tr><tr><td rowspan=1 colspan=1>5x LoD</td><td rowspan=1 colspan=1>100% (20/20)</td></tr><tr><td rowspan=3 colspan=1>Remel M6</td><td rowspan=1 colspan=1>0.5x LoD</td><td rowspan=1 colspan=1>90% (18/20</td></tr><tr><td rowspan=1 colspan=1>1x LoD</td><td rowspan=1 colspan=1>100% (20/20)</td></tr><tr><td rowspan=1 colspan=1>5x LoD</td><td rowspan=1 colspan=1>100% (20/20)</td></tr></table>

1 NP Samples $=$ Contrived positive samples generated with true negative clinical NP swab matrix

b. Collection Device Equivalency – RespDirect Collection Kit (eSTM) vs. VTM/UTM The purpose of this study was to demonstrate equivalency between two collection devices: (1) NP specimens collected into VTM/UTM and (2) NP specimens collected with the RespDirect Collection Kit (eSTM). For this study, contrived positive panels were prepared by adding one strain each of SARS-CoV-2 (WHO International Standard), Flu A (H3N2), Flu B (Victoria), and RSV B into paired, negative clinical NP eSTM and negative clinical NP VTM/UTM matrix collected from individuals with signs and symptoms of respiratory infection. Contrived panels were generated at both $2 \mathbf { x }$ and ${ 5 } \mathrm { x }$ LoD. SARS-CoV-2 was co-spiked with Flu A, Flu B, and RSV and was also tested independently. In total, 100 SARS-CoV-2 positives and 50 positives each for Flu A, Flu B, and RSV were evaluated. Additionally, 181 negative samples, consisting of negative clinical NP matrix, only, were included in the study. Results are shown in Table 17, below and demonstrate that the two collection devices are equivalent.

Table 17. Collection Device Equivalency Study Results   

<table><tr><td rowspan=1 colspan=1>Analyte</td><td rowspan=1 colspan=1>SampleConcentration</td><td rowspan=1 colspan=1>N per CollectionDevice</td><td rowspan=1 colspan=1>VTM/UTM% Positive (NPos/N Tested)</td><td rowspan=1 colspan=1>RespDirect %Positive (NPos/N Tested)</td></tr><tr><td rowspan=1 colspan=1>None (NegativeSamples)</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>181</td><td rowspan=1 colspan=1>0% (0/181)</td><td rowspan=1 colspan=1>0% (0/181)</td></tr><tr><td rowspan=2 colspan=1>SARS-CoV-2</td><td rowspan=1 colspan=1>2x</td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>100% (50/50)</td><td rowspan=1 colspan=1>98% (49/50)</td></tr><tr><td rowspan=1 colspan=1>5x</td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>100% (50/50)</td><td rowspan=1 colspan=1>100% (50/50)</td></tr><tr><td rowspan=2 colspan=1>Flu A (H3N2)</td><td rowspan=1 colspan=1>2x</td><td rowspan=1 colspan=1>25</td><td rowspan=1 colspan=1>100% (25/25)</td><td rowspan=1 colspan=1>100% (25/25)</td></tr><tr><td rowspan=1 colspan=1>5x</td><td rowspan=1 colspan=1>25</td><td rowspan=1 colspan=1>100% (25/25)</td><td rowspan=1 colspan=1>100% (25/25)</td></tr><tr><td rowspan=2 colspan=1>Flu B (Victoria)</td><td rowspan=1 colspan=1>2x</td><td rowspan=1 colspan=1>25</td><td rowspan=1 colspan=1>100% (25/25)</td><td rowspan=1 colspan=1>100% (25/25)</td></tr><tr><td rowspan=1 colspan=1>5x</td><td rowspan=1 colspan=1>25</td><td rowspan=1 colspan=1>100%(25/25)</td><td rowspan=1 colspan=1>100%b (25/25)</td></tr><tr><td rowspan=2 colspan=1>RSV B</td><td rowspan=1 colspan=1>2x</td><td rowspan=1 colspan=1>25</td><td rowspan=1 colspan=1>100% (25/25)</td><td rowspan=1 colspan=1>100% (25/25)</td></tr><tr><td rowspan=1 colspan=1>5x</td><td rowspan=1 colspan=1>25</td><td rowspan=1 colspan=1>100% (25/25)</td><td rowspan=1 colspan=1>100% (25/25)</td></tr></table>

# C Clinical Studies:

1. Clinical Sensitivity:

# Prospective Study

The clinical performance of the Panther Fusion SARS-CoV-2/Flu A/B/RSV assay was established in a multi-center study conducted with residual (leftover) and de-identified nasopharyngeal swab (NPS) specimens in VTM or UTM that were prospectively collected from patients with signs and symptoms of respiratory tract infections during periods of the 2020-2022 respiratory illness seasons. NPS specimens, from five geographically diverse clinical sites in the U.S., were enrolled and tested with the Panther Fusion SARS-CoV-2/Flu A/B/RSV assay at three U.S. testing sites.

The Panther Fusion SARS-CoV-2/Flu A/B/RSV assay was evaluated for SARS-CoV-2 performance by comparing the candidate device testing results to a composite comparator algorithm (CCA) consisting of two highly sensitive U.S. FDA EUA SARS-CoV-2 molecular tests and a validated PCR followed by bi-directional sequencing (PCR/BDS) assay. A final CCA result was assigned when two of the three composite comparator assays were in concordance. The comparator method utilized to establish performance for the Flu A, Flu B, and RSV targets was a U.S. FDA-cleared molecular Flu A/B/RSV assay. All comparator testing was performed in accordance with the respective package inserts.

The Panther Fusion SARS-CoV-2/Flu A/B/RSV was evaluated with prospectively (i.e., all comers between two time points that met the clinical study inclusion criteria) collected specimens. Both Category I specimens (i.e., prospectively collected, tested fresh) and Category II specimens (i.e., prospectively collected, tested frozen) were included in the study.

A total of 1949 residual NPS specimens were acquired and enrolled for the prospective clinical study. Of these 1949 specimens, 1056 were collected between January 2022 and

March 2022, while the remaining 893 were collected between November 2020 and March 2021. Forty-five (45) of these specimens were withdrawn from the study because of a reception delay that resulted in frozen specimens being received thawed $( \mathrm { n } { = } 4 4 )$ or a specimen not being handled according to the package insert $\scriptstyle ( \mathrm { n = 1 } )$ ). In addition, another 4 specimens were withdrawn because they yielded invalid results with the investigational device upon retesting $( \mathrm { n } { = } 4 )$ , bringing the total number of withdrawn samples to forty-nine. The initial prospective clinical study invalid rate was $0 . 6 \%$ (12/1904), and $0 . 2 \%$ (4/1904) following retesting.

This left 1900 prospective specimens enrolled in the evaluable population. For the SARSCoV-2 target, 13 specimens were excluded from analysis due to unknown infection status obtained from the CCA tests. This left a total of 1887 prospective samples evaluated for SARS-CoV-2, of which 790 $( 4 1 . 9 \% )$ were tested fresh and 1097 $( 5 8 . 1 \% )$ which were tested frozen with the Panther Fusion SARS-CoV-2/Flu A/B/RSV assay. For the Flu A, Flu B, and RSV targets, 63 specimens were excluded from analysis due to obtaining an invalid result from the comparator test. This left a total of 1837 valid prospective specimens evaluated for Flu A, Flu B, and RSV, of which 798 $( 4 3 . 4 \% )$ were tested fresh and 1039 $( 5 6 . 6 \% )$ were tested frozen with the Panther Fusion SARS-CoV-2/Flu A/B/RSV assay. Table 18 below provides a summary of demographic information for the 1900 specimens included in the prospective clinical study.

Table 18. Demographic Data for Prospectively Collected Specimens   

<table><tr><td rowspan=1 colspan=2></td><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>Site 1</td><td rowspan=1 colspan=1>Site 2</td><td rowspan=1 colspan=1>Site 3</td><td rowspan=1 colspan=1>Site 4</td><td rowspan=1 colspan=1>Site 5</td></tr><tr><td rowspan=3 colspan=1>Sex</td><td rowspan=1 colspan=1>Male</td><td rowspan=1 colspan=1>850 (44.7%)</td><td rowspan=1 colspan=1>99</td><td rowspan=1 colspan=1>209</td><td rowspan=1 colspan=1>316</td><td rowspan=1 colspan=1>216</td><td rowspan=1 colspan=1>10</td></tr><tr><td rowspan=1 colspan=1>Female</td><td rowspan=1 colspan=1>1049 (55.2%)</td><td rowspan=1 colspan=1>82</td><td rowspan=1 colspan=1>200</td><td rowspan=1 colspan=1>527</td><td rowspan=1 colspan=1>228</td><td rowspan=1 colspan=1>12</td></tr><tr><td rowspan=1 colspan=1>Unknown</td><td rowspan=1 colspan=1>1 (0.1%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=5 colspan=1>Age</td><td rowspan=1 colspan=1>&lt;5 years</td><td rowspan=1 colspan=1>388 (20.4%)</td><td rowspan=1 colspan=1>86</td><td rowspan=1 colspan=1>252</td><td rowspan=1 colspan=1>42</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>3</td></tr><tr><td rowspan=1 colspan=1>5-21 years</td><td rowspan=1 colspan=1>435 (22.9%)</td><td rowspan=1 colspan=1>77</td><td rowspan=1 colspan=1>156</td><td rowspan=1 colspan=1>173</td><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>12</td></tr><tr><td rowspan=1 colspan=1>22-40 years</td><td rowspan=1 colspan=1>372 (19.6%)</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>286</td><td rowspan=1 colspan=1>70</td><td rowspan=1 colspan=1>7</td></tr><tr><td rowspan=1 colspan=1>41-60 years</td><td rowspan=1 colspan=1>326 (17.2%)</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>212</td><td rowspan=1 colspan=1>111</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>&gt;60 years</td><td rowspan=1 colspan=1>379 (19.9%)</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>131</td><td rowspan=1 colspan=1>241</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>1900</td><td rowspan=1 colspan=1>181</td><td rowspan=1 colspan=1>409</td><td rowspan=1 colspan=1>844</td><td rowspan=1 colspan=1>444</td><td rowspan=1 colspan=1>22</td></tr></table>

A summary of the Panther Fusion SARS-CoV-2/Flu A/B/RSV assay prospective clinical study performance is provided in Table 19. Positive Percent Agreement (PPA) was calculated as $1 0 0 \% \times \left( \mathrm { T P } / \left( \mathrm { T P } + \mathrm { F N } \right) \right)$ . True positive (TP) indicates that both the Panther Fusion SARS-CoV-2/Flu A/B/RSV assay and the comparator method had a positive result for the specific analyte, and false negative (FN) indicates that the Panther Fusion SARSCoV-2/Flu A/B/RSV was negative while the comparator result was positive. Negative Percent Agreement (NPA) was calculated as $1 0 0 \% \times \left( \mathrm { T N / \left( T N + F P \right) } \right)$ . True negative (TN) indicates that both the Panther Fusion SARS-CoV-2/Flu A/B/RSV assay and the comparator method had negative results, and false positive (FP) indicates that the Panther Fusion SARSCoV-2/Flu A/B/RSV assay was positive while the comparator result was negative.

Specimens that obtained discordant results underwent additional testing with a U.S. FDA EUA SARS-CoV-2/Flu A/B/RSV molecular test, when sufficient sample volume remained.

Table 19. Panther Fusion SARS-CoV-2/Flu A/B/RSV Assay Performance   

<table><tr><td rowspan=2 colspan=2>Analyte</td><td rowspan=1 colspan=3>Positive Percent Agreement</td><td rowspan=1 colspan=3>Negative Percent Agreement</td></tr><tr><td rowspan=1 colspan=1>TP/(TP+FN)</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=1>TN/(TN+FP)</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=3 colspan=1>SARS-CoV-2</td><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>60/64</td><td rowspan=1 colspan=1>93.8</td><td rowspan=1 colspan=1>85.0-97.5</td><td rowspan=1 colspan=1>716/726</td><td rowspan=1 colspan=1>98.6</td><td rowspan=1 colspan=1>97.5-99.3</td></tr><tr><td rowspan=1 colspan=1>Frozen</td><td rowspan=1 colspan=1>318/326</td><td rowspan=1 colspan=1>97.5</td><td rowspan=1 colspan=1>95.2-98.8</td><td rowspan=1 colspan=1>758/771</td><td rowspan=1 colspan=1>98.3</td><td rowspan=1 colspan=1>97.1-99.0</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>378/390</td><td rowspan=1 colspan=1>96.9</td><td rowspan=1 colspan=1>94.7-98.2</td><td rowspan=1 colspan=1>1474/14972</td><td rowspan=1 colspan=1>98.5</td><td rowspan=1 colspan=1>97.7-99.0</td></tr><tr><td rowspan=3 colspan=1>Flu A</td><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>98/100</td><td rowspan=1 colspan=1>98.0</td><td rowspan=1 colspan=1>93.0-99.5</td><td rowspan=1 colspan=1>696/698</td><td rowspan=1 colspan=1>99.7</td><td rowspan=1 colspan=1>99.0-99.9</td></tr><tr><td rowspan=1 colspan=1>Frozen</td><td rowspan=1 colspan=1>23/23</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>85.7-100</td><td rowspan=1 colspan=1>1013/1016</td><td rowspan=1 colspan=1>99.7</td><td rowspan=1 colspan=1>99.1-99.9</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>121/123</td><td rowspan=1 colspan=1>98.4</td><td rowspan=1 colspan=1>94.3-99.6</td><td rowspan=1 colspan=1>1709/17143</td><td rowspan=1 colspan=1>99.7</td><td rowspan=1 colspan=1>99.3-99.9</td></tr><tr><td rowspan=3 colspan=1>Flu B</td><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>NC</td><td rowspan=1 colspan=1>NC</td><td rowspan=1 colspan=1>796/798</td><td rowspan=1 colspan=1>99.7</td><td rowspan=1 colspan=1>99.1-99.9</td></tr><tr><td rowspan=1 colspan=1>Frozen</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>NC</td><td rowspan=1 colspan=1>NC</td><td rowspan=1 colspan=1>1037/1039</td><td rowspan=1 colspan=1>99.8</td><td rowspan=1 colspan=1>99.3-99.9</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>NC</td><td rowspan=1 colspan=1>NC</td><td rowspan=1 colspan=1>1833/18374</td><td rowspan=1 colspan=1>99.8</td><td rowspan=1 colspan=1>99.4-99.9</td></tr><tr><td rowspan=3 colspan=1>RSV</td><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>11/13</td><td rowspan=1 colspan=1>84.6</td><td rowspan=1 colspan=1>57.8-95.7</td><td rowspan=1 colspan=1>785/785</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>99.5-100</td></tr><tr><td rowspan=1 colspan=1>Frozen</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>NC</td><td rowspan=1 colspan=1>NC</td><td rowspan=1 colspan=1>1039/1039</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>99.6-100</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>11/13</td><td rowspan=1 colspan=1>84.6</td><td rowspan=1 colspan=1>57.8-95.7</td><td rowspan=1 colspan=1>1824/1824</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>99.8-100</td></tr></table>

TP – true positive; FN – false negative; TN – true negative; FP – false positive; $\mathrm { N C } - \mathrm { N o t }$ calculable 1 Five (5) specimens with false negative SARS-CoV-2 results had sufficient sample volume remaining for discordant testing. All five specimens were positive for SARS-CoV-2 by a U.S. FDA EUA SARS-CoV-2/Flu A/B/RSV molecular test.

2 Eleven (11) specimens with false positive SARS-CoV-2 results had sufficient sample volume remaining for discordant testing. Seven of the specimens were negative for SARS-CoV-2 by a U.S. FDA EUA SARS-CoV-2/Flu A/B/RSV molecular test.

3 Two (2) specimens with false positive Flu A results had sufficient sample volume remaining for discordant testing. Both specimens were negative for Flu A by a U.S. FDA EUA SARS-CoV-2/Flu A/B/RSV molecular test.

4 One (1) specimens with false positive Flu B result had sufficient sample volume remaining for discordant testing. This specimen was negative for Flu B by a U.S. FDA EUA SARS-CoV-2/Flu A/B/RSV molecular test.

The Panther Fusion SARS-CoV-2/Flu A/B/RSV assay reported a total of 5 prospective specimens with multiple pathogen detections $0 . 2 6 \%$ of all prospective specimens, 5/1900; and $0 . 9 2 \%$ of all positive specimens, 5/542). All multiple detections identified contained two pathogens, with four specimens detecting SARS-CoV-2 and Flu A, and one specimen detecting SARS-CoV-2 and Flu B. In one specimen positive on the Panther Fusion SARSCoV-2/ Flu A/B/RSV assay for SARS-CoV-2 and Flu A, Flu A was not detected by the comparator test, and the multiple detection including Flu B was not detected on the comparator test (Table 20).

Table 20. Co-infections Detected by the Panther Fusion SARS-CoV-2/Flu A/B/RSV Assay in the Prospectively Collected Specimens   

<table><tr><td rowspan=1 colspan=1>Co-infectionCombination</td><td rowspan=1 colspan=1>Number ofSpecimens</td><td rowspan=1 colspan=1>DiscrepantCo-infections1</td></tr><tr><td rowspan=1 colspan=1>SARS-CoV-2 + Flu A</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>SARS-CoV-2 + Flu B</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>2</td></tr></table>

1 A discrepant co-infection was defined as a specimen that contains at least one pathogen that was detected by Panther Fusion SARS-CoV-2/Flu A/B/RSV assay but not detected by the comparator method.

# Retrospective Clinical Study

Flu B and RSV were of low prevalence and were not encountered in sufficiently large numbers during the prospective clinical study to adequately demonstrate assay performance. To supplement the results of the prospective clinical study, an evaluation of 95 preselected archived retrospective specimens was performed. These specimens were archived NPS in VTM or UTM that were collected between December 2019 and March 2020. Specimens were selected for enrollment in the study based solely on the historic qualitative result. In addition to evaluating Flu B and RSV positive specimens, Flu A positive specimens and negative specimens were included in the study. The 95 samples were distributed uniformly across all three clinical testing sites. A summary of the demographic information of the tested retrospective samples is provided in Table 21 below.

Table 21. Demographic Data for Retrospective Specimens   

<table><tr><td rowspan=1 colspan=2></td><td rowspan=1 colspan=1>Overall</td></tr><tr><td rowspan=3 colspan=1>Sex</td><td rowspan=1 colspan=1>Male</td><td rowspan=1 colspan=1>44 (46.3%)</td></tr><tr><td rowspan=1 colspan=1>Female</td><td rowspan=1 colspan=1>51 (53.7%)</td></tr><tr><td rowspan=1 colspan=1>Unknown</td><td rowspan=1 colspan=1>0 (0.0%)</td></tr><tr><td rowspan=5 colspan=1>Age</td><td rowspan=1 colspan=1>&gt;5 years</td><td rowspan=1 colspan=1>16 (16.8%)</td></tr><tr><td rowspan=1 colspan=1>5-21years</td><td rowspan=1 colspan=1>12 (12.6%)</td></tr><tr><td rowspan=1 colspan=1>22-40years</td><td rowspan=1 colspan=1>15 (15.8%)</td></tr><tr><td rowspan=1 colspan=1>41-60 years</td><td rowspan=1 colspan=1>16 (16.8%)</td></tr><tr><td rowspan=1 colspan=1>&gt;60 years</td><td rowspan=1 colspan=1>36 (37.9%)</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>95</td></tr></table>

The performance of the Panther Fusion SARS-CoV-2/Flu A/B/RSV assay was evaluated by comparing testing results against a U.S. FDA-cleared molecular Flu A/B/RSV assay. The Panther Fusion SARS-CoV-2/Flu A/B/RSV assay retrospective performance data, expressed as positive percent and negative percent agreements against the comparator method, are presented by in Table 22 below. Specimens that obtained discordant results underwent additional testing with a U.S. FDA EUA SARS-CoV-2/Flu A/B/RSV molecular test, when sufficient sample volume remained.

Table 22. Panther Fusion SARS-CoV-2/ Flu A/B/RSV Assay Clinical Performance in Retrospective Specimens   

<table><tr><td rowspan=1 colspan=1>Target</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>TP</td><td rowspan=1 colspan=1>FP</td><td rowspan=1 colspan=1>TN</td><td rowspan=1 colspan=1>FN</td><td rowspan=1 colspan=1>PPA%(95% CI)</td><td rowspan=1 colspan=1>NPA %(95% CI)</td></tr><tr><td rowspan=1 colspan=1>Flu A</td><td rowspan=1 colspan=1>95</td><td rowspan=1 colspan=1>27</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>66</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>93.1% (27/29)(78.0-98.1%)</td><td rowspan=1 colspan=1>100% (66/66)(94.5-100%)</td></tr><tr><td rowspan=1 colspan=1>Flu B</td><td rowspan=1 colspan=1>95</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>73</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>95.5% (21/22)(78.2-99.2%)</td><td rowspan=1 colspan=1>100% (73/73)(95.0-100%)</td></tr><tr><td rowspan=1 colspan=1>RSV</td><td rowspan=1 colspan=1>95</td><td rowspan=1 colspan=1>47</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>48</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100% (47/47)(92.4-100%)</td><td rowspan=1 colspan=1>100% (48/48)(92.6-100%)</td></tr></table>

N-number tested; TP – true positive; FN – false negative; TN – true negative; FP – false positive 1 One (1) specimen with a false negative Flu A result tested positive for Flu A with a U.S. FDA EUA SARS-CoV-2/Flu A/B/RSV molecular test, while one (1) specimen with a false negative Flu A result tested negative for Flu A with a U.S. FDA EUA SARS-CoV-2/Flu A/B/RSV molecular test. 2 One (1) specimen with a false negative Flu B result tested positive for Flu B with a U.S. FDA EUA SARS-CoV-2/Flu A/B/RSV molecular test.

Panther Fusion SARS-CoV-2/Flu A/B/RSV assay reported a total of 2 retrospective specimens with multiple pathogen detections $( 2 . 1 \%$ of all retrospective specimens, 2/95). One co-infection detected Flu A and Flu B, and one specimen detected Flu A with RSV. The comparator result was concordant with the candidate device in detecting the co-infection in both of these specimens (Table 23).

Table 23. Co-infections Detected by the Panther Fusion SARS-CoV-2/ Flu A/B/RSV Assay in Retrospective Specimens   

<table><tr><td rowspan=1 colspan=1>Co-infectionCombination</td><td rowspan=1 colspan=1>Number ofSpecimens</td><td rowspan=1 colspan=1>DiscrepantCo-infections1</td></tr><tr><td rowspan=1 colspan=1>Flu A+ Flu B</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Flu A + RSV</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td></tr></table>

1 A discrepant co-infection was defined as a specimen that contains at least one pathogen that was detected by Panther Fusion SARS-CoV-2/Flu A/B/RSV assay but not detected by the comparator method.

# 2. Clinical Specificity:

# See Section “Clinical Sensitivity” above.

3. Other Clinical Supportive Data (When 1. and 2. Are Not Applicable): Not applicable.

# D Clinical Cut-Off:

Not applicable.

# E Expected Values/Reference Range:

The Panther Fusion SARS-CoV-2/Flu A/B/RSV assay prospective clinical study included a total of 1900 prospectively collected NPS specimens, of which 1887 were evaluable for SARS-CoV-2 and 1837 were evaluable for Flu A, Flu B, and RSV. The number and percentage of cases positive for SARS-CoV-2, influenza A, influenza B, and RSV, as determined by the Panther Fusion SARS-CoV-2/Flu A/B/RSV assay, are presented below, stratified by collection period and site.

Table 24. Panther Fusion SARS-CoV-2/Flu A/B/RSV Expected Values by Specimen Collection Site for Specimens Collected from 11/20 to 3/21   

<table><tr><td rowspan=2 colspan=1>Pathogen</td><td rowspan=1 colspan=2>Overall</td><td rowspan=1 colspan=2>Site 1(New York)</td><td rowspan=1 colspan=2>Site 2(Missouri)</td><td rowspan=1 colspan=2>Site 3(Wisconsin)</td><td rowspan=1 colspan=2>Site 4(California)</td><td rowspan=1 colspan=2>Site 5(Texas)</td></tr><tr><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>Expectedvalue</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>Expectedvalue</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>Expectedvalue</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>Expectedvalue</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>Expectedvalue</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>Expectedvalue</td></tr><tr><td rowspan=1 colspan=1>SARS-CoV2</td><td rowspan=1 colspan=1>891</td><td rowspan=1 colspan=1>30.1%(268/891)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>425</td><td rowspan=1 colspan=1>16.0%(68/425)</td><td rowspan=1 colspan=1>444</td><td rowspan=1 colspan=1>44.6%(198/444)</td><td rowspan=1 colspan=1>22</td><td rowspan=1 colspan=1>9.1%(2/22)</td></tr><tr><td rowspan=1 colspan=1>Influenza A</td><td rowspan=1 colspan=1>831</td><td rowspan=1 colspan=1>0.2%(2/831)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>391</td><td rowspan=1 colspan=1>0.0%(0/391)</td><td rowspan=1 colspan=1>418</td><td rowspan=1 colspan=1>0.5%(2/418)</td><td rowspan=1 colspan=1>22</td><td rowspan=1 colspan=1>0%(0/22)</td></tr><tr><td rowspan=1 colspan=1>Influenza B</td><td rowspan=1 colspan=1>831</td><td rowspan=1 colspan=1>0.1%(1/831)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>391</td><td rowspan=1 colspan=1>0.3%(1/391)</td><td rowspan=1 colspan=1>418</td><td rowspan=1 colspan=1>0%(0/418)</td><td rowspan=1 colspan=1>22</td><td rowspan=1 colspan=1>0%(0/22)</td></tr><tr><td rowspan=1 colspan=1>RSV</td><td rowspan=1 colspan=1>831</td><td rowspan=1 colspan=1>0.0%(0/831)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>391</td><td rowspan=1 colspan=1>0.0%(0/391)</td><td rowspan=1 colspan=1>418</td><td rowspan=1 colspan=1>0%(0/418)</td><td rowspan=1 colspan=1>22</td><td rowspan=1 colspan=1>0%(0/22)</td></tr></table>

Table 25. Panther Fusion SARS-CoV-2/Flu A/B/RSV Expected Values by Specimen Collection Site for Specimens Collected from 01/22 to 03/22   

<table><tr><td rowspan=2 colspan=1>Pathogen</td><td rowspan=1 colspan=2>Overall</td><td rowspan=1 colspan=2>Site 1(New York)</td><td rowspan=1 colspan=2>Site 2(Missouri)</td><td rowspan=1 colspan=2>Site 3(Wisconsin)</td><td rowspan=1 colspan=2>Site 4(California)</td><td rowspan=1 colspan=2>Site 5(Texas)</td></tr><tr><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>Expectedvalue</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>Expectedvalue</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>Expectedvalue</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>Expectedvalue</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>Expectedvalue</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>Expectedvalue</td></tr><tr><td rowspan=1 colspan=1>SARS-CoV2</td><td rowspan=1 colspan=1>996</td><td rowspan=1 colspan=1>13.4%(133/996)</td><td rowspan=1 colspan=1>173</td><td rowspan=1 colspan=1>7.5%(13/173)</td><td rowspan=1 colspan=1>406</td><td rowspan=1 colspan=1>12.1%(49/406)</td><td rowspan=1 colspan=1>417</td><td rowspan=1 colspan=1>17.0%(71/417)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=1>Influenza A</td><td rowspan=1 colspan=1>1006</td><td rowspan=1 colspan=1>12.3%(124/1006)</td><td rowspan=1 colspan=1>181</td><td rowspan=1 colspan=1>2.8%(5/181)</td><td rowspan=1 colspan=1>409</td><td rowspan=1 colspan=1>15.2%(62/409)</td><td rowspan=1 colspan=1>416</td><td rowspan=1 colspan=1>13.7%(57/416)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=1>Influenza B</td><td rowspan=1 colspan=1>1006</td><td rowspan=1 colspan=1>0.3%(3/1006)</td><td rowspan=1 colspan=1>181</td><td rowspan=1 colspan=1>0.0%(0/181)</td><td rowspan=1 colspan=1>409</td><td rowspan=1 colspan=1>0.2%(1/409)</td><td rowspan=1 colspan=1>416</td><td rowspan=1 colspan=1>0.5%(2/416)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=1>RSV</td><td rowspan=1 colspan=1>1006</td><td rowspan=1 colspan=1>1.1%(11/1006)</td><td rowspan=1 colspan=1>181</td><td rowspan=1 colspan=1>3.3%(6/181)</td><td rowspan=1 colspan=1>409</td><td rowspan=1 colspan=1>1.2%(5/409)</td><td rowspan=1 colspan=1>416</td><td rowspan=1 colspan=1>0.0%(0/416)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>NA</td></tr></table>

NA-not applicable

# F Other Supportive Instrument Performance Characteristics Data: Not applicable.

# VIII Proposed Labeling:

The labeling supports or the finding of substantial equivalence for this device.

# IX Conclusion:

The submitted information in this premarket notification is complete and supports a substantial equivalence decision.