# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATION DECISION SUMMARY

I Background Information:

# B Applicant

Roche Molecular Systems, Inc.

# C Proprietary and Established Names

cobas SARS-CoV-2 & Influenza A/B for use on the cobas Liat System

# D Regulatory Information

<table><tr><td rowspan=1 colspan=1>ProductCode(s)</td><td rowspan=1 colspan=1>Classification</td><td rowspan=1 colspan=1>RegulationSection</td><td rowspan=1 colspan=1>Panel</td></tr><tr><td rowspan=1 colspan=1>QOF</td><td rowspan=1 colspan=1>Class II</td><td rowspan=1 colspan=1>21 CFR 866.3981 - DeviceTo Detect And IdentifyNucleic Acid Targets InRespiratory SpecimensFrom Microbial AgentsThat Cause The SARS-Cov-2 RespiratoryInfection And OtherMicrobial Agents When InA Multi-Target Test</td><td rowspan=1 colspan=1>MI - Microbiology</td></tr></table>

# II Submission/Device Overview:

# A Purpose for Submission:

The purpose of this submission is to show that the cobas SARS-CoV-2 & Influenza $\mathrm { A } / \mathrm { B }$ for use on the Liat System is substantially equivalent to the BioFire Respiratory Panel 2.1 (RP2.1) (DEN200031) and to obtain clearance for the cobas SARS-CoV-2 & Influenza A/B for use on the Liat System.

# B Measurand:

The cobas SARS-CoV-2 & Influenza A/B Assay targets both the $\mathrm { O R F 1 a / b }$ non-structural region and nucleocapsid protein gene that are unique to SARS-CoV-2, a well-conserved region of the

matrix gene of influenza A, and the non-structural protein gene of influenza B isolated from nasopharyngeal swab and nasal swab specimens from patients with signs and symptoms of respiratory tract infection.

# C Type of Test:

This assay is a multiplex nucleic acid assay for the qualitative detection and differentiation of SARS-CoV-2, influenza A and influenza B RNA through nucleic acid extraction, amplification, and detection using real-time RT-PCR. All steps of the assay are automated within the cobas Liat System, after scanning the specimen ID barcode, scanning the assay tube barcode, and the manual addition of sample into the assay tube.

# III Intended Use/Indications for Use:

A Intended Use(s): See Indications for Use below.

# B Indication(s) for Use:

The cobas SARS-CoV-2 & Influenza A/B nucleic acid test for use on the cobas Liat System (cobas SARS-CoV-2 & Influenza A/B) is an automated rapid multiplex real-time, reverse transcriptase polymerase chain reaction (RT-PCR) test intended for the simultaneous qualitative detection and differentiation of severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), influenza A, and/or influenza B virus nucleic acid in nasopharyngeal swab (NPS) and anterior nasal swab (ANS) specimens from individuals with signs and symptoms of respiratory tract infection. Clinical signs and symptoms of respiratory tract infection due to SARS-CoV-2 and influenza can be similar.

cobas SARS-CoV-2 & Influenza A/B is intended for use as an aid in the differential diagnosis of SARS-CoV-2, influenza A, and/or influenza B infection if used in conjunction with other clinical and epidemiological information, and laboratory findings. SARS-CoV-2, influenza A, and influenza B viral nucleic acid are generally detectable in NPS and ANS specimens during the acute phase of infection.

Positive results do not rule out co-infection with other organisms. The agent(s) detected by the cobas SARS-CoV-2 & Influenza A/B may not be the definitive cause of disease.

Negative results do not preclude SARS-CoV-2, influenza A, and/or influenza B infection. The results of this test should not be used as the sole basis for diagnosis, treatment, or other patient management decisions.

C Special Conditions for Use Statement(s): Rx - For Prescription Use Only

For in vitro diagnostic use only

D Special Instrument Requirements: For use with the cobas Liat System, only

# A Device Description:

# Overview:

The cobas Liat SARS-CoV-2 & Influenza A/B Nucleic Acid Test for Use on the cobas Liat System (cobas Liat SARS-CoV-2 & Influenza A/B) is a rapid, automated in vitro diagnostic test for the qualitative detection of SARS-CoV-2, influenza A, and influenza B RNA in nasopharyngeal swab (NPS) and nasal swab (NS) specimens eluted in viral transport media.

The assay targets both the ORF1 a/b non-structural region and nucleocapsid protein gene that are unique to SARS-CoV-2, a well-conserved region of the matrix gene of influenza A, and the nonstructural protein gene of influenza B. An Internal Process Control (IPC) is also included. The IPC is present to control for adequate processing of the target viruses and to monitor the presence of inhibitors in the sample preparation and RT-PCR.

The assay utilizes a single-use disposable cobas assay tube that holds the sample purification and PCR reagents and hosts the sample preparation and PCR processes. The cobas assay tube uses a flexible tube as a sample vessel. It contains all required unit dose reagents pre-packed in tube segments, separated by peelable seals, in the order of reagent use.

The cobas SARS-CoV-2 & Influenza A/B assay uses silica magnetic particle-based nucleic acid extraction and TaqMan probe-based real-time PCR amplification and detection. The cobas Liat Analyzer automates and integrates sample purification, nucleic acid amplification, and detection of the target sequence in biological samples. During the testing process, multiple sample processing actuators of the cobas Liat Analyzer compress the cobas Liat Tube to selectively release reagents from tube segments, move the sample from one segment to another, and control reaction volume, temperature, and incubation time. The cobas Liat Analyzer software controls and coordinates these actions to perform all required assay processes, including sample preparation, nucleic acid extraction, target enrichment, inhibitor removal, nucleic acid elution, and real-time PCR. All assay steps are performed within the closed and self-contained cobas SARS-CoV-2 & Influenza A/B assay tube.

The cobas SARS-CoV-2 & Influenza A/B for use on the Liat System contains the same sample preparation and PCR reaction chemistry as the previously cleared cobas Influenza A/B & RSV for use on the Liat System (K153544). Minor changes were made to the cleared assay that included removing the RSV primers and probes and adding the primers and probes for SARSCoV-2, as well as developing the cobas SARS-CoV-2 Positive Control.

# B Principle of Operation:

The cobas SARS-CoV-2 & Influenza A/B assay is performed on the cobas Liat Analyzer which automates and integrates sample purification, nucleic acid amplification, and detection of the target sequence in biological samples using real-time RT-PCR assays. The assay targets both the ORF1 a/b non-structural region and nucleocapsid protein gene that are unique to SARS-CoV-2, a well-conserved region of the matrix gene of influenza A, and the non-structural protein gene of influenza B. An Internal Process Control (IPC) is also included. The IPC is present to control for adequate processing of the target virus through steps of sample purification, nucleic acid amplification, and to monitor the presence of inhibitors in the RT-PCR processes.

# C Instrument Description Information:

1. Instrument Name: Cobas Liat Analyzer, including cobas Liat System software (core) version 3.3 or higher

2. Specimen Identification: Specimen identification is either entered manually or via barcode

3. Specimen Sampling and Handling:

Nasopharyngeal swab (NPS) or nasal swab (NS) specimens collected in $3 ~ \mathrm { m L }$ viral transport media or $0 . 9 \%$ physiological saline

4. Calibration:

The analyzer performs self-diagnostics during startup (initialization) and utilizes an advanced error diagnostics system to monitor the analyzer’s performance during an assay. Under normal operation, the analyzer alerts the operator if a malfunction or error is detected. The analyzer requires no adjustment or calibration from the operator.

5. Quality Control:

The assay contains an internal process control (IPC) comprised of a bacteriophage MS2 that is included in the Liat Assay Tube. The IPC is used to monitor specimen processing, amplification, detection and to monitor the presence of inhibitors in the RT-PCR reactions.

External Positive and Negative controls are included with cobas SARS-CoV-2 & Influenza A/B Quality Control Kit. The cobas SARS-CoV-2 Positive Control is a plasmid containing SARS-CoV-2 sequences in a buffer solution, that is used to rehydrate the cobas Influenza A/B Positive control lyophilized beads. The cobas SARS-CoV-2 & Influenza A/B Quality control kit also include cobas Dilution UTM as the negative control. External Controls are run during the Add cobas Liat SARS-CoV-2 & Influenza A/B Tube Lot procedure.

Additional External Controls should be tested in accordance with local, state, federal and/or accrediting organization requirements as applicable. The external controls were validated in the analytical, clinical, and flex studies.

# V Substantial Equivalence Information:

A Predicate Device Name(s): BioFire Respiratory Panel 2.1 (RP2.1)

B Predicate 510(k) Number(s): DEN200031

C Comparison with Predicate(s):

<table><tr><td colspan="1" rowspan="1">Device &amp; Predicate Device(s):</td><td colspan="1" rowspan="1">K223591</td><td colspan="1" rowspan="1">DEN200031</td></tr><tr><td colspan="1" rowspan="1">Device Trade Name</td><td colspan="1" rowspan="1">Cobas SARS-CoV-2 &amp;Influenza A/B for use on the</td><td colspan="1" rowspan="1">BioFire RP2.1 Panel</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">cobas Liat System</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Regulation Number/Name</td><td colspan="1" rowspan="1">Same</td><td colspan="1" rowspan="1">21 CFR 866.3981; Multi-Target Respiratory SpecimenNucleic Acid Test IncludingSARS-CoV-2 And OtherMicrobial Agents</td></tr><tr><td colspan="1" rowspan="1">Product Code(s)</td><td colspan="1" rowspan="1">Same</td><td colspan="1" rowspan="1">QOF</td></tr><tr><td colspan="1" rowspan="1">Prescription Use Only</td><td colspan="1" rowspan="1">Same</td><td colspan="1" rowspan="1">Yes</td></tr><tr><td colspan="1" rowspan="1">Intended Use/Indications For Use</td><td colspan="1" rowspan="1">The cobas SARS-CoV-2 &amp;Influenza A/B nucleic acidtest for use on the cobas LiatSystem (cobas SARS-CoV-2&amp; Influenza A/B) is anautomated rapid multiplexreal-time, reversetranscriptase polymerasechain reaction (RT-PCR) testintended for thesimultaneous qualitativedetection and differentiationof severe acute respiratorysyndrome coronavirus 2(SARS-CoV-2), influenza A,and/or influenza B virusnucleic acid innasopharyngeal swab (NPS)and anterior nasal swab(ANS) specimens fromindividuals with signs andsymptoms of respiratorytract infection. Clinical signsand symptoms of respiratorytract infection due to SARS-CoV-2 and influenza can besimilar.cobas SARS-CoV-2 &amp;Influenza A/B is intended foruse as an aid in thedifferential diagnosis ofSARS-CoV-2, influenza A,and/or influenza B infectionif used in conjunction withother clinical andepidemiological information,and laboratory findings.SARS-CoV-2, influenza A,and influenza B viral nucleicacid are generally detectablein NPS and ANS specimensduring the acute phase ofinfection.Positive results do not ruleout co-infection with otherorganisms. The agent(s)detected by the cobas SARS-CoV-2 &amp; Influenza A/B may</td><td colspan="1" rowspan="1">The BioFire RespiratoryPanel 2.1 (RP2.1) is a PCR-based multiplexed nucleicacid test intended for usewith the BioFire FilmArray2.0 or BioFire FilmArrayTorch systems for thesimultaneous qualitativedetection and identificationof multiple respiratory viraland bacterial nucleic acids innasopharyngeal swabs (NPS)obtained from individualssuspected of respiratory tractinfections, includingCOVID-19.The following organismtypes and subtypes areidentified using the BioFireRP2.1:Adenovirus,Coronavirus 229E,Coronavirus HKU1,Coronavirus NL63,Coronavirus OC43,Severe Acute RespiratorySyndrome Coronavirus(SARS-CoV-2),Human Metapneumovirus,HumanRhinovirus/Enterovirus,Influenza A, includingsubtypes H1, H1-2009, andH3,Influenza B,Parainfluenza Virus 1,Parainfluenza Virus 2,Parainfluenza Virus 3,Parainfluenza Virus 4,Respiratory SyncytialVirus,Bordetella parapertussis(IS1001),Bordetella pertussis (ptxP),</td></tr></table>

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>not be the definitive cause ofdisease.Negative results do notpreclude SARS-CoV-2,influenza A, and/or influenzaB infection. The results ofthis test should not be usedas the sole basis fordiagnosis, treatment, or otherpatient managementdecisions.</td><td rowspan=1 colspan=1>Chlamydia pneumoniae,andMycoplasma pneumoniaeNucleic acids from therespiratory viral andbacterial organismsidentified by this test aregenerally detectable in NPSspecimens during the acutephase of infection. Thedetection and identificationof specific viral and bacterialnucleic acids fromindividuals exhibiting signsand/or symptoms ofrespiratory infection isindicative of the presence ofthe identified microorganismand aids in the diagnosis ofrespiratory infection if usedin conjunction with otherclinical and epidemiologicalinformation. The results ofthis test should not be usedas the sole basis fordiagnosis, treatment, or otherpatient managementdecisions.Negative results in thesetting of a respiratoryillness may be due toinfection with pathogens thatare not detected by this test,or lower respiratory tractinfection that may not bedetected by an NPSspecimen. Positive results donot rule out coinfection withother organisms. Theagent(s) detected by theBioFire RP2.1 may not bethe definite cause of disease.Additional laboratory testing(e.g. bacterial and viralculture,immunofluorescence, andradiography) may benecessary when evaluating apatient with possiblerespiratory tract infection.</td><td></td></tr><tr><td rowspan=1 colspan=1>Sample Type(s)</td><td rowspan=1 colspan=1>Nasopharyngeal swabs,Nasal swabs</td><td rowspan=1 colspan=1>Nasopharyngeal swabs</td><td></td></tr><tr><td rowspan=1 colspan=1>Analyte Targets</td><td rowspan=1 colspan=1>SARS-CoV-2 ORF1 a/bnon-structural regionSARS-CoV-2nucleocapsid protein gene</td><td rowspan=1 colspan=1>For SARS-CoV-2 organismsspike protein (S) gene and•membrane protein (M)gene</td><td rowspan=1 colspan=1>]</td></tr></table>

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Influenza A matrix geneInfluenza B nonstructuralprotein gene</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Ancillary Collection Kits</td><td rowspan=1 colspan=1>Copan FLOQSwabs withUTM, UVT and otherswabs with other viraltransport media (VTM) -e.g., M4RT, M4, M5 andM60.9% Saline</td><td rowspan=1 colspan=1>Viral Transport Media(VTM)Saline (0.9%)</td></tr><tr><td rowspan=1 colspan=1>Sample Preparation</td><td rowspan=1 colspan=1>Same</td><td rowspan=1 colspan=1>Automated</td></tr><tr><td rowspan=1 colspan=1>Amplification Technology</td><td rowspan=1 colspan=1>Real-Time PCR</td><td rowspan=1 colspan=1>2 stage PCR</td></tr><tr><td rowspan=1 colspan=1>Detection Chemistry</td><td rowspan=1 colspan=1>Multiplex assay usingdifferent reporter dyes fortarget and control</td><td rowspan=1 colspan=1>Two Step Nested multiplexPCR:Reverse transcription,followed by a multiplexedfirst stage PCR reaction(PCR1).Multiple simultaneoussecond-stage PCRreactions (PCR2) toamplify sequences withinthe PCR1 products usingfluorescence doublestranded binding dye.Endpoint melting curvedata to detect target-specific amplicons</td></tr><tr><td rowspan=1 colspan=1>Controls Used</td><td rowspan=1 colspan=1>Same</td><td rowspan=1 colspan=1>Internal and Externalcontrols</td></tr><tr><td rowspan=1 colspan=1>Results Analysis</td><td rowspan=1 colspan=1>PCR Cycle thresholdanalysis</td><td rowspan=1 colspan=1>Endpoint melting curve datato detect target-specificamplicons</td></tr></table>

# VI Standards/Guidance Documents Referenced:

Class II Special Controls as per 21 CFR 866.3981   
Guidance for Industry and Food and Drug Administration Staff, Recommended Content   
and Format of Non-Clinical Bench Performance Testing Information in Premarket   
Submissions (December 20, 2019)   
Guidance for Industry and FDA Staff, Format for Traditional and Abbreviated 510(k)s   
(September 13, 2019)   
Guidance for Industry and Food and Drug Administration Staff, Refuse to Accept Policy   
for 510(k)s (April 21, 2022)   
Guidance for Industry and Food and Drug Administration Staff, The 510(k) Program:   
Evaluating Substantial Equivalence in Premarket Notifications [510(k)] (July 28, 2014)   
Guidance for Industry and FDA Staff, Establishing the Performance Characteristics of In   
Vitro Diagnostic Devices for the Detection or Detection and Differentiation of Influenza   
Viruses (July 7, 2011)

Guidance for Industry and FDA Staff, Highly Multiplexed Microbiological/ Medical Countermeasure In Vitro Nucleic Acie Based Diagnostic Devices (August 27, 2014) Recommendations for Clinical Laboratory Improvement Amendments of 1988 (CLIA) Waiver Applications for Manufacturers of In Vitro Diagnostic Devices - Guidance for Industry and Food and Drug Administration Staff (fda.gov)

VII Performance Characteristics:

# A Analytical Performance:

1. Precision/Reproducibility:

A reproducibility study was conducted assessing the total variability of the cobas Liat SARS$\mathrm { C o V } { - } 2 \ \&$ Influenza $\mathrm { A } / \mathrm { B }$ assay across operators, study sites, testing days, cobas Liat Analyzers, and cobas Liat assay tube lots. The cobas Liat SARS-CoV-2 & Influenza A/B assay was evaluated at three CLIA waived sites. Two (2) operators at each of the three sites tested a 3-member reproducibility panel in triplicate on five different days across 3 reagent lots, for a total of ${ \sim } 8 1 0$ tests, ${ \sim } 2 7 0$ tests/panel member (3 sites $\times 3$ lots/site $\times 5$ day/lot $\times 2$ operators/day $\times 3$ panel members/operator $\times 3$ replicates/panel member). Each site utilized a minimum of three Liat Analyzers. The reproducibility panel contained a true negative; a low positive and a moderate positive member co-formulated with SARS-CoV-2, influenza A, and influenza B.

The reproducibility panel samples were prepared by spiking SARS-CoV-2 (USA-WA1/2020, catalog number 0810587CFHI, ZeptoMetrix, NY, USA), influenza A virus (Brisbane/59/07- catalog number 0810244CF; ZeptoMetrix, NY, USA) and influenza B virus (Florida/04/06- catalog number 0810255CF; ZeptoMetrix, NY, USA) of known titer into negative simulated clinical matrix (Equivalency between NPS matrix, NS matrix and simulated clinical matrix was demonstrated in a matrix equivalency study, refer to section VII.B.2.) The moderate positive and low positive concentrations used for each of the strains corresponded to $3 \mathrm { x } . 5 \mathrm { x }$ LoD and 1x-2x LoD, respectively. The true negative sample was comprised of negative simulated clinical matrix.

Three (3) CLIA waived sites and six operators (two operators per site) participated in this reproducibility study. The six operators consisted of two Medical Assistants, a Point-of-Care Coordinator, a Research assistant, an Administrative assistant (billing), and a Lab Technician (institutional title only, no laboratory training) with no formal medical laboratory training. All operators had limited or no training or hands-on experience in conducting laboratory testing when the study initiated.

The six operators at the three sites tested the members of the reproducibility panel in triplicate on five non-consecutive days. Three (3) cobas Liat Analyzers were used at each site for a total of nine cobas Liat Analyzers. Each site also used approximately equal amounts of three different lots of cobas Liat SARS-CoV-2 & Influenza A/B assay tubes.

The reproducibility study results are shown in Table 1 and Table 2, respectively.

Table 1. Reproducibility Study- Qualitative Results

<table><tr><td rowspan=2 colspan=1>Target</td><td rowspan=2 colspan=1>Panel Conc.</td><td rowspan=1 colspan=4>% Agreement with Expected Results/(n Agreement/N Tested)(95% CI)</td></tr><tr><td rowspan=1 colspan=1>Site 1</td><td rowspan=1 colspan=1>Site 2</td><td rowspan=1 colspan=1>Site 3</td><td rowspan=1 colspan=1>Overall</td></tr><tr><td rowspan=3 colspan=1>SARS-CoV-2</td><td rowspan=1 colspan=1>Low Positive(1x-2x LoD)</td><td rowspan=1 colspan=1>100%(90/90)(95.9-100)</td><td rowspan=1 colspan=1>98.9%(89/90)(93.4-99.8)</td><td rowspan=1 colspan=1>97.6%(81/83)(91.6-99.3)</td><td rowspan=1 colspan=1>98.9%(260/263)(96.7-99.6)</td></tr><tr><td rowspan=1 colspan=1>Mod.Positive(3x-5x LoD)</td><td rowspan=1 colspan=1>98.9%(88/89)(93.9-99.8)</td><td rowspan=1 colspan=1>100%(89/89)(95.9-100)</td><td rowspan=1 colspan=1>100%(90/90)(95.9-100)</td><td rowspan=1 colspan=1>99.6%(267/268)(97.9-99.9)</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>100%(89/89)(95.9-100)</td><td rowspan=1 colspan=1>100%(90/90)(95.9-100)</td><td rowspan=1 colspan=1>100%(87/87)(95.8-100)</td><td rowspan=1 colspan=1>100%(266/266)(98.6-100)</td></tr><tr><td rowspan=3 colspan=1>Flu A</td><td rowspan=1 colspan=1>Low positive(1x-2x LoD)</td><td rowspan=1 colspan=1>100%(90/90)(95.9-100)</td><td rowspan=1 colspan=1>95.6%(86/90)(89.1-98.3)</td><td rowspan=1 colspan=1>100%(83/83)(95.6-100)</td><td rowspan=1 colspan=1>98.5%(259/263)(96.2-99.4)</td></tr><tr><td rowspan=1 colspan=1>Mod.Positive(3x-5x LoD)</td><td rowspan=1 colspan=1>100%(89/89)(95.9-100)</td><td rowspan=1 colspan=1>100%(89/89)(95.9-100)</td><td rowspan=1 colspan=1>100%(90/90)(95.9-100)</td><td rowspan=1 colspan=1>100%(268/268)(98.6-100)</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>100%(89/89)(95.9-100)</td><td rowspan=1 colspan=1>100%(90/90)(95.9-100)</td><td rowspan=1 colspan=1>100%(87/87)(95.8-100)</td><td rowspan=1 colspan=1>100%(266/266)(98.6-100)</td></tr><tr><td rowspan=3 colspan=1>Flu B</td><td rowspan=1 colspan=1>Low positive(1x-2x LoD)</td><td rowspan=1 colspan=1>100%(90/90)(95.9-100)</td><td rowspan=1 colspan=1>100%(90/90)(95.9-100)</td><td rowspan=1 colspan=1>100%(83/83)(95.6-100)</td><td rowspan=1 colspan=1>100%(263/263)(98.6-100)</td></tr><tr><td rowspan=1 colspan=1>Mod.Positive(3x- 5x LoD)</td><td rowspan=1 colspan=1>98.9%(88/89)(93.4-99.8)</td><td rowspan=1 colspan=1>100%(89/89)(95.9-100)</td><td rowspan=1 colspan=1>100%(90/90)(95.9-100)</td><td rowspan=1 colspan=1>99.6%(267/268)(97.9-100)</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>100%(89/89)(95.9-100)</td><td rowspan=1 colspan=1>100%(90/90)(95.9-100)</td><td rowspan=1 colspan=1>100%(87/87)(95.8-100)</td><td rowspan=1 colspan=1>100%(266/266)(98.6-100)</td></tr></table>

Mod $=$ moderate, Conc $\equiv$ Concentration; Note: Results are shown only for the intended targets. Panel members were all co-spiked with all targets, so results are presented three times

The cobas Liat SARS-CoV-2 & Influenza $\mathrm { A } / \mathrm { B }$ assay demonstrated $100 \%$ agreement for Flu A, and $9 9 . 6 \%$ agreement for SARS-CoV-2 and Flu B moderate positive panel members. For low positive panel members, the assay yielded $100 \%$ agreement for Flu B, and $9 8 . 9 \%$ and $9 8 . 5 \%$ for SARS-CoV-2 and Flu A, respectively (Table 1 above). A lower agreement for low positive panel members was expected, since the analyte concentration of their panel member analytes ranged between 1x and $2 \mathbf { x }$ the limit of detection, which is expected to yield $59 5 \%$ detection rate. This performance is acceptable and demonstrates acceptable assay reproducibility.

Table 2: Reproducibility Study- Ct Variability Analysis Results   

<table><tr><td colspan="1" rowspan="2">ViralTarget</td><td colspan="1" rowspan="2">PanelMemberConc</td><td colspan="1" rowspan="2">n/Na</td><td colspan="1" rowspan="2">MeanCtb</td><td colspan="2" rowspan="1">BetweenSites</td><td colspan="2" rowspan="1">BetweenLot</td><td colspan="2" rowspan="1">Between Day</td><td colspan="2" rowspan="1">BetweenOperator</td><td colspan="2" rowspan="1">Within-Run(Residual)</td><td colspan="2" rowspan="1">Total</td></tr><tr><td colspan="1" rowspan="1">SD</td><td colspan="1" rowspan="1">CV%</td><td colspan="1" rowspan="1">SD</td><td colspan="1" rowspan="1">CV%</td><td colspan="1" rowspan="1">SD</td><td colspan="1" rowspan="1">CV%</td><td colspan="1" rowspan="1">SD</td><td colspan="1" rowspan="1">CV%</td><td colspan="1" rowspan="1">SD</td><td colspan="1" rowspan="1">CV%</td><td colspan="1" rowspan="1">SDb</td><td colspan="1" rowspan="1">CV%c</td></tr><tr><td colspan="1" rowspan="1">SARS-CoV-2</td><td colspan="1" rowspan="1">1x-2xLOD</td><td colspan="1" rowspan="1">260/263</td><td colspan="1" rowspan="1">33.3</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">0.36</td><td colspan="1" rowspan="1">1.1</td><td colspan="1" rowspan="1">0.29</td><td colspan="1" rowspan="1">0.9</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">1.08</td><td colspan="1" rowspan="1">3.3</td><td colspan="1" rowspan="1">1.18</td><td colspan="1" rowspan="1">3.5</td></tr><tr><td colspan="1" rowspan="1">SARS-CoV-2</td><td colspan="1" rowspan="1">3x-5xLOD</td><td colspan="1" rowspan="1">267/268</td><td colspan="1" rowspan="1">32.1</td><td colspan="1" rowspan="1">0.31</td><td colspan="1" rowspan="1">1.0</td><td colspan="1" rowspan="1">0.46</td><td colspan="1" rowspan="1">1.4</td><td colspan="1" rowspan="1">0.29</td><td colspan="1" rowspan="1">0.9</td><td colspan="1" rowspan="1">0.07</td><td colspan="1" rowspan="1">0.2</td><td colspan="1" rowspan="1">0.74</td><td colspan="1" rowspan="1">2.3</td><td colspan="1" rowspan="1">0.97</td><td colspan="1" rowspan="1">3.0</td></tr><tr><td colspan="1" rowspan="1">Influenza A</td><td colspan="1" rowspan="1">1x-2xLOD</td><td colspan="1" rowspan="1">259/263</td><td colspan="1" rowspan="1">33.0</td><td colspan="1" rowspan="1">0.07</td><td colspan="1" rowspan="1">0.2</td><td colspan="1" rowspan="1">0.49</td><td colspan="1" rowspan="1">1.5</td><td colspan="1" rowspan="1">0.19</td><td colspan="1" rowspan="1">0.6</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">0.81</td><td colspan="1" rowspan="1">2.5</td><td colspan="1" rowspan="1">0.97</td><td colspan="1" rowspan="1">2.9</td></tr><tr><td colspan="1" rowspan="1">Influenza A</td><td colspan="1" rowspan="1">3x-5xLOD</td><td colspan="1" rowspan="1">268/268</td><td colspan="1" rowspan="1">31.9</td><td colspan="1" rowspan="1">0.26</td><td colspan="1" rowspan="1">0.8</td><td colspan="1" rowspan="1">0.44</td><td colspan="1" rowspan="1">1.4</td><td colspan="1" rowspan="1">0.23</td><td colspan="1" rowspan="1">0.7</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">0.56</td><td colspan="1" rowspan="1">1.7</td><td colspan="1" rowspan="1">0.79</td><td colspan="1" rowspan="1">2.5</td></tr><tr><td colspan="1" rowspan="1">Influenza B</td><td colspan="1" rowspan="1">1x-2xLOD</td><td colspan="1" rowspan="1">263/263</td><td colspan="1" rowspan="1">30.2</td><td colspan="1" rowspan="1">0.15</td><td colspan="1" rowspan="1">0.5</td><td colspan="1" rowspan="1">0.38</td><td colspan="1" rowspan="1">1.3</td><td colspan="1" rowspan="1">0.50</td><td colspan="1" rowspan="1">1.6</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">0.66</td><td colspan="1" rowspan="1">2.2</td><td colspan="1" rowspan="1">0.92</td><td colspan="1" rowspan="1">3.1</td></tr><tr><td colspan="1" rowspan="1">Influenza B</td><td colspan="1" rowspan="1">3x-5xLOD</td><td colspan="1" rowspan="1">267/268</td><td colspan="1" rowspan="1">29.3</td><td colspan="1" rowspan="1">0.09</td><td colspan="1" rowspan="1">0.3</td><td colspan="1" rowspan="1">0.29</td><td colspan="1" rowspan="1">1.0</td><td colspan="1" rowspan="1">0.29</td><td colspan="1" rowspan="1">1.0</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">0.96</td><td colspan="1" rowspan="1">3.3</td><td colspan="1" rowspan="1">1.05</td><td colspan="1" rowspan="1">3.6</td></tr></table>

Ct: cycle threshold, $\mathrm { C V } \%$ : percent coefficient of variation, LOD: limit of detection, SARS-CoV-2: severe acute respiratory syndrome coronavirus 2, SD: standard deviation. $\mathrm { ^ a n }$ is the number of positive tests, which contribute $\mathrm { C t }$ values to the analysis. N is the total number of valid tests for the panel member. b The Mean and total SD were estimated using the PROC MIXED procedure. c Total $\mathrm { C V } ( \% ) = ( \mathrm { S D } / \mathrm { M e a n } ) ^ { \ast } 1 0 0 ,$ .

The total $\mathrm { C t }$ variability, as measured by the standard deviation, was less than or equal to 1.08 across all target viruses and concentrations. For all positive panel members, the within-run factor (i.e., random error) was the largest contributor to total variability. These results indicate that the reproducibility of the cobas SARS-CoV-2 & Influenza A/B assay on the Liat system is robust in NPS samples.

# 2. Linearity:

Not applicable; this is a qualitative assay.

3. Analytical Specificity/Interference:

Analytical Reactivity (Inclusivity)

# a. In silico

The inclusivity of the cobas Liat SARS-CoV-2 & Influenza $\mathrm { A } / \mathrm { B }$ assay was evaluated using in silico analysis of the forward primers, reverse primers, and probes for the SARS-CoV-2, Flu A and Flu B target systems in relation to sequences available in the NCBI and GISAID gene databases. For SARS-CoV-2, all available sequences up to March 15, 2023, from the GISAID $( > 1 4 . 3$ million sequences) and NCBI ${ \ } ^ { \prime > 6 . 5 }$ million sequences) gene databases were evaluated. For Flu A and Flu B, available sequences from GISAID gene databases as of January 26, 2023, were evaluated. In total, ${ > } 2 0$ million SARS-CoV-2 and approximatley180 thousand Flu A sequences and approximately 38 thousand H1N1pdm09 sequences were assessed. The SARS-CoV-2 sequences included the following lineages and variants of concern (VOC) or variants of interest (VOI) that may have important epidemiological, immunological, or pathogenic properties from a public health perspective: Delta, Alpha, Omicron BA.1, Omicron BA.2, Omicron BA.4, Omicron BA.5, Omicron BQ.1, Omicron BQ.1.1, Omicron BF.7, Gamma, Epsilon, Iota, Beta, Mu, Zeta, Kappa, Eta, Lamba, and Theta.

Based on the in silico analysis of GISAID and NCBI sequences available up to March 15, 2023 for SARS-CoV-2, only two sequences in GISAID $( 0 . 0 0 0 0 1 \% )$ have the changes that were predicted to affect detection and performance of the test, and they were deposited prior to July 2022 and were likely sequencing errors due to low sequence quality. Therefore, the cobas Liat SARS-CoV-2 & Influenza $\mathrm { A } / \mathrm { B }$ assay is predicted to detect all 20,972,883 SARS-CoV-2 sequences evaluated.

Based on in silico analysis of all sequences available from January 26, 2023, to in GISAID database, the cobas Liat SARS-CoV-2 & Influenza A/B assay is predicted to detect all the current circulating Flu A and Flu B strains (including UK H1N1 mutations and the WHO recommended vaccine strains for 2023-2024).

# $b$ . Wet-Testing

This study was performed to determine the analytical reactivity of the cobas Liat SARSCoV-2 & Influenza $\mathrm { A } / \mathrm { B }$ assay with clinically relevant strains of SARS-CoV-2. The inclusivity panel was prepared by spiking various inactivated SARS-CoV-2 viral strains into pooled negative clinical matrix at concentration at, below, and above LoD and testing in triplicate. The strains evaluated and the lowest concentration that achieved $100 \%$ reactivity are shown in Table 3, below.

Table 3: Summary of SARS-CoV-2 Inclusivity Testing   

<table><tr><td rowspan=1 colspan=1>Isolate/Variant Name</td><td rowspan=1 colspan=1>PangoLineage</td><td rowspan=1 colspan=1>WHOLabel</td><td rowspan=1 colspan=1>TestConcentration(copies/mL)</td><td rowspan=1 colspan=1>SARS-CoV-2</td><td rowspan=1 colspan=1>InfluenzaA</td><td rowspan=1 colspan=1>InfluenzaB</td></tr><tr><td rowspan=1 colspan=1>SARS-CoV-2 Italy-INMI1</td><td rowspan=1 colspan=1>not listed</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>2.0E+01</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>SARS-CoV-2 HongKong/VM20001061/2020</td><td rowspan=1 colspan=1>A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>2.0E+01</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>.</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>SARS-CoV-2England/204820464/2020</td><td rowspan=1 colspan=1>B.1.1.7</td><td rowspan=1 colspan=1>Alpha</td><td rowspan=1 colspan=1>5.0E+00</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>.</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>SARS-CoV-2 SouthAfrica/KRISP-K005325/2020</td><td rowspan=1 colspan=1>B.1.351</td><td rowspan=1 colspan=1>Beta</td><td rowspan=1 colspan=1>2.0E+01</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>USA/COR-22-063113/2022</td><td rowspan=1 colspan=1>BA5.5</td><td rowspan=1 colspan=1>Omicron</td><td rowspan=1 colspan=1>6.00E+00</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>USA/GA-EHC-2811C/2021</td><td rowspan=1 colspan=1>BA.1</td><td rowspan=1 colspan=1>Omicron</td><td rowspan=1 colspan=1>1.50E+00</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>hCoV-19/USA/MD-HP40900/2022</td><td rowspan=1 colspan=1>B.1.1.529,XBB.1.5</td><td rowspan=1 colspan=1>Omicron</td><td rowspan=1 colspan=1>6.00E+00</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>hCoV-19/USA/MD-HP38861/2022</td><td rowspan=1 colspan=1>B.1.1.529,BQ.1.1</td><td rowspan=1 colspan=1>Omicron</td><td rowspan=1 colspan=1>1.20E+01</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>hCoV-19/USA/MD-HP38288/2022</td><td rowspan=1 colspan=1>B.1.1.529,BF.7</td><td rowspan=1 colspan=1>Omicron</td><td rowspan=1 colspan=1>1.20E+01</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>hCoV-19/USA/MD-HP30386/2022</td><td rowspan=1 colspan=1>B.1.1.529,BA.4</td><td rowspan=1 colspan=1>Omicron</td><td rowspan=1 colspan=1>6.00E+00</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>USA/MD-HP24556/2022</td><td rowspan=1 colspan=1>BA.2.3</td><td rowspan=1 colspan=1>Omicron</td><td rowspan=1 colspan=1>1.20E+01</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>USA/MD-HP20874/2021</td><td rowspan=1 colspan=1>B.1.1.529</td><td rowspan=1 colspan=1>Omicron</td><td rowspan=1 colspan=1>6.00E+00</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>hCoV-19/USA/CA-Stanford-15 S02/2021</td><td rowspan=1 colspan=1>B.1.617.1</td><td rowspan=1 colspan=1>Kappa</td><td rowspan=1 colspan=1>1.20E+01</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>USA/NY-Wadsworth-21025952/2021</td><td rowspan=1 colspan=1>B.1.526</td><td rowspan=1 colspan=1>Iota</td><td rowspan=1 colspan=1>3.60E+01</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>hCoV-19/USA/PHC658/2021</td><td rowspan=1 colspan=1>B.1.617.2</td><td rowspan=1 colspan=1>Delta</td><td rowspan=1 colspan=1>1.20E+01</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>hCoV-19/Japan/TY7-503/2021</td><td rowspan=1 colspan=1>P.1</td><td rowspan=1 colspan=1>Gamma</td><td rowspan=1 colspan=1>1.20E+01</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr></table>

The results from this study demonstrate that the cobas Liat SARS-CoV-2 & Influenza A/B assay can detect multiple clinically relevant strains of SARS-CoV-2.

Influenza A and influenza B oligonucleotide sets used in the cobas Liat SARS-CoV-2 & Influenza $\mathrm { A } / \mathrm { B }$ are the same sets used in the FDA cleared cobas Liat Influenza A/B & RSV, where 28 influenza A and 15 influenza B strains were tested and shown to be

inclusive. The tested influenza A strains included 14 influenza A/H1 strains (including 3 H1N1 pdm09 strains), 12 influenza A/H3 strains (including 1 H3N2 strain), 1 influenza A/H7N9 strain and 1 influenza A/H5N1 reassortant strain. The tested influenza B strains included both the Victoria lineage and Yamagata lineage (refer to the package insert for the cobas Liat Influenza A/B & RSV and the Decision Summary for K153544).

Cross-Reactivity/Microbial Interference

# Wet- Testing

This study evaluated the analytical specificity (cross-reactivity) and sensitivity of the cobas SARS-CoV-2 & Influenza A/B for use on the cobas Liat System in the presence of non-targeted microorganisms that may be found in a respiratory tract clinical specimen. Forty-two (42) non-target microorganisms (Table 4) were evaluated in the study. Panel members were composed of one non-target microorganism spiked into negative clinical NP swab matrix at approximately $1 . 0 \mathrm { E } { + 0 5 }$ units/mL for viruses and approximately $1 . 0 \mathrm { E } { + 0 6 }$ unit/mL for bacteria and fungi (Units/mL: $\mathrm { E B / m L }$ , CFU/mL, IU/mL, $\mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ , particles/mL, copies $\mathrm { \Delta } / \mathrm { m L }$ , or $\mathrm { P F U / m L }$ ), except for Pneumocystis jirovecii, which was tested at $5 . 0 \mathrm { E } { + 0 3 }$ units $/ \mathrm { m L }$ and was a positive clinical sample diluted 1:10 in negative NP clinical matrix. To evaluate cross-reactivity, each panel was tested once in the absence of the target organisms. To evaluate microbial interference (negative effect on sensitivity), each panel was tested once in the presence of SARS-CoV-2 (USAWA1/2020), Flu A H1N1 (Brisbane 59/07), and Flu B (Florida/04/06) at 3x LoD. No cross-reactivity or microbial interference was observed except for SARS-CoV-1, which at a concentration of $1 \mathrm { x } 1 0 ^ { 5 } \mathrm { P F U / m L }$ inhibited detection of SARS-CoV-2 (at 3x LoD). SARS-CoV-1 no longer inhibited detection of SARS-CoV-2 when SARS-CoV-1 was diluted and tested at a concentration of $1 \mathrm { x } 1 0 ^ { 4 } \ : \mathrm { P F U / m L }$ .

Table 4: Organisms Evaluated for Cross-Reactivity and Microbial Interference   

<table><tr><td colspan="1" rowspan="1">Organism</td><td colspan="1" rowspan="1">Conc.*</td><td colspan="1" rowspan="1">Organism</td><td colspan="1" rowspan="1">Conc.*</td></tr><tr><td colspan="2" rowspan="1">Viruses</td><td colspan="2" rowspan="1">Bacteria</td></tr><tr><td colspan="1" rowspan="1">Adenovirus</td><td colspan="1" rowspan="1">1x105</td><td colspan="1" rowspan="1">Bordetella pertussis</td><td colspan="1" rowspan="1">1x106</td></tr><tr><td colspan="1" rowspan="1">Cytomegalovirus</td><td colspan="1" rowspan="1">1x105</td><td colspan="1" rowspan="1">Bordetella parapertussis</td><td colspan="1" rowspan="1">1x106</td></tr><tr><td colspan="1" rowspan="1">Epstein-Barr virus</td><td colspan="1" rowspan="1">1x105</td><td colspan="1" rowspan="1">Chlamydia pneumoniae</td><td colspan="1" rowspan="1">1x106</td></tr><tr><td colspan="1" rowspan="1">Human Enterovirus D</td><td colspan="1" rowspan="1">1x105</td><td colspan="1" rowspan="1">Corynebacterium flavescens</td><td colspan="1" rowspan="1">1x106</td></tr><tr><td colspan="1" rowspan="1">Human Rhinovirus B</td><td colspan="1" rowspan="1">1x105</td><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">1x106</td></tr><tr><td colspan="1" rowspan="1">Human coronavirus 229E</td><td colspan="1" rowspan="1">1x105</td><td colspan="1" rowspan="1">Fusobacterium necrophorumsubsp. necrophorum</td><td colspan="1" rowspan="1">1x106</td></tr><tr><td colspan="1" rowspan="1">Human coronavirus HKU1</td><td colspan="1" rowspan="1">1x105</td><td colspan="1" rowspan="1">Haemophilus influenzae</td><td colspan="1" rowspan="1">1x106</td></tr><tr><td colspan="1" rowspan="1">Human coronavirus NL63</td><td colspan="1" rowspan="1">1x105</td><td colspan="1" rowspan="1">Lactobacillus crispatus</td><td colspan="1" rowspan="1">1x106</td></tr><tr><td colspan="1" rowspan="1">Human Metapneumovirus 27</td><td colspan="1" rowspan="1">1x105</td><td colspan="1" rowspan="1">Legionella pneumophila</td><td colspan="1" rowspan="1">1x106</td></tr><tr><td colspan="1" rowspan="1">MERS Coronavirus</td><td colspan="1" rowspan="1">1x105</td><td colspan="1" rowspan="1">Moraxella catarrhalis</td><td colspan="1" rowspan="1">1x106</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza Virus Type 1</td><td colspan="1" rowspan="1">1x105</td><td colspan="1" rowspan="1">Mycobacterium tuberculosis</td><td colspan="1" rowspan="1">1x106</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza Virus Type 2</td><td colspan="1" rowspan="1">1x105</td><td colspan="1" rowspan="1">Mycoplasma genitalium</td><td colspan="1" rowspan="1">1x106</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza Virus Type 3</td><td colspan="1" rowspan="1">1x105</td><td colspan="1" rowspan="1">Mycoplasma pneumoniae</td><td colspan="1" rowspan="1">1x106</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza Virus Type 4 (4A)</td><td colspan="1" rowspan="1">1x105</td><td colspan="1" rowspan="1">Neisseria flava</td><td colspan="1" rowspan="1">1x106</td></tr><tr><td colspan="1" rowspan="1">Respiratory Syncytial Virus (Strain A2)</td><td colspan="1" rowspan="1">1x10^}$</td><td colspan="1" rowspan="1">Neisseria meningitidis</td><td colspan="1" rowspan="1">1x106</td></tr><tr><td colspan="1" rowspan="2">SARS Coronavirus, Urbani (SARS-CoV-1)</td><td colspan="1" rowspan="2">1x105 and1x104</td><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">1x106</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus epidermidis</td><td colspan="1" rowspan="1">1x106</td></tr><tr><td colspan="1" rowspan="1">Measles</td><td colspan="1" rowspan="1">1x105</td><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">1x106</td></tr><tr><td colspan="1" rowspan="1">Mumps</td><td colspan="1" rowspan="1">1x105</td><td colspan="1" rowspan="1">Streptococcus pneumoniae</td><td colspan="1" rowspan="1">1x106</td></tr><tr><td colspan="2" rowspan="1">Fungi</td><td colspan="1" rowspan="1">Streptococcus pyogenes</td><td colspan="1" rowspan="1">1x106</td></tr><tr><td colspan="1" rowspan="1">Aspergillus Flavus var. flavus</td><td colspan="1" rowspan="1">1x106</td><td colspan="1" rowspan="1">Streptococcus salivarius</td><td colspan="1" rowspan="1">1x106</td></tr><tr><td>Candida albicans</td><td>1x106</td><td colspan="2">Nasal Wash</td><td>1:10 dilution</td></tr><tr><td>Pneumocystis jirovecii</td><td>5x103</td><td>Nasal Wash specimen</td><td>of patient sample</td><td rowspan="3"></td></tr><tr><td>Pneumocystis jirovecii clinical sample</td><td>1:10 dilution of patient sample</td><td></td><td></td></tr></table>

\*EB/mL, CFU/mL, $\mathrm { I U / m L }$ , $\mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ , particles/mL, copies/mL, or PFU/mL

# Interfering Substances

This study evaluated the performance of the cobas Liat SARS-CoV-2 & Influenza A/B test in the presence of medications, over the counter products, and other potentially interfering substances found in a clinical respiratory specimen. Assay results were evaluated to determine if the presence of potentially interfering substances in target analyte negative or target analyte positive samples had an effect on assay performance. The analyte negative pools contained negative NPS in UTM clinical matrix and the potentially interfering substances, only. The analyte positive pools contained negative NPS in UTM clinical matrix, the potentially interfering substance(s), and one representative strain of each targeted analyte (i.e., SARS-CoV-2, Flu A and Flu B) spiked to a final testing concentration of $3 \mathbf { x }$ LoD. Three replicates were tested per pool both in the presence and absence of the target analytes. The substances evaluated are denoted in Table 5 below. All target-spiked panels were $100 \%$ positive, and all target-negative panels were $100 \%$ negative, indicating that none of the evaluated substances interfered with the cobas Liat SARS-CoV-2 & Influenza A/B test.

Table 5: Endogenous and Exogenous Substances Evaluated   

<table><tr><td rowspan=1 colspan=1>Potential Interferent</td><td rowspan=1 colspan=1>Active Ingredient</td><td rowspan=1 colspan=1>Concentration1</td></tr><tr><td rowspan=1 colspan=1>Mucin: bovine submaxillary gland, type I-S</td><td rowspan=1 colspan=1>Purified mucin protein</td><td rowspan=1 colspan=1>5 mg/mL</td></tr><tr><td rowspan=1 colspan=1>Blood</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>5% (v/v)</td></tr><tr><td rowspan=1 colspan=1>Peripheral blood mononuclear cell (PBMC)</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>1.0E+06 cell/mL</td></tr><tr><td rowspan=1 colspan=1>Nasal spray - Afrin / Anefrin</td><td rowspan=1 colspan=1>Oxymetazoline</td><td rowspan=1 colspan=1>5% (v/v)</td></tr><tr><td rowspan=1 colspan=1>Nasal corticosteroids - Flonase</td><td rowspan=1 colspan=1>Fluticasone</td><td rowspan=1 colspan=1>5% (v/v)</td></tr><tr><td rowspan=1 colspan=1>Nasal gel - Zicam</td><td rowspan=1 colspan=1>Galphimia glauca, Histaminumhydrochloricum, Luffa operculata, Sulphur</td><td rowspan=1 colspan=1>5% (v/v)</td></tr><tr><td rowspan=1 colspan=1>Throat lozenges, oral anesthetic and analgesic- Cepacol</td><td rowspan=1 colspan=1>Benzocaine, Menthol</td><td rowspan=1 colspan=1>5 mg/mL</td></tr><tr><td rowspan=1 colspan=1>Antibiotic, nasal ointment - Bactroban</td><td rowspan=1 colspan=1>Mupirocin</td><td rowspan=1 colspan=1>5 mg/mL</td></tr><tr><td rowspan=1 colspan=1>Antiviral drug - Relenza</td><td rowspan=1 colspan=1>Zanamivir</td><td rowspan=1 colspan=1>5 mg/mL</td></tr><tr><td rowspan=1 colspan=1>Antiviral drug - Tamiflu</td><td rowspan=1 colspan=1>Oseltamivir</td><td rowspan=1 colspan=1>7.5 mg/mL</td></tr><tr><td rowspan=1 colspan=1>Antimicrobial, systemic</td><td rowspan=1 colspan=1>Tobramycin</td><td rowspan=1 colspan=1>4 μg/mL</td></tr></table>

1 v/v: volume by volume

# Competitive Interference

# a. Traditional Competitive Inhibition

The purpose of this study was to demonstrate that samples tested with the cobas SARS-CoV-2 & Influenza A/B test that are co-infected with multiple types of targeted microorganisms do not inhibit the detection of either one (competitive interference). One representative strains of SARS-CoV-2 (US-WA1/2020), one representative strain of influenza A (H1N1 Brisbane/59/07) and two representative strains of influenza B (B/Florida/04/06 and Colorado/06/2017) were evaluated. Traditional co-infection

panels were made by spiking one target organism at low concentration $( { \sim } 3 \mathrm { x } \ L \mathrm { o D } )$ in negative clinical NP swab UTM matrix, while the high concentration target virus was progressively decreased until the 3x LoD low targets achieved a $100 \%$ hit rate. The on-target analyte combinations evaluated and the results of the traditional competitive inhibition study are shown in Table 6.

Table 6: Traditional Competitive Interference Study Sample Panel Composition & Study Results   

<table><tr><td rowspan=1 colspan=3>Target 1 (High)</td><td rowspan=1 colspan=2>Target 2 (3x LoD)</td><td rowspan=1 colspan=2>Target 3 (3x LoD)</td><td rowspan=1 colspan=3>% Detected (Hit Rate)</td></tr><tr><td rowspan=1 colspan=1>ViralStrain</td><td rowspan=1 colspan=1>Conc.ID50/mL</td><td rowspan=1 colspan=1>Ct</td><td rowspan=1 colspan=1>ViralStrain</td><td rowspan=1 colspan=1>Conc.ID50/mL</td><td rowspan=1 colspan=1>ViralStrain</td><td rowspan=1 colspan=1>Conc.ID50/mL</td><td rowspan=1 colspan=1>FluA</td><td rowspan=1 colspan=1>FluB</td><td rowspan=1 colspan=1>SARS-CoV-2</td></tr><tr><td rowspan=1 colspan=1>Flu A(H1N1</td><td rowspan=1 colspan=1>1.40E+04(14,000,000x LoD)</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>Flu B(Florida)</td><td rowspan=1 colspan=1>1.20E-02</td><td rowspan=1 colspan=1>SARS-CoV-2</td><td rowspan=1 colspan=1>3.60E-02</td><td rowspan=1 colspan=1>100%(3/3)</td><td rowspan=1 colspan=1>100%(3/3)</td><td rowspan=1 colspan=1>100%(3/3)</td></tr><tr><td rowspan=1 colspan=1>Flu B(Florida)</td><td rowspan=1 colspan=1>3.20E+02(80,000x LoD)</td><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>Flu A(H1N1)</td><td rowspan=1 colspan=1>3.00E-03</td><td rowspan=1 colspan=1>SARS-CoV-2</td><td rowspan=1 colspan=1>3.60E-02</td><td rowspan=1 colspan=1>66%(2/3)</td><td rowspan=1 colspan=1>100%(3/3)</td><td rowspan=1 colspan=1>0%(0/3)</td></tr><tr><td rowspan=1 colspan=1>Flu B(Florida)</td><td rowspan=1 colspan=1>1.60E+02(40,000x LoD)</td><td rowspan=1 colspan=1>18</td><td rowspan=1 colspan=1>Flu A(H1N1)</td><td rowspan=1 colspan=1>3.00E-03</td><td rowspan=1 colspan=1>SARS-CoV-2</td><td rowspan=1 colspan=1>3.60E-02</td><td rowspan=1 colspan=1>100%(3/3)</td><td rowspan=1 colspan=1>100%(3/3)</td><td rowspan=1 colspan=1>0%(0/3)</td></tr><tr><td rowspan=1 colspan=1>Flu B(Florida)</td><td rowspan=1 colspan=1>4.00E+01(10,000x LoD)</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>Flu A(H1N1)</td><td rowspan=1 colspan=1>3.00E-03</td><td rowspan=1 colspan=1>SARS-CoV-2</td><td rowspan=1 colspan=1>3.60E-02</td><td rowspan=1 colspan=1>100%(3/3)</td><td rowspan=1 colspan=1>100%(3/3)</td><td rowspan=1 colspan=1>66%(2/3)</td></tr><tr><td rowspan=1 colspan=1>Flu B(Florida)</td><td rowspan=1 colspan=1>2.00E+01(5,000x LoD)</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>Flu A(H1N1)</td><td rowspan=1 colspan=1>3.00E-03</td><td rowspan=1 colspan=1>SARS-CoV-2</td><td rowspan=1 colspan=1>3.60E-02</td><td rowspan=1 colspan=1>100%(3/3)</td><td rowspan=1 colspan=1>100%(3/3)</td><td rowspan=1 colspan=1>100%(3/3)</td></tr><tr><td rowspan=1 colspan=1>Flu B*(Colorado)</td><td rowspan=1 colspan=1>1.40E+04</td><td rowspan=1 colspan=1>19</td><td rowspan=1 colspan=1>Flu A(H1N1)</td><td rowspan=1 colspan=1>NT</td><td rowspan=1 colspan=1>SARS-CoV-2</td><td rowspan=1 colspan=1>3.60E-02</td><td rowspan=1 colspan=1>NT</td><td rowspan=1 colspan=1>100%(3/3)</td><td rowspan=1 colspan=1>33%(1/3)</td></tr><tr><td rowspan=1 colspan=1>Flu B(Colorado)</td><td rowspan=1 colspan=1>7.00E+03</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>Flu A(H1N1)</td><td rowspan=1 colspan=1>NT</td><td rowspan=1 colspan=1>SARS-CoV-2</td><td rowspan=1 colspan=1>3.60E-02</td><td rowspan=1 colspan=1>NT</td><td rowspan=1 colspan=1>100%(3/3)</td><td rowspan=1 colspan=1>100%(3/3)</td></tr><tr><td rowspan=1 colspan=1>SARS-CoV-2</td><td rowspan=1 colspan=1>4.80E+01(4,000x LoD)</td><td rowspan=1 colspan=1>23</td><td rowspan=1 colspan=1>Flu A(H1N1)</td><td rowspan=1 colspan=1>3.00E-03</td><td rowspan=1 colspan=1>Flu B(Florida)</td><td rowspan=1 colspan=1>1.20E-02</td><td rowspan=1 colspan=1>66%(2/3)</td><td rowspan=1 colspan=1>66%(2/3)</td><td rowspan=1 colspan=1>100%(3/3)</td></tr><tr><td rowspan=1 colspan=1>SARS-CoV-2</td><td rowspan=1 colspan=1>3.60E+01(3,000x LoD)</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>Flu A(H1N1)</td><td rowspan=1 colspan=1>3.00E-03</td><td rowspan=1 colspan=1>Flu B(Florida)</td><td rowspan=1 colspan=1>1.20E-02</td><td rowspan=1 colspan=1>100%(3/3)</td><td rowspan=1 colspan=1>100%(3/3)</td><td rowspan=1 colspan=1>100%(3/3)</td></tr></table>

$\mathrm { N T } =$ Not tested, Conc. $\equiv$ Concentration \* LoD was established with influenza B/Colorado/06/2017, therefore relative LoD concentration in unknown

Interference was observed for the following co-infection conditions:

SARS-CoV-2 $( 3 \mathrm { x } \mathrm { L o D } )$ in the presence of Flu B(Florida) (high concentrations of   
$3 . 2 0 \mathrm { E } { + 0 2 }$ , $1 . 6 0 \mathrm { E } { + } 0 2$ , and $4 . 0 0 \mathrm { E } { + } 0 1$ $\mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ ). Interference was no longer observed at a Flu B (Florida) concentration of $2 . 0 0 \mathrm { E } { + } 0 1$ $\mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ .   
SARS-CoV-2 (3x LoD) in the presence of Flu B(Colorado) (high concentration of $1 . 4 0 \mathrm { E } { + } 0 4 \mathrm { T C I D } _ { 5 0 } / \mathrm { m L } $ . Interference was no longer observed at a Flu B (Colorado) concentration of $7 . 0 0 \mathrm { E } { + } 0 3 \mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ .   
Flu A $( 3 \mathrm { x } \ \mathrm { L o D } )$ in the presence of Flu B(Florida) (high concentration of $3 . 2 0 \mathrm { E } { + } 0 2 $ $\mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ ). Interference was no longer observed at a Flu B (Florida) concentration of $1 . 6 0 \mathrm { E } { + 0 2 } \ : \mathrm { T C I D } 5 0 \mathrm { / m L }$ .   
Flu A and Flu B $\left( 3 \mathbf { x } \ L \mathbf { o } \mathbf { D } \right)$ in the presence of SARS-CoV-2 (high concentration of $4 . 8 0 \mathrm { E } { + } 0 1 $ $\mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ ). Interference was no longer observed at a SARS-CoV-2   
concentration of $3 . 6 0 \mathrm { E } { + } 0 1$ $\mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ .

# b. Reverse Competitive Inhibition

A reverse competitive inhibition study was performed using the same viral strains utilized for the traditional study, with two lots of SARS-CoV-2 (US-WA1/2020). Co-infection panels were formulated where high concentrations of influenza B or SARS-CoV-2 were fixed at concentrations that produced Ct values of 15 and 16, respectively, which are frequently observed in positive clinical specimens. The low concentration target virus was progressively increased until it could be detected with a $100 \%$ hit rate. The on-target analyte combinations evaluated and the results of the reverse competitive inhibition study are shown in Table 7.

Table 7: Reverse Competitive Interference Study Sample Panel Composition & Study Results   

<table><tr><td rowspan=1 colspan=3>Target 1 (High)</td><td rowspan=1 colspan=2>Target 2</td><td rowspan=1 colspan=2>Target 3</td><td rowspan=1 colspan=3>% Detected (Hit Rate)</td></tr><tr><td rowspan=1 colspan=1>ViralStrain</td><td rowspan=1 colspan=1>Conc.ID50/mL</td><td rowspan=1 colspan=1>Ct</td><td rowspan=1 colspan=1>ViralStrain</td><td rowspan=1 colspan=1>Conc.D50/mL</td><td rowspan=1 colspan=1>ViralStrain</td><td rowspan=1 colspan=1>Conc.ID50/mL</td><td rowspan=1 colspan=1>FluA</td><td rowspan=1 colspan=1>FluB</td><td rowspan=1 colspan=1>SARS-CoV-2</td></tr><tr><td rowspan=1 colspan=1>Flu B(Florida)</td><td rowspan=1 colspan=1>1.00E+03(250,000x LoD)</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>Flu A(H1N1)</td><td rowspan=4 colspan=1>NT</td><td rowspan=1 colspan=1>SARS-CoV-2</td><td rowspan=1 colspan=1>4.50E-01(38x LoD)</td><td rowspan=1 colspan=1>NT</td><td rowspan=1 colspan=1>100%(3/3)</td><td rowspan=1 colspan=1>100%(3/3)</td></tr><tr><td rowspan=1 colspan=1>Flu B(Colorado)</td><td rowspan=1 colspan=1>3.20E+05</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>Flu A(H1N1)</td><td rowspan=1 colspan=1>SARS-CoV-2</td><td rowspan=1 colspan=1>4.50E-01(38x LoD)</td><td rowspan=1 colspan=1>NT</td><td rowspan=1 colspan=1>100%(3/3)</td><td rowspan=1 colspan=1>100%(3/3)</td></tr><tr><td rowspan=1 colspan=1>SARS-CoV-2</td><td rowspan=1 colspan=1>8.50E+03(&gt;700,00x LoD)</td><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>Flu A(H1N1)</td><td rowspan=1 colspan=1>Flu B(Florida)</td><td rowspan=1 colspan=1>3.00E+00(750x LoD)</td><td rowspan=1 colspan=1>NT</td><td rowspan=1 colspan=1>100%(3/3)</td><td rowspan=1 colspan=1>100%(3/3)</td></tr><tr><td rowspan=1 colspan=1>SARS-CoV-2</td><td rowspan=1 colspan=1>5.00E+03(&gt;400,000x LoD)</td><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>Flu A(H1N1)</td><td rowspan=1 colspan=1>Flu B(Florida)</td><td rowspan=1 colspan=1>3.20E+00(800x LoD)</td><td rowspan=1 colspan=1>NT</td><td rowspan=1 colspan=1>100%(3/3)</td><td rowspan=1 colspan=1>100%(3/3)</td></tr><tr><td rowspan=1 colspan=1>SARS-CoV-2</td><td rowspan=1 colspan=1>8.50E+03(&gt;700,000x LoD)</td><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>Flu A(H1N1)</td><td rowspan=1 colspan=1>8.00E-01(800x LoD)</td><td rowspan=1 colspan=1>Flu B(Florida)</td><td rowspan=2 colspan=1>NT</td><td rowspan=1 colspan=1>100%(3/3)</td><td rowspan=1 colspan=1>NT</td><td rowspan=1 colspan=1>100%(3/3)</td></tr><tr><td rowspan=1 colspan=1>SARS-CoV-2</td><td rowspan=1 colspan=1>5.00E+03(&gt;400,000x LoD)</td><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>Flu A(H1N1)</td><td rowspan=1 colspan=1>8.00E-01(800x LoD)</td><td rowspan=1 colspan=1>Flu B(Florida)</td><td rowspan=1 colspan=1>100%(3/3)</td><td rowspan=1 colspan=1>NT</td><td rowspan=1 colspan=1>100%(3/3)</td></tr></table>

Conc. $\underline { { \underline { { \mathbf { \Pi } } } } } =$ concentration, $\mathrm { N T } =$ Not tested

The results of the reverse competitive inhibition study showed that in the presence of these high target concentrations of influenza B, the detection of SARS-CoV-2 virus was achieved at 4.50E-01 $\mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ ; the impact on influenza A virus detection was not evaluated. In the presence of high target concentrations of SARS-CoV-2, the detection of influenza A and influenza B viruses was achieved at 8.00E-01 $\mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ and between $3 . 0 0 \mathrm { E } + 0 0 - 3 . 2 0 \mathrm { E } + 0 0 \mathrm { T C I D } _ { 5 0 } / \mathrm { m I }$ , respectively.

4. Assay Reportable Range: Not applicable; this is a qualitative assay.

5. Traceability, Stability, Expected Values (Controls, Calibrators, or Methods):

# a. Controls

The assay contains an Internal Process Control (IPC) added to each test specimen and external positive and negative controls. For more information, see Section IV.C.5. Quality Control, above.

# $b$ . Sample Stability

Stability studies have been performed to support the following specimen stability claims:

Primary NPS or NS swab specimens collected in transport media (UTM, VTM, M4, M4RT, M5, and M6) may be stored up to 4 hours at room temperature (15- $3 0 ^ { \circ } \mathrm { C } )$ , up to 72 hours refrigerated $\left( 2 \mathrm { - } 8 ^ { \circ } \mathrm { C } \right)$ , or frozen at $\mathrm { \leq } \mathrm { - } 7 0 ^ { \circ } \mathrm { C }$ if not tested within 72 hours of collection.   
Primary NPS or NS swabs specimens collected in $0 . 9 \%$ physiological saline may be stored up to 4 hours at room temperature $( 1 5 { - } 3 0 ^ { \circ } \mathrm { C } )$ or up to 72 hours   
refrigerated $\left( 2 \mathrm { - } 8 ^ { \circ } \mathrm { C } \right)$ .

Specimens transferred into the cobas SARS-CoV-2 & Influenza A/B Assy tube may be stored up to 4 hours at room temperature $( 1 5 . 3 0 ^ { \circ } \mathrm { C } )$ .

# c. Kit Stability

Stability of the cobas Liat SARS-CoV-2 & Influenza A/B assay tubes and cobas Liat SARS-CoV-2 Positive Control were evaluated in real time. Stability of cobas Liat Flu A/B Positive Control and cobas Liat Dilution UTM (negative control) were not assessed, as stability for these reagents has been previously established for cobas Liat Influenza A/B & RSV (K153544). The study data supports the claimed storage conditions for the cobas Liat SARS-CoV-2 & Influenza A/B assay tubes and cobas Liat SARS-Cov-2 Positive Control.

# Shipping Stability

The purpose of this study was to demonstrate that exposure to extreme hot or cold temperatures potentially encountered by the cobas SARS-CoV-2 & Influenza $\mathrm { A } / \mathrm { B }$ assay tubes during shipment would not impact the performance of the assay. Three pilot lots of assay tube for the cobas SARS-CoV-2 & Influenza A/B assay were tested in this study.

The cobas Liat SARS-CoV-2 & Influenza A/B assay tubes were stored at conditions representing the upper and lower limits of Category F classified kits (i.e., $3 7 ^ { \circ } \mathrm { C }$ and $- 2 0 ^ { \circ } \mathrm { C } )$ ). Assay tubes were stored at $3 7 ^ { \circ } \mathrm { C }$ for 14 days and $- 2 0 ^ { \circ } \mathrm { C }$ for 7 days. Five positive replicates per lot were prepared with cobas SARS-CoV-2 Positive Control and cobas Influenza $\mathrm { A } / \mathrm { B }$ Positive control and tested along with one replicate of cobas Liat Dilution UTM (negative control), for a total of 18 tests performed for each time point.

All runs tested gave the expected results for all positive and negative replicates. This study demonstrates that cobas SARS-CoV-2 & Influenza $\mathrm { A } / \mathrm { B }$ assay tubes are not altered by exposure to extreme hot or cold temperatures that may be encountered during shipping.

6. Detection Limit:

# a. SARS-CoV-2

To determine the LoD for the SARS-CoV-2 targets, heat inactivated cultured SARS-CoV-2 virus (USA-WA1/2020, catalog number 0810587CFHI, ZeptoMetrix, NY, USA) was serially diluted in pooled negative clinical nasopharyngeal swab matrix (NNPS). Five concentration levels of a 2-fold serial dilution series, were tested with 20 replicates per concentration except for the highest concentration level, where only 10 replicates were tested. Three Pilot lots of Assay tubes (approximately equal number of replicates per lot), 20 cobas Liat analyzers (4-5 runs per analyzer) and two independent dilution series (equal number of replicates per dilution series) were used in the study. The LoD, or the lowest concentration which yielded a positive rate of $2 9 5 \%$ , was determined to be 0.012 $\mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ (12 copies $\mathrm { \Delta } m \mathrm { \Delta } L$ ). Of note, only one LoD is reported for the SARS-CoV-2 N gene and RdRp targets, as both are labeled with the same fluorophore and therefore indistinguishable from one another. The results of the SARS-CoV-2 LoD study are summarized in Table 9.

Table 9. SARS-CoV-2 LoD Summary Data

<table><tr><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>ConcentrationTCID50/mL</td><td rowspan=1 colspan=1>Concentrationcopies/mL</td><td rowspan=1 colspan=1>ReplicatesTested</td><td rowspan=1 colspan=1>Detection Rate (#Pos/Total)</td><td rowspan=1 colspan=1>MeanCt*</td></tr><tr><td rowspan=5 colspan=1>USA-WA1/2020</td><td rowspan=1 colspan=1>0.048</td><td rowspan=1 colspan=1>49</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>100% (10/10)</td><td rowspan=1 colspan=1>32.6</td></tr><tr><td rowspan=1 colspan=1>0.024</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>100% (20/20)</td><td rowspan=1 colspan=1>33.5</td></tr><tr><td rowspan=1 colspan=1>0.012</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>100% (20/20)</td><td rowspan=1 colspan=1>35.2</td></tr><tr><td rowspan=1 colspan=1>0.006</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>70% (14/20)</td><td rowspan=1 colspan=1>35.7</td></tr><tr><td rowspan=1 colspan=1>0.003</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>25% (5/20)</td><td rowspan=1 colspan=1>36.7</td></tr></table>

\* Mean Ct values were calculated from positive samples only Note: Concentration of viral stock in copies $/ \mathrm { m L }$ was quantified using reverse transcriptase digital PCR with target specific PCR primers and probe sets designed to amplify SARS-CoV-2

# $b$ . WHO International Standard for SARS-CoV-2

The LoD using WHO International Standard for SARS-CoV-2 RNA (NIBSC code: 20/146) was determined by reconstituting the WHO Standard to $0 . 5 ~ \mathrm { m L }$ according to the WHO NIBSC code: 20/146 Instructions for use (Version 1.0, Dated 14-Dec-2020). Following reconstitution, the WHO Standard was diluted to an intermediate stock (IS) concentration in UTM.

WHO International Standard was serially diluted in pooled negative nasopharyngeal swabs matrix. Five concentration levels were tested with 24 replicates at each level across three lots of assay tubes (8 replicates per lot). Three independent dilution series were used in the study with an approximately equal numbers of replicates per dilution series. The LoD, or the lowest concentration which yielded a positive rate of $2 9 5 \%$ , was determined to be $6 2 . 5 \ \mathrm { I U / m L }$ .

The results of the detection rate and LoD are shown in Table 10 below.

Table 10. WHO International Standard for SARS-CoV-2 RNA LoD Results   

<table><tr><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>ConcentrationIU/mL</td><td rowspan=1 colspan=1>ReplicatesTested</td><td rowspan=1 colspan=1>Detection Rate(#Pos/Total)</td><td rowspan=1 colspan=1>MeanCt*</td></tr><tr><td rowspan=5 colspan=1>WHO International Standard forSARS-CoV-2 RNA(NIBSC code: 20/146)</td><td rowspan=1 colspan=1>125</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>100% (24/24)</td><td rowspan=1 colspan=1>32.1</td></tr><tr><td rowspan=1 colspan=1>62.5</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>100% (24/24)</td><td rowspan=1 colspan=1>33.2</td></tr><tr><td rowspan=1 colspan=1>31.25</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>71% (17/24)</td><td rowspan=1 colspan=1>34.5</td></tr><tr><td rowspan=1 colspan=1>15.625</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>50% (12/24)</td><td rowspan=1 colspan=1>35.4</td></tr><tr><td rowspan=1 colspan=1>7.8125</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>42% (10/24)</td><td rowspan=1 colspan=1>35.2</td></tr></table>

\* Mean Ct values were calculated from positive samples only

# c. Influenza A/B

The detection limit for influenza A and B was established in the original cleared cobas Influenza A/B & RSV Test for use with the Liat System (K153544). The Limit of Detection (LoD) was evaluated using 3 strains of influenza A and 2 strains of influenza B. The LoD was determined by limiting dilution studies using these titered viruses. The viruses were spiked into negative nasopharyngeal swab (NPS) in UTM sample matrix. The LoD was determined to be $2 { \times } 1 0 ^ { - 3 } - 2 { \times } \mathrm { i } 0 ^ { - 2 }$ $\Gamma \mathrm { C I D } _ { 5 0 } / \mathrm { m L }$ for influenza A strains, and $2 { \times } 1 0 ^ { - 3 } - 4 { \times } 1 0 ^ { - 3 }$ $\mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ for influenza B strains. The LoD results for influenza A and B are summarized in Table 11.

Table 11. LoD Determination for Influenza A and Influenza B Strains (K153544)   

<table><tr><td rowspan=1 colspan=1>Virus Strain</td><td rowspan=1 colspan=1>LoD (TCID50/mL)</td></tr><tr><td rowspan=1 colspan=1>A/Brisbane/10/07</td><td rowspan=1 colspan=1>2.0 × 10-2</td></tr><tr><td rowspan=1 colspan=1>A/Brisbane/59/07</td><td rowspan=1 colspan=1>2.0 × 10-3</td></tr><tr><td rowspan=1 colspan=1>A/NY/01/2009</td><td rowspan=1 colspan=1>2.0 × 10-2</td></tr><tr><td rowspan=1 colspan=1>B/Florida/04/06</td><td rowspan=1 colspan=1>2.0 × 10-3</td></tr><tr><td rowspan=1 colspan=1>B/Malaysia/2506/04</td><td rowspan=1 colspan=1>4.0 × 10-3</td></tr></table>

The LoD for co-analyte spiked samples with cultured influenza A virus strain Brisbane/59/07 (catalog number 0810244CF; ZeptoMetrix, NY, USA), influenza B virus strain Florida/04/06 (catalog number 0810255CF; ZeptoMetrix, NY, USA) and SARS-CoV-2 (USA-WA1/2020, catalog number 0810587CFHI, ZeptoMetrix, NY, USA) was also evaluated with cobas SARS-CoV-2 & Influenza A/B for the Liat System and shown to be equivalent to single analyte spiked samples.

# 7. Assay Cut-Off:

A target result call is determined individually for each target (SARS-CoV-2, Influenza A, Influenza B) channel and the internal process control channel based on the combination of: Ct, Amplitude (AMP), and other validity features. A final result is determined for the targets based on the combination of the target result call, internal process control result call, curve validity (including Ct and Amplitude cutoffs), and the sample input volume. The sample input volume is determined to be low when sample input is less than the defined minimum volume requirements. Only the SARS-CoV-2 target channel uses Amp as a part of the result evaluation to allow a better margin for specificity.

8. Accuracy (Instrument): Not Applicable

9. Carry-Over:

A carry-over study was conducted for the original cleared cobas Influenza A/B & RSV test for the Liat System (K153544) and demonstrated that there was no carry-over or cross contamination observed. The cobas SARS-CoV-2 & Influenza A/B for the Liat System instrumentation and workflow are identical to the cobas Influenza A/B & RSV for the Liat System, therefore additional carry-over studies are not required.

# B Comparison Studies:

1. Method Comparison with Predicate Device: Not applicable.

# 2. Matrix Comparison:

Some analytical studies were conducted using simulated matrix, therefore equivalency between simulated and real clinical matrix was evaluated. The simulated matrix consisted of $2 . 5 ~ \mathrm { m g / m L }$ mucin and 250,000 HCT-15 cells/mL in UTM. Matrix equivalency was evaluated by spiking cultured SARS-CoV-2 (USA-WA1/2020, catalog number 0810587CFHI, ZeptoMetrix, NY, USA), influenza A (Brisbane/59/07, catalog number 0810244CF, ZeptoMetrix, NY, USA), and influenza B (Florida/04/06, catalog number

0810255CF; ZeptoMetrix, NY, USA) viruses in simulated clinical matrix (SIM) and both natural nasopharyngeal swab (NPS) and nasal swab (NS) in UTM at $0 \mathbf { x }$ (negative samples), $2 \mathbf { x }$ and 5x LoD (approximately). For simulated matrix, 10 replicates of negative samples, 30 replicates of positive samples at ${ \sim } 2 \mathbf { x }$ LoD and 10 replicates of positive samples at ${ \sim } 5 \mathbf { x }$ LoD were tested. For natural NPS and NS matrix in UTM, the same number of replicates and concentrations as SIM were tested but divided equally between NPS and NS matrices. One lot of Assay tubes and 10 cobas Liat analyzers (equal number of replicates per analyzer) were used in the study. The acceptance criteria to demonstrate equivalency was $0 \%$ detection for negative samples, $2 9 5 \%$ detection for positive samples at $2 \mathbf { x }$ LoD and $100 \%$ detection for positive samples at 5x LoD for each target.

Negative sample testing showed $0 \%$ positive detection rate for all targets in all matrices, 5x LoD sample showed $100 \%$ detection rate for all targets in all matrices, and $2 \mathbf { x }$ LoD sample testing showed $100 \%$ detection rate for both simulated matrix and natural clinical specimen (NPS/NS) matrix for influenza A and influenza B. A $9 7 \%$ detection rate for SARS-CoV-2 at $2 \mathbf { x }$ LoD was observed in both simulated matrix and clinical specimen (NPS/NS) matrix. There was no difference in detection rates between simulated clinical matrix and NPS & NS clinical UTM matrix, therefore the study acceptance criteria were met. The results of the matrix equivalency study are summarized in Table 12.

Table 12. Summary of Matrix Equivalency Results   

<table><tr><td rowspan=1 colspan=1>Target</td><td rowspan=1 colspan=1>SampleConcentration</td><td rowspan=1 colspan=1>Matrix</td><td rowspan=1 colspan=1>Detection Rate %(#positive/#tested)</td></tr><tr><td rowspan=6 colspan=1>SARS-CoV-2</td><td rowspan=2 colspan=1>Negative</td><td rowspan=1 colspan=1>NPS &amp; NS</td><td rowspan=1 colspan=1>0% (0/10)</td></tr><tr><td rowspan=1 colspan=1>SIM</td><td rowspan=1 colspan=1>0% (0/10)</td></tr><tr><td rowspan=2 colspan=1>2x LoD</td><td rowspan=1 colspan=1>NPS &amp; NS</td><td rowspan=1 colspan=1>97% (29/30)</td></tr><tr><td rowspan=1 colspan=1>SIM</td><td rowspan=1 colspan=1>97% (29/30)</td></tr><tr><td rowspan=2 colspan=1>5x LoD</td><td rowspan=1 colspan=1>NPS &amp; NS</td><td rowspan=1 colspan=1>100% (10/10)</td></tr><tr><td rowspan=1 colspan=1>SIM</td><td rowspan=1 colspan=1>100% (10/10)</td></tr><tr><td rowspan=6 colspan=1>Influenza A</td><td rowspan=2 colspan=1>Negative</td><td rowspan=1 colspan=1>NPS &amp; NS</td><td rowspan=1 colspan=1>0% (0/10)</td></tr><tr><td rowspan=1 colspan=1>SIM</td><td rowspan=1 colspan=1>0% (0/10)</td></tr><tr><td rowspan=2 colspan=1>2x LoD</td><td rowspan=1 colspan=1>NPS &amp; NS</td><td rowspan=1 colspan=1>100% (30/30)</td></tr><tr><td rowspan=1 colspan=1>SIM</td><td rowspan=1 colspan=1>100%(30/30)</td></tr><tr><td rowspan=2 colspan=1>5x LoD</td><td rowspan=1 colspan=1>NPS &amp; NS</td><td rowspan=1 colspan=1>100% (10/10)</td></tr><tr><td rowspan=1 colspan=1>SIM</td><td rowspan=1 colspan=1>100% (10/10)</td></tr><tr><td rowspan=6 colspan=1>Influenza B</td><td rowspan=2 colspan=1>Negative</td><td rowspan=1 colspan=1>NPS &amp; NS</td><td rowspan=1 colspan=1>0% (0/10)</td></tr><tr><td rowspan=1 colspan=1>SIM</td><td rowspan=1 colspan=1>0% (0/10)</td></tr><tr><td rowspan=2 colspan=1>2x LoD</td><td rowspan=1 colspan=1>NPS &amp; NS</td><td rowspan=1 colspan=1>100% (30/30)</td></tr><tr><td rowspan=1 colspan=1>SIM</td><td rowspan=1 colspan=1>100%(30/30)</td></tr><tr><td rowspan=2 colspan=1>5x LoD</td><td rowspan=1 colspan=1>NPS &amp; NS</td><td rowspan=1 colspan=1>100%(10/10)</td></tr><tr><td rowspan=1 colspan=1>SIM</td><td rowspan=1 colspan=1>100% (10/10)</td></tr></table>

NPS: Nasopharyngeal Swab in UTM; NS: Nasal Swab in UTM; SIM: Simulated Clinical Matrix

The results showed that the assay was able to correctly detect the presence of the viral targets suspended in all matrices demonstrating equivalent performance between simulated clinical matrix and natural NPS and NS in UTM when tested with the cobas SARS-CoV-2 & Influenza $\mathrm { A } / \mathrm { B }$ for use on the cobas Liat System.

3. Collection Media Equivalency (UTM, M4RT, and $0 . 9 \%$ physiological saline)

Collection media equivalency was evaluated by spiking cultured SARS-CoV-2 (USAWA1/2020, catalog number 0810587CFHI- $\phantom { - } 0 . 5 \mathrm { m L }$ , ZeptoMetrix, NY, USA), influenza A (Brisbane/59/07, catalog number 0810244CF, ZeptoMetrix, NY, USA), and influenza B (Florida/04/06, catalog number 0810255CF; ZeptoMetrix, NY, USA) viruses at $0 \mathbf { x }$ (negative samples), 2x and 5x LoD into negative nasopharyngeal swabs (NPS) collected in UTM, M4RT and Saline $( 0 . 9 \% \mathrm { N a C l } )$ . For each collection media type, 10 replicates of negative samples, 30 replicates of positive samples at $2 \mathbf { x }$ LoD and 10 replicates of positive samples at 5x LoD were tested. One lot of Assay tubes and 15 cobas Liat analyzers (approximately equal number of replicates per analyzer) were used in the study. The acceptance criteria to demonstrate equivalency was $0 \%$ detection for negative samples, $2 9 5 \%$ detection for positive samples at 2x LoD and $100 \%$ detection for positive samples at 5x LoD for each target.

Negative sample testing showed $0 \%$ detection rate for all targets in all collection media types. 5x LoD sample testing showed $100 \%$ detection rate for all targets for UTM, M4RT, and Saline. 2x LoD sample testing showed $100 \%$ detection rate for UTM, M4RT and Saline for all targets, except for influenza A. At $2 \mathrm { x } \mathrm { L o D }$ in saline, influenza A generated a $9 7 \%$ detection rate with 29 out of 30 samples being detected. All levels for all targets met the study acceptance criteria for detection rate. Additionally, there was no statistical difference observed between UTM & M4RT or between UTM & SAL detection rate proportions. The results of the media equivalency study are summarized in Table 13.

Table 13. Summary of Collection Media Equivalency Results   

<table><tr><td rowspan=1 colspan=1>Target</td><td rowspan=1 colspan=1>SampleConcentration</td><td rowspan=1 colspan=1>NPS CollectionMedia</td><td rowspan=1 colspan=1>Detection Rate %(#positive/#tested)</td></tr><tr><td rowspan=9 colspan=1>SARS-CoV-2</td><td rowspan=3 colspan=1>Negative</td><td rowspan=1 colspan=1>UTM</td><td rowspan=1 colspan=1>0% (0/10)</td></tr><tr><td rowspan=1 colspan=1>M4RT</td><td rowspan=1 colspan=1>0% (0/10)</td></tr><tr><td rowspan=1 colspan=1>Saline</td><td rowspan=1 colspan=1>0% (0/10)</td></tr><tr><td rowspan=3 colspan=1>2x LoD</td><td rowspan=1 colspan=1>UTM</td><td rowspan=1 colspan=1>100% (30/30)</td></tr><tr><td rowspan=1 colspan=1>M4RT</td><td rowspan=1 colspan=1>100% (30/30)</td></tr><tr><td rowspan=1 colspan=1>Saline</td><td rowspan=1 colspan=1>100% (30/30)</td></tr><tr><td rowspan=3 colspan=1>5x LoD</td><td rowspan=1 colspan=1>UTM</td><td rowspan=1 colspan=1>100% (10/10)</td></tr><tr><td rowspan=1 colspan=1>M4RT</td><td rowspan=1 colspan=1>100% (10/10)</td></tr><tr><td rowspan=1 colspan=1>Saline</td><td rowspan=1 colspan=1>100% (10/10)</td></tr><tr><td rowspan=9 colspan=1>Influenza A</td><td rowspan=3 colspan=1>Negative</td><td rowspan=1 colspan=1>UTM</td><td rowspan=1 colspan=1>0% (0/10)</td></tr><tr><td rowspan=1 colspan=1>M4RT</td><td rowspan=1 colspan=1>0% (0/10)</td></tr><tr><td rowspan=1 colspan=1>Saline</td><td rowspan=1 colspan=1>0% (0/10)</td></tr><tr><td rowspan=3 colspan=1>2x LoD</td><td rowspan=1 colspan=1>UTM</td><td rowspan=1 colspan=1>100% (30/30)</td></tr><tr><td rowspan=1 colspan=1>M4RT</td><td rowspan=1 colspan=1>100% (30/30)</td></tr><tr><td rowspan=1 colspan=1>Saline</td><td rowspan=1 colspan=1>97% (29/30)</td></tr><tr><td rowspan=3 colspan=1>5x LoD</td><td rowspan=1 colspan=1>UTM</td><td rowspan=1 colspan=1>100% (10/10)</td></tr><tr><td rowspan=1 colspan=1>M4RT</td><td rowspan=1 colspan=1>100% (10/10)</td></tr><tr><td rowspan=1 colspan=1>Saline</td><td rowspan=1 colspan=1>100% (10/10)</td></tr><tr><td rowspan=9 colspan=1>Influenza B</td><td rowspan=3 colspan=1>Negative</td><td rowspan=1 colspan=1>UTM</td><td rowspan=1 colspan=1>0% (0/10)</td></tr><tr><td rowspan=1 colspan=1>M4RT</td><td rowspan=1 colspan=1>0% (0/10)</td></tr><tr><td rowspan=1 colspan=1>Saline</td><td rowspan=1 colspan=1>0% (0/10)</td></tr><tr><td rowspan=3 colspan=1>2x LoD</td><td rowspan=1 colspan=1>UTM</td><td rowspan=1 colspan=1>100% (30/30)</td></tr><tr><td rowspan=1 colspan=1>M4RT</td><td rowspan=1 colspan=1>100% (30/30)</td></tr><tr><td rowspan=1 colspan=1>Saline</td><td rowspan=1 colspan=1>100% (30/30)</td></tr><tr><td rowspan=3 colspan=1>5x LoD</td><td rowspan=1 colspan=1>UTM</td><td rowspan=1 colspan=1>100% (10/10)</td></tr><tr><td rowspan=1 colspan=1>M4RT</td><td rowspan=1 colspan=1>100% (10/10)</td></tr><tr><td rowspan=1 colspan=1>Saline</td><td rowspan=1 colspan=1>100% (10/10)</td></tr></table>

UTM: Universal Transport Medium; M4RT: Remel Micro Test M4RT; Saline: $0 . 9 \%$ physiological saline

The results showed that the assay was able to correctly detect the presence of the viral targets suspended in all collection media types demonstrating that UTM, M4RT, and $0 . 9 \%$ saline media are acceptable collection and transport media for use with the cobas SARS-CoV-2 & Influenza A/B for use on the cobas Liat System.

# C Clinical Studies:

# 1. Clinical Sensitivity:

The clinical performance of the cobas SARS-CoV-2 & Influenza A/B test for the detection of SARS-CoV-2, influenza A, and influenza B was separately evaluated using unpaired retrospective and paired prospective clinical NPS and NS samples collected from individuals with signs and symptoms of respiratory viral infection. Testing of clinical samples was performed with the cobas SARS-CoV-2 & Influenza A/B for use on the Liat System at 10 point-of-care healthcare facilities (e.g., emergency rooms, outpatient clinics, and physician offices). For the SARS-CoV-2 target, results from cobas SARS-CoV-2 & Influenza A/B were compared to results from three highly sensitive FDA-authorized laboratory-based RTPCR EUA assays (composite comparator method). For influenza A/B targets, results from cobas SARS-CoV-2 & Influenza A/B were compared to results from an acceptable molecular assay for influenza (comparator method).

# Prospective Study:

Prospective clinical specimens were collected and tested from February–June 2022. In total, prospectively collected paired NPS and NS specimens from 640 evaluable individuals were included in the analysis population for the evaluation of cobas Liat SARS-CoV-2 & Influenza A/B. Of these, 24 NPS and $2 4 \ : \mathrm { N S }$ samples were excluded or non-evaluable. For NPS, $1 3 \mathrm { N P S }$ specimens had no comparator results due to incidents (11) or were missing or not tested (2), and 11 NPS specimen results from cobas Liat SARS-CoV-2 & Influenza A/B were non-evaluable due to protocol deviation (8), not tested (1), or invalid results (2). For NS, 11 NS specimens had no comparator results due to incidents (9) or were missing/not tested (2), and $1 3 \mathrm { N S }$ specimen results from cobas Lait SARS-CoV-2 & Influenza A/B were non-evaluable due to protocol deviation (8) or invalid results (5). The remaining 616 prospective NPS and NS samples were included for analysis. Table 14 below provides a summary of the demographic information for the 640 subjects enrolled in the clinical study.

Table 14. Subject Demographics- Prospective Symptomatic Population   

<table><tr><td colspan="1" rowspan="1">Characteristics</td><td colspan="1" rowspan="1">SymptomaticSubjects</td></tr><tr><td colspan="1" rowspan="1">Total, N</td><td colspan="1" rowspan="1">640</td></tr><tr><td colspan="1" rowspan="1">Age (years)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Mean</td><td colspan="1" rowspan="1">31.0</td></tr><tr><td colspan="1" rowspan="1">Standard Deviation</td><td colspan="1" rowspan="1">21.06</td></tr><tr><td colspan="1" rowspan="1">Median</td><td colspan="1" rowspan="1">29.0</td></tr><tr><td colspan="1" rowspan="1">Range (minimum - maximum)</td><td colspan="1" rowspan="1">0.0 - 86.0</td></tr><tr><td colspan="1" rowspan="1">Age group (years), n (%)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">&lt;= 18</td><td colspan="1" rowspan="1">233 (36.4%)</td></tr><tr><td colspan="1" rowspan="1">19 to 39</td><td colspan="1" rowspan="1">171 (26.7%)</td></tr><tr><td colspan="1" rowspan="1">40 to 64</td><td colspan="1" rowspan="1">197 (30.8%)</td></tr><tr><td colspan="1" rowspan="1">&gt;=65</td><td colspan="1" rowspan="1">39 (6.1%)</td></tr><tr><td colspan="1" rowspan="1">Sex at Birth, n (%)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Male</td><td colspan="1" rowspan="1">306 (47.8%)</td></tr><tr><td colspan="1" rowspan="1">Female</td><td colspan="1" rowspan="1">334 (52.2%)</td></tr><tr><td colspan="1" rowspan="1">Ethnicity, n (%)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Hispanic / Latino</td><td colspan="1" rowspan="1">90 (14.1%)</td></tr><tr><td colspan="1" rowspan="1">Not Hispanic / Not Latino</td><td colspan="1" rowspan="1">404 (63.1%)</td></tr><tr><td colspan="1" rowspan="1">Not Reporteda</td><td colspan="1" rowspan="1">132 (20.6%)</td></tr><tr><td colspan="1" rowspan="1">Unknown</td><td colspan="1" rowspan="1">14 (2.2%)</td></tr><tr><td colspan="1" rowspan="1">Race, n (%)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">American Indian / Alaskan Native</td><td colspan="1" rowspan="1">2 (0.3%)</td></tr><tr><td colspan="1" rowspan="1">Asian</td><td colspan="1" rowspan="1">22 (3.4%)</td></tr><tr><td colspan="1" rowspan="1">Black / African-American</td><td colspan="1" rowspan="1">98 (15.3%)</td></tr><tr><td colspan="1" rowspan="1">Native Hawaiian / Pacific Islander</td><td colspan="1" rowspan="1">5 (0.8%)</td></tr><tr><td colspan="1" rowspan="1">White</td><td colspan="1" rowspan="1">322 (50.3%)</td></tr><tr><td colspan="1" rowspan="1">Other</td><td colspan="1" rowspan="1">38 (5.9%)</td></tr><tr><td colspan="1" rowspan="1">Not Reporteda</td><td colspan="1" rowspan="1">153 (23.9%)</td></tr></table>

a A clinical site that was a mobile drive-through site was not able to collect race/ethnicity identification from subjects.

No coinfections with SARS-CoV-2 and influenza $\mathrm { A } / \mathrm { B }$ were detected by the comparator methods or the candidate device. No prospective specimens tested in this performance evaluation were influenza B positive by the comparator method.

As shown in Table 15 for prospective symptomatic subjects, 101 NPS specimens tested positive for SARS-CoV-2 with both the cobas SARS-CoV-2 & Influenza A/B test on cobas Liat System and the composite comparator; five SARS-CoV-2 composite comparator positive specimens tested negative for SARS-CoV-2 with the cobas SARS-CoV-2 & Influenza A/B test. A total of 507 NPS specimens tested negative for SARS-CoV-2 with both the cobas SARS-CoV-2 & Influenza A/B test and the composite comparator; three SARSCoV-2 composite comparator negative specimens tested positive for SARS-CoV-2 with the cobas SARS-CoV-2 & Influenza A/B test. All discordant SARS-CoV-2 results showed late Ct values, which are indicative of NPS specimens from individuals with viral loads near or below the limit of detection of both cobas SARS-CoV-2 & Influenza A/B and the composite comparator methods.

For SARS-CoV-2, the results of the clinical performance evaluation using NPS specimens from prospective symptomatic subjects demonstrated $9 5 . 3 \%$ positive percent agreement (PPA) (101/106; $9 5 \%$ score CI: $8 9 . 4 \% - 9 8 . 0 \% )$ and $9 9 . 4 \%$ negative percent agreement (NPA) (507/510; $9 5 \%$ score CI: $9 8 . 3 \% - 9 9 . 8 \% )$ as compared to the composite comparator method.

Table 15. Clinical performance comparison – SARS-CoV-2 for prospective NPS specimens   

<table><tr><td rowspan=2 colspan=2></td><td rowspan=1 colspan=2>Composite Comparator MethodSARS-CoV-2 Result</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td></tr><tr><td rowspan=2 colspan=1>cobas SARS-CoV-2 &amp; Influenza A/Bon cobas Liat SystemNasopharyngeal Swab (NPS)</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>101</td><td rowspan=1 colspan=1>3</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>507</td></tr></table>

PPA $9 5 . 3 \%$ ( $9 5 \%$ CI: $8 9 . 4 \% - 9 8 . 0 \%$ ) NPA $9 9 . 4 \%$ ( $9 5 \%$ CI: $9 8 . 3 \% - 9 9 . 8 \%$ )

As shown in Table 16 for prospective symptomatic subjects, 105 NS specimens tested positive for SARS-CoV-2 with both the cobas SARS-CoV-2 & Influenza A/B test on cobas Liat System and the composite comparator; four SARS-CoV-2 composite comparator positive specimens tested negative for SARS-CoV-2 with the cobas SARS-CoV-2 & Influenza A/B test. A total of 503 NS specimens tested negative for SARS-CoV-2 with both the cobas SARS-CoV-2 & Influenza A/B test and the composite comparator; four SARSCoV-2 composite comparator negative specimens tested positive for SARS-CoV-2 with the cobas SARS-CoV-2 & Influenza A/B test. All eight of the discordant SARS-CoV-2 results showed late Ct values, which are indicative of NS specimens from individuals potentially with viral loads near or below the limit of detection of both cobas SARS-CoV-2 & Influenza $\mathrm { A } / \mathrm { B }$ and the composite comparator methods.

For SARS-CoV-2, the results of the clinical performance evaluation using NS specimens from prospective symptomatic subjects demonstrated $9 6 . 3 \%$ PPA (105/109; $9 5 \%$ score CI: $9 0 . 9 \% - 9 8 . 6 \% )$ and $9 9 . 2 \%$ NPA (503/507; $9 5 \%$ score CI: $9 8 . 0 \% - 9 9 . 7 \% )$ as compared to the composite comparator method.

Table 16. Clinical performance comparison – SARS-CoV-2 for prospective NS specimens   

<table><tr><td rowspan=2 colspan=2></td><td rowspan=1 colspan=2>Composite Comparator MethodSARS-CoV-2 Result</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td></tr><tr><td rowspan=2 colspan=1>cobas SARS-CoV-2 &amp; InfluenzaA/B on cobas Liat SystemNasal Swab (NS)</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>105</td><td rowspan=1 colspan=1>4</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>4a</td><td rowspan=1 colspan=1>503</td></tr></table>

PPA $9 6 . 3 \%$ $9 5 \%$ CI: $9 0 . 9 \% - 9 8 . 6 \%$ NPA $9 9 . 2 \%$ $9 5 \%$ CI: $9 8 . 0 \% - 9 9 . 7 \% )$   
Note: The nasal swabs were comprised of healthcare provider-collected nasal swab specimens and nasal swab specimens self-collected on-site with healthcare provider instructions.

As shown in Table 17 for prospective symptomatic subjects, 18 NPS specimens tested positive for influenza A with both the cobas SARS-CoV-2 & Influenza A/B test on cobas Liat System and the comparator assay; one influenza A comparator positive specimen tested negative for influenza A with the cobas SARS-CoV-2 & Influenza A/B test. A total of 595

NPS specimens tested negative for influenza A with both the cobas SARS-CoV-2 & Influenza $\mathrm { A } / \mathrm { B }$ test and the comparator assay; two influenza A comparator negative specimens tested positive for influenza A with the cobas SARS-CoV-2 & Influenza A/B test.

For influenza A, the results of the clinical performance evaluation using NPS specimens from prospective symptomatic subjects demonstrated $9 4 . 7 \%$ PPA (18/19; $9 5 \%$ score CI: $7 5 . 4 \%$ - $9 9 . 1 \% )$ and $9 9 . 7 \%$ NPA (595/597; $9 5 \%$ score CI: $9 8 . 8 \% - 9 9 . 9 \% )$ as compared to the comparator method.

Table 17. Clinical performance comparison – Influenza A for prospective NPS specimens   

<table><tr><td rowspan=2 colspan=2></td><td rowspan=1 colspan=2>Comparator MethodInfluenza A Result</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td></tr><tr><td rowspan=2 colspan=1>cobas SARS-CoV-2 &amp; Influenza A/Bon cobas Liat SystemNasopharyngeal Swab (NPS)</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>18</td><td rowspan=1 colspan=1>2</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>595</td></tr></table>

PPA $9 4 . 7 \%$ $9 5 \%$ CI: $7 5 . 4 \% - 9 9 . 1 \%$ NPA $9 9 . 7 \%$ $9 5 \%$ CI: $9 8 . 8 \% - 9 9 . 9 \% )$

As shown in Table 18 for prospective symptomatic subjects, all $2 0 \mathrm { N S }$ specimens tested positive for influenza A with both the cobas SARS-CoV-2 & Influenza $\mathrm { A } / \mathrm { B }$ test on cobas Liat System and the comparator assay. A total of $5 9 5 \mathrm { N S }$ specimens tested negative for influenza A with both the cobas SARS-CoV-2 & Influenza A/B test and the comparator assay; one influenza A comparator negative specimens tested positive for influenza A with the cobas SARS-CoV-2 & Influenza A/B test.

For influenza A, the results of the clinical performance evaluation using NS specimens from prospective symptomatic subjects demonstrated $1 0 0 . 0 \%$ PPA (20/20; $9 5 \%$ score CI: $8 3 . 9 \%$ - $1 0 0 . 0 \% )$ and $9 9 . 8 \%$ NPA (595/596; $9 5 \%$ score CI: $9 9 . 1 \% - 1 0 0 . 0 \% )$ as compared to the comparator method.

Table 18. Clinical performance comparison – Influenza A for prospective NS specimens   

<table><tr><td rowspan=2 colspan=2></td><td rowspan=1 colspan=2>Comparator MethodInfluenza A Result</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td></tr><tr><td rowspan=2 colspan=1>cobas SARS-CoV-2 &amp; InfluenzaA/B on cobas Liat SystemNasal Swab (NS)</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>595</td></tr></table>

Note: The nasal swabs were comprised of healthcare provider-collected nasal swab specimens and nasal swab specimens self-collected on-site with healthcare provider instructions.

For prospective symptomatic subjects, PPA was not calculable for influenza B because no fresh NPS or NS specimens were influenza B positive by the comparator method. For influenza B, the results of the clinical performance evaluation using NPS specimens from prospective symptomatic subjects demonstrated $1 0 0 . 0 \%$ NPA (616/616; $9 5 \%$ score CI: $9 9 . 4 \% - 1 0 0 . 0 \% )$ as compared to the comparator method. The results of the clinical performance evaluation using NS specimens from prospective symptomatic subjects demonstrated $1 0 0 . 0 \%$ NPA (616/616; $9 5 \%$ score CI: $9 9 . 4 \% - 1 0 0 . 0 \% )$ as compared to the comparator method.

# Retrospective Study:

To supplement the prospective data for influenza A and influenza B, retrospective frozen positive and negative NPS $\scriptstyle ( \mathrm { n = 1 } 7 8 )$ and NS $\mathrm { ( n = 1 9 0 }$ ) specimens prospectively obtained during the 2013-2014, 2014-2015, and 2019-2020 flu seasons and during the COVID-19 pandemic (March–June 2021) were distributed to 4 of the 10 sites and worked into the daily workflow of sites for testing. Subject demographic data was not available for the retrospective samples.

Of the178 retrospective NPS specimens (44 influenza A positive, 22 influenza B-positive, and 112 negative) that were tested at sites, two retrospective NPS samples were nonevaluable due to obtaining invalid results from the comparator device, and three obtained invalid results for influenza B with the candidate device, leaving 176 evaluable retrospective NPS samples for influenza A and 173 for influenza B. Of the 190 retrospective NS specimens (37 influenza A-positive, 35 influenza B-positive, and 118 negative) that were tested at sites, three retrospective NS samples were non-evaluable due to obtaining invalid results from the comparator device, and one was aborted by the candidate device, leaving 186 evaluable retrospective NS samples for influenza A and influenza B.

As shown in Table 19 for retrospective NPS specimens, the results of the clinical performance evaluation for influenza A demonstrated $9 7 . 7 \%$ PPA (43/44; $9 5 \%$ score CI: $8 8 . 2 \% - 9 9 . 6 \%$ and $9 9 . 2 \%$ NPA (131/132; $9 5 \%$ score CI: $9 5 . 8 \% - 9 9 . 9 \% )$ as compared to the comparator method.

Table 19. Clinical performance comparison – Influenza A for retrospective NPS specimens   

<table><tr><td rowspan=2 colspan=2></td><td rowspan=1 colspan=2>Comparator MethodInfluenza A Result</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td></tr><tr><td rowspan=2 colspan=1>cobas SARS-CoV-2 &amp;Influenza A/B on cobas LiatSystemNasopharyngeal Swab (NPS)</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>43</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>131</td></tr></table>

PPA $9 7 . 7 \%$ $9 5 \%$ CI: $8 8 . 2 \% - 9 9 . 6 \%$ $9 9 . 2 \%$ $9 5 \%$ CI: $9 5 . 8 \% - 9 9 . 9 \% )$ NPA

As shown in Table 20 for retrospective NS specimens, the results of the clinical performance evaluation for influenza A demonstrated $9 7 . 2 \%$ PPA (35/36; $9 5 \%$ score CI: $8 5 . 8 \% - 9 9 . 5 \% )$ and $1 0 0 . 0 \%$ NPA (150/150; $9 5 \%$ score CI: $9 7 . 5 \% - 1 0 0 . 0 \% )$ as compared to the comparator method.

Table 20. Clinical performance comparison – Influenza A for retrospective NS specimens   

<table><tr><td rowspan=2 colspan=2></td><td rowspan=1 colspan=2>Comparator MethodInfluenza A Result</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td></tr><tr><td rowspan=2 colspan=1>cobas SARS-CoV-2 &amp; InfluenzaA/B on cobas Liat SystemNasal Swab (NS)</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>35</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>150</td></tr></table>

PPA $9 7 . 2 \%$ $9 5 \%$ CI: $8 5 . 8 \% - 9 9 . 5 \%$ ) NPA $1 0 0 . 0 \%$ $9 5 \%$ CI: $9 7 . 5 \% - 1 0 0 . 0 \% )$

As shown in Table 21 for retrospective NPS specimens, the results of the clinical performance evaluation for influenza B demonstrated $1 0 0 . 0 \%$ PPA (22/22; $9 5 \%$ score CI: $8 5 . 1 \% \sim 1 0 0 . 0 \% )$ and $1 0 0 . 0 \%$ NPA (151/151; $9 5 \%$ score CI: $9 7 . 5 \% - 1 0 0 . 0 \% )$ as compared to the comparator method.

Table 21. Clinical performance comparison – Influenza B for retrospective NPS specimens   

<table><tr><td rowspan=2 colspan=2></td><td rowspan=1 colspan=2>Comparator MethodInfluenza B Result</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td></tr><tr><td rowspan=2 colspan=1>cobas SARS-CoV-2 &amp;Influenza A/B on cobas LiatSystemNasopharyngeal Swab (NPS)</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>22</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>151</td></tr></table>

PPA $1 0 0 . 0 \%$ $9 5 \%$ CI: $8 5 . 1 \% \sim 1 0 0 . 0 \% )$ NPA $1 0 0 . 0 \%$ $9 5 \%$ CI: $9 7 . 5 \% - 1 0 0 . 0 \% )$

As shown in Table 22 for retrospective NS specimens, the results of the clinical performance evaluation for influenza B demonstrated $1 0 0 . 0 \%$ PPA (32/32; $9 5 \%$ score CI: $8 9 . 3 \%$ - $1 0 0 . 0 \% )$ and $1 0 0 . 0 \%$ NPA (154/154; $9 5 \%$ score CI: $9 7 . 6 \% - 1 0 0 . 0 \% )$ as compared to the comparator method.

Table 22. Clinical performance comparison – Influenza B for retrospective NS specimens   

<table><tr><td colspan="2" rowspan="2"></td><td colspan="2" rowspan="1">Comparator MethodInfluenza B Result</td></tr><tr><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">cobas SARS-CoV-2 &amp; InfluenzaA/B on cobas Liat System</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">32</td><td colspan="1" rowspan="1">0</td></tr><tr><td>Nasal Swab (NS)</td><td>Negative</td><td>0</td><td>154</td></tr></table>

PPA $1 0 0 . 0 \%$ $9 5 \%$ CI: $8 9 . 3 \% - 1 0 0 . 0 \% )$ NPA $1 0 0 . 0 \%$ $9 5 \%$ CI: $9 7 . 6 \% - 1 0 0 . 0 \% )$

# Invalid rate:

Tables 23 and 24 describe the number of samples from the enrolled prospective and retrospective populations that obtained invalid or failed results with the cobas SARS-CoV-2 & Influenza $\mathrm { A } / \mathrm { B }$ for use on the Liat System.

Table 23. Numbers of Valid and Failed/Invalid Test Results From Specimens From Valid QC Batches by Lot, Site/Instrument and Operator - Nasopharyngeal swab (NPS)   

<table><tr><td rowspan=1 colspan=4></td><td rowspan=1 colspan=4>Invalid Test Results</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=5></td><td rowspan=1 colspan=2>Incidents</td><td rowspan=1 colspan=2>cYanmaGYellowb</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Factor Number</td><td rowspan=1 colspan=1>TotalTestsFromValid QCBatchesN</td><td rowspan=1 colspan=1>FailedTests{a}n %)</td><td rowspan=1 colspan=1>InvalidResultsn (%)</td><td rowspan=1 colspan=1>InstrumentErrorsn (%)</td><td rowspan=1 colspan=1>OtherIncidents{b}n (%)</td><td rowspan=1 colspan=1>ProtocolDeviationsn (%)</td><td rowspan=1 colspan=1>TestsWithValidResultsn (%)</td></tr><tr><td rowspan=1 colspan=1>Lot</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>130</td><td rowspan=1 colspan=1>1 (0.8)</td><td rowspan=1 colspan=1>2 (1.5)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>127 (97.7)</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>382</td><td rowspan=1 colspan=1>3 (0.8)</td><td rowspan=1 colspan=1>1 (0.3)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>6 (1.6)</td><td rowspan=1 colspan=1>372 (97.4)</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>185</td><td rowspan=1 colspan=1>2 (1.1)</td><td rowspan=1 colspan=1>3 (1.6)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>3 (1.6)</td><td rowspan=1 colspan=1>177 (95.7)</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>14 (100.0)</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>117</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>1 (0.9)</td><td rowspan=1 colspan=1>116 (99.1)</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>828</td><td rowspan=1 colspan=1>6 (0.7)</td><td rowspan=1 colspan=1>6 (0.7)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>10 (1.2)</td><td rowspan=1 colspan=1>806 (97.3)</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr><tr><td rowspan=1 colspan=1>Site</td><td rowspan=1 colspan=1>ABQ</td><td rowspan=1 colspan=1>144</td><td rowspan=1 colspan=1>2 (1.4)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>142 (98.6)</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>APR</td><td rowspan=1 colspan=1>248</td><td rowspan=1 colspan=1>1 (0.4)</td><td rowspan=1 colspan=1>2 (0.8)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>245 (98.8)</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>BNR</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>6 (100.0)</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>FRA</td><td rowspan=1 colspan=1>80</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>6 (7.5)</td><td rowspan=1 colspan=1>74 (92.5)</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>HCD</td><td rowspan=1 colspan=1>54</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>54 (100.0)</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>IND</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>9 (100.0)</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>LAS</td><td rowspan=1 colspan=1>26</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>2 (7.7)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>2 (7.7)</td><td rowspan=1 colspan=1>22 (84.6)</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>NSU</td><td rowspan=1 colspan=1>125</td><td rowspan=1 colspan=1>3 (2.4)</td><td rowspan=1 colspan=1>1 (0.8)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>121 (96.8)</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>UCC</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>1 (3.3)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>1 (3.3)</td><td rowspan=1 colspan=1>28 (93.3)</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>URM</td><td rowspan=1 colspan=1>106</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>1 (0.9)</td><td rowspan=1 colspan=1>105 (99.1)</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>828</td><td rowspan=1 colspan=1>6 (0.7)</td><td rowspan=1 colspan=1>6 (0.7)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>10 (1.2)</td><td rowspan=1 colspan=1>806 (97.3)</td></tr><tr><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr><tr><td rowspan=1 colspan=1>Site/Instrument</td><td rowspan=1 colspan=1>1-M1-E-00977</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>1 (4.2)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>23 (95.8)</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1-M1-E-23597</td><td rowspan=1 colspan=1>18</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>18 (100.0)</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1-M1-E-23608</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>21 (100.0)</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1-M1-E-23707</td><td rowspan=1 colspan=1>22</td><td rowspan=1 colspan=1>1 (4.5)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>21 (95.5)</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1-M1-E-23746</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>40 (100.0)</td></tr></table>

<table><tr><td rowspan=1 colspan=4>Invalid Test Results</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2>Incidents</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Factor Number</td><td rowspan=1 colspan=1>Total TestsFromValiCBatchesN</td><td rowspan=1 colspan=1>FailedTests{a}n(%</td><td rowspan=1 colspan=1>Invalidesultsn(%)</td><td rowspan=1 colspan=1>InstrumentErrorsn(%)</td><td rowspan=1 colspan=1>OtherI ncident{bn (%)</td><td rowspan=1 colspan=1>ProtocolDeviationsn (%)</td><td rowspan=1 colspan=1>Tests With ValidResultsn (%)</td></tr><tr><td rowspan=1 colspan=1>1-M1-E-23749</td><td rowspan=1 colspan=1>19</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>19 (100.0)</td></tr><tr><td rowspan=1 colspan=1>2-M1-E-18316</td><td rowspan=1 colspan=1>94</td><td rowspan=1 colspan=1>1 (1.1)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>93 (98.9)</td></tr><tr><td rowspan=1 colspan=1>2-M1-E-18734</td><td rowspan=1 colspan=1>148</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>2 (1.4)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>146 (98.6)</td></tr><tr><td rowspan=1 colspan=1>2-M1-E-18748</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>2 (100.0)</td></tr><tr><td rowspan=1 colspan=1>2-M1-E-23806</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>4 (100.0)</td></tr><tr><td rowspan=1 colspan=1>3-M1-E-00398</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>1 (100.0)</td></tr><tr><td rowspan=1 colspan=1>3-M1-E-23431</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>5 (100.0)</td></tr><tr><td rowspan=1 colspan=1>3-M1-E-23539</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td></tr><tr><td rowspan=1 colspan=1>3-M1-E-23887</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td></tr><tr><td rowspan=1 colspan=1>4-M1-E-15317</td><td rowspan=1 colspan=1>41</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>6 (14.6)</td><td rowspan=1 colspan=1>35 (85.4)</td></tr><tr><td rowspan=1 colspan=1>4-M1-E-23992</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>8 (100.0)</td></tr><tr><td rowspan=1 colspan=1>4-M1-E-24030</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>8 (100.0)</td></tr><tr><td rowspan=1 colspan=1>4-M1-E-24050</td><td rowspan=1 colspan=1>23</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>23 (100.0)</td></tr><tr><td rowspan=1 colspan=1>5-M1-E-00679</td><td rowspan=1 colspan=1>34</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>34 (100.0)</td></tr><tr><td rowspan=1 colspan=1>5-M1-E-00839</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>4 (100.0)</td></tr><tr><td rowspan=1 colspan=1>5-M1-E-23721</td><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>14 (100.0)</td></tr><tr><td rowspan=1 colspan=1>5-M1-E-23823</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>1 (100.0)</td></tr><tr><td rowspan=1 colspan=1>5-M1-E-23877</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>1 (100.0)</td></tr><tr><td rowspan=1 colspan=1>6-M1-E-00366</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>3 (100.0)</td></tr><tr><td rowspan=1 colspan=1>6-M1-E-18724</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>4 (100.0)</td></tr><tr><td rowspan=1 colspan=1>6-M1-E-23079</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>2 (100.0)</td></tr><tr><td rowspan=1 colspan=1>7-M1-E-23553</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>1 (100.0)</td></tr><tr><td rowspan=1 colspan=1>7-M1-E-23563</td><td rowspan=1 colspan=1>25</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>2 (8.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>2 (8.0)</td><td rowspan=1 colspan=1>21 (84.0)</td></tr><tr><td rowspan=1 colspan=1>8-M1-E-15370</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>12 (100.0)</td></tr><tr><td rowspan=1 colspan=1>8-M1-E-24060</td><td rowspan=1 colspan=1>32</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>1 (3.1)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>31 (96.9)</td></tr><tr><td rowspan=1 colspan=1>8-M1-E-24061</td><td rowspan=1 colspan=1>73</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>73 (100.0)</td></tr><tr><td rowspan=1 colspan=1>8-M1-E-24062</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>3 (37.5)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>5 (62.5)</td></tr><tr><td rowspan=1 colspan=1>9-M1-E-15331</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0( 0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>1 (100.0)</td></tr><tr><td rowspan=1 colspan=1>9-M1-E-24075</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>1 (11.1)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>8 (88.9)</td></tr><tr><td rowspan=1 colspan=1>9-M1-E-24082</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>10 (100.0)</td></tr><tr><td rowspan=1 colspan=1>9-M1-E-24086</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>1( 16.7)</td><td rowspan=1 colspan=1>5 (83.3)</td></tr><tr><td rowspan=1 colspan=1>9-M1-E-24088</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>4 (100.0)</td></tr><tr><td rowspan=1 colspan=1>10-M1-E-21999</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>1( 20.0)</td><td rowspan=1 colspan=1>4 (80.0)</td></tr><tr><td rowspan=1 colspan=1>10-M1-E-24041</td><td rowspan=1 colspan=1>101</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>101 (100.0)</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>828</td><td rowspan=1 colspan=1>6 (0.7)</td><td rowspan=1 colspan=1>6 (0.7)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>10 (1.2)</td><td rowspan=1 colspan=1>806 (97.3)</td></tr><tr><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td rowspan=1 colspan=1></td></tr></table>

<table><tr><td rowspan=1 colspan=4></td><td rowspan=1 colspan=4>Invalid Test Results</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=5></td><td rowspan=1 colspan=2>Incidents</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Factor Number</td><td rowspan=1 colspan=1>TotalTestsFromValid QCBatchesN</td><td rowspan=1 colspan=1>FailedTests{a}n (%)</td><td rowspan=1 colspan=1>InvalidResultsn (%)</td><td rowspan=1 colspan=1>InstrumentErrorsn (%)</td><td rowspan=1 colspan=1>OtherIncidents${b}n (%)</td><td rowspan=1 colspan=1>ProtocolDeviationsn (%)</td><td rowspan=1 colspan=1>TestsWithValidResultsn (%)</td></tr><tr><td rowspan=1 colspan=1>Operator</td><td rowspan=1 colspan=1>ABQ1</td><td rowspan=1 colspan=1>56</td><td rowspan=1 colspan=1>1 (1.8)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>55 (98.2)</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>ABQ2</td><td rowspan=1 colspan=1>35</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>35 (100.0)</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>ABQ3</td><td rowspan=1 colspan=1>53</td><td rowspan=1 colspan=1>1 (1.9)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>52 (98.1)</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>APR2</td><td rowspan=1 colspan=1>174</td><td rowspan=1 colspan=1>1 (0.6)</td><td rowspan=1 colspan=1>2 (1.1)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>171 (98.3)</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>APR4</td><td rowspan=1 colspan=1>74</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>74 (100.0)</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>BNR1</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>5 (100.0)</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>BNR2</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>1 (100.0)</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>FRA1</td><td rowspan=1 colspan=1>29</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>1 (3.4)</td><td rowspan=1 colspan=1>28 (96.6)</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>FRA2</td><td rowspan=1 colspan=1>32</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>32 (100.0)</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>FRA4</td><td rowspan=1 colspan=1>19</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>5 (26.3)</td><td rowspan=1 colspan=1>14 (73.7)</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>HCD1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>1 (100.0)</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>HCD2</td><td rowspan=1 colspan=1>26</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>26 (100.0)</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>HCD3</td><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>14 (100.0)</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>HCD4</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>3 (100.0)</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>HCD5</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>10 (100.0)</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>IND1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>1 (100.0)</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>IND2</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>8 (100.0)</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>LAS1</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>2 (13.3)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>13 (86.7)</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>LAS2</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>2 (18.2)</td><td rowspan=1 colspan=1>9(81.8)</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>NSU1</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>1 (12.5)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>7 (87.5)</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>NSU2</td><td rowspan=1 colspan=1>52</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>1 (1.9)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>51 (98.1)</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>NSU3</td><td rowspan=1 colspan=1>65</td><td rowspan=1 colspan=1>2 (3.1)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>63 (96.9)</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>UCC2</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>1 (4.2)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>1 (4.2)</td><td rowspan=1 colspan=1>22 (91.7)</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>UCC4</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>6 (100.0)</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>URM1</td><td rowspan=1 colspan=1>18</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>18 (100.0)</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>URM2</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>1 (9.1)</td><td rowspan=1 colspan=1>10 (90.9)</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>URM3</td><td rowspan=1 colspan=1>56</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>56 (100.0)</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>URM4</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>12 (100.0)</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>URM5</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>9 (100.0)</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>828</td><td rowspan=1 colspan=1>6 (0.7)</td><td rowspan=1 colspan=1>6 (0.7)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>10 (1.2)</td><td rowspan=1 colspan=1>806 (97.3)</td></tr></table>

Note: This table includes both prospective and retrospective sample results. Note: A quality control (QC) batch is defined as the set of external control runs (typically 1 SARS-CoV-2 & Influenza A/B– Positive and 1 SARS-CoV-2 Influenza A/B- Negative). Note: An invalid QC batch is one where at least one external control is invalid or was not performed or there is an incident or protocol deviation that invalidates the entire QC batch. $\{ \mathbf { a } \}$ A failed QC batch is one where a failed test occurred for at least one of the external controls (SARS-CoV-2 & Influenza A/B-Positive or SARS-CoV-2 & Influenza A/B-Negative external control) and the external control was not retested. $\{ { \bf b } \}$ Other incidents include: reagent, operational and other errors.

Table 24. Numbers of Valid and Failed/Invalid Test Results From Specimens From Valid   

<table><tr><td colspan="9" rowspan="1">QC Batches by Lot, Site/Instrument and Operator - Nasal swab (NS)</td></tr><tr><td colspan="4" rowspan="1"></td><td colspan="4" rowspan="1">Invalid Test Results</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="5" rowspan="1"></td><td colspan="2" rowspan="1">Incidents</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Factor Number</td><td colspan="1" rowspan="1">TotalTests From Valid CBatchesN</td><td colspan="1" rowspan="1">Failed Testsa}n(%</td><td colspan="1" rowspan="1">InvalidRResultsn(%)</td><td colspan="1" rowspan="1">InstrumentErrorsn(%)</td><td colspan="1" rowspan="1">OtherIncdentbn (%)</td><td colspan="1" rowspan="1">Protocol Deviationsn (%)</td><td colspan="1" rowspan="1">Tests WithValid Resultsn (%)</td></tr><tr><td colspan="1" rowspan="1">Lot</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">129</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">3 (2.3)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">1 (0.8)</td><td colspan="1" rowspan="1">125 (96.9)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">354</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">7(2.0)</td><td colspan="1" rowspan="1">347 (98.0)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">161</td><td colspan="1" rowspan="1">1 (0.6)</td><td colspan="1" rowspan="1">2 (1.2)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">2 (1.2)</td><td colspan="1" rowspan="1">156 (96.9)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">14</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">14 (100.0)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">176</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">2 (1.1)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">1 (0.6)</td><td colspan="1" rowspan="1">173 (98.3)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Total</td><td colspan="1" rowspan="1">834</td><td colspan="1" rowspan="1">1 (0.1)</td><td colspan="1" rowspan="1">7(0.8)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">11 (1.3)</td><td colspan="1" rowspan="1">815 (97.7)</td></tr><tr><td colspan="1" rowspan="1"></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr><tr><td colspan="1" rowspan="1">Site</td><td colspan="1" rowspan="1">ABQ</td><td colspan="1" rowspan="1">89</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">2 (2.2)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">87 (97.8)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">APR</td><td colspan="1" rowspan="1">286</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">4(1.4)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">1 (0.3)</td><td colspan="1" rowspan="1">281 (98.3)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">BNR</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">6 (100.0)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">FRA</td><td colspan="1" rowspan="1">102</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">1 (1.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">7(6.9)</td><td colspan="1" rowspan="1">94 (92.2)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">HCD</td><td colspan="1" rowspan="1">54</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">54 (100.0)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">IND</td><td colspan="1" rowspan="1">9</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">9(100.0)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">LAS</td><td colspan="1" rowspan="1">22</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">1 (4.5)</td><td colspan="1" rowspan="1">21 (95.5)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">NSU</td><td colspan="1" rowspan="1">131</td><td colspan="1" rowspan="1">1 (0.8)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">130 (99.2)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">UCC</td><td colspan="1" rowspan="1">29</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">1 (3.4)</td><td colspan="1" rowspan="1">28 (96.6)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">URM</td><td colspan="1" rowspan="1">106</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">1 (0.9)</td><td colspan="1" rowspan="1">105 (99.1)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Total</td><td colspan="1" rowspan="1">834</td><td colspan="1" rowspan="1">1 (0.1)</td><td colspan="1" rowspan="1">7(0.8)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">11 (1.3)</td><td colspan="1" rowspan="1">815 (97.7)</td></tr><tr><td colspan="1" rowspan="1"></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Site/Instrument</td><td colspan="1" rowspan="1">1-M1-E-00977</td><td colspan="1" rowspan="1">16</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">16 (100.0)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1-M1-E-23597</td><td colspan="1" rowspan="1">11</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">11 (100.0)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1-M1-E-23608</td><td colspan="1" rowspan="1">12</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">12 (100.0)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1-M1-E-23707</td><td colspan="1" rowspan="1">9</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">9 (100.0)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1-M1-E-23746</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">10 (100.0)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1-M1-E-23749</td><td colspan="1" rowspan="1">31</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">2 (6.5)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">29 (93.5)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2-M1-E-18316</td><td colspan="1" rowspan="1">153</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">2 (1.3)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">151 (98.7)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2-M1-E-18734</td><td colspan="1" rowspan="1">104</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">2 (1.9)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">102 (98.1)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2-M1-E-18748</td><td colspan="1" rowspan="1">18</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">18 (100.0)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2-M1-E-23806</td><td colspan="1" rowspan="1">11</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">1 (9.1)</td><td colspan="1" rowspan="1">10 (90.9)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">3-M1-E-00398</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">1 (100.0)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">3-M1-E-23431</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">1 (100.0)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">3-M1-E-23539</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">4(100.0)</td></tr><tr><td colspan="4" rowspan="1"></td><td colspan="4" rowspan="1">Invalid Test Results</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="4" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="2" rowspan="1">Incidents</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Factor Number</td><td colspan="1" rowspan="1">TotalTests FromValid QCBatchesN</td><td colspan="1" rowspan="1">FailedTests{a}n (%)</td><td colspan="1" rowspan="1">InvalidResultsn (%)</td><td colspan="1" rowspan="1">InstrumentErrorsn (%)</td><td colspan="1" rowspan="1">OtherIncidents{b}}$n (%)</td><td colspan="1" rowspan="1">Protocol Deviationsn (%)</td><td colspan="1" rowspan="1">Tests WithValidResultsn (%)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">3-M1-E-23887</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">4-M1-E-15317</td><td colspan="1" rowspan="1">35</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">1 (2.9)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">1 (2.9)</td><td colspan="1" rowspan="1">33 (94.3)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">4-M1-E-23992</td><td colspan="1" rowspan="1">24</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">5 (20.8)</td><td colspan="1" rowspan="1">19 (79.2)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">4-M1-E-24030</td><td colspan="1" rowspan="1">14</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">14 (100.0)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">4-M1-E-24050</td><td colspan="1" rowspan="1">29</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">1 (3.4)</td><td colspan="1" rowspan="1">28 (96.6)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">5-M1-E-00679</td><td colspan="1" rowspan="1">29</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">29 (100.0)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">5-M1-E-00839</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">4 (100.0)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">5-M1-E-23721</td><td colspan="1" rowspan="1">16</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">16 (100.0)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">5-M1-E-23823</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">1 (100.0)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">5-M1-E-23877</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">4 (100.0)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">6-M1-E-00366</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">4 (100.0)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">6-M1-E-18724</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">3 (100.0)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">6-M1-E-23079</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">7-M1-E-23553</td><td colspan="1" rowspan="1">22</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">1 (4.5)</td><td colspan="1" rowspan="1">21 (95.5)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">7-M1-E-23563</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">8-M1-E-15370</td><td colspan="1" rowspan="1">69</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">69 (100.0)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">8-M1-E-24060</td><td colspan="1" rowspan="1">17</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">17 (100.0)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">8-M1-E-24061</td><td colspan="1" rowspan="1">15</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">15 (100.0)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">8-M1-E-24062</td><td colspan="1" rowspan="1">30</td><td colspan="1" rowspan="1">1 (3.3)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">29 (96.7)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">9-M1-E-15331</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">9-M1-E-24075</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">1 (16.7)</td><td colspan="1" rowspan="1">5 (83.3)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">9-M1-E-24082</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">4 (100.0)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">9-M1-E-24086</td><td colspan="1" rowspan="1">8</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">8 (100.0)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">9-M1-E-24088</td><td colspan="1" rowspan="1">11</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">11 (100.0)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Total</td><td colspan="1" rowspan="1">834</td><td colspan="1" rowspan="1">1 (0.1)</td><td colspan="1" rowspan="1">7(0.8)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">11 (1.3)</td><td colspan="1" rowspan="1">815 (97.7)</td></tr><tr><td colspan="3" rowspan="1"></td><td colspan="5" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Operator</td><td colspan="1" rowspan="1">ABQ1</td><td colspan="1" rowspan="1">16</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">2 (12.5)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">14 (87.5)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">ABQ2</td><td colspan="1" rowspan="1">33</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">33 (100.0)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">ABQ3</td><td colspan="1" rowspan="1">40</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">40 (100.0)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">APR2</td><td colspan="1" rowspan="1">211</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">3 (1.4)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">1 (0.5)</td><td colspan="1" rowspan="1">207 (98.1)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">APR4</td><td colspan="1" rowspan="1">75</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">1 (1.3)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">74 (98.7)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">BNR1</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">5 (100.0)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">BNR2</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">1 (100.0)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">FRA1</td><td colspan="1" rowspan="1">38</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">1 (2.6)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">1 (2.6)</td><td colspan="1" rowspan="1">36 (94.7)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">FRA2</td><td colspan="1" rowspan="1">44</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">44 (100.0)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">FRA4</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">0 (0.0)</td><td colspan="1" rowspan="1">6 (30.0)</td><td colspan="1" rowspan="1">14 (70.0)</td></tr></table>

<table><tr><td rowspan=1 colspan=4>Invalid Test Results</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=2>Incidents</td><td rowspan=1 colspan=2></td></tr><tr><td rowspan=1 colspan=1>Factor Number</td><td rowspan=1 colspan=1>TotalTestsFromValid QCBatchesN</td><td rowspan=1 colspan=1>FailedTests{a}n (%)</td><td rowspan=1 colspan=1>InvalidResultsn (%)</td><td rowspan=1 colspan=1>InstrumentErrorsn (%)</td><td rowspan=1 colspan=1>OtherIncidents{b}n (%)</td><td rowspan=1 colspan=1>Protocol Devationsn (%)</td><td rowspan=1 colspan=1>TestsWithValidResultsn (%)</td></tr><tr><td rowspan=1 colspan=1>HCD1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>1 (100.0)</td></tr><tr><td rowspan=1 colspan=1>HCD2</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>24 (100.0)</td></tr><tr><td rowspan=1 colspan=1>HCD3</td><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>17 (100.0)</td></tr><tr><td rowspan=1 colspan=1>HCD4</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>3 (100.0)</td></tr><tr><td rowspan=1 colspan=1>HCD5</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>9 (100.0)</td></tr><tr><td rowspan=1 colspan=1>IND1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>1 (100.0)</td></tr><tr><td rowspan=1 colspan=1>IND2</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>8 (100.0)</td></tr><tr><td rowspan=1 colspan=1>LAS1</td><td rowspan=1 colspan=1>18</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>1 (5.6)</td><td rowspan=1 colspan=1>17 (94.4)</td></tr><tr><td rowspan=1 colspan=1>LAS2</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>4 (100.0)</td></tr><tr><td rowspan=1 colspan=1>NSU1</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>7 (100.0)</td></tr><tr><td rowspan=1 colspan=1>NSU2</td><td rowspan=1 colspan=1>70</td><td rowspan=1 colspan=1>1 (1.4)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>69 (98.6)</td></tr><tr><td rowspan=1 colspan=1>NSU3</td><td rowspan=1 colspan=1>54</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>54 (100.0)</td></tr><tr><td rowspan=1 colspan=1>UCC2</td><td rowspan=1 colspan=1>23</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>1 (4.3)</td><td rowspan=1 colspan=1>22 (95.7)</td></tr><tr><td rowspan=1 colspan=1>UCC4</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>6 (100.0)</td></tr><tr><td rowspan=1 colspan=1>URM1</td><td rowspan=1 colspan=1>18</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>18 (100.0)</td></tr><tr><td rowspan=1 colspan=1>URM2</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>1 (9.1)</td><td rowspan=1 colspan=1>10 (90.9)</td></tr><tr><td rowspan=1 colspan=1>URM3</td><td rowspan=1 colspan=1>56</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>56 (100.0)</td></tr><tr><td rowspan=1 colspan=1>URM4</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>12 (100.0)</td></tr><tr><td rowspan=1 colspan=1>URM5</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>9 (100.0)</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>834</td><td rowspan=1 colspan=1>1 (0.1)</td><td rowspan=1 colspan=1>7 (0.8)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>0 (0.0)</td><td rowspan=1 colspan=1>11 (1.3)</td><td rowspan=1 colspan=1>815 (97.7)</td></tr></table>

Note: This table includes both prospective and retrospective sample results. Note: A quality control (QC) batch is defined as the set of external control runs (typically 1 SARS-CoV-2 & Influenza A/B– Positive and 1 SARS-CoV-2 Influenza A/B- Negative). Note: An invalid QC batch is one where at least one external control is invalid or was not performed or there is an incident or protocol deviation that invalidates the entire QC batch. $\{ \mathbf { a } \}$ A failed QC batch is one where a failed test occurred for at least one of the external controls (SARS-CoV-2 & Influenza A/B-Positive or SARS-CoV-2 & Influenza A/B-Negative external control) and the external control was not retested. $\{ { \bf b } \}$ Other incidents include: reagent, operational and other errors.

2. Clinical Specificity: See Section “Clinical Sensitivity” above

3. Other Clinical Supportive Data (When 1. and 2. Are Not Applicable): Not applicable

D Clinical Cut-Off: Not applicable

E Expected Values/Reference Range:

For the prospective clinical performance evaluation of cobas SARS-CoV-2 & Influenza A/B, paired NPS and NS specimens from 640 evaluable subjects, including 616 evaluable results, were freshly collected and tested at 10 point-of-care clinical sites in the United States during February–June 2022. Expected value (as determined by cobas SARS-CoV-2 & Influenza A/B) summaries for prospective specimens, stratified by specimen collection/testing site are presented for SARS-CoV-2 and influenza A targets in Table 25 and Table 26, respectively.

No prospective fresh specimens tested in this performance evaluation were influenza B positive by either cobas SARS-CoV-2 & Influenza $\mathrm { A } / \mathrm { B }$ or comparator test methods.

Table 25. Expected value summary by clinical site for prospective clinical evaluation for SARSCoV-2 from February-June 2022   

<table><tr><td rowspan=2 colspan=1>ClinicalSiteID</td><td rowspan=2 colspan=1>Site location</td><td rowspan=1 colspan=3>NPS Specimens</td><td rowspan=1 colspan=3>NS Specimens</td></tr><tr><td rowspan=1 colspan=1>TotalNo.</td><td rowspan=1 colspan=1>No. Positivefor SARS-CoV-2</td><td rowspan=1 colspan=1>ExpectedValue</td><td rowspan=1 colspan=1>TotalNo.</td><td rowspan=1 colspan=1>No. Positivefor SARS-CoV-2</td><td rowspan=1 colspan=1>ExpectedValue</td></tr><tr><td rowspan=1 colspan=2>Overall</td><td rowspan=1 colspan=1>616</td><td rowspan=1 colspan=1>104</td><td rowspan=1 colspan=1>16.9%</td><td rowspan=1 colspan=1>616</td><td rowspan=1 colspan=1>109</td><td rowspan=1 colspan=1>17.7%</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>Albuquerque,NM</td><td rowspan=1 colspan=1>23</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>22</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>4.5%</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>Vienna, VA</td><td rowspan=1 colspan=1>241</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>12.4%</td><td rowspan=1 colspan=1>240</td><td rowspan=1 colspan=1>34</td><td rowspan=1 colspan=1>14.2%</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>Northridge,CA</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>Savannah,GA</td><td rowspan=1 colspan=1>46</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>26.1%</td><td rowspan=1 colspan=1>46</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>26.1%</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>North Miami,FL</td><td rowspan=1 colspan=1>52</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>23.1%</td><td rowspan=1 colspan=1>52</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>21.2%</td></tr><tr><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>Indianapolis,IN</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>11.1%</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>12.5%</td></tr><tr><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>Las Vegas,NV</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%</td></tr><tr><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>Evanston, IL</td><td rowspan=1 colspan=1>89</td><td rowspan=1 colspan=1>27</td><td rowspan=1 colspan=1>30.3%</td><td rowspan=1 colspan=1>89</td><td rowspan=1 colspan=1>27</td><td rowspan=1 colspan=1>30.3%</td></tr><tr><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>Seneca, SC</td><td rowspan=1 colspan=1>25</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>4.0%</td><td rowspan=1 colspan=1>28</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>7.1%</td></tr><tr><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>Rochester,NY</td><td rowspan=1 colspan=1>105</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>20.0%</td><td rowspan=1 colspan=1>105</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>20.0%</td></tr></table>

Table 26. Expected value summary by clinical site for prospective clinical evaluation for influenza A from February-June 2022   

<table><tr><td colspan="1" rowspan="2">ClinicalSite ID</td><td colspan="1" rowspan="2">Site location</td><td colspan="3" rowspan="1">NPS Specimens</td><td colspan="3" rowspan="1">NS Specimens</td></tr><tr><td colspan="1" rowspan="1">TotalNo.</td><td colspan="1" rowspan="1">No. Positivefor InfluenzaA</td><td colspan="1" rowspan="1">ExpectedValue</td><td colspan="1" rowspan="1">TotalNo.</td><td colspan="1" rowspan="1">No. Positivefor InfluenzaA</td><td colspan="1" rowspan="1">ExpectedValue</td></tr><tr><td colspan="2" rowspan="1">Overall</td><td colspan="1" rowspan="1">616</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">3.2%</td><td colspan="1" rowspan="1">616</td><td colspan="1" rowspan="1">21</td><td colspan="1" rowspan="1">3.4%</td></tr><tr><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Albuquerque,NM</td><td colspan="1" rowspan="1">23</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">4.3%</td><td colspan="1" rowspan="1">22</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">4.5%</td></tr><tr><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">Vienna, VA</td><td colspan="1" rowspan="1">241</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">2.5%</td><td colspan="1" rowspan="1">240</td><td colspan="1" rowspan="1">7</td><td colspan="1" rowspan="1">2.9%</td></tr><tr><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">Northridge,CA</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0%</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0%</td></tr><tr><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">Savannah,GA</td><td colspan="1" rowspan="1">46</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">4.3%</td><td colspan="1" rowspan="1">46</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">4.3%</td></tr><tr><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">North Miami,FL</td><td colspan="1" rowspan="1">52</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0%</td><td colspan="1" rowspan="1">52</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0%</td></tr><tr><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">Indianapolis,IN</td><td colspan="1" rowspan="1">9</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0%</td><td colspan="1" rowspan="1">8</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0%</td></tr><tr><td colspan="1" rowspan="1">7</td><td colspan="1" rowspan="1">Las Vegas,NV</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0%</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0%</td></tr><tr><td colspan="1" rowspan="1">8</td><td colspan="1" rowspan="1">Evanston, IL</td><td colspan="1" rowspan="1">89</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">2.2%</td><td colspan="1" rowspan="1">89</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">2.2%</td></tr><tr><td colspan="1" rowspan="1">9</td><td colspan="1" rowspan="1">Seneca, SC</td><td colspan="1" rowspan="1">25</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">8.0%</td><td colspan="1" rowspan="1">28</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">7.1%</td></tr><tr><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">Rochester,NY</td><td colspan="1" rowspan="1">105</td><td colspan="1" rowspan="1">7</td><td colspan="1" rowspan="1">6.7%</td><td colspan="1" rowspan="1">105</td><td colspan="1" rowspan="1">7</td><td colspan="1" rowspan="1">6.7%</td></tr></table>

# VIII Proposed Labeling:

The labeling supports or the finding of substantial equivalence for this device.

# IX Conclusion:

The submitted information in this premarket notification is complete and supports a substantial equivalence decision.