# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATIONDECISION SUMMARYASSAY AND INSTRUMENT

I Background Information:

A 510(k) Number K230451   
B Applicant Hologic, Inc.   
C Proprietary and Established Names Aptima Chlamydia trachomatis Assay   
D Regulatory Information

<table><tr><td rowspan=1 colspan=1>ProductCode(s)</td><td rowspan=1 colspan=1>Classification</td><td rowspan=1 colspan=1>RegulationSection</td><td rowspan=1 colspan=1>Panel</td></tr><tr><td rowspan=1 colspan=1>MKZ</td><td rowspan=1 colspan=1>Class I, reserved</td><td rowspan=1 colspan=1>21 CFR 866.3120 -Chlamydia SerologicalReagents</td><td rowspan=1 colspan=1>MI - Microbiology</td></tr></table>

# II Submission/Device Overview:

# A Purpose for Submission:

The Aptima CT assay (K043072) was previously cleared for use with the Tigris DTS platform (K063451) for endocervical and male urethral swabs, female and male urine, clinician- and patient-collected vaginal swabs, and PreservCyt liquid Pap specimens. With this submission, Hologic seeks clearance for the use of the assay on the Panther instrument with patient-collected vaginal swab specimens, and female and male urine specimens on the Panther system.

# B Measurand:

Chlamydia trachomatis (CT) ribosomal RNA

C Type of Test: Target-mediated amplificated nucleic acid probe test

# III Intended Use/Indications for Use:

# A Intended Use(s):

The Aptima Chlamydia trachomatis (CT) assay is an in vitro qualitative nucleic acid amplification test (NAAT) for the detection of ribosomal RNA (rRNA) from Chlamydia trachomatis to aid in the diagnosis of chlamydial urogenital disease using the Panther system.

The assay may be used to test the following specimens from symptomatic or asymptomatic individuals: patient-collected vaginal swab specimens1 (in a clinical setting); and female and male urine specimens.

1 Patient-collected vaginal swab specimens are an option for screening women when a pelvic exam is not otherwise indicated. The Aptima Multitest Swab Specimen Collection Kit has not been evaluated for home use.

B Indication(s) for Use: Same as the Intended Use above

C Special Conditions for Use Statement(s): Rx - For Prescription Use Only

D Special Instrument Requirements: Panther System

# IV Device/System Characteristics:

# A Device Description:

The Aptima CT assay is a nucleic acid amplification test that utilizes the target capture (TC), transcription-mediated amplification (TMA) and hybridization protection assay (HPA) technologies for the qualitative detection of Chlamydia trachomatis. Specimens are collected and transferred using their respective collection kit. The transport solution in the kit tube releases the rRNA target that are then captured by magnetic microparticles. A unique set of primers is used to amplify the target and the amplicon is detected by nucleic acid hybridization. Assay test results are automatically interpreted by the Panther System Aptima CT assay software.

This assay is similar to the Aptima Assay for Chlamydia trachomatis (K063451) in that the target organism is $C .$ . trachomatis. The primary differences between the two assays are the platform on which the assay is performed. There are no modifications to assay reagents.

# B Principle of Operation:

The Aptima CT assay combines the technologies of target capture, TMA, and HPA. Specimens are collected and transferred into their respective specimen transport tubes. The transport solution in these tubes releases the rRNA target and protects it from degradation during storage. The target rRNA molecule is isolated from the specimens by use of a capture oligomer via target capture that utilizes magnetic microparticles. The capture oligomer contains a sequence complementary to a specific region of the target molecule, as well as a string of deoxyadenosine residues. During the hybridization step, the sequence specific region of the capture oligomer binds to a specific region of the target molecule. The capture oligomer:target complex is then

captured out of solution by decreasing the temperature of the reaction to room temperature. This temperature reduction allows hybridization to occur between the deoxyadenosine region on the capture oligomer and the polydeoxythymidine molecules that are covalently attached to the magnetic particles. The microparticles, including the captured target molecule bound to them, are pulled to the side of the reaction vessel using magnets and the supernatant is aspirated. The particles are washed to remove residual specimen matrix that may contain amplification reaction inhibitors. After the target capture steps are completed, the specimens are ready for amplification. Target amplification assays are based on the ability of complementary oligonucleotide primers to specifically anneal and allow enzymatic amplification of the target nucleic acid strands. The Hologic TMA reaction replicates a specific region of the 16S rRNA from CT via DNA intermediates. A unique set of primers is used for the target molecule. Detection of the rRNA amplification product sequences (amplicon) is achieved using nucleic acid hybridization. A single-stranded chemiluminescent DNA probe, which is complementary to a region of the target amplicon, is labeled with an acridinium ester molecule. The labeled DNA probe combines with amplicon to form stable RNA:DNA hybrids. The Selection Reagent differentiates hybridized from unhybridized probe, eliminating the generation of signal from unhybridized probe. During the detection step, light emitted from the labeled RNA:DNA hybrids is measured as photon signals in a luminometer, and are reported as Relative Light Units (RLU).

# C Instrument Description Information:

1. Instrument Name: Panther System

2. Specimen Identification: By handheld barcode reader and manual entry

3. Specimen Sampling and Handling: Fully automated

4. Calibration: The Aptima CT assay requires no calibration.

The Panther System undergoes preventative maintenance every 12 months, which includes luminometer calibration.

5. Quality Control: The Aptima CT assay Controls Kit includes 5 vials each of Positive and Negative Controls which are ready to use.

# V Substantial Equivalence Information:

A Predicate Device Name(s): Aptima Assay for Chlamydia trachomatis

B Predicate 510(k) Number(s): K063451

# C Comparison with Predicate(s):

<table><tr><td colspan="1" rowspan="1">Device &amp; PredicateDevice(s):</td><td colspan="1" rowspan="1">K230451</td><td colspan="1" rowspan="1">K063451</td></tr><tr><td colspan="1" rowspan="1">Device Trade Name</td><td colspan="1" rowspan="1">Aptima Chlamydiatrachomatis assay(Panther)</td><td colspan="1" rowspan="1">Aptima Assay forChlamydia trachomatis(Tigris)</td></tr><tr><td colspan="1" rowspan="1">General DeviceCharacteristic Similarities</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Intended Use/IndicationsFor Use</td><td colspan="1" rowspan="1">The Aptima Chlamydiatrachomatis (CT) assayis an in vitro qualitativenucleic acidamplification test(NAAT) for thedetection of ribosomalRNA (rRNA) fromChlamydia trachomatisto aid in the diagnosisof chlamydialurogenital disease usingthe Panther system.The assay may be usedto test the followingspecimens fromsymptomatic orasymptomaticindividuals: patient-collected vaginalswabs1(in a clinicalsetting); and female andmale urine specimens.1Patient-collectedvaginal swab specimensare an option forscreening women whena pelvic exam is nototherwise indicated.The Aptima MultitestSwab SpecimenCollection kit has notbeen evaluated forhome use.</td><td colspan="1" rowspan="1">The Aptima Assay forChlamydia trachomatisassay is a targetamplification nucleicacid probe test thatutilizes target capturefor the in vitroqualitative detection ofribosomal RNA (rRNA)from Chlamydiatrachomatis (CT) to aidin the diagnosis ofchlamydial urogenitaldisease using the TigrisDTS AutomatedAnalyzer or semi-automatedinstrumentation asspecified. The assaymay be used to test thefollowing specimensfrom symptomaticindividuals: clinician-collected endocervical,vaginal and maleurethral swabspecimens; and femaleand male urinespecimens. The assaymay be used to test thefollowing specimensfrom asymptomaticindividuals: clinician-collected endocervical,vaginal and maleurethral swabspecimens; patient-collected vaginal swab</td></tr><tr><td colspan="1" rowspan="1">′</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">specimens1; and femaleand male urinespecimens. This assay isalso intended for usewith the testing ofgynecologicalspecimens, from bothsymptomatic andasymptomatic patients,collected in thePreservCyt solution andprocessed with theCytyc ThinPrep 2000System.1Patient-collectedvaginal swab specimensare an option forscreening women whena pelvic exam is nototherwise indicated.The Aptima MultitestSwab SpecimenCollection Kit is not forhome use.</td></tr><tr><td colspan="1" rowspan="1">Technology Principle ofOperation</td><td colspan="1" rowspan="1">Target Capture (TC),Transcription-MediatedAmplification (TMA),HybridizationProtection Assay (HPA)</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Assay Targets</td><td colspan="1" rowspan="1">Chlamydia trachomatisrRNA</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Assay Results</td><td colspan="1" rowspan="1">Qualitative</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Function</td><td colspan="1" rowspan="1">Detection of rRNAfrom Chlamydiatrachomatis</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">General DeviceCharacteristic Differences</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Platform</td><td colspan="1" rowspan="1">Automated PANTHERSystem</td><td colspan="1" rowspan="1">Automated TIGRISSystem</td></tr><tr><td colspan="1" rowspan="1">Specimen Types</td><td colspan="1" rowspan="1">Female Specimens:Patient-collectedvaginal swabUrineMale Specimens:</td><td colspan="1" rowspan="1">Female Specimens:Clinician -collectedvaginal swabPatient-collectedvaginal swab(asymptomatic</td></tr><tr><td></td><td>Urine • O O • •</td><td>only) Endocervical swab Gynecological specimens in PreservCyt solution Urine Male Specimens: Urethral swab Urine</td></tr></table>

# VI Standards/Guidance Documents Referenced:

CLSI EP05-A3 (Reaffirmed: September 2019) 7-251 Evaluation of Precision of Qualitative Measurement Procedures; Approved Guideline – Third Edition

CLSI EP07 $3 ^ { \mathrm { r d } }$ Edition 7-275 Interference Testing in Clinical Chemistry

CLSI EP12-A2 7-152 User Protocol of Evaluation of Qualitative Test Performance; Approved Guideline – Second Edition

CLSI EP15-A3 7-253 User Verification of Precision and Estimation of Bias; Approved Guideline – Second Edition

CLSI EP17-A2 7-233 Evaluation of Detection Capability for Clinical Laboratory Measurement Procedures; Approved Guideline – Second Edition

CLSI EP25-A (Replaces EP25-P) 7-235 Evaluation of Stability of In Vitro Diagnostic Reagents; Approved Guideline

CLSI MM03- $\cdot 3 ^ { \mathrm { r d } }$ Edition (Replaces MM03-A2) 7-260 Molecular Diagnostic Methods for Infectious Diseases

CLSI MM13- $\cdot 2 ^ { \mathrm { n d } }$ Edition 7-300 Collection Transport Preparation and Storage of Specimens for Molecular Methods

IEC 62304 Edition 1.1 2015-06 Consolidated Version 13-79. Medical device software – Software life cycle processes

ISO 14971 Third Edition 2019-12 5-125 Medical devices – Application of risk management to medical devices

IEC 60601-1-2 Edition 4.0 2014-02 19-8 Medical electrical equipment – Part 1-2: General requirements for basic safety and essential performance – Collateral Standard: Electromagnetic disturbances – Requirements and tests

# A Analytical Performance:

# 1. Precision/Reproducibility:

Reproducibility

The reproducibility study was conducted over six days, at two external and one internal sites using the Panther System. Aptima CT assay reproducibility was evaluated across two assay lots with six operators. Two operators at each of the three testing sites performed a total of six Aptima CT assay runs per kit lot for a total of 36 runs per kit lot. Each run consisted of a 4-member precision panel: one CT-negative and three CT-positive panel members. A range of low and high positive panel members were created by spiking swab transport medium with CT organisms at concentrations from $0 . 2 5 \mathrm { \ I F U / m L }$ to $2 5 \mathrm { \ I F U / m L }$ . Reproducibility was determined by calculating the agreement (with a $9 5 \%$ two-sided confidence interval) with expected results for each panel member by site, by lot, and overall. Signal variability was calculated 1) within runs, 2) between runs, 3) between operators, 4) between sites/instruments, and 5) between reagent lots. Agreement values were $100 \%$ for all panel members. Two invalid results, one at each of the external sites, were excluded from the analyses.

Table 1. Reproducibility Study Data   

<table><tr><td rowspan=2 colspan=1>PanelMember</td><td rowspan=2 colspan=1>TargetedConc(IFU/mL)</td><td rowspan=2 colspan=1>N</td><td rowspan=2 colspan=1>%Agrmt</td><td rowspan=2 colspan=1>MeanRLU</td><td rowspan=1 colspan=2>BetweenSites</td><td rowspan=1 colspan=2>Between-Operators</td><td rowspan=1 colspan=2>Between-Lot</td><td rowspan=1 colspan=2>Between-Run</td><td rowspan=1 colspan=2>Within-Run</td><td rowspan=1 colspan=2>Total</td></tr><tr><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>CV(%)</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>CV(%)</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>CV(%)</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>CV(%)</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>CV(%)</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>CV(%)</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1071</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>1.5</td><td rowspan=1 colspan=1>0.8</td><td rowspan=1 colspan=1>49.7</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>4.9</td><td rowspan=1 colspan=1>1.5</td><td rowspan=1 colspan=1>101.1</td><td rowspan=1 colspan=1>1.7</td><td rowspan=1 colspan=1>112.8</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0.25</td><td rowspan=1 colspan=1>108</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>7339.0</td><td rowspan=1 colspan=1>272.0</td><td rowspan=1 colspan=1>3.7</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>80.0</td><td rowspan=1 colspan=1>1.1</td><td rowspan=1 colspan=1>98.2</td><td rowspan=1 colspan=1>1.3</td><td rowspan=1 colspan=1>142.0</td><td rowspan=1 colspan=1>1.9</td><td rowspan=1 colspan=1>331.9</td><td rowspan=1 colspan=1>4.5</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>2.5</td><td rowspan=1 colspan=1>108</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>7387.6</td><td rowspan=1 colspan=1>307.8</td><td rowspan=1 colspan=1>4.2</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>97.9</td><td rowspan=1 colspan=1>1.3</td><td rowspan=1 colspan=1>139.9</td><td rowspan=1 colspan=1>1.9</td><td rowspan=1 colspan=1>114.0</td><td rowspan=1 colspan=1>1.5</td><td rowspan=1 colspan=1>370.0</td><td rowspan=1 colspan=1>5.0</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>25</td><td rowspan=1 colspan=1>1071</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>7424.4</td><td rowspan=1 colspan=1>285.6</td><td rowspan=1 colspan=1>3.8</td><td rowspan=1 colspan=1>39.6</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>136.9</td><td rowspan=1 colspan=1>1.8</td><td rowspan=1 colspan=1>91.3</td><td rowspan=1 colspan=1>1.2</td><td rowspan=1 colspan=1>138.7</td><td rowspan=1 colspan=1>1.9</td><td rowspan=1 colspan=1>359.8</td><td rowspan=1 colspan=1>4.8</td></tr></table>

Agrmt $=$ agreement with expected result, Conc $=$ concentration, $\mathrm { C V } =$ coefficient of variation, RLU $=$ relative light unit, $\mathbf { S D = }$ standard deviation Notes: The RLU value reported by the software is the total measured RLU divided by 1000 with the digits after the decimal point truncated. Variability from some factors may be numerically negative. In these cases, SD and CV are shown as 0.0. 1 One invalid result was excluded from the analysis.

# Within-Laboratory Precision

Within lab repeatability was evaluated using a 6-member precision panel. The negative panel member consisted of swab transport medium (STM) only and the positive panel members were created by spiking STM with concentrations of CT organisms ranging from 0.25 to $2 { , } 5 0 0 \ \mathrm { I F U / m L }$ . Testing was performed by three operators using three separate Panther instruments. Two reagent lots were used on each instrument, at two runs per day, with three replicates, over 12 days. The percent agreement with the remaining panels to the expected results was $100 \%$ , with the lower-bound of the two-sided $9 5 \%$ confidence interval $9 7 . 4 \%$ for each panel member.

Table 2. With-in Laboratory Precision, Percent Agreement, $9 5 \%$ Confidence Interval   

<table><tr><td rowspan=2 colspan=1>Panel</td><td rowspan=2 colspan=1>CT concentration</td><td rowspan=2 colspan=1>#Positive/Tested</td><td rowspan=2 colspan=1>%Positive</td><td rowspan=2 colspan=1>%Agreement</td><td rowspan=1 colspan=2>95% Confidence Interval</td></tr><tr><td rowspan=1 colspan=1>Lower</td><td rowspan=1 colspan=1>Upper</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0/144</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>97.4</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0.25 IFU/mL or 0.5 fg rRNA/assay</td><td rowspan=1 colspan=1>144/144</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>97.4</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>2.5 IFU/mL or 5 fg rRNA/assay</td><td rowspan=1 colspan=1>144/144</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>97.4</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>25 IFU/mL or 50 fg rRNA/assay</td><td rowspan=1 colspan=1>144/144</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>97.4</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>250 IFU/mL or 500 fg rRNA/assay</td><td rowspan=1 colspan=1>144/144</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>97.4</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>2,500 IFU/mL or 5,000 fgrRNA/assay</td><td rowspan=1 colspan=1>144/144</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>97.4</td><td rowspan=1 colspan=1>100</td></tr></table>

Table 3. Panther System Aptima CT Assay Precision Data   

<table><tr><td rowspan=2 colspan=1>Panel</td><td rowspan=2 colspan=1>N</td><td rowspan=2 colspan=1>MeanRLU(x1000)</td><td rowspan=2 colspan=1>%Agrmt</td><td rowspan=1 colspan=2>Between-Instrument</td><td rowspan=1 colspan=2>Between-Lot</td><td rowspan=1 colspan=2>Between-Run</td><td rowspan=1 colspan=2>Within-Run</td><td rowspan=1 colspan=2>Total</td></tr><tr><td rowspan=1 colspan=1>SD(x1000)</td><td rowspan=1 colspan=1>CV(%)</td><td rowspan=1 colspan=1>SD(x1000)</td><td rowspan=1 colspan=1>CV(%)</td><td rowspan=1 colspan=1>SD(x1000)</td><td rowspan=1 colspan=1>CV(%)</td><td rowspan=1 colspan=1>SD(x1000)</td><td rowspan=1 colspan=1>CV(%)</td><td rowspan=1 colspan=1>SD(x1000)</td><td rowspan=1 colspan=1>CV(%)</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>144</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>18.7</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>28.9</td><td rowspan=1 colspan=1>2.0</td><td rowspan=1 colspan=1>110.0</td><td rowspan=1 colspan=1>2.1</td><td rowspan=1 colspan=1>115.3</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>144</td><td rowspan=1 colspan=1>7670</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>348.2</td><td rowspan=1 colspan=1>4.5</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>430.3</td><td rowspan=1 colspan=1>5.6</td><td rowspan=1 colspan=1>153.1</td><td rowspan=1 colspan=1>2.0</td><td rowspan=1 colspan=1>574.3</td><td rowspan=1 colspan=1>7.5</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>144</td><td rowspan=1 colspan=1>7638</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>376.2</td><td rowspan=1 colspan=1>4.9</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>436.4</td><td rowspan=1 colspan=1>5.7</td><td rowspan=1 colspan=1>124.8</td><td rowspan=1 colspan=1>1.6</td><td rowspan=1 colspan=1>589.5</td><td rowspan=1 colspan=1>7.7</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>144</td><td rowspan=1 colspan=1>7657</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>370.2</td><td rowspan=1 colspan=1>4.8</td><td rowspan=1 colspan=1>98.8</td><td rowspan=1 colspan=1>1.3</td><td rowspan=1 colspan=1>430.2</td><td rowspan=1 colspan=1>5.6</td><td rowspan=1 colspan=1>125.1</td><td rowspan=1 colspan=1>1.6</td><td rowspan=1 colspan=1>589.5</td><td rowspan=1 colspan=1>7.7</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>144</td><td rowspan=1 colspan=1>7661</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>391.0</td><td rowspan=1 colspan=1>5.1</td><td rowspan=1 colspan=1>121.0</td><td rowspan=1 colspan=1>1.6</td><td rowspan=1 colspan=1>441.4</td><td rowspan=1 colspan=1>5.8</td><td rowspan=1 colspan=1>131.3</td><td rowspan=1 colspan=1>1.7</td><td rowspan=1 colspan=1>616.1</td><td rowspan=1 colspan=1>8.0</td></tr><tr><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>144</td><td rowspan=1 colspan=1>7598</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>391.8</td><td rowspan=1 colspan=1>5.2</td><td rowspan=1 colspan=1>128.3</td><td rowspan=1 colspan=1>1.7</td><td rowspan=1 colspan=1>450.2</td><td rowspan=1 colspan=1>5.9</td><td rowspan=1 colspan=1>112.3</td><td rowspan=1 colspan=1>1.5</td><td rowspan=1 colspan=1>620.7</td><td rowspan=1 colspan=1>8.2</td></tr></table>

Agrmt $=$ agreement, $\mathrm { C V } =$ coefficient of variation, RLU $=$ relative light unit, $\mathrm { S D = }$ standard deviation

2. Linearity: Not applicable.

3. Analytical Specificity/Interference: Please refer to K063451 for Analytical Specificity/Interference study.

4. Assay Reportable Range: Not applicable. This is a qualitative assay with a binary output (positive/negative).

5. Traceability, Stability, Expected Values (Controls, Calibrators, or Methods): Please refer to the decision summary of K063451 for stability study data.

6. Detection Limit: Initially panels were prepared by spiking CT organisms into pools of urine, vaginal swab matrix and STM at rRNA equivalents of 0.25 IFU/mL and 2.5 IFU/mL. Panels were tested on three Panther systems using two lots of reagents in replicates of 60 per panel member. The analytical sensitivity claim for the Aptima CT assay is $2 . 5 \ : \mathrm { I F U / m L }$ . The limit of detection (LoD) for the Aptima CT assay was further confirmed by testing two quantified strains (serovars G and E) of CT spiked into negative pools of vaginal matrix Each serovar was detected with greater than $9 5 \%$ positivity at less than $2 . 5 \mathrm { \ I F U / m L }$ ( $9 5 \%$ detection at 0.00267 IFU/mL for serovar E and 0.00441 IFU/mL for serovar G). The analytical sensitivity of the Aptima CT assay was evaluated for 15 serovars on the DTS system, including serovars E and G, the two serovars giving a similar result on the Panther System.

# 7. Assay Cut-Off:

Assay test results are automatically interpreted by the Aptima assay software. A test results may be negative, equivocal, positive, or invalid as determined by the toral RLU in the detection step as shown in Table 4.

Table 4. Test Interpretation   

<table><tr><td rowspan=1 colspan=1>Test Interpretation</td><td rowspan=1 colspan=1>Total RLU (x1000)</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>0*&lt;50</td></tr><tr><td rowspan=1 colspan=1>Equivocal</td><td rowspan=1 colspan=1>50 to &lt; 100</td></tr><tr><td rowspan=1 colspan=1>LowRLU Positive1</td><td rowspan=1 colspan=1>100tto&lt; 5,000</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>5,000 to &lt; 12,000</td></tr><tr><td rowspan=1 colspan=1>Invalid</td><td rowspan=1 colspan=1>0* or &gt; 12,000</td></tr></table>

$^ * \mathrm { A }$ zero $( 0 \mathrm { ~ x ~ } 1 0 0 0 )$ RLU result on the run report represents a value between zero and 999 RLU. RLU values less than 690 on the Panther System will be reported as invalid. 1 In the low positive range, data suggest positive results should be interpreted carefully, with the understanding that the likelihood of a false positive may be higher than a true positive.

8. Accuracy (Instrument): Not applicable.

# 9. Carry-Over:

Hologic performed a carryover study using high-target sample dilutions of $C .$ . trachomatis rRNA in STM for a final concentration of $5 \mathrm { x } 1 0 ^ { 5 }$ fg rRNA/reaction (rRNA equivalent of $2 . 5 \mathrm { x } 1 0 ^ { 5 } \mathrm { I F U / m L }$ ). Each run consisted of at least $20 \%$ high positive CT samples interspersed with negative samples. Carryover was assessed over five runs on each of six Panther instruments, with a total of 5878 negative samples and $1 5 6 0 \mathrm { C T }$ high target samples. Eleven false positive results were produced, giving an overall carryover rate of $0 . 1 9 \%$ with a $9 5 \%$ confidence interval of $0 . 1 0 \%$ to $0 . 3 3 \%$ .

# B Comparison Studies:

1. Method Comparison with Predicate Device: See Clinical Studies section below.

2. Matrix Comparison: Not applicable.

# C Clinical Studies:

Aptima CT assay performance was evaluated in a prospective clinical study, using patientcollected first catch male and female urine specimens, and patient-collected vaginal swab specimens. Specimens were collected from 4413 male and female subjects in a multicenter study. Participants were sexually active individuals at least 14 years of age, attending one of 11 participating geographically diverse clinical collection sites, with or without symptoms of an STI infection.

Up to five specimens were collected from each female subject (four patient-collected vaginal swabs, one first-catch urine), and one first catch urine specimen was collected from each male subject. All specimens were collected by the subject at the clinical sites.

Three clinical testing sites performed Aptima CT assay testing. All comparator testing was completed at a single central laboratory. The clinical performance of the Aptima CT assay with vaginal swabs and male urine was evaluated using a Patient Infected Status (PIS) algorithm, and female urine was evaluated against Composite Comparator Algorithm (CCA), testing each sample with up to three FDA-cleared NAATs for detection of CT. Specimens were categorized as “infected” (PIS) for vaginal swabs and male urine and as “positive” (CCA) for female urine. CT PIS for PVS samples was determined based on the combined PVS and urine result with two different FDA-cleared NAATs. One or more positive result obtained with each assay designates the PIS as “infected”. CT PIS for male urine and CT CCA for female urine were determined by testing each sample with up to three FDA-cleared NAATs. Specimens were categorized as PIS “infected” for male urine and as CCA “positive” for female urine if a positive result occurred in at least two of the comparator NAAT results, and as “not infected”/ “negative” if at least two of the comparator results were negative; the third (tie-breaker) NAAT was only required if the first two NAAT results were discordant.

Overall, 4247 subjects were included in the clinical performance evaluation. Ages ranged from 14-84 years of age, with the average age range of 34.5 years. Female subjects comprised $5 3 . 2 \%$ of the subject population, $9 8 . 7 \%$ of whom were not pregnant. Of the 4247 subjects, 2283 were women and 1964 were men. Symptoms were reported in 1939 subjects $( 4 5 . 7 \% )$ . Samples that originally met the inclusion criteria were excluded if deemed unsuitable or non-evaluable for testing based on the respective sample collection package insert.

Ninety-seven (97) female urine specimens were excluded from analysis, including 55 from a single site. Of these 97 excluded specimens, 54 were excluded because the PIS could not be determined, and the remaining were due to a sample handling deviation. Of the specimens collected, 6592 were processed in valid Aptima CT assay runs. Of these, 213 samples generated invalid results upon initial testing (invalid rate of $3 . 2 \%$ , $9 5 \%$ CI: $2 . 8 \% 6 - 3 . 7 \%$ ); after retesting, 31 $( 0 . 5 \% )$ samples remained invalid.

A total of 6415 specimens from the 4247 evaluable subjects were included in the analyses comparing Aptima CT assay results to the PIS or CCA interpretation: 2265 patient-collected vaginal swabs, 2186 female urine specimens, and 1964 male urine specimens.

Performance characteristics of the Aptima CT assay are shown below in Tables 5 and 6.

Table 5. Clinical Performance of the Aptima CT Assay on the Panther System Compared to PIS, by Symptom Status   

<table><tr><td rowspan=1 colspan=1>SpecimenType</td><td rowspan=1 colspan=1>SymptomStatus</td><td rowspan=1 colspan=1>n</td><td rowspan=1 colspan=1>TP</td><td rowspan=1 colspan=1>FP1</td><td rowspan=1 colspan=1>TN</td><td rowspan=1 colspan=1>FN2</td><td rowspan=1 colspan=1>Sensitivity %(95% CI)</td><td rowspan=1 colspan=1>Specificity %(95% CI)</td></tr><tr><td rowspan=1 colspan=1>PVS</td><td rowspan=1 colspan=1>Asymptomatic</td><td rowspan=1 colspan=1>1163</td><td rowspan=1 colspan=1>87</td><td rowspan=1 colspan=1>4a</td><td rowspan=1 colspan=1>1069</td><td rowspan=1 colspan=1>3c</td><td rowspan=1 colspan=1>96.7 (90.7, 98.9)</td><td rowspan=1 colspan=1>99.6 (99.0, 99.9)</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Symptomatic</td><td rowspan=1 colspan=1>1102</td><td rowspan=1 colspan=1>89</td><td rowspan=1 colspan=1>6b</td><td rowspan=1 colspan=1>1001</td><td rowspan=1 colspan=1>6f</td><td rowspan=1 colspan=1>93.7 (86.9, 97.1)</td><td rowspan=1 colspan=1>99.4 (98.7, 99.7)</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>All</td><td rowspan=1 colspan=1>2265</td><td rowspan=1 colspan=1>176</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>2070</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>95.1 (91.0, 97.4)</td><td rowspan=1 colspan=1>99.5 (99.1, 99.7)</td></tr><tr><td rowspan=1 colspan=1>MU</td><td rowspan=1 colspan=1>Asymptomatic</td><td rowspan=1 colspan=1>1136</td><td rowspan=1 colspan=1>56</td><td rowspan=1 colspan=1>1c</td><td rowspan=1 colspan=1>1078</td><td rowspan=1 colspan=1>1g</td><td rowspan=1 colspan=1>98.2 (90.7, 99.7)</td><td rowspan=1 colspan=1>99.9 (99.5, 100)</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Symptomatic</td><td rowspan=1 colspan=1>828</td><td rowspan=1 colspan=1>85</td><td rowspan=1 colspan=1>4d</td><td rowspan=1 colspan=1>738</td><td rowspan=1 colspan=1>1h</td><td rowspan=1 colspan=1>98.8 (93.7, 99.8)</td><td rowspan=1 colspan=1>99.5 (98.6, 99.8)</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>All</td><td rowspan=1 colspan=1>1964</td><td rowspan=1 colspan=1>141</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>1816</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>98.6 (95.0, 99.6)</td><td rowspan=1 colspan=1>99.7 (99.4, 99.9)</td></tr></table>

$\mathrm { P V S } =$ patient-collected vaginal swab, $\mathrm { T P } =$ true positive, $\mathrm { F P = }$ false positive, $\mathrm { T N } =$ true negative, $\mathrm { F N = }$ false negative, $\mathbf { M U } =$ male urine   
1 Specimens were retested by an alternative CT NAAT assay with the following results (# positive results / # samples tested): a $1 / 4 , ^ { \textrm { b } } 1 / 6 , ^ { \textrm { c } } 0 / 1 , ^ { \textrm { d } } 0 / 4$   
2 Specimens were retested by an alternative CT NAAT assay with the following results (# negative results / # samples tested): e 1/3, f 1/6, g 0/1, $^ \mathrm { h } 1 / 1$

The CT specimen specific positive and negative percent agreements (PPA and NPA) between the Aptima CT assay and up to three FDA-cleared NAATs with female urine specimen are shown below in Table 6.

Table 6. CT Specimen Specific Agreement for Female Urine, by Symptom Status   

<table><tr><td rowspan=1 colspan=1>SpecimenType</td><td rowspan=1 colspan=1>SymptomStatus</td><td rowspan=1 colspan=1>n</td><td rowspan=1 colspan=1>CCA+ACT+</td><td rowspan=1 colspan=1>CCA-ACT+1</td><td rowspan=1 colspan=1>CCA-ACT-</td><td rowspan=1 colspan=1>CCA+ACT-2</td><td rowspan=1 colspan=1>PPA%(95% CI)</td><td rowspan=1 colspan=1>NPA%(95% CI)</td></tr><tr><td rowspan=1 colspan=1>FU</td><td rowspan=1 colspan=1>Asymptomatic</td><td rowspan=1 colspan=1>1136</td><td rowspan=1 colspan=1>77</td><td rowspan=1 colspan=1>2a</td><td rowspan=1 colspan=1>1055</td><td rowspan=1 colspan=1>2c</td><td rowspan=1 colspan=1>97.5 (91.2, 99.3)</td><td rowspan=1 colspan=1>99.8 (99.3, 99.9)</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Symptomatic</td><td rowspan=1 colspan=1>1050</td><td rowspan=1 colspan=1>74</td><td rowspan=1 colspan=1>7b</td><td rowspan=1 colspan=1>968</td><td rowspan=1 colspan=1>1d</td><td rowspan=1 colspan=1>98.7 (92.8, 99.8)</td><td rowspan=1 colspan=1>99.3 (98.5, 99.7)</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>All</td><td rowspan=1 colspan=1>2186</td><td rowspan=1 colspan=1>151</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>2023</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>98.1 (94.4, 99.3)</td><td rowspan=1 colspan=1>99.6 (99.2, 99.8)</td></tr></table>

$\mathrm { F U } =$ female urine, $\mathrm { C C A } =$ composite comparator algorithm, $\mathbf { A C T } =$ Aptima CT assay, $\mathrm { P P A } =$ positive percent   
agreement, $\mathrm { \Delta N P A = }$ negative percent agreement, $\mathrm { C I } =$ confidence interval   
1 Specimens were retested by an alternative CT NAAT assay with the following results (# positive results / # samples   
tested): $^ { \mathrm { a } } 0 / 2$ , $^ { \mathrm { b } } 2 / 7$   
2 Specimens were retested by an alternative CT NAAT assay with the following results $\#$ negative results / #   
samples tested): $^ { \mathrm { c } } 2 / 2 , ^ { \mathrm { d } } 1 / 1$

The sensitivity of the Aptima CT assay on the Panther system in female urine specimens was also assessed against vaginal swabs tested on three NAATs. The data showed that the Aptima CT assay performance female urine specimens is $9 . 0 \%$ lower when compared to vaginal swab tested on three NAATs.

D Clinical Cut-Off: Not applicable.

# E Expected Values/Reference Range:

For evaluable specimens, the observed positivity rates from testing with Aptima CT assay were $7 . 3 \%$ (160/2186) for female urine specimens, $8 . 2 \%$ (186/2265) for patient-collected vaginal swab specimens, and $7 . 4 \%$ (146/1964) for male urine specimens.

# VIII Proposed Labeling:

The labeling is acceptable and supports the finding of substantial equivalence for this device.

# IX Conclusion:

The submitted information in this premarket notification is complete and supports a substantial equivalence decision.