# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATION DECISION SUMMARY

I Background Information:

# B Applicant

BD Integrated Diagnostic Solutions

# C Proprietary and Established Names

BD Respiratory Viral Panel (BD RVP) for BD MAX System; BD Respiratory Viral Panel-SCV2 (BD RVP-SCV2) for BD MAX System

# D Regulatory Information

<table><tr><td>Product Code(s)</td><td>Classification</td><td>Regulation Section 21 CFR 866.3981 - Device</td><td>Panel</td></tr><tr><td>QOF</td><td>Class II</td><td>To Detect And Identify Nucleic Acid Targets In Respiratory Specimens From Microbial Agents That Cause The SARS- Cov-2 Respiratory Infection And Other Microbial Agents When In A Multi-Target Test</td><td>MI - Microbiology</td></tr><tr><td>QQX</td><td>Class II</td><td>21 CFR 866.3981 - Device to detect and identify nucleic acid targets in respiratory specimens from microbial agents that cause the SARS-CoV-2 respiratory infection and other microbial agents when in a multi-target test</td><td>MI - Microbiology</td></tr></table>

# II Submission/Device Overview:

# A Purpose for Submission:

# BD Respiratory Viral Panel (BD RVP)

The purpose of this submission is to demonstrate that the BD Respiratory Viral Panel (SARSCoV-2, Influenza A, Influenza B and RSV) for use on the BD MAX System, is substantially equivalent to the BioFire Respiratory Panel 2.1 (RP2.1) (DEN200031) and to obtain clearance for the BD Respiratory Viral Panel (SARS-CoV-2, Influenza A, Influenza B and RSV) Assay.

# BD Respiratory Viral Panel-SCV2 (BD RVP-SCV2)

The purpose of this submission is to demonstrate that the BD Respiratory Viral Panel-SCV2 (SARS-CoV-2) for use on the BD MAX System, is substantially equivalent to the BioFire Respiratory Panel 2.1 (RP2.1) (DEN200031) and to obtain clearance for the BD Respiratory Viral Panel-SCV2 (SARS-CoV-2) Assay.

# B Measurand:

# BD Respiratory Viral Panel (BD RVP)

The BD Respiratory Viral Panel detects and identifies nucleic acids from the following pathogens: Severe Acute Respiratory Syndrome (SARS)-Coronavirus-2 (SARS-CoV-2), Influenza A, Influenza B, and Respiratory Syncytial Virus (RSV).

For each specimen, the amplification and detection of SARS-CoV-2, influenza A, influenza B, RSV and the internal control, human RNase P, occurs in a single reaction with the use of multiplexed primers and probes. The multiplexed primers and probes target RNA from the nucleocapsid phosphoprotein gene (N1 and N2 regions) of SARS-CoV-2, matrix (M1) gene of influenza A, matrix (M1) gene and hemagglutinin (HA) gene of influenza B, N gene and M gene from RSV and the RNase $\mathrm { P }$ gene from the human genome.

A total of eighteen (18) primers are involved in the DNA amplification process and nine (9) molecular probes are involved in the detection process of the BD Respiratory Viral Panel for BD MAX System.

# BD Respiratory Viral Panel-SCV2 (BD RVP-SCV2)

The BD Respiratory Viral Panel-SCV2 detects and identifies nucleic acids from the following pathogens: Severe Acute Respiratory Syndrome (SARS)-Coronavirus-2 (SARS-CoV-2).

The BD Respiratory Viral Panel-SCV2 is produced using the same manufacturing processes and assay reagents as the Respiratory Viral Panel, but requires a different Assay Definition File (ADF) which, via software-mediated masking, only reports results for the SARS-CoV-2 target.

# C Type of Test:

Multiplex nucleic acid assay for use with the BD MAX System for the qualitative detection of viral pathogens in individuals with signs and symptoms of respiratory tract infection consistent with COVID-19, Influenza A, Influenza B, or RSV.

# III Intended Use/Indications for Use:

# A Intended Use(s):

BD Respiratory Viral Panel for BD MAX System:

BD Respiratory Viral Panel for BD MAX System is an automated multiplexed real-time reverse transcriptase polymerase chain reaction (RT- PCR) test intended for the simultaneous, qualitative detection and differentiation of severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), influenza A, influenza B, and/or respiratory syncytial virus (RSV) nucleic acid in nasopharyngeal swab (NPS) and anterior nasal swab (ANS) specimens from individuals with signs and symptoms of respiratory tract infection. Clinical signs and symptoms of respiratory tract infection due to SARS-CoV-2, influenza, and RSV can be similar.

BD Respiratory Viral Panel for BD MAX System is intended for use as an aid in the differential diagnosis of SARS-CoV-2, influenza A, influenza B, and/or RSV infection if used in conjunction with other clinical and epidemiological information, and laboratory findings. SARS-CoV- 2, influenza A, influenza B, and RSV viral nucleic acid are generally detectable in NPS and ANS specimens during the acute phase of infection.

Positive results do not rule out co-infection with other organisms. The agent(s) detected by the BD Respiratory Viral Panel for BD MAX System may not be the definitive cause of disease.

Negative results do not preclude SARS-CoV-2, influenza A, influenza B, and/or RSV infection.

The results of this test should not be used as the sole basis for diagnosis, treatment, or other patient management decisions.

BD Respiratory Viral Panel-SCV2 for BD MAX System:

BD Respiratory Viral Panel-SCV2 for BD MAX System is an automated multiplexed real-time reverse transcriptase polymerase chain reaction (RT-PCR) test intended for the simultaneous, qualitative detection of severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) viral nucleic acid in nasopharyngeal swab (NPS) and anterior nasal swab (ANS) specimens from individuals with signs and symptoms of respiratory tract infection. SARS-CoV-2 viral RNA is generally detectable in NPS and ANS specimens during the acute phase of infection.

The BD Respiratory Viral Panel-SCV2 for BD MAX System is intended for use as an aid in the diagnosis of SARS-CoV-2 infection if used in conjunction with other clinical and epidemiological information, and laboratory findings.

Positive results do not rule out co-infection with other organisms. The agent detected by the BD Respiratory Viral Panel-SCV2 for BD MAX System may not be the definitive cause of disease.

Negative results do not preclude SARS-CoV-2 infection.

The results of this test should not be used as the sole basis for diagnosis, treatment, or other patient management decisions.

B Indication(s) for Use: See Intended Use(s)

C Special Conditions for Use Statement(s): Rx - For Prescription Use Only

For in vitro diagnostic use only

D Special Instrument Requirements:

The BD Respiratory Viral Panel Assay and BD Respiratory Viral Panel-SCV2 Assay are performed on the BD MAX System.

# IV Device/System Characteristics:

A Device Description:

# BD Respiratory Viral Panel (BD RVP) and BD Respiratory Viral Panel-SCV2 (BD RVPSCV2)

The BD Respiratory Viral Panel (BD RVP) and BD Respiratory Viral Panel-SCV2 (BD RVPSCV2) along with the BD MAX System comprise an instrument with associated hardware and accessories, disposable microfluidic cartridges, master mixes, unitized reagent strips, and extraction reagents. The instrument automates sample preparation including target lysis, Total Nucleic Acid (TNA) extraction and concentration, reagent rehydration, target nucleic acid amplification and detection using real-time PCR. The assay includes a Sample Processing Control (SPC) that is present in the Extraction Tube. The SPC monitors RNA extraction steps, thermal cycling steps, reagent integrity and the presence of inhibitory substances. The BD MAX System software automatically interprets test results.

For the BD Respiratory Viral Panel for BD MAX System the analytes reported are SARS-CoV-2, Influenza A, Influenza B and RSV. For the BD Respiratory Viral Panel-SCV2 for BD MAX System, the analyte is SARS-CoV-2. The difference in analyte(s) reported is determined by an assay definition file (ADF).

# B Principle of Operation:

# Test Principle

The BD Respiratory Viral Panel for BD MAX System and BD Respiratory Viral Panel-SCV2 for BD MAX System assays are designed for use with a nasopharyngeal or anterior nasal swabs collected in BD Universal Viral Transport System (UVT) or Copan Universal Transport Media System (UTM). Once collected, the UVT/UTM patient sample is vortexed and $7 5 0 \mathrm { u l }$ is transferred to the BD Molecular RVP Sample Buffer Tube (SBT) provided with the BD

Respiratory Viral Panel for BD MAX System. placed in the BD MAX System. For all sample types the SBTs are vortexed and then loaded into the BD MAX system along with the Unitized Reagent Strips, Master Mix, Extraction Tubes, and PCR Cartridges. No further operator intervention is necessary.

The BD RVP Unitized Reagent Strip contains a combination of lytic and extraction reagents designed to perform cell lysis and TNA extraction. Nucleic acids released from the target organisms are captured on magnetic affinity beads. The beads, together with the bound nucleic acids, are washed and the nucleic acids are eluted by a combination of heat and pH variation. Eluted TNA is added to neutralization buffer, mixed, and transferred to BD Respiratory Viral Panel master mix for rehydration. After reconstitution, the BD MAX System dispenses a fixed volume of RT-PCR-ready solution containing extracted nucleic acids into the PCR Cartridge. Microvalves on the cartridge are sealed by the system prior to initiating PCR in order to contain the amplification mixture and thus prevent evaporation and contamination.

The amplified cDNA targets are detected using hydrolysis (TaqMan) probes, labeled at one end with a fluorescent reporter dye (fluorophore), and at the other end, with a quencher moiety. Probes labeled with different fluorophores are used to detect the target analytes in different optical channels of the BD MAX System. When the probes are in their native state, the fluorescence of the fluorophore is quenched due to its proximity to the quencher. However, in the presence of target cDNA, the probes hybridize to their complementary sequences and are hydrolyzed by the $5 ' - 3 '$ exonuclease activity of the DNA polymerase as it synthesizes the nascent strand along the cDNA template. As a result, the fluorophores are separated from the quencher molecules and fluorescence is emitted. The amount of fluorescence detected in the optical channels is directly proportional to the quantity of the corresponding probe that is hydrolyzed. The BD MAX System monitors these signals at each cycle of the PCR and interprets the data at the end of the reaction to provide qualitative test results for each analyte.

Identification of SARS-CoV-2, influenza A, influenza B, RSV, and RNAse P occurs by the use of target-specific primers and fluorescent-labeled probes that hybridize to conserved regions in the viral genomes (Table 1).

Table 1. Assay Primer and Probe Targets   

<table><tr><td rowspan=1 colspan=1>Analyte</td><td rowspan=1 colspan=1>Gene Targeted</td><td rowspan=1 colspan=1>InstrumentChannel</td></tr><tr><td rowspan=1 colspan=1>SARS-CoV-2</td><td rowspan=1 colspan=1>Nucleocapsid gene (N1 and N2 regions)</td><td rowspan=1 colspan=1>FAM</td></tr><tr><td rowspan=1 colspan=1>Influenza A Virus</td><td rowspan=1 colspan=1>Matrix (M1) Gene</td><td rowspan=1 colspan=1>Cy5</td></tr><tr><td rowspan=1 colspan=1>Influenza B Virus</td><td rowspan=1 colspan=1>Matrix (M1) &amp; Hemagglutinin (HA) Genes</td><td rowspan=1 colspan=1>Cy5.5</td></tr><tr><td rowspan=1 colspan=1>Respiratory Syncytial Virus A/B</td><td rowspan=1 colspan=1>N &amp; M Genes</td><td rowspan=1 colspan=1>VIC</td></tr><tr><td rowspan=1 colspan=1>Extraction Control</td><td rowspan=1 colspan=1>Human RNAse P gene</td><td rowspan=1 colspan=1>ROX</td></tr></table>

# C Instrument Description Information:

1. Instrument Name: BD MAX System, software version 5.14A

2. Specimen Identification: Specimen identification can be entered either via barcode scanning or by manual entry.

3. Specimen Sampling and Handling:

Use of the BD Respiratory Viral Panel or the BD Respiratory Viral Panel-SCV2 requires either a nasopharyngeal swab (NPS) or anterior nasal swab (ANS) to be collected according to standard procedures and expressed into BD Universal Transport System (UVT) or Copan Universal Transport Media System (UTM). Note: the BD UVT and the Copan UTM are a similar device with identical chemical formulations. See Section B. Comparison Studies/Matrix Comparison.

$7 5 0 \mathrm { u L }$ of patient sample is transferred to the sample buffer tube (SBT) provided with the BD Respiratory Viral Panel for BD MAX System kit, which contains $4 . 5 \%$ Triton X-100 Reduced, and placed in the BD MAX sample rack. Unitized Reagent Strips are placed in the sample rack and securely seated. Foil-sealed dried Extraction Tubes and Master Mix Tubes are snapped into the appropriate positions on each Unitized Reagent Strip. The sample rack is placed in the BD MAX instrument along with a BD PCR cartridge. The BD MAX System automates running of the assay and result reporting.

The BD Respiratory Viral Panel Assay Extraction Tube and Unitized Reagent Strip (URS) contain a combination of lytic and extraction reagents designed to perform cell lysis and Total Nucleic Acid (TNA) extraction. $6 0 0 \mu \mathrm { L }$ is transferred from the sample buffer tube into the URS strip for cell lysis. Following cell lysis, the released nucleic acids are captured by magnetic affinity beads. The beads with the bound nucleic acids are washed and the TNA is eluted, and neutralization buffer added. $1 2 . 5 \mu \mathrm { L }$ of eluted TNA is then transferred to BD Respiratory Viral Panel Master Mix containing RT-polymerase, dNTPs, primers and probes. This final rehydrated Master Mix is transferred to a BD PCR cartridge for the initiation of reverse transcriptase PCR mediated conversion of RNA to cDNA and subsequent real-time PCR.

4. Calibration:

The BD MAX system preventative maintenance is performed twice per year by a BD Field Service Engineer.

5. Quality Control:

# Control Materials

# a) External Controls

External Control materials (assay run controls) are not provided as part of the BD Respiratory Viral Panel for BD MAX System. External Positive and Negative Controls are not used by the BD MAX System software for the purpose of sample test result interpretation. External Controls are treated as if they were patient samples. BD recommends the use of Microbiologics controls (Table 2) which were utilized during assay validation.

Laboratories must establish the number, type and frequency of testing control materials according to guidelines or requirements of local, provincial, state, federal, and/or country regulations or accreditation organizations in order to monitor the effectiveness of the entire analytical process.

Table 2. Recommended External Controls   

<table><tr><td rowspan=1 colspan=1>External Controls</td><td rowspan=1 colspan=1>Catalog #</td></tr><tr><td rowspan=1 colspan=1>Microbiologics Helix Elite Synthetic StandardSARS-CoV-2 Synthetic RNA (N gene Targets)</td><td rowspan=1 colspan=1>HE0060S</td></tr><tr><td rowspan=1 colspan=1>Microbiologics Helix Elite InactivatedSARS-CoV-2 Whole Virus (Pellet)</td><td rowspan=1 colspan=1>HE0065N</td></tr><tr><td rowspan=1 colspan=1>Microbiologics Helix Elite Inactivated SARS-CoV-2 Whole Virus(Swab)</td><td rowspan=1 colspan=1>HE0066NS</td></tr><tr><td rowspan=1 colspan=1>Microbiologics Helix Elite Inactivated StandardInactivated influenza A/B andRespiratory Syncytial Virus</td><td rowspan=1 colspan=1>HE0044N</td></tr><tr><td rowspan=1 colspan=1>Microbiologics Helix Elite Inactivated StandardNegative Cellularity Control (Pellet)</td><td rowspan=1 colspan=1>HE0058N</td></tr><tr><td rowspan=1 colspan=1>Microbiologics Helix Elite Flu/RSV/SARS-CoV-2 Control Panel(Inactivated Swab)</td><td rowspan=1 colspan=1>8246</td></tr><tr><td rowspan=1 colspan=1>Microbiologics Helix Elite Inactivated StandardNegative Cellularity Control (Swab)</td><td rowspan=1 colspan=1>HE0067NS</td></tr></table>

# b) Extraction and Internal Amplification Control

The human RNase P gene is present in all appropriately collected patient samples. It is coamplified with SARS-CoV-2, influenza A, influenza B and RSV gene targets (if present) and will serve as both an endogenous nucleic acid extraction control and internal amplification control. In the event that SARS-CoV-2, influenza A, influenza B, RSV are negative, an RNase P result must be positive for the SARS-CoV-2, influenza A, influenza B, RSV results to be valid negative results. When either SARS-CoV-2, influenza A, influenza B and/or RSV target results are positive, RNase P result is ignored. An Unresolved (UNR) result is indicative of specimen-associated inhibition or reagent failure.

# V Substantial Equivalence Information:

A Predicate Device Name(s): BioFire Respiratory Panel 2.1 (RP2.1)   
B Predicate 510(k) Number(s): DEN200031   
C Comparison with Predicate(s):

Table 3. BD Respiratory Viral Panel for BD MAX System Substantial Equivalence Comparison   

<table><tr><td>Device &amp; Predicate</td><td>K230956</td><td>DEN200031</td></tr><tr><td></td><td></td><td></td></tr></table>

<table><tr><td colspan="1" rowspan="2">Device Trade Name</td><td colspan="1" rowspan="2">BD Respiratory Viral Panel for BDMAX System</td><td colspan="1" rowspan="2">BioFire Respiratory Panel 2.1(RP2.1)</td><td></td></tr><tr><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Regulation Number</td><td colspan="1" rowspan="1">21 CFR 866.3981</td><td colspan="1" rowspan="1">Same</td><td colspan="1" rowspan="3"></td></tr><tr><td colspan="1" rowspan="1">Regulation Name</td><td colspan="1" rowspan="1">Multi-Target Respiratory SpecimenNucleic Acid Test Including SARS-Cov-2 And Other Microbial Agents</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Product Code</td><td colspan="1" rowspan="1">QOF</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">IntendedUse/Indications ForUse</td><td colspan="1" rowspan="1">BD Respiratory Viral Panel for BDMAX System is an automatedmultiplexed real-time reversetranscriptase polymerase chainreaction (RT- PCR) test intended forthe simultaneous, qualitative detectionand differentiation of severe acuterespiratory syndrome coronavirus 2(SARS-CoV-2), influenza A,influenza B, and/or respiratorysyncytial virus (RSV) nucleic acid innasopharyngeal swab (NPS) andanterior nasal swab (ANS) specimensfrom individuals with signs andsymptoms of respiratory tractinfection. Clinical signs andsymptoms of respiratory tractinfection due to SARS-CoV-2,influenza, and RSV can be similar.BD Respiratory Viral Panel for BDMAX System is intended for use as anaid in the differential diagnosis ofSARS-CoV-2, influenza A, influenzaB, and/or RSV infection if used inconjunction with other clinical andepidemiological information, andlaboratory findings. SARS-CoV- 2,influenza A, influenza B, and RSVviral nucleic acid are generallydetectable in NPS and ANSspecimens during the acute phase ofinfection.Positive results do not rule out co-infection with other organisms. Theagent(s) detected by the BDRespiratory Viral Panel for BD MAXSystem may not be the definitivecause of disease.Negative results do not precludeSARS-CoV-2, influenza A, influenzaB, and/or RSV infection.</td><td colspan="2" rowspan="1">The BIOFIRE RespiratoryPanel 2.1 (RP2.1) is a PCRbased multiplexed nucleic acidtest intended for use with theBIOFIRE FilmArray 2.0 orBIOFIRE FilmArray Torchsystems for the simultaneousqualitative detection andidentification of multiplerespiratory viral and bacterialnucleic acids innasopharyngeal swabs (NPS)obtained from individualssuspected of respiratory tractinfections, including COVID-19.The following organism typesand subtypes are identifiedusing the BIOFIRE RP2.1:Adenovirus,Coronavirus 229ECoronavirus HKU1Coronavirus NL63Coronavirus OC43Severe AcuteRespiratorySyndromeCoronavirus-2(SARS-CoV-2)Human MetapneumovirusHuman Rhinovirus/EnterovirusInfluenza A, includingsubtypes H1, H1-2009, andH3Influenza BParainfluenza Virus 1Parainfluenza Virus 2Parainfluenza Virus 3Parainfluenza Virus 4Respiratory Syncytial VirusBordetella parapertussis(IS1001)Bordetella pertussis (ptxP)Chlamydia pneumoniae, andMycoplasma pneumoniaeNucleic acids from the</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">used as the sole basis for diagnosis,treatment, or other patientmanagement decisions.</td><td colspan="1" rowspan="1">organisms identified by thistest are generally detectable inNPS specimens during theacute phase of infection. Thedetection and identification ofspecific viral and bacterialnucleic acids from individualsexhibiting signs and/orsymptoms of respiratoryinfection is indicative of thepresence of the identifiedmicroorganism and aids in thediagnosis of respiratoryinfection if used inconjunction with other clinicaland epidemiologicalinformation. The results of thistest should not be used as thesole basis for diagnosis,treatment, or other patientmanagement decisions.Negative results in the settingof a respiratory illness may bedue to infection with pathogensthat are not detected by thistest, or lower respiratory tractinfection that may not bedetected by an NPS specimen.Positive results do not rule outcoinfection with otherorganisms. The agent(s)detected by the BIOFIRERP2.1 may not be the definitecause of disease. Additionallaboratory testing (e.g.,bacterial and viral culture,immune-fluorescence, andradiography) may be necessarywhen evaluating a patient withpossible respiratory tractinfection.</td></tr><tr><td colspan="1" rowspan="1">Condition for use</td><td colspan="1" rowspan="1">For prescription useFor in vitro diagnostic use only.</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Sample Types</td><td colspan="1" rowspan="1">Nasopharyngeal swab specimen Nasalswab specimen</td><td colspan="1" rowspan="1">Nasopharyngeal swab specimen</td></tr><tr><td colspan="1" rowspan="1">Patient Population</td><td colspan="1" rowspan="1">Individuals suspected of respiratoryviral infection, including COVID-19.</td><td colspan="1" rowspan="1">Individuals suspected ofrespiratory tract infections,including COVID-19</td></tr><tr><td colspan="1" rowspan="1">Analyte Targets</td><td colspan="1" rowspan="1">The following organism types areidentified using the BDRespiratory Viral Panel for BDMAX System:Severe Acute RespiratorySyndrome Coronavirus (SARS-CoV-2)</td><td colspan="1" rowspan="1">The following organism typesand subtypes are identifiedusing the BioFire RP2.1:•  AdenovirusCoronavirus 229ECoronavirus HKU1,Coronavirus NL63</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Influenza AInfluenza B andRespiratory Syncytial Virus(RSV)</td><td colspan="1" rowspan="1">Coronavirus OC43Severe Acute RespiratorySyndrome Coronavirus(SARS-CoV-2)Human MetapneumovirusHumanRhinovirus/EnterovirusInfluenza A,including subtypesH1, H1-2009, andH3,Influenza B,Parainfluenza Virus 1,Parainfluenza Virus 2,Parainfluenza Virus 3,Parainfluenza Virus 4,Respiratory SyncytialVirus,Bordetella parapertussis(IS1001),Bordetella pertussis(ptxP),Chlamydia pneumoniae,andMycoplasma pneumoniae</td></tr><tr><td colspan="1" rowspan="1">Sample PreparationProcedure</td><td colspan="1" rowspan="1">Automated by BD MAX System</td><td colspan="1" rowspan="1">Automated by BioFireFilmArray 2.0 or BioFireFilmArray Torch systems</td></tr><tr><td colspan="1" rowspan="1">AmplificationTechnology</td><td colspan="1" rowspan="1">Real-Time PCR</td><td colspan="1" rowspan="1">Nested multiplex RT-PCR</td></tr><tr><td colspan="1" rowspan="1">Analyte</td><td colspan="1" rowspan="1">RNA</td><td colspan="1" rowspan="1">RNA/DNA</td></tr><tr><td colspan="1" rowspan="1">Detection Chemistry</td><td colspan="1" rowspan="1">Paired reporter and quencherfluorescence labeled probes (TaqManTechnology) using fluorescenceresonance energy transfer</td><td colspan="1" rowspan="1">Two Step Nested multiplexPCR: -Reverse transcription,followed by a multiplexed firststage PCR reaction (PCR1).Multiple simultaneous secondstage PCR reactions (PCR2) toamplify sequences within thePCR1 products usingfluorescence double strandedbinding dye. Endpoint meltingcurve data to detect targetspecific amplicons</td></tr><tr><td colspan="1" rowspan="1">Control used</td><td colspan="1" rowspan="1">The RNA Internal Control(RNase P)2.External Positive and negativecontrols</td><td colspan="1" rowspan="1">Two process controls:1RNA Process Control (IC)PCR2 Control (A positiveresult indicates that PCR2was successful)</td></tr><tr><td colspan="1" rowspan="1">Result Analysis</td><td colspan="1" rowspan="1">Based on PCR cycle thresholdanalysis</td><td colspan="1" rowspan="1">Endpoint melting curve data todetect target-specificamplicons</td></tr><tr><td colspan="1" rowspan="1">Test Interpretation</td><td colspan="1" rowspan="1">Automated test interpretation and</td><td colspan="1" rowspan="1">Same</td></tr></table>

Table 4. BD Respiratory Viral Panel-SCV2 for BD MAX System Substantial Equivalence Comparison   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>report generation. User cannot accessraw data.</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Time to Result</td><td rowspan=1 colspan=1>About 2 hours</td><td rowspan=1 colspan=1>About 45 min</td></tr></table>

<table><tr><td rowspan=1 colspan=1>Device &amp; PredicateDevice(s):</td><td rowspan=1 colspan=1>K230956</td><td rowspan=1 colspan=2>DEN200031</td><td rowspan=6 colspan=1>DEN200031BioFire Respiratory Panel 2.1(RP2.1)SameSameQOFThe BIOFIRE RespiratoryPanel 2.1 (RP2.1) is a PCRbased multiplexed nucleicacid test intended for usewith the BIOFIRE FilmArray2.0 or BIOFIRE FilmArrayTorch systems for thesimultaneous qualitativedetection and identificationof multiple respiratory viraland bacterial nucleic acids innasopharyngeal swabs (NPS)obtained from individualssuspected of respiratory tractinfections, including COVID-19.The following organismtypes and subtypes areidentified using the BIOFIRERP2.1:AdenovirusCoronavirus 229E•   Coronavirus HKU1Coronavirus NL63Coronavirus OC43Severe AcuteRespiratory SyndromeCoronavirus (SARS-CoV-2)HumanMetapneumovirusHumanRhinovirus/EnterovirusInfluenza A, includingsubtypes H1, H1-2009,and H3Influenza BParainfluenza Virus 1</td></tr><tr><td rowspan=1 colspan=1>Device Trade Name</td><td rowspan=1 colspan=1>BD Respiratory Viral Panel-SCV2for BD MAX System</td><td rowspan=1 colspan=2>BioFire Respiratory Panel 2.1(RP2.1)</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Regulation Number</td><td rowspan=1 colspan=1>21 CFR 866.3981</td><td rowspan=1 colspan=2>Same</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Regulation Name</td><td rowspan=1 colspan=1>Multi-Target Respiratory SpecimenNucleic Acid Test Including Sars-Cov-2 And Other Microbial Agents</td><td rowspan=1 colspan=2>Same</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Product Code</td><td rowspan=1 colspan=1>QOF</td><td rowspan=1 colspan=2>QOF</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>IntendedUse/Indications ForUse</td><td rowspan=1 colspan=1>BD Respiratory Viral Panel-SCV2 forBD MAX System is an automatedmultiplexed real-time reversetranscriptase polymerase chainreaction (RT-PCR) test intended forthe simultaneous, qualitative detectionof severe acute respiratory syndromecoronavirus 2 (SARS-CoV-2) viralnucleic acid in nasopharyngeal swab(NPS) and anterior nasal swab (ANS)specimens from individuals with signsand symptoms of respiratory tractinfection. SARS-CoV-2 viral RNA isgenerally detectable in NPS and ANSspecimens during the acute phase ofinfection.The BD Respiratory Viral Panel-SCV2 for BD MAX System isintended for use as an aid in thediagnosis of SARS-CoV-2 infection ifused in conjunction with other clinicaland epidemiological information, andlaboratory findings.Positive results do not rule out co-infection with other organisms. Theagent detected by the BD RespiratoryViral Panel-SCV2 for BD MAXSystem may not be the definitivecause of disease.Negative results do not precludeSARS-CoV-2 infection.</td><td></td><td></td></tr></table>

<table><tr><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2">The results of this test should not beused as the sole basis for diagnosis,treatment, or other patientmanagement decisions.</td><td colspan="1" rowspan="1">Parainfluenza Virus 2Parainfluenza Virus 3Parainfluenza Virus 4Respiratory SyncytialVirusBordetella parapertussis(IS1001)Bordetella pertussis(ptxP)Chlamydia pneumoniae,andMycoplasmapneumoniaeNucleic acids from therespiratory viral and bacterialorganisms identified by thistest are generally detectablein NPS specimens during the</td></tr><tr><td colspan="1" rowspan="1">acute phase of infection. Thedetection and identification ofspecific viral and bacterialnucleic acids fromindividuals exhibiting signsand/or symptoms ofrespiratory infection isindicative of the presence ofthe identified microorganismand aids in the diagnosis ofrespiratory infection if usedin conjunction with otherclinical and epidemiologicalinformation. The results ofthis test should not be used asthe sole basis for diagnosis,treatment, or other patientmanagement decisions.Negative results in the settingof a respiratory illness may bedue to infection with pathogensthat are not detected by thistest, or lower respiratory tractinfection that may not bedetected by an NPS specimen.Positive results do not rule outcoinfection with otherorganisms. The agent(s)detected by the BIOFIRERP2.1 may not be the definitecause of disease. Additionallaboratory testing (e.g.,bacterial and viral culture,immunofluorescence, andradiography) may be necessarywhen evaluating a patient withpossible respiratory tractinfection.</td></tr><tr><td colspan="1" rowspan="1">Condition for use</td><td colspan="1" rowspan="1">For prescription use</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">EY</td><td></td><td></td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">For in vitro diagnostic use only.</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Sample Types</td><td colspan="1" rowspan="1">Nasopharyngeal swab specimen Nasalswab specimen</td><td colspan="1" rowspan="1">Nasopharyngeal swab specimen</td></tr><tr><td colspan="1" rowspan="1">Patient Population</td><td colspan="1" rowspan="1">Individuals suspected of COVID-19by their healthcare provider</td><td colspan="1" rowspan="1">Individuals suspected ofrespiratory tract infections,including COVID-19</td></tr><tr><td colspan="1" rowspan="1">Analyte Targets</td><td colspan="1" rowspan="1">SARS-CoV-2</td><td colspan="1" rowspan="1">The following organism typesand subtypes are identifiedusing the BioFire RP2.1:Adenovirus,Coronavirus 229E,Coronavirus HKU1,Coronavirus NL63,Coronavirus OC43,Severe Acute RespiratorySyndrome Coronavirus(SARS-CoV-2),HumanMetapneumovirus,HumanRhinovirus/Enterovirus,Influenza A, includingsubtypes H1, H1-2009,and H3,Influenza B,Parainfluenza Virus 1,Parainfluenza Virus 2,Parainfluenza Virus 3,Parainfluenza Virus 4,Respiratory   SyncytialVirus,Bordetella parapertussis(IS1001),Bordetella     pertussis(ptxP),Chlamydia pneumoniae,andMycoplasma pneumoniae</td></tr><tr><td colspan="1" rowspan="1">Sample PreparationProcedure</td><td colspan="1" rowspan="1">Automated by BD MAX System</td><td colspan="1" rowspan="1">Automated by BioFireFilmArray 2.0 or BioFireFilmArray Torch systems</td></tr><tr><td colspan="1" rowspan="1">AmplificationTechnology</td><td colspan="1" rowspan="1">Real-Time PCR</td><td colspan="1" rowspan="1">Nested multiplex RT-PCR</td></tr><tr><td colspan="1" rowspan="1">Analyte</td><td colspan="1" rowspan="1">RNA</td><td colspan="1" rowspan="1">RNA/DNA</td></tr><tr><td colspan="1" rowspan="1">Detection Chemistry</td><td colspan="1" rowspan="1">Paired reporter and quencherfluorescence labeled probes (TaqManTechnology) using fluorescenceresonance energy transfer</td><td colspan="1" rowspan="1">Two Step Nested multiplexPCR: -Reverse transcription,followed by a multiplexed firststage PCR reaction (PCR1).Multiple simultaneous secondstage PCR reactions (PCR2) toamplify sequences within thePCR1 products using doublestranded binding dye. Endpointmelting curve data to detecttarget specific amplicons.</td></tr><tr><td colspan="1" rowspan="1">Control used</td><td colspan="1" rowspan="1">1. The RNA Internal Control (RNaseP2. External Positive and negativecontrols</td><td colspan="1" rowspan="1">Two process controls:1. RNA Process Control (IC)2. PCR2 Control (A positiveresult indicates that PCR2 wassuccessful)</td></tr><tr><td colspan="1" rowspan="1">Result Analysis</td><td colspan="1" rowspan="1">Based on PCR cycle thresholdanalysis</td><td colspan="1" rowspan="1">Endpoint melting curve data todetect target-specificamplicons</td></tr><tr><td colspan="1" rowspan="1">Test Interpretation</td><td colspan="1" rowspan="1">Automated test interpretation andreport generation. User cannot accessraw data.</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Time to Result</td><td colspan="1" rowspan="1">About 2 hours</td><td colspan="1" rowspan="1">About 45 min</td></tr></table>

# VI Standards/Guidance Documents Referenced:

# Quality System

• ISO 13485:2016-Medical devices -- Quality management systems   
• 21 CFR 820: Quality System Regulations (QSR)   
• IVDR 2017/746: REGULATION (EU) 2017/746 OF THE EUROPEAN PARLIAMENT AND OF THE COUNCIL of 5 April 2017 on in vitro diagnostic medical devices and repealing Directive 98/79/EC and Commission Decision 2010/227/EU

# Regulatory

The 510(k) Program: Evaluating Substantial Equivalence in Premarket Notifications [510(k)], Guidance for Industry and Food and Drug Administration Staff (July 2014)

Electronic Submission Template for Medical Device 510(k) Submissions, Guidance for Industry and Food and Drug Administration Staff (September 2022)

# Risk Management

ISO 14971: 2019 - Medical devices — Application of risk management to medical devices

# Labeling

• ISO 15223-1: 2021 (“Medical devices -Symbols to be used with medical device labels, labelling, and information to be supplied -Part 1: General requirements”)   
• ISO 20417:2021 - Medical devices — Information to be supplied by the manufacturer   
• 21 CFR 809 Subpart B –Labeling before the device is shipped in interstate commerce.   
• Unique Device Identification: Policy Regarding Compliance Dates for Class I and Unclassified Devices, Direct Marking, and Global Unique Device Identification

# Database Requirements for Certain Devices Guidance for Industry and Food and Drug Administration Staff (July 2022)

# Human Factor

• IEC 62366-1: 2015 Ed. 1.1 (2020) – Medical Devices - Part 1 Application of usability engineering to medical devices - Edition 1.1 Applying Human Factor and Usability Engineering to Medical Devices: Guidance for Industry and Food and Drug Administration Staff (February 2016)

# Design / Performance

ISO DTS 5798: Quality Practice for detection of Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2) by nucleic acid amplification methods   
• CLSI EP5-A3: Evaluation of Precision Performance of Quantitative Measurement Methods; Approved Guideline-Second Edition.   
• CLSI EP12-A2: User Protocol for Evaluation of Qualitative test Performance: Approved Guideline-Second Edition.   
• CLSI EP07-A2: Interference Testing in Clinical Chemistry: Approved GuidelineSecond Edition   
• CLSI EP17-A2: Protocols for Determination of Limits of Detection and Limits of Quantitation: Approved Guideline-Second Edition.   
• CLSI MM13-A: Collection, Transport, Preparation, and Storage of Specimens for Molecular Methods; Approved Guideline.   
• CLSI MM17-A: Verification and Validation of Multiplex Nucleic Acid Assays.   
• MDCG 2021-21 August 2021: Guidance on performance evaluation of SARSCoV-2 in vitro diagnostic medical devices   
• SARS-CoV-2 Common Specifications: Annex XIII of Commission Implementing Regulation (EU) 2022/1107 of 4 July 2022 laying down common specifications for certain class D in vitro diagnostic medical devices in accordance with Regulation (EU) 2017/746 of the European Parliament and of the Council (Text with EEA relevance)   
• Class II Special Controls Guidance Document: Respiratory Viral Panel Multiplex Nucleic Assay. October 9, 2009.   
• Instructions and requirements for Emergency Use Listing (EUL) submission: In vitro diagnostics detecting SARS-CoV-2 nucleic acid and rapid diagnostics tests detecting SARS-CoV-2 antigens (WHO PQDx_347 v4) Policy for Coronavirus Disease-2019 Tests During the Public Health Emergency (Revised) - Immediately in Effect Guidance for Clinical Laboratories, Commercial Manufacturers, and Food and Drug Administration Staff

# Clinical

• ISO 14155:2020 - Clinical investigation of medical devices for human subjects — Good clinical practice   
• ISO 20916:2019 - In vitro diagnostic medical devices — Clinical performance studies using specimens from human subjects — Good study practice   
• 21 CFR Part 50 - Protection of Human Subjects – eCFR   
• 21 CFR Part 54 – Financial Disclosure by Clinical Investigators   
• 21 CFR Part 56 - Institutional Review Boards - eCFR   
• 21 CFR Parts 812 - Investigational Device Exemptions – eCFR   
• Declaration of Helsinki,   
• Good Clinical Practice (ICH E6)   
• Good Clinical Practice (GCP)

# VII Performance Characteristics

# A Analytical Performance:

Note: The BD Respiratory Viral Panel – SCV2 for BD MAX System (Panel detects SARSCoV-2, Influenza A, Influenza B, RSV) is comprised of the same formulation, composition, and principle of operation as the BD Respiratory Viral Panel for BD MAX System; however, the BD Respiratory Viral Panel-SCV2 for BD MAX System utilizes a different Assay Definition File (ADF), which masks the results for influenza A/B and RSV and only reports the detection of SARS-CoV-2. As such, the supporting analytical and clinical validation data generated for the BD Respiratory Viral Panel for BD MAX System also supports validation of the BD Respiratory Viral Panel – SCV2 for BD MAX System.

1. Precision/Reproducibility:

# a. Within-Laboratory Precision

Within-laboratory precision was evaluated for the BD Respiratory Viral Panel at one (1) site with one (1) reagent lot. Testing was performed over twelve (12) days, with two (2) operators performing two (2) runs per day for a total of forty-eight (48) runs. The viral materials used to generate the positive panel members were contrived in simulated nasopharyngeal matrix (See: B. Comparison Studies/Matrix Comparison, below) and included SARS-CoV-2, Influenza A, Influenza B, and RSV. Each panel member was tested in three (3) replicates. The following target concentrations were used for each target organism contained in each panel member:

• Moderate positive (MP): 3x LoD • Low Positive (LP): 2x LoD • True Negative (TN): No target

The results are shown in Table 5.

Table 5. Overall Precision Study Results Using One Lot of the BD Respiratory Viral Panel (Percent Agreement with Expected Results)   

<table><tr><td colspan="1" rowspan="1">Sample Concentration</td><td colspan="1" rowspan="1">SARS-CoV-2Percent Agreement(N), 95% CI</td><td colspan="1" rowspan="1">Flu A PercentAgreement(N), 95% CI</td><td colspan="1" rowspan="1">Flu B PercentAgreement(N), 95% CI</td><td colspan="1" rowspan="1">RSV PercentAgreement(N), 95% CI</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td></tr><tr><td>Moderate Positive (3x LoD)</td><td>(144/144) 97.4-100</td><td>(144/144) 97.4-100</td><td>(144/144) 97.4-100</td><td>(144/144) 97.4-100</td></tr><tr><td>Low Positive (2x LoD)</td><td>100% (144/144) 97.4-100</td><td>97.2% (140/144) 93.1-98.9</td><td>99.3% (143/144) 96.2-99.9</td><td>99.3% (143/144) 96.2-99.9</td></tr><tr><td>True Negativea</td><td>100% (288/288) 98.7-100</td><td>100% (288/288) 98.7-100</td><td>100% (288/288) 98.7-100</td><td>100% (288/288) 98.7-100</td></tr></table>

a For the True Negative category, the reported agreement indicates percent of negative results.

# b. Reproducibility

For the Site-to-Site reproducibility study, three (3) sites (two external and one internal) were provided the same panels as described above for the Precision study. Each site performed testing on five (5) distinct days (consecutive or not), wherein each day, one (1) panel was tested by two (2) technologists. Each panel member was tested in three (3) replicates.

The qualitative reproducibility is presented below in Table 6 by target analyte. Ct, internal criterion used to determine a final assay result, was selected as an additional means of assessing assay reproducibility. Overall mean Ct values with variance components (SD and $\% C V _ { \perp }$ are shown in Table 7.

Table 6. Site-to Site Reproducibility Study using (1) Lot of the BD Respiratory Viral Panel (Percent Agreement with Expected Results)   

<table><tr><td rowspan=1 colspan=1>Sample Concentration</td><td rowspan=1 colspan=1>SARS-CoV-2PercentAgreement(N), 95% CI</td><td rowspan=1 colspan=1>Flu APercentAgreement(N), 95% CI</td><td rowspan=1 colspan=1>Flu BPercentAgreement(N), 95% CI</td><td rowspan=1 colspan=1>RSVPercentAgreement(N), 95% CI</td></tr><tr><td rowspan=1 colspan=1>Moderate Positive (3x LoD)</td><td rowspan=1 colspan=1>100%(90/90)95.9-100</td><td rowspan=1 colspan=1>97.8%(88/90)92.3-99.4</td><td rowspan=1 colspan=1>100%(90/90)95.9-100</td><td rowspan=1 colspan=1>100%(90/90)95.9-100</td></tr><tr><td rowspan=1 colspan=1>Low Positive (2x LoD)</td><td rowspan=1 colspan=1>100%(90/90)95.9-100</td><td rowspan=1 colspan=1>96.7%(87/90)90.7-98.9</td><td rowspan=1 colspan=1>100%(90/90)95.9-100</td><td rowspan=1 colspan=1>100%(90/90)95.9-100</td></tr><tr><td rowspan=1 colspan=1>True Negativea</td><td rowspan=1 colspan=1>100%(180/180)97.9-100</td><td rowspan=1 colspan=1>100%(180/180)97.9-100</td><td rowspan=1 colspan=1>100%(180/180)97.9-100</td><td rowspan=1 colspan=1>100%(180/180)97.9-100</td></tr></table>

a For the True Negative category, the reported agreement indicates percent of negative results.

Table 7. Site-to-Site Reproducibility Across Sites, Days, Runs, and Replicates – Ct Values   

<table><tr><td colspan="1" rowspan="2">Target</td><td colspan="1" rowspan="2">Level</td><td colspan="1" rowspan="2">N</td><td colspan="1" rowspan="2">MeanCt</td><td colspan="2" rowspan="1">Within Run</td><td colspan="2" rowspan="1">Between Run</td><td colspan="2" rowspan="1">Between Day</td><td colspan="2" rowspan="1">Between Site</td><td colspan="2" rowspan="1">Total</td></tr><tr><td colspan="1" rowspan="1">SD</td><td colspan="1" rowspan="1">CV (%)</td><td colspan="1" rowspan="1">SD</td><td colspan="1" rowspan="1">CV (%)</td><td colspan="1" rowspan="1">SD</td><td colspan="1" rowspan="1">CV (%)</td><td colspan="1" rowspan="1">SD</td><td colspan="1" rowspan="1">CV (%)</td><td colspan="1" rowspan="1">SD</td><td colspan="1" rowspan="1">CV (%)</td></tr><tr><td colspan="1" rowspan="1">CoV-2</td><td colspan="1" rowspan="1">LP</td><td colspan="1" rowspan="1">90</td><td colspan="1" rowspan="1">33.3</td><td colspan="1" rowspan="1">0.74</td><td colspan="1" rowspan="1">2.2</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">0.54</td><td colspan="1" rowspan="1">1.6</td><td colspan="1" rowspan="1">0.92</td><td colspan="1" rowspan="1">2.8</td></tr><tr><td colspan="1" rowspan="1">CoV-2</td><td colspan="1" rowspan="1">MP</td><td colspan="1" rowspan="1">90</td><td colspan="1" rowspan="1">33.0</td><td colspan="1" rowspan="1">0.45</td><td colspan="1" rowspan="1">1.4</td><td colspan="1" rowspan="1">0.08</td><td colspan="1" rowspan="1">0.2</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">0.74</td><td colspan="1" rowspan="1">2.2</td><td colspan="1" rowspan="1">0.87</td><td colspan="1" rowspan="1">2.6</td></tr><tr><td colspan="1" rowspan="1">Flu A</td><td colspan="1" rowspan="1">LP</td><td colspan="1" rowspan="1">87</td><td colspan="1" rowspan="1">34.9</td><td colspan="1" rowspan="1">1.31</td><td colspan="1" rowspan="1">3.8</td><td colspan="1" rowspan="1">0.36</td><td colspan="1" rowspan="1">1.0</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">0.51</td><td colspan="1" rowspan="1">1.5</td><td colspan="1" rowspan="1">1.45</td><td colspan="1" rowspan="1">4.2</td></tr><tr><td colspan="1" rowspan="1">Flu A</td><td colspan="1" rowspan="1">MP</td><td colspan="1" rowspan="1">88</td><td colspan="1" rowspan="1">33.5</td><td colspan="1" rowspan="1">1.03</td><td colspan="1" rowspan="1">3.1</td><td colspan="1" rowspan="1">0.37</td><td colspan="1" rowspan="1">1.1</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">0.20</td><td colspan="1" rowspan="1">0.6</td><td colspan="1" rowspan="1">1.11</td><td colspan="1" rowspan="1">3.3</td></tr><tr><td colspan="1" rowspan="1">Flu B</td><td colspan="1" rowspan="1">LP</td><td colspan="1" rowspan="1">90</td><td colspan="1" rowspan="1">33.6</td><td colspan="1" rowspan="1">1.25</td><td colspan="1" rowspan="1">3.7</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">0.29</td><td colspan="1" rowspan="1">0.9</td><td colspan="1" rowspan="1">0.20</td><td colspan="1" rowspan="1">0.6</td><td colspan="1" rowspan="1">1.29</td><td colspan="1" rowspan="1">3.9</td></tr><tr><td colspan="1" rowspan="1">Flu B</td><td colspan="1" rowspan="1">MP</td><td colspan="1" rowspan="1">90</td><td colspan="1" rowspan="1">33.0</td><td colspan="1" rowspan="1">0.67</td><td colspan="1" rowspan="1">2.0</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">0.20</td><td colspan="1" rowspan="1">0.6</td><td colspan="1" rowspan="1">0.15</td><td colspan="1" rowspan="1">0.5</td><td colspan="1" rowspan="1">0.72</td><td colspan="1" rowspan="1">2.2</td></tr><tr><td colspan="1" rowspan="1">RSV</td><td colspan="1" rowspan="1">LP</td><td colspan="1" rowspan="1">90</td><td colspan="1" rowspan="1">32.4</td><td colspan="1" rowspan="1">1.32</td><td colspan="1" rowspan="1">4.1</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">1.32</td><td colspan="1" rowspan="1">4.1</td></tr><tr><td colspan="1" rowspan="1">RSV</td><td colspan="1" rowspan="1">MP</td><td colspan="1" rowspan="1">90</td><td colspan="1" rowspan="1">31.9</td><td colspan="1" rowspan="1">0.92</td><td colspan="1" rowspan="1">2.9</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">0.13</td><td colspan="1" rowspan="1">0.4</td><td colspan="1" rowspan="1">0.92</td><td colspan="1" rowspan="1">2.9</td></tr></table>

For the Lot-to-Lot reproducibility study, one (1) internal site was provided the same panels as described for the Precision study above. Three (3) reagent lots were tested across five (5) distinct days (consecutive or not) using one (1) BD MAX, wherein each day, two (2) panels were tested by two (2) technologists. Each panel member was tested in three (3) replicates.

The qualitative reproducibility is presented below in Table 8 by target analyte. Ct, internal criterion used to determine a final assay result, was selected as an additional means of assessing assay reproducibility. Overall mean Ct values with variance components (SD and $\% C V$ ) are shown in Table 9.

Table 8. Lot-to-Lot Reproducibility Study Results using Three (3) Lots of the BD Respiratory Viral Panel (Percent Agreement with Expected Results)   

<table><tr><td rowspan=1 colspan=1>Sample Concentration</td><td rowspan=1 colspan=1>SARS-CoV-2(N), 95% CI</td><td rowspan=1 colspan=1>Flu A(N), 95% CI</td><td rowspan=1 colspan=1>Flu B(N), 95% CI</td><td rowspan=1 colspan=1>RSV(N), 95% CI</td></tr><tr><td rowspan=1 colspan=1>Moderate Positive (3x LoD)</td><td rowspan=1 colspan=1>99.4%(179/180)96.9-99.9</td><td rowspan=1 colspan=1>100%(180/180)97.9-100</td><td rowspan=1 colspan=1>98.9%(178/180)96.0-99.7</td><td rowspan=1 colspan=1>100%(180/180)97.9-100</td></tr><tr><td rowspan=1 colspan=1>Low Positive (2x LoD)</td><td rowspan=1 colspan=1>100%(180/180)97.9-100</td><td rowspan=1 colspan=1>97.8%(176/180)94.4-99.1</td><td rowspan=1 colspan=1>100%(180/180)97.9-100</td><td rowspan=1 colspan=1>100%(180/180)97.9-100</td></tr><tr><td rowspan=1 colspan=1>True Negativea</td><td rowspan=1 colspan=1>100%(360/360)98.9-100</td><td rowspan=1 colspan=1>100%(360/360)98.9-100</td><td rowspan=1 colspan=1>100%(360/360)98.9-100</td><td rowspan=1 colspan=1>100%(360/360)98.9-100</td></tr></table>

a – For the True Negative category, the reported agreement indicates percent of negative results.

Table 9. Lot-to-Lot Reproducibility across Operators, Days, Runs, and Replicates – Ct Values   

<table><tr><td colspan="1" rowspan="2">TargetLevel</td><td colspan="1" rowspan="2">TargetLevel</td><td colspan="1" rowspan="2">N</td><td colspan="1" rowspan="2">MeanCt</td><td colspan="2" rowspan="1">Lot</td><td colspan="2" rowspan="1">Day</td><td colspan="2" rowspan="1">Operator</td><td colspan="2" rowspan="1">Run</td><td colspan="2" rowspan="1">Within RunRepeatability</td><td colspan="2" rowspan="1">Total</td></tr><tr><td colspan="1" rowspan="1">SD</td><td colspan="1" rowspan="1">CV(%)</td><td colspan="1" rowspan="1">SD</td><td colspan="1" rowspan="1">CV(%)</td><td colspan="1" rowspan="1">SD</td><td colspan="1" rowspan="1">CV(%)</td><td colspan="1" rowspan="1">SD</td><td colspan="1" rowspan="1">CV(%)</td><td colspan="1" rowspan="1">SD</td><td colspan="1" rowspan="1">CV(%)</td><td colspan="1" rowspan="1">SD</td><td colspan="1" rowspan="1">CV(%)</td></tr><tr><td colspan="1" rowspan="1">CoV-2</td><td colspan="1" rowspan="1">MP</td><td colspan="1" rowspan="1">179</td><td colspan="1" rowspan="1">33.7</td><td colspan="1" rowspan="1">0.26</td><td colspan="1" rowspan="1">0.8</td><td colspan="1" rowspan="1">0.06</td><td colspan="1" rowspan="1">0.2</td><td colspan="1" rowspan="1">0.10</td><td colspan="1" rowspan="1">0.3</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">0.61</td><td colspan="1" rowspan="1">1.8</td><td colspan="1" rowspan="1">0.67</td><td colspan="1" rowspan="1">2.0</td></tr><tr><td colspan="1" rowspan="1">CoV-2</td><td colspan="1" rowspan="1">LP</td><td colspan="1" rowspan="1">180</td><td colspan="1" rowspan="1">33.9</td><td colspan="1" rowspan="1">0.24</td><td colspan="1" rowspan="1">0.7</td><td colspan="1" rowspan="1">0.18</td><td colspan="1" rowspan="1">0.5</td><td colspan="1" rowspan="1">0.12</td><td colspan="1" rowspan="1">0.3</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">0.64</td><td colspan="1" rowspan="1">1.9</td><td colspan="1" rowspan="1">0.72</td><td colspan="1" rowspan="1">2.1</td></tr><tr><td colspan="1" rowspan="1">Flu A</td><td colspan="1" rowspan="1">MP</td><td colspan="1" rowspan="1">180</td><td colspan="1" rowspan="1">33.2</td><td colspan="1" rowspan="1">0.26</td><td colspan="1" rowspan="1">0.8</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">1.05</td><td colspan="1" rowspan="1">3.2</td><td colspan="1" rowspan="1">1.08</td><td colspan="1" rowspan="1">3.3</td></tr><tr><td colspan="1" rowspan="1">Flu A</td><td colspan="1" rowspan="1">LP</td><td colspan="1" rowspan="1">176</td><td colspan="1" rowspan="1">34.3</td><td colspan="1" rowspan="1">0.44</td><td colspan="1" rowspan="1">1.3</td><td colspan="1" rowspan="1">0.44</td><td colspan="1" rowspan="1">1.3</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">0.23</td><td colspan="1" rowspan="1">0.7</td><td colspan="1" rowspan="1">1.40</td><td colspan="1" rowspan="1">4.1</td><td colspan="1" rowspan="1">1.55</td><td colspan="1" rowspan="1">4.5</td></tr><tr><td colspan="1" rowspan="1">Flu B</td><td colspan="1" rowspan="1">MP</td><td colspan="1" rowspan="1">178</td><td colspan="1" rowspan="1">33.3</td><td colspan="1" rowspan="1">0.30</td><td colspan="1" rowspan="1">0.9</td><td colspan="1" rowspan="1">0.36</td><td colspan="1" rowspan="1">1.1</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">0.16</td><td colspan="1" rowspan="1">0.5</td><td colspan="1" rowspan="1">1.30</td><td colspan="1" rowspan="1">3.9</td><td colspan="1" rowspan="1">1.39</td><td colspan="1" rowspan="1">4.2</td></tr><tr><td colspan="1" rowspan="1">Flu B</td><td colspan="1" rowspan="1">LP</td><td colspan="1" rowspan="1">180</td><td colspan="1" rowspan="1">34.1</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">0.17</td><td colspan="1" rowspan="1">0.5</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">0.22</td><td colspan="1" rowspan="1">0.6</td><td colspan="1" rowspan="1">1.20</td><td colspan="1" rowspan="1">3.5</td><td colspan="1" rowspan="1">1.23</td><td colspan="1" rowspan="1">3.6</td></tr><tr><td colspan="1" rowspan="2">Target</td><td colspan="1" rowspan="2">Level</td><td colspan="1" rowspan="2">N</td><td colspan="1" rowspan="2">MeanCt</td><td colspan="2" rowspan="1">Lot</td><td colspan="2" rowspan="1">Day</td><td colspan="2" rowspan="1">Operator</td><td colspan="2" rowspan="1">Run</td><td colspan="2" rowspan="1">Within Run(Repeatability</td><td colspan="2" rowspan="1">Total</td></tr><tr><td colspan="1" rowspan="1">SD</td><td colspan="1" rowspan="1">CV(%)</td><td colspan="1" rowspan="1">SD</td><td colspan="1" rowspan="1">CV(%)</td><td colspan="1" rowspan="1">SD</td><td colspan="1" rowspan="1">CV(%)</td><td colspan="1" rowspan="1">SD</td><td colspan="1" rowspan="1">CV(%)</td><td colspan="1" rowspan="1">SD</td><td colspan="1" rowspan="1">CV(%)</td><td colspan="1" rowspan="1">SD</td><td colspan="1" rowspan="1">CV(%)</td></tr><tr><td colspan="1" rowspan="1">RSV</td><td colspan="1" rowspan="1">MP</td><td colspan="1" rowspan="1">180</td><td colspan="1" rowspan="1">31.9</td><td colspan="1" rowspan="1">0.58</td><td colspan="1" rowspan="1">1.8</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">1.12</td><td colspan="1" rowspan="1">3.5</td><td colspan="1" rowspan="1">1.26</td><td colspan="1" rowspan="1">4.0</td></tr><tr><td colspan="1" rowspan="1">RSV</td><td colspan="1" rowspan="1">LP</td><td colspan="1" rowspan="1">180</td><td colspan="1" rowspan="1">32.7</td><td colspan="1" rowspan="1">0.60</td><td colspan="1" rowspan="1">1.8</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">0.05</td><td colspan="1" rowspan="1">0.1</td><td colspan="1" rowspan="1">0.87</td><td colspan="1" rowspan="1">2.7</td><td colspan="1" rowspan="1">1.06</td><td colspan="1" rowspan="1">3.2</td></tr></table>

2. Linearity: Not applicable. This is a qualitative assay.

3. Analytical Specificity/Interference: Analytical Reactivity (Inclusivity)

a. Wet-Testing

SARS-CoV-2, FluA, FluB, and RSV

This study was performed to determine the analytical reactivity of the BD Respiratory Viral Panel BD MAX System assay to detect clinically relevant strains, serotypes, or subtypes of the target species (i.e., SARS-CoV-2, Flu A, Flu B, and RSV). The inclusivity panel was prepared by spiking various heat-inactivated SARS-CoV-2 strains and purified cultures of Flu A/Flu B/RSV virus serotypes/subtypes/strains encompassing temporal and geographical diversity into negative clinical NP swab VTM/UTM matrix at a concentration of ${ \sim } 3 \mathbf { x }$ LoD and testing in triplicate.

Strains that did not yield $100 \%$ reactivity at $3 \mathbf { x }$ LoD were prepared at higher concentrations and tested until the minimum concentration that achieved $100 \%$ reactivity was reached. The strains evaluated and the lowest concentration that achieved $100 \%$ reactivity are shown in Table 10.

Table 10. Inclusivity Specifications and Results   

<table><tr><td colspan="1" rowspan="2">Subtype</td><td colspan="1" rowspan="2">Type</td><td colspan="1" rowspan="2">XLoD</td><td colspan="4" rowspan="1">Result</td></tr><tr><td colspan="1" rowspan="1">SARS-CoV-2</td><td colspan="1" rowspan="1">Flu A</td><td colspan="1" rowspan="1">FluB</td><td colspan="1" rowspan="1">RSV</td></tr><tr><td colspan="1" rowspan="10">SA-S2</td><td colspan="6" rowspan="1">SARS-CoV-2</td></tr><tr><td colspan="1" rowspan="1">Hong Kong/VM200001061/2020</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">Italy-INMI1</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">Alpha (B.1.1.7) USA/CA CDC 5574/2020</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">Alpha (B.1.1.7) England/204820464/2020</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">Beta (B.1.351) South Africa/KRISP-K005325/2020</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">Kappa (B.1.617.1) USA/CA-Stanford-15 S02/2021</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">Gamma (P1) Japan/TY7-503/2021</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">Delta (B.1.617.2) USA/PHC658/2021</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">Iota (B.1.526_2021) NY-Wadsworth-21025952-01/2021</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="1">Zeta (P2_2021) NY-Wadsworth-21006055-01/2021</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">Omicron (USA/GA-EHC-2811C/2021)</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="14">INIH</td><td colspan="1" rowspan="1">Influenza A</td><td colspan="1" rowspan="1">a A</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">A/H1N1(pdm09)/Bangladesh/3002/2015</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">A/H1N1(pdm09)/Iowa/53/2015</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">A/H1N1(pdm09)/Michigan/272/2017</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">A/H1N1(pdm09)/Michigan/45/2015</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">A/H1N1(pdm09)/St. Petersburg/61/2015</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">1/3a</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">A/H1N1(pdm09)/Wisconsin/505/2018</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">A/H1N1(pdm09)/Wisconsin/588/2019</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">A/H1N1(pdm09)AVR/Louisiana/08/2013</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">A/H1N1(pdm09)AVR/Maryland/08/2013</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">A/H1N1(pdm09)AVR/New York/18/2009</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">A/H1N1(pdm09)AVR/North Carolina/4/2014</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">A/H1N1/Guangdong-Maonan/SWL 1536/19</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">A/H1N1(pdm09)/Idaho/07/2018</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="12">HNE2</td><td colspan="1" rowspan="1">A/H3N2/Alaska/232/2015</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">A/H3N2/Arizona/45/2018</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">A/H3N2/California/02/2014</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">A/H3N2/Hong Kong/2671/19</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">A/H3N2/Norway/466/14</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">A/H3N2/Perth/16/09</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">A/H3N2/Singapore/INFIMH-16-0019/2016</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">A/H3N2/South Australia/55/14</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">A/H3N2/Stockholm/6/14</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">A/H3N2/Texas/71/2017</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">A/H3N2/Victoria/361/11</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">A/H3N2/Wisconsin/04/2018</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">H5N1</td><td colspan="1" rowspan="1">A/H5N1/common magpie/Hong Kong/645/2006</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">H5N2</td><td colspan="1" rowspan="1">A/H5N2/pheasant/New Jersey/1355/1998</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">H7N2</td><td colspan="1" rowspan="1">A/H7N2/turkey/Virginia/4529/2002</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="2">H7N7</td><td colspan="1" rowspan="2">A/H7N7/mallard/Netherlands/12/2000</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">1/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="2">H7N9</td><td colspan="1" rowspan="2">A/H7N9/Anhui/1/2013</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">1/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="2">H9N2</td><td colspan="1" rowspan="2">A/H9N2/chicken/Hong Kong/G9/1997</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">2/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="6">Ve</td><td colspan="1" rowspan="1">Influenza B</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">B/Colorado/6/2017</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">B/Hawaii/01/2018</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">B/Hong Kong/286/2017</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">B/Missouri/12/2018</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">B/Nevada/3/2011</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="5">Ra</td><td colspan="1" rowspan="1">B/Guangdong-Liwan/1133/2014</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">B/Indiana/17/2017</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">B/Oklahoma/10/2018</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">B/Utah/9/2014</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="1">B/Wisconsin/10/2016</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">0/3</td></tr><tr><td colspan="1" rowspan="4">RSV A</td><td colspan="1" rowspan="1">Respiratory Syncytial Virus (RSV)</td><td colspan="1" rowspan="1">ial Viru</td><td colspan="1" rowspan="1">(RSV</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">(RSV-A) 12/2014</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">(RSV-A)2/2015</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">(RSV-A) 4/2015</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">1/3a</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="2">RSV B</td><td colspan="1" rowspan="1">(RSV-B) 12/2014</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">1/3b</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">(RSV-B)3/2015</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">3/3</td></tr></table>

a-Positive result for SARS-CoV-2 amplification during the testing of strain A/H1N1(pdm09)/St. Petersburg/61/2015. Curve analysis shows clear amplification of the influenza A target and RNaseP endogenous control, no amplification of the influenza B and RSV targets, and weak/delayed amplification of the SARS-CoV-2 target. The curve is suggestive of a viral titer below the limit of detection for SARS-CoV-2 target.

b-Amplification of targets other than RSV. Curve analysis shows clear amplification of the RNaseP endogenous control and RSV targets, no amplification of the influenza A or B targets, and weak/delayed amplification of the SARS-CoV-2 target. No root cause could be identified.

The results from this study demonstrate that the BD Respiratory Viral Panel BD MAX System is capable of detecting multiple clinically relevant strains of SARS-CoV-2, Flu A, Flu B, and RSV.

# b. In-silico

The inclusivity of the BD MAX System BD Respiratory Viral Panel SARS-CoV-2/Flu A/B/RSV assay was evaluated using in silico analysis of the forward primers, reverse primers, and probes for the SARS-CoV-2, Flu A, Flu B and RSV target systems in relation to sequences available in the NCBI GenBank and GISAID gene databases.

Database Retrieval and Alignments of RSV, FluA and FluB Primer/Probe Sets

Sequences were retrieved from the NCBI GenBank nucleotide database for each species. The sequence coordinates of the RNA design regions were identified using blastn alignments between a reference sequence and each species dataset. These regions were extracted from all sequences and subsequently pairwise aligned using R Bioconductor, requiring global alignments of each oligo against local alignments to the subject. Ambiguous bases were called as a match when the paired nucleotide matched one of the corresponding bases, outlined in standard IUPAC code conventions. (Dixon, Bielka, & Cantor, 1986) Primers were grouped by sets and in the case of RSV, where there are two redundant detection systems; results were compared across primer-sets to determine overall system capability to detect each individual isolate.

NCBI accession numbers were used to retrieve associated meta-data from the GenBank files for each, and the results were used to identify the lineage, type and/or host of each sequenced isolate when available

Based on in silico analysis of all sequences available as of February 16, 2023 in GISAID and NCBI databases, the BD MAX System BD Respiratory Viral Panel SARS-CoV-2/Flu A/B/RSV assay is predicted to detect $2 9 9 . 8 \%$ Flu A, ${ \sim } 1 0 0 \%$ for Flu B, and $2 9 9 . 4 \%$ for RSV A/B.

Database Retrieval and Alignments of SARS-CoV-2 Primer/Probe Sets

All available full-length SARS-CoV-2 genome sequences were retrieved from the EpiCoV database at https://www.gisaid.org as of February 16, 2023. Sequences meeting internal quality control criteria $( \mathtt { n } = 9 , 3 3 0 , 3 5 ]$ 1), and the nucleocapsid N gene sequence was extracted by comparison to reference sequence “SARS-CoV- 2/human/USA/WA-CDC-WA1/2020” (NCBI Accession MN985325). To evaluate the potential impact of sequence variation on the

BD MAX SARS-CoV2 N1 and N2 PCR primer sets, all included sequences were compared through alignment with the BD MAX SARS-CoV-2 primers and probes.

Based on the in-silico analysis of GISAID and NCBI sequences available up to February 16, 2023 for SARS-CoV-2, the BD MAX System BD Respiratory Viral Panel SARS-CoV-2/Flu A/B/RSV assay, is predicted to detect $2 9 9 . 4 \%$ of the sequenced isolates $( \mathtt { n } = 9 , 3 3 0 , 3 5 1$ ).

Cross-Reactivity/Microbial Interference

# a. Exclusivity In-silico Assessment

The aim of this study is to perform an in-silico search for nucleic acid sequences from species other than the intended targets that match the assay primers and probes. This analysis serves to identify potential cross reactivity concerns due to primer and probe interactions with genetic material from species other than the intended targets. The analyses for primer/probe interactions for SARS-CoV-2, Flu A/B, and RSV were performed separately.

No clinically relevant cross-reactivity was identified for SARS-CoV-2, Influenza A, Influenza B, or RSV.

# b. Wet-Testing

The purpose of this study was to evaluate the analytical specificity (crossreactivity) and microbial interference of the BD Respiratory Viral Panel for BD MAX System and BD Respiratory Viral Panel – SCV2 for BD MAX System. Analytical specificity or cross-reactivity is defined as the ability to generate negative results for the analytes (SARS-CoV-2, Flu A/B, RSV, and RNase P) in the presence of viruses, yeasts, fungi, and bacteria that are phylogeneticallyrelated or likely to be found in respiratory tract clinical specimens, and potential microbial interference is assessing whether the candidate device can generate positive results for the analytes in the presence of viruses, yeasts, fungi, and bacteria that are phylogenetically-related or likely to be found in respiratory tract clinical specimens. Three (3) replicates in simulated nasopharyngeal matrix (See: B. Comparison Studies/Matrix Comparison, below) with and without the target analytes were tested for each potential cross-reactant/ microbial interferent listed in Table 11 below and must give a negative SARS-Cov-2. Flu A, Flu B, and RSV result in order to be deemed non-cross-reactive, and a positive SARS-CoV-2, Flu A, Flu B, and RSV result in order to be deemed non-microbial interferent. No cross-reactivity or microbial interference was observed at the concentrations tested.

Table 11. Cross-reactivity and Microbial Interference Study Microorganisms Evaluated by Wet-Testing   

<table><tr><td>Target</td><td>Concentration of Target Tested in SBT</td><td>Positive Testing (assessing potential microbial interference)</td><td>Negative Testing (assessing potential cross-</td></tr><tr><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td><td colspan="4" rowspan="1"></td><td colspan="1" rowspan="1">reactivity)</td></tr><tr><td colspan="1" rowspan="1">SARS-CovVv-2</td><td colspan="1" rowspan="1">Flu A</td><td colspan="1" rowspan="1">Flu B</td><td colspan="1" rowspan="1">RSV</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Adenovirus - Type 1</td><td colspan="1" rowspan="1">1.00E+05 TCID50/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Adenovirus - Type 4</td><td colspan="1" rowspan="1">1.00E+05 TCID50/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Adenovirus - Type 7</td><td colspan="1" rowspan="1">1.00E+05 TCID50/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Aspergillus flavus</td><td colspan="1" rowspan="1">1.00E+06 CFU/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Aspergillus fumigatus</td><td colspan="1" rowspan="1">1.00E+06 CFU/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Aspergillus niger</td><td colspan="1" rowspan="1">1.00E+06 CFU/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Aspergillus terreus</td><td colspan="1" rowspan="1">1.00E+06 CFU/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Bordetella parapertussis</td><td colspan="1" rowspan="1">1.00E+06 CFU/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Bordetella pertussis</td><td colspan="1" rowspan="1">1.00E+06 CFU/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Candida albicans</td><td colspan="1" rowspan="1">1.00E+06 CFU/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Chlamydia pneumonia</td><td colspan="1" rowspan="1">1.00E+06 IFU/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Corynebacteriumdiphtheriae</td><td colspan="1" rowspan="1">1.00E+06 CFU/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Cytomegalovirus</td><td colspan="1" rowspan="1">1.00E+05 copies/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Enterovirus B (Echovirus 6)</td><td colspan="1" rowspan="1">1.00E+05 units/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Enterovirus C(Coxsackievirus A16)</td><td colspan="1" rowspan="1">1.00E+05 TCID50/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Enterovirus D</td><td colspan="1" rowspan="1">1.00E+05 TCID50/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Epstein Barr virus</td><td colspan="1" rowspan="1">1.00E+05 copies/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">1.00E+06 CFU/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Fusobacteriumnecrophorum</td><td colspan="1" rowspan="1">1.00E+06 CFU/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Haemophilus influenzae</td><td colspan="1" rowspan="1">1.00E+06 CFU/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Herpes simplex virus Type 1</td><td colspan="1" rowspan="1">1.00E+05 TCID50/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Herpes simplex virusType 2</td><td colspan="1" rowspan="1">1.00E+05 TCID50/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Human coronavirus 229E</td><td colspan="1" rowspan="1">1.00E+05 TCID50/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Human coronavirus HKU1</td><td colspan="1" rowspan="1">1.00E+05 GC/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Human coronavirus NL63</td><td colspan="1" rowspan="1">1.00E+07 copies/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Human coronavirus OC43</td><td colspan="1" rowspan="1">1.00E+05 GE/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Human Metapneumovirus(hMPV)</td><td colspan="1" rowspan="1">1.00E+05 TCID50/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus acidophilus</td><td colspan="1" rowspan="1">1.00E+06 CFU/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Legionella pneumophila</td><td colspan="1" rowspan="1">1.00E+06 CFU/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Measles</td><td colspan="1" rowspan="1">1.00E+05 TCID50/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">MERS-coronavirus</td><td colspan="1" rowspan="1">1.00E+07 copies/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Moraxella catarrhalis</td><td colspan="1" rowspan="1">1.00E+06 CFU/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Mumps</td><td colspan="1" rowspan="1">1.00E+05 TCID50/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Mycoplasma genitalium</td><td colspan="1" rowspan="1">1.82E+05 cells/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Mycobacterium tuberculosis</td><td colspan="1" rowspan="1">1.00E+06 copies/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Mycoplasma pneumoniae</td><td colspan="1" rowspan="1">1.00E+06 CFU/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Neisseria meningitidis</td><td colspan="1" rowspan="1">1.00E+06 CFU/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Neisseria gonorrhoeae</td><td colspan="1" rowspan="1">1.00E+06 CFU/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza virus 1</td><td colspan="1" rowspan="1">1.00E+05 TCID50/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza virus 2</td><td colspan="1" rowspan="1">2.12E+05 TCID50/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza virus 3</td><td colspan="1" rowspan="1">1.00E+05 TCID50/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza virus 4</td><td colspan="1" rowspan="1">1.00E+05 TCID5/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Pneumocystis jirovecii(PJP)</td><td colspan="1" rowspan="1">1.00E+06 cells/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Pooled human expressednasopharyngeal swabmatrix</td><td colspan="1" rowspan="1">n/a</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">1.00E+06 CFU/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Rhinovirus</td><td colspan="1" rowspan="1">1.00E+05 TCID50/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">SARS-coronavirus</td><td colspan="1" rowspan="1">1.00E+05 GE /mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">1.00E+06 CFU/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus epidermis</td><td colspan="1" rowspan="1">1.00E+06 CFU/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Streptococcus pneumoniae</td><td colspan="1" rowspan="1">1.00E+06 CFU/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Streptococcus pyogenes</td><td colspan="1" rowspan="1">1.00E+06 CFU/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Streptococcus salivarius</td><td colspan="1" rowspan="1">1.00E+06 CFU/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Varicella-zoster virus</td><td colspan="1" rowspan="1">1.00E+07 copies/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr></table>

1 CFU $=$ Colony Forming Units; $\mathrm { G E = }$ Genome Equivalents; IFU $=$ Inclusion Forming Units; $\mathrm { T C I D } _ { 5 0 } =$ Median Tissue Culture Infectious Dose

# Interfering Substances

This study evaluated the performance of the BD Respiratory Viral Panel for BD MAX System and BD Respiratory Viral Panel – SCV2 for BD MAX System in the presence of medications, over the counter products, and other potentially interfering substances found in a clinical respiratory specimen. Assay results were evaluated to determine if the presence of potentially interfering substances in target analyte negative or target analyte positive samples had an effect on assay performance. The study utilized simulated nasopharyngeal matrix containing mucin and human DNA to establish the interference claims (See: B. Comparison Studies/Matrix Comparison, for validation of equivalency between matrices).

Each interferent was tested with three (3) positive replicates containing SARS-CoV-2, Influenza A, Influenza B, and RSV and three negative replicates containing only the interferent with the simulated nasopharyngeal matrix for a total of six (6) samples per interferent level. Three (3) positive replicates containing SARS-CoV-2, Influenza A, Influenza B, and RSV were run without interferent to demonstrate passing acceptance criteria for positive samples. Whole blood (Human) at $2 \%$ v/v was found to interfere with the assay of both SARS-CoV-2 and Influenza B. When the amount of blood was titrated downward to $0 . 2 \% \mathrm { v / v }$ , it no longer interfered with either assay. Table 12.

Table 12. Interfering Substance Results   

<table><tr><td colspan="1" rowspan="2">Substance</td><td colspan="1" rowspan="2">ActiveIngredient</td><td colspan="1" rowspan="2">ConcentrationTested</td><td colspan="4" rowspan="1">Positive Testing(Positive/Total)</td><td colspan="1" rowspan="2">Negative Testing(Negative /M Total)</td><td colspan="1" rowspan="2">Result</td></tr><tr><td colspan="1" rowspan="1">SARS-CoV-2</td><td colspan="1" rowspan="1">InfluenzaA</td><td colspan="1" rowspan="1">InfluenzaB</td><td colspan="1" rowspan="1">RSV</td></tr><tr><td colspan="1" rowspan="1">None</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td></tr><tr><td colspan="1" rowspan="2">Oralanestheticandanalgesic</td><td colspan="1" rowspan="1">Benzocaine</td><td colspan="1" rowspan="2">0.8 mg/mL</td><td colspan="1" rowspan="2">3/3</td><td colspan="1" rowspan="2">3/3</td><td colspan="1" rowspan="2">3/3</td><td colspan="1" rowspan="2">3/3</td><td colspan="1" rowspan="2">3/3</td><td colspan="1" rowspan="2">NI</td></tr><tr><td colspan="1" rowspan="1">Menthol</td></tr><tr><td colspan="1" rowspan="4">Biologicals</td><td colspan="1" rowspan="1">Purified Mucin</td><td colspan="1" rowspan="1">60 μg/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">NI</td></tr><tr><td colspan="1" rowspan="2">WholeBlood(human)</td><td colspan="1" rowspan="1">2% v/v</td><td colspan="1" rowspan="1">1/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">1/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">I</td></tr><tr><td colspan="1" rowspan="1">0.2% v/v</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">NI</td></tr><tr><td colspan="1" rowspan="1">Leukocytes</td><td colspan="1" rowspan="1">2% v/v</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">NI</td></tr><tr><td colspan="1" rowspan="4">Nasal Sprays /Drops</td><td colspan="1" rowspan="1">Zinc</td><td colspan="1" rowspan="1">1 mg/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">NI</td></tr><tr><td colspan="1" rowspan="1">Phenylephrine</td><td colspan="1" rowspan="1">5% v/v</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">NI</td></tr><tr><td colspan="1" rowspan="1">Oxymetazoline</td><td colspan="1" rowspan="1">5% v/v</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">NI</td></tr><tr><td colspan="1" rowspan="1">SodiumChloride withpreservatives</td><td colspan="1" rowspan="1">5% v/v</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">NI</td></tr><tr><td colspan="1" rowspan="7">Corticosteroids</td><td colspan="1" rowspan="1">Beclomethasone</td><td colspan="1" rowspan="1">17% v/v</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">NI</td></tr><tr><td colspan="1" rowspan="1">Dexamethasone</td><td colspan="1" rowspan="1">17% v/v</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">NI</td></tr><tr><td colspan="1" rowspan="1">Flunisolide</td><td colspan="1" rowspan="1">17% v/v</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">NI</td></tr><tr><td colspan="1" rowspan="1">Triamcinolone</td><td colspan="1" rowspan="1">17% v/v</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">NI</td></tr><tr><td colspan="1" rowspan="1">Budesonide</td><td colspan="1" rowspan="1">17% v/v</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">NI</td></tr><tr><td colspan="1" rowspan="1">Mometasone</td><td colspan="1" rowspan="1">17% v/v</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">NI</td></tr><tr><td colspan="1" rowspan="1">Fluticasone</td><td colspan="1" rowspan="1">17% v/v</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">NI</td></tr><tr><td colspan="1" rowspan="2">Nasal Gel -HomeopathicAllergy Relief</td><td colspan="1" rowspan="1">Luffaoperculata</td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2">−</td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td></tr><tr><td colspan="1" rowspan="1">Sulfur</td></tr><tr><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="1">Galphimia glauca</td><td colspan="1" rowspan="2">5% v/v</td><td colspan="1" rowspan="2">3/3</td><td colspan="1" rowspan="2">3/3</td><td colspan="1" rowspan="2">3/3</td><td colspan="1" rowspan="2">3/3</td><td colspan="1" rowspan="2">3/3</td><td colspan="1" rowspan="2">NI</td></tr><tr><td colspan="1" rowspan="1">Histaminumhydrocloricum</td></tr><tr><td colspan="1" rowspan="1">Antiviral Drug</td><td colspan="1" rowspan="1">Zanamivir</td><td colspan="1" rowspan="1">3.3 mg/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">NI</td></tr><tr><td colspan="1" rowspan="1">Antibiotic</td><td colspan="1" rowspan="1">Mupirocin</td><td colspan="1" rowspan="1">10 mg/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">NI</td></tr><tr><td colspan="1" rowspan="1">Antibacterial</td><td colspan="1" rowspan="1">Tobramycin</td><td colspan="1" rowspan="1">4 μg/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">NI</td></tr><tr><td colspan="9" rowspan="1">NI = Non-Interferent, I = Interferent</td></tr></table>

FluMist Quadrivalent

In a supplementary interfering substances study, the FluMist Quadrivalent vaccine (MedImmune, LLC) was assayed for its impact on the performance of the BD Respiratory Viral Panel for BD MAX System and the BD Respiratory Viral Panel – SCV2 for BD MAX System. FluMist vaccine contains live Influenza A and B strains and was found to interfere with the BD Respiratory Viral Panel at concentrations greater than 6.67E- $12 \%$ in nasopharyngeal specimens. Tables 13 & 14.

Table 13. Interfering Substance Positive Testing Results   

<table><tr><td rowspan=2 colspan=1>Substance</td><td rowspan=2 colspan=1>ActiveIngredient</td><td rowspan=2 colspan=1>ConcentrationTested</td><td rowspan=1 colspan=4>Positive Testing(Positive/Total)</td><td rowspan=2 colspan=1>Result</td></tr><tr><td rowspan=1 colspan=1>SARS- CoV-2</td><td rowspan=1 colspan=1>InfluenzaA</td><td rowspan=1 colspan=1>InfluenzaB</td><td rowspan=1 colspan=1>RSV</td></tr><tr><td rowspan=1 colspan=1>None</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>N/A</td></tr><tr><td rowspan=4 colspan=1>FluMistQuadrivalent</td><td rowspan=4 colspan=1>Live,attenuatedFlu A andFlu B strains</td><td rowspan=1 colspan=1>6.67%</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>NI</td></tr><tr><td rowspan=1 colspan=1>6.67E-04%</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>NI</td></tr><tr><td rowspan=1 colspan=1>6.67E-08%</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>NI</td></tr><tr><td rowspan=1 colspan=1>6.67E-12%</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>NI</td></tr><tr><td rowspan=1 colspan=8>NI = Non-Interferent, I = Interferent</td></tr></table>

Table 14. Interfering Substance Negative Testing Results   

<table><tr><td rowspan=2 colspan=1>Substance</td><td rowspan=2 colspan=1>ActiveIngredient</td><td rowspan=2 colspan=1>ConcentrationTested</td><td rowspan=1 colspan=4>Negative Testing(Negative/Total)</td><td rowspan=2 colspan=1>Result</td></tr><tr><td rowspan=1 colspan=1>SARS- CoV-2</td><td rowspan=1 colspan=1>InfluenzaA</td><td rowspan=1 colspan=1>InfluenzaB</td><td rowspan=1 colspan=1>RSV</td></tr><tr><td rowspan=4 colspan=1>FluMistQuadrivalent</td><td rowspan=4 colspan=1>Live,attenuatedFlu A andFlu B strains</td><td rowspan=1 colspan=1>6.67%</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>I</td></tr><tr><td rowspan=1 colspan=1>6.67E-04%</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>I</td></tr><tr><td rowspan=1 colspan=1>6.67E-08%</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>2/3</td><td rowspan=1 colspan=1>2/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>I</td></tr><tr><td rowspan=1 colspan=1>6.67E-12%</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>NI</td></tr></table>

NI = Non-Interferent, $\mathrm { I } =$ Interferent

# Competitive Interference

The purpose of this study was to demonstrate the ability of the BD Respiratory Viral Panel and BD Respiratory Viral Panel – SCV2 for the BD Max system assay to detect the targeted analytes in cases of simulated mixed infection, when three (3) viral targets are low and one (1) target is high (competitive interference).

One (1) representative strain of each targeted organism (SARS-CoV-2, Flu A, Flu B and RSV) was tested. Co-infection panels were made by spiking one (1) target organism at high concentration $\mathrm { 1 . 0 E ^ { + } 0 6 }$ $\mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ or $\mathrm { c p / m L }$ ) and three (3) other target organisms at low concentration (2x LoD) in simulated nasopharyngeal matrix with heat-inactivated virus (See: B. Comparison Studies/Matrix Comparison, for validation of equivalency between matrices). The study was performed with 20 replicates per condition. Acceptance criteria $2 9 5 \%$ . None of the analytes present at a very high concentration interfered with the detection of low levels of the other three analytes. See Table 15.

Table 15. Mixed Infection Results for the BD Respiratory Viral Panel for the BD MAX System   

<table><tr><td rowspan=2 colspan=1>Low (2X LoD)/High (1E+6 cp/mL)</td><td rowspan=1 colspan=4>Positivity</td></tr><tr><td rowspan=1 colspan=1>SARS-CoV-2</td><td rowspan=1 colspan=1>Flu A</td><td rowspan=1 colspan=1>Flu B</td><td rowspan=1 colspan=1>RSV</td></tr><tr><td rowspan=1 colspan=1>High (SCV2)</td><td rowspan=1 colspan=1>100%(20/20)</td><td rowspan=1 colspan=1>100%(20/20)</td><td rowspan=1 colspan=1>100%(20/20)</td><td rowspan=1 colspan=1>100%(20/20)</td></tr><tr><td rowspan=1 colspan=1>High (Flu A)</td><td rowspan=1 colspan=1>100%(20/20)</td><td rowspan=1 colspan=1>100%(20/20)</td><td rowspan=1 colspan=1>100%(20/20)</td><td rowspan=1 colspan=1>100%(20/20)</td></tr><tr><td rowspan=1 colspan=1>High (Flu B)</td><td rowspan=1 colspan=1>100%(20/20)</td><td rowspan=1 colspan=1>95%(19/20)</td><td rowspan=1 colspan=1>100%(20/20)</td><td rowspan=1 colspan=1>100%(20/20)</td></tr><tr><td rowspan=1 colspan=1>High (RSV)</td><td rowspan=1 colspan=1>95%(19/20)</td><td rowspan=1 colspan=1>95%(19/20)</td><td rowspan=1 colspan=1>95%(19/20)</td><td rowspan=1 colspan=1>100%(20/20)</td></tr><tr><td rowspan=1 colspan=1>All Low</td><td rowspan=1 colspan=1>100%(20/20)</td><td rowspan=1 colspan=1>95%(19/20)</td><td rowspan=1 colspan=1>100%(0/20)</td><td rowspan=1 colspan=1>100%((20/20)</td></tr></table>

4. Assay Reportable Range: Not applicable; this is a qualitative assay.

5. Traceability, Stability, Expected Values (Controls, Calibrators, or Methods):

# Traceability

This study was conducted in order to establish the limit of detection (LoD) of the SARSCoV-2 target using the WHO international standard [Heat inactivated, England/02/2020, NIBSC code: 20/146] with the BD Respiratory Viral Panel and BD Respiratory Viral Panel-SCV2 for BD MAX System. The data generated from this study will be used to support the conformity of the assay to common specifications in the EU for SARS-CoV-2 assays. The LoD of the WHO international standard was determined in both clinical nasal matrix and clinical nasopharyngeal matrix.

The LoD confirmation was performed with 20 replicates for each matrix type used, thus aligning with CLSI 17-A2 Evaluation of the Detection Capability for Clinical Laboratory Measurement Procedures and WHO PQDx_347. The LoD of SARS-CoV-2 [England/02/2020, NIBSC code: 20/146] was determined to be $7 2 6 \ : \mathrm { I U / m L }$ . Results are shown in Table 16.

Table 16. WHO International Standard LoD Results   

<table><tr><td colspan="1" rowspan="2">Level</td><td colspan="1" rowspan="2">Concentration(IU/mL)</td><td colspan="2" rowspan="1">Positivity</td><td colspan="2" rowspan="1">SARS-CoV-2 Ct</td><td colspan="2" rowspan="1">RNaseP Ct</td></tr><tr><td colspan="1" rowspan="1">Nasopharyngeal</td><td colspan="1" rowspan="1">Nasal</td><td colspan="1" rowspan="1">Nasopharyngeal</td><td colspan="1" rowspan="1">Nasal</td><td colspan="1" rowspan="1">Nasopharyngeal</td><td colspan="1" rowspan="1">Nasal</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">3x</td><td colspan="1" rowspan="1">726</td><td colspan="1" rowspan="1">(20/20)</td><td colspan="1" rowspan="1">(20/20)</td><td colspan="1" rowspan="1">35.0</td><td colspan="1" rowspan="1">35.1</td><td colspan="1" rowspan="1">22.2</td><td colspan="1" rowspan="1">23.7</td></tr><tr><td colspan="1" rowspan="1">1x</td><td colspan="1" rowspan="1">242</td><td colspan="1" rowspan="1">50%(10/20)</td><td colspan="1" rowspan="1">65%(13/20)</td><td colspan="1" rowspan="1">37.0</td><td colspan="1" rowspan="1">37.2</td><td colspan="1" rowspan="1">22.1</td><td colspan="1" rowspan="1">23.9</td></tr><tr><td colspan="1" rowspan="1">0.3x</td><td colspan="1" rowspan="1">80.7</td><td colspan="1" rowspan="1">30%(6/20)</td><td colspan="1" rowspan="1">35%(7/20)</td><td colspan="1" rowspan="1">36.8</td><td colspan="1" rowspan="1">37.3</td><td colspan="1" rowspan="1">22.0</td><td colspan="1" rowspan="1">23.8</td></tr></table>

a) Specimen Stability

Specimen stability studies were performed to support the following stability claims:

Table 17. BD Respiratory Viral Panel Assay for BD MAX System Specimen Stability   

<table><tr><td rowspan=1 colspan=1>Specimen Stability</td><td rowspan=1 colspan=1>Temperature</td><td rowspan=1 colspan=1>Duration</td></tr><tr><td rowspan=2 colspan=1>In UVT/UTM</td><td rowspan=1 colspan=1>25 ± 2 °</td><td rowspan=1 colspan=1>12 hours</td></tr><tr><td rowspan=1 colspan=1>2-8°</td><td rowspan=1 colspan=1>72 hours</td></tr><tr><td rowspan=1 colspan=1>In BD Molecular RVP Sample Buffer</td><td rowspan=1 colspan=1>25 ± 2 °</td><td rowspan=1 colspan=1>24 hours</td></tr><tr><td rowspan=1 colspan=1>Tube</td><td rowspan=1 colspan=1>2-8 </td><td rowspan=1 colspan=1>120 hours</td></tr></table>

Freeze/thaw specimen stability studies were performed and indicate that SARS-CoV-2, Influenza A, Influenza B, and RSV are stable for up to two (2) freeze / thaw cycles in either clinical nasopharyngeal matrix or clinical nasal matrix expressed in UVT/UTM (neat) or post transfer of UVT/UTM matrix into sample buffer tubes (nested). Once thawed, samples are stable for up to 12 hours at $2 5 \pm 2 { } ^ { \mathrm { ~ o } } \mathrm { C }$ . Specimens were constructed using natural clinical nasopharyngeal or anterior nasal matrix spiked at 3X LoD for each analyte.

# b) Reagent Kit Stability

To evaluate reagent kit stability, the reagents were refrigerated at $( 5 { \pm } 3 ^ { \circ } \mathrm { C } )$ and stored at “room temperature” (RT $2 5 \pm 2 ^ { \circ } \mathrm { C } )$ in parallel. At defined time points during the study, the Master Mix (MM) and Extraction Tube $( \mathrm { T x } )$ foil pouches are being 1) unsealed, 2) opened, 3) closed and then, 4) refrigerated and stored at RT in parallel to simulate in-use conditions across the targeted temperature range of $2 { - } 2 5 ^ { \circ } \mathrm { C }$ . Reagent tubes are stable for up to 31 days at $2 { - } 2 5 ~ ^ { \circ } \mathrm { C }$ after initial opening and re-sealing of the pouch.

# c) Reagent Kit Shipping Stability

The study also assessed reagent kit shipping stability. The purpose is to verify the stability of the reagents throughout their stored shelf-life at $2 { - } 2 5 ^ { \circ } \mathrm { C }$ after being exposed to conditions which could reasonably be encountered during normal shipping conditions. Simulated shipping stability was performed by subjecting reagents to shipping conditions. This is performed in an environmental chamber that adjusts both temperature and humidity to mimic shipping conditions. After shipping exposure, reagents are tested for a post-exposure baseline and subsequently included in the real-time stability arm of the study.

6. Detection Limit:

SARS-CoV-2 LoD Determination in Clinical NP and NS Matrix in UVT/UTM

The purpose of this study was to establish the Limit of Detection (LoD) for SARS-CoV-2 in clinical nasopharyngeal (NP) and nasal (NS) matrices for the BD Respiratory Viral Panel for BD MAX System.

LoD studies determine the lowest detectable concentration of virus at which approximately $9 5 \%$ of all (true positive) replicates test positive. LoD was estimated using probit analysis. Confirmation of the estimated LoD with heat-inactivated SARS-CoV-2 (2019-nCoV/USAWA1/2020) was performed with one (1) reagent lot in replicates of 20 prepared in clinical nasopharyngeal and clinical nasal matrix. Refer to Table 18 for the BD Respiratory Viral Panel-SCV2 LoD for SARS-CoV-2 in clinical nasopharyngeal matrix in UVT/UTM. The LoD for SARS-CoV-2 in NP matrix in UVT/UTM was determined to be 700 copies/mL.

Table 18. LoD determination for SARS-CoV-2 in Clinical NP Matrix   

<table><tr><td rowspan=1 colspan=3>Concentration</td><td rowspan=1 colspan=1>Positivity</td><td rowspan=1 colspan=2>Mean Ct</td></tr><tr><td rowspan=1 colspan=1>IU/mL</td><td rowspan=1 colspan=1>Copies/mL</td><td rowspan=1 colspan=1>TCID50/mL</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>SARS-CoV-2</td><td rowspan=1 colspan=1>RNaseP</td></tr><tr><td rowspan=1 colspan=1>112</td><td rowspan=1 colspan=1>323</td><td rowspan=1 colspan=1>4.96E+00</td><td rowspan=1 colspan=1>90%(18/20)</td><td rowspan=1 colspan=1>37.1</td><td rowspan=1 colspan=1>23.4</td></tr><tr><td rowspan=1 colspan=1>153</td><td rowspan=1 colspan=1>442</td><td rowspan=1 colspan=1>6.79E+00</td><td rowspan=1 colspan=1>90%(18/20)</td><td rowspan=1 colspan=1>35.4</td><td rowspan=1 colspan=1>24.2</td></tr><tr><td rowspan=1 colspan=1>194</td><td rowspan=1 colspan=1>561</td><td rowspan=1 colspan=1>8.62E+00</td><td rowspan=1 colspan=1>85%(17/20)</td><td rowspan=1 colspan=1>35.4</td><td rowspan=1 colspan=1>22.5</td></tr><tr><td rowspan=1 colspan=1>242</td><td rowspan=1 colspan=1>700</td><td rowspan=1 colspan=1>1.08E+01</td><td rowspan=1 colspan=1>95%(19/20)</td><td rowspan=1 colspan=1>34.2</td><td rowspan=1 colspan=1>23.3</td></tr></table>

The LoD determination for SARS-CoV-2 in clinical anterior nasal matrix in UVT/UTM was carried out using the data from the NPS study as a starting point. Results are shown in Table 19. Results for NP matrix, show the LoD for SARS-CoV-2 in ANS Matrix in UVT/UTM is 700 copies/mL.

Table 19. LoD determination for SARS-CoV-2 in Clinical ANS Matrix   

<table><tr><td rowspan=1 colspan=3>Concentration</td><td rowspan=1 colspan=1>Positivity</td><td rowspan=1 colspan=2>Mean Ct</td></tr><tr><td rowspan=1 colspan=1>IU/mL</td><td rowspan=1 colspan=1>Copies/mL</td><td rowspan=1 colspan=1>TCID50/mL</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>SARS-CoV-2</td><td rowspan=1 colspan=1>RNaseP</td></tr><tr><td rowspan=1 colspan=1>153</td><td rowspan=1 colspan=1>442</td><td rowspan=1 colspan=1>6.79E+00</td><td rowspan=1 colspan=1>80%(16/20)</td><td rowspan=1 colspan=1>36.3</td><td rowspan=1 colspan=1>25.5</td></tr><tr><td rowspan=1 colspan=1>242</td><td rowspan=1 colspan=1>700</td><td rowspan=1 colspan=1>1.08E+01</td><td rowspan=1 colspan=1>100%(20/20)</td><td rowspan=1 colspan=1>35.9</td><td rowspan=1 colspan=1>25.9</td></tr></table>

FluA, FluB, and RSV LoD Determination in Clinical NPS and ANS Matrix in UVT/UTM

The LoDs of live strains of Influenza A [H1N1/Brisbane] & [H3N2/Kansas/14/17], Influenza B [Victoria/Colorado] & [Yamagata/Phuket/3073/13], RSV A/B were determined in both NP and

ANS clinical matrices in UVT/UTM. Twenty (20) replicates of each dilution were carried out. Table 20 shows the compiled LoD determinations for Flu A, FluB, and RSV A/B in clinical NP matrix in UVT/UTM.

# Clinical Nasopharyngeal Matrix

Table 20. Analytical LoDs for Influenza A, Influenza B, and RSB A/B in NP Matrix in UVT/UTM   

<table><tr><td rowspan=1 colspan=1>Strain ID</td><td rowspan=1 colspan=2>Confirmed Limit of Detection</td></tr><tr><td rowspan=1 colspan=1>Influenza A/H1N1/Brisbane</td><td rowspan=1 colspan=1>500 copies/mL</td><td rowspan=1 colspan=1>5.59E-03 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Influenza A/H3N2/Kansas</td><td rowspan=1 colspan=1>500 copies/mL</td><td rowspan=1 colspan=1>2.75E-01 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Influenza B/Colorado</td><td rowspan=1 colspan=1>33 copies/mL</td><td rowspan=1 colspan=1>6.80E-03 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Influenza B/Phuket/3073/13</td><td rowspan=1 colspan=1>100 copies/mL</td><td rowspan=1 colspan=1>2.90E-02 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>RSV A</td><td rowspan=1 colspan=1>143 copies/mL</td><td rowspan=1 colspan=1>3.09E-02 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>RSV B</td><td rowspan=1 colspan=1>687 copies/mL</td><td rowspan=1 colspan=1>1.68E-02 TCID50/mL</td></tr></table>

# Clinical Anterior Nasal Matrix

Twenty replicates of each dilution were carried out. Table 21 shows the compiled LoD determinations for Flu A, FluB, and RSV A/B in clinical ANS matrix in UVT/UTM.

Table 21 Analytical LoDs for Influenza A, Influenza B, and RSB A/B in ANS Matrix in UVT/UTM   

<table><tr><td rowspan=1 colspan=1>Strain ID</td><td rowspan=1 colspan=2>Confirmed Limit of Detection</td></tr><tr><td rowspan=1 colspan=1>Influenza A/H1N1/Brisbane</td><td rowspan=1 colspan=1>500 copies/mL</td><td rowspan=1 colspan=1>5.59E-03 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Influenza A/H3N2/Kansas</td><td rowspan=1 colspan=1>500 copies/mL</td><td rowspan=1 colspan=1>2.75E-01 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Influenza B/Colorado</td><td rowspan=1 colspan=1>33 copies/mL</td><td rowspan=1 colspan=1>6.80E-03 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Influenza B/Phuket/3073/13</td><td rowspan=1 colspan=1>33 copies/mL</td><td rowspan=1 colspan=1>9.56E-03 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>RSV A</td><td rowspan=1 colspan=1>143 copies/mL</td><td rowspan=1 colspan=1>3.09E-02 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>RSVB</td><td rowspan=1 colspan=1>229 copies/mL</td><td rowspan=1 colspan=1>5.61E-03 TCID50/mL</td></tr></table>

# 7. Assay Cut-Off

This series of experiments was conducted in order to determine assay thresholds for the SARS-CoV-2, Flu A, Flu B, and RSV target assays by utilizing both contrived positive and negative nasopharyngeal swab (NPS) and simulated nasopharyngeal (SIM) specimens in UVT/UTM. NP specimens were representative for both NPS and anterior nasal swab (ANS) specimen types as NPS specimens are considered worst case. NPS specimens provide the most amount of background material while simulated matrix represents a specimen with the least amount of background material. Specimens were tested over six (6) master mix (MM) reagent lots across fifteen (15) instruments over three (3) test days.

The assay cut-off for the BD Respiratory Viral Panel for BD MAX was established based on PCR-based metrics taken together by the BD MAX software algorithm to make the qualitative decision whether a curve is to be considered positive or negative. These metrics (e.g., Ct, RFU endpoints, signal-to-noise ratios) are initially set by default parameters defined by the instrument. As the product undergoes product development, the data are supplemented, and the algorithm is adjusted (“trained”) using viral cultures spiked into clinical background matrices at levels surrounding the limit of detection and expected clinical range.

8. Accuracy (Instrument): Not Applicable

9. Carry-Over:

Carryover / Cross-Contamination

This study was conducted to investigate within-run and between-run carryover while processing samples with high viral load of SARS-CoV-2 in the BD Respiratory Viral Panel for BD MAX System. High positive samples contained heat inactivated SARS-CoV-2 spiked into pooled nasal swab matrix at a concentration of ${ \geq } 1 . 9 4 \mathrm { E } { + } 0 7$ copies/mL. The negative samples consisted of simulated nasopharyngeal (NP) matrix without any target analyte. Twelve (12) replicates of the high positive panel member and 12 replicates of the negative panel member were tested in nine (9) runs by alternating negative and positive samples, using three BD MAX Systems. A total of 108 positive and 108 negative samples were tested. Of the 108 negative samples tested, one (1) false positive result was obtained $( 0 . 9 3 \%$ , $9 5 \%$ CI: $0 . 1 6 { - } 5 . 0 6 \%$ ).

# B Comparison Studies:

1. Method Comparison with Predicate Device:

Not Applicable

2. Matrix Comparison:

Matrix Equivalency

The purpose of this study was to demonstrate the functional equivalency among clinical nasopharyngeal swab (NPS) specimens in either BD Universal Viral Transport (UVT/UTM) media, clinical nasal swab specimens expressed in either BD Universal Viral Transport (UVT/UTM) media and simulated nasopharyngeal/nasal matrix (contains mucin and human cells/DNA) when tested with the BD Respiratory Viral Panel for BD MAX System assay. The study utilized contrived low (2x LoD) and moderate (5x LoD) positive samples and negative samples. The viral samples employed were: heatinactivated SARS-CoV-2 (USA-WA/2020); Influenza A/H1N1/Brisbane; Influenza B/Yamagata/Phuket/3073/13, and RSV/A. The results are shown in

Table 22 and support that the BD MAX System BD Respiratory Viral Panel SARS-CoV-2/Flu A/B/RSV assay can be used with the evaluated viral transport media types, and the simulated nasopharyngeal/nasal matrix (contains mucin and human cells/DNA) is similar to the natural nasopharyngeal/nasal swabs in UVT/UTM matrix with regard to the candidate assay performance.

Table 22 Matrix Equivalency Results   

<table><tr><td rowspan=2 colspan=1>Conc.</td><td rowspan=2 colspan=1>Matrix</td><td rowspan=1 colspan=4>Positivity</td><td rowspan=1 colspan=5>Ct</td></tr><tr><td rowspan=1 colspan=1>SARS-CoV-2</td><td rowspan=1 colspan=1>InfluenzaA</td><td rowspan=1 colspan=1>InfluenzaB</td><td rowspan=1 colspan=1>RSV</td><td rowspan=1 colspan=1>SARS-CoV- 2</td><td rowspan=1 colspan=1>InfluenzaA</td><td rowspan=1 colspan=1>InfluenzaB</td><td rowspan=1 colspan=1>RSV</td><td rowspan=1 colspan=1>RNase P</td></tr><tr><td rowspan=6 colspan=1>Negative</td><td rowspan=2 colspan=1>Clinical NPMatrix</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0%</td><td rowspan=2 colspan=1>n/a</td><td rowspan=2 colspan=1>n/a</td><td rowspan=2 colspan=1>n/a</td><td rowspan=2 colspan=1>n/a</td><td rowspan=2 colspan=1>22.1</td></tr><tr><td rowspan=1 colspan=1>(0/15)</td><td rowspan=1 colspan=1>(0/15)</td><td rowspan=1 colspan=1>(0/15)</td><td rowspan=1 colspan=1>(0/15)</td></tr><tr><td rowspan=2 colspan=1>ClinicalNasalMatrix</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0%</td><td rowspan=2 colspan=1>n/a</td><td rowspan=2 colspan=1>n/a</td><td rowspan=2 colspan=1>n/a</td><td rowspan=2 colspan=1>n/a</td><td rowspan=2 colspan=1>24.6</td></tr><tr><td rowspan=1 colspan=1>(0/15)</td><td rowspan=1 colspan=1>(0/15)</td><td rowspan=1 colspan=1>(0/15)</td><td rowspan=1 colspan=1>(0/15)</td></tr><tr><td rowspan=2 colspan=1>SimulatedNP/NSMatrix</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0%</td><td rowspan=2 colspan=1>n/a</td><td rowspan=2 colspan=1>n/a</td><td rowspan=2 colspan=1>n/a</td><td rowspan=2 colspan=1>n/a</td><td rowspan=2 colspan=1>23.2</td></tr><tr><td rowspan=1 colspan=1>(0/15)</td><td rowspan=1 colspan=1>(0/15)</td><td rowspan=1 colspan=1>(0/15)</td><td rowspan=1 colspan=1>(0/15)</td></tr><tr><td rowspan=6 colspan=1>2X LOD</td><td rowspan=2 colspan=1>Clinical NPMatrix</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>97%</td><td rowspan=1 colspan=1>100%</td><td rowspan=2 colspan=1>33.9</td><td rowspan=2 colspan=1>33.2</td><td rowspan=2 colspan=1>33.2</td><td rowspan=2 colspan=1>32.4</td><td rowspan=2 colspan=1>22.6</td></tr><tr><td rowspan=1 colspan=1>(30/30)</td><td rowspan=1 colspan=1>(30/30)</td><td rowspan=1 colspan=1>(29/30)</td><td rowspan=1 colspan=1>(30/30)</td></tr><tr><td rowspan=2 colspan=1>ClinicalNasalMatrix</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=2 colspan=1>34.1</td><td rowspan=2 colspan=1>33.1</td><td rowspan=2 colspan=1>33.8</td><td rowspan=2 colspan=1>32.6</td><td rowspan=2 colspan=1>24.3</td></tr><tr><td rowspan=1 colspan=1>(30/30)</td><td rowspan=1 colspan=1>(30/30)</td><td rowspan=1 colspan=1>(30/30)</td><td rowspan=1 colspan=1>(30/30)</td></tr><tr><td rowspan=2 colspan=1>SimulatedNP/NSMatrix</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>97%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=2 colspan=1>33.7</td><td rowspan=2 colspan=1>32.6</td><td rowspan=2 colspan=1>34.0</td><td rowspan=2 colspan=1>32.5</td><td rowspan=2 colspan=1>23.4</td></tr><tr><td rowspan=1 colspan=1>(30/30)</td><td rowspan=1 colspan=1>(29/30)a</td><td rowspan=1 colspan=1>(30/30)</td><td rowspan=1 colspan=1>(30/30)</td></tr><tr><td rowspan=6 colspan=1>5X LOD</td><td rowspan=2 colspan=1>Clinical NPMatrix</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=2 colspan=1>31.7</td><td rowspan=2 colspan=1>31.5</td><td rowspan=2 colspan=1>32.3</td><td rowspan=2 colspan=1>30.8</td><td rowspan=2 colspan=1>22.5</td></tr><tr><td rowspan=1 colspan=1>(15/15)</td><td rowspan=1 colspan=1>(15/15)</td><td rowspan=1 colspan=1>(15/15)</td><td rowspan=1 colspan=1>(15/15)</td></tr><tr><td rowspan=2 colspan=1>ClinicalNasalMatrix</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=2 colspan=1>32.1</td><td rowspan=2 colspan=1>31.5</td><td rowspan=2 colspan=1>32.6</td><td rowspan=2 colspan=1>31.0</td><td rowspan=2 colspan=1>24.6</td></tr><tr><td rowspan=1 colspan=1>(15/15)</td><td rowspan=1 colspan=1>(15/15)</td><td rowspan=1 colspan=1>(15/15)</td><td rowspan=1 colspan=1>(15/15)</td></tr><tr><td rowspan=2 colspan=1>SimulatedNP/NSMatrix</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=2 colspan=1>31.8</td><td rowspan=2 colspan=1>31.9</td><td rowspan=2 colspan=1>32.4</td><td rowspan=2 colspan=1>31.3</td><td rowspan=2 colspan=1>24.0</td></tr><tr><td rowspan=1 colspan=1>(15/15)</td><td rowspan=1 colspan=1>(15/15)</td><td rowspan=1 colspan=1>(15/15)</td><td rowspan=1 colspan=1>(15/15)</td></tr></table>

a There was one non-reportable result during the study

# C Clinical Studies:

# Prospective Clinical Study

The clinical performance of the BD Respiratory Viral Panel for BD MAX System was established in a multi-center prospective study conducted with paired nasopharyngeal swab (NPS) and anterior nasal swab (ANS) specimens in UVT/UTM, collected following receipt of informed consent from study participants. All specimens were prospectively collected (i.e., all comers between two time points that met the clinical study inclusion criteria) from patients with signs and symptoms of respiratory tract infections during the 2022 respiratory illness season. Subject matched, paired NPS and ANS specimens were collected from six (6) geographically diverse clinical sites, four (4) in the U.S. and two (2) in Europe and were

tested with the BD Respiratory Viral Panel for BD MAX System at four (4) U.S. testing sites.

The BD Respiratory Viral Panel for BD MAX System was evaluated for SARS-CoV-2 performance by comparing the candidate device testing results to a composite comparator algorithm (CCA) consisting of three (3) highly sensitive U.S. FDA EUA SARS-CoV-2 molecular tests. A final CCA result was assigned when two of the three composite comparator assays were in concordance. The comparator method utilized to establish performance for the Flu A, Flu B, and RSV targets was a U.S. FDA-cleared molecular Flu A/B/RSV assay. All comparator testing was performed in accordance with the respective package inserts.

The BD Respiratory Viral Panel for BD MAX System was evaluated with both Category I specimens (i.e., prospectively collected, tested fresh) and Category II specimens (i.e., prospectively collected, tested frozen/thawed).

A total of 2005 paired NPS and ANS specimens were enrolled for the prospective clinical study between January 2022 and August 2022. Category I specimens were collected from April 2022 to August 2022, while Category II specimens were collected from January 2022 to April 2022.

Of these 2005 subjects, 360 were excluded as non-compliant (protocol deviations, etc.). This left a total of 1645 Compliant Subjects. A demographic summary of the compliant subjects is shown in Table 23.

For the SARS-CoV-2 evaluation, an additional $1 0 0 \mathrm { N P }$ specimens were excluded (protocol deviations, etc.) leaving 1545 NPS specimens for performance evaluation. Similarly, an additional 84 NS specimens were excluded (protocol deviations, etc.) leaving 1561 ANS specimens for evaluation.

Of the 1545 valid NPS specimens, 1021 $( 6 6 . 1 \% )$ were tested fresh (Category I specimens) and 524 $( 3 3 . 9 \% )$ were tested frozen/thawed (Category II specimens) with the BD Respiratory Viral Panel for BD MAX System. Of the 1561 valid ANS specimens, 1021 $( 6 5 . 4 \% )$ were tested fresh and 540 $( 3 4 . 6 \% )$ were tested frozen/thawed with the BD Respiratory Viral Panel for BD MAX System.

For the Flu A, Flu B, and RSV evaluations, an additional $8 3 \mathrm { N S }$ specimens were excluded (protocol deviations, etc.) leaving a total of $1 5 6 2 \mathrm { N P S }$ specimens. For the NS specimens, an additional 81 specimens were excluded (protocol deviations, etc.) leaving 1564 ANS specimens for evaluation.

Of the 1562 valid NPS specimens, 1022 $( 6 5 . 4 \% )$ were tested fresh and 540 $( 3 4 . 6 \% )$ were tested frozen/thawed with the BD Respiratory Viral Panel for BD MAX. Of the 1564 valid NS specimens, 1025 $( 6 5 . 5 \% )$ were tested fresh and 539 $( 3 4 . 5 \% )$ were tested frozen/thawed with the BD Respiratory Viral Panel for BD MAX System.

Table 23. Prospective Specimens: Demographic Summary of Compliant Subjects   

<table><tr><td rowspan=1 colspan=1>Demographics</td><td rowspan=1 colspan=1>Characteristics</td><td rowspan=1 colspan=1>Total (N=1645)</td></tr><tr><td rowspan=2 colspan=1>Gender</td><td rowspan=1 colspan=1>Female</td><td rowspan=1 colspan=1>61.8% (1016/1645)</td></tr><tr><td rowspan=1 colspan=1>Male</td><td rowspan=1 colspan=1>38.2% (629/1645)</td></tr><tr><td rowspan=4 colspan=1>Age Group</td><td rowspan=1 colspan=1>0 -5 years</td><td rowspan=1 colspan=1>1.2% (20/1645)</td></tr><tr><td rowspan=1 colspan=1>6 - 21 years</td><td rowspan=1 colspan=1>10.3% (170/1645)</td></tr><tr><td rowspan=1 colspan=1>22 - 59 years</td><td rowspan=1 colspan=1>58.2% (957/1645)</td></tr><tr><td rowspan=1 colspan=1>&gt; 59 years</td><td rowspan=1 colspan=1>30.3% (498/1645)</td></tr><tr><td rowspan=3 colspan=1>Ethnicity</td><td rowspan=1 colspan=1>Hispanic / Latino</td><td rowspan=1 colspan=1>47.1% (775/1645)</td></tr><tr><td rowspan=1 colspan=1>Non-Hispanic / Latino</td><td rowspan=1 colspan=1>49.5% (814/1645)</td></tr><tr><td rowspan=1 colspan=1>Not Reported</td><td rowspan=1 colspan=1>3.4% (56/1645)</td></tr><tr><td rowspan=8 colspan=1>Race</td><td rowspan=1 colspan=1>American Indian or Alaska Native</td><td rowspan=1 colspan=1>0.3% (5/1645)</td></tr><tr><td rowspan=1 colspan=1>Asian</td><td rowspan=1 colspan=1>0.3% (5/1645)</td></tr><tr><td rowspan=1 colspan=1>Black or African American</td><td rowspan=1 colspan=1>13.3% (219/1645)</td></tr><tr><td rowspan=1 colspan=1>Native Hawaiian or Other Pacific Islander</td><td rowspan=1 colspan=1>0.0% (0/1645)</td></tr><tr><td rowspan=1 colspan=1>White</td><td rowspan=1 colspan=1>84.8% (1395/1645)</td></tr><tr><td rowspan=1 colspan=1>Mixed Race</td><td rowspan=1 colspan=1>0.9% (15/1645)</td></tr><tr><td rowspan=1 colspan=1>Not Reported</td><td rowspan=1 colspan=1>0.1% (1/1645)</td></tr><tr><td rowspan=1 colspan=1>Unknown</td><td rowspan=1 colspan=1>0.3% (5/1645)</td></tr><tr><td rowspan=4 colspan=1>Patient Population</td><td rowspan=1 colspan=1>Outpatient</td><td rowspan=1 colspan=1>95.9% (1578/1645)</td></tr><tr><td rowspan=1 colspan=1>Hospitalized</td><td rowspan=1 colspan=1>3.0% (49/1645)</td></tr><tr><td rowspan=1 colspan=1>Emergency</td><td rowspan=1 colspan=1>1.0% (17/1645)</td></tr><tr><td rowspan=1 colspan=1>Unknown</td><td rowspan=1 colspan=1>0.1% (1/1645)</td></tr><tr><td rowspan=3 colspan=1>Immuno-compromised</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>3.2% (52/1645)</td></tr><tr><td rowspan=1 colspan=1>No</td><td rowspan=1 colspan=1>95.2% (1566/1645)</td></tr><tr><td rowspan=1 colspan=1>Unknown</td><td rowspan=1 colspan=1>1.6% (27/1645)</td></tr></table>

A summary of the BD Respiratory Viral Panel for BD MAX System assay prospective clinical study performance is provided in Tables 24 & 25. Positive Percent Agreement (PPA) was calculated as $1 0 0 \% \times \left( \mathrm { T P } / \left( \mathrm { T P } + \mathrm { F N } \right) \right)$ . True positive (TP) indicates that both the BD Respiratory Viral Panel for BD MAX System assay and the comparator method had a positive result for the specific analyte, and false negative (FN) indicates that the BD Respiratory Viral Panel for BD MAX System was negative while the comparator result was positive. Negative Percent Agreement (NPA) was calculated as $1 0 0 \% \times \left( \mathrm { T N / \left( T N + F P \right) } \right)$ . True negative (TN) indicates that both the BD Respiratory Viral Panel for BD MAX System assay and the comparator method had negative results, and false positive (FP) indicates that the BD Respiratory Viral Panel for BD MAX System assay was positive while the comparator result was negative.

Table 24. BD Respiratory Viral Panel for BD MAX System Performance with Prospective NPS Specimens (Fresh, Frozen, and Fresh/Frozen Combined)   

<table><tr><td colspan="2" rowspan="2">Analyte</td><td colspan="3" rowspan="1">Positive Percent Agreement</td><td colspan="3" rowspan="1">Negative Percent Agreement</td></tr><tr><td colspan="1" rowspan="1">TP/(TP+FN)</td><td colspan="1" rowspan="1">%</td><td colspan="1" rowspan="1">95% CI</td><td colspan="1" rowspan="1">TN/(TN+FP)</td><td colspan="1" rowspan="1">%</td><td colspan="1" rowspan="1">95% CI</td></tr><tr><td colspan="1" rowspan="3">SARS-CoV-2a</td><td colspan="1" rowspan="1">Fresh</td><td colspan="1" rowspan="1">370/372</td><td colspan="1" rowspan="1">99.5</td><td colspan="1" rowspan="1">98.1-99.9</td><td colspan="1" rowspan="1">641/649</td><td colspan="1" rowspan="1">98.8</td><td colspan="1" rowspan="1">97.6-99.4</td></tr><tr><td colspan="1" rowspan="1">Frozen</td><td colspan="1" rowspan="1">147/151</td><td colspan="1" rowspan="1">97.4</td><td colspan="1" rowspan="1">93.4-99.0</td><td colspan="1" rowspan="1">358/373</td><td colspan="1" rowspan="1">96.0</td><td colspan="1" rowspan="1">93.5-97.5</td></tr><tr><td colspan="1" rowspan="1">Combined</td><td colspan="1" rowspan="1">517/523</td><td colspan="1" rowspan="1">98.9</td><td colspan="1" rowspan="1">97.5-99.5</td><td colspan="1" rowspan="1">999/1022</td><td colspan="1" rowspan="1">97.7</td><td colspan="1" rowspan="1">96.6-98.5</td></tr><tr><td colspan="1" rowspan="2">Flu Ab</td><td colspan="1" rowspan="1">Fresh</td><td colspan="1" rowspan="1">58/60</td><td colspan="1" rowspan="1">96.7</td><td colspan="1" rowspan="1">88.6-99.1</td><td colspan="1" rowspan="1">955/962</td><td colspan="1" rowspan="1">99.3</td><td colspan="1" rowspan="1">98.5-99.6</td></tr><tr><td colspan="1" rowspan="1">Frozen</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">20.7-100</td><td colspan="1" rowspan="1">539/539</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">99.3-100</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Combined</td><td colspan="1" rowspan="1">59/61</td><td colspan="1" rowspan="1">96.7</td><td colspan="1" rowspan="1">88.8-99.1</td><td colspan="1" rowspan="1">1494/1501</td><td colspan="1" rowspan="1">99.5</td><td colspan="1" rowspan="1">99.0-99.8</td></tr><tr><td colspan="1" rowspan="3">Flu Bc</td><td colspan="1" rowspan="1">Fresh</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">NC</td><td colspan="1" rowspan="1">1021/1022</td><td colspan="1" rowspan="1">99.9</td><td colspan="1" rowspan="1">99.4-100</td></tr><tr><td colspan="1" rowspan="1">Frozen</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">NC</td><td colspan="1" rowspan="1">540/540</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">99.3-100</td></tr><tr><td colspan="1" rowspan="1">Combined</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">NC</td><td colspan="1" rowspan="1">1561/1562</td><td colspan="1" rowspan="1">99.9</td><td colspan="1" rowspan="1">99.6-100</td></tr><tr><td colspan="1" rowspan="3">RSV</td><td colspan="1" rowspan="1">Fresh</td><td colspan="1" rowspan="1">11/11</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">74.1-100</td><td colspan="1" rowspan="1">1011/1011</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">99.6-100</td></tr><tr><td colspan="1" rowspan="1">Frozen</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">20.7-100</td><td colspan="1" rowspan="1">539/539</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">99.3-100</td></tr><tr><td colspan="1" rowspan="1">Combined</td><td colspan="1" rowspan="1">12/12</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">75.8-100</td><td colspan="1" rowspan="1">1550/1550</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">99.8-100</td></tr></table>

TP – true positive; FN – false negative; TN – true negative; FP – false positive; NC – not calculable a SARS-CoV-2 was detected in 3/6 FN specimens with all three composite comparator methods. SARS-CoV-2 was detected in $1 5 / 2 3 \ \mathrm { F P }$ specimens with one of the three composite comparator methods.

b Flu A was detected in both FN specimens when tested with an independent molecular method. Flu A was detected in 3/7 FP specimens when tested with an independent molecular method. Flu A was Equivocal in 1/7 FP specimens when tested with an independent method.

c Flu B was not detected in the single FP specimen when tested with an independent molecular method.

Table 25. BD Respiratory Viral Panel for BD MAX System Performance with Prospective ANS Specimens (Fresh, Frozen, and Fresh/Frozen Combined)   

<table><tr><td rowspan=2 colspan=2>Analyte</td><td rowspan=1 colspan=3>Positive Percent Agreement</td><td rowspan=1 colspan=3>Negative Percent Agreement</td></tr><tr><td rowspan=1 colspan=1>TP/(TP+FN)</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=1>TN/(TN+FP)</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=3 colspan=1>SARS-CoV-2a</td><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>340/344</td><td rowspan=1 colspan=1>98.8</td><td rowspan=1 colspan=1>97.0-99.5</td><td rowspan=1 colspan=1>665/677</td><td rowspan=1 colspan=1>98.2</td><td rowspan=1 colspan=1>96.9-99.0</td></tr><tr><td rowspan=1 colspan=1>Frozen</td><td rowspan=1 colspan=1>138/142</td><td rowspan=1 colspan=1>97.2</td><td rowspan=1 colspan=1>93.0-98.9</td><td rowspan=1 colspan=1>385/398</td><td rowspan=1 colspan=1>96.7</td><td rowspan=1 colspan=1>94.5-98.1</td></tr><tr><td rowspan=1 colspan=1>Combined</td><td rowspan=1 colspan=1>478/486</td><td rowspan=1 colspan=1>98.4</td><td rowspan=1 colspan=1>96.8-99.2</td><td rowspan=1 colspan=1>1050/1075</td><td rowspan=1 colspan=1>97.7</td><td rowspan=1 colspan=1>96.6-98.4</td></tr><tr><td rowspan=3 colspan=1>Flu Ab</td><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>61/63</td><td rowspan=1 colspan=1>96.8</td><td rowspan=1 colspan=1>89.1-99.1</td><td rowspan=1 colspan=1>958/962</td><td rowspan=1 colspan=1>99.6</td><td rowspan=1 colspan=1>98.9-99.8</td></tr><tr><td rowspan=1 colspan=1>Frozen</td><td rowspan=1 colspan=1>1/1</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>20.7-100</td><td rowspan=1 colspan=1>538/538</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>99.3-100</td></tr><tr><td rowspan=1 colspan=1>Combined</td><td rowspan=1 colspan=1>62/64</td><td rowspan=1 colspan=1>96.9</td><td rowspan=1 colspan=1>89.3</td><td rowspan=1 colspan=1>1496/1500</td><td rowspan=1 colspan=1>99.7</td><td rowspan=1 colspan=1>99.3-99.9</td></tr><tr><td rowspan=3 colspan=1>Flu B</td><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>NC</td><td rowspan=1 colspan=1>1025/1025</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>99.3-100</td></tr><tr><td rowspan=1 colspan=1>Frozen</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>NC</td><td rowspan=1 colspan=1>539/539</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>99.3-100</td></tr><tr><td rowspan=1 colspan=1>Combined</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>NC</td><td rowspan=1 colspan=1>1564/1564</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>99.8-100</td></tr><tr><td rowspan=3 colspan=1>RSVc</td><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>11/11</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>74.1-100</td><td rowspan=1 colspan=1>1013/1014</td><td rowspan=1 colspan=1>99.9</td><td rowspan=1 colspan=1>99.4-100</td></tr><tr><td rowspan=1 colspan=1>Frozen</td><td rowspan=1 colspan=1>0/1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0-79.3</td><td rowspan=1 colspan=1>538/538</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>99.3-100</td></tr><tr><td rowspan=1 colspan=1>Combined</td><td rowspan=1 colspan=1>11/12</td><td rowspan=1 colspan=1>91.7</td><td rowspan=1 colspan=1>64.6-98.5</td><td rowspan=1 colspan=1>1551/1552</td><td rowspan=1 colspan=1>99.9</td><td rowspan=1 colspan=1>99.6-100</td></tr></table>

TP – true positive; FN – false negative; TN – true negative; FP – false positive; NC – not calculable a SARS-CoV-2 was detected in $2 / 8 \ \mathrm { F N }$ specimens with all three composite comparator methods. SARS-CoV-2 was detected in 17/25 FP specimens with one of the three composite comparator methods.

b Flu A was not detected in both FN specimens when tested with an independent molecular method. Flu A was detected in 1/4 FP specimens when tested with an independent molecular method.

c RSV was detected in the single FN specimen when tested with an independent molecular method. RSV was detected in the single FP specimen when tested with an independent molecular method.

# Non-Reportable Results Rates

Table 26 and Table 27 provide the non-reportable results rates for nasopharyngeal (NP) and anterior nasal swab (ANS) specimens, respectively.

Table 26. Prospective NP Specimens: SARS-CoV-2/Flu/RSV UNR, IND, INC and Total Non-reportable Rate   

<table><tr><td colspan="1" rowspan="2">SampleType(NP)</td><td colspan="2" rowspan="1">Unresolved (UNR)</td><td colspan="2" rowspan="1">Indeterminate (IND)</td><td colspan="2" rowspan="1">Incomplete (INC)</td><td colspan="2" rowspan="1">Total Non-Reportable(UNR+IND+INC)</td></tr><tr><td colspan="1" rowspan="1">Initial(95% CI)</td><td colspan="1" rowspan="1">Valid Repeat(95% CI)</td><td colspan="1" rowspan="1">Initial(95% CI)</td><td colspan="1" rowspan="1">Valid Repeat(95% CI)</td><td colspan="1" rowspan="1">Initial(95% CI)</td><td colspan="1" rowspan="1">Valid Repeat(95% CI)</td><td colspan="1" rowspan="1">Initial(95% CI)</td><td colspan="1" rowspan="1">Valid Repeat(95% CI)</td></tr><tr><td>Fresh</td><td>0.0% (0/1022) (0.0%, 0.4%)</td><td>0.0% (0/1022) (0.0%, 0.4%) 0.2%</td><td>0.0% (0/1022) (0.0%, 0.4%) 0.0%</td><td>0.0% (0/1022) (0.0%, 0.4%)</td><td>1.3% (13/1022) (0.7%, 2.2%)</td><td>0.0% (0/1022) (0.0%, 0.4%)</td><td>1.3% (13/1022) (0.7%, 2.2%)</td><td>0.0% (0/1022) (0.0%, 0.4%)</td></tr><tr><td>Frozen</td><td>0.2% (1/541) (0.0%, 1.0%) 0.1%</td><td>(1/541) (0.0%, 1.0%) 0.1%</td><td>(0/541) (0.0%, 0.7%) 0.0%</td><td>0.0% (0/541) (0.0%, 0.7%) 0.0%</td><td>0.0% (0/541) (0.0%, 0.7%) 0.8%</td><td>0.0% (0/541) (0.0%, 0.7%) 0.0%</td><td>0.2% (1/541) (0.0%, 1.0%) 0.9%</td><td>0.2% (1/541) (0.0%, 1.0%) 0.1%</td></tr><tr><td>Total</td><td>(1/1563) (0.0%, 0.4%)</td><td>(1/1563) (0.0%, 0.4%)</td><td>(0/1563) (0.0%, 0.2%)</td><td>(0/1563) (0.0%, 0.2%)</td><td>(13/1563) (0.5%, 1.4%)</td><td>(0/1563) (0.0%, 0.2%)</td><td>(14/1563) (0.5%, 1.5%)</td><td>(1/1563) (0.0%, 0.4%)</td></tr></table>

Table 27. Prospective ANS Specimens: SARS-CoV-2/Flu/RSV UNR, IND, INC and Total Non-reportable Rate   

<table><tr><td rowspan=2 colspan=1>SampleType(ANS)</td><td rowspan=1 colspan=2>Unresolved (UNR)</td><td rowspan=1 colspan=2>Indeterminate (IND)</td><td rowspan=1 colspan=2>Incomplete (INC)</td><td rowspan=1 colspan=2>Total Non-Reportable(UNR+IND+INC)</td></tr><tr><td rowspan=1 colspan=1>Initial(95% CI)</td><td rowspan=1 colspan=1>Valid Repeat(95% CI)</td><td rowspan=1 colspan=1>Initial(95% CI)</td><td rowspan=1 colspan=1>Valid Repeat(95% CI)</td><td rowspan=1 colspan=1>Initial(95% CI)</td><td rowspan=1 colspan=1>Valid Repeat(95% CI)</td><td rowspan=1 colspan=1>Initial(95% CI)</td><td rowspan=1 colspan=1>Valid Repeat(95% CI)</td></tr><tr><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>0.1%(1/1027)(0.0%, 0.5%)</td><td rowspan=1 colspan=1>0.1%(1/1027)(0.0%, 0.5%)</td><td rowspan=1 colspan=1>0.0%(0/1027)(0.0%, 0.4%)</td><td rowspan=1 colspan=1>0.0%(0/1027)(0.0%, 0.4%)</td><td rowspan=1 colspan=1>1.3%(13/1027)(0.7%, 2.2%)</td><td rowspan=1 colspan=1>0.0%(0/1027)(0.0%, 0.4%)</td><td rowspan=1 colspan=1>1.4%(14/1027)(0.8%, 2.3%)</td><td rowspan=1 colspan=1>0.1%(1/1027)(0.0%, 0.5%)</td></tr><tr><td rowspan=1 colspan=1>Frozen</td><td rowspan=1 colspan=1>0.2%(1/541)(0.0%, 1.0%)</td><td rowspan=1 colspan=1>0.2%(1/541)(0.0%, 1.0%)</td><td rowspan=1 colspan=1>0.4%(2/541)(0.1%, 1.3%)</td><td rowspan=1 colspan=1>0.0%(0/541)(0.0%, 0.7%)</td><td rowspan=1 colspan=1>0.0%(0/541)(0.0%, 0.7%)</td><td rowspan=1 colspan=1>0.0%(0/541)(0.0%, 0.7%)</td><td rowspan=1 colspan=1>0.6%(3/541)(0.2%, 1.6%)</td><td rowspan=1 colspan=1>0.2%(1/541)(0.0%, 1.0%)</td></tr><tr><td rowspan=2 colspan=1>Total</td><td rowspan=2 colspan=1>0.1%(2/1568)(0.0%, 0.5%)</td><td rowspan=1 colspan=1>0.1%(2/1568)</td><td rowspan=1 colspan=1>0.1%(2/1568)</td><td rowspan=1 colspan=1>0.0%(0/1568)</td><td rowspan=1 colspan=1>0.8%(13/1568)</td><td rowspan=1 colspan=1>0.0%(0/1568)</td><td rowspan=2 colspan=1>1.1%(17/1568)(0.7%, 1.7%)</td><td rowspan=2 colspan=1>0.1%(2/1568)(0.0%, 0.5%)</td></tr><tr><td rowspan=1 colspan=1>(0.0%, 0.5%)</td><td rowspan=1 colspan=1>(0.0%, 0.5%)</td><td rowspan=1 colspan=1>(0.0%, 0.2%)</td><td rowspan=1 colspan=1>(0.5%, 1.4%)</td><td rowspan=1 colspan=1>(0.0%, 0.2%)</td></tr></table>

# Co-infections Detected by the BD Respiratory Viral Panel for BD MAX Assay in the Prospectively Collected Specimens

The BD Respiratory Viral Panel for BD MAX System detected a total of 4 of the 5 coinfections (3 ANS, 1 NP) in which SARS-CoV-2 and Flu A were detected by the comparators. Only Flu A was detected by the candidate in the remaining sample (ANS).

The candidate assay detected 2 additional samples with SARS-CoV-2 and Flu A co-infections (1 ANS, 1 NP) and 1 additional sample with SARS-CoV-2 and Flu B co-infection (NP). However, the comparators detected only SARS-CoV-2 in all three samples.

# Retrospective Clinical Study

Flu B and RSV were of low prevalence and were not encountered in sufficiently large numbers during the prospective clinical study to adequately demonstrate assay performance. To supplement the results of the prospective clinical study, an evaluation of 20 RSV standard of care (SoC) positive and 14 Flu B SoC positive Category III ANS specimens was undertaken. These specimens, in addition to negative specimens collected under a BD collection protocol, in UVT/UTM, were collected between February 2021 and February 2023 and were randomized and de-identified. Five (5) of the 20 RSV SoC positive samples provided by the source laboratory, that were not confirmed as positive by the comparator, are included in the total $\mathrm { N } { = } 1 8 7$ . They were counted as part of the Negative Percent Agreement (NPA). One (1) specimen generated an Unresolved (UNR) assay result for both Flu B and RSV and was excluded from the performance analysis. A summary of the demographic information of the retrospective ANS samples is provided in Table 28.

Table 28. Retrospective ANS Specimens. Demographic Summary of Compliant Subjects   

<table><tr><td rowspan=1 colspan=1>Demographics</td><td rowspan=1 colspan=1>Characteristics</td><td rowspan=1 colspan=1>Total (N=187)</td></tr><tr><td rowspan=2 colspan=1>Gender</td><td rowspan=1 colspan=1>Female</td><td rowspan=1 colspan=1>50.8% (95/187)</td></tr><tr><td rowspan=1 colspan=1>Male</td><td rowspan=1 colspan=1>49.2%(92/187)</td></tr><tr><td rowspan=4 colspan=1>Age Group</td><td rowspan=1 colspan=1>0 - 5 years</td><td rowspan=1 colspan=1>8.6% (16/187)</td></tr><tr><td rowspan=1 colspan=1>6 - 21 years</td><td rowspan=1 colspan=1>16.0%(30/187)</td></tr><tr><td rowspan=1 colspan=1>22 - 59 years</td><td rowspan=1 colspan=1>62.0%(116/187)</td></tr><tr><td rowspan=1 colspan=1>&gt; 59 years</td><td rowspan=1 colspan=1>13.4% (25/187)</td></tr></table>

A summary of the BD Respiratory Viral Panel for BD MAX System assay retrospective ANS specimen clinical study performance is provided in Table 29.

Table 29. BD Respiratory Viral Panel for BD MAX System Flu B and RSV Performance with Retrospective ANS Specimens   

<table><tr><td rowspan=2 colspan=2>Analyte1</td><td rowspan=1 colspan=3>Positive Percent Agreement</td><td rowspan=1 colspan=3>Negative Percent Agreement</td></tr><tr><td rowspan=1 colspan=1>TP/(TP+FN)</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=1>TN/(TN+FP)</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=1>Flu B</td><td rowspan=1 colspan=1>Retrospective</td><td rowspan=1 colspan=1>12/12</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>75.7%-100%</td><td rowspan=1 colspan=1>172/174</td><td rowspan=1 colspan=1>98.9%</td><td rowspan=1 colspan=1>95.9%-99.7%</td></tr><tr><td rowspan=1 colspan=1>RSV</td><td rowspan=1 colspan=1>Retrospective</td><td rowspan=1 colspan=1>15/15</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>79.6%-100%</td><td rowspan=1 colspan=1>170/171</td><td rowspan=1 colspan=1>99.4%</td><td rowspan=1 colspan=1>96.8%-99.9%</td></tr></table>

TP – true positive; FN – false negative; TN – true negative; FP – false positive 1 One specimen generated an Unresolved (UNR) assay result for both Flu B and RSV. An UNR is generated when the target result and the Sample Processing Control (SPC) are both negative and no instrument error was recorded.

In addition, clinical performance characteristics of the BD Respiratory Viral Panel for BD MAX System were also assessed from a total of 240 frozen retrospective nasopharyngeal swabs (NPS) in UVT/UTM obtained from two (2) external sources with historical positive or negative results for either influenza B or RSV. The specimens were collected as part of routine patient care between December 2019 and January 2022. All the specimens were tested in a blinded and randomized fashion with the BD Respiratory Viral Panel for BD MAX System at least three different testing sites and FDA-cleared high sensitivity RT- PCR assay as comparator at one reference testing site. Table 30 summarizes the demographic information (gender and age group) by percentage for compliant subjects for retrospective NPS specimens.

Table 30. Retrospective NPS Specimens: Demographic Summary of Compliant Subjects   

<table><tr><td rowspan=1 colspan=1>Demographics</td><td rowspan=1 colspan=1>Characteristics</td><td rowspan=1 colspan=1>Total (N=240)</td></tr><tr><td rowspan=3 colspan=1>Gender</td><td rowspan=1 colspan=1>Female</td><td rowspan=1 colspan=1>37.5% (90/240)</td></tr><tr><td rowspan=1 colspan=1>Male</td><td rowspan=1 colspan=1>37.5% (90/240)</td></tr><tr><td rowspan=1 colspan=1>Unknown</td><td rowspan=1 colspan=1>25.0% (60/240)</td></tr><tr><td rowspan=2 colspan=1>Age Group</td><td rowspan=1 colspan=1>0 - 5 years</td><td rowspan=1 colspan=1>26.7% (64/240)</td></tr><tr><td rowspan=1 colspan=1>6 - 21 years</td><td rowspan=1 colspan=1>20.8% (50/240)</td></tr></table>

<table><tr><td>22 - 59 years</td><td>32.5% (78/240)</td></tr><tr><td>&gt; 59 years</td><td>20.0% (48/240)</td></tr></table>

A summary of the BD Respiratory Viral Panel for BD MAX System assay retrospective NPS clinical study performance is provided in Table 31.

Table 31. BD Respiratory Viral Panel for BD MAX System Flu B and RSV Performance with Retrospective NPS Specimens   

<table><tr><td rowspan=2 colspan=2>Analyte</td><td rowspan=1 colspan=3>Positive Percent Agreement</td><td rowspan=1 colspan=3>Negative Percent Agreement</td></tr><tr><td rowspan=1 colspan=1>TP/(TP+FN)</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=1>TN/(TN+FP)</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=1>Flu B</td><td rowspan=1 colspan=1>Retrospective</td><td rowspan=1 colspan=1>58/58</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>93.8%-100%</td><td rowspan=1 colspan=1>180/182</td><td rowspan=1 colspan=1>98.9%</td><td rowspan=1 colspan=1>96.1%-99.7%</td></tr><tr><td rowspan=1 colspan=1>RSV</td><td rowspan=1 colspan=1>Retrospective</td><td rowspan=1 colspan=1>62/63</td><td rowspan=1 colspan=1>98.4%</td><td rowspan=1 colspan=1>91.5%-99.7%</td><td rowspan=1 colspan=1>177/177</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>97.9%-100%</td></tr></table>

TP – true positive; FN – false negative; TN – true negative; FP – false positive

1. Other Clinical Supportive Data (When 1. and 2. Are Not Applicable): Not applicable.

D Clinical Cut-Off: Not applicable.

# E Expected Values/Reference Range:

The BD MAX RVP SARS-CoV-2/Flu A/B/RSV assay prospective clinical study included a total of 1562 prospectively collected NPS specimens and 1566 prospectively collected ANS specimens. The stratification of positivity rates by analyte (as determined by the BD MAX RVP SARS-CoV-2/Flu A/B/RSV test) and location for the prospective samples is shown in Table 32. Sample collection was carried out at 8 geographic locations. For the CTMD site, the same principal investigator had oversight of three clinics (two (2) in Florida and one (1) in North Carolina). Data from those three (3) clinics were aggregated.

Table 32. Prospective Specimens1: Positivity Rates based on BD Max RVP   

<table><tr><td colspan="1" rowspan="1">Target</td><td colspan="1" rowspan="1">Collection Site²</td><td colspan="1" rowspan="1">NPS</td><td colspan="1" rowspan="1">ANS</td></tr><tr><td colspan="1" rowspan="7">SARS-CoV-2</td><td colspan="1" rowspan="1">BUL</td><td colspan="1" rowspan="1">60.9% (28/46)</td><td colspan="1" rowspan="1">59.6% (28/47)</td></tr><tr><td colspan="1" rowspan="1">CMR</td><td colspan="1" rowspan="1">18.9% (62/328)</td><td colspan="1" rowspan="1">17.1% (56/328)</td></tr><tr><td colspan="1" rowspan="1">CTMD</td><td colspan="1" rowspan="1">39.9% (311/780)</td><td colspan="1" rowspan="1">37.2% (291/783)</td></tr><tr><td colspan="1" rowspan="1">MAC</td><td colspan="1" rowspan="1">38.0% (101/266)</td><td colspan="1" rowspan="1">35.5% (94/265)</td></tr><tr><td colspan="1" rowspan="1">ROM</td><td colspan="1" rowspan="1">28.9% (37/128)</td><td colspan="1" rowspan="1">25.8% (33/128)</td></tr><tr><td colspan="1" rowspan="1">VHMC</td><td colspan="1" rowspan="1">14.3% (2/14)</td><td colspan="1" rowspan="1">13.3% (2/15)</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">34.6% (541/1562)</td><td colspan="1" rowspan="1">32.2% (504/1566)</td></tr><tr><td colspan="1" rowspan="6">Flu A</td><td colspan="1" rowspan="1">BUL</td><td colspan="1" rowspan="1">0.0% (0/46)</td><td colspan="1" rowspan="1">0.0% (0/47)</td></tr><tr><td colspan="1" rowspan="1">CMR</td><td colspan="1" rowspan="1">3.4% (11/328)</td><td colspan="1" rowspan="1">3.0% (10/328)</td></tr><tr><td colspan="1" rowspan="1">CTMD</td><td colspan="1" rowspan="1">6.7% (52/780)</td><td colspan="1" rowspan="1">6.6% (52/783)</td></tr><tr><td colspan="1" rowspan="1">MAC</td><td colspan="1" rowspan="1">0.0% (0/266)</td><td colspan="1" rowspan="1">0.0% (0/265)</td></tr><tr><td colspan="1" rowspan="1">ROM</td><td colspan="1" rowspan="1">0.0% (0/128)</td><td colspan="1" rowspan="1">0.0% (0/128)</td></tr><tr><td colspan="1" rowspan="1">VHMC</td><td colspan="1" rowspan="1">21.4% (3/14)</td><td colspan="1" rowspan="1">26.7% (4/15)</td></tr><tr><td colspan="1" rowspan="1">EY</td><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">4.2% (66/1562)</td><td colspan="1" rowspan="1">4.2% (66/1566)</td></tr><tr><td colspan="1" rowspan="7">Flu B</td><td colspan="1" rowspan="1">BUL</td><td colspan="1" rowspan="1">0.0% (0/46)</td><td colspan="1" rowspan="1">0.0% (0/47)</td></tr><tr><td colspan="1" rowspan="1">CMR</td><td colspan="1" rowspan="1">0.0% (0/328)</td><td colspan="1" rowspan="1">0.0% (0/328)</td></tr><tr><td colspan="1" rowspan="1">CTMD</td><td colspan="1" rowspan="1">0.1% (1/780)</td><td colspan="1" rowspan="1">0.0% (0/783)</td></tr><tr><td colspan="1" rowspan="1">MAC</td><td colspan="1" rowspan="1">0.0% (0/266)</td><td colspan="1" rowspan="1">0.0% (0/265)</td></tr><tr><td colspan="1" rowspan="1">ROM</td><td colspan="1" rowspan="1">0.0% (0/128)</td><td colspan="1" rowspan="1">0.0% (0/128)</td></tr><tr><td colspan="1" rowspan="1">VHMC</td><td colspan="1" rowspan="1">0.0% (0/14)</td><td colspan="1" rowspan="1">0.0% (0/15)</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">0.1% (1/1562)</td><td colspan="1" rowspan="1">0.0% (0/1566)</td></tr><tr><td colspan="1" rowspan="7">RSV</td><td colspan="1" rowspan="1">BUL</td><td colspan="1" rowspan="1">0.0% (0/46)</td><td colspan="1" rowspan="1">0.0% (0/47)</td></tr><tr><td colspan="1" rowspan="1">CMR</td><td colspan="1" rowspan="1">0.9% (3/328)</td><td colspan="1" rowspan="1">0.9% (3/328)</td></tr><tr><td colspan="1" rowspan="1">CTMD</td><td colspan="1" rowspan="1">1.0% (8/780)</td><td colspan="1" rowspan="1">1.1% (9/783)</td></tr><tr><td colspan="1" rowspan="1">MAC</td><td colspan="1" rowspan="1">0.0% (0/266)</td><td colspan="1" rowspan="1">0.0% (0/265)</td></tr><tr><td colspan="1" rowspan="1">ROM</td><td colspan="1" rowspan="1">0.8% (1/128)</td><td colspan="1" rowspan="1">0.0% (0/128)</td></tr><tr><td colspan="1" rowspan="1">VHMC</td><td colspan="1" rowspan="1">0.0% (0/14)</td><td colspan="1" rowspan="1">0.0% (0/15)</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">0.8% (12/1562)</td><td colspan="1" rowspan="1">0.8% (12/1566)</td></tr></table>

1 Calculations are based on specimens that are compliant at the subject level, and compliant and reportable for MAX, regardless of the comparator compliancy and results. $^ { 2 } \mathrm { B U L }$ (Sofia, Bulgaria), CMR (Providence, RI), CTMD (Palm Springs, FL / Ashville, NC / Coconut Creek, FL), MAC (Savannah, GA), ROM (Bucuresti, Romania), VHMC (Phoenix, AZ)

# F Other Supportive Instrument Performance Characteristics Data:

Not Applicable

# VIII Proposed Labeling:

The labeling supports the finding of substantial equivalence for this device.

# IX Conclusion:

The submitted information in this premarket notification is complete and supports a substantial equivalence decision.