# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATIONDECISION SUMMARYASSAY AND INSTRUMENT

I Background Information:

A 510(k) Number K230440   
B Applicant Cepheid   
C Proprietary and Established Names Xpert Xpress CoV-2 plus   
D Regulatory Information

<table><tr><td>Product Code(s)</td><td>Classification</td><td>Regulation Section 21 CFR 866.3981 - Device</td><td>Panel</td></tr><tr><td>QQX</td><td>Class II</td><td>To Detect And Identify Nucleic Acid Targets In Respiratory Specimens From Microbial Agents That Cause The SARS- Cov-2 Respiratory Infection And Other Microbial Agents When In A Multi-Target Test</td><td>MI - Microbiology</td></tr><tr><td>00I</td><td>Class I</td><td>21 CFR 862.2570 - Instrumentation for clinical multiplex test systems</td><td>CH - Clinical Chemistry</td></tr></table>

# II Submission/Device Overview:

# A Purpose for Submission:

To obtain a substantial equivalence determination for the Xpert Xpress CoV-2 plus test performed on the Cepheid GeneXpert Instrument Systems (GeneXpert Dx, GeneXpert Infinity48s and GeneXpert Infinity-80 systems).

# B Measurand:

Nucleocapsid (N2), envelope (E), and RNA-dependent RNA polymerase (RdRP) genes of SARS-CoV-2.

# C Type of Test:

The Xpert Xpress CoV-2 plus test is a real-time reverse transcription polymerase chain reaction (RT-PCR) based in vitro diagnostic test for the qualitative detection of viral RNA from SARSCoV-2 in nasopharyngeal swab (NPS) and anterior nasal swab (NS) specimens.

# III Intended Use/Indications for Use:

A Intended Use(s): See Indications for Use below.

# B Indication(s) for Use:

The Xpert Xpress CoV-2 plus test, performed on the GeneXpert Dx and GeneXpert Infinity Systems, is a rapid real-time RT-PCR test intended for the qualitative detection of SARS-CoV-2 RNA in nasopharyngeal and anterior nasal swab specimens collected from individuals with signs and symptoms of respiratory tract infection.

The Xpert Xpress CoV-2 plus test is intended for use as an aid in the diagnosis of COVID-19 if used in conjunction with other clinical, epidemiologic, and laboratory findings. Positive results are indicative of the presence of SARS-CoV-2 RNA. Positive results do not rule out bacterial infection or co-infection with other pathogens.

Negative results do not preclude SARS-CoV-2 infection. The results of this test should not be used as the sole basis for diagnosis and patient management decisions. .

C Special Conditions for Use Statement(s): Rx - For Prescription Use Only

For in vitro diagnostic use Only

D Special Instrument Requirements:

The Xpert Xpress CoV-2 plus test is performed on the Cepheid GeneXpert Instrument Systems (GeneXpert Dx, GeneXpert Infinity-48s and GeneXpert Infinity-80s systems).

# IV Device/System Characteristics:

# A Device Description:

The Xpert Xpress CoV-2 plus test is a rapid, automated in vitro diagnostic test for the qualitative detection of viral RNA from SARS-CoV-2 in nasopharyngeal swab (NPS) and anterior nasal swab (NS) specimens obtained from individuals with signs and symptoms of respiratory tract infection.

The Xpert Xpress CoV-2 plus test is performed on the Cepheid GeneXpert Instrument Systems (GeneXpert Dx, GeneXpert Infinity-48s and GeneXpert Infinity-80 systems), which consist of an instrument, computer and preloaded software for running tests and viewing the results. The GeneXpert Instrument Systems automate and integrate sample preparation, nucleic acid extraction and amplification, and detection of the target sequences in simple or complex samples using real-time reverse transcription (RT)-polymerase chain reaction (PCR) and PCR assays. Depending on the instrument, the GeneXpert Instrument Systems can have from 1 and up to 80 randomly accessible modules, each capable of performing separate sample preparation and realtime RT-PCR and PCR tests. Each module contains a syringe drive for dispensing fluids (i.e., the syringe drive activates the plunger that works in concert with the rotary valve in the cartridge to move fluids between chambers), an ultrasonic horn for lysing cells or spores, and a proprietary ICORE thermocycler for performing real-time RT-PCR and PCR as well as detection. The systems require the use of single-use disposable cartridges that hold the RT-PCR reagents and host sample purification, nucleic acid amplification, and detection of the target sequences. Because the cartridges are self-contained, cross-contamination between cartridges during the testing process is minimized.

The Xpert Xpress CoV-2 plus test includes reagents for the detection of viral RNA from SARSCoV-2 in NPS and NS specimens. The primers and probes in the Xpert Xpress CoV-2 plus test are designed to amplify and detect sequences in the genes that encode the following SARS-CoV2 proteins: nucleocapsid (N2), envelope (E), and RNA-dependent RNA polymerase (RdRP). A Sample Processing Control (SPC) and a Probe Check Control (PCC) are also included in the cartridge utilized by the GeneXpert instrument. The SPC is present to control for adequate processing of the sample and to monitor for the presence of potential inhibitor(s) in the RT-PCR reaction. The SPC also ensures that the RT-PCR reaction conditions (temperature and time) are appropriate for the amplification reaction and that the RT-PCR reagents are functional. The PCC verifies reagent rehydration, PCR tube filling, and confirms that all reaction components are present in the cartridge including monitoring for probe integrity and dye stability.

The Xpert Xpress CoV-2 plus test is designed for use with NPS or NS specimen collected with nylon flocked swabs and placed into a viral transport medium (VTM), Universal Transport Medium (UTM) or eNAT.

# B Principle of Operation:

The Xpert Xpress CoV-2 plus test utilizes automated reverse transcription cDNA synthesis from RNA and real-time polymerase chain reaction (PCR) for gene specific sequence amplification. Detection of the amplified DNA is by fluorogenic target-specific probe hybridization followed by $5 '$ -nuclease cleavage of the probe to release the fluorophore (TaqMan).

The technology utilizes lyophilized reagents in the form of freeze-dried beads, which provide all the components (enzymes, primers, probes, salts, buffers and divalent metal ion) to support PCR when reconstituted.

# C Instrument Description Information:

Cepheid GeneXpert Instrument Systems [GeneXpert Dx (versions 4.7b or higher), GeneXpert Infinity-48s and GeneXpert Infinity-80s systems (versions 6.4b or higher)].

2. Specimen Identification: Anterior nasal (NS) or nasopharyngeal (NPS) swab specimen identification is automated.

3. Specimen Sampling and Handling:

The NPS or NS specimen is collected and placed into a transport tube containing $3 ~ \mathrm { m L }$ of Viral Transport Medium (VTM)/ Universal Transport Medium (UTM) or $2 \mathrm { m L }$ of eNAT. The specimen is briefly mixed by rapidly inverting the collection tube 5 times. Using the supplied transfer pipette, the specimen is transferred to the sample chamber of the Xpert Xpress CoV-2 plus cartridge. The GeneXpert cartridge is loaded onto the GeneXpert instrument, which performs hands-off, automated specimen processing, and real-time RTPCR for detection of viral RNA.

# 4. Calibration:

GeneXpert instruments are calibrated at the factory. Routine calibration of the GeneXpert Instrument systems may be performed by Cepheid Field Service Engineers during annual maintenance.

5. Quality Control:

Internal Controls: Each cartridge includes a Sample Processing Control (SPC) and Probe Check Control (PCC).

Sample Processing Control (SPC) – The SPC is an encapsidated RNA pseudovirus that ensures adequate lysis of target virus, monitors the presence of PCR inhibitors, and verifies the use of proper PCR conditions. The SPC passes if it meets the validated acceptance criteria. The SPC result should be PASS in a sample that is negative for all three SARS-CoV-2 target analytes and can be NEGATIVE or POSITIVE in a sample containing detectable levels of one or more of the SARSCoV-2 target analytes.

Probe Check Control (PCC) – Before the start of the PCR reaction, the GeneXpert System measures the fluorescence signal from the probes to monitor bead rehydration, reaction tube filling, probe integrity, and dye stability. The PCC passes if it meets the validated acceptance criteria.

External Controls:

External controls should be used in accordance with local, state, and federal accrediting organizations as applicable.

# V Substantial Equivalence Information:

A Predicate Device Name(s): BioFire COVID-19 Test 2   
B Predicate 510(k) Number(s): K211079

C Comparison with Predicate(s):

<table><tr><td colspan="1" rowspan="1">Device &amp; PredicateDevice(s):</td><td colspan="1" rowspan="1">K230440</td><td colspan="1" rowspan="1">K211079</td></tr><tr><td colspan="1" rowspan="1">Device Trade Name</td><td colspan="1" rowspan="1">Xpert Xpress CoV-2 plus</td><td colspan="1" rowspan="1">BioFire COVID-19 Test 2</td></tr><tr><td colspan="1" rowspan="1">Product Code</td><td colspan="1" rowspan="1">Same</td><td colspan="1" rowspan="1">QQX</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Respiratory Specimen Nucleic AcidSARS-CoV-2 Test</td></tr><tr><td colspan="1" rowspan="1">RegulationNumber/Name</td><td colspan="1" rowspan="1">Same</td><td colspan="1" rowspan="1">21 CFR 866.3981Devices to detect and identify nucleicacid targets in respiratory samplesfrom microbial agents that cause theSARS-CoV-2 respiratory infectionand other microbial agents when in amulti-analyte test</td></tr><tr><td colspan="1" rowspan="1">Device Class</td><td colspan="1" rowspan="1">Same</td><td colspan="1" rowspan="1">II (Special Controls)</td></tr><tr><td colspan="1" rowspan="1">General DeviceCharacteristicSimilarities</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">IntendedUse/Indications ForUse</td><td colspan="1" rowspan="1">The Xpert Xpress CoV-2 plustest, performed on theGeneXpert Dx and GeneXpertInfinity Systems, is a rapidreal-time RT-PCR testintended for the qualitativedetection of SARS-CoV-2RNA in nasopharyngeal andanterior nasal swab specimenscollected from individualswith signs and symptoms ofrespiratory tract infection.The Xpert Xpress CoV-2 plustest is intended for use as anaid in the diagnosis ofCOVID-19 if used inconjunction with other clinical,epidemiologic, and laboratoryfindings. Positive results areindicative of the presence ofSARS-CoV-2 RNA. Positiveresults do not rule out bacterialinfection or co-infection withother pathogens.Negative results do notpreclude SARS-CoV-2infection. The results of thistest should not be used as thesole basis for diagnosis andpatient management decisions.</td><td colspan="1" rowspan="1">The BioFire COVID-19 Test 2 is aqualitative nested multiplexed RT-PCR in vitro diagnostic test intendedfor use with the BioFire FilmArray2.0 and BioFire FilmArray TorchSystems. The BioFire COVID-19Test 2 detects nucleic acids fromsevere acute respiratory syndromecoronavirus 2 (SARS-CoV-2) innasopharyngeal swabs (NPS) fromsymptomatic individuals suspected ofCOVID-19 by their healthcareprovider.Results are for the identification ofSARSCoV-2 RNA. The SARS-CoV-2 RNA is generally detectable inNPS specimens during the acutephase of infection. Positive resultsare indicative of the presence ofSARS-CoV-2 RNA; clinicalcorrelation with patient history andother diagnostic information isnecessary to determine patientinfection status. Positive results donot rule out co-infection with otherpathogens.Results are meant to be used inconjunction with other clinical,epidemiologic, and laboratory data,in accordance with theguidelines provided by the relevantpublic</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">health authorities. The BioFireCOVID-19 Test 2 is intended for useby trained medical and laboratoryprofessionals in a laboratory settingor under the supervision of a trainedlaboratory professional.</td></tr><tr><td colspan="1" rowspan="1">Analyte Target</td><td colspan="1" rowspan="1">Same</td><td colspan="1" rowspan="1">SARS-CoV-2</td></tr><tr><td colspan="1" rowspan="1">Control</td><td colspan="1" rowspan="1">Same</td><td colspan="1" rowspan="1">Internal Controls</td></tr><tr><td colspan="1" rowspan="1">Test Format</td><td colspan="1" rowspan="1">Same</td><td colspan="1" rowspan="1">Single Use</td></tr><tr><td colspan="1" rowspan="1">Result Interpretation</td><td colspan="1" rowspan="1">Same</td><td colspan="1" rowspan="1">Automated</td></tr><tr><td colspan="1" rowspan="1">Test Result</td><td colspan="1" rowspan="1">Same</td><td colspan="1" rowspan="1">Qualitative</td></tr><tr><td colspan="1" rowspan="1">Automation</td><td colspan="1" rowspan="1">Same</td><td colspan="1" rowspan="1">Automated Nucleic Acid Extraction,Detection and Results Interpretation</td></tr><tr><td colspan="1" rowspan="1">Intended Environmentfor Use</td><td colspan="1" rowspan="1">Same</td><td colspan="1" rowspan="1">Professional use</td></tr><tr><td colspan="1" rowspan="1">Prescription Use Only</td><td colspan="1" rowspan="1">Same</td><td colspan="1" rowspan="1">Yes</td></tr><tr><td colspan="1" rowspan="1">General DeviceCharacteristicDifferences</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Technology</td><td colspan="1" rowspan="1">Real-time reverse transcriptionpolymerase chain reaction(RT-qPCR).</td><td colspan="1" rowspan="1">Nested multiplex RT-PCR followedby high resolution melting analysis toconfirm identity of amplifiedproduct.</td></tr><tr><td colspan="1" rowspan="1">SARS-CoV-2 TargetType</td><td colspan="1" rowspan="1">E, N2, RdRP</td><td colspan="1" rowspan="1">Multiple Targets</td></tr><tr><td colspan="1" rowspan="1">Specimen Type</td><td colspan="1" rowspan="1">•Nasopharyngeal swab (NPS)Anterior nasal swab (NS)</td><td colspan="1" rowspan="1">Nasopharyngeal swab (NPS)</td></tr><tr><td colspan="1" rowspan="1">Transport Media</td><td colspan="1" rowspan="1">Universal Transport Medium(UTM) / Viral TransportMedium (VTM)eNAT</td><td colspan="1" rowspan="1">Transport MediumSaline</td></tr><tr><td colspan="1" rowspan="1">Instrumentation</td><td colspan="1" rowspan="1">Cepheid GeneXpertInstrument Systems</td><td colspan="1" rowspan="1">FilmArray 2.0 or FilmArray Torchsystems</td></tr><tr><td colspan="1" rowspan="1">Internal Control</td><td colspan="1" rowspan="1">Sample Processing Control(SPC)Probe Check Control (PCC)</td><td colspan="1" rowspan="1">Sample Processing Control•PCR and Melt Analysis Control</td></tr><tr><td colspan="1" rowspan="1">Time to Result</td><td colspan="1" rowspan="1">∼ 30 minutes</td><td colspan="1" rowspan="1">~ 45 minutes</td></tr></table>

# VI Standards/Guidance Documents Referenced:

Class II Special Controls as per 21 CFR 866.3981   
Guidance for Industry and FDA Staff: General Principles of Software Validation, issued January 11, 2002. Guidance for the Content of Premarket Submissions for Software Contained in Medical Devices, May 11, 2005.   
• Guidance for Industry and FDA Staff: Statistical Guidance on Reporting Results from Studies Evaluating Diagnostic Tests, issued on March 13, 2007. Clinical and Laboratory Standards Institute (CLSI) documents: EP05-A3 (Evaluation of Precision of Quantitative Measurement Procedures, 3rd Edition). Clinical and Laboratory Standards Institute (CLSI) documents: EP07-A3 (Interference Testing in Clinical Chemistry, 3rd Edition).   
• Clinical and Laboratory Standards Institute (CLSI) documents: EP12-A2 (User Protocol For Evaluation Of Qualitative Test Performance; Approved Guideline, $2 ^ { \mathrm { n d } }$ Edition).   
• Clinical and Laboratory Standards Institute (CLSI) documents: EP17-A2 (Evaluation of Detection Capability for Clinical Laboratory Measurement Procedures, 2nd Edition).

# VII Performance Characteristics (if/when applicable):

# A Analytical Performance:

1. Reproducibility and Within-Laboratory Precision Study:

# a) Reproducibility Study

The reproducibility of the Xpert Xpress CoV-2 plus test was established at three sites (two external and one internal) using a three-member panel, including one negative sample, one low positive ${ \mathrm { \sim } } 1 . 5 \mathrm { x }$ LoD) sample and one moderate positive $\left( \sim 3 \mathrm { x } \ L \mathrm { o D } \right)$ sample. The negative sample consisted of simulated NS/NPS matrix without target microorganism or target RNA. The positive samples were contrived samples in a simulated NS/NPS matrix prepared by spiking the NATtrol inactivated SARS-CoV- 2 virus strain USA-WA1/2020 (ZeptoMetrix). Testing was conducted over six days with the GeneXpert Dx and Infinity systems, using three lots of Xpert Xpress CoV-2 plus cartridges at three participating sites each with two operators to yield a total of 144 observations per panel member (3 Sites x 2 Operators x 3 Lots x 2 Days/Lot x 2 Runs x 2 Replicates $= 1 4 4$ observations/panel member). The external controls, consisting of one positive and one negative control, were tested prior to testing panel samples at the site on each day of the study.

The percent agreement of the qualitative results for SARS-CoV-2 detection for each panel member analyzed by each of the six operators and each site is shown in Table 1. In addition, the overall percent agreement for each sample $\%$ total agreement) and the two-sided Wilson Score confidence intervals (CI) are presented in the last column.

Table 1: Summary of the Reproducibility Study Results - $\%$ Agreement   

<table><tr><td colspan="1" rowspan="2">PanelMember</td><td colspan="3" rowspan="1">Site 1</td><td colspan="3" rowspan="1">Site 2</td><td colspan="3" rowspan="1">Site 3</td><td colspan="1" rowspan="2">% TotalAgreementand 95% CI</td></tr><tr><td colspan="1" rowspan="1">Op1</td><td colspan="1" rowspan="1">Op2</td><td colspan="1" rowspan="1">Site</td><td colspan="1" rowspan="1">Op1</td><td colspan="1" rowspan="1">Op2</td><td colspan="1" rowspan="1">Site</td><td colspan="1" rowspan="1">Op1</td><td colspan="1" rowspan="1">Op2</td><td colspan="1" rowspan="1">Site</td></tr><tr><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">100%(24/24)</td><td colspan="1" rowspan="1">95.8%(23/24)</td><td colspan="1" rowspan="1">97.9%(47/48)</td><td colspan="1" rowspan="1">100%(24/24)</td><td colspan="1" rowspan="1">100%(24/24)</td><td colspan="1" rowspan="1">100%(48/48)</td><td colspan="1" rowspan="1">100%(24/24)</td><td colspan="1" rowspan="1">100%(23/23)a</td><td colspan="1" rowspan="1">100%(47/47)</td><td colspan="1" rowspan="1">99.3%(142/143)[96.1%-99.9%]</td></tr><tr><td colspan="1" rowspan="1">SARS-CoV-2Low Pos</td><td colspan="1" rowspan="1">100%(24/24)</td><td colspan="1" rowspan="1">100%(24/24)</td><td colspan="1" rowspan="1">100%(48/48)</td><td colspan="1" rowspan="1">100%(24/24)</td><td colspan="1" rowspan="1">100%(24/24)</td><td colspan="1" rowspan="1">100%(48/48)</td><td colspan="1" rowspan="1">100%(24/24)</td><td colspan="1" rowspan="1">100%(24/24)</td><td colspan="1" rowspan="1">100%(48/48)</td><td colspan="1" rowspan="1">100%(144/144)[97.4% - 100%]</td></tr><tr><td colspan="1" rowspan="1">SARS-CoV-2Mod Pos</td><td colspan="1" rowspan="1">100%(24/24)</td><td colspan="1" rowspan="1">100%(24/24)</td><td colspan="1" rowspan="1">100%(48/48)</td><td colspan="1" rowspan="1">100%(24/24)</td><td colspan="1" rowspan="1">100%(24/24)</td><td colspan="1" rowspan="1">100%(48/48)</td><td colspan="1" rowspan="1">100%(24/24)</td><td colspan="1" rowspan="1">100%(24/24)</td><td colspan="1" rowspan="1">100%(48/48)</td><td colspan="1" rowspan="1">100%(144/144)[97.4% - 100%]</td></tr></table>

Abbreviation: Op, operator; Pos, positive; Mod, moderate; CI, confidence interval a. One sample was non-determinate on both initial and retest and was excluded from the analyses.

The evaluation of reproducibility and within-laboratory precision of the underlying Ct values for E, N2 and RdRP obtained in the Xpert Xpress CoV-2 plus test was performed. The mean, standard deviation (SD), and coefficient of variation (CV) between-sites, between-operators, between-lots, between-days, between-runs and within-run for each panel member are presented in Table 2.

Table 2: Summary of Reproducibility Results – Nested ANOVA by Coefficient of Variation   

<table><tr><td rowspan=2 colspan=1>Sample</td><td rowspan=2 colspan=1>Analyte</td><td rowspan=2 colspan=1>N</td><td rowspan=2 colspan=1>MeanCt</td><td rowspan=1 colspan=2>BetweenSites</td><td rowspan=1 colspan=2>BetweenOperators</td><td rowspan=1 colspan=2>BetweenLots</td><td rowspan=1 colspan=2>BetweenDay</td><td rowspan=1 colspan=2>BetweenRuns</td><td rowspan=1 colspan=2>Within Run</td><td rowspan=1 colspan=2>Total</td></tr><tr><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>CV(%)</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>CV(%)</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>CV(%)</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>CV(%)</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>CV(%)</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>CV(%)</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>CV(%)</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>SPC</td><td rowspan=1 colspan=1>143</td><td rowspan=1 colspan=1>31.1</td><td rowspan=1 colspan=1>0.29</td><td rowspan=1 colspan=1>0.9</td><td rowspan=1 colspan=1>0.16</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>0.42</td><td rowspan=1 colspan=1>1.3</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1.47</td><td rowspan=1 colspan=1>4.7</td><td rowspan=1 colspan=1>1.56</td><td rowspan=1 colspan=1>5.0</td></tr><tr><td rowspan=1 colspan=1>SARS-</td><td rowspan=1 colspan=1>E</td><td rowspan=1 colspan=1>144</td><td rowspan=1 colspan=1>34.0</td><td rowspan=1 colspan=1>0.06</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>0.19</td><td rowspan=1 colspan=1>0.6</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.14</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.55</td><td rowspan=1 colspan=1>1.6</td><td rowspan=1 colspan=1>0.60</td><td rowspan=1 colspan=1>1.8</td></tr><tr><td rowspan=2 colspan=1>CoV-2Low Pos</td><td rowspan=1 colspan=1>N2</td><td rowspan=1 colspan=1>144</td><td rowspan=1 colspan=1>37.5</td><td rowspan=1 colspan=1>0.12</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>0.11</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.68</td><td rowspan=1 colspan=1>1.8</td><td rowspan=1 colspan=1>0.73</td><td rowspan=1 colspan=1>1.9</td></tr><tr><td rowspan=1 colspan=1>RdRP</td><td rowspan=1 colspan=1>144</td><td rowspan=1 colspan=1>36.3</td><td rowspan=1 colspan=1>0.07</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.25</td><td rowspan=1 colspan=1>0.7</td><td rowspan=1 colspan=1>0.06</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.61</td><td rowspan=1 colspan=1>1.7</td><td rowspan=1 colspan=1>0.66</td><td rowspan=1 colspan=1>1.8</td></tr><tr><td rowspan=2 colspan=1>SARS-CoV-2</td><td rowspan=1 colspan=1>E</td><td rowspan=1 colspan=1>144</td><td rowspan=1 colspan=1>32.5</td><td rowspan=1 colspan=1>0.31</td><td rowspan=1 colspan=1>0.9</td><td rowspan=1 colspan=1>0.03</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.47</td><td rowspan=1 colspan=1>1.4</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1.86</td><td rowspan=1 colspan=1>5.7</td><td rowspan=1 colspan=1>1.95</td><td rowspan=1 colspan=1>6.0</td></tr><tr><td rowspan=1 colspan=1>N2</td><td rowspan=1 colspan=1>144</td><td rowspan=1 colspan=1>36.2</td><td rowspan=1 colspan=1>0.16</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.29</td><td rowspan=1 colspan=1>0.8</td><td rowspan=1 colspan=1>0.17</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.65</td><td rowspan=1 colspan=1>1.8</td><td rowspan=1 colspan=1>0.75</td><td rowspan=1 colspan=1>2.1</td></tr><tr><td rowspan=1 colspan=1>Mod Pos</td><td rowspan=1 colspan=1>RdRP</td><td rowspan=1 colspan=1>144</td><td rowspan=1 colspan=1>34.8</td><td rowspan=1 colspan=1>0.33</td><td rowspan=1 colspan=1>1.0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.62</td><td rowspan=1 colspan=1>1.8</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2.18</td><td rowspan=1 colspan=1>6.2</td><td rowspan=1 colspan=1>2.29</td><td rowspan=1 colspan=1>6.6</td></tr></table>

# b) Within-Laboratory Precision Study

The within-laboratory precision of the Xpert Xpress CoV-2 plus test was evaluated at a single site using a three-member panel, including one negative sample, one low positive $( \mathord { \sim } 1 . 5 \mathrm { x }$ LoD) sample and one moderate positive $( { \sim } 3 \mathrm { x } \ L \mathrm { o D } )$ sample. The negative sample consisted of simulated NS/NPS swab matrix without target microorganism or target RNA. The positive samples were contrived samples in a simulated NS/NPS matrix prepared by spiking the NATtrol inactivated SARS-CoV- 2 virus strain USA-WA1/2020 (ZeptoMetrix).

Testing was conducted over 20 days, using one lot of Xpert Xpress CoV-2 plus cartridges at a single site and with one operator to yield a total of eighty observations per panel member (1 Site $_ { \textrm { x 1 } }$ Operator x 1 Lot x 20 Days x 2 Runs x 2 Replicates $= 8 0$ observations/panel member). The agreement of the qualitative results for SARS-CoV-2 detection for each panel member, including the overall percent agreement for each sample $\%$ total agreement) and the two-sided Wilson Score confidence intervals (CI), are presented in Table 3. The mean, standard deviation (SD), and coefficient of variation (CV) between-days, between-runs, and within-run for each panel member are presented in Table 4.

Table 3: Summary of the With-in Laboratory Precision Results $- \%$ Agreement   

<table><tr><td rowspan=1 colspan=1>Panel Member</td><td rowspan=1 colspan=1>Agreement</td><td rowspan=1 colspan=1>% Total Agreement and 95%CI by Panel Member</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>80/80</td><td rowspan=1 colspan=1>100%(95.4%-100.0%)</td></tr><tr><td rowspan=1 colspan=1>SARS-CoV-2 Low Positive</td><td rowspan=1 colspan=1>80/80</td><td rowspan=1 colspan=1>100%(95.4%-100.0%)</td></tr><tr><td rowspan=1 colspan=1>SARS-CoV-2 Mod Positive</td><td rowspan=1 colspan=1>80/80</td><td rowspan=1 colspan=1>100%(95.4%-100.0%)</td></tr></table>

Table 4: Within-Laboratory Precision – Nested ANOVA by Coefficient of Variation   

<table><tr><td rowspan=2 colspan=1>Sample</td><td rowspan=2 colspan=1>Analyteof CtValues</td><td rowspan=2 colspan=1>N</td><td rowspan=2 colspan=1>MeanCt</td><td rowspan=1 colspan=2>BetweenDays</td><td rowspan=1 colspan=2>BetweenRuns</td><td rowspan=1 colspan=2>WithinRun</td><td rowspan=1 colspan=2>Total</td></tr><tr><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>CV</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>SPC</td><td rowspan=1 colspan=1>80</td><td rowspan=1 colspan=1>30.34</td><td rowspan=1 colspan=1>0.26</td><td rowspan=1 colspan=1>0.8</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>1.03</td><td rowspan=1 colspan=1>3.4</td><td rowspan=1 colspan=1>1.06</td><td rowspan=1 colspan=1>3.5</td></tr><tr><td rowspan=3 colspan=1>SARS-CoV-2~1.5 xLoD</td><td rowspan=1 colspan=1>E</td><td rowspan=1 colspan=1>80</td><td rowspan=1 colspan=1>34.12</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.21</td><td rowspan=1 colspan=1>0.6</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>1.5</td><td rowspan=1 colspan=1>0.55</td><td rowspan=1 colspan=1>1.6</td></tr><tr><td rowspan=1 colspan=1>N2</td><td rowspan=1 colspan=1>80</td><td rowspan=1 colspan=1>37.66</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.11</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>0.57</td><td rowspan=1 colspan=1>1.5</td><td rowspan=1 colspan=1>0.58</td><td rowspan=1 colspan=1>1.5</td></tr><tr><td rowspan=1 colspan=1>RdRP</td><td rowspan=1 colspan=1>80</td><td rowspan=1 colspan=1>36.77</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.66</td><td rowspan=1 colspan=1>1.8</td><td rowspan=1 colspan=1>0.66</td><td rowspan=1 colspan=1>1.8</td></tr><tr><td rowspan=3 colspan=1>SARS-CoV-2~3 xLoD</td><td rowspan=1 colspan=1>E</td><td rowspan=1 colspan=1>80</td><td rowspan=1 colspan=1>33.37</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>1.03</td><td rowspan=1 colspan=1>3.1</td><td rowspan=1 colspan=1>1.03</td><td rowspan=1 colspan=1>3.1</td></tr><tr><td rowspan=1 colspan=1>N2</td><td rowspan=1 colspan=1>80</td><td rowspan=1 colspan=1>36.89</td><td rowspan=1 colspan=1>0.17</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>1.02</td><td rowspan=1 colspan=1>2.8</td><td rowspan=1 colspan=1>1.03</td><td rowspan=1 colspan=1>2.8</td></tr><tr><td rowspan=1 colspan=1>RdRP</td><td rowspan=1 colspan=1>80</td><td rowspan=1 colspan=1>35.99</td><td rowspan=1 colspan=1>0.27</td><td rowspan=1 colspan=1>0.7</td><td rowspan=1 colspan=1>0.37</td><td rowspan=1 colspan=1>1.0</td><td rowspan=1 colspan=1>0.95</td><td rowspan=1 colspan=1>2.6</td><td rowspan=1 colspan=1>1.05</td><td rowspan=1 colspan=1>2.9</td></tr></table>

$^ { * } \mathrm { C V } = \mathrm { S D } / \mathrm { M e a n } \mathrm { C t } ^ { \ast } 1 0 0$

2. Linearity: Not applicable; this is a qualitative assay.

3. Analytical Reactivity:

a) In silico Inclusivity Analysis

i) The inclusivity of Xpert Xpress CoV-2 plus primers was evaluated on June $3 0 ^ { \mathrm { t h } }$ , 2022 using in silico analysis of the assay amplicons in relation to 11,650,640 SARS-CoV-2 sequences available in the GISAID gene database for three targets, E, N2 and RdRP. The 11,650,640 SARS-CoV-2 sequences were separated into the lineages of interest based on the Pango Lineage assigned to each genome by GISAID, and those with ambiguous nucleotides were removed. Thus, the following inclusivity analyses focuses on the combined, non-ambiguous sequences from the variants of interest and variants of concern as of June $3 0 ^ { \mathrm { t h } }$ , 2022. These constituted 10,469,612 sequences for the E target, 10,587,381 sequences for the N2 target and 10,333,656 sequences for the RdRP target. Table 5 summarizes the effective predicted inclusivity for E, N2 and RdRP amplicons for the variants of interests and concern.

Table 5: Predicted Inclusivity for E, N2 and RdRP Amplicons for SARS-CoV-2 Variants of Interests and Concern   

<table><tr><td rowspan=1 colspan=1>SARS-CoV-2Target Amplicon</td><td rowspan=1 colspan=1>Exact Match</td><td rowspan=1 colspan=1>1 Mismatch a</td><td rowspan=1 colspan=1>2 or MoreMismatches</td><td rowspan=1 colspan=1>PredictedInclusivity</td></tr><tr><td rowspan=1 colspan=1>E</td><td rowspan=1 colspan=1>10,420,248 of 10,469,612 total(99.5%)</td><td rowspan=1 colspan=1>48,562(0.5%)</td><td rowspan=1 colspan=1>802(0.01%)</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>N2</td><td rowspan=1 colspan=1>10,386,068 of 10,587,381 total(98.1%)</td><td rowspan=1 colspan=1>196,336(1.9%)</td><td rowspan=1 colspan=1>4,977(0.05%)</td><td rowspan=1 colspan=1>99.95%</td></tr><tr><td rowspan=1 colspan=1>RdRP</td><td rowspan=1 colspan=1>10,247,146 of 10,333,656 total(99.2%)</td><td rowspan=1 colspan=1>85,373(0.8%)</td><td rowspan=1 colspan=1>1,137(0.01%)</td><td rowspan=1 colspan=1>100%</td></tr></table>

a. Single-nucleotide mismatches are predicted to not impact the performance of the test.

The in silico inclusivity of the Xpert Xpress CoV-2 plus probe oligonucleotides for E, N2 and RdRP were also assessed against the top 20 most frequent matches in the GISAID EpiCoV sequence database as of June 15th, 2022, which constituted 10,310, 839 for the E target, 10,428,014 for the N2 target and 10,178,602 for the RdRP target. For each of the probe oligonucleotides used in the Xpert Xpress CoV-2 plus test, Table 6 summarizes the number sequences as well as the corresponding percentage of sequences from this dataset with exact match, 1 mismatch/insertion, and 2 or more mismatches/insertions in the alignment.

Table 6: Predicted Inclusivity for E, N2 and RdRP Probes for SARS-CoV-2 Variants of Interests and Concern   

<table><tr><td rowspan=1 colspan=1>SARS-CoV-2Target Probe</td><td rowspan=1 colspan=1>Exact Match</td><td rowspan=1 colspan=1>1 Mismatch/Insertion a</td><td rowspan=1 colspan=1>2 or MoreMismatches/Insertions</td><td rowspan=1 colspan=1>PredictedInclusivity</td></tr><tr><td rowspan=1 colspan=1>E</td><td rowspan=1 colspan=1>10,300,688 of 10,310,839 total(99.9%)</td><td rowspan=1 colspan=1>9,853(0.1%)</td><td rowspan=1 colspan=1>22(0.0002%)</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>N2</td><td rowspan=1 colspan=1>10,351,581 of 10,428,014 total(99.3%)</td><td rowspan=1 colspan=1>72,957(0.7%)</td><td rowspan=1 colspan=1>0(0%)</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>RdRP</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>10,140,254 of 10,178602 total(99.6%)</td><td rowspan=1 colspan=1>37,492(0.4%)</td><td rowspan=1 colspan=1>99.6%</td></tr></table>

a. Single-nucleotide mismatches/insertions are predicted to not impact the performance of the test.

ii) Additional evaluation of inclusivity of Xpert Xpress CoV-2 plus primers was conducted on July 31, 2023 using in silico analysis of the assay amplicons in relation to SARS-CoV-2 sequences available in the GISAID gene database for three targets, E, N2 and RdRP. U.S.- specific sequences were then partitioned into the four time periods of 14, 30, 90 and 180 days (Tables 7-10) prior to the cutoff date of July 31, 2023 and were stratified by number and frequency of sequences with exact or mismatched sequences. The in silico inclusivity statistics for each time period are presented in Tables 7-10. In each of these tables, the results for the two N2 probes were combined into a single row which represents the best match of either probe.

Table 7: Inclusivity Statistic - Sequences Collected Within 14 Days Prior to July 31, 2023   

<table><tr><td rowspan=1 colspan=2>SARS-CoV-2Target</td><td rowspan=1 colspan=5>In Silico Inclusivity Statistics</td></tr><tr><td rowspan=1 colspan=2></td><td rowspan=1 colspan=1>Exact Match</td><td rowspan=1 colspan=1>1 Mismatch</td><td rowspan=1 colspan=1>2 Mismatches</td><td rowspan=1 colspan=1>3 Mismatches</td><td rowspan=1 colspan=1>3+ Mismatches</td></tr><tr><td rowspan=3 colspan=1>E</td><td rowspan=1 colspan=1>Forward</td><td rowspan=1 colspan=1>97.7% (173/177)</td><td rowspan=1 colspan=1>2.3% (4/177)</td><td rowspan=1 colspan=1>0.0% (0/177)</td><td rowspan=1 colspan=1>0.0% (0/177)</td><td rowspan=1 colspan=1>0.0% (0/177)</td></tr><tr><td rowspan=1 colspan=1>Probe</td><td rowspan=1 colspan=1>100.0% (177/177)</td><td rowspan=1 colspan=1>0.0% (0/177)</td><td rowspan=1 colspan=1>0.0% (0/177)</td><td rowspan=1 colspan=1>0.0% (0/177)</td><td rowspan=1 colspan=1>0.0% (0/177)</td></tr><tr><td rowspan=1 colspan=1>Reverse</td><td rowspan=1 colspan=1>100.0% (177/177)</td><td rowspan=1 colspan=1>0.0% (0/177)</td><td rowspan=1 colspan=1>0.0% (0/177)</td><td rowspan=1 colspan=1>0.0% (0/177)</td><td rowspan=1 colspan=1>0.0% (0/177)</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Exact Match</td><td rowspan=1 colspan=1>1 Mismatch</td><td rowspan=1 colspan=1>2 Mismatches</td><td rowspan=1 colspan=1>3 Mismatches</td><td rowspan=1 colspan=1>3+ Mismatches</td></tr><tr><td rowspan=3 colspan=1>N2</td><td rowspan=1 colspan=1>Forward</td><td rowspan=1 colspan=1>97.7% (172/176)</td><td rowspan=1 colspan=1>2.3% (4/176)</td><td rowspan=1 colspan=1>0.0% (0/176)</td><td rowspan=1 colspan=1>0.0% (0/176)</td><td rowspan=1 colspan=1>0.0% (0/176)</td></tr><tr><td rowspan=1 colspan=1>Probe</td><td rowspan=1 colspan=1>100% (176/176)</td><td rowspan=1 colspan=1>0.0% (0/176)</td><td rowspan=1 colspan=1>0.0% (0/176)</td><td rowspan=1 colspan=1>0.0% (0/176)</td><td rowspan=1 colspan=1>0.0% (0/176)</td></tr><tr><td rowspan=1 colspan=1>Reverse</td><td rowspan=1 colspan=1>99.4% (175/176)</td><td rowspan=1 colspan=1>0.6% (1/176)</td><td rowspan=1 colspan=1>0.0% (0/176)</td><td rowspan=1 colspan=1>0.0% (0/176)</td><td rowspan=1 colspan=1>0.0% (0/176)</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Exact Match</td><td rowspan=1 colspan=1>1 Mismatch</td><td rowspan=1 colspan=1>2 Mismatches</td><td rowspan=1 colspan=1>3 Mismatches</td><td rowspan=1 colspan=1>3+ Mismatches</td></tr><tr><td rowspan=4 colspan=1>RdRP</td><td rowspan=1 colspan=1>Forward</td><td rowspan=1 colspan=1>99.4% (172/173)</td><td rowspan=1 colspan=1>0.6% (1/173)</td><td rowspan=1 colspan=1>0.0% (0/173)</td><td rowspan=1 colspan=1>0.0% (0/173)</td><td rowspan=1 colspan=1>0.0% (0/173)</td></tr><tr><td rowspan=1 colspan=1>Reverse</td><td rowspan=1 colspan=1>100.0% (173/173)</td><td rowspan=1 colspan=1>0.0% (0/173)</td><td rowspan=1 colspan=1>0.0% (0/173)</td><td rowspan=1 colspan=1>0.0% (0/173)</td><td rowspan=1 colspan=1>0.0% (0/173)</td></tr><tr><td rowspan=2 colspan=1>Probe</td><td rowspan=1 colspan=1>1 Insertion*</td><td rowspan=1 colspan=1>1 Insertion +1 Mismatch</td><td rowspan=1 colspan=1>1 Insertion +2 Mimatches</td><td rowspan=1 colspan=1>1 Insertion +3 Mimatches</td><td rowspan=1 colspan=1>1 Insertion +&gt; 3 Mismatches</td></tr><tr><td rowspan=1 colspan=1>99.4% (172/173)</td><td rowspan=1 colspan=1>0.6% (1/173)</td><td rowspan=1 colspan=1>0.0% (0/173)</td><td rowspan=1 colspan=1>0.0% (0/173)</td><td rowspan=1 colspan=1>0.0% (0/173)</td></tr></table>

RdRp probe design lacks dC at position 14 from the $_ { 5 } '$ -end of the oligonucleotide.

As shown in Table 7, there are two instances where a specific oligonucleotide was observed to have a mismatch frequency greater than $1 \%$ in the 14-day period. The total number of sequences collected in the 14-day period is small, which can be a source of bias and may not be representative of the test performance. Consistent with this account, there were no instances where a specific primer/probe sequence was observed with greater than $1 \%$ frequency of

mismatch in any of the longer time stratifications (30 days, 90 days and 180 days), which would warrant a risk assessment for adverse impact to the test performance. Based on the built-in redundancy of the Xpert Xpress CoV-2 plus test’s SARS-CoV-2 amplification system (i.e., 3 independent targets, only 1 of 3 must be detected to assign a positive result), it is not anticipated that any of the evaluated sequences would be missed by the Xpert Xpress CoV-2 plus test.

Table 8: Inclusivity Statistic - Sequences Collected Within 30 Days Prior to July 31, 2023   

<table><tr><td rowspan=1 colspan=2>SARS-CoV-2Target</td><td rowspan=1 colspan=5>In Silico Inclusivity Statistics</td></tr><tr><td rowspan=1 colspan=2></td><td rowspan=1 colspan=1>Exact Match</td><td rowspan=1 colspan=1>1 Mismatch</td><td rowspan=1 colspan=1>2 Mismatches</td><td rowspan=1 colspan=1>3 Mismatches</td><td rowspan=1 colspan=1>3+ Mismatches</td></tr><tr><td rowspan=3 colspan=1>E</td><td rowspan=1 colspan=1>Forward</td><td rowspan=1 colspan=1>99.2% (2505/2526)</td><td rowspan=1 colspan=1>0.8% (21/2526)</td><td rowspan=1 colspan=1>0.0% (0/2526)</td><td rowspan=1 colspan=1>0.0% (0/2526)</td><td rowspan=1 colspan=1>0.0% (0/2526)</td></tr><tr><td rowspan=1 colspan=1>Probe</td><td rowspan=1 colspan=1>99.8% (2522/2526)</td><td rowspan=1 colspan=1>0.2% (4/2526)</td><td rowspan=1 colspan=1>0.0% (0/2526)</td><td rowspan=1 colspan=1>0.0% (0/2526)</td><td rowspan=1 colspan=1>0.0% (0/2526)</td></tr><tr><td rowspan=1 colspan=1>Reverse</td><td rowspan=1 colspan=1>100.0%(2525/2526)</td><td rowspan=1 colspan=1>0.0% (1/2526)</td><td rowspan=1 colspan=1>0.0% (0/2526)</td><td rowspan=1 colspan=1>0.0% (0/2526)</td><td rowspan=1 colspan=1>0.0% (0/2526)</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Exact Match</td><td rowspan=1 colspan=1>1 Mismatch</td><td rowspan=1 colspan=1>2 Mismatches</td><td rowspan=1 colspan=1>3 Mismatches</td><td rowspan=1 colspan=1>3+ Mismatches</td></tr><tr><td rowspan=3 colspan=1>N2</td><td rowspan=1 colspan=1>Forward</td><td rowspan=1 colspan=1>99.2% (2509/2528)</td><td rowspan=1 colspan=1>0.8% (19/2528)</td><td rowspan=1 colspan=1>0.0% (0/2528)</td><td rowspan=1 colspan=1>0.0% (0/2528)</td><td rowspan=1 colspan=1>0.0% (0/2528)</td></tr><tr><td rowspan=1 colspan=1>Probe</td><td rowspan=1 colspan=1>99.8% (2522/2528)</td><td rowspan=1 colspan=1>0.2% (6/2528)</td><td rowspan=1 colspan=1>0.0% (0/2528)</td><td rowspan=1 colspan=1>0.0% (0/2528)</td><td rowspan=1 colspan=1>0.0% (0/2528)</td></tr><tr><td rowspan=1 colspan=1>Reverse</td><td rowspan=1 colspan=1>99.3% (2510/2528)</td><td rowspan=1 colspan=1>0.7% (18/2528)</td><td rowspan=1 colspan=1>0.0% (0/2528)</td><td rowspan=1 colspan=1>0.0% (0/2528)</td><td rowspan=1 colspan=1>0.0% (0/2528)</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Exact Match</td><td rowspan=1 colspan=1>1 Mismatch</td><td rowspan=1 colspan=1>2 Mismatches</td><td rowspan=1 colspan=1>3 Mismatches</td><td rowspan=1 colspan=1>3+ Mismatches</td></tr><tr><td rowspan=4 colspan=1>RdRp</td><td rowspan=1 colspan=1>Forward</td><td rowspan=1 colspan=1>99.6% (2488/2499)</td><td rowspan=1 colspan=1>0.4% (11/2499)</td><td rowspan=1 colspan=1>0.0% (0/2499)</td><td rowspan=1 colspan=1>0.0% (0/2499)</td><td rowspan=1 colspan=1>0.0% (0/2499)</td></tr><tr><td rowspan=1 colspan=1>Reverse</td><td rowspan=1 colspan=1>99.9% (2497/2499)</td><td rowspan=1 colspan=1>0.1% (2/2499)</td><td rowspan=1 colspan=1>0.0% (0/2499)</td><td rowspan=1 colspan=1>0.0% (0/2499)</td><td rowspan=1 colspan=1>0.0% (0/2499)</td></tr><tr><td rowspan=2 colspan=1>Probe</td><td rowspan=1 colspan=1>1 Insertion*</td><td rowspan=1 colspan=1>1 Insertion +1 Mismatch</td><td rowspan=1 colspan=1>1 Insertion +2 Mismatches</td><td rowspan=1 colspan=1>1 Insertion +3 Mismatches</td><td rowspan=1 colspan=1>1 Insertion +&gt; 3 Mismatches</td></tr><tr><td rowspan=1 colspan=1>99.5% (2486/2499)</td><td rowspan=1 colspan=1>0.5% (13/2499)</td><td rowspan=1 colspan=1>0.0% (0/2499)</td><td rowspan=1 colspan=1>0.0% (0/2499)</td><td rowspan=1 colspan=1>0.0% (0/2499)</td></tr></table>

\* RdRp probe design lacks dC at position 14 from the $_ 5 '$ -end of the oligonucleotide.

Table 9: Inclusivity Statistic - Sequences Collected Within 90 Days Prior to July 31, 2023   

<table><tr><td rowspan=1 colspan=2>SARS-CoV-2Target</td><td rowspan=1 colspan=5>In Silico Inclusivity Statistics</td></tr><tr><td rowspan=1 colspan=2></td><td rowspan=1 colspan=1>Exact Match</td><td rowspan=1 colspan=1>1 Mismatch</td><td rowspan=1 colspan=1>2 Mismatches</td><td rowspan=1 colspan=1>3 Mismatches</td><td rowspan=1 colspan=1>3+ Mismatches</td></tr><tr><td rowspan=3 colspan=1>E</td><td rowspan=1 colspan=1>Forward</td><td rowspan=1 colspan=1>99.3%(25156/25321)</td><td rowspan=1 colspan=1>0.7% (165/25321)</td><td rowspan=1 colspan=1>0.0% (0/25321)</td><td rowspan=1 colspan=1>0.0% (0/25321)</td><td rowspan=1 colspan=1>0.0% (0/25321)</td></tr><tr><td rowspan=1 colspan=1>Probe</td><td rowspan=1 colspan=1>99.9%(25292/25321)</td><td rowspan=1 colspan=1>0.1% (29/25321)</td><td rowspan=1 colspan=1>0.0% (0/25321)</td><td rowspan=1 colspan=1>0.0% (0/25321)</td><td rowspan=1 colspan=1>0.0% (0/25321)</td></tr><tr><td rowspan=1 colspan=1>Reverse</td><td rowspan=1 colspan=1>99.9%(25304/25321)</td><td rowspan=1 colspan=1>0.1% (16/25321)</td><td rowspan=1 colspan=1>0.0% (1/25321)</td><td rowspan=1 colspan=1>0.0% (0/25321)</td><td rowspan=1 colspan=1>0.0% (0/25321)</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Exact Match</td><td rowspan=1 colspan=1>1 Mismatch</td><td rowspan=1 colspan=1>2 Mismatches</td><td rowspan=1 colspan=1>3 Mismatches</td><td rowspan=1 colspan=1>3+ Mismatches</td></tr><tr><td rowspan=3 colspan=1>N2</td><td rowspan=1 colspan=1>Forward</td><td rowspan=1 colspan=1>99.2%(25272/25483)</td><td rowspan=1 colspan=1>0.8% (211/25483)</td><td rowspan=1 colspan=1>0.0% (0/25483)</td><td rowspan=1 colspan=1>0.0% (0/25483)</td><td rowspan=1 colspan=1>0.0% (0/25483)</td></tr><tr><td rowspan=1 colspan=1>Probe</td><td rowspan=1 colspan=1>99.6%(25389/25483)</td><td rowspan=1 colspan=1>0.4% (93/25483)</td><td rowspan=1 colspan=1>0.0% (1/25483)</td><td rowspan=1 colspan=1>0.0% (0/25483)</td><td rowspan=1 colspan=1>0.0% (0/25483)</td></tr><tr><td rowspan=1 colspan=1>Reverse</td><td rowspan=1 colspan=1>99.5%(25353/25483)</td><td rowspan=1 colspan=1>0.5% (130/25483)</td><td rowspan=1 colspan=1>0.0% (0/25483)</td><td rowspan=1 colspan=1>0.0% (0/25483)</td><td rowspan=1 colspan=1>0.0% (0/25483)</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Exact Match</td><td rowspan=1 colspan=1>1 Mismatch</td><td rowspan=1 colspan=1>2 Mismatches</td><td rowspan=1 colspan=1>3 Mismatches</td><td rowspan=1 colspan=1>3+ Mismatches</td></tr><tr><td rowspan=4 colspan=1>RdRp</td><td rowspan=1 colspan=1>Forward</td><td rowspan=1 colspan=1>99.7%(25216/25302)</td><td rowspan=1 colspan=1>0.3% (86/25302)</td><td rowspan=1 colspan=1>0.0% (0/25302)</td><td rowspan=1 colspan=1>0.0% (0/25302)</td><td rowspan=1 colspan=1>0.0% (0/25302)</td></tr><tr><td rowspan=1 colspan=1>Reverse</td><td rowspan=1 colspan=1>99.8%(25242/25302)</td><td rowspan=1 colspan=1>0.2% (60/25302)</td><td rowspan=1 colspan=1>0.0% (0/25302)</td><td rowspan=1 colspan=1>0.0% (0/25302)</td><td rowspan=1 colspan=1>0.0% (0/25302)</td></tr><tr><td rowspan=2 colspan=1>Probe</td><td rowspan=1 colspan=1>1 Insertion*</td><td rowspan=1 colspan=1>1 Insertion +1 Mismatch</td><td rowspan=1 colspan=1>1 Insertion +2 Mismatches</td><td rowspan=1 colspan=1>1 Insertion +3 Mismatches</td><td rowspan=1 colspan=1>1 Insertion +&gt; 3 Mismatches</td></tr><tr><td rowspan=1 colspan=1>99.6%(25192/25302)</td><td rowspan=1 colspan=1>0.4% (110/25302)</td><td rowspan=1 colspan=1>0.0% (0/25302)</td><td rowspan=1 colspan=1>0.0% (0/25302)</td><td rowspan=1 colspan=1>0.0% (0/25302)</td></tr></table>

\* RdRp probe design lacks dC at position 14 from the $_ 5 \cdot$ -end of the oligonucleotide.

Table 10: Inclusivity Statistic - Sequences Collected Within 180 Days Prior to July 31, 2023   

<table><tr><td rowspan=1 colspan=2>SARS-CoV-2Target</td><td rowspan=1 colspan=5>In Silico Inclusivity Statistics</td></tr><tr><td rowspan=1 colspan=2></td><td rowspan=1 colspan=1>Exact Match</td><td rowspan=1 colspan=1>1 Mismatch</td><td rowspan=1 colspan=1>2 Mismatches</td><td rowspan=1 colspan=1>3 Mismatches</td><td rowspan=1 colspan=1>3+ Mismatches</td></tr><tr><td rowspan=3 colspan=1>E</td><td rowspan=1 colspan=1>Forward</td><td rowspan=1 colspan=1>99.5%(147492/148212)</td><td rowspan=1 colspan=1>0.5% (717/148212)</td><td rowspan=1 colspan=1>0.0% (3/148212)</td><td rowspan=1 colspan=1>0.0% (0/148212)</td><td rowspan=1 colspan=1>0.0% (0/148212)</td></tr><tr><td rowspan=1 colspan=1>Probe</td><td rowspan=1 colspan=1>99.8%(147963/148212)</td><td rowspan=1 colspan=1>0.2% (249/148212)</td><td rowspan=1 colspan=1>0.0% (0/148212)</td><td rowspan=1 colspan=1>0.0% (0/148212)</td><td rowspan=1 colspan=1>0.0% (0/148212)</td></tr><tr><td rowspan=1 colspan=1>Reverse</td><td rowspan=1 colspan=1>99.9%(148106/148212)</td><td rowspan=1 colspan=1>0.1% (105/148212)</td><td rowspan=1 colspan=1>0.0% (1/148212)</td><td rowspan=1 colspan=1>0.0% (0/148212)</td><td rowspan=1 colspan=1>0.0% (0/148212)</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Exact Match</td><td rowspan=1 colspan=1>1 Mismatch</td><td rowspan=1 colspan=1>2 Mismatches</td><td rowspan=1 colspan=1>3 Mismatches</td><td rowspan=1 colspan=1>3+ Mismatches</td></tr><tr><td rowspan=3 colspan=1>N2</td><td rowspan=1 colspan=1>Forward</td><td rowspan=1 colspan=1>99.1%(149625/150916)</td><td rowspan=1 colspan=1>0.9%(1287/150916)</td><td rowspan=1 colspan=1>0.0% (4/150916)</td><td rowspan=1 colspan=1>0.0% (0/150916)</td><td rowspan=1 colspan=1>0.0% (0/150916)</td></tr><tr><td rowspan=1 colspan=1>Probe</td><td rowspan=1 colspan=1>99.8%(150597/150916)</td><td rowspan=1 colspan=1>0.2% (315/150916)</td><td rowspan=1 colspan=1>0.0% (4/150916)</td><td rowspan=1 colspan=1>0.0% (0/150916)</td><td rowspan=1 colspan=1>0.0% (0/150916)</td></tr><tr><td rowspan=1 colspan=1>Reverse</td><td rowspan=1 colspan=1>99.6%(150373/150916)</td><td rowspan=1 colspan=1>0.4% (543/150916)</td><td rowspan=1 colspan=1>0.0% (0/150916)</td><td rowspan=1 colspan=1>0.0% (0/150916)</td><td rowspan=1 colspan=1>0.0% (0/150916)</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Exact Match</td><td rowspan=1 colspan=1>1 Mismatch</td><td rowspan=1 colspan=1>2 Mismatches</td><td rowspan=1 colspan=1>3 Mismatches</td><td rowspan=1 colspan=1>3+ Mismatches</td></tr><tr><td rowspan=4 colspan=1>RdRp</td><td rowspan=1 colspan=1>Forward</td><td rowspan=1 colspan=1>99.8%(146465/146703)</td><td rowspan=1 colspan=1>0.2% (238/146703)</td><td rowspan=1 colspan=1>0.0% (0/146703)</td><td rowspan=1 colspan=1>0.0% (0/146703)</td><td rowspan=1 colspan=1>0.0% (0/146703)</td></tr><tr><td rowspan=1 colspan=1>Reverse</td><td rowspan=1 colspan=1>99.9%(146489/146703)</td><td rowspan=1 colspan=1>0.1% (214/146703)</td><td rowspan=1 colspan=1>0.0% (0/146703)</td><td rowspan=1 colspan=1>0.0% (0/146703)</td><td rowspan=1 colspan=1>0.0% (0/146703)</td></tr><tr><td rowspan=2 colspan=1>Probe</td><td rowspan=1 colspan=1>1 Insertion*</td><td rowspan=1 colspan=1>1 Insertion +1 Mismatch</td><td rowspan=1 colspan=1>1 Insertion +2 Mismatches</td><td rowspan=1 colspan=1>1 Insertion +3 Mismatches</td><td rowspan=1 colspan=1>1 Insertion +&gt; 3 Mismatches</td></tr><tr><td rowspan=1 colspan=1>99.7%(146204/146703)</td><td rowspan=1 colspan=1>0.3% (499/146703)</td><td rowspan=1 colspan=1>0.0% (0/146703)</td><td rowspan=1 colspan=1>0.0% (0/146703)</td><td rowspan=1 colspan=1>0.0% (0/146703)</td></tr></table>

\* RdRp probe design lacks dC at position 14 from the $_ 5 \cdot$ -end of the oligonucleotide.

# b) Inclusivity Wet-Testing

In addition to the in silico analysis of the SARS-CoV-2 primers and probes, the inclusivity of the Xpert Xpress CoV-2 plus test was evaluated by wet testing against multiple strains of SARS$\mathrm { C o V - } 2$ at ${ \sim } 3 \mathbf { x }$ LoD. A total of 61 strains comprised of 23 intact SARS-CoV-2 viral particles (18 inactivated viral cultures and 5 BSL-3 live viral cultures), and 38 SARS-CoV-2 in vitro RNA transcripts representing variant strains were tested in this study with the Xpert Xpress CoV-2 plus test. Three replicates were tested for each strain. All SARS-CoV-2 strains tested positive in all three replicates. Results are shown in Table 11.

Table 11: Analytical Reactivity (Inclusivity) of the Xpert Xpress CoV-2 plus Test   

<table><tr><td colspan="1" rowspan="2">CountNo.</td><td colspan="1" rowspan="2">SARS-CoV-2 Strain</td><td colspan="1" rowspan="2">TestedConcentration</td><td colspan="1" rowspan="2">SARS-CoV-2Test Results</td><td colspan="3" rowspan="1">Number of Positive ResultsObtained out of the Total Number ofValid Replicates</td></tr><tr><td colspan="1" rowspan="1">E</td><td colspan="1" rowspan="1">N2</td><td colspan="1" rowspan="1">RdRP</td></tr><tr><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">2019-nCoV/Italy-INMI1 a</td><td colspan="1" rowspan="1">5 TCID50/mL</td><td colspan="1" rowspan="1">POS</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">England/204820464/2020 a</td><td colspan="1" rowspan="1">0.5 TCID50/mL</td><td colspan="1" rowspan="1">POS</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">Hong Kong/VM20001061/2020 a</td><td colspan="1" rowspan="1">0.25 TCID50/mL</td><td colspan="1" rowspan="1">POS</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">South Africa/KRISP-K005325/2020 a</td><td colspan="1" rowspan="1">0.25 TCID50/mL</td><td colspan="1" rowspan="1">POS</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">USA/CA_CDC_5574/2020 a</td><td colspan="1" rowspan="1">0.25 TCID50/mL</td><td colspan="1" rowspan="1">POS</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">USA/MD-HP01542/2021 a</td><td colspan="1" rowspan="1">1.3e3 genomeequivalents/mL</td><td colspan="1" rowspan="1">POS</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">7</td><td colspan="1" rowspan="1">USA/GA-EHC-2811C/2021 a</td><td colspan="1" rowspan="1">1.3e3 genomeequivents/mL</td><td colspan="1" rowspan="1">POS</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">8</td><td colspan="1" rowspan="1">NY-Wadsworth-103677-01/2020 a</td><td colspan="1" rowspan="1">0.15 TCID50/mL</td><td colspan="1" rowspan="1">POS</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">9</td><td colspan="1" rowspan="1">NY-Wadsworth-21006055-01/2021 a</td><td colspan="1" rowspan="1">0.04 TCID50/mL</td><td colspan="1" rowspan="1">POS</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">NY-Wadsworth-21025952-01/2021 a(Isolate 1)</td><td colspan="1" rowspan="1">0.625 TCID50/mL</td><td colspan="1" rowspan="1">POS</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">11</td><td colspan="1" rowspan="1">NY-Wadsworth-21018781-01/2021 a(Isolate 2)</td><td colspan="1" rowspan="1">0.625 TCID50/mL</td><td colspan="1" rowspan="1">POS</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">12</td><td colspan="1" rowspan="1">NY-Wadsworth-21033899-01/2021 a</td><td colspan="1" rowspan="1">1.25 TCID50/mL</td><td colspan="1" rowspan="1">POS</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">13</td><td colspan="1" rowspan="1">NY-Wadsworth-33126-01/2020 a</td><td colspan="1" rowspan="1">0.15 TCID50/mL</td><td colspan="1" rowspan="1">POS</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">14</td><td colspan="1" rowspan="1">USA/CA-Stanford-15_S02/2021a</td><td colspan="1" rowspan="1">0.04 TCID50/mL</td><td colspan="1" rowspan="1">POS</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">15</td><td colspan="1" rowspan="1">USA/PHC658/2021a</td><td colspan="1" rowspan="1">25 TCID50/mL</td><td colspan="1" rowspan="1">POS</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">16</td><td colspan="1" rowspan="1">Japan/TY7-503/2021a</td><td colspan="1" rowspan="1">0.75 TCID50/mL</td><td colspan="1" rowspan="1">POS</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">17</td><td colspan="1" rowspan="1">hCoV-19/USA/MD-HP30386/2022 a</td><td colspan="1" rowspan="1">27.5 copies/mL</td><td colspan="1" rowspan="1">POS</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">18</td><td colspan="1" rowspan="1">hCoV-19/USA/COR-22-063113/2022 a</td><td colspan="1" rowspan="1">19 copies/mL</td><td colspan="1" rowspan="1">POS</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">19</td><td colspan="1" rowspan="1">Australia/VIC01/2020 b</td><td colspan="1" rowspan="1">1.2e3 copies/mL</td><td colspan="1" rowspan="1">POS</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">Belgium/ULG/10004/2020 </td><td colspan="1" rowspan="1">1.2e3 copies/mL</td><td colspan="1" rowspan="1">POS</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">21</td><td colspan="1" rowspan="1">England/205041766/2020 b</td><td colspan="1" rowspan="1">1.2e3 copies/mL</td><td colspan="1" rowspan="1">POS</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">22</td><td colspan="1" rowspan="1">England/MILK-9E05B3/2020 b</td><td colspan="1" rowspan="1">1.2e3 copies/mL</td><td colspan="1" rowspan="1">POS</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">23</td><td colspan="1" rowspan="1">England/SHEF-C05B2/2020 b</td><td colspan="1" rowspan="1">1.2e3 copies/mL</td><td colspan="1" rowspan="1">POS</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">24</td><td colspan="1" rowspan="1">France/HF2393/2020 b</td><td colspan="1" rowspan="1">1.2e3 copies/mL</td><td colspan="1" rowspan="1">POS</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">25</td><td colspan="1" rowspan="1">Hong Kong/HKU-211129-001/2021 b</td><td colspan="1" rowspan="1">1.2e3 copies/mL</td><td colspan="1" rowspan="1">POS</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">26</td><td colspan="1" rowspan="1">Iceland/5/2020 b</td><td colspan="1" rowspan="1">1.2e3 copies/mL</td><td colspan="1" rowspan="1">POS</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">27</td><td colspan="1" rowspan="1">India/CT-ILSGS00361/2021 b</td><td colspan="1" rowspan="1">1.2e3 copies/mL</td><td colspan="1" rowspan="1">POS</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">28</td><td colspan="1" rowspan="1">India/MH-NCCS-P1162000182735/2021 b</td><td colspan="1" rowspan="1">1.2e3 copies/mL</td><td colspan="1" rowspan="1">POS</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">29</td><td colspan="1" rowspan="1">India/MH-SEQ-221 S66 R1 001/2021 b</td><td colspan="1" rowspan="1">1.2e3 copies/mL</td><td colspan="1" rowspan="1">POS</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">30</td><td colspan="1" rowspan="1">Japan/Hu_DP_Kng_19-020/2020</td><td colspan="1" rowspan="1">1.2e3 copies/mL</td><td colspan="1" rowspan="1">POS</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">31</td><td colspan="1" rowspan="1">Japan/IC-0564/2021 b</td><td colspan="1" rowspan="1">1.2e3 copies/mL</td><td colspan="1" rowspan="1">POS</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">32</td><td colspan="1" rowspan="1">Portugal/PT9543/2021b</td><td colspan="1" rowspan="1">1.2e3 copies/mL</td><td colspan="1" rowspan="1">POS</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">33</td><td colspan="1" rowspan="1">South Africa/KRISP-EC-K005299/2020 b</td><td colspan="1" rowspan="1">1.2e3 copies/mL</td><td colspan="1" rowspan="1">POS</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">34</td><td colspan="1" rowspan="1">Taiwan/NTU02/2020 b</td><td colspan="1" rowspan="1">1.2e3 copies/mL</td><td colspan="1" rowspan="1">POS</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">35</td><td colspan="1" rowspan="1">USA/CA9/2020 b</td><td colspan="1" rowspan="1">1.2e3 copies/mL</td><td colspan="1" rowspan="1">POS</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">36</td><td colspan="1" rowspan="1">USA/CA-PC101P/2020b</td><td colspan="1" rowspan="1">1.2e3 copies/mL</td><td colspan="1" rowspan="1">POS</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">37</td><td colspan="1" rowspan="1">USA/CA-CZB-12943/2020 b</td><td colspan="1" rowspan="1">1.2e3 copies/mL</td><td colspan="1" rowspan="1">POS</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">38</td><td colspan="1" rowspan="1">USA/MN2-MDH2/2020 b</td><td colspan="1" rowspan="1">1.2e3 copies/mL</td><td colspan="1" rowspan="1">POS</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">39</td><td colspan="1" rowspan="1">USA/NY-MSHSPSP-PV24650/2020 b</td><td colspan="1" rowspan="1">1.2e3 copies/mL</td><td colspan="1" rowspan="1">POS</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">40</td><td colspan="1" rowspan="1">USA/TX1/2020 b</td><td colspan="1" rowspan="1">1.2e3 copies/mL</td><td colspan="1" rowspan="1">POS</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">41</td><td colspan="1" rowspan="1">USA/WA2/2020 b</td><td colspan="1" rowspan="1">1.2e3 copies/mL</td><td colspan="1" rowspan="1">POS</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">42</td><td colspan="1" rowspan="1">USA/WA-CDC-UW21061750277/2021b</td><td colspan="1" rowspan="1">1.2e3 copies/mL</td><td colspan="1" rowspan="1">POS</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">43</td><td colspan="1" rowspan="1">Wuhan-Hu-1 b</td><td colspan="1" rowspan="1">1.2e3 copies/mL</td><td colspan="1" rowspan="1">POS</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">44</td><td colspan="1" rowspan="1">Germany/BavPat1/2020c</td><td colspan="1" rowspan="1">1.2e3 genomecopies/mL</td><td colspan="1" rowspan="1">POS</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">45</td><td colspan="1" rowspan="1">Singapore/2/2020</td><td colspan="1" rowspan="1">300 genomeequivalents/mL</td><td colspan="1" rowspan="1">POS</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">46</td><td colspan="1" rowspan="1">USA/AZ1/2020</td><td colspan="1" rowspan="1">300 genomeequivalents/mL</td><td colspan="1" rowspan="1">POS</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">47</td><td colspan="1" rowspan="1">USA/CA1/2020</td><td colspan="1" rowspan="1">1.2e3 genomeequivalents/mL</td><td colspan="1" rowspan="1">POS</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">48</td><td colspan="1" rowspan="1">USA/CA2/2020</td><td colspan="1" rowspan="1">1.2e3 genomeequivalents/mL</td><td colspan="1" rowspan="1">POS</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">49</td><td colspan="1" rowspan="1">USA/CA3/2020c</td><td colspan="1" rowspan="1">400 genomeequivalents/mL</td><td colspan="1" rowspan="1">POS</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">50</td><td colspan="1" rowspan="1">USA/CA4/2020c</td><td colspan="1" rowspan="1">1.2e3 genomeequivalents/mL</td><td colspan="1" rowspan="1">POS</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">51</td><td colspan="1" rowspan="1">USA/IL1/2020</td><td colspan="1" rowspan="1">150 genomeequivalents/mL</td><td colspan="1" rowspan="1">POS</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">52</td><td colspan="1" rowspan="1">USA/New York-PV08001/2020 c</td><td colspan="1" rowspan="1">1.2e3 genomecopies/mL</td><td colspan="1" rowspan="1">POS</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">53</td><td colspan="1" rowspan="1">USA/New York-PV08410/2020c</td><td colspan="1" rowspan="1">300 genomeequivalents/mL</td><td colspan="1" rowspan="1">POS</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">54</td><td colspan="1" rowspan="1">USA/New York-PV08449/2020 c</td><td colspan="1" rowspan="1">300 genomeequivalents/mL</td><td colspan="1" rowspan="1">POS</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">55</td><td colspan="1" rowspan="1">USA/New York-PV09158/2020 c</td><td colspan="1" rowspan="1">300 genomeequivalents/mL</td><td colspan="1" rowspan="1">POS</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">56</td><td colspan="1" rowspan="1">USA/WI1/2020</td><td colspan="1" rowspan="1">1.2e3 genomeequivalents/mL</td><td colspan="1" rowspan="1">POS</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">57</td><td colspan="1" rowspan="1">hCoV-19/USA/MD-HP38861/2022 d</td><td colspan="1" rowspan="1">0.29 TCID50/mL</td><td colspan="1" rowspan="1">POS</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">58</td><td colspan="1" rowspan="1">hCoV-19/USA/MD-HP38960/2022 d</td><td colspan="1" rowspan="1">0.043 TCID50/mL</td><td colspan="1" rowspan="1">POS</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">59</td><td colspan="1" rowspan="1">hCoV-19/USA/CO-CDPHE-2102544747/2021 d</td><td colspan="1" rowspan="1">0.028 TCID50/mL</td><td colspan="1" rowspan="1">POS</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">60</td><td colspan="1" rowspan="1">hCoV-19/USA/MD-HP38288/2022 d</td><td colspan="1" rowspan="1">0.24 TCID50/mL</td><td colspan="1" rowspan="1">POS</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">61</td><td colspan="1" rowspan="1">hCoV-19/USA/MD-HP40900/2022 d</td><td colspan="1" rowspan="1">0.72 TCID50/mL</td><td colspan="1" rowspan="1">POS</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr></table>

a. Inactivated viral culture fluid (intact viral particles) b. Synthetic, in vitro RNA transcript, TWIST controls c. Genomic RNA d. BSL-3 live viral culture fluid (intact viral particles) e. One of 3 replicates reported ERROR. The run was successfully repeated to obtain 3 valid replicates.

# 4. Analytical Specificity/Interference:

# a) In Silico Analysis

The analytical specificity/cross-reactivity of the Xpert Xpress CoV-2 plus plan included evaluation of the SARS-CoV-2 test primer and probes with potentially cross-reactive microorganisms by in silico analysis. The analysis was conducted by mapping the primers and probes of the Xpert Xpress CoV-2 plus individually to the microorganism sequences downloaded from the NCBI database. E primers and probes are not specific for SARS-COV2 and will detect Human and Bat SARS-coronavirus. No potential unintended cross reactivity with other organisms listed in Table 12 is expected based on the in silico analysis.

Table 12: Microorganisms Analyzed in the in silico Analysis for the SARS-CoV-2 Target   

<table><tr><td colspan="1" rowspan="1">Microorganisms from the Same GeneticFamily</td><td colspan="1" rowspan="1">High Priority Organisms</td></tr><tr><td colspan="1" rowspan="1">Human coronavirus 229E</td><td colspan="1" rowspan="1">Adenovirus (e.g., C1 Ad. 71)</td></tr><tr><td colspan="1" rowspan="1">Human coronavirus OC43</td><td colspan="1" rowspan="1">Cytomegalovirus</td></tr><tr><td colspan="1" rowspan="1">Human coronavirus HKU1</td><td colspan="1" rowspan="1">Enterovirus (e.g., EV68)</td></tr><tr><td colspan="1" rowspan="1">Human coronavirus NL63</td><td colspan="1" rowspan="1">Epstein-Barr virus</td></tr><tr><td colspan="1" rowspan="1">SARS-coronavirus</td><td colspan="1" rowspan="1">Human Metapneumovirus (hMPV)</td></tr><tr><td colspan="1" rowspan="1">MERS-coronavirus</td><td colspan="1" rowspan="1">Influenza A &amp; B</td></tr><tr><td colspan="1" rowspan="1">Bat coronavirus</td><td colspan="1" rowspan="1">Measles</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Mumps</td></tr><tr><td colspan="1" rowspan="1">EY</td><td colspan="1" rowspan="1">Parainfluenza virus 1-4</td></tr><tr><td colspan="1" rowspan="1">EY</td><td colspan="1" rowspan="1">Parechovirus</td></tr><tr><td colspan="1" rowspan="1">EY</td><td colspan="1" rowspan="1">Respiratory syncytial virus</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Rhinovirus</td></tr><tr><td colspan="1" rowspan="1">EY</td><td colspan="1" rowspan="1">Bacillus anthracis (Anthrax)</td></tr><tr><td colspan="1" rowspan="1">EY</td><td colspan="1" rowspan="1">Bordetella pertussis</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Bordetella parapertussis</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Chlamydia pneumoniae</td></tr><tr><td colspan="1" rowspan="1">EY</td><td colspan="1" rowspan="1">Chlamydia psittaci</td></tr><tr><td colspan="1" rowspan="1">EY</td><td colspan="1" rowspan="1">Corynebacterium diphtheriae</td></tr><tr><td colspan="1" rowspan="1">EY</td><td colspan="1" rowspan="1">Coxiella burnetii (Q-Fever)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Escherichia coli</td></tr><tr><td colspan="1" rowspan="1">EY</td><td colspan="1" rowspan="1">Fusobacterium necrophorum</td></tr><tr><td colspan="1" rowspan="1">EY</td><td colspan="1" rowspan="1">Haemophilus influenzae</td></tr><tr><td colspan="1" rowspan="1">EY</td><td colspan="1" rowspan="1">Lactobacillus sp.</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Legionella non-pneumophila</td></tr><tr><td colspan="1" rowspan="1">EY</td><td colspan="1" rowspan="1">Legionella pneumophila</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Leptospira</td></tr><tr><td colspan="1" rowspan="1">EY</td><td colspan="1" rowspan="1">Moraxella catarrhalis</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Mycobacterium tuberculosis</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Mycoplasma genitalium</td></tr><tr><td colspan="1" rowspan="1">EY</td><td colspan="1" rowspan="1">Mycoplasma pneumoniae</td></tr><tr><td colspan="1" rowspan="1">EY</td><td colspan="1" rowspan="1">Neisseria elongata</td></tr><tr><td colspan="1" rowspan="1">EY</td><td colspan="1" rowspan="1">Neisseria meningitidis</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Pneumocystis jirovecii (PJP)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Staphylococcus aureus</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Staphylococcus epidermidis</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Staphylococcus salivarius</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Streptococcus pneumoniae</td></tr><tr><td colspan="1" rowspan="1">EY</td><td colspan="1" rowspan="1">Streptococcus pyogenes</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Aspergillus sp</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Candida albicans</td></tr></table>

# b) Cross-reactivity Wet-Testing

In addition to in silico analysis of the SARS-CoV-2 primers and probes for cross-reactivity, the analytical specificity of the Xpert Xpress CoV-2 plus test was evaluated by bench-testing a panel of 62 microorganisms comprising (4) human coronaviruses, (1) MERS-coronavirus, (1) SARS-coronavirus, (20) other respiratory viruses, (30) respiratory bacteria, (2) yeast strains, (3) fungal strains, and (1) human nasal wash fluid representing a diverse microbial flora in the human respiratory tract.

The intact viruses were tested at concentrations of ${ \geq } 1 0 ^ { 5 }$ $\Gamma \mathrm { C I D } _ { 5 0 } / \mathrm { m L }$ , $\ge 1 0 ^ { 5 } \mathrm { C E I D } _ { 5 0 } / \mathrm { m L }$ , and ${ \geq } 1 0 ^ { 5 }$ copies $ { / \mathrm { m L } }$ . Bacteria and yeast were tested at $\geq 1 0 ^ { 6 } \mathrm { C F U / m L }$ . The bacteria Chlamydia pneumoniae and Mycoplasma pneumoniae were tested at concentrations o $\mathrm { f } \geq 1 0 ^ { 6 } \mathrm { I F U / m L }$ and $\ge 1 0 ^ { 6 } \mathrm { C C U / m L }$ , respectively. The live strains of Coxiella burnetii, Lactobacillus reuteri, and Neisseria meningitides were not available, therefore genomic DNA at $\geq 1 0 ^ { 6 }$ genome equivalent copies/mL was used. The in vitro transcribed (IVT) RNA sample was tested at $\geq 1 0 ^ { 6 }$ genome equivalent/mL for human Coronavirus HKU1, and the genomic RNA sample was tested at $\geq 1 0 ^ { 6 }$ genome equivalent/mL for SARS-coronavirus Urbani.

The microorganisms being evaluated for cross-reactivity were tested in groups or individually and spiked into negative simulated NPS/NS background matrix for testing. If a sample with grouped microorganisms produced a SARS-CoV-2 positive result, then each member of the group was tested separately. If the individual non-target organism yielded a positive result, retesting was performed at lower concentrations until a concentration that no longer produces a false positive result was identified.

The results of the cross-reactivity study demonstrated that the primer/probe sets included in the Xpert Xpress CoV-2 plus do not cross-react with the nucleic acids from non-intended respiratory microorganisms and phylogenetically related human coronavirus species. The one exception was the SARS-coronavirus Urbani which yielded SARS-CoV-2 POSITIVE test result. The cross-reactivity for the E gene target was expected with the SARS-coronavirus Urbani strain which has the highly conserved E gene target shared among coronaviruses in the B lineage Betacoronavirus. Results are shown in Table 13.

Table 13: Cross-reactivity testing of the Xpert Xpress CoV-2 plus Test   

<table><tr><td rowspan=2 colspan=1>Respiratory Microorganisms</td><td rowspan=2 colspan=1>TestGroup</td><td rowspan=2 colspan=1>Tested Concentration</td><td rowspan=2 colspan=1>FinalSARS-CoV-2Result Call-Out</td><td rowspan=1 colspan=3>Number of Positive ResultsObtained from Total Numberof Replicates Tested</td></tr><tr><td rowspan=1 colspan=1>E</td><td rowspan=1 colspan=1>N2</td><td rowspan=1 colspan=1>RdRP</td></tr><tr><td rowspan=1 colspan=1>Human coronavirus, 229E</td><td rowspan=3 colspan=1>G1</td><td rowspan=1 colspan=1>1.1e5 TCID50/mL</td><td rowspan=3 colspan=1>NEG</td><td rowspan=3 colspan=1>0/3</td><td rowspan=3 colspan=1>0/3</td><td rowspan=3 colspan=1>0/3</td></tr><tr><td rowspan=1 colspan=1>Human coronavirus, OC43</td><td rowspan=1 colspan=1>1.1e5 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>MERS-coronavirus</td><td rowspan=1 colspan=1>1.1e5 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Human coronavirus, NL63</td><td rowspan=1 colspan=1>G2</td><td rowspan=1 colspan=1>1.1e5 TCID50/mL</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td></tr><tr><td rowspan=1 colspan=1>Human coronavirus, HKU1a</td><td rowspan=1 colspan=1>G3</td><td rowspan=1 colspan=1>1.1e6 genome cp/mL</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td></tr><tr><td rowspan=1 colspan=1>SARS-coronavirus, Urbani a</td><td rowspan=1 colspan=1>G4</td><td rowspan=1 colspan=1>1.1e6 genome cp/mL</td><td rowspan=1 colspan=1>POS</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td></tr><tr><td rowspan=1 colspan=1>Influenza A H1N1 (pdm2009),Michigan/272/2017</td><td rowspan=3 colspan=1>G5</td><td rowspan=1 colspan=1>1.1e5 TCID50/mL</td><td rowspan=3 colspan=1>NEG</td><td rowspan=3 colspan=1>0/3</td><td rowspan=3 colspan=1>0/3</td><td rowspan=3 colspan=1>0/3</td></tr><tr><td rowspan=1 colspan=1>Influenza B (Victoria Lineage),Hawaii/01/2018 (NA D197N)</td><td rowspan=1 colspan=1>1.1e5 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>RSV-A, Strain: 4/2015 Isolate #1</td><td rowspan=1 colspan=1>1.1e5 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Adenovirus Type 1</td><td rowspan=3 colspan=1>G6</td><td rowspan=1 colspan=1>1.1e5 TCID50/mL</td><td rowspan=3 colspan=1>NEG</td><td rowspan=3 colspan=1>0/3</td><td rowspan=3 colspan=1>0/3</td><td rowspan=3 colspan=1>0/3</td></tr><tr><td rowspan=1 colspan=1>Adenovirus Type 7A</td><td rowspan=1 colspan=1>1.1e5 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Cytomegalovirus</td><td rowspan=1 colspan=1>1.1e5 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Echovirus</td><td rowspan=7 colspan=1>G7</td><td rowspan=1 colspan=1>1.1e5 TCID50/mL</td><td rowspan=7 colspan=1>NEG</td><td rowspan=7 colspan=1>0/3</td><td rowspan=7 colspan=1>0/3</td><td rowspan=7 colspan=1>0/3</td></tr><tr><td rowspan=1 colspan=1>Enterovirus, D68 strain US/KY/14-18953</td><td rowspan=1 colspan=1>1.1e5 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Epstein Barr Virus (Human Herpes Virus 4[Hhv-4])</td><td rowspan=1 colspan=1>1.1e5 cp/mL</td></tr><tr><td rowspan=1 colspan=1>Herpes Simplex Virus (HSV) type 1</td><td rowspan=1 colspan=1>1.1e5 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Human metapneumovirus(hMPV-5, type B1)</td><td rowspan=1 colspan=1>1.1e5 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Measles</td><td rowspan=1 colspan=1>1.1e5 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Mumps virus</td><td rowspan=1 colspan=1>1.1e5 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Human parainfluenza Type 1</td><td rowspan=5 colspan=1>G8</td><td rowspan=1 colspan=1>1.1e5 TCID50/mL</td><td rowspan=5 colspan=1>NEG</td><td rowspan=5 colspan=1>0/3</td><td rowspan=5 colspan=1>0/3</td><td rowspan=5 colspan=1>0/3</td></tr><tr><td rowspan=1 colspan=1>Human parainfluenza Type 2</td><td rowspan=1 colspan=1>1.1e5 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Human parainfluenza Type 3</td><td rowspan=1 colspan=1>1.1e5 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Human parainfluenza Type 4</td><td rowspan=1 colspan=1>1.1e5 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Rhinovirus, Type 1A b</td><td rowspan=1 colspan=1>4.5e4 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Acinetobacter baumannii</td><td rowspan=7 colspan=1>G9</td><td rowspan=1 colspan=1>1.1e6 CFU/mL</td><td rowspan=7 colspan=1>NEG</td><td rowspan=7 colspan=1>0/3</td><td rowspan=7 colspan=1>0/3</td><td rowspan=7 colspan=1>0/3</td></tr><tr><td rowspan=1 colspan=1>Burkholderia cepacia</td><td rowspan=1 colspan=1>1.1e6 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Candida albicans</td><td rowspan=1 colspan=1>1.1e6 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Candida parapsilosis</td><td rowspan=1 colspan=1>1.1e6 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Bordetella pertussis</td><td rowspan=1 colspan=1>1.1e6 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Chlamydia pneumoniae</td><td rowspan=1 colspan=1>1.1e6 IFU/mL</td></tr><tr><td rowspan=1 colspan=1>Citrobacter freundii</td><td rowspan=1 colspan=1>1.1e6 CFU/mL</td></tr></table>

<table><tr><td rowspan=2 colspan=1>Respiratory Microorganisms</td><td rowspan=2 colspan=5>TestGroup</td><td rowspan=2 colspan=1>Tested Concentration</td><td rowspan=2 colspan=1>FinalSARS-CoV-2Result Call-Out</td><td rowspan=1 colspan=3>Number of Positive ResultsObtained from Total Numberof Replicates Tested</td></tr><tr><td rowspan=1 colspan=1>E</td><td rowspan=1 colspan=1>N2</td><td rowspan=1 colspan=1>RdRP</td></tr><tr><td rowspan=1 colspan=1>Corynebacterium xerosis</td><td rowspan=6 colspan=5>G10</td><td rowspan=1 colspan=1>1.1e6 CFU/mL</td><td rowspan=6 colspan=1>NEG</td><td rowspan=6 colspan=1>0/3</td><td rowspan=6 colspan=1>0/3</td><td rowspan=6 colspan=1>0/3</td></tr><tr><td rowspan=1 colspan=1>Escherichia coli</td><td rowspan=1 colspan=1>1.1e6 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Enterococcus faecalis</td><td rowspan=1 colspan=1>1.1e6 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Hemophilus influenzae</td><td rowspan=1 colspan=1>1.1e6 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Legionella sppd</td><td rowspan=1 colspan=1>1.1e6 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Moraxella catarrhalis</td><td rowspan=1 colspan=1>1.1e6 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Mycobacterium tuberculosis (avirulent)</td><td rowspan=6 colspan=5>G11</td><td rowspan=1 colspan=1>1.1e6 CFU/mL</td><td rowspan=6 colspan=1>NEG</td><td rowspan=6 colspan=1>0/3</td><td rowspan=6 colspan=1>0/3</td><td rowspan=6 colspan=1>0/3</td></tr><tr><td rowspan=1 colspan=1>Mycoplasma pneumoniae</td><td rowspan=1 colspan=1>1.1e6 CCU/mL</td></tr><tr><td rowspan=1 colspan=1>Neisseria mucosa</td><td rowspan=1 colspan=1>1.1e6 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Propionibacterium acnes (= Cutibacteriumacnes) Z144</td><td rowspan=1 colspan=1>1.1e6 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Pseudomonas aeruginosa, Z139</td><td rowspan=1 colspan=1>1.1e6 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus aureus</td><td rowspan=1 colspan=1>1.1e6 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus epidermidis</td><td rowspan=2 colspan=5></td><td rowspan=1 colspan=1>1.1e6 CFU/mL</td><td rowspan=6 colspan=1>NEG</td><td rowspan=13 colspan=1>0/3</td><td rowspan=13 colspan=1>0/3</td><td rowspan=13 colspan=1>0/3</td></tr><tr><td rowspan=1 colspan=1>Staphyloccus haemolyticus</td><td rowspan=1 colspan=4></td><td rowspan=1 colspan=1>1.1e6 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Streptococcus agalactiae</td><td rowspan=1 colspan=2></td><td rowspan=1 colspan=2></td><td></td><td rowspan=1 colspan=1>1.1e6 CFU/mL</td></tr><tr><td rowspan=2 colspan=1>Streptococcus pneumoniae</td><td rowspan=2 colspan=1>−</td><td rowspan=1 colspan=3></td><td></td><td rowspan=2 colspan=1>1.1e6 CFU/mL</td></tr><tr><td rowspan=1 colspan=2></td><td></td><td></td><td rowspan=2 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Streptococcus pyogenes</td><td rowspan=1 colspan=2></td><td></td><td rowspan=1 colspan=2>2</td><td rowspan=1 colspan=1>1.1e6 CFU/mL</td></tr><tr><td rowspan=3 colspan=1>Streptococcus salivarius</td><td rowspan=3 colspan=3></td><td rowspan=3 colspan=2></td><td rowspan=3 colspan=1>1.1e6 CFU/mL</td><td></td></tr><tr><td rowspan=6 colspan=5></td><td></td></tr><tr><td rowspan=5 colspan=1></td></tr><tr><td rowspan=3 colspan=1>Streptococcus sanguinis</td><td></td></tr><tr><td></td></tr><tr><td rowspan=1 colspan=1>1.1e6 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Pneumocystis jirovecii (PJP)</td><td rowspan=1 colspan=1>1.1e6 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Lactobacillus reuteri, F275 c</td><td rowspan=2 colspan=5>G13</td><td rowspan=1 colspan=1>1.1e6 genome cp/mL</td><td rowspan=2 colspan=1>NEG</td><td rowspan=2 colspan=1>0/3</td><td rowspan=2 colspan=1>0/3</td><td rowspan=2 colspan=1>0/3</td></tr><tr><td rowspan=1 colspan=1>Neisseria meningitides c</td><td rowspan=1 colspan=1>1.1e6 genome cp/mL</td></tr><tr><td rowspan=1 colspan=1>Pooled human nasal wash</td><td rowspan=1 colspan=5>G14</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td></tr><tr><td rowspan=1 colspan=1>Influenza C (Taylor/1233/1947)</td><td rowspan=1 colspan=5>G15</td><td rowspan=1 colspan=1>1.1e5 CEID50/mL</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td></tr><tr><td rowspan=1 colspan=1>Rhinovirus, Type 1Ab</td><td rowspan=1 colspan=5>G16</td><td rowspan=1 colspan=1>1.1e5 TCID50/mL</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td></tr><tr><td rowspan=1 colspan=1>Aspergillus flavus</td><td rowspan=1 colspan=5>G17</td><td rowspan=1 colspan=1>1.35e7 CFU/mL</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td></tr><tr><td rowspan=1 colspan=1>Aspergillus fumigatus</td><td rowspan=1 colspan=5>G18</td><td rowspan=1 colspan=1>3.67e6 CFU/mL</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td></tr><tr><td rowspan=1 colspan=1>Coxiella burnetii</td><td rowspan=1 colspan=5>G19</td><td rowspan=1 colspan=1>2.5e6 genome cp/mL</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td></tr><tr><td rowspan=1 colspan=1>Leptospira broomii</td><td rowspan=1 colspan=5>G20</td><td rowspan=1 colspan=1>2.9e6 CFU/mL</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td></tr><tr><td rowspan=1 colspan=1>Parechovirus type 1</td><td rowspan=1 colspan=5>G21</td><td rowspan=1 colspan=1>3.39e5 TCID50/mL</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td></tr><tr><td rowspan=1 colspan=1>Fusobacterium necrophorum</td><td rowspan=1 colspan=5>G22</td><td rowspan=1 colspan=1>6.52e6 CFU/mL</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td></tr><tr><td rowspan=1 colspan=1>Mycoplasma genitalium </td><td rowspan=1 colspan=5>G23</td><td rowspan=1 colspan=1>3.6e6 genome cp/mL</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0/3</td></tr></table>

a. Microorganism in the form of genomic RNA were tested in Tris-EDTA $^ +$ ([NH4]2SO4) buffer using an assay definition file (ADF) without sample preparation. b. Rhinovirus, Type 1A was initially tested at $4 . 5 \ : \mathrm { e 4 }$ $\Gamma \mathrm { C I D } _ { 5 0 } / \mathrm { m L }$ in test group G8 using virus lot 325725. It was re-tested individually (G16) at a higher concentration of 1.1e5 $\mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ using virus lot 326527. c. Microorganisms in the form of genomic DNA were tested in simulated NPS/NS background matrix using the ADF with full sample preparation. d. Legionella pneumophila was tested in this study.

# c) Microbial Interference

A microbial interference study was performed to assess the inhibitory effects of commensal microorganisms potentially encountered in upper respiratory tract specimens on the performance of the Xpert Xpress CoV-2 plus test. A panel of 18 commensal microorganisms, consisting of 15 viral strains and 3 bacterial strains was tested. Contrived samples consisted of SARS-CoV-2 virus spiked at $1 . 1 5 \mathrm { x } { - 3 \mathrm { x } }$ LoD into simulated NPS/NS matrix in the presence of fifteen (15) commensal virus strains and three (3) commensal bacterial strains (Haemophilus influenzae, Staphylococcus aureus, Staphylococcus epidermidis) spiked at their respective concentrations listed in Table 14. SARS-CoV-2 virus and each potential microbial interference strain combination was tested in replicates of eight. All eight positive replicate samples were correctly identified as SARS-CoV-2 POSITIVE using the Xpert

Xpress CoV-2 plus test. No interference by the listed above commensal viral or bacterial strains was reported at the concentrations tested.

Table 14: Microbial Interference Study Results   

<table><tr><td rowspan=1 colspan=1>Commensal Strain</td><td rowspan=1 colspan=1>ConcentrationTested</td><td rowspan=1 colspan=1>Number ofCorrect TestResults/ NumberTested</td></tr><tr><td rowspan=1 colspan=1>Adenovirus Type 1Ca</td><td rowspan=1 colspan=1>1x105TCID50/mL</td><td rowspan=1 colspan=1>8/8</td></tr><tr><td rowspan=1 colspan=1>Adenovirus Type 2Cb</td><td rowspan=1 colspan=1>1x105 TCID50/mL</td><td rowspan=1 colspan=1>8/8c</td></tr><tr><td rowspan=1 colspan=1>Adenovirus Type 3Bb</td><td rowspan=1 colspan=1>1x105TCID50/mL</td><td rowspan=1 colspan=1>8/8</td></tr><tr><td rowspan=1 colspan=1>Human Coronavirus-OC43a</td><td rowspan=1 colspan=1>1x105 copies/mL</td><td rowspan=1 colspan=1>8/8</td></tr><tr><td rowspan=1 colspan=1>HumanCoronavirus-229Eb</td><td rowspan=1 colspan=1>1x105 TCID50/mL</td><td rowspan=1 colspan=1>8/8</td></tr><tr><td rowspan=1 colspan=1>HumanCoronavirus-NL63b</td><td rowspan=1 colspan=1>1x105TCID50/mL</td><td rowspan=1 colspan=1>8/8</td></tr><tr><td rowspan=1 colspan=1>Human Coronavirus-HKU1b</td><td rowspan=1 colspan=1>1x105 copies/mL</td><td rowspan=1 colspan=1>8/8</td></tr><tr><td rowspan=1 colspan=1>Metapneumovirus 5, Type B1a</td><td rowspan=1 colspan=1>1x105TCID50/mL</td><td rowspan=1 colspan=1>8/8</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza Type 1a</td><td rowspan=1 colspan=1>1x105 TCID50/mL</td><td rowspan=1 colspan=1>8/8</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza Type 2a</td><td rowspan=1 colspan=1>1x105TCID50/mL</td><td rowspan=1 colspan=1>8/8</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza Type 3a</td><td rowspan=1 colspan=1>1x105 TCID50/mL</td><td rowspan=1 colspan=1>8/8</td></tr><tr><td rowspan=1 colspan=1>Rhinovirus Type 1Aa</td><td rowspan=1 colspan=1>1x105 PFU/mL</td><td rowspan=1 colspan=1>8/8</td></tr><tr><td rowspan=1 colspan=1>Influenza A H1N1b</td><td rowspan=1 colspan=1>1x105CEID50/mL</td><td rowspan=1 colspan=1>8/8</td></tr><tr><td rowspan=1 colspan=1>Influenza Bb</td><td rowspan=1 colspan=1>1x105TCID50/mL</td><td rowspan=1 colspan=1>8/8d</td></tr><tr><td rowspan=1 colspan=1>Respiratory Syncytial Virus Ab</td><td rowspan=1 colspan=1>1x105 TCID50/mL</td><td rowspan=1 colspan=1>8/8</td></tr><tr><td rowspan=1 colspan=1>Haemophilus influenzaea</td><td rowspan=1 colspan=1>1x107 CFU/mL</td><td rowspan=1 colspan=1>8/8</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus aureusa</td><td rowspan=1 colspan=1>1x107 CFU/mL</td><td rowspan=1 colspan=1>8/8</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus epidermidisa</td><td rowspan=1 colspan=1>1x107 CFU/mL</td><td rowspan=1 colspan=1>8/8</td></tr></table>

a. These commensal strains were tested with SARS-CoV-2 at a concentration of 3x LoD. b. These commensal strains were tested with SARS-CoV-2 at a concentration of $1 . 1 5 \mathrm { x }$ LoD. c. Two of 8 replicates reported as ERROR. The runs were successfully repeated to obtain 8 valid replicates. d. One of 8 replicates reported as ERROR. The run was successfully repeated to obtain 8 valid replicates.

# d) Endogenous/Exogenous Interference Substances

An interfering substances study was performed to evaluate if substances naturally present or artificially introduced into the nasal cavity or nasopharynx interfere with the Xpert Xpress CoV-2 plus test performance. Positive and negative samples were prepared in simulated NPS/NS matrix. Each interfering substance was tested at the concentration listed below (Table 15) in the absence of SARS-CoV-2 and when SARS-CoV-2 is present at 3x LoD. All specimens were tested in eight replicates on the Xpert Xpress CoV-2 plus test according to the test procedure. For substances that resulted in an INVALID test result, the concentration of the substance was reduced by half and re-tested.

Expected results were obtained for all interfering substances tested, except for Mucin (Type I-S) $( 2 . 5 \mathrm { m g / m L } )$ and Fluticasone Propionate Nasal Spray $( 5 ~ \mu \mathrm { g / m L } )$ . Table 15 below presents the concentration at which no interference was seen for each substance tested.

Table 15: Interfering Substances Study Results   

<table><tr><td rowspan=1 colspan=1>Substance</td><td rowspan=1 colspan=1>ConcentrationTested</td><td rowspan=1 colspan=1>Number SARS-CoV-2 Negative /Number of ValidReplicates Tested</td><td rowspan=1 colspan=1>Number SARS-CoV-2 Positive /Number of ValidReplicates Tested</td></tr><tr><td rowspan=1 colspan=1>SARS-CoV-2 Negative(No Substance) Control</td><td rowspan=1 colspan=1>100% (v/v)</td><td rowspan=1 colspan=1>8/8</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>SARS-CoV-2 Positive(No substance) Control</td><td rowspan=1 colspan=1>100% (v/v)</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>8/8</td></tr><tr><td rowspan=1 colspan=1>Afrin</td><td rowspan=1 colspan=1>15% (v/v)</td><td rowspan=1 colspan=1>8/8</td><td rowspan=1 colspan=1>8/8</td></tr><tr><td rowspan=1 colspan=1>Albuterol Sulfate</td><td rowspan=1 colspan=1>0.83 mg/mL</td><td rowspan=1 colspan=1>8/8</td><td rowspan=1 colspan=1>8/8</td></tr><tr><td rowspan=1 colspan=1>BD Universal Transport Medium</td><td rowspan=1 colspan=1>100 (v/v)</td><td rowspan=1 colspan=1>8/8</td><td rowspan=1 colspan=1>8/8</td></tr><tr><td rowspan=1 colspan=1>Blood</td><td rowspan=1 colspan=1>2% (v/v)</td><td rowspan=1 colspan=1>8/8</td><td rowspan=1 colspan=1>8/8</td></tr><tr><td rowspan=1 colspan=1>Copan Swab M</td><td rowspan=1 colspan=1>100 (v/v)</td><td rowspan=1 colspan=1>8/8</td><td rowspan=1 colspan=1>8/8</td></tr><tr><td rowspan=1 colspan=1>FluMist</td><td rowspan=1 colspan=1>6.7% (v/v)</td><td rowspan=1 colspan=1>8/8</td><td rowspan=1 colspan=1>8/8</td></tr><tr><td rowspan=2 colspan=1>Fluticasone Propionate Nasal Spray</td><td rowspan=1 colspan=1>5 μg/mL</td><td rowspan=1 colspan=1>7/8a</td><td rowspan=1 colspan=1>7/8a</td></tr><tr><td rowspan=1 colspan=1>2.5 μg/mL</td><td rowspan=1 colspan=1>8/8</td><td rowspan=1 colspan=1>8/8</td></tr><tr><td rowspan=1 colspan=1>Ibuprofen</td><td rowspan=1 colspan=1>21.9 mg/dL</td><td rowspan=1 colspan=1>8/8</td><td rowspan=1 colspan=1>8/8</td></tr><tr><td rowspan=1 colspan=1>Leukocytes</td><td rowspan=1 colspan=1>1 x 106 cells/mL</td><td rowspan=1 colspan=1>8/8</td><td rowspan=1 colspan=1>8/8</td></tr><tr><td rowspan=1 colspan=1>Menthol</td><td rowspan=1 colspan=1>1.7 mg/mL</td><td rowspan=1 colspan=1>8/8</td><td rowspan=1 colspan=1>8/8</td></tr><tr><td rowspan=1 colspan=1>Mucin (Type II)</td><td rowspan=1 colspan=1>0.1% (w/v)</td><td rowspan=1 colspan=1>8/8</td><td rowspan=1 colspan=1>8/8</td></tr><tr><td rowspan=2 colspan=1>Mucin (Type I-S)</td><td rowspan=1 colspan=1>2.5 mg/mL</td><td rowspan=1 colspan=1>7/8a</td><td rowspan=1 colspan=1>8/8</td></tr><tr><td rowspan=1 colspan=1>1.25 mg/mL</td><td rowspan=1 colspan=1>8/8</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>Mupirocin</td><td rowspan=1 colspan=1>10 mg/mL</td><td rowspan=1 colspan=1>8/8</td><td rowspan=1 colspan=1>8/8</td></tr><tr><td rowspan=1 colspan=1>Human Peripheral Blood MononuclearCells (PBMC)</td><td rowspan=1 colspan=1>1 x 103 cells/μL</td><td rowspan=1 colspan=1>8/8</td><td rowspan=1 colspan=1>8/8</td></tr><tr><td rowspan=1 colspan=1>PHNY</td><td rowspan=1 colspan=1>15% (v/v)</td><td rowspan=1 colspan=1>8/8</td><td rowspan=1 colspan=1>8/8</td></tr><tr><td rowspan=1 colspan=1>Remel M4RT</td><td rowspan=1 colspan=1>100% (v/v)</td><td rowspan=1 colspan=1>8/8</td><td rowspan=1 colspan=1>8/8</td></tr><tr><td rowspan=1 colspan=1>Remel M5</td><td rowspan=1 colspan=1>100% (v/v)</td><td rowspan=1 colspan=1>8/8</td><td rowspan=1 colspan=1>8/8</td></tr><tr><td rowspan=1 colspan=1>Saline</td><td rowspan=1 colspan=1>15% (v/v)</td><td rowspan=1 colspan=1>8/8</td><td rowspan=1 colspan=1>8/8</td></tr><tr><td rowspan=1 colspan=1>Snuff</td><td rowspan=1 colspan=1>1% (w/v)</td><td rowspan=1 colspan=1>8/8</td><td rowspan=1 colspan=1>8/8</td></tr><tr><td rowspan=1 colspan=1>Tamiflu</td><td rowspan=1 colspan=1>7.5 mg/mL</td><td rowspan=1 colspan=1>8/8</td><td rowspan=1 colspan=1>8/8</td></tr><tr><td rowspan=1 colspan=1>Tobramycin</td><td rowspan=1 colspan=1>4 μg/mL</td><td rowspan=1 colspan=1>8/8</td><td rowspan=1 colspan=1>8/8</td></tr><tr><td rowspan=1 colspan=1>Zicam</td><td rowspan=1 colspan=1>15% (w/v)</td><td rowspan=1 colspan=1>8/8</td><td rowspan=1 colspan=1>8/8</td></tr><tr><td rowspan=1 colspan=1>Zinc</td><td rowspan=1 colspan=1>0.1 μg/mL</td><td rowspan=1 colspan=1>8/8</td><td rowspan=1 colspan=1>8/8</td></tr></table>

a. One of 8 replicates reported INVALID test result, indicating interference from the substance. The substance was subsequently tested with 8 replicates at half the initial concentration and no interference was observed.

5. Assay Reportable Range: Not applicable; this is a qualitative assay.

6. Traceability, Stability, Expected Values (Controls, Calibrators, or Methods):

a) Assay Controls

Please refer to Instrument Description Information (Section C.5) above for assay controls.

b) Specimen Stability Studies

Room Temperature (15 to $3 0 ^ { \circ } C )$ and Refrigerated (2 to $g \circ C ,$ ) Stability Studies A study was conducted to evaluate the stability of NPS specimens stored in UTM/VTM, followed by testing with the Xpert Xpress $\mathrm { C o V - } 2$ plus test. Clinical NPS specimens collected in UTM/VTM and eNAT, and confirmed SARS-CoV-2 negative were pooled for this study. The positive sample consisted of the NATtrol inactivated SARS-CoV-2 virus strain USA

WA1/2020 (ZeptoMetrix Inc., USA) spiked at $2 . 7 5 \mathrm { x }$ LoD (1108 copies $/ \mathrm { m L }$ ) into the pooled negative clinical NPS in UTM/VTM and into the pooled clinical NPS in eNAT. The same negative clinical matrix pool was used to provide the negative sample. Aliquots of each positive and negative sample were stored at $1 5 \mathrm { { ^ \circ C } }$ and $3 0 \mathrm { { } ^ { \circ } C }$ to cover the extremes of the recommended room temperature storage temperature range, and at $2 \mathrm { ^ { \circ } C }$ and $8 ^ { \circ } \mathrm { C }$ to cover the extremes of the recommended refrigerated storage temperature range. All four storage temperatures were monitored continuously by automated temperature monitoring and access control system. Eight (8) replicates of the positive sample and 4 replicates of the negative sample were tested at $\mathrm { T } { = } 0$ , 4 hours, 8 hours, 24 hours, 48 hours and 50 hours after storage at $1 5 \mathrm { { ^ \circ C } }$ and $3 0 \mathrm { { } ^ { \circ } C }$ . Eight (8) replicates of the positive sample and 4 replicates of the negative sample were tested at $\mathrm { T } { = } 0$ , 1 day, 2 days, 4 days, 6 days, 7 days, and 8 days after storage at $2 \mathrm { ^ { \circ } C }$ and $8 \mathrm { { } ^ { \circ } C }$ .

External positive and negative controls, run daily during the study, reported the expected Xpert Xpress CoV-2 plus test results. Aliquots of the negative and positive specimens stored at $1 5 \mathrm { { ^ \circ C } }$ , $3 0 \mathrm { { } ^ { \circ } C }$ , $2 \mathrm { ^ { \circ } C }$ and $8 \mathrm { { } ^ { \circ } C }$ , and tested at their respective timepoints gave expected results.

The study data supports the following specimen stability claims:

NPS and NS specimens can be stored in UTM/VTM or in eNAT matrix at refrigerated $( 2 ^ { \circ } \mathrm { C } - 8 ^ { \circ } \mathrm { C } )$ temperatures for 7 days and at room temperature ( $1 5 ^ { \circ } \mathrm { C }$ - $3 0 ^ { \circ } \mathrm { C }$ for up to 48 hours until testing is performed on the GeneXpert Instrument Systems.

Freeze/Thaw Testing for Samples Frozen at ${ } _ { - 2 0 ^ { \circ } C }$ or - ${ } _ { - 8 0 ^ { \circ } C }$ in VTM or eNAT A study was performed to determine equivalency of frozen SARS-CoV-2 samples compared to fresh samples when tested with the Xpert Xpress CoV-2 plus test. Positive samples used for the fresh and frozen specimen equivalency study were prepared by spiking the NATtrol SARS-CoV-2 (USA-WA1/2020) strain into pooled negative clinical NPS-UTM/VTM and NPS-eNAT at concentrations of $< 1 \mathbf { x }$ LoD, $1 { - } 3 \mathbf { x }$ LoD and 5x LoD. A negative sample consisting of negative clinical NPS-UTM/VTM and NPS-eNAT matrices were also included in the study. The positive and negative samples were tested fresh, and each sample was subsequently allowed to undergo two freeze-thaw cycles consisting of freezing at $- 8 0 \mathrm { { } ^ { \circ } C }$ for at least 24 hours and thawing at room temperature prior to testing. Thawed samples were kept on ice prior to testing.

The study data supports the following specimen stability claims:   
Frozen clinical NPS-UTM/VTM, clinical NS-UTM/VTM and clinical NPS-eNAT specimens positive for SARS-CoV- 2 can undergo one cycle of freeze-thaw before testing with the Xpert Xpress CoV-2 plus test.

# 7. Analytical Sensitivity/Limit of Detection (LoD):

The LoD of the Xpert Xpress CoV-2 plus test was established using NATtrol inactivated SARS-CoV-2 virus strain USA-WA1/2020 (ZeptoMetrix Inc., USA) diluted into pooled negative clinical NPS matrix and negative clinical NS matrix. Clinical NPS specimens were collected in VTM/UTM and in eNAT, while clinical NS specimens were collected in VTM/UTM only. The LoD is defined as the lowest concentration for SARS-CoV-2 strain at which $9 5 \%$ (19/20) of replicates yield a positive result.

The virus strain was initially estimated in a range finding study at ${ \tt > } 1 0$ concentrations in replicates of 20 per concentration of virus. Range finding was performed with two cartridge lots. Probit analysis was performed on the range finding results to estimate the LoDs, and the estimated LoDs were subsequently confirmed using two lots of Xpert Xpress CoV-2 plus cartridges. For the confirmatory study, testing was also performed in replicates of 20 per cartridge lot. The highest (least sensitive) LoD value for the two lots evaluated in the confirmatory study was reported as the final LoD for the Xpert Xpress CoV-2 plus test. The claimed LoD for SARS-CoV-2 is 403 copies/ mL in Clinical NPS-UTM/VTM (Table 16). An equivalent LoD (403 copies/ mL) was also established for SARS-CoV-2 in Clinical NPSeNAT. The claimed LoD is 462 copies $ { / \mathrm { m L } }$ in Clinical NS-UTM/VTM (Table 17).

Table 16: Results for Verification of SARS-CoV-2 LoD in Clinical NPS-UTM/VTM   

<table><tr><td rowspan=1 colspan=1>Virus Strain</td><td rowspan=1 colspan=1>Reagent Lot</td><td rowspan=1 colspan=1>LoD(copies/mL)</td><td rowspan=1 colspan=1>PositiveResults outof # ValidReplicates</td><td rowspan=1 colspan=1>%Positive</td><td rowspan=1 colspan=1>MeanE Ct</td><td rowspan=1 colspan=1>MeanN2 Ct</td><td rowspan=1 colspan=1>MeanRdRP Ct</td></tr><tr><td rowspan=2 colspan=1>NATtrolSARS-CoV-2</td><td rowspan=1 colspan=1>Reagent Lot 1</td><td rowspan=1 colspan=1>403</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>34.3</td><td rowspan=1 colspan=1>38.3</td><td rowspan=1 colspan=1>36.6</td></tr><tr><td rowspan=1 colspan=1>Reagent Lot 2</td><td rowspan=1 colspan=1>403</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>34.1</td><td rowspan=1 colspan=1>37.7</td><td rowspan=1 colspan=1>36.5</td></tr></table>

Table 17: Results for Verification of SARS-CoV-2 LoD in Clinical NS-UTM/VTM   

<table><tr><td rowspan=1 colspan=1>Virus Strain</td><td rowspan=1 colspan=1>Reagent Lot</td><td rowspan=1 colspan=1>LoD(copies/mL)</td><td rowspan=1 colspan=1>PositiveResults outof # ofValidReplicates</td><td rowspan=1 colspan=1>%Positive</td><td rowspan=1 colspan=1>MeanE Ct</td><td rowspan=1 colspan=1>MeanN2 Ct</td><td rowspan=1 colspan=1>MeanRdRP Ct</td></tr><tr><td rowspan=2 colspan=1>NATtrolSARS-CoV-2</td><td rowspan=1 colspan=1>Reagent Lot 1</td><td rowspan=1 colspan=1>462</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>34.4</td><td rowspan=1 colspan=1>38.9</td><td rowspan=1 colspan=1>36.8</td></tr><tr><td rowspan=1 colspan=1>Reagent Lot 2</td><td rowspan=1 colspan=1>462</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>34.1</td><td rowspan=1 colspan=1>37.6</td><td rowspan=1 colspan=1>36.1</td></tr></table>

The analytical sensitivity of the Xpert Xpress CoV-2 plus test was also estimated by testing WHO First International Standard for SARS-CoV-2 RNA in pooled negative clinical NSUTM/VTM matrix. The study was performed in a random and blinded fashion. The LoD for the First WHO International Standard for SARS-CoV-2 RNA tested in NS-UTM/VTM matrix with the Xpert Xpress CoV-2 plus test was $1 0 0 0 \mathrm { \ : I U / m L }$ (Table 18).

Table 18: LoD Concentration, Positivity Rate and Mean Ct Values for SARS-CoV-2 for the First WHO International Standard for SARS-CoV-2 RNA in Clinical NS-UTM/VTM Matrix   

<table><tr><td rowspan=2 colspan=1>Virus</td><td rowspan=2 colspan=1>LoDConcentration</td><td rowspan=2 colspan=1>SARS-CoV-2 PositiveResults /TestReplicates</td><td rowspan=1 colspan=2>E Target</td><td rowspan=1 colspan=2>N2 Target</td><td rowspan=1 colspan=2>RdRP Target</td></tr><tr><td rowspan=1 colspan=1>PositiveResults / TestReplicates</td><td rowspan=1 colspan=1>MeanCt</td><td rowspan=1 colspan=1>PositiveResults / TestReplicates</td><td rowspan=1 colspan=1>MeanCt</td><td rowspan=1 colspan=1>PositiveResults / TestReplicates</td><td rowspan=1 colspan=1>MeanCt</td></tr><tr><td rowspan=1 colspan=1>1st WHOInternationalStandard forSARS-CoV-2RNA Virus</td><td rowspan=1 colspan=1>1000 IU/mL</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>34.3</td><td rowspan=1 colspan=1>19/20</td><td rowspan=1 colspan=1>39.5</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>36.5</td></tr></table>

# 8. Assay Cut-Off:

The Xpert Xpress CoV-2 plus test detects defined cycle threshold (Ct) ranges for the SARSCoV-2 targets and SPC. The Ct cut-offs are hard coded values and are included as automatic calculations in the assay definition file (ADF) of the Xpert Xpress CoV-2 plus test.

# 9. Carry-Over Contamination Study:

A study was conducted to assess whether the single-use, self-contained Xpert Xpress CoV-2 plus cartridge prevents specimen and amplicon carryover by testing a negative sample immediately after testing of a very high positive sample in the same GeneXpert module. The study utilized two GeneXpert instrument modules; both modules were cleaned prior to starting the study. The negative sample used in this study consisted of simulated NPS/NS matrix and the positive sample consisted of high SARS-CoV-2 virus concentration (inactivated SARS-CoV-2 USAWA1/2020 at $5 \mathrm { x } 1 0 ^ { 4 }$ copies/mL) spiked into negative NPS/NS matrix. The negative sample was tested in a GeneXpert module at the start of the study. Following the initial testing of the negative sample, the high positive sample was processed in the same GeneXpert module immediately followed by another negative sample. This was repeated 20 times in the same module, resulting in 20 positives and 21 negatives for the module. The study was repeated using a second GeneXpert module for a total of 40 positive and 42 negative samples. All 40 positive samples were correctly reported as SARS-CoV-2 POSITIVE and all 42 negative samples were correctly reported as SARS-CoV-2 NEGATIVE with the Xpert Xpress CoV-2 plus test. No specimen or amplicon carry-over contamination was observed in this study.

# B Comparison Studies:

# 1. Method Comparison with Predicate Device:

Not applicable. Please refer to the Clinical Studies Section of this document.

# 2. Matrix Comparison:

The following studies were performed to establish equivalent performance of Xpert Xpress CoV2 plus between (a) clinical nasopharyngeal swab (NPS), clinical anterior nasal swab (NS) and simulated NPS/NS matrices in Universal Transport Medium (UTM)/Viral Transport Medium (VTM), and (b) clinical NPS in UTM/VTM and eNAT.

Aliquots of clinical NPS-UTM/VTM, clinical NS-UTM/VTM, and simulated NPS/NS matrices were screened with the Xpert Xpress CoV-2 plus test and negative clinical samples were pooled. NATtrol inactivated SARS-CoV-2 virus strain USA-WA1/2020 (ZeptoMetrix Inc., USA) was spiked into the negative matrices at 0.05x, 1.5x, 5x, and 10x LoD. Aliquots of the negative and SARS-CoV-2 positive matrices were tested in replicates 10 (except for 30 replicates in case of 1.5x LoD). Expected results were obtained for all replicates tested.

A similar study as described above, except the simulated NPS/NS matrix, was used to establish equivalent performance between clinical NPS-UTM/VTM and clinical NPS-eNAT. Expected results were obtained for all replicates tested.

# Clinical Studies:

The clinical performance of the Xpert Xpress CoV-2 plus test was evaluated in a multi-site, observational and method comparison study in 2022 that included 32 geographically diverse sites in the United States (US). Of the 32 sites, 5 sites participated in specimen collection only, 26 sites performed Xpert testing and specimen collection, and 1 site performed Xpert testing as well as comparator and discrepant testing.

To be enrolled in the study, patients had to be presenting at the study centers showing signs and symptoms of respiratory infection. One nasal (NS) or nasopharyngeal (NPS) swab collected in a randomized and blinded fashion from each patient was tested using the Xpert Xpress CoV-2 plus test side by side with an FDA-cleared molecular respiratory panel that includes SARS-CoV-2. The demographics of study participants are presented below (Table 19). Of the 4047 specimens collected from individuals showing signs and symptoms of respiratory infection, a total of 3075 specimens (869 NPS and 1871 NS) passed all study criteria and were used for performance calculations. The Positive Percent Agreement (PPA) and Negative Percent Agreement of Xpert Xpress CoV-2 plus test with the FDA cleared molecular comparator test are presented in Table 20 below. Based on the specimens that yielded non-determinate results (INVALID, ERROR, and NO RESULT) with Xpert Xpress $\mathrm { C o V - } 2 p l u s$ , the non-determinate rate was $1 . 6 \%$ (32/2029) with NPS and $1 . 4 \%$ (28/2018) with NS specimens.

Table 19: Demographic Information of Participants Symptomatic of Respiratory Infection   

<table><tr><td colspan="4" rowspan="1">Iable T9: Demograpiic Iinormatlon of Participalts Syilpto</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">NPS(N=1879)</td><td colspan="1" rowspan="1">NS(N=1871)</td><td colspan="1" rowspan="1">Overall(N=3750)</td></tr><tr><td colspan="1" rowspan="1">Gender</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Female</td><td colspan="1" rowspan="1">1107(58.9%)</td><td colspan="1" rowspan="1">1171(62.6%)</td><td colspan="1" rowspan="1">2278 (60.7%)</td></tr><tr><td colspan="1" rowspan="1">Male</td><td colspan="1" rowspan="1">772(41.1%)</td><td colspan="1" rowspan="1">700 (37.4%)</td><td colspan="1" rowspan="1">1472 (39.3%)</td></tr><tr><td colspan="1" rowspan="1">Age Group (Years)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">≤5</td><td colspan="1" rowspan="1">8 (0.4%)</td><td colspan="1" rowspan="1">64 (3.4%)</td><td colspan="1" rowspan="1">72 (1.9%)</td></tr><tr><td colspan="1" rowspan="1">6-21</td><td colspan="1" rowspan="1">389(20.7%)</td><td colspan="1" rowspan="1">380 (20.3%)</td><td colspan="1" rowspan="1">769 (20.5%)</td></tr><tr><td colspan="1" rowspan="1">22-59</td><td colspan="1" rowspan="1">1228(65.4%)</td><td colspan="1" rowspan="1">1167(62.4%)</td><td colspan="1" rowspan="1">2395 (63.9%)</td></tr><tr><td colspan="1" rowspan="1">≥60</td><td colspan="1" rowspan="1">254(13.5%)</td><td colspan="1" rowspan="1">260 (13.9%)</td><td colspan="1" rowspan="1">514 (13.7%)</td></tr><tr><td colspan="1" rowspan="1">Race</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">American Indian or AlaskaNative</td><td colspan="1" rowspan="1">4 (0.2%)</td><td colspan="1" rowspan="1">4 (0.2%)</td><td colspan="1" rowspan="1">8 (0.2%)</td></tr><tr><td colspan="1" rowspan="1">Asian</td><td colspan="1" rowspan="1">44 (2.3%)</td><td colspan="1" rowspan="1">44 (2.4%)</td><td colspan="1" rowspan="1">88 (2.3%)</td></tr><tr><td colspan="1" rowspan="1">Black or African American</td><td colspan="1" rowspan="1">525(27.9%)</td><td colspan="1" rowspan="1">524 (28.0%)</td><td colspan="1" rowspan="1">1049 (28.0%)</td></tr><tr><td colspan="1" rowspan="1">White</td><td colspan="1" rowspan="1">1236(65.8%)</td><td colspan="1" rowspan="1">1222(65.3%)</td><td colspan="1" rowspan="1">2458 (65.5%)</td></tr><tr><td colspan="1" rowspan="1">Native Hawaiian or OtherPacific Islander</td><td colspan="1" rowspan="1">4 (0.2%)</td><td colspan="1" rowspan="1">0 (0%)</td><td colspan="1" rowspan="1">4 (0.1%)</td></tr><tr><td colspan="1" rowspan="1">Black or AfricanAmerican, White</td><td colspan="1" rowspan="1">7 (0.4%)</td><td colspan="1" rowspan="1">8 (0.4%)</td><td colspan="1" rowspan="1">15 (0.4%)</td></tr><tr><td colspan="1" rowspan="1">Other Mixed (each N≤5)</td><td colspan="1" rowspan="1">7 (0.4%)</td><td colspan="1" rowspan="1">2 (0.1%)</td><td colspan="1" rowspan="1">9 (0.2%)</td></tr><tr><td colspan="1" rowspan="1">Participant declined toanswer, or unknown</td><td colspan="1" rowspan="1">52 (2.8%)</td><td colspan="1" rowspan="1">67 (3.6%)</td><td colspan="1" rowspan="1">119 (3.2%)</td></tr><tr><td colspan="1" rowspan="1">Ethnicity</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Hispanic</td><td colspan="1" rowspan="1">170 (9.0%)</td><td colspan="1" rowspan="1">165 (8.8%)</td><td colspan="1" rowspan="1">335 (8.9%)</td></tr><tr><td colspan="1" rowspan="1">Non-Hispanic</td><td colspan="1" rowspan="1">1681(89.5%)</td><td colspan="1" rowspan="1">1668(89.2%)</td><td colspan="1" rowspan="1">3349 (89.3%)</td></tr><tr><td colspan="1" rowspan="1">Participant declined toanswer, or unknown</td><td colspan="1" rowspan="1">28 (1.5%)</td><td colspan="1" rowspan="1">38 (2.0%)</td><td colspan="1" rowspan="1">66 (1.8%)</td></tr><tr><td colspan="1" rowspan="1">Specimen Testing</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Fresh</td><td colspan="1" rowspan="1">1852(98.6%)</td><td colspan="1" rowspan="1">1845(98.6%)</td><td colspan="1" rowspan="1">3697 (98.6%)</td></tr><tr><td colspan="1" rowspan="1">Frozen</td><td colspan="1" rowspan="1">27 (1.4%)</td><td colspan="1" rowspan="1">26 (1.4%)</td><td colspan="1" rowspan="1">53 (1.4%)</td></tr><tr><td colspan="1" rowspan="1">TestingEnvironment</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Laboratory/NPT</td><td colspan="1" rowspan="1">996(53.0%)</td><td colspan="1" rowspan="1">971 (51.9%)</td><td colspan="1" rowspan="1">1967 (52.5%)</td></tr><tr><td colspan="1" rowspan="1">CW</td><td colspan="1" rowspan="1">883(47.0%)</td><td colspan="1" rowspan="1">900 (48.1%)</td><td colspan="1" rowspan="1">1783 (47.5%)</td></tr><tr><td colspan="1" rowspan="1">Vaccine Status</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Vaccinated</td><td colspan="1" rowspan="1">1336(71.1%)</td><td colspan="1" rowspan="1">1344(71.8%)</td><td colspan="1" rowspan="1">2680 (71.5%)</td></tr><tr><td colspan="1" rowspan="1">Not Vaccinated</td><td colspan="1" rowspan="1">523(27.8%)</td><td colspan="1" rowspan="1">508 (27.2%)</td><td colspan="1" rowspan="1">1031 (27.5%)</td></tr><tr><td colspan="1" rowspan="1">Unknown</td><td colspan="1" rowspan="1">20 (1.1%)</td><td colspan="1" rowspan="1">19 (1.0%)</td><td colspan="1" rowspan="1">39 (1.0%)</td></tr></table>

Table 20: Performance of Xpert Xpress CoV-2 plus in Symptomatic Individuals (Nasal and Nasopharyngeal Swabs Combined)   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=3>Comparator Result</td></tr><tr><td rowspan=1 colspan=1>Xpert Xpress CoV-2 plus</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>574</td><td rowspan=1 colspan=1>55</td><td rowspan=1 colspan=1>629</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>3110</td><td rowspan=1 colspan=1>3121</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>585</td><td rowspan=1 colspan=1>3165</td><td rowspan=1 colspan=1>3750</td></tr><tr><td rowspan=1 colspan=1>PPA (95%CI)</td><td rowspan=1 colspan=3>98.1% (96.7% - 98.9%)</td></tr><tr><td rowspan=1 colspan=1>NPA (95% CI)</td><td rowspan=1 colspan=3>98.3% (97.7% - 98.7%)</td></tr></table>

# C Clinical Cut-Off:

Not applicable.

# D Expected Values/Reference Range:

Expected values as determined by Xpert Xpress CoV-2 plus are presented for nasopharyngeal swab (NPS) and anterior nasal swab (NS) specimen types (Table 21).

Table 21: Positivity Rate by Specimen Type   

<table><tr><td colspan="2">Specimen Type</td></tr><tr><td>NPS</td><td>NS</td></tr><tr><td>17%</td><td>16.5%</td></tr><tr><td>(320/1879)</td><td>(309/1871)</td></tr></table>

# VIII Proposed Labeling:

The labeling supports the finding of substantial equivalence for this device.

# IX Conclusion:

The submitted information in this premarket notification is complete and supports a substantial equivalence decision.