# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATIONDECISION SUMMARYASSAY AND INSTRUMENT

I Background Information:

A 510(k) Number K231329   
B Applicant Hologic, Inc.   
C Proprietary and Established Names Aptima Neisseria gonorrhoeae Assay   
D Regulatory Information

<table><tr><td rowspan=1 colspan=1>ProductCode(s)</td><td rowspan=1 colspan=1>Classification</td><td rowspan=1 colspan=1>RegulationSection</td><td rowspan=1 colspan=1>Panel</td></tr><tr><td rowspan=1 colspan=1>LSL</td><td rowspan=1 colspan=1>Class II</td><td rowspan=1 colspan=1>21 CFR 866.3390 -Neisseria Spp. DirectSerological Test Reagents</td><td rowspan=1 colspan=1>MI - Microbiology</td></tr><tr><td rowspan=1 colspan=1>QEP</td><td rowspan=1 colspan=1>Class II</td><td rowspan=1 colspan=1>21 CFR 866.3393 - Deviceto detect nucleic acidsfrom non-viralmicroorganism(s) causingsexually transmittedinfections and associatedresistance marker(s)</td><td rowspan=1 colspan=1>MI - Microbiology</td></tr></table>

# II Submission/Device Overview:

# A Purpose for Submission:

This submission is a Traditional 510(k) to obtain clearance for the Aptima Neisseria gonorrhoeae Assay on the Panther System. The Aptima Neisseria gonorrhoeae Assay (K043144, K063664) was previously cleared for use with the Tigris DTS platform for endocervical and male urethral swabs, female and male urine, clinician- and patient-collected (asymptomatic only) vaginal swabs, and PreservCyt liquid Pap specimens. With this submission, Hologic seeks clearance for the Aptima Neisseria gonorrhoeae Assay on the Panther using male urine specimens.

B Measurand: Neisseria gonorrhoeae ribosomal RNA   
C Type of Test: Target-mediated amplification nucleic acid probe test

III Intended Use/Indications for Use:

A Intended Use(s): See Indications for Use below.

# B Indication(s) for Use:

The Aptima Neisseria gonorrhoeae (GC) assay is an in vitro qualitative nucleic acid amplification (NAAT) test for the detection of ribosomal RNA (rRNA) from Neisseria gonorrhoeae (GC) to aid in the diagnosis of gonococcal urogenital disease using the Panther System. The assay may be used to test male urine specimens from symptomatic and asymptomatic individuals.

C Special Conditions for Use Statement(s): Rx - For Prescription Use Only

D Special Instrument Requirements: Panther System

# IV Device/System Characteristics:

# A Device Description:

The Aptima Neisseria gonorrhoeae Assay is a nucleic acid amplification test that utilizes the target capture (TC), transcription-mediated amplification (TMA) and hybridization protection assay (HPA) technologies for the qualitative detection of Neisseria gonorrhoeae. Male urine is collected and transferred using the collection kit. The transport solution in the kit tube releases the rRNA target that are then captured by magnetic microparticles. A unique set of primers is used to amplify the target and the amplicon is detected by nucleic acid hybridization. Assay test results are automatically interpreted by the Panther System Aptima GC assay software. This assay is similar to the Aptima Assay for Neisseria gonorrhoeae (K063664) in that the target organism is Neisseria gonorrhoeae. The primary differences between the two assays are the platform on which the assay is performed. There are no modifications to assay reagents.

# B Principle of Operation:

The Aptima Neisseria gonorrhoeae Assay combines the technologies of target capture, TMA, and HPA. Male urine is collected and transferred using the collection kit. The transport solution in these tubes releases the rRNA target and protects it from degradation during storage. The target rRNA molecule is isolated from the specimens by use of a capture oligomer via target capture that utilizes magnetic microparticles. The capture oligomer contains a sequence complementary to a specific region of the target molecule, as well as a string of deoxyadenosine residues. During the hybridization step, the sequence specific region of the capture oligomer binds to a specific region of the target molecule. The capture oligomer: target complex is then captured out of solution by decreasing the temperature of the reaction to room temperature. This temperature reduction allows hybridization to occur between the deoxyadenosine region on the capture oligomer and the polydeoxythymidine molecules that are covalently attached to the magnetic particles. The microparticles, including the captured target molecule bound to them, are pulled to the side of the reaction vessel using magnets and the supernatant is aspirated. The particles are washed to remove residual specimen matrix that may contain amplification reaction inhibitors. After the target capture steps are completed, the specimens are ready for amplification. Target amplification assays are based on the ability of complementary oligonucleotide primers to specifically anneal and allow enzymatic amplification of the target nucleic acid strands. The Hologic TMA reaction replicates a specific region of the 16S rRNA from GC via DNA intermediates. A unique set of primers is used for the target molecule. Detection of the rRNA amplification product sequences (amplicon) is achieved using nucleic acid hybridization. A single-stranded chemiluminescent DNA probe, which is complementary to a region of the target amplicon, is labeled with an acridinium ester molecule. The labeled DNA probe combines with amplicon to form stable RNA:DNA hybrids. The Selection Reagent differentiates hybridized from unhybridized probe, eliminating the generation of signal from unhybridized probe. During the detection step, light emitted from the labeled RNA:DNA hybrids is measured as photon signals in a luminometer and are reported as Relative Light Units (RLU).

# C Instrument Description Information:

1. Instrument Name: Panther System

2. Specimen Identification: By handheld barcode reader and manual entry.

3. Specimen Sampling and Handling: Fully automated.

4. Calibration: The Panther System undergoes preventative maintenance every 12 months, which includes luminometer calibration. The Aptima Neisseria gonorrhoeae Assay requires no calibration.

5. Quality Control: The Aptima Neisseria gonorrhoeae Assay Controls Kit includes 5 vials each of Positive and Negative Controls which are ready to use.

# V Substantial Equivalence Information:

A Predicate Device Name(s): GEN-PROBE APTIMA Assay for Neisseria gonorrhoeae   
B Predicate 510(k) Number(s): K063664

C Comparison with Predicate(s):   

<table><tr><td colspan="1" rowspan="1">Device &amp; PredicateDevice(s):</td><td colspan="1" rowspan="1">K231329</td><td colspan="1" rowspan="1">K063664</td></tr><tr><td colspan="1" rowspan="1">Device Trade Name</td><td colspan="1" rowspan="1">Aptima Neisseria gonorrhoeaeAssay (Panther)</td><td colspan="1" rowspan="1">GEN-PROBE APTIMA Assay forNeisseria gonorrhoeae</td></tr><tr><td colspan="1" rowspan="1">General DeviceCharacteristicSimilarities</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">IntendedUse/Indications ForUse</td><td colspan="1" rowspan="1">The Aptima Neisseriagonorrhoeae (GC) assay is anin vitro qualitative nucleicacid amplification (NAAT)test for the detection ofribosomal RNA (rRNA) fromNeisseria gonorrhoeae (GC)to aid in the diagnosis ofgonococcal urogenital diseaseusing the Panther System. Theassay may be used to test maleurine specimens fromsymptomatic andasymptomatic individuals.</td><td colspan="1" rowspan="1">The Aptima Neisseria gonorrhoeaeassay is a target amplification nucleicacid probe test that utilizes targetcapture for the in vitro qualitativedetection of ribosomal RNA (rRNA)from Neisseria gonorrhoeae (GC) toaid in the diagnosis of gonococcalurogenital disease using the TigrisDTS System or semi-automatedinstrumentation as specified. Theassay may be used to test thefollowing specimens fromsymptomatic individuals: clinician-collected endocervical, vaginal andmale urethral swab specimens; andfemale and male urine specimens. Theassay may be used to testthe following specimens fromasymptomatic individuals: clinician-collected endocervical and vaginalswab specimens, patient-collectedvaginal swab specimens1; and femaleand male urine specimens. The assayis also intended for use with the testingof gynecological specimens, fromboth symptomatic and asymptomaticpatients, collected in the PreservCytsolution.Patient-collected vaginal swabspecimens are an option for screeningwomen when otherwise indicated. TheAptima Multitest Swab SpecimenCollection Kit is not for home use.</td></tr><tr><td colspan="1" rowspan="1">Technology Principleof Operation</td><td colspan="1" rowspan="1">Target Capture (TC),Transcription-MediatedAmplification (TMA),</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Hybridization ProtectionAssay (HPA)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Assay Targets</td><td colspan="1" rowspan="1">Neisseria gonorrhoeae rRNA</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Assay Results</td><td colspan="1" rowspan="1">Qualitative</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Function</td><td colspan="1" rowspan="1">Detection of rRNA fromNeisseria gonorrhoeae</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">General DeviceCharacteristicDifferences</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Platform</td><td colspan="1" rowspan="1">Automated Panther System</td><td colspan="1" rowspan="1">Automated Tigris System</td></tr><tr><td colspan="1" rowspan="1">Specimen Types</td><td colspan="1" rowspan="1">Male urine specimens</td><td colspan="1" rowspan="1">Female specimens:Clinician-collected vaginal swab•Patient-collected vaginal swab(asymptomatic only)Endocervical swab•Gynecological specimens inPreservCyt solutionUrineMale Specimens:UrineUrethral</td></tr></table>

# VI Standards/Guidance Documents Referenced:

• CLSI EP05-A3 (Reaffirmed: September 2019) 7-251 Evaluation of Precision of Qualitative Measurement Procedures; Approved Guideline – Third Edition CLSI EP07 3rd Edition 7-275 Interference Testing in Clinical Chemistry CLSI EP12-A2 7-152 User Protocol of Evaluation of Qualitative Test Performance; Approved Guideline – Second Edition CLSI EP15-A3 7-253 User Verification of Precision and Estimation of Bias; Approved Guideline – Second Edition   
• CLSI EP17-A2 7-233 Evaluation of Detection Capability for Clinical Laboratory Measurement Procedures; Approved Guideline – Second Edition   
• CLSI EP25-A (Replaces EP25-P) 7-235 Evaluation of Stability of In Vitro Diagnostic Reagents; Approved Guideline CLSI MM03-3rd Edition (Replaces MM03-A2) 7-260 Molecular Diagnostic Methods for Infectious Diseases   
• CLSI MM13-2nd Edition 7-300 Collection Transport Preparation and Storage of Specimens for Molecular Methods

• IEC 62304 Edition 1.1 2015-06 Consolidated Version 13-79. Medical device software – Software life cycle processes ISO 14971 Third Edition 2019-12 5-125 Medical devices – Application of risk management to medical devices Interference Testing in Clinical Chemistry. IEC 60601-1-2 Edition 4.0 2014-02 19-8 Medical electrical equipment – Part 1-2: General requirements for basic safety and essential performance – Collateral Standard: Electromagnetic disturbances – Requirements and tests.

# VII Performance Characteristics (if/when applicable):

A Analytical Performance:

# 1. Reproducibility Study

Reproducibility study was done using three panels, one negative and two GC positive members. The positive samples were clinical samples selected based on the RLU values of the result interpretation range. The reproducibility study was conducted in three sites (2 external and one internal) on TIGRIS and Panther system by two operators at each site. For each instrument at each site each operator performed 2 runs per day over 3 days for a total of 12 runs performed over a period of 6 days. At each of three sites approximately 108 samples were tested on each instrument. Three hundred twenty-four samples were tested on each instrument across the sites. The percent agreement with expected results for all panels was $100 \%$ . The summary of the results is shown in Table 1 below.

Table 1: Reproducibility on the Panther system.   

<table><tr><td rowspan=2 colspan=1>Panelmember</td><td rowspan=2 colspan=1>N</td><td rowspan=2 colspan=1>%Agreement withExpected results</td><td rowspan=2 colspan=1>MeanRLU</td><td rowspan=1 colspan=2>Betweensites</td><td rowspan=1 colspan=2>Betweenoperators</td><td rowspan=1 colspan=2>Betweenlots</td><td rowspan=1 colspan=2>Betweenruns</td><td rowspan=1 colspan=2>Withinruns</td><td rowspan=1 colspan=2>Total</td></tr><tr><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>CV%</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>CV%</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>CV%</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>CV%</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>CV%</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>CV%</td></tr><tr><td rowspan=1 colspan=1>GCnegative</td><td rowspan=1 colspan=1>108</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>2.1</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>1.4</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>11.9</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>12.0</td></tr><tr><td rowspan=1 colspan=1>GCLowpositive</td><td rowspan=1 colspan=1>107</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>6196.9</td><td rowspan=1 colspan=1>247.5</td><td rowspan=1 colspan=1>4.0</td><td rowspan=1 colspan=1>237</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>315.1</td><td rowspan=1 colspan=1>5.1</td><td rowspan=1 colspan=1>136.5</td><td rowspan=1 colspan=1>2.2</td><td rowspan=1 colspan=1>187.4</td><td rowspan=1 colspan=1>3.0</td><td rowspan=1 colspan=1>463.5</td><td rowspan=1 colspan=1>7.5</td></tr><tr><td rowspan=1 colspan=1>GCPositive</td><td rowspan=1 colspan=1>105</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>926.1</td><td rowspan=1 colspan=1>81.2</td><td rowspan=1 colspan=1>8.8</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>98.4</td><td rowspan=1 colspan=1>10.6</td><td rowspan=1 colspan=1>86.2</td><td rowspan=1 colspan=1>9.3</td><td rowspan=1 colspan=1>361.5</td><td rowspan=1 colspan=1>39.0</td><td rowspan=1 colspan=1>392.9</td><td rowspan=1 colspan=1>42.4</td></tr></table>

2. Precision/Reproducibility:

# Within-Laboratory Precision

Within lab precision study using positive panel members consisting of Neisseria gonorrheae cells spiked into pooled negative urine matrix at three different concentrations: 3x LoD (Low Positive), ${ \ > } 3 \mathbf { x }$ and $- 5 \mathbf { x }$ LoD (Moderate Positive), and ${ \displaystyle \mathrm {  ~ \ p { \bf 1 0 x } ~ } }$ LoD (High positive). The negative panel member consisted of unspiked negative urine matrix. Testing was conducted over the course of at least 20 non-consecutive days using two lots of reagents on three Panther systems by three operators performing at least two daily runs. Agreement with

expected results was $100 \%$ for all four panel members (Table 2). The summary of the result is shown in Table 3 below.

# Table 2: Within-Laboratory Precision on the Panther system, Percent Agreement with Expected Results

Table 3: Within-Laboratory Precision on the Panther system based on RLU values   

<table><tr><td rowspan=2 colspan=1>Panel member</td><td rowspan=2 colspan=1>N</td><td rowspan=2 colspan=1>MeankRLU</td><td rowspan=1 colspan=2>BetweenLot</td><td rowspan=1 colspan=2>Betweeninstrument</td><td rowspan=1 colspan=2>Betweenoperator</td><td rowspan=1 colspan=2>BetweenDay</td><td rowspan=1 colspan=2>Betweenrun</td><td rowspan=1 colspan=2>WithinRun</td><td rowspan=1 colspan=2>Total</td></tr><tr><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>CV(%)</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>CV(%)</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>CV(%)</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>CV(%)</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>CV(%)</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>CV(%)</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>CV(%)</td></tr><tr><td rowspan=1 colspan=1>N.gonorrhoeaeLow Positive</td><td rowspan=1 colspan=1>162</td><td rowspan=1 colspan=1>520.1</td><td rowspan=1 colspan=1>65.05</td><td rowspan=1 colspan=1>12.51</td><td rowspan=1 colspan=1>38.39</td><td rowspan=1 colspan=1>7.38</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>14.84</td><td rowspan=1 colspan=1>2.85</td><td rowspan=1 colspan=1>20.11</td><td rowspan=1 colspan=1>3.87</td><td rowspan=1 colspan=1>62.34</td><td rowspan=1 colspan=1>3.87</td><td rowspan=1 colspan=1>101.07</td><td rowspan=1 colspan=1>19.43</td></tr><tr><td rowspan=1 colspan=1>N.gonorrhoeae,moderatepositive</td><td rowspan=1 colspan=1>162</td><td rowspan=1 colspan=1>809.4</td><td rowspan=1 colspan=1>90.32</td><td rowspan=1 colspan=1>11.16</td><td rowspan=1 colspan=1>52.74</td><td rowspan=1 colspan=1>6.52</td><td rowspan=1 colspan=1>7.71</td><td rowspan=1 colspan=1>0.95</td><td rowspan=1 colspan=1>15.00</td><td rowspan=1 colspan=1>1.85</td><td rowspan=1 colspan=1>24.09</td><td rowspan=1 colspan=1>2.98</td><td rowspan=1 colspan=1>84.36</td><td rowspan=1 colspan=1>2.98</td><td rowspan=1 colspan=1>137.55</td><td rowspan=1 colspan=1>16.99</td></tr><tr><td rowspan=1 colspan=1>N.gonorrhoeae,High Positive</td><td rowspan=1 colspan=1>162</td><td rowspan=1 colspan=1>1529.4</td><td rowspan=1 colspan=1>182.18</td><td rowspan=1 colspan=1>11.91</td><td rowspan=1 colspan=1>78.41</td><td rowspan=1 colspan=1>5.13</td><td rowspan=1 colspan=1>19.09</td><td rowspan=1 colspan=1>1.25</td><td rowspan=1 colspan=1>51.45</td><td rowspan=1 colspan=1>3.36</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>111.09</td><td rowspan=1 colspan=1>7.26</td><td rowspan=1 colspan=1>233.86</td><td rowspan=1 colspan=1>15.29</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>162</td><td rowspan=1 colspan=1>4.2</td><td rowspan=1 colspan=1>1.19</td><td rowspan=1 colspan=1>28.36</td><td rowspan=1 colspan=1>0.87</td><td rowspan=1 colspan=1>20.80</td><td rowspan=1 colspan=1>0.51</td><td rowspan=1 colspan=1>12.28</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>3.52</td><td rowspan=1 colspan=1>84.17</td><td rowspan=1 colspan=1>3.85</td><td rowspan=1 colspan=1>92.04</td></tr></table>

<table><tr><td rowspan=1 colspan=1>Panel member</td><td rowspan=1 colspan=1>Relativetargetconcentration(xLoD)</td><td rowspan=1 colspan=1>N.gonorrhoeaeconcentration(CFU/mL)</td><td rowspan=1 colspan=1>N pos/Ntested</td><td rowspan=1 colspan=1>%positive</td><td rowspan=1 colspan=1>% Agreement(95% CI)</td></tr><tr><td rowspan=1 colspan=1>N.gonorrhoeae,Low Positive</td><td rowspan=1 colspan=1>~3</td><td rowspan=1 colspan=1>0.1431</td><td rowspan=1 colspan=1>162/162</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100% (97.7-100)</td></tr><tr><td rowspan=1 colspan=1>N.gonorrhoeae,Moderate Positive</td><td rowspan=1 colspan=1>~54.9</td><td rowspan=1 colspan=1>0.2417</td><td rowspan=1 colspan=1>162/162</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100% (97.7-100)</td></tr><tr><td rowspan=1 colspan=1>N.gonorrhoeae,High Positive</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>0.5426</td><td rowspan=1 colspan=1>162/162</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100% (97.7-100)</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0/162</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>100% (97.7-100)</td></tr></table>

3. Linearity: N/A

# 4. Analytical Specificity/Interference:

The Analytical Specificity/Interference study was evaluated in K063664. The list of the organisms tested are listed below in Table 4.

Table 4: Analytical Specificity   

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Organism</td></tr><tr><td rowspan=1 colspan=1>Achromobacter xerosis</td><td rowspan=1 colspan=1>Escherichia coli</td><td rowspan=1 colspan=1>Neisseria mucosa</td></tr><tr><td rowspan=1 colspan=1>Acinetobacter calcoaceticus</td><td rowspan=1 colspan=1>Flavobacterium meningosepticum</td><td rowspan=1 colspan=1>Neisseria sicca</td></tr><tr><td rowspan=1 colspan=1>Acinetobacter Iwoffi</td><td rowspan=1 colspan=1>Fusobacterium nucleatum</td><td rowspan=1 colspan=1>Neisseria subflava</td></tr><tr><td rowspan=1 colspan=1>Actinomyces israelii</td><td rowspan=1 colspan=1>Gardnerella vaginalis</td><td rowspan=1 colspan=1>Neisseria perflava</td></tr><tr><td rowspan=1 colspan=1>Actinomyces pyogenes</td><td rowspan=1 colspan=1>Gemella haemolysans</td><td rowspan=1 colspan=1>Neisseria polysaccharea</td></tr><tr><td rowspan=1 colspan=1>Aerococcus viridans</td><td rowspan=1 colspan=1>Haemophilus ducreyi</td><td rowspan=1 colspan=1>Paracoccus denitrificans</td></tr><tr><td rowspan=1 colspan=1>Aeromonas hydrophila</td><td rowspan=1 colspan=1>Haemophilus influenzae</td><td rowspan=1 colspan=1>Peptostreptococcus anaerobius</td></tr><tr><td rowspan=1 colspan=1>Agrobacterium radiobacter</td><td rowspan=1 colspan=1>Herpes simplex virus I</td><td rowspan=1 colspan=1>Peptostreptococcus productus</td></tr><tr><td rowspan=1 colspan=1>Alcaligenes faecalis</td><td rowspan=1 colspan=1>Herpes simplex virus II</td><td rowspan=1 colspan=1>Plesiomonas shigelloides</td></tr><tr><td rowspan=1 colspan=1>Bacillus subtilis</td><td rowspan=1 colspan=1>Human papilloma virus 16</td><td rowspan=1 colspan=1>Propionibacterium acnes</td></tr><tr><td rowspan=1 colspan=1>Bacteriodes fragilis</td><td rowspan=1 colspan=1>Kingella dentrificans</td><td rowspan=1 colspan=1>Proteus mirabilis</td></tr><tr><td rowspan=1 colspan=1>Bacteriodes ureolyticus</td><td rowspan=1 colspan=1>Kingella kingae</td><td rowspan=1 colspan=1>Proteus vulgaris</td></tr><tr><td rowspan=1 colspan=1>Bifidobacterium adolescentis</td><td rowspan=1 colspan=1>Klebsiella oxytoca</td><td rowspan=1 colspan=1>Providencia stuartii</td></tr><tr><td rowspan=1 colspan=1>Bifidobacterium brevi</td><td rowspan=1 colspan=1>Klebsiella pneumoniae</td><td rowspan=1 colspan=1>Pseudomonas aeruginosa</td></tr><tr><td rowspan=1 colspan=1>Branhamella catarrhalis</td><td rowspan=1 colspan=1>Lactobacillus acidophilus</td><td rowspan=1 colspan=1>Pseudomonas fluorescens</td></tr><tr><td rowspan=1 colspan=1>Brevibacterium linens</td><td rowspan=1 colspan=1>Lactobacillus brevis</td><td rowspan=1 colspan=1>Pseudomonas putida</td></tr><tr><td rowspan=1 colspan=1>Campylobacter jejuni</td><td rowspan=1 colspan=1>Lactobacillus jensonii</td><td rowspan=1 colspan=1>Rahnella aquatilis</td></tr><tr><td rowspan=1 colspan=1>Candida albicans</td><td rowspan=1 colspan=1>Lactobacillus lactis</td><td rowspan=1 colspan=1>Rhodospirillum rubrum</td></tr><tr><td rowspan=1 colspan=1>Candida glabrata</td><td rowspan=1 colspan=1>Legionella pneumophila</td><td rowspan=1 colspan=1>Saccharomyces cerevisiae</td></tr><tr><td rowspan=1 colspan=1>Candida parapsilosis</td><td rowspan=1 colspan=1>Leuconostoc paramensenteroides</td><td rowspan=1 colspan=1>Salmonella minnesota</td></tr><tr><td rowspan=1 colspan=1>Candida tropicalis</td><td rowspan=1 colspan=1>Listeria monocytogenes</td><td rowspan=1 colspan=1>Salmonella typhimurium</td></tr><tr><td rowspan=1 colspan=1>Chlamydia pneumoniae</td><td rowspan=1 colspan=1>Micrococcus luteus</td><td rowspan=1 colspan=1>Serratia marcescens</td></tr><tr><td rowspan=1 colspan=1>Chlamydia psittaci</td><td rowspan=1 colspan=1>Moraxella lacunata</td><td rowspan=1 colspan=1>Staphylococcus saprophyticus</td></tr><tr><td rowspan=1 colspan=1>Chlamydia trachomatis</td><td rowspan=1 colspan=1>EY</td><td rowspan=1 colspan=1>EY</td></tr><tr><td rowspan=1 colspan=1>Chromobacterium violaceum</td><td rowspan=1 colspan=1>Moraxella osloensis</td><td rowspan=1 colspan=1>Staphylococcus aureus</td></tr><tr><td rowspan=1 colspan=1>Citrobacter freundii</td><td rowspan=1 colspan=1>Morganella morganii</td><td rowspan=1 colspan=1>Staphylococcus epidermidis</td></tr><tr><td rowspan=1 colspan=1>Clostridium perfringens</td><td rowspan=1 colspan=1>Mycobacterium smegmatis</td><td rowspan=1 colspan=1>Streptococcus agalactiae</td></tr><tr><td rowspan=1 colspan=1>Corynebacterium genitalium</td><td rowspan=1 colspan=1>Mycoplasma genitalium</td><td rowspan=1 colspan=1>Streptococcus bovis</td></tr><tr><td rowspan=1 colspan=1>Corynebacterium xerosis</td><td rowspan=1 colspan=1>Mycoplasma hominis</td><td rowspan=1 colspan=1>Streptococcus mitis</td></tr><tr><td rowspan=1 colspan=1>Cryptococcus neoformans</td><td rowspan=1 colspan=1>N. meningitidis Serogroup A</td><td rowspan=1 colspan=1>Streptococcus mutans</td></tr><tr><td rowspan=1 colspan=1>Cytomegalovirus</td><td rowspan=1 colspan=1>N. meningitidis Serogroup B</td><td rowspan=1 colspan=1>Streptococcus pneumoniae</td></tr><tr><td rowspan=1 colspan=1>Deinococcus radiodurans</td><td rowspan=1 colspan=1>N. meningitidis Serogroup C</td><td rowspan=1 colspan=1>Streptococcus pyogenes</td></tr><tr><td rowspan=1 colspan=1>Derxia gummosa</td><td rowspan=1 colspan=1>N. meningitidis Serogroup D</td><td rowspan=1 colspan=1>Streptococcus salivarius</td></tr><tr><td rowspan=1 colspan=1>Eikenella corrodens</td><td rowspan=1 colspan=1>N. meningitidis Serogroup Y</td><td rowspan=1 colspan=1>Streptococcus sanguis</td></tr><tr><td rowspan=1 colspan=1>Enterobacter aerogenes</td><td rowspan=1 colspan=1>N. meningitidis Serogroup W135</td><td rowspan=1 colspan=1>Streptomyces griseinus</td></tr><tr><td rowspan=1 colspan=1>Enterobacter cloacae</td><td rowspan=1 colspan=1>Neisseria cinerea</td><td rowspan=1 colspan=1>Trichomonas vaginalis</td></tr><tr><td rowspan=1 colspan=1>Entercoccus avium</td><td rowspan=1 colspan=1>Neisseria dentrificans</td><td rowspan=1 colspan=1>Ureaplasma urealyticum</td></tr><tr><td rowspan=1 colspan=1>Entercoccus faecalis</td><td rowspan=1 colspan=1>Neisseria elongata</td><td rowspan=1 colspan=1>Vibrio parahaemolyticus</td></tr><tr><td rowspan=1 colspan=1>Entercoccus faecium</td><td rowspan=1 colspan=1>Neisseria flava</td><td rowspan=1 colspan=1>Yersinia enterocolitica</td></tr><tr><td rowspan=1 colspan=1>Erwinia herbicola</td><td rowspan=1 colspan=1>Neisseria flavescens</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Erysipelothrix rhusiopathiae</td><td rowspan=1 colspan=1>Neisseria lactamica</td><td rowspan=1 colspan=1>EY</td></tr></table>

# Interference

The following interfering substances were individually spiked into urine specimens: $10 \%$ blood, contraceptive jelly, spermicide, moisturizer, hemorrhoidal anesthetic, body oil, powder, anti-fungal cream, vaginal lubricants, feminine spray and leukocytes ( $1 . 0 \mathrm { ~ x ~ } 1 0 6$ cells/mL). The following interfering substances were individually spiked into urine specimens: $30 \%$ blood, urine analytes, protein, glucose, ketones, bilirubin, nitrate,

urobilinogen, pH 4 (acidic), $\mathsf { p H } 9$ (alkaline), leukocytes $\mathrm { 1 . 0 \times 1 0 6 ~ c e l l s / m L }$ ), cellular debris, vitamins, minerals, acetaminophen, aspirin and ibuprofen. Fresh blood was added to clinical pools of urine specimens and then tested for potential assay interference in the presence and absence of GC target. No interference was observed with any of the tested substances.

Additional testing was performed for this submission to evaluate the effect of seminal fluid, mucin and Chlamydia trachomatis (CT) on the performance of the Aptima GC Assay when present in urine specimens. The two potentially interfering substances and organism (CT) were tested separately in the presence or absence of the analyte. Neisseria gonorrhoeae (ATCC 49226) was added to negative male urine matrix to be tested at a final concentration of 3x LoD. Each substance was tested in three replicates. All tests returned expected positive and negative results at the interferent concentrations shown below.

Table 5: Interference study with male urine   

<table><tr><td rowspan=1 colspan=1>Interferent</td><td rowspan=1 colspan=1>Concentration</td></tr><tr><td rowspan=1 colspan=1>Seminal Fluid</td><td rowspan=1 colspan=1>1.0% (vol/vol)</td></tr><tr><td rowspan=1 colspan=1>Mucin</td><td rowspan=1 colspan=1>1.0% (wt./vol)</td></tr><tr><td rowspan=1 colspan=1>C. trachomatis</td><td rowspan=1 colspan=1>1x106 IFU/mL</td></tr></table>

5. Assay Reportable Range:

N/A. This is a qualitative assay.

6. Traceability, Stability, (Controls, Calibrators, or Methods):

Specimen Stability

The specimen stability study was accepted in previous submission (K043144) and is applicable for this submission. Male urine specimen collected and transferred to Aptima urine specimen transport tube is stable at $2 \mathrm { { } ^ { \circ } C }$ to $3 0 \mathrm { { } ^ { \circ } C }$ for 30 days for testing. If longer storage is required urine sample in the transport tube can be stored at $- 2 0 ^ { \circ } \mathrm { C }$ to $- 7 0 \mathrm { { } ^ { \circ } C }$ for 12 months after collection.

# 7. Detection Limit:

The limit of detection (LoD) was tested and confirmed with dilution panels prepared using two strains of GC organisms spiked into pooled negative urine. Testing evaluated one antibiotic susceptible strain, Neisseria gonorrhoeae ATCC 49226 (GP1803), and one antibiotic resistant stain, Neisseria gonorrhoeae WHO X/NCTC 13820 (GP2730). The dilution panels were tested on three Panther instruments with two reagent lots. At least 20 replicates were run for each concentration for each reagent lot for each strain. The LoD for urine specimens, defined as the target concentration that can be detected in $9 5 \%$ of the replicates tested, was determined to be 0.04933 CFU/mL for ATCC 49226 and 0.03986 CFU/mL for stain X/NCTC 13820.

8. Assay Cut-Off:

Assay test results are automatically interpreted by the Aptima assay software. A test results may be negative, equivocal, positive, or invalid as determined by the toral RLU in the detection step as shown in Table 7.

Table 7: Result Interpretation   

<table><tr><td rowspan=1 colspan=1>Test Interpretation</td><td rowspan=1 colspan=1>Total RLU (x1000)</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>0* to &lt; 50</td></tr><tr><td rowspan=1 colspan=1>Equivocal</td><td rowspan=1 colspan=1>50 to &lt; 100</td></tr><tr><td rowspan=1 colspan=1>LowRLU positive</td><td rowspan=1 colspan=1>100 to &lt; 2,000</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>2,000 to &lt; 12,000</td></tr><tr><td rowspan=1 colspan=1>Invalid</td><td rowspan=1 colspan=1>0* or &gt; 12,000</td></tr></table>

$^ { * } \mathrm { A }$ zero $( 0 \mathrm { ~ x ~ } 1 0 0 0 )$ RLU result on the run report represents a value between zero and 999 RLU. RLU values less than 690 on the Panther System will be reported as invalid. $1 \ \mathrm { I n }$ the low positive range, data suggest positive results should be interpreted carefully, with the understanding that the likelihood of a false positive may be higher than a true positive.

9. Accuracy (Instrument): Not Applicable

10. Carry-Over:

A multi-run analytical study was conducted using spiked panels on three Panther systems. Carryover was assessed using approximately $20 \%$ high titer GC samples $( > 2 \mathrm { x } 1 0 ^ { 5 }$ cells/mL or rRNA equivalent) dispersed between negative samples. Testing was carried out using five (5) runs on each of three Panther systems with a total of 2941 negative samples. The overall carryover rate was $0 . 0 7 \%$ with a $9 5 \%$ confidence interval of $0 . 0 2 \substack { - 0 . 2 5 \% }$ .

# B Comparison Studies:

1. Method Comparison with Predicate Device: N/A

2. Matrix Comparison: N/A

# C Clinical Studies:

1. Clinical Performance:

Aptima GC clinical performance was evaluated in a clinical comparison study, using fresh and frozen prospective patient-collected first catch male urine specimens. Overall, 2085 males were prospectively enrolled in a multicenter study. Participants were sexually active individuals at least 14 years of age, attending one of eleven (11) participating clinical collection sites, with or without symptoms of an STI infection.

One first-catch urine specimen was collected from each male subject. Four urine samples were prepared from each first-catch urine specimen to be processed for testing with the investigational assay and three FDA cleared NAAT comparator assays. Three clinical testing sites performed Aptima GC testing; all comparator testing was completed at a single central laboratory. The results of the Aptima GC Assay were compared to a patient infected status (PIS), based on results from three FDA-cleared NAATs. Specimens were categorized as infected if a positive result occurred in at least two of the comparator NAATs, and as not infected if at least two of the comparator results were negative; the third (tiebreaker) comparator assay was only required if the first two comparator results were discordant.

The calculated performance estimates are shown below.

Among 2085 collected samples, 1958 were evaluable and 127 samples were excluded from the calculation (12 samples were withdrawn, 50 withdrawn Aptima MU, 13 AGC missing/final Invalid results, 51 PIS cannot be determined and 1 equivocal by AGC result). The total invalid rate with Aptima GC assay is $0 . 6 \%$ .

Table 8. Clinical Performance of the Aptima Neisseria gonorrhoeae Assay in Male Urine Samples on the Panther System by Symptom Status (Evaluable Specimens).   

<table><tr><td rowspan=1 colspan=1>SymptomStatus</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>TP</td><td rowspan=1 colspan=1>FP</td><td rowspan=1 colspan=1>TN</td><td rowspan=1 colspan=1>FN</td><td rowspan=1 colspan=1>Prev(%)</td><td rowspan=1 colspan=1>Sensitivity %(95% CI)1</td><td rowspan=1 colspan=1>Specificity %(955C)</td><td rowspan=1 colspan=1>PPV %(95% CI)²</td><td rowspan=1 colspan=1>NPV %(95% CI)²</td></tr><tr><td rowspan=1 colspan=1>All</td><td rowspan=1 colspan=1>1958</td><td rowspan=1 colspan=1>125</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1830</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>6.5</td><td rowspan=1 colspan=1>98.4 (94.4,99.6)</td><td rowspan=1 colspan=1>99.9 (99.7,100)</td><td rowspan=1 colspan=1>99.2 (95.8,100)</td><td rowspan=1 colspan=1>99.9 (99.6,100)</td></tr><tr><td rowspan=1 colspan=1>Symptomatic</td><td rowspan=1 colspan=1>825</td><td rowspan=1 colspan=1>105</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>717</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>98.1 (93.4,99.5)</td><td rowspan=1 colspan=1>99.9 (99.2,100)</td><td rowspan=1 colspan=1>99.1 (95.1,100)</td><td rowspan=1 colspan=1>99.7 (99.0,100)</td></tr><tr><td rowspan=1 colspan=1>Asymptomatic</td><td rowspan=1 colspan=1>1133</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1113</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1.8</td><td rowspan=1 colspan=1>100 (83.9, 100)</td><td rowspan=1 colspan=1>100 (99.7,100)</td><td rowspan=1 colspan=1>100 (84.4,100)</td><td rowspan=1 colspan=1>100 (99.7,100)</td></tr></table>

$\mathrm { F N = }$ false negative, $\mathrm { F P = }$ false positive Prev $=$ prevalence, $\mathrm { T N } =$ true negative, $\mathrm { T P } =$ true positive. Note: This table includes the subset of specimens in the Evaluable Specimen Population that have a valid negative or positive AGC assay result. 1Score CI.2PPV $9 5 \%$ CI computed from the exact $9 5 \%$ CI for the positive likelihood ratio, NPV $9 5 \%$ CI computed from the exact $9 5 \%$ CI for the negative likelihood ratio.

One equivocal result was observed by Aptima GC and was negative by PIS. This sample was excluded from the calculation of performance; If the equivocal result is considered discordant (i.e., false positive), the calculated performance estimates are : Sensitivity: $9 8 . 4 3 \% 9 5 \%$ CI: $( 9 4 . 4 4 \% \cdot 9 9 . 5 7 \% )$ ) and Specificity: $9 9 . 8 9 \%$ , $9 5 \%$ CI: $( 9 9 . 6 0 \% - 9 9 . 9 7 \% )$ ).If the equivocal result is included in the calculation as true negative , the performance estimates are: Sensitivity: $9 8 . 4 3 \%$ $9 5 \%$ CI: $( 9 4 . 4 4 \% - 9 9 . 5 7 \% )$ ) and Specificity: $9 9 . 9 5 \%$ , $9 5 \%$ CI: $( 9 9 . 6 9 \% - 9 9 . 9 9 \% )$ ).

Table 9. Neisseria gonorrhoeae Patient Infected Status for Male Urine Specimens (Evaluable Specimen Population).   

<table><tr><td colspan="1" rowspan="2">Patient InfectedStatus</td><td colspan="2" rowspan="1">Primary</td><td colspan="1" rowspan="1">Tiebreaker</td><td colspan="1" rowspan="1">AGC</td><td colspan="2" rowspan="1">Symptom Status</td></tr><tr><td colspan="1" rowspan="1">NAAT 1Result</td><td colspan="1" rowspan="1">NAAT 2Result</td><td colspan="1" rowspan="1">NAAT 3Result</td><td colspan="1" rowspan="1">AssayResult</td><td colspan="1" rowspan="1">Symptomatic</td><td colspan="1" rowspan="1">Asymptomatic</td></tr><tr><td colspan="1" rowspan="1">Infected</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">97</td><td colspan="1" rowspan="1">19</td></tr><tr><td colspan="1" rowspan="1">Infected</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">Infected</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">NR</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">Infected</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">Infected</td><td colspan="1" rowspan="1">NR</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">Non-infected</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">Non-infected</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">2</td></tr><tr><td colspan="1" rowspan="1">Non-infected</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">689</td><td colspan="1" rowspan="1">1079</td></tr><tr><td colspan="1" rowspan="1">Non-infected</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">=</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">Non-infected</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">NR</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">Non-infected</td><td colspan="1" rowspan="1">NR</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">26</td><td colspan="1" rowspan="1">32</td></tr></table>

- $=$ negative, $+ =$ positive, AGC assay $=$ Aptima Neisseria gonorrhoeae assay on the Panther system, $\mathrm { { N / A } = n o t }$ applicable, $\mathrm { N R } =$ no result, Note: The equal symbol $( = )$ represents a final equivocal result; the sample was negative for GC based on the comparator assays. The sample was not included in the final calculation of performance estimates.

2. Clinical Specificity: See Above.

D Clinical Cut-Off: N/A

# E Expected Values/Reference Range:

The positivity of Neisseria gonorrhoeae, as determined by the Aptima GC assay, during the clinical study is shown in Table 10 below.

# Table 10. Positivity of N. gonorrhoeae in Male Urine as Determined by the Aptima Neisseria gonorrhoeae (GC) assay, stratified by Clinical Sites.

<table><tr><td rowspan=1 colspan=1>Site</td><td rowspan=1 colspan=1>% Positivity</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>21.7 (38/175)</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0.8 (3/373)</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0 (0/61)</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>0 (0/13)</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>8.3 (34/409)</td></tr><tr><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>9.4 (29/307)</td></tr><tr><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>5.3 (12/225)</td></tr><tr><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>0 (0/32)</td></tr><tr><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>0 (0/218)</td></tr><tr><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>11.0 (10/91)</td></tr><tr><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>0 (0/54)</td></tr><tr><td rowspan=1 colspan=1>ALL</td><td rowspan=1 colspan=1>6.4 (126/1958)</td></tr></table>

# F Other Supportive Instrument Performance Characteristics Data:

Not applicable

# VIII Proposed Labeling:

The labeling is acceptable and supports the finding of substantial equivalence for this device.

# IX Conclusion:

The submitted information in this premarket notification is complete and supports a substantial equivalence decision.