# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATION DECISION SUMMARY

I Background Information:

A 510(k) Number K231481   
B Applicant Cepheid   
C Proprietary and Established Names Xpert Xpress CoV-2/Flu/RSV plus   
D Regulatory Information

<table><tr><td>Product Code(s)</td><td>Classification</td><td>Regulation Section</td><td>Panel</td></tr><tr><td>QOF</td><td>Class II</td><td>21 CFR 866.3981 - Device To Detect And Identify Nucleic Acid Targets In Respiratory Specimens From Microbial Agents That Cause The SARS- Cov-2 Respiratory Infection And Other Microbial</td><td>MI - Microbiology</td></tr><tr><td>0OI</td><td>Class II</td><td>Agents When In A Multi-Target Test 21 CFR 862.2570 - Instrumentation for clinical multiplex test systems</td><td>CH - Clinical Chemistry</td></tr></table>

# II Submission/Device Overview:

# A Purpose for Submission:

To obtain 510(k) clearance for the Xpert Xpress CoV-2/Flu/RSV plus test for use on the GeneXpert Dx and Infinity Instrument Systems.

# B Measurand:

Conserved RNA sequences within the genes encoding the nucleocapsid protein (N), envelope protein (E) and RNA-dependent RNA polymerase protein (RdRP) of SARS-CoV-2 viruses; the matrix protein (M), basic polymerase protein 2 (PB2), and polymerase acidic protein (PA) of influenza A viruses; the matrix protein (M) and non-structural protein (NS) of influenza B viruses; and the nucleocapsid protein of respiratory syncytial virus (RSV) A and B viruses.

# C Type of Test:

A multiplexed, real-time reverse transcriptase polymerase chain reaction (RT-PCR) assay for the qualitative detection and differentiation of SARS-CoV-2, influenza A, influenza B, and RSV viral RNA from nasopharyngeal swab (NPS) specimens and anterior nasal swab (NS) specimens using the GeneXpert Instrument Systems platform.

# III Intended Use/Indications for Use:

A Intended Use(s): See Indications for Use below.

# B Indication(s) for Use:

The Xpert Xpress CoV-2/Flu/RSV plus test, performed on the GeneXpert Dx and GeneXpert Infinity Systems, is an automated multiplexed real-time reverse transcriptase polymerase chain reaction (RT-PCR) test intended for use in the simultaneous in vitro qualitative detection and differentiation of severe acute respiratory syndrome coronavirus (SARS-CoV-2), influenza A, influenza B, and/or respiratory syncytial virus (RSV) viral RNA in nasopharyngeal swab and anterior nasal swab specimens collected from individuals with signs and symptoms of respiratory tract infection. Clinical signs and symptoms of respiratory tract infection due to SARS-CoV-2, influenza A, influenza B, and RSV can be similar.

The Xpert Xpress CoV-2/Flu/RSV plus is intended for use in the differential detection of SARSCoV-2, influenza A, influenza B and/or RSV RNA and aids in the diagnosis of COVID-19, influenza and/or RSV infections if used in conjunction with other clinical and epidemiological information, and laboratory findings. SARS-CoV-2, influenza A, influenza B, and RSV viral RNA are generally detectable in nasopharyngeal swab and anterior nasal swab specimens during the acute phase of infection.

Positive results are indicative of the presence of the identified virus, but do not rule out bacterial infection or co-infection with other pathogens not detected by the test. The agent(s) detected by the Xpert Xpress CoV-2/Flu/RSV plus test may not be the definite cause of disease.

Negative results do not preclude SARS-CoV-2, influenza A, influenza B, and/or RSV infection. The results of this test should not be used as the sole basis for diagnosis, treatment, or other patient management decisions.

C Special Conditions for Use Statement(s): Rx - For Prescription Use Only For in vitro diagnostic use only

D Special Instrument Requirements: GeneXpert Instrument Systems:

GeneXpert Dx Systems (GX-I, GX-II, GX-IV, GX-XVI) GeneXpert Infinity Systems (Infinity-48s and Infinity-80)

# IV Device/System Characteristics:

# A Device Description:

The Xpert Xpress CoV-2/Flu/RSV plus test, performed on the GeneXpert Instrument (GeneXpert Dx, GeneXpert Infinity-48s and GeneXpert Infinity-80 systems), is a multiplex nucleic acid amplification test for the qualitative detection and differentiation of SARS-CoV-2, influenza A, influenza B, and RSV viral RNA from NPS and NS swab specimens.

To perform the Xpert Xpress CoV-2/Flu/RSV plus test, NPS and NS swab specimens from patients are collected with nylon flocked swabs and placed into either $3 ~ \mathrm { m L }$ viral transport medium (VTM) or $2 \mathrm { m L }$ eNAT. With a supplied transfer pipette, eluted NPS and NS swab specimens are loaded into the sample chamber of a single-use, self-contained Xpert Xpress CoV2/Flu/RSV plus cartridge. Each assay cartridge contains separate chambers for sample loading, sample processing, and target amplification by real-time RT-PCR, and contains all the reagents necessary to carry out these processes. Because the cartridges are self-contained, and specimens never contact working parts of the instrument modules, cross-contamination between samples is minimized.

The assay cartridge containing the patient sample is inserted into the GeneXpert Instrument System, which performs fully automated and integrated sample preparation and real-time RTPCR for the Xpert Xpress CoV-2/Flu/RSV plus test in approximately 36 minutes. The Xpert Xpress CoV-2/Flu/RSV plus test can be run in one of five different modes, varying in the number of target analyte(s) tested for and reported. The five modes are: SARS-CoV-2 only; SARS-CoV-2 and Flu; SARS-CoV-2/Flu/RSV; Flu only; or Flu and RSV. For the SARS-CoV-2 only and Flu only modes, there is an option to activate an Early Assay Termination (EAT) function, which will provide earlier time to results in high titer specimens if the signal from the target nucleic acid reaches a predetermined threshold before the assay has finished running. When EAT is activated, the earliest time to a positive result is $\sim 2 5$ minutes. The GeneXpert Dx and Infinity Systems have one to 80 randomly accessible modules that are each capable of performing separate sample preparation and real-time PCR and RT-PCR tests. Each module contains a syringe drive for dispensing fluids (i.e., the syringe drive activates the plunger that works in concert with the rotary valve in the cartridge to move fluids between chambers), an ultrasonic horn for lysing cells or spores, and a proprietary I-CORE thermocycler for performing real-time PCR and RT-PCR detection.

After completion of the test, the assay results are interpreted by the GeneXpert software from measured fluorescent signals and embedded calculation algorithms and are shown in the “View Results” window.

# B Principle of Operation:

The Xpert Xpress CoV-2/Flu/RSV plus test is a nucleic acid-based test using real-time RT-PCR. After addition of the specimen to the Sample Chamber of the Xpert Xpress CoV-2/Flu/RSV plus test cartridge, the cartridge is loaded onto the GeneXpert Instrument System platform. The instrument then performs automated sample processing including RNA extraction, followed by reverse transcription, amplification, detection, and reporting of results. The results are interpreted automatically by the GeneXpert System and are shown in the View Results window.

# C Instrument Description Information:

1. Instrument Name:

• GeneXpert Dx Systems: GeneXpert Dx software version 4.7b or higher GeneXpert Infinity Systems: Xpertise software version 6.4b or higher

# 2. Specimen Identification:

To perform a test, the user selects the “Create Test” (Dx Systems) or “Orders” (Infinity Systems) icon, scans the cartridge barcode, enters the sample ID, or scans the sample ID, and

loads the cartridge into the instrument module that has the green blinking light (for the Dx Systems) or onto the conveyor belt (for the Infinity Systems) to start the test.

# 3. Specimen Sampling and Handling:

NPS and NS swab specimens are collected using nylon flocked swabs and eluted into $3 ~ \mathrm { m L }$ VTM or $2 \mathrm { m L }$ eNAT. At the testing facility, the operator mixes the specimen by rapidly inverting the specimen transport tube 5 times. A transfer pipette provided with the Xpert Xpress CoV-2/Flu/RSV plus is used to transfer an aliquot of the specimen into the Sample Chamber of the open test cartridge. After closing the cartridge lid, the operator loads the cartridge onto the applicable GeneXpert instrument for testing.

# 4. Calibration:

GeneXpert instruments are calibrated at the factory. Routine calibration of the GeneXpert Instrument systems may be performed by Cepheid Field Service Engineers during annual maintenance.

# 5. Quality Control:

The Xpert Xpress CoV-2/Flu/RSV plus test includes two internal controls: a sample processing control (SPC), and a probe check control (PCC).

# Sample Processing Control (SPC)

The sample processing control is a non-infectious armored RNA pseudovirus that ensures adequate lysis of target virus, monitors the presence of PCR inhibitors, and verifies the use of proper PCR conditions. The SPC should be POSITIVE in a sample that is negative for all four SARS-CoV-2, influenza A, influenza B and RSV target analytes, and can be NEGATIVE or POSITIVE in a sample containing detectable levels of one or more of the target analytes.

# Probe Check Control (PCC)

The probe check control is present to control for sufficient reagent rehydration, PCR tube filling, probe integrity and dye stability. All assay reagents must be present and intact for the PCC to pass the validated acceptance criteria. If any of the PCC conditions fail, the result is reported as an ERROR and the test must be repeated using a new assay cartridge.

# External Controls

Optional external quality control materials are available from Zeptometrix. Specifically, the following external controls may be used with the Xpert Xpress CoV-2/Flu/RSV plus test:

• External Positive Control: NATtrol Flu/RSV/SARS-CoV-2; Cat # NATFRC-6C-IVD External Negative Control: Coxsackievirus A9; Cat # NATCV9-6C-IVD

All external controls must be used in accordance with local, state, and federal accrediting organizations, as applicable.

# V Substantial Equivalence Information:

A Predicate Device Name(s): BioFire Respiratory Panel 2.1 (RP2.1)

B Predicate 510(k) Number(s): DEN200031

# C Comparison with Predicate(s):

<table><tr><td>Device &amp; Predicate Device(s):</td><td>K231481</td><td>DEN200031</td></tr><tr><td>Device Trade Name</td><td>Xpert Xpress CoV-2/Flu/RSV plus</td><td>BioFire Respiratory Panel 2.1</td></tr><tr><td>Regulation Number and Name</td><td>Same</td><td>21 CFR 866.3981; Devices to detect and identify nucleic acid targets in respiratory samples from microbial agents that cause the SARS-CoV-2 respiratory infection and other microbial agents when in a multi-analyte</td></tr><tr><td>Product Code</td><td>Same</td><td>test. QOF</td></tr><tr><td>Technology/Detection</td><td>Real-time reverse transcription polymerase chain reaction (RT-qPCR)</td><td>Nested multiplex RT-PCR followed by high resolution melting analysis to confirm identity of amplified product.</td></tr><tr><td>Intended Use/Indications for Use</td><td>The Xpert Xpress CoV- 2/Flu/RSV plus test, performed on the GeneXpert Dx and GeneXpert Infinity Systems, is an automated multiplexed real-time reverse transcriptase polymerase chain reaction (RT-PCR) test intended for use in the simultaneous in vitro qualitative detection and differentiation of severe acute respiratory syndrome coronavirus (SARS-CoV-2), influenza A, influenza B, and/or respiratory syncytial virus (RSV) viral RNA in nasopharyngeal swab and anterior nasal swab specimens collected from individuals with signs and symptoms of respiratory tract infection. Clinical signs and symptoms of respiratory tract infection due to SARS-CoV-2, influenza A, influenza B, and RSV can be similar. The Xpert Xpress CoV- 2/Flu/RSV plus is intended</td><td>The BioFire Respiratory Panel 2.1 (RP2.1) is a PCR-based multiplexed nucleic acid test intended for use with the BioFire FilmArray 2.0 or BioFire FilmArray Torch Systems for the simultaneous qualitative detection and identification of multiple respiratory viral and bacterial nucleic acids in nasopharyngeal swabs (NPS) obtained from individuals suspected of respiratory tract infections, including COVID-19. Nucleic acids from the respiratory viral and bacterial organisms identified by this test are generally detectable in NPS specimens during the acute phase of infection. The detection and identification of specific viral and bacterial nucleic acids from individuals exhibiting signs and/or symptoms of respiratory infection is indicative of the presence of the identified microorganism and aids in the diagnosis of</td></tr><tr><td colspan="3">detection of SARS-CoV-2, conjunction with other clinical</td></tr><tr><td>disease. decisions.</td><td>influenza A, influenza B and/or RSV RNA and aids in the diagnosis of COVID-19, influenza and/or RSV infections if used in conjunction with other clinical and epidemiological information, and laboratory findings. SARS-CoV-2, influenza A, influenza B, and RSV viral RNA are generally detectable in nasopharyngeal swab and anterior nasal swab specimens during the acute phase of infection. Positive results are indicative of the presence of the identified virus, but do not rule out bacterial infection or co-infection with other pathogens not detected by the test. The agent(s) detected by the Xpert Xpress CoV- 2/Flu/RSV plus test may not be the definite cause of Negative results do not preclude SARS-CoV-2, influenza A, influenza B, and/or RSV infection. The results of this test should not be used as the sole basis for diagnosis, treatment, or other patient management that may not be detected by an NPS specimen. Positive results</td><td>and epidemiological information. The results of this test should not be used as the sole basis for diagnosis, treatment, or other patient management decisions. The following organism types and subtypes are identified using the BioFire RP2.1: Adenovirus Coronavirus 229E Coronavirus HKU1 Coronavirus NL63 Coronavirus OC43 Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2) Human Metapneumovirus Human Rhinovirus/Enterovirus Influenza A, including • subtypes H1, H3 and H1- 2009 Influenza B Parainfluenza Virus 1 Parainfluenza Virus 2 Parainfluenza Virus 3 Parainfluenza Virus 4 Respiratory Syncytial Virus Bordetella parapertussis Bordetella pertussis • Chlamydia pneumoniae Mycoplasma pneumoniae Negative results in the setting of respiratory illness may be due to infection with pathogens that are not detected by this test, or lower respiratory tract infection</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">evaluating a patient withpossible respiratory tractinfection.</td></tr><tr><td colspan="1" rowspan="1">Test Format</td><td colspan="1" rowspan="1">Same</td><td colspan="1" rowspan="1">Single Use</td></tr><tr><td colspan="1" rowspan="1">Automated Test Processes</td><td colspan="1" rowspan="1">Same</td><td colspan="1" rowspan="1">Nucleic Acid Extraction,Detection and ResultsInterpretation</td></tr><tr><td colspan="1" rowspan="1">Results Reported</td><td colspan="1" rowspan="1">Same</td><td colspan="1" rowspan="1">Qualitative</td></tr><tr><td colspan="1" rowspan="1">Specimen Type</td><td colspan="1" rowspan="1">Nasopharyngeal (NPS) swaband Anterior Nasal (NS)swab</td><td colspan="1" rowspan="1">Nasopharyngeal (NPS) swab</td></tr><tr><td colspan="1" rowspan="1">Transport Media</td><td colspan="1" rowspan="1">Universal TransportMedium (UTM)/ViralTransport Medium (VTM),eNAT</td><td colspan="1" rowspan="1">Universal Transport Medium(UTM)/ Viral TransportMedium (VTM), saline</td></tr><tr><td colspan="1" rowspan="1">Internal Control</td><td colspan="1" rowspan="1">Sample Processing Control(SP), Probe Check Control(PCC)</td><td colspan="1" rowspan="1">Sample Processing Control,PCR and Melt AnalysisControl</td></tr><tr><td colspan="1" rowspan="1">Instrument Systems</td><td colspan="1" rowspan="1">GeneXpert Dx and InfinityInstrument Systems</td><td colspan="1" rowspan="1">BioFire FilmArray 2.0 orBioFire FilmArray TorchSystems</td></tr><tr><td colspan="1" rowspan="1">Time to Result</td><td colspan="1" rowspan="1">~36 minutes for samplepreparation and RT-PCR</td><td colspan="1" rowspan="1">~45 minutes</td></tr></table>

# VI Standards/Guidance Documents Referenced:

# Standards

ISO 14971. Medical Devices – Application of Risk Management to Medical Devices (2019) CLSI EP12-A2. User Protocol for Evaluation of Qualitative Test Performance; Approved Guideline – Second Edition   
• CLSI EP17-A2. Evaluation of Detection Capability for Clinical Laboratory Measurement Procedures; Approved Guideline – Second Edition CLSI EP25-A. Evaluation of Stability of In Vitro Diagnostic Reagents; Approved Guideline. CLSI EP05-A3. Evaluation of Precision of Quantitative Measurement Procedures; Approved Guideline – Third Edition. CLSI MM03-A3. Molecular Diagnostic Methods for Infectious Diseases; Approved Guidance – Third Edition. CLSI EP07. Testing in Clinical Chemistry. Third Edition. CLSI MM17. Verification and Validation of Multiplex Nucleic Acid Assays. Second Edition.   
• CLSI MM13. Collection, Transport, Preparation, and Storage of Specimens for Molecular Methods. Second Edition.

• Class II Special Controls as per 21 CFR 866.3981.

# Guidance Documents

Guidance for Industry and FDA Staff, Format for Traditional and Abbreviated 510(k)s (September 13, 2019). Guidance for Industry and FDA Staff, Appropriate Use of Voluntary Consensus Standards in Premarket Submissions for Medical Devices (September 14, 2018). Guidance for Sponsors, Institutional Review Boards, Clinical Investigators and FDA Staff, Guidance on Informed Consent for In Vitro Diagnostic Device Studies Using Leftover Human Specimens that are Not Individually Identifiable (April 25, 2006). Guidance for Industry and FDA Staff, Class II Special Controls Guidance Document: Instrumentation for Clinical Multiplex Test Systems (March 10, 2005). Guidance for Industry and FDA Staff, Guidance for the Content of Premarket Submissions for Software Contained in Medical Devices (May 11, 2005). Guidance for Industry and FDA Staff, General Principles of Software Validation (January 11, 2002). Guidance for Industry and FDA Staff, Content of Premarket Submissions for Management of Cybersecurity in Medical Devices (October 2, 2014). Guidance for Industry, Cybersecurity for Networked Medical Devices Containing Offthe-Shelf (OTS) Software (January 14, 2005). • Guidance for Industry and FDA Staff, Off-the-Shelf Software Use in Medical Devices (September 27, 2019). Guidance for Industry and FDA Staff, eCopy Program for Medical Device Submissions (April 27, 2020). Guidance for Test Developers and FDA Staff, Policy for Evaluating Impact of Viral Mutations on COVID-19 Tests (Revised); (January 12, 2023). Guidance for Industry, Other Stakeholders, and FDA Staff, Transition Plan for Medical Devices Issued Emergency Use Authorizations (EUA) During the Coronavirus Disease 2019 (COVID-19) Public Health Emergency (March 27, 2023).

# VII Performance Characteristics (if/when applicable):

# A Analytical Performance:

1. Precision/Reproducibility:

a. Within-Laboratory Precision

Within-laboratory precision was evaluated at a single site using the Xpert Xpress CoV2/Flu/RSV plus test run on the GeneXpert Infinity System. A total of nine contrived panels containing known quantities of the target analytes were prepared in simulated matrix, consisting of $2 . 5 \%$ (w/v) porcine mucin, $1 \% ( \mathrm { v / v } )$ human whole blood in $0 . 8 5 \%$ sodium chloride (NaCl) formulated in a $1 \mathbf { x }$ PBS solution with $1 5 \%$ glycerol, which was then diluted at a 1:40 ratio to a concentration of $2 . 5 \%$ (v/v) in UTM/VTM. Data supporting the use of simulated matrix can be found in section VII Performance Characteristics.B.2.Sample Matrix and Transport Media Equivalency Study, later in this document. The viral materials used to generate the positive panel members are denoted in Table 1. The contrived positive panels consisted of single target spiked

samples at low concentrations ${ \mathrm { \sim } } 1 . 5 \mathrm { x }$ LoD) and moderate concentrations $\left( \sim 3 \mathrm { x } \ L \mathrm { o D } \right)$ (Table 2). The study was conducted with one operator and one cartridge lot over the course of 20 days. Each panel member was tested in duplicate twice per day on 20 different days generating a total of 80 replicates per panel member (1 Site $_ { \textrm { x 1 } }$ Operator x 1 Lot x 20 Days x 2 Runs per Day x 2 Replicates per run).

Table 1. Viral Strains for Within-Laboratory Precision Study   

<table><tr><td rowspan=1 colspan=1>Description</td><td rowspan=1 colspan=1>Vendor</td><td rowspan=1 colspan=1>Catalog Number</td></tr><tr><td rowspan=1 colspan=1>SARS-CoV-2 (USA-WA1/2020),inactivated</td><td rowspan=1 colspan=1>Zeptometrix</td><td rowspan=1 colspan=1>NATSARS(COV2)-ST</td></tr><tr><td rowspan=1 colspan=1>Influenza A/Idaho/07/2018, cultured</td><td rowspan=1 colspan=1>Influenza ReagentResource (IRR)</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=1>Influenza B/Wisconsin/10/2016,cultured</td><td rowspan=1 colspan=1>Influenza ReagentResource (IRR)</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=1>RSV B/Wash/18537/62,cultured</td><td rowspan=1 colspan=1>Cepheid (internalstrain)</td><td rowspan=1 colspan=1>NA</td></tr></table>

Table 2. Precision Study Sample Panel   

<table><tr><td rowspan=1 colspan=1>Panel ID</td><td rowspan=1 colspan=1>Description</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>Negative</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>SARS-CoV-2 (~1.5x LoD)</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>SARS-CoV-2 (~3x LoD)</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>Flu A (~1.5x LoD)</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>Flu A (~3x LoD)</td></tr><tr><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>Flu B (~1.5x LoD)</td></tr><tr><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>Flu B (~3x LoD)</td></tr><tr><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>RSV B (~1.5x LoD)</td></tr><tr><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>RSV B (~3x LoD)</td></tr></table>

The qualitative (i.e., $\%$ agreement with expected results) and Ct results from the study are illustrated in Table 3 and Table 4, respectively.

Table 3. Within-Laboratory Precision Study – Qualitative Results   

<table><tr><td rowspan=1 colspan=1>PanelID</td><td rowspan=1 colspan=1>Panel Member</td><td rowspan=1 colspan=1>% Positive(pos n/valid n)</td><td rowspan=1 colspan=1>% Agreement withExpected Results(95% CI)</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>0% (0/80)</td><td rowspan=1 colspan=1>100% (95.4-100%)</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>SARS-CoV-2 (~1.5x LoD)</td><td rowspan=1 colspan=1>98.8% (79/80)</td><td rowspan=1 colspan=1>98.8% ((93.3-99.8%)</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>SARS-CoV-2 (~3x LoD)</td><td rowspan=1 colspan=1>100% (80/80)</td><td rowspan=1 colspan=1>100%(95.4-100%)</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>Flu A (~1.5x LoD)</td><td rowspan=1 colspan=1>97.5% (78/80)</td><td rowspan=1 colspan=1>97.5%(91.3-99.3%)</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>Flu A (~3x LoD)</td><td rowspan=1 colspan=1>100% (80/80)</td><td rowspan=1 colspan=1>100%(95.4-100%)</td></tr><tr><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>FluB3 (~1.5x LoD)</td><td rowspan=1 colspan=1>96.3%b (77/80)</td><td rowspan=1 colspan=1>96.3%(89.5-98.7%)</td></tr><tr><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>Flu B (~3x LoD)</td><td rowspan=1 colspan=1>100% (80/80)</td><td rowspan=1 colspan=1>100%(95.4-100%)</td></tr><tr><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>RSV B (~1.5x LoD)</td><td rowspan=1 colspan=1>97.5% (78/80)</td><td rowspan=1 colspan=1>97.5%(91.3-99.3%)</td></tr><tr><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>RSV B (~3x LoD)</td><td rowspan=1 colspan=1>100% (80/80)</td><td rowspan=1 colspan=1>100% (95.4-100%)</td></tr></table>

All low positive $( \mathord { \sim } 1 . 5 \mathrm { x } )$ panel members were positive $2 9 6 . 3 \%$ . All moderate positive $( \sim 3 \mathbf { x } )$ panel members were $100 \%$ positive for the spiked target analytes. The negative panel member was $0 . 0 \%$ positive for SARS-CoV-2, Flu A, Flu B, and RSV, (Table 3, above).

Table 4. Within-Laboratory Precision Study – Ct Signal Variability Analysis Results   

<table><tr><td rowspan=2 colspan=1>PanelMember</td><td rowspan=2 colspan=1>Target</td><td rowspan=2 colspan=1>N</td><td rowspan=2 colspan=1>MeanCt</td><td rowspan=1 colspan=2>BetweenDays</td><td rowspan=1 colspan=2>BetweenRuns</td><td rowspan=1 colspan=2>WithinTest</td><td rowspan=1 colspan=2>Total</td></tr><tr><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>CV(%)</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>CV(%)</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>CV(%)</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>CV(%)</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>SPC</td><td rowspan=1 colspan=1>80</td><td rowspan=1 colspan=1>28.93</td><td rowspan=1 colspan=1>0.31</td><td rowspan=1 colspan=1>1.1</td><td rowspan=1 colspan=1>0.21</td><td rowspan=1 colspan=1>0.7</td><td rowspan=1 colspan=1>0.76</td><td rowspan=1 colspan=1>2.6</td><td rowspan=1 colspan=1>0.85</td><td rowspan=1 colspan=1>2.9</td></tr><tr><td rowspan=1 colspan=1>SARS-CoV-2~1.5x LoD</td><td rowspan=2 colspan=1>SARS-CoV-2</td><td rowspan=1 colspan=1>79</td><td rowspan=1 colspan=1>36.95</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.45</td><td rowspan=1 colspan=1>1.2</td><td rowspan=1 colspan=1>1.05</td><td rowspan=1 colspan=1>2.8</td><td rowspan=1 colspan=1>1.14</td><td rowspan=1 colspan=1>3.1</td></tr><tr><td rowspan=1 colspan=1>SARS-CoV-2~3x LoD</td><td rowspan=1 colspan=1>80</td><td rowspan=1 colspan=1>35.69</td><td rowspan=1 colspan=1>0.21</td><td rowspan=1 colspan=1>0.6</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.78</td><td rowspan=1 colspan=1>2.2</td><td rowspan=1 colspan=1>0.81</td><td rowspan=1 colspan=1>2.3</td></tr><tr><td rowspan=2 colspan=1>Flu A~1.5x LoD</td><td rowspan=1 colspan=1>Flu A1</td><td rowspan=1 colspan=1>80</td><td rowspan=1 colspan=1>35.49</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.33</td><td rowspan=1 colspan=1>0.9</td><td rowspan=1 colspan=1>1.27</td><td rowspan=1 colspan=1>3.6</td><td rowspan=1 colspan=1>1.31</td><td rowspan=1 colspan=1>3.7</td></tr><tr><td rowspan=1 colspan=1>Flu A2</td><td rowspan=1 colspan=1>55</td><td rowspan=1 colspan=1>38.29</td><td rowspan=1 colspan=1>0.58</td><td rowspan=1 colspan=1>1.5</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>1.50</td><td rowspan=1 colspan=1>3.9</td><td rowspan=1 colspan=1>1.61</td><td rowspan=1 colspan=1>4.2</td></tr><tr><td rowspan=2 colspan=1>Flu A~3x LoD</td><td rowspan=1 colspan=1>Flu A1</td><td rowspan=1 colspan=1>80</td><td rowspan=1 colspan=1>34.30</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.23</td><td rowspan=1 colspan=1>0.7</td><td rowspan=1 colspan=1>0.59</td><td rowspan=1 colspan=1>1.7</td><td rowspan=1 colspan=1>0.63</td><td rowspan=1 colspan=1>1.8</td></tr><tr><td rowspan=1 colspan=1>Flu A2</td><td rowspan=1 colspan=1>75</td><td rowspan=1 colspan=1>36.56</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.53</td><td rowspan=1 colspan=1>1.5</td><td rowspan=1 colspan=1>0.67</td><td rowspan=1 colspan=1>1.8</td><td rowspan=1 colspan=1>0.86</td><td rowspan=1 colspan=1>2.3</td></tr><tr><td rowspan=1 colspan=1>Flu B~1.5x LoD</td><td rowspan=2 colspan=1>Flu B</td><td rowspan=1 colspan=1>79</td><td rowspan=1 colspan=1>35.57</td><td rowspan=1 colspan=1>0.27</td><td rowspan=1 colspan=1>0.8</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>1.35</td><td rowspan=1 colspan=1>3.8</td><td rowspan=1 colspan=1>1.37</td><td rowspan=1 colspan=1>3.9</td></tr><tr><td rowspan=1 colspan=1>Flu B∼3x LoD</td><td rowspan=1 colspan=1>80</td><td rowspan=1 colspan=1>34.35</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.84</td><td rowspan=1 colspan=1>2.5</td><td rowspan=1 colspan=1>0.84</td><td rowspan=1 colspan=1>2.5</td></tr><tr><td rowspan=1 colspan=1>RSV~1.5x LoD</td><td rowspan=2 colspan=1>RSV</td><td rowspan=1 colspan=1>80</td><td rowspan=1 colspan=1>35.78</td><td rowspan=1 colspan=1>0.24</td><td rowspan=1 colspan=1>0.7</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>1.02</td><td rowspan=1 colspan=1>2.8</td><td rowspan=1 colspan=1>1.04</td><td rowspan=1 colspan=1>2.9</td></tr><tr><td rowspan=1 colspan=1>RSV~3x LoD</td><td rowspan=1 colspan=1>80</td><td rowspan=1 colspan=1>34.75</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.34</td><td rowspan=1 colspan=1>1.0</td><td rowspan=1 colspan=1>0.61</td><td rowspan=1 colspan=1>1.7</td><td rowspan=1 colspan=1>0.69</td><td rowspan=1 colspan=1>2.0</td></tr></table>

$\mathrm { C t } =$ cycle threshold; $\mathrm { C V } =$ coefficient of variation; $\mathbf { S D = }$ standard deviation

The mean and variability analysis between days, between runs, within runs, and overall (total) for Ct values is shown in Table 4. Overall $\% \mathrm { C V }$ was $\leq 4 . 2 \%$ . The greatest source of variability was from Flu A at ${ \sim } 1 . 5 \mathbf { x }$ LoD (Flu A2 Ct), which had a within-run $\% C V$ of $3 . 9 \%$ . Overall variability was low, and the study demonstrates assay variability within an acceptable range.

# b. Reproducibility

A blinded, multi-site reproducibility study was conducted to assess the total variability of the Xpert Xpress CoV-2/Flu/RSV plus test across operators, study sites, testing days, runs, instruments (GeneXpert Dx and Infinity Systems), and reagent lots.

The same nine contrived panels evaluated in the Within-Laboratory Precision Study (see Table 2) were also tested in the Reproducibility Study. The study was performed by two operators at each of three testing sites. At each site three cartridge reagent lots were evaluated. One site performed testing using the GeneXpert Infinity System, while the remaining two sites conducted testing with the GeneXpert Dx System. Each panel member was tested in duplicate twice per day on 6 different days generating a total of 144 replicates per panel member (3 Sites $_ { \textrm { X 2 } }$ Operators per site x 3 Lots x 2 Days/Lot x 2 Runs per Day x 2 Replicates per run).

The qualitative (i.e., $\%$ agreement with expected results) and quantitative results from the study are illustrated in Table 5 and Table 6, respectively.

Table 5. Reproducibility Study – Qualitative Results   

<table><tr><td rowspan=3 colspan=1>PanelMember</td><td rowspan=1 colspan=9>% Agreement with Expected Results</td><td rowspan=3 colspan=1>Overall%Agreement(5% CI)</td></tr><tr><td rowspan=1 colspan=3>Site 1 (Infinity)</td><td rowspan=1 colspan=3>Site 2 (Dx)</td><td rowspan=1 colspan=2>Site 3 (Dx)</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Op1</td><td rowspan=1 colspan=1>Op2</td><td rowspan=1 colspan=1>Site</td><td rowspan=1 colspan=1>Op1</td><td rowspan=1 colspan=1>Op2</td><td rowspan=1 colspan=1>Site</td><td rowspan=1 colspan=1>Op1</td><td rowspan=1 colspan=1>Op2</td><td rowspan=1 colspan=1>Site</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>100%(24/24)</td><td rowspan=1 colspan=1>100%(24/24)</td><td rowspan=1 colspan=1>100%(48/48)</td><td rowspan=1 colspan=1>100%(24/24)</td><td rowspan=1 colspan=1>100%(24/24)</td><td rowspan=1 colspan=1>100%(48/48)</td><td rowspan=1 colspan=1>100%(24/24)</td><td rowspan=1 colspan=1>100%(24/24)</td><td rowspan=1 colspan=1>100%(48/48)</td><td rowspan=1 colspan=1>100%(144/144)(97.4-100)</td></tr><tr><td rowspan=1 colspan=1>SARS-CoV-2(~1.5xLoD</td><td rowspan=1 colspan=1>100%(24/24)</td><td rowspan=1 colspan=1>100%(24/24)</td><td rowspan=1 colspan=1>100%(48/48)</td><td rowspan=1 colspan=1>100%(24/24)</td><td rowspan=1 colspan=1>100%(24/24)</td><td rowspan=1 colspan=1>100%(48/48)</td><td rowspan=1 colspan=1>100%(24/24)</td><td rowspan=1 colspan=1>100%(24/24)</td><td rowspan=1 colspan=1>100%(48/48)</td><td rowspan=1 colspan=1>100%(144/144)(97.4-100)</td></tr><tr><td rowspan=1 colspan=1>SARS-CoV-2(~3xLoD</td><td rowspan=1 colspan=1>100%(24/24)</td><td rowspan=1 colspan=1>100%(24/24)</td><td rowspan=1 colspan=1>100%(48/48)</td><td rowspan=1 colspan=1>100%(24/24)</td><td rowspan=1 colspan=1>100%(24/24)</td><td rowspan=1 colspan=1>100%(48/48)</td><td rowspan=1 colspan=1>100%(24/24)</td><td rowspan=1 colspan=1>100%(24/24)</td><td rowspan=1 colspan=1>100%(48/48)</td><td rowspan=1 colspan=1>100%(144/144)(97.4-00)</td></tr><tr><td rowspan=1 colspan=1>Flu A(~1.5xLoD</td><td rowspan=1 colspan=1>100%(24/24)</td><td rowspan=1 colspan=1>100%(24/24)</td><td rowspan=1 colspan=1>100%(48/48)</td><td rowspan=1 colspan=1>100%(24/24)</td><td rowspan=1 colspan=1>100%(24/24)</td><td rowspan=1 colspan=1>100%(48/48)</td><td rowspan=1 colspan=1>100%(24/24)</td><td rowspan=1 colspan=1>100%(24/24)</td><td rowspan=1 colspan=1>100%(48/48)</td><td rowspan=1 colspan=1>100%(144/144)(97.4-100)</td></tr><tr><td rowspan=1 colspan=1>Flu A(~3xLoD</td><td rowspan=1 colspan=1>100%(24/24)</td><td rowspan=1 colspan=1>100%(24/24)</td><td rowspan=1 colspan=1>100%(48/48)</td><td rowspan=1 colspan=1>100%(24/24)</td><td rowspan=1 colspan=1>100%(24/24)</td><td rowspan=1 colspan=1>100%(48/48)</td><td rowspan=1 colspan=1>100%(24/24)</td><td rowspan=1 colspan=1>100%(24/24)</td><td rowspan=1 colspan=1>100%(48/48)</td><td rowspan=1 colspan=1>100%(144/144)(97.4-100)</td></tr><tr><td rowspan=1 colspan=1>Flu B(~1.5xLoD</td><td rowspan=1 colspan=1>100%(24/24)</td><td rowspan=1 colspan=1>100%(24/24)</td><td rowspan=1 colspan=1>100%(48/48)</td><td rowspan=1 colspan=1>95.8%(23/24)</td><td rowspan=1 colspan=1>95.8%(23/24)</td><td rowspan=1 colspan=1>95.8%(46/48)</td><td rowspan=1 colspan=1>100%(24/24)</td><td rowspan=1 colspan=1>100%(24/24)</td><td rowspan=1 colspan=1>100%(48/48)</td><td rowspan=1 colspan=1>98.6%(142/144)(95.1-99.6)</td></tr><tr><td rowspan=1 colspan=1>Flu B(~3xLoD</td><td rowspan=1 colspan=1>100%(24/24)</td><td rowspan=1 colspan=1>100%(24/24)</td><td rowspan=1 colspan=1>100%(48/48)</td><td rowspan=1 colspan=1>100%(24/24)</td><td rowspan=1 colspan=1>100%(24/24)</td><td rowspan=1 colspan=1>100%(48/48)</td><td rowspan=1 colspan=1>100%(23/23)</td><td rowspan=1 colspan=1>95.8%(23/24)</td><td rowspan=1 colspan=1>97.9%(46/47)</td><td rowspan=1 colspan=1>99.3%(142/143)a(96.1-99.9)</td></tr><tr><td rowspan=1 colspan=1>RSV B(~1.5xLoD</td><td rowspan=1 colspan=1>100%(24/24)</td><td rowspan=1 colspan=1>100%(24/24)</td><td rowspan=1 colspan=1>100%(48/48)</td><td rowspan=1 colspan=1>95.8%(23/24)</td><td rowspan=1 colspan=1>100%(24/24)</td><td rowspan=1 colspan=1>97.9%(47/48)</td><td rowspan=1 colspan=1>100%(24/24)</td><td rowspan=1 colspan=1>100%(24/24)</td><td rowspan=1 colspan=1>100%(48/48)</td><td rowspan=1 colspan=1>99.3%(143/144)(96.2-99.9)</td></tr><tr><td rowspan=1 colspan=1>RSV B(~3xLoD</td><td rowspan=1 colspan=1>100%(24/24)</td><td rowspan=1 colspan=1>100%(24/24)</td><td rowspan=1 colspan=1>100%(48/48)</td><td rowspan=1 colspan=1>100%(24/24)</td><td rowspan=1 colspan=1>100%(24/24)</td><td rowspan=1 colspan=1>100%(48/48)</td><td rowspan=1 colspan=1>100%(24/24)</td><td rowspan=1 colspan=1>100%(24/24)</td><td rowspan=1 colspan=1>100%(48/48)</td><td rowspan=1 colspan=1>100%(144/144)(97.4-100)</td></tr></table>

$\mathrm { O p } =$ Operator a One replicate was excluded because it was run on a day when an external positive control produced an incorrect result but was inadvertently not retested.

The Xpert Xpress CoV-2/Flu/RSV plus test demonstrated $100 \%$ agreement with expected results for all panel members except for the Flu B low positive $( 9 8 . 6 \% )$ , Flu B moderate positive $( 9 9 . 3 \% )$ , and the RSV B low positive $( 9 9 . 3 \% )$ . A lower percent agreement for low positive panel members was expected, since the analyte concentration of the panel member analytes was close to the limit of detection (i.e., ${ \sim } 1 . 5 \mathbf { x }$ LoD), which is expected to yield $2 9 5 \%$ detection rate. The results of the study demonstrate acceptable assay reproducibility for the Xpert Xpress CoV-2/Flu/RSV plus test.

<table><tr><td rowspan=3 colspan=1>Panel Member</td><td rowspan=3 colspan=1>Target</td><td rowspan=3 colspan=1>N</td><td rowspan=3 colspan=1>MeanCt</td><td rowspan=1 colspan=14>Variance Source</td></tr><tr><td rowspan=1 colspan=2>Site</td><td rowspan=1 colspan=2>Operator</td><td rowspan=1 colspan=2>Lot</td><td rowspan=1 colspan=2>Day</td><td rowspan=1 colspan=2>Run</td><td rowspan=1 colspan=2>Within-Run</td><td rowspan=1 colspan=2>Total</td></tr><tr><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>CV(%)</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>CV(%)</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>CV(%)</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>CV(%)</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>CV(%)</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>CV(%)</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>CV(%)</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>SPC</td><td rowspan=1 colspan=1>144</td><td rowspan=1 colspan=1>30.8</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.9</td><td rowspan=1 colspan=1>2.9</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>1.5</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>1.3</td><td rowspan=1 colspan=1>4.2</td><td rowspan=1 colspan=1>1.6</td><td rowspan=1 colspan=1>5.3</td></tr><tr><td rowspan=1 colspan=1>SARS-CoV-2Low Pos</td><td rowspan=1 colspan=1>SARS-</td><td rowspan=1 colspan=1>144</td><td rowspan=1 colspan=1>37.4</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>0.7</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>1.1</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>1.4</td></tr><tr><td rowspan=1 colspan=1>SARS-CoV-2Mod Pos</td><td rowspan=1 colspan=1>CoV-2</td><td rowspan=1 colspan=1>144</td><td rowspan=1 colspan=1>36.2</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>1.0</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>1.2</td></tr><tr><td rowspan=2 colspan=1>Flu A Low Pos</td><td rowspan=1 colspan=1>Flu A1</td><td rowspan=1 colspan=1>144</td><td rowspan=1 colspan=1>35.7</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>0.6</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.6</td><td rowspan=1 colspan=1>1.6</td><td rowspan=1 colspan=1>0.6</td><td rowspan=1 colspan=1>1.7</td></tr><tr><td rowspan=1 colspan=1>Flu A2</td><td rowspan=1 colspan=1>135b</td><td rowspan=1 colspan=1>37.9</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>0.8</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>1.1</td><td rowspan=1 colspan=1>0.9</td><td rowspan=1 colspan=1>2.5</td><td rowspan=1 colspan=1>1.1</td><td rowspan=1 colspan=1>2.9</td></tr><tr><td rowspan=2 colspan=1>Flu A Mod Posa</td><td rowspan=1 colspan=1>Flu A1</td><td rowspan=1 colspan=1>144</td><td rowspan=1 colspan=1>34.7</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>1.2</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>1.3</td></tr><tr><td rowspan=1 colspan=1>Flu A2</td><td rowspan=1 colspan=1>144</td><td rowspan=1 colspan=1>36.6</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>1.5</td><td rowspan=1 colspan=1>0.6</td><td rowspan=1 colspan=1>1.5</td></tr><tr><td rowspan=1 colspan=1>Flu B Low Pos</td><td rowspan=2 colspan=1>Flu B</td><td rowspan=1 colspan=1>144</td><td rowspan=1 colspan=1>36.3</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>0.8</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>0.7</td><td rowspan=1 colspan=1>0.7</td><td rowspan=1 colspan=1>2.1</td><td rowspan=1 colspan=1>0.8</td><td rowspan=1 colspan=1>2.3</td></tr><tr><td rowspan=1 colspan=1>Flu B Mod Pos</td><td rowspan=1 colspan=1>142c</td><td rowspan=1 colspan=1>35.1</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>0.8</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.7</td><td rowspan=1 colspan=1>2.0</td><td rowspan=1 colspan=1>0.8</td><td rowspan=1 colspan=1>2.2</td></tr><tr><td rowspan=1 colspan=1>RSV B Low Pos</td><td rowspan=2 colspan=1>RSV</td><td rowspan=1 colspan=1>144</td><td rowspan=1 colspan=1>35.8</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.6</td><td rowspan=1 colspan=1>1.7</td><td rowspan=1 colspan=1>0.6</td><td rowspan=1 colspan=1>1.8</td></tr><tr><td rowspan=1 colspan=1>RSV B Mod Pos</td><td rowspan=1 colspan=1>144</td><td rowspan=1 colspan=1>34.8</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>1.4</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>1.5</td></tr></table>

$\mathrm { C t } =$ cycle threshold; $\mathrm { C V } =$ coefficient of variation; $\mathbf { S D = }$ standard deviation a One replicate from Site 2 used the reagent lot 102 instead of the intended reagent lot 401. In the analysis, this replicate was included in the calculation for reagent lot 401. b Nine replicates were excluded due to not having Flu A2 Ct values. c One replicate was excluded due to not having a Flu B Ct value; another replicate was excluded because it was run on a day when an external positive control produced an incorrect result, but the sample was inadvertently not retested.

The mean and variability analysis between sites, operators, lots, days, runs, within-runs, and overall (total) for Ct values is shown in Table 6. Overall $\% \mathrm { C V } \ \mathrm { w a s } \leq 5 . 3 \%$ . The greatest source of variability was from the negative samples, which had a within-run $\% \mathrm { C V }$ of $4 . 2 \%$ . Overall variability was low, and the study demonstrates assay variability within an acceptable range.

2. Linearity: Not Applicable; this is a qualitative assay.

3. Analytical Specificity/Interference:

Analytical Reactivity (Inclusivity)

a. Wet-Testing

An analytical reactivity study was conducted to evaluate the ability of the Xpert Xpress CoV-2/Flu/RSV plus test to detect multiple SARS-CoV-2, influenza and RSV strains that are temporally and geographically diverse. Testing was performed on 102 different strains (18 SARS-CoV-2, 48 Flu A, 21 Flu B, and 15 RSV) in triplicate using inactivated strains for SARS-CoV-2 and cultured virus for influenza and RSV, unless noted. Each strain was spiked into negative simulated matrix at analyte concentrations of ${ \sim } 3 \mathbf { x }$ the LoD. Data supporting the use of simulated matrix can be found in section VII Performance Characteristics.B.2.Sample Matrix and Transport Media Equivalency

Study, later in this document. The influenza A testing included 20 A/H1N1 (pre-2009 seasonal and 2009 pandemic) strains, 16 A/H3N2 strains, and 12 avian strains (including the following 9 subtypes: H5N1, H5N2, H6N2, H7N2, H7N3, H2N2, H7N9, and H9N2). The 21 Influenza B strains included both the Victoria and Yamagata lineage. The 15 RSV strains included RSV types A and B.

The Xpert Xpress CoV-2/Flu/RSV plus test exhibited a broad range of reactivity, as all SARS-CoV-2, influenza A, influenza B and RSV strains tested positive in all three replicates. Results from the study are shown in Table 7.

Table 7. Inclusivity Wet-Testing Study Results   

<table><tr><td rowspan=2 colspan=1>Virus</td><td rowspan=2 colspan=1>Strain</td><td rowspan=2 colspan=1>Target Conc.</td><td rowspan=1 colspan=4>Result</td></tr><tr><td rowspan=1 colspan=1>SARS-CoV-2</td><td rowspan=1 colspan=1>FluA</td><td rowspan=1 colspan=1>Flu B</td><td rowspan=1 colspan=1>RSV</td></tr><tr><td rowspan=16 colspan=1>SARS-CoV-2</td><td rowspan=1 colspan=1>NATtrol SARS-CoV-2 USA-WA1/2020</td><td rowspan=1 colspan=1>412 copies/mL</td><td rowspan=1 colspan=1>POS</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>NEG</td></tr><tr><td rowspan=1 colspan=1>SARS-CoV-2/HongKong/VM20001061/2020</td><td rowspan=1 colspan=1>0.03 TCID50/mL</td><td rowspan=1 colspan=1>POSa</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>NEG</td></tr><tr><td rowspan=1 colspan=1>SARS-CoV-2/Italy-INM11</td><td rowspan=1 colspan=1>1 TCID50/mL</td><td rowspan=1 colspan=1>POS</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>NEG</td></tr><tr><td rowspan=1 colspan=1>SARS-CoV-2/Africa/KRISPK005325/2020 (Beta)</td><td rowspan=1 colspan=1>0.025 TCID50/mL</td><td rowspan=1 colspan=1>POS</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>NEG</td></tr><tr><td rowspan=1 colspan=1>SARS-CoV-2/England/204820464/2020</td><td rowspan=1 colspan=1>0.05 TCID50/mL</td><td rowspan=1 colspan=1>POS</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>NEG</td></tr><tr><td rowspan=1 colspan=1>SARS-CoV-2 (NY-Wadsworth-21033899-01/2021) P1_2021(Gamma)</td><td rowspan=1 colspan=1>0.01 TCID50/mL</td><td rowspan=1 colspan=1>POS</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>NEG</td></tr><tr><td rowspan=1 colspan=1>SARS-CoV-2 (NY-Wadsworth-21006055-01/2021) P2_2021(Zeta)</td><td rowspan=1 colspan=1>0.03 TCID50/mL</td><td rowspan=1 colspan=1>POS</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>NEG</td></tr><tr><td rowspan=1 colspan=1>SARS-CoV-2 (NYWadsworth-21025952-01/2021) B.1.526_2021(Iota)</td><td rowspan=1 colspan=1>0.1 TCID50/mL</td><td rowspan=1 colspan=1>POS</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>NEG</td></tr><tr><td rowspan=1 colspan=1>SARS-CoV-2 (NY-Wadsworth-103677-01/2020) B.1 2020</td><td rowspan=1 colspan=1>0.003 TCID50/mL</td><td rowspan=1 colspan=1>POS</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>NEG</td></tr><tr><td rowspan=1 colspan=1>SARS-CoV-2 (NY-Wadsworth-33126-01/2020) B.1.595 2020</td><td rowspan=1 colspan=1>0.0015TCID50/mL</td><td rowspan=1 colspan=1>POS</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>NEG</td></tr><tr><td rowspan=1 colspan=1>SARS-CoV-2 (USA/CA-Stanford-15 S02/2021) B.1.617.1 (Kappa)</td><td rowspan=1 colspan=1>1.7 TCID50/mL</td><td rowspan=1 colspan=1>POS</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>NEG</td></tr><tr><td rowspan=1 colspan=1>SARS-CoV-2(USA/PHC658/2021) B.1.617.2(Delta)</td><td rowspan=1 colspan=1>0.01 TCID50/mL</td><td rowspan=1 colspan=1>POS</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>NEG</td></tr><tr><td rowspan=1 colspan=1>SARS-CoV-2 (USA/MDHP01542/2021) B.1.351 (Beta)</td><td rowspan=1 colspan=1>100 (genomeequivalents/mL)</td><td rowspan=1 colspan=1>POS</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>NEG</td></tr><tr><td rowspan=1 colspan=1>SARS-CoV-2 (USA/GA-EHC-2811C/20221) B.1.1.529(Omicron)</td><td rowspan=1 colspan=1>100 (genomeequivalents/mL)</td><td rowspan=1 colspan=1>POS</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>NEG</td></tr><tr><td rowspan=1 colspan=1>SARS-CoV-2 RNA,USA/WA2/2020 (C09)b</td><td rowspan=1 colspan=1>100 copies/mL</td><td rowspan=1 colspan=1>POS</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>NEG</td></tr><tr><td rowspan=1 colspan=1>SARS-CoV-2RNA,England/205041766/2020(C14) (alpha)b</td><td rowspan=1 colspan=1>100 copies/mL</td><td rowspan=1 colspan=1>POS</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>NEG</td></tr></table>

<table><tr><td rowspan=2 colspan=2>Virus</td><td rowspan=2 colspan=1>Strain</td><td rowspan=2 colspan=1>Target Conc.</td><td rowspan=1 colspan=4>Result</td></tr><tr><td rowspan=1 colspan=1>SARS-ov-2</td><td rowspan=1 colspan=1>FluA</td><td rowspan=1 colspan=1>Flu B</td><td rowspan=1 colspan=1>RSV</td></tr><tr><td rowspan=3 colspan=2></td><td rowspan=1 colspan=1>SARS-CoV-2 RNA,England/MILK-9E05B3/2020(C15) (alpha)b</td><td rowspan=1 colspan=1>200 copies/mL</td><td rowspan=1 colspan=1>POS</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>NEG</td></tr><tr><td rowspan=1 colspan=1>SARS-CoV-2 RNA /Japan(Brazil)/IC-0564/2021 (C17)(gaamma)</td><td rowspan=1 colspan=1>100 copies/mL</td><td rowspan=1 colspan=1>POS</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>NEG</td></tr><tr><td rowspan=1 colspan=1>EY</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=13 colspan=2>Flu A H1N1(pre-2009)</td><td rowspan=1 colspan=1>A/swine/Iowa/15/30</td><td rowspan=1 colspan=1>10 TCID50/mL</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>POS</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>NEG</td></tr><tr><td rowspan=1 colspan=1>A/WS/33</td><td rowspan=1 colspan=1>0.6 CEID50/mL</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>POS</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>NEG</td></tr><tr><td rowspan=1 colspan=1>A/PR/8/34</td><td rowspan=1 colspan=1>1.25 CEID50/mL</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>POS</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>NEG</td></tr><tr><td rowspan=1 colspan=1>A/Mal/302/54</td><td rowspan=1 colspan=1>0.156 CEID50/mL</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>POS</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>NEG</td></tr><tr><td rowspan=1 colspan=1>A/Denver/1/57</td><td rowspan=1 colspan=1>1.5 CEID50/mL</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>POS</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>NEG</td></tr><tr><td rowspan=1 colspan=1>A/New Jersey/8/76</td><td rowspan=1 colspan=1>5 CEID50/mL</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>POS</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>NEG</td></tr><tr><td rowspan=1 colspan=1>A/New Caledonia/20/1999</td><td rowspan=1 colspan=1>0.10 TCID50/mL</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>POS</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>NEG</td></tr><tr><td rowspan=1 colspan=1>A/New York/55/2004</td><td rowspan=1 colspan=1>9 TCID50/mL</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>POS</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>NEG</td></tr><tr><td rowspan=1 colspan=1>A/Solomon Island/3/2006</td><td rowspan=1 colspan=1>0.0159TCID50/mL</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>POS</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>NEG</td></tr><tr><td rowspan=1 colspan=1>A/Taiwan/42/06</td><td rowspan=1 colspan=1>0.002 TCID50/mL</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>POS</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>NEG</td></tr><tr><td rowspan=1 colspan=1>A/Brisbane/59/2007</td><td rowspan=1 colspan=1>0.008 TCID50/mL</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>POS</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>NEG</td></tr><tr><td rowspan=1 colspan=1>A/Swine/NY/02/2009</td><td rowspan=1 colspan=1>3.2 TCID50/mL</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>POS</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>NEG</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=26 colspan=2>Flu A H1N1(pdm 2009)Flu AH3N2</td><td rowspan=1 colspan=1>A/Colorado/14/2012</td><td rowspan=1 colspan=1>0.04 TCID50/mL</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>POS</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>NEG</td></tr><tr><td rowspan=1 colspan=1>A/Michigan/45/2015</td><td rowspan=1 colspan=1>15 CEID50/mL</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>POS</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>NEG</td></tr><tr><td rowspan=1 colspan=1>A/Iowa/53/2015</td><td rowspan=1 colspan=1>6 CEID50/mL</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>POS</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>NEG</td></tr><tr><td rowspan=1 colspan=1>A/Michigan/272/2017</td><td rowspan=1 colspan=1>0.07 TCID50/mL</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>POS</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>NEG</td></tr><tr><td rowspan=1 colspan=1>A/Idaho/07/2018</td><td rowspan=1 colspan=1>0.0159TCID50/mL</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>POS</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>NEG</td></tr><tr><td rowspan=1 colspan=1>A/Wisconsin/505/2018</td><td rowspan=1 colspan=1>0.08 TCID50/mL</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>POS</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>NEG</td></tr><tr><td rowspan=1 colspan=1>A/Hawaii/66/2019</td><td rowspan=1 colspan=1>100 CEID50/mL</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>POS</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>NEG</td></tr><tr><td rowspan=1 colspan=1>A/Indiana/02/2020</td><td rowspan=1 colspan=1>NAc</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>POS</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>NEG</td></tr><tr><td rowspan=1 colspan=1>EY</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>A/Aichi/2/68</td><td rowspan=1 colspan=1>2 CEID50/mL</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>POS</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>NEG</td></tr><tr><td rowspan=1 colspan=1>A/Hong Kong/8/68</td><td rowspan=1 colspan=1>0.25 CEID50/mL</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>POS</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>NEG</td></tr><tr><td rowspan=1 colspan=1>A/Port Chalmers/1/73</td><td rowspan=1 colspan=1>8 CEID50/mL</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>POS</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>NEG</td></tr><tr><td rowspan=1 colspan=1>A/Hawaii/15/2001</td><td rowspan=1 colspan=1>33 CEID50/mL</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>POS</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>NEG</td></tr><tr><td rowspan=1 colspan=1>A/Wisconsin/67/05c</td><td rowspan=1 colspan=1>0.22 TCID50/mL</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>POS</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>NEG</td></tr><tr><td rowspan=1 colspan=1>A/Brisbane/10/2007</td><td rowspan=1 colspan=1>0.003 TCID50/mL</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>POS</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>NEG</td></tr><tr><td rowspan=1 colspan=1>A/Minnesota/11/2010</td><td rowspan=1 colspan=1>2.4 CEID50/mL</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>POS</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>NEG</td></tr><tr><td rowspan=1 colspan=1>A/Indiana/08/2011</td><td rowspan=1 colspan=1>0.02 TCID50/mL</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>POS</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>NEG</td></tr><tr><td rowspan=1 colspan=1>A/Texas/50/2012</td><td rowspan=1 colspan=1>0.008 TCID50/mL</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>POS</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>NEG</td></tr><tr><td rowspan=1 colspan=1>A/Alaska/232/2015</td><td rowspan=1 colspan=1>2 CEID50/mL</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>POS</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>NEG</td></tr><tr><td rowspan=1 colspan=1>A/Singapore/INFIMH-16-0019/2016</td><td rowspan=1 colspan=1>2.5 CEID50/mL</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>POS</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>NEG</td></tr><tr><td rowspan=1 colspan=1>A/Texas/71/2017</td><td rowspan=1 colspan=1>1 FFU/mL</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>POS</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>NEG</td></tr><tr><td rowspan=1 colspan=1>A/Kansas/14/2017</td><td rowspan=1 colspan=1>0.15 FFU/mL</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>POS</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>NEG</td></tr><tr><td rowspan=1 colspan=1>A/Wisconsin/04/2018d</td><td rowspan=1 colspan=1>0.15 FFU/mL</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>POS</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>NEG</td></tr><tr><td rowspan=1 colspan=1>A/Arizona/45/2018</td><td rowspan=1 colspan=1>2 FFU/mL</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>POS</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>NEG</td></tr><tr><td rowspan=1 colspan=1>A/Hong Kong/45/2019</td><td rowspan=1 colspan=1>0.8 FFU/mL</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>POS</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>NEG</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=4 colspan=2>AvianFlu A</td><td rowspan=1 colspan=1>A/Mallard/NY/6750/78 (H2N2)</td><td rowspan=1 colspan=1>&lt; 1 pg/uL</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>POS</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>NEG</td></tr><tr><td rowspan=1 colspan=1>A/duck/Hunan/795/2002 (H5N1)</td><td rowspan=1 colspan=1>&lt; 1 pg/uL</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>POS</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>NEG</td></tr><tr><td rowspan=1 colspan=1>A/Vietnam/1194/2004 (H5N1)</td><td rowspan=1 colspan=1>&lt; 1 pg/uL</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>POS</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>NEG</td></tr><tr><td rowspan=1 colspan=1>A/Anhui/01/2005 (H5N1)</td><td rowspan=1 colspan=1>&lt; 1 pg/uL</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>POS</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>NEG</td></tr></table>

<table><tr><td colspan="1" rowspan="2">Virus</td><td colspan="1" rowspan="2">Strain</td><td colspan="1" rowspan="2">Target Conc.</td><td colspan="4" rowspan="1">Result</td></tr><tr><td colspan="1" rowspan="1">SARS-Cv-2</td><td colspan="1" rowspan="1">FluA</td><td colspan="1" rowspan="1">Flu B</td><td colspan="1" rowspan="1">RSV</td></tr><tr><td colspan="1" rowspan="9"></td><td colspan="1" rowspan="1">A/Japanese white eye/HongKong/1038/2006 (H5N1)</td><td colspan="1" rowspan="1">&lt; 1 pg/uL</td><td colspan="1" rowspan="1">NEG</td><td colspan="1" rowspan="1">POS</td><td colspan="1" rowspan="1">NEG</td><td colspan="1" rowspan="1">NEG</td></tr><tr><td colspan="1" rowspan="1">A/mallard/WI/34/75 (H5N2)</td><td colspan="1" rowspan="1">&lt; 1 pg/uL</td><td colspan="1" rowspan="1">NEG</td><td colspan="1" rowspan="1">POS</td><td colspan="1" rowspan="1">NEG</td><td colspan="1" rowspan="1">NEG</td></tr><tr><td colspan="1" rowspan="1">A/turkey/Massachusetts/3740/1965 (H6N2)</td><td colspan="1" rowspan="1">0.1 fg/uL</td><td colspan="1" rowspan="1">NEG</td><td colspan="1" rowspan="1">POS</td><td colspan="1" rowspan="1">NEG</td><td colspan="1" rowspan="1">NEG</td></tr><tr><td colspan="1" rowspan="1">A/duck/LTC-10-82743 (H7N2)</td><td colspan="1" rowspan="1">5 fg/uL</td><td colspan="1" rowspan="1">NEG</td><td colspan="1" rowspan="1">POS</td><td colspan="1" rowspan="1">NEG</td><td colspan="1" rowspan="1">NEG</td></tr><tr><td colspan="1" rowspan="1">A/chicken/New Jersey/15086/3 7N</td><td colspan="1" rowspan="1">4 fg/uL</td><td colspan="1" rowspan="1">NEG</td><td colspan="1" rowspan="1">POS</td><td colspan="1" rowspan="1">NEG</td><td colspan="1" rowspan="1">NEG</td></tr><tr><td colspan="1" rowspan="1">A/Anhui/1/2013 (H7N9)</td><td colspan="1" rowspan="1">0.612 ng/uL</td><td colspan="1" rowspan="1">NEG</td><td colspan="1" rowspan="1">POS</td><td colspan="1" rowspan="1">NEG</td><td colspan="1" rowspan="1">NEG</td></tr><tr><td colspan="1" rowspan="1">A/Shanghai/1/2013 (H7N9)</td><td colspan="1" rowspan="1">NAf</td><td colspan="1" rowspan="1">NEG</td><td colspan="1" rowspan="1">POS</td><td colspan="1" rowspan="1">NEG</td><td colspan="1" rowspan="1">NEG</td></tr><tr><td colspan="1" rowspan="1">A/chicken/New Jersey/12220/1997(H9N2)</td><td colspan="1" rowspan="1">0.05 pg/uL</td><td colspan="1" rowspan="1">NEG</td><td colspan="1" rowspan="1">POS</td><td colspan="1" rowspan="1">NEG</td><td colspan="1" rowspan="1">NEG</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="6">Flu B</td><td colspan="1" rowspan="1">B/Lee/40</td><td colspan="1" rowspan="1">0.08 PFU/mL</td><td colspan="1" rowspan="1">NEG</td><td colspan="1" rowspan="1">NEG</td><td colspan="1" rowspan="1">POS</td><td colspan="1" rowspan="1">NEG</td></tr><tr><td colspan="1" rowspan="1">B/Allen/45</td><td colspan="1" rowspan="1">0.25 CEID50/mL</td><td colspan="1" rowspan="1">NEG</td><td colspan="1" rowspan="1">NEG</td><td colspan="1" rowspan="1">POS</td><td colspan="1" rowspan="1">NEG</td></tr><tr><td colspan="1" rowspan="1">B/GL/1739/54</td><td colspan="1" rowspan="1">0.50 CEID50/mL</td><td colspan="1" rowspan="1">NEG</td><td colspan="1" rowspan="1">NEG</td><td colspan="1" rowspan="1">POS</td><td colspan="1" rowspan="1">NEG</td></tr><tr><td colspan="1" rowspan="1">B/Maryland/1/59</td><td colspan="1" rowspan="1">0.2 CEID50/mL</td><td colspan="1" rowspan="1">NEG</td><td colspan="1" rowspan="1">NEG</td><td colspan="1" rowspan="1">POS</td><td colspan="1" rowspan="1">NEG</td></tr><tr><td colspan="1" rowspan="1">B/Taiwan/2/62</td><td colspan="1" rowspan="1">0.7 CEID50/mL</td><td colspan="1" rowspan="1">NEG</td><td colspan="1" rowspan="1">NEG</td><td colspan="1" rowspan="1">POS</td><td colspan="1" rowspan="1">NEG</td></tr><tr><td colspan="1" rowspan="1">B/Hong Kong/5/72</td><td colspan="1" rowspan="1">1 CEID50/mL</td><td colspan="1" rowspan="1">NEG</td><td colspan="1" rowspan="1">NEG</td><td colspan="1" rowspan="1">POS</td><td colspan="1" rowspan="1">NEG</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">E</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="9">Flu B (VictoriaLineage</td><td colspan="1" rowspan="1">B/Panama/45/90</td><td colspan="1" rowspan="1">0.125 TCID50/mL</td><td colspan="1" rowspan="1">NEG</td><td colspan="1" rowspan="1">NEG</td><td colspan="1" rowspan="1">POS</td><td colspan="1" rowspan="1">NEG</td></tr><tr><td colspan="1" rowspan="1">B/Malaysia/2506/04</td><td colspan="1" rowspan="1">0.001 TCID50/mL</td><td colspan="1" rowspan="1">NEG</td><td colspan="1" rowspan="1">NEG</td><td colspan="1" rowspan="1">POS</td><td colspan="1" rowspan="1">NEG</td></tr><tr><td colspan="1" rowspan="1">B/Florida/02/06</td><td colspan="1" rowspan="1">0.004 TCID50/mL</td><td colspan="1" rowspan="1">NEG</td><td colspan="1" rowspan="1">NEG</td><td colspan="1" rowspan="1">POS</td><td colspan="1" rowspan="1">NEG</td></tr><tr><td colspan="1" rowspan="1">B/Brisbane/60/2008</td><td colspan="1" rowspan="1">0.005 TCID50/mL</td><td colspan="1" rowspan="1">NEG</td><td colspan="1" rowspan="1">NEG</td><td colspan="1" rowspan="1">POS</td><td colspan="1" rowspan="1">NEG</td></tr><tr><td colspan="1" rowspan="1">B/Maryland/15/2016</td><td colspan="1" rowspan="1">0.06 TCID50/mL</td><td colspan="1" rowspan="1">NEG</td><td colspan="1" rowspan="1">NEG</td><td colspan="1" rowspan="1">POS</td><td colspan="1" rowspan="1">NEG</td></tr><tr><td colspan="1" rowspan="1">B/Colorado/6/2017</td><td colspan="1" rowspan="1">0.01 TCID50/mL</td><td colspan="1" rowspan="1">NEG</td><td colspan="1" rowspan="1">NEG</td><td colspan="1" rowspan="1">POS</td><td colspan="1" rowspan="1">NEG</td></tr><tr><td colspan="1" rowspan="1">B/Hawaii/01/2018</td><td colspan="1" rowspan="1">1 TCID50/mL</td><td colspan="1" rowspan="1">NEG</td><td colspan="1" rowspan="1">NEG</td><td colspan="1" rowspan="1">POS</td><td colspan="1" rowspan="1">NEG</td></tr><tr><td colspan="1" rowspan="1">B/Missouri/12/2018 (NA D197E)</td><td colspan="1" rowspan="1">1.2 TCID50/mL</td><td colspan="1" rowspan="1">NEG</td><td colspan="1" rowspan="1">NEG</td><td colspan="1" rowspan="1">POS</td><td colspan="1" rowspan="1">NEG</td></tr><tr><td colspan="1" rowspan="1">B/Washington/02/2019</td><td colspan="1" rowspan="1">60 TCID50/mL</td><td colspan="1" rowspan="1">NEG</td><td colspan="1" rowspan="1">NEG</td><td colspan="1" rowspan="1">POS</td><td colspan="1" rowspan="1">NEG</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="6">Flu BYamagataineage</td><td colspan="1" rowspan="1">B/Florida/07/2004</td><td colspan="1" rowspan="1">0.03 TCID50/mL</td><td colspan="1" rowspan="1">NEG</td><td colspan="1" rowspan="1">NEG</td><td colspan="1" rowspan="1">POS</td><td colspan="1" rowspan="1">NEG</td></tr><tr><td colspan="1" rowspan="1">B/Florida/04/06</td><td colspan="1" rowspan="1">0.03 TCID50/mL</td><td colspan="1" rowspan="1">NEG</td><td colspan="1" rowspan="1">NEG</td><td colspan="1" rowspan="1">POS</td><td colspan="1" rowspan="1">NEG</td></tr><tr><td colspan="1" rowspan="1">B/Wisconsin/01/2010</td><td colspan="1" rowspan="1">0.025 CEID50/mL</td><td colspan="1" rowspan="1">NEG</td><td colspan="1" rowspan="1">NEG</td><td colspan="1" rowspan="1">POS</td><td colspan="1" rowspan="1">NEG</td></tr><tr><td colspan="1" rowspan="1">B/Wisconsin/10/2016</td><td colspan="1" rowspan="1">2 TCID50/mL</td><td colspan="1" rowspan="1">NEG</td><td colspan="1" rowspan="1">NEG</td><td colspan="1" rowspan="1">POS</td><td colspan="1" rowspan="1">NEG</td></tr><tr><td colspan="1" rowspan="1">B/Indiana/17/2017</td><td colspan="1" rowspan="1">0.5 TCID50/mL</td><td colspan="1" rowspan="1">NEG</td><td colspan="1" rowspan="1">NEG</td><td colspan="1" rowspan="1">POS</td><td colspan="1" rowspan="1">NEG</td></tr><tr><td colspan="1" rowspan="1">B/Oklahoma/10/2018</td><td colspan="1" rowspan="1">1 TCID50/mL</td><td colspan="1" rowspan="1">NEG</td><td colspan="1" rowspan="1">NEG</td><td colspan="1" rowspan="1">POS</td><td colspan="1" rowspan="1">NEG</td></tr><tr><td colspan="1" rowspan="14">RSV ARSV B</td><td colspan="1" rowspan="1">E</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">RSV-A/NY</td><td colspan="1" rowspan="1">0.386 TCID50/mL</td><td colspan="1" rowspan="1">NEG</td><td colspan="1" rowspan="1">NEG</td><td colspan="1" rowspan="1">NEG</td><td colspan="1" rowspan="1">POS</td></tr><tr><td colspan="1" rowspan="1">RSV-A/WI-629.8.2/2007</td><td colspan="1" rowspan="1">0.50 TCID50/mL</td><td colspan="1" rowspan="1">NEG</td><td colspan="1" rowspan="1">NEG</td><td colspan="1" rowspan="1">NEG</td><td colspan="1" rowspan="1">POS</td></tr><tr><td colspan="1" rowspan="1">RSV-A/WI/629-11-1 2008</td><td colspan="1" rowspan="1">0.50 TCID50/mL</td><td colspan="1" rowspan="1">NEG</td><td colspan="1" rowspan="1">NEG</td><td colspan="1" rowspan="1">NEG</td><td colspan="1" rowspan="1">POS</td></tr><tr><td colspan="1" rowspan="1">RSV-A, Strain: 4/2015 Isolate #1</td><td colspan="1" rowspan="1">0.03 TCID50/mL</td><td colspan="1" rowspan="1">NEG</td><td colspan="1" rowspan="1">NEG</td><td colspan="1" rowspan="1">NEG</td><td colspan="1" rowspan="1">POS</td></tr><tr><td colspan="1" rowspan="1">RSV-A (2014, Isolate 342)</td><td colspan="1" rowspan="1">0.38 IU/mL</td><td colspan="1" rowspan="1">NEG</td><td colspan="1" rowspan="1">NEG</td><td colspan="1" rowspan="1">NEG</td><td colspan="1" rowspan="1">POS</td></tr><tr><td colspan="1" rowspan="1">RSV-A (A2 cpts-248 mutant)</td><td colspan="1" rowspan="1">1600 copies/mL</td><td colspan="1" rowspan="1">NEG</td><td colspan="1" rowspan="1">NEG</td><td colspan="1" rowspan="1">NEG</td><td colspan="1" rowspan="1">POS</td></tr><tr><td colspan="1" rowspan="1">RSV-A (2000/3-4)</td><td colspan="1" rowspan="1">0.0015TCID50/mL</td><td colspan="1" rowspan="1">NEG</td><td colspan="1" rowspan="1">NEG</td><td colspan="1" rowspan="1">NEG</td><td colspan="1" rowspan="1">POS</td></tr><tr><td colspan="1" rowspan="1">RSV-A (2001/3-12)</td><td colspan="1" rowspan="1">0.28 TCID50/mL</td><td colspan="1" rowspan="1">NEG</td><td colspan="1" rowspan="1">NEG</td><td colspan="1" rowspan="1">NEG</td><td colspan="1" rowspan="1">POS</td></tr><tr><td colspan="1" rowspan="1">RSV-A (1997/12-35)</td><td colspan="1" rowspan="1">0.5 TCID50/mL</td><td colspan="1" rowspan="1">NEG</td><td colspan="1" rowspan="1">NEG</td><td colspan="1" rowspan="1">NEG</td><td colspan="1" rowspan="1">POS</td></tr><tr><td colspan="1" rowspan="1">RSV-A (Homosapiens/ARG/177/2006)</td><td colspan="1" rowspan="1">0.089 TCID50/mL</td><td colspan="1" rowspan="1">NEG</td><td colspan="1" rowspan="1">NEG</td><td colspan="1" rowspan="1">NEG</td><td colspan="1" rowspan="1">POS</td></tr><tr><td colspan="1" rowspan="1">RSV-A (1998/3-2)</td><td colspan="1" rowspan="1">0.0089TCID50/mL</td><td colspan="1" rowspan="1">NEG</td><td colspan="1" rowspan="1">NEG</td><td colspan="1" rowspan="1">NEG</td><td colspan="1" rowspan="1">POS</td></tr><tr><td colspan="1" rowspan="1">E</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">RSV-B/WV14617/85</td><td colspan="1" rowspan="1">0.04 TCID50/mL</td><td colspan="1" rowspan="1">NEG</td><td colspan="1" rowspan="1">NEG</td><td colspan="1" rowspan="1">NEG</td><td colspan="1" rowspan="1">POS</td></tr><tr><td colspan="1" rowspan="1">SARS-CoV-2</td><td colspan="1" rowspan="1">FluA</td><td colspan="1" rowspan="1">Flu B</td><td colspan="1" rowspan="1">RSV</td></tr><tr><td colspan="1" rowspan="3"></td><td colspan="1" rowspan="1">RSV-B-CH93(18)-18-01</td><td colspan="1" rowspan="1">0.004 TCID50/mL</td><td colspan="1" rowspan="1">NEG</td><td colspan="1" rowspan="1">NEG</td><td colspan="1" rowspan="1">NEG</td><td colspan="1" rowspan="1">POS</td></tr><tr><td colspan="1" rowspan="1">RSV-B (12/2014, Isolate #1)</td><td colspan="1" rowspan="1">0.008 TCID50/mL</td><td colspan="1" rowspan="1">NEG</td><td colspan="1" rowspan="1">NEG</td><td colspan="1" rowspan="1">NEG</td><td colspan="1" rowspan="1">POS</td></tr><tr><td colspan="1" rowspan="1">RSV-B (cp23 Clone 1A2)</td><td colspan="1" rowspan="1">4200 copies/mL</td><td colspan="1" rowspan="1">NEG</td><td colspan="1" rowspan="1">NEG</td><td colspan="1" rowspan="1">NEG</td><td colspan="1" rowspan="1">POS</td></tr></table>

a One of three replicates was Invalid. The run was successfully repeated to obtain three valid replicates. b In vitro transcripts from Twist Biosciences.

c Influenza A/Indiana/02/2020 virus was without titer and the stock was diluted 48,000-fold in simulated matrix for testing.

d One of three replicates yielded an ERROR result. The run was successfully repeated to obtain three valid replicates.

Purified viral RNA in TE and diluted in simulated matrix was tested due to biosafety regulations.

Inactivated avian influenza A (H7N9) viral RNA without viral titer was diluted 100,000-fold in simulated matrix for testing due to biosafety regulations.

# b. SARS-CoV-2 In silico Analysis

The inclusivity of the Xpert Xpress CoV-2/Flu/RSV plus test was evaluated using in silico analyses of assay amplicons in relation to the SARS-CoV-2 sequences available in the GISAID EpiCoV database as of June 15, 2022. For this analysis, the sequences were separated into lineages of interest based on the Pango Lineage assigned to each genome by GISAID, and those with ambiguous nucleotides were removed. BLAST (Basic Local Alignment Search Tool) was used to compare similarity between the amplicon sequences and the sequences in GISAID. The inclusivity analyses focused on the combined, nonambiguous sequences from variants of concern (VOC) or variants of interest (VOI) that may have important epidemiological, immunological, or pathogenic properties from a public health perspective, including Delta, Alpha, Omicron BA.1, Omicron BA.2, Omicron BA.2.12.1, Omicron BA.3, Omicron BA.4, Omicron BA.5, Eta, Iota, Kappa, Mu, Lambda, Zeta, and Theta, amongst others.

The evaluation included the following number of sequences per SARS-CoV-2 target: 10,310,839 sequences for the E target, 10,428,014 sequences for the N2 target, and 10,178,602 sequences for the RdRp target. The results of this analysis are shown in Table 8, stratified by the number of nucleotide mismatches between the amplicon and evaluated sequences.

Table 8. Results of SARS-CoV-2 In Silico Analyses   

<table><tr><td rowspan=1 colspan=1>Amplicon</td><td rowspan=1 colspan=1>Exact Match</td><td rowspan=1 colspan=1>1 Mismatch</td><td rowspan=1 colspan=1>2 or MoreMismatches</td><td rowspan=1 colspan=1>% Total &lt;2Mismatches</td></tr><tr><td rowspan=1 colspan=1>E gene target</td><td rowspan=1 colspan=1>99.5%(10,262,080 of10,310,839)</td><td rowspan=1 colspan=1>0.5%(47,959 of10,310,839)</td><td rowspan=1 colspan=1>0.01%(800 of10,310,839)</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>N2 gene target</td><td rowspan=1 colspan=1>98.1%(10,228,739 of10,428,014)</td><td rowspan=1 colspan=1>1.9%(194,319 of10,428,014)</td><td rowspan=1 colspan=1>0.05%(4,956 of10,428,014)</td><td rowspan=1 colspan=1>99.9%</td></tr><tr><td rowspan=1 colspan=1>RdRp gene target</td><td rowspan=1 colspan=1>99.2%(10,092,873 of10,178,602)</td><td rowspan=1 colspan=1>0.8%(84,595 of10,178,602)</td><td rowspan=1 colspan=1>0.01%(1,134 of10,178,602)</td><td rowspan=1 colspan=1>100%</td></tr></table>

Based on the built-in redundancy of the Xpert Xpress CoV-2/Flu/RSV plus test’s SARS$\mathrm { C o V - } 2$ amplification system (i.e., 3 independent targets, only 1 of 3 must be detected to assign a positive result), it is not anticipated that any of the evaluated SARS-CoV-2 sequences would be missed by the Xpert Xpress CoV-2/Flu/RSV plus test.

Cross-Reactivity/Microbial Interference

# a. Wet-Testing

# i. Cross-Reactivity

The analytical specificity (cross-reactivity) of the Xpert Xpress CoV-2/Flu/RSV plus test was evaluated by testing a panel of non-targeted microorganisms that may be found in a respiratory tract clinical specimen. Forty-eight (48) non-target microorganisms (Table 9) were evaluated in the study. Panels were composed of 1 to 8 different non-target microorganisms spiked into simulated matrix at ${ \ge } \bar { 1 } \mathrm { x } 1 0 ^ { 5 }$ $\mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ (for viruses) and $\geq 1 \mathrm { x } 1 0 ^ { 6 } \mathrm { C F U / m L }$ (for bacteria and fungi), with the following exceptions: A live strain of Lactobacillus reutri was unavailable and therefore genomic DNA was tested at ${ \geq } 1 \mathbf { x } 1 0 ^ { 6 }$ copies/mL; similarly, whole, enveloped human coronavirus HKU1 was unavailable and therefore synthetic RNA was tested at ${ \geq } 1 \mathbf { x } 1 0 ^ { 6 }$ copies/mL. To evaluate cross-reactivity, each panel was tested in triplicate in the absence of the target microorganisms. No cross-reactivity was observed at the concentrations tested.

Table 9. Non-Target Microorganisms Evaluated During Cross-Reactivity Testing   

<table><tr><td rowspan=1 colspan=1>Microorganism</td><td rowspan=1 colspan=1>Concentration</td><td rowspan=1 colspan=1>Microorganism</td><td rowspan=1 colspan=1>Concentration</td></tr><tr><td rowspan=1 colspan=2>Viruses</td><td rowspan=1 colspan=2>Bacteria</td></tr><tr><td rowspan=1 colspan=1>Human coronavirus NL63</td><td rowspan=1 colspan=1>1.17x105 TCID50/mL</td><td rowspan=1 colspan=1>Acinetobacter baumannii</td><td rowspan=1 colspan=1>1.30x107 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>MERS-coronavirus</td><td rowspan=1 colspan=1>1.17x105 TCID50/mL</td><td rowspan=1 colspan=1>Bordetella pertussis</td><td rowspan=1 colspan=1>6.40x107 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Human coronavirus 229E</td><td rowspan=1 colspan=1>1.21x105 TCID50/mL</td><td rowspan=1 colspan=1>Burkholderia cepacia</td><td rowspan=1 colspan=1>1.90x10° CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Human coronavirus OC43</td><td rowspan=1 colspan=1>1.02x105 TCID50/mL</td><td rowspan=1 colspan=1>Citrobacter freundii</td><td rowspan=1 colspan=1>1.73x10° CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Human coronavirus HKU1a</td><td rowspan=1 colspan=1>1.23x106 copies/mL</td><td rowspan=1 colspan=1>Corynebacterium sp.</td><td rowspan=1 colspan=1>1.27x107 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Adenovirus Type 1</td><td rowspan=1 colspan=1>4.07x105 TCID50/mL</td><td rowspan=1 colspan=1>Enterococcus faecalis</td><td rowspan=1 colspan=1>5.87x107 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Adenovirus Type 7</td><td rowspan=1 colspan=1>1.15x105 TCID50/mL</td><td rowspan=1 colspan=1>Escherichia coli</td><td rowspan=1 colspan=1>1.55x108 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Cytomegalovirus</td><td rowspan=1 colspan=1>1.00x105 TCID50/mL</td><td rowspan=1 colspan=1>Hemophilus influenzae</td><td rowspan=1 colspan=1>6.62x106 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Echovirus</td><td rowspan=1 colspan=1>1.14x105 TCID50/mL</td><td rowspan=1 colspan=1>Lactobacillus reuterib</td><td rowspan=1 colspan=1>5.0x107 copies/mL</td></tr><tr><td rowspan=1 colspan=1>Enterovirus</td><td rowspan=1 colspan=1>2.80x105 TCID50/mL</td><td rowspan=1 colspan=1>Legionella pneumophila</td><td rowspan=1 colspan=1>1.42x108 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Epstein Barr Virus</td><td rowspan=1 colspan=1>5.60x106 TCID50/mL</td><td rowspan=1 colspan=1>Moraxella catarrhalis</td><td rowspan=1 colspan=1>2.46x106 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>HSV</td><td rowspan=1 colspan=1>1.97x105 TCID50/mL</td><td rowspan=1 colspan=1>Mycoplasma pneumoniae</td><td rowspan=1 colspan=1>2.70x106 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Human metapneumovirus</td><td rowspan=1 colspan=1>4.07x105 TCID50/mL</td><td rowspan=1 colspan=1>Neisseria meningitides</td><td rowspan=1 colspan=1>4.20x106 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Human parainfluenza Type 1</td><td rowspan=1 colspan=1>1.0x105 TCID50/mL</td><td rowspan=1 colspan=1>Neisseria mucosa</td><td rowspan=1 colspan=1>1.00x108 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Human parainfluenza Type 2</td><td rowspan=1 colspan=1>1.2x105 TCID50/mL</td><td rowspan=1 colspan=1>Propionibacterium acnes</td><td rowspan=1 colspan=1>8.25x107 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Human parainfluenza Type 3</td><td rowspan=1 colspan=1>1.2x105 TCID50/mL</td><td rowspan=1 colspan=1>Pseudomonas aeruginosa</td><td rowspan=1 colspan=1>1.05x107 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Human parainfluenza Type 4</td><td rowspan=1 colspan=1>1.19x106 TCID50/mL</td><td rowspan=1 colspan=1>Staphylococcus haemolyticus</td><td rowspan=1 colspan=1>2.66x106 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Measles</td><td rowspan=1 colspan=1>1.20x105 TCID50/mL</td><td rowspan=1 colspan=1>Staphylococcus aureus</td><td rowspan=1 colspan=1>5.87x107 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Mumps virus</td><td rowspan=1 colspan=1>1.20x105 TCID50/mL</td><td rowspan=1 colspan=1>Staphylococcus epidermidis</td><td rowspan=1 colspan=1>2.47x107 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Rhinovirus Type 1A</td><td rowspan=1 colspan=1>1.00x105 TCID50/mL</td><td rowspan=1 colspan=1>Streptococcus agalactiae</td><td rowspan=1 colspan=1>1.75x107 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Fungi</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Streptococcus pneumoniae</td><td rowspan=1 colspan=1>2.26x107CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Candida albicans</td><td rowspan=1 colspan=1>6.30x106 CFU/mL</td><td rowspan=1 colspan=1>Streptococcus pyogenes</td><td rowspan=1 colspan=1>9.00x106 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Candida parapsilosis</td><td rowspan=1 colspan=1>1.45x106 CFU/mL</td><td rowspan=1 colspan=1>Streptococcus salivarius</td><td rowspan=1 colspan=1>4.19x106 CFU/mL</td></tr><tr><td rowspan=3 colspan=2></td><td rowspan=1 colspan=1>Streptococcus sanguinis</td><td rowspan=1 colspan=1>8.67x106 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Chlamydia pneumoniae</td><td rowspan=1 colspan=1>1.20x106 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Mycobacterium tuberculosis(avirulent)</td><td rowspan=1 colspan=1>1.20x106 CFU/mL</td></tr></table>

a Live virus was not available. Synthetic RNA was used.   
b Live organism was not available. Genomic DNA was used.

# ii. Microbial Interference

A microbial interference study was conducted to assess potential inhibitory effects of a select panel of 10 non-target microorganisms (7 viruses and 3 bacteria) that may be found in a human respiratory specimen (Table 10). Viral strains were tested at $\geq$ $\mathrm { 1 x 1 0 ^ { 5 } T C I D 5 0 / m L }$ (unless otherwise noted) and bacteria were tested at $1 \mathrm { x } 1 0 ^ { 6 }$ $\mathrm { C F U / m L }$ in the presence of one strain each of SARS-CoV-2 (USA/WA/1/2020), influenza A (A/Idaho/07/2018), influenza B (B/Washington/2/2019), RSV A (A/Australia/2/61) and RSV B (B/9320/MA/77) spiked individually at 3x LoD in negative simulated matrix. Data supporting the use of simulated matrix can be found in section VII Performance Characteristics.B.2.Sample Matrix and Transport Media Equivalency Study, later in this document. Testing was performed with replicates of eight for each of the target strain/potentially interfering strain combinations, and a negative simulated matrix control. Under the conditions of the study, no inhibitory effects were observed for each of the non-target microorganisms tested in the presence of the target analytes.

Table 10. Non-Target Microorganisms Tested in the Microbial Interference Study   

<table><tr><td rowspan=1 colspan=1>Non-Target Microorganism</td><td rowspan=1 colspan=1>Concentration</td></tr><tr><td rowspan=1 colspan=1>Adenovirus Type 1C</td><td rowspan=1 colspan=1>1.0x105 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Human coronavirus OC43</td><td rowspan=1 colspan=1>1.0x105 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Human Metapneumovirus 5, Type B1</td><td rowspan=1 colspan=1>1.0x105 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza Type 1</td><td rowspan=1 colspan=1>1.0x105 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza Type 2</td><td rowspan=1 colspan=1>1.0x105TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza Type 3</td><td rowspan=1 colspan=1>1.0x105TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Rhinovirus Type 1A</td><td rowspan=1 colspan=1>1.0x105 PFU/mL</td></tr><tr><td rowspan=1 colspan=1>Haemophilus influenzae</td><td rowspan=1 colspan=1>1.0x106CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus aureus</td><td rowspan=1 colspan=1>1.0x107 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus epidermidis</td><td rowspan=1 colspan=1>1.0x107 CFU/mL</td></tr></table>

# b. In silico

For SARS-CoV-2 in silico analyses, sequences for 45 microorganisms that are closely related or commonly encountered in a respiratory clinical specimen, were downloaded from GenBank. These sequences were analyzed with BLAST for homology to the primer/probe sequences for the SARS-CoV-2 targets included in the assay. For Flu and RSV in silico analyses, all sequences in GenBank (which encompasses essentially all species) were downloaded and were evaluated with BLAST for homology to the primer/probe sequences for the Flu A, Flu B, and RSV targets included in the assay. In addition to performing homology analyses with primer/probe sequences, homology to the amplicon was also assessed for SARS-CoV-2, Flu A, Flu B, and RSV. All sequences were downloaded from GenBank during the week of July 15, 2022.

For the SARS-CoV-2, Flu, and RSV in silico analyses, a non-target microorganism was determined to have no cross-reactivity when its sequence had $- 8 0 \%$ homology with all the assay’s primers and probes When $2 8 0 \%$ homology was identified for a single assay target primer or probe, an analysis of the remaining target primer/probe sequences was performed to determine if they exhibited $2 8 0 \%$ homology and thus may generate an

amplicon that could be detected. For the SARS-CoV-2, Flu A, Flu B, and RSV in silico amplicon analyses, a non-target microorganism was determined to have no crossreactivity if the amplicon shared $- 8 0 \%$ homology to the non-target sequences.

For SARS-CoV-2, the E-gene target forward primer, reverse primer, and probe exhibited $5 8 0 \%$ homology to SARS-coronavirus from bats and humans and therefore these microorganisms may be amplified and detected. None of the remaining evaluated microorganisms have ${ > } 8 0 \%$ homology to a SARS-CoV-2 target forward primer, reverse primer and probe and therefore are not expected to be amplified or detected.

For Flu A, Flu B, and RSV B, there were no sequences from organisms expected to be found in a human respiratory tract sample for which the forward primer, reverse primer, and probe all had $2 8 0 \%$ homology. The RSV A primer and probe oligonucleotides exhibited $2 8 0 \%$ homology with two Pangolin RSV A isolates. Therefore, the RSV A primers and probe may cross-react with Pangolin RSV A if the strain is circulating in a human population and present in a sample tested with the Xpert Xpress CoV-2/Flu/RSV plus test. While there was homology ${ > } 8 0 \%$ to human genomic DNA, the matches were to different chromosomal regions, and there were no cases where a forward and revere primer for a specific target matched to the same human genomic DNA fragment. For Flu A, Flu B, and RSV B, in silico amplicon analyses produced no matches with $5 8 0 \%$ homology.

While no matches of the RSV A amplicon to genomic sequences from non-RSV species sequences of ${ > } 8 0 \%$ were observed, the RSV A amplicon shared a $9 5 \%$ identify with two Pangolin RSV A isolates.

# Interfering Substances

The performance of the Xpert Xpress CoV-2/Flu/RSV plus test was evaluated in the presence of medically and/or physiologically relevant concentrations of potentially interfering substances (listed in Table 11 below) that may be present in NPS or NS specimens.

Table 11. Potentially Interfering Substances Tested   

<table><tr><td colspan="1" rowspan="1">Substance/Class</td><td colspan="1" rowspan="1">Description/ActiveIngredient</td><td colspan="1" rowspan="1">Concentration Tested</td></tr><tr><td colspan="1" rowspan="1">Control</td><td colspan="1" rowspan="1">Simulated NPS/NS Matrix</td><td colspan="1" rowspan="1">100% (v/v)</td></tr><tr><td colspan="1" rowspan="1">Beta-Adrenergic Bronchodilator</td><td colspan="1" rowspan="1">Albuterol Sulfate</td><td colspan="1" rowspan="1">0.83 mg/mL (equivalentto 1 dose per day)</td></tr><tr><td colspan="1" rowspan="1">Afrin Nasal Spray</td><td colspan="1" rowspan="1">Oxymetazoline</td><td colspan="1" rowspan="1">15% (v/v)</td></tr><tr><td colspan="1" rowspan="1">BD Universal Transport Medium</td><td colspan="1" rowspan="1">Transport Medium</td><td colspan="1" rowspan="1">100% (v/v)</td></tr><tr><td colspan="1" rowspan="1">Remel M4RT</td><td colspan="1" rowspan="1">Transport Medium</td><td colspan="1" rowspan="1">100% (v/v)</td></tr><tr><td colspan="1" rowspan="1">Remel M5</td><td colspan="1" rowspan="1">Transport Medium</td><td colspan="1" rowspan="1">100% (v/v)</td></tr><tr><td colspan="1" rowspan="1">Copan Swab M Transport Medium</td><td colspan="1" rowspan="1">Transport Medium</td><td colspan="1" rowspan="1">100% (v/v)</td></tr><tr><td colspan="1" rowspan="1">Blood</td><td colspan="1" rowspan="1">Blood (Human)</td><td colspan="1" rowspan="1">2% (v/v)</td></tr><tr><td colspan="1" rowspan="4">Vaccine</td><td colspan="1" rowspan="4">FluMist QuadrivalentVaccine</td><td colspan="1" rowspan="1">6.7% (v/v)</td></tr><tr><td colspan="1" rowspan="1">6.7x10-4 % (v/v)</td></tr><tr><td colspan="1" rowspan="1">6.7x10-6 % (v/v)</td></tr><tr><td colspan="1" rowspan="1">6.7x10-7 % (v/v)</td></tr><tr><td colspan="1" rowspan="1">Nasal Corticosteroid</td><td colspan="1" rowspan="1">Fluticasone Propionate</td><td colspan="1" rowspan="1">5 ug/mL</td></tr><tr><td colspan="1" rowspan="2">Human Cells</td><td colspan="1" rowspan="2">Human Peripheral BloodMononuclear Cells (PBMC)</td><td colspan="1" rowspan="1">1x106 cells/mL</td></tr><tr><td colspan="1" rowspan="1">0.5x106 cells/mL</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">E</td><td colspan="1" rowspan="1">0.25x106 cells/mL</td></tr><tr><td colspan="1" rowspan="1">Nonsteroidal Anti-InflammatoryDrug</td><td colspan="1" rowspan="1">Ibuprofen 200 mg/tablet</td><td colspan="1" rowspan="1">5% (w/v)</td></tr><tr><td colspan="1" rowspan="1">Throat Lozenges, Oral Anestheticand Analgesic</td><td colspan="1" rowspan="1">Benzocaine, Menthol</td><td colspan="1" rowspan="1">1.7 mg/mL</td></tr><tr><td colspan="1" rowspan="1">Mucin</td><td colspan="1" rowspan="1">Purified Mucin protein(Bovine or porcinesubmaxillary gland)</td><td colspan="1" rowspan="1">1% (w/v)</td></tr><tr><td colspan="1" rowspan="1">Antibiotic, Nasal Ointment</td><td colspan="1" rowspan="1">Mupirocin</td><td colspan="1" rowspan="1">10 mg/mL</td></tr><tr><td colspan="1" rowspan="1">Nasal Drops</td><td colspan="1" rowspan="1">Phenylephrine, 1%</td><td colspan="1" rowspan="1">15% (v/v)</td></tr><tr><td colspan="1" rowspan="1">Saline Nasal Spray</td><td colspan="1" rowspan="1">Sodium Chloride (0.65%)</td><td colspan="1" rowspan="1">15% (v/v)</td></tr><tr><td colspan="1" rowspan="4">Tobacco Product  Snuff</td><td colspan="1" rowspan="4">Nicotine</td><td colspan="1" rowspan="1">1% (w/v)</td></tr><tr><td colspan="1" rowspan="1">0.5% (w/v)</td></tr><tr><td colspan="1" rowspan="1">0.25% (w/v)</td></tr><tr><td colspan="1" rowspan="1">0.1% (w/v)</td></tr><tr><td colspan="1" rowspan="1">Anti-Viral Drugs (Tamiflu)</td><td colspan="1" rowspan="1">Zanamivir</td><td colspan="1" rowspan="1">7.5 mg/mL</td></tr><tr><td colspan="1" rowspan="1">Antibacterial, Systemic</td><td colspan="1" rowspan="1">Tobramycin</td><td colspan="1" rowspan="1">4 ug/mL</td></tr><tr><td colspan="1" rowspan="2">Nasal Gel (Zicam)</td><td colspan="1" rowspan="2">Luffa opperculata, Galphimiaglauca, Histaminumhydrochloricum Sulfer(0.05%)</td><td colspan="1" rowspan="1">15% (w/v)</td></tr><tr><td colspan="1" rowspan="1">7.5% (w/v)</td></tr><tr><td colspan="1" rowspan="1">Zinc</td><td colspan="1" rowspan="1">Zinc Gluconate</td><td colspan="1" rowspan="1">0.1 ug/mL</td></tr></table>

For this study, eight negative replicates were tested per potentially interfering substance to determine the effects on the performance of the SPC. In addition, eight positive replicates were tested per substance for one SARS-CoV-2 strain, six influenza strains and four RSV strains. The panel of positive samples included one SARS-CoV-2 strain (USA-WA1/2020), two $2 0 0 9 \mathrm { H } 1 \mathrm { N } 1$ strains (A/California/7/2009 and A/Idaho/07/2018), two influenza A H3N2 strains (A/Hong Kong/45/2019 and A/Victoria/361/2011), two influenza B strains (B/Wisconsin/10/2016 and B/Washington/2/2019), and four RSV strains (A/2/Australia/61, A/Long/MD/56, B/9320/MA/77, and B/WA/18537/62) spiked into simulated matrix to an analyte concentration of $3 \mathbf { x }$ LoD. Data supporting the use of simulated matrix can be found in section VII Performance Characteristics.B.2.Sample Matrix and Transport Media Equivalency Study, later in this document. If interference was observed for a particular substance /organism combination, the substance was diluted and retested until interference was no longer observed.

Interference was not observed for any of the evaluated substances, except FluMist, Human Peripheral Blood Mononuclear Cells (PBMCs), Snuff, and Zicam. The test results for these substances are shown in Table 12.

Table 12. Substances that Interfered with Detection of at Least One Target Organism   

<table><tr><td colspan="1" rowspan="3">Substance</td><td colspan="1" rowspan="3">Conc. Tested</td><td colspan="12" rowspan="1"># Of Correct Results/Number Tested</td></tr><tr><td colspan="1" rowspan="2">Noviruscontrol</td><td colspan="1" rowspan="2">SARS-CoV-2</td><td colspan="2" rowspan="1">Flu A 2009H1N1a</td><td colspan="2" rowspan="1">Flu AH3N2b</td><td colspan="2" rowspan="1">Flu Bc</td><td colspan="2" rowspan="1">RSV Ad</td><td colspan="2" rowspan="1">RSV Be</td></tr><tr><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">2</td></tr><tr><td colspan="1" rowspan="3">FluMist</td><td colspan="1" rowspan="1">6.7%</td><td colspan="1" rowspan="1">8/8</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">NA</td></tr><tr><td colspan="1" rowspan="1">6.7x10-4 %(v/v)</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">7/8</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">0/8</td><td colspan="1" rowspan="1">0/8</td><td colspan="1" rowspan="1">2/8</td><td colspan="1" rowspan="1">0/8</td></tr><tr><td colspan="1" rowspan="1">6.7x10-6 %(v/v)</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">8/8</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">8/8</td><td colspan="1" rowspan="1">7/8</td><td colspan="1" rowspan="1">8/8g</td><td colspan="1" rowspan="1">8/8</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">6.7x10-7 %(v/v)</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">8/8f</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">NA</td></tr><tr><td colspan="1" rowspan="3">HumanPBMC</td><td colspan="1" rowspan="1">1 x 106cells/mL</td><td colspan="1" rowspan="1">8/8</td><td colspan="1" rowspan="1">8/8</td><td colspan="1" rowspan="1">8/8g</td><td colspan="1" rowspan="1">8/8g</td><td colspan="1" rowspan="1">8/8</td><td colspan="1" rowspan="1">8/8</td><td colspan="1" rowspan="1">8/8</td><td colspan="1" rowspan="1">6/8</td><td colspan="1" rowspan="1">8/8g</td><td colspan="1" rowspan="1">8/8</td><td colspan="1" rowspan="1">8/8g</td><td colspan="1" rowspan="1">8/8g</td></tr><tr><td colspan="1" rowspan="1">0.5 x 106cells/mL</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">7/8</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">NA</td></tr><tr><td colspan="1" rowspan="1">0.25 x 106cells/mL</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">8/8</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">NA</td></tr><tr><td colspan="1" rowspan="4">Snuff</td><td colspan="1" rowspan="1">1% (w/v)</td><td colspan="1" rowspan="1">8/8</td><td colspan="1" rowspan="1">8/8</td><td colspan="1" rowspan="1">6/8</td><td colspan="1" rowspan="1">8/8</td><td colspan="1" rowspan="1">8/8g</td><td colspan="1" rowspan="1">8/8</td><td colspan="1" rowspan="1">8/8</td><td colspan="1" rowspan="1">4/8g</td><td colspan="1" rowspan="1">8/8</td><td colspan="1" rowspan="1">8/8</td><td colspan="1" rowspan="1">8/8</td><td colspan="1" rowspan="1">8/8h</td></tr><tr><td colspan="1" rowspan="1">0.5% (w/v)</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">7/8</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">3/8</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">NA</td></tr><tr><td colspan="1" rowspan="1">0.25% (w/v)</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">8/8</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">7/8</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">NA</td></tr><tr><td colspan="1" rowspan="1">0.1% (w/v)</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">8/8</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">NA</td></tr><tr><td colspan="1" rowspan="2">Zicam</td><td colspan="1" rowspan="1">15% (w/v)</td><td colspan="1" rowspan="1">16/16</td><td colspan="1" rowspan="1">8/8</td><td colspan="1" rowspan="1">7/8</td><td colspan="1" rowspan="1">8/8</td><td colspan="1" rowspan="1">8/8</td><td colspan="1" rowspan="1">8/8</td><td colspan="1" rowspan="1">8/8f</td><td colspan="1" rowspan="1">5/8</td><td colspan="1" rowspan="1">7/8</td><td colspan="1" rowspan="1">8/8</td><td colspan="1" rowspan="1">8/8</td><td colspan="1" rowspan="1">8/8</td></tr><tr><td colspan="1" rowspan="1">7.5% (w/v)</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">8/8</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">8/8</td><td colspan="1" rowspan="1">8/8</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">NA</td></tr></table>

a Flu A 2009 H1: (1) A/California/7/2009; (2) A/Idaho/07/2018 b Flu A H3N2: (1) A/Hong Kong/45/2019; (2) A/Victoria/361/2011 c Flu B: (1) B/Wisconsin/10/2016; (2) B/Washington/2/2019 $^ { \mathrm { d } } \mathrm { R S V }$ A: (1) A/2/Australia/61; (2) A/Long/MD/56 e RSV B: (1) B/9320/MA/77; (2) B/WA/18537/62

f One replicate reported NO RESULT. The run was successfully repeated to obtain the required number of valid replicates.

g One replicate reported ERROR. The run was successfully repeated to obtain the required number of valid replicates.

h Two replicates reported ERROR. The two runs were successfully repeated to obtain the required number of valid replicates.

# Competitive Interference

The impact of competitive interference, caused by co-infections with on-target analytes, was evaluated for the Xpert Xpress CoV-2/Flu/RSV plus test by testing contrived samples consisting of individual SARS-CoV-2, Flu A, Flu B or RSV strains at 3x LoD in the presence of different target strains at high concentrations $( \ge 1 \mathrm { x } 1 0 ^ { 5 }$ RNA copies/mL, as determined by droplet digital PCR (ddPCR)). For this study, competitive interference was assessed using one strain each of SARS-CoV-2 (USA/WA/1/2020), influenza A (A/Idaho/07/2018), influenza B (B/Washington/2/2019), RSV A (A/Australia/2/61) and RSV B (B/9320/MA/77). Testing for each target strain (at low concentration) and each potential competitive strain (at high concentration) was performed in triplicate. No competitive interference was observed if all replicates for the low concentration target yielded positive results. If competitive interference was observed, the concentration of the competing virus was reduced by 10-fold increments until interference was no longer observed. The on-target analyte combinations evaluated, and the results of this evaluation are shown in Table 13.

Table 13. Competitive Interference Study Sample Panel Composition & Study Results   

<table><tr><td colspan="2" rowspan="1">Target 1 (Low conc.)</td><td colspan="2" rowspan="1">Target 2 (High conc.)</td><td colspan="4" rowspan="1">% Detected (# Detected / # Tested)</td></tr><tr><td colspan="1" rowspan="1">Virus</td><td colspan="1" rowspan="1">Conc.</td><td colspan="1" rowspan="1">Virus</td><td colspan="1" rowspan="1">Conc.(RNAcopies/mL)</td><td colspan="1" rowspan="1">SARS-CoV-2</td><td colspan="1" rowspan="1">Flu A</td><td colspan="1" rowspan="1">Flu B</td><td colspan="1" rowspan="1">RSV</td></tr><tr><td colspan="1" rowspan="3">SARS-CoV-2</td><td colspan="1" rowspan="3">414copies/mL(3x LoD)</td><td colspan="1" rowspan="1">Flu A</td><td colspan="1" rowspan="1">1.7x108</td><td colspan="1" rowspan="1">100%(3/3)</td><td colspan="1" rowspan="1">100%(3/3)</td><td colspan="1" rowspan="1">0%(0/3)</td><td colspan="1" rowspan="1">0%(0/3)</td></tr><tr><td colspan="1" rowspan="1">Flu B</td><td colspan="1" rowspan="1">1.4x105</td><td colspan="1" rowspan="1">100%(3/3)</td><td colspan="1" rowspan="1">0%(0/3)</td><td colspan="1" rowspan="1">100%(3/3)</td><td colspan="1" rowspan="1">0%(0/3)</td></tr><tr><td colspan="1" rowspan="1">RSV A</td><td colspan="1" rowspan="1">4.6x106</td><td colspan="1" rowspan="1">100%(3/3)</td><td colspan="1" rowspan="1">0%(0/3)</td><td colspan="1" rowspan="1">0%(0/3)</td><td colspan="1" rowspan="1">100%(3/3)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">RSV B</td><td colspan="1" rowspan="1">1.9x105</td><td colspan="1" rowspan="1">100%(3/3)</td><td colspan="1" rowspan="1">0%(0/3)</td><td colspan="1" rowspan="1">0%(0/3)</td><td colspan="1" rowspan="1">100%(3/3)</td></tr><tr><td colspan="1" rowspan="4">Flu A</td><td colspan="1" rowspan="4">0.021TCID50/mL(3x LoD)</td><td colspan="1" rowspan="1">SARS-CoV-2</td><td colspan="1" rowspan="1">1.0x106</td><td colspan="1" rowspan="1">100%(3/3)</td><td colspan="1" rowspan="1">100%(3/3)</td><td colspan="1" rowspan="1">0%(0/3)</td><td colspan="1" rowspan="1">0%(0/3)</td></tr><tr><td colspan="1" rowspan="1">Flu B</td><td colspan="1" rowspan="1">1.4x105</td><td colspan="1" rowspan="1">0%(0/3)</td><td colspan="1" rowspan="1">100%(3/3)</td><td colspan="1" rowspan="1">100%(3/3)</td><td colspan="1" rowspan="1">0%(0/3)</td></tr><tr><td colspan="1" rowspan="1">RSV A</td><td colspan="1" rowspan="1">4.6x106</td><td colspan="1" rowspan="1">0%(0/3)</td><td colspan="1" rowspan="1">100%(3/3)</td><td colspan="1" rowspan="1">0%(0/3)</td><td colspan="1" rowspan="1">100%(3/3)</td></tr><tr><td colspan="1" rowspan="1">RSV B</td><td colspan="1" rowspan="1">1.9x105</td><td colspan="1" rowspan="1">0%(0/3)</td><td colspan="1" rowspan="1">100%(3/3)</td><td colspan="1" rowspan="1">0%(0/3)</td><td colspan="1" rowspan="1">100%(3/3)</td></tr><tr><td colspan="1" rowspan="8">Flu B</td><td colspan="1" rowspan="8">38.7CEID50/mL(3x LoD)</td><td colspan="1" rowspan="2">SARS-CoV-2</td><td colspan="1" rowspan="1">1.0x106</td><td colspan="1" rowspan="1">100%(3/3)</td><td colspan="1" rowspan="1">0%(0/3)</td><td colspan="1" rowspan="1">33%(1/3)</td><td colspan="1" rowspan="1">0%(0/3)</td></tr><tr><td colspan="1" rowspan="1">1.0x105</td><td colspan="1" rowspan="1">100%(3/3)</td><td colspan="1" rowspan="1">0%(0/3)</td><td colspan="1" rowspan="1">100%(3/3)</td><td colspan="1" rowspan="1">0%(0/3)</td></tr><tr><td colspan="1" rowspan="4">Flu A</td><td colspan="1" rowspan="1">1.7x108</td><td colspan="1" rowspan="1">0%(0/3)</td><td colspan="1" rowspan="1">100%(3/3)</td><td colspan="1" rowspan="1">0%(0/3)</td><td colspan="1" rowspan="1">0%(0/3)</td></tr><tr><td colspan="1" rowspan="1">1.7x107</td><td colspan="1" rowspan="1">0%(0/3)</td><td colspan="1" rowspan="1">100%(3/3)</td><td colspan="1" rowspan="1">0%(0/3)</td><td colspan="1" rowspan="1">0%(0/3)</td></tr><tr><td colspan="1" rowspan="1">1.7x106</td><td colspan="1" rowspan="1">0%(0/3)</td><td colspan="1" rowspan="1">100%(3/3)</td><td colspan="1" rowspan="1">66.6%(2/3)</td><td colspan="1" rowspan="1">0%(0/3)</td></tr><tr><td colspan="1" rowspan="1">1.7x105</td><td colspan="1" rowspan="1">0%(0/3)</td><td colspan="1" rowspan="1">100%(3/3)</td><td colspan="1" rowspan="1">100%(3/3)</td><td colspan="1" rowspan="1">0%(0/3)</td></tr><tr><td colspan="1" rowspan="1">RSV A</td><td colspan="1" rowspan="1">4.6x106</td><td colspan="1" rowspan="1">0%(0/3)</td><td colspan="1" rowspan="1">0%(0/3)</td><td colspan="1" rowspan="1">100%(3/3)</td><td colspan="1" rowspan="1">100%(3/3)</td></tr><tr><td colspan="1" rowspan="1">RSVB</td><td colspan="1" rowspan="1">1.9x105</td><td colspan="1" rowspan="1">0%(0/3)</td><td colspan="1" rowspan="1">0%(0/3)</td><td colspan="1" rowspan="1">100%(3/3)</td><td colspan="1" rowspan="1">100%(3/3)</td></tr><tr><td colspan="1" rowspan="5">RSV A</td><td colspan="1" rowspan="5">0.99TCID50/mL(3x LoD)</td><td colspan="1" rowspan="1">SARS-CoV-2</td><td colspan="1" rowspan="1">1.0x106</td><td colspan="1" rowspan="1">100%(3/3)</td><td colspan="1" rowspan="1">0%(0/3)</td><td colspan="1" rowspan="1">0%(0/3)</td><td colspan="1" rowspan="1">100%(3/3)</td></tr><tr><td colspan="1" rowspan="3">Flu A</td><td colspan="1" rowspan="1">1.7x108</td><td colspan="1" rowspan="1">0%(0/3)</td><td colspan="1" rowspan="1">100%(3/3)</td><td colspan="1" rowspan="1">0%(0/3)</td><td colspan="1" rowspan="1">0%(0/3)</td></tr><tr><td colspan="1" rowspan="1">1.7x107</td><td colspan="1" rowspan="1">0%(0/3)</td><td colspan="1" rowspan="1">100%(3/3)</td><td colspan="1" rowspan="1">0%(0/3)</td><td colspan="1" rowspan="1">0%(0/3)</td></tr><tr><td colspan="1" rowspan="1">1.7x106</td><td colspan="1" rowspan="1">0%(0/3)</td><td colspan="1" rowspan="1">100%(3/3)</td><td colspan="1" rowspan="1">0%(0/3)</td><td colspan="1" rowspan="1">100%(3/3)</td></tr><tr><td colspan="1" rowspan="1">Flu B</td><td colspan="1" rowspan="1">1.4x105</td><td colspan="1" rowspan="1">0%(0/3)</td><td colspan="1" rowspan="1">0%(0/3)</td><td colspan="1" rowspan="1">100%(3/3)</td><td colspan="1" rowspan="1">100%(3/3)</td></tr><tr><td colspan="1" rowspan="3">RSVB</td><td colspan="1" rowspan="3">1.11TCID50/mL(3x LoD)</td><td colspan="1" rowspan="1">SARS-CoV-2</td><td colspan="1" rowspan="1">1.0x106</td><td colspan="1" rowspan="1">100%(3/3)</td><td colspan="1" rowspan="1">0%(0/3)</td><td colspan="1" rowspan="1">0%(0/3)</td><td colspan="1" rowspan="1">100%(3/3)</td></tr><tr><td colspan="1" rowspan="1">Flu A</td><td colspan="1" rowspan="1">1.7x108</td><td colspan="1" rowspan="1">0%(0/3)</td><td colspan="1" rowspan="1">100%(3/3)</td><td colspan="1" rowspan="1">0%(0/3)</td><td colspan="1" rowspan="1">100%(3/3)</td></tr><tr><td colspan="1" rowspan="1">Flu B</td><td colspan="1" rowspan="1">1.4x105</td><td colspan="1" rowspan="1">0%(0/3)</td><td colspan="1" rowspan="1">0%(0/3)</td><td colspan="1" rowspan="1">100%(3/3)</td><td colspan="1" rowspan="1">100%(3/3)</td></tr></table>

The study showed that Flu A at concentrations ${ \mathrm { > } } 1 . 7 \mathbf { x } 1 0 ^ { 5 }$ RNA copies $/ \mathrm { m L }$ inhibited detection of Flu B at $3 \mathbf { x }$ LoD, and at Flu A concentrations ${ > } 1 . 7 \mathrm { x } 1 0 ^ { 6 }$ RNA copies/mL inhibited detection of RSV A at 3x LoD (Table 13). In addition, SARS-CoV-2 at concentrations above $1 \mathrm { x } 1 0 ^ { 5 } \mathrm { R N A }$ copies/mL inhibited detection of Flu B at $3 \mathbf { x }$ LoD (Table 13). No other competitive interference was observed for the potential co-infections evaluated in the study, at the concentrations tested.

4. Assay Reportable Range: Not Applicable; this is a qualitative assay.

a. Controls

# External Control Evaluation

Two Zeptometrix NATtrol Molecular Controls, comprised of inactivated viruses in a purified protein matrix, were evaluated as external controls with the Xpert Xpress CoV2/Flu/RSV plus test. Testing was performed with 20 replicates of three separate lots of the NATtrol SARS-CoV-2/Flu/RSV Positive Control (cat# NATFRC-6C-IVD) and the NATtrol SARS-CoV-2/Flu/RSV Negative Control (cat# NATCV9-6C-IVD) totaling 120 runs. All 120 runs provided valid and accurate results except for the following: A single Positive Control replicate was negative for Flu B and RSV.

# b. Sample Stability

Room Temperature (15 to $3 0 ^ { \circ } C )$ ) and Refrigerated (2 to $g \circ C ,$ Stability Studies A stability study was conducted to establish transport and storage claims for eluted NPS specimens to be analyzed with the Xpert Xpress CoV-2/Flu/RSV plus test. Specimens included one SARS-CoV-2 strain (USA-WA1/2020), one influenza A strain (A/Idaho/07/2018), one influenza B strain (B/Washington/2/2019), and one RSV strain (RSV A/2/Australia/61) prepared in clinical NPS-UTM matrix and clinical NPS-eNAT matrix at a concentration of 3x LoD. Eight replicates of each of the positive specimens were tested at T0 (fresh) and at multiple time points following storage at $2 \mathrm { ^ { \circ } C }$ , $8 \mathrm { { } ^ { \circ } C }$ , $1 5 \mathrm { { } ^ { \circ } C }$ , and $3 0 \mathrm { { } ^ { \circ } C }$ . Negative samples were also included and tested in four replicates for each storage condition and temperature. Under the conditions of the study, all positive and negative specimens at all storage conditions and temperatures tested were correctly identified using the Xpert Xpress CoV-2/Flu/RSV plus. The study data supports the following specimen stability claims:

• VTM and eNAT: room temp $( 1 5 . 3 0 ^ { \circ } \mathrm { C } )$ for up to 48 hours • For VTM: refrigerated $( 2 { - } 8 ^ { \circ } \mathrm { C } )$ for up to seven days • eNAT: refrigerated $\left( 2 \mathrm { - } 8 ^ { \circ } \mathrm { C } \right)$ for up to six days

Freeze/Thaw Testing for Samples Frozen at ${ } _ { - 2 0 ^ { \circ } C }$ or ${ } _ { - 8 0 ^ { \circ } C }$ in VTM or eNAT The performance of the Xpert Xpress CoV-2/Flu/RSV plus test with fresh and frozen specimens was evaluated for equivalency by testing co-spiked samples containing one SARS-CoV-2 strain (USA-WA1/2020), one influenza A strain (A/Idaho/07/2018), one influenza B strain (B/Washington/2/2019), and one RSV strain (RSV B/9320/MA/77) prepared in pooled negative clinical NPS-VTM matrix and pooled negative clinical NPSeNAT matrix at a concentration of <1x LoD, 2x LoD, and 5x LoD. Negative samples were also included and consisted of negative clinical NPS-VTM and negative clinical NPS-eNAT matrix.

Positive samples spiked at ${ < } 1 \mathbf { x }$ LoD, 5x LoD, and negative samples were tested in replicates of 10 per condition, while positive samples prepared at $2 \mathbf { x }$ were tested in replicates of 30 per condition. All positive and negative samples were tested under three conditions: fresh, after one freeze-thaw cycle, and after two freeze-thaw cycles. For each freeze-thaw cycle, the samples were placed at either $- 2 0 ^ { \circ } \mathrm { C }$ or $- 8 0 ^ { \circ } \mathrm { C }$ for 24 hours after which they were thawed on ice prior to analysis with the Xpert Xpress CoV-2/Flu/RSV plus test.

For storage at ${ } _ { - 2 0 } \mathrm { { } ^ { \circ } C }$ under all conditions, all replicates for samples spiked at $2 \mathbf { x }$ and ${ 5 } \mathbf { x }$ LoD were positive, except for the following: one fresh sample spiked at $2 \mathbf { x }$ LoD in VTM was negative for Flu A, while a different fresh sample spiked at $2 \mathbf { x }$ LoD in VTM was negative for RSV. All negative samples were negative for all conditions tested. As expected, detection varied for samples spiked at ${ < } 1 \mathbf { x }$ LoD.

For storage at $- 8 0 ^ { \circ } \mathrm { C }$ under all conditions, all samples spiked at $2 \mathbf { x }$ and 5x LoD were positive, while all negative samples were negative. As expected, detection varied for samples spiked at ${ < } 1 \mathbf { x }$ LoD.

The data from this study supports that NPS and NS samples collected in VTM and eNAT can undergo up to 1 freeze/thaw cycle when stored at ${ } _ { - 2 0 } \mathrm { { } ^ { \circ } C }$ or $- 8 0 ^ { \circ } \mathrm { C }$ .

# c. Kit Stability

Real-time kit stability data for the Xpert Xpress CoV-2/Flu/RSV plus test is currently available for eight months at the extremes of the recommended storage temperature range of $2 \mathrm { ^ { \circ } C }$ to $2 8 ^ { \circ } \mathrm { C }$ . Results to date show that average Ct values from the stability testing meet the pre-defined performance criteria. Testing is ongoing for up to 37 months on seven cartridge lots.

# d. Cartridge Hold Time

Xpert Xpress CoV-2/Flu/RSV plus samples that are prepared for testing may wait up to four and a half hours on the GeneXpert Infinity Instruments before a module becomes available. During this wait time, targeted viral RNA may degrade or become unstable such that an originally low positive sample is rendered “NEGATIVE.” Assay performance out to five and a half hours was evaluated under three storage conditions (ambient, $2 5 ^ { \circ } \mathrm { C } / 7 5 \%$ relative humidity (RH), and $3 5 ^ { \circ } \mathrm { C }$ ) with a single lot of the Xpert Xpress CoV-2/Flu/RSV plus test. Testing was performed on eight replicates each of one SARS-CoV-2 strain (USA-WA1/2020), one influenza A strain (A/Idaho/07/2018), one influenza B strain (B/Washington/2/2019), and one RSV strain (RSV A/2/Australia/61) spiked into negative simulated NPS matrix at an analyte concentration of 3x LoD. Negative specimens consisting of simulated NPS matrix were also included in the study. For negative specimens, eight replicates were collected for each condition and time point. The results of this study support a cartridge hold time of 4.5 hours.

# 6. Detection Limit:

The LoD of the Xpert Xpress CoV-2/Flu/RSV plus test was established using the WHO International Standard for SARS-CoV-2 (NIBSC, 20/146), 1 inactivated SARS-CoV-2 strain (USA-WA1/2020), and viral cultures of two Flu A H1 strains, 2 Flu A H3 strains, 2 Flu B strains (One Yamagata and one Victoria lineage), 2 RSV A strains, and 2 RSV B strains diluted into pooled negative clinical NPS matrix and negative clinical NS matrix. The LoD is defined as the lowest concentration for each strain at which $9 5 \%$ (19/20) of replicates yield a positive result.

Each virus strain was initially tested in a range finding study at ${ \ge } 5$ concentrations in replicates of 20 per concentration of virus. Range finding was performed with two cartridge lots for each virus/strain. Probit analysis was performed on the range finding results to estimate the LoDs, and the estimated LoDs were subsequently confirmed using two lots of Xpert Xpress CoV-2/Flu/RSV plus cartridges. For the confirmatory study, testing was also performed in replicates of 20 per concentration of virus. The highest (least sensitive) LoD value for the two lots evaluated in the confirmatory story was reported as the final LoD for the Xpert Xpress CoV-2/Flu/RSV plus test (see Table 14). The LoD for co-analyte spiked samples was also evaluated and shown to be equivalent to single analyte spiked samples.

Table 14. Xpert Xpress CoV-2/Flu/RSV plus Confirmatory LoD Study Results   

<table><tr><td rowspan=2 colspan=1>Virus/Strain</td><td rowspan=1 colspan=2>Confirmed LoD Concentration</td></tr><tr><td rowspan=1 colspan=1>NPS matrix</td><td rowspan=1 colspan=1>NS matrix</td></tr><tr><td rowspan=1 colspan=1>SARS-CoV-2, WHO International Standard</td><td rowspan=1 colspan=1>94 IU/mL</td><td rowspan=1 colspan=1>143 IU/mL</td></tr><tr><td rowspan=1 colspan=1>SARS-CoV-2, USA-WA1/2020</td><td rowspan=1 colspan=1>138 copies/mL</td><td rowspan=1 colspan=1>64 copies/mL</td></tr><tr><td rowspan=1 colspan=1>Flu A/Idaho/07/2018</td><td rowspan=1 colspan=1>0.007 TCID50/mL</td><td rowspan=1 colspan=1>0.012 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Flu A/California/07/2009</td><td rowspan=1 colspan=1>0.0022 TCID50/mL</td><td rowspan=1 colspan=1>0.0028 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Flu A/Hong Kong/45/2019</td><td rowspan=1 colspan=1>0.44 FFU/mL</td><td rowspan=1 colspan=1>0.49 FFU/mL</td></tr><tr><td rowspan=1 colspan=1>Flu A/Victoria/361/2011</td><td rowspan=1 colspan=1>0.05 TCID50/mL</td><td rowspan=1 colspan=1>0.065 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Flu B/Washington/2/2019</td><td rowspan=1 colspan=1>12.9 CEID50/mL</td><td rowspan=1 colspan=1>26.3 CEID50/mL</td></tr><tr><td rowspan=1 colspan=1>Flu B/Wisconsin/10/2016</td><td rowspan=1 colspan=1>2.4 TCID50/mL</td><td rowspan=1 colspan=1>2.41 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>RSV A/2/Australia/61</td><td rowspan=1 colspan=1>0.33 TCID50/mL</td><td rowspan=1 colspan=1>0.28 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>RSV A/Long/MD/56</td><td rowspan=1 colspan=1>0.17 TCID50/mL</td><td rowspan=1 colspan=1>0.22 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>RSV B/9320/MA/77</td><td rowspan=1 colspan=1>0.37 TCID50/mL</td><td rowspan=1 colspan=1>0.24 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>RSV B/Wash/18537/62</td><td rowspan=1 colspan=1>0.2 TCID50/mL</td><td rowspan=1 colspan=1>0.4 TCID50/mL</td></tr></table>

# 7. Assay Cut-Off:

The Xpert Xpress CoV-2/Flu/RSV plus test includes defined Ct value cutoffs for each of the viral targets as well as the SPC. These values were determined during pre-clinical testing and were subsequently confirmed in the clinical study. The Ct cutoffs are included as automatic calculations in the assay definition file (ADF) of the Xpert Xpress CoV-2/Flu/RSV plus test.

8. Accuracy (Instrument): Not Applicable.

9. Carry-Over:

A study was conducted to assess whether the single-use, self-contained Xpert Xpress CoV2/Flu/RSV plus cartridge prevents specimen and amplicon carryover by testing a negative sample immediately after testing of a high positive sample in the same GeneXpert module. The negative sample used in this study consisted of simulated NPS/NS matrix and the positive sample consisted of Flu B and SARS-CoV-2 spiked at high concentrations (i.e., Flu B/Wisconsin/10/2016 was spiked at $1 . 0 \mathrm { x } 1 0 ^ { 6 } \mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ and SARS-CoV-2 USA-WA1/2020 at $1 \mathrm { x } 1 0 ^ { 4 }$ copies $/ \mathrm { m L }$ ) into simulated NPS/NS matrix. The negative sample was tested in a GeneXpert module at the start of the study. Following the initial testing of the negative sample, the high positive sample was processed in the same GeneXpert module followed by another negative sample. This was repeated 20 times in the same module, resulting in 20 positives and 21 negatives for the module. The study was repeated using a second GeneXpert module for a total of 40 positive and 42 negative samples. All positive samples were correctly reported as positive, and all negative samples were correctly reported as negative. No specimen or amplicon carry-over contamination was observed.

# B Comparison Studies:

1. Method Comparison with Predicate Device: Not Applicable.

2. Matrix Comparison:

Sample Matrix and Transport Media Equivalency Study

The objective of this study was to establish equivalent performance of the Xpert Xpress CoV-2/Flu/RSV plus test between the following conditions:

a. clinical NPS matrix (in UTM/VTM), clinical NS matrix (in UTM/VTM), and simulated NPS/NS matrix in UTM/VTM. b. clinical NPS matrix (in UTM/VTM) and clinical NPS matrix in eNAT.

Simulated NPS/NS background matrix consisted of $2 . 5 \%$ (w/v) porcine mucin and $1 \%$ (v/v) human whole blood in $0 . 8 5 \%$ sodium chloride (NaCl) formulated in 1x PBS solution with $1 5 \%$ glycerol and diluted into UTM/VTM at a 1:40 ratio. For this study, one SARS-CoV-2 strain (USA-WA1/2020), one influenza A strain (A/Idaho/07/2018), one Flu B strain (B/Wisconsin/10/2016) and two RSV strains (RSV A/2/Australia/61 and RSV B/9320/MA/77) were seeded into pooled negative matrices or simulated matrices at ${ < } 1 \mathbf { x }$ LoD, $1 . 5 \mathrm { x }$ LoD, 5x LoD, and $1 0 \mathrm { x }$ LoD. Aliquots of negative matrices were also included in the evaluation. Positive samples spiked at ${ < } 1 \mathbf { x }$ LoD, 5x LoD, $1 0 \mathrm { x }$ LoD and negative samples were tested in replicates of 10 per condition, while positive samples prepared at $1 . 5 \mathrm { x }$ LoD were tested in replicates of 30 per condition

For the conditions outlined in (a), all replicates for all samples spiked at 1.5x, 5x, and $1 0 \mathrm { x }$ LoD were positive for all evaluated matrices. All negative samples were negative for all matrices. As expected, detection varied for samples spiked at ${ < } 1 \mathbf { x }$ LoD.

For the conditions outlined in (b), all replicates for all samples spiked at 1.5x, 5x, and $1 0 \mathrm { x }$ LoD were positive for all evaluated matrices. All negative samples were negative for all matrices. As expected, detection varied for samples spiked at ${ < } 1 \mathrm { x } \mathrm { L o D }$ .

The results for the conditions evaluated in (a) indicate that performance of the Xpert Xpress CoV-2/Flu/RSV plus test is equivalent with viruses seeded into clinical NPS matrix (in UTM/VTM), clinical NS matrix (in UTM/VTM), and simulated NPS/NS matrix in UTM/VTM and thus support use of simulated matrix in select analytical validation studies. The results for the conditions evaluated in (b) reveal that performance of the Xpert Xpress CoV-2/Flu/RSV plus test is equivalent with viruses seeded into clinical NPS matrix (in UTM/VTM) and clinical NPS matrix in eNAT.

# C Clinical Studies:

# 1. Prospective Study

The clinical performance of the Xpert Xpress CoV-2/Flu/RSV plus was established in a multi-center study conducted with nasopharyngeal swab (NPS) and anterior nasal swab (NS) specimens in VTM/UTM collected from individuals with signs and symptoms of respiratory infection. Specimens were prospectively collected (i.e., all comers between two time points that met the clinical study inclusion criteria) during portions of the 2021-2022 viral respiratory illness season. Due to low prevalence of Flu and RSV during the 2021-2022 viral respiratory illness season, archived prospectively collected specimens collected during the 2016-2017 influenza season were also enrolled. Specimens prospectively collected in 2022 were tested fresh (Category I) or after freezing (Category II). All specimens enrolled in 2016- 2017 were Category II. In total, thirty-three (33) diverse clinical sites in the U.S. were involved in the prospective studies. Of these 33 sites, 5 sites participated in specimen collection only, 27 performed Xpert Xpress CoV-2/Flu/RSV plus testing and 1 site performed Xpert Xpress CoV-2/Flu/RSV plus testing, in addition to comparator and discordant analysis testing.

A total of 6,987 NPS and NS specimens were enrolled in the study. Of these, 6,100 specimens were enrolled in 2022 while the remaining 897 were collected in 2016-2017. Of the 6987 specimens, 19 were found to be ineligible after enrollment, 841 were excluded due to protocol deviations (e.g., specimen mislabeled, specimen not stored appropriately, etc.), and 6 were excluded because Xpert Xpress CoV-2/Flu/RSV plus test and/or comparator results were not provided by the testing sites. This left 6121 specimens for evaluation, of which 5331 (2672 NPS and $2 6 5 9 \mathrm { N S } )$ were collected in 2022 and 790 (422 NPS and 368 NS) were collected in 2016-2017. Patient demographic information for all evaluable specimens collected in 2022 and 2016-2017 is presented in Table 15.

Table 15. Demographic Data for Prospectively Collected Specimens   

<table><tr><td colspan="1" rowspan="1">Specimens Collectedin 2021-2022</td><td colspan="1" rowspan="1">NPS(N=2672)</td><td colspan="1" rowspan="1">NS(N=2659)</td><td colspan="1" rowspan="1">Overall(N=5331)</td></tr><tr><td colspan="1" rowspan="1">Gender</td><td colspan="3" rowspan="1">EY</td></tr><tr><td colspan="1" rowspan="1">Female</td><td colspan="1" rowspan="1">1568(58.7%)</td><td colspan="1" rowspan="1">1634(61.5%)</td><td colspan="1" rowspan="1">3202(60.1%)</td></tr><tr><td colspan="1" rowspan="1">Male</td><td colspan="1" rowspan="1">1104(41.3%)</td><td colspan="1" rowspan="1">1025(38.5%)</td><td colspan="1" rowspan="1">2129(39.9%)</td></tr><tr><td colspan="1" rowspan="1">Age Group (Years)</td><td colspan="3" rowspan="1">EY</td></tr><tr><td colspan="1" rowspan="1">≤5</td><td colspan="1" rowspan="1">9(0.3%)</td><td colspan="1" rowspan="1">183(6.9%)</td><td colspan="1" rowspan="1">192(3.6%)</td></tr><tr><td colspan="1" rowspan="1">6-21</td><td colspan="1" rowspan="1">623(23.3%)</td><td colspan="1" rowspan="1">562(21.1%)</td><td colspan="1" rowspan="1">1185(22.2%)</td></tr><tr><td colspan="1" rowspan="1">22-59</td><td colspan="1" rowspan="1">1676(62.7%)</td><td colspan="1" rowspan="1">1553(58.4%)</td><td colspan="1" rowspan="1">3229(60.6%)</td></tr><tr><td colspan="1" rowspan="1">≥60</td><td colspan="1" rowspan="1">364(13.6%)</td><td colspan="1" rowspan="1">361(13.6%)</td><td colspan="1" rowspan="1">725(13.6%)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Specimens Collectedin 2016-2017</td><td colspan="1" rowspan="1">NPS(N=422)</td><td colspan="1" rowspan="1">NS(N=368)</td><td colspan="1" rowspan="1">Overall(N=790)</td></tr><tr><td colspan="1" rowspan="1">Gender</td><td colspan="3" rowspan="1">EY</td></tr><tr><td colspan="1" rowspan="1">Female</td><td colspan="1" rowspan="1">211(50.0%)</td><td colspan="1" rowspan="1">223(60.6%)</td><td colspan="1" rowspan="1">434(54.9%)</td></tr><tr><td colspan="1" rowspan="1">Male</td><td colspan="1" rowspan="1">211(50.0%)</td><td colspan="1" rowspan="1">145(39.4%)</td><td colspan="1" rowspan="1">356(45.1%)</td></tr><tr><td colspan="1" rowspan="1">Age Group (Years)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">≤5</td><td colspan="1" rowspan="1">164</td><td colspan="1" rowspan="1">144</td><td colspan="1" rowspan="1">308</td></tr><tr><td colspan="1" rowspan="1">6-21</td><td colspan="1" rowspan="1">85</td><td colspan="1" rowspan="1">72</td><td colspan="1" rowspan="1">157</td></tr><tr><td colspan="1" rowspan="1">22-59</td><td colspan="1" rowspan="1">134</td><td colspan="1" rowspan="1">111</td><td colspan="1" rowspan="1">245</td></tr><tr><td colspan="1" rowspan="1">≥60</td><td colspan="1" rowspan="1">39</td><td colspan="1" rowspan="1">41</td><td colspan="1" rowspan="1">80</td></tr></table>

The Xpert Xpress CoV-2/Flu/RSV plus test was evaluated for SARS-CoV-2 performance by comparing to a U.S. FDA-cleared molecular respiratory panel that includes SARS-CoV-2. For SARS-CoV-2 performance evaluations, only specimens collected in 2022 were evaluated, as specimens collected prior to the COVID-19 pandemic (i.e., 2016-2017) were assumed to be SARS-CoV-2 negative. Of the 5331 evaluable specimens collected in 2022 and evaluated for investigational device SARS-CoV-2 performance, 261 were excluded due to non-evaluable/invalid comparator tests results, while an additional 19 were excluded because they were non-determinate (ND) upon retest. This left 5051 specimens (2536 NPS and 2515 NS) with valid Xpert Xpress CoV-2/Flu/RSV plus test and comparator SARSCoV-2 results. Of the 2536 NPS specimens, $9 8 . 8 \%$ (2505/2536) were tested fresh, while $1 . 2 \%$ (31/2536) were tested frozen with the Xpert Xpress CoV-2/Flu/RSV plus. Of the 2515 NS specimens, $9 9 . 0 \%$ (2489/2515) were tested fresh, while $1 . 0 \%$ (26/2515) were tested frozen with the Xpert Xpress CoV-2/Flu/RSV plus.

The Xpert Xpress CoV-2/Flu/RSV plus test was evaluated for Flu A, Flu B, and RSV performance by comparing to a U.S. FDA-cleared molecular Flu A/B/RSV assay. For Flu and RSV performance calculations, specimens collected in both 2022 and 2016-2017 were evaluated. Of the 6,121 evaluable specimens collected during these time periods and evaluated for investigational device Flu and RSV performance, 142 were excluded due to non-evaluable/invalid comparator results, while an additional 25 were excluded because they yielded ND results upon retest. This left 5954 specimens (3011 NPS and 2943 NS) with valid Xpert Xpress CoV-2/Flu/RSV plus test and comparator Flu A, Flu B, and RSV results. Of the 3011 NPS specimens, $8 5 . 1 \%$ (2562/3011) were tested fresh, while $1 4 . 9 \%$ (449/3011) were tested frozen with the Xpert Xpress CoV-2/Flu/RSV plus. Of the 2943 NS specimens, $8 6 . 7 \%$ (2553/2943) were tested fresh, while $1 3 . 3 \%$ (390/2943) were tested frozen with the Xpert Xpress CoV-2/Flu/RSV plus.

The Xpert Xpress CoV-2/Flu/RSV plus test ND rate for specimens collected in 2022 and 2016-2017, combined, was initially $2 . 4 \%$ (148/6121) and decreased to $0 . 4 \%$ (25/6121) upon retest. For specimens collected in 2022, the ND was rate initially $2 . 5 \%$ (132/5331) and decreased to $0 . 4 \%$ (19/5331) upon retest. For specimens collected in 2016-2017, the ND rate was initially $2 . 0 \%$ (16/790) and decreased to $0 . 8 \%$ (6/790) upon retest.

A summary of the Xpert Xpress CoV-2/Flu/RSV plus test prospective clinical study performance is provided in Table 16 for NPS specimens and Table 17 for NS specimens. Positive Percent Agreement (PPA) was calculated as $1 0 0 \% \times \left( \mathrm { T P } / \left( \mathrm { T P } + \mathrm { F N } \right) \right)$ . True positive (TP) indicates that both the Xpert Xpress $\mathrm { C o V - } 2 / \mathrm { F l u } / \mathrm { R S V } p l u s$ and the comparator method had a positive result for the specific analyte, and false negative (FN) indicates that the Xpert Xpress CoV-2/Flu/RSV plus was negative while the comparator result was positive. Negative Percent Agreement (NPA) was calculated as $1 0 0 \% \times \left( \mathrm { T N / \left( T N + F P \right) } \right)$ . True negative (TN) indicates that both the Xpert Xpress CoV-2/Flu/RSV plus and the comparator method had negative results, and false positive (FP) indicates that the Xpert Xpress CoV-2/Flu/RSV plus was positive while the comparator result was negative. Specimens that obtained discordant SARS-CoV-2 results underwent additional testing with a U.S. FDA EUA SARS-CoV-2 molecular test, while specimens that obtained discordant Flu A, Flu B, or RSV results underwent additional testing with a U.S. FDA-cleared molecular respiratory panel.

Table 16. Xpert Xpress CoV-2/Flu/RSV plus Assay Clinical Performance with NPS Specimens   

<table><tr><td rowspan=2 colspan=2>Analyte</td><td rowspan=1 colspan=3>Positive Percent Agreement</td><td rowspan=1 colspan=3>Negative Percent Agreement</td></tr><tr><td rowspan=1 colspan=1>TP/(TP+FN)</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=1>TN/(TN+FP)</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=3 colspan=1>SARS-CoV-2</td><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>454/468</td><td rowspan=1 colspan=1>97.0</td><td rowspan=1 colspan=1>95.0-98.2</td><td rowspan=1 colspan=1>2000/2037</td><td rowspan=1 colspan=1>98.2</td><td rowspan=1 colspan=1>97.5-98.7</td></tr><tr><td rowspan=1 colspan=1>Frozen</td><td rowspan=1 colspan=1>8/8</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>67.6-100</td><td rowspan=1 colspan=1>23/23</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>85.7-100</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>462/476a</td><td rowspan=1 colspan=1>97.1</td><td rowspan=1 colspan=1>95.0-98.2</td><td rowspan=1 colspan=1>2023/2060b</td><td rowspan=1 colspan=1>98.2</td><td rowspan=1 colspan=1>97.5-98.7</td></tr><tr><td rowspan=3 colspan=1>Flu A</td><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>98/100</td><td rowspan=1 colspan=1>98.0</td><td rowspan=1 colspan=1>93.0-99.5</td><td rowspan=1 colspan=1>2451/2462</td><td rowspan=1 colspan=1>99.6</td><td rowspan=1 colspan=1>99.2-99.8</td></tr><tr><td rowspan=1 colspan=1>Frozen</td><td rowspan=1 colspan=1>93/93</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>96.0-100</td><td rowspan=1 colspan=1>343/356</td><td rowspan=1 colspan=1>96.3</td><td rowspan=1 colspan=1>93.9-97.9</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>191/193c</td><td rowspan=1 colspan=1>99.0</td><td rowspan=1 colspan=1>96.3-99.7</td><td rowspan=1 colspan=1>2794/2818d</td><td rowspan=1 colspan=1>99.1</td><td rowspan=1 colspan=1>98.7-99.4</td></tr><tr><td rowspan=3 colspan=1>Flu B</td><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>2562/2562</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>99.9-100</td></tr><tr><td rowspan=1 colspan=1>Frozen</td><td rowspan=1 colspan=1>57/59</td><td rowspan=1 colspan=1>96.6</td><td rowspan=1 colspan=1>88.5-99.1</td><td rowspan=1 colspan=1>390/390</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>99.0-100</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>57/59e</td><td rowspan=1 colspan=1>96.6</td><td rowspan=1 colspan=1>88.5-99.1</td><td rowspan=1 colspan=1>2952/2952</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>99.9-100</td></tr><tr><td rowspan=3 colspan=1>RSV</td><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>12/12</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>75.8-100</td><td rowspan=1 colspan=1>2550/2550</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>99.8-100</td></tr><tr><td rowspan=1 colspan=1>Frozen</td><td rowspan=1 colspan=1>59/60</td><td rowspan=1 colspan=1>98.3</td><td rowspan=1 colspan=1>91.1-99.7</td><td rowspan=1 colspan=1>389/389</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>99.0-100</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>71/72f</td><td rowspan=1 colspan=1>98.6</td><td rowspan=1 colspan=1>92.5-99.8</td><td rowspan=1 colspan=1>2939/2939</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>99.9-100</td></tr></table>

TP-true positive; FN-false negative; TN-true negative; FP-false positive; NA-not applicable a Discrepant test results from a U.S. FDA EUA SARS-CoV-2 molecular test: 3/14 specimens were SARS-CoV-2 positive, 10/14 were SARS-CoV-2 negative, and 1/14 was invalid.

b Discrepant test results from a U.S. FDA EUA SARS-CoV-2 molecular test: 14/37 were SARS-CoV2 positive and 23/37 were SARS-CoV-2 negative.

c Discrepant test results from a U.S. FDA-cleared molecular respiratory panel: 1/2 were Flu A positive and 1/2 were Flu A negative.

d Discrepant test results from a U.S. FDA-cleared molecular respiratory panel: 9/24 were Flu A positive, 3/24 were Flu A negative, and 12/24 were not tested due to specimens being stored for a longer duration than recommended per the package insert.

e Neither specimen that yielded a Flu B false negative result was tested with a U.S. FDA-cleared molecular respiratory panel due to the specimens being stored for a longer duration than recommended in the package insert.

f The specimen that yielded the RSV false negative result was not tested with a U.S. FDA-cleared molecular Flu A/B/RSV assay due to the specimens being stored for a longer duration than recommended in the package insert.

Table 17. Xpert Xpress CoV-2/Flu/RSV plus Assay Clinical Performance with NS Specimens   

<table><tr><td rowspan=2 colspan=2>Analyte</td><td rowspan=1 colspan=3>Positive Percent Agreement</td><td rowspan=1 colspan=3>Negative Percent Agreement</td></tr><tr><td rowspan=1 colspan=1>TP/(TP+FN)</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=1>TN/(TN+FP)</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=3 colspan=1>SARS-CoV-2</td><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>442/449</td><td rowspan=1 colspan=1>98.4</td><td rowspan=1 colspan=1>96.8-99.2</td><td rowspan=1 colspan=1>2017/2040</td><td rowspan=1 colspan=1>98.8</td><td rowspan=1 colspan=1>98.3-99.2</td></tr><tr><td rowspan=1 colspan=1>Frozen</td><td rowspan=1 colspan=1>6/7</td><td rowspan=1 colspan=1>85.7</td><td rowspan=1 colspan=1>48.7-97.4</td><td rowspan=1 colspan=1>18/19</td><td rowspan=1 colspan=1>94.7</td><td rowspan=1 colspan=1>75.4-99.1</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>448/456&quot;</td><td rowspan=1 colspan=1>98.2</td><td rowspan=1 colspan=1>96.6-99.1</td><td rowspan=1 colspan=1>2035/2059b</td><td rowspan=1 colspan=1>98.8</td><td rowspan=1 colspan=1>98.3-99.2</td></tr><tr><td rowspan=3 colspan=1>Flu A</td><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>130/134</td><td rowspan=1 colspan=1>97.0</td><td rowspan=1 colspan=1>92.6-98.8</td><td rowspan=1 colspan=1>2413/2419</td><td rowspan=1 colspan=1>99.8</td><td rowspan=1 colspan=1>99.5-99.9</td></tr><tr><td rowspan=1 colspan=1>Frozen</td><td rowspan=1 colspan=1>66/66</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>94.5-100</td><td rowspan=1 colspan=1>312/324</td><td rowspan=1 colspan=1>96.3</td><td rowspan=1 colspan=1>93.6-97.9</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>196/200c</td><td rowspan=1 colspan=1>98.0</td><td rowspan=1 colspan=1>95.0-99.2</td><td rowspan=1 colspan=1>2725/2743d</td><td rowspan=1 colspan=1>99.3</td><td rowspan=1 colspan=1>99.0-99.6</td></tr><tr><td rowspan=3 colspan=1>Flu B</td><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>2553/2553</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>99.8-100</td></tr><tr><td rowspan=1 colspan=1>Frozen</td><td rowspan=1 colspan=1>34/34</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>89.9-100</td><td rowspan=1 colspan=1>353/356</td><td rowspan=1 colspan=1>99.2</td><td rowspan=1 colspan=1>97.6-99.7</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>34/34</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>89.9-100</td><td rowspan=1 colspan=1>2906/2909e</td><td rowspan=1 colspan=1>99.9</td><td rowspan=1 colspan=1>99.7-100</td></tr><tr><td rowspan=1 colspan=1>RSV</td><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>14/15</td><td rowspan=1 colspan=1>93.3</td><td rowspan=1 colspan=1>70.2-98.8</td><td rowspan=1 colspan=1>2538/2538</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>99.8-100</td></tr></table>

<table><tr><td rowspan=1 colspan=1>Frozen</td><td rowspan=1 colspan=1>55/57</td><td rowspan=1 colspan=1>96.5</td><td rowspan=1 colspan=1>88.1-99.0</td><td rowspan=1 colspan=1>333/333</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>98.9-100</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>69/72f</td><td rowspan=1 colspan=1>95.8</td><td rowspan=1 colspan=1>88.5-98.6</td><td rowspan=1 colspan=1>2871/2871</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>99.9-100</td></tr></table>

<table><tr><td>TP-true positive; FN-false negative; TN-true negative; FP-false positive; NA-not applicable a Discrepant test results from a U.S. FDA EUA SARS-CoV-2 molecular test: 2/8 specimens were SARS-CoV-2 positive and 6/8 were SARS-CoV-2 negative. b Discrepant test results from a U.S. FDA EUA SARS-CoV-2 molecular test: 7/24 specimens were SARS-CoV-2 positive, 15/24 were SARS-CoV-2 negative, 1/24 was invalid, and 1/24 was not</td></tr><tr><td>retested. cDiscrepant test results from a U.S. FDA-cleared molecular respiratory panel: 2/4 specimens were Flu</td></tr><tr><td>A positive, and 2/4 specimens were Flu A negative.</td></tr><tr><td>d Discrepant test results from a U.S. FDA-cleared molecular respiratory panel: 4/18 were Flu A positive, 1/18 Flu were Flu A negative, and 13/18 were not retested due to the specimens being stored</td></tr><tr><td>for a longer duration than recommended in the package insert. e Discrepant test results from a U.S. FDA-cleared molecular respiratory panel: None of the three</td></tr><tr><td>specimens that yielded FP Flu B results were rested due to the specimens being stored for a longer</td></tr><tr><td>duration than recommended in the package insert. f Discrepant test results from a U.S. FDA-cleared molecular respiratory panel: 1/3 specimens were RSV positive, 2/3 were not retested due to the specimens being stored for a longer duration than</td></tr></table>

The number of specimens with positive results for more than one or more target analyte by the Xpert Xpress CoV-2/Flu/RSV plus and/or comparator assay(s) is presented in Table 18 and Table 19, where bolded values indicate concordant results. As shown in Table 18, a total of 4921 specimens collected in 2022 yielded valid results for all four targets for both the Xpert Xpress CoV-2/Flu/RSV plus and comparator assays. Note that Flu B is not presented in Table 18, as no Flu B positives were obtained during the 2022 study. The rate of coinfection for Xpert Xpress CoV-2/Flu/RSV plus was $0 . 3 \%$ (3/1200) and the rate of coinfection by the comparators was $0 . 3 \%$ (3/1154). In total, 4 co-infections were detected by either the Xpert Xpress CoV-2/Flu/RSV plus test or the comparator assays for specimens collected in 2022. Two (2) of these co-infections were discordant between the Xpert Xpress CoV-2/Flu/RSV plus and the comparator assays (see Table 18).

Table 18. Summary of Multi-Target Detections for the Xpert Xpress CoV-2/Flu/RSV plus Test – NPS and NS Specimens Collected in 2022   

<table><tr><td rowspan=2 colspan=2></td><td rowspan=1 colspan=7>Comparator Assay Results*, **</td></tr><tr><td rowspan=1 colspan=1>SARS-CoV-2</td><td rowspan=1 colspan=1>SARS-CoV-2&amp; Flu A</td><td rowspan=1 colspan=1>Flu A</td><td rowspan=1 colspan=1>RSV</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>Co-InfectionRate (%)</td></tr><tr><td rowspan=7 colspan=1>Sp //-  S</td><td rowspan=1 colspan=1>SARS-CoV-2</td><td rowspan=1 colspan=1>876</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>57</td><td rowspan=1 colspan=1>934</td><td rowspan=6 colspan=1>0.3%(0/1200)</td></tr><tr><td rowspan=1 colspan=1>SARS-CoV-2&amp; Flu A</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3</td></tr><tr><td rowspan=1 colspan=1>Flu A</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>220</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>237</td></tr><tr><td rowspan=1 colspan=1>RSV</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>26</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>26</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>22</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>3693</td><td rowspan=1 colspan=1>3721</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>898</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>226</td><td rowspan=1 colspan=1>27</td><td rowspan=1 colspan=1>3767</td><td rowspan=1 colspan=1>4921</td></tr><tr><td rowspan=1 colspan=1>Co-InfectionRate (%)</td><td rowspan=1 colspan=6>0.3% (3/1154)</td><td rowspan=1 colspan=1></td></tr></table>

\*The comparator assay for SARS-CoV-2 was a U.S. FDA-cleared molecular respiratory panel that includes SARS-CoV-2. The comparator assay for Flu A, Flu B, and RSV was a U.S. FDA-cleared molecular Flu A/B/RSV assay.

\*\*Flu B is not presented as there were no Flu B detections in the 2022 prospective clinical study by either the Xpert Xpress CoV-2/Flu/RSV plus or the comparator assay(s).

As shown in Table 19, a total of 5,594 specimens, collected in 2022 and 2016-2017, yielded valid results for Flu A, Flu B, and RSV by both the Xpert Xpress CoV-2/Flu/RSV plus and comparator assay. The co-infection rate for Xpert Xpress CoV-2/Flu/RSV plus was $1 . 7 \%$ (11/652) and the co-infection rate by the comparator was $1 . 1 \%$ (7/623). In total, $1 3 \ \mathrm { c o - }$ infections were detected by either the Xpert Xpress CoV-2/Flu/RSV plus test or the comparator assay, of which 8 were discordant (see Table 19).

Table 19. Summary of Multi-Target Detections for the Xpert Xpress CoV-2/Flu/RSV plus Test – NPS and NS Specimens Collected in 2016-2017 and 2022   

<table><tr><td rowspan=2 colspan=2></td><td rowspan=1 colspan=9>Comparator Assay Results*</td></tr><tr><td rowspan=1 colspan=1>Flu A</td><td rowspan=1 colspan=1>Flu B</td><td rowspan=1 colspan=1>RSV</td><td rowspan=1 colspan=1>Flu A&amp;Flu B</td><td rowspan=1 colspan=1>Flu A&amp;RSV</td><td rowspan=1 colspan=1>Flu B&amp;RSV</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>Co-InfectionRate (%)</td></tr><tr><td rowspan=9 colspan=1>Su S//-  S</td><td rowspan=1 colspan=1>Flu A</td><td rowspan=1 colspan=1>381</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>36</td><td rowspan=1 colspan=1>419</td><td rowspan=8 colspan=1>1.7%(11/652)</td></tr><tr><td rowspan=1 colspan=1>Flu B</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>85</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>87</td></tr><tr><td rowspan=1 colspan=1>RSV</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>135</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>135</td></tr><tr><td rowspan=1 colspan=1>Flu A &amp;Flu B</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>6</td></tr><tr><td rowspan=1 colspan=1>Flu A &amp;RSV</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>4</td></tr><tr><td rowspan=1 colspan=1>Flu B &amp;RSV</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>5292</td><td rowspan=1 colspan=1>5302</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>387</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>139</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>5331</td><td rowspan=1 colspan=1>5954</td></tr><tr><td rowspan=1 colspan=1>Co-InfectionRate (%)</td><td rowspan=1 colspan=8>1.1% (7/623)</td><td rowspan=1 colspan=1></td></tr></table>

\*The comparator assay for Flu A, Flu B, and RSV was a U.S. FDA-cleared molecular Flu A/B/RSV assay.

2. Other Clinical Supportive Data (When 1. and 2. Are Not Applicable): Not Applicable.

D Clinical Cut-Off: Not Applicable.

# E Expected Values/Reference Range:

The 2022 prospective clinical study included a total of 5051 specimens (2536 NPS and 2515 NS) with valid Xpert Xpress CoV-2/Flu/RSV plus test and comparator SARS-CoV-2 results. The

2022 and 2016-2017 prospective clinical studies included 5172 specimens (2593 NPS and 2579 NS) and 782 specimens (418 NPS and 364 NS), respectively, with valid Flu A, Flu B, and RSV results. The number and percentage of cases positive for SARS-CoV-2, Flu A, Flu B, and RSV, as determined by the Xpert Xpress CoV-2/Flu/RSV plus test are shown in Table 20, stratified by collection period and age group, and Table 21, stratified by collection period and specimen type.

Table 20. Cepheid Xpert Xpress CoV-2/Flu/RSV plus - Expected Values Stratified by Collection Period and Age Group   

<table><tr><td rowspan=3 colspan=1>Target</td><td rowspan=1 colspan=5>Expected Values for Prospectively Collected Specimens from 2022(Category I and Category II)a</td></tr><tr><td rowspan=2 colspan=1>Overall</td><td rowspan=1 colspan=4>Age Group (years)</td></tr><tr><td rowspan=1 colspan=1>&lt;5</td><td rowspan=1 colspan=1>6-21</td><td rowspan=1 colspan=1>22-59</td><td rowspan=1 colspan=1>&gt;60</td></tr><tr><td rowspan=1 colspan=1>SARS-CoV-2</td><td rowspan=1 colspan=1>19.2%(971/5051)</td><td rowspan=1 colspan=1>11.3%(21/186)</td><td rowspan=1 colspan=1>13.8%(152/1099)</td><td rowspan=1 colspan=1>21.7%(667/3076)</td><td rowspan=1 colspan=1>19.0%(131/690)</td></tr><tr><td rowspan=1 colspan=1>Flu A</td><td rowspan=1 colspan=1>4.8%(250/5172)</td><td rowspan=1 colspan=1>6.6%(12/181)</td><td rowspan=1 colspan=1>9.2%(104/1136)</td><td rowspan=1 colspan=1>3.8%(119/3152)</td><td rowspan=1 colspan=1>2.1%(15/703)</td></tr><tr><td rowspan=1 colspan=1>Flu B</td><td rowspan=1 colspan=1>0.0%(0/5172)</td><td rowspan=1 colspan=1>0.0%(0/181)</td><td rowspan=1 colspan=1>0.0%(0/1136)</td><td rowspan=1 colspan=1>0.0%(0/3152)</td><td rowspan=1 colspan=1>0.0%(0/703)</td></tr><tr><td rowspan=1 colspan=1>RSV</td><td rowspan=1 colspan=1>0.5%(26/5172)</td><td rowspan=1 colspan=1>1.1%(2/181)</td><td rowspan=1 colspan=1>0.7%(8/1136)</td><td rowspan=1 colspan=1>0.5%(15/3152)</td><td rowspan=1 colspan=1>0.1%(1/703)</td></tr><tr><td rowspan=3 colspan=1>Target</td><td rowspan=1 colspan=5>Expected Values for Prospectively Collected Specimens from 2016-2017(Category II)</td></tr><tr><td rowspan=2 colspan=1>Overall</td><td rowspan=1 colspan=4>Age Group (years)</td></tr><tr><td rowspan=1 colspan=1>&lt;5</td><td rowspan=1 colspan=1>6-21</td><td rowspan=1 colspan=1>22-59</td><td rowspan=1 colspan=1>&gt;60</td></tr><tr><td rowspan=1 colspan=1>SARS-CoV-2</td><td rowspan=1 colspan=5>N/Ab</td></tr><tr><td rowspan=1 colspan=1>Flu A</td><td rowspan=1 colspan=1>22.9%(179/782)</td><td rowspan=1 colspan=1>18.4%(56/305)</td><td rowspan=1 colspan=1>40.4%(63/156)</td><td rowspan=1 colspan=1>16.1%(39/242)</td><td rowspan=1 colspan=1>26.6%(21/79)</td></tr><tr><td rowspan=1 colspan=1>Flu B</td><td rowspan=1 colspan=1>12.0%(94/782)</td><td rowspan=1 colspan=1>10.2%(31/305)</td><td rowspan=1 colspan=1>19.2%(30/156)</td><td rowspan=1 colspan=1>12.0%(29/242)</td><td rowspan=1 colspan=1>5.1%(4/79)</td></tr><tr><td rowspan=1 colspan=1>RSV</td><td rowspan=1 colspan=1>14.6%(114/782)</td><td rowspan=1 colspan=1>30.8%(94/305)</td><td rowspan=1 colspan=1>0.6%(1/156)</td><td rowspan=1 colspan=1>3.7%(9/242)</td><td rowspan=1 colspan=1>12.7%(10/79)</td></tr></table>

N/A-Not Applicable a Fifty-nine (59) of the specimens prospectively collected in 2022 were Category II. b Specimens collected prior to the COVID-19 pandemic were expected to be negative for SARS-CoV-2 and were only tested for the Flu A, Flu B, and RSV targets.

Table 21. Cepheid Xpert Xpress CoV-2/Flu/RSV plus - Expected Values Stratified by Collection Period and Specimen Type   

<table><tr><td rowspan=2 colspan=1>Target</td><td rowspan=1 colspan=3>Expected Values for ProspectivelyCollected specimens from 2022(Category I and Category II)a</td><td rowspan=1 colspan=3>Expected Values for ProspectivelyCollected Specimens from 2016-2017(Category II)</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>NPS</td><td rowspan=1 colspan=1>NS</td><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>NPS</td><td rowspan=1 colspan=1>NS</td></tr><tr><td rowspan=1 colspan=1>SARS-CoV-2</td><td rowspan=1 colspan=1>19.2%(971/5051)</td><td rowspan=1 colspan=1>19.7%(499/2536)</td><td rowspan=1 colspan=1>18.8%(472/2515)</td><td rowspan=1 colspan=3>N/Ab</td></tr><tr><td rowspan=1 colspan=1>Flu A</td><td rowspan=1 colspan=1>4.8%(250/5172)</td><td rowspan=1 colspan=1>4.3%(111/2593)</td><td rowspan=1 colspan=1>5.4%(139/2579)</td><td rowspan=1 colspan=1>22.9%(179/782)</td><td rowspan=1 colspan=1>24.9%(104/418)</td><td rowspan=1 colspan=1>20.6%(75/364)</td></tr><tr><td rowspan=1 colspan=1>Flu B</td><td rowspan=1 colspan=1>0.0%(0/5172)</td><td rowspan=1 colspan=1>0.0%(0/2593)</td><td rowspan=1 colspan=1>0.0%(0/2579)</td><td rowspan=1 colspan=1>12.0%(94/782)</td><td rowspan=1 colspan=1>13.6%(57/418)</td><td rowspan=1 colspan=1>10.2%(37/364)</td></tr><tr><td rowspan=1 colspan=1>RSV</td><td rowspan=1 colspan=1>0.5%(26/5172)</td><td rowspan=1 colspan=1>0.5%(12/2593)</td><td rowspan=1 colspan=1>0.5%(14/2579)</td><td rowspan=1 colspan=1>14.6%(114/782)</td><td rowspan=1 colspan=1>14.1%(59/418)</td><td rowspan=1 colspan=1>15.1%(55/364)</td></tr></table>

N/A-Not Applicable   
a Fifty-nine (59) of the specimens prospectively collected in 2022 were Category II.   
b Specimens collected prior to the COVID-19 pandemic were expected to be negative for SARS-CoV-2 and were only tested for the Flu A, Flu B, and RSV targets.

F Other Supportive Instrument Performance Characteristics Data: Not Applicable.

# VIII Proposed Labeling:

The labeling supports the finding of substantial equivalence for this device.

# IX Conclusion:

The submitted information in this premarket notification is complete and supports a substantial equivalence decision.