Luminex Molecular Diagnostics, Inc. Kate Goscha   
Senior Manager, Regulatory Affairs 439 University Avenue   
Toronto, Ontario M5G 1Y8   
Canada

Re: K231758 Trade/Device Name: NxTAG Respiratory Pathogen Panel v2 (NxTAG RPP v2) Regulation Number: 21 CFR 866.3981 Regulation Name: Device To Detect And Identify Nucleic Acid Targets In Respiratory Specimens From Microbial Agents That Cause The SARS-Cov-2 Respiratory Infection And Other Microbial Agents When In A Multi-Target Test Regulatory Class: Class II Product Code: QOF Dated: June 15, 2023 Received: June 16, 2023

Dear Kate Goscha:

We have reviewed your section 510(k) premarket notification of intent to market the device referenced above and have determined the device is substantially equivalent (for the indications for use stated in the enclosure) to legally marketed predicate devices marketed in interstate commerce prior to May 28, 1976, the enactment date of the Medical Device Amendments, or to devices that have been reclassified in accordance with the provisions of the Federal Food, Drug, and Cosmetic Act (the Act) that do not require approval of a premarket approval application (PMA). You may, therefore, market the device, subject to the general controls provisions of the Act. Although this letter refers to your product as a device, please be aware that some cleared products may instead be combination products. The 510(k) Premarket Notification Database available at https://www.accessdata.fda.gov/scripts/cdrh/cfdocs/cfpmn/pmn.cfm identifies combination product submissions. The general controls provisions of the Act include requirements for annual registration, listing of devices, good manufacturing practice, labeling, and prohibitions against misbranding and adulteration. Please note: CDRH does not evaluate information related to contract liability warranties. We remind you, however, that device labeling must be truthful and not misleading.

If your device is classified (see above) into either class II (Special Controls) or class III (PMA), it may be subject to additional controls. Existing major regulations affecting your device can be found in the Code of Federal Regulations, Title 21, Parts 800 to 898. In addition, FDA may publish further announcements concerning your device in the Federal Register.

Additional information about changes that may require a new premarket notification are provided in the FDA guidance documents entitled "Deciding When to Submit a 510(k) for a Change to an Existing Device"

(https://www.fda.gov/media/99812/download) and "Deciding When to Submit a 510(k) for a Software Change to an Existing Device" (https://www.fda.gov/media/99785/download).

Your device is also subject to, among other requirements, the Quality System (QS) regulation (21 CFR Part 820), which includes, but is not limited to, 21 CFR 820.30, Design controls; 21 CFR 820.90, Nonconforming product; and 21 CFR 820.100, Corrective and preventive action. Please note that regardless of whether a change requires premarket review, the QS regulation requires device manufacturers to review and approve changes to device design and production (21 CFR 820.30 and 21 CFR 820.70) and document changes and approvals in the device master record (21 CFR 820.181).

Please be advised that FDA's issuance of a substantial equivalence determination does not mean that FDA has made a determination that your device complies with other requirements of the Act or any Federal statutes and regulations administered by other Federal agencies. You must comply with all the Act's requirements, including, but not limited to: registration and listing (21 CFR Part 807); labeling (21 CFR Part 801 and Part 809); medical device reporting (reporting of medical device-related adverse events) (21 CFR Part 803) for devices or postmarketing safety reporting (21 CFR Part 4, Subpart B) for combination products (see https://www.fda.gov/combination-products/guidance-regulatory-information/postmarketing-safetyreporting-combination-products); good manufacturing practice requirements as set forth in the quality systems (QS) regulation (21 CFR Part 820) for devices or current good manufacturing practices (21 CFR Part 4, Subpart A) for combination products; and, if applicable, the electronic product radiation control provisions (Sections 531-542 of the Act); 21 CFR Parts 1000-1050.

Also, please note the regulation entitled, "Misbranding by reference to premarket notification" (21 CFR 807.97). For questions regarding the reporting of adverse events under the MDR regulation (21 CFR Part 803), please go to https://www.fda.gov/medical-devices/medical-device-safety/medical-device-reportingmdr-how-report-medical-device-problems.

For comprehensive regulatory information about medical devices and radiation-emitting products, including information about labeling regulations, please see Device Advice (https://www.fda.gov/medical  
devices/device-advice-comprehensive-regulatory-assistance) and CDRH Learn   
(https://www.fda.gov/training-and-continuing-education/cdrh-learn). Additionally, you may contact the Division of Industry and Consumer Education (DICE) to ask a question about a specific regulatory topic. See the DICE website (https://www.fda.gov/medical-devices/device-advice-comprehensive-regulatoryassistance/contact-us-division-industry-and-consumer-education-dice) for more information or contact DICE by email (DICE@fda.hhs.gov) or phone (1-800-638-2041 or 301-796-7100).

# Sincerely, Joseph Briggs -S

Joseph Briggs   
Deputy Branch Chief   
Division of Microbiology Devices   
OHT7: Office of In Vitro Diagnostics   
Office of Product Evaluation and Quality   
Center for Devices and Radiological Health

Enclosure

510(k) Number (if known) K231758

Device Name NxTAG $\textsuperscript { \textregistered }$ Respiratory Pathogen Panel v2 (NxTAG $\textsuperscript { \textregistered }$ RPP v2)

Indications for Use (Describe)   
The NxTAG $^ \mathrm { \textregistered }$ Respiratory Pathogen Panel v2 (NxTAG $^ \mathrm { \textregistered }$ RPP v2) is a multiplexed polymerase chain reaction (PCR) test intended for the simultaneous, qualitative detection and identification of multiple respiratory viral and bacterial nucleic acids in nasopharyngeal swab specimens obtained from individuals with signs and symptoms of respiratory tract infection, including COVID-19.

The following organism types and subtypes are identified and differentiated using the NxTAG RPPv2:

Viral Targets: Influenza A, Influenza A H1, Influenza A H1pdm09, Influenza A H3, Influenza B, Respiratory Syncytial Virus A, Respiratory Syncytial Virus B, SARS-CoV-2, Coronavirus 229E, Coronavirus OC43, Coronavirus NL63, Coronavirus HKU1, Human Metapneumovirus, Rhinovirus/Enterovirus, Adenovirus, Parainfluenza 1, Parainfluenza 2, Parainfluenza 3, Parainfluenza 4

Bacterial Targets: Chlamydia pneumoniae, Mycoplasma pneumoniae

Nucleic acids from the viral and bacterial organisms identified by this test are generally detectable in nasopharyngeal specimens during the acute phase of infection. The detection and identification of specific viral and bacterial nucleic acids from individuals exhibiting signs and/or symptoms of respiratory infection are indicative of the presence of the identified microorganism and aids in diagnosis if used in conjunction with other clinical and epidemiological information, and laboratory findings. The results of this test should not be used as the sole basis for diagnosis, treatment, or other patient management decisions.

Negative results in the setting of a respiratory illness may be due to infection with pathogens that are not detected by this test, or lower respiratory tract infection that may not be detected by a nasopharyngeal swab specimen. Positive results do not rule out coinfection with other organisms. The agent(s) detected by the NxTAG RPP v2 may not be the definite cause of disease.

Additional laboratory testing (e.g., bacterial and viral culture, immunofluorescence, and radiography) may be necessary when evaluating a patient with possible respiratory tract infection.

The NxTAG $^ \mathrm { \textregistered }$ Respiratory Pathogen Panel v2 is indicated for use with the Luminex $^ \mathrm { \textregistered }$ MAGPIX $\textsuperscript { \textregistered }$ Instrument and xPONENT $\textsuperscript { \textregistered }$ and SYNCT™ software.

This section applies only to requirements of the Paperwork Reduction Act of 1995.

# \*DO NOT SEND YOUR COMPLETED FORM TO THE PRA STAFF EMAIL ADDRESS BELOW.\*

The burden time for this collection of information is estimated to average 79 hours per response, including the time to review instructions, search existing data sources, gather and maintain the data needed and complete and review the collection of information. Send comments regarding this burden estimate or any other aspect of this information collection, including suggestions for reducing this burden, to:

Department of Health and Human Services Food and Drug Administration Office of Chief Information Officer Paperwork Reduction Act (PRA) Staff PRAStaff@fda.hhs.gov

“An agency may not conduct or sponsor, and a person is not required to respond to, a collection of information unless it displays a currently valid OMB number.”

# 10.0 510(k) Summary

This Summary of 510(k) safety and effectiveness information is being submitted in accordance with the requirements of 21 CFR 807.92.

Date Prepared: February 27, 2024

A. 510(k) Number: K231758   
B. Purpose for Submission: Traditional 510(k)   
C. Measurand:

The assay detects and identifies nucleic acids of the following respiratory pathogens: Influenza A, Influenza A H1, Influenza A H1pdm09, Influenza A H3, Influenza B, Respiratory Syncytial Virus A, Respiratory Syncytial Virus B, Severe Acute Respiratory Syndrome Coronavirus (SARS-CoV-2), Coronavirus 229E, Coronavirus OC43, Coronavirus NL63, Coronavirus HKU1, Human Metapneumovirus, Rhinovirus/Enterovirus, Adenovirus, Parainfluenza 1, Parainfluenza 2, Parainfluenza 3, Parainfluenza 4, Chlamydia pneumoniae, and Mycoplasma pneumoniae.

# D. Type of Test:

A multiplexed nucleic acid test intended for use with the Luminex® MAGPIX® Instrument, and xPONENT® and SYNCT™ software, for the qualitative detection and identification of multiple respiratory viral and bacterial nucleic acids in nasopharyngeal swabs (NPS) obtained from individuals with clinical signs and symptoms of a respiratory tract infection, including COVID-19.

# E. Applicant:

Kate Goscha   
439 University Avenue   
Toronto, Ontario M5G 1Y8   
Canada   
(608) 203-8909   
Luminex Molecular Diagnostics, Inc.

# F. Proprietary and Established Names:

NxTAG® Respiratory Pathogen Panel v2 (NxTAG® RPP v2)

# G. Regulatory Information:

1. Regulation Section:   
21 CFR 866.3981

2. Classification: Class II (special controls)

3. Product Code(s): QOF

4. Panel: Microbiology (83)

# H. Indications for Use:

1. Indication(s) for use:

The $\mathsf { N } \times \mathsf { T } \mathsf { A G } ^ { \textregistered }$ Respiratory Pathogen Panel v2 (NxTAG® RPP v2) is a multiplexed polymerase chain reaction (PCR) test intended for the simultaneous, qualitative detection and identification of multiple respiratory viral and bacterial nucleic acids in nasopharyngeal swab specimens obtained from individuals with signs and symptoms of respiratory tract infection, including COVID-19.

The following organism types and subtypes are identified and differentiated using the NxTAG RPP v2:

<table><tr><td colspan="2" rowspan="1">Viral Targets</td></tr><tr><td colspan="1" rowspan="1">Influenza A</td><td colspan="1" rowspan="1">Coronavirus NL63</td></tr><tr><td colspan="1" rowspan="1">Influenza A H1</td><td colspan="1" rowspan="1">Coronavirus HKU1</td></tr><tr><td colspan="1" rowspan="1">Influenza A H1pdm09</td><td colspan="1" rowspan="1">Human Metapneumovirus</td></tr><tr><td colspan="1" rowspan="1">Influenza A H3</td><td colspan="1" rowspan="1">Rhinovirus/Enterovirus</td></tr><tr><td colspan="1" rowspan="1">Influenza B</td><td colspan="1" rowspan="1">Adenovirus</td></tr><tr><td colspan="1" rowspan="1">Respiratory Syncytial Virus A</td><td colspan="1" rowspan="1">Parainfluenza 1</td></tr><tr><td colspan="1" rowspan="1">Respiratory Syncytial Virus B</td><td colspan="1" rowspan="1">Parainfluenza 2</td></tr><tr><td colspan="1" rowspan="1">SARS-CoV-2</td><td colspan="1" rowspan="1">Parainfluenza 3</td></tr><tr><td colspan="1" rowspan="1">Coronavirus 229E</td><td colspan="1" rowspan="1">Parainfluenza 4</td></tr><tr><td colspan="1" rowspan="1">Coronavirus OC43</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="2" rowspan="1">Bacterial Targets</td></tr><tr><td colspan="1" rowspan="1">Chlamydia pneumoniae</td><td colspan="1" rowspan="1">Mycoplasma pneumoniae</td></tr></table>

Nucleic acids from the viral and bacterial organisms identified by this test are generally detectable in nasopharyngeal specimens during the acute phase of infection. The detection and identification of specific viral and bacterial nucleic acids from individuals exhibiting signs and/or symptoms of respiratory infection are indicative of the presence of the identified microorganism and aids in diagnosis if used in conjunction with other clinical and epidemiological information, and laboratory findings. The results of this test should not be used as the sole basis for diagnosis, treatment, or other patient management decisions.

Negative results in the setting of a respiratory illness may be due to infection with pathogens that are not detected by this test, or lower respiratory tract infection that may not be detected by a nasopharyngeal swab specimen. Positive results do not rule out coinfection with other organisms. The agent(s) detected by the NxTAG RPP v2 may not be the definite cause of disease.

Additional laboratory testing (e.g., bacterial and viral culture, immunofluorescence, and radiography) may be necessary when evaluating a patient with possible respiratory tract infection.

The $\mathsf { N } \times \mathsf { T } \mathsf { A G } ^ { \textregistered }$ Respiratory Pathogen Panel v2 is indicated for use with the Luminex® MAGPIX® Instrument and xPONENT® and SYNCT™ software.

2. Special conditions for use statement(s): For prescription use only. For in vitro diagnostic use.

3. Special instrument requirements:

Sample Processing: o Extraction: bioMérieux NUCLISENS® easyMAG® or EMAG® nucleic acid extraction systems o Amplification: IVD-labeled Thermal Cycler required Sample Analysis: Luminex® MAGPIX® Instrument using xPONENT® and SYNCT™ software

# I. Device Description:

The $\mathsf { N } \times \mathsf { T } \mathsf { A G } ^ { \otimes }$ Respiratory Pathogen Panel v2 $\mathsf { N } \times \mathsf { T } \mathsf { A G } ^ { \otimes }$ RPP v2) is designed to simultaneously detect and identify 21 different potential pathogens of respiratory tract infections, including the novel coronavirus SARS-CoV-2, from a single NPS specimen in transport medium. NxTAG® RPP v2 is compatible with Luminex’s MAGPIX Instrument, and xPONENT® and SYNCT™ software. It incorporates multiplex Reverse Transcriptase Polymerase Chain Reaction (RT-PCR) with the Luminex® proprietary universal tag sorting system on the Luminex platform to easily detect the 21 respiratory pathogen targets.

Samples are extracted using the IVD‐labeled bioMérieux NucliSENS® easyMag® or EMAG® extraction systems. Extracted total nucleic acid is then added to the sealed 96‐well micro plate by piercing the seal with pipette tips. Each reaction well is pre‐plated with two Lyophilized Bead Reagents (LBRs) that contain all the required reagents including primer mixes, bead mix, and enzyme buffer systems. Once the LBRs are resuspended, the reaction wells are re‐sealed using the foils provided in the kit. The sealed plate can be placed inside the thermocycler. The reaction is amplified via RT‐PCR and the reaction product undergoes near simultaneous bead hybridization within the sealed reaction wells. The hybridized, tagged beads are then sorted and read on the Luminex® MAGPIX® instrument. The MAGPIX® instrument generates a signal in the form of a median fluorescence intensity (MFI) value for each bead population.

The signals are analyzed using the $\mathsf { N x T A G } ^ { \otimes }$ Respiratory Pathogen Panel v2 Assay File for SYNCT™ Software, providing a reliable, qualitative call for each of the 21 targets and internal controls within each reaction well.

# J. Substantial Equivalence Information:

1. Predicate device name(s): BioFire Respiratory Panel 2.1 (RP2.1)

2. Predicate 510(k) number(s): DEN200031

3. Comparison with predicate:

<table><tr><td rowspan=1 colspan=1>Device &amp; Predicate Device(s):</td><td rowspan=1 colspan=1>K231758</td><td rowspan=1 colspan=1>DEN200031</td></tr><tr><td rowspan=1 colspan=1>Device Trade Name</td><td rowspan=1 colspan=1>NxTAG Respiratory PathogenPanel v2</td><td rowspan=1 colspan=1>BioFire Respiratory Panel 2.1</td></tr><tr><td rowspan=1 colspan=1>General Device CharacteristicSimilarities</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Intended Use/Indications ForUse</td><td rowspan=1 colspan=1>The Luminex NxTAG RespiratoryPathogen Panel v2 (NxTAG RPPv2) is a multiplexed polymerasechain reaction (PCR) testintended for the simultaneous,qualitative detection andidentification of multiplerespiratory viral and bacterialnucleic acids in nasopharyngealswab specimens obtained fromindividuals with signs andsymptoms of respiratory tractinfection, including COVID-19.The following organism typesand subtypes are identified anddifferentiated using the NxTAGRPP v2:Viral targets:</td><td rowspan=1 colspan=1>The BioFire Respiratory Panel2.1 (RP2.1) is a PCR-basedmultiplexed nucleic acid testintended for use with theBioFire FilmArray 2.0 or BioFireFilmArray Torch systems for thesimultaneous qualitativedetection and identification ofmultiple respiratory viral andbacterial nucleic acids innasopharyngeal swabs (NPS)obtained from individualssuspected of respiratory tractinfections, including COVID-19.The following organism typesand subtypes are identifiedusing the BioFire RP2.1:Adenovirus</td></tr></table>

<table><tr><td></td><td>Influenza A</td></tr><tr><td>Influenza A H1</td><td>Coronavirus HKU1</td></tr><tr><td>Influenza A H1pdm09</td><td>Coronavirus NL63</td></tr><tr><td>Influenza A H3</td><td>Coronavirus OC43</td></tr><tr><td>Influenza B</td><td>Severe Acute Respiratory</td></tr><tr><td>Respiratory Syncytial Virus A</td><td>Syndrome Coronavirus (SARS-</td></tr><tr><td>Respiratory Syncytial Virus B</td><td>CoV-2)</td></tr><tr><td>SARS-CoV-2</td><td>Human Metapneumovirus</td></tr><tr><td>Coronavirus 229E</td><td>Human Rhinovirus/Enterovirus</td></tr><tr><td>Coronavirus OC43</td><td>Influenza A, including subtypes H1, H1-2009, and H3</td></tr><tr><td>Coronavirus NL63</td><td>Influenza B</td></tr><tr><td>Coronavirus HKU1</td><td>Parainfluenza Virus 1</td></tr><tr><td>Human Metapneumovirus</td><td>Parainfluenza Virus 2</td></tr><tr><td>Rhinovirus/Enterovirus</td><td>Parainfluenza Virus 3</td></tr><tr><td>Adenovirus</td><td>Parainfluenza Virus 4</td></tr><tr><td>Parainfluenza Virus 1</td><td>Respiratory Syncytial Virus</td></tr><tr><td>Parainfluenza Virus 2</td><td>Bordetella parapertussis</td></tr><tr><td>Parainfluenza Virus 3</td><td>(IS1001)</td></tr><tr><td>Parainfluenza Virus 4</td><td>Bordetella pertussis (prxP)</td></tr><tr><td>Bacterial targets:</td><td>Chlamydia pneumoniae and</td></tr><tr><td>Chlamydia pneumoniae</td><td>Mycoplasma pneumoniae</td></tr><tr><td>Mycoplasma pneumoniae</td><td>Nucleic acids from the</td></tr><tr><td>Nucleic acids from the viral and</td><td>respiratory viral and bacterial</td></tr><tr><td>bacterial organisms identified</td><td>organisms identified by this test</td></tr><tr><td>by this test are generally</td><td>are generally detectable in NPS</td></tr><tr><td>detectable in nasopharyngeal</td><td>specimens during the acute</td></tr><tr><td>specimens during the acute</td><td>phase of infection. The detection and identification of</td></tr><tr><td>phase of infection. The</td><td>specific viral and bacterial</td></tr><tr><td>detection and identification of</td><td></td></tr><tr><td></td><td>nucleic acids from individuals</td></tr><tr><td>specific viral and bacterial</td><td></td></tr><tr><td>nucleic acids from individuals</td><td>exhibiting signs and/or</td></tr></table>

<table><tr><td colspan="1" rowspan="1">Specimen Type</td><td colspan="1" rowspan="1">Same</td><td colspan="1" rowspan="1">Nasopharyngeal swabs</td></tr><tr><td colspan="1" rowspan="1">Patient Population</td><td colspan="1" rowspan="1">Individuals with signs andsymptoms of respiratory tractinfection, including COVID-19</td><td colspan="1" rowspan="1">Individuals suspected ofrespiratory tract infections,including COVID-19</td></tr><tr><td colspan="1" rowspan="1">Organisms Detected</td><td colspan="1" rowspan="1">Same except for:addition of assay fordifferentiation ofRespiratory Syncytial Virus Aand Bomission of assays forBordetella pertussis andBordetella parapertussis</td><td colspan="1" rowspan="1">Viruses:AdenovirusCoronavirus 229ECoronavirus HKU1Coronavirus NL63Coronavirus OC43Severe Acute RespiratorySyndrome Coronavirus 2 (SARS-CoV-2)Human MetapneumovirusHuman Rhinovirus/EnterovirusInfluenza A VirusSubtypes: H1, H3, H1-2009Influenza B VirusParainfluenza Virus 1Parainfluenza Virus 2Parainfluenza Virus 3Parainfluenza Virus 4Respiratory Syncytial VirusBacteria:Bordetella parapertussisBordetella pertussisChlamydia pneumoniaeMycoplasma pneumoniae</td></tr><tr><td colspan="1" rowspan="1">Technology</td><td colspan="1" rowspan="1">Same</td><td colspan="1" rowspan="1">PCR amplification</td></tr><tr><td colspan="1" rowspan="1">General Device CharacteristicDifferences</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">System</td><td colspan="1" rowspan="1">Separate instruments fornucleic acid extraction, PCRamplification and detection</td><td colspan="1" rowspan="1">Integrated nucleic acidextraction, amplification, anddetection in a sealed vessel.</td></tr><tr><td colspan="1" rowspan="1">Assay Read</td><td colspan="1" rowspan="1">MAGPIX Instrument</td><td colspan="1" rowspan="1">BIOFIRE FilmARRAY 2.0 orBIOFIRE FilmArray TorchSystems</td></tr><tr><td colspan="1" rowspan="1">Detection</td><td colspan="1" rowspan="1">Hybridization of amplifiedproducts with fluorescently</td><td colspan="1" rowspan="1">Array-based melt curve analysis</td></tr><tr><td></td><td>labeled beads, sorting of tagged products</td><td></td></tr></table>

# K. Standards/Guidance Documents Referenced:

# Guidance

Highly Multiplexed Microbiological/Medical Countermeasure In Vitro Nucleic Acid   
Based Diagnostic Devices Guidance for Industry and Food and Drug Administration   
Staff – August 2014   
Respiratory Viral Panel Multiplex Nucleic Acid Assay – Class II Special Controls   
Guidance for Industry and FDA Staff – October 9, 2009   
Guidance for Industry and FDA Staff Statistical Guidance on Reporting Results from   
Studies Evaluating Diagnostic Tests, March 13, 2007   
Guidance on Informed Consent for In Vitro Diagnostic Device Studies using Leftover   
Human Specimens that are Not Individually Identifiable, January 2006   
Policy for Coronavirus Disease-2019 Tests during the Public Health Emergency   
(Revised). Immediately in Effect Guidance for Clinical Laboratories, Commercial   
Manufacturers, and Food and Drug Administration Staff, May 11, 2020

# Standards

• CLSI, EP05-A3, Evaluation of Precision of Quantitative Measurement Procedures; Approved Guideline – Third Edition CLSI, EP07, Interference Testing in Clinical Chemistry; 3rd Edition CLSI, EP12-A2, User Protocol for Evaluation of Qualitative Test Performance; Approved Guideline – Second Edition CLSI, EP17-A2, Evaluation of Detection Capability for Clinical Laboratory Measurement Procedures; Approved Guideline – Second Edition CLSI, EP24-A2, Assessment of the Diagnostic Accuracy of Laboratory Tests Using Receiver Operating Characteristic Curves; Approved Guideline–Second Edition CLSI, EP25-A, Evaluation of Stability of In Vitro Diagnostic Reagents; Approved Guideline   
• CLSI, EP37, Supplemental Tables for Interference Testing in Clinical Chemistry – First Edition CLSI, MM03, Molecular Diagnostic Methods for Infectious Diseases; Approved Guideline – Third Edition CLSI, MM09-A2, Nucleic Acid Sequencing Methods in Diagnostic Laboratory Medicine: Approved Guideline -- Second Edition CLSI, MM17, Validation and Verification of Multiplex Nucleic Acid Assays – Second Edition

# Luminex

NxTAG® Respiratory Pathogen Panel v2 Traditional 510(k) Submission

CLSI, MM18-A, Interpretive Criteria for Identification of Bacteria and Fungi by DNA   
Target Sequencing; Approved Guideline   
ISO 14971:2019, Medical Devices – Application of Risk Management to Medical   
Devices   
EN ISO 23640:2015, In vitro diagnostic medical devices – Evaluation of stability of in   
vitro diagnostics reagents

# L. Test Principle:

The $\mathsf { N } \times \mathsf { T } \mathsf { A G } ^ { \otimes }$ Respiratory Pathogen Panel v2 (NxTAG® RPP v2) incorporates multiplex Reverse Transcriptase Polymerase Chain Reaction (RT-PCR) with the Luminex® proprietary universal tag sorting system on the Luminex platform to detect respiratory pathogen targets. Extracted total nucleic acid is added to pre-plated, Lyophilized Bead Reagents (LBRs), and mixed to resuspend the reaction reagents. The reaction is amplified via RT-PCR and the reaction product undergoes near simultaneous microsphere hybridization within the sealed reaction well. The hybridized, tagged microspheres are then sorted and read on the MAGPIX® instrument. The generated signals are analyzed using the $\mathsf { N x T A G } ^ { \otimes }$ Respiratory Pathogen Panel v2 Assay File for SYNCT™ Software, providing a reliable, qualitative call for each of the targets and internal controls within each reaction well.

# M. Performance Characteristics:

# 1. Analytical performance:

a. Precision (Reproducibility and Repeatability):

# Site-to-Site Reproducibility

A site-to-site reproducibility study was performed to assess the total variability of the NxTAG® Respiratory Pathogen Panel (NxTAG RPP v2) assay across study sites, operators, testing days, and instruments. Two operators at each of the 3 sites tested a 9-member reproducibility panel in 4 replicates on 5 non-consecutive days, for a total of 30 runs (3 sites x 2 operators $\textsf { x 5 }$ days). For each member of the 9-member panel, a total of 120 data points (30 runs x 4 replicates) were generated using 1 assay kit lot. The reproducibility panel comprised of a negative sample and 4 multi-analyte (MA) samples prepared in negative simulated matrix (NSM) at two concentrations, Low Positive $( 1 . 5 \mathsf { x } - 3 \mathsf { x } \mathsf { L o D } )$ and Moderate Positive $( 5 \mathsf { x } - 9 \mathsf { x } \mathsf { L o D } )$ ). The test concentration for Influenza A H3 (in sample MA2) is based on the LoD for the subtype. Since the Influenza A H3 matrix LoD is 3-fold less sensitive than the subtype, the expected results for the matrix at $1 . 5 \times$ of the subtype LoD was either “positive” or “negative

# Luminex

(i.e. both results were acceptable based on the respective LoDs of the Influenza A and H3 targets). Therefore, the Influenza A matrix result in sample MA2 could not be assessed and was excluded from data analysis. The results demonstrated reproducibility of the NxTAG RPP v2 assay across 3 sites with an overall percent agreement of $9 9 . 9 0 \%$ . The summary of results is shown in Table 1.

Table 1: NxTAG® RPP v2 Site-to-site Reproducibility   

<table><tr><td rowspan=2 colspan=1>Sample</td><td rowspan=2 colspan=1>Target</td><td rowspan=2 colspan=1>Sample Type</td><td rowspan=1 colspan=5>Agreement with Expected Results</td></tr><tr><td rowspan=1 colspan=1>Site 1</td><td rowspan=1 colspan=1>Site 2</td><td rowspan=1 colspan=1>Site 3</td><td rowspan=1 colspan=2>Overall(All Sites)</td></tr><tr><td rowspan=8 colspan=1>MA1</td><td rowspan=2 colspan=1>Influenza AH1pdm09</td><td rowspan=1 colspan=1>Low Positive</td><td rowspan=1 colspan=1>40/40(100%)</td><td rowspan=1 colspan=1>40/40(100%)</td><td rowspan=1 colspan=1>40/40(100%)</td><td rowspan=1 colspan=1>120/120(100%)</td><td rowspan=2 colspan=1>240/240(100%)</td></tr><tr><td rowspan=1 colspan=1>Moderate Positive</td><td rowspan=1 colspan=1>40/40(100%)</td><td rowspan=1 colspan=1>40/40(100%)</td><td rowspan=1 colspan=1>40/40(100%)</td><td rowspan=1 colspan=1>120/120(100%)</td></tr><tr><td rowspan=2 colspan=1>RespiratorySyncytial VirusA</td><td rowspan=1 colspan=1>Low Positive</td><td rowspan=1 colspan=1>40/40(100%)</td><td rowspan=1 colspan=1>40/40(100%)</td><td rowspan=1 colspan=1>40/40(100%)</td><td rowspan=1 colspan=1>120/120(100%)</td><td rowspan=2 colspan=1>240/240(100%)</td></tr><tr><td rowspan=1 colspan=1>Moderate Positive</td><td rowspan=1 colspan=1>40/40(100%)</td><td rowspan=1 colspan=1>40/40(100%)</td><td rowspan=1 colspan=1>40/40(100%)</td><td rowspan=1 colspan=1>120/120(100%)</td></tr><tr><td rowspan=2 colspan=1>Rhinovirusa</td><td rowspan=1 colspan=1>Low Positive</td><td rowspan=1 colspan=1>40/40(100%)</td><td rowspan=1 colspan=1>40/40(100%)</td><td rowspan=1 colspan=1>40/40(100%)</td><td rowspan=1 colspan=1>120/120(100%)</td><td rowspan=2 colspan=1>240/240(100%)</td></tr><tr><td rowspan=1 colspan=1>Moderate Positive</td><td rowspan=1 colspan=1>40/40(100%)</td><td rowspan=1 colspan=1>40/40(100%)</td><td rowspan=1 colspan=1>40/40(100%)</td><td rowspan=1 colspan=1>120/120(100%)</td></tr><tr><td rowspan=2 colspan=1>All othertargets</td><td rowspan=1 colspan=1>Low Positive</td><td rowspan=1 colspan=1>719/720(99.86)</td><td rowspan=1 colspan=1>720/720(100%)</td><td rowspan=1 colspan=1>720/720(100%)</td><td rowspan=1 colspan=1>2159/2160(99.95%)</td><td rowspan=2 colspan=1>4316/4320(99.91%)</td></tr><tr><td rowspan=1 colspan=1>Moderate Positive</td><td rowspan=1 colspan=1>718/720(99.72%)</td><td rowspan=1 colspan=1>720/720(100%)</td><td rowspan=1 colspan=1>719/720(99.86%)</td><td rowspan=1 colspan=1>2157/2160(99.86%)</td></tr><tr><td rowspan=6 colspan=1>MA2</td><td rowspan=2 colspan=1>Influenza A H3</td><td rowspan=1 colspan=1>Low Positive</td><td rowspan=1 colspan=1>39/40(97.50%)</td><td rowspan=1 colspan=1>40/40(100%)</td><td rowspan=1 colspan=1>40/40(100%)</td><td rowspan=1 colspan=1>119/120(99.17%)</td><td rowspan=2 colspan=1>239/240(99.58%)</td></tr><tr><td rowspan=1 colspan=1>Moderate Positive</td><td rowspan=1 colspan=1>40/40(100%)</td><td rowspan=1 colspan=1>40/40(100%)</td><td rowspan=1 colspan=1>40/40(100%)</td><td rowspan=1 colspan=1>120/120(100%)</td></tr><tr><td rowspan=2 colspan=1>RespiratorySyncytial VirusB</td><td rowspan=1 colspan=1>Low Positive</td><td rowspan=1 colspan=1>40/40(100%)</td><td rowspan=1 colspan=1>40/40(100%)</td><td rowspan=1 colspan=1>40/40(100%)</td><td rowspan=1 colspan=1>120/120(100%)</td><td rowspan=2 colspan=1>240/240(100%)</td></tr><tr><td rowspan=1 colspan=1>Moderate Positive</td><td rowspan=1 colspan=1>40/40(100%)</td><td rowspan=1 colspan=1>40/40(100%)</td><td rowspan=1 colspan=1>40/40(100%)</td><td rowspan=1 colspan=1>120/120(100%)</td></tr><tr><td rowspan=2 colspan=1>All othertargets</td><td rowspan=1 colspan=1>Low Positive</td><td rowspan=1 colspan=1>717/720(99.58%)</td><td rowspan=1 colspan=1>720/720(100%)</td><td rowspan=1 colspan=1>720/720(100%)</td><td rowspan=1 colspan=1>2157/2160(99.86%)</td><td rowspan=2 colspan=1>4434/4440b(99.86%)</td></tr><tr><td rowspan=1 colspan=1>Moderate Positive</td><td rowspan=1 colspan=1>757/760(99.61%)</td><td rowspan=1 colspan=1>760/760(100%)</td><td rowspan=1 colspan=1>760/760(100%)</td><td rowspan=1 colspan=1>2277/2280(99.87%)</td></tr><tr><td rowspan=6 colspan=1>MA3</td><td rowspan=2 colspan=1>Influenza B</td><td rowspan=1 colspan=1>Low Positive</td><td rowspan=1 colspan=1>39/40(97.50%)</td><td rowspan=1 colspan=1>40/40(100%)</td><td rowspan=1 colspan=1>40/40(100%)</td><td rowspan=1 colspan=1>119/120(99.17%)</td><td rowspan=2 colspan=1>239/240(99.58%)</td></tr><tr><td rowspan=1 colspan=1>Moderate Positive</td><td rowspan=1 colspan=1>40/40(100%)</td><td rowspan=1 colspan=1>40/40(100%)</td><td rowspan=1 colspan=1>40/40(100%)</td><td rowspan=1 colspan=1>120/120(100%)</td></tr><tr><td rowspan=2 colspan=1>Parainfluenzavirus 3</td><td rowspan=1 colspan=1>Low Positive</td><td rowspan=1 colspan=1>39/40(97.50%)</td><td rowspan=1 colspan=1>40/40(100%)</td><td rowspan=1 colspan=1>40/40(100%)</td><td rowspan=1 colspan=1>119/120(99.17%)</td><td rowspan=2 colspan=1>239/240(99.58%)</td></tr><tr><td rowspan=1 colspan=1>Moderate Positive</td><td rowspan=1 colspan=1>40/40(100%)</td><td rowspan=1 colspan=1>40/40(100%)</td><td rowspan=1 colspan=1>40/40(100%)</td><td rowspan=1 colspan=1>120/120(100%)</td></tr><tr><td rowspan=2 colspan=1>Mycoplasmapneumoniae</td><td rowspan=1 colspan=1>Low Positive</td><td rowspan=1 colspan=1>39/40(97.50%)</td><td rowspan=1 colspan=1>40/40(100%)</td><td rowspan=1 colspan=1>40/40(100%)</td><td rowspan=1 colspan=1>119/120(99.17%)</td><td rowspan=2 colspan=1>239/240(99.58%)</td></tr><tr><td rowspan=1 colspan=1>Moderate Positive</td><td rowspan=1 colspan=1>40/40(100%)</td><td rowspan=1 colspan=1>40/40(100%)</td><td rowspan=1 colspan=1>40/40(100%)</td><td rowspan=1 colspan=1>120/120(100%)</td></tr></table>

Confidential & Restricted 510(k) Summary

NxTAG® Respiratory Pathogen Panel v2 Traditional 510(k) Submission   

<table><tr><td rowspan=2 colspan=1>Sample</td><td rowspan=2 colspan=1>Target</td><td rowspan=2 colspan=1>Sample Type</td><td rowspan=1 colspan=5>Agreement with Expected Results</td></tr><tr><td rowspan=1 colspan=1>Site 1</td><td rowspan=1 colspan=1>Site 2</td><td rowspan=1 colspan=1>Site 3</td><td rowspan=1 colspan=2>Overall(All Sites)</td></tr><tr><td rowspan=2 colspan=1></td><td rowspan=2 colspan=1>All othertargets</td><td rowspan=1 colspan=1>Low Positive</td><td rowspan=1 colspan=1>718/720(99.72%)</td><td rowspan=1 colspan=1>720/720(100%)</td><td rowspan=1 colspan=1>720/720(100%)</td><td rowspan=1 colspan=1>2158/2160(99.91%)</td><td rowspan=2 colspan=1>4317/4320(99.93%)</td></tr><tr><td rowspan=1 colspan=1>Moderate Positive</td><td rowspan=1 colspan=1>719/720(99.86%)</td><td rowspan=1 colspan=1>720/720(100%)</td><td rowspan=1 colspan=1>720/720(100%)</td><td rowspan=1 colspan=1>2159/2160(99.95%)</td></tr><tr><td rowspan=8 colspan=1>MA4</td><td rowspan=2 colspan=1>SARS-CoV-2</td><td rowspan=1 colspan=1>Low Positive</td><td rowspan=1 colspan=1>40/40(100%)</td><td rowspan=1 colspan=1>40/40(100%)</td><td rowspan=1 colspan=1>40/40(100%)</td><td rowspan=1 colspan=1>120/120(100%)</td><td rowspan=2 colspan=1>240/240(100%)</td></tr><tr><td rowspan=1 colspan=1>Moderate Positive</td><td rowspan=1 colspan=1>40/40(100%)</td><td rowspan=1 colspan=1>40/40(100%)</td><td rowspan=1 colspan=1>40/40(100%)</td><td rowspan=1 colspan=1>120/120(100%)</td></tr><tr><td rowspan=2 colspan=1>HumanMetapneu-movirus</td><td rowspan=1 colspan=1>Low Positive</td><td rowspan=1 colspan=1>40/40(100%)</td><td rowspan=1 colspan=1>40/40(100%)</td><td rowspan=1 colspan=1>40/40(100%)</td><td rowspan=1 colspan=1>120/120(100%)</td><td rowspan=2 colspan=1>240/240(100%)</td></tr><tr><td rowspan=1 colspan=1>Moderate Positive</td><td rowspan=1 colspan=1>40/40(100%)</td><td rowspan=1 colspan=1>40/40(100%)</td><td rowspan=1 colspan=1>40/40(100%)</td><td rowspan=1 colspan=1>120/120(100%)</td></tr><tr><td rowspan=2 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>Low Positive</td><td rowspan=1 colspan=1>40/40(100%)</td><td rowspan=1 colspan=1>40/40(100%)</td><td rowspan=1 colspan=1>40/40(100%)</td><td rowspan=1 colspan=1>120/120(100%)</td><td rowspan=2 colspan=1>240/240(100%)</td></tr><tr><td rowspan=1 colspan=1>Moderate Positive</td><td rowspan=1 colspan=1>40/40(100%)</td><td rowspan=1 colspan=1>40/40(100%)</td><td rowspan=1 colspan=1>40/40(100%)</td><td rowspan=1 colspan=1>120/120(100%)</td></tr><tr><td rowspan=2 colspan=1>All othertargets</td><td rowspan=1 colspan=1>Low Positive</td><td rowspan=1 colspan=1>716/720(99.44%)</td><td rowspan=1 colspan=1>720/720(100%)</td><td rowspan=1 colspan=1>720/720(100%)</td><td rowspan=1 colspan=1>2156/2160(99.81%)</td><td rowspan=2 colspan=1>4315/4320(99.88%)</td></tr><tr><td rowspan=1 colspan=1>Moderate Positive</td><td rowspan=1 colspan=1>719/720(99.86%)</td><td rowspan=1 colspan=1>720/720(100%)</td><td rowspan=1 colspan=1>720/720(100%)</td><td rowspan=1 colspan=1>2159/2160(99.96%)</td></tr><tr><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>840/840(100%)</td><td rowspan=1 colspan=1>840/840(100%)</td><td rowspan=1 colspan=1>840/840(100%)</td><td rowspan=1 colspan=2>2520/2520(100%)</td></tr><tr><td rowspan=1 colspan=3>Overall Agreement with Expected Results(all targets and all test levels)</td><td rowspan=1 colspan=1>7499/7520(99.72%)</td><td rowspan=1 colspan=1>7520/7520 (100%)</td><td rowspan=1 colspan=1>7519/7520(99.99%)</td><td rowspan=1 colspan=2>22538/22560(99.90%)</td></tr></table>

a Reported by NxTAG RPP v2 as Rhinovirus/Enterovirus b Excludes results of Influenza A Matrix target at Low Positive concentration.

# Lot-to-Lot Reproducibility

A lot-to-lot reproducibility study was performed to assess the total variability of the $\mathsf { N x T A G } ^ { \otimes }$ Respiratory Pathogen Panel (NxTAG RPP v2) assay across 3 assay kit lots with unique lots of critical reagents, including enzymes, buffers, primers, MagPlex microspheres, and dNTPs. One operator at 1 site tested 17-member reproducibility panel in 10 replicates on each of the 3 assay kit lots, for a total of 30 data points (10 replicates x 3 assay lots) for each member. The reproducibility panel comprised of a negative sample and 8 multi-analyte samples prepared in negative simulated matrix (NSM) at two concentrations, Low Positive (1.5x - 3x LoD) and Moderate Positive (5x - 9x LoD). The test concentration for Influenza A H3 (in sample MA2) is based on the LoD for the subtype. Since the Influenza A H3 matrix LoD is 3-fold less sensitive than the subtype, the expected results for the matrix at $\mathsf { 1 . 5 x }$ of the subtype LoD was either “positive” or “negative” (i.e. both results were acceptable based on the respective LoDs of the Influenza A and H3 targets). Therefore, the Influenza A matrix result in sample MA2 could not be assessed and was excluded from data analysis. The results demonstrated reproducibility of the NxTAG RPP v2 assay across 3 assay kit lots with an overall percent agreement of $9 9 . 9 5 \%$ . The summary of results is shown in Table 2.

Table 2: NxTAG® RPP $\pmb { v } \pmb { 2 }$ Lot-to-lot Reproducibility   

<table><tr><td colspan="1" rowspan="2">Sample</td><td colspan="1" rowspan="2">Target</td><td colspan="1" rowspan="2">Sample Type</td><td colspan="5" rowspan="1">Agreement with Expected Results</td></tr><tr><td colspan="1" rowspan="1">Lot 1</td><td colspan="1" rowspan="1">Lot 2</td><td colspan="1" rowspan="1">Lot 3</td><td colspan="2" rowspan="1">Overall(All Lots)</td></tr><tr><td colspan="1" rowspan="8">MA1</td><td colspan="1" rowspan="2">Influenza A 2009H1N1</td><td colspan="1" rowspan="1">Low Positive</td><td colspan="1" rowspan="1">10/10(100%)</td><td colspan="1" rowspan="1">10/10(100%)</td><td colspan="1" rowspan="1">10/10(100%)</td><td colspan="1" rowspan="1">30/30(100%)</td><td colspan="1" rowspan="2">60/60(100%)</td></tr><tr><td colspan="1" rowspan="1">ModeratePositive</td><td colspan="1" rowspan="1">10/10(100%)</td><td colspan="1" rowspan="1">10/10(100%)</td><td colspan="1" rowspan="1">10/10(100%)</td><td colspan="1" rowspan="1">30/30(100%)</td></tr><tr><td colspan="1" rowspan="2">Respiratory SyncytialVirus A</td><td colspan="1" rowspan="1">Low Positive</td><td colspan="1" rowspan="1">10/10(100%)</td><td colspan="1" rowspan="1">10/10(100%)</td><td colspan="1" rowspan="1">10/10(100%)</td><td colspan="1" rowspan="1">30/30(100%)</td><td colspan="1" rowspan="2">60/60(100%)</td></tr><tr><td colspan="1" rowspan="1">ModeratePositive</td><td colspan="1" rowspan="1">10/10(100%)</td><td colspan="1" rowspan="1">10/10(100%)</td><td colspan="1" rowspan="1">10/10(100%)</td><td colspan="1" rowspan="1">30/30(100%)</td></tr><tr><td colspan="1" rowspan="2">Rhinovirusa</td><td colspan="1" rowspan="1">Low Positive</td><td colspan="1" rowspan="1">10/10(100%)</td><td colspan="1" rowspan="1">10/10(100%)</td><td colspan="1" rowspan="1">10/10(100%)</td><td colspan="1" rowspan="1">30/30(100%)</td><td colspan="1" rowspan="2">60/60(100%)</td></tr><tr><td colspan="1" rowspan="1">ModeratePositive</td><td colspan="1" rowspan="1">10/10(100%)</td><td colspan="1" rowspan="1">10/10(100%)</td><td colspan="1" rowspan="1">10/10(100%)</td><td colspan="1" rowspan="1">30/30(100%)</td></tr><tr><td colspan="1" rowspan="2">All other targets</td><td colspan="1" rowspan="1">Low Positive</td><td colspan="1" rowspan="1">180/180(100%)</td><td colspan="1" rowspan="1">180/180(100%)</td><td colspan="1" rowspan="1">180/180(100%)</td><td colspan="1" rowspan="1">540/540(100%)</td><td colspan="1" rowspan="2">1080/1080(100%)</td></tr><tr><td colspan="1" rowspan="1">ModeratePositive</td><td colspan="1" rowspan="1">180/180(100%)</td><td colspan="1" rowspan="1">180/180(100%)</td><td colspan="1" rowspan="1">180/180(100%)</td><td colspan="1" rowspan="1">540/540(100%)</td></tr><tr><td colspan="1" rowspan="3">MA2</td><td colspan="1" rowspan="2">Influenza A H3</td><td colspan="1" rowspan="1">Low Positive</td><td colspan="1" rowspan="1">10/10(100%)</td><td colspan="1" rowspan="1">10/10(100%)</td><td colspan="1" rowspan="1">10/10(100%)</td><td colspan="1" rowspan="1">30/30(100%)</td><td colspan="1" rowspan="2">60/60(100%)</td></tr><tr><td colspan="1" rowspan="1">ModeratePositive</td><td colspan="1" rowspan="1">10/10(100%)</td><td colspan="1" rowspan="1">10/10(100%)</td><td colspan="1" rowspan="1">10/10(100%)</td><td colspan="1" rowspan="1">30/30(100%)</td></tr><tr><td colspan="1" rowspan="1">Respiratory SyncytialVirus B</td><td colspan="1" rowspan="1">Low Positive</td><td colspan="1" rowspan="1">10/10(100%)</td><td colspan="1" rowspan="1">10/10(100%)</td><td colspan="1" rowspan="1">10/10(100%)</td><td colspan="1" rowspan="1">30/30(100%)</td><td colspan="1" rowspan="1">60/60(100%)</td></tr><tr><td colspan="1" rowspan="3"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">ModeratePositive</td><td colspan="1" rowspan="1">10/10(100%)</td><td colspan="1" rowspan="1">10/10(100%)</td><td colspan="1" rowspan="1">10/10(100%)</td><td colspan="1" rowspan="1">30/30(100%)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="2">All other targets</td><td colspan="1" rowspan="1">Low Positive</td><td colspan="1" rowspan="1">180/180(100%)</td><td colspan="1" rowspan="1">180/180(100%)</td><td colspan="1" rowspan="1">180/180(100%)</td><td colspan="1" rowspan="1">540/540(100%)</td><td colspan="1" rowspan="2">1110/1110(100%))</td></tr><tr><td colspan="1" rowspan="1">ModeratePositive</td><td colspan="1" rowspan="1">190/190(100%)</td><td colspan="1" rowspan="1">190/190(100%)</td><td colspan="1" rowspan="1">190/190(100%)</td><td colspan="1" rowspan="1">570/570(100%)</td></tr><tr><td colspan="1" rowspan="8">MA3</td><td colspan="1" rowspan="2">Influenza B</td><td colspan="1" rowspan="1">Low Positive</td><td colspan="1" rowspan="1">10/10(100%)</td><td colspan="1" rowspan="1">10/10(100%)</td><td colspan="1" rowspan="1">10/10(100%)</td><td colspan="1" rowspan="1">30/30(100%)</td><td colspan="1" rowspan="2">60/60(100%)</td></tr><tr><td colspan="1" rowspan="1">ModeratePositive</td><td colspan="1" rowspan="1">10/10(100%)</td><td colspan="1" rowspan="1">10/10(100%)</td><td colspan="1" rowspan="1">10/10(100%)</td><td colspan="1" rowspan="1">30/30(100%)</td></tr><tr><td colspan="1" rowspan="2">Parainfluenza virus 3</td><td colspan="1" rowspan="1">Low Positive</td><td colspan="1" rowspan="1">10/10(100%)</td><td colspan="1" rowspan="1">10/10(100%)</td><td colspan="1" rowspan="1">10/10(100%)</td><td colspan="1" rowspan="1">30/30(100%)</td><td colspan="1" rowspan="2">60/60(100%)</td></tr><tr><td colspan="1" rowspan="1">ModeratePositive</td><td colspan="1" rowspan="1">10/10(100%)</td><td colspan="1" rowspan="1">10/10(100%)</td><td colspan="1" rowspan="1">10/10(100%)</td><td colspan="1" rowspan="1">30/30(100%)</td></tr><tr><td colspan="1" rowspan="2">Mycoplasmapneumoniae</td><td colspan="1" rowspan="1">Low Positive</td><td colspan="1" rowspan="1">10/10(100%)</td><td colspan="1" rowspan="1">10/10(100%)</td><td colspan="1" rowspan="1">10/10(100%)</td><td colspan="1" rowspan="1">30/30(100%)</td><td colspan="1" rowspan="1">60/60</td></tr><tr><td colspan="1" rowspan="1">ModeratePositive</td><td colspan="1" rowspan="1">10/10(100%)</td><td colspan="1" rowspan="1">10/10(100%)</td><td colspan="1" rowspan="1">10/10(100%)</td><td colspan="1" rowspan="1">30/30(100%)</td><td colspan="1" rowspan="1">(100%)</td></tr><tr><td colspan="1" rowspan="2">All other targets</td><td colspan="1" rowspan="1">Low Positive</td><td colspan="1" rowspan="1">180/180(100%)</td><td colspan="1" rowspan="1">180/180(100%)</td><td colspan="1" rowspan="1">180/180(100%)</td><td colspan="1" rowspan="1">540/540(100%)</td><td colspan="1" rowspan="2">1080/1080(100%)</td></tr><tr><td colspan="1" rowspan="1">ModeratePositive</td><td colspan="1" rowspan="1">180/180(100%)</td><td colspan="1" rowspan="1">180/180(100%)</td><td colspan="1" rowspan="1">180/180(100%)</td><td colspan="1" rowspan="1">540/540(100%)</td></tr><tr><td colspan="1" rowspan="8">MA4</td><td colspan="1" rowspan="2">SARS-CoV-2</td><td colspan="1" rowspan="1">Low Positive</td><td colspan="1" rowspan="1">10/10(100%)</td><td colspan="1" rowspan="1">10/10(100%)</td><td colspan="1" rowspan="1">10/10(100%)</td><td colspan="1" rowspan="1">30/30(100%)</td><td colspan="1" rowspan="1">60/60</td></tr><tr><td colspan="1" rowspan="1">ModeratePositive</td><td colspan="1" rowspan="1">10/10(100%)</td><td colspan="1" rowspan="1">10/10(100%)</td><td colspan="1" rowspan="1">10/10(100%)</td><td colspan="1" rowspan="1">30/30(100%)</td><td colspan="1" rowspan="1">(100%)</td></tr><tr><td colspan="1" rowspan="2">HumanMetapneumovirus</td><td colspan="1" rowspan="1">Low Positive</td><td colspan="1" rowspan="1">10/10(100%)</td><td colspan="1" rowspan="1">10/10(100%)</td><td colspan="1" rowspan="1">10/10(100%)</td><td colspan="1" rowspan="1">30/30(100%)</td><td colspan="1" rowspan="2">60/60(100%)</td></tr><tr><td colspan="1" rowspan="1">ModeratePositive</td><td colspan="1" rowspan="1">10/10(100%)</td><td colspan="1" rowspan="1">10/10(100%)</td><td colspan="1" rowspan="1">10/10(100%)</td><td colspan="1" rowspan="1">30/30(100%)</td></tr><tr><td colspan="1" rowspan="2">Adenovirus</td><td colspan="1" rowspan="1">Low Positive</td><td colspan="1" rowspan="1">10/10(100%)</td><td colspan="1" rowspan="1">10/10(100%)</td><td colspan="1" rowspan="1">10/10(100%)</td><td colspan="1" rowspan="1">30/30(100%)</td><td colspan="1" rowspan="1">60/60</td></tr><tr><td colspan="1" rowspan="1">ModeratePositive</td><td colspan="1" rowspan="1">10/10(100%)</td><td colspan="1" rowspan="1">10/10(100%)</td><td colspan="1" rowspan="1">10/10(100%)</td><td colspan="1" rowspan="1">30/30(100%)</td><td colspan="1" rowspan="1">(100%)</td></tr><tr><td colspan="1" rowspan="2">All other targets</td><td colspan="1" rowspan="1">Low Positive</td><td colspan="1" rowspan="1">180/180(100%)</td><td colspan="1" rowspan="1">180/180(100%)</td><td colspan="1" rowspan="1">180/180(100%)</td><td colspan="1" rowspan="1">540/540(100%)</td><td colspan="1" rowspan="1">1080/1080</td></tr><tr><td colspan="1" rowspan="1">ModeratePositive</td><td colspan="1" rowspan="1">180/180(100%)</td><td colspan="1" rowspan="1">180/180(100%)</td><td colspan="1" rowspan="1">180/180(100%)</td><td colspan="1" rowspan="1">540/540(100%)</td><td colspan="1" rowspan="1">(100%)</td></tr><tr><td colspan="1" rowspan="4">MA5</td><td colspan="1" rowspan="2">Influenza A Matrix</td><td colspan="1" rowspan="1">Low Positive</td><td colspan="1" rowspan="1">10/10(100%)</td><td colspan="1" rowspan="1">10/10(100%)</td><td colspan="1" rowspan="1">10/10(100%)</td><td colspan="1" rowspan="1">30/30(100%)</td><td colspan="1" rowspan="1">60/60</td></tr><tr><td colspan="1" rowspan="1">ModeratePositive</td><td colspan="1" rowspan="1">10/10(100%)</td><td colspan="1" rowspan="1">10/10(100%)</td><td colspan="1" rowspan="1">10/10(100%)</td><td colspan="1" rowspan="1">30/30(100%)</td><td colspan="1" rowspan="1">(100%)</td></tr><tr><td colspan="1" rowspan="2">Coronavirus NL63</td><td colspan="1" rowspan="1">Low Positive</td><td colspan="1" rowspan="1">10/10(100%)</td><td colspan="1" rowspan="1">10/10(100%)</td><td colspan="1" rowspan="1">10/10(100%)</td><td colspan="1" rowspan="1">30/30(100%)</td><td colspan="1" rowspan="2">60/60(100%)</td></tr><tr><td colspan="1" rowspan="1">Moderate</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">30/30</td></tr></table>

Confidential & Restricted

<table><tr><td rowspan=2 colspan=1>Sample</td><td rowspan=2 colspan=1>Target</td><td rowspan=2 colspan=1>Sample Type</td><td rowspan=1 colspan=5>Agreement with Expected Results</td></tr><tr><td rowspan=1 colspan=1>Lot 1</td><td rowspan=1 colspan=1>Lot 2</td><td rowspan=1 colspan=1>Lot 3</td><td rowspan=1 colspan=2>Overall(All Lots)</td></tr><tr><td rowspan=5 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>(100%)</td><td rowspan=1 colspan=1>(100%)</td><td rowspan=1 colspan=1>(100%)</td><td rowspan=1 colspan=1>(100%)</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=2 colspan=1>Coronavirus HKU1</td><td rowspan=1 colspan=1>Low Positive</td><td rowspan=1 colspan=1>10/10(100%)</td><td rowspan=1 colspan=1>10/10(100%)</td><td rowspan=1 colspan=1>10/10(100%)</td><td rowspan=1 colspan=1>30/30(100%)</td><td rowspan=2 colspan=1>60/60(100%)</td></tr><tr><td rowspan=1 colspan=1>ModeratePositive</td><td rowspan=1 colspan=1>10/10(100%)</td><td rowspan=1 colspan=1>10/10(100%)</td><td rowspan=1 colspan=1>10/10(100%)</td><td rowspan=1 colspan=1>30/30(100%)</td></tr><tr><td rowspan=2 colspan=1>All other targets</td><td rowspan=1 colspan=1>Low Positive</td><td rowspan=1 colspan=1>180/180(100%)</td><td rowspan=1 colspan=1>179/180(99.44%)</td><td rowspan=1 colspan=1>180/180(100%)</td><td rowspan=1 colspan=1>539/540(99.81%)</td><td rowspan=2 colspan=1>1079/1080(99.90%)</td></tr><tr><td rowspan=1 colspan=1>ModeratePositive</td><td rowspan=1 colspan=1>180/180(100%)</td><td rowspan=1 colspan=1>180/180(100%)</td><td rowspan=1 colspan=1>180/180(100%)</td><td rowspan=1 colspan=1>540/540(100%)</td></tr><tr><td rowspan=8 colspan=1>MA6</td><td rowspan=2 colspan=1>Influenza A H1</td><td rowspan=1 colspan=1>Low Positive</td><td rowspan=1 colspan=1>10/10(100%)</td><td rowspan=1 colspan=1>10/10(100%)</td><td rowspan=1 colspan=1>10/10(100%)</td><td rowspan=1 colspan=1>30/30(100%)</td><td rowspan=2 colspan=1>60/60(100%)</td></tr><tr><td rowspan=1 colspan=1>ModeratePositive</td><td rowspan=1 colspan=1>10/10(100%)</td><td rowspan=1 colspan=1>10/10(100%)</td><td rowspan=1 colspan=1>10/10(100%)</td><td rowspan=1 colspan=1>30/30(100%)</td></tr><tr><td rowspan=2 colspan=1>Parainfluenza virus 1</td><td rowspan=1 colspan=1>Low Positive</td><td rowspan=1 colspan=1>10/10(100%)</td><td rowspan=1 colspan=1>10/10(100%)</td><td rowspan=1 colspan=1>10/10(100%)</td><td rowspan=1 colspan=1>30/30(100%)</td><td rowspan=2 colspan=1>60/60(100%)</td></tr><tr><td rowspan=1 colspan=1>ModeratePositive</td><td rowspan=1 colspan=1>10/10(100%)</td><td rowspan=1 colspan=1>10/10(100%)</td><td rowspan=1 colspan=1>10/10(100%)</td><td rowspan=1 colspan=1>30/30(100%)</td></tr><tr><td rowspan=2 colspan=1>Chlamydiapneumoniae</td><td rowspan=1 colspan=1>Low Positive</td><td rowspan=1 colspan=1>10/10(100%)</td><td rowspan=1 colspan=1>10/10(100%)</td><td rowspan=1 colspan=1>10/10(100%)</td><td rowspan=1 colspan=1>30/30(100%)</td><td rowspan=2 colspan=1>60/60(100%)</td></tr><tr><td rowspan=1 colspan=1>ModeratePositive</td><td rowspan=1 colspan=1>10/10(100%)</td><td rowspan=1 colspan=1>10/10(100%)</td><td rowspan=1 colspan=1>10/10(100%)</td><td rowspan=1 colspan=1>30/30(100%)</td></tr><tr><td rowspan=2 colspan=1>All other targets</td><td rowspan=1 colspan=1>Low Positive</td><td rowspan=1 colspan=1>180/180(100%)</td><td rowspan=1 colspan=1>180/180(100%)</td><td rowspan=1 colspan=1>180/180(100%)</td><td rowspan=1 colspan=1>540/540(100%)</td><td rowspan=2 colspan=1>1079/1080(99.91%)</td></tr><tr><td rowspan=1 colspan=1>ModeratePositive</td><td rowspan=1 colspan=1>180/180(100%)</td><td rowspan=1 colspan=1>179/180(99.44%)</td><td rowspan=1 colspan=1>180/180(100%)</td><td rowspan=1 colspan=1>539/540(99.81%)</td></tr><tr><td rowspan=8 colspan=1>MA7</td><td rowspan=2 colspan=1>Parainfluenza virus 2</td><td rowspan=1 colspan=1>Low Positive</td><td rowspan=1 colspan=1>10/10(100%)</td><td rowspan=1 colspan=1>10/10(100%)</td><td rowspan=1 colspan=1>10/10(100%)</td><td rowspan=1 colspan=1>30/30(100%)</td><td rowspan=1 colspan=1>60/60</td></tr><tr><td rowspan=1 colspan=1>ModeratePositive</td><td rowspan=1 colspan=1>10/10(100%)</td><td rowspan=1 colspan=1>10/10(100%)</td><td rowspan=1 colspan=1>10/10(100%)</td><td rowspan=1 colspan=1>30/30(100%)</td><td rowspan=1 colspan=1>(100%)</td></tr><tr><td rowspan=2 colspan=1>Parainfluenza virus 4(subtype 4B)c</td><td rowspan=1 colspan=1>Low Positive</td><td rowspan=1 colspan=1>10/10(100%)</td><td rowspan=1 colspan=1>10/10(100%)</td><td rowspan=1 colspan=1>10/10(100%)</td><td rowspan=1 colspan=1>30/30(100%)</td><td rowspan=2 colspan=1>60/60(100%)</td></tr><tr><td rowspan=1 colspan=1>ModeratePositive</td><td rowspan=1 colspan=1>10/10(100%)</td><td rowspan=1 colspan=1>10/10(100%)</td><td rowspan=1 colspan=1>10/10(100%)</td><td rowspan=1 colspan=1>30/30(100%)</td></tr><tr><td rowspan=2 colspan=1>Coronavirus 229E</td><td rowspan=1 colspan=1>Low Positive</td><td rowspan=1 colspan=1>10/10(100%)</td><td rowspan=1 colspan=1>10/10(100%)</td><td rowspan=1 colspan=1>10/10(100%)</td><td rowspan=1 colspan=1>30/30(100%)</td><td rowspan=2 colspan=1>60/60(100%)</td></tr><tr><td rowspan=1 colspan=1>Low Positive</td><td rowspan=1 colspan=1>10/10(100%)</td><td rowspan=1 colspan=1>10/10(100%)</td><td rowspan=1 colspan=1>10/10(100%)</td><td rowspan=1 colspan=1>30/30(100%)</td></tr><tr><td rowspan=2 colspan=1>All other targets</td><td rowspan=1 colspan=1>ModeratePositive</td><td rowspan=1 colspan=1>180/180(100%)</td><td rowspan=1 colspan=1>180/180(100%)</td><td rowspan=1 colspan=1>179/180(99.44%)</td><td rowspan=1 colspan=1>539/540(99.81%)</td><td rowspan=2 colspan=1>1079/1080(99.91%)</td></tr><tr><td rowspan=1 colspan=1>Low Positive</td><td rowspan=1 colspan=1>180/180(100%)</td><td rowspan=1 colspan=1>180/180(100%)</td><td rowspan=1 colspan=1>180/180(100%)</td><td rowspan=1 colspan=1>540/540(100%)</td></tr><tr><td rowspan=2 colspan=1>MA8</td><td rowspan=2 colspan=1>Parainfluenza virus 4(subtype 4A)c</td><td rowspan=1 colspan=1>ModeratePositive</td><td rowspan=1 colspan=1>10/10(100%)</td><td rowspan=1 colspan=1>10/10(100%)</td><td rowspan=1 colspan=1>10/10(100%)</td><td rowspan=1 colspan=1>30/30(100%)</td><td rowspan=1 colspan=1>60/60</td></tr><tr><td rowspan=1 colspan=1>Low Positive</td><td rowspan=1 colspan=1>10/10(100%)</td><td rowspan=1 colspan=1>10/10(100%)</td><td rowspan=1 colspan=1>10/10(100%)</td><td rowspan=1 colspan=1>30/30(100%)</td><td rowspan=1 colspan=1>(100%)</td></tr></table>

Confidential & Restricted 510(k) Summary

<table><tr><td rowspan=2 colspan=1>Sample</td><td rowspan=2 colspan=1>Target</td><td rowspan=2 colspan=1>Sample Type</td><td rowspan=1 colspan=5>Agreement with Expected Results</td></tr><tr><td rowspan=1 colspan=1>Lot 1</td><td rowspan=1 colspan=1>Lot 2</td><td rowspan=1 colspan=1>Lot 3</td><td rowspan=1 colspan=2>Overall(All Lots)</td></tr><tr><td rowspan=4 colspan=1></td><td rowspan=2 colspan=1>Coronavirus OC43</td><td rowspan=1 colspan=1>ModeratePositive</td><td rowspan=1 colspan=1>10/10(100%)</td><td rowspan=1 colspan=1>10/10(100%)</td><td rowspan=1 colspan=1>10/10(100%)</td><td rowspan=1 colspan=1>30/30(100%)</td><td rowspan=2 colspan=1>60/60(100%)</td></tr><tr><td rowspan=1 colspan=1>Low Positive</td><td rowspan=1 colspan=1>10/10(100%)</td><td rowspan=1 colspan=1>10/10(100%)</td><td rowspan=1 colspan=1>10/10(100%)</td><td rowspan=1 colspan=1>30/30(100%)</td></tr><tr><td rowspan=2 colspan=1>All other targets</td><td rowspan=1 colspan=1>ModeratePositive</td><td rowspan=1 colspan=1>190/190(100%)</td><td rowspan=1 colspan=1>190/190(100%)</td><td rowspan=1 colspan=1>190/190(100%)</td><td rowspan=1 colspan=1>570/570(100%)</td><td rowspan=2 colspan=1>1140/1140(100%)</td></tr><tr><td rowspan=1 colspan=1>Low Positive</td><td rowspan=1 colspan=1>190/190(100%)</td><td rowspan=1 colspan=1>190/190(100%)</td><td rowspan=1 colspan=1>190/190(100%)</td><td rowspan=1 colspan=1>570/570(100%)</td></tr><tr><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>208/210(99.04%)</td><td rowspan=1 colspan=1>210/210(100%)</td><td rowspan=1 colspan=1>210/210(100%)</td><td rowspan=1 colspan=2>628/630 (99.68%)</td></tr><tr><td rowspan=1 colspan=3>Overall Agreement with Expected Results(all targets and all test levels)</td><td rowspan=1 colspan=1>3558/3560(99.94%)</td><td rowspan=1 colspan=1>3558/3560(99.94%)</td><td rowspan=1 colspan=1>3559/3560(99.97%)</td><td rowspan=1 colspan=2>10675/10680 (99.95%)</td></tr></table>

a Reported by NxTAG RPP v2 as Rhinovirus/Enterovirus b Excludes results of Influenza A Matrix target at Low Positive concentration. c NxTAG RPP v2 does not distinguish Parainfluenza virus subtypes 4A and 4B which are both reported as Parainfluenza virus 4.

b. Linearity/assay reportable range: Not applicable. The NxTAG® RPP v2 assay is a qualitative assay.

c. Traceability, Stability, Expected values (controls, calibrators, or methods):

i) Controls:

(a) Internal Control

Bacteriophage MS2 is the internal control for the assay. This internal positive control is added to each specimen prior to extraction. This internal control allows the user to ascertain whether the assay is functioning properly. Failure to detect the MS2 control indicates a failure at either the extraction step, the reverse-transcription step, the PCR step, or the detection step, and may be indicative of the presence of amplification inhibitors, thereby preventing false negative results.

# (b) External Controls

External positive and negative controls should be used in accordance with local, state, federal accrediting organizations, as applicable.

• Positive Control – Positive controls are not included in the NxTAG® Respiratory Pathogen Panel v2 assay, but are recommended to be included in every run, as a good laboratory practice.

• Negative Amplification Control (No Template Control (NTC)) – The negative amplification control is RNase- free water.

• Negative Extraction Control (NEC) – The negative extraction control is the sample collection medium that has undergone the entire assay procedure, starting from extraction.

ii) Stability:

(a) Specimen Stability

Stability of raw specimens in Universal Transport Media (UTM), and MicroTest™ M4RT, as well as, stability of extracted specimens were evaluated at temperatures ranging between ${ 2 ^ { \circ } \mathsf C }$ to ${ 8 ^ { \circ } C }$ and $- 7 0 ^ { \circ } \mathsf { C } \pm 5 ^ { \circ } \mathsf { C }$ on the $\mathsf { N x T A G } ^ { \otimes }$

RPP v2 assay. For raw specimens prepared in negative clinical matrix in UTM and extracted specimens, all targets probed by the assay were evaluated with a set of 17 samples: eight multi-analyte samples contrived in negative clinical matrix at two concentrations, Low positive $( 1 . 5 \mathsf { x } - 3 \mathsf { x } \mathsf { \ L o D } )$ and Moderate positive $( 5 \mathsf { x } - 9 \mathsf { x } \mathsf { L o D } )$ , and a negative sample (negative clinical matrix only). For raw specimens prepared in negative clinical matrix in M4RT, a subset of targets in four multi-analyte samples consisting of representative organisms and genome types, at two concentrations Low positive $( 1 . 5 { \mathsf { x } } - 3 { \mathsf { x } } \lfloor { \mathsf { o } } { \mathsf { D } } )$ and Moderate positive $( 5 \mathsf { x } - 9 \mathsf { x } \mathsf { L o D } )$ , as well as a negative sample were evaluated. Ten replicates were tested for each condition.

All samples met the acceptance criteria, and the data support a raw specimen stability claim of 7 days at $2 ^ { \circ } \mathsf { C }$ to ${ 8 ^ { \circ } C }$ and 12 months at $- 7 0 ^ { \circ } \mathsf { C } \pm 5 ^ { \circ } \mathsf { C }$ for both transport media tested, and an extracted specimen stability claim of 4 hours at $2 ^ { \circ } \mathsf { C }$ to ${ 8 ^ { \circ } C }$ and 12 months at $- 7 0 ^ { \circ } \mathsf { C } \pm 5 ^ { \circ } \mathsf { C }$ .

(b) Device Stability

A shelf-life study was conducted to evaluate the real-time stability of $\mathsf { N x T A G } ^ { \otimes }$ RPP v2 at the recommended storage conditions of $2 - 8 ^ { \circ } \mathsf C$ . Real-time stability was assessed using Positive Controls that cover all analytes probed by the assay, extracted MS2 (internal control), and no template control. Results of this study demonstrated that NxTAG® RPP v2 is stable for at least 12 months when stored at $2 - 8 ^ { \circ } C .$ .

In-use, open-pouch/vial stability of $\mathsf { N } \times \mathsf { T } \mathsf { A G } ^ { \otimes }$ RPP v2 was also evaluated by mimicking in-use conditions of the kit, which was stored at the recommended storage conditions of $2 - 8 ^ { \circ } \mathsf C$ . Stability was assessed using Positive Controls that cover all or a subset of analytes probed by the assay, MS2 (Internal Control) and a No Template Control. The study demonstrated $100 \%$ agreement between calls made with the $\mathsf { N x T A G } ^ { \otimes }$ RPP v2 assay on a cold block for an hour vs. the baseline and calls made with NxTAG® RPP v2 assay on a cold block that was replaced every 15 to 20 minutes vs. the baseline, and confirmed the stability of the $\mathsf { N x T A G } ^ { \otimes }$ RPP v2 assay after it is opened and resealed for 6 cycles over 5 weeks, at the recommended storage conditions.

# d. Detection Limit:

The Limit of Detection (LoD) of the $\mathsf { N } \times \mathsf { T } \mathsf { A G } ^ { \otimes }$ RPP v2 assay for each target was assessed by testing simulated samples prepared from high-titer cultured material obtained from commercial suppliers or characterized clinical specimens. For each target, a 3-fold dilution series was prepared in negative clinical matrix (NCM,

# Luminex

pooled negative nasopharyngeal swabs in Universal Transport Medium), extracted using bioMérieux® EMAG® extraction system, and tested with $\mathsf { N x T A G } ^ { \otimes }$ RPP v2 assay. The preliminary LoD for each target was confirmed by preparing and testing 20 replicates. The LoD concentration for each target was defined as the lowest concentration at which $\geq$ $9 5 \%$ (≥ 19/20) of the replicates can reproducibly be detected. A droplet digital PCR (ddPCR) assay was performed to quantitate clinical specimens and those culture stocks where titer information in copy number was not available. A summary of the confirmed LoD for each target is listed in Table 3. In addition, confirmation of LoD was performed for targets in Multi-Analyte (MA) samples prepared in NCM. Each MA sample consisted of 2 to 4 target analytes and 8 MA samples cover all targets probed by the NxTAG® RPP v2 assay. Confirmation of the single-analyte LoD in MA samples supported the use of MA samples in $\mathsf { N x T A G } ^ { \otimes }$ RPP v2 analytical studies.

Table 3: NxTAG® RPP v2 Limit of Detection   

<table><tr><td rowspan=2 colspan=1>Target</td><td rowspan=2 colspan=1>Strain/Isolate</td><td rowspan=2 colspan=1>Supplier/Part Number</td><td rowspan=1 colspan=2>LoD Concentration</td><td rowspan=2 colspan=1># Detected/# Tested</td></tr><tr><td rowspan=1 colspan=1>Copies/mL</td><td rowspan=1 colspan=1>In SupplierUnit</td></tr><tr><td rowspan=3 colspan=1>Influenza A(Matrix)</td><td rowspan=1 colspan=1>A/Brisbane/59/07</td><td rowspan=1 colspan=1>ZeptoMetrix0810244CF</td><td rowspan=1 colspan=1>1.19E+02</td><td rowspan=1 colspan=1>2.83E-02TCID50/mL</td><td rowspan=1 colspan=1>20/20</td></tr><tr><td rowspan=1 colspan=1>A/NY/02/09</td><td rowspan=1 colspan=1>ZeptoMetrix0810109CFN</td><td rowspan=1 colspan=1>3.28E+02</td><td rowspan=1 colspan=1>3.74E-02TCID50/mL</td><td rowspan=1 colspan=1>20/20</td></tr><tr><td rowspan=1 colspan=1>A/Wisconsin/67/05</td><td rowspan=1 colspan=1>ZeptoMetrix0810252CF</td><td rowspan=1 colspan=1>1.68E+02</td><td rowspan=1 colspan=1>6.45E-01TCID5o/mL</td><td rowspan=1 colspan=1>20/20</td></tr><tr><td rowspan=1 colspan=1>Influenza A H1(Subtype)</td><td rowspan=1 colspan=1>A/Brisbane/59/07</td><td rowspan=1 colspan=1>ZeptoMetrix0810244CF</td><td rowspan=1 colspan=1>1.60E+03</td><td rowspan=1 colspan=1>3.82E-01TCID5o/mL</td><td rowspan=1 colspan=1>20/20</td></tr><tr><td rowspan=1 colspan=1>Influenza A H1pdm09(Subtype)</td><td rowspan=1 colspan=1>A/NY/02/09</td><td rowspan=1 colspan=1>ZeptoMetrix0810109CFN</td><td rowspan=1 colspan=1>9.84E+02</td><td rowspan=1 colspan=1>1.12E-01TCID50/mL</td><td rowspan=1 colspan=1>19/20</td></tr><tr><td rowspan=1 colspan=1>Influenza A H3(Subtype)</td><td rowspan=1 colspan=1>A/Wisconsin/67/05</td><td rowspan=1 colspan=1>ZeptoMetrix0810252CF</td><td rowspan=1 colspan=1>5.60E+01</td><td rowspan=1 colspan=1>2.15E-01TCID5o/mL</td><td rowspan=1 colspan=1>20/20</td></tr><tr><td rowspan=1 colspan=1>Influenza B</td><td rowspan=1 colspan=1>B/Florida/02/06</td><td rowspan=1 colspan=1>ZeptoMetrix0810037CF</td><td rowspan=1 colspan=1>6.33E+01</td><td rowspan=1 colspan=1>9.67E-01TCID5o/mL</td><td rowspan=1 colspan=1>19/20</td></tr><tr><td rowspan=1 colspan=1>Respiratory SyncytialVirus A</td><td rowspan=1 colspan=1>A2</td><td rowspan=1 colspan=1>ATCC VR-1540</td><td rowspan=1 colspan=1>4.97E+03</td><td rowspan=1 colspan=1>3.77E+01PFU/mL</td><td rowspan=1 colspan=1>19/20</td></tr><tr><td rowspan=1 colspan=1>Respiratory SyncytialVirus B</td><td rowspan=1 colspan=1>18537</td><td rowspan=1 colspan=1>ATCC VR-1580</td><td rowspan=1 colspan=1>7.21E+03</td><td rowspan=1 colspan=1>3.20E-01PFU/mL</td><td rowspan=1 colspan=1>20/20</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza Virus 1</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>ZeptoMetrix0810014CF</td><td rowspan=1 colspan=1>6.92E+02</td><td rowspan=1 colspan=1>7.64E-01TCID5o/mL</td><td rowspan=1 colspan=1>20/20</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza virus 2</td><td rowspan=1 colspan=1>Greer</td><td rowspan=1 colspan=1>ATCC VR-92</td><td rowspan=1 colspan=1>3.45E+02</td><td rowspan=1 colspan=1>7.32E-01TCID50/mL</td><td rowspan=1 colspan=1>20/20</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza virus 3</td><td rowspan=1 colspan=1>C243</td><td rowspan=1 colspan=1>ATCC VR-93</td><td rowspan=1 colspan=1>1.01E+03</td><td rowspan=1 colspan=1>1.10E+02TCID5o/mL</td><td rowspan=1 colspan=1>20/20</td></tr></table>

NxTAG® Respiratory Pathogen Panel v2 Traditional 510(k) Submission   

<table><tr><td rowspan=2 colspan=1>Target</td><td rowspan=2 colspan=1>Strain/Isolate</td><td rowspan=2 colspan=1>Supplier/Part Number</td><td rowspan=1 colspan=2>LoD Concentration</td><td rowspan=2 colspan=1># Detected/# Tested</td></tr><tr><td rowspan=1 colspan=1>Copies/mL</td><td rowspan=1 colspan=1>In SupplierUnit</td></tr><tr><td rowspan=2 colspan=1>Parainfluenza virus 4</td><td rowspan=1 colspan=1>Type 4A</td><td rowspan=1 colspan=1>ZeptoMetrix0810060CF</td><td rowspan=1 colspan=1>1.69E+04</td><td rowspan=1 colspan=1>8.58E-01TCID50/mL</td><td rowspan=1 colspan=1>20/20</td></tr><tr><td rowspan=1 colspan=1>Type 4B, CH 19503</td><td rowspan=1 colspan=1>ATCC VR-1377</td><td rowspan=1 colspan=1>7.15E+03</td><td rowspan=1 colspan=1>5.99E+01TCID5o/mL</td><td rowspan=1 colspan=1>20/20</td></tr><tr><td rowspan=1 colspan=1>SARS-CoV-2</td><td rowspan=1 colspan=1>USA-WA1/2020</td><td rowspan=1 colspan=1>ATCC VR-1986HK</td><td rowspan=1 colspan=1>5.00E+02</td><td rowspan=1 colspan=1>7.68E+00TCID50/mL</td><td rowspan=1 colspan=1>19/20</td></tr><tr><td rowspan=1 colspan=1>Coronavirus 229E</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>ATCC VR-740</td><td rowspan=1 colspan=1>3.81E+02</td><td rowspan=1 colspan=1>1.22E-01TCID50/mL</td><td rowspan=1 colspan=1>19/20</td></tr><tr><td rowspan=1 colspan=1>Coronavirus NL63</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>ZeptoMetrix0810228CF</td><td rowspan=1 colspan=1>1.00E+02</td><td rowspan=1 colspan=1>6.45E-03TCID50/mL</td><td rowspan=1 colspan=1>20/20</td></tr><tr><td rowspan=1 colspan=1>Coronavirus OC43</td><td rowspan=1 colspan=1>Betacoronavirus 1</td><td rowspan=1 colspan=1>ATCC VR-1558</td><td rowspan=1 colspan=1>4.55E+03</td><td rowspan=1 colspan=1>7.32E-02TCID5o/mL</td><td rowspan=1 colspan=1>20/20</td></tr><tr><td rowspan=1 colspan=1>Coronavirus HKU1</td><td rowspan=1 colspan=1>Genotype B</td><td rowspan=1 colspan=1>Clinical Sample</td><td rowspan=1 colspan=1>4.18E+03</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>19/20</td></tr><tr><td rowspan=2 colspan=1>HumanMetapneumovirus</td><td rowspan=1 colspan=1>hMPV-16, Type A1,IA10-2003</td><td rowspan=1 colspan=1>ZeptoMetrix0810161CF</td><td rowspan=1 colspan=1>7.15E+01</td><td rowspan=1 colspan=1>5.76E-02TCID5o/mL</td><td rowspan=1 colspan=1>20/20</td></tr><tr><td rowspan=1 colspan=1>hMPV-3, Type B1,Peru2-2002</td><td rowspan=1 colspan=1>ZeptoMetrix0810156CF</td><td rowspan=1 colspan=1>2.62E+02</td><td rowspan=1 colspan=1>1.78E-02TCID5o/mL</td><td rowspan=1 colspan=1>20/20</td></tr><tr><td rowspan=2 colspan=1>Rhinovirus/Enterovirus</td><td rowspan=1 colspan=1>Rhinovirus50-525-CV54[V-192-001-021]</td><td rowspan=1 colspan=1>ATCC VR-1195</td><td rowspan=1 colspan=1>1.54E+03</td><td rowspan=1 colspan=1>6.87E+01TCID50/mL</td><td rowspan=1 colspan=1>20/20</td></tr><tr><td rowspan=1 colspan=1>EnterovirusSpecies D, Type 682007 Isolate</td><td rowspan=1 colspan=1>ZeptoMetrix0810237CF</td><td rowspan=1 colspan=1>3.53E+03</td><td rowspan=1 colspan=1>2.30E+00TCID50/mL</td><td rowspan=1 colspan=1>20/20</td></tr><tr><td rowspan=3 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>Species B, Type 142006 isolate</td><td rowspan=1 colspan=1>ZeptoMetrix0810108CF</td><td rowspan=1 colspan=1>1.42E+03</td><td rowspan=1 colspan=1>1.44E-01TCID5o/mL</td><td rowspan=1 colspan=1>19/20</td></tr><tr><td rowspan=1 colspan=1>Species C, Type 1</td><td rowspan=1 colspan=1>ZeptoMetrix0810050CF</td><td rowspan=1 colspan=1>2.01E+04</td><td rowspan=1 colspan=1>9.28E+01TCID50/mL</td><td rowspan=1 colspan=1>20/20</td></tr><tr><td rowspan=1 colspan=1>Species E, Type 4</td><td rowspan=1 colspan=1>ZeptoMetrix0810070CF</td><td rowspan=1 colspan=1>7.33E+03</td><td rowspan=1 colspan=1>1.91E-01TCID5o/mL</td><td rowspan=1 colspan=1>19/20</td></tr><tr><td rowspan=1 colspan=1>Chlamydiapneumoniae</td><td rowspan=1 colspan=1>TW-183</td><td rowspan=1 colspan=1>ATCC VR-2282</td><td rowspan=1 colspan=1>2.38E+02</td><td rowspan=1 colspan=1>3.74E+01IFU/mL</td><td rowspan=1 colspan=1>20/20</td></tr><tr><td rowspan=1 colspan=1>Mycoplasmapneumoniae</td><td rowspan=1 colspan=1>M129</td><td rowspan=1 colspan=1>ZeptoMetrix 0801579</td><td rowspan=1 colspan=1>3.23E+03</td><td rowspan=1 colspan=1>5.56E+01CCU/mL</td><td rowspan=1 colspan=1>19/20</td></tr></table>

e. Analytical Sensitivity for the First WHO International Standard for SARS-CoV-2

The Analytical Sensitivity of the $\mathsf { N x T A G } ^ { \otimes }$ RPP v2 assay for the WHO standard for SARS-CoV-2 was evaluated.

# Luminex

The preliminary LoD was determined by preparing a 10-fold dilution series in negative clinical matrix (NCM, pooled negative nasopharyngeal swabs in Universal Transport Medium) and testing each dilution level in triplicate. The LoD was confirmed by testing 20 replicates of sample prepared at preliminary LoD, as well as 20 replicates of samples prepared 3-fold above and below this preliminary LoD.

The Analytical Sensitivity was defined as the lowest concentration at which $\geq 9 5 \%$ of the replicates tested generated a positive call. The summary of the results is shown in Table 4.

Table 4. Analytical Sensitivity of NxTAG® RPP v2 for the First WHO International Standard for SARS-CoV-2.   

<table><tr><td rowspan=1 colspan=1>Target</td><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>Supplier/Part Number</td><td rowspan=1 colspan=1>Concentration(IU/mL)</td><td rowspan=1 colspan=1># Detected / #tested</td></tr><tr><td rowspan=1 colspan=1>SARS-CoV-2</td><td rowspan=1 colspan=1>Heat inactivatedEngland/02/2020 isolate</td><td rowspan=1 colspan=1>NIBSC20/146</td><td rowspan=1 colspan=1>7.70E+05</td><td rowspan=1 colspan=1>20/20 (100%)</td></tr></table>

# f. Analytical Reactivity (Inclusivity)

Analytical Reactivity (Inclusivity) of the NxTAG® RPP v2 assay was assessed by testing a total of 193 pathogen strains/isolates (168 reactivity strains and $2 5 ~ \mathsf { L o D }$ strains). The strains tested represent the diversity of the targets probed by NxTAG® RPP v2. Each strain was prepared in negative simulated matrix (NSM) and tested in triplicate on the NxTAG RPP v2 assay. A droplet digital PCR (ddPCR) assay was performed to quantitate clinical specimens and those culture stocks where titer information in copy number was not available. For influenza A results, the concentration at which both the influenza A matrix and the subtype of that strain were detected by NxTAG® RPP v2 is shown. When the influenza A matrix and the subtype were detected at different concentrations, the concentration for each target is listed separately. The NxTAG® RPP v2 assay is capable of detecting the Influenza A matrix for all strains, including Influenza A H5, H7, and H9. Specimens containing Influenza A H5, H7, and H9 strains are expected to be reported as “Influenza A” only. The applicable subtype of all strains of influenza A H1, A H1pdm09 and A H3 with the exception of Influenza A A/Denver/1/57 H1 were detected successfully. A summary of the results, including the strain identity and the concentration at which they were detected, are shown in Table 5.

# Table 5. Results of NxTAG® RPP v2 Analytical Reactivity

NxTAG® Respiratory Pathogen Panel v2 Traditional 510(k) Submission   

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>Supplier / PartNumber</td><td rowspan=1 colspan=1>ConcentrationTested(Copies/mL)</td><td rowspan=1 colspan=1># Detected/# Tested</td></tr><tr><td rowspan=6 colspan=1>Influenza A H1</td><td rowspan=1 colspan=1>A/Brisbane/59/07*</td><td rowspan=1 colspan=1>ZeptoMetrix0810244CF</td><td rowspan=1 colspan=1>4.81E+03</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>A/New Caledonia/20/99</td><td rowspan=1 colspan=1>ZeptoMetrix0810036CF</td><td rowspan=1 colspan=1>4.81E+03</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>A/Solomon Island/03/06</td><td rowspan=1 colspan=1>ZeptoMetrix0810036CFN</td><td rowspan=1 colspan=1>4.81E+03</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>A/Taiwan/42/06</td><td rowspan=1 colspan=1>ZeptoMetrix0810036CF(New PN:0810247CF)</td><td rowspan=1 colspan=1>4.81E+03</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=2 colspan=1>A/Denver/1/57</td><td rowspan=2 colspan=1>ATCC VR-546</td><td rowspan=1 colspan=1>4.81E+03</td><td rowspan=1 colspan=1>3/3 (Matrix)</td></tr><tr><td rowspan=1 colspan=1>5.77E+07a</td><td rowspan=1 colspan=1>0/3 (H1 Subtype)</td></tr><tr><td rowspan=15 colspan=1>Influenza A H1pdm09</td><td rowspan=1 colspan=1>A/NY/02/09*</td><td rowspan=1 colspan=1>ZeptoMetrix0810109CFN</td><td rowspan=1 colspan=1>5.11E+03</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>A/SwineNY/01/2009(New name: A/NY/01/09)</td><td rowspan=1 colspan=1>ZeptoMetrix0810109CFN(New PN:0810248CF)</td><td rowspan=1 colspan=1>5.11E+03</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>A/SwineNY/03/2009(New name: A/NY/03/09)</td><td rowspan=1 colspan=1>ZeptoMetrix0810109CFN(New PN:0810249CF)</td><td rowspan=1 colspan=1>5.11E+03</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>A/Swine/Canada/6294/09</td><td rowspan=1 colspan=1>ZeptoMetrix0810109CFJ</td><td rowspan=1 colspan=1>5.11E+03</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>A/California/07/09</td><td rowspan=1 colspan=1>ZeptoMetrix0810165CF</td><td rowspan=1 colspan=1>5.11E+03</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>A/Mexico/4108/09</td><td rowspan=1 colspan=1>ZeptoMetrix0810166CF</td><td rowspan=1 colspan=1>5.11E+03</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>A/Michigan/45/15</td><td rowspan=1 colspan=1>ZeptoMetrix0810538CF</td><td rowspan=1 colspan=1>5.11E+03</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>A/Brisbane/02/18</td><td rowspan=1 colspan=1>ZeptoMetrix0810585CF</td><td rowspan=1 colspan=1>5.11E+03</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>A/Virginia/ATCC1/2009</td><td rowspan=1 colspan=1>ATCC VR-1736</td><td rowspan=1 colspan=1>5.11E+03</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>A/Netherlands/2629/2009</td><td rowspan=1 colspan=1>BEI NR-19823</td><td rowspan=1 colspan=1>5.11E+03</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>A/Houston/3H/2009</td><td rowspan=1 colspan=1>BEI NR-20340</td><td rowspan=1 colspan=1>5.11E+03</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>A/Brownsville/31H/2009</td><td rowspan=1 colspan=1>BEI NR-20344</td><td rowspan=1 colspan=1>5.11E+03</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>A/Dominican Republic/7293/2013</td><td rowspan=1 colspan=1>IRR FR-1298</td><td rowspan=1 colspan=1>5.11E+03</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>A/Massachusetts/15/2013</td><td rowspan=1 colspan=1>IRR FR-1319</td><td rowspan=1 colspan=1>5.11E+03</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>A/Swine/1976/31</td><td rowspan=1 colspan=1>ATCC VR-99</td><td rowspan=1 colspan=1>4.81E+03</td><td rowspan=1 colspan=1>3/3 (Matrix)</td></tr></table>

NxTAG® Respiratory Pathogen Panel v2 Traditional 510(k) Submission   

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>Supplier / PartNumber</td><td rowspan=1 colspan=1>ConcentrationTested(Copies/mL)</td><td rowspan=1 colspan=1># Detected/# Tested</td></tr><tr><td rowspan=3 colspan=1></td><td rowspan=1 colspan=1>cYanmaGtaYellowb</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1.83E+07</td><td rowspan=1 colspan=1>3/3 (H1pdm09Subtype)</td></tr><tr><td rowspan=2 colspan=1>A/Swine/lowa/15/30</td><td rowspan=2 colspan=1>ATCC VR-333</td><td rowspan=1 colspan=1>4.81E+03</td><td rowspan=1 colspan=1>3/3 (Matrix)</td></tr><tr><td rowspan=1 colspan=1>2.25E+07</td><td rowspan=1 colspan=1>3/3(H1pdm09Subtype)</td></tr><tr><td rowspan=21 colspan=1>Influenza A H3</td><td rowspan=1 colspan=1>A/Wisconsin/67/05*</td><td rowspan=1 colspan=1>ZeptoMetrix0810252CF</td><td rowspan=1 colspan=1>1.68E+02</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=2 colspan=1>A/Brisbane/10/07</td><td rowspan=2 colspan=1>ZeptoMetrix0810138CF</td><td rowspan=1 colspan=1>5.04E+02</td><td rowspan=1 colspan=1>3/3 (Matrix)</td></tr><tr><td rowspan=1 colspan=1>1.68E+02</td><td rowspan=1 colspan=1>3/3 (H3 Subtype)</td></tr><tr><td rowspan=1 colspan=1>A/Texas/50/12</td><td rowspan=1 colspan=1>ZeptoMetrix0810238CF</td><td rowspan=1 colspan=1>5.04E+02</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>A/Perth/16/09</td><td rowspan=1 colspan=1>ZeptoMetrix0810138CF(New PN:0810251CF)</td><td rowspan=1 colspan=1>1.68E+02</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=2 colspan=1>A/Hong Kong/4801/14</td><td rowspan=2 colspan=1>ZeptoMetrix0810526CF</td><td rowspan=1 colspan=1>4.54E+03</td><td rowspan=1 colspan=1>3/3 (Matrix)</td></tr><tr><td rowspan=1 colspan=1>1.51E+03</td><td rowspan=1 colspan=1>3/3 (H3 Subtype)</td></tr><tr><td rowspan=1 colspan=1>A/Singapore/INFIMH-16-0019/16</td><td rowspan=1 colspan=1>ZeptoMetrix0810574CF</td><td rowspan=1 colspan=1>1.51E+03</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=2 colspan=1>A/Kansas/14/17</td><td rowspan=2 colspan=1>ZeptoMetrix0810586CF</td><td rowspan=1 colspan=1>5.04E+02</td><td rowspan=1 colspan=1>3/3 (Matrix)</td></tr><tr><td rowspan=1 colspan=1>1.68E+02</td><td rowspan=1 colspan=1>3/3 (H3 Subtype)</td></tr><tr><td rowspan=2 colspan=1>A/Hong Kong/8/68</td><td rowspan=2 colspan=1>ATCC VR-544</td><td rowspan=1 colspan=1>1.68E+02</td><td rowspan=1 colspan=1>3/3 (Matrix)</td></tr><tr><td rowspan=1 colspan=1>4.08E+04</td><td rowspan=1 colspan=1>3/3 (H3 Subtype)</td></tr><tr><td rowspan=2 colspan=1>A/Alice</td><td rowspan=2 colspan=1>ATCC VR-776</td><td rowspan=1 colspan=1>1.68E+02</td><td rowspan=1 colspan=1>3/3 (Matrix)</td></tr><tr><td rowspan=1 colspan=1>8.04E+03</td><td rowspan=1 colspan=1>3/3 (H3 Subtype)</td></tr><tr><td rowspan=2 colspan=1>A/Port Chalmers/1/73</td><td rowspan=2 colspan=1>ATCC VR-810</td><td rowspan=1 colspan=1>1.51E+03</td><td rowspan=1 colspan=1>3/3 (Matrix)</td></tr><tr><td rowspan=1 colspan=1>5.04E+02</td><td rowspan=1 colspan=1>3/3 (H3 Subtype)</td></tr><tr><td rowspan=2 colspan=1>A/Sydney/5/1997</td><td rowspan=2 colspan=1>BEI NR-12278</td><td rowspan=1 colspan=1>4.54E+03</td><td rowspan=1 colspan=1>3/3 (Matrix)</td></tr><tr><td rowspan=1 colspan=1>1.68E+02</td><td rowspan=1 colspan=1>3/3 (H3 Subtype)</td></tr><tr><td rowspan=1 colspan=1>A/Santiago/7981/2006</td><td rowspan=1 colspan=1>IRR FR-336</td><td rowspan=1 colspan=1>1.68E+02</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>A/Henan/Jinshui/147/2007</td><td rowspan=1 colspan=1>IRR FR-365</td><td rowspan=1 colspan=1>1.68E+02</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>A/Brisbane/9/2006</td><td rowspan=1 colspan=1>IRR FR-366</td><td rowspan=1 colspan=1>1.68E+02</td><td rowspan=1 colspan=1>3/3</td></tr></table>

NxTAG® Respiratory Pathogen Panel v2 Traditional 510(k) Submission   

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>Supplier / PartNumber</td><td rowspan=1 colspan=1>ConcentrationTested(Copies/mL)</td><td rowspan=1 colspan=1># Detected/# Tested</td></tr><tr><td rowspan=12 colspan=1></td><td rowspan=1 colspan=1>A/Nepal/921/2006</td><td rowspan=1 colspan=1>IRR FR-367</td><td rowspan=1 colspan=1>1.68E+02</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=2 colspan=1>A/Florida/2/2006</td><td rowspan=2 colspan=1>IRR FR-368</td><td rowspan=1 colspan=1>5.04E+02</td><td rowspan=1 colspan=1>3/3 (Matrix)</td></tr><tr><td rowspan=1 colspan=1>1.68E+02</td><td rowspan=1 colspan=1>3/3 (H3 Subtype)</td></tr><tr><td rowspan=1 colspan=1>South Australia/55/14</td><td rowspan=1 colspan=1>0810512CF</td><td rowspan=1 colspan=1>1.68E+02</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=2 colspan=1>Stockholm/6/14</td><td rowspan=2 colspan=1>0810513CF</td><td rowspan=1 colspan=1>5.04E+02</td><td rowspan=1 colspan=1>3/3 (Matrix)</td></tr><tr><td rowspan=1 colspan=1>1.68E+02</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Norway/466/14</td><td rowspan=1 colspan=1>0810514CF</td><td rowspan=1 colspan=1>1.68E+02</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Hong Kong/2671/19</td><td rowspan=1 colspan=1>0810609CF</td><td rowspan=1 colspan=1>5.04E+02</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>A/California/2/2014</td><td rowspan=1 colspan=1>VR-1938</td><td rowspan=1 colspan=1>1.68E+02</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>A/Switzerland/9715293/2013</td><td rowspan=1 colspan=1>VR-183</td><td rowspan=1 colspan=1>1.68E+02</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Clinical Sample</td><td rowspan=1 colspan=1>500-NEG-161</td><td rowspan=1 colspan=1>1.68E+02</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Clinical Sample</td><td rowspan=1 colspan=1>500-NEG-199</td><td rowspan=1 colspan=1>5.04E+02</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=4 colspan=1>Influenza A H5b</td><td rowspan=1 colspan=1>A/Anhui/01/2005 (H5N1)-PR8-IBCDC-RG6</td><td rowspan=1 colspan=1>IRR FR-735</td><td rowspan=1 colspan=1>5.04E+02</td><td rowspan=1 colspan=1>3/3 (Matrix only)</td></tr><tr><td rowspan=1 colspan=1>A/Egypt/N03072/2010 (H5N1)-PR8-IDCDC-RG29</td><td rowspan=1 colspan=1>IRR FR-1065</td><td rowspan=1 colspan=1>5.04E+02</td><td rowspan=1 colspan=1>3/3 (Matrix only)</td></tr><tr><td rowspan=1 colspan=1>A/pheasant/New Jersey/1355/1998(H5N2)-PR8-IBCDC-4</td><td rowspan=1 colspan=1>IRR FR-771</td><td rowspan=1 colspan=1>5.04E+02</td><td rowspan=1 colspan=1>3/3 (Matrix only)</td></tr><tr><td rowspan=1 colspan=1>A/Hubei/1/2010 (H5N1)-PR8-IDCDC-RG30</td><td rowspan=1 colspan=1>IRR FR-1066</td><td rowspan=1 colspan=1>5.04E+02</td><td rowspan=1 colspan=1>3/3 (Matrix only)</td></tr><tr><td rowspan=2 colspan=1>Influenza A H7b</td><td rowspan=1 colspan=1>A/turkey/Virginia/4529/2002(H7N2) x PR8-IBCDC-5</td><td rowspan=1 colspan=1>IRR FR-772</td><td rowspan=1 colspan=1>5.04E+02</td><td rowspan=1 colspan=1>3/3 (Matrix only)</td></tr><tr><td rowspan=1 colspan=1>A/mallard/Netherlands/12/2000(H7N7)/PR8-IBCDC-1</td><td rowspan=1 colspan=1>IRR FR-773</td><td rowspan=1 colspan=1>5.04E+02</td><td rowspan=1 colspan=1>3/3 (Matrix only)</td></tr><tr><td rowspan=1 colspan=1>Influenza A H9b</td><td rowspan=1 colspan=1>A/Hong Kong/33982/2009 (H9N2)-PR8-IDCDC-RG26</td><td rowspan=1 colspan=1>IRR FR-1068</td><td rowspan=1 colspan=1>5.04E+02</td><td rowspan=1 colspan=1>3/3 (Matrix only)</td></tr><tr><td rowspan=4 colspan=1>Influenza Bc(Yamagata Lineage)</td><td rowspan=1 colspan=1>B/Florida/02/06*</td><td rowspan=1 colspan=1>ZeptoMetrix0810037CF</td><td rowspan=1 colspan=1>1.90E+02</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>B/Massachusetts/2/12</td><td rowspan=1 colspan=1>ZeptoMetrix0810239CF</td><td rowspan=1 colspan=1>1.90E+02</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>B/Wisconsin/1/10</td><td rowspan=1 colspan=1>ZeptoMetrix0810241CF</td><td rowspan=1 colspan=1>1.90E+02</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>B/Florida/04/06</td><td rowspan=1 colspan=1>ZeptoMetrix0810037CF(New PN:0810255CF)</td><td rowspan=1 colspan=1>1.90E+02</td><td rowspan=1 colspan=1>3/3</td></tr></table>

NxTAG® Respiratory Pathogen Panel v2 Traditional 510(k) Submission   

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>Supplier / PartNumber</td><td rowspan=1 colspan=1>ConcentrationTested(Copies/mL)</td><td rowspan=1 colspan=1># Detected/# Tested</td></tr><tr><td rowspan=5 colspan=1></td><td rowspan=1 colspan=1>B/Florida/07/04</td><td rowspan=1 colspan=1>ZeptoMetrix0810037CF (NewPN: 0810256CF)</td><td rowspan=1 colspan=1>1.90E+02</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>B/Panama/45/90</td><td rowspan=1 colspan=1>ZeptoMetrix0810037CF(New PN:0810259CF)</td><td rowspan=1 colspan=1>1.90E+02</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>B/Phuket/3073/13</td><td rowspan=1 colspan=1>ZeptoMetrix0810515CF</td><td rowspan=1 colspan=1>1.90E+02</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>B/Bangladesh/5972/2007</td><td rowspan=1 colspan=1>IRR FR-450</td><td rowspan=1 colspan=1>1.90E+02</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>B/Hubei-Wujiagang/158/2009</td><td rowspan=1 colspan=1>IRR FR-469</td><td rowspan=1 colspan=1>1.90E+02</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=7 colspan=1>Influenza Bc(Victoria Lineage)</td><td rowspan=1 colspan=1>B/Brisbane/33/08</td><td rowspan=1 colspan=1>ZeptoMetrix0810037CF(New PN:0810253CF)</td><td rowspan=1 colspan=1>1.90E+02</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>B/Brisbane/60/08</td><td rowspan=1 colspan=1>ZeptoMetrix0810037CF(New PN:0810254CF)</td><td rowspan=1 colspan=1>1.90E+02</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>B/Malaysia/2506/04</td><td rowspan=1 colspan=1>ZeptoMetrix0810258CF</td><td rowspan=1 colspan=1>1.90E+02</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>B/Colorado/06/17</td><td rowspan=1 colspan=1>ZeptoMetrix0810573CF</td><td rowspan=1 colspan=1>5.69E+02</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>B/Hong Kong/259/2010</td><td rowspan=1 colspan=1>IRR FR-663</td><td rowspan=1 colspan=1>1.90E+02</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>B/New Jersey/1/2012</td><td rowspan=1 colspan=1>IRR FR-1270</td><td rowspan=1 colspan=1>1.90E+02</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>B/Texas/02/2013</td><td rowspan=1 colspan=1>IRR FR-1302</td><td rowspan=1 colspan=1>1.90E+02</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=3 colspan=1>RSVA</td><td rowspan=1 colspan=1>A2*</td><td rowspan=1 colspan=1>ATCC VR-1540</td><td rowspan=1 colspan=1>1.49E+04</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>2006 Isolate</td><td rowspan=1 colspan=1>ZeptoMetrix0810040ACF</td><td rowspan=1 colspan=1>1.49E+04</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Long</td><td rowspan=1 colspan=1>ATCC VR-26</td><td rowspan=1 colspan=1>1.49E+04</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=5 colspan=1>RSVB</td><td rowspan=1 colspan=1>18537*</td><td rowspan=1 colspan=1>ATCC VR-1580</td><td rowspan=1 colspan=1>2.16E+04</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>CH93(18)-18</td><td rowspan=1 colspan=1>ZeptoMetrix0810040CF</td><td rowspan=1 colspan=1>2.16E+04</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>9320</td><td rowspan=1 colspan=1>ATCC VR-955</td><td rowspan=1 colspan=1>2.16E+04</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>B WV/14617/85</td><td rowspan=1 colspan=1>ATCC VR-1400</td><td rowspan=1 colspan=1>2.16E+04</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>B1</td><td rowspan=1 colspan=1>BEI NR-4052</td><td rowspan=1 colspan=1>2.16E+04</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>PIV1</td><td rowspan=1 colspan=1>Type 1*</td><td rowspan=1 colspan=1>ZeptoMetrix0810014CF</td><td rowspan=1 colspan=1>2.08E+03</td><td rowspan=1 colspan=1>3/3</td></tr></table>

Confidential & Restricted 510(k) Summary

NxTAG® Respiratory Pathogen Panel v2 Traditional 510(k) Submission   

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>Supplier / PartNumber</td><td rowspan=1 colspan=1>ConcentrationTested(Copies/mL)</td><td rowspan=1 colspan=1># Detected/# Tested</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>C35</td><td rowspan=1 colspan=1>ATCC VR-94</td><td rowspan=1 colspan=1>2.08E+03</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=2 colspan=1>PIV2</td><td rowspan=1 colspan=1>Greer*</td><td rowspan=1 colspan=1>ATCC VR-92</td><td rowspan=1 colspan=1>1.03E+03</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Type 2</td><td rowspan=1 colspan=1>ZeptoMetrix0810015CF</td><td rowspan=1 colspan=1>1.03E+03</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=4 colspan=1>PIV3</td><td rowspan=1 colspan=1>C 243*</td><td rowspan=1 colspan=1>ATCC VR-93</td><td rowspan=1 colspan=1>3.04E+03</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Type 3</td><td rowspan=1 colspan=1>ZeptoMetrix0810016CF</td><td rowspan=1 colspan=1>3.04E+03</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>ATCC-2011-5</td><td rowspan=1 colspan=1>ATCC VR-1782</td><td rowspan=1 colspan=1>3.04E+03</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>NIH 47885</td><td rowspan=1 colspan=1>BEI NR-3233</td><td rowspan=1 colspan=1>3.04E+03</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=4 colspan=1>PIV4</td><td rowspan=1 colspan=1>Type 4A*</td><td rowspan=1 colspan=1>ZeptoMetrix0810060CF</td><td rowspan=1 colspan=1>5.08E+04</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>M-25; Type 4A</td><td rowspan=1 colspan=1>ATCC VR-1378</td><td rowspan=1 colspan=1>5.08E+04</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>CH 19503; Type 4B*</td><td rowspan=1 colspan=1>ATCC VR-1377</td><td rowspan=1 colspan=1>2.14E+04</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Type 4B</td><td rowspan=1 colspan=1>ZeptoMetrix0810060BCF</td><td rowspan=1 colspan=1>2.14E+04</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=13 colspan=1>SARS-CoV-2</td><td rowspan=1 colspan=1>2019-nCoV/USA-WA1/2020*</td><td rowspan=1 colspan=1>ATCC VR-1986HK</td><td rowspan=1 colspan=1>1.50E+03</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>HongKong/VM2000106/2020</td><td rowspan=1 colspan=1>ZeptoMetrix0810590CFHI</td><td rowspan=1 colspan=1>1.50E+03</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>USA-WA1/2020</td><td rowspan=1 colspan=1>ZeptoMetrix0810587CFHI</td><td rowspan=1 colspan=1>1.50E+03</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>BetaCoV/Germany/BavPat1/2020p.1d</td><td rowspan=1 colspan=1>EVAg 026N-03889</td><td rowspan=1 colspan=1>2.73E+03</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>2019-nCoV/Italy-INMI1d</td><td rowspan=1 colspan=1>EVAg 008N-03894</td><td rowspan=1 colspan=1>2.73E+03</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>England/02/2020d</td><td rowspan=1 colspan=1>BEI NR-52499</td><td rowspan=1 colspan=1>2.73E+03e</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Singapore/2/20202d</td><td rowspan=1 colspan=1>BEI NR-52501</td><td rowspan=1 colspan=1>2.73E+03e</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>USA-IL1/2020d</td><td rowspan=1 colspan=1>BEI NR-52503</td><td rowspan=1 colspan=1>2.73E+03e</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>USA-CA1/2020d</td><td rowspan=1 colspan=1>BEI NR-52504</td><td rowspan=1 colspan=1>2.73E+03e</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>USA-AZ1/2020d</td><td rowspan=1 colspan=1>BEI NR-52505</td><td rowspan=1 colspan=1>2.73E+03e</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>USA-WI11/2020d</td><td rowspan=1 colspan=1>BEI NR-52506</td><td rowspan=1 colspan=1>2.73E+03e</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>USA-CA3/2020d</td><td rowspan=1 colspan=1>BEI NR-52507</td><td rowspan=1 colspan=1>2.73E+03e</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>USA-CA4/2020d</td><td rowspan=1 colspan=1>BEI NR-52508</td><td rowspan=1 colspan=1>2.73E+03e</td><td rowspan=1 colspan=1>3/3</td></tr></table>

NxTAG® Respiratory Pathogen Panel v2 Traditional 510(k) Submission   

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>Supplier / PartNumber</td><td rowspan=1 colspan=1>ConcentrationTested(Copies/mL)</td><td rowspan=1 colspan=1># Detected/# Tested</td></tr><tr><td rowspan=17 colspan=1></td><td rowspan=1 colspan=1>USA-CA2/2020d</td><td rowspan=1 colspan=1>BEI NR-52509</td><td rowspan=1 colspan=1>2.73E+03e</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Chile/Santiago_op4d1/2020d</td><td rowspan=1 colspan=1>BEI NR-52510</td><td rowspan=1 colspan=1>2.73E+03e</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>New York-PV08410/2020d</td><td rowspan=1 colspan=1>BEI NR-53518</td><td rowspan=1 colspan=1>2.73E+03e</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>USA/CA_CDC_5574/2020, HeatInactivated</td><td rowspan=1 colspan=1>BEI NR-55245</td><td rowspan=1 colspan=1>1.50E+03e</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Alpha (B1.1.7)/UK Variant</td><td rowspan=1 colspan=1>Clinical Specimen</td><td rowspan=1 colspan=1>1.50E+03</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Epsilon (B1.429)/California Variant</td><td rowspan=1 colspan=1>Clinical Specimen</td><td rowspan=1 colspan=1>1.50E+03</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Epsilon (B1.429)/California Variant</td><td rowspan=1 colspan=1>Clinical Specimen</td><td rowspan=1 colspan=1>1.50E+03</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Delta (B.1.617.2)</td><td rowspan=1 colspan=1>Clinical Specimen</td><td rowspan=1 colspan=1>1.50E+03</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Delta (B.1.617.2)</td><td rowspan=1 colspan=1>Clinical Specimen</td><td rowspan=1 colspan=1>1.50E+03</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Delta (B.1.617.2)</td><td rowspan=1 colspan=1>Clinical Specimen</td><td rowspan=1 colspan=1>1.50E+03</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Delta (B.1.617.2)</td><td rowspan=1 colspan=1>Clinical Specimen</td><td rowspan=1 colspan=1>1.50E+03</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Delta (B.1.617.2)</td><td rowspan=1 colspan=1>Clinical Specimen</td><td rowspan=1 colspan=1>1.50E+03</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Omicron (B.1.1.529 and BAlineages)</td><td rowspan=1 colspan=1>Clinical Specimen</td><td rowspan=1 colspan=1>1.50E+03</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Omicron (B.1.1.529 and BAlineages)</td><td rowspan=1 colspan=1>Clinical Specimen</td><td rowspan=1 colspan=1>1.50E+03</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Omicron (B.1.1.529 and BAlineages)</td><td rowspan=1 colspan=1>Clinical Specimen</td><td rowspan=1 colspan=1>1.50E+03</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Omicron (B.1.1.529 and BAlineages)</td><td rowspan=1 colspan=1>Clinical Specimen</td><td rowspan=1 colspan=1>1.50E+03</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Omicron (B.1.1.529 and BAlineages)</td><td rowspan=1 colspan=1>Clinical Specimen</td><td rowspan=1 colspan=1>1.50E+03</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=2 colspan=1>Coronavirus 229E</td><td rowspan=1 colspan=1>229E*</td><td rowspan=1 colspan=1>ATCC VR-740</td><td rowspan=1 colspan=1>1.14E+03</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>229E</td><td rowspan=1 colspan=1>ZeptoMetrix0810229CF</td><td rowspan=1 colspan=1>1.14E+03</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=2 colspan=1>Coronavirus NL63</td><td rowspan=1 colspan=1>NL63*</td><td rowspan=1 colspan=1>ZeptoMetrix0810228CF</td><td rowspan=1 colspan=1>3.00E+02</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>NL63</td><td rowspan=1 colspan=1>BEI NR-470</td><td rowspan=1 colspan=1>3.00E+02</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=2 colspan=1>Coronavirus OC43</td><td rowspan=1 colspan=1>Betacoronavirus 1*</td><td rowspan=1 colspan=1>ATCC VR-1558</td><td rowspan=1 colspan=1>1.36E+04</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>OC43</td><td rowspan=1 colspan=1>ZeptoMetrix0810024CF</td><td rowspan=1 colspan=1>1.36E+04</td><td rowspan=1 colspan=1>3/3</td></tr></table>

NxTAG® Respiratory Pathogen Panel v2 Traditional 510(k) Submission   

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>Supplier / PartNumber</td><td rowspan=1 colspan=1>ConcentrationTested(Copies/mL)</td><td rowspan=1 colspan=1># Detected/# Tested</td></tr><tr><td rowspan=4 colspan=1>Coronavirus HKU1</td><td rowspan=1 colspan=1>HKU1, Genotype B*</td><td rowspan=1 colspan=1>Clinical Specimen</td><td rowspan=1 colspan=1>1.25E+04</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>HKU1, Genotype B</td><td rowspan=1 colspan=1>Clinical Specimen</td><td rowspan=1 colspan=1>1.25E+04</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>HKU1, Genotype A</td><td rowspan=1 colspan=1>Clinical Specimen</td><td rowspan=1 colspan=1>1.25E+04</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>HKU1, Genotype A</td><td rowspan=1 colspan=1>Clinical Specimen</td><td rowspan=1 colspan=1>1.25E+04</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=9 colspan=1>HumanMetapneumovirus</td><td rowspan=1 colspan=1>Type A1, IA10-2003, hMPV-16*</td><td rowspan=1 colspan=1>ZeptoMetrix0810161CF</td><td rowspan=1 colspan=1>2.15E+02</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Type A1, IA3-2002, hMPV-9</td><td rowspan=1 colspan=1>ZeptoMetrix0810160CF</td><td rowspan=1 colspan=1>2.15E+02</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Type A2, IA27-2004, hMPV-27</td><td rowspan=1 colspan=1>ZeptoMetrix0810164CF</td><td rowspan=1 colspan=1>2.15E+02</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Type A2, DHI 26583</td><td rowspan=1 colspan=1>Clinical Specimen</td><td rowspan=1 colspan=1>2.15E+02</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Type B1, Peru2-2002, hMPV-3*</td><td rowspan=1 colspan=1>ZeptoMetrix0810156CF</td><td rowspan=1 colspan=1>7.85E+02</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Type B1, Peru3-2003, hMPV-5</td><td rowspan=1 colspan=1>ZeptoMetrix0810158CF</td><td rowspan=1 colspan=1>7.85E+02</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Type B2, Peru1-2002, hMPV-4</td><td rowspan=1 colspan=1>ZeptoMetrix0810157CF</td><td rowspan=1 colspan=1>7.85E+02</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Type B2, Peru6-2003, hMPV-8</td><td rowspan=1 colspan=1>ZeptoMetrix0810159CF</td><td rowspan=1 colspan=1>7.85E+02</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Type B2, IA18-2003, hMPV-18</td><td rowspan=1 colspan=1>ZeptoMetrix0810162CF</td><td rowspan=1 colspan=1>7.85E+02</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=9 colspan=1>Rhinovirusf</td><td rowspan=1 colspan=1>Species A, Type 85, strain 50-525-CV54 [V-192-001-021]*</td><td rowspan=1 colspan=1>ATCC VR-1195</td><td rowspan=1 colspan=1>4.61E+03</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Species A, Type 1A</td><td rowspan=1 colspan=1>ZeptoMetrix0810012CFN</td><td rowspan=1 colspan=1>4.61E+03</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Species A, Type 2, strain HGP</td><td rowspan=1 colspan=1>ATCC VR-482</td><td rowspan=1 colspan=1>4.61E+03</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Species A, Type 7, strain 68-CV11</td><td rowspan=1 colspan=1>ATCC VR-1601</td><td rowspan=1 colspan=1>4.61E+03</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Species A, Type 16</td><td rowspan=1 colspan=1>ZeptoMetrix0810285CF</td><td rowspan=1 colspan=1>4.61E+03</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Species A, Type 34, strain 137-3</td><td rowspan=1 colspan=1>ATCC VR-1365</td><td rowspan=1 colspan=1>4.61E+03</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Species A, Type 39, strain 209</td><td rowspan=1 colspan=1>ATCC VR-340</td><td rowspan=1 colspan=1>4.61E+03</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Species A, Type 54, strain FO1-3774</td><td rowspan=1 colspan=1>ATCC VR-1661</td><td rowspan=1 colspan=1>4.61E+03</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Species A, Type 57, strain Ch47</td><td rowspan=1 colspan=1>ATCC VR-1600</td><td rowspan=1 colspan=1>4.61E+03</td><td rowspan=1 colspan=1>3/3</td></tr></table>

NxTAG® Respiratory Pathogen Panel v2 Traditional 510(k) Submission   

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>Supplier / PartNumber</td><td rowspan=1 colspan=1>ConcentrationTested(Copies/mL)</td><td rowspan=1 colspan=1># Detected/# Tested</td></tr><tr><td rowspan=12 colspan=1></td><td rowspan=1 colspan=1>Species A, Type 77, strain 130-63[V-185-001-021]</td><td rowspan=1 colspan=1>ATCC VR-1187</td><td rowspan=1 colspan=1>4.61E+03</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Species B, Type 3, strain FEB</td><td rowspan=1 colspan=1>ATCC VR-483</td><td rowspan=1 colspan=1>4.61E+03</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Species B, Type 14, strain 1059</td><td rowspan=1 colspan=1>ATCC VR-284</td><td rowspan=1 colspan=1>4.61E+03</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Species B, Type 17, strain 33342</td><td rowspan=1 colspan=1>ATCC VR-1663</td><td rowspan=1 colspan=1>4.61E+03</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Species B, Type 27, strain 5870[5870-CV28] (NIAID V-144-001-021)</td><td rowspan=1 colspan=1>ATCC VR-1137</td><td rowspan=1 colspan=1>4.61E+03</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Species B, Type 42, strain 56822</td><td rowspan=1 colspan=1>ATCC VR-338</td><td rowspan=1 colspan=1>4.61E+03</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Species B, Type 83, strain Baylor 7[V-190-001-021]</td><td rowspan=1 colspan=1>ATCC VR-1193</td><td rowspan=1 colspan=1>4.61E+03</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Type C</td><td rowspan=1 colspan=1>Clinical Specimen</td><td rowspan=1 colspan=1>4.61E+03</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Type C</td><td rowspan=1 colspan=1>Clinical Specimen</td><td rowspan=1 colspan=1>4.61E+03</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Type C</td><td rowspan=1 colspan=1>Clinical Specimen</td><td rowspan=1 colspan=1>4.61E+03</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Type C</td><td rowspan=1 colspan=1>Clinical Specimen</td><td rowspan=1 colspan=1>4.61E+03</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Type C</td><td rowspan=1 colspan=1>Clinical Specimen</td><td rowspan=1 colspan=1>4.61E+03</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=9 colspan=1>Enterovirus</td><td rowspan=1 colspan=1>Species D, Type 68, 2007 isolate*</td><td rowspan=1 colspan=1>ZeptoMetrix0810237CF</td><td rowspan=1 colspan=1>1.06E+04</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Species A, Human Enterovirus 71,strain H</td><td rowspan=1 colspan=1>ATCC VR-1432</td><td rowspan=1 colspan=1>1.06E+04</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Species A, Human Coxsackie A10,strain M.K. (Kowalik)</td><td rowspan=1 colspan=1>ATCC VR-168</td><td rowspan=1 colspan=1>1.06E+04</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Species B, Human CoxsackievirusB1, strain Conn-5</td><td rowspan=1 colspan=1>ATCC VR-28</td><td rowspan=1 colspan=1>1.06E+04</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Species B, Human CoxsackievirusB4, strain J.V.B. (Benschoten)</td><td rowspan=1 colspan=1>ATCC VR-184</td><td rowspan=1 colspan=1>1.06E+04</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Species B, Human Echovirus 11,strain Gregory</td><td rowspan=1 colspan=1>ATCC VR-41</td><td rowspan=1 colspan=1>2.86E+05</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Species B, Human Echovirus 13,strain Del Carmen</td><td rowspan=1 colspan=1>ATCC VR-1054(New PN: VR-43)</td><td rowspan=1 colspan=1>1.06E+04</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Species B, Enterovirus Type 69,strain Toluca-1 [V-068-001-021]</td><td rowspan=1 colspan=1>ATCC VR-1077</td><td rowspan=1 colspan=1>1.06E+04</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Species C, Human coxsackievirusA21, strain Kuykendall</td><td rowspan=1 colspan=1>ATCC VR-850</td><td rowspan=1 colspan=1>1.06E+04</td><td rowspan=1 colspan=1>3/3</td></tr></table>

NxTAG® Respiratory Pathogen Panel v2 Traditional 510(k) Submission   

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>Supplier / PartNumber</td><td rowspan=1 colspan=1>ConcentrationTested(Copies/mL)</td><td rowspan=1 colspan=1># Detected/# Tested</td></tr><tr><td rowspan=7 colspan=1></td><td rowspan=1 colspan=1>Species C, Human coxsackievirusA24, strain DN-19</td><td rowspan=1 colspan=1>ATCC VR-1662</td><td rowspan=1 colspan=1>1.06E+04</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Species D, Type 68, 2014 Isolate</td><td rowspan=1 colspan=1>ZeptoMetrix0810300CF</td><td rowspan=1 colspan=1>1.06E+04</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Species D, Type 68, strainUS/MO/14-18947</td><td rowspan=1 colspan=1>ATCC VR-1823</td><td rowspan=1 colspan=1>1.06E+04</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Species D, Type 68, strain US/IL/14-18952</td><td rowspan=1 colspan=1>ATCC VR-1824</td><td rowspan=1 colspan=1>1.06E+04</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Species D, Type 68, strainUS/KY/14-18953</td><td rowspan=1 colspan=1>ATCC VR-1825</td><td rowspan=1 colspan=1>1.06E+04</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Species D, Type 68, strain Fermon</td><td rowspan=1 colspan=1>ATCC VR-1076(New PN: VR-1826)</td><td rowspan=1 colspan=1>1.06E+04</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Species D, Type 70, strain J670/71</td><td rowspan=1 colspan=1>ATCC VR-836</td><td rowspan=1 colspan=1>1.06E+04</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=11 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>Species B, Type 14, 2006 isolate*</td><td rowspan=1 colspan=1>ZeptoMetrix0810108CF</td><td rowspan=1 colspan=1>7.38E+03</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Species B, Type 3</td><td rowspan=1 colspan=1>ZeptoMetrix0810062CF</td><td rowspan=1 colspan=1>7.38E+03</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Species B, Type 7, strain Gomen</td><td rowspan=1 colspan=1>ATCC VR-7</td><td rowspan=1 colspan=1>7.38E+03</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Species B, Type 7A</td><td rowspan=1 colspan=1>ZeptoMetrix0810021CF</td><td rowspan=1 colspan=1>7.38E+03</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Species B, Type 21, AV-1645 [128]</td><td rowspan=1 colspan=1>ATCC VR-1098(New PN: VR-256)</td><td rowspan=1 colspan=1>7.38E+03</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Species C, Type 1*</td><td rowspan=1 colspan=1>ZeptoMetrix0810050CF</td><td rowspan=1 colspan=1>6.02E+04</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Species C, Type 1, strain Adenoid71</td><td rowspan=1 colspan=1>ATCC VR-1</td><td rowspan=1 colspan=1>6.02E+04</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Species C, Type 2</td><td rowspan=1 colspan=1>ZeptoMetrix0810110CF</td><td rowspan=1 colspan=1>6.02E+04</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Species C, Type 5</td><td rowspan=1 colspan=1>ZeptoMetrix0810020CF</td><td rowspan=1 colspan=1>6.02E+04</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Species C, Type 6</td><td rowspan=1 colspan=1>ZeptoMetrix0810111CF</td><td rowspan=1 colspan=1>6.02E+04</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Species E, Type 4*</td><td rowspan=1 colspan=1>ZeptoMetrix0810070CF</td><td rowspan=1 colspan=1>2.20E+04</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=2 colspan=1>C. pneumoniae</td><td rowspan=1 colspan=1>TW-183*</td><td rowspan=1 colspan=1>ATCC VR-2282</td><td rowspan=1 colspan=1>7.13E+02</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>TWAR (CDC/CWL-029)</td><td rowspan=1 colspan=1>ATCC VR-1310</td><td rowspan=1 colspan=1>7.13E+02</td><td rowspan=1 colspan=1>3/3</td></tr></table>

NxTAG® Respiratory Pathogen Panel v2 Traditional 510(k) Submission   

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>Supplier / PartNumber</td><td rowspan=1 colspan=1>ConcentrationTested(Copies/mL)</td><td rowspan=1 colspan=1># Detected/# Tested</td></tr><tr><td rowspan=2 colspan=1></td><td rowspan=1 colspan=1>TWAR 2023</td><td rowspan=1 colspan=1>ATCC VR-1356</td><td rowspan=1 colspan=1>7.13E+02</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>AR-39</td><td rowspan=1 colspan=1>ATCC 53592</td><td rowspan=1 colspan=1>7.13E+02</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=4 colspan=1>M. pneumoniae</td><td rowspan=1 colspan=1>M129*</td><td rowspan=1 colspan=1>ZeptoMetrix0801579</td><td rowspan=1 colspan=1>1.29E+04</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>[M52]</td><td rowspan=1 colspan=1>ATCC 15293</td><td rowspan=1 colspan=1>1.29E+04</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>FH strain of Eaton Agent [NCTC10119]</td><td rowspan=1 colspan=1>ATCC 15531-TTR</td><td rowspan=1 colspan=1>1.29E+04</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Mutant 22</td><td rowspan=1 colspan=1>ATCC 39505</td><td rowspan=1 colspan=1>1.29E+04</td><td rowspan=1 colspan=1>3/3</td></tr></table>

\* Indicates a LoD strain. a Highest possible concentration. b NxTAG RPP v2 does not differentiate Influenza A H5, H7 or H9, all of which are reported as Influenza A c NxTAG RPP v2 does not differentiate the Yamagata and Victoria lineages, both of which are reported as influenza B d Samples were obtained as RNA. The RNA was diluted in extracted negative simulated matrix to a concentration that represented $1 . 5 0 \mathsf { E } { + } 0 3$ copies/mL in a raw sample. e Concentration units for these strains are Genome equivalents/mL. f Reported by NxTAG RPP v2 as Rhinovirus/Enterovirus

# Analytical Reactivity In Silico Analysis

Based on in silico inclusivity analysis, it is predicted that the SARS-CoV-2 sequences available from GISAID EpiCoV database as of November 30, 2023, including sequences from all defined variants of concern or interest, are $100 \%$ detectable by NxTAG® Respiratory Pathogen Panel v2 (NxTAG RPP v2) assay.

Influenza A and B inclusivity was assessed with sequences available from the GISAID EpiFlu database between January 1, 2017 and May 5, 2023, as well as between January 1, 2000 and December 31, 2008. The assay oligos for Influenza A, Influenza A H1 (including H1pdm09), Influenza A H3, and Influenza B are predicted to have ${ \sim } 9 9 \%$ inclusivity against the analyzed sequences.

For all targets other than SARS-CoV-2 and the influenza viruses, in silico inclusivity analysis was performed with sequences available from the GenBank® Nucleotide (nt) database as of April 8, 2023. Based on this analysis, $29 6 \%$ of sequences of each analyte were predicted to be detected by NxTAG® RPP v2, except for Parainfluenza Virus 2 $( \sim 9 2 \% )$ and untyped strains of Parainfluenza Virus 4 $( \sim 9 4 \% )$ , which exhibited lower homology.

g. Analytical Specificity

# i) Cross-Reactivity

Analytical Specificity (Exclusivity) of the $\mathsf { N } \times \mathsf { T } \mathsf { A G } ^ { \textregistered }$ Respiratory Pathogen Panel v2 $( { \mathsf { N } } \times { \mathsf { T } } { \mathsf { A G } } ^ { \otimes }$ RPP v2) assay was assessed with pathogens that cause respiratory infections or those that may be found in respiratory specimens. Sixty-three (63) organisms (82 strains total) were tested, including 41 pathogens that are not detected by NxTAG® RPP v2, and pooled nasal wash (referred to as “Off-panel organisms” – Table 6) and 22 that are detected by the assay (referred to as “OnPanel organisms” – Table 7). Each strain was prepared in negative simulated matrix (NSM) to reach the high positive concentration and tested in triplicate on the $\mathsf { N x T A G } ^ { \otimes }$ RPP v2 assay. None of the off-panel or on-panel organisms tested showed cross-reactivity, with the exception of one strain: Enterovirus (Species D, Type 68, US/IL/14-18952). This strain generated a false positive call for Influenza A H3 when it was tested at $\ge 1 . 0 0 \mathsf { E } + 0 3 \mathsf { T C l D } _ { 5 0 } / \mathsf { m L }$ , although the influenza A matrix gene target was negative. No false positive call was generated when the strain was tested at $1 . 0 0 5 \substack { + 0 2 \top \mathsf { C } 1 } \mathsf { D } _ { 5 0 } / \mathsf { m L }$ and five other isolates of Enterovirus D showed no evidence of cross-reaction.

Table 6: NxTAG® RPP v2 Analytical Specificity (Off-Panel Organisms)   

<table><tr><td colspan="1" rowspan="1">Off-Panel Organisms</td><td colspan="2" rowspan="1">Concentration Tested</td><td colspan="1" rowspan="1">Cross-ReactivityDetected</td></tr><tr><td colspan="1" rowspan="1">Aspergillus flavus</td><td colspan="1" rowspan="1">1.00E+06</td><td colspan="1" rowspan="1">CFU/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Aspergillus fumigatus</td><td colspan="1" rowspan="1">1.00E+06</td><td colspan="1" rowspan="1">CFU/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Bordetella parapertussis</td><td colspan="1" rowspan="1">1.00E+06</td><td colspan="1" rowspan="1">CFU/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Bordetella pertussis</td><td colspan="1" rowspan="1">1.00E+06</td><td colspan="1" rowspan="1">CFU/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Candida albicans</td><td colspan="1" rowspan="1">1.00E+06</td><td colspan="1" rowspan="1">CFU/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Chlamydia trachomatis</td><td colspan="1" rowspan="1">1.00E+06</td><td colspan="1" rowspan="1">IFU/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium diphtheriae</td><td colspan="1" rowspan="1">1.00E+06</td><td colspan="1" rowspan="1">CFU/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium striatum</td><td colspan="1" rowspan="1">1.00E+06</td><td colspan="1" rowspan="1">CFU/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Cytomegalovirus</td><td colspan="1" rowspan="1">1.00E+05</td><td colspan="1" rowspan="1">TCID5o/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Epstein Barr Virus</td><td colspan="1" rowspan="1">1.00E+07</td><td colspan="1" rowspan="1">Copies/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">1.00E+06</td><td colspan="1" rowspan="1">CFU/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Fusobacterium necrophorum</td><td colspan="1" rowspan="1">1.00E+06</td><td colspan="1" rowspan="1">CFU/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Haemophilus influenzae</td><td colspan="1" rowspan="1">1.00E+06</td><td colspan="1" rowspan="1">CFU/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Herpes Simplex virus Type 1</td><td colspan="1" rowspan="1">1.00E+05</td><td colspan="1" rowspan="1">TCID50/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Human Bocavirus</td><td colspan="1" rowspan="1">1.00E+07</td><td colspan="1" rowspan="1">Copies/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">1.00E+06</td><td colspan="1" rowspan="1">CFU/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus acidophilus</td><td colspan="1" rowspan="1">1.00E+06</td><td colspan="1" rowspan="1">CFU/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus plantarum</td><td colspan="1" rowspan="1">1.00E+06</td><td colspan="1" rowspan="1">CFU/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Legionella (Tatlockia) micdadei</td><td colspan="1" rowspan="1">1.00E+06</td><td colspan="1" rowspan="1">CFU/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Legionella pneumophila</td><td colspan="1" rowspan="1">1.00E+06</td><td colspan="1" rowspan="1">CFU/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Measles Virus</td><td colspan="1" rowspan="1">1.00E+05</td><td colspan="1" rowspan="1">TCID50/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">MERS-coronavirus</td><td colspan="1" rowspan="1">1.00E+05</td><td colspan="1" rowspan="1">TCID50/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Moraxella catarrhalis</td><td colspan="1" rowspan="1">1.00E+06</td><td colspan="1" rowspan="1">CFU/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Mumps Virus</td><td colspan="1" rowspan="1">1.00E+05</td><td colspan="1" rowspan="1">TCID50/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Mycobacterium tuberculosis</td><td colspan="1" rowspan="1">1.00E+06</td><td colspan="1" rowspan="1">CFU/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Mycoplasma genitalium</td><td colspan="1" rowspan="1">1.00E+051</td><td colspan="1" rowspan="1">CCU/mL(Approximate)</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Mycoplasma hominis</td><td colspan="1" rowspan="1">1.00E+06</td><td colspan="1" rowspan="1">CCU/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Neisseria elongata</td><td colspan="1" rowspan="1">1.00E+06</td><td colspan="1" rowspan="1">CFU/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Neisseria gonorrhoeae</td><td colspan="1" rowspan="1">1.00E+06</td><td colspan="1" rowspan="1">CFU/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Neisseria meningitidis</td><td colspan="1" rowspan="1">1.00E+06</td><td colspan="1" rowspan="1">CFU/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Pneumocystis carinii</td><td colspan="1" rowspan="1">1.00E+06</td><td colspan="1" rowspan="1">nuclei/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">1.00E+06</td><td colspan="1" rowspan="1">CFU/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Serratia marcescens</td><td colspan="1" rowspan="1">1.00E+06</td><td colspan="1" rowspan="1">CFU/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">1.00E+06</td><td colspan="1" rowspan="1">CFU/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus epidermidis</td><td colspan="1" rowspan="1">1.00E+06</td><td colspan="1" rowspan="1">CFU/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Streptococcus agalactiae</td><td colspan="1" rowspan="1">1.00E+06</td><td colspan="1" rowspan="1">CFU/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Streptococcus pneumoniae</td><td colspan="1" rowspan="1">1.00E+06</td><td colspan="1" rowspan="1">CFU/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Streptococcus pyogenes</td><td colspan="1" rowspan="1">1.00E+06</td><td colspan="1" rowspan="1">CFU/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Streptococcus salivarius</td><td colspan="1" rowspan="1">1.00E+06</td><td colspan="1" rowspan="1">CFU/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">SARS-coronavirus</td><td colspan="1" rowspan="1">3.01E+051</td><td colspan="1" rowspan="1">Copies/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Varicella Zoster Virus</td><td colspan="1" rowspan="1">1.00E+05</td><td colspan="1" rowspan="1">TCID50/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">N/A (Pooled Nasal Wash)</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">None</td></tr></table>

1Highest concentration based on the available stock.

Table 7: NxTAG® RPP v2 Analytical Specificity (On-Panel Organisms)   

<table><tr><td rowspan=1 colspan=1>On-Panel Organisms (strain/subtype)</td><td rowspan=1 colspan=2>Concentration Tested</td><td rowspan=1 colspan=1>Cross-ReactivityDetected</td></tr><tr><td rowspan=1 colspan=1>Influenza A H1N1 (Brisbane/59/07)</td><td rowspan=1 colspan=1>1.00E+05</td><td rowspan=1 colspan=1>TCID5o/mL</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Influenza A 2009 H1N1 (A/NY/02/09)</td><td rowspan=1 colspan=1>1.00E+05</td><td rowspan=1 colspan=1>TCID50/mL</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Influenza A H3N2 (Wisconsin/67/05)</td><td rowspan=1 colspan=1>1.00E+05</td><td rowspan=1 colspan=1>TCID50/mL</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Influenza B (Florida/02/06) (LN: 325286)</td><td rowspan=1 colspan=1>1.00E+05</td><td rowspan=1 colspan=1>TCID50/mL</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Influenza B (Florida/02/06) (LN: 325345)</td><td rowspan=1 colspan=1>1.00E+05</td><td rowspan=1 colspan=1>TCID5o/mL</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Influenza B (Florida/02/06) (LN: 307551)</td><td rowspan=1 colspan=1>1.00E+05</td><td rowspan=1 colspan=1>TCID50/mL</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Influenza B (B/Brisbane/33/2008)</td><td rowspan=1 colspan=1>1.00E+05</td><td rowspan=1 colspan=1>TCID50/mL</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Influenza B (B/Massachusetts/2/12)</td><td rowspan=1 colspan=1>1.00E+05</td><td rowspan=1 colspan=1>TCID5o/mL</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Influenza B (B/Wisconsin/1/2010)</td><td rowspan=1 colspan=1>1.00E+05</td><td rowspan=1 colspan=1>TCID5o/mL</td><td rowspan=1 colspan=1>None</td></tr></table>

Confidential & Restricted 510(k) Summary

<table><tr><td colspan="1" rowspan="1">On-Panel Organisms (strain/subtype)</td><td colspan="2" rowspan="1">Concentration Tested</td><td colspan="1" rowspan="1">Cross-ReactivityDetected</td></tr><tr><td colspan="1" rowspan="1">Influenza B (B/Brigit (B/Russia/69))</td><td colspan="1" rowspan="1">1.00E+05</td><td colspan="1" rowspan="1">CEID50/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Influenza B (B/Hong Kong/5/72)</td><td colspan="1" rowspan="1">1.00E+05</td><td colspan="1" rowspan="1">CEID5o/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Influenza B (B/Russia/69)</td><td colspan="1" rowspan="1">1.00E+05</td><td colspan="1" rowspan="1">CEID50/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Influenza B (B/GreLakes/1739/1954)</td><td colspan="1" rowspan="1">1.00E+05</td><td colspan="1" rowspan="1">CEID50/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Influenza B (B/Bangladesh/5972/2007)</td><td colspan="1" rowspan="1">1.00E+05</td><td colspan="1" rowspan="1">TCID5o/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Influenza B (B/Hong Kong/259/2010)</td><td colspan="1" rowspan="1">1.00E+05</td><td colspan="1" rowspan="1">CEID5o/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Influenza B (B/Texas/02/2013)</td><td colspan="1" rowspan="1">1.00E+05</td><td colspan="1" rowspan="1">TCID50/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Influenza B (B/Hubei-Wujiagang/158/2009)</td><td colspan="1" rowspan="1">1.00E+05</td><td colspan="1" rowspan="1">CEID50/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Influenza B (B/New Jersey/1/2012)</td><td colspan="1" rowspan="1">1.00E+05</td><td colspan="1" rowspan="1">TCID5o/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Influenza B (B/Brisbane/3/2007)</td><td colspan="1" rowspan="1">1.00E+05</td><td colspan="1" rowspan="1">CEID50/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Human Respiratory Syncytial Virus A (A2)</td><td colspan="1" rowspan="1">1.00E+05</td><td colspan="1" rowspan="1">PFU/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Human Respiratory Syncytial Virus B (18537)</td><td colspan="1" rowspan="1">7.00E+04^</td><td colspan="1" rowspan="1">PFU/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">SARS-CoV-2 (USA-WA-1/2020)</td><td colspan="1" rowspan="1">1.00E+07</td><td colspan="1" rowspan="1">Copies/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Coronavirus 229E</td><td colspan="1" rowspan="1">1.00E+05</td><td colspan="1" rowspan="1">TCID5o/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Coronavirus OC43</td><td colspan="1" rowspan="1">1.00E+05</td><td colspan="1" rowspan="1">TCID50/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Coronavirus NL63</td><td colspan="1" rowspan="1">1.00E+05</td><td colspan="1" rowspan="1">TCIDso/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Coronavirus HKU1</td><td colspan="1" rowspan="1">1.00E+06a</td><td colspan="1" rowspan="1">Copies/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Human Metapneumovirus (hMPV-16, Type A1,IA10-2003)</td><td colspan="1" rowspan="1">1.00E+05</td><td colspan="1" rowspan="1">TCID50/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Human Metapneumovirus (hMPV-2, Type B1,Peru2-2002)</td><td colspan="1" rowspan="1">3.89E+04a</td><td colspan="1" rowspan="1">TCID5o/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Rhinovirus (Type 85, 50-525-CV54 [V-192-001-021]</td><td colspan="1" rowspan="1">1.00E+05</td><td colspan="1" rowspan="1">TCID50/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Enterovirus (Species D, Type 68, 2007 Isolate)</td><td colspan="1" rowspan="1">1.00E+05</td><td colspan="1" rowspan="1">TCID50/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Enterovirus (Species D, Type 68, US/MO/14-18947)</td><td colspan="1" rowspan="1">1.00E+05</td><td colspan="1" rowspan="1">TCID5o/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Enterovirus (Species D, Type 68, US/IL/14-18952)</td><td colspan="1" rowspan="1">1.00E+03</td><td colspan="1" rowspan="1">TCID50/mL</td><td colspan="1" rowspan="1">Influenza AH3</td></tr><tr><td colspan="1" rowspan="1">Enterovirus (Species D, Type 68, US/KY/14-18953)</td><td colspan="1" rowspan="1">1.00E+05</td><td colspan="1" rowspan="1">TCID50/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Enterovirus (Species D, Type 68, Fermon)</td><td colspan="1" rowspan="1">1.00E+05</td><td colspan="1" rowspan="1">TCID50/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Enterovirus (68, 2014 Isolate 1)</td><td colspan="1" rowspan="1">1.00E+05</td><td colspan="1" rowspan="1">TCID50/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Adenovirus (Species B; Type 14, 2006 isolate)</td><td colspan="1" rowspan="1">1.00E+05</td><td colspan="1" rowspan="1">TCID50/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza virus 1 (Type 1)</td><td colspan="1" rowspan="1">1.00E+05</td><td colspan="1" rowspan="1">TCID5o/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza virus 2 (Greer)</td><td colspan="1" rowspan="1">1.00E+05</td><td colspan="1" rowspan="1">TCID5o/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza virus 3 (C243)</td><td colspan="1" rowspan="1">1.00E+05</td><td colspan="1" rowspan="1">TCID50/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza virus 4 (Subtype A)</td><td colspan="1" rowspan="1">1.00E+05</td><td colspan="1" rowspan="1">TCID5o/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza virus 4 (Subtype B; CH 19503)</td><td colspan="1" rowspan="1">1.00E+05</td><td colspan="1" rowspan="1">TCID5o/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Chlamydia pneumoniae (TW-183)</td><td colspan="1" rowspan="1">1.00E+06</td><td colspan="1" rowspan="1">IFU/mL</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Mycoplasma pneumoniae (M129)</td><td colspan="1" rowspan="1">1.00E+06</td><td colspan="1" rowspan="1">CCU/mL</td><td colspan="1" rowspan="1">None</td></tr></table>

aHighest concentration possible based on the available stock.

# Analytical Cross-Reactivity In Silico Analysis

For in silico exclusivity assessment of the assay oligos against on-panel and off-panel organisms listed in Table 8 below, based on analysis of sequences available in the GenBank® Nucleotide (nt) database as of April 9, 2023, the following potential cross-reactivity is predicted:

SARS-CoV-2 oligos are likely to detect some SARS-related coronavirus strains, as well as some bat coronavirus and bat SARS-like coronavirus strains. One bat 229E-like coronavirus sequence (KT253270) is likely to be detected by the Coronavirus 229E oligos at high viral titer.

Table 8. Potential Cross-Reactive Organisms assessed in the In Silico Exclusivity Analysis   

<table><tr><td>On-Panel Organisms</td><td colspan="2">Off-Panel Organisms</td></tr><tr><td>Adenovirus</td><td>Bat SARS-like Coronavirus Bat SARS-like Coronavirus</td><td>Chlamydia psittaci</td></tr><tr><td>Enterovirus</td><td>HKU5</td><td>Coxiella burnetii</td></tr><tr><td>Human coronavirus 229E</td><td>Herpes Simplex Virus 2 (HSV2)</td><td>Cryptococcus neoformans</td></tr><tr><td>Human coronavirus HKU1</td><td>Human Bocavirus</td><td>Fusobacterium necrophorum</td></tr><tr><td>Human coronavirus NL63</td><td>Human Herpes Virus 6 (HHV6)</td><td>Haemophilus influenza</td></tr><tr><td>Human coronavirus OC43</td><td>Human Parechovirus (HPeV)</td><td>Histoplasma capsulatum</td></tr><tr><td>Human metapneumovirus (hMPV)</td><td>Influenza C</td><td>Klebsiella (Enterobacter) aerogenes</td></tr><tr><td>Influenza A</td><td>MERS-coronavirus</td><td>Klebsiella oxytoca</td></tr><tr><td>Influenza B</td><td>SARS-coronavirus</td><td>Legionella pneumophila</td></tr><tr><td>Parainfluenza virus 1</td><td>Acinetobacter calcoaceticus</td><td>Leptospira interrogans</td></tr><tr><td>Parainfluenza virus 2</td><td>Arcanobacterium haemolyticum</td><td>Mycobacterium tuberculosis</td></tr><tr><td>Parainfluenza virus 3</td><td>Aspergillus fumigatus</td><td>Mycoplasma orale</td></tr><tr><td>Parainfluenza virus 4</td><td>Aspergillus flavus</td><td>Pneumocystis jirovecii (PJP)</td></tr><tr><td>Respiratory syncytial virus A</td><td>Bacillus anthracis</td><td>Proteus mirabilis</td></tr><tr><td>Respiratory syncytial virus B</td><td>Blastomyces dennatitidis</td><td>Pseudomonas aeruginosa</td></tr><tr><td>Rhinovirus</td><td>Bordetella avium</td><td>Staphylococcus epidermidis</td></tr><tr><td>SARS-CoV-2</td><td>Bordetella bronchiseptica</td><td>Stenotrophomonas maltophilia</td></tr><tr><td>Chlamydia pneumoniae</td><td>Bordetella hinzii</td><td>Streptococcus dysgalactiae</td></tr><tr><td>Mycoplasma pneumoniae</td><td>Bordetella holmesii</td><td></td></tr><tr><td></td><td></td><td>Streptococcus pneumonia</td></tr><tr><td></td><td>Bordetella pertussis</td><td>Streptococcus pyogenes</td></tr><tr><td></td><td>Burkholderia cepacia</td><td>Streptococcus salivarius</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Candida albicans</td><td colspan="1" rowspan="1">Ureaplasma urealyticum</td></tr></table>

# ii) Microbial Interference

The performance of NxTAG® RPP v2 in the presence of potentially interfering offpanel organisms (i.e., pathogens that are not detected by $\mathsf { N x T A G } ^ { \otimes }$ RPP v2) was evaluated by testing 11 off-panel organisms against 22 on-panel organisms detected by the assay. The samples were prepared in negative simulated matrix (NSM), with on-panel targets in multi-analyte samples at Low-Moderate Positive concentration (3x-6x LoD) and pools of off-panel organisms at the high positive concentration. Each combination was tested in triplicate on the NxTAG® RPP v2 assay. No off-panel organism present at high positive concentration interfered with the detection of any on-panel organism present at low positive concentration. The summary of results is shown in Table 9.

Table 9: NxTAG® RPP v2 Microbial Interference   

<table><tr><td rowspan=1 colspan=1>Off-Panel Organisms</td><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>Concentration Tested</td><td rowspan=1 colspan=1>MicrobialInterferenceDetected</td></tr><tr><td rowspan=1 colspan=1>Bordetella pertussis</td><td rowspan=1 colspan=1>A639</td><td rowspan=1 colspan=1>1.00E+06CFU/mL</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Cytomegalovirus</td><td rowspan=1 colspan=1>Merlin</td><td rowspan=1 colspan=1>1.00E+05TCID5o/mL</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Corynebacterium diphtheriae</td><td rowspan=1 colspan=1>Z116</td><td rowspan=1 colspan=1>1.00E+06CFU/mL</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Haemophilus influenzae</td><td rowspan=1 colspan=1>Type b; MinnA</td><td rowspan=1 colspan=1>1.00E+06CFU/mL</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Measles Virus</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>1.00E+05TCID50/mL</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Moraxella catarrhalis</td><td rowspan=1 colspan=1>Strain NE 11</td><td rowspan=1 colspan=1>1.00E+06CFU/mL</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Mumps Virus</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>1.00E+05TCID5o/mL</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Neisseria meningitides</td><td rowspan=1 colspan=1>Serotype A</td><td rowspan=1 colspan=1>1.00E+06CFU/mL</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Pseudomonas aeruginosa</td><td rowspan=1 colspan=1>Clinical isolate</td><td rowspan=1 colspan=1>1.00E+06CFU/mL</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus aureus</td><td rowspan=1 colspan=1>102-04</td><td rowspan=1 colspan=1>1.00E+06CFU/mL</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Streptococcus pneumoniae</td><td rowspan=1 colspan=1>Z022</td><td rowspan=1 colspan=1>1.00E+06CFU/mL</td><td rowspan=1 colspan=1>None</td></tr></table>

# iii) Competitive Interference (Co-infection)

The performance of $\mathsf { N } \times \mathsf { T } \mathsf { A G } ^ { \otimes }$ RPP v2 assay in the presence of potentially interfering on-panel organisms (i.e., pathogens that are detected by $\mathsf { N } \times \mathsf { T } \mathsf { A G } ^ { \textregistered }$ RPP v2) was

# Luminex

evaluated by testing samples that contain multiple on-panel targets, which included clinically relevant co-infections that occur in respiratory samples. The samples were prepared in negative simulated matrix (NSM), with on-panel organisms (in either single-analyte or multi-analyte samples) at Low-Moderate Positive concentration (3x-6x LoD), and the potentially interfering on-panel organisms at high positive concentration. Each combination was tested in triplicate on the $\mathsf { N x T A G } ^ { \otimes }$ RPP v2 assay. No on-panel organism present at high positive concentration interfered with the detection of any other on-panel organism present at low-moderate positive concentration. The summary of results is shown in Table 10.

Table 10: NxTAG® RPP v2 Competitive Interference   

<table><tr><td rowspan=1 colspan=3>High Positive</td><td rowspan=1 colspan=1>Low-Moderate Positive</td><td rowspan=2 colspan=1>CompetitiveInterferenceDetected</td></tr><tr><td rowspan=1 colspan=1>Target</td><td rowspan=1 colspan=2>Concentration</td><td rowspan=1 colspan=1>Target</td></tr><tr><td rowspan=10 colspan=1>Influenza A H1pdm09(strain: A/NY/02/09)</td><td rowspan=10 colspan=2>1.00E+05TCID5o/mL</td><td rowspan=1 colspan=1>Respiratory Syncytial Virus A</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Rhinovirus</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Respiratory Syncytial Virus B</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Influenza A H3</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>Influenza B</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Parainfluenza virus 3</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Mycoplasma pneumoniae</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>SARS-CoV-2</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Human Metapneumovirus</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=10 colspan=1>Influenza A H3(Strain:Wisconsin/67/05)</td><td rowspan=10 colspan=2>1.00E+05TCID50/mL</td><td rowspan=1 colspan=1>Influenza A H1pdm09</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Respiratory Syncytial Virus A</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Rhinovirus</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Respiratory Syncytial Virus B</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Influenza B</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza virus 3</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Mycoplasma pneumoniae</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>SARS-CoV-2</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Human Metapneumovirus</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Influenza B</td><td rowspan=1 colspan=2>1.00E+05</td><td rowspan=1 colspan=1>Influenza A H1pdm09</td><td rowspan=1 colspan=1>None</td></tr></table>

<table><tr><td colspan="2" rowspan="1">High Positive</td><td colspan="1" rowspan="1">Low-Moderate Positive</td><td colspan="1" rowspan="2">CompetitiveInterferenceDetected</td></tr><tr><td colspan="1" rowspan="1">Target</td><td colspan="1" rowspan="1">Concentration</td><td colspan="1" rowspan="1">Target</td></tr><tr><td colspan="1" rowspan="9">(Strain: Florida/02/06)</td><td colspan="1" rowspan="9">TCID50/mL</td><td colspan="1" rowspan="1">Respiratory Syncytial Virus A</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Rhinovirus</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Respiratory Syncytial Virus B</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Influenza A H3</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza virus 3</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Mycoplasma pneumoniae</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">SARS-CoV-2</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Human Metapneumovirus</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Adenovirus</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="10">Respiratory SyncytialVirus B(Strain: B/18537)</td><td colspan="1" rowspan="10">6.30E+04aPFU/mL</td><td colspan="1" rowspan="1">Influenza A H1pdm09</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Respiratory Syncytial Virus A</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Rhinovirus</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Influenza A H3</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Influenza B</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza virus 3</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Mycoplasma pneumoniae</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">SARS-CoV-2</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Human Metapneumovirus</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Adenovirus</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="10">Respiratory SyncytialVirus B(Strain:B/WV/14617/85)</td><td colspan="1" rowspan="10">1.00E+05TCID50/mL</td><td colspan="1" rowspan="1">Influenza A H1pdm09</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Respiratory Syncytial Virus A</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Rhinovirus</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Influenza A H3</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Influenza B</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza virus 3</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Mycoplasma pneumoniae</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">SARS-CoV-2</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Human Metapneumovirus</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Adenovirus</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="2">SARS-CoV-2(Strain: USA-WA-</td><td colspan="1" rowspan="2">1.00E+07Copies/mL</td><td colspan="1" rowspan="1">Influenza A H1pdm09</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Respiratory Syncytial Virus A</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="8">1/2020)</td><td colspan="1" rowspan="8"></td><td colspan="1" rowspan="1">Rhinovirus</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Respiratory Syncytial Virus B</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Influenza A H3</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Influenza B</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza virus 3</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Mycoplasma pneumoniae</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Human Metapneumovirus</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Adenovirus</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="11">Coronavirus NL63(Strain: NL63)</td><td colspan="1" rowspan="11">1.00E+05TCID50/mL</td><td colspan="1" rowspan="1">Influenza A H1pdm09</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Respiratory Syncytial Virus A</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Rhinovirus</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Respiratory Syncytial Virus B</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Influenza A H3</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Influenza B</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza virus 3</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Mycoplasma pneumoniae</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">SARS-CoV-2</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Human Metapneumovirus</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Adenovirus</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="11">HumanMetapneumovirus(Strain: hMPV-16, TypeA1, IA10-2003)</td><td colspan="1" rowspan="11">1.00E+05TCID50/mL</td><td colspan="1" rowspan="1">Influenza A H1pdm09</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Respiratory Syncytial Virus A</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Rhinovirus</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Respiratory Syncytial Virus B</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Influenza A H3</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Influenza B</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza virus 3</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Mycoplasma pneumoniae</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">SARS-CoV-2</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Human Metapneumovirus</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Adenovirus</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Human</td><td colspan="1" rowspan="1">3.50E+04a</td><td colspan="1" rowspan="1">Influenza A H1pdm09</td><td colspan="1" rowspan="1">None</td></tr></table>

Confidential & Restricted

<table><tr><td colspan="2" rowspan="1">High Positive</td><td colspan="1" rowspan="1">Low-Moderate Positive</td><td colspan="1" rowspan="2">CompetitiveInterferenceDetected</td></tr><tr><td colspan="1" rowspan="1">Target</td><td colspan="1" rowspan="1">Concentration</td><td colspan="1" rowspan="1">Target</td></tr><tr><td colspan="1" rowspan="9">Metapneumovirus(Strain: hMPV-3, TypeB1, Peru2-2002)</td><td colspan="1" rowspan="9">TCID5o/mL</td><td colspan="1" rowspan="1">Respiratory Syncytial Virus A</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Rhinovirus</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Respiratory Syncytial Virus B</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Influenza A H3</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Influenza B</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza virus 3</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Mycoplasma pneumoniae</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">SARS-CoV-2</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Adenovirus</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="10">HumanMetapneumovirus(Strain: hMPV-5, TypeB1, Peru3-2003 G gene)</td><td colspan="1" rowspan="10">1.00E+05TCID5o/mL</td><td colspan="1" rowspan="1">Influenza A H1pdm09</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Respiratory Syncytial Virus A</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Rhinovirus</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Respiratory Syncytial Virus B</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Influenza A H3</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Influenza B</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza virus 3</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Mycoplasma pneumoniae</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">SARS-CoV-2</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Adenovirus</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="10">Rhinovirus(Strain: Type 85, 50-525-CV54 [V-192-001-021)</td><td colspan="1" rowspan="10">1.00E+05TCID50/mL</td><td colspan="1" rowspan="1">Influenza A H1pdm09</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Respiratory Syncytial Virus A</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Respiratory Syncytial Virus B</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Influenza A H3</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Influenza B</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza virus 3</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Mycoplasma pneumoniae</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">SARS-CoV-2</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Human Metapneumovirus</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Adenovirus</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Adenovirus B</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Influenza A H1pdm09</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="9">(Strain: Type 14, 2006isolate)</td><td colspan="1" rowspan="9">1.00E+05TCID50/mL</td><td colspan="1" rowspan="1">Respiratory Syncytial Virus A</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Rhinovirus</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Respiratory Syncytial Virus B</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Influenza A H3</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Influenza B</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza virus 3</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Mycoplasma pneumoniae</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">SARS-CoV-2</td><td colspan="1" rowspan="1">None</td></tr><tr><td colspan="1" rowspan="1">Human Metapneumovirus</td><td colspan="1" rowspan="1">None</td></tr></table>

a The highest possible concentration was tested.

Additional competitive interference was evaluated for SARS-CoV-2. Low Positive SARS-CoV-2 (3x LoD) was tested against the remaining $1 3 ~ { \mathsf { N X T A G } } ^ { \otimes }$ RPP v2 assay targets that were not tested with the SARS-CoV-2 containing multi-analyte combinations. High Positive targets were tested at $1 . 0 0 \mathsf { E } \mathsf { + } 0 6 \mathsf { C C U / m L }$ or IFU/mL for bacteria; $1 . 0 0 \mathsf { E } + 0 5 \mathsf { T C I D } _ { 5 0 } / \mathsf { m L } , \mathsf { P F U } / \mathsf { m L }$ , or $1 . 0 0 { \mathsf { E } } { + } 0 7 $ Copies/mL, or highest possible concentration for virus. Samples were prepared in negative simulated matrix and each combination was tested in triplicate with the assay. No interference of SARSCoV-2 low positive detection was observed (Table 11).

Table 11: NxTAG® RPP v2 Competitive Interference Study – Additional Tests Performed with SARS-CoV-2   

<table><tr><td rowspan=1 colspan=2>High Positive</td><td rowspan=1 colspan=5>Low Positive</td><td rowspan=2 colspan=1>CompetitiveInterferenceDetected</td></tr><tr><td rowspan=1 colspan=1>Target</td><td rowspan=1 colspan=1>Concentration</td><td rowspan=1 colspan=5>Target</td></tr><tr><td rowspan=1 colspan=1>Influenza A H1</td><td rowspan=1 colspan=1>1.00E+05 TCID50/mL</td><td rowspan=3 colspan=5>SARS-CoV-2</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Respiratory Syncytial Virus A</td><td rowspan=1 colspan=1>1.00E+05PFU/mL</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza virus 1</td><td rowspan=1 colspan=1>1.00E+05TCID50/mL</td><td rowspan=1 colspan=3></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza virus 2</td><td rowspan=1 colspan=1>1.00E+05TCID50/mL</td><td rowspan=2 colspan=5></td><td rowspan=1 colspan=1>−</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza virus 3</td><td rowspan=1 colspan=1>1.00E+05 TCID50/mL</td><td rowspan=1 colspan=1>None</td></tr></table>

Confidential & Restricted

<table><tr><td rowspan=1 colspan=1>Parainfluenza virus 4Aa</td><td rowspan=1 colspan=1>1.00E+05 TCID50/mL</td><td rowspan=8 colspan=1>b</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza virus 4Ba</td><td rowspan=1 colspan=1>1.00E+05 TCID50/mL</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Coronavirus 229E</td><td rowspan=1 colspan=1>1.00E+05 TCID5o/mL</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Coronavirus OC43</td><td rowspan=1 colspan=1>1.00E+05 TCID50/mL</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Coronavirus HKU1</td><td rowspan=1 colspan=1>1.00E+06bCopies/mL</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Enterovirus</td><td rowspan=1 colspan=1>1.00E+05 TCID50/mL</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Chlamydia pneumoniae</td><td rowspan=1 colspan=1>1.00E+06 IFU/mL</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Mycoplasma pneumoniae</td><td rowspan=1 colspan=1>1.00E+06 CCU/mL</td><td rowspan=1 colspan=1>None</td></tr></table>

a Reported by NxTAG RPP as Parainfluenza virus 4 b The highest possible concentration was tested c Reported by NxTAG RPP v2 as Rhinovirus/Enterovirus

# iv) Interfering Substances:

The performance of $\mathsf { N } \times \mathsf { T } \mathsf { A G } ^ { \textregistered }$ RPP v2 in the presence of potentially interfering substances was assessed. Twenty non-microbial substances commonly found in respiratory specimens were tested on the assay alone or in the presence of pathogens detected by the assay in multi-analyte samples. The samples were prepared in negative simulated matrix (NSM), with the on-panel organisms at LowModerate Positive concentration (3x-6x LoD) and the potentially interfering substance at the concentration listed in Table 14. All samples were tested in triplicate on the $\mathsf { N x T A G } ^ { \otimes }$ RPP v2 assay. None of the substances tested interfered with the detection of on-panel organisms present in the sample, with the exception of menthol and FluMist®. Menthol interfered with the detection of Coronavirus OC43 at $1 \% \ \mathsf { w } / \mathsf { v } ;$ ; no interference was observed when menthol was tested at $0 . 5 \% \ \mathsf { w } / \mathsf { v }$ . FluMist® generated positive calls for Influenza A (matrix), Influenza A H1pdm09, Influenza A H3, and Influenza B for all replicates it was present in. These positive calls are expected as FluMist® contains attenuated Influenza A H1N1, Influenza A H3N2, and Influenza B strains. Positive influenza results obtained in a patient who received FluMist® prior to sample collection may be due to detection of vaccine virus and may mask a true positive result due to infection by one or more of these analytes. The list of potentially interfering substances and the concentrations tested are shown in Table 12.

Table 12: NxTAG® RPP v2 Interfering Substances   

<table><tr><td rowspan=1 colspan=1>Potential Interferent</td><td rowspan=1 colspan=1>Active Ingredient</td><td rowspan=1 colspan=1>Concentration Tested</td></tr><tr><td rowspan=1 colspan=1>Human Whole Blood</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>5% (v/v)</td></tr></table>

Confidential & Restricted

NxTAG® Respiratory Pathogen Panel v2 Traditional 510(k) Submission   

<table><tr><td rowspan=1 colspan=1>Potential Interferent</td><td rowspan=1 colspan=1>Active Ingredient</td><td rowspan=1 colspan=1>Concentration Tested</td></tr><tr><td rowspan=1 colspan=1>Human Genomic DNA</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>20 ng/μL</td></tr><tr><td rowspan=1 colspan=1>Mucin</td><td rowspan=1 colspan=1>Mucin</td><td rowspan=1 colspan=1>100 μg/mL</td></tr><tr><td rowspan=1 colspan=1>Phenylephrine</td><td rowspan=1 colspan=1>Phenylephrine</td><td rowspan=1 colspan=1>0.03 μg/mL</td></tr><tr><td rowspan=1 colspan=1>Beclomethasonedipropionate</td><td rowspan=1 colspan=1>Beclomethasonedipropionate</td><td rowspan=1 colspan=1>25 μg/mL</td></tr><tr><td rowspan=1 colspan=1>Dexamethasone</td><td rowspan=1 colspan=1>Dexamethasone</td><td rowspan=1 colspan=1>12 μg/mL</td></tr><tr><td rowspan=1 colspan=1>Flunisolide</td><td rowspan=1 colspan=1>Flunisolide</td><td rowspan=1 colspan=1>5 μg/mL</td></tr><tr><td rowspan=1 colspan=1>Triamcinolone acetonide</td><td rowspan=1 colspan=1>Triamcinolone acetonide</td><td rowspan=1 colspan=1>22 μg/mL</td></tr><tr><td rowspan=1 colspan=1>Budesonide</td><td rowspan=1 colspan=1>Budesonide</td><td rowspan=1 colspan=1>6.30E-03 μg/mL</td></tr><tr><td rowspan=1 colspan=1>Mometasone furoate</td><td rowspan=1 colspan=1>Mometasone furoate</td><td rowspan=1 colspan=1>4.50E-04 μg/mL</td></tr><tr><td rowspan=1 colspan=1>Fluticasone proprionate</td><td rowspan=1 colspan=1>Fluticasone</td><td rowspan=1 colspan=1>1.26E-03 μg/mL</td></tr><tr><td rowspan=2 colspan=1>Drixoral</td><td rowspan=2 colspan=1>Oxymetaxoline</td><td rowspan=1 colspan=1>10% (v/v)</td></tr><tr><td rowspan=1 colspan=1>15% (v/v)</td></tr><tr><td rowspan=2 colspan=1>ZICAM® Allergy Relief</td><td rowspan=2 colspan=1>Galphimia GlaucaHistaminumHydrochloricumLuffa operculataSulfur</td><td rowspan=1 colspan=1>1% (v/v)</td></tr><tr><td rowspan=1 colspan=1>5% (v/v)</td></tr><tr><td rowspan=2 colspan=1>Salinex®</td><td rowspan=2 colspan=1>Sodium Chloride</td><td rowspan=1 colspan=1>1% (v/v)</td></tr><tr><td rowspan=1 colspan=1>15% (v/v)</td></tr><tr><td rowspan=1 colspan=1>Mupirocin</td><td rowspan=1 colspan=1>Mupirocin</td><td rowspan=1 colspan=1>1.5 μg/mL</td></tr><tr><td rowspan=2 colspan=1>Tobramycin</td><td rowspan=2 colspan=1>Tobramycin</td><td rowspan=1 colspan=1>33 μg/mL</td></tr><tr><td rowspan=1 colspan=1>600 μg/mL</td></tr><tr><td rowspan=1 colspan=1>Zanamivir</td><td rowspan=1 colspan=1>Zanamivir</td><td rowspan=1 colspan=1>100 μg/mL</td></tr><tr><td rowspan=1 colspan=1>FluMist®</td><td rowspan=1 colspan=1>Influenza A H1N1,Influenza A H3N2,Influenza B Yamagatalineage, Influenza B VictoriaLineage</td><td rowspan=1 colspan=1>0.5% (v/v) a</td></tr><tr><td rowspan=1 colspan=1>Benzocaine</td><td rowspan=1 colspan=1>Benzocaine</td><td rowspan=1 colspan=1>10% (w/v)</td></tr><tr><td rowspan=2 colspan=1>Menthol</td><td rowspan=2 colspan=1>Menthol</td><td rowspan=1 colspan=1>1%(w/v)b</td></tr><tr><td rowspan=1 colspan=1>0.5% (w/v)</td></tr><tr><td rowspan=1 colspan=1>Leukocyte</td><td rowspan=1 colspan=1>Leukocyte</td><td rowspan=1 colspan=1>1.00E+03 cells/μL</td></tr><tr><td rowspan=1 colspan=1>Early Defence Nasal Spray®</td><td rowspan=1 colspan=1>Zinc</td><td rowspan=1 colspan=1>5% (v/v)</td></tr></table>

a FluMist® demonstrated interference with the detection of Influenza A (matrix), Influenza A H1pdm09, Influenza A H3, and Influenza B at $0 . 5 \% \lor / \lor$ . b Menthol demonstrated interference with the detection of Coronavirus OC43 at $1 \% \mathrm { ~ w / v } ;$ no interference was observed at $0 . 5 \% \boldsymbol { \mathrm { w } } / \boldsymbol { \mathrm { v } }$ .

# v) Carry-Over/Cross-Contamination:

# Luminex

The risk of carry-over and cross-contamination events for the NxTAG® RPP v2 assay was assessed by testing samples at high titer in alternating pattern with negative samples. Two representative targets, SARS-CoV-2 (viral) and Mycoplasma pneumoniae (bacterial), were each extracted in an alternating arrangement with the negative sample across two easyMAG instruments. The extracts were then tested in an alternating arrangement with the NxTAG® RPP v2 assay. Two false positives were observed for the SARS-CoV-2 target; one after a negative sample and the other after a high titer sample. The samples were re-run to determine whether the extracts were contaminated, and the re-run results confirmed the contamination of the extracts. Following this, the samples were re-prepared and re-tested, generating the expected results.

# h. Assay cut-off

Thresholds for the NxTAG® RPP v2 assay are provided in Table 13.

Table 13. MFI and MDD Thresholds (Cut-Off Values) for NxTAG® RPP v2   

<table><tr><td rowspan=1 colspan=1>Analyte</td><td rowspan=1 colspan=1>MFI Threshold</td><td rowspan=1 colspan=1>MDD Threshold</td></tr><tr><td rowspan=1 colspan=1>Influenza A</td><td rowspan=1 colspan=1>45</td><td rowspan=1 colspan=1>35</td></tr><tr><td rowspan=1 colspan=1>Influenza A H1 (H1-A)</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>75</td></tr><tr><td rowspan=1 colspan=1>Influenza A H1pdm09 (H1-B)</td><td rowspan=1 colspan=1>55</td><td rowspan=1 colspan=1>45</td></tr><tr><td rowspan=1 colspan=1>Influenza A H3</td><td rowspan=1 colspan=1>80</td><td rowspan=1 colspan=1>50</td></tr><tr><td rowspan=1 colspan=1>Influenza B</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>40</td></tr><tr><td rowspan=1 colspan=1>RSV A</td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>45</td></tr><tr><td rowspan=1 colspan=1>RSV B</td><td rowspan=1 colspan=1>45</td><td rowspan=1 colspan=1>35</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza 1</td><td rowspan=1 colspan=1>65</td><td rowspan=1 colspan=1>50</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza 2</td><td rowspan=1 colspan=1>70</td><td rowspan=1 colspan=1>55</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza 3</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>50</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza 4A a</td><td rowspan=1 colspan=1>70</td><td rowspan=1 colspan=1>55</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza 4B a</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>45</td></tr><tr><td rowspan=1 colspan=1>SARS-CoV-2 (ORF1ab) b</td><td rowspan=1 colspan=1>35</td><td rowspan=1 colspan=1>30</td></tr><tr><td rowspan=1 colspan=1>SARS-CoV-2 (M) b</td><td rowspan=1 colspan=1>35</td><td rowspan=1 colspan=1>30</td></tr><tr><td rowspan=1 colspan=1>Coronavirus 229E</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>50</td></tr><tr><td rowspan=1 colspan=1>Coronavirus NL63</td><td rowspan=1 colspan=1>75</td><td rowspan=1 colspan=1>60</td></tr><tr><td rowspan=1 colspan=1>CoronavirusOC43</td><td rowspan=1 colspan=1>70</td><td rowspan=1 colspan=1>55</td></tr><tr><td rowspan=1 colspan=1>Coronavirus HKU1</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>40</td></tr><tr><td rowspan=1 colspan=1>Metapneumovirus</td><td rowspan=1 colspan=1>55</td><td rowspan=1 colspan=1>40</td></tr><tr><td rowspan=1 colspan=1>Rhinovirus/Enterovirus</td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>40</td></tr><tr><td rowspan=1 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>75</td><td rowspan=1 colspan=1>65</td></tr><tr><td rowspan=1 colspan=1>Chlamydia pneumoniae</td><td rowspan=1 colspan=1>45</td><td rowspan=1 colspan=1>40</td></tr></table>

Confidential & Restricted

<table><tr><td rowspan=1 colspan=1>Analyte</td><td rowspan=1 colspan=1>MFI Threshold</td><td rowspan=1 colspan=1>MDD Threshold</td></tr><tr><td rowspan=1 colspan=1>Mycoplasma pneumoniae</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>30</td></tr><tr><td rowspan=1 colspan=1>Internal Control</td><td rowspan=1 colspan=1>120</td><td rowspan=1 colspan=1>100</td></tr></table>

a NxTAG RPP v2 does not differentiate Parainfluenza virus 4A and 4B, both of which are reported as Parainfluenza virus 4 b NxTAG RPP v2 reports a single combined result for the SARS-CoV-2 ORF1ab and M gene targets

# 2. Comparison Studies:

a. Method comparison with predicate device: Not applicable.

b. Matrix and Multi-Analyte Sample Comparison:

Equivalency between multi-analyte (MA) and single target samples, and NCM and negative simulated matrix (NSM) were assessed to demonstrate the validity of using MA samples and/or NSM in applicable analytical studies. Eight (8) MA samples, covering all targets probed by NxTAG® RPP v2, were evaluated. Each MA sample consisted of 2-3 targets and was prepared in both NCM and NSM at or near the LoD concentration $( 1 \mathsf { x } - 2 \mathsf { x } \mathsf { L o D } )$ confirmed with single target samples in NCM. Twenty (20) replicates of each MA were tested, and all targets in MA samples generated $\geq 9 5 \%$ positivity. The results demonstrate equivalency between single-analyte and multi-analyte samples as well as between NCM and NSM.

The multi-analyte sample composition and the equivalency test results are shown in Table 14.

Table 14: NxTAG® RPP v2 Multi-Analyte Vs. Single Analyte, and Negative Clinical Matrix Vs. Negative Simulated Matrix Equivalency   

<table><tr><td rowspan=2 colspan=1>Multi-analyteSample</td><td rowspan=2 colspan=1>Organism</td><td rowspan=2 colspan=1>Confirmed LoD(Copies/mL)</td><td rowspan=1 colspan=2>Positivity (%)</td></tr><tr><td rowspan=1 colspan=1>MA Sample inNCM</td><td rowspan=1 colspan=1>MA Sample inNSM</td></tr><tr><td rowspan=3 colspan=1>MA-1</td><td rowspan=1 colspan=1>Influenza A 2009 H1N1 (subtype)</td><td rowspan=1 colspan=1>9.84E+02</td><td rowspan=1 colspan=1>20/20 (100%)</td><td rowspan=1 colspan=1>20/20 (100%)</td></tr><tr><td rowspan=1 colspan=1>Respiratory Syncytial Virus A</td><td rowspan=1 colspan=1>4.97E+03</td><td rowspan=1 colspan=1>19/20 (95%)</td><td rowspan=1 colspan=1>20/20 (100%)</td></tr><tr><td rowspan=1 colspan=1>Rhinovirus</td><td rowspan=1 colspan=1>1.54E+03</td><td rowspan=1 colspan=1>20/20 (100%)</td><td rowspan=1 colspan=1>20/20 (100%)</td></tr><tr><td rowspan=2 colspan=1>MA-2</td><td rowspan=1 colspan=1>Influenza A H3 (subtype)</td><td rowspan=1 colspan=1>5.60E+01</td><td rowspan=1 colspan=1>20/20 (100%)</td><td rowspan=1 colspan=1>20/20 (100%)</td></tr><tr><td rowspan=1 colspan=1>Respiratory Syncytial Virus B</td><td rowspan=1 colspan=1>7.21E+03</td><td rowspan=1 colspan=1>20/20 (100%)</td><td rowspan=1 colspan=1>20/20 (100%)</td></tr></table>

Confidential & Restricted

<table><tr><td rowspan=2 colspan=1>Multi-analyteSample</td><td rowspan=2 colspan=1>Organism</td><td rowspan=2 colspan=1>Confirmed LoD(Copies/mL)</td><td rowspan=1 colspan=2>Positivity (%)</td></tr><tr><td rowspan=1 colspan=1>MA Sample inNCM</td><td rowspan=1 colspan=1>MA Sample inNSM</td></tr><tr><td rowspan=3 colspan=1>MA-3</td><td rowspan=1 colspan=1>Influenza B</td><td rowspan=1 colspan=1>6.33E+01</td><td rowspan=1 colspan=1>20/20 (100%)</td><td rowspan=1 colspan=1>19/20 (95%)</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza virus 3</td><td rowspan=1 colspan=1>1.01E+03</td><td rowspan=1 colspan=1>20/20 (100%)</td><td rowspan=1 colspan=1>20/20 (100%)</td></tr><tr><td rowspan=1 colspan=1>Mycoplasma pneumoniae</td><td rowspan=1 colspan=1>3.23E+03</td><td rowspan=1 colspan=1>20/20 (100%)</td><td rowspan=1 colspan=1>20/20 (100%)</td></tr><tr><td rowspan=3 colspan=1>MA-4</td><td rowspan=1 colspan=1>SARS-CoV-2</td><td rowspan=1 colspan=1>5.00E+02</td><td rowspan=1 colspan=1>20/20 (100%)</td><td rowspan=1 colspan=1>20/20 (100%)</td></tr><tr><td rowspan=1 colspan=1>Human Metapneumovirus</td><td rowspan=1 colspan=1>2.62E+02</td><td rowspan=1 colspan=1>20/20 (100%)</td><td rowspan=1 colspan=1>19/20 (95%)</td></tr><tr><td rowspan=1 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>1.42E+03</td><td rowspan=1 colspan=1>20/20 (100%)</td><td rowspan=1 colspan=1>20/20 (100%)</td></tr><tr><td rowspan=3 colspan=1>MA-5</td><td rowspan=1 colspan=1>Influenza A H3 (Matrix)</td><td rowspan=1 colspan=1>1.68E+02</td><td rowspan=1 colspan=1>20/20 (100%)</td><td rowspan=1 colspan=1>20/20 (100%)</td></tr><tr><td rowspan=1 colspan=1>Coronavirus NL63</td><td rowspan=1 colspan=1>1.00E+02</td><td rowspan=1 colspan=1>20/20 (100%)</td><td rowspan=1 colspan=1>20/20 (100%)</td></tr><tr><td rowspan=1 colspan=1>Coronavirus HKU1</td><td rowspan=1 colspan=1>4.18E+03</td><td rowspan=1 colspan=1>20/20 (100%)</td><td rowspan=1 colspan=1>19/20 (95%)</td></tr><tr><td rowspan=3 colspan=1>MA-6</td><td rowspan=1 colspan=1>Influenza H1 (subtype)</td><td rowspan=1 colspan=1>1.60E+03</td><td rowspan=1 colspan=1>20/20 (100%)</td><td rowspan=1 colspan=1>19/20 (95%)</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza virus 1</td><td rowspan=1 colspan=1>6.92E+02</td><td rowspan=1 colspan=1>20/20 (100%)</td><td rowspan=1 colspan=1>20/20 (100%)</td></tr><tr><td rowspan=1 colspan=1>Chlamydia pneumoniae</td><td rowspan=1 colspan=1>2.38E+02</td><td rowspan=1 colspan=1>20/20 (100%)</td><td rowspan=1 colspan=1>20/20 (100%)</td></tr><tr><td rowspan=3 colspan=1>MA-7</td><td rowspan=1 colspan=1>Parainfluenza virus 2</td><td rowspan=1 colspan=1>3.45E+02</td><td rowspan=1 colspan=1>19/20 (95%)</td><td rowspan=1 colspan=1>19/20 (95%)</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza virus 4B</td><td rowspan=1 colspan=1>7.15E+03</td><td rowspan=1 colspan=1>20/20 (100%)</td><td rowspan=1 colspan=1>20/20 (100%)</td></tr><tr><td rowspan=1 colspan=1>Coronavirus 229E</td><td rowspan=1 colspan=1>3.81E+02</td><td rowspan=1 colspan=1>20/20 (100%)</td><td rowspan=1 colspan=1>19/20 (95%)</td></tr><tr><td rowspan=2 colspan=1>MA-8</td><td rowspan=1 colspan=1>Parainfluenza virus 4A</td><td rowspan=1 colspan=1>1.69E+04</td><td rowspan=1 colspan=1>20/20 (100%)</td><td rowspan=1 colspan=1>20/20 (100%)</td></tr><tr><td rowspan=1 colspan=1>Coronavirus OC43</td><td rowspan=1 colspan=1>4.55E+03</td><td rowspan=1 colspan=1>20/20 (100%)</td><td rowspan=1 colspan=1>20/20 (100%)</td></tr></table>

# c. Specimen Collection Device Comparison:

The equivalency between three swab types (swabs with a nylon flocked tip, polyester tip, and rayon tip) for the collection of samples for testing on the NxTAG® RPP v2 was evaluated. The study assessed the transfer of samples from the swab to the collection media. Four multi-analyte (MA) samples consisting of targets detected by the NxTAG® RPP v2 assay were prepared in negative simulated matrix (NSM) and tested at a final concentration of 3x-6x LoD, along with a negative sample.

Samples prepared using swabs with a nylon flocked tip and a polyester tip generated $100 \%$ positivity for all targets present in the respective multi-analyte samples and $0 \%$ positivity for all targets for the negative sample.

# Luminex

Samples prepared using swabs with a rayon tip only generated $100 \%$ positivity for all targets present in the MA1 and MA2 samples, and $0 \%$ positivity for all targets for the negative sample. For the MA3 sample, two of three replicates generated negative results for all targets (Influenza B, Parainfluenza virus 3, and M. pneumoniae) and for the MA4 sample, one of three replicates generated a negative result for Adenovirus.

The results demonstrate equivalency between swabs with nylon flocked tip and polyester tip (Table 15). Based on the results of this study, rayon-tipped swabs should not be used to collect specimens for use with the NxTAG RPP v2 assay.

Table 15: NxTAG® RPP v2 Assay Swab Equivalency   

<table><tr><td rowspan=2 colspan=1>Sample</td><td rowspan=2 colspan=1>Organism</td><td rowspan=1 colspan=3>Target Positivity</td></tr><tr><td rowspan=1 colspan=1>Nylon Flocked</td><td rowspan=1 colspan=1>Polyester</td><td rowspan=1 colspan=1>Rayon</td></tr><tr><td rowspan=3 colspan=1>MA1</td><td rowspan=1 colspan=1>Influenza A 2009 H1N1 (subtype)</td><td rowspan=1 colspan=1>100% (3/3)</td><td rowspan=1 colspan=1>100% (3/3)</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>Respiratory Syncytial Virus A</td><td rowspan=1 colspan=1>100% (3/3)</td><td rowspan=1 colspan=1>100% (3/3)</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>Rhinovirus</td><td rowspan=1 colspan=1>100% (3/3)</td><td rowspan=1 colspan=1>100% (3/3)</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=2 colspan=1>MA2</td><td rowspan=1 colspan=1>Influenza A H3 (subtype)</td><td rowspan=1 colspan=1>100% (3/3)</td><td rowspan=1 colspan=1>100% (3/3)</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>Respiratory Syncytial Virus B</td><td rowspan=1 colspan=1>100% (3/3)</td><td rowspan=1 colspan=1>100% (3/3)</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=3 colspan=1>MA3</td><td rowspan=1 colspan=1>Influenza B</td><td rowspan=1 colspan=1>100% (3/3)</td><td rowspan=1 colspan=1>100% (3/3)</td><td rowspan=1 colspan=1>33% (1/3)</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza virus 3</td><td rowspan=1 colspan=1>100% (3/3)</td><td rowspan=1 colspan=1>100% (3/3)</td><td rowspan=1 colspan=1>33% (1/3)</td></tr><tr><td rowspan=1 colspan=1>Mycoplasma pneumoniae</td><td rowspan=1 colspan=1>100% (3/3)</td><td rowspan=1 colspan=1>100% (3/3)</td><td rowspan=1 colspan=1>33% (1/3)</td></tr><tr><td rowspan=3 colspan=1>MA4</td><td rowspan=1 colspan=1>SARS-CoV-2</td><td rowspan=1 colspan=1>100% (3/3)</td><td rowspan=1 colspan=1>100% (3/3)</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>Human Metapneumovirus</td><td rowspan=1 colspan=1>100% (3/3)</td><td rowspan=1 colspan=1>100% (3/3)</td><td rowspan=1 colspan=1>100% (3/3)</td></tr><tr><td rowspan=1 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>100% (3/3)</td><td rowspan=1 colspan=1>100% (3/3)</td><td rowspan=1 colspan=1>67% (2/3)</td></tr><tr><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>0% (0/3)</td><td rowspan=1 colspan=1>0% (0/3)</td><td rowspan=1 colspan=1>0% (0/3)</td></tr></table>

d. Specimen Collection Media and Extraction Comparison:

Equivalency between collection media; Universal Transport Media (UTM), and Remel MicroTest™ M4RT (M4RT), and between two extraction systems (bioMérieux’s NucliSENS® easyMAG® and $\mathsf { E M A G } ^ { \otimes }$ ) was assessed to verify their compatibility with the NxTAG® RPP v2 assay.

# Luminex

To assess collection media equivalency, four multi-analyte samples consisting of representative targets were contrived in pooled negative nasopharyngeal swabs (NPS) collected in UTM (NCM), and in pooled negative NPS collected in M4RT (NCM-M4RT). The samples, prepared at 3 dilution levels: Above LoD (5x -9x LoD), At/Near LoD (1x-2x LoD), and Below LoD $( 1 / 3 x { - } 2 / 3 \times \mathsf { L o D } )$ , along with a negative sample (negative clinical matrix alone), were extracted using the EMAG system and tested on the NxTAG® RPP v2 assay.

All samples prepared in NCM and extracted using easyMAG generated $\geq 9 5 \%$ positivity for all targets at concentrations that were $\leq 2 \times$ LoD concentration confirmed in using the EMAG in the LoD study. The results demonstrate equivalency between extractions by EMAG and easyMAG for use with NxTAG® RPP v2. Results are summarized in Table 16.

Table 16: NxTAG® RPP v2 Collection Media Equivalency   

<table><tr><td rowspan=2 colspan=1>Organism</td><td rowspan=2 colspan=1>Strain information</td><td rowspan=2 colspan=1>Concentration(Copies/mL)</td><td rowspan=2 colspan=1>SampleType(LoD)</td><td rowspan=1 colspan=2>Target Positivity (%)</td></tr><tr><td rowspan=1 colspan=1>NCM (UTM)</td><td rowspan=1 colspan=1>NCM-M4RT</td></tr><tr><td rowspan=3 colspan=1>Influenza A 2009H1N1</td><td rowspan=3 colspan=1>A/NY/02/09</td><td rowspan=1 colspan=1>8.52E+03</td><td rowspan=1 colspan=1>Above</td><td rowspan=1 colspan=1>10/10(100%)</td><td rowspan=1 colspan=1>10/10 (100%)</td></tr><tr><td rowspan=1 colspan=1>1.70E+03</td><td rowspan=1 colspan=1>At/Near</td><td rowspan=1 colspan=1>30/30 (100%)</td><td rowspan=1 colspan=1>30/30 (100%)</td></tr><tr><td rowspan=1 colspan=1>5.68E+02</td><td rowspan=1 colspan=1>Below</td><td rowspan=1 colspan=1>10/10 (100%)</td><td rowspan=1 colspan=1>7/10 (70%)</td></tr><tr><td rowspan=3 colspan=1>RespiratorySyncytial Virus A</td><td rowspan=3 colspan=1>A2</td><td rowspan=1 colspan=1>2.48E+04</td><td rowspan=1 colspan=1>Above</td><td rowspan=1 colspan=1>10/10 (100%)</td><td rowspan=1 colspan=1>10/10 (100%)</td></tr><tr><td rowspan=1 colspan=1>4.97E+03</td><td rowspan=1 colspan=1>At/Near</td><td rowspan=1 colspan=1>30/30 (100%)</td><td rowspan=1 colspan=1>30/30 (100%)</td></tr><tr><td rowspan=1 colspan=1>1.66E+03</td><td rowspan=1 colspan=1>Below</td><td rowspan=1 colspan=1>6/10 (60%)</td><td rowspan=1 colspan=1>5/10 (50%)</td></tr><tr><td rowspan=3 colspan=1>Rhinovirus</td><td rowspan=3 colspan=1>50-525-CV54</td><td rowspan=1 colspan=1>7.68E+03</td><td rowspan=1 colspan=1>Above</td><td rowspan=1 colspan=1>10/10 (100%)</td><td rowspan=1 colspan=1>10/10 (100%)</td></tr><tr><td rowspan=1 colspan=1>1.54E+03</td><td rowspan=1 colspan=1>At/Near</td><td rowspan=1 colspan=1>30/30 (100%)</td><td rowspan=1 colspan=1>30/30 (100%)</td></tr><tr><td rowspan=1 colspan=1>5.12E+02</td><td rowspan=1 colspan=1>Below</td><td rowspan=1 colspan=1>10/10 (100%)</td><td rowspan=1 colspan=1>9/10 (90%)</td></tr><tr><td rowspan=3 colspan=1>Influenza A H3</td><td rowspan=3 colspan=1>A/Wisconsin/67/05</td><td rowspan=1 colspan=1>2.80E+02</td><td rowspan=1 colspan=1>Above</td><td rowspan=1 colspan=1>10/10 (100%)</td><td rowspan=1 colspan=1>10/10 (100%)</td></tr><tr><td rowspan=1 colspan=1>5.60E+01</td><td rowspan=1 colspan=1>At/Near</td><td rowspan=1 colspan=1>30/30 (100%)</td><td rowspan=1 colspan=1>30/30 (100%)</td></tr><tr><td rowspan=1 colspan=1>1.87E+01</td><td rowspan=1 colspan=1>Below</td><td rowspan=1 colspan=1>4/10 (40%)</td><td rowspan=1 colspan=1>4/10 (40%)</td></tr><tr><td rowspan=3 colspan=1>RespiratorySyncytial Virus B</td><td rowspan=3 colspan=1>18537</td><td rowspan=1 colspan=1>3.60E+04</td><td rowspan=1 colspan=1>Above</td><td rowspan=1 colspan=1>10/10 (100%)</td><td rowspan=1 colspan=1>10/10 (100%)</td></tr><tr><td rowspan=1 colspan=1>7.21E+03</td><td rowspan=1 colspan=1>At/Near</td><td rowspan=1 colspan=1>30/30 (100%)</td><td rowspan=1 colspan=1>30/30 (100%)</td></tr><tr><td rowspan=1 colspan=1>2.40E+03</td><td rowspan=1 colspan=1>Below</td><td rowspan=1 colspan=1>6/10 (60%)</td><td rowspan=1 colspan=1>3/10 (30%)</td></tr><tr><td rowspan=3 colspan=1>Influenza B</td><td rowspan=3 colspan=1>B/Florida/02/06</td><td rowspan=1 colspan=1>3.16E+02</td><td rowspan=1 colspan=1>Above</td><td rowspan=1 colspan=1>10/10 (100%)</td><td rowspan=1 colspan=1>10/10 (100%)</td></tr><tr><td rowspan=1 colspan=1>6.33E+01</td><td rowspan=1 colspan=1>At/Near</td><td rowspan=1 colspan=1>29/30 (97%)</td><td rowspan=1 colspan=1>29/30 (97%)</td></tr><tr><td rowspan=1 colspan=1>2.11E+01</td><td rowspan=1 colspan=1>Below</td><td rowspan=1 colspan=1>5/10 (50%)</td><td rowspan=1 colspan=1>4/10 (40%)</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza virus</td><td rowspan=1 colspan=1>C243</td><td rowspan=1 colspan=1>5.07E+03</td><td rowspan=1 colspan=1>Above</td><td rowspan=1 colspan=1>10/10 (100%)</td><td rowspan=1 colspan=1>10/10 (100%)</td></tr></table>

NxTAG® Respiratory Pathogen Panel v2 Traditional 510(k) Submission   
Table 17: NxTAG® RPP v2 Extractor Equivalency   

<table><tr><td rowspan=2 colspan=1>Organism</td><td rowspan=2 colspan=1>Strain information</td><td rowspan=2 colspan=1>Concentration(Copies/mL)</td><td rowspan=2 colspan=1>SampleType(LoD)</td><td rowspan=1 colspan=2>Target Positivity (%)</td></tr><tr><td rowspan=1 colspan=1>NCM (UTM)</td><td rowspan=1 colspan=1>NCM-M4RT</td></tr><tr><td rowspan=3 colspan=1>3</td><td rowspan=3 colspan=1></td><td rowspan=1 colspan=1>2.03E+03</td><td rowspan=1 colspan=1>At/Near</td><td rowspan=1 colspan=1>30/30 (100%)</td><td rowspan=1 colspan=1>29/30 (97%)</td></tr><tr><td rowspan=1 colspan=1>1.01E+03</td><td rowspan=1 colspan=1>At/Near</td><td rowspan=1 colspan=1>26/30 (87%)</td><td rowspan=1 colspan=1>22/30 (73%)</td></tr><tr><td rowspan=1 colspan=1>3.38E+02</td><td rowspan=1 colspan=1>Below</td><td rowspan=1 colspan=1>0/10 (0%)</td><td rowspan=1 colspan=1>2/10 (20%)</td></tr><tr><td rowspan=3 colspan=1>Mycoplasmapneumoniae</td><td rowspan=3 colspan=1>M129</td><td rowspan=1 colspan=1>2.15E+04</td><td rowspan=1 colspan=1>Above</td><td rowspan=1 colspan=1>10/10 (100%)</td><td rowspan=1 colspan=1>10/10 (100%)</td></tr><tr><td rowspan=1 colspan=1>4.30E+03</td><td rowspan=1 colspan=1>At/Near</td><td rowspan=1 colspan=1>30/30 (100%)</td><td rowspan=1 colspan=1>29/30 (97%)</td></tr><tr><td rowspan=1 colspan=1>1.43E+03</td><td rowspan=1 colspan=1>Below</td><td rowspan=1 colspan=1>3/10 (30%)</td><td rowspan=1 colspan=1>4/10 (40%)</td></tr><tr><td rowspan=3 colspan=1>SARS-CoV-2</td><td rowspan=3 colspan=1>USA-WA1/2020</td><td rowspan=1 colspan=1>2.50E+03</td><td rowspan=1 colspan=1>Above</td><td rowspan=1 colspan=1>10/10 (100%)</td><td rowspan=1 colspan=1>10/10 (100%)</td></tr><tr><td rowspan=1 colspan=1>5.00E+02</td><td rowspan=1 colspan=1>At/Near</td><td rowspan=1 colspan=1>30/30 (100%)</td><td rowspan=1 colspan=1>30/30 (100%)</td></tr><tr><td rowspan=1 colspan=1>1.67E+02</td><td rowspan=1 colspan=1>Below</td><td rowspan=1 colspan=1>10/10 (100%)</td><td rowspan=1 colspan=1>7/10 (70%)</td></tr><tr><td rowspan=3 colspan=1>HumanMetapneumovirus</td><td rowspan=3 colspan=1>hMPV-3, Type B1,Peru2-2002</td><td rowspan=1 colspan=1>1.31E+03</td><td rowspan=1 colspan=1>Above</td><td rowspan=1 colspan=1>10/10 (100%)</td><td rowspan=1 colspan=1>10/10 (100%)</td></tr><tr><td rowspan=1 colspan=1>2.62E+02</td><td rowspan=1 colspan=1>At/Near</td><td rowspan=1 colspan=1>29/30 (97%)</td><td rowspan=1 colspan=1>30/30 (100%)</td></tr><tr><td rowspan=1 colspan=1>8.73E+01</td><td rowspan=1 colspan=1>Below</td><td rowspan=1 colspan=1>5/10 (50%)</td><td rowspan=1 colspan=1>7/10 (70%)</td></tr><tr><td rowspan=4 colspan=1>Adenovirus</td><td rowspan=4 colspan=1>Type 14</td><td rowspan=1 colspan=1>1.23E+04</td><td rowspan=1 colspan=1>Above</td><td rowspan=1 colspan=1>10/10 (100%)</td><td rowspan=1 colspan=1>10/10 (100%)</td></tr><tr><td rowspan=1 colspan=1>3.69E+03</td><td rowspan=1 colspan=1>At/Near</td><td rowspan=1 colspan=1>30/30 (100%)</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>2.46E+03</td><td rowspan=1 colspan=1>At/Near</td><td rowspan=1 colspan=1>28/30 (93%)</td><td rowspan=1 colspan=1>30/30 (100%)</td></tr><tr><td rowspan=1 colspan=1>8.20E+02</td><td rowspan=1 colspan=1>Below</td><td rowspan=1 colspan=1>4/10 (40%)</td><td rowspan=1 colspan=1>0/10 (0%)</td></tr><tr><td rowspan=1 colspan=1>N/A(Negative Sample)</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>0/10(0%)</td><td rowspan=1 colspan=1>0/10 (0%)</td></tr></table>

To demonstrate extractor equivalency, eight multi-analyte samples covering all targets probed by NxTAG® RPP v2 were tested. Multi-analyte samples were contrived in NCM, extracted on the easyMAG system, and tested on the NxTAG® RPP v2 assay.

All samples prepared in NCM and extracted using easyMAG generated $\geq 9 5 \%$ positivity for all targets at concentrations that were within $_ { 2 \times }$ LoD of the confirmed concentration in multi-analyte samples prepared in NCM and extracted using the EMAG. The results demonstrate equivalency between extractions by EMAG and easyMAG for use with $\mathsf { N x T A G } ^ { \otimes }$ RPP v2. Results are summarized in Table 17.

<table><tr><td rowspan=2 colspan=1>Multi-analyteSample</td><td rowspan=2 colspan=1>Organism</td><td rowspan=2 colspan=1>Strain information</td><td rowspan=1 colspan=3>easyMAG (in NCM)</td></tr><tr><td rowspan=1 colspan=1>Concentration(Copies/mL)</td><td rowspan=1 colspan=1>Sample Type(LoD)</td><td rowspan=1 colspan=1>Positivity (%)</td></tr><tr><td rowspan=3 colspan=1>MA1</td><td rowspan=1 colspan=1>Influenza A 2009 H1N1</td><td rowspan=1 colspan=1>A/NY/02/09</td><td rowspan=1 colspan=1>1.70E+03</td><td rowspan=1 colspan=1>At/Near</td><td rowspan=1 colspan=1>20/20 (100%)</td></tr><tr><td rowspan=1 colspan=1>Respiratory Syncytial Virus A</td><td rowspan=1 colspan=1>A2</td><td rowspan=1 colspan=1>4.97E+03</td><td rowspan=1 colspan=1>At/Near</td><td rowspan=1 colspan=1>20/20 (100%)</td></tr><tr><td rowspan=1 colspan=1>Rhinovirus</td><td rowspan=1 colspan=1>50-525-CV54</td><td rowspan=1 colspan=1>5.12E+02</td><td rowspan=1 colspan=1>Below</td><td rowspan=1 colspan=1>20/20 (100%)</td></tr><tr><td rowspan=2 colspan=1>MA2</td><td rowspan=1 colspan=1>Influenza A H3</td><td rowspan=1 colspan=1>A/Wisconsin/67/05</td><td rowspan=1 colspan=1>5.60E+01</td><td rowspan=1 colspan=1>At/Near</td><td rowspan=1 colspan=1>20/20 (100%)</td></tr><tr><td rowspan=1 colspan=1>Respiratory Syncytial Virus B</td><td rowspan=1 colspan=1>18537</td><td rowspan=1 colspan=1>7.21E+03</td><td rowspan=1 colspan=1>At/Near</td><td rowspan=1 colspan=1>20/20 (100%)</td></tr><tr><td rowspan=3 colspan=1>MA3</td><td rowspan=1 colspan=1>Influenza B</td><td rowspan=1 colspan=1>B/Florida/02/06</td><td rowspan=1 colspan=1>6.33E+01</td><td rowspan=1 colspan=1>At/Near</td><td rowspan=1 colspan=1>20/20 (100%)</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza virus 3</td><td rowspan=1 colspan=1>C 243</td><td rowspan=1 colspan=1>1.01E+03</td><td rowspan=1 colspan=1>At/Near</td><td rowspan=1 colspan=1>20/20 (100%)</td></tr><tr><td rowspan=1 colspan=1>Mycoplasma pneumoniae</td><td rowspan=1 colspan=1>M129</td><td rowspan=1 colspan=1>4.30E+03</td><td rowspan=1 colspan=1>At/Near</td><td rowspan=1 colspan=1>20/20 (100%)</td></tr><tr><td rowspan=3 colspan=1>MA4</td><td rowspan=1 colspan=1>SARS-CoV-2</td><td rowspan=1 colspan=1>USA-WA1/2020</td><td rowspan=1 colspan=1>5.00E+02</td><td rowspan=1 colspan=1>At/Near</td><td rowspan=1 colspan=1>20/20 (100%)</td></tr><tr><td rowspan=1 colspan=1>Human Metapneumovirus</td><td rowspan=1 colspan=1>hMPV-3, Type B1,Peru2-2002</td><td rowspan=1 colspan=1>2.62E+02</td><td rowspan=1 colspan=1>At/Near</td><td rowspan=1 colspan=1>20/20 (100%)</td></tr><tr><td rowspan=1 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>Type 14</td><td rowspan=1 colspan=1>2.46E+03</td><td rowspan=1 colspan=1>At/Near</td><td rowspan=1 colspan=1>20/20 (100%)</td></tr><tr><td rowspan=3 colspan=1>MA5</td><td rowspan=1 colspan=1>Influenza A Matrix</td><td rowspan=1 colspan=1>A/Wisconsin/67/05</td><td rowspan=1 colspan=1>1.68E+02</td><td rowspan=1 colspan=1>At/Near</td><td rowspan=1 colspan=1>20/20 (100%)</td></tr><tr><td rowspan=1 colspan=1>Coronavirus NL63</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>1.00E+02</td><td rowspan=1 colspan=1>At/Near</td><td rowspan=1 colspan=1>20/20 (100%)</td></tr><tr><td rowspan=1 colspan=1>Coronavirus HKU1</td><td rowspan=1 colspan=1>Type B</td><td rowspan=1 colspan=1>4.18E+03</td><td rowspan=1 colspan=1>At/Near</td><td rowspan=1 colspan=1>20/20 (100%)</td></tr><tr><td rowspan=3 colspan=1>MA6</td><td rowspan=1 colspan=1>Influenza H1</td><td rowspan=1 colspan=1>A/Brisbane/59/07</td><td rowspan=1 colspan=1>5.35E+02</td><td rowspan=1 colspan=1>Below</td><td rowspan=1 colspan=1>19/20 (95%)</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza virus 1</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>6.92E+02</td><td rowspan=1 colspan=1>Below</td><td rowspan=1 colspan=1>20/20 (100%)</td></tr><tr><td rowspan=1 colspan=1>Chlamydia pneumoniae</td><td rowspan=1 colspan=1>TW-183</td><td rowspan=1 colspan=1>7.93E+01</td><td rowspan=1 colspan=1>Below</td><td rowspan=1 colspan=1>20/20 (100%)</td></tr><tr><td rowspan=3 colspan=1>MA7</td><td rowspan=1 colspan=1>Parainfluenza virus 2</td><td rowspan=1 colspan=1>Greer</td><td rowspan=1 colspan=1>3.45E+02</td><td rowspan=1 colspan=1>At/Near</td><td rowspan=1 colspan=1>19/20 (95%)</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza virus 4B</td><td rowspan=1 colspan=1>CH 19503</td><td rowspan=1 colspan=1>7.15E+03</td><td rowspan=1 colspan=1>At/Near</td><td rowspan=1 colspan=1>20/20 (100%)</td></tr><tr><td rowspan=1 colspan=1>Coronavirus 229E</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>3.81E+02</td><td rowspan=1 colspan=1>At/Near</td><td rowspan=1 colspan=1>20/20 (100%)</td></tr><tr><td rowspan=2 colspan=1>MA8</td><td rowspan=1 colspan=1>Parainfluenza virus 4A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>5.65E+03</td><td rowspan=1 colspan=1>Below</td><td rowspan=1 colspan=1>19/20 (95%)</td></tr><tr><td rowspan=1 colspan=1>Coronavirus OC43</td><td rowspan=1 colspan=1>Betacoronavirus 1</td><td rowspan=1 colspan=1>4.55E+03</td><td rowspan=1 colspan=1>At/Near</td><td rowspan=1 colspan=1>20/20 (100%)</td></tr></table>

# 3. Clinical Performance:

A multi-site clinical study established the clinical performance of the $\mathsf { N } \times \mathsf { T } \mathsf { A G } ^ { \otimes }$ RPP v2 assay for the detection and identification of nucleic acids from multiple respiratory viruses and bacteria extracted from upper respiratory tract specimens collected from individuals with clinical signs and symptoms of a respiratory tract infection. The clinical performance of the $\mathsf { N x T A G } ^ { \otimes }$ RPP v2 assay was evaluated using clinical specimens prospectively collected between October 2022 and April 2023 from five geographically diverse clinical sites within the United States. The clinical study utilized leftover, deidentified specimens collected from pediatric and adult patients exhibiting clinical signs and symptoms of a respiratory tract infection.

The NxTAG® RPP v2 results were compared to those obtained with an FDA-cleared molecular assay and PCR/bi-directional sequencing for influenza A subtyping. The twostep PCR analysis followed by BDS assays employed two independent sets of validated PCR assays. The primers used for PCR analysis and sequencing assays, where possible, were designed to amplify distinct regions from the investigational device. PCR analysis composite positive specimens were confirmed by BDS assays. See Table 18 for comparator method testing by target.

Table 18: Prospective Comparator Method Algorithm   

<table><tr><td rowspan=1 colspan=1>NxTAG® RPP v2 Target</td><td rowspan=1 colspan=1>Comparator Method</td></tr><tr><td rowspan=1 colspan=1>Adenovirus</td><td rowspan=19 colspan=1>FDA cleared molecular assay</td></tr><tr><td rowspan=1 colspan=1>Enterovirus/Rhinovirus</td></tr><tr><td rowspan=1 colspan=1>Influenza A</td></tr><tr><td rowspan=1 colspan=1>Influenza A H3</td></tr><tr><td rowspan=1 colspan=1>Influenza B</td></tr><tr><td rowspan=1 colspan=1>Respiratory Syncytial Virus A</td></tr><tr><td rowspan=1 colspan=1>Respiratory Syncytial Virus B</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza 1</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza 2</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza 3</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza 4</td></tr><tr><td rowspan=1 colspan=1>Coronavirus 229E</td></tr><tr><td rowspan=1 colspan=1>Coronavirus NL63</td></tr><tr><td rowspan=1 colspan=1>Coronavirus OC43</td></tr><tr><td rowspan=1 colspan=1>Coronavirus HKU1</td></tr><tr><td rowspan=1 colspan=1>Human Metapneumovirus</td></tr><tr><td rowspan=2 colspan=1>Chlamydia pneumoniae</td><td rowspan=2 colspan=1></td></tr><tr><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Mycoplasma pneumoniae</td></tr><tr><td rowspan=1 colspan=1>Influenza A H1</td><td rowspan=2 colspan=1>FDA cleared molecular assay followed bycomposite of PCR followed by BDS NAATs*</td></tr><tr><td rowspan=1 colspan=1>Influenza A H1pdm09</td></tr><tr><td rowspan=1 colspan=1>SARS-CoV-2</td><td rowspan=1 colspan=1>FDA cleared molecular assay</td></tr></table>

\* NAAT – Nucleic Acid Amplification Test

A total of 1844 prospective specimens, collected from five geographically diverse US sites were initially enrolled in the study, of which 19 were excluded from the analysis of performance (duplicate specimen, not from a unique subject (11), absence of signs and symptoms (5), improper labeling (2), operator error (1)). Nineteen of the remaining 1825 specimens initially produced invalid NxTAG RPP v2 results, of which 14 resolved upon repeat testing. Therefore, a total of 1820 prospectively collected specimens

# Luminex

generated valid NxTAG RPP v2 results after allowing for a single retest. Certain additional specimens were excluded from performance calculations for specific analytes based on the availability of valid results for the applicable comparator method. Clinical runs and re-runs using the $\mathsf { N } \times \mathsf { T } \mathsf { A G } ^ { \textregistered }$ RPP v2 assay were tested on the MAGPIX System by trained operators at three sites. Prospective specimen (Arm 1) testing occurred between March 2023 and April 2023.

For targets that exhibited low prevalence rates in the prospective study cohort, the prospective specimen set was supplemented with 320 pre-selected left-over, deidentified specimens (Arm 2) sourced from six sites in the United States. Pre-selected specimens were identified by Standard of Care (SoC) results and confirmed by BDS testing prior to enrollment in the study. Of the preselected specimens, 11 were excluded from the analysis of performance because confirmatory testing could not be completed. In addition, 3 specimens initially produced invalid NxTAG RPP v2 results, of which 2 resolved upon repeat testing. Therefore, a total of 308 preselected specimens were included in the analysis of performance. To minimize bias, pre-selected specimens were tested in a randomized, blinded manner with negative specimens at four sites. Pre-selected specimen (Arm 2) testing occurred between December 2021 and May 2023.

To supplement the number of clinical specimens positive for Chlamydia pneumoniae, RSV B, pre-2009 pandemic Influenza A H1N1, and Coronavirus 229E in the prospective and pre-selected arms of the study, additional testing was performed using contrived specimens. Contrived specimens were prepared by spiking representative strains into unique negative human nasopharyngeal specimens at 2x Limit of Detection (LoD), 10x LoD, and $1 0 0 \times$ LoD, for all strains except influenza A H1N1 which was prepared at 2x and ${ 1 0 \times }$ LoD. A total of 199 specimens were contrived and tested as part of Arm 3.

To minimize bias, contrived specimens were blinded, randomized, and tested along with Arm 2 positive and negative clinical specimens at two testing sites between December 2021 and January 2022. Results from contrived specimens were analyzed separately from the prospective and pre-selected data sets.

Out of the 199 specimens included in the contrived study analysis, 198 $( 9 9 . 5 0 \% )$ generated valid $\mathsf { N x T A G } ^ { \otimes }$ RPP v2 Assay results (i.e., Positive or Negative) on the first attempt. There was one specimen $( 0 . 5 0 \% )$ with an invalid result on the initial run. This specimen generated a valid result after a single retest for a final success rate of $100 \%$ .

The invalid rate for prospective, pre-selected, and contrived specimens combined was $0 . 9 8 \%$ (23/2344) after the initial run. Of the 23 specimens with initial invalid results, 17 $( 0 . 7 3 \% )$ specimens generated valid NxTAG® RPP v2 results after a single retest, three $( 0 . 1 3 \% )$ specimens remained invalid on repeat, and three $( 0 . 1 3 \% )$ specimens were not retested due to volume limitations.

# Luminex

For each target in the $\mathsf { N x T A G } ^ { \otimes }$ RPP v2 Assay, the performance (Positive Percent Agreement, Negative Percent Agreement, and $9 5 \%$ confidence interval) of the NxTAG® RPP v2 Assay as compared to the reference method are summarized in Tables 19 and 20 for prospective and pre-selected specimen analysis, respectively. The performance of the $\mathsf { N x T A G } ^ { \otimes }$ RPP v2 Assay for contrived specimens is presented separately in Table 21. For each of the contrived specimens $( \mathsf { n } = \mathsf { 1 9 9 } )$ , negative results were obtained for all other analytes included on the panel that are not listed in Table 21.

Table 19: NxTAG® RPP v2 Performance for the Prospective Data Set   

<table><tr><td colspan="2" rowspan="1">Pathogen Target</td><td colspan="3" rowspan="1">Positive Percent Agreement</td><td colspan="3" rowspan="1">Negative Percent Agreement</td></tr><tr><td colspan="2" rowspan="1"></td><td colspan="1" rowspan="1">TP /(TP+FN)</td><td colspan="1" rowspan="1">PPA (%)</td><td colspan="1" rowspan="1">95% Cl</td><td colspan="1" rowspan="1">TN /(TN+FP)</td><td colspan="1" rowspan="1">NPA (%)</td><td colspan="1" rowspan="1">95% Cl</td></tr><tr><td colspan="8" rowspan="1">Viruses</td></tr><tr><td colspan="1" rowspan="3">Adenovirus</td><td colspan="1" rowspan="1">Fresh</td><td colspan="1" rowspan="1">39/39</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">91%-100%</td><td colspan="1" rowspan="1">733/736</td><td colspan="1" rowspan="1">99.6%</td><td colspan="1" rowspan="1">99%-100%</td></tr><tr><td colspan="1" rowspan="1">Frozen</td><td colspan="1" rowspan="1">55/59</td><td colspan="1" rowspan="1">93.2%</td><td colspan="1" rowspan="1">84%-97%</td><td colspan="1" rowspan="1">976/980</td><td colspan="1" rowspan="1">99.6%</td><td colspan="1" rowspan="1">99%-100%</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">94/98a</td><td colspan="1" rowspan="1">95.9%</td><td colspan="1" rowspan="1">90%-98%</td><td colspan="1" rowspan="1">1709/1716b</td><td colspan="1" rowspan="1">99.6%</td><td colspan="1" rowspan="1">99%-100%</td></tr><tr><td colspan="1" rowspan="3">Coronavirus 229E</td><td colspan="1" rowspan="1">Fresh</td><td colspan="1" rowspan="1">2/2</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">34%-100%</td><td colspan="1" rowspan="1">773/773</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%-100%</td></tr><tr><td colspan="1" rowspan="1">Frozen</td><td colspan="1" rowspan="1">5/5</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">57%-100%</td><td colspan="1" rowspan="1">1033/1034</td><td colspan="1" rowspan="1">99.9%</td><td colspan="1" rowspan="1">99%-100%</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">7/7</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">65%-100%</td><td colspan="1" rowspan="1">1806/1807</td><td colspan="1" rowspan="1">99.9%</td><td colspan="1" rowspan="1">100%-100%</td></tr><tr><td colspan="1" rowspan="3">Coronavirus HKU1</td><td colspan="1" rowspan="1">Fresh</td><td colspan="1" rowspan="1">8/8</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">68%-100%</td><td colspan="1" rowspan="1">767/767</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%-100%</td></tr><tr><td colspan="1" rowspan="1">Frozen</td><td colspan="1" rowspan="1">12/13</td><td colspan="1" rowspan="1">92.3%</td><td colspan="1" rowspan="1">67%-99%</td><td colspan="1" rowspan="1">1026/1026</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%-100%</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">20/21c</td><td colspan="1" rowspan="1">95.2%</td><td colspan="1" rowspan="1">77%-99%</td><td colspan="1" rowspan="1">1793/1793</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%-100%</td></tr><tr><td colspan="1" rowspan="3">Coronavirus NL63</td><td colspan="1" rowspan="1">Fresh</td><td colspan="1" rowspan="1">25/27</td><td colspan="1" rowspan="1">92.6%</td><td colspan="1" rowspan="1">77%-98%</td><td colspan="1" rowspan="1">748/748</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">99%-100%</td></tr><tr><td colspan="1" rowspan="1">Frozen</td><td colspan="1" rowspan="1">23/25</td><td colspan="1" rowspan="1">92%</td><td colspan="1" rowspan="1">75%-98%</td><td colspan="1" rowspan="1">1014/1014</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%-100%</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">48/52</td><td colspan="1" rowspan="1">92.3%</td><td colspan="1" rowspan="1">82%-97%</td><td colspan="1" rowspan="1">1762/1762</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%-100%</td></tr><tr><td colspan="1" rowspan="3">Coronavirus OC43</td><td colspan="1" rowspan="1">Fresh</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">72%-100%</td><td colspan="1" rowspan="1">765/765</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%-100%</td></tr><tr><td colspan="1" rowspan="1">Frozen</td><td colspan="1" rowspan="1">29/29</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">88%-100%</td><td colspan="1" rowspan="1">1010/1010</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%-100%</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">39/39</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">91%-100%</td><td colspan="1" rowspan="1">1775/1775</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%-100%</td></tr><tr><td colspan="1" rowspan="3">HumanMetapneumovirus</td><td colspan="1" rowspan="1">Fresh</td><td colspan="1" rowspan="1">87/87</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">96%-100%</td><td colspan="1" rowspan="1">680/688</td><td colspan="1" rowspan="1">98.8%</td><td colspan="1" rowspan="1">98%-99%</td></tr><tr><td colspan="1" rowspan="1">Frozen</td><td colspan="1" rowspan="1">70/70</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">95%-100%</td><td colspan="1" rowspan="1">963/969</td><td colspan="1" rowspan="1">99.4%</td><td colspan="1" rowspan="1">99%-100%</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">157/157</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">98%-100%</td><td colspan="1" rowspan="1">1643/1657d</td><td colspan="1" rowspan="1">99.2%</td><td colspan="1" rowspan="1">99%-99%</td></tr><tr><td colspan="1" rowspan="3">Influenza A</td><td colspan="1" rowspan="1">Fresh</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">84%-100%</td><td colspan="1" rowspan="1">753/755</td><td colspan="1" rowspan="1">99.7%</td><td colspan="1" rowspan="1">99%-100%</td></tr><tr><td colspan="1" rowspan="1">Frozen</td><td colspan="1" rowspan="1">54/54</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">93%-100%</td><td colspan="1" rowspan="1">984/985</td><td colspan="1" rowspan="1">99.9%</td><td colspan="1" rowspan="1">99%-100%</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">74/74</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">95%-100%</td><td colspan="1" rowspan="1">1737/1740e</td><td colspan="1" rowspan="1">99.8%</td><td colspan="1" rowspan="1">99%-100%</td></tr><tr><td colspan="1" rowspan="3">Influenza AH1pdm09</td><td colspan="1" rowspan="1">Fresh</td><td colspan="1" rowspan="1">9/9</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">70%-100%</td><td colspan="1" rowspan="1">765/765</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%-100%</td></tr><tr><td colspan="1" rowspan="1">Frozen</td><td colspan="1" rowspan="1">22/22</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">85%-100%</td><td colspan="1" rowspan="1">1017/1017</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%-100%</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">31/31</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">89%-100%</td><td colspan="1" rowspan="1">1782/1782</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%-100%</td></tr><tr><td colspan="1" rowspan="3">Influenza A H1</td><td colspan="1" rowspan="1">Fresh</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">774/774</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%-100%</td></tr><tr><td colspan="1" rowspan="1">Frozen</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">1039/1039</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%-100%</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">1813/1813</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%-100%</td></tr><tr><td colspan="1" rowspan="2">Influenza A H3</td><td colspan="1" rowspan="1">Fresh</td><td colspan="1" rowspan="1">11/11</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">74%-100%</td><td colspan="1" rowspan="1">764/764</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">99%-100%</td></tr><tr><td colspan="1" rowspan="1">Frozen</td><td colspan="1" rowspan="1">34/36</td><td colspan="1" rowspan="1">94.4%</td><td colspan="1" rowspan="1">82%-98%</td><td colspan="1" rowspan="1">1002/1003</td><td colspan="1" rowspan="1">99.9%</td><td colspan="1" rowspan="1">99%-100%</td></tr><tr><td colspan="2" rowspan="1"></td><td colspan="1" rowspan="1">TP / (TP+FN)</td><td colspan="1" rowspan="1">PPA (%)</td><td colspan="1" rowspan="1">95% CI</td><td colspan="1" rowspan="1">TN /(TN+FP)</td><td colspan="1" rowspan="1">NPA (%)</td><td colspan="1" rowspan="1">95% Cl</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">45/47f</td><td colspan="1" rowspan="1">95.7%</td><td colspan="1" rowspan="1">86%-99%</td><td colspan="1" rowspan="1">1766/1767</td><td colspan="1" rowspan="1">99.9%</td><td colspan="1" rowspan="1">100%-100%</td></tr><tr><td colspan="1" rowspan="3">Influenza B</td><td colspan="1" rowspan="1">Fresh</td><td colspan="1" rowspan="1">5/5</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">57%-100%</td><td colspan="1" rowspan="1">770/770</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%-100%</td></tr><tr><td colspan="1" rowspan="1">Frozen</td><td colspan="1" rowspan="1">6/6</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">61%-100%</td><td colspan="1" rowspan="1">1033/1033</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%-100%</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">11/11</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">74%-100%</td><td colspan="1" rowspan="1">1803/1803</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%-100%</td></tr><tr><td colspan="1" rowspan="3">Parainfluenza 1</td><td colspan="1" rowspan="1">Fresh</td><td colspan="1" rowspan="1">7/7</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">65%-100%</td><td colspan="1" rowspan="1">768/768</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%-100%</td></tr><tr><td colspan="1" rowspan="1">Frozen</td><td colspan="1" rowspan="1">11/11</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">74%-100%</td><td colspan="1" rowspan="1">1028/1028</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%-100%</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">18/18</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">82%-100%</td><td colspan="1" rowspan="1">1796/1796</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%-100%</td></tr><tr><td colspan="1" rowspan="3">Parainfluenza 2</td><td colspan="1" rowspan="1">Fresh</td><td colspan="1" rowspan="1">4/5</td><td colspan="1" rowspan="1">80%</td><td colspan="1" rowspan="1">38%-96%</td><td colspan="1" rowspan="1">770/770</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%-100%</td></tr><tr><td colspan="1" rowspan="1">Frozen</td><td colspan="1" rowspan="1">5/5</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">57%-100%</td><td colspan="1" rowspan="1">1034/1034</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%-100%</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">9/10</td><td colspan="1" rowspan="1">90%</td><td colspan="1" rowspan="1">60%-98%</td><td colspan="1" rowspan="1">1804/1804</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%-100%</td></tr><tr><td colspan="1" rowspan="3">Parainfluenza 3</td><td colspan="1" rowspan="1">Fresh</td><td colspan="1" rowspan="1">19/19</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">83%-100%</td><td colspan="1" rowspan="1">756/756</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">99%-100%</td></tr><tr><td colspan="1" rowspan="1">Frozen</td><td colspan="1" rowspan="1">23/23</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">86%-100%</td><td colspan="1" rowspan="1">1015/1016</td><td colspan="1" rowspan="1">99.9%</td><td colspan="1" rowspan="1">99%-100%</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">42/42</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">92%-100%</td><td colspan="1" rowspan="1">1771/1772</td><td colspan="1" rowspan="1">99.9%</td><td colspan="1" rowspan="1">100%-100%</td></tr><tr><td colspan="1" rowspan="3">Parainfluenza 4</td><td colspan="1" rowspan="1">Fresh</td><td colspan="1" rowspan="1">2/3</td><td colspan="1" rowspan="1">66.7%</td><td colspan="1" rowspan="1">21%-94%</td><td colspan="1" rowspan="1">770/772</td><td colspan="1" rowspan="1">99.7%</td><td colspan="1" rowspan="1">99%-100%</td></tr><tr><td colspan="1" rowspan="1">Frozen</td><td colspan="1" rowspan="1">11/12</td><td colspan="1" rowspan="1">91.7%</td><td colspan="1" rowspan="1">65%-99%</td><td colspan="1" rowspan="1">1026/1027</td><td colspan="1" rowspan="1">99.9%</td><td colspan="1" rowspan="1">99%-100%</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">13/158</td><td colspan="1" rowspan="1">86.7%</td><td colspan="1" rowspan="1">62%-96%</td><td colspan="1" rowspan="1">1796/1799h</td><td colspan="1" rowspan="1">99.8%</td><td colspan="1" rowspan="1">100%-100%</td></tr><tr><td colspan="1" rowspan="3">RSV A</td><td colspan="1" rowspan="1">Fresh</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">72%-100%</td><td colspan="1" rowspan="1">764/765</td><td colspan="1" rowspan="1">99.9%</td><td colspan="1" rowspan="1">99%-100%</td></tr><tr><td colspan="1" rowspan="1">Frozen</td><td colspan="1" rowspan="1">45/45</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">92%-100%</td><td colspan="1" rowspan="1">992/994</td><td colspan="1" rowspan="1">99.8%</td><td colspan="1" rowspan="1">99%-100%</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">55/55</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">93%-100%</td><td colspan="1" rowspan="1">1756/1759</td><td colspan="1" rowspan="1">99.8%</td><td colspan="1" rowspan="1">99%-100%</td></tr><tr><td colspan="1" rowspan="3">RSV B</td><td colspan="1" rowspan="1">Fresh</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">44%-100%</td><td colspan="1" rowspan="1">772/772</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%-100%</td></tr><tr><td colspan="1" rowspan="1">Frozen</td><td colspan="1" rowspan="1">17/17</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">82%-100%</td><td colspan="1" rowspan="1">1022/1022</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%-100%</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">84%-100%</td><td colspan="1" rowspan="1">1794/1794</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%-100%</td></tr><tr><td colspan="1" rowspan="3">Rhinovirus /Enterovirus</td><td colspan="1" rowspan="1">Fresh</td><td colspan="1" rowspan="1">123/132</td><td colspan="1" rowspan="1">93.2%</td><td colspan="1" rowspan="1">88%-96%</td><td colspan="1" rowspan="1">643/643</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">99%-100%</td></tr><tr><td colspan="1" rowspan="1">Frozen</td><td colspan="1" rowspan="1">228/237</td><td colspan="1" rowspan="1">96.2%</td><td colspan="1" rowspan="1">93%-98%</td><td colspan="1" rowspan="1">801/802</td><td colspan="1" rowspan="1">99.9%</td><td colspan="1" rowspan="1">99%-100%</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">351/369</td><td colspan="1" rowspan="1">95.1%</td><td colspan="1" rowspan="1">92%-97%</td><td colspan="1" rowspan="1">1444/1445</td><td colspan="1" rowspan="1">99.9%</td><td colspan="1" rowspan="1">100%-100%</td></tr><tr><td colspan="1" rowspan="3">SARS-CoV-2</td><td colspan="1" rowspan="1">Fresh</td><td colspan="1" rowspan="1">103/106</td><td colspan="1" rowspan="1">97.2%</td><td colspan="1" rowspan="1">92%-99%</td><td colspan="1" rowspan="1">656/660</td><td colspan="1" rowspan="1">99.4%</td><td colspan="1" rowspan="1">98%-100%</td></tr><tr><td colspan="1" rowspan="1">Frozen</td><td colspan="1" rowspan="1">126/128</td><td colspan="1" rowspan="1">98.4%</td><td colspan="1" rowspan="1">94%-100%</td><td colspan="1" rowspan="1">902/909</td><td colspan="1" rowspan="1">99.2%</td><td colspan="1" rowspan="1">98%-100%</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">229/234k</td><td colspan="1" rowspan="1">97.9%</td><td colspan="1" rowspan="1">95%-99%</td><td colspan="1" rowspan="1">1558/1569'</td><td colspan="1" rowspan="1">99.3%</td><td colspan="1" rowspan="1">99%-100%</td></tr><tr><td colspan="8" rowspan="1">Bacteria</td></tr><tr><td colspan="1" rowspan="3">Chlamydiapneumoniae</td><td colspan="1" rowspan="1">Fresh</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">775/775</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%-100%</td></tr><tr><td colspan="1" rowspan="1">Frozen</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">1039/1039</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%-100%</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">1814/1814</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%-100%</td></tr><tr><td colspan="1" rowspan="3">Mycoplasmapneumoniae</td><td colspan="1" rowspan="1">Fresh</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">775/775</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%-100%</td></tr><tr><td colspan="1" rowspan="1">Frozen</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">1039/1039</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%-100%</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">1814/1814</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%-100%</td></tr><tr><td colspan="1" rowspan="1">aTwe of the four Ader</td><td colspan="1" rowspan="1">is Ealse Noga</td><td colspan="1" rowspan="1">ives wore negatiy</td><td colspan="1" rowspan="1">2 by BDc and t</td><td colspan="1" rowspan="1">ve wore net tosto</td><td colspan="1" rowspan="1">due to volume li</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr></table>

aTwo of the four Adenovirus False Negatives were negative by BDS and two were not tested due to volume limitations. bFour of the seven Adenovirus False Positives were positive by the molecular SoC assay. cThe one prospective Coronavirus HKU1 False Negative was negative by the molecular SoC assay. dTwo of the fourteen Human Metapneumovirus False Positives were positive by the molecular SoC assay. Seven samples could not be tested due to volume limitations. eThe Influenza A subtype was detected by the reference method for three of the three Influenza A False Positives. fOne of the two Influenza A H3 False Negatives was negative by the molecular SoC assay. gOne of the two Parainfluenza 4 False Negatives was negative by the molecular SoC assay.

# Luminex

NxTAG® Respiratory Pathogen Panel v2 Traditional 510(k) Submission

hTwo of the three Parainfluenza 4 False Positives were positive by the molecular SoC assay.   
iOne of the three RSV A False Positives was positive by the molecular SoC assay.   
jEight of the eighteen Rhinovirus/Enterovirus False Negatives were negative by BDS, and three False Negatives were negative by the molecular SoC assay. Five samples could not be tested due to volume limitations.   
kTwo of the five SARS-CoV-2 False Negatives were negative by the molecular SoC assay.   
lSix of the eleven SARS-CoV-2 False Positives were positive by the molecular SoC assay.

Table 20: NxTAG® RPP v2 Performance for the Pre-Selected Data Set   

<table><tr><td colspan="2" rowspan="2">Pathogen Target</td><td colspan="3" rowspan="1">Positive Percent Agreement</td><td colspan="3" rowspan="1">Negative Percent Agreement</td></tr><tr><td colspan="1" rowspan="1">TP/(TP+FN)</td><td colspan="1" rowspan="1">PPA (%)</td><td colspan="1" rowspan="1">95% Cl</td><td colspan="1" rowspan="1">TN /(TN+FP)</td><td colspan="1" rowspan="1">NPA(%)</td><td colspan="1" rowspan="1">95% Cl</td></tr><tr><td colspan="8" rowspan="1">Viruses</td></tr><tr><td colspan="1" rowspan="1">Adenovirus</td><td colspan="1" rowspan="1">Pre-Selected</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">21%-100%</td><td colspan="1" rowspan="1">307/307</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">99%-100%</td></tr><tr><td colspan="1" rowspan="1">Coronavirus 229E</td><td colspan="1" rowspan="1">Pre-Selected</td><td colspan="1" rowspan="1">11/11</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">74%-100%</td><td colspan="1" rowspan="1">297/297</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">99%-100%</td></tr><tr><td colspan="1" rowspan="1">Coronavirus HKU1</td><td colspan="1" rowspan="1">Pre-Selected</td><td colspan="1" rowspan="1">30/32</td><td colspan="1" rowspan="1">93.8%</td><td colspan="1" rowspan="1">80%-98%</td><td colspan="1" rowspan="1">276/276</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">99%-100%</td></tr><tr><td colspan="1" rowspan="1">Coronavirus NL63</td><td colspan="1" rowspan="1">Pre-Selected</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">308/308</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">99%-100%</td></tr><tr><td colspan="1" rowspan="1">Coronavirus OC43</td><td colspan="1" rowspan="1">Pre-Selected</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">308/308</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">99%-100%</td></tr><tr><td colspan="1" rowspan="1">HumanMetapneumovirus</td><td colspan="1" rowspan="1">Pre-Selected</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">308/308</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">99%-100%</td></tr><tr><td colspan="1" rowspan="1">Influenza A</td><td colspan="1" rowspan="1">Pre-Selected</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">89%-100%</td><td colspan="1" rowspan="1">277/278</td><td colspan="1" rowspan="1">99.6%</td><td colspan="1" rowspan="1">98%-100%</td></tr><tr><td colspan="1" rowspan="1">Influenza AH1pdm09</td><td colspan="1" rowspan="1">Pre-Selected</td><td colspan="1" rowspan="1">29/30</td><td colspan="1" rowspan="1">96.7%</td><td colspan="1" rowspan="1">83%-99%</td><td colspan="1" rowspan="1">278/278</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">99%-100%</td></tr><tr><td colspan="1" rowspan="1">Influenza A H1</td><td colspan="1" rowspan="1">Pre-Selected</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">277/278</td><td colspan="1" rowspan="1">99.6%</td><td colspan="1" rowspan="1">98%-100%</td></tr><tr><td colspan="1" rowspan="1">Influenza A H3</td><td colspan="1" rowspan="1">Pre-Selected</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">278/278</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">99%-100%</td></tr><tr><td colspan="1" rowspan="1">Influenza B</td><td colspan="1" rowspan="1">Pre-Selected</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">89%-100%</td><td colspan="1" rowspan="1">278/278</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">99%-100%</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza 1</td><td colspan="1" rowspan="1">Pre-Selected</td><td colspan="1" rowspan="1">29/29</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">88%-100%</td><td colspan="1" rowspan="1">279/279</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">99%-100%</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza 2</td><td colspan="1" rowspan="1">Pre-Selected</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">89%-100%</td><td colspan="1" rowspan="1">278/278</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">99%-100%</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza 3</td><td colspan="1" rowspan="1">Pre-Selected</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">307/307</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">99%-100%</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza 4</td><td colspan="1" rowspan="1">Pre-Selected</td><td colspan="1" rowspan="1">15/16</td><td colspan="1" rowspan="1">93.8%</td><td colspan="1" rowspan="1">72%-99%</td><td colspan="1" rowspan="1">292/292</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">99%-100%</td></tr><tr><td colspan="1" rowspan="1">RSV A</td><td colspan="1" rowspan="1">Pre-Selected</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">305/307</td><td colspan="1" rowspan="1">99.3%</td><td colspan="1" rowspan="1">98%-100%</td></tr><tr><td colspan="1" rowspan="1">RSV B</td><td colspan="1" rowspan="1">Pre-Selected</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">306/307</td><td colspan="1" rowspan="1">99.7%</td><td colspan="1" rowspan="1">98%-100%</td></tr><tr><td colspan="1" rowspan="1">Rhinovirus /Enterovirus</td><td colspan="1" rowspan="1">Pre-Selected</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">21%-100%</td><td colspan="1" rowspan="1">302/306</td><td colspan="1" rowspan="1">98.7%</td><td colspan="1" rowspan="1">97%-99%</td></tr><tr><td colspan="1" rowspan="1">SARS-CoV-2</td><td colspan="1" rowspan="1">Pre-Selected</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">0/0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td></tr><tr><td colspan="8" rowspan="1">Bacteria</td></tr><tr><td colspan="1" rowspan="1">Chlamydiapneumoniae</td><td colspan="1" rowspan="1">Pre-Selected</td><td colspan="1" rowspan="1">14/14</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">78%-100%</td><td colspan="1" rowspan="1">293/294</td><td colspan="1" rowspan="1">99.7%</td><td colspan="1" rowspan="1">98%-100%</td></tr><tr><td colspan="1" rowspan="1">TP/(TP+FN)</td><td colspan="1" rowspan="1">PPA (%)</td><td colspan="1" rowspan="1">95% Cl</td><td colspan="1" rowspan="1">TN /(TN+FP)</td><td colspan="1" rowspan="1">NPA(%)</td><td colspan="1" rowspan="1">95% CI</td></tr><tr><td colspan="8" rowspan="1">Viruses</td></tr><tr><td colspan="1" rowspan="1">Mycoplasmapneumoniae</td><td colspan="1" rowspan="1">Pre-Selected</td><td colspan="1" rowspan="1">48/52</td><td colspan="1" rowspan="1">92.3%</td><td colspan="1" rowspan="1">82%-97%</td><td colspan="1" rowspan="1">256/256</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">99%-100%</td></tr></table>

Table 21: NxTAG® RPP v2 Performance for the Contrived Data Set   

<table><tr><td rowspan=2 colspan=2>Pathogen Target</td><td rowspan=1 colspan=3>Positive Percent Agreement</td><td rowspan=1 colspan=3>Negative Percent Agreement</td></tr><tr><td rowspan=1 colspan=1>TP /(TP+FN)</td><td rowspan=1 colspan=1>PPA (%)</td><td rowspan=1 colspan=1>95% Cl</td><td rowspan=1 colspan=1>TN /(TN+FP)</td><td rowspan=1 colspan=1>NPA (%)</td><td rowspan=1 colspan=1>95% Cl</td></tr><tr><td rowspan=1 colspan=1>Analyte</td><td rowspan=1 colspan=1>LoD</td><td rowspan=1 colspan=6>Viruses</td></tr><tr><td rowspan=4 colspan=1>Coronavirus229E</td><td rowspan=1 colspan=1>2x</td><td rowspan=1 colspan=1>25/25</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>87%-100%</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td></tr><tr><td rowspan=1 colspan=1>10x</td><td rowspan=1 colspan=1>12/12</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>76%-100%</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td></tr><tr><td rowspan=1 colspan=1>100x</td><td rowspan=1 colspan=1>12/12</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>76%-100%</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td></tr><tr><td rowspan=1 colspan=1>Combined</td><td rowspan=1 colspan=1>49/49</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>93%-100%</td><td rowspan=1 colspan=1>150/150</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>98%-100%</td></tr><tr><td rowspan=4 colspan=1>Influenza A(matrix)</td><td rowspan=1 colspan=1>2x</td><td rowspan=1 colspan=1>26/26</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>87%-100%</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td></tr><tr><td rowspan=1 colspan=1>10x</td><td rowspan=1 colspan=1>24/24</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>86%-100%</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td></tr><tr><td rowspan=1 colspan=1>100x</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td></tr><tr><td rowspan=1 colspan=1>Combined</td><td rowspan=1 colspan=1>50/50</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>93%-100%</td><td rowspan=1 colspan=1>149/149</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>97%-100%</td></tr><tr><td rowspan=4 colspan=1>Influenza A H1(subtype)</td><td rowspan=1 colspan=1>2x</td><td rowspan=1 colspan=1>26/26</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>87%-100%</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td></tr><tr><td rowspan=1 colspan=1>10x</td><td rowspan=1 colspan=1>24/24</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>86%-100%</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td></tr><tr><td rowspan=1 colspan=1>100x</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td></tr><tr><td rowspan=1 colspan=1>Combined</td><td rowspan=1 colspan=1>50/50</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>93%-100%</td><td rowspan=1 colspan=1>149/149</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>97%-100%</td></tr><tr><td rowspan=4 colspan=1>RSV B</td><td rowspan=1 colspan=1>2x</td><td rowspan=1 colspan=1>24/25</td><td rowspan=1 colspan=1>96.0%</td><td rowspan=1 colspan=1>80%-99%</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td></tr><tr><td rowspan=1 colspan=1>10x</td><td rowspan=1 colspan=1>13/13</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>77%-100%</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td></tr><tr><td rowspan=1 colspan=1>100x</td><td rowspan=1 colspan=1>12/12</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>76%-100%</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td></tr><tr><td rowspan=1 colspan=1>Combined</td><td rowspan=1 colspan=1>49/50</td><td rowspan=1 colspan=1>98.0%</td><td rowspan=1 colspan=1>90%-100%</td><td rowspan=1 colspan=1>148/149</td><td rowspan=1 colspan=1>99.3%</td><td rowspan=1 colspan=1>96%-100%</td></tr><tr><td rowspan=1 colspan=1>Analyte</td><td rowspan=1 colspan=1>LoD</td><td rowspan=1 colspan=2>cYanmaGYellowb</td><td rowspan=1 colspan=3>Bacteria</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=4 colspan=1>Chlamydiapneumoniae</td><td rowspan=1 colspan=1>2x</td><td rowspan=1 colspan=1>25/25</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>87%-100%</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td></tr><tr><td rowspan=1 colspan=1>10x</td><td rowspan=1 colspan=1>12/13</td><td rowspan=1 colspan=1>92.3%</td><td rowspan=1 colspan=1>67%-99%</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td></tr><tr><td rowspan=1 colspan=1>100x</td><td rowspan=1 colspan=1>12/12</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>76%-100%</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td></tr><tr><td rowspan=1 colspan=1>Combined</td><td rowspan=1 colspan=1>49/50</td><td rowspan=1 colspan=1>98.0%</td><td rowspan=1 colspan=1>90%-100%</td><td rowspan=1 colspan=1>149/149</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>97%-100%</td></tr></table>

The study results demonstrate that the diagnostic accuracy of the NxTAG® RPP v2 assay is acceptable for the detection and identification of respiratory bacteria and viruses from NPS specimens collected from patients exhibiting clinical signs and symptoms of RTI.

4. Expected values/Reference range:

Table 22: NxTAG® RPP v2 Expected Values for Prospective Specimens by Age   

<table><tr><td colspan="1" rowspan="1"></td><td colspan="2" rowspan="1">0-1 years</td><td colspan="2" rowspan="1">&gt;1-5 years</td><td colspan="2" rowspan="1">&gt;5-21 years</td><td colspan="2" rowspan="1">&gt;21-65 years</td><td colspan="2" rowspan="1">&gt; 65 years</td><td colspan="2" rowspan="1">Unknown</td><td colspan="2" rowspan="1">Overall</td></tr><tr><td colspan="1" rowspan="1">Target (Analyte)</td><td colspan="1" rowspan="1">#Pos</td><td colspan="1" rowspan="1">(%)</td><td colspan="1" rowspan="1">#Pos</td><td colspan="1" rowspan="1">(%)</td><td colspan="1" rowspan="1">#Pos</td><td colspan="1" rowspan="1">(%)</td><td colspan="1" rowspan="1">#Pos</td><td colspan="1" rowspan="1">(%)</td><td colspan="1" rowspan="1">#Pos</td><td colspan="1" rowspan="1">(%)</td><td colspan="1" rowspan="1">#Pos</td><td colspan="1" rowspan="1">(%)</td><td colspan="1" rowspan="1">#Pos</td><td colspan="1" rowspan="1">(%)</td></tr><tr><td colspan="1" rowspan="1">Adenovirus</td><td colspan="1" rowspan="1">34</td><td colspan="1" rowspan="1">10.1%(34/337)</td><td colspan="1" rowspan="1">40</td><td colspan="1" rowspan="1">15.8%(40/253)</td><td colspan="1" rowspan="1">22</td><td colspan="1" rowspan="1">6.7%(22/329)</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">0.8% (5/641)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0%0/22</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0%0/12)</td><td colspan="1" rowspan="1">101</td><td colspan="1" rowspan="1">5.6%(101/1814)</td></tr><tr><td colspan="1" rowspan="1">Chlamydia pneumoniae</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0% (0/337)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0%(0/253)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0%0/329)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0% (0/641)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0%/24)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0%0/12)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0% (0/1814)</td></tr><tr><td colspan="1" rowspan="1">Coronavirus 229E</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0.3% (1/337)</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0.4%(1/253)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0%(0/329)</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">0.8% (5/641)</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0.4%(1/242)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0%/12)</td><td colspan="1" rowspan="1">8</td><td colspan="1" rowspan="1">0.4% (8/1814)</td></tr><tr><td colspan="1" rowspan="1">Coronavirus HKU1</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0.9% (3/337)</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">2.0%253)</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">1.2%(4/329)</td><td colspan="1" rowspan="1">8</td><td colspan="1" rowspan="1">1.2% (8/641)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0%(0/242)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0% 0/12)</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">1.1%(20/1814)</td></tr><tr><td colspan="1" rowspan="1">Coronavirus NL63</td><td colspan="1" rowspan="1">17</td><td colspan="1" rowspan="1">5.0% (17/337)</td><td colspan="1" rowspan="1">12</td><td colspan="1" rowspan="1">4.7%(12/253)</td><td colspan="1" rowspan="1">8</td><td colspan="1" rowspan="1">2.4%(8/329)</td><td colspan="1" rowspan="1">9</td><td colspan="1" rowspan="1">1.4% (9/641)</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0.8%(2/242)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0%(0/12)</td><td colspan="1" rowspan="1">48</td><td colspan="1" rowspan="1">2.6%(48/1814)</td></tr><tr><td colspan="1" rowspan="1">Coronavirus OC43</td><td colspan="1" rowspan="1">16</td><td colspan="1" rowspan="1">4.7% (16/337)</td><td colspan="1" rowspan="1">8</td><td colspan="1" rowspan="1">3.2%(8/253</td><td colspan="1" rowspan="1">7</td><td colspan="1" rowspan="1">2.1%(7/329)</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">0.9% (6/641)</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0.8%(2/242)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0%0/12)</td><td colspan="1" rowspan="1">39</td><td colspan="1" rowspan="1">2.1%(39/1814)</td></tr><tr><td colspan="1" rowspan="1">HumanMetapneumovirus</td><td colspan="1" rowspan="1">45</td><td colspan="1" rowspan="1">13.4%45/337)</td><td colspan="1" rowspan="1">41</td><td colspan="1" rowspan="1">16.2%(41/253)</td><td colspan="1" rowspan="1">29</td><td colspan="1" rowspan="1">8.8%(29/329)</td><td colspan="1" rowspan="1">39</td><td colspan="1" rowspan="1">6.1% (39/641)</td><td colspan="1" rowspan="1">14</td><td colspan="1" rowspan="1">5.8%(14/242)</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">25%(/12)</td><td colspan="1" rowspan="1">171</td><td colspan="1" rowspan="1">9.4%(171/1814)</td></tr><tr><td colspan="1" rowspan="1">Influenza A</td><td colspan="1" rowspan="1">7</td><td colspan="1" rowspan="1">2.1% (7/337)</td><td colspan="1" rowspan="1">8</td><td colspan="1" rowspan="1">3.2%/253)</td><td colspan="1" rowspan="1">25</td><td colspan="1" rowspan="1">7.6%(25/329)</td><td colspan="1" rowspan="1">27</td><td colspan="1" rowspan="1">4.2% (27/641)</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">4.1%(10/242)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0% (0/12)</td><td colspan="1" rowspan="1">77</td><td colspan="1" rowspan="1">4.2%(77/1814)</td></tr><tr><td colspan="1" rowspan="1">Influenza A H1pdm09</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0.9% (3/337)</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">1.2%(3/253)</td><td colspan="1" rowspan="1">8</td><td colspan="1" rowspan="1">2.4%(8/329)</td><td colspan="1" rowspan="1">14</td><td colspan="1" rowspan="1">2.2% (14/641)</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">1.2%(3/242)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0%0/12)</td><td colspan="1" rowspan="1">31</td><td colspan="1" rowspan="1">1.7%(31/1813)</td></tr><tr><td colspan="1" rowspan="1">Influenza A H1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0% (0/337)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0%/253)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0%(0/328)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0% (0/641)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0%0/24)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0%0/12)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0% (0/1813)</td></tr><tr><td colspan="1" rowspan="1">Influenza A H3</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">1.2% (4/337)</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">2.4%(6/253</td><td colspan="1" rowspan="1">18</td><td colspan="1" rowspan="1">5.5%(18/329)</td><td colspan="1" rowspan="1">11</td><td colspan="1" rowspan="1">1.7% (11/641)</td><td colspan="1" rowspan="1">7</td><td colspan="1" rowspan="1">2.9%(7/242)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0%/12)</td><td colspan="1" rowspan="1">46</td><td colspan="1" rowspan="1">2.5%(46/1814)</td></tr><tr><td colspan="1" rowspan="1">Influenza B</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0% (0/337)</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">1.2%3/ /253)</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0.9%(3/329)</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">0.8% (5/641)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0% (/24)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0% (0/12)</td><td colspan="1" rowspan="1">11</td><td colspan="1" rowspan="1">0.6%(11/1814)</td></tr><tr><td colspan="1" rowspan="1">Mycoplasmapneumoniae</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0% (0/337)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0%/253)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0%0/329</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0% (0/641)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0%(0/242)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0%0/12)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0% (0/1814)</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza 1</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">1.8% (6/337)</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">1.2%(3/253)</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0.3%(1/329)</td><td colspan="1" rowspan="1">7</td><td colspan="1" rowspan="1">1.1% (7/641)</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0.4%(1/242)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0% 0/12)</td><td colspan="1" rowspan="1">18</td><td colspan="1" rowspan="1">1.0%(18/1814)</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza 2</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0.3% (1/337)</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0.4%(1/253)</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0.9%(3/329)</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0.2% (1/641)</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">1.2%(3/242)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0% 0/12)</td><td colspan="1" rowspan="1">9</td><td colspan="1" rowspan="1">0.5% (9/1814)</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza 3</td><td colspan="1" rowspan="1">15</td><td colspan="1" rowspan="1">4.5% (15/337)</td><td colspan="1" rowspan="1">16</td><td colspan="1" rowspan="1">6.3%(16/253)</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">1.5%(5/329)</td><td colspan="1" rowspan="1">7</td><td colspan="1" rowspan="1">1.1% (7/641)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0%(0/242)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0%0/12)</td><td colspan="1" rowspan="1">43</td><td colspan="1" rowspan="1">2.4%(43/1814)</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza 4</td><td colspan="1" rowspan="1">7</td><td colspan="1" rowspan="1">2.1% (7/337)</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0.4%/253)</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">1.2%(4/329)</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">0.6% (4/641)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0%0/4)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0% 0/12)</td><td colspan="1" rowspan="1">16</td><td colspan="1" rowspan="1">0.9%(16/1814)</td></tr><tr><td colspan="1" rowspan="1">RSV A</td><td colspan="1" rowspan="1">28</td><td colspan="1" rowspan="1">8.3% (28/337)</td><td colspan="1" rowspan="1">9</td><td colspan="1" rowspan="1">3.6%(9/253)</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0.6%(2/329)</td><td colspan="1" rowspan="1">11</td><td colspan="1" rowspan="1">1.7% (11/641)</td><td colspan="1" rowspan="1">8</td><td colspan="1" rowspan="1">3.3%(8/242)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0%(0/12)</td><td colspan="1" rowspan="1">58</td><td colspan="1" rowspan="1">3.2%(58/1814)</td></tr><tr><td colspan="1" rowspan="1">RSV B</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">3.0% (10/337)</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">1.6%(4/253)</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0.3%(1/329)</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">0.6% (4/641)</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0.4%/242)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0% 0/12)</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">1.1%(20/1814)</td></tr><tr><td colspan="1" rowspan="1">Rhinovirus/ Enterovirus</td><td colspan="1" rowspan="1">113</td><td colspan="1" rowspan="1">33.5%(113/337)</td><td colspan="1" rowspan="1">75</td><td colspan="1" rowspan="1">29.6%(75/253</td><td colspan="1" rowspan="1">75</td><td colspan="1" rowspan="1">22.8%(75/329)</td><td colspan="1" rowspan="1">65</td><td colspan="1" rowspan="1">10.1%(65/641)</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">8.3%(20/24)</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">33.3%/12)</td><td colspan="1" rowspan="1">352</td><td colspan="1" rowspan="1">19.4%(352/1814)</td></tr><tr><td colspan="1" rowspan="1">SARS-CoV-2</td><td colspan="1" rowspan="1">32</td><td colspan="1" rowspan="1">9.5% (32/336)</td><td colspan="1" rowspan="1">12</td><td colspan="1" rowspan="1">4.8%(12/251)</td><td colspan="1" rowspan="1">21</td><td colspan="1" rowspan="1">6.5%(21/325)</td><td colspan="1" rowspan="1">118</td><td colspan="1" rowspan="1">18.4%(118/640)</td><td colspan="1" rowspan="1">56</td><td colspan="1" rowspan="1">23.3%(56/240)</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">9.1%1/1)</td><td colspan="1" rowspan="1">240</td><td colspan="1" rowspan="1">13.3%(240/1803)</td></tr></table>

# Luminex

# N. Proposed Labeling:

The labeling provided in the submission satisfies the requirements of 21 CFR 809.10.

# O. Conclusion:

The submitted information in this premarket notification is complete and supports a substantial equivalence decision.