# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATIONDECISION SUMMARYASSAY AND INSTRUMENT

I Background Information:

# B Applicant

Luminex Molecular Diagnostics, Inc.

# C Proprietary and Established Names

NxTAG Respiratory Pathogen Panel v2 (NxTAG RPP v2)

# D Regulatory Information

<table><tr><td rowspan=1 colspan=1>ProductCode(s)</td><td rowspan=1 colspan=1>Classification</td><td rowspan=1 colspan=1>RegulationSection</td><td rowspan=1 colspan=1>Panel</td></tr><tr><td rowspan=1 colspan=1>QOF</td><td rowspan=1 colspan=1>Class II</td><td rowspan=1 colspan=1>21 CFR 866.3981 - DeviceTo Detect And IdentifyNucleic Acid Targets InRespiratory SpecimensFrom Microbial AgentsThat Cause The SARS-Cov-2 RespiratoryInfection And OtherMicrobial Agents When InA Multi-Target Test</td><td rowspan=1 colspan=1>MI - Microbiology</td></tr></table>

# II Submission/Device Overview:

# A Purpose for Submission:

To obtain a substantial equivalence determination for the NxTAG Respiratory Pathogen Panel v2 (NxTAG RPP v2).

# B Measurand:

The NxTAG RPP v2 detects and identifies nucleic acids from the following pathogens: Adenovirus, Coronavirus 229E, Coronavirus OC43, Coronavirus NL63, Coronavirus HKU1, Human Metapneumovirus, Influenza A virus with subtyping of Influenza A H1, Influenza A

H1pdm09 and Influenza A H3 (reported separately), Influenza B virus, Parainfluenza Virus 1, Parainfluenza Virus 2, Parainfluenza Virus 3, Parainfluenza Virus 4, Respiratory Syncytial Virus A, Respiratory Syncytial Virus B, Rhinovirus/Enterovirus, Severe Acute Respiratory Syndrome (SARS)-Coronavirus-2, Chlamydia pneumoniae and Mycoplasma pneumoniae.

# C Type of Test:

Multiplex nucleic acid assay for use with the MAGPIX Instrument for the qualitative detection of viral and/or bacterial pathogens in nasopharyngeal swab specimens from patients with signs and symptoms of respiratory tract infection.

# III Intended Use/Indications for Use:

A Intended Use(s): See Indications for Use below.

# B Indication(s) for Use:

The NxTAG Respiratory Pathogen Panel v2 (NxTAG RPP v2) is a multiplexed polymerase chain reaction (PCR) test intended for the simultaneous, qualitative detection and identification of multiple respiratory viral and bacterial nucleic acids in nasopharyngeal swab specimens obtained from individuals with signs and symptoms of respiratory tract infection, including COVID-19.

The following organism types and subtypes are identified and differentiated using the NxTAG RPP v2:

<table><tr><td rowspan=1 colspan=2>Viral Targets</td></tr><tr><td rowspan=1 colspan=1>Influenza A</td><td rowspan=1 colspan=1>Coronavirus NL63</td></tr><tr><td rowspan=1 colspan=1>Influenza A H1</td><td rowspan=1 colspan=1>Coronavirus HKU1</td></tr><tr><td rowspan=1 colspan=1>Influenza A H1pdm09</td><td rowspan=1 colspan=1>Human Metapneumovirus</td></tr><tr><td rowspan=1 colspan=1>Influenza A H3</td><td rowspan=1 colspan=1>Rhinovirus/Enterovirus</td></tr><tr><td rowspan=1 colspan=1>Influenza B</td><td rowspan=1 colspan=1>Adenovirus</td></tr><tr><td rowspan=1 colspan=1>Respiratory Syncytial Virus A</td><td rowspan=1 colspan=1>Parainfluenza virus 1</td></tr><tr><td rowspan=1 colspan=1>Respiratory Syncytial Virus B</td><td rowspan=1 colspan=1>Parainfluenza virus 2</td></tr><tr><td rowspan=1 colspan=1>SARS-CoV-2</td><td rowspan=1 colspan=1>Parainfluenza virus 3</td></tr><tr><td rowspan=1 colspan=1>Coronavirus 229E</td><td rowspan=1 colspan=1>Parainfluenza virus 4</td></tr><tr><td rowspan=1 colspan=1>Coronavirus OC43</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=2>Bacterial Targets</td></tr><tr><td rowspan=1 colspan=1>Chlamydia pneumoniae</td><td rowspan=1 colspan=1>Mycoplasma pneumoniae</td></tr></table>

Nucleic acids from the viral and bacterial organisms identified by this test are generally detectable in nasopharyngeal specimens during the acute phase of infection. The detection and identification of specific viral and bacterial nucleic acids from individuals exhibiting signs and/or symptoms of respiratory infection are indicative of the presence of the identified microorganism and aids in diagnosis if used in conjunction with other clinical and epidemiological information, and laboratory findings. The results of this test should not be used as the sole basis for diagnosis, treatment, or other patient management decisions.

Negative results in the setting of a respiratory illness may be due to infection with pathogens that are not detected by this test, or lower respiratory tract infection that may not be detected by a nasopharyngeal swab specimen. Positive results do not rule out coinfection with other organisms. The agent(s) detected by the NxTAG RPP v2 may not be the definite cause of disease.

Additional laboratory testing (e.g., bacterial and viral culture, immunofluorescence, and radiography) may be necessary when evaluating a patient with possible respiratory tract infection.

The NxTAG Respiratory Pathogen Panel v2 is indicated for use with the Luminex MAGPIX Instrument and xPONENT and SYNCT software.

C Special Conditions for Use Statement(s): Rx - For Prescription Use Only

For in vitro diagnostic use

D Special Instrument Requirements:

Luminex MAGPIX Instrument   
bioMérieux NUCLISENS easyMAG Instrument or EMAG Instrument   
IVD-Labeled Thermal Cycler

# IV Device/System Characteristics:

# A Device Description:

The NxTAG Respiratory Pathogen Panel v2 (NxTAG RPP v2) is a qualitative reversetranscription polymerase chain reaction (RT-PCR) in vitro diagnostic assay for the simultaneous detection and discrimination of 21 viral and bacterial respiratory pathogens in nasopharyngeal swab specimens from patients with signs and symptoms of respiratory tract infection. The assay uses the proprietary Luminex bead hybridization tag sorting system to discriminate amplified products and is performed on the Luminex MAGPIX Instrument equipped with xPONENT software following nucleic acid extraction and PCR amplification using off-the-shelf instrument systems. Signals obtained with the MAGPIX Instrument are analyzed using the NxTAG RPP v2 Assay File for SYNCT Software.

The NxTAG RPP v2 differs from the original NxTAG Respiratory Pathogen Panel (NxTAG RPP; K152386/K193167) as shown in Table 1. Other minor differences include changes to the fluorescence thresholds for certain assays and the Internal Control.

Table 1. Changes to the NxTAG RPP implemented for the NxTAG RPP v2   

<table><tr><td rowspan=1 colspan=1>Target Analyte</td><td rowspan=1 colspan=1>Description of Change for NxTAG RPP v2</td></tr><tr><td rowspan=1 colspan=1>Influenza A</td><td rowspan=1 colspan=1>Modified primers for improved inclusivity</td></tr><tr><td rowspan=1 colspan=1>Influenza A H1</td><td rowspan=1 colspan=1>Differentiation of subtypes H1 and H1pdm09</td></tr><tr><td rowspan=1 colspan=1>Influenza A H3</td><td rowspan=1 colspan=1>Modified primers for improved inclusivity</td></tr><tr><td rowspan=1 colspan=1>Human Bocavirus</td><td rowspan=1 colspan=1>Results not reported</td></tr><tr><td rowspan=1 colspan=1>Human Metapneumovirus</td><td rowspan=1 colspan=1>New target region for improved inclusivity</td></tr><tr><td rowspan=1 colspan=1>Rhinovirus/Enterovirus</td><td rowspan=1 colspan=1>Modified formulation for improved specificity</td></tr><tr><td rowspan=1 colspan=1>SARS-CoV-2</td><td rowspan=1 colspan=1>New analyte</td></tr></table>

# B Principle of Operation:

Nucleic acids are extracted for analysis from nasopharyngeal swab samples in transport medium using either the IVD-labeled bioMérieux NucliSENS easyMAG or EMAG extraction systems. The extracted nucleic acids are added to 96-well plates containing lyophilized PCR primers, hybridization beads and amplification enzymes. Following rehydration, the plates are sealed and loaded into a thermocycler for reverse transcription and PCR amplification. The amplified PCR products hybridize to the tagged beads in near real-time and upon completion of the reaction, the sealed plate is loaded into the MAGPIX Instrument for automated bead sorting and fluorescence reading using xPONENT $4 . 3 { \mathrm { u } } 2$ software. The data are analyzed using the NxTAG RPP v2 Assay File for SYNCT $1 . 1 \mathrm { u } 2$ software and results for each analyte are reported as “detected”, “not detected” or “invalid”. Details of the method of result interpretation for influenza A and its associated subtypes are provided in Table 2.

Table 2. Method of result interpretation for influenza A   

<table><tr><td rowspan=2 colspan=1>Final Result</td><td rowspan=1 colspan=4>NxTAG RPP v2 Analyte</td><td rowspan=2 colspan=1>ApplicableFootnotes</td></tr><tr><td rowspan=1 colspan=1>Influenza A</td><td rowspan=1 colspan=1>Influenza A H1</td><td rowspan=1 colspan=1>Influenza AH1pdm09</td><td rowspan=1 colspan=1>Influenza A H3</td></tr><tr><td rowspan=1 colspan=1>Influenza ANot Detected</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=2 colspan=1>Influenza A H1</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=2 colspan=1>Influenza A2009 H1N1</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=2 colspan=1>Influenza A H1 andInfluenza A H1pdm09</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>2</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>1,2</td></tr><tr><td rowspan=2 colspan=1>Influenza A H3</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>--</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=2 colspan=1>Influenza A H3 andInfluenza A H1</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>2</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>1,2</td></tr><tr><td rowspan=2 colspan=1>Influenza A H3 andInfluenza A H1pdm09</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>2</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>1,2</td></tr><tr><td rowspan=2 colspan=1>Influenza A H1,Influenza A H1pdm09 andInfluenza A H3</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>2</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>Influenza A(no sub-type detected)</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>3</td></tr></table>

Detection of influenza A H1, A H1pdm09 or A H3 subtypes without a corresponding positive result for influenza A may occur at low target levels, be due to contamination or be indicative of a mutation in the matrix gene target region used for detection of influenza A. Retesting is recommended prior to reporting test results if clinically indicated or for epidemiological investigation. Further investigation may be warranted if the same result is generated upon retesting.

2 Co-detection of multiple influenza A subtypes (H1 and/or H1pdm09 and/or H3) may be due to coinfection, false-positive results due to contamination or the presence of a multi-valent influenza virus vaccine in the sample. Retesting is recommended prior to reporting test results. If the same result is obtained upon retest, the presence of multiple subtypes should be confirmed by alternative FDA-cleared methods.

3 Detection of influenza A without a corresponding positive result for an influenza A subtype may occur at low target levels, be due to contamination or be indicative of the presence of a novel influenza A strain. If a specimen is confirmed positive for influenza A on retesting but produces negative test results for A H1, A H1pdm09 and A H3, the appropriate local, state, or federal public health authorities should be notified to determine necessary measures for verification and to determine whether the specimen contains a novel strain of Influenza A.

# C Instrument Description Information:

1. Instrument Name:

Luminex MAGPIX Instrument   
bioMérieux NUCLISENS easyMAG Instrument or EMAG Instrument   
IVD-labeled thermal cycler

2. Specimen Identification:

Specimen identification numbers may be entered into the SYNCT Software manually or using the optional barcode reader.

3. Specimen Sampling and Handling:

The NxTAG RPP v2 assay is intended for use with nasopharyngeal swabs collected in Universal Transport Medium (UTM, Copan Diagnostics) or MicroTest M4RT medium (Remel) using a nylon flocked or polyester swab. Specimens may be stored for up to 7 days at $2 { - } 8 ~ ^ { \circ } \mathrm { C }$ or up to 12 months at $\le - 7 0 { } ^ { \circ } \mathrm { C }$ .

The respective locations of Positive and Negative External Controls and patient samples on the MAGPIX assay plate must be defined in the SYNCT Software. For controls, the expected results for each analyte must also be assigned. Each sample and control within a run must have a unique identifier. The locations of the External Controls should be selected to enable correct orientation of the MAGPIX plate.

The SYNCT software may be used to select which test results to report for each sample prior to analysis based on the test order. Masked results will not be reported. A default Test Panel that includes all available analytes is provided but, for convenience, users also have the option to create custom Test Panels from within the list of available NxTAG RPP v2 target analytes. Targets that were previously masked, may be unmasked by reanalyzing the data in the SYNCT software upon receipt of an appropriate test order.

# 4. Calibration:

Calibration of the MAGPIX Instrument using the MAGPIX Calibration Kit is required at least weekly to normalize the settings for the classification and reporter optical channels. Following calibration, the MAGPIX Performance Verification Kit must be used to verify the integrity of the system. Daily use of the MAGPIX Performance Verification Kit to verify the calibration and optical integrity of the MAGPIX system and the fluidics channels is also recommended. The MAGPIX Calibration Kit and MAGPIX Performance Verification Kit are provided separately from the NxTAG RPP v2.

# 5. Quality Control:

Prior to processing, bacteriophage MS2 is added to each patient sample or External Control as an Internal Control to monitor the integrity of nucleic acid extraction, reverse transcription, PCR amplification and detection. External Positive Controls for use with the

NxTAG RPP v2 assay are not provided with the assay kit but are recommended for use in accordance with local, state and/or federal guidelines and good laboratory practices.

Luminex recommends inclusion of a Negative Extraction Control (transport medium) to monitor for contamination. Optionally, a Negative Amplification Control comprised of RNase-free water may be included on each amplification plate to monitor for contamination downstream from the nucleic acid extraction process. The Negative Extraction Control and Negative Amplification Control should produce negative test results for all analytes.

If the External Positive Control, Negative Amplification Control and Negative Extraction Control do not produce the expected results, patient results should not be reported.

# V Substantial Equivalence Information:

A Predicate Device Name(s): BioFire Respiratory Panel 2.1 (RP2.1)

B Predicate 510(k) Number(s): DEN200031

C Comparison with Predicate(s):

<table><tr><td rowspan=1 colspan=1>Device &amp; PredicateDevice(s):</td><td rowspan=1 colspan=1>K231758</td><td rowspan=1 colspan=1>DEN200031</td></tr><tr><td rowspan=1 colspan=1>Device Trade Name</td><td rowspan=1 colspan=1>NxTAG RespiratoryPathogen Panel v2</td><td rowspan=1 colspan=1>BioFire Respiratory Panel 2.1</td></tr><tr><td rowspan=1 colspan=1>General DeviceCharacteristic Similarities</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Intended Use/Indications ForUse</td><td rowspan=1 colspan=1>The NxTAG RespiratoryPathogen Panel v2 (NxTAGRPP v2) is a multiplexedpolymerase chain reaction(PCR) test intended for thesimultaneous, qualitativedetection and identificationof multiple respiratory viraland bacterial nucleic acids innasopharyngeal swabspecimens obtained fromindividuals with signs andsymptoms of respiratory tractinfection, including COVID-19.The following organismtypes and subtypes are</td><td rowspan=1 colspan=1>The BioFire RespiratoryPanel 2.1 (RP2.1) is a PCR-based multiplexed nucleicacid test intended for usewith the BioFire FilmArray2.0 or BioFire FilmArrayTorch systems for thesimultaneous qualitativedetection and identificationof multiple respiratory viraland bacterial nucleic acids innasopharyngeal swabs (NPS)obtained from individualssuspected of respiratory tractinfections, including COVID-19.The following organismtypes and subtypes are</td></tr></table>

<table><tr><td rowspan="18">identified and differentiated identified using the BioFire</td></tr><tr><td>using the NxTAG RPP v2: RP2.1 Adenovirus</td></tr><tr><td>Viral targets: Coronavirus 229E</td></tr><tr><td>Influenza A Influenza A H1 Coronavirus HKU1</td></tr><tr><td></td></tr><tr><td>Influenza A H1pdm09 Coronavirus NL63</td></tr><tr><td>Influenza A H3 Coronavirus OC43</td></tr><tr><td>Influenza B Severe Acute Respiratory Syndrome Coronavirus Respiratory Syncytial Virus</td></tr><tr><td>(SARS-CoV-2) A</td></tr><tr><td>Human Metapneumovirus Respiratory Syncytial Virus B Human</td></tr><tr><td>Rhinovirus/Enterovirus SARS-CoV-2</td></tr><tr><td>Influenza A, including Coronavirus 229E subtypes H1, H1-2009, and</td></tr><tr><td>Coronavirus OC43 H3</td></tr><tr><td>Coronavirus NL63 Influenza B</td></tr><tr><td>Coronavirus HKU1 Parainfluenza Virus 1</td></tr><tr><td>Human Metapneumovirus Parainfluenza Virus 2</td></tr><tr><td>Rhinovirus/Enterovirus Parainfluenza Virus 3</td></tr><tr><td>Adenovirus Parainfluenza Virus 4</td></tr><tr><td>Parainfluenza Virus 1 Respiratory Syncytial Virus</td></tr><tr><td>Parainfluenza Virus 2 Bordetella parapertussis</td></tr><tr><td>(IS1001) Parainfluenza Virus 3</td></tr><tr><td>Bordetella pertussis (prxP) Parainfluenza Virus 4</td></tr><tr><td>Chlamydia pneumoniae and Bacterial targets:</td></tr><tr><td>Chlamydia pneumoniae Mycoplasma pneumoniae</td></tr><tr><td>Nucleic acids from the Mycoplasma pneumoniae respiratory viral and bacterial Nucleic acids from the viral organisms identified by this</td></tr><tr><td></td></tr><tr><td>and bacterial organisms test are generally detectable identified by this test are</td></tr><tr><td>in NPS specimens during the generally detectable in acute phase of infection. The detection and identification</td></tr><tr><td>nasopharyngeal specimens during the acute phase of</td></tr><tr><td>of specific viral and bacterial</td></tr><tr><td>infection. The detection and nucleic acids from identification of specific viral individuals exhibiting signs</td></tr></table>

<table><tr><td colspan="1" rowspan="5"></td><td colspan="6" rowspan="5">information, and laboratoryfindings. The results of thistest should not be used as thesole basis for diagnosis,treatment, or other patientmanagement decisions.Negative results in the settingof a respiratory illness maybe due to infection withpathogens that are notdetected by this test, or lowerrespiratory tract infection thatmay not be detected by anasopharyngeal swabspecimen. Positive results donot rule out coinfection withother organisms. The agent(s)detected by the NxTAG RPPv2 may not be the definitecause of disease.Additional laboratory testing(e.g., bacterial and viralculture, immunofluorescence,and radiography) may benecessary when evaluating apatient with possiblerespiratory tract infection.The NxTAG RespiratoryPathogen Panel v2 isindicated for use with theLuminex MAGPIXInstrument and xPONENTand SYNCT software.</td><td colspan="1" rowspan="5">this test should not be used asthe sole basis for diagnosis,treatment, or other patientmanagement decisions.Negative results in the settingof a respiratory illness maybe due to infection withpathogens that are notdetected by this test, or lowerrespiratory tract infection thatmay not be detected by anNPS specimen. Positiveresults do not rule outcoinfection with otherorganisms. The agent(s)detected by the BioFireRP2.1 may not be the definitecause of disease. Additionallaboratory testing (e.g.bacterial and viral culture,immunofluorescence, andradiography) may benecessary when evaluating apatient with possiblerespiratory tract infection.</td></tr><tr><td colspan="2" rowspan="2">pathogens that are not</td><td colspan="2" rowspan="1">that</td></tr><tr><td colspan="3" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">t1</td></tr><tr><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Specimen Type</td><td colspan="6" rowspan="1">Same</td><td colspan="1" rowspan="1">Nasopharyngeal swabs</td></tr><tr><td colspan="1" rowspan="1">Patient Population</td><td colspan="6" rowspan="1">Individuals with signs andsymptoms of respiratory tractinfection, including COVID-19</td><td colspan="1" rowspan="1">Individuals suspected ofrespiratory tract infections,including COVID-19</td></tr><tr><td colspan="1" rowspan="1">Organisms Detected</td><td colspan="6" rowspan="1">Same except for:</td><td colspan="1" rowspan="1">Viruses:Adenovirus</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">adifferentiation ofRespiratory SyncytialVirus A and B) omission of assays forBordetella pertussis andBordetella parapertussis</td><td colspan="1" rowspan="1">Coronavirus 229ECoronavirus HKU1Coronavirus NL63Coronavirus OC43Severe Acute RespiratorySyndrome Coronavirus 2(SARS-CoV-2)Human MetapneumovirusHumanRhinovirus/EnterovirusInfluenza A VirusSubtypes: H1, H3, H1-2009Influenza B VirusParainfluenza Virus 1Parainfluenza Virus 2Parainfluenza Virus 3Parainfluenza Virus 4Respiratory Syncytial VirusBacteria:Bordetella parapertussisBordetella pertussisChlamydia pneumoniaeMycoplasma pneumoniae</td></tr><tr><td colspan="1" rowspan="1">Technology</td><td colspan="1" rowspan="1">Same</td><td colspan="1" rowspan="1">PCR amplification</td></tr><tr><td colspan="1" rowspan="1">General DeviceCharacteristic Differences</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">System</td><td colspan="1" rowspan="1">Separate instruments fornucleic acid extraction, PCRamplification and detection</td><td colspan="1" rowspan="1">Integrated nucleic acidextraction, amplification anddetection in a sealed vessel.</td></tr><tr><td colspan="1" rowspan="1">Assay Read</td><td colspan="1" rowspan="1">MAGPIX Instrument</td><td colspan="1" rowspan="1">BIOFIRE FilmArray 2.0 orBIOFIRE FilmArray TorchSystems</td></tr><tr><td colspan="1" rowspan="1">Detection</td><td colspan="1" rowspan="1">Hybridization of amplifiedproducts with fluorescentlylabeled beads, sorting oftagged products</td><td colspan="1" rowspan="1">Array-based melt curveanalysis</td></tr></table>

# VI Standards/Guidance Documents Referenced:

CLSI. Evaluation of Precision of Quantitative Measurement Procedures; Approved Guideline.   
$3 ^ { \mathrm { r d } }$ ed. CLSI document EP05-A3. Wayne, PA: Clinical and Laboratory Standards Institute; 2014.

CLSI. Interference Testing in Clinical Chemistry. 3rd ed. CLSI guideline EP07. Wayne, PA: Clinical and Laboratory Standards Institute; 2018.

CLSI. User Protocol for Evaluation of Qualitative Test Performance; Approved Guideline. $2 ^ { \mathrm { n d } }$   
ed. CLSI document EP12-A2. Wayne, PA: Clinical and Laboratory Standards Institute; 2008.

CLSI. Evaluation of Detection Capability for Clinical Laboratory Measurement Procedures; Approved Guideline. $2 ^ { \mathrm { n d } }$ ed. CLSI document EP17-A2. Wayne, PA: Clinical and Laboratory Standards Institute; 2012.

CLSI. Assessment of the Diagnostic Accuracy of Laboratory Tests Using Receiver Operating Characteristic Curves; Approved Guideline. $2 ^ { \mathrm { n d } }$ ed. CLSI document EP24-A2. Wayne, PA: Clinical and Laboratory Standards Institute; 2011.

CLSI. Evaluation of Stability of In Vitro Diagnostic Reagents; Approved Guideline. CLSI document EP25-A. Wayne, PA: Clinical and Laboratory Standards Institute; 2009.

CLSI. Supplementary Tables for Interference Testing in Clinical Chemistry. $1 ^ { \mathrm { s t } }$ ed. CLSI supplement EP37. Wayne, PA: Clinical and Laboratory Standards Institute; 2018.

ISO 14971: 2019. Medical devices – Application of risk management to medical devices.

BS EN ISO 23640: 2015. In vitro diagnostic medical devices – Evaluation of stability of in vitro diagnostic reagents.

CLSI. Molecular Diagnostic Methods for Infectious Diseases. 3rd ed. CLSI report MM03.   
Wayne, PA: Clinical and Laboratory Standards Institute; 2015.

CLSI. Nucleic Acid Sequencing Methods in Diagnostic Laboratory Medicine; Approved Guideline. $2 ^ { \mathrm { n d } }$ ed. CLSI document MM09-A2. Wayne, PA: Clinical and Laboratory Standards Institute; 2014.

CLSI. Validation and Verification of Multiplex Nucleic Acid Assays. $2 ^ { \mathrm { n d } }$ ed. CLSI guideline MM17. Wayne, PA: Clinical and Laboratory Standards Institute; 2018.

CLSI. Interpretive Criteria for Identification of Bacteria and Fungi by DNA Target Sequencing; Approved Guideline. CLSI document MM18-A. Wayne, PA: Clinical and Laboratory Standards Institute; 2008.

# VII Performance Characteristics (if/when applicable):

# A Analytical Performance:

1. Precision/Reproducibility:

Lot-to-Lot and Within-Run Precision The lot-to-lot precision of the NxTAG RPP v2 was evaluated using three lots of assay kits, each containing unique lots of critical raw materials/sub-components. The panel of samples for analysis comprised simulated nasopharyngeal swab matrix that was spiked with different combinations of all the NxTAG RPP v2 analytes, as well as a negative sample (Table 3). Please refer to Section VII A (6) for data supporting use of simulated nasopharyngeal swab matrix for Analytical Studies, as well as testing of samples containing multiple analytes.

Each positive panel member was prepared at both low and moderate target levels (1.5X [low] and 5X LoD [moderate], respectively, except where noted). Each panel member at each

concentration was tested a total of 10 times with each lot of reagents (total 30 replicates per panel member per target level). There was close to $100 \%$ positive and negative agreement with the expected results for each panel member at each target level with all three lots of reagents (Table 4). These results are acceptable.

Table 3. Panel members used to evaluate lot-to-lot assay precision   

<table><tr><td rowspan=1 colspan=1>Sample</td><td rowspan=1 colspan=1>Analyte</td><td rowspan=1 colspan=1>Strain/Isolate</td><td rowspan=1 colspan=1>Source</td><td rowspan=1 colspan=1>Catalogue #</td></tr><tr><td rowspan=3 colspan=1>1</td><td rowspan=1 colspan=1>Influenza A H1pdm09 (subtype) 1, 2,3</td><td rowspan=1 colspan=1>A/NY/02/09</td><td rowspan=1 colspan=1>ZeptoMetrix</td><td rowspan=1 colspan=1>0810109CFN</td></tr><tr><td rowspan=1 colspan=1>Respiratory Syncytial Virus A</td><td rowspan=1 colspan=1>A2</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>VR-1540</td></tr><tr><td rowspan=1 colspan=1>Rhinovirus</td><td rowspan=1 colspan=1>50-525-CV54</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>VR-1195</td></tr><tr><td rowspan=2 colspan=1>2</td><td rowspan=1 colspan=1>Influenza A H3 (subtype) 1,4</td><td rowspan=1 colspan=1>Wisconsin/67/05</td><td rowspan=1 colspan=1>ZeptoMetrix</td><td rowspan=1 colspan=1>0810252CF</td></tr><tr><td rowspan=1 colspan=1>Respiratory Syncytial Virus B</td><td rowspan=1 colspan=1>18537</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>VR-1580</td></tr><tr><td rowspan=3 colspan=1>3</td><td rowspan=1 colspan=1>Influenza B</td><td rowspan=1 colspan=1>B/Florida/02/06</td><td rowspan=1 colspan=1>ZeptoMetrix</td><td rowspan=1 colspan=1>0810037CF</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza Virus 3</td><td rowspan=1 colspan=1>C 243</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>VR-93</td></tr><tr><td rowspan=1 colspan=1>Mycoplasma pneumoniae 5</td><td rowspan=1 colspan=1>M129</td><td rowspan=1 colspan=1>ZeptoMetrix</td><td rowspan=1 colspan=1>0801579</td></tr><tr><td rowspan=3 colspan=1>4</td><td rowspan=1 colspan=1>SARS-CoV-2</td><td rowspan=1 colspan=1>USA-WA1/2020</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>VR-1986HK</td></tr><tr><td rowspan=1 colspan=1>Human Metapneumovirus</td><td rowspan=1 colspan=1>hMPV-3, Type B1, Peru2-2002</td><td rowspan=1 colspan=1>ZeptoMetrix</td><td rowspan=1 colspan=1>0810156CF</td></tr><tr><td rowspan=1 colspan=1>Adenovirus B 2</td><td rowspan=1 colspan=1>Type 14, 2006 isolate</td><td rowspan=1 colspan=1>ZeptoMetrix</td><td rowspan=1 colspan=1>0810108CF</td></tr><tr><td rowspan=3 colspan=1>5</td><td rowspan=1 colspan=1>Influenza A H3 (matrix)</td><td rowspan=1 colspan=1>Wisconsin/67/05</td><td rowspan=1 colspan=1>ZeptoMetrix</td><td rowspan=1 colspan=1>0810252CF</td></tr><tr><td rowspan=1 colspan=1>Coronavirus NL63</td><td rowspan=1 colspan=1>--</td><td rowspan=1 colspan=1>ZeptoMetrix</td><td rowspan=1 colspan=1>0810228CF</td></tr><tr><td rowspan=1 colspan=1>Coronavirus HKU1</td><td rowspan=1 colspan=1>Genotype B</td><td rowspan=1 colspan=1>Clinical</td><td rowspan=1 colspan=1>--</td></tr><tr><td rowspan=3 colspan=1>6</td><td rowspan=1 colspan=1>Influenza A H1 (subtype) 1, 6</td><td rowspan=1 colspan=1>A/Brisbane/59/07</td><td rowspan=1 colspan=1>ZeptoMetrix</td><td rowspan=1 colspan=1>0810244CF</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza Virus 1</td><td rowspan=1 colspan=1>--</td><td rowspan=1 colspan=1>ZeptoMetrix</td><td rowspan=1 colspan=1>0810014CF</td></tr><tr><td rowspan=1 colspan=1>Chlamydia pneumoniae</td><td rowspan=1 colspan=1>TW-183</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>VR-2282</td></tr><tr><td rowspan=3 colspan=1>7</td><td rowspan=1 colspan=1>Parainfluenza Virus 2</td><td rowspan=1 colspan=1>Greer</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>VR-92</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza Virus 4B</td><td rowspan=1 colspan=1>CH 19503</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>VR-1377</td></tr><tr><td rowspan=1 colspan=1>Coronavirus 229E</td><td rowspan=1 colspan=1>--</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>VR-740</td></tr><tr><td rowspan=2 colspan=1>8</td><td rowspan=1 colspan=1>Parainfluenza Virus 4A</td><td rowspan=1 colspan=1>--</td><td rowspan=1 colspan=1>ZeptoMetrix</td><td rowspan=1 colspan=1>0810060CF</td></tr><tr><td rowspan=1 colspan=1>Coronavirus OC43</td><td rowspan=1 colspan=1>Betacoronavirus 1</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>VR-1558</td></tr><tr><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>--</td><td rowspan=1 colspan=1>--</td><td rowspan=1 colspan=1>--</td></tr></table>

Samples 1-8 were formulated at $1 . 5 \mathrm { X }$ (Low) and 5X LoD (Moderate), except where noted. 1 Formulated based on the LoD for the applicable influenza A subtype. 2 Formulated at approximately 3X and 9X LoD. 3 Expected results at the Low target level based on the respective analytical sensitivities of the H1pdm09 subtyping and influenza A assays: H1pdm09 subtype “positive”, Influenza A “positive”. 4 Expected results at the Low target level based on the respective analytical sensitivities of the H3 subtyping and influenza A assays: H3 subtype “positive”, Influenza A “positive” or “negative”. 5 Formulated at approximately 2X and 7X LoD. 6 Expected results at the Low target level based on the respective analytical sensitivities of the H1 subtyping and influenza A assays: H1 subtype “positive”, Influenza A “positive”.

Table 4. Lot-to-lot agreement with expected results   

<table><tr><td colspan="1" rowspan="2">Sample</td><td colspan="1" rowspan="2">NxTAG RPP v2 Analyte</td><td colspan="1" rowspan="2">Level</td><td colspan="5" rowspan="1">Percent Agreement</td></tr><tr><td colspan="1" rowspan="1">Lot 1</td><td colspan="1" rowspan="1">Lot 2</td><td colspan="1" rowspan="1">Lot 3</td><td colspan="1" rowspan="1">Total</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="8">1</td><td colspan="1" rowspan="2">Influenza A H1pdm09(subtype) 1.2</td><td colspan="1" rowspan="1">Low 3</td><td colspan="1" rowspan="1">100 (10/10)</td><td colspan="1" rowspan="1">100 (10/10)</td><td colspan="1" rowspan="1">100 (10/10)</td><td colspan="1" rowspan="1">100 (30/30)</td><td colspan="1" rowspan="2">100 (60/60)</td></tr><tr><td colspan="1" rowspan="1">Mod</td><td colspan="1" rowspan="1">100 (10/10)</td><td colspan="1" rowspan="1">100 (10/10)</td><td colspan="1" rowspan="1">100 (10/10)</td><td colspan="1" rowspan="1">100 (30/30)</td><td colspan="1" rowspan="1">−</td></tr><tr><td colspan="1" rowspan="2">Respiratory Syncytial Virus A</td><td colspan="1" rowspan="1">Low</td><td colspan="1" rowspan="1">100 (10/10)</td><td colspan="1" rowspan="1">100 (10/10)</td><td colspan="1" rowspan="1">100 (10/10)</td><td colspan="1" rowspan="1">100 (30/30)</td><td colspan="1" rowspan="2">100 (60/60)</td></tr><tr><td colspan="1" rowspan="1">Mod</td><td colspan="1" rowspan="1">100 (10/10)</td><td colspan="1" rowspan="1">100 (10/10)</td><td colspan="1" rowspan="1">100 (10/10)</td><td colspan="1" rowspan="1">100 (30/30)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="2">Rhinovirus/Enterovirus</td><td colspan="1" rowspan="1">Low</td><td colspan="1" rowspan="1">100 (10/10)</td><td colspan="1" rowspan="1">100 (10/10)</td><td colspan="1" rowspan="1">100 (10/10)</td><td colspan="1" rowspan="1">100 (30/30)</td><td colspan="1" rowspan="2">100 (60/60)</td></tr><tr><td colspan="1" rowspan="1">Mod</td><td colspan="1" rowspan="1">100 (10/10)</td><td colspan="1" rowspan="1">100 (10/10)</td><td colspan="1" rowspan="1">100 (10/10)</td><td colspan="1" rowspan="1">100 (30/30)</td></tr><tr><td colspan="1" rowspan="2">All Other</td><td colspan="1" rowspan="1">Low</td><td colspan="1" rowspan="1">100 (180/180)</td><td colspan="1" rowspan="1">100 (180/180)</td><td colspan="1" rowspan="1">100 (180/180)</td><td colspan="1" rowspan="1">100 (540/540)</td><td colspan="1" rowspan="2">100(1080/1080)</td></tr><tr><td colspan="1" rowspan="1">Mod</td><td colspan="1" rowspan="1">100 (180/180)</td><td colspan="1" rowspan="1">100 (180/180)</td><td colspan="1" rowspan="1">100 (180/180)</td><td colspan="1" rowspan="1">100 (540/540)</td></tr><tr><td colspan="1" rowspan="6">2</td><td colspan="1" rowspan="2">Influenza A H3 (subtype) 1</td><td colspan="1" rowspan="1">Low 4</td><td colspan="1" rowspan="1">100 (10/10)</td><td colspan="1" rowspan="1">100 (10/10)</td><td colspan="1" rowspan="1">100 (10/10)</td><td colspan="1" rowspan="1">100 (30/30)</td><td colspan="1" rowspan="2">100 (60/60)</td></tr><tr><td colspan="1" rowspan="1">Mod</td><td colspan="1" rowspan="1">100 (10/10)</td><td colspan="1" rowspan="1">100 (10/10)</td><td colspan="1" rowspan="1">100 (10/10)</td><td colspan="1" rowspan="1">100 (30/30)</td></tr><tr><td colspan="1" rowspan="2">Respiratory Syncytial Virus B</td><td colspan="1" rowspan="1">Low</td><td colspan="1" rowspan="1">100 (10/10)</td><td colspan="1" rowspan="1">100 (10/10)</td><td colspan="1" rowspan="1">100 (10/10)</td><td colspan="1" rowspan="1">100 (30/30)</td><td colspan="1" rowspan="2">100 (60/60)</td></tr><tr><td colspan="1" rowspan="1">Mod</td><td colspan="1" rowspan="1">100 (10/10)</td><td colspan="1" rowspan="1">100 (10/10)</td><td colspan="1" rowspan="1">100 (10/10)</td><td colspan="1" rowspan="1">100 (30/30)</td></tr><tr><td colspan="1" rowspan="2">All Other</td><td colspan="1" rowspan="1">Low</td><td colspan="1" rowspan="1">100 (190/190)</td><td colspan="1" rowspan="1">100 (190/190)</td><td colspan="1" rowspan="1">100 (190/190)</td><td colspan="1" rowspan="1">100 (570/570)</td><td colspan="1" rowspan="2">100(1140/1140)</td></tr><tr><td colspan="1" rowspan="1">Mod</td><td colspan="1" rowspan="1">100 (190/190)</td><td colspan="1" rowspan="1">100 (190/190)</td><td colspan="1" rowspan="1">100 (190/190)</td><td colspan="1" rowspan="1">100 (570/570)</td></tr><tr><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">Influenza B</td><td colspan="1" rowspan="1">Low</td><td colspan="1" rowspan="1">100 (10/10)</td><td colspan="1" rowspan="1">100 (10/10)</td><td colspan="1" rowspan="1">100 (10/10)</td><td colspan="1" rowspan="1">100 (30/30)</td><td colspan="1" rowspan="1">100 (60/60)</td></tr><tr><td colspan="1" rowspan="7">b</td><td colspan="1" rowspan="1">EY</td><td colspan="1" rowspan="1">Mod</td><td colspan="1" rowspan="1">100 (10/10)</td><td colspan="1" rowspan="1">100 (10/10)</td><td colspan="1" rowspan="1">100 (10/10)</td><td colspan="1" rowspan="1">100 (30/30)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="2">Parainfluenza virus 3</td><td colspan="1" rowspan="1">Low</td><td colspan="1" rowspan="1">100 (10/10)</td><td colspan="1" rowspan="1">100 (10/10)</td><td colspan="1" rowspan="1">100 (10/10)</td><td colspan="1" rowspan="1">100 (30/30)</td><td colspan="1" rowspan="2">100 (60/60)</td></tr><tr><td colspan="1" rowspan="1">Mod</td><td colspan="1" rowspan="1">100 (10/10)</td><td colspan="1" rowspan="1">100 (10/10)</td><td colspan="1" rowspan="1">100 (10/10)</td><td colspan="1" rowspan="1">100 (30/30)</td></tr><tr><td colspan="1" rowspan="2">Mycoplasma pneumoniae 5</td><td colspan="1" rowspan="1">Low</td><td colspan="1" rowspan="1">100 (10/10)</td><td colspan="1" rowspan="1">1000 (10/10)</td><td colspan="1" rowspan="1">100 (10/10)</td><td colspan="1" rowspan="1">100 (30/30)</td><td colspan="1" rowspan="2">100 (60/60)</td></tr><tr><td colspan="1" rowspan="1">Mod</td><td colspan="1" rowspan="1">100 (10/10)</td><td colspan="1" rowspan="1">100 (10/10)</td><td colspan="1" rowspan="1">100 (10/10)</td><td colspan="1" rowspan="1">100 (30/30)</td></tr><tr><td colspan="1" rowspan="2">All Other</td><td colspan="1" rowspan="1">Low</td><td colspan="1" rowspan="1">100 (180/180)</td><td colspan="1" rowspan="1">100 (180/180)</td><td colspan="1" rowspan="1">100 (180/180)</td><td colspan="1" rowspan="1">100 (540/540)</td><td colspan="1" rowspan="2">100(1080/1080)</td></tr><tr><td colspan="1" rowspan="1">Mod</td><td colspan="1" rowspan="1">100 (180/180)</td><td colspan="1" rowspan="1">100 (180/180)</td><td colspan="1" rowspan="1">100 (180/180)</td><td colspan="1" rowspan="1">100 (540/540)</td></tr><tr><td colspan="1" rowspan="8">4</td><td colspan="1" rowspan="2">SARS-CoV-2</td><td colspan="1" rowspan="1">Low</td><td colspan="1" rowspan="1">100 (10/10)</td><td colspan="1" rowspan="1">100 (10/10)</td><td colspan="1" rowspan="1">100 (10/10)</td><td colspan="1" rowspan="1">100 (30/30)</td><td colspan="1" rowspan="2">100 (60/60)</td></tr><tr><td colspan="1" rowspan="1">Mod</td><td colspan="1" rowspan="1">100 (10/10)</td><td colspan="1" rowspan="1">100 (10/10)</td><td colspan="1" rowspan="1">100 (10/10)</td><td colspan="1" rowspan="1">100 (30/30)</td></tr><tr><td colspan="1" rowspan="2">Human Metapneumovirus</td><td colspan="1" rowspan="1">Low</td><td colspan="1" rowspan="1">100 (10/10)</td><td colspan="1" rowspan="1">100 (10/10)</td><td colspan="1" rowspan="1">100 (10/10)</td><td colspan="1" rowspan="1">100 (30/30)</td><td colspan="1" rowspan="2">100 (60/60)</td></tr><tr><td colspan="1" rowspan="1">Mod</td><td colspan="1" rowspan="1">100 (10/10)</td><td colspan="1" rowspan="1">100 (10/10)</td><td colspan="1" rowspan="1">100 (10/10)</td><td colspan="1" rowspan="1">100 (30/30)</td></tr><tr><td colspan="1" rowspan="2">Adenovirus 2</td><td colspan="1" rowspan="1">Low</td><td colspan="1" rowspan="1">100 (10/10)</td><td colspan="1" rowspan="1">100 (10/10)</td><td colspan="1" rowspan="1">100 (10/10)</td><td colspan="1" rowspan="1">100 (30/30)</td><td colspan="1" rowspan="2">100 (60/60)</td></tr><tr><td colspan="1" rowspan="1">Mod</td><td colspan="1" rowspan="1">100 (10/10)</td><td colspan="1" rowspan="1">100 (10/10)</td><td colspan="1" rowspan="1">100 (10/10)</td><td colspan="1" rowspan="1">100 (30/30)</td></tr><tr><td colspan="1" rowspan="2">All Other</td><td colspan="1" rowspan="1">Low</td><td colspan="1" rowspan="1">100 (180/180)</td><td colspan="1" rowspan="1">100 (180/180)</td><td colspan="1" rowspan="1">100 (180/180)</td><td colspan="1" rowspan="1">100 (540/540)</td><td colspan="1" rowspan="2">100(1080/1080)</td></tr><tr><td colspan="1" rowspan="1">Mod</td><td colspan="1" rowspan="1">100 (180/180)</td><td colspan="1" rowspan="1">100 (180/180)</td><td colspan="1" rowspan="1">100 (180/180)</td><td colspan="1" rowspan="1">100 (540/540)</td></tr><tr><td colspan="1" rowspan="8">5</td><td colspan="1" rowspan="2">Influenza A (matrix)</td><td colspan="1" rowspan="1">Low</td><td colspan="1" rowspan="1">100 (10/10)</td><td colspan="1" rowspan="1">100 (10/10)</td><td colspan="1" rowspan="1">100 (10/10)</td><td colspan="1" rowspan="1">100 (30/30)</td><td colspan="1" rowspan="2">100 (60/60)</td></tr><tr><td colspan="1" rowspan="1">Mod</td><td colspan="1" rowspan="1">100 (10/10)</td><td colspan="1" rowspan="1">100 (10/10)</td><td colspan="1" rowspan="1">100 (10/10)</td><td colspan="1" rowspan="1">100 (30/30)</td></tr><tr><td colspan="1" rowspan="2">Coronavirus NL63</td><td colspan="1" rowspan="1">Low</td><td colspan="1" rowspan="1">100 (10/10)</td><td colspan="1" rowspan="1">100 (10/10)</td><td colspan="1" rowspan="1">100 (10/10)</td><td colspan="1" rowspan="1">100 (30/30)</td><td colspan="1" rowspan="2">100 (60/60)</td></tr><tr><td colspan="1" rowspan="1">Mod</td><td colspan="1" rowspan="1">100 (10/10)</td><td colspan="1" rowspan="1">100 (10/10)</td><td colspan="1" rowspan="1">100 (10/10)</td><td colspan="1" rowspan="1">100 (30/30)</td></tr><tr><td colspan="1" rowspan="2">Coronavirus HKU1</td><td colspan="1" rowspan="1">Low</td><td colspan="1" rowspan="1">100 (10/10)</td><td colspan="1" rowspan="1">100 (10/10)</td><td colspan="1" rowspan="1">100 (10/10)</td><td colspan="1" rowspan="1">100 (30/30)</td><td colspan="1" rowspan="2">100 (60/60)</td></tr><tr><td colspan="1" rowspan="1">Mod</td><td colspan="1" rowspan="1">100 (10/10)</td><td colspan="1" rowspan="1">100 (10/10)</td><td colspan="1" rowspan="1">100 (10/10)</td><td colspan="1" rowspan="1">100 (30/30)</td></tr><tr><td colspan="1" rowspan="2">All Other</td><td colspan="1" rowspan="1">Low</td><td colspan="1" rowspan="1">100 (180/180)</td><td colspan="1" rowspan="1">99.4 (179/180)</td><td colspan="1" rowspan="1">100 (180/180)</td><td colspan="1" rowspan="1">99.8 (539/540)</td><td colspan="1" rowspan="2">99.9(1079/1080)</td></tr><tr><td colspan="1" rowspan="1">Mod</td><td colspan="1" rowspan="1">100 (180/180)</td><td colspan="1" rowspan="1">100 (180/180)</td><td colspan="1" rowspan="1">100 (180/180)</td><td colspan="1" rowspan="1">100 (540/540)</td></tr><tr><td colspan="1" rowspan="8">6</td><td colspan="1" rowspan="2">Influenza A H1 (subtype) 1</td><td colspan="1" rowspan="1">Low 6</td><td colspan="1" rowspan="1">100 (10/10)</td><td colspan="1" rowspan="1">100 (10/10)</td><td colspan="1" rowspan="1">100 (10/10)</td><td colspan="1" rowspan="1">100 (30/30)</td><td colspan="1" rowspan="2">100 (60/60)</td></tr><tr><td colspan="1" rowspan="1">Mod</td><td colspan="1" rowspan="1">100 (10/10)</td><td colspan="1" rowspan="1">100 (10/10)</td><td colspan="1" rowspan="1">100 (10/10)</td><td colspan="1" rowspan="1">100 (30/30)</td></tr><tr><td colspan="1" rowspan="2">Parainfluenza virus 1</td><td colspan="1" rowspan="1">Low</td><td colspan="1" rowspan="1">100 (10/10)</td><td colspan="1" rowspan="1">100 (10/10)</td><td colspan="1" rowspan="1">100 (10/10)</td><td colspan="1" rowspan="1">100 (30/30)</td><td colspan="1" rowspan="2">100 (60/60)</td></tr><tr><td colspan="1" rowspan="1">Mod</td><td colspan="1" rowspan="1">100 (10/10)</td><td colspan="1" rowspan="1">100 (10/10)</td><td colspan="1" rowspan="1">100 (10/10)</td><td colspan="1" rowspan="1">100 (30/30)</td></tr><tr><td colspan="1" rowspan="2">Chlamydia pneumoniae</td><td colspan="1" rowspan="1">Low</td><td colspan="1" rowspan="1">100 (10/10)</td><td colspan="1" rowspan="1">100 (10/10)</td><td colspan="1" rowspan="1">100 (10/10)</td><td colspan="1" rowspan="1">100 (30/30)</td><td colspan="1" rowspan="2">100 (60/60)</td></tr><tr><td colspan="1" rowspan="1">Mod</td><td colspan="1" rowspan="1">100 (10/10)</td><td colspan="1" rowspan="1">100 (10/10)</td><td colspan="1" rowspan="1">100 (10/10)</td><td colspan="1" rowspan="1">100 (30/30)</td></tr><tr><td colspan="1" rowspan="2">All Other</td><td colspan="1" rowspan="1">Low</td><td colspan="1" rowspan="1">100 (180/180)</td><td colspan="1" rowspan="1">100 (180/180)</td><td colspan="1" rowspan="1">100 (180/180)</td><td colspan="1" rowspan="1">100 (540/540)</td><td colspan="1" rowspan="2">99.9(1079/1080)</td></tr><tr><td colspan="1" rowspan="1">Mod</td><td colspan="1" rowspan="1">100 (180/180)</td><td colspan="1" rowspan="1">99.4 (179/180)</td><td colspan="1" rowspan="1">100 (180/180)</td><td colspan="1" rowspan="1">99.8 (539/540)</td></tr><tr><td colspan="1" rowspan="8">7</td><td colspan="1" rowspan="2">Parainfluenza virus 2</td><td colspan="1" rowspan="1">Low</td><td colspan="1" rowspan="1">100 (10/10)</td><td colspan="1" rowspan="1">100 (10/10)</td><td colspan="1" rowspan="1">100 (10/10)</td><td colspan="1" rowspan="1">100 (30/30)</td><td colspan="1" rowspan="2">100 (60/60)</td></tr><tr><td colspan="1" rowspan="1">Mod</td><td colspan="1" rowspan="1">100 (10/10)</td><td colspan="1" rowspan="1">100 (10/10)</td><td colspan="1" rowspan="1">100 (10/10)</td><td colspan="1" rowspan="1">100 (30/30)</td></tr><tr><td colspan="1" rowspan="2">Parainfluenza virus 4</td><td colspan="1" rowspan="1">Low</td><td colspan="1" rowspan="1">100 (10/10)</td><td colspan="1" rowspan="1">100 (10/10)</td><td colspan="1" rowspan="1">100 (10/10)</td><td colspan="1" rowspan="1">100 (30/30)</td><td colspan="1" rowspan="2">100 (60/60)</td></tr><tr><td colspan="1" rowspan="1">Mod</td><td colspan="1" rowspan="1">100 (10/10)</td><td colspan="1" rowspan="1">100 (10/10)</td><td colspan="1" rowspan="1">100 (10/10)</td><td colspan="1" rowspan="1">100 (30/30)</td></tr><tr><td colspan="1" rowspan="2">Coronavirus 229E</td><td colspan="1" rowspan="1">Low</td><td colspan="1" rowspan="1">100 (10/10)</td><td colspan="1" rowspan="1">100 (10/10)</td><td colspan="1" rowspan="1">1000 (10/10)</td><td colspan="1" rowspan="1">100 (30/30)</td><td colspan="1" rowspan="2">100 (60/60)</td></tr><tr><td colspan="1" rowspan="1">Mod</td><td colspan="1" rowspan="1">100 (10/10)</td><td colspan="1" rowspan="1">100 (10/10)</td><td colspan="1" rowspan="1">100 (10/10)</td><td colspan="1" rowspan="1">100 (30/30)</td></tr><tr><td colspan="1" rowspan="2">All Other</td><td colspan="1" rowspan="1">Low</td><td colspan="1" rowspan="1">100 (180/180)</td><td colspan="1" rowspan="1">100 (180/180)</td><td colspan="1" rowspan="1">99.4 (179/180)</td><td colspan="1" rowspan="1">99.8 (539/540)</td><td colspan="1" rowspan="2">99.9(1079/1080)</td></tr><tr><td colspan="1" rowspan="1">Mod</td><td colspan="1" rowspan="1">100 (180/180)</td><td colspan="1" rowspan="1">100 (180/180)</td><td colspan="1" rowspan="1">100 (180/180)</td><td colspan="1" rowspan="1">100 (540/540)</td></tr><tr><td colspan="1" rowspan="6">8</td><td colspan="1" rowspan="2">Parainfluenza virus 4</td><td colspan="1" rowspan="1">Low</td><td colspan="1" rowspan="1">100 (10/10)</td><td colspan="1" rowspan="1">100 (10/10)</td><td colspan="1" rowspan="1">100 (10/10)</td><td colspan="1" rowspan="1">100 (30/30)</td><td colspan="1" rowspan="2">100 (60/60)</td></tr><tr><td colspan="1" rowspan="1">Mod</td><td colspan="1" rowspan="1">100 (10/10)</td><td colspan="1" rowspan="1">100 (10/10)</td><td colspan="1" rowspan="1">100 (10/10)</td><td colspan="1" rowspan="1">100 (30/30)</td></tr><tr><td colspan="1" rowspan="2">Coronavirus OC43</td><td colspan="1" rowspan="1">Low</td><td colspan="1" rowspan="1">100 (10/10)</td><td colspan="1" rowspan="1">100 (10/10)</td><td colspan="1" rowspan="1">100 (10/10)</td><td colspan="1" rowspan="1">100 (30/30)</td><td colspan="1" rowspan="2">100 (60/60)</td></tr><tr><td colspan="1" rowspan="1">Mod</td><td colspan="1" rowspan="1">100 (10/10)</td><td colspan="1" rowspan="1">100 (10/10)</td><td colspan="1" rowspan="1">100 (10/10)</td><td colspan="1" rowspan="1">100 (30/30)</td></tr><tr><td colspan="1" rowspan="2">All Other</td><td colspan="1" rowspan="1">Low</td><td colspan="1" rowspan="1">100 (190/190)</td><td colspan="1" rowspan="1">100 (190/190)</td><td colspan="1" rowspan="1">100 (190/190)</td><td colspan="1" rowspan="1">100 (570/570)</td><td colspan="1" rowspan="2">100(1140/1140)</td></tr><tr><td colspan="1" rowspan="1">Mod</td><td colspan="1" rowspan="1">100 (190/190)</td><td colspan="1" rowspan="1">100 (190/190)</td><td colspan="1" rowspan="1">100 (190/190)</td><td colspan="1" rowspan="1">100 (570/570)</td></tr><tr><td colspan="1" rowspan="1">9</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">99.0 (208/210)</td><td colspan="1" rowspan="1">100 (210/210)</td><td colspan="1" rowspan="1">100 (210/210)</td><td colspan="2" rowspan="1">99.7 (628/630)</td></tr></table>

N/A: Not applicable; Low: 1.5X LoD; Mod (Moderate): 5X LoD, except where noted 1 Formulated based on the LoD for the applicable influenza A subtype. 2 Formulated at approximately 3X (Low) and 9X LoD (Moderate). 3 Expected results at the Low target level based on the respective analytical sensitivities of the H1pdm09 subtyping and influenza A assays: H1pdm09 subtype “positive”, Influenza A “positive”. 4 Expected results at the Low target level based on the respective analytical sensitivities of the H3 subtyping and influenza A assays: H3 subtype “positive”, Influenza A “positive” or “negative”; agreement for Influenza A is included under “All Other” $\mathrm { \Delta } _ { \mathrm { n } } = 1 0$ per lot per level). 5 Formulated at approximately 2X (Low) and 7X LoD (Moderate). 6 Expected results at the Low target level based on the respective analytical sensitivities of the H1 subtyping and influenza A assays: H1 subtype “positive”, Influenza A “positive”.

# Site-to-Site and Within-Site (Operator-to-Operator) Reproducibility

The site-to-site and within site (operator-to-operator) reproducibility of the NxTAG RPP v2 was evaluated using a panel of samples that were tested by two operators at each of three study sites in replicates of four on five non-consecutive days, using a single reagent lot and unique set of critical equipment at each site (2 operators x 3 sites x 4 replicates x 5 days x 1 $\mathrm { l o t } = 1 2 0$ replicates per panel member). The panel of samples for analysis comprised simulated nasopharyngeal swab matrix that was spiked with different combinations of a subset of 11 of the NxTAG RPP v2 target analytes, as well as a negative sample (Table 5). Each positive panel member was prepared at both low and moderate target levels (1.5X and 5X LoD, respectively, except where noted). Positive and negative agreement for each panel member within and between study sites and operators was close to $100 \%$ (Table 6). These results are acceptable.

Table 5. Panel members used to evaluate site-to-site and within-site reproducibility   

<table><tr><td rowspan=1 colspan=1>Sample</td><td rowspan=1 colspan=1>Analyte</td><td rowspan=1 colspan=1>Strain/Isolate</td><td rowspan=1 colspan=1>Source</td><td rowspan=1 colspan=1>Catalogue #</td></tr><tr><td rowspan=3 colspan=1>1</td><td rowspan=1 colspan=1>Influenza A H1pdm09 (subtype) 1, 2, 3</td><td rowspan=1 colspan=1>A/NY/02/09</td><td rowspan=1 colspan=1>ZeptoMetrix</td><td rowspan=1 colspan=1>0810109CFN</td></tr><tr><td rowspan=1 colspan=1>Respiratory Syncytial Virus A</td><td rowspan=1 colspan=1>A2</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>VR-1540</td></tr><tr><td rowspan=1 colspan=1>Rhinovirus</td><td rowspan=1 colspan=1>50-525-CV54</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>VR-1195</td></tr><tr><td rowspan=2 colspan=1>2</td><td rowspan=1 colspan=1>Influenza A H3 (subtype) 1,4</td><td rowspan=1 colspan=1>Wisconsin/67/05</td><td rowspan=1 colspan=1>ZeptoMetrix</td><td rowspan=1 colspan=1>0810252CF</td></tr><tr><td rowspan=1 colspan=1>Respiratory Syncytial Virus B</td><td rowspan=1 colspan=1>18537</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>VR-1580</td></tr><tr><td rowspan=3 colspan=1>3</td><td rowspan=1 colspan=1>Influenza B</td><td rowspan=1 colspan=1>B/Florida/02/06</td><td rowspan=1 colspan=1>ZeptoMetrix</td><td rowspan=1 colspan=1>0810037CF</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza Virus 3</td><td rowspan=1 colspan=1>C 243</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>VR-93</td></tr><tr><td rowspan=1 colspan=1>Mycoplasma pneumoniae 5</td><td rowspan=1 colspan=1>M129</td><td rowspan=1 colspan=1>ZeptoMetrix</td><td rowspan=1 colspan=1>0801579</td></tr><tr><td rowspan=3 colspan=1>4</td><td rowspan=1 colspan=1>SARS-CoV-2</td><td rowspan=1 colspan=1>USA-WA1/2020</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>VR-1986HK</td></tr><tr><td rowspan=1 colspan=1>Human Metapneumovirus</td><td rowspan=1 colspan=1>hMPV-3, Type B1, Peru2-2002</td><td rowspan=1 colspan=1>ZeptoMetrix</td><td rowspan=1 colspan=1>0810156CF</td></tr><tr><td rowspan=1 colspan=1>Adenovirus B 2</td><td rowspan=1 colspan=1>Type 14, 2006 isolate</td><td rowspan=1 colspan=1>ZeptoMetrix</td><td rowspan=1 colspan=1>0810108CF</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>--</td><td rowspan=1 colspan=1>--</td><td rowspan=1 colspan=1>--</td></tr></table>

Samples 1-4 were formulated at both $1 . 5 \mathrm { X }$ (Low) and 5X LoD (Moderate), except where noted Formulated based on the LoD for the applicable influenza A subtype. 2 Formulated at approximately 3X (Low) and 9X LoD (Moderate). 3 Expected results based on the relative analytical sensitivities of the H1pdm09 subtyping and influenza A assays: H1pdm09 subtype “positive”, Influenza A “positive”. Expected results based on the relative analytical sensitivities of the H3 subtyping and influenza A assays: H3 subtype “positive”, Influenza A “positive” or “negative”. 5 Formulated at approximately 2X (Low) and 7X LoD (Moderate).

Table 6. Site-to-site agreement with expected results   

<table><tr><td colspan="1" rowspan="3">Sample</td><td colspan="1" rowspan="3">NxTAG RPP v2Analyte</td><td colspan="1" rowspan="3">PanelLevel</td><td colspan="7" rowspan="1">Percent Agreement</td></tr><tr><td colspan="2" rowspan="1">Site 1</td><td colspan="2" rowspan="1">Site 2</td><td colspan="2" rowspan="1">Site 3</td><td colspan="1" rowspan="2">Total</td></tr><tr><td colspan="1" rowspan="1">Operator 1</td><td colspan="1" rowspan="1">Operator 2</td><td colspan="1" rowspan="1">Operator 1</td><td colspan="1" rowspan="1">Operator 2</td><td colspan="1" rowspan="1">Operator 1</td><td colspan="1" rowspan="1">Operator 2</td></tr><tr><td colspan="1" rowspan="13">1</td><td colspan="1" rowspan="4">Influenza AH1pdm09(subtype) 1,2, 3</td><td colspan="1" rowspan="2">Low</td><td colspan="1" rowspan="1">100(20/20)</td><td colspan="1" rowspan="1">100(20/20)</td><td colspan="1" rowspan="1">100(20/20)</td><td colspan="1" rowspan="1">100(20/20)</td><td colspan="1" rowspan="1">100(20/20)</td><td colspan="1" rowspan="1">100(20/20)</td><td colspan="1" rowspan="2">100(120/120)</td></tr><tr><td colspan="2" rowspan="1">100(40/40)</td><td colspan="2" rowspan="1">100(40/40)</td><td colspan="2" rowspan="1">100(40/40)</td></tr><tr><td colspan="1" rowspan="2">Mod</td><td colspan="1" rowspan="1">100(20/20)</td><td colspan="1" rowspan="1">100(20/20)</td><td colspan="1" rowspan="1">100(20/20)</td><td colspan="1" rowspan="1">100(20/20)</td><td colspan="1" rowspan="1">100(20/20)</td><td colspan="1" rowspan="1">100(20/20)</td><td colspan="1" rowspan="2">100(120/120)</td></tr><tr><td colspan="2" rowspan="1">100(40/40)</td><td colspan="2" rowspan="1">100(40/40)</td><td colspan="2" rowspan="1">100(40/40)</td></tr><tr><td colspan="1" rowspan="4">RespiratorySyncytial Virus A</td><td colspan="1" rowspan="2">Low</td><td colspan="1" rowspan="1">100(20/20)</td><td colspan="1" rowspan="1">100(20/20)</td><td colspan="1" rowspan="1">100(20/20)</td><td colspan="1" rowspan="1">100(20/20)</td><td colspan="1" rowspan="1">100(20/20)</td><td colspan="1" rowspan="1">100(20/20)</td><td colspan="1" rowspan="2">100(120/120)</td></tr><tr><td colspan="2" rowspan="1">100(40/40)</td><td colspan="2" rowspan="1">100(40/40)</td><td colspan="2" rowspan="1">100(40/40)</td></tr><tr><td colspan="1" rowspan="2">Mod</td><td colspan="1" rowspan="1">100(20/20)</td><td colspan="1" rowspan="1">100(20/20)</td><td colspan="1" rowspan="1">100(20/20)</td><td colspan="1" rowspan="1">100(20/20)</td><td colspan="1" rowspan="1">100(20/20)</td><td colspan="1" rowspan="1">100(20/20)</td><td colspan="1" rowspan="2">100(120/120)</td></tr><tr><td colspan="2" rowspan="1">100(40/40)</td><td colspan="2" rowspan="1">100(40/40)</td><td colspan="2" rowspan="1">100(40/40)</td></tr><tr><td colspan="1" rowspan="4">Rhinovirus/Enterovirus</td><td colspan="1" rowspan="2">Low</td><td colspan="1" rowspan="1">100(20/20)</td><td colspan="1" rowspan="1">100(20/20)</td><td colspan="1" rowspan="1">100(20/20)</td><td colspan="1" rowspan="1">100(20/20)</td><td colspan="1" rowspan="1">100(20/20)</td><td colspan="1" rowspan="1">100(20/20)</td><td colspan="1" rowspan="2">100(120/120)</td></tr><tr><td colspan="2" rowspan="1">100(40/40)</td><td colspan="2" rowspan="1">100(40/40)</td><td colspan="2" rowspan="1">100(40/40)</td></tr><tr><td colspan="1" rowspan="2">Mod</td><td colspan="1" rowspan="1">100(20/20)</td><td colspan="1" rowspan="1">100(20/20)</td><td colspan="1" rowspan="1">100(20/20)</td><td colspan="1" rowspan="1">100(20/20)</td><td colspan="1" rowspan="1">100(20/20)</td><td colspan="1" rowspan="1">100(20/20)</td><td colspan="1" rowspan="2">100(120/120)</td></tr><tr><td colspan="2" rowspan="1">100(40/40)</td><td colspan="2" rowspan="1">100(40/40)</td><td colspan="2" rowspan="1">100(40/40)</td></tr><tr><td colspan="1" rowspan="1">All Other</td><td colspan="1" rowspan="1">Low</td><td colspan="1" rowspan="1">99.7(359/360)</td><td colspan="1" rowspan="1">100(360/360)</td><td colspan="1" rowspan="1">100(360/360)</td><td colspan="1" rowspan="1">100(360/360)</td><td colspan="1" rowspan="1">100(360/360)</td><td colspan="1" rowspan="1">100(360/360)</td><td colspan="1" rowspan="1">100(2159/2160)</td></tr><tr><td colspan="2" rowspan="1">Operator 1 Operator 2</td><td colspan="2" rowspan="1">Operator 1  Operator 2</td><td colspan="2" rowspan="1">Operator 1  Operator 2</td></tr><tr><td colspan="1" rowspan="3">b</td><td colspan="1" rowspan="3"></td><td colspan="1" rowspan="1"></td><td colspan="2" rowspan="1">99.9(719/720)</td><td colspan="2" rowspan="1">100(720/720)</td><td colspan="2" rowspan="1">100(720/720)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="2">Mod</td><td colspan="1" rowspan="1">99.7(359/360)</td><td colspan="1" rowspan="1">99.7(359/360)</td><td colspan="1" rowspan="1">100(360/360)</td><td colspan="1" rowspan="1">100(360/360)</td><td colspan="1" rowspan="1">99.7(359/360)</td><td colspan="1" rowspan="1">100(360/360)</td><td colspan="1" rowspan="2">99.9(2157/2160)</td></tr><tr><td colspan="2" rowspan="1">99.7(718/720)</td><td colspan="2" rowspan="1">100(720/720)</td><td colspan="2" rowspan="1">99.9(719/720)</td></tr><tr><td colspan="1" rowspan="12">2</td><td colspan="1" rowspan="4">Influenza A H3(subtype) 1,4</td><td colspan="1" rowspan="2">Low</td><td colspan="1" rowspan="1">95.0(19/20)</td><td colspan="1" rowspan="1">100(20/20)</td><td colspan="1" rowspan="1">100(20/20)</td><td colspan="1" rowspan="1">100(20/20)</td><td colspan="1" rowspan="1">100(20/20)</td><td colspan="1" rowspan="1">100(20/20)</td><td colspan="1" rowspan="2">99.2(119/120)</td></tr><tr><td colspan="2" rowspan="1">97.5(39/40)</td><td colspan="2" rowspan="1">100(40/40)</td><td colspan="2" rowspan="1">100(40/40)</td></tr><tr><td colspan="1" rowspan="2">Mod</td><td colspan="1" rowspan="1">100(20/20)</td><td colspan="1" rowspan="1">100(20/20)</td><td colspan="1" rowspan="1">100(20/20)</td><td colspan="1" rowspan="1">100(20/20)</td><td colspan="1" rowspan="1">100(20/20)</td><td colspan="1" rowspan="1">100(20/20)</td><td colspan="1" rowspan="2">100(120/120)</td></tr><tr><td colspan="2" rowspan="1">100(40/40)</td><td colspan="2" rowspan="1">100(40/40)</td><td colspan="2" rowspan="1">100(40/40)</td></tr><tr><td colspan="1" rowspan="4">RespiratorySyncytial Virus B</td><td colspan="1" rowspan="2">Low</td><td colspan="1" rowspan="1">100(20/20)</td><td colspan="1" rowspan="1">100(20/20)</td><td colspan="1" rowspan="1">100(20/20)</td><td colspan="1" rowspan="1">100(20/20)</td><td colspan="1" rowspan="1">100(20/20)</td><td colspan="1" rowspan="1">100(20/20)</td><td colspan="1" rowspan="2">100(120/120)</td></tr><tr><td colspan="2" rowspan="1">100(40/40)</td><td colspan="2" rowspan="1">100(40/40)</td><td colspan="2" rowspan="1">100(40/40)</td></tr><tr><td colspan="1" rowspan="2">Mod</td><td colspan="1" rowspan="1">100(20/20)</td><td colspan="1" rowspan="1">100(20/20)</td><td colspan="1" rowspan="1">100(20/20)</td><td colspan="1" rowspan="1">100(20/20)</td><td colspan="1" rowspan="1">100(20/20)</td><td colspan="1" rowspan="1">100(20/20)</td><td colspan="1" rowspan="2">100(120/120)</td></tr><tr><td colspan="2" rowspan="1">100(40/40)</td><td colspan="2" rowspan="1">100(40/40)</td><td colspan="2" rowspan="1">100(40/40)</td></tr><tr><td colspan="1" rowspan="4">All Other</td><td colspan="1" rowspan="2">Low</td><td colspan="1" rowspan="1">99.5(378/380)</td><td colspan="1" rowspan="1">99.7(379/380)</td><td colspan="1" rowspan="1">100(380/380)</td><td colspan="1" rowspan="1">100(380/380)</td><td colspan="1" rowspan="1">100(380/380)</td><td colspan="1" rowspan="1">100(380/380)</td><td colspan="1" rowspan="1">99.9(2277/2280)</td></tr><tr><td colspan="2" rowspan="1">99.6(757/760)</td><td colspan="2" rowspan="1">100(760/760)</td><td colspan="2" rowspan="1">100(760/760)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="2">Mod</td><td colspan="1" rowspan="1">99.7(379/380)</td><td colspan="1" rowspan="1">99.5(378/380)</td><td colspan="1" rowspan="1">100(380/380)</td><td colspan="1" rowspan="1">100(380/380)</td><td colspan="1" rowspan="1">100(380/380)</td><td colspan="1" rowspan="1">100(380/380)</td><td colspan="1" rowspan="2">99.9(2277/2280)</td></tr><tr><td colspan="2" rowspan="1">99.6(757/760)</td><td colspan="2" rowspan="1">100(760/760)</td><td colspan="2" rowspan="1">100(760/760)</td></tr><tr><td colspan="1" rowspan="16">3</td><td colspan="1" rowspan="4">Influenza B</td><td colspan="1" rowspan="2">Low</td><td colspan="1" rowspan="1">95.0(19/20)</td><td colspan="1" rowspan="1">100(20/20)</td><td colspan="1" rowspan="1">100(20/20)</td><td colspan="1" rowspan="1">100(20/20)</td><td colspan="1" rowspan="1">100(20/20)</td><td colspan="1" rowspan="1">100(20/20)</td><td colspan="1" rowspan="2">99.2(119/120)</td></tr><tr><td colspan="2" rowspan="1">97.5(39/40)</td><td colspan="2" rowspan="1">100(40/40)</td><td colspan="2" rowspan="1">100(40/40)</td></tr><tr><td colspan="1" rowspan="2">Mod</td><td colspan="1" rowspan="1">100(20/20)</td><td colspan="1" rowspan="1">100(20/20)</td><td colspan="1" rowspan="1">100(20/20)</td><td colspan="1" rowspan="1">100(20/20)</td><td colspan="1" rowspan="1">100(20/20)</td><td colspan="1" rowspan="1">100(20/20)</td><td colspan="1" rowspan="2">100(120/120)</td></tr><tr><td colspan="2" rowspan="1">100(40/40)</td><td colspan="2" rowspan="1">100(40/40)</td><td colspan="2" rowspan="1">100(40/40)</td></tr><tr><td colspan="1" rowspan="4">Parainfluenzavirus 3</td><td colspan="1" rowspan="2">Low</td><td colspan="1" rowspan="1">95.0(19/20)</td><td colspan="1" rowspan="1">100(20/20)</td><td colspan="1" rowspan="1">100(20/20)</td><td colspan="1" rowspan="1">100(20/20)</td><td colspan="1" rowspan="1">100(20/20)</td><td colspan="1" rowspan="1">100(20/20)</td><td colspan="1" rowspan="2">99.2(119/120)</td></tr><tr><td colspan="2" rowspan="1">97.5(39/40)</td><td colspan="2" rowspan="1">100(40/40)</td><td colspan="2" rowspan="1">100(40/40)</td></tr><tr><td colspan="1" rowspan="2">Mod</td><td colspan="1" rowspan="1">100(20/20)</td><td colspan="1" rowspan="1">100(20/20)</td><td colspan="1" rowspan="1">100(20/20)</td><td colspan="1" rowspan="1">100(20/20)</td><td colspan="1" rowspan="1">100(20/20)</td><td colspan="1" rowspan="1">100(20/20)</td><td colspan="1" rowspan="2">100(120/120)</td></tr><tr><td colspan="2" rowspan="1">100(40/40)</td><td colspan="2" rowspan="1">100(40/40)</td><td colspan="2" rowspan="1">100(40/40)</td></tr><tr><td colspan="1" rowspan="4">Mycoplasmapneumoniae 5</td><td colspan="1" rowspan="2">Low</td><td colspan="1" rowspan="1">95.0(19/20)</td><td colspan="1" rowspan="1">100(20/20)</td><td colspan="1" rowspan="1">100(20/20)</td><td colspan="1" rowspan="1">100(20/20)</td><td colspan="1" rowspan="1">100(20/20)</td><td colspan="1" rowspan="1">100(20/20)</td><td colspan="1" rowspan="2">99.2(119/120)</td></tr><tr><td colspan="2" rowspan="1">97.5(39/40)</td><td colspan="2" rowspan="1">100(40/40)</td><td colspan="2" rowspan="1">100(40/40)</td></tr><tr><td colspan="1" rowspan="2">Mod</td><td colspan="1" rowspan="1">100(20/20)</td><td colspan="1" rowspan="1">100(20/20)</td><td colspan="1" rowspan="1">100(20/20)</td><td colspan="1" rowspan="1">100(20/20)</td><td colspan="1" rowspan="1">100(20/20)</td><td colspan="1" rowspan="1">100(20/20)</td><td colspan="1" rowspan="2">100(120/120)</td></tr><tr><td colspan="2" rowspan="1">100(40/40)</td><td colspan="2" rowspan="1">100(40/40)</td><td colspan="2" rowspan="1">100(40/40)</td></tr><tr><td colspan="1" rowspan="4">All Other(Negative)</td><td colspan="1" rowspan="2">Low</td><td colspan="1" rowspan="1">99.5(358/360)</td><td colspan="1" rowspan="1">100(360/360)</td><td colspan="1" rowspan="1">100(360/360)</td><td colspan="1" rowspan="1">100(360/360)</td><td colspan="1" rowspan="1">100(360/360)</td><td colspan="1" rowspan="1">100(360/360)</td><td colspan="1" rowspan="2">99.9(2158/2160)</td></tr><tr><td colspan="2" rowspan="1">99.7(718/720)</td><td colspan="2" rowspan="1">100(720/720)</td><td colspan="2" rowspan="1">100(720/720)</td></tr><tr><td colspan="1" rowspan="2">Mod</td><td colspan="1" rowspan="1">99.7(359/360)</td><td colspan="1" rowspan="1">100(360/360)</td><td colspan="1" rowspan="1">100(360/360)</td><td colspan="1" rowspan="1">100(360/360)</td><td colspan="1" rowspan="1">100(360/360)</td><td colspan="1" rowspan="1">100(360/360)</td><td colspan="1" rowspan="2">100(2159/2160)</td></tr><tr><td colspan="2" rowspan="1">99.9(719/720)</td><td colspan="2" rowspan="1">100(720/720)</td><td colspan="2" rowspan="1">100(720/720)</td></tr><tr><td colspan="1" rowspan="3">4</td><td colspan="1" rowspan="3">SARS-CoV-2</td><td colspan="1" rowspan="2">Low</td><td colspan="1" rowspan="1">100(20/20)</td><td colspan="1" rowspan="1">100(20/20)</td><td colspan="1" rowspan="1">100(20/20)</td><td colspan="1" rowspan="1">100(20/20)</td><td colspan="1" rowspan="1">100(20/20)</td><td colspan="1" rowspan="1">100(20/20)</td><td colspan="1" rowspan="2">100(120/120)</td></tr><tr><td colspan="2" rowspan="1">100(40/40)</td><td colspan="2" rowspan="1">100(40/40)</td><td colspan="2" rowspan="1">100(40/40)</td></tr><tr><td colspan="1" rowspan="1">Mod</td><td colspan="1" rowspan="1">100(20/20)</td><td colspan="1" rowspan="1">100(20/20)</td><td colspan="1" rowspan="1">100(20/20)</td><td colspan="1" rowspan="1">100(20/20)</td><td colspan="1" rowspan="1">100(20/20)</td><td colspan="1" rowspan="1">100(20/20)</td><td colspan="1" rowspan="1">100(120/120)</td></tr><tr><td colspan="2" rowspan="1">Operator 1  Operator 2</td><td colspan="2" rowspan="1">Operator 1  Operator 2</td><td colspan="2" rowspan="1">Operator 1  Operator 2</td></tr><tr><td colspan="1" rowspan="13"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="2" rowspan="1">100(40/40)</td><td colspan="2" rowspan="1">100(40/40)</td><td colspan="2" rowspan="1">100(40/40)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="4">HumanMetapneumovirus</td><td colspan="1" rowspan="2">Low</td><td colspan="1" rowspan="1">100(20/20)</td><td colspan="1" rowspan="1">100(20/20)</td><td colspan="1" rowspan="1">100(20/20)</td><td colspan="1" rowspan="1">100(20/20)</td><td colspan="1" rowspan="1">100(20/20)</td><td colspan="1" rowspan="1">100(20/20)</td><td colspan="1" rowspan="2">100(120/120)</td></tr><tr><td colspan="2" rowspan="1">100(40/40)</td><td colspan="2" rowspan="1">100(40/40)</td><td colspan="2" rowspan="1">100(40/40)</td></tr><tr><td colspan="1" rowspan="2">Mod</td><td colspan="1" rowspan="1">100(20/20)</td><td colspan="1" rowspan="1">100(20/20)</td><td colspan="1" rowspan="1">100(20/20)</td><td colspan="1" rowspan="1">100(20/20)</td><td colspan="1" rowspan="1">100(20/20)</td><td colspan="1" rowspan="1">100(20/20)</td><td colspan="1" rowspan="2">100(120/120)</td></tr><tr><td colspan="2" rowspan="1">100(40/40)</td><td colspan="2" rowspan="1">100(40/40)</td><td colspan="2" rowspan="1">100(40/40)</td></tr><tr><td colspan="1" rowspan="4">Adenovirus 2</td><td colspan="1" rowspan="2">Low</td><td colspan="1" rowspan="1">100(20/20)</td><td colspan="1" rowspan="1">100(20/20)</td><td colspan="1" rowspan="1">100(20/20)</td><td colspan="1" rowspan="1">100(20/20)</td><td colspan="1" rowspan="1">100(20/20)</td><td colspan="1" rowspan="1">100(20/20)</td><td colspan="1" rowspan="2">100(120/120)</td></tr><tr><td colspan="1" rowspan="1">(40/40)</td><td colspan="1" rowspan="1">100(40/40)</td><td colspan="1" rowspan="1">100(40/40)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">100(40/40)</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="2">Mod</td><td colspan="1" rowspan="1">100(20/20)</td><td colspan="1" rowspan="1">100(20/20)</td><td colspan="1" rowspan="1">100(20/20)</td><td colspan="1" rowspan="1">100(20/20)</td><td colspan="1" rowspan="1">100(20/20)</td><td colspan="1" rowspan="1">100(20/20)</td><td colspan="1" rowspan="2">100(120/120)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">100(40/40)</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">100(40/40)</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="4">All Other(Negative)</td><td colspan="1" rowspan="2">Low</td><td colspan="1" rowspan="1">99.5(358/360)</td><td colspan="1" rowspan="1">99.5(358/360)</td><td colspan="1" rowspan="1">100(360/360)</td><td colspan="1" rowspan="1">100(360/360)</td><td colspan="1" rowspan="1">100(360/360)</td><td colspan="1" rowspan="1">100(360/360)</td><td colspan="1" rowspan="1">99.8(2156/2160)</td></tr><tr><td colspan="2" rowspan="1">99.5(716/720)</td><td colspan="1" rowspan="1">100(720/720)</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">100(720/720)</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="2">Mod</td><td colspan="1" rowspan="1">99.7(359/360)</td><td colspan="1" rowspan="1">100(360/360)</td><td colspan="1" rowspan="1">100(360/360)</td><td colspan="1" rowspan="1">100(360/360)</td><td colspan="1" rowspan="1">100(360/360)</td><td colspan="1" rowspan="1">100(360/360)</td><td colspan="1" rowspan="1">100(2159/2160)</td></tr><tr><td colspan="2" rowspan="1">99.9(719/720)</td><td colspan="1" rowspan="1">100(720/720)</td><td colspan="1" rowspan="1">100</td><td colspan="2" rowspan="1">100(720/720)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="2">5</td><td colspan="1" rowspan="2">Negative</td><td colspan="1" rowspan="2">N/A</td><td colspan="1" rowspan="1">100(420/420)</td><td colspan="1" rowspan="1">100(420/420)</td><td colspan="1" rowspan="1">100(420/420)</td><td colspan="1" rowspan="1">100(420/420)</td><td colspan="1" rowspan="1">100(420/420)</td><td colspan="1" rowspan="1">100(420/420)</td><td colspan="1" rowspan="1">100(2520/2520)</td></tr><tr><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1"></td></tr></table>

Low: 1.5X LoD; Mod (Moderate): 5X LoD, except where noted Neg.: Negative; N/A: Not applicable 1 Formulated based on the LoD for the applicable influenza A subtype. 2 Formulated at approximately 3X (Low) and 9X LoD (Moderate). 3 Expected results based on the respective analytical sensitivities of the H1pdm09 subtyping and influenza A assays: H1pdm09 subtype “positive”, Influenza A “positive”. 4 Expected results based on the relative analytical sensitivities of the H3 subtyping and influenza A assays: H3 subtype “positive”, Influenza A “positive” or “negative”; agreement for Influenza A is included under “All Other” $[ \boldsymbol { \mathrm { n } } = 4 0$ per site). 5 Formulated at approximately 2X (Low) and 7X LoD (Moderate).

2. Linearity:

Not applicable.

3. Analytical Specificity/Interference:

# In silico Analysis

An in silico BLAST analysis was performed to evaluate the potential for cross-reaction of the NxTAG RPP v2 oligonucleotides with nucleic acids from on- and off-panel species. The analysis was performed with all the primers and probes in the assay mixture against sequences in the GenBank nt sequence database as of April 9, 2023. Prediction of potential cross-reaction/interference was based on the percentage homology with the non-target sequences, the relative locations of potential hybridization (amplicon length), the location of any mismatches and predicted $\mathrm { T } _ { \mathrm { m } }$ . Based on this analysis, the NxTAG RPP v2 SARS-CoV-2 primers and probe were determined to cross-react with some bat coronavirus and bat-SARSlike coronavirus strains. In addition, the Coronavirus 229E primers and probe are predicted to cross-react with 229E-like coronavirus sequences from bats. This observation was confirmed empirically using synthetic template. The potential for cross-reaction of the Coronavirus

NL63 primers and probes with bat-coronaviruses was demonstrated in silico but was not confirmed through functional testing. Overall, in silico analysis demonstrated low likelihood for cross-reaction of the NxTAG RPP v2 assay primers and probes with non-target nucleic acids. However, the potential for cross-reaction with some bat SARS-like and 229E-like coronavirus strains is noted as a Limitation in the device labeling.

Laboratory Testing of Analytical Specificity

The analytical specificity of the NxTAG RPP v2 was evaluated by testing high concentrations of potentially cross-reactive on- and off-panel bacteria, fungi and viruses, in addition to pooled nasal wash. No unexpected cross-reaction was observed except with Enterovirus 68 (Species D, US/IL/14-18952), which produced false-positive results for Influenza A H3 (Tables 7 and 8). This is noted as a Limitation in the device labeling.

Table 7. Off-panel bacteria, fungi and viruses evaluated for potential cross-reaction   

<table><tr><td rowspan=1 colspan=1>Species</td><td rowspan=1 colspan=1>Strain/Isolate</td><td rowspan=1 colspan=1>Source/Catalogue No.</td><td rowspan=1 colspan=1>Concentrationer mL</td><td rowspan=1 colspan=1>Cross-reaction</td></tr><tr><td rowspan=1 colspan=1>Bordetella parapertussis</td><td rowspan=1 colspan=1>A747</td><td rowspan=1 colspan=1>Zeptometrix 0801641</td><td rowspan=1 colspan=1>1.00 x 106 CFU</td><td rowspan=1 colspan=1>None1</td></tr><tr><td rowspan=1 colspan=1>Bordetella pertussis</td><td rowspan=1 colspan=1>A639</td><td rowspan=1 colspan=1>Zeptometrix 0801459</td><td rowspan=1 colspan=1>1.00 x 106 CFU</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Chlamydia trachomatis</td><td rowspan=1 colspan=1>Z054; D-UW3; Serovar D</td><td rowspan=1 colspan=1>Zeptometrix 0801775</td><td rowspan=1 colspan=1>1.00 x 106 IFU</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Corynebacterium diphtheriae</td><td rowspan=1 colspan=1>Z116</td><td rowspan=1 colspan=1>Zeptometrix 0801882</td><td rowspan=1 colspan=1>1.00 x 106 CFU</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Corynebacterium striatum</td><td rowspan=1 colspan=1>FS1</td><td rowspan=1 colspan=1>Zeptometrix BAA-1851</td><td rowspan=1 colspan=1>1.00 x 106 CFU</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Escherichia coli</td><td rowspan=1 colspan=1>0157:H7; EDL 933</td><td rowspan=1 colspan=1>Zeptometrix 0801622</td><td rowspan=1 colspan=1>1.00 x 106 CFU</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Fusobacterium necrophorum</td><td rowspan=1 colspan=1>Z239</td><td rowspan=1 colspan=1>Zeptometrix 0804189</td><td rowspan=1 colspan=1>1.00 x 106 CFU</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Haemophilus influenzae</td><td rowspan=1 colspan=1>Type b; MinnA</td><td rowspan=1 colspan=1>Zeptometrix 0801680</td><td rowspan=1 colspan=1>1.00 x 106 CFU</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Klebsiella pneumoniae</td><td rowspan=1 colspan=1>Z135; IMP-13, CTX-M</td><td rowspan=1 colspan=1>Zeptometrix 0801904</td><td rowspan=1 colspan=1>1.00 x 106CFU</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Lactobacillus acidophilus</td><td rowspan=1 colspan=1>Z048</td><td rowspan=1 colspan=1>Zeptometrix 0801540</td><td rowspan=1 colspan=1>1.00 x 106 CFU</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Lactobacillus plantarum</td><td rowspan=1 colspan=1>Z524</td><td rowspan=1 colspan=1>Zeptometrix 0804436</td><td rowspan=1 colspan=1>1.00 x 106 CFU</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Legionella (Tatlockia) micdadei</td><td rowspan=1 colspan=1>Tatlock</td><td rowspan=1 colspan=1>Zeptometrix 0801576</td><td rowspan=1 colspan=1>1.00 x 106 CFU</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Legionella pneumophila</td><td rowspan=1 colspan=1>Philadelphia</td><td rowspan=1 colspan=1>Zeptometrix 0801645</td><td rowspan=1 colspan=1>1.00 x 106 CFU</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Moraxella catarrhalis</td><td rowspan=1 colspan=1>Strain NE 11</td><td rowspan=1 colspan=1>Zeptometrix 0801509</td><td rowspan=1 colspan=1>1.00 x 106 CFU</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Mycobacterium tuberculosis</td><td rowspan=1 colspan=1>H37Rx-1</td><td rowspan=1 colspan=1>Zeptometrix 0801660</td><td rowspan=1 colspan=1>1.00 x 106 CFU</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Mycoplasma genitalium</td><td rowspan=1 colspan=1>UMTB-10G</td><td rowspan=1 colspan=1>ATCC 49899</td><td rowspan=1 colspan=1>1.00 x 105CCU 2</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Mycoplasma hominis</td><td rowspan=1 colspan=1>Z317</td><td rowspan=1 colspan=1>Zeptometrix 0804011</td><td rowspan=1 colspan=1>1.00 x 106 CCU</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Neisseria elongate</td><td rowspan=1 colspan=1>Z071</td><td rowspan=1 colspan=1>Zeptometrix 0801510</td><td rowspan=1 colspan=1>1.00 x 106 CFU</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Neisseria gonorrhoeae</td><td rowspan=1 colspan=1>Z017</td><td rowspan=1 colspan=1>Zeptometrix 0801482</td><td rowspan=1 colspan=1>1.00 x 106 CFU</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Neisseria meningitidis</td><td rowspan=1 colspan=1>Serotype A</td><td rowspan=1 colspan=1>Zeptometrix 0801511</td><td rowspan=1 colspan=1>1.00 x 106 CFU</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Pseudomonas aeruginosa</td><td rowspan=1 colspan=1>Clinical isolate</td><td rowspan=1 colspan=1>Zeptometrix 0801519</td><td rowspan=1 colspan=1>1.00 x 106 CFU</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Serratia marcescens</td><td rowspan=1 colspan=1>Z053</td><td rowspan=1 colspan=1>Zeptometrix 0801723</td><td rowspan=1 colspan=1>1.00 x 106 CFU</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus aureus</td><td rowspan=1 colspan=1>102-04</td><td rowspan=1 colspan=1>Zeptometrix BAA-1765</td><td rowspan=1 colspan=1>1.00 x 106CFU</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus epidermidis</td><td rowspan=1 colspan=1>MRSE, RP62A</td><td rowspan=1 colspan=1>Zeptometrix 0801651</td><td rowspan=1 colspan=1>1.00 x 106CFU</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Streptococcus agalactiae</td><td rowspan=1 colspan=1>Z019</td><td rowspan=1 colspan=1>Zeptometrix 0801545</td><td rowspan=1 colspan=1>1.00 x 106 CFU</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Streptococcus pneumoniae</td><td rowspan=1 colspan=1>Z022</td><td rowspan=1 colspan=1>Zeptometrix 0801439</td><td rowspan=1 colspan=1>1.00 x 106 CFU</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Streptococcus pyogenes</td><td rowspan=1 colspan=1>Z018</td><td rowspan=1 colspan=1>Zeptometrix 0801512</td><td rowspan=1 colspan=1>1.00 x 106CFU</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Streptococcus salivarius</td><td rowspan=1 colspan=1>Z127</td><td rowspan=1 colspan=1>Zeptometrix 0801896</td><td rowspan=1 colspan=1>1.00 x 106 CFU</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Cytomegalovirus</td><td rowspan=1 colspan=1>Merlin</td><td rowspan=1 colspan=1>Zeptometrix 0810500CF</td><td rowspan=1 colspan=1>1.00 x 105 TCID50</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Epstein Barr Virus</td><td rowspan=1 colspan=1>B95-8</td><td rowspan=1 colspan=1>Zeptometrix 0810008CF</td><td rowspan=1 colspan=1>1.00 x 107 copies</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Herpes Simplex virus Type 1</td><td rowspan=1 colspan=1>Macintyre</td><td rowspan=1 colspan=1>Zeptometrix 0810005CF</td><td rowspan=1 colspan=1>1.00 x 105 TCID50</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Human Bocavirus</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>Clinical specimen</td><td rowspan=1 colspan=1>1.00 x 107 copies</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Measles Virus</td><td rowspan=1 colspan=1>--</td><td rowspan=1 colspan=1>Zeptometrix 0810025CF</td><td rowspan=1 colspan=1>1.00 x 105 TCID50</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>MERS-coronavirus</td><td rowspan=1 colspan=1>Florida/USA-2 Saudi Arabia 2014</td><td rowspan=1 colspan=1>Zeptometrix 0810575CFHI</td><td rowspan=1 colspan=1>1.00 x 105 TCID50</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Mumps Virus</td><td rowspan=1 colspan=1>--</td><td rowspan=1 colspan=1>Zeptometrix 0810079CF</td><td rowspan=1 colspan=1>1.00 x 105 TCID50</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>SARS-coronavirus</td><td rowspan=1 colspan=1>2003-00592</td><td rowspan=1 colspan=1>Zeptometrix NATSARS-ST</td><td rowspan=1 colspan=1>3.01 x 105 copies</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Varicella Zoster Virus</td><td rowspan=1 colspan=1>Ellen</td><td rowspan=1 colspan=1>Zeptometrix 0810171CF</td><td rowspan=1 colspan=1>1.00 x 105 TCID50</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Aspergillus flavus</td><td rowspan=1 colspan=1>Z013</td><td rowspan=1 colspan=1>Zeptometrix 0801598</td><td rowspan=1 colspan=1>1.00 x 106 CFU</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Aspergillus fumigatus</td><td rowspan=1 colspan=1>QM 1981</td><td rowspan=1 colspan=1>ATCC 1022</td><td rowspan=1 colspan=1>1.00 x 106 CFU</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Candida albicans</td><td rowspan=1 colspan=1>Z006</td><td rowspan=1 colspan=1>Zeptometrix 0801504</td><td rowspan=1 colspan=1>1.00 x 106 CFU</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Pneumocystis carinii</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>ATCC PRA-159</td><td rowspan=1 colspan=1>1.00 x 106nuclei</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Pooled nasal wash</td><td rowspan=1 colspan=1>--</td><td rowspan=1 colspan=1>--</td><td rowspan=1 colspan=1>--</td><td rowspan=1 colspan=1>None</td></tr></table>

CCU: Color Changing Units; CFU: Colony Forming Units; IFU: Inclusion Forming Units; TCID50: $5 0 \%$ Tissue Culture Infectious Dose 1 On initial testing, 1/3 replicates gave a positive result for Rhinovirus/Enterovirus; upon repeat testing of the same samples, 3/3 replicates produced negative results for all analytes. A further 3/3 replicates prepared in negative nasopharyngeal swab matrix also produced negative results for all analytes. 2 Approximate concentration based on supplier Certificate of Analysis.

Table 8. On-panel viruses and bacteria evaluated for potential cross-reaction   

<table><tr><td rowspan=1 colspan=1>Analyte</td><td rowspan=1 colspan=1>Strain/Isolate</td><td rowspan=1 colspan=1>Source/Catalogue No.</td><td rowspan=1 colspan=1>Concentrationper mL</td><td rowspan=1 colspan=1>Cross-reaction</td></tr><tr><td rowspan=1 colspan=1>Adenovirus B</td><td rowspan=1 colspan=1>Type 14, 2006 isolate</td><td rowspan=1 colspan=1>Zeptometrix 0810300CF</td><td rowspan=1 colspan=1>1.00 x 105 TCID50</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Coronavirus 229E</td><td rowspan=1 colspan=1>--</td><td rowspan=1 colspan=1>ATCC VR-740</td><td rowspan=1 colspan=1>1.00 x 105 TCID50</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Coronavirus HKU1</td><td rowspan=1 colspan=1>Genotype A</td><td rowspan=1 colspan=1>Clinical</td><td rowspan=1 colspan=1>1.00 x 106 copies</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Coronavirus NL63</td><td rowspan=1 colspan=1>--</td><td rowspan=1 colspan=1>Zeptometrix 0810228CF</td><td rowspan=1 colspan=1>1.00 x 105 TCID50</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Coronavirus OC43</td><td rowspan=1 colspan=1>--</td><td rowspan=1 colspan=1>ATCC VR-1558</td><td rowspan=1 colspan=1>1.00 x 105 TCID50</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Enterovirus</td><td rowspan=1 colspan=1>Species D, Type 68, 2014 Isolate 1</td><td rowspan=1 colspan=1>Zeptometrix 0810300CF</td><td rowspan=1 colspan=1>1.00 x 105 TCID50</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Enterovirus</td><td rowspan=1 colspan=1>Species D, Type 68, 2007 Isolate</td><td rowspan=1 colspan=1>Zeptometrix 0810237CF</td><td rowspan=1 colspan=1>1.00 x 105 TCID50</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Enterovirus</td><td rowspan=1 colspan=1>Species D, Type 68, Fermon</td><td rowspan=1 colspan=1>ATCC VR-1826</td><td rowspan=1 colspan=1>1.00 x 105 TCID50</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Enterovirus</td><td rowspan=1 colspan=1>Species D, Type 68, US/IL/14-18952</td><td rowspan=1 colspan=1>ATCC VR-1824</td><td rowspan=1 colspan=1>1.00 x 103 TCID50</td><td rowspan=1 colspan=1>Influenza A H31</td></tr><tr><td rowspan=1 colspan=1>Enterovirus</td><td rowspan=1 colspan=1>Species D, Type 68, US/KY/14-18953</td><td rowspan=1 colspan=1>ATCC VR-1825</td><td rowspan=1 colspan=1>1.00 x 105 TCID50</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Enterovirus</td><td rowspan=1 colspan=1>Species D, Type 68, US/MO/14-18947</td><td rowspan=1 colspan=1>ATCC VR-1823</td><td rowspan=1 colspan=1>1.00 x 105 TCID50</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Human Metapneumovirus</td><td rowspan=1 colspan=1>16, Type A1, IA10-2003</td><td rowspan=1 colspan=1>Zeptometrix 0810161CF</td><td rowspan=1 colspan=1>1.00 x 105 TCID50</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Human Metapneumovirus</td><td rowspan=1 colspan=1>3, Type B1, Peru2-2002</td><td rowspan=1 colspan=1>Zeptometrix 0810156CF</td><td rowspan=1 colspan=1>3.89 x 104 TCID502</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Influenza A H1pdm09</td><td rowspan=1 colspan=1>A/NY/02/09</td><td rowspan=1 colspan=1>Zeptometrix 0810109CFN</td><td rowspan=1 colspan=1>1.00 x 105 TCID50</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Influenza A H1</td><td rowspan=1 colspan=1>A/Brisbane/59/07</td><td rowspan=1 colspan=1>Zeptometrix 0810244CF</td><td rowspan=1 colspan=1>1.00 x 105 TCID50</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Influenza A H3</td><td rowspan=1 colspan=1>A/Wisconsin/67/05</td><td rowspan=1 colspan=1>Zeptometrix 0810252CF</td><td rowspan=1 colspan=1>1.00 x 105 TCID50</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Influenza B</td><td rowspan=1 colspan=1>B/Bangladesh/5972/2007</td><td rowspan=1 colspan=1>IRR FR-450</td><td rowspan=1 colspan=1>1.00 x 105 TCID50</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Influenza B</td><td rowspan=1 colspan=1>B/Brigit (B/Russia/69)</td><td rowspan=1 colspan=1>ATCC VR-786</td><td rowspan=1 colspan=1>1.00 x 105 CEID50</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Influenza B</td><td rowspan=1 colspan=1>B/Brisbane/3/2007</td><td rowspan=1 colspan=1>IRR FR-18</td><td rowspan=1 colspan=1>1.00 x 105 CEID50</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Influenza B</td><td rowspan=1 colspan=1>B/Brisbane/33/2008</td><td rowspan=1 colspan=1>Zeptometrix 0810253CF</td><td rowspan=1 colspan=1>1.00 x 105 TCID50</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Influenza B</td><td rowspan=1 colspan=1>B/GreLakes/1739/1954</td><td rowspan=1 colspan=1>BEI NR-3179</td><td rowspan=1 colspan=1>1.00 x 105 CEID50</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Influenza B</td><td rowspan=1 colspan=1>B/Hong Kong/259/2010</td><td rowspan=1 colspan=1>IRR FR-663</td><td rowspan=1 colspan=1>1.00 x 105 CEID50</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Influenza B</td><td rowspan=1 colspan=1>B/Hong Kong/5/72</td><td rowspan=1 colspan=1>ATCC VR-823</td><td rowspan=1 colspan=1>1.00 x 105 CEID50</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Influenza B</td><td rowspan=1 colspan=1>B/Hubei-Wujiagang/158/2009</td><td rowspan=1 colspan=1>IRR FR-469</td><td rowspan=1 colspan=1>1.00 x 105 CEID50</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Influenza B</td><td rowspan=1 colspan=1>B/Massachusetts/2/12</td><td rowspan=1 colspan=1>Zeptometrix 0810239CF</td><td rowspan=1 colspan=1>1.00 x 105 TCID50</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Influenza B</td><td rowspan=1 colspan=1>B/New Jersey/1/2012</td><td rowspan=1 colspan=1>IRR FR-1270</td><td rowspan=1 colspan=1>1.00 x 105 TCID50</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Influenza B</td><td rowspan=1 colspan=1>B/Russia/69</td><td rowspan=1 colspan=1>ATCC VR-790</td><td rowspan=1 colspan=1>1.00 x 105 CEID50</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Influenza B</td><td rowspan=1 colspan=1>B/Texas/02/2013</td><td rowspan=1 colspan=1>IRR FR-1302</td><td rowspan=1 colspan=1>1.00 x 105 TCID50</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Influenza B</td><td rowspan=1 colspan=1>B/Wisconsin/1/2010</td><td rowspan=1 colspan=1>Zeptometrix 0810241CF</td><td rowspan=1 colspan=1>1.00 x 105 TCID50</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Influenza B</td><td rowspan=1 colspan=1>B/Florida/02/06</td><td rowspan=1 colspan=1>Zeptometrix 0810037CF</td><td rowspan=1 colspan=1>1.00 x 105 TCID50</td><td rowspan=1 colspan=1>None 3</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza virus 1</td><td rowspan=1 colspan=1>Type 1</td><td rowspan=1 colspan=1>Zeptometrix 0810014CF</td><td rowspan=1 colspan=1>1.00 x 105 TCID50</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza virus 2</td><td rowspan=1 colspan=1>Greer</td><td rowspan=1 colspan=1>ATCC VR-92</td><td rowspan=1 colspan=1>1.00 x 105 TCID50</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza virus 3</td><td rowspan=1 colspan=1>C243</td><td rowspan=1 colspan=1>ATCC VR-93</td><td rowspan=1 colspan=1>1.00 x 105 TCID50</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza virus 4A</td><td rowspan=1 colspan=1>--</td><td rowspan=1 colspan=1>Zeptometrix 0810060CF</td><td rowspan=1 colspan=1>1.00 105 TCID50</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza virus 4B</td><td rowspan=1 colspan=1>CH 19503</td><td rowspan=1 colspan=1>ATCC VR-1377</td><td rowspan=1 colspan=1>1.00 x 105 TCID50</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Respiratory Syncytial Virus A</td><td rowspan=1 colspan=1>A2</td><td rowspan=1 colspan=1>ATCC VR-1540</td><td rowspan=1 colspan=1>1.00 x 105 PFU</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Respiratory Syncytial Virus B</td><td rowspan=1 colspan=1>18537</td><td rowspan=1 colspan=1>ATCC VR-1580</td><td rowspan=1 colspan=1>7.00 x 104 PFU</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Rhinovirus</td><td rowspan=1 colspan=1>Type 85, 50-525-CV54 [V-192-001-021]</td><td rowspan=1 colspan=1>ATCC VR-1195</td><td rowspan=1 colspan=1>1.00 x 105 TCID50</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>SARS-CoV-2</td><td rowspan=1 colspan=1>USA-WA-1/2020</td><td rowspan=1 colspan=1>ATCC VR-1986HK</td><td rowspan=1 colspan=1>1.00 x 107 copies</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Chlamydia pneumoniae</td><td rowspan=1 colspan=1>TW-183</td><td rowspan=1 colspan=1>ATCC VR-2282</td><td rowspan=1 colspan=1>1.00 x 106 IFU</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Mycoplasma pneumoniae</td><td rowspan=1 colspan=1>M129</td><td rowspan=1 colspan=1>Zeptometrix 0801579</td><td rowspan=1 colspan=1>1.00 x 106 CCU</td><td rowspan=1 colspan=1>None</td></tr></table>

CCU: Color Changing Units; $\mathrm { C E I D } _ { 5 0 }$ ; $5 0 \%$ Chicken Embryo Infectious Dose; IFU: Inclusion Forming Units; PFU: Plaque Forming Units; $\mathrm { T C I D } { 5 0 }$ : $5 0 \%$ Tissue Culture Infectious Dose 1 Cross-reaction observed at levels $\ge 1 . 0 0 \mathrm { ~ x ~ } 1 0 ^ { 3 } \mathrm { T C I D } 5 0 / \mathrm { m L }$ ; no cross-reaction observed at $1 . 0 0 \mathrm { ~ x ~ } 1 0 ^ { 2 }$ T $\mathrm { \dot { C } I D 5 0 / m L }$ ; this is noted as a Limitation in the device labeling. 2 Highest attainable concentration. 3 3/9 replicates produced false-positive results for Coronavirus 229E due to suspected contamination of the viral stock; positive results were not obtained with any other strains of Influenza B.

# Off-Panel Microbial Interference

The potential for interference with the NxTAG RPP by off-panel microorganisms that may be present in nasopharyngeal swab specimens was evaluated by testing 11 potentially crossreactive species (Table 9) at high concentration in the presence of low-moderate levels of 22 on-panel target analytes, including different subtypes of influenza A. Testing was performed with combinations of two or three off-panel analytes at high concentration in the presence of different combinations of three on-panel analytes, each at ${ \sim } 3 – 6 \mathrm { X }$ LoD. The expected positive results were obtained for each of the on-panel analytes, with no evidence of interference from the off-panel species.

Table 9. Off-panel species evaluated for potential interference with the NxTAG RPP v2   

<table><tr><td rowspan=1 colspan=1>Species/</td><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>Source/Catalogue No.</td><td rowspan=1 colspan=1>Concentrationper mL</td><td rowspan=1 colspan=1>Interference</td></tr><tr><td rowspan=1 colspan=1>Bordetella pertussis</td><td rowspan=1 colspan=1>A639</td><td rowspan=1 colspan=1>Zeptometrix 0801459</td><td rowspan=1 colspan=1>1.00 x 106 CFU</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Corynebacterium diphtheriae</td><td rowspan=1 colspan=1>Z116</td><td rowspan=1 colspan=1>Zeptometrix 0801882</td><td rowspan=1 colspan=1>1.00 x 106 CFU</td><td rowspan=1 colspan=1>None 1</td></tr><tr><td rowspan=1 colspan=1>Haemophilus influenzae</td><td rowspan=1 colspan=1>Type b; MinnA</td><td rowspan=1 colspan=1>Zeptometrix 0801680</td><td rowspan=1 colspan=1>1.00 x 106 CFU</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Moraxella catarrhalis</td><td rowspan=1 colspan=1>Strain NE 11</td><td rowspan=1 colspan=1>Zeptometrix 0801509</td><td rowspan=1 colspan=1>1.00 x 106 CFU</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Neisseria meningitidis</td><td rowspan=1 colspan=1>Serotype A</td><td rowspan=1 colspan=1>Zeptometrix 0801511</td><td rowspan=1 colspan=1>1.00 x 106 CFU</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Pseudomonas aeruginosa</td><td rowspan=1 colspan=1>Clinical</td><td rowspan=1 colspan=1>Zeptometrix 0801519</td><td rowspan=1 colspan=1>1.00 x 106 CFU</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus aureus</td><td rowspan=1 colspan=1>102-04</td><td rowspan=1 colspan=1>Zeptometrix BAA-1765</td><td rowspan=1 colspan=1>1.00 x 106 CFU</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Streptococcus pneumoniae</td><td rowspan=1 colspan=1>Z022</td><td rowspan=1 colspan=1>Zeptometrix 0801439</td><td rowspan=1 colspan=1>1.00 x 106 CFU</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Cytomegalovirus</td><td rowspan=1 colspan=1>Merlin</td><td rowspan=1 colspan=1>Zeptometrix 0810500CF</td><td rowspan=1 colspan=1>1.00 x 105 TCID50</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Measles Virus</td><td rowspan=1 colspan=1>--</td><td rowspan=1 colspan=1>Zeptometrix 0810025CF</td><td rowspan=1 colspan=1>1.00 x 105 TCID50</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Mumps Virus</td><td rowspan=1 colspan=1>--</td><td rowspan=1 colspan=1>Zeptometrix 0810079CF</td><td rowspan=1 colspan=1>1.00 x 105 TCID50</td><td rowspan=1 colspan=1>None</td></tr></table>

CFU: Colony Forming Units; $\mathrm { T C I D } 5 0$ : $5 0 \%$ Tissue Culture Infectious Dose On initial testing, 1/3 samples containing high levels of Corynebacterium diphtheriae produced false negative results for Coronavirus NL63 and Coronavirus HKU1 but produced the expected positive results for both analytes upon repeat.

# On-Panel Competitive Interference

The potential for competitive interference between viral and bacterial species targeted by the NxTAG RPP v2 assay was evaluated by testing combinations of on panel analytes at asymmetric concentrations. Analytes at low-moderate concentration (3-6X LoD) were combined into four groups (Table 10) that were each tested in the presence of a single high concentration analyte (Table 11) in simulated nasopharyngeal swab matrix. In all cases $[ \mathfrak { n } =$ 3 ea.), each analyte was successfully detected irrespective of the concentration tested, demonstrating the absence of assay interference.

Table 10. Combinations of on-panel analytes at 3-6X LoD used to evaluate potential competitive interference   

<table><tr><td rowspan=1 colspan=1>Sample</td><td rowspan=1 colspan=1>Analyte</td><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>Source/Catalogue No.</td></tr><tr><td rowspan=3 colspan=1>1</td><td rowspan=1 colspan=1>Influenza A H1pdm09 (subtype) 1</td><td rowspan=1 colspan=1>A/NY/02/09</td><td rowspan=1 colspan=1>Zeptometrix 0810109CFN</td></tr><tr><td rowspan=1 colspan=1>Respiratory Syncytial Virus A</td><td rowspan=1 colspan=1>A2</td><td rowspan=1 colspan=1>ATCC VR-1540</td></tr><tr><td rowspan=1 colspan=1>Rhinovirus</td><td rowspan=1 colspan=1>50-525-CV 54</td><td rowspan=1 colspan=1>ATCC VR-1195</td></tr><tr><td rowspan=2 colspan=1>2</td><td rowspan=1 colspan=1>Respiratory Syncytial Virus B</td><td rowspan=1 colspan=1>18537</td><td rowspan=1 colspan=1>ATCC VR-1580</td></tr><tr><td rowspan=1 colspan=1>Influenza A H3 (subtype)</td><td rowspan=1 colspan=1>A/Wisconsin/67/05</td><td rowspan=1 colspan=1>Zeptometrix 0810252CF</td></tr><tr><td rowspan=3 colspan=1>3</td><td rowspan=1 colspan=1>Influenza B</td><td rowspan=1 colspan=1>B/Florida/02/06</td><td rowspan=1 colspan=1>Zeptometrix 0810037CF</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza virus 3</td><td rowspan=1 colspan=1>C 243</td><td rowspan=1 colspan=1>ATCC VR-93</td></tr><tr><td rowspan=1 colspan=1>Mycoplasma pneumoniae</td><td rowspan=1 colspan=1>M129</td><td rowspan=1 colspan=1>Zeptometrix 0801579</td></tr><tr><td rowspan=3 colspan=1>4</td><td rowspan=1 colspan=1>Adenovirus B</td><td rowspan=1 colspan=1>Type 14</td><td rowspan=1 colspan=1>Zeptometrix 0810108CF</td></tr><tr><td rowspan=1 colspan=1>Human Metapneumovirus B1</td><td rowspan=1 colspan=1>hMPV-3</td><td rowspan=1 colspan=1>Zeptometrix 0810156CF</td></tr><tr><td rowspan=1 colspan=1>SARS-CoV-2</td><td rowspan=1 colspan=1>USA-WA1/2020</td><td rowspan=1 colspan=1>ATCC VR-1986HK</td></tr></table>

CCU: Color Changing Units; PFU: Plaque Forming Units; TCID50: $5 0 \%$ Tissue Culture Infectious Dose 1 Target level based on the LoD for the applicable subtype.

Table 11. On-panel analytes tested at high concentration to evaluate potential competitive interference   

<table><tr><td rowspan=1 colspan=1>Analyte</td><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>Source/Catalogue No.</td><td rowspan=1 colspan=1>Concentrationper mL</td></tr><tr><td rowspan=1 colspan=1>Adenovirus B</td><td rowspan=1 colspan=1>Type 14</td><td rowspan=1 colspan=1>Zeptometrix 0810108CF</td><td rowspan=1 colspan=1>1.00 x 105 TCID50</td></tr><tr><td rowspan=1 colspan=1>Coronavirus NL63</td><td rowspan=1 colspan=1>--</td><td rowspan=1 colspan=1>Zeptometrix 0810228CF</td><td rowspan=1 colspan=1>1.00 x 105 TCID50</td></tr><tr><td rowspan=1 colspan=1>Influenza A H1pdm09</td><td rowspan=1 colspan=1>A/NY/02/09</td><td rowspan=1 colspan=1>Zeptometrix 0810109CFN</td><td rowspan=1 colspan=1>1.00 x 105 TCID50</td></tr><tr><td rowspan=1 colspan=1>Influenza A H3</td><td rowspan=1 colspan=1>A/Wisconsin/67/05</td><td rowspan=1 colspan=1>Zeptometrix 0810252CF</td><td rowspan=1 colspan=1>1.00 x 105 TCID50</td></tr><tr><td rowspan=1 colspan=1>Influenza B</td><td rowspan=1 colspan=1>B/Florida/02/06</td><td rowspan=1 colspan=1>Zeptometrix 0810037CF</td><td rowspan=1 colspan=1>1.00 x 105 TCID50</td></tr><tr><td rowspan=1 colspan=1>Human Metapneumovirus</td><td rowspan=1 colspan=1>hMPV-16, Type A1, IA10-2003</td><td rowspan=1 colspan=1>Zeptometrix 0810161CF</td><td rowspan=1 colspan=1>1.00 x 105 TCID50</td></tr><tr><td rowspan=1 colspan=1>Human Metapneumovirus</td><td rowspan=1 colspan=1>hMPV-3, Type B1, Peru2-2002</td><td rowspan=1 colspan=1>Zeptometrix 0810156CF</td><td rowspan=1 colspan=1>3.50 x 104 TCID50</td></tr><tr><td rowspan=1 colspan=1>Human Metapneumovirus</td><td rowspan=1 colspan=1>hMPV-5, Type B1, Peru3-2003 G gene</td><td rowspan=1 colspan=1>Zeptometrix 0810158CF</td><td rowspan=1 colspan=1>1.00 x 105 TCID50</td></tr><tr><td rowspan=1 colspan=1>Respiratory Syncytial Virus B</td><td rowspan=1 colspan=1>B/18537</td><td rowspan=1 colspan=1>ATCC VR-1580</td><td rowspan=1 colspan=1>6.30 x 104 PFU</td></tr><tr><td rowspan=1 colspan=1>Respiratory Syncytial Virus B</td><td rowspan=1 colspan=1>B/WV/14617/85</td><td rowspan=1 colspan=1>ATCC VR-1400</td><td rowspan=1 colspan=1>1.00 x 105 TCID50</td></tr><tr><td rowspan=1 colspan=1>Rhinovirus</td><td rowspan=1 colspan=1>50-525-CV 54 [V-192-001-021]</td><td rowspan=1 colspan=1>ATCC VR-1195</td><td rowspan=1 colspan=1>1.00 x 105 TCID50</td></tr><tr><td rowspan=1 colspan=1>SARS-CoV-2</td><td rowspan=1 colspan=1>USA-WA1/2020</td><td rowspan=1 colspan=1>VR-1986HK</td><td rowspan=1 colspan=1>1.00 x 107 copies</td></tr></table>

PFU: Plaque Forming Units; TCID50: $5 0 \%$ Tissue Culture Infectious Dose

Because there is limited information with respect to co-infections involving SARS-CoV-2, additional evaluation of the potential for competitive interference was performed with low concentrations of SARS-CoV-2 (3X LoD) in the presence of high concentrations of 13 on-panel analytes that were not included in Table 11 (Table 12). In all cases $\mathrm { \Delta } \mathrm { n } = 3$ ea.), positive results were reported for both SARS-CoV-2 and the high concentration analyte, demonstrating the absence of competitive interference.

Table 12. On-panel analytes tested at high concentration to evaluate potential competitive interference with the detection of SARS-CoV-2   

<table><tr><td rowspan=1 colspan=1>Analyte</td><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>Source/Catalogue No.</td><td rowspan=1 colspan=1>Concentration per mL</td></tr><tr><td rowspan=1 colspan=1>Coronavirus 229E</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>ATCC VR-740</td><td rowspan=1 colspan=1>1.00 x 105 TCID50</td></tr><tr><td rowspan=1 colspan=1>Coronavirus OC43</td><td rowspan=1 colspan=1>Betacoronavirus 1</td><td rowspan=1 colspan=1>ATCC VR-1558</td><td rowspan=1 colspan=1>1.00 x 105 TCID50</td></tr><tr><td rowspan=1 colspan=1>Coronavirus HKU1</td><td rowspan=1 colspan=1>Clinical specimen</td><td rowspan=1 colspan=1>--</td><td rowspan=1 colspan=1>1.00 x 106 copies</td></tr><tr><td rowspan=1 colspan=1>Enterovirus</td><td rowspan=1 colspan=1>Type 68, 2007 isolate</td><td rowspan=1 colspan=1>Zeptometrix 0810237CF</td><td rowspan=1 colspan=1>1.00 x 105 PFU</td></tr><tr><td rowspan=1 colspan=1>Influenza A H1</td><td rowspan=1 colspan=1>A/Brisbane/59/07</td><td rowspan=1 colspan=1>Zeptometrix 0810244CF</td><td rowspan=1 colspan=1>1.00 x 105 TCID50</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza virus 1</td><td rowspan=1 colspan=1>Type 1</td><td rowspan=1 colspan=1>Zeptometrix 0810014CF</td><td rowspan=1 colspan=1>1.00 x 105 TCID50</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza virus 2</td><td rowspan=1 colspan=1>Greer</td><td rowspan=1 colspan=1>ATCC VR-92</td><td rowspan=1 colspan=1>1.00 x 105 TCID50</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza virus 3</td><td rowspan=1 colspan=1>C 243</td><td rowspan=1 colspan=1>ATCC VR-93</td><td rowspan=1 colspan=1>1.00 x 105 TCID50</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza virus 4A</td><td rowspan=1 colspan=1>Type 4A</td><td rowspan=1 colspan=1>Zeptometrix 0810060CF</td><td rowspan=1 colspan=1>1.00 x 105 TCID50</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza virus 4B</td><td rowspan=1 colspan=1>CH 19503</td><td rowspan=1 colspan=1>ATCC VR-1377</td><td rowspan=1 colspan=1>1.00 x 105 TCID50</td></tr><tr><td rowspan=1 colspan=1>Respiratory Syncytial Virus A</td><td rowspan=1 colspan=1>A2</td><td rowspan=1 colspan=1>ATCC VR-1540</td><td rowspan=1 colspan=1>1.00 x 105 PFU</td></tr><tr><td rowspan=1 colspan=1>Chlamydia pneumoniae</td><td rowspan=1 colspan=1>TW-183</td><td rowspan=1 colspan=1>ATCC VR-2282</td><td rowspan=1 colspan=1>1.00 x 106 IFU</td></tr><tr><td rowspan=1 colspan=1>Mycoplasma pneumoniae</td><td rowspan=1 colspan=1>M129</td><td rowspan=1 colspan=1>Zeptometrix 0801579</td><td rowspan=1 colspan=1>1.00 x 106 CCU</td></tr></table>

CCU: Color Changing Units; IFU: Inclusion Forming Units; PFU: Plaque Forming Units; $\mathrm { T C I D } _ { 5 0 }$ : $5 0 \%$ Tissue Culture Infectious Dose

# Interfering Substances

Testing was conducted to evaluate the potential for interference with the NxTAG RPP v2 assay by endogenous and exogenous substances that may be present in nasopharyngeal swab specimens. Eight multi-analyte samples (Table 13) comprised of simulated nasopharyngeal swab matrix containing on-panel analytes at 3-6X LoD were tested in triplicate in the presence of 22 potentially interfering substances (Table 14). Each substance was also added to negative simulated nasopharyngeal matrix to evaluate the potential for false positive results.

The expected results were obtained in all cases, except in the presence of menthol and FluMist influenza vaccine. Menthol $1 \%$ (w/v) was shown to interfere with the detection of Coronavirus

OC43, although no interference was observed at lower concentration $( 0 . 5 \% \mathrm { w } / \mathrm { v } )$ . Positive results for Influenza A, Influenza A H1pdm09, Influenza A H3 and Influenza B were obtained in the presence of FluMist because the vaccine contains attenuated strains of these analytes that are detectable at the levels used in this study. The potential for false results with menthol and FluMist is noted in the Limitations section of the device labeling.

Table 13. Multi-analyte panels used to evaluate the potential for assay interference by endogenous and exogenous substances   

<table><tr><td rowspan=1 colspan=1>Sample</td><td rowspan=1 colspan=1>Analyte 1</td><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>Source</td><td rowspan=1 colspan=1>Catalogue No.</td></tr><tr><td rowspan=3 colspan=1>1</td><td rowspan=1 colspan=1>Influenza A H1pdm09 (subtype) 2</td><td rowspan=1 colspan=1>A/NY/02/09</td><td rowspan=1 colspan=1>Zeptometrix</td><td rowspan=1 colspan=1>0810109CFN</td></tr><tr><td rowspan=1 colspan=1>Respiratory Syncytial Virus A</td><td rowspan=1 colspan=1>A2</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>VR-1540</td></tr><tr><td rowspan=1 colspan=1>Rhinovirus</td><td rowspan=1 colspan=1>50-525-CV54</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>VR-1195</td></tr><tr><td rowspan=2 colspan=1>2</td><td rowspan=1 colspan=1>Influenza A H3 (subtype) 2</td><td rowspan=1 colspan=1>A/Wisconsin/67/05</td><td rowspan=1 colspan=1>Zeptometrix</td><td rowspan=1 colspan=1>0810252CF</td></tr><tr><td rowspan=1 colspan=1>Respiratory Syncytial Virus B</td><td rowspan=1 colspan=1>CH 19503</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>VR-1377</td></tr><tr><td rowspan=3 colspan=1>3</td><td rowspan=1 colspan=1>Influenza B</td><td rowspan=1 colspan=1>B/Florida/02/06</td><td rowspan=1 colspan=1>Zeptometrix</td><td rowspan=1 colspan=1>0810037CF</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza virus 3</td><td rowspan=1 colspan=1>C 243</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>VR-93</td></tr><tr><td rowspan=1 colspan=1>Mycoplasma pneumoniae</td><td rowspan=1 colspan=1>M129</td><td rowspan=1 colspan=1>Zeptometrix</td><td rowspan=1 colspan=1>0801579</td></tr><tr><td rowspan=3 colspan=1>4</td><td rowspan=1 colspan=1>SARS-CoV-2</td><td rowspan=1 colspan=1>USA-WA1/2020</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>VR-1986HK</td></tr><tr><td rowspan=1 colspan=1>Human Metapneumovirus B1</td><td rowspan=1 colspan=1>hMPV-3</td><td rowspan=1 colspan=1>Zeptometrix</td><td rowspan=1 colspan=1>0810156CF</td></tr><tr><td rowspan=1 colspan=1>Adenovirus B</td><td rowspan=1 colspan=1>Type 14</td><td rowspan=1 colspan=1>Zeptometrix</td><td rowspan=1 colspan=1>0810108CF</td></tr><tr><td rowspan=3 colspan=1>5</td><td rowspan=1 colspan=1>Influenza A H3 (matrix)</td><td rowspan=1 colspan=1>A/Wisconsin/67/05</td><td rowspan=1 colspan=1>Zeptometrix</td><td rowspan=1 colspan=1>0810252CF</td></tr><tr><td rowspan=1 colspan=1>Coronavirus NL63</td><td rowspan=1 colspan=1>--</td><td rowspan=1 colspan=1>Zeptometrix</td><td rowspan=1 colspan=1>0810228CF</td></tr><tr><td rowspan=1 colspan=1>Coronavirus HKU1</td><td rowspan=1 colspan=1>Clinical specimen</td><td rowspan=1 colspan=1>--</td><td rowspan=1 colspan=1>--</td></tr><tr><td rowspan=3 colspan=1>6</td><td rowspan=1 colspan=1>Influenza H1 (subtype) 2</td><td rowspan=1 colspan=1>A/Brisbane/59/07</td><td rowspan=1 colspan=1>Zeptometrix</td><td rowspan=1 colspan=1>0810244CF</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza virus 1</td><td rowspan=1 colspan=1>--</td><td rowspan=1 colspan=1>Zeptometrix</td><td rowspan=1 colspan=1>0810014CF</td></tr><tr><td rowspan=1 colspan=1>Chlamydia pneumoniae</td><td rowspan=1 colspan=1>TW-183</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>VR-2282</td></tr><tr><td rowspan=3 colspan=1>7</td><td rowspan=1 colspan=1>Parainfluenza virus 2</td><td rowspan=1 colspan=1>Greer</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>VR-92</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza virus 4B</td><td rowspan=1 colspan=1>CH 19503</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>VR-1377</td></tr><tr><td rowspan=1 colspan=1>Coronavirus 229E</td><td rowspan=1 colspan=1>--</td><td rowspan=1 colspan=1>Zeptometrix</td><td rowspan=1 colspan=1>0810228CF</td></tr><tr><td rowspan=2 colspan=1>8</td><td rowspan=1 colspan=1>Parainfluenza virus 4A</td><td rowspan=1 colspan=1>--</td><td rowspan=1 colspan=1>Zeptometrix</td><td rowspan=1 colspan=1>0810060CF</td></tr><tr><td rowspan=1 colspan=1>Coronavirus OC43</td><td rowspan=1 colspan=1>Betacoronavirus 1</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>VR-1558</td></tr><tr><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>--</td><td rowspan=1 colspan=1>--</td><td rowspan=1 colspan=1>--</td></tr></table>

1 Each analyte was tested at 3-6X LoD. 2 Target level based on the LoD for the applicable subtype.

Table 14. Endogenous and exogenous substances evaluated for potential interference with the NxTAG RPP v2   

<table><tr><td rowspan=1 colspan=1>Source</td><td rowspan=1 colspan=1>Substance</td><td rowspan=1 colspan=1>Active Ingredient</td><td rowspan=1 colspan=1>Concentration</td><td rowspan=1 colspan=1>Interference</td></tr><tr><td rowspan=4 colspan=1>Endogenous</td><td rowspan=1 colspan=1>Human whole blood</td><td rowspan=1 colspan=1>--</td><td rowspan=1 colspan=1>5% (v/v/)</td><td rowspan=1 colspan=1>None 1</td></tr><tr><td rowspan=1 colspan=1>Human Genomic DNA</td><td rowspan=1 colspan=1>--</td><td rowspan=1 colspan=1>20 μg/mL</td><td rowspan=1 colspan=1>None 2</td></tr><tr><td rowspan=1 colspan=1>Leukocytes</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1.00 x 103 cells/μL</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Mucin</td><td rowspan=1 colspan=1>--</td><td rowspan=1 colspan=1>100 μg/mL</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=23 colspan=1>Exogenous</td><td rowspan=1 colspan=1>Beclomethasone dipropionate</td><td rowspan=1 colspan=1>--</td><td rowspan=1 colspan=1>25 μg/mL</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Benzocaine</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>10% (w/v)</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Budesonide</td><td rowspan=1 colspan=1>--</td><td rowspan=1 colspan=1>6.30 x 10-3 μg/mL</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Dexamethasone</td><td rowspan=1 colspan=1>--</td><td rowspan=1 colspan=1>12 μg/mL</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Early Defense Nasal Spray</td><td rowspan=1 colspan=1>Zinc</td><td rowspan=1 colspan=1>5% (v/v)</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>FluMist</td><td rowspan=1 colspan=1>Influenza A H1N1Influenza A H3N2Influenza B (Yamagata)Influenza B (Victoria)</td><td rowspan=1 colspan=1>0.5% (v/v)</td><td rowspan=1 colspan=1>Yes 3</td></tr><tr><td rowspan=1 colspan=1>Flunisolide</td><td rowspan=1 colspan=1>--</td><td rowspan=1 colspan=1>5 μg/mL</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Fluticasone propionate</td><td rowspan=1 colspan=1>--</td><td rowspan=1 colspan=1>1.26 x 10-3 µg/mL</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=2 colspan=1>Menthol</td><td rowspan=2 colspan=1>--</td><td rowspan=1 colspan=1>1% (w/v)</td><td rowspan=1 colspan=1>Yes 4</td></tr><tr><td rowspan=1 colspan=1>0.5% (w/v)</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Mometasone furoate</td><td rowspan=1 colspan=1>--</td><td rowspan=1 colspan=1>4.50 x 10-4 μg/mL</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Mupirocin</td><td rowspan=1 colspan=1>--</td><td rowspan=1 colspan=1>1.5 mg/mL</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=2 colspan=1>Drixoral</td><td rowspan=2 colspan=1>Oxymetaxoline</td><td rowspan=1 colspan=1>10% (v/v)</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>15% (v/v)</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Phenylephrine</td><td rowspan=1 colspan=1>--</td><td rowspan=1 colspan=1>0.03 μg/mL</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=2 colspan=1>Salinex</td><td rowspan=2 colspan=1>Sodium chloride</td><td rowspan=1 colspan=1>1% (v/v)</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>15% (v/v)</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=2 colspan=1>Tobramycin</td><td rowspan=2 colspan=1>--</td><td rowspan=1 colspan=1>33 μg/mL</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>600 μg/mL</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Triamcinolone acetonide</td><td rowspan=1 colspan=1>--</td><td rowspan=1 colspan=1>22 μg/mL</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Zanamivir</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>100 μg/mL</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=2 colspan=1>ZICAM Allergy Relief</td><td rowspan=2 colspan=1>Galphimia glauca,Histaminum hydrochoricum,Luffa operculate, Sulfur</td><td rowspan=1 colspan=1>1% (v/v)</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>5% (v/v)</td><td rowspan=1 colspan=1>None</td></tr></table>

On initial testing, 1/3 replicates with Sample 5 from Table 13 produced false negative results for Coronavirus NL63 and Coronavirus HKU1 and the result was confirmed upon retesting; however, upon testing with new samples $3 / 3$ replicates gave the expected positive results. 2 On initial testing, 1/3 replicates with Sample 2 from Table 13 produced a false negative result for RSV B; upon re-testing of the same sample, the expected result was obtained. 3 All samples containing FluMist produced positive results for influenza A, influenza A H1pdm09, influenza A H3 and influenza B; this is noted as a Limitation in the device labeling. 4 3/3 replicates in the presence of $1 \%$ (w/v) menthol produced false negative results for Coronavirus OC43; interference by menthol was alleviated at lower concentration $( 0 . 5 \% \mathrm { w } / \mathrm { v } )$ ; this is noted as a Limitation in the device labeling.

4. Assay Reportable Range: Not applicable.

5. Traceability, Stability, Expected Values (Controls, Calibrators, or Methods):

# Controls

Bacteriophage MS2 is added to each sample to be tested with the NxTAG RPP v2 prior to nucleic acid extraction as an internal control to monitor process and reagent integrity. Failure to detect the MS2 control (in conjunction with failure to detect any of the target analytes) is indicative of failure of one or more processes: nucleic acid extraction, reverse transcription PCR amplification and/or detection.

# External Controls

External Positive and Negative Controls are not provided with in the NxTAG RPP v2 assay kit but are recommended for inclusion in every assay run in accordance with good laboratory practice and applicable local, state and federal accrediting organizations. External Controls will be reported as “Pass”, “Fail” or “Invalid” in accordance with the expected values entered by the user in the SYNCT Software.

External Positive and Negative Controls were included in each run of the NxTAG RPP v2 assay in the Clinical Studies described in Section VII C. On each day of testing, the External Positive and Negative Controls were required to produce the expected results for the results obtained with clinical specimens to be considered acceptable.

# Specimen Stability

To evaluate the stability of the NxTAG RPP v2 target analytes in nasopharyngeal swab matrix, separate studies were performed with samples prepared in Universal Transport Medium (UTM, Copan Diagnostics) and MicroTest M4RT (M4RT, Remel). All the analytes targeted by the NxTAG RPP v2 assay were included in the analysis of performance with UTM, whereas only a representative subset was included in evaluating the stability of specimens prepared in M4RT (Table 15). The positive samples for testing were prepared at two concentrations (1.5X [Low] and 5X LoD [Moderate], unless otherwise noted in Table 15) and stored at $2 { - } 8 ~ ^ { \circ } \mathrm { C }$ or $- 7 0 \pm 5 \ ^ { \circ } \mathrm { C }$ prior to testing. The results of the study support the claimed stability of the target analytes in nasopharyngeal swab matrix collected in UTM or M4RT for up to 7 days at $2 { - } 8 ~ ^ { \circ } \mathrm { C }$ or 12 months at $- 7 0 \pm 5 { } ^ { \circ } { \mathrm { C } }$ .

Additional testing also demonstrated the stability of extracted nucleic acid after storage at 2-8 $^ \circ \mathrm { C }$ for up to 4 hours or $- 7 0 \pm 5 { } ^ { \circ } { \mathrm { C } }$ for up to 12 months.

Table 15. Multi-analyte samples used to evaluate specimen stability   

<table><tr><td rowspan=1 colspan=1>Sample</td><td rowspan=1 colspan=1>Analyte</td><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>Source</td><td rowspan=1 colspan=1>Catalogue No.</td></tr><tr><td rowspan=3 colspan=1>11</td><td rowspan=1 colspan=1>Influenza A H1pdm09 (subtype) 2, 3</td><td rowspan=1 colspan=1>A/NY/02/09</td><td rowspan=1 colspan=1>Zeptometrix</td><td rowspan=1 colspan=1>0810109CFN</td></tr><tr><td rowspan=1 colspan=1>Respiratory Syncytial Virus A</td><td rowspan=1 colspan=1>A2</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>VR-1540</td></tr><tr><td rowspan=1 colspan=1>Rhinovirus</td><td rowspan=1 colspan=1>50-525-CV54</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>VR-1195</td></tr><tr><td rowspan=2 colspan=1>21</td><td rowspan=1 colspan=1>Influenza A H3 (subtype) 2</td><td rowspan=1 colspan=1>A/Wisconsin/67/05</td><td rowspan=1 colspan=1>Zeptometrix</td><td rowspan=1 colspan=1>0810252CF</td></tr><tr><td rowspan=1 colspan=1>Respiratory Syncytial Virus B</td><td rowspan=1 colspan=1>CH 19503</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>VR-1377</td></tr><tr><td rowspan=3 colspan=1>31</td><td rowspan=1 colspan=1>Influenza B 2</td><td rowspan=1 colspan=1>B/Florida/02/06</td><td rowspan=1 colspan=1>Zeptometrix</td><td rowspan=1 colspan=1>0810037CF</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza virus 3</td><td rowspan=1 colspan=1>C 243</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>VR-93</td></tr><tr><td rowspan=1 colspan=1>Mycoplasma pneumoniae 4</td><td rowspan=1 colspan=1>M129</td><td rowspan=1 colspan=1>Zeptometrix</td><td rowspan=1 colspan=1>0801579</td></tr><tr><td rowspan=3 colspan=1>41</td><td rowspan=1 colspan=1>SARS-CoV-2</td><td rowspan=1 colspan=1>USA-WA1/2020</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>VR-1986HK</td></tr><tr><td rowspan=1 colspan=1>Human Metapneumovirus B1</td><td rowspan=1 colspan=1>hMPV-3</td><td rowspan=1 colspan=1>Zeptometrix</td><td rowspan=1 colspan=1>0810156CF</td></tr><tr><td rowspan=1 colspan=1>Adenovirus B 2, 3</td><td rowspan=1 colspan=1>Type 14</td><td rowspan=1 colspan=1>Zeptometrix</td><td rowspan=1 colspan=1>0810108CF</td></tr><tr><td rowspan=3 colspan=1>5</td><td rowspan=1 colspan=1>Influenza A H3 (matrix)</td><td rowspan=1 colspan=1>A/Wisconsin/67/05</td><td rowspan=1 colspan=1>Zeptometrix</td><td rowspan=1 colspan=1>0810252CF</td></tr><tr><td rowspan=1 colspan=1>Coronavirus NL63</td><td rowspan=1 colspan=1>--</td><td rowspan=1 colspan=1>Zeptometrix</td><td rowspan=1 colspan=1>0810228CF</td></tr><tr><td rowspan=1 colspan=1>Coronavirus HKU1</td><td rowspan=1 colspan=1>Clinical specimen</td><td rowspan=1 colspan=1>--</td><td rowspan=1 colspan=1>--</td></tr><tr><td rowspan=3 colspan=1>6</td><td rowspan=1 colspan=1>Influenza H1 (subtype) 2</td><td rowspan=1 colspan=1>A/Brisbane/59/07</td><td rowspan=1 colspan=1>Zeptometrix</td><td rowspan=1 colspan=1>0810244CF</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza virus 1</td><td rowspan=1 colspan=1>--</td><td rowspan=1 colspan=1>Zeptometrix</td><td rowspan=1 colspan=1>0810014CF</td></tr><tr><td rowspan=1 colspan=1>Chlamydia pneumoniae</td><td rowspan=1 colspan=1>TW-183</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>VR-2282</td></tr><tr><td rowspan=3 colspan=1>7</td><td rowspan=1 colspan=1>Parainfluenza virus 2</td><td rowspan=1 colspan=1>Greer</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>VR-92</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza virus 4B</td><td rowspan=1 colspan=1>CH 19503</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>VR-1377</td></tr><tr><td rowspan=1 colspan=1>Coronavirus 229E</td><td rowspan=1 colspan=1>--</td><td rowspan=1 colspan=1>Zeptometrix</td><td rowspan=1 colspan=1>0810228CF</td></tr><tr><td rowspan=2 colspan=1>8</td><td rowspan=1 colspan=1>Parainfluenza virus 4A</td><td rowspan=1 colspan=1>--</td><td rowspan=1 colspan=1>Zeptometrix</td><td rowspan=1 colspan=1>0810060CF</td></tr><tr><td rowspan=1 colspan=1>Coronavirus OC43</td><td rowspan=1 colspan=1>Betacoronavirus 1</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>VR-1558</td></tr><tr><td rowspan=1 colspan=1>91</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>--</td><td rowspan=1 colspan=1>--</td><td rowspan=1 colspan=1>--</td></tr></table>

1 Panel member included in evaluation of the stability of specimens in M4RT medium. 2 Target level based on the LoD for the applicable strain/subtype. 3 Formulated at approximately 3X (Low) and 9X LoD (Moderate). 4 Formulated at approximately 2X (Low) and 7X LoD (Moderate).

# Reagent Stability

A Real-time Stability Study to evaluate the stability of the NxTAG RPP v2 reagents under the recommended storage conditions of $2 { - } 8 ~ ^ { \circ } \mathrm { C }$ is currently on-going. The study includes three independent kit lots that were manufactured with unique lots of critical reagents. Each kit lot will be tested at 3-month intervals over the course of the study using a panel of Positive Controls that collectively include target nucleic acids for each assay on the panel, in addition to the MS2 bacteriophage Internal Control $\mathbf { \tilde { n } } = 3$ replicates per assay per kit lot at each test point). To-date, testing has been completed that supports the stability of NxTAG RPP v2 kits for up to 12 months from the date of manufacture when stored under the recommended conditions.

# In-use Reagent Stability

Lyophilized NxTAG RPP v2 assay reagents are supplied in foil-sealed 8-well strips that are packaged in sealed pouches of 96 reactions and stored at $2 { - } 8 ~ ^ { \circ } \mathrm { C }$ . Testing was performed to demonstrate the stability of the prefilled wells after removal from the original packaging, prior to sample addition. Additional testing was also conducted to support resealing of the original reagent pouch and storage of unused foil-sealed wells if less than a full 96 well plate of reactions is consumed. Under all conditions tested, the expected results were obtained, supporting the in-use stability of NxTAG RPP v2 reagents in accordance with the Instructions For Use.

# Limit of Detection

The analytical sensitivity of the NxTAG RPP v2 was evaluated by testing contrived specimens that were prepared by adding quantified, diluted stocks of representative viral and bacterial strains/isolates of each target analyte to pooled negative nasopharyngeal swab matrix in Universal Transport Medium (UTM) that was processed using the bioMérieux EMAG Extraction System. Each analyte was tested individually in a three-fold dilution series to estimate the limit of detection (LoD) which was then confirmed by testing an additional 20 replicates at or near the concentration that yielded 3/3 positive replicates in the preliminary titration. The confirmed LoD for each strain/isolate was defined as the lowest concentration that yielded $\geq 1 9 / 2 0$ positive replicates (i.e., $\ge 9 5 \%$ proportion positive) (Table 16).

Table 16. Analytical sensitivity of the NxTAG RPP $\mathbf { v } 2$ with representative strains/isolates of the target analytes (tested individually using the EMAG Extraction System)   

<table><tr><td colspan="1" rowspan="2">NxTAG RPP v2Analyte</td><td colspan="1" rowspan="2">Strain/Isolate Information</td><td colspan="1" rowspan="2">Source</td><td colspan="1" rowspan="2">CatalogueNo.</td><td colspan="2" rowspan="1">LoD Concentrationper mL</td></tr><tr><td colspan="1" rowspan="1">Copies 1</td><td colspan="1" rowspan="1">Supplier Units</td></tr><tr><td colspan="1" rowspan="3">Adenovirus</td><td colspan="1" rowspan="1">Type 14, Species B, 2006isolate</td><td colspan="1" rowspan="1">ZeptoMetrix</td><td colspan="1" rowspan="1">0810108CF</td><td colspan="1" rowspan="1">1,420</td><td colspan="1" rowspan="1">0.144 TCID50</td></tr><tr><td colspan="1" rowspan="1">Type 1, Species C</td><td colspan="1" rowspan="1">ZeptoMetrix</td><td colspan="1" rowspan="1">0810050CF</td><td colspan="1" rowspan="1">20,100</td><td colspan="1" rowspan="1">92.8 TCID50</td></tr><tr><td colspan="1" rowspan="1">Type 4, Species E</td><td colspan="1" rowspan="1">ZeptoMetrix</td><td colspan="1" rowspan="1">0810070CF</td><td colspan="1" rowspan="1">7,330</td><td colspan="1" rowspan="1">0.191 TCID50</td></tr><tr><td colspan="1" rowspan="1">Coronavirus 229E</td><td colspan="1" rowspan="1">--</td><td colspan="1" rowspan="1">ATCC</td><td colspan="1" rowspan="1">VR-740</td><td colspan="1" rowspan="1">381</td><td colspan="1" rowspan="1">0.122 TCID50</td></tr><tr><td colspan="1" rowspan="1">Coronavirus HKU1</td><td colspan="1" rowspan="1">Genotype A</td><td colspan="1" rowspan="1">Clinical</td><td colspan="1" rowspan="1">68972</td><td colspan="1" rowspan="1">4,180</td><td colspan="1" rowspan="1">--</td></tr><tr><td colspan="1" rowspan="1">Coronavirus NL63</td><td colspan="1" rowspan="1">--</td><td colspan="1" rowspan="1">ZeptoMetrix</td><td colspan="1" rowspan="1">0810228CF</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">0.00645 TCID50</td></tr><tr><td colspan="1" rowspan="1">Coronavirus OC43</td><td colspan="1" rowspan="1">--</td><td colspan="1" rowspan="1">ATCC</td><td colspan="1" rowspan="1">VR-1558</td><td colspan="1" rowspan="1">4,550</td><td colspan="1" rowspan="1">0.0732 TCID50</td></tr><tr><td colspan="1" rowspan="2">HumanMetapneumovirus</td><td colspan="1" rowspan="1">hMPV-16, Type A1, IA10-2003</td><td colspan="1" rowspan="1">ZeptoMetrix</td><td colspan="1" rowspan="1">0810161CF</td><td colspan="1" rowspan="1">71.5</td><td colspan="1" rowspan="1">0.0576 TCID50</td></tr><tr><td colspan="1" rowspan="1">hMPV-3, Type B1,Peru2-2002</td><td colspan="1" rowspan="1">ZeptoMetrix</td><td colspan="1" rowspan="1">0810156CF</td><td colspan="1" rowspan="1">262</td><td colspan="1" rowspan="1">0.0178 TCID50</td></tr><tr><td colspan="1" rowspan="3">Influenza A (matrix)</td><td colspan="1" rowspan="1">A/Brisbane/59/07 (H1)</td><td colspan="1" rowspan="1">ZeptoMetrix</td><td colspan="1" rowspan="1">0810244CF</td><td colspan="1" rowspan="1">119</td><td colspan="1" rowspan="1">0.0283 TCID50</td></tr><tr><td colspan="1" rowspan="1">A/NY/02/09 (H1pdm09)</td><td colspan="1" rowspan="1">ZeptoMetrix</td><td colspan="1" rowspan="1">0810109CFN</td><td colspan="1" rowspan="1">328</td><td colspan="1" rowspan="1">0.0374 TCID50</td></tr><tr><td colspan="1" rowspan="1">A/Wisconsin/67/05 (H3)</td><td colspan="1" rowspan="1">ZeptoMetrix</td><td colspan="1" rowspan="1">0810252CF</td><td colspan="1" rowspan="1">168</td><td colspan="1" rowspan="1">0.645 TCID50</td></tr><tr><td colspan="1" rowspan="1">Influenza A H1(subtype)</td><td colspan="1" rowspan="1">A/Brisbane/59/07</td><td colspan="1" rowspan="1">ZeptoMetrix</td><td colspan="1" rowspan="1">0810244CF</td><td colspan="1" rowspan="1">1,600</td><td colspan="1" rowspan="1">0.382 TCID50</td></tr><tr><td colspan="1" rowspan="1">Influenza A H1pdm09(subtype)</td><td colspan="1" rowspan="1">A/NY/02/09</td><td colspan="1" rowspan="1">ZeptoMetrix</td><td colspan="1" rowspan="1">0810109CFN</td><td colspan="1" rowspan="1">984</td><td colspan="1" rowspan="1">0.112 TCID50</td></tr><tr><td colspan="1" rowspan="1">Influenza A H3(subtype)</td><td colspan="1" rowspan="1">A/Wisconsin/67/05</td><td colspan="1" rowspan="1">ZeptoMetrix</td><td colspan="1" rowspan="1">0810252CF</td><td colspan="1" rowspan="1">56</td><td colspan="1" rowspan="1">0.215 TCID50</td></tr><tr><td colspan="1" rowspan="1">Influenza B</td><td colspan="1" rowspan="1">B/Florida/02/06</td><td colspan="1" rowspan="1">ZeptoMetrix</td><td colspan="1" rowspan="1">0810037CF</td><td colspan="1" rowspan="1">63.3</td><td colspan="1" rowspan="1">0.967 TCID50</td></tr><tr><td colspan="1" rowspan="5">Parainfluenza Virus</td><td colspan="1" rowspan="1">Serotype 1, N/A</td><td colspan="1" rowspan="1">ZeptoMetrix</td><td colspan="1" rowspan="1">0810014CF</td><td colspan="1" rowspan="1">692</td><td colspan="1" rowspan="1">0.764 TCID50</td></tr><tr><td colspan="1" rowspan="1">Serotype 2, Greer</td><td colspan="1" rowspan="1">ATCC</td><td colspan="1" rowspan="1">VR-92</td><td colspan="1" rowspan="1">345</td><td colspan="1" rowspan="1">0.732 TCID50</td></tr><tr><td colspan="1" rowspan="1">Serotype 3, C 243</td><td colspan="1" rowspan="1">ATCC</td><td colspan="1" rowspan="1">VR-93</td><td colspan="1" rowspan="1">1,010</td><td colspan="1" rowspan="1">110 TCID50</td></tr><tr><td colspan="1" rowspan="1">Serotype 4A</td><td colspan="1" rowspan="1">ZeptoMetrix</td><td colspan="1" rowspan="1">0810060CF</td><td colspan="1" rowspan="1">16,900</td><td colspan="1" rowspan="1">0.858 TCID50</td></tr><tr><td colspan="1" rowspan="1">Serotype 4B, CH 19503</td><td colspan="1" rowspan="1">ATCC</td><td colspan="1" rowspan="1">VR-1377</td><td colspan="1" rowspan="1">7,150</td><td colspan="1" rowspan="1">59.9 TCID50</td></tr><tr><td colspan="1" rowspan="1">Respiratory SyncytialVirus A</td><td colspan="1" rowspan="1">A2</td><td colspan="1" rowspan="1">ATCC</td><td colspan="1" rowspan="1">VR-1540</td><td colspan="1" rowspan="1">4,970</td><td colspan="1" rowspan="1">37.7 PFU</td></tr><tr><td colspan="1" rowspan="1">Respiratory SyncytialVirus B</td><td colspan="1" rowspan="1">18537</td><td colspan="1" rowspan="1">ATCC</td><td colspan="1" rowspan="1">VR-1580</td><td colspan="1" rowspan="1">7,205</td><td colspan="1" rowspan="1">0.320 PFU</td></tr><tr><td colspan="1" rowspan="2">Rhinovirus/Enterovirus</td><td colspan="1" rowspan="1">Rhinovirus,50-525-CV54(V-192-001-021)</td><td colspan="1" rowspan="1">ATCC</td><td colspan="1" rowspan="1">VR-1195</td><td colspan="1" rowspan="1">1,536</td><td colspan="1" rowspan="1">68.7 TCID50</td></tr><tr><td colspan="1" rowspan="1">Enterovirus Type 68,2007 Isolate</td><td colspan="1" rowspan="1">ZeptoMetrix</td><td colspan="1" rowspan="1">0810237CF</td><td colspan="1" rowspan="1">3,526</td><td colspan="1" rowspan="1">2.30 TCID50</td></tr><tr><td colspan="1" rowspan="1">SARS-CoV-2</td><td colspan="1" rowspan="1">USA-WA1/2020</td><td colspan="1" rowspan="1">ATCC</td><td colspan="1" rowspan="1">VR-1986HK</td><td colspan="1" rowspan="1">500</td><td colspan="1" rowspan="1">7.68 TCID50</td></tr><tr><td colspan="1" rowspan="1">Copies 1</td><td colspan="1" rowspan="1">SupplierUnits</td></tr><tr><td colspan="1" rowspan="1">Chlamydia pneumoniae</td><td colspan="1" rowspan="1">TW-183</td><td colspan="1" rowspan="1">ATCC</td><td colspan="1" rowspan="1">VR-2282</td><td colspan="1" rowspan="1">238</td><td colspan="1" rowspan="1">37.4 IFU</td></tr><tr><td colspan="1" rowspan="1">Mycoplasmapneumoniae</td><td colspan="1" rowspan="1">M129</td><td colspan="1" rowspan="1">ZeptoMetrix</td><td colspan="1" rowspan="1">0801579</td><td colspan="1" rowspan="1">3,230</td><td colspan="1" rowspan="1">55.6 CCU</td></tr></table>

CCU: Color Changing Units; Copies: copies of nucleic acid target as determined by quantitative PCR; IFU: Inclusion Forming Units; PFU: Plaque Forming Units; $\mathrm { T C I D } 5 0$ : $5 0 \%$ Tissue Culture Infectious Dose

Limit of Detection with Multi-analyte Samples and Simulated Nasopharyngeal Swab Matrix Additional testing was performed to evaluate the analytical sensitivity of the NxTAG RPP v2 assay with samples containing multiple target analytes and to evaluate use of simulated nasopharyngeal swab matrix in analytical studies. Samples containing combinations of two or three target species at concentrations equivalent to 1-2X the confirmed single target LoD in either negative clinical or simulated nasopharyngeal swab matrix were tested in parallel. The analyte combinations evaluated are shown in Table 17. Twenty replicates of each analyte combination were included in the study and all analytes were successfully detected $( \geq$ $9 5 \%$ proportion positive), demonstrating the absence of interference from the presence of multiple analytes at levels close to the LoD of assay and supporting use simulated nasopharyngeal swab matrix in other analytical studies to evaluate device performance.

Table 17. Multi-analyte samples used to evaluate the analytical sensitivity   

<table><tr><td rowspan=2 colspan=1>Sample</td><td rowspan=2 colspan=1>Analyte</td><td rowspan=2 colspan=1>Strain</td><td rowspan=2 colspan=1>Source</td><td rowspan=2 colspan=1>Catalogue No.</td><td rowspan=1 colspan=2>% Detected (n = 20)</td></tr><tr><td rowspan=1 colspan=1>NCM</td><td rowspan=1 colspan=1>NSM</td></tr><tr><td rowspan=3 colspan=1>1</td><td rowspan=1 colspan=1>Influenza A H1pdm09(subtype) 1, 2</td><td rowspan=1 colspan=1>A/NY/02/09</td><td rowspan=1 colspan=1>ZeptoMetrix</td><td rowspan=1 colspan=1>0810109CFN</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>Respiratory SyncytialVirus A</td><td rowspan=1 colspan=1>A2</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>VR-1540</td><td rowspan=1 colspan=1>95</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>Rhinovirus</td><td rowspan=1 colspan=1>50-525-CV54</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>VR-1195</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=2 colspan=1>2</td><td rowspan=1 colspan=1>Influenza A H3(subtype)1</td><td rowspan=1 colspan=1>A/Wisconsin/67/05</td><td rowspan=1 colspan=1>ZeptoMetrix</td><td rowspan=1 colspan=1>0810252CF</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>Respiratory SyncytialVirus B</td><td rowspan=1 colspan=1>18537</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>VR-1580</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=3 colspan=1>3</td><td rowspan=1 colspan=1>Influenza B</td><td rowspan=1 colspan=1>B/Florida/02/06</td><td rowspan=1 colspan=1>ZeptoMetrix</td><td rowspan=1 colspan=1>0810037CF</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>95</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza Virus 3</td><td rowspan=1 colspan=1>C 243</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>VR-93</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>M. pneumoniae 2</td><td rowspan=1 colspan=1>M129</td><td rowspan=1 colspan=1>ZeptoMetrix</td><td rowspan=1 colspan=1>0801579</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=3 colspan=1>4</td><td rowspan=1 colspan=1>SARS-CoV-2</td><td rowspan=1 colspan=1>USA-WA1/2020</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>VR-1986HK</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>HumanMetapneumovirus</td><td rowspan=1 colspan=1>hMPV-3, Type B1,Peru2-2002</td><td rowspan=1 colspan=1>ZeptoMetrix</td><td rowspan=1 colspan=1>0810156CF</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>95</td></tr><tr><td rowspan=1 colspan=1>Adenovirus B 2</td><td rowspan=1 colspan=1>Type 14, 2006 isolate</td><td rowspan=1 colspan=1>ZeptoMetrix</td><td rowspan=1 colspan=1>0810108CF</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=3 colspan=1>5</td><td rowspan=1 colspan=1>Influenza A H3 (matrix)</td><td rowspan=1 colspan=1>A/Wisconsin/67/05</td><td rowspan=1 colspan=1>ZeptoMetrix</td><td rowspan=1 colspan=1>0810252CF</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>Coronavirus NL63</td><td rowspan=1 colspan=1>NL63</td><td rowspan=1 colspan=1>ZeptoMetrix</td><td rowspan=1 colspan=1>0810228CF</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>Coronavirus HKU1</td><td rowspan=1 colspan=1>Genotype B</td><td rowspan=1 colspan=1>Clinical</td><td rowspan=1 colspan=1>HKU1-20210301</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>95</td></tr><tr><td rowspan=3 colspan=1>6</td><td rowspan=1 colspan=1>Influenza A H1(subtype) 1</td><td rowspan=1 colspan=1>A/Brisbane/59/07</td><td rowspan=1 colspan=1>ZeptoMetrix</td><td rowspan=1 colspan=1>0810244CF</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>95</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza Virus 1</td><td rowspan=1 colspan=1>--</td><td rowspan=1 colspan=1>ZeptoMetrix</td><td rowspan=1 colspan=1>0810014CF</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>C. pneumoniae</td><td rowspan=1 colspan=1>TW-183</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>VR-2282</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=3 colspan=1>7</td><td rowspan=1 colspan=1>Parainfluenza Virus 2</td><td rowspan=1 colspan=1>Greer</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>VR-92</td><td rowspan=1 colspan=1>95</td><td rowspan=1 colspan=1>95</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza Virus 4B</td><td rowspan=1 colspan=1>CH 19503</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>VR-1377</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>Coronavirus 229E</td><td rowspan=1 colspan=1>--</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>VR-740</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>95</td></tr><tr><td rowspan=2 colspan=1>8</td><td rowspan=1 colspan=1>Parainfluenza Virus 4A</td><td rowspan=1 colspan=1>--</td><td rowspan=1 colspan=1>ZeptoMetrix</td><td rowspan=1 colspan=1>0810060CF</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>Coronavirus OC43</td><td rowspan=1 colspan=1>--</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>VR-1558</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td></tr></table>

NCM: Nasopharyngeal Clinical Matrix; NSM: Nasopharyngeal Simulated Matrix 1 Target level based on the LoD for the influenza A subtype. 2 Formulated at 1-2X LoD; all other analytes tested at 1X LoD.

Comparison of Alternative Methods of Nucleic Acid Extraction The analytical sensitivity of the NxTAG RPP v2 assay was established using samples that were processed using the bioMérieux EMAG Extraction System and NUCLISENS extraction reagents. Additional testing was performed to evaluate the analytical sensitivity of the assay with the alternative NUCLISENS easyMAG Extraction System, which uses the same extraction reagents as the EMAG instrument. Representative strains/isolates of each of the NxTAG RPP v2 target analytes were diluted in negative nasopharyngeal swab matrix in UTM at levels close to the LoD established using the bioMérieux EMAG Extraction System (Table 16). The combinations of strains/isolates used was the same as that described in Table 15, above. For each of the analytes, $\ge 9 5 \%$ of replicates produced positive results at $\leq$ 2X the established LoD of the NxTAG RPP v2 assay (Table 18), demonstrating that the analytical sensitivity is similar with both nucleic acid extraction systems.

Table 18. Analytical sensitivity of the NxTAG RPP v2 with samples processed using the NUCLISENS easyMAG Extraction System   

<table><tr><td colspan="1" rowspan="1">NxTAG RPP v2 Analyte</td><td colspan="1" rowspan="1">Concentration(copies/mL)</td><td colspan="1" rowspan="1">X LoD 1</td><td colspan="1" rowspan="1">Positive/Tested (%)</td></tr><tr><td colspan="1" rowspan="2">Adenovirus</td><td colspan="1" rowspan="1">2,460</td><td colspan="1" rowspan="1">1.73</td><td colspan="1" rowspan="1">20/20 (100)</td></tr><tr><td colspan="1" rowspan="1">820</td><td colspan="1" rowspan="1">0.58</td><td colspan="1" rowspan="1">13/20 (65)</td></tr><tr><td colspan="1" rowspan="2">Coronavirus 229E</td><td colspan="1" rowspan="1">381</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">20/20 (100)</td></tr><tr><td colspan="1" rowspan="1">127</td><td colspan="1" rowspan="1">0.33</td><td colspan="1" rowspan="1">14/20 (70)</td></tr><tr><td colspan="1" rowspan="2">Coronavirus HKU1</td><td colspan="1" rowspan="1">4,180</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">20/20 (100)</td></tr><tr><td colspan="1" rowspan="1">1,390</td><td colspan="1" rowspan="1">0.33</td><td colspan="1" rowspan="1">15/20 (75)</td></tr><tr><td colspan="1" rowspan="2">Coronavirus NL63</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">20/20 (100)</td></tr><tr><td colspan="1" rowspan="1">33</td><td colspan="1" rowspan="1">0.33</td><td colspan="1" rowspan="1">18/20 (90)</td></tr><tr><td colspan="1" rowspan="2">Coronavirus OC43</td><td colspan="1" rowspan="1">4,550</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">20/20 (100)</td></tr><tr><td colspan="1" rowspan="1">1,520</td><td colspan="1" rowspan="1">0.33</td><td colspan="1" rowspan="1">16/20 (80)</td></tr><tr><td colspan="1" rowspan="2">Human Metapneumovirus</td><td colspan="1" rowspan="1">262</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">20/20 (100)</td></tr><tr><td colspan="1" rowspan="1">87</td><td colspan="1" rowspan="1">0.33</td><td colspan="1" rowspan="1">15/20 (75)</td></tr><tr><td colspan="1" rowspan="2">Influenza A (matrix)</td><td colspan="1" rowspan="1">168</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">20/20 (100)</td></tr><tr><td colspan="1" rowspan="1">56</td><td colspan="1" rowspan="1">0.33</td><td colspan="1" rowspan="1">17/20 (85)</td></tr><tr><td colspan="1" rowspan="2">Influenza A H1pdm09 (subtype)</td><td colspan="1" rowspan="1">1,700</td><td colspan="1" rowspan="1">1.73</td><td colspan="1" rowspan="1">20/20 (100)</td></tr><tr><td colspan="1" rowspan="1">567</td><td colspan="1" rowspan="1">0.58</td><td colspan="1" rowspan="1">17/20 (85)</td></tr><tr><td colspan="1" rowspan="2">Influenza A H3 (subtype)</td><td colspan="1" rowspan="1">56</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">20/20 (100)</td></tr><tr><td colspan="1" rowspan="1">19</td><td colspan="1" rowspan="1">0.33</td><td colspan="1" rowspan="1">16/20 (80)</td></tr><tr><td colspan="1" rowspan="2">Influenza H1 (subtype)</td><td colspan="1" rowspan="1">1,630</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">20/20 (100)</td></tr><tr><td colspan="1" rowspan="1">535</td><td colspan="1" rowspan="1">0.33</td><td colspan="1" rowspan="1">19/20 (95)</td></tr><tr><td colspan="1" rowspan="2">Influenza B</td><td colspan="1" rowspan="1">63</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">20/20 (100)</td></tr><tr><td colspan="1" rowspan="1">21</td><td colspan="1" rowspan="1">0.33</td><td colspan="1" rowspan="1">11/20 (55)</td></tr><tr><td colspan="1" rowspan="3">Parainfluenza Virus 1</td><td colspan="1" rowspan="1">692</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">20/20 (100)</td></tr><tr><td colspan="1" rowspan="1">231</td><td colspan="1" rowspan="1">0.33</td><td colspan="1" rowspan="1">20/20 (100)</td></tr><tr><td colspan="1" rowspan="1">77</td><td colspan="1" rowspan="1">0.11</td><td colspan="1" rowspan="1">8/20 (40)</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza Virus 2</td><td colspan="1" rowspan="1">345</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">19/20 (95)</td></tr><tr><td colspan="1" rowspan="2">Parainfluenza Virus 3</td><td colspan="1" rowspan="1">1,010</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">20/20 (100)</td></tr><tr><td colspan="1" rowspan="1">338</td><td colspan="1" rowspan="1">0.33</td><td colspan="1" rowspan="1">9/20 (45)</td></tr><tr><td colspan="1" rowspan="2">Parainfluenza Virus 4A 2</td><td colspan="1" rowspan="1">16,900</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">20/20 (100)</td></tr><tr><td colspan="1" rowspan="1">5,650</td><td colspan="1" rowspan="1">0.33</td><td colspan="1" rowspan="1">19/20 (95)</td></tr><tr><td colspan="1" rowspan="2">Parainfluenza Virus 4B 2</td><td colspan="1" rowspan="1">7,150</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">20/20 (100)</td></tr><tr><td colspan="1" rowspan="1">2,380</td><td colspan="1" rowspan="1">0.33</td><td colspan="1" rowspan="1">15/20 (75)</td></tr><tr><td colspan="1" rowspan="2">Respiratory Syncytial Virus A</td><td colspan="1" rowspan="1">4,970</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">20/20 (100)</td></tr><tr><td colspan="1" rowspan="1">1,660</td><td colspan="1" rowspan="1">0.33</td><td colspan="1" rowspan="1">16/20 (80)</td></tr><tr><td colspan="1" rowspan="2">Respiratory Syncytial Virus B</td><td colspan="1" rowspan="1">7,210</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">20/20 (100)</td></tr><tr><td colspan="1" rowspan="1">2,400</td><td colspan="1" rowspan="1">0.33</td><td colspan="1" rowspan="1">6/20 (30)</td></tr><tr><td colspan="1" rowspan="3">Rhinovirus 3</td><td colspan="1" rowspan="1">1,540</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">20/20 (100)</td></tr><tr><td colspan="1" rowspan="1">512</td><td colspan="1" rowspan="1">0.33</td><td colspan="1" rowspan="1">20/20 (100)</td></tr><tr><td colspan="1" rowspan="1">171</td><td colspan="1" rowspan="1">0.11</td><td colspan="1" rowspan="1">9/20 (45)</td></tr><tr><td colspan="1" rowspan="2">SARS-CoV-2</td><td colspan="1" rowspan="1">500</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">20/20 (100)</td></tr><tr><td colspan="1" rowspan="1">167</td><td colspan="1" rowspan="1">0.33</td><td colspan="1" rowspan="1">14/20 (70)</td></tr><tr><td colspan="1" rowspan="3">Chlamydia pneumoniae</td><td colspan="1" rowspan="1">238</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">20/20 (100)</td></tr><tr><td colspan="1" rowspan="1">79</td><td colspan="1" rowspan="1">0.33</td><td colspan="1" rowspan="1">20/20 (100)</td></tr><tr><td colspan="1" rowspan="1">26</td><td colspan="1" rowspan="1">0.11</td><td colspan="1" rowspan="1">7/20 (35)</td></tr><tr><td colspan="1" rowspan="2">Mycoplasma pneumoniae</td><td colspan="1" rowspan="1">4,300</td><td colspan="1" rowspan="1">1.33</td><td colspan="1" rowspan="1">20/20 (100)</td></tr><tr><td colspan="1" rowspan="1">1,430</td><td colspan="1" rowspan="1">0.44</td><td colspan="1" rowspan="1">15/20 (75)</td></tr></table>

1 Multiple of the LoD established with samples processed using the bioMérieux EMAG Extraction System (Table 16). 2 Reported by NxTAG RPP $\mathbf { v } 2$ as Parainfluenza virus 4. 3 Reported by NxTAG RPP v2 as Rhinovirus/Enterovirus.

Analytical Sensitivity with the WHO International Standard for SARS-CoV-2 The analytical sensitivity of the NxTAG RPP v2 was also evaluated using the WHO International Standard for SARS-CoV-2 (NIBSC 20/146: Heat inactivated England/02/2020). Serial dilutions of the standard were prepared in pooled nasopharyngeal swab matrix and tested. The LoD with the WHO International Standard, defined as the lowest concentration that produced $\ge 9 5 \%$ positive replicates, was determined to be $7 . 7 0 \times 1 0 ^ { 5 } \mathrm { I U / m L }$ .

# Inclusivity

# In silico Analysis

The inclusivity of the NxTAG RPP v2 SARS-CoV-2 primers and probes was evaluated using 5,859,379 SARS-CoV-2 sequences available in the GISAID EpiCoV database as of November 30, 2023, including sequences from all defined variants of concern or interest. The NxTAG RPP v2 assay is predicted to detect all the SARS-CoV-2 sequences evaluated.

The inclusivity of the NxTAG RPP v2 primers and probes for influenza A and B was evaluated using sequences in the GISAID EpiFlu database from two time periods: January 1, 2000 to December 31, 2008 for influenza A H1 and January 1, 2017 to May 5, 2023 for influenza A H1pdm09, influenza A H3 and influenza B. The NxTAG RPP v2 primers and probes for influenza A, the influenza A subtypes H1, H1pdm09 and H3, and influenza B were predicted to detect approximately $9 7 - 1 0 0 \%$ of the respective target sequences analyzed.

The inclusivity of the NxTAG RPP v2 primers and probes for all other analytes was evaluated using sequences of the target analytes that were deposited in the NCBI GenBank nt database as of April 8, 2023. Based on this analysis, $\ge 9 6 \%$ of sequences of each analyte were predicted to be detected, except for Parainfluenza Virus 2 $( \sim 9 2 \% )$ and untyped strains of Parainfluenza Virus 4 $( \sim 9 4 \% )$ , which exhibited increased potential for mismatches with the corresponding NxTAG RPP v2 primers and probes which could affect analytical and/or clinical sensitivity. The potential for false-negative results with certain strains of Parainfluenza Virus 2 and 4 based on in silico analysis is noted as Limitation in the device labeling.

# Laboratory Testing of Inclusivity

The inclusivity (analytical reactivity) of the NxTAG RPP v2 assay was evaluated using a panel of representative strains of each of the analytes on the panel. Testing of each strain/isolate was performed in at least triplicate by adding a quantified, diluted stock to simulated or natural nasopharyngeal swab matrix at a final concentration equivalent to 3X the claimed LoD for the analyte. If negative results were obtained with one or more

replicates, additional testing was performed at higher concentrations. The results of the study are summarized in Tables 19 – 30 and are acceptable.

Table 19. Strains of Adenovirus evaluated for inclusivity   

<table><tr><td rowspan=1 colspan=1>Species 1</td><td rowspan=1 colspan=1>Type</td><td rowspan=1 colspan=1>Strain/Isolate</td><td rowspan=1 colspan=1>Source</td><td rowspan=1 colspan=1>CatalogueNo.</td><td rowspan=1 colspan=1>ConcentrationDetected(copies/mL)</td></tr><tr><td rowspan=5 colspan=1>B</td><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>2006 isolate</td><td rowspan=1 colspan=1>ZeptoMetrix</td><td rowspan=1 colspan=1>0810108CF</td><td rowspan=1 colspan=1>7,380</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>--</td><td rowspan=1 colspan=1>ZeptoMetrix</td><td rowspan=1 colspan=1>0810062CF</td><td rowspan=1 colspan=1>7,380</td></tr><tr><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>Gomen</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>VR-7</td><td rowspan=1 colspan=1>7,380</td></tr><tr><td rowspan=1 colspan=1>7A</td><td rowspan=1 colspan=1>AV-1645 [128]</td><td rowspan=1 colspan=1>ZeptoMetrix</td><td rowspan=1 colspan=1>0810021CF</td><td rowspan=1 colspan=1>7,380</td></tr><tr><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>--</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>VR-256</td><td rowspan=1 colspan=1>7,380</td></tr><tr><td rowspan=5 colspan=1>C</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>--</td><td rowspan=1 colspan=1>ZeptoMetrix</td><td rowspan=1 colspan=1>0810050CF</td><td rowspan=1 colspan=1>60,200</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>Adenoid 71</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>VR-1</td><td rowspan=1 colspan=1>60,200</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>--</td><td rowspan=1 colspan=1>ZeptoMetrix</td><td rowspan=1 colspan=1>0810110CF</td><td rowspan=1 colspan=1>60,200</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>--</td><td rowspan=1 colspan=1>ZeptoMetrix</td><td rowspan=1 colspan=1>0810020CF</td><td rowspan=1 colspan=1>60,200</td></tr><tr><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>--</td><td rowspan=1 colspan=1>ZeptoMetrix</td><td rowspan=1 colspan=1>0810111CF</td><td rowspan=1 colspan=1>60,200</td></tr><tr><td rowspan=1 colspan=1>E</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>--</td><td rowspan=1 colspan=1>ZeptoMetrix</td><td rowspan=1 colspan=1>0810070CF</td><td rowspan=1 colspan=1>22,000</td></tr></table>

Bold text: strain/isolate used in the LoD Study 1 The NxTAG RPP v2 assay reports a single test result for all Adenovirus species combined (undifferentiated).

Table 20. Strains of the common human coronaviruses evaluated for inclusivity   

<table><tr><td rowspan=1 colspan=1>Species 1</td><td rowspan=1 colspan=1>Genotype</td><td rowspan=1 colspan=1>Source</td><td rowspan=1 colspan=1>Catalogue orLot No.</td><td rowspan=1 colspan=1>ConcentrationDetected (copies/mL)</td></tr><tr><td rowspan=2 colspan=1>229E</td><td rowspan=1 colspan=1>--</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>VR740</td><td rowspan=1 colspan=1>1,140</td></tr><tr><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>Zeptometrix</td><td rowspan=1 colspan=1>0810229CF</td><td rowspan=1 colspan=1>1,140</td></tr><tr><td rowspan=2 colspan=1>NL63</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>Zeptometrix</td><td rowspan=1 colspan=1>0810228CF</td><td rowspan=1 colspan=1>300</td></tr><tr><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>BEI Resources</td><td rowspan=1 colspan=1>NR-470</td><td rowspan=1 colspan=1>300</td></tr><tr><td rowspan=2 colspan=1>OC43</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>VR-1558</td><td rowspan=1 colspan=1>13,600</td></tr><tr><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>Zeptometrix</td><td rowspan=1 colspan=1>0810024CF</td><td rowspan=1 colspan=1>13,600</td></tr><tr><td rowspan=4 colspan=1>HKU1</td><td rowspan=1 colspan=1>B</td><td rowspan=1 colspan=1>Clinical Specimen</td><td rowspan=1 colspan=1>HKU1-20210301</td><td rowspan=1 colspan=1>12,500</td></tr><tr><td rowspan=1 colspan=1>B</td><td rowspan=1 colspan=1>Clinical Specimen</td><td rowspan=1 colspan=1>HKU1-176622</td><td rowspan=1 colspan=1>12,500</td></tr><tr><td rowspan=1 colspan=1>A</td><td rowspan=1 colspan=1>Clinical Specimen</td><td rowspan=1 colspan=1>68972</td><td rowspan=1 colspan=1>12,500</td></tr><tr><td rowspan=1 colspan=1>A</td><td rowspan=1 colspan=1>Clinical Specimen</td><td rowspan=1 colspan=1>69174</td><td rowspan=1 colspan=1>12,500</td></tr></table>

Bold text: strain/isolate used in the LoD Study 1 The NxTAG RPP v2 assay reports separate results for each Coronavirus species.

Table 21. Strains of Human Metapneumovirus evaluated for inclusivity   

<table><tr><td rowspan=1 colspan=1>Clade 1</td><td rowspan=1 colspan=1>Serotype</td><td rowspan=1 colspan=1>Strain/Isolate</td><td rowspan=1 colspan=1>Source</td><td rowspan=1 colspan=1>Catalogue No.</td><td rowspan=1 colspan=1>ConcentrationDetected(copies/mL)</td></tr><tr><td rowspan=2 colspan=1>A1</td><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>IA10-2003</td><td rowspan=1 colspan=1>Zeptometrix</td><td rowspan=1 colspan=1>0810161CF</td><td rowspan=1 colspan=1>215</td></tr><tr><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>IA3-2002</td><td rowspan=1 colspan=1>Zeptometrix</td><td rowspan=1 colspan=1>0810160CF</td><td rowspan=1 colspan=1>215</td></tr><tr><td rowspan=2 colspan=1>A2</td><td rowspan=1 colspan=1>27</td><td rowspan=1 colspan=1>IA27-2004</td><td rowspan=1 colspan=1>Zeptometrix</td><td rowspan=1 colspan=1>0810164CF</td><td rowspan=1 colspan=1>215</td></tr><tr><td rowspan=1 colspan=1>--</td><td rowspan=1 colspan=1>DHI 26583</td><td rowspan=1 colspan=1>SJH 030209</td><td rowspan=1 colspan=1>DHI 26583</td><td rowspan=1 colspan=1>215</td></tr><tr><td rowspan=2 colspan=1>B1</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>Peru2-2002</td><td rowspan=1 colspan=1>Zeptometrix</td><td rowspan=1 colspan=1>0810156CF</td><td rowspan=1 colspan=1>785</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>Peru3-2003</td><td rowspan=1 colspan=1>Zeptometrix</td><td rowspan=1 colspan=1>0810158CF</td><td rowspan=1 colspan=1>785</td></tr><tr><td rowspan=3 colspan=1>B2</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>Peru1-2002</td><td rowspan=1 colspan=1>Zeptometrix</td><td rowspan=1 colspan=1>0810157CF</td><td rowspan=1 colspan=1>785</td></tr><tr><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>Peru6-2003</td><td rowspan=1 colspan=1>Zeptometrix</td><td rowspan=1 colspan=1>0810159CF</td><td rowspan=1 colspan=1>785</td></tr><tr><td rowspan=1 colspan=1>18</td><td rowspan=1 colspan=1>IA18-2003</td><td rowspan=1 colspan=1>Zeptometrix</td><td rowspan=1 colspan=1>0810162CF</td><td rowspan=1 colspan=1>785</td></tr></table>

Bold text: strain/isolate used in the LoD Study 1 The NxTAG RPP $\mathbf { v } 2$ assay reports a single test result for all clades of Human Metapneumovirus combined (undifferentiated).

Table 22. Strains of Influenza A evaluated for inclusivity   

<table><tr><td rowspan=2 colspan=1>Subtype</td><td rowspan=2 colspan=1>Strain/Isolate</td><td rowspan=2 colspan=1>Source</td><td rowspan=2 colspan=1>CatalogueNo.</td><td rowspan=1 colspan=2>Concentration Detected(copies/mL)</td></tr><tr><td rowspan=1 colspan=1>Influenza A</td><td rowspan=1 colspan=1>Subtype</td></tr><tr><td rowspan=5 colspan=1>H1</td><td rowspan=1 colspan=1>A/Brisbane/59/07</td><td rowspan=1 colspan=1>ZeptoMetrix</td><td rowspan=1 colspan=1>0810244CF</td><td rowspan=1 colspan=1>4,810</td><td rowspan=1 colspan=1>4,810</td></tr><tr><td rowspan=1 colspan=1>A/New Caledonia/20/99</td><td rowspan=1 colspan=1>ZeptoMetrix</td><td rowspan=1 colspan=1>0810036CF</td><td rowspan=1 colspan=1>4,810</td><td rowspan=1 colspan=1>4,810</td></tr><tr><td rowspan=1 colspan=1>A/Solomon Island/03/06</td><td rowspan=1 colspan=1>ZeptoMetrix</td><td rowspan=1 colspan=1>0810036CFN</td><td rowspan=1 colspan=1>4,810</td><td rowspan=1 colspan=1>4,810</td></tr><tr><td rowspan=1 colspan=1>A/Taiwan/42/06</td><td rowspan=1 colspan=1>ZeptoMetrix</td><td rowspan=1 colspan=1>0810247CF</td><td rowspan=1 colspan=1>4,810</td><td rowspan=1 colspan=1>4,810</td></tr><tr><td rowspan=1 colspan=1>A/Denver/1/57</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>VR-546</td><td rowspan=1 colspan=1>4,810</td><td rowspan=1 colspan=1>Negative 2</td></tr><tr><td rowspan=16 colspan=1>H1pdm09</td><td rowspan=1 colspan=1>A/NY/02/09</td><td rowspan=1 colspan=1>ZeptoMetrix</td><td rowspan=1 colspan=1>0810109CFN</td><td rowspan=1 colspan=1>5,110</td><td rowspan=1 colspan=1>5,110</td></tr><tr><td rowspan=1 colspan=1>A/Swine/NY/01/2009 (A/NY/01/2009)</td><td rowspan=1 colspan=1>ZeptoMetrix</td><td rowspan=1 colspan=1>0810248CF</td><td rowspan=1 colspan=1>5,110</td><td rowspan=1 colspan=1>5,110</td></tr><tr><td rowspan=1 colspan=1>A/Swine/NY/03/09 (A/NY/03/2009)</td><td rowspan=1 colspan=1>ZeptoMetrix</td><td rowspan=1 colspan=1>0810249CF</td><td rowspan=1 colspan=1>5,110</td><td rowspan=1 colspan=1>5,110</td></tr><tr><td rowspan=1 colspan=1>A/Swine/Canada/6294/09</td><td rowspan=1 colspan=1>ZeptoMetrix</td><td rowspan=1 colspan=1>0810109CFJ</td><td rowspan=1 colspan=1>5,110</td><td rowspan=1 colspan=1>5,110</td></tr><tr><td rowspan=1 colspan=1>A/California/07/09</td><td rowspan=1 colspan=1>ZeptoMetrix</td><td rowspan=1 colspan=1>0810165CF</td><td rowspan=1 colspan=1>5,110</td><td rowspan=1 colspan=1>5,110</td></tr><tr><td rowspan=1 colspan=1>A/Mexico/4108/09</td><td rowspan=1 colspan=1>ZeptoMetrix</td><td rowspan=1 colspan=1>0810166CF</td><td rowspan=1 colspan=1>5,110</td><td rowspan=1 colspan=1>5,110</td></tr><tr><td rowspan=1 colspan=1>A/Michigan/45/15</td><td rowspan=1 colspan=1>ZeptoMetrix</td><td rowspan=1 colspan=1>0810538CF</td><td rowspan=1 colspan=1>5,110</td><td rowspan=1 colspan=1>5,110</td></tr><tr><td rowspan=1 colspan=1>A/Brisbane/02/18</td><td rowspan=1 colspan=1>ZeptoMetrix</td><td rowspan=1 colspan=1>0810585CF</td><td rowspan=1 colspan=1>5,110</td><td rowspan=1 colspan=1>5,110</td></tr><tr><td rowspan=1 colspan=1>A/Virginia/ATCC1/2009</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>VR-1736</td><td rowspan=1 colspan=1>5,110</td><td rowspan=1 colspan=1>5,110</td></tr><tr><td rowspan=1 colspan=1>A/Netherlands/2629/2009</td><td rowspan=1 colspan=1>BEI Resources</td><td rowspan=1 colspan=1>NR-19823</td><td rowspan=1 colspan=1>5,110</td><td rowspan=1 colspan=1>5,110</td></tr><tr><td rowspan=1 colspan=1>A/Houston/3H/2009</td><td rowspan=1 colspan=1>BEI Resources</td><td rowspan=1 colspan=1>NR-20340</td><td rowspan=1 colspan=1>5,110</td><td rowspan=1 colspan=1>5,110</td></tr><tr><td rowspan=1 colspan=1>A/Brownsville/31H/2009</td><td rowspan=1 colspan=1>BEI Resources</td><td rowspan=1 colspan=1>NR-20344</td><td rowspan=1 colspan=1>5,110</td><td rowspan=1 colspan=1>5,110</td></tr><tr><td rowspan=1 colspan=1>A/Dominican Republic/7293/2013</td><td rowspan=1 colspan=1>IRR</td><td rowspan=1 colspan=1>FR-1298</td><td rowspan=1 colspan=1>5,110</td><td rowspan=1 colspan=1>5,110</td></tr><tr><td rowspan=1 colspan=1>A/Massachusetts/15/2013</td><td rowspan=1 colspan=1>IRR</td><td rowspan=1 colspan=1>FR-1319</td><td rowspan=1 colspan=1>5,110</td><td rowspan=1 colspan=1>5,110</td></tr><tr><td rowspan=1 colspan=1>A/Swine/Iowa/15/30</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>VR-333</td><td rowspan=1 colspan=1>4,810</td><td rowspan=1 colspan=1>2.25 x 107</td></tr><tr><td rowspan=1 colspan=1>A/Swine/1976/31</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>VR-99</td><td rowspan=1 colspan=1>4,810</td><td rowspan=1 colspan=1>1.83 x 107</td></tr><tr><td rowspan=24 colspan=1>H3</td><td rowspan=1 colspan=1>A/Wisconsin/67/05</td><td rowspan=1 colspan=1>ZeptoMetrix</td><td rowspan=1 colspan=1>0810252CF</td><td rowspan=1 colspan=1>168</td><td rowspan=1 colspan=1>168</td></tr><tr><td rowspan=1 colspan=1>A/Brisbane/10/07</td><td rowspan=1 colspan=1>ZeptoMetrix</td><td rowspan=1 colspan=1>0810138CF</td><td rowspan=1 colspan=1>504</td><td rowspan=1 colspan=1>168</td></tr><tr><td rowspan=1 colspan=1>A/Texas/50/2012</td><td rowspan=1 colspan=1>ZeptoMetrix</td><td rowspan=1 colspan=1>0810238CF</td><td rowspan=1 colspan=1>504</td><td rowspan=1 colspan=1>504</td></tr><tr><td rowspan=1 colspan=1>A/Perth/16/09</td><td rowspan=1 colspan=1>ZeptoMetrix</td><td rowspan=1 colspan=1>0810251CF</td><td rowspan=1 colspan=1>168</td><td rowspan=1 colspan=1>168</td></tr><tr><td rowspan=1 colspan=1>A/Hong Kong/4801/14</td><td rowspan=1 colspan=1>ZeptoMetrix</td><td rowspan=1 colspan=1>0810526CF</td><td rowspan=1 colspan=1>4,540</td><td rowspan=1 colspan=1>1,510</td></tr><tr><td rowspan=1 colspan=1>A/Singapore/INFIMH-16-0019/16</td><td rowspan=1 colspan=1>ZeptoMetrix</td><td rowspan=1 colspan=1>0810574CF</td><td rowspan=1 colspan=1>1,510</td><td rowspan=1 colspan=1>1,510</td></tr><tr><td rowspan=1 colspan=1>A/Kansas/14/17</td><td rowspan=1 colspan=1>ZeptoMetrix</td><td rowspan=1 colspan=1>0810586CF</td><td rowspan=1 colspan=1>504</td><td rowspan=1 colspan=1>168</td></tr><tr><td rowspan=1 colspan=1>A/Hong Kong/8/68</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>VR-544</td><td rowspan=1 colspan=1>168</td><td rowspan=1 colspan=1>40,800 3</td></tr><tr><td rowspan=1 colspan=1>A/Alice/1973</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>VR-776</td><td rowspan=1 colspan=1>168</td><td rowspan=1 colspan=1>8,040 3</td></tr><tr><td rowspan=1 colspan=1>A/Port Chalmers/1/73</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>VR-810</td><td rowspan=1 colspan=1>1,510</td><td rowspan=1 colspan=1>504 3</td></tr><tr><td rowspan=1 colspan=1>A/Sydney/5/97</td><td rowspan=1 colspan=1>BEI Resources</td><td rowspan=1 colspan=1>NR-12278</td><td rowspan=1 colspan=1>4,540</td><td rowspan=1 colspan=1>168</td></tr><tr><td rowspan=1 colspan=1>A/Santiago/7981/2006</td><td rowspan=1 colspan=1>IRR</td><td rowspan=1 colspan=1>FR-336</td><td rowspan=1 colspan=1>168</td><td rowspan=1 colspan=1>168</td></tr><tr><td rowspan=1 colspan=1>A/Henan/Jinshui/147/07</td><td rowspan=1 colspan=1>IRR</td><td rowspan=1 colspan=1>FR-365</td><td rowspan=1 colspan=1>168</td><td rowspan=1 colspan=1>168</td></tr><tr><td rowspan=1 colspan=1>A/Brisbane/9/2006</td><td rowspan=1 colspan=1>IRR</td><td rowspan=1 colspan=1>FR-366</td><td rowspan=1 colspan=1>168</td><td rowspan=1 colspan=1>168</td></tr><tr><td rowspan=1 colspan=1>A/Nepal/921/2006</td><td rowspan=1 colspan=1>IRR</td><td rowspan=1 colspan=1>FR-367</td><td rowspan=1 colspan=1>168</td><td rowspan=1 colspan=1>168</td></tr><tr><td rowspan=1 colspan=1>A/Florida/2/2006</td><td rowspan=1 colspan=1>IRR</td><td rowspan=1 colspan=1>FR-368</td><td rowspan=1 colspan=1>504</td><td rowspan=1 colspan=1>168</td></tr><tr><td rowspan=1 colspan=1>A/South Australia/55/14</td><td rowspan=1 colspan=1>Zeptometrix</td><td rowspan=1 colspan=1>0810512CF</td><td rowspan=1 colspan=1>168</td><td rowspan=1 colspan=1>168</td></tr><tr><td rowspan=1 colspan=1>A/Stockholm/6/14</td><td rowspan=1 colspan=1>ZeptoMetrix</td><td rowspan=1 colspan=1>0810513CF</td><td rowspan=1 colspan=1>504</td><td rowspan=1 colspan=1>168</td></tr><tr><td rowspan=1 colspan=1>A/Norway/466/14</td><td rowspan=1 colspan=1>ZeptoMetrix</td><td rowspan=1 colspan=1>0810514CF</td><td rowspan=1 colspan=1>168</td><td rowspan=1 colspan=1>168</td></tr><tr><td rowspan=1 colspan=1>A/Hong Kong/2671/19</td><td rowspan=1 colspan=1>ZeptoMetrix</td><td rowspan=1 colspan=1>0810609CF</td><td rowspan=1 colspan=1>504</td><td rowspan=1 colspan=1>504</td></tr><tr><td rowspan=1 colspan=1>A/California/2/2014</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>VR-1938</td><td rowspan=1 colspan=1>168</td><td rowspan=1 colspan=1>168</td></tr><tr><td rowspan=1 colspan=1>A/Switzerland/9715293/2013</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>VR-183</td><td rowspan=1 colspan=1>168</td><td rowspan=1 colspan=1>168</td></tr><tr><td rowspan=1 colspan=1>Clinical Sample</td><td rowspan=1 colspan=1>--</td><td rowspan=1 colspan=1>500-NEG-161</td><td rowspan=1 colspan=1>168</td><td rowspan=1 colspan=1>168</td></tr><tr><td rowspan=1 colspan=1>Clinical Sample</td><td rowspan=1 colspan=1>--</td><td rowspan=1 colspan=1>500-NEG-199</td><td rowspan=1 colspan=1>504</td><td rowspan=1 colspan=1>504</td></tr><tr><td rowspan=4 colspan=1>H5</td><td rowspan=1 colspan=1>A/Anhui/01/2005 (H5N1)</td><td rowspan=1 colspan=1>IRR</td><td rowspan=1 colspan=1>FR-735</td><td rowspan=1 colspan=1>540</td><td rowspan=1 colspan=1>Not Applicable</td></tr><tr><td rowspan=1 colspan=1>A/Egypt/N03072/2010 (H5N1)</td><td rowspan=1 colspan=1>IRR</td><td rowspan=1 colspan=1>FR-1065</td><td rowspan=1 colspan=1>540</td><td rowspan=1 colspan=1>Not Applicable</td></tr><tr><td rowspan=1 colspan=1>A/Hubei/1/2010 (H5N1)</td><td rowspan=1 colspan=1>IRR</td><td rowspan=1 colspan=1>FR-1066</td><td rowspan=1 colspan=1>540</td><td rowspan=1 colspan=1>Not Applicable</td></tr><tr><td rowspan=1 colspan=1>A/pheasant/New Jersey/1355/1998 (H5N2)</td><td rowspan=1 colspan=1>IRR</td><td rowspan=1 colspan=1>FR-771</td><td rowspan=1 colspan=1>540</td><td rowspan=1 colspan=1>Not Applicable</td></tr><tr><td rowspan=2 colspan=1>H7</td><td rowspan=1 colspan=1>A/turkey/Virginia/4529/2002 (H7N2)</td><td rowspan=1 colspan=1>IRR</td><td rowspan=1 colspan=1>FR-772</td><td rowspan=1 colspan=1>540</td><td rowspan=1 colspan=1>Not Applicable</td></tr><tr><td rowspan=1 colspan=1>A/mallard/Netherlands/12/2000 (H7N7)</td><td rowspan=1 colspan=1>IRR</td><td rowspan=1 colspan=1>FR-773</td><td rowspan=1 colspan=1>540</td><td rowspan=1 colspan=1>Not Applicable</td></tr><tr><td rowspan=1 colspan=1>H9</td><td rowspan=1 colspan=1>A/Hong Kong/33982/2009 (H9N2)</td><td rowspan=1 colspan=1>IRR</td><td rowspan=1 colspan=1>FR-1068</td><td rowspan=1 colspan=1>540</td><td rowspan=1 colspan=1>Not Applicable</td></tr></table>

Bold text: strain/isolate used in the LoD Study 1 The NxTAG RPP v2 assay reports separate results for influenza A, influenza A H1, influenza A H1pdm09 and influenza A H3.

2 Reported negative for Influenza A H1 at $5 . 7 7 \times 1 0 ^ { 7 }$ copies/mL, the highest concentration attainable. 3 Sequence analysis identified 1 or more mismatches with the labeled NxTAG primer; however, strains exhibiting these mutations are not currently in circulation and therefore negligible effect on assay performance is predicted.

Table 23. Strains of Influenza B evaluated for inclusivity   

<table><tr><td rowspan=1 colspan=1>Lineage 1</td><td rowspan=1 colspan=1>Strain/Isolate</td><td rowspan=1 colspan=1>Source</td><td rowspan=1 colspan=1>Catalogue No.</td><td rowspan=1 colspan=1>ConcentrationDetected(copies/mL)</td></tr><tr><td rowspan=9 colspan=1>Yamagata</td><td rowspan=1 colspan=1>B/Florida/02/06</td><td rowspan=1 colspan=1>ZeptoMetrix</td><td rowspan=1 colspan=1>0810037CF</td><td rowspan=1 colspan=1>190</td></tr><tr><td rowspan=1 colspan=1>B/Massachusetts/2/2012</td><td rowspan=1 colspan=1>ZeptoMetrix</td><td rowspan=1 colspan=1>0810239CF</td><td rowspan=1 colspan=1>190</td></tr><tr><td rowspan=1 colspan=1>B/Florida/04/2006</td><td rowspan=1 colspan=1>ZeptoMetrix</td><td rowspan=1 colspan=1>0810255CF</td><td rowspan=1 colspan=1>190</td></tr><tr><td rowspan=1 colspan=1>B/Florida/07/04</td><td rowspan=1 colspan=1>ZeptoMetrix</td><td rowspan=1 colspan=1>0810256CF</td><td rowspan=1 colspan=1>190</td></tr><tr><td rowspan=1 colspan=1>B/Panama/45/90</td><td rowspan=1 colspan=1>ZeptoMetrix</td><td rowspan=1 colspan=1>0810259CF</td><td rowspan=1 colspan=1>190</td></tr><tr><td rowspan=1 colspan=1>B/Phuket/3073/13</td><td rowspan=1 colspan=1>ZeptoMetrix</td><td rowspan=1 colspan=1>0810515CF</td><td rowspan=1 colspan=1>190</td></tr><tr><td rowspan=1 colspan=1>B/Bangladesh/5972/07</td><td rowspan=1 colspan=1>IRR</td><td rowspan=1 colspan=1>FR-450</td><td rowspan=1 colspan=1>190</td></tr><tr><td rowspan=1 colspan=1>B/Hubei-Wujiagang/158/2009</td><td rowspan=1 colspan=1>IRR</td><td rowspan=1 colspan=1>FR-469</td><td rowspan=1 colspan=1>190</td></tr><tr><td rowspan=1 colspan=1>B/Wisconsin/1/2010</td><td rowspan=1 colspan=1>ZeptoMetrix</td><td rowspan=1 colspan=1>0810241CF</td><td rowspan=1 colspan=1>190</td></tr><tr><td rowspan=7 colspan=1>Victoria</td><td rowspan=1 colspan=1>B/Brisbane/33/08</td><td rowspan=1 colspan=1>ZeptoMetrix</td><td rowspan=1 colspan=1>0810253CF</td><td rowspan=1 colspan=1>190</td></tr><tr><td rowspan=1 colspan=1>B/Brisbane/60/08</td><td rowspan=1 colspan=1>ZeptoMetrix</td><td rowspan=1 colspan=1>0810254CF</td><td rowspan=1 colspan=1>190</td></tr><tr><td rowspan=1 colspan=1>B/Malaysia/2506/04</td><td rowspan=1 colspan=1>ZeptoMetrix</td><td rowspan=1 colspan=1>0810258CF</td><td rowspan=1 colspan=1>190</td></tr><tr><td rowspan=1 colspan=1>B/Colorado/06/17</td><td rowspan=1 colspan=1>ZeptoMetrix</td><td rowspan=1 colspan=1>0810573CF</td><td rowspan=1 colspan=1>569</td></tr><tr><td rowspan=1 colspan=1>B/Hong Kong/259/2010</td><td rowspan=1 colspan=1>IRR</td><td rowspan=1 colspan=1>FR-663</td><td rowspan=1 colspan=1>190</td></tr><tr><td rowspan=1 colspan=1>B/New Jersey/1/2012</td><td rowspan=1 colspan=1>IRR</td><td rowspan=1 colspan=1>FR-1270</td><td rowspan=1 colspan=1>190</td></tr><tr><td rowspan=1 colspan=1>B/Texas/02/2013</td><td rowspan=1 colspan=1>IRR</td><td rowspan=1 colspan=1>FR-1302</td><td rowspan=1 colspan=1>190</td></tr></table>

Bold text: strain/isolate used in the LoD Study 1 The NxTAG RPP v2 assay reports a single result for both lineages of influenza B (undifferentiated).

Table 24. Strains of Parainfluenza Virus evaluated for inclusivity   

<table><tr><td rowspan=1 colspan=1>Serotype 1</td><td rowspan=1 colspan=1>Strain/Isolate</td><td rowspan=1 colspan=1>Source</td><td rowspan=1 colspan=1>Catalogue No.</td><td rowspan=1 colspan=1>ConcentrationDetected (copies/mL)</td></tr><tr><td rowspan=2 colspan=1>1</td><td rowspan=1 colspan=1>Type 1</td><td rowspan=1 colspan=1>ZeptoMetrix</td><td rowspan=1 colspan=1>0810014CF</td><td rowspan=1 colspan=1>2,080</td></tr><tr><td rowspan=1 colspan=1>C35</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>VR-94</td><td rowspan=1 colspan=1>2,080</td></tr><tr><td rowspan=2 colspan=1>2</td><td rowspan=1 colspan=1>Greer</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>VR-92</td><td rowspan=1 colspan=1>1,030</td></tr><tr><td rowspan=1 colspan=1>Type 2</td><td rowspan=1 colspan=1>ZeptoMetrix</td><td rowspan=1 colspan=1>0810015CF</td><td rowspan=1 colspan=1>1,030</td></tr><tr><td rowspan=4 colspan=1>3</td><td rowspan=1 colspan=1>C 243</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>VR-93</td><td rowspan=1 colspan=1>3,040</td></tr><tr><td rowspan=1 colspan=1>Type 3</td><td rowspan=1 colspan=1>ZeptoMetrix</td><td rowspan=1 colspan=1>0810016CF</td><td rowspan=1 colspan=1>3,040</td></tr><tr><td rowspan=1 colspan=1>ATCC-2011-5</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>VR-1782</td><td rowspan=1 colspan=1>3,040</td></tr><tr><td rowspan=1 colspan=1>NIH47885</td><td rowspan=1 colspan=1>BEI</td><td rowspan=1 colspan=1>NR-3233</td><td rowspan=1 colspan=1>3,040</td></tr><tr><td rowspan=2 colspan=1>4A</td><td rowspan=1 colspan=1>Type 4A</td><td rowspan=1 colspan=1>ZeptoMetrix</td><td rowspan=1 colspan=1>0810060CF</td><td rowspan=1 colspan=1>50,800</td></tr><tr><td rowspan=1 colspan=1>M-25</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>VR-1378</td><td rowspan=1 colspan=1>50,800</td></tr><tr><td rowspan=2 colspan=1>4B</td><td rowspan=1 colspan=1>CH 19503</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>VR-1377</td><td rowspan=1 colspan=1>21,400</td></tr><tr><td rowspan=1 colspan=1>Type 4B</td><td rowspan=1 colspan=1>ZeptoMetrix</td><td rowspan=1 colspan=1>0810060BCF</td><td rowspan=1 colspan=1>21,400</td></tr></table>

Bold text: strain/isolate used in the LoD Study 1 The NxTAG RPP v2 assay reports separate results for RSV serotypes 1-4 (serotypes 4A and 4B are not differentiated).

Table 25. Strains of Respiratory Syncytial Virus evaluated for inclusivity   

<table><tr><td rowspan=1 colspan=1>Subtype 1</td><td rowspan=1 colspan=1>Strain/Isolate</td><td rowspan=1 colspan=1>Source</td><td rowspan=1 colspan=1>Catalogue No.</td><td rowspan=1 colspan=1>ConcentrationDetected (copies/mL)</td></tr><tr><td rowspan=3 colspan=1>A</td><td rowspan=1 colspan=1>A2</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>VR-1540</td><td rowspan=1 colspan=1>14,900</td></tr><tr><td rowspan=1 colspan=1>2006 Isolate</td><td rowspan=1 colspan=1>ZeptoMetrix</td><td rowspan=1 colspan=1>0810040ACF</td><td rowspan=1 colspan=1>14,900</td></tr><tr><td rowspan=1 colspan=1>Long</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>VR-26</td><td rowspan=1 colspan=1>14,900</td></tr><tr><td rowspan=5 colspan=1>B</td><td rowspan=1 colspan=1>18537</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>VR-1580</td><td rowspan=1 colspan=1>21,600</td></tr><tr><td rowspan=1 colspan=1>CH93(18)-18</td><td rowspan=1 colspan=1>ZeptoMetrix</td><td rowspan=1 colspan=1>0810040CF</td><td rowspan=1 colspan=1>21,600</td></tr><tr><td rowspan=1 colspan=1>9320</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>VR-955</td><td rowspan=1 colspan=1>21,600</td></tr><tr><td rowspan=1 colspan=1>B WV/14617/85</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>VR-1400</td><td rowspan=1 colspan=1>21,600</td></tr><tr><td rowspan=1 colspan=1>B1</td><td rowspan=1 colspan=1>BEI Resources</td><td rowspan=1 colspan=1>NR-4052</td><td rowspan=1 colspan=1>21,600</td></tr></table>

Bold text: strain/isolate used in the LoD Study 1 The NxTAG RPP v2 assay reports separate results for RSV subtypes A and B.

Table 26. Strains of Rhinovirus evaluated for inclusivity   

<table><tr><td rowspan=1 colspan=1>Species</td><td rowspan=1 colspan=1>Type</td><td rowspan=1 colspan=1>Strain/Isolate</td><td rowspan=1 colspan=1>Source</td><td rowspan=1 colspan=1>CatalogueNo.</td><td rowspan=1 colspan=1>ConcentrationDetected(copies/mL)</td></tr><tr><td rowspan=10 colspan=1>A</td><td rowspan=1 colspan=1>85</td><td rowspan=1 colspan=1>50-525-CV54 (V-192-001-021)</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>VR-1195</td><td rowspan=1 colspan=1>4,610</td></tr><tr><td rowspan=1 colspan=1>1A</td><td rowspan=1 colspan=1>--</td><td rowspan=1 colspan=1>Zeptometrix</td><td rowspan=1 colspan=1>0810012CFN</td><td rowspan=1 colspan=1>4,610</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>--</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>VR-482</td><td rowspan=1 colspan=1>4,610</td></tr><tr><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>68-CV11</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>VR-1601</td><td rowspan=1 colspan=1>4,610</td></tr><tr><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>--</td><td rowspan=1 colspan=1>Zeptometrix</td><td rowspan=1 colspan=1>0810285CF</td><td rowspan=1 colspan=1>4,610</td></tr><tr><td rowspan=1 colspan=1>34</td><td rowspan=1 colspan=1>137-3</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>VR-1365</td><td rowspan=1 colspan=1>4,610</td></tr><tr><td rowspan=1 colspan=1>39</td><td rowspan=1 colspan=1>209</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>VR-340</td><td rowspan=1 colspan=1>4,610</td></tr><tr><td rowspan=1 colspan=1>54</td><td rowspan=1 colspan=1>FO1-3774</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>VR-1661</td><td rowspan=1 colspan=1>4,610</td></tr><tr><td rowspan=1 colspan=1>57</td><td rowspan=1 colspan=1>Ch47</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>VR-1600</td><td rowspan=1 colspan=1>4,610</td></tr><tr><td rowspan=1 colspan=1>77</td><td rowspan=1 colspan=1>130-63 (V-185-001-021)</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>VR-1187</td><td rowspan=1 colspan=1>4,610</td></tr><tr><td rowspan=6 colspan=1>B</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>FEB</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>VR-483</td><td rowspan=1 colspan=1>4,610</td></tr><tr><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>1059</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>VR-284</td><td rowspan=1 colspan=1>4,610</td></tr><tr><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>33342</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>VR-1663</td><td rowspan=1 colspan=1>4,610</td></tr><tr><td rowspan=1 colspan=1>27</td><td rowspan=1 colspan=1>5870 (5870-CV28) (NIAID V-144-001-021)</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>VR-1137</td><td rowspan=1 colspan=1>4,610</td></tr><tr><td rowspan=1 colspan=1>42</td><td rowspan=1 colspan=1>56822</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>VR-338</td><td rowspan=1 colspan=1>4,610</td></tr><tr><td rowspan=1 colspan=1>83</td><td rowspan=1 colspan=1>Baylor 7 (V-190-001-021)</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>VR-1193</td><td rowspan=1 colspan=1>4,610</td></tr><tr><td rowspan=5 colspan=1>C</td><td rowspan=1 colspan=1>--</td><td rowspan=1 colspan=1>SARS-COV2-NEG017</td><td rowspan=1 colspan=1>Clinical Specimen</td><td rowspan=1 colspan=1>--</td><td rowspan=1 colspan=1>4,610</td></tr><tr><td rowspan=1 colspan=1>--</td><td rowspan=1 colspan=1>NPS/UTM NEG 178</td><td rowspan=1 colspan=1>Clinical Specimen</td><td rowspan=1 colspan=1>--</td><td rowspan=1 colspan=1>4,610</td></tr><tr><td rowspan=1 colspan=1>--</td><td rowspan=1 colspan=1>75466-1</td><td rowspan=1 colspan=1>Clinical Specimen</td><td rowspan=1 colspan=1>--</td><td rowspan=1 colspan=1>4,610</td></tr><tr><td rowspan=1 colspan=1>--</td><td rowspan=1 colspan=1>SARS-COV2-NEG004</td><td rowspan=1 colspan=1>Clinical Specimen</td><td rowspan=1 colspan=1>--</td><td rowspan=1 colspan=1>4,610</td></tr><tr><td rowspan=1 colspan=1>--</td><td rowspan=1 colspan=1>SARS-COV2-NEG012</td><td rowspan=1 colspan=1>Clinical Specimen</td><td rowspan=1 colspan=1>--</td><td rowspan=1 colspan=1>4,610</td></tr></table>

Bold text: strain/isolate used in the LoD Study The NxTAG RPP v2 assay reports a single test result for Rhinovirus/Enterovirus (undifferentiated). Strains of Enterovirus evaluated for inclusivity are shown in Table 27.

Table 27. Strains of Enterovirus evaluated for inclusivity 1   

<table><tr><td rowspan=1 colspan=1>Species</td><td rowspan=1 colspan=1>Serotype</td><td rowspan=1 colspan=1>Strain/Isolate</td><td rowspan=1 colspan=1>Source</td><td rowspan=1 colspan=1>CatalogueNo.</td><td rowspan=1 colspan=1>ConcentrationDetected(copies/mL)</td></tr><tr><td rowspan=2 colspan=1>A</td><td rowspan=1 colspan=1>Enterovirus 71</td><td rowspan=1 colspan=1>Strain H</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>VR-1432</td><td rowspan=1 colspan=1>10,600</td></tr><tr><td rowspan=1 colspan=1>Coxsackievirus A10</td><td rowspan=1 colspan=1>M.K. (Kowalik)</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>VR-168</td><td rowspan=1 colspan=1>10,600</td></tr><tr><td rowspan=5 colspan=1>B</td><td rowspan=1 colspan=1>Coxsackievirus B1</td><td rowspan=1 colspan=1>Conn-5</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>VR-28</td><td rowspan=1 colspan=1>10,600</td></tr><tr><td rowspan=1 colspan=1>Coxsackievirus B4</td><td rowspan=1 colspan=1>J.V.B. (Benschoten)</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>VR-184</td><td rowspan=1 colspan=1>10,600</td></tr><tr><td rowspan=1 colspan=1>Echovirus 11</td><td rowspan=1 colspan=1>Gregory</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>VR-41</td><td rowspan=1 colspan=1>286,000</td></tr><tr><td rowspan=1 colspan=1>Echovirus 13</td><td rowspan=1 colspan=1>Del Carmen</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>VR-43</td><td rowspan=1 colspan=1>10,600</td></tr><tr><td rowspan=1 colspan=1>Enterovirus Type 69</td><td rowspan=1 colspan=1>Toluca-1 [V- 068-001-021]</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>VR-1077</td><td rowspan=1 colspan=1>10,600</td></tr><tr><td rowspan=2 colspan=1>C</td><td rowspan=1 colspan=1>Coxsackievirus A21</td><td rowspan=1 colspan=1>Kuykendall</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>VR-850</td><td rowspan=1 colspan=1>10,600</td></tr><tr><td rowspan=1 colspan=1>Coxsackievirus A24</td><td rowspan=1 colspan=1>DN-19</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>VR-1662</td><td rowspan=1 colspan=1>10,600</td></tr><tr><td rowspan=6 colspan=1>D</td><td rowspan=1 colspan=1>Enterovirus 68</td><td rowspan=1 colspan=1>US/IL/14-18952</td><td rowspan=1 colspan=1>ZeptoMetrix</td><td rowspan=1 colspan=1>0810237CF</td><td rowspan=1 colspan=1>10,600</td></tr><tr><td rowspan=1 colspan=1>Enterovirus 68</td><td rowspan=1 colspan=1>2014 Isolate</td><td rowspan=1 colspan=1>ZeptoMetrix</td><td rowspan=1 colspan=1>0810300CF</td><td rowspan=1 colspan=1>10,600</td></tr><tr><td rowspan=1 colspan=1>Enterovirus 68</td><td rowspan=1 colspan=1>US/MO/14-18947</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>VR-1823</td><td rowspan=1 colspan=1>10,600</td></tr><tr><td rowspan=1 colspan=1>Enterovirus 68</td><td rowspan=1 colspan=1>US/KY/14-18953</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>VR-1825</td><td rowspan=1 colspan=1>10,600</td></tr><tr><td rowspan=1 colspan=1>Enterovirus 68</td><td rowspan=1 colspan=1>Fermon</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>VR-1826</td><td rowspan=1 colspan=1>10,600</td></tr><tr><td rowspan=1 colspan=1>Enterovirus 70</td><td rowspan=1 colspan=1>J670/71</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>VR-836</td><td rowspan=1 colspan=1>10,600</td></tr></table>

Bold text: strain/isolate used in the LoD Study The NxTAG RPP v2 assay reports a single test result for Rhinovirus/Enterovirus (undifferentiated). Strains/isolates of Rhinovirus evaluated for inclusivity are shown in Table 26.

Table 28. Strains of SARS-CoV-2 evaluated for inclusivity   

<table><tr><td rowspan=1 colspan=1>Strain/Isolate</td><td rowspan=1 colspan=1>Source</td><td rowspan=1 colspan=1>Catalogue orLot No.</td><td rowspan=1 colspan=1>Material</td><td rowspan=1 colspan=1>ConcentrationDetected(copies/mL)</td></tr><tr><td rowspan=1 colspan=1>2019-nCoV/USA WA1/2020</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>VR-1986HK</td><td rowspan=1 colspan=1>Heat inactivated</td><td rowspan=1 colspan=1>1,500</td></tr><tr><td rowspan=1 colspan=1>Hong Kong/VM2000106/2020</td><td rowspan=1 colspan=1>ZeptoMetrix</td><td rowspan=1 colspan=1>0810590CFHI</td><td rowspan=1 colspan=1>Heat inactivated</td><td rowspan=1 colspan=1>1,500</td></tr><tr><td rowspan=1 colspan=1>Isolate USA-WA1/2020</td><td rowspan=1 colspan=1>ZeptoMetrix</td><td rowspan=1 colspan=1>0810587CFHI</td><td rowspan=1 colspan=1>Heat inactivated</td><td rowspan=1 colspan=1>1,500</td></tr><tr><td rowspan=1 colspan=1>Strain BetaCoV/Germany/BavPat1/2020</td><td rowspan=1 colspan=1>EVAg</td><td rowspan=1 colspan=1>026N-03889</td><td rowspan=1 colspan=1>RNA</td><td rowspan=1 colspan=1>2,730 1</td></tr><tr><td rowspan=1 colspan=1>Strain 2019- nCoV/Italy-INMI1</td><td rowspan=1 colspan=1>EVAg</td><td rowspan=1 colspan=1>008N-03894</td><td rowspan=1 colspan=1>RNA</td><td rowspan=1 colspan=1>2,730 1</td></tr><tr><td rowspan=1 colspan=1>England/02/2020</td><td rowspan=1 colspan=1>BEI Resources</td><td rowspan=1 colspan=1>NR-52499</td><td rowspan=1 colspan=1>RNA</td><td rowspan=1 colspan=1>2,730 1</td></tr><tr><td rowspan=1 colspan=1>Singapore/2/2020</td><td rowspan=1 colspan=1>BEI Resources</td><td rowspan=1 colspan=1>NR-52501</td><td rowspan=1 colspan=1>RNA</td><td rowspan=1 colspan=1>2,730 1</td></tr><tr><td rowspan=1 colspan=1>USA-IL1/2020</td><td rowspan=1 colspan=1>BEI Resources</td><td rowspan=1 colspan=1>NR-52503</td><td rowspan=1 colspan=1>RNA</td><td rowspan=1 colspan=1>2,730 1</td></tr><tr><td rowspan=1 colspan=1>USA-CA1/2020</td><td rowspan=1 colspan=1>BEI Resources</td><td rowspan=1 colspan=1>NR-52504</td><td rowspan=1 colspan=1>RNA</td><td rowspan=1 colspan=1>2,730 1</td></tr><tr><td rowspan=1 colspan=1>USA-AZ1/2020</td><td rowspan=1 colspan=1>BEI Resources</td><td rowspan=1 colspan=1>NR-52505</td><td rowspan=1 colspan=1>RNA</td><td rowspan=1 colspan=1>2,730 1</td></tr><tr><td rowspan=1 colspan=1>USA-WI1/2020</td><td rowspan=1 colspan=1>BEI Resources</td><td rowspan=1 colspan=1>NR-52506</td><td rowspan=1 colspan=1>RNA</td><td rowspan=1 colspan=1>2,730 1</td></tr><tr><td rowspan=1 colspan=1>USA-CA3/2020</td><td rowspan=1 colspan=1>BEI Resources</td><td rowspan=1 colspan=1>NR-52507</td><td rowspan=1 colspan=1>RNA</td><td rowspan=1 colspan=1>2,730 1</td></tr><tr><td rowspan=1 colspan=1>USA-CA4/2020</td><td rowspan=1 colspan=1>BEI Resources</td><td rowspan=1 colspan=1>NR-52508</td><td rowspan=1 colspan=1>RNA</td><td rowspan=1 colspan=1>2,730 1</td></tr><tr><td rowspan=1 colspan=1>USA-CA2/2020</td><td rowspan=1 colspan=1>BEI Resources</td><td rowspan=1 colspan=1>NR-52509</td><td rowspan=1 colspan=1>RNA</td><td rowspan=1 colspan=1>2,730 1</td></tr><tr><td rowspan=1 colspan=1>Chile/Santiago op4d1/2020</td><td rowspan=1 colspan=1>BEI Resources</td><td rowspan=1 colspan=1>NR-52510</td><td rowspan=1 colspan=1>RNA</td><td rowspan=1 colspan=1>2,730 1</td></tr><tr><td rowspan=1 colspan=1>New York-PV08410/2020</td><td rowspan=1 colspan=1>BEI Resources</td><td rowspan=1 colspan=1>NR-53518</td><td rowspan=1 colspan=1>RNA</td><td rowspan=1 colspan=1>2,730 1</td></tr><tr><td rowspan=1 colspan=1>USA/CA CDC 5574/2020</td><td rowspan=1 colspan=1>BEI Resources</td><td rowspan=1 colspan=1>NR-55245</td><td rowspan=1 colspan=1>Heat inactivated</td><td rowspan=1 colspan=1>1,500</td></tr><tr><td rowspan=1 colspan=1>Alpha (B1.1.7)/UK Variant</td><td rowspan=1 colspan=1>Clinical</td><td rowspan=1 colspan=1>RP-1760</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>1,500</td></tr><tr><td rowspan=1 colspan=1>Epsilon (B1.429)/California Variant</td><td rowspan=1 colspan=1>Clinical</td><td rowspan=1 colspan=1>RP-1877</td><td rowspan=1 colspan=1>--</td><td rowspan=1 colspan=1>1,500</td></tr><tr><td rowspan=1 colspan=1>Epsilon (B1.429)/California Variant</td><td rowspan=1 colspan=1>Clinical</td><td rowspan=1 colspan=1>RP-1878</td><td rowspan=1 colspan=1>--</td><td rowspan=1 colspan=1>1,500</td></tr><tr><td rowspan=1 colspan=1>Delta (B.1.617.2)</td><td rowspan=1 colspan=1>Clinical</td><td rowspan=1 colspan=1>CoV-P-UTM-384</td><td rowspan=1 colspan=1>--</td><td rowspan=1 colspan=1>1,500</td></tr><tr><td rowspan=1 colspan=1>Delta (B.1.617.2)</td><td rowspan=1 colspan=1>Clinical</td><td rowspan=1 colspan=1>CoV-P-UTM-385</td><td rowspan=1 colspan=1>--</td><td rowspan=1 colspan=1>1,500</td></tr><tr><td rowspan=1 colspan=1>Delta (B.1.617.2)</td><td rowspan=1 colspan=1>Clinical</td><td rowspan=1 colspan=1>CoV-P-UTM-385</td><td rowspan=1 colspan=1>--</td><td rowspan=1 colspan=1>1,500</td></tr><tr><td rowspan=1 colspan=1>Delta (B.1.617.2)</td><td rowspan=1 colspan=1>Clinical</td><td rowspan=1 colspan=1>CoV-P-UTM-392</td><td rowspan=1 colspan=1>--</td><td rowspan=1 colspan=1>1,500</td></tr><tr><td rowspan=1 colspan=1>Delta (B.1.617.2)</td><td rowspan=1 colspan=1>Clinical</td><td rowspan=1 colspan=1>CoV-P-UTM-394</td><td rowspan=1 colspan=1>--</td><td rowspan=1 colspan=1>1,500</td></tr><tr><td rowspan=1 colspan=1>Omicron (B.1.1.529 and BA lineages)</td><td rowspan=1 colspan=1>Clinical</td><td rowspan=1 colspan=1>COV-053</td><td rowspan=1 colspan=1>--</td><td rowspan=1 colspan=1>1,500</td></tr><tr><td rowspan=1 colspan=1>Omicron (B.1.1.529 and BA lineages)</td><td rowspan=1 colspan=1>Clinical</td><td rowspan=1 colspan=1>COV-054</td><td rowspan=1 colspan=1>--</td><td rowspan=1 colspan=1>1,500</td></tr><tr><td rowspan=1 colspan=1>Omicron (B.1.1.529 and BA lineages)</td><td rowspan=1 colspan=1>Clinical</td><td rowspan=1 colspan=1>COV-055</td><td rowspan=1 colspan=1>--</td><td rowspan=1 colspan=1>1,500</td></tr><tr><td rowspan=1 colspan=1>Omicron (B.1.1.529 and BA lineages)</td><td rowspan=1 colspan=1>Clinical</td><td rowspan=1 colspan=1>COV-058</td><td rowspan=1 colspan=1>--</td><td rowspan=1 colspan=1>1,500</td></tr><tr><td rowspan=1 colspan=1>Omicron (B.1.1.529 and BA lineages)</td><td rowspan=1 colspan=1>Clinical</td><td rowspan=1 colspan=1>COV-059</td><td rowspan=1 colspan=1>--</td><td rowspan=1 colspan=1>1,500</td></tr></table>

Bold text: strain/isolate used in the LoD Study 1 RNA diluted in extracted negative simulated matrix to a concentration equivalent to 1,500 copies/mL of raw sample.

Table 29. Strains of the Chlamydia pneumoniae evaluated for inclusivity   

<table><tr><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>Source</td><td rowspan=1 colspan=1>Catalogue No.</td><td rowspan=1 colspan=1>Concentration Detected(copies/mL)</td></tr><tr><td rowspan=1 colspan=1>TW-183</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>VR-2282</td><td rowspan=1 colspan=1>713</td></tr><tr><td rowspan=1 colspan=1>TWAR (CDC/CWL-029)</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>VR-1310</td><td rowspan=1 colspan=1>713</td></tr><tr><td rowspan=1 colspan=1>TWAR 2023</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>VR-1356</td><td rowspan=1 colspan=1>713</td></tr><tr><td rowspan=1 colspan=1>AR-39</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>53592</td><td rowspan=1 colspan=1>713</td></tr></table>

Bold text: strain/isolate used in the LoD Study

Table 30. Strains of the Mycoplasma pneumoniae evaluated for inclusivity   

<table><tr><td rowspan=1 colspan=1>Strain/Isolate</td><td rowspan=1 colspan=1>Source</td><td rowspan=1 colspan=1>Catalogue No.</td><td rowspan=1 colspan=1>Concentration Detected(copies/mL)</td></tr><tr><td rowspan=1 colspan=1>M129</td><td rowspan=1 colspan=1>ZeptoMetrix</td><td rowspan=1 colspan=1>0801579</td><td rowspan=1 colspan=1>12,900</td></tr><tr><td rowspan=1 colspan=1>[M52]</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>15293</td><td rowspan=1 colspan=1>12,900</td></tr><tr><td rowspan=1 colspan=1>FH strain of Eaton Agent [NCTC 10119]</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>15531-TTR</td><td rowspan=1 colspan=1>12,900</td></tr><tr><td rowspan=1 colspan=1>Mutant 22</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>39505</td><td rowspan=1 colspan=1>12,900</td></tr></table>

Bold text: strain/isolate used in the LoD Study

The design of the NxTAG RPP v2 assay was based on that of the previously FDA-cleared NxTAG RPP (K152386/K193167), with various modifications aimed to improve inclusivity and specificity, for the addition of SARS-CoV-2 as a target analyte and for discrimination of Influenza A H1pdm09. Because of these changes, a new threshold analysis was performed using a combination of clinical and contrived specimens, as well as negative controls.

The thresholds for the NxTAG RPP v2 assay were established in a three-step optimization process:

a) Selection of an initial range of threshold values for each analyte;   
b) Determination of performance for each analyte using the range of potential threshold values;   
c) Optimization of the thresholds through Receiver Operating Characteristic (ROC) curve analysis.

Discrete data sets were used for preliminary threshold selection, performance evaluation (optimization) and validation. The final optimized thresholds were used to analyze the data obtained in the Analytical and Clinical Studies described in Sections VII A, B and C.

8. Accuracy (Instrument):

# Refer to Section VII C Clinical Studies.

# 9. Carry-Over:

The potential for contamination within and between runs of the NxTAG RPP v2 assay was evaluated by testing high concentrations of two representative analytes (SARS-CoV-2 and $M .$ . pneumoniae at $1 0 ^ { 5 } \mathrm { T C I D } _ { 5 0 }$ and $1 0 ^ { 6 } \mathrm { C C U / m L }$ , respectively) in simulated nasopharyngeal swab matrix, in conjunction with negative samples. Five MAGPIX runs were performed on two instruments using an alternating pattern of high positive and negative samples $[ { \tt n } = 1 2$ ea.) that were processed on two bioMérieux NucliSENS easyMAG extraction systems. On initial testing, 2/12 negative samples in one MAGPIX run gave positive results for SARS-CoV-2 (Table 31). Repreparation, extraction and testing of 12 additional positive and 12 additional negative samples on the same easyMAG and MAGPIX instruments produced the expected results. Overall, in conjunction with the results of the Prospective Clinical Study described in Section VIII $^ { © }$ , the likelihood of false positive results with the NxTAG RPP v2 assay due to cross-contamination was considered acceptable.

Table 31. Results from evaluation of run-to-run and sample-to-sample contamination   

<table><tr><td rowspan=1 colspan=1>MAGPIX Run</td><td rowspan=1 colspan=1>Analyte</td><td rowspan=1 colspan=1>Sample</td><td rowspan=1 colspan=1>Agreement (%)</td></tr><tr><td rowspan=2 colspan=1>1</td><td rowspan=2 colspan=1>SARS-CoV-2</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>12/12 (100)</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>12/12 (100)</td></tr><tr><td rowspan=2 colspan=1>2</td><td rowspan=2 colspan=1>M. pneumoniae</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>12/12 (100)</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>12/12 (100)</td></tr><tr><td rowspan=2 colspan=1>3</td><td rowspan=2 colspan=1>SARS-CoV-2</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>12/12 (100)</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>10/12 (83.3)</td></tr><tr><td rowspan=2 colspan=1>4</td><td rowspan=2 colspan=1>M. pneumoniae</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>12/12 (100)</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>12/12 (100)</td></tr><tr><td rowspan=2 colspan=1>5</td><td rowspan=2 colspan=1>SARS-CoV-2</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>12/12 (100)</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>12/12 (100)</td></tr><tr><td rowspan=4 colspan=1>Total</td><td rowspan=2 colspan=1>SARS-CoV-2</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>36/36 (100)</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>34/36 (94.4)</td></tr><tr><td rowspan=2 colspan=1>M. pneumoniae</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>24/24 (100)</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>24/24 (100)</td></tr></table>

SARS-CoV-2 Positive: $\overline { { 1 0 ^ { 5 } } }$ $\mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ M. pneumoniae Positive: $1 0 ^ { 6 } \mathrm { C C U / m L }$ Repreparation, extraction and testing of 12 additional positive and 12 additional negative samples produced the expected results ( $1 0 0 \%$ agreement for both positive and negative samples).

# B Comparison Studies:

1. Method Comparison with Predicate Device: Not applicable.

2. Matrix Comparison:

Comparison of Natural and Simulated Nasopharyngeal Swab Matrix Please refer to Section VII A (6) regarding use of simulated nasopharyngeal swab matrix in Analytical Studies to evaluate the performance of the NxTAG RPP v2 assay.

Comparison of Alternative Transport Media   
The compatibility of the NxTAG RPP v2 assay with nasopharyngeal swabs collected in alternative transport media was evaluated by testing contrived specimens composed of negative nasopharyngeal swab matrix containing representative NxTAG RPP v2 target analytes at levels above and below the LoD established with Universal Transport Medium (UTM, Copan Diagnostics) (Table 32). The NxTAG RPP V2 assay demonstrated similar analytical sensitivity $( \geq 9 5 \%$ detection at 1 - 3X LoD) with specimens prepared in both transport media evaluated, supporting the use of these media for specimen collection (Table 33).

Table 32. Representative NxTAG RPP v2 analytes used to evaluate alternative transport media   

<table><tr><td rowspan=1 colspan=1>Sample</td><td rowspan=1 colspan=1>Analyte</td><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>Source</td><td rowspan=1 colspan=1>Catalogue No.</td></tr><tr><td rowspan=3 colspan=1>1</td><td rowspan=1 colspan=1>Influenza A H1pdm09 (subtype) 1</td><td rowspan=1 colspan=1>A/NY/02/09</td><td rowspan=1 colspan=1>ZeptoMetrix</td><td rowspan=1 colspan=1>0810109CFN</td></tr><tr><td rowspan=1 colspan=1>Respiratory Syncytial Virus A</td><td rowspan=1 colspan=1>A2</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>VR-1540</td></tr><tr><td rowspan=1 colspan=1>Rhinovirus</td><td rowspan=1 colspan=1>50-525-CV54</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>VR-1195</td></tr><tr><td rowspan=2 colspan=1>2</td><td rowspan=1 colspan=1>Influenza A H3 (subtype) 1</td><td rowspan=1 colspan=1>A/Wisconsin/67/05</td><td rowspan=1 colspan=1>ZeptoMetrix</td><td rowspan=1 colspan=1>0810252CF</td></tr><tr><td rowspan=1 colspan=1>Respiratory Syncytial Virus B</td><td rowspan=1 colspan=1>18537</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>VR-1580</td></tr><tr><td rowspan=3 colspan=1>3</td><td rowspan=1 colspan=1>Influenza B</td><td rowspan=1 colspan=1>B/Florida/02/06</td><td rowspan=1 colspan=1>ZeptoMetrix</td><td rowspan=1 colspan=1>0810037CF</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza Virus 3</td><td rowspan=1 colspan=1>C 243</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>VR-93</td></tr><tr><td rowspan=1 colspan=1>Mycoplasma pneumoniae</td><td rowspan=1 colspan=1>M129</td><td rowspan=1 colspan=1>ZeptoMetrix</td><td rowspan=1 colspan=1>0801579</td></tr><tr><td rowspan=3 colspan=1>4</td><td rowspan=1 colspan=1>SARS-CoV-2</td><td rowspan=1 colspan=1>USA-WA1/2020</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>VR-1986HK</td></tr><tr><td rowspan=1 colspan=1>Human Metapneumovirus</td><td rowspan=1 colspan=1>hMPV-3, Type B1, Peru2-2002</td><td rowspan=1 colspan=1>ZeptoMetrix</td><td rowspan=1 colspan=1>0810156CF</td></tr><tr><td rowspan=1 colspan=1>Adenovirus B</td><td rowspan=1 colspan=1>Type 14, 2006 isolate</td><td rowspan=1 colspan=1>ZeptoMetrix</td><td rowspan=1 colspan=1>0810108CF</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>--</td><td rowspan=1 colspan=1>--</td><td rowspan=1 colspan=1>--</td></tr></table>

Target level based on the LoD for the applicable subtype.

Table 33. Comparison of analytical sensitivity with alternative transport media   

<table><tr><td rowspan=2 colspan=1>NxTAG RPP v2 Analyte</td><td rowspan=2 colspan=1>Concentration(copies/mL)</td><td rowspan=2 colspan=1>X LoD</td><td rowspan=1 colspan=2>Positive/Tested (%)</td></tr><tr><td rowspan=1 colspan=1>UTM</td><td rowspan=1 colspan=1>M4RT</td></tr><tr><td rowspan=4 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>12,300</td><td rowspan=1 colspan=1>8.7</td><td rowspan=1 colspan=1>10/10 (100)</td><td rowspan=1 colspan=1>10/10 (100)</td></tr><tr><td rowspan=1 colspan=1>3,690</td><td rowspan=1 colspan=1>2.6</td><td rowspan=1 colspan=1>30/30 (100)</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>2,460</td><td rowspan=1 colspan=1>1.7</td><td rowspan=1 colspan=1>28/30 (93)</td><td rowspan=1 colspan=1>30/30 (100)</td></tr><tr><td rowspan=1 colspan=1>820</td><td rowspan=1 colspan=1>0.6</td><td rowspan=1 colspan=1>4/10 (40)</td><td rowspan=1 colspan=1>0/10 (0)</td></tr><tr><td rowspan=3 colspan=1>Human Metapneumovirus</td><td rowspan=1 colspan=1>1,310</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>10/10 (100)</td><td rowspan=1 colspan=1>10/10 (100)</td></tr><tr><td rowspan=1 colspan=1>262</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>29/30 (97)</td><td rowspan=1 colspan=1>30/30 (100)</td></tr><tr><td rowspan=1 colspan=1>87</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>5/10 (50)</td><td rowspan=1 colspan=1>7/10 (70)</td></tr><tr><td rowspan=3 colspan=1>Influenza A H1pdm09 (subtype)</td><td rowspan=1 colspan=1>8,520</td><td rowspan=1 colspan=1>8.7</td><td rowspan=1 colspan=1>10/10 (100)</td><td rowspan=1 colspan=1>10/10 (100)</td></tr><tr><td rowspan=1 colspan=1>1,700</td><td rowspan=1 colspan=1>1.7</td><td rowspan=1 colspan=1>30/30 (100)</td><td rowspan=1 colspan=1>30/30 (100)</td></tr><tr><td rowspan=1 colspan=1>568</td><td rowspan=1 colspan=1>0.6</td><td rowspan=1 colspan=1>10/10 (100)</td><td rowspan=1 colspan=1>7/10 (70)</td></tr><tr><td rowspan=3 colspan=1>Influenza A H3 (subtype)</td><td rowspan=1 colspan=1>280</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>10/10 (100)</td><td rowspan=1 colspan=1>10/10 (100)</td></tr><tr><td rowspan=1 colspan=1>56</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>30/30 (100)</td><td rowspan=1 colspan=1>30/30 (100)</td></tr><tr><td rowspan=1 colspan=1>19</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>4/10 (40)</td><td rowspan=1 colspan=1>4/10 (40)</td></tr><tr><td rowspan=3 colspan=1>Influenza B</td><td rowspan=1 colspan=1>316</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>10/10 (100)</td><td rowspan=1 colspan=1>10/10 (100)</td></tr><tr><td rowspan=1 colspan=1>63</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>29/30 (97)</td><td rowspan=1 colspan=1>29/30 (97)</td></tr><tr><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>5/10 (50)</td><td rowspan=1 colspan=1>4/10 (40)</td></tr><tr><td rowspan=4 colspan=1>Parainfluenza Virus 3</td><td rowspan=1 colspan=1>5,070</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>10/10 (100)</td><td rowspan=1 colspan=1>10/10 (100)</td></tr><tr><td rowspan=1 colspan=1>2,030</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>30/30 (100)</td><td rowspan=1 colspan=1>29/30 (97) 1</td></tr><tr><td rowspan=1 colspan=1>1,010</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>26/30 (87)</td><td rowspan=1 colspan=1>22/30 (73)</td></tr><tr><td rowspan=1 colspan=1>338</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>0/10 (0)</td><td rowspan=1 colspan=1>2/10 (20)</td></tr><tr><td rowspan=3 colspan=1>Respiratory Syncytial Virus A</td><td rowspan=1 colspan=1>24,800</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>10/10 (100)</td><td rowspan=1 colspan=1>10/10 (100)</td></tr><tr><td rowspan=1 colspan=1>4,970</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>30/30 (100)</td><td rowspan=1 colspan=1>30/30 (100)</td></tr><tr><td rowspan=1 colspan=1>1,660</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>6/10 (60)</td><td rowspan=1 colspan=1>5/10 (50)</td></tr><tr><td rowspan=3 colspan=1>Respiratory Syncytial Virus B</td><td rowspan=1 colspan=1>36,000</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>10/10 (100)</td><td rowspan=1 colspan=1>10/10 (100)</td></tr><tr><td rowspan=1 colspan=1>7,210</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>30/30 (100)</td><td rowspan=1 colspan=1>30/30 (100)</td></tr><tr><td rowspan=1 colspan=1>2,400</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>6/10 (60)</td><td rowspan=1 colspan=1>3/10 (30)</td></tr><tr><td rowspan=3 colspan=1>Rhinovirus/Enterovirus</td><td rowspan=1 colspan=1>7,680</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>10/10 (100)</td><td rowspan=1 colspan=1>10/10 (100)</td></tr><tr><td rowspan=1 colspan=1>1,540</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>30/30 (100)</td><td rowspan=1 colspan=1>30/30 (100)</td></tr><tr><td rowspan=1 colspan=1>512</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>10/10 (100)</td><td rowspan=1 colspan=1>9/10 (90)</td></tr><tr><td rowspan=3 colspan=1>SARS-CoV-2</td><td rowspan=1 colspan=1>2,500</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>10/10 (100)</td><td rowspan=1 colspan=1>10/10 (100)</td></tr><tr><td rowspan=1 colspan=1>500</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>30/30 (100)</td><td rowspan=1 colspan=1>30/30 (100)</td></tr><tr><td rowspan=1 colspan=1>167</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>10/10 (100)</td><td rowspan=1 colspan=1>7/10 (70)</td></tr><tr><td rowspan=3 colspan=1>Mycoplasma pneumoniae</td><td rowspan=1 colspan=1>21,500</td><td rowspan=1 colspan=1>6.7</td><td rowspan=1 colspan=1>10/10 (100)</td><td rowspan=1 colspan=1>10/10 (100)</td></tr><tr><td rowspan=1 colspan=1>4,300</td><td rowspan=1 colspan=1>1.3</td><td rowspan=1 colspan=1>30/30 (100)</td><td rowspan=1 colspan=1>29/30 (97)</td></tr><tr><td rowspan=1 colspan=1>1,430</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>3/10 (30)</td><td rowspan=1 colspan=1>4/10 (40)</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>--</td><td rowspan=1 colspan=1>--</td><td rowspan=1 colspan=1>0/10 (0)</td><td rowspan=1 colspan=1>0/10 (0)</td></tr></table>

UTM: Universal Transport Medium (Copan Diagnostics); M4RT: MicroTest M4RT (Remel) 1 sample that produced a negative result exhibited an unexpectedly high signal for the MS2 Internal Control and was retested. The retest result was positive and was included in the analysis.

# Comparison of Alternative Swabs for Specimen Collection

The compatibility of the NxTAG RPP v2 assay with alternative types of collection swab was evaluated by testing flocked nylon, polyester and rayon swabs that were seeded with simulated nasopharyngeal swab matrix containing representative target analytes (Table 34). Swabs of each type were prepared and expressed in $1 ~ \mathrm { m L }$ Universal Transport Medium prior to testing with the NxTAG RPP v2 assay. All results for flocked nylon and polyester swabs were as expected. However, diminished recovery/detection from rayon swabs was observed for multiple analytes (Table 35). As a result, the device labeling includes a Limitation indicating that rayon nasopharyngeal swabs are not compatible with the NxTAG RPP v2 assay.

Table 34. Representative NxTAG RPP v2 analytes used to evaluate alternative collection swab for specimen collection   

<table><tr><td rowspan=1 colspan=1>Sample</td><td rowspan=1 colspan=1>Analyte</td><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>Source</td><td rowspan=1 colspan=1>Catalogue No.</td></tr><tr><td rowspan=3 colspan=1>1</td><td rowspan=1 colspan=1>Influenza A H1pdm09dm (subtype) 1</td><td rowspan=1 colspan=1>A/NY/02/09</td><td rowspan=1 colspan=1>ZeptoMetrix</td><td rowspan=1 colspan=1>0810109CFN</td></tr><tr><td rowspan=1 colspan=1>Respiratory Syncytial Virus A</td><td rowspan=1 colspan=1>A2</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>VR-1540</td></tr><tr><td rowspan=1 colspan=1>Rhinovirus</td><td rowspan=1 colspan=1>50-525-CV54</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>VR-1195</td></tr><tr><td rowspan=2 colspan=1>2</td><td rowspan=1 colspan=1>Influenza A H3 (subtype) 1</td><td rowspan=1 colspan=1>A/Wisconsin/67/05</td><td rowspan=1 colspan=1>ZeptoMetrix</td><td rowspan=1 colspan=1>0810252CF</td></tr><tr><td rowspan=1 colspan=1>Respiratory Syncytial Virus B</td><td rowspan=1 colspan=1>18537</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>VR-1580</td></tr><tr><td rowspan=3 colspan=1>3</td><td rowspan=1 colspan=1>Influenza B</td><td rowspan=1 colspan=1>B/Florida/02/06</td><td rowspan=1 colspan=1>ZeptoMetrix</td><td rowspan=1 colspan=1>0810037CF</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza Virus 3</td><td rowspan=1 colspan=1>C 243</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>VR-93</td></tr><tr><td rowspan=1 colspan=1>Mycoplasma pneumoniae</td><td rowspan=1 colspan=1>M129</td><td rowspan=1 colspan=1>ZeptoMetrix</td><td rowspan=1 colspan=1>0801579</td></tr><tr><td rowspan=3 colspan=1>4</td><td rowspan=1 colspan=1>SARS-CoV-2</td><td rowspan=1 colspan=1>USA-WA1/2020</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>VR-1986HK</td></tr><tr><td rowspan=1 colspan=1>Human Metapneumovirus</td><td rowspan=1 colspan=1>hMPV-3, Type B1, Peru2-2002</td><td rowspan=1 colspan=1>ZeptoMetrix</td><td rowspan=1 colspan=1>0810156CF</td></tr><tr><td rowspan=1 colspan=1>Adenovirus B</td><td rowspan=1 colspan=1>Type 14, 2006 isolate</td><td rowspan=1 colspan=1>ZeptoMetrix</td><td rowspan=1 colspan=1>0810108CF</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>--</td><td rowspan=1 colspan=1>--</td><td rowspan=1 colspan=1>--</td></tr></table>

1 Target level based on the LoD for the applicable subtype.

Table 35. Results obtained from evaluation of alternative collection swabs   

<table><tr><td rowspan=2 colspan=1>Sample</td><td rowspan=2 colspan=1>NxTAG RPP v2 Analyte</td><td rowspan=1 colspan=3>Agreement (%)</td></tr><tr><td rowspan=1 colspan=1>Nylon Flocked</td><td rowspan=1 colspan=1>Polyester</td><td rowspan=1 colspan=1>Rayon</td></tr><tr><td rowspan=3 colspan=1>1</td><td rowspan=1 colspan=1>Influenza A H1pdm09 (subtype)</td><td rowspan=1 colspan=1>3/3 (100)</td><td rowspan=1 colspan=1>3/3 (100)</td><td rowspan=1 colspan=1>3/3 (100)</td></tr><tr><td rowspan=1 colspan=1>Respiratory Syncytial Virus A</td><td rowspan=1 colspan=1>3/3 (100)</td><td rowspan=1 colspan=1>3/3 (100)</td><td rowspan=1 colspan=1>3/3 (100)</td></tr><tr><td rowspan=1 colspan=1>Rhinovirus/Enterovirus</td><td rowspan=1 colspan=1>3/3 (100)</td><td rowspan=1 colspan=1>3/3 (100)</td><td rowspan=1 colspan=1>3/3 (100)</td></tr><tr><td rowspan=2 colspan=1>2</td><td rowspan=1 colspan=1>Influenza A H3 (subtype)</td><td rowspan=1 colspan=1>3/3 (100)</td><td rowspan=1 colspan=1>3/3 (100)</td><td rowspan=1 colspan=1>3/3 (100)</td></tr><tr><td rowspan=1 colspan=1>Respiratory Syncytial Virus B</td><td rowspan=1 colspan=1>3/3 (100)</td><td rowspan=1 colspan=1>3/3 (100)</td><td rowspan=1 colspan=1>3/3 (100)</td></tr><tr><td rowspan=3 colspan=1>3</td><td rowspan=1 colspan=1>Influenza B</td><td rowspan=1 colspan=1>3/3 (100)</td><td rowspan=1 colspan=1>3/3 (100)</td><td rowspan=1 colspan=1>1/3 (33) 1</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza Virus 3</td><td rowspan=1 colspan=1>3/3 (100)</td><td rowspan=1 colspan=1>3/3 (100)</td><td rowspan=1 colspan=1>1/3 (33) 1</td></tr><tr><td rowspan=1 colspan=1>Mycoplasma pneumoniae</td><td rowspan=1 colspan=1>3/3 (100)</td><td rowspan=1 colspan=1>3/3 (100)</td><td rowspan=1 colspan=1>1/3 (33) 1</td></tr><tr><td rowspan=3 colspan=1>4</td><td rowspan=1 colspan=1>SARS-CoV-2</td><td rowspan=1 colspan=1>3/3 (100)</td><td rowspan=1 colspan=1>3/3 (100)</td><td rowspan=1 colspan=1>3/3 (100)</td></tr><tr><td rowspan=1 colspan=1>Human Metapneumovirus</td><td rowspan=1 colspan=1>3/3 (100)</td><td rowspan=1 colspan=1>3/3 (100)</td><td rowspan=1 colspan=1>3/3 (100)</td></tr><tr><td rowspan=1 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>3/3 (100)</td><td rowspan=1 colspan=1>3/3 (100)</td><td rowspan=1 colspan=1>2/3 (67) 2</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>3/3 (100)</td><td rowspan=1 colspan=1>3/3 (100)</td><td rowspan=1 colspan=1>3/3 (100)</td></tr></table>

1 Negative were obtained results for all 3 analytes (Influenza B, Parainfluenza Virus 3 and $M$ . pneumoniae) from 2 rayon swabs. 2 A negative result for Adenovirus was obtained from a single rayon swab.

# C Clinical Studies:

1. Clinical Sensitivity:

Study Design

The performance of the NxTAG RPP v2 was evaluated in a multi-site Clinical Study using a combination of nasopharyngeal swab specimens that were either:

a) prospectively collected and tested fresh (Category I specimens), or after freezing at ≤ - $. 7 0 ~ ^ { \circ } \mathrm { C }$ and thawing if they could not be processed within 7 days (Category II specimens), or   
b) preselected, archived specimens (Category III specimens), or   
c) contrived simulated specimens (Category IV specimens).

For inclusion in the study, Category I, II or III specimens were obtained from patients exhibiting signs and symptoms of respiratory tract infection. Samples for testing with the NxTAG RPP v2 assay were processed using either the bioMérieux easyMAG or EMAG instruments.

Prospectively Collected Specimens (Category I/II)

A total of 1,844 nasopharyngeal swab specimens from five geographically diverse sites in the United States were initially enrolled in the Prospective Clinical Study between October 2022 and April 2023. The performance of the NxTAG RPP v2 was compared to that of an analyte-specific molecular assay for SARS-CoV-2 that was cleared by FDA under 21 CFR 866.3981 (Product Code QQX) and an FDA-cleared molecular respiratory panel for all other analytes. Specimens that were reported positive for influenza A H1 by the comparator method were further characterized by PCR/bidirectional sequencing to distinguish the pandemic strain A H1pdm09 from other influenza A H1 strains. The prospectively collected specimens were tested using the NxTAG RPP v2 assay at three study sites.

Nineteen specimens were excluded from the analysis of performance as follows: duplicate specimen, not from a unique patient (11), not collected from the intended use population with signs and symptoms of respiratory tract infection (5), improper labeling (2) or NxTAG RPP v2 operator error (1). Nineteen of the remaining 1,825 specimens $( 1 . 0 \% )$ produced invalid NxTAG results on initial testing. Sixteen were retested, of which 14 produced valid positive or negative results and 2 were again reported as “invalid”. In total, 5 specimens $( 0 . 3 \% )$ were excluded from the analysis of performance due to invalid NxTAG RPP v2 test results. Additional specimens were also excluded from analysis of performance for specific analytes due to failure to perform the applicable comparator test method(s) or generation of invalid comparator test results.

The demographic characteristics of the 1,820 patients from whom specimens with valid NxTAG RPP v2 results were obtained are provided in Table 36.

Table 36. Characteristics of patients and specimens included in the Prospective Clinical Study   

<table><tr><td rowspan=1 colspan=2>Category</td><td rowspan=1 colspan=1>Number</td><td rowspan=1 colspan=1>Percent</td></tr><tr><td rowspan=3 colspan=1>Gender</td><td rowspan=1 colspan=1>Male</td><td rowspan=1 colspan=1>817</td><td rowspan=1 colspan=1>44.9</td></tr><tr><td rowspan=1 colspan=1>Female</td><td rowspan=1 colspan=1>1,003</td><td rowspan=1 colspan=1>55.1</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>1,820</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=7 colspan=1>Age</td><td rowspan=1 colspan=1>0-1</td><td rowspan=1 colspan=1>337</td><td rowspan=1 colspan=1>18.5</td></tr><tr><td rowspan=1 colspan=1>&gt;1-5</td><td rowspan=1 colspan=1>253</td><td rowspan=1 colspan=1>13.9</td></tr><tr><td rowspan=1 colspan=1>&gt;5-21</td><td rowspan=1 colspan=1>331</td><td rowspan=1 colspan=1>18.2</td></tr><tr><td rowspan=1 colspan=1>&gt; 21- 65</td><td rowspan=1 colspan=1>645</td><td rowspan=1 colspan=1>35.4</td></tr><tr><td rowspan=1 colspan=1>&gt; 65</td><td rowspan=1 colspan=1>242</td><td rowspan=1 colspan=1>13.3</td></tr><tr><td rowspan=1 colspan=1>Not Known</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>0.7</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>1,820</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=5 colspan=1>Patient Status</td><td rowspan=1 colspan=1>Outpatient</td><td rowspan=1 colspan=1>356</td><td rowspan=1 colspan=1>19.6</td></tr><tr><td rowspan=1 colspan=1>Hospitalized</td><td rowspan=1 colspan=1>304</td><td rowspan=1 colspan=1>16.7</td></tr><tr><td rowspan=1 colspan=1>Emergency Room</td><td rowspan=1 colspan=1>1,146</td><td rowspan=1 colspan=1>63.0</td></tr><tr><td rowspan=1 colspan=1>Not Known</td><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>0.8</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>1,820</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=5 colspan=1>Location</td><td rowspan=1 colspan=1>California</td><td rowspan=1 colspan=1>447</td><td rowspan=1 colspan=1>24.6</td></tr><tr><td rowspan=1 colspan=1>Florida</td><td rowspan=1 colspan=1>826</td><td rowspan=1 colspan=1>45.4</td></tr><tr><td rowspan=1 colspan=1>Texas</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>2.2</td></tr><tr><td rowspan=1 colspan=1>Wisconsin</td><td rowspan=1 colspan=1>507</td><td rowspan=1 colspan=1>27.9</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>1,820</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=3 colspan=1>Transport Medium</td><td rowspan=1 colspan=1>UTM</td><td rowspan=1 colspan=1>566</td><td rowspan=1 colspan=1>31.1</td></tr><tr><td rowspan=1 colspan=1>UVT</td><td rowspan=1 colspan=1>1,254</td><td rowspan=1 colspan=1>68.9</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>1,820</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=3 colspan=1>Specimen Category</td><td rowspan=1 colspan=1>Category I - Fresh</td><td rowspan=1 colspan=1>776</td><td rowspan=1 colspan=1>42.6</td></tr><tr><td rowspan=1 colspan=1>Category II - Frozen</td><td rowspan=1 colspan=1>1,044</td><td rowspan=1 colspan=1>57.4</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>1,820</td><td rowspan=1 colspan=1>100</td></tr></table>

UTM: Universal Transport Medium (Copan Diagnostics); UVT: Universal Viral Transport (BD)

# Preselected, Archived Specimens (Category III)

For analytes that exhibited low prevalence in the Prospective Clinical Study, 320 preselected, archived specimens that were initially reported as positive based on standard of care test results were obtained from six sites in the United States. The microbial content of each specimen was confirmed by PCR/bidirectional sequencing prior to enrollment in the study. Eleven specimens were excluded from the analysis of performance because confirmatory testing could not be completed. To avoid bias, testing of the remaining 309 archived specimens with the NxTAG RPP v2 assay was performed in a randomized, blinded manner at four study sites without further preselection based on their microbial content. Three specimens $( 0 . 9 \% )$ produced invalid NxTAG RPP v2 results on initial testing, of which 2 gave valid positive/negative results upon repeat, for a final invalid rate of $0 . 3 \%$ (1/309) and a final total of 308 preselected specimens for inclusion in the analysis of performance.

# Contrived Specimens (Category IV)

Positive clinical specimens could not be obtained in sufficient numbers to meet study requirements for some analytes for which additional testing was performed using contrived specimens. These were prepared by spiking known analyte-negative individual clinical specimens in Universal Transport Medium (UTM; Copan Diagnostics) using previously characterized bacterial or viral stocks of representative strains at 2X, 10X or 100X the established LoD. Testing of the contrived specimens with the NxTAG RPP v2 assay was performed at two study sites in a blinded manner. Results were compared to the known analyte content of each specimen. On initial testing 1/199 contrived specimens $( 0 . 5 \% )$ produced an invalid NxTAG RPP v2 test result. A valid result was obtained for this specimen upon retest.

# Clinical Performance

A summary of the results from testing prospectively collected and archived, preselected nasopharyngeal swabs is shown in Table 37.

Valid NxTAG RPP v2 test results were obtained from a total of 1,820 prospectively collected nasopharyngeal swab specimens. Positive Percent Agreement (PPA) ranged from $9 0 . 0 { - } 1 0 0 \%$ for prospectively collected specimens depending on the analyte, except for Parainfluenza Virus 4 for which the number of specimens tested was small $( \mathtt { n } = 1 5 )$ ). For archived, preselected specimens, PPA ranged from $9 2 . 3 \substack { - 1 0 0 \% }$ , depending on the analyte.

Negative Percent Agreement (NPA) ranged from $9 9 . 7 \mathrm { - } 1 0 0 \%$ for prospectively collected specimens and from $9 9 . 3 \substack { - 1 0 0 \% }$ for archived specimens.

Table 37. Clinical performance of the NxTAG RPP v2 with prospectively collected and archived (preselected) specimens   

<table><tr><td colspan="6" rowspan="1">Viruses</td></tr><tr><td colspan="1" rowspan="2">NxTAG RPP v2 Analyte</td><td colspan="1" rowspan="2">Study</td><td colspan="2" rowspan="1">Positive Agreement</td><td colspan="2" rowspan="1">Negative Agreement</td></tr><tr><td colspan="1" rowspan="1">Percent(95% CI)</td><td colspan="1" rowspan="1">TP/(TP+FN)</td><td colspan="1" rowspan="1">Percent(95% CI)</td><td colspan="1" rowspan="1">TN/(TN+FP)</td></tr><tr><td colspan="1" rowspan="4">Adenovirus</td><td colspan="1" rowspan="1">Prospective(Fresh)</td><td colspan="1" rowspan="1">100(91.0-100)</td><td colspan="1" rowspan="1">39/39</td><td colspan="1" rowspan="1">99.6(98.8-99.9)</td><td colspan="1" rowspan="1">733/736</td></tr><tr><td colspan="1" rowspan="1">Prospective(Frozen)</td><td colspan="1" rowspan="1">93.2(83.8-97.3)</td><td colspan="1" rowspan="1">55/59</td><td colspan="1" rowspan="1">99.6(99.0-99.8)</td><td colspan="1" rowspan="1">976/980</td></tr><tr><td colspan="1" rowspan="1">ProspectiveTotal</td><td colspan="1" rowspan="1">95.9(90.0-98.4)</td><td colspan="1" rowspan="1">94/98</td><td colspan="1" rowspan="1">99.6(99.2-99.8)</td><td colspan="1" rowspan="1">1709/1716</td></tr><tr><td colspan="1" rowspan="1">Archived</td><td colspan="1" rowspan="1">100(20.7-100)</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100(98.8-100)</td><td colspan="1" rowspan="1">307/307</td></tr><tr><td colspan="1" rowspan="4">Coronavirus 229E</td><td colspan="1" rowspan="1">Prospective(Fresh)</td><td colspan="1" rowspan="1">100(34.2-100)</td><td colspan="1" rowspan="1">2/2</td><td colspan="1" rowspan="1">100(99.5-100)</td><td colspan="1" rowspan="1">773/773</td></tr><tr><td colspan="1" rowspan="1">Prospective(Frozen)</td><td colspan="1" rowspan="1">100(56.6-100)</td><td colspan="1" rowspan="1">5/5</td><td colspan="1" rowspan="1">99.9(99.5-100)</td><td colspan="1" rowspan="1">1033/1034</td></tr><tr><td colspan="1" rowspan="1">ProspectiveTotal</td><td colspan="1" rowspan="1">100(64.6-100)</td><td colspan="1" rowspan="1">7/7</td><td colspan="1" rowspan="1">99.9(99.7-100)</td><td colspan="1" rowspan="1">1806/1807</td></tr><tr><td colspan="1" rowspan="1">Archived</td><td colspan="1" rowspan="1">100(74.1-100)</td><td colspan="1" rowspan="1">11/11</td><td colspan="1" rowspan="1">100(98.7-100)</td><td colspan="1" rowspan="1">297/297</td></tr><tr><td colspan="1" rowspan="4">Coronavirus HKU1</td><td colspan="1" rowspan="1">Prospective(Fresh)</td><td colspan="1" rowspan="1">100(67.6-100)</td><td colspan="1" rowspan="1">8/8</td><td colspan="1" rowspan="1">100(99.5-100)</td><td colspan="1" rowspan="1">767/767</td></tr><tr><td colspan="1" rowspan="1">Prospective(Frozen)</td><td colspan="1" rowspan="1">92.3(66.7-98.6)</td><td colspan="1" rowspan="1">12/13</td><td colspan="1" rowspan="1">100(99.6-100)</td><td colspan="1" rowspan="1">1026/1026</td></tr><tr><td colspan="1" rowspan="1">ProspectiveTotal</td><td colspan="1" rowspan="1">95.2(77.3-99.2)</td><td colspan="1" rowspan="1">20/21</td><td colspan="1" rowspan="1">100(99.8-100)</td><td colspan="1" rowspan="1">1793/1793</td></tr><tr><td colspan="1" rowspan="1">Archived</td><td colspan="1" rowspan="1">93.8(79.9-98.3)</td><td colspan="1" rowspan="1">30/32</td><td colspan="1" rowspan="1">100(98.6-100)</td><td colspan="1" rowspan="1">276/276</td></tr><tr><td colspan="1" rowspan="1">Coronavirus NL63</td><td colspan="1" rowspan="1">Prospective(Fresh)</td><td colspan="1" rowspan="1">92.6(76.6-97.9)</td><td colspan="1" rowspan="1">25/27</td><td colspan="1" rowspan="1">100(99.5-100)</td><td colspan="1" rowspan="1">748/748</td></tr><tr><td colspan="1" rowspan="3"></td><td colspan="1" rowspan="1">Prospective(Frozen)</td><td colspan="1" rowspan="1">92.0(75.0-97.8)</td><td colspan="1" rowspan="1">23/25</td><td colspan="1" rowspan="1">100(99.6-100)</td><td colspan="1" rowspan="1">1014/1014</td></tr><tr><td colspan="1" rowspan="1">ProspectiveTotal</td><td colspan="1" rowspan="1">92.3(81.8-97.0)</td><td colspan="1" rowspan="1">48/52</td><td colspan="1" rowspan="1">100(99.8-100)</td><td colspan="1" rowspan="1">1762/1762</td></tr><tr><td colspan="1" rowspan="1">Archived</td><td colspan="1" rowspan="1">NotApplicable</td><td colspan="1" rowspan="1">NotApplicable</td><td colspan="1" rowspan="1">100(98.8-100)</td><td colspan="1" rowspan="1">308/308</td></tr><tr><td colspan="1" rowspan="4">Coronavirus OC43</td><td colspan="1" rowspan="1">Prospective (FFresh)</td><td colspan="1" rowspan="1">100(72.3-100)</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">100(99.5-100)</td><td colspan="1" rowspan="1">765/765</td></tr><tr><td colspan="1" rowspan="1">Prospective(Frozen)</td><td colspan="1" rowspan="1">100(88.3-100)</td><td colspan="1" rowspan="1">29/29</td><td colspan="1" rowspan="1">100(99.6-100)</td><td colspan="1" rowspan="1">1010/1010</td></tr><tr><td colspan="1" rowspan="1">Prospective Totall</td><td colspan="1" rowspan="1">100(91.0-100)</td><td colspan="1" rowspan="1">39/39</td><td colspan="1" rowspan="1">100(99.8-100)</td><td colspan="1" rowspan="1">1775/1775</td></tr><tr><td colspan="1" rowspan="1">Archived</td><td colspan="1" rowspan="1">NotApplicable</td><td colspan="1" rowspan="1">NotApplicable</td><td colspan="1" rowspan="1">100(98.8-100)</td><td colspan="1" rowspan="1">308/308</td></tr><tr><td colspan="1" rowspan="4">Human Metapneumovirus</td><td colspan="1" rowspan="1">Prospective(Fresh)</td><td colspan="1" rowspan="1">100(95.8-100)</td><td colspan="1" rowspan="1">87/87</td><td colspan="1" rowspan="1">98.8(97.7-99.4)</td><td colspan="1" rowspan="1">680/688</td></tr><tr><td colspan="1" rowspan="1">Prospective(Frozen)</td><td colspan="1" rowspan="1">100(94.8-100)</td><td colspan="1" rowspan="1">70/70</td><td colspan="1" rowspan="1">99.4(98.7-99.7)</td><td colspan="1" rowspan="1">963/969</td></tr><tr><td colspan="1" rowspan="1">ProspectiveTotal</td><td colspan="1" rowspan="1">100(97.6-100)</td><td colspan="1" rowspan="1">157/157</td><td colspan="1" rowspan="1">99.2(98.6-99.5)</td><td colspan="1" rowspan="1">1643/1677</td></tr><tr><td colspan="1" rowspan="1">Archived</td><td colspan="1" rowspan="1">NotApplicable</td><td colspan="1" rowspan="1">NotApplicable</td><td colspan="1" rowspan="1">100(98.8-100)</td><td colspan="1" rowspan="1">308/308</td></tr><tr><td colspan="1" rowspan="4">Influenza A (matrix)</td><td colspan="1" rowspan="1">Prospective(Fresh)</td><td colspan="1" rowspan="1">100(83.9-100)</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">99.7(99.0-99.9)</td><td colspan="1" rowspan="1">753/755</td></tr><tr><td colspan="1" rowspan="1">Prospective(Frozen)</td><td colspan="1" rowspan="1">100(93.4-100)</td><td colspan="1" rowspan="1">54/54</td><td colspan="1" rowspan="1">99.9(9.4-100)</td><td colspan="1" rowspan="1">984/985</td></tr><tr><td colspan="1" rowspan="1">ProspectiveTotal</td><td colspan="1" rowspan="1">100(95.1-100)</td><td colspan="1" rowspan="1">74/74</td><td colspan="1" rowspan="1">99.8(99.5-99.9)</td><td colspan="1" rowspan="1">1737/1740</td></tr><tr><td colspan="1" rowspan="1">Archived</td><td colspan="1" rowspan="1">100(88.7-100)</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">99.6(98.0-99.9)</td><td colspan="1" rowspan="1">277/278</td></tr><tr><td colspan="1" rowspan="4">Influenza A H1 (subtype)</td><td colspan="1" rowspan="1">Prospective(Fresh)</td><td colspan="1" rowspan="1">NotApplicable</td><td colspan="1" rowspan="1">NotApplicable</td><td colspan="1" rowspan="1">100(99.5-100)</td><td colspan="1" rowspan="1">774/774</td></tr><tr><td colspan="1" rowspan="1">Prospective(Frozen)</td><td colspan="1" rowspan="1">NotApplicable</td><td colspan="1" rowspan="1">NotApplicable</td><td colspan="1" rowspan="1">100(99.6-100)</td><td colspan="1" rowspan="1">1039/1039</td></tr><tr><td colspan="1" rowspan="1">Prospective Total</td><td colspan="1" rowspan="1">NotApplicable</td><td colspan="1" rowspan="1">NotApplicable</td><td colspan="1" rowspan="1">100(99.8-100)</td><td colspan="1" rowspan="1">1813/1813</td></tr><tr><td colspan="1" rowspan="1">Archived</td><td colspan="1" rowspan="1">NotApplicable</td><td colspan="1" rowspan="1">NotApplicable</td><td colspan="1" rowspan="1">99.6(98.0-99.9)</td><td colspan="1" rowspan="1">277/278</td></tr><tr><td colspan="1" rowspan="4">Influenza A H1pdm09(subtype)</td><td colspan="1" rowspan="1">Prospective(Fresh)</td><td colspan="1" rowspan="1">100(70.1-100)</td><td colspan="1" rowspan="1">9/9</td><td colspan="1" rowspan="1">100(99.5-100)</td><td colspan="1" rowspan="1">765/765</td></tr><tr><td colspan="1" rowspan="1">Prospective(Frozen)</td><td colspan="1" rowspan="1">100(85.1-100)</td><td colspan="1" rowspan="1">22/22</td><td colspan="1" rowspan="1">100(99.6-100)</td><td colspan="1" rowspan="1">1017/1017</td></tr><tr><td colspan="1" rowspan="1">ProspectiveTotall</td><td colspan="1" rowspan="1">100(89.0-100)</td><td colspan="1" rowspan="1">31/31</td><td colspan="1" rowspan="1">100(99.8-100)</td><td colspan="1" rowspan="1">1782/1782</td></tr><tr><td colspan="1" rowspan="1">Archived</td><td colspan="1" rowspan="1">96.7(83.3-99.4)</td><td colspan="1" rowspan="1">29/30</td><td colspan="1" rowspan="1">100(98.6-100)</td><td colspan="1" rowspan="1">278/278</td></tr><tr><td colspan="1" rowspan="4">Influenza A H3 (subtype)</td><td colspan="1" rowspan="1">Prospective(Fresh)</td><td colspan="1" rowspan="1">100(74.1-100)</td><td colspan="1" rowspan="1">11/11</td><td colspan="1" rowspan="1">100(99.5-100)</td><td colspan="1" rowspan="1">764/764</td></tr><tr><td colspan="1" rowspan="1">Prospective(Frozen)</td><td colspan="1" rowspan="1">94.4(81.9-98.5)</td><td colspan="1" rowspan="1">34/36</td><td colspan="1" rowspan="1">99.9(99.4-100)</td><td colspan="1" rowspan="1">1002/1003</td></tr><tr><td colspan="1" rowspan="1">ProspectiveTotal</td><td colspan="1" rowspan="1">95.7(85.8-98.8)</td><td colspan="1" rowspan="1">45/47</td><td colspan="1" rowspan="1">99.9(99.7-100)</td><td colspan="1" rowspan="1">1766/1767</td></tr><tr><td colspan="1" rowspan="1">Archived</td><td colspan="1" rowspan="1">NotApplicable</td><td colspan="1" rowspan="1">NotApplicable</td><td colspan="1" rowspan="1">100(98.6-100)</td><td colspan="1" rowspan="1">278/278</td></tr><tr><td colspan="1" rowspan="4">Influenza B</td><td colspan="1" rowspan="1">Prospective(Fresh)</td><td colspan="1" rowspan="1">100(56.6-100)</td><td colspan="1" rowspan="1">5/5</td><td colspan="1" rowspan="1">100(99.5-100)</td><td colspan="1" rowspan="1">770/770</td></tr><tr><td colspan="1" rowspan="1">Prospective(Frozen)</td><td colspan="1" rowspan="1">100(61.0-100)</td><td colspan="1" rowspan="1">6/6</td><td colspan="1" rowspan="1">100(99.6-100)</td><td colspan="1" rowspan="1">1033/1033</td></tr><tr><td colspan="1" rowspan="1">ProspectiveTotal</td><td colspan="1" rowspan="1">100(74.1-100)</td><td colspan="1" rowspan="1">11/11</td><td colspan="1" rowspan="1">100(99.8-100)</td><td colspan="1" rowspan="1">1803/1803</td></tr><tr><td colspan="1" rowspan="1">Archived</td><td colspan="1" rowspan="1">100(88.7-100)</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">100(98.6-100)</td><td colspan="1" rowspan="1">278/278</td></tr><tr><td colspan="1" rowspan="4">Parainfluenza Virus 1</td><td colspan="1" rowspan="1">Prospective(Fresh)</td><td colspan="1" rowspan="1">10064.6-100)</td><td colspan="1" rowspan="1">7/7</td><td colspan="1" rowspan="1">100(99.5-100)</td><td colspan="1" rowspan="1">768/768</td></tr><tr><td colspan="1" rowspan="1">Prospective(Frozen)</td><td colspan="1" rowspan="1">100(74.1-100)</td><td colspan="1" rowspan="1">11/11</td><td colspan="1" rowspan="1">100(99.6-100)</td><td colspan="1" rowspan="1">1028/1028</td></tr><tr><td colspan="1" rowspan="1">ProspectiveTotal</td><td colspan="1" rowspan="1">100(82.4-100)</td><td colspan="1" rowspan="1">18/18</td><td colspan="1" rowspan="1">100(99.8-100)</td><td colspan="1" rowspan="1">1796/1796</td></tr><tr><td colspan="1" rowspan="1">Archived</td><td colspan="1" rowspan="1">100(88.3-100)</td><td colspan="1" rowspan="1">29/29</td><td colspan="1" rowspan="1">100(98.6-100)</td><td colspan="1" rowspan="1">279/279</td></tr><tr><td colspan="1" rowspan="4">Parainfluenza Virus 2</td><td colspan="1" rowspan="1">Prospective(Fresh)</td><td colspan="1" rowspan="1">80.0(37.6-96.4)</td><td colspan="1" rowspan="1">4/5</td><td colspan="1" rowspan="1">100(99.5-100)</td><td colspan="1" rowspan="1">770/770</td></tr><tr><td colspan="1" rowspan="1">Prospective(Frozen)</td><td colspan="1" rowspan="1">100(56.6-100)</td><td colspan="1" rowspan="1">5/5</td><td colspan="1" rowspan="1">100(99.6-100)</td><td colspan="1" rowspan="1">1034/1034</td></tr><tr><td colspan="1" rowspan="1">ProspectiveTotal</td><td colspan="1" rowspan="1">90.0(59.6-98.2)</td><td colspan="1" rowspan="1">9/10</td><td colspan="1" rowspan="1">100(99.8-100)</td><td colspan="1" rowspan="1">1804/1804</td></tr><tr><td colspan="1" rowspan="1">Archived</td><td colspan="1" rowspan="1">100(88.7-100)</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">100(98.6-100)</td><td colspan="1" rowspan="1">278/278</td></tr><tr><td colspan="1" rowspan="4">Parainfluenza Virus 3</td><td colspan="1" rowspan="1">Prospective(Fresh)</td><td colspan="1" rowspan="1">100(83.2-100)</td><td colspan="1" rowspan="1">19/19</td><td colspan="1" rowspan="1">100(99.5-100)</td><td colspan="1" rowspan="1">756/756</td></tr><tr><td colspan="1" rowspan="1">Prospective(Frozen)</td><td colspan="1" rowspan="1">100(85.7-100)</td><td colspan="1" rowspan="1">23/23</td><td colspan="1" rowspan="1">99.9(99.4-100)</td><td colspan="1" rowspan="1">1015/1016</td></tr><tr><td colspan="1" rowspan="1">ProspectiveTotal</td><td colspan="1" rowspan="1">100(91.6-100)</td><td colspan="1" rowspan="1">42/42</td><td colspan="1" rowspan="1">99.9(99.7-100)</td><td colspan="1" rowspan="1">1771/1772</td></tr><tr><td colspan="1" rowspan="1">Archived</td><td colspan="1" rowspan="1">NotApplicable</td><td colspan="1" rowspan="1">NotApplicable</td><td colspan="1" rowspan="1">100(98.8-100)</td><td colspan="1" rowspan="1">307/307</td></tr><tr><td colspan="1" rowspan="3">Parainfluenza Virus 4</td><td colspan="1" rowspan="1">Prospective(Fresh)</td><td colspan="1" rowspan="1">66.7(20.8-93.9)</td><td colspan="1" rowspan="1">2/3</td><td colspan="1" rowspan="1">99.7(99.1-99.9)</td><td colspan="1" rowspan="1">770/772</td></tr><tr><td colspan="1" rowspan="1">Prospective(Frozen)</td><td colspan="1" rowspan="1">91.7(64.6-98.5)</td><td colspan="1" rowspan="1">11/12</td><td colspan="1" rowspan="1">99.9(99.5-100)</td><td colspan="1" rowspan="1">1026/1027</td></tr><tr><td colspan="1" rowspan="1">ProspectiveTotal</td><td colspan="1" rowspan="1">86.7(62.1-96.3)</td><td colspan="1" rowspan="1">13/15</td><td colspan="1" rowspan="1">99.8(99.5-99.9)</td><td colspan="1" rowspan="1">1796/1799</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Archived</td><td colspan="1" rowspan="1">93.8(71.7-98.9)</td><td colspan="1" rowspan="1">15/16</td><td colspan="1" rowspan="1">100(98.7-100)</td><td colspan="1" rowspan="1">292/292</td></tr><tr><td colspan="1" rowspan="4">Respiratory Syncytial Virus A</td><td colspan="1" rowspan="1">Prospective(Fresh)</td><td colspan="1" rowspan="1">100(72.3-100)</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">99.9(99.3-100)</td><td colspan="1" rowspan="1">764/765</td></tr><tr><td colspan="1" rowspan="1">Prospective(Frozen)</td><td colspan="1" rowspan="1">100(92.1-100)</td><td colspan="1" rowspan="1">45/45</td><td colspan="1" rowspan="1">99.8(99.3-99.9)</td><td colspan="1" rowspan="1">992/994</td></tr><tr><td colspan="1" rowspan="1">ProspectiveTTotal</td><td colspan="1" rowspan="1">100(93.5-100)</td><td colspan="1" rowspan="1">55/55</td><td colspan="1" rowspan="1">99.8(99.5-99.9)</td><td colspan="1" rowspan="1">1756/1759</td></tr><tr><td colspan="1" rowspan="1">Archived</td><td colspan="1" rowspan="1">NotApplicable</td><td colspan="1" rowspan="1">NotApplicable</td><td colspan="1" rowspan="1">99.3(97.7-99.8)</td><td colspan="1" rowspan="1">305/307</td></tr><tr><td colspan="1" rowspan="4">Respiratory Syncytial Virus B</td><td colspan="1" rowspan="1">Prospective(Fresh)</td><td colspan="1" rowspan="1">100(43.9-100)</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">100(99.5-100)</td><td colspan="1" rowspan="1">772/772</td></tr><tr><td colspan="1" rowspan="1">Prospective(Frozen)</td><td colspan="1" rowspan="1">100(81.6-100)</td><td colspan="1" rowspan="1">17/17</td><td colspan="1" rowspan="1">100(99.6-100)</td><td colspan="1" rowspan="1">1022/1022</td></tr><tr><td colspan="1" rowspan="1">ProspectiveTotal</td><td colspan="1" rowspan="1">100(83.9-100)</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">100(99.8-100)</td><td colspan="1" rowspan="1">1794/1794</td></tr><tr><td colspan="1" rowspan="1">Archived</td><td colspan="1" rowspan="1">NotApplicable</td><td colspan="1" rowspan="1">NotApplicable</td><td colspan="1" rowspan="1">99.7(98.2-99.9)</td><td colspan="1" rowspan="1">306/307</td></tr><tr><td colspan="1" rowspan="4">Rhinovirus/Enterovirus</td><td colspan="1" rowspan="1">Prospective(Fresh)</td><td colspan="1" rowspan="1">93.2(87.6-96.4)</td><td colspan="1" rowspan="1">123/132</td><td colspan="1" rowspan="1">100(99.4-100)</td><td colspan="1" rowspan="1">643/643</td></tr><tr><td colspan="1" rowspan="1">Prospective(Frozen)</td><td colspan="1" rowspan="1">96.2(92.9-98.0)</td><td colspan="1" rowspan="1">228/237</td><td colspan="1" rowspan="1">99.9(99.3-100)</td><td colspan="1" rowspan="1">801/802</td></tr><tr><td colspan="1" rowspan="1">ProspectiveTotal</td><td colspan="1" rowspan="1">95.1(92.4-96.9)</td><td colspan="1" rowspan="1">351/369</td><td colspan="1" rowspan="1">99.9(99.6-100)</td><td colspan="1" rowspan="1">1444/1445</td></tr><tr><td colspan="1" rowspan="1">Archived</td><td colspan="1" rowspan="1">100(20.7-100)</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">98.7(96.7-99.5)</td><td colspan="1" rowspan="1">302/306</td></tr><tr><td colspan="1" rowspan="4">SARS-CoV-2</td><td colspan="1" rowspan="1">Prospective(Fresh)</td><td colspan="1" rowspan="1">97.2(92.0-99.0)</td><td colspan="1" rowspan="1">103/106</td><td colspan="1" rowspan="1">99.4(98.5-99.8)</td><td colspan="1" rowspan="1">656/660</td></tr><tr><td colspan="1" rowspan="1">Prospective(Frozen)</td><td colspan="1" rowspan="1">98.4(94.5-99.6)</td><td colspan="1" rowspan="1">126/128</td><td colspan="1" rowspan="1">99.2(98.4-99.6)</td><td colspan="1" rowspan="1">902/909</td></tr><tr><td colspan="1" rowspan="1">ProspectiveTotal</td><td colspan="1" rowspan="1">97.9(95.1-99.1)</td><td colspan="1" rowspan="1">229/234</td><td colspan="1" rowspan="1">99.3(98.7-99.6)</td><td colspan="1" rowspan="1">1558/1569</td></tr><tr><td colspan="1" rowspan="1">Archived</td><td colspan="1" rowspan="1">NotApplicable</td><td colspan="1" rowspan="1">NotApplicable</td><td colspan="1" rowspan="1">NotApplicable</td><td colspan="1" rowspan="1">NotApplicable</td></tr></table>

<table><tr><td colspan="6" rowspan="1">Bacteria</td></tr><tr><td colspan="1" rowspan="2">NxTAG RPP v2 Analyte</td><td colspan="1" rowspan="2">Study</td><td colspan="2" rowspan="1">Positive Agreement</td><td colspan="2" rowspan="1">Negative Agreement</td></tr><tr><td colspan="1" rowspan="1">Percent(95% CI)</td><td colspan="1" rowspan="1">TP/(TP+FN)</td><td colspan="1" rowspan="1">Percent(95% CI)</td><td colspan="1" rowspan="1">TN/(TN+FP)</td></tr><tr><td colspan="1" rowspan="3">Chlamydia pneumoniae</td><td colspan="1" rowspan="1">Prospective(Fresh)</td><td colspan="1" rowspan="1">NotApplicable</td><td colspan="1" rowspan="1">NotApplicable</td><td colspan="1" rowspan="1">100(99.5-100)</td><td colspan="1" rowspan="1">775/775</td></tr><tr><td colspan="1" rowspan="1">Prospective(Frozen)</td><td colspan="1" rowspan="1">NotApplicable</td><td colspan="1" rowspan="1">NotApplicable</td><td colspan="1" rowspan="1">100(99.6-100)</td><td colspan="1" rowspan="1">1039/1039</td></tr><tr><td colspan="1" rowspan="1">ProspectiveTotal</td><td colspan="1" rowspan="1">NotApplicable</td><td colspan="1" rowspan="1">NotApplicable</td><td colspan="1" rowspan="1">100(99.8-100)</td><td colspan="1" rowspan="1">1814/1814</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Archived</td><td colspan="1" rowspan="1">100(78.5-100)</td><td colspan="1" rowspan="1">14/14</td><td colspan="1" rowspan="1">99.7(98.1-99.9)</td><td colspan="1" rowspan="1">293/294</td></tr><tr><td colspan="1" rowspan="4">Mycoplasma pneumoniae</td><td colspan="1" rowspan="1">Prospective(Fresh)</td><td colspan="1" rowspan="1">NotApplicable</td><td colspan="1" rowspan="1">NotApplicable</td><td colspan="1" rowspan="1">100(99.5-100)</td><td colspan="1" rowspan="1">775/775</td></tr><tr><td colspan="1" rowspan="1">Prospective(Frozen)</td><td colspan="1" rowspan="1">NotApplicable</td><td colspan="1" rowspan="1">NotApplicable</td><td colspan="1" rowspan="1">100(99.6-100)</td><td colspan="1" rowspan="1">1039/1039</td></tr><tr><td colspan="1" rowspan="1">ProspectiveTotal</td><td colspan="1" rowspan="1">NotApplicable</td><td colspan="1" rowspan="1">NotApplicable</td><td colspan="1" rowspan="1">100(99.8-100)</td><td colspan="1" rowspan="1">1814/1814</td></tr><tr><td colspan="1" rowspan="1">Archived</td><td colspan="1" rowspan="1">92.3(81.8-97.0)</td><td colspan="1" rowspan="1">48/52</td><td colspan="1" rowspan="1">100(98.5-100)</td><td colspan="1" rowspan="1">256/256</td></tr></table>

FN: False Negative; FP: False Positive; TN: True Negative; TP: True Positive

# Contrived Specimens

Because some analytes exhibited low prevalence in the Prospective Clinical Study and insufficient archived positive specimens were available to demonstrate acceptable positive agreement, additional testing was performed with contrived specimens that were prepared by spiking quantified viral or bacterial stocks into unique individual nasopharyngeal swab specimens that were known to be negative for the NxTAG panel analytes (Table 38). The results of the study are described in Table 39 and show acceptable positive and negative agreement.

Table 38. Strains of bacteria and viruses used to prepare contrived specimens   

<table><tr><td rowspan=1 colspan=1>Analyte</td><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>Source</td><td rowspan=1 colspan=1>Catalog No.</td></tr><tr><td rowspan=1 colspan=1>Coronavirus 229E</td><td rowspan=1 colspan=1>229E</td><td rowspan=1 colspan=1>Zeptometrix</td><td rowspan=1 colspan=1>0810229CF</td></tr><tr><td rowspan=1 colspan=1>Influenza A H1</td><td rowspan=1 colspan=1>A/Brisbane/59/07</td><td rowspan=1 colspan=1>Zeptometrix</td><td rowspan=1 colspan=1>0810244CF</td></tr><tr><td rowspan=1 colspan=1>Human Bocavirus</td><td rowspan=1 colspan=1>Recombinant</td><td rowspan=1 colspan=1>Zeptometrix</td><td rowspan=1 colspan=1>0830039</td></tr><tr><td rowspan=1 colspan=1>Respiratory Syncytial Virus B</td><td rowspan=1 colspan=1>CH93(18)-18</td><td rowspan=1 colspan=1>Zeptometrix</td><td rowspan=1 colspan=1>0810040CF</td></tr><tr><td rowspan=1 colspan=1>Chlamydia pneumoniae</td><td rowspan=1 colspan=1>TW-183</td><td rowspan=1 colspan=1>ATCC</td><td rowspan=1 colspan=1>VR-2282</td></tr></table>

Table 39. Results from testing contrived specimens   

<table><tr><td rowspan=1 colspan=4>Viruses</td></tr><tr><td rowspan=2 colspan=1>NxTAG RPP v2 Analyte</td><td rowspan=2 colspan=1>Level</td><td rowspan=1 colspan=2>Agreement</td></tr><tr><td rowspan=1 colspan=1>Percent</td><td rowspan=1 colspan=1>Number</td></tr><tr><td rowspan=1 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>199/199</td></tr><tr><td rowspan=4 colspan=1>Coronavirus 229E</td><td rowspan=1 colspan=1>100X</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>12/12</td></tr><tr><td rowspan=1 colspan=1>10X</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>12/12</td></tr><tr><td rowspan=1 colspan=1>2X</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>25/25</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>149/149</td></tr><tr><td rowspan=1 colspan=1>Coronavirus HKU1</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>199/199</td></tr><tr><td rowspan=1 colspan=1>Coronavirus NL63</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>199/199</td></tr><tr><td rowspan=1 colspan=1>Coronavirus OC43</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>199/199</td></tr><tr><td rowspan=1 colspan=1>Human Metapneumovirus</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>199/199</td></tr><tr><td rowspan=3 colspan=1>Influenza A (matrix)</td><td rowspan=1 colspan=1>10X</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>24/24</td></tr><tr><td rowspan=1 colspan=1>2X</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>26/26</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>149/149</td></tr><tr><td rowspan=3 colspan=1>Influenza A H1 (subtype)</td><td rowspan=1 colspan=1>10X</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>24/24</td></tr><tr><td rowspan=1 colspan=1>2X</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>26/26</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>149/149</td></tr><tr><td rowspan=1 colspan=1>Influenza A H1pdm09 (subtype)</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>199/199</td></tr><tr><td rowspan=1 colspan=1>Influenza A H3 (subtype)</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>199/199</td></tr><tr><td rowspan=1 colspan=1>Influenza B</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>199/199</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza Virus 1</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>199/199</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza Virus 2</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>99.5</td><td rowspan=1 colspan=1>198/199</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza Virus 3</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>199/199</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza Virus 4</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>199/199</td></tr><tr><td rowspan=1 colspan=1>Respiratory Syncytial Virus A</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>199/199</td></tr><tr><td rowspan=4 colspan=1>Respiratory Syncytial Virus B</td><td rowspan=1 colspan=1>100X</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>13/13</td></tr><tr><td rowspan=1 colspan=1>10X</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>13/13</td></tr><tr><td rowspan=1 colspan=1>2X</td><td rowspan=1 colspan=1>96.0</td><td rowspan=1 colspan=1>24/25</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>99.3</td><td rowspan=1 colspan=1>148/149</td></tr><tr><td rowspan=1 colspan=1>Rhinovirus/Enterovirus</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>199/199</td></tr><tr><td rowspan=1 colspan=1>SARS-Coronavirus</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>199/199</td></tr></table>

<table><tr><td rowspan=1 colspan=4>Bacteria</td></tr><tr><td rowspan=2 colspan=1>NxTAG RPP v2 Analyte</td><td rowspan=2 colspan=1>Level</td><td rowspan=1 colspan=2>Agreement</td></tr><tr><td rowspan=1 colspan=1>Percent</td><td rowspan=1 colspan=1>Number</td></tr><tr><td rowspan=4 colspan=1>Chlamydia pneumoniae</td><td rowspan=1 colspan=1>100X</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>12/12</td></tr><tr><td rowspan=1 colspan=1>10X</td><td rowspan=1 colspan=1>92.3</td><td rowspan=1 colspan=1>12/13</td></tr><tr><td rowspan=1 colspan=1>2X</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>25/25</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>149/149</td></tr><tr><td rowspan=1 colspan=1>Mycoplasma pneumoniae</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>199/199</td></tr></table>

# Coinfections

The NxTAG RPP v2 assay reported detection of two or more target analytes from 125/1820 specimens $( 6 . 9 \% )$ in the Prospective Clinical Study (Table 40). Of these, 116 $( 9 2 . 8 \% )$ were positive for two viral pathogens while 9 specimens $( 7 . 2 \% )$ were positive for three different viruses.

# Table 40. Coinfections detected by the NxTAG RPP v2 assay in the Prospective Clinical Study

<table><tr><td rowspan=1 colspan=4>NxTAG RPP v2 Positive Result</td><td rowspan=1 colspan=2>False NxTAG RPP v2 Results vs Comparator</td></tr><tr><td rowspan=1 colspan=1>Analyte 1</td><td rowspan=1 colspan=1>Analyte 2</td><td rowspan=1 colspan=1>Analyte 3</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td></tr><tr><td rowspan=1 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>CoV HKU1</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>--</td><td rowspan=1 colspan=1>--</td></tr><tr><td rowspan=1 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>CoV NL63</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>--</td><td rowspan=1 colspan=1>--</td></tr><tr><td rowspan=1 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>CoV OC43</td><td rowspan=1 colspan=1>hMPV</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>hMPV (1)</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>hMPV</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>Adenovirus (1)</td><td rowspan=1 colspan=1>SARS-CoV-2 (1)</td></tr><tr><td rowspan=1 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>Influenza A H3</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>Influenza A H3 (1)</td><td rowspan=1 colspan=1>--</td></tr><tr><td rowspan=1 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>PIV 2</td><td rowspan=1 colspan=1>hMPV</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>--</td><td rowspan=1 colspan=1>--</td></tr><tr><td rowspan=1 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>PIV 3</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>--</td><td rowspan=1 colspan=1>--</td></tr><tr><td rowspan=1 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>PIV 3</td><td rowspan=1 colspan=1>CoV NL63</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>--</td><td rowspan=1 colspan=1>--</td></tr><tr><td rowspan=1 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>RSV A</td><td rowspan=1 colspan=1>Y</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>--</td><td rowspan=1 colspan=1>--</td></tr><tr><td rowspan=1 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>Rh/EV</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>18</td><td rowspan=1 colspan=1>Adenovirus (2)</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>SARS-CoV-2</td><td rowspan=1 colspan=1>Y</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>--</td><td rowspan=1 colspan=1>--</td></tr><tr><td rowspan=1 colspan=1>CoV 229E</td><td rowspan=1 colspan=1>RSV A</td><td rowspan=1 colspan=1>Y</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>RSV A (1)</td><td rowspan=1 colspan=1>--</td></tr><tr><td rowspan=1 colspan=1>CoV 229E</td><td rowspan=1 colspan=1>Rh/EV</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>Y</td><td rowspan=1 colspan=1>CoV NL63, SARS-CoV-2 (1)</td></tr><tr><td rowspan=1 colspan=1>CoV HKU1</td><td rowspan=1 colspan=1>hMPV</td><td rowspan=1 colspan=1>Y</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>--</td><td rowspan=1 colspan=1>--</td></tr><tr><td rowspan=1 colspan=1>CoV NL63</td><td rowspan=1 colspan=1>CoV HKU1</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>--</td><td rowspan=1 colspan=1>--</td></tr><tr><td rowspan=1 colspan=1>CoV NL63</td><td rowspan=1 colspan=1>CoV OC43</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>--</td><td rowspan=1 colspan=1>--</td></tr><tr><td rowspan=1 colspan=1>CoV NL63</td><td rowspan=1 colspan=1>hMPV</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>--</td><td rowspan=1 colspan=1>--</td></tr><tr><td rowspan=1 colspan=1>CoV NL63</td><td rowspan=1 colspan=1>Rh/EV</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>--</td><td rowspan=1 colspan=1>--</td></tr><tr><td rowspan=1 colspan=1>CoV OC43</td><td rowspan=1 colspan=1>hMPV</td><td rowspan=1 colspan=1>Y</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>--</td><td rowspan=1 colspan=1>--</td></tr><tr><td rowspan=1 colspan=1>CoV OC43</td><td rowspan=1 colspan=1>hMPV</td><td rowspan=1 colspan=1>Rh/EV</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>--</td></tr><tr><td rowspan=1 colspan=1>CoV OC43</td><td rowspan=1 colspan=1>RSV B</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>--</td><td rowspan=1 colspan=1>--</td></tr><tr><td rowspan=1 colspan=1>CoV OC43</td><td rowspan=1 colspan=1>Rh/EV</td><td rowspan=1 colspan=1>Y</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>--</td><td rowspan=1 colspan=1>--</td></tr><tr><td rowspan=1 colspan=1>hMPV</td><td rowspan=1 colspan=1>RSV A</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>hMPV (1)</td><td rowspan=1 colspan=1>--</td></tr><tr><td rowspan=1 colspan=1>hMPV</td><td rowspan=1 colspan=1>Rh/EV</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>hMPV (1)</td><td rowspan=1 colspan=1>Adenovirus (1)</td></tr><tr><td rowspan=1 colspan=1>hMPV</td><td rowspan=1 colspan=1>Rh/EV</td><td rowspan=1 colspan=1>SARS-CoV-2</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>--</td><td rowspan=1 colspan=1>CoV NL63 (1)</td></tr><tr><td rowspan=1 colspan=1>Influenza A H3</td><td rowspan=1 colspan=1>CoV 229E</td><td rowspan=1 colspan=1>Y</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>CoV 229E (1)</td><td rowspan=1 colspan=1>--</td></tr><tr><td rowspan=1 colspan=1>Influenza A H3</td><td rowspan=1 colspan=1>RSV A</td><td rowspan=1 colspan=1>Y</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>RSV A (1)</td><td rowspan=1 colspan=1>--</td></tr><tr><td rowspan=1 colspan=1>Influenza A H3</td><td rowspan=1 colspan=1>Rh/EV</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>--</td><td rowspan=1 colspan=1>--</td></tr><tr><td rowspan=1 colspan=1>Influenza A H3</td><td rowspan=1 colspan=1>SARS-CoV-2</td><td rowspan=1 colspan=1>Rh/EV</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>--</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>Influenza A H1pdm09</td><td rowspan=1 colspan=1>CoV HKU1</td><td rowspan=1 colspan=1>Y</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>--</td><td rowspan=1 colspan=1>--</td></tr><tr><td rowspan=1 colspan=1>Influenza A H1pdm09</td><td rowspan=1 colspan=1>Rh/EV</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>--</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>PIV 1</td><td rowspan=1 colspan=1>Rh/EV</td><td rowspan=1 colspan=1>Y</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>--</td><td rowspan=1 colspan=1>--</td></tr><tr><td rowspan=1 colspan=1>PIV 2</td><td rowspan=1 colspan=1>RSV A</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>--</td><td rowspan=1 colspan=1>--</td></tr><tr><td rowspan=1 colspan=1>PIV 3</td><td rowspan=1 colspan=1>CoV NL63</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>--</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>PIV 3</td><td rowspan=1 colspan=1>CoV NL63</td><td rowspan=1 colspan=1>hMPV</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>hMPV (1)</td><td rowspan=1 colspan=1>--</td></tr><tr><td rowspan=1 colspan=1>PIV 3</td><td rowspan=1 colspan=1>CoV OC43</td><td rowspan=1 colspan=1>Y</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>--</td></tr><tr><td rowspan=1 colspan=1>PIV 3</td><td rowspan=1 colspan=1>hMPV</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>--</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>PIV 3</td><td rowspan=1 colspan=1>RSV B</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>--</td><td rowspan=1 colspan=1>--</td></tr><tr><td rowspan=1 colspan=1>PIV 3</td><td rowspan=1 colspan=1>Rh/EV</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>PIV 3 (1)</td><td rowspan=1 colspan=1>--</td></tr><tr><td rowspan=1 colspan=1>PIV 4</td><td rowspan=1 colspan=1>Rh/EV</td><td rowspan=1 colspan=1>Y</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>PIV 4 (2)</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>PIV 4</td><td rowspan=1 colspan=1>SARS-CoV-2</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>PIV 4 (1)</td><td rowspan=1 colspan=1>--</td></tr><tr><td rowspan=1 colspan=1>PIV 4</td><td rowspan=1 colspan=1>SARS-CoV-2</td><td rowspan=1 colspan=1>Rh/EV</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>--</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>Rh/EV</td><td rowspan=1 colspan=1>RSV A</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>Rh/EV</td><td rowspan=1 colspan=1>RSV B</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>--</td></tr><tr><td rowspan=1 colspan=1>SARS-CoV-2</td><td rowspan=1 colspan=1>CoV HKU1</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>--</td><td rowspan=1 colspan=1>--</td></tr><tr><td rowspan=1 colspan=1>SARS-CoV-2</td><td rowspan=1 colspan=1>CoV OC43</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>SARS-CoV-2 (1)</td><td rowspan=1 colspan=1>--</td></tr><tr><td rowspan=1 colspan=1>SARS-CoV-2</td><td rowspan=1 colspan=1>CoV OC43</td><td rowspan=1 colspan=1>Rh/EV</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>--</td><td rowspan=1 colspan=1>--</td></tr><tr><td rowspan=1 colspan=1>SARS-CoV-2</td><td rowspan=1 colspan=1>RSV A</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>RSV A (1)</td><td rowspan=1 colspan=1>--</td></tr><tr><td rowspan=1 colspan=1>SARS-CoV-2</td><td rowspan=1 colspan=1>Rh/EV</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>SARS-CoV-2 (2)</td><td rowspan=1 colspan=1>--</td></tr></table>

CoV: Coronavirus; hMPV: Human Metapneumovirus; PIV: Parainfluenza Virus; Rh/EV: Rhinovirus/Enterovirus; RSV: Respiratory Syncytial Virus

2. Clinical Specificity: Refer to Section VII C (1), above.

3. Other Clinical Supportive Data (When 1. and 2. Are Not Applicable): Not applicable.

# D Clinical Cut-Off:

Not applicable

# E Expected Values/Reference Range:

The rates of positive results for the NxTAG RPP v2 analytes in the Prospective Clinical Study described in Section VII C are presented in Tables 41 and 42 stratified by study site and patient age, respectively.

Table 41. Expected values and prevalence of analytes in prospectively collected nasopharyngeal swab specimens, as determined by the NxTAG RPP v2 and comparator methods, respectively, stratified by site   

<table><tr><td colspan="1" rowspan="2">NxTAG RPP v2Analyte</td><td colspan="1" rowspan="2">Test Method</td><td colspan="6" rowspan="1">Percent Positive Stratified by Study Site</td></tr><tr><td colspan="1" rowspan="1">All</td><td colspan="1" rowspan="1">Site 1</td><td colspan="1" rowspan="1">Site 2</td><td colspan="1" rowspan="1">Site 3</td><td colspan="1" rowspan="1">Site 4</td><td colspan="1" rowspan="1">Site 5</td></tr><tr><td colspan="1" rowspan="2">Adenovirus</td><td colspan="1" rowspan="1">NxTAG</td><td colspan="1" rowspan="1">5.6(101/1814)</td><td colspan="1" rowspan="1">8.7(39/446)</td><td colspan="1" rowspan="1">0(0/40)</td><td colspan="1" rowspan="1">6.2(51/822)</td><td colspan="1" rowspan="1">3.0(10/334)</td><td colspan="1" rowspan="1">0.6(1/172)</td></tr><tr><td colspan="1" rowspan="1">Comparator</td><td colspan="1" rowspan="1">5.4(98/1814)</td><td colspan="1" rowspan="1">8.7(39/446)</td><td colspan="1" rowspan="1">0(0/40)</td><td colspan="1" rowspan="1">5.8(48/822)</td><td colspan="1" rowspan="1">3.0(10/334)</td><td colspan="1" rowspan="1">0.6(1/172)</td></tr><tr><td colspan="1" rowspan="2">Coronavirus HKU1</td><td colspan="1" rowspan="1">NxTAG</td><td colspan="1" rowspan="1">1.1(20/1814)</td><td colspan="1" rowspan="1">1.3(6/446)</td><td colspan="1" rowspan="1">0(0/40)</td><td colspan="1" rowspan="1">0.7(6/822)</td><td colspan="1" rowspan="1">1.5(5/334)</td><td colspan="1" rowspan="1">1.7(3/172)</td></tr><tr><td colspan="1" rowspan="1">Comparator</td><td colspan="1" rowspan="1">1.2(21/1814</td><td colspan="1" rowspan="1">1.3(6/446)</td><td colspan="1" rowspan="1">0(0/40)</td><td colspan="1" rowspan="1">0.9(7/822)</td><td colspan="1" rowspan="1">1.5(5/334)</td><td colspan="1" rowspan="1">1.7(3/172)</td></tr><tr><td colspan="1" rowspan="2">Coronavirus NL63</td><td colspan="1" rowspan="1">NxTAG</td><td colspan="1" rowspan="1">2.6(48/1814)</td><td colspan="1" rowspan="1">5.2(23/446)</td><td colspan="1" rowspan="1">5.0(2/40)</td><td colspan="1" rowspan="1">0.9(7/822)</td><td colspan="1" rowspan="1">3.0(13//334)</td><td colspan="1" rowspan="1">1.7(3/172)</td></tr><tr><td colspan="1" rowspan="1">Comparator</td><td colspan="1" rowspan="1">2.9(52/1814)</td><td colspan="1" rowspan="1">5.6(25/446)</td><td colspan="1" rowspan="1">5.0(2/40)</td><td colspan="1" rowspan="1">0.9(7/822)</td><td colspan="1" rowspan="1">4.2(14/334)</td><td colspan="1" rowspan="1">2.3(4/172)</td></tr><tr><td colspan="1" rowspan="2">Coronavirus OC43</td><td colspan="1" rowspan="1">NxTAG</td><td colspan="1" rowspan="1">2.1(39/1814)</td><td colspan="1" rowspan="1">4.5(20/446)</td><td colspan="1" rowspan="1">0(0/40)</td><td colspan="1" rowspan="1">1.3(11/822)</td><td colspan="1" rowspan="1">2.4(8/334)</td><td colspan="1" rowspan="1">0(0/172)</td></tr><tr><td colspan="1" rowspan="1">Comparator</td><td colspan="1" rowspan="1">2.1(39/1814)</td><td colspan="1" rowspan="1">4.5(20/446)</td><td colspan="1" rowspan="1">0(0/40)</td><td colspan="1" rowspan="1">1.3(11/822))</td><td colspan="1" rowspan="1">2.4(8/334)</td><td colspan="1" rowspan="1">0(0/172)</td></tr><tr><td colspan="1" rowspan="2">Coronavirus 229E</td><td colspan="1" rowspan="1">NxTAG</td><td colspan="1" rowspan="1">0.4(8/1814)</td><td colspan="1" rowspan="1">0.2(1/446)</td><td colspan="1" rowspan="1">0(0/40)</td><td colspan="1" rowspan="1">0.6(5/822)</td><td colspan="1" rowspan="1">0.3(1/334)</td><td colspan="1" rowspan="1">0.6(1/172)</td></tr><tr><td colspan="1" rowspan="1">Comparator</td><td colspan="1" rowspan="1">0.4(7/1814)</td><td colspan="1" rowspan="1">0.2(1/446)</td><td colspan="1" rowspan="1">0(0/40)</td><td colspan="1" rowspan="1">0.5(4/822)</td><td colspan="1" rowspan="1">0.3(1/334)</td><td colspan="1" rowspan="1">0.6(1/172)</td></tr><tr><td colspan="1" rowspan="2">Influenza A (matrix)</td><td colspan="1" rowspan="1">NxTAG</td><td colspan="1" rowspan="1">4.2(77/1814)</td><td colspan="1" rowspan="1">0.9(4/446)</td><td colspan="1" rowspan="1">0(0/40)</td><td colspan="1" rowspan="1">7.7(63/822)</td><td colspan="1" rowspan="1">2.7(9/334)</td><td colspan="1" rowspan="1">0.6(1/172)</td></tr><tr><td colspan="1" rowspan="1">Comparator</td><td colspan="1" rowspan="1">4.1(74/1814)</td><td colspan="1" rowspan="1">0.9(4/446)</td><td colspan="1" rowspan="1">0(0/40)</td><td colspan="1" rowspan="1">7.3(60/822)</td><td colspan="1" rowspan="1">2.7(9/334)</td><td colspan="1" rowspan="1">0.6(1/172)</td></tr><tr><td colspan="1" rowspan="2">Influenza A H1(subtype)</td><td colspan="1" rowspan="1">NxTAG</td><td colspan="1" rowspan="1">0(0/1813)</td><td colspan="1" rowspan="1">0(0/446)</td><td colspan="1" rowspan="1">0(0/40)</td><td colspan="1" rowspan="1">0(0/821)</td><td colspan="1" rowspan="1">0(0//334)</td><td colspan="1" rowspan="1">0(0/172)</td></tr><tr><td colspan="1" rowspan="1">Comparator</td><td colspan="1" rowspan="1">0(0/1813)</td><td colspan="1" rowspan="1">0(0/446)</td><td colspan="1" rowspan="1">0(0/40)</td><td colspan="1" rowspan="1">0(0/821)</td><td colspan="1" rowspan="1">0(0/334)</td><td colspan="1" rowspan="1">0(0/172)</td></tr><tr><td colspan="1" rowspan="2">Influenza AH1pdm09 (subtype)</td><td colspan="1" rowspan="1">NxTAG</td><td colspan="1" rowspan="1">1.7(31/1813)</td><td colspan="1" rowspan="1">0.7(3/446)</td><td colspan="1" rowspan="1">0(0/40)</td><td colspan="1" rowspan="1">2.3(19/821)</td><td colspan="1" rowspan="1">2.4(8/334)</td><td colspan="1" rowspan="1">0.6(1/172)</td></tr><tr><td colspan="1" rowspan="1">Comparator</td><td colspan="1" rowspan="1">1.7(31/1813)</td><td colspan="1" rowspan="1">0.7(3/446)</td><td colspan="1" rowspan="1">0(0/40)</td><td colspan="1" rowspan="1">2.3(19/821)</td><td colspan="1" rowspan="1">2.4(8/334)</td><td colspan="1" rowspan="1">0.6(1/172)</td></tr><tr><td colspan="1" rowspan="2">Influenza A H3(subtype)</td><td colspan="1" rowspan="1">NxTAG</td><td colspan="1" rowspan="1">2.5(46/1814)</td><td colspan="1" rowspan="1">0.4(2/446)</td><td colspan="1" rowspan="1">0(0/40)</td><td colspan="1" rowspan="1">5.4(44/822)</td><td colspan="1" rowspan="1">0(0/334)</td><td colspan="1" rowspan="1">0(0/172)</td></tr><tr><td colspan="1" rowspan="1">Comparator</td><td colspan="1" rowspan="1">2.6(47/1814)</td><td colspan="1" rowspan="1">0.7(3/446)</td><td colspan="1" rowspan="1">0(0/40)</td><td colspan="1" rowspan="1">5.4(44/822)</td><td colspan="1" rowspan="1">0(0/334)</td><td colspan="1" rowspan="1">0(0/172)</td></tr><tr><td colspan="1" rowspan="2">Influenza B</td><td colspan="1" rowspan="1">NxTAG</td><td colspan="1" rowspan="1">0.6(11/1814)</td><td colspan="1" rowspan="1">0(0/446)</td><td colspan="1" rowspan="1">0(0/40)</td><td colspan="1" rowspan="1">1.3(11/822)</td><td colspan="1" rowspan="1">0(0/334)</td><td colspan="1" rowspan="1">0(0/172)</td></tr><tr><td colspan="1" rowspan="1">Comparator</td><td colspan="1" rowspan="1">0.6(11/1814)</td><td colspan="1" rowspan="1">0(0/446)</td><td colspan="1" rowspan="1">0(0/40)</td><td colspan="1" rowspan="1">1.3(11/822)</td><td colspan="1" rowspan="1">0(0/334)</td><td colspan="1" rowspan="1">0(0/172)</td></tr><tr><td colspan="1" rowspan="2">HumanMetapneumovirus</td><td colspan="1" rowspan="1">NxTAG</td><td colspan="1" rowspan="1">9.4(171/1814)</td><td colspan="1" rowspan="1">17.3(77/446)</td><td colspan="1" rowspan="1">2.5(1/40)</td><td colspan="1" rowspan="1">4.6(38/822)</td><td colspan="1" rowspan="1">7.8(26/334)</td><td colspan="1" rowspan="1">16.9(29/172)</td></tr><tr><td colspan="1" rowspan="1">Comparator</td><td colspan="1" rowspan="1">8.7(157/1814)</td><td colspan="1" rowspan="1">15.7(70/446)</td><td colspan="1" rowspan="1">2.5(1/40)</td><td colspan="1" rowspan="1">4.5(37/822)</td><td colspan="1" rowspan="1">6.6(22/334)</td><td colspan="1" rowspan="1">15.7(27/172)</td></tr><tr><td colspan="1" rowspan="2">Parainfluenza Virus 1</td><td colspan="1" rowspan="1">NxTAG</td><td colspan="1" rowspan="1">1.0(18/1814)</td><td colspan="1" rowspan="1">1.6(7/446)</td><td colspan="1" rowspan="1">0(0/40)</td><td colspan="1" rowspan="1">1.1(9/822)</td><td colspan="1" rowspan="1">0(0/334)</td><td colspan="1" rowspan="1">1.2(2/172)</td></tr><tr><td colspan="1" rowspan="1">Comparator</td><td colspan="1" rowspan="1">1.0</td><td colspan="1" rowspan="1">1.6</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1.1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1.2</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">(18/1814)</td><td colspan="1" rowspan="1">(7/446)</td><td colspan="1" rowspan="1">(0/40)</td><td colspan="1" rowspan="1">(9/822)</td><td colspan="1" rowspan="1">(0/334)</td><td colspan="1" rowspan="1">(2/172)</td></tr><tr><td colspan="1" rowspan="2">Parainfluenza Virus 2</td><td colspan="1" rowspan="1">NxTAG</td><td colspan="1" rowspan="1">0.5(9/1814)</td><td colspan="1" rowspan="1">0(0/446)</td><td colspan="1" rowspan="1">0(0/40)</td><td colspan="1" rowspan="1">0.5(4/822)</td><td colspan="1" rowspan="1">1.5(5/334)</td><td colspan="1" rowspan="1">0(0/172)</td></tr><tr><td colspan="1" rowspan="1">Comparator</td><td colspan="1" rowspan="1">0.6(10/1814)</td><td colspan="1" rowspan="1">0(0/446)</td><td colspan="1" rowspan="1">0(0/40)</td><td colspan="1" rowspan="1">0.5(4/822)</td><td colspan="1" rowspan="1">1.8(6/334)</td><td colspan="1" rowspan="1">0(0/172)</td></tr><tr><td colspan="1" rowspan="2">Parainfluenza Virus 3</td><td colspan="1" rowspan="1">NxTAG</td><td colspan="1" rowspan="1">2.4(43/1814)</td><td colspan="1" rowspan="1">4.9(22/446)</td><td colspan="1" rowspan="1">0(0/40)</td><td colspan="1" rowspan="1">1.6(13/822)</td><td colspan="1" rowspan="1">0.9(3/334)</td><td colspan="1" rowspan="1">2.9(5/172)</td></tr><tr><td colspan="1" rowspan="1">Comparator</td><td colspan="1" rowspan="1">2.3(42/1814)</td><td colspan="1" rowspan="1">4.9(22/446)</td><td colspan="1" rowspan="1">0(0/40)</td><td colspan="1" rowspan="1">1.5(12/822)</td><td colspan="1" rowspan="1">0.9(3/334)</td><td colspan="1" rowspan="1">2.9(5/172)</td></tr><tr><td colspan="1" rowspan="2">Parainfluenza Virus 4</td><td colspan="1" rowspan="1">NxTAG</td><td colspan="1" rowspan="1">0.9(16/1814)</td><td colspan="1" rowspan="1">0.4(2/446)</td><td colspan="1" rowspan="1">0(0/40)</td><td colspan="1" rowspan="1">1.6(13/822)</td><td colspan="1" rowspan="1">0.3(1/334)</td><td colspan="1" rowspan="1">0(0/172)</td></tr><tr><td colspan="1" rowspan="1">Comparator</td><td colspan="1" rowspan="1">0.8(15/1814)</td><td colspan="1" rowspan="1">0.4(2/446)</td><td colspan="1" rowspan="1">0(0/40)</td><td colspan="1" rowspan="1">1.5(12/822)</td><td colspan="1" rowspan="1">0.3(1/334)</td><td colspan="1" rowspan="1">0(0/172)</td></tr><tr><td colspan="1" rowspan="2">Respiratory SyncytialVirus A</td><td colspan="1" rowspan="1">NxTAG</td><td colspan="1" rowspan="1">3.2(58/1814)</td><td colspan="1" rowspan="1">3.6(16/446)</td><td colspan="1" rowspan="1">0(0/40)</td><td colspan="1" rowspan="1">4.5(37/822)</td><td colspan="1" rowspan="1">1.5(5/334)</td><td colspan="1" rowspan="1">0(0/172)</td></tr><tr><td colspan="1" rowspan="1">Comparator</td><td colspan="1" rowspan="1">3.0(55/1814)</td><td colspan="1" rowspan="1">3.6(16/446)</td><td colspan="1" rowspan="1">0(0/40)</td><td colspan="1" rowspan="1">4.1(34/822)</td><td colspan="1" rowspan="1">1.5(5/334)</td><td colspan="1" rowspan="1">0(0/172)</td></tr><tr><td colspan="1" rowspan="2">Respiratory SyncytialVirus B</td><td colspan="1" rowspan="1">NxTAG</td><td colspan="1" rowspan="1">1.1(20/1814)</td><td colspan="1" rowspan="1">2.2(10/446)</td><td colspan="1" rowspan="1">0(0/40)</td><td colspan="1" rowspan="1">0.7(6/822)</td><td colspan="1" rowspan="1">0.9(3/334)</td><td colspan="1" rowspan="1">0.6(1/172)</td></tr><tr><td colspan="1" rowspan="1">Comparator</td><td colspan="1" rowspan="1">1.1(20/1814)</td><td colspan="1" rowspan="1">2.2(10/446)</td><td colspan="1" rowspan="1">0(0/40)</td><td colspan="1" rowspan="1">0.7(6/822)</td><td colspan="1" rowspan="1">0.9(3/334)</td><td colspan="1" rowspan="1">0.6(1/172)</td></tr><tr><td colspan="1" rowspan="2">Rhinovirus /Enterovirus</td><td colspan="1" rowspan="1">NxTAG</td><td colspan="1" rowspan="1">19.4(352/1814)</td><td colspan="1" rowspan="1">30.5(136/446)</td><td colspan="1" rowspan="1">2.5(1/40)</td><td colspan="1" rowspan="1">18.0(148/822)</td><td colspan="1" rowspan="1">12.6(42/334)</td><td colspan="1" rowspan="1">14.5(25/172)</td></tr><tr><td colspan="1" rowspan="1">Comparator</td><td colspan="1" rowspan="1">20.3(369/1814)</td><td colspan="1" rowspan="1">30.7(137/446)</td><td colspan="1" rowspan="1">5.0(2/40)</td><td colspan="1" rowspan="1">19.1(157/822)</td><td colspan="1" rowspan="1">13.2(44/334)</td><td colspan="1" rowspan="1">16.9(29/172)</td></tr><tr><td colspan="1" rowspan="2">SARS-CoV-2</td><td colspan="1" rowspan="1">NxTAG</td><td colspan="1" rowspan="1">13.3(240/1803)</td><td colspan="1" rowspan="1">5.4(24/444)</td><td colspan="1" rowspan="1">60.0(24/40)</td><td colspan="1" rowspan="1">13.6(112/824)</td><td colspan="1" rowspan="1">17.9(59/329)</td><td colspan="1" rowspan="1">12.7(21/166)</td></tr><tr><td colspan="1" rowspan="1">Comparator</td><td colspan="1" rowspan="1">13.0(234/1803)</td><td colspan="1" rowspan="1">5.2(23/444)</td><td colspan="1" rowspan="1">60.0(24/40)</td><td colspan="1" rowspan="1">13.0(107/824)</td><td colspan="1" rowspan="1">17.9(59/329)</td><td colspan="1" rowspan="1">12.7(21/166)</td></tr><tr><td colspan="1" rowspan="2">Chlamydiapneumoniae</td><td colspan="1" rowspan="1">NxTAG</td><td colspan="1" rowspan="1">0(0/1814)</td><td colspan="1" rowspan="1">0(0/446)</td><td colspan="1" rowspan="1">0(0/40)</td><td colspan="1" rowspan="1">0(0/822)</td><td colspan="1" rowspan="1">0(0/334)</td><td colspan="1" rowspan="1">0(0/172)</td></tr><tr><td colspan="1" rowspan="1">Comparator</td><td colspan="1" rowspan="1">0(0/1814)</td><td colspan="1" rowspan="1">0(0/446)</td><td colspan="1" rowspan="1">0(0/40)</td><td colspan="1" rowspan="1">0(0/822)</td><td colspan="1" rowspan="1">0(0/334)</td><td colspan="1" rowspan="1">0(0/172)</td></tr><tr><td colspan="1" rowspan="2">Mycoplasmapneumoniae</td><td colspan="1" rowspan="1">NxTAG</td><td colspan="1" rowspan="1">0(0/1814)</td><td colspan="1" rowspan="1">0(0/446)</td><td colspan="1" rowspan="1">0(0/40)</td><td colspan="1" rowspan="1">0(0/822)</td><td colspan="1" rowspan="1">0(0/334)</td><td colspan="1" rowspan="1">0(0/172)</td></tr><tr><td colspan="1" rowspan="1">Comparator</td><td colspan="1" rowspan="1">0(0/1814)</td><td colspan="1" rowspan="1">0(0/446)</td><td colspan="1" rowspan="1">0(0/40)</td><td colspan="1" rowspan="1">0(0/822)</td><td colspan="1" rowspan="1">0(0/334)</td><td colspan="1" rowspan="1">0(0/172)</td></tr></table>

Table 42. Expected values and prevalence of analytes in prospectively collected nasopharyngeal swab specimens, as determined by the NxTAG RPP v2 and comparator methods, respectively, stratified by patient age   

<table><tr><td colspan="1" rowspan="2">NxTAG RPP v2Analyte</td><td colspan="1" rowspan="2">TestMethod</td><td colspan="7" rowspan="1">Number Positive (%) Stratified by Patient Age (years)</td></tr><tr><td colspan="1" rowspan="1">All</td><td colspan="1" rowspan="1">0 to 1</td><td colspan="1" rowspan="1">&gt;1 to 5</td><td colspan="1" rowspan="1">&gt; 5 to 21</td><td colspan="1" rowspan="1">&gt; 21 to 65</td><td colspan="1" rowspan="1">&gt; 65</td><td colspan="1" rowspan="1">Unknown</td></tr><tr><td colspan="1" rowspan="2">Adenovirus</td><td colspan="1" rowspan="1">NxTAG</td><td colspan="1" rowspan="1">5.6(101/1814)</td><td colspan="1" rowspan="1">10.1(34/337)</td><td colspan="1" rowspan="1">15.8(40/253)</td><td colspan="1" rowspan="1">6.7(22/329)</td><td colspan="1" rowspan="1">0.8(5/641)</td><td colspan="1" rowspan="1">0(0/242)</td><td colspan="1" rowspan="1">0(0/12)</td></tr><tr><td colspan="1" rowspan="1">Comparator</td><td colspan="1" rowspan="1">5.4(98/1814</td><td colspan="1" rowspan="1">9.5(32/337)</td><td colspan="1" rowspan="1">15.0(38/253)</td><td colspan="1" rowspan="1">7.0(23/329)</td><td colspan="1" rowspan="1">0.8(5/641)</td><td colspan="1" rowspan="1">0(0/242)</td><td colspan="1" rowspan="1">0(0/12)</td></tr><tr><td colspan="1" rowspan="2">Coronavirus HKU1</td><td colspan="1" rowspan="1">NxTAG</td><td colspan="1" rowspan="1">1.1(20/1814)</td><td colspan="1" rowspan="1">0.9(3/337)</td><td colspan="1" rowspan="1">2.0(5/253)</td><td colspan="1" rowspan="1">1.2(4/329)</td><td colspan="1" rowspan="1">1.2(8/641)</td><td colspan="1" rowspan="1">0(0/242)</td><td colspan="1" rowspan="1">0(0/12)</td></tr><tr><td colspan="1" rowspan="1">Comparator</td><td colspan="1" rowspan="1">1.2(21/1814</td><td colspan="1" rowspan="1">0.9(3/337)</td><td colspan="1" rowspan="1">2.0(5/253)</td><td colspan="1" rowspan="1">1.2(4/329)</td><td colspan="1" rowspan="1">1.2(8/641)</td><td colspan="1" rowspan="1">0.4(1/242)</td><td colspan="1" rowspan="1">0(0/12)</td></tr><tr><td colspan="1" rowspan="2">Coronavirus NL63</td><td colspan="1" rowspan="1">NxTAG</td><td colspan="1" rowspan="1">2.6(48/1814)</td><td colspan="1" rowspan="1">5.0(17/337)</td><td colspan="1" rowspan="1">4.0(12/253)</td><td colspan="1" rowspan="1">2.4(8/329)</td><td colspan="1" rowspan="1">1.4(9/641)</td><td colspan="1" rowspan="1">0.8(2/242)</td><td colspan="1" rowspan="1">0(0/12)</td></tr><tr><td colspan="1" rowspan="1">Comparator</td><td colspan="1" rowspan="1">2.9(52/1814)</td><td colspan="1" rowspan="1">5.0(17/337)</td><td colspan="1" rowspan="1">5.1(13/253)</td><td colspan="1" rowspan="1">2.7(9/329)</td><td colspan="1" rowspan="1">1.6(10/641)</td><td colspan="1" rowspan="1">0.8(2/242)</td><td colspan="1" rowspan="1">8.3(1/12)</td></tr><tr><td colspan="1" rowspan="2">Coronavirus OC43</td><td colspan="1" rowspan="1">NxTAG</td><td colspan="1" rowspan="1">2.1(39/1814)</td><td colspan="1" rowspan="1">4.7(16/337)</td><td colspan="1" rowspan="1">3.2(8/253)</td><td colspan="1" rowspan="1">2.1(7/329)</td><td colspan="1" rowspan="1">0.9(6/641)</td><td colspan="1" rowspan="1">0.8(2/242)</td><td colspan="1" rowspan="1">0(0/12)</td></tr><tr><td colspan="1" rowspan="1">Comparator</td><td colspan="1" rowspan="1">2.1(39/1814)</td><td colspan="1" rowspan="1">4.7(16/337)</td><td colspan="1" rowspan="1">3.2(8/253)</td><td colspan="1" rowspan="1">2.1(7/329)</td><td colspan="1" rowspan="1">0.9(6/641)</td><td colspan="1" rowspan="1">0.8(2/242)</td><td colspan="1" rowspan="1">0(0/12)</td></tr><tr><td colspan="1" rowspan="2">Coronavirus 229E</td><td colspan="1" rowspan="1">NxTAG</td><td colspan="1" rowspan="1">0.4(8/1814)</td><td colspan="1" rowspan="1">0.3(1/337)</td><td colspan="1" rowspan="1">0.4(1/253)</td><td colspan="1" rowspan="1">0(0/329)</td><td colspan="1" rowspan="1">0.8(5/641)</td><td colspan="1" rowspan="1">0.4(1/242)</td><td colspan="1" rowspan="1">0(0/12)</td></tr><tr><td colspan="1" rowspan="1">Comparator</td><td colspan="1" rowspan="1">0.4(7/1814)</td><td colspan="1" rowspan="1">0.3(1/337)</td><td colspan="1" rowspan="1">0.4(1/253)</td><td colspan="1" rowspan="1">0(0/329)</td><td colspan="1" rowspan="1">0.8(5/641)</td><td colspan="1" rowspan="1">0(0/242)</td><td colspan="1" rowspan="1">0(0/12)</td></tr><tr><td colspan="1" rowspan="1">Influenza A (matrix)</td><td colspan="1" rowspan="1">NxTAG</td><td colspan="1" rowspan="1">4.2</td><td colspan="1" rowspan="1">2.1</td><td colspan="1" rowspan="1">3.2</td><td colspan="1" rowspan="1">7.6</td><td colspan="1" rowspan="1">4.2</td><td colspan="1" rowspan="1">4.1</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="2">NxTAG RPP v2Analyte</td><td colspan="1" rowspan="2">TestMthod</td><td colspan="7" rowspan="1">Number Positive (%) Stratified by Patient Age (years)</td></tr><tr><td colspan="1" rowspan="1">All</td><td colspan="1" rowspan="1">0 to 1</td><td colspan="1" rowspan="1">&gt; 1 to 5</td><td colspan="1" rowspan="1">&gt; 5 to 21</td><td colspan="1" rowspan="1">&gt; 21 to 65</td><td colspan="1" rowspan="1">&gt;65</td><td colspan="1" rowspan="1">Unknown</td></tr><tr><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">(77/1814)</td><td colspan="1" rowspan="1">(7/337)</td><td colspan="1" rowspan="1">(8/253)</td><td colspan="1" rowspan="1">(25/329)</td><td colspan="1" rowspan="1">(27/641)</td><td colspan="1" rowspan="1">(10/242)</td><td colspan="1" rowspan="1">(0/12)</td></tr><tr><td colspan="1" rowspan="1">Comparator</td><td colspan="1" rowspan="1">4.1(74/1814)</td><td colspan="1" rowspan="1">2.1(7/337)</td><td colspan="1" rowspan="1">3.2(8/253</td><td colspan="1" rowspan="1">7.6(25/329)</td><td colspan="1" rowspan="1">3.9(25/641)</td><td colspan="1" rowspan="1">3.7(9/242)</td><td colspan="1" rowspan="1">0(0/12)</td></tr><tr><td colspan="1" rowspan="2">Influenza A H1uubtype)</td><td colspan="1" rowspan="1">NxTAG</td><td colspan="1" rowspan="1">0(0/1813)</td><td colspan="1" rowspan="1">0(0/337)</td><td colspan="1" rowspan="1">0(0/253)</td><td colspan="1" rowspan="1">0(0/328)</td><td colspan="1" rowspan="1">0(0/641)</td><td colspan="1" rowspan="1">0(0/242)</td><td colspan="1" rowspan="1">0(0/12)</td></tr><tr><td colspan="1" rowspan="1">Comparator</td><td colspan="1" rowspan="1">0(0/1813)</td><td colspan="1" rowspan="1">0(0/337)</td><td colspan="1" rowspan="1">0(0/253)</td><td colspan="1" rowspan="1">0(0/328)</td><td colspan="1" rowspan="1">0(0/641)</td><td colspan="1" rowspan="1">0(0/242)</td><td colspan="1" rowspan="1">0(0/12)</td></tr><tr><td colspan="1" rowspan="2">Influenza AH1 pdm09 (subtype)</td><td colspan="1" rowspan="1">NxTAG</td><td colspan="1" rowspan="1">1.7(31/1813)</td><td colspan="1" rowspan="1">0.9(3/337)</td><td colspan="1" rowspan="1">1.2(3/253)</td><td colspan="1" rowspan="1">2.4(8/328)</td><td colspan="1" rowspan="1">2.2(14/641)</td><td colspan="1" rowspan="1">1.2(3/242)</td><td colspan="1" rowspan="1">0(0/12)</td></tr><tr><td colspan="1" rowspan="1">Comparator</td><td colspan="1" rowspan="1">1.7(31/1813)</td><td colspan="1" rowspan="1">0.9(3/337)</td><td colspan="1" rowspan="1">1.2(3/253)</td><td colspan="1" rowspan="1">2.4(8/328)</td><td colspan="1" rowspan="1">2.2(14/641)</td><td colspan="1" rowspan="1">1.2(3/242)</td><td colspan="1" rowspan="1">0(0/12)</td></tr><tr><td colspan="1" rowspan="2">Influenza A H3ubbtype)</td><td colspan="1" rowspan="1">NxTAG</td><td colspan="1" rowspan="1">2.5(46/1814)</td><td colspan="1" rowspan="1">1.2(4/337)</td><td colspan="1" rowspan="1">2.4(6/253)</td><td colspan="1" rowspan="1">5.5(18/329)</td><td colspan="1" rowspan="1">1.7(11/641)</td><td colspan="1" rowspan="1">2.9(7/242)</td><td colspan="1" rowspan="1">(0/12)</td></tr><tr><td colspan="1" rowspan="1">Comparator</td><td colspan="1" rowspan="1">2.6(47/1814)</td><td colspan="1" rowspan="1">1.3(5/337)</td><td colspan="1" rowspan="1">2.0(5/253)</td><td colspan="1" rowspan="1">5.5(18/329)</td><td colspan="1" rowspan="1">1.7(11/641)</td><td colspan="1" rowspan="1">3.3(8/242)</td><td colspan="1" rowspan="1">0(0/12)</td></tr><tr><td colspan="1" rowspan="2">Influenza B</td><td colspan="1" rowspan="1">NxTAG</td><td colspan="1" rowspan="1">0.6(11/1814)</td><td colspan="1" rowspan="1">0(0/337)</td><td colspan="1" rowspan="1">1.2(3/253)</td><td colspan="1" rowspan="1">0.9(3/329)</td><td colspan="1" rowspan="1">0.8(5/641)</td><td colspan="1" rowspan="1">0(0/242)</td><td colspan="1" rowspan="1">0(0/12)</td></tr><tr><td colspan="1" rowspan="1">Comparator</td><td colspan="1" rowspan="1">0.6(11/1814)</td><td colspan="1" rowspan="1">0(0/337)</td><td colspan="1" rowspan="1">1.2(3/253)</td><td colspan="1" rowspan="1">0.9(3/329)</td><td colspan="1" rowspan="1">0.8(5/641)</td><td colspan="1" rowspan="1">0(0/242)</td><td colspan="1" rowspan="1">0(0/12)</td></tr><tr><td colspan="1" rowspan="2">HumanMetapneumovirus</td><td colspan="1" rowspan="1">NxTAG</td><td colspan="1" rowspan="1">9.4(171/1814)</td><td colspan="1" rowspan="1">13.4(45/337)</td><td colspan="1" rowspan="1">16.2(41/253)</td><td colspan="1" rowspan="1">8.8(29/329)</td><td colspan="1" rowspan="1">6.1(39/641)</td><td colspan="1" rowspan="1">5.8(14/242)</td><td colspan="1" rowspan="1">312</td></tr><tr><td colspan="1" rowspan="1">Comparator</td><td colspan="1" rowspan="1">8.7(157/1814)</td><td colspan="1" rowspan="1">11.9(40/337)</td><td colspan="1" rowspan="1">15.4(39/253)</td><td colspan="1" rowspan="1">8.2(27/329)</td><td colspan="1" rowspan="1">5.9(38/641)</td><td colspan="1" rowspan="1">4.1(10/242)</td><td colspan="1" rowspan="1">25.0(3/12)</td></tr><tr><td colspan="1" rowspan="2">Parainfluenza VVirus 1</td><td colspan="1" rowspan="1">NxTAG</td><td colspan="1" rowspan="1">1.0(18/1814)</td><td colspan="1" rowspan="1">1.8(6/337)</td><td colspan="1" rowspan="1">1.2(3/253)</td><td colspan="1" rowspan="1">0.3(1/329)</td><td colspan="1" rowspan="1">1.1(7/641)</td><td colspan="1" rowspan="1">0.4(1/242)</td><td colspan="1" rowspan="1">0(0/12)</td></tr><tr><td colspan="1" rowspan="1">Comparator</td><td colspan="1" rowspan="1">1.0(18/1814)</td><td colspan="1" rowspan="1">1.8(6/337)</td><td colspan="1" rowspan="1">1.2(3/253)</td><td colspan="1" rowspan="1">0.3(1/329)</td><td colspan="1" rowspan="1">1.1(7/641)</td><td colspan="1" rowspan="1">0.4(1/242)</td><td colspan="1" rowspan="1">0(0/12)</td></tr><tr><td colspan="1" rowspan="2">ParainfluenzaVirus 2</td><td colspan="1" rowspan="1">NxTAG</td><td colspan="1" rowspan="1">0.5(9/1814)</td><td colspan="1" rowspan="1">0.3(1/337)</td><td colspan="1" rowspan="1">0.4(1/253)</td><td colspan="1" rowspan="1">0.9(3/329)</td><td colspan="1" rowspan="1">0.2(1/641)</td><td colspan="1" rowspan="1">1.2(3/242)</td><td colspan="1" rowspan="1">0(0/12)</td></tr><tr><td colspan="1" rowspan="1">Comparator</td><td colspan="1" rowspan="1">0.6(10/1814)</td><td colspan="1" rowspan="1">0.3(1/337)</td><td colspan="1" rowspan="1">0.8(2/253)</td><td colspan="1" rowspan="1">0.9(3/329)</td><td colspan="1" rowspan="1">0.2(1/641)</td><td colspan="1" rowspan="1">1.2(3/242)</td><td colspan="1" rowspan="1">0(0/12)</td></tr><tr><td colspan="1" rowspan="2">ParainfluenzaVirus 3</td><td colspan="1" rowspan="1">NxTAG</td><td colspan="1" rowspan="1">2.4(43/1814)</td><td colspan="1" rowspan="1">4.5(15/337)</td><td colspan="1" rowspan="1">6.3(16/253)</td><td colspan="1" rowspan="1">1.5(5/329)</td><td colspan="1" rowspan="1">1.1(7/641)</td><td colspan="1" rowspan="1">0(0/242)</td><td colspan="1" rowspan="1">0(0/12)</td></tr><tr><td colspan="1" rowspan="1">Comparator</td><td colspan="1" rowspan="1">2.3(42/1814)</td><td colspan="1" rowspan="1">4.2(14/337)</td><td colspan="1" rowspan="1">6.3(16/253)</td><td colspan="1" rowspan="1">1.5(5/329)</td><td colspan="1" rowspan="1">1.1(7/641)</td><td colspan="1" rowspan="1">0(0/242)</td><td colspan="1" rowspan="1">0(0/12)</td></tr><tr><td colspan="1" rowspan="2">ParainfluenzaVirus 4</td><td colspan="1" rowspan="1">NxTAG</td><td colspan="1" rowspan="1">0.9(16/1814)</td><td colspan="1" rowspan="1">2.1(7/337)</td><td colspan="1" rowspan="1">0.4(1/253)</td><td colspan="1" rowspan="1">1.2(4/329)</td><td colspan="1" rowspan="1">0.6(4/641)</td><td colspan="1" rowspan="1">0(0/242)</td><td colspan="1" rowspan="1">0(0/12)</td></tr><tr><td colspan="1" rowspan="1">Comparator</td><td colspan="1" rowspan="1">0.8(15/1814)</td><td colspan="1" rowspan="1">2.1(7/337)</td><td colspan="1" rowspan="1">0.4(1/253)</td><td colspan="1" rowspan="1">1.2(4/329)</td><td colspan="1" rowspan="1">0.5(3/641)</td><td colspan="1" rowspan="1">0(0/242)</td><td colspan="1" rowspan="1">O(0/12)</td></tr><tr><td colspan="1" rowspan="2">RespiratorySyncytial Virus A</td><td colspan="1" rowspan="1">NxTAG</td><td colspan="1" rowspan="1">3.2(58/1814)</td><td colspan="1" rowspan="1">8.3(28/337)</td><td colspan="1" rowspan="1">3.6(9/253)</td><td colspan="1" rowspan="1">0.6(2/329)</td><td colspan="1" rowspan="1">1.7(11/641)</td><td colspan="1" rowspan="1">3.3(8/242)</td><td colspan="1" rowspan="1">0(0/12)</td></tr><tr><td colspan="1" rowspan="1">Comparator</td><td colspan="1" rowspan="1">3.0(55/1814)</td><td colspan="1" rowspan="1">8.3(28/337)</td><td colspan="1" rowspan="1">3.6(9/253)</td><td colspan="1" rowspan="1">0.6(2/329)</td><td colspan="1" rowspan="1">1.2(8/641)</td><td colspan="1" rowspan="1">3.3(8/242)</td><td colspan="1" rowspan="1">O(0/12)</td></tr><tr><td colspan="1" rowspan="2">RespiratorySyncytial Virus B</td><td colspan="1" rowspan="1">NxTAG</td><td colspan="1" rowspan="1">1.1(20/1814)</td><td colspan="1" rowspan="1">3.0(10/337)</td><td colspan="1" rowspan="1">1.6(4/253)</td><td colspan="1" rowspan="1">0.3(1/329)</td><td colspan="1" rowspan="1">0.6(4/641)</td><td colspan="1" rowspan="1">0.4(1/242)</td><td colspan="1" rowspan="1">$(02</td></tr><tr><td colspan="1" rowspan="1">Comparator</td><td colspan="1" rowspan="1">1.1(20/1814)</td><td colspan="1" rowspan="1">3.0(10/337)</td><td colspan="1" rowspan="1">1.6(4/253)</td><td colspan="1" rowspan="1">0.3(1/329)</td><td colspan="1" rowspan="1">0.6(4/641)</td><td colspan="1" rowspan="1">0.4(1/242)</td><td colspan="1" rowspan="1">0(0/12)</td></tr><tr><td colspan="1" rowspan="2">Rhinovirus /Enterovirus</td><td colspan="1" rowspan="1">NxTAG</td><td colspan="1" rowspan="1">19.4(352/1814)</td><td colspan="1" rowspan="1">33.5(113/337)</td><td colspan="1" rowspan="1">29.6(75/253)</td><td colspan="1" rowspan="1">22.8(75/329)</td><td colspan="1" rowspan="1">10.1(65/641)</td><td colspan="1" rowspan="1">8.3(20/242)</td><td colspan="1" rowspan="1">33.3(4/12)</td></tr><tr><td colspan="1" rowspan="1">Comparator</td><td colspan="1" rowspan="1">20.3(369/1814)</td><td colspan="1" rowspan="1">34.7(117/337)</td><td colspan="1" rowspan="1">31.2(79/253)</td><td colspan="1" rowspan="1">23.7(78/329)</td><td colspan="1" rowspan="1">10.8(69/641)</td><td colspan="1" rowspan="1">8.7(21/242)</td><td colspan="1" rowspan="1">41.7(5/12)</td></tr><tr><td colspan="1" rowspan="2">SARS-CoV-2</td><td colspan="1" rowspan="1">NxTAG</td><td colspan="1" rowspan="1">13.3(240/1803)</td><td colspan="1" rowspan="1">9.5(32/336)</td><td colspan="1" rowspan="1">4.8(12/251)</td><td colspan="1" rowspan="1">6.5(21/325)</td><td colspan="1" rowspan="1">18.4(118/640)</td><td colspan="1" rowspan="1">23.3(56/240)</td><td colspan="1" rowspan="1">9.1(1/11)</td></tr><tr><td colspan="1" rowspan="1">Comparator</td><td colspan="1" rowspan="1">13.0(234/1803)</td><td colspan="1" rowspan="1">8.9(30/336)</td><td colspan="1" rowspan="1">4.8(12/251)</td><td colspan="1" rowspan="1">6.5(21/325)</td><td colspan="1" rowspan="1">18.4(118/640)</td><td colspan="1" rowspan="1">21.7(52/240)</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="2">Chlamydiapneumoniae</td><td colspan="1" rowspan="1">NxTAG</td><td colspan="1" rowspan="1">0(0/1814)</td><td colspan="1" rowspan="1">0(0/337)</td><td colspan="1" rowspan="1">0(0/253)</td><td colspan="1" rowspan="1">0(0/329)</td><td colspan="1" rowspan="1">0(0/641)</td><td colspan="1" rowspan="1">0(0/242)</td><td colspan="1" rowspan="1">0(0/12)</td></tr><tr><td colspan="1" rowspan="1">Comparator</td><td colspan="1" rowspan="1">0(0/1814)</td><td colspan="1" rowspan="1">0(0/337)</td><td colspan="1" rowspan="1">0(0/253)</td><td colspan="1" rowspan="1">0(0/329)</td><td colspan="1" rowspan="1">0(0/641)</td><td colspan="1" rowspan="1">0(0/242)</td><td colspan="1" rowspan="1">0(0/12)</td></tr><tr><td colspan="1" rowspan="2">Mycoplasmapneumoniae</td><td colspan="1" rowspan="1">NxTAG</td><td colspan="1" rowspan="1">0(0/1814)</td><td colspan="1" rowspan="1">0(0/337)</td><td colspan="1" rowspan="1">0(0/253)</td><td colspan="1" rowspan="1">0(0/329)</td><td colspan="1" rowspan="1">0(0/641)</td><td colspan="1" rowspan="1">0(0/242)</td><td colspan="1" rowspan="1">0(0/12)</td></tr><tr><td colspan="1" rowspan="1">Comparator</td><td colspan="1" rowspan="1">0(0/1814)</td><td colspan="1" rowspan="1">0(0/337)</td><td colspan="1" rowspan="1">0(0/253)</td><td colspan="1" rowspan="1">0(0/329)</td><td colspan="1" rowspan="1">0(0/641)</td><td colspan="1" rowspan="1">0(0/242)</td><td colspan="1" rowspan="1">0(0/12)</td></tr></table>

# F Other Supportive Instrument Performance Characteristics Data:

Not applicable.

# VIII Proposed Labeling:

The labeling supports the finding of substantial equivalence for this device.

# IX Conclusion:

The submitted information in this premarket notification is complete and supports a substantial equivalence decision.